---
stable_id: R-HSA-6799959
display_name: PGLYRP3,4 binds bacterial peptidoglycan
species: Homo sapiens
summary: Peptidoglycan recognition proteins (PGRPs or PGLYRPs) are innate immunity
  molecules that contain a conserved peptidoglycan‑binding type 2 amidase domain that
  is homologous to bacteriophage and bacterial type 2 amidases (Kang D et al. 1998;
  Liu C et al. 2001; Royet J and Dziarski R 2007; Royet J et al. 2011; Dziarski R
  et al. 2016). Mammals have a family of four PGRPs (PGLYRP1, 2, 3 & 4) that are differentially
  expressed in a cell-type or tissue-specific manner. Human PGLYRP3 and PGLYRP4 (also
  known as PGRP‑Ialpha and PGRP-Ibeta) are expressed in keratinocytes and epithelial
  cells and are found in the skin, eyes, salivary glands, throat, tongue, esophagus,
  stomach, and intestine (Liu C et al. 2001; Lu X et al. 2006). Like PGLYRP1, PGLYRP3
  and PGLYRP4 are secreted as disulfide‑linked homodimers (Guan R et al. 2004, 2005;
  Lu X et al. 2006). However, PGLYRP3 and PGLYRP4 preferentially form heterodimers
  when coexpressed in the same cells (Lu X et al. 2006). PGLYRP3, PGLYRP4, and PGLYRP3:PGLYRP4
  have Zn(2+)‑dependent bactericidal activity against both Gram‑positive and Gram‑negative
  bacteria at the physiological Zn(2+) concentrations found in serum, sweat, saliva,
  and other body fluids (Lu X et al. 2006; Wang M et al. 2007). PGLYRP3 and PGLYRP4
  are also active against Chlamydia trachomatis (Bobrovsky P et al. 2016). Killing
  of both Gram-positive and Gram-negative bacteria by PGLYRP3 and PGLYRP4 is synergistically
  enhanced by antimicrobial peptides (phospholipase A2, alpha- and beta-defensins,
  and bactericidal permeability-increasing protein (BPI)) (Wang M et al. 2007). The
  bactericidal activity of PGRPs requires their N‑glycosylation, as deglycosylation
  with N‑glycosidase abolished their bactericidal activity against Bacillus subtilis,
  Staphylococcus aureus, and Escherichia coli (Lu X et al. 2006; Wang M et al. 2007).<p>PGRPs
  are thought to kill bacteria by interacting with cell wall peptidoglycan and by
  inducing lethal stress response in bacteria, as opposed to the hydrolysis of peptidoglycan
  or permeabilizing bacterial membranes seen with other antibacterial peptides (Lu
  X et al. 2006; Wang M et al. 2007; Cho S et al. 2007; Kashyap DR et al. 2011, 2014).
  In Gram‑positive bacteria, including B. subtilis, PGLYRP1, 3 & 4 were found to enter
  the bacterial cell wall at the site of daughter cell separation during cell division
  and to bind to cell wall peptidoglycan in the vicinity of the cell membrane (Kashyap
  DR et al. 2011). Binding of PGLYRP3 to peptidoglycan induces a structural change
  in PGLYRP3 that locks peptidoglycan in the protein's bindings groove (Guan R et
  al. 2006). However, this binding did not inhibit the extracellular transglycosylation
  or transpeptidation steps in peptidoglycan synthesis (Kashyap DR et al. 2011), which
  are well-known targets for bactericidal antibiotics. Instead, this interaction of
  PGRP proteins with peptidoglycan activated the B. subtilis envelope stress response
  two‑component system CssR‑CssS that detects and disposes of misfolded proteins exported
  out of bacterial cells. This activation resulted in bacterial membrane depolarization,
  cessation of intracellular peptidoglycan, protein, RNA, and DNA synthesis and production
  of toxic hydroxyl radicals, which were responsible for bacterial death (Kashyap
  DR et al. 2011). In Gram‑negative bacteria (E. coli), PGRPs were found to bind the
  bacterial outer membrane activating the functionally homologous CpxA‑CpxR two‑component
  system (Kashyap DR et al. 2011). Genome expression arrays, qRT‑PCR, and biochemical
  tests showed that PGLYRP3 & 4 kill both E. coli and B. subtilis by inducing oxidative,
  thiol, and metal stress (Kashyap DR et al. 2014). <br>
---

# PGLYRP3,4 binds bacterial peptidoglycan
**Reactome ID:** [R-HSA-6799959](https://reactome.org/content/detail/R-HSA-6799959)
**Species:** Homo sapiens

## Summary

Peptidoglycan recognition proteins (PGRPs or PGLYRPs) are innate immunity molecules that contain a conserved peptidoglycan‑binding type 2 amidase domain that is homologous to bacteriophage and bacterial type 2 amidases (Kang D et al. 1998; Liu C et al. 2001; Royet J and Dziarski R 2007; Royet J et al. 2011; Dziarski R et al. 2016). Mammals have a family of four PGRPs (PGLYRP1, 2, 3 & 4) that are differentially expressed in a cell-type or tissue-specific manner. Human PGLYRP3 and PGLYRP4 (also known as PGRP‑Ialpha and PGRP-Ibeta) are expressed in keratinocytes and epithelial cells and are found in the skin, eyes, salivary glands, throat, tongue, esophagus, stomach, and intestine (Liu C et al. 2001; Lu X et al. 2006). Like PGLYRP1, PGLYRP3 and PGLYRP4 are secreted as disulfide‑linked homodimers (Guan R et al. 2004, 2005; Lu X et al. 2006). However, PGLYRP3 and PGLYRP4 preferentially form heterodimers when coexpressed in the same cells (Lu X et al. 2006). PGLYRP3, PGLYRP4, and PGLYRP3:PGLYRP4 have Zn(2+)‑dependent bactericidal activity against both Gram‑positive and Gram‑negative bacteria at the physiological Zn(2+) concentrations found in serum, sweat, saliva, and other body fluids (Lu X et al. 2006; Wang M et al. 2007). PGLYRP3 and PGLYRP4 are also active against Chlamydia trachomatis (Bobrovsky P et al. 2016). Killing of both Gram-positive and Gram-negative bacteria by PGLYRP3 and PGLYRP4 is synergistically enhanced by antimicrobial peptides (phospholipase A2, alpha- and beta-defensins, and bactericidal permeability-increasing protein (BPI)) (Wang M et al. 2007). The bactericidal activity of PGRPs requires their N‑glycosylation, as deglycosylation with N‑glycosidase abolished their bactericidal activity against Bacillus subtilis, Staphylococcus aureus, and Escherichia coli (Lu X et al. 2006; Wang M et al. 2007).<p>PGRPs are thought to kill bacteria by interacting with cell wall peptidoglycan and by inducing lethal stress response in bacteria, as opposed to the hydrolysis of peptidoglycan or permeabilizing bacterial membranes seen with other antibacterial peptides (Lu X et al. 2006; Wang M et al. 2007; Cho S et al. 2007; Kashyap DR et al. 2011, 2014). In Gram‑positive bacteria, including B. subtilis, PGLYRP1, 3 & 4 were found to enter the bacterial cell wall at the site of daughter cell separation during cell division and to bind to cell wall peptidoglycan in the vicinity of the cell membrane (Kashyap DR et al. 2011). Binding of PGLYRP3 to peptidoglycan induces a structural change in PGLYRP3 that locks peptidoglycan in the protein's bindings groove (Guan R et al. 2006). However, this binding did not inhibit the extracellular transglycosylation or transpeptidation steps in peptidoglycan synthesis (Kashyap DR et al. 2011), which are well-known targets for bactericidal antibiotics. Instead, this interaction of PGRP proteins with peptidoglycan activated the B. subtilis envelope stress response two‑component system CssR‑CssS that detects and disposes of misfolded proteins exported out of bacterial cells. This activation resulted in bacterial membrane depolarization, cessation of intracellular peptidoglycan, protein, RNA, and DNA synthesis and production of toxic hydroxyl radicals, which were responsible for bacterial death (Kashyap DR et al. 2011). In Gram‑negative bacteria (E. coli), PGRPs were found to bind the bacterial outer membrane activating the functionally homologous CpxA‑CpxR two‑component system (Kashyap DR et al. 2011). Genome expression arrays, qRT‑PCR, and biochemical tests showed that PGLYRP3 & 4 kill both E. coli and B. subtilis by inducing oxidative, thiol, and metal stress (Kashyap DR et al. 2014). <br>
