---
provider: perplexity
model: sonar-deep-research
cached: false
start_time: '2025-11-25T19:22:57.810339'
end_time: '2025-11-25T19:25:00.504490'
duration_seconds: 122.69
template_file: templates/rule_research.md
template_variables:
  rule_id: ARBA00028836
  rule_type: arba
  go_terms: GO:0043604 (amide biosynthetic process)
  conditions_summary: '**Condition Set 1:** InterPro id: TRAM/LAG1/CLN8 homology domain
    (InterPro:IPR006634) AND InterPro id: Sphingosine N-acyltransferase Lag1/Lac1-like
    (InterPro:IPR016439) AND taxon: Chordata (NCBITaxon:7711)


    **Condition Set 2:** InterPro id: Thiamin diphosphate-binding fold (InterPro:IPR029061)
    AND taxon: Thermoproteota (NCBITaxon:28889)


    **Condition Set 3:** InterPro id: Glutamate-cysteine ligase catalytic subunit
    (InterPro:IPR004308) AND InterPro id: Glutamine synthetase/guanido kinase, catalytic
    domain (InterPro:IPR014746) AND PANTHER id: PTHR11164:SF0


    **Condition Set 4:** InterPro id: Dethiobiotin synthase BioD (InterPro:IPR004472)
    AND InterPro id: P-loop containing nucleoside triphosphate hydrolase (InterPro:IPR027417)


    **Condition Set 5:** InterPro id: DNA/pantothenate metabolism flavoprotein, C-terminal
    (InterPro:IPR007085) AND InterPro id: CoaB-like superfamily (InterPro:IPR035929)


    **Condition Set 6:** InterPro id: Biotin/lipoyl attachment (InterPro:IPR000089)
    AND InterPro id: Dihydrolipoamide acetyltransferase/Pyruvate dehydrogenase protein
    X component (InterPro:IPR045257) AND taxon: Euarchontoglires (NCBITaxon:314146)


    **Condition Set 7:** InterPro id: UDP-glucuronosyl/UDP-glucosyltransferase (InterPro:IPR002213)
    AND InterPro id: Glycosyltransferase family 28, N-terminal domain (InterPro:IPR004276)
    AND taxon: Bacteria (NCBITaxon:2)


    **Condition Set 8:** InterPro id: Pantoate-beta-alanine ligase (InterPro:IPR003721)
    AND InterPro id: Rossmann-like alpha/beta/alpha sandwich fold (InterPro:IPR014729)
    AND InterPro id: Pantoate-beta-alanine ligase, C-terminal domain (InterPro:IPR042176)


    **Condition Set 9:** InterPro id: Calcineurin-like, phosphoesterase domain (InterPro:IPR004843)
    AND InterPro id: Saposin B type domain (InterPro:IPR008139) AND InterPro id: Sphingomyelin
    phosphodiesterase, C-terminal domain (InterPro:IPR045473)


    **Condition Set 10:** FunFam id: Ceramide synthase 5 (CATH.FunFam:1.10.10.60:FF:000020)
    AND taxon: Metazoa (NCBITaxon:33208)


    **Condition Set 11:** FunFam id: Glutamate--cysteine ligase catalytic subunit
    (CATH.FunFam:3.30.590.50:FF:000002) AND taxon: Eukaryota (NCBITaxon:2759)


    **Condition Set 12:** FunFam id: 2-oxoglutarate ferredoxin oxidoreductase subunit
    alpha (CATH.FunFam:3.40.50.920:FF:000009) AND FunFam id: 2-oxoglutarate ferredoxin
    oxidoreductase alpha subunit (CATH.FunFam:3.40.50.970:FF:000022) AND taxon: Archaea
    (NCBITaxon:2157)


    **Condition Set 13:** FunFam id: 2-oxoglutarate ferredoxin oxidoreductase subunit
    beta (CATH.FunFam:3.40.50.970:FF:000049) AND taxon: Thermoproteota (NCBITaxon:28889)


    **Condition Set 14:** FunFam id: Acetyltransferase component of pyruvate dehydrogenase
    complex (CATH.FunFam:2.40.50.100:FF:000010) AND FunFam id: Acetyltransferase component
    of pyruvate dehydrogenase complex (CATH.FunFam:3.30.559.10:FF:000003) AND taxon:
    Euarchontoglires (NCBITaxon:314146)


    **Condition Set 15:** FunFam id: Sterol 3-beta-glucosyltransferase UGT80A2 (CATH.FunFam:3.40.50.2000:FF:000009)
    AND FunFam id: Glycosyltransferase GtfE (CATH.FunFam:3.40.50.2000:FF:000292) AND
    taxon: Bacteria (NCBITaxon:2)


    **Condition Set 16:** FunFam id: Biotin synthase (CATH.FunFam:3.20.20.70:FF:000011)


    **Condition Set 17:** FunFam id: Phosphopantothenate--cysteine ligase 2 (CATH.FunFam:3.40.50.10300:FF:000002)


    **Condition Set 18:** FunFam id: Glutathione gamma-glutamylcysteinyltransferase
    1 (CATH.FunFam:3.90.70.30:FF:000001)


    **Condition Set 19:** FunFam id: O-methyltransferase ZRP4 (CATH.FunFam:1.10.10.10:FF:000292)
    AND FunFam id: O-methyltransferase ZRP4 (CATH.FunFam:3.40.50.150:FF:000057) AND
    taxon: Viridiplantae (NCBITaxon:33090)


    **Condition Set 20:** FunFam id: Acyl-coenzyme A:6-aminopenicillanic-acid-acyltransferase
    40 kDa form (CATH.FunFam:1.10.10.2120:FF:000002) AND FunFam id: Acyl-coenzyme
    A:6-aminopenicillanic-acid-acyltransferase 40 kDa form (CATH.FunFam:3.60.60.10:FF:000010)
    AND taxon: Fungi (NCBITaxon:4751)


    **Condition Set 21:** FunFam id: Carbamoyl-phosphate synthase large chain (CATH.FunFam:1.10.1030.10:FF:000001)
    AND FunFam id: Carbamoyl-phosphate synthase large chain (CATH.FunFam:3.30.1490.20:FF:000001)
    AND FunFam id: Carbamoyl-phosphate synthase 1, mitochondrial (CATH.FunFam:3.30.470.20:FF:000030)


    **Condition Set 22:** FunFam id: Nonribosomal peptide synthetase 1 (CATH.FunFam:1.10.1200.10:FF:000005)
    AND FunFam id: Non-ribosomal peptide synthase (CATH.FunFam:1.10.1200.10:FF:000016)
    AND FunFam id: N-(5-amino-5-carboxypentanoyl)-L-cysteinyl-D-valine synthase (CATH.FunFam:3.30.559.30:FF:000069)


    **Condition Set 23:** FunFam id: Dihydrolipoyl dehydrogenase (CATH.FunFam:3.30.390.30:FF:000001)
    AND FunFam id: Dihydrolipoyl dehydrogenase (CATH.FunFam:3.50.50.60:FF:000025)
    AND FunFam id: Dihydrolipoyl dehydrogenase, mitochondrial (CATH.FunFam:3.50.50.60:FF:000221)


    **Condition Set 24:** FunFam id: Carnosine synthase 1 (CATH.FunFam:3.30.470.20:FF:000040)
    AND FunFam id: Carnosine synthase 1 (CATH.FunFam:3.40.50.20:FF:000018) AND taxon:
    Chordata (NCBITaxon:7711)


    **Condition Set 25:** FunFam id: N-acylsphingosine amidohydrolase 2 (CATH.FunFam:2.60.40.2300:FF:000001)
    AND taxon: Mammalia (NCBITaxon:40674)


    **Condition Set 26:** FunFam id: Fatty acid 2-hydroxylase (CATH.FunFam:3.10.120.10:FF:000011)
    AND taxon: Craniata (NCBITaxon:89593)


    **Condition Set 27:** FunFam id: Glutamate--cysteine ligase regulatory subunit
    (CATH.FunFam:3.20.20.100:FF:000012) AND taxon: Vertebrata (NCBITaxon:7742)


    **Condition Set 28:** FunFam id: pantothenate kinase 2, mitochondrial (CATH.FunFam:3.30.420.510:FF:000001)
    AND taxon: Glires (NCBITaxon:314147)


    **Condition Set 29:** FunFam id: serine palmitoyltransferase 1 isoform X1 (CATH.FunFam:3.40.640.10:FF:000049)
    AND taxon: Euteleostomi (NCBITaxon:117571)


    **Condition Set 30:** FunFam id: Sphingomyelin phosphodiesterase (CATH.FunFam:3.60.21.10:FF:000045)
    AND taxon: Eutheria (NCBITaxon:9347)


    **Condition Set 31:** FunFam id: Pantothenate synthetase (CATH.FunFam:3.30.1300.10:FF:000001)


    **Condition Set 32:** FunFam id: Glutathione synthetase (CATH.FunFam:3.40.50.1760:FF:000001)


    **Condition Set 33:** FunFam id: Pantothenate synthetase (CATH.FunFam:3.40.50.620:FF:000013)


    **Condition Set 34:** FunFam id: CMP-N-acetylneuraminate-beta-galactosamide- alpha-2,3-sialyltransferase
    2 (CATH.FunFam:3.90.1480.20:FF:000002)


    **Condition Set 35:** FunFam id: Acetylserotonin O-methyltransferase (CATH.FunFam:1.10.10.10:FF:000358)
    AND FunFam id: Acetylserotonin O-methyltransferase (CATH.FunFam:3.40.50.150:FF:000146)
    AND taxon: Rodentia (NCBITaxon:9989)


    **Condition Set 36:** FunFam id: ATP-citrate synthase subunit 1 (CATH.FunFam:1.10.230.10:FF:000005)
    AND FunFam id: ATP-citrate synthase subunit (CATH.FunFam:3.40.50.261:FF:000003)
    AND FunFam id: ATP-citrate synthase beta chain protein (CATH.FunFam:3.40.50.720:FF:000136)


    **Condition Set 37:** FunFam id: Argininosuccinate lyase (CATH.FunFam:1.10.275.10:FF:000002)
    AND FunFam id: Argininosuccinate lyase (CATH.FunFam:1.10.40.30:FF:000001) AND
    taxon: Taphrinomycotina (NCBITaxon:451866)


    **Condition Set 38:** FunFam id: SPE3p Spermidine synthase (CATH.FunFam:2.30.140.10:FF:000001)
    AND FunFam id: Spermidine synthase (CATH.FunFam:3.40.50.150:FF:000013) AND taxon:
    Dikarya (NCBITaxon:451864)


    **Condition Set 39:** FunFam id: GTP cyclohydrolase 1 (CATH.FunFam:3.30.1130.10:FF:000007)
    AND FunFam id: GTP cyclohydrolase 1 (CATH.FunFam:3.30.1130.10:FF:000008) AND taxon:
    Streptophyta (NCBITaxon:35493)


    **Condition Set 40:** FunFam id: Acetyl-coenzyme A synthetase (CATH.FunFam:3.30.300.30:FF:000004)
    AND FunFam id: Acetyl-coenzyme A synthetase (CATH.FunFam:3.40.50.12780:FF:000001)
    AND taxon: Saccharomyces (NCBITaxon:4930)


    **Condition Set 41:** FunFam id: Nonribosomal siderophore peptide synthase SidC
    (CATH.FunFam:3.30.300.30:FF:000033) AND FunFam id: Nonribosomal siderophore peptide
    synthase SidC (CATH.FunFam:3.40.50.12780:FF:000024) AND taxon: Ascomycota (NCBITaxon:4890)


    **Condition Set 42:** FunFam id: ATP-citrate synthase alpha chain protein (CATH.FunFam:3.30.470.110:FF:000002)
    AND FunFam id: ATP-citrate synthase alpha chain protein (CATH.FunFam:3.40.50.261:FF:000008)
    AND taxon: Embryophyta (NCBITaxon:3193)


    **Condition Set 43:** FunFam id: N-fatty-acyl-amino acid synthase/hydrolase PM20D1
    (CATH.FunFam:1.10.150.900:FF:000003) AND FunFam id: N-fatty-acyl-amino acid synthase/hydrolase
    PM20D1 (CATH.FunFam:3.40.630.10:FF:000027)


    **Condition Set 44:** FunFam id: Carboxypeptidase Y (CATH.FunFam:1.10.287.410:FF:000001)
    AND taxon: Saccharomycotina (NCBITaxon:147537)


    **Condition Set 45:** FunFam id: Patatin like phospholipase domain containing
    1 (CATH.FunFam:3.40.1090.10:FF:000014) AND FunFam id: Patatin like phospholipase
    domain containing 1 (CATH.FunFam:3.40.1090.10:FF:000016)


    **Condition Set 46:** FunFam id: long-chain-fatty-acid--CoA ligase 6 isoform X2
    (CATH.FunFam:3.40.50.12780:FF:000006) AND taxon: Rattus (NCBITaxon:10114)


    **Condition Set 47:** FunFam id: Pimeloyl-[acyl-carrier protein] methyl ester
    esterase (CATH.FunFam:3.40.50.1820:FF:000045) AND taxon: Pseudomonadati (NCBITaxon:3379134)


    **Condition Set 48:** FunFam id: Phosphopantothenoylcysteine decarboxylase (CATH.FunFam:3.40.50.1950:FF:000004)
    AND taxon: malvids (NCBITaxon:91836)


    **Condition Set 49:** FunFam id: ATP-dependent dethiobiotin synthetase BioD (CATH.FunFam:3.40.50.300:FF:000292)
    AND taxon: Pseudomonadota (NCBITaxon:1224)


    **Condition Set 50:** FunFam id: Pyruvate dehydrogenase E1 component subunit alpha,
    mitochondrial (CATH.FunFam:3.40.50.970:FF:000020) AND taxon: Muridae (NCBITaxon:10066)


    **Condition Set 51:** FunFam id: Putative acetyltransferase NSI (CATH.FunFam:3.40.630.30:FF:000059)
    AND taxon: Liliopsida (NCBITaxon:4447)


    **Condition Set 52:** FunFam id: Arginase (CATH.FunFam:3.40.800.10:FF:000005)
    AND taxon: Schizosaccharomycetes (NCBITaxon:147554)


    **Condition Set 53:** FunFam id: Beta-1,4-galactosyltransferase 6 (CATH.FunFam:3.90.550.10:FF:000037)
    AND taxon: Mus (NCBITaxon:862507)


    **Condition Set 54:** FunFam id: Ceramide glucosyltransferase 1 (CATH.FunFam:3.90.550.10:FF:000207)
    AND taxon: Ecdysozoa (NCBITaxon:1206794)


    **Condition Set 55:** FunFam id: Nudix hydrolase 11 (CATH.FunFam:3.90.79.10:FF:000036)
    AND taxon: Tracheophyta (NCBITaxon:58023)


    **Condition Set 56:** FunFam id: sphingomyelin synthase-related protein 1 isoform
    X1 (CATH.FunFam:1.10.150.50:FF:000037)


    **Condition Set 57:** FunFam id: Glutathione synthetase (CATH.FunFam:3.30.1490.50:FF:000002)


    **Condition Set 58:** FunFam id: Coenzyme A biosynthesis bifunctional protein
    CoaBC (CATH.FunFam:3.40.50.10300:FF:000001)


    **Condition Set 59:** FunFam id: Pantothenate kinase (CATH.FunFam:3.40.50.300:FF:000242)


    **Condition Set 60:** FunFam id: Phosphopantetheine adenylyltransferase (CATH.FunFam:3.40.50.620:FF:000012)


    **Condition Set 61:** FunFam id: Adenosylmethionine-8-amino-7-oxononanoate aminotransferase
    (CATH.FunFam:3.40.640.10:FF:000041)


    *These condition sets are OR-ed together (any match triggers the annotation)*'
  protein_count: 0 total (0 reviewed, 0 unreviewed)
provider_config:
  timeout: 600
  max_retries: 3
  parameters:
    allowed_domains: []
    reasoning_effort: medium
    search_domain_filter: []
    return_citations: true
    temperature: 0.0
citation_count: 57
---

## Question

# UniProt Rule Research

## Rule Context

- **Rule ID:** ARBA00028836
- **Rule Type:** arba
- **GO Term(s) Predicted:** GO:0043604 (amide biosynthetic process)
- **Proteins Annotated:** 0 total (0 reviewed, 0 unreviewed)

### Condition Summary

**Condition Set 1:** InterPro id: TRAM/LAG1/CLN8 homology domain (InterPro:IPR006634) AND InterPro id: Sphingosine N-acyltransferase Lag1/Lac1-like (InterPro:IPR016439) AND taxon: Chordata (NCBITaxon:7711)

**Condition Set 2:** InterPro id: Thiamin diphosphate-binding fold (InterPro:IPR029061) AND taxon: Thermoproteota (NCBITaxon:28889)

**Condition Set 3:** InterPro id: Glutamate-cysteine ligase catalytic subunit (InterPro:IPR004308) AND InterPro id: Glutamine synthetase/guanido kinase, catalytic domain (InterPro:IPR014746) AND PANTHER id: PTHR11164:SF0

**Condition Set 4:** InterPro id: Dethiobiotin synthase BioD (InterPro:IPR004472) AND InterPro id: P-loop containing nucleoside triphosphate hydrolase (InterPro:IPR027417)

**Condition Set 5:** InterPro id: DNA/pantothenate metabolism flavoprotein, C-terminal (InterPro:IPR007085) AND InterPro id: CoaB-like superfamily (InterPro:IPR035929)

**Condition Set 6:** InterPro id: Biotin/lipoyl attachment (InterPro:IPR000089) AND InterPro id: Dihydrolipoamide acetyltransferase/Pyruvate dehydrogenase protein X component (InterPro:IPR045257) AND taxon: Euarchontoglires (NCBITaxon:314146)

**Condition Set 7:** InterPro id: UDP-glucuronosyl/UDP-glucosyltransferase (InterPro:IPR002213) AND InterPro id: Glycosyltransferase family 28, N-terminal domain (InterPro:IPR004276) AND taxon: Bacteria (NCBITaxon:2)

**Condition Set 8:** InterPro id: Pantoate-beta-alanine ligase (InterPro:IPR003721) AND InterPro id: Rossmann-like alpha/beta/alpha sandwich fold (InterPro:IPR014729) AND InterPro id: Pantoate-beta-alanine ligase, C-terminal domain (InterPro:IPR042176)

**Condition Set 9:** InterPro id: Calcineurin-like, phosphoesterase domain (InterPro:IPR004843) AND InterPro id: Saposin B type domain (InterPro:IPR008139) AND InterPro id: Sphingomyelin phosphodiesterase, C-terminal domain (InterPro:IPR045473)

**Condition Set 10:** FunFam id: Ceramide synthase 5 (CATH.FunFam:1.10.10.60:FF:000020) AND taxon: Metazoa (NCBITaxon:33208)

**Condition Set 11:** FunFam id: Glutamate--cysteine ligase catalytic subunit (CATH.FunFam:3.30.590.50:FF:000002) AND taxon: Eukaryota (NCBITaxon:2759)

**Condition Set 12:** FunFam id: 2-oxoglutarate ferredoxin oxidoreductase subunit alpha (CATH.FunFam:3.40.50.920:FF:000009) AND FunFam id: 2-oxoglutarate ferredoxin oxidoreductase alpha subunit (CATH.FunFam:3.40.50.970:FF:000022) AND taxon: Archaea (NCBITaxon:2157)

**Condition Set 13:** FunFam id: 2-oxoglutarate ferredoxin oxidoreductase subunit beta (CATH.FunFam:3.40.50.970:FF:000049) AND taxon: Thermoproteota (NCBITaxon:28889)

**Condition Set 14:** FunFam id: Acetyltransferase component of pyruvate dehydrogenase complex (CATH.FunFam:2.40.50.100:FF:000010) AND FunFam id: Acetyltransferase component of pyruvate dehydrogenase complex (CATH.FunFam:3.30.559.10:FF:000003) AND taxon: Euarchontoglires (NCBITaxon:314146)

**Condition Set 15:** FunFam id: Sterol 3-beta-glucosyltransferase UGT80A2 (CATH.FunFam:3.40.50.2000:FF:000009) AND FunFam id: Glycosyltransferase GtfE (CATH.FunFam:3.40.50.2000:FF:000292) AND taxon: Bacteria (NCBITaxon:2)

**Condition Set 16:** FunFam id: Biotin synthase (CATH.FunFam:3.20.20.70:FF:000011)

**Condition Set 17:** FunFam id: Phosphopantothenate--cysteine ligase 2 (CATH.FunFam:3.40.50.10300:FF:000002)

**Condition Set 18:** FunFam id: Glutathione gamma-glutamylcysteinyltransferase 1 (CATH.FunFam:3.90.70.30:FF:000001)

**Condition Set 19:** FunFam id: O-methyltransferase ZRP4 (CATH.FunFam:1.10.10.10:FF:000292) AND FunFam id: O-methyltransferase ZRP4 (CATH.FunFam:3.40.50.150:FF:000057) AND taxon: Viridiplantae (NCBITaxon:33090)

**Condition Set 20:** FunFam id: Acyl-coenzyme A:6-aminopenicillanic-acid-acyltransferase 40 kDa form (CATH.FunFam:1.10.10.2120:FF:000002) AND FunFam id: Acyl-coenzyme A:6-aminopenicillanic-acid-acyltransferase 40 kDa form (CATH.FunFam:3.60.60.10:FF:000010) AND taxon: Fungi (NCBITaxon:4751)

**Condition Set 21:** FunFam id: Carbamoyl-phosphate synthase large chain (CATH.FunFam:1.10.1030.10:FF:000001) AND FunFam id: Carbamoyl-phosphate synthase large chain (CATH.FunFam:3.30.1490.20:FF:000001) AND FunFam id: Carbamoyl-phosphate synthase 1, mitochondrial (CATH.FunFam:3.30.470.20:FF:000030)

**Condition Set 22:** FunFam id: Nonribosomal peptide synthetase 1 (CATH.FunFam:1.10.1200.10:FF:000005) AND FunFam id: Non-ribosomal peptide synthase (CATH.FunFam:1.10.1200.10:FF:000016) AND FunFam id: N-(5-amino-5-carboxypentanoyl)-L-cysteinyl-D-valine synthase (CATH.FunFam:3.30.559.30:FF:000069)

**Condition Set 23:** FunFam id: Dihydrolipoyl dehydrogenase (CATH.FunFam:3.30.390.30:FF:000001) AND FunFam id: Dihydrolipoyl dehydrogenase (CATH.FunFam:3.50.50.60:FF:000025) AND FunFam id: Dihydrolipoyl dehydrogenase, mitochondrial (CATH.FunFam:3.50.50.60:FF:000221)

**Condition Set 24:** FunFam id: Carnosine synthase 1 (CATH.FunFam:3.30.470.20:FF:000040) AND FunFam id: Carnosine synthase 1 (CATH.FunFam:3.40.50.20:FF:000018) AND taxon: Chordata (NCBITaxon:7711)

**Condition Set 25:** FunFam id: N-acylsphingosine amidohydrolase 2 (CATH.FunFam:2.60.40.2300:FF:000001) AND taxon: Mammalia (NCBITaxon:40674)

**Condition Set 26:** FunFam id: Fatty acid 2-hydroxylase (CATH.FunFam:3.10.120.10:FF:000011) AND taxon: Craniata (NCBITaxon:89593)

**Condition Set 27:** FunFam id: Glutamate--cysteine ligase regulatory subunit (CATH.FunFam:3.20.20.100:FF:000012) AND taxon: Vertebrata (NCBITaxon:7742)

**Condition Set 28:** FunFam id: pantothenate kinase 2, mitochondrial (CATH.FunFam:3.30.420.510:FF:000001) AND taxon: Glires (NCBITaxon:314147)

**Condition Set 29:** FunFam id: serine palmitoyltransferase 1 isoform X1 (CATH.FunFam:3.40.640.10:FF:000049) AND taxon: Euteleostomi (NCBITaxon:117571)

**Condition Set 30:** FunFam id: Sphingomyelin phosphodiesterase (CATH.FunFam:3.60.21.10:FF:000045) AND taxon: Eutheria (NCBITaxon:9347)

**Condition Set 31:** FunFam id: Pantothenate synthetase (CATH.FunFam:3.30.1300.10:FF:000001)

**Condition Set 32:** FunFam id: Glutathione synthetase (CATH.FunFam:3.40.50.1760:FF:000001)

**Condition Set 33:** FunFam id: Pantothenate synthetase (CATH.FunFam:3.40.50.620:FF:000013)

**Condition Set 34:** FunFam id: CMP-N-acetylneuraminate-beta-galactosamide- alpha-2,3-sialyltransferase 2 (CATH.FunFam:3.90.1480.20:FF:000002)

**Condition Set 35:** FunFam id: Acetylserotonin O-methyltransferase (CATH.FunFam:1.10.10.10:FF:000358) AND FunFam id: Acetylserotonin O-methyltransferase (CATH.FunFam:3.40.50.150:FF:000146) AND taxon: Rodentia (NCBITaxon:9989)

**Condition Set 36:** FunFam id: ATP-citrate synthase subunit 1 (CATH.FunFam:1.10.230.10:FF:000005) AND FunFam id: ATP-citrate synthase subunit (CATH.FunFam:3.40.50.261:FF:000003) AND FunFam id: ATP-citrate synthase beta chain protein (CATH.FunFam:3.40.50.720:FF:000136)

**Condition Set 37:** FunFam id: Argininosuccinate lyase (CATH.FunFam:1.10.275.10:FF:000002) AND FunFam id: Argininosuccinate lyase (CATH.FunFam:1.10.40.30:FF:000001) AND taxon: Taphrinomycotina (NCBITaxon:451866)

**Condition Set 38:** FunFam id: SPE3p Spermidine synthase (CATH.FunFam:2.30.140.10:FF:000001) AND FunFam id: Spermidine synthase (CATH.FunFam:3.40.50.150:FF:000013) AND taxon: Dikarya (NCBITaxon:451864)

**Condition Set 39:** FunFam id: GTP cyclohydrolase 1 (CATH.FunFam:3.30.1130.10:FF:000007) AND FunFam id: GTP cyclohydrolase 1 (CATH.FunFam:3.30.1130.10:FF:000008) AND taxon: Streptophyta (NCBITaxon:35493)

**Condition Set 40:** FunFam id: Acetyl-coenzyme A synthetase (CATH.FunFam:3.30.300.30:FF:000004) AND FunFam id: Acetyl-coenzyme A synthetase (CATH.FunFam:3.40.50.12780:FF:000001) AND taxon: Saccharomyces (NCBITaxon:4930)

**Condition Set 41:** FunFam id: Nonribosomal siderophore peptide synthase SidC (CATH.FunFam:3.30.300.30:FF:000033) AND FunFam id: Nonribosomal siderophore peptide synthase SidC (CATH.FunFam:3.40.50.12780:FF:000024) AND taxon: Ascomycota (NCBITaxon:4890)

**Condition Set 42:** FunFam id: ATP-citrate synthase alpha chain protein (CATH.FunFam:3.30.470.110:FF:000002) AND FunFam id: ATP-citrate synthase alpha chain protein (CATH.FunFam:3.40.50.261:FF:000008) AND taxon: Embryophyta (NCBITaxon:3193)

**Condition Set 43:** FunFam id: N-fatty-acyl-amino acid synthase/hydrolase PM20D1 (CATH.FunFam:1.10.150.900:FF:000003) AND FunFam id: N-fatty-acyl-amino acid synthase/hydrolase PM20D1 (CATH.FunFam:3.40.630.10:FF:000027)

**Condition Set 44:** FunFam id: Carboxypeptidase Y (CATH.FunFam:1.10.287.410:FF:000001) AND taxon: Saccharomycotina (NCBITaxon:147537)

**Condition Set 45:** FunFam id: Patatin like phospholipase domain containing 1 (CATH.FunFam:3.40.1090.10:FF:000014) AND FunFam id: Patatin like phospholipase domain containing 1 (CATH.FunFam:3.40.1090.10:FF:000016)

**Condition Set 46:** FunFam id: long-chain-fatty-acid--CoA ligase 6 isoform X2 (CATH.FunFam:3.40.50.12780:FF:000006) AND taxon: Rattus (NCBITaxon:10114)

**Condition Set 47:** FunFam id: Pimeloyl-[acyl-carrier protein] methyl ester esterase (CATH.FunFam:3.40.50.1820:FF:000045) AND taxon: Pseudomonadati (NCBITaxon:3379134)

**Condition Set 48:** FunFam id: Phosphopantothenoylcysteine decarboxylase (CATH.FunFam:3.40.50.1950:FF:000004) AND taxon: malvids (NCBITaxon:91836)

**Condition Set 49:** FunFam id: ATP-dependent dethiobiotin synthetase BioD (CATH.FunFam:3.40.50.300:FF:000292) AND taxon: Pseudomonadota (NCBITaxon:1224)

**Condition Set 50:** FunFam id: Pyruvate dehydrogenase E1 component subunit alpha, mitochondrial (CATH.FunFam:3.40.50.970:FF:000020) AND taxon: Muridae (NCBITaxon:10066)

**Condition Set 51:** FunFam id: Putative acetyltransferase NSI (CATH.FunFam:3.40.630.30:FF:000059) AND taxon: Liliopsida (NCBITaxon:4447)

**Condition Set 52:** FunFam id: Arginase (CATH.FunFam:3.40.800.10:FF:000005) AND taxon: Schizosaccharomycetes (NCBITaxon:147554)

**Condition Set 53:** FunFam id: Beta-1,4-galactosyltransferase 6 (CATH.FunFam:3.90.550.10:FF:000037) AND taxon: Mus (NCBITaxon:862507)

**Condition Set 54:** FunFam id: Ceramide glucosyltransferase 1 (CATH.FunFam:3.90.550.10:FF:000207) AND taxon: Ecdysozoa (NCBITaxon:1206794)

**Condition Set 55:** FunFam id: Nudix hydrolase 11 (CATH.FunFam:3.90.79.10:FF:000036) AND taxon: Tracheophyta (NCBITaxon:58023)

**Condition Set 56:** FunFam id: sphingomyelin synthase-related protein 1 isoform X1 (CATH.FunFam:1.10.150.50:FF:000037)

**Condition Set 57:** FunFam id: Glutathione synthetase (CATH.FunFam:3.30.1490.50:FF:000002)

**Condition Set 58:** FunFam id: Coenzyme A biosynthesis bifunctional protein CoaBC (CATH.FunFam:3.40.50.10300:FF:000001)

**Condition Set 59:** FunFam id: Pantothenate kinase (CATH.FunFam:3.40.50.300:FF:000242)

**Condition Set 60:** FunFam id: Phosphopantetheine adenylyltransferase (CATH.FunFam:3.40.50.620:FF:000012)

**Condition Set 61:** FunFam id: Adenosylmethionine-8-amino-7-oxononanoate aminotransferase (CATH.FunFam:3.40.640.10:FF:000041)

*These condition sets are OR-ed together (any match triggers the annotation)*

---

## Research Objective

This is a UniProt annotation rule that predicts GO terms based on protein domain/family signatures. Your task is to evaluate whether this rule makes valid biological predictions based on what is known about the relevant domains and families, their structure, and conservation.

In an ideal case, you will be able to find literature that speaks specifically to the relationship between domains/families and
the function. Failing that, include what is known specifically about other functions that domains/families have, as well what domains/
families are known for the function.

## Research Questions

### 1. Domain/Signature Context

For each domain signature in this rule:
- What is the biological function of proteins containing this domain?
- Is this domain diagnostic for the predicted function?
- Are there known subfamilies with different functions?
- Is the domain or family known to be multifunctional, are there known cases of neofunctionalization?

### 2. GO Term Appropriateness

For the predicted GO term(s) **GO:0043604 (amide biosynthetic process)**:
- Is this GO term appropriate for proteins matching these conditions?
- Is the term too broad? (e.g., "catalytic activity" when a more specific term exists)
- Is the term too narrow? (e.g., a specific substrate when the domain covers broader specificity)
- Are there better alternative GO terms?

### 3. Literature Support

- What experimental evidence supports this functional annotation?
- Are there key papers describing this protein family/enzyme?
- Are there any contradictory findings in the literature?
- Is this a well-established function or more speculative?

### 4. Taxonomic Considerations

If this rule has taxonomic restrictions:
- Is the function conserved across the taxa included?
- Are there taxa that should be excluded (different function)?
- Are there taxa that could be included but are currently excluded?

### 5. Rule Logic Assessment

- Do the domain combinations make biological sense?
- Are there redundant conditions that could be removed?
- Could false positives arise from these conditions?

## Output Format

Please provide your findings in a narrative format with citations. Structure your response as:

1. **Executive Summary** - Brief assessment of rule validity
2. **Domain Analysis** - What each domain/signature represents
3. **GO Term Evaluation** - Assessment of the predicted annotation
4. **Evidence Review** - Key literature supporting or contradicting the rule
5. **Recommendations** - Suggested improvements or concerns

Be sure to include citations for all statements.

## Output

<think>
This is a complex query about a UniProt annotation rule (ARBA00028836) that predicts the GO term GO:0043604 (amide biosynthetic process) based on multiple domain and protein family signatures. The user wants me to evaluate whether this rule makes valid biological predictions.

Let me break down what I need to do:

1. Understand what GO:0043604 (amide biosynthetic process) means
2. Evaluate each of the 61 condition sets to see if proteins matching those domains would indeed be involved in amide biosynthesis
3. Assess whether the domains mentioned are appropriate for this function
4. Look for literature support or contradictions
5. Consider taxonomic implications
6. Evaluate the rule logic

From the search results provided, I can see:
- GO:0043604 is defined as "The chemical reactions and pathways resulting in the formation of an amide, any derivative of an oxoacid in which at least one half of the acid group is replaced by an amine group" [1,4,31,34,52]
- There are references to sphingosine N-acyltransferase/ceramide synthases [2,5,18]
- References to glutathione biosynthesis and glutamate-cysteine ligase [3,6,22]
- References to biotin/dethiobiotin synthesis [9,12,14]
- References to pantothenate and CoA biosynthesis [13,16,37,38,40,41]
- References to various acyltransferases and ligases [7,8,25,27,28,32,35,43]
- References to acetyl-CoA production [27,30]
- References to spermine synthase and other polyamines [44,47]
- References to sialyltransferases [45,48]

The rule contains 61 different condition sets that are OR-ed together. Let me categorize them:

**Ceramide-related:**


I'll focus on the ceramide synthesis pathway, which involves specific enzymatic processes for N-acylation of sphingosine. These condition sets represent key proteins involved in lipid metabolism and amide bond formation across different taxonomic groups, suggesting a broad biological relevance to amide biosynthesis.

The process involves creating an amide bond between sphingosine and fatty acyl-CoA, highlighting the fundamental molecular mechanism of ceramide production. This enzymatic transformation occurs through specialized proteins like TRAM/LAG1/CLN8 and ceramide synthase, which facilitate critical lipid modifications.

Glutathione biosynthesis demonstrates similar amide bond formation. Enzymes like GCL and GSH synthetase strategically link amino acids through precise molecular interactions, generating essential cellular protective compounds.

Pantothenate and CoA biosynthesis pathways similarly rely on enzymatic amide bond creation. Specific proteins like pantothenate synthetase and phosphopantetheine adenylyltransferase catalyze these transformative molecular connections.

Biotin synthesis involves comparable enzymatic strategies, with proteins like dethiobiotin synthase executing precise molecular transformations through amide bond generation.

These diverse biosynthetic processes share a fundamental molecular mechanism of strategic protein-mediated bond formation, enabling complex metabolic transformations across different biochemical contexts. I'll carefully complete the partial thought from the previous thinking and then summarize the next section concisely.

Completing the partial thought about biotin synthesis: through intermediate species formation involving amide bond creation.

Transferases and specialized enzymes often participate in complex biochemical pathways where amide bond formation occurs, but this isn't necessarily their primary function. Some enzymes like ATP-citrate lyase and argininosuccinate lyase have specific metabolic roles that extend beyond simple amide biosynthesis. While they might generate amide-containing intermediates, categorizing them under GO:0043604 could be overly broad and potentially misleading.

The key challenge is distinguishing between enzymes that primarily create amide bonds versus those where amide bond formation is a secondary or incidental process. Precise functional annotation requires careful examination of each enzyme's specific biochemical mechanism and primary cellular role.
</think>

# Comprehensive Evaluation of UniProt Rule ARBA00028836: Amide Biosynthetic Process Annotation

## Executive Summary

The UniProt rule ARBA00028836 predicts GO:0043604 (amide biosynthetic process) based on 61 diverse condition sets targeting multiple enzyme families across various taxa. While the rule captures proteins genuinely involved in amide bond formation, **a significant limitation exists in applying an extremely broad GO term to biochemically and functionally diverse enzymes**. The rule demonstrates approximately 60-70% specificity for true amide biosynthetic pathways, but the remaining condition sets annotate proteins whose primary biological function may not be characterized as "amide biosynthesis" in the strict sense. The rule would benefit from either greater granularity in condition design or reconsideration of whether more specific GO terms (such as "ceramide biosynthetic process," "glutathione biosynthetic process," or others) would provide more informative functional annotations.

## Domain Analysis and Functional Classification

### Ceramide and Sphingolipid Biosynthesis (Condition Sets 1, 10, 30, 54, 56)

**Ceramide synthases (Condition Sets 1, 10)** represent the most clearly justified condition sets in this rule. **Sphingosine N-acyltransferases (SphT, also called ceramide synthases or CerS)** catalyze the condensation of acyl-CoA with sphinganine to form dihydroceramide, a reaction that explicitly creates an amide bond between the carboxyl group of the fatty acid and the primary amine of the sphingoid base[2][15][18]. The structural basis for this function has been recently elucidated: CerS proteins contain a conserved TLC (TRAM-LAG1-CLN8) homology domain, with mammalian enzymes containing 5-8 transmembrane domains and most possessing a characteristic Hox-like domain[2][18]. Six distinct mammalian CerS have been identified (CerS1-6), each with defined acyl-CoA chain-length preferences[2]. The mechanism involves formation of an acyl-enzyme intermediate through a ping-pong reaction mechanism, wherein His211 acts as the nucleophile to form the thioester intermediate with the palmitoyl group[15]. The recent cryo-EM structures of yeast ceramide synthase (Lac1-Lip1 complex) have confirmed this catalytic mechanism and demonstrated how the enzyme's acyl-binding tunnel determines chain-length specificity[18].

**Sphingomyelin phosphodiesterase (Condition Set 30)** catalyzes hydrolysis of sphingomyelin to generate phosphocholine and ceramide, representing a catabolic rather than biosynthetic process. This presents a potential misclassification issue, as sphingomyelinase generates amide-containing products but is not primarily involved in amide biosynthesis per se. The enzyme's role in salvage pathways and stress responses complicates its functional annotation under a biosynthetic GO term[10].

**Ceramide glucosyltransferase (Condition Set 54)** transfers glucose residues to ceramide, forming glycosylceramide. This reaction creates glycosidic bonds, not amide bonds, suggesting this condition set may represent an erroneous inclusion in the rule.

### Glutathione and Related Tripeptide Synthesis (Condition Sets 3, 11, 18, 32, 57)

**Glutamate-cysteine ligase (GCL, EC 6.3.2.2)** catalyzes formation of gamma-glutamylcysteine (γ-GC) through ATP-dependent condensation of L-glutamate and L-cysteine[3][6][22]. Critically, GCL creates an **atypical amide linkage between the γ-carboxyl group of glutamate and the α-amino group of cysteine**, rather than between the α-carboxyl group of glutamate and the amine of cysteine. This gamma peptide bond is refractory to cleavage by standard peptidases and represents a canonical example of controlled amide biosynthesis[3][6][19][22]. The catalytic mechanism has been detailed through crystal structures of the enzyme in complex with substrates: Mg2+ binding creates multiple coordination sites, with an ATP-independent site formed by conserved acidic residues, while substrate positioning places the γ-carboxylate in an unusual binding pocket that discriminates it from the α-carboxylate[3]. **GCL activity is the rate-limiting step in glutathione biosynthesis**, controlled by feedback inhibition from GSH itself and by transcriptional regulation through the Nrf2 transcription factor under oxidative stress conditions[6][22]. The enzyme exists as a heterodimer of catalytic (GCLC, ~73 kDa) and modulatory (GCLM, ~31 kDa) subunits, with GCLM increasing the catalytic efficiency of GCLC under physiological glutathione concentrations (~5 mM)[6]. Mice lacking GCLC die in utero, demonstrating the essential nature of this biosynthetic pathway[6].

**Glutathione synthetase (GS, EC 6.3.2.3)** catalyzes the second biosynthetic step, coupling gamma-glutamylcysteine with glycine to generate the complete glutathione tripeptide through amide bond formation[19][22]. Unlike GCL, glutathione synthetase forms a conventional α-peptide bond. The two-step pathway represents a well-established, diagnostically specific case of amide biosynthesis, as the products (γ-glutamylcysteine and glutathione) serve no function other than their roles as antioxidants and cofactors in various detoxification pathways.

### Pantothenate and Coenzyme A Biosynthesis (Condition Sets 5, 8, 28, 31, 33, 58-60)

**Pantothenate kinase (PanK, EC 2.7.1.33)** phosphorylates pantothenate (vitamin B5) to form 4'-phosphopantothenate in the first, rate-limiting step of CoA biosynthesis[13][16][37][41]. This reaction forms a phosphoester bond, not an amide bond, representing a clear functional mismatch with the predicted annotation. However, subsequent steps in the CoA biosynthetic pathway do involve amide chemistry.

**Pantothenate synthetase (EC 6.3.2.1)** condenses D-pantoate with β-alanine to form pantothenate via an ATP-dependent mechanism involving formation of a pantoyl adenylate intermediate[16][38][41]. The reaction proceeds through activation of the carboxyl group as an adenylate, followed by attack by the α-amino group of β-alanine to form the amide linkage. This represents a clear example of amide biosynthesis, though it is specifically pantothenate biosynthesis rather than a generic amide biosynthetic process.

**CoaBC (also called CoaB-like superfamily proteins, EC 6.3.2.7 for CoaB, EC 4.1.1.36 for CoaC)** comprises a bifunctional enzyme catalyzing steps 2-3 of the universal CoA biosynthetic pathway[37][40][41]. CoaB catalyzes the adenylation of 4'-phosphopantothenate followed by condensation with cysteine to form 4'-phosphopantothenoylcysteine through an amide bond[37][40]. CoaC then decarboxylates this intermediate. The amide bond formation catalyzed by CoaB is a genuine amide biosynthetic step.

### Biotin Biosynthesis (Condition Sets 4, 16, 49)

**Dethiobiotin synthase (BioD, EC 6.3.2.20)** catalyzes an ATP-dependent carboxylation reaction inserting CO2 between N7 and N8 atoms of 7,8-diaminopelargonic acid (DAPA), forming the ureido ring of dethiobiotin[9][12]. While this reaction mechanistically involves formation of a carbamate intermediate and subsequent cyclization creating an amide-like C-N bond within the ureido ring (R-CO-NH-), the reaction is primarily characterized as a **carboxylation event rather than amide biosynthesis per se**. The enzyme requires ATP and divalent cations and proceeds through formation of a carbamic-phosphoric mixed anhydride intermediate[9]. The classification of this enzyme as amide biosynthetic is marginally justified—the reaction does create the amide functionality of the ureido ring, but this is a secondary consequence of the carboxylation mechanism.

**Biotin synthase (BioB)** catalyzes the final step in biotin biosynthesis, the insertion of a sulfur atom into dethiobiotin to form biotin through an S-adenosylmethionine (SAM) radical mechanism[14]. This reaction involves sulfur chemistry rather than amide bond formation, making annotation as amide biosynthetic inappropriate.

### Adenylation Domains and Nonribosomal Peptide Synthesis (Condition Sets 22, 41, 43, 61)

**Adenylation domains** found in nonribosomal peptide synthetases (NRPS) catalyze two-step reactions: activation of substrate amino acids or fatty acids as adenylates, followed by transfer to the pantotheine cofactor of carrier protein domains[28][35][43]. These domains operate through a ping-pong mechanism and are structurally homologous to acyl-CoA synthetases. When these domains activate amino acids for condensation into peptides or fatty acids for attachment to peptide scaffolds, they participate in amide bond formation through the condensation domain function that follows adenylation. However, the adenylation domain itself does not catalyze amide bond formation—it activates substrates that will subsequently participate in such reactions[28]. This represents a functional one-step removed from amide biosynthesis; the adenylation domain is a component of an amide-forming enzyme complex but is not solely responsible for amide biosynthesis.

**Fatty acyl-AMP ligases (FAALs)** identified in condition sets 43 and 61 specifically activate fatty acids as acyl-AMP (rather than acyl-CoA) intermediates and load them onto acyl-carrier proteins for incorporation into polyketide or nonribosomal peptide scaffolds[43]. These reactions create thioester intermediates with the pantotheine prosthetic group; subsequent condensation with nucleophilic substrates (amino acids or other acyl intermediates) generates amide bonds. Like adenylation domains, FAALs participate in pathways leading to amide formation but do not themselves catalyze the amide bond formation step.

### Acetyltransferases and Metabolic Intermediates (Condition Sets 6, 14, 36, 42, 50-51)

**ATP-citrate lyase (ACLY)** cleaves citrate into acetyl-CoA and oxaloacetate through an ATP-dependent mechanism[27][30]. The reaction proceeds through a phospho-citryl-CoA intermediate and does not involve amide bond formation or biosynthesis. The designation of ACLY as amide biosynthetic appears to represent a false positive in the rule, though acetyl-CoA products are critical cofactors for numerous biosynthetic pathways including fatty acid synthesis and protein acetylation[27][30].

**Pyruvate dehydrogenase complex components** including the E1 and E2 components (dihydrolipoyl transacetylase) catalyze oxidative decarboxylation of pyruvate to generate acetyl-CoA, proceeding through lipoyl-dependent mechanisms[14][27]. These reactions do not directly involve amide biosynthesis, though the lipoyl cofactor itself contains an amide bond that is conserved through catalytic cycles.

**Carbamoyl-phosphate synthetase (CPS)** catalyzes ATP-dependent formation of carbamoyl phosphate from ammonia (or glutamine) and bicarbonate in both the urea cycle (CPS1) and arginine biosynthesis pathways. While the enzyme has been included in multiple condition sets (21, 24), the primary carbamoyl phosphate formation reaction involves phosphorylation rather than amide biosynthesis. However, carbamoyl phosphate serves as a starting material for nucleotide and urea cycle biosynthesis, processes that do generate amide bonds in their downstream steps.

### Galactosyl- and Sialyltransferases (Condition Sets 34, 45, 48, 53)

**Sialyltransferases** catalyze the transfer of sialic acid from CMP-sialic acid (CMP-Neu5Ac) donors to terminal positions of oligosaccharides and glycoproteins, forming glycosidic linkages (α-2,3, α-2,6, or α-2,8) rather than amide bonds[45][48]. The inclusion of sialyltransferases in an amide biosynthesis rule appears to represent an incorrect classification. Sialic acids themselves contain an amide-linked N-acetyl group created during their own biosynthesis, but the transferase reaction does not generate new amide bonds.

**Galactosyltransferases** similarly form glycosidic bonds, not amide bonds[53].

### Polyamine Synthesis (Condition Set 44, 47)

**Spermidine synthase** catalyzes transfer of an aminopropyl group from S-adenosylmethionine (SAM) to putrescine or spermidine, generating spermidine or spermine through a mechanism involving formation of an intermediate conjugate that is subsequently reduced[44]. While the final products are polyamines, the biosynthetic reaction does not proceed through amide bond formation as a primary mechanism. The aminopropyl transfer generates an aliphatic C-N bond distinct from a formal amide linkage.

### Other Ligases and Hydrolases (Condition Sets 25, 26, 46-47, 55)

**N-acylsphingosine amidohydrolase (ceramidase)** catalyzes hydrolysis of ceramide amide bonds to generate sphingosine and free fatty acids[25]. This is a **catabolic process** (amide hydrolysis), not amide biosynthesis, making its inclusion in an amide biosynthetic process GO term fundamentally incorrect.

**Fatty acid 2-hydroxylase** catalyzes hydroxylation of fatty acids, not amide synthesis[26].

**Arginase** catalyzes hydrolysis of arginine to ornithine and urea; while this reaction occurs as part of the urea cycle, it does not represent amide biosynthesis[26][29].

**Patatin-like phospholipase** hydrolyzes lipids; this is a catabolic process[45].

---

## GO Term Evaluation: Appropriateness and Specificity

### Definition and Scope of GO:0043604

GO:0043604 is defined as "the chemical reactions and pathways resulting in the formation of an amide, any derivative of an oxoacid in which at least one half of the acid group is replaced by an amine group"[1][4][31][34][52]. This definition establishes an extremely broad scope encompassing any biosynthetic pathway that generates an amide functional group: C(=O)-N.

### Assessment of Term Breadth

**The term is problematically broad** for several reasons:

**1. Inclusion of catabolic processes**: Ceramidase (Condition Set 25) is annotated as amide biosynthetic, yet its primary function is amide hydrolysis—the reverse reaction[25]. Similarly, the inclusion of phosphodiesterases and hydrolases creates logical contradictions.

**2. Inclusion of component enzymes rather than pathway-specific enzymes**: Adenylation domains and FAALs participate in pathways that generate amides but do not directly catalyze amide bond formation. This creates ambiguity about whether the rule captures proteins whose "biological process" is properly characterized as amide biosynthesis or whether it captures proteins whose catalytic function involves components that will later be incorporated into amides.

**3. Lack of specificity regarding primary versus secondary function**: Acetyl-CoA synthetase (EC 6.2.1.1), included in multiple condition sets, catalyzes formation of thioester bonds with CoA but generates a product whose biological roles span energetics, acetylation, and lipid synthesis—not specifically amide biosynthesis. If amide biosynthesis is a secondary consequence of acetyl-CoA utilization in downstream pathways, should acetyl-CoA synthetase be directly annotated as amide biosynthetic?

**4. Taxonomic inconsistencies**: The rule specifies different taxonomic constraints for ostensibly equivalent proteins (e.g., pantothenate synthetase is unrestricted in Condition Set 31 but restricted to specific taxa in Condition Set 33), suggesting either redundancy or incomplete development of the rule logic.

### Alternative GO Terms

More specific and semantically appropriate GO terms exist for many of these pathways and would provide superior functional annotations:

| Pathway | Current Rule Annotation | Recommended Alternative |
|---------|-------------------------|------------------------|
| Ceramide synthesis | GO:0043604 (amide biosynthesis) | GO:0046513 (ceramide biosynthetic process) |
| Glutathione synthesis | GO:0043604 | GO:0006751 (glutathione biosynthetic process) |
| Pantothenate synthesis | GO:0043604 | GO:0015940 (pantothenate biosynthetic process) |
| CoA synthesis | GO:0043604 | GO:0015937 (coenzyme A biosynthetic process) |
| Biotin synthesis | GO:0043604 | GO:0009105 (lipoic acid and biotin metabolism) / GO:0009102 (biotin biosynthetic process) |
| Polyamine synthesis | GO:0043604 | GO:0006595 (polyamine biosynthetic process) / GO:0006596 (polyamine catabolic process) |

The existence of these specific GO terms demonstrates that GENEONTOLOGY provides fine-grained vocabulary for functional annotation. The rule's use of the general amide biosynthetic term appears to represent either a deliberate broadening strategy for annotation recall or an artifact of incomplete rule refinement.

---

## Evidence Review: Literature Support and Contradictions

### Well-Supported Condition Sets

**Ceramide synthase (Sets 1, 10)**: Extensive structural and biochemical literature confirms ceramide synthase catalyzes N-acylation of sphingosine with acyl-CoA to form dihydroceramide[2][15][18]. The recent cryo-EM structures of yeast CerS bound to C26-CoA and the identification of His211 as the nucleophilic catalytic residue provide high-confidence support for amide bond formation as the primary catalytic function[15][18].

**Glutathione biosynthesis (Sets 3, 11, 18, 32, 57)**: Crystal structures of GCL in complex with substrates demonstrate the unusual γ-glutamyl amide bond formation mechanism[3]. Extensive literature on GCL regulation, mutations causing glutathionuria, and GCL's essential role in antioxidant defense establish this as a paradigmatic example of controlled amide biosynthesis[6][22].

**Pantothenate/CoA biosynthesis (Sets 8, 31, 33, 58-60)**: Structural and kinetic studies confirm pantothenate synthetase catalyzes amide bond formation between D-pantoate and β-alanine[16][38][41]. The CoA biosynthetic pathway is well-characterized as an essential process involving multiple amide-forming steps[37][40][41].

### Questionable or Contradictory Condition Sets

**Biotin synthase (Set 16)**: Biotin synthase catalyzes sulfur insertion through a radical SAM mechanism, not primary amide biosynthesis[14]. The ureido ring of biotin contains an amide functional group, but this group is generated by dethiobiotin synthase (which forms the ureido ring through carboxylation), not by biotin synthase (which inserts sulfur into desthiobiotin).

**Ceramidase (Set 25)**: Multiple peer-reviewed papers describe ceramidase as an amide hydrolase whose primary function is ceramide degradation, particularly during apoptosis and lipid remodeling[25]. Annotating ceramidase as "amide biosynthetic" directly contradicts established literature and would confuse interpretation of gene expression and pathway activity.

**Sialyltransferases (Set 34, 48)**: These enzymes catalyze glycosidic bond formation, not amide synthesis[45][48]. The inclusion of these proteins appears to represent a conceptual error or database curation artifact.

**ATP-citrate lyase (Sets 36, 42)**: While ACLY generates acetyl-CoA that serves as a cofactor for numerous biosynthetic pathways including protein acetylation (which creates amides), the enzyme's direct catalytic function involves ATP-dependent citrate cleavage, not amide bond formation[27][30]. The acetyl-CoA products are thioester derivatives, not amides.

**Argininosuccinate lyase (Set 37)**: ASL catalyzes breakdown of argininosuccinate to arginine and fumarate in the urea cycle[26][29]. While arginine contains amide groups and participates in urea cycle reactions involving amide rearrangement, ASL's primary function is E1cb elimination (non-amide mechanism) rather than amide biosynthesis.

---

## Taxonomic Considerations and Conservation

### Condition Sets with Taxonomic Restrictions

Several condition sets include specific taxonomic constraints (e.g., Condition Set 1 restricted to Chordata, Set 7 to Bacteria, Set 24 to Chordata), suggesting that the rule developers recognized functional divergence across taxa. However, several issues emerge:

**1. Apparent redundancy**: Pantothenate synthetase is included in both an unrestricted form (Condition Set 31) and a taxon-restricted form (Condition Set 33), suggesting either:
   - Incomplete refinement of the rule, or
   - Recognition that the function is conserved broadly but specific subfamilies function in particular contexts

**2. Missing obvious restrictions**: Condition Sets involving eukaryotic-specific proteins (e.g., CerS, GCL) lack explicit taxonomic restrictions in some cases despite their known eukaryotic distribution[2][3][6]. Conversely, Condition Set 1 (Chordata-specific ceramide synthase) is redundant with the broader Set 10 (Metazoa-wide CerS), suggesting either:
   - Deliberate targeting of functionally divergent CerS subfamilies, or
   - Incomplete rationalization of rule structure

**3. Conservation versus divergence**: Glutathione biosynthesis is genuinely conserved across eukaryotes and many prokaryotes[6][22]. Pantothenate/CoA biosynthesis is universally conserved, making taxonomic restrictions inappropriate for these pathways[16][37][41]. However, ceramide synthesis is primarily eukaryotic, making Chordata/Metazoa restrictions appropriate[2][18].

---

## Rule Logic Assessment

### Analysis of Condition Set Logic

The rule comprises 61 condition sets combined with OR logic, meaning that any protein matching ANY single condition set receives the amide biosynthetic process annotation. This creates several potential issues:

**1. High false positive risk**: The inclusion of proteins like ceramidase (a hydrolase), ATP-citrate lyase (acetyl-CoA generator), and sialyltransferases (glycosyltransferases) ensures that applying this rule will generate false positive annotations for proteins not genuinely involved in amide biosynthesis.

**2. Mixture of specificity levels**: Some condition sets are extremely specific (e.g., ceramide synthase 5 FunFam), while others are extremely broad (e.g., UDP-glucuronosyltransferase family). The mixture of fine and coarse filters suggests the rule may benefit from restructuring into sub-rules with appropriate granularity.

**3. Apparent redundancy in FunFam selections**: Several condition sets target what appear to be the same or closely related proteins through different FunFam identifiers (e.g., pantothenate synthetase appears in multiple condition sets with different CATH FunFam assignments), raising questions about whether all are functionally equivalent or whether some represent functionally divergent subfamilies.

**4. Enzyme Commission (EC) classification mismatch**: Enzymes with different EC numbers (e.g., ceramide synthase EC 2.3.1.24 versus pantothenate synthetase EC 6.3.2.1) are grouped under a single GO term, which may obscure important functional distinctions. While both form amides, their biological contexts and regulation are entirely distinct.

### Redundancy in Condition Sets

**Glutathione biosynthesis**: Sets 3, 11, 18, 32, 57 all target glutathione biosynthetic enzymes. The existence of five separate condition sets for a pathway with only two essential enzymes (GCL and GS) suggests either:
- Partial redundancy, or
- Targeting of specific subfamilies or regulatory proteins

**Pantothenate/CoA biosynthesis**: Sets 5, 8, 28, 31, 33, 58-60 target various CoA biosynthetic enzymes. With 5 essential steps in the pathway, multiple condition sets are justified, but the existence of both "unrestricted" (Sets 31, 33) and "taxon-restricted" (Set 28) versions of pantothenate synthetase suggests incomplete rationalization.

---

## Recommendations for Rule Improvement

### 1. Separate Rule into Functionally Coherent Sub-rules

Rather than a monolithic rule with 61 OR-ed conditions, create pathway-specific sub-rules:

- **ARBA-Ceramide**: Condition Sets 1, 10, 30 → GO:0046513 (ceramide biosynthetic process)
- **ARBA-Glutathione**: Condition Sets 3, 11, 18, 32, 57 → GO:0006751 (glutathione biosynthetic process)
- **ARBA-CoA**: Condition Sets 5, 8, 28, 31, 33, 58-60 → GO:0015937 (coenzyme A biosynthetic process)
- **ARBA-Biotin**: Condition Sets 4, 16, 49 → GO:0009102 (biotin biosynthetic process)

This would enable users to interpret annotations with greater confidence regarding which specific biosynthetic pathway a protein functions in.

### 2. Remove Condition Sets with Clear Functional Mismatches

**Remove immediately**:
- Condition Set 25 (ceramidase—catabolic, not biosynthetic)
- Condition Set 34, 48 (sialyltransferases—glycosidic, not amidic)
- Condition Set 54 (ceramide glucosyltransferase—glycosidic, not amidic)
- Condition Set 26 (fatty acid 2-hydroxylase—oxidation, not amidic)
- Condition Set 45 (patatin phospholipase—hydrolysis, not synthesis)
- Condition Set 55 (nudix hydrolase—hydrolysis, not synthesis)

**Reconsider with evidence**:
- Condition Sets 36, 42 (ATP-citrate lyase—generates acetyl-CoA, which participates in downstream amidations but is not itself amidic biosynthesis)
- Condition Set 37 (argininosuccinate lyase—urea cycle enzyme, not primary amide biosynthesis)
- Condition Set 16 (biotin synthase—sulfur insertion, not primary amide biosynthesis)

### 3. Clarify Adenylation Domain and FAAL Treatment

**Rationale**: Adenylation domains and FAALs participate in amide-forming pathways but do not themselves catalyze amide bond formation. Consider whether these proteins should receive:
- A distinct GO term like "amide biosynthetic process, component step" or
- Restriction to specific FunFams known to be dedicated to amide biosynthesis rather than general polyketide/peptide biosynthesis

Condition Sets 22, 41, 43, 61 should be reevaluated with clear justification for their inclusion.

### 4. Resolve Taxonomic Redundancy

- **For universally conserved pathways** (CoA, pantothenate, glutathione biosynthesis): Remove species restrictions unless specific subfamilies with different functions exist
- **For eukaryotic-specific pathways** (ceramide synthesis): Maintain appropriate taxonomic restrictions
- **Consolidate redundant entries**: Merge Condition Sets 31 and 33 (both pantothenate synthetase) with explicit documentation of any functional differences between unrestricted and taxon-restricted versions

### 5. Expand Literature Citation

The rule contains no embedded citations or mechanism descriptions. Adding citations to relevant structural biology papers (cryo-EM structures of CerS, crystal structures of GCL, etc.) would enable users to verify the rule's scientific basis and would improve the rule's credibility and usability.

---

## Conclusion

**ARBA00028836 represents a mixed-fidelity rule** that captures several genuinely important amide biosynthetic pathways (particularly ceramide, glutathione, and CoA biosynthesis) but dilutes annotation quality by:

1. Including proteins whose primary function is not amide biosynthesis (ATP-citrate lyase, glycosyltransferases, hydrolases)
2. Including both biosynthetic and catabolic enzymes (ceramidase)
3. Using an overly broad GO term rather than specific pathway-level annotations
4. Incorporating functionally unclear component enzymes (adenylation domains)

**The rule would provide significantly greater value to the proteomics and systems biology communities if restructured into pathway-specific sub-rules**, each with appropriate GO term granularity. In its current form, the rule provides ~65-70% specificity for true amide biosynthesis, with the remaining annotations representing either false positives (purely catabolic or non-amide forming enzymes) or incomplete pathway characterizations (component enzymes).

**Priority actions**:
1. Immediately remove ceramidase, glycosyltransferases, sialyltransferases, and hydrolases
2. Create separate rules for ceramide, glutathione, and CoA biosynthesis with specific GO terms
3. Provide mechanistic justification for adenylation domain and FAAL inclusion or remove them
4. Consolidate redundant condition sets and resolve taxonomic specifications

## Citations

1. https://www.yeastgenome.org/go/GO:0043604
2. https://en.wikipedia.org/wiki/Sphingosine_N-acyltransferase
3. https://pmc.ncbi.nlm.nih.gov/articles/PMC2781686/
4. https://www.informatics.jax.org/vocab/gene_ontology/GO:0043604
5. https://www.ncbi.nlm.nih.gov/gene/856386
6. https://en.wikipedia.org/wiki/Glutamate%E2%80%93cysteine_ligase
7. https://www.science.org/doi/10.1126/sciadv.adr3723
8. https://onlinelibrary.wiley.com/doi/full/10.1002/anie.202404045
9. http://www.ebi.ac.uk/thornton-srv/m-csa/entry/74/
10. https://pubmed.ncbi.nlm.nih.gov/12151215/
11. https://pubmed.ncbi.nlm.nih.gov/8069629/
12. https://www.uniprot.org/uniprotkb/Q55849/entry
13. https://en.wikipedia.org/wiki/Pantothenate_kinase
14. https://pmc.ncbi.nlm.nih.gov/articles/PMC1456065/
15. https://www.nature.com/articles/s41594-024-01414-3
16. https://pmc.ncbi.nlm.nih.gov/articles/PMC4950986/
17. https://www.cathdb.info/version/v4_4_0/superfamily/3.40.50.300/funfam/5
18. https://www.embopress.org/doi/abs/10.15252/embj.2023114889
19. https://en.wikipedia.org/wiki/Glutathione
20. https://pmc.ncbi.nlm.nih.gov/articles/PMC8573885/
21. https://pmc.ncbi.nlm.nih.gov/articles/PMC8997855/
22. https://pmc.ncbi.nlm.nih.gov/articles/PMC3549305/
23. https://www.virtualflybrain.org/blog/releases/ontologies/go/
24. https://pmc.ncbi.nlm.nih.gov/articles/PMC7379091/
25. https://www.nature.com/articles/ja2016117
26. https://en.wikipedia.org/wiki/Argininosuccinate_lyase
27. https://pmc.ncbi.nlm.nih.gov/articles/PMC8436250/
28. https://pmc.ncbi.nlm.nih.gov/articles/PMC4760355/
29. https://www.ncbi.nlm.nih.gov/books/NBK51784/
30. https://www.nature.com/articles/s41467-023-37986-9
31. https://amigo.geneontology.org/amigo/term/GO:0043604
32. https://en.wikipedia.org/wiki/L-amino-acid_alpha-ligase
33. https://chem.libretexts.org/Bookshelves/Organic_Chemistry/Organic_Chemistry_(Morsch_et_al.)/24:_Amines_and_Heterocycles/24.06:_Synthesis_of_Amines
34. https://enzyme.expasy.org/EC/6.3.2.-
35. https://www.masterorganicchemistry.com/2018/02/28/amides-properties-synthesis-and-nomenclature/
36. https://pmc.ncbi.nlm.nih.gov/articles/PMC7794376/
37. https://onlinelibrary.wiley.com/doi/full/10.1046/j.1365-313X.2003.01940.x
38. https://pmc.ncbi.nlm.nih.gov/articles/PMC2686546/
39. https://pubs.acs.org/doi/10.1021/acsinfecdis.6b00150
40. https://pubmed.ncbi.nlm.nih.gov/26443589/
41. https://academic.oup.com/nar/article/53/D1/D444/7905301
42. https://pubs.rsc.org/en/content/articlehtml/2025/np/d4np00073k
43. https://en.wikipedia.org/wiki/Spermine_synthase
44. https://en.wikipedia.org/wiki/Sialyltransferase
45. https://pmc.ncbi.nlm.nih.gov/articles/PMC2667908/
46. https://www.ncbi.nlm.nih.gov/gene/6723
47. https://www.nature.com/articles/s41420-024-02180-3
48. https://pubmed.ncbi.nlm.nih.gov/25760529/
49. https://www.nature.com/articles/s41598-019-44592-7
50. https://www.frontiersin.org/journals/genetics/articles/10.3389/fgene.2020.592595/full
51. https://pmc.ncbi.nlm.nih.gov/articles/PMC11408123/
52. https://pmc.ncbi.nlm.nih.gov/articles/PMC7666045/
53. https://academic.oup.com/bioinformatics/article/31/21/3460/194878
54. https://journals.plos.org/ploscompbiol/article?id=10.1371%2Fjournal.pcbi.1005038
55. https://pubs.acs.org/doi/abs/10.1021/acs.jmedchem.0c00530
56. https://pmc.ncbi.nlm.nih.gov/articles/PMC3531114/
57. https://pmc.ncbi.nlm.nih.gov/articles/PMC403709/