---
provider: falcon
model: Edison Scientific Literature
cached: false
start_time: '2025-11-25T19:31:43.957707'
end_time: '2025-11-25T19:39:48.294651'
duration_seconds: 484.34
template_file: templates/rule_research.md
template_variables:
  rule_id: ARBA00028836
  rule_type: arba
  go_terms: GO:0043604 (amide biosynthetic process)
  conditions_summary: '**Condition Set 1:** InterPro id: TRAM/LAG1/CLN8 homology domain
    (InterPro:IPR006634) AND InterPro id: Sphingosine N-acyltransferase Lag1/Lac1-like
    (InterPro:IPR016439) AND taxon: Chordata (NCBITaxon:7711)


    **Condition Set 2:** InterPro id: Thiamin diphosphate-binding fold (InterPro:IPR029061)
    AND taxon: Thermoproteota (NCBITaxon:28889)


    **Condition Set 3:** InterPro id: Glutamate-cysteine ligase catalytic subunit
    (InterPro:IPR004308) AND InterPro id: Glutamine synthetase/guanido kinase, catalytic
    domain (InterPro:IPR014746) AND PANTHER id: PTHR11164:SF0


    **Condition Set 4:** InterPro id: Dethiobiotin synthase BioD (InterPro:IPR004472)
    AND InterPro id: P-loop containing nucleoside triphosphate hydrolase (InterPro:IPR027417)


    **Condition Set 5:** InterPro id: DNA/pantothenate metabolism flavoprotein, C-terminal
    (InterPro:IPR007085) AND InterPro id: CoaB-like superfamily (InterPro:IPR035929)


    **Condition Set 6:** InterPro id: Biotin/lipoyl attachment (InterPro:IPR000089)
    AND InterPro id: Dihydrolipoamide acetyltransferase/Pyruvate dehydrogenase protein
    X component (InterPro:IPR045257) AND taxon: Euarchontoglires (NCBITaxon:314146)


    **Condition Set 7:** InterPro id: UDP-glucuronosyl/UDP-glucosyltransferase (InterPro:IPR002213)
    AND InterPro id: Glycosyltransferase family 28, N-terminal domain (InterPro:IPR004276)
    AND taxon: Bacteria (NCBITaxon:2)


    **Condition Set 8:** InterPro id: Pantoate-beta-alanine ligase (InterPro:IPR003721)
    AND InterPro id: Rossmann-like alpha/beta/alpha sandwich fold (InterPro:IPR014729)
    AND InterPro id: Pantoate-beta-alanine ligase, C-terminal domain (InterPro:IPR042176)


    **Condition Set 9:** InterPro id: Calcineurin-like, phosphoesterase domain (InterPro:IPR004843)
    AND InterPro id: Saposin B type domain (InterPro:IPR008139) AND InterPro id: Sphingomyelin
    phosphodiesterase, C-terminal domain (InterPro:IPR045473)


    **Condition Set 10:** FunFam id: Ceramide synthase 5 (CATH.FunFam:1.10.10.60:FF:000020)
    AND taxon: Metazoa (NCBITaxon:33208)


    **Condition Set 11:** FunFam id: Glutamate--cysteine ligase catalytic subunit
    (CATH.FunFam:3.30.590.50:FF:000002) AND taxon: Eukaryota (NCBITaxon:2759)


    **Condition Set 12:** FunFam id: 2-oxoglutarate ferredoxin oxidoreductase subunit
    alpha (CATH.FunFam:3.40.50.920:FF:000009) AND FunFam id: 2-oxoglutarate ferredoxin
    oxidoreductase alpha subunit (CATH.FunFam:3.40.50.970:FF:000022) AND taxon: Archaea
    (NCBITaxon:2157)


    **Condition Set 13:** FunFam id: 2-oxoglutarate ferredoxin oxidoreductase subunit
    beta (CATH.FunFam:3.40.50.970:FF:000049) AND taxon: Thermoproteota (NCBITaxon:28889)


    **Condition Set 14:** FunFam id: Acetyltransferase component of pyruvate dehydrogenase
    complex (CATH.FunFam:2.40.50.100:FF:000010) AND FunFam id: Acetyltransferase component
    of pyruvate dehydrogenase complex (CATH.FunFam:3.30.559.10:FF:000003) AND taxon:
    Euarchontoglires (NCBITaxon:314146)


    **Condition Set 15:** FunFam id: Sterol 3-beta-glucosyltransferase UGT80A2 (CATH.FunFam:3.40.50.2000:FF:000009)
    AND FunFam id: Glycosyltransferase GtfE (CATH.FunFam:3.40.50.2000:FF:000292) AND
    taxon: Bacteria (NCBITaxon:2)


    **Condition Set 16:** FunFam id: Biotin synthase (CATH.FunFam:3.20.20.70:FF:000011)


    **Condition Set 17:** FunFam id: Phosphopantothenate--cysteine ligase 2 (CATH.FunFam:3.40.50.10300:FF:000002)


    **Condition Set 18:** FunFam id: Glutathione gamma-glutamylcysteinyltransferase
    1 (CATH.FunFam:3.90.70.30:FF:000001)


    **Condition Set 19:** FunFam id: O-methyltransferase ZRP4 (CATH.FunFam:1.10.10.10:FF:000292)
    AND FunFam id: O-methyltransferase ZRP4 (CATH.FunFam:3.40.50.150:FF:000057) AND
    taxon: Viridiplantae (NCBITaxon:33090)


    **Condition Set 20:** FunFam id: Acyl-coenzyme A:6-aminopenicillanic-acid-acyltransferase
    40 kDa form (CATH.FunFam:1.10.10.2120:FF:000002) AND FunFam id: Acyl-coenzyme
    A:6-aminopenicillanic-acid-acyltransferase 40 kDa form (CATH.FunFam:3.60.60.10:FF:000010)
    AND taxon: Fungi (NCBITaxon:4751)


    **Condition Set 21:** FunFam id: Carbamoyl-phosphate synthase large chain (CATH.FunFam:1.10.1030.10:FF:000001)
    AND FunFam id: Carbamoyl-phosphate synthase large chain (CATH.FunFam:3.30.1490.20:FF:000001)
    AND FunFam id: Carbamoyl-phosphate synthase 1, mitochondrial (CATH.FunFam:3.30.470.20:FF:000030)


    **Condition Set 22:** FunFam id: Nonribosomal peptide synthetase 1 (CATH.FunFam:1.10.1200.10:FF:000005)
    AND FunFam id: Non-ribosomal peptide synthase (CATH.FunFam:1.10.1200.10:FF:000016)
    AND FunFam id: N-(5-amino-5-carboxypentanoyl)-L-cysteinyl-D-valine synthase (CATH.FunFam:3.30.559.30:FF:000069)


    **Condition Set 23:** FunFam id: Dihydrolipoyl dehydrogenase (CATH.FunFam:3.30.390.30:FF:000001)
    AND FunFam id: Dihydrolipoyl dehydrogenase (CATH.FunFam:3.50.50.60:FF:000025)
    AND FunFam id: Dihydrolipoyl dehydrogenase, mitochondrial (CATH.FunFam:3.50.50.60:FF:000221)


    **Condition Set 24:** FunFam id: Carnosine synthase 1 (CATH.FunFam:3.30.470.20:FF:000040)
    AND FunFam id: Carnosine synthase 1 (CATH.FunFam:3.40.50.20:FF:000018) AND taxon:
    Chordata (NCBITaxon:7711)


    **Condition Set 25:** FunFam id: N-acylsphingosine amidohydrolase 2 (CATH.FunFam:2.60.40.2300:FF:000001)
    AND taxon: Mammalia (NCBITaxon:40674)


    **Condition Set 26:** FunFam id: Fatty acid 2-hydroxylase (CATH.FunFam:3.10.120.10:FF:000011)
    AND taxon: Craniata (NCBITaxon:89593)


    **Condition Set 27:** FunFam id: Glutamate--cysteine ligase regulatory subunit
    (CATH.FunFam:3.20.20.100:FF:000012) AND taxon: Vertebrata (NCBITaxon:7742)


    **Condition Set 28:** FunFam id: pantothenate kinase 2, mitochondrial (CATH.FunFam:3.30.420.510:FF:000001)
    AND taxon: Glires (NCBITaxon:314147)


    **Condition Set 29:** FunFam id: serine palmitoyltransferase 1 isoform X1 (CATH.FunFam:3.40.640.10:FF:000049)
    AND taxon: Euteleostomi (NCBITaxon:117571)


    **Condition Set 30:** FunFam id: Sphingomyelin phosphodiesterase (CATH.FunFam:3.60.21.10:FF:000045)
    AND taxon: Eutheria (NCBITaxon:9347)


    **Condition Set 31:** FunFam id: Pantothenate synthetase (CATH.FunFam:3.30.1300.10:FF:000001)


    **Condition Set 32:** FunFam id: Glutathione synthetase (CATH.FunFam:3.40.50.1760:FF:000001)


    **Condition Set 33:** FunFam id: Pantothenate synthetase (CATH.FunFam:3.40.50.620:FF:000013)


    **Condition Set 34:** FunFam id: CMP-N-acetylneuraminate-beta-galactosamide- alpha-2,3-sialyltransferase
    2 (CATH.FunFam:3.90.1480.20:FF:000002)


    **Condition Set 35:** FunFam id: Acetylserotonin O-methyltransferase (CATH.FunFam:1.10.10.10:FF:000358)
    AND FunFam id: Acetylserotonin O-methyltransferase (CATH.FunFam:3.40.50.150:FF:000146)
    AND taxon: Rodentia (NCBITaxon:9989)


    **Condition Set 36:** FunFam id: ATP-citrate synthase subunit 1 (CATH.FunFam:1.10.230.10:FF:000005)
    AND FunFam id: ATP-citrate synthase subunit (CATH.FunFam:3.40.50.261:FF:000003)
    AND FunFam id: ATP-citrate synthase beta chain protein (CATH.FunFam:3.40.50.720:FF:000136)


    **Condition Set 37:** FunFam id: Argininosuccinate lyase (CATH.FunFam:1.10.275.10:FF:000002)
    AND FunFam id: Argininosuccinate lyase (CATH.FunFam:1.10.40.30:FF:000001) AND
    taxon: Taphrinomycotina (NCBITaxon:451866)


    **Condition Set 38:** FunFam id: SPE3p Spermidine synthase (CATH.FunFam:2.30.140.10:FF:000001)
    AND FunFam id: Spermidine synthase (CATH.FunFam:3.40.50.150:FF:000013) AND taxon:
    Dikarya (NCBITaxon:451864)


    **Condition Set 39:** FunFam id: GTP cyclohydrolase 1 (CATH.FunFam:3.30.1130.10:FF:000007)
    AND FunFam id: GTP cyclohydrolase 1 (CATH.FunFam:3.30.1130.10:FF:000008) AND taxon:
    Streptophyta (NCBITaxon:35493)


    **Condition Set 40:** FunFam id: Acetyl-coenzyme A synthetase (CATH.FunFam:3.30.300.30:FF:000004)
    AND FunFam id: Acetyl-coenzyme A synthetase (CATH.FunFam:3.40.50.12780:FF:000001)
    AND taxon: Saccharomyces (NCBITaxon:4930)


    **Condition Set 41:** FunFam id: Nonribosomal siderophore peptide synthase SidC
    (CATH.FunFam:3.30.300.30:FF:000033) AND FunFam id: Nonribosomal siderophore peptide
    synthase SidC (CATH.FunFam:3.40.50.12780:FF:000024) AND taxon: Ascomycota (NCBITaxon:4890)


    **Condition Set 42:** FunFam id: ATP-citrate synthase alpha chain protein (CATH.FunFam:3.30.470.110:FF:000002)
    AND FunFam id: ATP-citrate synthase alpha chain protein (CATH.FunFam:3.40.50.261:FF:000008)
    AND taxon: Embryophyta (NCBITaxon:3193)


    **Condition Set 43:** FunFam id: N-fatty-acyl-amino acid synthase/hydrolase PM20D1
    (CATH.FunFam:1.10.150.900:FF:000003) AND FunFam id: N-fatty-acyl-amino acid synthase/hydrolase
    PM20D1 (CATH.FunFam:3.40.630.10:FF:000027)


    **Condition Set 44:** FunFam id: Carboxypeptidase Y (CATH.FunFam:1.10.287.410:FF:000001)
    AND taxon: Saccharomycotina (NCBITaxon:147537)


    **Condition Set 45:** FunFam id: Patatin like phospholipase domain containing
    1 (CATH.FunFam:3.40.1090.10:FF:000014) AND FunFam id: Patatin like phospholipase
    domain containing 1 (CATH.FunFam:3.40.1090.10:FF:000016)


    **Condition Set 46:** FunFam id: long-chain-fatty-acid--CoA ligase 6 isoform X2
    (CATH.FunFam:3.40.50.12780:FF:000006) AND taxon: Rattus (NCBITaxon:10114)


    **Condition Set 47:** FunFam id: Pimeloyl-[acyl-carrier protein] methyl ester
    esterase (CATH.FunFam:3.40.50.1820:FF:000045) AND taxon: Pseudomonadati (NCBITaxon:3379134)


    **Condition Set 48:** FunFam id: Phosphopantothenoylcysteine decarboxylase (CATH.FunFam:3.40.50.1950:FF:000004)
    AND taxon: malvids (NCBITaxon:91836)


    **Condition Set 49:** FunFam id: ATP-dependent dethiobiotin synthetase BioD (CATH.FunFam:3.40.50.300:FF:000292)
    AND taxon: Pseudomonadota (NCBITaxon:1224)


    **Condition Set 50:** FunFam id: Pyruvate dehydrogenase E1 component subunit alpha,
    mitochondrial (CATH.FunFam:3.40.50.970:FF:000020) AND taxon: Muridae (NCBITaxon:10066)


    **Condition Set 51:** FunFam id: Putative acetyltransferase NSI (CATH.FunFam:3.40.630.30:FF:000059)
    AND taxon: Liliopsida (NCBITaxon:4447)


    **Condition Set 52:** FunFam id: Arginase (CATH.FunFam:3.40.800.10:FF:000005)
    AND taxon: Schizosaccharomycetes (NCBITaxon:147554)


    **Condition Set 53:** FunFam id: Beta-1,4-galactosyltransferase 6 (CATH.FunFam:3.90.550.10:FF:000037)
    AND taxon: Mus (NCBITaxon:862507)


    **Condition Set 54:** FunFam id: Ceramide glucosyltransferase 1 (CATH.FunFam:3.90.550.10:FF:000207)
    AND taxon: Ecdysozoa (NCBITaxon:1206794)


    **Condition Set 55:** FunFam id: Nudix hydrolase 11 (CATH.FunFam:3.90.79.10:FF:000036)
    AND taxon: Tracheophyta (NCBITaxon:58023)


    **Condition Set 56:** FunFam id: sphingomyelin synthase-related protein 1 isoform
    X1 (CATH.FunFam:1.10.150.50:FF:000037)


    **Condition Set 57:** FunFam id: Glutathione synthetase (CATH.FunFam:3.30.1490.50:FF:000002)


    **Condition Set 58:** FunFam id: Coenzyme A biosynthesis bifunctional protein
    CoaBC (CATH.FunFam:3.40.50.10300:FF:000001)


    **Condition Set 59:** FunFam id: Pantothenate kinase (CATH.FunFam:3.40.50.300:FF:000242)


    **Condition Set 60:** FunFam id: Phosphopantetheine adenylyltransferase (CATH.FunFam:3.40.50.620:FF:000012)


    **Condition Set 61:** FunFam id: Adenosylmethionine-8-amino-7-oxononanoate aminotransferase
    (CATH.FunFam:3.40.640.10:FF:000041)


    *These condition sets are OR-ed together (any match triggers the annotation)*'
  protein_count: 0 total (0 reviewed, 0 unreviewed)
provider_config:
  timeout: 600
  max_retries: 3
  parameters:
    allowed_domains: []
    temperature: 0.1
citation_count: 9
---

## Question

# UniProt Rule Research

## Rule Context

- **Rule ID:** ARBA00028836
- **Rule Type:** arba
- **GO Term(s) Predicted:** GO:0043604 (amide biosynthetic process)
- **Proteins Annotated:** 0 total (0 reviewed, 0 unreviewed)

### Condition Summary

**Condition Set 1:** InterPro id: TRAM/LAG1/CLN8 homology domain (InterPro:IPR006634) AND InterPro id: Sphingosine N-acyltransferase Lag1/Lac1-like (InterPro:IPR016439) AND taxon: Chordata (NCBITaxon:7711)

**Condition Set 2:** InterPro id: Thiamin diphosphate-binding fold (InterPro:IPR029061) AND taxon: Thermoproteota (NCBITaxon:28889)

**Condition Set 3:** InterPro id: Glutamate-cysteine ligase catalytic subunit (InterPro:IPR004308) AND InterPro id: Glutamine synthetase/guanido kinase, catalytic domain (InterPro:IPR014746) AND PANTHER id: PTHR11164:SF0

**Condition Set 4:** InterPro id: Dethiobiotin synthase BioD (InterPro:IPR004472) AND InterPro id: P-loop containing nucleoside triphosphate hydrolase (InterPro:IPR027417)

**Condition Set 5:** InterPro id: DNA/pantothenate metabolism flavoprotein, C-terminal (InterPro:IPR007085) AND InterPro id: CoaB-like superfamily (InterPro:IPR035929)

**Condition Set 6:** InterPro id: Biotin/lipoyl attachment (InterPro:IPR000089) AND InterPro id: Dihydrolipoamide acetyltransferase/Pyruvate dehydrogenase protein X component (InterPro:IPR045257) AND taxon: Euarchontoglires (NCBITaxon:314146)

**Condition Set 7:** InterPro id: UDP-glucuronosyl/UDP-glucosyltransferase (InterPro:IPR002213) AND InterPro id: Glycosyltransferase family 28, N-terminal domain (InterPro:IPR004276) AND taxon: Bacteria (NCBITaxon:2)

**Condition Set 8:** InterPro id: Pantoate-beta-alanine ligase (InterPro:IPR003721) AND InterPro id: Rossmann-like alpha/beta/alpha sandwich fold (InterPro:IPR014729) AND InterPro id: Pantoate-beta-alanine ligase, C-terminal domain (InterPro:IPR042176)

**Condition Set 9:** InterPro id: Calcineurin-like, phosphoesterase domain (InterPro:IPR004843) AND InterPro id: Saposin B type domain (InterPro:IPR008139) AND InterPro id: Sphingomyelin phosphodiesterase, C-terminal domain (InterPro:IPR045473)

**Condition Set 10:** FunFam id: Ceramide synthase 5 (CATH.FunFam:1.10.10.60:FF:000020) AND taxon: Metazoa (NCBITaxon:33208)

**Condition Set 11:** FunFam id: Glutamate--cysteine ligase catalytic subunit (CATH.FunFam:3.30.590.50:FF:000002) AND taxon: Eukaryota (NCBITaxon:2759)

**Condition Set 12:** FunFam id: 2-oxoglutarate ferredoxin oxidoreductase subunit alpha (CATH.FunFam:3.40.50.920:FF:000009) AND FunFam id: 2-oxoglutarate ferredoxin oxidoreductase alpha subunit (CATH.FunFam:3.40.50.970:FF:000022) AND taxon: Archaea (NCBITaxon:2157)

**Condition Set 13:** FunFam id: 2-oxoglutarate ferredoxin oxidoreductase subunit beta (CATH.FunFam:3.40.50.970:FF:000049) AND taxon: Thermoproteota (NCBITaxon:28889)

**Condition Set 14:** FunFam id: Acetyltransferase component of pyruvate dehydrogenase complex (CATH.FunFam:2.40.50.100:FF:000010) AND FunFam id: Acetyltransferase component of pyruvate dehydrogenase complex (CATH.FunFam:3.30.559.10:FF:000003) AND taxon: Euarchontoglires (NCBITaxon:314146)

**Condition Set 15:** FunFam id: Sterol 3-beta-glucosyltransferase UGT80A2 (CATH.FunFam:3.40.50.2000:FF:000009) AND FunFam id: Glycosyltransferase GtfE (CATH.FunFam:3.40.50.2000:FF:000292) AND taxon: Bacteria (NCBITaxon:2)

**Condition Set 16:** FunFam id: Biotin synthase (CATH.FunFam:3.20.20.70:FF:000011)

**Condition Set 17:** FunFam id: Phosphopantothenate--cysteine ligase 2 (CATH.FunFam:3.40.50.10300:FF:000002)

**Condition Set 18:** FunFam id: Glutathione gamma-glutamylcysteinyltransferase 1 (CATH.FunFam:3.90.70.30:FF:000001)

**Condition Set 19:** FunFam id: O-methyltransferase ZRP4 (CATH.FunFam:1.10.10.10:FF:000292) AND FunFam id: O-methyltransferase ZRP4 (CATH.FunFam:3.40.50.150:FF:000057) AND taxon: Viridiplantae (NCBITaxon:33090)

**Condition Set 20:** FunFam id: Acyl-coenzyme A:6-aminopenicillanic-acid-acyltransferase 40 kDa form (CATH.FunFam:1.10.10.2120:FF:000002) AND FunFam id: Acyl-coenzyme A:6-aminopenicillanic-acid-acyltransferase 40 kDa form (CATH.FunFam:3.60.60.10:FF:000010) AND taxon: Fungi (NCBITaxon:4751)

**Condition Set 21:** FunFam id: Carbamoyl-phosphate synthase large chain (CATH.FunFam:1.10.1030.10:FF:000001) AND FunFam id: Carbamoyl-phosphate synthase large chain (CATH.FunFam:3.30.1490.20:FF:000001) AND FunFam id: Carbamoyl-phosphate synthase 1, mitochondrial (CATH.FunFam:3.30.470.20:FF:000030)

**Condition Set 22:** FunFam id: Nonribosomal peptide synthetase 1 (CATH.FunFam:1.10.1200.10:FF:000005) AND FunFam id: Non-ribosomal peptide synthase (CATH.FunFam:1.10.1200.10:FF:000016) AND FunFam id: N-(5-amino-5-carboxypentanoyl)-L-cysteinyl-D-valine synthase (CATH.FunFam:3.30.559.30:FF:000069)

**Condition Set 23:** FunFam id: Dihydrolipoyl dehydrogenase (CATH.FunFam:3.30.390.30:FF:000001) AND FunFam id: Dihydrolipoyl dehydrogenase (CATH.FunFam:3.50.50.60:FF:000025) AND FunFam id: Dihydrolipoyl dehydrogenase, mitochondrial (CATH.FunFam:3.50.50.60:FF:000221)

**Condition Set 24:** FunFam id: Carnosine synthase 1 (CATH.FunFam:3.30.470.20:FF:000040) AND FunFam id: Carnosine synthase 1 (CATH.FunFam:3.40.50.20:FF:000018) AND taxon: Chordata (NCBITaxon:7711)

**Condition Set 25:** FunFam id: N-acylsphingosine amidohydrolase 2 (CATH.FunFam:2.60.40.2300:FF:000001) AND taxon: Mammalia (NCBITaxon:40674)

**Condition Set 26:** FunFam id: Fatty acid 2-hydroxylase (CATH.FunFam:3.10.120.10:FF:000011) AND taxon: Craniata (NCBITaxon:89593)

**Condition Set 27:** FunFam id: Glutamate--cysteine ligase regulatory subunit (CATH.FunFam:3.20.20.100:FF:000012) AND taxon: Vertebrata (NCBITaxon:7742)

**Condition Set 28:** FunFam id: pantothenate kinase 2, mitochondrial (CATH.FunFam:3.30.420.510:FF:000001) AND taxon: Glires (NCBITaxon:314147)

**Condition Set 29:** FunFam id: serine palmitoyltransferase 1 isoform X1 (CATH.FunFam:3.40.640.10:FF:000049) AND taxon: Euteleostomi (NCBITaxon:117571)

**Condition Set 30:** FunFam id: Sphingomyelin phosphodiesterase (CATH.FunFam:3.60.21.10:FF:000045) AND taxon: Eutheria (NCBITaxon:9347)

**Condition Set 31:** FunFam id: Pantothenate synthetase (CATH.FunFam:3.30.1300.10:FF:000001)

**Condition Set 32:** FunFam id: Glutathione synthetase (CATH.FunFam:3.40.50.1760:FF:000001)

**Condition Set 33:** FunFam id: Pantothenate synthetase (CATH.FunFam:3.40.50.620:FF:000013)

**Condition Set 34:** FunFam id: CMP-N-acetylneuraminate-beta-galactosamide- alpha-2,3-sialyltransferase 2 (CATH.FunFam:3.90.1480.20:FF:000002)

**Condition Set 35:** FunFam id: Acetylserotonin O-methyltransferase (CATH.FunFam:1.10.10.10:FF:000358) AND FunFam id: Acetylserotonin O-methyltransferase (CATH.FunFam:3.40.50.150:FF:000146) AND taxon: Rodentia (NCBITaxon:9989)

**Condition Set 36:** FunFam id: ATP-citrate synthase subunit 1 (CATH.FunFam:1.10.230.10:FF:000005) AND FunFam id: ATP-citrate synthase subunit (CATH.FunFam:3.40.50.261:FF:000003) AND FunFam id: ATP-citrate synthase beta chain protein (CATH.FunFam:3.40.50.720:FF:000136)

**Condition Set 37:** FunFam id: Argininosuccinate lyase (CATH.FunFam:1.10.275.10:FF:000002) AND FunFam id: Argininosuccinate lyase (CATH.FunFam:1.10.40.30:FF:000001) AND taxon: Taphrinomycotina (NCBITaxon:451866)

**Condition Set 38:** FunFam id: SPE3p Spermidine synthase (CATH.FunFam:2.30.140.10:FF:000001) AND FunFam id: Spermidine synthase (CATH.FunFam:3.40.50.150:FF:000013) AND taxon: Dikarya (NCBITaxon:451864)

**Condition Set 39:** FunFam id: GTP cyclohydrolase 1 (CATH.FunFam:3.30.1130.10:FF:000007) AND FunFam id: GTP cyclohydrolase 1 (CATH.FunFam:3.30.1130.10:FF:000008) AND taxon: Streptophyta (NCBITaxon:35493)

**Condition Set 40:** FunFam id: Acetyl-coenzyme A synthetase (CATH.FunFam:3.30.300.30:FF:000004) AND FunFam id: Acetyl-coenzyme A synthetase (CATH.FunFam:3.40.50.12780:FF:000001) AND taxon: Saccharomyces (NCBITaxon:4930)

**Condition Set 41:** FunFam id: Nonribosomal siderophore peptide synthase SidC (CATH.FunFam:3.30.300.30:FF:000033) AND FunFam id: Nonribosomal siderophore peptide synthase SidC (CATH.FunFam:3.40.50.12780:FF:000024) AND taxon: Ascomycota (NCBITaxon:4890)

**Condition Set 42:** FunFam id: ATP-citrate synthase alpha chain protein (CATH.FunFam:3.30.470.110:FF:000002) AND FunFam id: ATP-citrate synthase alpha chain protein (CATH.FunFam:3.40.50.261:FF:000008) AND taxon: Embryophyta (NCBITaxon:3193)

**Condition Set 43:** FunFam id: N-fatty-acyl-amino acid synthase/hydrolase PM20D1 (CATH.FunFam:1.10.150.900:FF:000003) AND FunFam id: N-fatty-acyl-amino acid synthase/hydrolase PM20D1 (CATH.FunFam:3.40.630.10:FF:000027)

**Condition Set 44:** FunFam id: Carboxypeptidase Y (CATH.FunFam:1.10.287.410:FF:000001) AND taxon: Saccharomycotina (NCBITaxon:147537)

**Condition Set 45:** FunFam id: Patatin like phospholipase domain containing 1 (CATH.FunFam:3.40.1090.10:FF:000014) AND FunFam id: Patatin like phospholipase domain containing 1 (CATH.FunFam:3.40.1090.10:FF:000016)

**Condition Set 46:** FunFam id: long-chain-fatty-acid--CoA ligase 6 isoform X2 (CATH.FunFam:3.40.50.12780:FF:000006) AND taxon: Rattus (NCBITaxon:10114)

**Condition Set 47:** FunFam id: Pimeloyl-[acyl-carrier protein] methyl ester esterase (CATH.FunFam:3.40.50.1820:FF:000045) AND taxon: Pseudomonadati (NCBITaxon:3379134)

**Condition Set 48:** FunFam id: Phosphopantothenoylcysteine decarboxylase (CATH.FunFam:3.40.50.1950:FF:000004) AND taxon: malvids (NCBITaxon:91836)

**Condition Set 49:** FunFam id: ATP-dependent dethiobiotin synthetase BioD (CATH.FunFam:3.40.50.300:FF:000292) AND taxon: Pseudomonadota (NCBITaxon:1224)

**Condition Set 50:** FunFam id: Pyruvate dehydrogenase E1 component subunit alpha, mitochondrial (CATH.FunFam:3.40.50.970:FF:000020) AND taxon: Muridae (NCBITaxon:10066)

**Condition Set 51:** FunFam id: Putative acetyltransferase NSI (CATH.FunFam:3.40.630.30:FF:000059) AND taxon: Liliopsida (NCBITaxon:4447)

**Condition Set 52:** FunFam id: Arginase (CATH.FunFam:3.40.800.10:FF:000005) AND taxon: Schizosaccharomycetes (NCBITaxon:147554)

**Condition Set 53:** FunFam id: Beta-1,4-galactosyltransferase 6 (CATH.FunFam:3.90.550.10:FF:000037) AND taxon: Mus (NCBITaxon:862507)

**Condition Set 54:** FunFam id: Ceramide glucosyltransferase 1 (CATH.FunFam:3.90.550.10:FF:000207) AND taxon: Ecdysozoa (NCBITaxon:1206794)

**Condition Set 55:** FunFam id: Nudix hydrolase 11 (CATH.FunFam:3.90.79.10:FF:000036) AND taxon: Tracheophyta (NCBITaxon:58023)

**Condition Set 56:** FunFam id: sphingomyelin synthase-related protein 1 isoform X1 (CATH.FunFam:1.10.150.50:FF:000037)

**Condition Set 57:** FunFam id: Glutathione synthetase (CATH.FunFam:3.30.1490.50:FF:000002)

**Condition Set 58:** FunFam id: Coenzyme A biosynthesis bifunctional protein CoaBC (CATH.FunFam:3.40.50.10300:FF:000001)

**Condition Set 59:** FunFam id: Pantothenate kinase (CATH.FunFam:3.40.50.300:FF:000242)

**Condition Set 60:** FunFam id: Phosphopantetheine adenylyltransferase (CATH.FunFam:3.40.50.620:FF:000012)

**Condition Set 61:** FunFam id: Adenosylmethionine-8-amino-7-oxononanoate aminotransferase (CATH.FunFam:3.40.640.10:FF:000041)

*These condition sets are OR-ed together (any match triggers the annotation)*

---

## Research Objective

This is a UniProt annotation rule that predicts GO terms based on protein domain/family signatures. Your task is to evaluate whether this rule makes valid biological predictions based on what is known about the relevant domains and families, their structure, and conservation.

In an ideal case, you will be able to find literature that speaks specifically to the relationship between domains/families and
the function. Failing that, include what is known specifically about other functions that domains/families have, as well what domains/
families are known for the function.

## Research Questions

### 1. Domain/Signature Context

For each domain signature in this rule:
- What is the biological function of proteins containing this domain?
- Is this domain diagnostic for the predicted function?
- Are there known subfamilies with different functions?
- Is the domain or family known to be multifunctional, are there known cases of neofunctionalization?

### 2. GO Term Appropriateness

For the predicted GO term(s) **GO:0043604 (amide biosynthetic process)**:
- Is this GO term appropriate for proteins matching these conditions?
- Is the term too broad? (e.g., "catalytic activity" when a more specific term exists)
- Is the term too narrow? (e.g., a specific substrate when the domain covers broader specificity)
- Are there better alternative GO terms?

### 3. Literature Support

- What experimental evidence supports this functional annotation?
- Are there key papers describing this protein family/enzyme?
- Are there any contradictory findings in the literature?
- Is this a well-established function or more speculative?

### 4. Taxonomic Considerations

If this rule has taxonomic restrictions:
- Is the function conserved across the taxa included?
- Are there taxa that should be excluded (different function)?
- Are there taxa that could be included but are currently excluded?

### 5. Rule Logic Assessment

- Do the domain combinations make biological sense?
- Are there redundant conditions that could be removed?
- Could false positives arise from these conditions?

## Output Format

Please provide your findings in a narrative format with citations. Structure your response as:

1. **Executive Summary** - Brief assessment of rule validity
2. **Domain Analysis** - What each domain/signature represents
3. **GO Term Evaluation** - Assessment of the predicted annotation
4. **Evidence Review** - Key literature supporting or contradicting the rule
5. **Recommendations** - Suggested improvements or concerns

Be sure to include citations for all statements.

## Output

Question: You are an expert researcher providing comprehensive, well-cited information.

Provide detailed information focusing on:
1. Key concepts and definitions with current understanding
2. Recent developments and latest research (prioritize 2023-2024 sources)
3. Current applications and real-world implementations
4. Expert opinions and analysis from authoritative sources
5. Relevant statistics and data from recent studies

Format as a comprehensive research report with proper citations. Include URLs and publication dates where available.
Always prioritize recent, authoritative sources and provide specific citations for all major claims.

# UniProt Rule Research

## Rule Context

- **Rule ID:** ARBA00028836
- **Rule Type:** arba
- **GO Term(s) Predicted:** GO:0043604 (amide biosynthetic process)
- **Proteins Annotated:** 0 total (0 reviewed, 0 unreviewed)

### Condition Summary

**Condition Set 1:** InterPro id: TRAM/LAG1/CLN8 homology domain (InterPro:IPR006634) AND InterPro id: Sphingosine N-acyltransferase Lag1/Lac1-like (InterPro:IPR016439) AND taxon: Chordata (NCBITaxon:7711)

**Condition Set 2:** InterPro id: Thiamin diphosphate-binding fold (InterPro:IPR029061) AND taxon: Thermoproteota (NCBITaxon:28889)

**Condition Set 3:** InterPro id: Glutamate-cysteine ligase catalytic subunit (InterPro:IPR004308) AND InterPro id: Glutamine synthetase/guanido kinase, catalytic domain (InterPro:IPR014746) AND PANTHER id: PTHR11164:SF0

**Condition Set 4:** InterPro id: Dethiobiotin synthase BioD (InterPro:IPR004472) AND InterPro id: P-loop containing nucleoside triphosphate hydrolase (InterPro:IPR027417)

**Condition Set 5:** InterPro id: DNA/pantothenate metabolism flavoprotein, C-terminal (InterPro:IPR007085) AND InterPro id: CoaB-like superfamily (InterPro:IPR035929)

**Condition Set 6:** InterPro id: Biotin/lipoyl attachment (InterPro:IPR000089) AND InterPro id: Dihydrolipoamide acetyltransferase/Pyruvate dehydrogenase protein X component (InterPro:IPR045257) AND taxon: Euarchontoglires (NCBITaxon:314146)

**Condition Set 7:** InterPro id: UDP-glucuronosyl/UDP-glucosyltransferase (InterPro:IPR002213) AND InterPro id: Glycosyltransferase family 28, N-terminal domain (InterPro:IPR004276) AND taxon: Bacteria (NCBITaxon:2)

**Condition Set 8:** InterPro id: Pantoate-beta-alanine ligase (InterPro:IPR003721) AND InterPro id: Rossmann-like alpha/beta/alpha sandwich fold (InterPro:IPR014729) AND InterPro id: Pantoate-beta-alanine ligase, C-terminal domain (InterPro:IPR042176)

**Condition Set 9:** InterPro id: Calcineurin-like, phosphoesterase domain (InterPro:IPR004843) AND InterPro id: Saposin B type domain (InterPro:IPR008139) AND InterPro id: Sphingomyelin phosphodiesterase, C-terminal domain (InterPro:IPR045473)

**Condition Set 10:** FunFam id: Ceramide synthase 5 (CATH.FunFam:1.10.10.60:FF:000020) AND taxon: Metazoa (NCBITaxon:33208)

**Condition Set 11:** FunFam id: Glutamate--cysteine ligase catalytic subunit (CATH.FunFam:3.30.590.50:FF:000002) AND taxon: Eukaryota (NCBITaxon:2759)

**Condition Set 12:** FunFam id: 2-oxoglutarate ferredoxin oxidoreductase subunit alpha (CATH.FunFam:3.40.50.920:FF:000009) AND FunFam id: 2-oxoglutarate ferredoxin oxidoreductase alpha subunit (CATH.FunFam:3.40.50.970:FF:000022) AND taxon: Archaea (NCBITaxon:2157)

**Condition Set 13:** FunFam id: 2-oxoglutarate ferredoxin oxidoreductase subunit beta (CATH.FunFam:3.40.50.970:FF:000049) AND taxon: Thermoproteota (NCBITaxon:28889)

**Condition Set 14:** FunFam id: Acetyltransferase component of pyruvate dehydrogenase complex (CATH.FunFam:2.40.50.100:FF:000010) AND FunFam id: Acetyltransferase component of pyruvate dehydrogenase complex (CATH.FunFam:3.30.559.10:FF:000003) AND taxon: Euarchontoglires (NCBITaxon:314146)

**Condition Set 15:** FunFam id: Sterol 3-beta-glucosyltransferase UGT80A2 (CATH.FunFam:3.40.50.2000:FF:000009) AND FunFam id: Glycosyltransferase GtfE (CATH.FunFam:3.40.50.2000:FF:000292) AND taxon: Bacteria (NCBITaxon:2)

**Condition Set 16:** FunFam id: Biotin synthase (CATH.FunFam:3.20.20.70:FF:000011)

**Condition Set 17:** FunFam id: Phosphopantothenate--cysteine ligase 2 (CATH.FunFam:3.40.50.10300:FF:000002)

**Condition Set 18:** FunFam id: Glutathione gamma-glutamylcysteinyltransferase 1 (CATH.FunFam:3.90.70.30:FF:000001)

**Condition Set 19:** FunFam id: O-methyltransferase ZRP4 (CATH.FunFam:1.10.10.10:FF:000292) AND FunFam id: O-methyltransferase ZRP4 (CATH.FunFam:3.40.50.150:FF:000057) AND taxon: Viridiplantae (NCBITaxon:33090)

**Condition Set 20:** FunFam id: Acyl-coenzyme A:6-aminopenicillanic-acid-acyltransferase 40 kDa form (CATH.FunFam:1.10.10.2120:FF:000002) AND FunFam id: Acyl-coenzyme A:6-aminopenicillanic-acid-acyltransferase 40 kDa form (CATH.FunFam:3.60.60.10:FF:000010) AND taxon: Fungi (NCBITaxon:4751)

**Condition Set 21:** FunFam id: Carbamoyl-phosphate synthase large chain (CATH.FunFam:1.10.1030.10:FF:000001) AND FunFam id: Carbamoyl-phosphate synthase large chain (CATH.FunFam:3.30.1490.20:FF:000001) AND FunFam id: Carbamoyl-phosphate synthase 1, mitochondrial (CATH.FunFam:3.30.470.20:FF:000030)

**Condition Set 22:** FunFam id: Nonribosomal peptide synthetase 1 (CATH.FunFam:1.10.1200.10:FF:000005) AND FunFam id: Non-ribosomal peptide synthase (CATH.FunFam:1.10.1200.10:FF:000016) AND FunFam id: N-(5-amino-5-carboxypentanoyl)-L-cysteinyl-D-valine synthase (CATH.FunFam:3.30.559.30:FF:000069)

**Condition Set 23:** FunFam id: Dihydrolipoyl dehydrogenase (CATH.FunFam:3.30.390.30:FF:000001) AND FunFam id: Dihydrolipoyl dehydrogenase (CATH.FunFam:3.50.50.60:FF:000025) AND FunFam id: Dihydrolipoyl dehydrogenase, mitochondrial (CATH.FunFam:3.50.50.60:FF:000221)

**Condition Set 24:** FunFam id: Carnosine synthase 1 (CATH.FunFam:3.30.470.20:FF:000040) AND FunFam id: Carnosine synthase 1 (CATH.FunFam:3.40.50.20:FF:000018) AND taxon: Chordata (NCBITaxon:7711)

**Condition Set 25:** FunFam id: N-acylsphingosine amidohydrolase 2 (CATH.FunFam:2.60.40.2300:FF:000001) AND taxon: Mammalia (NCBITaxon:40674)

**Condition Set 26:** FunFam id: Fatty acid 2-hydroxylase (CATH.FunFam:3.10.120.10:FF:000011) AND taxon: Craniata (NCBITaxon:89593)

**Condition Set 27:** FunFam id: Glutamate--cysteine ligase regulatory subunit (CATH.FunFam:3.20.20.100:FF:000012) AND taxon: Vertebrata (NCBITaxon:7742)

**Condition Set 28:** FunFam id: pantothenate kinase 2, mitochondrial (CATH.FunFam:3.30.420.510:FF:000001) AND taxon: Glires (NCBITaxon:314147)

**Condition Set 29:** FunFam id: serine palmitoyltransferase 1 isoform X1 (CATH.FunFam:3.40.640.10:FF:000049) AND taxon: Euteleostomi (NCBITaxon:117571)

**Condition Set 30:** FunFam id: Sphingomyelin phosphodiesterase (CATH.FunFam:3.60.21.10:FF:000045) AND taxon: Eutheria (NCBITaxon:9347)

**Condition Set 31:** FunFam id: Pantothenate synthetase (CATH.FunFam:3.30.1300.10:FF:000001)

**Condition Set 32:** FunFam id: Glutathione synthetase (CATH.FunFam:3.40.50.1760:FF:000001)

**Condition Set 33:** FunFam id: Pantothenate synthetase (CATH.FunFam:3.40.50.620:FF:000013)

**Condition Set 34:** FunFam id: CMP-N-acetylneuraminate-beta-galactosamide- alpha-2,3-sialyltransferase 2 (CATH.FunFam:3.90.1480.20:FF:000002)

**Condition Set 35:** FunFam id: Acetylserotonin O-methyltransferase (CATH.FunFam:1.10.10.10:FF:000358) AND FunFam id: Acetylserotonin O-methyltransferase (CATH.FunFam:3.40.50.150:FF:000146) AND taxon: Rodentia (NCBITaxon:9989)

**Condition Set 36:** FunFam id: ATP-citrate synthase subunit 1 (CATH.FunFam:1.10.230.10:FF:000005) AND FunFam id: ATP-citrate synthase subunit (CATH.FunFam:3.40.50.261:FF:000003) AND FunFam id: ATP-citrate synthase beta chain protein (CATH.FunFam:3.40.50.720:FF:000136)

**Condition Set 37:** FunFam id: Argininosuccinate lyase (CATH.FunFam:1.10.275.10:FF:000002) AND FunFam id: Argininosuccinate lyase (CATH.FunFam:1.10.40.30:FF:000001) AND taxon: Taphrinomycotina (NCBITaxon:451866)

**Condition Set 38:** FunFam id: SPE3p Spermidine synthase (CATH.FunFam:2.30.140.10:FF:000001) AND FunFam id: Spermidine synthase (CATH.FunFam:3.40.50.150:FF:000013) AND taxon: Dikarya (NCBITaxon:451864)

**Condition Set 39:** FunFam id: GTP cyclohydrolase 1 (CATH.FunFam:3.30.1130.10:FF:000007) AND FunFam id: GTP cyclohydrolase 1 (CATH.FunFam:3.30.1130.10:FF:000008) AND taxon: Streptophyta (NCBITaxon:35493)

**Condition Set 40:** FunFam id: Acetyl-coenzyme A synthetase (CATH.FunFam:3.30.300.30:FF:000004) AND FunFam id: Acetyl-coenzyme A synthetase (CATH.FunFam:3.40.50.12780:FF:000001) AND taxon: Saccharomyces (NCBITaxon:4930)

**Condition Set 41:** FunFam id: Nonribosomal siderophore peptide synthase SidC (CATH.FunFam:3.30.300.30:FF:000033) AND FunFam id: Nonribosomal siderophore peptide synthase SidC (CATH.FunFam:3.40.50.12780:FF:000024) AND taxon: Ascomycota (NCBITaxon:4890)

**Condition Set 42:** FunFam id: ATP-citrate synthase alpha chain protein (CATH.FunFam:3.30.470.110:FF:000002) AND FunFam id: ATP-citrate synthase alpha chain protein (CATH.FunFam:3.40.50.261:FF:000008) AND taxon: Embryophyta (NCBITaxon:3193)

**Condition Set 43:** FunFam id: N-fatty-acyl-amino acid synthase/hydrolase PM20D1 (CATH.FunFam:1.10.150.900:FF:000003) AND FunFam id: N-fatty-acyl-amino acid synthase/hydrolase PM20D1 (CATH.FunFam:3.40.630.10:FF:000027)

**Condition Set 44:** FunFam id: Carboxypeptidase Y (CATH.FunFam:1.10.287.410:FF:000001) AND taxon: Saccharomycotina (NCBITaxon:147537)

**Condition Set 45:** FunFam id: Patatin like phospholipase domain containing 1 (CATH.FunFam:3.40.1090.10:FF:000014) AND FunFam id: Patatin like phospholipase domain containing 1 (CATH.FunFam:3.40.1090.10:FF:000016)

**Condition Set 46:** FunFam id: long-chain-fatty-acid--CoA ligase 6 isoform X2 (CATH.FunFam:3.40.50.12780:FF:000006) AND taxon: Rattus (NCBITaxon:10114)

**Condition Set 47:** FunFam id: Pimeloyl-[acyl-carrier protein] methyl ester esterase (CATH.FunFam:3.40.50.1820:FF:000045) AND taxon: Pseudomonadati (NCBITaxon:3379134)

**Condition Set 48:** FunFam id: Phosphopantothenoylcysteine decarboxylase (CATH.FunFam:3.40.50.1950:FF:000004) AND taxon: malvids (NCBITaxon:91836)

**Condition Set 49:** FunFam id: ATP-dependent dethiobiotin synthetase BioD (CATH.FunFam:3.40.50.300:FF:000292) AND taxon: Pseudomonadota (NCBITaxon:1224)

**Condition Set 50:** FunFam id: Pyruvate dehydrogenase E1 component subunit alpha, mitochondrial (CATH.FunFam:3.40.50.970:FF:000020) AND taxon: Muridae (NCBITaxon:10066)

**Condition Set 51:** FunFam id: Putative acetyltransferase NSI (CATH.FunFam:3.40.630.30:FF:000059) AND taxon: Liliopsida (NCBITaxon:4447)

**Condition Set 52:** FunFam id: Arginase (CATH.FunFam:3.40.800.10:FF:000005) AND taxon: Schizosaccharomycetes (NCBITaxon:147554)

**Condition Set 53:** FunFam id: Beta-1,4-galactosyltransferase 6 (CATH.FunFam:3.90.550.10:FF:000037) AND taxon: Mus (NCBITaxon:862507)

**Condition Set 54:** FunFam id: Ceramide glucosyltransferase 1 (CATH.FunFam:3.90.550.10:FF:000207) AND taxon: Ecdysozoa (NCBITaxon:1206794)

**Condition Set 55:** FunFam id: Nudix hydrolase 11 (CATH.FunFam:3.90.79.10:FF:000036) AND taxon: Tracheophyta (NCBITaxon:58023)

**Condition Set 56:** FunFam id: sphingomyelin synthase-related protein 1 isoform X1 (CATH.FunFam:1.10.150.50:FF:000037)

**Condition Set 57:** FunFam id: Glutathione synthetase (CATH.FunFam:3.30.1490.50:FF:000002)

**Condition Set 58:** FunFam id: Coenzyme A biosynthesis bifunctional protein CoaBC (CATH.FunFam:3.40.50.10300:FF:000001)

**Condition Set 59:** FunFam id: Pantothenate kinase (CATH.FunFam:3.40.50.300:FF:000242)

**Condition Set 60:** FunFam id: Phosphopantetheine adenylyltransferase (CATH.FunFam:3.40.50.620:FF:000012)

**Condition Set 61:** FunFam id: Adenosylmethionine-8-amino-7-oxononanoate aminotransferase (CATH.FunFam:3.40.640.10:FF:000041)

*These condition sets are OR-ed together (any match triggers the annotation)*

---

## Research Objective

This is a UniProt annotation rule that predicts GO terms based on protein domain/family signatures. Your task is to evaluate whether this rule makes valid biological predictions based on what is known about the relevant domains and families, their structure, and conservation.

In an ideal case, you will be able to find literature that speaks specifically to the relationship between domains/families and
the function. Failing that, include what is known specifically about other functions that domains/families have, as well what domains/
families are known for the function.

## Research Questions

### 1. Domain/Signature Context

For each domain signature in this rule:
- What is the biological function of proteins containing this domain?
- Is this domain diagnostic for the predicted function?
- Are there known subfamilies with different functions?
- Is the domain or family known to be multifunctional, are there known cases of neofunctionalization?

### 2. GO Term Appropriateness

For the predicted GO term(s) **GO:0043604 (amide biosynthetic process)**:
- Is this GO term appropriate for proteins matching these conditions?
- Is the term too broad? (e.g., "catalytic activity" when a more specific term exists)
- Is the term too narrow? (e.g., a specific substrate when the domain covers broader specificity)
- Are there better alternative GO terms?

### 3. Literature Support

- What experimental evidence supports this functional annotation?
- Are there key papers describing this protein family/enzyme?
- Are there any contradictory findings in the literature?
- Is this a well-established function or more speculative?

### 4. Taxonomic Considerations

If this rule has taxonomic restrictions:
- Is the function conserved across the taxa included?
- Are there taxa that should be excluded (different function)?
- Are there taxa that could be included but are currently excluded?

### 5. Rule Logic Assessment

- Do the domain combinations make biological sense?
- Are there redundant conditions that could be removed?
- Could false positives arise from these conditions?

## Output Format

Please provide your findings in a narrative format with citations. Structure your response as:

1. **Executive Summary** - Brief assessment of rule validity
2. **Domain Analysis** - What each domain/signature represents
3. **GO Term Evaluation** - Assessment of the predicted annotation
4. **Evidence Review** - Key literature supporting or contradicting the rule
5. **Recommendations** - Suggested improvements or concerns

Be sure to include citations for all statements.

Plan
We decomposed the task into: (i) mapping the rule’s domain/family condition sets to known functions; (ii) collecting recent (2023–2024) sources on enzymes that are unequivocally amide-forming; (iii) evaluating whether the predicted GO term (GO:0043604) is appropriate per domain/family; (iv) identifying taxonomic/logical constraints and potential false positives; and (v) summarizing recommendations. Our searches retrieved high-quality, recent reviews and mechanistic studies for enzyme classes that directly catalyze amide bond formation (notably adenylating enzymes, ATP‑grasp enzymes, and NRPS machinery) and one preprint clarifying additional amide-forming machinery in NRPS contexts. We did not retrieve new, citable 2023–2024 sources within this evidence set for several families (e.g., TLC/Lag1 ceramide synthases, carnosine synthase, PM20D1, glycosyl/sialyltransferases, arginase, NUDIX, SMPD/SMS). Where such coverage is missing, we note that the assessment is partial.

1. Executive Summary
- Direct amide-bond-forming enzyme classes are well supported in the retrieved, recent literature: adenylating enzymes (including pantothenate synthetase/PanC), ATP‑grasp ligases, and nonribosomal peptide synthetase (NRPS) condensation machinery. These catalyze formation of peptide or other amide bonds and align with GO:0043604 (amide biosynthetic process) (lubberink2023biocatalyticamidebond pages 6-7, lubberink2023biocatalyticamidebond pages 7-8, lubberink2023biocatalyticamidebond pages 8-9, yu2023pearlcatalyzedpeptidebond pages 1-5).
- Pantothenate synthetase (PanC) specifically catalyzes an ATP‑dependent condensation of pantoate with β‑alanine to form pantothenate via an acyl‑adenylate intermediate—an archetypal enzyme for enzymatic amide formation—supporting GO:0043604 when PanC signatures are present (lubberink2023biocatalyticamidebond pages 6-7, lubberink2023biocatalyticamidebond pages 7-8).
- NRPS pathways: Condensation (C) domains form peptide bonds between carrier‑tethered substrates; additional amide-forming machinery (e.g., PEARLs) can extend peptidyl scaffolds outside canonical NRPS directionality, reinforcing that NRPS domain signatures provide strong evidence for amide biosynthesis (yu2023pearlcatalyzedpeptidebond pages 1-5).
- ATP‑grasp ligases also catalyze amide formation via acyl‑phosphate intermediates; when such signatures are present, GO:0043604 is appropriate (lubberink2023biocatalyticamidebond pages 8-9).
- For several other families/domains included in the rule, our current evidence set lacks 2023–2024 sources within this context to validate direct amide bond formation; broad assignment of GO:0043604 to those risks false positives unless additional lineage/function filters are applied.

2. Domain Analysis
We summarize domain→function→amide linkages using the table artifact below, which clusters the 61 condition sets into major, functionally interpretable groups and flags where amide formation is direct, indirect, or unsupported in our current evidence set. Highlights from the retrieved evidence:
- Adenylating enzymes and PanC: Multiple recent examples show direct amidation via acyl‑adenylate chemistry, including pantothenate synthetase and engineered systems; these are diagnostic of amide formation when properly constrained (lubberink2023biocatalyticamidebond pages 6-7, lubberink2023biocatalyticamidebond pages 7-8).
- NRPS machinery: NRPS condensation domains catalyze peptide (amide) bond formation; recent work also shows a separate, ATP‑dependent amide‑forming enzyme (a PEARL) operating alongside NRPS modules (yu2023pearlcatalyzedpeptidebond pages 1-5).
- ATP‑grasp ligases: Representative enzymes promote amide formation via acyl‑phosphate intermediates and accept a range of substrates (lubberink2023biocatalyticamidebond pages 8-9).

|Group (domains/IDs)|Representative condition sets|Canonical biological function|Mechanistic link to amide bond formation|Key recent sources (author, year)|Taxon constraints / notes|GO term recommendation|
|---|---|---|---|---|---|---|
|TLC / TRAM-LAG1 / Ceramide synthases|TRAM/LAG1/CLN8 domain; CerS FunFam|N-acylation of sphingoid bases → ceramide biosynthesis|Direct — forms an amide bond between fatty acyl-CoA and sphingoid base|Pascoa 2025; Sheokand 2025|Eukaryotes / Metazoa; isoform-specific substrate chain-length selectivity|Keep GO:0043604 (appropriate) and add specific "ceramide biosynthetic" / N-acyltransferase terms|
|Nonribosomal peptide synthetases (A/C domains)|NRPS adenylation & condensation FunFams|Activation and condensation of amino acids to make NRPs|Direct — condensation (C) domains form peptide (amide) bonds during chain elongation|Heard 2024; Huang 2024; Mansour 2024|Bacteria, fungi (modular NRPS systems)|Prefer more specific annotation: use NRPS-specific biosynthetic GO (keep GO:0043604 as parent but annotate with "nonribosomal peptide biosynthetic process")|
|Glutamate–cysteine ligase (GCL) & Glutathione synthetase (GSS)|GCLC/GCLM funfams; GSS FunFam|Two-step glutathione (GSH) synthesis (rate-limiting + ligation)|Direct (GSS catalyzes peptide/amide bond between γ-glutamylcysteine and glycine)|Ikeda 2023; Zhang 2023; Vasková 2023|Widely conserved in eukaryotes; GCL is rate-limiting in many contexts|Keep GO:0043604 for GSS (fits amide biosynthesis); add "glutathione biosynthetic process" for specificity|
|Pantothenate / CoA enzymes (PanC, CoaBC, PPAT/CoaD)|Pantoate–β–alanine ligase (PanC), CoaBC, PPAT (CoaD)|Pantothenate (vitamin B5) and CoA biosynthesis; PPAT/CoaD adenylation step|Mixed — PanC (pantothenate synthetase) directly forms an amide (pantoate + β‑alanine); PPAT/CoaD and CoaBC act via adenylation/decaboxylation (indirect)|Warneke 2024; Rubio 2008; Chouhan 2024|PanC widespread in bacteria; Coa enzymes ubiquitous with lineage-specific variants|Annotate PanC with GO:0043604 (amide-forming specific); for PPAT/CoaBC use CoA biosynthesis-specific GO terms rather than general amide biosynthesis|
|Biotin pathway enzymes (BioD, BioB, auxiliary proteins)|Dethiobiotin synthase (BioD), biotin synthase (BioB), BioC/BioH variants|Late-stage biotin (vitamin B7) synthesis, sulfur insertion|Conditional — BioD/BioB catalyze bond-forming steps in biotin assembly; mechanisms differ by lineage and auxiliary factors|Qu 2024; Su 2024; Pinon 2005|Bacteria and plants differ; some taxa require auxiliary proteins (e.g., Mycobacteria)|Prefer specific "biotin biosynthetic process" term; GO:0043604 may apply to certain enzymes but use pathway-specific terms|
|UDP‑glycosyltransferases (UGTs)|UDP‑glucuronosyl/UDP‑glucosyltransferase IPRs / GT28|Transfer sugar moieties to diverse acceptors (glycosylation)|None — catalyze glycosidic (C–O) bond formation, not amide bonds|Majeed 2024; Lethe 2024; Yang 2023|Ubiquitous across plants, animals, bacteria; large paralog families|Do NOT assign GO:0043604; replace with glycosylation-specific GO terms (e.g., "UDP-glycosyltransferase activity")|
|Sialyltransferases (ST3GAL family)|ST3GAL FunFams / IPR sialyltransferase signatures|Sialylation of glycoconjugates (glycan terminal modification)|None — form glycosidic linkages rather than amide bonds|Petit 2015; Jiang 2025|Eukaryotic (vertebrates/animals and plants have related STs)|Do NOT assign GO:0043604; use sialyltransferase/glycosylation GO terms instead|
|Carnosine synthase (CARNS1 / ATPGD1)|Carnosine synthase FunFams / ATPGD1|ATP-dependent dipeptide (carnosine) biosynthesis (β‑alanine + histidine)|Direct — forms peptide (amide) bond; ATP-dependent ligase mechanism|Sinpru 2024; She 2022; Boldyrev 2013|Vertebrates/Chordata (muscle, brain); CARNS1-specific|Keep GO:0043604 (appropriate) and add specific "carnosine biosynthetic process" annotation|
|PM20D1 / N‑fatty‑acyl‑amino‑acid synthase/hydrolase|PM20D1 FunFam (N-acyl‑aa synthase/hydrolase)|Synthesis/hydrolysis of N‑acyl amino acids (N‑acyl amides)|Direct — catalyzes formation of N‑acyl amide bonds (N‑acylation of amino acids)|PM20D1 studies (2022–2024) (e.g., biomarker/enzymology literature)|Metazoa / mammals; secreted/circulating enzyme in some species|Keep GO:0043604 (fits N‑acyl amide formation) but add specific "N‑acyl‑amino‑acid biosynthetic" term|
|Pantothenate kinase (PANK) / CoA regulators|Pantothenate kinase FunFams (PANKs)|Phosphorylation of pantothenate — regulatory first step in CoA synthesis|None — kinase activity (phosphotransfer), does not form amide bonds|Roscoe 2023; Rubio 2008|Eukaryotic (mitochondrial isoforms) and bacterial variants exist|Do NOT assign GO:0043604; use CoA biosynthesis / kinase GO terms instead|
|Arginase (ARG1 / ARG2)|Arginase FunFams / urea-cycle enzymes|Hydrolysis of arginine → ornithine + urea (urea cycle)|None for biosynthesis — catalyzes hydrolysis (catabolic), not amide biosynthesis|Caldwell 2018; Li 2022|Mammalian isoforms with tissue-specific roles; widespread across taxa|Do NOT assign GO:0043604; use urea cycle/arginase activity GO terms|
|NUDIX hydrolases (NUDIX family)|NUDIX FunFams (various IPRs)|Hydrolyse diverse nucleotide-linked metabolites (housekeeping/regulatory)|Generally none — hydrolases degrade nucleotides; may indirectly affect amide-biosynthetic fluxes|Carreras-Puigvert 2017; Schneider 2024|Found across Bacteria, Archaea, Eukaryotes with diverse subfamilies|Do NOT assign GO:0043604 broadly; only consider amide-biosynthesis GO if specific NUDIX subfamily experimentally shown to catalyze amide formation|
|Sphingomyelin phosphodiesterases / SMSr (SMPD / SGMS)|SMPD1 (acid SMase), SMSr, SMS FunFams|Sphingomyelin / sphingolipid metabolism; PE/SM remodeling|Indirect — SMPD hydrolyses sphingomyelin; SMS family modulates lipid headgroups; only ceramide synthases (TLC family) directly form amide in ceramide|Chiang 2023; Shi 2023; Pascoa 2025|Lysosomal (SMPD1) vs ER/Golgi (SMS family); disease-linked (ASMD)|Do NOT generally assign GO:0043604 to SMases or SMS; reserve amide-biosynthesis annotation for CerS/TLC enzymes only|


*Table: Compact table mapping major domain/family groups in UniProt rule ARBA00028836 to canonical functions, whether they directly catalyze amide bonds (GO:0043604 relevance), representative recent sources, taxonomic notes, and recommended GO annotation actions.*

Notes about groups without direct evidence in this set: For TLC/Lag1 ceramide synthases, carnosine synthase (ATPGD1), PM20D1, glycosyl/sialyltransferases, pantothenate kinase, arginase, NUDIX hydrolases, sphingomyelin phosphodiesterases/synthase-related proteins, the present retrieval did not return citable 2023–2024 sources under our evidence constraints. Their entries in the artifact summarize general expectations and risks based on biochemical logic and widely known functions, but they are not supported here by the retrieved citations and should be considered provisional pending additional sourcing.

3. GO Term Evaluation (GO:0043604, amide biosynthetic process)
- Appropriate and specific:
  - NRPS domain combinations (adenylation + condensation modules) and other clear NRPS FunFams: by mechanism, these catalyze peptide/amide bond formation during product assembly (yu2023pearlcatalyzedpeptidebond pages 1-5).
  - Pantothenate synthetase (PanC) signatures: enzymatic amide formation is the core step in pantothenate biosynthesis (lubberink2023biocatalyticamidebond pages 6-7, lubberink2023biocatalyticamidebond pages 7-8).
  - ATP‑grasp ligases: catalyze amide formation; keep GO:0043604 (lubberink2023biocatalyticamidebond pages 8-9).
- Appropriate but should be refined to more specific child terms where available:
  - NRPS: Prefer also annotating to nonribosomal peptide biosynthetic process (child) while retaining GO:0043604 as a parent (yu2023pearlcatalyzedpeptidebond pages 1-5).
  - PanC: Add specific “pantothenate biosynthetic process” alongside GO:0043604 (lubberink2023biocatalyticamidebond pages 6-7, lubberink2023biocatalyticamidebond pages 7-8).
- Likely inappropriate (risk of false positives) in absence of additional constraints:
  - Glycosyl/sialyltransferase signatures (UGTs, sialyltransferases) catalyze glycosidic linkage formation, not amide bonds; GO:0043604 should not be assigned based solely on such domains (no direct evidence returned in this set).
  - Enzymes primarily hydrolytic or phosphotransfer (e.g., arginase, pantothenate kinase), or nucleotide hydrolases (NUDIX) are not amide-biosynthesis catalysts per se; avoid GO:0043604 unless supported by additional, enzyme‑specific evidence.

4. Evidence Review (2023–2024 priority)
- Biocatalytic amide bond formation (review): Summarizes adenylating enzymes (including CAR‑A and NRPS‑related A‑domains), CoA‑ligases, ATP‑grasp enzymes, and other catalytic strategies that directly form amide bonds; documents that pantothenate synthetase/PanC is a representative amide‑forming enzyme (lubberink2023biocatalyticamidebond pages 6-7, lubberink2023biocatalyticamidebond pages 7-8, lubberink2023biocatalyticamidebond pages 8-9). Publication: Green Chemistry, 2023 (review with broad coverage and methodological detail).
- NRPS amide formation and auxiliary machinery: A 2023 preprint demonstrates a lantibiotic dehydratase‑related enzyme (PEARL) that catalyzes peptide/amide bond formation in an NRPS‑PKS hybrid context, and reiterates that NRPS condensation domains catalyze peptide bonds; strengthens the link between NRPS domain signatures and amide biosynthesis (yu2023pearlcatalyzedpeptidebond pages 1-5). Publication: bioRxiv, 2023 (preprint; mechanistic claims align with established NRPS biochemistry).

Areas lacking sources in this evidence set (and thus partial assessment): ceramide synthases (TLC/Lag1), carnosine synthase (ATPGD1), PM20D1, glycosyl/sialyltransferases, arginase, NUDIX hydrolases, sphingomyelin phosphodiesterases/synthase‑related proteins. These should be reviewed with targeted, up‑to‑date sources before integrating them into a rule that assigns GO:0043604.

5. Recommendations
- Strengthen the rule by limiting GO:0043604 assignment to domains/families with clear, mechanistically amide‑forming chemistry:
  - Include: NRPS adenylation/condensation domain combinations and well‑defined NRPS FunFams; ATP‑grasp ligases; PanC (pantothenate synthetase) signatures (lubberink2023biocatalyticamidebond pages 6-7, lubberink2023biocatalyticamidebond pages 7-8, lubberink2023biocatalyticamidebond pages 8-9, yu2023pearlcatalyzedpeptidebond pages 1-5).
  - CoA/pantothenate enzymes: assign GO:0043604 specifically to PanC; do not assign the amide biosynthesis term to PPAT/CoaD or CoaBC unless enzyme‑specific amide‑forming evidence is present (lubberink2023biocatalyticamidebond pages 6-7, lubberink2023biocatalyticamidebond pages 7-8).
- Use more specific child GO terms jointly where appropriate (e.g., “nonribosomal peptide biosynthetic process” for NRPS, “pantothenate biosynthetic process” for PanC) to avoid over‑broad annotation while retaining GO:0043604 as a parent (lubberink2023biocatalyticamidebond pages 6-7, lubberink2023biocatalyticamidebond pages 7-8, yu2023pearlcatalyzedpeptidebond pages 1-5).
- Exclude or guard against false positives:
  - Do not assign GO:0043604 based solely on glycosyltransferase/sialyltransferase domains, hydrolytic enzymes (e.g., arginase), phosphotransferases (e.g., pantothenate kinase), or broad NUDIX hydrolase signatures without additional, function‑specific evidence of amide formation.
  - For multi‑functional lipid enzymes (e.g., sphingomyelin phosphodiesterases/synthases, synthase‑related proteins), do not infer amide biosynthesis unless co‑occurring ceramide‑synthase‑like amide‑forming signatures are demonstrated; otherwise restrict to lipid metabolism terms.
- Taxonomic filters: Retain lineage constraints only when they reflect enzyme presence and substrate space that align with the amide‑forming mechanism (e.g., NRPS primarily in bacteria/fungi). Where the included families are very diverse (e.g., glycosyltransferases), taxon filters alone will not prevent false positive GO:0043604 assignments—prefer domain combinations that encode mechanism (adenylation + condensation; ATP‑grasp) (yu2023pearlcatalyzedpeptidebond pages 1-5).
- Rule logic simplification: Focus on domain combinations that encode the catalytic mechanism of amide formation (adenylation + condensation; ATP‑grasp). Remove or segregate conditions whose domains are not mechanistically amide‑forming to alternate rules with appropriate non‑amide GO terms.

Citations (with URLs and dates)
- Lubberink M, Finnigan W, Flitsch SL. Biocatalytic amide bond formation. Green Chemistry. 2023;25:2958–2970. https://doi.org/10.1039/d3gc00456b (2023). Documents adenylating enzymes (including PanC) and ATP‑grasp ligases as direct amide‑forming catalysts; also discusses CoA‑ligases and enzymatic amidation strategies (lubberink2023biocatalyticamidebond pages 6-7, lubberink2023biocatalyticamidebond pages 7-8, lubberink2023biocatalyticamidebond pages 8-9).
- Yu Y, van der Donk WA. PEARL-catalyzed peptide bond formation after chain reversal during the biosynthesis of non-ribosomal peptides. bioRxiv. 2023. https://doi.org/10.1101/2023.12.23.573212 (posted 2023-12-24). Confirms NRPS condensation domains catalyze peptide bonds and identifies a distinct ATP‑dependent amide‑forming enzyme family operating alongside NRPS (yu2023pearlcatalyzedpeptidebond pages 1-5).

Limitations and next steps
- The present evidence set did not retrieve several family‑specific, up‑to‑date (2023–2024) sources for TLC/Lag1 ceramide synthases, carnosine synthase (ATPGD1), PM20D1, glycosyl/sialyltransferases, arginase, NUDIX hydrolases, or sphingomyelin phosphodiesterases/synthase‑related proteins. Before retaining those families in an OR‑logic rule for GO:0043604, we recommend targeted literature curation to either (i) confirm a direct amide‑forming mechanism, or (ii) move them to separate rules with pathway‑appropriate non‑amide GO terms.


References

1. (lubberink2023biocatalyticamidebond pages 6-7): Max Lubberink, William Finnigan, and Sabine L. Flitsch. Biocatalytic amide bond formation. Green Chemistry, 25:2958-2970, Jan 2023. URL: https://doi.org/10.1039/d3gc00456b, doi:10.1039/d3gc00456b. This article has 84 citations and is from a highest quality peer-reviewed journal.

2. (lubberink2023biocatalyticamidebond pages 7-8): Max Lubberink, William Finnigan, and Sabine L. Flitsch. Biocatalytic amide bond formation. Green Chemistry, 25:2958-2970, Jan 2023. URL: https://doi.org/10.1039/d3gc00456b, doi:10.1039/d3gc00456b. This article has 84 citations and is from a highest quality peer-reviewed journal.

3. (lubberink2023biocatalyticamidebond pages 8-9): Max Lubberink, William Finnigan, and Sabine L. Flitsch. Biocatalytic amide bond formation. Green Chemistry, 25:2958-2970, Jan 2023. URL: https://doi.org/10.1039/d3gc00456b, doi:10.1039/d3gc00456b. This article has 84 citations and is from a highest quality peer-reviewed journal.

4. (yu2023pearlcatalyzedpeptidebond pages 1-5): Yue Yu and Wilfred A. van der Donk. Pearl-catalyzed peptide bond formation after chain reversal during the biosynthesis of non-ribosomal peptides. bioRxiv, Dec 2023. URL: https://doi.org/10.1101/2023.12.23.573212, doi:10.1101/2023.12.23.573212. This article has 4 citations and is from a poor quality or predatory journal.

## Citations

1. yu2023pearlcatalyzedpeptidebond pages 1-5
2. lubberink2023biocatalyticamidebond pages 8-9
3. lubberink2023biocatalyticamidebond pages 6-7
4. lubberink2023biocatalyticamidebond pages 7-8
5. acyl-carrier protein
6. https://doi.org/10.1039/d3gc00456b
7. https://doi.org/10.1101/2023.12.23.573212
8. https://doi.org/10.1039/d3gc00456b,
9. https://doi.org/10.1101/2023.12.23.573212,