---
pmid: '37207277'
title: Using brain cell-type-specific protein interactomes to interpret neurodevelopmental
  genetic signals in schizophrenia.
authors:
- Hsu YH
- Pintacuda G
- Liu R
- Nacu E
- Kim A
- Tsafou K
- Petrossian N
- Crotty W
- Suh JM
- Riseman J
- Martin JM
- Biagini JC
- Mena D
- Ching JKT
- Malolepsza E
- Li T
- Singh T
- Ge T
- Egri SB
- Tanenbaum B
- Stanclift CR
- Apffel AM
- Schizophrenia Working Group of the Psychiatric Genomics Consortium
- Stanley Global Asia Initiatives
- Carr SA
- Schenone M
- Jaffe J
- Fornelos N
- Huang H
- Eggan KC
- Lage K
journal: iScience
year: '2023'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC10189495
doi: 10.1016/j.isci.2023.106701
---

# Using brain cell-type-specific protein interactomes to interpret neurodevelopmental genetic signals in schizophrenia.
**Authors:** Hsu YH, Pintacuda G, Liu R, Nacu E, Kim A, Tsafou K, Petrossian N, Crotty W, Suh JM, Riseman J, Martin JM, Biagini JC, Mena D, Ching JKT, Malolepsza E, Li T, Singh T, Ge T, Egri SB, Tanenbaum B, Stanclift CR, Apffel AM, Schizophrenia Working Group of the Psychiatric Genomics Consortium, Stanley Global Asia Initiatives, Carr SA, Schenone M, Jaffe J, Fornelos N, Huang H, Eggan KC, Lage K
**Journal:** iScience (2023)
**DOI:** [10.1016/j.isci.2023.106701](https://doi.org/10.1016/j.isci.2023.106701)
**PMC:** [PMC10189495](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10189495/)

## Abstract

1. iScience. 2023 Apr 18;26(5):106701. doi: 10.1016/j.isci.2023.106701.
eCollection  2023 May 19.

Using brain cell-type-specific protein interactomes to interpret 
neurodevelopmental genetic signals in schizophrenia.

Hsu YH(1)(2)(3), Pintacuda G(1)(2)(4), Liu R(1)(5), Nacu E(1)(4), Kim A(1)(6), 
Tsafou K(1), Petrossian N(1), Crotty W(1), Suh JM(1), Riseman J(1), Martin 
JM(1), Biagini JC(1), Mena D(1), Ching JKT(1), Malolepsza E(7), Li T(1), Singh 
T(1)(5), Ge T(1)(8), Egri SB(9), Tanenbaum B(9), Stanclift CR(9), Apffel AM(9); 
Schizophrenia Working Group of the Psychiatric Genomics Consortium; Stanley 
Global Asia Initiatives; Carr SA(9), Schenone M(9), Jaffe J(9), Fornelos 
N(1)(2)(3), Huang H(1)(5), Eggan KC(1)(4), Lage K(1)(2)(3)(10).

Collaborators: Ripke S, Neale BM, Corvin A, Walters JT, Farh KH, Holmans PA, Lee 
P, Bulik-Sullivan B, Collier DA, Huang H, Pers TH, Agartz I, Agerbo E, Albus M, 
Alexander M, Amin F, Bacanu SA, Begemann M, Belliveau RA Jr, Bene J, Bergen SE, 
Bevilacqua E, Bigdeli TB, Black DW, Bruggeman R, Buccola NG, Buckner RL, Byerley 
W, Cahn W, Cai G, Campion D, Cantor RM, Carr VJ, Carrera N, Catts SV, Chambert 
KD, Chan RC, Chan RY, Chen EY, Cheng W, Cheung EF, Chong SA, Cloninger CR, Cohen 
D, Cohen N, Cormican P, Craddock N, Crowley JJ, Curtis D, Davidson M, Davis KL, 
Degenhardt F, Del Favero J, Demontis D, Dikeos D, Dinan T, Djurovic S, Donohoe 
G, Drapeau E, Duan J, Dudbridge F, Durmishi N, Eichhammer P, Eriksson J, 
Escott-Price V, Essioux L, Fanous AH, Farrell MS, Frank J, Franke L, Freedman R, 
Freimer NB, Friedl M, Friedman JI, Fromer M, Genovese G, Georgieva L, Giegling 
I, Giusti-Rodríguez P, Godard S, Goldstein JI, Golimbet V, Gopal S, Gratten J, 
de Haan L, Hammer C, Hamshere ML, Hansen M, Hansen T, Haroutunian V, Hartmann 
AM, Henskens FA, Herms S, Hirschhorn JN, Hoffmann P, Hofman A, Hollegaard MV, 
Hougaard DM, Ikeda M, Joa I, Julià A, Kahn RS, Kalaydjieva L, Karachanak-Yankova 
S, Karjalainen J, Kavanagh D, Keller MC, Kennedy JL, Khrunin A, Kim Y, Klovins 
J, Knowles JA, Konte B, Kucinskas V, Kucinskiene ZA, Kuzelova-Ptackova H, Kähler 
AK, Laurent C, Lee J, Lee SH, Legge SE, Lerer B, Li M, Li T, Liang KY, Lieberman 
J, Limborska S, Loughland CM, Lubinski J, Lönnqvist J, Macek M, Magnusson PK, 
Maher BS, Maier W, Mallet J, Marsal S, Mattheisen M, Mattingsdal M, McCarley RW, 
McDonald C, McIntosh AM, Meier S, Meijer CJ, Melegh B, Melle I, Mesholam-Gately 
RI, Metspalu A, Michie PT, Milani L, Milanova V, Mokrab Y, Morris DW, Mors O, 
Murphy KC, Murray RM, Myin-Germeys I, Müller-Myhsok B, Nelis M, Nenadic I, 
Nertney DA, Nestadt G, Nicodemus KK, Nikitina-Zake L, Nisenbaum L, Nordin A, 
O'Callaghan E, O'Dushlaine C, O'Neill FA, Oh SY, Olincy A, Olsen L, Van Os J, 
Pantelis C, Papadimitriou GN, Papiol S, Parkhomenko E, Pato MT, Paunio T, 
Pejovic-Milovancevic M, Perkins DO, Pietiläinen O, Pimm J, Pocklington AJ, 
Powell J, Price A, Pulver AE, Purcell SM, Quested D, Rasmussen HB, Reichenberg 
A, Reimers MA, Richards AL, Roffman JL, Roussos P, Ruderfer DM, Salomaa V, 
Sanders AR, Schall U, Schubert CR, Schulze TG, Schwab SG, Scolnick EM, Scott RJ, 
Seidman LJ, Shi J, Sigurdsson E, Silagadze T, Silverman JM, Sim K, Slominsky P, 
Smoller JW, So HC, Spencer CC, Stahl EA, Stefansson H, Steinberg S, Stogmann E, 
Straub RE, Strengman E, Strohmaier J, Stroup TS, Subramaniam M, Suvisaari J, 
Svrakic DM, Szatkiewicz JP, Söderman E, Thirumalai S, Toncheva D, Tosato S, 
Veijola J, Waddington J, Walsh D, Wang D, Wang Q, Webb BT, Weiser M, Wildenauer 
DB, Williams NM, Williams S, Witt SH, Wolen AR, Wong EH, Wormley BK, Xi HS, Zai 
CC, Zheng X, Zimprich F, Wray NR, Stefansson K, Visscher PM, Adolfsson R, 
Andreassen OA, Blackwood DH, Bramon E, Buxbaum JD, Børglum AD, Cichon S, Darvasi 
A, Domenici E, Ehrenreich H, Esko T, Gejman PV, Gill M, Gurling H, Hultman CM, 
Iwata N, Jablensky AV, Jönsson EG, Kendler KS, Kirov G, Knight J, Lencz T, 
Levinson DF, Li QS, Liu J, Malhotra AK, McCarroll SA, McQuillin A, Moran JL, 
Mortensen PB, Mowry BJ, Nöthen MM, Ophoff RA, Owen MJ, Palotie A, Pato CN, 
Petryshen TL, Posthuma D, Rietschel M, Riley BP, Rujescu D, Sham PC, Sklar P, St 
Clair D, Weinberger DR, Wendland JR, Werge T, Daly MJ, Sullivan PF, O'Donovan 
MC, Huang H, Qin S, Sawa A, Schwab SG, Kahn R, Hong KS, Shi W, Tsuang M, Itokawa 
M, Feng G, Liu J, Glatt SJ, Iwata N, Ikeda M, Ma X, Lee J, Tang J, Ruan Y, Liu 
R, Zhu F, Horiuchi Y, Lee BD, Joo EJ, Myung W, Ha K, Won HH, Baek JH, Chung YC, 
Kim SW, Wildenauer DB, Kusumawardhani A, Chen WJ, Hwu HG, Sim K, Hishimoto A, 
Otsuka I, Sora I, Toyota T, Yoshikawa T, Kunugi H, Hattori K, Ishiwata S, Numata 
S, Ohmori T, Arai M, Ozeki Y, Fujii K, Kim SJ, Lee HJ, Ahn YM, Kim SH, Akiyama 
K, Shimoda K, Kinoshita M.

Author information:
(1)Stanley Center for Psychiatric Research, Broad Institute of MIT and Harvard, 
Cambridge, MA 02142, USA.
(2)Novo Nordisk Foundation Center for Genomic Mechanisms of Disease, Broad 
Institute of MIT and Harvard, Cambridge, MA 02142, USA.
(3)Department of Surgery, Massachusetts General Hospital, Boston, MA 02114, USA.
(4)Harvard Stem Cell Institute and Department of Stem Cell and Regenerative 
Biology, Harvard University, Cambridge, MA 02138, USA.
(5)Analytic and Translational Genetics Unit, Department of Medicine, 
Massachusetts General Hospital, Boston, MA 02114, USA.
(6)Department of Computer Science, Johns Hopkins University, Baltimore, MD 
21218, USA.
(7)Genomics Platform, Broad Institute of MIT and Harvard, Cambridge, MA 02142, 
USA.
(8)Psychiatric and Neurodevelopmental Genetics Unit, Center for Genomic 
Medicine, Massachusetts General Hospital, Boston, MA 02114, USA.
(9)Proteomics Platform, Broad Institute of MIT and Harvard, Cambridge, MA 02142, 
USA.
(10)Institute of Biological Psychiatry, Mental Health Centre Sct. Hans, Mental 
Health Services Copenhagen, 4000 Roskilde, Denmark.

Genetics have nominated many schizophrenia risk genes and identified convergent 
signals between schizophrenia and neurodevelopmental disorders. However, 
functional interpretation of the nominated genes in the relevant brain cell 
types is often lacking. We executed interaction proteomics for six schizophrenia 
risk genes that have also been implicated in neurodevelopment in human induced 
cortical neurons. The resulting protein network is enriched for common variant 
risk of schizophrenia in Europeans and East Asians, is down-regulated in layer 
5/6 cortical neurons of individuals affected by schizophrenia, and can 
complement fine-mapping and eQTL data to prioritize additional genes in GWAS 
loci. A sub-network centered on HCN1 is enriched for common variant risk and 
contains proteins (HCN4 and AKAP11) enriched for rare protein-truncating 
mutations in individuals with schizophrenia and bipolar disorder. Our findings 
showcase brain cell-type-specific interactomes as an organizing framework to 
facilitate interpretation of genetic and transcriptomic data in schizophrenia 
and its related disorders.

© 2023 The Authors.

DOI: 10.1016/j.isci.2023.106701
PMCID: PMC10189495
PMID: 37207277

Conflict of interest statement: K.C.E. is a co-founder of Q-State Biosciences, 
Quralis, and Enclear, and currently employed at BioMarin Pharmaceutical. Other 
authors declare no competing interests.

## Full Text

IntroductionSchizophrenia is a debilitating psychiatric disorder occurring in ∼0.3% of the global population with severe repercussions for patients, families, and society.1,2 The last years have seen great advances in mapping the genetic architecture of schizophrenia, identifying hundreds of common and rare variants that confer risk for the disorder across diverse populations.3,4,5,6,7,8,9,10 These studies also revealed overlapping genetic signals between schizophrenia, autism spectrum disorders (ASD), and severe developmental disorders (DD), supporting the importance of neurodevelopmental processes in the pathophysiology of schizophrenia.9,10,11 However, although the identified schizophrenia risk genes provide a good entry point for systematic studies of the disorder and its related conditions, their molecular functions and interactions in the brain remain poorly understood, hindering the development of effective treatments and therapeutics.12,13In parallel, analyses of postmortem brains from individuals with schizophrenia and integration of genetic and transcriptomic data from human and mouse brains have converged on cortical excitatory neurons as a key biological conduit of genetically encoded risk.14,15,16,17 This suggests that systematic mapping of schizophrenia risk genes onto protein-protein interaction (PPI) networks in this cell type could reveal mechanisms and pathways underlying schizophrenia.18,19 A seminal study showed that adding extrinsic neuronal patterning to pluripotent stem cells (PSCs) overexpressing NGN2 generates glutamatergic induced neurons (iNs) that behave like cortical excitatory neurons at the molecular, morphological, and functional levels.20,21 Here, we leveraged this protocol to perform interaction proteomics for six schizophrenia risk genes that have also been implicated in neurodevelopment in a neuronal cell model. By integrating the resulting PPI networks with orthogonal datasets, we showed that our approach creates a unique opportunity to investigate the roles of schizophrenia risk genes and their associated pathways in a brain cell-type-specific context.

Introduction

Schizophrenia is a debilitating psychiatric disorder occurring in ∼0.3% of the global population with severe repercussions for patients, families, and society.1,2 The last years have seen great advances in mapping the genetic architecture of schizophrenia, identifying hundreds of common and rare variants that confer risk for the disorder across diverse populations.3,4,5,6,7,8,9,10 These studies also revealed overlapping genetic signals between schizophrenia, autism spectrum disorders (ASD), and severe developmental disorders (DD), supporting the importance of neurodevelopmental processes in the pathophysiology of schizophrenia.9,10,11 However, although the identified schizophrenia risk genes provide a good entry point for systematic studies of the disorder and its related conditions, their molecular functions and interactions in the brain remain poorly understood, hindering the development of effective treatments and therapeutics.12,13

In parallel, analyses of postmortem brains from individuals with schizophrenia and integration of genetic and transcriptomic data from human and mouse brains have converged on cortical excitatory neurons as a key biological conduit of genetically encoded risk.14,15,16,17 This suggests that systematic mapping of schizophrenia risk genes onto protein-protein interaction (PPI) networks in this cell type could reveal mechanisms and pathways underlying schizophrenia.18,19 A seminal study showed that adding extrinsic neuronal patterning to pluripotent stem cells (PSCs) overexpressing NGN2 generates glutamatergic induced neurons (iNs) that behave like cortical excitatory neurons at the molecular, morphological, and functional levels.20,21 Here, we leveraged this protocol to perform interaction proteomics for six schizophrenia risk genes that have also been implicated in neurodevelopment in a neuronal cell model. By integrating the resulting PPI networks with orthogonal datasets, we showed that our approach creates a unique opportunity to investigate the roles of schizophrenia risk genes and their associated pathways in a brain cell-type-specific context.

ResultsPrioritization of schizophrenia index genes with shared neurodevelopmental signalsTo identify schizophrenia risk genes as the basis of our interactome experiments we designed and executed a three-step procedure (Figure 1A and Table S1). First, we identified 445 genes (Set 1) in previously reported genome-wide significant loci from the Psychiatric Genomics Consortium (PGC) genome-wide association study3 (GWAS; phase 2). Second, we filtered this set to 37 genes (Set 2) within single protein-coding gene loci, excluding other genes in loci with more ambiguous association signals. Third, we integrated data from orthogonal studies (e.g., high-density genotyping, exome sequencing, and earlier targeted studies of individual genes; STAR Methods and Table S1) to identify a subset of 10 genes (Set 3) supported by multiple independent lines of evidence. Importantly, we used orthogonal evidence from rare variant studies of ASD/DD to prioritize schizophrenia risk genes that have also been implicated in neurodevelopmental conditions. We additionally included SYNGAP1 in the major histocompatibility complex (MHC) region in all three sets because of strong orthogonal evidence for its involvement in schizophrenia and neurodevelopmental disorders.Figure 1Selection of schizophrenia index genes and proteins for interactome experiments(A) Three-step procedure to identify Sets 1–3 by refining schizophrenia GWAS [PGC phase 2] data, where Set 3 was defined as ‘index genes’ and used as the basis for downstream experiments. Set 3 genes are color-coded based on the type of orthogonal evidence supporting their involvement in neuropsychiatric or neurodevelopmental phenotypes.(B) Cumulative density of gnomAD pLI scores for different gene sets. ‘Genome’ indicates genes in the pLI dataset [excluding Sets 1–3]; ‘Sets 1–3’ indicate genes in Sets 1–3 with available pLI scores; ‘SCHEMA 5% FDR’ and ‘SCHEMA EWS’ indicate genes with FDR < 0.05 or 3.7e-3 [exome-wide significance] in the SCHEMA exome sequencing analysis, respectively.(C) Frontal cortex RNA expression of gene sets across ten developmental stages. Median expression and standard error [SE] of each gene set were derived from the BrainSpan exon microarray dataset. ‘Random’ indicates genes randomly sampled from the BrainSpan dataset; ‘Sets 1–3’ indicate genes in Sets 1–3 with available BrainSpan data; ‘SCHEMA’ indicates exome-wide significant genes from SCHEMA. Shaded regions indicate median expression of genes with FDR < 3.7e-3 [exome-wide significance], 0.05, 0.25, or 0.5 in SCHEMA with darker gray indicating greater significance.(D) Western blot analysis of index proteins in iPSCs, NPCs [at day 3 of differentiation], iNs [at weeks 2–7 of differentiation], three cancer cell lines [TF-1, K562, U937], HEK293 cells, and mouse cortex. SATB2 and ZNF804A are excluded from this panel due to lack of detectable expression in iNs. See also Figure S1, Data S4, and Tables S1, S2, S3, and S4.Genes implicated in schizophrenia are under strong genetic selection and have elevated expression in the frontal cortex.9 Therefore, to assess the enrichment of schizophrenia risk genes in Sets 1–3, we compared their gnomAD22 pLI scores (i.e., the probability of being loss-of-function intolerant, where genes under strong selection have higher pLI scores) and BrainSpan23 expression to schizophrenia risk genes from the recent Schizophrenia Exome Sequencing Meta-Analysis (SCHEMA) study.9 In terms of pLI scores, Sets 1–3 all have increasingly higher scores compared to other genes in the genome (one-tailed KS test p = 1.1e-3, 1.7e-3, and 1.3e-5 for Sets 1–3, respectively; Figure 1B and Table S2). Set 3 scores are significantly higher than that of Set 2 (one-tailed KS test p = 5.8e-4), which in turn are higher than that of Set 1 (one-tailed KS test p = 0.019). Furthermore, Set 3 is under the same degree of constraint as the exome-wide significant (FDR < 3.7e-3) SCHEMA genes (two-tailed KS test p = 0.23). In terms of gene expression, Set 1 mirrors the expression profile of random genes during frontal cortical development up until adulthood; Set 2 has a postnatal expression profile that resembles SCHEMA genes with FDR of 0.25–0.5; and Set 3 has a postnatal expression profile that strongly mirrors the exome-wide significant SCHEMA genes (Figure 1C and Table S3). Although the lower prenatal expression of Set 3 compared to the SCHEMA genes may be reflecting different aspects of schizophrenia-related biology captured by common versus rare variants, respectively (Data S4), the pLI scores and postnatal expression patterns generally support our three-step approach to gene selection based on refining GWAS data and indicate that Set 3 is enriched for bona fide schizophrenia risk genes. We proceeded to use Set 3 as the starting point of our experiments and refer to these 11 genes and their encoded proteins as ‘index genes’ and ‘index proteins’, respectively.Interaction proteomics of schizophrenia index proteins in excitatory iNsTo study the expression patterns of the index proteins throughout iN maturation, we tested 58 commercially available antibodies and identified 31 with competency to detect the 11 index proteins (Table S4). We differentiated iPSCs into neural progenitor cells (NPCs; day 3) and iNs (weeks 2–7) and confirmed protein expression of CACNA1C, CACNB2, CSMD1, CUL3, GRIN2A, HCN1, RIMS1, SYNGAP1, and TCF4 in neuron lysates by western blot (Figures 1D and S1). SATB2 and ZNF804A lacked detectable neuronal expression or high-quality reagents and were excluded from further experiments. When comparing index protein expression between iNs, non-neuronal cells, and mouse cortex, CACNA1C, CSMD1, GRIN2A, HCN1, RIMS1, and SYNGAP1 displayed a neuron-specific expression profile.Next, we tested 42 antibodies for their ability to immunoprecipitate (IP) the index proteins (Table S4) and were able to IP seven index proteins (CACNA1C, CACNB2, CUL3, HCN1, RIMS1, SYNGAP1, TCF4) in ∼seven billion iNs for subsequent mass spectrometry (MS) analyses. In total, we carried out 23 IP-MS experiments at five neuronal differentiation time points. We performed quality control (QC) and analyzed each experiment using Genoppi,24 calculating the log2 fold change (FC) and corresponding statistical significance for each protein identified in the index protein IPs compared to the controls, and then defining proteins with log2 FC > 0 and FDR ≤ 0.1 as the significant interactors of the index protein. We disregarded four IP experiments that did not meet our QC criteria (i.e., the log2 FC correlation between replicates was < 0.5 or the index protein itself was not enriched at FDR ≤ 0.1). The remaining 19 high-quality IPs of CACNA1C, CUL3, HCN1, RIMS1, SYNGAP1, and TCF4 had a median replicate log2 FC correlation of 0.87, with the six index proteins enriched at a median FDR of 8.2e-4 (Figures 2A, 2B, S2A, S2B; Tables S5 and S6). In addition, we performed experimental and computational analyses to confirm that, despite using an inclusive FC cutoff (i.e., log2 FC > 0) to define index protein interactors in the IPs, the interactors showed no obvious quality differences across a range of different FCs (Data S5).Figure 2Cell-type-specific protein interactomes in cortical human neurons(A) Scatter plot showing log2 FC correlation between replicate 1 [x-axis] and replicate 2 [y-axis] of an IP of CACNA1C at week 3 of neuron differentiation [Pearson’s r = 0.74].(B) Volcano plot showing log2 FC [x-axis] and -log10 p-value [y-axis] of the CACNA1C IP from (A). For (A) and (B), the index protein [CACNA1C] is shown in red, significant interactors [log2 FC > 0 and FDR ≤ 0.1] in green, and non-interactors [i.e., other detected proteins] in blue. Known InWeb interactors are indicated by black border circles, with the subset that are significant in the IP highlighted in yellow [overlap p = 1.8e-2]. Calcium channel components [alpha, beta, and alpha2delta subunits] are in orange.(C) Replication rates of a subset of interactions tested in forward or reverse IPs followed by western blotting [IP-WB].(D) Pairwise co-expression Z-scores between index genes and their interactors [Int], non-interactors [NonInt], known InWeb interactors [InWeb], and all protein-coding genes [All] derived from a spatial transcriptomic dataset in human dorsolateral prefrontal cortex. Boxes and whiskers in violin plots indicate the interquartile range [IQR] and 1.5x IQR, respectively. Double asterisks indicate p < 0.05/6 [adjusting for six pairwise comparisons] as calculated by two-tailed Wilcoxon rank-sum tests. Number of gene pairs plotted for each gene type is indicated toward the bottom.(E) The combined interaction network of six index proteins resulting from 19 individual IPs. Index proteins and their interactors are indicated as red and purple nodes, respectively. Size and color of the interactor nodes scale with the number of index proteins linked to each interactor, with larger and darker nodes representing more recurrent interactors [distribution shown in upper right pie chart]. Edges represent protein interactions with colors indicating whether each interaction is known in InWeb [blue] or potentially novel [gray; distribution shown in lower right pie chart]. See also Figures S2 and S7, Data S5 and S6, and Tables S4, S5, S6, S7, S8, S9, and S10.The PPI data contain newly reported and reproducible interactionsImportantly, our experimental design was purposely aimed at generating inclusive PPI datasets that encompass non-stoichiometric interactions, rather than restricting to direct interactions and core molecular complexes (Data S6). In previous work, we have established that the replication rate of interactions identified using a similar IP-MS approach is 70–90%.24 In the current study, we were able to recapitulate these estimates using a two-pronged approach (Figures 2C and S3–S5, and Table S7). First, we performed western blots of 45 interactors for the six index proteins on repetition of the index protein IPs and validated 40 of the interactors (88.9% replication rate). In parallel, we performed reverse IPs using a panel of interactors as baits, and successfully detected the original index proteins in 23 out of 25 reverse IPs that showed bait enrichment (92.0% replication rate). The ∼90% replication rates are in general agreement with the 10% FDR cutoff we applied to identify significant interactions.Besides replicating a subset of the index protein interactors, we also used published datasets to assess the biological validity of our IP-MS data. First, ten of the 19 IPs are further supported by the observation that they are enriched for known protein interactors derived from the InWeb database25 (Table S5). As an example, in a CACNA1C IP performed in iNs at week 3 of differentiation, the significant interactors are enriched for known CACNA1C interactors in InWeb (p = 1.8e-2), including all known L-type calcium channel subunits: the extracellular CACNA2D1 and the intracellular CACNB1 and CACNB326 (Figures 2A and 2B). On the other hand, >94% of the interactors are not found in InWeb nor in an IP of CACNA1C executed in mouse heart tissue with the same antibody.27 This example illustrates that our neuron-derived PPI data capture both known and novel biology as expected, given that existing PPI datasets were mostly generated in non-neuronal context using different experimental methods (Data S6).We also used brain co-expression data to systematically benchmark all index protein interactors in our data, reasoning that on average, transcripts of interacting proteins would be more likely to co-localize across tissues, cell types, and developmental time points (STAR Methods). Indeed, we observed that the interactors usually have higher co-expression with the index proteins compared to the ‘non-interactors’ (i.e., non-significant proteins detected in IP-MS), known interactors in InWeb, and all protein-coding genes in a spatial transcriptomic dataset derived from human dorsolateral prefrontal cortex28 (Figure 2D). Similar trends were also observed in other expression datasets from human or mouse brains29,30 (Figure S2C). This not only indicates that the interactions we identified in in vitro neurons represent biology found in complex brain tissues, but further suggests that they may be more enriched for gene relationships in the brain compared to proteins generally expressed in neurons (represented by the non-interactors) or interactions found in non-neuronal context (represented by the InWeb interactors). In summary, the experimental replications, InWeb overlap, and brain co-expression results all support the quality, reproducibility, and biological relevance of the interactome data we have generated for the schizophrenia index proteins in human iNs.The combined PPI networks for six schizophrenia index proteinsWhen we compared multiple IPs of the same index protein across time points during neuronal maturation and between cell lines (for CACNA1C, HCN1, and SYNGAP1), we observed significant agreement in terms of the log2 FC correlation of all detected proteins (median correlation = 0.75; Figures S6A and S6C–S6E). When we clustered the IPs based on the percentage of overlap between significant interactors, we found that IPs from the earlier versus the later time points tend to fall into separate clusters, agreeing with known characteristics of maturing neurons generated using the NGN2-based protocol21 (Figures S6B and S6D–S6F and Table S8). However, we also observed relatively high percentages of overlap that are statistically significant across all time points (median percentage = 72%). Overall, these results indicate that our IPs from different time points capture a large proportion of overlapping biology, which is not unexpected given that differentiating iPSCs start to express neuronal markers soon after forcing NGN2 overexpression. Based on these observations, we decided to explore the combined PPI network of each index protein across time points in downstream analyses.We merged data from the 19 individual IPs to create nine additional consolidated datasets (Table S9). These datasets represent the combined PPI network of a single index protein across multiple experiments or time points (i.e., CACNA1C, HCN1, RIMS1, and SYNGAP1), the combined network of multiple index proteins at one time point (i.e., week 2, 3, 4, and 7), and the combined network of all six index proteins across all time points (i.e., ‘all combined’). The all combined network contains 1,238 interactors of the six index proteins and, like the CACNA1C IP highlighted above, >96% of the interactions in this network are potentially novel interactions not found in InWeb (Figure 2E and Table S5). Genes encoding these interactors have relatively high expression in the frontal cortex throughout brain development similar to schizophrenia risk genes reaching 5% FDR in SCHEMA (Figure S7A, Table S3, and Data S4). SynGO31 gene set analysis also found the network to be enriched for genes involved in various biological processes in the synapse (Figure S7B and Table S10). Overall, we successfully mapped the neuronal protein interactomes of six proteins that are transmembrane (CACNA1C, HCN1), cytosolic (CUL3, SYNGAP1), and involved in multiple neuronal signaling processes (RIMS1, SYNGAP1, TCF4). The resulting PPI networks include a high percentage of newly reported interactions and span many areas of the cell biology of cortical excitatory neurons.The PPI networks are enriched for genetic risks of schizophrenia, ASD, and DDTo test the networks for association to schizophrenia, we assessed the enrichment of common variant risk across the networks using PGC GWAS data containing schizophrenia cases and controls of European (EUR) or East Asian (EAS) ancestry.3,8 For these and all further network enrichment analyses, we created a more conservative version of the combined networks by excluding proteins that showed up as non-interactors in any of the source IPs (see ‘stringent interactors’ in Table S9). In total, we performed genetic analyses for 11 interaction networks, including the nine combined networks and two individual IP networks for CUL3 and TCF4. Importantly, our analyses were conditional on the non-interactors detected in our IP experiments, meaning that we tested whether interactors in the different networks are enriched for genetic risk compared to other iN-expressed proteins. If so, this would suggest that our networks are relevant to schizophrenia over and above the background proteome of the neuronal cell model.Using MAGMA,32 we found that many of the PPI networks are indeed enriched for schizophrenia risk when conditioned on other iN-expressed proteins. Notably, the enrichment is generally consistent across EUR and EAS ancestries (Figure 3A and Table S11). At a Bonferroni-corrected threshold (p < 0.05/22, adjusting for 11 networks and two ancestries), the week 4 network in both EUR and EAS ancestries, as well as the all combined, week 2, week 7, and HCN1 networks in EAS showed significant enrichment. In the cross-ancestry meta-analysis, the all combined, week 2, week 4, week 7, and HCN1 networks were all significant at the same Bonferroni-corrected threshold. In parallel, we further validated these findings using a genetic risk score (GRS) enrichment analysis method that estimates the genetic risk on holdout samples not included in GWAS and therefore is less sensitive to outliers (STAR Methods). Most results were replicated, including the enrichment signals for the all combined, week 4, and HCN1 networks in both ancestries and their meta-analysis (Figure S8B and Table S11).Figure 3Enrichment of common variant risks and transcriptional perturbations in the index protein interactomesNetworks tested are the combined network of all IPs [All combined], the combined networks at each time point [Week 2 to Week 7], the combined networks for CACNA1C, HCN1, RIMS1, and SYNGAP1, and the individual IP networks for CUL3 and TCF4; the number of genes in each network is shown in parentheses on the y-axes.(A) Common variant enrichment of schizophrenia [SCZ] or height in Europeans [EUR], East Asians [EAS], or their meta-analysis. Enrichment coefficients, standard errors [SE], and p-values were calculated using MAGMA. p < 0.05 or p < 0.05/22 [adjusting for 11 networks and two ancestries] results are highlighted in orange or red, respectively.(B) Common variant enrichment of SCZ, attention deficit hyperactivity disorder [ADHD], autism spectrum disorders [ASD], bipolar disorder [BIP], or major depressive disorder [MDD] calculated using MAGMA. Cross-ancestry meta-analysis results are shown for SCZ; EUR ancestry results are shown for other disorders. Enrichment coefficients reaching p < 0.05 or p < 0.05/22 significance are shown in the heatmap followed by single or double asterisks, respectively.(C) Enrichment of cell-type-specific differentially expressed genes [DEGs] in the prefrontal cortex of schizophrenia patients compared to controls; the number of DEGs in each cell type is shown in parentheses on the x-axis. p-values were calculated using one-tailed hypergeometric tests. Gene counts in overlaps reaching p < 0.05 or p < 0.05/220 [adjusting for 11 networks and 20 cell types] significance are shown in the heatmap followed by single or double asterisks, respectively. See also Figures S8 and S9, Tables S11 and S12.To further explore whether the genetic risk enrichment we observed in the PPI networks is specific to schizophrenia, we repeated the same analyses using GWAS data of other psychiatric disorders, including attention deficit hyperactivity disorder33 (ADHD), ASD,34 bipolar disorder35 (BIP), and major depressive disorder36 (MDD), as well as height37,38 as a control trait (Figures 3A, 3B, S8A, and S8B, and Table S11). Across these phenotypes, we only observed robust enrichment for ASD in the CUL3 network and the week 7 network which contains IP data of CUL3 and SYNGAP1, both of which have been previously linked to ASD.39,40 The networks that showed the most robust enrichment for schizophrenia (i.e., the all combined, week 4, and HCN1 networks) did not demonstrate enrichment for the other phenotypes.Besides analyzing data from common variants, we also tested whether the PPI networks are enriched for rare variant risks of schizophrenia,9 ASD,40 and DD,41 as well as for high gnomAD pLI scores (Figure S8C and Table S11). At a Bonferroni-corrected threshold (p < 0.05/11, adjusting for 11 networks), we found the CUL3 network to be enriched for DD genes, which agrees with the previously implicated role of CUL3 in developmental delay.39,42 In addition, the week 4 and RIMS1 networks are significantly enriched for genes with high pLI scores, indicating that some members of these networks are likely intolerant to loss-of-function mutations and may be involved in essential cell functions.Overall, these genetic analysis results indicate that most of the PPI network genes are specifically concentrating common variant risk of schizophrenia, whereas several sub-networks are also enriched for common or rare variant risks of ASD/DD, highlighting the shared biology between these disorders.The PPI networks implicate layer 5/6 cortical excitatory neurons in schizophreniaNext, we explored whether the PPI networks are enriched for brain-layer-specific transcriptional perturbations observed in patients with schizophrenia. We analyzed data from a recent single-cell RNA sequencing study,17which identified differentially expressed genes (DEGs) in individuals with schizophrenia versus controls in 20 annotated cell types in the prefrontal cortex. Many of our networks had nominally significant (p < 0.05) overlaps with the cell-type-specific DEGs in neuronal cell types, which include both upper and deep layer excitatory neurons and inhibitory interneurons (Figure 3C and Table S12). A population of layer 5/6 cortico-cortical projection neurons (‘Ex-L5-6CCb’) showed the most robust enrichment, reaching Bonferroni significance (p < 0.05/220, adjusting for 11 networks and 20 cell types) for the all combined, week 4, and week 7 networks. Subsequently, we separately analyzed the up- and down-regulated DEGs in this celltype, showing that the enrichment signals are strongly driven by the down-regulated DEGs (Figure S9 and Table S12). As DEGs in ‘Ex-L5-6CCb’ were also found to be enriched for genes implicated by schizophrenia GWAS,10 there is an intriguing convergence between our results and findings in patients that converge on deep layer cortical excitatory neurons as a key cell type for studying cellular networks involved in schizophrenia.The PPI data complement orthogonal approaches to prioritize risk genes from GWAS lociFunctionally interpreting GWAS data to identify causal genes based on genome-wide significant SNPs is a major challenge in the field of genetics. Because our analyses indicate that the combined PPI network of all IPs is genetically and transcriptionally relevant in schizophrenia, we used the network to prioritize additional schizophrenia risk genes in GWAS loci. We created a ‘social Manhattan plot’ by integrating our PPI data with the most recent PGC schizophrenia GWAS10 (phase 3), highlighting observed interactions between the index proteins and other proteins (locus proteins, hereafter) encoded by genes in the 287 genome-wide significant loci (Figure 4A and Table S13). In total, we identified 123 locus proteins in 74 loci that are linked to ≥1 index protein in the social Manhattan plot. We further intersected these locus proteins with those prioritized by fine-mapping (FINEMAP) or eQTL co-localization (summary-based Mendelian randomization, SMR) analysis, pinpointing nine proteins that were also prioritized by FINEMAP (ACTR1B, EPN2, GABBR2, KIAA1549, MSI2, NEGR1, NLGN4X, PDE4B, PTPRD) and three proteins that were also prioritized by SMR (PCDHA2, PCDHA8, SF3B1). In addition, our network was able to nominate candidate genes in 44 distinct loci that lack prioritization results from FINEMAP or SMR analysis.Figure 4Prioritizing genes in schizophrenia GWAS loci using brain cell-type-specific interactome data(A) Social Manhattan plot of genes encoding the index proteins [red] and their interactors [purple] in genome-wide significant loci in PGC schizophrenia GWAS [phase 3]. Size of the interactor nodes and their labels scale with the number of index genes linked to each interactor; those that were also prioritized by FINEMAP or SMR analysis are highlighted in magenta. Gray lines indicate observed protein-protein interactions in our data; interactions that have been replicated by IP-WB are highlighted in blue.(B) Cumulative density of gnomAD pLI scores for different gene sets. ‘Genome’ indicates genes in the pLI dataset [excluding PGC3 genes]; ‘PGC3’ indicates genes in PGC GWAS [phase 3] loci; ‘FINEMAP’, ‘SMR’, and ‘Network’ indicate PGC3 genes prioritized by FINEMAP, SMR, or our interactome data, respectively; ‘Overlap’ indicates genes overlapping between Network and FINEMAP or SMR; ‘SCHEMA 5% FDR’ and ‘SCHEMA EWS’ indicate genes with FDR < 0.05 or 3.7e-3 [exome-wide significance] in SCHEMA, respectively.(C) Frontal cortex RNA expression of gene sets across ten developmental stages. Median expression and standard error [SE] of each gene set were derived from the BrainSpan exon microarray dataset. ‘Random’ indicates genes randomly sampled from the BrainSpan dataset; ‘PGC3’, ‘FINEMAP’, ‘SMR’, ‘Network’, and ‘Overlap’ indicate gene sets as described in (B); ‘SCHEMA’ indicates exome-wide significant genes from SCHEMA. Shaded regions indicate median expression of genes with FDR < 3.7e-3 [exome-wide significance], 0.05, 0.25, or 0.5 in SCHEMA with darker gray indicating greater significance.(D) Western blot analysis on independent IPs of index proteins [named on the top] to detect the presence of selected locus proteins [named on the side of each gel]. Green and red circles at the bottom represented whether the tested interaction was significant or non-significant by IP-MS, respectively. Each lane represents 10% of the IP material analyzed by IP-MS. L = ladder. Molecular weights are in KDa. See also Figure S10 and Table S13.To further compare our network prioritization approach to FINEMAP or SMR, we looked at the pLI score distributions and BrainSpan expression profiles of several gene sets, including all genes in the GWAS loci (PGC3), genes prioritized by our network (Network), FINEMAP, or SMR, and genes prioritized by both our network and either FINEMAP or SMR (Overlap). The Network and FINEMAP genes both have significantly higher pLI scores compared to all PGC3 genes (one-tailed KS test p = 7.9e-6 and 2.0e-4, respectively) and genes in the same loci as the prioritized genes (one-tailed KS test p = 1.4e-6 and 1.1e-4, respectively; Figure 4B and Table S2). In contrast, the SMR genes did not show such enrichment. The Overlap genes have even higher pLI scores that are comparable to the SCHEMA genes with FDR < 0.05 (two-tailed KS test p = 0.36), although the differences between the Overlap genes and other genes in the Network, FINEMAP, or SMR supersets are not statistically significant. When looking at gene expression throughout frontal cortical development, the Network, FINEMAP, SMR, and Overlap genes all have elevated expression compared to all PGC3 genes. In particular, the Network and Overlap genes have higher expression than the FINEMAP and SMR genes; their postnatal expression profiles are close to that of the high-confidence SCHEMA genes (Figure 4C and Table S3).We also compared the Network genes to other PGC3 genes encoding iN-expressed proteins (i.e., non-interactors in Table S9) and observed that these two gene sets have comparable pLI scores and BrainSpan expression (Tables S2 and S3). These results indicate that the proteome of cultured neurons may contribute to the pLI score and expression enrichment we observed for the Network genes. However, by first showing that our PPI network is significantly enriched for schizophrenia genetic risk and transcriptional perturbations compared to the non-interactors, and then showing that the network genes in GWAS loci have higher pLI scores and frontal cortical expression throughout development compared to other genes in the loci, we demonstrate how our interactome data can complement orthogonal methods such as fine-mapping and eQTL co-localization analyses to nominate schizophrenia risk genes from GWAS data.Finally, to illustrate that many of the locus proteins we prioritized could be reproducible interactors of the index proteins, we repeated the index protein IPs and performed western blotting to detect selected interactors; or vice versa, we executed reverse IPs for the interactors followed by western blotting to detect the index proteins. In total, we replicated 25 interactions for 18 unique locus proteins using forward or reverse IPs, including eight proteins that were also prioritized by FINEMAP or SMR, as well as an interaction between two of the index proteins, HCN1 and SYNGAP1 (Figures 4A, S3, and S5, and Table S7). We also performed western blots for several locus proteins on all six index protein IPs in parallel, showing that their detection patterns agree with their significant interactions with distinct index proteins in IP-MS (Figures 4D and S10). These results support the reproducibility of the PPI data and the observed convergence between the index protein interactomes. Follow-up investigation on the prioritized locus proteins can be informative for determining whether their corresponding genes are true schizophrenia risk genes responsible for the genetic signals observed in GWAS.

Prioritization of schizophrenia index genes with shared neurodevelopmental signalsTo identify schizophrenia risk genes as the basis of our interactome experiments we designed and executed a three-step procedure (Figure 1A and Table S1). First, we identified 445 genes (Set 1) in previously reported genome-wide significant loci from the Psychiatric Genomics Consortium (PGC) genome-wide association study3 (GWAS; phase 2). Second, we filtered this set to 37 genes (Set 2) within single protein-coding gene loci, excluding other genes in loci with more ambiguous association signals. Third, we integrated data from orthogonal studies (e.g., high-density genotyping, exome sequencing, and earlier targeted studies of individual genes; STAR Methods and Table S1) to identify a subset of 10 genes (Set 3) supported by multiple independent lines of evidence. Importantly, we used orthogonal evidence from rare variant studies of ASD/DD to prioritize schizophrenia risk genes that have also been implicated in neurodevelopmental conditions. We additionally included SYNGAP1 in the major histocompatibility complex (MHC) region in all three sets because of strong orthogonal evidence for its involvement in schizophrenia and neurodevelopmental disorders.Figure 1Selection of schizophrenia index genes and proteins for interactome experiments(A) Three-step procedure to identify Sets 1–3 by refining schizophrenia GWAS [PGC phase 2] data, where Set 3 was defined as ‘index genes’ and used as the basis for downstream experiments. Set 3 genes are color-coded based on the type of orthogonal evidence supporting their involvement in neuropsychiatric or neurodevelopmental phenotypes.(B) Cumulative density of gnomAD pLI scores for different gene sets. ‘Genome’ indicates genes in the pLI dataset [excluding Sets 1–3]; ‘Sets 1–3’ indicate genes in Sets 1–3 with available pLI scores; ‘SCHEMA 5% FDR’ and ‘SCHEMA EWS’ indicate genes with FDR < 0.05 or 3.7e-3 [exome-wide significance] in the SCHEMA exome sequencing analysis, respectively.(C) Frontal cortex RNA expression of gene sets across ten developmental stages. Median expression and standard error [SE] of each gene set were derived from the BrainSpan exon microarray dataset. ‘Random’ indicates genes randomly sampled from the BrainSpan dataset; ‘Sets 1–3’ indicate genes in Sets 1–3 with available BrainSpan data; ‘SCHEMA’ indicates exome-wide significant genes from SCHEMA. Shaded regions indicate median expression of genes with FDR < 3.7e-3 [exome-wide significance], 0.05, 0.25, or 0.5 in SCHEMA with darker gray indicating greater significance.(D) Western blot analysis of index proteins in iPSCs, NPCs [at day 3 of differentiation], iNs [at weeks 2–7 of differentiation], three cancer cell lines [TF-1, K562, U937], HEK293 cells, and mouse cortex. SATB2 and ZNF804A are excluded from this panel due to lack of detectable expression in iNs. See also Figure S1, Data S4, and Tables S1, S2, S3, and S4.Genes implicated in schizophrenia are under strong genetic selection and have elevated expression in the frontal cortex.9 Therefore, to assess the enrichment of schizophrenia risk genes in Sets 1–3, we compared their gnomAD22 pLI scores (i.e., the probability of being loss-of-function intolerant, where genes under strong selection have higher pLI scores) and BrainSpan23 expression to schizophrenia risk genes from the recent Schizophrenia Exome Sequencing Meta-Analysis (SCHEMA) study.9 In terms of pLI scores, Sets 1–3 all have increasingly higher scores compared to other genes in the genome (one-tailed KS test p = 1.1e-3, 1.7e-3, and 1.3e-5 for Sets 1–3, respectively; Figure 1B and Table S2). Set 3 scores are significantly higher than that of Set 2 (one-tailed KS test p = 5.8e-4), which in turn are higher than that of Set 1 (one-tailed KS test p = 0.019). Furthermore, Set 3 is under the same degree of constraint as the exome-wide significant (FDR < 3.7e-3) SCHEMA genes (two-tailed KS test p = 0.23). In terms of gene expression, Set 1 mirrors the expression profile of random genes during frontal cortical development up until adulthood; Set 2 has a postnatal expression profile that resembles SCHEMA genes with FDR of 0.25–0.5; and Set 3 has a postnatal expression profile that strongly mirrors the exome-wide significant SCHEMA genes (Figure 1C and Table S3). Although the lower prenatal expression of Set 3 compared to the SCHEMA genes may be reflecting different aspects of schizophrenia-related biology captured by common versus rare variants, respectively (Data S4), the pLI scores and postnatal expression patterns generally support our three-step approach to gene selection based on refining GWAS data and indicate that Set 3 is enriched for bona fide schizophrenia risk genes. We proceeded to use Set 3 as the starting point of our experiments and refer to these 11 genes and their encoded proteins as ‘index genes’ and ‘index proteins’, respectively.

Prioritization of schizophrenia index genes with shared neurodevelopmental signals

To identify schizophrenia risk genes as the basis of our interactome experiments we designed and executed a three-step procedure (Figure 1A and Table S1). First, we identified 445 genes (Set 1) in previously reported genome-wide significant loci from the Psychiatric Genomics Consortium (PGC) genome-wide association study3 (GWAS; phase 2). Second, we filtered this set to 37 genes (Set 2) within single protein-coding gene loci, excluding other genes in loci with more ambiguous association signals. Third, we integrated data from orthogonal studies (e.g., high-density genotyping, exome sequencing, and earlier targeted studies of individual genes; STAR Methods and Table S1) to identify a subset of 10 genes (Set 3) supported by multiple independent lines of evidence. Importantly, we used orthogonal evidence from rare variant studies of ASD/DD to prioritize schizophrenia risk genes that have also been implicated in neurodevelopmental conditions. We additionally included SYNGAP1 in the major histocompatibility complex (MHC) region in all three sets because of strong orthogonal evidence for its involvement in schizophrenia and neurodevelopmental disorders.Figure 1Selection of schizophrenia index genes and proteins for interactome experiments(A) Three-step procedure to identify Sets 1–3 by refining schizophrenia GWAS [PGC phase 2] data, where Set 3 was defined as ‘index genes’ and used as the basis for downstream experiments. Set 3 genes are color-coded based on the type of orthogonal evidence supporting their involvement in neuropsychiatric or neurodevelopmental phenotypes.(B) Cumulative density of gnomAD pLI scores for different gene sets. ‘Genome’ indicates genes in the pLI dataset [excluding Sets 1–3]; ‘Sets 1–3’ indicate genes in Sets 1–3 with available pLI scores; ‘SCHEMA 5% FDR’ and ‘SCHEMA EWS’ indicate genes with FDR < 0.05 or 3.7e-3 [exome-wide significance] in the SCHEMA exome sequencing analysis, respectively.(C) Frontal cortex RNA expression of gene sets across ten developmental stages. Median expression and standard error [SE] of each gene set were derived from the BrainSpan exon microarray dataset. ‘Random’ indicates genes randomly sampled from the BrainSpan dataset; ‘Sets 1–3’ indicate genes in Sets 1–3 with available BrainSpan data; ‘SCHEMA’ indicates exome-wide significant genes from SCHEMA. Shaded regions indicate median expression of genes with FDR < 3.7e-3 [exome-wide significance], 0.05, 0.25, or 0.5 in SCHEMA with darker gray indicating greater significance.(D) Western blot analysis of index proteins in iPSCs, NPCs [at day 3 of differentiation], iNs [at weeks 2–7 of differentiation], three cancer cell lines [TF-1, K562, U937], HEK293 cells, and mouse cortex. SATB2 and ZNF804A are excluded from this panel due to lack of detectable expression in iNs. See also Figure S1, Data S4, and Tables S1, S2, S3, and S4.

Selection of schizophrenia index genes and proteins for interactome experiments

(A) Three-step procedure to identify Sets 1–3 by refining schizophrenia GWAS [PGC phase 2] data, where Set 3 was defined as ‘index genes’ and used as the basis for downstream experiments. Set 3 genes are color-coded based on the type of orthogonal evidence supporting their involvement in neuropsychiatric or neurodevelopmental phenotypes.

(B) Cumulative density of gnomAD pLI scores for different gene sets. ‘Genome’ indicates genes in the pLI dataset [excluding Sets 1–3]; ‘Sets 1–3’ indicate genes in Sets 1–3 with available pLI scores; ‘SCHEMA 5% FDR’ and ‘SCHEMA EWS’ indicate genes with FDR < 0.05 or 3.7e-3 [exome-wide significance] in the SCHEMA exome sequencing analysis, respectively.

(C) Frontal cortex RNA expression of gene sets across ten developmental stages. Median expression and standard error [SE] of each gene set were derived from the BrainSpan exon microarray dataset. ‘Random’ indicates genes randomly sampled from the BrainSpan dataset; ‘Sets 1–3’ indicate genes in Sets 1–3 with available BrainSpan data; ‘SCHEMA’ indicates exome-wide significant genes from SCHEMA. Shaded regions indicate median expression of genes with FDR < 3.7e-3 [exome-wide significance], 0.05, 0.25, or 0.5 in SCHEMA with darker gray indicating greater significance.

(D) Western blot analysis of index proteins in iPSCs, NPCs [at day 3 of differentiation], iNs [at weeks 2–7 of differentiation], three cancer cell lines [TF-1, K562, U937], HEK293 cells, and mouse cortex. SATB2 and ZNF804A are excluded from this panel due to lack of detectable expression in iNs. See also Figure S1, Data S4, and Tables S1, S2, S3, and S4.

Genes implicated in schizophrenia are under strong genetic selection and have elevated expression in the frontal cortex.9 Therefore, to assess the enrichment of schizophrenia risk genes in Sets 1–3, we compared their gnomAD22 pLI scores (i.e., the probability of being loss-of-function intolerant, where genes under strong selection have higher pLI scores) and BrainSpan23 expression to schizophrenia risk genes from the recent Schizophrenia Exome Sequencing Meta-Analysis (SCHEMA) study.9 In terms of pLI scores, Sets 1–3 all have increasingly higher scores compared to other genes in the genome (one-tailed KS test p = 1.1e-3, 1.7e-3, and 1.3e-5 for Sets 1–3, respectively; Figure 1B and Table S2). Set 3 scores are significantly higher than that of Set 2 (one-tailed KS test p = 5.8e-4), which in turn are higher than that of Set 1 (one-tailed KS test p = 0.019). Furthermore, Set 3 is under the same degree of constraint as the exome-wide significant (FDR < 3.7e-3) SCHEMA genes (two-tailed KS test p = 0.23). In terms of gene expression, Set 1 mirrors the expression profile of random genes during frontal cortical development up until adulthood; Set 2 has a postnatal expression profile that resembles SCHEMA genes with FDR of 0.25–0.5; and Set 3 has a postnatal expression profile that strongly mirrors the exome-wide significant SCHEMA genes (Figure 1C and Table S3). Although the lower prenatal expression of Set 3 compared to the SCHEMA genes may be reflecting different aspects of schizophrenia-related biology captured by common versus rare variants, respectively (Data S4), the pLI scores and postnatal expression patterns generally support our three-step approach to gene selection based on refining GWAS data and indicate that Set 3 is enriched for bona fide schizophrenia risk genes. We proceeded to use Set 3 as the starting point of our experiments and refer to these 11 genes and their encoded proteins as ‘index genes’ and ‘index proteins’, respectively.

Interaction proteomics of schizophrenia index proteins in excitatory iNsTo study the expression patterns of the index proteins throughout iN maturation, we tested 58 commercially available antibodies and identified 31 with competency to detect the 11 index proteins (Table S4). We differentiated iPSCs into neural progenitor cells (NPCs; day 3) and iNs (weeks 2–7) and confirmed protein expression of CACNA1C, CACNB2, CSMD1, CUL3, GRIN2A, HCN1, RIMS1, SYNGAP1, and TCF4 in neuron lysates by western blot (Figures 1D and S1). SATB2 and ZNF804A lacked detectable neuronal expression or high-quality reagents and were excluded from further experiments. When comparing index protein expression between iNs, non-neuronal cells, and mouse cortex, CACNA1C, CSMD1, GRIN2A, HCN1, RIMS1, and SYNGAP1 displayed a neuron-specific expression profile.Next, we tested 42 antibodies for their ability to immunoprecipitate (IP) the index proteins (Table S4) and were able to IP seven index proteins (CACNA1C, CACNB2, CUL3, HCN1, RIMS1, SYNGAP1, TCF4) in ∼seven billion iNs for subsequent mass spectrometry (MS) analyses. In total, we carried out 23 IP-MS experiments at five neuronal differentiation time points. We performed quality control (QC) and analyzed each experiment using Genoppi,24 calculating the log2 fold change (FC) and corresponding statistical significance for each protein identified in the index protein IPs compared to the controls, and then defining proteins with log2 FC > 0 and FDR ≤ 0.1 as the significant interactors of the index protein. We disregarded four IP experiments that did not meet our QC criteria (i.e., the log2 FC correlation between replicates was < 0.5 or the index protein itself was not enriched at FDR ≤ 0.1). The remaining 19 high-quality IPs of CACNA1C, CUL3, HCN1, RIMS1, SYNGAP1, and TCF4 had a median replicate log2 FC correlation of 0.87, with the six index proteins enriched at a median FDR of 8.2e-4 (Figures 2A, 2B, S2A, S2B; Tables S5 and S6). In addition, we performed experimental and computational analyses to confirm that, despite using an inclusive FC cutoff (i.e., log2 FC > 0) to define index protein interactors in the IPs, the interactors showed no obvious quality differences across a range of different FCs (Data S5).Figure 2Cell-type-specific protein interactomes in cortical human neurons(A) Scatter plot showing log2 FC correlation between replicate 1 [x-axis] and replicate 2 [y-axis] of an IP of CACNA1C at week 3 of neuron differentiation [Pearson’s r = 0.74].(B) Volcano plot showing log2 FC [x-axis] and -log10 p-value [y-axis] of the CACNA1C IP from (A). For (A) and (B), the index protein [CACNA1C] is shown in red, significant interactors [log2 FC > 0 and FDR ≤ 0.1] in green, and non-interactors [i.e., other detected proteins] in blue. Known InWeb interactors are indicated by black border circles, with the subset that are significant in the IP highlighted in yellow [overlap p = 1.8e-2]. Calcium channel components [alpha, beta, and alpha2delta subunits] are in orange.(C) Replication rates of a subset of interactions tested in forward or reverse IPs followed by western blotting [IP-WB].(D) Pairwise co-expression Z-scores between index genes and their interactors [Int], non-interactors [NonInt], known InWeb interactors [InWeb], and all protein-coding genes [All] derived from a spatial transcriptomic dataset in human dorsolateral prefrontal cortex. Boxes and whiskers in violin plots indicate the interquartile range [IQR] and 1.5x IQR, respectively. Double asterisks indicate p < 0.05/6 [adjusting for six pairwise comparisons] as calculated by two-tailed Wilcoxon rank-sum tests. Number of gene pairs plotted for each gene type is indicated toward the bottom.(E) The combined interaction network of six index proteins resulting from 19 individual IPs. Index proteins and their interactors are indicated as red and purple nodes, respectively. Size and color of the interactor nodes scale with the number of index proteins linked to each interactor, with larger and darker nodes representing more recurrent interactors [distribution shown in upper right pie chart]. Edges represent protein interactions with colors indicating whether each interaction is known in InWeb [blue] or potentially novel [gray; distribution shown in lower right pie chart]. See also Figures S2 and S7, Data S5 and S6, and Tables S4, S5, S6, S7, S8, S9, and S10.

Interaction proteomics of schizophrenia index proteins in excitatory iNs

To study the expression patterns of the index proteins throughout iN maturation, we tested 58 commercially available antibodies and identified 31 with competency to detect the 11 index proteins (Table S4). We differentiated iPSCs into neural progenitor cells (NPCs; day 3) and iNs (weeks 2–7) and confirmed protein expression of CACNA1C, CACNB2, CSMD1, CUL3, GRIN2A, HCN1, RIMS1, SYNGAP1, and TCF4 in neuron lysates by western blot (Figures 1D and S1). SATB2 and ZNF804A lacked detectable neuronal expression or high-quality reagents and were excluded from further experiments. When comparing index protein expression between iNs, non-neuronal cells, and mouse cortex, CACNA1C, CSMD1, GRIN2A, HCN1, RIMS1, and SYNGAP1 displayed a neuron-specific expression profile.

Next, we tested 42 antibodies for their ability to immunoprecipitate (IP) the index proteins (Table S4) and were able to IP seven index proteins (CACNA1C, CACNB2, CUL3, HCN1, RIMS1, SYNGAP1, TCF4) in ∼seven billion iNs for subsequent mass spectrometry (MS) analyses. In total, we carried out 23 IP-MS experiments at five neuronal differentiation time points. We performed quality control (QC) and analyzed each experiment using Genoppi,24 calculating the log2 fold change (FC) and corresponding statistical significance for each protein identified in the index protein IPs compared to the controls, and then defining proteins with log2 FC > 0 and FDR ≤ 0.1 as the significant interactors of the index protein. We disregarded four IP experiments that did not meet our QC criteria (i.e., the log2 FC correlation between replicates was < 0.5 or the index protein itself was not enriched at FDR ≤ 0.1). The remaining 19 high-quality IPs of CACNA1C, CUL3, HCN1, RIMS1, SYNGAP1, and TCF4 had a median replicate log2 FC correlation of 0.87, with the six index proteins enriched at a median FDR of 8.2e-4 (Figures 2A, 2B, S2A, S2B; Tables S5 and S6). In addition, we performed experimental and computational analyses to confirm that, despite using an inclusive FC cutoff (i.e., log2 FC > 0) to define index protein interactors in the IPs, the interactors showed no obvious quality differences across a range of different FCs (Data S5).Figure 2Cell-type-specific protein interactomes in cortical human neurons(A) Scatter plot showing log2 FC correlation between replicate 1 [x-axis] and replicate 2 [y-axis] of an IP of CACNA1C at week 3 of neuron differentiation [Pearson’s r = 0.74].(B) Volcano plot showing log2 FC [x-axis] and -log10 p-value [y-axis] of the CACNA1C IP from (A). For (A) and (B), the index protein [CACNA1C] is shown in red, significant interactors [log2 FC > 0 and FDR ≤ 0.1] in green, and non-interactors [i.e., other detected proteins] in blue. Known InWeb interactors are indicated by black border circles, with the subset that are significant in the IP highlighted in yellow [overlap p = 1.8e-2]. Calcium channel components [alpha, beta, and alpha2delta subunits] are in orange.(C) Replication rates of a subset of interactions tested in forward or reverse IPs followed by western blotting [IP-WB].(D) Pairwise co-expression Z-scores between index genes and their interactors [Int], non-interactors [NonInt], known InWeb interactors [InWeb], and all protein-coding genes [All] derived from a spatial transcriptomic dataset in human dorsolateral prefrontal cortex. Boxes and whiskers in violin plots indicate the interquartile range [IQR] and 1.5x IQR, respectively. Double asterisks indicate p < 0.05/6 [adjusting for six pairwise comparisons] as calculated by two-tailed Wilcoxon rank-sum tests. Number of gene pairs plotted for each gene type is indicated toward the bottom.(E) The combined interaction network of six index proteins resulting from 19 individual IPs. Index proteins and their interactors are indicated as red and purple nodes, respectively. Size and color of the interactor nodes scale with the number of index proteins linked to each interactor, with larger and darker nodes representing more recurrent interactors [distribution shown in upper right pie chart]. Edges represent protein interactions with colors indicating whether each interaction is known in InWeb [blue] or potentially novel [gray; distribution shown in lower right pie chart]. See also Figures S2 and S7, Data S5 and S6, and Tables S4, S5, S6, S7, S8, S9, and S10.

Cell-type-specific protein interactomes in cortical human neurons

(A) Scatter plot showing log2 FC correlation between replicate 1 [x-axis] and replicate 2 [y-axis] of an IP of CACNA1C at week 3 of neuron differentiation [Pearson’s r = 0.74].

(B) Volcano plot showing log2 FC [x-axis] and -log10 p-value [y-axis] of the CACNA1C IP from (A). For (A) and (B), the index protein [CACNA1C] is shown in red, significant interactors [log2 FC > 0 and FDR ≤ 0.1] in green, and non-interactors [i.e., other detected proteins] in blue. Known InWeb interactors are indicated by black border circles, with the subset that are significant in the IP highlighted in yellow [overlap p = 1.8e-2]. Calcium channel components [alpha, beta, and alpha2delta subunits] are in orange.

(C) Replication rates of a subset of interactions tested in forward or reverse IPs followed by western blotting [IP-WB].

(D) Pairwise co-expression Z-scores between index genes and their interactors [Int], non-interactors [NonInt], known InWeb interactors [InWeb], and all protein-coding genes [All] derived from a spatial transcriptomic dataset in human dorsolateral prefrontal cortex. Boxes and whiskers in violin plots indicate the interquartile range [IQR] and 1.5x IQR, respectively. Double asterisks indicate p < 0.05/6 [adjusting for six pairwise comparisons] as calculated by two-tailed Wilcoxon rank-sum tests. Number of gene pairs plotted for each gene type is indicated toward the bottom.

(E) The combined interaction network of six index proteins resulting from 19 individual IPs. Index proteins and their interactors are indicated as red and purple nodes, respectively. Size and color of the interactor nodes scale with the number of index proteins linked to each interactor, with larger and darker nodes representing more recurrent interactors [distribution shown in upper right pie chart]. Edges represent protein interactions with colors indicating whether each interaction is known in InWeb [blue] or potentially novel [gray; distribution shown in lower right pie chart]. See also Figures S2 and S7, Data S5 and S6, and Tables S4, S5, S6, S7, S8, S9, and S10.

The PPI data contain newly reported and reproducible interactionsImportantly, our experimental design was purposely aimed at generating inclusive PPI datasets that encompass non-stoichiometric interactions, rather than restricting to direct interactions and core molecular complexes (Data S6). In previous work, we have established that the replication rate of interactions identified using a similar IP-MS approach is 70–90%.24 In the current study, we were able to recapitulate these estimates using a two-pronged approach (Figures 2C and S3–S5, and Table S7). First, we performed western blots of 45 interactors for the six index proteins on repetition of the index protein IPs and validated 40 of the interactors (88.9% replication rate). In parallel, we performed reverse IPs using a panel of interactors as baits, and successfully detected the original index proteins in 23 out of 25 reverse IPs that showed bait enrichment (92.0% replication rate). The ∼90% replication rates are in general agreement with the 10% FDR cutoff we applied to identify significant interactions.Besides replicating a subset of the index protein interactors, we also used published datasets to assess the biological validity of our IP-MS data. First, ten of the 19 IPs are further supported by the observation that they are enriched for known protein interactors derived from the InWeb database25 (Table S5). As an example, in a CACNA1C IP performed in iNs at week 3 of differentiation, the significant interactors are enriched for known CACNA1C interactors in InWeb (p = 1.8e-2), including all known L-type calcium channel subunits: the extracellular CACNA2D1 and the intracellular CACNB1 and CACNB326 (Figures 2A and 2B). On the other hand, >94% of the interactors are not found in InWeb nor in an IP of CACNA1C executed in mouse heart tissue with the same antibody.27 This example illustrates that our neuron-derived PPI data capture both known and novel biology as expected, given that existing PPI datasets were mostly generated in non-neuronal context using different experimental methods (Data S6).We also used brain co-expression data to systematically benchmark all index protein interactors in our data, reasoning that on average, transcripts of interacting proteins would be more likely to co-localize across tissues, cell types, and developmental time points (STAR Methods). Indeed, we observed that the interactors usually have higher co-expression with the index proteins compared to the ‘non-interactors’ (i.e., non-significant proteins detected in IP-MS), known interactors in InWeb, and all protein-coding genes in a spatial transcriptomic dataset derived from human dorsolateral prefrontal cortex28 (Figure 2D). Similar trends were also observed in other expression datasets from human or mouse brains29,30 (Figure S2C). This not only indicates that the interactions we identified in in vitro neurons represent biology found in complex brain tissues, but further suggests that they may be more enriched for gene relationships in the brain compared to proteins generally expressed in neurons (represented by the non-interactors) or interactions found in non-neuronal context (represented by the InWeb interactors). In summary, the experimental replications, InWeb overlap, and brain co-expression results all support the quality, reproducibility, and biological relevance of the interactome data we have generated for the schizophrenia index proteins in human iNs.

The PPI data contain newly reported and reproducible interactions

Importantly, our experimental design was purposely aimed at generating inclusive PPI datasets that encompass non-stoichiometric interactions, rather than restricting to direct interactions and core molecular complexes (Data S6). In previous work, we have established that the replication rate of interactions identified using a similar IP-MS approach is 70–90%.24 In the current study, we were able to recapitulate these estimates using a two-pronged approach (Figures 2C and S3–S5, and Table S7). First, we performed western blots of 45 interactors for the six index proteins on repetition of the index protein IPs and validated 40 of the interactors (88.9% replication rate). In parallel, we performed reverse IPs using a panel of interactors as baits, and successfully detected the original index proteins in 23 out of 25 reverse IPs that showed bait enrichment (92.0% replication rate). The ∼90% replication rates are in general agreement with the 10% FDR cutoff we applied to identify significant interactions.

Besides replicating a subset of the index protein interactors, we also used published datasets to assess the biological validity of our IP-MS data. First, ten of the 19 IPs are further supported by the observation that they are enriched for known protein interactors derived from the InWeb database25 (Table S5). As an example, in a CACNA1C IP performed in iNs at week 3 of differentiation, the significant interactors are enriched for known CACNA1C interactors in InWeb (p = 1.8e-2), including all known L-type calcium channel subunits: the extracellular CACNA2D1 and the intracellular CACNB1 and CACNB326 (Figures 2A and 2B). On the other hand, >94% of the interactors are not found in InWeb nor in an IP of CACNA1C executed in mouse heart tissue with the same antibody.27 This example illustrates that our neuron-derived PPI data capture both known and novel biology as expected, given that existing PPI datasets were mostly generated in non-neuronal context using different experimental methods (Data S6).

We also used brain co-expression data to systematically benchmark all index protein interactors in our data, reasoning that on average, transcripts of interacting proteins would be more likely to co-localize across tissues, cell types, and developmental time points (STAR Methods). Indeed, we observed that the interactors usually have higher co-expression with the index proteins compared to the ‘non-interactors’ (i.e., non-significant proteins detected in IP-MS), known interactors in InWeb, and all protein-coding genes in a spatial transcriptomic dataset derived from human dorsolateral prefrontal cortex28 (Figure 2D). Similar trends were also observed in other expression datasets from human or mouse brains29,30 (Figure S2C). This not only indicates that the interactions we identified in in vitro neurons represent biology found in complex brain tissues, but further suggests that they may be more enriched for gene relationships in the brain compared to proteins generally expressed in neurons (represented by the non-interactors) or interactions found in non-neuronal context (represented by the InWeb interactors). In summary, the experimental replications, InWeb overlap, and brain co-expression results all support the quality, reproducibility, and biological relevance of the interactome data we have generated for the schizophrenia index proteins in human iNs.

The combined PPI networks for six schizophrenia index proteinsWhen we compared multiple IPs of the same index protein across time points during neuronal maturation and between cell lines (for CACNA1C, HCN1, and SYNGAP1), we observed significant agreement in terms of the log2 FC correlation of all detected proteins (median correlation = 0.75; Figures S6A and S6C–S6E). When we clustered the IPs based on the percentage of overlap between significant interactors, we found that IPs from the earlier versus the later time points tend to fall into separate clusters, agreeing with known characteristics of maturing neurons generated using the NGN2-based protocol21 (Figures S6B and S6D–S6F and Table S8). However, we also observed relatively high percentages of overlap that are statistically significant across all time points (median percentage = 72%). Overall, these results indicate that our IPs from different time points capture a large proportion of overlapping biology, which is not unexpected given that differentiating iPSCs start to express neuronal markers soon after forcing NGN2 overexpression. Based on these observations, we decided to explore the combined PPI network of each index protein across time points in downstream analyses.We merged data from the 19 individual IPs to create nine additional consolidated datasets (Table S9). These datasets represent the combined PPI network of a single index protein across multiple experiments or time points (i.e., CACNA1C, HCN1, RIMS1, and SYNGAP1), the combined network of multiple index proteins at one time point (i.e., week 2, 3, 4, and 7), and the combined network of all six index proteins across all time points (i.e., ‘all combined’). The all combined network contains 1,238 interactors of the six index proteins and, like the CACNA1C IP highlighted above, >96% of the interactions in this network are potentially novel interactions not found in InWeb (Figure 2E and Table S5). Genes encoding these interactors have relatively high expression in the frontal cortex throughout brain development similar to schizophrenia risk genes reaching 5% FDR in SCHEMA (Figure S7A, Table S3, and Data S4). SynGO31 gene set analysis also found the network to be enriched for genes involved in various biological processes in the synapse (Figure S7B and Table S10). Overall, we successfully mapped the neuronal protein interactomes of six proteins that are transmembrane (CACNA1C, HCN1), cytosolic (CUL3, SYNGAP1), and involved in multiple neuronal signaling processes (RIMS1, SYNGAP1, TCF4). The resulting PPI networks include a high percentage of newly reported interactions and span many areas of the cell biology of cortical excitatory neurons.

The combined PPI networks for six schizophrenia index proteins

When we compared multiple IPs of the same index protein across time points during neuronal maturation and between cell lines (for CACNA1C, HCN1, and SYNGAP1), we observed significant agreement in terms of the log2 FC correlation of all detected proteins (median correlation = 0.75; Figures S6A and S6C–S6E). When we clustered the IPs based on the percentage of overlap between significant interactors, we found that IPs from the earlier versus the later time points tend to fall into separate clusters, agreeing with known characteristics of maturing neurons generated using the NGN2-based protocol21 (Figures S6B and S6D–S6F and Table S8). However, we also observed relatively high percentages of overlap that are statistically significant across all time points (median percentage = 72%). Overall, these results indicate that our IPs from different time points capture a large proportion of overlapping biology, which is not unexpected given that differentiating iPSCs start to express neuronal markers soon after forcing NGN2 overexpression. Based on these observations, we decided to explore the combined PPI network of each index protein across time points in downstream analyses.

We merged data from the 19 individual IPs to create nine additional consolidated datasets (Table S9). These datasets represent the combined PPI network of a single index protein across multiple experiments or time points (i.e., CACNA1C, HCN1, RIMS1, and SYNGAP1), the combined network of multiple index proteins at one time point (i.e., week 2, 3, 4, and 7), and the combined network of all six index proteins across all time points (i.e., ‘all combined’). The all combined network contains 1,238 interactors of the six index proteins and, like the CACNA1C IP highlighted above, >96% of the interactions in this network are potentially novel interactions not found in InWeb (Figure 2E and Table S5). Genes encoding these interactors have relatively high expression in the frontal cortex throughout brain development similar to schizophrenia risk genes reaching 5% FDR in SCHEMA (Figure S7A, Table S3, and Data S4). SynGO31 gene set analysis also found the network to be enriched for genes involved in various biological processes in the synapse (Figure S7B and Table S10). Overall, we successfully mapped the neuronal protein interactomes of six proteins that are transmembrane (CACNA1C, HCN1), cytosolic (CUL3, SYNGAP1), and involved in multiple neuronal signaling processes (RIMS1, SYNGAP1, TCF4). The resulting PPI networks include a high percentage of newly reported interactions and span many areas of the cell biology of cortical excitatory neurons.

The PPI networks are enriched for genetic risks of schizophrenia, ASD, and DDTo test the networks for association to schizophrenia, we assessed the enrichment of common variant risk across the networks using PGC GWAS data containing schizophrenia cases and controls of European (EUR) or East Asian (EAS) ancestry.3,8 For these and all further network enrichment analyses, we created a more conservative version of the combined networks by excluding proteins that showed up as non-interactors in any of the source IPs (see ‘stringent interactors’ in Table S9). In total, we performed genetic analyses for 11 interaction networks, including the nine combined networks and two individual IP networks for CUL3 and TCF4. Importantly, our analyses were conditional on the non-interactors detected in our IP experiments, meaning that we tested whether interactors in the different networks are enriched for genetic risk compared to other iN-expressed proteins. If so, this would suggest that our networks are relevant to schizophrenia over and above the background proteome of the neuronal cell model.Using MAGMA,32 we found that many of the PPI networks are indeed enriched for schizophrenia risk when conditioned on other iN-expressed proteins. Notably, the enrichment is generally consistent across EUR and EAS ancestries (Figure 3A and Table S11). At a Bonferroni-corrected threshold (p < 0.05/22, adjusting for 11 networks and two ancestries), the week 4 network in both EUR and EAS ancestries, as well as the all combined, week 2, week 7, and HCN1 networks in EAS showed significant enrichment. In the cross-ancestry meta-analysis, the all combined, week 2, week 4, week 7, and HCN1 networks were all significant at the same Bonferroni-corrected threshold. In parallel, we further validated these findings using a genetic risk score (GRS) enrichment analysis method that estimates the genetic risk on holdout samples not included in GWAS and therefore is less sensitive to outliers (STAR Methods). Most results were replicated, including the enrichment signals for the all combined, week 4, and HCN1 networks in both ancestries and their meta-analysis (Figure S8B and Table S11).Figure 3Enrichment of common variant risks and transcriptional perturbations in the index protein interactomesNetworks tested are the combined network of all IPs [All combined], the combined networks at each time point [Week 2 to Week 7], the combined networks for CACNA1C, HCN1, RIMS1, and SYNGAP1, and the individual IP networks for CUL3 and TCF4; the number of genes in each network is shown in parentheses on the y-axes.(A) Common variant enrichment of schizophrenia [SCZ] or height in Europeans [EUR], East Asians [EAS], or their meta-analysis. Enrichment coefficients, standard errors [SE], and p-values were calculated using MAGMA. p < 0.05 or p < 0.05/22 [adjusting for 11 networks and two ancestries] results are highlighted in orange or red, respectively.(B) Common variant enrichment of SCZ, attention deficit hyperactivity disorder [ADHD], autism spectrum disorders [ASD], bipolar disorder [BIP], or major depressive disorder [MDD] calculated using MAGMA. Cross-ancestry meta-analysis results are shown for SCZ; EUR ancestry results are shown for other disorders. Enrichment coefficients reaching p < 0.05 or p < 0.05/22 significance are shown in the heatmap followed by single or double asterisks, respectively.(C) Enrichment of cell-type-specific differentially expressed genes [DEGs] in the prefrontal cortex of schizophrenia patients compared to controls; the number of DEGs in each cell type is shown in parentheses on the x-axis. p-values were calculated using one-tailed hypergeometric tests. Gene counts in overlaps reaching p < 0.05 or p < 0.05/220 [adjusting for 11 networks and 20 cell types] significance are shown in the heatmap followed by single or double asterisks, respectively. See also Figures S8 and S9, Tables S11 and S12.To further explore whether the genetic risk enrichment we observed in the PPI networks is specific to schizophrenia, we repeated the same analyses using GWAS data of other psychiatric disorders, including attention deficit hyperactivity disorder33 (ADHD), ASD,34 bipolar disorder35 (BIP), and major depressive disorder36 (MDD), as well as height37,38 as a control trait (Figures 3A, 3B, S8A, and S8B, and Table S11). Across these phenotypes, we only observed robust enrichment for ASD in the CUL3 network and the week 7 network which contains IP data of CUL3 and SYNGAP1, both of which have been previously linked to ASD.39,40 The networks that showed the most robust enrichment for schizophrenia (i.e., the all combined, week 4, and HCN1 networks) did not demonstrate enrichment for the other phenotypes.Besides analyzing data from common variants, we also tested whether the PPI networks are enriched for rare variant risks of schizophrenia,9 ASD,40 and DD,41 as well as for high gnomAD pLI scores (Figure S8C and Table S11). At a Bonferroni-corrected threshold (p < 0.05/11, adjusting for 11 networks), we found the CUL3 network to be enriched for DD genes, which agrees with the previously implicated role of CUL3 in developmental delay.39,42 In addition, the week 4 and RIMS1 networks are significantly enriched for genes with high pLI scores, indicating that some members of these networks are likely intolerant to loss-of-function mutations and may be involved in essential cell functions.Overall, these genetic analysis results indicate that most of the PPI network genes are specifically concentrating common variant risk of schizophrenia, whereas several sub-networks are also enriched for common or rare variant risks of ASD/DD, highlighting the shared biology between these disorders.

The PPI networks are enriched for genetic risks of schizophrenia, ASD, and DD

To test the networks for association to schizophrenia, we assessed the enrichment of common variant risk across the networks using PGC GWAS data containing schizophrenia cases and controls of European (EUR) or East Asian (EAS) ancestry.3,8 For these and all further network enrichment analyses, we created a more conservative version of the combined networks by excluding proteins that showed up as non-interactors in any of the source IPs (see ‘stringent interactors’ in Table S9). In total, we performed genetic analyses for 11 interaction networks, including the nine combined networks and two individual IP networks for CUL3 and TCF4. Importantly, our analyses were conditional on the non-interactors detected in our IP experiments, meaning that we tested whether interactors in the different networks are enriched for genetic risk compared to other iN-expressed proteins. If so, this would suggest that our networks are relevant to schizophrenia over and above the background proteome of the neuronal cell model.

Using MAGMA,32 we found that many of the PPI networks are indeed enriched for schizophrenia risk when conditioned on other iN-expressed proteins. Notably, the enrichment is generally consistent across EUR and EAS ancestries (Figure 3A and Table S11). At a Bonferroni-corrected threshold (p < 0.05/22, adjusting for 11 networks and two ancestries), the week 4 network in both EUR and EAS ancestries, as well as the all combined, week 2, week 7, and HCN1 networks in EAS showed significant enrichment. In the cross-ancestry meta-analysis, the all combined, week 2, week 4, week 7, and HCN1 networks were all significant at the same Bonferroni-corrected threshold. In parallel, we further validated these findings using a genetic risk score (GRS) enrichment analysis method that estimates the genetic risk on holdout samples not included in GWAS and therefore is less sensitive to outliers (STAR Methods). Most results were replicated, including the enrichment signals for the all combined, week 4, and HCN1 networks in both ancestries and their meta-analysis (Figure S8B and Table S11).Figure 3Enrichment of common variant risks and transcriptional perturbations in the index protein interactomesNetworks tested are the combined network of all IPs [All combined], the combined networks at each time point [Week 2 to Week 7], the combined networks for CACNA1C, HCN1, RIMS1, and SYNGAP1, and the individual IP networks for CUL3 and TCF4; the number of genes in each network is shown in parentheses on the y-axes.(A) Common variant enrichment of schizophrenia [SCZ] or height in Europeans [EUR], East Asians [EAS], or their meta-analysis. Enrichment coefficients, standard errors [SE], and p-values were calculated using MAGMA. p < 0.05 or p < 0.05/22 [adjusting for 11 networks and two ancestries] results are highlighted in orange or red, respectively.(B) Common variant enrichment of SCZ, attention deficit hyperactivity disorder [ADHD], autism spectrum disorders [ASD], bipolar disorder [BIP], or major depressive disorder [MDD] calculated using MAGMA. Cross-ancestry meta-analysis results are shown for SCZ; EUR ancestry results are shown for other disorders. Enrichment coefficients reaching p < 0.05 or p < 0.05/22 significance are shown in the heatmap followed by single or double asterisks, respectively.(C) Enrichment of cell-type-specific differentially expressed genes [DEGs] in the prefrontal cortex of schizophrenia patients compared to controls; the number of DEGs in each cell type is shown in parentheses on the x-axis. p-values were calculated using one-tailed hypergeometric tests. Gene counts in overlaps reaching p < 0.05 or p < 0.05/220 [adjusting for 11 networks and 20 cell types] significance are shown in the heatmap followed by single or double asterisks, respectively. See also Figures S8 and S9, Tables S11 and S12.

Enrichment of common variant risks and transcriptional perturbations in the index protein interactomes

Networks tested are the combined network of all IPs [All combined], the combined networks at each time point [Week 2 to Week 7], the combined networks for CACNA1C, HCN1, RIMS1, and SYNGAP1, and the individual IP networks for CUL3 and TCF4; the number of genes in each network is shown in parentheses on the y-axes.

(A) Common variant enrichment of schizophrenia [SCZ] or height in Europeans [EUR], East Asians [EAS], or their meta-analysis. Enrichment coefficients, standard errors [SE], and p-values were calculated using MAGMA. p < 0.05 or p < 0.05/22 [adjusting for 11 networks and two ancestries] results are highlighted in orange or red, respectively.

(B) Common variant enrichment of SCZ, attention deficit hyperactivity disorder [ADHD], autism spectrum disorders [ASD], bipolar disorder [BIP], or major depressive disorder [MDD] calculated using MAGMA. Cross-ancestry meta-analysis results are shown for SCZ; EUR ancestry results are shown for other disorders. Enrichment coefficients reaching p < 0.05 or p < 0.05/22 significance are shown in the heatmap followed by single or double asterisks, respectively.

(C) Enrichment of cell-type-specific differentially expressed genes [DEGs] in the prefrontal cortex of schizophrenia patients compared to controls; the number of DEGs in each cell type is shown in parentheses on the x-axis. p-values were calculated using one-tailed hypergeometric tests. Gene counts in overlaps reaching p < 0.05 or p < 0.05/220 [adjusting for 11 networks and 20 cell types] significance are shown in the heatmap followed by single or double asterisks, respectively. See also Figures S8 and S9, Tables S11 and S12.

To further explore whether the genetic risk enrichment we observed in the PPI networks is specific to schizophrenia, we repeated the same analyses using GWAS data of other psychiatric disorders, including attention deficit hyperactivity disorder33 (ADHD), ASD,34 bipolar disorder35 (BIP), and major depressive disorder36 (MDD), as well as height37,38 as a control trait (Figures 3A, 3B, S8A, and S8B, and Table S11). Across these phenotypes, we only observed robust enrichment for ASD in the CUL3 network and the week 7 network which contains IP data of CUL3 and SYNGAP1, both of which have been previously linked to ASD.39,40 The networks that showed the most robust enrichment for schizophrenia (i.e., the all combined, week 4, and HCN1 networks) did not demonstrate enrichment for the other phenotypes.

Besides analyzing data from common variants, we also tested whether the PPI networks are enriched for rare variant risks of schizophrenia,9 ASD,40 and DD,41 as well as for high gnomAD pLI scores (Figure S8C and Table S11). At a Bonferroni-corrected threshold (p < 0.05/11, adjusting for 11 networks), we found the CUL3 network to be enriched for DD genes, which agrees with the previously implicated role of CUL3 in developmental delay.39,42 In addition, the week 4 and RIMS1 networks are significantly enriched for genes with high pLI scores, indicating that some members of these networks are likely intolerant to loss-of-function mutations and may be involved in essential cell functions.

Overall, these genetic analysis results indicate that most of the PPI network genes are specifically concentrating common variant risk of schizophrenia, whereas several sub-networks are also enriched for common or rare variant risks of ASD/DD, highlighting the shared biology between these disorders.

The PPI networks implicate layer 5/6 cortical excitatory neurons in schizophreniaNext, we explored whether the PPI networks are enriched for brain-layer-specific transcriptional perturbations observed in patients with schizophrenia. We analyzed data from a recent single-cell RNA sequencing study,17which identified differentially expressed genes (DEGs) in individuals with schizophrenia versus controls in 20 annotated cell types in the prefrontal cortex. Many of our networks had nominally significant (p < 0.05) overlaps with the cell-type-specific DEGs in neuronal cell types, which include both upper and deep layer excitatory neurons and inhibitory interneurons (Figure 3C and Table S12). A population of layer 5/6 cortico-cortical projection neurons (‘Ex-L5-6CCb’) showed the most robust enrichment, reaching Bonferroni significance (p < 0.05/220, adjusting for 11 networks and 20 cell types) for the all combined, week 4, and week 7 networks. Subsequently, we separately analyzed the up- and down-regulated DEGs in this celltype, showing that the enrichment signals are strongly driven by the down-regulated DEGs (Figure S9 and Table S12). As DEGs in ‘Ex-L5-6CCb’ were also found to be enriched for genes implicated by schizophrenia GWAS,10 there is an intriguing convergence between our results and findings in patients that converge on deep layer cortical excitatory neurons as a key cell type for studying cellular networks involved in schizophrenia.

The PPI networks implicate layer 5/6 cortical excitatory neurons in schizophrenia

Next, we explored whether the PPI networks are enriched for brain-layer-specific transcriptional perturbations observed in patients with schizophrenia. We analyzed data from a recent single-cell RNA sequencing study,17which identified differentially expressed genes (DEGs) in individuals with schizophrenia versus controls in 20 annotated cell types in the prefrontal cortex. Many of our networks had nominally significant (p < 0.05) overlaps with the cell-type-specific DEGs in neuronal cell types, which include both upper and deep layer excitatory neurons and inhibitory interneurons (Figure 3C and Table S12). A population of layer 5/6 cortico-cortical projection neurons (‘Ex-L5-6CCb’) showed the most robust enrichment, reaching Bonferroni significance (p < 0.05/220, adjusting for 11 networks and 20 cell types) for the all combined, week 4, and week 7 networks. Subsequently, we separately analyzed the up- and down-regulated DEGs in this celltype, showing that the enrichment signals are strongly driven by the down-regulated DEGs (Figure S9 and Table S12). As DEGs in ‘Ex-L5-6CCb’ were also found to be enriched for genes implicated by schizophrenia GWAS,10 there is an intriguing convergence between our results and findings in patients that converge on deep layer cortical excitatory neurons as a key cell type for studying cellular networks involved in schizophrenia.

The PPI data complement orthogonal approaches to prioritize risk genes from GWAS lociFunctionally interpreting GWAS data to identify causal genes based on genome-wide significant SNPs is a major challenge in the field of genetics. Because our analyses indicate that the combined PPI network of all IPs is genetically and transcriptionally relevant in schizophrenia, we used the network to prioritize additional schizophrenia risk genes in GWAS loci. We created a ‘social Manhattan plot’ by integrating our PPI data with the most recent PGC schizophrenia GWAS10 (phase 3), highlighting observed interactions between the index proteins and other proteins (locus proteins, hereafter) encoded by genes in the 287 genome-wide significant loci (Figure 4A and Table S13). In total, we identified 123 locus proteins in 74 loci that are linked to ≥1 index protein in the social Manhattan plot. We further intersected these locus proteins with those prioritized by fine-mapping (FINEMAP) or eQTL co-localization (summary-based Mendelian randomization, SMR) analysis, pinpointing nine proteins that were also prioritized by FINEMAP (ACTR1B, EPN2, GABBR2, KIAA1549, MSI2, NEGR1, NLGN4X, PDE4B, PTPRD) and three proteins that were also prioritized by SMR (PCDHA2, PCDHA8, SF3B1). In addition, our network was able to nominate candidate genes in 44 distinct loci that lack prioritization results from FINEMAP or SMR analysis.Figure 4Prioritizing genes in schizophrenia GWAS loci using brain cell-type-specific interactome data(A) Social Manhattan plot of genes encoding the index proteins [red] and their interactors [purple] in genome-wide significant loci in PGC schizophrenia GWAS [phase 3]. Size of the interactor nodes and their labels scale with the number of index genes linked to each interactor; those that were also prioritized by FINEMAP or SMR analysis are highlighted in magenta. Gray lines indicate observed protein-protein interactions in our data; interactions that have been replicated by IP-WB are highlighted in blue.(B) Cumulative density of gnomAD pLI scores for different gene sets. ‘Genome’ indicates genes in the pLI dataset [excluding PGC3 genes]; ‘PGC3’ indicates genes in PGC GWAS [phase 3] loci; ‘FINEMAP’, ‘SMR’, and ‘Network’ indicate PGC3 genes prioritized by FINEMAP, SMR, or our interactome data, respectively; ‘Overlap’ indicates genes overlapping between Network and FINEMAP or SMR; ‘SCHEMA 5% FDR’ and ‘SCHEMA EWS’ indicate genes with FDR < 0.05 or 3.7e-3 [exome-wide significance] in SCHEMA, respectively.(C) Frontal cortex RNA expression of gene sets across ten developmental stages. Median expression and standard error [SE] of each gene set were derived from the BrainSpan exon microarray dataset. ‘Random’ indicates genes randomly sampled from the BrainSpan dataset; ‘PGC3’, ‘FINEMAP’, ‘SMR’, ‘Network’, and ‘Overlap’ indicate gene sets as described in (B); ‘SCHEMA’ indicates exome-wide significant genes from SCHEMA. Shaded regions indicate median expression of genes with FDR < 3.7e-3 [exome-wide significance], 0.05, 0.25, or 0.5 in SCHEMA with darker gray indicating greater significance.(D) Western blot analysis on independent IPs of index proteins [named on the top] to detect the presence of selected locus proteins [named on the side of each gel]. Green and red circles at the bottom represented whether the tested interaction was significant or non-significant by IP-MS, respectively. Each lane represents 10% of the IP material analyzed by IP-MS. L = ladder. Molecular weights are in KDa. See also Figure S10 and Table S13.To further compare our network prioritization approach to FINEMAP or SMR, we looked at the pLI score distributions and BrainSpan expression profiles of several gene sets, including all genes in the GWAS loci (PGC3), genes prioritized by our network (Network), FINEMAP, or SMR, and genes prioritized by both our network and either FINEMAP or SMR (Overlap). The Network and FINEMAP genes both have significantly higher pLI scores compared to all PGC3 genes (one-tailed KS test p = 7.9e-6 and 2.0e-4, respectively) and genes in the same loci as the prioritized genes (one-tailed KS test p = 1.4e-6 and 1.1e-4, respectively; Figure 4B and Table S2). In contrast, the SMR genes did not show such enrichment. The Overlap genes have even higher pLI scores that are comparable to the SCHEMA genes with FDR < 0.05 (two-tailed KS test p = 0.36), although the differences between the Overlap genes and other genes in the Network, FINEMAP, or SMR supersets are not statistically significant. When looking at gene expression throughout frontal cortical development, the Network, FINEMAP, SMR, and Overlap genes all have elevated expression compared to all PGC3 genes. In particular, the Network and Overlap genes have higher expression than the FINEMAP and SMR genes; their postnatal expression profiles are close to that of the high-confidence SCHEMA genes (Figure 4C and Table S3).We also compared the Network genes to other PGC3 genes encoding iN-expressed proteins (i.e., non-interactors in Table S9) and observed that these two gene sets have comparable pLI scores and BrainSpan expression (Tables S2 and S3). These results indicate that the proteome of cultured neurons may contribute to the pLI score and expression enrichment we observed for the Network genes. However, by first showing that our PPI network is significantly enriched for schizophrenia genetic risk and transcriptional perturbations compared to the non-interactors, and then showing that the network genes in GWAS loci have higher pLI scores and frontal cortical expression throughout development compared to other genes in the loci, we demonstrate how our interactome data can complement orthogonal methods such as fine-mapping and eQTL co-localization analyses to nominate schizophrenia risk genes from GWAS data.Finally, to illustrate that many of the locus proteins we prioritized could be reproducible interactors of the index proteins, we repeated the index protein IPs and performed western blotting to detect selected interactors; or vice versa, we executed reverse IPs for the interactors followed by western blotting to detect the index proteins. In total, we replicated 25 interactions for 18 unique locus proteins using forward or reverse IPs, including eight proteins that were also prioritized by FINEMAP or SMR, as well as an interaction between two of the index proteins, HCN1 and SYNGAP1 (Figures 4A, S3, and S5, and Table S7). We also performed western blots for several locus proteins on all six index protein IPs in parallel, showing that their detection patterns agree with their significant interactions with distinct index proteins in IP-MS (Figures 4D and S10). These results support the reproducibility of the PPI data and the observed convergence between the index protein interactomes. Follow-up investigation on the prioritized locus proteins can be informative for determining whether their corresponding genes are true schizophrenia risk genes responsible for the genetic signals observed in GWAS.

The PPI data complement orthogonal approaches to prioritize risk genes from GWAS loci

Functionally interpreting GWAS data to identify causal genes based on genome-wide significant SNPs is a major challenge in the field of genetics. Because our analyses indicate that the combined PPI network of all IPs is genetically and transcriptionally relevant in schizophrenia, we used the network to prioritize additional schizophrenia risk genes in GWAS loci. We created a ‘social Manhattan plot’ by integrating our PPI data with the most recent PGC schizophrenia GWAS10 (phase 3), highlighting observed interactions between the index proteins and other proteins (locus proteins, hereafter) encoded by genes in the 287 genome-wide significant loci (Figure 4A and Table S13). In total, we identified 123 locus proteins in 74 loci that are linked to ≥1 index protein in the social Manhattan plot. We further intersected these locus proteins with those prioritized by fine-mapping (FINEMAP) or eQTL co-localization (summary-based Mendelian randomization, SMR) analysis, pinpointing nine proteins that were also prioritized by FINEMAP (ACTR1B, EPN2, GABBR2, KIAA1549, MSI2, NEGR1, NLGN4X, PDE4B, PTPRD) and three proteins that were also prioritized by SMR (PCDHA2, PCDHA8, SF3B1). In addition, our network was able to nominate candidate genes in 44 distinct loci that lack prioritization results from FINEMAP or SMR analysis.Figure 4Prioritizing genes in schizophrenia GWAS loci using brain cell-type-specific interactome data(A) Social Manhattan plot of genes encoding the index proteins [red] and their interactors [purple] in genome-wide significant loci in PGC schizophrenia GWAS [phase 3]. Size of the interactor nodes and their labels scale with the number of index genes linked to each interactor; those that were also prioritized by FINEMAP or SMR analysis are highlighted in magenta. Gray lines indicate observed protein-protein interactions in our data; interactions that have been replicated by IP-WB are highlighted in blue.(B) Cumulative density of gnomAD pLI scores for different gene sets. ‘Genome’ indicates genes in the pLI dataset [excluding PGC3 genes]; ‘PGC3’ indicates genes in PGC GWAS [phase 3] loci; ‘FINEMAP’, ‘SMR’, and ‘Network’ indicate PGC3 genes prioritized by FINEMAP, SMR, or our interactome data, respectively; ‘Overlap’ indicates genes overlapping between Network and FINEMAP or SMR; ‘SCHEMA 5% FDR’ and ‘SCHEMA EWS’ indicate genes with FDR < 0.05 or 3.7e-3 [exome-wide significance] in SCHEMA, respectively.(C) Frontal cortex RNA expression of gene sets across ten developmental stages. Median expression and standard error [SE] of each gene set were derived from the BrainSpan exon microarray dataset. ‘Random’ indicates genes randomly sampled from the BrainSpan dataset; ‘PGC3’, ‘FINEMAP’, ‘SMR’, ‘Network’, and ‘Overlap’ indicate gene sets as described in (B); ‘SCHEMA’ indicates exome-wide significant genes from SCHEMA. Shaded regions indicate median expression of genes with FDR < 3.7e-3 [exome-wide significance], 0.05, 0.25, or 0.5 in SCHEMA with darker gray indicating greater significance.(D) Western blot analysis on independent IPs of index proteins [named on the top] to detect the presence of selected locus proteins [named on the side of each gel]. Green and red circles at the bottom represented whether the tested interaction was significant or non-significant by IP-MS, respectively. Each lane represents 10% of the IP material analyzed by IP-MS. L = ladder. Molecular weights are in KDa. See also Figure S10 and Table S13.

Prioritizing genes in schizophrenia GWAS loci using brain cell-type-specific interactome data

(A) Social Manhattan plot of genes encoding the index proteins [red] and their interactors [purple] in genome-wide significant loci in PGC schizophrenia GWAS [phase 3]. Size of the interactor nodes and their labels scale with the number of index genes linked to each interactor; those that were also prioritized by FINEMAP or SMR analysis are highlighted in magenta. Gray lines indicate observed protein-protein interactions in our data; interactions that have been replicated by IP-WB are highlighted in blue.

(B) Cumulative density of gnomAD pLI scores for different gene sets. ‘Genome’ indicates genes in the pLI dataset [excluding PGC3 genes]; ‘PGC3’ indicates genes in PGC GWAS [phase 3] loci; ‘FINEMAP’, ‘SMR’, and ‘Network’ indicate PGC3 genes prioritized by FINEMAP, SMR, or our interactome data, respectively; ‘Overlap’ indicates genes overlapping between Network and FINEMAP or SMR; ‘SCHEMA 5% FDR’ and ‘SCHEMA EWS’ indicate genes with FDR < 0.05 or 3.7e-3 [exome-wide significance] in SCHEMA, respectively.

(C) Frontal cortex RNA expression of gene sets across ten developmental stages. Median expression and standard error [SE] of each gene set were derived from the BrainSpan exon microarray dataset. ‘Random’ indicates genes randomly sampled from the BrainSpan dataset; ‘PGC3’, ‘FINEMAP’, ‘SMR’, ‘Network’, and ‘Overlap’ indicate gene sets as described in (B); ‘SCHEMA’ indicates exome-wide significant genes from SCHEMA. Shaded regions indicate median expression of genes with FDR < 3.7e-3 [exome-wide significance], 0.05, 0.25, or 0.5 in SCHEMA with darker gray indicating greater significance.

(D) Western blot analysis on independent IPs of index proteins [named on the top] to detect the presence of selected locus proteins [named on the side of each gel]. Green and red circles at the bottom represented whether the tested interaction was significant or non-significant by IP-MS, respectively. Each lane represents 10% of the IP material analyzed by IP-MS. L = ladder. Molecular weights are in KDa. See also Figure S10 and Table S13.

To further compare our network prioritization approach to FINEMAP or SMR, we looked at the pLI score distributions and BrainSpan expression profiles of several gene sets, including all genes in the GWAS loci (PGC3), genes prioritized by our network (Network), FINEMAP, or SMR, and genes prioritized by both our network and either FINEMAP or SMR (Overlap). The Network and FINEMAP genes both have significantly higher pLI scores compared to all PGC3 genes (one-tailed KS test p = 7.9e-6 and 2.0e-4, respectively) and genes in the same loci as the prioritized genes (one-tailed KS test p = 1.4e-6 and 1.1e-4, respectively; Figure 4B and Table S2). In contrast, the SMR genes did not show such enrichment. The Overlap genes have even higher pLI scores that are comparable to the SCHEMA genes with FDR < 0.05 (two-tailed KS test p = 0.36), although the differences between the Overlap genes and other genes in the Network, FINEMAP, or SMR supersets are not statistically significant. When looking at gene expression throughout frontal cortical development, the Network, FINEMAP, SMR, and Overlap genes all have elevated expression compared to all PGC3 genes. In particular, the Network and Overlap genes have higher expression than the FINEMAP and SMR genes; their postnatal expression profiles are close to that of the high-confidence SCHEMA genes (Figure 4C and Table S3).

We also compared the Network genes to other PGC3 genes encoding iN-expressed proteins (i.e., non-interactors in Table S9) and observed that these two gene sets have comparable pLI scores and BrainSpan expression (Tables S2 and S3). These results indicate that the proteome of cultured neurons may contribute to the pLI score and expression enrichment we observed for the Network genes. However, by first showing that our PPI network is significantly enriched for schizophrenia genetic risk and transcriptional perturbations compared to the non-interactors, and then showing that the network genes in GWAS loci have higher pLI scores and frontal cortical expression throughout development compared to other genes in the loci, we demonstrate how our interactome data can complement orthogonal methods such as fine-mapping and eQTL co-localization analyses to nominate schizophrenia risk genes from GWAS data.

Finally, to illustrate that many of the locus proteins we prioritized could be reproducible interactors of the index proteins, we repeated the index protein IPs and performed western blotting to detect selected interactors; or vice versa, we executed reverse IPs for the interactors followed by western blotting to detect the index proteins. In total, we replicated 25 interactions for 18 unique locus proteins using forward or reverse IPs, including eight proteins that were also prioritized by FINEMAP or SMR, as well as an interaction between two of the index proteins, HCN1 and SYNGAP1 (Figures 4A, S3, and S5, and Table S7). We also performed western blots for several locus proteins on all six index protein IPs in parallel, showing that their detection patterns agree with their significant interactions with distinct index proteins in IP-MS (Figures 4D and S10). These results support the reproducibility of the PPI data and the observed convergence between the index protein interactomes. Follow-up investigation on the prioritized locus proteins can be informative for determining whether their corresponding genes are true schizophrenia risk genes responsible for the genetic signals observed in GWAS.

DiscussionWe brought together advances in genetics, neuronal cell modeling, interaction proteomics, and integrative analytical approaches to study cell-type-specific pathway relationships of schizophrenia risk genes in human excitatory neurons. Although previous studies have used PPI data to interpret genetic signals in complex disorders,18,19 few leveraged neuronal cell models to generate PPI networks in a human cellular context that are important for neuropsychiatric disorders. We showed that the iN-derived PPI networks consist of many newly reported, reproducible (∼90% replication rate by western blot) interactions that capture gene relationships found in the human brain, illustrating the potential for biological pathway discovery based on neuron-specific PPI data.We performed genetic enrichment analyses to validate the relevance of our PPI networks to schizophrenia and related disorders. Strikingly, the all combined network and several sub-networks are enriched for common variant risk of schizophrenia across different populations (Europeans and East Asians) and analytical methods (MAGMA and the GRS method), indicating that the PPI data will be a rich substrate for follow-up investigation of universal molecular mechanisms of schizophrenia. In contrast, we did not observe enrichment for rare variant schizophrenia risk in the networks. Repeating the analysis using larger rare variant datasets in the future may allow us to determine whether this is due to biological differences between common versus rare variants of schizophrenia or because of power limitations in the current data.9 When we tested for genetic risk enrichment of several other disorders, we found a few networks to be enriched for common or rare variant risks of ASD/DD. This is expected given our choice to prioritize schizophrenia index genes using orthogonal evidence for ASD/DD and agrees with the neurodevelopmental model of schizophrenia.11 However, the all combined PPI network did not show enrichment for ASD/DD or other disorders that have strong genetic correlations with schizophrenia (e.g., BIP). The lack of enrichment may be because of sample size and power differences between the genetic datasets. Alternatively, it may be because the schizophrenia index genes are not all relevant to the other disorders (e.g., among the index genes, only CACNA1C is in genome-wide significant loci of the BIP GWAS).We also demonstrated how our PPI data can prioritize candidate genes from schizophrenia GWAS data, which is a main bottleneck in post-GWAS analysis.43 Although statistical fine-mapping can pinpoint the likely causal variants within GWAS loci, it is difficult to generate actionable biological hypotheses based on fine-mapped SNPs.44 Hence, functional genomics and data types such as eQTLs have been used to complement fine-mapping by providing biological context and identifying SNP-associated genes. A lesson that emerged from these gene prioritization approaches is the importance of using tissue- and cell-type-specific data to dissect genetic signals in a disease-relevant context.45 However, there is still an underrepresentation of data derived from highly specialized cell types that are difficult to culture in the lab. Furthermore, recent analysis revealed a limited overlap between GWAS signals and eQTLs, indicating a need for orthogonal approaches that do not rely on transcriptomic data.46 Our study provides a brain cell-type-specific, protein-centric dataset that could address some of these limitations. We used a ‘guilt-by-association’ approach to prioritize GWAS genes that interact with the high-confidence schizophrenia index genes at the protein level, thereby linking them to interactions, mechanisms, and pathways that can be investigated in a neuronal context. It is exciting that, with a PPI network anchored by just six index genes, we already obtained promising gene prioritization results that are both complementary to and distinct from fine-mapping and eQTL co-localization, which were performed using genome-scale datasets. This suggests that cell-type-specific PPI data may become increasingly useful and powerful for gene prioritization with increase in scale (e.g., data from additional index genes) and scope (e.g., data from different disease-relevant cell types).In our analyses, the HCN1 network emerged as a promising lead for follow-up investigation. This network is enriched for schizophrenia common variant risk in both Europeans and East Asians, suggesting that perturbed signaling through the hyperpolarization-activated cyclic nucleotide-gated potassium channel,47 a heterotetrameric complex consisting of HCN1-4, could play a role in schizophrenia. Two HCN1 interactor genes, HCN4 and AKAP11, are also enriched for schizophrenia-associated protein-truncating variants (PTVs) in SCHEMA (FDR = 4.2e-3 and 1.3e-2, respectively). In a meta-analysis of schizophrenia and bipolar disorder cases,48AKAP11 further emerged as an exome-wide significant (p = 2.8e-9) gene enriched for ultra-rare PTVs. The roles of PTVs in disease are commonly linked to decreased gene function and expression. Indeed, the HCN1 network is suggestively enriched for down-regulated DEGs in layer 5/6 cortico-cortical projection neurons (‘Ex-L5-6CCb’) of individuals with schizophrenia (Figure S9), supporting the hypothesis that members of the network may be involved in schizophrenia through loss-of-function or decreased expression. Together, these findings implicate a network involved in neuronal potassium signaling in schizophrenia, which also contains drug targets that can be explored in follow-up studies (Figure S11 and Table S14).Another intriguing finding from our results is the recurrent interaction between CACNA1C and C4A (observed in four out of five CACNA1C IP-MS experiments; Table S6), suggesting that the L-type calcium channel may be a functional binding site of the complement cascade in synaptic pruning of the developing prefrontal cortex. This interaction is consistent with the emerging evidence that complement-mediated modulation of synapse stability or function contributes to risk for schizophrenia.49,50 We were unable to identify suitable immunoreagents to perform follow-up IP of C4A (Table S4), likely because of its complex post-translational modifications. However, we also identified C3, another component of the complement cascade, as an interactor of CACNA1C in one of the CACNA1C IPs (Table S6). This interaction could be a perhaps better vantage point into the functional characterization of the interplay between synaptic biology and complement cascade, because it is more stable and therefore amenable to biochemical studies.Limitations of the studyWe acknowledge several limitations that should be considered when interpreting the results of our study. First, all IPs were performed in a salt/detergent environment optimized to identify both stable interactions (e.g., between members of core protein complexes) and more transient interactions. Although both may be biologically relevant to schizophrenia, we are unable to discriminate between them with our current IP-MS approach. Second, MS analyses of the IPs were associated with known technical biases, including incomplete coverage and underrepresentation of lowly expressed or highly hydrophobic proteins.51,52 Therefore, the IP-MS datasets generated in this study do not represent a saturated interactome for each index protein in iNs. Third, although independent IPs and reverse IPs followed by western blotting estimated a ∼90% replication rate in our PPI data, these assays shared some of the biases of IP-MS. Technologies such as multichannel microscopy,53 super-resolution microscopy,54 and cryo-EM55 could provide orthogonal validation of the identified interactions.Finally, our PPI networks were generated from IPs of six index proteins in excitatory neurons, thus they do not represent a complete interactome of all schizophrenia risk genes across different brain cell types. As large-scale rare variant association studies of schizophrenia (e.g., SCHEMA) were not yet available at the conception of our study, we chose to prioritize the common variant schizophrenia index genes using orthogonal rare variant evidence of ASD/DD. Consequently, our findings may be biased toward shared biology between schizophrenia and neurodevelopmental conditions, instead of biology involved more exclusively in schizophrenia. Furthermore, although our neuronal cell model mimics cortical excitatory neurons in the human brain, it may not capture molecular interactions that are influenced by interplay with other cell types (e.g., astrocytes) or only present in later developmental stages (e.g., adolescence). Despite these caveats, our genetic enrichment analyses indicate that the generated PPI networks are indeed capturing schizophrenia-relevant biology. In the future, applying our approach to a wider set of index genes and cell types implicated across the spectrum of schizophrenia and related neurodevelopmental conditions may allow us to further dissect the convergent versus divergent pathways underlying these conditions in the human brain.ConclusionsIn this study, we leveraged human iPSC-derived excitatory neurons to build PPI networks for six schizophrenia risk genes and integrated the networks with orthogonal data types. Going forward, with larger genetic datasets to identify disease risk genes, wider availability of IP-competent immunoreagents, and the ability to create stem cell models of other brain cell types at the scale required for systematic proteomic experiments, we expect that the approach described here can be applied to uncover additional insights into the biology of schizophrenia and provide rich orthogonal information that is not captured by other approaches such as GWAS, exome sequencing, single-cell RNA sequencing, and whole-proteome analyses. More generally, our study presents an organizing framework to study complex brain disorders and contributes to laying the foundation for a new functional genomic approach in psychiatry. In fact, we have applied the same framework to study ASD-associated genes prioritized by exome sequencing,56 showing that it can empower interpretation of data from both common and rare variant genetics across two different groups of disorders of the human brain.

We brought together advances in genetics, neuronal cell modeling, interaction proteomics, and integrative analytical approaches to study cell-type-specific pathway relationships of schizophrenia risk genes in human excitatory neurons. Although previous studies have used PPI data to interpret genetic signals in complex disorders,18,19 few leveraged neuronal cell models to generate PPI networks in a human cellular context that are important for neuropsychiatric disorders. We showed that the iN-derived PPI networks consist of many newly reported, reproducible (∼90% replication rate by western blot) interactions that capture gene relationships found in the human brain, illustrating the potential for biological pathway discovery based on neuron-specific PPI data.

We performed genetic enrichment analyses to validate the relevance of our PPI networks to schizophrenia and related disorders. Strikingly, the all combined network and several sub-networks are enriched for common variant risk of schizophrenia across different populations (Europeans and East Asians) and analytical methods (MAGMA and the GRS method), indicating that the PPI data will be a rich substrate for follow-up investigation of universal molecular mechanisms of schizophrenia. In contrast, we did not observe enrichment for rare variant schizophrenia risk in the networks. Repeating the analysis using larger rare variant datasets in the future may allow us to determine whether this is due to biological differences between common versus rare variants of schizophrenia or because of power limitations in the current data.9 When we tested for genetic risk enrichment of several other disorders, we found a few networks to be enriched for common or rare variant risks of ASD/DD. This is expected given our choice to prioritize schizophrenia index genes using orthogonal evidence for ASD/DD and agrees with the neurodevelopmental model of schizophrenia.11 However, the all combined PPI network did not show enrichment for ASD/DD or other disorders that have strong genetic correlations with schizophrenia (e.g., BIP). The lack of enrichment may be because of sample size and power differences between the genetic datasets. Alternatively, it may be because the schizophrenia index genes are not all relevant to the other disorders (e.g., among the index genes, only CACNA1C is in genome-wide significant loci of the BIP GWAS).

We also demonstrated how our PPI data can prioritize candidate genes from schizophrenia GWAS data, which is a main bottleneck in post-GWAS analysis.43 Although statistical fine-mapping can pinpoint the likely causal variants within GWAS loci, it is difficult to generate actionable biological hypotheses based on fine-mapped SNPs.44 Hence, functional genomics and data types such as eQTLs have been used to complement fine-mapping by providing biological context and identifying SNP-associated genes. A lesson that emerged from these gene prioritization approaches is the importance of using tissue- and cell-type-specific data to dissect genetic signals in a disease-relevant context.45 However, there is still an underrepresentation of data derived from highly specialized cell types that are difficult to culture in the lab. Furthermore, recent analysis revealed a limited overlap between GWAS signals and eQTLs, indicating a need for orthogonal approaches that do not rely on transcriptomic data.46 Our study provides a brain cell-type-specific, protein-centric dataset that could address some of these limitations. We used a ‘guilt-by-association’ approach to prioritize GWAS genes that interact with the high-confidence schizophrenia index genes at the protein level, thereby linking them to interactions, mechanisms, and pathways that can be investigated in a neuronal context. It is exciting that, with a PPI network anchored by just six index genes, we already obtained promising gene prioritization results that are both complementary to and distinct from fine-mapping and eQTL co-localization, which were performed using genome-scale datasets. This suggests that cell-type-specific PPI data may become increasingly useful and powerful for gene prioritization with increase in scale (e.g., data from additional index genes) and scope (e.g., data from different disease-relevant cell types).

In our analyses, the HCN1 network emerged as a promising lead for follow-up investigation. This network is enriched for schizophrenia common variant risk in both Europeans and East Asians, suggesting that perturbed signaling through the hyperpolarization-activated cyclic nucleotide-gated potassium channel,47 a heterotetrameric complex consisting of HCN1-4, could play a role in schizophrenia. Two HCN1 interactor genes, HCN4 and AKAP11, are also enriched for schizophrenia-associated protein-truncating variants (PTVs) in SCHEMA (FDR = 4.2e-3 and 1.3e-2, respectively). In a meta-analysis of schizophrenia and bipolar disorder cases,48AKAP11 further emerged as an exome-wide significant (p = 2.8e-9) gene enriched for ultra-rare PTVs. The roles of PTVs in disease are commonly linked to decreased gene function and expression. Indeed, the HCN1 network is suggestively enriched for down-regulated DEGs in layer 5/6 cortico-cortical projection neurons (‘Ex-L5-6CCb’) of individuals with schizophrenia (Figure S9), supporting the hypothesis that members of the network may be involved in schizophrenia through loss-of-function or decreased expression. Together, these findings implicate a network involved in neuronal potassium signaling in schizophrenia, which also contains drug targets that can be explored in follow-up studies (Figure S11 and Table S14).

Another intriguing finding from our results is the recurrent interaction between CACNA1C and C4A (observed in four out of five CACNA1C IP-MS experiments; Table S6), suggesting that the L-type calcium channel may be a functional binding site of the complement cascade in synaptic pruning of the developing prefrontal cortex. This interaction is consistent with the emerging evidence that complement-mediated modulation of synapse stability or function contributes to risk for schizophrenia.49,50 We were unable to identify suitable immunoreagents to perform follow-up IP of C4A (Table S4), likely because of its complex post-translational modifications. However, we also identified C3, another component of the complement cascade, as an interactor of CACNA1C in one of the CACNA1C IPs (Table S6). This interaction could be a perhaps better vantage point into the functional characterization of the interplay between synaptic biology and complement cascade, because it is more stable and therefore amenable to biochemical studies.

Limitations of the studyWe acknowledge several limitations that should be considered when interpreting the results of our study. First, all IPs were performed in a salt/detergent environment optimized to identify both stable interactions (e.g., between members of core protein complexes) and more transient interactions. Although both may be biologically relevant to schizophrenia, we are unable to discriminate between them with our current IP-MS approach. Second, MS analyses of the IPs were associated with known technical biases, including incomplete coverage and underrepresentation of lowly expressed or highly hydrophobic proteins.51,52 Therefore, the IP-MS datasets generated in this study do not represent a saturated interactome for each index protein in iNs. Third, although independent IPs and reverse IPs followed by western blotting estimated a ∼90% replication rate in our PPI data, these assays shared some of the biases of IP-MS. Technologies such as multichannel microscopy,53 super-resolution microscopy,54 and cryo-EM55 could provide orthogonal validation of the identified interactions.Finally, our PPI networks were generated from IPs of six index proteins in excitatory neurons, thus they do not represent a complete interactome of all schizophrenia risk genes across different brain cell types. As large-scale rare variant association studies of schizophrenia (e.g., SCHEMA) were not yet available at the conception of our study, we chose to prioritize the common variant schizophrenia index genes using orthogonal rare variant evidence of ASD/DD. Consequently, our findings may be biased toward shared biology between schizophrenia and neurodevelopmental conditions, instead of biology involved more exclusively in schizophrenia. Furthermore, although our neuronal cell model mimics cortical excitatory neurons in the human brain, it may not capture molecular interactions that are influenced by interplay with other cell types (e.g., astrocytes) or only present in later developmental stages (e.g., adolescence). Despite these caveats, our genetic enrichment analyses indicate that the generated PPI networks are indeed capturing schizophrenia-relevant biology. In the future, applying our approach to a wider set of index genes and cell types implicated across the spectrum of schizophrenia and related neurodevelopmental conditions may allow us to further dissect the convergent versus divergent pathways underlying these conditions in the human brain.

Limitations of the study

We acknowledge several limitations that should be considered when interpreting the results of our study. First, all IPs were performed in a salt/detergent environment optimized to identify both stable interactions (e.g., between members of core protein complexes) and more transient interactions. Although both may be biologically relevant to schizophrenia, we are unable to discriminate between them with our current IP-MS approach. Second, MS analyses of the IPs were associated with known technical biases, including incomplete coverage and underrepresentation of lowly expressed or highly hydrophobic proteins.51,52 Therefore, the IP-MS datasets generated in this study do not represent a saturated interactome for each index protein in iNs. Third, although independent IPs and reverse IPs followed by western blotting estimated a ∼90% replication rate in our PPI data, these assays shared some of the biases of IP-MS. Technologies such as multichannel microscopy,53 super-resolution microscopy,54 and cryo-EM55 could provide orthogonal validation of the identified interactions.

Finally, our PPI networks were generated from IPs of six index proteins in excitatory neurons, thus they do not represent a complete interactome of all schizophrenia risk genes across different brain cell types. As large-scale rare variant association studies of schizophrenia (e.g., SCHEMA) were not yet available at the conception of our study, we chose to prioritize the common variant schizophrenia index genes using orthogonal rare variant evidence of ASD/DD. Consequently, our findings may be biased toward shared biology between schizophrenia and neurodevelopmental conditions, instead of biology involved more exclusively in schizophrenia. Furthermore, although our neuronal cell model mimics cortical excitatory neurons in the human brain, it may not capture molecular interactions that are influenced by interplay with other cell types (e.g., astrocytes) or only present in later developmental stages (e.g., adolescence). Despite these caveats, our genetic enrichment analyses indicate that the generated PPI networks are indeed capturing schizophrenia-relevant biology. In the future, applying our approach to a wider set of index genes and cell types implicated across the spectrum of schizophrenia and related neurodevelopmental conditions may allow us to further dissect the convergent versus divergent pathways underlying these conditions in the human brain.

ConclusionsIn this study, we leveraged human iPSC-derived excitatory neurons to build PPI networks for six schizophrenia risk genes and integrated the networks with orthogonal data types. Going forward, with larger genetic datasets to identify disease risk genes, wider availability of IP-competent immunoreagents, and the ability to create stem cell models of other brain cell types at the scale required for systematic proteomic experiments, we expect that the approach described here can be applied to uncover additional insights into the biology of schizophrenia and provide rich orthogonal information that is not captured by other approaches such as GWAS, exome sequencing, single-cell RNA sequencing, and whole-proteome analyses. More generally, our study presents an organizing framework to study complex brain disorders and contributes to laying the foundation for a new functional genomic approach in psychiatry. In fact, we have applied the same framework to study ASD-associated genes prioritized by exome sequencing,56 showing that it can empower interpretation of data from both common and rare variant genetics across two different groups of disorders of the human brain.

Conclusions

In this study, we leveraged human iPSC-derived excitatory neurons to build PPI networks for six schizophrenia risk genes and integrated the networks with orthogonal data types. Going forward, with larger genetic datasets to identify disease risk genes, wider availability of IP-competent immunoreagents, and the ability to create stem cell models of other brain cell types at the scale required for systematic proteomic experiments, we expect that the approach described here can be applied to uncover additional insights into the biology of schizophrenia and provide rich orthogonal information that is not captured by other approaches such as GWAS, exome sequencing, single-cell RNA sequencing, and whole-proteome analyses. More generally, our study presents an organizing framework to study complex brain disorders and contributes to laying the foundation for a new functional genomic approach in psychiatry. In fact, we have applied the same framework to study ASD-associated genes prioritized by exome sequencing,56 showing that it can empower interpretation of data from both common and rare variant genetics across two different groups of disorders of the human brain.

STAR★MethodsKey resources table
REAGENT or RESOURCESOURCEIDENTIFIERAntibodiesACTBAbcamCat# ab8227; RRID: AB_2305186AKAP11InvitrogenCat# PA5-103369;RRID: AB_2852713APLP2Proteintech GroupCat# 15041-1-AP;RRID: AB_2289597ARID3BFortis Life SciencesCat# A302-565A;RRID: AB_2034823C3AbcamCat# ab97462;RRID: AB_10679468CACNA1CAlomone LabsCat# ACC-003;RRID: AB_2039771CACNB2AbnovaCat# H00000783-A01;RRID: AB_463642CACNB3Alomone LabsCat# ACC-008:RRID: AB_2039787CAMK2DAbcamCat# ab181052;RRID: AB_2891241CHD4AbcamCat# ab70469;RRID: AB_2229454CKBSanta Cruz BiotechnologyCat# sc-374072;RRID: AB_10947239CNTNAP1AbcamCat# ab34151;RRID: AB_869934COPS2Proteintech GroupCat# 10969-2-AP;RRID: AB_2276346COPS5Cell Signaling TechnologyCat# 6895;RRID: AB_10839271CSMD1AbcamCat# ab166908CTNND1Santa Cruz BiotechnologyCat# sc-23873;RRID: AB_2086394CUL3Fortis Life SciencesCat# A301-109A;RRID: AB_873023ELAVL2Proteintech GroupCat# 14008-1-AP;RRID: AB_2096356ELAVL3ABclonalCat# A6091;RRID: AB_2766743EPN2Fortis Life SciencesCat# A305-510A;RRID: AB_2891436FMR1 (FMRP)AbcamCat# ab17722;RRID: AB_2278530FXR1Proteintech GroupCat# 13194-1-AP;RRID: AB_2110702GABBR2 (GABAB2)AbcamCat# ab75838;RRID: AB_1310245GNAO1AbcamCat# ab154001GRIN2AAbcamCat# ab133265;RRID: AB_11158532HCN1Proteintech GroupCat# 55222-1-AP;RRID: AB_11182929HCN4AbcamCat# ab32675;RRID: AB_732770HNRNPA2B1AbcamCat# ab6102;RRID: AB_305293IGF2BP1Proteintech GroupCat# 22803-1-AP;RRID: AB_2879173IGF2BP2AbcamCat# ab124930;RRID: AB_11131218IGF2BP3Fortis Life SciencesCat# A303-426A;RRID: AB_10951696INAFortis Life SciencesCat# A305-441A;RRID: AB_2631832ISL1AbcamCat# ab109517;RRID: AB_10866454KIAA1549Fortis Life SciencesCat# A305-877A;RRID: AB_2891771LDB1Novus BiologicalsCat# NBP1-77832;RRID: AB_11038158MARK3LSBioCat# LS-B5226;RRID: AB_10851098MEF2CAtlas AntibodiesCat# AMAb90727;RRID: AB_2665645MSI2AbcamCat# ab76148;RRID: AB_1523981NEGR1Proteintech GroupCat# 13674-1-AP;RRID: AB_2877969NSG1Millipore SigmaCat# SAB4501190;RRID: AB_10745270PARP1Proteintech GroupCat# 13371-1-AP;RRID: AB_2160459PCDHA2Proteintech GroupCat# 10127-2-AP;RRID: AB_2158964PCDHB5Proteintech GroupCat# 19609-1-AP;RRID: AB_10792248PCM1Fortis Life SciencesCat# A301-150A;RRID: AB_873100PDE4BCell Signaling TechnologyCat# 72096S;RRID: AB_2799812PYGBProteintech GroupCat# 12075-1-AP;RRID: AB_2174885RALYAbcamCat# ab170105RIMS1Proteintech GroupCat# 24576-1-AP;RRID: AB_2879618RIMS1Santa Cruz BiotechnologyCat# sc-368540RUNX1T1AbcamCat# ab124269;RRID: AB_10976059SEMA4CSanta Cruz BiotechnologyCat# sc-136445;RRID: AB_10837020SF3B1Cell Signaling TechnologyCat# 14434S;RRID: AB_2798479SRPK2Fortis Life SciencesCat# A302-467A;RRID: AB_1944286SYNGAP1Cell Signaling TechnologyCat# 5539;RRID: AB_10694401TCF4GeneTexCat# GTX17912;RRID: AB_2887569XRCC5AbcamCat# ab80592;RRID: AB_1603758XRCC6AbcamCat# ab92450;RRID: AB_10562280V5 tagMBLCat# M167-11Mouse IgG1 monoclonal isotype controlCell Signaling TechnologyCat# 5415;RRID: AB_10829607Mouse IgG2a monoclonal isotype controlCell Signaling TechnologyCat# 61656;RRID: AB_2799613Rabbit IgG monoclonal isotype controlCell Signaling TechnologyCat# 3900;RRID: AB_1550038Rabbit IgG polyclonal isotype controlSigma-AldrichCat# I5006;RRID: AB_1163659Chemicals, peptides, and recombinant proteinsAccutaseGibcoCat# A11105Agilent RNA 6000 Pico kitAgilent TechnologiesCat# 5067-1513Ambion DNA-free DNase Treatment kitLife TechnologiesCat# AM1906B-27 supplementThermo ScientificCat# 17504044BDNFR&D SystemsCat# 248-BD/CF2100 Bioanalyzer InstrumentAgilent TechnologiesCat# G2939BABlotting-Grade BlockerBio-Rad LaboratoriesCat# 1706404CNTFR&D SystemsCat# 257-NTCryoStor CS10Stem Cell TechnologiesCat# 07930DMEM/F - 12Life TechnologiesCat# 11320033Doxycycline hyclateSigma-AldrichCat# 24390-14-5Fetal Bovine SerumR&D SystemsCat# S11195GDNFR&D SystemsCat# 212-GDGeltrexLife TechnologiesCat# A1413301GeneticinGibcoCat# 10131027D-(+)-GlucoseSigma-AldrichCat# 50-99-7GlutamaxLife TechnologiesCat# 35050079Gridded Plate 150 x 25 mmVWR InternationalCat# 25383-103Halt™ Protease and Phosphatase InhibitorThermo ScientificCat# 78442Laminin, MouseVWR InternationalCat# 47743-734LDN-193189Stem Cell TechnologiesCat# 1435934-00-1MEM NEAALife TechnologiesCat# 11140076Methanol ≥99.8%Fisher ScientificCat# 67-56-1N-2 SupplementLife TechnologiesCat# 17502048NeurobasalLife TechnologiesCat# 21103049NormocinInvitrogenCat# ant-nr-2NuPAGE™ 12%, Bis-Tris GelInvitrogenCat# NP0341PK2NuPAGE Transfer Buffer (20X)Life TechnologiesCat# NP00061PBSLife TechnologiesCat# 10010049PVDF/Filter Paper SandwichLife TechnologiesCat# LC2002ROCK Inhibitor (Y-27632)StemgentCat# 04-0012SB 431542TocrisCat# 1614StemFlex™ MediumLife TechnologiesCat# A3349401Microcentrifuge TubesVWR InternationalCat# 0011-702Sterile SleevesVWR InternationalCat# 414004-510TrizolLife TechnologiesCat# 15596026TruSeq Stranded Total RNA Library Prep KitIlluminaCat# 20020596TrypLELife TechnologiesCat# 12604039TrypsinPromegaCat# V5111XAV939StemgentCat# 04-00046Critical commercial assaysPierce™ Protein A/G Magnetic BeadsThermo ScientificCat# 88803SuperSignal™ West Femto Maximum Sensitivity SubstrateThermo ScientificCat# 34094Pierce™ IP Lysis BufferThermo ScientificCat# 87788EnGen Spy Cas9 NLSNew England BiolabsCat# M0646TDeposited dataIP-MS dataThis studyMassIVE: MSV000087514Experimental Models: Cell linesiPS line from human fibroblasts with integrated NGN2Novartis Institutes for BioMedical ResearchiPS hDFn 83/22 iNgn2#9; iPS3 in Nehme et al.21Human embryonic stem cells WA01WiCellWA01 H1; NIH registration no. 0043Software and algorithmsRicopiliLam et al.57https://data.broadinstitute.org/mpg/ricopili/Spectrum Mill (v7.0)Broad Proteomics Platformhttps://proteomics.broadinstitute.orgGenoppi (v1.0)Pintacuda et al.24https://github.com/lagelab/Genoppipropr (v4.2.6)Quinn et al.58https://github.com/tpq/proprMAGMA (v1.09b)deLeeuw et al.32https://ctg.cncr.nl/software/magmaPLINK (v1.9)Chang et al.59https://www.cog-genomics.org/plink/Custom codeThis studyhttps://github.com/lagelab/SCZ_PPI; https://doi.org/10.5281/zenodo.7814403
Resource availabilityLead contactFurther information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Kasper Lage (lage.kasper@mgh.harvard.edu).Materials availabilityThis study did not generate new unique reagents.Experimental model and subject detailsCell linesiNsiNs collected at time points under one week of differentiation were dissociated from plates with Accutase (Innovative Cell Technologies AT104-500) at 37°C for 5-10 min. iNs collected at time points beyond two weeks of differentiation were collected by removing media, adding 1x PBS, and swirling the plates until the neurons detached. Dissociated or detached cells were collected in 15 mL or 50 mL Falcon tubes depending on the number of cells and washed 3 times. Each wash consisted of centrifugation at 300 RCF for 3-5 min, discarding of the supernatant, and resuspension of the cell pellet in 1x PBS. After the final wash, the sample was centrifuged at 300 RCF for 3-5 min, supernatant was discarded, and cells were either lysed or flash frozen in liquid nitrogen and stored at -80°C. Lysis was performed according to manufacturer’s protocol using Pierce IP lysis buffer (Thermo Scientific #87788) with 1x Halt Protease and Phosphatase cocktail inhibitor (Thermo Scientific #7844). Protein concentration of the lysate was quantified using BCA Protein assay (Thermo Scientific #23227). When not used for IP-MS experiments, the lysate was aliquoted in low-bind microfuge tubes (Axygen Scientific #MCT-175-L-C), flash frozen and stored at -80°C.Cancer and HEK cellsWe used the following cell lines as controls in western blots: HEK-293 (ATCC CRL-1573), a human embryonic kidney cell line; TF-1 (ATCC CRL-2003), a human erythroleukemia cell line; K-562 (ATCC CCL-243), a human myelogenous leukemia cell line; and U-937 (ATCC CRL-1593.2), a histiocytic lymphoma cell line. All cell lines were grown on uncoated plates (Corning) according to vendor recommendations. Media was changed every 3 days and cells were passaged when reaching 70% confluency. All cell lines were incubated at 37 °C, 5% CO2. TrypLE (Thermo Scientific) was used to detach cells from plates.Mouse cortex sampleMouse cortices were isolated from p0 pups of C57BL/6 background, cut into small pieces, and flash frozen. Lysis was performed by adding Pierce IP lysis buffer to frozen pieces and immediately applying a handheld homogenizer (VWR pellet mixer #47747-370). All subsequent steps were the same as those for iNs.Method detailsiN differentiationGlutamatergic patterned induced neurons (iNs) were differentiated from male stem cells by conditional expression of the neuralizing transcription factor NGN2 as previously described,21 with the exception that N2 media was used instead of KSR media during days 0 to 3. iNs were re-passaged at day 3 of differentiation (i.e., 40,000 cells/cm2) on Geltrex (Thermo Scientific #A1413202) coated plates. To remove remaining proliferating cells, fluoro-deoxyuridine (Bioworld 40690016-2) was added to cell cultures at 10 μM on day 6 of differentiation. iNs used in a subset of the CACNA1C, HCN1, and TCF4 IP-MS experiments (Table S5) were differentiated from human embryonic stem cells WA01 (H1, NIH registration no. 0043) transduced with lentiviruses carrying TetO::Ngn2-Puro and reverse tetracycline-controlled transactivator (rtTA), custom-packaged by Alstem. All other IP-MS experiments used iNs generated from a clonally selected induced PSC line (iPS hDFn 83/22 iNgn2#9 [iPS3]) with TetO::Ngn2 and rtTA delivered via PiggyBAC.21Western blottingFor western blots, frozen lysates were thawed on ice, diluted to desired concentration in 1x PBS, and brought to 1x LDS using 4x LDS stock. Samples were denatured at various conditions based on what we identified to work best for each protein of interest. Prepared samples were run on NuPAGE 1.5mm 3-7% Tris-Acetate or 1mm 4-12% Bis-Tris gels (Thermo Scientific). Transfer was onto a nitrocellulose membrane using XCell wet transfer (Invitrogen), iBlot2 (Thermo Scientific), or Trans-Blot Turbo (BioRad). Membranes were blocked with 5% BSA or 5% milk diluted in TBST, incubated with primary antibody diluted in matching blocking buffer for at least 12 hat 4°C, and washed 3 x 10 min in TBST. Membranes were incubated with HRP-conjugated secondary antibody diluted in 5% milk for 1h at RT, washed 3 x 10 min in TBST, developed with enhanced chemiluminescent substrate (Thermo Scientific #34095), and imaged on ChemiDoc MP (BioRad). In ‘forward’ IP western validation, the immunoprecipitate of the index protein was loaded into the gel and incubated with the primary antibody of an interactor protein to determine if the interactor could be found in the immunoprecipitate; in ‘reverse’ IP western validation, the immunoprecipitate of an interactor was loaded into the gel and incubated with the primary antibody of the index protein to determine if the index protein could be found in the immunoprecipitate. Western blot antibodies used for each protein of interest are listed in Table S4.ImmunoprecipitationsFor immunoprecipitations (IPs) followed by western blots or Coomassie stain analysis, either fresh or previously frozen lysates were used. For IPs followed by mass spectrometry (MS) only fresh lysates were used. On day 1, the needed amounts of lysate and IP antibody were added to a 1.7 mL Axygen MaxyClear tube (MCT-175-L-C), then brought to a final volume of 1.3 mL with Pierce IP lysis buffer. Tubes were then rotated at 4°C overnight for 14-18 h. On day 2, the needed amount of beads per IP were apportioned to separate tubes, washed twice in cold lysis buffer, resuspended in 200 μL lysis buffer per tube, added to corresponding tubes, and rotated at 4°C for 2-4 h. After incubation, tubes were placed on ice, and beads were first washed once in 1mL cold lysis buffer and then twice in 1x cold PBS. To remove supernatants in between the steps, magnetic beads were placed on a magnetic rack on ice and agarose beads were spun at 2,000 x g for 3 min. Supernatants after overnight rotation and each wash step were collected for western blot analysis of IP quality. After the third wash, supernatant was removed, and beads were resuspended in 50 μL of PBS if the IP was to be processed by MS and 40 μL of PBS if the IP was to be processed by western blot or Coomassie stain. Samples were flash frozen in liquid nitrogen and stored at -80°C until further use. All IP experiments sent for MS consisted of 4 IP samples performed on cells from the same differentiation batch: 2 replicate experimental IPs using an antibody against the protein of interest (i.e., an index protein) and 2 replicate control IPs using a control IgG antibody. IP antibodies used for each protein of interest are listed in Table S4.Immunoprecipitations using V5 epitope tagged TCF4For immunoprecipitation of TCF4, an ORF of the TCF4 isoform A with a c-terminal V5-tag was acquired from the Broad Institute Genetic Perturbation Platform, and lentiviruses were generated from the construct by ALSTEM Inc. Lentiviruses were delivered to iN cultures at a multiplicity of infection (MOI) of 4 at day 3 of differentiation during re-passaging, and mock-transduced cells were used as controls. Cells were collected at day 6 of differentiation by washing 3 times with PBS, followed by lysis and scraping on the plate. Lysates were then processed and quantified as usual. For IP, the needed amount of lysate was added to a microfuge tube and brought to a final volume of 1.3 mL with Pierce IP lysis buffer, then incubated overnight with anti-v5 antibody directly coupled to magnetic beads (MBL International, M167-11). The remaining protocol is the same as used for the other IPs.Mass spectrometrySample preparationProteins were digested on beads using 90 μl of digestion buffer (2 M urea/50 mM Tris buffer with 1 mM DTT and 5 μg/mL Trypsin) for 1 hr, shaking at 1000 rpm. The suspension was then transferred to a new tube, and the beads were washed twice with 60 μL of wash buffer (2 M urea/50 mM Tris buffer). The wash buffer was added to the suspension with digestion. The digestion and wash process were repeated a second time pooling the suspensions with the suspensions from the first round. The pooled solution was reduced using 4 mM DTT for 30 minat 25°C shaking at 1000 rpm. The proteins were then alkylated using 10 mM iodoacetamide and incubating for 45 minat 25°C shaking at 1000 rpm and protected from light. Proteins were then digested with 0.5 μg of trypsin overnight at 25°C shaking at 700 rpm. The next day proteins were quenched using 40 μL of 10% formic acid and desalted using an Oasis Cartridge. Samples were vacuum dried and labeled with iTRAQ4 (Sciex Inc.) or TMT10 (Thermo Scientific) kits. Each iTRAQ 4-plex consisted of 2 replicate experimental IPs using an antibody against the protein of interest (i.e., an index protein), and 2 replicate control IPs using a control IgG antibody. The specific iTRAQ labels for each replicate pair are indicated in Table S5. For ‘RIMS1_wk4_2’, a TMT 10-plex experiment was conducted with antibodies from Santa Cruz Biotechnology (SC; 128C, 129N), Proteintech Group (PT; 126, 127N), and Synaptic Systems (SYSY; 129C, 130N). Two sets of replicate control IPs were included for PT (127C, 128N) and SC/SYSY (130C, 131). Only the results from the SC antibody met our quality control metrics and were reported.Liquid chromatography-tandem mass spectrometry (LC-MS/MS)Reconstituted peptides were separated on an online nanoflow EASY-nLC 1000 UHPLC system (Thermo Scientific) and analyzed on a benchtop Orbitrap Q Exactive Plus mass spectrometer (Thermo Scientific). The peptide samples were injected onto a capillary column (Picofrit with 10 μm tip opening/75 μm diameter, New Objective, PF360-75-10-N-5) packed in-house with 20 cm C18 silica material (1.9 μm ReproSil-Pur C18-AQ medium, Dr. Maisch GmbH, r119.aq). The UHPLC setup was connected with a custom-fit microadapting tee (360 μm, IDEX Health & Science, UH-753), and capillary columns were heated to 50 °C in column heater sleeves (Phoenix-ST) to reduce backpressure during UHPLC separation. Injected peptides were separated at a flow rate of 200 nL/min with a linear 150 min gradient from 94% solvent A (3% acetonitrile, 0.1% formic acid) to 35% solvent B (90% acetonitrile, 0.1% formic acid), followed by a linear 8 min gradient from 35% solvent B to 60% solvent B and a 3 min ramp to 90% B. The Q Exactive instrument was operated in the data-dependent mode acquiring HCD MS/MS scans (R=17,500 for iTRAQ4, or R=35,000 for TMT10) after each MS1 scan (R=70,000) on the 12 most abundant ions using an MS1 ion target of 3x106 ions and an MS2 target of 5x104 ions. The maximum ion time utilized for the MS/MS scans was 120 ms; the HCD-normalized collision energy was set to 28 for iTRAQ4 or 31 for TMT10; the dynamic exclusion time was set to 20s, and the peptide match and isotope exclusion functions were enabled.Quantification and statistical analysisRefining geneticdata to identify index genesThree-step procedure for selecting index genesStarting with 125 independent autosomal SNPs that reached genome-wide significance in the combined discovery-replication meta-analysis of the PGC schizophrenia GWAS3 (phase 2), Ricopili57 (https://data.broadinstitute.org/mpg/ricopili/) was used to map the 124 non-MHC (major histocompatibility complex) region SNPs to 445 genes (Set 1; Table S1) in linkage disequilibrium (LD) loci, whose boundaries were defined by SNPs in LD (r2> 0.6) with the index SNPs, ±50kb on either end. Next, we excluded SNPs in intergenic regions or in loci with multiple protein-coding genes, resulting in a list of 40 SNPs pointing to 37 unique protein-coding genes (Set 2) in single-gene loci. We further refined the 37 genes into a set of 10 genes (Set 3) based on strong orthogonal evidence supporting their involvement in psychiatric diseases (see below). We also included SYNGAP1 in the MHC region (which was excluded from the SNP-to-gene mapping process due to its LD complexity) in all 3 sets based on strong orthogonal evidence. In total, we identified 11 high-confidence schizophrenia risk genes and considered their encoded proteins as index proteins in the proteomic experiments.Orthogonal evidence for the index genesThe index genes were selected based on several types of orthogonal genetic or functional evidence, including: i] high-density genotyping experiments of individual genes (ZNF804A),60 ii] sequencing or copy number variant studies linking genes to autism spectrum disorders or neurodevelopmental delay (CACNA1C, CUL3, CSMD1, GRIN2A, SATB2 SYNGAP1, TCF4),39,61,62,63,64,65,66,67,68 iii] strong functional evidence supporting that the gene in question is causal (HCN1),69 and iv] multiple subunits from the same protein complex are encoded by genes in different single-gene loci linked to psychiatric diseases (CACNA1C, CACNB2, RIMS1).3,70,71 In addition, we annotated the 10 genes prioritized from single-gene GWAS loci using Hi-C chromatin interaction data from the developing human brain,72which may point to long-range regulation of genes outside of the loci in the context of schizophrenia; only three of these genes (CUL3, RIMS1, SATB2) lie in loci that exclusively interacted with long-range genes outside of the loci (Table S1).pLI score enrichment analysisWe extracted pLI scores from the gnomAD22 (v2.1.1) ‘pLoF Metrics by Gene TSV’ dataset. We performed one-tailed Kolmogorov-Smirnov (KS) tests to assess whether various gene sets are enriched for high pLI scores compared to other genes in the genome. The tested gene sets include: i] SCHEMA9 genes with FDR <3.7e-3 (exome-wide significance) and 0.5, ii] Sets 1-3 defined by our index gene selection steps, and iii] genes in PGC schizophrenia GWAS (phase 3) loci10 and subsets of these genes prioritized by FINEMAP/SMR analysis and/or our interaction data (see social manhattan plot). In addition, we performed one-tailed KS tests to compare the pLI scores of several gene sets that are subsets and supersets of each other, and two-tailed KS tests to compare a few disjoint sets. Table S2 summarizes all the comparisons performed. Finally, we also performed one-tailed KS tests to assess whether the stringent interactors in our interaction networks (Table S9) have enriched pLI scores compared to the non-interactors linked to each network.BrainSpan expression profilesWe obtained gene expression data in four distinct parts of the frontal cortex (dorsolateral prefrontal cortex [DFC], medial prefrontal cortex [MFC], ventrolateral prefrontal cortex [VFC], orbital frontal cortex [OFC]) across 10 developmental stages from the BrainSpan (https://www.brainspan.org) exon microarray dataset.23 At each developmental stage, we calculated the median and standard error of the expression values for various genes or gene sets including: i] SCHEMA genes with FDR <3.7e-3, 0.05, 0.25, and 0.5, ii] Sets 1-3 defined by our index gene selection steps, iii] index and interactor genes in our combined network of all IPs (Table S9), iv] genes in PGC schizophrenia GWAS (phase 3) loci and subsets of these genes prioritized by FINEMAP/SMR analysis and/or our interaction data (see social manhattan plot), and v] random genes sampled from the BrainSpan dataset for comparison against the other gene sets. Table S3 provides more detailed summary statistics for all gene sets.IP-MS data analysisSpectrum Mill analysisAll mass spectra were processed using Spectrum Mill (v7.0; https://proteomics.broadinstitute.org). For peptide identification, MS/MS spectra were searched against a sequence database for the UniProt human reference proteome, including isoforms, with a set of common laboratory contaminant proteins appended (2017: 65,068 entries, or 2014: 59,079 entries) as indicated in Table S5. Search parameters included: ESI-QEXACTIVE-HCD scoring parameters, trypsin enzyme specificity with a maximum of two missed cleavages, 40% minimum matched peak intensity, ±20 ppm precursor mass tolerance, ±20 ppm product mass tolerance. Carbamidomethylation of cysteines and iTRAQ4 or TMT10 full labeling of lysines and peptide n-termini were set as fixed modifications. Allowed variable modifications were oxidation of methionine (M), acetyl (ProtN-term), and deamidated (N), with a precursor MH+ shift range of -18 to 64 Da. Identities interpreted for individual spectra were automatically designated as valid by optimizing score and delta rank1-rank2 score thresholds separately for each precursor charge state in each LC-MS/MS while allowing a maximum target-decoy-based false discovery rate (FDR) of 1.0% at the spectrum level. Identified peptides were organized into protein groups and subgroups (isoforms and family members) with Spectrum Mill’s subgroup specific option enabled, so that peptides shared between subgroups are ignored for quantitation. TMT10 reporter ion intensities were corrected for isotopic impurities using Spectrum Mill’s afRICA correction method and correction factors obtained from the reagent manufacturer’s certificate of analysis. For quantitation at the peptide spectrum match level, log2 fold change (FC) reporter ion intensity ratios were calculated for each IP replicate. To obtain protein-level log2 FC values between index protein vs. control for each IP replicate, the median ratio was calculated from all peptide spectrum matches of subgroup specific peptides assigned to each protein subgroup.Genoppi analysisFor each IP-MS experiment, starting with the protein-level quantification report generated by Spectrum Mill, we applied median normalization to protein log2 FC values between index protein vs. control IPs for each replicate, and then performed downstream analyses using Genoppi24 (v1.0). Additional data processing was performed for two experiments pre-Genoppi: for ‘CACNA1C_wk7’ the log2 FC values for one replicate was right-shifted by a small value (0.005) to facilitate direct comparisons with all other volcano plots without changing the results; for ‘TCF4_wk1’ additional contaminant proteins found in the CRAPome73 database were removed. The Genoppi analyses include: i] calculating Pearson’s correlation of log2 FC values between IP replicates, ii] calculating average log2 FC, and corresponding P-value and FDR, for each protein across IP replicates using a two-tailed one-sample moderated t-test from the limma74 R package, iii] identifying statistically significant (log2 FC > 0 and FDR ≤0.1) index protein interactors (i.e., proteins with significantly higher abundance in the index protein IPs compared to the controls), iv] defining other non-significant (log2 FC ≤ 0 or FDR >0.1) proteins to be the ‘non-interactors’ (to serve as proxy for background proteome in enrichment analyses), and v] assessing overlap enrichment between the identified interactors and known interactors from InWeb25 (compared to the non-interactors) using a one-tailed hypergeometric test. We performed quality control of each IP-MS experiment using two criteria: i] the log2 FC correlation between replicates must be >0.5, and ii] the index protein itself must be significant (log2 FC > 0 and FDR ≤0.1). Experiments that failed to meet these criteria were excluded from further analysis; those that passed QC are summarized in Table S5, with their analysis results provided in Table S6.Co-expression analysisWe assessed pairwise co-expression between each index gene and all other protein-coding genes using data from four independent studies, including: i] Stickels et al.29: spatial transcriptomics in mouse neocortex (Puck_190921_19.digital_expression.txt.gz retrieved from: https://singlecell.broadinstitute.org/single_cell/study/SCP815/highly-sensitive-spatial-transcriptomics-at-near-cellular-resolution-with-slide-seqv2#study-download), ii] Maynard et al.28: spatial transcriptomics in human dorsolateral prefrontal cortex (count matrix retrieved from spatialLIBD R package: https://github.com/LieberInstitute/HumanPilot), iii] Velmeshev et al.30: single-cell RNA-seq in human cortex (rawMatrix.zip retrieved from: https://cells.ucsc.edu/?ds=autism), and iv] BrainSpan: bulk RNA-seq across human brain regions and developmental stages (“RNA-Seq Gencode v10 summarized to genes” dataset retrieved from: https://www.brainspan.org/static/download.html). We used two different methods for estimating co-expression to account for different statistical properties in these datasets. For the spatial transcriptomic datasets with very sparse gene expression matrices, we reasoned that the binary presence/absence of genes across physical locations would be the most informative, and therefore performed one-tailed Fisher’s exact tests to calculate the significance of co-occurrence for each gene pair across locations. For the single-cell RNA-seq and BrainSpan datasets, we calculated a proportionality metric, ⍴, for each gene pair using the propr R package58 (v4.2.6); this metric is analogous to conventional correlation measures but has been shown to be better at capturing functional associations between genes in RNA-seq data.75 After calculating either the Fisher’s exact P-values or the proportionality ⍴ values for all gene pairs involving each index gene, we then performed rank-based inverse normal transformation to convert the values into co-expression Z-scores (where a positive score indicates that a gene has higher than average co-expression with the index gene compared to the rest of the genome).We performed two-tailed Wilcoxon rank-sum tests to assess if the co-expression Z-scores between index genes and their interactors are significantly different from the scores between the index genes and other gene groups including: i] non-interactors detected in IP-MS, ii] known interactors from InWeb, and iii] all protein-coding genes.Consolidating IP-MS datasets into interaction networksWe consolidated IP-MS datasets for the same index proteins into index-protein-specific networks; datasets derived from the same time points into time point-specific networks; and all datasets into an all combined network. For each network, we defined ‘interactors’ as proteins that show up as significant interactors in ≥1 source IPs contributing to the network, and the matching ‘non-interactors’ as proteins that show up as non-interactors in ≥1 source IPs but never as interactors in these IPs. In addition, we defined ‘stringent interactors’ as interactors that never show up as non-interactors in the source IPs. Finally, we removed proteins whose HGNC gene symbols could not be mapped to Ensembl76 (GRCh37.p13) genomic positions and the index proteins for the source IPs from all 3 lists. The exclusion of index proteins ensured that enrichment signals in downstream analyses could not be driven by the index proteins themselves. Table S9 lists the interactors, non-interactors, and stringent interactors associated with each network.SynGO gene set analysisSynGO analysis of interactor genes in the all combined network was performed using the SynGO web browser31 (dataset version: 20210225; https://syngoportal.org). The SynGO ‘Biological Processes’ annotations and the ‘brain expressed’ background set were used in the analysis.Common variant enrichment analysis using MAGMATo perform common variant risk enrichment analysis for each interaction network using MAGMA32 (v1.09b), PGC GWAS summary statistics (https://www.med.unc.edu/pgc/download-results/) were obtained for schizophrenia (33,640 cases and 43,456 controls of EUR ancestry3; 22,778 cases and 35,362 controls of EAS ancestry8), ADHD (19,099 cases and 34,194 controls of EUR ancestry33), ASD (18,382 cases and 27,969 controls of EUR ancestry34), BIP (41,917 cases and 371,549 controls of EUR ancestry35), and MDD (170,756 cases and 329,443 controls of EUR ancestry36). GWAS summary statistics for height were obtained from the Neale Lab UK Biobank GWAS (round 2; 361,194 individuals of EUR ancestry37; https://www.nealelab.is/uk-biobank/) and Biobank Japan (159,095 individuals of EAS ancestry38; http://jenger.riken.jp/en/result). For each GWAS dataset, we first annotated variants to genes using the Human reference genome (GRCh37/hg19) with a flanking gene region of ±250kb. Variants on sex chromosomes, with minor allele frequency (MAF) ≤ 0.05 in the study, or within the MHC region (chr6:22.5M-33.5M) were excluded from analysis. Next, gene-based P-values were computed using the SNP-wise Mean model in MAGMA. LD was estimated from the 1000 Genomes Project77 phase 3 EUR or EAS panel to match the ancestry of each study. Competitive tests were used for gene-set comparison analysis between stringent interactors and non-interactors in each interaction network. Specifically, a linear regression model was built by MAGMA to test if genes within the interactor gene-set are more strongly associated with the phenotype of interest compared to the non-interactor gene-set. Inverse-variance weighted fixed-effect meta-analysis78 was used to combine results across EUR and EAS ancestries for schizophrenia and height, and a one-tailed P-value was calculated from the meta-analyzed Z-score.Common variant enrichment analysis using the GRS methodTo complement the MAGMA analysis, we used an alternative genetic risk score (GRS) method and individual-level genotypes from the schizophrenia GWAS study cohorts (24,764 cases and 30,655 controls of EUR ancestry; 8,960 cases and 8,284 controls of EAS ancestry) to assess genetic risk enrichment in the interaction networks. IRB approvals for accessing the individual-level data were obtained from the PGC and Stanley Global Asia Initiatives (Data S1-S3). For each study cohort, we removed variants in the MHC region and variants with MAF ≤0.05 or low imputation quality (INFO ≤0.8), and then mapped the remaining variants to the stringent interactors or non-interactors associated with each network with a flanking gene region of ±250kb. Next, we used PLINK59 (v1.9) and the in-sample LD to clump variants within the interactor or non-interactor gene-sets into independent association signals. The clumping was performed with a window of 250kb, R2> 0.2, and a P-value threshold of 1. The out-of-sample GRS for each individual and each gene-set was then calculated as ∑ log(OR) ∗G, in whichOR is the estimated odds ratio from the leave-one-out GWAS meta-analysis for the clumped index variants within the gene-set and G is the genotype dosage. To test if the stringent interactors in a network contribute more genetic risk compared to the non-interactors, we fit a linear regression model: GRSij
= Pi
+ Ij
+ Pi ∗ Ij
+ Covi, where i and j denote individual index and gene index, respectively, P is the case/control status, I is interactor/non-interactor status, P ∗I is an interaction term, and Cov includes 10 genetic principal components. We tested whether the P ∗I interaction term is significantly larger than zero, which captures whether the difference of GRS between cases and controls calculated in the interactor gene-set is significantly larger than the non-interactor gene-set. Inverse-variance weighted fixed-effect meta-analysis was used to combine the test statistics for the interaction term across study cohorts and ancestries, and a one-tailed P-value was calculated from the meta-analyzed Z-score. Finally, as a negative control, we performed an analogous GRS analysis for height; in this case, the GRS was calculated as ∑ log(beta) ∗G, where beta is the effect size estimate from the EUR or EAS height GWAS for clumped index variants.Rare variant enrichment analysisGene-based association statistics derived from exome sequencing data were obtained for schizophrenia (P-values in ‘P meta’ column from Table S5 of Singh et al.9), ASD (Q-values in ‘qval_dnccPTV’ column from Table S2 of Satterstrom et al.40), and DD (P-values in ‘denovoWEST_p_full’ column from Table S2 of Kaplanis et al.41). For each phenotype and each network, a one-tailed KS test was used to test whether the stringent interactors in the network have more significant association scores (i.e., smaller P-values or Q-values) compared to the non-interactors.Transcriptional perturbation enrichment analysisDifferentially expressed genes (DEGs) between schizophrenia patients vs. controls identified in 20 prefrontal cortex cell types were retrieved from Table S6 of Ruzicka et al.17 The up- and down-regulated DEGs were analyzed as a joint set in the primary analysis, and then as separate sets in the secondary follow-up analyses. One-tailed P-values were calculated using a hypergeometric distribution to assess the overlap enrichment between interactors in our networks and the cell-type-specific DEGs. For each hypergeometric test, the ‘population’ was defined as all stringent interactor or non-interactor genes associated with a network and ‘success in population’ was defined as the stringent interactors in the network. The ‘sample’ contained DEGs in a celltype that were found in the population and ‘success in sample’ was the overlap between the interactors and the DEGs.Social Manhattan plotGenes mapped to 287 genome-wide significant regions in the combined discovery-replication meta-analysis of the PGC schizophrenia GWAS (phase 3) were retrieved from Table S3 of the paper.10 We excluded non-coding genes and genes in the MHC region, and obtained the genomic positions of the remaining genes from Ensembl76 (GRCh37.p13). We additionally included SYNGAP1 by mapping it to the only MHC region SNP (rs140365013) included in the GWAS meta-analysis. Next, we intersected these GWAS genes with our PPI data to identify the subset of GWAS genes that are either an index gene or an interactor of an index gene. We generated a Manhattan plot for these genes using the GWAS P-values of their associated index SNPs and plotted links between the genes to indicate observed interactions in our 3. In addition, we retrieved the list of GWAS genes prioritized by FINEMAP or SMR analysis from Table S12 of the PGC paper and highlighted their overlap with genes prioritized by our data in the plot. For ease of visualization, GWAS P-values were capped at P = 1e-25 and genes in the PCDHA@ gene cluster were collapsed into one gene in the plot.HCN1 network drug target queryTo identify existing drugs targeting genes in the combined HCN1 network, we performed a batch query using the Open Targets Platform79 (https://platform.opentargets.org). For each resulting drug-target entry, the corresponding drug mechanism information was retrieved from ChEMBL80 (https://www.ebi.ac.uk/chembl/g/#browse/mechanisms_of_action).

STAR★Methods

Key resources table
REAGENT or RESOURCESOURCEIDENTIFIERAntibodiesACTBAbcamCat# ab8227; RRID: AB_2305186AKAP11InvitrogenCat# PA5-103369;RRID: AB_2852713APLP2Proteintech GroupCat# 15041-1-AP;RRID: AB_2289597ARID3BFortis Life SciencesCat# A302-565A;RRID: AB_2034823C3AbcamCat# ab97462;RRID: AB_10679468CACNA1CAlomone LabsCat# ACC-003;RRID: AB_2039771CACNB2AbnovaCat# H00000783-A01;RRID: AB_463642CACNB3Alomone LabsCat# ACC-008:RRID: AB_2039787CAMK2DAbcamCat# ab181052;RRID: AB_2891241CHD4AbcamCat# ab70469;RRID: AB_2229454CKBSanta Cruz BiotechnologyCat# sc-374072;RRID: AB_10947239CNTNAP1AbcamCat# ab34151;RRID: AB_869934COPS2Proteintech GroupCat# 10969-2-AP;RRID: AB_2276346COPS5Cell Signaling TechnologyCat# 6895;RRID: AB_10839271CSMD1AbcamCat# ab166908CTNND1Santa Cruz BiotechnologyCat# sc-23873;RRID: AB_2086394CUL3Fortis Life SciencesCat# A301-109A;RRID: AB_873023ELAVL2Proteintech GroupCat# 14008-1-AP;RRID: AB_2096356ELAVL3ABclonalCat# A6091;RRID: AB_2766743EPN2Fortis Life SciencesCat# A305-510A;RRID: AB_2891436FMR1 (FMRP)AbcamCat# ab17722;RRID: AB_2278530FXR1Proteintech GroupCat# 13194-1-AP;RRID: AB_2110702GABBR2 (GABAB2)AbcamCat# ab75838;RRID: AB_1310245GNAO1AbcamCat# ab154001GRIN2AAbcamCat# ab133265;RRID: AB_11158532HCN1Proteintech GroupCat# 55222-1-AP;RRID: AB_11182929HCN4AbcamCat# ab32675;RRID: AB_732770HNRNPA2B1AbcamCat# ab6102;RRID: AB_305293IGF2BP1Proteintech GroupCat# 22803-1-AP;RRID: AB_2879173IGF2BP2AbcamCat# ab124930;RRID: AB_11131218IGF2BP3Fortis Life SciencesCat# A303-426A;RRID: AB_10951696INAFortis Life SciencesCat# A305-441A;RRID: AB_2631832ISL1AbcamCat# ab109517;RRID: AB_10866454KIAA1549Fortis Life SciencesCat# A305-877A;RRID: AB_2891771LDB1Novus BiologicalsCat# NBP1-77832;RRID: AB_11038158MARK3LSBioCat# LS-B5226;RRID: AB_10851098MEF2CAtlas AntibodiesCat# AMAb90727;RRID: AB_2665645MSI2AbcamCat# ab76148;RRID: AB_1523981NEGR1Proteintech GroupCat# 13674-1-AP;RRID: AB_2877969NSG1Millipore SigmaCat# SAB4501190;RRID: AB_10745270PARP1Proteintech GroupCat# 13371-1-AP;RRID: AB_2160459PCDHA2Proteintech GroupCat# 10127-2-AP;RRID: AB_2158964PCDHB5Proteintech GroupCat# 19609-1-AP;RRID: AB_10792248PCM1Fortis Life SciencesCat# A301-150A;RRID: AB_873100PDE4BCell Signaling TechnologyCat# 72096S;RRID: AB_2799812PYGBProteintech GroupCat# 12075-1-AP;RRID: AB_2174885RALYAbcamCat# ab170105RIMS1Proteintech GroupCat# 24576-1-AP;RRID: AB_2879618RIMS1Santa Cruz BiotechnologyCat# sc-368540RUNX1T1AbcamCat# ab124269;RRID: AB_10976059SEMA4CSanta Cruz BiotechnologyCat# sc-136445;RRID: AB_10837020SF3B1Cell Signaling TechnologyCat# 14434S;RRID: AB_2798479SRPK2Fortis Life SciencesCat# A302-467A;RRID: AB_1944286SYNGAP1Cell Signaling TechnologyCat# 5539;RRID: AB_10694401TCF4GeneTexCat# GTX17912;RRID: AB_2887569XRCC5AbcamCat# ab80592;RRID: AB_1603758XRCC6AbcamCat# ab92450;RRID: AB_10562280V5 tagMBLCat# M167-11Mouse IgG1 monoclonal isotype controlCell Signaling TechnologyCat# 5415;RRID: AB_10829607Mouse IgG2a monoclonal isotype controlCell Signaling TechnologyCat# 61656;RRID: AB_2799613Rabbit IgG monoclonal isotype controlCell Signaling TechnologyCat# 3900;RRID: AB_1550038Rabbit IgG polyclonal isotype controlSigma-AldrichCat# I5006;RRID: AB_1163659Chemicals, peptides, and recombinant proteinsAccutaseGibcoCat# A11105Agilent RNA 6000 Pico kitAgilent TechnologiesCat# 5067-1513Ambion DNA-free DNase Treatment kitLife TechnologiesCat# AM1906B-27 supplementThermo ScientificCat# 17504044BDNFR&D SystemsCat# 248-BD/CF2100 Bioanalyzer InstrumentAgilent TechnologiesCat# G2939BABlotting-Grade BlockerBio-Rad LaboratoriesCat# 1706404CNTFR&D SystemsCat# 257-NTCryoStor CS10Stem Cell TechnologiesCat# 07930DMEM/F - 12Life TechnologiesCat# 11320033Doxycycline hyclateSigma-AldrichCat# 24390-14-5Fetal Bovine SerumR&D SystemsCat# S11195GDNFR&D SystemsCat# 212-GDGeltrexLife TechnologiesCat# A1413301GeneticinGibcoCat# 10131027D-(+)-GlucoseSigma-AldrichCat# 50-99-7GlutamaxLife TechnologiesCat# 35050079Gridded Plate 150 x 25 mmVWR InternationalCat# 25383-103Halt™ Protease and Phosphatase InhibitorThermo ScientificCat# 78442Laminin, MouseVWR InternationalCat# 47743-734LDN-193189Stem Cell TechnologiesCat# 1435934-00-1MEM NEAALife TechnologiesCat# 11140076Methanol ≥99.8%Fisher ScientificCat# 67-56-1N-2 SupplementLife TechnologiesCat# 17502048NeurobasalLife TechnologiesCat# 21103049NormocinInvitrogenCat# ant-nr-2NuPAGE™ 12%, Bis-Tris GelInvitrogenCat# NP0341PK2NuPAGE Transfer Buffer (20X)Life TechnologiesCat# NP00061PBSLife TechnologiesCat# 10010049PVDF/Filter Paper SandwichLife TechnologiesCat# LC2002ROCK Inhibitor (Y-27632)StemgentCat# 04-0012SB 431542TocrisCat# 1614StemFlex™ MediumLife TechnologiesCat# A3349401Microcentrifuge TubesVWR InternationalCat# 0011-702Sterile SleevesVWR InternationalCat# 414004-510TrizolLife TechnologiesCat# 15596026TruSeq Stranded Total RNA Library Prep KitIlluminaCat# 20020596TrypLELife TechnologiesCat# 12604039TrypsinPromegaCat# V5111XAV939StemgentCat# 04-00046Critical commercial assaysPierce™ Protein A/G Magnetic BeadsThermo ScientificCat# 88803SuperSignal™ West Femto Maximum Sensitivity SubstrateThermo ScientificCat# 34094Pierce™ IP Lysis BufferThermo ScientificCat# 87788EnGen Spy Cas9 NLSNew England BiolabsCat# M0646TDeposited dataIP-MS dataThis studyMassIVE: MSV000087514Experimental Models: Cell linesiPS line from human fibroblasts with integrated NGN2Novartis Institutes for BioMedical ResearchiPS hDFn 83/22 iNgn2#9; iPS3 in Nehme et al.21Human embryonic stem cells WA01WiCellWA01 H1; NIH registration no. 0043Software and algorithmsRicopiliLam et al.57https://data.broadinstitute.org/mpg/ricopili/Spectrum Mill (v7.0)Broad Proteomics Platformhttps://proteomics.broadinstitute.orgGenoppi (v1.0)Pintacuda et al.24https://github.com/lagelab/Genoppipropr (v4.2.6)Quinn et al.58https://github.com/tpq/proprMAGMA (v1.09b)deLeeuw et al.32https://ctg.cncr.nl/software/magmaPLINK (v1.9)Chang et al.59https://www.cog-genomics.org/plink/Custom codeThis studyhttps://github.com/lagelab/SCZ_PPI; https://doi.org/10.5281/zenodo.7814403

Key resources table

REAGENT or RESOURCESOURCEIDENTIFIERAntibodiesACTBAbcamCat# ab8227; RRID: AB_2305186AKAP11InvitrogenCat# PA5-103369;RRID: AB_2852713APLP2Proteintech GroupCat# 15041-1-AP;RRID: AB_2289597ARID3BFortis Life SciencesCat# A302-565A;RRID: AB_2034823C3AbcamCat# ab97462;RRID: AB_10679468CACNA1CAlomone LabsCat# ACC-003;RRID: AB_2039771CACNB2AbnovaCat# H00000783-A01;RRID: AB_463642CACNB3Alomone LabsCat# ACC-008:RRID: AB_2039787CAMK2DAbcamCat# ab181052;RRID: AB_2891241CHD4AbcamCat# ab70469;RRID: AB_2229454CKBSanta Cruz BiotechnologyCat# sc-374072;RRID: AB_10947239CNTNAP1AbcamCat# ab34151;RRID: AB_869934COPS2Proteintech GroupCat# 10969-2-AP;RRID: AB_2276346COPS5Cell Signaling TechnologyCat# 6895;RRID: AB_10839271CSMD1AbcamCat# ab166908CTNND1Santa Cruz BiotechnologyCat# sc-23873;RRID: AB_2086394CUL3Fortis Life SciencesCat# A301-109A;RRID: AB_873023ELAVL2Proteintech GroupCat# 14008-1-AP;RRID: AB_2096356ELAVL3ABclonalCat# A6091;RRID: AB_2766743EPN2Fortis Life SciencesCat# A305-510A;RRID: AB_2891436FMR1 (FMRP)AbcamCat# ab17722;RRID: AB_2278530FXR1Proteintech GroupCat# 13194-1-AP;RRID: AB_2110702GABBR2 (GABAB2)AbcamCat# ab75838;RRID: AB_1310245GNAO1AbcamCat# ab154001GRIN2AAbcamCat# ab133265;RRID: AB_11158532HCN1Proteintech GroupCat# 55222-1-AP;RRID: AB_11182929HCN4AbcamCat# ab32675;RRID: AB_732770HNRNPA2B1AbcamCat# ab6102;RRID: AB_305293IGF2BP1Proteintech GroupCat# 22803-1-AP;RRID: AB_2879173IGF2BP2AbcamCat# ab124930;RRID: AB_11131218IGF2BP3Fortis Life SciencesCat# A303-426A;RRID: AB_10951696INAFortis Life SciencesCat# A305-441A;RRID: AB_2631832ISL1AbcamCat# ab109517;RRID: AB_10866454KIAA1549Fortis Life SciencesCat# A305-877A;RRID: AB_2891771LDB1Novus BiologicalsCat# NBP1-77832;RRID: AB_11038158MARK3LSBioCat# LS-B5226;RRID: AB_10851098MEF2CAtlas AntibodiesCat# AMAb90727;RRID: AB_2665645MSI2AbcamCat# ab76148;RRID: AB_1523981NEGR1Proteintech GroupCat# 13674-1-AP;RRID: AB_2877969NSG1Millipore SigmaCat# SAB4501190;RRID: AB_10745270PARP1Proteintech GroupCat# 13371-1-AP;RRID: AB_2160459PCDHA2Proteintech GroupCat# 10127-2-AP;RRID: AB_2158964PCDHB5Proteintech GroupCat# 19609-1-AP;RRID: AB_10792248PCM1Fortis Life SciencesCat# A301-150A;RRID: AB_873100PDE4BCell Signaling TechnologyCat# 72096S;RRID: AB_2799812PYGBProteintech GroupCat# 12075-1-AP;RRID: AB_2174885RALYAbcamCat# ab170105RIMS1Proteintech GroupCat# 24576-1-AP;RRID: AB_2879618RIMS1Santa Cruz BiotechnologyCat# sc-368540RUNX1T1AbcamCat# ab124269;RRID: AB_10976059SEMA4CSanta Cruz BiotechnologyCat# sc-136445;RRID: AB_10837020SF3B1Cell Signaling TechnologyCat# 14434S;RRID: AB_2798479SRPK2Fortis Life SciencesCat# A302-467A;RRID: AB_1944286SYNGAP1Cell Signaling TechnologyCat# 5539;RRID: AB_10694401TCF4GeneTexCat# GTX17912;RRID: AB_2887569XRCC5AbcamCat# ab80592;RRID: AB_1603758XRCC6AbcamCat# ab92450;RRID: AB_10562280V5 tagMBLCat# M167-11Mouse IgG1 monoclonal isotype controlCell Signaling TechnologyCat# 5415;RRID: AB_10829607Mouse IgG2a monoclonal isotype controlCell Signaling TechnologyCat# 61656;RRID: AB_2799613Rabbit IgG monoclonal isotype controlCell Signaling TechnologyCat# 3900;RRID: AB_1550038Rabbit IgG polyclonal isotype controlSigma-AldrichCat# I5006;RRID: AB_1163659Chemicals, peptides, and recombinant proteinsAccutaseGibcoCat# A11105Agilent RNA 6000 Pico kitAgilent TechnologiesCat# 5067-1513Ambion DNA-free DNase Treatment kitLife TechnologiesCat# AM1906B-27 supplementThermo ScientificCat# 17504044BDNFR&D SystemsCat# 248-BD/CF2100 Bioanalyzer InstrumentAgilent TechnologiesCat# G2939BABlotting-Grade BlockerBio-Rad LaboratoriesCat# 1706404CNTFR&D SystemsCat# 257-NTCryoStor CS10Stem Cell TechnologiesCat# 07930DMEM/F - 12Life TechnologiesCat# 11320033Doxycycline hyclateSigma-AldrichCat# 24390-14-5Fetal Bovine SerumR&D SystemsCat# S11195GDNFR&D SystemsCat# 212-GDGeltrexLife TechnologiesCat# A1413301GeneticinGibcoCat# 10131027D-(+)-GlucoseSigma-AldrichCat# 50-99-7GlutamaxLife TechnologiesCat# 35050079Gridded Plate 150 x 25 mmVWR InternationalCat# 25383-103Halt™ Protease and Phosphatase InhibitorThermo ScientificCat# 78442Laminin, MouseVWR InternationalCat# 47743-734LDN-193189Stem Cell TechnologiesCat# 1435934-00-1MEM NEAALife TechnologiesCat# 11140076Methanol ≥99.8%Fisher ScientificCat# 67-56-1N-2 SupplementLife TechnologiesCat# 17502048NeurobasalLife TechnologiesCat# 21103049NormocinInvitrogenCat# ant-nr-2NuPAGE™ 12%, Bis-Tris GelInvitrogenCat# NP0341PK2NuPAGE Transfer Buffer (20X)Life TechnologiesCat# NP00061PBSLife TechnologiesCat# 10010049PVDF/Filter Paper SandwichLife TechnologiesCat# LC2002ROCK Inhibitor (Y-27632)StemgentCat# 04-0012SB 431542TocrisCat# 1614StemFlex™ MediumLife TechnologiesCat# A3349401Microcentrifuge TubesVWR InternationalCat# 0011-702Sterile SleevesVWR InternationalCat# 414004-510TrizolLife TechnologiesCat# 15596026TruSeq Stranded Total RNA Library Prep KitIlluminaCat# 20020596TrypLELife TechnologiesCat# 12604039TrypsinPromegaCat# V5111XAV939StemgentCat# 04-00046Critical commercial assaysPierce™ Protein A/G Magnetic BeadsThermo ScientificCat# 88803SuperSignal™ West Femto Maximum Sensitivity SubstrateThermo ScientificCat# 34094Pierce™ IP Lysis BufferThermo ScientificCat# 87788EnGen Spy Cas9 NLSNew England BiolabsCat# M0646TDeposited dataIP-MS dataThis studyMassIVE: MSV000087514Experimental Models: Cell linesiPS line from human fibroblasts with integrated NGN2Novartis Institutes for BioMedical ResearchiPS hDFn 83/22 iNgn2#9; iPS3 in Nehme et al.21Human embryonic stem cells WA01WiCellWA01 H1; NIH registration no. 0043Software and algorithmsRicopiliLam et al.57https://data.broadinstitute.org/mpg/ricopili/Spectrum Mill (v7.0)Broad Proteomics Platformhttps://proteomics.broadinstitute.orgGenoppi (v1.0)Pintacuda et al.24https://github.com/lagelab/Genoppipropr (v4.2.6)Quinn et al.58https://github.com/tpq/proprMAGMA (v1.09b)deLeeuw et al.32https://ctg.cncr.nl/software/magmaPLINK (v1.9)Chang et al.59https://www.cog-genomics.org/plink/Custom codeThis studyhttps://github.com/lagelab/SCZ_PPI; https://doi.org/10.5281/zenodo.7814403

Resource availabilityLead contactFurther information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Kasper Lage (lage.kasper@mgh.harvard.edu).Materials availabilityThis study did not generate new unique reagents.

Resource availability

Lead contactFurther information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Kasper Lage (lage.kasper@mgh.harvard.edu).

Lead contact

Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Kasper Lage (lage.kasper@mgh.harvard.edu).

Materials availabilityThis study did not generate new unique reagents.

Materials availability

This study did not generate new unique reagents.

Experimental model and subject detailsCell linesiNsiNs collected at time points under one week of differentiation were dissociated from plates with Accutase (Innovative Cell Technologies AT104-500) at 37°C for 5-10 min. iNs collected at time points beyond two weeks of differentiation were collected by removing media, adding 1x PBS, and swirling the plates until the neurons detached. Dissociated or detached cells were collected in 15 mL or 50 mL Falcon tubes depending on the number of cells and washed 3 times. Each wash consisted of centrifugation at 300 RCF for 3-5 min, discarding of the supernatant, and resuspension of the cell pellet in 1x PBS. After the final wash, the sample was centrifuged at 300 RCF for 3-5 min, supernatant was discarded, and cells were either lysed or flash frozen in liquid nitrogen and stored at -80°C. Lysis was performed according to manufacturer’s protocol using Pierce IP lysis buffer (Thermo Scientific #87788) with 1x Halt Protease and Phosphatase cocktail inhibitor (Thermo Scientific #7844). Protein concentration of the lysate was quantified using BCA Protein assay (Thermo Scientific #23227). When not used for IP-MS experiments, the lysate was aliquoted in low-bind microfuge tubes (Axygen Scientific #MCT-175-L-C), flash frozen and stored at -80°C.Cancer and HEK cellsWe used the following cell lines as controls in western blots: HEK-293 (ATCC CRL-1573), a human embryonic kidney cell line; TF-1 (ATCC CRL-2003), a human erythroleukemia cell line; K-562 (ATCC CCL-243), a human myelogenous leukemia cell line; and U-937 (ATCC CRL-1593.2), a histiocytic lymphoma cell line. All cell lines were grown on uncoated plates (Corning) according to vendor recommendations. Media was changed every 3 days and cells were passaged when reaching 70% confluency. All cell lines were incubated at 37 °C, 5% CO2. TrypLE (Thermo Scientific) was used to detach cells from plates.Mouse cortex sampleMouse cortices were isolated from p0 pups of C57BL/6 background, cut into small pieces, and flash frozen. Lysis was performed by adding Pierce IP lysis buffer to frozen pieces and immediately applying a handheld homogenizer (VWR pellet mixer #47747-370). All subsequent steps were the same as those for iNs.

Experimental model and subject details

Cell linesiNsiNs collected at time points under one week of differentiation were dissociated from plates with Accutase (Innovative Cell Technologies AT104-500) at 37°C for 5-10 min. iNs collected at time points beyond two weeks of differentiation were collected by removing media, adding 1x PBS, and swirling the plates until the neurons detached. Dissociated or detached cells were collected in 15 mL or 50 mL Falcon tubes depending on the number of cells and washed 3 times. Each wash consisted of centrifugation at 300 RCF for 3-5 min, discarding of the supernatant, and resuspension of the cell pellet in 1x PBS. After the final wash, the sample was centrifuged at 300 RCF for 3-5 min, supernatant was discarded, and cells were either lysed or flash frozen in liquid nitrogen and stored at -80°C. Lysis was performed according to manufacturer’s protocol using Pierce IP lysis buffer (Thermo Scientific #87788) with 1x Halt Protease and Phosphatase cocktail inhibitor (Thermo Scientific #7844). Protein concentration of the lysate was quantified using BCA Protein assay (Thermo Scientific #23227). When not used for IP-MS experiments, the lysate was aliquoted in low-bind microfuge tubes (Axygen Scientific #MCT-175-L-C), flash frozen and stored at -80°C.Cancer and HEK cellsWe used the following cell lines as controls in western blots: HEK-293 (ATCC CRL-1573), a human embryonic kidney cell line; TF-1 (ATCC CRL-2003), a human erythroleukemia cell line; K-562 (ATCC CCL-243), a human myelogenous leukemia cell line; and U-937 (ATCC CRL-1593.2), a histiocytic lymphoma cell line. All cell lines were grown on uncoated plates (Corning) according to vendor recommendations. Media was changed every 3 days and cells were passaged when reaching 70% confluency. All cell lines were incubated at 37 °C, 5% CO2. TrypLE (Thermo Scientific) was used to detach cells from plates.

iNsiNs collected at time points under one week of differentiation were dissociated from plates with Accutase (Innovative Cell Technologies AT104-500) at 37°C for 5-10 min. iNs collected at time points beyond two weeks of differentiation were collected by removing media, adding 1x PBS, and swirling the plates until the neurons detached. Dissociated or detached cells were collected in 15 mL or 50 mL Falcon tubes depending on the number of cells and washed 3 times. Each wash consisted of centrifugation at 300 RCF for 3-5 min, discarding of the supernatant, and resuspension of the cell pellet in 1x PBS. After the final wash, the sample was centrifuged at 300 RCF for 3-5 min, supernatant was discarded, and cells were either lysed or flash frozen in liquid nitrogen and stored at -80°C. Lysis was performed according to manufacturer’s protocol using Pierce IP lysis buffer (Thermo Scientific #87788) with 1x Halt Protease and Phosphatase cocktail inhibitor (Thermo Scientific #7844). Protein concentration of the lysate was quantified using BCA Protein assay (Thermo Scientific #23227). When not used for IP-MS experiments, the lysate was aliquoted in low-bind microfuge tubes (Axygen Scientific #MCT-175-L-C), flash frozen and stored at -80°C.

iNs collected at time points under one week of differentiation were dissociated from plates with Accutase (Innovative Cell Technologies AT104-500) at 37°C for 5-10 min. iNs collected at time points beyond two weeks of differentiation were collected by removing media, adding 1x PBS, and swirling the plates until the neurons detached. Dissociated or detached cells were collected in 15 mL or 50 mL Falcon tubes depending on the number of cells and washed 3 times. Each wash consisted of centrifugation at 300 RCF for 3-5 min, discarding of the supernatant, and resuspension of the cell pellet in 1x PBS. After the final wash, the sample was centrifuged at 300 RCF for 3-5 min, supernatant was discarded, and cells were either lysed or flash frozen in liquid nitrogen and stored at -80°C. Lysis was performed according to manufacturer’s protocol using Pierce IP lysis buffer (Thermo Scientific #87788) with 1x Halt Protease and Phosphatase cocktail inhibitor (Thermo Scientific #7844). Protein concentration of the lysate was quantified using BCA Protein assay (Thermo Scientific #23227). When not used for IP-MS experiments, the lysate was aliquoted in low-bind microfuge tubes (Axygen Scientific #MCT-175-L-C), flash frozen and stored at -80°C.

Cancer and HEK cellsWe used the following cell lines as controls in western blots: HEK-293 (ATCC CRL-1573), a human embryonic kidney cell line; TF-1 (ATCC CRL-2003), a human erythroleukemia cell line; K-562 (ATCC CCL-243), a human myelogenous leukemia cell line; and U-937 (ATCC CRL-1593.2), a histiocytic lymphoma cell line. All cell lines were grown on uncoated plates (Corning) according to vendor recommendations. Media was changed every 3 days and cells were passaged when reaching 70% confluency. All cell lines were incubated at 37 °C, 5% CO2. TrypLE (Thermo Scientific) was used to detach cells from plates.

Cancer and HEK cells

We used the following cell lines as controls in western blots: HEK-293 (ATCC CRL-1573), a human embryonic kidney cell line; TF-1 (ATCC CRL-2003), a human erythroleukemia cell line; K-562 (ATCC CCL-243), a human myelogenous leukemia cell line; and U-937 (ATCC CRL-1593.2), a histiocytic lymphoma cell line. All cell lines were grown on uncoated plates (Corning) according to vendor recommendations. Media was changed every 3 days and cells were passaged when reaching 70% confluency. All cell lines were incubated at 37 °C, 5% CO2. TrypLE (Thermo Scientific) was used to detach cells from plates.

Mouse cortex sampleMouse cortices were isolated from p0 pups of C57BL/6 background, cut into small pieces, and flash frozen. Lysis was performed by adding Pierce IP lysis buffer to frozen pieces and immediately applying a handheld homogenizer (VWR pellet mixer #47747-370). All subsequent steps were the same as those for iNs.

Mouse cortex sample

Mouse cortices were isolated from p0 pups of C57BL/6 background, cut into small pieces, and flash frozen. Lysis was performed by adding Pierce IP lysis buffer to frozen pieces and immediately applying a handheld homogenizer (VWR pellet mixer #47747-370). All subsequent steps were the same as those for iNs.

Method detailsiN differentiationGlutamatergic patterned induced neurons (iNs) were differentiated from male stem cells by conditional expression of the neuralizing transcription factor NGN2 as previously described,21 with the exception that N2 media was used instead of KSR media during days 0 to 3. iNs were re-passaged at day 3 of differentiation (i.e., 40,000 cells/cm2) on Geltrex (Thermo Scientific #A1413202) coated plates. To remove remaining proliferating cells, fluoro-deoxyuridine (Bioworld 40690016-2) was added to cell cultures at 10 μM on day 6 of differentiation. iNs used in a subset of the CACNA1C, HCN1, and TCF4 IP-MS experiments (Table S5) were differentiated from human embryonic stem cells WA01 (H1, NIH registration no. 0043) transduced with lentiviruses carrying TetO::Ngn2-Puro and reverse tetracycline-controlled transactivator (rtTA), custom-packaged by Alstem. All other IP-MS experiments used iNs generated from a clonally selected induced PSC line (iPS hDFn 83/22 iNgn2#9 [iPS3]) with TetO::Ngn2 and rtTA delivered via PiggyBAC.21Western blottingFor western blots, frozen lysates were thawed on ice, diluted to desired concentration in 1x PBS, and brought to 1x LDS using 4x LDS stock. Samples were denatured at various conditions based on what we identified to work best for each protein of interest. Prepared samples were run on NuPAGE 1.5mm 3-7% Tris-Acetate or 1mm 4-12% Bis-Tris gels (Thermo Scientific). Transfer was onto a nitrocellulose membrane using XCell wet transfer (Invitrogen), iBlot2 (Thermo Scientific), or Trans-Blot Turbo (BioRad). Membranes were blocked with 5% BSA or 5% milk diluted in TBST, incubated with primary antibody diluted in matching blocking buffer for at least 12 hat 4°C, and washed 3 x 10 min in TBST. Membranes were incubated with HRP-conjugated secondary antibody diluted in 5% milk for 1h at RT, washed 3 x 10 min in TBST, developed with enhanced chemiluminescent substrate (Thermo Scientific #34095), and imaged on ChemiDoc MP (BioRad). In ‘forward’ IP western validation, the immunoprecipitate of the index protein was loaded into the gel and incubated with the primary antibody of an interactor protein to determine if the interactor could be found in the immunoprecipitate; in ‘reverse’ IP western validation, the immunoprecipitate of an interactor was loaded into the gel and incubated with the primary antibody of the index protein to determine if the index protein could be found in the immunoprecipitate. Western blot antibodies used for each protein of interest are listed in Table S4.ImmunoprecipitationsFor immunoprecipitations (IPs) followed by western blots or Coomassie stain analysis, either fresh or previously frozen lysates were used. For IPs followed by mass spectrometry (MS) only fresh lysates were used. On day 1, the needed amounts of lysate and IP antibody were added to a 1.7 mL Axygen MaxyClear tube (MCT-175-L-C), then brought to a final volume of 1.3 mL with Pierce IP lysis buffer. Tubes were then rotated at 4°C overnight for 14-18 h. On day 2, the needed amount of beads per IP were apportioned to separate tubes, washed twice in cold lysis buffer, resuspended in 200 μL lysis buffer per tube, added to corresponding tubes, and rotated at 4°C for 2-4 h. After incubation, tubes were placed on ice, and beads were first washed once in 1mL cold lysis buffer and then twice in 1x cold PBS. To remove supernatants in between the steps, magnetic beads were placed on a magnetic rack on ice and agarose beads were spun at 2,000 x g for 3 min. Supernatants after overnight rotation and each wash step were collected for western blot analysis of IP quality. After the third wash, supernatant was removed, and beads were resuspended in 50 μL of PBS if the IP was to be processed by MS and 40 μL of PBS if the IP was to be processed by western blot or Coomassie stain. Samples were flash frozen in liquid nitrogen and stored at -80°C until further use. All IP experiments sent for MS consisted of 4 IP samples performed on cells from the same differentiation batch: 2 replicate experimental IPs using an antibody against the protein of interest (i.e., an index protein) and 2 replicate control IPs using a control IgG antibody. IP antibodies used for each protein of interest are listed in Table S4.Immunoprecipitations using V5 epitope tagged TCF4For immunoprecipitation of TCF4, an ORF of the TCF4 isoform A with a c-terminal V5-tag was acquired from the Broad Institute Genetic Perturbation Platform, and lentiviruses were generated from the construct by ALSTEM Inc. Lentiviruses were delivered to iN cultures at a multiplicity of infection (MOI) of 4 at day 3 of differentiation during re-passaging, and mock-transduced cells were used as controls. Cells were collected at day 6 of differentiation by washing 3 times with PBS, followed by lysis and scraping on the plate. Lysates were then processed and quantified as usual. For IP, the needed amount of lysate was added to a microfuge tube and brought to a final volume of 1.3 mL with Pierce IP lysis buffer, then incubated overnight with anti-v5 antibody directly coupled to magnetic beads (MBL International, M167-11). The remaining protocol is the same as used for the other IPs.Mass spectrometrySample preparationProteins were digested on beads using 90 μl of digestion buffer (2 M urea/50 mM Tris buffer with 1 mM DTT and 5 μg/mL Trypsin) for 1 hr, shaking at 1000 rpm. The suspension was then transferred to a new tube, and the beads were washed twice with 60 μL of wash buffer (2 M urea/50 mM Tris buffer). The wash buffer was added to the suspension with digestion. The digestion and wash process were repeated a second time pooling the suspensions with the suspensions from the first round. The pooled solution was reduced using 4 mM DTT for 30 minat 25°C shaking at 1000 rpm. The proteins were then alkylated using 10 mM iodoacetamide and incubating for 45 minat 25°C shaking at 1000 rpm and protected from light. Proteins were then digested with 0.5 μg of trypsin overnight at 25°C shaking at 700 rpm. The next day proteins were quenched using 40 μL of 10% formic acid and desalted using an Oasis Cartridge. Samples were vacuum dried and labeled with iTRAQ4 (Sciex Inc.) or TMT10 (Thermo Scientific) kits. Each iTRAQ 4-plex consisted of 2 replicate experimental IPs using an antibody against the protein of interest (i.e., an index protein), and 2 replicate control IPs using a control IgG antibody. The specific iTRAQ labels for each replicate pair are indicated in Table S5. For ‘RIMS1_wk4_2’, a TMT 10-plex experiment was conducted with antibodies from Santa Cruz Biotechnology (SC; 128C, 129N), Proteintech Group (PT; 126, 127N), and Synaptic Systems (SYSY; 129C, 130N). Two sets of replicate control IPs were included for PT (127C, 128N) and SC/SYSY (130C, 131). Only the results from the SC antibody met our quality control metrics and were reported.Liquid chromatography-tandem mass spectrometry (LC-MS/MS)Reconstituted peptides were separated on an online nanoflow EASY-nLC 1000 UHPLC system (Thermo Scientific) and analyzed on a benchtop Orbitrap Q Exactive Plus mass spectrometer (Thermo Scientific). The peptide samples were injected onto a capillary column (Picofrit with 10 μm tip opening/75 μm diameter, New Objective, PF360-75-10-N-5) packed in-house with 20 cm C18 silica material (1.9 μm ReproSil-Pur C18-AQ medium, Dr. Maisch GmbH, r119.aq). The UHPLC setup was connected with a custom-fit microadapting tee (360 μm, IDEX Health & Science, UH-753), and capillary columns were heated to 50 °C in column heater sleeves (Phoenix-ST) to reduce backpressure during UHPLC separation. Injected peptides were separated at a flow rate of 200 nL/min with a linear 150 min gradient from 94% solvent A (3% acetonitrile, 0.1% formic acid) to 35% solvent B (90% acetonitrile, 0.1% formic acid), followed by a linear 8 min gradient from 35% solvent B to 60% solvent B and a 3 min ramp to 90% B. The Q Exactive instrument was operated in the data-dependent mode acquiring HCD MS/MS scans (R=17,500 for iTRAQ4, or R=35,000 for TMT10) after each MS1 scan (R=70,000) on the 12 most abundant ions using an MS1 ion target of 3x106 ions and an MS2 target of 5x104 ions. The maximum ion time utilized for the MS/MS scans was 120 ms; the HCD-normalized collision energy was set to 28 for iTRAQ4 or 31 for TMT10; the dynamic exclusion time was set to 20s, and the peptide match and isotope exclusion functions were enabled.

Method details

iN differentiationGlutamatergic patterned induced neurons (iNs) were differentiated from male stem cells by conditional expression of the neuralizing transcription factor NGN2 as previously described,21 with the exception that N2 media was used instead of KSR media during days 0 to 3. iNs were re-passaged at day 3 of differentiation (i.e., 40,000 cells/cm2) on Geltrex (Thermo Scientific #A1413202) coated plates. To remove remaining proliferating cells, fluoro-deoxyuridine (Bioworld 40690016-2) was added to cell cultures at 10 μM on day 6 of differentiation. iNs used in a subset of the CACNA1C, HCN1, and TCF4 IP-MS experiments (Table S5) were differentiated from human embryonic stem cells WA01 (H1, NIH registration no. 0043) transduced with lentiviruses carrying TetO::Ngn2-Puro and reverse tetracycline-controlled transactivator (rtTA), custom-packaged by Alstem. All other IP-MS experiments used iNs generated from a clonally selected induced PSC line (iPS hDFn 83/22 iNgn2#9 [iPS3]) with TetO::Ngn2 and rtTA delivered via PiggyBAC.21

iN differentiation

Glutamatergic patterned induced neurons (iNs) were differentiated from male stem cells by conditional expression of the neuralizing transcription factor NGN2 as previously described,21 with the exception that N2 media was used instead of KSR media during days 0 to 3. iNs were re-passaged at day 3 of differentiation (i.e., 40,000 cells/cm2) on Geltrex (Thermo Scientific #A1413202) coated plates. To remove remaining proliferating cells, fluoro-deoxyuridine (Bioworld 40690016-2) was added to cell cultures at 10 μM on day 6 of differentiation. iNs used in a subset of the CACNA1C, HCN1, and TCF4 IP-MS experiments (Table S5) were differentiated from human embryonic stem cells WA01 (H1, NIH registration no. 0043) transduced with lentiviruses carrying TetO::Ngn2-Puro and reverse tetracycline-controlled transactivator (rtTA), custom-packaged by Alstem. All other IP-MS experiments used iNs generated from a clonally selected induced PSC line (iPS hDFn 83/22 iNgn2#9 [iPS3]) with TetO::Ngn2 and rtTA delivered via PiggyBAC.21

Western blottingFor western blots, frozen lysates were thawed on ice, diluted to desired concentration in 1x PBS, and brought to 1x LDS using 4x LDS stock. Samples were denatured at various conditions based on what we identified to work best for each protein of interest. Prepared samples were run on NuPAGE 1.5mm 3-7% Tris-Acetate or 1mm 4-12% Bis-Tris gels (Thermo Scientific). Transfer was onto a nitrocellulose membrane using XCell wet transfer (Invitrogen), iBlot2 (Thermo Scientific), or Trans-Blot Turbo (BioRad). Membranes were blocked with 5% BSA or 5% milk diluted in TBST, incubated with primary antibody diluted in matching blocking buffer for at least 12 hat 4°C, and washed 3 x 10 min in TBST. Membranes were incubated with HRP-conjugated secondary antibody diluted in 5% milk for 1h at RT, washed 3 x 10 min in TBST, developed with enhanced chemiluminescent substrate (Thermo Scientific #34095), and imaged on ChemiDoc MP (BioRad). In ‘forward’ IP western validation, the immunoprecipitate of the index protein was loaded into the gel and incubated with the primary antibody of an interactor protein to determine if the interactor could be found in the immunoprecipitate; in ‘reverse’ IP western validation, the immunoprecipitate of an interactor was loaded into the gel and incubated with the primary antibody of the index protein to determine if the index protein could be found in the immunoprecipitate. Western blot antibodies used for each protein of interest are listed in Table S4.

Western blotting

For western blots, frozen lysates were thawed on ice, diluted to desired concentration in 1x PBS, and brought to 1x LDS using 4x LDS stock. Samples were denatured at various conditions based on what we identified to work best for each protein of interest. Prepared samples were run on NuPAGE 1.5mm 3-7% Tris-Acetate or 1mm 4-12% Bis-Tris gels (Thermo Scientific). Transfer was onto a nitrocellulose membrane using XCell wet transfer (Invitrogen), iBlot2 (Thermo Scientific), or Trans-Blot Turbo (BioRad). Membranes were blocked with 5% BSA or 5% milk diluted in TBST, incubated with primary antibody diluted in matching blocking buffer for at least 12 hat 4°C, and washed 3 x 10 min in TBST. Membranes were incubated with HRP-conjugated secondary antibody diluted in 5% milk for 1h at RT, washed 3 x 10 min in TBST, developed with enhanced chemiluminescent substrate (Thermo Scientific #34095), and imaged on ChemiDoc MP (BioRad). In ‘forward’ IP western validation, the immunoprecipitate of the index protein was loaded into the gel and incubated with the primary antibody of an interactor protein to determine if the interactor could be found in the immunoprecipitate; in ‘reverse’ IP western validation, the immunoprecipitate of an interactor was loaded into the gel and incubated with the primary antibody of the index protein to determine if the index protein could be found in the immunoprecipitate. Western blot antibodies used for each protein of interest are listed in Table S4.

ImmunoprecipitationsFor immunoprecipitations (IPs) followed by western blots or Coomassie stain analysis, either fresh or previously frozen lysates were used. For IPs followed by mass spectrometry (MS) only fresh lysates were used. On day 1, the needed amounts of lysate and IP antibody were added to a 1.7 mL Axygen MaxyClear tube (MCT-175-L-C), then brought to a final volume of 1.3 mL with Pierce IP lysis buffer. Tubes were then rotated at 4°C overnight for 14-18 h. On day 2, the needed amount of beads per IP were apportioned to separate tubes, washed twice in cold lysis buffer, resuspended in 200 μL lysis buffer per tube, added to corresponding tubes, and rotated at 4°C for 2-4 h. After incubation, tubes were placed on ice, and beads were first washed once in 1mL cold lysis buffer and then twice in 1x cold PBS. To remove supernatants in between the steps, magnetic beads were placed on a magnetic rack on ice and agarose beads were spun at 2,000 x g for 3 min. Supernatants after overnight rotation and each wash step were collected for western blot analysis of IP quality. After the third wash, supernatant was removed, and beads were resuspended in 50 μL of PBS if the IP was to be processed by MS and 40 μL of PBS if the IP was to be processed by western blot or Coomassie stain. Samples were flash frozen in liquid nitrogen and stored at -80°C until further use. All IP experiments sent for MS consisted of 4 IP samples performed on cells from the same differentiation batch: 2 replicate experimental IPs using an antibody against the protein of interest (i.e., an index protein) and 2 replicate control IPs using a control IgG antibody. IP antibodies used for each protein of interest are listed in Table S4.

Immunoprecipitations

For immunoprecipitations (IPs) followed by western blots or Coomassie stain analysis, either fresh or previously frozen lysates were used. For IPs followed by mass spectrometry (MS) only fresh lysates were used. On day 1, the needed amounts of lysate and IP antibody were added to a 1.7 mL Axygen MaxyClear tube (MCT-175-L-C), then brought to a final volume of 1.3 mL with Pierce IP lysis buffer. Tubes were then rotated at 4°C overnight for 14-18 h. On day 2, the needed amount of beads per IP were apportioned to separate tubes, washed twice in cold lysis buffer, resuspended in 200 μL lysis buffer per tube, added to corresponding tubes, and rotated at 4°C for 2-4 h. After incubation, tubes were placed on ice, and beads were first washed once in 1mL cold lysis buffer and then twice in 1x cold PBS. To remove supernatants in between the steps, magnetic beads were placed on a magnetic rack on ice and agarose beads were spun at 2,000 x g for 3 min. Supernatants after overnight rotation and each wash step were collected for western blot analysis of IP quality. After the third wash, supernatant was removed, and beads were resuspended in 50 μL of PBS if the IP was to be processed by MS and 40 μL of PBS if the IP was to be processed by western blot or Coomassie stain. Samples were flash frozen in liquid nitrogen and stored at -80°C until further use. All IP experiments sent for MS consisted of 4 IP samples performed on cells from the same differentiation batch: 2 replicate experimental IPs using an antibody against the protein of interest (i.e., an index protein) and 2 replicate control IPs using a control IgG antibody. IP antibodies used for each protein of interest are listed in Table S4.

Immunoprecipitations using V5 epitope tagged TCF4For immunoprecipitation of TCF4, an ORF of the TCF4 isoform A with a c-terminal V5-tag was acquired from the Broad Institute Genetic Perturbation Platform, and lentiviruses were generated from the construct by ALSTEM Inc. Lentiviruses were delivered to iN cultures at a multiplicity of infection (MOI) of 4 at day 3 of differentiation during re-passaging, and mock-transduced cells were used as controls. Cells were collected at day 6 of differentiation by washing 3 times with PBS, followed by lysis and scraping on the plate. Lysates were then processed and quantified as usual. For IP, the needed amount of lysate was added to a microfuge tube and brought to a final volume of 1.3 mL with Pierce IP lysis buffer, then incubated overnight with anti-v5 antibody directly coupled to magnetic beads (MBL International, M167-11). The remaining protocol is the same as used for the other IPs.

Immunoprecipitations using V5 epitope tagged TCF4

For immunoprecipitation of TCF4, an ORF of the TCF4 isoform A with a c-terminal V5-tag was acquired from the Broad Institute Genetic Perturbation Platform, and lentiviruses were generated from the construct by ALSTEM Inc. Lentiviruses were delivered to iN cultures at a multiplicity of infection (MOI) of 4 at day 3 of differentiation during re-passaging, and mock-transduced cells were used as controls. Cells were collected at day 6 of differentiation by washing 3 times with PBS, followed by lysis and scraping on the plate. Lysates were then processed and quantified as usual. For IP, the needed amount of lysate was added to a microfuge tube and brought to a final volume of 1.3 mL with Pierce IP lysis buffer, then incubated overnight with anti-v5 antibody directly coupled to magnetic beads (MBL International, M167-11). The remaining protocol is the same as used for the other IPs.

Mass spectrometrySample preparationProteins were digested on beads using 90 μl of digestion buffer (2 M urea/50 mM Tris buffer with 1 mM DTT and 5 μg/mL Trypsin) for 1 hr, shaking at 1000 rpm. The suspension was then transferred to a new tube, and the beads were washed twice with 60 μL of wash buffer (2 M urea/50 mM Tris buffer). The wash buffer was added to the suspension with digestion. The digestion and wash process were repeated a second time pooling the suspensions with the suspensions from the first round. The pooled solution was reduced using 4 mM DTT for 30 minat 25°C shaking at 1000 rpm. The proteins were then alkylated using 10 mM iodoacetamide and incubating for 45 minat 25°C shaking at 1000 rpm and protected from light. Proteins were then digested with 0.5 μg of trypsin overnight at 25°C shaking at 700 rpm. The next day proteins were quenched using 40 μL of 10% formic acid and desalted using an Oasis Cartridge. Samples were vacuum dried and labeled with iTRAQ4 (Sciex Inc.) or TMT10 (Thermo Scientific) kits. Each iTRAQ 4-plex consisted of 2 replicate experimental IPs using an antibody against the protein of interest (i.e., an index protein), and 2 replicate control IPs using a control IgG antibody. The specific iTRAQ labels for each replicate pair are indicated in Table S5. For ‘RIMS1_wk4_2’, a TMT 10-plex experiment was conducted with antibodies from Santa Cruz Biotechnology (SC; 128C, 129N), Proteintech Group (PT; 126, 127N), and Synaptic Systems (SYSY; 129C, 130N). Two sets of replicate control IPs were included for PT (127C, 128N) and SC/SYSY (130C, 131). Only the results from the SC antibody met our quality control metrics and were reported.Liquid chromatography-tandem mass spectrometry (LC-MS/MS)Reconstituted peptides were separated on an online nanoflow EASY-nLC 1000 UHPLC system (Thermo Scientific) and analyzed on a benchtop Orbitrap Q Exactive Plus mass spectrometer (Thermo Scientific). The peptide samples were injected onto a capillary column (Picofrit with 10 μm tip opening/75 μm diameter, New Objective, PF360-75-10-N-5) packed in-house with 20 cm C18 silica material (1.9 μm ReproSil-Pur C18-AQ medium, Dr. Maisch GmbH, r119.aq). The UHPLC setup was connected with a custom-fit microadapting tee (360 μm, IDEX Health & Science, UH-753), and capillary columns were heated to 50 °C in column heater sleeves (Phoenix-ST) to reduce backpressure during UHPLC separation. Injected peptides were separated at a flow rate of 200 nL/min with a linear 150 min gradient from 94% solvent A (3% acetonitrile, 0.1% formic acid) to 35% solvent B (90% acetonitrile, 0.1% formic acid), followed by a linear 8 min gradient from 35% solvent B to 60% solvent B and a 3 min ramp to 90% B. The Q Exactive instrument was operated in the data-dependent mode acquiring HCD MS/MS scans (R=17,500 for iTRAQ4, or R=35,000 for TMT10) after each MS1 scan (R=70,000) on the 12 most abundant ions using an MS1 ion target of 3x106 ions and an MS2 target of 5x104 ions. The maximum ion time utilized for the MS/MS scans was 120 ms; the HCD-normalized collision energy was set to 28 for iTRAQ4 or 31 for TMT10; the dynamic exclusion time was set to 20s, and the peptide match and isotope exclusion functions were enabled.

Mass spectrometry

Sample preparationProteins were digested on beads using 90 μl of digestion buffer (2 M urea/50 mM Tris buffer with 1 mM DTT and 5 μg/mL Trypsin) for 1 hr, shaking at 1000 rpm. The suspension was then transferred to a new tube, and the beads were washed twice with 60 μL of wash buffer (2 M urea/50 mM Tris buffer). The wash buffer was added to the suspension with digestion. The digestion and wash process were repeated a second time pooling the suspensions with the suspensions from the first round. The pooled solution was reduced using 4 mM DTT for 30 minat 25°C shaking at 1000 rpm. The proteins were then alkylated using 10 mM iodoacetamide and incubating for 45 minat 25°C shaking at 1000 rpm and protected from light. Proteins were then digested with 0.5 μg of trypsin overnight at 25°C shaking at 700 rpm. The next day proteins were quenched using 40 μL of 10% formic acid and desalted using an Oasis Cartridge. Samples were vacuum dried and labeled with iTRAQ4 (Sciex Inc.) or TMT10 (Thermo Scientific) kits. Each iTRAQ 4-plex consisted of 2 replicate experimental IPs using an antibody against the protein of interest (i.e., an index protein), and 2 replicate control IPs using a control IgG antibody. The specific iTRAQ labels for each replicate pair are indicated in Table S5. For ‘RIMS1_wk4_2’, a TMT 10-plex experiment was conducted with antibodies from Santa Cruz Biotechnology (SC; 128C, 129N), Proteintech Group (PT; 126, 127N), and Synaptic Systems (SYSY; 129C, 130N). Two sets of replicate control IPs were included for PT (127C, 128N) and SC/SYSY (130C, 131). Only the results from the SC antibody met our quality control metrics and were reported.

Sample preparation

Proteins were digested on beads using 90 μl of digestion buffer (2 M urea/50 mM Tris buffer with 1 mM DTT and 5 μg/mL Trypsin) for 1 hr, shaking at 1000 rpm. The suspension was then transferred to a new tube, and the beads were washed twice with 60 μL of wash buffer (2 M urea/50 mM Tris buffer). The wash buffer was added to the suspension with digestion. The digestion and wash process were repeated a second time pooling the suspensions with the suspensions from the first round. The pooled solution was reduced using 4 mM DTT for 30 minat 25°C shaking at 1000 rpm. The proteins were then alkylated using 10 mM iodoacetamide and incubating for 45 minat 25°C shaking at 1000 rpm and protected from light. Proteins were then digested with 0.5 μg of trypsin overnight at 25°C shaking at 700 rpm. The next day proteins were quenched using 40 μL of 10% formic acid and desalted using an Oasis Cartridge. Samples were vacuum dried and labeled with iTRAQ4 (Sciex Inc.) or TMT10 (Thermo Scientific) kits. Each iTRAQ 4-plex consisted of 2 replicate experimental IPs using an antibody against the protein of interest (i.e., an index protein), and 2 replicate control IPs using a control IgG antibody. The specific iTRAQ labels for each replicate pair are indicated in Table S5. For ‘RIMS1_wk4_2’, a TMT 10-plex experiment was conducted with antibodies from Santa Cruz Biotechnology (SC; 128C, 129N), Proteintech Group (PT; 126, 127N), and Synaptic Systems (SYSY; 129C, 130N). Two sets of replicate control IPs were included for PT (127C, 128N) and SC/SYSY (130C, 131). Only the results from the SC antibody met our quality control metrics and were reported.

Liquid chromatography-tandem mass spectrometry (LC-MS/MS)Reconstituted peptides were separated on an online nanoflow EASY-nLC 1000 UHPLC system (Thermo Scientific) and analyzed on a benchtop Orbitrap Q Exactive Plus mass spectrometer (Thermo Scientific). The peptide samples were injected onto a capillary column (Picofrit with 10 μm tip opening/75 μm diameter, New Objective, PF360-75-10-N-5) packed in-house with 20 cm C18 silica material (1.9 μm ReproSil-Pur C18-AQ medium, Dr. Maisch GmbH, r119.aq). The UHPLC setup was connected with a custom-fit microadapting tee (360 μm, IDEX Health & Science, UH-753), and capillary columns were heated to 50 °C in column heater sleeves (Phoenix-ST) to reduce backpressure during UHPLC separation. Injected peptides were separated at a flow rate of 200 nL/min with a linear 150 min gradient from 94% solvent A (3% acetonitrile, 0.1% formic acid) to 35% solvent B (90% acetonitrile, 0.1% formic acid), followed by a linear 8 min gradient from 35% solvent B to 60% solvent B and a 3 min ramp to 90% B. The Q Exactive instrument was operated in the data-dependent mode acquiring HCD MS/MS scans (R=17,500 for iTRAQ4, or R=35,000 for TMT10) after each MS1 scan (R=70,000) on the 12 most abundant ions using an MS1 ion target of 3x106 ions and an MS2 target of 5x104 ions. The maximum ion time utilized for the MS/MS scans was 120 ms; the HCD-normalized collision energy was set to 28 for iTRAQ4 or 31 for TMT10; the dynamic exclusion time was set to 20s, and the peptide match and isotope exclusion functions were enabled.

Liquid chromatography-tandem mass spectrometry (LC-MS/MS)

Reconstituted peptides were separated on an online nanoflow EASY-nLC 1000 UHPLC system (Thermo Scientific) and analyzed on a benchtop Orbitrap Q Exactive Plus mass spectrometer (Thermo Scientific). The peptide samples were injected onto a capillary column (Picofrit with 10 μm tip opening/75 μm diameter, New Objective, PF360-75-10-N-5) packed in-house with 20 cm C18 silica material (1.9 μm ReproSil-Pur C18-AQ medium, Dr. Maisch GmbH, r119.aq). The UHPLC setup was connected with a custom-fit microadapting tee (360 μm, IDEX Health & Science, UH-753), and capillary columns were heated to 50 °C in column heater sleeves (Phoenix-ST) to reduce backpressure during UHPLC separation. Injected peptides were separated at a flow rate of 200 nL/min with a linear 150 min gradient from 94% solvent A (3% acetonitrile, 0.1% formic acid) to 35% solvent B (90% acetonitrile, 0.1% formic acid), followed by a linear 8 min gradient from 35% solvent B to 60% solvent B and a 3 min ramp to 90% B. The Q Exactive instrument was operated in the data-dependent mode acquiring HCD MS/MS scans (R=17,500 for iTRAQ4, or R=35,000 for TMT10) after each MS1 scan (R=70,000) on the 12 most abundant ions using an MS1 ion target of 3x106 ions and an MS2 target of 5x104 ions. The maximum ion time utilized for the MS/MS scans was 120 ms; the HCD-normalized collision energy was set to 28 for iTRAQ4 or 31 for TMT10; the dynamic exclusion time was set to 20s, and the peptide match and isotope exclusion functions were enabled.

Quantification and statistical analysisRefining geneticdata to identify index genesThree-step procedure for selecting index genesStarting with 125 independent autosomal SNPs that reached genome-wide significance in the combined discovery-replication meta-analysis of the PGC schizophrenia GWAS3 (phase 2), Ricopili57 (https://data.broadinstitute.org/mpg/ricopili/) was used to map the 124 non-MHC (major histocompatibility complex) region SNPs to 445 genes (Set 1; Table S1) in linkage disequilibrium (LD) loci, whose boundaries were defined by SNPs in LD (r2> 0.6) with the index SNPs, ±50kb on either end. Next, we excluded SNPs in intergenic regions or in loci with multiple protein-coding genes, resulting in a list of 40 SNPs pointing to 37 unique protein-coding genes (Set 2) in single-gene loci. We further refined the 37 genes into a set of 10 genes (Set 3) based on strong orthogonal evidence supporting their involvement in psychiatric diseases (see below). We also included SYNGAP1 in the MHC region (which was excluded from the SNP-to-gene mapping process due to its LD complexity) in all 3 sets based on strong orthogonal evidence. In total, we identified 11 high-confidence schizophrenia risk genes and considered their encoded proteins as index proteins in the proteomic experiments.Orthogonal evidence for the index genesThe index genes were selected based on several types of orthogonal genetic or functional evidence, including: i] high-density genotyping experiments of individual genes (ZNF804A),60 ii] sequencing or copy number variant studies linking genes to autism spectrum disorders or neurodevelopmental delay (CACNA1C, CUL3, CSMD1, GRIN2A, SATB2 SYNGAP1, TCF4),39,61,62,63,64,65,66,67,68 iii] strong functional evidence supporting that the gene in question is causal (HCN1),69 and iv] multiple subunits from the same protein complex are encoded by genes in different single-gene loci linked to psychiatric diseases (CACNA1C, CACNB2, RIMS1).3,70,71 In addition, we annotated the 10 genes prioritized from single-gene GWAS loci using Hi-C chromatin interaction data from the developing human brain,72which may point to long-range regulation of genes outside of the loci in the context of schizophrenia; only three of these genes (CUL3, RIMS1, SATB2) lie in loci that exclusively interacted with long-range genes outside of the loci (Table S1).pLI score enrichment analysisWe extracted pLI scores from the gnomAD22 (v2.1.1) ‘pLoF Metrics by Gene TSV’ dataset. We performed one-tailed Kolmogorov-Smirnov (KS) tests to assess whether various gene sets are enriched for high pLI scores compared to other genes in the genome. The tested gene sets include: i] SCHEMA9 genes with FDR <3.7e-3 (exome-wide significance) and 0.5, ii] Sets 1-3 defined by our index gene selection steps, and iii] genes in PGC schizophrenia GWAS (phase 3) loci10 and subsets of these genes prioritized by FINEMAP/SMR analysis and/or our interaction data (see social manhattan plot). In addition, we performed one-tailed KS tests to compare the pLI scores of several gene sets that are subsets and supersets of each other, and two-tailed KS tests to compare a few disjoint sets. Table S2 summarizes all the comparisons performed. Finally, we also performed one-tailed KS tests to assess whether the stringent interactors in our interaction networks (Table S9) have enriched pLI scores compared to the non-interactors linked to each network.BrainSpan expression profilesWe obtained gene expression data in four distinct parts of the frontal cortex (dorsolateral prefrontal cortex [DFC], medial prefrontal cortex [MFC], ventrolateral prefrontal cortex [VFC], orbital frontal cortex [OFC]) across 10 developmental stages from the BrainSpan (https://www.brainspan.org) exon microarray dataset.23 At each developmental stage, we calculated the median and standard error of the expression values for various genes or gene sets including: i] SCHEMA genes with FDR <3.7e-3, 0.05, 0.25, and 0.5, ii] Sets 1-3 defined by our index gene selection steps, iii] index and interactor genes in our combined network of all IPs (Table S9), iv] genes in PGC schizophrenia GWAS (phase 3) loci and subsets of these genes prioritized by FINEMAP/SMR analysis and/or our interaction data (see social manhattan plot), and v] random genes sampled from the BrainSpan dataset for comparison against the other gene sets. Table S3 provides more detailed summary statistics for all gene sets.IP-MS data analysisSpectrum Mill analysisAll mass spectra were processed using Spectrum Mill (v7.0; https://proteomics.broadinstitute.org). For peptide identification, MS/MS spectra were searched against a sequence database for the UniProt human reference proteome, including isoforms, with a set of common laboratory contaminant proteins appended (2017: 65,068 entries, or 2014: 59,079 entries) as indicated in Table S5. Search parameters included: ESI-QEXACTIVE-HCD scoring parameters, trypsin enzyme specificity with a maximum of two missed cleavages, 40% minimum matched peak intensity, ±20 ppm precursor mass tolerance, ±20 ppm product mass tolerance. Carbamidomethylation of cysteines and iTRAQ4 or TMT10 full labeling of lysines and peptide n-termini were set as fixed modifications. Allowed variable modifications were oxidation of methionine (M), acetyl (ProtN-term), and deamidated (N), with a precursor MH+ shift range of -18 to 64 Da. Identities interpreted for individual spectra were automatically designated as valid by optimizing score and delta rank1-rank2 score thresholds separately for each precursor charge state in each LC-MS/MS while allowing a maximum target-decoy-based false discovery rate (FDR) of 1.0% at the spectrum level. Identified peptides were organized into protein groups and subgroups (isoforms and family members) with Spectrum Mill’s subgroup specific option enabled, so that peptides shared between subgroups are ignored for quantitation. TMT10 reporter ion intensities were corrected for isotopic impurities using Spectrum Mill’s afRICA correction method and correction factors obtained from the reagent manufacturer’s certificate of analysis. For quantitation at the peptide spectrum match level, log2 fold change (FC) reporter ion intensity ratios were calculated for each IP replicate. To obtain protein-level log2 FC values between index protein vs. control for each IP replicate, the median ratio was calculated from all peptide spectrum matches of subgroup specific peptides assigned to each protein subgroup.Genoppi analysisFor each IP-MS experiment, starting with the protein-level quantification report generated by Spectrum Mill, we applied median normalization to protein log2 FC values between index protein vs. control IPs for each replicate, and then performed downstream analyses using Genoppi24 (v1.0). Additional data processing was performed for two experiments pre-Genoppi: for ‘CACNA1C_wk7’ the log2 FC values for one replicate was right-shifted by a small value (0.005) to facilitate direct comparisons with all other volcano plots without changing the results; for ‘TCF4_wk1’ additional contaminant proteins found in the CRAPome73 database were removed. The Genoppi analyses include: i] calculating Pearson’s correlation of log2 FC values between IP replicates, ii] calculating average log2 FC, and corresponding P-value and FDR, for each protein across IP replicates using a two-tailed one-sample moderated t-test from the limma74 R package, iii] identifying statistically significant (log2 FC > 0 and FDR ≤0.1) index protein interactors (i.e., proteins with significantly higher abundance in the index protein IPs compared to the controls), iv] defining other non-significant (log2 FC ≤ 0 or FDR >0.1) proteins to be the ‘non-interactors’ (to serve as proxy for background proteome in enrichment analyses), and v] assessing overlap enrichment between the identified interactors and known interactors from InWeb25 (compared to the non-interactors) using a one-tailed hypergeometric test. We performed quality control of each IP-MS experiment using two criteria: i] the log2 FC correlation between replicates must be >0.5, and ii] the index protein itself must be significant (log2 FC > 0 and FDR ≤0.1). Experiments that failed to meet these criteria were excluded from further analysis; those that passed QC are summarized in Table S5, with their analysis results provided in Table S6.Co-expression analysisWe assessed pairwise co-expression between each index gene and all other protein-coding genes using data from four independent studies, including: i] Stickels et al.29: spatial transcriptomics in mouse neocortex (Puck_190921_19.digital_expression.txt.gz retrieved from: https://singlecell.broadinstitute.org/single_cell/study/SCP815/highly-sensitive-spatial-transcriptomics-at-near-cellular-resolution-with-slide-seqv2#study-download), ii] Maynard et al.28: spatial transcriptomics in human dorsolateral prefrontal cortex (count matrix retrieved from spatialLIBD R package: https://github.com/LieberInstitute/HumanPilot), iii] Velmeshev et al.30: single-cell RNA-seq in human cortex (rawMatrix.zip retrieved from: https://cells.ucsc.edu/?ds=autism), and iv] BrainSpan: bulk RNA-seq across human brain regions and developmental stages (“RNA-Seq Gencode v10 summarized to genes” dataset retrieved from: https://www.brainspan.org/static/download.html). We used two different methods for estimating co-expression to account for different statistical properties in these datasets. For the spatial transcriptomic datasets with very sparse gene expression matrices, we reasoned that the binary presence/absence of genes across physical locations would be the most informative, and therefore performed one-tailed Fisher’s exact tests to calculate the significance of co-occurrence for each gene pair across locations. For the single-cell RNA-seq and BrainSpan datasets, we calculated a proportionality metric, ⍴, for each gene pair using the propr R package58 (v4.2.6); this metric is analogous to conventional correlation measures but has been shown to be better at capturing functional associations between genes in RNA-seq data.75 After calculating either the Fisher’s exact P-values or the proportionality ⍴ values for all gene pairs involving each index gene, we then performed rank-based inverse normal transformation to convert the values into co-expression Z-scores (where a positive score indicates that a gene has higher than average co-expression with the index gene compared to the rest of the genome).We performed two-tailed Wilcoxon rank-sum tests to assess if the co-expression Z-scores between index genes and their interactors are significantly different from the scores between the index genes and other gene groups including: i] non-interactors detected in IP-MS, ii] known interactors from InWeb, and iii] all protein-coding genes.Consolidating IP-MS datasets into interaction networksWe consolidated IP-MS datasets for the same index proteins into index-protein-specific networks; datasets derived from the same time points into time point-specific networks; and all datasets into an all combined network. For each network, we defined ‘interactors’ as proteins that show up as significant interactors in ≥1 source IPs contributing to the network, and the matching ‘non-interactors’ as proteins that show up as non-interactors in ≥1 source IPs but never as interactors in these IPs. In addition, we defined ‘stringent interactors’ as interactors that never show up as non-interactors in the source IPs. Finally, we removed proteins whose HGNC gene symbols could not be mapped to Ensembl76 (GRCh37.p13) genomic positions and the index proteins for the source IPs from all 3 lists. The exclusion of index proteins ensured that enrichment signals in downstream analyses could not be driven by the index proteins themselves. Table S9 lists the interactors, non-interactors, and stringent interactors associated with each network.SynGO gene set analysisSynGO analysis of interactor genes in the all combined network was performed using the SynGO web browser31 (dataset version: 20210225; https://syngoportal.org). The SynGO ‘Biological Processes’ annotations and the ‘brain expressed’ background set were used in the analysis.Common variant enrichment analysis using MAGMATo perform common variant risk enrichment analysis for each interaction network using MAGMA32 (v1.09b), PGC GWAS summary statistics (https://www.med.unc.edu/pgc/download-results/) were obtained for schizophrenia (33,640 cases and 43,456 controls of EUR ancestry3; 22,778 cases and 35,362 controls of EAS ancestry8), ADHD (19,099 cases and 34,194 controls of EUR ancestry33), ASD (18,382 cases and 27,969 controls of EUR ancestry34), BIP (41,917 cases and 371,549 controls of EUR ancestry35), and MDD (170,756 cases and 329,443 controls of EUR ancestry36). GWAS summary statistics for height were obtained from the Neale Lab UK Biobank GWAS (round 2; 361,194 individuals of EUR ancestry37; https://www.nealelab.is/uk-biobank/) and Biobank Japan (159,095 individuals of EAS ancestry38; http://jenger.riken.jp/en/result). For each GWAS dataset, we first annotated variants to genes using the Human reference genome (GRCh37/hg19) with a flanking gene region of ±250kb. Variants on sex chromosomes, with minor allele frequency (MAF) ≤ 0.05 in the study, or within the MHC region (chr6:22.5M-33.5M) were excluded from analysis. Next, gene-based P-values were computed using the SNP-wise Mean model in MAGMA. LD was estimated from the 1000 Genomes Project77 phase 3 EUR or EAS panel to match the ancestry of each study. Competitive tests were used for gene-set comparison analysis between stringent interactors and non-interactors in each interaction network. Specifically, a linear regression model was built by MAGMA to test if genes within the interactor gene-set are more strongly associated with the phenotype of interest compared to the non-interactor gene-set. Inverse-variance weighted fixed-effect meta-analysis78 was used to combine results across EUR and EAS ancestries for schizophrenia and height, and a one-tailed P-value was calculated from the meta-analyzed Z-score.Common variant enrichment analysis using the GRS methodTo complement the MAGMA analysis, we used an alternative genetic risk score (GRS) method and individual-level genotypes from the schizophrenia GWAS study cohorts (24,764 cases and 30,655 controls of EUR ancestry; 8,960 cases and 8,284 controls of EAS ancestry) to assess genetic risk enrichment in the interaction networks. IRB approvals for accessing the individual-level data were obtained from the PGC and Stanley Global Asia Initiatives (Data S1-S3). For each study cohort, we removed variants in the MHC region and variants with MAF ≤0.05 or low imputation quality (INFO ≤0.8), and then mapped the remaining variants to the stringent interactors or non-interactors associated with each network with a flanking gene region of ±250kb. Next, we used PLINK59 (v1.9) and the in-sample LD to clump variants within the interactor or non-interactor gene-sets into independent association signals. The clumping was performed with a window of 250kb, R2> 0.2, and a P-value threshold of 1. The out-of-sample GRS for each individual and each gene-set was then calculated as ∑ log(OR) ∗G, in whichOR is the estimated odds ratio from the leave-one-out GWAS meta-analysis for the clumped index variants within the gene-set and G is the genotype dosage. To test if the stringent interactors in a network contribute more genetic risk compared to the non-interactors, we fit a linear regression model: GRSij
= Pi
+ Ij
+ Pi ∗ Ij
+ Covi, where i and j denote individual index and gene index, respectively, P is the case/control status, I is interactor/non-interactor status, P ∗I is an interaction term, and Cov includes 10 genetic principal components. We tested whether the P ∗I interaction term is significantly larger than zero, which captures whether the difference of GRS between cases and controls calculated in the interactor gene-set is significantly larger than the non-interactor gene-set. Inverse-variance weighted fixed-effect meta-analysis was used to combine the test statistics for the interaction term across study cohorts and ancestries, and a one-tailed P-value was calculated from the meta-analyzed Z-score. Finally, as a negative control, we performed an analogous GRS analysis for height; in this case, the GRS was calculated as ∑ log(beta) ∗G, where beta is the effect size estimate from the EUR or EAS height GWAS for clumped index variants.Rare variant enrichment analysisGene-based association statistics derived from exome sequencing data were obtained for schizophrenia (P-values in ‘P meta’ column from Table S5 of Singh et al.9), ASD (Q-values in ‘qval_dnccPTV’ column from Table S2 of Satterstrom et al.40), and DD (P-values in ‘denovoWEST_p_full’ column from Table S2 of Kaplanis et al.41). For each phenotype and each network, a one-tailed KS test was used to test whether the stringent interactors in the network have more significant association scores (i.e., smaller P-values or Q-values) compared to the non-interactors.Transcriptional perturbation enrichment analysisDifferentially expressed genes (DEGs) between schizophrenia patients vs. controls identified in 20 prefrontal cortex cell types were retrieved from Table S6 of Ruzicka et al.17 The up- and down-regulated DEGs were analyzed as a joint set in the primary analysis, and then as separate sets in the secondary follow-up analyses. One-tailed P-values were calculated using a hypergeometric distribution to assess the overlap enrichment between interactors in our networks and the cell-type-specific DEGs. For each hypergeometric test, the ‘population’ was defined as all stringent interactor or non-interactor genes associated with a network and ‘success in population’ was defined as the stringent interactors in the network. The ‘sample’ contained DEGs in a celltype that were found in the population and ‘success in sample’ was the overlap between the interactors and the DEGs.Social Manhattan plotGenes mapped to 287 genome-wide significant regions in the combined discovery-replication meta-analysis of the PGC schizophrenia GWAS (phase 3) were retrieved from Table S3 of the paper.10 We excluded non-coding genes and genes in the MHC region, and obtained the genomic positions of the remaining genes from Ensembl76 (GRCh37.p13). We additionally included SYNGAP1 by mapping it to the only MHC region SNP (rs140365013) included in the GWAS meta-analysis. Next, we intersected these GWAS genes with our PPI data to identify the subset of GWAS genes that are either an index gene or an interactor of an index gene. We generated a Manhattan plot for these genes using the GWAS P-values of their associated index SNPs and plotted links between the genes to indicate observed interactions in our 3. In addition, we retrieved the list of GWAS genes prioritized by FINEMAP or SMR analysis from Table S12 of the PGC paper and highlighted their overlap with genes prioritized by our data in the plot. For ease of visualization, GWAS P-values were capped at P = 1e-25 and genes in the PCDHA@ gene cluster were collapsed into one gene in the plot.HCN1 network drug target queryTo identify existing drugs targeting genes in the combined HCN1 network, we performed a batch query using the Open Targets Platform79 (https://platform.opentargets.org). For each resulting drug-target entry, the corresponding drug mechanism information was retrieved from ChEMBL80 (https://www.ebi.ac.uk/chembl/g/#browse/mechanisms_of_action).

Quantification and statistical analysis

Refining geneticdata to identify index genesThree-step procedure for selecting index genesStarting with 125 independent autosomal SNPs that reached genome-wide significance in the combined discovery-replication meta-analysis of the PGC schizophrenia GWAS3 (phase 2), Ricopili57 (https://data.broadinstitute.org/mpg/ricopili/) was used to map the 124 non-MHC (major histocompatibility complex) region SNPs to 445 genes (Set 1; Table S1) in linkage disequilibrium (LD) loci, whose boundaries were defined by SNPs in LD (r2> 0.6) with the index SNPs, ±50kb on either end. Next, we excluded SNPs in intergenic regions or in loci with multiple protein-coding genes, resulting in a list of 40 SNPs pointing to 37 unique protein-coding genes (Set 2) in single-gene loci. We further refined the 37 genes into a set of 10 genes (Set 3) based on strong orthogonal evidence supporting their involvement in psychiatric diseases (see below). We also included SYNGAP1 in the MHC region (which was excluded from the SNP-to-gene mapping process due to its LD complexity) in all 3 sets based on strong orthogonal evidence. In total, we identified 11 high-confidence schizophrenia risk genes and considered their encoded proteins as index proteins in the proteomic experiments.Orthogonal evidence for the index genesThe index genes were selected based on several types of orthogonal genetic or functional evidence, including: i] high-density genotyping experiments of individual genes (ZNF804A),60 ii] sequencing or copy number variant studies linking genes to autism spectrum disorders or neurodevelopmental delay (CACNA1C, CUL3, CSMD1, GRIN2A, SATB2 SYNGAP1, TCF4),39,61,62,63,64,65,66,67,68 iii] strong functional evidence supporting that the gene in question is causal (HCN1),69 and iv] multiple subunits from the same protein complex are encoded by genes in different single-gene loci linked to psychiatric diseases (CACNA1C, CACNB2, RIMS1).3,70,71 In addition, we annotated the 10 genes prioritized from single-gene GWAS loci using Hi-C chromatin interaction data from the developing human brain,72which may point to long-range regulation of genes outside of the loci in the context of schizophrenia; only three of these genes (CUL3, RIMS1, SATB2) lie in loci that exclusively interacted with long-range genes outside of the loci (Table S1).

Refining geneticdata to identify index genes

Three-step procedure for selecting index genesStarting with 125 independent autosomal SNPs that reached genome-wide significance in the combined discovery-replication meta-analysis of the PGC schizophrenia GWAS3 (phase 2), Ricopili57 (https://data.broadinstitute.org/mpg/ricopili/) was used to map the 124 non-MHC (major histocompatibility complex) region SNPs to 445 genes (Set 1; Table S1) in linkage disequilibrium (LD) loci, whose boundaries were defined by SNPs in LD (r2> 0.6) with the index SNPs, ±50kb on either end. Next, we excluded SNPs in intergenic regions or in loci with multiple protein-coding genes, resulting in a list of 40 SNPs pointing to 37 unique protein-coding genes (Set 2) in single-gene loci. We further refined the 37 genes into a set of 10 genes (Set 3) based on strong orthogonal evidence supporting their involvement in psychiatric diseases (see below). We also included SYNGAP1 in the MHC region (which was excluded from the SNP-to-gene mapping process due to its LD complexity) in all 3 sets based on strong orthogonal evidence. In total, we identified 11 high-confidence schizophrenia risk genes and considered their encoded proteins as index proteins in the proteomic experiments.

Three-step procedure for selecting index genes

Starting with 125 independent autosomal SNPs that reached genome-wide significance in the combined discovery-replication meta-analysis of the PGC schizophrenia GWAS3 (phase 2), Ricopili57 (https://data.broadinstitute.org/mpg/ricopili/) was used to map the 124 non-MHC (major histocompatibility complex) region SNPs to 445 genes (Set 1; Table S1) in linkage disequilibrium (LD) loci, whose boundaries were defined by SNPs in LD (r2> 0.6) with the index SNPs, ±50kb on either end. Next, we excluded SNPs in intergenic regions or in loci with multiple protein-coding genes, resulting in a list of 40 SNPs pointing to 37 unique protein-coding genes (Set 2) in single-gene loci. We further refined the 37 genes into a set of 10 genes (Set 3) based on strong orthogonal evidence supporting their involvement in psychiatric diseases (see below). We also included SYNGAP1 in the MHC region (which was excluded from the SNP-to-gene mapping process due to its LD complexity) in all 3 sets based on strong orthogonal evidence. In total, we identified 11 high-confidence schizophrenia risk genes and considered their encoded proteins as index proteins in the proteomic experiments.

Orthogonal evidence for the index genesThe index genes were selected based on several types of orthogonal genetic or functional evidence, including: i] high-density genotyping experiments of individual genes (ZNF804A),60 ii] sequencing or copy number variant studies linking genes to autism spectrum disorders or neurodevelopmental delay (CACNA1C, CUL3, CSMD1, GRIN2A, SATB2 SYNGAP1, TCF4),39,61,62,63,64,65,66,67,68 iii] strong functional evidence supporting that the gene in question is causal (HCN1),69 and iv] multiple subunits from the same protein complex are encoded by genes in different single-gene loci linked to psychiatric diseases (CACNA1C, CACNB2, RIMS1).3,70,71 In addition, we annotated the 10 genes prioritized from single-gene GWAS loci using Hi-C chromatin interaction data from the developing human brain,72which may point to long-range regulation of genes outside of the loci in the context of schizophrenia; only three of these genes (CUL3, RIMS1, SATB2) lie in loci that exclusively interacted with long-range genes outside of the loci (Table S1).

Orthogonal evidence for the index genes

The index genes were selected based on several types of orthogonal genetic or functional evidence, including: i] high-density genotyping experiments of individual genes (ZNF804A),60 ii] sequencing or copy number variant studies linking genes to autism spectrum disorders or neurodevelopmental delay (CACNA1C, CUL3, CSMD1, GRIN2A, SATB2 SYNGAP1, TCF4),39,61,62,63,64,65,66,67,68 iii] strong functional evidence supporting that the gene in question is causal (HCN1),69 and iv] multiple subunits from the same protein complex are encoded by genes in different single-gene loci linked to psychiatric diseases (CACNA1C, CACNB2, RIMS1).3,70,71 In addition, we annotated the 10 genes prioritized from single-gene GWAS loci using Hi-C chromatin interaction data from the developing human brain,72which may point to long-range regulation of genes outside of the loci in the context of schizophrenia; only three of these genes (CUL3, RIMS1, SATB2) lie in loci that exclusively interacted with long-range genes outside of the loci (Table S1).

pLI score enrichment analysisWe extracted pLI scores from the gnomAD22 (v2.1.1) ‘pLoF Metrics by Gene TSV’ dataset. We performed one-tailed Kolmogorov-Smirnov (KS) tests to assess whether various gene sets are enriched for high pLI scores compared to other genes in the genome. The tested gene sets include: i] SCHEMA9 genes with FDR <3.7e-3 (exome-wide significance) and 0.5, ii] Sets 1-3 defined by our index gene selection steps, and iii] genes in PGC schizophrenia GWAS (phase 3) loci10 and subsets of these genes prioritized by FINEMAP/SMR analysis and/or our interaction data (see social manhattan plot). In addition, we performed one-tailed KS tests to compare the pLI scores of several gene sets that are subsets and supersets of each other, and two-tailed KS tests to compare a few disjoint sets. Table S2 summarizes all the comparisons performed. Finally, we also performed one-tailed KS tests to assess whether the stringent interactors in our interaction networks (Table S9) have enriched pLI scores compared to the non-interactors linked to each network.

pLI score enrichment analysis

We extracted pLI scores from the gnomAD22 (v2.1.1) ‘pLoF Metrics by Gene TSV’ dataset. We performed one-tailed Kolmogorov-Smirnov (KS) tests to assess whether various gene sets are enriched for high pLI scores compared to other genes in the genome. The tested gene sets include: i] SCHEMA9 genes with FDR <3.7e-3 (exome-wide significance) and 0.5, ii] Sets 1-3 defined by our index gene selection steps, and iii] genes in PGC schizophrenia GWAS (phase 3) loci10 and subsets of these genes prioritized by FINEMAP/SMR analysis and/or our interaction data (see social manhattan plot). In addition, we performed one-tailed KS tests to compare the pLI scores of several gene sets that are subsets and supersets of each other, and two-tailed KS tests to compare a few disjoint sets. Table S2 summarizes all the comparisons performed. Finally, we also performed one-tailed KS tests to assess whether the stringent interactors in our interaction networks (Table S9) have enriched pLI scores compared to the non-interactors linked to each network.

BrainSpan expression profilesWe obtained gene expression data in four distinct parts of the frontal cortex (dorsolateral prefrontal cortex [DFC], medial prefrontal cortex [MFC], ventrolateral prefrontal cortex [VFC], orbital frontal cortex [OFC]) across 10 developmental stages from the BrainSpan (https://www.brainspan.org) exon microarray dataset.23 At each developmental stage, we calculated the median and standard error of the expression values for various genes or gene sets including: i] SCHEMA genes with FDR <3.7e-3, 0.05, 0.25, and 0.5, ii] Sets 1-3 defined by our index gene selection steps, iii] index and interactor genes in our combined network of all IPs (Table S9), iv] genes in PGC schizophrenia GWAS (phase 3) loci and subsets of these genes prioritized by FINEMAP/SMR analysis and/or our interaction data (see social manhattan plot), and v] random genes sampled from the BrainSpan dataset for comparison against the other gene sets. Table S3 provides more detailed summary statistics for all gene sets.

BrainSpan expression profiles

We obtained gene expression data in four distinct parts of the frontal cortex (dorsolateral prefrontal cortex [DFC], medial prefrontal cortex [MFC], ventrolateral prefrontal cortex [VFC], orbital frontal cortex [OFC]) across 10 developmental stages from the BrainSpan (https://www.brainspan.org) exon microarray dataset.23 At each developmental stage, we calculated the median and standard error of the expression values for various genes or gene sets including: i] SCHEMA genes with FDR <3.7e-3, 0.05, 0.25, and 0.5, ii] Sets 1-3 defined by our index gene selection steps, iii] index and interactor genes in our combined network of all IPs (Table S9), iv] genes in PGC schizophrenia GWAS (phase 3) loci and subsets of these genes prioritized by FINEMAP/SMR analysis and/or our interaction data (see social manhattan plot), and v] random genes sampled from the BrainSpan dataset for comparison against the other gene sets. Table S3 provides more detailed summary statistics for all gene sets.

IP-MS data analysisSpectrum Mill analysisAll mass spectra were processed using Spectrum Mill (v7.0; https://proteomics.broadinstitute.org). For peptide identification, MS/MS spectra were searched against a sequence database for the UniProt human reference proteome, including isoforms, with a set of common laboratory contaminant proteins appended (2017: 65,068 entries, or 2014: 59,079 entries) as indicated in Table S5. Search parameters included: ESI-QEXACTIVE-HCD scoring parameters, trypsin enzyme specificity with a maximum of two missed cleavages, 40% minimum matched peak intensity, ±20 ppm precursor mass tolerance, ±20 ppm product mass tolerance. Carbamidomethylation of cysteines and iTRAQ4 or TMT10 full labeling of lysines and peptide n-termini were set as fixed modifications. Allowed variable modifications were oxidation of methionine (M), acetyl (ProtN-term), and deamidated (N), with a precursor MH+ shift range of -18 to 64 Da. Identities interpreted for individual spectra were automatically designated as valid by optimizing score and delta rank1-rank2 score thresholds separately for each precursor charge state in each LC-MS/MS while allowing a maximum target-decoy-based false discovery rate (FDR) of 1.0% at the spectrum level. Identified peptides were organized into protein groups and subgroups (isoforms and family members) with Spectrum Mill’s subgroup specific option enabled, so that peptides shared between subgroups are ignored for quantitation. TMT10 reporter ion intensities were corrected for isotopic impurities using Spectrum Mill’s afRICA correction method and correction factors obtained from the reagent manufacturer’s certificate of analysis. For quantitation at the peptide spectrum match level, log2 fold change (FC) reporter ion intensity ratios were calculated for each IP replicate. To obtain protein-level log2 FC values between index protein vs. control for each IP replicate, the median ratio was calculated from all peptide spectrum matches of subgroup specific peptides assigned to each protein subgroup.Genoppi analysisFor each IP-MS experiment, starting with the protein-level quantification report generated by Spectrum Mill, we applied median normalization to protein log2 FC values between index protein vs. control IPs for each replicate, and then performed downstream analyses using Genoppi24 (v1.0). Additional data processing was performed for two experiments pre-Genoppi: for ‘CACNA1C_wk7’ the log2 FC values for one replicate was right-shifted by a small value (0.005) to facilitate direct comparisons with all other volcano plots without changing the results; for ‘TCF4_wk1’ additional contaminant proteins found in the CRAPome73 database were removed. The Genoppi analyses include: i] calculating Pearson’s correlation of log2 FC values between IP replicates, ii] calculating average log2 FC, and corresponding P-value and FDR, for each protein across IP replicates using a two-tailed one-sample moderated t-test from the limma74 R package, iii] identifying statistically significant (log2 FC > 0 and FDR ≤0.1) index protein interactors (i.e., proteins with significantly higher abundance in the index protein IPs compared to the controls), iv] defining other non-significant (log2 FC ≤ 0 or FDR >0.1) proteins to be the ‘non-interactors’ (to serve as proxy for background proteome in enrichment analyses), and v] assessing overlap enrichment between the identified interactors and known interactors from InWeb25 (compared to the non-interactors) using a one-tailed hypergeometric test. We performed quality control of each IP-MS experiment using two criteria: i] the log2 FC correlation between replicates must be >0.5, and ii] the index protein itself must be significant (log2 FC > 0 and FDR ≤0.1). Experiments that failed to meet these criteria were excluded from further analysis; those that passed QC are summarized in Table S5, with their analysis results provided in Table S6.

IP-MS data analysis

Spectrum Mill analysisAll mass spectra were processed using Spectrum Mill (v7.0; https://proteomics.broadinstitute.org). For peptide identification, MS/MS spectra were searched against a sequence database for the UniProt human reference proteome, including isoforms, with a set of common laboratory contaminant proteins appended (2017: 65,068 entries, or 2014: 59,079 entries) as indicated in Table S5. Search parameters included: ESI-QEXACTIVE-HCD scoring parameters, trypsin enzyme specificity with a maximum of two missed cleavages, 40% minimum matched peak intensity, ±20 ppm precursor mass tolerance, ±20 ppm product mass tolerance. Carbamidomethylation of cysteines and iTRAQ4 or TMT10 full labeling of lysines and peptide n-termini were set as fixed modifications. Allowed variable modifications were oxidation of methionine (M), acetyl (ProtN-term), and deamidated (N), with a precursor MH+ shift range of -18 to 64 Da. Identities interpreted for individual spectra were automatically designated as valid by optimizing score and delta rank1-rank2 score thresholds separately for each precursor charge state in each LC-MS/MS while allowing a maximum target-decoy-based false discovery rate (FDR) of 1.0% at the spectrum level. Identified peptides were organized into protein groups and subgroups (isoforms and family members) with Spectrum Mill’s subgroup specific option enabled, so that peptides shared between subgroups are ignored for quantitation. TMT10 reporter ion intensities were corrected for isotopic impurities using Spectrum Mill’s afRICA correction method and correction factors obtained from the reagent manufacturer’s certificate of analysis. For quantitation at the peptide spectrum match level, log2 fold change (FC) reporter ion intensity ratios were calculated for each IP replicate. To obtain protein-level log2 FC values between index protein vs. control for each IP replicate, the median ratio was calculated from all peptide spectrum matches of subgroup specific peptides assigned to each protein subgroup.

Spectrum Mill analysis

All mass spectra were processed using Spectrum Mill (v7.0; https://proteomics.broadinstitute.org). For peptide identification, MS/MS spectra were searched against a sequence database for the UniProt human reference proteome, including isoforms, with a set of common laboratory contaminant proteins appended (2017: 65,068 entries, or 2014: 59,079 entries) as indicated in Table S5. Search parameters included: ESI-QEXACTIVE-HCD scoring parameters, trypsin enzyme specificity with a maximum of two missed cleavages, 40% minimum matched peak intensity, ±20 ppm precursor mass tolerance, ±20 ppm product mass tolerance. Carbamidomethylation of cysteines and iTRAQ4 or TMT10 full labeling of lysines and peptide n-termini were set as fixed modifications. Allowed variable modifications were oxidation of methionine (M), acetyl (ProtN-term), and deamidated (N), with a precursor MH+ shift range of -18 to 64 Da. Identities interpreted for individual spectra were automatically designated as valid by optimizing score and delta rank1-rank2 score thresholds separately for each precursor charge state in each LC-MS/MS while allowing a maximum target-decoy-based false discovery rate (FDR) of 1.0% at the spectrum level. Identified peptides were organized into protein groups and subgroups (isoforms and family members) with Spectrum Mill’s subgroup specific option enabled, so that peptides shared between subgroups are ignored for quantitation. TMT10 reporter ion intensities were corrected for isotopic impurities using Spectrum Mill’s afRICA correction method and correction factors obtained from the reagent manufacturer’s certificate of analysis. For quantitation at the peptide spectrum match level, log2 fold change (FC) reporter ion intensity ratios were calculated for each IP replicate. To obtain protein-level log2 FC values between index protein vs. control for each IP replicate, the median ratio was calculated from all peptide spectrum matches of subgroup specific peptides assigned to each protein subgroup.

Genoppi analysisFor each IP-MS experiment, starting with the protein-level quantification report generated by Spectrum Mill, we applied median normalization to protein log2 FC values between index protein vs. control IPs for each replicate, and then performed downstream analyses using Genoppi24 (v1.0). Additional data processing was performed for two experiments pre-Genoppi: for ‘CACNA1C_wk7’ the log2 FC values for one replicate was right-shifted by a small value (0.005) to facilitate direct comparisons with all other volcano plots without changing the results; for ‘TCF4_wk1’ additional contaminant proteins found in the CRAPome73 database were removed. The Genoppi analyses include: i] calculating Pearson’s correlation of log2 FC values between IP replicates, ii] calculating average log2 FC, and corresponding P-value and FDR, for each protein across IP replicates using a two-tailed one-sample moderated t-test from the limma74 R package, iii] identifying statistically significant (log2 FC > 0 and FDR ≤0.1) index protein interactors (i.e., proteins with significantly higher abundance in the index protein IPs compared to the controls), iv] defining other non-significant (log2 FC ≤ 0 or FDR >0.1) proteins to be the ‘non-interactors’ (to serve as proxy for background proteome in enrichment analyses), and v] assessing overlap enrichment between the identified interactors and known interactors from InWeb25 (compared to the non-interactors) using a one-tailed hypergeometric test. We performed quality control of each IP-MS experiment using two criteria: i] the log2 FC correlation between replicates must be >0.5, and ii] the index protein itself must be significant (log2 FC > 0 and FDR ≤0.1). Experiments that failed to meet these criteria were excluded from further analysis; those that passed QC are summarized in Table S5, with their analysis results provided in Table S6.

Genoppi analysis

For each IP-MS experiment, starting with the protein-level quantification report generated by Spectrum Mill, we applied median normalization to protein log2 FC values between index protein vs. control IPs for each replicate, and then performed downstream analyses using Genoppi24 (v1.0). Additional data processing was performed for two experiments pre-Genoppi: for ‘CACNA1C_wk7’ the log2 FC values for one replicate was right-shifted by a small value (0.005) to facilitate direct comparisons with all other volcano plots without changing the results; for ‘TCF4_wk1’ additional contaminant proteins found in the CRAPome73 database were removed. The Genoppi analyses include: i] calculating Pearson’s correlation of log2 FC values between IP replicates, ii] calculating average log2 FC, and corresponding P-value and FDR, for each protein across IP replicates using a two-tailed one-sample moderated t-test from the limma74 R package, iii] identifying statistically significant (log2 FC > 0 and FDR ≤0.1) index protein interactors (i.e., proteins with significantly higher abundance in the index protein IPs compared to the controls), iv] defining other non-significant (log2 FC ≤ 0 or FDR >0.1) proteins to be the ‘non-interactors’ (to serve as proxy for background proteome in enrichment analyses), and v] assessing overlap enrichment between the identified interactors and known interactors from InWeb25 (compared to the non-interactors) using a one-tailed hypergeometric test. We performed quality control of each IP-MS experiment using two criteria: i] the log2 FC correlation between replicates must be >0.5, and ii] the index protein itself must be significant (log2 FC > 0 and FDR ≤0.1). Experiments that failed to meet these criteria were excluded from further analysis; those that passed QC are summarized in Table S5, with their analysis results provided in Table S6.

Co-expression analysisWe assessed pairwise co-expression between each index gene and all other protein-coding genes using data from four independent studies, including: i] Stickels et al.29: spatial transcriptomics in mouse neocortex (Puck_190921_19.digital_expression.txt.gz retrieved from: https://singlecell.broadinstitute.org/single_cell/study/SCP815/highly-sensitive-spatial-transcriptomics-at-near-cellular-resolution-with-slide-seqv2#study-download), ii] Maynard et al.28: spatial transcriptomics in human dorsolateral prefrontal cortex (count matrix retrieved from spatialLIBD R package: https://github.com/LieberInstitute/HumanPilot), iii] Velmeshev et al.30: single-cell RNA-seq in human cortex (rawMatrix.zip retrieved from: https://cells.ucsc.edu/?ds=autism), and iv] BrainSpan: bulk RNA-seq across human brain regions and developmental stages (“RNA-Seq Gencode v10 summarized to genes” dataset retrieved from: https://www.brainspan.org/static/download.html). We used two different methods for estimating co-expression to account for different statistical properties in these datasets. For the spatial transcriptomic datasets with very sparse gene expression matrices, we reasoned that the binary presence/absence of genes across physical locations would be the most informative, and therefore performed one-tailed Fisher’s exact tests to calculate the significance of co-occurrence for each gene pair across locations. For the single-cell RNA-seq and BrainSpan datasets, we calculated a proportionality metric, ⍴, for each gene pair using the propr R package58 (v4.2.6); this metric is analogous to conventional correlation measures but has been shown to be better at capturing functional associations between genes in RNA-seq data.75 After calculating either the Fisher’s exact P-values or the proportionality ⍴ values for all gene pairs involving each index gene, we then performed rank-based inverse normal transformation to convert the values into co-expression Z-scores (where a positive score indicates that a gene has higher than average co-expression with the index gene compared to the rest of the genome).We performed two-tailed Wilcoxon rank-sum tests to assess if the co-expression Z-scores between index genes and their interactors are significantly different from the scores between the index genes and other gene groups including: i] non-interactors detected in IP-MS, ii] known interactors from InWeb, and iii] all protein-coding genes.

Co-expression analysis

We assessed pairwise co-expression between each index gene and all other protein-coding genes using data from four independent studies, including: i] Stickels et al.29: spatial transcriptomics in mouse neocortex (Puck_190921_19.digital_expression.txt.gz retrieved from: https://singlecell.broadinstitute.org/single_cell/study/SCP815/highly-sensitive-spatial-transcriptomics-at-near-cellular-resolution-with-slide-seqv2#study-download), ii] Maynard et al.28: spatial transcriptomics in human dorsolateral prefrontal cortex (count matrix retrieved from spatialLIBD R package: https://github.com/LieberInstitute/HumanPilot), iii] Velmeshev et al.30: single-cell RNA-seq in human cortex (rawMatrix.zip retrieved from: https://cells.ucsc.edu/?ds=autism), and iv] BrainSpan: bulk RNA-seq across human brain regions and developmental stages (“RNA-Seq Gencode v10 summarized to genes” dataset retrieved from: https://www.brainspan.org/static/download.html). We used two different methods for estimating co-expression to account for different statistical properties in these datasets. For the spatial transcriptomic datasets with very sparse gene expression matrices, we reasoned that the binary presence/absence of genes across physical locations would be the most informative, and therefore performed one-tailed Fisher’s exact tests to calculate the significance of co-occurrence for each gene pair across locations. For the single-cell RNA-seq and BrainSpan datasets, we calculated a proportionality metric, ⍴, for each gene pair using the propr R package58 (v4.2.6); this metric is analogous to conventional correlation measures but has been shown to be better at capturing functional associations between genes in RNA-seq data.75 After calculating either the Fisher’s exact P-values or the proportionality ⍴ values for all gene pairs involving each index gene, we then performed rank-based inverse normal transformation to convert the values into co-expression Z-scores (where a positive score indicates that a gene has higher than average co-expression with the index gene compared to the rest of the genome).

We performed two-tailed Wilcoxon rank-sum tests to assess if the co-expression Z-scores between index genes and their interactors are significantly different from the scores between the index genes and other gene groups including: i] non-interactors detected in IP-MS, ii] known interactors from InWeb, and iii] all protein-coding genes.

Consolidating IP-MS datasets into interaction networksWe consolidated IP-MS datasets for the same index proteins into index-protein-specific networks; datasets derived from the same time points into time point-specific networks; and all datasets into an all combined network. For each network, we defined ‘interactors’ as proteins that show up as significant interactors in ≥1 source IPs contributing to the network, and the matching ‘non-interactors’ as proteins that show up as non-interactors in ≥1 source IPs but never as interactors in these IPs. In addition, we defined ‘stringent interactors’ as interactors that never show up as non-interactors in the source IPs. Finally, we removed proteins whose HGNC gene symbols could not be mapped to Ensembl76 (GRCh37.p13) genomic positions and the index proteins for the source IPs from all 3 lists. The exclusion of index proteins ensured that enrichment signals in downstream analyses could not be driven by the index proteins themselves. Table S9 lists the interactors, non-interactors, and stringent interactors associated with each network.

Consolidating IP-MS datasets into interaction networks

We consolidated IP-MS datasets for the same index proteins into index-protein-specific networks; datasets derived from the same time points into time point-specific networks; and all datasets into an all combined network. For each network, we defined ‘interactors’ as proteins that show up as significant interactors in ≥1 source IPs contributing to the network, and the matching ‘non-interactors’ as proteins that show up as non-interactors in ≥1 source IPs but never as interactors in these IPs. In addition, we defined ‘stringent interactors’ as interactors that never show up as non-interactors in the source IPs. Finally, we removed proteins whose HGNC gene symbols could not be mapped to Ensembl76 (GRCh37.p13) genomic positions and the index proteins for the source IPs from all 3 lists. The exclusion of index proteins ensured that enrichment signals in downstream analyses could not be driven by the index proteins themselves. Table S9 lists the interactors, non-interactors, and stringent interactors associated with each network.

SynGO gene set analysisSynGO analysis of interactor genes in the all combined network was performed using the SynGO web browser31 (dataset version: 20210225; https://syngoportal.org). The SynGO ‘Biological Processes’ annotations and the ‘brain expressed’ background set were used in the analysis.

SynGO gene set analysis

SynGO analysis of interactor genes in the all combined network was performed using the SynGO web browser31 (dataset version: 20210225; https://syngoportal.org). The SynGO ‘Biological Processes’ annotations and the ‘brain expressed’ background set were used in the analysis.

Common variant enrichment analysis using MAGMATo perform common variant risk enrichment analysis for each interaction network using MAGMA32 (v1.09b), PGC GWAS summary statistics (https://www.med.unc.edu/pgc/download-results/) were obtained for schizophrenia (33,640 cases and 43,456 controls of EUR ancestry3; 22,778 cases and 35,362 controls of EAS ancestry8), ADHD (19,099 cases and 34,194 controls of EUR ancestry33), ASD (18,382 cases and 27,969 controls of EUR ancestry34), BIP (41,917 cases and 371,549 controls of EUR ancestry35), and MDD (170,756 cases and 329,443 controls of EUR ancestry36). GWAS summary statistics for height were obtained from the Neale Lab UK Biobank GWAS (round 2; 361,194 individuals of EUR ancestry37; https://www.nealelab.is/uk-biobank/) and Biobank Japan (159,095 individuals of EAS ancestry38; http://jenger.riken.jp/en/result). For each GWAS dataset, we first annotated variants to genes using the Human reference genome (GRCh37/hg19) with a flanking gene region of ±250kb. Variants on sex chromosomes, with minor allele frequency (MAF) ≤ 0.05 in the study, or within the MHC region (chr6:22.5M-33.5M) were excluded from analysis. Next, gene-based P-values were computed using the SNP-wise Mean model in MAGMA. LD was estimated from the 1000 Genomes Project77 phase 3 EUR or EAS panel to match the ancestry of each study. Competitive tests were used for gene-set comparison analysis between stringent interactors and non-interactors in each interaction network. Specifically, a linear regression model was built by MAGMA to test if genes within the interactor gene-set are more strongly associated with the phenotype of interest compared to the non-interactor gene-set. Inverse-variance weighted fixed-effect meta-analysis78 was used to combine results across EUR and EAS ancestries for schizophrenia and height, and a one-tailed P-value was calculated from the meta-analyzed Z-score.

Common variant enrichment analysis using MAGMA

To perform common variant risk enrichment analysis for each interaction network using MAGMA32 (v1.09b), PGC GWAS summary statistics (https://www.med.unc.edu/pgc/download-results/) were obtained for schizophrenia (33,640 cases and 43,456 controls of EUR ancestry3; 22,778 cases and 35,362 controls of EAS ancestry8), ADHD (19,099 cases and 34,194 controls of EUR ancestry33), ASD (18,382 cases and 27,969 controls of EUR ancestry34), BIP (41,917 cases and 371,549 controls of EUR ancestry35), and MDD (170,756 cases and 329,443 controls of EUR ancestry36). GWAS summary statistics for height were obtained from the Neale Lab UK Biobank GWAS (round 2; 361,194 individuals of EUR ancestry37; https://www.nealelab.is/uk-biobank/) and Biobank Japan (159,095 individuals of EAS ancestry38; http://jenger.riken.jp/en/result). For each GWAS dataset, we first annotated variants to genes using the Human reference genome (GRCh37/hg19) with a flanking gene region of ±250kb. Variants on sex chromosomes, with minor allele frequency (MAF) ≤ 0.05 in the study, or within the MHC region (chr6:22.5M-33.5M) were excluded from analysis. Next, gene-based P-values were computed using the SNP-wise Mean model in MAGMA. LD was estimated from the 1000 Genomes Project77 phase 3 EUR or EAS panel to match the ancestry of each study. Competitive tests were used for gene-set comparison analysis between stringent interactors and non-interactors in each interaction network. Specifically, a linear regression model was built by MAGMA to test if genes within the interactor gene-set are more strongly associated with the phenotype of interest compared to the non-interactor gene-set. Inverse-variance weighted fixed-effect meta-analysis78 was used to combine results across EUR and EAS ancestries for schizophrenia and height, and a one-tailed P-value was calculated from the meta-analyzed Z-score.

Common variant enrichment analysis using the GRS methodTo complement the MAGMA analysis, we used an alternative genetic risk score (GRS) method and individual-level genotypes from the schizophrenia GWAS study cohorts (24,764 cases and 30,655 controls of EUR ancestry; 8,960 cases and 8,284 controls of EAS ancestry) to assess genetic risk enrichment in the interaction networks. IRB approvals for accessing the individual-level data were obtained from the PGC and Stanley Global Asia Initiatives (Data S1-S3). For each study cohort, we removed variants in the MHC region and variants with MAF ≤0.05 or low imputation quality (INFO ≤0.8), and then mapped the remaining variants to the stringent interactors or non-interactors associated with each network with a flanking gene region of ±250kb. Next, we used PLINK59 (v1.9) and the in-sample LD to clump variants within the interactor or non-interactor gene-sets into independent association signals. The clumping was performed with a window of 250kb, R2> 0.2, and a P-value threshold of 1. The out-of-sample GRS for each individual and each gene-set was then calculated as ∑ log(OR) ∗G, in whichOR is the estimated odds ratio from the leave-one-out GWAS meta-analysis for the clumped index variants within the gene-set and G is the genotype dosage. To test if the stringent interactors in a network contribute more genetic risk compared to the non-interactors, we fit a linear regression model: GRSij
= Pi
+ Ij
+ Pi ∗ Ij
+ Covi, where i and j denote individual index and gene index, respectively, P is the case/control status, I is interactor/non-interactor status, P ∗I is an interaction term, and Cov includes 10 genetic principal components. We tested whether the P ∗I interaction term is significantly larger than zero, which captures whether the difference of GRS between cases and controls calculated in the interactor gene-set is significantly larger than the non-interactor gene-set. Inverse-variance weighted fixed-effect meta-analysis was used to combine the test statistics for the interaction term across study cohorts and ancestries, and a one-tailed P-value was calculated from the meta-analyzed Z-score. Finally, as a negative control, we performed an analogous GRS analysis for height; in this case, the GRS was calculated as ∑ log(beta) ∗G, where beta is the effect size estimate from the EUR or EAS height GWAS for clumped index variants.

Common variant enrichment analysis using the GRS method

To complement the MAGMA analysis, we used an alternative genetic risk score (GRS) method and individual-level genotypes from the schizophrenia GWAS study cohorts (24,764 cases and 30,655 controls of EUR ancestry; 8,960 cases and 8,284 controls of EAS ancestry) to assess genetic risk enrichment in the interaction networks. IRB approvals for accessing the individual-level data were obtained from the PGC and Stanley Global Asia Initiatives (Data S1-S3). For each study cohort, we removed variants in the MHC region and variants with MAF ≤0.05 or low imputation quality (INFO ≤0.8), and then mapped the remaining variants to the stringent interactors or non-interactors associated with each network with a flanking gene region of ±250kb. Next, we used PLINK59 (v1.9) and the in-sample LD to clump variants within the interactor or non-interactor gene-sets into independent association signals. The clumping was performed with a window of 250kb, R2> 0.2, and a P-value threshold of 1. The out-of-sample GRS for each individual and each gene-set was then calculated as ∑ log(OR) ∗G, in whichOR is the estimated odds ratio from the leave-one-out GWAS meta-analysis for the clumped index variants within the gene-set and G is the genotype dosage. To test if the stringent interactors in a network contribute more genetic risk compared to the non-interactors, we fit a linear regression model: GRSij
= Pi
+ Ij
+ Pi ∗ Ij
+ Covi, where i and j denote individual index and gene index, respectively, P is the case/control status, I is interactor/non-interactor status, P ∗I is an interaction term, and Cov includes 10 genetic principal components. We tested whether the P ∗I interaction term is significantly larger than zero, which captures whether the difference of GRS between cases and controls calculated in the interactor gene-set is significantly larger than the non-interactor gene-set. Inverse-variance weighted fixed-effect meta-analysis was used to combine the test statistics for the interaction term across study cohorts and ancestries, and a one-tailed P-value was calculated from the meta-analyzed Z-score. Finally, as a negative control, we performed an analogous GRS analysis for height; in this case, the GRS was calculated as ∑ log(beta) ∗G, where beta is the effect size estimate from the EUR or EAS height GWAS for clumped index variants.

Rare variant enrichment analysisGene-based association statistics derived from exome sequencing data were obtained for schizophrenia (P-values in ‘P meta’ column from Table S5 of Singh et al.9), ASD (Q-values in ‘qval_dnccPTV’ column from Table S2 of Satterstrom et al.40), and DD (P-values in ‘denovoWEST_p_full’ column from Table S2 of Kaplanis et al.41). For each phenotype and each network, a one-tailed KS test was used to test whether the stringent interactors in the network have more significant association scores (i.e., smaller P-values or Q-values) compared to the non-interactors.

Rare variant enrichment analysis

Gene-based association statistics derived from exome sequencing data were obtained for schizophrenia (P-values in ‘P meta’ column from Table S5 of Singh et al.9), ASD (Q-values in ‘qval_dnccPTV’ column from Table S2 of Satterstrom et al.40), and DD (P-values in ‘denovoWEST_p_full’ column from Table S2 of Kaplanis et al.41). For each phenotype and each network, a one-tailed KS test was used to test whether the stringent interactors in the network have more significant association scores (i.e., smaller P-values or Q-values) compared to the non-interactors.

Transcriptional perturbation enrichment analysisDifferentially expressed genes (DEGs) between schizophrenia patients vs. controls identified in 20 prefrontal cortex cell types were retrieved from Table S6 of Ruzicka et al.17 The up- and down-regulated DEGs were analyzed as a joint set in the primary analysis, and then as separate sets in the secondary follow-up analyses. One-tailed P-values were calculated using a hypergeometric distribution to assess the overlap enrichment between interactors in our networks and the cell-type-specific DEGs. For each hypergeometric test, the ‘population’ was defined as all stringent interactor or non-interactor genes associated with a network and ‘success in population’ was defined as the stringent interactors in the network. The ‘sample’ contained DEGs in a celltype that were found in the population and ‘success in sample’ was the overlap between the interactors and the DEGs.

Transcriptional perturbation enrichment analysis

Differentially expressed genes (DEGs) between schizophrenia patients vs. controls identified in 20 prefrontal cortex cell types were retrieved from Table S6 of Ruzicka et al.17 The up- and down-regulated DEGs were analyzed as a joint set in the primary analysis, and then as separate sets in the secondary follow-up analyses. One-tailed P-values were calculated using a hypergeometric distribution to assess the overlap enrichment between interactors in our networks and the cell-type-specific DEGs. For each hypergeometric test, the ‘population’ was defined as all stringent interactor or non-interactor genes associated with a network and ‘success in population’ was defined as the stringent interactors in the network. The ‘sample’ contained DEGs in a celltype that were found in the population and ‘success in sample’ was the overlap between the interactors and the DEGs.

Social Manhattan plotGenes mapped to 287 genome-wide significant regions in the combined discovery-replication meta-analysis of the PGC schizophrenia GWAS (phase 3) were retrieved from Table S3 of the paper.10 We excluded non-coding genes and genes in the MHC region, and obtained the genomic positions of the remaining genes from Ensembl76 (GRCh37.p13). We additionally included SYNGAP1 by mapping it to the only MHC region SNP (rs140365013) included in the GWAS meta-analysis. Next, we intersected these GWAS genes with our PPI data to identify the subset of GWAS genes that are either an index gene or an interactor of an index gene. We generated a Manhattan plot for these genes using the GWAS P-values of their associated index SNPs and plotted links between the genes to indicate observed interactions in our 3. In addition, we retrieved the list of GWAS genes prioritized by FINEMAP or SMR analysis from Table S12 of the PGC paper and highlighted their overlap with genes prioritized by our data in the plot. For ease of visualization, GWAS P-values were capped at P = 1e-25 and genes in the PCDHA@ gene cluster were collapsed into one gene in the plot.

Social Manhattan plot

Genes mapped to 287 genome-wide significant regions in the combined discovery-replication meta-analysis of the PGC schizophrenia GWAS (phase 3) were retrieved from Table S3 of the paper.10 We excluded non-coding genes and genes in the MHC region, and obtained the genomic positions of the remaining genes from Ensembl76 (GRCh37.p13). We additionally included SYNGAP1 by mapping it to the only MHC region SNP (rs140365013) included in the GWAS meta-analysis. Next, we intersected these GWAS genes with our PPI data to identify the subset of GWAS genes that are either an index gene or an interactor of an index gene. We generated a Manhattan plot for these genes using the GWAS P-values of their associated index SNPs and plotted links between the genes to indicate observed interactions in our 3. In addition, we retrieved the list of GWAS genes prioritized by FINEMAP or SMR analysis from Table S12 of the PGC paper and highlighted their overlap with genes prioritized by our data in the plot. For ease of visualization, GWAS P-values were capped at P = 1e-25 and genes in the PCDHA@ gene cluster were collapsed into one gene in the plot.

HCN1 network drug target queryTo identify existing drugs targeting genes in the combined HCN1 network, we performed a batch query using the Open Targets Platform79 (https://platform.opentargets.org). For each resulting drug-target entry, the corresponding drug mechanism information was retrieved from ChEMBL80 (https://www.ebi.ac.uk/chembl/g/#browse/mechanisms_of_action).

HCN1 network drug target query

To identify existing drugs targeting genes in the combined HCN1 network, we performed a batch query using the Open Targets Platform79 (https://platform.opentargets.org). For each resulting drug-target entry, the corresponding drug mechanism information was retrieved from ChEMBL80 (https://www.ebi.ac.uk/chembl/g/#browse/mechanisms_of_action).
