---
pmid: '31599702'
title: Reciprocal interactions between mtDNA and lifespan control in budding yeast.
authors:
- Garcia EJ
- de Jonge JJ
- Liao PC
- Stivison E
- Sing CN
- Higuchi-Sanabria R
- Boldogh IR
- Pon LA
journal: Mol Biol Cell
year: '2019'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC6857569
doi: 10.1091/mbc.E18-06-0356
---

# Reciprocal interactions between mtDNA and lifespan control in budding yeast.
**Authors:** Garcia EJ, de Jonge JJ, Liao PC, Stivison E, Sing CN, Higuchi-Sanabria R, Boldogh IR, Pon LA
**Journal:** Mol Biol Cell (2019)
**DOI:** [10.1091/mbc.E18-06-0356](https://doi.org/10.1091/mbc.E18-06-0356)
**PMC:** [PMC6857569](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6857569/)

## Abstract

1. Mol Biol Cell. 2019 Nov 15;30(24):2943-2952. doi: 10.1091/mbc.E18-06-0356.
Epub  2019 Oct 10.

Reciprocal interactions between mtDNA and lifespan control in budding yeast.

Garcia EJ(1), de Jonge JJ(1), Liao PC(1), Stivison E(1), Sing CN(1), 
Higuchi-Sanabria R(1), Boldogh IR(1), Pon LA(1).

Author information:
(1)Department of Pathology and Cell Biology and Institute of Human Nutrition, 
College of Physicians and Surgeons, Columbia University, New York, NY 10032.

Loss of mitochondrial DNA (mtDNA) results in loss of mitochondrial respiratory 
activity, checkpoint-regulated inhibition of cell cycle progression, defects in 
growth, and nuclear genome instability. However, after several generations, 
yeast cells can adapt to the loss of mtDNA. During this adaptation, rho0 cells, 
which have no mtDNA, exhibit increased growth rates and nuclear genome 
stabilization. Here, we report that an immediate response to loss of mtDNA is a 
decrease in replicative lifespan (RLS). Moreover, we find that adapted rho0 
cells bypass the mtDNA inheritance checkpoint, exhibit increased mitochondrial 
function, and undergo an increase in RLS as they adapt to the loss of mtDNA. 
Transcriptome analysis reveals that metabolic reprogramming to compensate for 
defects in mitochondrial function is an early event during adaptation and that 
up-regulation of stress response genes occurs later in the adaptation process. 
We also find that specific subtelomeric genes are silenced during adaptation to 
loss of mtDNA. Moreover, we find that deletion of SIR3, a subtelomeric gene 
silencing protein, inhibits silencing of subtelomeric genes associated with 
adaptation to loss of mtDNA, as well as adaptation-associated increases in 
mitochondrial function and RLS extension.

DOI: 10.1091/mbc.E18-06-0356
PMCID: PMC6857569
PMID: 31599702 [Indexed for MEDLINE]

## Full Text

INTRODUCTIONMitochondrial DNA (mtDNA) encodes subunits of the electron transport chain and ATP synthase, as well as components required for mitochondrial protein synthesis. For example, mtDNA of the budding yeast Saccharomyces cerevisiae encodes protein subunits of respiratory chain complexes III, IV, V, and the mitoribosome, as well as rRNAs and tRNAs (Contamine and Picard, 2000). Although mtDNA can be deleted in S. cerevisiae or in cultured mammalian cells (Nagley and Linnane, 1970; King and Attardi, 1989), it is essential in complex multicellular organisms. Indeed, mutations of human mtDNA have clinical manifestations in the brain, heart, skeletal muscle, kidney, and endocrine system (Wallace, 2005; Park and Larsson, 2011).There are also extensive links between mtDNA and lifespan control. For example, there is an age-associated increase in oxidative damage and mutations in mtDNA and a decrease in mitochondrial respiration in humans, mice, and mammalian cells (Muller-Hocker, 1989, 1990; Muller-Hocker et al., 1992; Trounce et al., 1989; Mecocci et al., 1993; Melov et al., 1995, 1997, 1999). Moreover, PolgAmut/mut mutator mice that carry mutations that inhibit the mtDNA proofreading activity of DNA polymerase gamma (PolgA) exhibit elevated levels of mtDNA mutation, premature aging, and phenotypes associated with aging in humans (Trifunovic et al., 2004; Kujoth et al., 2005). These findings raise the possibility that mutation of mtDNA may contribute to aging.However, it is not clear whether mutation or loss of mtDNA function is a cause or consequence of aging. The level of mtDNA mutations in homozygous PolgAmut/mut mice is highly variable and in some tissues more than an order of magnitude higher than that observed in aging humans (Khrapko et al., 2006). Moreover, heterozygous PolgA+/mut mice, which have lower levels of mtDNA mutations compared with homozygous PolgAmut/mut mice but ∼30–200 times higher than wild-type mice, do not exhibit any premature aging or reduction in lifespan (Vermulst et al., 2007). Thus, while PolgA mutator mice are widely used to study diseases associated with mutations of mtDNA, it is not clear that they model the normal aging process.Studies in budding yeast have not provided a clearer understanding of the links between mtDNA and lifespan control. Aging studies in yeast can model two distinct forms of cellular aging. Chronological lifespan (CLS), the survival time of stationary-phase, nondividing yeast cells, is a model for stress resistance in postmitotic cells (MacLean et al., 2001). Replicative lifespan (RLS), the number of times that a cell can divide prior to senescence, is a model for aging of division-competent cells (Mortimer and Johnston, 1959). Budding yeast cells exhibit an increase in mutation or loss of mtDNA as they undergo replicative aging (Veatch et al., 2009). On the other hand, deletion of mtDNA can result in an increase in RLS and the observed lifespan extension is not due to loss of mitochondrial respiratory activity or reduced oxidative stress in mitochondria (Woo and Poyton, 2009). However, other studies indicate that loss of mtDNA can increase, decrease, or have no effect on RLS (Kirchman et al., 1999; Kaeberlein et al., 2005). Thus, the link between loss of mtDNA and aging remains elusive in yeast and other eukaryotes.We reevaluated the effect of mtDNA on yeast RLS, in part because yeast cells adapt to loss of mtDNA. The immediate response to loss of mtDNA in yeast is loss of respiratory activity, activation of the mtDNA inheritance checkpoint, reduced growth rate, and a high rate of nuclear genome instability (Slonimski et al., 1968; Veatch et al., 2009; Crider et al., 2012). The mtDNA inheritance checkpoint inhibits progression from G1 to S phase in response to loss of mtDNA and is regulated by Rad53p, a component of the DNA damage checkpoint signaling pathway (Crider et al., 2012). The nuclear genome instability observed in cells without mtDNA (referred to as rho0 cells) is a consequence of decreased mitochondrial membrane potential (∆Ψ), which in turn results in defects in the formation of iron–sulfur clusters, cofactors that are essential for the normal function of proteins including those that affect nuclear genome integrity (Veatch et al., 2009).Early studies revealed that rho0 cells adapt to loss of the mitochondrial genome. During this process, they exhibit increased growth rates and nuclear genome stability. Adaptation to loss of mtDNA is affected by environmental factors including pH, temperature, nutrient availability, antioxidants, and coculture with cells that have mtDNA (Veatch et al., 2009; Dirick et al., 2014). Here, we report that loss of mtDNA results in a decrease in RLS, and that one consequence of adaptation to loss of mtDNA is an extension of RLS. Moreover, we obtained evidence for a role for subtelomeric gene silencing in the process of rho0 cell adaptation (Dang et al., 2009). In yeast, as in other eukaryotes, telomeres act as caps at the ends of chromosomes to protect them from exonuclease degradation and end-to-end fusions. The DNA repeat TG1-3 at the ends of all yeast chromosomes binds to conserved proteins that regulate telomere length, transcription, and packaging and is both necessary and sufficient to provide telomere function (Shampay et al., 1984; Walmsley et al., 1984; Wellinger and Zakian, 1989; Grunstein, 1997). Other studies support a role for conserved lifespan regulatory proteins in subtelomeric gene silencing. Specifically, a complex consisting of the Sirtuins Sir2p, Sir3p, and Sir4p is recruited to telomeres and subtelomeric regions, where they catalyze deacetylation of histones adjacent to the nucleosome, which leads to chromatin condensation and gene silencing (Park and Lustig, 2000; Rusche et al., 2003; Altaf et al., 2007; Dang et al., 2009; Kozak et al., 2010). We find that specific subtelomeric genes are silenced in yeast as they adapt to loss of mtDNA, and that Sir3p is required for silencing of at least three different subtelomeric genes, as well as for improved mitochondrial function and RLS extension during this adaptation process.

INTRODUCTION

Mitochondrial DNA (mtDNA) encodes subunits of the electron transport chain and ATP synthase, as well as components required for mitochondrial protein synthesis. For example, mtDNA of the budding yeast Saccharomyces cerevisiae encodes protein subunits of respiratory chain complexes III, IV, V, and the mitoribosome, as well as rRNAs and tRNAs (Contamine and Picard, 2000). Although mtDNA can be deleted in S. cerevisiae or in cultured mammalian cells (Nagley and Linnane, 1970; King and Attardi, 1989), it is essential in complex multicellular organisms. Indeed, mutations of human mtDNA have clinical manifestations in the brain, heart, skeletal muscle, kidney, and endocrine system (Wallace, 2005; Park and Larsson, 2011).

There are also extensive links between mtDNA and lifespan control. For example, there is an age-associated increase in oxidative damage and mutations in mtDNA and a decrease in mitochondrial respiration in humans, mice, and mammalian cells (Muller-Hocker, 1989, 1990; Muller-Hocker et al., 1992; Trounce et al., 1989; Mecocci et al., 1993; Melov et al., 1995, 1997, 1999). Moreover, PolgAmut/mut mutator mice that carry mutations that inhibit the mtDNA proofreading activity of DNA polymerase gamma (PolgA) exhibit elevated levels of mtDNA mutation, premature aging, and phenotypes associated with aging in humans (Trifunovic et al., 2004; Kujoth et al., 2005). These findings raise the possibility that mutation of mtDNA may contribute to aging.

However, it is not clear whether mutation or loss of mtDNA function is a cause or consequence of aging. The level of mtDNA mutations in homozygous PolgAmut/mut mice is highly variable and in some tissues more than an order of magnitude higher than that observed in aging humans (Khrapko et al., 2006). Moreover, heterozygous PolgA+/mut mice, which have lower levels of mtDNA mutations compared with homozygous PolgAmut/mut mice but ∼30–200 times higher than wild-type mice, do not exhibit any premature aging or reduction in lifespan (Vermulst et al., 2007). Thus, while PolgA mutator mice are widely used to study diseases associated with mutations of mtDNA, it is not clear that they model the normal aging process.

Studies in budding yeast have not provided a clearer understanding of the links between mtDNA and lifespan control. Aging studies in yeast can model two distinct forms of cellular aging. Chronological lifespan (CLS), the survival time of stationary-phase, nondividing yeast cells, is a model for stress resistance in postmitotic cells (MacLean et al., 2001). Replicative lifespan (RLS), the number of times that a cell can divide prior to senescence, is a model for aging of division-competent cells (Mortimer and Johnston, 1959). Budding yeast cells exhibit an increase in mutation or loss of mtDNA as they undergo replicative aging (Veatch et al., 2009). On the other hand, deletion of mtDNA can result in an increase in RLS and the observed lifespan extension is not due to loss of mitochondrial respiratory activity or reduced oxidative stress in mitochondria (Woo and Poyton, 2009). However, other studies indicate that loss of mtDNA can increase, decrease, or have no effect on RLS (Kirchman et al., 1999; Kaeberlein et al., 2005). Thus, the link between loss of mtDNA and aging remains elusive in yeast and other eukaryotes.

We reevaluated the effect of mtDNA on yeast RLS, in part because yeast cells adapt to loss of mtDNA. The immediate response to loss of mtDNA in yeast is loss of respiratory activity, activation of the mtDNA inheritance checkpoint, reduced growth rate, and a high rate of nuclear genome instability (Slonimski et al., 1968; Veatch et al., 2009; Crider et al., 2012). The mtDNA inheritance checkpoint inhibits progression from G1 to S phase in response to loss of mtDNA and is regulated by Rad53p, a component of the DNA damage checkpoint signaling pathway (Crider et al., 2012). The nuclear genome instability observed in cells without mtDNA (referred to as rho0 cells) is a consequence of decreased mitochondrial membrane potential (∆Ψ), which in turn results in defects in the formation of iron–sulfur clusters, cofactors that are essential for the normal function of proteins including those that affect nuclear genome integrity (Veatch et al., 2009).

Early studies revealed that rho0 cells adapt to loss of the mitochondrial genome. During this process, they exhibit increased growth rates and nuclear genome stability. Adaptation to loss of mtDNA is affected by environmental factors including pH, temperature, nutrient availability, antioxidants, and coculture with cells that have mtDNA (Veatch et al., 2009; Dirick et al., 2014). Here, we report that loss of mtDNA results in a decrease in RLS, and that one consequence of adaptation to loss of mtDNA is an extension of RLS. Moreover, we obtained evidence for a role for subtelomeric gene silencing in the process of rho0 cell adaptation (Dang et al., 2009). In yeast, as in other eukaryotes, telomeres act as caps at the ends of chromosomes to protect them from exonuclease degradation and end-to-end fusions. The DNA repeat TG1-3 at the ends of all yeast chromosomes binds to conserved proteins that regulate telomere length, transcription, and packaging and is both necessary and sufficient to provide telomere function (Shampay et al., 1984; Walmsley et al., 1984; Wellinger and Zakian, 1989; Grunstein, 1997). Other studies support a role for conserved lifespan regulatory proteins in subtelomeric gene silencing. Specifically, a complex consisting of the Sirtuins Sir2p, Sir3p, and Sir4p is recruited to telomeres and subtelomeric regions, where they catalyze deacetylation of histones adjacent to the nucleosome, which leads to chromatin condensation and gene silencing (Park and Lustig, 2000; Rusche et al., 2003; Altaf et al., 2007; Dang et al., 2009; Kozak et al., 2010). We find that specific subtelomeric genes are silenced in yeast as they adapt to loss of mtDNA, and that Sir3p is required for silencing of at least three different subtelomeric genes, as well as for improved mitochondrial function and RLS extension during this adaptation process.

RESULTS AND DISCUSSIONYeast adapt to loss of mtDNAWe confirmed previous findings (Veatch et al., 2009; Dirick et al., 2014) that rho0 cells adapt to loss of mtDNA. Freshly prepared rho0 cells form small and large colonies on solid media (Figure 1, A and B). Cells from both small and large rho0 cell colonies have no respiratory activity and grow significantly more slowly than rho+ cells, which contain mtDNA (Figure 1C). However, cells from large rho0 colonies exhibit higher growth rates than cells from small rho0 colonies (Figure 1C). It is likely that the large colonies represent a population of rho0 cells that have adapted to the loss of mtDNA and thus exhibit faster growth rates. Therefore, we will refer to the cells from small and large colonies of rho0 strains as unadapted (UA) and adapted, respectively.FIGURE 1:Yeast adapt to loss of mtDNA. (A) Distribution of yeast colony area from rho0 (newly generated rho0 cells), rho0 UA source (rho0 cells derived from UA small rho0 colonies), and rho0 A source (rho0 cells derived from adapted large rho0 colonies). Representative trial from three independent experiments. The dotted line indicates colony area threshold criterion used to define adapted colonies. The p values indicate statistically significant differences between the average colony sizes of the strains (n = 137–322 colonies measured per condition; ****p < 0.0001, by Kruskal–Wallis test with Dunn’s post-hoc test for multiple comparisons). (B) The percentage of UA and adapted colonies, according to colony area criteria used in A, from newly generated rho0, rho0 UA, and rho0 A sources. The bar represents the average percentage of colonies of each size ± SEM in three independent experiments (n = 68–322 colonies per experiment per condition; ***p < 0.001; and ****p < 0.0001, by one-way ANOVA with Tukey post-hoc test). (C) Growth rates of rho+, rho0 UA, and rho0 A cells. The bar shows pooled average ± SEM of the maximum OD600/h from three independent experiments (n = 6–12 replicates per conditions; **p < 0.01; and ***p < 0.001, by one-way ANOVA with Tukey post-hoc test). (D) Quantitation of progression from G1 to G2 for rho+, rho0 UA, and rho0 A cells. Cells were incubated with mating pheromone (alpha factor), which arrests cells in the G1 phase of the cell cycle. Progression of cells from G1 to G2 stages of the cell cycle was monitored after release from G1 using flow cytometry to measure the levels of propidium iodine stained DNA. Progression was measured as the fold change in the fraction of cells in G1 phase at the time specified, relative to the fraction of cells that were in G1 at the time of release from alpha factor-induced G1 arrest (cells in G1 at t0/cells in G1 at tx, 50,000 events measured per timepoint per strain).Interestingly, we find that UA rho0 cells give rise to cells that form small colonies and exhibit low growth rates, but they also give rise to cells that form large colonies and exhibit high growth rates. Thus, UA rho0 cells give rise to both UA and adapted rho0 cells. In contrast, adapted rho0 cells give rise only to adapted cells, which form large colonies and exhibit high growth rates (Figure 1, A–C). These data confirm previous findings that the adaptation observed in rho0 cells is heritable (Dirick et al., 2014). Moreover, rho0 cells continue to adapt as they are propagated. We find that the colonies produced from adapted rho0 cells are significantly larger than those obtained from newly generated rho0 cells (Figure 1B).Our initial observation of rho0 adaptation was made in cells where mtDNA was eliminated by treatment with EtBr. However, we also observe adaptation in rho0 cells in which mtDNA has been lost as a result of the deletion of MGM101 (Supplemental Figure S1), which encodes a protein that mediates mtDNA repair and is required for mtDNA maintenance (Chen et al., 1993). Adaptation to loss of mtDNA has also been documented in yeast that undergo spontaneous mtDNA loss and in yeast in which mtDNA loss was induced by expression of a dominant-negative form of the mtDNA polymerase MIP1 (Veatch et al., 2009; Dirick et al., 2014). Thus, this adaptation is a general response to loss of mtDNA and not a consequence of the method used to delete mtDNA. The collective findings that yeast cells adapt to loss of mtDNA raise the possibility that mammalian rho0 cells can also adapt to loss of mtDNA.Finally, we find that the increase in growth rate that occurs in adapted rho0 colonies is due at least in part to bypass of the mtDNA inheritance checkpoint (Figure 1D). We monitored cell cycle progression in synchronized yeast cells using flow cytometry to measure DNA content. Wild-type rho+ cells, which contain mtDNA, transition from G1 to G2 phase 60–130 min after release from G1 arrest. In contrast, cells from small rho0 colonies exhibit severe defects in transition from G1 to S phase. Finally, cells from large rho0 colonies progress through the cell cycle similarly to rho+ cells. In the example shown, the lag time for entry into the cell cycle after release from G1 arrest and the cycling times are shorter in adapted rho0 cells than in rho+ cells.Effects of mtDNA on lifespan and mitochondrial redox stateOne consequence of loss of mtDNA and the associated mitochondrial respiratory activity is a decrease in ∆Ψ. Previous studies revealed that ∆Ψ increases as yeast adapt to loss of mtDNA (Veatch et al., 2009). To further characterize the adaptation process, we studied mitochondrial redox state in UA and adapted rho0 cells using a redox-sensing variant of GFP (roGFP) (Figure 2, A and B) (Hanson et al., 2004). Our previous studies using mitochondria-­targeted roGFP and other biosensors revealed that fitter mitochondria that are more reduced, contain less mitochondrial superoxide, and have higher ∆Ψ are preferentially inherited by yeast daughter cells and that this affects yeast cell fitness and lifespan (McFaline-Figueroa et al., 2011).FIGURE 2:Adaptation to loss of mtDNA results in increased lifespan and mitochondrial quality. (A) Representative images of redox state of mitochondria in WT rho+, rho0 UA, and rho0 A cells measured with mito-roGFP1. Reduced:oxidized mito-roGFP ratio images are shown. Color scale in the bottom panel shows the dynamic range of ratios, with warmer colors indicating a more reducing environment. (B) Quantitation of reduced:oxidized mito-roGFP ratios in WT rho+, rho0 UA, rho0 A, and rho0 AA cells. The box indicates the middle quartile with the midline representing the median; whiskers show the minimum and maximum values. Representative trial from three independent experiments (n = 84–104 for each condition, *p < 0.05, ****p < 0.0001, by one-way ANOVA with Tukey post-hoc test for multiple comparisons). (C) Mean generation time was measured during RLS determination shown in D and E, as the time elapsed between the emergence of two consecutive buds. Bars show average ± SEM for one independent experiment (n = 30–51 cells per condition; **p < 0.01; and ****p < 0.0001, by Kruskal–Wallis test with Dunn’s post-hoc test for multiple comparisons). (D, E) RLS determination for WT rho+, rho0 UA, and rho0 A cells. (n = 40–52 starting new daughters per condition. Statistical significance between RLS survival curves was tested with Log-rank (Mantel–Cox) test where p < 0.05).Loss of mtDNA or adaptation to that loss does not affect mitochondrial quality control during inheritance: daughter cells inherit mitochondria that are more reduced and therefore higher functioning in adapted and UA rho0 cells (Supplemental Figure S2). However, mitochondria in UA rho0 cells are significantly more oxidized compared with mitochondria in rho+ cells. Moreover, mitochondrial redox state improves in rho0 cells as they adapt. Here, we evaluated mitochondrial redox state during early and later stages of adaptation (3 and 5 d after deletion of mtDNA, respectively). We detect a subtle but statistically significant increase in the reducing potential of mitochondria during early stages of adaptation. Furthermore, mitochondrial reducing potential continues to increase during late stages of adaptation, approaching levels observed in rho+ cells (Figure 2, A and B).Interestingly, the more reducing mitochondrial environment observed on adaptation of rho0 cells is not accompanied by lower mitochondrial superoxide levels. Using dihydroethidium (DHE) to detect superoxides in living yeast cells, we confirmed our previous findings that all detectable superoxides in yeast colocalize with mitochondria (McFaline-Figueroa et al., 2011). Beyond this, we find that deletion of mtDNA results in loss of all detectable superoxides in mitochondria and that mitochondrial superoxide levels do not change as cells adapt to loss of mtDNA. (Supplemental Figure S2).Thus, we detect two additional events that occur during adaptation to loss of mtDNA: bypass of the mtDNA inheritance checkpoint and improved mitochondrial redox state. Interestingly, loss of mtDNA has no effect on the mitochondrial quality control mechanisms that promote inheritance of higher-functioning mitochondria by yeast daughter cells (McFaline-Figueroa et al., 2011). Moreover, since loss of mtDNA results in loss of all detectable mitochondrial ROS, a phenotype that is stable during rho0 cell adaptation, mitochondrial ROS is therefore not responsible for changes in the redox state of the organelle as rho0 cells adapt.Equally important, we find that rho0 cells undergo an extension of RLS as they adapt (Figure 2, C–E). The mean RLS of rho+ cells in the genetic background used in these studies is 20–25 generations. The RLS of UA and adapted rho0 cells is variable. However, the mean RLS of UA rho0 cells is always significantly lower than that of rho+ cells, and the generation time of UA rho0 cells is longer. In contrast, the mean RLS of adapted rho0 cells is higher than that of UA rho0 cells (unpublished data). In the example shown, the RLS of the adapted rho0 cells is greater than that of rho+ cells.Thus, we detect reciprocal interactions between mtDNA and lifespan in budding yeast. Loss of mtDNA results in reduced RLS. Conversely, RLS extension is one consequence of the adaptation of yeast to loss of mtDNA. Our findings provide evidence for a role for mtDNA in lifespan control in yeast and raise the possibility that loss of mtDNA or mtDNA function may also affect lifespan in other eukaryotes. Collectively, our findings reconcile previous observations that loss of mtDNA has diverse effects on RLS in yeast: the variable RLS observed in rho0 cells may be a consequence of analysis of RLS in cells in different states of adaption to loss of mtDNA.Subtelomeric genes are silenced in adapted rho 0 cellsSince adaptation is heritable, we used RNA-Seq to compare the transcriptomes of UA rho0 cells and rho0 cells at different stages of adaptation (Figure 3 and Supplemental Table S1). Here, we used rho0 cells 3 d after EtBR-mediated loss of mtDNA as a model for early-stage adaptation and rho0 cells >10 d after loss of mtDNA induced by deletion of MGM101 as a model for late-stage adaptation.FIGURE 3:Transient up-regulation of genes occurs during adaptation to loss of mtDNA. Revigo plot of GO terms associated with genes that are up-regulated in early-stage adapted rho0 cells compared with UA rho0 cells (A) and later-stage compared with early-stage adapted rho0 cells (B). Bubbles with cooler colors represent more significant p values; the size of the bubble indicates the frequency of the GO term. X and Y coordinates are derived by applying multidimensional scaling to a matrix of the GO terms’ semantic similarities. (C) Average fold change of transcript levels for subtelomeric genes HSP30 and ADE17 in early-stage adapted rho0 cells (A) and late-stage adapted cells (AA), relative to WT rho0 UA cells. Fold change was calculated as 2-ΔΔCT with actin serving as the endogenous control for each sample. Averages and SEM from n = 3 independent trials are shown (*p < 0.05, **p < 0.001 by one-way ANOVA with Dunnett’s multiple comparisons test).We find that metabolic reprogramming to compensate for loss of mitochondrial metabolic activity occurs during early stages of adaptation in rho0 cells and identified biomarkers that can be used to assess rho0 adaptation (Figure 3 and Supplemental Table S1). Specifically, we find that pathways for amino acid and purine biosynthesis, essential functions of mitochondria, are up-regulated during early-stage adaptation in rho0 cells. Consistent with this, our quantitative PCR (qPCR) analysis indicates that the level of mRNA for ADE17, which encodes a purine biosynthetic enzyme, increases during early stages of adaption and, during later stages of adaptation, returns to levels observed in UA rho0 cells (Figure 3; Supplemental Figure S3A).In contrast, we find that oxidant stress response genes are up-regulated during late stages of adaptation (Figure 3 and Supplemental Table S1). For example, spermine is a polyamine that is induced by stressors in prokaryotes and eukaryotes, can protect DNA from oxidative damage, decreases sensitivity to oxidative stress, and has been implicated in activation of stress response gene expression (Rider et al., 2007; Pegg and Michael, 2010). We find that spermine transport and biosynthesis are up-regulated during late-stage adaptation of rho0 cells. As described above, mitochondria become more reducing, but we see no obvious change in mitochondrial superoxide levels during adaptation to loss of mtDNA. Therefore, it is possible that the change in mitochondrial redox state observed in adapted rho0 cells is due to up-regulation of spermine and other oxidant stress response factors.It is interesting that oxidant stress response genes are up-regulated in late-stage adapted rho0 cells, even though there is no change in mitochondrial superoxide levels in UA and adapted rho0 cells. On the other hand, we detect superoxides that do not colocalize with mitochondria in UA rho0 cells. Therefore, it is possible that extramitochondrial ROS drives the mitochondrial antioxidant response.Our transcriptome analysis also revealed that specific subtelomeric genes are preferentially silenced in rho0 cells in both early and late stages of adaptation (Supplemental Tables S1 and S2). We find that 24% of the transcripts that are down-regulated during early adaptation are encoded by genes that lie within 25 kb of the telomere and are therefore subtelomeric. Indeed, the subtelomeric genes that appear to be silenced in adapted rho0 cells are present in 11 of the 16 yeast chromosomes (Figure 4, A and B).FIGURE 4:Subtelomeric gene silencing occurs during adaptation to loss of mtDNA. (A) The most down-regulated genes in early stage (A) vs. UA rho0 cells, listed according to their chromosomal loci. Down-regulated genes were defined as those exhibiting a Log2 fold change <–1.5 and statistical significance of p < 0.05. Distance from telomere denotes the distance from the gene locus to the closest telomere in the chromosome. (B) Histogram showing distribution of distances from nearest telomere of the 13 most down-regulated genes shown in A. One gene with a distance of 93 kb is not shown. (C) Average fold change of transcript levels for subtelomeric genes PAU24, DAN1, and FIT3 in early-stage adapted rho0 (A), and late-stage adapted cells (AA), compared with WT rho0 UA. Fold change was calculated as 2-ΔΔCT with actin serving as the endogenous control for each sample. Averages and SEM from n = 3 independent trials are shown (**p < 0.01, ****p < 0.0001 by one-way ANOVA with Dunnett’s multiple comparisons test).Early studies supported the model that subtelomeric silencing is a consequence of spreading of silencing mediators (e.g., the Sir2/3/4 protein complex) from telomeres to subtelomeres (Gottschling et al., 1990; Renauld et al., 1993). However, other studies raised questions regarding the generality of this model. Indeed, transcriptome analysis and chromatin immunoprecipitation studies revealed that the Sir2/3/4 complex localizes to discrete, noncontinuous sites on subtelomeres and is responsible for silencing of only 6% of the genes in subtelomeres (Ellahi et al., 2015). Thus, available evidence indicates that telomere positioning effects do not contribute to subtelomeric gene silencing by the Sir2/3/4 complex. Our transcriptome analysis indicates that only a limited number of genes are silenced within subtelomeres in adapted rho0 cells. In cases where more than one subtelomeric gene appears to be silenced, genes are discontinuous. Thus, there are no obvious telomere positioning effects in the subtelomeric gene silencing we observe in adapted rho0 cells.On the other hand, many of the subtelomeric genes that are silenced during rho0 cell adaptation are functionally related. Previous studies indicate that newly generated rho0 cells exhibit a transcription signature of iron starvation (e.g., increased expression of iron transport and homeostasis proteins), which likely reflects compensatory mechanisms to promote the essential process of iron–sulfur cluster formation in mitochondria (Veatch et al., 2009). We find that some of the subtelomeric genes that are silenced during early and late adaptation encode iron transport proteins. The other major class of subtelomeric genes that are silenced in both early- and late-stage adaptation are stress response genes including proteins in the seripauperin multigene family (Supplemental Tables S3 and S4).Indeed, using qPCR to quantify cellular mRNA levels, we confirmed that the transcripts of two seripauperin family proteins (PAU24 and DAN1) and one iron transport gene (FIT3) that are silenced in adapted rho0 cells based on RNA-Seq analysis are present in reduced levels in adapted compared with UA rho0 cells (Figure 4C; Supplemental Figure S3B). Importantly, these genes are silenced during both early and later stages of adaptation, indicating that subtelomeric silencing is involved in the regulation of adaptation.SIR3 is required for adaptation to loss of mtDNATo determine whether adaptation to loss of mtDNA is due to subtelomeric silencing, we studied the effect of deletion of SIR3 on adaptation in rho0 cells. We deleted mtDNA in sir3∆ cells and found that sir3∆ rho0 cells form large and small colonies when propagated on solid media. We also find that small sir3∆ rho0 cell colonies give rise to large and small colonies, while large sir3∆ rho0 cell colonies give rise primarily to large colonies (Figure 5A). Thus, sir3∆ rho0 cells have the capacity to adapt to loss of mtDNA.FIGURE 5:Sir3p is required for efficient adaptation to loss of mtDNA. (A) Percentage of UA and A colonies, according to colony area criteria used in Figure 1A, arising from newly generated sir3Δ rho0 and from UA and adapted colonies propagated from newly generated sir3Δ rho0 cells. The bar shows the average percentage distribution of colony sizes ± SEM of three independent experiments (n = 85–394 per condition per experiment; **p < 0.01, and ****p < 0.0001 by one-way ANOVA with Tukey post-hoc test). (B) Growth rates of sir3Δ rho+ cells and cells from UA (gray column) and adapted (A, light blue) sir3Δ rho0 colonies. Bars show pooled average ± SEM of maximum OD600/h from three independent experiments (n = 6–12 replicates per conditions; ***p < 0.001 and ****p < 0.0001, by one-way ANOVA with Tukey post-hoc test). (C) Gene silencing rate changes were determined by the change in ΔCt between 3 and 5 d of adaptation for PAU24, DAN1, and FIT3 in rho0 and sir3∆ rho0 cells. Averages and SEM from n = 3 independent trials are shown (*p < 0.05, by Student’s t test). (D) Representative images of redox state of mitochondria measured with mito-roGFP1 in sir3Δ rho+ cells, and in sir3Δ rho0 cells from UA and adapted (A) colonies. (E) Quantitation of reduced:oxidized mito-roGFP ratios in sir3Δ rho+ cells, and sir3Δ rho0 cells from UA and A colonies. Box and whiskers are defined as in Figure 2 (n = 48–89 for each condition; ****p < 0.0001, by one-way ANOVA with Tukey post-hoc test). (F) RLS measurements for sir3Δ rho0 UA and A cells performed as described in Materials and Methods (n = 40 new daughters observed per condition per RLS experiment. Statistical significance between RLS survival curves was tested with Log-rank (Mantel–Cox) where p < 0.05).However, we find that sir3∆ rho0 cells exhibit a fundamentally different adaptation phenotype compared with rho0 cells that contain wild-type SIR3. First, in contrast to adapted rho0 cells, which exhibit higher rates of growth in liquid media compared with UA rho0 cells, the growth rate of cells from large sir3∆ rho0 cell colonies in liquid media is significantly lower than that of UA sir3∆ rho0 cells (Figure 5B; Supplemental Figure S4A). Although it seems counterintuitive that cells from large sir3∆ rho0 colonies exhibit lower growth rates compared with cells from small sir3∆ rho0 colonies, there is a precedent for differential yeast growth rates on solid versus liquid media. For example, yeast bearing a deletion in one of the FLO genes, which are required for adhesion (flocculation) of yeast cells, exhibit increased colony size on solid media as well as a severe reduction in growth rate in liquid media compared with wild-type cells (e.g., Di Gianvito et al., 2017). Indeed, since genes that affect cell wall mannoproteins are silenced during adaptation to loss of mtDNA, it is possible that the large colony size of some sir3∆ rho0 cells is due to effects on yeast cell adhesion.Equally important, we detect a statistically significant decrease in the silencing rate of three subtelomeric genes (FIT3, DAN1, and PAU24) in sir3∆ rho0 cells compared with rho0 cells during the adaptation process (Figure 5C; Supplemental Figure S4B). In addition, there is no detectable increase in mitochondrial reducing potential during adaptation of sir3∆ rho0 cells (Figure 5, D and E). Finally, we find that the RLS of cells from adapted and UA sir3∆ rho0 cell colonies is indistinguishable. Thus, there is no improvement in mitochondrial redox state or extension of RLS in adapted sir3∆ rho0 cells (Figure 5F).Our findings support the model that Sir3p-dependent subtelomeric gene silencing is responsible for improved mitochondrial function and extended RLS associated with adaptation to loss of mtDNA. While it is not clear whether the increase in mitochondrial redox state is causative in the RLS-adaptive phenotype, our previous studies (Higuchi et al., 2013) indicate that rendering mitochondria more reducing is sufficient to extend RLS in yeast. Thus, it is possible that the lifespan extension observed in yeast that adapt to loss of mtDNA is due to the change in mitochondrial redox state.Previous studies revealed a role for lifespan-regulating genes in rho0 cell survival. Most yeast strains can tolerate loss of mtDNA. However, yeast carrying certain mutations (e.g., mitochondrial protein import or protein quality control) require mtDNA for survival (Dunn and Jensen, 2003; Senapin et al., 2003). Interestingly, deletion of conserved lifespan-regulating genes can suppress the lethality observed on loss of mtDNA in these “petite-negative” strains. Specifically, deletion of any of several proteins in the 60S ribosomal subunit (rpl13a, 37b, 12a, 20b, 19a, 6b, 14a, 43b, 34b, 35b, and 12b as well as rpp1b and 4a) can extend lifespan and suppress the lethality of loss of mtDNA in petite-negative yeast (Dunn et al., 2006). We do not detect changes in the levels of the transcripts of any of these RLS-extending genes. Thus, the lifespan extension observed in adapted rho0 cells is not due to alterations in the expression of ribosomal genes that affect rho0 cell viability.Previous studies also revealed that elevated mitochondrial ROS results in extension of CLS in yeast. The extended CLS observed in these studies is a consequence of Rad53p-dependent silencing of the DNA demethylase Rph1p and the resulting increase in subtelomeric gene silencing by Sir3p (Schroeder et al., 2013). While the lifespan extension observed in yeast exposed to elevated mitochondrial ROS and in yeast that are adapting to loss of mtDNA are both Sir3p-dependent, there is evidence that these processes are mechanistically distinct. First, yeast undergoing chronological and replicative aging are in different stages of the yeast life cycle. Chronological aging occurs in yeast that have encountered nutrient limitations and/or accumulation of damaging metabolites and cannot undergo cell division. In contrast, yeast undergoing replicative aging are cell division-competent and not nutrient-­limited. Second, there is no detectable mitochondrial superoxide in UA rho0 cells or adapted rho0 cells at any stage of adaptation. Third, with the exception of one gene, there is no overlap between the 43 genes that are repressed in stationary-phase yeast with elevated mitochondrial ROS and the 42 genes that are down-regulated in early-stage adapted (A) rho0 cells, or the 93 genes that are repressed in later-stage adapted (AA) rho0 cells. Indeed, the genes that are down-regulated in adapted rho0 cells are different from the genes that undergo Sir2/3/4-dependent down-regulation in rho+ cells (Ellahi et al., 2015). Finally, we find that silencing of the three subtelomeric genes analyzed that are down-regulated in adapted rho0 cells does not require RPH1 (unpublished data). These findings provide additional support for the notion that differential Sir3p-dependent subtelomeric gene silencing events occur in response to different environmental or cellular conditions. That is, Sir3p regulates different genes in dividing rho+ cells in oxidatively stressed quiescent rho+ cells and in rho0 cells that are adapting to loss of mtDNA.Overall, our studies reveal new links between mtDNA and lifespan control. Loss of mtDNA results in reduced RLS. On the other hand, extension of RLS is one consequence of adaptation to loss of mtDNA. We also identified distinct transcriptome signatures during early and late stages of adaptation to loss of mtDNA. Metabolic programming to compensate for loss of key mitochondrial functions occurs early in the adaptation process while up-regulation of specific stress response genes occurs later in that process. Finally, we find that silencing of some subtelomeric genes occurs during early and later stages of adaptation and obtained evidence for a role for Sir3p in this process and for improved mitochondrial function and extended RLS during rho0 cell adaptation. Ongoing studies focus on the gene(s) responsible for the RLS extension that occurs during adaptation to loss of mtDNA and how those genes contribute to that process.

RESULTS AND DISCUSSION

Yeast adapt to loss of mtDNAWe confirmed previous findings (Veatch et al., 2009; Dirick et al., 2014) that rho0 cells adapt to loss of mtDNA. Freshly prepared rho0 cells form small and large colonies on solid media (Figure 1, A and B). Cells from both small and large rho0 cell colonies have no respiratory activity and grow significantly more slowly than rho+ cells, which contain mtDNA (Figure 1C). However, cells from large rho0 colonies exhibit higher growth rates than cells from small rho0 colonies (Figure 1C). It is likely that the large colonies represent a population of rho0 cells that have adapted to the loss of mtDNA and thus exhibit faster growth rates. Therefore, we will refer to the cells from small and large colonies of rho0 strains as unadapted (UA) and adapted, respectively.FIGURE 1:Yeast adapt to loss of mtDNA. (A) Distribution of yeast colony area from rho0 (newly generated rho0 cells), rho0 UA source (rho0 cells derived from UA small rho0 colonies), and rho0 A source (rho0 cells derived from adapted large rho0 colonies). Representative trial from three independent experiments. The dotted line indicates colony area threshold criterion used to define adapted colonies. The p values indicate statistically significant differences between the average colony sizes of the strains (n = 137–322 colonies measured per condition; ****p < 0.0001, by Kruskal–Wallis test with Dunn’s post-hoc test for multiple comparisons). (B) The percentage of UA and adapted colonies, according to colony area criteria used in A, from newly generated rho0, rho0 UA, and rho0 A sources. The bar represents the average percentage of colonies of each size ± SEM in three independent experiments (n = 68–322 colonies per experiment per condition; ***p < 0.001; and ****p < 0.0001, by one-way ANOVA with Tukey post-hoc test). (C) Growth rates of rho+, rho0 UA, and rho0 A cells. The bar shows pooled average ± SEM of the maximum OD600/h from three independent experiments (n = 6–12 replicates per conditions; **p < 0.01; and ***p < 0.001, by one-way ANOVA with Tukey post-hoc test). (D) Quantitation of progression from G1 to G2 for rho+, rho0 UA, and rho0 A cells. Cells were incubated with mating pheromone (alpha factor), which arrests cells in the G1 phase of the cell cycle. Progression of cells from G1 to G2 stages of the cell cycle was monitored after release from G1 using flow cytometry to measure the levels of propidium iodine stained DNA. Progression was measured as the fold change in the fraction of cells in G1 phase at the time specified, relative to the fraction of cells that were in G1 at the time of release from alpha factor-induced G1 arrest (cells in G1 at t0/cells in G1 at tx, 50,000 events measured per timepoint per strain).Interestingly, we find that UA rho0 cells give rise to cells that form small colonies and exhibit low growth rates, but they also give rise to cells that form large colonies and exhibit high growth rates. Thus, UA rho0 cells give rise to both UA and adapted rho0 cells. In contrast, adapted rho0 cells give rise only to adapted cells, which form large colonies and exhibit high growth rates (Figure 1, A–C). These data confirm previous findings that the adaptation observed in rho0 cells is heritable (Dirick et al., 2014). Moreover, rho0 cells continue to adapt as they are propagated. We find that the colonies produced from adapted rho0 cells are significantly larger than those obtained from newly generated rho0 cells (Figure 1B).Our initial observation of rho0 adaptation was made in cells where mtDNA was eliminated by treatment with EtBr. However, we also observe adaptation in rho0 cells in which mtDNA has been lost as a result of the deletion of MGM101 (Supplemental Figure S1), which encodes a protein that mediates mtDNA repair and is required for mtDNA maintenance (Chen et al., 1993). Adaptation to loss of mtDNA has also been documented in yeast that undergo spontaneous mtDNA loss and in yeast in which mtDNA loss was induced by expression of a dominant-negative form of the mtDNA polymerase MIP1 (Veatch et al., 2009; Dirick et al., 2014). Thus, this adaptation is a general response to loss of mtDNA and not a consequence of the method used to delete mtDNA. The collective findings that yeast cells adapt to loss of mtDNA raise the possibility that mammalian rho0 cells can also adapt to loss of mtDNA.Finally, we find that the increase in growth rate that occurs in adapted rho0 colonies is due at least in part to bypass of the mtDNA inheritance checkpoint (Figure 1D). We monitored cell cycle progression in synchronized yeast cells using flow cytometry to measure DNA content. Wild-type rho+ cells, which contain mtDNA, transition from G1 to G2 phase 60–130 min after release from G1 arrest. In contrast, cells from small rho0 colonies exhibit severe defects in transition from G1 to S phase. Finally, cells from large rho0 colonies progress through the cell cycle similarly to rho+ cells. In the example shown, the lag time for entry into the cell cycle after release from G1 arrest and the cycling times are shorter in adapted rho0 cells than in rho+ cells.

Yeast adapt to loss of mtDNA

We confirmed previous findings (Veatch et al., 2009; Dirick et al., 2014) that rho0 cells adapt to loss of mtDNA. Freshly prepared rho0 cells form small and large colonies on solid media (Figure 1, A and B). Cells from both small and large rho0 cell colonies have no respiratory activity and grow significantly more slowly than rho+ cells, which contain mtDNA (Figure 1C). However, cells from large rho0 colonies exhibit higher growth rates than cells from small rho0 colonies (Figure 1C). It is likely that the large colonies represent a population of rho0 cells that have adapted to the loss of mtDNA and thus exhibit faster growth rates. Therefore, we will refer to the cells from small and large colonies of rho0 strains as unadapted (UA) and adapted, respectively.

Yeast adapt to loss of mtDNA. (A) Distribution of yeast colony area from rho0 (newly generated rho0 cells), rho0 UA source (rho0 cells derived from UA small rho0 colonies), and rho0 A source (rho0 cells derived from adapted large rho0 colonies). Representative trial from three independent experiments. The dotted line indicates colony area threshold criterion used to define adapted colonies. The p values indicate statistically significant differences between the average colony sizes of the strains (n = 137–322 colonies measured per condition; ****p < 0.0001, by Kruskal–Wallis test with Dunn’s post-hoc test for multiple comparisons). (B) The percentage of UA and adapted colonies, according to colony area criteria used in A, from newly generated rho0, rho0 UA, and rho0 A sources. The bar represents the average percentage of colonies of each size ± SEM in three independent experiments (n = 68–322 colonies per experiment per condition; ***p < 0.001; and ****p < 0.0001, by one-way ANOVA with Tukey post-hoc test). (C) Growth rates of rho+, rho0 UA, and rho0 A cells. The bar shows pooled average ± SEM of the maximum OD600/h from three independent experiments (n = 6–12 replicates per conditions; **p < 0.01; and ***p < 0.001, by one-way ANOVA with Tukey post-hoc test). (D) Quantitation of progression from G1 to G2 for rho+, rho0 UA, and rho0 A cells. Cells were incubated with mating pheromone (alpha factor), which arrests cells in the G1 phase of the cell cycle. Progression of cells from G1 to G2 stages of the cell cycle was monitored after release from G1 using flow cytometry to measure the levels of propidium iodine stained DNA. Progression was measured as the fold change in the fraction of cells in G1 phase at the time specified, relative to the fraction of cells that were in G1 at the time of release from alpha factor-induced G1 arrest (cells in G1 at t0/cells in G1 at tx, 50,000 events measured per timepoint per strain).

Interestingly, we find that UA rho0 cells give rise to cells that form small colonies and exhibit low growth rates, but they also give rise to cells that form large colonies and exhibit high growth rates. Thus, UA rho0 cells give rise to both UA and adapted rho0 cells. In contrast, adapted rho0 cells give rise only to adapted cells, which form large colonies and exhibit high growth rates (Figure 1, A–C). These data confirm previous findings that the adaptation observed in rho0 cells is heritable (Dirick et al., 2014). Moreover, rho0 cells continue to adapt as they are propagated. We find that the colonies produced from adapted rho0 cells are significantly larger than those obtained from newly generated rho0 cells (Figure 1B).

Our initial observation of rho0 adaptation was made in cells where mtDNA was eliminated by treatment with EtBr. However, we also observe adaptation in rho0 cells in which mtDNA has been lost as a result of the deletion of MGM101 (Supplemental Figure S1), which encodes a protein that mediates mtDNA repair and is required for mtDNA maintenance (Chen et al., 1993). Adaptation to loss of mtDNA has also been documented in yeast that undergo spontaneous mtDNA loss and in yeast in which mtDNA loss was induced by expression of a dominant-negative form of the mtDNA polymerase MIP1 (Veatch et al., 2009; Dirick et al., 2014). Thus, this adaptation is a general response to loss of mtDNA and not a consequence of the method used to delete mtDNA. The collective findings that yeast cells adapt to loss of mtDNA raise the possibility that mammalian rho0 cells can also adapt to loss of mtDNA.

Finally, we find that the increase in growth rate that occurs in adapted rho0 colonies is due at least in part to bypass of the mtDNA inheritance checkpoint (Figure 1D). We monitored cell cycle progression in synchronized yeast cells using flow cytometry to measure DNA content. Wild-type rho+ cells, which contain mtDNA, transition from G1 to G2 phase 60–130 min after release from G1 arrest. In contrast, cells from small rho0 colonies exhibit severe defects in transition from G1 to S phase. Finally, cells from large rho0 colonies progress through the cell cycle similarly to rho+ cells. In the example shown, the lag time for entry into the cell cycle after release from G1 arrest and the cycling times are shorter in adapted rho0 cells than in rho+ cells.

Effects of mtDNA on lifespan and mitochondrial redox stateOne consequence of loss of mtDNA and the associated mitochondrial respiratory activity is a decrease in ∆Ψ. Previous studies revealed that ∆Ψ increases as yeast adapt to loss of mtDNA (Veatch et al., 2009). To further characterize the adaptation process, we studied mitochondrial redox state in UA and adapted rho0 cells using a redox-sensing variant of GFP (roGFP) (Figure 2, A and B) (Hanson et al., 2004). Our previous studies using mitochondria-­targeted roGFP and other biosensors revealed that fitter mitochondria that are more reduced, contain less mitochondrial superoxide, and have higher ∆Ψ are preferentially inherited by yeast daughter cells and that this affects yeast cell fitness and lifespan (McFaline-Figueroa et al., 2011).FIGURE 2:Adaptation to loss of mtDNA results in increased lifespan and mitochondrial quality. (A) Representative images of redox state of mitochondria in WT rho+, rho0 UA, and rho0 A cells measured with mito-roGFP1. Reduced:oxidized mito-roGFP ratio images are shown. Color scale in the bottom panel shows the dynamic range of ratios, with warmer colors indicating a more reducing environment. (B) Quantitation of reduced:oxidized mito-roGFP ratios in WT rho+, rho0 UA, rho0 A, and rho0 AA cells. The box indicates the middle quartile with the midline representing the median; whiskers show the minimum and maximum values. Representative trial from three independent experiments (n = 84–104 for each condition, *p < 0.05, ****p < 0.0001, by one-way ANOVA with Tukey post-hoc test for multiple comparisons). (C) Mean generation time was measured during RLS determination shown in D and E, as the time elapsed between the emergence of two consecutive buds. Bars show average ± SEM for one independent experiment (n = 30–51 cells per condition; **p < 0.01; and ****p < 0.0001, by Kruskal–Wallis test with Dunn’s post-hoc test for multiple comparisons). (D, E) RLS determination for WT rho+, rho0 UA, and rho0 A cells. (n = 40–52 starting new daughters per condition. Statistical significance between RLS survival curves was tested with Log-rank (Mantel–Cox) test where p < 0.05).Loss of mtDNA or adaptation to that loss does not affect mitochondrial quality control during inheritance: daughter cells inherit mitochondria that are more reduced and therefore higher functioning in adapted and UA rho0 cells (Supplemental Figure S2). However, mitochondria in UA rho0 cells are significantly more oxidized compared with mitochondria in rho+ cells. Moreover, mitochondrial redox state improves in rho0 cells as they adapt. Here, we evaluated mitochondrial redox state during early and later stages of adaptation (3 and 5 d after deletion of mtDNA, respectively). We detect a subtle but statistically significant increase in the reducing potential of mitochondria during early stages of adaptation. Furthermore, mitochondrial reducing potential continues to increase during late stages of adaptation, approaching levels observed in rho+ cells (Figure 2, A and B).Interestingly, the more reducing mitochondrial environment observed on adaptation of rho0 cells is not accompanied by lower mitochondrial superoxide levels. Using dihydroethidium (DHE) to detect superoxides in living yeast cells, we confirmed our previous findings that all detectable superoxides in yeast colocalize with mitochondria (McFaline-Figueroa et al., 2011). Beyond this, we find that deletion of mtDNA results in loss of all detectable superoxides in mitochondria and that mitochondrial superoxide levels do not change as cells adapt to loss of mtDNA. (Supplemental Figure S2).Thus, we detect two additional events that occur during adaptation to loss of mtDNA: bypass of the mtDNA inheritance checkpoint and improved mitochondrial redox state. Interestingly, loss of mtDNA has no effect on the mitochondrial quality control mechanisms that promote inheritance of higher-functioning mitochondria by yeast daughter cells (McFaline-Figueroa et al., 2011). Moreover, since loss of mtDNA results in loss of all detectable mitochondrial ROS, a phenotype that is stable during rho0 cell adaptation, mitochondrial ROS is therefore not responsible for changes in the redox state of the organelle as rho0 cells adapt.Equally important, we find that rho0 cells undergo an extension of RLS as they adapt (Figure 2, C–E). The mean RLS of rho+ cells in the genetic background used in these studies is 20–25 generations. The RLS of UA and adapted rho0 cells is variable. However, the mean RLS of UA rho0 cells is always significantly lower than that of rho+ cells, and the generation time of UA rho0 cells is longer. In contrast, the mean RLS of adapted rho0 cells is higher than that of UA rho0 cells (unpublished data). In the example shown, the RLS of the adapted rho0 cells is greater than that of rho+ cells.Thus, we detect reciprocal interactions between mtDNA and lifespan in budding yeast. Loss of mtDNA results in reduced RLS. Conversely, RLS extension is one consequence of the adaptation of yeast to loss of mtDNA. Our findings provide evidence for a role for mtDNA in lifespan control in yeast and raise the possibility that loss of mtDNA or mtDNA function may also affect lifespan in other eukaryotes. Collectively, our findings reconcile previous observations that loss of mtDNA has diverse effects on RLS in yeast: the variable RLS observed in rho0 cells may be a consequence of analysis of RLS in cells in different states of adaption to loss of mtDNA.

Effects of mtDNA on lifespan and mitochondrial redox state

One consequence of loss of mtDNA and the associated mitochondrial respiratory activity is a decrease in ∆Ψ. Previous studies revealed that ∆Ψ increases as yeast adapt to loss of mtDNA (Veatch et al., 2009). To further characterize the adaptation process, we studied mitochondrial redox state in UA and adapted rho0 cells using a redox-sensing variant of GFP (roGFP) (Figure 2, A and B) (Hanson et al., 2004). Our previous studies using mitochondria-­targeted roGFP and other biosensors revealed that fitter mitochondria that are more reduced, contain less mitochondrial superoxide, and have higher ∆Ψ are preferentially inherited by yeast daughter cells and that this affects yeast cell fitness and lifespan (McFaline-Figueroa et al., 2011).

Adaptation to loss of mtDNA results in increased lifespan and mitochondrial quality. (A) Representative images of redox state of mitochondria in WT rho+, rho0 UA, and rho0 A cells measured with mito-roGFP1. Reduced:oxidized mito-roGFP ratio images are shown. Color scale in the bottom panel shows the dynamic range of ratios, with warmer colors indicating a more reducing environment. (B) Quantitation of reduced:oxidized mito-roGFP ratios in WT rho+, rho0 UA, rho0 A, and rho0 AA cells. The box indicates the middle quartile with the midline representing the median; whiskers show the minimum and maximum values. Representative trial from three independent experiments (n = 84–104 for each condition, *p < 0.05, ****p < 0.0001, by one-way ANOVA with Tukey post-hoc test for multiple comparisons). (C) Mean generation time was measured during RLS determination shown in D and E, as the time elapsed between the emergence of two consecutive buds. Bars show average ± SEM for one independent experiment (n = 30–51 cells per condition; **p < 0.01; and ****p < 0.0001, by Kruskal–Wallis test with Dunn’s post-hoc test for multiple comparisons). (D, E) RLS determination for WT rho+, rho0 UA, and rho0 A cells. (n = 40–52 starting new daughters per condition. Statistical significance between RLS survival curves was tested with Log-rank (Mantel–Cox) test where p < 0.05).

Loss of mtDNA or adaptation to that loss does not affect mitochondrial quality control during inheritance: daughter cells inherit mitochondria that are more reduced and therefore higher functioning in adapted and UA rho0 cells (Supplemental Figure S2). However, mitochondria in UA rho0 cells are significantly more oxidized compared with mitochondria in rho+ cells. Moreover, mitochondrial redox state improves in rho0 cells as they adapt. Here, we evaluated mitochondrial redox state during early and later stages of adaptation (3 and 5 d after deletion of mtDNA, respectively). We detect a subtle but statistically significant increase in the reducing potential of mitochondria during early stages of adaptation. Furthermore, mitochondrial reducing potential continues to increase during late stages of adaptation, approaching levels observed in rho+ cells (Figure 2, A and B).

Interestingly, the more reducing mitochondrial environment observed on adaptation of rho0 cells is not accompanied by lower mitochondrial superoxide levels. Using dihydroethidium (DHE) to detect superoxides in living yeast cells, we confirmed our previous findings that all detectable superoxides in yeast colocalize with mitochondria (McFaline-Figueroa et al., 2011). Beyond this, we find that deletion of mtDNA results in loss of all detectable superoxides in mitochondria and that mitochondrial superoxide levels do not change as cells adapt to loss of mtDNA. (Supplemental Figure S2).

Thus, we detect two additional events that occur during adaptation to loss of mtDNA: bypass of the mtDNA inheritance checkpoint and improved mitochondrial redox state. Interestingly, loss of mtDNA has no effect on the mitochondrial quality control mechanisms that promote inheritance of higher-functioning mitochondria by yeast daughter cells (McFaline-Figueroa et al., 2011). Moreover, since loss of mtDNA results in loss of all detectable mitochondrial ROS, a phenotype that is stable during rho0 cell adaptation, mitochondrial ROS is therefore not responsible for changes in the redox state of the organelle as rho0 cells adapt.

Equally important, we find that rho0 cells undergo an extension of RLS as they adapt (Figure 2, C–E). The mean RLS of rho+ cells in the genetic background used in these studies is 20–25 generations. The RLS of UA and adapted rho0 cells is variable. However, the mean RLS of UA rho0 cells is always significantly lower than that of rho+ cells, and the generation time of UA rho0 cells is longer. In contrast, the mean RLS of adapted rho0 cells is higher than that of UA rho0 cells (unpublished data). In the example shown, the RLS of the adapted rho0 cells is greater than that of rho+ cells.

Thus, we detect reciprocal interactions between mtDNA and lifespan in budding yeast. Loss of mtDNA results in reduced RLS. Conversely, RLS extension is one consequence of the adaptation of yeast to loss of mtDNA. Our findings provide evidence for a role for mtDNA in lifespan control in yeast and raise the possibility that loss of mtDNA or mtDNA function may also affect lifespan in other eukaryotes. Collectively, our findings reconcile previous observations that loss of mtDNA has diverse effects on RLS in yeast: the variable RLS observed in rho0 cells may be a consequence of analysis of RLS in cells in different states of adaption to loss of mtDNA.

Subtelomeric genes are silenced in adapted rho 0 cellsSince adaptation is heritable, we used RNA-Seq to compare the transcriptomes of UA rho0 cells and rho0 cells at different stages of adaptation (Figure 3 and Supplemental Table S1). Here, we used rho0 cells 3 d after EtBR-mediated loss of mtDNA as a model for early-stage adaptation and rho0 cells >10 d after loss of mtDNA induced by deletion of MGM101 as a model for late-stage adaptation.FIGURE 3:Transient up-regulation of genes occurs during adaptation to loss of mtDNA. Revigo plot of GO terms associated with genes that are up-regulated in early-stage adapted rho0 cells compared with UA rho0 cells (A) and later-stage compared with early-stage adapted rho0 cells (B). Bubbles with cooler colors represent more significant p values; the size of the bubble indicates the frequency of the GO term. X and Y coordinates are derived by applying multidimensional scaling to a matrix of the GO terms’ semantic similarities. (C) Average fold change of transcript levels for subtelomeric genes HSP30 and ADE17 in early-stage adapted rho0 cells (A) and late-stage adapted cells (AA), relative to WT rho0 UA cells. Fold change was calculated as 2-ΔΔCT with actin serving as the endogenous control for each sample. Averages and SEM from n = 3 independent trials are shown (*p < 0.05, **p < 0.001 by one-way ANOVA with Dunnett’s multiple comparisons test).We find that metabolic reprogramming to compensate for loss of mitochondrial metabolic activity occurs during early stages of adaptation in rho0 cells and identified biomarkers that can be used to assess rho0 adaptation (Figure 3 and Supplemental Table S1). Specifically, we find that pathways for amino acid and purine biosynthesis, essential functions of mitochondria, are up-regulated during early-stage adaptation in rho0 cells. Consistent with this, our quantitative PCR (qPCR) analysis indicates that the level of mRNA for ADE17, which encodes a purine biosynthetic enzyme, increases during early stages of adaption and, during later stages of adaptation, returns to levels observed in UA rho0 cells (Figure 3; Supplemental Figure S3A).In contrast, we find that oxidant stress response genes are up-regulated during late stages of adaptation (Figure 3 and Supplemental Table S1). For example, spermine is a polyamine that is induced by stressors in prokaryotes and eukaryotes, can protect DNA from oxidative damage, decreases sensitivity to oxidative stress, and has been implicated in activation of stress response gene expression (Rider et al., 2007; Pegg and Michael, 2010). We find that spermine transport and biosynthesis are up-regulated during late-stage adaptation of rho0 cells. As described above, mitochondria become more reducing, but we see no obvious change in mitochondrial superoxide levels during adaptation to loss of mtDNA. Therefore, it is possible that the change in mitochondrial redox state observed in adapted rho0 cells is due to up-regulation of spermine and other oxidant stress response factors.It is interesting that oxidant stress response genes are up-regulated in late-stage adapted rho0 cells, even though there is no change in mitochondrial superoxide levels in UA and adapted rho0 cells. On the other hand, we detect superoxides that do not colocalize with mitochondria in UA rho0 cells. Therefore, it is possible that extramitochondrial ROS drives the mitochondrial antioxidant response.Our transcriptome analysis also revealed that specific subtelomeric genes are preferentially silenced in rho0 cells in both early and late stages of adaptation (Supplemental Tables S1 and S2). We find that 24% of the transcripts that are down-regulated during early adaptation are encoded by genes that lie within 25 kb of the telomere and are therefore subtelomeric. Indeed, the subtelomeric genes that appear to be silenced in adapted rho0 cells are present in 11 of the 16 yeast chromosomes (Figure 4, A and B).FIGURE 4:Subtelomeric gene silencing occurs during adaptation to loss of mtDNA. (A) The most down-regulated genes in early stage (A) vs. UA rho0 cells, listed according to their chromosomal loci. Down-regulated genes were defined as those exhibiting a Log2 fold change <–1.5 and statistical significance of p < 0.05. Distance from telomere denotes the distance from the gene locus to the closest telomere in the chromosome. (B) Histogram showing distribution of distances from nearest telomere of the 13 most down-regulated genes shown in A. One gene with a distance of 93 kb is not shown. (C) Average fold change of transcript levels for subtelomeric genes PAU24, DAN1, and FIT3 in early-stage adapted rho0 (A), and late-stage adapted cells (AA), compared with WT rho0 UA. Fold change was calculated as 2-ΔΔCT with actin serving as the endogenous control for each sample. Averages and SEM from n = 3 independent trials are shown (**p < 0.01, ****p < 0.0001 by one-way ANOVA with Dunnett’s multiple comparisons test).Early studies supported the model that subtelomeric silencing is a consequence of spreading of silencing mediators (e.g., the Sir2/3/4 protein complex) from telomeres to subtelomeres (Gottschling et al., 1990; Renauld et al., 1993). However, other studies raised questions regarding the generality of this model. Indeed, transcriptome analysis and chromatin immunoprecipitation studies revealed that the Sir2/3/4 complex localizes to discrete, noncontinuous sites on subtelomeres and is responsible for silencing of only 6% of the genes in subtelomeres (Ellahi et al., 2015). Thus, available evidence indicates that telomere positioning effects do not contribute to subtelomeric gene silencing by the Sir2/3/4 complex. Our transcriptome analysis indicates that only a limited number of genes are silenced within subtelomeres in adapted rho0 cells. In cases where more than one subtelomeric gene appears to be silenced, genes are discontinuous. Thus, there are no obvious telomere positioning effects in the subtelomeric gene silencing we observe in adapted rho0 cells.On the other hand, many of the subtelomeric genes that are silenced during rho0 cell adaptation are functionally related. Previous studies indicate that newly generated rho0 cells exhibit a transcription signature of iron starvation (e.g., increased expression of iron transport and homeostasis proteins), which likely reflects compensatory mechanisms to promote the essential process of iron–sulfur cluster formation in mitochondria (Veatch et al., 2009). We find that some of the subtelomeric genes that are silenced during early and late adaptation encode iron transport proteins. The other major class of subtelomeric genes that are silenced in both early- and late-stage adaptation are stress response genes including proteins in the seripauperin multigene family (Supplemental Tables S3 and S4).Indeed, using qPCR to quantify cellular mRNA levels, we confirmed that the transcripts of two seripauperin family proteins (PAU24 and DAN1) and one iron transport gene (FIT3) that are silenced in adapted rho0 cells based on RNA-Seq analysis are present in reduced levels in adapted compared with UA rho0 cells (Figure 4C; Supplemental Figure S3B). Importantly, these genes are silenced during both early and later stages of adaptation, indicating that subtelomeric silencing is involved in the regulation of adaptation.

Subtelomeric genes are silenced in adapted rho 0 cells

Since adaptation is heritable, we used RNA-Seq to compare the transcriptomes of UA rho0 cells and rho0 cells at different stages of adaptation (Figure 3 and Supplemental Table S1). Here, we used rho0 cells 3 d after EtBR-mediated loss of mtDNA as a model for early-stage adaptation and rho0 cells >10 d after loss of mtDNA induced by deletion of MGM101 as a model for late-stage adaptation.

Transient up-regulation of genes occurs during adaptation to loss of mtDNA. Revigo plot of GO terms associated with genes that are up-regulated in early-stage adapted rho0 cells compared with UA rho0 cells (A) and later-stage compared with early-stage adapted rho0 cells (B). Bubbles with cooler colors represent more significant p values; the size of the bubble indicates the frequency of the GO term. X and Y coordinates are derived by applying multidimensional scaling to a matrix of the GO terms’ semantic similarities. (C) Average fold change of transcript levels for subtelomeric genes HSP30 and ADE17 in early-stage adapted rho0 cells (A) and late-stage adapted cells (AA), relative to WT rho0 UA cells. Fold change was calculated as 2-ΔΔCT with actin serving as the endogenous control for each sample. Averages and SEM from n = 3 independent trials are shown (*p < 0.05, **p < 0.001 by one-way ANOVA with Dunnett’s multiple comparisons test).

We find that metabolic reprogramming to compensate for loss of mitochondrial metabolic activity occurs during early stages of adaptation in rho0 cells and identified biomarkers that can be used to assess rho0 adaptation (Figure 3 and Supplemental Table S1). Specifically, we find that pathways for amino acid and purine biosynthesis, essential functions of mitochondria, are up-regulated during early-stage adaptation in rho0 cells. Consistent with this, our quantitative PCR (qPCR) analysis indicates that the level of mRNA for ADE17, which encodes a purine biosynthetic enzyme, increases during early stages of adaption and, during later stages of adaptation, returns to levels observed in UA rho0 cells (Figure 3; Supplemental Figure S3A).

In contrast, we find that oxidant stress response genes are up-regulated during late stages of adaptation (Figure 3 and Supplemental Table S1). For example, spermine is a polyamine that is induced by stressors in prokaryotes and eukaryotes, can protect DNA from oxidative damage, decreases sensitivity to oxidative stress, and has been implicated in activation of stress response gene expression (Rider et al., 2007; Pegg and Michael, 2010). We find that spermine transport and biosynthesis are up-regulated during late-stage adaptation of rho0 cells. As described above, mitochondria become more reducing, but we see no obvious change in mitochondrial superoxide levels during adaptation to loss of mtDNA. Therefore, it is possible that the change in mitochondrial redox state observed in adapted rho0 cells is due to up-regulation of spermine and other oxidant stress response factors.

It is interesting that oxidant stress response genes are up-regulated in late-stage adapted rho0 cells, even though there is no change in mitochondrial superoxide levels in UA and adapted rho0 cells. On the other hand, we detect superoxides that do not colocalize with mitochondria in UA rho0 cells. Therefore, it is possible that extramitochondrial ROS drives the mitochondrial antioxidant response.

Our transcriptome analysis also revealed that specific subtelomeric genes are preferentially silenced in rho0 cells in both early and late stages of adaptation (Supplemental Tables S1 and S2). We find that 24% of the transcripts that are down-regulated during early adaptation are encoded by genes that lie within 25 kb of the telomere and are therefore subtelomeric. Indeed, the subtelomeric genes that appear to be silenced in adapted rho0 cells are present in 11 of the 16 yeast chromosomes (Figure 4, A and B).

Subtelomeric gene silencing occurs during adaptation to loss of mtDNA. (A) The most down-regulated genes in early stage (A) vs. UA rho0 cells, listed according to their chromosomal loci. Down-regulated genes were defined as those exhibiting a Log2 fold change <–1.5 and statistical significance of p < 0.05. Distance from telomere denotes the distance from the gene locus to the closest telomere in the chromosome. (B) Histogram showing distribution of distances from nearest telomere of the 13 most down-regulated genes shown in A. One gene with a distance of 93 kb is not shown. (C) Average fold change of transcript levels for subtelomeric genes PAU24, DAN1, and FIT3 in early-stage adapted rho0 (A), and late-stage adapted cells (AA), compared with WT rho0 UA. Fold change was calculated as 2-ΔΔCT with actin serving as the endogenous control for each sample. Averages and SEM from n = 3 independent trials are shown (**p < 0.01, ****p < 0.0001 by one-way ANOVA with Dunnett’s multiple comparisons test).

Early studies supported the model that subtelomeric silencing is a consequence of spreading of silencing mediators (e.g., the Sir2/3/4 protein complex) from telomeres to subtelomeres (Gottschling et al., 1990; Renauld et al., 1993). However, other studies raised questions regarding the generality of this model. Indeed, transcriptome analysis and chromatin immunoprecipitation studies revealed that the Sir2/3/4 complex localizes to discrete, noncontinuous sites on subtelomeres and is responsible for silencing of only 6% of the genes in subtelomeres (Ellahi et al., 2015). Thus, available evidence indicates that telomere positioning effects do not contribute to subtelomeric gene silencing by the Sir2/3/4 complex. Our transcriptome analysis indicates that only a limited number of genes are silenced within subtelomeres in adapted rho0 cells. In cases where more than one subtelomeric gene appears to be silenced, genes are discontinuous. Thus, there are no obvious telomere positioning effects in the subtelomeric gene silencing we observe in adapted rho0 cells.

On the other hand, many of the subtelomeric genes that are silenced during rho0 cell adaptation are functionally related. Previous studies indicate that newly generated rho0 cells exhibit a transcription signature of iron starvation (e.g., increased expression of iron transport and homeostasis proteins), which likely reflects compensatory mechanisms to promote the essential process of iron–sulfur cluster formation in mitochondria (Veatch et al., 2009). We find that some of the subtelomeric genes that are silenced during early and late adaptation encode iron transport proteins. The other major class of subtelomeric genes that are silenced in both early- and late-stage adaptation are stress response genes including proteins in the seripauperin multigene family (Supplemental Tables S3 and S4).

Indeed, using qPCR to quantify cellular mRNA levels, we confirmed that the transcripts of two seripauperin family proteins (PAU24 and DAN1) and one iron transport gene (FIT3) that are silenced in adapted rho0 cells based on RNA-Seq analysis are present in reduced levels in adapted compared with UA rho0 cells (Figure 4C; Supplemental Figure S3B). Importantly, these genes are silenced during both early and later stages of adaptation, indicating that subtelomeric silencing is involved in the regulation of adaptation.

SIR3 is required for adaptation to loss of mtDNATo determine whether adaptation to loss of mtDNA is due to subtelomeric silencing, we studied the effect of deletion of SIR3 on adaptation in rho0 cells. We deleted mtDNA in sir3∆ cells and found that sir3∆ rho0 cells form large and small colonies when propagated on solid media. We also find that small sir3∆ rho0 cell colonies give rise to large and small colonies, while large sir3∆ rho0 cell colonies give rise primarily to large colonies (Figure 5A). Thus, sir3∆ rho0 cells have the capacity to adapt to loss of mtDNA.FIGURE 5:Sir3p is required for efficient adaptation to loss of mtDNA. (A) Percentage of UA and A colonies, according to colony area criteria used in Figure 1A, arising from newly generated sir3Δ rho0 and from UA and adapted colonies propagated from newly generated sir3Δ rho0 cells. The bar shows the average percentage distribution of colony sizes ± SEM of three independent experiments (n = 85–394 per condition per experiment; **p < 0.01, and ****p < 0.0001 by one-way ANOVA with Tukey post-hoc test). (B) Growth rates of sir3Δ rho+ cells and cells from UA (gray column) and adapted (A, light blue) sir3Δ rho0 colonies. Bars show pooled average ± SEM of maximum OD600/h from three independent experiments (n = 6–12 replicates per conditions; ***p < 0.001 and ****p < 0.0001, by one-way ANOVA with Tukey post-hoc test). (C) Gene silencing rate changes were determined by the change in ΔCt between 3 and 5 d of adaptation for PAU24, DAN1, and FIT3 in rho0 and sir3∆ rho0 cells. Averages and SEM from n = 3 independent trials are shown (*p < 0.05, by Student’s t test). (D) Representative images of redox state of mitochondria measured with mito-roGFP1 in sir3Δ rho+ cells, and in sir3Δ rho0 cells from UA and adapted (A) colonies. (E) Quantitation of reduced:oxidized mito-roGFP ratios in sir3Δ rho+ cells, and sir3Δ rho0 cells from UA and A colonies. Box and whiskers are defined as in Figure 2 (n = 48–89 for each condition; ****p < 0.0001, by one-way ANOVA with Tukey post-hoc test). (F) RLS measurements for sir3Δ rho0 UA and A cells performed as described in Materials and Methods (n = 40 new daughters observed per condition per RLS experiment. Statistical significance between RLS survival curves was tested with Log-rank (Mantel–Cox) where p < 0.05).However, we find that sir3∆ rho0 cells exhibit a fundamentally different adaptation phenotype compared with rho0 cells that contain wild-type SIR3. First, in contrast to adapted rho0 cells, which exhibit higher rates of growth in liquid media compared with UA rho0 cells, the growth rate of cells from large sir3∆ rho0 cell colonies in liquid media is significantly lower than that of UA sir3∆ rho0 cells (Figure 5B; Supplemental Figure S4A). Although it seems counterintuitive that cells from large sir3∆ rho0 colonies exhibit lower growth rates compared with cells from small sir3∆ rho0 colonies, there is a precedent for differential yeast growth rates on solid versus liquid media. For example, yeast bearing a deletion in one of the FLO genes, which are required for adhesion (flocculation) of yeast cells, exhibit increased colony size on solid media as well as a severe reduction in growth rate in liquid media compared with wild-type cells (e.g., Di Gianvito et al., 2017). Indeed, since genes that affect cell wall mannoproteins are silenced during adaptation to loss of mtDNA, it is possible that the large colony size of some sir3∆ rho0 cells is due to effects on yeast cell adhesion.Equally important, we detect a statistically significant decrease in the silencing rate of three subtelomeric genes (FIT3, DAN1, and PAU24) in sir3∆ rho0 cells compared with rho0 cells during the adaptation process (Figure 5C; Supplemental Figure S4B). In addition, there is no detectable increase in mitochondrial reducing potential during adaptation of sir3∆ rho0 cells (Figure 5, D and E). Finally, we find that the RLS of cells from adapted and UA sir3∆ rho0 cell colonies is indistinguishable. Thus, there is no improvement in mitochondrial redox state or extension of RLS in adapted sir3∆ rho0 cells (Figure 5F).Our findings support the model that Sir3p-dependent subtelomeric gene silencing is responsible for improved mitochondrial function and extended RLS associated with adaptation to loss of mtDNA. While it is not clear whether the increase in mitochondrial redox state is causative in the RLS-adaptive phenotype, our previous studies (Higuchi et al., 2013) indicate that rendering mitochondria more reducing is sufficient to extend RLS in yeast. Thus, it is possible that the lifespan extension observed in yeast that adapt to loss of mtDNA is due to the change in mitochondrial redox state.Previous studies revealed a role for lifespan-regulating genes in rho0 cell survival. Most yeast strains can tolerate loss of mtDNA. However, yeast carrying certain mutations (e.g., mitochondrial protein import or protein quality control) require mtDNA for survival (Dunn and Jensen, 2003; Senapin et al., 2003). Interestingly, deletion of conserved lifespan-regulating genes can suppress the lethality observed on loss of mtDNA in these “petite-negative” strains. Specifically, deletion of any of several proteins in the 60S ribosomal subunit (rpl13a, 37b, 12a, 20b, 19a, 6b, 14a, 43b, 34b, 35b, and 12b as well as rpp1b and 4a) can extend lifespan and suppress the lethality of loss of mtDNA in petite-negative yeast (Dunn et al., 2006). We do not detect changes in the levels of the transcripts of any of these RLS-extending genes. Thus, the lifespan extension observed in adapted rho0 cells is not due to alterations in the expression of ribosomal genes that affect rho0 cell viability.Previous studies also revealed that elevated mitochondrial ROS results in extension of CLS in yeast. The extended CLS observed in these studies is a consequence of Rad53p-dependent silencing of the DNA demethylase Rph1p and the resulting increase in subtelomeric gene silencing by Sir3p (Schroeder et al., 2013). While the lifespan extension observed in yeast exposed to elevated mitochondrial ROS and in yeast that are adapting to loss of mtDNA are both Sir3p-dependent, there is evidence that these processes are mechanistically distinct. First, yeast undergoing chronological and replicative aging are in different stages of the yeast life cycle. Chronological aging occurs in yeast that have encountered nutrient limitations and/or accumulation of damaging metabolites and cannot undergo cell division. In contrast, yeast undergoing replicative aging are cell division-competent and not nutrient-­limited. Second, there is no detectable mitochondrial superoxide in UA rho0 cells or adapted rho0 cells at any stage of adaptation. Third, with the exception of one gene, there is no overlap between the 43 genes that are repressed in stationary-phase yeast with elevated mitochondrial ROS and the 42 genes that are down-regulated in early-stage adapted (A) rho0 cells, or the 93 genes that are repressed in later-stage adapted (AA) rho0 cells. Indeed, the genes that are down-regulated in adapted rho0 cells are different from the genes that undergo Sir2/3/4-dependent down-regulation in rho+ cells (Ellahi et al., 2015). Finally, we find that silencing of the three subtelomeric genes analyzed that are down-regulated in adapted rho0 cells does not require RPH1 (unpublished data). These findings provide additional support for the notion that differential Sir3p-dependent subtelomeric gene silencing events occur in response to different environmental or cellular conditions. That is, Sir3p regulates different genes in dividing rho+ cells in oxidatively stressed quiescent rho+ cells and in rho0 cells that are adapting to loss of mtDNA.Overall, our studies reveal new links between mtDNA and lifespan control. Loss of mtDNA results in reduced RLS. On the other hand, extension of RLS is one consequence of adaptation to loss of mtDNA. We also identified distinct transcriptome signatures during early and late stages of adaptation to loss of mtDNA. Metabolic programming to compensate for loss of key mitochondrial functions occurs early in the adaptation process while up-regulation of specific stress response genes occurs later in that process. Finally, we find that silencing of some subtelomeric genes occurs during early and later stages of adaptation and obtained evidence for a role for Sir3p in this process and for improved mitochondrial function and extended RLS during rho0 cell adaptation. Ongoing studies focus on the gene(s) responsible for the RLS extension that occurs during adaptation to loss of mtDNA and how those genes contribute to that process.

SIR3 is required for adaptation to loss of mtDNA

To determine whether adaptation to loss of mtDNA is due to subtelomeric silencing, we studied the effect of deletion of SIR3 on adaptation in rho0 cells. We deleted mtDNA in sir3∆ cells and found that sir3∆ rho0 cells form large and small colonies when propagated on solid media. We also find that small sir3∆ rho0 cell colonies give rise to large and small colonies, while large sir3∆ rho0 cell colonies give rise primarily to large colonies (Figure 5A). Thus, sir3∆ rho0 cells have the capacity to adapt to loss of mtDNA.

Sir3p is required for efficient adaptation to loss of mtDNA. (A) Percentage of UA and A colonies, according to colony area criteria used in Figure 1A, arising from newly generated sir3Δ rho0 and from UA and adapted colonies propagated from newly generated sir3Δ rho0 cells. The bar shows the average percentage distribution of colony sizes ± SEM of three independent experiments (n = 85–394 per condition per experiment; **p < 0.01, and ****p < 0.0001 by one-way ANOVA with Tukey post-hoc test). (B) Growth rates of sir3Δ rho+ cells and cells from UA (gray column) and adapted (A, light blue) sir3Δ rho0 colonies. Bars show pooled average ± SEM of maximum OD600/h from three independent experiments (n = 6–12 replicates per conditions; ***p < 0.001 and ****p < 0.0001, by one-way ANOVA with Tukey post-hoc test). (C) Gene silencing rate changes were determined by the change in ΔCt between 3 and 5 d of adaptation for PAU24, DAN1, and FIT3 in rho0 and sir3∆ rho0 cells. Averages and SEM from n = 3 independent trials are shown (*p < 0.05, by Student’s t test). (D) Representative images of redox state of mitochondria measured with mito-roGFP1 in sir3Δ rho+ cells, and in sir3Δ rho0 cells from UA and adapted (A) colonies. (E) Quantitation of reduced:oxidized mito-roGFP ratios in sir3Δ rho+ cells, and sir3Δ rho0 cells from UA and A colonies. Box and whiskers are defined as in Figure 2 (n = 48–89 for each condition; ****p < 0.0001, by one-way ANOVA with Tukey post-hoc test). (F) RLS measurements for sir3Δ rho0 UA and A cells performed as described in Materials and Methods (n = 40 new daughters observed per condition per RLS experiment. Statistical significance between RLS survival curves was tested with Log-rank (Mantel–Cox) where p < 0.05).

However, we find that sir3∆ rho0 cells exhibit a fundamentally different adaptation phenotype compared with rho0 cells that contain wild-type SIR3. First, in contrast to adapted rho0 cells, which exhibit higher rates of growth in liquid media compared with UA rho0 cells, the growth rate of cells from large sir3∆ rho0 cell colonies in liquid media is significantly lower than that of UA sir3∆ rho0 cells (Figure 5B; Supplemental Figure S4A). Although it seems counterintuitive that cells from large sir3∆ rho0 colonies exhibit lower growth rates compared with cells from small sir3∆ rho0 colonies, there is a precedent for differential yeast growth rates on solid versus liquid media. For example, yeast bearing a deletion in one of the FLO genes, which are required for adhesion (flocculation) of yeast cells, exhibit increased colony size on solid media as well as a severe reduction in growth rate in liquid media compared with wild-type cells (e.g., Di Gianvito et al., 2017). Indeed, since genes that affect cell wall mannoproteins are silenced during adaptation to loss of mtDNA, it is possible that the large colony size of some sir3∆ rho0 cells is due to effects on yeast cell adhesion.

Equally important, we detect a statistically significant decrease in the silencing rate of three subtelomeric genes (FIT3, DAN1, and PAU24) in sir3∆ rho0 cells compared with rho0 cells during the adaptation process (Figure 5C; Supplemental Figure S4B). In addition, there is no detectable increase in mitochondrial reducing potential during adaptation of sir3∆ rho0 cells (Figure 5, D and E). Finally, we find that the RLS of cells from adapted and UA sir3∆ rho0 cell colonies is indistinguishable. Thus, there is no improvement in mitochondrial redox state or extension of RLS in adapted sir3∆ rho0 cells (Figure 5F).

Our findings support the model that Sir3p-dependent subtelomeric gene silencing is responsible for improved mitochondrial function and extended RLS associated with adaptation to loss of mtDNA. While it is not clear whether the increase in mitochondrial redox state is causative in the RLS-adaptive phenotype, our previous studies (Higuchi et al., 2013) indicate that rendering mitochondria more reducing is sufficient to extend RLS in yeast. Thus, it is possible that the lifespan extension observed in yeast that adapt to loss of mtDNA is due to the change in mitochondrial redox state.

Previous studies revealed a role for lifespan-regulating genes in rho0 cell survival. Most yeast strains can tolerate loss of mtDNA. However, yeast carrying certain mutations (e.g., mitochondrial protein import or protein quality control) require mtDNA for survival (Dunn and Jensen, 2003; Senapin et al., 2003). Interestingly, deletion of conserved lifespan-regulating genes can suppress the lethality observed on loss of mtDNA in these “petite-negative” strains. Specifically, deletion of any of several proteins in the 60S ribosomal subunit (rpl13a, 37b, 12a, 20b, 19a, 6b, 14a, 43b, 34b, 35b, and 12b as well as rpp1b and 4a) can extend lifespan and suppress the lethality of loss of mtDNA in petite-negative yeast (Dunn et al., 2006). We do not detect changes in the levels of the transcripts of any of these RLS-extending genes. Thus, the lifespan extension observed in adapted rho0 cells is not due to alterations in the expression of ribosomal genes that affect rho0 cell viability.

Previous studies also revealed that elevated mitochondrial ROS results in extension of CLS in yeast. The extended CLS observed in these studies is a consequence of Rad53p-dependent silencing of the DNA demethylase Rph1p and the resulting increase in subtelomeric gene silencing by Sir3p (Schroeder et al., 2013). While the lifespan extension observed in yeast exposed to elevated mitochondrial ROS and in yeast that are adapting to loss of mtDNA are both Sir3p-dependent, there is evidence that these processes are mechanistically distinct. First, yeast undergoing chronological and replicative aging are in different stages of the yeast life cycle. Chronological aging occurs in yeast that have encountered nutrient limitations and/or accumulation of damaging metabolites and cannot undergo cell division. In contrast, yeast undergoing replicative aging are cell division-competent and not nutrient-­limited. Second, there is no detectable mitochondrial superoxide in UA rho0 cells or adapted rho0 cells at any stage of adaptation. Third, with the exception of one gene, there is no overlap between the 43 genes that are repressed in stationary-phase yeast with elevated mitochondrial ROS and the 42 genes that are down-regulated in early-stage adapted (A) rho0 cells, or the 93 genes that are repressed in later-stage adapted (AA) rho0 cells. Indeed, the genes that are down-regulated in adapted rho0 cells are different from the genes that undergo Sir2/3/4-dependent down-regulation in rho+ cells (Ellahi et al., 2015). Finally, we find that silencing of the three subtelomeric genes analyzed that are down-regulated in adapted rho0 cells does not require RPH1 (unpublished data). These findings provide additional support for the notion that differential Sir3p-dependent subtelomeric gene silencing events occur in response to different environmental or cellular conditions. That is, Sir3p regulates different genes in dividing rho+ cells in oxidatively stressed quiescent rho+ cells and in rho0 cells that are adapting to loss of mtDNA.

Overall, our studies reveal new links between mtDNA and lifespan control. Loss of mtDNA results in reduced RLS. On the other hand, extension of RLS is one consequence of adaptation to loss of mtDNA. We also identified distinct transcriptome signatures during early and late stages of adaptation to loss of mtDNA. Metabolic programming to compensate for loss of key mitochondrial functions occurs early in the adaptation process while up-regulation of specific stress response genes occurs later in that process. Finally, we find that silencing of some subtelomeric genes occurs during early and later stages of adaptation and obtained evidence for a role for Sir3p in this process and for improved mitochondrial function and extended RLS during rho0 cell adaptation. Ongoing studies focus on the gene(s) responsible for the RLS extension that occurs during adaptation to loss of mtDNA and how those genes contribute to that process.

MATERIALS AND METHODSYeast strains and growth conditionsAll S. cerevisiae strains used in this study are derivatives of the wild-type BY4741 strain (MATa his3∆1 leu2∆0 met15∆0 ura3∆0) from Open Biosystems (Huntsville, AL). Yeast cells were cultivated and manipulated as described previously (Sherman, 2002). The rho0 cells were generated by treatment with EtBr as described in Fox et al. (1991). Briefly, each strain was grown in SC containing 25 μg/ml ethidium bromide (EtBr; Sigma, St. Louis, MO) for 48 h at 30°C with shaking at 220 rpm. Then cells were spread on YPD and incubated for 5 d at 30°C. The rho0 status was confirmed by lack of growth on plates containing a nonfermentable carbon source and an absence of mtDNA by DAPI (4’,6-diamidino-2-phenylindole) staining.For some studies, rho0 cells were generated by replacing MGM101 in rho+ cells with knockout cassettes containing the selectable marker LEU2 through homologous recombination according to previously described protocols (Longtine et al., 1998; Gauss et al., 2005). The primers, 5′ CTAAAAAAGGAAAGAAAGGACAAGTAGGAAGATCAGCGTACGTGCAGGTCGACAACCCTTAAT 3′ and 5′ ATATACTTACTAAAATTAGCTTATATGGTTCGCATATTGAGCAGCGTACGGATATCACCTA 3′, were used to amplify LEU2 from pOM13. Deletion of MGM101 was confirmed by PCR amplification of the locus using the following primers: 5′ CGAAATTTATCGACAGAATAATGG 3′ and 5′ GTACTGACACTACGCACTACC 3′.For each experiment needing mgm101Δ rho0 cells, MGM101 was freshly deleted to avoid further adaptation of rho0 cells during normal handling, passage and subculturing. Likewise, new EtBr-generated rho0 cells were generated for each experiment. SIR3 was deleted from BY4741 using the following primers: 5′ TTAAGAAAGTTGTTTTGTTCTAACAATTGGATTAGCTAAATGCAGGTCGACAACCCTTAAT 3′ and 5′ CATAGGCATATCTATGGCGGAAGTGAAAATGAATGTTGGTGGGCAGCGTACGGATATCACCTA 3′ to amplify URA3 from pOM12. Deletion of SIR3 was confirmed by PCR amplification of the locus using primers 5′ CACATAAGCAGCCCTTTCATC 3′ and 5′ GAATACAGAAGAGACTGCATG 3′.Colony size determinationYeast from the indicated colony type (rho0 strains obtained by EtBr treatment, or subcultured adapted or UA colonies from parental rho0 strains) were diluted and spread on YPD plates to generate single colonies. After 5 d of growth at 30°C, single colonies were imaged using a ChemiDoc MP (Bio-Rad). Images were processed with the open-source colony counting software OpenCFU to automatically count and measure the size of each colony (Geissmann, 2013). False-positive colonies resulting from noise were automatically removed by applying a –1 filter or manually removed from analysis. Colonies of area <0.72 mm2 (100 pixels) were categorized as small/UA, while colonies of area >0.72 mm2 were categorized as large/adapted.Growth ratesGrowth curves were measured using an automated plate reader (Tecan; Infinite M200, Research Triangle Park, NC). Each strain was grown to mid–log phase in rich, glucose-based media (YPD) and diluted to an OD600 of 0.07 (2.0 × 106 cells/ml). A diluted strain of 10 µl was added to a well containing 200 µl YPD in a 96-well plate. Cells were propagated at 30°C without shaking, and an optical density at 600 nm (OD600) was measured every 20 min for 72 h. For each strain, three independent colonies were tested in quadruplicate. The maximum growth rate was calculated using the greatest change in OD600 over a 240-min interval in 72 h.Cell cycle assayCell cycle progression was assessed by measuring amount of DNA as described previously (Breeden, 1997; Fortuna et al., 2001; Crider et al., 2012). Mid–log phase yeast were synchronized by incubating cells with 10–100 µM α-factor for 2.5 h on YPD with shaking at 30°C. Cells were released from arrest by washing with fresh YPD media and transferred to pheromone-free media. Cells were collected and fixed in 70% ethanol 0, 20, 40, 60, 80, 100, 120, 150, and 180 min after release from arrest. Cell cycle progression was assessed by measuring DNA content as described previously (Breeden, 1997; Fortuna et al., 2001; Crider et al., 2012). Briefly, cells were washed and digested with 250 µg/ml RNase in sodium citrate buffer, pH 7.5, for 2 h at 50°C. After RNase digestion, cells were digested with 20 mg/ml Proteinase K for 2 h at 50°C. DNA was stained resuspending cells in citrate buffer containing 16 µg/ml propidium iodide. DNA content was measured using a fluorescence-activated cell analyzer (LSRII, BD), and 50,000 events were recorded for each timepoint. The percentage of G1, S, and G2 phase cells was determined using FlowJo (FlowJo LLC, Ashland, OR).RLS determinationRLS measurements were performed as described previously (Erjavec et al., 2008) without alpha-factor synchronization. Frozen yeast strain stocks (stored at –80°C) were grown on YPD plates at 30°C, and rho0 cells were obtained by ethidium bromide treatment as previously described. Single colonies of each yeast strain were suspended in liquid YPD and grown at 30°C with shaking to mid–log phase (OD600 0.1–0.3). A 2-µl aliquot of cell suspension was applied to a YPD plate. Small-budded cells were isolated and arranged in a matrix using a micromanipulator mounted on a dissecting microscope (Zeiss, Thornwood, NY, or Singer Instruments, Watchet, UK). When the small buds completed growth, their mother cells were removed and discarded, and the remaining daughter cells were named virgin mother cells. After each replication, the new bud was removed and discarded. This was continued until all replication ceased. The mean generation time and number of daughter cells produced by each virgin mother cell were recorded.Assessing mitochondrial function using mitochondria-targeted roGFPMitochondrial redox state was measured as previously described in McFaline-Figueroa et al. (2011). Strains were transformed with a centromeric plasmid expressing mito-roGFP1 targeted to mitochondria using the ATP9 mitochondrial targeting sequence prior to EtBr treatment. After EtBr treatment as described previously, ∼40–50 small rho0 colonies and ∼8–10 large rho0 colonies were selected and grown for 12 h on selective media to obtain UA and A, respectively. These adapted cells were grown for another 24 h to obtain lAA. Images were acquired with an Axioskop 2 microscope equipped with a 100×/1.4 Plan-Apochromat objective (Zeiss, Thornwood, NY), an Orca-ER cooled CCD camera (Hamamatsu), and a pE-4000 LED illumination system (coolLED, Andover, UK). Images were acquired using NIS Elements 4.60 Lambda software (Nikon, Melville, NY). Oxidized and reduced channels were excited using a 365-nm LED and a 470-nm LED with a ET470/40× filter, respectively. All channels were acquired with a modified GFP filter (Zeiss filter 46 HE without excitation filter, dichroic FT 515, emission 535/30). Images were deconvolved using 60 iterations (100% confidence criterion) of a constrained iterative restoration algorithm and a theoretical PSF based on a 507-nm emission wavelength using Volocity 6.3 (Perkin-Elmer, Waltham, MA). The reduced:oxidized ratio channel was calculated by dividing the intensity of the reduced channel (λex = 470 nm, λem = 525 nm) by the intensity of the oxidized channel (λex = 365 nm, λem = 525 nm) after background subtraction and thresholding for each channel individually.RNA sequencingTranscriptome was analyzed as previously described in Vevea et al. (2015). RNA was extracted from mid–log phase A and UA rho0 yeast cells using the RNeasy kit (Qiagen, Germantown, MD). RNA quality was analyzed with Agilent 2100 Bioanalyzer using a Plant RNA Nano chip, and only RNA samples with RNA Integrity Number (RIN) scores >9 were used for subsequent RNA-Seq. The mRNA library was generated using Illumina TruSeq RNA prep kit after poly-A pull-down enrichment of mRNA from total RNA samples. RNA-Seq was performed on an Illumina HiSeq2500 generating 200 million 100-base pair single-end reads per lane, with 10 samples multiplexed per lane (average 30 million raw reads per sample) by the Columbia Genome Center. Data were analyzed using Tophat, Cufflinks, and Cuffdiff protocol as described on the Galaxy platform (Afgan et al., 2016). Differential expression was analyzed with DESeq2, an open-source differential gene expression analysis based on the negative binomial distribution (Love et al., 2014). Differentially expressed genes were then analyzed using Funspec to group the large sets of up-regulated and down-regulated genes into gene ontology (GO) terms (Robinson et al., 2002).cDNA synthesis and quantitative PCRRNA was extracted from mid–log phase UA and A rho0 cells using the RNeasy kit (Qiagen, Germantown, MD). RNA quality was analyzed as previously described and only RNA samples with RIN scores >9 were used for subsequent RT-PCR analysis. Genomic DNA contamination was removed from RNA samples with TURBO DNA-free Kit (Ambion, Carslbad, CA). DNA-free RNA (1 µg ) was used for cDNA synthesis performed with SuperScript IV First-Strand Synthesis System (Invitrogen, Waltham, MA). cDNA was diluted and used for quantitative PCR with PowerUp SYBR Green Master Mix (Applied Biosystems, Carlsbad, CA) for adapted, superadapted, and UA samples. Primers for qPCR were designed with NCBI Primer Blast (www.ncbi.nlm.nih.gov/tools/primer-blast/) with a PCR product size of 100 base pairs and max Tm difference of 2°C. Primer specificity and amplification efficiency for each primer set were validated with a standard curve. ACT1 was amplified with 5′ TCGCCTTGGACTTCGAACAA 3′ and 5′ CAAAGCTTCTGGGGCTCTGA 3′, ADE17 with 5′ CGTGACGCTGGTTTTCCAAT 3′ and 5′ CACCATGAACGGCAGGATGT 3′, HSP30 with 5′ GCTACGGACGATGTGGAAGA 3′ and 5′ CAGGTTCGGGTTCGTGGATT 3′, PAU24 with 5′ GGTATTGCCCCAGACCAAGT 3′ and 5′ GCACTAGAGATGGCTGGCTT 3′, DAN1 with 5′ GTACTGACAGCACCGTCACA 3′, and 5′ GCTTTGGAGGAGACTGGCTT 3′ and FIT3 with 5′ TTGTCTGGACTGGTGAAGGC 3′ and 5′ GTGGTTGCAGTGGTTGAAGC 3′. For each specified gene, ΔCT was calculated as CTtarget gene-CTactin, while fold change was calculated as 2-ΔΔCT with actin serving as the endogenous control for each sample. For each gene-sample pair, a no-template control and nonreverse-transcriptase control were performed to control for genomic DNA contamination.To determine the silencing rate, RNA was extracted from the same mid–log phase cells treated with EtBr for 3 and 5 d and analyzed by RT-qPCR. The silencing rate is calculated as the changes of the ΔCT between 3 and 5 d.DHE superoxide stainingMitochondrial superoxide was visualized by staining live cells with DHE as previously described in McFaline-Figueroa et al. (2011). Mid–log yeast cells propagated on SC liquid medium were incubated with 40 µM DHE dissolved in DMSO (Molecular Probes, Eugene, OR) for 30 min at 30°C, washed 2× with SC, and imaged without fixation (McFaline-Figueroa et al., 2011). DHE was excited using a 561-nm LED and imaged with the microscope previously described with a dual eGFP/mCherry cube (59222; Chroma, Bellows Falls, VT). DHE images were deconvolved using 60 iterations (100% confidence criterion) of a constrained iterative restoration algorithm and a theoretical PSF based on a 610-nm emission wavelength using Volocity 6.3.Statistical methods and data representationAll data were analyzed for normal distribution with the D’Agostino and Pearson normality test. The p values for simple two-group comparison were determined with a two-tailed Student’s t test for parametric distributions and a Mann–Whitney test for nonparametric data. For multiple group comparisons, p values were determined with a one-way analysis of variance (ANOVA) with Tukey’s post-hoc test for parametric distributions and Kruskal–Wallis test with Dunn’s post-hoc test for nonparametric distributions. The Log-rank (Mantel–Cox) test was used to test statistical significance between RLS survival curves. GraphPad Prism7 (GraphPad Software) was used to conduct all statistical analysis and to create all graphs. Bar graphs show the mean and SEM; in box and whiskers graphs, the box represents the middle quartile, the midline represents the median, and whiskers show the minimum and maximum values. For RLS graphs, survival graphs are shown where the remaining percentage of viable cells is plotted over generation number. For all tests, p values are classified as follows: *p < 0.05, **p < 0.01, ***p < 0.001, ****p < 0.0001, unless otherwise noted in the figure legends.

MATERIALS AND METHODS

Yeast strains and growth conditionsAll S. cerevisiae strains used in this study are derivatives of the wild-type BY4741 strain (MATa his3∆1 leu2∆0 met15∆0 ura3∆0) from Open Biosystems (Huntsville, AL). Yeast cells were cultivated and manipulated as described previously (Sherman, 2002). The rho0 cells were generated by treatment with EtBr as described in Fox et al. (1991). Briefly, each strain was grown in SC containing 25 μg/ml ethidium bromide (EtBr; Sigma, St. Louis, MO) for 48 h at 30°C with shaking at 220 rpm. Then cells were spread on YPD and incubated for 5 d at 30°C. The rho0 status was confirmed by lack of growth on plates containing a nonfermentable carbon source and an absence of mtDNA by DAPI (4’,6-diamidino-2-phenylindole) staining.For some studies, rho0 cells were generated by replacing MGM101 in rho+ cells with knockout cassettes containing the selectable marker LEU2 through homologous recombination according to previously described protocols (Longtine et al., 1998; Gauss et al., 2005). The primers, 5′ CTAAAAAAGGAAAGAAAGGACAAGTAGGAAGATCAGCGTACGTGCAGGTCGACAACCCTTAAT 3′ and 5′ ATATACTTACTAAAATTAGCTTATATGGTTCGCATATTGAGCAGCGTACGGATATCACCTA 3′, were used to amplify LEU2 from pOM13. Deletion of MGM101 was confirmed by PCR amplification of the locus using the following primers: 5′ CGAAATTTATCGACAGAATAATGG 3′ and 5′ GTACTGACACTACGCACTACC 3′.For each experiment needing mgm101Δ rho0 cells, MGM101 was freshly deleted to avoid further adaptation of rho0 cells during normal handling, passage and subculturing. Likewise, new EtBr-generated rho0 cells were generated for each experiment. SIR3 was deleted from BY4741 using the following primers: 5′ TTAAGAAAGTTGTTTTGTTCTAACAATTGGATTAGCTAAATGCAGGTCGACAACCCTTAAT 3′ and 5′ CATAGGCATATCTATGGCGGAAGTGAAAATGAATGTTGGTGGGCAGCGTACGGATATCACCTA 3′ to amplify URA3 from pOM12. Deletion of SIR3 was confirmed by PCR amplification of the locus using primers 5′ CACATAAGCAGCCCTTTCATC 3′ and 5′ GAATACAGAAGAGACTGCATG 3′.

Yeast strains and growth conditions

All S. cerevisiae strains used in this study are derivatives of the wild-type BY4741 strain (MATa his3∆1 leu2∆0 met15∆0 ura3∆0) from Open Biosystems (Huntsville, AL). Yeast cells were cultivated and manipulated as described previously (Sherman, 2002). The rho0 cells were generated by treatment with EtBr as described in Fox et al. (1991). Briefly, each strain was grown in SC containing 25 μg/ml ethidium bromide (EtBr; Sigma, St. Louis, MO) for 48 h at 30°C with shaking at 220 rpm. Then cells were spread on YPD and incubated for 5 d at 30°C. The rho0 status was confirmed by lack of growth on plates containing a nonfermentable carbon source and an absence of mtDNA by DAPI (4’,6-diamidino-2-phenylindole) staining.

For some studies, rho0 cells were generated by replacing MGM101 in rho+ cells with knockout cassettes containing the selectable marker LEU2 through homologous recombination according to previously described protocols (Longtine et al., 1998; Gauss et al., 2005). The primers, 5′ CTAAAAAAGGAAAGAAAGGACAAGTAGGAAGATCAGCGTACGTGCAGGTCGACAACCCTTAAT 3′ and 5′ ATATACTTACTAAAATTAGCTTATATGGTTCGCATATTGAGCAGCGTACGGATATCACCTA 3′, were used to amplify LEU2 from pOM13. Deletion of MGM101 was confirmed by PCR amplification of the locus using the following primers: 5′ CGAAATTTATCGACAGAATAATGG 3′ and 5′ GTACTGACACTACGCACTACC 3′.

For each experiment needing mgm101Δ rho0 cells, MGM101 was freshly deleted to avoid further adaptation of rho0 cells during normal handling, passage and subculturing. Likewise, new EtBr-generated rho0 cells were generated for each experiment. SIR3 was deleted from BY4741 using the following primers: 5′ TTAAGAAAGTTGTTTTGTTCTAACAATTGGATTAGCTAAATGCAGGTCGACAACCCTTAAT 3′ and 5′ CATAGGCATATCTATGGCGGAAGTGAAAATGAATGTTGGTGGGCAGCGTACGGATATCACCTA 3′ to amplify URA3 from pOM12. Deletion of SIR3 was confirmed by PCR amplification of the locus using primers 5′ CACATAAGCAGCCCTTTCATC 3′ and 5′ GAATACAGAAGAGACTGCATG 3′.

Colony size determinationYeast from the indicated colony type (rho0 strains obtained by EtBr treatment, or subcultured adapted or UA colonies from parental rho0 strains) were diluted and spread on YPD plates to generate single colonies. After 5 d of growth at 30°C, single colonies were imaged using a ChemiDoc MP (Bio-Rad). Images were processed with the open-source colony counting software OpenCFU to automatically count and measure the size of each colony (Geissmann, 2013). False-positive colonies resulting from noise were automatically removed by applying a –1 filter or manually removed from analysis. Colonies of area <0.72 mm2 (100 pixels) were categorized as small/UA, while colonies of area >0.72 mm2 were categorized as large/adapted.

Colony size determination

Yeast from the indicated colony type (rho0 strains obtained by EtBr treatment, or subcultured adapted or UA colonies from parental rho0 strains) were diluted and spread on YPD plates to generate single colonies. After 5 d of growth at 30°C, single colonies were imaged using a ChemiDoc MP (Bio-Rad). Images were processed with the open-source colony counting software OpenCFU to automatically count and measure the size of each colony (Geissmann, 2013). False-positive colonies resulting from noise were automatically removed by applying a –1 filter or manually removed from analysis. Colonies of area <0.72 mm2 (100 pixels) were categorized as small/UA, while colonies of area >0.72 mm2 were categorized as large/adapted.

Growth ratesGrowth curves were measured using an automated plate reader (Tecan; Infinite M200, Research Triangle Park, NC). Each strain was grown to mid–log phase in rich, glucose-based media (YPD) and diluted to an OD600 of 0.07 (2.0 × 106 cells/ml). A diluted strain of 10 µl was added to a well containing 200 µl YPD in a 96-well plate. Cells were propagated at 30°C without shaking, and an optical density at 600 nm (OD600) was measured every 20 min for 72 h. For each strain, three independent colonies were tested in quadruplicate. The maximum growth rate was calculated using the greatest change in OD600 over a 240-min interval in 72 h.

Growth rates

Growth curves were measured using an automated plate reader (Tecan; Infinite M200, Research Triangle Park, NC). Each strain was grown to mid–log phase in rich, glucose-based media (YPD) and diluted to an OD600 of 0.07 (2.0 × 106 cells/ml). A diluted strain of 10 µl was added to a well containing 200 µl YPD in a 96-well plate. Cells were propagated at 30°C without shaking, and an optical density at 600 nm (OD600) was measured every 20 min for 72 h. For each strain, three independent colonies were tested in quadruplicate. The maximum growth rate was calculated using the greatest change in OD600 over a 240-min interval in 72 h.

Cell cycle assayCell cycle progression was assessed by measuring amount of DNA as described previously (Breeden, 1997; Fortuna et al., 2001; Crider et al., 2012). Mid–log phase yeast were synchronized by incubating cells with 10–100 µM α-factor for 2.5 h on YPD with shaking at 30°C. Cells were released from arrest by washing with fresh YPD media and transferred to pheromone-free media. Cells were collected and fixed in 70% ethanol 0, 20, 40, 60, 80, 100, 120, 150, and 180 min after release from arrest. Cell cycle progression was assessed by measuring DNA content as described previously (Breeden, 1997; Fortuna et al., 2001; Crider et al., 2012). Briefly, cells were washed and digested with 250 µg/ml RNase in sodium citrate buffer, pH 7.5, for 2 h at 50°C. After RNase digestion, cells were digested with 20 mg/ml Proteinase K for 2 h at 50°C. DNA was stained resuspending cells in citrate buffer containing 16 µg/ml propidium iodide. DNA content was measured using a fluorescence-activated cell analyzer (LSRII, BD), and 50,000 events were recorded for each timepoint. The percentage of G1, S, and G2 phase cells was determined using FlowJo (FlowJo LLC, Ashland, OR).

Cell cycle assay

Cell cycle progression was assessed by measuring amount of DNA as described previously (Breeden, 1997; Fortuna et al., 2001; Crider et al., 2012). Mid–log phase yeast were synchronized by incubating cells with 10–100 µM α-factor for 2.5 h on YPD with shaking at 30°C. Cells were released from arrest by washing with fresh YPD media and transferred to pheromone-free media. Cells were collected and fixed in 70% ethanol 0, 20, 40, 60, 80, 100, 120, 150, and 180 min after release from arrest. Cell cycle progression was assessed by measuring DNA content as described previously (Breeden, 1997; Fortuna et al., 2001; Crider et al., 2012). Briefly, cells were washed and digested with 250 µg/ml RNase in sodium citrate buffer, pH 7.5, for 2 h at 50°C. After RNase digestion, cells were digested with 20 mg/ml Proteinase K for 2 h at 50°C. DNA was stained resuspending cells in citrate buffer containing 16 µg/ml propidium iodide. DNA content was measured using a fluorescence-activated cell analyzer (LSRII, BD), and 50,000 events were recorded for each timepoint. The percentage of G1, S, and G2 phase cells was determined using FlowJo (FlowJo LLC, Ashland, OR).

RLS determinationRLS measurements were performed as described previously (Erjavec et al., 2008) without alpha-factor synchronization. Frozen yeast strain stocks (stored at –80°C) were grown on YPD plates at 30°C, and rho0 cells were obtained by ethidium bromide treatment as previously described. Single colonies of each yeast strain were suspended in liquid YPD and grown at 30°C with shaking to mid–log phase (OD600 0.1–0.3). A 2-µl aliquot of cell suspension was applied to a YPD plate. Small-budded cells were isolated and arranged in a matrix using a micromanipulator mounted on a dissecting microscope (Zeiss, Thornwood, NY, or Singer Instruments, Watchet, UK). When the small buds completed growth, their mother cells were removed and discarded, and the remaining daughter cells were named virgin mother cells. After each replication, the new bud was removed and discarded. This was continued until all replication ceased. The mean generation time and number of daughter cells produced by each virgin mother cell were recorded.

RLS determination

RLS measurements were performed as described previously (Erjavec et al., 2008) without alpha-factor synchronization. Frozen yeast strain stocks (stored at –80°C) were grown on YPD plates at 30°C, and rho0 cells were obtained by ethidium bromide treatment as previously described. Single colonies of each yeast strain were suspended in liquid YPD and grown at 30°C with shaking to mid–log phase (OD600 0.1–0.3). A 2-µl aliquot of cell suspension was applied to a YPD plate. Small-budded cells were isolated and arranged in a matrix using a micromanipulator mounted on a dissecting microscope (Zeiss, Thornwood, NY, or Singer Instruments, Watchet, UK). When the small buds completed growth, their mother cells were removed and discarded, and the remaining daughter cells were named virgin mother cells. After each replication, the new bud was removed and discarded. This was continued until all replication ceased. The mean generation time and number of daughter cells produced by each virgin mother cell were recorded.

Assessing mitochondrial function using mitochondria-targeted roGFPMitochondrial redox state was measured as previously described in McFaline-Figueroa et al. (2011). Strains were transformed with a centromeric plasmid expressing mito-roGFP1 targeted to mitochondria using the ATP9 mitochondrial targeting sequence prior to EtBr treatment. After EtBr treatment as described previously, ∼40–50 small rho0 colonies and ∼8–10 large rho0 colonies were selected and grown for 12 h on selective media to obtain UA and A, respectively. These adapted cells were grown for another 24 h to obtain lAA. Images were acquired with an Axioskop 2 microscope equipped with a 100×/1.4 Plan-Apochromat objective (Zeiss, Thornwood, NY), an Orca-ER cooled CCD camera (Hamamatsu), and a pE-4000 LED illumination system (coolLED, Andover, UK). Images were acquired using NIS Elements 4.60 Lambda software (Nikon, Melville, NY). Oxidized and reduced channels were excited using a 365-nm LED and a 470-nm LED with a ET470/40× filter, respectively. All channels were acquired with a modified GFP filter (Zeiss filter 46 HE without excitation filter, dichroic FT 515, emission 535/30). Images were deconvolved using 60 iterations (100% confidence criterion) of a constrained iterative restoration algorithm and a theoretical PSF based on a 507-nm emission wavelength using Volocity 6.3 (Perkin-Elmer, Waltham, MA). The reduced:oxidized ratio channel was calculated by dividing the intensity of the reduced channel (λex = 470 nm, λem = 525 nm) by the intensity of the oxidized channel (λex = 365 nm, λem = 525 nm) after background subtraction and thresholding for each channel individually.

Assessing mitochondrial function using mitochondria-targeted roGFP

Mitochondrial redox state was measured as previously described in McFaline-Figueroa et al. (2011). Strains were transformed with a centromeric plasmid expressing mito-roGFP1 targeted to mitochondria using the ATP9 mitochondrial targeting sequence prior to EtBr treatment. After EtBr treatment as described previously, ∼40–50 small rho0 colonies and ∼8–10 large rho0 colonies were selected and grown for 12 h on selective media to obtain UA and A, respectively. These adapted cells were grown for another 24 h to obtain lAA. Images were acquired with an Axioskop 2 microscope equipped with a 100×/1.4 Plan-Apochromat objective (Zeiss, Thornwood, NY), an Orca-ER cooled CCD camera (Hamamatsu), and a pE-4000 LED illumination system (coolLED, Andover, UK). Images were acquired using NIS Elements 4.60 Lambda software (Nikon, Melville, NY). Oxidized and reduced channels were excited using a 365-nm LED and a 470-nm LED with a ET470/40× filter, respectively. All channels were acquired with a modified GFP filter (Zeiss filter 46 HE without excitation filter, dichroic FT 515, emission 535/30). Images were deconvolved using 60 iterations (100% confidence criterion) of a constrained iterative restoration algorithm and a theoretical PSF based on a 507-nm emission wavelength using Volocity 6.3 (Perkin-Elmer, Waltham, MA). The reduced:oxidized ratio channel was calculated by dividing the intensity of the reduced channel (λex = 470 nm, λem = 525 nm) by the intensity of the oxidized channel (λex = 365 nm, λem = 525 nm) after background subtraction and thresholding for each channel individually.

RNA sequencingTranscriptome was analyzed as previously described in Vevea et al. (2015). RNA was extracted from mid–log phase A and UA rho0 yeast cells using the RNeasy kit (Qiagen, Germantown, MD). RNA quality was analyzed with Agilent 2100 Bioanalyzer using a Plant RNA Nano chip, and only RNA samples with RNA Integrity Number (RIN) scores >9 were used for subsequent RNA-Seq. The mRNA library was generated using Illumina TruSeq RNA prep kit after poly-A pull-down enrichment of mRNA from total RNA samples. RNA-Seq was performed on an Illumina HiSeq2500 generating 200 million 100-base pair single-end reads per lane, with 10 samples multiplexed per lane (average 30 million raw reads per sample) by the Columbia Genome Center. Data were analyzed using Tophat, Cufflinks, and Cuffdiff protocol as described on the Galaxy platform (Afgan et al., 2016). Differential expression was analyzed with DESeq2, an open-source differential gene expression analysis based on the negative binomial distribution (Love et al., 2014). Differentially expressed genes were then analyzed using Funspec to group the large sets of up-regulated and down-regulated genes into gene ontology (GO) terms (Robinson et al., 2002).

RNA sequencing

Transcriptome was analyzed as previously described in Vevea et al. (2015). RNA was extracted from mid–log phase A and UA rho0 yeast cells using the RNeasy kit (Qiagen, Germantown, MD). RNA quality was analyzed with Agilent 2100 Bioanalyzer using a Plant RNA Nano chip, and only RNA samples with RNA Integrity Number (RIN) scores >9 were used for subsequent RNA-Seq. The mRNA library was generated using Illumina TruSeq RNA prep kit after poly-A pull-down enrichment of mRNA from total RNA samples. RNA-Seq was performed on an Illumina HiSeq2500 generating 200 million 100-base pair single-end reads per lane, with 10 samples multiplexed per lane (average 30 million raw reads per sample) by the Columbia Genome Center. Data were analyzed using Tophat, Cufflinks, and Cuffdiff protocol as described on the Galaxy platform (Afgan et al., 2016). Differential expression was analyzed with DESeq2, an open-source differential gene expression analysis based on the negative binomial distribution (Love et al., 2014). Differentially expressed genes were then analyzed using Funspec to group the large sets of up-regulated and down-regulated genes into gene ontology (GO) terms (Robinson et al., 2002).

cDNA synthesis and quantitative PCRRNA was extracted from mid–log phase UA and A rho0 cells using the RNeasy kit (Qiagen, Germantown, MD). RNA quality was analyzed as previously described and only RNA samples with RIN scores >9 were used for subsequent RT-PCR analysis. Genomic DNA contamination was removed from RNA samples with TURBO DNA-free Kit (Ambion, Carslbad, CA). DNA-free RNA (1 µg ) was used for cDNA synthesis performed with SuperScript IV First-Strand Synthesis System (Invitrogen, Waltham, MA). cDNA was diluted and used for quantitative PCR with PowerUp SYBR Green Master Mix (Applied Biosystems, Carlsbad, CA) for adapted, superadapted, and UA samples. Primers for qPCR were designed with NCBI Primer Blast (www.ncbi.nlm.nih.gov/tools/primer-blast/) with a PCR product size of 100 base pairs and max Tm difference of 2°C. Primer specificity and amplification efficiency for each primer set were validated with a standard curve. ACT1 was amplified with 5′ TCGCCTTGGACTTCGAACAA 3′ and 5′ CAAAGCTTCTGGGGCTCTGA 3′, ADE17 with 5′ CGTGACGCTGGTTTTCCAAT 3′ and 5′ CACCATGAACGGCAGGATGT 3′, HSP30 with 5′ GCTACGGACGATGTGGAAGA 3′ and 5′ CAGGTTCGGGTTCGTGGATT 3′, PAU24 with 5′ GGTATTGCCCCAGACCAAGT 3′ and 5′ GCACTAGAGATGGCTGGCTT 3′, DAN1 with 5′ GTACTGACAGCACCGTCACA 3′, and 5′ GCTTTGGAGGAGACTGGCTT 3′ and FIT3 with 5′ TTGTCTGGACTGGTGAAGGC 3′ and 5′ GTGGTTGCAGTGGTTGAAGC 3′. For each specified gene, ΔCT was calculated as CTtarget gene-CTactin, while fold change was calculated as 2-ΔΔCT with actin serving as the endogenous control for each sample. For each gene-sample pair, a no-template control and nonreverse-transcriptase control were performed to control for genomic DNA contamination.To determine the silencing rate, RNA was extracted from the same mid–log phase cells treated with EtBr for 3 and 5 d and analyzed by RT-qPCR. The silencing rate is calculated as the changes of the ΔCT between 3 and 5 d.

cDNA synthesis and quantitative PCR

RNA was extracted from mid–log phase UA and A rho0 cells using the RNeasy kit (Qiagen, Germantown, MD). RNA quality was analyzed as previously described and only RNA samples with RIN scores >9 were used for subsequent RT-PCR analysis. Genomic DNA contamination was removed from RNA samples with TURBO DNA-free Kit (Ambion, Carslbad, CA). DNA-free RNA (1 µg ) was used for cDNA synthesis performed with SuperScript IV First-Strand Synthesis System (Invitrogen, Waltham, MA). cDNA was diluted and used for quantitative PCR with PowerUp SYBR Green Master Mix (Applied Biosystems, Carlsbad, CA) for adapted, superadapted, and UA samples. Primers for qPCR were designed with NCBI Primer Blast (www.ncbi.nlm.nih.gov/tools/primer-blast/) with a PCR product size of 100 base pairs and max Tm difference of 2°C. Primer specificity and amplification efficiency for each primer set were validated with a standard curve. ACT1 was amplified with 5′ TCGCCTTGGACTTCGAACAA 3′ and 5′ CAAAGCTTCTGGGGCTCTGA 3′, ADE17 with 5′ CGTGACGCTGGTTTTCCAAT 3′ and 5′ CACCATGAACGGCAGGATGT 3′, HSP30 with 5′ GCTACGGACGATGTGGAAGA 3′ and 5′ CAGGTTCGGGTTCGTGGATT 3′, PAU24 with 5′ GGTATTGCCCCAGACCAAGT 3′ and 5′ GCACTAGAGATGGCTGGCTT 3′, DAN1 with 5′ GTACTGACAGCACCGTCACA 3′, and 5′ GCTTTGGAGGAGACTGGCTT 3′ and FIT3 with 5′ TTGTCTGGACTGGTGAAGGC 3′ and 5′ GTGGTTGCAGTGGTTGAAGC 3′. For each specified gene, ΔCT was calculated as CTtarget gene-CTactin, while fold change was calculated as 2-ΔΔCT with actin serving as the endogenous control for each sample. For each gene-sample pair, a no-template control and nonreverse-transcriptase control were performed to control for genomic DNA contamination.

To determine the silencing rate, RNA was extracted from the same mid–log phase cells treated with EtBr for 3 and 5 d and analyzed by RT-qPCR. The silencing rate is calculated as the changes of the ΔCT between 3 and 5 d.

DHE superoxide stainingMitochondrial superoxide was visualized by staining live cells with DHE as previously described in McFaline-Figueroa et al. (2011). Mid–log yeast cells propagated on SC liquid medium were incubated with 40 µM DHE dissolved in DMSO (Molecular Probes, Eugene, OR) for 30 min at 30°C, washed 2× with SC, and imaged without fixation (McFaline-Figueroa et al., 2011). DHE was excited using a 561-nm LED and imaged with the microscope previously described with a dual eGFP/mCherry cube (59222; Chroma, Bellows Falls, VT). DHE images were deconvolved using 60 iterations (100% confidence criterion) of a constrained iterative restoration algorithm and a theoretical PSF based on a 610-nm emission wavelength using Volocity 6.3.

DHE superoxide staining

Mitochondrial superoxide was visualized by staining live cells with DHE as previously described in McFaline-Figueroa et al. (2011). Mid–log yeast cells propagated on SC liquid medium were incubated with 40 µM DHE dissolved in DMSO (Molecular Probes, Eugene, OR) for 30 min at 30°C, washed 2× with SC, and imaged without fixation (McFaline-Figueroa et al., 2011). DHE was excited using a 561-nm LED and imaged with the microscope previously described with a dual eGFP/mCherry cube (59222; Chroma, Bellows Falls, VT). DHE images were deconvolved using 60 iterations (100% confidence criterion) of a constrained iterative restoration algorithm and a theoretical PSF based on a 610-nm emission wavelength using Volocity 6.3.

Statistical methods and data representationAll data were analyzed for normal distribution with the D’Agostino and Pearson normality test. The p values for simple two-group comparison were determined with a two-tailed Student’s t test for parametric distributions and a Mann–Whitney test for nonparametric data. For multiple group comparisons, p values were determined with a one-way analysis of variance (ANOVA) with Tukey’s post-hoc test for parametric distributions and Kruskal–Wallis test with Dunn’s post-hoc test for nonparametric distributions. The Log-rank (Mantel–Cox) test was used to test statistical significance between RLS survival curves. GraphPad Prism7 (GraphPad Software) was used to conduct all statistical analysis and to create all graphs. Bar graphs show the mean and SEM; in box and whiskers graphs, the box represents the middle quartile, the midline represents the median, and whiskers show the minimum and maximum values. For RLS graphs, survival graphs are shown where the remaining percentage of viable cells is plotted over generation number. For all tests, p values are classified as follows: *p < 0.05, **p < 0.01, ***p < 0.001, ****p < 0.0001, unless otherwise noted in the figure legends.

Statistical methods and data representation

All data were analyzed for normal distribution with the D’Agostino and Pearson normality test. The p values for simple two-group comparison were determined with a two-tailed Student’s t test for parametric distributions and a Mann–Whitney test for nonparametric data. For multiple group comparisons, p values were determined with a one-way analysis of variance (ANOVA) with Tukey’s post-hoc test for parametric distributions and Kruskal–Wallis test with Dunn’s post-hoc test for nonparametric distributions. The Log-rank (Mantel–Cox) test was used to test statistical significance between RLS survival curves. GraphPad Prism7 (GraphPad Software) was used to conduct all statistical analysis and to create all graphs. Bar graphs show the mean and SEM; in box and whiskers graphs, the box represents the middle quartile, the midline represents the median, and whiskers show the minimum and maximum values. For RLS graphs, survival graphs are shown where the remaining percentage of viable cells is plotted over generation number. For all tests, p values are classified as follows: *p < 0.05, **p < 0.01, ***p < 0.001, ****p < 0.0001, unless otherwise noted in the figure legends.

Supplementary MaterialClick here for additional data file.

Supplementary Material

Click here for additional data file.
