---
pmid: '25519916'
title: The EHEC-host interactome reveals novel targets for the translocated intimin
  receptor.
authors:
- Blasche S
- Arens S
- Ceol A
- Siszler G
- Schmidt MA
- Häuser R
- Schwarz F
- Wuchty S
- Aloy P
- Uetz P
- Stradal T
- Koegl M
journal: Sci Rep
year: '2014'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC4269881
doi: 10.1038/srep07531
---

# The EHEC-host interactome reveals novel targets for the translocated intimin receptor.
**Authors:** Blasche S, Arens S, Ceol A, Siszler G, Schmidt MA, Häuser R, Schwarz F, Wuchty S, Aloy P, Uetz P, Stradal T, Koegl M
**Journal:** Sci Rep (2014)
**DOI:** [10.1038/srep07531](https://doi.org/10.1038/srep07531)
**PMC:** [PMC4269881](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4269881/)

## Abstract

1. Sci Rep. 2014 Dec 18;4:7531. doi: 10.1038/srep07531.

The EHEC-host interactome reveals novel targets for the translocated intimin 
receptor.

Blasche S(1), Arens S(2), Ceol A(3), Siszler G(1), Schmidt MA(4), Häuser R(1), 
Schwarz F(1), Wuchty S(5), Aloy P(6), Uetz P(7), Stradal T(8), Koegl M(1).

Author information:
(1)Genomics and Proteomics Core Facilities, German Cancer Research Center, Im 
Neuenheimer Feld 580, 69120 Heidelberg, Germany.
(2)Institute of Molecular Cell Biology, University of Münster, Schlossplatz 5, 
D-48149 Münster.
(3)1] Joint IRB-BSC Program in Computational Biology, Institute for Research in 
Biomedicine (IRB Barcelona), Barcelona, Spain [2] Center for Genomic Science of 
IIT@SEMM, Fondazione Istituto Italiano di Tecnologia (IIT), Via Adamello 16, 
20139 Milan - Italy.
(4)Institute of Infectiology, ZMBE, University of Münster, Von-Esmarch-Str. 56, 
D-48149 Münster.
(5)1] Dept. of Computer Science, Univ. of Miami, 1365 Memorial Drive, Coral 
Gables, FL 33146, USA [2] Center for Computational Science, Univ. of Miami, 1365 
Memorial Drive, Coral Gables, FL 33146, USA.
(6)1] Joint IRB-BSC Program in Computational Biology, Institute for Research in 
Biomedicine (IRB Barcelona), Barcelona, Spain [2] Institució Catalana de Recerca 
i Estudis Avançats (ICREA), Barcelona, Spain.
(7)Center for the Study of Biological Complexity, Virginia Commonwealth 
University, Richmond, VA 23284, USA.
(8)1] Institute of Molecular Cell Biology, University of Münster, Schlossplatz 
5, D-48149 Münster [2] Helmholtz Centre for Infection Research, Inhoffenstrasse 
7, D-38124 Braunschweig.

Enterohemorrhagic E. coli (EHEC) manipulate their human host through at least 39 
effector proteins which hijack host processes through direct protein-protein 
interactions (PPIs). To identify their protein targets in the host cells, we 
performed yeast two-hybrid screens, allowing us to find 48 high-confidence 
protein-protein interactions between 15 EHEC effectors and 47 human host 
proteins. In comparison to other bacteria and viruses we found that EHEC 
effectors bind more frequently to hub proteins as well as to proteins that 
participate in a higher number of protein complexes. The data set includes six 
new interactions that involve the translocated intimin receptor (TIR), namely 
HPCAL1, HPCAL4, NCALD, ARRB1, PDE6D, and STK16. We compared these TIR 
interactions in EHEC and enteropathogenic E. coli (EPEC) and found that five 
interactions were conserved. Notably, the conserved interactions included those 
of serine/threonine kinase 16 (STK16), hippocalcin-like 1 (HPCAL1) as well as 
neurocalcin-delta (NCALD). These proteins co-localize with the infection sites 
of EPEC. Furthermore, our results suggest putative functions of poorly 
characterized effectors (EspJ, EspY1). In particular, we observed that EspJ is 
connected to the microtubule system while EspY1 appears to be involved in 
apoptosis/cell cycle regulation.

DOI: 10.1038/srep07531
PMCID: PMC4269881
PMID: 25519916 [Indexed for MEDLINE]

## Full Text

Enterohemorrhagic E. coli (EHEC) are pathogenic E. coli that produce shiga toxins similar to Shigella dysenteriae12. Aside from bloody and non-bloody self-limiting diarrhea, EHEC can cause so-called hemolytic uremic syndrome (HUS), a disease that is characterized by hemolytic anemia, acute and occasionally chronic renal failure as well as thrombocytopenia. The public health impact of EHEC infections is considerable34 as a consequence of their low infectious dose (<100 colony forming units) and the corresponding frequency of systemic complications in patients. EHEC colonization utilizes a type three-secretion system (T3SS) and up to 62 putative effector proteins. The T3SS system directly translocates at least 39 effectors into the cytoplasm of their human host cells5. The combined action of these effectors results in brush border remodeling and the formation of attaching and effacing (A/E) lesions in the small intestine that are typical for EHEC and enteropathogenic E. coli (EPEC) infections. Furthermore, effectors manipulate several host cell signaling pathways to allow successful colonization of the host. Especially the assembly of pedestals through actin-cytoskeleton reorganization is considered important for host cell colonization, a step that is substantially conditioned by the translocated intimin receptor (TIR)6. TIR is involved in several distinct cellular processes besides actin pedestal assembly, such as down-regulation of Map-dependent filopodia formation and suppression of inflammatory cytokine production6789. Furthermore, TIR interacts with numerous host cellular targets such as 14-3-3tau, alpha-actinin, cortacin, CK18, IQGAP1, IRTKS, IRSp53, Nck, PI3K, talin, vinculin and AnxA2101112131415161718192021 as reviewed in ref 7. Only recently two novel interactions with SHP-1 and SHP-2 were published89. Although several other novel EHEC - host protein interactions have been described that involve the effectors EspG, EspZ, NleB and NleF892223242526 our knowledge of interactions between effector and host cell proteins is far from complete. To fill this gap we mapped the first comprehensive, experimentally derived network of binary EHEC-host interactions which allowed us to investigate the interactions between the translocated intimin receptor TIR and its human targets in more detail.

Results and DiscussionThe EHEC effector-host protein interaction mapIn 2006 Tobe et al. identified 62 putative EHEC effector proteins through computational analysis and experimentally confirmed that 39 effectors were indeed translocated into host cells5. While several already have been demonstrated to modify certain host functions (reviewed in ref 7), many effector proteins remain functionally poorly characterized. To investigate the global landscape of effector-host interactions we systematically screened 34 of the 39 effectors verified by Tobe et al. using a well established, semi-automated comprehensive Yeast 2-Hybrid (Y2H) screening protocol2728 (Figure 1, Supplementary Table S1). Each effector was fused to the GAL4-DBD (DNA binding domain of the GAL4 transcription factor) on both its N- and C-terminus. Similarly, we built two different prey screening libraries from human open reading frame (ORFeome) collections293031. Each consisted of a single pool of over 10,000 individual human protein-coding ORFs that were either fused C- or N-terminally to the AD (activation domain) of GAL4. The N- and C-terminally-tagged effector bait constructs were then screened individually against both ORFeome libraries to increase the detection sensitivity3233 as well as against a cDNA prey library (Human Universal, Clontech). These screens resulted in 1,025 positive yeast colonies from which we successfully retrieved 328 unique EHEC bait-human prey pairs. We removed 207 pairs with prey sequences that were not located in the protein coding region of genes (3′-UTR artifacts), interactions involving randomly activating bait constructs (i.e. baits that returned exclusively multiple non-reproducible hits), or interactions including prey proteins known to interact non-selectively with many different bait proteins. In the latter case we excluded prey constructs known to bind to more than five bait proteins28 (Schwarz et al., unpublished). Out of the remaining 121 interactions 86 were detected only once, i.e., a prey could be assigned to the known bait protein only from one unique positive yeast colony. Using pairwise Y2H assays we retested all 121 pairs that had human full-length ORF constructs. We cloned 78 out of 90 human interactors into prey plasmids and were able to reproduce 35 out of 82 tested PPIs (42.7%). All 23 pairs that had two or more colonies in the screens were retested positive; pairs with only a single colony had a success rate of 20.7%. In the final dataset, we included all PPIs that were isolated multiple times, as well as those that were independently reproduced. In total, our approach yielded 48 individual EHEC-host PPIs of high confidence (Table 1 and for more details Supplementary Table S2) where the majority of 36 PPIs was identified by using a C-terminal ORFeome prey library. In comparison, the N-terminal ORFeome prey library and the cDNA prey libraries isolated clearly fewer interactions (seven PPIs each). Only two interactions were found in all the different ORFeome prey libraries (NleF with Caspase 9 and dihydrofolate reductase).To generate a combined protein interaction network including the 48 newly found interacions, we extensively reviewed the existing literature and curated 130 effector-host interactions, including 76 EHEC-host protein interactions. Furthermore, we found a few interactions that were inferred from protein complexes although no experimental evidence for a direct interaction appears to be available.The combined network is shown in Figure 2 (for a complete list of EHEC-host interactions and references see Supplementary Table S3). We found that five out of the discovered 48 pathogen-host pairs were previously published (Tir-BAIAP2L1 ( = IRTKS)14, Tir-BAIAP2 ( = IRSp53)15, Map-NHERF2 ( = SLC9A3R2)34, EspF-SNX935 and NleF-CASP923).From the literature, EHEC-host interactions are known for 18 out of 39 effector proteins described by Tobe et al. Our screens allowed us to find PPIs that involved 15 out of 39 effectors including EspJ and EspY1 that previously were not reported to interact with the host. While cellular targets of the remaining 13 effectors, EspB, EspF, EspG, EspO, Map, NleA, NleB; NleC, NleD, NleF, NleH, TccP (EspFu) and TIR, were previously known, we determined 31 novel interactions.A global analysis of EHEC-host interactions compared to other pathogensTo compare proteins targeted by EHEC with those of other pathogens, we compared our data to 8,297 protein interactions between proteins of B. anthracis, F. tularensis and Y. pestis and 3,363 human host proteins36. Furthermore, we compiled a set of 3,156 interactions between viral proteins of the Epstein-Barr, HIV, Hepatitis, Herpes, Influenza, Papilloma and Vaccinia virus and 1,778 human host proteins37. We find that many human host proteins were targeted by both bacterial and viral proteins (Figure 3A).To investigate the impact on pathways, we determined the enrichment of proteins targeted by EHEC effectors in pathways that we collected from the molecular signature database38. We observed a significant enrichment of a small set of mostly signaling as well as infection-related pathways (Figure 3B).In general, the majority of human target proteins interact with one effector protein, a characteristic that is shared with bacterial and viral targets (Figure 4A). Next, we wondered whether pathogen proteins interact with highly connected human proteins (hubs) or rather focus on less frequently interacting proteins in the human interactome. Figure 4B indicates that bacterial as well as viral proteins are predominately enriched in groups of highly connected host proteins. In comparison, targets of EHEC effector proteins surprisingly show a significantly stronger enrichment signal. Calculating shortest path lengths (Figure 4C) we observed that proteins targeted by EHEC effector proteins had shorter paths to other human proteins than the corresponding targets of other bacteria and of viruses (Student's t-test, p < 10−20). A similar trend for binding to highly connected proteins is apparent in the enrichment of targeted proteins in protein complexes, as shown in Figure 4D. In comparison to bacterial and viral targets we observed that targets of EHEC effector proteins appear significantly more frequently in protein complexes.EspY1 interacts with proteins involved in apoptosis and cell cycle regulationWhile neither a function nor a human interacting protein was previously known for EspY1, we detected eight novel interactions that involved this effector protein. EspY1 appears to be primarily involved in apoptosis and cell cycle regulation as these functions are shared among its host targets calpain-3, 26S protease regulatory subunit 4, and PCI domain-containing protein 2. Such interactions may complement other EHEC effector proteins that target host proteins known to block apoptosis while assuring host cell survival. For instance, NleF interacts with caspases 4,8, and 923, while NleH1/NleH2 inhibit the anti-apoptotic protein Bax inhibitor-1 (BI-139). Furthermore, NleD cleaves JNK1 and JNK2, potentially affecting AP-1 activity in EHEC and EPEC-infected cells40.EspJ and its interactions with the cytoskeleton and phagocytosisEspJ is another poorly characterized effector that we found to interact with four proteins in our screens. Manually reviewing the function of its binding partners we found that EspJ was functionally linked to the cytoskeleton in three cases: (i) EspJ binds to the guanine nucleotide exchange factor Synembryn-A (RIC8A) that may be involved in signaling and the cytoskeleton41 (ii) Furthermore, EspJ interacts with intraflagellar transport protein 20 homolog (IFT20) that may play a role in the trafficking of ciliary membrane proteins from the Golgi complex to the cilium42. (iii) EspJ also interacts with centromere protein H (CENHP), a component of the nucleosome-associated complex that plays a role in assembly of kinetochore proteins, mitotic progression and chromosome segregation434445.EspJ was shown to inhibit FCYR and CR3-mediated phagocytosis (immune globulin and complement system-mediated, respectively)46, thereby avoiding the internalization of the bacterial cell and degradation by macrophages. However, the mechanism as well as any human binding partners that could explain these observations remained unknown. Another effector, EspH, is known to counteract FCYR phagocytosis by binding to Rho-GEFs (ARHGEF11, ARHGEF1, LARG) while blocking Rho activation through actin cytoskeletal remodeling46. Finally, EspF, inhibits phagocytosis probably through its actin cytoskeleton-related binding partners (actin, profilin, sorting nexin 9; reviewed in ref 7). The microtubule system is essential for optimal FCYR-mediated phagocytosis47. We speculate that EspJ may specifically affect the microtubule system to block phagocytosis while EspH and EspF interfere with such processes through targeting the actin cytoskeleton.EspO1 and EspF interact with cell cycle regulators and apoptosis proteinsEspO1-1: The Shigella flexneri effector OspE (an ortholog of EspO1-1) was shown to bind to integrin-linked kinase (ILK) at focal adhesions while reinforcing host cell adherence to the basement membrane4849. As for EHEC EspO1-1 we discovered another interaction with fem-1 homolog b (Fem1b), a component of an E3 ubiquitin-protein ligase complex acting as substrate recognition subunit50. Fem1b interacts with the checkpoint kinase CHEK1 to sense replication stress51 and can act as a proapoptotic protein in cancer cells525354. EHEC EspO1-1 may be another candidate that could act anti-apoptotically through its Fem1b interaction. In particular, targeting of the ubiquitin system by type III secreted factors is a common theme in pathogenesis of many bacteria (reviewed in ref 55).EspF1: EspF1 is a highly multifunctional effector that localizes to mitochondria, the cytosol, nucleus, apical and lateral membrane and tight junctions56. EPEC EspF1 interacts with 14-3-3-zeta, ABCF2, actin, ARP2, CK-18, N-WASP, profilin, ZO-1/ZO-2 and SNX9 (sorting nexin 9). These interactions potentially allow EPEC to disrupt mitochondria, tight junctions, nucleolus, intermediate filaments, PI3K-dependent phagocytosis, and membrane remodeling. Such processes are involved in apoptosis and counteract phagocytosis735575859606162. Specifically, we found that EHEC EspF1 interacts with sorting nexin 33 (SNX33) and mitotic arrest deficient like 2 (MAD2L2) as well as with SNX9, an interaction that was previously found for EPEC and EHEC EspF. Sorting nexins are a large evolutionary conserved protein family involved in cargo sorting through the endosomal network63. SNX9 and SNX33 - both members of the SNX9 subfamily - are (together with SNX18) required for progression and completion of mitosis64. This role in cell cycle regulation is in line with EspF's binding to MAD2L2, a protein participating in the spindle assembly checkpoint65, translesional DNA synthesis in S phase cells6667 and in Cdc20 homolog 1 (Cdh1) regulation68.Notably, MAD2L2 is also targeted by the Shigella effector IpaB, leading to a cell cycle arrest in the G2/M phase69, a cellular event that is also induced during EPEC infection70.TIR interactions in vitro and localization in vivoTo validate the quality of our data we focused on interactions isolated for the translocated intimin receptor (TIR). Our Y2H screen found eight human proteins interacting with TIR, namely arrestin beta 1 (ARRB1), BAI1-associated protein 2 (BAIAP2), BAI1-associated protein 2-like 1 (BAIAP2L1), hippocalcin-like 1 (HPCAL1), neurocalcin delta (NCALD), serine/threonine kinase 16 (STK16), hippocalcin like 4 (HPCAL4) and phosphodiesterase 6D (PDE6D).Interaction validation: To validate, we tested these interactions using LUMIER assays7172 (Figure 5A): The effector proteins were purified using a Protein A tag and co-purified binding partners fused to luciferase were detected by luminescence. This confirmed all Y2H interactions, including the interactions of TIR with BAIAP2 and BAIAP2L1 that have been previously found with EHEC and EPEC TIR1415.Five interactions are conserved in EHEC and EPEC: Since the TIRs of EHEC and EPEC share 58% of their amino acid sequence73 and carry out similar though not identical functions in these bacteria, we wondered if the isolated TIREHEC interactors would also bind TIREPEC. We used the N- and C-terminal cytosolic domains and a full-length construct of TIR to compare TIR of EHEC and EPEC (Figure 5B). We used BAIAP2 and BAIAP2L1 as positive controls as they are known to interact with both TIREPEC and TIREHEC1415. HPCAL1, NCALD and STK16 interacted with TIRs from both EPEC and EHEC, representing novel conserved targets of TIR. However, we could not detect any interactions between TIR and ARRB1 and HPCAL4 in EPEC.HPCAL1, NCALD and STK16 co-localize with EPEC infection sites in COS-7 cellsTo test the in vivo-relevance of the conserved TIREPEC interactors we expressed the human binders as YFP-tagged fusion proteins in COS-7 cells and infected the cells with EPEC afterwards. Subsequently, we assessed co-localization of the YFP constructs with the infection sites. As expected, BAIAP2L1 (Figure 5C) and BAIAP2 (data not shown) clearly co-localized with EPEC infection sites1415. As for HPCAL1, STK16 and NCALD, that interact with TIREPEC, we found that only HPCAL1 and STK16 clearly co-localized with EPEC infection sites despite stronger signals obtained for BAIAP2 and BAIAP2L1. In case of NCALD, the signal seems to be enhanced in the vicinity of EPEC infection sites, but not directly where bacteria and TIR are located. The localization is somewhat reminiscent of NHERF2 and Annexin A2, which also localize to membrane areas surrounding the bacteria21. As expected, ARRB1 and HPCAL4 (not shown) do not co-localize with EPEC infection sites, as they interact specifically with TIREHEC.In summary, we discovered another group of TIREHEC interactors, including three calcium binders of the VILIP family, namely HPCAL1, HPCAL4 and NCALD with highly similar amino acid sequences (62% identity). Although mainly known as proteins involved in neuronal signaling in the central nervous system7475, each gene is expressed in tissues of the digestive tract7677. The interactions of TIREHEC with the three calcium binding proteins are noteworthy, since the roles of Ca2+ signaling during EPEC infection are contentious78. Some studies found evidence supporting a role of Ca2+ in pedestal formation167980, others did not81. The interactions of TIR are summarized in Figure 6, including previously published interactions.

Results and Discussion

The EHEC effector-host protein interaction mapIn 2006 Tobe et al. identified 62 putative EHEC effector proteins through computational analysis and experimentally confirmed that 39 effectors were indeed translocated into host cells5. While several already have been demonstrated to modify certain host functions (reviewed in ref 7), many effector proteins remain functionally poorly characterized. To investigate the global landscape of effector-host interactions we systematically screened 34 of the 39 effectors verified by Tobe et al. using a well established, semi-automated comprehensive Yeast 2-Hybrid (Y2H) screening protocol2728 (Figure 1, Supplementary Table S1). Each effector was fused to the GAL4-DBD (DNA binding domain of the GAL4 transcription factor) on both its N- and C-terminus. Similarly, we built two different prey screening libraries from human open reading frame (ORFeome) collections293031. Each consisted of a single pool of over 10,000 individual human protein-coding ORFs that were either fused C- or N-terminally to the AD (activation domain) of GAL4. The N- and C-terminally-tagged effector bait constructs were then screened individually against both ORFeome libraries to increase the detection sensitivity3233 as well as against a cDNA prey library (Human Universal, Clontech). These screens resulted in 1,025 positive yeast colonies from which we successfully retrieved 328 unique EHEC bait-human prey pairs. We removed 207 pairs with prey sequences that were not located in the protein coding region of genes (3′-UTR artifacts), interactions involving randomly activating bait constructs (i.e. baits that returned exclusively multiple non-reproducible hits), or interactions including prey proteins known to interact non-selectively with many different bait proteins. In the latter case we excluded prey constructs known to bind to more than five bait proteins28 (Schwarz et al., unpublished). Out of the remaining 121 interactions 86 were detected only once, i.e., a prey could be assigned to the known bait protein only from one unique positive yeast colony. Using pairwise Y2H assays we retested all 121 pairs that had human full-length ORF constructs. We cloned 78 out of 90 human interactors into prey plasmids and were able to reproduce 35 out of 82 tested PPIs (42.7%). All 23 pairs that had two or more colonies in the screens were retested positive; pairs with only a single colony had a success rate of 20.7%. In the final dataset, we included all PPIs that were isolated multiple times, as well as those that were independently reproduced. In total, our approach yielded 48 individual EHEC-host PPIs of high confidence (Table 1 and for more details Supplementary Table S2) where the majority of 36 PPIs was identified by using a C-terminal ORFeome prey library. In comparison, the N-terminal ORFeome prey library and the cDNA prey libraries isolated clearly fewer interactions (seven PPIs each). Only two interactions were found in all the different ORFeome prey libraries (NleF with Caspase 9 and dihydrofolate reductase).To generate a combined protein interaction network including the 48 newly found interacions, we extensively reviewed the existing literature and curated 130 effector-host interactions, including 76 EHEC-host protein interactions. Furthermore, we found a few interactions that were inferred from protein complexes although no experimental evidence for a direct interaction appears to be available.The combined network is shown in Figure 2 (for a complete list of EHEC-host interactions and references see Supplementary Table S3). We found that five out of the discovered 48 pathogen-host pairs were previously published (Tir-BAIAP2L1 ( = IRTKS)14, Tir-BAIAP2 ( = IRSp53)15, Map-NHERF2 ( = SLC9A3R2)34, EspF-SNX935 and NleF-CASP923).From the literature, EHEC-host interactions are known for 18 out of 39 effector proteins described by Tobe et al. Our screens allowed us to find PPIs that involved 15 out of 39 effectors including EspJ and EspY1 that previously were not reported to interact with the host. While cellular targets of the remaining 13 effectors, EspB, EspF, EspG, EspO, Map, NleA, NleB; NleC, NleD, NleF, NleH, TccP (EspFu) and TIR, were previously known, we determined 31 novel interactions.

The EHEC effector-host protein interaction map

In 2006 Tobe et al. identified 62 putative EHEC effector proteins through computational analysis and experimentally confirmed that 39 effectors were indeed translocated into host cells5. While several already have been demonstrated to modify certain host functions (reviewed in ref 7), many effector proteins remain functionally poorly characterized. To investigate the global landscape of effector-host interactions we systematically screened 34 of the 39 effectors verified by Tobe et al. using a well established, semi-automated comprehensive Yeast 2-Hybrid (Y2H) screening protocol2728 (Figure 1, Supplementary Table S1). Each effector was fused to the GAL4-DBD (DNA binding domain of the GAL4 transcription factor) on both its N- and C-terminus. Similarly, we built two different prey screening libraries from human open reading frame (ORFeome) collections293031. Each consisted of a single pool of over 10,000 individual human protein-coding ORFs that were either fused C- or N-terminally to the AD (activation domain) of GAL4. The N- and C-terminally-tagged effector bait constructs were then screened individually against both ORFeome libraries to increase the detection sensitivity3233 as well as against a cDNA prey library (Human Universal, Clontech). These screens resulted in 1,025 positive yeast colonies from which we successfully retrieved 328 unique EHEC bait-human prey pairs. We removed 207 pairs with prey sequences that were not located in the protein coding region of genes (3′-UTR artifacts), interactions involving randomly activating bait constructs (i.e. baits that returned exclusively multiple non-reproducible hits), or interactions including prey proteins known to interact non-selectively with many different bait proteins. In the latter case we excluded prey constructs known to bind to more than five bait proteins28 (Schwarz et al., unpublished). Out of the remaining 121 interactions 86 were detected only once, i.e., a prey could be assigned to the known bait protein only from one unique positive yeast colony. Using pairwise Y2H assays we retested all 121 pairs that had human full-length ORF constructs. We cloned 78 out of 90 human interactors into prey plasmids and were able to reproduce 35 out of 82 tested PPIs (42.7%). All 23 pairs that had two or more colonies in the screens were retested positive; pairs with only a single colony had a success rate of 20.7%. In the final dataset, we included all PPIs that were isolated multiple times, as well as those that were independently reproduced. In total, our approach yielded 48 individual EHEC-host PPIs of high confidence (Table 1 and for more details Supplementary Table S2) where the majority of 36 PPIs was identified by using a C-terminal ORFeome prey library. In comparison, the N-terminal ORFeome prey library and the cDNA prey libraries isolated clearly fewer interactions (seven PPIs each). Only two interactions were found in all the different ORFeome prey libraries (NleF with Caspase 9 and dihydrofolate reductase).

To generate a combined protein interaction network including the 48 newly found interacions, we extensively reviewed the existing literature and curated 130 effector-host interactions, including 76 EHEC-host protein interactions. Furthermore, we found a few interactions that were inferred from protein complexes although no experimental evidence for a direct interaction appears to be available.

The combined network is shown in Figure 2 (for a complete list of EHEC-host interactions and references see Supplementary Table S3). We found that five out of the discovered 48 pathogen-host pairs were previously published (Tir-BAIAP2L1 ( = IRTKS)14, Tir-BAIAP2 ( = IRSp53)15, Map-NHERF2 ( = SLC9A3R2)34, EspF-SNX935 and NleF-CASP923).

From the literature, EHEC-host interactions are known for 18 out of 39 effector proteins described by Tobe et al. Our screens allowed us to find PPIs that involved 15 out of 39 effectors including EspJ and EspY1 that previously were not reported to interact with the host. While cellular targets of the remaining 13 effectors, EspB, EspF, EspG, EspO, Map, NleA, NleB; NleC, NleD, NleF, NleH, TccP (EspFu) and TIR, were previously known, we determined 31 novel interactions.

A global analysis of EHEC-host interactions compared to other pathogensTo compare proteins targeted by EHEC with those of other pathogens, we compared our data to 8,297 protein interactions between proteins of B. anthracis, F. tularensis and Y. pestis and 3,363 human host proteins36. Furthermore, we compiled a set of 3,156 interactions between viral proteins of the Epstein-Barr, HIV, Hepatitis, Herpes, Influenza, Papilloma and Vaccinia virus and 1,778 human host proteins37. We find that many human host proteins were targeted by both bacterial and viral proteins (Figure 3A).To investigate the impact on pathways, we determined the enrichment of proteins targeted by EHEC effectors in pathways that we collected from the molecular signature database38. We observed a significant enrichment of a small set of mostly signaling as well as infection-related pathways (Figure 3B).In general, the majority of human target proteins interact with one effector protein, a characteristic that is shared with bacterial and viral targets (Figure 4A). Next, we wondered whether pathogen proteins interact with highly connected human proteins (hubs) or rather focus on less frequently interacting proteins in the human interactome. Figure 4B indicates that bacterial as well as viral proteins are predominately enriched in groups of highly connected host proteins. In comparison, targets of EHEC effector proteins surprisingly show a significantly stronger enrichment signal. Calculating shortest path lengths (Figure 4C) we observed that proteins targeted by EHEC effector proteins had shorter paths to other human proteins than the corresponding targets of other bacteria and of viruses (Student's t-test, p < 10−20). A similar trend for binding to highly connected proteins is apparent in the enrichment of targeted proteins in protein complexes, as shown in Figure 4D. In comparison to bacterial and viral targets we observed that targets of EHEC effector proteins appear significantly more frequently in protein complexes.

A global analysis of EHEC-host interactions compared to other pathogens

To compare proteins targeted by EHEC with those of other pathogens, we compared our data to 8,297 protein interactions between proteins of B. anthracis, F. tularensis and Y. pestis and 3,363 human host proteins36. Furthermore, we compiled a set of 3,156 interactions between viral proteins of the Epstein-Barr, HIV, Hepatitis, Herpes, Influenza, Papilloma and Vaccinia virus and 1,778 human host proteins37. We find that many human host proteins were targeted by both bacterial and viral proteins (Figure 3A).

To investigate the impact on pathways, we determined the enrichment of proteins targeted by EHEC effectors in pathways that we collected from the molecular signature database38. We observed a significant enrichment of a small set of mostly signaling as well as infection-related pathways (Figure 3B).

In general, the majority of human target proteins interact with one effector protein, a characteristic that is shared with bacterial and viral targets (Figure 4A). Next, we wondered whether pathogen proteins interact with highly connected human proteins (hubs) or rather focus on less frequently interacting proteins in the human interactome. Figure 4B indicates that bacterial as well as viral proteins are predominately enriched in groups of highly connected host proteins. In comparison, targets of EHEC effector proteins surprisingly show a significantly stronger enrichment signal. Calculating shortest path lengths (Figure 4C) we observed that proteins targeted by EHEC effector proteins had shorter paths to other human proteins than the corresponding targets of other bacteria and of viruses (Student's t-test, p < 10−20). A similar trend for binding to highly connected proteins is apparent in the enrichment of targeted proteins in protein complexes, as shown in Figure 4D. In comparison to bacterial and viral targets we observed that targets of EHEC effector proteins appear significantly more frequently in protein complexes.

EspY1 interacts with proteins involved in apoptosis and cell cycle regulationWhile neither a function nor a human interacting protein was previously known for EspY1, we detected eight novel interactions that involved this effector protein. EspY1 appears to be primarily involved in apoptosis and cell cycle regulation as these functions are shared among its host targets calpain-3, 26S protease regulatory subunit 4, and PCI domain-containing protein 2. Such interactions may complement other EHEC effector proteins that target host proteins known to block apoptosis while assuring host cell survival. For instance, NleF interacts with caspases 4,8, and 923, while NleH1/NleH2 inhibit the anti-apoptotic protein Bax inhibitor-1 (BI-139). Furthermore, NleD cleaves JNK1 and JNK2, potentially affecting AP-1 activity in EHEC and EPEC-infected cells40.

EspY1 interacts with proteins involved in apoptosis and cell cycle regulation

While neither a function nor a human interacting protein was previously known for EspY1, we detected eight novel interactions that involved this effector protein. EspY1 appears to be primarily involved in apoptosis and cell cycle regulation as these functions are shared among its host targets calpain-3, 26S protease regulatory subunit 4, and PCI domain-containing protein 2. Such interactions may complement other EHEC effector proteins that target host proteins known to block apoptosis while assuring host cell survival. For instance, NleF interacts with caspases 4,8, and 923, while NleH1/NleH2 inhibit the anti-apoptotic protein Bax inhibitor-1 (BI-139). Furthermore, NleD cleaves JNK1 and JNK2, potentially affecting AP-1 activity in EHEC and EPEC-infected cells40.

EspJ and its interactions with the cytoskeleton and phagocytosisEspJ is another poorly characterized effector that we found to interact with four proteins in our screens. Manually reviewing the function of its binding partners we found that EspJ was functionally linked to the cytoskeleton in three cases: (i) EspJ binds to the guanine nucleotide exchange factor Synembryn-A (RIC8A) that may be involved in signaling and the cytoskeleton41 (ii) Furthermore, EspJ interacts with intraflagellar transport protein 20 homolog (IFT20) that may play a role in the trafficking of ciliary membrane proteins from the Golgi complex to the cilium42. (iii) EspJ also interacts with centromere protein H (CENHP), a component of the nucleosome-associated complex that plays a role in assembly of kinetochore proteins, mitotic progression and chromosome segregation434445.EspJ was shown to inhibit FCYR and CR3-mediated phagocytosis (immune globulin and complement system-mediated, respectively)46, thereby avoiding the internalization of the bacterial cell and degradation by macrophages. However, the mechanism as well as any human binding partners that could explain these observations remained unknown. Another effector, EspH, is known to counteract FCYR phagocytosis by binding to Rho-GEFs (ARHGEF11, ARHGEF1, LARG) while blocking Rho activation through actin cytoskeletal remodeling46. Finally, EspF, inhibits phagocytosis probably through its actin cytoskeleton-related binding partners (actin, profilin, sorting nexin 9; reviewed in ref 7). The microtubule system is essential for optimal FCYR-mediated phagocytosis47. We speculate that EspJ may specifically affect the microtubule system to block phagocytosis while EspH and EspF interfere with such processes through targeting the actin cytoskeleton.

EspJ and its interactions with the cytoskeleton and phagocytosis

EspJ is another poorly characterized effector that we found to interact with four proteins in our screens. Manually reviewing the function of its binding partners we found that EspJ was functionally linked to the cytoskeleton in three cases: (i) EspJ binds to the guanine nucleotide exchange factor Synembryn-A (RIC8A) that may be involved in signaling and the cytoskeleton41 (ii) Furthermore, EspJ interacts with intraflagellar transport protein 20 homolog (IFT20) that may play a role in the trafficking of ciliary membrane proteins from the Golgi complex to the cilium42. (iii) EspJ also interacts with centromere protein H (CENHP), a component of the nucleosome-associated complex that plays a role in assembly of kinetochore proteins, mitotic progression and chromosome segregation434445.

EspJ was shown to inhibit FCYR and CR3-mediated phagocytosis (immune globulin and complement system-mediated, respectively)46, thereby avoiding the internalization of the bacterial cell and degradation by macrophages. However, the mechanism as well as any human binding partners that could explain these observations remained unknown. Another effector, EspH, is known to counteract FCYR phagocytosis by binding to Rho-GEFs (ARHGEF11, ARHGEF1, LARG) while blocking Rho activation through actin cytoskeletal remodeling46. Finally, EspF, inhibits phagocytosis probably through its actin cytoskeleton-related binding partners (actin, profilin, sorting nexin 9; reviewed in ref 7). The microtubule system is essential for optimal FCYR-mediated phagocytosis47. We speculate that EspJ may specifically affect the microtubule system to block phagocytosis while EspH and EspF interfere with such processes through targeting the actin cytoskeleton.

EspO1 and EspF interact with cell cycle regulators and apoptosis proteinsEspO1-1: The Shigella flexneri effector OspE (an ortholog of EspO1-1) was shown to bind to integrin-linked kinase (ILK) at focal adhesions while reinforcing host cell adherence to the basement membrane4849. As for EHEC EspO1-1 we discovered another interaction with fem-1 homolog b (Fem1b), a component of an E3 ubiquitin-protein ligase complex acting as substrate recognition subunit50. Fem1b interacts with the checkpoint kinase CHEK1 to sense replication stress51 and can act as a proapoptotic protein in cancer cells525354. EHEC EspO1-1 may be another candidate that could act anti-apoptotically through its Fem1b interaction. In particular, targeting of the ubiquitin system by type III secreted factors is a common theme in pathogenesis of many bacteria (reviewed in ref 55).EspF1: EspF1 is a highly multifunctional effector that localizes to mitochondria, the cytosol, nucleus, apical and lateral membrane and tight junctions56. EPEC EspF1 interacts with 14-3-3-zeta, ABCF2, actin, ARP2, CK-18, N-WASP, profilin, ZO-1/ZO-2 and SNX9 (sorting nexin 9). These interactions potentially allow EPEC to disrupt mitochondria, tight junctions, nucleolus, intermediate filaments, PI3K-dependent phagocytosis, and membrane remodeling. Such processes are involved in apoptosis and counteract phagocytosis735575859606162. Specifically, we found that EHEC EspF1 interacts with sorting nexin 33 (SNX33) and mitotic arrest deficient like 2 (MAD2L2) as well as with SNX9, an interaction that was previously found for EPEC and EHEC EspF. Sorting nexins are a large evolutionary conserved protein family involved in cargo sorting through the endosomal network63. SNX9 and SNX33 - both members of the SNX9 subfamily - are (together with SNX18) required for progression and completion of mitosis64. This role in cell cycle regulation is in line with EspF's binding to MAD2L2, a protein participating in the spindle assembly checkpoint65, translesional DNA synthesis in S phase cells6667 and in Cdc20 homolog 1 (Cdh1) regulation68.Notably, MAD2L2 is also targeted by the Shigella effector IpaB, leading to a cell cycle arrest in the G2/M phase69, a cellular event that is also induced during EPEC infection70.

EspO1 and EspF interact with cell cycle regulators and apoptosis proteins

EspO1-1: The Shigella flexneri effector OspE (an ortholog of EspO1-1) was shown to bind to integrin-linked kinase (ILK) at focal adhesions while reinforcing host cell adherence to the basement membrane4849. As for EHEC EspO1-1 we discovered another interaction with fem-1 homolog b (Fem1b), a component of an E3 ubiquitin-protein ligase complex acting as substrate recognition subunit50. Fem1b interacts with the checkpoint kinase CHEK1 to sense replication stress51 and can act as a proapoptotic protein in cancer cells525354. EHEC EspO1-1 may be another candidate that could act anti-apoptotically through its Fem1b interaction. In particular, targeting of the ubiquitin system by type III secreted factors is a common theme in pathogenesis of many bacteria (reviewed in ref 55).

EspF1: EspF1 is a highly multifunctional effector that localizes to mitochondria, the cytosol, nucleus, apical and lateral membrane and tight junctions56. EPEC EspF1 interacts with 14-3-3-zeta, ABCF2, actin, ARP2, CK-18, N-WASP, profilin, ZO-1/ZO-2 and SNX9 (sorting nexin 9). These interactions potentially allow EPEC to disrupt mitochondria, tight junctions, nucleolus, intermediate filaments, PI3K-dependent phagocytosis, and membrane remodeling. Such processes are involved in apoptosis and counteract phagocytosis735575859606162. Specifically, we found that EHEC EspF1 interacts with sorting nexin 33 (SNX33) and mitotic arrest deficient like 2 (MAD2L2) as well as with SNX9, an interaction that was previously found for EPEC and EHEC EspF. Sorting nexins are a large evolutionary conserved protein family involved in cargo sorting through the endosomal network63. SNX9 and SNX33 - both members of the SNX9 subfamily - are (together with SNX18) required for progression and completion of mitosis64. This role in cell cycle regulation is in line with EspF's binding to MAD2L2, a protein participating in the spindle assembly checkpoint65, translesional DNA synthesis in S phase cells6667 and in Cdc20 homolog 1 (Cdh1) regulation68.

Notably, MAD2L2 is also targeted by the Shigella effector IpaB, leading to a cell cycle arrest in the G2/M phase69, a cellular event that is also induced during EPEC infection70.

TIR interactions in vitro and localization in vivoTo validate the quality of our data we focused on interactions isolated for the translocated intimin receptor (TIR). Our Y2H screen found eight human proteins interacting with TIR, namely arrestin beta 1 (ARRB1), BAI1-associated protein 2 (BAIAP2), BAI1-associated protein 2-like 1 (BAIAP2L1), hippocalcin-like 1 (HPCAL1), neurocalcin delta (NCALD), serine/threonine kinase 16 (STK16), hippocalcin like 4 (HPCAL4) and phosphodiesterase 6D (PDE6D).Interaction validation: To validate, we tested these interactions using LUMIER assays7172 (Figure 5A): The effector proteins were purified using a Protein A tag and co-purified binding partners fused to luciferase were detected by luminescence. This confirmed all Y2H interactions, including the interactions of TIR with BAIAP2 and BAIAP2L1 that have been previously found with EHEC and EPEC TIR1415.Five interactions are conserved in EHEC and EPEC: Since the TIRs of EHEC and EPEC share 58% of their amino acid sequence73 and carry out similar though not identical functions in these bacteria, we wondered if the isolated TIREHEC interactors would also bind TIREPEC. We used the N- and C-terminal cytosolic domains and a full-length construct of TIR to compare TIR of EHEC and EPEC (Figure 5B). We used BAIAP2 and BAIAP2L1 as positive controls as they are known to interact with both TIREPEC and TIREHEC1415. HPCAL1, NCALD and STK16 interacted with TIRs from both EPEC and EHEC, representing novel conserved targets of TIR. However, we could not detect any interactions between TIR and ARRB1 and HPCAL4 in EPEC.

TIR interactions in vitro and localization in vivo

To validate the quality of our data we focused on interactions isolated for the translocated intimin receptor (TIR). Our Y2H screen found eight human proteins interacting with TIR, namely arrestin beta 1 (ARRB1), BAI1-associated protein 2 (BAIAP2), BAI1-associated protein 2-like 1 (BAIAP2L1), hippocalcin-like 1 (HPCAL1), neurocalcin delta (NCALD), serine/threonine kinase 16 (STK16), hippocalcin like 4 (HPCAL4) and phosphodiesterase 6D (PDE6D).

Interaction validation: To validate, we tested these interactions using LUMIER assays7172 (Figure 5A): The effector proteins were purified using a Protein A tag and co-purified binding partners fused to luciferase were detected by luminescence. This confirmed all Y2H interactions, including the interactions of TIR with BAIAP2 and BAIAP2L1 that have been previously found with EHEC and EPEC TIR1415.

Five interactions are conserved in EHEC and EPEC: Since the TIRs of EHEC and EPEC share 58% of their amino acid sequence73 and carry out similar though not identical functions in these bacteria, we wondered if the isolated TIREHEC interactors would also bind TIREPEC. We used the N- and C-terminal cytosolic domains and a full-length construct of TIR to compare TIR of EHEC and EPEC (Figure 5B). We used BAIAP2 and BAIAP2L1 as positive controls as they are known to interact with both TIREPEC and TIREHEC1415. HPCAL1, NCALD and STK16 interacted with TIRs from both EPEC and EHEC, representing novel conserved targets of TIR. However, we could not detect any interactions between TIR and ARRB1 and HPCAL4 in EPEC.

HPCAL1, NCALD and STK16 co-localize with EPEC infection sites in COS-7 cellsTo test the in vivo-relevance of the conserved TIREPEC interactors we expressed the human binders as YFP-tagged fusion proteins in COS-7 cells and infected the cells with EPEC afterwards. Subsequently, we assessed co-localization of the YFP constructs with the infection sites. As expected, BAIAP2L1 (Figure 5C) and BAIAP2 (data not shown) clearly co-localized with EPEC infection sites1415. As for HPCAL1, STK16 and NCALD, that interact with TIREPEC, we found that only HPCAL1 and STK16 clearly co-localized with EPEC infection sites despite stronger signals obtained for BAIAP2 and BAIAP2L1. In case of NCALD, the signal seems to be enhanced in the vicinity of EPEC infection sites, but not directly where bacteria and TIR are located. The localization is somewhat reminiscent of NHERF2 and Annexin A2, which also localize to membrane areas surrounding the bacteria21. As expected, ARRB1 and HPCAL4 (not shown) do not co-localize with EPEC infection sites, as they interact specifically with TIREHEC.In summary, we discovered another group of TIREHEC interactors, including three calcium binders of the VILIP family, namely HPCAL1, HPCAL4 and NCALD with highly similar amino acid sequences (62% identity). Although mainly known as proteins involved in neuronal signaling in the central nervous system7475, each gene is expressed in tissues of the digestive tract7677. The interactions of TIREHEC with the three calcium binding proteins are noteworthy, since the roles of Ca2+ signaling during EPEC infection are contentious78. Some studies found evidence supporting a role of Ca2+ in pedestal formation167980, others did not81. The interactions of TIR are summarized in Figure 6, including previously published interactions.

HPCAL1, NCALD and STK16 co-localize with EPEC infection sites in COS-7 cells

To test the in vivo-relevance of the conserved TIREPEC interactors we expressed the human binders as YFP-tagged fusion proteins in COS-7 cells and infected the cells with EPEC afterwards. Subsequently, we assessed co-localization of the YFP constructs with the infection sites. As expected, BAIAP2L1 (Figure 5C) and BAIAP2 (data not shown) clearly co-localized with EPEC infection sites1415. As for HPCAL1, STK16 and NCALD, that interact with TIREPEC, we found that only HPCAL1 and STK16 clearly co-localized with EPEC infection sites despite stronger signals obtained for BAIAP2 and BAIAP2L1. In case of NCALD, the signal seems to be enhanced in the vicinity of EPEC infection sites, but not directly where bacteria and TIR are located. The localization is somewhat reminiscent of NHERF2 and Annexin A2, which also localize to membrane areas surrounding the bacteria21. As expected, ARRB1 and HPCAL4 (not shown) do not co-localize with EPEC infection sites, as they interact specifically with TIREHEC.

In summary, we discovered another group of TIREHEC interactors, including three calcium binders of the VILIP family, namely HPCAL1, HPCAL4 and NCALD with highly similar amino acid sequences (62% identity). Although mainly known as proteins involved in neuronal signaling in the central nervous system7475, each gene is expressed in tissues of the digestive tract7677. The interactions of TIREHEC with the three calcium binding proteins are noteworthy, since the roles of Ca2+ signaling during EPEC infection are contentious78. Some studies found evidence supporting a role of Ca2+ in pedestal formation167980, others did not81. The interactions of TIR are summarized in Figure 6, including previously published interactions.

ConclusionsIn this study we determined 43 novel and 5 previously published pathogen-host interactions of EHEC effector proteins. Including data from the literature, our network comparisons showed that EHEC effectors attack the host interactome more specifically than bacteria and viruses.The quality of the interactome was exemplarily confirmed for six novel TIR interactions by LUMIER assay. Furthermore, we provide evidence for the biological relevance of three interactions, involving NCALD, HPCAL1 and STK16, based on their recruitment to EPEC infection sites.Notably, these three TIR interactors are small calcium binding proteins that are composed of four consecutive tandem repeated EF-hand motifs. They all belong to the superfamily of neuronal calcium sensors8283 and were found in TIR-dependent signaling complexes, in addition to AnxA221, IQGAP and calmodulin16. Early upon pathogen contact, microvilli effacement is induced, which was also shown to involve calcium signaling and calcium binding proteins such as calpain84. Calcium has been implicated in microvilli- effacement and disruption8485 but requires detailed further analysis for a better understanding of EHEC/EPEC pathogenicity.

Conclusions

In this study we determined 43 novel and 5 previously published pathogen-host interactions of EHEC effector proteins. Including data from the literature, our network comparisons showed that EHEC effectors attack the host interactome more specifically than bacteria and viruses.

The quality of the interactome was exemplarily confirmed for six novel TIR interactions by LUMIER assay. Furthermore, we provide evidence for the biological relevance of three interactions, involving NCALD, HPCAL1 and STK16, based on their recruitment to EPEC infection sites.

Notably, these three TIR interactors are small calcium binding proteins that are composed of four consecutive tandem repeated EF-hand motifs. They all belong to the superfamily of neuronal calcium sensors8283 and were found in TIR-dependent signaling complexes, in addition to AnxA221, IQGAP and calmodulin16. Early upon pathogen contact, microvilli effacement is induced, which was also shown to involve calcium signaling and calcium binding proteins such as calpain84. Calcium has been implicated in microvilli- effacement and disruption8485 but requires detailed further analysis for a better understanding of EHEC/EPEC pathogenicity.

MethodsCloningEffector ORFs of EHEC5 were cloned from E. coli O157:H7 str. Sakai DNA using the Gateway® Technology (Invitrogen) (see Supplementary Table S1). Each effector was amplified in a two-step PCR using gene specific primers and the universal second primers attB1 and attB2 for the attachment of the Gateway attB sites. Primer design, PCR and cloning followed manufacturer's instructions. Resulting PCR products were shuttled into the pDONR221 entry vector via Gateway® BP reactions. Correctness of entry clones was confirmed by sequencing. Then the ORFs were shuttled into Gateway-compatible pGBT9 and pGBKCg plasmids2732 using an LR reaction (Invitrogen). For the LUMIER tests, TIREHEC was shuttled into pTREX-dest30-ntPrA72. Human ORFs of the TIR binders were shuttled from entry clones (collections293031) into pcDNA3-Rluc-GW72 and for the infection/localization assay into pEYFP-N186.For the amplification of the TIREHEC amino-and carboxy-terminal fragments the gene specific primers 5′-A GGC TCC ACC ATG AAA ATA ACA AAC TAT ATA CTG CC-3′ (forw) + 5′-C TGG GTG GAT YCA ACT TTC CCT CCC GTA AAG-3′ (rev) and 5′-A GGC TCC ACC ATG GGC GGG CAG CAG TTT ATT TGG-3′ (forw) + 5′-C TGG GTG GAT YCA TTT CGA TGC ATT TAC CAT TGA G-3′ (rev) were used, respectively.TIREPEC was cloned from E. coli O127:H6 str. E2348/69. The amino- and carboxy terminal regions as well as full length constructs were amplified using the gene specific primer pairs 5′-A GGC TCC ACC ATG CCT ATT GGT AAC CTT GG-3′ (forw) + 5′-C TGG GTG GAT YCA ATC GGT GGT TGT AGG ATC-3′ (rev), 5′-A GGC TCC ACC ATG GAA AGC AAT GCA CAG GCG C-3′ (forw) + 5′-C TGG GTG GAT YCA AAC GAA ACG TAC TGG TCC C-3 (rev) and 5′-A GGC TCC ACC ATG CCT ATT GGT AAC CTT GG-3′ (forw) + 5′-C TGG GTG GAT YCA AAC GAA ACG TAC TGG TCC C-3′ (rev), respectively. Depending on whether a stop is required or not Y can be either T or C (stop or no stop). The second PCR and cloning were performed as described for the EHEC effectors.Y2H prey librariesThe Mammalian Gene Collection (MGC)30 and the human ORF collection from the German Center for Genome Research (RZPD)293031 were shuttled as pools into pGAD424-GW (Clontech, Gateway-compatible variant) and pGADCg (Clontech32), using an LR reaction. After counter selection in E. coli DH10B prey plasmids were then transfected into yeast strain Y187 (Clontech). The cDNA prey library was purchased from Clontech (Human Universal, Clontech 638874).Y2H screeningY2H pool screening was done as described in refs 27, 28 (Figure 1).Pairwise Y2H testsIn binary Y2H assays, a single bait and a single prey plasmid were transformed in the bait and prey yeast strains, CG-1945 and Y187, respectively (Clontech). Bait and prey were mated on YPDA plates at 30°C over night or for 3 h at 100 rpm in liquid YPDA containing 40% PEG. Diploids were harvested and grown in ΔLW medium for 2 days and then plated on selective agar plates (ΔLWH). To adjust stringency, various concentrations ranging from 0 to 150 mM of 3-amino-1, 2, 4-triazole were added to the selective agar plates. To check for auto-activation all baits were tested against Y187 harboring empty prey vector (negative control). Details can be found in ref 87.LUMIER assayLuciferase based pull down assays were performed as described in ref 23.Infection assayTransfection of the YFP constructs, infection assays, and image analysis was performed as described in ref 15.Computational analysisHuman protein interactionsWe utilized a total of 28,627 high quality protein interactions between 8,495 human proteins from the HINT database88, accounting for both binary and co-complex interactions.Human canonical pathway informationAs a source of canonical pathway information, we used 1,320 pathways from the Molecular Signatures Database38 that were largely obtained from the NCI Pathway Interaction Database, Reactome and Biocarta.Bacteria-host and Virus-host protein interactionsAs for Bacteria we utilized 2,989 interactions of B. anthracis, 1,347 interactions of F. tularensis and 3,961 interactions of Y. pestis with human host proteins36. Utilizing the IntAct database37, we collected 330 interactions of the Epstein-Barr virus, 557 interactions of the HIV virus, 766 interactions of the Hepatitis virus, 332 interactions of the Herpes virus, 756 interactions of the Influenza virus, 238 interactions from the Papilloma virus and 177 interactions of the Vaccinia virus with human host proteins.Human protein complexesWe utilized 1,843 protein complexes in H. sapiens from the CORUM database89, that collects information about experimentally determined protein complexes from the literature.Enrichment analysis as a function of degreeWe grouped human proteins according to their number of interactions in an underlying human protein interaction network. We represented each group by N≥k proteins that had at least k interactions and calculated the number of targeted proteins i, Ni,≥k in each group. Randomly picking targeted genes we defined  as their enrichment where was the corresponding random number of targeted proteins among all Ni,≥k proteins. After averaging Ei over 10,000 randomizations Ei > 1 pointed to an enrichment and vice versa, while Ei ~ 1 indicated a random process90.

CloningEffector ORFs of EHEC5 were cloned from E. coli O157:H7 str. Sakai DNA using the Gateway® Technology (Invitrogen) (see Supplementary Table S1). Each effector was amplified in a two-step PCR using gene specific primers and the universal second primers attB1 and attB2 for the attachment of the Gateway attB sites. Primer design, PCR and cloning followed manufacturer's instructions. Resulting PCR products were shuttled into the pDONR221 entry vector via Gateway® BP reactions. Correctness of entry clones was confirmed by sequencing. Then the ORFs were shuttled into Gateway-compatible pGBT9 and pGBKCg plasmids2732 using an LR reaction (Invitrogen). For the LUMIER tests, TIREHEC was shuttled into pTREX-dest30-ntPrA72. Human ORFs of the TIR binders were shuttled from entry clones (collections293031) into pcDNA3-Rluc-GW72 and for the infection/localization assay into pEYFP-N186.For the amplification of the TIREHEC amino-and carboxy-terminal fragments the gene specific primers 5′-A GGC TCC ACC ATG AAA ATA ACA AAC TAT ATA CTG CC-3′ (forw) + 5′-C TGG GTG GAT YCA ACT TTC CCT CCC GTA AAG-3′ (rev) and 5′-A GGC TCC ACC ATG GGC GGG CAG CAG TTT ATT TGG-3′ (forw) + 5′-C TGG GTG GAT YCA TTT CGA TGC ATT TAC CAT TGA G-3′ (rev) were used, respectively.TIREPEC was cloned from E. coli O127:H6 str. E2348/69. The amino- and carboxy terminal regions as well as full length constructs were amplified using the gene specific primer pairs 5′-A GGC TCC ACC ATG CCT ATT GGT AAC CTT GG-3′ (forw) + 5′-C TGG GTG GAT YCA ATC GGT GGT TGT AGG ATC-3′ (rev), 5′-A GGC TCC ACC ATG GAA AGC AAT GCA CAG GCG C-3′ (forw) + 5′-C TGG GTG GAT YCA AAC GAA ACG TAC TGG TCC C-3 (rev) and 5′-A GGC TCC ACC ATG CCT ATT GGT AAC CTT GG-3′ (forw) + 5′-C TGG GTG GAT YCA AAC GAA ACG TAC TGG TCC C-3′ (rev), respectively. Depending on whether a stop is required or not Y can be either T or C (stop or no stop). The second PCR and cloning were performed as described for the EHEC effectors.

Effector ORFs of EHEC5 were cloned from E. coli O157:H7 str. Sakai DNA using the Gateway® Technology (Invitrogen) (see Supplementary Table S1). Each effector was amplified in a two-step PCR using gene specific primers and the universal second primers attB1 and attB2 for the attachment of the Gateway attB sites. Primer design, PCR and cloning followed manufacturer's instructions. Resulting PCR products were shuttled into the pDONR221 entry vector via Gateway® BP reactions. Correctness of entry clones was confirmed by sequencing. Then the ORFs were shuttled into Gateway-compatible pGBT9 and pGBKCg plasmids2732 using an LR reaction (Invitrogen). For the LUMIER tests, TIREHEC was shuttled into pTREX-dest30-ntPrA72. Human ORFs of the TIR binders were shuttled from entry clones (collections293031) into pcDNA3-Rluc-GW72 and for the infection/localization assay into pEYFP-N186.

For the amplification of the TIREHEC amino-and carboxy-terminal fragments the gene specific primers 5′-A GGC TCC ACC ATG AAA ATA ACA AAC TAT ATA CTG CC-3′ (forw) + 5′-C TGG GTG GAT YCA ACT TTC CCT CCC GTA AAG-3′ (rev) and 5′-A GGC TCC ACC ATG GGC GGG CAG CAG TTT ATT TGG-3′ (forw) + 5′-C TGG GTG GAT YCA TTT CGA TGC ATT TAC CAT TGA G-3′ (rev) were used, respectively.

TIREPEC was cloned from E. coli O127:H6 str. E2348/69. The amino- and carboxy terminal regions as well as full length constructs were amplified using the gene specific primer pairs 5′-A GGC TCC ACC ATG CCT ATT GGT AAC CTT GG-3′ (forw) + 5′-C TGG GTG GAT YCA ATC GGT GGT TGT AGG ATC-3′ (rev), 5′-A GGC TCC ACC ATG GAA AGC AAT GCA CAG GCG C-3′ (forw) + 5′-C TGG GTG GAT YCA AAC GAA ACG TAC TGG TCC C-3 (rev) and 5′-A GGC TCC ACC ATG CCT ATT GGT AAC CTT GG-3′ (forw) + 5′-C TGG GTG GAT YCA AAC GAA ACG TAC TGG TCC C-3′ (rev), respectively. Depending on whether a stop is required or not Y can be either T or C (stop or no stop). The second PCR and cloning were performed as described for the EHEC effectors.

Y2H prey librariesThe Mammalian Gene Collection (MGC)30 and the human ORF collection from the German Center for Genome Research (RZPD)293031 were shuttled as pools into pGAD424-GW (Clontech, Gateway-compatible variant) and pGADCg (Clontech32), using an LR reaction. After counter selection in E. coli DH10B prey plasmids were then transfected into yeast strain Y187 (Clontech). The cDNA prey library was purchased from Clontech (Human Universal, Clontech 638874).

Y2H prey libraries

The Mammalian Gene Collection (MGC)30 and the human ORF collection from the German Center for Genome Research (RZPD)293031 were shuttled as pools into pGAD424-GW (Clontech, Gateway-compatible variant) and pGADCg (Clontech32), using an LR reaction. After counter selection in E. coli DH10B prey plasmids were then transfected into yeast strain Y187 (Clontech). The cDNA prey library was purchased from Clontech (Human Universal, Clontech 638874).

Y2H screeningY2H pool screening was done as described in refs 27, 28 (Figure 1).

Y2H screening

Y2H pool screening was done as described in refs 27, 28 (Figure 1).

Pairwise Y2H testsIn binary Y2H assays, a single bait and a single prey plasmid were transformed in the bait and prey yeast strains, CG-1945 and Y187, respectively (Clontech). Bait and prey were mated on YPDA plates at 30°C over night or for 3 h at 100 rpm in liquid YPDA containing 40% PEG. Diploids were harvested and grown in ΔLW medium for 2 days and then plated on selective agar plates (ΔLWH). To adjust stringency, various concentrations ranging from 0 to 150 mM of 3-amino-1, 2, 4-triazole were added to the selective agar plates. To check for auto-activation all baits were tested against Y187 harboring empty prey vector (negative control). Details can be found in ref 87.

Pairwise Y2H tests

In binary Y2H assays, a single bait and a single prey plasmid were transformed in the bait and prey yeast strains, CG-1945 and Y187, respectively (Clontech). Bait and prey were mated on YPDA plates at 30°C over night or for 3 h at 100 rpm in liquid YPDA containing 40% PEG. Diploids were harvested and grown in ΔLW medium for 2 days and then plated on selective agar plates (ΔLWH). To adjust stringency, various concentrations ranging from 0 to 150 mM of 3-amino-1, 2, 4-triazole were added to the selective agar plates. To check for auto-activation all baits were tested against Y187 harboring empty prey vector (negative control). Details can be found in ref 87.

LUMIER assayLuciferase based pull down assays were performed as described in ref 23.

LUMIER assay

Luciferase based pull down assays were performed as described in ref 23.

Infection assayTransfection of the YFP constructs, infection assays, and image analysis was performed as described in ref 15.

Infection assay

Transfection of the YFP constructs, infection assays, and image analysis was performed as described in ref 15.

Computational analysisHuman protein interactionsWe utilized a total of 28,627 high quality protein interactions between 8,495 human proteins from the HINT database88, accounting for both binary and co-complex interactions.Human canonical pathway informationAs a source of canonical pathway information, we used 1,320 pathways from the Molecular Signatures Database38 that were largely obtained from the NCI Pathway Interaction Database, Reactome and Biocarta.Bacteria-host and Virus-host protein interactionsAs for Bacteria we utilized 2,989 interactions of B. anthracis, 1,347 interactions of F. tularensis and 3,961 interactions of Y. pestis with human host proteins36. Utilizing the IntAct database37, we collected 330 interactions of the Epstein-Barr virus, 557 interactions of the HIV virus, 766 interactions of the Hepatitis virus, 332 interactions of the Herpes virus, 756 interactions of the Influenza virus, 238 interactions from the Papilloma virus and 177 interactions of the Vaccinia virus with human host proteins.Human protein complexesWe utilized 1,843 protein complexes in H. sapiens from the CORUM database89, that collects information about experimentally determined protein complexes from the literature.Enrichment analysis as a function of degreeWe grouped human proteins according to their number of interactions in an underlying human protein interaction network. We represented each group by N≥k proteins that had at least k interactions and calculated the number of targeted proteins i, Ni,≥k in each group. Randomly picking targeted genes we defined  as their enrichment where was the corresponding random number of targeted proteins among all Ni,≥k proteins. After averaging Ei over 10,000 randomizations Ei > 1 pointed to an enrichment and vice versa, while Ei ~ 1 indicated a random process90.

Computational analysis

Human protein interactionsWe utilized a total of 28,627 high quality protein interactions between 8,495 human proteins from the HINT database88, accounting for both binary and co-complex interactions.

Human protein interactions

We utilized a total of 28,627 high quality protein interactions between 8,495 human proteins from the HINT database88, accounting for both binary and co-complex interactions.

Human canonical pathway informationAs a source of canonical pathway information, we used 1,320 pathways from the Molecular Signatures Database38 that were largely obtained from the NCI Pathway Interaction Database, Reactome and Biocarta.

Human canonical pathway information

As a source of canonical pathway information, we used 1,320 pathways from the Molecular Signatures Database38 that were largely obtained from the NCI Pathway Interaction Database, Reactome and Biocarta.

Bacteria-host and Virus-host protein interactionsAs for Bacteria we utilized 2,989 interactions of B. anthracis, 1,347 interactions of F. tularensis and 3,961 interactions of Y. pestis with human host proteins36. Utilizing the IntAct database37, we collected 330 interactions of the Epstein-Barr virus, 557 interactions of the HIV virus, 766 interactions of the Hepatitis virus, 332 interactions of the Herpes virus, 756 interactions of the Influenza virus, 238 interactions from the Papilloma virus and 177 interactions of the Vaccinia virus with human host proteins.

Bacteria-host and Virus-host protein interactions

As for Bacteria we utilized 2,989 interactions of B. anthracis, 1,347 interactions of F. tularensis and 3,961 interactions of Y. pestis with human host proteins36. Utilizing the IntAct database37, we collected 330 interactions of the Epstein-Barr virus, 557 interactions of the HIV virus, 766 interactions of the Hepatitis virus, 332 interactions of the Herpes virus, 756 interactions of the Influenza virus, 238 interactions from the Papilloma virus and 177 interactions of the Vaccinia virus with human host proteins.

Human protein complexesWe utilized 1,843 protein complexes in H. sapiens from the CORUM database89, that collects information about experimentally determined protein complexes from the literature.

Human protein complexes

We utilized 1,843 protein complexes in H. sapiens from the CORUM database89, that collects information about experimentally determined protein complexes from the literature.

Enrichment analysis as a function of degreeWe grouped human proteins according to their number of interactions in an underlying human protein interaction network. We represented each group by N≥k proteins that had at least k interactions and calculated the number of targeted proteins i, Ni,≥k in each group. Randomly picking targeted genes we defined  as their enrichment where was the corresponding random number of targeted proteins among all Ni,≥k proteins. After averaging Ei over 10,000 randomizations Ei > 1 pointed to an enrichment and vice versa, while Ei ~ 1 indicated a random process90.

Enrichment analysis as a function of degree

We grouped human proteins according to their number of interactions in an underlying human protein interaction network. We represented each group by N≥k proteins that had at least k interactions and calculated the number of targeted proteins i, Ni,≥k in each group. Randomly picking targeted genes we defined  as their enrichment where was the corresponding random number of targeted proteins among all Ni,≥k proteins. After averaging Ei over 10,000 randomizations Ei > 1 pointed to an enrichment and vice versa, while Ei ~ 1 indicated a random process90.

The experiments were performed by S.B., S.A., G.S and M.A.S. Experiments were designed and conceived by S.B., M.K., P.U., T.S. and P.A. Data was analyzed by S.B., R.H., A.C., S.W. and F.S. The paper was written by S.B., R.H., P.U., S.W. and M.K.

Supplementary MaterialSupplementary InformationSupplementary Tables S1-S3

Supplementary Material

Supplementary Information

Supplementary Tables S1-S3
