---
pmid: '25473596'
title: Comprehensive analysis of interacting proteins and genome-wide location studies
  of the Sas3-dependent NuA3 histone acetyltransferase complex.
authors:
- Vicente-Muñoz S
- Romero P
- Magraner-Pardo L
- Martinez-Jimenez CP
- Tordera V
- Pamblanco M
journal: FEBS Open Bio
year: '2014'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC4248121
doi: 10.1016/j.fob.2014.11.001
---

# Comprehensive analysis of interacting proteins and genome-wide location studies of the Sas3-dependent NuA3 histone acetyltransferase complex.
**Authors:** Vicente-Muñoz S, Romero P, Magraner-Pardo L, Martinez-Jimenez CP, Tordera V, Pamblanco M
**Journal:** FEBS Open Bio (2014)
**DOI:** [10.1016/j.fob.2014.11.001](https://doi.org/10.1016/j.fob.2014.11.001)
**PMC:** [PMC4248121](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4248121/)

## Abstract

1. FEBS Open Bio. 2014 Nov 8;4:996-1006. doi: 10.1016/j.fob.2014.11.001. 
eCollection 2014.

Comprehensive analysis of interacting proteins and genome-wide location studies 
of the Sas3-dependent NuA3 histone acetyltransferase complex.

Vicente-Muñoz S(1), Romero P(2), Magraner-Pardo L(2), Martinez-Jimenez CP(2), 
Tordera V(2), Pamblanco M(2).

Author information:
(1)Structural Biochemistry Laboratory, Centro de Investigación Príncipe Felipe 
(CIPF), Eduardo Primo Yúfera, 3, 46012 València, Spain.
(2)Departament de Bioquímica i Biologia Molecular, Universitat de València, 
C/Dr. Moliner 50, 46100 Burjassot, València, Spain.

Histone acetylation affects several aspects of gene regulation, from chromatin 
remodelling to gene expression, by modulating the interplay between chromatin 
and key transcriptional regulators. The exact molecular mechanism underlying 
acetylation patterns and crosstalk with other epigenetic modifications requires 
further investigation. In budding yeast, these epigenetic markers are produced 
partly by histone acetyltransferase enzymes, which act as multi-protein 
complexes. The Sas3-dependent NuA3 complex has received less attention than 
other histone acetyltransferases (HAT), such as Gcn5-dependent complexes. Here, 
we report our analysis of Sas3p-interacting proteins using tandem affinity 
purification (TAP), coupled with mass spectrometry. This analysis revealed 
Pdp3p, a recently described component of NuA3, to be one of the most abundant 
Sas3p-interacting proteins. The PDP3 gene, was TAP-tagged and protein complex 
purification confirmed that Pdp3p co-purified with the NuA3 protein complex, 
histones, and several transcription-related and chromatin remodelling proteins. 
Our results also revealed that the protein complexes associated with Sas3p 
presented HAT activity even in the absence of Gcn5p and vice versa. We also 
provide evidence that Sas3p cannot substitute Gcn5p in acetylation of lysine 9 
in histone H3 in vivo. Genome-wide occupancy of Sas3p using ChIP-on-chip tiled 
microarrays showed that Sas3p was located preferentially within the 5'-half of 
the coding regions of target genes, indicating its probable involvement in the 
transcriptional elongation process. Hence, this work further characterises the 
function and regulation of the NuA3 complex by identifying novel 
post-translational modifications in Pdp3p, additional Pdp3p-co-purifying 
chromatin regulatory proteins involved in chromatin-modifying complex dynamics 
and gene regulation, and a subset of genes whose transcriptional elongation is 
controlled by this complex.

DOI: 10.1016/j.fob.2014.11.001
PMCID: PMC4248121
PMID: 25473596

## Full Text

1IntroductionMany regulatory processes of eukaryotic organisms are determined not only by the genome sequence, but also by epigenetic information. Epigenetic modifications generate heritable alterations in gene expression, although the primary nucleotide sequence remains unaltered [1–3]. Major epigenetic events include the methylation of DNA bases and post-translational modifications (PTMs) of histones. DNA methylation has been frequently described as an epigenetic modification related to gene silencing [4]. Meanwhile, histone PTMs typically contribute to the regulation of chromatin-related processes, modulating either directly the chromatin packing by altering the net charge of the histones [5] or indirectly, by regulating the DNA accessibility through nucleosome remodelling. The presence or absence of particular PTMs are likely to determine the sequential recruitment of the PTM enzymes or protein complexes implicated in gene regulation [6–9]. Therefore histone modifications are important in coordinating epigenetic crosstalk. Several types of enzymatic activities are involved in histone modifications [10], of which histone acetyltransferases (HATs) are the best-studied group [11,12]. HAT enzymes acetylate specific lysine residues located mainly on the amino terminal tail of histones [13–16] thereby participating in the recruitment of trans-acting regulatory factors [6–9].Eleven different proteins exhibiting HAT activity, at least in vitro, have been described to date in the Saccharomyces Genome Database (SGD, www.yeastgenome.org): Elp3p/Hpa1p, Esa1p, Gcn5p, Hat1p, Hpa2p, Hpa3p, Nut1p, Rtt109, Sas2p, Sas3p and Taf1p. Most of these enzymes have physical and functional connections with transcriptional regulation and belong to complexes composed of several subunits, which can modulate their activities. The fact that some of these proteins work as a catalytic subunit of more than one HAT complex, such as Gcn5p, which is a component of SAGA, SLIK/SALSA, ADA and HAT-A2 HAT yeast complexes [11], might explain the diversity of their functions. Histone H3 acetylation at lysines 9 (H3K9) and 14 (H3K14) strongly correlates with transcriptional activity and peaks beyond the transcription start site (TSS) of active genes [17]. Studies suggest that Gcn5p is the HAT responsible for most of these acetylation events in vivo
[18–20], which is in accordance with observations that Gcn5p is generally recruited to the promoters of active genes, as previously described for H3K9 and H3K14 acetylation marks [17,21]. Gcn5p also regulates replication-coupled nucleosome activity partly by promoting histone H3 association with CAF-1 (Chromatin Association Factor) via H3 acetylation [22]. Two other HATs also specifically acetylate H3K9 and H3K14 in vivo: the MYST family HAT Sas3p (Something About Silencing protein) and Elp3p (Elongator protein) [18,20]. Sas3p is the catalytic subunit of the NuA3-HAT complex [23] which also contains Eaf6p, Taf14p, Nto1p and Yng1p. Recruitment of NuA3 to nucleosomes in vivo requires the methylation of H3K4 and H3K36, which are catalysed by the Set1p and the Set2p histone methyltransferases, respectively [24]. The function of Sas3p appears to be critical in the absence of Gcn5p [19]. More specifically, deletion of SAS3 in WT cells does not elicit obvious phenotypes, however diminished Sas3p activity in a gcn5Δ null mutant results in cell cycle progression defects, and the complete loss of SAS3 function in gcn5Δ cells leads to cell death [19]. In addition, genome-wide mapping studies have established that Sas3p and Gcn5p are recruited to many of the same actively transcribed genes and that the binding sites of both HATs correlate with H3K14 acetylation [25]. Furthermore, genome-wide mapping approaches suggest that the SAGA-component Gcn5p [26–28] and Yng1p, the histone-binding subunit of NuA3 complex [29], are involved in transcriptional elongation, since their as both has been shown to bind to the coding region of target genes. These observations strongly suggest that the Sas3p and Gcn5p histone acetyltransferases are critical for active transcription, although the molecular mechanisms underlying their regulation have not yet been fully elucidated. In particular, since mutations of the above-mentioned histone H3 lysines result in only mild phenotypes [30,31], the essential function revealed by gcn5Δsas3Δ mutant lethality most likely goes beyond H3 acetylation. Interestingly, in the gcn5 and sas3 histone acetyltransferase double mutant, the inactivation of the chromatin remodelling complex Isw1a triggers the restoration of the RNAPII recruitment to active genes, which emphasise the importance of a fine-tuned balanced between chromatin modifying and remodelling activities for optimal transcription and cell growth [32]. Moreover, previously unanticipated differential roles for Sas3p and Gcn5p in DNA damage response and cell cycle regulation have been recently shown in relation to the β-1,3-glucanosyltransferase Gas1p, which regulates transcriptional silencing independently of its activity on the cell wall [33].Although Gcn5p has been extensively studied, much less is known about Sas3p. To explore Sas3p functions, we designed a proteomic approach based on the tandem affinity purification (TAP) strategy coupled to a mass spectrometry (MS) analysis and additional genomic studies by ChIP-on-chip assays. TAP-tagged wild-type (WT) and deletion mutant strains were constructed, and in this work, we particularly focused on the purification and characterisation of the active HAT complexes associated with Sas3p and Gcn5p. Notably, we found novel physical associations involving NuA3, nucleosomal histones, Pdp3p, and a variety of proteins involved in chromatin dynamics. At the genomic level, the ChIP-on-chip analyses suggest a role for Sas3p in transcriptional elongation. Altogether, these results significantly improve our knowledge regarding the functionality and dynamic interactions of Sas3p in relation to the presence of Gcn5p. Finally, we also report the likelihood of a functionally relevant crosstalk between NuA3 and Pdp3p.

Introduction

Many regulatory processes of eukaryotic organisms are determined not only by the genome sequence, but also by epigenetic information. Epigenetic modifications generate heritable alterations in gene expression, although the primary nucleotide sequence remains unaltered [1–3]. Major epigenetic events include the methylation of DNA bases and post-translational modifications (PTMs) of histones. DNA methylation has been frequently described as an epigenetic modification related to gene silencing [4]. Meanwhile, histone PTMs typically contribute to the regulation of chromatin-related processes, modulating either directly the chromatin packing by altering the net charge of the histones [5] or indirectly, by regulating the DNA accessibility through nucleosome remodelling. The presence or absence of particular PTMs are likely to determine the sequential recruitment of the PTM enzymes or protein complexes implicated in gene regulation [6–9]. Therefore histone modifications are important in coordinating epigenetic crosstalk. Several types of enzymatic activities are involved in histone modifications [10], of which histone acetyltransferases (HATs) are the best-studied group [11,12]. HAT enzymes acetylate specific lysine residues located mainly on the amino terminal tail of histones [13–16] thereby participating in the recruitment of trans-acting regulatory factors [6–9].

Eleven different proteins exhibiting HAT activity, at least in vitro, have been described to date in the Saccharomyces Genome Database (SGD, www.yeastgenome.org): Elp3p/Hpa1p, Esa1p, Gcn5p, Hat1p, Hpa2p, Hpa3p, Nut1p, Rtt109, Sas2p, Sas3p and Taf1p. Most of these enzymes have physical and functional connections with transcriptional regulation and belong to complexes composed of several subunits, which can modulate their activities. The fact that some of these proteins work as a catalytic subunit of more than one HAT complex, such as Gcn5p, which is a component of SAGA, SLIK/SALSA, ADA and HAT-A2 HAT yeast complexes [11], might explain the diversity of their functions. Histone H3 acetylation at lysines 9 (H3K9) and 14 (H3K14) strongly correlates with transcriptional activity and peaks beyond the transcription start site (TSS) of active genes [17]. Studies suggest that Gcn5p is the HAT responsible for most of these acetylation events in vivo
[18–20], which is in accordance with observations that Gcn5p is generally recruited to the promoters of active genes, as previously described for H3K9 and H3K14 acetylation marks [17,21]. Gcn5p also regulates replication-coupled nucleosome activity partly by promoting histone H3 association with CAF-1 (Chromatin Association Factor) via H3 acetylation [22]. Two other HATs also specifically acetylate H3K9 and H3K14 in vivo: the MYST family HAT Sas3p (Something About Silencing protein) and Elp3p (Elongator protein) [18,20]. Sas3p is the catalytic subunit of the NuA3-HAT complex [23] which also contains Eaf6p, Taf14p, Nto1p and Yng1p. Recruitment of NuA3 to nucleosomes in vivo requires the methylation of H3K4 and H3K36, which are catalysed by the Set1p and the Set2p histone methyltransferases, respectively [24]. The function of Sas3p appears to be critical in the absence of Gcn5p [19]. More specifically, deletion of SAS3 in WT cells does not elicit obvious phenotypes, however diminished Sas3p activity in a gcn5Δ null mutant results in cell cycle progression defects, and the complete loss of SAS3 function in gcn5Δ cells leads to cell death [19]. In addition, genome-wide mapping studies have established that Sas3p and Gcn5p are recruited to many of the same actively transcribed genes and that the binding sites of both HATs correlate with H3K14 acetylation [25]. Furthermore, genome-wide mapping approaches suggest that the SAGA-component Gcn5p [26–28] and Yng1p, the histone-binding subunit of NuA3 complex [29], are involved in transcriptional elongation, since their as both has been shown to bind to the coding region of target genes. These observations strongly suggest that the Sas3p and Gcn5p histone acetyltransferases are critical for active transcription, although the molecular mechanisms underlying their regulation have not yet been fully elucidated. In particular, since mutations of the above-mentioned histone H3 lysines result in only mild phenotypes [30,31], the essential function revealed by gcn5Δsas3Δ mutant lethality most likely goes beyond H3 acetylation. Interestingly, in the gcn5 and sas3 histone acetyltransferase double mutant, the inactivation of the chromatin remodelling complex Isw1a triggers the restoration of the RNAPII recruitment to active genes, which emphasise the importance of a fine-tuned balanced between chromatin modifying and remodelling activities for optimal transcription and cell growth [32]. Moreover, previously unanticipated differential roles for Sas3p and Gcn5p in DNA damage response and cell cycle regulation have been recently shown in relation to the β-1,3-glucanosyltransferase Gas1p, which regulates transcriptional silencing independently of its activity on the cell wall [33].

Although Gcn5p has been extensively studied, much less is known about Sas3p. To explore Sas3p functions, we designed a proteomic approach based on the tandem affinity purification (TAP) strategy coupled to a mass spectrometry (MS) analysis and additional genomic studies by ChIP-on-chip assays. TAP-tagged wild-type (WT) and deletion mutant strains were constructed, and in this work, we particularly focused on the purification and characterisation of the active HAT complexes associated with Sas3p and Gcn5p. Notably, we found novel physical associations involving NuA3, nucleosomal histones, Pdp3p, and a variety of proteins involved in chromatin dynamics. At the genomic level, the ChIP-on-chip analyses suggest a role for Sas3p in transcriptional elongation. Altogether, these results significantly improve our knowledge regarding the functionality and dynamic interactions of Sas3p in relation to the presence of Gcn5p. Finally, we also report the likelihood of a functionally relevant crosstalk between NuA3 and Pdp3p.

2Results and discussion2.1Purification of Sas3p reveals its association with histones and Pdp3pSas3p is the HAT catalytic subunit of the NuA3 complex. Sas3p, as well as the extensively studied Gcn5p HAT, act on nucleosomal H3, specifically at K9 and K14 [34], as primary sites of acetylation [35]. To study how the absence of one of these two HATs might affect the composition and/or the acetylation activity of the HAT complexes, new strains were constructed. Firstly, we assessed whether the integration of the TAP tag into the genome affects the growth of yeast strains. As shown in Fig. 1, TAP-tagging or SAS3 gene deletion did not produce defective growth at either 30 °C or 37 °C when compared to the corresponding WT cells (BMA64-1A and BY4742, Fig. 1 upper and bottom panels, respectively). On the contrary, the growth defects observed for deleted or TAP-tagged GCN5 (Fig. 1) can be partially explained by previous observations that gcn5Δ mutant cells accumulate in the G2/M phase of the cell cycle [31]. Phenotypic growth defects in the absence of GCN5 depended on the strain background, and no drastic visual defects were observed in the BY4742 background if compared with BMA64-1A. As a control, HAT2-TAP hif1Δ was used as a temperature-sensitive strain (Fig. 1).Sas3p is involved not only in chromatin modification through histone acetylation, but also in chromatin silencing in the silent mating-type cassettes and telomeres. By means of different technical approaches, it has been proposed that 28 proteins physically interact with Sas3p (BIOGRID database). In the present study, SAS3 was TAP-tagged and purified to further explore these interactions. The cell extract preparation and purification for TAP assays were carried out under sufficiently gentle conditions to maintain the integrity of the protein complex. Purification of a non-TAP-tagged strain was used as a negative control to rule out non-specific interactions [36]. Moreover, the typically observed non-specific purifying proteins were also ruled out, such as abundant housekeeping enzymes, translational factors, ribosomal, and heat-shock proteins [37]. The MS identification of the SAS3-TAP co-purified proteins using a nano LC–MS/MS analysis, which was performed using the QSTAR-XL system, is shown in Table 1. Notably, besides the components of the NuA3 complex (Sas3p, Nto1p, Taf30/14p, Yng1p and Eaf6p) and core histones, one of the best represented proteins was Pdp3p (UniProtKB: Q06188), with a 66.8% protein sequence coverage and an emPAI score of 4.56 (Table 1). In this context, it should also be mentioned that the only minor differences were observed among the proteins identified in the extracts from the SAS3-TAP WT or gcn5Δ mutant cells. Thus, in the GCN5-deleted strain, we identified Rim1p, Sec39p and Sas2p (data not shown) instead of Phb2p, Yhz9p and Ubi3p, which were identified in WT SAS3-TAP cells (Table 1). To confirm the physical interactions of Pdp3p with the NuA3 complex components and histones, the PDP3 gene was TAP-tagged and the purified proteins were analysed by MS. First, a portion of the chromatography eluate was analysed by nano LC MALDI-TOF–TOF with the 4700 PA system, which enabled the identification of the four core nucleosomal histones, and only a few peptides corresponding to Taf14p, Yng1p and Eaf6p components of NuA3 (Table 1). Secondly, cell extracts were prepared from a larger number of harvested cells. Approximately 36 μg of TAP-purified proteins were obtained and resolved by SDS–polyacrylamide gel electrophoresis (Fig. 2A). Protein bands were gel-excised and analysed by LC–MS/MS, performed using a 5600 Triple TOF system (Fig. 2B). Protein identification of the major band (slice 1) was carried out by the Protein Pilot software using Paragon Algorithm (AB Sciex). The analysis revealed new PTMs in Pdp3p (methylation at K79 and K126, ubiquitination at K14 and K63, and phosphorylation at S148, S150, S161 and S173; Fig. 2C). Our results corroborated one of the two phosphorylation events described in the BIOGRID database (i.e. at S148). Additionally, analyses of slice 2 identified Eaf6p and the four core histones (Fig. 2B). A thorough PTM search identified several modifications in H3: K23ac, K56ac, K36me, K79me and tri-methylated K79 (Fig. 2C). It must be noted that the histone H3 modifications we identified have been previously linked to active chromatin and genome stability (BIOGRID data). A total of 381 proteins were identified in slices 3 + 4 + 5, including common background proteins such as abundant housekeeping enzymes, ribosomal and heat-shock proteins, and translational factors (data not shown). As shown in Fig. 2B, Pdp3p co-purified not only with the core histones and all the components of the NuA3 complex, but also with several proteins involved in chromatin modification or remodelling.The network displayed by the STRING representation [38] generated by 22 Pdp3-interacting proteins identified in our analysis (Fig. 3A) shows the increased interacting complexity compared to the network published on BIOGRID (Fig. 3B). Among the transcription-related proteins, we found Ino80p (chromatin remodelling in replication and transcription), Chd1p (a chromatin remodeler, that binds methylated histone), Elp3p (the HAT subunit of elongator complex) and Spt5p (RNAPII core binding, transcription elongation by RNAPII). In particular, the identification of Spt16p and Pob3p, protein components of the yFACT nucleosome assembly complex, and Rfc1p, according to others studies [39], provides insights into the mechanism of NuA3-associated transcription and chromatin regulation.Previous studies have reported that Sas3p, core histones, Spt16p and Pdp3p, are co-purified with Yta7p tagged with Protein A [40]. However, an I-DIRT (Isotopic-Differentiation of Interactions as Random or Targeted) analysis applied to YNG1-TAP cells indicated that Pdp3p was a contaminant of NuA3 purification and not a true component of the complex [29]. Pdp3p has recently been described as a nuclear PWWP domain-containing protein [41] that has been physically associated with Sas3p [37]. Additionally, Gilbert and co-workers [41] have recently reported two RNAPII components (Rpb2p and Rpb4p) as Pdp3-interacting proteins, in addition to the NuA3 core-associated proteins and histones. Notably, among the Pdp3-interacting proteins identified in our analyses, we found the RNAPII component Rpb1p (represented by 16 peptides, with a protein sequence coverage of 11.6%).In order to investigate whether the absence of Sas3p can affect the nature of the proteins associated with Gcn5p and/or its HAT activity, TAP–MS analyses were carried out with complexes purified from the GCN5-TAP WT and GCN5-TAP sas3Δ mutant strains. The LC–MS/MS analyses of a portion of eluates from the TAP purification showed that Gcn5p co-purifies with all the components of the SAGA complex, except Sgf11p, Sus1p (components of the deubiquitination module of SAGA) and Chd1p, a chromatin remodeler (Table 2). These three proteins and Ubp8p were not identified when Sas3p was absent. Interestingly, protein Tra1p, which functions as a recruitment module for SAGA and is also a member of the NuA4 complex, was the most abundant protein in the GCN5-TAP purification from WT cells and one of the minor proteins recovered in the sas3Δ mutant. In both strains, Ahc1p, a component of the Ada2/Gcn5/Ada3 transcription activation complex [42], was found among the less representative proteins.Overall, our results provide a comprehensive list of physically interacting proteins, thus complementing previous data from low-throughput studies [40,41] and high-throughput studies compiled in the BIOGRID database. The protein network generated in the present work connects Pdp3p with a variety of chromatin-related proteins. These data suggest that novel mechanisms could be involved in the increase in Pdp3p in response to DNA replication stress [43] and its relocation to the cytosol in response to hypoxia [44] reported previously. Therefore, further research is needed to elucidate whether these proteins, Pdp3p and Sas3p, as well as another component of NuA3, together with the core of histones, are functionally linked.2.2The TAP-purified Sas3p complex acetylates nucleosomal histone H3Sas3p and Gcn5p are HAT enzymes that belong to the NuA3 and the SAGA complex, respectively. Both HAT complexes acetylate nucleosomal histones including H3K9 and H3K14 among other residues [34]. The protein complexes associated with Sas3p and purified by tandem affinity chromatography acetylated histones H4, and preferentially H3, when assayed with chicken erythrocyte oligonucleosomes in the presence or absence of Gcn5p and vice versa (Fig. 4). Interestingly, the activity observed for the NuA3 complex purified from the sas3Δ mutant, and also for the SAGA complex purified from the gcn5Δ mutant, was stronger in comparison to the WT. However, we cannot rule out that these differences might be due to a larger quantity of enzyme in the sample. In previous studies, by using the TAP strategy described herein, we were able to purify the HAT B complex that presented Hat1 activity on free histone H4 (results not shown). Thus, we can conclude that the experimental conditions we used to purify HAT complexes are sufficiently gentle to preserve their enzymatic activity.2.3The absence of Sas3p or Gcn5p has different effects on the in vivo acetylation levels of H3 at K9 and K14It is well-accepted that H3K14 is the main in vitro target for both histone acetyltransferases Sas3p and Gcn5p, although both proteins also acetylate H3K9. It has been reported that these histone modifications play an important role in gene transcriptional activation [24–29]. Fig. 5 shows the results obtained from the Western blot analyses of the total protein extracts employed to assess the in vivo acetylation levels of histone H3 at K14 and K9. Compared to the WT cells, loss of SAS3 or GCN5 did not significantly affect the total histone H3 levels (αH3 C-terminal–antibody). Strikingly, the H3K9 acetylation was practically absent in gcn5Δ cells, but remained virtually unaltered in sas3Δ cells. This shows that Sas3p cannot replace the acetylating function of Gcn5p on H3K9. However, the H3K14 acetylation levels were similar to the WT cells in both mutants. These data suggest that both Sas3p and Gcn5p contribute to reciprocally restore the H3K14ac level, because no change was detected when either of them was absent. Together, these data suggest that Sas3p has a similar but non-redundant function to Gcn5p (Fig. 5).2.4Sas3 preferentially binds to the 5′-half of the coding region of target genesIn a previous genome-wide study [25], we demonstrated that Sas3p, like Gcn5p, was recruited to a pool of intensely transcribed genes, and that a considerable overlap was observed between the two cohorts of the Sas3p- and Gcn5p-bound gene pools. However, because the arrays only had a single probe for each gene, it was not possible to determine the location of Sas3p in relation to the ORF. Here we investigated the genome-wide occupancy of Sas3p by ChIP-on-chip using tiled microarrays (244K, Agilent) with an average probe spatial resolution of 50 nt. Use of this technique showed that H3K14 (and H3K9) acetylation is enriched in promoter regions and the TSS of active genes [17,45]. Because Sas3p is a HAT, whose main target in vitro is H3K14, we expected to detect the majority of Sas3p binding to the promoter regions of genes. Surprisingly, we found that Sas3p was associated with the coding regions of genes, with peak enrichment 400 bp from the TSS within the ORF, and that this enrichment dropped substantially towards the 3′ region of the ORF (Fig. 6A). This result is similar to that obtained for Yng1p genome-wide occupancy [29], which also belongs to the NuA3 complex, and suggests that this complex might be involved in transcriptional elongation, at least in an initial step of the process. We employed a graphical representation (real-length representation), where the Sas3p location profiles were centred on the actual TSS [46]. The data for each probe corresponds to their real distance from the TSS for each gene up to the total transcript length, with a maximum of 3000 bp. A similar result was obtained with a graphical representation of the genome-wide data using a metagene profile, where genes were expanded or compressed to fit onto a hypothetical gene of average length (data not shown). Next we analysed whether the positive correlation observed between Sas3p binding and transcriptional activity [25] could be reproduced at a higher level when studying the Sas3p association with the coding regions of the most highly transcribed genes. Fig. 6B shows how the relative Sas3p binding level rises for a set of genes grouped according to the reported presence of RNA polymerase II (RNAPII) [47] (up to 25%, from 25% to 75% and over 75%) in the body of the genes. Moreover, the main correlation between the presence of both RNAPII and Sas3p proteins was observed, not only within the first 1000 bp from the TSS, but also upstream of the coding region. Thus, these results provide new evidence for the involvement of Sas3p in transcriptional elongation, which is in agreement with previous reports [25] showing that Sas3p acts as a general activator of transcription. As both functions have also been described for Gcn5p, the results of the present work provide new evidence to support the hypothesis that Gcn5p and Sas3p share redundant functions [19,25].The genome-wide pattern of Sas3p binding to the genes observed in Fig. 6A might be due to general non-specific Sas3p binding or may result from high Sas3p levels in specific classes of genes, which together form the observed pattern. To investigate this, we analysed the binding pattern of Sas3p to genes. Of the 5603 genes showing probes in the ORF, 5450 had more than three probes inside the ORF. Of these, 2644 genes showed no trend (the type H genes in Fig. 7A). The remaining genes were classified into four clusters according to the Sas3p binding pattern. Cluster 1 included those genes where Sas3p tended to be located at the 5′ of the ORF (936 type L + N genes in Fig. 7B), which, therefore, represents the group that best fits the profile obtained at the genomic level. Cluster 2 contains the genes where Sas3p tended to be located at the 3′ end of the ORF (706 type J + Z genes in Fig. 7C). Cluster 3 included the genes for which Sas3p tended to be located at both the 5′ and 3′ ends of the ORF (405 type U genes in Fig. 7D). Finally, cluster 4 contains the genes for which Sas3p tended to be located preferentially at the central region of the ORF (759 type A + S + X genes in Fig. 7E). Interestingly, a gene ontology (GO) analysis, which was performed separately on the genes belonging to the four clusters, revealed that only the genes included in cluster 1 were statistically grouped into GO categories (Table 3). Among the most statistically significant processes, the over-representation of the intracellular organelle-related categories is noteworthy. Taken together, these findings strongly suggest that Sas3p is present at the 5′ end of the ORF of a set of approximately 1000 genes involved in essential cellular functions, such as organelle biosynthesis and the regulation of biological processes.In summary, we have performed proteomic and genomic approaches to provide relevant insight to further extend our knowledge on the NuA3 histone acetyltransferase complex and its role in transcriptional regulation. We studied the crosstalk between Sas3p and Gcn5p, characterised the proteins that co-purified with Sas3p in active HAT complexes and studied its dependency on the presence or absence of Gcn5p. Our work provides experimental-based evidence for the Pdp3 protein–protein interaction in the NuA3 HAT complex, characterising its post-translational modifications, to broaden its protein network among chromatin interacting proteins. Complementarily, the high throughput genomic analyses indicated that Sas3p binding takes place preferentially on the 5′ end of the ORF, mostly in genes involved in intracellular organelle-related processes.

Results and discussion

2.1Purification of Sas3p reveals its association with histones and Pdp3pSas3p is the HAT catalytic subunit of the NuA3 complex. Sas3p, as well as the extensively studied Gcn5p HAT, act on nucleosomal H3, specifically at K9 and K14 [34], as primary sites of acetylation [35]. To study how the absence of one of these two HATs might affect the composition and/or the acetylation activity of the HAT complexes, new strains were constructed. Firstly, we assessed whether the integration of the TAP tag into the genome affects the growth of yeast strains. As shown in Fig. 1, TAP-tagging or SAS3 gene deletion did not produce defective growth at either 30 °C or 37 °C when compared to the corresponding WT cells (BMA64-1A and BY4742, Fig. 1 upper and bottom panels, respectively). On the contrary, the growth defects observed for deleted or TAP-tagged GCN5 (Fig. 1) can be partially explained by previous observations that gcn5Δ mutant cells accumulate in the G2/M phase of the cell cycle [31]. Phenotypic growth defects in the absence of GCN5 depended on the strain background, and no drastic visual defects were observed in the BY4742 background if compared with BMA64-1A. As a control, HAT2-TAP hif1Δ was used as a temperature-sensitive strain (Fig. 1).Sas3p is involved not only in chromatin modification through histone acetylation, but also in chromatin silencing in the silent mating-type cassettes and telomeres. By means of different technical approaches, it has been proposed that 28 proteins physically interact with Sas3p (BIOGRID database). In the present study, SAS3 was TAP-tagged and purified to further explore these interactions. The cell extract preparation and purification for TAP assays were carried out under sufficiently gentle conditions to maintain the integrity of the protein complex. Purification of a non-TAP-tagged strain was used as a negative control to rule out non-specific interactions [36]. Moreover, the typically observed non-specific purifying proteins were also ruled out, such as abundant housekeeping enzymes, translational factors, ribosomal, and heat-shock proteins [37]. The MS identification of the SAS3-TAP co-purified proteins using a nano LC–MS/MS analysis, which was performed using the QSTAR-XL system, is shown in Table 1. Notably, besides the components of the NuA3 complex (Sas3p, Nto1p, Taf30/14p, Yng1p and Eaf6p) and core histones, one of the best represented proteins was Pdp3p (UniProtKB: Q06188), with a 66.8% protein sequence coverage and an emPAI score of 4.56 (Table 1). In this context, it should also be mentioned that the only minor differences were observed among the proteins identified in the extracts from the SAS3-TAP WT or gcn5Δ mutant cells. Thus, in the GCN5-deleted strain, we identified Rim1p, Sec39p and Sas2p (data not shown) instead of Phb2p, Yhz9p and Ubi3p, which were identified in WT SAS3-TAP cells (Table 1). To confirm the physical interactions of Pdp3p with the NuA3 complex components and histones, the PDP3 gene was TAP-tagged and the purified proteins were analysed by MS. First, a portion of the chromatography eluate was analysed by nano LC MALDI-TOF–TOF with the 4700 PA system, which enabled the identification of the four core nucleosomal histones, and only a few peptides corresponding to Taf14p, Yng1p and Eaf6p components of NuA3 (Table 1). Secondly, cell extracts were prepared from a larger number of harvested cells. Approximately 36 μg of TAP-purified proteins were obtained and resolved by SDS–polyacrylamide gel electrophoresis (Fig. 2A). Protein bands were gel-excised and analysed by LC–MS/MS, performed using a 5600 Triple TOF system (Fig. 2B). Protein identification of the major band (slice 1) was carried out by the Protein Pilot software using Paragon Algorithm (AB Sciex). The analysis revealed new PTMs in Pdp3p (methylation at K79 and K126, ubiquitination at K14 and K63, and phosphorylation at S148, S150, S161 and S173; Fig. 2C). Our results corroborated one of the two phosphorylation events described in the BIOGRID database (i.e. at S148). Additionally, analyses of slice 2 identified Eaf6p and the four core histones (Fig. 2B). A thorough PTM search identified several modifications in H3: K23ac, K56ac, K36me, K79me and tri-methylated K79 (Fig. 2C). It must be noted that the histone H3 modifications we identified have been previously linked to active chromatin and genome stability (BIOGRID data). A total of 381 proteins were identified in slices 3 + 4 + 5, including common background proteins such as abundant housekeeping enzymes, ribosomal and heat-shock proteins, and translational factors (data not shown). As shown in Fig. 2B, Pdp3p co-purified not only with the core histones and all the components of the NuA3 complex, but also with several proteins involved in chromatin modification or remodelling.The network displayed by the STRING representation [38] generated by 22 Pdp3-interacting proteins identified in our analysis (Fig. 3A) shows the increased interacting complexity compared to the network published on BIOGRID (Fig. 3B). Among the transcription-related proteins, we found Ino80p (chromatin remodelling in replication and transcription), Chd1p (a chromatin remodeler, that binds methylated histone), Elp3p (the HAT subunit of elongator complex) and Spt5p (RNAPII core binding, transcription elongation by RNAPII). In particular, the identification of Spt16p and Pob3p, protein components of the yFACT nucleosome assembly complex, and Rfc1p, according to others studies [39], provides insights into the mechanism of NuA3-associated transcription and chromatin regulation.Previous studies have reported that Sas3p, core histones, Spt16p and Pdp3p, are co-purified with Yta7p tagged with Protein A [40]. However, an I-DIRT (Isotopic-Differentiation of Interactions as Random or Targeted) analysis applied to YNG1-TAP cells indicated that Pdp3p was a contaminant of NuA3 purification and not a true component of the complex [29]. Pdp3p has recently been described as a nuclear PWWP domain-containing protein [41] that has been physically associated with Sas3p [37]. Additionally, Gilbert and co-workers [41] have recently reported two RNAPII components (Rpb2p and Rpb4p) as Pdp3-interacting proteins, in addition to the NuA3 core-associated proteins and histones. Notably, among the Pdp3-interacting proteins identified in our analyses, we found the RNAPII component Rpb1p (represented by 16 peptides, with a protein sequence coverage of 11.6%).In order to investigate whether the absence of Sas3p can affect the nature of the proteins associated with Gcn5p and/or its HAT activity, TAP–MS analyses were carried out with complexes purified from the GCN5-TAP WT and GCN5-TAP sas3Δ mutant strains. The LC–MS/MS analyses of a portion of eluates from the TAP purification showed that Gcn5p co-purifies with all the components of the SAGA complex, except Sgf11p, Sus1p (components of the deubiquitination module of SAGA) and Chd1p, a chromatin remodeler (Table 2). These three proteins and Ubp8p were not identified when Sas3p was absent. Interestingly, protein Tra1p, which functions as a recruitment module for SAGA and is also a member of the NuA4 complex, was the most abundant protein in the GCN5-TAP purification from WT cells and one of the minor proteins recovered in the sas3Δ mutant. In both strains, Ahc1p, a component of the Ada2/Gcn5/Ada3 transcription activation complex [42], was found among the less representative proteins.Overall, our results provide a comprehensive list of physically interacting proteins, thus complementing previous data from low-throughput studies [40,41] and high-throughput studies compiled in the BIOGRID database. The protein network generated in the present work connects Pdp3p with a variety of chromatin-related proteins. These data suggest that novel mechanisms could be involved in the increase in Pdp3p in response to DNA replication stress [43] and its relocation to the cytosol in response to hypoxia [44] reported previously. Therefore, further research is needed to elucidate whether these proteins, Pdp3p and Sas3p, as well as another component of NuA3, together with the core of histones, are functionally linked.

Purification of Sas3p reveals its association with histones and Pdp3p

Sas3p is the HAT catalytic subunit of the NuA3 complex. Sas3p, as well as the extensively studied Gcn5p HAT, act on nucleosomal H3, specifically at K9 and K14 [34], as primary sites of acetylation [35]. To study how the absence of one of these two HATs might affect the composition and/or the acetylation activity of the HAT complexes, new strains were constructed. Firstly, we assessed whether the integration of the TAP tag into the genome affects the growth of yeast strains. As shown in Fig. 1, TAP-tagging or SAS3 gene deletion did not produce defective growth at either 30 °C or 37 °C when compared to the corresponding WT cells (BMA64-1A and BY4742, Fig. 1 upper and bottom panels, respectively). On the contrary, the growth defects observed for deleted or TAP-tagged GCN5 (Fig. 1) can be partially explained by previous observations that gcn5Δ mutant cells accumulate in the G2/M phase of the cell cycle [31]. Phenotypic growth defects in the absence of GCN5 depended on the strain background, and no drastic visual defects were observed in the BY4742 background if compared with BMA64-1A. As a control, HAT2-TAP hif1Δ was used as a temperature-sensitive strain (Fig. 1).

Sas3p is involved not only in chromatin modification through histone acetylation, but also in chromatin silencing in the silent mating-type cassettes and telomeres. By means of different technical approaches, it has been proposed that 28 proteins physically interact with Sas3p (BIOGRID database). In the present study, SAS3 was TAP-tagged and purified to further explore these interactions. The cell extract preparation and purification for TAP assays were carried out under sufficiently gentle conditions to maintain the integrity of the protein complex. Purification of a non-TAP-tagged strain was used as a negative control to rule out non-specific interactions [36]. Moreover, the typically observed non-specific purifying proteins were also ruled out, such as abundant housekeeping enzymes, translational factors, ribosomal, and heat-shock proteins [37]. The MS identification of the SAS3-TAP co-purified proteins using a nano LC–MS/MS analysis, which was performed using the QSTAR-XL system, is shown in Table 1. Notably, besides the components of the NuA3 complex (Sas3p, Nto1p, Taf30/14p, Yng1p and Eaf6p) and core histones, one of the best represented proteins was Pdp3p (UniProtKB: Q06188), with a 66.8% protein sequence coverage and an emPAI score of 4.56 (Table 1). In this context, it should also be mentioned that the only minor differences were observed among the proteins identified in the extracts from the SAS3-TAP WT or gcn5Δ mutant cells. Thus, in the GCN5-deleted strain, we identified Rim1p, Sec39p and Sas2p (data not shown) instead of Phb2p, Yhz9p and Ubi3p, which were identified in WT SAS3-TAP cells (Table 1). To confirm the physical interactions of Pdp3p with the NuA3 complex components and histones, the PDP3 gene was TAP-tagged and the purified proteins were analysed by MS. First, a portion of the chromatography eluate was analysed by nano LC MALDI-TOF–TOF with the 4700 PA system, which enabled the identification of the four core nucleosomal histones, and only a few peptides corresponding to Taf14p, Yng1p and Eaf6p components of NuA3 (Table 1). Secondly, cell extracts were prepared from a larger number of harvested cells. Approximately 36 μg of TAP-purified proteins were obtained and resolved by SDS–polyacrylamide gel electrophoresis (Fig. 2A). Protein bands were gel-excised and analysed by LC–MS/MS, performed using a 5600 Triple TOF system (Fig. 2B). Protein identification of the major band (slice 1) was carried out by the Protein Pilot software using Paragon Algorithm (AB Sciex). The analysis revealed new PTMs in Pdp3p (methylation at K79 and K126, ubiquitination at K14 and K63, and phosphorylation at S148, S150, S161 and S173; Fig. 2C). Our results corroborated one of the two phosphorylation events described in the BIOGRID database (i.e. at S148). Additionally, analyses of slice 2 identified Eaf6p and the four core histones (Fig. 2B). A thorough PTM search identified several modifications in H3: K23ac, K56ac, K36me, K79me and tri-methylated K79 (Fig. 2C). It must be noted that the histone H3 modifications we identified have been previously linked to active chromatin and genome stability (BIOGRID data). A total of 381 proteins were identified in slices 3 + 4 + 5, including common background proteins such as abundant housekeeping enzymes, ribosomal and heat-shock proteins, and translational factors (data not shown). As shown in Fig. 2B, Pdp3p co-purified not only with the core histones and all the components of the NuA3 complex, but also with several proteins involved in chromatin modification or remodelling.

The network displayed by the STRING representation [38] generated by 22 Pdp3-interacting proteins identified in our analysis (Fig. 3A) shows the increased interacting complexity compared to the network published on BIOGRID (Fig. 3B). Among the transcription-related proteins, we found Ino80p (chromatin remodelling in replication and transcription), Chd1p (a chromatin remodeler, that binds methylated histone), Elp3p (the HAT subunit of elongator complex) and Spt5p (RNAPII core binding, transcription elongation by RNAPII). In particular, the identification of Spt16p and Pob3p, protein components of the yFACT nucleosome assembly complex, and Rfc1p, according to others studies [39], provides insights into the mechanism of NuA3-associated transcription and chromatin regulation.

Previous studies have reported that Sas3p, core histones, Spt16p and Pdp3p, are co-purified with Yta7p tagged with Protein A [40]. However, an I-DIRT (Isotopic-Differentiation of Interactions as Random or Targeted) analysis applied to YNG1-TAP cells indicated that Pdp3p was a contaminant of NuA3 purification and not a true component of the complex [29]. Pdp3p has recently been described as a nuclear PWWP domain-containing protein [41] that has been physically associated with Sas3p [37]. Additionally, Gilbert and co-workers [41] have recently reported two RNAPII components (Rpb2p and Rpb4p) as Pdp3-interacting proteins, in addition to the NuA3 core-associated proteins and histones. Notably, among the Pdp3-interacting proteins identified in our analyses, we found the RNAPII component Rpb1p (represented by 16 peptides, with a protein sequence coverage of 11.6%).

In order to investigate whether the absence of Sas3p can affect the nature of the proteins associated with Gcn5p and/or its HAT activity, TAP–MS analyses were carried out with complexes purified from the GCN5-TAP WT and GCN5-TAP sas3Δ mutant strains. The LC–MS/MS analyses of a portion of eluates from the TAP purification showed that Gcn5p co-purifies with all the components of the SAGA complex, except Sgf11p, Sus1p (components of the deubiquitination module of SAGA) and Chd1p, a chromatin remodeler (Table 2). These three proteins and Ubp8p were not identified when Sas3p was absent. Interestingly, protein Tra1p, which functions as a recruitment module for SAGA and is also a member of the NuA4 complex, was the most abundant protein in the GCN5-TAP purification from WT cells and one of the minor proteins recovered in the sas3Δ mutant. In both strains, Ahc1p, a component of the Ada2/Gcn5/Ada3 transcription activation complex [42], was found among the less representative proteins.

Overall, our results provide a comprehensive list of physically interacting proteins, thus complementing previous data from low-throughput studies [40,41] and high-throughput studies compiled in the BIOGRID database. The protein network generated in the present work connects Pdp3p with a variety of chromatin-related proteins. These data suggest that novel mechanisms could be involved in the increase in Pdp3p in response to DNA replication stress [43] and its relocation to the cytosol in response to hypoxia [44] reported previously. Therefore, further research is needed to elucidate whether these proteins, Pdp3p and Sas3p, as well as another component of NuA3, together with the core of histones, are functionally linked.

2.2The TAP-purified Sas3p complex acetylates nucleosomal histone H3Sas3p and Gcn5p are HAT enzymes that belong to the NuA3 and the SAGA complex, respectively. Both HAT complexes acetylate nucleosomal histones including H3K9 and H3K14 among other residues [34]. The protein complexes associated with Sas3p and purified by tandem affinity chromatography acetylated histones H4, and preferentially H3, when assayed with chicken erythrocyte oligonucleosomes in the presence or absence of Gcn5p and vice versa (Fig. 4). Interestingly, the activity observed for the NuA3 complex purified from the sas3Δ mutant, and also for the SAGA complex purified from the gcn5Δ mutant, was stronger in comparison to the WT. However, we cannot rule out that these differences might be due to a larger quantity of enzyme in the sample. In previous studies, by using the TAP strategy described herein, we were able to purify the HAT B complex that presented Hat1 activity on free histone H4 (results not shown). Thus, we can conclude that the experimental conditions we used to purify HAT complexes are sufficiently gentle to preserve their enzymatic activity.

The TAP-purified Sas3p complex acetylates nucleosomal histone H3

Sas3p and Gcn5p are HAT enzymes that belong to the NuA3 and the SAGA complex, respectively. Both HAT complexes acetylate nucleosomal histones including H3K9 and H3K14 among other residues [34]. The protein complexes associated with Sas3p and purified by tandem affinity chromatography acetylated histones H4, and preferentially H3, when assayed with chicken erythrocyte oligonucleosomes in the presence or absence of Gcn5p and vice versa (Fig. 4). Interestingly, the activity observed for the NuA3 complex purified from the sas3Δ mutant, and also for the SAGA complex purified from the gcn5Δ mutant, was stronger in comparison to the WT. However, we cannot rule out that these differences might be due to a larger quantity of enzyme in the sample. In previous studies, by using the TAP strategy described herein, we were able to purify the HAT B complex that presented Hat1 activity on free histone H4 (results not shown). Thus, we can conclude that the experimental conditions we used to purify HAT complexes are sufficiently gentle to preserve their enzymatic activity.

2.3The absence of Sas3p or Gcn5p has different effects on the in vivo acetylation levels of H3 at K9 and K14It is well-accepted that H3K14 is the main in vitro target for both histone acetyltransferases Sas3p and Gcn5p, although both proteins also acetylate H3K9. It has been reported that these histone modifications play an important role in gene transcriptional activation [24–29]. Fig. 5 shows the results obtained from the Western blot analyses of the total protein extracts employed to assess the in vivo acetylation levels of histone H3 at K14 and K9. Compared to the WT cells, loss of SAS3 or GCN5 did not significantly affect the total histone H3 levels (αH3 C-terminal–antibody). Strikingly, the H3K9 acetylation was practically absent in gcn5Δ cells, but remained virtually unaltered in sas3Δ cells. This shows that Sas3p cannot replace the acetylating function of Gcn5p on H3K9. However, the H3K14 acetylation levels were similar to the WT cells in both mutants. These data suggest that both Sas3p and Gcn5p contribute to reciprocally restore the H3K14ac level, because no change was detected when either of them was absent. Together, these data suggest that Sas3p has a similar but non-redundant function to Gcn5p (Fig. 5).

The absence of Sas3p or Gcn5p has different effects on the in vivo acetylation levels of H3 at K9 and K14

It is well-accepted that H3K14 is the main in vitro target for both histone acetyltransferases Sas3p and Gcn5p, although both proteins also acetylate H3K9. It has been reported that these histone modifications play an important role in gene transcriptional activation [24–29]. Fig. 5 shows the results obtained from the Western blot analyses of the total protein extracts employed to assess the in vivo acetylation levels of histone H3 at K14 and K9. Compared to the WT cells, loss of SAS3 or GCN5 did not significantly affect the total histone H3 levels (αH3 C-terminal–antibody). Strikingly, the H3K9 acetylation was practically absent in gcn5Δ cells, but remained virtually unaltered in sas3Δ cells. This shows that Sas3p cannot replace the acetylating function of Gcn5p on H3K9. However, the H3K14 acetylation levels were similar to the WT cells in both mutants. These data suggest that both Sas3p and Gcn5p contribute to reciprocally restore the H3K14ac level, because no change was detected when either of them was absent. Together, these data suggest that Sas3p has a similar but non-redundant function to Gcn5p (Fig. 5).

2.4Sas3 preferentially binds to the 5′-half of the coding region of target genesIn a previous genome-wide study [25], we demonstrated that Sas3p, like Gcn5p, was recruited to a pool of intensely transcribed genes, and that a considerable overlap was observed between the two cohorts of the Sas3p- and Gcn5p-bound gene pools. However, because the arrays only had a single probe for each gene, it was not possible to determine the location of Sas3p in relation to the ORF. Here we investigated the genome-wide occupancy of Sas3p by ChIP-on-chip using tiled microarrays (244K, Agilent) with an average probe spatial resolution of 50 nt. Use of this technique showed that H3K14 (and H3K9) acetylation is enriched in promoter regions and the TSS of active genes [17,45]. Because Sas3p is a HAT, whose main target in vitro is H3K14, we expected to detect the majority of Sas3p binding to the promoter regions of genes. Surprisingly, we found that Sas3p was associated with the coding regions of genes, with peak enrichment 400 bp from the TSS within the ORF, and that this enrichment dropped substantially towards the 3′ region of the ORF (Fig. 6A). This result is similar to that obtained for Yng1p genome-wide occupancy [29], which also belongs to the NuA3 complex, and suggests that this complex might be involved in transcriptional elongation, at least in an initial step of the process. We employed a graphical representation (real-length representation), where the Sas3p location profiles were centred on the actual TSS [46]. The data for each probe corresponds to their real distance from the TSS for each gene up to the total transcript length, with a maximum of 3000 bp. A similar result was obtained with a graphical representation of the genome-wide data using a metagene profile, where genes were expanded or compressed to fit onto a hypothetical gene of average length (data not shown). Next we analysed whether the positive correlation observed between Sas3p binding and transcriptional activity [25] could be reproduced at a higher level when studying the Sas3p association with the coding regions of the most highly transcribed genes. Fig. 6B shows how the relative Sas3p binding level rises for a set of genes grouped according to the reported presence of RNA polymerase II (RNAPII) [47] (up to 25%, from 25% to 75% and over 75%) in the body of the genes. Moreover, the main correlation between the presence of both RNAPII and Sas3p proteins was observed, not only within the first 1000 bp from the TSS, but also upstream of the coding region. Thus, these results provide new evidence for the involvement of Sas3p in transcriptional elongation, which is in agreement with previous reports [25] showing that Sas3p acts as a general activator of transcription. As both functions have also been described for Gcn5p, the results of the present work provide new evidence to support the hypothesis that Gcn5p and Sas3p share redundant functions [19,25].The genome-wide pattern of Sas3p binding to the genes observed in Fig. 6A might be due to general non-specific Sas3p binding or may result from high Sas3p levels in specific classes of genes, which together form the observed pattern. To investigate this, we analysed the binding pattern of Sas3p to genes. Of the 5603 genes showing probes in the ORF, 5450 had more than three probes inside the ORF. Of these, 2644 genes showed no trend (the type H genes in Fig. 7A). The remaining genes were classified into four clusters according to the Sas3p binding pattern. Cluster 1 included those genes where Sas3p tended to be located at the 5′ of the ORF (936 type L + N genes in Fig. 7B), which, therefore, represents the group that best fits the profile obtained at the genomic level. Cluster 2 contains the genes where Sas3p tended to be located at the 3′ end of the ORF (706 type J + Z genes in Fig. 7C). Cluster 3 included the genes for which Sas3p tended to be located at both the 5′ and 3′ ends of the ORF (405 type U genes in Fig. 7D). Finally, cluster 4 contains the genes for which Sas3p tended to be located preferentially at the central region of the ORF (759 type A + S + X genes in Fig. 7E). Interestingly, a gene ontology (GO) analysis, which was performed separately on the genes belonging to the four clusters, revealed that only the genes included in cluster 1 were statistically grouped into GO categories (Table 3). Among the most statistically significant processes, the over-representation of the intracellular organelle-related categories is noteworthy. Taken together, these findings strongly suggest that Sas3p is present at the 5′ end of the ORF of a set of approximately 1000 genes involved in essential cellular functions, such as organelle biosynthesis and the regulation of biological processes.In summary, we have performed proteomic and genomic approaches to provide relevant insight to further extend our knowledge on the NuA3 histone acetyltransferase complex and its role in transcriptional regulation. We studied the crosstalk between Sas3p and Gcn5p, characterised the proteins that co-purified with Sas3p in active HAT complexes and studied its dependency on the presence or absence of Gcn5p. Our work provides experimental-based evidence for the Pdp3 protein–protein interaction in the NuA3 HAT complex, characterising its post-translational modifications, to broaden its protein network among chromatin interacting proteins. Complementarily, the high throughput genomic analyses indicated that Sas3p binding takes place preferentially on the 5′ end of the ORF, mostly in genes involved in intracellular organelle-related processes.

Sas3 preferentially binds to the 5′-half of the coding region of target genes

In a previous genome-wide study [25], we demonstrated that Sas3p, like Gcn5p, was recruited to a pool of intensely transcribed genes, and that a considerable overlap was observed between the two cohorts of the Sas3p- and Gcn5p-bound gene pools. However, because the arrays only had a single probe for each gene, it was not possible to determine the location of Sas3p in relation to the ORF. Here we investigated the genome-wide occupancy of Sas3p by ChIP-on-chip using tiled microarrays (244K, Agilent) with an average probe spatial resolution of 50 nt. Use of this technique showed that H3K14 (and H3K9) acetylation is enriched in promoter regions and the TSS of active genes [17,45]. Because Sas3p is a HAT, whose main target in vitro is H3K14, we expected to detect the majority of Sas3p binding to the promoter regions of genes. Surprisingly, we found that Sas3p was associated with the coding regions of genes, with peak enrichment 400 bp from the TSS within the ORF, and that this enrichment dropped substantially towards the 3′ region of the ORF (Fig. 6A). This result is similar to that obtained for Yng1p genome-wide occupancy [29], which also belongs to the NuA3 complex, and suggests that this complex might be involved in transcriptional elongation, at least in an initial step of the process. We employed a graphical representation (real-length representation), where the Sas3p location profiles were centred on the actual TSS [46]. The data for each probe corresponds to their real distance from the TSS for each gene up to the total transcript length, with a maximum of 3000 bp. A similar result was obtained with a graphical representation of the genome-wide data using a metagene profile, where genes were expanded or compressed to fit onto a hypothetical gene of average length (data not shown). Next we analysed whether the positive correlation observed between Sas3p binding and transcriptional activity [25] could be reproduced at a higher level when studying the Sas3p association with the coding regions of the most highly transcribed genes. Fig. 6B shows how the relative Sas3p binding level rises for a set of genes grouped according to the reported presence of RNA polymerase II (RNAPII) [47] (up to 25%, from 25% to 75% and over 75%) in the body of the genes. Moreover, the main correlation between the presence of both RNAPII and Sas3p proteins was observed, not only within the first 1000 bp from the TSS, but also upstream of the coding region. Thus, these results provide new evidence for the involvement of Sas3p in transcriptional elongation, which is in agreement with previous reports [25] showing that Sas3p acts as a general activator of transcription. As both functions have also been described for Gcn5p, the results of the present work provide new evidence to support the hypothesis that Gcn5p and Sas3p share redundant functions [19,25].

The genome-wide pattern of Sas3p binding to the genes observed in Fig. 6A might be due to general non-specific Sas3p binding or may result from high Sas3p levels in specific classes of genes, which together form the observed pattern. To investigate this, we analysed the binding pattern of Sas3p to genes. Of the 5603 genes showing probes in the ORF, 5450 had more than three probes inside the ORF. Of these, 2644 genes showed no trend (the type H genes in Fig. 7A). The remaining genes were classified into four clusters according to the Sas3p binding pattern. Cluster 1 included those genes where Sas3p tended to be located at the 5′ of the ORF (936 type L + N genes in Fig. 7B), which, therefore, represents the group that best fits the profile obtained at the genomic level. Cluster 2 contains the genes where Sas3p tended to be located at the 3′ end of the ORF (706 type J + Z genes in Fig. 7C). Cluster 3 included the genes for which Sas3p tended to be located at both the 5′ and 3′ ends of the ORF (405 type U genes in Fig. 7D). Finally, cluster 4 contains the genes for which Sas3p tended to be located preferentially at the central region of the ORF (759 type A + S + X genes in Fig. 7E). Interestingly, a gene ontology (GO) analysis, which was performed separately on the genes belonging to the four clusters, revealed that only the genes included in cluster 1 were statistically grouped into GO categories (Table 3). Among the most statistically significant processes, the over-representation of the intracellular organelle-related categories is noteworthy. Taken together, these findings strongly suggest that Sas3p is present at the 5′ end of the ORF of a set of approximately 1000 genes involved in essential cellular functions, such as organelle biosynthesis and the regulation of biological processes.

In summary, we have performed proteomic and genomic approaches to provide relevant insight to further extend our knowledge on the NuA3 histone acetyltransferase complex and its role in transcriptional regulation. We studied the crosstalk between Sas3p and Gcn5p, characterised the proteins that co-purified with Sas3p in active HAT complexes and studied its dependency on the presence or absence of Gcn5p. Our work provides experimental-based evidence for the Pdp3 protein–protein interaction in the NuA3 HAT complex, characterising its post-translational modifications, to broaden its protein network among chromatin interacting proteins. Complementarily, the high throughput genomic analyses indicated that Sas3p binding takes place preferentially on the 5′ end of the ORF, mostly in genes involved in intracellular organelle-related processes.

3Materials and methods3.1Yeast strains and culture mediaThe Saccharomyces cerevisiae yeast strains used in this study are listed in Table 4. Yeast cells were grown at 30 °C in yeast peptone dextrose (YPD) medium containing, or not, supplementary adenine. Epitope tags and gene replacements were introduced into the genome by PCR-mediated one-step gene replacement. The TAP-tagged yeast strains were constructed by gene targeting at the 3′ end of the ORF with a tag variant (CBP-T7-TEV-Protein A-KAN MX6) of the original TAP tag [48], which we constructed in B. Seraphin’s laboratory. Growth was assayed by spotting serial dilutions of fresh yeast cells exponentially cultured on YPDA and incubated at different temperatures.3.2Preparation of cell extracts for tandem affinity purification (TAP)At least 6 L of cultured cells grown to an absorbance of OD600 ≈ 2.0 were harvested, instantly frozen in liquid nitrogen and cryo-ground 6 times during a 3-min period at maximum speed using a MM301 ball mill (Retsch). Frozen powdered cells were transferred into 100-mL containers and were stored at −80 °C until used. Cell extracts were prepared using 2 mL/g of cells of a buffer 40 mM HEPES, pH 7.9, 350 mM NaCl, 10% glycerol and 0.1% Nonidet NP-40, containing protease inhibitors 2 mM benzamidine, 1 mM PMSF, 2 mg/mL leupeptine, 2 mg/mL pepstatin A, 2.4 mg/mL chymostatin and 10 μL/mL Trasylol (Bayer). All the affinity purification steps were carried out at 4 °C, basically as described elsewhere [49]. To increase the SAS3-TAP protein purification yield, instead of commercial resins, we used IgG- (I4506, Sigma) and Calmodulin (P1431, Sigma)-coated Dynabeads (M-280, Invitrogen), prepared following the manufacturer’s instructions. For PDP3-TAP, we used commercial resin IgG-Sepharose (17-0969-01, Amersham) and Calmodulin-coated Dynabeads. Two antibodies were used to detect the TAP-tag: P1291 Sigma and CAB1001 Open Biosystems, which respectively recognise the Protein A or the calmodulin-binding peptide regions of the TAP tag. TAP-purified protein complexes were either directly analysed by MS, or dried, and then solved on sample buffer and resolved in 10% SDS–PAGE. Gel was stained with Colloidal Blue (R0571 Fermentas) prior to the MS analysis.3.3In vitro histone acetyltransferase specificity assaysThe PhosphorImager HAT assay was performed as previously described [50], with minor modifications. HAT activities were determined over the whole chromatographic TAP-eluates (5 or 10 μL). Sample fractions were incubated for 20 min at 30 °C with chicken erythrocyte oligonucleosomes or free core histones used as a control and prepared as previously described [51] and with 0.005 μCi [1-14C]acetyl-CoA in a final volume of 16 μL. The reactions were finalised by adding 4 μL of 5× SDS sample loading buffer. The histones in the reaction mixtures were resolved by SDS–PAGE (16%). After staining/fixing and drying, gels were exposed to the BAS-SR IP phosphor-storage image plate (FujiFilm) for different times. Phosphor screens were scanned in a FujiFilm FLA-3000 fluorescent imager analyser.3.4Histone immunoblotting assaysFor the histone Western blots, cells were collected at OD600 ≈ 0.8. WCEs were prepared as described elsewhere [52] and were resolved by SDS–PAGE (16%). Protein gels were transferred to 0.2-μm pore nitrocellulose membranes (Protan, Whatman), as described elsewhere [53]. Membranes, after staining with Ponceau S and subsequent washes, were processed according to the ECL Advance Western Blotting detection system (RNP 2135, GE Healthcare) instructions. The primary antibodies were the following: α-H3Ct (ab1791, Abcam), α-H3K9ac (06-942, Millipore) and α-H3K14ac (07-353, Upstate), and were used at a 10-fold dilution as suggested by the manufacturer. The secondary antibody was horseradish peroxidase-linked anti-rabbit (GE Healthcare), employed at a dilution of 1/40,000.3.5ChIP-on-chip experimentsChIP-on-chip experiments were performed as previously described [45], but with some modifications. Briefly, the antibody used for the genome-wide location analysis was 2 μg of rat anti-HA 3F10 (Roche). The rat antibody was coupled to a suspension of 100 μL of 50% (v/v) Protein G Sepharose 4FF (Amersham Biosciences) equilibrated in lysis buffer (50 mM HEPES-KOH, pH 7.5, 140 mM NaCl, 1 mM EDTA, 1% Triton X-100, 0.1% sodium deoxycholate, 1 mM Phenylmethylsulfonyl fluoride, 1 mM benzamidine, and 1 pill of protease inhibitor cocktail (Roche) was dissolved in every 25 mL of buffer) supplemented with 5 mg/mL of BSA. Next, 2000 ng of DNA immunoprecipitated from the BQS1217 strain (SAS3-HA tagged) and DNA immunoprecipitated from the BY4742 strain (WT) were labelled with Cyanine 5-dUTP and Cyanine 3-dUTP, respectively, using the ‘CGH Labelling Kit’ (Invitrogen, p/n 18095-011) following the manufacturer’s instructions. Labelled DNA was hybridised to the Yeast Whole Genome ChIP-on-chip Microarray (Agilent p/n G4491A, AMADID: 014741), specifically designed for location analyses.Data were processed by ChIP Analytics, 1.3 (Agilent). Blank subtraction normalisation, and inter-array and intra-array median normalisation were performed. Probes were lifted over to the sacCer3 genome version (April 2011). Transcript boundaries [46] were downloaded from SGD (R64-1-1, http://www.yeastgenome.org). Raw and processed data were deposited in GEO: GSE56889. Plots were performed using R and Bioconductor [54]. Probes were binned to 200 bp, and the mean and its confident intervals (t-test, 95% confidence) were plotted. Experiments were carried out in duplicate.3.6Mass spectrometry analysisThe TAP-tagged co-purified proteins were treated and identified by LC–MS/MS at the 95% confidence level, as previously described [36,55]. Briefly, the proteins from the TAP-eluates were precipitated with TCA/acetone, solved and digested with trypsin. The resulting peptides were loaded into LC Packings Pep Map C18 columns and chromatographed by elution with a linear gradient of 5% to 45–50% of acetonitrile in 0.1% formic acid for 120 min. The peptides from the Sas3p sample were analysed in a nano ESI Q-TOF mass spectrometer (QSTAR XL, ABI) and those from the Pdp3p sample in a MALDI-TOF–TOF (4700 Proteomic Analyser, ABI). Searches were made in the Swiss-Prot database using the MASCOT and Protein Pilot search engines.Slices from the gel corresponding to PDP3-TAP were excised and digested with sequencing grade trypsin (Promega), as described elsewhere [56]. The digestion mixture was dried in a vacuum centrifuge, and dissolved in 2% ACN, 0.1% TFA (samples 1 and 2 in 10 μL; samples 4,5 and 6 in 10 μL). Then 5 μL of each sample were loaded into a trap column (Nano LC Column, 3 μ C18-CL, 75 um × 15 cm; Eksigen) and desalted with 0.1% TFA at 2 μL/min for 10 min. Peptides were loaded into an analytical column (LC Column, 3 μ C18-CL, 75 um × 25 cm, Eksigen), equilibrated in 5% acetonitrile 0.1% FA (formic acid). Peptide elution was carried out with a linear gradient of 5–40% B in 30 min (A: 0.1% FA; B: ACN, 0.1% FA) at a flow rate of 300 nL/min. Peptides were analysed in a nano ESI qQTOF mass spectrometer (5600 TripleTOF, AB Sciex). TripleTOF was operated in the information-dependent acquisition mode, in which a 0.25-s TOF MS scan from 350 to 1250 m/z, was performed, followed by 0.05-s production scans from 100 to 1500 m/z on the 50 most intense 2–5 charged ions. MS/MS information was sent to MASCOT v 2.3.02 or to PARAGON via v 4.5 (AB Sciex). Samples 3, 4 and 5 were searched combined for customer convenience.3.6.1MASCOT search engine (matrix–science)A database search was conducted in Swiss-Prot. Searches were made with tryptic specificity, and by allowing one missed cleavage and tolerance on the mass measurement of 50 ppm in the MS mode and of 0.6 Da for the MS/MS ions. The carbamidomethylation of Cys was used as a fixed modification, and oxidation of Met and deamidation of Asn and Gln were employed as variable modifications. The proteins with a score higher than homology or the significance threshold were identified with confidence ⩾95%. The protein score in the results reported from an MS/MS search was derived from the ions scores.3.6.2Protein Pilot v4.5 search engine (AB Sciex)Protein Pilot default parameters were used to generate a peak list directly from the 5600 Triple TOF wiff files. The Paragon algorithm of Protein Pilot was used to search the Expasy protein database (538585 sequences; 191240774 residues) with the following parameters: trypsin specificity, cys-alkylation, taxonomy restriction to yeast, and search effort set at rapid. To avoid using the same spectral evidence in more than one protein, the identified proteins were grouped based on the MS/MS spectra by the Protein Pilot Progroup algorithm. Thus the proteins sharing MS/MS spectra were clustered, regardless of the peptide sequence assigned. The protein within each group that explained more spectral data with confidence was shown as the primary protein of the group. Only the proteins of the group for which individual evidence existed (unique peptides with enough confidence) were also listed, usually toward the end of the protein list. The proteins with an unused score >1.3 were identified with confidence ⩾95%.3.7Evidence of databank submissionThe mass spectrometry proteomics data have been deposited in the ProteomeXchange Consortium (http://proteomecentral.proteomexchange.org) via the PRIDE partner repository with dataset identifiers PXD001089 and DOI http://dx.doi.org/10.6019/PXD001089 and PXD001090 with DOI http://dx.doi.org/10.6019/PXD001090. The protein interactions from this publication have been submitted to the IMEx (http://www.imexconsortium.org) consortium through IntAct (PMID: 22121220) and, assigned the identifier IM-23292. The data deposited in GEO (GSE56889) are accessible to reviewers through this link: http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?token=kryfcgkmrdsfpab&acc=GSE56889).

Materials and methods

3.1Yeast strains and culture mediaThe Saccharomyces cerevisiae yeast strains used in this study are listed in Table 4. Yeast cells were grown at 30 °C in yeast peptone dextrose (YPD) medium containing, or not, supplementary adenine. Epitope tags and gene replacements were introduced into the genome by PCR-mediated one-step gene replacement. The TAP-tagged yeast strains were constructed by gene targeting at the 3′ end of the ORF with a tag variant (CBP-T7-TEV-Protein A-KAN MX6) of the original TAP tag [48], which we constructed in B. Seraphin’s laboratory. Growth was assayed by spotting serial dilutions of fresh yeast cells exponentially cultured on YPDA and incubated at different temperatures.

Yeast strains and culture media

The Saccharomyces cerevisiae yeast strains used in this study are listed in Table 4. Yeast cells were grown at 30 °C in yeast peptone dextrose (YPD) medium containing, or not, supplementary adenine. Epitope tags and gene replacements were introduced into the genome by PCR-mediated one-step gene replacement. The TAP-tagged yeast strains were constructed by gene targeting at the 3′ end of the ORF with a tag variant (CBP-T7-TEV-Protein A-KAN MX6) of the original TAP tag [48], which we constructed in B. Seraphin’s laboratory. Growth was assayed by spotting serial dilutions of fresh yeast cells exponentially cultured on YPDA and incubated at different temperatures.

3.2Preparation of cell extracts for tandem affinity purification (TAP)At least 6 L of cultured cells grown to an absorbance of OD600 ≈ 2.0 were harvested, instantly frozen in liquid nitrogen and cryo-ground 6 times during a 3-min period at maximum speed using a MM301 ball mill (Retsch). Frozen powdered cells were transferred into 100-mL containers and were stored at −80 °C until used. Cell extracts were prepared using 2 mL/g of cells of a buffer 40 mM HEPES, pH 7.9, 350 mM NaCl, 10% glycerol and 0.1% Nonidet NP-40, containing protease inhibitors 2 mM benzamidine, 1 mM PMSF, 2 mg/mL leupeptine, 2 mg/mL pepstatin A, 2.4 mg/mL chymostatin and 10 μL/mL Trasylol (Bayer). All the affinity purification steps were carried out at 4 °C, basically as described elsewhere [49]. To increase the SAS3-TAP protein purification yield, instead of commercial resins, we used IgG- (I4506, Sigma) and Calmodulin (P1431, Sigma)-coated Dynabeads (M-280, Invitrogen), prepared following the manufacturer’s instructions. For PDP3-TAP, we used commercial resin IgG-Sepharose (17-0969-01, Amersham) and Calmodulin-coated Dynabeads. Two antibodies were used to detect the TAP-tag: P1291 Sigma and CAB1001 Open Biosystems, which respectively recognise the Protein A or the calmodulin-binding peptide regions of the TAP tag. TAP-purified protein complexes were either directly analysed by MS, or dried, and then solved on sample buffer and resolved in 10% SDS–PAGE. Gel was stained with Colloidal Blue (R0571 Fermentas) prior to the MS analysis.

Preparation of cell extracts for tandem affinity purification (TAP)

At least 6 L of cultured cells grown to an absorbance of OD600 ≈ 2.0 were harvested, instantly frozen in liquid nitrogen and cryo-ground 6 times during a 3-min period at maximum speed using a MM301 ball mill (Retsch). Frozen powdered cells were transferred into 100-mL containers and were stored at −80 °C until used. Cell extracts were prepared using 2 mL/g of cells of a buffer 40 mM HEPES, pH 7.9, 350 mM NaCl, 10% glycerol and 0.1% Nonidet NP-40, containing protease inhibitors 2 mM benzamidine, 1 mM PMSF, 2 mg/mL leupeptine, 2 mg/mL pepstatin A, 2.4 mg/mL chymostatin and 10 μL/mL Trasylol (Bayer). All the affinity purification steps were carried out at 4 °C, basically as described elsewhere [49]. To increase the SAS3-TAP protein purification yield, instead of commercial resins, we used IgG- (I4506, Sigma) and Calmodulin (P1431, Sigma)-coated Dynabeads (M-280, Invitrogen), prepared following the manufacturer’s instructions. For PDP3-TAP, we used commercial resin IgG-Sepharose (17-0969-01, Amersham) and Calmodulin-coated Dynabeads. Two antibodies were used to detect the TAP-tag: P1291 Sigma and CAB1001 Open Biosystems, which respectively recognise the Protein A or the calmodulin-binding peptide regions of the TAP tag. TAP-purified protein complexes were either directly analysed by MS, or dried, and then solved on sample buffer and resolved in 10% SDS–PAGE. Gel was stained with Colloidal Blue (R0571 Fermentas) prior to the MS analysis.

3.3In vitro histone acetyltransferase specificity assaysThe PhosphorImager HAT assay was performed as previously described [50], with minor modifications. HAT activities were determined over the whole chromatographic TAP-eluates (5 or 10 μL). Sample fractions were incubated for 20 min at 30 °C with chicken erythrocyte oligonucleosomes or free core histones used as a control and prepared as previously described [51] and with 0.005 μCi [1-14C]acetyl-CoA in a final volume of 16 μL. The reactions were finalised by adding 4 μL of 5× SDS sample loading buffer. The histones in the reaction mixtures were resolved by SDS–PAGE (16%). After staining/fixing and drying, gels were exposed to the BAS-SR IP phosphor-storage image plate (FujiFilm) for different times. Phosphor screens were scanned in a FujiFilm FLA-3000 fluorescent imager analyser.

In vitro histone acetyltransferase specificity assays

The PhosphorImager HAT assay was performed as previously described [50], with minor modifications. HAT activities were determined over the whole chromatographic TAP-eluates (5 or 10 μL). Sample fractions were incubated for 20 min at 30 °C with chicken erythrocyte oligonucleosomes or free core histones used as a control and prepared as previously described [51] and with 0.005 μCi [1-14C]acetyl-CoA in a final volume of 16 μL. The reactions were finalised by adding 4 μL of 5× SDS sample loading buffer. The histones in the reaction mixtures were resolved by SDS–PAGE (16%). After staining/fixing and drying, gels were exposed to the BAS-SR IP phosphor-storage image plate (FujiFilm) for different times. Phosphor screens were scanned in a FujiFilm FLA-3000 fluorescent imager analyser.

3.4Histone immunoblotting assaysFor the histone Western blots, cells were collected at OD600 ≈ 0.8. WCEs were prepared as described elsewhere [52] and were resolved by SDS–PAGE (16%). Protein gels were transferred to 0.2-μm pore nitrocellulose membranes (Protan, Whatman), as described elsewhere [53]. Membranes, after staining with Ponceau S and subsequent washes, were processed according to the ECL Advance Western Blotting detection system (RNP 2135, GE Healthcare) instructions. The primary antibodies were the following: α-H3Ct (ab1791, Abcam), α-H3K9ac (06-942, Millipore) and α-H3K14ac (07-353, Upstate), and were used at a 10-fold dilution as suggested by the manufacturer. The secondary antibody was horseradish peroxidase-linked anti-rabbit (GE Healthcare), employed at a dilution of 1/40,000.

Histone immunoblotting assays

For the histone Western blots, cells were collected at OD600 ≈ 0.8. WCEs were prepared as described elsewhere [52] and were resolved by SDS–PAGE (16%). Protein gels were transferred to 0.2-μm pore nitrocellulose membranes (Protan, Whatman), as described elsewhere [53]. Membranes, after staining with Ponceau S and subsequent washes, were processed according to the ECL Advance Western Blotting detection system (RNP 2135, GE Healthcare) instructions. The primary antibodies were the following: α-H3Ct (ab1791, Abcam), α-H3K9ac (06-942, Millipore) and α-H3K14ac (07-353, Upstate), and were used at a 10-fold dilution as suggested by the manufacturer. The secondary antibody was horseradish peroxidase-linked anti-rabbit (GE Healthcare), employed at a dilution of 1/40,000.

3.5ChIP-on-chip experimentsChIP-on-chip experiments were performed as previously described [45], but with some modifications. Briefly, the antibody used for the genome-wide location analysis was 2 μg of rat anti-HA 3F10 (Roche). The rat antibody was coupled to a suspension of 100 μL of 50% (v/v) Protein G Sepharose 4FF (Amersham Biosciences) equilibrated in lysis buffer (50 mM HEPES-KOH, pH 7.5, 140 mM NaCl, 1 mM EDTA, 1% Triton X-100, 0.1% sodium deoxycholate, 1 mM Phenylmethylsulfonyl fluoride, 1 mM benzamidine, and 1 pill of protease inhibitor cocktail (Roche) was dissolved in every 25 mL of buffer) supplemented with 5 mg/mL of BSA. Next, 2000 ng of DNA immunoprecipitated from the BQS1217 strain (SAS3-HA tagged) and DNA immunoprecipitated from the BY4742 strain (WT) were labelled with Cyanine 5-dUTP and Cyanine 3-dUTP, respectively, using the ‘CGH Labelling Kit’ (Invitrogen, p/n 18095-011) following the manufacturer’s instructions. Labelled DNA was hybridised to the Yeast Whole Genome ChIP-on-chip Microarray (Agilent p/n G4491A, AMADID: 014741), specifically designed for location analyses.Data were processed by ChIP Analytics, 1.3 (Agilent). Blank subtraction normalisation, and inter-array and intra-array median normalisation were performed. Probes were lifted over to the sacCer3 genome version (April 2011). Transcript boundaries [46] were downloaded from SGD (R64-1-1, http://www.yeastgenome.org). Raw and processed data were deposited in GEO: GSE56889. Plots were performed using R and Bioconductor [54]. Probes were binned to 200 bp, and the mean and its confident intervals (t-test, 95% confidence) were plotted. Experiments were carried out in duplicate.

ChIP-on-chip experiments

ChIP-on-chip experiments were performed as previously described [45], but with some modifications. Briefly, the antibody used for the genome-wide location analysis was 2 μg of rat anti-HA 3F10 (Roche). The rat antibody was coupled to a suspension of 100 μL of 50% (v/v) Protein G Sepharose 4FF (Amersham Biosciences) equilibrated in lysis buffer (50 mM HEPES-KOH, pH 7.5, 140 mM NaCl, 1 mM EDTA, 1% Triton X-100, 0.1% sodium deoxycholate, 1 mM Phenylmethylsulfonyl fluoride, 1 mM benzamidine, and 1 pill of protease inhibitor cocktail (Roche) was dissolved in every 25 mL of buffer) supplemented with 5 mg/mL of BSA. Next, 2000 ng of DNA immunoprecipitated from the BQS1217 strain (SAS3-HA tagged) and DNA immunoprecipitated from the BY4742 strain (WT) were labelled with Cyanine 5-dUTP and Cyanine 3-dUTP, respectively, using the ‘CGH Labelling Kit’ (Invitrogen, p/n 18095-011) following the manufacturer’s instructions. Labelled DNA was hybridised to the Yeast Whole Genome ChIP-on-chip Microarray (Agilent p/n G4491A, AMADID: 014741), specifically designed for location analyses.

Data were processed by ChIP Analytics, 1.3 (Agilent). Blank subtraction normalisation, and inter-array and intra-array median normalisation were performed. Probes were lifted over to the sacCer3 genome version (April 2011). Transcript boundaries [46] were downloaded from SGD (R64-1-1, http://www.yeastgenome.org). Raw and processed data were deposited in GEO: GSE56889. Plots were performed using R and Bioconductor [54]. Probes were binned to 200 bp, and the mean and its confident intervals (t-test, 95% confidence) were plotted. Experiments were carried out in duplicate.

3.6Mass spectrometry analysisThe TAP-tagged co-purified proteins were treated and identified by LC–MS/MS at the 95% confidence level, as previously described [36,55]. Briefly, the proteins from the TAP-eluates were precipitated with TCA/acetone, solved and digested with trypsin. The resulting peptides were loaded into LC Packings Pep Map C18 columns and chromatographed by elution with a linear gradient of 5% to 45–50% of acetonitrile in 0.1% formic acid for 120 min. The peptides from the Sas3p sample were analysed in a nano ESI Q-TOF mass spectrometer (QSTAR XL, ABI) and those from the Pdp3p sample in a MALDI-TOF–TOF (4700 Proteomic Analyser, ABI). Searches were made in the Swiss-Prot database using the MASCOT and Protein Pilot search engines.Slices from the gel corresponding to PDP3-TAP were excised and digested with sequencing grade trypsin (Promega), as described elsewhere [56]. The digestion mixture was dried in a vacuum centrifuge, and dissolved in 2% ACN, 0.1% TFA (samples 1 and 2 in 10 μL; samples 4,5 and 6 in 10 μL). Then 5 μL of each sample were loaded into a trap column (Nano LC Column, 3 μ C18-CL, 75 um × 15 cm; Eksigen) and desalted with 0.1% TFA at 2 μL/min for 10 min. Peptides were loaded into an analytical column (LC Column, 3 μ C18-CL, 75 um × 25 cm, Eksigen), equilibrated in 5% acetonitrile 0.1% FA (formic acid). Peptide elution was carried out with a linear gradient of 5–40% B in 30 min (A: 0.1% FA; B: ACN, 0.1% FA) at a flow rate of 300 nL/min. Peptides were analysed in a nano ESI qQTOF mass spectrometer (5600 TripleTOF, AB Sciex). TripleTOF was operated in the information-dependent acquisition mode, in which a 0.25-s TOF MS scan from 350 to 1250 m/z, was performed, followed by 0.05-s production scans from 100 to 1500 m/z on the 50 most intense 2–5 charged ions. MS/MS information was sent to MASCOT v 2.3.02 or to PARAGON via v 4.5 (AB Sciex). Samples 3, 4 and 5 were searched combined for customer convenience.3.6.1MASCOT search engine (matrix–science)A database search was conducted in Swiss-Prot. Searches were made with tryptic specificity, and by allowing one missed cleavage and tolerance on the mass measurement of 50 ppm in the MS mode and of 0.6 Da for the MS/MS ions. The carbamidomethylation of Cys was used as a fixed modification, and oxidation of Met and deamidation of Asn and Gln were employed as variable modifications. The proteins with a score higher than homology or the significance threshold were identified with confidence ⩾95%. The protein score in the results reported from an MS/MS search was derived from the ions scores.3.6.2Protein Pilot v4.5 search engine (AB Sciex)Protein Pilot default parameters were used to generate a peak list directly from the 5600 Triple TOF wiff files. The Paragon algorithm of Protein Pilot was used to search the Expasy protein database (538585 sequences; 191240774 residues) with the following parameters: trypsin specificity, cys-alkylation, taxonomy restriction to yeast, and search effort set at rapid. To avoid using the same spectral evidence in more than one protein, the identified proteins were grouped based on the MS/MS spectra by the Protein Pilot Progroup algorithm. Thus the proteins sharing MS/MS spectra were clustered, regardless of the peptide sequence assigned. The protein within each group that explained more spectral data with confidence was shown as the primary protein of the group. Only the proteins of the group for which individual evidence existed (unique peptides with enough confidence) were also listed, usually toward the end of the protein list. The proteins with an unused score >1.3 were identified with confidence ⩾95%.

Mass spectrometry analysis

The TAP-tagged co-purified proteins were treated and identified by LC–MS/MS at the 95% confidence level, as previously described [36,55]. Briefly, the proteins from the TAP-eluates were precipitated with TCA/acetone, solved and digested with trypsin. The resulting peptides were loaded into LC Packings Pep Map C18 columns and chromatographed by elution with a linear gradient of 5% to 45–50% of acetonitrile in 0.1% formic acid for 120 min. The peptides from the Sas3p sample were analysed in a nano ESI Q-TOF mass spectrometer (QSTAR XL, ABI) and those from the Pdp3p sample in a MALDI-TOF–TOF (4700 Proteomic Analyser, ABI). Searches were made in the Swiss-Prot database using the MASCOT and Protein Pilot search engines.

Slices from the gel corresponding to PDP3-TAP were excised and digested with sequencing grade trypsin (Promega), as described elsewhere [56]. The digestion mixture was dried in a vacuum centrifuge, and dissolved in 2% ACN, 0.1% TFA (samples 1 and 2 in 10 μL; samples 4,5 and 6 in 10 μL). Then 5 μL of each sample were loaded into a trap column (Nano LC Column, 3 μ C18-CL, 75 um × 15 cm; Eksigen) and desalted with 0.1% TFA at 2 μL/min for 10 min. Peptides were loaded into an analytical column (LC Column, 3 μ C18-CL, 75 um × 25 cm, Eksigen), equilibrated in 5% acetonitrile 0.1% FA (formic acid). Peptide elution was carried out with a linear gradient of 5–40% B in 30 min (A: 0.1% FA; B: ACN, 0.1% FA) at a flow rate of 300 nL/min. Peptides were analysed in a nano ESI qQTOF mass spectrometer (5600 TripleTOF, AB Sciex). TripleTOF was operated in the information-dependent acquisition mode, in which a 0.25-s TOF MS scan from 350 to 1250 m/z, was performed, followed by 0.05-s production scans from 100 to 1500 m/z on the 50 most intense 2–5 charged ions. MS/MS information was sent to MASCOT v 2.3.02 or to PARAGON via v 4.5 (AB Sciex). Samples 3, 4 and 5 were searched combined for customer convenience.

3.6.1MASCOT search engine (matrix–science)A database search was conducted in Swiss-Prot. Searches were made with tryptic specificity, and by allowing one missed cleavage and tolerance on the mass measurement of 50 ppm in the MS mode and of 0.6 Da for the MS/MS ions. The carbamidomethylation of Cys was used as a fixed modification, and oxidation of Met and deamidation of Asn and Gln were employed as variable modifications. The proteins with a score higher than homology or the significance threshold were identified with confidence ⩾95%. The protein score in the results reported from an MS/MS search was derived from the ions scores.

MASCOT search engine (matrix–science)

A database search was conducted in Swiss-Prot. Searches were made with tryptic specificity, and by allowing one missed cleavage and tolerance on the mass measurement of 50 ppm in the MS mode and of 0.6 Da for the MS/MS ions. The carbamidomethylation of Cys was used as a fixed modification, and oxidation of Met and deamidation of Asn and Gln were employed as variable modifications. The proteins with a score higher than homology or the significance threshold were identified with confidence ⩾95%. The protein score in the results reported from an MS/MS search was derived from the ions scores.

3.6.2Protein Pilot v4.5 search engine (AB Sciex)Protein Pilot default parameters were used to generate a peak list directly from the 5600 Triple TOF wiff files. The Paragon algorithm of Protein Pilot was used to search the Expasy protein database (538585 sequences; 191240774 residues) with the following parameters: trypsin specificity, cys-alkylation, taxonomy restriction to yeast, and search effort set at rapid. To avoid using the same spectral evidence in more than one protein, the identified proteins were grouped based on the MS/MS spectra by the Protein Pilot Progroup algorithm. Thus the proteins sharing MS/MS spectra were clustered, regardless of the peptide sequence assigned. The protein within each group that explained more spectral data with confidence was shown as the primary protein of the group. Only the proteins of the group for which individual evidence existed (unique peptides with enough confidence) were also listed, usually toward the end of the protein list. The proteins with an unused score >1.3 were identified with confidence ⩾95%.

Protein Pilot v4.5 search engine (AB Sciex)

Protein Pilot default parameters were used to generate a peak list directly from the 5600 Triple TOF wiff files. The Paragon algorithm of Protein Pilot was used to search the Expasy protein database (538585 sequences; 191240774 residues) with the following parameters: trypsin specificity, cys-alkylation, taxonomy restriction to yeast, and search effort set at rapid. To avoid using the same spectral evidence in more than one protein, the identified proteins were grouped based on the MS/MS spectra by the Protein Pilot Progroup algorithm. Thus the proteins sharing MS/MS spectra were clustered, regardless of the peptide sequence assigned. The protein within each group that explained more spectral data with confidence was shown as the primary protein of the group. Only the proteins of the group for which individual evidence existed (unique peptides with enough confidence) were also listed, usually toward the end of the protein list. The proteins with an unused score >1.3 were identified with confidence ⩾95%.

3.7Evidence of databank submissionThe mass spectrometry proteomics data have been deposited in the ProteomeXchange Consortium (http://proteomecentral.proteomexchange.org) via the PRIDE partner repository with dataset identifiers PXD001089 and DOI http://dx.doi.org/10.6019/PXD001089 and PXD001090 with DOI http://dx.doi.org/10.6019/PXD001090. The protein interactions from this publication have been submitted to the IMEx (http://www.imexconsortium.org) consortium through IntAct (PMID: 22121220) and, assigned the identifier IM-23292. The data deposited in GEO (GSE56889) are accessible to reviewers through this link: http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?token=kryfcgkmrdsfpab&acc=GSE56889).

Evidence of databank submission

The mass spectrometry proteomics data have been deposited in the ProteomeXchange Consortium (http://proteomecentral.proteomexchange.org) via the PRIDE partner repository with dataset identifiers PXD001089 and DOI http://dx.doi.org/10.6019/PXD001089 and PXD001090 with DOI http://dx.doi.org/10.6019/PXD001090. The protein interactions from this publication have been submitted to the IMEx (http://www.imexconsortium.org) consortium through IntAct (PMID: 22121220) and, assigned the identifier IM-23292. The data deposited in GEO (GSE56889) are accessible to reviewers through this link: http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?token=kryfcgkmrdsfpab&acc=GSE56889).
