---
pmid: '25404562'
title: Conservation of an intricate circuit for crucial modifications of the tRNAPhe
  anticodon loop in eukaryotes.
authors:
- Guy MP
- Phizicky EM
journal: RNA
year: '2015'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC4274638
doi: 10.1261/rna.047639.114
---

# Conservation of an intricate circuit for crucial modifications of the tRNAPhe anticodon loop in eukaryotes.
**Authors:** Guy MP, Phizicky EM
**Journal:** RNA (2015)
**DOI:** [10.1261/rna.047639.114](https://doi.org/10.1261/rna.047639.114)
**PMC:** [PMC4274638](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4274638/)

## Abstract

1. RNA. 2015 Jan;21(1):61-74. doi: 10.1261/rna.047639.114. Epub 2014 Nov 17.

Conservation of an intricate circuit for crucial modifications of the tRNAPhe 
anticodon loop in eukaryotes.

Guy MP(1), Phizicky EM(2).

Author information:
(1)Department of Biochemistry and Biophysics, Center for RNA Biology, University 
of Rochester School of Medicine, Rochester, New York 14642, USA.
(2)Department of Biochemistry and Biophysics, Center for RNA Biology, University 
of Rochester School of Medicine, Rochester, New York 14642, USA 
eric_phizicky@urmc.rochester.edu.

Post-transcriptional tRNA modifications are critical for efficient and accurate 
translation, and have multiple different roles. Lack of modifications often 
leads to different biological consequences in different organisms, and in humans 
is frequently associated with neurological disorders. We investigate here the 
conservation of a unique circuitry for anticodon loop modification required for 
healthy growth in the yeast Saccharomyces cerevisiae. S. cerevisiae Trm7 
interacts separately with Trm732 and Trm734 to 2'-O-methylate three substrate 
tRNAs at anticodon loop residues C₃₂ and N₃₄, and these modifications are 
required for efficient wybutosine formation at m(1)G₃₇ of tRNA(Phe). Moreover, 
trm7Δ and trm732Δ trm734Δ mutants grow poorly due to lack of functional 
tRNA(Phe). It is unknown if this circuitry is conserved and important for 
tRNA(Phe) modification in other eukaryotes, but a likely human TRM7 ortholog is 
implicated in nonsyndromic X-linked intellectual disability. We find that the 
distantly related yeast Schizosaccharomyces pombe has retained this circuitry 
for anticodon loop modification, that S. pombe trm7Δ and trm734Δ mutants have 
more severe phenotypes than the S. cerevisiae mutants, and that tRNA(Phe) is the 
major biological target. Furthermore, we provide evidence that Trm7 and Trm732 
function is widely conserved throughout eukaryotes, since human FTSJ1 and THADA, 
respectively, complement growth defects of S. cerevisiae trm7Δ and trm732Δ 
trm734Δ mutants by modifying C₃₂ of tRNA(Phe), each working with the 
corresponding S. cerevisiae partner protein. These results suggest widespread 
importance of 2'-O-methylation of the tRNA anticodon loop, implicate tRNA(Phe) 
as the crucial substrate, and suggest that this modification circuitry is 
important for human neuronal development.

© 2014 Guy and Phizicky; Published by Cold Spring Harbor Laboratory Press for 
the RNA Society.

DOI: 10.1261/rna.047639.114
PMCID: PMC4274638
PMID: 25404562 [Indexed for MEDLINE]

## Full Text

INTRODUCTIONPost-transcriptional modification of tRNA is universally required for accurate and efficient translation. Modifications are found in all characterized tRNA species (Machnicka et al. 2013), and are highly conserved within each domain of life (Grosjean 2009). Modifications have a number of different roles, with well documented examples including modulating the efficiency and specificity of charging (Muramatsu et al. 1988; Pütz et al. 1994), altering the specificity of decoding (Johansson et al. 2008), maintaining the frame for decoding (Urbonavicius et al. 2001), and preventing decay of pre-tRNA (Kadaba et al. 2004) and mature tRNA (Alexandrov et al. 2006; Chernyakov et al. 2008).Many tRNA modifications have a similar biological impact on different organisms. For example, the genes responsible for modification of residue A34 to I34 (inosine) in the wobble position of tRNAs are essential in the bacterium Escherichia coli (Wolf et al. 2002), in the yeast Saccharomyces cerevisiae (Gerber and Keller 1999), and in the distantly related yeast Schizosaccharomyces pombe (Kim et al. 2010), and RNAi against a putative homolog results in 29% embryonic lethality in the nematode Caenorhabditis elegans (Fernandez et al. 2005). Similarly, lack of the terminal methyl group of mcm5U34 (5-methoxycarbonylmethyluridine) in S. cerevisiae due to mutation of TRM9 results in sensitivity to aminoglycosides (Kalhor and Clarke 2003) and DNA damaging agents (Begley et al. 2007) but no other growth defects, depletion of the human protein results in DNA damage sensitivity but no other growth defects (Fu et al. 2010), and mice lacking TRM9 (Alkbh8−/−) appear normal (Songe-Møller et al. 2010).In contrast, many modifications often do not have precisely the same biological impact on different organisms. Thus, for example, the genes that specify the m1A58 (1-methyladensosine) modification found in many eukaryotic tRNAs are essential in S. cerevisiae (Anderson et al. 1998) due to turnover of pre-tRNAiMet by the nuclear surveillance pathway (Kadaba et al. 2004), but, the corresponding gene knockouts are each viable in S. pombe (Kim et al. 2010), although one knockout strain that was examined grows slowly and is more sensitive to oxidative stress (Zuin et al. 2008). Similarly, the genes required for t6A37 (N6-threonlycarbamoyladenosine) formation are essential in E. coli and Haloferax volcanii, but not in S. cerevisiae, although mutants grow poorly (El Yacoubi et al. 2011; Srinivasan et al. 2011; Naor et al. 2012).Several modifications with more modest phenotypes in S. cerevisiae also appear to have a markedly different biological impact on different organisms. Thus, for example, lack of i6A37 (N6-isopentenyladenosine) due to mutation of MOD5 in S. cerevisiae, results in reduced nonsense suppression (Laten et al. 1978; Dihanich et al. 1987), but no obvious growth defect, whereas S. pombe mod5▵ (Sp tit1▵) mutants grow slowly on glycerol or rapamycin (Lamichhane et al. 2013), C. elegans mod-5 mutants (gro-1) have slowed embryogenesis and development and an increased life span (Lemieux et al. 2001), and mutations in human MOD5 (TRIT1) have been linked to encephalopathy and epilepsy due to mitochondrial defects (Yarham et al. 2014).Although only a subset of tRNA substrates of a modification enzyme are often responsible for the known phenotypes of a mutation in the corresponding modification gene (Phizicky and Alfonzo 2010; Guy et al. 2012), it is unclear if the same subset of tRNA species are equally important in each organism. For example, in S. cerevisiae all phenotypes associated with mutations in the ELP complex (which forms the cm5U moiety found in mcm5U34, ncm5U34, [5-carbamoylmethyluridine], and mcm5s2U34 [5-methoxycarbonylmethyl-2-thiouridine]) are due to lack of functional tRNALys(UUU), tRNAGln(UUG), and tRNAGlu(UUC), although 11 tRNAs are targets of the ELP complex (Esberg et al. 2006; Johansson et al. 2008; Chen et al. 2011b); in contrast, it appears that only overexpression of tRNALys(UUU) is required in S. pombe to suppress most of the sensitivity to oxidative stress in an Sp elp3 mutant (which lacks both the cm5 moiety and the s2 moiety of mcm5s2U), although tRNAGln(UUG) and tRNAGlu(UUC) have the same mcm5s2U modification (Fernández-Vázquez et al. 2013).The roles and biological effects of 2′-O-methylation of the anticodon loop are of particular interest because of prior work in S. cerevisiae showing a severe phenotype of mutants, specificity of that phenotype for modification of only one substrate, and an intricate circuitry for modification of tRNA substrates (Pintard et al. 2002; Guy et al. 2012). S. cerevisiae Trm7 is required for 2′-O-methylation of the anticodon loop of tRNAPhe, tRNATrp, and tRNALeu(UAA) at C32 (forming 2′-O-methylcytidine, Cm32) and at N34 (forming Gm34, Cm34, and ncm5Um34, respectively); and Sc trm7▵ mutants have a severe growth defect (Pintard et al. 2002). Interestingly, Sc Trm7 interacts with Sc Trm732 to form Cm32, and separately interacts with Sc Trm734 to form Nm34, and both modifications are required to efficiently drive formation of yW37 (wybutosine) at m1G37 on tRNAPhe (Guy et al. 2012) by Sc Tyw1–Tyw4 (Fig. 1A; Noma et al. 2006). Furthermore, overexpression of only tRNAPhe fully suppresses the growth defect and the aminoglycoside sensitivity of Sc trm7▵ mutants. In addition, the growth defect of an Sc trm7▵ mutant requires loss of both the Cm32 and Gm34 modifications of tRNAPhe (and the accompanying loss of yW37) since neither an Sc trm732▵ nor an Sc trm734▵ single mutant has any observable growth defect, whereas the double mutant is as sick as an Sc trm7▵ mutant and is equally suppressed by overexpression of tRNAPhe (Guy et al. 2012).FIGURE 1.Trm7 modification machinery in eukaryotes. (A) Schematic of the anticodon loop of tRNAPhe and tRNALeu(UAA) from S. cerevisiae. S. cerevisiae Trm7 (Sc Trm7) requires Sc Trm732 and Sc Trm734 to form Cm32, and Nm34, respectively, on tRNAPhe and tRNALeu(UAA). Cm32 and Gm34 modification then drive yW formation from m1G on tRNAPhe. Wider arrow for Gm34 indicates that yW formation is more dependent on this modification than on Cm32. Predicted human homologs of Sc Trm7, Sc Trm732, and Sc Trm734 are in brackets. (B) Amino acid sequence alignment of Sc Trm7 with putative homologs from S. pombe and H. sapiens. (C) Schematic representation of Sc Trm732 aligned with putative homologs from S. pombe and H. sapiens. Inset box is an amino acid alignment of the DUF2428 domain found in these proteins.The goal of the work described here is to determine if this intricate circuitry, the biologically significant tRNA target, and the importance of Trm7 modifications are conserved, focusing on S. pombe, whose lineage diverged from that of S. cerevisiae ∼1.1 billion yr ago (Hedges 2002). Available evidence is equivocal for conservation of all these features among eukaryotes. Although Trm7 is highly conserved in each of 25 divergent eukaryotic genomes examined (Fig. 1B) (see Materials and Methods), Trm732 homologs can only be identified in 22 genomes, and Trm734 homologs only in 14 genomes, and both protein families have little overall sequence similarity, (∼23% and 21% overall identity between the Sc proteins and their predicted human orthologs THADA [Trm732] [Fig. 1C] and WDR6 [Trm734], respectively [Shi et al. 2011]). Moreover, S. cerevisiae Trm734 has been implicated in regulation of Ty1 transposition (Nyswaner et al. 2008) and endoplasmic recycling (Shi et al. 2011), suggesting that Trm734 family proteins from other organisms might have other roles in addition to, or instead of, tRNA modification.It also appears that TRM7 has an important but differing biological impact on different organisms, since a high throughput screen in S. pombe indicated that the putative TRM7 gene was essential (Kim et al. 2010), and since mutations in a putative human TRM7 homolog (FTSJ1) are associated with nonsyndromic X-linked intellectual disability (NSXLID) (Freude et al. 2004; Ramser et al. 2004; Froyen et al. 2007; Takano et al. 2008).We report here that the circuitry and biologically important substrate for 2′-O-methylation of the tRNA anticodon loop are conserved in S. pombe, and that S. pombe trm7Δ and trm734Δ mutants are viable, but with more severe growth defects than in S. cerevisiae. Furthermore, we provide evidence that this circuitry is retained in other eukaryotes including humans, suggesting that defective 2′-O-methylation of tRNA is linked to NSXLID.

INTRODUCTION

Post-transcriptional modification of tRNA is universally required for accurate and efficient translation. Modifications are found in all characterized tRNA species (Machnicka et al. 2013), and are highly conserved within each domain of life (Grosjean 2009). Modifications have a number of different roles, with well documented examples including modulating the efficiency and specificity of charging (Muramatsu et al. 1988; Pütz et al. 1994), altering the specificity of decoding (Johansson et al. 2008), maintaining the frame for decoding (Urbonavicius et al. 2001), and preventing decay of pre-tRNA (Kadaba et al. 2004) and mature tRNA (Alexandrov et al. 2006; Chernyakov et al. 2008).

Many tRNA modifications have a similar biological impact on different organisms. For example, the genes responsible for modification of residue A34 to I34 (inosine) in the wobble position of tRNAs are essential in the bacterium Escherichia coli (Wolf et al. 2002), in the yeast Saccharomyces cerevisiae (Gerber and Keller 1999), and in the distantly related yeast Schizosaccharomyces pombe (Kim et al. 2010), and RNAi against a putative homolog results in 29% embryonic lethality in the nematode Caenorhabditis elegans (Fernandez et al. 2005). Similarly, lack of the terminal methyl group of mcm5U34 (5-methoxycarbonylmethyluridine) in S. cerevisiae due to mutation of TRM9 results in sensitivity to aminoglycosides (Kalhor and Clarke 2003) and DNA damaging agents (Begley et al. 2007) but no other growth defects, depletion of the human protein results in DNA damage sensitivity but no other growth defects (Fu et al. 2010), and mice lacking TRM9 (Alkbh8−/−) appear normal (Songe-Møller et al. 2010).

In contrast, many modifications often do not have precisely the same biological impact on different organisms. Thus, for example, the genes that specify the m1A58 (1-methyladensosine) modification found in many eukaryotic tRNAs are essential in S. cerevisiae (Anderson et al. 1998) due to turnover of pre-tRNAiMet by the nuclear surveillance pathway (Kadaba et al. 2004), but, the corresponding gene knockouts are each viable in S. pombe (Kim et al. 2010), although one knockout strain that was examined grows slowly and is more sensitive to oxidative stress (Zuin et al. 2008). Similarly, the genes required for t6A37 (N6-threonlycarbamoyladenosine) formation are essential in E. coli and Haloferax volcanii, but not in S. cerevisiae, although mutants grow poorly (El Yacoubi et al. 2011; Srinivasan et al. 2011; Naor et al. 2012).

Several modifications with more modest phenotypes in S. cerevisiae also appear to have a markedly different biological impact on different organisms. Thus, for example, lack of i6A37 (N6-isopentenyladenosine) due to mutation of MOD5 in S. cerevisiae, results in reduced nonsense suppression (Laten et al. 1978; Dihanich et al. 1987), but no obvious growth defect, whereas S. pombe mod5▵ (Sp tit1▵) mutants grow slowly on glycerol or rapamycin (Lamichhane et al. 2013), C. elegans mod-5 mutants (gro-1) have slowed embryogenesis and development and an increased life span (Lemieux et al. 2001), and mutations in human MOD5 (TRIT1) have been linked to encephalopathy and epilepsy due to mitochondrial defects (Yarham et al. 2014).

Although only a subset of tRNA substrates of a modification enzyme are often responsible for the known phenotypes of a mutation in the corresponding modification gene (Phizicky and Alfonzo 2010; Guy et al. 2012), it is unclear if the same subset of tRNA species are equally important in each organism. For example, in S. cerevisiae all phenotypes associated with mutations in the ELP complex (which forms the cm5U moiety found in mcm5U34, ncm5U34, [5-carbamoylmethyluridine], and mcm5s2U34 [5-methoxycarbonylmethyl-2-thiouridine]) are due to lack of functional tRNALys(UUU), tRNAGln(UUG), and tRNAGlu(UUC), although 11 tRNAs are targets of the ELP complex (Esberg et al. 2006; Johansson et al. 2008; Chen et al. 2011b); in contrast, it appears that only overexpression of tRNALys(UUU) is required in S. pombe to suppress most of the sensitivity to oxidative stress in an Sp elp3 mutant (which lacks both the cm5 moiety and the s2 moiety of mcm5s2U), although tRNAGln(UUG) and tRNAGlu(UUC) have the same mcm5s2U modification (Fernández-Vázquez et al. 2013).

The roles and biological effects of 2′-O-methylation of the anticodon loop are of particular interest because of prior work in S. cerevisiae showing a severe phenotype of mutants, specificity of that phenotype for modification of only one substrate, and an intricate circuitry for modification of tRNA substrates (Pintard et al. 2002; Guy et al. 2012). S. cerevisiae Trm7 is required for 2′-O-methylation of the anticodon loop of tRNAPhe, tRNATrp, and tRNALeu(UAA) at C32 (forming 2′-O-methylcytidine, Cm32) and at N34 (forming Gm34, Cm34, and ncm5Um34, respectively); and Sc trm7▵ mutants have a severe growth defect (Pintard et al. 2002). Interestingly, Sc Trm7 interacts with Sc Trm732 to form Cm32, and separately interacts with Sc Trm734 to form Nm34, and both modifications are required to efficiently drive formation of yW37 (wybutosine) at m1G37 on tRNAPhe (Guy et al. 2012) by Sc Tyw1–Tyw4 (Fig. 1A; Noma et al. 2006). Furthermore, overexpression of only tRNAPhe fully suppresses the growth defect and the aminoglycoside sensitivity of Sc trm7▵ mutants. In addition, the growth defect of an Sc trm7▵ mutant requires loss of both the Cm32 and Gm34 modifications of tRNAPhe (and the accompanying loss of yW37) since neither an Sc trm732▵ nor an Sc trm734▵ single mutant has any observable growth defect, whereas the double mutant is as sick as an Sc trm7▵ mutant and is equally suppressed by overexpression of tRNAPhe (Guy et al. 2012).

Trm7 modification machinery in eukaryotes. (A) Schematic of the anticodon loop of tRNAPhe and tRNALeu(UAA) from S. cerevisiae. S. cerevisiae Trm7 (Sc Trm7) requires Sc Trm732 and Sc Trm734 to form Cm32, and Nm34, respectively, on tRNAPhe and tRNALeu(UAA). Cm32 and Gm34 modification then drive yW formation from m1G on tRNAPhe. Wider arrow for Gm34 indicates that yW formation is more dependent on this modification than on Cm32. Predicted human homologs of Sc Trm7, Sc Trm732, and Sc Trm734 are in brackets. (B) Amino acid sequence alignment of Sc Trm7 with putative homologs from S. pombe and H. sapiens. (C) Schematic representation of Sc Trm732 aligned with putative homologs from S. pombe and H. sapiens. Inset box is an amino acid alignment of the DUF2428 domain found in these proteins.

The goal of the work described here is to determine if this intricate circuitry, the biologically significant tRNA target, and the importance of Trm7 modifications are conserved, focusing on S. pombe, whose lineage diverged from that of S. cerevisiae ∼1.1 billion yr ago (Hedges 2002). Available evidence is equivocal for conservation of all these features among eukaryotes. Although Trm7 is highly conserved in each of 25 divergent eukaryotic genomes examined (Fig. 1B) (see Materials and Methods), Trm732 homologs can only be identified in 22 genomes, and Trm734 homologs only in 14 genomes, and both protein families have little overall sequence similarity, (∼23% and 21% overall identity between the Sc proteins and their predicted human orthologs THADA [Trm732] [Fig. 1C] and WDR6 [Trm734], respectively [Shi et al. 2011]). Moreover, S. cerevisiae Trm734 has been implicated in regulation of Ty1 transposition (Nyswaner et al. 2008) and endoplasmic recycling (Shi et al. 2011), suggesting that Trm734 family proteins from other organisms might have other roles in addition to, or instead of, tRNA modification.

It also appears that TRM7 has an important but differing biological impact on different organisms, since a high throughput screen in S. pombe indicated that the putative TRM7 gene was essential (Kim et al. 2010), and since mutations in a putative human TRM7 homolog (FTSJ1) are associated with nonsyndromic X-linked intellectual disability (NSXLID) (Freude et al. 2004; Ramser et al. 2004; Froyen et al. 2007; Takano et al. 2008).

We report here that the circuitry and biologically important substrate for 2′-O-methylation of the tRNA anticodon loop are conserved in S. pombe, and that S. pombe trm7Δ and trm734Δ mutants are viable, but with more severe growth defects than in S. cerevisiae. Furthermore, we provide evidence that this circuitry is retained in other eukaryotes including humans, suggesting that defective 2′-O-methylation of tRNA is linked to NSXLID.

RESULTSThe putative S. pombe trm7+ gene is required for Cm and Gm formation on tRNAPhe and mutants are barely viableTo determine if S. cerevisiae Trm7 function is conserved in eukaryotes, we generated and examined S. pombe strains lacking the likely trm7+ gene (SPAC4F10.03c). Consistent with previous high throughput results (Kim et al. 2010), we were unable to generate S. pombe trm7Δ mutant haploids from a heterozygous diploid (relevant genotype: spac4F10.03cΔ::kanMX/SPAC4F10.03c) by sporulation and selection for the mutant haploid (data not shown). However, by first introducing a ura4+ plasmid expressing S. pombe trm7+ under control of the Pnmt1 promoter (no message in thiamine), we could recover an Sp trm7Δ [Pnmt1 Sp trm7+
ura4+] haploid after sporulation on Edinburgh minimal medium (EMM, permissive conditions) in the presence of selective drug; moreover, this Sp trm7Δ [Pnmt1 Sp trm7+
ura4+] haploid was viable (albeit barely) when plated to EMM containing 5-fluoroorotic acid (5-FOA) to select against the ura4+ plasmid. Indeed, after streaking the Sp trm7Δ [Pnmt1 Sp trm7+
ura4+] strain to EMM containing 0.5 mg/L 5-FOA at 30°C, it took 11 d of growth to attain visible colonies, compared with 3 d for wild-type cells (data not shown). Control experiments demonstrated that the growth defect was due to the trm7Δ mutation of the Sp trm7Δ [Pnmt1 Sp trm7+
ura4+] strain, since introduction of a [Pnmt1** Sp trm7+
LEU2] plasmid (expressing trm7+ under control of the low strength no message in thiamine promoter), resulted in healthy growth after plating to EMM containing 5-FOA (Fig. 2A). The growth defect of Sp trm7Δ mutants thus appears to be more severe than that observed in the S. cerevisiae trm7Δ mutant, which is sick but viable (Pintard et al. 2002; Guy et al. 2012). Under repressive conditions in liquid YES medium, the Sp trm7Δ [Pnmt1** Sp trm7+
LEU2] strain (although much healthier than the Sp trm7Δ haploid) had a generation time of 320 min, compared with 148 min for the wild-type strain (a difference of 2.16-fold) (Table 1), which was still more than the difference of 1.94-fold in generation times observed for an S. cerevisiae trm7Δ mutant compared with its wild-type control. Furthermore, this difference was even greater in selective EMM-Leu + thiamine, with a generation time of 603 min for the Sp trm7Δ [Pnmt1** Sp trm7+
LEU2] strain, compared with 238 min for the wild-type control strain (a difference of 2.53-fold), whereas the Sp trm7Δ [Pnmt1** Sp trm7+
LEU2] strain grew nearly as well as the wild-type strain in permissive conditions (EMM-Leu), with a difference in generation times of only 1.05-fold (Table 1).FIGURE 2.S. pombe trm7Δ mutants are barely viable, and trm7Δ [Pnmt1** Sp trm7+] strains grown in repressive conditions have reduced Cm32, Gm34, and yW37 modification of tRNAPhe. (A) S. pombe trm7▵ mutants are barely viable. S. pombe strains with the indicated plasmids were grown in EMM overnight, diluted to OD600 of ∼0.5 in H2O, and serially diluted 10-fold in H2O, and then 2 µL was spotted onto indicated media, followed by incubation for 4 d at 30°C. (B) HPLC traces of tRNAPhe from an S. pombe trm7Δ [Pnmt1** Sp trm7+] strain. tRNAPhe from indicated strains grown in EMM-Leu with thiamine (+T, repressive conditions) or without thiamine (permissive conditions) was digested to nucleosides and analyzed by HPLC as described in Materials and Methods. (C) Levels of yW on tRNAPhe substantially decrease in an S. pombe trm7Δ [Pnmt1** Sp trm7+] strain grown under repressive conditions. tRNAPhe from indicated strains grown in EMM-Leu with thiamine (+T) was analyzed as described in panel B under conditions optimized for detecting yW as described in Materials and Methods.TABLE 1.Comparison of generation times for trm7▵, trm732▵, and trm734▵ mutant strains from S. pombe and S. cerevisiae at 30°CBecause it was extremely difficult to obtain and grow an Sp trm7Δ strain for analysis of modifications, we instead examined the effect of reduced levels of Sp Trm7 on modification of tRNAPhe by growing the Sp trm7Δ [Pnmt1** Sp trm7+
LEU2] strain in EMM-Leu containing thiamine to repress Sp Trm7 expression, followed by purification of tRNAPhe and analysis of its nucleoside content by HPLC. Under these conditions, Cm levels of tRNAPhe were substantially reduced compared with those of a wild-type control strain grown in the same medium (0.13 versus 0.93 moles/mole) (Table 2; Fig. 2B), or to those when the Sp trm7Δ [Pnmt1** Sp trm7+
LEU2] strain was grown under permissive (EMM-Leu) conditions (0.90 moles/mole). Similarly, Gm levels were reduced in tRNAPhe from the Sp trm7Δ [Pnmt1** Sp trm7+
LEU2] strain grown in repressive conditions (EMM-Leu containing thiamine) relative to those from the wild-type strain grown under the same conditions (0.04 versus 0.99 moles/mole), or to those when the strain was grown under permissive (EMM-Leu) conditions (0.90 moles/mole). In contrast, the control modification Ψ (pseudouridine) showed little variation in these strains (Table 2).TABLE 2.HPLC analysis of tRNAPhe nucleoside content from an S. pombe trm7▵ strainWe also found evidence that yW modification was reduced upon repression of trm7+ expression in the Sp trm7Δ [Pnmt1** Sp trm7+
LEU2] strain by growth in thiamine-containing medium. Under these conditions, m1G levels of tRNAPhe increased substantially compared with those from the wild-type strain (1.80 versus 1.16 moles/mole), suggesting that yW formation from m1G was reduced (Table 2; Fig. 2B). Direct measurement by HPLC (Noma et al. 2006) confirmed that levels of yW were reduced in tRNAPhe from the Sp trm7Δ [Pnmt1** Sp trm7+
LEU2] strain grown under repressive conditions, to 20% of those from tRNAPhe of wild-type cells (Fig. 2C). Thus, just as in S. cerevisiae (Guy et al. 2012), S. pombe mutants lacking Cm32 and Gm34 due to loss of trm7+ also appear to have reduced synthesis of yW from m1G in tRNAPhe.Curiously, levels of the control modifications m2G (N2-methylguanosine) and m7G (7-methylguanosine) were unexpectedly reduced in tRNAPhe purified from different strains grown in EMM. m7G was reproducibly and significantly reduced whenever thiamine was missing from the medium, from 0.55 to 0.28 moles/mole for the wild-type strain, and from 0.46 to 0.25 moles/mole for the Sp trm7Δ [Pnmt1** Sp trm7+
LEU2] strain (Table 2; Fig. 2B). m2G levels in tRNAPhe were reproducibly different in different strains, with high levels in the Sp trm7Δ [Pnmt1** Sp trm7+
LEU2] strain in the presence of thiamine (0.75 moles/mole), and reduced levels when thiamine was absent (0.12 moles/mole), and with significantly lower levels in the wild-type strain (0.29 moles/mole), further reduced in the absence of thiamine (0.11 moles/mole).Overexpression of tRNAPhe suppresses the growth defect of S. pombe trm7Δ mutantsWe also found that tRNAPhe was the important Trm7 substrate in S. pombe, since overexpression of this tRNA restored healthy growth to Sp trm7Δ mutants on 5-FOA medium, whereas overexpression of tRNATrp or tRNALeu(UAA), which were not 2′-O-methylated in S. pombe (data not shown), did not (Fig. 3A). Indeed, the Sp trm7Δ mutant overexpressing tRNAPhe grew nearly as well as the wild-type control strain on EMM at 25°C and 30°C (and a bit more poorly on YES medium), but was slightly temperature sensitive at 33°C and 37°C (Fig. 3B). As expected for a strain lacking Trm7, tRNAPhe from the Sp trm7Δ [tRNAPhe] strain lacked detectable Cm and Gm (Table 2), had increased m1G levels compared with those from the wild-type strain overexpressing tRNAPhe (2.18 versus 1.62 moles/mole), and had undectable levels of yW (Fig. 3C). Thus, the important Trm7 substrate in S. pombe is tRNAPhe, just as in S. cerevisiae (Guy et al. 2012).FIGURE 3.The biologically important Sp Trm7 tRNA substrate is tRNAPhe. (A) Overexpression of tRNAPhe suppresses the near lethality of S. pombe trm7▵ mutants. Strains with plasmids as indicated were grown overnight in EMM-Leu and analyzed as in Figure 2A, after incubation for 3 d at 30°C. (B) Overexpression of tRNAPhe in S. pombe trm7▵ mutants does not completely restore wild-type growth at 37°C. Strains as indicated were purified on 5-FOA to ensure loss of the [Pnmt1** Sp trm7+
ura4+] plasmid, grown overnight in EMM-Leu, and then growth was analyzed as indicated in Figure 2A. (C) tRNAPhe from the S. pombe trm7▵ [tRNAPhe] strain lacks yW. tRNAPhe purified from the indicated strains after growth in EMM-Leu was analyzed as described in Figure 2C.The putative S. pombe trm732+ and trm734+ genes are required for Cm and Gm formation, respectively, on tRNAPheTo further determine if the modification circuitry for the anticodon loop of tRNAPhe is conserved between S. cerevisiae and S. pombe, we analyzed the putative S. pombe trm732▵ (SPCC1494.07) and trm734▵ (SPBC1306.02) strains. An Sp trm732▵ haploid mutant obtained by sporulation of the heterozygous diploid lacked detectable Cm in its tRNAPhe and had normal levels of Gm compared with that from wild type (0.88 versus 0.90 moles/mole) (Table 3; Fig. 4A) and did not have any obvious growth defect on plates or in liquid YES medium (Table 1; Fig. 4B), similar to the lack of an obvious growth defect observed for the Sc trm732▵ mutant (Table 1; Guy et al. 2012).FIGURE 4.S. pombe trm734▵ mutants are sick due to hypomodified tRNAPhe. (A) HPLC traces of tRNAPhe from S. pombe trm732▵ and S. pombe trm734▵ mutants. tRNAPhe was purified from the indicated strains grown in YES medium and then digested to nucleosides and analyzed by HPLC. (B) S. pombe trm732▵ mutants are healthy. Indicated strains were grown in YES medium overnight, and analyzed on YES plates as described in Figure 2A. (C) S. pombe trm734▵ mutants have a growth defect on EMM. Indicated strains were grown overnight in EMM, diluted, and plated on EMM as indicated. (D) S. pombe trm734▵ mutants have a substantial growth defect on rich medium at various temperatures. Indicated strains were transformed with indicated ura4+ plasmids, grown in EMM-Ura, diluted, and plated on YES medium at the indicated temperatures after 3 d. (E) Overexpression of tRNAPhe suppresses the slow growth of S. pombe trm734▵ mutants. Strains with plasmids as indicated were grown in EMM-Leu overnight, and analyzed as in Figure 2A, after incubation for 3 d at 30°C. Strains were then purified on medium containing 5-FOA, grown overnight in EMM-Leu, diluted, spotted, and analyzed at the indicated temperatures on YES medium. (F) Levels of yW in tRNAPhe are decreased in S. pombe trm732▵ and S. pombe trm734▵ mutants. tRNAPhe was purified from the indicated strains after growth in YES medium and analyzed as described in Figure 2C.TABLE 3.HPLC analysis of tRNAPhe nucleoside content from S. pombe trm732▵ and S. pombe trm734▵ strainsHowever, an Sp trm734▵ [Sp trm734+
ura4+] haploid obtained by sporulation of the heterozygous diploid containing the plasmid grew poorly when plated on EMM containing 5-FOA (Fig. 4C), and the resulting Sp trm734▵ strain lacked Gm34 in its tRNAPhe, and had Cm levels comparable to those of wild type (0.86 versus 0.91 moles/mole) (Table 3; Fig. 4A). The Sp trm734▵ mutant had a significant growth defect on YES medium on plates or in liquid (generation time of 258 min, compared with 148 min for wild type) (Table 1; Fig. 4D), whereas the Sc trm734▵ mutant lacked any obvious growth defect on plates or in liquid medium (generation time of 70 min, compared with 66 min for wild type) (Table 1; Guy et al. 2012), demonstrating that Trm734 has a more prominent role in S. pombe than in S. cerevisiae. Furthermore, since overexpression of tRNAPhe nearly completely suppressed the slow growth phenotype of the Sp trm734▵ strain over a range of temperatures (Fig. 4E), we conclude that lack of Gm34 of tRNAPhe is the cause of the Sp trm734▵ defect.In addition, our analysis showed that, as in S. cerevisiae (Guy et al. 2012), yW formation was impaired in the Sp trm734▵ mutant (44% of wild-type levels) and to a lesser extent in the Sp trm732▵ mutant (73%) (Fig. 4F), consistent with the increased m1G levels (Table 3; Fig. 4A). We also note that m2G levels on tRNAPhe are substantially higher in the Sp trm734▵ mutant compared with those from a wild-type strain when grown in EMM (0.61 versus 0.19 moles/mole).FTSJ1 is the human Trm7 ortholog and requires S. cerevisiae Trm732 to catalyze Cm32 modification on tRNAPhe in S. cerevisiaeTo investigate the conservation of function of metazoan genes involved in 2′-O-methylation of the tRNA anticodon loop, we examined complementation of the corresponding S. cerevisiae mutants by introduction of appropriate constructs expressing metazoan TRM7 and/or its metazoan TRM732 and TRM734 partners. To identify TRM7 homologs, we examined Sp trm7+ and predicted TRM7 orthologs from D. melanogaster and humans by introduction of a high copy [2µ LEU2 PGAL TRM7] plasmid expressing the corresponding ortholog under galactose control to an Sc trm7Δ [Sc TRM7 URA3] strain, followed by analysis of growth after plating on medium containing 5-FOA and galactose. We found that the slow growth phenotype of Sc trm7Δ mutants was efficiently suppressed by expression of Sp Trm7 (∼61% identical to Sc Trm7), D. melanogaster ORF CG5220 (Dm TRM7, ∼43% identical to Sc Trm7), and human FTSJ1 (∼50% identical to Sc Trm7) but not human FTSJ2 (∼34% identical to Sc Trm7) (Fig. 5A), after selection on medium containing 5-FOA. Furthermore, the resulting Sc trm7Δ [2µ LEU2 PGAL FTSJ1] strain grew nearly as well as the wild-type S. cerevisiae strain (Fig. 5A, right panel), strongly suggesting that FTSJ1 is the human ortholog of Trm7.FIGURE 5.FTSJ1 is the human ortholog of S. cerevisiae Trm7, and its function in S. cerevisiae requires Trm732 for Cm32 modification. (A) Expression of S. pombe trm7+, D. melanogaster TRM7, or human FTSJ1 suppresses the slow growth S. cerevisiae trm7▵ mutants. Strains with plasmids as indicated were grown overnight in S-Leu medium containing raffinose and galactose, diluted to OD600 of ∼0.5 in H2O, and serially diluted 10-fold in H2O, and then 2 µL was spotted onto S medium containing 5-FOA, raffinose, and galactose, followed by incubation for 3 d at 30°C. (Right panel) Selected strains were then purified on medium containing 5-FOA, grown overnight in S-Leu medium containing raffinose and galactose, and analyzed by spotting to YP medium containing raffinose and galactose. (B) FTSJ1 requires Sc TRM732 to suppress the slow growth of Sc trm7▵ mutants. Strains were analyzed as in A.The ability of human FTSJ1 and the other tested TRM7 orthologs to complement the growth defect of an S. cerevisiae trm7Δ mutant strain was surprising because Sc Trm7 and Sp Trm7 have a strict requirement for their corresponding Trm732 and Trm734 partner proteins to generate Cm32 and Nm34, respectively (Guy et al. 2012). Because tRNAPhe is the Sc Trm7 substrate that must be modified for healthy growth in S. cerevisiae (Guy et al. 2012), we examined 2′-O-methylation of tRNAPhe from the Sc trm7Δ [2µ LEU2 PGAL FTSJ1] strain. We found that tRNAPhe purified from this strain had high levels of Cm (0.89 versus 0.90 moles/mole in wild type), but no detectable Gm (Table 4), suggesting that human FTSJ1 is working in concert with Sc Trm732 to modify tRNAPhe. The healthy growth of this Sc trm7Δ [2µ LEU2 PGAL FTSJ1] strain is consistent with our previous observation that the Cm32 modification of tRNAPhe is sufficient for healthy growth in S. cerevisiae (Guy et al. 2012).TABLE 4.HPLC analysis of tRNAPhe nucleoside content from S. cerevisiae trm7▵ [FTSJ1] and S. cerevisiae trm7▵ trm732▵ [FTSJ1 THADA] strainsThe occurrence of Cm in tRNAPhe from an Sc trm7Δ [2µ LEU2 PGAL FTSJ1] strain suggested either that FTSJ1 does not require an interacting partner for Cm32 catalysis, or that it works with S. cerevisiae Trm732. Since the slow growth phenotype of the Sc trm7Δ trm732Δ strain was not suppressed by introduction of the [2µ LEU2 PGAL FTSJ1] plasmid and no Cm formation was observed on tRNAPhe purified from this strain (Table 4; Fig. 5B), we infer that FTSJ1 works with Sc Trm732 for formation of Cm32 on tRNAPhe. Since tRNALeu(UAA) from an Sc trm7Δ [2µ LEU2 PGAL FTSJ1] strain lacked both Cm and ncm5Um (Table 5; Fig. 1A), the activity of FTSJ1 with Trm732 in S. cerevisiae does not extend to all Sc Trm7 substrates.TABLE 5.HPLC analysis of tRNALeu(UAA) nucleoside content from an S. cerevisiae trm7▵ [FTSJ1] strainWe note that because FTSJ1 is expressed from a high copy PGAL plasmid in these experiments, it is formally possible that FTSJ1 is not the true Trm7 homolog. However, this seems unlikely due to the high sequence similarity between FTSJ1 and Trm7, and the requirement of Sc Trm732 function for FTSJ1 activity in S. cerevisiae.THADA is the human homolog of Trm732 and works with ScTrm7 for Cm32 modificationAlthough Trm732 homologs have only ∼20% sequence identity, we find that the putative human Trm732 homolog THADA functions efficiently to replace Sc Trm732. Thus, the slow growth of an Sc trm7Δ trm732Δ strain was suppressed by expression of FTSJ1 and THADA, but not by FTSJ1 alone (Fig. 6A). Since tRNAPhe from this Sc trm7Δ trm732Δ [2µ PGAL
FTSJ1 PGAL
THADA] strain had identical Cm levels to that from the wild-type strain (0.90 moles/mole) (Table 4) and no detectable Gm, and since tRNALeu(UAA) from this strain had substantial Cm modification (0.36 moles/mole) and no ncm5Um (Table 5), we conclude that THADA is the human Trm732 homolog and functionally interacts with FTSJ1. The ability of FTSJ1 to modify C32 of tRNALeu(UAA) in combination with THADA but not with Sc Trm732 may be due to higher levels of THADA relative to endogenous levels of yeast Trm732, and/or more efficient partnering of FTSJ1 and THADA.FIGURE 6.S. cerevisiae Trm7 works with human THADA or S. pombe Trm732 for function. (A) THADA is the human ortholog of S. cerevisiae TRM7. Strains with plasmids as indicated were analyzed as in Figure 5A. (B) S. pombe trm732+ suppresses the slow growth of S. cerevisiae trm732▵ trm734▵ mutants. Strains were analyzed as in Figure 5A.We also find evidence that S. cerevisiae Trm7 can interact with human THADA for Cm32 formation, since expression of THADA suppressed the slow growth of an Sc trm732Δ trm734Δ strain (Fig. 6A), resulting in tRNAPhe with 0.76 moles/mole Cm (Table 4). Thus, our data demonstrate that human (FTSJ1) or S. cerevisiae Trm7 can each utilize either human (THADA) or S. cerevisiae Trm732 to catalyze formation of Cm32 on tRNAPhe in S. cerevisiae. S. cerevisiae Trm7 also appears to work with S. pombe Trm732, because expression of Sp trm732+ suppressed the slow growth of the Sc trm732Δ trm734Δ strain (Fig. 6B).

The putative S. pombe trm7+ gene is required for Cm and Gm formation on tRNAPhe and mutants are barely viableTo determine if S. cerevisiae Trm7 function is conserved in eukaryotes, we generated and examined S. pombe strains lacking the likely trm7+ gene (SPAC4F10.03c). Consistent with previous high throughput results (Kim et al. 2010), we were unable to generate S. pombe trm7Δ mutant haploids from a heterozygous diploid (relevant genotype: spac4F10.03cΔ::kanMX/SPAC4F10.03c) by sporulation and selection for the mutant haploid (data not shown). However, by first introducing a ura4+ plasmid expressing S. pombe trm7+ under control of the Pnmt1 promoter (no message in thiamine), we could recover an Sp trm7Δ [Pnmt1 Sp trm7+
ura4+] haploid after sporulation on Edinburgh minimal medium (EMM, permissive conditions) in the presence of selective drug; moreover, this Sp trm7Δ [Pnmt1 Sp trm7+
ura4+] haploid was viable (albeit barely) when plated to EMM containing 5-fluoroorotic acid (5-FOA) to select against the ura4+ plasmid. Indeed, after streaking the Sp trm7Δ [Pnmt1 Sp trm7+
ura4+] strain to EMM containing 0.5 mg/L 5-FOA at 30°C, it took 11 d of growth to attain visible colonies, compared with 3 d for wild-type cells (data not shown). Control experiments demonstrated that the growth defect was due to the trm7Δ mutation of the Sp trm7Δ [Pnmt1 Sp trm7+
ura4+] strain, since introduction of a [Pnmt1** Sp trm7+
LEU2] plasmid (expressing trm7+ under control of the low strength no message in thiamine promoter), resulted in healthy growth after plating to EMM containing 5-FOA (Fig. 2A). The growth defect of Sp trm7Δ mutants thus appears to be more severe than that observed in the S. cerevisiae trm7Δ mutant, which is sick but viable (Pintard et al. 2002; Guy et al. 2012). Under repressive conditions in liquid YES medium, the Sp trm7Δ [Pnmt1** Sp trm7+
LEU2] strain (although much healthier than the Sp trm7Δ haploid) had a generation time of 320 min, compared with 148 min for the wild-type strain (a difference of 2.16-fold) (Table 1), which was still more than the difference of 1.94-fold in generation times observed for an S. cerevisiae trm7Δ mutant compared with its wild-type control. Furthermore, this difference was even greater in selective EMM-Leu + thiamine, with a generation time of 603 min for the Sp trm7Δ [Pnmt1** Sp trm7+
LEU2] strain, compared with 238 min for the wild-type control strain (a difference of 2.53-fold), whereas the Sp trm7Δ [Pnmt1** Sp trm7+
LEU2] strain grew nearly as well as the wild-type strain in permissive conditions (EMM-Leu), with a difference in generation times of only 1.05-fold (Table 1).FIGURE 2.S. pombe trm7Δ mutants are barely viable, and trm7Δ [Pnmt1** Sp trm7+] strains grown in repressive conditions have reduced Cm32, Gm34, and yW37 modification of tRNAPhe. (A) S. pombe trm7▵ mutants are barely viable. S. pombe strains with the indicated plasmids were grown in EMM overnight, diluted to OD600 of ∼0.5 in H2O, and serially diluted 10-fold in H2O, and then 2 µL was spotted onto indicated media, followed by incubation for 4 d at 30°C. (B) HPLC traces of tRNAPhe from an S. pombe trm7Δ [Pnmt1** Sp trm7+] strain. tRNAPhe from indicated strains grown in EMM-Leu with thiamine (+T, repressive conditions) or without thiamine (permissive conditions) was digested to nucleosides and analyzed by HPLC as described in Materials and Methods. (C) Levels of yW on tRNAPhe substantially decrease in an S. pombe trm7Δ [Pnmt1** Sp trm7+] strain grown under repressive conditions. tRNAPhe from indicated strains grown in EMM-Leu with thiamine (+T) was analyzed as described in panel B under conditions optimized for detecting yW as described in Materials and Methods.TABLE 1.Comparison of generation times for trm7▵, trm732▵, and trm734▵ mutant strains from S. pombe and S. cerevisiae at 30°CBecause it was extremely difficult to obtain and grow an Sp trm7Δ strain for analysis of modifications, we instead examined the effect of reduced levels of Sp Trm7 on modification of tRNAPhe by growing the Sp trm7Δ [Pnmt1** Sp trm7+
LEU2] strain in EMM-Leu containing thiamine to repress Sp Trm7 expression, followed by purification of tRNAPhe and analysis of its nucleoside content by HPLC. Under these conditions, Cm levels of tRNAPhe were substantially reduced compared with those of a wild-type control strain grown in the same medium (0.13 versus 0.93 moles/mole) (Table 2; Fig. 2B), or to those when the Sp trm7Δ [Pnmt1** Sp trm7+
LEU2] strain was grown under permissive (EMM-Leu) conditions (0.90 moles/mole). Similarly, Gm levels were reduced in tRNAPhe from the Sp trm7Δ [Pnmt1** Sp trm7+
LEU2] strain grown in repressive conditions (EMM-Leu containing thiamine) relative to those from the wild-type strain grown under the same conditions (0.04 versus 0.99 moles/mole), or to those when the strain was grown under permissive (EMM-Leu) conditions (0.90 moles/mole). In contrast, the control modification Ψ (pseudouridine) showed little variation in these strains (Table 2).TABLE 2.HPLC analysis of tRNAPhe nucleoside content from an S. pombe trm7▵ strainWe also found evidence that yW modification was reduced upon repression of trm7+ expression in the Sp trm7Δ [Pnmt1** Sp trm7+
LEU2] strain by growth in thiamine-containing medium. Under these conditions, m1G levels of tRNAPhe increased substantially compared with those from the wild-type strain (1.80 versus 1.16 moles/mole), suggesting that yW formation from m1G was reduced (Table 2; Fig. 2B). Direct measurement by HPLC (Noma et al. 2006) confirmed that levels of yW were reduced in tRNAPhe from the Sp trm7Δ [Pnmt1** Sp trm7+
LEU2] strain grown under repressive conditions, to 20% of those from tRNAPhe of wild-type cells (Fig. 2C). Thus, just as in S. cerevisiae (Guy et al. 2012), S. pombe mutants lacking Cm32 and Gm34 due to loss of trm7+ also appear to have reduced synthesis of yW from m1G in tRNAPhe.Curiously, levels of the control modifications m2G (N2-methylguanosine) and m7G (7-methylguanosine) were unexpectedly reduced in tRNAPhe purified from different strains grown in EMM. m7G was reproducibly and significantly reduced whenever thiamine was missing from the medium, from 0.55 to 0.28 moles/mole for the wild-type strain, and from 0.46 to 0.25 moles/mole for the Sp trm7Δ [Pnmt1** Sp trm7+
LEU2] strain (Table 2; Fig. 2B). m2G levels in tRNAPhe were reproducibly different in different strains, with high levels in the Sp trm7Δ [Pnmt1** Sp trm7+
LEU2] strain in the presence of thiamine (0.75 moles/mole), and reduced levels when thiamine was absent (0.12 moles/mole), and with significantly lower levels in the wild-type strain (0.29 moles/mole), further reduced in the absence of thiamine (0.11 moles/mole).

The putative S. pombe trm7+ gene is required for Cm and Gm formation on tRNAPhe and mutants are barely viable

To determine if S. cerevisiae Trm7 function is conserved in eukaryotes, we generated and examined S. pombe strains lacking the likely trm7+ gene (SPAC4F10.03c). Consistent with previous high throughput results (Kim et al. 2010), we were unable to generate S. pombe trm7Δ mutant haploids from a heterozygous diploid (relevant genotype: spac4F10.03cΔ::kanMX/SPAC4F10.03c) by sporulation and selection for the mutant haploid (data not shown). However, by first introducing a ura4+ plasmid expressing S. pombe trm7+ under control of the Pnmt1 promoter (no message in thiamine), we could recover an Sp trm7Δ [Pnmt1 Sp trm7+
ura4+] haploid after sporulation on Edinburgh minimal medium (EMM, permissive conditions) in the presence of selective drug; moreover, this Sp trm7Δ [Pnmt1 Sp trm7+
ura4+] haploid was viable (albeit barely) when plated to EMM containing 5-fluoroorotic acid (5-FOA) to select against the ura4+ plasmid. Indeed, after streaking the Sp trm7Δ [Pnmt1 Sp trm7+
ura4+] strain to EMM containing 0.5 mg/L 5-FOA at 30°C, it took 11 d of growth to attain visible colonies, compared with 3 d for wild-type cells (data not shown). Control experiments demonstrated that the growth defect was due to the trm7Δ mutation of the Sp trm7Δ [Pnmt1 Sp trm7+
ura4+] strain, since introduction of a [Pnmt1** Sp trm7+
LEU2] plasmid (expressing trm7+ under control of the low strength no message in thiamine promoter), resulted in healthy growth after plating to EMM containing 5-FOA (Fig. 2A). The growth defect of Sp trm7Δ mutants thus appears to be more severe than that observed in the S. cerevisiae trm7Δ mutant, which is sick but viable (Pintard et al. 2002; Guy et al. 2012). Under repressive conditions in liquid YES medium, the Sp trm7Δ [Pnmt1** Sp trm7+
LEU2] strain (although much healthier than the Sp trm7Δ haploid) had a generation time of 320 min, compared with 148 min for the wild-type strain (a difference of 2.16-fold) (Table 1), which was still more than the difference of 1.94-fold in generation times observed for an S. cerevisiae trm7Δ mutant compared with its wild-type control. Furthermore, this difference was even greater in selective EMM-Leu + thiamine, with a generation time of 603 min for the Sp trm7Δ [Pnmt1** Sp trm7+
LEU2] strain, compared with 238 min for the wild-type control strain (a difference of 2.53-fold), whereas the Sp trm7Δ [Pnmt1** Sp trm7+
LEU2] strain grew nearly as well as the wild-type strain in permissive conditions (EMM-Leu), with a difference in generation times of only 1.05-fold (Table 1).

S. pombe trm7Δ mutants are barely viable, and trm7Δ [Pnmt1** Sp trm7+] strains grown in repressive conditions have reduced Cm32, Gm34, and yW37 modification of tRNAPhe. (A) S. pombe trm7▵ mutants are barely viable. S. pombe strains with the indicated plasmids were grown in EMM overnight, diluted to OD600 of ∼0.5 in H2O, and serially diluted 10-fold in H2O, and then 2 µL was spotted onto indicated media, followed by incubation for 4 d at 30°C. (B) HPLC traces of tRNAPhe from an S. pombe trm7Δ [Pnmt1** Sp trm7+] strain. tRNAPhe from indicated strains grown in EMM-Leu with thiamine (+T, repressive conditions) or without thiamine (permissive conditions) was digested to nucleosides and analyzed by HPLC as described in Materials and Methods. (C) Levels of yW on tRNAPhe substantially decrease in an S. pombe trm7Δ [Pnmt1** Sp trm7+] strain grown under repressive conditions. tRNAPhe from indicated strains grown in EMM-Leu with thiamine (+T) was analyzed as described in panel B under conditions optimized for detecting yW as described in Materials and Methods.

Comparison of generation times for trm7▵, trm732▵, and trm734▵ mutant strains from S. pombe and S. cerevisiae at 30°C

Because it was extremely difficult to obtain and grow an Sp trm7Δ strain for analysis of modifications, we instead examined the effect of reduced levels of Sp Trm7 on modification of tRNAPhe by growing the Sp trm7Δ [Pnmt1** Sp trm7+
LEU2] strain in EMM-Leu containing thiamine to repress Sp Trm7 expression, followed by purification of tRNAPhe and analysis of its nucleoside content by HPLC. Under these conditions, Cm levels of tRNAPhe were substantially reduced compared with those of a wild-type control strain grown in the same medium (0.13 versus 0.93 moles/mole) (Table 2; Fig. 2B), or to those when the Sp trm7Δ [Pnmt1** Sp trm7+
LEU2] strain was grown under permissive (EMM-Leu) conditions (0.90 moles/mole). Similarly, Gm levels were reduced in tRNAPhe from the Sp trm7Δ [Pnmt1** Sp trm7+
LEU2] strain grown in repressive conditions (EMM-Leu containing thiamine) relative to those from the wild-type strain grown under the same conditions (0.04 versus 0.99 moles/mole), or to those when the strain was grown under permissive (EMM-Leu) conditions (0.90 moles/mole). In contrast, the control modification Ψ (pseudouridine) showed little variation in these strains (Table 2).

HPLC analysis of tRNAPhe nucleoside content from an S. pombe trm7▵ strain

We also found evidence that yW modification was reduced upon repression of trm7+ expression in the Sp trm7Δ [Pnmt1** Sp trm7+
LEU2] strain by growth in thiamine-containing medium. Under these conditions, m1G levels of tRNAPhe increased substantially compared with those from the wild-type strain (1.80 versus 1.16 moles/mole), suggesting that yW formation from m1G was reduced (Table 2; Fig. 2B). Direct measurement by HPLC (Noma et al. 2006) confirmed that levels of yW were reduced in tRNAPhe from the Sp trm7Δ [Pnmt1** Sp trm7+
LEU2] strain grown under repressive conditions, to 20% of those from tRNAPhe of wild-type cells (Fig. 2C). Thus, just as in S. cerevisiae (Guy et al. 2012), S. pombe mutants lacking Cm32 and Gm34 due to loss of trm7+ also appear to have reduced synthesis of yW from m1G in tRNAPhe.

Curiously, levels of the control modifications m2G (N2-methylguanosine) and m7G (7-methylguanosine) were unexpectedly reduced in tRNAPhe purified from different strains grown in EMM. m7G was reproducibly and significantly reduced whenever thiamine was missing from the medium, from 0.55 to 0.28 moles/mole for the wild-type strain, and from 0.46 to 0.25 moles/mole for the Sp trm7Δ [Pnmt1** Sp trm7+
LEU2] strain (Table 2; Fig. 2B). m2G levels in tRNAPhe were reproducibly different in different strains, with high levels in the Sp trm7Δ [Pnmt1** Sp trm7+
LEU2] strain in the presence of thiamine (0.75 moles/mole), and reduced levels when thiamine was absent (0.12 moles/mole), and with significantly lower levels in the wild-type strain (0.29 moles/mole), further reduced in the absence of thiamine (0.11 moles/mole).

Overexpression of tRNAPhe suppresses the growth defect of S. pombe trm7Δ mutantsWe also found that tRNAPhe was the important Trm7 substrate in S. pombe, since overexpression of this tRNA restored healthy growth to Sp trm7Δ mutants on 5-FOA medium, whereas overexpression of tRNATrp or tRNALeu(UAA), which were not 2′-O-methylated in S. pombe (data not shown), did not (Fig. 3A). Indeed, the Sp trm7Δ mutant overexpressing tRNAPhe grew nearly as well as the wild-type control strain on EMM at 25°C and 30°C (and a bit more poorly on YES medium), but was slightly temperature sensitive at 33°C and 37°C (Fig. 3B). As expected for a strain lacking Trm7, tRNAPhe from the Sp trm7Δ [tRNAPhe] strain lacked detectable Cm and Gm (Table 2), had increased m1G levels compared with those from the wild-type strain overexpressing tRNAPhe (2.18 versus 1.62 moles/mole), and had undectable levels of yW (Fig. 3C). Thus, the important Trm7 substrate in S. pombe is tRNAPhe, just as in S. cerevisiae (Guy et al. 2012).FIGURE 3.The biologically important Sp Trm7 tRNA substrate is tRNAPhe. (A) Overexpression of tRNAPhe suppresses the near lethality of S. pombe trm7▵ mutants. Strains with plasmids as indicated were grown overnight in EMM-Leu and analyzed as in Figure 2A, after incubation for 3 d at 30°C. (B) Overexpression of tRNAPhe in S. pombe trm7▵ mutants does not completely restore wild-type growth at 37°C. Strains as indicated were purified on 5-FOA to ensure loss of the [Pnmt1** Sp trm7+
ura4+] plasmid, grown overnight in EMM-Leu, and then growth was analyzed as indicated in Figure 2A. (C) tRNAPhe from the S. pombe trm7▵ [tRNAPhe] strain lacks yW. tRNAPhe purified from the indicated strains after growth in EMM-Leu was analyzed as described in Figure 2C.

Overexpression of tRNAPhe suppresses the growth defect of S. pombe trm7Δ mutants

We also found that tRNAPhe was the important Trm7 substrate in S. pombe, since overexpression of this tRNA restored healthy growth to Sp trm7Δ mutants on 5-FOA medium, whereas overexpression of tRNATrp or tRNALeu(UAA), which were not 2′-O-methylated in S. pombe (data not shown), did not (Fig. 3A). Indeed, the Sp trm7Δ mutant overexpressing tRNAPhe grew nearly as well as the wild-type control strain on EMM at 25°C and 30°C (and a bit more poorly on YES medium), but was slightly temperature sensitive at 33°C and 37°C (Fig. 3B). As expected for a strain lacking Trm7, tRNAPhe from the Sp trm7Δ [tRNAPhe] strain lacked detectable Cm and Gm (Table 2), had increased m1G levels compared with those from the wild-type strain overexpressing tRNAPhe (2.18 versus 1.62 moles/mole), and had undectable levels of yW (Fig. 3C). Thus, the important Trm7 substrate in S. pombe is tRNAPhe, just as in S. cerevisiae (Guy et al. 2012).

The biologically important Sp Trm7 tRNA substrate is tRNAPhe. (A) Overexpression of tRNAPhe suppresses the near lethality of S. pombe trm7▵ mutants. Strains with plasmids as indicated were grown overnight in EMM-Leu and analyzed as in Figure 2A, after incubation for 3 d at 30°C. (B) Overexpression of tRNAPhe in S. pombe trm7▵ mutants does not completely restore wild-type growth at 37°C. Strains as indicated were purified on 5-FOA to ensure loss of the [Pnmt1** Sp trm7+
ura4+] plasmid, grown overnight in EMM-Leu, and then growth was analyzed as indicated in Figure 2A. (C) tRNAPhe from the S. pombe trm7▵ [tRNAPhe] strain lacks yW. tRNAPhe purified from the indicated strains after growth in EMM-Leu was analyzed as described in Figure 2C.

The putative S. pombe trm732+ and trm734+ genes are required for Cm and Gm formation, respectively, on tRNAPheTo further determine if the modification circuitry for the anticodon loop of tRNAPhe is conserved between S. cerevisiae and S. pombe, we analyzed the putative S. pombe trm732▵ (SPCC1494.07) and trm734▵ (SPBC1306.02) strains. An Sp trm732▵ haploid mutant obtained by sporulation of the heterozygous diploid lacked detectable Cm in its tRNAPhe and had normal levels of Gm compared with that from wild type (0.88 versus 0.90 moles/mole) (Table 3; Fig. 4A) and did not have any obvious growth defect on plates or in liquid YES medium (Table 1; Fig. 4B), similar to the lack of an obvious growth defect observed for the Sc trm732▵ mutant (Table 1; Guy et al. 2012).FIGURE 4.S. pombe trm734▵ mutants are sick due to hypomodified tRNAPhe. (A) HPLC traces of tRNAPhe from S. pombe trm732▵ and S. pombe trm734▵ mutants. tRNAPhe was purified from the indicated strains grown in YES medium and then digested to nucleosides and analyzed by HPLC. (B) S. pombe trm732▵ mutants are healthy. Indicated strains were grown in YES medium overnight, and analyzed on YES plates as described in Figure 2A. (C) S. pombe trm734▵ mutants have a growth defect on EMM. Indicated strains were grown overnight in EMM, diluted, and plated on EMM as indicated. (D) S. pombe trm734▵ mutants have a substantial growth defect on rich medium at various temperatures. Indicated strains were transformed with indicated ura4+ plasmids, grown in EMM-Ura, diluted, and plated on YES medium at the indicated temperatures after 3 d. (E) Overexpression of tRNAPhe suppresses the slow growth of S. pombe trm734▵ mutants. Strains with plasmids as indicated were grown in EMM-Leu overnight, and analyzed as in Figure 2A, after incubation for 3 d at 30°C. Strains were then purified on medium containing 5-FOA, grown overnight in EMM-Leu, diluted, spotted, and analyzed at the indicated temperatures on YES medium. (F) Levels of yW in tRNAPhe are decreased in S. pombe trm732▵ and S. pombe trm734▵ mutants. tRNAPhe was purified from the indicated strains after growth in YES medium and analyzed as described in Figure 2C.TABLE 3.HPLC analysis of tRNAPhe nucleoside content from S. pombe trm732▵ and S. pombe trm734▵ strainsHowever, an Sp trm734▵ [Sp trm734+
ura4+] haploid obtained by sporulation of the heterozygous diploid containing the plasmid grew poorly when plated on EMM containing 5-FOA (Fig. 4C), and the resulting Sp trm734▵ strain lacked Gm34 in its tRNAPhe, and had Cm levels comparable to those of wild type (0.86 versus 0.91 moles/mole) (Table 3; Fig. 4A). The Sp trm734▵ mutant had a significant growth defect on YES medium on plates or in liquid (generation time of 258 min, compared with 148 min for wild type) (Table 1; Fig. 4D), whereas the Sc trm734▵ mutant lacked any obvious growth defect on plates or in liquid medium (generation time of 70 min, compared with 66 min for wild type) (Table 1; Guy et al. 2012), demonstrating that Trm734 has a more prominent role in S. pombe than in S. cerevisiae. Furthermore, since overexpression of tRNAPhe nearly completely suppressed the slow growth phenotype of the Sp trm734▵ strain over a range of temperatures (Fig. 4E), we conclude that lack of Gm34 of tRNAPhe is the cause of the Sp trm734▵ defect.In addition, our analysis showed that, as in S. cerevisiae (Guy et al. 2012), yW formation was impaired in the Sp trm734▵ mutant (44% of wild-type levels) and to a lesser extent in the Sp trm732▵ mutant (73%) (Fig. 4F), consistent with the increased m1G levels (Table 3; Fig. 4A). We also note that m2G levels on tRNAPhe are substantially higher in the Sp trm734▵ mutant compared with those from a wild-type strain when grown in EMM (0.61 versus 0.19 moles/mole).

The putative S. pombe trm732+ and trm734+ genes are required for Cm and Gm formation, respectively, on tRNAPhe

To further determine if the modification circuitry for the anticodon loop of tRNAPhe is conserved between S. cerevisiae and S. pombe, we analyzed the putative S. pombe trm732▵ (SPCC1494.07) and trm734▵ (SPBC1306.02) strains. An Sp trm732▵ haploid mutant obtained by sporulation of the heterozygous diploid lacked detectable Cm in its tRNAPhe and had normal levels of Gm compared with that from wild type (0.88 versus 0.90 moles/mole) (Table 3; Fig. 4A) and did not have any obvious growth defect on plates or in liquid YES medium (Table 1; Fig. 4B), similar to the lack of an obvious growth defect observed for the Sc trm732▵ mutant (Table 1; Guy et al. 2012).

S. pombe trm734▵ mutants are sick due to hypomodified tRNAPhe. (A) HPLC traces of tRNAPhe from S. pombe trm732▵ and S. pombe trm734▵ mutants. tRNAPhe was purified from the indicated strains grown in YES medium and then digested to nucleosides and analyzed by HPLC. (B) S. pombe trm732▵ mutants are healthy. Indicated strains were grown in YES medium overnight, and analyzed on YES plates as described in Figure 2A. (C) S. pombe trm734▵ mutants have a growth defect on EMM. Indicated strains were grown overnight in EMM, diluted, and plated on EMM as indicated. (D) S. pombe trm734▵ mutants have a substantial growth defect on rich medium at various temperatures. Indicated strains were transformed with indicated ura4+ plasmids, grown in EMM-Ura, diluted, and plated on YES medium at the indicated temperatures after 3 d. (E) Overexpression of tRNAPhe suppresses the slow growth of S. pombe trm734▵ mutants. Strains with plasmids as indicated were grown in EMM-Leu overnight, and analyzed as in Figure 2A, after incubation for 3 d at 30°C. Strains were then purified on medium containing 5-FOA, grown overnight in EMM-Leu, diluted, spotted, and analyzed at the indicated temperatures on YES medium. (F) Levels of yW in tRNAPhe are decreased in S. pombe trm732▵ and S. pombe trm734▵ mutants. tRNAPhe was purified from the indicated strains after growth in YES medium and analyzed as described in Figure 2C.

HPLC analysis of tRNAPhe nucleoside content from S. pombe trm732▵ and S. pombe trm734▵ strains

However, an Sp trm734▵ [Sp trm734+
ura4+] haploid obtained by sporulation of the heterozygous diploid containing the plasmid grew poorly when plated on EMM containing 5-FOA (Fig. 4C), and the resulting Sp trm734▵ strain lacked Gm34 in its tRNAPhe, and had Cm levels comparable to those of wild type (0.86 versus 0.91 moles/mole) (Table 3; Fig. 4A). The Sp trm734▵ mutant had a significant growth defect on YES medium on plates or in liquid (generation time of 258 min, compared with 148 min for wild type) (Table 1; Fig. 4D), whereas the Sc trm734▵ mutant lacked any obvious growth defect on plates or in liquid medium (generation time of 70 min, compared with 66 min for wild type) (Table 1; Guy et al. 2012), demonstrating that Trm734 has a more prominent role in S. pombe than in S. cerevisiae. Furthermore, since overexpression of tRNAPhe nearly completely suppressed the slow growth phenotype of the Sp trm734▵ strain over a range of temperatures (Fig. 4E), we conclude that lack of Gm34 of tRNAPhe is the cause of the Sp trm734▵ defect.

In addition, our analysis showed that, as in S. cerevisiae (Guy et al. 2012), yW formation was impaired in the Sp trm734▵ mutant (44% of wild-type levels) and to a lesser extent in the Sp trm732▵ mutant (73%) (Fig. 4F), consistent with the increased m1G levels (Table 3; Fig. 4A). We also note that m2G levels on tRNAPhe are substantially higher in the Sp trm734▵ mutant compared with those from a wild-type strain when grown in EMM (0.61 versus 0.19 moles/mole).

FTSJ1 is the human Trm7 ortholog and requires S. cerevisiae Trm732 to catalyze Cm32 modification on tRNAPhe in S. cerevisiaeTo investigate the conservation of function of metazoan genes involved in 2′-O-methylation of the tRNA anticodon loop, we examined complementation of the corresponding S. cerevisiae mutants by introduction of appropriate constructs expressing metazoan TRM7 and/or its metazoan TRM732 and TRM734 partners. To identify TRM7 homologs, we examined Sp trm7+ and predicted TRM7 orthologs from D. melanogaster and humans by introduction of a high copy [2µ LEU2 PGAL TRM7] plasmid expressing the corresponding ortholog under galactose control to an Sc trm7Δ [Sc TRM7 URA3] strain, followed by analysis of growth after plating on medium containing 5-FOA and galactose. We found that the slow growth phenotype of Sc trm7Δ mutants was efficiently suppressed by expression of Sp Trm7 (∼61% identical to Sc Trm7), D. melanogaster ORF CG5220 (Dm TRM7, ∼43% identical to Sc Trm7), and human FTSJ1 (∼50% identical to Sc Trm7) but not human FTSJ2 (∼34% identical to Sc Trm7) (Fig. 5A), after selection on medium containing 5-FOA. Furthermore, the resulting Sc trm7Δ [2µ LEU2 PGAL FTSJ1] strain grew nearly as well as the wild-type S. cerevisiae strain (Fig. 5A, right panel), strongly suggesting that FTSJ1 is the human ortholog of Trm7.FIGURE 5.FTSJ1 is the human ortholog of S. cerevisiae Trm7, and its function in S. cerevisiae requires Trm732 for Cm32 modification. (A) Expression of S. pombe trm7+, D. melanogaster TRM7, or human FTSJ1 suppresses the slow growth S. cerevisiae trm7▵ mutants. Strains with plasmids as indicated were grown overnight in S-Leu medium containing raffinose and galactose, diluted to OD600 of ∼0.5 in H2O, and serially diluted 10-fold in H2O, and then 2 µL was spotted onto S medium containing 5-FOA, raffinose, and galactose, followed by incubation for 3 d at 30°C. (Right panel) Selected strains were then purified on medium containing 5-FOA, grown overnight in S-Leu medium containing raffinose and galactose, and analyzed by spotting to YP medium containing raffinose and galactose. (B) FTSJ1 requires Sc TRM732 to suppress the slow growth of Sc trm7▵ mutants. Strains were analyzed as in A.The ability of human FTSJ1 and the other tested TRM7 orthologs to complement the growth defect of an S. cerevisiae trm7Δ mutant strain was surprising because Sc Trm7 and Sp Trm7 have a strict requirement for their corresponding Trm732 and Trm734 partner proteins to generate Cm32 and Nm34, respectively (Guy et al. 2012). Because tRNAPhe is the Sc Trm7 substrate that must be modified for healthy growth in S. cerevisiae (Guy et al. 2012), we examined 2′-O-methylation of tRNAPhe from the Sc trm7Δ [2µ LEU2 PGAL FTSJ1] strain. We found that tRNAPhe purified from this strain had high levels of Cm (0.89 versus 0.90 moles/mole in wild type), but no detectable Gm (Table 4), suggesting that human FTSJ1 is working in concert with Sc Trm732 to modify tRNAPhe. The healthy growth of this Sc trm7Δ [2µ LEU2 PGAL FTSJ1] strain is consistent with our previous observation that the Cm32 modification of tRNAPhe is sufficient for healthy growth in S. cerevisiae (Guy et al. 2012).TABLE 4.HPLC analysis of tRNAPhe nucleoside content from S. cerevisiae trm7▵ [FTSJ1] and S. cerevisiae trm7▵ trm732▵ [FTSJ1 THADA] strainsThe occurrence of Cm in tRNAPhe from an Sc trm7Δ [2µ LEU2 PGAL FTSJ1] strain suggested either that FTSJ1 does not require an interacting partner for Cm32 catalysis, or that it works with S. cerevisiae Trm732. Since the slow growth phenotype of the Sc trm7Δ trm732Δ strain was not suppressed by introduction of the [2µ LEU2 PGAL FTSJ1] plasmid and no Cm formation was observed on tRNAPhe purified from this strain (Table 4; Fig. 5B), we infer that FTSJ1 works with Sc Trm732 for formation of Cm32 on tRNAPhe. Since tRNALeu(UAA) from an Sc trm7Δ [2µ LEU2 PGAL FTSJ1] strain lacked both Cm and ncm5Um (Table 5; Fig. 1A), the activity of FTSJ1 with Trm732 in S. cerevisiae does not extend to all Sc Trm7 substrates.TABLE 5.HPLC analysis of tRNALeu(UAA) nucleoside content from an S. cerevisiae trm7▵ [FTSJ1] strainWe note that because FTSJ1 is expressed from a high copy PGAL plasmid in these experiments, it is formally possible that FTSJ1 is not the true Trm7 homolog. However, this seems unlikely due to the high sequence similarity between FTSJ1 and Trm7, and the requirement of Sc Trm732 function for FTSJ1 activity in S. cerevisiae.

FTSJ1 is the human Trm7 ortholog and requires S. cerevisiae Trm732 to catalyze Cm32 modification on tRNAPhe in S. cerevisiae

To investigate the conservation of function of metazoan genes involved in 2′-O-methylation of the tRNA anticodon loop, we examined complementation of the corresponding S. cerevisiae mutants by introduction of appropriate constructs expressing metazoan TRM7 and/or its metazoan TRM732 and TRM734 partners. To identify TRM7 homologs, we examined Sp trm7+ and predicted TRM7 orthologs from D. melanogaster and humans by introduction of a high copy [2µ LEU2 PGAL TRM7] plasmid expressing the corresponding ortholog under galactose control to an Sc trm7Δ [Sc TRM7 URA3] strain, followed by analysis of growth after plating on medium containing 5-FOA and galactose. We found that the slow growth phenotype of Sc trm7Δ mutants was efficiently suppressed by expression of Sp Trm7 (∼61% identical to Sc Trm7), D. melanogaster ORF CG5220 (Dm TRM7, ∼43% identical to Sc Trm7), and human FTSJ1 (∼50% identical to Sc Trm7) but not human FTSJ2 (∼34% identical to Sc Trm7) (Fig. 5A), after selection on medium containing 5-FOA. Furthermore, the resulting Sc trm7Δ [2µ LEU2 PGAL FTSJ1] strain grew nearly as well as the wild-type S. cerevisiae strain (Fig. 5A, right panel), strongly suggesting that FTSJ1 is the human ortholog of Trm7.

FTSJ1 is the human ortholog of S. cerevisiae Trm7, and its function in S. cerevisiae requires Trm732 for Cm32 modification. (A) Expression of S. pombe trm7+, D. melanogaster TRM7, or human FTSJ1 suppresses the slow growth S. cerevisiae trm7▵ mutants. Strains with plasmids as indicated were grown overnight in S-Leu medium containing raffinose and galactose, diluted to OD600 of ∼0.5 in H2O, and serially diluted 10-fold in H2O, and then 2 µL was spotted onto S medium containing 5-FOA, raffinose, and galactose, followed by incubation for 3 d at 30°C. (Right panel) Selected strains were then purified on medium containing 5-FOA, grown overnight in S-Leu medium containing raffinose and galactose, and analyzed by spotting to YP medium containing raffinose and galactose. (B) FTSJ1 requires Sc TRM732 to suppress the slow growth of Sc trm7▵ mutants. Strains were analyzed as in A.

The ability of human FTSJ1 and the other tested TRM7 orthologs to complement the growth defect of an S. cerevisiae trm7Δ mutant strain was surprising because Sc Trm7 and Sp Trm7 have a strict requirement for their corresponding Trm732 and Trm734 partner proteins to generate Cm32 and Nm34, respectively (Guy et al. 2012). Because tRNAPhe is the Sc Trm7 substrate that must be modified for healthy growth in S. cerevisiae (Guy et al. 2012), we examined 2′-O-methylation of tRNAPhe from the Sc trm7Δ [2µ LEU2 PGAL FTSJ1] strain. We found that tRNAPhe purified from this strain had high levels of Cm (0.89 versus 0.90 moles/mole in wild type), but no detectable Gm (Table 4), suggesting that human FTSJ1 is working in concert with Sc Trm732 to modify tRNAPhe. The healthy growth of this Sc trm7Δ [2µ LEU2 PGAL FTSJ1] strain is consistent with our previous observation that the Cm32 modification of tRNAPhe is sufficient for healthy growth in S. cerevisiae (Guy et al. 2012).

HPLC analysis of tRNAPhe nucleoside content from S. cerevisiae trm7▵ [FTSJ1] and S. cerevisiae trm7▵ trm732▵ [FTSJ1 THADA] strains

The occurrence of Cm in tRNAPhe from an Sc trm7Δ [2µ LEU2 PGAL FTSJ1] strain suggested either that FTSJ1 does not require an interacting partner for Cm32 catalysis, or that it works with S. cerevisiae Trm732. Since the slow growth phenotype of the Sc trm7Δ trm732Δ strain was not suppressed by introduction of the [2µ LEU2 PGAL FTSJ1] plasmid and no Cm formation was observed on tRNAPhe purified from this strain (Table 4; Fig. 5B), we infer that FTSJ1 works with Sc Trm732 for formation of Cm32 on tRNAPhe. Since tRNALeu(UAA) from an Sc trm7Δ [2µ LEU2 PGAL FTSJ1] strain lacked both Cm and ncm5Um (Table 5; Fig. 1A), the activity of FTSJ1 with Trm732 in S. cerevisiae does not extend to all Sc Trm7 substrates.

HPLC analysis of tRNALeu(UAA) nucleoside content from an S. cerevisiae trm7▵ [FTSJ1] strain

We note that because FTSJ1 is expressed from a high copy PGAL plasmid in these experiments, it is formally possible that FTSJ1 is not the true Trm7 homolog. However, this seems unlikely due to the high sequence similarity between FTSJ1 and Trm7, and the requirement of Sc Trm732 function for FTSJ1 activity in S. cerevisiae.

THADA is the human homolog of Trm732 and works with ScTrm7 for Cm32 modificationAlthough Trm732 homologs have only ∼20% sequence identity, we find that the putative human Trm732 homolog THADA functions efficiently to replace Sc Trm732. Thus, the slow growth of an Sc trm7Δ trm732Δ strain was suppressed by expression of FTSJ1 and THADA, but not by FTSJ1 alone (Fig. 6A). Since tRNAPhe from this Sc trm7Δ trm732Δ [2µ PGAL
FTSJ1 PGAL
THADA] strain had identical Cm levels to that from the wild-type strain (0.90 moles/mole) (Table 4) and no detectable Gm, and since tRNALeu(UAA) from this strain had substantial Cm modification (0.36 moles/mole) and no ncm5Um (Table 5), we conclude that THADA is the human Trm732 homolog and functionally interacts with FTSJ1. The ability of FTSJ1 to modify C32 of tRNALeu(UAA) in combination with THADA but not with Sc Trm732 may be due to higher levels of THADA relative to endogenous levels of yeast Trm732, and/or more efficient partnering of FTSJ1 and THADA.FIGURE 6.S. cerevisiae Trm7 works with human THADA or S. pombe Trm732 for function. (A) THADA is the human ortholog of S. cerevisiae TRM7. Strains with plasmids as indicated were analyzed as in Figure 5A. (B) S. pombe trm732+ suppresses the slow growth of S. cerevisiae trm732▵ trm734▵ mutants. Strains were analyzed as in Figure 5A.We also find evidence that S. cerevisiae Trm7 can interact with human THADA for Cm32 formation, since expression of THADA suppressed the slow growth of an Sc trm732Δ trm734Δ strain (Fig. 6A), resulting in tRNAPhe with 0.76 moles/mole Cm (Table 4). Thus, our data demonstrate that human (FTSJ1) or S. cerevisiae Trm7 can each utilize either human (THADA) or S. cerevisiae Trm732 to catalyze formation of Cm32 on tRNAPhe in S. cerevisiae. S. cerevisiae Trm7 also appears to work with S. pombe Trm732, because expression of Sp trm732+ suppressed the slow growth of the Sc trm732Δ trm734Δ strain (Fig. 6B).

THADA is the human homolog of Trm732 and works with ScTrm7 for Cm32 modification

Although Trm732 homologs have only ∼20% sequence identity, we find that the putative human Trm732 homolog THADA functions efficiently to replace Sc Trm732. Thus, the slow growth of an Sc trm7Δ trm732Δ strain was suppressed by expression of FTSJ1 and THADA, but not by FTSJ1 alone (Fig. 6A). Since tRNAPhe from this Sc trm7Δ trm732Δ [2µ PGAL
FTSJ1 PGAL
THADA] strain had identical Cm levels to that from the wild-type strain (0.90 moles/mole) (Table 4) and no detectable Gm, and since tRNALeu(UAA) from this strain had substantial Cm modification (0.36 moles/mole) and no ncm5Um (Table 5), we conclude that THADA is the human Trm732 homolog and functionally interacts with FTSJ1. The ability of FTSJ1 to modify C32 of tRNALeu(UAA) in combination with THADA but not with Sc Trm732 may be due to higher levels of THADA relative to endogenous levels of yeast Trm732, and/or more efficient partnering of FTSJ1 and THADA.

S. cerevisiae Trm7 works with human THADA or S. pombe Trm732 for function. (A) THADA is the human ortholog of S. cerevisiae TRM7. Strains with plasmids as indicated were analyzed as in Figure 5A. (B) S. pombe trm732+ suppresses the slow growth of S. cerevisiae trm732▵ trm734▵ mutants. Strains were analyzed as in Figure 5A.

We also find evidence that S. cerevisiae Trm7 can interact with human THADA for Cm32 formation, since expression of THADA suppressed the slow growth of an Sc trm732Δ trm734Δ strain (Fig. 6A), resulting in tRNAPhe with 0.76 moles/mole Cm (Table 4). Thus, our data demonstrate that human (FTSJ1) or S. cerevisiae Trm7 can each utilize either human (THADA) or S. cerevisiae Trm732 to catalyze formation of Cm32 on tRNAPhe in S. cerevisiae. S. cerevisiae Trm7 also appears to work with S. pombe Trm732, because expression of Sp trm732+ suppressed the slow growth of the Sc trm732Δ trm734Δ strain (Fig. 6B).

DISCUSSIONOur results demonstrate that the entire circuitry for tRNAPhe anticodon loop modification established for S. cerevisiae has been retained in the yeast S. pombe, requiring Trm7 to act with Trm732 to 2′-O-methylate C32, and with Trm734 to 2′-O-methylate G34, leading to efficient conversion of m1G37 to yW. Furthermore, we have provided strong evidence that FTSJ1 and THADA are the human Trm7 and Trm732 orthologs, and that the human and S. cerevisiae proteins can act interchangeably together to catalyze 2′-O-methylation of C32 of tRNAPhe (Table 4). THADA and S. pombe trm732+ each complemented the Trm732 defect of Sc trm732▵ trm734Δ mutants, but have little overall conservation (∼20% identity, mostly clustered in the small DUF2428 region) (Fig. 1C), suggesting that there may be substantial structural homology in less conserved regions, such as in the predicted armadillo repeats (Tewari et al. 2010).Since the S. cerevisiae and S. pombe lineages diverged from one another ∼1.1 billion yr ago, and since humans and fungal lineages diverged ∼1.6 billion yr ago (Hedges 2002), it seems likely that this anticodon loop modification circuitry is widely conserved among eukaryotes. The absence of identifiable Trm732 and Trm734 orthologs in some eukaryotes may indeed be due to lack of universal conservation of the corresponding genes in some organisms, perhaps because of increased amounts of tRNAPhe in those organisms. Alternatively the lack of identifiable Trm732 and Trm734 orthologs may reflect the poor sequence conservation of these proteins, or may be due to alternative 2′-O-methylation pathways, such as by a phylogenetically distinct anticodon loop 2′-O-methyltransferase family (Tkaczuk et al. 2007), or by a Box C/D guide RNA (Joardar et al. 2011).We note that experiments to test the function of WDR6, the predicted human Trm734 homolog, by complementation of the appropriate S. cerevisiae mutants have been inconclusive. Coexpression of human WDR6 with FTSJ1 does not suppress the slow growth of a Sc trm7Δ trm732Δ strain (data not shown), but WDR6 was not expressed well in S. cerevisiae (data not shown). Nonetheless, because Trm734 function is conserved in S. pombe and S. cerevisiae, it seems plausible that WDR6 and the other Trm734 family members will be required for Gm34 formation on tRNAPhe in their corresponding organisms.Further evidence that this tRNAPhe anticodon loop modification circuitry is conserved in humans derives from the observation that Ehrlich ascites tumors and neuroblastoma cells lacking O2yW37 (peroxywybutosine) on tRNAPhe also lack Cm32 and Gm34 modifications on tRNAPhe (Kuchino et al. 1982), consistent with the requirement for 2′-O-methylation of the anticodon loop of tRNAPhe as a prerequisite for yW37 formation, and fueling speculation that these tRNA defects arise from defective FTSJ1 function.Our data also provide further evidence indicating that levels of tRNA modifications are regulated by cellular growth conditions. Thus, we found that m7G and m2G levels on tRNAPhe are decreased in S. pombe cells grown in EMM in the absence of thiamine (Table 2), similar to the changes in tRNA modification levels observed in S. cerevisiae cells grown under cellular stress conditions or in cells that have undergone growth arrest (Chan et al. 2010; Chan et al. 2012; Preston et al. 2013). Furthermore, the finding that m2G levels of tRNAPhe are near normal in the Sptrm7Δ and Sptrm734Δ mutants (compared with those from wild-type cells grown in EMM) implies the existence of a compensatory modification mechanism, similar to that observed in trm9Δ mutants, which have acquired ncm5U (and ncm5s2U) in the absence of mcm5U34 (and mcm5s2U) (Chen et al. 2011a), but in this case the m2G is likely on a different residue.Our finding that tRNAPhe is the biologically important Trm7 substrate in both S. cerevisiae and S. pombe (Fig. 3; Guy et al. 2012) suggests that tRNAPhe may be the important substrate throughout eukaryotes. This is consistent with our finding that human FTSJ1 and THADA readily modified C32 of tRNAPhe in S. cerevisiae, and that tRNAPhe from 16 of 17 eukaryotes examined contains Cm32 and Gm34 (Machnicka et al. 2013). However, we note that overexpression of tRNAPhe did not completely suppress the growth defect of Sp trm7▵ mutants, particularly at high temperature (Fig. 3). This could occur if there are other tRNA species that require modification for full function at this temperature, if the tRNAPhe is not sufficiently overexpressed at this temperature to overcome the defect in decoding, or if part of the defect in tRNAPhe occurs at a step after binding of the tRNA to the A-site of the ribosome, since additional copies of hypomodified tRNAs should not affect translation after this step. Because overexpression of tRNAPhe nearly completely suppressed the slow growth phenotype of Sp trm734▵ mutants at all temperatures, the defect is almost certainly caused by loss of Gm34 on tRNAPhe (Fig. 4E).Although there is conservation of the Trm7 circuitry for tRNAPhe anticodon loop modification and for the importance of these modifications in S. pombe and S. cerevisiae, there are two crucial differences in the biological consequences of mutations in the corresponding genes in the two organisms. First, a trm7Δ mutation was more deleterious to growth in S. pombe than in S. cerevisiae; thus the Sp trm7Δ [Pnmt1** Sp trm7+
LEU2] strain (although itself much healthier than the Sp trm7Δ haploid) still had a generation time 2.16-fold higher than the wild-type strain (Table 1), which was slightly more than the difference of 1.94-fold observed for an S. cerevisiae trm7Δ mutant compared with its wild-type control. Second, an S. pombe trm734Δ mutant had a severe growth defect, with a 1.7-fold increased generation time relative to the wild type strain, whereas an S. cerevisiae trm734Δ mutant had 1.06-fold difference in generation time. Furthermore, our identification of FTSJ1 as the human Sc TRM7 ortholog suggests that defects in the human gene result in a relatively mild, albeit medically serious, condition, since FTSJ1 splice site, nonsense, and deletion mutations are consistently associated with NSXLID (Freude et al. 2004; Ramser et al. 2004; Froyen et al. 2007; Takano et al. 2008). NSXLID may occur in these patients because specific human tRNAs have a greater requirement for Trm7 modification in development of the central nervous system (CNS) than in other tissues; indeed, recent results suggest that mutation of a tRNA isodecoder expressed specifically in the CNS can lead to ribosome stalling and contribute to neurodegeneration in certain mutant mouse strains (Ishimura et al. 2014).Our results demonstrating that THADA is the human Trm732 ortholog further suggest that 2′-O-methylation of the tRNA anticodon loop may be associated with human health. THADA is associated with epithelial thyroid adenomas (Rippe et al. 2003); and genome wide association studies have implicated THADA alleles in type 2 diabetes (Zeggini et al. 2008) and polycystic ovary syndrome (Chen et al. 2011c). However, the biological significance of these associations is not known, and the linkage to Nm32 modification remains to be determined.NSXLID associated with defective FTSJ1 (Freude et al. 2004; Ramser et al. 2004; Froyen et al. 2007; Takano et al. 2008) adds to a growing list of neurological disorders associated with defective tRNA modification. This list includes intellectual disability associated with a point mutation in hADAT3 , the predicted homolog of a subunit of the yeast tRNA A34 deaminase (Alazami et al. 2013); a frameshift mutation in hTRMT1 (Najmabadi et al. 2011), which has tRNA m2,2G26 (N2,N2-dimethylguanosine) methyltransferase activity (Liu and Strâby 2000); mutations in NSUN2 (Abbasi-Moheb et al. 2012; Khan et al. 2012; Martinez et al. 2012), which modifies C34, C48, C49, and C50 on target tRNAs to m5C; and mutations in hELP2 (Najmabadi et al. 2011), a member of the ELP complex responsible for formation of the cm5U moiety found on mcm5U34, ncm5U34, mcm5s2U34 and related modifications. In addition, familial disautonomia is associated with mutations in hELP1 (IKBAKP) (Anderson et al. 2001). The ELP complex has been reported to have functions in addition to tRNA modification in humans (Creppe et al. 2009; Okada et al. 2010), suggesting the possibility that these disease associations could be due to non-tRNA related defects. Regardless, it is notable that many of the associations between defective tRNA modification and neurological disorders are linked to wobble residue 34, which is required for efficient and accurate decoding of mRNA (Agris et al. 2007; Johansson et al. 2008), suggesting that neurological development is extremely sensitive to defects in translation. It remains to be seen whether FTSJ1-associated NSXLID is specifically due to Nm34 defects, to Nm32 defects, or to both.

Our results demonstrate that the entire circuitry for tRNAPhe anticodon loop modification established for S. cerevisiae has been retained in the yeast S. pombe, requiring Trm7 to act with Trm732 to 2′-O-methylate C32, and with Trm734 to 2′-O-methylate G34, leading to efficient conversion of m1G37 to yW. Furthermore, we have provided strong evidence that FTSJ1 and THADA are the human Trm7 and Trm732 orthologs, and that the human and S. cerevisiae proteins can act interchangeably together to catalyze 2′-O-methylation of C32 of tRNAPhe (Table 4). THADA and S. pombe trm732+ each complemented the Trm732 defect of Sc trm732▵ trm734Δ mutants, but have little overall conservation (∼20% identity, mostly clustered in the small DUF2428 region) (Fig. 1C), suggesting that there may be substantial structural homology in less conserved regions, such as in the predicted armadillo repeats (Tewari et al. 2010).

Since the S. cerevisiae and S. pombe lineages diverged from one another ∼1.1 billion yr ago, and since humans and fungal lineages diverged ∼1.6 billion yr ago (Hedges 2002), it seems likely that this anticodon loop modification circuitry is widely conserved among eukaryotes. The absence of identifiable Trm732 and Trm734 orthologs in some eukaryotes may indeed be due to lack of universal conservation of the corresponding genes in some organisms, perhaps because of increased amounts of tRNAPhe in those organisms. Alternatively the lack of identifiable Trm732 and Trm734 orthologs may reflect the poor sequence conservation of these proteins, or may be due to alternative 2′-O-methylation pathways, such as by a phylogenetically distinct anticodon loop 2′-O-methyltransferase family (Tkaczuk et al. 2007), or by a Box C/D guide RNA (Joardar et al. 2011).

We note that experiments to test the function of WDR6, the predicted human Trm734 homolog, by complementation of the appropriate S. cerevisiae mutants have been inconclusive. Coexpression of human WDR6 with FTSJ1 does not suppress the slow growth of a Sc trm7Δ trm732Δ strain (data not shown), but WDR6 was not expressed well in S. cerevisiae (data not shown). Nonetheless, because Trm734 function is conserved in S. pombe and S. cerevisiae, it seems plausible that WDR6 and the other Trm734 family members will be required for Gm34 formation on tRNAPhe in their corresponding organisms.

Further evidence that this tRNAPhe anticodon loop modification circuitry is conserved in humans derives from the observation that Ehrlich ascites tumors and neuroblastoma cells lacking O2yW37 (peroxywybutosine) on tRNAPhe also lack Cm32 and Gm34 modifications on tRNAPhe (Kuchino et al. 1982), consistent with the requirement for 2′-O-methylation of the anticodon loop of tRNAPhe as a prerequisite for yW37 formation, and fueling speculation that these tRNA defects arise from defective FTSJ1 function.

Our data also provide further evidence indicating that levels of tRNA modifications are regulated by cellular growth conditions. Thus, we found that m7G and m2G levels on tRNAPhe are decreased in S. pombe cells grown in EMM in the absence of thiamine (Table 2), similar to the changes in tRNA modification levels observed in S. cerevisiae cells grown under cellular stress conditions or in cells that have undergone growth arrest (Chan et al. 2010; Chan et al. 2012; Preston et al. 2013). Furthermore, the finding that m2G levels of tRNAPhe are near normal in the Sptrm7Δ and Sptrm734Δ mutants (compared with those from wild-type cells grown in EMM) implies the existence of a compensatory modification mechanism, similar to that observed in trm9Δ mutants, which have acquired ncm5U (and ncm5s2U) in the absence of mcm5U34 (and mcm5s2U) (Chen et al. 2011a), but in this case the m2G is likely on a different residue.

Our finding that tRNAPhe is the biologically important Trm7 substrate in both S. cerevisiae and S. pombe (Fig. 3; Guy et al. 2012) suggests that tRNAPhe may be the important substrate throughout eukaryotes. This is consistent with our finding that human FTSJ1 and THADA readily modified C32 of tRNAPhe in S. cerevisiae, and that tRNAPhe from 16 of 17 eukaryotes examined contains Cm32 and Gm34 (Machnicka et al. 2013). However, we note that overexpression of tRNAPhe did not completely suppress the growth defect of Sp trm7▵ mutants, particularly at high temperature (Fig. 3). This could occur if there are other tRNA species that require modification for full function at this temperature, if the tRNAPhe is not sufficiently overexpressed at this temperature to overcome the defect in decoding, or if part of the defect in tRNAPhe occurs at a step after binding of the tRNA to the A-site of the ribosome, since additional copies of hypomodified tRNAs should not affect translation after this step. Because overexpression of tRNAPhe nearly completely suppressed the slow growth phenotype of Sp trm734▵ mutants at all temperatures, the defect is almost certainly caused by loss of Gm34 on tRNAPhe (Fig. 4E).

Although there is conservation of the Trm7 circuitry for tRNAPhe anticodon loop modification and for the importance of these modifications in S. pombe and S. cerevisiae, there are two crucial differences in the biological consequences of mutations in the corresponding genes in the two organisms. First, a trm7Δ mutation was more deleterious to growth in S. pombe than in S. cerevisiae; thus the Sp trm7Δ [Pnmt1** Sp trm7+
LEU2] strain (although itself much healthier than the Sp trm7Δ haploid) still had a generation time 2.16-fold higher than the wild-type strain (Table 1), which was slightly more than the difference of 1.94-fold observed for an S. cerevisiae trm7Δ mutant compared with its wild-type control. Second, an S. pombe trm734Δ mutant had a severe growth defect, with a 1.7-fold increased generation time relative to the wild type strain, whereas an S. cerevisiae trm734Δ mutant had 1.06-fold difference in generation time. Furthermore, our identification of FTSJ1 as the human Sc TRM7 ortholog suggests that defects in the human gene result in a relatively mild, albeit medically serious, condition, since FTSJ1 splice site, nonsense, and deletion mutations are consistently associated with NSXLID (Freude et al. 2004; Ramser et al. 2004; Froyen et al. 2007; Takano et al. 2008). NSXLID may occur in these patients because specific human tRNAs have a greater requirement for Trm7 modification in development of the central nervous system (CNS) than in other tissues; indeed, recent results suggest that mutation of a tRNA isodecoder expressed specifically in the CNS can lead to ribosome stalling and contribute to neurodegeneration in certain mutant mouse strains (Ishimura et al. 2014).

Our results demonstrating that THADA is the human Trm732 ortholog further suggest that 2′-O-methylation of the tRNA anticodon loop may be associated with human health. THADA is associated with epithelial thyroid adenomas (Rippe et al. 2003); and genome wide association studies have implicated THADA alleles in type 2 diabetes (Zeggini et al. 2008) and polycystic ovary syndrome (Chen et al. 2011c). However, the biological significance of these associations is not known, and the linkage to Nm32 modification remains to be determined.

NSXLID associated with defective FTSJ1 (Freude et al. 2004; Ramser et al. 2004; Froyen et al. 2007; Takano et al. 2008) adds to a growing list of neurological disorders associated with defective tRNA modification. This list includes intellectual disability associated with a point mutation in hADAT3 , the predicted homolog of a subunit of the yeast tRNA A34 deaminase (Alazami et al. 2013); a frameshift mutation in hTRMT1 (Najmabadi et al. 2011), which has tRNA m2,2G26 (N2,N2-dimethylguanosine) methyltransferase activity (Liu and Strâby 2000); mutations in NSUN2 (Abbasi-Moheb et al. 2012; Khan et al. 2012; Martinez et al. 2012), which modifies C34, C48, C49, and C50 on target tRNAs to m5C; and mutations in hELP2 (Najmabadi et al. 2011), a member of the ELP complex responsible for formation of the cm5U moiety found on mcm5U34, ncm5U34, mcm5s2U34 and related modifications. In addition, familial disautonomia is associated with mutations in hELP1 (IKBAKP) (Anderson et al. 2001). The ELP complex has been reported to have functions in addition to tRNA modification in humans (Creppe et al. 2009; Okada et al. 2010), suggesting the possibility that these disease associations could be due to non-tRNA related defects. Regardless, it is notable that many of the associations between defective tRNA modification and neurological disorders are linked to wobble residue 34, which is required for efficient and accurate decoding of mRNA (Agris et al. 2007; Johansson et al. 2008), suggesting that neurological development is extremely sensitive to defects in translation. It remains to be seen whether FTSJ1-associated NSXLID is specifically due to Nm34 defects, to Nm32 defects, or to both.

MATERIALS AND METHODSIdentification of Trm7, Trm732, and Trm734 sequence homologs from diverse eukaryotic genomesBLAST searches (http://blast.ncbi.nlm.nih.gov/) for Trm7, Trm732, and Trm734 homologs were performed against sequenced genomes from a diverse set of eukaryotes including representatives from all five eukaryotic supergroups (Adl et al. 2012), including Amoebazoa (Dictyostelium discoideum), Archaeplastida (Arabidopsis thaliana, Cyanidioschyzon merolae, Ostreococcus tauri, Oryza sativa, and Zea mays), Excavata (Giardia intestinalis, Naegleria gruberi), Opisthokonta (Bombyx mori, Caenorhabditis elegans, Danio rerio, Drosophila melanogaster, Homo sapiens, Monosiga brevicollis, Mus musculus, Saccharomyces cerevisiae, Schizosaccharomyces pombe, and Xenopus tropicalis), and SAR (Stramenopiles + Alveolates + Rhizaria) (Cryptomonas paramaecium, Guillardia theta, Phytophthora infestans, Tetrahymena thermophile, Thalassiosira pseudonana, Toxoplasma gondii, and Trypanosoma brucei).Yeast strainsYeast strains are listed in Table 6. The S. pombe haploid Sp trm7Δ::kanMX [Pnmt1 Sp trm7+
ura4+] (yMG1052A) strain was generated by transformation of the Sp trm7Δ::kanMX heterozygous diploid with pMG360A [Pnmt1 Sp trm7+ura4+], followed by sporulation on EMM lacking uracil supplemented with 500 mg/L G418, selection of haploids, and PCR confirmation of the knockout. The haploid S. pombe trm732Δ::kanMX mutant strain (yMG958B) was generated by sporulation of the heterzygous diploid on yeast extract with 3% dextrose (YE medium) with 200 mg/L G418, selection of haploids, and PCR confirmation. The haploid S. pombe trm734▵ [PSp trm734 Sp trm734+
ura4+] mutant strain (yMG1289-1) was generated by transformation of the Sp trm734Δ::kanMX heterozygous diploid with a LEU2 tRNAPhe plasmid (pMG308C), followed by sporulation on YE medium with 200 mg/L G418, selection of haploids and PCR verification. This strain was then transformed with pMG426G (PSp trm734 Sp trm734+
ura4+), followed by selection of leu−/ura+ colonies, and PCR confirmation. The S. cerevisiae trm7Δ::bleR [URA3 Sc TRM7] (yMG348-1), Sc trm732Δ::bleR (yMG814-1), Sc trm734Δ::bleR (yMG724-5), and Sc trm734Δ::bleR trm732Δ::kanMX [Sc TRM734 URA3 CEN] (yMG818-1) strains were described previously (Guy et al. 2012). Double mutant S. cerevisiae trm7Δ strains were constructed by PCR amplification of DNA from the appropriate YKO collection kanMX strain (Open Biosystems), followed by transformation of the DNA into yMG348-1.TABLE 6.Strains used in this studyPlasmidsPlasmids used in this study are listed in Table 7. The S. pombe vector expressing S. pombe trm7+ under control of the Pnmt1 (no message in thiamine) (pMG360A) and Pnmt1** (low strength no message in thiamine) promoters (pMG527B) were constructed by PCR of the Sp trm7+ cDNA ORF from an S. pombe cDNA library (Fikes et al. 1990) and insertion into the XhoI and SmaI sites of pREP4X or pREP81X vectors, respectively. The S. pombe vector expressing Sp trm734+ under control of the native Sp trm734+promoter (pMG426G) was constructed by PCR of Sp trm734+ from S. pombe genomic DNA and insertion at the PstI and XhoI sites of pREP4X to remove the Pnmt1 promoter sequence. tRNA expression vectors were generated by PCR of the appropriate tRNA fragment from genomic DNA and insertion at the Pst1 and XhoI sites of pREP3X. For expression in S. cerevisiae, human FTSJ1 and THADA ORFs were cloned from cDNA plasmids (Open Biosystems), as was D. melanogaster ORF CG5220 (Dm TRM7) (Drosophila Genomics Resource Center). S. pombe trm732+ ORF was cloned from an S. pombe cDNA library (Fikes et al. 1990). ORFs were then inserted by ligation independent cloning (LIC) into [2µ PGAL1,10] S. cerevisiae dual ORF expression vectors which express ORFs under PGAL1 control with a C-terminal PT tag (ORF-3C site-HA epitope-His6- ZZ domain of protein A), and ORFs under PGAL10 control with no tag, essentially as described previously (Quartley et al. 2009; Guy et al. 2012). All plasmids were confirmed by sequencing before use.TABLE 7.Plasmids used in this studyIsolation and purification of tRNAS. pombe strains were grown at 30°C to mid-log phase in YE medium supplemented with 225 mg/L adenine, lysine, histidine, leucine, and uracil (YES), or in EMM with appropriate supplements at 225 mg/L. For analysis of tRNA from the Sp trm7Δ [Pnmt1** Sp trm7+
LEU2] and wild-type strains under repressive conditions, thiamine was added to EMM-Leu at 5 mg/L. S. cerevisiae strains were grown at 30°C to mid-log phase in S dropout medium containing 2% raffinose and 2% galactose. Bulk low molecular weight RNA was extracted from 300 OD-mL pellets, and appropriate 5′ biotinylated oligonucleotides were used to purify tRNA as previously described (Jackman et al. 2003).HPLC and analysis of tRNAPurified tRNA was digested with P1 nuclease and phosphatase as previously described, and nucleosides were subjected to HPLC analysis essentially as previously described (Jackman et al. 2003). For tRNAPhe, HPLC was done at pH 7.0 to maximize separation of Gm and m1G as previously described (Guy et al. 2012). For detection of yW, HPLC was performed with buffers and gradients essentially as previously described (Noma et al. 2006).

MATERIALS AND METHODS

Identification of Trm7, Trm732, and Trm734 sequence homologs from diverse eukaryotic genomesBLAST searches (http://blast.ncbi.nlm.nih.gov/) for Trm7, Trm732, and Trm734 homologs were performed against sequenced genomes from a diverse set of eukaryotes including representatives from all five eukaryotic supergroups (Adl et al. 2012), including Amoebazoa (Dictyostelium discoideum), Archaeplastida (Arabidopsis thaliana, Cyanidioschyzon merolae, Ostreococcus tauri, Oryza sativa, and Zea mays), Excavata (Giardia intestinalis, Naegleria gruberi), Opisthokonta (Bombyx mori, Caenorhabditis elegans, Danio rerio, Drosophila melanogaster, Homo sapiens, Monosiga brevicollis, Mus musculus, Saccharomyces cerevisiae, Schizosaccharomyces pombe, and Xenopus tropicalis), and SAR (Stramenopiles + Alveolates + Rhizaria) (Cryptomonas paramaecium, Guillardia theta, Phytophthora infestans, Tetrahymena thermophile, Thalassiosira pseudonana, Toxoplasma gondii, and Trypanosoma brucei).

Identification of Trm7, Trm732, and Trm734 sequence homologs from diverse eukaryotic genomes

BLAST searches (http://blast.ncbi.nlm.nih.gov/) for Trm7, Trm732, and Trm734 homologs were performed against sequenced genomes from a diverse set of eukaryotes including representatives from all five eukaryotic supergroups (Adl et al. 2012), including Amoebazoa (Dictyostelium discoideum), Archaeplastida (Arabidopsis thaliana, Cyanidioschyzon merolae, Ostreococcus tauri, Oryza sativa, and Zea mays), Excavata (Giardia intestinalis, Naegleria gruberi), Opisthokonta (Bombyx mori, Caenorhabditis elegans, Danio rerio, Drosophila melanogaster, Homo sapiens, Monosiga brevicollis, Mus musculus, Saccharomyces cerevisiae, Schizosaccharomyces pombe, and Xenopus tropicalis), and SAR (Stramenopiles + Alveolates + Rhizaria) (Cryptomonas paramaecium, Guillardia theta, Phytophthora infestans, Tetrahymena thermophile, Thalassiosira pseudonana, Toxoplasma gondii, and Trypanosoma brucei).

Yeast strainsYeast strains are listed in Table 6. The S. pombe haploid Sp trm7Δ::kanMX [Pnmt1 Sp trm7+
ura4+] (yMG1052A) strain was generated by transformation of the Sp trm7Δ::kanMX heterozygous diploid with pMG360A [Pnmt1 Sp trm7+ura4+], followed by sporulation on EMM lacking uracil supplemented with 500 mg/L G418, selection of haploids, and PCR confirmation of the knockout. The haploid S. pombe trm732Δ::kanMX mutant strain (yMG958B) was generated by sporulation of the heterzygous diploid on yeast extract with 3% dextrose (YE medium) with 200 mg/L G418, selection of haploids, and PCR confirmation. The haploid S. pombe trm734▵ [PSp trm734 Sp trm734+
ura4+] mutant strain (yMG1289-1) was generated by transformation of the Sp trm734Δ::kanMX heterozygous diploid with a LEU2 tRNAPhe plasmid (pMG308C), followed by sporulation on YE medium with 200 mg/L G418, selection of haploids and PCR verification. This strain was then transformed with pMG426G (PSp trm734 Sp trm734+
ura4+), followed by selection of leu−/ura+ colonies, and PCR confirmation. The S. cerevisiae trm7Δ::bleR [URA3 Sc TRM7] (yMG348-1), Sc trm732Δ::bleR (yMG814-1), Sc trm734Δ::bleR (yMG724-5), and Sc trm734Δ::bleR trm732Δ::kanMX [Sc TRM734 URA3 CEN] (yMG818-1) strains were described previously (Guy et al. 2012). Double mutant S. cerevisiae trm7Δ strains were constructed by PCR amplification of DNA from the appropriate YKO collection kanMX strain (Open Biosystems), followed by transformation of the DNA into yMG348-1.TABLE 6.Strains used in this study

Yeast strains

Yeast strains are listed in Table 6. The S. pombe haploid Sp trm7Δ::kanMX [Pnmt1 Sp trm7+
ura4+] (yMG1052A) strain was generated by transformation of the Sp trm7Δ::kanMX heterozygous diploid with pMG360A [Pnmt1 Sp trm7+ura4+], followed by sporulation on EMM lacking uracil supplemented with 500 mg/L G418, selection of haploids, and PCR confirmation of the knockout. The haploid S. pombe trm732Δ::kanMX mutant strain (yMG958B) was generated by sporulation of the heterzygous diploid on yeast extract with 3% dextrose (YE medium) with 200 mg/L G418, selection of haploids, and PCR confirmation. The haploid S. pombe trm734▵ [PSp trm734 Sp trm734+
ura4+] mutant strain (yMG1289-1) was generated by transformation of the Sp trm734Δ::kanMX heterozygous diploid with a LEU2 tRNAPhe plasmid (pMG308C), followed by sporulation on YE medium with 200 mg/L G418, selection of haploids and PCR verification. This strain was then transformed with pMG426G (PSp trm734 Sp trm734+
ura4+), followed by selection of leu−/ura+ colonies, and PCR confirmation. The S. cerevisiae trm7Δ::bleR [URA3 Sc TRM7] (yMG348-1), Sc trm732Δ::bleR (yMG814-1), Sc trm734Δ::bleR (yMG724-5), and Sc trm734Δ::bleR trm732Δ::kanMX [Sc TRM734 URA3 CEN] (yMG818-1) strains were described previously (Guy et al. 2012). Double mutant S. cerevisiae trm7Δ strains were constructed by PCR amplification of DNA from the appropriate YKO collection kanMX strain (Open Biosystems), followed by transformation of the DNA into yMG348-1.

Strains used in this study

PlasmidsPlasmids used in this study are listed in Table 7. The S. pombe vector expressing S. pombe trm7+ under control of the Pnmt1 (no message in thiamine) (pMG360A) and Pnmt1** (low strength no message in thiamine) promoters (pMG527B) were constructed by PCR of the Sp trm7+ cDNA ORF from an S. pombe cDNA library (Fikes et al. 1990) and insertion into the XhoI and SmaI sites of pREP4X or pREP81X vectors, respectively. The S. pombe vector expressing Sp trm734+ under control of the native Sp trm734+promoter (pMG426G) was constructed by PCR of Sp trm734+ from S. pombe genomic DNA and insertion at the PstI and XhoI sites of pREP4X to remove the Pnmt1 promoter sequence. tRNA expression vectors were generated by PCR of the appropriate tRNA fragment from genomic DNA and insertion at the Pst1 and XhoI sites of pREP3X. For expression in S. cerevisiae, human FTSJ1 and THADA ORFs were cloned from cDNA plasmids (Open Biosystems), as was D. melanogaster ORF CG5220 (Dm TRM7) (Drosophila Genomics Resource Center). S. pombe trm732+ ORF was cloned from an S. pombe cDNA library (Fikes et al. 1990). ORFs were then inserted by ligation independent cloning (LIC) into [2µ PGAL1,10] S. cerevisiae dual ORF expression vectors which express ORFs under PGAL1 control with a C-terminal PT tag (ORF-3C site-HA epitope-His6- ZZ domain of protein A), and ORFs under PGAL10 control with no tag, essentially as described previously (Quartley et al. 2009; Guy et al. 2012). All plasmids were confirmed by sequencing before use.TABLE 7.Plasmids used in this study

Plasmids used in this study are listed in Table 7. The S. pombe vector expressing S. pombe trm7+ under control of the Pnmt1 (no message in thiamine) (pMG360A) and Pnmt1** (low strength no message in thiamine) promoters (pMG527B) were constructed by PCR of the Sp trm7+ cDNA ORF from an S. pombe cDNA library (Fikes et al. 1990) and insertion into the XhoI and SmaI sites of pREP4X or pREP81X vectors, respectively. The S. pombe vector expressing Sp trm734+ under control of the native Sp trm734+promoter (pMG426G) was constructed by PCR of Sp trm734+ from S. pombe genomic DNA and insertion at the PstI and XhoI sites of pREP4X to remove the Pnmt1 promoter sequence. tRNA expression vectors were generated by PCR of the appropriate tRNA fragment from genomic DNA and insertion at the Pst1 and XhoI sites of pREP3X. For expression in S. cerevisiae, human FTSJ1 and THADA ORFs were cloned from cDNA plasmids (Open Biosystems), as was D. melanogaster ORF CG5220 (Dm TRM7) (Drosophila Genomics Resource Center). S. pombe trm732+ ORF was cloned from an S. pombe cDNA library (Fikes et al. 1990). ORFs were then inserted by ligation independent cloning (LIC) into [2µ PGAL1,10] S. cerevisiae dual ORF expression vectors which express ORFs under PGAL1 control with a C-terminal PT tag (ORF-3C site-HA epitope-His6- ZZ domain of protein A), and ORFs under PGAL10 control with no tag, essentially as described previously (Quartley et al. 2009; Guy et al. 2012). All plasmids were confirmed by sequencing before use.

Plasmids used in this study

Isolation and purification of tRNAS. pombe strains were grown at 30°C to mid-log phase in YE medium supplemented with 225 mg/L adenine, lysine, histidine, leucine, and uracil (YES), or in EMM with appropriate supplements at 225 mg/L. For analysis of tRNA from the Sp trm7Δ [Pnmt1** Sp trm7+
LEU2] and wild-type strains under repressive conditions, thiamine was added to EMM-Leu at 5 mg/L. S. cerevisiae strains were grown at 30°C to mid-log phase in S dropout medium containing 2% raffinose and 2% galactose. Bulk low molecular weight RNA was extracted from 300 OD-mL pellets, and appropriate 5′ biotinylated oligonucleotides were used to purify tRNA as previously described (Jackman et al. 2003).

Isolation and purification of tRNA

S. pombe strains were grown at 30°C to mid-log phase in YE medium supplemented with 225 mg/L adenine, lysine, histidine, leucine, and uracil (YES), or in EMM with appropriate supplements at 225 mg/L. For analysis of tRNA from the Sp trm7Δ [Pnmt1** Sp trm7+
LEU2] and wild-type strains under repressive conditions, thiamine was added to EMM-Leu at 5 mg/L. S. cerevisiae strains were grown at 30°C to mid-log phase in S dropout medium containing 2% raffinose and 2% galactose. Bulk low molecular weight RNA was extracted from 300 OD-mL pellets, and appropriate 5′ biotinylated oligonucleotides were used to purify tRNA as previously described (Jackman et al. 2003).

HPLC and analysis of tRNAPurified tRNA was digested with P1 nuclease and phosphatase as previously described, and nucleosides were subjected to HPLC analysis essentially as previously described (Jackman et al. 2003). For tRNAPhe, HPLC was done at pH 7.0 to maximize separation of Gm and m1G as previously described (Guy et al. 2012). For detection of yW, HPLC was performed with buffers and gradients essentially as previously described (Noma et al. 2006).

HPLC and analysis of tRNA

Purified tRNA was digested with P1 nuclease and phosphatase as previously described, and nucleosides were subjected to HPLC analysis essentially as previously described (Jackman et al. 2003). For tRNAPhe, HPLC was done at pH 7.0 to maximize separation of Gm and m1G as previously described (Guy et al. 2012). For detection of yW, HPLC was performed with buffers and gradients essentially as previously described (Noma et al. 2006).
