---
pmid: '25293756'
title: A proteomic strategy for global analysis of plant protein complexes.
authors:
- Aryal UK
- Xiong Y
- McBride Z
- Kihara D
- Xie J
- Hall MC
- Szymanski DB
journal: Plant Cell
year: '2014'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC4247564
doi: 10.1105/tpc.114.127563
---

# A proteomic strategy for global analysis of plant protein complexes.
**Authors:** Aryal UK, Xiong Y, McBride Z, Kihara D, Xie J, Hall MC, Szymanski DB
**Journal:** Plant Cell (2014)
**DOI:** [10.1105/tpc.114.127563](https://doi.org/10.1105/tpc.114.127563)
**PMC:** [PMC4247564](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4247564/)

## Abstract

1. Plant Cell. 2014 Oct;26(10):3867-82. doi: 10.1105/tpc.114.127563. Epub 2014
Oct  7.

A proteomic strategy for global analysis of plant protein complexes.

Aryal UK(1), Xiong Y(2), McBride Z(2), Kihara D(3), Xie J(4), Hall MC(1), 
Szymanski DB(5).

Author information:
(1)Department of Biochemistry, Purdue University, West Lafayette, Indiana 47907.
(2)Department of Biological Sciences, Purdue University, West Lafayette, Indiana 
47907.
(3)Department of Biological Sciences, Purdue University, West Lafayette, Indiana 
47907 Department of Computer Science, Purdue University, West Lafayette, Indiana 
47907.
(4)Department of Statistics, Purdue University, West Lafayette, Indiana 47907.
(5)Department of Biological Sciences, Purdue University, West Lafayette, Indiana 
47907 Department of Agronomy, Purdue University, West Lafayette, Indiana 47907 
dszyman@purdue.edu.

Global analyses of protein complex assembly, composition, and location are 
needed to fully understand how cells coordinate diverse metabolic, mechanical, 
and developmental activities. The most common methods for proteome-wide analysis 
of protein complexes rely on affinity purification-mass spectrometry or yeast 
two-hybrid approaches. These methods are time consuming and are not suitable for 
many plant species that are refractory to transformation or genome-wide cloning 
of open reading frames. Here, we describe the proof of concept for a method 
allowing simultaneous global analysis of endogenous protein complexes that 
begins with intact leaves and combines chromatographic separation of extracts 
from subcellular fractions with quantitative label-free protein abundance 
profiling by liquid chromatography-coupled mass spectrometry. Applying this 
approach to the crude cytosolic fraction of Arabidopsis thaliana leaves using 
size exclusion chromatography, we identified hundreds of cytosolic proteins that 
appeared to exist as components of stable protein complexes. The reliability of 
the method was validated by protein immunoblot analysis and comparisons with 
published size exclusion chromatography data and the masses of known complexes. 
The method can be implemented with appropriate instrumentation, is applicable to 
any biological system, and has the potential to be further developed to 
characterize the composition of protein complexes and measure the dynamics of 
protein complex localization and assembly under different conditions.

© 2014 American Society of Plant Biologists. All rights reserved.

DOI: 10.1105/tpc.114.127563
PMCID: PMC4247564
PMID: 25293756 [Indexed for MEDLINE]

## Full Text

Abstract

Global analyses of protein complex assembly, composition, and location are needed to fully understand how cells coordinate diverse metabolic, mechanical, and developmental activities. The most common methods for proteome-wide analysis of protein complexes rely on affinity purification-mass spectrometry or yeast two-hybrid approaches. These methods are time consuming and are not suitable for many plant species that are refractory to transformation or genome-wide cloning of open reading frames. Here, we describe the proof of concept for a method allowing simultaneous global analysis of endogenous protein complexes that begins with intact leaves and combines chromatographic separation of extracts from subcellular fractions with quantitative label-free protein abundance profiling by liquid chromatography-coupled mass spectrometry. Applying this approach to the crude cytosolic fraction of Arabidopsis thaliana leaves using size exclusion chromatography, we identified hundreds of cytosolic proteins that appeared to exist as components of stable protein complexes. The reliability of the method was validated by protein immunoblot analysis and comparisons with published size exclusion chromatography data and the masses of known complexes. The method can be implemented with appropriate instrumentation, is applicable to any biological system, and has the potential to be further developed to characterize the composition of protein complexes and measure the dynamics of protein complex localization and assembly under different conditions.

INTRODUCTION

Protein complexes, defined as the quaternary structure of multiple polypeptides that physically associate with one another, are the cornerstones of cellular control mechanisms ( Alberts, 1998 ; Srere, 2000 ). Invariably, plant cells use combinations of protein complexes to adjust their metabolism ( Winkel, 2004 ), growth ( Szymanski, 2005 ; Ingram and Waites, 2006 ), and physiology ( Yi and Deng, 2005 ) as a function of ever-changing developmental states and environmental conditions. In this context, protein complexes can be considered as fundamental building blocks of cell biology that enable precise control within and between cellular pathways ( Hartwell et al., 1999 ; Good et al., 2011 ). This view is widely supported by genetic data, in which null alleles of different subunits of a protein complex often display identical phenotypes, and reveal genes and proteins that function as part of a common pathway. Therefore, in order to gain broad insight into the function and integration of cellular pathways, deep knowledge about protein complex formation and dynamics is needed.

This is a tall order to fill. In Arabidopsis thaliana leaves, roughly ∼16,000 genes are expressed in specific cell types ( Marks et al., 2009 ) and there are no unifying rules for protein oligomerization: The complexes are diverse in terms of composition, stability, and function ( Nooren and Thornton, 2003 ). For example, many protein complexes assemble to do mechanical work at a specific place and time. The highly organized long-distance intracellular transport system of the cell has, at its core, protein complexes containing myosin motors that link regulated cargo selection to ATP-dependent transport on another protein complex, the actin cytoskeleton. Many other protein complexes use ATP to function in the context of protein complex remodeling ( Kressler et al., 2008 ) or protein turnover ( Pickart and Cohen, 2004 ). In other instances, complex mechanical tasks are executed by evolutionarily conserved protein complexes that cooperate, for example, to drive specific organelle fusion events ( Ohya et al., 2009 ; Stroupe et al., 2009 ), generate transport vesicles at specialized subdomains of organelle surfaces ( Barlowe et al., 1994 ; Kaksonen et al., 2003 ), or segregate replicated chromosomes during cell division ( Karsenti and Vernos, 2001 ; Masoud et al., 2013 ). Metabolic pathways and enzyme systems are strongly affected by protein complex formation. Homo-oligomeric protein complexes can create new intermolecular interfaces and highly efficient enzyme complexes with clustered active sites ( Marianayagam et al., 2004 ; Chen et al., 2011 ). Hetero-oligomeric protein complexes that contain sequential enzymes in a metabolic pathway can shield metabolites from the bulk cytosol and promote metabolite flux into specific final products ( Srere, 1987 ). In other instances, protein complex formation can negatively regulate an enzyme, alter its substrate specificity, or dictate its subcellular localization ( Winkel, 2004 ).

The biological significance of protein complex formation is perhaps most frequently considered in the context of signal transduction. The protein complex hardware for information flow assembles through regulated physical interactions among subunits that give the complex stability, yet allow conformational or compositional changes in response to ligand binding or other types of input signals. During signaling, protein complexes function dynamically as coincidence detectors to efficiently couple signal inputs from multiple pathways into a specific output behavior in the cell ( Stradal and Scita, 2006 ; Pawson, 2007 ). Distribution of shared subunits among protein complexes with distinct functions is another mechanism to coordinate information flow and cellular activities ( Krause et al., 2004 ; Huberts and van der Klei, 2010 ). The highly specific, contingent, and plastic behaviors of protein complexes generate robust signaling systems that could not be constructed with monomeric proteins and posttranslational modifications.

Protein complexes are typically assembled through distinct binary interactions between pairs of subunits. Binding of two proteins cannot be effectively gleaned from genome sequencing and bioinformatics-based protein predictions alone. Consequently, extensive effort has been directed toward generating proteome-wide maps of protein-protein interactions in model organisms from all kingdoms of life. Clever and effective techniques for large-scale detection of pairwise protein-protein interactions have been developed, such as phage display ( Jacobsson and Frykberg, 1996 ), protein chips ( Zhu et al., 2001 ), and the yeast two-hybrid system ( Fields and Sternglanz, 1994 ; Jansen et al., 2003 ). Successful large-scale yeast two-hybrid screens of soluble proteins have been conducted in the model plant Arabidopsis ( Yu et al., 2008 ; Mukhtar et al., 2011 ); however, there are technical limitations to the yeast two-hybrid system ( Jansen and Gerstein, 2004 ; Wodak et al., 2009 ), and it is only capable of detecting binary interactions. Tandem affinity purification ( TAP ) ( Rigaut et al., 1999 ) and related affinity purification strategies employ trans-gene expression technology to introduce affinity tags that allow biochemical isolation of all proteins in a stable complex via a single uniform protocol followed by mass spectrometry ( MS ) to identify copurifying proteins ( Gavin et al., 2006 ). In plants, TAP has been used to efficiently discover protein complexes and analyze important aspects of cell cycle control, 14-3-3 client proteins, and other basic cellular processes ( Braun et al., 2013 ). A disadvantage of TAP is that each tagged protein must be analyzed individually, and broad proteome coverage would require the generation of thousands of transgenic lines and greater numbers of parallel purifications and MS analyses. The expression level of the tagged proteins and the effect of the affinity tag on protein function are important technical considerations for this technology ( Wodak et al., 2009 ). Importantly, TAP is not possible in most crop plants because it is not feasible to generate the enormous number of high-quality transgenic lines that would be needed.

An alternative approach to protein complex analysis is to capitalize on the parallel protein detection and quantification capability of liquid chromatography-tandem mass spectrometry ( LC-MS/MS ) to analyze endogenous protein complexes ( Dong et al., 2008 ; Liu et al., 2008 ). In plants, the potential benefits of label-free MS protein quantification have been discussed ( Thelen and Peck, 2007 ), and these methods have been used to analyze the size and putative composition of endogenous protein complexes in the chloroplast stroma that were separated by native gel electrophoresis ( Peltier et al., 2006 ) and size exclusion chromatography ( SEC ) ( Olinares et al., 2010 ). This powerful approach requires only a well-annotated genome sequence and retains useful information on subcellular localization. In order to use profiling-based methods to predict protein complex composition from a cell fraction, hundreds of LC-MS/MS samples from different column fractions would be needed to sufficiently reduce the confounding effects of chance coelution. Such strategies are being developed in human cell culture systems, in which MS -based abundance profiling of native complexes has been coupled with different types of column chromatography and bioinformatic analyses to make global predictions about protein complex composition ( Havugimana et al., 2012 ; Kristensen et al., 2012 ) or analyze how protein complexes can change as a function of growth factor treatment ( Kristensen et al., 2012 ) or posttranscriptional control ( Kirkwood et al., 2013 ). However, at present, the extent to which LC-MS/MS -based profiling methods are sufficient to accurately predict the interactome is not known, and similar methods have not been applied to an intact organ, which could provide a useful physiological system to analyze protein complex behaviors.

We seek to develop a protein complex analysis technology for plants that couples cell fractionation and multidimensional chromatographic separation of protein complexes with label-free LC-MS/MS protein abundance profiling across all chromatography fractions. As a first step toward this goal, we begin with intact leaves of Arabidopsis and report on the use of LC-MS/MS to measure the relative abundance profiles and apparent masses of soluble proteins following their separation by SEC . The cytosol is an important intracellular compartment for cellular organization and metabolism and has been previously subjected to a proteomic analysis in Arabidopsis protoplasts ( Ito et al., 2011 ). Application of our method to the crude cytosolic fraction from leaf extracts revealed hundreds of proteins with apparent masses indicative of stable multimeric complexes, many of which appear to be novel. This article demonstrates the feasibility of using label-free quantitative MS to simultaneously analyze the oligomerization state and localization of endogenous protein complexes and sets the stage for the addition of other chromatographic steps that should allow the determination of protein complex composition.

DISCUSSION

Plant cells respond to endogenous signals and changing environmental conditions as a system in which the activities of thousands of protein complexes are modulated to generate an adaptive response. Accordingly, there is a strong need to develop new methods to measure the size, location, and composition of protein complexes, as well as monitor their dynamics under different conditions. The ability of mass spectrometry to simultaneously detect and quantify thousands of proteins makes it an appealing technology to address this need. For example, it has already been shown that organelle purification, coupled with native gel electrophoresis and quantitative protein MS , can identify proteins that reside in large, megadalton complexes in the chloroplast stroma ( Olinares et al., 2010 ). In this article, we began with intact leaves and combined cell fractionation, SEC , and LC-MS/MS to globally analyze the oligomeric state of cytosolic proteins. This method is reliable and unbiased because it monitors the assembly status of endogenous protein complexes on an SEC column and only requires a well-annotated genome to enable accurate protein identification. Therefore, it is easily applicable to many species and subcellular fractions other than the cytosol.

Our protein complex analysis platform provides reliable estimates for the mass of soluble proteins. The technical advance is that the apparent masses of ∼1300 leaf-expressed proteins were determined in parallel, using a gel-free, label-free, LC-MS/MS workflow. This analysis provided cytosol localization and oligomerization data for 83 proteins of unknown function, many of which appear to be in a protein complex. Although we used spectral counts to quantify MS signal and define chromatographic profiles, other label-free quantitative approaches, including MS1 extracted ion chromatogram integration, or new data-independent acquisition, will likely provide better quantification. Overall, our measurements of protein mass were reproducible and accurate. Our method was validated by comparing the LC-MS/MS -derived mass estimations with those obtained by protein immunoblots ( Figures 4B and 4C ) and those reported in previous studies ( Figures 4F and 4G ). Using the proteasome as an example, we demonstrated the precise cofractionation of subunits of a large complex and the ability to distinguish distinct forms of a complex ( Figures 4D and 4E ). Importantly, we demonstrated the ability to use a clustering algorithm to identify those proteins with highly similar chromatographic elution profiles. This analysis will be critical as we further develop this technology to identify the components of novel complexes.

We used our apparent mass measurements of proteins in the soluble cytosolic fraction to predict whether or not each exists in a stable protein complex using the R app . Based on comparisons of our R app values to the subunit composition of known complexes, we demonstrated it to be a reasonable metric for oligomeric state, and it allowed us to define a population of proteins that putatively exist in stable multimeric complexes (those with an R app greater than 2.0). Furthermore, because an R app value of 2 is associated with a roughly 4 fraction shift in SEC mobility compared with its monomeric form, we believe these measurements and assignments are influenced negligibly by variation in the SEC and our LC-MS/MS analyses. We acknowledge that use of R app has limitations, and in some cases, we are likely to misidentify large proteins as monomeric in cases where they interact with proteins that are relatively small. However, overall our R app predictions agree very well with the oligomerization state of known protein complexes, and provide a useful benchmark for more directed studies.

Our proteomic pipeline detected many large, apparently novel, protein complexes ( Supplemental Table 2 ), a few of which will be highlighted below. In some instances, we detected proteins with broad elution profiles and multiple SEC peaks. For example, the glycolytic enzymes ALDOLASE and GAPC are distinct compared with other glycolytic enzymes in that their elution profiles are very broad and complex ( Figure 6B ). ALDOLASE and GAPC are known moonlighting proteins ( Lee et al., 2002 ; Huberts and van der Klei, 2010 ; Tunio et al., 2010 ), and their complex elution profiles might reflect the existence of compositionally and functionally distinct protein complexes. We may have also uncovered moonlighting functions for proteins involved in protein translation. There are 20 cytosolic aminoacyl t-RNA synthetases/ligases in Arabidopsis , and we detected 11 of them in our study ( Supplemental Figure 4 ), all of relatively low abundance. This result is expected because the primary function of the enzymes, the specific aminoacylation of tRNAs, occurs in the nucleus ( Lund and Dahlberg, 1998 ). Of the 11 tRNA ligases, two had an obvious quaternary structure. GLUTAMINYL-tRNA LIGASE/AT5G26710 is an 81-kD protein that was reproducibly detected in high mass fractions with R app values of 7.1 and 8.8 ( Supplemental Table 2 and Supplemental Figure 4 ). ISOLEUCINE-tRNA LIGASE/AT4G10320 is a 135-kD protein and is the only other tRNA ligase that had a peak abundance in high mass fractions. In many organisms, tRNA ligases have multiple functions that are independent of protein synthesis ( Jahn et al., 1992 ; Martinis et al., 1999 ; Park et al., 2005 ), and perhaps that is also the case in Arabidopsis .

The dehydrin proteins are classic transcriptional markers for signaling output of pathways that allow plants to effectively adapt to the osmotic and oxidative stresses associated with drought ( Saavedra et al., 2006 ). The dehydrins are a group of small, highly charged proteins present during normal growth conditions and that may interact with membranes during stress conditions. The dehydrin COR47 is a classic marker for osmotic stress ( Welin et al., 1995 ), and like other dehydrins, it is almost always analyzed in terms of its transcriptional upregulation in response to cold or osmotic stress. We detected the 30-kD COR47/AT1G20440 protein, a relatively abundant protein with reproducible elution peaks in the ∼200 kD mass fraction ( Supplemental Table 2 and Supplemental Figure 5 ). We also detected the Arabidopsis homolog of maize ( Zea mays ) TANGLED, which is an important cytoskeletal control protein that positions a specialized microtubule structure termed the preprophase band to define the future plane of cell division and location of new cell wall formation during cytokinesis ( Walker et al., 2007 ). The 49-kD TANGLED was weakly detected, but reproducibly peaked in high mass fractions in the two biological replicates ( Supplemental Table 2 and Supplemental Figure 5 ). Despite the variation in apparent mass, our data indicated that TANGLED exists in a relatively large, stable, oligomeric cytosolic complex. TANGLED is recruited from a cytosolic pool to the preprophase band in a cell-cycle-dependent manner ( Rasmussen et al., 2011 ). It will be important to determine how the oligomerization state of TANGLED and its cytosolic pool relates to its cortical functions during cell division.

Our experimental system is expected to detect proteins that are partitioned between the cytosol and the nucleoplasm; indeed, there were a number of proteins with known nuclear functions as apparent subunits of large cytosolic complexes. We cannot rule out the possibility that this is an artifact of contamination; however, we did not detect abundant nuclear proteins such as histones, high mobility group chromatin-associated proteins, or multiple subunits of RNA polymerase complexes. We therefore propose that our data set reveals nuclear proteins that may either be regulated by cytosolic localization or they may have independent functions in the cytosol. We may have uncovered a cytosolic function for a domesticated transposase. We detected the hAT-family transposase named DAYSLEEPER/AT3G42170 in a large cytosolic complex ( Supplemental Table 2 and Supplemental Figure 6 ). It is a domesticated transposase that is present in angiosperms and has a proven importance for plant development in Arabidopsis ( Bundock and Hooykaas, 2005 ; Knip et al., 2012 ). We detected the 30-kD SC35-LIKE SPLICING FACTOR 30A/SCL30A as a component of a large complex greater than 700 kD in both biological replicates that may have cytosolic functions ( Supplemental Table 2 ). We also detected putative transcription factors in our analysis of cytosolic protein complexes. An 86-kD myb-like TRF class putative transcription factor (AT1G58220) was detected in a high mass complex greater than 700 kD in both replicates ( Supplemental Table 2 and Supplemental Figure 6 ). The expression of AT1G58220 is induced by jasmonic acid and auxin ( Yanhui et al., 2006 ). Perhaps the nuclear localization and function of AT1G58220 is conditional and its subcellular partitioning is mediated by this large cytosolic protein complex. All of these complexes merit further study.

One immediate application of our method is to analyze the oligomerization state of enzymes that function within a specific metabolic pathway and how it might change under different growth conditions. For example, glycolysis is perhaps the most ancient and conserved pathway in primary metabolism and in many instances enzyme activity is related to oligomerization ( Plaxton, 1996 ). There is experimental evidence in plant and microbial systems for physical associations among enzymes in the pathway that could mediate substrate channeling and/or efficient delivery of pyruvate, the end product of the pathway, into mitochondria ( Brandina et al., 2006 ; Gavin et al., 2002 ). In the hypothetical extreme case in which substrate channeling is an inherent and stable property of all enzymes in the glycolytic pathway, one would expect to detect coelution of sequential enzymes. This overly simplistic regulatory scheme is not supported by our experimental data ( Figure 6 ). For example, PHOSPHOGLUCOMUTASE (PGM), SUGAR ISOMERASE (SIS), and PHOSPHOFRUCTOKINASE (PGK) are sequential enzymes in the early steps of the pathway and they all have distinct elution peaks with PGM and SIS detected as monomers ( Figure 6B ). Along similar lines, the later acting glycolytic enzymes, PHOSPHOGLYCERATE KINASE (PGK), PHOSPHOGLYCERATE MUTASE (GPM), and ENOLASE2 (ENOS2) have distinct elution profiles and therefore do not appear to exist primarily as stable subunits of a hetero-oligomeric complex. However, higher order protein complexes containing sequential enzymes may exist as transient complexes that are not detected using our method, or they may assemble more efficiently on organelle surfaces ( Giegé et al., 2003 ; Graham et al., 2007 ).

In conclusion, the work described here provides a strong starting point for the parallel analysis of protein complexes using gel-free, quantitative MS . This particular SEC -based application provides a new way to globally analyze the oligomerization state of a system of protein complexes. Our data set points to the existence of many new, unexplored, protein complexes. We hope that these biochemical road maps to protein complexes will be used by the research community to accelerate the analysis of specific proteins of interest. This protein complex analysis method can also be expanded to analyze the dynamics of a large population of cytosolic proteins under different developmental or environmental conditions. The application also needs to be adapted to include the many plant protein complexes that associate with membrane surfaces. These microsomal fractions were discarded in this study. We already have developed robust methods for protein complex solubilization and SEC separation ( Basu et al., 2008 ; Kotchoni et al., 2009 ), and we will expand this technology to include organelle-associated proteins.

A major challenge in the field is to use LC-MS/MS profiling to globally predict protein complex composition ( Havugimana et al., 2012 ; Kristensen et al., 2012 ; Kirkwood et al., 2013 ). In the future, we will combine orthogonal chromatographic separations (either serially or in parallel) with LC-MS/MS profiling to extend the resolution space for stable complexes. Development of clustering analyses for the multidimensional chromatographic separations, combined with the quantitative spectral information and bioinformatic tools is expected to allow precise definition of subunit composition in novel protein assemblies. Other improvements will include RUBISCO depletion methods ( Aryal et al., 2012 ) to minimize suppression of cofractionating proteins, more extensive proteome coverage using improved analytical instrumentation (liquid chromatography and MS ), and improved accuracy of quantitative elution profiles by replacement of spectral counting with MS1 peak integration of detected peptides. These efforts will provide powerful new tools to broadly analyze the composition, localization, and dynamics of protein complex systems in the cell.
