---
pmid: '22634751'
title: Mutations in the PCNA-binding domain of CDKN1C cause IMAGe syndrome.
authors:
- Arboleda VA
- Lee H
- Parnaik R
- Fleming A
- Banerjee A
- Ferraz-de-Souza B
- Délot EC
- Rodriguez-Fernandez IA
- Braslavsky D
- Bergadá I
- Dell'Angelica EC
- Nelson SF
- Martinez-Agosto JA
- Achermann JC
- Vilain E
journal: Nat Genet
year: '2012'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC3386373
doi: 10.1038/ng.2275
---

# Mutations in the PCNA-binding domain of CDKN1C cause IMAGe syndrome.
**Authors:** Arboleda VA, Lee H, Parnaik R, Fleming A, Banerjee A, Ferraz-de-Souza B, Délot EC, Rodriguez-Fernandez IA, Braslavsky D, Bergadá I, Dell'Angelica EC, Nelson SF, Martinez-Agosto JA, Achermann JC, Vilain E
**Journal:** Nat Genet (2012)
**DOI:** [10.1038/ng.2275](https://doi.org/10.1038/ng.2275)
**PMC:** [PMC3386373](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3386373/)

## Abstract

1. Nat Genet. 2012 May 27;44(7):788-92. doi: 10.1038/ng.2275.

Mutations in the PCNA-binding domain of CDKN1C cause IMAGe syndrome.

Arboleda VA(1), Lee H, Parnaik R, Fleming A, Banerjee A, Ferraz-de-Souza B, 
Délot EC, Rodriguez-Fernandez IA, Braslavsky D, Bergadá I, Dell'Angelica EC, 
Nelson SF, Martinez-Agosto JA, Achermann JC, Vilain E.

Author information:
(1)Department of Human Genetics, David Geffen School of Medicine, University of 
California, Los Angeles, California, USA.

Comment in
    Nat Genet. 2012 Jun 27;44(7):737-8. doi: 10.1038/ng.2336.

IMAGe syndrome (intrauterine growth restriction, metaphyseal dysplasia, adrenal 
hypoplasia congenita and genital anomalies) is an undergrowth developmental 
disorder with life-threatening consequences. An identity-by-descent analysis in 
a family with IMAGe syndrome identified a 17.2-Mb locus on chromosome 11p15 that 
segregated in the affected family members. Targeted exon array capture of the 
disease locus, followed by high-throughput genomic sequencing and validation by 
dideoxy sequencing, identified missense mutations in the imprinted gene CDKN1C 
(also known as P57KIP2) in two familial and four unrelated patients. A familial 
analysis showed an imprinted mode of inheritance in which only maternal 
transmission of the mutation resulted in IMAGe syndrome. CDKN1C inhibits 
cell-cycle progression, and we found that targeted expression of 
IMAGe-associated CDKN1C mutations in Drosophila caused severe eye growth defects 
compared to wild-type CDKN1C, suggesting a gain-of-function mechanism. All 
IMAGe-associated mutations clustered in the PCNA-binding domain of CDKN1C and 
resulted in loss of PCNA binding, distinguishing them from the mutations of 
CDKN1C that cause Beckwith-Wiedemann syndrome, an overgrowth syndrome.

DOI: 10.1038/ng.2275
PMCID: PMC3386373
PMID: 22634751 [Indexed for MEDLINE]

Conflict of interest statement: Competing financial interests The authors 
declare no competing financial interests.

## Full Text

IntroductoryIMAGE Syndrome (Intrauterine growth restriction, Metaphyseal dysplasia, Adrenal hypoplasia congenita, and Genital anomalies) is an undergrowth developmental disorder with life-threatening consequences1. Identity-by-descent analysis in a family with IMAGE syndrome2 identified a 17.2 megabase (Mb) locus on 11p15 that segregated in affected family members. Targeted exon array capture of the disease locus, followed by high-throughput genomic sequencing and validated by dideoxysequencing, identified missense mutations in imprinted gene CDKN1C (P57KIP2) in two familial and four unrelated patients. Familial analysis demonstrated an imprinted mode of inheritance where only maternal transmission of the mutation resulted in IMAGE syndrome. CDKN1C inhibits cell-cycle progression3 and targeted expression of IMAGE-associated CDKN1C mutations in Drosophila caused severe eye growth defects compared to wild type CDKN1C, suggesting a gain-of-function mechanism. All IMAGE-associated mutations clustered in the PCNA-binding domain of CDKN1C and resulted in loss of PCNA binding, distinguishing them from CDKN1C mutations that cause Beckwith-Wiedemann Syndrome, an overgrowth syndrome4.Since the initial description of IMAGE syndrome (OMIM #300290)1, a number of isolated and familial cases have been reported1,5–11. In order to identify a causative gene for IMAGE syndrome, we performed a 250K Nsp Affymetrix SNP Array on seven affected and one unaffected sibling from Family A, a five-generation family from Argentina (Fig. 1a). Further analysis using a custom script detected a 17.2 Mb identical-by-descent (IBD) region on chromosome 11, shared by seven affected family members but not an unaffected sibling (Fig. 1b), with a LOD score of 5.4. Despite the multisystem involvement of IMAGE syndrome, we did not identify a contiguous gene deletion or duplication in the affected individuals (Supplementary Fig. 1).To determine the causative mutation, we performed targeted high-throughput genomic sequencing of all the exons within a conservative IBD region. An Agilent 244K custom CGH array was designed to capture all exons and splice sites within the region spanning 0–22.6 Mb on chromosome 11. In total, five custom bar-coded genomic DNA libraries (Family A, V-1 and V-6; unrelated Patients 1, 2, and 3) were prepared, pooled, and captured on a single custom array, and the DNA enriched for the IBD region was sequenced on one lane of the Illumina Genome Analyzer II. Our targeted approach yielded an average coverage of 32x. Patient 3 had a significantly lower coverage of 9x across the targeted intervals and was not used in the initial bioinformatics analysis. In the remaining four samples analyzed, ~85% of all targeted regions were covered at ≥10x.The pedigree and IBD analysis led us to hypothesize that IMAGE syndrome was inherited as a rare autosomal dominant disorder. Our bioinformatics analysis required that both V-1 and V-6 share the same rare gene variant, and at least one of the non-related IMAGE syndrome patients harbor a rare variant (defined as a variant not present in dbSNP129) in the same gene. This approach identified a single gene, CDKN1C.Upon further examination, we noted that CDKN1C was captured and sequenced at a much lower rate compared to other targeted genes due to a high GC-content of ≥80%. To compound this low gene coverage, we re-sequenced CDKN1C by dideoxysequencing (primers listed in Supplementary Table 1) in all five individuals sequenced by high-throughput sequencing and in an additional sporadic case (Patient 4). Affected individuals from Family A carried a c.825T>G change resulting in a F276V missense mutation. The four unrelated patients with IMAGE syndrome harbored four mutations in CDKN1C: F276S, R279P, D274N, and K278E (Table 1). In total, we identified five rare heterozygous missense mutations in CDKN1C that cluster within six amino acids of the PCNA-binding domain12 (Fig. 2a). All variants localized to a highly conserved region13 (Fig. 2b) and are predicted to be “damaging” to the structure and function of CDKN1C by Polyphen analysis14.CDKN1C is located on chromosome 11 and encodes a protein known to play a key role in inhibiting cell cycle progression. In most tissues, the paternal allele is repressed by distant imprinting control regions, such that expression is primarily from the maternal allele15,16. Inheritance of IMAGE syndrome in Family A was only through maternal transmission of the CDKN1C mutation (Fig. 1a). Sequencing for the c.825T>G mutation in 24 members from Family A confirmed that only individuals who inherited the mutation on the maternal allele are affected. A c.825T>G mutation inherited on the paternal allele was not expressed, presumably because of epigenetic silencing of the mutated allele.To confirm the pathogenicity of these mutations, we used an in vivo functional model in which IMAGE-associated human CDKN1C mutants were expressed in Drosophila melanogaster using the GAL4 UAS system17. Ubiquitous overexpression of wild type or mutant CDKN1C resulted in early larval lethality. Targeted expression of the IMAGE-associated CDKN1C mutants resulted in altered wing vein patterning and decreased wing size (see Supplementary Fig. 2 & 3). Expression of wild type CDKN1C restricted to the eye did not have any effects on adult eye size, while expression of IMAGE-associated CDKN1C mutants showed a moderate to severe reduction in eye size (Fig. 3).Overexpression of IMAGE-associated CDKN1C mutants in HEK293T cells did not interfere with the ability of CDKN1C to inhibit the cell cycle in G0/G1 through binding of the CDK-domain (Supplementary Fig. 4). These data suggest that IMAGE-mutations within the PCNA-binding domain do not inhibit cell-cycle progression and likely act through a different mechanism resulting in IMAGE syndrome.CDKN1C is located within an imprinted cluster of genes that regulate prenatal and postnatal growth and development. Genetic alterations in CDKN1C have been shown to give rise to Beckwith-Wiedemann Syndrome (BWS; OMIM #130650), an overgrowth disorder18,19 Here, we show that an undergrowth condition, IMAGE syndrome, is caused by domain-specific mutations in the maternally inherited allele of CDKN1C.Both BWS human patients and CDKN1C−/− knockout mice exhibit adrenal hyperplasia20,21 in contrast to the adrenal hypoplasia in IMAGE patients. We therefore verified that CDKN1C mRNA and protein are expressed in the developing human adrenal gland (Fig. 4). Quantitative RT-PCR demonstrated that the expression of CDKN1C is greater in adrenal tissue during early human development than in brain or muscle (Fig. 4a). Immunohistochemistry showed strongest expression of CDKN1C within a subset of cells in the subcapsular or developing definitive zone of the adrenal gland (Fig. 4b).To determine if BWS and IMAGE mutations work through the same mechanism, we repeated the above functional studies with BWS-specific mutants. In the cell cycle analysis, transfection of the BWS-mutant CDKN1C-L42P resulted in a loss of cell cycle inhibition at G0/G1. Ubiquitous expression of BWS-associated CDKN1C mutations in Drosophila melanogaster were early larval lethal, however targeted expression had no effect on eye size, wing size, or wing vein patterning (Fig. 3g,h, Supplementary Fig. 2e,2f,3e,3f). Thus, in vitro and in vivo, BWS mutants have different effects relative to the IMAGE mutants suggesting that domain-specific mutations have differential effects on cell cycle progression and developmental processes.BWS-associated CDKN1C mutations are either missense mutations localized to the cyclin-dependent kinase binding domain or nonsense mutations, both of which result in protein loss-of-function, over proliferation, and predisposition to cancer22 due to loss of cell-cycle inhibition3. In contrast, we show that missense mutations localized to a highly conserved region of the PCNA-binding domain of CDKN1C in IMAGE syndrome resulted in excess inhibition of growth and differentiation—a gain of function (Fig. 5).Since two of the five mutations fall into a putative nuclear localization signal, we determined whether IMAGE mutants interfere with active nuclear transport of CDKN1C. H295R and M1 cells transfected with GFP-CDKN1C fusion constructs (Supplementary Fig. 5), showed that none of the tested IMAGE mutants interfered with active transport mechanisms or with binding affinity to α-importin.Since IMAGE mutations cluster in a domain known to bind PCNA, we performed co-immunoprecipitation experiments to test the effect of IMAGE mutations on PCNA binding. HEK293T cells were transfected with flag-tagged CDKN1C constructs bearing the wild type or IMAGE alleles (F276V and K278E). Endogenous PCNA was recovered from the wild type but not IMAGE immunoprecipitates (Fig. 4c), suggesting that PCNA binding is disrupted in mutants.Since one of the roles of PCNA is to facilitate ubiquitination of cell cycle proteins23, we investigated the role of IMAGE mutations in PCNA-dependent ubiquitination of CDKN1C. HEK293T cells were co-transfected with flag-tagged wild type or IMAGE-mutant F276V CDKN1C and with HA-tagged Ubiquitin (12 kDa) and subjected to co-immunoprecipitation. CDKN1C migrates at ~50 kDa, and therefore we expect the mono-, di-, and poly-ubiquitinated CDKN1C to migrate at ~62 kDa or higher depending on the number and the branching of the ubiquitin moieties. Here we show that a band at 63 kDa, the approximate size of a monoubiquitinated CDKN1C protein, is present in the wild type CDKN1C but absent in the IMAGE mutant sample (Fig.4d).Our data reveal a role for PCNA binding in a specific ubiquitination modification of CDKN1C. Many cell cycle proteins are subject to PCNA-dependent ubiquitination24, which has pleiotropic effects. Monoubiquitination, as observed in our data, may have a number of functional consequences, such as modulation of protein localization, of protein interactions, and of proteosomal degradation25,26,27. The latter is less likely because it typically requires, at minimum, tetra-ubiquitination28, but cannot be ruled out without information on CDKN1C protein stability.Next generation sequencing has emerged as a powerful tool in identifying rare Mendelian disease genes, utilizing existing linkage analysis data to identify a disease gene in an unbiased fashion. Our findings show that missense mutations in the PCNA-binding domain have an inhibitory effect on growth, via loss of binding of PCNA to CDKN1C, thereby altering ubiquitination of CDKN1C and presumably promoting its function. The contrast between BWS and IMAGE mutations in CDKN1C highlights the dual and opposing effects of specific CDKN1C mutations. Mutations within the PCNA-binding site of CDKN1C blocked in vivo growth and differentiation and may illuminate novel mechanisms regulating cell transformation12, tumor growth, and cell cycle progression.

Introductory

IMAGE Syndrome (Intrauterine growth restriction, Metaphyseal dysplasia, Adrenal hypoplasia congenita, and Genital anomalies) is an undergrowth developmental disorder with life-threatening consequences1. Identity-by-descent analysis in a family with IMAGE syndrome2 identified a 17.2 megabase (Mb) locus on 11p15 that segregated in affected family members. Targeted exon array capture of the disease locus, followed by high-throughput genomic sequencing and validated by dideoxysequencing, identified missense mutations in imprinted gene CDKN1C (P57KIP2) in two familial and four unrelated patients. Familial analysis demonstrated an imprinted mode of inheritance where only maternal transmission of the mutation resulted in IMAGE syndrome. CDKN1C inhibits cell-cycle progression3 and targeted expression of IMAGE-associated CDKN1C mutations in Drosophila caused severe eye growth defects compared to wild type CDKN1C, suggesting a gain-of-function mechanism. All IMAGE-associated mutations clustered in the PCNA-binding domain of CDKN1C and resulted in loss of PCNA binding, distinguishing them from CDKN1C mutations that cause Beckwith-Wiedemann Syndrome, an overgrowth syndrome4.

Since the initial description of IMAGE syndrome (OMIM #300290)1, a number of isolated and familial cases have been reported1,5–11. In order to identify a causative gene for IMAGE syndrome, we performed a 250K Nsp Affymetrix SNP Array on seven affected and one unaffected sibling from Family A, a five-generation family from Argentina (Fig. 1a). Further analysis using a custom script detected a 17.2 Mb identical-by-descent (IBD) region on chromosome 11, shared by seven affected family members but not an unaffected sibling (Fig. 1b), with a LOD score of 5.4. Despite the multisystem involvement of IMAGE syndrome, we did not identify a contiguous gene deletion or duplication in the affected individuals (Supplementary Fig. 1).

To determine the causative mutation, we performed targeted high-throughput genomic sequencing of all the exons within a conservative IBD region. An Agilent 244K custom CGH array was designed to capture all exons and splice sites within the region spanning 0–22.6 Mb on chromosome 11. In total, five custom bar-coded genomic DNA libraries (Family A, V-1 and V-6; unrelated Patients 1, 2, and 3) were prepared, pooled, and captured on a single custom array, and the DNA enriched for the IBD region was sequenced on one lane of the Illumina Genome Analyzer II. Our targeted approach yielded an average coverage of 32x. Patient 3 had a significantly lower coverage of 9x across the targeted intervals and was not used in the initial bioinformatics analysis. In the remaining four samples analyzed, ~85% of all targeted regions were covered at ≥10x.

The pedigree and IBD analysis led us to hypothesize that IMAGE syndrome was inherited as a rare autosomal dominant disorder. Our bioinformatics analysis required that both V-1 and V-6 share the same rare gene variant, and at least one of the non-related IMAGE syndrome patients harbor a rare variant (defined as a variant not present in dbSNP129) in the same gene. This approach identified a single gene, CDKN1C.

Upon further examination, we noted that CDKN1C was captured and sequenced at a much lower rate compared to other targeted genes due to a high GC-content of ≥80%. To compound this low gene coverage, we re-sequenced CDKN1C by dideoxysequencing (primers listed in Supplementary Table 1) in all five individuals sequenced by high-throughput sequencing and in an additional sporadic case (Patient 4). Affected individuals from Family A carried a c.825T>G change resulting in a F276V missense mutation. The four unrelated patients with IMAGE syndrome harbored four mutations in CDKN1C: F276S, R279P, D274N, and K278E (Table 1). In total, we identified five rare heterozygous missense mutations in CDKN1C that cluster within six amino acids of the PCNA-binding domain12 (Fig. 2a). All variants localized to a highly conserved region13 (Fig. 2b) and are predicted to be “damaging” to the structure and function of CDKN1C by Polyphen analysis14.

CDKN1C is located on chromosome 11 and encodes a protein known to play a key role in inhibiting cell cycle progression. In most tissues, the paternal allele is repressed by distant imprinting control regions, such that expression is primarily from the maternal allele15,16. Inheritance of IMAGE syndrome in Family A was only through maternal transmission of the CDKN1C mutation (Fig. 1a). Sequencing for the c.825T>G mutation in 24 members from Family A confirmed that only individuals who inherited the mutation on the maternal allele are affected. A c.825T>G mutation inherited on the paternal allele was not expressed, presumably because of epigenetic silencing of the mutated allele.

To confirm the pathogenicity of these mutations, we used an in vivo functional model in which IMAGE-associated human CDKN1C mutants were expressed in Drosophila melanogaster using the GAL4 UAS system17. Ubiquitous overexpression of wild type or mutant CDKN1C resulted in early larval lethality. Targeted expression of the IMAGE-associated CDKN1C mutants resulted in altered wing vein patterning and decreased wing size (see Supplementary Fig. 2 & 3). Expression of wild type CDKN1C restricted to the eye did not have any effects on adult eye size, while expression of IMAGE-associated CDKN1C mutants showed a moderate to severe reduction in eye size (Fig. 3).

Overexpression of IMAGE-associated CDKN1C mutants in HEK293T cells did not interfere with the ability of CDKN1C to inhibit the cell cycle in G0/G1 through binding of the CDK-domain (Supplementary Fig. 4). These data suggest that IMAGE-mutations within the PCNA-binding domain do not inhibit cell-cycle progression and likely act through a different mechanism resulting in IMAGE syndrome.

CDKN1C is located within an imprinted cluster of genes that regulate prenatal and postnatal growth and development. Genetic alterations in CDKN1C have been shown to give rise to Beckwith-Wiedemann Syndrome (BWS; OMIM #130650), an overgrowth disorder18,19 Here, we show that an undergrowth condition, IMAGE syndrome, is caused by domain-specific mutations in the maternally inherited allele of CDKN1C.

Both BWS human patients and CDKN1C−/− knockout mice exhibit adrenal hyperplasia20,21 in contrast to the adrenal hypoplasia in IMAGE patients. We therefore verified that CDKN1C mRNA and protein are expressed in the developing human adrenal gland (Fig. 4). Quantitative RT-PCR demonstrated that the expression of CDKN1C is greater in adrenal tissue during early human development than in brain or muscle (Fig. 4a). Immunohistochemistry showed strongest expression of CDKN1C within a subset of cells in the subcapsular or developing definitive zone of the adrenal gland (Fig. 4b).

To determine if BWS and IMAGE mutations work through the same mechanism, we repeated the above functional studies with BWS-specific mutants. In the cell cycle analysis, transfection of the BWS-mutant CDKN1C-L42P resulted in a loss of cell cycle inhibition at G0/G1. Ubiquitous expression of BWS-associated CDKN1C mutations in Drosophila melanogaster were early larval lethal, however targeted expression had no effect on eye size, wing size, or wing vein patterning (Fig. 3g,h, Supplementary Fig. 2e,2f,3e,3f). Thus, in vitro and in vivo, BWS mutants have different effects relative to the IMAGE mutants suggesting that domain-specific mutations have differential effects on cell cycle progression and developmental processes.

BWS-associated CDKN1C mutations are either missense mutations localized to the cyclin-dependent kinase binding domain or nonsense mutations, both of which result in protein loss-of-function, over proliferation, and predisposition to cancer22 due to loss of cell-cycle inhibition3. In contrast, we show that missense mutations localized to a highly conserved region of the PCNA-binding domain of CDKN1C in IMAGE syndrome resulted in excess inhibition of growth and differentiation—a gain of function (Fig. 5).

Since two of the five mutations fall into a putative nuclear localization signal, we determined whether IMAGE mutants interfere with active nuclear transport of CDKN1C. H295R and M1 cells transfected with GFP-CDKN1C fusion constructs (Supplementary Fig. 5), showed that none of the tested IMAGE mutants interfered with active transport mechanisms or with binding affinity to α-importin.

Since IMAGE mutations cluster in a domain known to bind PCNA, we performed co-immunoprecipitation experiments to test the effect of IMAGE mutations on PCNA binding. HEK293T cells were transfected with flag-tagged CDKN1C constructs bearing the wild type or IMAGE alleles (F276V and K278E). Endogenous PCNA was recovered from the wild type but not IMAGE immunoprecipitates (Fig. 4c), suggesting that PCNA binding is disrupted in mutants.

Since one of the roles of PCNA is to facilitate ubiquitination of cell cycle proteins23, we investigated the role of IMAGE mutations in PCNA-dependent ubiquitination of CDKN1C. HEK293T cells were co-transfected with flag-tagged wild type or IMAGE-mutant F276V CDKN1C and with HA-tagged Ubiquitin (12 kDa) and subjected to co-immunoprecipitation. CDKN1C migrates at ~50 kDa, and therefore we expect the mono-, di-, and poly-ubiquitinated CDKN1C to migrate at ~62 kDa or higher depending on the number and the branching of the ubiquitin moieties. Here we show that a band at 63 kDa, the approximate size of a monoubiquitinated CDKN1C protein, is present in the wild type CDKN1C but absent in the IMAGE mutant sample (Fig.4d).

Our data reveal a role for PCNA binding in a specific ubiquitination modification of CDKN1C. Many cell cycle proteins are subject to PCNA-dependent ubiquitination24, which has pleiotropic effects. Monoubiquitination, as observed in our data, may have a number of functional consequences, such as modulation of protein localization, of protein interactions, and of proteosomal degradation25,26,27. The latter is less likely because it typically requires, at minimum, tetra-ubiquitination28, but cannot be ruled out without information on CDKN1C protein stability.

Next generation sequencing has emerged as a powerful tool in identifying rare Mendelian disease genes, utilizing existing linkage analysis data to identify a disease gene in an unbiased fashion. Our findings show that missense mutations in the PCNA-binding domain have an inhibitory effect on growth, via loss of binding of PCNA to CDKN1C, thereby altering ubiquitination of CDKN1C and presumably promoting its function. The contrast between BWS and IMAGE mutations in CDKN1C highlights the dual and opposing effects of specific CDKN1C mutations. Mutations within the PCNA-binding site of CDKN1C blocked in vivo growth and differentiation and may illuminate novel mechanisms regulating cell transformation12, tumor growth, and cell cycle progression.

METHODSStudy SubjectsAll participants were patients diagnosed clinically with IMAGE syndrome. This study was approved by the Institutional Review Board (IRB) at the University of California, Los Angeles, the Hospital de Niños Ricardo Gutierrez in Argentina, or the Institute of Child Health at University College London. All participants provided informed consent. Phenotypes are summarized in Table 1.Identity-by-descent analysisGenomic DNA from 7 affected individuals (IV-10, V-1, V-2, V-5, V6, V-7, V-12) and 1 non-affected individual (V-13) from a large Argentine family (Family A) were genotyped on the Affymetrix 250K Nsp1 SNP arrays, as per manufacturers’ protocol. Familial relationships were confirmed through checking the sharing statistics in all pairs of samples and ancestral identity-by-descent (IBD) analysis was performed using a custom script (B. Merriman, available on request). The IBD analysis script was designed to search for long continuous intervals compatible with a common extended haplotype among all the affected individuals but not with the unaffected individuals.29 A conservative error rate of 1% was used to allow the algorithm to tolerate possible genotyping errors. A rare dominant model of inheritance was assumed with rare frequency of this haplotype of 0.1% and penetrance of 100% under the assumption that the carriers (parents of the affected individuals) were not showing the phenotype for a different reason (i.e. imprinting) than low penetrance of the disease phenotype.Capture and Sequencing of Genomic DNAFor the capture of the genes within the IBD interval, we used Agilent Custom 244K comparative genomic hybridization Array. The 60-bp oligonucleotide probes were tiled every 20-bp against all exonic regions on chromosome 11 between 2.45-20.15Mb (hg18, March 2006, build 36.1) and every 30-bp in the flanking 5’ and 3’ regions spanning 0-2.45 Mb and 20.15-22.6 Mb. We included all gene models identified in RefSeq, Genbank, CCDS and UniProt. Location and sequence of all probes are available upon request.Genomic DNA libraries were created for patients V-1 an V-6 from Family A and 3 isolated cases of IMAGE1 (Patient1, 2, and 3) following manufacturer’s protocol (Illumina protocol Preparing Samples for Sequencing Genomic DNA, p/n 11251892 Rev. A) except for the adaptor ligation step where we used custom-made, internally validated bar-coded adaptors. After the PCR amplification, five bar-coded libraries were pooled together and captured by hybridization to custom-designed CGH array, as described in 30. After the capture, the array was washed and the captured DNA was eluted, amplified and diluted to 10nM final concentration based on the Qubit concentration measure and Agilent Bioanalyzer. One flowcell lane of single-end sequencing was performed at the UCLA Genomic Sequencing Center on the Illumina Genome Analyzer II for 76x cycles following manufacturer’s protocol. The base-calling was performed by the real time analysis software (Illumina).Sequence Data AnalysisThe Illumina output files (.qseq) were converted to fastq formats using BFAST31,32 script ill2fastq.pl and then parsed into multiple fastq files, each for one unique barcode. Only the reads with 100% matching barcode sequences were carried over to the second set of fastq files. The sequence reads in each fastq file were aligned to the human reference genome (hg18, March 2006, build 36.1) using Novoalign from Novocraft Short Read Alignment Package. The output format was set to SAM and default settings were used for all options. Using SAMtools, the SAM file of each sample was converted to a BAM file, sorted and merged and potential PCR duplicates were removed using Picard33. The variants, both single nucleotide variants (SNVs) and small INDELs (insertions and deletions), within the captured coding exonic intervals were called using the SAMtools pileup tool. For SNV calling, the last 5 bases were trimmed and only the reads lacking INDELs were retained. For INDEL calling, only the reads that contained one contiguous INDEL, not occurring on either end of the read, were retained34. The variants were further annotated using the SeqWare project and loaded into the SeqWare QueryEngine database35. Variants from each sample with the following criteria were identified: (i) variant base or INDEL observed at least twice and at ≥5% of the total coverage per base, (ii) variant observed at least once on both forward and reverse strands, (iii) SNV quality score ≥10. As IMAGE syndrome is a rare condition, only variants with coding consequences not present in dbSNP129 were further analyzed.CDKN1C SequencingAll CDKN1C (RefSeq: NM_000076.2) mutations were PCRed using Phusion HF polymerase (NEB) with 5% DMSO and 0.1M Betaine. PCR products were sent for dideoxysequencing at Laragen, Inc. PCR primers used are listed in Supplementary Table 1. All mutation locations were reported using CCDS7738.1 and P49918 as the normal transcript and protein sequences, respectively.Plasmid ConstructsThe cDNA of CDKN1C cloned into pBluescript was purchased from ATCC(#99411)36. Mutations were generated using site-directed mutagenesis (Stratagene, primers in Supplementary Table 1). 4 mutants were created, corresponding to BWS mutations (p.L42P and c.826delT) and IMAGE syndrome mutations (p.F276V and p.K278E). Wild type and mutant versions of CDKN1C were subcloned into pCDNA3.1(+) and pEGFP-C2 for mammalian cell culture experiments, into pFLAG-pcDNA3.1 for the immunoprecipitation, and into pUAST for the generation of transgenic flies.Quantitative Reverse Transcription PCRHuman tissue from 7–8 weeks post-conception (wpc) was provided by the Medical Research Council/Wellcome Trust-funded Human Developmental Biology Resource with Research Ethics Committee approval and informed consent. RNA was extracted from adrenal, brain and muscle samples using the TRIzol reagent and first-strand cDNA was generated using SuperScriptR II reverse transcriptase (Invitrogen). Expression of CDKN1C transcript was assessed by quantitative PCR using the StepOnePlus Real-time PCR System, TaqManR Gene Expression Assays for human CDKN1C (Hs00175938_m1) and human GAPDH as endogenous control (4333764T; all Applied Biosystems). Data were analyzed with StepOne software v2.1 according to the 2-ΔΔCT method.ImmunofluorescenceFourteen-micron sections of human fetal adrenal tissue (8 weeks post conception) were fixed briefly in 4% PFA and blocked in 1% BSA before incubating overnight with antibody to CDKN1C (Fisher AFMA121866) and CYP11A1 (Sigma HPA016436). Primary antibodies were detected using Alexa647 goat anti-mouse (Invitrogen, A21235) and Alexa555 (Invitrogen, A21429) goat anti-rabbit conjugates. Nuclei were counterstained with DAPI. IMAGEs were collected on a Zeiss 710 confocal microscope (Carl Zeiss).ImmunoprecipitationHEK293T cells were transfected with constructs encoding Flag-CDKN1C with Lipofectamine 2000 (Invitrogen). For ubiquitination assays, cells were co-transfected with pCI-neo-(HA)3-human Ub37 construct and treated 3 hours in 10μM MG-132 (Millipore) prior to cell lysis. Flag-CDKN1C was immunoprecipitated from cell lysates using the ANTI-FLAG M2-Agarose Affinity Gel (Sigma-Aldrich). Western blot was performed on immunoprecipitated samples and cell lysates with primary antibody to HA (Covance MMS101R), Flag (Abcam, ab1162), PCNA (Abcam ab29), and CDKN1C (Santa Cruz biotechnology, sc-1040). Secondary antibodies used were goat-anti-mouse-HRP (Biorad, 1:20,000) and goat-anti-rabbit-HRP (Santa Cruz, 1:5,000).Drosophila experimentsFive independent constructs, CDKN1CWT, CDKN1CL42P, CKDN1C826delT, CDKN1CF276V, and CDKN1CK278E were injected into embryos and each construct generated multiple independent transgenic lines. Overexpression was achieved by using the GAL4-UAS system17 and the following drivers: Ubi-gal4 (ubiquitous expression), ey-gal4 (eye-specific expression), MS1096-gal4 (wing-specific expression), and salPE-gal4 (wing pouch-specific expression). MS1096-gal4 is expressed in the entire wing imaginal disc but at higher levels in the dorsal compartment. This higher level of expression in the dorsal compartment causes enhanced phenotypic effects in the dorsal versus the ventral side of the wing. salPE-gal4 is specifically expressed in the pouch of the wing disc, which only gives rise to the wing proper. All UAS-CDKN1C constructs were larval lethal when expressed with Ubi-gal4, confirming their expression efficiency. At least two independent lines were used for each experiment and all yielded similar results. All eye images were taken on a Leica Z16 AP0 Camera. Wing IMAGEs were taken on Leica DFC 300 FX R2 Camera.Flow CytometryWild-type or mutant versions of CDKN1C were co-transfected with pCMV-GFP into serum starved HEK293T cells using Lipofectamine 2000 (Invitrogen). After 24 hours, cells were grown in media containing 5% serum for 48-hours. Cells were resuspended in a hypotonic buffer with propidium iodide and GFP-positive events were analyzed on a Becton Dickinson FACScan Analytic Flow Cytometer. All experiments were performed in 2 biological replicates. Statistical significance was assessed using a 2-proportion z-test. Flow cytometry was performed in the UCLA Johansson Comprehensive Cancer Center (JCCC) and Center for AIDS Research Flow Cytometry Core Facility.Analysis of nuclear/cytoplasmic distribution of GFP-fusion proteinsH295R cells were transfected using Lipofectamine 2000. After incubation for 24 hours, cells were fixed in 4% paraformaldehyde and IHC was performed using an anti-GFP antibody (Invitrogen), a FITC-labeled secondary antibody (Jackson Lab), and a mounting medium with DAPI (VectorLabs). Cells were imaged on an Olympus AX70 microscope.M1 fibroblasts were transfected with 2ug/well of purified expression plasmid using the X-tremeGENE HP DNA transfection reagent (Roche). 24-hours after transfection, cells were fixed in 2% formaldehyde and analyzed using fluorescence microscopy. Fluorescent and bright-field images of randomly selected fields containing transfected M1 cells were saved electronically using a 'blind-code' file nomenclature. The distribution of GFP-fusion proteins in each transfected cell was annotated by an experienced observer (who was unaware of the blind code) as one of the following: (1) nuclear only, (2) nuclear and cytoplasmic, and (3) cytoplasmic only.

Study SubjectsAll participants were patients diagnosed clinically with IMAGE syndrome. This study was approved by the Institutional Review Board (IRB) at the University of California, Los Angeles, the Hospital de Niños Ricardo Gutierrez in Argentina, or the Institute of Child Health at University College London. All participants provided informed consent. Phenotypes are summarized in Table 1.

Study Subjects

All participants were patients diagnosed clinically with IMAGE syndrome. This study was approved by the Institutional Review Board (IRB) at the University of California, Los Angeles, the Hospital de Niños Ricardo Gutierrez in Argentina, or the Institute of Child Health at University College London. All participants provided informed consent. Phenotypes are summarized in Table 1.

Identity-by-descent analysisGenomic DNA from 7 affected individuals (IV-10, V-1, V-2, V-5, V6, V-7, V-12) and 1 non-affected individual (V-13) from a large Argentine family (Family A) were genotyped on the Affymetrix 250K Nsp1 SNP arrays, as per manufacturers’ protocol. Familial relationships were confirmed through checking the sharing statistics in all pairs of samples and ancestral identity-by-descent (IBD) analysis was performed using a custom script (B. Merriman, available on request). The IBD analysis script was designed to search for long continuous intervals compatible with a common extended haplotype among all the affected individuals but not with the unaffected individuals.29 A conservative error rate of 1% was used to allow the algorithm to tolerate possible genotyping errors. A rare dominant model of inheritance was assumed with rare frequency of this haplotype of 0.1% and penetrance of 100% under the assumption that the carriers (parents of the affected individuals) were not showing the phenotype for a different reason (i.e. imprinting) than low penetrance of the disease phenotype.

Identity-by-descent analysis

Genomic DNA from 7 affected individuals (IV-10, V-1, V-2, V-5, V6, V-7, V-12) and 1 non-affected individual (V-13) from a large Argentine family (Family A) were genotyped on the Affymetrix 250K Nsp1 SNP arrays, as per manufacturers’ protocol. Familial relationships were confirmed through checking the sharing statistics in all pairs of samples and ancestral identity-by-descent (IBD) analysis was performed using a custom script (B. Merriman, available on request). The IBD analysis script was designed to search for long continuous intervals compatible with a common extended haplotype among all the affected individuals but not with the unaffected individuals.29 A conservative error rate of 1% was used to allow the algorithm to tolerate possible genotyping errors. A rare dominant model of inheritance was assumed with rare frequency of this haplotype of 0.1% and penetrance of 100% under the assumption that the carriers (parents of the affected individuals) were not showing the phenotype for a different reason (i.e. imprinting) than low penetrance of the disease phenotype.

Capture and Sequencing of Genomic DNAFor the capture of the genes within the IBD interval, we used Agilent Custom 244K comparative genomic hybridization Array. The 60-bp oligonucleotide probes were tiled every 20-bp against all exonic regions on chromosome 11 between 2.45-20.15Mb (hg18, March 2006, build 36.1) and every 30-bp in the flanking 5’ and 3’ regions spanning 0-2.45 Mb and 20.15-22.6 Mb. We included all gene models identified in RefSeq, Genbank, CCDS and UniProt. Location and sequence of all probes are available upon request.Genomic DNA libraries were created for patients V-1 an V-6 from Family A and 3 isolated cases of IMAGE1 (Patient1, 2, and 3) following manufacturer’s protocol (Illumina protocol Preparing Samples for Sequencing Genomic DNA, p/n 11251892 Rev. A) except for the adaptor ligation step where we used custom-made, internally validated bar-coded adaptors. After the PCR amplification, five bar-coded libraries were pooled together and captured by hybridization to custom-designed CGH array, as described in 30. After the capture, the array was washed and the captured DNA was eluted, amplified and diluted to 10nM final concentration based on the Qubit concentration measure and Agilent Bioanalyzer. One flowcell lane of single-end sequencing was performed at the UCLA Genomic Sequencing Center on the Illumina Genome Analyzer II for 76x cycles following manufacturer’s protocol. The base-calling was performed by the real time analysis software (Illumina).

Capture and Sequencing of Genomic DNA

For the capture of the genes within the IBD interval, we used Agilent Custom 244K comparative genomic hybridization Array. The 60-bp oligonucleotide probes were tiled every 20-bp against all exonic regions on chromosome 11 between 2.45-20.15Mb (hg18, March 2006, build 36.1) and every 30-bp in the flanking 5’ and 3’ regions spanning 0-2.45 Mb and 20.15-22.6 Mb. We included all gene models identified in RefSeq, Genbank, CCDS and UniProt. Location and sequence of all probes are available upon request.

Genomic DNA libraries were created for patients V-1 an V-6 from Family A and 3 isolated cases of IMAGE1 (Patient1, 2, and 3) following manufacturer’s protocol (Illumina protocol Preparing Samples for Sequencing Genomic DNA, p/n 11251892 Rev. A) except for the adaptor ligation step where we used custom-made, internally validated bar-coded adaptors. After the PCR amplification, five bar-coded libraries were pooled together and captured by hybridization to custom-designed CGH array, as described in 30. After the capture, the array was washed and the captured DNA was eluted, amplified and diluted to 10nM final concentration based on the Qubit concentration measure and Agilent Bioanalyzer. One flowcell lane of single-end sequencing was performed at the UCLA Genomic Sequencing Center on the Illumina Genome Analyzer II for 76x cycles following manufacturer’s protocol. The base-calling was performed by the real time analysis software (Illumina).

Sequence Data AnalysisThe Illumina output files (.qseq) were converted to fastq formats using BFAST31,32 script ill2fastq.pl and then parsed into multiple fastq files, each for one unique barcode. Only the reads with 100% matching barcode sequences were carried over to the second set of fastq files. The sequence reads in each fastq file were aligned to the human reference genome (hg18, March 2006, build 36.1) using Novoalign from Novocraft Short Read Alignment Package. The output format was set to SAM and default settings were used for all options. Using SAMtools, the SAM file of each sample was converted to a BAM file, sorted and merged and potential PCR duplicates were removed using Picard33. The variants, both single nucleotide variants (SNVs) and small INDELs (insertions and deletions), within the captured coding exonic intervals were called using the SAMtools pileup tool. For SNV calling, the last 5 bases were trimmed and only the reads lacking INDELs were retained. For INDEL calling, only the reads that contained one contiguous INDEL, not occurring on either end of the read, were retained34. The variants were further annotated using the SeqWare project and loaded into the SeqWare QueryEngine database35. Variants from each sample with the following criteria were identified: (i) variant base or INDEL observed at least twice and at ≥5% of the total coverage per base, (ii) variant observed at least once on both forward and reverse strands, (iii) SNV quality score ≥10. As IMAGE syndrome is a rare condition, only variants with coding consequences not present in dbSNP129 were further analyzed.

Sequence Data Analysis

The Illumina output files (.qseq) were converted to fastq formats using BFAST31,32 script ill2fastq.pl and then parsed into multiple fastq files, each for one unique barcode. Only the reads with 100% matching barcode sequences were carried over to the second set of fastq files. The sequence reads in each fastq file were aligned to the human reference genome (hg18, March 2006, build 36.1) using Novoalign from Novocraft Short Read Alignment Package. The output format was set to SAM and default settings were used for all options. Using SAMtools, the SAM file of each sample was converted to a BAM file, sorted and merged and potential PCR duplicates were removed using Picard33. The variants, both single nucleotide variants (SNVs) and small INDELs (insertions and deletions), within the captured coding exonic intervals were called using the SAMtools pileup tool. For SNV calling, the last 5 bases were trimmed and only the reads lacking INDELs were retained. For INDEL calling, only the reads that contained one contiguous INDEL, not occurring on either end of the read, were retained34. The variants were further annotated using the SeqWare project and loaded into the SeqWare QueryEngine database35. Variants from each sample with the following criteria were identified: (i) variant base or INDEL observed at least twice and at ≥5% of the total coverage per base, (ii) variant observed at least once on both forward and reverse strands, (iii) SNV quality score ≥10. As IMAGE syndrome is a rare condition, only variants with coding consequences not present in dbSNP129 were further analyzed.

CDKN1C SequencingAll CDKN1C (RefSeq: NM_000076.2) mutations were PCRed using Phusion HF polymerase (NEB) with 5% DMSO and 0.1M Betaine. PCR products were sent for dideoxysequencing at Laragen, Inc. PCR primers used are listed in Supplementary Table 1. All mutation locations were reported using CCDS7738.1 and P49918 as the normal transcript and protein sequences, respectively.

CDKN1C Sequencing

All CDKN1C (RefSeq: NM_000076.2) mutations were PCRed using Phusion HF polymerase (NEB) with 5% DMSO and 0.1M Betaine. PCR products were sent for dideoxysequencing at Laragen, Inc. PCR primers used are listed in Supplementary Table 1. All mutation locations were reported using CCDS7738.1 and P49918 as the normal transcript and protein sequences, respectively.

Plasmid ConstructsThe cDNA of CDKN1C cloned into pBluescript was purchased from ATCC(#99411)36. Mutations were generated using site-directed mutagenesis (Stratagene, primers in Supplementary Table 1). 4 mutants were created, corresponding to BWS mutations (p.L42P and c.826delT) and IMAGE syndrome mutations (p.F276V and p.K278E). Wild type and mutant versions of CDKN1C were subcloned into pCDNA3.1(+) and pEGFP-C2 for mammalian cell culture experiments, into pFLAG-pcDNA3.1 for the immunoprecipitation, and into pUAST for the generation of transgenic flies.

Plasmid Constructs

The cDNA of CDKN1C cloned into pBluescript was purchased from ATCC(#99411)36. Mutations were generated using site-directed mutagenesis (Stratagene, primers in Supplementary Table 1). 4 mutants were created, corresponding to BWS mutations (p.L42P and c.826delT) and IMAGE syndrome mutations (p.F276V and p.K278E). Wild type and mutant versions of CDKN1C were subcloned into pCDNA3.1(+) and pEGFP-C2 for mammalian cell culture experiments, into pFLAG-pcDNA3.1 for the immunoprecipitation, and into pUAST for the generation of transgenic flies.

Quantitative Reverse Transcription PCRHuman tissue from 7–8 weeks post-conception (wpc) was provided by the Medical Research Council/Wellcome Trust-funded Human Developmental Biology Resource with Research Ethics Committee approval and informed consent. RNA was extracted from adrenal, brain and muscle samples using the TRIzol reagent and first-strand cDNA was generated using SuperScriptR II reverse transcriptase (Invitrogen). Expression of CDKN1C transcript was assessed by quantitative PCR using the StepOnePlus Real-time PCR System, TaqManR Gene Expression Assays for human CDKN1C (Hs00175938_m1) and human GAPDH as endogenous control (4333764T; all Applied Biosystems). Data were analyzed with StepOne software v2.1 according to the 2-ΔΔCT method.

Quantitative Reverse Transcription PCR

Human tissue from 7–8 weeks post-conception (wpc) was provided by the Medical Research Council/Wellcome Trust-funded Human Developmental Biology Resource with Research Ethics Committee approval and informed consent. RNA was extracted from adrenal, brain and muscle samples using the TRIzol reagent and first-strand cDNA was generated using SuperScriptR II reverse transcriptase (Invitrogen). Expression of CDKN1C transcript was assessed by quantitative PCR using the StepOnePlus Real-time PCR System, TaqManR Gene Expression Assays for human CDKN1C (Hs00175938_m1) and human GAPDH as endogenous control (4333764T; all Applied Biosystems). Data were analyzed with StepOne software v2.1 according to the 2-ΔΔCT method.

ImmunofluorescenceFourteen-micron sections of human fetal adrenal tissue (8 weeks post conception) were fixed briefly in 4% PFA and blocked in 1% BSA before incubating overnight with antibody to CDKN1C (Fisher AFMA121866) and CYP11A1 (Sigma HPA016436). Primary antibodies were detected using Alexa647 goat anti-mouse (Invitrogen, A21235) and Alexa555 (Invitrogen, A21429) goat anti-rabbit conjugates. Nuclei were counterstained with DAPI. IMAGEs were collected on a Zeiss 710 confocal microscope (Carl Zeiss).

Immunofluorescence

Fourteen-micron sections of human fetal adrenal tissue (8 weeks post conception) were fixed briefly in 4% PFA and blocked in 1% BSA before incubating overnight with antibody to CDKN1C (Fisher AFMA121866) and CYP11A1 (Sigma HPA016436). Primary antibodies were detected using Alexa647 goat anti-mouse (Invitrogen, A21235) and Alexa555 (Invitrogen, A21429) goat anti-rabbit conjugates. Nuclei were counterstained with DAPI. IMAGEs were collected on a Zeiss 710 confocal microscope (Carl Zeiss).

ImmunoprecipitationHEK293T cells were transfected with constructs encoding Flag-CDKN1C with Lipofectamine 2000 (Invitrogen). For ubiquitination assays, cells were co-transfected with pCI-neo-(HA)3-human Ub37 construct and treated 3 hours in 10μM MG-132 (Millipore) prior to cell lysis. Flag-CDKN1C was immunoprecipitated from cell lysates using the ANTI-FLAG M2-Agarose Affinity Gel (Sigma-Aldrich). Western blot was performed on immunoprecipitated samples and cell lysates with primary antibody to HA (Covance MMS101R), Flag (Abcam, ab1162), PCNA (Abcam ab29), and CDKN1C (Santa Cruz biotechnology, sc-1040). Secondary antibodies used were goat-anti-mouse-HRP (Biorad, 1:20,000) and goat-anti-rabbit-HRP (Santa Cruz, 1:5,000).

Immunoprecipitation

HEK293T cells were transfected with constructs encoding Flag-CDKN1C with Lipofectamine 2000 (Invitrogen). For ubiquitination assays, cells were co-transfected with pCI-neo-(HA)3-human Ub37 construct and treated 3 hours in 10μM MG-132 (Millipore) prior to cell lysis. Flag-CDKN1C was immunoprecipitated from cell lysates using the ANTI-FLAG M2-Agarose Affinity Gel (Sigma-Aldrich). Western blot was performed on immunoprecipitated samples and cell lysates with primary antibody to HA (Covance MMS101R), Flag (Abcam, ab1162), PCNA (Abcam ab29), and CDKN1C (Santa Cruz biotechnology, sc-1040). Secondary antibodies used were goat-anti-mouse-HRP (Biorad, 1:20,000) and goat-anti-rabbit-HRP (Santa Cruz, 1:5,000).

Drosophila experimentsFive independent constructs, CDKN1CWT, CDKN1CL42P, CKDN1C826delT, CDKN1CF276V, and CDKN1CK278E were injected into embryos and each construct generated multiple independent transgenic lines. Overexpression was achieved by using the GAL4-UAS system17 and the following drivers: Ubi-gal4 (ubiquitous expression), ey-gal4 (eye-specific expression), MS1096-gal4 (wing-specific expression), and salPE-gal4 (wing pouch-specific expression). MS1096-gal4 is expressed in the entire wing imaginal disc but at higher levels in the dorsal compartment. This higher level of expression in the dorsal compartment causes enhanced phenotypic effects in the dorsal versus the ventral side of the wing. salPE-gal4 is specifically expressed in the pouch of the wing disc, which only gives rise to the wing proper. All UAS-CDKN1C constructs were larval lethal when expressed with Ubi-gal4, confirming their expression efficiency. At least two independent lines were used for each experiment and all yielded similar results. All eye images were taken on a Leica Z16 AP0 Camera. Wing IMAGEs were taken on Leica DFC 300 FX R2 Camera.

Drosophila experiments

Five independent constructs, CDKN1CWT, CDKN1CL42P, CKDN1C826delT, CDKN1CF276V, and CDKN1CK278E were injected into embryos and each construct generated multiple independent transgenic lines. Overexpression was achieved by using the GAL4-UAS system17 and the following drivers: Ubi-gal4 (ubiquitous expression), ey-gal4 (eye-specific expression), MS1096-gal4 (wing-specific expression), and salPE-gal4 (wing pouch-specific expression). MS1096-gal4 is expressed in the entire wing imaginal disc but at higher levels in the dorsal compartment. This higher level of expression in the dorsal compartment causes enhanced phenotypic effects in the dorsal versus the ventral side of the wing. salPE-gal4 is specifically expressed in the pouch of the wing disc, which only gives rise to the wing proper. All UAS-CDKN1C constructs were larval lethal when expressed with Ubi-gal4, confirming their expression efficiency. At least two independent lines were used for each experiment and all yielded similar results. All eye images were taken on a Leica Z16 AP0 Camera. Wing IMAGEs were taken on Leica DFC 300 FX R2 Camera.

Flow CytometryWild-type or mutant versions of CDKN1C were co-transfected with pCMV-GFP into serum starved HEK293T cells using Lipofectamine 2000 (Invitrogen). After 24 hours, cells were grown in media containing 5% serum for 48-hours. Cells were resuspended in a hypotonic buffer with propidium iodide and GFP-positive events were analyzed on a Becton Dickinson FACScan Analytic Flow Cytometer. All experiments were performed in 2 biological replicates. Statistical significance was assessed using a 2-proportion z-test. Flow cytometry was performed in the UCLA Johansson Comprehensive Cancer Center (JCCC) and Center for AIDS Research Flow Cytometry Core Facility.

Flow Cytometry

Wild-type or mutant versions of CDKN1C were co-transfected with pCMV-GFP into serum starved HEK293T cells using Lipofectamine 2000 (Invitrogen). After 24 hours, cells were grown in media containing 5% serum for 48-hours. Cells were resuspended in a hypotonic buffer with propidium iodide and GFP-positive events were analyzed on a Becton Dickinson FACScan Analytic Flow Cytometer. All experiments were performed in 2 biological replicates. Statistical significance was assessed using a 2-proportion z-test. Flow cytometry was performed in the UCLA Johansson Comprehensive Cancer Center (JCCC) and Center for AIDS Research Flow Cytometry Core Facility.

Analysis of nuclear/cytoplasmic distribution of GFP-fusion proteinsH295R cells were transfected using Lipofectamine 2000. After incubation for 24 hours, cells were fixed in 4% paraformaldehyde and IHC was performed using an anti-GFP antibody (Invitrogen), a FITC-labeled secondary antibody (Jackson Lab), and a mounting medium with DAPI (VectorLabs). Cells were imaged on an Olympus AX70 microscope.M1 fibroblasts were transfected with 2ug/well of purified expression plasmid using the X-tremeGENE HP DNA transfection reagent (Roche). 24-hours after transfection, cells were fixed in 2% formaldehyde and analyzed using fluorescence microscopy. Fluorescent and bright-field images of randomly selected fields containing transfected M1 cells were saved electronically using a 'blind-code' file nomenclature. The distribution of GFP-fusion proteins in each transfected cell was annotated by an experienced observer (who was unaware of the blind code) as one of the following: (1) nuclear only, (2) nuclear and cytoplasmic, and (3) cytoplasmic only.

Analysis of nuclear/cytoplasmic distribution of GFP-fusion proteins

H295R cells were transfected using Lipofectamine 2000. After incubation for 24 hours, cells were fixed in 4% paraformaldehyde and IHC was performed using an anti-GFP antibody (Invitrogen), a FITC-labeled secondary antibody (Jackson Lab), and a mounting medium with DAPI (VectorLabs). Cells were imaged on an Olympus AX70 microscope.

M1 fibroblasts were transfected with 2ug/well of purified expression plasmid using the X-tremeGENE HP DNA transfection reagent (Roche). 24-hours after transfection, cells were fixed in 2% formaldehyde and analyzed using fluorescence microscopy. Fluorescent and bright-field images of randomly selected fields containing transfected M1 cells were saved electronically using a 'blind-code' file nomenclature. The distribution of GFP-fusion proteins in each transfected cell was annotated by an experienced observer (who was unaware of the blind code) as one of the following: (1) nuclear only, (2) nuclear and cytoplasmic, and (3) cytoplasmic only.

Supplementary Material1

Supplementary Material
