---
pmid: '10393948'
title: A maternally methylated CpG island in KvLQT1 is associated with an antisense
  paternal transcript and loss of imprinting in Beckwith-Wiedemann syndrome.
authors:
- Smilinich NJ
- Day CD
- Fitzpatrick GV
- Caldwell GM
- Lossie AC
- Cooper PR
- Smallwood AC
- Joyce JA
- Schofield PN
- Reik W
- Nicholls RD
- Weksberg R
- Driscoll DJ
- Maher ER
- Shows TB
- Higgins MJ
journal: Proc Natl Acad Sci U S A
year: '1999'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC22188
doi: 10.1073/pnas.96.14.8064
---

# A maternally methylated CpG island in KvLQT1 is associated with an antisense paternal transcript and loss of imprinting in Beckwith-Wiedemann syndrome.
**Authors:** Smilinich NJ, Day CD, Fitzpatrick GV, Caldwell GM, Lossie AC, Cooper PR, Smallwood AC, Joyce JA, Schofield PN, Reik W, Nicholls RD, Weksberg R, Driscoll DJ, Maher ER, Shows TB, Higgins MJ
**Journal:** Proc Natl Acad Sci U S A (1999)
**DOI:** [10.1073/pnas.96.14.8064](https://doi.org/10.1073/pnas.96.14.8064)
**PMC:** [PMC22188](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC22188/)

## Abstract

1. Proc Natl Acad Sci U S A. 1999 Jul 6;96(14):8064-9. doi: 
10.1073/pnas.96.14.8064.

A maternally methylated CpG island in KvLQT1 is associated with an antisense 
paternal transcript and loss of imprinting in Beckwith-Wiedemann syndrome.

Smilinich NJ(1), Day CD, Fitzpatrick GV, Caldwell GM, Lossie AC, Cooper PR, 
Smallwood AC, Joyce JA, Schofield PN, Reik W, Nicholls RD, Weksberg R, Driscoll 
DJ, Maher ER, Shows TB, Higgins MJ.

Author information:
(1)Department of Cancer Genetics, Roswell Park Cancer Institute, Elm and Carlton 
Streets, Buffalo, NY 14263, USA.

Loss of imprinting at IGF2, generally through an H19-independent mechanism, is 
associated with a large percentage of patients with the overgrowth and cancer 
predisposition condition Beckwith-Wiedemann syndrome (BWS). Imprinting control 
elements are proposed to exist within the KvLQT1 locus, because multiple 
BWS-associated chromosome rearrangements disrupt this gene. We have identified 
an evolutionarily conserved, maternally methylated CpG island (KvDMR1) in an 
intron of the KvLQT1 gene. Among 12 cases of BWS with normal H19 methylation, 5 
showed demethylation of KvDMR1 in fibroblast or lymphocyte DNA; whereas, in 4 
cases of BWS with H19 hypermethylation, methylation at KvDMRl was normal. Thus, 
inactivation of H19 and hypomethylation at KvDMR1 (or an associated phenomenon) 
represent distinct epigenetic anomalies associated with biallelic expression of 
IGF2. Reverse transcription-PCR analysis of the human and syntenic mouse loci 
identified the presence of a KvDMR1-associated RNA transcribed exclusively from 
the paternal allele and in the opposite orientation with respect to the 
maternally expressed KvLQT1 gene. We propose that KvDMR1 and/or its associated 
antisense RNA (KvLQT1-AS) represents an additional imprinting control element or 
center in the human 11p15.5 and mouse distal 7 imprinted domains.

DOI: 10.1073/pnas.96.14.8064
PMCID: PMC22188
PMID: 10393948 [Indexed for MEDLINE]

## Full Text

Abstract

Loss of imprinting at IGF2 , generally through an H19 -independent mechanism, is associated with a large percentage of patients with the overgrowth and cancer predisposition condition Beckwith–Wiedemann syndrome (BWS). Imprinting control elements are proposed to exist within the KvLQT1 locus, because multiple BWS-associated chromosome rearrangements disrupt this gene. We have identified an evolutionarily conserved, maternally methylated CpG island ( KvDMR1 ) in an intron of the KvLQT1 gene. Among 12 cases of BWS with normal H19 methylation, 5 showed demethylation of KvDMR1 in fibroblast or lymphocyte DNA; whereas, in 4 cases of BWS with H19 hypermethylation, methylation at KvDMRl was normal. Thus, inactivation of H19 and hypomethylation at KvDMR1 (or an associated phenomenon) represent distinct epigenetic anomalies associated with biallelic expression of IGF2 . Reverse transcription–PCR analysis of the human and syntenic mouse loci identified the presence of a KvDMR1 -associated RNA transcribed exclusively from the paternal allele and in the opposite orientation with respect to the maternally expressed KvLQT1 gene. We propose that KvDMR1 and/or its associated antisense RNA ( KvLQT1-AS ) represents an additional imprinting control element or center in the human 11p15.5 and mouse distal 7 imprinted domains.

DISCUSSION

This work describes an imprinted CpG island in an intron of KvLQT1 that is methylated on the active (maternal) allele of this gene and associated with an oppositely oriented RNA transcript expressed from the repressed (paternal) KvLQT1 locus. This situation is reminiscent of the “imprinting box” in region 2 of the mouse Igf2r locus ( 8 ), which has recently been shown to be necessary for the correct imprinted expression of Igf2r transgenes ( 16 ). Moreover, transgenes showing repression of Igf2r after paternal transmission expressed an antisense transcript dependent on this CpG island ( 16 ). Thus, KvLQT1 can be added to the increasing number of endogenous imprinted genes shown to overlap with imprinted antisense transcripts ( 13 – 17 ). Similar to the situation at the Igf2r and UBE3A loci, and unlike that at Igf2 and Zpf127 / ZPF127 ( 13 – 15 ), KvLQT1-AS / Kvlqt1-AS is imprinted in the opposite direction compared with the associated protein-coding gene and therefore conforms to the expression competition model of genomic imprinting ( 19 , 49 ). On the other hand, if the overlapping antisense transcripts associated with the similarly imprinted Igf2 and ZPF127 / Zpf127 loci carry out regulatory functions, these functions may not be related to genomic imprinting and/or are likely to act through a different mechanism. It is interesting to note that the three oppositely imprinted antisense RNAs described to date are associated with maternally expressed imprinted genes (16, 17, and this study).

Based on expression analysis in Dnmt 1−/− mice, Caspary et al. ( 37 ) showed Kvlqt1 to be an indirect target of methylation and predicted the existence of a maternally methylated locus and associated paternally expressed RNA within Kvlqt1 , with the paternal transcript competing with Kvlqt1 for expression. One possibility is that Kvlqt1-AS is an imprintor gene that competes with the target-imprinted gene ( 19 ) Kvlqt1 for expression and is silenced directly by DNA methylation. Down-regulation of Kvlqt1-AS expression during developmental relaxation of Kvlqt1 imprinting ( 37 , 44 – 46 ) would lend support to the notion of a functional role for the antisense RNA transcription in Kvlqt1 imprinting. A second possibility is that KvDMR1 acts as an insulator or boundary element as recently suggested for the core element upstream of the H19 gene ( 50 , 51 ). In this model, KvDMR1 would block the promoter of Kvlqt1 and/or other genes in the vicinity from interacting with enhancers, and antisense RNA transcript levels would not necessarily change during development. Although Kvlqt1 has maternal-specific expression during early embryonic growth in all mice tested, the developmental regulation of Kvlqt1 imprinting varies considerably between strains ( 37 , 44 – 46 ). Whether this variability is related to differences in methylation at KvDMR1 or elsewhere within the gene is unknown; however, consistent with KvDMR1 being a imprinting control element, differential methylation is maintained (at least at the Eag I sites tested; Fig. 3 c ) in adult kidney DNA where biallelic expression of Kvlqt1 is evident in the same tissue from (C57BL/6J × PWK)F 1 offspring (G.V.F. and M.J.H., unpublished work). In this respect, it will also be important to compare the expression of KvLQT1 and KvLQT1-AS in BWS patients with and without methylation at KvDMR1 .

The H19 locus is not a domain-wide imprinting control element, because, unlike Ins2 and Igf2 , the imprinting of p57 Kip2 , Kvlqt1 , and Mash2 is unaffected in H19 deletion mice ( 37 ). This finding supports earlier conclusions resulting from the observation that the majority of informative patients with BWS have LOI at IGF2 ( 32 ) but retain normal methylation and monoallelic expression at H19 ( 30 – 32 ). This lack of reciprocity was also noted for the BWS inv (11) reported by Brown et al. ( 35 ) as well as the inv (11) case described here. These observations suggest the existence of an H19 -independent mechanism for the regulation of IGF2 imprinting and, together with the results of the study of the H19 knockout mouse ( 36 , 37 ), predict the existence of a separately regulated imprinted domain and at least one additional cis-acting imprinting control element or center in 11p15.5 (and mouse distal 7). A subset of DMRs have been termed gametic imprints because of their establishment in the germ line and maintenance throughout development ( 2 ). Transgenic and targeted mutation analyses in the mouse have shown the importance of at least two of these sites in the regulation of genomic imprinting, namely the upstream DMR of H19 and region 2 in Igf2r ( 16 , 50 ). Furthermore, the maternally methylated DMR at exon 1 of the SNRPN gene is included in the smallest microdeletions found in patients with Praeder–Willi syndrome showing imprinting-center defects ( 52 ). Although direct studies of purified oocytes are needed for confirmation, the lack of methylation at KvDMR1 in sperm and its enrichment in ovaries suggest that this locus is also a gametic imprinting mark and could therefore represent a critical control element or imprinting center in 11p15.5 (and distal chromosome 7 in the mouse). The loss of methylation at KvDMR1 in patients with BWS with normal H19 methylation and biallelic expression of IGF2 and the existence of the associated paternally expressed antisense RNA suggest the hypothesis that this locus regulates the imprinted expression of KvLQT1 , IGF2 , and perhaps other imprinted genes in the domain. Functional disruption of KvDMR1 , as evidenced by loss of methylation, may account for a majority (five of nine patients studied here) of non-UPD, nonrearrangement BWS cases without mutations at p57 KIP2 . A limitation of this study is the general unavailability of patient tissues affected by overgrowth. It is formally possible that changes in methylation at KvDMR1 in fibroblasts and lymphocytes from patients with BWS may reflect tissue-specific differences or occasional loss of methylation in these cell types, and may not accurately reflect the methylation status in affected tissues. However, in an analysis of 17 normal individuals (Fig. 3 a and data not shown) including DNA from four lymphoblast and two fibroblast cell lines, three lymphocyte samples, and eight fetal hearts, no departure from the differentially methylated pattern shown in Fig. 3 a has been observed. On the other hand, the proportion of patients with BWS and epigenetic changes at KvDMR1 might even be greater if analysis of the tissues actually affected by overgrowth were available for molecular examination. Analysis of genomic sequence indicates the presence of multiple CpG islands within the 11p15.5 imprinted domain (C.D.D., G.V.F., and M.J.H., unpublished work). The assessment of methylation at these sites in both normal and patient DNA will help to determine whether their methylation patterns show allelic specificity and are as stringently controlled as the methylation pattern for KvDMR1 . Definitive answers to questions regarding the function of KvDMR1 and its associated paternal transcript await the generation of mouse models with specific mutations in this region and further detailed study of the molecular pathology of patients with BWS.
