---
pmid: '19401441'
title: Identification of Apurinic/apyrimidinic endonuclease 1 (APE1) as the endoribonuclease
  that cleaves c-myc mRNA.
authors:
- Barnes T
- Kim WC
- Mantha AK
- Kim SE
- Izumi T
- Mitra S
- Lee CH
journal: Nucleic Acids Res
year: '2009'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC2709568
doi: 10.1093/nar/gkp275
---

# Identification of Apurinic/apyrimidinic endonuclease 1 (APE1) as the endoribonuclease that cleaves c-myc mRNA.
**Authors:** Barnes T, Kim WC, Mantha AK, Kim SE, Izumi T, Mitra S, Lee CH
**Journal:** Nucleic Acids Res (2009)
**DOI:** [10.1093/nar/gkp275](https://doi.org/10.1093/nar/gkp275)
**PMC:** [PMC2709568](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2709568/)

## Abstract

1. Nucleic Acids Res. 2009 Jul;37(12):3946-58. doi: 10.1093/nar/gkp275. Epub 2009
 Apr 28.

Identification of Apurinic/apyrimidinic endonuclease 1 (APE1) as the 
endoribonuclease that cleaves c-myc mRNA.

Barnes T(1), Kim WC, Mantha AK, Kim SE, Izumi T, Mitra S, Lee CH.

Author information:
(1)Chemistry Program, University of Northern British Columbia, 3333 University 
Way, Prince George, British Columbia V2N 4Z9, Canada.

Endonucleolytic cleavage of the coding region determinant (CRD) of c-myc mRNA 
appears to play a critical role in regulating c-myc mRNA turnover. Using 
(32)P-labeled c-myc CRD RNA as substrate, we have purified and identified two 
endoribonucleases from rat liver polysomes that are capable of cleaving the 
transcript in vitro. A 17-kDa enzyme was identified as RNase1. 
Apurinic/apyrimidinic (AP) DNA endonuclease 1 (APE1) was identified as the 
35-kDa endoribonuclease that preferentially cleaves in between UA and CA 
dinucleotides of c-myc CRD RNA. APE1 was further confirmed to be the 35-kDa 
endoribonuclease because: (i) the endoribonuclease activity of the purified 
35-kDa native enzyme was specifically immuno-depleted with APE1 monoclonal 
antibody, and (ii) recombinant human APE1 generated identical RNA cleavage 
patterns as the native liver enzyme. Studies using E96A and H309N mutants of 
APE1 suggest that the endoribonuclease activity for c-myc CRD RNA shares the 
same active center with the AP-DNA endonuclease activity. Transient knockdown of 
APE1 in HeLa cells led to increased steady-state level of c-myc mRNA and its 
half-life. We conclude that the ability to cleave RNA dinucleotides is a 
previously unidentified function of APE1 and it can regulate c-myc mRNA level 
possibly via its endoribonuclease activity.

DOI: 10.1093/nar/gkp275
PMCID: PMC2709568
PMID: 19401441 [Indexed for MEDLINE]

## Full Text

INTRODUCTIONIn higher eukaryotes, including mammals, endonucleolytic cleavage intermediates were shown to be generated during decay of a number of mRNAs (1). For instance, mRNA decay intermediates have been described for the transferrin receptor (2), insulin-like growth factor II (3), α–globin (4), β–globin (5), c-myc (6,7), MDR1 (8), hepatitis B virus (9) and mRNAs containing premature termination codons (10). For most systems, the responsible endoribonuclease has not been identified. This is in part due to the difficulty in identifying this group of enzymes which so far appeared to be substrate-specific and have no sequence homology to known nucleases (1).To fully understand the mechanism and significance of endonucleolytic cleavage in the control of mRNA degradation and abundance, the responsible endoribonuclease must be identified and characterized. To date, endoribonucleases that have been shown to cleave vertebrate mRNA are: Ras GTPase-activating protein-SH3 domain binding protein (G3BP) that cleaves between cytosine and adenine residues at 3′ UTR of mouse c-myc mRNA (11), an estrogen-regulated polysomal endoribonuclease termed PMR1 that cleaves albumin and vitellogenin mRNAs (12), ErEN that cleaves α-globin (4), activator of RNA Decay (ARD-1) (13), RNase L (14), endoplasmic reticulum-associated type 1 transmembrane protein (IRE1) (15) and Argonaute2 (16). Rrp44, a component of eukaryotic exosome (17,18), and SMG6, a protein involved in metazoan nonsense-mediated decay pathway (19,20), have recently demonstrated to possess endonucleolytic activity. Such surprising findings underscore the significance of endonucleolytic cleavage and suggest that this mode of RNA cleavage to control mRNA decay should be re-examined.The proto-oncogene c-Myc has been implicated in the development of virtually all types of human cancers (21). The c-myc mRNA can be degraded via two distinct pathways. One pathway involves deadenylation followed by 3′-5′ exonucleolytic degradation catalyzed by 3′-5′exoribonucleases (22). The c-myc mRNA can also be degraded endonucleolytically, as discovered using a polysome-based in vitro mRNA decay assay (23). An endoribonuclease is believed to target the exposed region of polysome-associated c-myc mRNA referred to as the c-myc coding region determinant or CRD. Indeed, recent evidence suggests that translational pausing at the CRD could result in a ribosome-deficient region that is susceptible to endonucleolytic attack (24). Several studies have confirmed that the coding region of c-myc mRNA, including the CRD, is involved in the regulation of c-myc mRNA stability in cells (25–29). Furthermore, endonucleolytic decay intermediates for c-myc mRNA have been detected in cells (6,7), which provided further support for the significance of endonucleolytic cleavage. However, the endoribonuclease(s) that can cleave CRD of c-myc mRNA remained to be identified. Here, we describe the identification of the apurinic/apyrimidinic (AP) DNA endonuclease 1 (APE1), also referred to as Ref-1, as an endoribonuclease that can cleave c-myc mRNA in vitro. We further show that APE1 could in fact control c-myc mRNA level and half-life in vivo.

INTRODUCTION

In higher eukaryotes, including mammals, endonucleolytic cleavage intermediates were shown to be generated during decay of a number of mRNAs (1). For instance, mRNA decay intermediates have been described for the transferrin receptor (2), insulin-like growth factor II (3), α–globin (4), β–globin (5), c-myc (6,7), MDR1 (8), hepatitis B virus (9) and mRNAs containing premature termination codons (10). For most systems, the responsible endoribonuclease has not been identified. This is in part due to the difficulty in identifying this group of enzymes which so far appeared to be substrate-specific and have no sequence homology to known nucleases (1).

To fully understand the mechanism and significance of endonucleolytic cleavage in the control of mRNA degradation and abundance, the responsible endoribonuclease must be identified and characterized. To date, endoribonucleases that have been shown to cleave vertebrate mRNA are: Ras GTPase-activating protein-SH3 domain binding protein (G3BP) that cleaves between cytosine and adenine residues at 3′ UTR of mouse c-myc mRNA (11), an estrogen-regulated polysomal endoribonuclease termed PMR1 that cleaves albumin and vitellogenin mRNAs (12), ErEN that cleaves α-globin (4), activator of RNA Decay (ARD-1) (13), RNase L (14), endoplasmic reticulum-associated type 1 transmembrane protein (IRE1) (15) and Argonaute2 (16). Rrp44, a component of eukaryotic exosome (17,18), and SMG6, a protein involved in metazoan nonsense-mediated decay pathway (19,20), have recently demonstrated to possess endonucleolytic activity. Such surprising findings underscore the significance of endonucleolytic cleavage and suggest that this mode of RNA cleavage to control mRNA decay should be re-examined.

The proto-oncogene c-Myc has been implicated in the development of virtually all types of human cancers (21). The c-myc mRNA can be degraded via two distinct pathways. One pathway involves deadenylation followed by 3′-5′ exonucleolytic degradation catalyzed by 3′-5′exoribonucleases (22). The c-myc mRNA can also be degraded endonucleolytically, as discovered using a polysome-based in vitro mRNA decay assay (23). An endoribonuclease is believed to target the exposed region of polysome-associated c-myc mRNA referred to as the c-myc coding region determinant or CRD. Indeed, recent evidence suggests that translational pausing at the CRD could result in a ribosome-deficient region that is susceptible to endonucleolytic attack (24). Several studies have confirmed that the coding region of c-myc mRNA, including the CRD, is involved in the regulation of c-myc mRNA stability in cells (25–29). Furthermore, endonucleolytic decay intermediates for c-myc mRNA have been detected in cells (6,7), which provided further support for the significance of endonucleolytic cleavage. However, the endoribonuclease(s) that can cleave CRD of c-myc mRNA remained to be identified. Here, we describe the identification of the apurinic/apyrimidinic (AP) DNA endonuclease 1 (APE1), also referred to as Ref-1, as an endoribonuclease that can cleave c-myc mRNA in vitro. We further show that APE1 could in fact control c-myc mRNA level and half-life in vivo.

MATERIALS AND METHODSPurification of the native rat liver endoribonucleasesThe native endoribonucleases were purified from juvenile frozen rat livers as previously described (30) except for the following changes: (i) Reactive blue-4 chromatographic step was omitted, (ii) dialysis rather than dilution was performed at each step to remove excess KCl, (iii) Superdex 75 Hi Load 16/60 (GE Healthcare, Quebec) gel filtration was used as the last preparative step and (iv) RNasin was omitted in the standard endoribonuclease assay. One unit (U) of enzyme was defined as the quantity of purified enzyme (up to heparin-Sepharose column) required to cleave 90 fmol of 5′-32P-labeled c-myc CRD RNA substrate in 5 min at 37°C under the standard endoribonuclease assay described below. For gel filtration purified enzyme, 1 U of 10–20-kDa or 30–40-kDa enzyme was defined as the amount of the enzyme required to cleave 90 fmol of 5′-32P-labeled c-myc CRD RNA substrate under conditions as described above.Preparation of radiolabeled nucleic acidsTo synthesize human c-myc CRD RNA corresponding to nts 1705–1792, the plasmid pGEM4Z-myc 1705–1792 was linearized and in vitro transcribed as previously described (30,31). The RNA was then 5′-labeled with γ-[32P]-ATP using T4 polynucleotide kinase (30,31). This RNA substrate was used throughout this study unless otherwise indicated. For confirming recombinant APE1 as an endoribonuclease, PCR-amplified DNA template corresponding to c-myc CRD nts 1730–1766 was used to transcribe RNA using T7 RNA polymerase as previously described (30). For internal labeling, α-[32P]-UTP was used during transcription. For 5′end labeling, in-vitro transcribed RNA was 5′-labeled using T4 polynucleotide kinase as described previously (30,31). The oligonucleotide 5′CAAGGTAGTrUATCCTTG-3′ corresponding to c-myc CRD nts 1742–1757 (synthesized by IDT Inc.) was also 5′-labeled and used as substrate in the endoribonuclease assay.In vitro endoribonuclease assay and mapping of RNA cleavage sitesThe standard endoribonuclease assay has been described previously (30,31). The standard 20-μl reaction mixture used for this assay included 2 mM DTT, 1.0 U of RNasin, 2 mM magnesium acetate, 50 mM potassium acetate, 0.1 mM spermidine, 350 fmol of 5′-end-labeled 32P-RNA (∼5 × 104 c.p.m.) and 10 mM Tris–HCl, pH 7.4. Reactions were incubated for 5 min at 37°C unless otherwise indicated, placed in liquid nitrogen, and then at 80–90°C to inactivate the enzyme. Five microliters of loading dye (9 M urea, 0.2% xylene cyanol, 0.2% bromophenol blue) were added to the reaction sample, and 5 μl of the reaction mixtures were subjected to electrophoresis in 8% or 12% polyacrylamide, 7 M urea gel depending on the type of experiments. Gels were then dried and exposed to PhosphorImager screen (Cyclone PhosphorImager). To determine if the native enzyme was N-glycosylated, 100 U of N-glycosidase F (Roche Diagnostics, Germany) was incubated with 3.0 ml of post heparin-Sepharose samples overnight at 30°C before subjecting the samples to gel filtration analysis. The 0.5-ml fractions were collected and analyzed for the presence of endoribonuclease activity as described above. To determine if the native enzyme was composed of multi-subunits linked by disulfide bonds, 3.0 ml of post heparin-Sepharose samples was incubated with 250 mM DTT for 1 h at 4°C before subjecting the samples to gel filtration analysis followed by endoribonuclease assay. For mapping RNA cleavage sites, RNase T1 digestion and alkaline hydrolysis of 5′-radiolabeled RNA were performed as described previously (30) and samples were separated on a 12% polyacrylamide/7 M urea gel.Western blot analysisProtein samples were separated in a 12.5% polyacrylamide/SDS Lammeli gel system, transferred to a nitrocellulose membrane and incubated against APE1 monoclonal antibody (Affinity Bioreagents, Colorado) or RNase A polyclonal antibody (GeneTex Inc, San Antonio). For re-use, some blots were stripped by incubating at 50–55°C with gentle shaking in 63 mM Tris. pH 6.7, 2% SDS, 100 mM β-mercaptoethanol. Full-range rainbow marker (GE Healthcare, Quebec) was used to identify size of bands. Sizes of the marker proteins in kDa are: 250, 160, 105, 75, 50, 35, 30, 25, 15 and 10.Immunodepletion of the native 35-kDa purified endoribonucleasePIERCE Seize X Protein A Immunoprecipitation kit (MJS BioLynx Inc, Ontario) was used to assess whether the endoribonuclease activity from the native 35-kDa purified enzyme was attributed to APE1. Preparation of the spin cup containing Protein A cross-linked to 50 μg of APE1 or syntaxin18 monoclonal antibody was performed according to the manufacturer's instruction. After equilibrating with the binding/wash buffer (8 mM sodium phosphate, 2 mM potassium phosphate, 140 mM NaCl, 10 mM KCl, pH 7.4), 400 μl of gel filtration purified 35-kDa native enzyme were loaded and incubated with gentle rocking for 2 h at 4°C. Spin cups were spun at 3000 r.p.m. for 30 s and the flow through buffer was used for analysis. Four hundred microliters of binding/wash buffer was added to the mix, and the spin cups were spun again. This was repeated a total of three times and washed buffer from each spin kept for analysis. Finally, bound proteins were eluted with 3 × 200 μl elution buffer (primary amine solution, pH 2.8) followed by immediate neutralization with equal volume of Tris–Cl, pH 9.5.Purification of recombinant proteinsThe plasmid pET15b-hAPE1 containing human APE1 cDNA was used to express the recombinant APE1 in BL21(DE3) cells. The His-tagged APE1 was first purified using Ni-NTA column chromatography as described (32). Following removal of the His-tag with thrombin, the recombinant protein was further purified by Superdex HiPrep FPLC (GE Healthcare). Just prior to use, the recombinant protein was dialyzed for 5 h against 10 mM Tris–HCl, pH 7.4, 2 mM DTT, 2 mM magnesium acetate, and 50 mM potassium acetate, with two buffer changes. The recombinant human mutant APE1, H309N and E96A, and l-3-hydroxyacyl-CoA dehydrogenase (HADHSC) were purified in the same manner as described above (33). The recombinant annexin III was purchased from GenWay Biotech (San Diego).Cell culture and siRNA transfectionHuman cervical cancer cell line HeLa (ATCC) was cultured in MEM medium supplemented with 10% fetal bovine serum (Invitrogen) at 37°C in 5% CO2. The day before transfection, ∼1.0 × 105 cells were plated per well in 6-well plates. Transient transfection of 60 nM siRNAs was carried out using Lipofectamine 2000 reagent (Invitrogen) as according to the manufacturer's instructions. The double-stranded Dicer substrate RNAi directed against APE1 mRNA was chemically synthesized (IDT Inc.). The sense and antisense sequences were: r(GUCUGGUACGACUGGAGUACCGG)dCA and r(UGCCGGUACUCCAGUCGUACCAGACCU). As control, the DS Scrambled Negative (IDT Inc.) was used. The sense and antisense sequences of DS Scrambled Negative were: r(CUUCCUCUCUUUCUCUCCCUUGU)dGA and r(UCACAAGGGAGAGAAAGAGAGGAAGGA). Cells from duplicate wells in each experiment were subjected to either total RNA extraction as described below or to cell lysate isolation as previously described (33).Total RNA extraction and quantitative reverse transcription-PCRTotal RNA was extracted from cells using TRIzol reagent (Invitrogen) as according to the manufacturer's instructions. APE1, c-myc and β–actin mRNA levels were examined by quantitative real-time reverse transcription-PCR (qRT-PCR). The first strand cDNA synthesis was performed using QuantiTect RT kit (Qiagen) on 1 μg of total RNA, and the qRT-PCR was performed using iQ SYBR Green Supermix (Bio-Rad) on an iQ5 Multicolor Real-Time PCR Detection System (Bio-Rad). The PCR primers synthesized by IDT Inc. were: APE1 forward primer, 5′-TGG AAT GTG GAT GGG CTT CGA GCC-3′ and APE1 reverse primer, 5′-AAG GAG CTG ACC AGT ATT GAT GA-3′; c-myc forward primer, 5′-ACG AAA CTT TGC CCA TAG CA-3′ and c-myc reverse primer, 5′ GCA AGG AGA GCC TTT CAG AG-3′; β-actin forward primer, 5′-TTG CCG ACA GGA TGC AGA AGG A-3′ and β-actin reverse primer, 5′-AGG TGG ACA GCG AGG CCA GGA T-3′. The cycling protocol consisted of 95°C for 3 min and 40 cycles of denaturation at 95°C for 10 s, annealing at 52°C for 30 s. To confirm amplification specificity, we performed a melting curve analysis at the end of each cycling. Each sample was run in triplicate. The data were analyzed using iQ5 optical system software. Serial dilutions were carried out for each total RNA sample and reverse-transcribed under the above-mentioned conditions for each primer set to ensure amplification with efficiencies near 100%. CT values for target genes (APE1 and c-myc) and reference gene (β-actin) were then used in the comparative CT method or commonly known as the 2−ΔΔCT method (34) to determine the expression level of target gene in APE1-knockdown samples relative to the DS Scrambled Negative-treated sample.Statistical analysisFor statistical analysis, Student's t-test was performed for Figure 8 and linear regression analysis was performed for Figure 9 using Prism 3.0 software (GraphPad, Inc., San Diego, CA, USA).
Figure 1.Purification and identification of proteins that co-purified with endoribonuclease activity. (A) Autoradiograph of an endoribonuclease assay in gel filtration fractions (elution volume 20–76) using 350 fmol 5′ 32P-labeled CRD c-myc RNA as substrate. The endoribonuclease assay was performed for 5 min at 37°C under the standard condition as described in the ‘Materials and Methods’ section. Filled arrow indicates undigested substrate RNA and unfilled arrow indicates endonucleolytic decay products. (B) Silver-stained SDS–PAGE gel of pooled samples from gel filtration in (A). (C) Coomasie blue-stained SDS–PAGE gel of pooled samples from three separate gel filtration columns. Arrows indicate protein bands that were excised and subjected to LC-MS/MS protein identification analysis.
Figure 2.Mapping the cleavage sites generated by the purified 17-kDa and 35-kDa native endoribonucleases. (A) Three hundred and fifty femtomole 5′ -labeled c-myc CRD RNA were subjected to post-heparin-Sepharose purified enzyme (lanes 3 and 9), 30–40-kDa fraction purified enzyme (lanes 4–6 and 12 and 13) and 10–20-kDa fraction purified enzyme (lanes 10 and 11) from gel filtration column under the standard endoribonuclease assay. The radiolabeled RNA was also subjected to alkaline hydrolysis (lanes 7 and 14), bovine pancreatic RNase A (1 U) (lane 8), and RNase T1 digestion (1 U) (lane 1) as described previously (30,31). Samples were run on a 12% polyacrylamide/7 M urea gel. The amount of 30–40-kDa fraction purified enzyme used were 0.75 U (lane 4), 0.6 U (lane 5), 0.4 U (lane 6), 0.2 U (lane 12) and 0.1 U (lane 13). The amount of 10–20-kDa fraction purified enzyme used were 3 U (lane 10) and 1 U (lane 11). Asterisks indicate the 1751UA dinucleotide preferentially cleaved by the 30–40-kDa fraction enzyme. Arrows on the left indicate guanosine cleavage sites generated by RNase T1 and the numbers indicate position of nucleotide sequence. Numbers on the right indicate the cleavage sites generated by the endoribonucleases. (B) Secondary structure of c-myc CRD RNA and the cleavage sites generated by the 35-kDa endoribonuclease.
Figure 3.Presence of RNase1 and APE1 in partially purified native enzyme. Western blot analysis of pooled samples from gel filtration (lanes 3–8) as detected by RNase A (upper panel) or APE1 (lower panel) antibody. Lane 1 contains 5 μg of recombinant bovine pancreatic RNase A.
Figure 4.Immuno-depletion of the 35-kDa endoribonuclease activity using APE1 or syntaxin18 antibody. (A) Partially purified 35-kDa native enzyme was subjected to Seize X Protein A spin column which has been cross-linked with anti-APE1 antibody. Four microliters of washed (lanes 6 and 7) and eluted samples (lanes 8–10) were tested against 350 fmol 5′-labeled c-myc CRD RNA as described in ‘Materials and Methods’ section. Lane 1 contains the RNA only without any treatment with proteins. Lanes 2 and 3 contain 2 U and 3 U, respectively, of partially purified native enzyme from heparin-Sepharose column. Lanes 4 and 5 are 0.75 U and 1 U, respectively, of pre-loaded partially purified 30–40-kDa fraction native enzyme from gel filtration. Filled arrow indicates the intact c-myc CRD RNA and the decay products are shown with a bracket and unfilled arrow. (B) Western blot analysis of samples from (A) as detected using anti-APE1 antibody. Lane 1 contains 0.5 μg recombinant APE1 and lane 2 has the partially purified 30–40-kDa fraction native enzyme. FT is flow-through from the spin column. (C) As in (A), partially purified 35-kDa native enzyme was subjected to spin column which has been cross-linked with anti-syntaxin18 antibody. Lane 1 contains 2 U of partially purified native enzyme from heparin-Sepharose column. Four microliters of flow through (lane 2), washed (lane 3) and eluted samples (lanes 4 and 5) were tested against 5′-labeled c-myc CRD RNA as described in ‘Materials and methods’ section.
Figure 5.Endoribonuclease activity of recombinant APE1 on c-myc CRD RNA. (A) Left panel, Coomasie blue-stained SDS–PAGE gel of recombinant human APE1 at 5 μg (lane 2) and 1 μg (lane 3). Lane 1 is molecular weight marker. Right panel, western blot analysis of 3 μg of recombinant human APE1 generated by us (lane 1) or in Dr. Hickson's laboratory (lane 2), as detected by specific monoclonal antibody against APE1. (B) Left panel, 350 fmol 5′ -labeled c-myc CRD RNA were treated with 0.1 U of the partially purified 35-kDa native enzyme for 5 min (lanes 2 and 3), or with 0.05 μg of purified recombinant human APE1 generated in Dr. Hickson's laboratory (lane 4). Lane 1 had no protein added. Right panel, 350 fmol 5′ -labeled c-myc CRD RNA were treated with 0.1 μg of purified and renatured recombinant human APE1 for 5 min (lane 2), 10 min (lane 3) and 20 min (lane 4), or with 0.1 U of the partially purified 35-kDa native enzyme for 5 min (lane 5), 10 min (lane 6) and 20 min (lane 7) under the standard endoribonuclease assay. Lane 1 had no protein added. Samples were run on 8% polyarylamide/7 M urea gel. Numbers on the right indicate cleavage sites generated by the enzymes.
Figure 6.Endoribonuclease activity of recombinant APE1 polypeptide. (A) Left panel, 350 fmol 5′ -labeled c-myc CRD RNA were treated with the purified and renatured wild-type APE1 (lanes 2 and 9), H309N (lanes 3–5) or E96A (lanes 6–8) mutant APE1 for 5 min at 37°C at the amount indicated under the standard endoribonuclease assay containing 20 μl reaction. Lanes 1 and 10, no protein added. Right panel, 350 fmol 5′-labeled c-myc CRD RNA were treated with the purified and renatured wild-type recombinant APE1 (lane 2) or N-terminus truncated APE1, ND42 (lane 3). Lane 1, no protein added. Samples were run on 8% polyarylamide/7 M urea gel. Numbers on the right indicate cleavage sites generated by the enzymes. (B) 5′-labeled c-myc CRD RNA were treated with purified recombinant human APE1 (lanes 2 and 3), HADHSC (lanes 4 and 5) or annexin3 (lanes 6 and 7) for 5 min at 37°C at the amount indicated under the standard endoribonuclease assay.
Figure 7.APE1 is an endoribonuclease. c-myc CRD RNA corresponding to nts 1730–1766 was either 5′-labeled with 32P-γ-ATP (A) or uniformly labeled with 32P-α-UTP (B) before subjecting to 0.1 μg purified recombinant human APE1 for 2 min (lane 2) or 5 min (lane 3) under the standard endoribonuclease assay. Samples were run on 8% polyarylamide/7 M urea gel. The secondary structure of RNA is on the right of each figure and arrows indicate the cleavage sites generated by APE1. Asterisks on the structure in (B) indicate radiolabeled uridine. Schematic RNA secondary structures on the right represent 5′fragment of cleavage products while those on the left in (B) represent 3′fragment of cleavage products. (C) 5′-labeled oligonucleotide corresponding to nts 1742–1757 of c-myc CRD was treated for 5 min with increasing concentrations (0.08–0.8 μg) of recombinant APE1 (lanes 3–9) or with 2 U partially purified native APE1 from heparin sepharose column (lane 2). Samples were run on 12% polyarylamide/7 M urea gel. The secondary structure of the oligo is on the right panel and the arrow indicates the RNA cleavage site generated by APE1. Schematic secondary structures on the right represent the full-length oligo (top) or 5′fragment of the cleavage product (bottom).
Figure 8.APE1 knockdown increases c-myc mRNA level in HeLa cells. (A) Western blot analysis of APE1 and β-actin protein levels in HeLa cells at 24 and 48 h after transfection with 20 μM Control-dsRNAi or 20 μM APE1-dsRNAi. (B) APE1 and β-actin mRNA levels in HeLa cells transfected with 20 μM APE1-dsRNAi or 20 μM Control-dsRNAi for 24 and 48 h were analyzed by quantitative real-time RT-PCR (top panel). CT values obtained in triplicates from separate transfection experiments were used in the comparative CT method calculation as described in ‘Materials and Methods’ section. Data are mean ± SD of three independent experiments (*, P < 0.05). The APE1/β-actin mRNA ratios are presented as a percentage relative to Control-dsRNAi (top panel). The same RNA samples were analyzed for c-myc mRNA levels and the c-myc/β-actin mRNA ratios are presented as a percentage relative to Control-dsRNAi (bottom panel).
Figure 9.Effect of APE1 knockdown on c-myc mRNA decay in HeLa cells. Twenty-four hours after plating at 1.0 × 105 cells/well, cells were transfected with either 60 nM Control-dsRNAi or 60 nM APE1-dsRNAi. After a further 30 h incubation, cells were subjected to transcriptional inhibition by the addition of 200 µM DRB. Total RNA was then extracted from cells after 20, 40 and 60 min of further incubation. c-myc and β–actin mRNA levels were then quantified as described in Figure 8. After normalizing the levels of c-myc mRNA to β-actin mRNA, c-myc/β-actin mRNA ratios at different time points were expressed as percentage to that at 0 min. Data are the mean ± SD of two independent experiments. Linear regression analysis was performed to compare the two treatment groups and statistically significant difference was found in the slope (P < 0.05).

MATERIALS AND METHODS

Purification of the native rat liver endoribonucleasesThe native endoribonucleases were purified from juvenile frozen rat livers as previously described (30) except for the following changes: (i) Reactive blue-4 chromatographic step was omitted, (ii) dialysis rather than dilution was performed at each step to remove excess KCl, (iii) Superdex 75 Hi Load 16/60 (GE Healthcare, Quebec) gel filtration was used as the last preparative step and (iv) RNasin was omitted in the standard endoribonuclease assay. One unit (U) of enzyme was defined as the quantity of purified enzyme (up to heparin-Sepharose column) required to cleave 90 fmol of 5′-32P-labeled c-myc CRD RNA substrate in 5 min at 37°C under the standard endoribonuclease assay described below. For gel filtration purified enzyme, 1 U of 10–20-kDa or 30–40-kDa enzyme was defined as the amount of the enzyme required to cleave 90 fmol of 5′-32P-labeled c-myc CRD RNA substrate under conditions as described above.

Purification of the native rat liver endoribonucleases

The native endoribonucleases were purified from juvenile frozen rat livers as previously described (30) except for the following changes: (i) Reactive blue-4 chromatographic step was omitted, (ii) dialysis rather than dilution was performed at each step to remove excess KCl, (iii) Superdex 75 Hi Load 16/60 (GE Healthcare, Quebec) gel filtration was used as the last preparative step and (iv) RNasin was omitted in the standard endoribonuclease assay. One unit (U) of enzyme was defined as the quantity of purified enzyme (up to heparin-Sepharose column) required to cleave 90 fmol of 5′-32P-labeled c-myc CRD RNA substrate in 5 min at 37°C under the standard endoribonuclease assay described below. For gel filtration purified enzyme, 1 U of 10–20-kDa or 30–40-kDa enzyme was defined as the amount of the enzyme required to cleave 90 fmol of 5′-32P-labeled c-myc CRD RNA substrate under conditions as described above.

Preparation of radiolabeled nucleic acidsTo synthesize human c-myc CRD RNA corresponding to nts 1705–1792, the plasmid pGEM4Z-myc 1705–1792 was linearized and in vitro transcribed as previously described (30,31). The RNA was then 5′-labeled with γ-[32P]-ATP using T4 polynucleotide kinase (30,31). This RNA substrate was used throughout this study unless otherwise indicated. For confirming recombinant APE1 as an endoribonuclease, PCR-amplified DNA template corresponding to c-myc CRD nts 1730–1766 was used to transcribe RNA using T7 RNA polymerase as previously described (30). For internal labeling, α-[32P]-UTP was used during transcription. For 5′end labeling, in-vitro transcribed RNA was 5′-labeled using T4 polynucleotide kinase as described previously (30,31). The oligonucleotide 5′CAAGGTAGTrUATCCTTG-3′ corresponding to c-myc CRD nts 1742–1757 (synthesized by IDT Inc.) was also 5′-labeled and used as substrate in the endoribonuclease assay.

Preparation of radiolabeled nucleic acids

To synthesize human c-myc CRD RNA corresponding to nts 1705–1792, the plasmid pGEM4Z-myc 1705–1792 was linearized and in vitro transcribed as previously described (30,31). The RNA was then 5′-labeled with γ-[32P]-ATP using T4 polynucleotide kinase (30,31). This RNA substrate was used throughout this study unless otherwise indicated. For confirming recombinant APE1 as an endoribonuclease, PCR-amplified DNA template corresponding to c-myc CRD nts 1730–1766 was used to transcribe RNA using T7 RNA polymerase as previously described (30). For internal labeling, α-[32P]-UTP was used during transcription. For 5′end labeling, in-vitro transcribed RNA was 5′-labeled using T4 polynucleotide kinase as described previously (30,31). The oligonucleotide 5′CAAGGTAGTrUATCCTTG-3′ corresponding to c-myc CRD nts 1742–1757 (synthesized by IDT Inc.) was also 5′-labeled and used as substrate in the endoribonuclease assay.

In vitro endoribonuclease assay and mapping of RNA cleavage sitesThe standard endoribonuclease assay has been described previously (30,31). The standard 20-μl reaction mixture used for this assay included 2 mM DTT, 1.0 U of RNasin, 2 mM magnesium acetate, 50 mM potassium acetate, 0.1 mM spermidine, 350 fmol of 5′-end-labeled 32P-RNA (∼5 × 104 c.p.m.) and 10 mM Tris–HCl, pH 7.4. Reactions were incubated for 5 min at 37°C unless otherwise indicated, placed in liquid nitrogen, and then at 80–90°C to inactivate the enzyme. Five microliters of loading dye (9 M urea, 0.2% xylene cyanol, 0.2% bromophenol blue) were added to the reaction sample, and 5 μl of the reaction mixtures were subjected to electrophoresis in 8% or 12% polyacrylamide, 7 M urea gel depending on the type of experiments. Gels were then dried and exposed to PhosphorImager screen (Cyclone PhosphorImager). To determine if the native enzyme was N-glycosylated, 100 U of N-glycosidase F (Roche Diagnostics, Germany) was incubated with 3.0 ml of post heparin-Sepharose samples overnight at 30°C before subjecting the samples to gel filtration analysis. The 0.5-ml fractions were collected and analyzed for the presence of endoribonuclease activity as described above. To determine if the native enzyme was composed of multi-subunits linked by disulfide bonds, 3.0 ml of post heparin-Sepharose samples was incubated with 250 mM DTT for 1 h at 4°C before subjecting the samples to gel filtration analysis followed by endoribonuclease assay. For mapping RNA cleavage sites, RNase T1 digestion and alkaline hydrolysis of 5′-radiolabeled RNA were performed as described previously (30) and samples were separated on a 12% polyacrylamide/7 M urea gel.

In vitro endoribonuclease assay and mapping of RNA cleavage sites

The standard endoribonuclease assay has been described previously (30,31). The standard 20-μl reaction mixture used for this assay included 2 mM DTT, 1.0 U of RNasin, 2 mM magnesium acetate, 50 mM potassium acetate, 0.1 mM spermidine, 350 fmol of 5′-end-labeled 32P-RNA (∼5 × 104 c.p.m.) and 10 mM Tris–HCl, pH 7.4. Reactions were incubated for 5 min at 37°C unless otherwise indicated, placed in liquid nitrogen, and then at 80–90°C to inactivate the enzyme. Five microliters of loading dye (9 M urea, 0.2% xylene cyanol, 0.2% bromophenol blue) were added to the reaction sample, and 5 μl of the reaction mixtures were subjected to electrophoresis in 8% or 12% polyacrylamide, 7 M urea gel depending on the type of experiments. Gels were then dried and exposed to PhosphorImager screen (Cyclone PhosphorImager). To determine if the native enzyme was N-glycosylated, 100 U of N-glycosidase F (Roche Diagnostics, Germany) was incubated with 3.0 ml of post heparin-Sepharose samples overnight at 30°C before subjecting the samples to gel filtration analysis. The 0.5-ml fractions were collected and analyzed for the presence of endoribonuclease activity as described above. To determine if the native enzyme was composed of multi-subunits linked by disulfide bonds, 3.0 ml of post heparin-Sepharose samples was incubated with 250 mM DTT for 1 h at 4°C before subjecting the samples to gel filtration analysis followed by endoribonuclease assay. For mapping RNA cleavage sites, RNase T1 digestion and alkaline hydrolysis of 5′-radiolabeled RNA were performed as described previously (30) and samples were separated on a 12% polyacrylamide/7 M urea gel.

Western blot analysisProtein samples were separated in a 12.5% polyacrylamide/SDS Lammeli gel system, transferred to a nitrocellulose membrane and incubated against APE1 monoclonal antibody (Affinity Bioreagents, Colorado) or RNase A polyclonal antibody (GeneTex Inc, San Antonio). For re-use, some blots were stripped by incubating at 50–55°C with gentle shaking in 63 mM Tris. pH 6.7, 2% SDS, 100 mM β-mercaptoethanol. Full-range rainbow marker (GE Healthcare, Quebec) was used to identify size of bands. Sizes of the marker proteins in kDa are: 250, 160, 105, 75, 50, 35, 30, 25, 15 and 10.

Western blot analysis

Protein samples were separated in a 12.5% polyacrylamide/SDS Lammeli gel system, transferred to a nitrocellulose membrane and incubated against APE1 monoclonal antibody (Affinity Bioreagents, Colorado) or RNase A polyclonal antibody (GeneTex Inc, San Antonio). For re-use, some blots were stripped by incubating at 50–55°C with gentle shaking in 63 mM Tris. pH 6.7, 2% SDS, 100 mM β-mercaptoethanol. Full-range rainbow marker (GE Healthcare, Quebec) was used to identify size of bands. Sizes of the marker proteins in kDa are: 250, 160, 105, 75, 50, 35, 30, 25, 15 and 10.

Immunodepletion of the native 35-kDa purified endoribonucleasePIERCE Seize X Protein A Immunoprecipitation kit (MJS BioLynx Inc, Ontario) was used to assess whether the endoribonuclease activity from the native 35-kDa purified enzyme was attributed to APE1. Preparation of the spin cup containing Protein A cross-linked to 50 μg of APE1 or syntaxin18 monoclonal antibody was performed according to the manufacturer's instruction. After equilibrating with the binding/wash buffer (8 mM sodium phosphate, 2 mM potassium phosphate, 140 mM NaCl, 10 mM KCl, pH 7.4), 400 μl of gel filtration purified 35-kDa native enzyme were loaded and incubated with gentle rocking for 2 h at 4°C. Spin cups were spun at 3000 r.p.m. for 30 s and the flow through buffer was used for analysis. Four hundred microliters of binding/wash buffer was added to the mix, and the spin cups were spun again. This was repeated a total of three times and washed buffer from each spin kept for analysis. Finally, bound proteins were eluted with 3 × 200 μl elution buffer (primary amine solution, pH 2.8) followed by immediate neutralization with equal volume of Tris–Cl, pH 9.5.

Immunodepletion of the native 35-kDa purified endoribonuclease

PIERCE Seize X Protein A Immunoprecipitation kit (MJS BioLynx Inc, Ontario) was used to assess whether the endoribonuclease activity from the native 35-kDa purified enzyme was attributed to APE1. Preparation of the spin cup containing Protein A cross-linked to 50 μg of APE1 or syntaxin18 monoclonal antibody was performed according to the manufacturer's instruction. After equilibrating with the binding/wash buffer (8 mM sodium phosphate, 2 mM potassium phosphate, 140 mM NaCl, 10 mM KCl, pH 7.4), 400 μl of gel filtration purified 35-kDa native enzyme were loaded and incubated with gentle rocking for 2 h at 4°C. Spin cups were spun at 3000 r.p.m. for 30 s and the flow through buffer was used for analysis. Four hundred microliters of binding/wash buffer was added to the mix, and the spin cups were spun again. This was repeated a total of three times and washed buffer from each spin kept for analysis. Finally, bound proteins were eluted with 3 × 200 μl elution buffer (primary amine solution, pH 2.8) followed by immediate neutralization with equal volume of Tris–Cl, pH 9.5.

Purification of recombinant proteinsThe plasmid pET15b-hAPE1 containing human APE1 cDNA was used to express the recombinant APE1 in BL21(DE3) cells. The His-tagged APE1 was first purified using Ni-NTA column chromatography as described (32). Following removal of the His-tag with thrombin, the recombinant protein was further purified by Superdex HiPrep FPLC (GE Healthcare). Just prior to use, the recombinant protein was dialyzed for 5 h against 10 mM Tris–HCl, pH 7.4, 2 mM DTT, 2 mM magnesium acetate, and 50 mM potassium acetate, with two buffer changes. The recombinant human mutant APE1, H309N and E96A, and l-3-hydroxyacyl-CoA dehydrogenase (HADHSC) were purified in the same manner as described above (33). The recombinant annexin III was purchased from GenWay Biotech (San Diego).

Purification of recombinant proteins

The plasmid pET15b-hAPE1 containing human APE1 cDNA was used to express the recombinant APE1 in BL21(DE3) cells. The His-tagged APE1 was first purified using Ni-NTA column chromatography as described (32). Following removal of the His-tag with thrombin, the recombinant protein was further purified by Superdex HiPrep FPLC (GE Healthcare). Just prior to use, the recombinant protein was dialyzed for 5 h against 10 mM Tris–HCl, pH 7.4, 2 mM DTT, 2 mM magnesium acetate, and 50 mM potassium acetate, with two buffer changes. The recombinant human mutant APE1, H309N and E96A, and l-3-hydroxyacyl-CoA dehydrogenase (HADHSC) were purified in the same manner as described above (33). The recombinant annexin III was purchased from GenWay Biotech (San Diego).

Cell culture and siRNA transfectionHuman cervical cancer cell line HeLa (ATCC) was cultured in MEM medium supplemented with 10% fetal bovine serum (Invitrogen) at 37°C in 5% CO2. The day before transfection, ∼1.0 × 105 cells were plated per well in 6-well plates. Transient transfection of 60 nM siRNAs was carried out using Lipofectamine 2000 reagent (Invitrogen) as according to the manufacturer's instructions. The double-stranded Dicer substrate RNAi directed against APE1 mRNA was chemically synthesized (IDT Inc.). The sense and antisense sequences were: r(GUCUGGUACGACUGGAGUACCGG)dCA and r(UGCCGGUACUCCAGUCGUACCAGACCU). As control, the DS Scrambled Negative (IDT Inc.) was used. The sense and antisense sequences of DS Scrambled Negative were: r(CUUCCUCUCUUUCUCUCCCUUGU)dGA and r(UCACAAGGGAGAGAAAGAGAGGAAGGA). Cells from duplicate wells in each experiment were subjected to either total RNA extraction as described below or to cell lysate isolation as previously described (33).

Cell culture and siRNA transfection

Human cervical cancer cell line HeLa (ATCC) was cultured in MEM medium supplemented with 10% fetal bovine serum (Invitrogen) at 37°C in 5% CO2. The day before transfection, ∼1.0 × 105 cells were plated per well in 6-well plates. Transient transfection of 60 nM siRNAs was carried out using Lipofectamine 2000 reagent (Invitrogen) as according to the manufacturer's instructions. The double-stranded Dicer substrate RNAi directed against APE1 mRNA was chemically synthesized (IDT Inc.). The sense and antisense sequences were: r(GUCUGGUACGACUGGAGUACCGG)dCA and r(UGCCGGUACUCCAGUCGUACCAGACCU). As control, the DS Scrambled Negative (IDT Inc.) was used. The sense and antisense sequences of DS Scrambled Negative were: r(CUUCCUCUCUUUCUCUCCCUUGU)dGA and r(UCACAAGGGAGAGAAAGAGAGGAAGGA). Cells from duplicate wells in each experiment were subjected to either total RNA extraction as described below or to cell lysate isolation as previously described (33).

Total RNA extraction and quantitative reverse transcription-PCRTotal RNA was extracted from cells using TRIzol reagent (Invitrogen) as according to the manufacturer's instructions. APE1, c-myc and β–actin mRNA levels were examined by quantitative real-time reverse transcription-PCR (qRT-PCR). The first strand cDNA synthesis was performed using QuantiTect RT kit (Qiagen) on 1 μg of total RNA, and the qRT-PCR was performed using iQ SYBR Green Supermix (Bio-Rad) on an iQ5 Multicolor Real-Time PCR Detection System (Bio-Rad). The PCR primers synthesized by IDT Inc. were: APE1 forward primer, 5′-TGG AAT GTG GAT GGG CTT CGA GCC-3′ and APE1 reverse primer, 5′-AAG GAG CTG ACC AGT ATT GAT GA-3′; c-myc forward primer, 5′-ACG AAA CTT TGC CCA TAG CA-3′ and c-myc reverse primer, 5′ GCA AGG AGA GCC TTT CAG AG-3′; β-actin forward primer, 5′-TTG CCG ACA GGA TGC AGA AGG A-3′ and β-actin reverse primer, 5′-AGG TGG ACA GCG AGG CCA GGA T-3′. The cycling protocol consisted of 95°C for 3 min and 40 cycles of denaturation at 95°C for 10 s, annealing at 52°C for 30 s. To confirm amplification specificity, we performed a melting curve analysis at the end of each cycling. Each sample was run in triplicate. The data were analyzed using iQ5 optical system software. Serial dilutions were carried out for each total RNA sample and reverse-transcribed under the above-mentioned conditions for each primer set to ensure amplification with efficiencies near 100%. CT values for target genes (APE1 and c-myc) and reference gene (β-actin) were then used in the comparative CT method or commonly known as the 2−ΔΔCT method (34) to determine the expression level of target gene in APE1-knockdown samples relative to the DS Scrambled Negative-treated sample.

Total RNA extraction and quantitative reverse transcription-PCR

Total RNA was extracted from cells using TRIzol reagent (Invitrogen) as according to the manufacturer's instructions. APE1, c-myc and β–actin mRNA levels were examined by quantitative real-time reverse transcription-PCR (qRT-PCR). The first strand cDNA synthesis was performed using QuantiTect RT kit (Qiagen) on 1 μg of total RNA, and the qRT-PCR was performed using iQ SYBR Green Supermix (Bio-Rad) on an iQ5 Multicolor Real-Time PCR Detection System (Bio-Rad). The PCR primers synthesized by IDT Inc. were: APE1 forward primer, 5′-TGG AAT GTG GAT GGG CTT CGA GCC-3′ and APE1 reverse primer, 5′-AAG GAG CTG ACC AGT ATT GAT GA-3′; c-myc forward primer, 5′-ACG AAA CTT TGC CCA TAG CA-3′ and c-myc reverse primer, 5′ GCA AGG AGA GCC TTT CAG AG-3′; β-actin forward primer, 5′-TTG CCG ACA GGA TGC AGA AGG A-3′ and β-actin reverse primer, 5′-AGG TGG ACA GCG AGG CCA GGA T-3′. The cycling protocol consisted of 95°C for 3 min and 40 cycles of denaturation at 95°C for 10 s, annealing at 52°C for 30 s. To confirm amplification specificity, we performed a melting curve analysis at the end of each cycling. Each sample was run in triplicate. The data were analyzed using iQ5 optical system software. Serial dilutions were carried out for each total RNA sample and reverse-transcribed under the above-mentioned conditions for each primer set to ensure amplification with efficiencies near 100%. CT values for target genes (APE1 and c-myc) and reference gene (β-actin) were then used in the comparative CT method or commonly known as the 2−ΔΔCT method (34) to determine the expression level of target gene in APE1-knockdown samples relative to the DS Scrambled Negative-treated sample.

Statistical analysisFor statistical analysis, Student's t-test was performed for Figure 8 and linear regression analysis was performed for Figure 9 using Prism 3.0 software (GraphPad, Inc., San Diego, CA, USA).
Figure 1.Purification and identification of proteins that co-purified with endoribonuclease activity. (A) Autoradiograph of an endoribonuclease assay in gel filtration fractions (elution volume 20–76) using 350 fmol 5′ 32P-labeled CRD c-myc RNA as substrate. The endoribonuclease assay was performed for 5 min at 37°C under the standard condition as described in the ‘Materials and Methods’ section. Filled arrow indicates undigested substrate RNA and unfilled arrow indicates endonucleolytic decay products. (B) Silver-stained SDS–PAGE gel of pooled samples from gel filtration in (A). (C) Coomasie blue-stained SDS–PAGE gel of pooled samples from three separate gel filtration columns. Arrows indicate protein bands that were excised and subjected to LC-MS/MS protein identification analysis.
Figure 2.Mapping the cleavage sites generated by the purified 17-kDa and 35-kDa native endoribonucleases. (A) Three hundred and fifty femtomole 5′ -labeled c-myc CRD RNA were subjected to post-heparin-Sepharose purified enzyme (lanes 3 and 9), 30–40-kDa fraction purified enzyme (lanes 4–6 and 12 and 13) and 10–20-kDa fraction purified enzyme (lanes 10 and 11) from gel filtration column under the standard endoribonuclease assay. The radiolabeled RNA was also subjected to alkaline hydrolysis (lanes 7 and 14), bovine pancreatic RNase A (1 U) (lane 8), and RNase T1 digestion (1 U) (lane 1) as described previously (30,31). Samples were run on a 12% polyacrylamide/7 M urea gel. The amount of 30–40-kDa fraction purified enzyme used were 0.75 U (lane 4), 0.6 U (lane 5), 0.4 U (lane 6), 0.2 U (lane 12) and 0.1 U (lane 13). The amount of 10–20-kDa fraction purified enzyme used were 3 U (lane 10) and 1 U (lane 11). Asterisks indicate the 1751UA dinucleotide preferentially cleaved by the 30–40-kDa fraction enzyme. Arrows on the left indicate guanosine cleavage sites generated by RNase T1 and the numbers indicate position of nucleotide sequence. Numbers on the right indicate the cleavage sites generated by the endoribonucleases. (B) Secondary structure of c-myc CRD RNA and the cleavage sites generated by the 35-kDa endoribonuclease.
Figure 3.Presence of RNase1 and APE1 in partially purified native enzyme. Western blot analysis of pooled samples from gel filtration (lanes 3–8) as detected by RNase A (upper panel) or APE1 (lower panel) antibody. Lane 1 contains 5 μg of recombinant bovine pancreatic RNase A.
Figure 4.Immuno-depletion of the 35-kDa endoribonuclease activity using APE1 or syntaxin18 antibody. (A) Partially purified 35-kDa native enzyme was subjected to Seize X Protein A spin column which has been cross-linked with anti-APE1 antibody. Four microliters of washed (lanes 6 and 7) and eluted samples (lanes 8–10) were tested against 350 fmol 5′-labeled c-myc CRD RNA as described in ‘Materials and Methods’ section. Lane 1 contains the RNA only without any treatment with proteins. Lanes 2 and 3 contain 2 U and 3 U, respectively, of partially purified native enzyme from heparin-Sepharose column. Lanes 4 and 5 are 0.75 U and 1 U, respectively, of pre-loaded partially purified 30–40-kDa fraction native enzyme from gel filtration. Filled arrow indicates the intact c-myc CRD RNA and the decay products are shown with a bracket and unfilled arrow. (B) Western blot analysis of samples from (A) as detected using anti-APE1 antibody. Lane 1 contains 0.5 μg recombinant APE1 and lane 2 has the partially purified 30–40-kDa fraction native enzyme. FT is flow-through from the spin column. (C) As in (A), partially purified 35-kDa native enzyme was subjected to spin column which has been cross-linked with anti-syntaxin18 antibody. Lane 1 contains 2 U of partially purified native enzyme from heparin-Sepharose column. Four microliters of flow through (lane 2), washed (lane 3) and eluted samples (lanes 4 and 5) were tested against 5′-labeled c-myc CRD RNA as described in ‘Materials and methods’ section.
Figure 5.Endoribonuclease activity of recombinant APE1 on c-myc CRD RNA. (A) Left panel, Coomasie blue-stained SDS–PAGE gel of recombinant human APE1 at 5 μg (lane 2) and 1 μg (lane 3). Lane 1 is molecular weight marker. Right panel, western blot analysis of 3 μg of recombinant human APE1 generated by us (lane 1) or in Dr. Hickson's laboratory (lane 2), as detected by specific monoclonal antibody against APE1. (B) Left panel, 350 fmol 5′ -labeled c-myc CRD RNA were treated with 0.1 U of the partially purified 35-kDa native enzyme for 5 min (lanes 2 and 3), or with 0.05 μg of purified recombinant human APE1 generated in Dr. Hickson's laboratory (lane 4). Lane 1 had no protein added. Right panel, 350 fmol 5′ -labeled c-myc CRD RNA were treated with 0.1 μg of purified and renatured recombinant human APE1 for 5 min (lane 2), 10 min (lane 3) and 20 min (lane 4), or with 0.1 U of the partially purified 35-kDa native enzyme for 5 min (lane 5), 10 min (lane 6) and 20 min (lane 7) under the standard endoribonuclease assay. Lane 1 had no protein added. Samples were run on 8% polyarylamide/7 M urea gel. Numbers on the right indicate cleavage sites generated by the enzymes.
Figure 6.Endoribonuclease activity of recombinant APE1 polypeptide. (A) Left panel, 350 fmol 5′ -labeled c-myc CRD RNA were treated with the purified and renatured wild-type APE1 (lanes 2 and 9), H309N (lanes 3–5) or E96A (lanes 6–8) mutant APE1 for 5 min at 37°C at the amount indicated under the standard endoribonuclease assay containing 20 μl reaction. Lanes 1 and 10, no protein added. Right panel, 350 fmol 5′-labeled c-myc CRD RNA were treated with the purified and renatured wild-type recombinant APE1 (lane 2) or N-terminus truncated APE1, ND42 (lane 3). Lane 1, no protein added. Samples were run on 8% polyarylamide/7 M urea gel. Numbers on the right indicate cleavage sites generated by the enzymes. (B) 5′-labeled c-myc CRD RNA were treated with purified recombinant human APE1 (lanes 2 and 3), HADHSC (lanes 4 and 5) or annexin3 (lanes 6 and 7) for 5 min at 37°C at the amount indicated under the standard endoribonuclease assay.
Figure 7.APE1 is an endoribonuclease. c-myc CRD RNA corresponding to nts 1730–1766 was either 5′-labeled with 32P-γ-ATP (A) or uniformly labeled with 32P-α-UTP (B) before subjecting to 0.1 μg purified recombinant human APE1 for 2 min (lane 2) or 5 min (lane 3) under the standard endoribonuclease assay. Samples were run on 8% polyarylamide/7 M urea gel. The secondary structure of RNA is on the right of each figure and arrows indicate the cleavage sites generated by APE1. Asterisks on the structure in (B) indicate radiolabeled uridine. Schematic RNA secondary structures on the right represent 5′fragment of cleavage products while those on the left in (B) represent 3′fragment of cleavage products. (C) 5′-labeled oligonucleotide corresponding to nts 1742–1757 of c-myc CRD was treated for 5 min with increasing concentrations (0.08–0.8 μg) of recombinant APE1 (lanes 3–9) or with 2 U partially purified native APE1 from heparin sepharose column (lane 2). Samples were run on 12% polyarylamide/7 M urea gel. The secondary structure of the oligo is on the right panel and the arrow indicates the RNA cleavage site generated by APE1. Schematic secondary structures on the right represent the full-length oligo (top) or 5′fragment of the cleavage product (bottom).
Figure 8.APE1 knockdown increases c-myc mRNA level in HeLa cells. (A) Western blot analysis of APE1 and β-actin protein levels in HeLa cells at 24 and 48 h after transfection with 20 μM Control-dsRNAi or 20 μM APE1-dsRNAi. (B) APE1 and β-actin mRNA levels in HeLa cells transfected with 20 μM APE1-dsRNAi or 20 μM Control-dsRNAi for 24 and 48 h were analyzed by quantitative real-time RT-PCR (top panel). CT values obtained in triplicates from separate transfection experiments were used in the comparative CT method calculation as described in ‘Materials and Methods’ section. Data are mean ± SD of three independent experiments (*, P < 0.05). The APE1/β-actin mRNA ratios are presented as a percentage relative to Control-dsRNAi (top panel). The same RNA samples were analyzed for c-myc mRNA levels and the c-myc/β-actin mRNA ratios are presented as a percentage relative to Control-dsRNAi (bottom panel).
Figure 9.Effect of APE1 knockdown on c-myc mRNA decay in HeLa cells. Twenty-four hours after plating at 1.0 × 105 cells/well, cells were transfected with either 60 nM Control-dsRNAi or 60 nM APE1-dsRNAi. After a further 30 h incubation, cells were subjected to transcriptional inhibition by the addition of 200 µM DRB. Total RNA was then extracted from cells after 20, 40 and 60 min of further incubation. c-myc and β–actin mRNA levels were then quantified as described in Figure 8. After normalizing the levels of c-myc mRNA to β-actin mRNA, c-myc/β-actin mRNA ratios at different time points were expressed as percentage to that at 0 min. Data are the mean ± SD of two independent experiments. Linear regression analysis was performed to compare the two treatment groups and statistically significant difference was found in the slope (P < 0.05).

Statistical analysis

For statistical analysis, Student's t-test was performed for Figure 8 and linear regression analysis was performed for Figure 9 using Prism 3.0 software (GraphPad, Inc., San Diego, CA, USA).
Figure 1.Purification and identification of proteins that co-purified with endoribonuclease activity. (A) Autoradiograph of an endoribonuclease assay in gel filtration fractions (elution volume 20–76) using 350 fmol 5′ 32P-labeled CRD c-myc RNA as substrate. The endoribonuclease assay was performed for 5 min at 37°C under the standard condition as described in the ‘Materials and Methods’ section. Filled arrow indicates undigested substrate RNA and unfilled arrow indicates endonucleolytic decay products. (B) Silver-stained SDS–PAGE gel of pooled samples from gel filtration in (A). (C) Coomasie blue-stained SDS–PAGE gel of pooled samples from three separate gel filtration columns. Arrows indicate protein bands that were excised and subjected to LC-MS/MS protein identification analysis.
Figure 2.Mapping the cleavage sites generated by the purified 17-kDa and 35-kDa native endoribonucleases. (A) Three hundred and fifty femtomole 5′ -labeled c-myc CRD RNA were subjected to post-heparin-Sepharose purified enzyme (lanes 3 and 9), 30–40-kDa fraction purified enzyme (lanes 4–6 and 12 and 13) and 10–20-kDa fraction purified enzyme (lanes 10 and 11) from gel filtration column under the standard endoribonuclease assay. The radiolabeled RNA was also subjected to alkaline hydrolysis (lanes 7 and 14), bovine pancreatic RNase A (1 U) (lane 8), and RNase T1 digestion (1 U) (lane 1) as described previously (30,31). Samples were run on a 12% polyacrylamide/7 M urea gel. The amount of 30–40-kDa fraction purified enzyme used were 0.75 U (lane 4), 0.6 U (lane 5), 0.4 U (lane 6), 0.2 U (lane 12) and 0.1 U (lane 13). The amount of 10–20-kDa fraction purified enzyme used were 3 U (lane 10) and 1 U (lane 11). Asterisks indicate the 1751UA dinucleotide preferentially cleaved by the 30–40-kDa fraction enzyme. Arrows on the left indicate guanosine cleavage sites generated by RNase T1 and the numbers indicate position of nucleotide sequence. Numbers on the right indicate the cleavage sites generated by the endoribonucleases. (B) Secondary structure of c-myc CRD RNA and the cleavage sites generated by the 35-kDa endoribonuclease.
Figure 3.Presence of RNase1 and APE1 in partially purified native enzyme. Western blot analysis of pooled samples from gel filtration (lanes 3–8) as detected by RNase A (upper panel) or APE1 (lower panel) antibody. Lane 1 contains 5 μg of recombinant bovine pancreatic RNase A.
Figure 4.Immuno-depletion of the 35-kDa endoribonuclease activity using APE1 or syntaxin18 antibody. (A) Partially purified 35-kDa native enzyme was subjected to Seize X Protein A spin column which has been cross-linked with anti-APE1 antibody. Four microliters of washed (lanes 6 and 7) and eluted samples (lanes 8–10) were tested against 350 fmol 5′-labeled c-myc CRD RNA as described in ‘Materials and Methods’ section. Lane 1 contains the RNA only without any treatment with proteins. Lanes 2 and 3 contain 2 U and 3 U, respectively, of partially purified native enzyme from heparin-Sepharose column. Lanes 4 and 5 are 0.75 U and 1 U, respectively, of pre-loaded partially purified 30–40-kDa fraction native enzyme from gel filtration. Filled arrow indicates the intact c-myc CRD RNA and the decay products are shown with a bracket and unfilled arrow. (B) Western blot analysis of samples from (A) as detected using anti-APE1 antibody. Lane 1 contains 0.5 μg recombinant APE1 and lane 2 has the partially purified 30–40-kDa fraction native enzyme. FT is flow-through from the spin column. (C) As in (A), partially purified 35-kDa native enzyme was subjected to spin column which has been cross-linked with anti-syntaxin18 antibody. Lane 1 contains 2 U of partially purified native enzyme from heparin-Sepharose column. Four microliters of flow through (lane 2), washed (lane 3) and eluted samples (lanes 4 and 5) were tested against 5′-labeled c-myc CRD RNA as described in ‘Materials and methods’ section.
Figure 5.Endoribonuclease activity of recombinant APE1 on c-myc CRD RNA. (A) Left panel, Coomasie blue-stained SDS–PAGE gel of recombinant human APE1 at 5 μg (lane 2) and 1 μg (lane 3). Lane 1 is molecular weight marker. Right panel, western blot analysis of 3 μg of recombinant human APE1 generated by us (lane 1) or in Dr. Hickson's laboratory (lane 2), as detected by specific monoclonal antibody against APE1. (B) Left panel, 350 fmol 5′ -labeled c-myc CRD RNA were treated with 0.1 U of the partially purified 35-kDa native enzyme for 5 min (lanes 2 and 3), or with 0.05 μg of purified recombinant human APE1 generated in Dr. Hickson's laboratory (lane 4). Lane 1 had no protein added. Right panel, 350 fmol 5′ -labeled c-myc CRD RNA were treated with 0.1 μg of purified and renatured recombinant human APE1 for 5 min (lane 2), 10 min (lane 3) and 20 min (lane 4), or with 0.1 U of the partially purified 35-kDa native enzyme for 5 min (lane 5), 10 min (lane 6) and 20 min (lane 7) under the standard endoribonuclease assay. Lane 1 had no protein added. Samples were run on 8% polyarylamide/7 M urea gel. Numbers on the right indicate cleavage sites generated by the enzymes.
Figure 6.Endoribonuclease activity of recombinant APE1 polypeptide. (A) Left panel, 350 fmol 5′ -labeled c-myc CRD RNA were treated with the purified and renatured wild-type APE1 (lanes 2 and 9), H309N (lanes 3–5) or E96A (lanes 6–8) mutant APE1 for 5 min at 37°C at the amount indicated under the standard endoribonuclease assay containing 20 μl reaction. Lanes 1 and 10, no protein added. Right panel, 350 fmol 5′-labeled c-myc CRD RNA were treated with the purified and renatured wild-type recombinant APE1 (lane 2) or N-terminus truncated APE1, ND42 (lane 3). Lane 1, no protein added. Samples were run on 8% polyarylamide/7 M urea gel. Numbers on the right indicate cleavage sites generated by the enzymes. (B) 5′-labeled c-myc CRD RNA were treated with purified recombinant human APE1 (lanes 2 and 3), HADHSC (lanes 4 and 5) or annexin3 (lanes 6 and 7) for 5 min at 37°C at the amount indicated under the standard endoribonuclease assay.
Figure 7.APE1 is an endoribonuclease. c-myc CRD RNA corresponding to nts 1730–1766 was either 5′-labeled with 32P-γ-ATP (A) or uniformly labeled with 32P-α-UTP (B) before subjecting to 0.1 μg purified recombinant human APE1 for 2 min (lane 2) or 5 min (lane 3) under the standard endoribonuclease assay. Samples were run on 8% polyarylamide/7 M urea gel. The secondary structure of RNA is on the right of each figure and arrows indicate the cleavage sites generated by APE1. Asterisks on the structure in (B) indicate radiolabeled uridine. Schematic RNA secondary structures on the right represent 5′fragment of cleavage products while those on the left in (B) represent 3′fragment of cleavage products. (C) 5′-labeled oligonucleotide corresponding to nts 1742–1757 of c-myc CRD was treated for 5 min with increasing concentrations (0.08–0.8 μg) of recombinant APE1 (lanes 3–9) or with 2 U partially purified native APE1 from heparin sepharose column (lane 2). Samples were run on 12% polyarylamide/7 M urea gel. The secondary structure of the oligo is on the right panel and the arrow indicates the RNA cleavage site generated by APE1. Schematic secondary structures on the right represent the full-length oligo (top) or 5′fragment of the cleavage product (bottom).
Figure 8.APE1 knockdown increases c-myc mRNA level in HeLa cells. (A) Western blot analysis of APE1 and β-actin protein levels in HeLa cells at 24 and 48 h after transfection with 20 μM Control-dsRNAi or 20 μM APE1-dsRNAi. (B) APE1 and β-actin mRNA levels in HeLa cells transfected with 20 μM APE1-dsRNAi or 20 μM Control-dsRNAi for 24 and 48 h were analyzed by quantitative real-time RT-PCR (top panel). CT values obtained in triplicates from separate transfection experiments were used in the comparative CT method calculation as described in ‘Materials and Methods’ section. Data are mean ± SD of three independent experiments (*, P < 0.05). The APE1/β-actin mRNA ratios are presented as a percentage relative to Control-dsRNAi (top panel). The same RNA samples were analyzed for c-myc mRNA levels and the c-myc/β-actin mRNA ratios are presented as a percentage relative to Control-dsRNAi (bottom panel).
Figure 9.Effect of APE1 knockdown on c-myc mRNA decay in HeLa cells. Twenty-four hours after plating at 1.0 × 105 cells/well, cells were transfected with either 60 nM Control-dsRNAi or 60 nM APE1-dsRNAi. After a further 30 h incubation, cells were subjected to transcriptional inhibition by the addition of 200 µM DRB. Total RNA was then extracted from cells after 20, 40 and 60 min of further incubation. c-myc and β–actin mRNA levels were then quantified as described in Figure 8. After normalizing the levels of c-myc mRNA to β-actin mRNA, c-myc/β-actin mRNA ratios at different time points were expressed as percentage to that at 0 min. Data are the mean ± SD of two independent experiments. Linear regression analysis was performed to compare the two treatment groups and statistically significant difference was found in the slope (P < 0.05).

Purification and identification of proteins that co-purified with endoribonuclease activity. (A) Autoradiograph of an endoribonuclease assay in gel filtration fractions (elution volume 20–76) using 350 fmol 5′ 32P-labeled CRD c-myc RNA as substrate. The endoribonuclease assay was performed for 5 min at 37°C under the standard condition as described in the ‘Materials and Methods’ section. Filled arrow indicates undigested substrate RNA and unfilled arrow indicates endonucleolytic decay products. (B) Silver-stained SDS–PAGE gel of pooled samples from gel filtration in (A). (C) Coomasie blue-stained SDS–PAGE gel of pooled samples from three separate gel filtration columns. Arrows indicate protein bands that were excised and subjected to LC-MS/MS protein identification analysis.

Mapping the cleavage sites generated by the purified 17-kDa and 35-kDa native endoribonucleases. (A) Three hundred and fifty femtomole 5′ -labeled c-myc CRD RNA were subjected to post-heparin-Sepharose purified enzyme (lanes 3 and 9), 30–40-kDa fraction purified enzyme (lanes 4–6 and 12 and 13) and 10–20-kDa fraction purified enzyme (lanes 10 and 11) from gel filtration column under the standard endoribonuclease assay. The radiolabeled RNA was also subjected to alkaline hydrolysis (lanes 7 and 14), bovine pancreatic RNase A (1 U) (lane 8), and RNase T1 digestion (1 U) (lane 1) as described previously (30,31). Samples were run on a 12% polyacrylamide/7 M urea gel. The amount of 30–40-kDa fraction purified enzyme used were 0.75 U (lane 4), 0.6 U (lane 5), 0.4 U (lane 6), 0.2 U (lane 12) and 0.1 U (lane 13). The amount of 10–20-kDa fraction purified enzyme used were 3 U (lane 10) and 1 U (lane 11). Asterisks indicate the 1751UA dinucleotide preferentially cleaved by the 30–40-kDa fraction enzyme. Arrows on the left indicate guanosine cleavage sites generated by RNase T1 and the numbers indicate position of nucleotide sequence. Numbers on the right indicate the cleavage sites generated by the endoribonucleases. (B) Secondary structure of c-myc CRD RNA and the cleavage sites generated by the 35-kDa endoribonuclease.

Presence of RNase1 and APE1 in partially purified native enzyme. Western blot analysis of pooled samples from gel filtration (lanes 3–8) as detected by RNase A (upper panel) or APE1 (lower panel) antibody. Lane 1 contains 5 μg of recombinant bovine pancreatic RNase A.

Immuno-depletion of the 35-kDa endoribonuclease activity using APE1 or syntaxin18 antibody. (A) Partially purified 35-kDa native enzyme was subjected to Seize X Protein A spin column which has been cross-linked with anti-APE1 antibody. Four microliters of washed (lanes 6 and 7) and eluted samples (lanes 8–10) were tested against 350 fmol 5′-labeled c-myc CRD RNA as described in ‘Materials and Methods’ section. Lane 1 contains the RNA only without any treatment with proteins. Lanes 2 and 3 contain 2 U and 3 U, respectively, of partially purified native enzyme from heparin-Sepharose column. Lanes 4 and 5 are 0.75 U and 1 U, respectively, of pre-loaded partially purified 30–40-kDa fraction native enzyme from gel filtration. Filled arrow indicates the intact c-myc CRD RNA and the decay products are shown with a bracket and unfilled arrow. (B) Western blot analysis of samples from (A) as detected using anti-APE1 antibody. Lane 1 contains 0.5 μg recombinant APE1 and lane 2 has the partially purified 30–40-kDa fraction native enzyme. FT is flow-through from the spin column. (C) As in (A), partially purified 35-kDa native enzyme was subjected to spin column which has been cross-linked with anti-syntaxin18 antibody. Lane 1 contains 2 U of partially purified native enzyme from heparin-Sepharose column. Four microliters of flow through (lane 2), washed (lane 3) and eluted samples (lanes 4 and 5) were tested against 5′-labeled c-myc CRD RNA as described in ‘Materials and methods’ section.

Endoribonuclease activity of recombinant APE1 on c-myc CRD RNA. (A) Left panel, Coomasie blue-stained SDS–PAGE gel of recombinant human APE1 at 5 μg (lane 2) and 1 μg (lane 3). Lane 1 is molecular weight marker. Right panel, western blot analysis of 3 μg of recombinant human APE1 generated by us (lane 1) or in Dr. Hickson's laboratory (lane 2), as detected by specific monoclonal antibody against APE1. (B) Left panel, 350 fmol 5′ -labeled c-myc CRD RNA were treated with 0.1 U of the partially purified 35-kDa native enzyme for 5 min (lanes 2 and 3), or with 0.05 μg of purified recombinant human APE1 generated in Dr. Hickson's laboratory (lane 4). Lane 1 had no protein added. Right panel, 350 fmol 5′ -labeled c-myc CRD RNA were treated with 0.1 μg of purified and renatured recombinant human APE1 for 5 min (lane 2), 10 min (lane 3) and 20 min (lane 4), or with 0.1 U of the partially purified 35-kDa native enzyme for 5 min (lane 5), 10 min (lane 6) and 20 min (lane 7) under the standard endoribonuclease assay. Lane 1 had no protein added. Samples were run on 8% polyarylamide/7 M urea gel. Numbers on the right indicate cleavage sites generated by the enzymes.

Endoribonuclease activity of recombinant APE1 polypeptide. (A) Left panel, 350 fmol 5′ -labeled c-myc CRD RNA were treated with the purified and renatured wild-type APE1 (lanes 2 and 9), H309N (lanes 3–5) or E96A (lanes 6–8) mutant APE1 for 5 min at 37°C at the amount indicated under the standard endoribonuclease assay containing 20 μl reaction. Lanes 1 and 10, no protein added. Right panel, 350 fmol 5′-labeled c-myc CRD RNA were treated with the purified and renatured wild-type recombinant APE1 (lane 2) or N-terminus truncated APE1, ND42 (lane 3). Lane 1, no protein added. Samples were run on 8% polyarylamide/7 M urea gel. Numbers on the right indicate cleavage sites generated by the enzymes. (B) 5′-labeled c-myc CRD RNA were treated with purified recombinant human APE1 (lanes 2 and 3), HADHSC (lanes 4 and 5) or annexin3 (lanes 6 and 7) for 5 min at 37°C at the amount indicated under the standard endoribonuclease assay.

APE1 is an endoribonuclease. c-myc CRD RNA corresponding to nts 1730–1766 was either 5′-labeled with 32P-γ-ATP (A) or uniformly labeled with 32P-α-UTP (B) before subjecting to 0.1 μg purified recombinant human APE1 for 2 min (lane 2) or 5 min (lane 3) under the standard endoribonuclease assay. Samples were run on 8% polyarylamide/7 M urea gel. The secondary structure of RNA is on the right of each figure and arrows indicate the cleavage sites generated by APE1. Asterisks on the structure in (B) indicate radiolabeled uridine. Schematic RNA secondary structures on the right represent 5′fragment of cleavage products while those on the left in (B) represent 3′fragment of cleavage products. (C) 5′-labeled oligonucleotide corresponding to nts 1742–1757 of c-myc CRD was treated for 5 min with increasing concentrations (0.08–0.8 μg) of recombinant APE1 (lanes 3–9) or with 2 U partially purified native APE1 from heparin sepharose column (lane 2). Samples were run on 12% polyarylamide/7 M urea gel. The secondary structure of the oligo is on the right panel and the arrow indicates the RNA cleavage site generated by APE1. Schematic secondary structures on the right represent the full-length oligo (top) or 5′fragment of the cleavage product (bottom).

APE1 knockdown increases c-myc mRNA level in HeLa cells. (A) Western blot analysis of APE1 and β-actin protein levels in HeLa cells at 24 and 48 h after transfection with 20 μM Control-dsRNAi or 20 μM APE1-dsRNAi. (B) APE1 and β-actin mRNA levels in HeLa cells transfected with 20 μM APE1-dsRNAi or 20 μM Control-dsRNAi for 24 and 48 h were analyzed by quantitative real-time RT-PCR (top panel). CT values obtained in triplicates from separate transfection experiments were used in the comparative CT method calculation as described in ‘Materials and Methods’ section. Data are mean ± SD of three independent experiments (*, P < 0.05). The APE1/β-actin mRNA ratios are presented as a percentage relative to Control-dsRNAi (top panel). The same RNA samples were analyzed for c-myc mRNA levels and the c-myc/β-actin mRNA ratios are presented as a percentage relative to Control-dsRNAi (bottom panel).

Effect of APE1 knockdown on c-myc mRNA decay in HeLa cells. Twenty-four hours after plating at 1.0 × 105 cells/well, cells were transfected with either 60 nM Control-dsRNAi or 60 nM APE1-dsRNAi. After a further 30 h incubation, cells were subjected to transcriptional inhibition by the addition of 200 µM DRB. Total RNA was then extracted from cells after 20, 40 and 60 min of further incubation. c-myc and β–actin mRNA levels were then quantified as described in Figure 8. After normalizing the levels of c-myc mRNA to β-actin mRNA, c-myc/β-actin mRNA ratios at different time points were expressed as percentage to that at 0 min. Data are the mean ± SD of two independent experiments. Linear regression analysis was performed to compare the two treatment groups and statistically significant difference was found in the slope (P < 0.05).

RESULTSPurification and identification of the 35-kDa liver endoribonucleaseWe previously reported partial purification of an endoribonuclease from rat liver for cleaving the c-myc CRD RNA (30). Five major proteins of ∼10–35-kDa size were co-purified with this activity. To determine the identity of the responsible enzyme, we first repeated earlier enzyme purification steps with slight modifications as described in the ‘Materials and Methods’ section. Two endoribonuclease activities from the final gel filtration column at elution volumes 46–50 ml and 62–68 ml, corresponding to 30–40-kDa and 10–20-kDa sizes, were prominent (Figure 1A). Pooled protein fractions visualized by silver-staining in SDS–PAGE gel (Figure 1B) shows a distinct protein band of ∼35 kDa at elution volume 40–50 ml and four protein bands from 10–20 kDa at elution volume 60–66 ml. These fractions were separately pooled from three separate preparations and visualized by Coomasie blue-staining after SDS–PAGE (Figure 1C). Gel slices at around protein bands 1–6 (Figure 1C) were excised for LC/MS mass spectrometry protein identification analysis at the Genome BC Proteomics Centre, University of Victoria. Based on molecular weight of proteins, number of matched peptides, and percentage of amino acid sequence against the ‘rodentia’ protein database, the top three proteins that matched for each band were selected (Table 1). The three major proteins, L-3-hydroxyacyl-CoA dehydrogenase (HADHSC), annexin III and AP endonuclease (APE1), were identified as possible candidates for the 35-kDa endoribonuclease. The band in between band 2 and 3 in Figure 1C was determined to be HADHSC by LC/MS spectrometry (data not shown). This band may in fact represent a truncated or non-post-translationally modified form of HADHSC. In addition to the criteria described above, HADHSC was chosen because it contains a predicted RNA-binding Rossman fold motif (35) and annexin III was chosen because a related protein, annexin II, has been shown to bind human c-myc RNA (36). Both HADHSC and annexin III were present in the gel filtration fractions as determined by western analysis (data not shown). However, neither purified recombinant HADHSC nor annexin III exhibited any endoribonuclease activity (see later in Figure 6B). APE1 then became the prime candidate because it is a known multifunctional protein with DNA-specific endonuclease and RNase H-like activities (37–39). To further confirm its identity, we performed duplicate LC/MS mass spectrometry on individual 35–38-kDa protein in the gel slices isolated after heparin-Sepharose chromatography. Both HADHSC and APE1 were again two of the matched proteins (data not shown), suggesting that both proteins co-purified with the 35-kDa endoribonuclease activity.
Table 1.Summary of LC/MS mass spectrometry data and peptide analysis to identify purified proteins following gel filtrationProtein band #Top three protein matches from Rodentia speciesAmino sequence coverage (%)Number of matched peptides11) Apurinic/apyrimidinic endonuclease lyase (AP endonuclease/APE1) (35.8 kDa)1) 32%1) 72) Annexin III (36.5 kDa)2) 38%2) 103) Aldo-keto reductase E1 (34.8 kDa)3) 40%3) 112L-3-hydroxyacyl-CoA dehydrogenase (HADHSC) (34 kDa)55%831) Peroxisomal enoyl hydratase-like protein (36.5 kDa)1) 46%1) 122) HADHSC (34 kDa)2) 20%2) 33) Glutathione S-transferase (25.6 kDa)3) 54%3) 74Cyclophilin B (23 kDa)56%145Pancreatic ribonuclease A (RNase1) (17 kDa)45%56Cytochrome C (12.5 kDa)55%11The 17-kDa endoribonuclease belongs to the RNase A superfamilyBased on the above identification criteria, protein bands 4, 5 and 6 (Figure 1C) were convincingly identified as cyclophilin B, pancreatic rat ribonuclease A family member RNase1, and cytochrome c respectively (Table 1). We reasoned that the 17-kDa RNase 1 is most likely responsible for the 10–20-kDa endoribonuclease activity based on the following observations: (i) RNase 1 is the only known protein with ribonuclease activity identified from 10–20-kDa protein bands 4, 5 and 6, (ii) RNA sequence cleavage specificity of the partially purified 10–20-kDa native enzyme from gel filtration resembled, yet was not identical, to that exhibited by the bovine pancreatic RNase A (compare lanes 10 and 11 to lane 8 in Figure 2A), and (iii) polyclonal antibody against RNase A detected a protein band at around 17-kDa range in elution volume 60–80 ml from gel filtration column (lanes 7 and 8, Figure 3) which correlated with the endoribonuclease activity (Figure 1A), and finally the presence of 17-kDa protein band on silver-stained SDS–PAGE gel (Figure 1B).The 35-kDa endoribonuclease activity is not attributable to a modified form of pancreatic RNase AThe cleavage sites on c-myc CRD RNA generated by the purified native enzyme in the heparin-Sepharose and gel filtration fractions with proteins of 10–20-kDa and 30–40-kDa size range were mapped as previously described using RNase T1 digest and alkaline hydrolysis (30,31). The 30–40-kDa fraction appeared to exhibit similarity with the 10–20-kDa fractions in sequence cleavage specificity (compare lanes 4, 5, 6, 12 and 13 with lanes 10 and 11, Figure 2A). However, there were some differences in that the 30–40-kDa fractions had stronger preference for the 1751UA dinucleotide (shown by asterisks) (Figure 2A). The 30–40-kDa fractions also had some preference for the 1771CA and 1773UA dinucleotides (Figure 2A).Given the similarity in RNA cleavage patterns between the 30–40-kDa and the 10–20-kDa fractions, together with the observations that the polyclonal antibody against RNase A detected a faint band corresponding to 30-kDa (lane 6 in Figure 3) and 35-kDa (data not shown) polypeptides on western blots, we considered whether the 30–40-kDa endoribonuclease activity is contributed by a modified form of pancreatic RNase A. This is plausible because the dimeric RNase A is known to exist (40), and a monomeric N-glycosylated RNase A of approximately 24–36-kDa size has been reported (41). To test the possibility that the 35-kDa endoribonuclease is a disulfide-linked dimeric RNase A, we treated the purified post-heparin Sepharose fraction with 250 mM DTT before gel filtration analysis. Endoribonuclease analysis of the fractions showed two distinct activities (data not shown) similar to the profile of the original sample (Figure 1A). We therefore concluded that the 35-kDa endoribonuclease is unlikely to be a disulfide-linked dimeric form of 17-kDa RNase1, and is likely a monomeric protein. To test the possibility that the 35-kDa endoribonuclease is a N-glycosylated RNase A, we treated the heparin Sepharose fraction with 100 U endoglycosidase F before gel filtration chromatography. Our results (data not shown) show that the elution profile of endoribonuclease activity remained unchanged compared to the original sample. Thus, the 35-kDa endoribonuclease is unlikely to be N-glycosylated RNase A.Immuno-depletion of the 35-kDa endoribonuclease with APE1 monoclonal antibodyOur results so far suggested that the 35-kDa endoribonuclease was APE1 which we then tested by western analysis of the 30–40-kDa fractions from gel filtration. Figure 3 shows that APE1 is present only in the elution volume 45–55 ml (lane 6, Figure 3, bottom panel) which correlates with the 35-kDa endoribonuclease activity (Figure 1A). We performed immunodepletion studies of the purified native 35-kDa endoribonuclease with the APE1 antibody. Endoribonuclease analysis of the fractions eluted from an APE1 monoclonal antibody spin column is shown in Figure 4A. Washed fractions (lanes 6 and 7) had no endoribonuclease activity while the earlier elution fractions (lanes 8 and 9) contained endoribonuclease activity. Endoribonuclease activity in Elution 1 (lane 8) was as high as those from partially purified post-heparin-Sepharose sample (lanes 2 and 3). In both cases, excess nuclease activity led to almost complete degradation of RNA substrate. Figure 4B shows western blot analysis of samples taken from the same immunoprecipitation experiment described in Figure 4A. There was striking correlation between the endoribonuclease activity in the pre-loaded purified native 30–40-kDa gel filtration fraction, Elutions 1 and 2, and the presence of APE1 in those fractions (compare lanes 2, 6 and 7 in Figure 4B with lanes 4, 5, 8 and 9 in Figure 4A). In addition, there was correlation between the absence of endoribonuclease activity in the wash fractions, and the absence of APE1 in those fractions (compare lanes 4 and 5 in Figure 4B with lanes 6 and 7 in Figure 4A). To confirm the specificity of the immunodepletion experiment using APE1 antibody, we performed similar experiments using another antibody cross-linked to the spin column, which in this case was syntaxin18 antibody. Figure 4C shows that endoribonuclease analysis of the fractions eluted from syntaxin18 antibody spin column. The flow through (lane 2) and washed fractions (lane 3) showed strong nuclease activity whereas the Elution 1 and 2 (lanes 4 and 5) had no endoribonuclease activity. This control experiment confirmed that the immunodepletion of endoribonuclease activity observed in Figure 4A was produced specifically by the presence of anti-APE1 antibody.Recombinant human APE1 and purified native 35-kDa endoribonuclease exhibited similar RNA cleavage patternWe then directly tested whether the recombinant human APE1 possesses endoribonuclease activity for c-myc CRD RNA. The purified proteins were almost homogenous with more than 95% pure based on Coomasie blue-staining after SDS–PAGE (Figure 5A, left panel) and identity checked by western analysis (Figure 5A, right panel). The minor, lower-molecular-weight band in the immunoblot assay (Figure 5A, right panel) and the SDS–PAGE gel (Figure 5A, lane 2 in left panel) was likely to be a N-terminal cleavage product of APE1 (42). This is commonly observed as the protein is quite susceptible to specific hydrolysis upon boiling. Endoribonuclease analysis confirmed that recombinant APE1 from two separate sources (one was prepared by us and the other from Dr. Ian Hickson's lab, University of Oxford) exhibited endoribonuclease activity against c-myc CRD RNA with a distinct preference for the 1751UA dinucleotide (Figure 5B, left panel). Furthermore, when the recombinant APE1 was reduced, denatured with guanidine hydrochloride and renatured, the renatured recombinant APE1 exhibits the same cleavage specificity (lanes 2–4, right panel in Figure 5B) as the native enzyme (lanes 5–7, right panel in Figure 5B) with 1751UA still being the dominant cleavage site.Specificity and identification of active residues in APE1 for endoribonuclease activitySpecific residues Glu96 and His309 have been identified as critical for the AP DNA endonuclease (37,43), exonuclease (44) and RNase H activities of APE1 (39). To examine whether the c-myc CRD RNA-cleaving activity of APE1 requires the same amino acid residues as these nuclease activities of APE1, E96A and H309N APE1 mutant polypeptides were purified under identical purification methods as the wild-type APE1 and tested for endoribonuclease activity under our standard endoribonuclease assay. Figure 6A shows the lack of RNA cleaving activity of the E96A mutant at up to 0.5 μg of the protein (lanes 6–8, Figure 6A). H309N mutant also did not exhibit any endoribonuclease activity at 0.1 and 0.3 μg (lanes 3 and 5, Figure 6A). Interestingly, at a higher concentration (0.5 μg), the H309N mutant cleaved CRD RNA at 1727CA, 1768CA, 1771CA, 1773UA and 1775CA but not at 1757UA, 1751UA, 1747UA and 1742CA (compare lane 4 to lanes 2 and 9, Figure 6A). These results suggest that the endoribonuclease activity of APE1 against c-myc CRD RNA shares, to some extent, common critical amino acid residues with AP DNA endonuclease, RNase H and exonuclease activities. ND42 mutant APE1 missing the first 42 amino acids at the N-terminus demonstrated activity which is similar to the wild-type APE1 (Figure 6A, right panel). This suggests that the N-terminus domain, at least the first 42 amino acids, does not participate in the RNA-cleaving of APE1.To further confirm specificity of the endoribonuclease activity exhibited by the wild-type and APE1 mutants, we also tested other recombinant proteins. Figure 6B shows that the lack of endoribonuclease activity exhibited by recombinant HADHSC (lanes 4 and 5) and annexin III (lanes 6 and 7) while the wild-type APE1 (lanes 2 and 3) exhibited the typical endoribonuclease activity. Overall, the results in Figure 6 confirms the specificity of the endoribonuclease activity exhibited by the purified wild-type APE1 and absence of RNase A-like contaminants amongst other purified recombinant proteins including the E96A and H309N APE1 mutants.APE1 is an endoribonucleaseTo confirm that APE1 is indeed an endoribonuclease, we challenged the enzyme with two new substrates. First, we tested APE1 on 32P-5′-labeled or 32P-UTP internally labeled RNA corresponding to nts 1730–1766 of c-myc CRD RNA. Figure 7A shows the 5′ fragment of the cleavage products generated by APE1 on the 5′-labeled RNA. Figure 7B shows the cleavage products generated when the RNA was internally labeled with 32P-UTP. The corresponding 5′ and 3′ cleavage fragments were clearly visible indicating the endonucleolytic cleavage of the RNA substrate by APE1. To further confirm this, we tested APE1 on a 5′-labeled DNA oligo corresponding to nts 1742–1757 of c-myc CRD DNA with an incorporation of an uridine at position 1751. As expected, being an endoribonuclease APE1 was only capable of cleaving at position 1751 to generate the 5′-fragment of the oligo (Figure 7C). A recent study has shown that APE1 can cleave AP-site-containing single-stranded RNA (45,46), presumably at the phosphodiester bond immediately 5′ to the AP-site. Therefore, it is expected that removal of any bases, including uracil, in single-stranded region of RNA will be susceptible to the abasic RNA-endonuclease activity of APE1.APE1 knockdown upregulates c-myc mRNA expressionTo assess if APE1 can in fact regulate c-myc mRNA expression in cells, we employed siRNA to knockdown APE1 expression in HeLa cells. We previously established that about 80% transfection efficiency can be achieved using Lipofectamine 2000 (33). We used double-stranded Dicer substrate RNAi (dsRNAi) against APE1 (APE1-dsRNAi) based on siRNA sequences used successfully to knockdown APE1 (47). As a negative control, we used DS Scrambled Negative (Control-dsRNAi) with randomly rearranged nucleotides. The effect of APE1-dsRNAi on APE1 protein and mRNA levels was monitored over a 2-day period. The levels of APE1 mRNA were examined using qRT-PCR and normalized against β-actin mRNA levels. Figure 8B (top panel) shows that APE1 mRNA was reduced by 80% at 24 h and 85% at 48 h. This was accompanied by reduction of APE1 protein of 60% and 70% at 24 and 48 h, respectively, as determined by western analysis (Figure 8A).The effect of APE1 knockdown on c-myc mRNA expression was investigated. Using the same total RNA samples as above, we found 1.7-fold and 4.6-fold increase at 24 and 48 h, respectively, in the steady-state c-myc mRNA levels upon knockdown of APE1 (Figure 8B, bottom panel). To assess whether elevated c-myc mRNA levels upon APE1 knockdown were due to more stable transcript, the decay of c-myc mRNA was examined. We transfected HeLa cells with either APE1-dsRNAi or Control-dsRNAi for 30 h, followed by treatment with 200 μM 5,6-dichloro-1-β-d-ribobenzimidazole (DRB) to inhibit transcription. Total RNA extracted at various time points were analyzed for c-myc and β-actin mRNA levels by qRT-PCR. c-myc mRNA level was normalized to β-actin mRNA level at each time point and then expressed as percentage to the level at 0 min (Figure 9). As shown in Figure 9, the levels of c-myc mRNA in the Control-dsRNAi-treated cells exhibited decay with a half-life of about 30 min. In contrast, the levels of c-myc mRNA in APE1-dsRNAi-treated cells were relatively stable up to 60 min.

Purification and identification of the 35-kDa liver endoribonucleaseWe previously reported partial purification of an endoribonuclease from rat liver for cleaving the c-myc CRD RNA (30). Five major proteins of ∼10–35-kDa size were co-purified with this activity. To determine the identity of the responsible enzyme, we first repeated earlier enzyme purification steps with slight modifications as described in the ‘Materials and Methods’ section. Two endoribonuclease activities from the final gel filtration column at elution volumes 46–50 ml and 62–68 ml, corresponding to 30–40-kDa and 10–20-kDa sizes, were prominent (Figure 1A). Pooled protein fractions visualized by silver-staining in SDS–PAGE gel (Figure 1B) shows a distinct protein band of ∼35 kDa at elution volume 40–50 ml and four protein bands from 10–20 kDa at elution volume 60–66 ml. These fractions were separately pooled from three separate preparations and visualized by Coomasie blue-staining after SDS–PAGE (Figure 1C). Gel slices at around protein bands 1–6 (Figure 1C) were excised for LC/MS mass spectrometry protein identification analysis at the Genome BC Proteomics Centre, University of Victoria. Based on molecular weight of proteins, number of matched peptides, and percentage of amino acid sequence against the ‘rodentia’ protein database, the top three proteins that matched for each band were selected (Table 1). The three major proteins, L-3-hydroxyacyl-CoA dehydrogenase (HADHSC), annexin III and AP endonuclease (APE1), were identified as possible candidates for the 35-kDa endoribonuclease. The band in between band 2 and 3 in Figure 1C was determined to be HADHSC by LC/MS spectrometry (data not shown). This band may in fact represent a truncated or non-post-translationally modified form of HADHSC. In addition to the criteria described above, HADHSC was chosen because it contains a predicted RNA-binding Rossman fold motif (35) and annexin III was chosen because a related protein, annexin II, has been shown to bind human c-myc RNA (36). Both HADHSC and annexin III were present in the gel filtration fractions as determined by western analysis (data not shown). However, neither purified recombinant HADHSC nor annexin III exhibited any endoribonuclease activity (see later in Figure 6B). APE1 then became the prime candidate because it is a known multifunctional protein with DNA-specific endonuclease and RNase H-like activities (37–39). To further confirm its identity, we performed duplicate LC/MS mass spectrometry on individual 35–38-kDa protein in the gel slices isolated after heparin-Sepharose chromatography. Both HADHSC and APE1 were again two of the matched proteins (data not shown), suggesting that both proteins co-purified with the 35-kDa endoribonuclease activity.
Table 1.Summary of LC/MS mass spectrometry data and peptide analysis to identify purified proteins following gel filtrationProtein band #Top three protein matches from Rodentia speciesAmino sequence coverage (%)Number of matched peptides11) Apurinic/apyrimidinic endonuclease lyase (AP endonuclease/APE1) (35.8 kDa)1) 32%1) 72) Annexin III (36.5 kDa)2) 38%2) 103) Aldo-keto reductase E1 (34.8 kDa)3) 40%3) 112L-3-hydroxyacyl-CoA dehydrogenase (HADHSC) (34 kDa)55%831) Peroxisomal enoyl hydratase-like protein (36.5 kDa)1) 46%1) 122) HADHSC (34 kDa)2) 20%2) 33) Glutathione S-transferase (25.6 kDa)3) 54%3) 74Cyclophilin B (23 kDa)56%145Pancreatic ribonuclease A (RNase1) (17 kDa)45%56Cytochrome C (12.5 kDa)55%11

Purification and identification of the 35-kDa liver endoribonuclease

We previously reported partial purification of an endoribonuclease from rat liver for cleaving the c-myc CRD RNA (30). Five major proteins of ∼10–35-kDa size were co-purified with this activity. To determine the identity of the responsible enzyme, we first repeated earlier enzyme purification steps with slight modifications as described in the ‘Materials and Methods’ section. Two endoribonuclease activities from the final gel filtration column at elution volumes 46–50 ml and 62–68 ml, corresponding to 30–40-kDa and 10–20-kDa sizes, were prominent (Figure 1A). Pooled protein fractions visualized by silver-staining in SDS–PAGE gel (Figure 1B) shows a distinct protein band of ∼35 kDa at elution volume 40–50 ml and four protein bands from 10–20 kDa at elution volume 60–66 ml. These fractions were separately pooled from three separate preparations and visualized by Coomasie blue-staining after SDS–PAGE (Figure 1C). Gel slices at around protein bands 1–6 (Figure 1C) were excised for LC/MS mass spectrometry protein identification analysis at the Genome BC Proteomics Centre, University of Victoria. Based on molecular weight of proteins, number of matched peptides, and percentage of amino acid sequence against the ‘rodentia’ protein database, the top three proteins that matched for each band were selected (Table 1). The three major proteins, L-3-hydroxyacyl-CoA dehydrogenase (HADHSC), annexin III and AP endonuclease (APE1), were identified as possible candidates for the 35-kDa endoribonuclease. The band in between band 2 and 3 in Figure 1C was determined to be HADHSC by LC/MS spectrometry (data not shown). This band may in fact represent a truncated or non-post-translationally modified form of HADHSC. In addition to the criteria described above, HADHSC was chosen because it contains a predicted RNA-binding Rossman fold motif (35) and annexin III was chosen because a related protein, annexin II, has been shown to bind human c-myc RNA (36). Both HADHSC and annexin III were present in the gel filtration fractions as determined by western analysis (data not shown). However, neither purified recombinant HADHSC nor annexin III exhibited any endoribonuclease activity (see later in Figure 6B). APE1 then became the prime candidate because it is a known multifunctional protein with DNA-specific endonuclease and RNase H-like activities (37–39). To further confirm its identity, we performed duplicate LC/MS mass spectrometry on individual 35–38-kDa protein in the gel slices isolated after heparin-Sepharose chromatography. Both HADHSC and APE1 were again two of the matched proteins (data not shown), suggesting that both proteins co-purified with the 35-kDa endoribonuclease activity.
Table 1.Summary of LC/MS mass spectrometry data and peptide analysis to identify purified proteins following gel filtrationProtein band #Top three protein matches from Rodentia speciesAmino sequence coverage (%)Number of matched peptides11) Apurinic/apyrimidinic endonuclease lyase (AP endonuclease/APE1) (35.8 kDa)1) 32%1) 72) Annexin III (36.5 kDa)2) 38%2) 103) Aldo-keto reductase E1 (34.8 kDa)3) 40%3) 112L-3-hydroxyacyl-CoA dehydrogenase (HADHSC) (34 kDa)55%831) Peroxisomal enoyl hydratase-like protein (36.5 kDa)1) 46%1) 122) HADHSC (34 kDa)2) 20%2) 33) Glutathione S-transferase (25.6 kDa)3) 54%3) 74Cyclophilin B (23 kDa)56%145Pancreatic ribonuclease A (RNase1) (17 kDa)45%56Cytochrome C (12.5 kDa)55%11

Summary of LC/MS mass spectrometry data and peptide analysis to identify purified proteins following gel filtration

The 17-kDa endoribonuclease belongs to the RNase A superfamilyBased on the above identification criteria, protein bands 4, 5 and 6 (Figure 1C) were convincingly identified as cyclophilin B, pancreatic rat ribonuclease A family member RNase1, and cytochrome c respectively (Table 1). We reasoned that the 17-kDa RNase 1 is most likely responsible for the 10–20-kDa endoribonuclease activity based on the following observations: (i) RNase 1 is the only known protein with ribonuclease activity identified from 10–20-kDa protein bands 4, 5 and 6, (ii) RNA sequence cleavage specificity of the partially purified 10–20-kDa native enzyme from gel filtration resembled, yet was not identical, to that exhibited by the bovine pancreatic RNase A (compare lanes 10 and 11 to lane 8 in Figure 2A), and (iii) polyclonal antibody against RNase A detected a protein band at around 17-kDa range in elution volume 60–80 ml from gel filtration column (lanes 7 and 8, Figure 3) which correlated with the endoribonuclease activity (Figure 1A), and finally the presence of 17-kDa protein band on silver-stained SDS–PAGE gel (Figure 1B).

The 17-kDa endoribonuclease belongs to the RNase A superfamily

Based on the above identification criteria, protein bands 4, 5 and 6 (Figure 1C) were convincingly identified as cyclophilin B, pancreatic rat ribonuclease A family member RNase1, and cytochrome c respectively (Table 1). We reasoned that the 17-kDa RNase 1 is most likely responsible for the 10–20-kDa endoribonuclease activity based on the following observations: (i) RNase 1 is the only known protein with ribonuclease activity identified from 10–20-kDa protein bands 4, 5 and 6, (ii) RNA sequence cleavage specificity of the partially purified 10–20-kDa native enzyme from gel filtration resembled, yet was not identical, to that exhibited by the bovine pancreatic RNase A (compare lanes 10 and 11 to lane 8 in Figure 2A), and (iii) polyclonal antibody against RNase A detected a protein band at around 17-kDa range in elution volume 60–80 ml from gel filtration column (lanes 7 and 8, Figure 3) which correlated with the endoribonuclease activity (Figure 1A), and finally the presence of 17-kDa protein band on silver-stained SDS–PAGE gel (Figure 1B).

The 35-kDa endoribonuclease activity is not attributable to a modified form of pancreatic RNase AThe cleavage sites on c-myc CRD RNA generated by the purified native enzyme in the heparin-Sepharose and gel filtration fractions with proteins of 10–20-kDa and 30–40-kDa size range were mapped as previously described using RNase T1 digest and alkaline hydrolysis (30,31). The 30–40-kDa fraction appeared to exhibit similarity with the 10–20-kDa fractions in sequence cleavage specificity (compare lanes 4, 5, 6, 12 and 13 with lanes 10 and 11, Figure 2A). However, there were some differences in that the 30–40-kDa fractions had stronger preference for the 1751UA dinucleotide (shown by asterisks) (Figure 2A). The 30–40-kDa fractions also had some preference for the 1771CA and 1773UA dinucleotides (Figure 2A).Given the similarity in RNA cleavage patterns between the 30–40-kDa and the 10–20-kDa fractions, together with the observations that the polyclonal antibody against RNase A detected a faint band corresponding to 30-kDa (lane 6 in Figure 3) and 35-kDa (data not shown) polypeptides on western blots, we considered whether the 30–40-kDa endoribonuclease activity is contributed by a modified form of pancreatic RNase A. This is plausible because the dimeric RNase A is known to exist (40), and a monomeric N-glycosylated RNase A of approximately 24–36-kDa size has been reported (41). To test the possibility that the 35-kDa endoribonuclease is a disulfide-linked dimeric RNase A, we treated the purified post-heparin Sepharose fraction with 250 mM DTT before gel filtration analysis. Endoribonuclease analysis of the fractions showed two distinct activities (data not shown) similar to the profile of the original sample (Figure 1A). We therefore concluded that the 35-kDa endoribonuclease is unlikely to be a disulfide-linked dimeric form of 17-kDa RNase1, and is likely a monomeric protein. To test the possibility that the 35-kDa endoribonuclease is a N-glycosylated RNase A, we treated the heparin Sepharose fraction with 100 U endoglycosidase F before gel filtration chromatography. Our results (data not shown) show that the elution profile of endoribonuclease activity remained unchanged compared to the original sample. Thus, the 35-kDa endoribonuclease is unlikely to be N-glycosylated RNase A.

The 35-kDa endoribonuclease activity is not attributable to a modified form of pancreatic RNase A

The cleavage sites on c-myc CRD RNA generated by the purified native enzyme in the heparin-Sepharose and gel filtration fractions with proteins of 10–20-kDa and 30–40-kDa size range were mapped as previously described using RNase T1 digest and alkaline hydrolysis (30,31). The 30–40-kDa fraction appeared to exhibit similarity with the 10–20-kDa fractions in sequence cleavage specificity (compare lanes 4, 5, 6, 12 and 13 with lanes 10 and 11, Figure 2A). However, there were some differences in that the 30–40-kDa fractions had stronger preference for the 1751UA dinucleotide (shown by asterisks) (Figure 2A). The 30–40-kDa fractions also had some preference for the 1771CA and 1773UA dinucleotides (Figure 2A).

Given the similarity in RNA cleavage patterns between the 30–40-kDa and the 10–20-kDa fractions, together with the observations that the polyclonal antibody against RNase A detected a faint band corresponding to 30-kDa (lane 6 in Figure 3) and 35-kDa (data not shown) polypeptides on western blots, we considered whether the 30–40-kDa endoribonuclease activity is contributed by a modified form of pancreatic RNase A. This is plausible because the dimeric RNase A is known to exist (40), and a monomeric N-glycosylated RNase A of approximately 24–36-kDa size has been reported (41). To test the possibility that the 35-kDa endoribonuclease is a disulfide-linked dimeric RNase A, we treated the purified post-heparin Sepharose fraction with 250 mM DTT before gel filtration analysis. Endoribonuclease analysis of the fractions showed two distinct activities (data not shown) similar to the profile of the original sample (Figure 1A). We therefore concluded that the 35-kDa endoribonuclease is unlikely to be a disulfide-linked dimeric form of 17-kDa RNase1, and is likely a monomeric protein. To test the possibility that the 35-kDa endoribonuclease is a N-glycosylated RNase A, we treated the heparin Sepharose fraction with 100 U endoglycosidase F before gel filtration chromatography. Our results (data not shown) show that the elution profile of endoribonuclease activity remained unchanged compared to the original sample. Thus, the 35-kDa endoribonuclease is unlikely to be N-glycosylated RNase A.

Immuno-depletion of the 35-kDa endoribonuclease with APE1 monoclonal antibodyOur results so far suggested that the 35-kDa endoribonuclease was APE1 which we then tested by western analysis of the 30–40-kDa fractions from gel filtration. Figure 3 shows that APE1 is present only in the elution volume 45–55 ml (lane 6, Figure 3, bottom panel) which correlates with the 35-kDa endoribonuclease activity (Figure 1A). We performed immunodepletion studies of the purified native 35-kDa endoribonuclease with the APE1 antibody. Endoribonuclease analysis of the fractions eluted from an APE1 monoclonal antibody spin column is shown in Figure 4A. Washed fractions (lanes 6 and 7) had no endoribonuclease activity while the earlier elution fractions (lanes 8 and 9) contained endoribonuclease activity. Endoribonuclease activity in Elution 1 (lane 8) was as high as those from partially purified post-heparin-Sepharose sample (lanes 2 and 3). In both cases, excess nuclease activity led to almost complete degradation of RNA substrate. Figure 4B shows western blot analysis of samples taken from the same immunoprecipitation experiment described in Figure 4A. There was striking correlation between the endoribonuclease activity in the pre-loaded purified native 30–40-kDa gel filtration fraction, Elutions 1 and 2, and the presence of APE1 in those fractions (compare lanes 2, 6 and 7 in Figure 4B with lanes 4, 5, 8 and 9 in Figure 4A). In addition, there was correlation between the absence of endoribonuclease activity in the wash fractions, and the absence of APE1 in those fractions (compare lanes 4 and 5 in Figure 4B with lanes 6 and 7 in Figure 4A). To confirm the specificity of the immunodepletion experiment using APE1 antibody, we performed similar experiments using another antibody cross-linked to the spin column, which in this case was syntaxin18 antibody. Figure 4C shows that endoribonuclease analysis of the fractions eluted from syntaxin18 antibody spin column. The flow through (lane 2) and washed fractions (lane 3) showed strong nuclease activity whereas the Elution 1 and 2 (lanes 4 and 5) had no endoribonuclease activity. This control experiment confirmed that the immunodepletion of endoribonuclease activity observed in Figure 4A was produced specifically by the presence of anti-APE1 antibody.

Immuno-depletion of the 35-kDa endoribonuclease with APE1 monoclonal antibody

Our results so far suggested that the 35-kDa endoribonuclease was APE1 which we then tested by western analysis of the 30–40-kDa fractions from gel filtration. Figure 3 shows that APE1 is present only in the elution volume 45–55 ml (lane 6, Figure 3, bottom panel) which correlates with the 35-kDa endoribonuclease activity (Figure 1A). We performed immunodepletion studies of the purified native 35-kDa endoribonuclease with the APE1 antibody. Endoribonuclease analysis of the fractions eluted from an APE1 monoclonal antibody spin column is shown in Figure 4A. Washed fractions (lanes 6 and 7) had no endoribonuclease activity while the earlier elution fractions (lanes 8 and 9) contained endoribonuclease activity. Endoribonuclease activity in Elution 1 (lane 8) was as high as those from partially purified post-heparin-Sepharose sample (lanes 2 and 3). In both cases, excess nuclease activity led to almost complete degradation of RNA substrate. Figure 4B shows western blot analysis of samples taken from the same immunoprecipitation experiment described in Figure 4A. There was striking correlation between the endoribonuclease activity in the pre-loaded purified native 30–40-kDa gel filtration fraction, Elutions 1 and 2, and the presence of APE1 in those fractions (compare lanes 2, 6 and 7 in Figure 4B with lanes 4, 5, 8 and 9 in Figure 4A). In addition, there was correlation between the absence of endoribonuclease activity in the wash fractions, and the absence of APE1 in those fractions (compare lanes 4 and 5 in Figure 4B with lanes 6 and 7 in Figure 4A). To confirm the specificity of the immunodepletion experiment using APE1 antibody, we performed similar experiments using another antibody cross-linked to the spin column, which in this case was syntaxin18 antibody. Figure 4C shows that endoribonuclease analysis of the fractions eluted from syntaxin18 antibody spin column. The flow through (lane 2) and washed fractions (lane 3) showed strong nuclease activity whereas the Elution 1 and 2 (lanes 4 and 5) had no endoribonuclease activity. This control experiment confirmed that the immunodepletion of endoribonuclease activity observed in Figure 4A was produced specifically by the presence of anti-APE1 antibody.

Recombinant human APE1 and purified native 35-kDa endoribonuclease exhibited similar RNA cleavage patternWe then directly tested whether the recombinant human APE1 possesses endoribonuclease activity for c-myc CRD RNA. The purified proteins were almost homogenous with more than 95% pure based on Coomasie blue-staining after SDS–PAGE (Figure 5A, left panel) and identity checked by western analysis (Figure 5A, right panel). The minor, lower-molecular-weight band in the immunoblot assay (Figure 5A, right panel) and the SDS–PAGE gel (Figure 5A, lane 2 in left panel) was likely to be a N-terminal cleavage product of APE1 (42). This is commonly observed as the protein is quite susceptible to specific hydrolysis upon boiling. Endoribonuclease analysis confirmed that recombinant APE1 from two separate sources (one was prepared by us and the other from Dr. Ian Hickson's lab, University of Oxford) exhibited endoribonuclease activity against c-myc CRD RNA with a distinct preference for the 1751UA dinucleotide (Figure 5B, left panel). Furthermore, when the recombinant APE1 was reduced, denatured with guanidine hydrochloride and renatured, the renatured recombinant APE1 exhibits the same cleavage specificity (lanes 2–4, right panel in Figure 5B) as the native enzyme (lanes 5–7, right panel in Figure 5B) with 1751UA still being the dominant cleavage site.

Recombinant human APE1 and purified native 35-kDa endoribonuclease exhibited similar RNA cleavage pattern

We then directly tested whether the recombinant human APE1 possesses endoribonuclease activity for c-myc CRD RNA. The purified proteins were almost homogenous with more than 95% pure based on Coomasie blue-staining after SDS–PAGE (Figure 5A, left panel) and identity checked by western analysis (Figure 5A, right panel). The minor, lower-molecular-weight band in the immunoblot assay (Figure 5A, right panel) and the SDS–PAGE gel (Figure 5A, lane 2 in left panel) was likely to be a N-terminal cleavage product of APE1 (42). This is commonly observed as the protein is quite susceptible to specific hydrolysis upon boiling. Endoribonuclease analysis confirmed that recombinant APE1 from two separate sources (one was prepared by us and the other from Dr. Ian Hickson's lab, University of Oxford) exhibited endoribonuclease activity against c-myc CRD RNA with a distinct preference for the 1751UA dinucleotide (Figure 5B, left panel). Furthermore, when the recombinant APE1 was reduced, denatured with guanidine hydrochloride and renatured, the renatured recombinant APE1 exhibits the same cleavage specificity (lanes 2–4, right panel in Figure 5B) as the native enzyme (lanes 5–7, right panel in Figure 5B) with 1751UA still being the dominant cleavage site.

Specificity and identification of active residues in APE1 for endoribonuclease activitySpecific residues Glu96 and His309 have been identified as critical for the AP DNA endonuclease (37,43), exonuclease (44) and RNase H activities of APE1 (39). To examine whether the c-myc CRD RNA-cleaving activity of APE1 requires the same amino acid residues as these nuclease activities of APE1, E96A and H309N APE1 mutant polypeptides were purified under identical purification methods as the wild-type APE1 and tested for endoribonuclease activity under our standard endoribonuclease assay. Figure 6A shows the lack of RNA cleaving activity of the E96A mutant at up to 0.5 μg of the protein (lanes 6–8, Figure 6A). H309N mutant also did not exhibit any endoribonuclease activity at 0.1 and 0.3 μg (lanes 3 and 5, Figure 6A). Interestingly, at a higher concentration (0.5 μg), the H309N mutant cleaved CRD RNA at 1727CA, 1768CA, 1771CA, 1773UA and 1775CA but not at 1757UA, 1751UA, 1747UA and 1742CA (compare lane 4 to lanes 2 and 9, Figure 6A). These results suggest that the endoribonuclease activity of APE1 against c-myc CRD RNA shares, to some extent, common critical amino acid residues with AP DNA endonuclease, RNase H and exonuclease activities. ND42 mutant APE1 missing the first 42 amino acids at the N-terminus demonstrated activity which is similar to the wild-type APE1 (Figure 6A, right panel). This suggests that the N-terminus domain, at least the first 42 amino acids, does not participate in the RNA-cleaving of APE1.To further confirm specificity of the endoribonuclease activity exhibited by the wild-type and APE1 mutants, we also tested other recombinant proteins. Figure 6B shows that the lack of endoribonuclease activity exhibited by recombinant HADHSC (lanes 4 and 5) and annexin III (lanes 6 and 7) while the wild-type APE1 (lanes 2 and 3) exhibited the typical endoribonuclease activity. Overall, the results in Figure 6 confirms the specificity of the endoribonuclease activity exhibited by the purified wild-type APE1 and absence of RNase A-like contaminants amongst other purified recombinant proteins including the E96A and H309N APE1 mutants.

Specificity and identification of active residues in APE1 for endoribonuclease activity

Specific residues Glu96 and His309 have been identified as critical for the AP DNA endonuclease (37,43), exonuclease (44) and RNase H activities of APE1 (39). To examine whether the c-myc CRD RNA-cleaving activity of APE1 requires the same amino acid residues as these nuclease activities of APE1, E96A and H309N APE1 mutant polypeptides were purified under identical purification methods as the wild-type APE1 and tested for endoribonuclease activity under our standard endoribonuclease assay. Figure 6A shows the lack of RNA cleaving activity of the E96A mutant at up to 0.5 μg of the protein (lanes 6–8, Figure 6A). H309N mutant also did not exhibit any endoribonuclease activity at 0.1 and 0.3 μg (lanes 3 and 5, Figure 6A). Interestingly, at a higher concentration (0.5 μg), the H309N mutant cleaved CRD RNA at 1727CA, 1768CA, 1771CA, 1773UA and 1775CA but not at 1757UA, 1751UA, 1747UA and 1742CA (compare lane 4 to lanes 2 and 9, Figure 6A). These results suggest that the endoribonuclease activity of APE1 against c-myc CRD RNA shares, to some extent, common critical amino acid residues with AP DNA endonuclease, RNase H and exonuclease activities. ND42 mutant APE1 missing the first 42 amino acids at the N-terminus demonstrated activity which is similar to the wild-type APE1 (Figure 6A, right panel). This suggests that the N-terminus domain, at least the first 42 amino acids, does not participate in the RNA-cleaving of APE1.

To further confirm specificity of the endoribonuclease activity exhibited by the wild-type and APE1 mutants, we also tested other recombinant proteins. Figure 6B shows that the lack of endoribonuclease activity exhibited by recombinant HADHSC (lanes 4 and 5) and annexin III (lanes 6 and 7) while the wild-type APE1 (lanes 2 and 3) exhibited the typical endoribonuclease activity. Overall, the results in Figure 6 confirms the specificity of the endoribonuclease activity exhibited by the purified wild-type APE1 and absence of RNase A-like contaminants amongst other purified recombinant proteins including the E96A and H309N APE1 mutants.

APE1 is an endoribonucleaseTo confirm that APE1 is indeed an endoribonuclease, we challenged the enzyme with two new substrates. First, we tested APE1 on 32P-5′-labeled or 32P-UTP internally labeled RNA corresponding to nts 1730–1766 of c-myc CRD RNA. Figure 7A shows the 5′ fragment of the cleavage products generated by APE1 on the 5′-labeled RNA. Figure 7B shows the cleavage products generated when the RNA was internally labeled with 32P-UTP. The corresponding 5′ and 3′ cleavage fragments were clearly visible indicating the endonucleolytic cleavage of the RNA substrate by APE1. To further confirm this, we tested APE1 on a 5′-labeled DNA oligo corresponding to nts 1742–1757 of c-myc CRD DNA with an incorporation of an uridine at position 1751. As expected, being an endoribonuclease APE1 was only capable of cleaving at position 1751 to generate the 5′-fragment of the oligo (Figure 7C). A recent study has shown that APE1 can cleave AP-site-containing single-stranded RNA (45,46), presumably at the phosphodiester bond immediately 5′ to the AP-site. Therefore, it is expected that removal of any bases, including uracil, in single-stranded region of RNA will be susceptible to the abasic RNA-endonuclease activity of APE1.

APE1 is an endoribonuclease

To confirm that APE1 is indeed an endoribonuclease, we challenged the enzyme with two new substrates. First, we tested APE1 on 32P-5′-labeled or 32P-UTP internally labeled RNA corresponding to nts 1730–1766 of c-myc CRD RNA. Figure 7A shows the 5′ fragment of the cleavage products generated by APE1 on the 5′-labeled RNA. Figure 7B shows the cleavage products generated when the RNA was internally labeled with 32P-UTP. The corresponding 5′ and 3′ cleavage fragments were clearly visible indicating the endonucleolytic cleavage of the RNA substrate by APE1. To further confirm this, we tested APE1 on a 5′-labeled DNA oligo corresponding to nts 1742–1757 of c-myc CRD DNA with an incorporation of an uridine at position 1751. As expected, being an endoribonuclease APE1 was only capable of cleaving at position 1751 to generate the 5′-fragment of the oligo (Figure 7C). A recent study has shown that APE1 can cleave AP-site-containing single-stranded RNA (45,46), presumably at the phosphodiester bond immediately 5′ to the AP-site. Therefore, it is expected that removal of any bases, including uracil, in single-stranded region of RNA will be susceptible to the abasic RNA-endonuclease activity of APE1.

APE1 knockdown upregulates c-myc mRNA expressionTo assess if APE1 can in fact regulate c-myc mRNA expression in cells, we employed siRNA to knockdown APE1 expression in HeLa cells. We previously established that about 80% transfection efficiency can be achieved using Lipofectamine 2000 (33). We used double-stranded Dicer substrate RNAi (dsRNAi) against APE1 (APE1-dsRNAi) based on siRNA sequences used successfully to knockdown APE1 (47). As a negative control, we used DS Scrambled Negative (Control-dsRNAi) with randomly rearranged nucleotides. The effect of APE1-dsRNAi on APE1 protein and mRNA levels was monitored over a 2-day period. The levels of APE1 mRNA were examined using qRT-PCR and normalized against β-actin mRNA levels. Figure 8B (top panel) shows that APE1 mRNA was reduced by 80% at 24 h and 85% at 48 h. This was accompanied by reduction of APE1 protein of 60% and 70% at 24 and 48 h, respectively, as determined by western analysis (Figure 8A).The effect of APE1 knockdown on c-myc mRNA expression was investigated. Using the same total RNA samples as above, we found 1.7-fold and 4.6-fold increase at 24 and 48 h, respectively, in the steady-state c-myc mRNA levels upon knockdown of APE1 (Figure 8B, bottom panel). To assess whether elevated c-myc mRNA levels upon APE1 knockdown were due to more stable transcript, the decay of c-myc mRNA was examined. We transfected HeLa cells with either APE1-dsRNAi or Control-dsRNAi for 30 h, followed by treatment with 200 μM 5,6-dichloro-1-β-d-ribobenzimidazole (DRB) to inhibit transcription. Total RNA extracted at various time points were analyzed for c-myc and β-actin mRNA levels by qRT-PCR. c-myc mRNA level was normalized to β-actin mRNA level at each time point and then expressed as percentage to the level at 0 min (Figure 9). As shown in Figure 9, the levels of c-myc mRNA in the Control-dsRNAi-treated cells exhibited decay with a half-life of about 30 min. In contrast, the levels of c-myc mRNA in APE1-dsRNAi-treated cells were relatively stable up to 60 min.

APE1 knockdown upregulates c-myc mRNA expression

To assess if APE1 can in fact regulate c-myc mRNA expression in cells, we employed siRNA to knockdown APE1 expression in HeLa cells. We previously established that about 80% transfection efficiency can be achieved using Lipofectamine 2000 (33). We used double-stranded Dicer substrate RNAi (dsRNAi) against APE1 (APE1-dsRNAi) based on siRNA sequences used successfully to knockdown APE1 (47). As a negative control, we used DS Scrambled Negative (Control-dsRNAi) with randomly rearranged nucleotides. The effect of APE1-dsRNAi on APE1 protein and mRNA levels was monitored over a 2-day period. The levels of APE1 mRNA were examined using qRT-PCR and normalized against β-actin mRNA levels. Figure 8B (top panel) shows that APE1 mRNA was reduced by 80% at 24 h and 85% at 48 h. This was accompanied by reduction of APE1 protein of 60% and 70% at 24 and 48 h, respectively, as determined by western analysis (Figure 8A).

The effect of APE1 knockdown on c-myc mRNA expression was investigated. Using the same total RNA samples as above, we found 1.7-fold and 4.6-fold increase at 24 and 48 h, respectively, in the steady-state c-myc mRNA levels upon knockdown of APE1 (Figure 8B, bottom panel). To assess whether elevated c-myc mRNA levels upon APE1 knockdown were due to more stable transcript, the decay of c-myc mRNA was examined. We transfected HeLa cells with either APE1-dsRNAi or Control-dsRNAi for 30 h, followed by treatment with 200 μM 5,6-dichloro-1-β-d-ribobenzimidazole (DRB) to inhibit transcription. Total RNA extracted at various time points were analyzed for c-myc and β-actin mRNA levels by qRT-PCR. c-myc mRNA level was normalized to β-actin mRNA level at each time point and then expressed as percentage to the level at 0 min (Figure 9). As shown in Figure 9, the levels of c-myc mRNA in the Control-dsRNAi-treated cells exhibited decay with a half-life of about 30 min. In contrast, the levels of c-myc mRNA in APE1-dsRNAi-treated cells were relatively stable up to 60 min.

DISCUSSIONWe set the goal of identifying endoribonuclease(s) that are capable of cleaving c-myc CRD RNA in vitro for four specific reasons. First, endonucleolytic cleavage of c-myc mRNA has been shown to occur in vivo (6,7,25–29), implying the presence of the responsible endoribonuclease(s) in cells. Second, many vertebrate mRNA decay intermediates have been detected but only a handful of endoribonucleases that degrade specific mRNAs have been identified (1). Third, our understanding about the role of endoribonucleases in mRNA decay and gene expression in vertebrates is still unclear. Lastly, given that c-Myc is implicated in human cancers, characterization of enzymes that degrade c-myc mRNA may help develop new therapeutic approach via destruction of c-myc mRNA. In this study, we describe the purification and identification of RNase1 as a 17-kDa enzyme and APE1 as a 35-kDa endoribonuclease that are capable of cleaving c-myc CRD RNA in vitro.RNase1 is a known endoribonuclease, belonging to the RNase A superfamily of ribonucleases (48). Therefore, it is not surprising to see its ability to cleave c-myc CRD RNA in vitro. However, whether RNase1 has a role in cleaving c-myc CRD RNA, and therefore in the regulation of c-myc mRNA level in vivo remains unknown. In addition, APE1 was identified as the 35-kDa endoribonuclease that cleaved c-myc mRNA in vitro. Four findings confirm that APE1 is the protein that contributes to the 35-kDa endoribonuclease activity. (i) Two independent sets of LC/MS-MS data identified APE1 with 32% amino acid sequence coverage and seven matched peptides (Table 1). (ii) As determined by western analysis, APE1 is present in the 30–40-kDa fractions from gel filtration which contained the endoribonuclease activity (Figures 1A and 3). (iii) The endoribonuclease activity was specifically immuno-depleted with APE1 monoclonal antibody (Figure 4). (iv) The cleavage pattern generated on c-myc CRD RNA by the recombinant human APE1 is similar to that generated by the purified native 35-kDa endoribonuclease from rat liver (Figure 5B).Overall, the endoribonuclease activity of the purified native APE1 and the recombinant APE1 resemble RNase A-like activity, a standard contaminating ribonuclease in RNA studies. However, the following evidence strongly argue against the possibility that the observed endoribonuclease activity of APE1 is due to RNase A contaminants: (i) APE1 antibody which does not cross react with RNase A, specifically immuno-depleted the endoribonuclease activity of the purified native APE1 (Figure 4); (ii) gel filtration used in the last step to partially purify the 35-kDa native APE1 would have excluded any possible RNase A contaminants which are typically below 20 kDa in size; (iii) the cleavage pattern generated by the purified native APE1 on c-myc CRD RNA (Figure 2) and other RNAs are similar but not identical (data not shown) to that generated by RNase A; and (iv) at higher concentrations, other recombinant proteins including mutant APE1 E96A which was purified under identical conditions failed to exhibit any endoribonuclease activity (Figure 6).APE1 has been identified as an unusual, multi-functional protein (37,38). It endonucleolytically cleaves DNA at AP sites, and is a key enzyme in the base excision repair pathway (37). APE1 also provides the major redox activity for AP-1, p53, H1F1alpha and other transcription factors (49). In addition to its AP DNA endonuclease activity, APE1 has 3′ -5′ DNA exonuclease (50) and 3′ phoshodiesterase activities (37). APE1 has also been shown to bind to a fragment of human topoisomerase IIα RNA and possesses RNase H activity (39) but the overall biological significance of this function has remained unclear. We have not examined the RNase H activity of recombinant APE1 but the previously partially purified native enzyme which contained both RNase1 and APE1 did not appear to have any RNase H activity (30). Here, we show that recombinant APE1 cleaves specific sequences on the c-myc CRD RNA, namely in between CA and UA dinucleotides (Figure 5B). To our knowledge, this is the first demonstration of a sequence specific endoribonuclease activity intrinsic to APE1. The endoribonuclease activity of APE1 on c-myc CRD RNA appears to share some of the same active site with other nuclease activities of the protein because both E96A and H309N APE1 mutants were inactive or less active in cleaving c-myc CRD RNA.To initiate studies on examining the RNA-cleaving potential of APE1 in cells, we used the siRNA technology to transiently knockdown APE1 and then measure c-myc mRNA expression in HeLa cells. Figure 8 shows that transient knockdown of APE1 at the protein and mRNA levels lead to significant increase in steady-state c-myc mRNA. Such observation could be attributed to: (i) more stable c-myc mRNA due to less APE1 in cleaving c-myc mRNA, or/and (ii) decrease in transcriptional repressor of c-myc gene due to less APE1 in keeping some transcriptional factors active. To test the former hypothesis, we assessed the decay of c-myc mRNA upon knockdown of APE1. We found that indeed c-myc mRNA was more stable upon knockdown of APE1 supporting the notion that APE1 can cleave c-myc mRNA and control its abundance in cells. Interestingly, one of the c-myc mRNA endonucleolytic decay intermediates previously detected in cells (7) corresponded to 1727CA, which is a site cleave by APE1 (Figure 2). Our results of increased c-myc mRNA upon knockdown of APE1 is in contrast to a recent report of a modest decrease in c-myc mRNA using micro-array approach (51). This discrepancy could due to the differences in the experimental design. These investigators used stable siRNA-inducible cells that were knockdown for 10 days upon stimulation with siRNA expression and as pointed out by the investigators, compensatory mechanisms and indirect effects may set in for the observed molecular results (51).We have observed stable association of APE1 with YB-1 and hnRNP-H both of which are involved in RNA metabolism (52). In addition to its expected localization to the nucleus, APE1 is also found in the cytoplasm and in some cases, appears to be exclusively located there (38). Such observations and the finding of redistribution of the protein between the nucleus and cytoplasm in some cancers had many researchers baffled for some time (38,49). APE1 has been reported to associate with ribosomes in motor neurons and also possibly in highly proliferative cells including hepatocytes (38), and such observations had led to the speculation that it may have an undefined role in ribosomal function (49). Indeed, a recent report provided strong evidence that the N-terminus of APE1 is physically associated with a number of proteins known to be involved in ribosome biogenesis and RNA processing (46).Interestingly, we have purified APE1 from rat liver polysomal fraction which supports its role in RNA processing. We had earlier shown that APE1 interacts with an RNA-binding protein hnRNP-L in negative regulation of the parathyroid hormone gene (53). Of particular interest is the report that hnRNP-L can regulate the stability of human VEGF mRNA through its AU-rich element during hypoxia (54). Thus, given the evidence provided in this study and the prior evidence, it is tempting to speculate that a previously unknown function of cytoplasmic APE1 is its ability to serve as an endoribonuclease for mRNAs including c-myc mRNA. APE1 has recently been shown to cleave AP-site-containing single-stranded RNA (45,46), providing further support that the enzyme is involved in RNA metabolism in cells.We have previously shown that an RNA-binding protein, CRD-BP, can protect c-myc CRD RNA from degradation by the partially purified native enzyme (55). With the identification of APE1 as a responsible enzyme for the control of c-myc mRNA, we are now in a position to directly investigate if APE1 interacts with CRD-BP in controlling the turnover of c-myc mRNA and other mRNAs in cells.

We set the goal of identifying endoribonuclease(s) that are capable of cleaving c-myc CRD RNA in vitro for four specific reasons. First, endonucleolytic cleavage of c-myc mRNA has been shown to occur in vivo (6,7,25–29), implying the presence of the responsible endoribonuclease(s) in cells. Second, many vertebrate mRNA decay intermediates have been detected but only a handful of endoribonucleases that degrade specific mRNAs have been identified (1). Third, our understanding about the role of endoribonucleases in mRNA decay and gene expression in vertebrates is still unclear. Lastly, given that c-Myc is implicated in human cancers, characterization of enzymes that degrade c-myc mRNA may help develop new therapeutic approach via destruction of c-myc mRNA. In this study, we describe the purification and identification of RNase1 as a 17-kDa enzyme and APE1 as a 35-kDa endoribonuclease that are capable of cleaving c-myc CRD RNA in vitro.

RNase1 is a known endoribonuclease, belonging to the RNase A superfamily of ribonucleases (48). Therefore, it is not surprising to see its ability to cleave c-myc CRD RNA in vitro. However, whether RNase1 has a role in cleaving c-myc CRD RNA, and therefore in the regulation of c-myc mRNA level in vivo remains unknown. In addition, APE1 was identified as the 35-kDa endoribonuclease that cleaved c-myc mRNA in vitro. Four findings confirm that APE1 is the protein that contributes to the 35-kDa endoribonuclease activity. (i) Two independent sets of LC/MS-MS data identified APE1 with 32% amino acid sequence coverage and seven matched peptides (Table 1). (ii) As determined by western analysis, APE1 is present in the 30–40-kDa fractions from gel filtration which contained the endoribonuclease activity (Figures 1A and 3). (iii) The endoribonuclease activity was specifically immuno-depleted with APE1 monoclonal antibody (Figure 4). (iv) The cleavage pattern generated on c-myc CRD RNA by the recombinant human APE1 is similar to that generated by the purified native 35-kDa endoribonuclease from rat liver (Figure 5B).

Overall, the endoribonuclease activity of the purified native APE1 and the recombinant APE1 resemble RNase A-like activity, a standard contaminating ribonuclease in RNA studies. However, the following evidence strongly argue against the possibility that the observed endoribonuclease activity of APE1 is due to RNase A contaminants: (i) APE1 antibody which does not cross react with RNase A, specifically immuno-depleted the endoribonuclease activity of the purified native APE1 (Figure 4); (ii) gel filtration used in the last step to partially purify the 35-kDa native APE1 would have excluded any possible RNase A contaminants which are typically below 20 kDa in size; (iii) the cleavage pattern generated by the purified native APE1 on c-myc CRD RNA (Figure 2) and other RNAs are similar but not identical (data not shown) to that generated by RNase A; and (iv) at higher concentrations, other recombinant proteins including mutant APE1 E96A which was purified under identical conditions failed to exhibit any endoribonuclease activity (Figure 6).

APE1 has been identified as an unusual, multi-functional protein (37,38). It endonucleolytically cleaves DNA at AP sites, and is a key enzyme in the base excision repair pathway (37). APE1 also provides the major redox activity for AP-1, p53, H1F1alpha and other transcription factors (49). In addition to its AP DNA endonuclease activity, APE1 has 3′ -5′ DNA exonuclease (50) and 3′ phoshodiesterase activities (37). APE1 has also been shown to bind to a fragment of human topoisomerase IIα RNA and possesses RNase H activity (39) but the overall biological significance of this function has remained unclear. We have not examined the RNase H activity of recombinant APE1 but the previously partially purified native enzyme which contained both RNase1 and APE1 did not appear to have any RNase H activity (30). Here, we show that recombinant APE1 cleaves specific sequences on the c-myc CRD RNA, namely in between CA and UA dinucleotides (Figure 5B). To our knowledge, this is the first demonstration of a sequence specific endoribonuclease activity intrinsic to APE1. The endoribonuclease activity of APE1 on c-myc CRD RNA appears to share some of the same active site with other nuclease activities of the protein because both E96A and H309N APE1 mutants were inactive or less active in cleaving c-myc CRD RNA.

To initiate studies on examining the RNA-cleaving potential of APE1 in cells, we used the siRNA technology to transiently knockdown APE1 and then measure c-myc mRNA expression in HeLa cells. Figure 8 shows that transient knockdown of APE1 at the protein and mRNA levels lead to significant increase in steady-state c-myc mRNA. Such observation could be attributed to: (i) more stable c-myc mRNA due to less APE1 in cleaving c-myc mRNA, or/and (ii) decrease in transcriptional repressor of c-myc gene due to less APE1 in keeping some transcriptional factors active. To test the former hypothesis, we assessed the decay of c-myc mRNA upon knockdown of APE1. We found that indeed c-myc mRNA was more stable upon knockdown of APE1 supporting the notion that APE1 can cleave c-myc mRNA and control its abundance in cells. Interestingly, one of the c-myc mRNA endonucleolytic decay intermediates previously detected in cells (7) corresponded to 1727CA, which is a site cleave by APE1 (Figure 2). Our results of increased c-myc mRNA upon knockdown of APE1 is in contrast to a recent report of a modest decrease in c-myc mRNA using micro-array approach (51). This discrepancy could due to the differences in the experimental design. These investigators used stable siRNA-inducible cells that were knockdown for 10 days upon stimulation with siRNA expression and as pointed out by the investigators, compensatory mechanisms and indirect effects may set in for the observed molecular results (51).

We have observed stable association of APE1 with YB-1 and hnRNP-H both of which are involved in RNA metabolism (52). In addition to its expected localization to the nucleus, APE1 is also found in the cytoplasm and in some cases, appears to be exclusively located there (38). Such observations and the finding of redistribution of the protein between the nucleus and cytoplasm in some cancers had many researchers baffled for some time (38,49). APE1 has been reported to associate with ribosomes in motor neurons and also possibly in highly proliferative cells including hepatocytes (38), and such observations had led to the speculation that it may have an undefined role in ribosomal function (49). Indeed, a recent report provided strong evidence that the N-terminus of APE1 is physically associated with a number of proteins known to be involved in ribosome biogenesis and RNA processing (46).

Interestingly, we have purified APE1 from rat liver polysomal fraction which supports its role in RNA processing. We had earlier shown that APE1 interacts with an RNA-binding protein hnRNP-L in negative regulation of the parathyroid hormone gene (53). Of particular interest is the report that hnRNP-L can regulate the stability of human VEGF mRNA through its AU-rich element during hypoxia (54). Thus, given the evidence provided in this study and the prior evidence, it is tempting to speculate that a previously unknown function of cytoplasmic APE1 is its ability to serve as an endoribonuclease for mRNAs including c-myc mRNA. APE1 has recently been shown to cleave AP-site-containing single-stranded RNA (45,46), providing further support that the enzyme is involved in RNA metabolism in cells.

We have previously shown that an RNA-binding protein, CRD-BP, can protect c-myc CRD RNA from degradation by the partially purified native enzyme (55). With the identification of APE1 as a responsible enzyme for the control of c-myc mRNA, we are now in a position to directly investigate if APE1 interacts with CRD-BP in controlling the turnover of c-myc mRNA and other mRNAs in cells.

Conflict of interest statement. None declared.
