---
pmid: '12702766'
title: Characterization of cells and gene-targeted mice deficient for the p53-binding
  kinase homeodomain-interacting protein kinase 1 (HIPK1).
authors:
- Kondo S
- Lu Y
- Debbas M
- Lin AW
- Sarosi I
- Itie A
- Wakeham A
- Tuan J
- Saris C
- Elliott G
- Ma W
- Benchimol S
- Lowe SW
- Mak TW
- Thukral SK
journal: Proc Natl Acad Sci U S A
year: '2003'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC154362
doi: 10.1073/pnas.0530308100
---

# Characterization of cells and gene-targeted mice deficient for the p53-binding kinase homeodomain-interacting protein kinase 1 (HIPK1).
**Authors:** Kondo S, Lu Y, Debbas M, Lin AW, Sarosi I, Itie A, Wakeham A, Tuan J, Saris C, Elliott G, Ma W, Benchimol S, Lowe SW, Mak TW, Thukral SK
**Journal:** Proc Natl Acad Sci U S A (2003)
**DOI:** [10.1073/pnas.0530308100](https://doi.org/10.1073/pnas.0530308100)
**PMC:** [PMC154362](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC154362/)

## Abstract

1. Proc Natl Acad Sci U S A. 2003 Apr 29;100(9):5431-6. doi: 
10.1073/pnas.0530308100. Epub 2003 Apr 17.

Characterization of cells and gene-targeted mice deficient for the p53-binding 
kinase homeodomain-interacting protein kinase 1 (HIPK1).

Kondo S(1), Lu Y, Debbas M, Lin AW, Sarosi I, Itie A, Wakeham A, Tuan J, Saris 
C, Elliott G, Ma W, Benchimol S, Lowe SW, Mak TW, Thukral SK.

Author information:
(1)Advanced Medical Discovery Institute, and Ontario Cancer Institute and 
Department of Medical Biophysics, University of Toronto, ON, Canada M5G 2C1.

The tumor suppressor p53 is regulated in part by binding to cellular proteins. 
We used p53 as bait in the yeast two-hybrid system and isolated 
homeodomain-interacting protein kinase 1 (HIPK1) as a p53-binding protein. 
Deletion analysis showed that amino acids 100-370 of p53 and amino acids 
885-1093 of HIPK1 were sufficient for HIPK1-p53 interaction. HIPK1 was capable 
of autophosphorylation and specific serine phosphorylation of p53. The HIPK1 
gene was highly expressed in human breast cancer cell lines and oncogenically 
transformed mouse embryonic fibroblasts. HIPK1 was localized to human chromosome 
band 1p13, a site frequently altered in cancers. Gene-targeted HIPK1-/- mice 
were grossly normal but oncogenically transformed HIPK1 -/- mouse embryonic 
fibroblasts exhibited reduced transcription of Mdm2 and were more susceptible 
than transformed HIPK1+/+ cells to apoptosis induced by DNA damage. 
Carcinogen-treated HIPK1 -/- mice developed fewer and smaller skin tumors than 
HIPK1+/+ mice. HIPK1 may thus play a role in tumorigenesis, perhaps by means of 
the regulation of p53 and/or Mdm2.

DOI: 10.1073/pnas.0530308100
PMCID: PMC154362
PMID: 12702766 [Indexed for MEDLINE]

## Full Text

Abstract

The tumor suppressor p53 is regulated in part by binding to cellular proteins. We used p53 as bait in the yeast two-hybrid system and isolated homeodomain-interacting protein kinase 1 (HIPK1) as a p53-binding protein. Deletion analysis showed that amino acids 100–370 of p53 and amino acids 885-1093 of HIPK1 were sufficient for HIPK1–p53 interaction. HIPK1 was capable of autophosphorylation and specific serine phosphorylation of p53. The HIPK1 gene was highly expressed in human breast cancer cell lines and oncogenically transformed mouse embryonic fibroblasts. HIPK1 was localized to human chromosome band 1p13, a site frequently altered in cancers. Gene-targeted HIPK1 −/− mice were grossly normal but oncogenically transformed HIPK1 −/− mouse embryonic fibroblasts exhibited reduced transcription of Mdm2 and were more susceptible than transformed HIPK1 +/+ cells to apoptosis induced by DNA damage. Carcinogen-treated HIPK1 −/− mice developed fewer and smaller skin tumors than HIPK1 +/+ mice. HIPK1 may thus play a role in tumorigenesis, perhaps by means of the regulation of p53 and/or Mdm2.

Colony Transformation, Apoptosis, and Reintroduction of HIPK1.

For colony transformation assays, cells were suspended in 0.4% low-melting agarose in DMEM containing 10% FCS and plated on solidified 0.53% agarose at 1 × 10 3 cells per well on six-well plates. Colony numbers were scored after 10 days. For apoptosis assays, transformed MEFs (1 × 10 5 per well) were plated in six-well plates and treated for 24 h with doxorubicin (0.2 μg/ml), etoposide (1 μM), or γ-irradiation (20 Gy). Cell viability was determined by negative staining for the vital dye 7-aminoactinomycin D followed by flow cytometry. For the reintroduction of HIPK1 into HIPK1 −/−-transformed MEFs, cells were electroporated with pcDNA3.1/Zeo vector containing HIPK1 cDNA or the control hemagglutinin (HA) sequence and selected in medium containing 0.3 mg/ml zeocin (Invitrogen).

Results

When human WT p53 (amino acids 71–393) was used as bait in a yeast two-hybrid assay ( 13 ) to screen a transformed rat-embryonic-fibroblast cDNA library, a previously undescribed kinase, HIPK1, was identified. Human 293 cells expressing endogenous p53 were then transfected with a plasmid expressing a Myc-tagged HIPK1 fusion protein expressing amino  acids 678-1149 of human HIPK1, or with empty Myc-vector. In immunoprecipitation experiments, a distinct protein band at 65 kDa was observed only in extracts of cells transfected with the Myc-HIPK1 vector (Fig. 1 A ), indicating that HIPK1 can form a stable intracellular complex with p53. Sequence analysis of the full-length human HIPK1 cDNA showed that the HIPK1 protein contains a putative serine-threonine kinase domain ( 17 ), and in situ hybridization revealed that HIPK1 localizes to human chromosome band 1p13, a region frequently altered in human breast cancers (ref. 18 ; data not shown). Deletion analyses using yeast two-hybrid assays showed that a 271-aa region of p53 spanning amino acids 100–370 was sufficient for association with HIPK1 (Fig. 1 B ). This region overlaps the p53 domain known to be involved in binding to specific DNA sequences but does not include the p53-transactivation domain or the nonspecific DNA-interaction domain ( 12 ). A region of HIPK1 spanning amino acids 885-1093 (which does not include the kinase domain) was found to be necessary for binding to p53 (Fig. 1 C ).

High-stringency Northern blot analysis revealed that the ≈9 kb of HIPK1 mRNA was expressed in heart, brain, placenta, skeletal muscle, and pancreas (Fig. 2 A ). Interestingly, in 12 of 14 human breast cancer cell lines tested, HIPK1 mRNA was elevated, compared with levels in normal mammary epithelial cells and a nontumorigenic cell line (Fig. 2 B ). In two of these cell lines (MCF7 and BT20-T), HIPK1 was expressed at >10 times control levels, suggesting that HIPK1 might be an oncogene.

We next investigated whether HIPK1 phosphorylated p53. HIPK1 protein immunoprecipitated from lysates of COS cells in which murine HIPK1 was transiently expressed were subjected to in vitro kinase assays using p53 as substrate. The results showed that, in vitro , HIPK1 was able to phosphorylate itself, but not p53 (Fig. 3 A ). Because the phosphorylation of p53 by HIPK1 might require a cellular context, we looked at the phosphorylation status of p53 in transformed MEFs generated from either HIPK1 +/+ or HIPK1 −/− mice (see below). After labeling of MEFs with 32 P in vivo , lysates were immunoprecipitated with anti-p53 antibody and the electrophoresed p53 band was isolated and subjected to tryptic phosphopeptide analysis. The presence of HIPK1 led to the appearance of a unique phosphopeptide spot that was absent in cells lacking HIPK1 (Fig. 3 B ). Phosphoamino acid analysis of this spot revealed phospho-serine exclusively (Fig. 3 B Inset ). The relative mobility of the HIPK1-induced phosphopeptide was consistent with its being the multiply phosphorylated N-terminal 27-aa tryptic peptide of p53 identified in previous reports ( 19 , 20 ). Thus, HIPK1 can induce serine phosphorylation of p53 in vivo .

To determine the physiological role of HIPK1, HIPK1-deficient mice were created by conventional gene targeting (see Methods ). The genotypes of HIPK1 +/− and HIPK1 −/− mice were confirmed by Southern analysis and deletion of HIPK1 was confirmed by Northern analysis of RNA from primary MEFs and ES cells (data not shown). HIPK1 −/− mice were born at the expected Mendelian frequency, displayed no gross abnormalities of growth or development, and were fertile. No spontaneous tumors were observed in HIPK1 −/− mice over a 1-yr period.

To examine the role of HIPK1 in oncogenesis, E1A and H- Ras oncogenes (E1A/Ras) were introduced into HIPK1 +/+ and HIPK1 −/− primary MEFs by retroviral gene transfer ( 21 ). Cellular transformation was achieved in both cases, as determined by tumor formation in nude mice (data not shown). Expression of HIPK1 was increased in four independent clones of E1A/Ras-transformed HIPK1 +/+ MEFs compared with nontransformed WT MEFs (Fig. 4 A ). In colony-formation assays in semisolid medium, HIPK1 −/−-transformed MEFs formed 2-fold fewer colonies than HIPK1 +/+-transformed MEFs, and 20-fold fewer than p53 −/−-transformed MEFs (Fig. 4 B and data not shown). Thus, the presence of HIPK1 enhances malignant transformation by E1A/Ras.

In primary HIPK1 −/− MEFs and thymocytes, DNA damage triggered normal p53-dependent cell-cycle arrest and apoptosis (data not shown). However, when HIPK1 −/−-transformed MEFs were treated with high doses of γ-irradiation (20 Gy; Fig. 4 C ), doxorubicin (0.2 μg/ml), or etoposide (1 μM) (data not shown), apoptosis was increased. These results imply that HIPK1 is involved in regulating apoptosis caused by DNA damage in oncogenically transformed cells.

To investigate whether HIPK1 modulates the transcriptional activity of p53, we carried out Northern blot analyses of the expression of the p53-inducible genes Mdm-2 , p21 , and thrombospondin 1 ( TSP1 ) in oncogenically transformed MEFs. As controls, we used two lines of E1A/Ras-transformed MEFs expressing a spontaneous mutation of p53 that were established in our laboratory; these cells are functionally null for p53. As expected, basal levels of Mdm2 , p21 , and TSP-1 transcripts all were decreased in transformed MEFs expressing mutant p53 compared with transformed MEFs expressing WT p53 (Fig. 5 A Upper ). However, in oncogenically transformed HIPK1 −/− MEFs expressing WT p53, basal levels of both Mdm2 transcripts (Fig. 5 A Lower ) and protein (Fig. 5 B ) were decreased, whereas basal expression of p21 and TSP1 transcripts was unaffected (Fig. 5 A Lower ). Luciferase reporter assays (see Methods ) showed that the Mdm2 promoter was less active (42.2% of WT value) in HIPK1 −/−-transformed MEFs than in HIPK1 +/+-transformed MEFs, whereas the p21 promoter was transactivated to a similar degree in cells of both genotypes (Fig. 5 C ). Furthermore, when full-length mouse HIPK1 cDNA was transiently transfected into HIPK1 −/−-transformed MEFs and luciferase activity was determined, a 4- to 6-fold increase in Mdm2 promoter activation was observed (data not shown). Thus, expression of the p53 target gene Mdm2 is influenced at the transcriptional stage by HIPK1.

To confirm that loss of HIPK1 was responsible for the phenotype of transformed HIPK1 −/− MEFs, we generated stably transfected “reintroduced HIPK1 ” (re- HIPK1 ) cell lines by using the full-length mouse HIPK1 cDNA and Zeocin selection (see Methods ). The reintroduction of HIPK1 into HIPK1 −/−-transformed MEFs increased the expression of Mdm2 transcripts over that in controls (data not shown) and restored resistance to doxorubicin and γ-irradiation (Fig. 6 ).

To examine the role of HIPK1 in oncogenesis in vivo , mice were treated with repeated applications of the skin carcinogen DMBA ( 22 , 23 ). After 14 weeks, fewer (Fig. 7 ) and smaller ( Inset ) tumors had developed on the skin of treated HIPK1 −/− mice than on that of heterozygous or WT littermates. Mean tumor numbers were as follows: 1.3 tumors per HIPK1 −/− mouse ( n = 13); 4.0 tumors per +/− mouse ( n = 11); and 3.6 tumors per +/+ mouse ( n = 8). In a second experiment of 17 weeks, mean tumor numbers were as follows: 1.6 tumors per −/− mouse ( n = 10) and 3.0 tumors per +/− mouse ( n = 11). No malignant tumors were found in 23 HIPK1 −/− mice examined, although they did arise in WT and heterozygous mice (2/8 and 3/22, respectively) by 15 weeks. The vast majority of skin tumors were identified as papillomas, with the remainder diagnosed histologically as keratoacanthomas or squamous cell carcinomas (Table 1 ). These data indicate that HIPK1 is involved in (at least) malignant squamous cell tumor formation in vivo , and suggest that, like Mdm2, HIPK1 can promote oncogenesis.

Discussion

How might HIPK1 modulate p53 function? In our in vivo assays, p53 underwent specific serine phosphorylation only when HIPK1 was present. Serine phosphorylation at specific residues has been shown to govern p53's ability to transactivate particular target genes ( 24 , 25 ). HIPK1 may itself be phosphorylating p53, or HIPK1 binding to p53 may facilitate serine phosphorylation of p53 by another protein kinase. Another possibility is that the conformation of p53, and thus, its activity might be altered by the binding of HIPK1. Interaction of p53 with c-Abl (independent of its kinase activity) has been shown to induce a conformational change in p53 such that p53 tetramers are stabilized and DNA-binding activity is enhanced ( 26 ).

Kim et al. ( 11 ) have suggested that mouse HIPK1 may act as a transcriptional cofactor (repressor) interacting with the homeoproteins NKx-1.2 and NK-1. We have shown that human HIPK1 interacts with p53 through a domain that does not overlap with that which is involved in homeoprotein interaction. Several other proteins can interact with HIPK1 through its p53-binding domain (Y.L. and M.D., unpublished data), including Hic5 ( 27 ), UBC9 ( 28 ), and zyxin ( 29 ). Interactions with Hic5 and zyxin are particularly interesting because these proteins are recruited into focal adhesions and provide a link with the extracellular matrix (focal adhesion complex; refs. 29 and 30 ). Interestingly, HIPK1 is expressed at high levels in oncogenically transformed rat embryonic fibroblasts and MCF7 cells, whereas Hic5 levels are undetectable. Subsequent analyses of a large number of tumor and normal cell lines have shown that expression patterns of HIPK1 and Hic5 are negatively correlated (Y.L. and M.D., unpublished data). It is possible that, in untransformed cells, HIPK1 is retained in the cytoplasm through its interactions with focal adhesion proteins such as Hic5. Under conditions that promote transformation and reduce Hic5 levels, HIPK1 might be freed to transmigrate to the nucleus, interact with nuclear p53, and modulate p53-dependent function such that oncogenesis is enhanced.

That HIPK1 may be an oncogene was supported by our examination of HIPK1-deficient mice. Although HIPK1 −/− mice were phenotypically normal, they were more resistant to the development of DMBA-induced skin tumors (particularly benign papillomas) than WT littermates. The loss of HIPK1 also had a protective effect with respect to malignant progression. Similarly, in colony-formation assays, HIPK1 −/−-transformed MEF colonies grew more slowly than the equivalent HIPK1 +/+ cells. Finally, HIPK1 was up-regulated in tumor cell lines. We speculate that these differences in malignant progression may result from differences in downstream events mediated by p53. Such differences and their relation to HIPK1 warrant future study.
