---
pmid: '17085477'
title: ZNF366 is an estrogen receptor corepressor that acts through CtBP and histone
  deacetylases.
authors:
- Lopez-Garcia J
- Periyasamy M
- Thomas RS
- Christian M
- Leao M
- Jat P
- Kindle KB
- Heery DM
- Parker MG
- Buluwela L
- Kamalati T
- Ali S
journal: Nucleic Acids Res
year: '2006'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC1693901
doi: 10.1093/nar/gkl875
---

# ZNF366 is an estrogen receptor corepressor that acts through CtBP and histone deacetylases.
**Authors:** Lopez-Garcia J, Periyasamy M, Thomas RS, Christian M, Leao M, Jat P, Kindle KB, Heery DM, Parker MG, Buluwela L, Kamalati T, Ali S
**Journal:** Nucleic Acids Res (2006)
**DOI:** [10.1093/nar/gkl875](https://doi.org/10.1093/nar/gkl875)
**PMC:** [PMC1693901](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1693901/)

## Abstract

1. Nucleic Acids Res. 2006;34(21):6126-36. doi: 10.1093/nar/gkl875. Epub 2006 Nov
 3.

ZNF366 is an estrogen receptor corepressor that acts through CtBP and histone 
deacetylases.

Lopez-Garcia J(1), Periyasamy M, Thomas RS, Christian M, Leao M, Jat P, Kindle 
KB, Heery DM, Parker MG, Buluwela L, Kamalati T, Ali S.

Author information:
(1)Department of Oncology, Imperial College London, Du Cane Road, London W12 
0NN, UK.

The regulation of gene expression by estrogen receptor-alpha (ERalpha) requires 
the coordinated and temporal recruitment of diverse sets of transcriptional 
co-regulator complexes, which mediate nucleosome remodelling and histone 
modification. Using ERalpha as bait in a yeast two-hybrid screen, we have 
identified a novel ERalpha-interacting protein, ZNF366, which is a potent 
corepressor of ERalpha activity. The interaction between ZNF366 and ERalpha has 
been confirmed in vitro and in vivo, and is mediated by the zinc finger domains 
of the two proteins. Further, we show that ZNF366 acts as a corepressor by 
interacting with other known ERalpha corepressors, namely RIP140 and CtBP, to 
inhibit expression of estrogen-responsive genes in vivo. Together, our results 
indicate that ZNF366 may play an important role in regulating the expression of 
genes in response to estrogen.

DOI: 10.1093/nar/gkl875
PMCID: PMC1693901
PMID: 17085477 [Indexed for MEDLINE]

## Full Text

INTRODUCTIONEstrogens play diverse roles in the body, most notably in the development and maintenance of the female and the male, reproductive systems and secondary sexual characteristics (1). Estrogens also play a central role in promoting breast cancer growth (2), as well as being implicated in uterine and ovarian cancers (3,4), and are also implicated in the physiology of the brain, bone and the cardiovascular system, as evidenced by the increased risk of cardiovascular disease and osteoporosis engendered by the decline in estrogen levels during menopause (1).Estrogen action is mediated by two highly related estrogen receptors (ERα and ERβ), which are members of the ligand-activated nuclear receptor (NR) superfamily of transcription factors (5,6). NRs are characterized by a DNA binding domain (DBD), comprised of two zinc fingers, which mediate receptor dimerization and binding to specific response elements in the promoters of target genes. Binding of the ligand to the ligand binding domain (LBD), located C-terminal to the DBD, results in a conformational change in the LBD and activation of the intrinsic transcription activation function AF2, which facilitates the recruitment of transcriptional coactivators (7,8). Transcription activation requires cooperation of AF2 with a region N-terminal to the DBD that encodes transcription activation function AF1, which is often, as in the case of ERα and ERβ, regulated by phosphorylation at specific serine residues (9–11).The liganded estrogen receptors regulate gene expression by direct binding to DNA at estrogen response elements in target genes, resulting in the recruitment of diverse transcriptional coregulators, including the SWI–SNF complexes that remodel chromatin to alter nucleosomal organization in an ATP-dependent manner (12), the p160 family of coactivators (SRC1/NCo-A1, TIF-2/GRIP1 and AIB1/pCIP/ACTR/RAC3/TRAM1) and TRAP/DRIP complexes. (13–15). The p160 coactivators facilitate the recruitment of other proteins, including CBP, its homologue p300 and p/CAF, which possess intrinsic histone acetyltransferase activity, as well as histone methyltransferases CARM1 and PRMT1 that methylate arginine residues in histone tails (16,17). The thyroid receptor–associated protein (TRAP) complex, similar to or identical with the vitamin D3 receptor–interacting protein (DRIP) complex, which is also similar in many respects to the Mediator complex acts to bridge RNA polymerase II with basal transcription factors and transcription activators. These and other coregulators are recruited to gene promoters in a sequential/ordered manner, resulting in cycles of chromatin remodelling and modification that facilitate transcription (13,15,18).The ERα LBD is composed of 12 α-helices packed in three layers, with a central hydrophobic pocket that accommodates the ligand (7). Helix 12, together with helices 3, 4 and 5, form a coactivator-binding groove. Most coactivators recruited by agonist-bound ERα contain LXXLL motifs, which form a two-turn amphipathic α-helix that fits into the coactivator-binding groove in the ERα LBD (8,19–22). In addition to the well-characterized recruitment of coactivators to the LBD/AF2, coactivator interaction with AF1 of estrogen receptors has also been described, the interaction being influenced by the phosphorylation status of the receptor in some cases (23–25).When unliganded, some DNA-bound NR recruit the corepressors NCoR and SMRT and associated protein complexes implicated in transcriptional repression and histone deacetylation, these complexes being dissociated upon ligand binding (13–15,26). NCoR/SMRT bind to NR through CoRNR boxes, LXXI/HIXXXL/I motifs, that form a more extended α-helix than the LXXLL motifs, with helix 12 in the LBD being displaced from the conformation it occupies in the agonist-bound LBD (27,28). NCoR/SMRT is also recruited by antagonist-bound ERα to inhibit gene expression (29–31).In addition to stimulating gene expression, estrogen-bound ER represses the expression of many genes. Indeed, gene profiling studies show that the down-regulation of gene expression is a significant feature of the response to estrogen in the ERα-positive, estrogen-responsive MCF7 breast cancer cell line (32). This is likely to involve transcriptional corepressors, such as LCoR and RIP140, which can be recruited to the agonist-bound ERα via LXXLL motifs (20,33,34). Repression by LCoR and RIP140 occurs through HDAC-dependent and -independent mechanisms and involves the recruitment of HDACs and the C-terminal binding protein (CtBP) corepressor (34–39). CtBP, originally identified based on its interaction with the C-terminal end of adenovirus E1A via the sequence PLDLS, is highly conserved in higher eukaryotes and plays a critical role in development (40,41). Other transcription factors also interact with CtBP1, and the highly related CtBP2, through PXDLS motifs. Although, the mechanisms by which CtBP acts as a corepressor have not been fully defined, a recent study has identified CtBP complexes that contain HDACs and histone lysine methyltransferases (42).The ERα DBD participates in the recruitment of transcriptional co-regulator proteins. These include the coactivator XBP-1 (43), which modulates ERα signalling both in the absence and presence of estrogen, the signal transducer and activator of transcription-5 (STAT 5) (44) and the corepressor TAF-Ib that has been shown to decrease ERα acetylation (45). Here, we report that ZNF366, which encodes an evolutionarily conserved zinc finger protein, interacts with the ERα DBD. We also show that ZNF366 represses ERα activity through association with RIP140, CtBP and histone deacetylases.

INTRODUCTION

Estrogens play diverse roles in the body, most notably in the development and maintenance of the female and the male, reproductive systems and secondary sexual characteristics (1). Estrogens also play a central role in promoting breast cancer growth (2), as well as being implicated in uterine and ovarian cancers (3,4), and are also implicated in the physiology of the brain, bone and the cardiovascular system, as evidenced by the increased risk of cardiovascular disease and osteoporosis engendered by the decline in estrogen levels during menopause (1).

Estrogen action is mediated by two highly related estrogen receptors (ERα and ERβ), which are members of the ligand-activated nuclear receptor (NR) superfamily of transcription factors (5,6). NRs are characterized by a DNA binding domain (DBD), comprised of two zinc fingers, which mediate receptor dimerization and binding to specific response elements in the promoters of target genes. Binding of the ligand to the ligand binding domain (LBD), located C-terminal to the DBD, results in a conformational change in the LBD and activation of the intrinsic transcription activation function AF2, which facilitates the recruitment of transcriptional coactivators (7,8). Transcription activation requires cooperation of AF2 with a region N-terminal to the DBD that encodes transcription activation function AF1, which is often, as in the case of ERα and ERβ, regulated by phosphorylation at specific serine residues (9–11).

The liganded estrogen receptors regulate gene expression by direct binding to DNA at estrogen response elements in target genes, resulting in the recruitment of diverse transcriptional coregulators, including the SWI–SNF complexes that remodel chromatin to alter nucleosomal organization in an ATP-dependent manner (12), the p160 family of coactivators (SRC1/NCo-A1, TIF-2/GRIP1 and AIB1/pCIP/ACTR/RAC3/TRAM1) and TRAP/DRIP complexes. (13–15). The p160 coactivators facilitate the recruitment of other proteins, including CBP, its homologue p300 and p/CAF, which possess intrinsic histone acetyltransferase activity, as well as histone methyltransferases CARM1 and PRMT1 that methylate arginine residues in histone tails (16,17). The thyroid receptor–associated protein (TRAP) complex, similar to or identical with the vitamin D3 receptor–interacting protein (DRIP) complex, which is also similar in many respects to the Mediator complex acts to bridge RNA polymerase II with basal transcription factors and transcription activators. These and other coregulators are recruited to gene promoters in a sequential/ordered manner, resulting in cycles of chromatin remodelling and modification that facilitate transcription (13,15,18).

The ERα LBD is composed of 12 α-helices packed in three layers, with a central hydrophobic pocket that accommodates the ligand (7). Helix 12, together with helices 3, 4 and 5, form a coactivator-binding groove. Most coactivators recruited by agonist-bound ERα contain LXXLL motifs, which form a two-turn amphipathic α-helix that fits into the coactivator-binding groove in the ERα LBD (8,19–22). In addition to the well-characterized recruitment of coactivators to the LBD/AF2, coactivator interaction with AF1 of estrogen receptors has also been described, the interaction being influenced by the phosphorylation status of the receptor in some cases (23–25).

When unliganded, some DNA-bound NR recruit the corepressors NCoR and SMRT and associated protein complexes implicated in transcriptional repression and histone deacetylation, these complexes being dissociated upon ligand binding (13–15,26). NCoR/SMRT bind to NR through CoRNR boxes, LXXI/HIXXXL/I motifs, that form a more extended α-helix than the LXXLL motifs, with helix 12 in the LBD being displaced from the conformation it occupies in the agonist-bound LBD (27,28). NCoR/SMRT is also recruited by antagonist-bound ERα to inhibit gene expression (29–31).

In addition to stimulating gene expression, estrogen-bound ER represses the expression of many genes. Indeed, gene profiling studies show that the down-regulation of gene expression is a significant feature of the response to estrogen in the ERα-positive, estrogen-responsive MCF7 breast cancer cell line (32). This is likely to involve transcriptional corepressors, such as LCoR and RIP140, which can be recruited to the agonist-bound ERα via LXXLL motifs (20,33,34). Repression by LCoR and RIP140 occurs through HDAC-dependent and -independent mechanisms and involves the recruitment of HDACs and the C-terminal binding protein (CtBP) corepressor (34–39). CtBP, originally identified based on its interaction with the C-terminal end of adenovirus E1A via the sequence PLDLS, is highly conserved in higher eukaryotes and plays a critical role in development (40,41). Other transcription factors also interact with CtBP1, and the highly related CtBP2, through PXDLS motifs. Although, the mechanisms by which CtBP acts as a corepressor have not been fully defined, a recent study has identified CtBP complexes that contain HDACs and histone lysine methyltransferases (42).

The ERα DBD participates in the recruitment of transcriptional co-regulator proteins. These include the coactivator XBP-1 (43), which modulates ERα signalling both in the absence and presence of estrogen, the signal transducer and activator of transcription-5 (STAT 5) (44) and the corepressor TAF-Ib that has been shown to decrease ERα acetylation (45). Here, we report that ZNF366, which encodes an evolutionarily conserved zinc finger protein, interacts with the ERα DBD. We also show that ZNF366 represses ERα activity through association with RIP140, CtBP and histone deacetylases.

MATERIALS AND METHODSPlasmidsThe mammalian expression plasmids and reporter genes have previously been described (37,46–49). Site-directed mutagenesis was used to introduce an EcoRI site 5′ to the GAL4 translation initiation site in the pBridge yeast expression plasmid (BD Biosciences, UK), enabling cDNA sequences encoding ERα and ERα−ΔLBD to be cloned at this position following removal of the GAL4 sequences encoded between the introduced EcoRI site and the multiple cloning site in pBridge. The pACTII-ZNF366 clone isolated from the yeast 2-hybrid screening encodes sequences corresponding to 439–2761 bp of the ZNF366 mRNA sequence with the accession number NM_152625 in the NCBI database (). The full-length ZNF366 open reading frame was reconstituted from the pACTII-ZNF366 clone and IMAGE EST clone 5204702 (accession no. BI770486), to generate pCMVSPORT6-ZNF366 in which ZNF366 is C-terminally FLAG-tagged. ZNF366 deletion and point mutants were generated by site-directed mutagenesis according to manufacturer's protocols (Stratagene, UK).Yeast 2-hybrid screeningPL1α (MATα ura3-Δ1 his3-Δ200 leu2-Δ1 trp1::ERE)1-URA3 yeast strain (50) was transformed with pBridge(Mod)-ERα-ΔLBD, together with a human placental cDNA expression library (BD Biosciences, UK), using the Alkali-Cation yeast transformation kit (BIO 101 systems, UK). Following transformation, the cells were plated on 15 cm trp− leu− ura− plates. Positive clones arising from the screening of 2 × 104 transformants were re-screened and plasmid DNAs were isolated using the lyticase method from BD Biosciences, UK. Plasmids from positive clones were re-transformed, together with pBridge(Mod)-ERα-ΔLBD or pBridge(Mod)-ERα and interactions confirmed by growth on trp− leu− ura− plates.Northern blottingMultiple tissue northern blots MTN I and MTN II (BD Biosciences, Europe) were probed following 32P-labelling of the ZNF366 cDNA isolated from the pACTII-ZNF366 clone, as described (51).Protein expression, purification and glutathione S-transferase (GST)-based interaction assayIn vitro transcription/translations were performed using TNT rabbit reticulocyte lysates (Promega, UK), in the presence of [35S]-labelled methionine. GST proteins were induced and Escherichia coli lysates prepared as described previously (33). For pulldowns, GST fusion proteins were purified by affinity chromatography on glutathione-agarose beads and retained as 50% slurry in 20 mM HEPES (pH 7.6), 100 mM KCl, 1 mM EDTA, 1 mM DTT, 20% glycerol, supplemented with protease inhibitors. A total of 100 μl volumes of glutathione-agarose bead slurry loaded with 10 μg of GST fusion proteins were then used directly in binding assays with 10 μl of radiolabelled in vitro translation reactions and 890 μl of low salt buffer [50 mM HEPES (pH 7.6), 250 mM NaCl, 0.5% NP-40, 5 mM EDTA, 0.1% BSA, 0.5 mM DTT, 0.005% SDS and protease inhibitors]. Following 1 h incubation at room temperature, the beads were washed twice with low salt buffer and twice with high-salt buffer (low salt buffer, but with 1 M NaCl). Samples were boiled for 10 min in 80 μl of Laemmli buffer and fractionated by SDS–PAGE. Gels were dried and autoradiographed.Reporter gene assaysCOS-1 cells were maintained in DMEM, supplemented with 5% fetal calf serum (FCS). For transient transfection, cells were seeded in 24-well plates in DMEM lacking phenol red and supplemented with 5% dextran-coated charcoal-stripped FCS (DSS). Following seeding for 24 h, the cells were transfected using Fugene 6 (Roche Diagnostics, UK), with 100 ng of luciferase reporter gene and amounts of expression plasmids as indicated in the figure legends. E2 (10 nM), 4-hydroxytamoxifen (OHT; 100 nM) or ICI 182, 780 (ICI; 100 nM) were added as appropriate. Since the ligands were prepared in ethanol, an equal volume of ethanol was added to the no ligand controls. Luciferase activities were determined using the Dual-Glo Luciferase Assay kit (Promega, UK). For the other reporter gene assays, cells were maintained in DMEM, supplemented with 5% FCS and transfections carried out as above.Immunoprecipitations and immunoblottingCOS-1 cells were plated in 9 cm dishes in DMEM supplemented with 5% FCS 16 to 24 h prior to transfection. The cells were transfected with 5 μg of the ZNF366-FLAG and ERα expression plasmids using Lipofectamine 2000 (Invitrogen, UK). Following transfection for 48 h, the cells were lysed in RIPA buffer [150 mM NaCl, 1% NP-40, 0.5% deoxycholic acid, 0.1% SDS and 50 mM Tris–HCl (pH 7.5)] containing protease inhibitors. Lysates (2 mg) were immunoprecipitated (IP) using the M2 anti-FLAG mouse monoclonal antibody (Sigma–Aldrich, UK), or using an anti-ERα antibody (6F11; Novocastra, UK). Control IPs was carried out using mouse IgG (Sigma–Aldrich, UK). IPs were resolved by SDS–PAGE and immunoblotted using horseradish peroxidase (HRP)-labelled HA antibody (Sigma–Aldrich, UK) or using an anti-ERα rabbit polyclonal antibody HC20 (Santa Cruz, UK). Co-IP of ZNF366-FLAG with CtBP was carried out as above, except that a mouse monoclonal CtBP antibody (sc-17759; Santa Cruz) was used for the IPs and a rabbit polyclonal CtBP antibody (sc-11390; Santa Cruz) was used for immunoblotting.MCF7 cells cultured for 3 days in DMEM lacking phenol red and supplemented with 5% DSS, were transfected with 1 μg of ZNF366-FLAG or vector control, using Fugene 6. The media were replaced with media containing E2 (10 nM) or vehicle, 24 h following transfection and the cells were harvested after a further 24 h. Immunoblotting was performed using antibodies for cathepsin D (ab6313; Abcam, UK), progesterone receptor (SC538; Santa Cruz Biotechnologies, UK), FLAG-M2 and β-actin (ab6276; Abcam, UK).ImmunofluorescenceCOS-1 cells plated on glass coverslips placed in 24-well plates in DMEM lacking phenol red and containing 5% DSS, were transiently transfected with 50 ng of ZNF366-FLAG and/or [ERα-ΔNLS (HE257G; (48)] using Fugene 6. Five hours following transfection, culture media were replaced by fresh media containing E2 (100 nM), OHT (1 μM) or ICI 182, 780 (100 nM), or an equal volume of vehicle (ethanol), as appropriate. 24 h later, cells were fixed by the addition of 4% formaldehyde for 10 min at room temperature, washed with phosphate-buffered saline (PBS) and 0.1 M glycine was added for 10 min to neutralize the formaldehyde. Following further washing with PBS, the cells were permeabilized in 1% Triton/PBS for 5 min. After washing with PBS, the cells were incubated at 37°C for 1 h with the 6F11 ERα antibody (1:50 dilution) and rabbit polyclonal FLAG antiserum (Santa Cruz Biotechnology, UK) (1:350 dilution). The cells were washed and incubated for 1 h at 37°C with Alexa Fluor 488 goat anti-mouse immunoglobulins (green) and Alexa Fluor 594 goat anti-rabbit immunoglobulins (red) (1:3000 dilution). The coverslips were mounted on microscope slides using mountant containing Dapi (Vector Laboratories, UK) and immunofluorescence observed using a Zeiss LSM510 confocal microscope.Growth assaysMCF7 and MDA-MB-231 cells (2 × 105 cells per well) were seeded in 6-well plates in DMEM lacking phenol red and containing 5% DSS. After 48 h the cells were transfected with 1 μg of ZNF366 or empty vector using Fugene 6. E2 (10 nM) was added after 24 h and cell numbers determined using a haemocytometer after a further 48 h.RNA interference and RT–PCR analysis of gene expressionPE04 cells (2.5 × 105 cells) seeded in 6-well plates in RPMI lacking phenol red and containing 5% DSS, were transfected with double-stranded RNA oligonucleotides for ZNF366, lamin A/C or a non-targeting siRNA (Ambion, UK), in serum-free DMEM lacking phenol red, using Oligofectamine (Invitrogen, UK), according to manufacturer's protocols. After 4 h the medium was changed to DMEM lacking phenol red, supplemented with 5% DSS and containing E2 (10 nM), as appropriate. RNA was prepared after a further 24 h and RT–PCR carried out using primers with the sequences: 5′-GGACCAGCTTCAGTCACCTTTCCAGTGGTGGCC-3′ and 5′-GGGAGTAAAGCTGGTGCCTGGGGCACAGGTCACG-3′ (GREB1), 5′-CCCCATCCAGTACAACTGCT-3′ and 5′-CTTCACGTCAGAGTGGACGA-3′ (ZNF366), 5′-GCGTACGGCTCTCATCAACT-3′ and 5′-GACACTGGAGGCAGAAGAGC-3′ (Lamin A/C) and 5′-TCCCATCACCATCTTCCA-3′ and 5′-CATCACGCCACAGTTTCC-3′ (GAPDH).

MATERIALS AND METHODS

PlasmidsThe mammalian expression plasmids and reporter genes have previously been described (37,46–49). Site-directed mutagenesis was used to introduce an EcoRI site 5′ to the GAL4 translation initiation site in the pBridge yeast expression plasmid (BD Biosciences, UK), enabling cDNA sequences encoding ERα and ERα−ΔLBD to be cloned at this position following removal of the GAL4 sequences encoded between the introduced EcoRI site and the multiple cloning site in pBridge. The pACTII-ZNF366 clone isolated from the yeast 2-hybrid screening encodes sequences corresponding to 439–2761 bp of the ZNF366 mRNA sequence with the accession number NM_152625 in the NCBI database (). The full-length ZNF366 open reading frame was reconstituted from the pACTII-ZNF366 clone and IMAGE EST clone 5204702 (accession no. BI770486), to generate pCMVSPORT6-ZNF366 in which ZNF366 is C-terminally FLAG-tagged. ZNF366 deletion and point mutants were generated by site-directed mutagenesis according to manufacturer's protocols (Stratagene, UK).

The mammalian expression plasmids and reporter genes have previously been described (37,46–49). Site-directed mutagenesis was used to introduce an EcoRI site 5′ to the GAL4 translation initiation site in the pBridge yeast expression plasmid (BD Biosciences, UK), enabling cDNA sequences encoding ERα and ERα−ΔLBD to be cloned at this position following removal of the GAL4 sequences encoded between the introduced EcoRI site and the multiple cloning site in pBridge. The pACTII-ZNF366 clone isolated from the yeast 2-hybrid screening encodes sequences corresponding to 439–2761 bp of the ZNF366 mRNA sequence with the accession number NM_152625 in the NCBI database (). The full-length ZNF366 open reading frame was reconstituted from the pACTII-ZNF366 clone and IMAGE EST clone 5204702 (accession no. BI770486), to generate pCMVSPORT6-ZNF366 in which ZNF366 is C-terminally FLAG-tagged. ZNF366 deletion and point mutants were generated by site-directed mutagenesis according to manufacturer's protocols (Stratagene, UK).

Yeast 2-hybrid screeningPL1α (MATα ura3-Δ1 his3-Δ200 leu2-Δ1 trp1::ERE)1-URA3 yeast strain (50) was transformed with pBridge(Mod)-ERα-ΔLBD, together with a human placental cDNA expression library (BD Biosciences, UK), using the Alkali-Cation yeast transformation kit (BIO 101 systems, UK). Following transformation, the cells were plated on 15 cm trp− leu− ura− plates. Positive clones arising from the screening of 2 × 104 transformants were re-screened and plasmid DNAs were isolated using the lyticase method from BD Biosciences, UK. Plasmids from positive clones were re-transformed, together with pBridge(Mod)-ERα-ΔLBD or pBridge(Mod)-ERα and interactions confirmed by growth on trp− leu− ura− plates.

Yeast 2-hybrid screening

PL1α (MATα ura3-Δ1 his3-Δ200 leu2-Δ1 trp1::ERE)1-URA3 yeast strain (50) was transformed with pBridge(Mod)-ERα-ΔLBD, together with a human placental cDNA expression library (BD Biosciences, UK), using the Alkali-Cation yeast transformation kit (BIO 101 systems, UK). Following transformation, the cells were plated on 15 cm trp− leu− ura− plates. Positive clones arising from the screening of 2 × 104 transformants were re-screened and plasmid DNAs were isolated using the lyticase method from BD Biosciences, UK. Plasmids from positive clones were re-transformed, together with pBridge(Mod)-ERα-ΔLBD or pBridge(Mod)-ERα and interactions confirmed by growth on trp− leu− ura− plates.

Northern blottingMultiple tissue northern blots MTN I and MTN II (BD Biosciences, Europe) were probed following 32P-labelling of the ZNF366 cDNA isolated from the pACTII-ZNF366 clone, as described (51).

Northern blotting

Multiple tissue northern blots MTN I and MTN II (BD Biosciences, Europe) were probed following 32P-labelling of the ZNF366 cDNA isolated from the pACTII-ZNF366 clone, as described (51).

Protein expression, purification and glutathione S-transferase (GST)-based interaction assayIn vitro transcription/translations were performed using TNT rabbit reticulocyte lysates (Promega, UK), in the presence of [35S]-labelled methionine. GST proteins were induced and Escherichia coli lysates prepared as described previously (33). For pulldowns, GST fusion proteins were purified by affinity chromatography on glutathione-agarose beads and retained as 50% slurry in 20 mM HEPES (pH 7.6), 100 mM KCl, 1 mM EDTA, 1 mM DTT, 20% glycerol, supplemented with protease inhibitors. A total of 100 μl volumes of glutathione-agarose bead slurry loaded with 10 μg of GST fusion proteins were then used directly in binding assays with 10 μl of radiolabelled in vitro translation reactions and 890 μl of low salt buffer [50 mM HEPES (pH 7.6), 250 mM NaCl, 0.5% NP-40, 5 mM EDTA, 0.1% BSA, 0.5 mM DTT, 0.005% SDS and protease inhibitors]. Following 1 h incubation at room temperature, the beads were washed twice with low salt buffer and twice with high-salt buffer (low salt buffer, but with 1 M NaCl). Samples were boiled for 10 min in 80 μl of Laemmli buffer and fractionated by SDS–PAGE. Gels were dried and autoradiographed.

Protein expression, purification and glutathione S-transferase (GST)-based interaction assay

In vitro transcription/translations were performed using TNT rabbit reticulocyte lysates (Promega, UK), in the presence of [35S]-labelled methionine. GST proteins were induced and Escherichia coli lysates prepared as described previously (33). For pulldowns, GST fusion proteins were purified by affinity chromatography on glutathione-agarose beads and retained as 50% slurry in 20 mM HEPES (pH 7.6), 100 mM KCl, 1 mM EDTA, 1 mM DTT, 20% glycerol, supplemented with protease inhibitors. A total of 100 μl volumes of glutathione-agarose bead slurry loaded with 10 μg of GST fusion proteins were then used directly in binding assays with 10 μl of radiolabelled in vitro translation reactions and 890 μl of low salt buffer [50 mM HEPES (pH 7.6), 250 mM NaCl, 0.5% NP-40, 5 mM EDTA, 0.1% BSA, 0.5 mM DTT, 0.005% SDS and protease inhibitors]. Following 1 h incubation at room temperature, the beads were washed twice with low salt buffer and twice with high-salt buffer (low salt buffer, but with 1 M NaCl). Samples were boiled for 10 min in 80 μl of Laemmli buffer and fractionated by SDS–PAGE. Gels were dried and autoradiographed.

Reporter gene assaysCOS-1 cells were maintained in DMEM, supplemented with 5% fetal calf serum (FCS). For transient transfection, cells were seeded in 24-well plates in DMEM lacking phenol red and supplemented with 5% dextran-coated charcoal-stripped FCS (DSS). Following seeding for 24 h, the cells were transfected using Fugene 6 (Roche Diagnostics, UK), with 100 ng of luciferase reporter gene and amounts of expression plasmids as indicated in the figure legends. E2 (10 nM), 4-hydroxytamoxifen (OHT; 100 nM) or ICI 182, 780 (ICI; 100 nM) were added as appropriate. Since the ligands were prepared in ethanol, an equal volume of ethanol was added to the no ligand controls. Luciferase activities were determined using the Dual-Glo Luciferase Assay kit (Promega, UK). For the other reporter gene assays, cells were maintained in DMEM, supplemented with 5% FCS and transfections carried out as above.

Reporter gene assays

COS-1 cells were maintained in DMEM, supplemented with 5% fetal calf serum (FCS). For transient transfection, cells were seeded in 24-well plates in DMEM lacking phenol red and supplemented with 5% dextran-coated charcoal-stripped FCS (DSS). Following seeding for 24 h, the cells were transfected using Fugene 6 (Roche Diagnostics, UK), with 100 ng of luciferase reporter gene and amounts of expression plasmids as indicated in the figure legends. E2 (10 nM), 4-hydroxytamoxifen (OHT; 100 nM) or ICI 182, 780 (ICI; 100 nM) were added as appropriate. Since the ligands were prepared in ethanol, an equal volume of ethanol was added to the no ligand controls. Luciferase activities were determined using the Dual-Glo Luciferase Assay kit (Promega, UK). For the other reporter gene assays, cells were maintained in DMEM, supplemented with 5% FCS and transfections carried out as above.

Immunoprecipitations and immunoblottingCOS-1 cells were plated in 9 cm dishes in DMEM supplemented with 5% FCS 16 to 24 h prior to transfection. The cells were transfected with 5 μg of the ZNF366-FLAG and ERα expression plasmids using Lipofectamine 2000 (Invitrogen, UK). Following transfection for 48 h, the cells were lysed in RIPA buffer [150 mM NaCl, 1% NP-40, 0.5% deoxycholic acid, 0.1% SDS and 50 mM Tris–HCl (pH 7.5)] containing protease inhibitors. Lysates (2 mg) were immunoprecipitated (IP) using the M2 anti-FLAG mouse monoclonal antibody (Sigma–Aldrich, UK), or using an anti-ERα antibody (6F11; Novocastra, UK). Control IPs was carried out using mouse IgG (Sigma–Aldrich, UK). IPs were resolved by SDS–PAGE and immunoblotted using horseradish peroxidase (HRP)-labelled HA antibody (Sigma–Aldrich, UK) or using an anti-ERα rabbit polyclonal antibody HC20 (Santa Cruz, UK). Co-IP of ZNF366-FLAG with CtBP was carried out as above, except that a mouse monoclonal CtBP antibody (sc-17759; Santa Cruz) was used for the IPs and a rabbit polyclonal CtBP antibody (sc-11390; Santa Cruz) was used for immunoblotting.MCF7 cells cultured for 3 days in DMEM lacking phenol red and supplemented with 5% DSS, were transfected with 1 μg of ZNF366-FLAG or vector control, using Fugene 6. The media were replaced with media containing E2 (10 nM) or vehicle, 24 h following transfection and the cells were harvested after a further 24 h. Immunoblotting was performed using antibodies for cathepsin D (ab6313; Abcam, UK), progesterone receptor (SC538; Santa Cruz Biotechnologies, UK), FLAG-M2 and β-actin (ab6276; Abcam, UK).

Immunoprecipitations and immunoblotting

COS-1 cells were plated in 9 cm dishes in DMEM supplemented with 5% FCS 16 to 24 h prior to transfection. The cells were transfected with 5 μg of the ZNF366-FLAG and ERα expression plasmids using Lipofectamine 2000 (Invitrogen, UK). Following transfection for 48 h, the cells were lysed in RIPA buffer [150 mM NaCl, 1% NP-40, 0.5% deoxycholic acid, 0.1% SDS and 50 mM Tris–HCl (pH 7.5)] containing protease inhibitors. Lysates (2 mg) were immunoprecipitated (IP) using the M2 anti-FLAG mouse monoclonal antibody (Sigma–Aldrich, UK), or using an anti-ERα antibody (6F11; Novocastra, UK). Control IPs was carried out using mouse IgG (Sigma–Aldrich, UK). IPs were resolved by SDS–PAGE and immunoblotted using horseradish peroxidase (HRP)-labelled HA antibody (Sigma–Aldrich, UK) or using an anti-ERα rabbit polyclonal antibody HC20 (Santa Cruz, UK). Co-IP of ZNF366-FLAG with CtBP was carried out as above, except that a mouse monoclonal CtBP antibody (sc-17759; Santa Cruz) was used for the IPs and a rabbit polyclonal CtBP antibody (sc-11390; Santa Cruz) was used for immunoblotting.

MCF7 cells cultured for 3 days in DMEM lacking phenol red and supplemented with 5% DSS, were transfected with 1 μg of ZNF366-FLAG or vector control, using Fugene 6. The media were replaced with media containing E2 (10 nM) or vehicle, 24 h following transfection and the cells were harvested after a further 24 h. Immunoblotting was performed using antibodies for cathepsin D (ab6313; Abcam, UK), progesterone receptor (SC538; Santa Cruz Biotechnologies, UK), FLAG-M2 and β-actin (ab6276; Abcam, UK).

ImmunofluorescenceCOS-1 cells plated on glass coverslips placed in 24-well plates in DMEM lacking phenol red and containing 5% DSS, were transiently transfected with 50 ng of ZNF366-FLAG and/or [ERα-ΔNLS (HE257G; (48)] using Fugene 6. Five hours following transfection, culture media were replaced by fresh media containing E2 (100 nM), OHT (1 μM) or ICI 182, 780 (100 nM), or an equal volume of vehicle (ethanol), as appropriate. 24 h later, cells were fixed by the addition of 4% formaldehyde for 10 min at room temperature, washed with phosphate-buffered saline (PBS) and 0.1 M glycine was added for 10 min to neutralize the formaldehyde. Following further washing with PBS, the cells were permeabilized in 1% Triton/PBS for 5 min. After washing with PBS, the cells were incubated at 37°C for 1 h with the 6F11 ERα antibody (1:50 dilution) and rabbit polyclonal FLAG antiserum (Santa Cruz Biotechnology, UK) (1:350 dilution). The cells were washed and incubated for 1 h at 37°C with Alexa Fluor 488 goat anti-mouse immunoglobulins (green) and Alexa Fluor 594 goat anti-rabbit immunoglobulins (red) (1:3000 dilution). The coverslips were mounted on microscope slides using mountant containing Dapi (Vector Laboratories, UK) and immunofluorescence observed using a Zeiss LSM510 confocal microscope.

Immunofluorescence

COS-1 cells plated on glass coverslips placed in 24-well plates in DMEM lacking phenol red and containing 5% DSS, were transiently transfected with 50 ng of ZNF366-FLAG and/or [ERα-ΔNLS (HE257G; (48)] using Fugene 6. Five hours following transfection, culture media were replaced by fresh media containing E2 (100 nM), OHT (1 μM) or ICI 182, 780 (100 nM), or an equal volume of vehicle (ethanol), as appropriate. 24 h later, cells were fixed by the addition of 4% formaldehyde for 10 min at room temperature, washed with phosphate-buffered saline (PBS) and 0.1 M glycine was added for 10 min to neutralize the formaldehyde. Following further washing with PBS, the cells were permeabilized in 1% Triton/PBS for 5 min. After washing with PBS, the cells were incubated at 37°C for 1 h with the 6F11 ERα antibody (1:50 dilution) and rabbit polyclonal FLAG antiserum (Santa Cruz Biotechnology, UK) (1:350 dilution). The cells were washed and incubated for 1 h at 37°C with Alexa Fluor 488 goat anti-mouse immunoglobulins (green) and Alexa Fluor 594 goat anti-rabbit immunoglobulins (red) (1:3000 dilution). The coverslips were mounted on microscope slides using mountant containing Dapi (Vector Laboratories, UK) and immunofluorescence observed using a Zeiss LSM510 confocal microscope.

Growth assaysMCF7 and MDA-MB-231 cells (2 × 105 cells per well) were seeded in 6-well plates in DMEM lacking phenol red and containing 5% DSS. After 48 h the cells were transfected with 1 μg of ZNF366 or empty vector using Fugene 6. E2 (10 nM) was added after 24 h and cell numbers determined using a haemocytometer after a further 48 h.

Growth assays

MCF7 and MDA-MB-231 cells (2 × 105 cells per well) were seeded in 6-well plates in DMEM lacking phenol red and containing 5% DSS. After 48 h the cells were transfected with 1 μg of ZNF366 or empty vector using Fugene 6. E2 (10 nM) was added after 24 h and cell numbers determined using a haemocytometer after a further 48 h.

RNA interference and RT–PCR analysis of gene expressionPE04 cells (2.5 × 105 cells) seeded in 6-well plates in RPMI lacking phenol red and containing 5% DSS, were transfected with double-stranded RNA oligonucleotides for ZNF366, lamin A/C or a non-targeting siRNA (Ambion, UK), in serum-free DMEM lacking phenol red, using Oligofectamine (Invitrogen, UK), according to manufacturer's protocols. After 4 h the medium was changed to DMEM lacking phenol red, supplemented with 5% DSS and containing E2 (10 nM), as appropriate. RNA was prepared after a further 24 h and RT–PCR carried out using primers with the sequences: 5′-GGACCAGCTTCAGTCACCTTTCCAGTGGTGGCC-3′ and 5′-GGGAGTAAAGCTGGTGCCTGGGGCACAGGTCACG-3′ (GREB1), 5′-CCCCATCCAGTACAACTGCT-3′ and 5′-CTTCACGTCAGAGTGGACGA-3′ (ZNF366), 5′-GCGTACGGCTCTCATCAACT-3′ and 5′-GACACTGGAGGCAGAAGAGC-3′ (Lamin A/C) and 5′-TCCCATCACCATCTTCCA-3′ and 5′-CATCACGCCACAGTTTCC-3′ (GAPDH).

RNA interference and RT–PCR analysis of gene expression

PE04 cells (2.5 × 105 cells) seeded in 6-well plates in RPMI lacking phenol red and containing 5% DSS, were transfected with double-stranded RNA oligonucleotides for ZNF366, lamin A/C or a non-targeting siRNA (Ambion, UK), in serum-free DMEM lacking phenol red, using Oligofectamine (Invitrogen, UK), according to manufacturer's protocols. After 4 h the medium was changed to DMEM lacking phenol red, supplemented with 5% DSS and containing E2 (10 nM), as appropriate. RNA was prepared after a further 24 h and RT–PCR carried out using primers with the sequences: 5′-GGACCAGCTTCAGTCACCTTTCCAGTGGTGGCC-3′ and 5′-GGGAGTAAAGCTGGTGCCTGGGGCACAGGTCACG-3′ (GREB1), 5′-CCCCATCCAGTACAACTGCT-3′ and 5′-CTTCACGTCAGAGTGGACGA-3′ (ZNF366), 5′-GCGTACGGCTCTCATCAACT-3′ and 5′-GACACTGGAGGCAGAAGAGC-3′ (Lamin A/C) and 5′-TCCCATCACCATCTTCCA-3′ and 5′-CATCACGCCACAGTTTCC-3′ (GAPDH).

RESULTSIdentification of ZNF366We utilized yeast strain PL1α, encoding an integrated estrogen-responsive URA3 gene (50), for screening of a human placental cDNA expression library for proteins that interact with an ERα deletion mutant lacking the LBD (ERα-ΔLBD). Screening of 2 × 104 transformants yielded 24 positive clones. One of these encoded the C-terminal portion of ZNF366, described previously based on gene prediction of genomic DNA sequence of human chromosome 5q13.2. Human ESTs were identified using the NCBI site () by carrying out a BLAST search against the database of human ESTs, using sequences derived from the yeast 2-hybrid clone. The exact intron/exon structure of ZNF366 was established by complete DNA sequencing of overlapping EST IMAGE clones with GenBank accession nos BI523869, 5201353, BI770486 and BE552137 to generate an mRNA sequence encoding a predicted polypeptide of 744 amino acids (Figure 1A), encoded within 5 exons. Based on the homology with the Fugu fZF1 gene, this was subsequently confirmed by Gilligan et al. (52).Homology searches identified genes in mouse, rat, dog, chicken and Xenopus tropicalis, having very high amino acid sequence homology (93% or greater) in the zinc finger region and all of which encode 11 zinc fingers (Figure 1B). Regions N- and C-terminal to the zinc fingers were also well conserved, albeit less so than the zinc finger region. Together, the high degree of sequence conservation indicates that the identified vertebrate genes represent orthologues of human ZNF366. Support for this idea is provided by additional homology searches, which indicated that the human gene most closely related to ZNF366 is ZNF710, also encoding an 11 zinc finger protein, the zinc finger regions of human ZNF366 and ZNF710 showing amino acid sequence identity of 78%, considerably lower than the identity between human and the putative ZNF366 homologue in chicken or Xenopus. Moreover, no significant homology was detectable between human ZNF366 and ZNF710 outside the zinc finger region. All other human genes demonstrating homology to human ZNF366 displayed amino acid sequence identities no better than 39%, when the zinc finger region of ZNF366 was used in the BLAST searching. Together with the previous report demonstrating, based on very high (88%) amino acid sequence homology in the zinc finger region, as well as conservation of the chromosome location of human ZNF366 and the Fugu fZF1 gene (52), these analyses indicate that ZNF366 is highly conserved in vertebrate evolution.Northern blotting of human tissue RNAs demonstrated that ZNF366 is broadly expressed at varying levels in human adult tissues, with highest expression in heart, placenta, muscle and spleen, with possible alternative splice forms in the liver and muscle (Figure 1C).Interaction of ZNF366 with ER-αTo further confirm interaction between ERα and ZNF366, whole cell lysates were prepared from COS-1 cells transiently transfected with ERα and FLAG-tagged ZNF366, in the presence of E2. IP with FLAG antibody, followed by immunoblotting using the HC20 rabbit polyclonal ERα antiserum, showed that ERα interacts with ZNF366 (Figure 2A), whilst IP of ERα co-immunoprecipitated ZNF366-FLAG (Figure 2B), indicating that ERα and ZNF366 interact in vivo.As expected, PL1α cells co-transformed with pBridge(Mod)-ERα-ΔLBD and pACTII-ZNF366 grew on minimal medium lacking lacking uracil (Figure 2C). However, for full-length ERα, productive interaction, as assayed by growth in the absence of uracil, was ligand-dependent, requiring the addition of estrogen (17β-estradiol; E2) or an anti-estrogen 4-hydroxytamoxifen (OHT).In order to confirm the ligand requirement for the interaction between ERα and/or ZNF366 in mammalian cells, COS-1 cells were transfected with ERα−ΔNLS, which is excluded from the nucleus (48). ZNF366-FLAG was exclusively localized to the nuclei (Figure 2D), whilst ERα−ΔNLS was always cytoplasmic in the presence of E2, or in the presence of the anti-estrogens OHT or ICI 182, 780 (ICI), as well as in the absence of ligand, when expressed alone (data not shown). In cells co-transfected with ZNF366-FLAG and ERα−ΔNLS, ERα−ΔNLS re-localized to the nucleus when E2, OHT or ICI were present, but not in the absence of ligand, indicative of ligand-dependent interaction between the two proteins. These data further demonstrate that ERα−ΔNLS and ZNF366 interact in vivo and in agreement with the yeast 2-hybrid data, show that the in vivo interaction requires estrogen agonist or antagonist binding by ERα.In contrast, in vitro binding assays showed that ERα bound to GST-ZNF366 in the absence of ligand, as well as in the presence of E2 or anti-estrogens, although the interaction appeared to be greater in the presence of E2 and ICI (Figure 3C), which may suggest that the interaction between ERα and ZNF366 in vivo is regulated by other factors (see Discussion). ERα can be phosphorylated at serine 118 within AF1, and substitution of this residue by alanine significantly reduces ERα activity (53). Substitution of leucine-539 and leucine-540 in the LBD also dramatically reduces ERα activity, by preventing coactivator recruitment (54). Substitution of serine 118 or of leucine-539/540 did not inhibit ZNF366 interaction with ERα, suggesting that AF1 and the LBD/AF2 may not be involved in the interaction of ZNF366 with ERα.GST pulldowns were performed to delineate the regions of ERα and ZNF366 required for their interaction. ERα deletion mutants lacking AF1 (ERα−ΔAF1) or the LBD (ERα−ΔLBD) interacted with ZNF366, as did the isolated ERα DBD (Figure 3D), whereas the interaction with the LBD was weak, suggesting that the ERα DBD is required for interaction with ZNF366. GST fusion proteins encoding the N- and C-terminal regions of ZNF366, ZNF366(9–251) and ZNF366(558–744) respectively, did not interact with ERα, indicating that the zinc finger region is required for the interaction. Interestingly, this assay suggests that the interaction requires independent binding to several zinc fingers, since ZNF366(9–452) and ZNF366(455–744), which do not overlap, both interacted with ERα.ZNF366 is a corepressor for ERαIn reporter gene assays, co-transfection with increasing amounts of ZNF366 showed a dose-dependent inhibition of ERα activity (Figure 4A). ZNF366 repressed AF1 (ERα-ΔLBD; Figure 4B) and AF2 (ERα-ΔAF1; Figure 4C), in agreement with the GST pulldowns that indicate interaction of ZNF366 with the ERα DBD. A trans-repression assay was employed to see whether ZNF366 is a repressor and encodes autonomous repression domain(s). For this, a luciferase reporter gene under the control of LexA and Gal4 binding sites upstream of an E1A TATA box was used. As expected, LexA-VP16 stimulated reporter gene expression (Figure 4D), with this activity being reduced in a dose-dependent manner by the Gal4 DBD fused to the NR corepressors RIP140 (Gal4-RIP140), as described previously (37). ZNF366 similarly repressed reporter gene activity in a dose-dependent manner, confirmative of its activity as a transcriptional repressor.In order to address the potential activity of ZNF366 on ERα-regulated gene expression, we looked for human cell lines in which the genes were co-expressed (data not shown). The PE04 ovarian cancer cell line was found to express both ERα (3) and ZNF366 (Figure 4E). Transfection with siRNA for ZNF366 resulted in down-regulation of ZNF366 expression and concomitant increase in the expression of the estrogen-responsive GREB1 and TERT genes, indicating that ZNF366 is involved in the regulation of estrogen-responsive gene expression in vivo.ZNF366 interacts with CtBP in vitro and in vivoSince ZNF366 can repress ligand-stimulated ERα activity, we wondered whether it interacts with other corepressors that are known to associate with ERα in a ligand-dependent manner. Amongst these is RIP140 (33), which acts by recruiting HDACs and C-terminal binding protein (CtBP) (37). In GST pulldown assays, ZNF366 interacted with RIP140 1–415 and 753–1158 amino acids (Figure 5A), which encode repression domains RD1 and RD4 (37). This interaction appeared to require the C-terminal-most zinc fingers 8–11 of ZNF366 (Figure 5B).Interestingly, ZNF366 also interacted with CtBP1 (Figure 5A), the interaction apparently requiring sequences C-terminal to the zinc fingers (Figure 5B). Many proteins interact with CtBP through sequence motifs having the consensus sequence PXDLS, with a lysine residue two amino acids C-terminal to the serine also often being present (41). Two such motifs, 590-PFDLS(QK)-596 and 645-PEDLS(TK)-651 (Figure 1) are located within the region of ZNF366 required for interaction with CtBP. Mutation of the CtBP motifs by substituting the proline (P) and aspartic acid (D) residues by alanines, prevented interaction between ZNF366 and CtBP in GST pulldown assays (Figure 5C) and in a mammalian two-hybrid assay (Figure 5D). Mutation of M1 or the M2 motif reduced the interaction between ZNF366 and CtBP1. In these assays, the N-terminal CtBP motif (M1) appeared to be more important than the C-terminal motif (M2).In the trans-repression assay, mutation of the N-terminal-most CtBP binding motif in ZNF366 (M1) partially relieved the repression of LexA-VP16 (Figure 5E), whereas mutation of the second motif (M2) did not significantly relieve the repression and mutation of both motifs almost completely abolished the repression by ZNF366. These findings show that the interaction between ZNF366 and CtBP is important for the repression activity of ZNF366, the interaction being mediated by two CtBP-interaction motifs, with both motifs being required for the interaction with CtBP, although motif M1 may be more important than M2 for the interaction.Whole cell lysates prepared from COS-1 cells transiently transfected with CtBP1 and FLAG-tagged ZNF366 immunoprecipitated using a CtBP antibody resulted in co-IP of FLAG-ZNF366 (Figure 5F). In the reciprocal experiment, CtBP1 was co-immunoprecipitated with FLAG-ZNF366 (Figure 5G). CtBP1 was not co-immunoprecipitated with ZNF366 in which the CtBP binding motifs were mutated. Collectively these data demonstrate that ZNF366 interacts in vitro and in vivo with CtBP1, the interaction being mediated by two CtBP binding motifs. In agreement with these findings substitution of the CtBP motifs in ZNF366 significantly reduced the repression of ERα activity by ZNF366 (Figure 6A).Gene repression by transcriptional corepressors, including RIP140 and CtBP, frequently requires HDAC recruitment and histone deacetylation. The HDAC inhibitor suberoylanilide hydroxamic acid (SAHA) relieved the repression of ERα activity by ZNF366 (Figure 6B), whilst GST pulldowns showed that ZNF366 interacts with HDACs 1, 3 and 6 (Figure 6C), indicating that transcriptional repression by ZNF366 is mediated, at least in part through histone deacetylation.ZNF366 represses the expression of estrogen-responsive genes in breast cancer cellsThe majority of breast cancers express ERα, and the growth of ERα-positive breast tumours is stimulated by estrogen, as evidenced by the utility of anti-estrogens and inhibitors of estrogen biosynthesis in breast cancer treatment (2). The MCF7 breast cancer cell line expresses ERα and grows in response to estrogen, its growth being inhibited by anti-estrogens. Further, MCF7 cells demonstrate estrogen-stimulated expression of a number of well-characterized estrogen-responsive genes, including cathepsin D and pS2. In order to evaluate the effect of ZNF366 on ERα-regulation of these genomically encoded estrogen-responsive genes, MCF7 cells were transfected with ZNF366. This resulted in a marked reduction in expression of both cathepsin D and pS2 (Figure 7A). MCF7 cell growth was also reduced following ZNF366 transfection (Figure 7B), whilst growth of an ERα-negative breast cancer cell line that is not estrogen-responsive, was not inhibited by ZNF366 (Figure 7C).

Identification of ZNF366We utilized yeast strain PL1α, encoding an integrated estrogen-responsive URA3 gene (50), for screening of a human placental cDNA expression library for proteins that interact with an ERα deletion mutant lacking the LBD (ERα-ΔLBD). Screening of 2 × 104 transformants yielded 24 positive clones. One of these encoded the C-terminal portion of ZNF366, described previously based on gene prediction of genomic DNA sequence of human chromosome 5q13.2. Human ESTs were identified using the NCBI site () by carrying out a BLAST search against the database of human ESTs, using sequences derived from the yeast 2-hybrid clone. The exact intron/exon structure of ZNF366 was established by complete DNA sequencing of overlapping EST IMAGE clones with GenBank accession nos BI523869, 5201353, BI770486 and BE552137 to generate an mRNA sequence encoding a predicted polypeptide of 744 amino acids (Figure 1A), encoded within 5 exons. Based on the homology with the Fugu fZF1 gene, this was subsequently confirmed by Gilligan et al. (52).Homology searches identified genes in mouse, rat, dog, chicken and Xenopus tropicalis, having very high amino acid sequence homology (93% or greater) in the zinc finger region and all of which encode 11 zinc fingers (Figure 1B). Regions N- and C-terminal to the zinc fingers were also well conserved, albeit less so than the zinc finger region. Together, the high degree of sequence conservation indicates that the identified vertebrate genes represent orthologues of human ZNF366. Support for this idea is provided by additional homology searches, which indicated that the human gene most closely related to ZNF366 is ZNF710, also encoding an 11 zinc finger protein, the zinc finger regions of human ZNF366 and ZNF710 showing amino acid sequence identity of 78%, considerably lower than the identity between human and the putative ZNF366 homologue in chicken or Xenopus. Moreover, no significant homology was detectable between human ZNF366 and ZNF710 outside the zinc finger region. All other human genes demonstrating homology to human ZNF366 displayed amino acid sequence identities no better than 39%, when the zinc finger region of ZNF366 was used in the BLAST searching. Together with the previous report demonstrating, based on very high (88%) amino acid sequence homology in the zinc finger region, as well as conservation of the chromosome location of human ZNF366 and the Fugu fZF1 gene (52), these analyses indicate that ZNF366 is highly conserved in vertebrate evolution.Northern blotting of human tissue RNAs demonstrated that ZNF366 is broadly expressed at varying levels in human adult tissues, with highest expression in heart, placenta, muscle and spleen, with possible alternative splice forms in the liver and muscle (Figure 1C).

Identification of ZNF366

We utilized yeast strain PL1α, encoding an integrated estrogen-responsive URA3 gene (50), for screening of a human placental cDNA expression library for proteins that interact with an ERα deletion mutant lacking the LBD (ERα-ΔLBD). Screening of 2 × 104 transformants yielded 24 positive clones. One of these encoded the C-terminal portion of ZNF366, described previously based on gene prediction of genomic DNA sequence of human chromosome 5q13.2. Human ESTs were identified using the NCBI site () by carrying out a BLAST search against the database of human ESTs, using sequences derived from the yeast 2-hybrid clone. The exact intron/exon structure of ZNF366 was established by complete DNA sequencing of overlapping EST IMAGE clones with GenBank accession nos BI523869, 5201353, BI770486 and BE552137 to generate an mRNA sequence encoding a predicted polypeptide of 744 amino acids (Figure 1A), encoded within 5 exons. Based on the homology with the Fugu fZF1 gene, this was subsequently confirmed by Gilligan et al. (52).

Homology searches identified genes in mouse, rat, dog, chicken and Xenopus tropicalis, having very high amino acid sequence homology (93% or greater) in the zinc finger region and all of which encode 11 zinc fingers (Figure 1B). Regions N- and C-terminal to the zinc fingers were also well conserved, albeit less so than the zinc finger region. Together, the high degree of sequence conservation indicates that the identified vertebrate genes represent orthologues of human ZNF366. Support for this idea is provided by additional homology searches, which indicated that the human gene most closely related to ZNF366 is ZNF710, also encoding an 11 zinc finger protein, the zinc finger regions of human ZNF366 and ZNF710 showing amino acid sequence identity of 78%, considerably lower than the identity between human and the putative ZNF366 homologue in chicken or Xenopus. Moreover, no significant homology was detectable between human ZNF366 and ZNF710 outside the zinc finger region. All other human genes demonstrating homology to human ZNF366 displayed amino acid sequence identities no better than 39%, when the zinc finger region of ZNF366 was used in the BLAST searching. Together with the previous report demonstrating, based on very high (88%) amino acid sequence homology in the zinc finger region, as well as conservation of the chromosome location of human ZNF366 and the Fugu fZF1 gene (52), these analyses indicate that ZNF366 is highly conserved in vertebrate evolution.

Northern blotting of human tissue RNAs demonstrated that ZNF366 is broadly expressed at varying levels in human adult tissues, with highest expression in heart, placenta, muscle and spleen, with possible alternative splice forms in the liver and muscle (Figure 1C).

Interaction of ZNF366 with ER-αTo further confirm interaction between ERα and ZNF366, whole cell lysates were prepared from COS-1 cells transiently transfected with ERα and FLAG-tagged ZNF366, in the presence of E2. IP with FLAG antibody, followed by immunoblotting using the HC20 rabbit polyclonal ERα antiserum, showed that ERα interacts with ZNF366 (Figure 2A), whilst IP of ERα co-immunoprecipitated ZNF366-FLAG (Figure 2B), indicating that ERα and ZNF366 interact in vivo.As expected, PL1α cells co-transformed with pBridge(Mod)-ERα-ΔLBD and pACTII-ZNF366 grew on minimal medium lacking lacking uracil (Figure 2C). However, for full-length ERα, productive interaction, as assayed by growth in the absence of uracil, was ligand-dependent, requiring the addition of estrogen (17β-estradiol; E2) or an anti-estrogen 4-hydroxytamoxifen (OHT).In order to confirm the ligand requirement for the interaction between ERα and/or ZNF366 in mammalian cells, COS-1 cells were transfected with ERα−ΔNLS, which is excluded from the nucleus (48). ZNF366-FLAG was exclusively localized to the nuclei (Figure 2D), whilst ERα−ΔNLS was always cytoplasmic in the presence of E2, or in the presence of the anti-estrogens OHT or ICI 182, 780 (ICI), as well as in the absence of ligand, when expressed alone (data not shown). In cells co-transfected with ZNF366-FLAG and ERα−ΔNLS, ERα−ΔNLS re-localized to the nucleus when E2, OHT or ICI were present, but not in the absence of ligand, indicative of ligand-dependent interaction between the two proteins. These data further demonstrate that ERα−ΔNLS and ZNF366 interact in vivo and in agreement with the yeast 2-hybrid data, show that the in vivo interaction requires estrogen agonist or antagonist binding by ERα.In contrast, in vitro binding assays showed that ERα bound to GST-ZNF366 in the absence of ligand, as well as in the presence of E2 or anti-estrogens, although the interaction appeared to be greater in the presence of E2 and ICI (Figure 3C), which may suggest that the interaction between ERα and ZNF366 in vivo is regulated by other factors (see Discussion). ERα can be phosphorylated at serine 118 within AF1, and substitution of this residue by alanine significantly reduces ERα activity (53). Substitution of leucine-539 and leucine-540 in the LBD also dramatically reduces ERα activity, by preventing coactivator recruitment (54). Substitution of serine 118 or of leucine-539/540 did not inhibit ZNF366 interaction with ERα, suggesting that AF1 and the LBD/AF2 may not be involved in the interaction of ZNF366 with ERα.GST pulldowns were performed to delineate the regions of ERα and ZNF366 required for their interaction. ERα deletion mutants lacking AF1 (ERα−ΔAF1) or the LBD (ERα−ΔLBD) interacted with ZNF366, as did the isolated ERα DBD (Figure 3D), whereas the interaction with the LBD was weak, suggesting that the ERα DBD is required for interaction with ZNF366. GST fusion proteins encoding the N- and C-terminal regions of ZNF366, ZNF366(9–251) and ZNF366(558–744) respectively, did not interact with ERα, indicating that the zinc finger region is required for the interaction. Interestingly, this assay suggests that the interaction requires independent binding to several zinc fingers, since ZNF366(9–452) and ZNF366(455–744), which do not overlap, both interacted with ERα.

Interaction of ZNF366 with ER-α

To further confirm interaction between ERα and ZNF366, whole cell lysates were prepared from COS-1 cells transiently transfected with ERα and FLAG-tagged ZNF366, in the presence of E2. IP with FLAG antibody, followed by immunoblotting using the HC20 rabbit polyclonal ERα antiserum, showed that ERα interacts with ZNF366 (Figure 2A), whilst IP of ERα co-immunoprecipitated ZNF366-FLAG (Figure 2B), indicating that ERα and ZNF366 interact in vivo.

As expected, PL1α cells co-transformed with pBridge(Mod)-ERα-ΔLBD and pACTII-ZNF366 grew on minimal medium lacking lacking uracil (Figure 2C). However, for full-length ERα, productive interaction, as assayed by growth in the absence of uracil, was ligand-dependent, requiring the addition of estrogen (17β-estradiol; E2) or an anti-estrogen 4-hydroxytamoxifen (OHT).

In order to confirm the ligand requirement for the interaction between ERα and/or ZNF366 in mammalian cells, COS-1 cells were transfected with ERα−ΔNLS, which is excluded from the nucleus (48). ZNF366-FLAG was exclusively localized to the nuclei (Figure 2D), whilst ERα−ΔNLS was always cytoplasmic in the presence of E2, or in the presence of the anti-estrogens OHT or ICI 182, 780 (ICI), as well as in the absence of ligand, when expressed alone (data not shown). In cells co-transfected with ZNF366-FLAG and ERα−ΔNLS, ERα−ΔNLS re-localized to the nucleus when E2, OHT or ICI were present, but not in the absence of ligand, indicative of ligand-dependent interaction between the two proteins. These data further demonstrate that ERα−ΔNLS and ZNF366 interact in vivo and in agreement with the yeast 2-hybrid data, show that the in vivo interaction requires estrogen agonist or antagonist binding by ERα.

In contrast, in vitro binding assays showed that ERα bound to GST-ZNF366 in the absence of ligand, as well as in the presence of E2 or anti-estrogens, although the interaction appeared to be greater in the presence of E2 and ICI (Figure 3C), which may suggest that the interaction between ERα and ZNF366 in vivo is regulated by other factors (see Discussion). ERα can be phosphorylated at serine 118 within AF1, and substitution of this residue by alanine significantly reduces ERα activity (53). Substitution of leucine-539 and leucine-540 in the LBD also dramatically reduces ERα activity, by preventing coactivator recruitment (54). Substitution of serine 118 or of leucine-539/540 did not inhibit ZNF366 interaction with ERα, suggesting that AF1 and the LBD/AF2 may not be involved in the interaction of ZNF366 with ERα.

GST pulldowns were performed to delineate the regions of ERα and ZNF366 required for their interaction. ERα deletion mutants lacking AF1 (ERα−ΔAF1) or the LBD (ERα−ΔLBD) interacted with ZNF366, as did the isolated ERα DBD (Figure 3D), whereas the interaction with the LBD was weak, suggesting that the ERα DBD is required for interaction with ZNF366. GST fusion proteins encoding the N- and C-terminal regions of ZNF366, ZNF366(9–251) and ZNF366(558–744) respectively, did not interact with ERα, indicating that the zinc finger region is required for the interaction. Interestingly, this assay suggests that the interaction requires independent binding to several zinc fingers, since ZNF366(9–452) and ZNF366(455–744), which do not overlap, both interacted with ERα.

ZNF366 is a corepressor for ERαIn reporter gene assays, co-transfection with increasing amounts of ZNF366 showed a dose-dependent inhibition of ERα activity (Figure 4A). ZNF366 repressed AF1 (ERα-ΔLBD; Figure 4B) and AF2 (ERα-ΔAF1; Figure 4C), in agreement with the GST pulldowns that indicate interaction of ZNF366 with the ERα DBD. A trans-repression assay was employed to see whether ZNF366 is a repressor and encodes autonomous repression domain(s). For this, a luciferase reporter gene under the control of LexA and Gal4 binding sites upstream of an E1A TATA box was used. As expected, LexA-VP16 stimulated reporter gene expression (Figure 4D), with this activity being reduced in a dose-dependent manner by the Gal4 DBD fused to the NR corepressors RIP140 (Gal4-RIP140), as described previously (37). ZNF366 similarly repressed reporter gene activity in a dose-dependent manner, confirmative of its activity as a transcriptional repressor.In order to address the potential activity of ZNF366 on ERα-regulated gene expression, we looked for human cell lines in which the genes were co-expressed (data not shown). The PE04 ovarian cancer cell line was found to express both ERα (3) and ZNF366 (Figure 4E). Transfection with siRNA for ZNF366 resulted in down-regulation of ZNF366 expression and concomitant increase in the expression of the estrogen-responsive GREB1 and TERT genes, indicating that ZNF366 is involved in the regulation of estrogen-responsive gene expression in vivo.

ZNF366 is a corepressor for ERα

In reporter gene assays, co-transfection with increasing amounts of ZNF366 showed a dose-dependent inhibition of ERα activity (Figure 4A). ZNF366 repressed AF1 (ERα-ΔLBD; Figure 4B) and AF2 (ERα-ΔAF1; Figure 4C), in agreement with the GST pulldowns that indicate interaction of ZNF366 with the ERα DBD. A trans-repression assay was employed to see whether ZNF366 is a repressor and encodes autonomous repression domain(s). For this, a luciferase reporter gene under the control of LexA and Gal4 binding sites upstream of an E1A TATA box was used. As expected, LexA-VP16 stimulated reporter gene expression (Figure 4D), with this activity being reduced in a dose-dependent manner by the Gal4 DBD fused to the NR corepressors RIP140 (Gal4-RIP140), as described previously (37). ZNF366 similarly repressed reporter gene activity in a dose-dependent manner, confirmative of its activity as a transcriptional repressor.

In order to address the potential activity of ZNF366 on ERα-regulated gene expression, we looked for human cell lines in which the genes were co-expressed (data not shown). The PE04 ovarian cancer cell line was found to express both ERα (3) and ZNF366 (Figure 4E). Transfection with siRNA for ZNF366 resulted in down-regulation of ZNF366 expression and concomitant increase in the expression of the estrogen-responsive GREB1 and TERT genes, indicating that ZNF366 is involved in the regulation of estrogen-responsive gene expression in vivo.

ZNF366 interacts with CtBP in vitro and in vivoSince ZNF366 can repress ligand-stimulated ERα activity, we wondered whether it interacts with other corepressors that are known to associate with ERα in a ligand-dependent manner. Amongst these is RIP140 (33), which acts by recruiting HDACs and C-terminal binding protein (CtBP) (37). In GST pulldown assays, ZNF366 interacted with RIP140 1–415 and 753–1158 amino acids (Figure 5A), which encode repression domains RD1 and RD4 (37). This interaction appeared to require the C-terminal-most zinc fingers 8–11 of ZNF366 (Figure 5B).Interestingly, ZNF366 also interacted with CtBP1 (Figure 5A), the interaction apparently requiring sequences C-terminal to the zinc fingers (Figure 5B). Many proteins interact with CtBP through sequence motifs having the consensus sequence PXDLS, with a lysine residue two amino acids C-terminal to the serine also often being present (41). Two such motifs, 590-PFDLS(QK)-596 and 645-PEDLS(TK)-651 (Figure 1) are located within the region of ZNF366 required for interaction with CtBP. Mutation of the CtBP motifs by substituting the proline (P) and aspartic acid (D) residues by alanines, prevented interaction between ZNF366 and CtBP in GST pulldown assays (Figure 5C) and in a mammalian two-hybrid assay (Figure 5D). Mutation of M1 or the M2 motif reduced the interaction between ZNF366 and CtBP1. In these assays, the N-terminal CtBP motif (M1) appeared to be more important than the C-terminal motif (M2).In the trans-repression assay, mutation of the N-terminal-most CtBP binding motif in ZNF366 (M1) partially relieved the repression of LexA-VP16 (Figure 5E), whereas mutation of the second motif (M2) did not significantly relieve the repression and mutation of both motifs almost completely abolished the repression by ZNF366. These findings show that the interaction between ZNF366 and CtBP is important for the repression activity of ZNF366, the interaction being mediated by two CtBP-interaction motifs, with both motifs being required for the interaction with CtBP, although motif M1 may be more important than M2 for the interaction.Whole cell lysates prepared from COS-1 cells transiently transfected with CtBP1 and FLAG-tagged ZNF366 immunoprecipitated using a CtBP antibody resulted in co-IP of FLAG-ZNF366 (Figure 5F). In the reciprocal experiment, CtBP1 was co-immunoprecipitated with FLAG-ZNF366 (Figure 5G). CtBP1 was not co-immunoprecipitated with ZNF366 in which the CtBP binding motifs were mutated. Collectively these data demonstrate that ZNF366 interacts in vitro and in vivo with CtBP1, the interaction being mediated by two CtBP binding motifs. In agreement with these findings substitution of the CtBP motifs in ZNF366 significantly reduced the repression of ERα activity by ZNF366 (Figure 6A).Gene repression by transcriptional corepressors, including RIP140 and CtBP, frequently requires HDAC recruitment and histone deacetylation. The HDAC inhibitor suberoylanilide hydroxamic acid (SAHA) relieved the repression of ERα activity by ZNF366 (Figure 6B), whilst GST pulldowns showed that ZNF366 interacts with HDACs 1, 3 and 6 (Figure 6C), indicating that transcriptional repression by ZNF366 is mediated, at least in part through histone deacetylation.

ZNF366 interacts with CtBP in vitro and in vivo

Since ZNF366 can repress ligand-stimulated ERα activity, we wondered whether it interacts with other corepressors that are known to associate with ERα in a ligand-dependent manner. Amongst these is RIP140 (33), which acts by recruiting HDACs and C-terminal binding protein (CtBP) (37). In GST pulldown assays, ZNF366 interacted with RIP140 1–415 and 753–1158 amino acids (Figure 5A), which encode repression domains RD1 and RD4 (37). This interaction appeared to require the C-terminal-most zinc fingers 8–11 of ZNF366 (Figure 5B).

Interestingly, ZNF366 also interacted with CtBP1 (Figure 5A), the interaction apparently requiring sequences C-terminal to the zinc fingers (Figure 5B). Many proteins interact with CtBP through sequence motifs having the consensus sequence PXDLS, with a lysine residue two amino acids C-terminal to the serine also often being present (41). Two such motifs, 590-PFDLS(QK)-596 and 645-PEDLS(TK)-651 (Figure 1) are located within the region of ZNF366 required for interaction with CtBP. Mutation of the CtBP motifs by substituting the proline (P) and aspartic acid (D) residues by alanines, prevented interaction between ZNF366 and CtBP in GST pulldown assays (Figure 5C) and in a mammalian two-hybrid assay (Figure 5D). Mutation of M1 or the M2 motif reduced the interaction between ZNF366 and CtBP1. In these assays, the N-terminal CtBP motif (M1) appeared to be more important than the C-terminal motif (M2).

In the trans-repression assay, mutation of the N-terminal-most CtBP binding motif in ZNF366 (M1) partially relieved the repression of LexA-VP16 (Figure 5E), whereas mutation of the second motif (M2) did not significantly relieve the repression and mutation of both motifs almost completely abolished the repression by ZNF366. These findings show that the interaction between ZNF366 and CtBP is important for the repression activity of ZNF366, the interaction being mediated by two CtBP-interaction motifs, with both motifs being required for the interaction with CtBP, although motif M1 may be more important than M2 for the interaction.

Whole cell lysates prepared from COS-1 cells transiently transfected with CtBP1 and FLAG-tagged ZNF366 immunoprecipitated using a CtBP antibody resulted in co-IP of FLAG-ZNF366 (Figure 5F). In the reciprocal experiment, CtBP1 was co-immunoprecipitated with FLAG-ZNF366 (Figure 5G). CtBP1 was not co-immunoprecipitated with ZNF366 in which the CtBP binding motifs were mutated. Collectively these data demonstrate that ZNF366 interacts in vitro and in vivo with CtBP1, the interaction being mediated by two CtBP binding motifs. In agreement with these findings substitution of the CtBP motifs in ZNF366 significantly reduced the repression of ERα activity by ZNF366 (Figure 6A).

Gene repression by transcriptional corepressors, including RIP140 and CtBP, frequently requires HDAC recruitment and histone deacetylation. The HDAC inhibitor suberoylanilide hydroxamic acid (SAHA) relieved the repression of ERα activity by ZNF366 (Figure 6B), whilst GST pulldowns showed that ZNF366 interacts with HDACs 1, 3 and 6 (Figure 6C), indicating that transcriptional repression by ZNF366 is mediated, at least in part through histone deacetylation.

ZNF366 represses the expression of estrogen-responsive genes in breast cancer cellsThe majority of breast cancers express ERα, and the growth of ERα-positive breast tumours is stimulated by estrogen, as evidenced by the utility of anti-estrogens and inhibitors of estrogen biosynthesis in breast cancer treatment (2). The MCF7 breast cancer cell line expresses ERα and grows in response to estrogen, its growth being inhibited by anti-estrogens. Further, MCF7 cells demonstrate estrogen-stimulated expression of a number of well-characterized estrogen-responsive genes, including cathepsin D and pS2. In order to evaluate the effect of ZNF366 on ERα-regulation of these genomically encoded estrogen-responsive genes, MCF7 cells were transfected with ZNF366. This resulted in a marked reduction in expression of both cathepsin D and pS2 (Figure 7A). MCF7 cell growth was also reduced following ZNF366 transfection (Figure 7B), whilst growth of an ERα-negative breast cancer cell line that is not estrogen-responsive, was not inhibited by ZNF366 (Figure 7C).

ZNF366 represses the expression of estrogen-responsive genes in breast cancer cells

The majority of breast cancers express ERα, and the growth of ERα-positive breast tumours is stimulated by estrogen, as evidenced by the utility of anti-estrogens and inhibitors of estrogen biosynthesis in breast cancer treatment (2). The MCF7 breast cancer cell line expresses ERα and grows in response to estrogen, its growth being inhibited by anti-estrogens. Further, MCF7 cells demonstrate estrogen-stimulated expression of a number of well-characterized estrogen-responsive genes, including cathepsin D and pS2. In order to evaluate the effect of ZNF366 on ERα-regulation of these genomically encoded estrogen-responsive genes, MCF7 cells were transfected with ZNF366. This resulted in a marked reduction in expression of both cathepsin D and pS2 (Figure 7A). MCF7 cell growth was also reduced following ZNF366 transfection (Figure 7B), whilst growth of an ERα-negative breast cancer cell line that is not estrogen-responsive, was not inhibited by ZNF366 (Figure 7C).

DISCUSSIONZinc finger proteins constitute a very large family of transcriptional regulators and can be further subdivided into groupings based on the type of zinc finger present, as well as by the presence of additional motifs elsewhere in the protein that mediate protein–protein interactions and transcriptional regulation. Sequence analysis of ZNF366 shows that it encodes encodes a protein containing 11 Kruppel-type C2H2 zinc fingers, which is highly conserved in vertebrate evolution. However, ZNF366 does not belong to any of the major subfamilies of the Kruppel zinc finger family and shows most significant amino acid sequence similarity to one other Kruppel zinc finger protein, ZNF710, of unknown function, where the homology is restricted to the Kruppel zinc finger region. However, several other Kruppel-type zinc finger proteins act as transcriptional repressors, including ZNF217, a putative oncogene that is amplified and overexpressed in breast and other cancers (55,56), and copurifies with a CtBP corepressor complex (42). Further, the Bcl11 Kruppel zinc finger gene, which is translocated in B-cell chronic lymphocytic leukemias, acts as a corepressor for the COUP-TF NR (57).Although we did not investigate the potential of ZNF366 to bind DNA, alter reporter gene activities in yeast or in mammalian cells in the absence of ERα (see Results and Supplementary Figure 1A). Further, ZNF366 did not bind to estrogen response elements in gelshift assays (Supplementary Figure 1B). Recruitment of ERα to the estrogen-responsive pS2 gene promoter was also not inhibited by ZNF366 expression in MCF7 cells (Supplementary Figure 1C). Collectively, these findings indicate that the repression of estrogen-responsive reporter genes by ZNF366 does not involve inhibition of DNA binding by ERα.ZNF366 appears to be recruited to estrogen-responsive genes through interaction of the zinc finger region of ZNF366 with the zinc finger region (DBD) of ERα. Whilst the zinc finger region was required for ZNF366 interaction with ERα, the exact sequence requirements for the interaction with ERα were not established, although non-overlapping regions of ZNF366, encoding zinc fingers 1–7 or 8–11 were sufficient for the interaction. Whilst the interaction of ZNF366 and ERα did not require ligand for in vitro assays, the interaction was apparently better in the presence of estrogen. Further, in vivo assays demonstrated a requirement for estrogen or anti-estrogen binding for the interaction between ZNF366 and ERα. The in vivo requirement for ligand binding may be influenced by post-translational modifications. Additionally, steroid receptors, including ERα, are complexed in the unliganded state, with the Hsp90 chaperone complex, required for appropriate folding of steroid receptors (58). The Hsp90-steroid receptor also likely interferes with steroid receptor interaction with some proteins. Ligand binding results in a conformational change in steroid receptors and Hsp90 dissociation. This could explain the estrogen and anti-estrogen regulation of ZNF366 recruitment by ERα.In vitro, ZNF366 also interacted with other steroid receptors (ERβ, androgen and glucocorticoid receptors), as well as the non-steroid retinoic acid, retinoid X and peroxisome proliferators-activated receptors (data not shown). In all cases the ZNF366 zinc finger region mediated the interaction, with no interaction being detected for the region C-terminal to the zinc finger region (558–744 amino acids), with the exception of retinoid X receptor-α (RXRα). In this case, ligand-stimulated interaction of RXRα with 558–744 amino acids was observed, indicating that the interaction between ZNF366 and RXRα is mechanistically distinct from the interaction of ZNF366 with other NRs, perhaps requiring the potential LXXLL motif located near the C-terminus of ZNF366.ZNF366 inhibited ligand-dependent transactivation by ERα in a dose-dependent manner and functioned as a repressor when tethered to DNA by the GAL4 DBD. Its interaction with the ERα DBD suggests that the observed repression is not due simply to prevention of coactivator recruitment, nor does ZNF366 inhibit DNA binding by ERα (data not shown). Rather, ZNF366 appears to recruit multiple factors that act to repress transcription. Hence, the HDAC inhibitor SAHA partially relieved the repression of ERα by ZNF366 and in vitro binding assays showed that ZNF366 interacts with Class I HDACs 1 and 3, as well as the Class II HDAC6, indicating that the corepressor activity of ZNF366 is, at least in part, HDAC-dependent.Several different corepressor complexes that are associated with NRs have been identified, most notably N-CoR–SMRT complexes that include HDACs (59). These are usually recruited to unliganded or antagonist-bound NRs, such as the tamoxifen-bound ERα (60). RIP140 is unusual in being a corepressor that is recruited by agonist-bound NRs. The repressive activity of RIP140 is achieved by the recruitment of class I HDACs and CtBP1 (35–39,61). CtBP1 and the related protein CtBP2 are potent corepressors that are present in protein complexes containing HDACs and histone lysine methyltransferases (42), and its corepressor activity is mediated through HDAC-dependent and—independent mechanisms, the HDAC-independent mechanisms likely involving PcG complexes (62). ZNF366 interacted with RIP140, the interaction requiring 455–558 amino acids of ZNF366, which encode zinc fingers 7–11. ZNF366 also interacted with CtBP1 in vitro and in vivo, the interaction being mediated by two PXDLS CtBP-interaction motifs located C-terminal to the zinc fingers in ZNF366. Mutation of the CtBP-interacting motifs prevented the interaction of ZNF366 and CtBP1 and relieved repression of ERα activity by ZNF366, confirming the importance of CtBP recruitment for the corepressor activity of ZNF366. However, the mutant ZNF366 still significantly repressed ERα activity, likely due to the fact that it directly interacts with HDACs and with RIP140.These studies suggest that ZNF366 acts as a corepressor for ERα. In agreement with these findings, transfection of ZNF366 into the estrogen-responsive and ERα-positive MCF7 breast cancer cell line, which does not express ZNF366 (data not shown), reduced expression of genomically encoded ERα-regulated genes. Moreover, expression of ZNF366 inhibited MCF7 cell growth in response to estrogen, whereas ZNF366 expression did not inhibit growth of the ERα-negative, MDA-MB-231 breast cancer cell line that is not estrogen-responsive. Finally, RNAi-mediated down-regulation of ZNF366 in the ERα-positive PE04 ovarian cell line, stimulated expression of the estrogen-regulated GREB1 and TERT genes, further evidence for the in vivo role of ZNF366 as a corepressor for ERα.In summary, we have identified a novel ERα-interacting protein ZNF366, which represses ligand-dependent ERα transactivation by recruitment of multiple factors, to regulate the expression of estrogen-responsive genes. ZNF366 is widely expressed in adult tissues and our preliminary findings of interaction between ZNF366 and many other NRs suggest that ZNF366 may have a widespread role as a NR corepressor, in addition to its action as an ERα corepressor, as defined in this study.

Zinc finger proteins constitute a very large family of transcriptional regulators and can be further subdivided into groupings based on the type of zinc finger present, as well as by the presence of additional motifs elsewhere in the protein that mediate protein–protein interactions and transcriptional regulation. Sequence analysis of ZNF366 shows that it encodes encodes a protein containing 11 Kruppel-type C2H2 zinc fingers, which is highly conserved in vertebrate evolution. However, ZNF366 does not belong to any of the major subfamilies of the Kruppel zinc finger family and shows most significant amino acid sequence similarity to one other Kruppel zinc finger protein, ZNF710, of unknown function, where the homology is restricted to the Kruppel zinc finger region. However, several other Kruppel-type zinc finger proteins act as transcriptional repressors, including ZNF217, a putative oncogene that is amplified and overexpressed in breast and other cancers (55,56), and copurifies with a CtBP corepressor complex (42). Further, the Bcl11 Kruppel zinc finger gene, which is translocated in B-cell chronic lymphocytic leukemias, acts as a corepressor for the COUP-TF NR (57).

Although we did not investigate the potential of ZNF366 to bind DNA, alter reporter gene activities in yeast or in mammalian cells in the absence of ERα (see Results and Supplementary Figure 1A). Further, ZNF366 did not bind to estrogen response elements in gelshift assays (Supplementary Figure 1B). Recruitment of ERα to the estrogen-responsive pS2 gene promoter was also not inhibited by ZNF366 expression in MCF7 cells (Supplementary Figure 1C). Collectively, these findings indicate that the repression of estrogen-responsive reporter genes by ZNF366 does not involve inhibition of DNA binding by ERα.

ZNF366 appears to be recruited to estrogen-responsive genes through interaction of the zinc finger region of ZNF366 with the zinc finger region (DBD) of ERα. Whilst the zinc finger region was required for ZNF366 interaction with ERα, the exact sequence requirements for the interaction with ERα were not established, although non-overlapping regions of ZNF366, encoding zinc fingers 1–7 or 8–11 were sufficient for the interaction. Whilst the interaction of ZNF366 and ERα did not require ligand for in vitro assays, the interaction was apparently better in the presence of estrogen. Further, in vivo assays demonstrated a requirement for estrogen or anti-estrogen binding for the interaction between ZNF366 and ERα. The in vivo requirement for ligand binding may be influenced by post-translational modifications. Additionally, steroid receptors, including ERα, are complexed in the unliganded state, with the Hsp90 chaperone complex, required for appropriate folding of steroid receptors (58). The Hsp90-steroid receptor also likely interferes with steroid receptor interaction with some proteins. Ligand binding results in a conformational change in steroid receptors and Hsp90 dissociation. This could explain the estrogen and anti-estrogen regulation of ZNF366 recruitment by ERα.

In vitro, ZNF366 also interacted with other steroid receptors (ERβ, androgen and glucocorticoid receptors), as well as the non-steroid retinoic acid, retinoid X and peroxisome proliferators-activated receptors (data not shown). In all cases the ZNF366 zinc finger region mediated the interaction, with no interaction being detected for the region C-terminal to the zinc finger region (558–744 amino acids), with the exception of retinoid X receptor-α (RXRα). In this case, ligand-stimulated interaction of RXRα with 558–744 amino acids was observed, indicating that the interaction between ZNF366 and RXRα is mechanistically distinct from the interaction of ZNF366 with other NRs, perhaps requiring the potential LXXLL motif located near the C-terminus of ZNF366.

ZNF366 inhibited ligand-dependent transactivation by ERα in a dose-dependent manner and functioned as a repressor when tethered to DNA by the GAL4 DBD. Its interaction with the ERα DBD suggests that the observed repression is not due simply to prevention of coactivator recruitment, nor does ZNF366 inhibit DNA binding by ERα (data not shown). Rather, ZNF366 appears to recruit multiple factors that act to repress transcription. Hence, the HDAC inhibitor SAHA partially relieved the repression of ERα by ZNF366 and in vitro binding assays showed that ZNF366 interacts with Class I HDACs 1 and 3, as well as the Class II HDAC6, indicating that the corepressor activity of ZNF366 is, at least in part, HDAC-dependent.

Several different corepressor complexes that are associated with NRs have been identified, most notably N-CoR–SMRT complexes that include HDACs (59). These are usually recruited to unliganded or antagonist-bound NRs, such as the tamoxifen-bound ERα (60). RIP140 is unusual in being a corepressor that is recruited by agonist-bound NRs. The repressive activity of RIP140 is achieved by the recruitment of class I HDACs and CtBP1 (35–39,61). CtBP1 and the related protein CtBP2 are potent corepressors that are present in protein complexes containing HDACs and histone lysine methyltransferases (42), and its corepressor activity is mediated through HDAC-dependent and—independent mechanisms, the HDAC-independent mechanisms likely involving PcG complexes (62). ZNF366 interacted with RIP140, the interaction requiring 455–558 amino acids of ZNF366, which encode zinc fingers 7–11. ZNF366 also interacted with CtBP1 in vitro and in vivo, the interaction being mediated by two PXDLS CtBP-interaction motifs located C-terminal to the zinc fingers in ZNF366. Mutation of the CtBP-interacting motifs prevented the interaction of ZNF366 and CtBP1 and relieved repression of ERα activity by ZNF366, confirming the importance of CtBP recruitment for the corepressor activity of ZNF366. However, the mutant ZNF366 still significantly repressed ERα activity, likely due to the fact that it directly interacts with HDACs and with RIP140.

These studies suggest that ZNF366 acts as a corepressor for ERα. In agreement with these findings, transfection of ZNF366 into the estrogen-responsive and ERα-positive MCF7 breast cancer cell line, which does not express ZNF366 (data not shown), reduced expression of genomically encoded ERα-regulated genes. Moreover, expression of ZNF366 inhibited MCF7 cell growth in response to estrogen, whereas ZNF366 expression did not inhibit growth of the ERα-negative, MDA-MB-231 breast cancer cell line that is not estrogen-responsive. Finally, RNAi-mediated down-regulation of ZNF366 in the ERα-positive PE04 ovarian cell line, stimulated expression of the estrogen-regulated GREB1 and TERT genes, further evidence for the in vivo role of ZNF366 as a corepressor for ERα.

In summary, we have identified a novel ERα-interacting protein ZNF366, which represses ligand-dependent ERα transactivation by recruitment of multiple factors, to regulate the expression of estrogen-responsive genes. ZNF366 is widely expressed in adult tissues and our preliminary findings of interaction between ZNF366 and many other NRs suggest that ZNF366 may have a widespread role as a NR corepressor, in addition to its action as an ERα corepressor, as defined in this study.

SUPPLEMENTARY DATASupplementary Data are available at NAR online.

SUPPLEMENTARY DATA

Supplementary Data are available at NAR online.

Supplementary Material[Supplementary Data]

Supplementary Material

[Supplementary Data]
