---
pmid: '26655473'
title: Pre-Anchoring of Pin1 to Unphosphorylated c-Myc in a Fuzzy Complex Regulates
  c-Myc Activity.
authors:
- Helander S
- Montecchio M
- Pilstål R
- Su Y
- Kuruvilla J
- Elvén M
- Ziauddin JME
- Anandapadamanaban M
- Cristobal S
- Lundström P
- Sears RC
- Wallner B
- Sunnerhagen M
journal: Structure
year: '2015'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC4807222
doi: 10.1016/j.str.2015.10.010
---

# Pre-Anchoring of Pin1 to Unphosphorylated c-Myc in a Fuzzy Complex Regulates c-Myc Activity.
**Authors:** Helander S, Montecchio M, Pilstål R, Su Y, Kuruvilla J, Elvén M, Ziauddin JME, Anandapadamanaban M, Cristobal S, Lundström P, Sears RC, Wallner B, Sunnerhagen M
**Journal:** Structure (2015)
**DOI:** [10.1016/j.str.2015.10.010](https://doi.org/10.1016/j.str.2015.10.010)
**PMC:** [PMC4807222](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4807222/)

## Abstract

1. Structure. 2015 Dec 1;23(12):2267-2279. doi: 10.1016/j.str.2015.10.010. Epub 
2015 Nov 19.

Pre-Anchoring of Pin1 to Unphosphorylated c-Myc in a Fuzzy Complex Regulates 
c-Myc Activity.

Helander S(1), Montecchio M(1), Pilstål R(2), Su Y(3), Kuruvilla J(4), Elvén 
M(1), Ziauddin JME(1), Anandapadamanaban M(1), Cristobal S(4), Lundström P(1), 
Sears RC(3), Wallner B(2), Sunnerhagen M(5).

Author information:
(1)Division of Chemistry, Department of Physics, Chemistry and Biology, 
Linköping University, 58183 Linköping, Sweden.
(2)Division of Bioinformatics, Department of Physics, Chemistry and Biology, 
Linköping University, 58183 Linköping, Sweden.
(3)Department of Molecular and Medical Genetics, Oregon Health and Science 
University, Portland, OR 92739, USA.
(4)Department of Clinical and Experimental Medicine, Cell Biology, Faculty of 
Health Science Linköping University, 58183 Linköping, Sweden.
(5)Division of Chemistry, Department of Physics, Chemistry and Biology, 
Linköping University, 58183 Linköping, Sweden. Electronic address: 
maria.sunnerhagen@liu.se.

Hierarchic phosphorylation and concomitant Pin1-mediated proline isomerization 
of the oncoprotein c-Myc controls its cellular stability and activity. However, 
the molecular basis for Pin1 recognition and catalysis of c-Myc and other 
multisite, disordered substrates in cell regulation and disease is unclear. By 
nuclear magnetic resonance, surface plasmon resonance, and molecular modeling, 
we show that Pin1 subdomains jointly pre-anchor unphosphorylated c-Myc1-88 in 
the Pin1 interdomain cleft in a disordered, or "fuzzy", complex at the herein 
named Myc Box 0 (MB0) conserved region N-terminal to the highly conserved Myc 
Box I (MBI). Ser62 phosphorylation in MBI intensifies previously transient 
MBI-Pin1 interactions in c-Myc1-88 binding, and increasingly engages Pin1PPIase 
and its catalytic region with maintained MB0 interactions. In cellular assays, 
MB0 mutated c-Myc shows decreased Pin1 interaction, increased protein half-life, 
but lowered rates of Myc-driven transcription and cell proliferation. We propose 
that dynamic Pin1 recognition of MB0 contributes to the regulation of c-Myc 
activity in cells.

Copyright © 2015 Elsevier Ltd. All rights reserved.

DOI: 10.1016/j.str.2015.10.010
PMCID: PMC4807222
PMID: 26655473 [Indexed for MEDLINE]

## Full Text

INTRODUCTION

Among the most fundamental processes in cell biology is the regulation of the cell cycle and, thereby, cell growth. A key player in this process is the peptidyl-prolyl isomerase Pin1 that has been shown to function as a molecular timer by acting as a switch of various cell-signaling processes ( Liou et al., 2011 ; Lu et al., 2007 ). One of the targets of Pin1 is the proto-oncogenic transcription factor c-Myc, which is a universal regulator of cell growth, apoptosis, and proliferation in both normal and tumor cells ( Lin et al., 2012 ; Meyer and Penn, 2008 ; Nie et al., 2012 ). While the pro-proliferative properties of c-Myc are activated by phosphorylation of Ser62, its cellular stability and degradation are controlled by subsequent phosphorylation at Thr58, both sites being located in the conserved Myc Box I (MBI) region of its N-terminal transcriptional activation domain (TAD) ( Hann, 2006 ). The time window of phosphorylation and dephosphorylation at these two sites is affected by Pin1-mediated cis - trans isomerization of the Ser62-Pro motif, thereby regulating specific interactions of c-Myc with modifying kinases, predominantly acting on trans substrates ( Brown et al., 1999 ; Lu et al., 2002a ) and phosphatases, which control the pathway of c-Myc activation and ubiquitin-mediated degradation ( Figure 1A ) ( Farrell and Sears, 2014 ; Sears, 2004 ), and which have been shown to act on cis substrates ( Werner-Allen et al., 2011 ). Pin1 also facilitates the dynamic binding of c-Myc to target gene promoters, enhancing association with transcriptional co-activators and transcriptional activation of target genes, thus potentiating c-Myc’s oncogenic activity ( Farrell et al., 2013 ). The importance of regulatory coupling c-Myc activation with its subsequent degradation in maintaining normal cell growth is evidenced by cancer-associated mutations at or near Thr58 and Ser62 that result in maintaining c-Myc in its activated, Ser62-phosphory-lated state ( Bahram et al., 2000 ; Wang et al., 2011 ), and many regulators of c-Myc that bind to this region are themselves either oncogenes or tumor suppressors ( Tu et al., 2015 ). Although Pin1 promotes c-Myc degradation in normal cells, this activity is uncoupled in cancer cells where both proteins have oncogenic activities and are overexpressed, and Pin1 functions only as a c-Myc co-activator ( Farrell et al., 2013 ). Potential uncoupling mechanisms involve reduced Axin1 function, which scaffolds c-Myc degradation ( Arnold et al., 2009 ; Zhang et al., 2012b ) and Pin1’s downregulation of Fbw7, an E3 ligase controlling degradation of a number of oncoproteins including c-Myc ( Min et al., 2012 ).

Pin1 comprises two independently folded subdomains: Pin1 WW (residues 6–39) and Pin1 PPIase (residues 50–163), connected by a flexible linker ( Bayer et al., 2003 ; Jacobs et al., 2003 ; Ranganathan et al., 1997 ). The catalytic proline isomerization activity toward pSer/pThr-Pro motifs entirely resides in Pin1 PPIase ( Lu et al., 1999 ; Ranganathan et al., 1997 ). The Pin1 PPIase alone binds weakly to native, phosphorylated peptide targets ( K D > 390 μM) ( De et al., 2012 ; Verdecia et al., 2000 ), with significant affinities ( K d range 0.5–90 μM) observed only for peptides selected from library screens ( Duncan et al., 2011 ; Namanja et al., 2011 ; Verdecia et al., 2000 ) or designed inhibitors ( Namanja et al., 2011 ; Zhang et al., 2012a ). Pin1 WW binds pSer/ pThr-Pro-containing peptides with variable affinities ( K d > 7 μM) ( Lu et al., 1999 ; Verdecia et al., 2000 ), preferably in the extended, trans conformation ( De et al., 2012 ; Namanja et al., 2011 ; Verdecia et al., 2000 ; Wintjens et al., 2001 ). In intact Pin1, the two domains create an interdomain cleft, distant from the Pin1 PPIase active site, where phosphorylated target peptides ( K d 5–80 μM) ( Verdecia et al., 2000 ) as well as buffer components such as polyethylene glycol (PEG) bind in well-defined conformations (reviewed in Matena et al., 2013 ). Pin1 WW binding at phosphorylated pSer/pThr-Pro sites has been proposed to increase the local effective concentration of substrate and activity on neighboring phosphorylated sites ( Jacobs et al., 2003 ; Lu et al., 1999 ), to facilitate substrate transfer ( De et al., 2012 ; Lu et al., 1999 ; Wintjens et al., 2001 ), and/or to sequester trans conformations away from the active site, thereby providing directionality to the cis-trans conversion ( De et al., 2012 ; Namanja et al., 2011 ; Wintjens et al., 2001 ). Still, the functional reason for binding of targeted peptides to an interdomain cleft distant from the Pin1 active site remains unresolved.

Recent studies suggest that collaborative and possibly allosteric mechanisms jointly involve both Pin1 domains in target binding (reviewed in Peng, 2015 ). Binding of shorter peptide substrates and small molecules to Pin1 has been shown to affect interdomain mobility and linker dynamics ( Jacobs et al., 2003 ), and increased affinity and isomerization of phosphorylated peptides binding to Pin1 PPIase has been shown in the presence of PEG-induced transient domain interactions ( Matena et al., 2013 ). Interactions between the two domains have been shown to allosterically affect the isomerization activity by an internal dynamic circuit through the Pin1 PPIase interior ( Namanja et al., 2011 ), as well as through residues in the domain interface ( Wilson et al., 2013 ), both recently supported by molecular simulations ( Guo et al., 2015 ). However, to understand how the dual-domain protein Pin1 acts on its longer, multiply phosphorylated, and often intrinsically disordered substrates ( Lu and Zhou, 2007 ), the interaction with such substrates needs to be studied in structural and dynamic detail, but as yet such studies have not been achieved.

We recently characterized the structural and dynamic properties of the most N-terminal part of the c-Myc TAD domain (c-Myc 1–88 ), by nuclear magnetic resonance (NMR) and surface plasmon resonance (SPR), detailing also its dynamic and multivalent interactions with the tumor suppressor Bin1 ( Andresen et al., 2012 ). We found that c-Myc 1–88 contains two transiently ordered regions: the well-characterized MBI region and a less studied, more N-terminal region conserved in c-, N-, and L-Myc ( Cowling and Cole, 2006 ; Legouy et al., 1987 ) ( Figures 1B and 1C ). While MBI is a well-known c-Myc interaction site, and comprises the phosphorylation sites directing c-Myc stability ( Meyer and Penn, 2008 ), so far no clear functional role has been attributed to the transiently ordered region N-terminal to MBI.

Here, we show for the first time how a longer Pin1 substrate, c-Myc 1–88 , interacts with both domains of Pin1, and how this interaction is affected by phosphorylation. By SPR, NMR, and cellular assays, we show that the transiently ordered, unphosphorylated c-Myc region comprising c-Myc residues 13–32, which we henceforth refer to as “Myc Box 0” (MB0; Figures 1B and 1C ), serves as a dynamic anchoring site for Pin1 on c-Myc 1–88 , both in the absence and presence of Ser62 phosphorylation. Molecular simulations, restricted by experimental data, show how Pin1 WW and PPIase domains jointly bind the MB0 region in a dynamic complex, thus facilitating MBI interaction with the Pin1 active site. Our results suggest that Pin1 binding to MB0 affects the cellular activity window of the c-Myc oncoprotein.

DISCUSSION

The Pin1 enzyme is well known for its capability of cis-trans isomerization of phosphorylated (p)Ser/(p)Thr-Pro target sequences, and specifically acts on intrinsically disordered substrates in cell regulation to direct their regulatory activities ( Liou et al., 2011 ; Lu et al., 2007 ). However, limited light has been shed on how Pin1 interacts with longer substrates, since the present molecular knowledge predominantly has been gained from Pin1 studies of interactions with small molecules or peptides. The present work significantly advances our understanding of Pin1-substrate interactions by showing, using a wide spectrum of biophysical and cellular methods, that Pin1 recognition of its well-known target c-Myc involves pre-anchoring to an unphosphorylated conserved motif distal from the phosphorylated target site for Pin1-mediated proline isomerization, which is biologically critical and structurally significant. Specifically, we find that Pin1 binds unphosphorylated c-Myc in a Pin1 interdomain-anchored dynamic, or fuzzy, complex at a well-conserved region here designated MB0 ( Figure 1 ), and we show that such anchoring is structurally compatible with concomitant transient binding to the Pin1 PPIase active site region of the MBI (p)Ser/(p)Thr-Pro Pin1 target site(s) for cis / trans isomerization ( Figure 6 ), in full agreement with our NMR data ( Figures 3 , 4 , and 5 ). Importantly, disturbing Pin1 binding to its unphosphorylated MB0 anchor site results in impaired c-Myc transcriptional activity and reduced c-Myc function in cellular proliferation assays.

A compelling advantage of Pin1 pre-anchoring to unphosphorylated, distal site(s) would be to increase the local concentration of substrate presented to the active site and thereby further increase the catalytic efficiency of Pin1. Our study substantiates this, since we in our NMR experiments directly observe a significant recognition of the unphosphorylated MBI region by intact Pin1 on MB0 binding ( Figure 3 ), indicating that this recognition is facilitated by an increase in local concentration due to Pin1-MB0 pre-anchoring. When c-Myc 1–88 is phosphorylated at Ser62, we find that Pin1 distinctly recognizes the phosphorylation site in MBI but still affects the N-terminal, non-phosphorylated MB0 ( Figure 3 ), suggesting a maintained role of this interaction in the phosphorylated state. Furthermore, the mere localization of MB0 binding to the interdomain cleft ( Figures 5 and 6 ) may also mediate increased catalytic activity. Recent work has shown enhanced interdomain interactions in Pin1 by small-molecule binding at the Pin1 subdomain interface ( Matena et al., 2013 ), and critical interdomain contact residues such as Ile28 have been shown to affect Pin1 catalytic activity and substrate binding affinity ( Wilson et al., 2013 ). Notably, however, simultaneous tight binding of designed bivalent Pin1 binders to both WW and PPIase creates inhibition of Pin1 activity ( Daum et al., 2007 ), suggesting that intrinsic dynamics in binding both Pin1 sites is required for native substrate activity.

Our results show that mutations in the MB0 Pin1-anchoring region dramatically decrease Ras-dependent transformation and simultaneously show prolonged c-Myc half-life. At first glance this may appear confusing, since several stabilizing c-Myc mutations have been shown to increase cell proliferation ( Chang et al., 2000 ; Salghetti et al., 1999 ). However, our results are compatible with recent data demonstrating that Pin1 isomerization at pSer62-Pro63 in c-Myc promotes c-Myc DNA binding and target gene activation ( Figure 1 ; Farell et al., 2013 ) even though it can also facilitate S62-dephosphorylation by protein phosphatase 2A and increase c-Myc turnover ( Figure 1 ; Yeh et al., 2004 ; Arnold and Sears, 2006 ). Thus, a Myc mutant with deficient Pin1 interaction would indeed be more stable and Ser62 phosphorylated, but would not promote transcriptional and/or oncogenic activity. Since our biophysical results indicate that Pin1 binding to the MB0 site N-terminal to the c-Myc pSer62-Pro motif serves to allosterically activate Pin1 and increase the frequency of encounters between the MBI target sequence and the PPIase active site, it is indeed a tenable hypothesis that with disturbed MB0 pre-recognition, both the Pin1 isomerase activity and the affinity between Myc and Pin1 will be too low to be biologically efficient. This would result in a higher prevalence of Myc isomers that are less active in transcriptional activity in the Myc-accessible cellular activity window, with concomitant lowering of cell growth, as shown in our MB0 mutants.

We suggest that the biological requirement for both Pin1 subdomains to reach full Pin1 functionality is based on the biophysical requirements for joint subdomain interaction to longer substrates, which is necessary to allosterically elevate enzymatic activity and specificity to the levels required for efficient biological regulation. Recent studies show that interdomain dynamics is affected by binding of small molecules or substrate peptides/analogs to the active site and/or to the interdomain region ( Bayer et al., 2003 ; Jacobs et al., 2003 ; Matena et al., 2013 ; Namanja et al., 2007 ; Wilson et al., 2013 ). Our data support and extend these findings by showing how Pin1 integrates recognition to both these sites in binding to a longer substrate. Importantly, we show that not only key residues, such as Pin1 WW -Ile28 and its counterpart Ala140 in Pin1 PPIase , but a wide range of interdomain residues in both Pin1 subdomains are differentially affected with respect to the phosphorylation state of the binding peptide ( Figure 5 ). A recent study performed by molecular dynamics simulations proposes that the two Pin1 domains are allosterically regulated through two pathways: a first quiescent state called Path 1, and a second state (Path 2) which is sequentially activated upon substrate binding to the WW pocket ( Guo et al., 2015 ). Interestingly, in our experimental work with a long substrate, we find that the CSPs in Pin1 created by the binding with c-Myc 1–88 resembles the Path 1 pattern, while with pS62-cMyc 1–88 a pattern comparable with Path 2 emerges ( Figure 5 ). Notably, while the WW pocket is involved in binding both phosphorylated and unphosphorylated c-Myc 1–88 , it is only in the presence of pS62-c-Myc 1–88 that the second pathway becomes active. Therefore, we propose that the presence of the phosphate in the active site represents the real trigger factor for the activation of Path 2, given joint binding of the substrate protein to the WW pocket. Thus, the allosteric communication between the two Pin1 domains is not a one- way signal initiated by Pin1 WW binding, but it is a multistep process whereby Path 1 is activated by substrate binding to the WW domain and Path 2 by the phosphate binding the PPIase active site. Taken together, a comparison between results relating to Pin1 allostery in our experimental data and in the molecular dynamics simulations ( Guo et al., 2015 ) reveals interesting similarities, but also different implications concerning the activation mechanism of allosteric pathways.

The highly dynamic association between non-phosphorylated c-Myc 1–88 and Pin1 shows the characteristics of a fuzzy complex ( Fuxreiter and Tompa, 2012 ): alternate conformations are allowed in the bound state, and large parts of the bound c-Myc 1–88 peptide have dynamic properties closely resembling the unbound state. The unphosphorylated Myc-Pin1 complex must therefore be considered as an ensemble of conformations where, although one orientation of c-Myc in the interdomain cleft is prevalent, a wide variety of chain-wrapping modes around Pin1 are allowed in the bound ensemble ( Figures 6 and S5 ). Redistribution among multiple states within the bound ensemble could well occur on Myc phosphorylation, which could also increase the propensities for MBI binding to the Pin1 interdomain cleft, as suggested by Pin1 WW binding to both MB0 and MBI upon Ser62 phosphorylation ( Figure 3E ) as well as by previous crystal structures showing phosphorylated short peptides or peptide analogs bound in the interdomain cleft ( Matena et al., 2013 ; Verdecia et al., 2000 ). The juggling of substrates on and off the active site, as well as the enzymatic efficiency ( De et al., 2012 ), may thus be controlled by dynamically shifting the ensemble of interdomain Pin1 bound states, as indicated by the requirement of both domains for efficient catalysis and by the multiple modes of Myc-Pin1 recognition identified here. Furthermore, the fuzzy complex with a flexible 30-residue (>60 Å) linker between the Myc MB0 and MBI binding sites ( Figure 6 ), together with the dynamics of the bound complex ( Figure 4 ), will allow for facile kinase/phosphatase access to Ser/Thr-Pro sites in MBI with maintained MB0 association of Pin1 throughout the regulatory cycle ( Figure 1 ). Our findings, together with the possible presence of distal motifs in other Pin1-dependent pro-proliferative transcription factors ( Figure S7E ), suggest that dynamic pre-recognition of distal motifs by Pin1 could play a more general role in the timing of cellular events in growth and differentiation.
