---
pmid: '23075851'
title: DAXX envelops a histone H3.3-H4 dimer for H3.3-specific recognition.
authors:
- Elsässer SJ
- Huang H
- Lewis PW
- Chin JW
- Allis CD
- Patel DJ
journal: Nature
year: '2012'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC4056191
doi: 10.1038/nature11608
---

# DAXX envelops a histone H3.3-H4 dimer for H3.3-specific recognition.
**Authors:** Elsässer SJ, Huang H, Lewis PW, Chin JW, Allis CD, Patel DJ
**Journal:** Nature (2012)
**DOI:** [10.1038/nature11608](https://doi.org/10.1038/nature11608)
**PMC:** [PMC4056191](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4056191/)

## Abstract

1. Nature. 2012 Nov 22;491(7425):560-5. doi: 10.1038/nature11608. Epub 2012 Oct
17.

DAXX envelops a histone H3.3-H4 dimer for H3.3-specific recognition.

Elsässer SJ(1), Huang H, Lewis PW, Chin JW, Allis CD, Patel DJ.

Author information:
(1)Laboratory of Chromatin Biology and Epigenetics, The Rockefeller University, 
New York, New York 10065, USA.

Histone chaperones represent a structurally and functionally diverse family of 
histone-binding proteins that prevent promiscuous interactions of histones 
before their assembly into chromatin. DAXX is a metazoan histone chaperone 
specific to the evolutionarily conserved histone variant H3.3. Here we report 
the crystal structures of the DAXX histone-binding domain with a histone H3.3-H4 
dimer, including mutants within DAXX and H3.3, together with in vitro and in 
vivo functional studies that elucidate the principles underlying H3.3 
recognition specificity. Occupying 40% of the histone surface-accessible area, 
DAXX wraps around the H3.3-H4 dimer, with complex formation accompanied by 
structural transitions in the H3.3-H4 histone fold. DAXX uses an extended 
α-helical conformation to compete with major inter-histone, DNA and ASF1 
interaction sites. Our structural studies identify recognition elements that 
read out H3.3-specific residues, and functional studies address the 
contributions of Gly 90 in H3.3 and Glu 225 in DAXX to chaperone-mediated H3.3 
variant recognition specificity.

DOI: 10.1038/nature11608
PMCID: PMC4056191
PMID: 23075851 [Indexed for MEDLINE]

## Full Text

The biology of histone proteins encompasses their synthesis in the cytosol, nuclear import and incorporation into nucleosomes, as well as subsequent eviction from chromatin, redeposition, storage or degradation1,2. Exchange of relatively minor histone variants into distinct, nonrandom genomic locations represents a long-standing mechanism for introducing variation into the chromatin polymer. One compelling example of this mechanism is provided by the mammalian histone H3 family, wherein specialized H3 proteins exist at centromeric (CENPA) and noncentromeric (H3.1, H3.2, H3.3) locations with important distinct functional consequences3. H3.3, for example, has long been correlated with ‘active’ chromatin, but recently has been found associated with other genomic loci, including telomeres4,5. H3.3 enrichment at telomeres and other genomic loci is achieved by specialized chaperone assembly systems, one of which, DAXX, is highlighted in this study.

DAXX is a histone chaperone specific to the evolutionarily conserved histone replacement variant H3.3 (refs 6, 7) that has been shown to deposit H3.3 at heterochromatic loci in vivo in cooperation with the ATP-dependent chromatin remodeller ATRX4,5. Although direct mechanistic links are missing, ATRX/DAXX-mediated heterochromatin maintenance has been implicated in telomere stability, and the suppression of pancreatic neuroendocrine tumours (panNET)8,9 and paediatric glioblastomas10, underscoring the importance of gaining more structural and functional insights into DAXX–H3.3 biology.

Our understanding of the mechanism of histone shuttling, handover between different chaperone systems, and histone transfer onto and off DNA has been hampered by the as-yet-limited number of co-structures of histone–chaperone complexes1,11. Here we report on the crystal structure of the histone-binding domain (HBD) of the histone chaperone DAXX bound to H3.3–H4 dimer, wherein the DAXX HBD wraps around the H3.3–H4 dimer, as well as in vitro and in vivo functional studies to account for the specificity of DAXX for histone variant H3.3.

Structure of the DAXX–H3.3–H4 complexOn the basis of previous biochemical studies7, we identified the minimal HBD of DAXX (residues 178–389) that formed a biochemically stable complex with H3.3–H4. We found that introducing a small number of rigidifying substitutions from the centromeric H3 variant CENPA12,13 into the H3.3 sequence enabled crystallization of this complex. Distinct crystal forms of DAXX–H3.3–H4 complexes with either five (S96A, Y99F, G102A, A111T and M120F) or seven (additional A75C and F84W) rigidifying substitutions in H3.3 (in magenta background, Fig. 1a) yielded structures refined to 1.95 Å and 2.60 Å resolution, respectively (X-ray statistics in Supplementary Table 1). The five- and seven-substituent structures of the DAXX–H3.3–H4 complexes superpose quite well (stereo view in Supplementary Fig. 1b) with a root mean squared deviation of 0.65 Å. Notably, these rigidifying substitutions do not abrogate the formation of a complex with DAXX in vivo (Supplementary Fig. 2a, b, d).The DAXX HBD adopts an all α-helical fold and wraps around the H3.3–H4 dimer through formation of a 1:1:1 ternary complex, thereby burying 4,500 Å2 of surface area (measured using ArealMol in CCP4 program; 40% of the histone surface) in the process (Fig. 1b). The unique variant-specific Ala 87-Ala 88-Ile 89-Gly 90 segment located on α2 helix of H3.3 (in blue background, Fig. 1a) is positioned at the junctional core of H3.3–H4 (boxed segment in red in Fig. 1b) and is surrounded by loop and helical segments in the complex (expanded view in Fig. 1c). Helices α1 and α2 of DAXX form a coiled-coil (hereafter designated as ‘tower’) that packs against a hydrophobic patch of the H3.3 αN helix (Supplementary Fig. 3a, d). The tower helices of DAXX bury a contiguous 1,800 Å2 surface-accessible area spanning αN, α1 and α2 helices and loops of H3.3 (Fig. 1b, c and Supplementary Fig. 1a). The tower helices are followed by a fully ordered 20-residue linker, threading along the DAXX α5 helix and connecting to a helix bundle of α3, α4 and the base of α5. The α5 helix of DAXX is kinked and aligned antiparallel to the α2 helix of H3.3 in the complex (Fig. 1b), thereby tracking another hydrophobic patch on H3.3 (Supplementary Fig. 3b, e). Finally, the α6 helix of DAXX forms an antiparallel coiled-coil pair with the α2 helix of H4 (Fig. 1b) and spans H4 along its amino-terminal half and the interface between H3.3 and H4 along its carboxy-terminal half (Fig. 1b and Supplementary Fig. 3c, f).Highlighting the evolution of DAXX as a histone chaperone, highly conserved residues among metazoan species in the DAXX histone-binding domain cluster along the histone interface (Supplementary Fig. 4a, b). Formation of a DAXX–H3.3–H4 complex seems to require both hydrogen bonding, electrostatic and hydrophobic interactions and is sensitive to single-point Ala mutations of conserved DAXX residues Ser 220, Tyr 222, Arg 251, Phe 317, Arg 328, Asp 331 and Asp 349 (Fig. 1d and Supplementary Fig. 5), as monitored in vivo (Fig. 1e).We note that there is a marked similarity between the antiparallel coiled-coil formed by the α6 helix of DAXX (in magenta) and the α2 helix of H4 (in green) in the DAXX–H3.3–H4 complex (Fig. 1f) with that formed by the sole helix of HJURP (in orange) and α2 helix of CENPA (in dark blue) in the HJURP–CENPA–H4 complex14–16 (Fig. 1g). Remarkably, the observed coiled-coil interactions are mediated by the interfacial DXXLXXXL(X)13VIXKY helical segment common to both chaperones (Fig. 1f, g and Supplementary Fig. 6a, b). Mutations of interfacial H4 residues abrogate in vitro binding of the isolated DAXX α6 helix to H3.3–H4 (Supplementary Fig. 6c), revealing that the α6 helix of DAXX has intrinsic propensity to bind H4 (Fig. 1f), and not H3, as the corresponding helix does in the HJURP complex (Fig. 1g). Although functionally distinct, we therefore conclude that DAXX and HJURP histone chaperones have evolved a common mode of contacting the histone core.

Structure of the DAXX–H3.3–H4 complex

On the basis of previous biochemical studies7, we identified the minimal HBD of DAXX (residues 178–389) that formed a biochemically stable complex with H3.3–H4. We found that introducing a small number of rigidifying substitutions from the centromeric H3 variant CENPA12,13 into the H3.3 sequence enabled crystallization of this complex. Distinct crystal forms of DAXX–H3.3–H4 complexes with either five (S96A, Y99F, G102A, A111T and M120F) or seven (additional A75C and F84W) rigidifying substitutions in H3.3 (in magenta background, Fig. 1a) yielded structures refined to 1.95 Å and 2.60 Å resolution, respectively (X-ray statistics in Supplementary Table 1). The five- and seven-substituent structures of the DAXX–H3.3–H4 complexes superpose quite well (stereo view in Supplementary Fig. 1b) with a root mean squared deviation of 0.65 Å. Notably, these rigidifying substitutions do not abrogate the formation of a complex with DAXX in vivo (Supplementary Fig. 2a, b, d).

The DAXX HBD adopts an all α-helical fold and wraps around the H3.3–H4 dimer through formation of a 1:1:1 ternary complex, thereby burying 4,500 Å2 of surface area (measured using ArealMol in CCP4 program; 40% of the histone surface) in the process (Fig. 1b). The unique variant-specific Ala 87-Ala 88-Ile 89-Gly 90 segment located on α2 helix of H3.3 (in blue background, Fig. 1a) is positioned at the junctional core of H3.3–H4 (boxed segment in red in Fig. 1b) and is surrounded by loop and helical segments in the complex (expanded view in Fig. 1c). Helices α1 and α2 of DAXX form a coiled-coil (hereafter designated as ‘tower’) that packs against a hydrophobic patch of the H3.3 αN helix (Supplementary Fig. 3a, d). The tower helices of DAXX bury a contiguous 1,800 Å2 surface-accessible area spanning αN, α1 and α2 helices and loops of H3.3 (Fig. 1b, c and Supplementary Fig. 1a). The tower helices are followed by a fully ordered 20-residue linker, threading along the DAXX α5 helix and connecting to a helix bundle of α3, α4 and the base of α5. The α5 helix of DAXX is kinked and aligned antiparallel to the α2 helix of H3.3 in the complex (Fig. 1b), thereby tracking another hydrophobic patch on H3.3 (Supplementary Fig. 3b, e). Finally, the α6 helix of DAXX forms an antiparallel coiled-coil pair with the α2 helix of H4 (Fig. 1b) and spans H4 along its amino-terminal half and the interface between H3.3 and H4 along its carboxy-terminal half (Fig. 1b and Supplementary Fig. 3c, f).

Highlighting the evolution of DAXX as a histone chaperone, highly conserved residues among metazoan species in the DAXX histone-binding domain cluster along the histone interface (Supplementary Fig. 4a, b). Formation of a DAXX–H3.3–H4 complex seems to require both hydrogen bonding, electrostatic and hydrophobic interactions and is sensitive to single-point Ala mutations of conserved DAXX residues Ser 220, Tyr 222, Arg 251, Phe 317, Arg 328, Asp 331 and Asp 349 (Fig. 1d and Supplementary Fig. 5), as monitored in vivo (Fig. 1e).

We note that there is a marked similarity between the antiparallel coiled-coil formed by the α6 helix of DAXX (in magenta) and the α2 helix of H4 (in green) in the DAXX–H3.3–H4 complex (Fig. 1f) with that formed by the sole helix of HJURP (in orange) and α2 helix of CENPA (in dark blue) in the HJURP–CENPA–H4 complex14–16 (Fig. 1g). Remarkably, the observed coiled-coil interactions are mediated by the interfacial DXXLXXXL(X)13VIXKY helical segment common to both chaperones (Fig. 1f, g and Supplementary Fig. 6a, b). Mutations of interfacial H4 residues abrogate in vitro binding of the isolated DAXX α6 helix to H3.3–H4 (Supplementary Fig. 6c), revealing that the α6 helix of DAXX has intrinsic propensity to bind H4 (Fig. 1f), and not H3, as the corresponding helix does in the HJURP complex (Fig. 1g). Although functionally distinct, we therefore conclude that DAXX and HJURP histone chaperones have evolved a common mode of contacting the histone core.

DAXX competes with histone–DNA contactsOverlay of the DAXX HBD bound to a H3.3–H4 dimer with that of the histone dimer in the context of the nucleosome (Fig. 2a, b and Supplementary Fig. 7a, b) suggests that the DAXX HBD competes with major histone–DNA contacts. It is readily apparent that α1 and α2 helices of the tower segment of DAXX in the DAXX–H3.3–H4 ternary complex would severely clash with the DNA wrapped around the H3–H4 core (Fig. 2a). Six acidic residues in the L1 loop of DAXX HBD track a basic patch on H3.3 that contacts the DNA within the nucleosome.Furthermore, a shift in the positioning of the αN helix by three residues and an unwinding of the C terminus of the α2 helix in the DAXX complex is apparent (Fig. 2c). The αN helix of H3.3, which forms extensive contacts with the DNA at its entry and exit sites on the nucleosome17, flips over in the DAXX complex by 180°, and is covered by α1 and α2 helices of DAXX in the complex (αN helix in yellow and conformational transition indicated by red arrow in Fig. 2b). The DAXX HBD–H3.3–H4 complex shows no detectable tetrasome assembly activity (Fig. 2d), indicating that other parts of DAXX probably help to mediate the unwrapping of the HBD and release of histone dimer to facilitate chromatin assembly, as previously reported for full-length DAXX in vitro7.In the nucleosome structure, α2 and α3 helices of H3 from opposing H3–H4 dimers constitute the central four-helix bundle formed in the (H3–H4)2 tetramer. At the H3–H3′ interface (coloured in blue for H3 and dark blue for H3′ in Fig. 2e), hydrogen bonds between His 113′ of H3′ and Arg 116 and Asp 123 of H3 (red box, Fig. 2e) stabilize the four-helix bundle. In the DAXX–H3.3–H4 complex, H3.3 undergoes a conformational change (Supplementary Fig. 8a), such that His 113 is retracted by 7 Å without forming new hydrogen-bonding contacts (black boxes, Fig. 2e, f), whereas Arg 116 of H3.3 undergoes 5 Å movement and flips by 180°, whereby it forms a new hydrogen-bonding network with conserved Asp 285 and Asp 288 of DAXX(red box in Fig. 2f; for conserved alignment of Asp residues, see Supplementary Fig. 9). Ile 112 undergoes a small movement and packs against Tyr 286, Gln 314 and Phe 317 of DAXX in the complex (dashed-line black boxes, Fig. 2e, f; see also Supplementary Fig. 8b).The stabilization of the novel conformation of the C-terminal α2-helix segment of H3.3 in the DAXX complex suggests that a H3–H3′ contact in the form of a four-helix bundle is no longer possible on formation of the ternary DAXX–H3.3–H4 complex. Indeed, the central H3–H3′ Cys 110 residues can be crosslinked in the H3.3–H4 tetramer, but not in the presence of DAXX HBD (Fig. 2g). Thus, DAXX, on binding to H3.3–H4, prevents histone tetramer formation.

DAXX competes with histone–DNA contacts

Overlay of the DAXX HBD bound to a H3.3–H4 dimer with that of the histone dimer in the context of the nucleosome (Fig. 2a, b and Supplementary Fig. 7a, b) suggests that the DAXX HBD competes with major histone–DNA contacts. It is readily apparent that α1 and α2 helices of the tower segment of DAXX in the DAXX–H3.3–H4 ternary complex would severely clash with the DNA wrapped around the H3–H4 core (Fig. 2a). Six acidic residues in the L1 loop of DAXX HBD track a basic patch on H3.3 that contacts the DNA within the nucleosome.

Furthermore, a shift in the positioning of the αN helix by three residues and an unwinding of the C terminus of the α2 helix in the DAXX complex is apparent (Fig. 2c). The αN helix of H3.3, which forms extensive contacts with the DNA at its entry and exit sites on the nucleosome17, flips over in the DAXX complex by 180°, and is covered by α1 and α2 helices of DAXX in the complex (αN helix in yellow and conformational transition indicated by red arrow in Fig. 2b). The DAXX HBD–H3.3–H4 complex shows no detectable tetrasome assembly activity (Fig. 2d), indicating that other parts of DAXX probably help to mediate the unwrapping of the HBD and release of histone dimer to facilitate chromatin assembly, as previously reported for full-length DAXX in vitro7.

In the nucleosome structure, α2 and α3 helices of H3 from opposing H3–H4 dimers constitute the central four-helix bundle formed in the (H3–H4)2 tetramer. At the H3–H3′ interface (coloured in blue for H3 and dark blue for H3′ in Fig. 2e), hydrogen bonds between His 113′ of H3′ and Arg 116 and Asp 123 of H3 (red box, Fig. 2e) stabilize the four-helix bundle. In the DAXX–H3.3–H4 complex, H3.3 undergoes a conformational change (Supplementary Fig. 8a), such that His 113 is retracted by 7 Å without forming new hydrogen-bonding contacts (black boxes, Fig. 2e, f), whereas Arg 116 of H3.3 undergoes 5 Å movement and flips by 180°, whereby it forms a new hydrogen-bonding network with conserved Asp 285 and Asp 288 of DAXX(red box in Fig. 2f; for conserved alignment of Asp residues, see Supplementary Fig. 9). Ile 112 undergoes a small movement and packs against Tyr 286, Gln 314 and Phe 317 of DAXX in the complex (dashed-line black boxes, Fig. 2e, f; see also Supplementary Fig. 8b).

The stabilization of the novel conformation of the C-terminal α2-helix segment of H3.3 in the DAXX complex suggests that a H3–H3′ contact in the form of a four-helix bundle is no longer possible on formation of the ternary DAXX–H3.3–H4 complex. Indeed, the central H3–H3′ Cys 110 residues can be crosslinked in the H3.3–H4 tetramer, but not in the presence of DAXX HBD (Fig. 2g). Thus, DAXX, on binding to H3.3–H4, prevents histone tetramer formation.

DAXX and ASF1 compete for H3.3–H4 dimerThe universal H3–H4 dimer chaperone, ASF1/CIA, binds to the H3–H3′ interface through formation of an extended β-sheet alignment (Fig. 3a)18,19. As apparent from a comparison of their complexes with H3–H4, both DAXX and ASF1 chaperones compete for the C-terminal α2-helix segment of H3.3 as well as clashing with adjacent structural elements. In addition, DAXX and ASF1 sequester the C-terminal tail of H4 in distinct conformations (Figs 1b and 3a; C-terminal tail in yellow in black boxed region). In the DAXX complex, the C-terminal tail is buried in a groove formed by the α5 helix of DAXX, α2 helix of H3.3 and α3 helix of H4, and is anchored through a network of hydrogen-bonding interactions (Fig. 3b). Notably, in both DAXX and ASF1 complexes (Fig. 1b and 3a, respectively), Phe 100 of H4 is buried in a hydrophobic pocket (Supplementary Fig. 10a, b, respectively).In accordance with these structural incompatibilities, pull-down experiments in vitro show that ASF1 binds free H3.3–H4, as well as H3.3–H4 from a preformed DAXX–H3.3–H4 complex, but does not pull down the intact DAXX–H3.3–H4 complex (Supplementary Fig. 11). Biochemical purification from HeLa nuclear extract indicates that ASF1 and DAXX do not exist in a common complex with histones in vivo (Supplementary Fig. 12a, b). We therefore wondered if direct competition between histone chaperones contributes to the partitioning of histone H3.3 into distinct histone chaperone complexes. Indeed, reduction of the protein levels of ASF1A and ASF1B by combined RNA interference (RNAi) increased the amount of DAXX that associated with H3.3 in vivo (Fig. 3c, left column). Similarly, a C110E mutation of H3.3 that selectively disrupts ASF1 binding increased the association with DAXX in vivo (Fig. 3d). Conversely, overexpression of ASF1B and, to a marginal extent, ASF1A reduced the levels of DAXX pulled down with H3.3 in vivo (Fig. 3c, right column). In conclusion, formation of DAXX and ASF1 complexes represents concurrent events in vivo.

DAXX and ASF1 compete for H3.3–H4 dimer

The universal H3–H4 dimer chaperone, ASF1/CIA, binds to the H3–H3′ interface through formation of an extended β-sheet alignment (Fig. 3a)18,19. As apparent from a comparison of their complexes with H3–H4, both DAXX and ASF1 chaperones compete for the C-terminal α2-helix segment of H3.3 as well as clashing with adjacent structural elements. In addition, DAXX and ASF1 sequester the C-terminal tail of H4 in distinct conformations (Figs 1b and 3a; C-terminal tail in yellow in black boxed region). In the DAXX complex, the C-terminal tail is buried in a groove formed by the α5 helix of DAXX, α2 helix of H3.3 and α3 helix of H4, and is anchored through a network of hydrogen-bonding interactions (Fig. 3b). Notably, in both DAXX and ASF1 complexes (Fig. 1b and 3a, respectively), Phe 100 of H4 is buried in a hydrophobic pocket (Supplementary Fig. 10a, b, respectively).

In accordance with these structural incompatibilities, pull-down experiments in vitro show that ASF1 binds free H3.3–H4, as well as H3.3–H4 from a preformed DAXX–H3.3–H4 complex, but does not pull down the intact DAXX–H3.3–H4 complex (Supplementary Fig. 11). Biochemical purification from HeLa nuclear extract indicates that ASF1 and DAXX do not exist in a common complex with histones in vivo (Supplementary Fig. 12a, b). We therefore wondered if direct competition between histone chaperones contributes to the partitioning of histone H3.3 into distinct histone chaperone complexes. Indeed, reduction of the protein levels of ASF1A and ASF1B by combined RNA interference (RNAi) increased the amount of DAXX that associated with H3.3 in vivo (Fig. 3c, left column). Similarly, a C110E mutation of H3.3 that selectively disrupts ASF1 binding increased the association with DAXX in vivo (Fig. 3d). Conversely, overexpression of ASF1B and, to a marginal extent, ASF1A reduced the levels of DAXX pulled down with H3.3 in vivo (Fig. 3c, right column). In conclusion, formation of DAXX and ASF1 complexes represents concurrent events in vivo.

Structural basis of H3.3 specificityThe major histone types H3.1/2 differ from variant H3.3 at no more than five positions, with three of them clustered within one short segment at the base of the α2 helix of H3.3, namely Ala 87-Ala-Ile-Gly 90 in H3.3 replaced by Ser 87-Ala-Val-Met 90 in H3.1/2 (Fig. 4a, bottom). We analysed the effect of single point mutations in the N-terminal H3.3 tail (S31A) and the ‘AAIG’ motif (A87S, I89V and G90M) on association with DAXX in vivo; although A87S and I89V had little effect by themselves, G90M reduced DAXX binding by, 50% and concomitantly pulled down CAF1 subunit p150 (Fig. 4a), which normally is exclusively associated with H3.2 (ref. 20). Conversely, H3.2 was not found to pull down detectable amounts of DAXX, but the reciprocal M90G mutation in H3.2 rescued DAXX binding to, 50% of H3.3 levels in vivo (Fig. 4a). Therefore, it seems that Gly 90 is a dominant contributor to chaperone specificity in vivo, with positions Ile 87 and Ser 89 having potentially a synergistic effect with Gly 90.The local environment of the H3.3 AAIG segment in the DAXX HBD–H3.3–H4 complex is shown in Fig. 4b and highlights an extensive hydrogen-bonding network with inclusion of crystallographic water molecules in the vicinity of Gly 90 (Supplementary Fig. 13). Charged or polar side chains from the DAXX tower helices (Gln 206, Tyr 222, Glu 225, Lys 229), as well as residues Gly 337 and Cys 338 from the DAXX L5 linker, were found to line a solvent-filled cavity that is confined by the H3.3 αN and α2 helices. We identify ten bound water molecules that form a total of 26 hydrogen bonds with the main and side chains of DAXX and H3.3 (Fig. 4b).Relying again on the five stabilizing H3.3 substitutions, we solved the crystal structure of DAXX bound to H3.3(G90M)–H4 and refined it to 2.05 Å resolution (X-ray statistics in Supplementary Table 2) and compared alignment of side chains in the vicinity of the H3-variant-specific segment (Fig. 4c). The structure of the G90M-substituted H3.3 complex shows only four bound water molecules and a total of eight hydrogen bonds (Fig. 4c and Supplementary Figs 13b and 14). Importantly, the structure of the G90A-substituted H3.3 complex refined to 2.20 Å resolution (X-ray statistics in Supplementary Table 2) also shows fewer water molecules and hydrogen bonds (Supplementary Fig. 15a). Thus, it is not simply an issue of the larger Met 90 displacing the remaining solvent molecules, as essentially the same effect is observed with the smaller Ala 90 side chain. In conclusion, the integrity of the hydrogen bond network in the vicinity of position 90 seems to favour Gly (in H3.3) over Met (in H3.1/2) or Ala in its ternary complex with H4 and DAXX.

Structural basis of H3.3 specificity

The major histone types H3.1/2 differ from variant H3.3 at no more than five positions, with three of them clustered within one short segment at the base of the α2 helix of H3.3, namely Ala 87-Ala-Ile-Gly 90 in H3.3 replaced by Ser 87-Ala-Val-Met 90 in H3.1/2 (Fig. 4a, bottom). We analysed the effect of single point mutations in the N-terminal H3.3 tail (S31A) and the ‘AAIG’ motif (A87S, I89V and G90M) on association with DAXX in vivo; although A87S and I89V had little effect by themselves, G90M reduced DAXX binding by, 50% and concomitantly pulled down CAF1 subunit p150 (Fig. 4a), which normally is exclusively associated with H3.2 (ref. 20). Conversely, H3.2 was not found to pull down detectable amounts of DAXX, but the reciprocal M90G mutation in H3.2 rescued DAXX binding to, 50% of H3.3 levels in vivo (Fig. 4a). Therefore, it seems that Gly 90 is a dominant contributor to chaperone specificity in vivo, with positions Ile 87 and Ser 89 having potentially a synergistic effect with Gly 90.

The local environment of the H3.3 AAIG segment in the DAXX HBD–H3.3–H4 complex is shown in Fig. 4b and highlights an extensive hydrogen-bonding network with inclusion of crystallographic water molecules in the vicinity of Gly 90 (Supplementary Fig. 13). Charged or polar side chains from the DAXX tower helices (Gln 206, Tyr 222, Glu 225, Lys 229), as well as residues Gly 337 and Cys 338 from the DAXX L5 linker, were found to line a solvent-filled cavity that is confined by the H3.3 αN and α2 helices. We identify ten bound water molecules that form a total of 26 hydrogen bonds with the main and side chains of DAXX and H3.3 (Fig. 4b).

Relying again on the five stabilizing H3.3 substitutions, we solved the crystal structure of DAXX bound to H3.3(G90M)–H4 and refined it to 2.05 Å resolution (X-ray statistics in Supplementary Table 2) and compared alignment of side chains in the vicinity of the H3-variant-specific segment (Fig. 4c). The structure of the G90M-substituted H3.3 complex shows only four bound water molecules and a total of eight hydrogen bonds (Fig. 4c and Supplementary Figs 13b and 14). Importantly, the structure of the G90A-substituted H3.3 complex refined to 2.20 Å resolution (X-ray statistics in Supplementary Table 2) also shows fewer water molecules and hydrogen bonds (Supplementary Fig. 15a). Thus, it is not simply an issue of the larger Met 90 displacing the remaining solvent molecules, as essentially the same effect is observed with the smaller Ala 90 side chain. In conclusion, the integrity of the hydrogen bond network in the vicinity of position 90 seems to favour Gly (in H3.3) over Met (in H3.1/2) or Ala in its ternary complex with H4 and DAXX.

The DAXX tower provides specificity to Gly 90To rationalize how the local perturbation by the H3 variant substitutions affects overall complex stability, we dissected the role of structural elements and individual residues lining the solvent-filled pocket surrounding the H3.3 AAIG motif. Notably, the H3.3 αN helix that packs against both H3.3 α1 and α2 helices and the DAXX tower helices in the DAXX–H3.3–H4 complex (Fig. 5a) is disordered in other known histone chaperone complexes13,17–19, indicating that it is positioned and stabilized by hydrogen-bonding and hydrophobic packing interactions with the DAXX tower (Fig. 5a). The side chains of Ser 57 and Glu 59 of the αN helix of H3.3 participate in a hydrogen bond network with Tyr 222, Glu 225, Lys 229 and Arg 251 of DAXX in the vicinity of key variant residues of H3.3 in the complex (Fig. 5a). The tower helices make important contacts with the H3.3 αN helix, as deletion of residues 1–60 of H3.3, but not 1–43, abrogates binding of the DAXX tower (residues 183–251) and full-length DAXX in vitro (Fig. 5b and Supplementary Fig. 16a, respectively). In vitro binding experiments also show that H3.3 Gln 93 and Glu 94, which appear to anchor the H3.3 α1 and αN helices relative to the central α2 helix (Fig. 4b), are required for tower binding (Supplementary 16b). The tower extends several polar side chains—namely Gln 206, Glu 225 and Lys 229—towards the H3.3 α2 helix that are fully solvated in the wild-type complex (Fig. 4b) but less so in the G90M (Fig. 4c) and G90A (Supplementary Fig. 15a) mutant complexes. Replacement of these side chains with aliphatic side chains disrupts tower binding in vitro (Supplementary Fig. 17), corroborating the importance of hydrogen bonds and polar contacts in this region.In vitro, the DAXX tower alone recapitulates (Fig. 5c) the in vivo pull-down data for H3.3 Gly 90 (Fig. 4a), indicating that it confers at least part of the H3.3 variant specificity to the entire protein. Intriguingly, tower binding to H3.2 could be rescued with a M90G mutation, but not a M90A mutation in vitro (Fig. 5c), suggesting that insertion of any side chain at position 90 and concomitant loss of water molecules can be discriminated by the DAXX tower.

The DAXX tower provides specificity to Gly 90

To rationalize how the local perturbation by the H3 variant substitutions affects overall complex stability, we dissected the role of structural elements and individual residues lining the solvent-filled pocket surrounding the H3.3 AAIG motif. Notably, the H3.3 αN helix that packs against both H3.3 α1 and α2 helices and the DAXX tower helices in the DAXX–H3.3–H4 complex (Fig. 5a) is disordered in other known histone chaperone complexes13,17–19, indicating that it is positioned and stabilized by hydrogen-bonding and hydrophobic packing interactions with the DAXX tower (Fig. 5a). The side chains of Ser 57 and Glu 59 of the αN helix of H3.3 participate in a hydrogen bond network with Tyr 222, Glu 225, Lys 229 and Arg 251 of DAXX in the vicinity of key variant residues of H3.3 in the complex (Fig. 5a). The tower helices make important contacts with the H3.3 αN helix, as deletion of residues 1–60 of H3.3, but not 1–43, abrogates binding of the DAXX tower (residues 183–251) and full-length DAXX in vitro (Fig. 5b and Supplementary Fig. 16a, respectively). In vitro binding experiments also show that H3.3 Gln 93 and Glu 94, which appear to anchor the H3.3 α1 and αN helices relative to the central α2 helix (Fig. 4b), are required for tower binding (Supplementary 16b). The tower extends several polar side chains—namely Gln 206, Glu 225 and Lys 229—towards the H3.3 α2 helix that are fully solvated in the wild-type complex (Fig. 4b) but less so in the G90M (Fig. 4c) and G90A (Supplementary Fig. 15a) mutant complexes. Replacement of these side chains with aliphatic side chains disrupts tower binding in vitro (Supplementary Fig. 17), corroborating the importance of hydrogen bonds and polar contacts in this region.

In vitro, the DAXX tower alone recapitulates (Fig. 5c) the in vivo pull-down data for H3.3 Gly 90 (Fig. 4a), indicating that it confers at least part of the H3.3 variant specificity to the entire protein. Intriguingly, tower binding to H3.2 could be rescued with a M90G mutation, but not a M90A mutation in vitro (Fig. 5c), suggesting that insertion of any side chain at position 90 and concomitant loss of water molecules can be discriminated by the DAXX tower.

Discrimination of G90A/M by DAXX(E225A)In the process of systematically mutating DAXX tower residues coordinating water molecules, we found that K229A and Q206A reduced binding of the tower, whereas E225A did not (Supplementary Fig. 17). Notably, when tested against a set of H3.3 Gly 90 mutants (Fig. 5d and Supplementary Fig. 18), the E225A DAXX tower tolerated various side chains at position 90. Importantly, H3.2 with a single M90G substitution was bound by wild-type, E225A and Y222F DAXX tower (Fig. 5d). The more stringent H3.2 M90A mutant was only bound by the E225A mutant, but not by the K229A and Y222F mutants (Fig. 5d). This unique gain-of-function effect of the E225A mutant suggested a special role for E225A in discriminating side-chain insertions at H3.3 position 90. In the wild-type crystals of the ternary complex, residue Glu 225 of the DAXX tower positions a water molecule (number 10) close to Gly 90 (3.4 Å and 3.5 Å to the backbone Cα and N atoms, respectively) (Fig. 4b), which is evicted in the structures of the ternary complexes of G90M (Fig. 4c) and G90A (Supplementary Fig. 15a) mutants, providing evidence that the coordinated water contributes to the discrimination of the side chain insertions at H3.3 position 90 by DAXX Glu 225.From our in vitro studies on the DAXX tower, we proposed that an E225A mutation would also alleviate the constraints of full-length DAXX for H3.3 Gly 90 in vivo. Indeed, when co-expressed with H3.3 or H3.2 in vivo, the DAXX(E225A) mutant associated with both, although more weakly with H3.2, whereas wild-type DAXX binding was markedly reduced for H3.2 (Fig. 5e). Therefore, removal of the Glu 225 side chain of DAXX induced a gain-of-function effect on binding H3.2 in vivo.To follow up on this hypothesis, we have also solved structures of five-substituent complexes containing the DAXX(E225A) mutant and the dual DAXX(E225A)–H3.3(G90A) mutants refined to 1.95 and 2.20 Å resolution, respectively (Fig. 5f and Supplementary Fig. 15b, respectively; X-ray statistics in Supplementary Table 2). In the E225A mutant structure, the hydration network is minimally perturbed, with a new water, number 11, occupying the same position as that of the Glu 225 carboxylate group (Fig. 5f and Supplementary Fig. 13c, compare with Fig. 4b). Similarly, the hydration network is retained in the structure of the dual E225A/G90A mutant (Supplementary Fig. 15b) relative to the E225A mutant (Fig. 5f). These observations highlight the contribution of water-mediated interactions in molecular recognition and discrimination21 of histone variants by the H3.3–H4 chaperone DAXX.The implications of the structure–function studies reported here promise to be far-reaching in understanding how variation in histone sequence allows functional diversification of histone deposition pathways in maintaining specialized chromatin structure at a wide range of genomic loci, including telomeres.

Discrimination of G90A/M by DAXX(E225A)

In the process of systematically mutating DAXX tower residues coordinating water molecules, we found that K229A and Q206A reduced binding of the tower, whereas E225A did not (Supplementary Fig. 17). Notably, when tested against a set of H3.3 Gly 90 mutants (Fig. 5d and Supplementary Fig. 18), the E225A DAXX tower tolerated various side chains at position 90. Importantly, H3.2 with a single M90G substitution was bound by wild-type, E225A and Y222F DAXX tower (Fig. 5d). The more stringent H3.2 M90A mutant was only bound by the E225A mutant, but not by the K229A and Y222F mutants (Fig. 5d). This unique gain-of-function effect of the E225A mutant suggested a special role for E225A in discriminating side-chain insertions at H3.3 position 90. In the wild-type crystals of the ternary complex, residue Glu 225 of the DAXX tower positions a water molecule (number 10) close to Gly 90 (3.4 Å and 3.5 Å to the backbone Cα and N atoms, respectively) (Fig. 4b), which is evicted in the structures of the ternary complexes of G90M (Fig. 4c) and G90A (Supplementary Fig. 15a) mutants, providing evidence that the coordinated water contributes to the discrimination of the side chain insertions at H3.3 position 90 by DAXX Glu 225.

From our in vitro studies on the DAXX tower, we proposed that an E225A mutation would also alleviate the constraints of full-length DAXX for H3.3 Gly 90 in vivo. Indeed, when co-expressed with H3.3 or H3.2 in vivo, the DAXX(E225A) mutant associated with both, although more weakly with H3.2, whereas wild-type DAXX binding was markedly reduced for H3.2 (Fig. 5e). Therefore, removal of the Glu 225 side chain of DAXX induced a gain-of-function effect on binding H3.2 in vivo.

To follow up on this hypothesis, we have also solved structures of five-substituent complexes containing the DAXX(E225A) mutant and the dual DAXX(E225A)–H3.3(G90A) mutants refined to 1.95 and 2.20 Å resolution, respectively (Fig. 5f and Supplementary Fig. 15b, respectively; X-ray statistics in Supplementary Table 2). In the E225A mutant structure, the hydration network is minimally perturbed, with a new water, number 11, occupying the same position as that of the Glu 225 carboxylate group (Fig. 5f and Supplementary Fig. 13c, compare with Fig. 4b). Similarly, the hydration network is retained in the structure of the dual E225A/G90A mutant (Supplementary Fig. 15b) relative to the E225A mutant (Fig. 5f). These observations highlight the contribution of water-mediated interactions in molecular recognition and discrimination21 of histone variants by the H3.3–H4 chaperone DAXX.

The implications of the structure–function studies reported here promise to be far-reaching in understanding how variation in histone sequence allows functional diversification of histone deposition pathways in maintaining specialized chromatin structure at a wide range of genomic loci, including telomeres.

METHODSProtein expression and purificationFor preparation of histone H3.3–H4 tetramers and DAXX HBD–H3.3–H4 complexes in vitro, human DAXX fragment 183–398 or 183–417, human histone H3.3 and codon-optimized H4 (gift from S. Yokoyama) was cloned into pRUTH5 to yield an N-terminal His6-TEV tag (courtesy of A. Ruthenburg). Single or multiple point mutations were introduced by standard mutagenesis PCR procedures. Histones and DAXX HBD were individually expressed in BL21 Star(DE3) cells (Invitrogen) into inclusion bodies for 4–6 h at 37 °C. Inclusion bodies were resolubilized in 6 M guanidine-HCl, 1 M NaCl, 50 mM Tris-HCl pH 8, and purified on a Ni-NTA affinity column. To prepare H3–H4 tetramers, equimolar ratios of H3 and H4 were mixed in refolding buffer (4 M guanidine, 50 mM MOPS pH 7, 1 M NaCl, 5 mM EDTA, 15% glycerol) and dialysed against 50 mM MOPS pH 7, 1 M NaCl, 1 mM EDTA, 10% glycerol.For purification of the native DAXX HBD–H3.3–H4 complex, DAXX (178–389) was cloned into a modified RSFDuet-1 vector (Novagen), with an N-terminal His6-SUMO tag and human histone H3.3 and H4 were cloned into pETDuet-1 vector (Novagen). Either five (S96A, Y99F, G102A, A111T and M120F) or seven (additional A75C and F84W) rigidifying substitutions in H3.3 were introduced by standard mutagenesis PCR procedure. These two plasmids were transformed into BL21 (DE3) RIL cell strain (Stratagene) for co-expression. The cells were grown in LB medium at 37 °C to an OD600 nm value about 1.0 and induced with 1 mM isopropyl β-d-1-thiogalactopyranoside (IPTG) for another 4 h. The DAXX–H3.3–H4 protein complex was purified first on a Ni-NTA affinity column. After removing the SUMO tag, the resulting complex was loaded on a cation exchange HiTrap SP FF column and further purified by gel filtration with a HiLoad 16/600 Superdex 200 column (GE Healthcare). The final high-purity complex was concentrated to 50–100 mg ml−1 in a pH7.5 buffer containing 20 mM Tris-HCl, 1 M NaCl and 2 mM DTT. The DAXX–H3.3(G90M)–H4, DAXX–H3.3(G90A)–H4, DAXX(E225A)–H3.3–H4 and DAXX(E225A)–H3.3(G90A)–H4 complexes were purified as described above.CrystallizationTo yield initial crystals for structure determination, DAXX HBD–H3.3–H4 was refolded in vitro from equimolar amounts of DAXX HBD (residues 183–398), H3 containing seven substituents (A75C, F84W, S96A, Y99F, G102A, A111T and M120F) and H4 (residues 20–100), mixed in refolding buffer, by dialysis against 50 mM MOPS pH 7, 1 M NaCl, 1 mM EDTA, 10% glycerol. After insoluble precipitate was removed by centrifugation, the complex was treated with TEV protease (1% w/w), concentrated to 20 mg ml−1 and purified on a 16/60 Superdex 200 size exclusion column, equilibrated and run in crystal screening buffer (10 mM MOPS pH 7, 500 mM NaCl, 0.1 mM EDTA, 0.1 mM PMSF, 1 mM TCEP). Peak fractions were pooled and concentrated to 50 mg ml−1 in spin concentrators (Amicon Ultra, Millipore).The 2.6 Å diffracting crystal of the DAXX–H3.3–H4 containing seven substituents formed in sitting-drop vapour-diffusion setups with 2.5 M NaCl, 150 mM Na/K-phosphate pH 6.2, 5% glycerol after a few days at 20 °C.For crystals of the five substituent (S96A, Y99F, G102A, A111T and M120F) H3.3 and all additional mutants, the respective DAXX–H3.3–H4 complex at a concentration of 52 mg ml−1 was used for sparse-matrix and grid-search screens (Hampton Research and Qiagen) and an initial condition (1.8 M Na/K-phosphate, pH 6.9) was identified. The crystals were subsequently reproduced and improved in 1.8 M Na/K-phosphate, pH 7.0 using the hanging-drop vapour-diffusion method at 4 °C. It took more than 1 month for the crystals to grow to a suitable size. The crystals of DAXX–H3.3(G90M)–H4, DAXX–H3.3(G90A)–H4, DAXX(E225A)–H3.3–H4 and DAXX(E225A)–H3.3(G90A)–H4 were micro-seeded with crystals from the DAXX–H3.3–H4 complex and grown in 1.8 M Na/K-phosphate, pH 6.8 or 7.0, at 4 °C. It took more than two months for the crystals to grow to suitable size for data collection. All the crystals were soaked in a cryoprotectant made from mother liquor supplemented with 20% glycerol, before flash freezing in liquid nitrogen.Structure determinationThe data sets for DAXX–H3.3–H4 were collected at 1.075 Å on beamline X29 (Brookhaven NSLS), whereas the data sets for DAXX–H3.3(G90M)–H4, DAXX–H3.3(G90A)–H4, DAXX(E225A)–H3.3–H4 and DAXX(E225A)–H3.3(G90A)–H4 were collected at 0.979 Å on 24-ID-C/E NE-CAT beamline (Advanced Photo Source, Argonne National Laboratory). All the data sets were processed by using the HKL 2000 program. The initial structure for DAXX–H3.3–H4 was solved by molecular replacement program BALBES22 with a final search model based on Protein Data Bank 2HUE and manually refined and built using Coot23 and Refmac524 in CCP4 program suite25. The final structures of the five-substituent and seven-substituent ternary complexes were refined to 1.95 Å and 2.60 Å resolution respectively using PHENIX26. Supplementary Table 1 summarizes the statistics for data collection and structural refinement. The structure of DAXX–H3.3(G90M)–H4, DAXX–H3.3(G90A)–H4, DAXX(E225A)–H3.3–H4 and DAXX(E225A)–H3.3(G90A)–H4 were solved by molecular replacement in PHASER27 using the structure of DAXX–H3.3–H4 as a search model. Supplementary Table 2 summarizes the statistics for data collection and structural refinement for the mutant ternary complexes.Mononucleosome assembly assayTwofold dilutions of 8 pmol of H3.3–H4 or DAXX HBD–H3.3–H4 was mixed with 2 pmol 84-bp DNA (derived from the Widom 601 sequence) in 20 µl assembly buffer AB1 (10 mM HEPES pH 7.5, 40 mM KCl, 60 mM NaCl, 2 mM MgCl, 0.5 mM EGTA). Reaction was incubated for 3 h at room temperature and an aliquot analysed on a 6% PAGE in 0.5× TBE.GST pull downsFull-length ASF1A, a DAXX fragment spanning the tower helices (residues 183–251), and the α6 helix (residues 349–384) were cloned into a pET-based vector with N-terminal GST-tag (pRUTH2, courtesy of A. Ruthenburg). GST fusion proteins were expressed in Rosetta2(DE3)pLysS cells (Novagen) at 25 °C for 6 h, purified using BugBuster protein extraction reagent (Novagen) and magnetic glutathione resin (Pierce). Pull downs were carried out on 2 µl of magnetic bead slurry saturated with GST-fusion proteins and washed in binding buffer (20 mM phosphate buffer pH7.4, 1 M NaCl, 10% glycerol, 0.01% Triton X-100, 0.01% CHAPS, 1 mM DTT). Wild-type histone H3.3–H4 and H3.2–H4 tetramers, as well as all mutants, were prepared and purified as described above. 20 µg (at least tenfold excess) of H3–H4 tetramers in 500 µl binding buffer was added to the immobilized DAXX fusion proteins and incubated for 10 min. Beads were separated from solution and washed quickly with three times 1 ml of binding buffer before eluting in 20 µl of binding buffer supplemented with 25 mM fresh glutathione.Cysteine chemical crosslinkingBismaleimidoethane (BMOE, Pierce 22323) was dissolved at 20 mM concentration in DMSO. In each reaction, 10 µM H3.3–H4 dimers or DAXX HBD–H3.3–H4 trimeric complex were present in 100 µl reaction volume. The reaction buffer was 20 mM MOPS pH 7, 150 mM NaCl, 1 mM EDTA, 0.5 mM TCEP. BMOE was rapidly added from stock solution to 50 µM concentration. Reactions were incubated for 30 min at room temperature, and stopped by adding 6× laemmli sample buffer including 1% DTT, boiled and run on a SDS–PAGE.Transient transfection of 293T and HeLa cells and co-immunoprecipitationHuman histone H3.3 and mutants, as well as ASF1A and ASF1B, were cloned into a pCDH vector. Full-length human DAXX was cloned into pRK5 with an N-terminal Flag epitope tag. For transfection, 293T cells were seeded in 6-well plates at 2 × 105 cells per well in DMEM supplemented with 10% FBS, 1× penicillin/streptomycin. After 12–24 h, cells were transfected with 2.5 µg plasmid with Transit LT1 reagent (Mirus) according to the manufacturers manual. Cell were collected after 48 h by scraping and tituration in warm DMEM medium and pelleted. Cell pellet was washed with PBS and subsequently with hypotonic buffer (15 mM HEPES pH 7.5, 30 mM KCl, 5 mM MgCl2). For lysis, cell pellet was re-suspended in 250 µl hypotonic buffer supplemented with 0.02% Nonident P-40, 0.8 mM PMSF, 1× EDTA-free Protease Inhibitor Cocktail (Roche), 10 µg ml−1 RNase I. Lysis was allowed to proceed for 15 min with rotation at 4 °C. 22 µl 5 M NaCl was added (final 400 mM NaCl) under agitation to extract nuclear proteins. Extract was rotated for 15 min at 4 °C and spun 30 min at 4 °C at 20,000g. Supernatant was incubated with 25 µl anti-Flag Affinity gel (Sigma-Aldrich) for 4 h at 4 °C. Beads were washed 2× 5 min with hypotonic buffer plus 400 mM NaCl. Bound proteins were eluted with 1× laemmli SDS sample buffer (63 mM Tris-HCl pH 6.8, 10% glycerol, 2% SDS). DTT was added to eluate to 1% before boiling and running on an SDS PAGE.RNAiHeLa cells stably expressing Flag-HA-tagged H3.3 and H3.1 were a gift from G. Almouzni20 siRNAs against ASF1A/B were ordered from Dharmacon, from published sequences (ASF1A, 5′-AAGUGAAGAAUACGAUCAAGU(dTdT)-3′, ASF1B, 5′-AACAACGAGUACCUCAACCCU(dTdT)-3′, (refs 28, 29). HeLa cells were transfected in 6-well plates with Oligofectamin (Invitrogen) according to the manufacturer’s protocol and collected after 72 h as described for transient transfections above.Co-transfection of DAXX and histone mutantsWild-type pRK5-Flag–hDAXX or hDAXX point mutants were transiently transfected into 293T cells for 48 h (30 µg DNA per 2.0 × 107 cells). Cells were collected and lysed (20 mM HEPES, pH 7.9, 1 mM EDTA, 0.1% Triton X-100, 500 mM KCl, 2 mM 2-mercaptoethanol, 0.8 mM PMSF, 2× Protease inhibitor cocktail (Roche)) before Flag-M2 immunoprecipitation. Immunoprecipitated material was washed extensively (6× 1 ml of 20 mM HEPES, pH 7.9, 1 mM EDTA, 0.1% Triton X-100, 1 M KCl, 2 mM 2-mercaptoethanol, 0.4 mM PMSF) before elution (70 mM glycine, pH 2.5, 150 mM NaCl). pRK5-Flag–hDAXX(E225A) (8 µg DNA per 2.0 × 107 cells) and pCDH-HA–H3.3 or pCDH-HA–H3.2 (2 µg DNA per 2.0 × 107) were co-transfected into 293T cells for DAXX–H3.3–H4 co-immunoprecipitations.

Protein expression and purificationFor preparation of histone H3.3–H4 tetramers and DAXX HBD–H3.3–H4 complexes in vitro, human DAXX fragment 183–398 or 183–417, human histone H3.3 and codon-optimized H4 (gift from S. Yokoyama) was cloned into pRUTH5 to yield an N-terminal His6-TEV tag (courtesy of A. Ruthenburg). Single or multiple point mutations were introduced by standard mutagenesis PCR procedures. Histones and DAXX HBD were individually expressed in BL21 Star(DE3) cells (Invitrogen) into inclusion bodies for 4–6 h at 37 °C. Inclusion bodies were resolubilized in 6 M guanidine-HCl, 1 M NaCl, 50 mM Tris-HCl pH 8, and purified on a Ni-NTA affinity column. To prepare H3–H4 tetramers, equimolar ratios of H3 and H4 were mixed in refolding buffer (4 M guanidine, 50 mM MOPS pH 7, 1 M NaCl, 5 mM EDTA, 15% glycerol) and dialysed against 50 mM MOPS pH 7, 1 M NaCl, 1 mM EDTA, 10% glycerol.For purification of the native DAXX HBD–H3.3–H4 complex, DAXX (178–389) was cloned into a modified RSFDuet-1 vector (Novagen), with an N-terminal His6-SUMO tag and human histone H3.3 and H4 were cloned into pETDuet-1 vector (Novagen). Either five (S96A, Y99F, G102A, A111T and M120F) or seven (additional A75C and F84W) rigidifying substitutions in H3.3 were introduced by standard mutagenesis PCR procedure. These two plasmids were transformed into BL21 (DE3) RIL cell strain (Stratagene) for co-expression. The cells were grown in LB medium at 37 °C to an OD600 nm value about 1.0 and induced with 1 mM isopropyl β-d-1-thiogalactopyranoside (IPTG) for another 4 h. The DAXX–H3.3–H4 protein complex was purified first on a Ni-NTA affinity column. After removing the SUMO tag, the resulting complex was loaded on a cation exchange HiTrap SP FF column and further purified by gel filtration with a HiLoad 16/600 Superdex 200 column (GE Healthcare). The final high-purity complex was concentrated to 50–100 mg ml−1 in a pH7.5 buffer containing 20 mM Tris-HCl, 1 M NaCl and 2 mM DTT. The DAXX–H3.3(G90M)–H4, DAXX–H3.3(G90A)–H4, DAXX(E225A)–H3.3–H4 and DAXX(E225A)–H3.3(G90A)–H4 complexes were purified as described above.

Protein expression and purification

For preparation of histone H3.3–H4 tetramers and DAXX HBD–H3.3–H4 complexes in vitro, human DAXX fragment 183–398 or 183–417, human histone H3.3 and codon-optimized H4 (gift from S. Yokoyama) was cloned into pRUTH5 to yield an N-terminal His6-TEV tag (courtesy of A. Ruthenburg). Single or multiple point mutations were introduced by standard mutagenesis PCR procedures. Histones and DAXX HBD were individually expressed in BL21 Star(DE3) cells (Invitrogen) into inclusion bodies for 4–6 h at 37 °C. Inclusion bodies were resolubilized in 6 M guanidine-HCl, 1 M NaCl, 50 mM Tris-HCl pH 8, and purified on a Ni-NTA affinity column. To prepare H3–H4 tetramers, equimolar ratios of H3 and H4 were mixed in refolding buffer (4 M guanidine, 50 mM MOPS pH 7, 1 M NaCl, 5 mM EDTA, 15% glycerol) and dialysed against 50 mM MOPS pH 7, 1 M NaCl, 1 mM EDTA, 10% glycerol.

For purification of the native DAXX HBD–H3.3–H4 complex, DAXX (178–389) was cloned into a modified RSFDuet-1 vector (Novagen), with an N-terminal His6-SUMO tag and human histone H3.3 and H4 were cloned into pETDuet-1 vector (Novagen). Either five (S96A, Y99F, G102A, A111T and M120F) or seven (additional A75C and F84W) rigidifying substitutions in H3.3 were introduced by standard mutagenesis PCR procedure. These two plasmids were transformed into BL21 (DE3) RIL cell strain (Stratagene) for co-expression. The cells were grown in LB medium at 37 °C to an OD600 nm value about 1.0 and induced with 1 mM isopropyl β-d-1-thiogalactopyranoside (IPTG) for another 4 h. The DAXX–H3.3–H4 protein complex was purified first on a Ni-NTA affinity column. After removing the SUMO tag, the resulting complex was loaded on a cation exchange HiTrap SP FF column and further purified by gel filtration with a HiLoad 16/600 Superdex 200 column (GE Healthcare). The final high-purity complex was concentrated to 50–100 mg ml−1 in a pH7.5 buffer containing 20 mM Tris-HCl, 1 M NaCl and 2 mM DTT. The DAXX–H3.3(G90M)–H4, DAXX–H3.3(G90A)–H4, DAXX(E225A)–H3.3–H4 and DAXX(E225A)–H3.3(G90A)–H4 complexes were purified as described above.

CrystallizationTo yield initial crystals for structure determination, DAXX HBD–H3.3–H4 was refolded in vitro from equimolar amounts of DAXX HBD (residues 183–398), H3 containing seven substituents (A75C, F84W, S96A, Y99F, G102A, A111T and M120F) and H4 (residues 20–100), mixed in refolding buffer, by dialysis against 50 mM MOPS pH 7, 1 M NaCl, 1 mM EDTA, 10% glycerol. After insoluble precipitate was removed by centrifugation, the complex was treated with TEV protease (1% w/w), concentrated to 20 mg ml−1 and purified on a 16/60 Superdex 200 size exclusion column, equilibrated and run in crystal screening buffer (10 mM MOPS pH 7, 500 mM NaCl, 0.1 mM EDTA, 0.1 mM PMSF, 1 mM TCEP). Peak fractions were pooled and concentrated to 50 mg ml−1 in spin concentrators (Amicon Ultra, Millipore).The 2.6 Å diffracting crystal of the DAXX–H3.3–H4 containing seven substituents formed in sitting-drop vapour-diffusion setups with 2.5 M NaCl, 150 mM Na/K-phosphate pH 6.2, 5% glycerol after a few days at 20 °C.For crystals of the five substituent (S96A, Y99F, G102A, A111T and M120F) H3.3 and all additional mutants, the respective DAXX–H3.3–H4 complex at a concentration of 52 mg ml−1 was used for sparse-matrix and grid-search screens (Hampton Research and Qiagen) and an initial condition (1.8 M Na/K-phosphate, pH 6.9) was identified. The crystals were subsequently reproduced and improved in 1.8 M Na/K-phosphate, pH 7.0 using the hanging-drop vapour-diffusion method at 4 °C. It took more than 1 month for the crystals to grow to a suitable size. The crystals of DAXX–H3.3(G90M)–H4, DAXX–H3.3(G90A)–H4, DAXX(E225A)–H3.3–H4 and DAXX(E225A)–H3.3(G90A)–H4 were micro-seeded with crystals from the DAXX–H3.3–H4 complex and grown in 1.8 M Na/K-phosphate, pH 6.8 or 7.0, at 4 °C. It took more than two months for the crystals to grow to suitable size for data collection. All the crystals were soaked in a cryoprotectant made from mother liquor supplemented with 20% glycerol, before flash freezing in liquid nitrogen.

Crystallization

To yield initial crystals for structure determination, DAXX HBD–H3.3–H4 was refolded in vitro from equimolar amounts of DAXX HBD (residues 183–398), H3 containing seven substituents (A75C, F84W, S96A, Y99F, G102A, A111T and M120F) and H4 (residues 20–100), mixed in refolding buffer, by dialysis against 50 mM MOPS pH 7, 1 M NaCl, 1 mM EDTA, 10% glycerol. After insoluble precipitate was removed by centrifugation, the complex was treated with TEV protease (1% w/w), concentrated to 20 mg ml−1 and purified on a 16/60 Superdex 200 size exclusion column, equilibrated and run in crystal screening buffer (10 mM MOPS pH 7, 500 mM NaCl, 0.1 mM EDTA, 0.1 mM PMSF, 1 mM TCEP). Peak fractions were pooled and concentrated to 50 mg ml−1 in spin concentrators (Amicon Ultra, Millipore).

The 2.6 Å diffracting crystal of the DAXX–H3.3–H4 containing seven substituents formed in sitting-drop vapour-diffusion setups with 2.5 M NaCl, 150 mM Na/K-phosphate pH 6.2, 5% glycerol after a few days at 20 °C.

For crystals of the five substituent (S96A, Y99F, G102A, A111T and M120F) H3.3 and all additional mutants, the respective DAXX–H3.3–H4 complex at a concentration of 52 mg ml−1 was used for sparse-matrix and grid-search screens (Hampton Research and Qiagen) and an initial condition (1.8 M Na/K-phosphate, pH 6.9) was identified. The crystals were subsequently reproduced and improved in 1.8 M Na/K-phosphate, pH 7.0 using the hanging-drop vapour-diffusion method at 4 °C. It took more than 1 month for the crystals to grow to a suitable size. The crystals of DAXX–H3.3(G90M)–H4, DAXX–H3.3(G90A)–H4, DAXX(E225A)–H3.3–H4 and DAXX(E225A)–H3.3(G90A)–H4 were micro-seeded with crystals from the DAXX–H3.3–H4 complex and grown in 1.8 M Na/K-phosphate, pH 6.8 or 7.0, at 4 °C. It took more than two months for the crystals to grow to suitable size for data collection. All the crystals were soaked in a cryoprotectant made from mother liquor supplemented with 20% glycerol, before flash freezing in liquid nitrogen.

Structure determinationThe data sets for DAXX–H3.3–H4 were collected at 1.075 Å on beamline X29 (Brookhaven NSLS), whereas the data sets for DAXX–H3.3(G90M)–H4, DAXX–H3.3(G90A)–H4, DAXX(E225A)–H3.3–H4 and DAXX(E225A)–H3.3(G90A)–H4 were collected at 0.979 Å on 24-ID-C/E NE-CAT beamline (Advanced Photo Source, Argonne National Laboratory). All the data sets were processed by using the HKL 2000 program. The initial structure for DAXX–H3.3–H4 was solved by molecular replacement program BALBES22 with a final search model based on Protein Data Bank 2HUE and manually refined and built using Coot23 and Refmac524 in CCP4 program suite25. The final structures of the five-substituent and seven-substituent ternary complexes were refined to 1.95 Å and 2.60 Å resolution respectively using PHENIX26. Supplementary Table 1 summarizes the statistics for data collection and structural refinement. The structure of DAXX–H3.3(G90M)–H4, DAXX–H3.3(G90A)–H4, DAXX(E225A)–H3.3–H4 and DAXX(E225A)–H3.3(G90A)–H4 were solved by molecular replacement in PHASER27 using the structure of DAXX–H3.3–H4 as a search model. Supplementary Table 2 summarizes the statistics for data collection and structural refinement for the mutant ternary complexes.

Structure determination

The data sets for DAXX–H3.3–H4 were collected at 1.075 Å on beamline X29 (Brookhaven NSLS), whereas the data sets for DAXX–H3.3(G90M)–H4, DAXX–H3.3(G90A)–H4, DAXX(E225A)–H3.3–H4 and DAXX(E225A)–H3.3(G90A)–H4 were collected at 0.979 Å on 24-ID-C/E NE-CAT beamline (Advanced Photo Source, Argonne National Laboratory). All the data sets were processed by using the HKL 2000 program. The initial structure for DAXX–H3.3–H4 was solved by molecular replacement program BALBES22 with a final search model based on Protein Data Bank 2HUE and manually refined and built using Coot23 and Refmac524 in CCP4 program suite25. The final structures of the five-substituent and seven-substituent ternary complexes were refined to 1.95 Å and 2.60 Å resolution respectively using PHENIX26. Supplementary Table 1 summarizes the statistics for data collection and structural refinement. The structure of DAXX–H3.3(G90M)–H4, DAXX–H3.3(G90A)–H4, DAXX(E225A)–H3.3–H4 and DAXX(E225A)–H3.3(G90A)–H4 were solved by molecular replacement in PHASER27 using the structure of DAXX–H3.3–H4 as a search model. Supplementary Table 2 summarizes the statistics for data collection and structural refinement for the mutant ternary complexes.

Mononucleosome assembly assayTwofold dilutions of 8 pmol of H3.3–H4 or DAXX HBD–H3.3–H4 was mixed with 2 pmol 84-bp DNA (derived from the Widom 601 sequence) in 20 µl assembly buffer AB1 (10 mM HEPES pH 7.5, 40 mM KCl, 60 mM NaCl, 2 mM MgCl, 0.5 mM EGTA). Reaction was incubated for 3 h at room temperature and an aliquot analysed on a 6% PAGE in 0.5× TBE.

Mononucleosome assembly assay

Twofold dilutions of 8 pmol of H3.3–H4 or DAXX HBD–H3.3–H4 was mixed with 2 pmol 84-bp DNA (derived from the Widom 601 sequence) in 20 µl assembly buffer AB1 (10 mM HEPES pH 7.5, 40 mM KCl, 60 mM NaCl, 2 mM MgCl, 0.5 mM EGTA). Reaction was incubated for 3 h at room temperature and an aliquot analysed on a 6% PAGE in 0.5× TBE.

GST pull downsFull-length ASF1A, a DAXX fragment spanning the tower helices (residues 183–251), and the α6 helix (residues 349–384) were cloned into a pET-based vector with N-terminal GST-tag (pRUTH2, courtesy of A. Ruthenburg). GST fusion proteins were expressed in Rosetta2(DE3)pLysS cells (Novagen) at 25 °C for 6 h, purified using BugBuster protein extraction reagent (Novagen) and magnetic glutathione resin (Pierce). Pull downs were carried out on 2 µl of magnetic bead slurry saturated with GST-fusion proteins and washed in binding buffer (20 mM phosphate buffer pH7.4, 1 M NaCl, 10% glycerol, 0.01% Triton X-100, 0.01% CHAPS, 1 mM DTT). Wild-type histone H3.3–H4 and H3.2–H4 tetramers, as well as all mutants, were prepared and purified as described above. 20 µg (at least tenfold excess) of H3–H4 tetramers in 500 µl binding buffer was added to the immobilized DAXX fusion proteins and incubated for 10 min. Beads were separated from solution and washed quickly with three times 1 ml of binding buffer before eluting in 20 µl of binding buffer supplemented with 25 mM fresh glutathione.

GST pull downs

Full-length ASF1A, a DAXX fragment spanning the tower helices (residues 183–251), and the α6 helix (residues 349–384) were cloned into a pET-based vector with N-terminal GST-tag (pRUTH2, courtesy of A. Ruthenburg). GST fusion proteins were expressed in Rosetta2(DE3)pLysS cells (Novagen) at 25 °C for 6 h, purified using BugBuster protein extraction reagent (Novagen) and magnetic glutathione resin (Pierce). Pull downs were carried out on 2 µl of magnetic bead slurry saturated with GST-fusion proteins and washed in binding buffer (20 mM phosphate buffer pH7.4, 1 M NaCl, 10% glycerol, 0.01% Triton X-100, 0.01% CHAPS, 1 mM DTT). Wild-type histone H3.3–H4 and H3.2–H4 tetramers, as well as all mutants, were prepared and purified as described above. 20 µg (at least tenfold excess) of H3–H4 tetramers in 500 µl binding buffer was added to the immobilized DAXX fusion proteins and incubated for 10 min. Beads were separated from solution and washed quickly with three times 1 ml of binding buffer before eluting in 20 µl of binding buffer supplemented with 25 mM fresh glutathione.

Cysteine chemical crosslinkingBismaleimidoethane (BMOE, Pierce 22323) was dissolved at 20 mM concentration in DMSO. In each reaction, 10 µM H3.3–H4 dimers or DAXX HBD–H3.3–H4 trimeric complex were present in 100 µl reaction volume. The reaction buffer was 20 mM MOPS pH 7, 150 mM NaCl, 1 mM EDTA, 0.5 mM TCEP. BMOE was rapidly added from stock solution to 50 µM concentration. Reactions were incubated for 30 min at room temperature, and stopped by adding 6× laemmli sample buffer including 1% DTT, boiled and run on a SDS–PAGE.

Cysteine chemical crosslinking

Bismaleimidoethane (BMOE, Pierce 22323) was dissolved at 20 mM concentration in DMSO. In each reaction, 10 µM H3.3–H4 dimers or DAXX HBD–H3.3–H4 trimeric complex were present in 100 µl reaction volume. The reaction buffer was 20 mM MOPS pH 7, 150 mM NaCl, 1 mM EDTA, 0.5 mM TCEP. BMOE was rapidly added from stock solution to 50 µM concentration. Reactions were incubated for 30 min at room temperature, and stopped by adding 6× laemmli sample buffer including 1% DTT, boiled and run on a SDS–PAGE.

Transient transfection of 293T and HeLa cells and co-immunoprecipitationHuman histone H3.3 and mutants, as well as ASF1A and ASF1B, were cloned into a pCDH vector. Full-length human DAXX was cloned into pRK5 with an N-terminal Flag epitope tag. For transfection, 293T cells were seeded in 6-well plates at 2 × 105 cells per well in DMEM supplemented with 10% FBS, 1× penicillin/streptomycin. After 12–24 h, cells were transfected with 2.5 µg plasmid with Transit LT1 reagent (Mirus) according to the manufacturers manual. Cell were collected after 48 h by scraping and tituration in warm DMEM medium and pelleted. Cell pellet was washed with PBS and subsequently with hypotonic buffer (15 mM HEPES pH 7.5, 30 mM KCl, 5 mM MgCl2). For lysis, cell pellet was re-suspended in 250 µl hypotonic buffer supplemented with 0.02% Nonident P-40, 0.8 mM PMSF, 1× EDTA-free Protease Inhibitor Cocktail (Roche), 10 µg ml−1 RNase I. Lysis was allowed to proceed for 15 min with rotation at 4 °C. 22 µl 5 M NaCl was added (final 400 mM NaCl) under agitation to extract nuclear proteins. Extract was rotated for 15 min at 4 °C and spun 30 min at 4 °C at 20,000g. Supernatant was incubated with 25 µl anti-Flag Affinity gel (Sigma-Aldrich) for 4 h at 4 °C. Beads were washed 2× 5 min with hypotonic buffer plus 400 mM NaCl. Bound proteins were eluted with 1× laemmli SDS sample buffer (63 mM Tris-HCl pH 6.8, 10% glycerol, 2% SDS). DTT was added to eluate to 1% before boiling and running on an SDS PAGE.

Transient transfection of 293T and HeLa cells and co-immunoprecipitation

Human histone H3.3 and mutants, as well as ASF1A and ASF1B, were cloned into a pCDH vector. Full-length human DAXX was cloned into pRK5 with an N-terminal Flag epitope tag. For transfection, 293T cells were seeded in 6-well plates at 2 × 105 cells per well in DMEM supplemented with 10% FBS, 1× penicillin/streptomycin. After 12–24 h, cells were transfected with 2.5 µg plasmid with Transit LT1 reagent (Mirus) according to the manufacturers manual. Cell were collected after 48 h by scraping and tituration in warm DMEM medium and pelleted. Cell pellet was washed with PBS and subsequently with hypotonic buffer (15 mM HEPES pH 7.5, 30 mM KCl, 5 mM MgCl2). For lysis, cell pellet was re-suspended in 250 µl hypotonic buffer supplemented with 0.02% Nonident P-40, 0.8 mM PMSF, 1× EDTA-free Protease Inhibitor Cocktail (Roche), 10 µg ml−1 RNase I. Lysis was allowed to proceed for 15 min with rotation at 4 °C. 22 µl 5 M NaCl was added (final 400 mM NaCl) under agitation to extract nuclear proteins. Extract was rotated for 15 min at 4 °C and spun 30 min at 4 °C at 20,000g. Supernatant was incubated with 25 µl anti-Flag Affinity gel (Sigma-Aldrich) for 4 h at 4 °C. Beads were washed 2× 5 min with hypotonic buffer plus 400 mM NaCl. Bound proteins were eluted with 1× laemmli SDS sample buffer (63 mM Tris-HCl pH 6.8, 10% glycerol, 2% SDS). DTT was added to eluate to 1% before boiling and running on an SDS PAGE.

RNAiHeLa cells stably expressing Flag-HA-tagged H3.3 and H3.1 were a gift from G. Almouzni20 siRNAs against ASF1A/B were ordered from Dharmacon, from published sequences (ASF1A, 5′-AAGUGAAGAAUACGAUCAAGU(dTdT)-3′, ASF1B, 5′-AACAACGAGUACCUCAACCCU(dTdT)-3′, (refs 28, 29). HeLa cells were transfected in 6-well plates with Oligofectamin (Invitrogen) according to the manufacturer’s protocol and collected after 72 h as described for transient transfections above.

HeLa cells stably expressing Flag-HA-tagged H3.3 and H3.1 were a gift from G. Almouzni20 siRNAs against ASF1A/B were ordered from Dharmacon, from published sequences (ASF1A, 5′-AAGUGAAGAAUACGAUCAAGU(dTdT)-3′, ASF1B, 5′-AACAACGAGUACCUCAACCCU(dTdT)-3′, (refs 28, 29). HeLa cells were transfected in 6-well plates with Oligofectamin (Invitrogen) according to the manufacturer’s protocol and collected after 72 h as described for transient transfections above.

Co-transfection of DAXX and histone mutantsWild-type pRK5-Flag–hDAXX or hDAXX point mutants were transiently transfected into 293T cells for 48 h (30 µg DNA per 2.0 × 107 cells). Cells were collected and lysed (20 mM HEPES, pH 7.9, 1 mM EDTA, 0.1% Triton X-100, 500 mM KCl, 2 mM 2-mercaptoethanol, 0.8 mM PMSF, 2× Protease inhibitor cocktail (Roche)) before Flag-M2 immunoprecipitation. Immunoprecipitated material was washed extensively (6× 1 ml of 20 mM HEPES, pH 7.9, 1 mM EDTA, 0.1% Triton X-100, 1 M KCl, 2 mM 2-mercaptoethanol, 0.4 mM PMSF) before elution (70 mM glycine, pH 2.5, 150 mM NaCl). pRK5-Flag–hDAXX(E225A) (8 µg DNA per 2.0 × 107 cells) and pCDH-HA–H3.3 or pCDH-HA–H3.2 (2 µg DNA per 2.0 × 107) were co-transfected into 293T cells for DAXX–H3.3–H4 co-immunoprecipitations.

Co-transfection of DAXX and histone mutants

Wild-type pRK5-Flag–hDAXX or hDAXX point mutants were transiently transfected into 293T cells for 48 h (30 µg DNA per 2.0 × 107 cells). Cells were collected and lysed (20 mM HEPES, pH 7.9, 1 mM EDTA, 0.1% Triton X-100, 500 mM KCl, 2 mM 2-mercaptoethanol, 0.8 mM PMSF, 2× Protease inhibitor cocktail (Roche)) before Flag-M2 immunoprecipitation. Immunoprecipitated material was washed extensively (6× 1 ml of 20 mM HEPES, pH 7.9, 1 mM EDTA, 0.1% Triton X-100, 1 M KCl, 2 mM 2-mercaptoethanol, 0.4 mM PMSF) before elution (70 mM glycine, pH 2.5, 150 mM NaCl). pRK5-Flag–hDAXX(E225A) (8 µg DNA per 2.0 × 107 cells) and pCDH-HA–H3.3 or pCDH-HA–H3.2 (2 µg DNA per 2.0 × 107) were co-transfected into 293T cells for DAXX–H3.3–H4 co-immunoprecipitations.

Supplementary MaterialSupplementary Figures and TablesSupplementary Video

Supplementary Material
