---
pmid: '25009770'
title: 'Discovery of novel interacting partners of PSMD9, a proteasomal chaperone:
  Role of an Atypical and versatile PDZ-domain motif interaction and identification
  of putative functional modules.'
authors:
- Sangith N
- Srinivasaraghavan K
- Sahu I
- Desai A
- Medipally S
- Somavarappu AK
- Verma C
- Venkatraman P
journal: FEBS Open Bio
year: '2014'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC4087146
doi: 10.1016/j.fob.2014.05.005
---

# Discovery of novel interacting partners of PSMD9, a proteasomal chaperone: Role of an Atypical and versatile PDZ-domain motif interaction and identification of putative functional modules.
**Authors:** Sangith N, Srinivasaraghavan K, Sahu I, Desai A, Medipally S, Somavarappu AK, Verma C, Venkatraman P
**Journal:** FEBS Open Bio (2014)
**DOI:** [10.1016/j.fob.2014.05.005](https://doi.org/10.1016/j.fob.2014.05.005)
**PMC:** [PMC4087146](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4087146/)

## Abstract

1. FEBS Open Bio. 2014 Jun 6;4:571-83. doi: 10.1016/j.fob.2014.05.005.
eCollection  2014.

Discovery of novel interacting partners of PSMD9, a proteasomal chaperone: Role 
of an Atypical and versatile PDZ-domain motif interaction and identification of 
putative functional modules.

Sangith N(1), Srinivasaraghavan K(2), Sahu I(1), Desai A(1), Medipally S(1), 
Somavarappu AK(1), Verma C(3), Venkatraman P(1).

Author information:
(1)Advanced Centre for Treatment, Research and Education in Cancer (ACTREC), 
Tata Memorial Centre (TMC), Kharghar, Navi Mumbai 410210, India.
(2)Bioinformatics Institute ASTAR, 30 Biopolis Street, #07-01 Matrix, Singapore 
138671, Singapore ; Experimental Therapeutics Centre (A*STAR), 31 Biopolis 
Street, #03-01 Helios, Singapore 138669, Singapore.
(3)Bioinformatics Institute ASTAR, 30 Biopolis Street, #07-01 Matrix, Singapore 
138671, Singapore ; School of Biological Sciences, Nanyang Technological 
University, 60 Nanyang Drive, Singapore 637551, Singapore ; Department of 
Biological Sciences, National University of Singapore, 14 Science Drive 4, 
Singapore 117543, Singapore.

Erratum in
    FEBS Open Bio. 2015 Jun 28;5:731-2. doi: 10.1016/j.fob.2015.06.010.

PSMD9 (Proteasome Macropain non-ATPase subunit 9), a proteasomal assembly 
chaperone, harbors an uncharacterized PDZ-like domain. Here we report the 
identification of five novel interacting partners of PSMD9 and provide the first 
glimpse at the structure of the PDZ-domain, including the molecular details of 
the interaction. We based our strategy on two propositions: (a) proteins with 
conserved C-termini may share common functions and (b) PDZ domains interact with 
C-terminal residues of proteins. Screening of C-terminal peptides followed by 
interactions using full-length recombinant proteins, we discovered hnRNPA1 (an 
RNA binding protein), S14 (a ribosomal protein), CSH1 (a growth hormone), E12 (a 
transcription factor) and IL6 receptor as novel PSMD9-interacting partners. 
Through multiple techniques and structural insights, we clearly demonstrate for 
the first time that human PDZ domain interacts with the predicted Short Linear 
Sequence Motif (SLIM) at the C-termini of the client proteins. These 
interactions are also recapitulated in mammalian cells. Together, these results 
are suggestive of the role of PSMD9 in transcriptional regulation, mRNA 
processing and editing, hormone and receptor activity and protein translation. 
Our proof-of-principle experiments endorse a novel and quick method for the 
identification of putative interacting partners of similar PDZ-domain proteins 
from the proteome and for discovering novel functions.

DOI: 10.1016/j.fob.2014.05.005
PMCID: PMC4087146
PMID: 25009770

## Full Text

1IntroductionAlmost every cellular pathway involved in the biology and homeostasis of a eukaryotic organism is regulated by the Ubiquitin Proteasome System (UPS) [1]. Impairment in the function of UPS components results in the accumulation of proteins leading to cellular stress and apoptosis [2]. While the role of proteasome in normal biology and disease is by and large well studied, the precise mechanism, the sequence and the structural requirements for substrate recognition, direct and indirect protein–protein interactions required for recruiting a substrate to the proteasome, remain obscure [3]. The structure and the domain functions of various 19S subunits and their role in proteasome dependent and independent functions are unclear. We recently showed that a 13 residue peptide of the A-helix from myoglobin acts as an anchor while a floppy region, the ‘F-helix’ acts as an initiator of proteasome mediated ubiquitin independent degradation of apomyoglobin [4]. We identified new interacting partners of gankyrin, a chaperone of the proteasome assembly and an oncoprotein by recognizing proteins that share EEVD, a conserved Short Linear Sequence Motif (SLIM) seen at the gankyrin and S6 ATPase interface [5]. Interaction between gankyrin and chloride intracellular channel protein through the conserved hot spot site enhances the migratory potential of breast carcinoma cell line. In addition, we demonstrated a role for Sug 1, an ATPase of the proteasome in transcriptional regulation of MHC proteins [6]. We described a novel role of PSMD9–hnRNPA1 interaction in basal and signal induced NF-κB activation via enhanced proteasomal degradation of IκBα [7]. We show that in this signaling pathway, proteasome bound PSMD9 acts as a subunit acceptor and hnRNPA1 as a shuttle receptor that recruits IκBα for degradation. Here, we exploit the presence of PDZ domain in PSMD9, a non-ATPase subunit, and a chaperone, of proteasome assembly to identify novel interacting partners and suggest putative functions of this biologically important molecule.

Introduction

Almost every cellular pathway involved in the biology and homeostasis of a eukaryotic organism is regulated by the Ubiquitin Proteasome System (UPS) [1]. Impairment in the function of UPS components results in the accumulation of proteins leading to cellular stress and apoptosis [2]. While the role of proteasome in normal biology and disease is by and large well studied, the precise mechanism, the sequence and the structural requirements for substrate recognition, direct and indirect protein–protein interactions required for recruiting a substrate to the proteasome, remain obscure [3]. The structure and the domain functions of various 19S subunits and their role in proteasome dependent and independent functions are unclear. We recently showed that a 13 residue peptide of the A-helix from myoglobin acts as an anchor while a floppy region, the ‘F-helix’ acts as an initiator of proteasome mediated ubiquitin independent degradation of apomyoglobin [4]. We identified new interacting partners of gankyrin, a chaperone of the proteasome assembly and an oncoprotein by recognizing proteins that share EEVD, a conserved Short Linear Sequence Motif (SLIM) seen at the gankyrin and S6 ATPase interface [5]. Interaction between gankyrin and chloride intracellular channel protein through the conserved hot spot site enhances the migratory potential of breast carcinoma cell line. In addition, we demonstrated a role for Sug 1, an ATPase of the proteasome in transcriptional regulation of MHC proteins [6]. We described a novel role of PSMD9–hnRNPA1 interaction in basal and signal induced NF-κB activation via enhanced proteasomal degradation of IκBα [7]. We show that in this signaling pathway, proteasome bound PSMD9 acts as a subunit acceptor and hnRNPA1 as a shuttle receptor that recruits IκBα for degradation. Here, we exploit the presence of PDZ domain in PSMD9, a non-ATPase subunit, and a chaperone, of proteasome assembly to identify novel interacting partners and suggest putative functions of this biologically important molecule.

2Material and methods2.1PlasmidsPSMD9 cDNA (Origene Technologies) was amplified and ligated into pRSETA vector between BamHI and EcoRI sites. hnRNPA1 and S14 ribosomal protein cDNA was generated by RT-PCR from RNA extracted from HEK293 cells. E12, growth hormone and the FN3 domain of IL6 receptor were amplified from the cDNA obtained from Harvard Institute of Proteomics. hnRNPA1 was ligated in pGEX4T1 (GE Amersham). FN3 domain was cloned in pGEX4T1 between BamHI and XhoI. S14, ribosomal protein, growth hormone and E12 were cloned in pMALC5 between BamHI and EcoRI sites. Mutations generated by site directed mutagenesis were confirmed by sequencing. PSMD9 was cloned in pCMV10 3X FLAG between HindIII and EcoRI sites. In doxycycline inducible pTRIPZ vector, PSMD9 was cloned between AgeI and EcoRI sites. All the interacting partners of PSMD9 were cloned in HA-pcDNA3.1 (A gift from Dr. Sorab Dalal, ACTREC) between BamHI and XhoI sites. Also see primers (Table S6).2.2Expression and purification of recombinant proteinsAll recombinant proteins were expressed in E.coli BL21 DE (3) using 100 μM IPTG at 20 °C for 16 h. His-PSMD9 and its mutants were purified by Ni-NTA chromatography (Qiagen); GST, GST-hnRNPA1, GST-FN3 and its mutants were purified using glutathione sepharose (GE Amersham); MBP and MBP-S14, E12 and growth hormone were purified using amylose resin (NEB), according to manufacturer’s protocol.2.3ELISA with tetra-peptidesN-terminal biotinylated tetra-peptides were procured from GenPro Biotech, India, (Biotin-KGG-XXXX, where XXXX represents the tetra-peptide sequence) and reconstituted to 25 mM with 100% DMSO and further diluted to 5 mM with distilled water. Anti-PSMD9 (Abcam) antibody in 0.1 M sodium carbonate buffer, pH 9.5 was coated on Nunc-Immuno™ MicroWell™ 96 well solid plates and incubated for 16 h at 4 °C. Wells were blocked with 2% BSA in TBST (10 mM Tris pH8, 138 mM NaCl and 0.5% Tween-20) for 1 h at 37 °C. His-tagged PSMD9 or its mutant proteins (5 μg/ml), diluted in TBST (containing 0.1% BSA) were added and incubated at 37 °C for 1 h. Plates were washed, and biotinylated peptides (in TBST with 0.1% BSA) were added to the wells and incubated for 1 h at 37 °C. The plates were washed with TBST vigorously after each incubation step. Finally, streptavidin alkaline phosphatase (Sigma), at a dilution of 1:2000 in TBST containing 0.1% BSA was added to all wells. After incubation for 1 h at 37 °C, binding was detected by the addition of para-Nitro phenyl phosphate (PNPP) (Bangalore Genei, India), the substrate of alkaline phosphatase and color developed was read at 405 nm (Spectramax 190, Molecular Devices). Wells that lack PSMD9 and wells that lack anti-PSMD9 antibody were taken as negative controls.2.4ELISA for PSMD9-hnRNPA1 and PSMD9-growth hormone interactionGST-hnRNPA1, its mutants and GST only (control; 5 μg/ml) or MBP-growth hormone and MBP only (control; 5 μg/ml) were coated as described for the PSMD9 antibody (Section3.2). All incubations were performed as described for the peptide ELISA (Section3.2). Different concentrations of His-tagged PSMD9 or its mutant proteins were (in TBST containing 0.1% BSA) added to the coated plates. After incubation, anti-his antibody (Cell Signaling) was added at a dilution of 1:2000, incubated and washed. HRP conjugated anti-mouse antibody (GE Amersham) (at 1:3000 dilution) was then added. After incubation and washes, HRP substrate TMB (1X) was added to all the wells. Reaction was stopped using 2 M sulfuric acid before recording the readings at 450 nm. Wells not coated with GST-hnRNPA1 and wells in which PSMD9 or the mutants were not added served as negative controls. For the competition assays, recombinant his-PSMD9 was incubated with different concentrations of GRRF/GRRG or SCGF/SCGG/SGGF peptides for 1 h at 37 °C and then added to wells containing GST-hnRNPA1 or MBP-GH respectively.2.5In vitro pull down assayRecombinant GST, GST- hnRNPA1, and its mutants (baits) were allowed to bind with glutathione sepharose beads (GE Amersham) in Transport Buffer (TB, 20 mM HEPES pH 7.9, 110 mM potassium acetate, 5 mM sodium acetate, 0.5 mM EGTA and 1 mM DTT) for 1 h at 4 °C. Beads were washed, following which PSMD9 or its mutants (in TB 0.1% BSA) were incubated with each bait for 4 h at 4 °C. Binding was monitored by Western blot using anti-His antibody (Cell Signaling). Cell lysates of MBP, MBP-S14, growth hormone, E12 or their respective C-terminal mutants were allowed to bind with amylose resin (NEB) in Transport Buffer for 1 h at 4 °C. Further incubations with PSMD9 or mutants were performed as described above except that anti- His antibody (Cell Signaling) was used to detect bound PSMD9.2.6Homology modelingThere is currently no crystal structure available for PSMD9 protein. A homology model of PDZ domain of PSMD9 was thus constructed using comparative modeling method, by comparing the sequence of this target protein with sequence of other related proteins (template) for which experimental structures are available. BLAST search showed that the PDZ domain shares 42% sequence similarity with PDZ2 domain of harmonin and sequence alignment between the two reveals that this sequence similarity is distributed throughout the sequence. Solution structure of PDZ2 domain of harmonin bound with C-terminal peptide of cadherin23 (PDB code 2KBS) [8] was chosen as a template for the homology modeling. Modeller, a program for comparative protein structure modeling by satisfaction of spatial restraints [9] was used for generation of the homology model. Several homology models were built based on structural information from the template, and model that showed good stereochemical property was selected for further use.2.7Peptide docking3D structure of peptides GRRF and SCGF was generated using Xleap module in Amber11 [10]. Peptide in its extended conformation was docked with the generated model of PDZ domain of PSMD9 protein. Peptide docking was carried out with two different docking programs, HADDOCK [11] and ATTRACT [12]. For HADDOCK, a binding site was defined using residues Leu124, Gly125, Gln126, Glu128 and Gln181 within the canonical pocket. No information regarding the binding site was given while using ATTRACT and a complete blind docking was performed using this program. Both the docking programs were validated earlier, by docking a set of co-crystallized peptides into the canonical pocket of the corresponding PDZ domains, and the docked conformations of each peptide had rmsd values 1.5–2.5 A with the corresponding experimental structures.2.8Molecular dynamics simulationsGenerated homology model of PDZ domain, peptide GRRF (derived from C-terminus of hnRNPA1) - PDZ complex (PDZ-GRRF) and peptide SCGF – PDZ complexes (PDZ-SCGF) (both the canonical and non-canonical binding mode) were used as the starting structure for MD simulations. Mutated structures of the protein Q181G and thetriple β-sheet mutant L124G/Q126G/E128G were also generated by replacing (mutating) the respective residues in PyMol. Hydrogen atoms were added to the WT and mutant experimental structures using the Xleap module of the Amber11 package. N-terminus of the GRRF and SCGF peptide was capped by acetylation (ACE). Simulation systems were neutralized by the addition of counter ions. The neutralized system was solvated with TIP3P [13] water molecules to form a truncated octahedral box with at least 10 Ǻ separating the solute atoms and the edges of the box. MD simulations were carried out with the Sander module of the AMBER11 package in combination with the parm03 force field [14]. All systems were first subjected to 100 steps of energy minimization. The protein was initially harmonically restrained (25 kcal mol−1 Ǻ2) to the energy minimized coordinates, and MD simulations were initiated by heating the system to 300 K in steps of 100 K followed by gradual removal of the positional restraints, and a 1 ns unrestrained equilibration at 300 K. The resulting system was used as starting structure for production MD run. For each case, three independent (using different initial random velocities) MD simulations were carried out starting from the well equilibrated structure. Each MD simulation was carried out for 100 ns and conformations were recorded every 10 ps. All MD simulations were carried out in explicit solvent at 300 K. During all the simulations, the long-range electrostatic interactions were treated with the particle mesh Ewald [15] method using a real space distance cutoff of 9 Ǻ. The settle [16] algorithm was used to constrain bond vibrations involving hydrogen atoms, which allowed time step of 2 fs during the simulations. Simulation trajectories were visualized using VMD [17] and figures were generated using PyMol.2.9ImmunoprecipitationFLAG-PSMD9 and HA tagged interacting partners were overexpressed in HEK293 cells. Lysates were added either to M2-Agarose (Sigma) or to anti HA-agarose beads and incubated for 3 h at 4 °C to immunoprecipitate the complex. Either anti-HA antibody or anti-FLAG antibody (Sigma) was used for detection.2.10Circular dichroism of PSMD9 and its mutantsFar-UV CD spectrum (Jasco, J815) of PSMD9-WT and its mutant proteins were recorded between 260 nm and 190 nm in a 2 mm path length cuvette. A protein concentration of 2 μM, in a volume of 500 μl (10 mM phosphate buffer (pH 7.5)) was used for collecting data at 20 °C. Data were normalized to obtain molar ellipticity values and fitted using Dichroweb’s CONTIN software.2.11Tryptophan fluorescence of PSMD9 and mutantsTryptophan fluorescence of PSMD9-WT and PSMD9-PDZ-mutants was recorded at a concentration of 1.5 μM. Emission spectra between 310 and 400 nm were collected upon excitation at 295 nm with a slit width of 5 nm and scan speed of 50 nm/s using Fluorolog HORIBA fluorimeter.2.12Western blottingSamples were separated on 15% SDS PAGE gels and Western blots were performed using standard protocols. Depending on the protein under study, anti-His antibody (mouse monoclonal, Cell Signaling), anti-FLAG antibody (rabbit polyclonal, Sigma) or anti-HA antibody (rabbit polyclonal, Sigma) were used.

Material and methods

2.1PlasmidsPSMD9 cDNA (Origene Technologies) was amplified and ligated into pRSETA vector between BamHI and EcoRI sites. hnRNPA1 and S14 ribosomal protein cDNA was generated by RT-PCR from RNA extracted from HEK293 cells. E12, growth hormone and the FN3 domain of IL6 receptor were amplified from the cDNA obtained from Harvard Institute of Proteomics. hnRNPA1 was ligated in pGEX4T1 (GE Amersham). FN3 domain was cloned in pGEX4T1 between BamHI and XhoI. S14, ribosomal protein, growth hormone and E12 were cloned in pMALC5 between BamHI and EcoRI sites. Mutations generated by site directed mutagenesis were confirmed by sequencing. PSMD9 was cloned in pCMV10 3X FLAG between HindIII and EcoRI sites. In doxycycline inducible pTRIPZ vector, PSMD9 was cloned between AgeI and EcoRI sites. All the interacting partners of PSMD9 were cloned in HA-pcDNA3.1 (A gift from Dr. Sorab Dalal, ACTREC) between BamHI and XhoI sites. Also see primers (Table S6).

PSMD9 cDNA (Origene Technologies) was amplified and ligated into pRSETA vector between BamHI and EcoRI sites. hnRNPA1 and S14 ribosomal protein cDNA was generated by RT-PCR from RNA extracted from HEK293 cells. E12, growth hormone and the FN3 domain of IL6 receptor were amplified from the cDNA obtained from Harvard Institute of Proteomics. hnRNPA1 was ligated in pGEX4T1 (GE Amersham). FN3 domain was cloned in pGEX4T1 between BamHI and XhoI. S14, ribosomal protein, growth hormone and E12 were cloned in pMALC5 between BamHI and EcoRI sites. Mutations generated by site directed mutagenesis were confirmed by sequencing. PSMD9 was cloned in pCMV10 3X FLAG between HindIII and EcoRI sites. In doxycycline inducible pTRIPZ vector, PSMD9 was cloned between AgeI and EcoRI sites. All the interacting partners of PSMD9 were cloned in HA-pcDNA3.1 (A gift from Dr. Sorab Dalal, ACTREC) between BamHI and XhoI sites. Also see primers (Table S6).

2.2Expression and purification of recombinant proteinsAll recombinant proteins were expressed in E.coli BL21 DE (3) using 100 μM IPTG at 20 °C for 16 h. His-PSMD9 and its mutants were purified by Ni-NTA chromatography (Qiagen); GST, GST-hnRNPA1, GST-FN3 and its mutants were purified using glutathione sepharose (GE Amersham); MBP and MBP-S14, E12 and growth hormone were purified using amylose resin (NEB), according to manufacturer’s protocol.

Expression and purification of recombinant proteins

All recombinant proteins were expressed in E.coli BL21 DE (3) using 100 μM IPTG at 20 °C for 16 h. His-PSMD9 and its mutants were purified by Ni-NTA chromatography (Qiagen); GST, GST-hnRNPA1, GST-FN3 and its mutants were purified using glutathione sepharose (GE Amersham); MBP and MBP-S14, E12 and growth hormone were purified using amylose resin (NEB), according to manufacturer’s protocol.

2.3ELISA with tetra-peptidesN-terminal biotinylated tetra-peptides were procured from GenPro Biotech, India, (Biotin-KGG-XXXX, where XXXX represents the tetra-peptide sequence) and reconstituted to 25 mM with 100% DMSO and further diluted to 5 mM with distilled water. Anti-PSMD9 (Abcam) antibody in 0.1 M sodium carbonate buffer, pH 9.5 was coated on Nunc-Immuno™ MicroWell™ 96 well solid plates and incubated for 16 h at 4 °C. Wells were blocked with 2% BSA in TBST (10 mM Tris pH8, 138 mM NaCl and 0.5% Tween-20) for 1 h at 37 °C. His-tagged PSMD9 or its mutant proteins (5 μg/ml), diluted in TBST (containing 0.1% BSA) were added and incubated at 37 °C for 1 h. Plates were washed, and biotinylated peptides (in TBST with 0.1% BSA) were added to the wells and incubated for 1 h at 37 °C. The plates were washed with TBST vigorously after each incubation step. Finally, streptavidin alkaline phosphatase (Sigma), at a dilution of 1:2000 in TBST containing 0.1% BSA was added to all wells. After incubation for 1 h at 37 °C, binding was detected by the addition of para-Nitro phenyl phosphate (PNPP) (Bangalore Genei, India), the substrate of alkaline phosphatase and color developed was read at 405 nm (Spectramax 190, Molecular Devices). Wells that lack PSMD9 and wells that lack anti-PSMD9 antibody were taken as negative controls.

ELISA with tetra-peptides

N-terminal biotinylated tetra-peptides were procured from GenPro Biotech, India, (Biotin-KGG-XXXX, where XXXX represents the tetra-peptide sequence) and reconstituted to 25 mM with 100% DMSO and further diluted to 5 mM with distilled water. Anti-PSMD9 (Abcam) antibody in 0.1 M sodium carbonate buffer, pH 9.5 was coated on Nunc-Immuno™ MicroWell™ 96 well solid plates and incubated for 16 h at 4 °C. Wells were blocked with 2% BSA in TBST (10 mM Tris pH8, 138 mM NaCl and 0.5% Tween-20) for 1 h at 37 °C. His-tagged PSMD9 or its mutant proteins (5 μg/ml), diluted in TBST (containing 0.1% BSA) were added and incubated at 37 °C for 1 h. Plates were washed, and biotinylated peptides (in TBST with 0.1% BSA) were added to the wells and incubated for 1 h at 37 °C. The plates were washed with TBST vigorously after each incubation step. Finally, streptavidin alkaline phosphatase (Sigma), at a dilution of 1:2000 in TBST containing 0.1% BSA was added to all wells. After incubation for 1 h at 37 °C, binding was detected by the addition of para-Nitro phenyl phosphate (PNPP) (Bangalore Genei, India), the substrate of alkaline phosphatase and color developed was read at 405 nm (Spectramax 190, Molecular Devices). Wells that lack PSMD9 and wells that lack anti-PSMD9 antibody were taken as negative controls.

2.4ELISA for PSMD9-hnRNPA1 and PSMD9-growth hormone interactionGST-hnRNPA1, its mutants and GST only (control; 5 μg/ml) or MBP-growth hormone and MBP only (control; 5 μg/ml) were coated as described for the PSMD9 antibody (Section3.2). All incubations were performed as described for the peptide ELISA (Section3.2). Different concentrations of His-tagged PSMD9 or its mutant proteins were (in TBST containing 0.1% BSA) added to the coated plates. After incubation, anti-his antibody (Cell Signaling) was added at a dilution of 1:2000, incubated and washed. HRP conjugated anti-mouse antibody (GE Amersham) (at 1:3000 dilution) was then added. After incubation and washes, HRP substrate TMB (1X) was added to all the wells. Reaction was stopped using 2 M sulfuric acid before recording the readings at 450 nm. Wells not coated with GST-hnRNPA1 and wells in which PSMD9 or the mutants were not added served as negative controls. For the competition assays, recombinant his-PSMD9 was incubated with different concentrations of GRRF/GRRG or SCGF/SCGG/SGGF peptides for 1 h at 37 °C and then added to wells containing GST-hnRNPA1 or MBP-GH respectively.

ELISA for PSMD9-hnRNPA1 and PSMD9-growth hormone interaction

GST-hnRNPA1, its mutants and GST only (control; 5 μg/ml) or MBP-growth hormone and MBP only (control; 5 μg/ml) were coated as described for the PSMD9 antibody (Section3.2). All incubations were performed as described for the peptide ELISA (Section3.2). Different concentrations of His-tagged PSMD9 or its mutant proteins were (in TBST containing 0.1% BSA) added to the coated plates. After incubation, anti-his antibody (Cell Signaling) was added at a dilution of 1:2000, incubated and washed. HRP conjugated anti-mouse antibody (GE Amersham) (at 1:3000 dilution) was then added. After incubation and washes, HRP substrate TMB (1X) was added to all the wells. Reaction was stopped using 2 M sulfuric acid before recording the readings at 450 nm. Wells not coated with GST-hnRNPA1 and wells in which PSMD9 or the mutants were not added served as negative controls. For the competition assays, recombinant his-PSMD9 was incubated with different concentrations of GRRF/GRRG or SCGF/SCGG/SGGF peptides for 1 h at 37 °C and then added to wells containing GST-hnRNPA1 or MBP-GH respectively.

2.5In vitro pull down assayRecombinant GST, GST- hnRNPA1, and its mutants (baits) were allowed to bind with glutathione sepharose beads (GE Amersham) in Transport Buffer (TB, 20 mM HEPES pH 7.9, 110 mM potassium acetate, 5 mM sodium acetate, 0.5 mM EGTA and 1 mM DTT) for 1 h at 4 °C. Beads were washed, following which PSMD9 or its mutants (in TB 0.1% BSA) were incubated with each bait for 4 h at 4 °C. Binding was monitored by Western blot using anti-His antibody (Cell Signaling). Cell lysates of MBP, MBP-S14, growth hormone, E12 or their respective C-terminal mutants were allowed to bind with amylose resin (NEB) in Transport Buffer for 1 h at 4 °C. Further incubations with PSMD9 or mutants were performed as described above except that anti- His antibody (Cell Signaling) was used to detect bound PSMD9.

In vitro pull down assay

Recombinant GST, GST- hnRNPA1, and its mutants (baits) were allowed to bind with glutathione sepharose beads (GE Amersham) in Transport Buffer (TB, 20 mM HEPES pH 7.9, 110 mM potassium acetate, 5 mM sodium acetate, 0.5 mM EGTA and 1 mM DTT) for 1 h at 4 °C. Beads were washed, following which PSMD9 or its mutants (in TB 0.1% BSA) were incubated with each bait for 4 h at 4 °C. Binding was monitored by Western blot using anti-His antibody (Cell Signaling). Cell lysates of MBP, MBP-S14, growth hormone, E12 or their respective C-terminal mutants were allowed to bind with amylose resin (NEB) in Transport Buffer for 1 h at 4 °C. Further incubations with PSMD9 or mutants were performed as described above except that anti- His antibody (Cell Signaling) was used to detect bound PSMD9.

2.6Homology modelingThere is currently no crystal structure available for PSMD9 protein. A homology model of PDZ domain of PSMD9 was thus constructed using comparative modeling method, by comparing the sequence of this target protein with sequence of other related proteins (template) for which experimental structures are available. BLAST search showed that the PDZ domain shares 42% sequence similarity with PDZ2 domain of harmonin and sequence alignment between the two reveals that this sequence similarity is distributed throughout the sequence. Solution structure of PDZ2 domain of harmonin bound with C-terminal peptide of cadherin23 (PDB code 2KBS) [8] was chosen as a template for the homology modeling. Modeller, a program for comparative protein structure modeling by satisfaction of spatial restraints [9] was used for generation of the homology model. Several homology models were built based on structural information from the template, and model that showed good stereochemical property was selected for further use.

Homology modeling

There is currently no crystal structure available for PSMD9 protein. A homology model of PDZ domain of PSMD9 was thus constructed using comparative modeling method, by comparing the sequence of this target protein with sequence of other related proteins (template) for which experimental structures are available. BLAST search showed that the PDZ domain shares 42% sequence similarity with PDZ2 domain of harmonin and sequence alignment between the two reveals that this sequence similarity is distributed throughout the sequence. Solution structure of PDZ2 domain of harmonin bound with C-terminal peptide of cadherin23 (PDB code 2KBS) [8] was chosen as a template for the homology modeling. Modeller, a program for comparative protein structure modeling by satisfaction of spatial restraints [9] was used for generation of the homology model. Several homology models were built based on structural information from the template, and model that showed good stereochemical property was selected for further use.

2.7Peptide docking3D structure of peptides GRRF and SCGF was generated using Xleap module in Amber11 [10]. Peptide in its extended conformation was docked with the generated model of PDZ domain of PSMD9 protein. Peptide docking was carried out with two different docking programs, HADDOCK [11] and ATTRACT [12]. For HADDOCK, a binding site was defined using residues Leu124, Gly125, Gln126, Glu128 and Gln181 within the canonical pocket. No information regarding the binding site was given while using ATTRACT and a complete blind docking was performed using this program. Both the docking programs were validated earlier, by docking a set of co-crystallized peptides into the canonical pocket of the corresponding PDZ domains, and the docked conformations of each peptide had rmsd values 1.5–2.5 A with the corresponding experimental structures.

Peptide docking

3D structure of peptides GRRF and SCGF was generated using Xleap module in Amber11 [10]. Peptide in its extended conformation was docked with the generated model of PDZ domain of PSMD9 protein. Peptide docking was carried out with two different docking programs, HADDOCK [11] and ATTRACT [12]. For HADDOCK, a binding site was defined using residues Leu124, Gly125, Gln126, Glu128 and Gln181 within the canonical pocket. No information regarding the binding site was given while using ATTRACT and a complete blind docking was performed using this program. Both the docking programs were validated earlier, by docking a set of co-crystallized peptides into the canonical pocket of the corresponding PDZ domains, and the docked conformations of each peptide had rmsd values 1.5–2.5 A with the corresponding experimental structures.

2.8Molecular dynamics simulationsGenerated homology model of PDZ domain, peptide GRRF (derived from C-terminus of hnRNPA1) - PDZ complex (PDZ-GRRF) and peptide SCGF – PDZ complexes (PDZ-SCGF) (both the canonical and non-canonical binding mode) were used as the starting structure for MD simulations. Mutated structures of the protein Q181G and thetriple β-sheet mutant L124G/Q126G/E128G were also generated by replacing (mutating) the respective residues in PyMol. Hydrogen atoms were added to the WT and mutant experimental structures using the Xleap module of the Amber11 package. N-terminus of the GRRF and SCGF peptide was capped by acetylation (ACE). Simulation systems were neutralized by the addition of counter ions. The neutralized system was solvated with TIP3P [13] water molecules to form a truncated octahedral box with at least 10 Ǻ separating the solute atoms and the edges of the box. MD simulations were carried out with the Sander module of the AMBER11 package in combination with the parm03 force field [14]. All systems were first subjected to 100 steps of energy minimization. The protein was initially harmonically restrained (25 kcal mol−1 Ǻ2) to the energy minimized coordinates, and MD simulations were initiated by heating the system to 300 K in steps of 100 K followed by gradual removal of the positional restraints, and a 1 ns unrestrained equilibration at 300 K. The resulting system was used as starting structure for production MD run. For each case, three independent (using different initial random velocities) MD simulations were carried out starting from the well equilibrated structure. Each MD simulation was carried out for 100 ns and conformations were recorded every 10 ps. All MD simulations were carried out in explicit solvent at 300 K. During all the simulations, the long-range electrostatic interactions were treated with the particle mesh Ewald [15] method using a real space distance cutoff of 9 Ǻ. The settle [16] algorithm was used to constrain bond vibrations involving hydrogen atoms, which allowed time step of 2 fs during the simulations. Simulation trajectories were visualized using VMD [17] and figures were generated using PyMol.

Molecular dynamics simulations

Generated homology model of PDZ domain, peptide GRRF (derived from C-terminus of hnRNPA1) - PDZ complex (PDZ-GRRF) and peptide SCGF – PDZ complexes (PDZ-SCGF) (both the canonical and non-canonical binding mode) were used as the starting structure for MD simulations. Mutated structures of the protein Q181G and thetriple β-sheet mutant L124G/Q126G/E128G were also generated by replacing (mutating) the respective residues in PyMol. Hydrogen atoms were added to the WT and mutant experimental structures using the Xleap module of the Amber11 package. N-terminus of the GRRF and SCGF peptide was capped by acetylation (ACE). Simulation systems were neutralized by the addition of counter ions. The neutralized system was solvated with TIP3P [13] water molecules to form a truncated octahedral box with at least 10 Ǻ separating the solute atoms and the edges of the box. MD simulations were carried out with the Sander module of the AMBER11 package in combination with the parm03 force field [14]. All systems were first subjected to 100 steps of energy minimization. The protein was initially harmonically restrained (25 kcal mol−1 Ǻ2) to the energy minimized coordinates, and MD simulations were initiated by heating the system to 300 K in steps of 100 K followed by gradual removal of the positional restraints, and a 1 ns unrestrained equilibration at 300 K. The resulting system was used as starting structure for production MD run. For each case, three independent (using different initial random velocities) MD simulations were carried out starting from the well equilibrated structure. Each MD simulation was carried out for 100 ns and conformations were recorded every 10 ps. All MD simulations were carried out in explicit solvent at 300 K. During all the simulations, the long-range electrostatic interactions were treated with the particle mesh Ewald [15] method using a real space distance cutoff of 9 Ǻ. The settle [16] algorithm was used to constrain bond vibrations involving hydrogen atoms, which allowed time step of 2 fs during the simulations. Simulation trajectories were visualized using VMD [17] and figures were generated using PyMol.

2.9ImmunoprecipitationFLAG-PSMD9 and HA tagged interacting partners were overexpressed in HEK293 cells. Lysates were added either to M2-Agarose (Sigma) or to anti HA-agarose beads and incubated for 3 h at 4 °C to immunoprecipitate the complex. Either anti-HA antibody or anti-FLAG antibody (Sigma) was used for detection.

Immunoprecipitation

FLAG-PSMD9 and HA tagged interacting partners were overexpressed in HEK293 cells. Lysates were added either to M2-Agarose (Sigma) or to anti HA-agarose beads and incubated for 3 h at 4 °C to immunoprecipitate the complex. Either anti-HA antibody or anti-FLAG antibody (Sigma) was used for detection.

2.10Circular dichroism of PSMD9 and its mutantsFar-UV CD spectrum (Jasco, J815) of PSMD9-WT and its mutant proteins were recorded between 260 nm and 190 nm in a 2 mm path length cuvette. A protein concentration of 2 μM, in a volume of 500 μl (10 mM phosphate buffer (pH 7.5)) was used for collecting data at 20 °C. Data were normalized to obtain molar ellipticity values and fitted using Dichroweb’s CONTIN software.

Circular dichroism of PSMD9 and its mutants

Far-UV CD spectrum (Jasco, J815) of PSMD9-WT and its mutant proteins were recorded between 260 nm and 190 nm in a 2 mm path length cuvette. A protein concentration of 2 μM, in a volume of 500 μl (10 mM phosphate buffer (pH 7.5)) was used for collecting data at 20 °C. Data were normalized to obtain molar ellipticity values and fitted using Dichroweb’s CONTIN software.

2.11Tryptophan fluorescence of PSMD9 and mutantsTryptophan fluorescence of PSMD9-WT and PSMD9-PDZ-mutants was recorded at a concentration of 1.5 μM. Emission spectra between 310 and 400 nm were collected upon excitation at 295 nm with a slit width of 5 nm and scan speed of 50 nm/s using Fluorolog HORIBA fluorimeter.

Tryptophan fluorescence of PSMD9 and mutants

Tryptophan fluorescence of PSMD9-WT and PSMD9-PDZ-mutants was recorded at a concentration of 1.5 μM. Emission spectra between 310 and 400 nm were collected upon excitation at 295 nm with a slit width of 5 nm and scan speed of 50 nm/s using Fluorolog HORIBA fluorimeter.

2.12Western blottingSamples were separated on 15% SDS PAGE gels and Western blots were performed using standard protocols. Depending on the protein under study, anti-His antibody (mouse monoclonal, Cell Signaling), anti-FLAG antibody (rabbit polyclonal, Sigma) or anti-HA antibody (rabbit polyclonal, Sigma) were used.

Western blotting

Samples were separated on 15% SDS PAGE gels and Western blots were performed using standard protocols. Depending on the protein under study, anti-His antibody (mouse monoclonal, Cell Signaling), anti-FLAG antibody (rabbit polyclonal, Sigma) or anti-HA antibody (rabbit polyclonal, Sigma) were used.

3Results3.1A screen for putative PSMD9 interacting partners and validation using full length proteinsMany methods capitalize on the ability of the PDZ domains to recognize C-terminal residues in proteins to primarily define their binding specificity [18–22]. Peptide libraries have been created, and peptides derived from the C-terminus of the human proteome have been used by various investigators [18,23–25]. We chose C-terminal peptides of the human proteome as baits to identify novel interacting partners of PSMD9. Premise for this study is that modification-independent, sequence specific recognition is central to many biological processes, and rules inherent to this recognition process can bring together proteins of very different functions under a master regulator. Chung et al., had classified proteome from drosophila/yeast/human by recognizing conserved C-terminal residues in some of these proteins [26]. These C-terminal peptides were tested here for the following reasons. (1) Most high-throughput studies are optimized for selecting peptides with high affinity while many protein–protein interactions are of low affinity and, therefore, are likely to be missed. (2) If the corresponding protein/proteins were to interact, one could quickly move to associated functions, and finally (3) such a guided approach prevents identification of those peptides that are not represented in the human proteome and, are physiologically irrelevant. Due to financial constraints, thirteen among the thirty conserved tetra peptides from the human proteome were chosen. These sequences differ in charge, hydrophobicity and size and represent some of the known sequence specificity seen with other PDZ domains. AGHM, the C-terminus of E12 transcription factor, the human homolog of rat E2, was specifically included. E12 was shown to interact with Bridge 1 (homolog of PSMD9 with a PDZ domain) during insulin signaling [27–29]. We cloned, expressed and purified human PSMD9 and used enzyme-linked immunosorbent assay (ELISA), to test for binding of the peptides. GRRF, SCGF and AGHM peptides bound to PSMD9 to an appreciable extent with SCGF demonstrating highest affinity (Fig. 1A and Fig. S1A and B). SCGF and GRRF resemble class III PDZ peptides with the sequence motif-X-[D/E/K/R]-X-Φ where Φ is hydrophobic, and X is any residue. GRRF forms the C-terminus of hnRNPA1 isoforms while SCGF belongs to growth hormone (CSH1; referred from henceforth as GH). To test if the corresponding full length proteins would interact with PSMD9, we cloned and expressed the longer isoform of hnRNPA1 as a GST fusion protein, GH and E12 as MBP fusion proteins. PSMD9 was expressed as a His-Tag protein. Affinity pull-down followed by Western blot showed that the three full length proteins interact with PSMD9 (Fig. 1B–D). While hnRNPA1 (Fig 1E andTable S4) and E12 binding (Fig 1C) were clearly affected by simple C-terminal substitution (Phe to Gly), GH binding to PSMD9 was not affected to any measurable extent (Fig. 1D). Deletion of C-terminal seven residues compromised binding of GH severely (Fig. 1D) and not surprisingly those of hnRNPA1 and E12, as well (Fig. 1B and C). These interactions were further confirmed using ELISA and the estimated dissociation constant Kd for PSMD9-hnRNPA1 interaction is 1.33 ± 0.16 μM and of PSMD9-GH interaction is 0.74 ± 0.04 μM and ΔG for the interaction between PSMD9 and WT-hnRNPA1 or GH were calculated to be 6.9 ± 0.04 and 7.1 ± 0.09 kcal/mol, respectively. Peptide GRRF and not GRRG inhibited hnRNPA1 binding (Ki of 326.5 ± 0.25 μM) confirming the importance of C-terminal residues (Fig. 1G) in this interaction. Again, as seen with the C-terminal substitutions of GH, inhibition of GH-PSMD9 interaction by SCGG was as good as SCGF and the Ki values for these peptides were 36.7 ± 0.29 and 35.6 ± 0.24 μM, respectively (Fig. 1H). These pairwise interactions and the role of C-terminal residues were confirmed in mammalian cells using co-immunoprecipitation assays (Fig. 1I and J).3.2The fine specificity of SCG derivativesUnlike hnRNPA1 GRRG mutant, mutant GH with a C-terminal substituted SCGG binds to the PDZ domain of PSMD9 and interaction is inhibited only upon deletion of C-terminal residues (Δ7 mutant). To identify the minimal motif important for GH interaction, we engineered ΔGF, and ΔCGF mutants of GH and interaction with PSMD9 was tested by pull down and ELISA (Fig. 2 A and B and Table S4). While ΔGF mutant bound with PSMD9, deletion of one more residue, Cysteine, ΔGFC, impaired the interaction. By ELISA, the estimated Kd values were 0.8 ± 0.02 μM for ΔGF and 2.6 ± 0.011 μM for ΔCGF mutant. The % occupancy of GH was unaltered in the ΔGF mutant but was reduced to ∼45% in the case of the ΔCGF mutant. This result emphasizes the importance of P−2 residue in interaction with PSMD9. The importance of the P−2 Cys was further confirmed by demonstrating the failure of peptide SGGF to inhibit the binding of GH to PSMD9 (Fig. 2C). As noted before both SCGF and SCGG can inhibit binding between the two proteins.Our results help to clarify some of the observations made earlier with respect to Nas-2-Rpt5 interaction in yeast (PSMD9 homolog and the ATPase subunit of the 19S regulatory particle). Here, single C-terminal residue deletion in Rpt5 did not affect its binding to Nas-2 that made the authors conclude that the PDZ like domain of Nas2 may not confirm to the classical description [30]. Based on our results on human PSMD9 using similar pull down assays, other comparative studies and quantitative analysis, we show that the precise role of the C-terminal residues in the interaction is likely to be context dependent. In the case of hnRNPA1 (GRRF) and E12 (AGHM), bulk of the binding energy is derived from the C-terminal residue much like the classical PDZ domains. In GH with SCGF at the C-terminus, however, the terminal residue is less important. These differences are also reflected in the binding affinity of the three peptides to PSMD9. While GRRF binds weakly (Kd 651.7 ± 76 μM), peptide SCGF binds tightly to PSMD9 (Kd 8.6 ± 1.2 μM). One possible explanation is that these peptides may bind in different modes or orientations at the binding groove (discussed below). While results observed with the C-terminal peptides can be readily extrapolated to protein binding, stable binding of the full length protein may require additional interactions. It is also likely that, besides the canonical α–β groove, the protein, may bind elsewhere on PSMD9 perhaps at an allosteric site while the C-terminal sequence acts as initial recognition element that docks the protein at the canonical site.3.3Role of PDZ domain in interaction: modeling and site directed mutagenesisTo better understand the role of the C-terminal residues and PDZ domain in binding and recognition, we modeled the structure of PDZ and carried out extensive molecular dynamic simulations and peptide docking studies (supplementary methods). Several docking poses were created. Upon visual inspection of all the docked poses, a peptide-protein complex similar to that seen in the co-crystals of other PDZ-peptide complex with Phe at the fourth position was chosen. In this conformation, the peptide binds in an extended, antiparallel manner through canonical interactions that extend the beta sheet by an additional strand (Fig. 3A and B). The hydrophobic side chain of Phe4 of the peptide is deeply buried in the hydrophobic pocket formed by Leu124 from β2, Val139, from β3, Leu153 from β4, Ile159, Phe 162, from β4. The peptide further interacts with the beta sheet mainly through backbone/side chain hydrogen bonds with residues Leu124, Gly125, Gln126, Glu128 of β2 of the PDZ domain (Fig. 2B). In addition, the side chain of Arg2 of the peptide forms a salt bridge with the side chain of Glu128 from β2. During MD simulation, the alpha/beta binding groove (canonical binding site) of apo PDZ showed increased flexibility (Supplementary Fig. S2). The α2/β2 binding pocket was partially deformed/destabilized (either collapses or widens), and is stabilized upon peptide binding. Increased flexibility of PDZ domains in their apo form have been reported by others [31]. The intrinsic flexibility of PDZ domains is a key determinant that allows them to recognize a wide repertoire of peptide ligands. Throughout the protein-peptide simulation, Phe4 remains deeply buried in the hydrophobic pocket (Movie1:http://web.bii.a-star.edu.sg/bmad/PDZ/PDZ-PEP-WT-Top.mpg). Charge-charge interactions between Arg2 and Glu128 on β2 are preserved during the 100 ns simulation. The bound conformation of the peptide was further stabilized via backbone hydrogen bond interactions with residues Leu124, Gly125, Gln126 and Glu128 from β2 in the canonical binding site.In the complex where Phe4 was mutated to Gly, the peptide unbinds from the canonical binding site within ∼5–10 ns and doesn’t bind again (Movie 2:http://web.bii.a-star.edu.sg/bmad/PDZ/PDZ-PEP_GRRG-Top.mpg). Although the peptide stays close to the canonical site due to charge-charge interactions with the protein residues, it undergoes translation and rotations that prevent it from rebinding in the canonical interaction mode. Thus, our MD simulations suggest that the burial of Phe in the hydrophobic pocket is crucial for the stabilization of this peptide in its bound conformation. Based on peptide docking and MD simulations (Movie 3:http://web.bii.a-star.edu.sg/bmad/PDZ/PDZPEP_L124G_Q126G_E128G-Top.mpg and Movie 4: http://web.bii.a-star.edu.sg/bmad/PDZ/PDZ-PEP_Q181G-Top.mpg), three single amino acid mutations F162G, L153G, Q181G and a triple mutation, L124G/Q126G/E128G were generated. In vitro pull-down shows that these mutations affect GH and hnRNPA1 binding to PSMD9 (Fig. 3C and D). Mutation of residue L173 (to Gly), part of the α2 helix, not involved in the interaction, did not affect the binding of peptide or the proteins (Fig. 3E and Table S1). MD simulations support this finding as the L173G PSMD9 mutant maintains the peptide in a stably bound form (not shown).These results together, confirm the domain-motif interaction between PDZ domain of PSMD9 and the C-terminal region of the interacting proteins. The instability of the peptide-free forms is reflected in the secondary structure of these proteins determined by circular dichroism. While WT PSMD9 records 49% helicity, the L173G mutant shows 43% helical structure, Q181G mutant 39%, L153G mutant 45% and the F162G mutant shows 42% helical structure (Fig. 3F, Table 1 and Appendix Eq.(1)) [32]. Tryptophan fluorescence of these mutant proteins is less affected (Fig. 3G).3.4Identification of putative functional modules regulated by PSMD9Although GRRF and SCGF were motifs under which several family members (12 and 13 respectively) were grouped by Chung et al., a detailed analysis and further curation using UniProt data (ftp://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/) indicated that there was only one unique protein under each family. There are four isoforms within the GRRF family and ten isoforms within the SCGF family (Table S2). We re-analyzed other 28 families and found that, in the vast majority of the cases, the proteins grouped under each peptide family are primarily isoforms (Table S2). Although isoforms are homologous in sequence, their functions can be mutually exclusive or even counteractive [33,34]. To better define the role of C-terminus in functional grouping beyond isoforms, and predict the modules that may be regulated by PSMD9, we analyzed C-terminal variants of GRRF and SCGF from the human proteome. There are ten variants of GRRF where X is C, E, G I, K, L, N, P, Q or R (Table S3). SCGL at the C-terminus of IL6 receptor was a single variant of SCGF. We screened seven variants of GRRF (GRRG was already tested as a control) i.e., GRRC, GRRE, GRRI, GRRL, GRRN, GRRQ and GRRR as well as the SCGL peptide for binding to PSMD9 by ELISA (Fig. 4A, Table S5). Peptides GRRL, GRRI, GRRQ, GRRC, GRRR and SCGL bound to PSMD9. GRRI and GRRL binding affinity were comparable to GRRF. GRRI belongs to a hypothetical protein. GRRL belongs to S14. S14 is part of the ribosome and like hnRNPA1 is an RNA binding protein also involved in protein translation[35]. GRRC and GRRR surprisingly bound with 12–14-fold higher affinity than GRRF. GRRC belongs to endothelial receptor protein and GRRR to UPF2, a protein involved in mRNA metabolism. Like SCGF, SCGL (from IL6 receptor) bound to PSMD9 with better affinity than GRRF or its variants. We tested full length S14 and IL-6 receptor C-terminal domain for binding to PSMD9 using in vitro pull down assay, and both were found to interact with PSMD9. As in hnRNPA1, C-terminal substitution abrogated binding of S14 and remarkably as seen with GH, binding of IL-6 receptor was unaffected by the C-terminal Gly substitution but was inhibited upon deletion (Fig. 4B and D). Again, similar to hnRNPA1-PDZ interaction, all the PSMD9 PDZ mutants L153G, F162G, Q181G and the triple mutant L124G/Q126G/E128G, either did not recognize or bound less well to WT S14 and the FN3 domain of the IL6 receptor with intact C-terminal residues (Fig. 4C and E).To test whether observed in vitro interactions can be extended to interactions within the cellular milieu, we cloned and trans-expressed S14 ribosomal protein and the FN3 domain of IL6 receptor and their respective C-terminal mutants, in HEK293 cells. Immunoprecipitation results clearly confirm all in vitro observations (Fig. 4F and G).

3.1A screen for putative PSMD9 interacting partners and validation using full length proteinsMany methods capitalize on the ability of the PDZ domains to recognize C-terminal residues in proteins to primarily define their binding specificity [18–22]. Peptide libraries have been created, and peptides derived from the C-terminus of the human proteome have been used by various investigators [18,23–25]. We chose C-terminal peptides of the human proteome as baits to identify novel interacting partners of PSMD9. Premise for this study is that modification-independent, sequence specific recognition is central to many biological processes, and rules inherent to this recognition process can bring together proteins of very different functions under a master regulator. Chung et al., had classified proteome from drosophila/yeast/human by recognizing conserved C-terminal residues in some of these proteins [26]. These C-terminal peptides were tested here for the following reasons. (1) Most high-throughput studies are optimized for selecting peptides with high affinity while many protein–protein interactions are of low affinity and, therefore, are likely to be missed. (2) If the corresponding protein/proteins were to interact, one could quickly move to associated functions, and finally (3) such a guided approach prevents identification of those peptides that are not represented in the human proteome and, are physiologically irrelevant. Due to financial constraints, thirteen among the thirty conserved tetra peptides from the human proteome were chosen. These sequences differ in charge, hydrophobicity and size and represent some of the known sequence specificity seen with other PDZ domains. AGHM, the C-terminus of E12 transcription factor, the human homolog of rat E2, was specifically included. E12 was shown to interact with Bridge 1 (homolog of PSMD9 with a PDZ domain) during insulin signaling [27–29]. We cloned, expressed and purified human PSMD9 and used enzyme-linked immunosorbent assay (ELISA), to test for binding of the peptides. GRRF, SCGF and AGHM peptides bound to PSMD9 to an appreciable extent with SCGF demonstrating highest affinity (Fig. 1A and Fig. S1A and B). SCGF and GRRF resemble class III PDZ peptides with the sequence motif-X-[D/E/K/R]-X-Φ where Φ is hydrophobic, and X is any residue. GRRF forms the C-terminus of hnRNPA1 isoforms while SCGF belongs to growth hormone (CSH1; referred from henceforth as GH). To test if the corresponding full length proteins would interact with PSMD9, we cloned and expressed the longer isoform of hnRNPA1 as a GST fusion protein, GH and E12 as MBP fusion proteins. PSMD9 was expressed as a His-Tag protein. Affinity pull-down followed by Western blot showed that the three full length proteins interact with PSMD9 (Fig. 1B–D). While hnRNPA1 (Fig 1E andTable S4) and E12 binding (Fig 1C) were clearly affected by simple C-terminal substitution (Phe to Gly), GH binding to PSMD9 was not affected to any measurable extent (Fig. 1D). Deletion of C-terminal seven residues compromised binding of GH severely (Fig. 1D) and not surprisingly those of hnRNPA1 and E12, as well (Fig. 1B and C). These interactions were further confirmed using ELISA and the estimated dissociation constant Kd for PSMD9-hnRNPA1 interaction is 1.33 ± 0.16 μM and of PSMD9-GH interaction is 0.74 ± 0.04 μM and ΔG for the interaction between PSMD9 and WT-hnRNPA1 or GH were calculated to be 6.9 ± 0.04 and 7.1 ± 0.09 kcal/mol, respectively. Peptide GRRF and not GRRG inhibited hnRNPA1 binding (Ki of 326.5 ± 0.25 μM) confirming the importance of C-terminal residues (Fig. 1G) in this interaction. Again, as seen with the C-terminal substitutions of GH, inhibition of GH-PSMD9 interaction by SCGG was as good as SCGF and the Ki values for these peptides were 36.7 ± 0.29 and 35.6 ± 0.24 μM, respectively (Fig. 1H). These pairwise interactions and the role of C-terminal residues were confirmed in mammalian cells using co-immunoprecipitation assays (Fig. 1I and J).

A screen for putative PSMD9 interacting partners and validation using full length proteins

Many methods capitalize on the ability of the PDZ domains to recognize C-terminal residues in proteins to primarily define their binding specificity [18–22]. Peptide libraries have been created, and peptides derived from the C-terminus of the human proteome have been used by various investigators [18,23–25]. We chose C-terminal peptides of the human proteome as baits to identify novel interacting partners of PSMD9. Premise for this study is that modification-independent, sequence specific recognition is central to many biological processes, and rules inherent to this recognition process can bring together proteins of very different functions under a master regulator. Chung et al., had classified proteome from drosophila/yeast/human by recognizing conserved C-terminal residues in some of these proteins [26]. These C-terminal peptides were tested here for the following reasons. (1) Most high-throughput studies are optimized for selecting peptides with high affinity while many protein–protein interactions are of low affinity and, therefore, are likely to be missed. (2) If the corresponding protein/proteins were to interact, one could quickly move to associated functions, and finally (3) such a guided approach prevents identification of those peptides that are not represented in the human proteome and, are physiologically irrelevant. Due to financial constraints, thirteen among the thirty conserved tetra peptides from the human proteome were chosen. These sequences differ in charge, hydrophobicity and size and represent some of the known sequence specificity seen with other PDZ domains. AGHM, the C-terminus of E12 transcription factor, the human homolog of rat E2, was specifically included. E12 was shown to interact with Bridge 1 (homolog of PSMD9 with a PDZ domain) during insulin signaling [27–29]. We cloned, expressed and purified human PSMD9 and used enzyme-linked immunosorbent assay (ELISA), to test for binding of the peptides. GRRF, SCGF and AGHM peptides bound to PSMD9 to an appreciable extent with SCGF demonstrating highest affinity (Fig. 1A and Fig. S1A and B). SCGF and GRRF resemble class III PDZ peptides with the sequence motif-X-[D/E/K/R]-X-Φ where Φ is hydrophobic, and X is any residue. GRRF forms the C-terminus of hnRNPA1 isoforms while SCGF belongs to growth hormone (CSH1; referred from henceforth as GH). To test if the corresponding full length proteins would interact with PSMD9, we cloned and expressed the longer isoform of hnRNPA1 as a GST fusion protein, GH and E12 as MBP fusion proteins. PSMD9 was expressed as a His-Tag protein. Affinity pull-down followed by Western blot showed that the three full length proteins interact with PSMD9 (Fig. 1B–D). While hnRNPA1 (Fig 1E andTable S4) and E12 binding (Fig 1C) were clearly affected by simple C-terminal substitution (Phe to Gly), GH binding to PSMD9 was not affected to any measurable extent (Fig. 1D). Deletion of C-terminal seven residues compromised binding of GH severely (Fig. 1D) and not surprisingly those of hnRNPA1 and E12, as well (Fig. 1B and C). These interactions were further confirmed using ELISA and the estimated dissociation constant Kd for PSMD9-hnRNPA1 interaction is 1.33 ± 0.16 μM and of PSMD9-GH interaction is 0.74 ± 0.04 μM and ΔG for the interaction between PSMD9 and WT-hnRNPA1 or GH were calculated to be 6.9 ± 0.04 and 7.1 ± 0.09 kcal/mol, respectively. Peptide GRRF and not GRRG inhibited hnRNPA1 binding (Ki of 326.5 ± 0.25 μM) confirming the importance of C-terminal residues (Fig. 1G) in this interaction. Again, as seen with the C-terminal substitutions of GH, inhibition of GH-PSMD9 interaction by SCGG was as good as SCGF and the Ki values for these peptides were 36.7 ± 0.29 and 35.6 ± 0.24 μM, respectively (Fig. 1H). These pairwise interactions and the role of C-terminal residues were confirmed in mammalian cells using co-immunoprecipitation assays (Fig. 1I and J).

3.2The fine specificity of SCG derivativesUnlike hnRNPA1 GRRG mutant, mutant GH with a C-terminal substituted SCGG binds to the PDZ domain of PSMD9 and interaction is inhibited only upon deletion of C-terminal residues (Δ7 mutant). To identify the minimal motif important for GH interaction, we engineered ΔGF, and ΔCGF mutants of GH and interaction with PSMD9 was tested by pull down and ELISA (Fig. 2 A and B and Table S4). While ΔGF mutant bound with PSMD9, deletion of one more residue, Cysteine, ΔGFC, impaired the interaction. By ELISA, the estimated Kd values were 0.8 ± 0.02 μM for ΔGF and 2.6 ± 0.011 μM for ΔCGF mutant. The % occupancy of GH was unaltered in the ΔGF mutant but was reduced to ∼45% in the case of the ΔCGF mutant. This result emphasizes the importance of P−2 residue in interaction with PSMD9. The importance of the P−2 Cys was further confirmed by demonstrating the failure of peptide SGGF to inhibit the binding of GH to PSMD9 (Fig. 2C). As noted before both SCGF and SCGG can inhibit binding between the two proteins.Our results help to clarify some of the observations made earlier with respect to Nas-2-Rpt5 interaction in yeast (PSMD9 homolog and the ATPase subunit of the 19S regulatory particle). Here, single C-terminal residue deletion in Rpt5 did not affect its binding to Nas-2 that made the authors conclude that the PDZ like domain of Nas2 may not confirm to the classical description [30]. Based on our results on human PSMD9 using similar pull down assays, other comparative studies and quantitative analysis, we show that the precise role of the C-terminal residues in the interaction is likely to be context dependent. In the case of hnRNPA1 (GRRF) and E12 (AGHM), bulk of the binding energy is derived from the C-terminal residue much like the classical PDZ domains. In GH with SCGF at the C-terminus, however, the terminal residue is less important. These differences are also reflected in the binding affinity of the three peptides to PSMD9. While GRRF binds weakly (Kd 651.7 ± 76 μM), peptide SCGF binds tightly to PSMD9 (Kd 8.6 ± 1.2 μM). One possible explanation is that these peptides may bind in different modes or orientations at the binding groove (discussed below). While results observed with the C-terminal peptides can be readily extrapolated to protein binding, stable binding of the full length protein may require additional interactions. It is also likely that, besides the canonical α–β groove, the protein, may bind elsewhere on PSMD9 perhaps at an allosteric site while the C-terminal sequence acts as initial recognition element that docks the protein at the canonical site.

The fine specificity of SCG derivatives

Unlike hnRNPA1 GRRG mutant, mutant GH with a C-terminal substituted SCGG binds to the PDZ domain of PSMD9 and interaction is inhibited only upon deletion of C-terminal residues (Δ7 mutant). To identify the minimal motif important for GH interaction, we engineered ΔGF, and ΔCGF mutants of GH and interaction with PSMD9 was tested by pull down and ELISA (Fig. 2 A and B and Table S4). While ΔGF mutant bound with PSMD9, deletion of one more residue, Cysteine, ΔGFC, impaired the interaction. By ELISA, the estimated Kd values were 0.8 ± 0.02 μM for ΔGF and 2.6 ± 0.011 μM for ΔCGF mutant. The % occupancy of GH was unaltered in the ΔGF mutant but was reduced to ∼45% in the case of the ΔCGF mutant. This result emphasizes the importance of P−2 residue in interaction with PSMD9. The importance of the P−2 Cys was further confirmed by demonstrating the failure of peptide SGGF to inhibit the binding of GH to PSMD9 (Fig. 2C). As noted before both SCGF and SCGG can inhibit binding between the two proteins.

Our results help to clarify some of the observations made earlier with respect to Nas-2-Rpt5 interaction in yeast (PSMD9 homolog and the ATPase subunit of the 19S regulatory particle). Here, single C-terminal residue deletion in Rpt5 did not affect its binding to Nas-2 that made the authors conclude that the PDZ like domain of Nas2 may not confirm to the classical description [30]. Based on our results on human PSMD9 using similar pull down assays, other comparative studies and quantitative analysis, we show that the precise role of the C-terminal residues in the interaction is likely to be context dependent. In the case of hnRNPA1 (GRRF) and E12 (AGHM), bulk of the binding energy is derived from the C-terminal residue much like the classical PDZ domains. In GH with SCGF at the C-terminus, however, the terminal residue is less important. These differences are also reflected in the binding affinity of the three peptides to PSMD9. While GRRF binds weakly (Kd 651.7 ± 76 μM), peptide SCGF binds tightly to PSMD9 (Kd 8.6 ± 1.2 μM). One possible explanation is that these peptides may bind in different modes or orientations at the binding groove (discussed below). While results observed with the C-terminal peptides can be readily extrapolated to protein binding, stable binding of the full length protein may require additional interactions. It is also likely that, besides the canonical α–β groove, the protein, may bind elsewhere on PSMD9 perhaps at an allosteric site while the C-terminal sequence acts as initial recognition element that docks the protein at the canonical site.

3.3Role of PDZ domain in interaction: modeling and site directed mutagenesisTo better understand the role of the C-terminal residues and PDZ domain in binding and recognition, we modeled the structure of PDZ and carried out extensive molecular dynamic simulations and peptide docking studies (supplementary methods). Several docking poses were created. Upon visual inspection of all the docked poses, a peptide-protein complex similar to that seen in the co-crystals of other PDZ-peptide complex with Phe at the fourth position was chosen. In this conformation, the peptide binds in an extended, antiparallel manner through canonical interactions that extend the beta sheet by an additional strand (Fig. 3A and B). The hydrophobic side chain of Phe4 of the peptide is deeply buried in the hydrophobic pocket formed by Leu124 from β2, Val139, from β3, Leu153 from β4, Ile159, Phe 162, from β4. The peptide further interacts with the beta sheet mainly through backbone/side chain hydrogen bonds with residues Leu124, Gly125, Gln126, Glu128 of β2 of the PDZ domain (Fig. 2B). In addition, the side chain of Arg2 of the peptide forms a salt bridge with the side chain of Glu128 from β2. During MD simulation, the alpha/beta binding groove (canonical binding site) of apo PDZ showed increased flexibility (Supplementary Fig. S2). The α2/β2 binding pocket was partially deformed/destabilized (either collapses or widens), and is stabilized upon peptide binding. Increased flexibility of PDZ domains in their apo form have been reported by others [31]. The intrinsic flexibility of PDZ domains is a key determinant that allows them to recognize a wide repertoire of peptide ligands. Throughout the protein-peptide simulation, Phe4 remains deeply buried in the hydrophobic pocket (Movie1:http://web.bii.a-star.edu.sg/bmad/PDZ/PDZ-PEP-WT-Top.mpg). Charge-charge interactions between Arg2 and Glu128 on β2 are preserved during the 100 ns simulation. The bound conformation of the peptide was further stabilized via backbone hydrogen bond interactions with residues Leu124, Gly125, Gln126 and Glu128 from β2 in the canonical binding site.In the complex where Phe4 was mutated to Gly, the peptide unbinds from the canonical binding site within ∼5–10 ns and doesn’t bind again (Movie 2:http://web.bii.a-star.edu.sg/bmad/PDZ/PDZ-PEP_GRRG-Top.mpg). Although the peptide stays close to the canonical site due to charge-charge interactions with the protein residues, it undergoes translation and rotations that prevent it from rebinding in the canonical interaction mode. Thus, our MD simulations suggest that the burial of Phe in the hydrophobic pocket is crucial for the stabilization of this peptide in its bound conformation. Based on peptide docking and MD simulations (Movie 3:http://web.bii.a-star.edu.sg/bmad/PDZ/PDZPEP_L124G_Q126G_E128G-Top.mpg and Movie 4: http://web.bii.a-star.edu.sg/bmad/PDZ/PDZ-PEP_Q181G-Top.mpg), three single amino acid mutations F162G, L153G, Q181G and a triple mutation, L124G/Q126G/E128G were generated. In vitro pull-down shows that these mutations affect GH and hnRNPA1 binding to PSMD9 (Fig. 3C and D). Mutation of residue L173 (to Gly), part of the α2 helix, not involved in the interaction, did not affect the binding of peptide or the proteins (Fig. 3E and Table S1). MD simulations support this finding as the L173G PSMD9 mutant maintains the peptide in a stably bound form (not shown).These results together, confirm the domain-motif interaction between PDZ domain of PSMD9 and the C-terminal region of the interacting proteins. The instability of the peptide-free forms is reflected in the secondary structure of these proteins determined by circular dichroism. While WT PSMD9 records 49% helicity, the L173G mutant shows 43% helical structure, Q181G mutant 39%, L153G mutant 45% and the F162G mutant shows 42% helical structure (Fig. 3F, Table 1 and Appendix Eq.(1)) [32]. Tryptophan fluorescence of these mutant proteins is less affected (Fig. 3G).

Role of PDZ domain in interaction: modeling and site directed mutagenesis

To better understand the role of the C-terminal residues and PDZ domain in binding and recognition, we modeled the structure of PDZ and carried out extensive molecular dynamic simulations and peptide docking studies (supplementary methods). Several docking poses were created. Upon visual inspection of all the docked poses, a peptide-protein complex similar to that seen in the co-crystals of other PDZ-peptide complex with Phe at the fourth position was chosen. In this conformation, the peptide binds in an extended, antiparallel manner through canonical interactions that extend the beta sheet by an additional strand (Fig. 3A and B). The hydrophobic side chain of Phe4 of the peptide is deeply buried in the hydrophobic pocket formed by Leu124 from β2, Val139, from β3, Leu153 from β4, Ile159, Phe 162, from β4. The peptide further interacts with the beta sheet mainly through backbone/side chain hydrogen bonds with residues Leu124, Gly125, Gln126, Glu128 of β2 of the PDZ domain (Fig. 2B). In addition, the side chain of Arg2 of the peptide forms a salt bridge with the side chain of Glu128 from β2. During MD simulation, the alpha/beta binding groove (canonical binding site) of apo PDZ showed increased flexibility (Supplementary Fig. S2). The α2/β2 binding pocket was partially deformed/destabilized (either collapses or widens), and is stabilized upon peptide binding. Increased flexibility of PDZ domains in their apo form have been reported by others [31]. The intrinsic flexibility of PDZ domains is a key determinant that allows them to recognize a wide repertoire of peptide ligands. Throughout the protein-peptide simulation, Phe4 remains deeply buried in the hydrophobic pocket (Movie1:http://web.bii.a-star.edu.sg/bmad/PDZ/PDZ-PEP-WT-Top.mpg). Charge-charge interactions between Arg2 and Glu128 on β2 are preserved during the 100 ns simulation. The bound conformation of the peptide was further stabilized via backbone hydrogen bond interactions with residues Leu124, Gly125, Gln126 and Glu128 from β2 in the canonical binding site.

In the complex where Phe4 was mutated to Gly, the peptide unbinds from the canonical binding site within ∼5–10 ns and doesn’t bind again (Movie 2:http://web.bii.a-star.edu.sg/bmad/PDZ/PDZ-PEP_GRRG-Top.mpg). Although the peptide stays close to the canonical site due to charge-charge interactions with the protein residues, it undergoes translation and rotations that prevent it from rebinding in the canonical interaction mode. Thus, our MD simulations suggest that the burial of Phe in the hydrophobic pocket is crucial for the stabilization of this peptide in its bound conformation. Based on peptide docking and MD simulations (Movie 3:http://web.bii.a-star.edu.sg/bmad/PDZ/PDZPEP_L124G_Q126G_E128G-Top.mpg and Movie 4: http://web.bii.a-star.edu.sg/bmad/PDZ/PDZ-PEP_Q181G-Top.mpg), three single amino acid mutations F162G, L153G, Q181G and a triple mutation, L124G/Q126G/E128G were generated. In vitro pull-down shows that these mutations affect GH and hnRNPA1 binding to PSMD9 (Fig. 3C and D). Mutation of residue L173 (to Gly), part of the α2 helix, not involved in the interaction, did not affect the binding of peptide or the proteins (Fig. 3E and Table S1). MD simulations support this finding as the L173G PSMD9 mutant maintains the peptide in a stably bound form (not shown).

These results together, confirm the domain-motif interaction between PDZ domain of PSMD9 and the C-terminal region of the interacting proteins. The instability of the peptide-free forms is reflected in the secondary structure of these proteins determined by circular dichroism. While WT PSMD9 records 49% helicity, the L173G mutant shows 43% helical structure, Q181G mutant 39%, L153G mutant 45% and the F162G mutant shows 42% helical structure (Fig. 3F, Table 1 and Appendix Eq.(1)) [32]. Tryptophan fluorescence of these mutant proteins is less affected (Fig. 3G).

3.4Identification of putative functional modules regulated by PSMD9Although GRRF and SCGF were motifs under which several family members (12 and 13 respectively) were grouped by Chung et al., a detailed analysis and further curation using UniProt data (ftp://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/) indicated that there was only one unique protein under each family. There are four isoforms within the GRRF family and ten isoforms within the SCGF family (Table S2). We re-analyzed other 28 families and found that, in the vast majority of the cases, the proteins grouped under each peptide family are primarily isoforms (Table S2). Although isoforms are homologous in sequence, their functions can be mutually exclusive or even counteractive [33,34]. To better define the role of C-terminus in functional grouping beyond isoforms, and predict the modules that may be regulated by PSMD9, we analyzed C-terminal variants of GRRF and SCGF from the human proteome. There are ten variants of GRRF where X is C, E, G I, K, L, N, P, Q or R (Table S3). SCGL at the C-terminus of IL6 receptor was a single variant of SCGF. We screened seven variants of GRRF (GRRG was already tested as a control) i.e., GRRC, GRRE, GRRI, GRRL, GRRN, GRRQ and GRRR as well as the SCGL peptide for binding to PSMD9 by ELISA (Fig. 4A, Table S5). Peptides GRRL, GRRI, GRRQ, GRRC, GRRR and SCGL bound to PSMD9. GRRI and GRRL binding affinity were comparable to GRRF. GRRI belongs to a hypothetical protein. GRRL belongs to S14. S14 is part of the ribosome and like hnRNPA1 is an RNA binding protein also involved in protein translation[35]. GRRC and GRRR surprisingly bound with 12–14-fold higher affinity than GRRF. GRRC belongs to endothelial receptor protein and GRRR to UPF2, a protein involved in mRNA metabolism. Like SCGF, SCGL (from IL6 receptor) bound to PSMD9 with better affinity than GRRF or its variants. We tested full length S14 and IL-6 receptor C-terminal domain for binding to PSMD9 using in vitro pull down assay, and both were found to interact with PSMD9. As in hnRNPA1, C-terminal substitution abrogated binding of S14 and remarkably as seen with GH, binding of IL-6 receptor was unaffected by the C-terminal Gly substitution but was inhibited upon deletion (Fig. 4B and D). Again, similar to hnRNPA1-PDZ interaction, all the PSMD9 PDZ mutants L153G, F162G, Q181G and the triple mutant L124G/Q126G/E128G, either did not recognize or bound less well to WT S14 and the FN3 domain of the IL6 receptor with intact C-terminal residues (Fig. 4C and E).To test whether observed in vitro interactions can be extended to interactions within the cellular milieu, we cloned and trans-expressed S14 ribosomal protein and the FN3 domain of IL6 receptor and their respective C-terminal mutants, in HEK293 cells. Immunoprecipitation results clearly confirm all in vitro observations (Fig. 4F and G).

Identification of putative functional modules regulated by PSMD9

Although GRRF and SCGF were motifs under which several family members (12 and 13 respectively) were grouped by Chung et al., a detailed analysis and further curation using UniProt data (ftp://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/) indicated that there was only one unique protein under each family. There are four isoforms within the GRRF family and ten isoforms within the SCGF family (Table S2). We re-analyzed other 28 families and found that, in the vast majority of the cases, the proteins grouped under each peptide family are primarily isoforms (Table S2). Although isoforms are homologous in sequence, their functions can be mutually exclusive or even counteractive [33,34]. To better define the role of C-terminus in functional grouping beyond isoforms, and predict the modules that may be regulated by PSMD9, we analyzed C-terminal variants of GRRF and SCGF from the human proteome. There are ten variants of GRRF where X is C, E, G I, K, L, N, P, Q or R (Table S3). SCGL at the C-terminus of IL6 receptor was a single variant of SCGF. We screened seven variants of GRRF (GRRG was already tested as a control) i.e., GRRC, GRRE, GRRI, GRRL, GRRN, GRRQ and GRRR as well as the SCGL peptide for binding to PSMD9 by ELISA (Fig. 4A, Table S5). Peptides GRRL, GRRI, GRRQ, GRRC, GRRR and SCGL bound to PSMD9. GRRI and GRRL binding affinity were comparable to GRRF. GRRI belongs to a hypothetical protein. GRRL belongs to S14. S14 is part of the ribosome and like hnRNPA1 is an RNA binding protein also involved in protein translation[35]. GRRC and GRRR surprisingly bound with 12–14-fold higher affinity than GRRF. GRRC belongs to endothelial receptor protein and GRRR to UPF2, a protein involved in mRNA metabolism. Like SCGF, SCGL (from IL6 receptor) bound to PSMD9 with better affinity than GRRF or its variants. We tested full length S14 and IL-6 receptor C-terminal domain for binding to PSMD9 using in vitro pull down assay, and both were found to interact with PSMD9. As in hnRNPA1, C-terminal substitution abrogated binding of S14 and remarkably as seen with GH, binding of IL-6 receptor was unaffected by the C-terminal Gly substitution but was inhibited upon deletion (Fig. 4B and D). Again, similar to hnRNPA1-PDZ interaction, all the PSMD9 PDZ mutants L153G, F162G, Q181G and the triple mutant L124G/Q126G/E128G, either did not recognize or bound less well to WT S14 and the FN3 domain of the IL6 receptor with intact C-terminal residues (Fig. 4C and E).

To test whether observed in vitro interactions can be extended to interactions within the cellular milieu, we cloned and trans-expressed S14 ribosomal protein and the FN3 domain of IL6 receptor and their respective C-terminal mutants, in HEK293 cells. Immunoprecipitation results clearly confirm all in vitro observations (Fig. 4F and G).

4DiscussionOur results taken together indicate that PSMD9 carries a versatile PDZ domain and interacts with residues at the C-terminus of proteins that are non-homologous in sequence, but carry a signature Short Linear Sequence Motif. Although the number of peptides screened here is limited, substantial information can be inferred from the binding of peptides and proteins to the PDZ domain of PSMD9 and their mutant forms. Given that the information on the structure and functions of PSMD9 (and other 19S subunits) is minimal, the results reported here are highly significant. However, some amount of speculation drawing support from our own studies and those from the literature is necessary to appreciate the significance of the results.4.1On the origin of affinity differencesWe had included 8 out of 10 C-terminal variants of GRRF, and SCGL a single variant of SCGF, from the human proteome and peptide AGHM from transcription factor E12, for their ability to interact with PSMD9. The Kd for each of these peptides is summarized in Table S3. Based on the affinity of the peptides, these variants can be classified into three groups – Group I or low affinity binders, Group II or high affinity binders and Group III tight binders or the top ranking peptides. In the Group I peptides, hydrophobic residues such as F, L, I at P0 provide specificity. In Group II peptides, Cys or Arg at P0 increases affinity by 10-fold as compared to that of Group I peptides. These two amino acids seem very different from each other and from the Group I peptides in terms of their physical properties and binding preference of the PSMD9 PDZ domain seems very intricate. The binding pocket of PSMD9 seems better adapted to bind to residues that are not bulky or highly hydrophobic explaining the high affinity binding of GRRR and GRRC. On the other hand, both Cys and Arg show characteristics of hydrophobic residues. For example, based on the hydrophobicity index Cys is classified along with Phe [36–39] and arginine, although one of the least hydrophobic amino acids, shows very interesting properties. Arginine solubilizes aggregation prone proteins helps in the elution of proteins bound to phenylsepharose column and has wide application in the purification and solubilization of inclusion bodies [40,41]. Arginine like GuHCL interacts with almost all amino acids and preferentially with aromatic residues [40], but unlike GuHCl, Arginine is not a denaturant [42]. This probably explains why these two amino acids like the hydrophobic residues occupy the P0 position. Arg substitution for a Phe in the interior of a protein will result in destabilization but less likely to do so at the protein interface.Based on binding affinity, peptide AGHM will also fall under the Group I peptides, and methionine is known to be a hydrophobic residue. GRRE with a negatively charged C-terminus and GRRG with a small but relatively hydrophobic residue at P0 do not bind to PSMD9. These results indicate that P0 residue and not GRR is a major determinant of binding specificity in these peptides.Compared to all peptides tested, the top ranking Group III peptides, SCG variants SCGF and SCGL bind with the highest affinity – Kd for the two peptides is four to five times less than the Group II peptides. The hydrophobicity of the C-terminal residue in SCGF and SCGL is clearly not important for binding and recognition as it can be readily replaced by a Gly. It seems that, in this set of peptides, the P−1, P−2 or P−3 residues are more important for high affinity interaction. By systematically deleting residues from the C-terminus, we identified Cys at P−2 position to be very important for interaction. In accordance with these results, peptide SGGF was unable to inhibit the interaction between GH and PSMD9.It is clear that, in the absence of high resolution crystal structure of the complexes coupled with kinetic, thermodynamic studies using mutant peptides, it would be impossible to precisely define the molecular basis of affinity differences and positional occupancy of residues. In the absence of these details, we will have to consider different possibilities that may account for the binding preferences and affinity.Reports from 20 complex structures of PDZ domains with C-terminal peptides of proteins indicate that the aminoacid at the P0 position has no specific conformational preference in the Ramachandran plot. In contrast the P−1, P−2 and P−3 residues show a strong preference and occupy either a strand or an extended conformation [43]. Such a conformational preference especially of the P−2 residue may explain the high affinity interaction seen with SCGX peptides. Alternatively, SCGF and SCGL peptides may mimic the internal sequences in proteins that bind PDZ domains and the Cys at P−2 may occupy the hydrophobic pocket formed by L153 and Phe 162 residues, mutations of which affect interaction. Flexibility in the binding modes is not uncommon to the peptide-PDZ domain interactions. For e.g., the P(−2) residue in some of the PDZ ligands are known to interact with αB-1 and αB-5 residues on the PDZ domain [43,44]. These residues normally interact with the P0 residues in the ligand. In the crystal structure of Dvl2 PDZ domain bound to a noncanonical C-terminal sequence, P−3 residue was seen to occupy the binding position utilized by a P−2 residue [45]. Secondary structure of the PSMD9 mutant proteins F162G (42%) and L153G (45%; very close to WT ∼48%) were not dramatically altered compared to other mutant proteins which bind the peptide (L173G 42%) or those that do not (Q181G 38%; Table 1).Three modes of peptide binding to PDZ domains in proteins GRASP, PDLIM and MAST4 have been identified. In the structures of GRASP-peptide complexes, it is striking that the two chains of the protein bind to the same peptide in two different binding modes. Comparing these structures, a perpendicular mode, an intermediate mode – both speculated to be kinetic intermediates- and a stable canonical binding mode have been described [44]. We can draw parallels from these studies and propose the following: there exists a conformational ensemble of peptide-PSMD9 complexes. The Group I peptides, probably frequent the non-canonical or perpendicular orientation seen with other PDZ binding peptides. This orientation will rely heavily on the burial of the C-terminal residue for affinity. The Group II peptides GRRC and GRRR peptides probably frequent the intermediate population wherein the P0 residue is anchored. Peptides SCGF and SCGL populate the extended conformation in the canonical mode (although simulations propose a stable binding in the reverse orientation). The entropy cost of binding is probably paid for the SCGF and SCGL peptides as described for other protein derived C-terminal peptides bound to their cognate PDZ domain that may explain the high affinity interaction.In our MD simulation studies, SCGF was unstable in the canonical binding mode but binds stably in a fully extended form, in the reverse orientation (Supplementary Fig. S3). Reverse binding modes of peptides have also been reported in literature, where the same peptide binds in opposite orientation i.e., N′–C′ or C′–N′ termini e.g., peptides binding to chaperone DnaK, Calmodulin and SH3 proteins [46–48]. If SCGF or SCGL peptides bind in a reverse orientation with the hydroxyl-group of the Ser residue substituting for the Phe carboxyl residue, mutation of F162 or L153 residues to Gly, will affect the binding, as seen in Supplementary Fig. S4. However, since the mutant peptide SGGF is unable to bind to PSMD9 or inhibit the binding of GH to PSMD9, the Ser residue seems unimportant for interaction. Therefore, binding of SCGF/L in the reverse orientation as a probable determinant of high affinity interaction seems less likely.4.2On the number of binding sites and the mechanism of bindingMutation of the C-terminal Phe in GRRF, or Leu in GRRL or Met in AGHM, to Gly in the respective peptides or proteins inhibits interaction. Commensurate with these results, while GRRF competitively inhibits approximately 69% of the binding between hnRNPA1 and PSMD9 the mutant peptide GRRG, is unable to do so. However, substitution and even deletion of few C-terminal residues does not completely prevent binding of proteins. In all the cases, the fractional occupancy (like Vmax in enzyme catalysis) is maximally affected by the C-terminal mutations while the affinity per se as determined by Kd (like Km for substrate binding) is less affected. In addition, the affinity differences between peptide or protein binding to PSMD9 is large. This difference is especially striking with the Group I peptides, the Kd of which are in the high μM range (∼600 μM) while the proteins bind with low micromolar affinity (1 μM). This vast discrepancy may be explained by the following: (a) C-terminal residues act primarily as signatures or bar codes that are read by the PDZ domain of PSMD9; (b) maximal affinity is contributed by a binding motif elsewhere in the protein or the binding of extended residues at the N-termini of the protein. Peptide affinity, however increases only marginally upon extension of the N-terminal residues. For example, the nine residue C-terminal sequence of E12, with the tetrapeptide motif AGHM, interacts approximately two times more strongly than the short tetrapeptide AGHM (data not shown); and (c) the C-terminal sequences are stabilized by the structure of the full length protein. Based on these possibilities we propose a two state binding model for the interaction between PSMD9 and its client proteins – an initial weak recognition phase mediated by the C-terminal residues which act as specificity determinants followed by its consolidation via interaction of a secondary binding motif. Initial recognition of C-terminal residues in proteins by the PDZ domain may allow binding of the secondary site to an allosteric pocket on PSMD9 (Supplementary Fig. S5). These may or may not be accompanied by conformational changes in the proteins. The binding disparity between Group II peptides or the SCGF and SCGL peptides and their corresponding proteins although is much less (of the order of 8-fold), the fractional occupancy of the C-terminal deletion mutant, ΔCGF of GH is ∼45% of WT indicating that the same mechanism is probably operational. The plasticity of the PDZ domain and contribution of the overall structure to the binding is well illustrated in [31,49]. The partial loss in affinity upon mutations of residues present in different secondary structural elements of the PSMD9-PDZ domain is probably a reflection of the same paradigm.It is also possible that the binding affinities are a reflection of the associated functions of the proteins. For example, we have demonstrated that hnRNPA1 acts as a novel shuttle receptor [7] that recruits IκBα for degradation by the proteasome. PSMD9 by interacting with hnRNPA1 and the 26S proteasome helps in anchoring IκBα and accelerating degradation. In this process, hnRNPA1 is likely to be recycled. By analogy, S14 may also act as a shuttle receptor that recruits proteins like MDM2 for degradation (Fig. 5B, see Section 4.3 for details) and must itself be released intact. Therefore, the C-terminus of these proteins may bind weakly to PSMD9. GH and IL6 receptors are the proposed direct substrates of the proteasome (Fig. 5D see Section 4.3 for details). In addition to the polyubiquitin binding, initial recognition of the C-terminal residues by PSMD9 with high affinity may be very important for the stable binding of such direct substrates to provide fatal directionality for degradation (Fig. 5D).4.3On the functional annotation of PSMD9 and its role in quality control by the proteasomeIn this third part, we speculate on the probable regulatory role of PSMD9 by inferring on the role of its interacting partners. It is interesting that these proteins perform very different functions in the cell. At first glance, these interactions seem unusual, and the real physiological relevance may not be apparent. But detailed literature study provides substantial support for the plausible physiological role of these interactions in mammalian cells. hnRNPA1 is known to interact with IκBα in murine cells, and this interaction somehow accelerates degradation of IκBα resulting in NF-κB activation [50]. A possible functional conservation can be expected in human cells, and one may anticipate PSMD9 to regulate NF-κB signaling via IκBα degradation. In the manuscript that we published recently, we show that hnRNPA1 is a shuttle receptor that recruits IκBα for degradation and PSMD9 acts as a subunit acceptor and anchors hnRNPA1 to facilitate degradation of IκBα by the proteasome [7]. Association of proteasome with ribosome has been documented in the literature [51–53]. Whether S14 and PSMD9 interaction provides the structural scaffold for this interaction and what may be the functional consequence of this interaction in protein translation will be an interesting future investigation. In addition, S14 is known to bind to MDM2, which prevents the ability of this E3 ligase to ubiquitinate p53 thereby preventing proteasomal degradation of p53, leading to stabilization and activation of p53 [54]. Depending on whether or not the interaction between PSMD9 and S14, S14 and MDM2 are mutually exclusive, PSMD9 may influence ubiquitination, stability and functions of p53. By drawing a parallel from our studies on hnRNPA1 and PSMD9 interaction in IκBα degradation, we provide an alternate possibility for the fate of MDM2 and p53. We speculate that S14, similar to hnRNPA1 may also act as a shuttle receptor which recruits MDM2 for degradation by the proteasome (may be under similar stress conditions). Proteasome associated PSMD9 may anchor S14 (like it does hnRNPA1) to facilitate degradation of MDM2 (like IκBα) by the proteasome.GRRR as mentioned before belongs to UPF2, yet another protein involved in RNA metabolism. UPF2 is part of a post-splicing multiprotein complex which regulate mRNA nuclear export and responsible for the detection of exported mRNAs with truncated open reading frames, resulting in nonsense-mediated mRNA decay [55].Two other PSMD9 interacting partners GH (CHS1) and IL6 receptor are implicated in chemokine signaling [56,57]. However both GH and the IL6 receptor with SCGL at the C-termini created by alternate splicing are soluble, secreted proteins. In order to be exported out of the cell, these proteins must follow the ER-Golgi traffic [58]. Secreted proteins are inserted co-translationally into the ER lumen. Upon achieving their final folded state and post-translational modifications, these proteins are transported out of ER, through Golgi and finally out of the plasma membrane. However under stress or stimuli induced signaling, when the load on ER is more, quality control mechanisms must ensure that misfolded proteins are degraded. Misfolded and immature proteins are retro translocated by ER resident translocon and aided by ubiquitin or specialized proteins like Kar2p/BiP they are actively pulled out. These proteins are bound by the ER associated proteasomes and degraded [59–65]. Here, we speculate that while other ubiquitin binding proteins on the 19S regulatory complex of the proteasome may bind to the polyubiquitinated GH or IL6 as classically demonstrated for other proteins, PSMD9 would trap the misfolded GH or IL6R (and other such ERAD proteins) via the specific interaction of the PDZ domain with the respective C-terminal motif. Such interaction would prevent their premature release back into ER, ensure directionality and rapid clearance by the proteasome. Degradation is likely to occur at a basal rate as a routine quality control measure. The rate would be accelerated upon signal induction to truncate and attenuate the signaling process upon withdrawal of the stimuli or in response to ER stress. Therefore, PSMD9 may act as a general molecular chaperone that exerts quality control.Soluble IL6R formed by splicing or proteolysis also bind to gp130 present on cells that lack the IL6 receptors. Soluble IL6R-IL6 complex is involved in what is called as the ‘trans-signaling’ an important mediator of inflammation and chemokine signaling in cancer [66]. Signaling may be attenuated by the receptor mediated endocytosis [67]. By binding to its receptor, GH may also follow receptor mediated endocytosis. The fate of such proteins engulfed by the process of endocytosis and how these may encounter the proteasome is described elegantly [60,68–71]. Some of these proteins routed by endocytosis may become shared substrates of the ‘cytosolic’ proteasomes and lysosomal enzymes. Different parts of the same sequence may be degraded by these degradation machineries. Our experiments designed to verify the binding of these proteins to PSMD9 in HEK293 cells upon co-expression is justified as these interactions are likely to be post endocytosis or post extraction events. Future lines of exciting studies include the characterization of the subcellular loci of these proteins and their ultimate and probably differential fate at the ‘hands’ of proteasome and lysosomes. GRRC belongs to endothelial receptor protein, yet another type I transmembrane protein, involved in signaling by EPCR [72].E12-PSMD9 interaction is likely to influence transcriptional regulation (like Bridge-1 in insulin signaling). PSMD9 may act either as a coactivator or as a repressor of many transcription events. Whether this regulatory role would involve 19S, or the entire 26S proteasome again or a proteasome independent role at the chromatin remains to be seen.With all these examples, the grand or unifying role of PSMD9 seems to be to ensure quality control and regulate the magnitude of signaling or transcriptional programs (working model Fig. 5). The probable mechanism is likely to involve the proteasome and its proteolytic components. However, other regulatory steps involving an independent pool of PSMD9 and its interacting partners within the protein–protein interaction network cannot be ruled out.

Our results taken together indicate that PSMD9 carries a versatile PDZ domain and interacts with residues at the C-terminus of proteins that are non-homologous in sequence, but carry a signature Short Linear Sequence Motif. Although the number of peptides screened here is limited, substantial information can be inferred from the binding of peptides and proteins to the PDZ domain of PSMD9 and their mutant forms. Given that the information on the structure and functions of PSMD9 (and other 19S subunits) is minimal, the results reported here are highly significant. However, some amount of speculation drawing support from our own studies and those from the literature is necessary to appreciate the significance of the results.

4.1On the origin of affinity differencesWe had included 8 out of 10 C-terminal variants of GRRF, and SCGL a single variant of SCGF, from the human proteome and peptide AGHM from transcription factor E12, for their ability to interact with PSMD9. The Kd for each of these peptides is summarized in Table S3. Based on the affinity of the peptides, these variants can be classified into three groups – Group I or low affinity binders, Group II or high affinity binders and Group III tight binders or the top ranking peptides. In the Group I peptides, hydrophobic residues such as F, L, I at P0 provide specificity. In Group II peptides, Cys or Arg at P0 increases affinity by 10-fold as compared to that of Group I peptides. These two amino acids seem very different from each other and from the Group I peptides in terms of their physical properties and binding preference of the PSMD9 PDZ domain seems very intricate. The binding pocket of PSMD9 seems better adapted to bind to residues that are not bulky or highly hydrophobic explaining the high affinity binding of GRRR and GRRC. On the other hand, both Cys and Arg show characteristics of hydrophobic residues. For example, based on the hydrophobicity index Cys is classified along with Phe [36–39] and arginine, although one of the least hydrophobic amino acids, shows very interesting properties. Arginine solubilizes aggregation prone proteins helps in the elution of proteins bound to phenylsepharose column and has wide application in the purification and solubilization of inclusion bodies [40,41]. Arginine like GuHCL interacts with almost all amino acids and preferentially with aromatic residues [40], but unlike GuHCl, Arginine is not a denaturant [42]. This probably explains why these two amino acids like the hydrophobic residues occupy the P0 position. Arg substitution for a Phe in the interior of a protein will result in destabilization but less likely to do so at the protein interface.Based on binding affinity, peptide AGHM will also fall under the Group I peptides, and methionine is known to be a hydrophobic residue. GRRE with a negatively charged C-terminus and GRRG with a small but relatively hydrophobic residue at P0 do not bind to PSMD9. These results indicate that P0 residue and not GRR is a major determinant of binding specificity in these peptides.Compared to all peptides tested, the top ranking Group III peptides, SCG variants SCGF and SCGL bind with the highest affinity – Kd for the two peptides is four to five times less than the Group II peptides. The hydrophobicity of the C-terminal residue in SCGF and SCGL is clearly not important for binding and recognition as it can be readily replaced by a Gly. It seems that, in this set of peptides, the P−1, P−2 or P−3 residues are more important for high affinity interaction. By systematically deleting residues from the C-terminus, we identified Cys at P−2 position to be very important for interaction. In accordance with these results, peptide SGGF was unable to inhibit the interaction between GH and PSMD9.It is clear that, in the absence of high resolution crystal structure of the complexes coupled with kinetic, thermodynamic studies using mutant peptides, it would be impossible to precisely define the molecular basis of affinity differences and positional occupancy of residues. In the absence of these details, we will have to consider different possibilities that may account for the binding preferences and affinity.Reports from 20 complex structures of PDZ domains with C-terminal peptides of proteins indicate that the aminoacid at the P0 position has no specific conformational preference in the Ramachandran plot. In contrast the P−1, P−2 and P−3 residues show a strong preference and occupy either a strand or an extended conformation [43]. Such a conformational preference especially of the P−2 residue may explain the high affinity interaction seen with SCGX peptides. Alternatively, SCGF and SCGL peptides may mimic the internal sequences in proteins that bind PDZ domains and the Cys at P−2 may occupy the hydrophobic pocket formed by L153 and Phe 162 residues, mutations of which affect interaction. Flexibility in the binding modes is not uncommon to the peptide-PDZ domain interactions. For e.g., the P(−2) residue in some of the PDZ ligands are known to interact with αB-1 and αB-5 residues on the PDZ domain [43,44]. These residues normally interact with the P0 residues in the ligand. In the crystal structure of Dvl2 PDZ domain bound to a noncanonical C-terminal sequence, P−3 residue was seen to occupy the binding position utilized by a P−2 residue [45]. Secondary structure of the PSMD9 mutant proteins F162G (42%) and L153G (45%; very close to WT ∼48%) were not dramatically altered compared to other mutant proteins which bind the peptide (L173G 42%) or those that do not (Q181G 38%; Table 1).Three modes of peptide binding to PDZ domains in proteins GRASP, PDLIM and MAST4 have been identified. In the structures of GRASP-peptide complexes, it is striking that the two chains of the protein bind to the same peptide in two different binding modes. Comparing these structures, a perpendicular mode, an intermediate mode – both speculated to be kinetic intermediates- and a stable canonical binding mode have been described [44]. We can draw parallels from these studies and propose the following: there exists a conformational ensemble of peptide-PSMD9 complexes. The Group I peptides, probably frequent the non-canonical or perpendicular orientation seen with other PDZ binding peptides. This orientation will rely heavily on the burial of the C-terminal residue for affinity. The Group II peptides GRRC and GRRR peptides probably frequent the intermediate population wherein the P0 residue is anchored. Peptides SCGF and SCGL populate the extended conformation in the canonical mode (although simulations propose a stable binding in the reverse orientation). The entropy cost of binding is probably paid for the SCGF and SCGL peptides as described for other protein derived C-terminal peptides bound to their cognate PDZ domain that may explain the high affinity interaction.In our MD simulation studies, SCGF was unstable in the canonical binding mode but binds stably in a fully extended form, in the reverse orientation (Supplementary Fig. S3). Reverse binding modes of peptides have also been reported in literature, where the same peptide binds in opposite orientation i.e., N′–C′ or C′–N′ termini e.g., peptides binding to chaperone DnaK, Calmodulin and SH3 proteins [46–48]. If SCGF or SCGL peptides bind in a reverse orientation with the hydroxyl-group of the Ser residue substituting for the Phe carboxyl residue, mutation of F162 or L153 residues to Gly, will affect the binding, as seen in Supplementary Fig. S4. However, since the mutant peptide SGGF is unable to bind to PSMD9 or inhibit the binding of GH to PSMD9, the Ser residue seems unimportant for interaction. Therefore, binding of SCGF/L in the reverse orientation as a probable determinant of high affinity interaction seems less likely.

On the origin of affinity differences

We had included 8 out of 10 C-terminal variants of GRRF, and SCGL a single variant of SCGF, from the human proteome and peptide AGHM from transcription factor E12, for their ability to interact with PSMD9. The Kd for each of these peptides is summarized in Table S3. Based on the affinity of the peptides, these variants can be classified into three groups – Group I or low affinity binders, Group II or high affinity binders and Group III tight binders or the top ranking peptides. In the Group I peptides, hydrophobic residues such as F, L, I at P0 provide specificity. In Group II peptides, Cys or Arg at P0 increases affinity by 10-fold as compared to that of Group I peptides. These two amino acids seem very different from each other and from the Group I peptides in terms of their physical properties and binding preference of the PSMD9 PDZ domain seems very intricate. The binding pocket of PSMD9 seems better adapted to bind to residues that are not bulky or highly hydrophobic explaining the high affinity binding of GRRR and GRRC. On the other hand, both Cys and Arg show characteristics of hydrophobic residues. For example, based on the hydrophobicity index Cys is classified along with Phe [36–39] and arginine, although one of the least hydrophobic amino acids, shows very interesting properties. Arginine solubilizes aggregation prone proteins helps in the elution of proteins bound to phenylsepharose column and has wide application in the purification and solubilization of inclusion bodies [40,41]. Arginine like GuHCL interacts with almost all amino acids and preferentially with aromatic residues [40], but unlike GuHCl, Arginine is not a denaturant [42]. This probably explains why these two amino acids like the hydrophobic residues occupy the P0 position. Arg substitution for a Phe in the interior of a protein will result in destabilization but less likely to do so at the protein interface.

Based on binding affinity, peptide AGHM will also fall under the Group I peptides, and methionine is known to be a hydrophobic residue. GRRE with a negatively charged C-terminus and GRRG with a small but relatively hydrophobic residue at P0 do not bind to PSMD9. These results indicate that P0 residue and not GRR is a major determinant of binding specificity in these peptides.

Compared to all peptides tested, the top ranking Group III peptides, SCG variants SCGF and SCGL bind with the highest affinity – Kd for the two peptides is four to five times less than the Group II peptides. The hydrophobicity of the C-terminal residue in SCGF and SCGL is clearly not important for binding and recognition as it can be readily replaced by a Gly. It seems that, in this set of peptides, the P−1, P−2 or P−3 residues are more important for high affinity interaction. By systematically deleting residues from the C-terminus, we identified Cys at P−2 position to be very important for interaction. In accordance with these results, peptide SGGF was unable to inhibit the interaction between GH and PSMD9.

It is clear that, in the absence of high resolution crystal structure of the complexes coupled with kinetic, thermodynamic studies using mutant peptides, it would be impossible to precisely define the molecular basis of affinity differences and positional occupancy of residues. In the absence of these details, we will have to consider different possibilities that may account for the binding preferences and affinity.

Reports from 20 complex structures of PDZ domains with C-terminal peptides of proteins indicate that the aminoacid at the P0 position has no specific conformational preference in the Ramachandran plot. In contrast the P−1, P−2 and P−3 residues show a strong preference and occupy either a strand or an extended conformation [43]. Such a conformational preference especially of the P−2 residue may explain the high affinity interaction seen with SCGX peptides. Alternatively, SCGF and SCGL peptides may mimic the internal sequences in proteins that bind PDZ domains and the Cys at P−2 may occupy the hydrophobic pocket formed by L153 and Phe 162 residues, mutations of which affect interaction. Flexibility in the binding modes is not uncommon to the peptide-PDZ domain interactions. For e.g., the P(−2) residue in some of the PDZ ligands are known to interact with αB-1 and αB-5 residues on the PDZ domain [43,44]. These residues normally interact with the P0 residues in the ligand. In the crystal structure of Dvl2 PDZ domain bound to a noncanonical C-terminal sequence, P−3 residue was seen to occupy the binding position utilized by a P−2 residue [45]. Secondary structure of the PSMD9 mutant proteins F162G (42%) and L153G (45%; very close to WT ∼48%) were not dramatically altered compared to other mutant proteins which bind the peptide (L173G 42%) or those that do not (Q181G 38%; Table 1).

Three modes of peptide binding to PDZ domains in proteins GRASP, PDLIM and MAST4 have been identified. In the structures of GRASP-peptide complexes, it is striking that the two chains of the protein bind to the same peptide in two different binding modes. Comparing these structures, a perpendicular mode, an intermediate mode – both speculated to be kinetic intermediates- and a stable canonical binding mode have been described [44]. We can draw parallels from these studies and propose the following: there exists a conformational ensemble of peptide-PSMD9 complexes. The Group I peptides, probably frequent the non-canonical or perpendicular orientation seen with other PDZ binding peptides. This orientation will rely heavily on the burial of the C-terminal residue for affinity. The Group II peptides GRRC and GRRR peptides probably frequent the intermediate population wherein the P0 residue is anchored. Peptides SCGF and SCGL populate the extended conformation in the canonical mode (although simulations propose a stable binding in the reverse orientation). The entropy cost of binding is probably paid for the SCGF and SCGL peptides as described for other protein derived C-terminal peptides bound to their cognate PDZ domain that may explain the high affinity interaction.

In our MD simulation studies, SCGF was unstable in the canonical binding mode but binds stably in a fully extended form, in the reverse orientation (Supplementary Fig. S3). Reverse binding modes of peptides have also been reported in literature, where the same peptide binds in opposite orientation i.e., N′–C′ or C′–N′ termini e.g., peptides binding to chaperone DnaK, Calmodulin and SH3 proteins [46–48]. If SCGF or SCGL peptides bind in a reverse orientation with the hydroxyl-group of the Ser residue substituting for the Phe carboxyl residue, mutation of F162 or L153 residues to Gly, will affect the binding, as seen in Supplementary Fig. S4. However, since the mutant peptide SGGF is unable to bind to PSMD9 or inhibit the binding of GH to PSMD9, the Ser residue seems unimportant for interaction. Therefore, binding of SCGF/L in the reverse orientation as a probable determinant of high affinity interaction seems less likely.

4.2On the number of binding sites and the mechanism of bindingMutation of the C-terminal Phe in GRRF, or Leu in GRRL or Met in AGHM, to Gly in the respective peptides or proteins inhibits interaction. Commensurate with these results, while GRRF competitively inhibits approximately 69% of the binding between hnRNPA1 and PSMD9 the mutant peptide GRRG, is unable to do so. However, substitution and even deletion of few C-terminal residues does not completely prevent binding of proteins. In all the cases, the fractional occupancy (like Vmax in enzyme catalysis) is maximally affected by the C-terminal mutations while the affinity per se as determined by Kd (like Km for substrate binding) is less affected. In addition, the affinity differences between peptide or protein binding to PSMD9 is large. This difference is especially striking with the Group I peptides, the Kd of which are in the high μM range (∼600 μM) while the proteins bind with low micromolar affinity (1 μM). This vast discrepancy may be explained by the following: (a) C-terminal residues act primarily as signatures or bar codes that are read by the PDZ domain of PSMD9; (b) maximal affinity is contributed by a binding motif elsewhere in the protein or the binding of extended residues at the N-termini of the protein. Peptide affinity, however increases only marginally upon extension of the N-terminal residues. For example, the nine residue C-terminal sequence of E12, with the tetrapeptide motif AGHM, interacts approximately two times more strongly than the short tetrapeptide AGHM (data not shown); and (c) the C-terminal sequences are stabilized by the structure of the full length protein. Based on these possibilities we propose a two state binding model for the interaction between PSMD9 and its client proteins – an initial weak recognition phase mediated by the C-terminal residues which act as specificity determinants followed by its consolidation via interaction of a secondary binding motif. Initial recognition of C-terminal residues in proteins by the PDZ domain may allow binding of the secondary site to an allosteric pocket on PSMD9 (Supplementary Fig. S5). These may or may not be accompanied by conformational changes in the proteins. The binding disparity between Group II peptides or the SCGF and SCGL peptides and their corresponding proteins although is much less (of the order of 8-fold), the fractional occupancy of the C-terminal deletion mutant, ΔCGF of GH is ∼45% of WT indicating that the same mechanism is probably operational. The plasticity of the PDZ domain and contribution of the overall structure to the binding is well illustrated in [31,49]. The partial loss in affinity upon mutations of residues present in different secondary structural elements of the PSMD9-PDZ domain is probably a reflection of the same paradigm.It is also possible that the binding affinities are a reflection of the associated functions of the proteins. For example, we have demonstrated that hnRNPA1 acts as a novel shuttle receptor [7] that recruits IκBα for degradation by the proteasome. PSMD9 by interacting with hnRNPA1 and the 26S proteasome helps in anchoring IκBα and accelerating degradation. In this process, hnRNPA1 is likely to be recycled. By analogy, S14 may also act as a shuttle receptor that recruits proteins like MDM2 for degradation (Fig. 5B, see Section 4.3 for details) and must itself be released intact. Therefore, the C-terminus of these proteins may bind weakly to PSMD9. GH and IL6 receptors are the proposed direct substrates of the proteasome (Fig. 5D see Section 4.3 for details). In addition to the polyubiquitin binding, initial recognition of the C-terminal residues by PSMD9 with high affinity may be very important for the stable binding of such direct substrates to provide fatal directionality for degradation (Fig. 5D).

On the number of binding sites and the mechanism of binding

Mutation of the C-terminal Phe in GRRF, or Leu in GRRL or Met in AGHM, to Gly in the respective peptides or proteins inhibits interaction. Commensurate with these results, while GRRF competitively inhibits approximately 69% of the binding between hnRNPA1 and PSMD9 the mutant peptide GRRG, is unable to do so. However, substitution and even deletion of few C-terminal residues does not completely prevent binding of proteins. In all the cases, the fractional occupancy (like Vmax in enzyme catalysis) is maximally affected by the C-terminal mutations while the affinity per se as determined by Kd (like Km for substrate binding) is less affected. In addition, the affinity differences between peptide or protein binding to PSMD9 is large. This difference is especially striking with the Group I peptides, the Kd of which are in the high μM range (∼600 μM) while the proteins bind with low micromolar affinity (1 μM). This vast discrepancy may be explained by the following: (a) C-terminal residues act primarily as signatures or bar codes that are read by the PDZ domain of PSMD9; (b) maximal affinity is contributed by a binding motif elsewhere in the protein or the binding of extended residues at the N-termini of the protein. Peptide affinity, however increases only marginally upon extension of the N-terminal residues. For example, the nine residue C-terminal sequence of E12, with the tetrapeptide motif AGHM, interacts approximately two times more strongly than the short tetrapeptide AGHM (data not shown); and (c) the C-terminal sequences are stabilized by the structure of the full length protein. Based on these possibilities we propose a two state binding model for the interaction between PSMD9 and its client proteins – an initial weak recognition phase mediated by the C-terminal residues which act as specificity determinants followed by its consolidation via interaction of a secondary binding motif. Initial recognition of C-terminal residues in proteins by the PDZ domain may allow binding of the secondary site to an allosteric pocket on PSMD9 (Supplementary Fig. S5). These may or may not be accompanied by conformational changes in the proteins. The binding disparity between Group II peptides or the SCGF and SCGL peptides and their corresponding proteins although is much less (of the order of 8-fold), the fractional occupancy of the C-terminal deletion mutant, ΔCGF of GH is ∼45% of WT indicating that the same mechanism is probably operational. The plasticity of the PDZ domain and contribution of the overall structure to the binding is well illustrated in [31,49]. The partial loss in affinity upon mutations of residues present in different secondary structural elements of the PSMD9-PDZ domain is probably a reflection of the same paradigm.

It is also possible that the binding affinities are a reflection of the associated functions of the proteins. For example, we have demonstrated that hnRNPA1 acts as a novel shuttle receptor [7] that recruits IκBα for degradation by the proteasome. PSMD9 by interacting with hnRNPA1 and the 26S proteasome helps in anchoring IκBα and accelerating degradation. In this process, hnRNPA1 is likely to be recycled. By analogy, S14 may also act as a shuttle receptor that recruits proteins like MDM2 for degradation (Fig. 5B, see Section 4.3 for details) and must itself be released intact. Therefore, the C-terminus of these proteins may bind weakly to PSMD9. GH and IL6 receptors are the proposed direct substrates of the proteasome (Fig. 5D see Section 4.3 for details). In addition to the polyubiquitin binding, initial recognition of the C-terminal residues by PSMD9 with high affinity may be very important for the stable binding of such direct substrates to provide fatal directionality for degradation (Fig. 5D).

4.3On the functional annotation of PSMD9 and its role in quality control by the proteasomeIn this third part, we speculate on the probable regulatory role of PSMD9 by inferring on the role of its interacting partners. It is interesting that these proteins perform very different functions in the cell. At first glance, these interactions seem unusual, and the real physiological relevance may not be apparent. But detailed literature study provides substantial support for the plausible physiological role of these interactions in mammalian cells. hnRNPA1 is known to interact with IκBα in murine cells, and this interaction somehow accelerates degradation of IκBα resulting in NF-κB activation [50]. A possible functional conservation can be expected in human cells, and one may anticipate PSMD9 to regulate NF-κB signaling via IκBα degradation. In the manuscript that we published recently, we show that hnRNPA1 is a shuttle receptor that recruits IκBα for degradation and PSMD9 acts as a subunit acceptor and anchors hnRNPA1 to facilitate degradation of IκBα by the proteasome [7]. Association of proteasome with ribosome has been documented in the literature [51–53]. Whether S14 and PSMD9 interaction provides the structural scaffold for this interaction and what may be the functional consequence of this interaction in protein translation will be an interesting future investigation. In addition, S14 is known to bind to MDM2, which prevents the ability of this E3 ligase to ubiquitinate p53 thereby preventing proteasomal degradation of p53, leading to stabilization and activation of p53 [54]. Depending on whether or not the interaction between PSMD9 and S14, S14 and MDM2 are mutually exclusive, PSMD9 may influence ubiquitination, stability and functions of p53. By drawing a parallel from our studies on hnRNPA1 and PSMD9 interaction in IκBα degradation, we provide an alternate possibility for the fate of MDM2 and p53. We speculate that S14, similar to hnRNPA1 may also act as a shuttle receptor which recruits MDM2 for degradation by the proteasome (may be under similar stress conditions). Proteasome associated PSMD9 may anchor S14 (like it does hnRNPA1) to facilitate degradation of MDM2 (like IκBα) by the proteasome.GRRR as mentioned before belongs to UPF2, yet another protein involved in RNA metabolism. UPF2 is part of a post-splicing multiprotein complex which regulate mRNA nuclear export and responsible for the detection of exported mRNAs with truncated open reading frames, resulting in nonsense-mediated mRNA decay [55].Two other PSMD9 interacting partners GH (CHS1) and IL6 receptor are implicated in chemokine signaling [56,57]. However both GH and the IL6 receptor with SCGL at the C-termini created by alternate splicing are soluble, secreted proteins. In order to be exported out of the cell, these proteins must follow the ER-Golgi traffic [58]. Secreted proteins are inserted co-translationally into the ER lumen. Upon achieving their final folded state and post-translational modifications, these proteins are transported out of ER, through Golgi and finally out of the plasma membrane. However under stress or stimuli induced signaling, when the load on ER is more, quality control mechanisms must ensure that misfolded proteins are degraded. Misfolded and immature proteins are retro translocated by ER resident translocon and aided by ubiquitin or specialized proteins like Kar2p/BiP they are actively pulled out. These proteins are bound by the ER associated proteasomes and degraded [59–65]. Here, we speculate that while other ubiquitin binding proteins on the 19S regulatory complex of the proteasome may bind to the polyubiquitinated GH or IL6 as classically demonstrated for other proteins, PSMD9 would trap the misfolded GH or IL6R (and other such ERAD proteins) via the specific interaction of the PDZ domain with the respective C-terminal motif. Such interaction would prevent their premature release back into ER, ensure directionality and rapid clearance by the proteasome. Degradation is likely to occur at a basal rate as a routine quality control measure. The rate would be accelerated upon signal induction to truncate and attenuate the signaling process upon withdrawal of the stimuli or in response to ER stress. Therefore, PSMD9 may act as a general molecular chaperone that exerts quality control.Soluble IL6R formed by splicing or proteolysis also bind to gp130 present on cells that lack the IL6 receptors. Soluble IL6R-IL6 complex is involved in what is called as the ‘trans-signaling’ an important mediator of inflammation and chemokine signaling in cancer [66]. Signaling may be attenuated by the receptor mediated endocytosis [67]. By binding to its receptor, GH may also follow receptor mediated endocytosis. The fate of such proteins engulfed by the process of endocytosis and how these may encounter the proteasome is described elegantly [60,68–71]. Some of these proteins routed by endocytosis may become shared substrates of the ‘cytosolic’ proteasomes and lysosomal enzymes. Different parts of the same sequence may be degraded by these degradation machineries. Our experiments designed to verify the binding of these proteins to PSMD9 in HEK293 cells upon co-expression is justified as these interactions are likely to be post endocytosis or post extraction events. Future lines of exciting studies include the characterization of the subcellular loci of these proteins and their ultimate and probably differential fate at the ‘hands’ of proteasome and lysosomes. GRRC belongs to endothelial receptor protein, yet another type I transmembrane protein, involved in signaling by EPCR [72].E12-PSMD9 interaction is likely to influence transcriptional regulation (like Bridge-1 in insulin signaling). PSMD9 may act either as a coactivator or as a repressor of many transcription events. Whether this regulatory role would involve 19S, or the entire 26S proteasome again or a proteasome independent role at the chromatin remains to be seen.With all these examples, the grand or unifying role of PSMD9 seems to be to ensure quality control and regulate the magnitude of signaling or transcriptional programs (working model Fig. 5). The probable mechanism is likely to involve the proteasome and its proteolytic components. However, other regulatory steps involving an independent pool of PSMD9 and its interacting partners within the protein–protein interaction network cannot be ruled out.

On the functional annotation of PSMD9 and its role in quality control by the proteasome

In this third part, we speculate on the probable regulatory role of PSMD9 by inferring on the role of its interacting partners. It is interesting that these proteins perform very different functions in the cell. At first glance, these interactions seem unusual, and the real physiological relevance may not be apparent. But detailed literature study provides substantial support for the plausible physiological role of these interactions in mammalian cells. hnRNPA1 is known to interact with IκBα in murine cells, and this interaction somehow accelerates degradation of IκBα resulting in NF-κB activation [50]. A possible functional conservation can be expected in human cells, and one may anticipate PSMD9 to regulate NF-κB signaling via IκBα degradation. In the manuscript that we published recently, we show that hnRNPA1 is a shuttle receptor that recruits IκBα for degradation and PSMD9 acts as a subunit acceptor and anchors hnRNPA1 to facilitate degradation of IκBα by the proteasome [7]. Association of proteasome with ribosome has been documented in the literature [51–53]. Whether S14 and PSMD9 interaction provides the structural scaffold for this interaction and what may be the functional consequence of this interaction in protein translation will be an interesting future investigation. In addition, S14 is known to bind to MDM2, which prevents the ability of this E3 ligase to ubiquitinate p53 thereby preventing proteasomal degradation of p53, leading to stabilization and activation of p53 [54]. Depending on whether or not the interaction between PSMD9 and S14, S14 and MDM2 are mutually exclusive, PSMD9 may influence ubiquitination, stability and functions of p53. By drawing a parallel from our studies on hnRNPA1 and PSMD9 interaction in IκBα degradation, we provide an alternate possibility for the fate of MDM2 and p53. We speculate that S14, similar to hnRNPA1 may also act as a shuttle receptor which recruits MDM2 for degradation by the proteasome (may be under similar stress conditions). Proteasome associated PSMD9 may anchor S14 (like it does hnRNPA1) to facilitate degradation of MDM2 (like IκBα) by the proteasome.

GRRR as mentioned before belongs to UPF2, yet another protein involved in RNA metabolism. UPF2 is part of a post-splicing multiprotein complex which regulate mRNA nuclear export and responsible for the detection of exported mRNAs with truncated open reading frames, resulting in nonsense-mediated mRNA decay [55].

Two other PSMD9 interacting partners GH (CHS1) and IL6 receptor are implicated in chemokine signaling [56,57]. However both GH and the IL6 receptor with SCGL at the C-termini created by alternate splicing are soluble, secreted proteins. In order to be exported out of the cell, these proteins must follow the ER-Golgi traffic [58]. Secreted proteins are inserted co-translationally into the ER lumen. Upon achieving their final folded state and post-translational modifications, these proteins are transported out of ER, through Golgi and finally out of the plasma membrane. However under stress or stimuli induced signaling, when the load on ER is more, quality control mechanisms must ensure that misfolded proteins are degraded. Misfolded and immature proteins are retro translocated by ER resident translocon and aided by ubiquitin or specialized proteins like Kar2p/BiP they are actively pulled out. These proteins are bound by the ER associated proteasomes and degraded [59–65]. Here, we speculate that while other ubiquitin binding proteins on the 19S regulatory complex of the proteasome may bind to the polyubiquitinated GH or IL6 as classically demonstrated for other proteins, PSMD9 would trap the misfolded GH or IL6R (and other such ERAD proteins) via the specific interaction of the PDZ domain with the respective C-terminal motif. Such interaction would prevent their premature release back into ER, ensure directionality and rapid clearance by the proteasome. Degradation is likely to occur at a basal rate as a routine quality control measure. The rate would be accelerated upon signal induction to truncate and attenuate the signaling process upon withdrawal of the stimuli or in response to ER stress. Therefore, PSMD9 may act as a general molecular chaperone that exerts quality control.

Soluble IL6R formed by splicing or proteolysis also bind to gp130 present on cells that lack the IL6 receptors. Soluble IL6R-IL6 complex is involved in what is called as the ‘trans-signaling’ an important mediator of inflammation and chemokine signaling in cancer [66]. Signaling may be attenuated by the receptor mediated endocytosis [67]. By binding to its receptor, GH may also follow receptor mediated endocytosis. The fate of such proteins engulfed by the process of endocytosis and how these may encounter the proteasome is described elegantly [60,68–71]. Some of these proteins routed by endocytosis may become shared substrates of the ‘cytosolic’ proteasomes and lysosomal enzymes. Different parts of the same sequence may be degraded by these degradation machineries. Our experiments designed to verify the binding of these proteins to PSMD9 in HEK293 cells upon co-expression is justified as these interactions are likely to be post endocytosis or post extraction events. Future lines of exciting studies include the characterization of the subcellular loci of these proteins and their ultimate and probably differential fate at the ‘hands’ of proteasome and lysosomes. GRRC belongs to endothelial receptor protein, yet another type I transmembrane protein, involved in signaling by EPCR [72].

E12-PSMD9 interaction is likely to influence transcriptional regulation (like Bridge-1 in insulin signaling). PSMD9 may act either as a coactivator or as a repressor of many transcription events. Whether this regulatory role would involve 19S, or the entire 26S proteasome again or a proteasome independent role at the chromatin remains to be seen.

With all these examples, the grand or unifying role of PSMD9 seems to be to ensure quality control and regulate the magnitude of signaling or transcriptional programs (working model Fig. 5). The probable mechanism is likely to involve the proteasome and its proteolytic components. However, other regulatory steps involving an independent pool of PSMD9 and its interacting partners within the protein–protein interaction network cannot be ruled out.

5Conflict of interestAuthors declare that there is no conflict of interest.

Conflict of interest

Authors declare that there is no conflict of interest.
