---
pmid: '36727601'
title: Evidence for existence of an apoptosis-inducing BH3-only protein, sayonara,
  in Drosophila.
authors:
- Ikegawa Y
- Combet C
- Groussin M
- Navratil V
- Safar-Remali S
- Shiota T
- Aouacheria A
- Yoo SK
journal: EMBO J
year: '2023'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC10107002
doi: 10.15252/embj.2021110454
---

# Evidence for existence of an apoptosis-inducing BH3-only protein, sayonara, in Drosophila.
**Authors:** Ikegawa Y, Combet C, Groussin M, Navratil V, Safar-Remali S, Shiota T, Aouacheria A, Yoo SK
**Journal:** EMBO J (2023)
**DOI:** [10.15252/embj.2021110454](https://doi.org/10.15252/embj.2021110454)
**PMC:** [PMC10107002](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10107002/)

## Abstract

1. EMBO J. 2023 Apr 17;42(8):e110454. doi: 10.15252/embj.2021110454. Epub 2023
Feb  2.

Evidence for existence of an apoptosis-inducing BH3-only protein, sayonara, in 
Drosophila.

Ikegawa Y(1)(2), Combet C(3), Groussin M(4)(5)(6), Navratil V(7)(8), 
Safar-Remali S(3), Shiota T(9)(10), Aouacheria A(11), Yoo SK(1)(12).

Author information:
(1)Laboratory for Homeodynamics, RIKEN BDR, Kobe, Japan.
(2)Graduate School of Biostudies, Kyoto University, Kyoto, Japan.
(3)Centre de Recherche en Cancérologie de Lyon, UMR Inserm U1052, CNRS 5286, 
Université Claude Bernard Lyon 1, Centre Léon Bérard, Lyon, France.
(4)Laboratoire de Biométrie et Biologie Evolutive, Université de Lyon, 
Université Claude Bernard Lyon 1, CNRS, Villeurbanne, France.
(5)Department of Biological Engineering, Massachusetts Institute of Technology, 
Cambridge, MA, USA.
(6)Institute of Clinical Molecular Biology, Kiel University, Kiel, Germany.
(7)PRABI, Rhône-Alpes Bioinformatics Center, Université Lyon 1, Villeurbanne, 
France.
(8)UMS 3601, Institut Français de Bioinformatique, IFB-Core, Évry, France.
(9)Organization for Promotion of Tenure Track, University of Miyazaki, Miyazaki, 
Japan.
(10)Frontier Science Research Center, University of Miyazaki, Miyazaki, Japan.
(11)ISEM, Institut des Sciences de l'Evolution de Montpellier, UMR 5554, CNRS, 
IRD, EPHE, Université de Montpellier, Montpellier, France.
(12)Physiological Genetics Laboratory, RIKEN CPR, Kobe, Japan.

Cells need to sense stresses to initiate the execution of the dormant cell death 
program. Since the discovery of the first BH3-only protein Bad, BH3-only 
proteins have been recognized as indispensable stress sensors that induce 
apoptosis. BH3-only proteins have so far not been identified in Drosophila 
despite their importance in other organisms. Here, we identify the first 
Drosophila BH3-only protein and name it sayonara. Sayonara induces apoptosis in 
a BH3 motif-dependent manner and interacts genetically and biochemically with 
the BCL-2 homologous proteins, Buffy and Debcl. There is a positive feedback 
loop between Sayonara-mediated caspase activation and autophagy. The BH3 motif 
of sayonara phylogenetically appeared at the time of the ancestral gene 
duplication that led to the formation of Buffy and Debcl in the dipteran 
lineage. To our knowledge, this is the first identification of a bona fide 
BH3-only protein in Drosophila, thus providing a unique example of how cell 
death mechanisms can evolve both through time and across taxa.

© 2023 The Authors. Published under the terms of the CC BY 4.0 license.

DOI: 10.15252/embj.2021110454
PMCID: PMC10107002
PMID: 36727601 [Indexed for MEDLINE]

Conflict of interest statement: The authors declare that they have no conflict 
of interest.

## Full Text

IntroductionApoptosis research blossomed from the finding of Ced3, a caspase, in C. elegans (Ellis & Horvitz, 1986; Igaki & Miura, 2004), followed by elucidation of the detailed molecular mechanisms of caspase activation in C. elegans, Drosophila and mammals (Budihardjo et al, 1999; Igaki & Miura, 2004; Riedl & Shi, 2004; Fan et al, 2005). In both mammals and C. elegans, BH3‐only proteins work as important stress sensors that initiate events leading to caspase activation through BCL‐2 proteins (Bouillet & Strasser, 2002; Giam et al, 2008; Lomonosova & Chinnadurai, 2008; Shamas‐Din et al, 2011; Happo et al, 2012; Aouacheria et al, 2015; Doerflinger et al, 2015). Contrary to the situation of worms and mammals, it has been regarded over the last two decades that BH3‐only proteins that induce caspase activation do not exist in Drosophila in spite of their importance in other organisms (Kuranaga & Miura, 2002; Quinn et al, 2003; Kumar & Cakouros, 2004; Denton et al, 2013; Banjara et al, 2020). For this reason, the current literature suggests that Drosophila, possibly all insects, might have evolved to possess a unique cell death program, which is primarily regulated by the inhibitor of apoptosis (IAP) family of proteins through their antagonists such as Reaper, Hid, and Grim (Goyal et al, 2000).Since the initial finding of BH3‐only proteins functioning as stress sensors that initiate caspase activation in mammals and C. elegans (Yang et al, 1995a; Conradt & Horvitz, 1998), the repertoire of BH3‐only proteins expanded, now including structurally and functionally dissimilar proteins that have only distantly related BH3 sequences, which is widely denoted by the minimal hexameric sequence L‐X(3)‐G(or A, S)‐D (Bouillet & Strasser, 2002; Giam et al, 2008; Lomonosova & Chinnadurai, 2008; Shamas‐Din et al, 2011; Happo et al, 2012; Aouacheria et al, 2013, 2015; Doerflinger et al, 2015), and that belong to protein families with roles sometimes unrelated to cell death (Aouacheria et al, 2013, 2015). Interestingly, proteins such as Atg6 (Beclin) or BNIP3 were reported to contain a BH3 motif in humans and rodents (Aouacheria et al, 2005; Chinnadurai et al, 2008; Sinha & Levine, 2008), but the sequences of their respective homologs in Drosophila appear to be devoid of any recognizable BH3‐like motif (Fig EV1A and B). Their functions are not known to be related to either caspase activation or apoptosis in flies. In this report, we focus on the classic type of BH3‐only proteins that have a typical BH3 sequence signature and that are able to activate caspases through BCL‐2 family proteins.Figure EV1Alignment of noncanonical BH3‐only proteins and a structure of CG14044
Alignment of Beclin in humans, chicken, xenopus, zebrafish, and fly demonstrates that the BH3 motif (the red rectangle is the core sequence of the BH3 motif; the pink rectangle is the sequence surrounding the core BH3 motif) is not conserved in the fly. An * (asterisk) indicates positions which have a single, perfectly conserved residue. (A:) (colon) indicates conservation between groups of strongly similar properties. (A.) (period) indicates conservation between groups of weakly similar properties.Alignment of Bnip3 in humans, chicken, xenopus, zebrafish, and fly demonstrates that the BH3 motif (the red rectangle is the core sequence of the BH3 motif; the pink rectangle is the sequence surrounding the core BH3 motif) is not conserved in the fly.The BH3 motif (LAYNLGVIGDARK) of CG14044 exists in the interhelix region of the protein structure, which was predicted by alphafold 2.A representative image of overlayed adult wings: black, + (control); magenta, Synr WT; blue, Synr ΔBH3.Amino acid sequence of the coiled‐coil domain substitution mutant. All the hydrophobic and charged amino acids were changed to glycine.HA‐tagged Synr was expressed in the wing pouch. The BH3 deletion or BH3 amino acid change does not reduce the Synr expression levels compared with Synr WT. Two independent pictures of each condition are shown.
Data information: Scale bars, 500 μm in (D) and 50 μm in (F).
Source data are available online for this figure.

Introduction

Apoptosis research blossomed from the finding of Ced3, a caspase, in C. elegans (Ellis & Horvitz, 1986; Igaki & Miura, 2004), followed by elucidation of the detailed molecular mechanisms of caspase activation in C. elegans, Drosophila and mammals (Budihardjo et al, 1999; Igaki & Miura, 2004; Riedl & Shi, 2004; Fan et al, 2005). In both mammals and C. elegans, BH3‐only proteins work as important stress sensors that initiate events leading to caspase activation through BCL‐2 proteins (Bouillet & Strasser, 2002; Giam et al, 2008; Lomonosova & Chinnadurai, 2008; Shamas‐Din et al, 2011; Happo et al, 2012; Aouacheria et al, 2015; Doerflinger et al, 2015). Contrary to the situation of worms and mammals, it has been regarded over the last two decades that BH3‐only proteins that induce caspase activation do not exist in Drosophila in spite of their importance in other organisms (Kuranaga & Miura, 2002; Quinn et al, 2003; Kumar & Cakouros, 2004; Denton et al, 2013; Banjara et al, 2020). For this reason, the current literature suggests that Drosophila, possibly all insects, might have evolved to possess a unique cell death program, which is primarily regulated by the inhibitor of apoptosis (IAP) family of proteins through their antagonists such as Reaper, Hid, and Grim (Goyal et al, 2000).

Since the initial finding of BH3‐only proteins functioning as stress sensors that initiate caspase activation in mammals and C. elegans (Yang et al, 1995a; Conradt & Horvitz, 1998), the repertoire of BH3‐only proteins expanded, now including structurally and functionally dissimilar proteins that have only distantly related BH3 sequences, which is widely denoted by the minimal hexameric sequence L‐X(3)‐G(or A, S)‐D (Bouillet & Strasser, 2002; Giam et al, 2008; Lomonosova & Chinnadurai, 2008; Shamas‐Din et al, 2011; Happo et al, 2012; Aouacheria et al, 2013, 2015; Doerflinger et al, 2015), and that belong to protein families with roles sometimes unrelated to cell death (Aouacheria et al, 2013, 2015). Interestingly, proteins such as Atg6 (Beclin) or BNIP3 were reported to contain a BH3 motif in humans and rodents (Aouacheria et al, 2005; Chinnadurai et al, 2008; Sinha & Levine, 2008), but the sequences of their respective homologs in Drosophila appear to be devoid of any recognizable BH3‐like motif (Fig EV1A and B). Their functions are not known to be related to either caspase activation or apoptosis in flies. In this report, we focus on the classic type of BH3‐only proteins that have a typical BH3 sequence signature and that are able to activate caspases through BCL‐2 family proteins.

Alignment of noncanonical BH3‐only proteins and a structure of CG14044

Alignment of Beclin in humans, chicken, xenopus, zebrafish, and fly demonstrates that the BH3 motif (the red rectangle is the core sequence of the BH3 motif; the pink rectangle is the sequence surrounding the core BH3 motif) is not conserved in the fly. An * (asterisk) indicates positions which have a single, perfectly conserved residue. (A:) (colon) indicates conservation between groups of strongly similar properties. (A.) (period) indicates conservation between groups of weakly similar properties.Alignment of Bnip3 in humans, chicken, xenopus, zebrafish, and fly demonstrates that the BH3 motif (the red rectangle is the core sequence of the BH3 motif; the pink rectangle is the sequence surrounding the core BH3 motif) is not conserved in the fly.The BH3 motif (LAYNLGVIGDARK) of CG14044 exists in the interhelix region of the protein structure, which was predicted by alphafold 2.A representative image of overlayed adult wings: black, + (control); magenta, Synr WT; blue, Synr ΔBH3.Amino acid sequence of the coiled‐coil domain substitution mutant. All the hydrophobic and charged amino acids were changed to glycine.HA‐tagged Synr was expressed in the wing pouch. The BH3 deletion or BH3 amino acid change does not reduce the Synr expression levels compared with Synr WT. Two independent pictures of each condition are shown.

Alignment of Beclin in humans, chicken, xenopus, zebrafish, and fly demonstrates that the BH3 motif (the red rectangle is the core sequence of the BH3 motif; the pink rectangle is the sequence surrounding the core BH3 motif) is not conserved in the fly. An * (asterisk) indicates positions which have a single, perfectly conserved residue. (A:) (colon) indicates conservation between groups of strongly similar properties. (A.) (period) indicates conservation between groups of weakly similar properties.

Alignment of Bnip3 in humans, chicken, xenopus, zebrafish, and fly demonstrates that the BH3 motif (the red rectangle is the core sequence of the BH3 motif; the pink rectangle is the sequence surrounding the core BH3 motif) is not conserved in the fly.

The BH3 motif (LAYNLGVIGDARK) of CG14044 exists in the interhelix region of the protein structure, which was predicted by alphafold 2.

A representative image of overlayed adult wings: black, + (control); magenta, Synr WT; blue, Synr ΔBH3.

Amino acid sequence of the coiled‐coil domain substitution mutant. All the hydrophobic and charged amino acids were changed to glycine.

HA‐tagged Synr was expressed in the wing pouch. The BH3 deletion or BH3 amino acid change does not reduce the Synr expression levels compared with Synr WT. Two independent pictures of each condition are shown.

Data information: Scale bars, 500 μm in (D) and 50 μm in (F).

Source data are available online for this figure.

ResultsDiscovery of a BH3‐only protein, sayonara
We explored whether there is truly no fly BH3‐only protein that activates caspase, as has been believed in the field over the last few decades. By performing in silico database analysis using BLAST searches at UniProt, we found a gene, CG14044 (Q9VMY2), whose product could be weakly aligned with the region of human BIK that includes the BH3 motif (Fig 1A). In fact, the region of this protein satisfies the definition of the BH3 motif. In addition to the BH3 motif, CG14044 has two coiled‐coil domains but no other BH motif (Fig 1B). The BH3 motif exists in the interhelix region of the protein structure (Fig EV1C), which was predicted by alphafold 2 (Jumper et al, 2021).Figure 1Discovery of a BH3‐only protein, sayonara

Alignment of human BIK and fly CG14044. A part of CG14044 can be aligned with a part of BIK that contains the BH3 motif. Φ, Σ, Z, and Γ indicate hydrophobic, small, acidic, and hydrophilic amino acid residues, respectively. The minimal sequence is L‐X(3)‐G(or A, S)‐D.Schematic representation of CG14044.Neighbor‐joining phylogenetic tree of Sayonara orthologs from representative ecdysozoan species. Colored circles indicate bootstrap support. Alignment of the region encompassing the L1‐X2‐X3‐X4‐X5‐D core pattern characteristic of most BH3 sequences is shown (with conserved leucine and aspartic acid residues in white on black background). The BCL2DB (Rech de Laval et al, 2014; Aouacheria et al, 2019) position‐specific scoring matrix corresponding to the BH3 motif was used to scan the listed protein sequences for the presence of a BH3 motif (green box depicting positive hits). From this profile‐based prediction, a BH3 motif can only be inferred in Sayonara sequences from the Brachyceran infraorder Muscomorpha.Ectopic expression of CG14044 with act‐Gal4 induces lethality during the larval stage.Ectopic expression of CG14044 in the wing pouch (GFP area indicated by the dotted line) induces caspase activation and DNA fragmentation (TUNEL) in the larval disc, melanization in the pupal wing (arrowhead), and a structural defect in the adult wing.Schematic representation of Sayonara WT and the mutant that lacks the BH3 motif.Deletion of the BH3 motif suppresses the effect of Synr on the wing size.Deletion of the BH3 motif suppresses Sayonara‐induced caspase activation, which was detected by the caspase sensor GC3Ai.Quantification of GC3Ai‐positive areas with Synr WT and the BH3 motif deletion mutant.Schematic representation of Sayonara WT and coiled‐coil domain deletion mutants.Deletion of the coiled‐coil domains suppresses the effect of Synr on the wing size.Amino acid sequence of the BH3 motif substitution mutant. The four well‐conserved hydrophobic amino acids were changed to hydrophilic glutamic acid.Amino acid substitution at the BH3 motif or the coiled‐coil domain suppresses the effect of Synr on the wing size.
Data information: Statistical significance was determined using one‐way ANOVA with Dunnett's post hoc test. Scale bars, 100 μm in (E) (cleaved caspase) and (H), 20 μm in (E) (TUNEL) and 500 μm in (E) (pupae and adult wing).
Source data are available online for this figure.
CG14044 belongs to the Panther family PTHR21974, which is a relatively large, evolutionarily conserved family with its defining coiled‐coil regions (http://www.pantherdb.org/panther/family.do?clsAccession=PTHR21974). Sequence and phylogenetic analyses indicate that the BH3 motif of CG14044 evolved within this protein group in the lineage leading to extant Diptera (Fig 1C). Because the BH3‐only member BIK is found only in vertebrates (Rech de Laval et al, 2014), we interpret that the initial discovery of CG14044 using alignment with BIK was likely a fortuitous one and that the detected similarity to the BIK BH3 motif is the result of convergence or of a random event, that is, analogy rather than homology. This is consistent with the fact that most BH3‐containing proteins are phylogenetically unrelated (Aouacheria et al, 2005, 2013) and with the view that BH3 motifs form a novel class of linear motifs (Aouacheria et al, 2015).To investigate the function of CG14044, we first ectopically expressed CG14044 in the whole body of Drosophila using act‐Gal4 and UAS‐CG14044 (FlyORF F003800). This led to complete lethality (Fig 1D) during the larval stage. Next, we expressed CG14044 in the wing pouch to examine whether it can induce apoptosis. Expression of CG14044 induced caspase activation and DNA fragmentation in the wing disc, black melanization in the pupal wing and abnormal wings in adults (Fig 1E). In sum, CG14044 has a BH3 motif and induces caspase activation and cell death. We named this gene sayonara (synr), meaning “good bye” in Japanese.BH3‐only proteins classically induce caspase activation and apoptosis in a BH3 motif‐dependent manner. To examine the importance of the BH3 motif of Synr, we generated new transgenic lines that have UAS‐synr WT or UAS‐synr BH3 motif mutant, which lacks the BH3 motif (Fig 1F), using the same insertion site attp2. Compared with the FlyORF stock, the newly generated UAS‐synr WT induced a milder wing phenotype, reducing the wing size (Fig EV1D). The BH3 mutant did not induce the adult wing size reduction or caspase activation, which was detected with a caspase sensor, GC3Ai (Schott et al, 2017) (Figs 1G–I and EV1D). These data indicate that Synr induces caspase activation and apoptosis in a BH3 motif‐dependent manner. Interestingly, in addition to the BH3 motif, the coiled‐coil regions were also important for inducing the adult wing phenotype, as demonstrated by the phenotypes of coiled‐coil domain (ccd) 1 and/or 2 deletion mutants (Fig 1J and K). We also made additional transgenics that have amino acid changes in either the BH3 motif or the coiled‐coil domains rather than deletion of the regions. For the BH3 motif, we changed the four conserved hydrophobic amino acids to glutamate, as previously performed (Chen et al, 2005) (Fig 1L). For the coiled‐coil domain formation, repetition of hydrophobic, hydrophilic, and charged amino acids is crucial (Mason & Arndt, 2004). Thus, we changed all the hydrophobic amino acids and charged amino acids to glycine (Fig EV1E). These mutations reversed the wing phenotype induced by synr WT (Fig 1M). To take further controls of the experimental setting, we also created transgenics with HA‐tagged Synr. We confirmed that the BH3 mutations do not reduce the protein expression levels of Synr (Fig EV1F). These data indicate that the effect of the BH3 motif mutation is not due to reduced protein expression.Synr interacts with Debcl and Buffy both genetically and biochemicallyBH3‐only proteins work through BCL‐2 proteins (Bouillet & Strasser, 2002; Giam et al, 2008; Lomonosova & Chinnadurai, 2008; Shamas‐Din et al, 2011; Happo et al, 2012; Aouacheria et al, 2015; Doerflinger et al, 2015). Drosophila has two BCL‐2 proteins, Buffy and Debcl (Brachmann et al, 2000; Colussi et al, 2000; Igaki et al, 2000; Zhang et al, 2000; Quinn et al, 2003; Igaki & Miura, 2004; Doumanis et al, 2007). Debcl was originally identified as a proapoptotic BCL‐2 protein (Brachmann et al, 2000; Colussi et al, 2000; Igaki et al, 2000; Zhang et al, 2000) whereas Buffy as an antiapoptotic BCL‐2 protein (Quinn et al, 2003). Yet, later studies suggested that whether they behave as an anti or proapoptotic factor depends on cell types and physiological contexts (Igaki & Miura, 2004; Senoo‐Matsuda et al, 2005; Doumanis et al, 2007; Clavier et al, 2015). We found that inhibition of either Buffy or Debcl suppresses the adult wing phenotype induced by Synr (Fig 2A–C and E). We also found that inhibition of the executioner caspase, Dcp‐1, inhibits the Synr‐induced wing phenotype (Fig 2D and E). Consistent with the adult wing results, inhibition of Debcl or Buffy suppressed Synr‐mediated caspase activation, which was detected by cleavage of Dcp‐1 and GC3Ai in the wing disc (Fig 2F and G). These genetic epistasis data demonstrate that synr and the BCL‐2 family members Debcl and Buffy, genetically interact.Figure 2Synr interacts with Debcl and Buffy both genetically and biochemically
A–DThe Synr‐induced wing structural defect is suppressed by knockdown of Debcl, Buffy or DCP1.EQuantification of the adult wing size with RNAis for apoptotic genes.FSynr‐induced caspase activation is suppressed by knockdown of Debcl.GGC3Ai signals are suppressed by knockdown of Debcl or Buffy (quantification is shown in Fig EV2H).H, ILysates from HEK293T cells that express Flag‐Debcl or Flag‐Buffy were incubated with GST or GST‐Synr bound to glutathione Sepharose. Flag‐Debcl or Flag‐Buffy is pulled down with GST‐Synr but not with GST.JDeletion of neither the coiled‐coil domain nor the BH3 motif suppresses binding Flag‐Debcl to GST‐Synr.KQuantification of the GST pull‐down assay shown in (J).LDeletion of neither the coiled‐coil domain nor the BH3 motif suppresses binding of Flag‐Buffy to GST‐Synr.MQuantification of the GST pull‐down assay shown in (L).NAmino acids that were replaced with BPA are shown. Tyrosine124 is within the BH3 motif and phenylalanine141 is in a close proximity to the BH3 motif. Tyrosine340 is for a negative control.ODebcl can be photo‐crosslinked with Synr at BPA that corresponds to Y124 and F141. BPA at Y340 was used as a negative control. Red arrowheads indicate covalently crosslinked products.PBuffy is also photo‐crosslinked with Synr at BPA that corresponds to Y124 and F141. Red arrowheads indicate covalently crosslinked products.QGST‐Synr pulls down both Flag‐Buffy and HA‐Debcl simultaneously, suggesting a possibility of complex formation among Synr, Buffy and Debcl.RHA‐Debcl is detected after two series of immunoprecipitation. The first immunoprecipitation was performed for Flag‐Buffy, followed by Flag peptide‐mediated elution and secondary precipitation for Myc‐Synr. HA‐Debcl can be detected after this sequential immunoprecipitation.
Data information: Statistical significance was determined using one‐way repeated measures ANOVA with Dunnett's post hoc test. Scale bars, 500 μm in (A) and 50 μm in (F) and (G).
Source data are available online for this figure.
To examine whether they biochemically interact, we investigated biochemical binding of Synr to Debcl or Buffy. We performed GST pull‐down assays using GST‐Synr bound to glutathione Sepharose, and Buffy or Debcl, which was expressed in HEK293T cells. We found that Synr interacts with both BCL‐2 proteins (Fig 2H and I). This strongly supports the idea to posit Synr as the caspase‐activating BH3‐only protein that interacts genetically and biochemically with BCL‐2 proteins.We then investigated which part of Synr is important for binding to Debcl or Buffy. In most BH3‐only proteins, the BH3 motif is crucial for binding to BCL‐2 proteins (Bouillet & Strasser, 2002; Giam et al, 2008; Lomonosova & Chinnadurai, 2008; Shamas‐Din et al, 2011; Happo et al, 2012; Aouacheria et al, 2015; Doerflinger et al, 2015). Interestingly, deletion of either the BH3 motif or the coiled‐coil domain did not affect Synr's binding to the BCL‐2 proteins (Fig 2J–M). This indicates that Synr's physical interaction with Buffy or Debcl can occur without the BH3 motif. This seemed at a glance contradictory from the genetic data that Synr's BH3 motif is indispensable for caspase activation (Fig 1F–M). We hypothesized that the BH3 motif may still interact with the BCL‐2 proteins and play an important role albeit being dispensable for their gross interaction. To test this hypothesis, we assessed the interaction of Synr's BH3 motif and BCL‐2 proteins by performing site‐specific photo‐crosslinking (Chin et al, 2002; Shiota et al, 2011). We incorporated p‐benzoyl‐L‐phenylalanine (BPA) either in the BH3 motif (124Y) or in its proximity (141F) (Fig 2N). This experimentation enables assessment of the physical interaction between the BH3 motif and Buffy/Debcl. We found that UV‐induced crosslinking occurs between the inserted BPA and Buffy/Debcl, demonstrating their physical interaction (Fig 2O and P).Additionally, we found that Synr can pull down Debcl and Buffy simultaneously (Fig 2Q), suggesting a possibility that the three proteins make a complex. To further verify this possibility, we performed a sequential pull‐down experiment by pulling down Flag‐Buffy, followed by elution with the Flag peptide and second pull‐down of Myc‐Synr. This sequential pull‐down experiment led to co‐immunoprecipitation of HA‐Debcl (Fig 2R). Based on these biochemical data and genetic data, we speculate that Synr makes a complex with Buffy and Debcl, where Synr's BH3 motif could exert a crucial effect on the protein complex's function. It is of note that Buffy and Debcl themselves interact with each other (Quinn et al, 2003), indicating that all the three proteins can interact with each other. The idea that Synr makes a complex with Debcl and Buffy is consistent with our genetic data that inhibition of either Debcl or Buffy is sufficient to suppress the synr‐induced phenotype.Feedback loop between apoptosis and autophagyWhile we were characterizing the effects of synr expression, we unexpectedly found that Synr induces autophagy, which was detected by punctate accumulation of Lysotracker (Fig 3A) and mCherry‐Atg8a (Fig 3B). Synr also colocalized with Atg8a and Rab27, which label autophagosomes (Nagy et al, 2015; Underwood et al, 2020), in the salivary gland and the wing disc (Figs 3C and EV2A). The relationship between autophagy and apoptosis is enigmatic (Gump & Thorburn, 2011; Marino et al, 2014; Bialik et al, 2018; Doherty & Baehrecke, 2018). Autophagy could occur as a byproduct of apoptosis or could affect the apoptotic process. While autophagy is in principle a mechanism for cell survival, it can induce caspase‐dependent apoptosis (Scott et al, 2007; Mohseni et al, 2009; Nagata et al, 2019) as well as caspase‐independent cell death, which is called autophagic cell death (Berry & Baehrecke, 2007; Bialik et al, 2018). Additionally, caspase can also induce autophagy (Hou et al, 2008). Which role autophagy plays depends on cellular contexts. Interestingly, inhibition of autophagy suppressed both the Synr‐induced wing phenotype and cell death detected by propidium iodide and TUNEL (Figs 3D–G and EV2B–D), suggesting that Synr‐induced autophagy is not just a byproduct of caspase‐mediated apoptosis or counteractive for apoptosis, rather it has a positive contribution to the cell death phenotype. In addition to Synr‐induced cell death, we also found that reaper (rpr), which activates apoptosis through DIAP1 inhibition (Goyal et al, 2000), induces autophagy (Fig EV2E and F) and that the rpr‐induced phenotype in the adult wing is suppressed by autophagy inhibition (Fig EV2G).Figure 3Feedback loop between apoptosis and autophagy
AEctopic expression of Synr induces autolysosome accumulation, which was detected by Lysotracker, in the wing disc.BEctopic expression of Synr also induces autophagosome accumulation, which was detected by mCherry‐Atg8a. The dotted square regions are magnified in the right pictures.CSynr is colocalized with mCherry‐Atg8a (arrowheads) in the wing disc and the salivary gland at 112 h AEL.D–FKnockdown of Atg2 or Atg8a suppresses the wing structural defect that is induced by Synr.G
Atg2 knockdown suppresses Synr‐induced cell death, which was detected by TUNEL staining.H, ISynr‐induced caspase activation is suppressed by inhibition of autophagy (Atg2, Atg8a RNAis) in a similar manner to apoptosis inhibition (Debcl, Dcp‐1 RNAis).JSynr‐induced autolysosome accumulation is suppressed by Dcp‐1 knockdown.
Data information: Statistical significance was determined using one‐way ANOVA with Dunnett's post hoc test. Scale bars, 100 μm in (A) and (H), 50 μm in (B) and (J), 10 μm in (C) and (G), and 500 μm in (D).
Source data are available online for this figure.
Figure EV2Synr‐induced autophagy
Synr is colocalized with Rab27 (arrowheads) in the wing disc and the salivary gland at 112 h AEL.The Synr‐induced wing structural defect is suppressed by autophagy inhibition.Quantification of the wing size with a combination of Synr expression and autophagy inhibition.
Atg2 knockdown suppresses Synr‐induced cell death, which was detected by propidium iodide (PI).A DIAP1 inhibitor, reaper induces autolysosome accumulation in the wing disc.Reaper also induces autophagosome accumulation in the wing disc. The yellow‐dotted square region is magnified in the right picture.Reaper‐induced wing defects are suppressed by autophagy inhibition. Wing phenotypes were divided into five classes based on the severity as indicated in pictures.Knockdown of Debcl, Buffy, or Atg2 significantly suppresses caspase activation compared with the control RNAi for luciferase.
Data information: Statistical significance was determined using one‐way ANOVA with Dunnett's post hoc test. Scale bars, 10 μm in (A), 500 μm in (B), and 50 μm in (D–F).
Source data are available online for this figure.
We further clarified how autophagy regulates Synr‐induced cell death. Autophagy inhibition suppressed caspase activation by Synr, in a similar manner to inhibition of Dcp‐1 or Debcl (Figs 3H and I, and EV2H). On the contrary, Dcp‐1 inhibition suppressed Synr‐induced Lysotracker puncta formation, suggesting that caspase is necessary for Synr‐induced autophagy (Fig 3J). These data indicate that there is a positive feedback loop between apoptosis and autophagy, activating each other at least in a context of Synr‐induced cell death.Endogenous function of SynrFinally, we tested endogenous functions of synr. In vertebrates, BH3‐only proteins often function downstream of p53 (Happo et al, 2012). We found that the p53‐induced wing phenotype was well suppressed by synr RNAis that we newly generated (Figs 4A and EV3A). synr RNAis also suppressed p53‐induced caspase activation (Figs 4B and EV3B), indicating that endogenous synr regulates p53‐mediated apoptosis.Figure 4Endogenous function of Synr
Inhibition of Synr suppresses the wing defect that is induced by p53, a potent apoptosis inducer.Inhibition of Synr suppresses p53‐induced caspase activation.Inhibition of Synr in the midgut enterocytes suppresses starvation‐induced caspase activation in the midgut.Inhibition of Synr in the midgut enterocytes enhances resistance to starvation.The synr mutant also exhibits enhanced resistance to starvation.A proposed model of Synr‐induced cell death. Synr, which functions together with Buffy and Debcl, induces initial caspase activation, which leads to feedback loop between apoptosis and autophagy.
Data information: Statistical significance was determined using a two‐tailed unpaired t‐test (B), one‐way ANOVA with Holm‐Sidak's multiple comparisons test (C), and a log‐rank (Mantel–Cox) test (D, E). Scale bar, 500 μm in (A).
Source data are available online for this figure.
Figure EV3Knockdown and knockout of synr

A newly generated synr RNAi line can suppress the effect of ectopic synr expression on wings.Two RNAis for synr can suppress p53‐mediated caspase activation, which is detected by GC3Ai.Schematic of MiMIC (Minos‐mediated integration cassette) insertion in synr (CGCG14044MI09290).A design of primers to detect MiMIC insertion.Using the primers, the MiMIC insertion was confirmed by PCR. During extensive outcross of CGCG14044MI09290 to OregonR, MiMIC insertion was confirmed by PCR.The extensively outcrossed synr mutant demonstrates a slight, but reproducible developmental delay during development.
Data information: Statistical significance was determined using one‐way ANOVA with Holm‐Sidak's multiple comparisons test (B) and a log‐rank (Mantel–Cox) test (F). Scale bar, 500 μm in (A).
Source data are available online for this figure.
We also examined more physiological functions of synr. Starvation induces caspase activation in the gut enterocytes (O'Brien et al, 2011). We found that synr inhibition suppressed starvation‐induced caspase activation (Fig 4C). Interestingly, synr mutation or enterocyte‐specific synr knockdown enhanced resistance to starvation (Figs 4D and E, and EV3C–E), supporting a physiological function of endogenous synr.It is of note that the synr mutant was viable, suggesting that synr is not necessary for apoptosis during development, which is similar to mutants of Buffy or Debcl (Sevrioukov et al, 2007). However, we reproducibly found that the extensively outcrossed synr mutant develops slightly faster than control (Fig EV3F), suggesting that synr could play some role during development.

Discovery of a BH3‐only protein, sayonara
We explored whether there is truly no fly BH3‐only protein that activates caspase, as has been believed in the field over the last few decades. By performing in silico database analysis using BLAST searches at UniProt, we found a gene, CG14044 (Q9VMY2), whose product could be weakly aligned with the region of human BIK that includes the BH3 motif (Fig 1A). In fact, the region of this protein satisfies the definition of the BH3 motif. In addition to the BH3 motif, CG14044 has two coiled‐coil domains but no other BH motif (Fig 1B). The BH3 motif exists in the interhelix region of the protein structure (Fig EV1C), which was predicted by alphafold 2 (Jumper et al, 2021).Figure 1Discovery of a BH3‐only protein, sayonara

Alignment of human BIK and fly CG14044. A part of CG14044 can be aligned with a part of BIK that contains the BH3 motif. Φ, Σ, Z, and Γ indicate hydrophobic, small, acidic, and hydrophilic amino acid residues, respectively. The minimal sequence is L‐X(3)‐G(or A, S)‐D.Schematic representation of CG14044.Neighbor‐joining phylogenetic tree of Sayonara orthologs from representative ecdysozoan species. Colored circles indicate bootstrap support. Alignment of the region encompassing the L1‐X2‐X3‐X4‐X5‐D core pattern characteristic of most BH3 sequences is shown (with conserved leucine and aspartic acid residues in white on black background). The BCL2DB (Rech de Laval et al, 2014; Aouacheria et al, 2019) position‐specific scoring matrix corresponding to the BH3 motif was used to scan the listed protein sequences for the presence of a BH3 motif (green box depicting positive hits). From this profile‐based prediction, a BH3 motif can only be inferred in Sayonara sequences from the Brachyceran infraorder Muscomorpha.Ectopic expression of CG14044 with act‐Gal4 induces lethality during the larval stage.Ectopic expression of CG14044 in the wing pouch (GFP area indicated by the dotted line) induces caspase activation and DNA fragmentation (TUNEL) in the larval disc, melanization in the pupal wing (arrowhead), and a structural defect in the adult wing.Schematic representation of Sayonara WT and the mutant that lacks the BH3 motif.Deletion of the BH3 motif suppresses the effect of Synr on the wing size.Deletion of the BH3 motif suppresses Sayonara‐induced caspase activation, which was detected by the caspase sensor GC3Ai.Quantification of GC3Ai‐positive areas with Synr WT and the BH3 motif deletion mutant.Schematic representation of Sayonara WT and coiled‐coil domain deletion mutants.Deletion of the coiled‐coil domains suppresses the effect of Synr on the wing size.Amino acid sequence of the BH3 motif substitution mutant. The four well‐conserved hydrophobic amino acids were changed to hydrophilic glutamic acid.Amino acid substitution at the BH3 motif or the coiled‐coil domain suppresses the effect of Synr on the wing size.
Data information: Statistical significance was determined using one‐way ANOVA with Dunnett's post hoc test. Scale bars, 100 μm in (E) (cleaved caspase) and (H), 20 μm in (E) (TUNEL) and 500 μm in (E) (pupae and adult wing).
Source data are available online for this figure.
CG14044 belongs to the Panther family PTHR21974, which is a relatively large, evolutionarily conserved family with its defining coiled‐coil regions (http://www.pantherdb.org/panther/family.do?clsAccession=PTHR21974). Sequence and phylogenetic analyses indicate that the BH3 motif of CG14044 evolved within this protein group in the lineage leading to extant Diptera (Fig 1C). Because the BH3‐only member BIK is found only in vertebrates (Rech de Laval et al, 2014), we interpret that the initial discovery of CG14044 using alignment with BIK was likely a fortuitous one and that the detected similarity to the BIK BH3 motif is the result of convergence or of a random event, that is, analogy rather than homology. This is consistent with the fact that most BH3‐containing proteins are phylogenetically unrelated (Aouacheria et al, 2005, 2013) and with the view that BH3 motifs form a novel class of linear motifs (Aouacheria et al, 2015).To investigate the function of CG14044, we first ectopically expressed CG14044 in the whole body of Drosophila using act‐Gal4 and UAS‐CG14044 (FlyORF F003800). This led to complete lethality (Fig 1D) during the larval stage. Next, we expressed CG14044 in the wing pouch to examine whether it can induce apoptosis. Expression of CG14044 induced caspase activation and DNA fragmentation in the wing disc, black melanization in the pupal wing and abnormal wings in adults (Fig 1E). In sum, CG14044 has a BH3 motif and induces caspase activation and cell death. We named this gene sayonara (synr), meaning “good bye” in Japanese.BH3‐only proteins classically induce caspase activation and apoptosis in a BH3 motif‐dependent manner. To examine the importance of the BH3 motif of Synr, we generated new transgenic lines that have UAS‐synr WT or UAS‐synr BH3 motif mutant, which lacks the BH3 motif (Fig 1F), using the same insertion site attp2. Compared with the FlyORF stock, the newly generated UAS‐synr WT induced a milder wing phenotype, reducing the wing size (Fig EV1D). The BH3 mutant did not induce the adult wing size reduction or caspase activation, which was detected with a caspase sensor, GC3Ai (Schott et al, 2017) (Figs 1G–I and EV1D). These data indicate that Synr induces caspase activation and apoptosis in a BH3 motif‐dependent manner. Interestingly, in addition to the BH3 motif, the coiled‐coil regions were also important for inducing the adult wing phenotype, as demonstrated by the phenotypes of coiled‐coil domain (ccd) 1 and/or 2 deletion mutants (Fig 1J and K). We also made additional transgenics that have amino acid changes in either the BH3 motif or the coiled‐coil domains rather than deletion of the regions. For the BH3 motif, we changed the four conserved hydrophobic amino acids to glutamate, as previously performed (Chen et al, 2005) (Fig 1L). For the coiled‐coil domain formation, repetition of hydrophobic, hydrophilic, and charged amino acids is crucial (Mason & Arndt, 2004). Thus, we changed all the hydrophobic amino acids and charged amino acids to glycine (Fig EV1E). These mutations reversed the wing phenotype induced by synr WT (Fig 1M). To take further controls of the experimental setting, we also created transgenics with HA‐tagged Synr. We confirmed that the BH3 mutations do not reduce the protein expression levels of Synr (Fig EV1F). These data indicate that the effect of the BH3 motif mutation is not due to reduced protein expression.

Discovery of a BH3‐only protein, sayonara

We explored whether there is truly no fly BH3‐only protein that activates caspase, as has been believed in the field over the last few decades. By performing in silico database analysis using BLAST searches at UniProt, we found a gene, CG14044 (Q9VMY2), whose product could be weakly aligned with the region of human BIK that includes the BH3 motif (Fig 1A). In fact, the region of this protein satisfies the definition of the BH3 motif. In addition to the BH3 motif, CG14044 has two coiled‐coil domains but no other BH motif (Fig 1B). The BH3 motif exists in the interhelix region of the protein structure (Fig EV1C), which was predicted by alphafold 2 (Jumper et al, 2021).

Discovery of a BH3‐only protein, sayonara

Alignment of human BIK and fly CG14044. A part of CG14044 can be aligned with a part of BIK that contains the BH3 motif. Φ, Σ, Z, and Γ indicate hydrophobic, small, acidic, and hydrophilic amino acid residues, respectively. The minimal sequence is L‐X(3)‐G(or A, S)‐D.Schematic representation of CG14044.Neighbor‐joining phylogenetic tree of Sayonara orthologs from representative ecdysozoan species. Colored circles indicate bootstrap support. Alignment of the region encompassing the L1‐X2‐X3‐X4‐X5‐D core pattern characteristic of most BH3 sequences is shown (with conserved leucine and aspartic acid residues in white on black background). The BCL2DB (Rech de Laval et al, 2014; Aouacheria et al, 2019) position‐specific scoring matrix corresponding to the BH3 motif was used to scan the listed protein sequences for the presence of a BH3 motif (green box depicting positive hits). From this profile‐based prediction, a BH3 motif can only be inferred in Sayonara sequences from the Brachyceran infraorder Muscomorpha.Ectopic expression of CG14044 with act‐Gal4 induces lethality during the larval stage.Ectopic expression of CG14044 in the wing pouch (GFP area indicated by the dotted line) induces caspase activation and DNA fragmentation (TUNEL) in the larval disc, melanization in the pupal wing (arrowhead), and a structural defect in the adult wing.Schematic representation of Sayonara WT and the mutant that lacks the BH3 motif.Deletion of the BH3 motif suppresses the effect of Synr on the wing size.Deletion of the BH3 motif suppresses Sayonara‐induced caspase activation, which was detected by the caspase sensor GC3Ai.Quantification of GC3Ai‐positive areas with Synr WT and the BH3 motif deletion mutant.Schematic representation of Sayonara WT and coiled‐coil domain deletion mutants.Deletion of the coiled‐coil domains suppresses the effect of Synr on the wing size.Amino acid sequence of the BH3 motif substitution mutant. The four well‐conserved hydrophobic amino acids were changed to hydrophilic glutamic acid.Amino acid substitution at the BH3 motif or the coiled‐coil domain suppresses the effect of Synr on the wing size.

Alignment of human BIK and fly CG14044. A part of CG14044 can be aligned with a part of BIK that contains the BH3 motif. Φ, Σ, Z, and Γ indicate hydrophobic, small, acidic, and hydrophilic amino acid residues, respectively. The minimal sequence is L‐X(3)‐G(or A, S)‐D.

Schematic representation of CG14044.

Neighbor‐joining phylogenetic tree of Sayonara orthologs from representative ecdysozoan species. Colored circles indicate bootstrap support. Alignment of the region encompassing the L1‐X2‐X3‐X4‐X5‐D core pattern characteristic of most BH3 sequences is shown (with conserved leucine and aspartic acid residues in white on black background). The BCL2DB (Rech de Laval et al, 2014; Aouacheria et al, 2019) position‐specific scoring matrix corresponding to the BH3 motif was used to scan the listed protein sequences for the presence of a BH3 motif (green box depicting positive hits). From this profile‐based prediction, a BH3 motif can only be inferred in Sayonara sequences from the Brachyceran infraorder Muscomorpha.

Ectopic expression of CG14044 with act‐Gal4 induces lethality during the larval stage.

Ectopic expression of CG14044 in the wing pouch (GFP area indicated by the dotted line) induces caspase activation and DNA fragmentation (TUNEL) in the larval disc, melanization in the pupal wing (arrowhead), and a structural defect in the adult wing.

Schematic representation of Sayonara WT and the mutant that lacks the BH3 motif.

Deletion of the BH3 motif suppresses the effect of Synr on the wing size.

Deletion of the BH3 motif suppresses Sayonara‐induced caspase activation, which was detected by the caspase sensor GC3Ai.

Quantification of GC3Ai‐positive areas with Synr WT and the BH3 motif deletion mutant.

Schematic representation of Sayonara WT and coiled‐coil domain deletion mutants.

Deletion of the coiled‐coil domains suppresses the effect of Synr on the wing size.

Amino acid sequence of the BH3 motif substitution mutant. The four well‐conserved hydrophobic amino acids were changed to hydrophilic glutamic acid.

Amino acid substitution at the BH3 motif or the coiled‐coil domain suppresses the effect of Synr on the wing size.

Data information: Statistical significance was determined using one‐way ANOVA with Dunnett's post hoc test. Scale bars, 100 μm in (E) (cleaved caspase) and (H), 20 μm in (E) (TUNEL) and 500 μm in (E) (pupae and adult wing).

Source data are available online for this figure.

CG14044 belongs to the Panther family PTHR21974, which is a relatively large, evolutionarily conserved family with its defining coiled‐coil regions (http://www.pantherdb.org/panther/family.do?clsAccession=PTHR21974). Sequence and phylogenetic analyses indicate that the BH3 motif of CG14044 evolved within this protein group in the lineage leading to extant Diptera (Fig 1C). Because the BH3‐only member BIK is found only in vertebrates (Rech de Laval et al, 2014), we interpret that the initial discovery of CG14044 using alignment with BIK was likely a fortuitous one and that the detected similarity to the BIK BH3 motif is the result of convergence or of a random event, that is, analogy rather than homology. This is consistent with the fact that most BH3‐containing proteins are phylogenetically unrelated (Aouacheria et al, 2005, 2013) and with the view that BH3 motifs form a novel class of linear motifs (Aouacheria et al, 2015).

To investigate the function of CG14044, we first ectopically expressed CG14044 in the whole body of Drosophila using act‐Gal4 and UAS‐CG14044 (FlyORF F003800). This led to complete lethality (Fig 1D) during the larval stage. Next, we expressed CG14044 in the wing pouch to examine whether it can induce apoptosis. Expression of CG14044 induced caspase activation and DNA fragmentation in the wing disc, black melanization in the pupal wing and abnormal wings in adults (Fig 1E). In sum, CG14044 has a BH3 motif and induces caspase activation and cell death. We named this gene sayonara (synr), meaning “good bye” in Japanese.

BH3‐only proteins classically induce caspase activation and apoptosis in a BH3 motif‐dependent manner. To examine the importance of the BH3 motif of Synr, we generated new transgenic lines that have UAS‐synr WT or UAS‐synr BH3 motif mutant, which lacks the BH3 motif (Fig 1F), using the same insertion site attp2. Compared with the FlyORF stock, the newly generated UAS‐synr WT induced a milder wing phenotype, reducing the wing size (Fig EV1D). The BH3 mutant did not induce the adult wing size reduction or caspase activation, which was detected with a caspase sensor, GC3Ai (Schott et al, 2017) (Figs 1G–I and EV1D). These data indicate that Synr induces caspase activation and apoptosis in a BH3 motif‐dependent manner. Interestingly, in addition to the BH3 motif, the coiled‐coil regions were also important for inducing the adult wing phenotype, as demonstrated by the phenotypes of coiled‐coil domain (ccd) 1 and/or 2 deletion mutants (Fig 1J and K). We also made additional transgenics that have amino acid changes in either the BH3 motif or the coiled‐coil domains rather than deletion of the regions. For the BH3 motif, we changed the four conserved hydrophobic amino acids to glutamate, as previously performed (Chen et al, 2005) (Fig 1L). For the coiled‐coil domain formation, repetition of hydrophobic, hydrophilic, and charged amino acids is crucial (Mason & Arndt, 2004). Thus, we changed all the hydrophobic amino acids and charged amino acids to glycine (Fig EV1E). These mutations reversed the wing phenotype induced by synr WT (Fig 1M). To take further controls of the experimental setting, we also created transgenics with HA‐tagged Synr. We confirmed that the BH3 mutations do not reduce the protein expression levels of Synr (Fig EV1F). These data indicate that the effect of the BH3 motif mutation is not due to reduced protein expression.

Synr interacts with Debcl and Buffy both genetically and biochemicallyBH3‐only proteins work through BCL‐2 proteins (Bouillet & Strasser, 2002; Giam et al, 2008; Lomonosova & Chinnadurai, 2008; Shamas‐Din et al, 2011; Happo et al, 2012; Aouacheria et al, 2015; Doerflinger et al, 2015). Drosophila has two BCL‐2 proteins, Buffy and Debcl (Brachmann et al, 2000; Colussi et al, 2000; Igaki et al, 2000; Zhang et al, 2000; Quinn et al, 2003; Igaki & Miura, 2004; Doumanis et al, 2007). Debcl was originally identified as a proapoptotic BCL‐2 protein (Brachmann et al, 2000; Colussi et al, 2000; Igaki et al, 2000; Zhang et al, 2000) whereas Buffy as an antiapoptotic BCL‐2 protein (Quinn et al, 2003). Yet, later studies suggested that whether they behave as an anti or proapoptotic factor depends on cell types and physiological contexts (Igaki & Miura, 2004; Senoo‐Matsuda et al, 2005; Doumanis et al, 2007; Clavier et al, 2015). We found that inhibition of either Buffy or Debcl suppresses the adult wing phenotype induced by Synr (Fig 2A–C and E). We also found that inhibition of the executioner caspase, Dcp‐1, inhibits the Synr‐induced wing phenotype (Fig 2D and E). Consistent with the adult wing results, inhibition of Debcl or Buffy suppressed Synr‐mediated caspase activation, which was detected by cleavage of Dcp‐1 and GC3Ai in the wing disc (Fig 2F and G). These genetic epistasis data demonstrate that synr and the BCL‐2 family members Debcl and Buffy, genetically interact.Figure 2Synr interacts with Debcl and Buffy both genetically and biochemically
A–DThe Synr‐induced wing structural defect is suppressed by knockdown of Debcl, Buffy or DCP1.EQuantification of the adult wing size with RNAis for apoptotic genes.FSynr‐induced caspase activation is suppressed by knockdown of Debcl.GGC3Ai signals are suppressed by knockdown of Debcl or Buffy (quantification is shown in Fig EV2H).H, ILysates from HEK293T cells that express Flag‐Debcl or Flag‐Buffy were incubated with GST or GST‐Synr bound to glutathione Sepharose. Flag‐Debcl or Flag‐Buffy is pulled down with GST‐Synr but not with GST.JDeletion of neither the coiled‐coil domain nor the BH3 motif suppresses binding Flag‐Debcl to GST‐Synr.KQuantification of the GST pull‐down assay shown in (J).LDeletion of neither the coiled‐coil domain nor the BH3 motif suppresses binding of Flag‐Buffy to GST‐Synr.MQuantification of the GST pull‐down assay shown in (L).NAmino acids that were replaced with BPA are shown. Tyrosine124 is within the BH3 motif and phenylalanine141 is in a close proximity to the BH3 motif. Tyrosine340 is for a negative control.ODebcl can be photo‐crosslinked with Synr at BPA that corresponds to Y124 and F141. BPA at Y340 was used as a negative control. Red arrowheads indicate covalently crosslinked products.PBuffy is also photo‐crosslinked with Synr at BPA that corresponds to Y124 and F141. Red arrowheads indicate covalently crosslinked products.QGST‐Synr pulls down both Flag‐Buffy and HA‐Debcl simultaneously, suggesting a possibility of complex formation among Synr, Buffy and Debcl.RHA‐Debcl is detected after two series of immunoprecipitation. The first immunoprecipitation was performed for Flag‐Buffy, followed by Flag peptide‐mediated elution and secondary precipitation for Myc‐Synr. HA‐Debcl can be detected after this sequential immunoprecipitation.
Data information: Statistical significance was determined using one‐way repeated measures ANOVA with Dunnett's post hoc test. Scale bars, 500 μm in (A) and 50 μm in (F) and (G).
Source data are available online for this figure.
To examine whether they biochemically interact, we investigated biochemical binding of Synr to Debcl or Buffy. We performed GST pull‐down assays using GST‐Synr bound to glutathione Sepharose, and Buffy or Debcl, which was expressed in HEK293T cells. We found that Synr interacts with both BCL‐2 proteins (Fig 2H and I). This strongly supports the idea to posit Synr as the caspase‐activating BH3‐only protein that interacts genetically and biochemically with BCL‐2 proteins.We then investigated which part of Synr is important for binding to Debcl or Buffy. In most BH3‐only proteins, the BH3 motif is crucial for binding to BCL‐2 proteins (Bouillet & Strasser, 2002; Giam et al, 2008; Lomonosova & Chinnadurai, 2008; Shamas‐Din et al, 2011; Happo et al, 2012; Aouacheria et al, 2015; Doerflinger et al, 2015). Interestingly, deletion of either the BH3 motif or the coiled‐coil domain did not affect Synr's binding to the BCL‐2 proteins (Fig 2J–M). This indicates that Synr's physical interaction with Buffy or Debcl can occur without the BH3 motif. This seemed at a glance contradictory from the genetic data that Synr's BH3 motif is indispensable for caspase activation (Fig 1F–M). We hypothesized that the BH3 motif may still interact with the BCL‐2 proteins and play an important role albeit being dispensable for their gross interaction. To test this hypothesis, we assessed the interaction of Synr's BH3 motif and BCL‐2 proteins by performing site‐specific photo‐crosslinking (Chin et al, 2002; Shiota et al, 2011). We incorporated p‐benzoyl‐L‐phenylalanine (BPA) either in the BH3 motif (124Y) or in its proximity (141F) (Fig 2N). This experimentation enables assessment of the physical interaction between the BH3 motif and Buffy/Debcl. We found that UV‐induced crosslinking occurs between the inserted BPA and Buffy/Debcl, demonstrating their physical interaction (Fig 2O and P).Additionally, we found that Synr can pull down Debcl and Buffy simultaneously (Fig 2Q), suggesting a possibility that the three proteins make a complex. To further verify this possibility, we performed a sequential pull‐down experiment by pulling down Flag‐Buffy, followed by elution with the Flag peptide and second pull‐down of Myc‐Synr. This sequential pull‐down experiment led to co‐immunoprecipitation of HA‐Debcl (Fig 2R). Based on these biochemical data and genetic data, we speculate that Synr makes a complex with Buffy and Debcl, where Synr's BH3 motif could exert a crucial effect on the protein complex's function. It is of note that Buffy and Debcl themselves interact with each other (Quinn et al, 2003), indicating that all the three proteins can interact with each other. The idea that Synr makes a complex with Debcl and Buffy is consistent with our genetic data that inhibition of either Debcl or Buffy is sufficient to suppress the synr‐induced phenotype.

Synr interacts with Debcl and Buffy both genetically and biochemically

BH3‐only proteins work through BCL‐2 proteins (Bouillet & Strasser, 2002; Giam et al, 2008; Lomonosova & Chinnadurai, 2008; Shamas‐Din et al, 2011; Happo et al, 2012; Aouacheria et al, 2015; Doerflinger et al, 2015). Drosophila has two BCL‐2 proteins, Buffy and Debcl (Brachmann et al, 2000; Colussi et al, 2000; Igaki et al, 2000; Zhang et al, 2000; Quinn et al, 2003; Igaki & Miura, 2004; Doumanis et al, 2007). Debcl was originally identified as a proapoptotic BCL‐2 protein (Brachmann et al, 2000; Colussi et al, 2000; Igaki et al, 2000; Zhang et al, 2000) whereas Buffy as an antiapoptotic BCL‐2 protein (Quinn et al, 2003). Yet, later studies suggested that whether they behave as an anti or proapoptotic factor depends on cell types and physiological contexts (Igaki & Miura, 2004; Senoo‐Matsuda et al, 2005; Doumanis et al, 2007; Clavier et al, 2015). We found that inhibition of either Buffy or Debcl suppresses the adult wing phenotype induced by Synr (Fig 2A–C and E). We also found that inhibition of the executioner caspase, Dcp‐1, inhibits the Synr‐induced wing phenotype (Fig 2D and E). Consistent with the adult wing results, inhibition of Debcl or Buffy suppressed Synr‐mediated caspase activation, which was detected by cleavage of Dcp‐1 and GC3Ai in the wing disc (Fig 2F and G). These genetic epistasis data demonstrate that synr and the BCL‐2 family members Debcl and Buffy, genetically interact.

Synr interacts with Debcl and Buffy both genetically and biochemically

A–DThe Synr‐induced wing structural defect is suppressed by knockdown of Debcl, Buffy or DCP1.EQuantification of the adult wing size with RNAis for apoptotic genes.FSynr‐induced caspase activation is suppressed by knockdown of Debcl.GGC3Ai signals are suppressed by knockdown of Debcl or Buffy (quantification is shown in Fig EV2H).H, ILysates from HEK293T cells that express Flag‐Debcl or Flag‐Buffy were incubated with GST or GST‐Synr bound to glutathione Sepharose. Flag‐Debcl or Flag‐Buffy is pulled down with GST‐Synr but not with GST.JDeletion of neither the coiled‐coil domain nor the BH3 motif suppresses binding Flag‐Debcl to GST‐Synr.KQuantification of the GST pull‐down assay shown in (J).LDeletion of neither the coiled‐coil domain nor the BH3 motif suppresses binding of Flag‐Buffy to GST‐Synr.MQuantification of the GST pull‐down assay shown in (L).NAmino acids that were replaced with BPA are shown. Tyrosine124 is within the BH3 motif and phenylalanine141 is in a close proximity to the BH3 motif. Tyrosine340 is for a negative control.ODebcl can be photo‐crosslinked with Synr at BPA that corresponds to Y124 and F141. BPA at Y340 was used as a negative control. Red arrowheads indicate covalently crosslinked products.PBuffy is also photo‐crosslinked with Synr at BPA that corresponds to Y124 and F141. Red arrowheads indicate covalently crosslinked products.QGST‐Synr pulls down both Flag‐Buffy and HA‐Debcl simultaneously, suggesting a possibility of complex formation among Synr, Buffy and Debcl.RHA‐Debcl is detected after two series of immunoprecipitation. The first immunoprecipitation was performed for Flag‐Buffy, followed by Flag peptide‐mediated elution and secondary precipitation for Myc‐Synr. HA‐Debcl can be detected after this sequential immunoprecipitation.

The Synr‐induced wing structural defect is suppressed by knockdown of Debcl, Buffy or DCP1.

Quantification of the adult wing size with RNAis for apoptotic genes.

Synr‐induced caspase activation is suppressed by knockdown of Debcl.

GC3Ai signals are suppressed by knockdown of Debcl or Buffy (quantification is shown in Fig EV2H).

Lysates from HEK293T cells that express Flag‐Debcl or Flag‐Buffy were incubated with GST or GST‐Synr bound to glutathione Sepharose. Flag‐Debcl or Flag‐Buffy is pulled down with GST‐Synr but not with GST.

Deletion of neither the coiled‐coil domain nor the BH3 motif suppresses binding Flag‐Debcl to GST‐Synr.

Quantification of the GST pull‐down assay shown in (J).

Deletion of neither the coiled‐coil domain nor the BH3 motif suppresses binding of Flag‐Buffy to GST‐Synr.

Quantification of the GST pull‐down assay shown in (L).

Amino acids that were replaced with BPA are shown. Tyrosine124 is within the BH3 motif and phenylalanine141 is in a close proximity to the BH3 motif. Tyrosine340 is for a negative control.

Debcl can be photo‐crosslinked with Synr at BPA that corresponds to Y124 and F141. BPA at Y340 was used as a negative control. Red arrowheads indicate covalently crosslinked products.

Buffy is also photo‐crosslinked with Synr at BPA that corresponds to Y124 and F141. Red arrowheads indicate covalently crosslinked products.

GST‐Synr pulls down both Flag‐Buffy and HA‐Debcl simultaneously, suggesting a possibility of complex formation among Synr, Buffy and Debcl.

HA‐Debcl is detected after two series of immunoprecipitation. The first immunoprecipitation was performed for Flag‐Buffy, followed by Flag peptide‐mediated elution and secondary precipitation for Myc‐Synr. HA‐Debcl can be detected after this sequential immunoprecipitation.

Data information: Statistical significance was determined using one‐way repeated measures ANOVA with Dunnett's post hoc test. Scale bars, 500 μm in (A) and 50 μm in (F) and (G).

Source data are available online for this figure.

To examine whether they biochemically interact, we investigated biochemical binding of Synr to Debcl or Buffy. We performed GST pull‐down assays using GST‐Synr bound to glutathione Sepharose, and Buffy or Debcl, which was expressed in HEK293T cells. We found that Synr interacts with both BCL‐2 proteins (Fig 2H and I). This strongly supports the idea to posit Synr as the caspase‐activating BH3‐only protein that interacts genetically and biochemically with BCL‐2 proteins.

We then investigated which part of Synr is important for binding to Debcl or Buffy. In most BH3‐only proteins, the BH3 motif is crucial for binding to BCL‐2 proteins (Bouillet & Strasser, 2002; Giam et al, 2008; Lomonosova & Chinnadurai, 2008; Shamas‐Din et al, 2011; Happo et al, 2012; Aouacheria et al, 2015; Doerflinger et al, 2015). Interestingly, deletion of either the BH3 motif or the coiled‐coil domain did not affect Synr's binding to the BCL‐2 proteins (Fig 2J–M). This indicates that Synr's physical interaction with Buffy or Debcl can occur without the BH3 motif. This seemed at a glance contradictory from the genetic data that Synr's BH3 motif is indispensable for caspase activation (Fig 1F–M). We hypothesized that the BH3 motif may still interact with the BCL‐2 proteins and play an important role albeit being dispensable for their gross interaction. To test this hypothesis, we assessed the interaction of Synr's BH3 motif and BCL‐2 proteins by performing site‐specific photo‐crosslinking (Chin et al, 2002; Shiota et al, 2011). We incorporated p‐benzoyl‐L‐phenylalanine (BPA) either in the BH3 motif (124Y) or in its proximity (141F) (Fig 2N). This experimentation enables assessment of the physical interaction between the BH3 motif and Buffy/Debcl. We found that UV‐induced crosslinking occurs between the inserted BPA and Buffy/Debcl, demonstrating their physical interaction (Fig 2O and P).

Additionally, we found that Synr can pull down Debcl and Buffy simultaneously (Fig 2Q), suggesting a possibility that the three proteins make a complex. To further verify this possibility, we performed a sequential pull‐down experiment by pulling down Flag‐Buffy, followed by elution with the Flag peptide and second pull‐down of Myc‐Synr. This sequential pull‐down experiment led to co‐immunoprecipitation of HA‐Debcl (Fig 2R). Based on these biochemical data and genetic data, we speculate that Synr makes a complex with Buffy and Debcl, where Synr's BH3 motif could exert a crucial effect on the protein complex's function. It is of note that Buffy and Debcl themselves interact with each other (Quinn et al, 2003), indicating that all the three proteins can interact with each other. The idea that Synr makes a complex with Debcl and Buffy is consistent with our genetic data that inhibition of either Debcl or Buffy is sufficient to suppress the synr‐induced phenotype.

Feedback loop between apoptosis and autophagyWhile we were characterizing the effects of synr expression, we unexpectedly found that Synr induces autophagy, which was detected by punctate accumulation of Lysotracker (Fig 3A) and mCherry‐Atg8a (Fig 3B). Synr also colocalized with Atg8a and Rab27, which label autophagosomes (Nagy et al, 2015; Underwood et al, 2020), in the salivary gland and the wing disc (Figs 3C and EV2A). The relationship between autophagy and apoptosis is enigmatic (Gump & Thorburn, 2011; Marino et al, 2014; Bialik et al, 2018; Doherty & Baehrecke, 2018). Autophagy could occur as a byproduct of apoptosis or could affect the apoptotic process. While autophagy is in principle a mechanism for cell survival, it can induce caspase‐dependent apoptosis (Scott et al, 2007; Mohseni et al, 2009; Nagata et al, 2019) as well as caspase‐independent cell death, which is called autophagic cell death (Berry & Baehrecke, 2007; Bialik et al, 2018). Additionally, caspase can also induce autophagy (Hou et al, 2008). Which role autophagy plays depends on cellular contexts. Interestingly, inhibition of autophagy suppressed both the Synr‐induced wing phenotype and cell death detected by propidium iodide and TUNEL (Figs 3D–G and EV2B–D), suggesting that Synr‐induced autophagy is not just a byproduct of caspase‐mediated apoptosis or counteractive for apoptosis, rather it has a positive contribution to the cell death phenotype. In addition to Synr‐induced cell death, we also found that reaper (rpr), which activates apoptosis through DIAP1 inhibition (Goyal et al, 2000), induces autophagy (Fig EV2E and F) and that the rpr‐induced phenotype in the adult wing is suppressed by autophagy inhibition (Fig EV2G).Figure 3Feedback loop between apoptosis and autophagy
AEctopic expression of Synr induces autolysosome accumulation, which was detected by Lysotracker, in the wing disc.BEctopic expression of Synr also induces autophagosome accumulation, which was detected by mCherry‐Atg8a. The dotted square regions are magnified in the right pictures.CSynr is colocalized with mCherry‐Atg8a (arrowheads) in the wing disc and the salivary gland at 112 h AEL.D–FKnockdown of Atg2 or Atg8a suppresses the wing structural defect that is induced by Synr.G
Atg2 knockdown suppresses Synr‐induced cell death, which was detected by TUNEL staining.H, ISynr‐induced caspase activation is suppressed by inhibition of autophagy (Atg2, Atg8a RNAis) in a similar manner to apoptosis inhibition (Debcl, Dcp‐1 RNAis).JSynr‐induced autolysosome accumulation is suppressed by Dcp‐1 knockdown.
Data information: Statistical significance was determined using one‐way ANOVA with Dunnett's post hoc test. Scale bars, 100 μm in (A) and (H), 50 μm in (B) and (J), 10 μm in (C) and (G), and 500 μm in (D).
Source data are available online for this figure.
Figure EV2Synr‐induced autophagy
Synr is colocalized with Rab27 (arrowheads) in the wing disc and the salivary gland at 112 h AEL.The Synr‐induced wing structural defect is suppressed by autophagy inhibition.Quantification of the wing size with a combination of Synr expression and autophagy inhibition.
Atg2 knockdown suppresses Synr‐induced cell death, which was detected by propidium iodide (PI).A DIAP1 inhibitor, reaper induces autolysosome accumulation in the wing disc.Reaper also induces autophagosome accumulation in the wing disc. The yellow‐dotted square region is magnified in the right picture.Reaper‐induced wing defects are suppressed by autophagy inhibition. Wing phenotypes were divided into five classes based on the severity as indicated in pictures.Knockdown of Debcl, Buffy, or Atg2 significantly suppresses caspase activation compared with the control RNAi for luciferase.
Data information: Statistical significance was determined using one‐way ANOVA with Dunnett's post hoc test. Scale bars, 10 μm in (A), 500 μm in (B), and 50 μm in (D–F).
Source data are available online for this figure.
We further clarified how autophagy regulates Synr‐induced cell death. Autophagy inhibition suppressed caspase activation by Synr, in a similar manner to inhibition of Dcp‐1 or Debcl (Figs 3H and I, and EV2H). On the contrary, Dcp‐1 inhibition suppressed Synr‐induced Lysotracker puncta formation, suggesting that caspase is necessary for Synr‐induced autophagy (Fig 3J). These data indicate that there is a positive feedback loop between apoptosis and autophagy, activating each other at least in a context of Synr‐induced cell death.

Feedback loop between apoptosis and autophagy

While we were characterizing the effects of synr expression, we unexpectedly found that Synr induces autophagy, which was detected by punctate accumulation of Lysotracker (Fig 3A) and mCherry‐Atg8a (Fig 3B). Synr also colocalized with Atg8a and Rab27, which label autophagosomes (Nagy et al, 2015; Underwood et al, 2020), in the salivary gland and the wing disc (Figs 3C and EV2A). The relationship between autophagy and apoptosis is enigmatic (Gump & Thorburn, 2011; Marino et al, 2014; Bialik et al, 2018; Doherty & Baehrecke, 2018). Autophagy could occur as a byproduct of apoptosis or could affect the apoptotic process. While autophagy is in principle a mechanism for cell survival, it can induce caspase‐dependent apoptosis (Scott et al, 2007; Mohseni et al, 2009; Nagata et al, 2019) as well as caspase‐independent cell death, which is called autophagic cell death (Berry & Baehrecke, 2007; Bialik et al, 2018). Additionally, caspase can also induce autophagy (Hou et al, 2008). Which role autophagy plays depends on cellular contexts. Interestingly, inhibition of autophagy suppressed both the Synr‐induced wing phenotype and cell death detected by propidium iodide and TUNEL (Figs 3D–G and EV2B–D), suggesting that Synr‐induced autophagy is not just a byproduct of caspase‐mediated apoptosis or counteractive for apoptosis, rather it has a positive contribution to the cell death phenotype. In addition to Synr‐induced cell death, we also found that reaper (rpr), which activates apoptosis through DIAP1 inhibition (Goyal et al, 2000), induces autophagy (Fig EV2E and F) and that the rpr‐induced phenotype in the adult wing is suppressed by autophagy inhibition (Fig EV2G).

Feedback loop between apoptosis and autophagy

AEctopic expression of Synr induces autolysosome accumulation, which was detected by Lysotracker, in the wing disc.BEctopic expression of Synr also induces autophagosome accumulation, which was detected by mCherry‐Atg8a. The dotted square regions are magnified in the right pictures.CSynr is colocalized with mCherry‐Atg8a (arrowheads) in the wing disc and the salivary gland at 112 h AEL.D–FKnockdown of Atg2 or Atg8a suppresses the wing structural defect that is induced by Synr.G
Atg2 knockdown suppresses Synr‐induced cell death, which was detected by TUNEL staining.H, ISynr‐induced caspase activation is suppressed by inhibition of autophagy (Atg2, Atg8a RNAis) in a similar manner to apoptosis inhibition (Debcl, Dcp‐1 RNAis).JSynr‐induced autolysosome accumulation is suppressed by Dcp‐1 knockdown.

Ectopic expression of Synr induces autolysosome accumulation, which was detected by Lysotracker, in the wing disc.

Ectopic expression of Synr also induces autophagosome accumulation, which was detected by mCherry‐Atg8a. The dotted square regions are magnified in the right pictures.

Synr is colocalized with mCherry‐Atg8a (arrowheads) in the wing disc and the salivary gland at 112 h AEL.

Knockdown of Atg2 or Atg8a suppresses the wing structural defect that is induced by Synr.

Atg2 knockdown suppresses Synr‐induced cell death, which was detected by TUNEL staining.

Synr‐induced caspase activation is suppressed by inhibition of autophagy (Atg2, Atg8a RNAis) in a similar manner to apoptosis inhibition (Debcl, Dcp‐1 RNAis).

Synr‐induced autolysosome accumulation is suppressed by Dcp‐1 knockdown.

Data information: Statistical significance was determined using one‐way ANOVA with Dunnett's post hoc test. Scale bars, 100 μm in (A) and (H), 50 μm in (B) and (J), 10 μm in (C) and (G), and 500 μm in (D).

Source data are available online for this figure.

Synr‐induced autophagy

Synr is colocalized with Rab27 (arrowheads) in the wing disc and the salivary gland at 112 h AEL.The Synr‐induced wing structural defect is suppressed by autophagy inhibition.Quantification of the wing size with a combination of Synr expression and autophagy inhibition.
Atg2 knockdown suppresses Synr‐induced cell death, which was detected by propidium iodide (PI).A DIAP1 inhibitor, reaper induces autolysosome accumulation in the wing disc.Reaper also induces autophagosome accumulation in the wing disc. The yellow‐dotted square region is magnified in the right picture.Reaper‐induced wing defects are suppressed by autophagy inhibition. Wing phenotypes were divided into five classes based on the severity as indicated in pictures.Knockdown of Debcl, Buffy, or Atg2 significantly suppresses caspase activation compared with the control RNAi for luciferase.

Synr is colocalized with Rab27 (arrowheads) in the wing disc and the salivary gland at 112 h AEL.

The Synr‐induced wing structural defect is suppressed by autophagy inhibition.

Quantification of the wing size with a combination of Synr expression and autophagy inhibition.

Atg2 knockdown suppresses Synr‐induced cell death, which was detected by propidium iodide (PI).

A DIAP1 inhibitor, reaper induces autolysosome accumulation in the wing disc.

Reaper also induces autophagosome accumulation in the wing disc. The yellow‐dotted square region is magnified in the right picture.

Reaper‐induced wing defects are suppressed by autophagy inhibition. Wing phenotypes were divided into five classes based on the severity as indicated in pictures.

Knockdown of Debcl, Buffy, or Atg2 significantly suppresses caspase activation compared with the control RNAi for luciferase.

Data information: Statistical significance was determined using one‐way ANOVA with Dunnett's post hoc test. Scale bars, 10 μm in (A), 500 μm in (B), and 50 μm in (D–F).

Source data are available online for this figure.

We further clarified how autophagy regulates Synr‐induced cell death. Autophagy inhibition suppressed caspase activation by Synr, in a similar manner to inhibition of Dcp‐1 or Debcl (Figs 3H and I, and EV2H). On the contrary, Dcp‐1 inhibition suppressed Synr‐induced Lysotracker puncta formation, suggesting that caspase is necessary for Synr‐induced autophagy (Fig 3J). These data indicate that there is a positive feedback loop between apoptosis and autophagy, activating each other at least in a context of Synr‐induced cell death.

Endogenous function of SynrFinally, we tested endogenous functions of synr. In vertebrates, BH3‐only proteins often function downstream of p53 (Happo et al, 2012). We found that the p53‐induced wing phenotype was well suppressed by synr RNAis that we newly generated (Figs 4A and EV3A). synr RNAis also suppressed p53‐induced caspase activation (Figs 4B and EV3B), indicating that endogenous synr regulates p53‐mediated apoptosis.Figure 4Endogenous function of Synr
Inhibition of Synr suppresses the wing defect that is induced by p53, a potent apoptosis inducer.Inhibition of Synr suppresses p53‐induced caspase activation.Inhibition of Synr in the midgut enterocytes suppresses starvation‐induced caspase activation in the midgut.Inhibition of Synr in the midgut enterocytes enhances resistance to starvation.The synr mutant also exhibits enhanced resistance to starvation.A proposed model of Synr‐induced cell death. Synr, which functions together with Buffy and Debcl, induces initial caspase activation, which leads to feedback loop between apoptosis and autophagy.
Data information: Statistical significance was determined using a two‐tailed unpaired t‐test (B), one‐way ANOVA with Holm‐Sidak's multiple comparisons test (C), and a log‐rank (Mantel–Cox) test (D, E). Scale bar, 500 μm in (A).
Source data are available online for this figure.
Figure EV3Knockdown and knockout of synr

A newly generated synr RNAi line can suppress the effect of ectopic synr expression on wings.Two RNAis for synr can suppress p53‐mediated caspase activation, which is detected by GC3Ai.Schematic of MiMIC (Minos‐mediated integration cassette) insertion in synr (CGCG14044MI09290).A design of primers to detect MiMIC insertion.Using the primers, the MiMIC insertion was confirmed by PCR. During extensive outcross of CGCG14044MI09290 to OregonR, MiMIC insertion was confirmed by PCR.The extensively outcrossed synr mutant demonstrates a slight, but reproducible developmental delay during development.
Data information: Statistical significance was determined using one‐way ANOVA with Holm‐Sidak's multiple comparisons test (B) and a log‐rank (Mantel–Cox) test (F). Scale bar, 500 μm in (A).
Source data are available online for this figure.
We also examined more physiological functions of synr. Starvation induces caspase activation in the gut enterocytes (O'Brien et al, 2011). We found that synr inhibition suppressed starvation‐induced caspase activation (Fig 4C). Interestingly, synr mutation or enterocyte‐specific synr knockdown enhanced resistance to starvation (Figs 4D and E, and EV3C–E), supporting a physiological function of endogenous synr.It is of note that the synr mutant was viable, suggesting that synr is not necessary for apoptosis during development, which is similar to mutants of Buffy or Debcl (Sevrioukov et al, 2007). However, we reproducibly found that the extensively outcrossed synr mutant develops slightly faster than control (Fig EV3F), suggesting that synr could play some role during development.

Endogenous function of Synr

Finally, we tested endogenous functions of synr. In vertebrates, BH3‐only proteins often function downstream of p53 (Happo et al, 2012). We found that the p53‐induced wing phenotype was well suppressed by synr RNAis that we newly generated (Figs 4A and EV3A). synr RNAis also suppressed p53‐induced caspase activation (Figs 4B and EV3B), indicating that endogenous synr regulates p53‐mediated apoptosis.

Endogenous function of Synr

Inhibition of Synr suppresses the wing defect that is induced by p53, a potent apoptosis inducer.Inhibition of Synr suppresses p53‐induced caspase activation.Inhibition of Synr in the midgut enterocytes suppresses starvation‐induced caspase activation in the midgut.Inhibition of Synr in the midgut enterocytes enhances resistance to starvation.The synr mutant also exhibits enhanced resistance to starvation.A proposed model of Synr‐induced cell death. Synr, which functions together with Buffy and Debcl, induces initial caspase activation, which leads to feedback loop between apoptosis and autophagy.

Inhibition of Synr suppresses the wing defect that is induced by p53, a potent apoptosis inducer.

Inhibition of Synr suppresses p53‐induced caspase activation.

Inhibition of Synr in the midgut enterocytes suppresses starvation‐induced caspase activation in the midgut.

Inhibition of Synr in the midgut enterocytes enhances resistance to starvation.

The synr mutant also exhibits enhanced resistance to starvation.

A proposed model of Synr‐induced cell death. Synr, which functions together with Buffy and Debcl, induces initial caspase activation, which leads to feedback loop between apoptosis and autophagy.

Data information: Statistical significance was determined using a two‐tailed unpaired t‐test (B), one‐way ANOVA with Holm‐Sidak's multiple comparisons test (C), and a log‐rank (Mantel–Cox) test (D, E). Scale bar, 500 μm in (A).

Source data are available online for this figure.

Knockdown and knockout of synr

A newly generated synr RNAi line can suppress the effect of ectopic synr expression on wings.Two RNAis for synr can suppress p53‐mediated caspase activation, which is detected by GC3Ai.Schematic of MiMIC (Minos‐mediated integration cassette) insertion in synr (CGCG14044MI09290).A design of primers to detect MiMIC insertion.Using the primers, the MiMIC insertion was confirmed by PCR. During extensive outcross of CGCG14044MI09290 to OregonR, MiMIC insertion was confirmed by PCR.The extensively outcrossed synr mutant demonstrates a slight, but reproducible developmental delay during development.

A newly generated synr RNAi line can suppress the effect of ectopic synr expression on wings.

Two RNAis for synr can suppress p53‐mediated caspase activation, which is detected by GC3Ai.

Schematic of MiMIC (Minos‐mediated integration cassette) insertion in synr (CGCG14044MI09290).

A design of primers to detect MiMIC insertion.

Using the primers, the MiMIC insertion was confirmed by PCR. During extensive outcross of CGCG14044MI09290 to OregonR, MiMIC insertion was confirmed by PCR.

The extensively outcrossed synr mutant demonstrates a slight, but reproducible developmental delay during development.

Data information: Statistical significance was determined using one‐way ANOVA with Holm‐Sidak's multiple comparisons test (B) and a log‐rank (Mantel–Cox) test (F). Scale bar, 500 μm in (A).

Source data are available online for this figure.

We also examined more physiological functions of synr. Starvation induces caspase activation in the gut enterocytes (O'Brien et al, 2011). We found that synr inhibition suppressed starvation‐induced caspase activation (Fig 4C). Interestingly, synr mutation or enterocyte‐specific synr knockdown enhanced resistance to starvation (Figs 4D and E, and EV3C–E), supporting a physiological function of endogenous synr.

It is of note that the synr mutant was viable, suggesting that synr is not necessary for apoptosis during development, which is similar to mutants of Buffy or Debcl (Sevrioukov et al, 2007). However, we reproducibly found that the extensively outcrossed synr mutant develops slightly faster than control (Fig EV3F), suggesting that synr could play some role during development.

DiscussionHere, we report the original identification and characterization of the first BH3‐only protein that induces caspase activation in Drosophila. This challenges the dogma that Drosophila is devoid of a BH3‐only protein acting in cell death control.One obvious question arising from our work is why a BH3‐only protein was not found for over 20 years in Drosophila. First, in silico prediction of BH3‐only proteins is notoriously difficult because of the degenerate (low complexity) nature of the BH3 motif signature (Aouacheria et al, 2013, 2015), thereby requiring experimental validation through laborious and time‐consuming in vitro and in vivo assays. Another reason for missing the fly BH3‐only protein for long could be related to the important roles DIAP1 and its regulators Rpr, Hid, and grim play in Drosophila, which probably led to underappreciation of the BCL‐2 pathway.Synr‐induced caspase activation induces autophagy, which also requires caspase for its activation. Previous reports indicated that autophagy can induce caspase‐dependent apoptosis (Scott et al, 2007; Mohseni et al, 2009; Nagata et al, 2019). On the contrary, caspases can also induce autophagy (Hou et al, 2008). We also observed that Rpr‐mediated caspase activation induces autophagy (Fig EV2F–H). Our results integrate these observations in different systems and propose a feedback mechanism in a single setting (Fig 4F). We speculate that the feedback loop between caspase and autophagy could be of importance especially when caspase induction is relatively weak. Currently, we do not know how exactly caspase and autophagy components such as Atg proteins interact with each other. Discovering the detailed mechanistic basis of the coupling of apoptosis and autophagy is an important avenue for further study.Contrary to the BCL‐2 family of homologous proteins, which have additional BH motifs and share globular structures and common ancestry, most BH3‐only proteins are phylogenetically unrelated (Aouacheria et al, 2005, 2013). Now defined as a particular class of short linear motifs (Aouacheria et al, 2015), BH3 motifs are expected to easily evolve de novo through convergent evolution or random incidence (homoplasy), and not exclusively through divergent evolution like BCL‐2 family members (Aouacheria et al, 2015). Our discovery that the BH3 motif of Synr was a dipteran synapomorphy fits well into this paradigm. For example, although Cil‐7 in C. elegans and Synr, both of which belong to the same protein family (PTHR21974), share similar coiled‐coil domains in their C‐terminal halves, their N‐terminal regions greatly vary in length and amino acid sequence and Cil‐7 does not have any recognizable BH3 motif.Based on our genetic and biochemical data, we propose that Synr makes a complex with Debcl and Buffy, which is critical for caspase activation. In this model, both Debcl and Buffy function as proapoptotic BCL‐2 proteins. This is reminiscent of the direct model rather than the indirect model in mammalian BH3‐only proteins (Bouillet & Strasser, 2002; Giam et al, 2008; Lomonosova & Chinnadurai, 2008; Shamas‐Din et al, 2011; Happo et al, 2012; Aouacheria et al, 2015; Doerflinger et al, 2015). One unique aspect of the BH3 motif of Synr is, in spite of its importance in Synr proapoptotic activity, being dispensable for Synr overall binding to Debcl/Buffy. At the same time, the BH3 motif binds to Debcl/Buffy, as demonstrated by the photo‐crosslinking at the BH3 motif. We speculate that the local binding of Synr BH3 and Debcl/Buffy is critical for the function of the protein complex. Future protein structural analyses will reveal the molecular details by which Synr‐BH3 motif exerts its function.Our findings provide a unique insight into how a molecular machinery of apoptosis can be formed through possible coevolution of several genes. Our phylogenetic analyses suggest that a recognizable BH3 motif of synr appeared (Fig 1C) at the time (or soon afterwards) a gene duplication produced the pair of paralogous genes Buffy and Debcl in Diptera (Fig EV4A). We inferred and synthesized the sequence of the putative ancestor of Buffy/Debcl, which we named dBorg0, based on ancestral protein reconstruction (Fig EV4B). dBorg0 could biochemically bind to all of Synr WT, the coiled‐coil deletion mutant and the BH3 deletion mutant (Fig EV4C–F), which is similar to Debcl and Buffy. Based on the evolutionary history of Synr, Debcl and Buffy and biochemical binding data (Figs 2H–R and EV4), we speculate the following evolutionary scenario. A BH3 motif evolved in the coiled‐coil domain‐containing protein in a subset of Diptera (that led to extant Muscomorpha or Brachycera), giving birth to synr with a BH3 motif. At a similar timing, in the lineage that led to modern Diptera, duplication of the ancestral BCL‐2 homologous gene dBorg0 led to the generation of the Debcl and Buffy pair of paralogous proteins. The two paralogs underwent functional divergence and subsequent specialization of their protein–protein interaction network, leading to functionalization of the apoptosis‐inducing machinery complex Synr‐Debcl‐Buffy.Figure EV4Evolution of the Buffy–Debcl orthology group in insects
APhylogenetic subtree depicting the Buffy–Debcl duplication in Diptera. The Buffy–Debcl ancestral protein before duplication was named dBorg0, since Debcl and Buffy are also known as dBorg‐1 and dBorg‐2, respectively. The scale bar indicates 0.5 substitution per site.BThe amino acid sequence of dBorg0 was predicted based on ancestral protein reconstruction (see Materials and Methods). Multiple sequence alignment of Drosophila Debcl (also known as dBorg‐1) and Buffy (dBorg‐2) amino acid sequences with that of their putative ancestor dBorg0.C–FLysates from HEK293T cells that express Flag‐dBorg0 were incubated with GST, GST‐Synr WT, GST‐Synr ΔBH3, or GST‐Synr Δcoiled‐coil mutant, which was bound to glutathione Sepharose. Synr WT, Synr BH3 mutant, and Synr coiled‐coil domain mutant can pull down dBorg0 in a similar manner, indicating that the coiled‐coil region and the BH3 motif are dispensable for Synr's binding to dBorg0, which is similar to Buffy and Debcl.

Source data are available online for this figure.

Here, we report the original identification and characterization of the first BH3‐only protein that induces caspase activation in Drosophila. This challenges the dogma that Drosophila is devoid of a BH3‐only protein acting in cell death control.

One obvious question arising from our work is why a BH3‐only protein was not found for over 20 years in Drosophila. First, in silico prediction of BH3‐only proteins is notoriously difficult because of the degenerate (low complexity) nature of the BH3 motif signature (Aouacheria et al, 2013, 2015), thereby requiring experimental validation through laborious and time‐consuming in vitro and in vivo assays. Another reason for missing the fly BH3‐only protein for long could be related to the important roles DIAP1 and its regulators Rpr, Hid, and grim play in Drosophila, which probably led to underappreciation of the BCL‐2 pathway.

Synr‐induced caspase activation induces autophagy, which also requires caspase for its activation. Previous reports indicated that autophagy can induce caspase‐dependent apoptosis (Scott et al, 2007; Mohseni et al, 2009; Nagata et al, 2019). On the contrary, caspases can also induce autophagy (Hou et al, 2008). We also observed that Rpr‐mediated caspase activation induces autophagy (Fig EV2F–H). Our results integrate these observations in different systems and propose a feedback mechanism in a single setting (Fig 4F). We speculate that the feedback loop between caspase and autophagy could be of importance especially when caspase induction is relatively weak. Currently, we do not know how exactly caspase and autophagy components such as Atg proteins interact with each other. Discovering the detailed mechanistic basis of the coupling of apoptosis and autophagy is an important avenue for further study.

Contrary to the BCL‐2 family of homologous proteins, which have additional BH motifs and share globular structures and common ancestry, most BH3‐only proteins are phylogenetically unrelated (Aouacheria et al, 2005, 2013). Now defined as a particular class of short linear motifs (Aouacheria et al, 2015), BH3 motifs are expected to easily evolve de novo through convergent evolution or random incidence (homoplasy), and not exclusively through divergent evolution like BCL‐2 family members (Aouacheria et al, 2015). Our discovery that the BH3 motif of Synr was a dipteran synapomorphy fits well into this paradigm. For example, although Cil‐7 in C. elegans and Synr, both of which belong to the same protein family (PTHR21974), share similar coiled‐coil domains in their C‐terminal halves, their N‐terminal regions greatly vary in length and amino acid sequence and Cil‐7 does not have any recognizable BH3 motif.

Based on our genetic and biochemical data, we propose that Synr makes a complex with Debcl and Buffy, which is critical for caspase activation. In this model, both Debcl and Buffy function as proapoptotic BCL‐2 proteins. This is reminiscent of the direct model rather than the indirect model in mammalian BH3‐only proteins (Bouillet & Strasser, 2002; Giam et al, 2008; Lomonosova & Chinnadurai, 2008; Shamas‐Din et al, 2011; Happo et al, 2012; Aouacheria et al, 2015; Doerflinger et al, 2015). One unique aspect of the BH3 motif of Synr is, in spite of its importance in Synr proapoptotic activity, being dispensable for Synr overall binding to Debcl/Buffy. At the same time, the BH3 motif binds to Debcl/Buffy, as demonstrated by the photo‐crosslinking at the BH3 motif. We speculate that the local binding of Synr BH3 and Debcl/Buffy is critical for the function of the protein complex. Future protein structural analyses will reveal the molecular details by which Synr‐BH3 motif exerts its function.

Our findings provide a unique insight into how a molecular machinery of apoptosis can be formed through possible coevolution of several genes. Our phylogenetic analyses suggest that a recognizable BH3 motif of synr appeared (Fig 1C) at the time (or soon afterwards) a gene duplication produced the pair of paralogous genes Buffy and Debcl in Diptera (Fig EV4A). We inferred and synthesized the sequence of the putative ancestor of Buffy/Debcl, which we named dBorg0, based on ancestral protein reconstruction (Fig EV4B). dBorg0 could biochemically bind to all of Synr WT, the coiled‐coil deletion mutant and the BH3 deletion mutant (Fig EV4C–F), which is similar to Debcl and Buffy. Based on the evolutionary history of Synr, Debcl and Buffy and biochemical binding data (Figs 2H–R and EV4), we speculate the following evolutionary scenario. A BH3 motif evolved in the coiled‐coil domain‐containing protein in a subset of Diptera (that led to extant Muscomorpha or Brachycera), giving birth to synr with a BH3 motif. At a similar timing, in the lineage that led to modern Diptera, duplication of the ancestral BCL‐2 homologous gene dBorg0 led to the generation of the Debcl and Buffy pair of paralogous proteins. The two paralogs underwent functional divergence and subsequent specialization of their protein–protein interaction network, leading to functionalization of the apoptosis‐inducing machinery complex Synr‐Debcl‐Buffy.

Evolution of the Buffy–Debcl orthology group in insects

APhylogenetic subtree depicting the Buffy–Debcl duplication in Diptera. The Buffy–Debcl ancestral protein before duplication was named dBorg0, since Debcl and Buffy are also known as dBorg‐1 and dBorg‐2, respectively. The scale bar indicates 0.5 substitution per site.BThe amino acid sequence of dBorg0 was predicted based on ancestral protein reconstruction (see Materials and Methods). Multiple sequence alignment of Drosophila Debcl (also known as dBorg‐1) and Buffy (dBorg‐2) amino acid sequences with that of their putative ancestor dBorg0.C–FLysates from HEK293T cells that express Flag‐dBorg0 were incubated with GST, GST‐Synr WT, GST‐Synr ΔBH3, or GST‐Synr Δcoiled‐coil mutant, which was bound to glutathione Sepharose. Synr WT, Synr BH3 mutant, and Synr coiled‐coil domain mutant can pull down dBorg0 in a similar manner, indicating that the coiled‐coil region and the BH3 motif are dispensable for Synr's binding to dBorg0, which is similar to Buffy and Debcl.

Phylogenetic subtree depicting the Buffy–Debcl duplication in Diptera. The Buffy–Debcl ancestral protein before duplication was named dBorg0, since Debcl and Buffy are also known as dBorg‐1 and dBorg‐2, respectively. The scale bar indicates 0.5 substitution per site.

The amino acid sequence of dBorg0 was predicted based on ancestral protein reconstruction (see Materials and Methods). Multiple sequence alignment of Drosophila Debcl (also known as dBorg‐1) and Buffy (dBorg‐2) amino acid sequences with that of their putative ancestor dBorg0.

Lysates from HEK293T cells that express Flag‐dBorg0 were incubated with GST, GST‐Synr WT, GST‐Synr ΔBH3, or GST‐Synr Δcoiled‐coil mutant, which was bound to glutathione Sepharose. Synr WT, Synr BH3 mutant, and Synr coiled‐coil domain mutant can pull down dBorg0 in a similar manner, indicating that the coiled‐coil region and the BH3 motif are dispensable for Synr's binding to dBorg0, which is similar to Buffy and Debcl.

Source data are available online for this figure.

Materials and Methods
Drosophila husbandryFlies were maintained as described previously (Yoo et al, 2016). The fly food was composed of 0.8% agar, 10% glucose, 4.5% corn flour, 3.72% dry yeast, 0.4% propionic acid, and 0.3% butyl p‐hydroxybenzoate. All experiments were performed at 25°C except Figs 4A and B, and EV3B, which were conducted at 18°C.
Drosophila stocksThe fly stocks used in this study are listed in Dataset EV1. We also generated the following fly stocks by combining the listed stocks:

nub‐gal4, UAS‐GFP; UAS‐synr

nub‐gal4; UAS‐GC3Ai

nub‐gal4; UAS‐synr

nub‐gal4, UAS‐GFP; act‐mCherry‐Atg8a

vg‐gal4, UAS‐GFP; UAS‐rpr

nub‐gal4; UAS‐p53

5966GS‐gal4; UAS‐GC3Ai

synr
−/− was made by extensive outcross of CG14044

MI09290
 (50506, Bloomington Stock Center) to Oregon R. The MiMIC insertion was confirmed by PCR.Immunofluorescence and imagingWing discs or adult midguts were immunostained as described previously (Nishida et al, 2021; Sasaki et al, 2021). We used the following antibodies and fluorescent dyes at the indicated dilutions: cleaved Dcp1 antibody (1:500; 9578, Cell Signaling Technology), HA antibody (1:500; 901513, BAB), DAPI (1:500; D9542, Sigma), and Alexa Fluor secondary antibodies (1:500; A11008 and A11036, Thermo Fisher). Fluorescence images were acquired with a confocal microscope (Zeiss LSM 780, 880, 900). To measure the wing size, wings were imaged with a stereo microscope (Nikon SMZ18). The wing size was quantified using Fiji. To quantify the percentage of GC3Ai‐positive areas, images were binarized with a certain intensity threshold with Fiji, and the percentage was calculated to the area with gene expression, which was determined by the GC3Ai background signal and morphology.
DNA constructionFor making transgenic flies, synr WT (FBpp0078629), synr BH3 motif deletion mutant, synr ccd deletion mutant, synr BH3 motif amino acid change mutant, or synr ccd amino acid change mutant was cloned into the pUASTattB vector. synr BH3 deletion mutant was made by overlap PCR to remove the BH3 motif (residues 122–131). synr coiled‐coil domain mutants were also generated similarly. We removed residues 162–182 for synr dcc1, residues 199–219 for synr dcc2, and residues 162–219 for synr dcc1‐2. Regarding AA substitution mutants, BH3 motif AA mutant was generated by overlap PCR. The ccd AA mutant was newly synthesized by GenScript.The amino acid sequences of BH3 motif AA mutant and ccd AA mutant are the following:
BH3 motif AA mutant, EAYNEGVEAAAEK (122–134); ccd AA mutant, ccd1, GGGTGGGGGTGGGSYGGGNGG (162–182), ccd2, GTGGGGSGGGQGGTGGGTGGG (199–219)
For HA‐tagged Synr, synr WT, ΔBH3, and BH3AA mutants were once inserted into the pCMV‐HA‐N vector (Takara #635690). Then, the tagged synr constructs were amplified by PCR and cloned into the pUASTattB vector. To knockdown synr, we generated shRNAi lines based on the Trip protocol by Jian‐Quan Ni and Norbert Perrimon at Harvard Medical School (https://fgr.hms.harvard.edu/files/fly/files/2ndgenprotocol.pdf). The following oligos were annealed and cloned into the pVALIUM20 vector.
synr ‐RNAi:
ctagcagtTCGGCAAGTCCTAGCCGTCAAtagttatattcaagcataTTGACGGCTAGGACTTGCCGAgcgaattcgcTCGGCAAGTCCTAGCCGTCAAtatgcttgaatataactaTTGACGGCTAGGACTTGCCGAactg

synr ‐RNAi‐2:
ctagcagtAACGAGCTGTACGTAAAGCAAtagttatattcaagcataTTGCTTTACGTACAGCTCGTTgcgaattcgcAACGAGCTGTACGTAAAGCAAtatgcttgaatataactaTTGCTTTACGTACAGCTCGTTactg
Transgenic flies were generated by BestGene.For protein expression in E. coli, synr WT, ΔBH3, and Δccd1‐2 mutants were cloned into a pCold‐GST DNA (Takara #3372) with an N‐terminal GST tag.For protein expression in HEK293T cells, both Buffy (FBpp0087182) and Debcl (FBpp0085443) were cloned into a Flag vector (a gift from Dr Nishimura), where the Flag tag was added to the N terminus. We also generated HA‐Debcl with pCMV‐HA‐N vector (Takara #635690). Myc tag was also added at the N terminus of synr with pCMV‐Myc vector (Takara #635689). Regarding the dBorg0 experiment, based on the amino acid sequence of dBorg0 (Fig EV4B), the following DNA sequence was synthesized with the human codon usage and cloned into the Flag vector (GenScript): ATGCTGAGCACCACCGCCCCCCCCACCACCACCACCACCGCCAGCAGCAACCCCAGCCCCAACAGCAACAGCGTGAGCCCCAGCAGAAACCTGGCCAACCTGCACAGCAGCCCCCTGAACGGCGACCACCACCCCCCCAGCAGCAGAAGAAGACTGAGCAACGTGAGCGACGTGGTGACCAGAAAGCTGAGCAGCACCATCGGCTGGAAGCAGGCCGTGATCCCCAGCCAGGACATCATCACCCAGGGCAAGTGCCTGTGCGGCCAGTACATCAGAAGCAGACTGAAGAGAAGCGGCGTGTTCAACAGAAAGCTGGGCCTGCAGAGACTGAGAAGCATCGTGGGCGTGCCCACCGTGCACGTGGTGAGAGAGGTGTTCCCCGCCCTGCTGAGCCTGGGCGAGGAGCTGGAGAGAATGCACCCCAGAGTGTACACCAACGTGGCCAGACAGATCAGCAGAAGCCCCAGAGGCGAGCTGAAGGAGCCCGACACCGCCCCCGTGCTGCTGAGCGCCATCGCCAGAGACCTGTTCAGAAGCGACATCACCTGGGGCAAGGTGGTGAGCCTGTTCGCCATCGCCGGCGGCCTGGCCGTGGACTGCGTGAGACAGGGCCACCCCGACTACCTGCAGAAGCTGATCGAGGGCGTGGCCGACGTGATCGAGGACGACCTGGCCACCTGGATCGCCGAGAACGGCGGCTGGGTGGGCCTGGCCAACCACGTGAGACCCCCCCAGGAGGAGATCACCTTCGTGGGCAGATGCCTGGGCCTGCTGGCCCTGTTCATGGCCGTGAAGTTCCTGGTGTACCTGATCCTGCAGTGGCTGCAGAAGCTGGAGAGCCAGTACGACAGCCAGCTGTAA.To generate synr mutant constructs for BPA photo‐crosslinking assay, a codon of the target amino acids was changed to the amber codon TAG in the GST‐synr plasmid by PCR with Primestar Mutagenesis Basal kit (Takara #R046A).Protein preparationGST proteins (GST only, GST‐Synr, GST‐Synr ΔBH3, and GST‐Synr Δccd) were expressed in BL21 E. coli (Takara #9126) and purified by affinity for glutathione Sepharose (GE healthcare). After preculture for 4 h, we applied a cold shock according to the manufacturer's instruction and incubated in LB media with 0.1 mM IPTG at 15°C for 20 h. Cell pellets were isolated by centrifugation and lysed by sonication in the E. coli lysis buffer (PBS, 1% NP40, 250 μM PMSF). GST proteins were purified by affinity for glutathione Sepharose. After washing beads, lysates that contain Flag‐tagged proteins or HA‐tagged proteins were added. To make the lysates, HEK293T cells were lysed in the cell lysis buffer (pH 7.4 50 mM Hepes, 2% NP40, 150 mM NaCl, 5% Glycerol, 1 mM MgCl2, 1 mM MnCl2, 10 mM NaF, 1 mM Na3VO4, 10 μg/ml Leupeptin, 2 μg/ml Pepstatin, 0.1% Aprotinin, 1% Protease inhibitor cocktail (Roche, cOmplete, EDTA free), 0.1% Phosphatase inhibitor cocktail 2 (P5726, Sigma), 0.1% Phosphate inhibitor cocktail 3 (P0044, Sigma), 250 μM PMSF and 1 mM DTT), and cell lysates were cleaned by centrifugation. After washing the GST‐bound beads with the cell lysis buffer, proteins were eluted by heating at 95°C for 5 min in the 2× sample buffer and they were separated by SDS–PAGE (8% SDS–PAGE gels).Protein preparation for photo‐crosslinking assayPlasmids for GST proteins and pEVOL‐pBpF (#31190), which includes tRNA synthetase/tRNA pair to incorporate pBPA into proteins, were co‐transformed to pG‐Tf2/BL21 E. coli (Takara #9124), and they were cultured LB medium with 1 mM p‐benzoyl‐L‐phenylalanine and 0.1% L‐arabinose. Then, cell lysis, GST purification, and incubation with lysates that contain Flag‐tagged protein or HA‐tagged protein were performed as described previously. Lysates were cleaned by centrifugation, and the GST‐bound beads were exposed to UV for 15 min. Washing and elution were performed as described previously. Proteins were separated by SDS–PAGE (6% SDS–PAGE gels).Sequential IP
All of Myc‐Synr, HA‐Debcl, and Flag‐Buffy were expressed in HEK293T cells. Their lysates were incubated with the monoclonal mouse ANTI‐FLAG M2 antibody (F1804, Sigma) and Protein A/G Magnetic Beads (#88802, Thermo Fisher) for 4 h. After washing beads, proteins were eluted with 200 μg/ml 3XFlag peptide and then elution was again incubated with new Protein A/G Magnetic Beads and Myc‐tag mouse antibody (9B11, #2276, Cell Signaling Technology) O/N. After washing the beads with the cell lysis buffer, proteins were eluted by heating at 95°C for 5 min in the 2× sample buffer and they were separated by SDS–PAGE (8% SDS–PAGE gels).Western blotting analysisProteins were separated on an SDS–PAGE gel and transferred to a nitrocellulose membrane (Bio‐Rad Laboratories, Inc.). After blocking with 5% BSA, the membranes were immunoblotted with the following antibodies and fluorescent dyes at the indicated dilutions: GST rabbit antibody (1:1,000; 2622S, Cell Signaling Technology, or 1:1,000; 10000‐0‐AP, Proteintech), Monoclonal mouse ANTI‐FLAG M2 antibody (1:1,000; F1804, Sigma), HA‐Tag Rabbit antibody (1:1,000; 3724, Cell Signaling Technology), Anti‐HA.11 Epitope Tag mouse antibody (1:1,000; 901513, BioLegend), Myc‐tag mouse antibody (1:1,000, #2276, Cell Signaling Technology), IRDye® 680RD Goat anti‐Rabbit IgG Secondary Antibody (1:10,000; 925‐68071, LI‐COR), and IRDye® 800CW Goat anti‐Mouse IgG Secondary Antibody (1:10,000; 925‐32210, LI‐COR). Bands were visualized with a near‐infrared fluorescence imaging scanner (LI‐COR, Odyssey CLx). The signal intensity was measured with Image Studio from LI‐COR.Terminal deoxynucleotidyl transferase dUTP nick end labeling (TUNEL) assayThe TUNEL assay was performed using the ApopTag Red in situ apoptosis detection kit (Millipore) as described previously (Ciesielski et al, 2022). Wandering 3rd instar larvae were dissected in 1×PBS and fixed for 30 min in 1×PBS with 4% paraformaldehyde at room temperature (RT). After fixation, samples were washed with PBS 0.1% Triton‐X100 and incubated in equilibration buffer (Apop Tag kit; Millipore) for 10 s. Then, samples were incubated in reaction buffer (TdT enzyme; ratio 7:3; Apop Tag kit) at 37°C for 1 h. The TdT reaction mix was replaced with stop buffer (diluted 1:34 in dH2O; Apop Tag kit) and incubated for 10 min at RT. Samples were then washed with PBS 0.1% Triton‐X100 3 times and incubated with antidigoxigenin antibody solution (diluted 31:34 in blocking solution; ApopTag kit) overnight at 4°C. Samples were then washed with PBS 0.1% Triton‐X100 3 times again and mounted on the slide glass.Lysotracker stainingWandering 3rd instar larvae were dissected in PBS and incubated with 50 μM LysoTracker Red DND‐99 (L7528, Invitrogen) in PBS for 5 min. After washing with PBS, samples were fixed with 4% PFA in PBS for 20 min. Then, samples were washed once with PBSTx and incubated with PBSTx containing DAPI (4 μg/ml) for 5 min. After washing with PBSTx, we observed the samples immediately with a confocal microscope.
PI stainingWing discs in L3 larvae were dissected in PBS, incubated with Propidium Iodide (100 nM, 341‐07881, Fujifilm) in PBS immediately after dissection, then fixed with 4% PFA in PBS, and washed in PBSTx.Starvation assayFor both measuring the starvation resistance and caspase activation, 5–7‐day‐old virgin female flies were used. The starvation food contained only 1.5% agar. To measure starvation resistance, we counted the number of dead flies every day. For the analysis of caspase activation, we specifically induced GC3Ai expression in ECs with 5966‐GS. We added 100 μM RU486 (M1732, TCI) to both the normal food and the starvation food.Genotyping for CG14044MI09290

To confirm CG14044MI09290 has a MIMIC insertion, we designed the primers to detect 500 bp surrounding the MIMIC insertion site. Since MIMIC is too large to amplify, the PCR does not work in CG14044MI09290.Quantification of developmental timingTo quantify developmental timing, we calculated the pupariation rate. Mated females were allowed to lay eggs on a grape agar plate for 8 h at 25°C. First, instar larvae were collected from the grape agar plate and placed into the normal food. Each vial contained 50 larvae. The number of new pupae of each genotype was recorded every 12 h. Pupariation rates were calculated as the number of pupariated flies divided by the total number of pupae at the end of each experiment.Sayonara orthologue identification, alignment, and phylogenetic reconstructionTo infer the evolutionary history of Sayonara, we first queried Panther (16.0) to identify gene family members related to CG14044. We found that CG14044 belonged to Panther family PTHR21974. As our aim was to characterize the emergence of a BH3 motif in the Sayonara orthogroup rather than to build a comprehensive phylogeny with remote homologs, we restricted our analysis to nonredundant sequences from Ecdysozoa. Among 816 hits matching Panther family PTHR21974, we extracted 109 protein sequences from which 69 were selected for further analysis. Manual curation was performed to exclude identical, nearly identical or partial sequences as well as sequences from overrepresented taxa (e.g., Drosophila). BLAST searches against NCBI and UniProtKB databases were conducted one‐by‐one to find additional sequences in taxa of interest leading to a final dataset containing 41 representative sequences. Sequence E0W198 from Pediculus humanus (Psocodea) was unusual and excluded from subsequent analysis in that it has a significantly truncated C terminus. Sequences were aligned using MUSCLE, and the alignment was edited manually at ambiguous sites and at N‐ and C‐terminal ends. A neighbor‐joining tree was constructed using the BioNJ algorithm as implemented in SeaView (version 5) and Poisson distances (gaps included), with 1,000 bootstrap replicates. The tree was rooted using CIL‐7 sequences from Caenorhabditis. The topology of our final tree is in accordance with previously published general phylogenies of insects and Diptera (Misof et al, 2014).Profile‐based methods for domain composition and BH3 motif predictionSayonara domain composition was assessed using HMM profile searches with the HMMER package (v3.0 March 2010). Briefly, we searched the UniProtKB 2021_03 database to retrieve homologs of fly CG14044 using the BLASTp algorithm, and the new sequences (obtained with E‐value < 1e−6, length ≥ 135 or 341 residues) were aligned using Clustal W 1.8 to construct two HMM profiles with hmmbuild. The full‐length sequences (n = 52) of Sayonara (378 residues in Drosophila melanogaster) or the first 150 amino acids (present in n = 41 sequences) were used as inputs to build a first profile for the whole Sayonara orthogroup and a second profile specific for the N‐terminal variable domain. These two profiles were searched against the UniProtKB 2021_03 sequence database by hmmsearch, resulting in 215 and 60 hits, respectively (E‐value ≤ 0.001). Venn diagram analysis indicated that 60 sequences were common to both profiles, 155 sequences were exclusive to the first profile (full‐length sequences), and no sequence was exclusive to the second profile (N‐terminal domain). We predicted the presence or absence of BH3 motifs in the N‐terminal domain of Sayonara orthologous sequences using evolutionary information provided by a position‐specific scoring matrix (PSSM) profile. As previously described (Rech de Laval et al, 2014; Aouacheria et al, 2019), a PSSM profile of BH3 motifs was computed as part of an ab initio motif discovery procedure by running the meme program of the MEME software suite 4.9.0 using a set of 158 amino acid sequences of BCL‐2 homologous proteins and BH3‐only proteins. The PSSM was used to scan the set of 41 selected sequences for BH3 motif occurrences using the program mast from the MEME suite.Characterization of the 
Buffy–Debcl
 duplication eventA total of 224 amino acid sequences (present in 201 different species) showing significative similarity to Drosophila Buffy and Debcl were collected from translated insect transcriptomes produced by the 1KITE (1K Insect Transcriptome Evolution) project (Misof et al, 2014). A profile HMM specific to both paralogous protein sequences was built and then used to screen the 1KITE transcriptome assemblies with the program hmmsearch (after translating the transcripts into all six possible reading frames). Only hits with a global e‐value ≤ 10−5 and that were predicted to contain a BH motif (using Batch CD‐search) were considered for subsequent analysis. Buffy and Debcl protein sequences were also searched against the nonredundant protein (NR) and Reference Sequence (RefSeq) databases using BLASTP, leading to an additional set of 39 sequences (in 25 species). All 263 sequences were aligned with MUSCLE and trimmed with GBLOCKS. The phylogenetic tree was reconstructed using the UL3 amino acid substitution model with PhyML‐Structure (Guindon & Gascuel, 2003; Le & Gascuel, 2010) and rooted using Collembola as the outgroup.Ancestral sequence reconstruction and experimental resurrectionAncestral sequence reconstruction (ASR) was carried out as described previously (Groussin et al, 2015), accounting for the uncertainty in the reconstruction of the gene tree and gene duplication and loss. We used Phyldog (Boussau et al, 2013) to reconcile the gene tree with the species tree. We used 18S ribosomal RNA sequences, when available in the NR database (32 sequences), to reconstruct the species tree. The reconciled species tree‐aware gene tree was used as the guiding tree in Prank to realign Buffy–Debcl sequences. We used bppML (Dutheil & Boussau, 2008) to compare the capacity of different site‐homogeneous and site‐heterogenous models substitution models to fit the data and found that the site‐heterogenous EX_EHO model was best at fitting the data according to the BIC criterion. We then performed ASR with the ML parameter estimates obtained with the EX_EHO model, using bppAncestor (Dutheil & Boussau, 2008) with the marginal ASR approach (Yang et al, 1995b). For a given site at a given internal node of the reconciled tree, the amino acid having the maximum posterior probability was inferred as the putative ancestral amino acid. Gap positions in ancestral sequences were introduced a posteriori based on inferences performed by Prank during sequence alignment. The putative Buffy–Debcl ancestor was named “Borg0” after dBorg‐1 (one of the synonyms for Debcl) and dBorg‐2 (Buffy).Statistical analysisStatistical tests used are indicated in the figure captions. Sample sizes were determined empirically based on the observed effects. Blinding was not applicable because the investigators who set up the experiments were the same with the ones doing sample collection and analyses. However, each experiment was associated with proper controls, and samples were collected and analyzed under identical conditions. All the statistical analyses were conducted in GraphPad Prism.

Materials and Methods

Drosophila husbandryFlies were maintained as described previously (Yoo et al, 2016). The fly food was composed of 0.8% agar, 10% glucose, 4.5% corn flour, 3.72% dry yeast, 0.4% propionic acid, and 0.3% butyl p‐hydroxybenzoate. All experiments were performed at 25°C except Figs 4A and B, and EV3B, which were conducted at 18°C.

Drosophila husbandry

Flies were maintained as described previously (Yoo et al, 2016). The fly food was composed of 0.8% agar, 10% glucose, 4.5% corn flour, 3.72% dry yeast, 0.4% propionic acid, and 0.3% butyl p‐hydroxybenzoate. All experiments were performed at 25°C except Figs 4A and B, and EV3B, which were conducted at 18°C.

Drosophila stocksThe fly stocks used in this study are listed in Dataset EV1. We also generated the following fly stocks by combining the listed stocks:

nub‐gal4, UAS‐GFP; UAS‐synr

nub‐gal4; UAS‐GC3Ai

nub‐gal4; UAS‐synr

nub‐gal4, UAS‐GFP; act‐mCherry‐Atg8a

vg‐gal4, UAS‐GFP; UAS‐rpr

nub‐gal4; UAS‐p53

5966GS‐gal4; UAS‐GC3Ai

synr
−/− was made by extensive outcross of CG14044

MI09290
 (50506, Bloomington Stock Center) to Oregon R. The MiMIC insertion was confirmed by PCR.

Drosophila stocks

The fly stocks used in this study are listed in Dataset EV1. We also generated the following fly stocks by combining the listed stocks:

nub‐gal4, UAS‐GFP; UAS‐synr

nub‐gal4; UAS‐GC3Ai

nub‐gal4; UAS‐synr

nub‐gal4, UAS‐GFP; act‐mCherry‐Atg8a

vg‐gal4, UAS‐GFP; UAS‐rpr

nub‐gal4; UAS‐p53

5966GS‐gal4; UAS‐GC3Ai

synr
−/− was made by extensive outcross of CG14044

MI09290
 (50506, Bloomington Stock Center) to Oregon R. The MiMIC insertion was confirmed by PCR.

nub‐gal4, UAS‐GFP; UAS‐synr

nub‐gal4; UAS‐GC3Ai

nub‐gal4; UAS‐synr

nub‐gal4, UAS‐GFP; act‐mCherry‐Atg8a

vg‐gal4, UAS‐GFP; UAS‐rpr

nub‐gal4; UAS‐p53

5966GS‐gal4; UAS‐GC3Ai

Immunofluorescence and imagingWing discs or adult midguts were immunostained as described previously (Nishida et al, 2021; Sasaki et al, 2021). We used the following antibodies and fluorescent dyes at the indicated dilutions: cleaved Dcp1 antibody (1:500; 9578, Cell Signaling Technology), HA antibody (1:500; 901513, BAB), DAPI (1:500; D9542, Sigma), and Alexa Fluor secondary antibodies (1:500; A11008 and A11036, Thermo Fisher). Fluorescence images were acquired with a confocal microscope (Zeiss LSM 780, 880, 900). To measure the wing size, wings were imaged with a stereo microscope (Nikon SMZ18). The wing size was quantified using Fiji. To quantify the percentage of GC3Ai‐positive areas, images were binarized with a certain intensity threshold with Fiji, and the percentage was calculated to the area with gene expression, which was determined by the GC3Ai background signal and morphology.

Immunofluorescence and imaging

Wing discs or adult midguts were immunostained as described previously (Nishida et al, 2021; Sasaki et al, 2021). We used the following antibodies and fluorescent dyes at the indicated dilutions: cleaved Dcp1 antibody (1:500; 9578, Cell Signaling Technology), HA antibody (1:500; 901513, BAB), DAPI (1:500; D9542, Sigma), and Alexa Fluor secondary antibodies (1:500; A11008 and A11036, Thermo Fisher). Fluorescence images were acquired with a confocal microscope (Zeiss LSM 780, 880, 900). To measure the wing size, wings were imaged with a stereo microscope (Nikon SMZ18). The wing size was quantified using Fiji. To quantify the percentage of GC3Ai‐positive areas, images were binarized with a certain intensity threshold with Fiji, and the percentage was calculated to the area with gene expression, which was determined by the GC3Ai background signal and morphology.

DNA constructionFor making transgenic flies, synr WT (FBpp0078629), synr BH3 motif deletion mutant, synr ccd deletion mutant, synr BH3 motif amino acid change mutant, or synr ccd amino acid change mutant was cloned into the pUASTattB vector. synr BH3 deletion mutant was made by overlap PCR to remove the BH3 motif (residues 122–131). synr coiled‐coil domain mutants were also generated similarly. We removed residues 162–182 for synr dcc1, residues 199–219 for synr dcc2, and residues 162–219 for synr dcc1‐2. Regarding AA substitution mutants, BH3 motif AA mutant was generated by overlap PCR. The ccd AA mutant was newly synthesized by GenScript.The amino acid sequences of BH3 motif AA mutant and ccd AA mutant are the following:
BH3 motif AA mutant, EAYNEGVEAAAEK (122–134); ccd AA mutant, ccd1, GGGTGGGGGTGGGSYGGGNGG (162–182), ccd2, GTGGGGSGGGQGGTGGGTGGG (199–219)
For HA‐tagged Synr, synr WT, ΔBH3, and BH3AA mutants were once inserted into the pCMV‐HA‐N vector (Takara #635690). Then, the tagged synr constructs were amplified by PCR and cloned into the pUASTattB vector. To knockdown synr, we generated shRNAi lines based on the Trip protocol by Jian‐Quan Ni and Norbert Perrimon at Harvard Medical School (https://fgr.hms.harvard.edu/files/fly/files/2ndgenprotocol.pdf). The following oligos were annealed and cloned into the pVALIUM20 vector.
synr ‐RNAi:
ctagcagtTCGGCAAGTCCTAGCCGTCAAtagttatattcaagcataTTGACGGCTAGGACTTGCCGAgcgaattcgcTCGGCAAGTCCTAGCCGTCAAtatgcttgaatataactaTTGACGGCTAGGACTTGCCGAactg

synr ‐RNAi‐2:
ctagcagtAACGAGCTGTACGTAAAGCAAtagttatattcaagcataTTGCTTTACGTACAGCTCGTTgcgaattcgcAACGAGCTGTACGTAAAGCAAtatgcttgaatataactaTTGCTTTACGTACAGCTCGTTactg
Transgenic flies were generated by BestGene.For protein expression in E. coli, synr WT, ΔBH3, and Δccd1‐2 mutants were cloned into a pCold‐GST DNA (Takara #3372) with an N‐terminal GST tag.For protein expression in HEK293T cells, both Buffy (FBpp0087182) and Debcl (FBpp0085443) were cloned into a Flag vector (a gift from Dr Nishimura), where the Flag tag was added to the N terminus. We also generated HA‐Debcl with pCMV‐HA‐N vector (Takara #635690). Myc tag was also added at the N terminus of synr with pCMV‐Myc vector (Takara #635689). Regarding the dBorg0 experiment, based on the amino acid sequence of dBorg0 (Fig EV4B), the following DNA sequence was synthesized with the human codon usage and cloned into the Flag vector (GenScript): ATGCTGAGCACCACCGCCCCCCCCACCACCACCACCACCGCCAGCAGCAACCCCAGCCCCAACAGCAACAGCGTGAGCCCCAGCAGAAACCTGGCCAACCTGCACAGCAGCCCCCTGAACGGCGACCACCACCCCCCCAGCAGCAGAAGAAGACTGAGCAACGTGAGCGACGTGGTGACCAGAAAGCTGAGCAGCACCATCGGCTGGAAGCAGGCCGTGATCCCCAGCCAGGACATCATCACCCAGGGCAAGTGCCTGTGCGGCCAGTACATCAGAAGCAGACTGAAGAGAAGCGGCGTGTTCAACAGAAAGCTGGGCCTGCAGAGACTGAGAAGCATCGTGGGCGTGCCCACCGTGCACGTGGTGAGAGAGGTGTTCCCCGCCCTGCTGAGCCTGGGCGAGGAGCTGGAGAGAATGCACCCCAGAGTGTACACCAACGTGGCCAGACAGATCAGCAGAAGCCCCAGAGGCGAGCTGAAGGAGCCCGACACCGCCCCCGTGCTGCTGAGCGCCATCGCCAGAGACCTGTTCAGAAGCGACATCACCTGGGGCAAGGTGGTGAGCCTGTTCGCCATCGCCGGCGGCCTGGCCGTGGACTGCGTGAGACAGGGCCACCCCGACTACCTGCAGAAGCTGATCGAGGGCGTGGCCGACGTGATCGAGGACGACCTGGCCACCTGGATCGCCGAGAACGGCGGCTGGGTGGGCCTGGCCAACCACGTGAGACCCCCCCAGGAGGAGATCACCTTCGTGGGCAGATGCCTGGGCCTGCTGGCCCTGTTCATGGCCGTGAAGTTCCTGGTGTACCTGATCCTGCAGTGGCTGCAGAAGCTGGAGAGCCAGTACGACAGCCAGCTGTAA.To generate synr mutant constructs for BPA photo‐crosslinking assay, a codon of the target amino acids was changed to the amber codon TAG in the GST‐synr plasmid by PCR with Primestar Mutagenesis Basal kit (Takara #R046A).

DNA construction

For making transgenic flies, synr WT (FBpp0078629), synr BH3 motif deletion mutant, synr ccd deletion mutant, synr BH3 motif amino acid change mutant, or synr ccd amino acid change mutant was cloned into the pUASTattB vector. synr BH3 deletion mutant was made by overlap PCR to remove the BH3 motif (residues 122–131). synr coiled‐coil domain mutants were also generated similarly. We removed residues 162–182 for synr dcc1, residues 199–219 for synr dcc2, and residues 162–219 for synr dcc1‐2. Regarding AA substitution mutants, BH3 motif AA mutant was generated by overlap PCR. The ccd AA mutant was newly synthesized by GenScript.

The amino acid sequences of BH3 motif AA mutant and ccd AA mutant are the following:
BH3 motif AA mutant, EAYNEGVEAAAEK (122–134); ccd AA mutant, ccd1, GGGTGGGGGTGGGSYGGGNGG (162–182), ccd2, GTGGGGSGGGQGGTGGGTGGG (199–219)

BH3 motif AA mutant, EAYNEGVEAAAEK (122–134); ccd AA mutant, ccd1, GGGTGGGGGTGGGSYGGGNGG (162–182), ccd2, GTGGGGSGGGQGGTGGGTGGG (199–219)

For HA‐tagged Synr, synr WT, ΔBH3, and BH3AA mutants were once inserted into the pCMV‐HA‐N vector (Takara #635690). Then, the tagged synr constructs were amplified by PCR and cloned into the pUASTattB vector. To knockdown synr, we generated shRNAi lines based on the Trip protocol by Jian‐Quan Ni and Norbert Perrimon at Harvard Medical School (https://fgr.hms.harvard.edu/files/fly/files/2ndgenprotocol.pdf). The following oligos were annealed and cloned into the pVALIUM20 vector.

synr ‐RNAi:
ctagcagtTCGGCAAGTCCTAGCCGTCAAtagttatattcaagcataTTGACGGCTAGGACTTGCCGAgcgaattcgcTCGGCAAGTCCTAGCCGTCAAtatgcttgaatataactaTTGACGGCTAGGACTTGCCGAactg

ctagcagtTCGGCAAGTCCTAGCCGTCAAtagttatattcaagcataTTGACGGCTAGGACTTGCCGAgcg

aattcgcTCGGCAAGTCCTAGCCGTCAAtatgcttgaatataactaTTGACGGCTAGGACTTGCCGAactg

synr ‐RNAi‐2:
ctagcagtAACGAGCTGTACGTAAAGCAAtagttatattcaagcataTTGCTTTACGTACAGCTCGTTgcgaattcgcAACGAGCTGTACGTAAAGCAAtatgcttgaatataactaTTGCTTTACGTACAGCTCGTTactg

ctagcagtAACGAGCTGTACGTAAAGCAAtagttatattcaagcataTTGCTTTACGTACAGCTCGTTgcg

aattcgcAACGAGCTGTACGTAAAGCAAtatgcttgaatataactaTTGCTTTACGTACAGCTCGTTactg

Transgenic flies were generated by BestGene.

For protein expression in E. coli, synr WT, ΔBH3, and Δccd1‐2 mutants were cloned into a pCold‐GST DNA (Takara #3372) with an N‐terminal GST tag.

For protein expression in HEK293T cells, both Buffy (FBpp0087182) and Debcl (FBpp0085443) were cloned into a Flag vector (a gift from Dr Nishimura), where the Flag tag was added to the N terminus. We also generated HA‐Debcl with pCMV‐HA‐N vector (Takara #635690). Myc tag was also added at the N terminus of synr with pCMV‐Myc vector (Takara #635689). Regarding the dBorg0 experiment, based on the amino acid sequence of dBorg0 (Fig EV4B), the following DNA sequence was synthesized with the human codon usage and cloned into the Flag vector (GenScript): ATGCTGAGCACCACCGCCCCCCCCACCACCACCACCACCGCCAGCAGCAACCCCAGCCCCAACAGCAACAGCGTGAGCCCCAGCAGAAACCTGGCCAACCTGCACAGCAGCCCCCTGAACGGCGACCACCACCCCCCCAGCAGCAGAAGAAGACTGAGCAACGTGAGCGACGTGGTGACCAGAAAGCTGAGCAGCACCATCGGCTGGAAGCAGGCCGTGATCCCCAGCCAGGACATCATCACCCAGGGCAAGTGCCTGTGCGGCCAGTACATCAGAAGCAGACTGAAGAGAAGCGGCGTGTTCAACAGAAAGCTGGGCCTGCAGAGACTGAGAAGCATCGTGGGCGTGCCCACCGTGCACGTGGTGAGAGAGGTGTTCCCCGCCCTGCTGAGCCTGGGCGAGGAGCTGGAGAGAATGCACCCCAGAGTGTACACCAACGTGGCCAGACAGATCAGCAGAAGCCCCAGAGGCGAGCTGAAGGAGCCCGACACCGCCCCCGTGCTGCTGAGCGCCATCGCCAGAGACCTGTTCAGAAGCGACATCACCTGGGGCAAGGTGGTGAGCCTGTTCGCCATCGCCGGCGGCCTGGCCGTGGACTGCGTGAGACAGGGCCACCCCGACTACCTGCAGAAGCTGATCGAGGGCGTGGCCGACGTGATCGAGGACGACCTGGCCACCTGGATCGCCGAGAACGGCGGCTGGGTGGGCCTGGCCAACCACGTGAGACCCCCCCAGGAGGAGATCACCTTCGTGGGCAGATGCCTGGGCCTGCTGGCCCTGTTCATGGCCGTGAAGTTCCTGGTGTACCTGATCCTGCAGTGGCTGCAGAAGCTGGAGAGCCAGTACGACAGCCAGCTGTAA.

To generate synr mutant constructs for BPA photo‐crosslinking assay, a codon of the target amino acids was changed to the amber codon TAG in the GST‐synr plasmid by PCR with Primestar Mutagenesis Basal kit (Takara #R046A).

Protein preparationGST proteins (GST only, GST‐Synr, GST‐Synr ΔBH3, and GST‐Synr Δccd) were expressed in BL21 E. coli (Takara #9126) and purified by affinity for glutathione Sepharose (GE healthcare). After preculture for 4 h, we applied a cold shock according to the manufacturer's instruction and incubated in LB media with 0.1 mM IPTG at 15°C for 20 h. Cell pellets were isolated by centrifugation and lysed by sonication in the E. coli lysis buffer (PBS, 1% NP40, 250 μM PMSF). GST proteins were purified by affinity for glutathione Sepharose. After washing beads, lysates that contain Flag‐tagged proteins or HA‐tagged proteins were added. To make the lysates, HEK293T cells were lysed in the cell lysis buffer (pH 7.4 50 mM Hepes, 2% NP40, 150 mM NaCl, 5% Glycerol, 1 mM MgCl2, 1 mM MnCl2, 10 mM NaF, 1 mM Na3VO4, 10 μg/ml Leupeptin, 2 μg/ml Pepstatin, 0.1% Aprotinin, 1% Protease inhibitor cocktail (Roche, cOmplete, EDTA free), 0.1% Phosphatase inhibitor cocktail 2 (P5726, Sigma), 0.1% Phosphate inhibitor cocktail 3 (P0044, Sigma), 250 μM PMSF and 1 mM DTT), and cell lysates were cleaned by centrifugation. After washing the GST‐bound beads with the cell lysis buffer, proteins were eluted by heating at 95°C for 5 min in the 2× sample buffer and they were separated by SDS–PAGE (8% SDS–PAGE gels).

Protein preparation

GST proteins (GST only, GST‐Synr, GST‐Synr ΔBH3, and GST‐Synr Δccd) were expressed in BL21 E. coli (Takara #9126) and purified by affinity for glutathione Sepharose (GE healthcare). After preculture for 4 h, we applied a cold shock according to the manufacturer's instruction and incubated in LB media with 0.1 mM IPTG at 15°C for 20 h. Cell pellets were isolated by centrifugation and lysed by sonication in the E. coli lysis buffer (PBS, 1% NP40, 250 μM PMSF). GST proteins were purified by affinity for glutathione Sepharose. After washing beads, lysates that contain Flag‐tagged proteins or HA‐tagged proteins were added. To make the lysates, HEK293T cells were lysed in the cell lysis buffer (pH 7.4 50 mM Hepes, 2% NP40, 150 mM NaCl, 5% Glycerol, 1 mM MgCl2, 1 mM MnCl2, 10 mM NaF, 1 mM Na3VO4, 10 μg/ml Leupeptin, 2 μg/ml Pepstatin, 0.1% Aprotinin, 1% Protease inhibitor cocktail (Roche, cOmplete, EDTA free), 0.1% Phosphatase inhibitor cocktail 2 (P5726, Sigma), 0.1% Phosphate inhibitor cocktail 3 (P0044, Sigma), 250 μM PMSF and 1 mM DTT), and cell lysates were cleaned by centrifugation. After washing the GST‐bound beads with the cell lysis buffer, proteins were eluted by heating at 95°C for 5 min in the 2× sample buffer and they were separated by SDS–PAGE (8% SDS–PAGE gels).

Protein preparation for photo‐crosslinking assayPlasmids for GST proteins and pEVOL‐pBpF (#31190), which includes tRNA synthetase/tRNA pair to incorporate pBPA into proteins, were co‐transformed to pG‐Tf2/BL21 E. coli (Takara #9124), and they were cultured LB medium with 1 mM p‐benzoyl‐L‐phenylalanine and 0.1% L‐arabinose. Then, cell lysis, GST purification, and incubation with lysates that contain Flag‐tagged protein or HA‐tagged protein were performed as described previously. Lysates were cleaned by centrifugation, and the GST‐bound beads were exposed to UV for 15 min. Washing and elution were performed as described previously. Proteins were separated by SDS–PAGE (6% SDS–PAGE gels).

Protein preparation for photo‐crosslinking assay

Plasmids for GST proteins and pEVOL‐pBpF (#31190), which includes tRNA synthetase/tRNA pair to incorporate pBPA into proteins, were co‐transformed to pG‐Tf2/BL21 E. coli (Takara #9124), and they were cultured LB medium with 1 mM p‐benzoyl‐L‐phenylalanine and 0.1% L‐arabinose. Then, cell lysis, GST purification, and incubation with lysates that contain Flag‐tagged protein or HA‐tagged protein were performed as described previously. Lysates were cleaned by centrifugation, and the GST‐bound beads were exposed to UV for 15 min. Washing and elution were performed as described previously. Proteins were separated by SDS–PAGE (6% SDS–PAGE gels).

Sequential IP
All of Myc‐Synr, HA‐Debcl, and Flag‐Buffy were expressed in HEK293T cells. Their lysates were incubated with the monoclonal mouse ANTI‐FLAG M2 antibody (F1804, Sigma) and Protein A/G Magnetic Beads (#88802, Thermo Fisher) for 4 h. After washing beads, proteins were eluted with 200 μg/ml 3XFlag peptide and then elution was again incubated with new Protein A/G Magnetic Beads and Myc‐tag mouse antibody (9B11, #2276, Cell Signaling Technology) O/N. After washing the beads with the cell lysis buffer, proteins were eluted by heating at 95°C for 5 min in the 2× sample buffer and they were separated by SDS–PAGE (8% SDS–PAGE gels).

Sequential IP

All of Myc‐Synr, HA‐Debcl, and Flag‐Buffy were expressed in HEK293T cells. Their lysates were incubated with the monoclonal mouse ANTI‐FLAG M2 antibody (F1804, Sigma) and Protein A/G Magnetic Beads (#88802, Thermo Fisher) for 4 h. After washing beads, proteins were eluted with 200 μg/ml 3XFlag peptide and then elution was again incubated with new Protein A/G Magnetic Beads and Myc‐tag mouse antibody (9B11, #2276, Cell Signaling Technology) O/N. After washing the beads with the cell lysis buffer, proteins were eluted by heating at 95°C for 5 min in the 2× sample buffer and they were separated by SDS–PAGE (8% SDS–PAGE gels).

Western blotting analysisProteins were separated on an SDS–PAGE gel and transferred to a nitrocellulose membrane (Bio‐Rad Laboratories, Inc.). After blocking with 5% BSA, the membranes were immunoblotted with the following antibodies and fluorescent dyes at the indicated dilutions: GST rabbit antibody (1:1,000; 2622S, Cell Signaling Technology, or 1:1,000; 10000‐0‐AP, Proteintech), Monoclonal mouse ANTI‐FLAG M2 antibody (1:1,000; F1804, Sigma), HA‐Tag Rabbit antibody (1:1,000; 3724, Cell Signaling Technology), Anti‐HA.11 Epitope Tag mouse antibody (1:1,000; 901513, BioLegend), Myc‐tag mouse antibody (1:1,000, #2276, Cell Signaling Technology), IRDye® 680RD Goat anti‐Rabbit IgG Secondary Antibody (1:10,000; 925‐68071, LI‐COR), and IRDye® 800CW Goat anti‐Mouse IgG Secondary Antibody (1:10,000; 925‐32210, LI‐COR). Bands were visualized with a near‐infrared fluorescence imaging scanner (LI‐COR, Odyssey CLx). The signal intensity was measured with Image Studio from LI‐COR.

Western blotting analysis

Proteins were separated on an SDS–PAGE gel and transferred to a nitrocellulose membrane (Bio‐Rad Laboratories, Inc.). After blocking with 5% BSA, the membranes were immunoblotted with the following antibodies and fluorescent dyes at the indicated dilutions: GST rabbit antibody (1:1,000; 2622S, Cell Signaling Technology, or 1:1,000; 10000‐0‐AP, Proteintech), Monoclonal mouse ANTI‐FLAG M2 antibody (1:1,000; F1804, Sigma), HA‐Tag Rabbit antibody (1:1,000; 3724, Cell Signaling Technology), Anti‐HA.11 Epitope Tag mouse antibody (1:1,000; 901513, BioLegend), Myc‐tag mouse antibody (1:1,000, #2276, Cell Signaling Technology), IRDye® 680RD Goat anti‐Rabbit IgG Secondary Antibody (1:10,000; 925‐68071, LI‐COR), and IRDye® 800CW Goat anti‐Mouse IgG Secondary Antibody (1:10,000; 925‐32210, LI‐COR). Bands were visualized with a near‐infrared fluorescence imaging scanner (LI‐COR, Odyssey CLx). The signal intensity was measured with Image Studio from LI‐COR.

Terminal deoxynucleotidyl transferase dUTP nick end labeling (TUNEL) assayThe TUNEL assay was performed using the ApopTag Red in situ apoptosis detection kit (Millipore) as described previously (Ciesielski et al, 2022). Wandering 3rd instar larvae were dissected in 1×PBS and fixed for 30 min in 1×PBS with 4% paraformaldehyde at room temperature (RT). After fixation, samples were washed with PBS 0.1% Triton‐X100 and incubated in equilibration buffer (Apop Tag kit; Millipore) for 10 s. Then, samples were incubated in reaction buffer (TdT enzyme; ratio 7:3; Apop Tag kit) at 37°C for 1 h. The TdT reaction mix was replaced with stop buffer (diluted 1:34 in dH2O; Apop Tag kit) and incubated for 10 min at RT. Samples were then washed with PBS 0.1% Triton‐X100 3 times and incubated with antidigoxigenin antibody solution (diluted 31:34 in blocking solution; ApopTag kit) overnight at 4°C. Samples were then washed with PBS 0.1% Triton‐X100 3 times again and mounted on the slide glass.

Terminal deoxynucleotidyl transferase dUTP nick end labeling (TUNEL) assay

The TUNEL assay was performed using the ApopTag Red in situ apoptosis detection kit (Millipore) as described previously (Ciesielski et al, 2022). Wandering 3rd instar larvae were dissected in 1×PBS and fixed for 30 min in 1×PBS with 4% paraformaldehyde at room temperature (RT). After fixation, samples were washed with PBS 0.1% Triton‐X100 and incubated in equilibration buffer (Apop Tag kit; Millipore) for 10 s. Then, samples were incubated in reaction buffer (TdT enzyme; ratio 7:3; Apop Tag kit) at 37°C for 1 h. The TdT reaction mix was replaced with stop buffer (diluted 1:34 in dH2O; Apop Tag kit) and incubated for 10 min at RT. Samples were then washed with PBS 0.1% Triton‐X100 3 times and incubated with antidigoxigenin antibody solution (diluted 31:34 in blocking solution; ApopTag kit) overnight at 4°C. Samples were then washed with PBS 0.1% Triton‐X100 3 times again and mounted on the slide glass.

Lysotracker stainingWandering 3rd instar larvae were dissected in PBS and incubated with 50 μM LysoTracker Red DND‐99 (L7528, Invitrogen) in PBS for 5 min. After washing with PBS, samples were fixed with 4% PFA in PBS for 20 min. Then, samples were washed once with PBSTx and incubated with PBSTx containing DAPI (4 μg/ml) for 5 min. After washing with PBSTx, we observed the samples immediately with a confocal microscope.

Lysotracker staining

Wandering 3rd instar larvae were dissected in PBS and incubated with 50 μM LysoTracker Red DND‐99 (L7528, Invitrogen) in PBS for 5 min. After washing with PBS, samples were fixed with 4% PFA in PBS for 20 min. Then, samples were washed once with PBSTx and incubated with PBSTx containing DAPI (4 μg/ml) for 5 min. After washing with PBSTx, we observed the samples immediately with a confocal microscope.

PI stainingWing discs in L3 larvae were dissected in PBS, incubated with Propidium Iodide (100 nM, 341‐07881, Fujifilm) in PBS immediately after dissection, then fixed with 4% PFA in PBS, and washed in PBSTx.

PI staining

Wing discs in L3 larvae were dissected in PBS, incubated with Propidium Iodide (100 nM, 341‐07881, Fujifilm) in PBS immediately after dissection, then fixed with 4% PFA in PBS, and washed in PBSTx.

Starvation assayFor both measuring the starvation resistance and caspase activation, 5–7‐day‐old virgin female flies were used. The starvation food contained only 1.5% agar. To measure starvation resistance, we counted the number of dead flies every day. For the analysis of caspase activation, we specifically induced GC3Ai expression in ECs with 5966‐GS. We added 100 μM RU486 (M1732, TCI) to both the normal food and the starvation food.

Starvation assay

For both measuring the starvation resistance and caspase activation, 5–7‐day‐old virgin female flies were used. The starvation food contained only 1.5% agar. To measure starvation resistance, we counted the number of dead flies every day. For the analysis of caspase activation, we specifically induced GC3Ai expression in ECs with 5966‐GS. We added 100 μM RU486 (M1732, TCI) to both the normal food and the starvation food.

Genotyping for CG14044MI09290

To confirm CG14044MI09290 has a MIMIC insertion, we designed the primers to detect 500 bp surrounding the MIMIC insertion site. Since MIMIC is too large to amplify, the PCR does not work in CG14044MI09290.

Genotyping for CG14044MI09290

To confirm CG14044MI09290 has a MIMIC insertion, we designed the primers to detect 500 bp surrounding the MIMIC insertion site. Since MIMIC is too large to amplify, the PCR does not work in CG14044MI09290.

Quantification of developmental timingTo quantify developmental timing, we calculated the pupariation rate. Mated females were allowed to lay eggs on a grape agar plate for 8 h at 25°C. First, instar larvae were collected from the grape agar plate and placed into the normal food. Each vial contained 50 larvae. The number of new pupae of each genotype was recorded every 12 h. Pupariation rates were calculated as the number of pupariated flies divided by the total number of pupae at the end of each experiment.

Quantification of developmental timing

To quantify developmental timing, we calculated the pupariation rate. Mated females were allowed to lay eggs on a grape agar plate for 8 h at 25°C. First, instar larvae were collected from the grape agar plate and placed into the normal food. Each vial contained 50 larvae. The number of new pupae of each genotype was recorded every 12 h. Pupariation rates were calculated as the number of pupariated flies divided by the total number of pupae at the end of each experiment.

Sayonara orthologue identification, alignment, and phylogenetic reconstructionTo infer the evolutionary history of Sayonara, we first queried Panther (16.0) to identify gene family members related to CG14044. We found that CG14044 belonged to Panther family PTHR21974. As our aim was to characterize the emergence of a BH3 motif in the Sayonara orthogroup rather than to build a comprehensive phylogeny with remote homologs, we restricted our analysis to nonredundant sequences from Ecdysozoa. Among 816 hits matching Panther family PTHR21974, we extracted 109 protein sequences from which 69 were selected for further analysis. Manual curation was performed to exclude identical, nearly identical or partial sequences as well as sequences from overrepresented taxa (e.g., Drosophila). BLAST searches against NCBI and UniProtKB databases were conducted one‐by‐one to find additional sequences in taxa of interest leading to a final dataset containing 41 representative sequences. Sequence E0W198 from Pediculus humanus (Psocodea) was unusual and excluded from subsequent analysis in that it has a significantly truncated C terminus. Sequences were aligned using MUSCLE, and the alignment was edited manually at ambiguous sites and at N‐ and C‐terminal ends. A neighbor‐joining tree was constructed using the BioNJ algorithm as implemented in SeaView (version 5) and Poisson distances (gaps included), with 1,000 bootstrap replicates. The tree was rooted using CIL‐7 sequences from Caenorhabditis. The topology of our final tree is in accordance with previously published general phylogenies of insects and Diptera (Misof et al, 2014).

Sayonara orthologue identification, alignment, and phylogenetic reconstruction

To infer the evolutionary history of Sayonara, we first queried Panther (16.0) to identify gene family members related to CG14044. We found that CG14044 belonged to Panther family PTHR21974. As our aim was to characterize the emergence of a BH3 motif in the Sayonara orthogroup rather than to build a comprehensive phylogeny with remote homologs, we restricted our analysis to nonredundant sequences from Ecdysozoa. Among 816 hits matching Panther family PTHR21974, we extracted 109 protein sequences from which 69 were selected for further analysis. Manual curation was performed to exclude identical, nearly identical or partial sequences as well as sequences from overrepresented taxa (e.g., Drosophila). BLAST searches against NCBI and UniProtKB databases were conducted one‐by‐one to find additional sequences in taxa of interest leading to a final dataset containing 41 representative sequences. Sequence E0W198 from Pediculus humanus (Psocodea) was unusual and excluded from subsequent analysis in that it has a significantly truncated C terminus. Sequences were aligned using MUSCLE, and the alignment was edited manually at ambiguous sites and at N‐ and C‐terminal ends. A neighbor‐joining tree was constructed using the BioNJ algorithm as implemented in SeaView (version 5) and Poisson distances (gaps included), with 1,000 bootstrap replicates. The tree was rooted using CIL‐7 sequences from Caenorhabditis. The topology of our final tree is in accordance with previously published general phylogenies of insects and Diptera (Misof et al, 2014).

Profile‐based methods for domain composition and BH3 motif predictionSayonara domain composition was assessed using HMM profile searches with the HMMER package (v3.0 March 2010). Briefly, we searched the UniProtKB 2021_03 database to retrieve homologs of fly CG14044 using the BLASTp algorithm, and the new sequences (obtained with E‐value < 1e−6, length ≥ 135 or 341 residues) were aligned using Clustal W 1.8 to construct two HMM profiles with hmmbuild. The full‐length sequences (n = 52) of Sayonara (378 residues in Drosophila melanogaster) or the first 150 amino acids (present in n = 41 sequences) were used as inputs to build a first profile for the whole Sayonara orthogroup and a second profile specific for the N‐terminal variable domain. These two profiles were searched against the UniProtKB 2021_03 sequence database by hmmsearch, resulting in 215 and 60 hits, respectively (E‐value ≤ 0.001). Venn diagram analysis indicated that 60 sequences were common to both profiles, 155 sequences were exclusive to the first profile (full‐length sequences), and no sequence was exclusive to the second profile (N‐terminal domain). We predicted the presence or absence of BH3 motifs in the N‐terminal domain of Sayonara orthologous sequences using evolutionary information provided by a position‐specific scoring matrix (PSSM) profile. As previously described (Rech de Laval et al, 2014; Aouacheria et al, 2019), a PSSM profile of BH3 motifs was computed as part of an ab initio motif discovery procedure by running the meme program of the MEME software suite 4.9.0 using a set of 158 amino acid sequences of BCL‐2 homologous proteins and BH3‐only proteins. The PSSM was used to scan the set of 41 selected sequences for BH3 motif occurrences using the program mast from the MEME suite.

Profile‐based methods for domain composition and BH3 motif prediction

Sayonara domain composition was assessed using HMM profile searches with the HMMER package (v3.0 March 2010). Briefly, we searched the UniProtKB 2021_03 database to retrieve homologs of fly CG14044 using the BLASTp algorithm, and the new sequences (obtained with E‐value < 1e−6, length ≥ 135 or 341 residues) were aligned using Clustal W 1.8 to construct two HMM profiles with hmmbuild. The full‐length sequences (n = 52) of Sayonara (378 residues in Drosophila melanogaster) or the first 150 amino acids (present in n = 41 sequences) were used as inputs to build a first profile for the whole Sayonara orthogroup and a second profile specific for the N‐terminal variable domain. These two profiles were searched against the UniProtKB 2021_03 sequence database by hmmsearch, resulting in 215 and 60 hits, respectively (E‐value ≤ 0.001). Venn diagram analysis indicated that 60 sequences were common to both profiles, 155 sequences were exclusive to the first profile (full‐length sequences), and no sequence was exclusive to the second profile (N‐terminal domain). We predicted the presence or absence of BH3 motifs in the N‐terminal domain of Sayonara orthologous sequences using evolutionary information provided by a position‐specific scoring matrix (PSSM) profile. As previously described (Rech de Laval et al, 2014; Aouacheria et al, 2019), a PSSM profile of BH3 motifs was computed as part of an ab initio motif discovery procedure by running the meme program of the MEME software suite 4.9.0 using a set of 158 amino acid sequences of BCL‐2 homologous proteins and BH3‐only proteins. The PSSM was used to scan the set of 41 selected sequences for BH3 motif occurrences using the program mast from the MEME suite.

Characterization of the 
Buffy–Debcl
 duplication eventA total of 224 amino acid sequences (present in 201 different species) showing significative similarity to Drosophila Buffy and Debcl were collected from translated insect transcriptomes produced by the 1KITE (1K Insect Transcriptome Evolution) project (Misof et al, 2014). A profile HMM specific to both paralogous protein sequences was built and then used to screen the 1KITE transcriptome assemblies with the program hmmsearch (after translating the transcripts into all six possible reading frames). Only hits with a global e‐value ≤ 10−5 and that were predicted to contain a BH motif (using Batch CD‐search) were considered for subsequent analysis. Buffy and Debcl protein sequences were also searched against the nonredundant protein (NR) and Reference Sequence (RefSeq) databases using BLASTP, leading to an additional set of 39 sequences (in 25 species). All 263 sequences were aligned with MUSCLE and trimmed with GBLOCKS. The phylogenetic tree was reconstructed using the UL3 amino acid substitution model with PhyML‐Structure (Guindon & Gascuel, 2003; Le & Gascuel, 2010) and rooted using Collembola as the outgroup.

Characterization of the 
Buffy–Debcl
 duplication event

A total of 224 amino acid sequences (present in 201 different species) showing significative similarity to Drosophila Buffy and Debcl were collected from translated insect transcriptomes produced by the 1KITE (1K Insect Transcriptome Evolution) project (Misof et al, 2014). A profile HMM specific to both paralogous protein sequences was built and then used to screen the 1KITE transcriptome assemblies with the program hmmsearch (after translating the transcripts into all six possible reading frames). Only hits with a global e‐value ≤ 10−5 and that were predicted to contain a BH motif (using Batch CD‐search) were considered for subsequent analysis. Buffy and Debcl protein sequences were also searched against the nonredundant protein (NR) and Reference Sequence (RefSeq) databases using BLASTP, leading to an additional set of 39 sequences (in 25 species). All 263 sequences were aligned with MUSCLE and trimmed with GBLOCKS. The phylogenetic tree was reconstructed using the UL3 amino acid substitution model with PhyML‐Structure (Guindon & Gascuel, 2003; Le & Gascuel, 2010) and rooted using Collembola as the outgroup.

Ancestral sequence reconstruction and experimental resurrectionAncestral sequence reconstruction (ASR) was carried out as described previously (Groussin et al, 2015), accounting for the uncertainty in the reconstruction of the gene tree and gene duplication and loss. We used Phyldog (Boussau et al, 2013) to reconcile the gene tree with the species tree. We used 18S ribosomal RNA sequences, when available in the NR database (32 sequences), to reconstruct the species tree. The reconciled species tree‐aware gene tree was used as the guiding tree in Prank to realign Buffy–Debcl sequences. We used bppML (Dutheil & Boussau, 2008) to compare the capacity of different site‐homogeneous and site‐heterogenous models substitution models to fit the data and found that the site‐heterogenous EX_EHO model was best at fitting the data according to the BIC criterion. We then performed ASR with the ML parameter estimates obtained with the EX_EHO model, using bppAncestor (Dutheil & Boussau, 2008) with the marginal ASR approach (Yang et al, 1995b). For a given site at a given internal node of the reconciled tree, the amino acid having the maximum posterior probability was inferred as the putative ancestral amino acid. Gap positions in ancestral sequences were introduced a posteriori based on inferences performed by Prank during sequence alignment. The putative Buffy–Debcl ancestor was named “Borg0” after dBorg‐1 (one of the synonyms for Debcl) and dBorg‐2 (Buffy).

Ancestral sequence reconstruction and experimental resurrection

Ancestral sequence reconstruction (ASR) was carried out as described previously (Groussin et al, 2015), accounting for the uncertainty in the reconstruction of the gene tree and gene duplication and loss. We used Phyldog (Boussau et al, 2013) to reconcile the gene tree with the species tree. We used 18S ribosomal RNA sequences, when available in the NR database (32 sequences), to reconstruct the species tree. The reconciled species tree‐aware gene tree was used as the guiding tree in Prank to realign Buffy–Debcl sequences. We used bppML (Dutheil & Boussau, 2008) to compare the capacity of different site‐homogeneous and site‐heterogenous models substitution models to fit the data and found that the site‐heterogenous EX_EHO model was best at fitting the data according to the BIC criterion. We then performed ASR with the ML parameter estimates obtained with the EX_EHO model, using bppAncestor (Dutheil & Boussau, 2008) with the marginal ASR approach (Yang et al, 1995b). For a given site at a given internal node of the reconciled tree, the amino acid having the maximum posterior probability was inferred as the putative ancestral amino acid. Gap positions in ancestral sequences were introduced a posteriori based on inferences performed by Prank during sequence alignment. The putative Buffy–Debcl ancestor was named “Borg0” after dBorg‐1 (one of the synonyms for Debcl) and dBorg‐2 (Buffy).

Statistical analysisStatistical tests used are indicated in the figure captions. Sample sizes were determined empirically based on the observed effects. Blinding was not applicable because the investigators who set up the experiments were the same with the ones doing sample collection and analyses. However, each experiment was associated with proper controls, and samples were collected and analyzed under identical conditions. All the statistical analyses were conducted in GraphPad Prism.

Statistical analysis

Statistical tests used are indicated in the figure captions. Sample sizes were determined empirically based on the observed effects. Blinding was not applicable because the investigators who set up the experiments were the same with the ones doing sample collection and analyses. However, each experiment was associated with proper controls, and samples were collected and analyzed under identical conditions. All the statistical analyses were conducted in GraphPad Prism.

Disclosure and competing interests statementThe authors declare that they have no conflict of interest.

Disclosure and competing interests statement

The authors declare that they have no conflict of interest.

Supporting informationExpanded View Figures PDFClick here for additional data file.Dataset EV1Click here for additional data file.Source Data for Expanded ViewClick here for additional data file.PDF+Click here for additional data file.Source Data for Figure 1Click here for additional data file.Source Data for Figure 2Click here for additional data file.Source Data for Figure 3Click here for additional data file.Source Data for Figure 4Click here for additional data file.

Supporting information

Expanded View Figures PDF

Click here for additional data file.

Dataset EV1

Click here for additional data file.

Source Data for Expanded View

Click here for additional data file.

Click here for additional data file.

Source Data for Figure 1

Click here for additional data file.

Source Data for Figure 2

Click here for additional data file.

Source Data for Figure 3

Click here for additional data file.

Source Data for Figure 4

Click here for additional data file.
