---
pmid: '15282549'
title: LRIG1 restricts growth factor signaling by enhancing receptor ubiquitylation
  and degradation.
authors:
- Gur G
- Rubin C
- Katz M
- Amit I
- Citri A
- Nilsson J
- Amariglio N
- Henriksson R
- Rechavi G
- Hedman H
- Wides R
- Yarden Y
journal: EMBO J
year: '2004'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC514515
doi: 10.1038/sj.emboj.7600342
---

# LRIG1 restricts growth factor signaling by enhancing receptor ubiquitylation and degradation.
**Authors:** Gur G, Rubin C, Katz M, Amit I, Citri A, Nilsson J, Amariglio N, Henriksson R, Rechavi G, Hedman H, Wides R, Yarden Y
**Journal:** EMBO J (2004)
**DOI:** [10.1038/sj.emboj.7600342](https://doi.org/10.1038/sj.emboj.7600342)
**PMC:** [PMC514515](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC514515/)

## Abstract

1. EMBO J. 2004 Aug 18;23(16):3270-81. doi: 10.1038/sj.emboj.7600342. Epub 2004
Jul  29.

LRIG1 restricts growth factor signaling by enhancing receptor ubiquitylation and 
degradation.

Gur G(1), Rubin C, Katz M, Amit I, Citri A, Nilsson J, Amariglio N, Henriksson 
R, Rechavi G, Hedman H, Wides R, Yarden Y.

Author information:
(1)Department of Biological Regulation, The Weizmann Institute of Science, 
Rehovot, Israel.

Kekkon proteins negatively regulate the epidermal growth factor receptor (EGFR) 
during oogenesis in Drosophila. Their structural relative in mammals, LRIG1, is 
a transmembrane protein whose inactivation in rodents promotes skin hyperplasia, 
suggesting involvement in EGFR regulation. We report upregulation of LRIG1 
transcript and protein upon EGF stimulation, and physical association of the 
encoded protein with the four EGFR orthologs of mammals. Upregulation of LRIG1 
is followed by enhanced ubiquitylation and degradation of EGFR. The underlying 
mechanism involves recruitment of c-Cbl, an E3 ubiquitin ligase that 
simultaneously ubiquitylates EGFR and LRIG1 and sorts them for degradation. We 
conclude that LRIG1 evolved in mammals as a feedback negative attenuator of 
signaling by receptor tyrosine kinases.

DOI: 10.1038/sj.emboj.7600342
PMCID: PMC514515
PMID: 15282549 [Indexed for MEDLINE]

## Full Text

Abstract

Kekkon proteins negatively regulate the epidermal growth factor receptor (EGFR) during oogenesis in Drosophila . Their structural relative in mammals, LRIG1, is a transmembrane protein whose inactivation in rodents promotes skin hyperplasia, suggesting involvement in EGFR regulation. We report upregulation of LRIG1 transcript and protein upon EGF stimulation, and physical association of the encoded protein with the four EGFR orthologs of mammals. Upregulation of LRIG1 is followed by enhanced ubiquitylation and degradation of EGFR. The underlying mechanism involves recruitment of c-Cbl, an E3 ubiquitin ligase that simultaneously ubiquitylates EGFR and LRIG1 and sorts them for degradation. We conclude that LRIG1 evolved in mammals as a feedback negative attenuator of signaling by receptor tyrosine kinases.

Introduction

Cell fate determination, as well as rapid responses to extracellular cues, is critically mediated by soluble growth factors and their transmembrane receptors. Once activated by growth factors, the receptors simultaneously launch both ‘positive signals', which lead to cell stimulation, and ‘negative signals', which regulate the amplitude and duration of these positive signals (reviewed in Dikic and Giordano, 2003 ). A delicate balance between positive and negative signals is critical for normal cell homeostasis, and its disturbance is often implicated in disease development. An example is provided by the ErbB family of receptor tyrosine kinases (RTKs) and their ligands, soluble factors of the epidermal growth factor (EGF) family ( Yarden and Sliwkowski, 2001 ). The prevalence of positive signals, promoted by autocrine loops, overexpressed or mutated ErbB receptors, as well as unleashed downstream effectors, leads to excessive cell proliferation and often associates with human cancer.

Unlike positive signals, which are relatively well understood, the nature and mediators of signal desensitization are only beginning to be unraveled. A major source of information on negative ErbB signals arises from developmental genetics of invertebrate organisms like Caenorhabditis elegans and Drosophila . A major mediator of definitive negative signals, Sli-1/c-Cbl, was discovered in both worms and mammals. In C. elegans , loss of Sli-1 leads to excessive vulva formation, and naturally occurring aberrant forms of c-Cbl are oncogenic in mammals ( Thien et al , 2001 ). Studies in mammalian cells identified c-Cbl as an E3 ubiquitin ligase, which tags ligand-activated ErbB-1 molecules with ubiquitin, thereby promoting their sorting to degradation in lysosomes (reviewed in Shtiegman and Yarden, 2003 ). In insects, a group of negative regulators undergo transcriptional up-regulation following activation of the EGF-receptor (reviewed in Shilo, 2003 ). An example is kekkon-1 , which encodes a transmembrane protein that physically binds to and inhibits EGF receptor molecules ( Ghiglione et al , 1999 ). The six leucine-rich repeats (LRRs) of Kekkon-1 are necessary for recognition of EGFR, and for consequent inhibition of activation by growth factors ( Ghiglione et al , 2003 ; Alvarado et al , 2004 ). The multiple Kekkon proteins of insects have no clear orthologs in mammals ( MacLaren et al , 2004 ). However, the three mammalian LRIG genes share domain organization with Kekkons ( Suzuki et al , 1996 ; Nilsson et al , 2001 ; Guo et al , 2004 ; Holmlund et al , 2004 ). The extracellular regions of both the murine Lrig1/Lig-1 ( Suzuki et al , 1996 ) and the human LRIG1 ( Nilsson et al , 2001 ) share 15 LRRs followed by three immunoglobulin (Ig) domains. Interestingly, disruption of the LRIG1 gene in mice resulted in fertile animals that developed skin defects ( Suzuki et al , 2002 ).

The structural similarity of LRIG1 to Kekkons predicts that LRIG1 will interact with and restrict ErbB signaling in mammals, but no relevant data have been reported. Consistent with negative signaling, LRIG1 is downregulated in tumors of renal cell carcinoma ( Thomasson et al , 2004 ). The present study addressed the possible physical and functional interactions between human LRIG1 and ErbB receptors. We report direct ErbB–LRIG1 interactions, which inhibit ErbB signaling through a mechanism that involves enhancement of receptor ubiquitylation and accelerated intracellular degradation.

Discussion

Our study establishes for the first time a role for LRIG1, an LRR-Ig protein, as a feedback negative regulator of RTKs in mammalian cells. LRIG1 belongs to a distinct family, which is widely represented in both vertebrates and invertebrates ( Figure 1A ). Single LRIG orthologs exist in the nematodes C. elegans and C. briggsae . Likewise, a single clear ortholog exists in each sequenced insect genome, namely: bee-Apis mellifera , mosquito-Anopholes gambiae , Drosophila melanogaster and D. pseudoobscura . Moreover, a single LRIG homolog appears in the urochordate Ciona intestinalis , indicating that it is a ‘pre-duplication' member of the chordate LRIG family ( Guo et al , 2004 ). The six Kekkon genes have no clear orthologs in vertebrates or in nematodes, and they are distantly related to the LRIG proteins. Hence, it can be concluded that Kekkons and the LRIG ortholog in D. melanogaster have co-existed in insects as separate genes over extensive evolutionary time. Nevertheless, the findings reported in this study and in other studies ( Ghiglione et al , 1999 , 2003 ; Alvarado et al , 2004 ) imply that both LRIG1 and Kekkons evolved as negative regulators of RTKs, although the mechanisms underlying their actions appear distinct: inhibition of receptor activation in the case of Kekkons ( Ghiglione et al , 1999 , 2003 ), and accelerated receptor degradation in the case of LRIG1.

By identifying Cbl-mediated receptor degradation as a mechanism that underlies negative signaling by LRIG1, our study links LRIG1 to a growing list of proteins that help desensitize RTK signaling. Negative regulators of RTK signaling may be divided into two groups ( Dikic and Giordano, 2003 ). The immediate early regulators, such as Ras-GAP, Cbl and the cytoplasmic tyrosine kinase Ack/Ark, are ubiquitously expressed proteins whose function requires no de novo protein synthesis. The delayed negative regulators, such as Sprouty (reviewed in Guy et al , 2003 ) and RALT ( Fiorini et al , 2002 ), are transcriptionally induced upon receptor stimulation and, therefore, they reach their peak of activity only 2–3 h after stimulation. LRIG1 belongs to the latter group of delayed regulators (see Figure 1B and C ). Although both Kekkon-1 and LRIG1 physically interact with all four members of the ErbB family ( Ghiglione et al , 2003 ; Figure 1D ), their mechanisms of action differ. Conceivably, the shared LRRs and Ig-like domains of LRIG1 and Kekkon-1 enable recognition of extracellular motifs common to all ErbB proteins ( Figure 2B ), but the divergent cytoplasmic domain of LRIG1 evolved an ability to recruit c-Cbl ( Figure 4D ).

In light of the finding that LRIG1 acts within a framework of a negative feedback loop, it is interesting to consider the common attributes of the currently known delayed negative regulators. These proteins not only share time courses of transcriptional induction by RTKs but also physically associate with activated receptors. Thus, the newly synthesized Argos and LRIG1 bind to the extracellular region of the EGF receptor ( Schweitzer et al , 1995 ; and Figure 1D ), whereas RALT ( Fiorini et al , 2002 ) and Sprouty proteins ( Guy et al , 2003 ) bind either directly or indirectly to the cytoplasmic domains of RTKs. Interestingly, like LRIG1 ( Figure 5C ), RALT ( Fiorini et al , 2002 ) and Sprouty proteins ( Guy et al , 2003 ) undergo ubiquitylation and subsequent degradation, suggesting similar modes of post-translational control.

The following mechanism of LRIG1 modulation of RTKs emerges from our studies. In resting cells, LRIG1 and other delayed negative regulators are only weakly expressed, but upon RTK stimulation, the LRIG1 gene undergoes transcriptional activation. A few hours from ErbB activation, LRIG1 level reaches a high steady state, which probably enables physical attachment to all available ErbB receptors. Like RALT, but unlike Cbl, LRIG1 binds to both ligand-activated and nonactivated receptors. Likewise, the juxtamembrane domain of LRIG1 constitutively recruits a third component, c-Cbl, whose E3 ubiquitin ligase remains sedentary unless phosphorylated on a specific tyrosine residue ( Levkowitz et al , 1999 ; Thien et al , 2001 ). Upon ligand binding and activation of c-Cbl, all components of the putative complex undergo ubiquitylation, but LRIG1 dissociates and undergoes proteasomal degradation once a Cbl–receptor complex is internalized. An important feature of the proposed model is the ability of LRIG1 to sort for degradation receptors, which are otherwise uncoupled to the Cbl-mediated pathway of endocytosis and degradation. This feature has been demonstrated with Y1045F-ErbB-1 ( Figure 6A ), and it implies that LRIG1 evolved as a surrogate mechanism that enables Cbl recruitment and subsequent receptor inactivation.

Along with signaling and evolutionary aspects, our results may bear relevance to pathologies in which RTKs are involved. For example, ErbB proteins have been implicated in human cancer of epithelial and glial origins, due to frequent amplification and aberrations of the corresponding genes, as well as occurrence of autocrine loops (reviewed in Yarden and Sliwkowski, 2001 ). Hence, it will be interesting to test the possibility that oncogenic ErbB versions, such as EGFRvIII, escape regulation by LRIG1. Consistent with antagonistic interactions in cancer, a recent survey of renal cell carcinomas discovered reciprocal relationships between an overexpressed ErbB-1 and a downregulated LRIG1 ( Thomasson et al , 2004 ). The psoriasiform epidermal hyperplasia of Lrig1/Lig-1 -defective mice, as well as inverse relationships between LRIG1 levels and proliferative ability of psoriatic human keratinocytes ( Suzuki et al , 2002 ), are additional manifestations of the inhibitory potential of LRIG1. Nevertheless, further research, and more surveys of clinical specimens, will be necessary to address the proposition that LRIG1 can suppress oncogenic signals by means of recruiting an E3 ubiquitin ligase to active growth factor receptors.
