---
pmid: '33938021'
title: Genomewide Association Studies of LRRK2 Modifiers of Parkinson's Disease.
authors:
- Lai D
- Alipanahi B
- Fontanillas P
- Schwantes-An TH
- Aasly J
- Alcalay RN
- Beecham GW
- Berg D
- Bressman S
- Brice A
- Brockman K
- Clark L
- Cookson M
- Das S
- Van Deerlin V
- Follett J
- Farrer MJ
- Trinh J
- Gasser T
- Goldwurm S
- Gustavsson E
- Klein C
- Lang AE
- Langston JW
- Latourelle J
- Lynch T
- Marder K
- Marras C
- Martin ER
- McLean CY
- Mejia-Santana H
- Molho E
- Myers RH
- Nuytemans K
- Ozelius L
- Payami H
- Raymond D
- Rogaeva E
- Rogers MP
- Ross OA
- Samii A
- Saunders-Pullman R
- Schüle B
- Schulte C
- Scott WK
- Tanner C
- Tolosa E
- Tomkins JE
- Vilas D
- Trojanowski JQ
- 23andMe Research Team
- Uitti R
- Vance JM
- Visanji NP
- Wszolek ZK
- Zabetian CP
- Mirelman A
- Giladi N
- Orr Urtreger A
- Cannon P
- Fiske B
- Foroud T
journal: Ann Neurol
year: '2021'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC8252519
doi: 10.1002/ana.26094
---

# Genomewide Association Studies of LRRK2 Modifiers of Parkinson's Disease.
**Authors:** Lai D, Alipanahi B, Fontanillas P, Schwantes-An TH, Aasly J, Alcalay RN, Beecham GW, Berg D, Bressman S, Brice A, Brockman K, Clark L, Cookson M, Das S, Van Deerlin V, Follett J, Farrer MJ, Trinh J, Gasser T, Goldwurm S, Gustavsson E, Klein C, Lang AE, Langston JW, Latourelle J, Lynch T, Marder K, Marras C, Martin ER, McLean CY, Mejia-Santana H, Molho E, Myers RH, Nuytemans K, Ozelius L, Payami H, Raymond D, Rogaeva E, Rogers MP, Ross OA, Samii A, Saunders-Pullman R, Schüle B, Schulte C, Scott WK, Tanner C, Tolosa E, Tomkins JE, Vilas D, Trojanowski JQ, 23andMe Research Team, Uitti R, Vance JM, Visanji NP, Wszolek ZK, Zabetian CP, Mirelman A, Giladi N, Orr Urtreger A, Cannon P, Fiske B, Foroud T
**Journal:** Ann Neurol (2021)
**DOI:** [10.1002/ana.26094](https://doi.org/10.1002/ana.26094)
**PMC:** [PMC8252519](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8252519/)

## Abstract

1. Ann Neurol. 2021 Jul;90(1):76-88. doi: 10.1002/ana.26094. Epub 2021 May 17.

Genomewide Association Studies of LRRK2 Modifiers of Parkinson's Disease.

Lai D(#)(1), Alipanahi B(#)(2)(3), Fontanillas P(#)(2), Schwantes-An TH(1), 
Aasly J(4), Alcalay RN(5), Beecham GW(6), Berg D(7)(8), Bressman S(9), Brice 
A(10), Brockman K(8)(11), Clark L(12), Cookson M(13), Das S(2), Van Deerlin 
V(14), Follett J(15), Farrer MJ(15), Trinh J(16), Gasser T(8)(11), Goldwurm 
S(17), Gustavsson E(18), Klein C(16), Lang AE(19), Langston JW(20), Latourelle 
J(21), Lynch T(22), Marder K(23), Marras C(19), Martin ER(6), McLean CY(2)(24), 
Mejia-Santana H(25), Molho E(26), Myers RH(27), Nuytemans K(6), Ozelius L(9), 
Payami H(28), Raymond D(9), Rogaeva E(29), Rogers MP(30), Ross OA(31)(32), Samii 
A(33), Saunders-Pullman R(9), Schüle B(34), Schulte C(8)(11), Scott WK(6), 
Tanner C(35), Tolosa E(36), Tomkins JE(37), Vilas D(36), Trojanowski JQ(14); 
23andMe Research Team(2); Uitti R(38), Vance JM(6), Visanji NP(19), Wszolek 
ZK(38), Zabetian CP(33), Mirelman A(39), Giladi N(39), Orr Urtreger A(39), 
Cannon P(2), Fiske B(40), Foroud T(1).

Author information:
(1)Department of Medical and Molecular Genetics, Indiana University School of 
Medicine, Indianapolis, IN.
(2)23andMe, Inc., Sunnyvale, CA.
(3)Google LLC, Palo Alto, CA.
(4)Department of Neurology, St. Olavs Hospital, Trondheim, Norway.
(5)Department of Neurology, Columbia University, New York, NY.
(6)John P. Hussman Institute for Human Genomics and Dr. John T. Macdonald 
Department of Human Genetics, University of Miami, Miller School of Medicine, 
Miami, FL.
(7)Department of Neurology, Christian-Albrechts-University of Kiel, Kiel, 
Germany.
(8)Department of Neurodegenerative Diseases, Hertie Institute for Clinical Brain 
Research, University of Tübingen, Tübingen, Germany.
(9)Department of Neurology, Icahn School of Medicine at Mount Sinai, New York, 
NY.
(10)Sorbonne Université, Institut du Cerveau et de la Moelle épinière (ICM), 
AP-HP, Inserm, CNRS, University Hospital Pitié-Salpêtrière, Paris, France.
(11)German Center for Neurodegenerative Diseases (DZNE), Tübingen, Germany.
(12)Department of Pathology and Cell Biology, Columbia University, New York, NY.
(13)Laboratory of Neurogenetics, National Institute of Aging, National Institute 
of Health, Bethesda, MD.
(14)Department of Pathology and Laboratory Medicine, University of Pennsylvania, 
Philadelphia, PA.
(15)Laboratory of Neurogenetics and Neuroscience, Fixel Institute for 
Neurological Diseases, McKnight Brain Institute, L5-101D, UF Clinical and 
Translational Science Institute, University of Florida, Gainesville, FL.
(16)Institute of Neurogenetics, University of Luebeck, Luebeck, Germany.
(17)Parkinson Institute, ASST "G.Pini-CTO, Milan, Italy.
(18)Centre for Applied Neurogenetics, University of British Columbia, Vancouver, 
Canada.
(19)The Edmond J. Safra Program in Parkinson's Disease and the Morton and Gloria 
Shulman Movement Disorders Clinic, Toronto Western Hospital, Toronto, Canada.
(20)Departments of Neurology, Neuroscience, and Pathology, Stanford University 
School of Medicine, Stanford, CA.
(21)GNS Healthcare, Cambridge, MA.
(22)Dublin Neurological Institute at the Mater Misericordiae University 
Hospital, Conway Institute of Biomolecular and Biomedical Research, University 
College Dublin, Dublin, Ireland.
(23)Department of Neurology and Psychiatry, Taub Institute and Sergievsky 
Center, Columbia University Vagelos College of Physicians and Surgeons, New 
York, NY.
(24)Google LLC, Cambridge, MA.
(25)Gertrude H. Sergievsky Center, Columbia University, New York, NY.
(26)Department of Neurology, Albany Medical College, Albany, NY.
(27)Department of Neurology, Boston University, Boston, MA.
(28)Department of Neurology, University of Alabama at Birmingham, Birmingham, 
AL.
(29)Tanz Centre for Research in Neurodegenerative Diseases and Department of 
Neurology, University of Toronto, Toronto, Canada.
(30)Department of General Surgery, University of South Florida Morsani College 
of Medicine, Tampa, FL.
(31)Departments of Neuroscience and Clinical Genomics, Mayo Clinic, 
Jacksonville, FL.
(32)School of Medicine and Medical Science, University College Dublin, Dublin, 
Ireland.
(33)VA Puget Sound Health Care System and Department of Neurology, University of 
Washington, Seattle, WA.
(34)Department of Pathology, Stanford University School of Medicine, Stanford, 
CA.
(35)University of California, San Francisco Veterans Affairs Health Care System, 
San Francisco, CA.
(36)Parkinson Disease and Movement Disorders Unit, Hospital Clínic Universitari, 
Institut d'Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), University 
of Barcelona (UB), Centro de Investigación Biomédica en Red sobre Enfermedades 
Neurodegenerativas (CIBERNED), Barcelona, Spain.
(37)School of Pharmacy, University of Reading, Reading, UK.
(38)Department of Neurology, Mayo Clinic, Jacksonville, FL.
(39)Tel Aviv Sourasky Medical Center, Sackler Faculty of Medicine and Sagol 
School of Neuroscience, Tel Aviv University, Tel Aviv, Israel.
(40)The Michael J. Fox Foundation for Parkinson's Research, New York, NY.
(#)Contributed equally

OBJECTIVE: The aim of this study was to search for genes/variants that modify 
the effect of LRRK2 mutations in terms of penetrance and age-at-onset of 
Parkinson's disease.
METHODS: We performed the first genomewide association study of penetrance and 
age-at-onset of Parkinson's disease in LRRK2 mutation carriers (776 cases and 
1,103 non-cases at their last evaluation). Cox proportional hazard models and 
linear mixed models were used to identify modifiers of penetrance and 
age-at-onset of LRRK2 mutations, respectively. We also investigated whether a 
polygenic risk score derived from a published genomewide association study of 
Parkinson's disease was able to explain variability in penetrance and 
age-at-onset in LRRK2 mutation carriers.
RESULTS: A variant located in the intronic region of CORO1C on chromosome 12 
(rs77395454; p value = 2.5E-08, beta = 1.27, SE = 0.23, risk allele: C) met 
genomewide significance for the penetrance model. Co-immunoprecipitation 
analyses of LRRK2 and CORO1C supported an interaction between these 2 proteins. 
A region on chromosome 3, within a previously reported linkage peak for 
Parkinson's disease susceptibility, showed suggestive associations in both 
models (penetrance top variant: p value = 1.1E-07; age-at-onset top variant: p 
value = 9.3E-07). A polygenic risk score derived from publicly available 
Parkinson's disease summary statistics was a significant predictor of 
penetrance, but not of age-at-onset.
INTERPRETATION: This study suggests that variants within or near CORO1C may 
modify the penetrance of LRRK2 mutations. In addition, common Parkinson's 
disease associated variants collectively increase the penetrance of LRRK2 
mutations. ANN NEUROL 2021;90:82-94.

© 2021 The Authors. Annals of Neurology published by Wiley Periodicals LLC on 
behalf of American Neurological Association.

DOI: 10.1002/ana.26094
PMCID: PMC8252519
PMID: 33938021 [Indexed for MEDLINE]

Conflict of interest statement: B.A., P.F., P.C., S.D., C.Y.M., and members of 
the 23andMe Research Team are current or former employees of 23andMe, Inc., and 
hold stock or stock options in 23andMe.

## Full Text

Parkinson's disease (PD) is the second most common neurodegenerative disease in older adults.
1
 Several genes showing autosomal dominant (SNCA, LRRK2, and VPS35) or recessive (PRKN, PINK1, and DJ‐1) inheritance patterns have been identified as the cause of familial PD. These genes harbor rare, high penetrance mutations that explain up to 10% of familial PD cases in different populations.
1
, 
2
 Recently, large genomewide association studies (GWAS) have identified over 90 loci with small individual effects on disease risk in both familial and sporadic PD.
3
, 
4

Mutations in LRRK2 are among the most common genetic causes of PD.
1
, 
2
 The most frequent mutation is G2019S (rs34637584), which explains up to 10% of familial PD cases and 1% to 2% of all PD cases.
2
, 
5
 Among patients with PD, the frequency of the G2019S mutation is approximately 3% in Europeans, 16% to 19% in Ashkenazi Jews and up to 42% in Arab‐Berbers.
6
, 
7
, 
8
, 
9
, 
10
, 
11
, 
12
, 
13
, 
14
, 
15
 Estimates of the risk for developing PD among LRRK2 G2019S mutation carriers range from 15% to 85%.
16
, 
17
, 
18
, 
19
 To explain the incomplete penetrance of G2019S, it has long been hypothesized that there are other genes/variants outside of LRRK2 acting to modify its effect (LRRK2 modifiers). Identification of LRRK2 modifiers could aid the development of novel prevention and treatment strategies for PD.

Most studies of LRRK2 modifiers, to date, have focused on candidate genes. Because the protein product of LRRK2 may interact with α‐synuclein (encoded by SNCA), and tau (encoded by MAPT),
20
, 
21
 variants in SNCA and MAPT were widely investigated. However, the results have been inconsistent, possibly due to small sample sizes and differences in variants and populations investigated.
22
, 
23
, 
24
, 
25
, 
26
, 
27
, 
28
, 
29
, 
30
 Other PD associated genes, such as GBA,
29

BST1,
29

GAK,
30
 and PARK16

29
, 
31
, 
32
 have also been investigated. However, the number of studies is limited and findings remain to be replicated. Genomewide searches for LRRK2 modifiers are sparse and limited to linkage studies. Using 85 LRRK2 carriers from 38 families, a genomewide linkage study of LRRK2 modifiers found a suggestive linkage region at 1q32 (limit of detection [LOD] = 2.43); but that study did not identify any candidate genes/variants underlying the linkage peak.
33
 A genomewide linkage scan in Arab‐Berber PD families found DNM3 as a LRRK2 modifier.
34
 This finding was not independently replicated, although it was still significant in a meta‐analysis including the participants reported in the original finding.
25
, 
35
 GWASs have successfully detected many disease genes/variants, including those associated with PD. However, to date, no GWAS for LRRK2 modifiers has been reported, probably due to limitations in sample size and corresponding statistical power.

In this study, we recruited LRRK2 mutation carriers from multiple centers and performed the first GWAS to identify genes/variants that modify the penetrance and age‐at‐onset of PD among LRRK2 mutation carriers. Using the largest cohort to date, which consisted of 1,879 LRRK2 mutation carriers (including 776 PD cases), one genomewide significant association signal was found in the intronic region of the CORO1C gene. Using co‐immunoprecipitation analyses, we demonstrated that the protein product of CORO1C interacted with LRRK2. In addition, we found that a polygenic risk score (PRS) derived from publicly available PD GWAS summary statistics, was associated with penetrance, but not age‐at‐onset, of PD in LRRK2 mutation carriers.

Methods
Study Participants
The studies and the LRRK2 mutation carriers were grouped into 3 cohorts. The first cohort was primarily identified from The Michael J. Fox Foundation's LRRK2 Consortium and consisted of research sites worldwide (referred to as the MJFF consortium cohort). We searched PubMed and identified study groups that reported LRRK2 mutation carriers and then asked them to participate in this study (PubMed IDs: 16240353, 16333314, 18986508, and 16960813).
36
, 
37
, 
38
, 
39
 We also made announcements at international conferences to recruit more study mutation carriers. Details can be found in their publications.
36
, 
37
, 
38
, 
39
 To maximize participation and facilitate uniform data preparation across sites, a minimal dataset was submitted for all subjects that included LRRK2 mutation status, sex, age‐at‐onset (for PD cases), age at last evaluation (for non‐PD participants), and pedigree information, along with the availability of a minimal amount of DNA (approximately 2 ug). The minimal phenotypic data were sent to Indiana University and the subjects were assigned a unique identifier. The second cohort was from Tel Aviv University, Israel (referred to as the Israel cohort). Participants were of Ashkenazi origin and recruited from the Movement Disorders Unit at Tel Aviv Medical Center. PD diagnosis was confirmed by a movement disorders specialist and clinical disease status (PD or not diagnosed as PD) was evaluated at the time of blood draw for genetic testing. The third cohort (referred to as the 23andMe cohort) consisted of research participants of the personal genetics company 23andMe, Inc. who were LRRK2 G2019S carriers and whose PD status was known. Individuals who reported via an online survey that they had been diagnosed with PD by a medical professional, were asked to provide their age at diagnosis. For individuals who affirmed at least once that they had not been diagnosed with PD, their age at the most recent completion of the survey was recorded. The institutional review board at each participating site approved this study.
Genotyping, Quality Review, and Imputation
All study participants were genotyped on the Illumina Omni 2.5 Exome Array version 1.1 (Illumina, San Diego, CA, USA), except 166 participants from the Israel cohort, who were genotyped on an earlier version of the same array (version 1.0). This array has common, rare, and exonic variants that were selected from diverse world population samples included in the 1000 Genomes Project. In total, there were > 2.58 M variants, including > 567 K exonic variants. Participants from the MJFF consortium and 23andMe were genotyped at the Center for Inherited Disease Research (CIDR) at Johns Hopkins University (Baltimore, MD, USA). The Israel cohort was genotyped at Tel Aviv University and 2 samples from the MJFF consortium were included for quality control. There were 134 duplicated and unexpected identical participants among all 3 cohorts. Pairwise concordance rates were all > 99.97%, showing high consistency among the 2 genotyping laboratories and the 2 versions of the Illumina array.Variants with genotypic missing rates > 5% and nonpolymorphic variants were excluded. In addition, variants with A/T or C/G alleles were also excluded due to strand ambiguity. Hardy–Weinberg equilibrium (HWE) was not used to filter variants because these participants were ascertained to be LRRK2 mutation carriers and this participant selection scheme would directly violate HWE and remove potential LRRK2 modifiers from the analysis.To confirm the reported pedigree structure and detect cryptic relatedness, we used a set of 56,184 high quality (missing rate < 2%, HWE p values > 0.001), common (minor allele frequency [MAF] > 0.1), and independent (linkage disequilibrium as measured by r2 < 0.5) variants to calculate the pairwise identity by descent using PLINK.
40
 Reported pedigree structures were revised accordingly, if necessary. Mendelian error checking was performed in the revised pedigree structure. Any inconsistent genotypes were set to missing. The same set of variants was also used to estimate the principal components (PCs) of population stratification using Eigenstrat.
41
 All samples were imputed to the Haplotype Reference Consortium (http://www.haplotype-reference-consortium.org/) using Minimac3.
42
 A total of 725,802 high quality genotyped variants were selected for imputation (MAF > 3%, HWE p value > 0.0001, and missing rate < 5%). EAGLE version 2.4
43
 was used to phase genotyped variants for each sample. After filtering out variants with poor imputation quality score (R2 < 0.6) and checking for Mendelian inconsistencies using PLINK,
40
 a final dataset of 7,934,276 imputed and genotyped variants was used for association analyses.
Genomewide Association Studies
Our association analysis tested two models: (1) variants modifying the penetrance for PD among LRRK2 mutation carriers (penetrance model), and (2) variants modifying the age‐at‐onset for PD among LRRK2 mutation carriers (age‐at‐onset model). For the penetrance model (including PD cases and those not diagnosed as PD at last evaluation), the association analysis was designed to identify variants associated with the time to PD diagnosis or last evaluation for undiagnosed mutation carriers. For the age‐at‐onset model (PD cases only), the association analysis tested whether variants contributed to the age‐at‐onset for PD cases among LRRK2 mutation carriers.For the penetrance model, a mixed effect Cox proportional hazard model (frailty model) was used with sex, 10 PCs, array, and cohort indicators as covariates. Family relationships were adjusted by using a kinship matrix calculated using R package COXME (https://cran.r-project.org/web/packages/coxme/index.html). For the age‐at‐onset model, a linear mixed model was fit with the same covariates as the penetrance model and a kinship matrix to adjust family relationships. Although adjusting 10 PCs and family relationships could minimize the effects of population stratifications and shared genetic and environment factors among family members, the use of this mixed samples was designed to search for common genetic variants that have the same effects in participants from all populations and could miss population‐specific or family specific findings. Variants with MAF > 1% were tested for association in these 2 models. In addition to the single variant analyses, we performed gene‐based association analyses for both the penetrance and age‐at‐onset models. We focused on rare exonic and splicing variants, based on annotations from Variant Effect Predictor (https://useast.ensembl.org/info/docs/tools/vep/index.html), and restricting to variants with MAF < 3%. Only genotyped variants (N = 725,802) were used in the gene‐based analyses due to the low quality of imputation for rare variants. The R package COXME was used to perform all analyses (https://cran.r-project.org/web/packages/coxme/index.html). Conditional analysis was conducted using the most significant variant in an associated region as a covariate, and additional signals within the associated region were determined based on p values < 0.01.
Cell Culture, Co‐Immunoprecipitation, and Antibodies
To test whether the protein product of the identified gene interacts with LRRK2, we performed co‐immunoprecipitation analyses. HEK293FT (R70007; Invitrogen) cells were maintained as previously described
44
 and transiently transfected with pEGFP‐C1, pEGFP‐CORO1C, or co‐transfected with 3xFlag‐LRRK2 using Lipofectamine 2000 (Thermofisher Scientific) for 16 hours. Cells were washed once with ice cold PBS and lysed in 1 ml of GFP‐trap buffer (20 mM Tris–HCl pH 7.5, 150 mM NaCl, 1 mM EDTA, 50 mM NaF, 0.3% Triton X‐100, 5% Glycerol, Halt phosphatase inhibitor cocktail [Thermofisher Scientific], and protease inhibitor cocktail [Roche]) or Flag/GFP buffer (20 mM Tris–HCl pH 7.5, 300 mM NaCl, 1 mM EDTA, 50 mM NaF, 0.3% Triton X‐100, 5% Glycerol, Halt phosphatase inhibitor cocktail [Thermofisher Scientific], and protease inhibitor cocktail [Roche]) for 30 minutes on ice, followed by centrifugation at 4°C for 10 minutes at 13,900 g to obtain the supernatant. The supernatant was then incubated with either pre‐equilibrated GFP‐Trap agarose beads (ChromoTek) for 1 hour at 4°C on an end over end rotator to recover the GFP tag or pre‐cleared with Pierce Protein A/G Agarose (Thermofisher Scientific; 20 minutes at 4°C). Pre‐cleared lysates were incubated with 1 μg of monoclonal anti‐flag antibody (F3165; Millipore‐Sigma) or rabbit polyclonal anti‐GFP (Ab290; Abcam) for 2 hours at 4°C on an end over end rotator. Antibody complexes were captured by Protein A/G Agarose beads for 2 hours. Bead complexes were washed 6 times with lysis buffer, followed by elution in 1 times loading dye containing 2% β‐mercaptoethanol for 6 minutes at 95°C. All samples were resolved by Western blot as previously described.
45
 Each co‐immunoprecipitation was repeated in 2 to 3 independent experiments. Primary antibodies used in this study are as follows: mouse monoclonal anti‐FLAG M2 (F3165) was purchased from Thermofisher Scientific; Rabbit polyclonal to GFP was from Abcam (ab290), and anti‐LRRK2 was from UC Davis/NIH NeuroMab Facility (clone N241A/34).
In Silico Functional Studies
To evaluate whether the genomewide significant findings had immediate biological consequences on gene expression (expression quantitative trait locus [eQTL]) of nearby genes, we searched Open Targets Genetics (https://genetics.opentargets.org/) and GTEx (https://www.gtexportal.org/). In addition, protein–protein interaction (PPI) data were assessed to identify whether the protein product of the nominated gene either interacts directly with LRRK2 or has common interactors that are shared with LRRK2, using Protein Interaction Network Online Tool (PINOT) version 1.0
46
 queried on June 16, 2020 (http://www.reading.ac.uk/bioinf/PINOT/PINOT_form.html). We also performed chromatin interaction mapping to check whether our top findings interact with LRRK2 distantly. Functional Mapping and Annotation of GenomeWide Association Studies (FUMA: https://fuma.ctglab.nl/) was used to perform chromatin interaction mapping.
47
 Hi‐C data for chromatin interaction mapping were from Schmitt et al 2016 and Giusti‐Rodriguez et al 2019,
48
, 
49
 and are available in FUMA.
Polygenic Risk Score Analyses
In the largest GWAS analysis of PD susceptibility to date, Nalls et al meta‐analyzed 17 datasets with 56,306 PD cases or proxy‐cases and 1.4 million controls.
4
 Based on their results, they developed a PRS using summary statistics of 1,805 variants that can explain 26% of PD heritability.
4
 In this study, we performed PRS analysis using these 1,805 variants. Detailed information about how to select these 1,805 variants was described in Nalls et al.
4
 Because we were searching for LRRK2 modifiers, variants in the LRRK2 region (chr12: 40,118,913‐41,263,086) were excluded. The PRS was calculated as a weighted summation of effective alleles with the logarithm of odds ratios as the weights. This derived PRS was used to fit the same models with the same set of covariates as described for the genomewide association analyses using the R package COXME.

Study Participants
The studies and the LRRK2 mutation carriers were grouped into 3 cohorts. The first cohort was primarily identified from The Michael J. Fox Foundation's LRRK2 Consortium and consisted of research sites worldwide (referred to as the MJFF consortium cohort). We searched PubMed and identified study groups that reported LRRK2 mutation carriers and then asked them to participate in this study (PubMed IDs: 16240353, 16333314, 18986508, and 16960813).
36
, 
37
, 
38
, 
39
 We also made announcements at international conferences to recruit more study mutation carriers. Details can be found in their publications.
36
, 
37
, 
38
, 
39
 To maximize participation and facilitate uniform data preparation across sites, a minimal dataset was submitted for all subjects that included LRRK2 mutation status, sex, age‐at‐onset (for PD cases), age at last evaluation (for non‐PD participants), and pedigree information, along with the availability of a minimal amount of DNA (approximately 2 ug). The minimal phenotypic data were sent to Indiana University and the subjects were assigned a unique identifier. The second cohort was from Tel Aviv University, Israel (referred to as the Israel cohort). Participants were of Ashkenazi origin and recruited from the Movement Disorders Unit at Tel Aviv Medical Center. PD diagnosis was confirmed by a movement disorders specialist and clinical disease status (PD or not diagnosed as PD) was evaluated at the time of blood draw for genetic testing. The third cohort (referred to as the 23andMe cohort) consisted of research participants of the personal genetics company 23andMe, Inc. who were LRRK2 G2019S carriers and whose PD status was known. Individuals who reported via an online survey that they had been diagnosed with PD by a medical professional, were asked to provide their age at diagnosis. For individuals who affirmed at least once that they had not been diagnosed with PD, their age at the most recent completion of the survey was recorded. The institutional review board at each participating site approved this study.

Study Participants

The studies and the LRRK2 mutation carriers were grouped into 3 cohorts. The first cohort was primarily identified from The Michael J. Fox Foundation's LRRK2 Consortium and consisted of research sites worldwide (referred to as the MJFF consortium cohort). We searched PubMed and identified study groups that reported LRRK2 mutation carriers and then asked them to participate in this study (PubMed IDs: 16240353, 16333314, 18986508, and 16960813).
36
, 
37
, 
38
, 
39
 We also made announcements at international conferences to recruit more study mutation carriers. Details can be found in their publications.
36
, 
37
, 
38
, 
39
 To maximize participation and facilitate uniform data preparation across sites, a minimal dataset was submitted for all subjects that included LRRK2 mutation status, sex, age‐at‐onset (for PD cases), age at last evaluation (for non‐PD participants), and pedigree information, along with the availability of a minimal amount of DNA (approximately 2 ug). The minimal phenotypic data were sent to Indiana University and the subjects were assigned a unique identifier. The second cohort was from Tel Aviv University, Israel (referred to as the Israel cohort). Participants were of Ashkenazi origin and recruited from the Movement Disorders Unit at Tel Aviv Medical Center. PD diagnosis was confirmed by a movement disorders specialist and clinical disease status (PD or not diagnosed as PD) was evaluated at the time of blood draw for genetic testing. The third cohort (referred to as the 23andMe cohort) consisted of research participants of the personal genetics company 23andMe, Inc. who were LRRK2 G2019S carriers and whose PD status was known. Individuals who reported via an online survey that they had been diagnosed with PD by a medical professional, were asked to provide their age at diagnosis. For individuals who affirmed at least once that they had not been diagnosed with PD, their age at the most recent completion of the survey was recorded. The institutional review board at each participating site approved this study.

Genotyping, Quality Review, and Imputation
All study participants were genotyped on the Illumina Omni 2.5 Exome Array version 1.1 (Illumina, San Diego, CA, USA), except 166 participants from the Israel cohort, who were genotyped on an earlier version of the same array (version 1.0). This array has common, rare, and exonic variants that were selected from diverse world population samples included in the 1000 Genomes Project. In total, there were > 2.58 M variants, including > 567 K exonic variants. Participants from the MJFF consortium and 23andMe were genotyped at the Center for Inherited Disease Research (CIDR) at Johns Hopkins University (Baltimore, MD, USA). The Israel cohort was genotyped at Tel Aviv University and 2 samples from the MJFF consortium were included for quality control. There were 134 duplicated and unexpected identical participants among all 3 cohorts. Pairwise concordance rates were all > 99.97%, showing high consistency among the 2 genotyping laboratories and the 2 versions of the Illumina array.Variants with genotypic missing rates > 5% and nonpolymorphic variants were excluded. In addition, variants with A/T or C/G alleles were also excluded due to strand ambiguity. Hardy–Weinberg equilibrium (HWE) was not used to filter variants because these participants were ascertained to be LRRK2 mutation carriers and this participant selection scheme would directly violate HWE and remove potential LRRK2 modifiers from the analysis.To confirm the reported pedigree structure and detect cryptic relatedness, we used a set of 56,184 high quality (missing rate < 2%, HWE p values > 0.001), common (minor allele frequency [MAF] > 0.1), and independent (linkage disequilibrium as measured by r2 < 0.5) variants to calculate the pairwise identity by descent using PLINK.
40
 Reported pedigree structures were revised accordingly, if necessary. Mendelian error checking was performed in the revised pedigree structure. Any inconsistent genotypes were set to missing. The same set of variants was also used to estimate the principal components (PCs) of population stratification using Eigenstrat.
41
 All samples were imputed to the Haplotype Reference Consortium (http://www.haplotype-reference-consortium.org/) using Minimac3.
42
 A total of 725,802 high quality genotyped variants were selected for imputation (MAF > 3%, HWE p value > 0.0001, and missing rate < 5%). EAGLE version 2.4
43
 was used to phase genotyped variants for each sample. After filtering out variants with poor imputation quality score (R2 < 0.6) and checking for Mendelian inconsistencies using PLINK,
40
 a final dataset of 7,934,276 imputed and genotyped variants was used for association analyses.

Genotyping, Quality Review, and Imputation

All study participants were genotyped on the Illumina Omni 2.5 Exome Array version 1.1 (Illumina, San Diego, CA, USA), except 166 participants from the Israel cohort, who were genotyped on an earlier version of the same array (version 1.0). This array has common, rare, and exonic variants that were selected from diverse world population samples included in the 1000 Genomes Project. In total, there were > 2.58 M variants, including > 567 K exonic variants. Participants from the MJFF consortium and 23andMe were genotyped at the Center for Inherited Disease Research (CIDR) at Johns Hopkins University (Baltimore, MD, USA). The Israel cohort was genotyped at Tel Aviv University and 2 samples from the MJFF consortium were included for quality control. There were 134 duplicated and unexpected identical participants among all 3 cohorts. Pairwise concordance rates were all > 99.97%, showing high consistency among the 2 genotyping laboratories and the 2 versions of the Illumina array.

Variants with genotypic missing rates > 5% and nonpolymorphic variants were excluded. In addition, variants with A/T or C/G alleles were also excluded due to strand ambiguity. Hardy–Weinberg equilibrium (HWE) was not used to filter variants because these participants were ascertained to be LRRK2 mutation carriers and this participant selection scheme would directly violate HWE and remove potential LRRK2 modifiers from the analysis.

To confirm the reported pedigree structure and detect cryptic relatedness, we used a set of 56,184 high quality (missing rate < 2%, HWE p values > 0.001), common (minor allele frequency [MAF] > 0.1), and independent (linkage disequilibrium as measured by r2 < 0.5) variants to calculate the pairwise identity by descent using PLINK.
40
 Reported pedigree structures were revised accordingly, if necessary. Mendelian error checking was performed in the revised pedigree structure. Any inconsistent genotypes were set to missing. The same set of variants was also used to estimate the principal components (PCs) of population stratification using Eigenstrat.
41
 All samples were imputed to the Haplotype Reference Consortium (http://www.haplotype-reference-consortium.org/) using Minimac3.
42
 A total of 725,802 high quality genotyped variants were selected for imputation (MAF > 3%, HWE p value > 0.0001, and missing rate < 5%). EAGLE version 2.4
43
 was used to phase genotyped variants for each sample. After filtering out variants with poor imputation quality score (R2 < 0.6) and checking for Mendelian inconsistencies using PLINK,
40
 a final dataset of 7,934,276 imputed and genotyped variants was used for association analyses.

Genomewide Association Studies
Our association analysis tested two models: (1) variants modifying the penetrance for PD among LRRK2 mutation carriers (penetrance model), and (2) variants modifying the age‐at‐onset for PD among LRRK2 mutation carriers (age‐at‐onset model). For the penetrance model (including PD cases and those not diagnosed as PD at last evaluation), the association analysis was designed to identify variants associated with the time to PD diagnosis or last evaluation for undiagnosed mutation carriers. For the age‐at‐onset model (PD cases only), the association analysis tested whether variants contributed to the age‐at‐onset for PD cases among LRRK2 mutation carriers.For the penetrance model, a mixed effect Cox proportional hazard model (frailty model) was used with sex, 10 PCs, array, and cohort indicators as covariates. Family relationships were adjusted by using a kinship matrix calculated using R package COXME (https://cran.r-project.org/web/packages/coxme/index.html). For the age‐at‐onset model, a linear mixed model was fit with the same covariates as the penetrance model and a kinship matrix to adjust family relationships. Although adjusting 10 PCs and family relationships could minimize the effects of population stratifications and shared genetic and environment factors among family members, the use of this mixed samples was designed to search for common genetic variants that have the same effects in participants from all populations and could miss population‐specific or family specific findings. Variants with MAF > 1% were tested for association in these 2 models. In addition to the single variant analyses, we performed gene‐based association analyses for both the penetrance and age‐at‐onset models. We focused on rare exonic and splicing variants, based on annotations from Variant Effect Predictor (https://useast.ensembl.org/info/docs/tools/vep/index.html), and restricting to variants with MAF < 3%. Only genotyped variants (N = 725,802) were used in the gene‐based analyses due to the low quality of imputation for rare variants. The R package COXME was used to perform all analyses (https://cran.r-project.org/web/packages/coxme/index.html). Conditional analysis was conducted using the most significant variant in an associated region as a covariate, and additional signals within the associated region were determined based on p values < 0.01.

Genomewide Association Studies

Our association analysis tested two models: (1) variants modifying the penetrance for PD among LRRK2 mutation carriers (penetrance model), and (2) variants modifying the age‐at‐onset for PD among LRRK2 mutation carriers (age‐at‐onset model). For the penetrance model (including PD cases and those not diagnosed as PD at last evaluation), the association analysis was designed to identify variants associated with the time to PD diagnosis or last evaluation for undiagnosed mutation carriers. For the age‐at‐onset model (PD cases only), the association analysis tested whether variants contributed to the age‐at‐onset for PD cases among LRRK2 mutation carriers.

For the penetrance model, a mixed effect Cox proportional hazard model (frailty model) was used with sex, 10 PCs, array, and cohort indicators as covariates. Family relationships were adjusted by using a kinship matrix calculated using R package COXME (https://cran.r-project.org/web/packages/coxme/index.html). For the age‐at‐onset model, a linear mixed model was fit with the same covariates as the penetrance model and a kinship matrix to adjust family relationships. Although adjusting 10 PCs and family relationships could minimize the effects of population stratifications and shared genetic and environment factors among family members, the use of this mixed samples was designed to search for common genetic variants that have the same effects in participants from all populations and could miss population‐specific or family specific findings. Variants with MAF > 1% were tested for association in these 2 models. In addition to the single variant analyses, we performed gene‐based association analyses for both the penetrance and age‐at‐onset models. We focused on rare exonic and splicing variants, based on annotations from Variant Effect Predictor (https://useast.ensembl.org/info/docs/tools/vep/index.html), and restricting to variants with MAF < 3%. Only genotyped variants (N = 725,802) were used in the gene‐based analyses due to the low quality of imputation for rare variants. The R package COXME was used to perform all analyses (https://cran.r-project.org/web/packages/coxme/index.html). Conditional analysis was conducted using the most significant variant in an associated region as a covariate, and additional signals within the associated region were determined based on p values < 0.01.

Cell Culture, Co‐Immunoprecipitation, and Antibodies
To test whether the protein product of the identified gene interacts with LRRK2, we performed co‐immunoprecipitation analyses. HEK293FT (R70007; Invitrogen) cells were maintained as previously described
44
 and transiently transfected with pEGFP‐C1, pEGFP‐CORO1C, or co‐transfected with 3xFlag‐LRRK2 using Lipofectamine 2000 (Thermofisher Scientific) for 16 hours. Cells were washed once with ice cold PBS and lysed in 1 ml of GFP‐trap buffer (20 mM Tris–HCl pH 7.5, 150 mM NaCl, 1 mM EDTA, 50 mM NaF, 0.3% Triton X‐100, 5% Glycerol, Halt phosphatase inhibitor cocktail [Thermofisher Scientific], and protease inhibitor cocktail [Roche]) or Flag/GFP buffer (20 mM Tris–HCl pH 7.5, 300 mM NaCl, 1 mM EDTA, 50 mM NaF, 0.3% Triton X‐100, 5% Glycerol, Halt phosphatase inhibitor cocktail [Thermofisher Scientific], and protease inhibitor cocktail [Roche]) for 30 minutes on ice, followed by centrifugation at 4°C for 10 minutes at 13,900 g to obtain the supernatant. The supernatant was then incubated with either pre‐equilibrated GFP‐Trap agarose beads (ChromoTek) for 1 hour at 4°C on an end over end rotator to recover the GFP tag or pre‐cleared with Pierce Protein A/G Agarose (Thermofisher Scientific; 20 minutes at 4°C). Pre‐cleared lysates were incubated with 1 μg of monoclonal anti‐flag antibody (F3165; Millipore‐Sigma) or rabbit polyclonal anti‐GFP (Ab290; Abcam) for 2 hours at 4°C on an end over end rotator. Antibody complexes were captured by Protein A/G Agarose beads for 2 hours. Bead complexes were washed 6 times with lysis buffer, followed by elution in 1 times loading dye containing 2% β‐mercaptoethanol for 6 minutes at 95°C. All samples were resolved by Western blot as previously described.
45
 Each co‐immunoprecipitation was repeated in 2 to 3 independent experiments. Primary antibodies used in this study are as follows: mouse monoclonal anti‐FLAG M2 (F3165) was purchased from Thermofisher Scientific; Rabbit polyclonal to GFP was from Abcam (ab290), and anti‐LRRK2 was from UC Davis/NIH NeuroMab Facility (clone N241A/34).

Cell Culture, Co‐Immunoprecipitation, and Antibodies

To test whether the protein product of the identified gene interacts with LRRK2, we performed co‐immunoprecipitation analyses. HEK293FT (R70007; Invitrogen) cells were maintained as previously described
44
 and transiently transfected with pEGFP‐C1, pEGFP‐CORO1C, or co‐transfected with 3xFlag‐LRRK2 using Lipofectamine 2000 (Thermofisher Scientific) for 16 hours. Cells were washed once with ice cold PBS and lysed in 1 ml of GFP‐trap buffer (20 mM Tris–HCl pH 7.5, 150 mM NaCl, 1 mM EDTA, 50 mM NaF, 0.3% Triton X‐100, 5% Glycerol, Halt phosphatase inhibitor cocktail [Thermofisher Scientific], and protease inhibitor cocktail [Roche]) or Flag/GFP buffer (20 mM Tris–HCl pH 7.5, 300 mM NaCl, 1 mM EDTA, 50 mM NaF, 0.3% Triton X‐100, 5% Glycerol, Halt phosphatase inhibitor cocktail [Thermofisher Scientific], and protease inhibitor cocktail [Roche]) for 30 minutes on ice, followed by centrifugation at 4°C for 10 minutes at 13,900 g to obtain the supernatant. The supernatant was then incubated with either pre‐equilibrated GFP‐Trap agarose beads (ChromoTek) for 1 hour at 4°C on an end over end rotator to recover the GFP tag or pre‐cleared with Pierce Protein A/G Agarose (Thermofisher Scientific; 20 minutes at 4°C). Pre‐cleared lysates were incubated with 1 μg of monoclonal anti‐flag antibody (F3165; Millipore‐Sigma) or rabbit polyclonal anti‐GFP (Ab290; Abcam) for 2 hours at 4°C on an end over end rotator. Antibody complexes were captured by Protein A/G Agarose beads for 2 hours. Bead complexes were washed 6 times with lysis buffer, followed by elution in 1 times loading dye containing 2% β‐mercaptoethanol for 6 minutes at 95°C. All samples were resolved by Western blot as previously described.
45
 Each co‐immunoprecipitation was repeated in 2 to 3 independent experiments. Primary antibodies used in this study are as follows: mouse monoclonal anti‐FLAG M2 (F3165) was purchased from Thermofisher Scientific; Rabbit polyclonal to GFP was from Abcam (ab290), and anti‐LRRK2 was from UC Davis/NIH NeuroMab Facility (clone N241A/34).

In Silico Functional Studies
To evaluate whether the genomewide significant findings had immediate biological consequences on gene expression (expression quantitative trait locus [eQTL]) of nearby genes, we searched Open Targets Genetics (https://genetics.opentargets.org/) and GTEx (https://www.gtexportal.org/). In addition, protein–protein interaction (PPI) data were assessed to identify whether the protein product of the nominated gene either interacts directly with LRRK2 or has common interactors that are shared with LRRK2, using Protein Interaction Network Online Tool (PINOT) version 1.0
46
 queried on June 16, 2020 (http://www.reading.ac.uk/bioinf/PINOT/PINOT_form.html). We also performed chromatin interaction mapping to check whether our top findings interact with LRRK2 distantly. Functional Mapping and Annotation of GenomeWide Association Studies (FUMA: https://fuma.ctglab.nl/) was used to perform chromatin interaction mapping.
47
 Hi‐C data for chromatin interaction mapping were from Schmitt et al 2016 and Giusti‐Rodriguez et al 2019,
48
, 
49
 and are available in FUMA.

In Silico Functional Studies

To evaluate whether the genomewide significant findings had immediate biological consequences on gene expression (expression quantitative trait locus [eQTL]) of nearby genes, we searched Open Targets Genetics (https://genetics.opentargets.org/) and GTEx (https://www.gtexportal.org/). In addition, protein–protein interaction (PPI) data were assessed to identify whether the protein product of the nominated gene either interacts directly with LRRK2 or has common interactors that are shared with LRRK2, using Protein Interaction Network Online Tool (PINOT) version 1.0
46
 queried on June 16, 2020 (http://www.reading.ac.uk/bioinf/PINOT/PINOT_form.html). We also performed chromatin interaction mapping to check whether our top findings interact with LRRK2 distantly. Functional Mapping and Annotation of GenomeWide Association Studies (FUMA: https://fuma.ctglab.nl/) was used to perform chromatin interaction mapping.
47
 Hi‐C data for chromatin interaction mapping were from Schmitt et al 2016 and Giusti‐Rodriguez et al 2019,
48
, 
49
 and are available in FUMA.

Polygenic Risk Score Analyses
In the largest GWAS analysis of PD susceptibility to date, Nalls et al meta‐analyzed 17 datasets with 56,306 PD cases or proxy‐cases and 1.4 million controls.
4
 Based on their results, they developed a PRS using summary statistics of 1,805 variants that can explain 26% of PD heritability.
4
 In this study, we performed PRS analysis using these 1,805 variants. Detailed information about how to select these 1,805 variants was described in Nalls et al.
4
 Because we were searching for LRRK2 modifiers, variants in the LRRK2 region (chr12: 40,118,913‐41,263,086) were excluded. The PRS was calculated as a weighted summation of effective alleles with the logarithm of odds ratios as the weights. This derived PRS was used to fit the same models with the same set of covariates as described for the genomewide association analyses using the R package COXME.

Polygenic Risk Score Analyses

In the largest GWAS analysis of PD susceptibility to date, Nalls et al meta‐analyzed 17 datasets with 56,306 PD cases or proxy‐cases and 1.4 million controls.
4
 Based on their results, they developed a PRS using summary statistics of 1,805 variants that can explain 26% of PD heritability.
4
 In this study, we performed PRS analysis using these 1,805 variants. Detailed information about how to select these 1,805 variants was described in Nalls et al.
4
 Because we were searching for LRRK2 modifiers, variants in the LRRK2 region (chr12: 40,118,913‐41,263,086) were excluded. The PRS was calculated as a weighted summation of effective alleles with the logarithm of odds ratios as the weights. This derived PRS was used to fit the same models with the same set of covariates as described for the genomewide association analyses using the R package COXME.

ResultsStudy participants from the 3 cohorts are summarized in Table 1. In total, 1,879 participants (853 individuals from 294 families and 1,026 singletons) were included in the analyses. Among them, 776 had, or self‐reported, a PD diagnosis and 1,103 were not classified as affected with PD at the last evaluation. The majority of participants were G2019S carriers, only 4% carried other LRRK2 mutations as reported by the contributing sites, all from the MJFF consortium cohort. In the 23andMe cohort, 85% of participants were not diagnosed with PD and most of them were less than 50 years of age at the time of their last evaluation. Based on PCs, the majority of participants were of European ancestry.TABLE 1Summary of Study CohortsCohortsMJFF consortiumIsrael23andMeTotalNumber of participants
768

185

926

1,879
% PD cases (N)
67% (512)

66% (122)

15% (142)

41% (776)
% Females (N)
49% (378)

53% (98)

52% (480)

51% (956)
Mean age at last evaluation (SD) among non‐PD
56.2 (15.8)

53.6 (14.3)

45.9 (17.3)

48.7 (17.4)
Mean age at PD diagnosis (SD) among cases
56.9 (12.2)

57.5 (11.4)

59.4 (10)

57.4 (11.7)

LRRK2 mutation (% of total)

G2019S

699 (91%)

185 (100%)

926 (100%)

1810 (96%)

Non‐G2019S

69 (9%)

N/A

N/A

69 (4%)

Families
Number of participants with families (% total)
473 (61%)

96 (52%)

284 (31%)

853 (45%)
Total number of families
138

38

118

294
Average (max) family size
3.4 (17)

2.5 (4)

2.4 (10)

2.9
Average PD ratio in families
0.56

0.43

0.12

0.37
Ancestries (% of total)European ancestry91%100%89%91%
Ashkenazi Jewish
37%100%48%49%African‐American or Latinos ancestry9%0%11%9%MJFF = Michael J. Fox Foundation; N/A = not applicable; PD = Parkinson's disease.Manhattan plots for the single variant analyses of the penetrance and age‐at‐onset models are shown in Figure 1A and 1B. Q‐Q plots for both models are show in Figure 1C. No obvious bias was detected in either model; and genomic controls were 1.055 and 1.052 for the penetrance model and age‐at‐onset model, respectively. Twelve loci showed variants with p values < 1.0E‐6 (ie, meet the threshold for suggestive significance) in either the penetrance model or the age‐at‐onset model (Supplementary Table S1). One variant on chromosome 12 reached genomewide significance (rs77395454, p value = 2.5E‐08) in the penetrance model (Table 2, Supplementary Table S1). Conditional analysis suggested that there were no additional association signals in this locus. The top variant (rs77395454) on the chromosome 12 region is located in an intron of CORO1C (coronin 1C; Fig 2). The causal haplotype(s) spanned SELPLG, CORO1C, and SSH1, with most of the variants within CORO1C. Figure 3A–C shows the survival curves stratified by rs77395454 genotypes for all samples, familial samples, and unrelated samples, respectively. Heterozygous rs77395454 carriers (20 familial and 37 unrelated samples) had an increased risk of PD. Six other loci met suggestive significance (p value < 1.0E‐6) for the penetrance model (see Table 2).FIGURE 1Manhattan and Q‐Q plots of single variant analysis of penetrance and age‐at‐onset models. Y‐axis is the –log (p value) for associations. X‐axis is physical position of the variants across the genome. The horizontal line indicates genome‐wide significance. (A) Penetrance model; (B) age‐at‐onset model; and (C) Q‐Q plots of penetrance model (left) and age‐at‐onset model (right). [Color figure can be viewed at www.annalsofneurology.org]TABLE 2Variants That Have P Values < 1.0E‐6 in the Penetrance ModelCHRBPrsidAllelesGeneAnnotationMAFBETASE
p valueG2019S only p value1221,173,137rs141686162A/G
HLX, DUSP10
Intergenic
0.01

0.42

0.28
0.130.863124,083,400rs145611031C/G
KALRN
Intron
0.02

1.14

0.23
5.2E‐072.3E‐073140,288,373rs150382576A/G
CLSTN2
3'UTR
0.02

0.78

0.22
5.3E‐042.1E‐033152,841,926rs59679443A/G
RAP2B, ARHGEF26
Intergenic
0.04

0.62

0.17
3.1E‐041.6E‐043152,932,435rs16846845G/C
RAP2B, ARHGEF26
Intergenic
0.05

0.83

0.16
1.1E‐07
4.5E‐08
4160,854,320rs12272007A/G
LOC107986324
Intergenic
0.01

1.22

0.36
6.0E‐041.9E‐035115,786,384rs73781088C/T
SEMA6A
Intron
0.03

0.47

0.19
0.010.0589,520,115rs28398294G/A
TNKS
Intron
0.03

1.09

0.22
5.7E‐071.1E‐069127,532,973rs148922482C/T
NR6A1
Intron
0.01

0.90

0.28
1.4E‐033.7E‐0311120,585,515rs28470321G/A
GRIK4
Intron
0.01

1.91

0.36
9.0E‐086.1E‐08
12

109,080,567

rs77395454

C/T

CORO1C

Intron

0.02

1.27

0.23

2.5E‐08

1.0E‐06
1490,982,388rs76788674A/G
CALM1, TTC7B
Intergenic
0.03

0.78

0.16
7.1E‐072.8E‐06X123,652,525rs185981774A/G
TENM1
Intron
0.02

0.86

0.17
5.4E‐073.9E‐07Note: Genomewide significant variant is in bold.CHR = chromosome; MAF = minor allele frequency.FIGURE 2Regional association plot of the chromosome 12 region for the penetrance model. Y‐axis is the –log (p value) for associations. X‐axis denotes physical positions on the chromosome (Mb). The color scale shows the extent of linkage disequilibrium (LD; as measured by r2) between each variant and the top variant (indicated by the purple diamond) with larger r2 indicating greater LD. Peaks indicate the recombination hot spots. SNP = single nucleotide polymorphism. [Color figure can be viewed at www.annalsofneurology.org]FIGURE 3Cumulative incidence of PD stratified by rs77395454 genotypes. Dashed lines indicate 95% confidence interval. Due to the low MAF of rs77395454 and therefore the small number of CC genotype carriers, only participants with TT and CT genotypes are shown. (A) All samples; (B) familial samples; and (C) unrelated samples. MAF = minor allele frequency; SNP = single nucleotide polymorphism. [Color figure can be viewed at www.annalsofneurology.org]For the age‐at‐onset model, no chromosomal region reached genomewide significance, but 7 loci met the suggestive association threshold. Except for variants on chromosome 3 identified in both models, rs73781088 on chromosome 5 (intron of SEMA6A) for the age‐at‐onset model and rs28398284 on chromosome 8 (intron of TNKS) for the penetrance model, all other variants had no or marginal LD support. Variants on chromosome 3 from both models cover the same region but identified different haplotypes.For comparison purposes, we also performed analyses limited to only the LRRK2 G2019S carriers; overall the results were comparable (see Table 2, Supplementary Table S1) but rs16846845 on chromosome 3 (p value = 4.5E‐08) was genomewide significant for the penetrance model. In addition, we performed analyses using only individuals of predicted Ashkenazi Jewish ancestry. Results are less significant due to dramatically decreased sample sizes as shown in Supplementary Table S2. No genomewide significant results were detected in the gene‐based analysis using exonic variants for either model.We next sought evidence of a physical interaction between CORO1C and LRRK2 using co‐immunoprecipitation analyses. GFP‐tagged CORO1C or eGFP empty vector control were transfected into HEK293FT and lysates were incubated with anti‐GFP agarose beads to recover the GFP tag; relative to empty vector control, eGFP‐CORO1C co‐precipitated endogenous LRRK2 from HEK293FT cells (Fig 4A). To further support this interaction, 3xFlag‐tagged LRRK2 with either GFP‐tagged CORO1C or eGFP empty vector control were cotransfected into HEK293FT and, after 16 hours, lysates were incubated with anti‐GFP or anti‐Flag antibodies to immunoprecipitate the GFP or Flag tag, respectively. Immunoprecipitation of LRRK2 via the Flag antibody co‐precipitated eGFP‐CORO1C, but not the eGFP from the empty vector control. (Fig 4B). Reciprocally, immunoprecipitation of the GFP tag via a GFP antibody co‐precipitated Flag‐LRRK2 only in the presence of GFP‐CORO1C but not the empty vector GFP control (Fig 4C). Thus, co‐immunoprecipitation analysis of Flag‐LRRK2 and GFP‐CORO1C support an interaction between these 2 proteins.FIGURE 4Evidence of a LRRK2‐CORO1C complex. (A) Immunoprecipitation (IP) analysis of lysates from HEK293FT cells transiently expressing eGFP‐C1 empty vector control or eGFP‐CORO1C. IP with GFP‐Trap was followed with immunoblot analysis with an anti‐GFP (bottom panel) or an anti‐LRRK2 antibody (top panel) as indicated. Input denotes whole cell lysate material that was used for immunoprecipitation analysis. (B, C) Co‐immunoprecipitation analysis of lysates from HEK293FT cells transiently co‐expressing 3xflag (3FL) epitope‐tagged LRRK2 with either eGFP‐C1 empty vector control or eGFP‐CORO1C. Epitope‐tagged proteins were recovered from lysates using anti‐Flag (IP: Flag) or anti‐GFP (IP: GFP) antibodies, followed by immunoblot detection using anti‐Flag (top panels) and anti‐GFP (bottom panels) antibodies. Input denotes whole cell lysate starting material from HEK293FT cells used for immunoprecipitation analysis; +/− indicates cDNA transfections.By searching Open Targets Genetics and GTEx, we found that the most significant variant, rs77395454, is an eQTL of CORO1C in blood and MYO1H in visceral adipose (omentum) but not in any brain tissues. The minor allele (C allele) is associated with higher expression of CORO1C and MYO1H. We did not find any previous report that LRRK2 interacts with CORO1C or MYO1H directly, however, there are several proteins that are the common interactors of both LRRK2 and CORO1C: ABCE1, ACTR2, CDC42, DAPK1, MYO1C, RAC1, and TP53, as identified by using PINOT,
46
 and it remains to be determined whether the interaction of LRRK2 with CORO1C is within a single complex or dependent upon these common interacting proteins. Chromatin interaction mapping did not find any variant that interacts with LRRK2 distantly.Among those 1,805 variants that were obtained from the study of Nalls et al 2019,
4
 20 variants were not present in our datasets. An additional 27 variants were located in the LRRK2 region and were excluded, and 1,758 variants were included in the PRS calculation. The PRS was a significant predictor in the penetrance model (p value = 7.8E‐4) but not in the age‐at‐onset model (p value = 0.75). These results suggest that a high genetic risk of PD significantly increases the chance of developing PD among LRRK2 mutation carriers.

Study participants from the 3 cohorts are summarized in Table 1. In total, 1,879 participants (853 individuals from 294 families and 1,026 singletons) were included in the analyses. Among them, 776 had, or self‐reported, a PD diagnosis and 1,103 were not classified as affected with PD at the last evaluation. The majority of participants were G2019S carriers, only 4% carried other LRRK2 mutations as reported by the contributing sites, all from the MJFF consortium cohort. In the 23andMe cohort, 85% of participants were not diagnosed with PD and most of them were less than 50 years of age at the time of their last evaluation. Based on PCs, the majority of participants were of European ancestry.

Summary of Study Cohorts

MJFF = Michael J. Fox Foundation; N/A = not applicable; PD = Parkinson's disease.

Manhattan plots for the single variant analyses of the penetrance and age‐at‐onset models are shown in Figure 1A and 1B. Q‐Q plots for both models are show in Figure 1C. No obvious bias was detected in either model; and genomic controls were 1.055 and 1.052 for the penetrance model and age‐at‐onset model, respectively. Twelve loci showed variants with p values < 1.0E‐6 (ie, meet the threshold for suggestive significance) in either the penetrance model or the age‐at‐onset model (Supplementary Table S1). One variant on chromosome 12 reached genomewide significance (rs77395454, p value = 2.5E‐08) in the penetrance model (Table 2, Supplementary Table S1). Conditional analysis suggested that there were no additional association signals in this locus. The top variant (rs77395454) on the chromosome 12 region is located in an intron of CORO1C (coronin 1C; Fig 2). The causal haplotype(s) spanned SELPLG, CORO1C, and SSH1, with most of the variants within CORO1C. Figure 3A–C shows the survival curves stratified by rs77395454 genotypes for all samples, familial samples, and unrelated samples, respectively. Heterozygous rs77395454 carriers (20 familial and 37 unrelated samples) had an increased risk of PD. Six other loci met suggestive significance (p value < 1.0E‐6) for the penetrance model (see Table 2).

Manhattan and Q‐Q plots of single variant analysis of penetrance and age‐at‐onset models. Y‐axis is the –log (p value) for associations. X‐axis is physical position of the variants across the genome. The horizontal line indicates genome‐wide significance. (A) Penetrance model; (B) age‐at‐onset model; and (C) Q‐Q plots of penetrance model (left) and age‐at‐onset model (right). [Color figure can be viewed at www.annalsofneurology.org]

Variants That Have P Values < 1.0E‐6 in the Penetrance Model

Note: Genomewide significant variant is in bold.

CHR = chromosome; MAF = minor allele frequency.

Regional association plot of the chromosome 12 region for the penetrance model. Y‐axis is the –log (p value) for associations. X‐axis denotes physical positions on the chromosome (Mb). The color scale shows the extent of linkage disequilibrium (LD; as measured by r2) between each variant and the top variant (indicated by the purple diamond) with larger r2 indicating greater LD. Peaks indicate the recombination hot spots. SNP = single nucleotide polymorphism. [Color figure can be viewed at www.annalsofneurology.org]

Cumulative incidence of PD stratified by rs77395454 genotypes. Dashed lines indicate 95% confidence interval. Due to the low MAF of rs77395454 and therefore the small number of CC genotype carriers, only participants with TT and CT genotypes are shown. (A) All samples; (B) familial samples; and (C) unrelated samples. MAF = minor allele frequency; SNP = single nucleotide polymorphism. [Color figure can be viewed at www.annalsofneurology.org]

For the age‐at‐onset model, no chromosomal region reached genomewide significance, but 7 loci met the suggestive association threshold. Except for variants on chromosome 3 identified in both models, rs73781088 on chromosome 5 (intron of SEMA6A) for the age‐at‐onset model and rs28398284 on chromosome 8 (intron of TNKS) for the penetrance model, all other variants had no or marginal LD support. Variants on chromosome 3 from both models cover the same region but identified different haplotypes.

For comparison purposes, we also performed analyses limited to only the LRRK2 G2019S carriers; overall the results were comparable (see Table 2, Supplementary Table S1) but rs16846845 on chromosome 3 (p value = 4.5E‐08) was genomewide significant for the penetrance model. In addition, we performed analyses using only individuals of predicted Ashkenazi Jewish ancestry. Results are less significant due to dramatically decreased sample sizes as shown in Supplementary Table S2. No genomewide significant results were detected in the gene‐based analysis using exonic variants for either model.

We next sought evidence of a physical interaction between CORO1C and LRRK2 using co‐immunoprecipitation analyses. GFP‐tagged CORO1C or eGFP empty vector control were transfected into HEK293FT and lysates were incubated with anti‐GFP agarose beads to recover the GFP tag; relative to empty vector control, eGFP‐CORO1C co‐precipitated endogenous LRRK2 from HEK293FT cells (Fig 4A). To further support this interaction, 3xFlag‐tagged LRRK2 with either GFP‐tagged CORO1C or eGFP empty vector control were cotransfected into HEK293FT and, after 16 hours, lysates were incubated with anti‐GFP or anti‐Flag antibodies to immunoprecipitate the GFP or Flag tag, respectively. Immunoprecipitation of LRRK2 via the Flag antibody co‐precipitated eGFP‐CORO1C, but not the eGFP from the empty vector control. (Fig 4B). Reciprocally, immunoprecipitation of the GFP tag via a GFP antibody co‐precipitated Flag‐LRRK2 only in the presence of GFP‐CORO1C but not the empty vector GFP control (Fig 4C). Thus, co‐immunoprecipitation analysis of Flag‐LRRK2 and GFP‐CORO1C support an interaction between these 2 proteins.

Evidence of a LRRK2‐CORO1C complex. (A) Immunoprecipitation (IP) analysis of lysates from HEK293FT cells transiently expressing eGFP‐C1 empty vector control or eGFP‐CORO1C. IP with GFP‐Trap was followed with immunoblot analysis with an anti‐GFP (bottom panel) or an anti‐LRRK2 antibody (top panel) as indicated. Input denotes whole cell lysate material that was used for immunoprecipitation analysis. (B, C) Co‐immunoprecipitation analysis of lysates from HEK293FT cells transiently co‐expressing 3xflag (3FL) epitope‐tagged LRRK2 with either eGFP‐C1 empty vector control or eGFP‐CORO1C. Epitope‐tagged proteins were recovered from lysates using anti‐Flag (IP: Flag) or anti‐GFP (IP: GFP) antibodies, followed by immunoblot detection using anti‐Flag (top panels) and anti‐GFP (bottom panels) antibodies. Input denotes whole cell lysate starting material from HEK293FT cells used for immunoprecipitation analysis; +/− indicates cDNA transfections.

By searching Open Targets Genetics and GTEx, we found that the most significant variant, rs77395454, is an eQTL of CORO1C in blood and MYO1H in visceral adipose (omentum) but not in any brain tissues. The minor allele (C allele) is associated with higher expression of CORO1C and MYO1H. We did not find any previous report that LRRK2 interacts with CORO1C or MYO1H directly, however, there are several proteins that are the common interactors of both LRRK2 and CORO1C: ABCE1, ACTR2, CDC42, DAPK1, MYO1C, RAC1, and TP53, as identified by using PINOT,
46
 and it remains to be determined whether the interaction of LRRK2 with CORO1C is within a single complex or dependent upon these common interacting proteins. Chromatin interaction mapping did not find any variant that interacts with LRRK2 distantly.

Among those 1,805 variants that were obtained from the study of Nalls et al 2019,
4
 20 variants were not present in our datasets. An additional 27 variants were located in the LRRK2 region and were excluded, and 1,758 variants were included in the PRS calculation. The PRS was a significant predictor in the penetrance model (p value = 7.8E‐4) but not in the age‐at‐onset model (p value = 0.75). These results suggest that a high genetic risk of PD significantly increases the chance of developing PD among LRRK2 mutation carriers.

DiscussionTwo major unresolved questions in PD research are why some, but not all, LRRK2 mutation carriers develop PD, and why the age‐at‐onset is so variable in those that do. This work represents the first GWAS study to report LRRK2 modifiers of PD penetrance and age‐at‐onset. One variant on chromosome 12 reached genomewide significance in the penetrance model (rs77395454 in an intronic region of CORO1C). Several loci reached suggestive significance in either the penetrance model or the age‐at‐onset model. One region on chromosome 3 showed suggestive associations in both models and reached genomewide significance in penetrance model when focused on G2019S carriers only. PRS derived from a publicly available PD GWAS was a significant predictor of penetrance of PD among LRRK2 mutation carriers.The genomewide significant variant, rs77395454 on chromosome 12, is located in an intronic region of CORO1C. Our co‐immunoprecipitation experiments from HEK293 cells found LRRK2 interacted with CORO1C. In addition, there also are several proteins that are common interactors of both LRRK2 and CORO1C. Two of them, CDC42 and RAC1, have previously been validated as modifiers of LRRK2‐mediated neurite shortening (reviewed in Boon et al 2014),
50
 suggesting that both CORO1C and LRRK2 might have effects on the actin cytoskeleton. Furthermore, a recent APEX2 screen identified that CORO1C is physically proximate to LRRK2 in cells.
51
 Notably, the protein expression of Coro1c is significantly higher in Lrrk2 knockout mice in vivo, as shown by proteomics and validated by Western blotting.
52
 The accumulation of CORO1C in knockout mice might represent compensation for diminished LRRK2 function. The CORO1C protein is a member of the WD repeat protein family that has been implicated in signal transduction, gene regulation (https://www.ncbi.nlm.nih.gov/gene/23603). In a zebrafish model of spinal muscular atrophy, overexpression of CORO1C rescued the phenotype caused by SMN deficiency.
53
 Using mass spectrometry, Malty et al showed that the product of CORO1C interacts with mitochondrial proteins associated with neurodegeneration.
54
 Collectively, these complementary results support that CORO1C is a more likely functional interactor of LRRK2 and all of these warrant more in‐depth cell and in vitro studies, including mapping the domains of LRRK2 responsible for the interaction between LRRK2 and CORO1C. However, it is possible that other genes in this region may underpin the observed association. For example, the protein product of SSH1 regulates actin filament dynamics, which has been linked to LRRK2 mutations.
55
, 
56

SELPLG has been linked to neuropsychiatric disorders, such as conduct disorder.
57
 Further studies are needed to conclusively determine the gene(s) underlying the observed association.Multiple variants on chromosome 3 were supported by both models, although they identified different associated haplotypes. The most significant variants were rs16846845 in the penetrance model and rs150382576 in the age‐at‐onset model. Furthermore, rs16846845 reached genomewide significance in G2019S only analysis for penetrance model. This region is under a known linkage peak for PD (LOD = 2.5).
58
 In the study by Gao et al, 2 variants (rs902432 and rs755763) had LOD scores > 2 in different analysis models.
58
 These 2 variants are about 850 Kb upstream and 200 Kb downstream from variants identified in our study, respectively. This is consistent with our findings that top variants in either model, and variants in LD with them, were physically distinct from each other. A nearby region was also linked to PD (LOD = 3.6) in an Amish Parkinsonism pedigree linkage study performed by Lee et al.
59
 In both Gao et al and Lee et al linkage studies, no candidate genes were nominated due to the large size of the reported linkage regions.
58
, 
59
 Variants that we identified are not located in any gene and the nearest gene is RAP2B, a member of the RAS oncogene family. However, its role in PD is unknown.Rs73781088 on chromosome 5 is in the intronic region of SEMA6A, which is broadly expressed in the brain. This gene is associated with amyotrophic lateral sclerosis.
60
 Rs28398294 on chromosome 8 is in the intronic region of TNKS, which is also broadly expressed in the brain. This region has been linked to Alzheimer's disease.
61
 Rs141686162 on chromosome 1 is in an intergenic region near DUSP10, which has been associated with progressive supranuclear palsy in a recent study.
62
 All of these findings warrant further study to investigate their potential roles in modifying the effect of LRRK2 mutations.We also examined the variants previously reported as LRRK2 modifiers in other studies. Thirteen variants from 7 genes passed our QC (rs4273468 from BST1; rs2421947 from DNM3; rs1564282 from GAK; rs1052553, rs242562, and rs2435207 from MAPT; rs823144 from PARK16; rs11931074, rs1372525, rs181489, rs2583988, and rs356219 from SNCA; rs11578699 from VAMP4).
20
, 
21
, 
22
, 
23
, 
24
, 
25
, 
26
, 
27
, 
28
, 
29
, 
30
, 
31
, 
32
, 
34
, 
35
 Only 4 variants from 3 genes had p values < 0.05: rs823144 from PARK16 in the penetrance model (p value = 0.01); rs1564282 from GAK in both the penetrance (p value = 0.03) and age‐at‐onset models (p value = 7.1E‐03); rs2345207 (p value = 5.1E‐04); and rs1052553 (p value = 0.02) from MAPT in the penetrance model. Unfortunately, because some individuals in our study may have also been included in previous studies where these candidate genes were first reported, our findings do not represent independent replication. However, our results showed that previously reported variants in BST1, DNM3, SNCA, and VAMP4 were not replicated, and whether they are LRRK2 modifiers remains equivocal.The significant effect of the PRS in the penetrance model supports the polygenic nature of the LRRK2 modifiers (ie, there are many genetic variants each with a small effect that collectively have a significant effect on the risk of PD in LRRK2 mutation carriers). This result is in line with the recent analysis of Iwaki et al.
63
 In that study, a PRS was derived using 89 genomewide significant variants (some variants were also included in our PRS) identified in a PD GWAS of Nalls et al.
4
 Iwaki et al found that the PRS was significantly associated with LRRK2 G2019S penetrance. Potential overlap between the participants in our study and that of Iwaki et al means that the results of these studies do not represent independent replication. We did not detect a significant association in the age‐at‐onset model. One reason for this may be the smaller sample size (less than half of that in the penetrance model, only 776 affected from 1,879 total participants analyzed), and the resulting lack of statistical power. Another possible explanation is that the PRS was derived from a GWAS comparing PD cases and controls, and these risk‐associated genes/variants are not necessarily associated with age‐at‐onset. Note, some participants of our study were included in the study of Nalls et al.
4
 Although we were unable to directly check for overlapping samples and the results were potentially biased, the overlapping samples is at most 0.13% of the total sample in Nalls et al, therefore, our samples had a minimal influence on the weight estimation that was used to calculate the PRS.There are several limitations of this study. First, despite the effort to enroll as many participants as possible, the sample size of this study still resulted in only modest statistical power. With this sample size, assuming a linear model, for a variant with MAF 3%, a change of at least 6 years of age‐at‐onset can be detected with 80% power at a genomewide significant level. Second, to maximize the number of eligible studies to join this collaboration, our inclusion criteria was quite minimal. Although this approach dramatically increased the sample size, many potentially important covariates were not collected and could not be adjusted for in subsequent analyses. Third, approximately 96% of our participants were G2019S carriers. However, there are carriers of other LRRK2 mutations in the MJFF cohort. Although in a sensitivity analysis using only G2019S carriers, we observed similar effects for those top variants that we identified in both models, these mutations may have different effects that cannot be detected in the small number of carriers. Fourth, our study cohorts consisted of family participants and unrelated participants. Family history was not collected for every participant. Therefore, some unrelated individuals may be sporadic PD and have different penetrance from familial PD participants. Fifth, there was a lack of information on subjects with subtle signs of PD but who did not yet merit a diagnosis of PD. Sixth, although we included 10 PCs to adjust population stratifications, there may still exist fine‐scale population stratifications that cannot be detected by those 10 PCs thus could potentially cause false positive findings. Nevertheless, we detected a genomewide significant variant. We provide experimental data to show CORO1C and LRRK2 interact, and support that observation by proteomics literature. The PRS analysis suggested that there is unlikely to be one or several single LRRK2 modifiers, but similar to overall PD risk, penetrance of LRRK2 mutations is affected by multiple genetic variants. Given the significant therapeutic efforts underway to develop targets for patients with PD carrying LRRK2 mutations, further replication of these results is essential. Furthermore, the genetic variants identified in this study and the PRS evaluated in the LRRK2 mutation carriers, may be used in the future to make personalized prevention and treatment possible.

Two major unresolved questions in PD research are why some, but not all, LRRK2 mutation carriers develop PD, and why the age‐at‐onset is so variable in those that do. This work represents the first GWAS study to report LRRK2 modifiers of PD penetrance and age‐at‐onset. One variant on chromosome 12 reached genomewide significance in the penetrance model (rs77395454 in an intronic region of CORO1C). Several loci reached suggestive significance in either the penetrance model or the age‐at‐onset model. One region on chromosome 3 showed suggestive associations in both models and reached genomewide significance in penetrance model when focused on G2019S carriers only. PRS derived from a publicly available PD GWAS was a significant predictor of penetrance of PD among LRRK2 mutation carriers.

The genomewide significant variant, rs77395454 on chromosome 12, is located in an intronic region of CORO1C. Our co‐immunoprecipitation experiments from HEK293 cells found LRRK2 interacted with CORO1C. In addition, there also are several proteins that are common interactors of both LRRK2 and CORO1C. Two of them, CDC42 and RAC1, have previously been validated as modifiers of LRRK2‐mediated neurite shortening (reviewed in Boon et al 2014),
50
 suggesting that both CORO1C and LRRK2 might have effects on the actin cytoskeleton. Furthermore, a recent APEX2 screen identified that CORO1C is physically proximate to LRRK2 in cells.
51
 Notably, the protein expression of Coro1c is significantly higher in Lrrk2 knockout mice in vivo, as shown by proteomics and validated by Western blotting.
52
 The accumulation of CORO1C in knockout mice might represent compensation for diminished LRRK2 function. The CORO1C protein is a member of the WD repeat protein family that has been implicated in signal transduction, gene regulation (https://www.ncbi.nlm.nih.gov/gene/23603). In a zebrafish model of spinal muscular atrophy, overexpression of CORO1C rescued the phenotype caused by SMN deficiency.
53
 Using mass spectrometry, Malty et al showed that the product of CORO1C interacts with mitochondrial proteins associated with neurodegeneration.
54
 Collectively, these complementary results support that CORO1C is a more likely functional interactor of LRRK2 and all of these warrant more in‐depth cell and in vitro studies, including mapping the domains of LRRK2 responsible for the interaction between LRRK2 and CORO1C. However, it is possible that other genes in this region may underpin the observed association. For example, the protein product of SSH1 regulates actin filament dynamics, which has been linked to LRRK2 mutations.
55
, 
56

SELPLG has been linked to neuropsychiatric disorders, such as conduct disorder.
57
 Further studies are needed to conclusively determine the gene(s) underlying the observed association.

Multiple variants on chromosome 3 were supported by both models, although they identified different associated haplotypes. The most significant variants were rs16846845 in the penetrance model and rs150382576 in the age‐at‐onset model. Furthermore, rs16846845 reached genomewide significance in G2019S only analysis for penetrance model. This region is under a known linkage peak for PD (LOD = 2.5).
58
 In the study by Gao et al, 2 variants (rs902432 and rs755763) had LOD scores > 2 in different analysis models.
58
 These 2 variants are about 850 Kb upstream and 200 Kb downstream from variants identified in our study, respectively. This is consistent with our findings that top variants in either model, and variants in LD with them, were physically distinct from each other. A nearby region was also linked to PD (LOD = 3.6) in an Amish Parkinsonism pedigree linkage study performed by Lee et al.
59
 In both Gao et al and Lee et al linkage studies, no candidate genes were nominated due to the large size of the reported linkage regions.
58
, 
59
 Variants that we identified are not located in any gene and the nearest gene is RAP2B, a member of the RAS oncogene family. However, its role in PD is unknown.

Rs73781088 on chromosome 5 is in the intronic region of SEMA6A, which is broadly expressed in the brain. This gene is associated with amyotrophic lateral sclerosis.
60
 Rs28398294 on chromosome 8 is in the intronic region of TNKS, which is also broadly expressed in the brain. This region has been linked to Alzheimer's disease.
61
 Rs141686162 on chromosome 1 is in an intergenic region near DUSP10, which has been associated with progressive supranuclear palsy in a recent study.
62
 All of these findings warrant further study to investigate their potential roles in modifying the effect of LRRK2 mutations.

We also examined the variants previously reported as LRRK2 modifiers in other studies. Thirteen variants from 7 genes passed our QC (rs4273468 from BST1; rs2421947 from DNM3; rs1564282 from GAK; rs1052553, rs242562, and rs2435207 from MAPT; rs823144 from PARK16; rs11931074, rs1372525, rs181489, rs2583988, and rs356219 from SNCA; rs11578699 from VAMP4).
20
, 
21
, 
22
, 
23
, 
24
, 
25
, 
26
, 
27
, 
28
, 
29
, 
30
, 
31
, 
32
, 
34
, 
35
 Only 4 variants from 3 genes had p values < 0.05: rs823144 from PARK16 in the penetrance model (p value = 0.01); rs1564282 from GAK in both the penetrance (p value = 0.03) and age‐at‐onset models (p value = 7.1E‐03); rs2345207 (p value = 5.1E‐04); and rs1052553 (p value = 0.02) from MAPT in the penetrance model. Unfortunately, because some individuals in our study may have also been included in previous studies where these candidate genes were first reported, our findings do not represent independent replication. However, our results showed that previously reported variants in BST1, DNM3, SNCA, and VAMP4 were not replicated, and whether they are LRRK2 modifiers remains equivocal.

The significant effect of the PRS in the penetrance model supports the polygenic nature of the LRRK2 modifiers (ie, there are many genetic variants each with a small effect that collectively have a significant effect on the risk of PD in LRRK2 mutation carriers). This result is in line with the recent analysis of Iwaki et al.
63
 In that study, a PRS was derived using 89 genomewide significant variants (some variants were also included in our PRS) identified in a PD GWAS of Nalls et al.
4
 Iwaki et al found that the PRS was significantly associated with LRRK2 G2019S penetrance. Potential overlap between the participants in our study and that of Iwaki et al means that the results of these studies do not represent independent replication. We did not detect a significant association in the age‐at‐onset model. One reason for this may be the smaller sample size (less than half of that in the penetrance model, only 776 affected from 1,879 total participants analyzed), and the resulting lack of statistical power. Another possible explanation is that the PRS was derived from a GWAS comparing PD cases and controls, and these risk‐associated genes/variants are not necessarily associated with age‐at‐onset. Note, some participants of our study were included in the study of Nalls et al.
4
 Although we were unable to directly check for overlapping samples and the results were potentially biased, the overlapping samples is at most 0.13% of the total sample in Nalls et al, therefore, our samples had a minimal influence on the weight estimation that was used to calculate the PRS.

There are several limitations of this study. First, despite the effort to enroll as many participants as possible, the sample size of this study still resulted in only modest statistical power. With this sample size, assuming a linear model, for a variant with MAF 3%, a change of at least 6 years of age‐at‐onset can be detected with 80% power at a genomewide significant level. Second, to maximize the number of eligible studies to join this collaboration, our inclusion criteria was quite minimal. Although this approach dramatically increased the sample size, many potentially important covariates were not collected and could not be adjusted for in subsequent analyses. Third, approximately 96% of our participants were G2019S carriers. However, there are carriers of other LRRK2 mutations in the MJFF cohort. Although in a sensitivity analysis using only G2019S carriers, we observed similar effects for those top variants that we identified in both models, these mutations may have different effects that cannot be detected in the small number of carriers. Fourth, our study cohorts consisted of family participants and unrelated participants. Family history was not collected for every participant. Therefore, some unrelated individuals may be sporadic PD and have different penetrance from familial PD participants. Fifth, there was a lack of information on subjects with subtle signs of PD but who did not yet merit a diagnosis of PD. Sixth, although we included 10 PCs to adjust population stratifications, there may still exist fine‐scale population stratifications that cannot be detected by those 10 PCs thus could potentially cause false positive findings. Nevertheless, we detected a genomewide significant variant. We provide experimental data to show CORO1C and LRRK2 interact, and support that observation by proteomics literature. The PRS analysis suggested that there is unlikely to be one or several single LRRK2 modifiers, but similar to overall PD risk, penetrance of LRRK2 mutations is affected by multiple genetic variants. Given the significant therapeutic efforts underway to develop targets for patients with PD carrying LRRK2 mutations, further replication of these results is essential. Furthermore, the genetic variants identified in this study and the PRS evaluated in the LRRK2 mutation carriers, may be used in the future to make personalized prevention and treatment possible.

T.F. and P.C. contributed to the conception and design of the study. D.L., B.A., P.F., P.C., and T.F. drafted the text and prepared the figures. D.L., B.A., P.F., T.S., J.A., R.N.A., G.W.B., D.B., S.B., A.B., K.B., L.C., M.C., S.D., V.V.D., M.F., J.Trinh, T.G., S.G., E.G., C.K., A.E.L., J.W.L., J.L., T.L., K.M., C.M., E.R.M., C.Y.M., H.M., E.M., R.H.M., K.N., L.O., H.P., D.R., E.R., M.P.R., O.A.R., A.S., R.S., B.S., C.S., W.K.S., C.T., E.T., J.E.T., D.V., J.Trojanowski, R.U., J.M.V., N.P.V., Z.K.W., C.P.Z., A.M., N.G., A.O.U., and B.F. contributed to acquisition and analysis of data. All authors reviewed and approved the submission.

Potential Conflicts of InterestB.A., P.F., P.C., S.D., C.Y.M., and members of the 23andMe Research Team are current or former employees of 23andMe, Inc., and hold stock or stock options in 23andMe.

Potential Conflicts of Interest

B.A., P.F., P.C., S.D., C.Y.M., and members of the 23andMe Research Team are current or former employees of 23andMe, Inc., and hold stock or stock options in 23andMe.

Data AvailabilityAggregate‐level data included in this study will be made available to qualified investigators upon request. Investigators interested in receiving 23andMe data, either alone or in combination with data from other cohorts, will need to sign a Data Transfer Agreement with 23andMe that protects 23andMe research participant privacy, and should visit https://research.23andme.com/dataset-access/ to submit a request.

Data Availability

Aggregate‐level data included in this study will be made available to qualified investigators upon request. Investigators interested in receiving 23andMe data, either alone or in combination with data from other cohorts, will need to sign a Data Transfer Agreement with 23andMe that protects 23andMe research participant privacy, and should visit https://research.23andme.com/dataset-access/ to submit a request.

Supporting information
Supplemental Table S1 Variants that have p values < 1.0E‐6 in either penetrance model or age‐at‐onset model.Click here for additional data file.
Supplemental Table S2 Results of AJ ancestry sample only.Click here for additional data file.

Supporting information

Supplemental Table S1 Variants that have p values < 1.0E‐6 in either penetrance model or age‐at‐onset model.

Click here for additional data file.

Supplemental Table S2 Results of AJ ancestry sample only.

Click here for additional data file.
