---
pmid: '32197073'
title: De Novo Frameshift Variants in the Neuronal Splicing Factor NOVA2 Result in
  a Common C-Terminal Extension and Cause a Severe Form of Neurodevelopmental Disorder.
authors:
- Mattioli F
- Hayot G
- Drouot N
- Isidor B
- Courraud J
- Hinckelmann MV
- Mau-Them FT
- Sellier C
- Goldman A
- Telegrafi A
- Boughton A
- Gamble C
- Moutton S
- Quartier A
- Jean N
- Van Ness P
- Grotto S
- Nambot S
- Douglas G
- Si YC
- Chelly J
- Shad Z
- Kaplan E
- Dineen R
- Golzio C
- Charlet-Berguerand N
- Mandel JL
- Piton A
journal: Am J Hum Genet
year: '2020'
full_text_available: false
pmcid: PMC7118572
doi: 10.1016/j.ajhg.2020.02.013
---

# De Novo Frameshift Variants in the Neuronal Splicing Factor NOVA2 Result in a Common C-Terminal Extension and Cause a Severe Form of Neurodevelopmental Disorder.
**Authors:** Mattioli F, Hayot G, Drouot N, Isidor B, Courraud J, Hinckelmann MV, Mau-Them FT, Sellier C, Goldman A, Telegrafi A, Boughton A, Gamble C, Moutton S, Quartier A, Jean N, Van Ness P, Grotto S, Nambot S, Douglas G, Si YC, Chelly J, Shad Z, Kaplan E, Dineen R, Golzio C, Charlet-Berguerand N, Mandel JL, Piton A
**Journal:** Am J Hum Genet (2020)
**DOI:** [10.1016/j.ajhg.2020.02.013](https://doi.org/10.1016/j.ajhg.2020.02.013)
**PMC:** [PMC7118572](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7118572/)

## Abstract

1. Am J Hum Genet. 2020 Apr 2;106(4):438-452. doi: 10.1016/j.ajhg.2020.02.013.
Epub  2020 Mar 19.

De Novo Frameshift Variants in the Neuronal Splicing Factor NOVA2 Result in a 
Common C-Terminal Extension and Cause a Severe Form of Neurodevelopmental 
Disorder.

Mattioli F(1), Hayot G(1), Drouot N(1), Isidor B(2), Courraud J(1), Hinckelmann 
MV(1), Mau-Them FT(3), Sellier C(1), Goldman A(4), Telegrafi A(5), Boughton 
A(6), Gamble C(6), Moutton S(7), Quartier A(1), Jean N(7), Van Ness P(4), Grotto 
S(8), Nambot S(7), Douglas G(5), Si YC(5), Chelly J(9), Shad Z(10), Kaplan 
E(10), Dineen R(10), Golzio C(1), Charlet-Berguerand N(1), Mandel JL(11), Piton 
A(12).

Author information:
(1)Institut de Génétique et de Biologie Moléculaire et Cellulaire, Illkirch 
67400, France; Centre National de la Recherche Scientifique, UMR7104, Illkirch 
67400, France; Institut National de la Santé et de la Recherche Médicale, U964, 
Illkirch 67400, France; Université de Strasbourg, Illkirch 67400, France.
(2)Service de Génétique Médicale, CHU de Nantes, Nantes 44093, France.
(3)Laboratoire de Génétique Moléculaire, UF Innovation en diagnostic génomique 
des maladies rares, Plateau Technique de Biologie, Centre Hospitalier 
Universitaire de Dijon, Dijon 21070, France; INSERM U1231, LNC UMR1231 GAD, 
Burgundy University, Dijon 21070, France.
(4)Department of Neurology, Neurophysiology Section, Baylor College of Medicine, 
Houston, TX 77030, USA.
(5)GeneDx, Gaithersburg, MD 20877, USA.
(6)Cook Children's Genetics Fort Worth, TX 76104, USA.
(7)INSERM U1231, LNC UMR1231 GAD, Burgundy University, Dijon 21070, France; 
Centre de Génétique et Centre de référence "Anomalies du Développement et 
Syndromes Malformatifs," Hôpital d'Enfants, Centre Hospitalier Universitaire de 
Dijon, Dijon 21070, France.
(8)Service de Génétique Médicale, AP-HP Robert-Debré, Paris 75019, France.
(9)Institut de Génétique et de Biologie Moléculaire et Cellulaire, Illkirch 
67400, France; Centre National de la Recherche Scientifique, UMR7104, Illkirch 
67400, France; Institut National de la Santé et de la Recherche Médicale, U964, 
Illkirch 67400, France; Université de Strasbourg, Illkirch 67400, France; 
Laboratory of Genetic Diagnostic, Hôpitaux Universitaires de Strasbourg, 
Strasbourg 67000, France.
(10)Department of Genetics, University of Illinois College of Medicine, Chicago, 
IL 60607, USA.
(11)Institut de Génétique et de Biologie Moléculaire et Cellulaire, Illkirch 
67400, France; Centre National de la Recherche Scientifique, UMR7104, Illkirch 
67400, France; Institut National de la Santé et de la Recherche Médicale, U964, 
Illkirch 67400, France; Université de Strasbourg, Illkirch 67400, France; 
University of Strasbourg Institute of Advanced Studies, Strasbourg 67000, 
France.
(12)Institut de Génétique et de Biologie Moléculaire et Cellulaire, Illkirch 
67400, France; Centre National de la Recherche Scientifique, UMR7104, Illkirch 
67400, France; Institut National de la Santé et de la Recherche Médicale, U964, 
Illkirch 67400, France; Université de Strasbourg, Illkirch 67400, France; 
Laboratory of Genetic Diagnostic, Hôpitaux Universitaires de Strasbourg, 
Strasbourg 67000, France. Electronic address: piton@igbmc.fr.

The neuro-oncological ventral antigen 2 (NOVA2) protein is a major factor 
regulating neuron-specific alternative splicing (AS), previously associated with 
an acquired neurologic condition, the paraneoplastic opsoclonus-myoclonus ataxia 
(POMA). We report here six individuals with de novo frameshift variants in NOVA2 
affected with a severe neurodevelopmental disorder characterized by intellectual 
disability (ID), motor and speech delay, autistic features, hypotonia, feeding 
difficulties, spasticity or ataxic gait, and abnormal brain MRI. The six 
variants lead to the same reading frame, adding a common proline rich C-terminal 
part instead of the last KH RNA binding domain. We detected 41 genes 
differentially spliced after NOVA2 downregulation in human neural cells. The 
NOVA2 variant protein shows decreased ability to bind target RNA sequences and 
to regulate target AS events. It also fails to complement the effect on neurite 
outgrowth induced by NOVA2 downregulation in vitro and to rescue alterations of 
retinotectal axonal pathfinding induced by loss of NOVA2 ortholog in zebrafish. 
Our results suggest a partial loss-of-function mechanism rather than a full 
heterozygous loss-of-function, although a specific contribution of the novel 
C-terminal extension cannot be excluded.

Copyright © 2020 American Society of Human Genetics. Published by Elsevier Inc. 
All rights reserved.

DOI: 10.1016/j.ajhg.2020.02.013
PMCID: PMC7118572
PMID: 32197073 [Indexed for MEDLINE]

Conflict of interest statement: A.T., G.D., and Y.C.S. are employees of GeneDx.
