---
pmid: '31467278'
title: Maximizing binary interactome mapping with a minimal number of assays.
authors:
- Choi SG
- Olivet J
- Cassonnet P
- Vidalain PO
- Luck K
- Lambourne L
- Spirohn K
- Lemmens I
- Dos Santos M
- Demeret C
- Jones L
- Rangarajan S
- Bian W
- Coutant EP
- Janin YL
- van der Werf S
- Trepte P
- Wanker EE
- De Las Rivas J
- Tavernier J
- Twizere JC
- Hao T
- Hill DE
- Vidal M
- Calderwood MA
- Jacob Y
journal: Nat Commun
year: '2019'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC6715725
doi: 10.1038/s41467-019-11809-2
---

# Maximizing binary interactome mapping with a minimal number of assays.
**Authors:** Choi SG, Olivet J, Cassonnet P, Vidalain PO, Luck K, Lambourne L, Spirohn K, Lemmens I, Dos Santos M, Demeret C, Jones L, Rangarajan S, Bian W, Coutant EP, Janin YL, van der Werf S, Trepte P, Wanker EE, De Las Rivas J, Tavernier J, Twizere JC, Hao T, Hill DE, Vidal M, Calderwood MA, Jacob Y
**Journal:** Nat Commun (2019)
**DOI:** [10.1038/s41467-019-11809-2](https://doi.org/10.1038/s41467-019-11809-2)
**PMC:** [PMC6715725](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6715725/)

## Abstract

1. Nat Commun. 2019 Aug 29;10(1):3907. doi: 10.1038/s41467-019-11809-2.

Maximizing binary interactome mapping with a minimal number of assays.

Choi SG(1)(2)(3), Olivet J(1)(2)(3)(4), Cassonnet P(5), Vidalain PO(6), Luck 
K(1)(2)(3), Lambourne L(1)(2)(3), Spirohn K(1)(2)(3), Lemmens I(7)(8), Dos 
Santos M(5), Demeret C(5), Jones L(9), Rangarajan S(1)(2)(3), Bian W(1)(2)(3), 
Coutant EP(10), Janin YL(10), van der Werf S(5), Trepte P(11)(12), Wanker 
EE(11), De Las Rivas J(13), Tavernier J(7)(8), Twizere JC(4), Hao T(1)(2)(3), 
Hill DE(1)(2)(3), Vidal M(14)(15), Calderwood MA(16)(17)(18), Jacob Y(19)(20).

Author information:
(1)Center for Cancer Systems Biology (CCSB), Dana-Farber Cancer Institute 
(DFCI), 450 Brookline Avenue, Boston, MA, 02215, USA.
(2)Department of Genetics, Blavatnik Institute, Harvard Medical School (HMS), 77 
Avenue Louis Pasteur, Boston, MA, 02115, USA.
(3)Department of Cancer Biology, Dana-Farber Cancer Institute, 450 Brookline 
Avenue, Boston, MA, 02215, USA.
(4)Laboratory of Viral Interactomes, Unit of Molecular Biology of Diseases, 
Groupe Interdisciplinaire de Génomique Appliquée (GIGA Institute), University of 
Liège, 7 Place du 20 Août, 4000, Liège, Belgium.
(5)Département de Virologie, Unité de Génétique Moléculaire des Virus à ARN 
(GMVR), Institut Pasteur, UMR3569, Centre National de la Recherche Scientifique 
(CNRS), Université Paris Diderot, Sorbonne Paris Cité, 28 rue du Docteur Roux, 
75015, Paris, France.
(6)Équipe Chimie, Biologie, Modélisation et Immunologie pour la Thérapie 
(CBMIT), Laboratoire de Chimie et Biochimie Pharmacologiques et Toxicologiques 
(LCBPT), Centre Interdisciplinaire Chimie Biologie-Paris (CICB-Paris), UMR8601, 
CNRS, Université Paris Descartes, 45 rue des Saints-Pères, 75006, Paris, France.
(7)Center for Medical Biotechnology, Vlaams Instituut voor Biotechnologie (VIB), 
3 Albert Baertsoenkaai, 9000, Ghent, Belgium.
(8)Cytokine Receptor Laboratory (CRL), Department of Biomolecular Medicine, 
Faculty of Medicine and Health Sciences, Ghent University, 3 Albert 
Baertsoenkaai, 9000, Ghent, Belgium.
(9)Centre de Bioinformatique, Biostatistique et Biologie Intégrative (C3BI), 
Institut Pasteur, 28 rue du Docteur Roux, 75015, Paris, France.
(10)Département de Biologie Structurale et Chimie, Unité de Chimie et 
Biocatalyse, Institut Pasteur, UMR3523, CNRS, 28 rue du Docteur Roux, 75015, 
Paris, France.
(11)Neuroproteomics, Max Delbrück Center for Molecular Medicine, 10 
Robert-Rössle-Str., 13125, Berlin, Germany.
(12)Brain Development and Disease, Institute of Molecular Biotechnology of the 
Austrian Academy of Sciences (IMBA), 3 Dr. Bohr-Gasse, 1030, Vienna, Austria.
(13)Cancer Research Center (CiC-IBMCC, CSIC/USAL), Consejo Superior de 
Investigaciones Científicas (CSIC), University of Salamanca (USAL), Campus 
Miguel de Unamuno, 37007, Salamanca, Spain.
(14)Center for Cancer Systems Biology (CCSB), Dana-Farber Cancer Institute 
(DFCI), 450 Brookline Avenue, Boston, MA, 02215, USA. 
marc_vidal@dfci.harvard.edu.
(15)Department of Genetics, Blavatnik Institute, Harvard Medical School (HMS), 
77 Avenue Louis Pasteur, Boston, MA, 02115, USA. marc_vidal@dfci.harvard.edu.
(16)Center for Cancer Systems Biology (CCSB), Dana-Farber Cancer Institute 
(DFCI), 450 Brookline Avenue, Boston, MA, 02215, USA. 
michael_calderwood@dfci.harvard.edu.
(17)Department of Genetics, Blavatnik Institute, Harvard Medical School (HMS), 
77 Avenue Louis Pasteur, Boston, MA, 02115, USA. 
michael_calderwood@dfci.harvard.edu.
(18)Department of Cancer Biology, Dana-Farber Cancer Institute, 450 Brookline 
Avenue, Boston, MA, 02215, USA. michael_calderwood@dfci.harvard.edu.
(19)Center for Cancer Systems Biology (CCSB), Dana-Farber Cancer Institute 
(DFCI), 450 Brookline Avenue, Boston, MA, 02215, USA. yves.jacob@pasteur.fr.
(20)Département de Virologie, Unité de Génétique Moléculaire des Virus à ARN 
(GMVR), Institut Pasteur, UMR3569, Centre National de la Recherche Scientifique 
(CNRS), Université Paris Diderot, Sorbonne Paris Cité, 28 rue du Docteur Roux, 
75015, Paris, France. yves.jacob@pasteur.fr.

Complementary assays are required to comprehensively map complex biological 
entities such as genomes, proteomes and interactome networks. However, how 
various assays can be optimally combined to approach completeness while 
maintaining high precision often remains unclear. Here, we propose a framework 
for binary protein-protein interaction (PPI) mapping based on optimally 
combining assays and/or assay versions to maximize detection of true positive 
interactions, while avoiding detection of random protein pairs. We have 
engineered a novel NanoLuc two-hybrid (N2H) system that integrates 12 different 
versions, differing by protein expression systems and tagging configurations. 
The resulting union of N2H versions recovers as many PPIs as 10 distinct assays 
combined. Thus, to further improve PPI mapping, developing alternative versions 
of existing assays might be as productive as designing completely new assays. 
Our findings should be applicable to systematic mapping of other biological 
landscapes.

DOI: 10.1038/s41467-019-11809-2
PMCID: PMC6715725
PMID: 31467278 [Indexed for MEDLINE]

Conflict of interest statement: The authors declare no competing interests.

## Full Text

IntroductionComplex biological entities such as genomes, proteomes, and interactome networks require multiple, orthogonal approaches to be fully characterized. Even in the case of DNA, a highly homogenous macromolecule, sequencing and assembly of large genomes such as the human genome requires combinations of complementary methodologies including short-read sequencing as those provided by Illumina platforms, long-read DNA sequencing as in PacBio and/or Nanopore technologies, and HiC-related chromatin mapping tools1.Because polypeptides are more heterogeneous and structurally complex macromolecules than DNA2, determining protein features at the scale of the whole proteome is considerably more challenging than characterizing nucleic acid sequences. For instance, to expand the detectable portion of a proteome by mass-spectrometry, negative peptide ionization approaches are necessary to complement canonical positive-mode strategies3. Another example is protein crystallization that often requires hundreds of different conditions to be tested, including combining parameters such as ionic strength, temperature, pH, salt, protein concentrations, and others4. Structural proteomics, which determines protein structures at proteome-scale5 also requires a palette of technologies such as X-ray crystallography, nuclear magnetic resonance (NMR), and/or cryogenic electron microscopy (cryo-EM) to reasonably cover proteins and protein complexes of different sizes, solubilities, and flexibilities6,7.In short, the mapping of large macromolecular landscapes usually requires multiple technologies that can be combined to complement each other’s relative shortcomings. However, for most comprehensive mapping projects, it remains unknown how to optimally combine various experimental methods in order to achieve maximal detection and specificity by a minimal number of assays.Macromolecules including DNA, RNA, and proteins do not work in isolation but assemble into molecular machines, signaling cascades and metabolic pathways through interactions with other macromolecules, altogether forming complex interactome networks that are critical for biological systems8,9. As protein–protein interactions (PPIs) play a pivotal role in these networks, it is essential to comprehensively identify binary PPIs10–12 and co-complex protein associations13–15, both at the scale of signaling pathways16,17, molecular machines18,19 and network modules20, and at the scale of the whole proteome12.Although medium- to high-throughput binary PPI detection assays have been available for three decades, since the original description of the yeast two-hybrid (Y2H) system21, it remains impossible to completely map interactome networks using any assay in isolation. This is well illustrated by the fact that most binary PPI assays developed so far only detect about a third of well-described benchmark interactions from a positive reference set (PRS) under conditions that limit, but not necessarily completely prevent, the detection of negative control protein pairs from a random reference set (RRS)22–27. However, since different assays23 or assay versions28,29 often detect overlapping, yet distinct subsets of interactions, combining them should significantly increase overall detection.In order to comprehensively map binary PPIs both at the scale of signaling pathways and protein complexes and at the scale of whole proteomes, it should thus be possible to establish an optimal combination of assays and/or assay versions to capture nearly all interactions in a defined search space while ensuring high specificity (Fig. 1a). Our goal here was to systematically analyze experimental variables that influence binary PPI detection across multiple assays, and integrate such knowledge into the rational design of a binary PPI detection framework involving a minimal number of assays with maximal recovery.Fig. 1A framework to combine assays and maximize overall detection. a Combining multiple assays and/or assay versions to increase detection under maximum specificity. b A multi-dimensional space of experimental conditions for each binary PPI assay. c Comparing combinations of different, distinct assays (left) to combinations of multiple assay versions explored by the single, highly versatile NanoLuc 2-hybrid (N2H) system (right)We evaluated a class of binary PPI assays that are based on exogenously expressing pairs of hybrid proteins, collectively referred to as Tag1-X and Tag2-Y, where X and Y can be any protein, and Tag1 and Tag2 are two polypeptidic moieties, the physical proximity of which can be readily monitored. For example, in the Y2H system, X and Y interacting proteins restore physical proximity between the DNA binding domain (DB) and the activation domain (AD) of a transcription factor upon exogenous expression of DB-X and AD-Y hybrid proteins21,30.In addition to assessing different assays defined by specific tag pairs, we also evaluated assay version differences due to: (i) the relative orientation or swap of the tested proteins X and Y, differentiating between Tag1-X/Tag2-Y and Tag1-Y/Tag2-X; (ii) the tagging configurations where we distinguish protein fusions from either the N- or C-terminal extremities of X and Y, e.g. Tag1-X/Tag2-Y vs. X-Tag1/Y-Tag2; and (iii) the exogenous expression systems, either in yeast, in mammalian cells, or in vitro (cell-free) (Fig. 1b).To uniformly compare these different experimental conditions within a single assay, we have engineered a PPI detection technology based on the functional reconstitution of NanoLuc, a small 19 kDa, high-performance bioluminescent protein31. Using this NanoLuc two-hybrid (N2H) system in combination with highly versatile vectors capable of driving protein expression as N- or C-terminal fusions of assay tags in all three environments, we show that permuting various experimental conditions in a single assay can achieve similar PPI detection as through the continuous addition of fundamentally different techniques (Fig. 1c). Our results suggest that only a few versatile assays should be sufficient to achieve near saturation with maximal precision in medium- and high-throughput proteome-scale PPI mapping projects. This finding might also be of value for large-scale mapping of other complex biological landscapes.

Introduction

Complex biological entities such as genomes, proteomes, and interactome networks require multiple, orthogonal approaches to be fully characterized. Even in the case of DNA, a highly homogenous macromolecule, sequencing and assembly of large genomes such as the human genome requires combinations of complementary methodologies including short-read sequencing as those provided by Illumina platforms, long-read DNA sequencing as in PacBio and/or Nanopore technologies, and HiC-related chromatin mapping tools1.

Because polypeptides are more heterogeneous and structurally complex macromolecules than DNA2, determining protein features at the scale of the whole proteome is considerably more challenging than characterizing nucleic acid sequences. For instance, to expand the detectable portion of a proteome by mass-spectrometry, negative peptide ionization approaches are necessary to complement canonical positive-mode strategies3. Another example is protein crystallization that often requires hundreds of different conditions to be tested, including combining parameters such as ionic strength, temperature, pH, salt, protein concentrations, and others4. Structural proteomics, which determines protein structures at proteome-scale5 also requires a palette of technologies such as X-ray crystallography, nuclear magnetic resonance (NMR), and/or cryogenic electron microscopy (cryo-EM) to reasonably cover proteins and protein complexes of different sizes, solubilities, and flexibilities6,7.

In short, the mapping of large macromolecular landscapes usually requires multiple technologies that can be combined to complement each other’s relative shortcomings. However, for most comprehensive mapping projects, it remains unknown how to optimally combine various experimental methods in order to achieve maximal detection and specificity by a minimal number of assays.

Macromolecules including DNA, RNA, and proteins do not work in isolation but assemble into molecular machines, signaling cascades and metabolic pathways through interactions with other macromolecules, altogether forming complex interactome networks that are critical for biological systems8,9. As protein–protein interactions (PPIs) play a pivotal role in these networks, it is essential to comprehensively identify binary PPIs10–12 and co-complex protein associations13–15, both at the scale of signaling pathways16,17, molecular machines18,19 and network modules20, and at the scale of the whole proteome12.

Although medium- to high-throughput binary PPI detection assays have been available for three decades, since the original description of the yeast two-hybrid (Y2H) system21, it remains impossible to completely map interactome networks using any assay in isolation. This is well illustrated by the fact that most binary PPI assays developed so far only detect about a third of well-described benchmark interactions from a positive reference set (PRS) under conditions that limit, but not necessarily completely prevent, the detection of negative control protein pairs from a random reference set (RRS)22–27. However, since different assays23 or assay versions28,29 often detect overlapping, yet distinct subsets of interactions, combining them should significantly increase overall detection.

In order to comprehensively map binary PPIs both at the scale of signaling pathways and protein complexes and at the scale of whole proteomes, it should thus be possible to establish an optimal combination of assays and/or assay versions to capture nearly all interactions in a defined search space while ensuring high specificity (Fig. 1a). Our goal here was to systematically analyze experimental variables that influence binary PPI detection across multiple assays, and integrate such knowledge into the rational design of a binary PPI detection framework involving a minimal number of assays with maximal recovery.Fig. 1A framework to combine assays and maximize overall detection. a Combining multiple assays and/or assay versions to increase detection under maximum specificity. b A multi-dimensional space of experimental conditions for each binary PPI assay. c Comparing combinations of different, distinct assays (left) to combinations of multiple assay versions explored by the single, highly versatile NanoLuc 2-hybrid (N2H) system (right)

A framework to combine assays and maximize overall detection. a Combining multiple assays and/or assay versions to increase detection under maximum specificity. b A multi-dimensional space of experimental conditions for each binary PPI assay. c Comparing combinations of different, distinct assays (left) to combinations of multiple assay versions explored by the single, highly versatile NanoLuc 2-hybrid (N2H) system (right)

We evaluated a class of binary PPI assays that are based on exogenously expressing pairs of hybrid proteins, collectively referred to as Tag1-X and Tag2-Y, where X and Y can be any protein, and Tag1 and Tag2 are two polypeptidic moieties, the physical proximity of which can be readily monitored. For example, in the Y2H system, X and Y interacting proteins restore physical proximity between the DNA binding domain (DB) and the activation domain (AD) of a transcription factor upon exogenous expression of DB-X and AD-Y hybrid proteins21,30.

In addition to assessing different assays defined by specific tag pairs, we also evaluated assay version differences due to: (i) the relative orientation or swap of the tested proteins X and Y, differentiating between Tag1-X/Tag2-Y and Tag1-Y/Tag2-X; (ii) the tagging configurations where we distinguish protein fusions from either the N- or C-terminal extremities of X and Y, e.g. Tag1-X/Tag2-Y vs. X-Tag1/Y-Tag2; and (iii) the exogenous expression systems, either in yeast, in mammalian cells, or in vitro (cell-free) (Fig. 1b).

To uniformly compare these different experimental conditions within a single assay, we have engineered a PPI detection technology based on the functional reconstitution of NanoLuc, a small 19 kDa, high-performance bioluminescent protein31. Using this NanoLuc two-hybrid (N2H) system in combination with highly versatile vectors capable of driving protein expression as N- or C-terminal fusions of assay tags in all three environments, we show that permuting various experimental conditions in a single assay can achieve similar PPI detection as through the continuous addition of fundamentally different techniques (Fig. 1c). Our results suggest that only a few versatile assays should be sufficient to achieve near saturation with maximal precision in medium- and high-throughput proteome-scale PPI mapping projects. This finding might also be of value for large-scale mapping of other complex biological landscapes.

ResultsCombining binary PPI assays under maximized specificityDifferent binary PPI assays often detect distinct subsets of interactions23, thus combining several assays should increase the overall recovery in a given search space of the proteome (Fig. 1a). However, what remains unclear is how binary PPI assay combinations affect the overall specificity, i.e. the fraction of true interactions detected in a given search space over the fraction of spurious pairs reported.Ten years ago, our group introduced an empirical framework to benchmark binary PPI assays based on: (i) a list of ~100 well-documented positive control PPIs, constituting the first version of a PRS, or hsPRS-v1, (ii) a list of ~100 pairs of proteins randomly picked among the ~2 x 108 pairwise combinations of human proteins, constituting version 1 of a RRS, or hsRRS-v1;22 and (iii) the Gateway cloning of all open reading frames (ORFs) involved, allowing identical constructs to be used across many different assays32. We and others have since cross-examined the following binary PPI assays against each other: (i) Y2H23, (ii) yellow fluorescent protein–protein complementation assay (YFP-PCA)23, (iii) mammalian protein–protein interaction trap (MAPPIT)23, (iv) well-based nucleic acid programmable protein arrays (wNAPPA)23, (v) luminescence-based mammalian interactome (LUMIER)23, (vi) kinase substrate sensor (KISS)25, (vii) Gaussia princeps complementation assay (GPCA)24,33, (viii) dual luminescence-based co-immunoprecipitation (DULIP)26, and (ix) bioluminescence-based two-hybrid (LuTHy)27, combining bioluminescence resonance energy transfer (LuTHy-BRET) and luminescence-based co-precipitation (LuTHy-LuC) assays (Fig. 1c, left panel).Under the various conditions used in these studies, each of the corresponding 10 assays, representing the union of only one or two version(s) (Supplementary Table 1), recovered between 21 and 39% of hsPRS-v1 pairs, with the negative control hsRRS-v1 detection ranging between 0 and 4% (Supplementary Fig. 1a). However, to directly compare the different assays side-by-side and combine them homogeneously, an identical RRS threshold should be applied to each individual assay. After applying an identical RRS cutoff, increasing from 1 to 10%, to each of the above-mentioned assays, we observed that the union of the ten different assays detected between 70% and 91% of positive control hsPRS-v1 pairs with 8% and 59% of random hsRRS-v1 pairs scoring positive, respectively (Supplementary Fig. 1b). Therefore, to avoid accumulation of random pairs affecting the quality of future analyses, each assay should be calibrated such that none of the RRS pairs are scored positive. After reanalyzing published data under these conditions, the recovered fractions of hsPRS-v1 pairs ranged from 3 to 33% for individual assays performed in only one or two version(s) (Fig. 2a), while the cumulative detection reached 65% when all assays were combined (Fig. 2b and Supplementary Fig. 1c). This suggested that many different assays, each associated with specific tag pairs and plasmids, would be needed to reach full coverage.Fig. 2Recovery of positive control PRS pairs by multiple assays or assay versions. a Fractions of hsPRS-v1 pairs recovered by ten well-established assays when an identical threshold of no hsRRS-v1 pairs scoring positive is applied. b Cumulative hsPRS-v1 recovery rates when assays in (a) are combined (individual data points are displayed in Supplementary Fig. 1c). c Fractions of hsPRS-v2 pairs recovered by different versions of GPCA, KISS, MAPPIT, Y2H and NanoBiT when an identical threshold of no hsRRS-v2 pairs scoring positive is applied. Horizontal shaded blue area indicates the average recovery rate ± S.D. for GPCA, KISS, MAPPIT and Y2H versions. d Cumulative hsPRS-v2 recovery rates when versions of GPCA, KISS, MAPPIT and Y2H presented in (c) are combined (individual data points are displayed in Supplementary Fig. 2d). Error bars indicate standard errors of proportions (a, c) or standard deviations (b, d)Benchmarking binary PPI assays and assay versionsWe considered the possibility that the failure of the 10 assays described above (Fig. 2a, b) to detect nearly all PRS interactions might be due to ORF clone quality issues and/or problematic interaction annotations. To avoid such possibilities, we re-examined all protein pairs from the original reference sets: hsPRS-v1 and hsRRS-v123. As a first step, only fully sequence-verified, full-length clonal ORFs were isolated (Supplementary Fig. 2a). Resulting pairs were then filtered based on updated literature information and annotations for binary PPIs (Supplementary Table 2, Supplementary Data 1). Finally, 60 PRS and 78 RRS pairs were assembled into a second-generation of reference sets: hsPRS-v2 and hsRRS-v2, respectively (see Methods for details and Supplementary Fig. 2b). After repeating four assays that were originally benchmarked against hsPRS-v1 and hsRRS-v1 with the second generation of reference sets, the cumulative PRS recovery rate reached 50 and 58% with -v1 and -v2 sets respectively, when none of the RRS pairs were scored positive (Supplementary Fig. 2c).As the failure of 10 well-established assays to detect nearly all binary PPIs did not arise from problems with the original reference sets, we decided to evaluate whether expanding the number of experimental conditions, or assay versions (Fig. 1b), covered by any single technique could increase its coverage and ultimately reduce the number of assays required. To investigate this, we benchmarked several versions of Y2H, MAPPIT, KISS, and GPCA against the second-generation of reference sets, hsPRS-v2 and hsRRS-v2. These four assays and corresponding versions (10 in total) were implemented as summarized in Supplementary Table 1. In particular, we constructed Gateway-compatible vectors for GPCA to cover all four possibilities of tagging configurations with this assay, i.e. Tag1-X/Tag2-Y, X-Tag1/Tag2-Y, Tag1-X/Y-Tag2, and X-Tag1/Y-Tag2 or, simply, N1N2, C1N2, N1C2, and C1C2, respectively. Between 8 and 37% of hsPRS-v2 pairs were detected by the 10 versions of GPCA, MAPPIT, KISS and Y2H when none of the hsRRS-v2 pairs were scored positive (Fig. 2c), which represented an average PRS detection rate of 22.8 ± 8.8% (S.D.) (Fig. 2c, shaded blue area).When we considered all protein pairs identified by the union of these 10 versions corresponding to four assays, the hsPRS-v2 recovery rate reached 63% (Fig. 2d and Supplementary Fig. 2d), similar to what was obtained by combining ten distinct assays, performed in only one or two version(s), as published in the literature (65%; Fig. 2b). This result demonstrates that the combination of four distinct assays exploring different experimental conditions can recover as many PPIs as 10 fundamentally different techniques limited to a few versions.Toward developing a highly versatile binary PPI assayIn an attempt to further minimize the overall number of multi-version assays required to maximize binary PPI detection and, by extension, reduce the associated cloning efforts and lab resources, we decided to develop a highly versatile binary PPI detection system opening access to all assay versions defined by the three experimental parameters presented in Fig. 1b.Although after reanalyzing published data, 1.3-fold and 1.4-fold increases in PRS detection were observed when switching from one orientation to the other (i.e. Tag1-X/Tag2-Y to Tag1-Y/Tag2-X), or from one configuration to another (i.e. N1N2 to C1N2), respectively (Supplementary Fig. 3a, b), the impact of permuting expression systems remained unclear as no assay had been used to systematically explore this parameter. However, several high-quality PPI datasets have been obtained using heterologous expression of protein pairs in yeast12,34, in mammalian cells35,36 and via cell-free coupled transcription-translation systems in vitro37,38.After surveying existing binary interaction assays, it appeared that split-protein reconstitution was the most suitable technology to cover the three experimental parameters presented in Fig. 1b. Indeed, compared to other binary PPI methods involving cell-specific responses or antibody-based captures, split-protein-based assays enable detection of PPIs through the simple reassembly of a functional reporter protein (Fig. 1c). Among the different reporters available today, NanoLuc, a highly stable 19 kDa protein derived from the luciferase complex of the deep-sea shrimp O. gracilirostris, is gaining increasing attention for the development of binary PPI detection assays, as highlighted by recent publications31,39–41. Following the single-step addition of imidazopyrazinone substrates, NanoLuc instantly produces an intense and sustainable glow-type luminescence with a specific activity about 100-fold greater than that of other luciferases, allowing its detection in vitro as well as in living cells and animals31,42,43. Therefore, split-NanoLuc-based technology appears to be the ideal candidate for the high-throughput detection of binary PPIs under various experimental conditions.Among the recent studies reporting a split-NanoLuc binary PPI assay, only one extensively fragmented the enzyme to test different tag pairs in an unbiased way (Supplementary Table 3)39. After selecting a cleavage site between amino acids 156 and 157, the authors heavily modified the corresponding fragments to reduce their association affinity. The resulting moieties, 11S (156 amino acids) and 114 (11 amino acids), were then used to develop the NanoLuc Binary Technology (NanoBiT) assay39.Although NanoBiT was originally designed for expression of protein pairs in mammalian cells, we decided to evaluate whether fragments 11S and 114 could be used to further develop our new, highly versatile binary PPI detection system (Fig. 1c, right panel). To investigate this, we first adapted the available NanoBiT vectors with the reported original split site39 to achieve different tagging configurations with Gateway-cloning compatibility and then benchmarked all four pairing combinations, i.e. N1N2, C1N2, N1C2, and C1C2, against hsPRS-v2 and hsRRS-v2. The benchmarking of these four NanoBiT versions in conditions where none of the hsRRS-v2 pairs are scored positive showed that between 3 and 25% of hsPRS-v2 pairs could be recovered (Fig. 2c). These results revealed optimal performances when the larger NanoBiT fragment, 11S (156 amino acids), was fused to the C-terminal extremity of the tested proteins, as demonstrated by PRS recovery rates of 22% and 25% for the C1N2 and C1C2 configurations, respectively, similar to those from four other well-established assays (Fig. 2c). These observations were in agreement with the author’s original assumption that the C-terminus tagging of proteins with 11S might impose minimal steric constraints39.Nanoluc 2-Hybrid (N2H): a highly versatile binary PPI assayAlthough NanoBiT appears to be a powerful tool to study PPI kinetics in live cells, the comprehensive benchmarking of this assay revealed two underperforming versions, N1N2 and N1C2, that might be associated with the inherent properties of the selected tag pair39. Therefore, we decided to empirically define a new split site for NanoLuc in an unbiased way. To do this, we first extensively fragmented the full-length enzyme by transposon-based insertional mutagenesis (Supplementary Fig. 4a). Complementation of the resulting fragments, F1 and F2 (58 pairs in total), was then systematically tested using a proximity assay in vitro (Supplementary Fig. 4b), and a cleavage site between amino acids 65 and 66 was selected (Fig. 3a) to build the NanoLuc 2-hybrid (N2H) assay (Fig. 1c, right panel; Supplementary Fig. 4c).Fig. 3Development of the highly versatile N2H assay. a Determination of a new split site for NanoLuc. Dark blue bar and red arrow indicate the selected NanoLuc fragments. RLU relative luminescence units, NLuc NanoLuc. Blank control corresponds to reagents without any NanoLuc fragment. (b) Tripartite promoter and specific activities of selected N2H fragments, F1 and F2, normalized to full-length NanoLuc when expressed in mammalian cells (mN2H; orange), in yeast (yN2H; purple), or in vitro (vN2H; gray), from the same plasmid. For “empty”: no protein or fragment was expressed. CMV cytomegalovirus, HSV-TK herpes simplex virus-thymidine kinase, UAS upstream activating sequence, IRES internal ribosome entry site. Red arrow displayed on NanoLuc structure (PDB: 5IBO) indicates the cleavage area. c Reproducibility of N2H assay in three expression environments (mN2H, yN2H, and vN2H) for the N1N2 tagging configuration. Each triangle in the different graphs represents a single tested protein pair. d Normalized luminescence signals for 12 versions of N2H benchmarked against hsPRS-v2 and hsRRS-v2. Blue squares and purple circles indicate PRS and RRS pairs, respectively. The threshold of no RRS pair scoring positive is indicated (horizontal line) for each assay version. e Fraction of hsPRS-v2 pairs recovered by 12 N2H assay versions when none of the hsRRS-v2 pairs are scored positive, corresponding to the thresholds displayed in (d). Horizontal shaded blue area indicates the average recovery rate ± S.D. for all assay versions reported in Fig. 2c. f Cumulative hsPRS-v2 recovery rates when versions of N2H presented in (e) are combined (individual data points are displayed in Supplementary Fig. 5a). Error bars indicate standard deviations (b, f) or standard errors of proportions (e)In order to comprehensively cover the experimental parameters presented in Fig. 1b with the single N2H assay, we constructed four Gateway-compatible plasmids from which N2H fragments, F1 (amino acids 1–65) and F2 (amino acids 66–171), can be expressed as either N- or C-terminal fusions of any pair of proteins X and Y in yeast, in mammalian cells or in vitro (cell-free). To drive expression in all three environments with each individual plasmid, we engineered a tripartite promoter of ~1 kb that can easily be introduced into any vector of interest (Fig. 3b). The performance of this 3-in-1 promoter was evaluated by measuring the expression of full-length and fragmented NanoLuc in yeast (yN2H), in mammalian cells (mN2H) and in vitro (vN2H). We obtained intense signals ranging between 108 and 109 relative luminescence units (RLU) when the full-length NanoLuc was expressed in each individual environment. Furthermore, compared to the full-length enzyme, individually expressed NanoLuc fragments did not produce any significant signal. More importantly, the two fragments expressed together did not generate any strong self-assembly luminescence that could interfere with the accurate measurement of binary PPIs. Taken together, these results demonstrate the full functionality of both the tripartite promoter and the selected pair of NanoLuc fragments (Fig. 3b).By combining NanoLuc fragments with the tripartite promoter and the Gateway-cloning technology into four uniform expression vectors (pDEST-N2H-N1, -N2, -C1, and -C2), all 24 assay versions (4x3x2) presented in Fig. 1b can thus be implemented with the single N2H assay. The Gateway-compatibility of N2H destination plasmids also renders the entire human ORFeome collection of over 18,000 entry clones44 directly amenable to this assay.As N2H was the first binary PPI detection tool in which proteins could be expressed as either N- or C-terminal fusions of protein tags in yeast (yN2H), in mammalian cells (mN2H) or in vitro (vN2H) from the same Gateway-compatible destination vector (Figs. 1b, c, 3b), it was important to benchmark the robustness of PPI-mediated readouts in each of the three environments. Using hsPRS-v2 and hsRRS-v2 pairs, we compared bioluminescence signals between two independently replicated experiments in yN2H, mN2H, and vN2H. As highlighted by the R2 > 0.97, a robust reproducibility was observed in each case when all 138 pairs from hsPRS-v2 and hsRRS-v2 were tested. In addition, a large dynamic range of luminescence readout, between 102 and 107 RLU, was consistently observed in each environment (Fig. 3c).To compare the performance of N2H assay versions in detecting reference binary PPIs over random protein pairs, we normalized luminescence signals and separated hsPRS-v2 from hsRRS-v2. No major differences were observed in the overall luminescence ratios as a fraction of PRS pairs always generated higher signals than that from random pairs in each case (Fig. 3d). As observed for C1C2-mN2H or C1C2-yN2H, for example, the only differences in PRS detection rates between assay versions came from one or a few RRS pairs that generated higher luminescence signals than the average. These version-specific artifacts ultimately reduced the fraction of PRS pairs recovered as their corresponding thresholds of no RRS pair scoring positive increased (Fig. 3d). The same observation was also made for well-established assays (Fig. 2a, c). Thus, in conditions where none of the RRS pairs are scored positive (threshold displayed for each individual version in Fig. 3d), the 12 tested N2H assay versions detected between 7 and 32% of hsPRS-v2 pairs (Fig. 3e), demonstrating similar performances compared to five other PPI detection methods (Fig. 2c) that averaged 20.4 ± 9.9% (S.D.) PRS recovery (Fig. 3e, shaded blue area). When these 12 N2H versions were combined, a PRS detection of 60% was yielded (Fig. 3f and Supplementary Fig. 5a), similar to the 65% obtained by combining either 10 different, version-limited assays from the literature23–28 (Fig. 2a, b) or five non-N2H, multi-version assays (Fig. 2c and Supplementary Fig. 5b).These results highlight the advantages of using a single, highly versatile PPI detection tool, such as N2H, to reduce the overall cloning efforts and lab resources (Fig. 4a–c), as the same set of expression vectors are used to explore several assay versions at a time (Fig. 4d) whilst recovering as many PPIs as a combination of many different techniques (Fig. 4e). The published assays in Fig. 4 are presented in Fig. 2a and Supplementary Table 1.Fig. 4Invested resources for various combinations of binary PPI assays and PRS detection rates. Numbers of (a) assays, (b) plasmids, (c) clonings per pair, and (d) tested versions required to achieve the PRS recovery rates presented in (e). PRS recovery rates presented in Fig. 4e correspond to those found in Fig. 2b (published), Supplementary Fig. 5b (non-N2H) and Fig. 3f (N2H). Except for Y2H-N1C2, Y2H-C1C2, and Y2H-C1N2 for which unambiguous titrations could not be obtained from the original study28, all published assay versions benchmarked against hsPRS-v1 and hsRRS-v1 were used (Fig. 2a, Supplementary Table 1)Combining multi-version assays to maximize PPI detectionWith all assays in hand to systematically evaluate the impact of the three variables presented in Fig. 1b, we decided to combine the different PPI assay versions previously tested against the improved reference sets (Figs. 2c, 3e) with respect to each individual parameter. All analyses were conducted in conditions where none of the random pairs are scored positive.As the impact of permuting proteins X and Y from one tag to the other (Tag1-X:Tag2-Y vs. Tag1-Y:Tag2-X) had already been comprehensively investigated by six different assays in the literature23,26,27 (Supplementary Table 1), we only repeated Y2H and MAPPIT to explore this parameter. While a single, randomly picked orientation yielded PRS recovery rates of 19% and 32%, switching to the second orientation increased detection to 25% and 40% for Y2H and MAPPIT, respectively. Importantly, cumulative PPI detection increased from 38% to 47% when a single or two orientations were combined for both assays, respectively (Fig. 5a). This 1.2-fold (47/38) increase was consistent with our analysis of published results23,26,27 (Supplementary Fig. 3a) and confirmed that permuting the orientation of tested proteins is a valuable option to increase PPI detection.Fig. 5Impact of permuting assay parameters on binary PPI detection. a–c PPIs detected by permuting (a) protein orientations in Y2H and MAPPIT, (b) tagging configurations in KISS (only two versions tested), NanoBiT, GPCA, mN2H, yN2H and vN2H, and (c) protein expression environments in N2H. Error bars in (a-c) indicate standard deviationsTo probe the influence of different tagging configurations on PPI detection, i.e. the N- or C-terminal positioning of the assay tags, we systematically evaluated the contribution of each pairing combination (N1N2, C1N2, N1C2, and C1C2) with KISS, NanoBiT, GPCA, mN2H, yN2H, and vN2H. For KISS, the analysis was restricted to the two published configurations: N1N2 and C1N225. Using mN2H as a first example, we observed an average PRS detection rate of 19% for a single, randomly picked tagging configuration (Fig. 5b). When two of the four possible tagging configurations were used, the PRS recovery rate was further increased to 28%. Finally, when three and all four tagging configurations were combined, 34 and 37% of hsPRS-v2 pairs were identified (Fig. 5b). Thus, using more than one tagging configuration substantially increased the detection of binary PPIs by mN2H. Similar observations were made for the other assays as the PRS recovery rates were increased from an average 26% to 42% for KISS, where only two configurations were tested, and from 14% to 28% for NanoBiT, 19% to 35% for GPCA, 21% to 45% for yN2H and 19% to 33% for vN2H when only one or all four configurations were tested, respectively. The same trend was observed when combining all tested versions and a 1.6-fold increase in PRS detection obtained by extending assays from a single to multiple configurations (Fig. 5b, union). These results illustrate the gain of sensitivity achieved when performing the same assay with different combinations of tagging configurations, and confirm the preliminary observations made after reanalyzing published data23,25 (Supplementary Fig. 3b).To evaluate the impact of different protein expression systems on binary PPI detection, we implemented different versions of the N2H assay in yeast, in mammalian cells and in vitro (cell-free). All four tagging configurations (Fig. 1b, c) were systematically tested in each environment. For the N1N2 version, an average PRS detection rate of 24% was reached when performing the assay in a single environment. This recovery was increased to 35% when switching to a second expression environment. Finally, after combining detected PRS pairs from all three environments, the cumulative recovery rate reached 42% for the N1N2 versions of N2H (Fig. 5c). When we expanded the analysis to the other tagging configurations, N1C2, C1C2, and C1N2, we consistently observed that performing the assay with different expression systems allowed the identification of a larger fraction of PPIs (Fig. 5c). Indeed, a robust growth of cumulative PRS detection was observed for N1C2 (from 18 to 40%), C1C2 (from 11 to 22%), and C1N2 (from 24 to 43%) when PPIs were tested in just one or all three different expression environments, respectively. Importantly, the cumulative PPI recovery rate for all 12 N2H assay versions tested here was increased from 38% to 60% when using one or all three expression systems, respectively, which represented an overall 1.6-fold increase in PPI detection (Fig. 5c, union). This result also highlights that the performance of the single multi-version N2H assay (60% PRS recovery) was significantly higher than that of any other method presented in this study (Fig. 2a, c).Taken together, these observations demonstrate the benefit of permuting experimental conditions in binary PPI assays to increase detection (Fig. 5a-c). What remained unclear was the overall complementarity of assay versions when they are all combined. To visualize the different subsets of interactions recovered by the six different assays analyzed in this study (Figs. 2c, 3e), we displayed the performance of their corresponding 26 assay versions in the form of a checkerboard plot. Under conditions where none of the hsRRS-v2 pairs are scored positive, the different versions detected distinct but partially overlapping subsets of PPIs, the union of which reached 78% (Fig. 6). Importantly, as already observed in Fig. 3f, the combination of 12 N2H versions detected 60% of PRS pairs, which means that this single assay recovered over three-quarters of all PPIs detected in this study (Fig. 6). Taken together, these results showed that a single assay exploring multiple dimensions of the parameter space (Fig. 1b, c) will be sufficient to map a large fraction of binary PPIs in a defined search space.Fig. 6Complementarity of different assay versions. Detection of individual hsPRS-v2 pairs (left panel) by the assay versions used in Fig. 5a–c, under conditions where none of the hsRRS-v2 pairs (right panel) are scored positive. Detected hsPRS-v2 pairs are indicated by blue squares: light blue corresponds to individual assay versions and dark blue to the union of assay versions into distinct assays or combinations of assays. Identity of pairs can be found in Supplementary Table 4

Combining binary PPI assays under maximized specificityDifferent binary PPI assays often detect distinct subsets of interactions23, thus combining several assays should increase the overall recovery in a given search space of the proteome (Fig. 1a). However, what remains unclear is how binary PPI assay combinations affect the overall specificity, i.e. the fraction of true interactions detected in a given search space over the fraction of spurious pairs reported.Ten years ago, our group introduced an empirical framework to benchmark binary PPI assays based on: (i) a list of ~100 well-documented positive control PPIs, constituting the first version of a PRS, or hsPRS-v1, (ii) a list of ~100 pairs of proteins randomly picked among the ~2 x 108 pairwise combinations of human proteins, constituting version 1 of a RRS, or hsRRS-v1;22 and (iii) the Gateway cloning of all open reading frames (ORFs) involved, allowing identical constructs to be used across many different assays32. We and others have since cross-examined the following binary PPI assays against each other: (i) Y2H23, (ii) yellow fluorescent protein–protein complementation assay (YFP-PCA)23, (iii) mammalian protein–protein interaction trap (MAPPIT)23, (iv) well-based nucleic acid programmable protein arrays (wNAPPA)23, (v) luminescence-based mammalian interactome (LUMIER)23, (vi) kinase substrate sensor (KISS)25, (vii) Gaussia princeps complementation assay (GPCA)24,33, (viii) dual luminescence-based co-immunoprecipitation (DULIP)26, and (ix) bioluminescence-based two-hybrid (LuTHy)27, combining bioluminescence resonance energy transfer (LuTHy-BRET) and luminescence-based co-precipitation (LuTHy-LuC) assays (Fig. 1c, left panel).Under the various conditions used in these studies, each of the corresponding 10 assays, representing the union of only one or two version(s) (Supplementary Table 1), recovered between 21 and 39% of hsPRS-v1 pairs, with the negative control hsRRS-v1 detection ranging between 0 and 4% (Supplementary Fig. 1a). However, to directly compare the different assays side-by-side and combine them homogeneously, an identical RRS threshold should be applied to each individual assay. After applying an identical RRS cutoff, increasing from 1 to 10%, to each of the above-mentioned assays, we observed that the union of the ten different assays detected between 70% and 91% of positive control hsPRS-v1 pairs with 8% and 59% of random hsRRS-v1 pairs scoring positive, respectively (Supplementary Fig. 1b). Therefore, to avoid accumulation of random pairs affecting the quality of future analyses, each assay should be calibrated such that none of the RRS pairs are scored positive. After reanalyzing published data under these conditions, the recovered fractions of hsPRS-v1 pairs ranged from 3 to 33% for individual assays performed in only one or two version(s) (Fig. 2a), while the cumulative detection reached 65% when all assays were combined (Fig. 2b and Supplementary Fig. 1c). This suggested that many different assays, each associated with specific tag pairs and plasmids, would be needed to reach full coverage.Fig. 2Recovery of positive control PRS pairs by multiple assays or assay versions. a Fractions of hsPRS-v1 pairs recovered by ten well-established assays when an identical threshold of no hsRRS-v1 pairs scoring positive is applied. b Cumulative hsPRS-v1 recovery rates when assays in (a) are combined (individual data points are displayed in Supplementary Fig. 1c). c Fractions of hsPRS-v2 pairs recovered by different versions of GPCA, KISS, MAPPIT, Y2H and NanoBiT when an identical threshold of no hsRRS-v2 pairs scoring positive is applied. Horizontal shaded blue area indicates the average recovery rate ± S.D. for GPCA, KISS, MAPPIT and Y2H versions. d Cumulative hsPRS-v2 recovery rates when versions of GPCA, KISS, MAPPIT and Y2H presented in (c) are combined (individual data points are displayed in Supplementary Fig. 2d). Error bars indicate standard errors of proportions (a, c) or standard deviations (b, d)

Combining binary PPI assays under maximized specificity

Different binary PPI assays often detect distinct subsets of interactions23, thus combining several assays should increase the overall recovery in a given search space of the proteome (Fig. 1a). However, what remains unclear is how binary PPI assay combinations affect the overall specificity, i.e. the fraction of true interactions detected in a given search space over the fraction of spurious pairs reported.

Ten years ago, our group introduced an empirical framework to benchmark binary PPI assays based on: (i) a list of ~100 well-documented positive control PPIs, constituting the first version of a PRS, or hsPRS-v1, (ii) a list of ~100 pairs of proteins randomly picked among the ~2 x 108 pairwise combinations of human proteins, constituting version 1 of a RRS, or hsRRS-v1;22 and (iii) the Gateway cloning of all open reading frames (ORFs) involved, allowing identical constructs to be used across many different assays32. We and others have since cross-examined the following binary PPI assays against each other: (i) Y2H23, (ii) yellow fluorescent protein–protein complementation assay (YFP-PCA)23, (iii) mammalian protein–protein interaction trap (MAPPIT)23, (iv) well-based nucleic acid programmable protein arrays (wNAPPA)23, (v) luminescence-based mammalian interactome (LUMIER)23, (vi) kinase substrate sensor (KISS)25, (vii) Gaussia princeps complementation assay (GPCA)24,33, (viii) dual luminescence-based co-immunoprecipitation (DULIP)26, and (ix) bioluminescence-based two-hybrid (LuTHy)27, combining bioluminescence resonance energy transfer (LuTHy-BRET) and luminescence-based co-precipitation (LuTHy-LuC) assays (Fig. 1c, left panel).

Under the various conditions used in these studies, each of the corresponding 10 assays, representing the union of only one or two version(s) (Supplementary Table 1), recovered between 21 and 39% of hsPRS-v1 pairs, with the negative control hsRRS-v1 detection ranging between 0 and 4% (Supplementary Fig. 1a). However, to directly compare the different assays side-by-side and combine them homogeneously, an identical RRS threshold should be applied to each individual assay. After applying an identical RRS cutoff, increasing from 1 to 10%, to each of the above-mentioned assays, we observed that the union of the ten different assays detected between 70% and 91% of positive control hsPRS-v1 pairs with 8% and 59% of random hsRRS-v1 pairs scoring positive, respectively (Supplementary Fig. 1b). Therefore, to avoid accumulation of random pairs affecting the quality of future analyses, each assay should be calibrated such that none of the RRS pairs are scored positive. After reanalyzing published data under these conditions, the recovered fractions of hsPRS-v1 pairs ranged from 3 to 33% for individual assays performed in only one or two version(s) (Fig. 2a), while the cumulative detection reached 65% when all assays were combined (Fig. 2b and Supplementary Fig. 1c). This suggested that many different assays, each associated with specific tag pairs and plasmids, would be needed to reach full coverage.Fig. 2Recovery of positive control PRS pairs by multiple assays or assay versions. a Fractions of hsPRS-v1 pairs recovered by ten well-established assays when an identical threshold of no hsRRS-v1 pairs scoring positive is applied. b Cumulative hsPRS-v1 recovery rates when assays in (a) are combined (individual data points are displayed in Supplementary Fig. 1c). c Fractions of hsPRS-v2 pairs recovered by different versions of GPCA, KISS, MAPPIT, Y2H and NanoBiT when an identical threshold of no hsRRS-v2 pairs scoring positive is applied. Horizontal shaded blue area indicates the average recovery rate ± S.D. for GPCA, KISS, MAPPIT and Y2H versions. d Cumulative hsPRS-v2 recovery rates when versions of GPCA, KISS, MAPPIT and Y2H presented in (c) are combined (individual data points are displayed in Supplementary Fig. 2d). Error bars indicate standard errors of proportions (a, c) or standard deviations (b, d)

Recovery of positive control PRS pairs by multiple assays or assay versions. a Fractions of hsPRS-v1 pairs recovered by ten well-established assays when an identical threshold of no hsRRS-v1 pairs scoring positive is applied. b Cumulative hsPRS-v1 recovery rates when assays in (a) are combined (individual data points are displayed in Supplementary Fig. 1c). c Fractions of hsPRS-v2 pairs recovered by different versions of GPCA, KISS, MAPPIT, Y2H and NanoBiT when an identical threshold of no hsRRS-v2 pairs scoring positive is applied. Horizontal shaded blue area indicates the average recovery rate ± S.D. for GPCA, KISS, MAPPIT and Y2H versions. d Cumulative hsPRS-v2 recovery rates when versions of GPCA, KISS, MAPPIT and Y2H presented in (c) are combined (individual data points are displayed in Supplementary Fig. 2d). Error bars indicate standard errors of proportions (a, c) or standard deviations (b, d)

Benchmarking binary PPI assays and assay versionsWe considered the possibility that the failure of the 10 assays described above (Fig. 2a, b) to detect nearly all PRS interactions might be due to ORF clone quality issues and/or problematic interaction annotations. To avoid such possibilities, we re-examined all protein pairs from the original reference sets: hsPRS-v1 and hsRRS-v123. As a first step, only fully sequence-verified, full-length clonal ORFs were isolated (Supplementary Fig. 2a). Resulting pairs were then filtered based on updated literature information and annotations for binary PPIs (Supplementary Table 2, Supplementary Data 1). Finally, 60 PRS and 78 RRS pairs were assembled into a second-generation of reference sets: hsPRS-v2 and hsRRS-v2, respectively (see Methods for details and Supplementary Fig. 2b). After repeating four assays that were originally benchmarked against hsPRS-v1 and hsRRS-v1 with the second generation of reference sets, the cumulative PRS recovery rate reached 50 and 58% with -v1 and -v2 sets respectively, when none of the RRS pairs were scored positive (Supplementary Fig. 2c).As the failure of 10 well-established assays to detect nearly all binary PPIs did not arise from problems with the original reference sets, we decided to evaluate whether expanding the number of experimental conditions, or assay versions (Fig. 1b), covered by any single technique could increase its coverage and ultimately reduce the number of assays required. To investigate this, we benchmarked several versions of Y2H, MAPPIT, KISS, and GPCA against the second-generation of reference sets, hsPRS-v2 and hsRRS-v2. These four assays and corresponding versions (10 in total) were implemented as summarized in Supplementary Table 1. In particular, we constructed Gateway-compatible vectors for GPCA to cover all four possibilities of tagging configurations with this assay, i.e. Tag1-X/Tag2-Y, X-Tag1/Tag2-Y, Tag1-X/Y-Tag2, and X-Tag1/Y-Tag2 or, simply, N1N2, C1N2, N1C2, and C1C2, respectively. Between 8 and 37% of hsPRS-v2 pairs were detected by the 10 versions of GPCA, MAPPIT, KISS and Y2H when none of the hsRRS-v2 pairs were scored positive (Fig. 2c), which represented an average PRS detection rate of 22.8 ± 8.8% (S.D.) (Fig. 2c, shaded blue area).When we considered all protein pairs identified by the union of these 10 versions corresponding to four assays, the hsPRS-v2 recovery rate reached 63% (Fig. 2d and Supplementary Fig. 2d), similar to what was obtained by combining ten distinct assays, performed in only one or two version(s), as published in the literature (65%; Fig. 2b). This result demonstrates that the combination of four distinct assays exploring different experimental conditions can recover as many PPIs as 10 fundamentally different techniques limited to a few versions.

Benchmarking binary PPI assays and assay versions

We considered the possibility that the failure of the 10 assays described above (Fig. 2a, b) to detect nearly all PRS interactions might be due to ORF clone quality issues and/or problematic interaction annotations. To avoid such possibilities, we re-examined all protein pairs from the original reference sets: hsPRS-v1 and hsRRS-v123. As a first step, only fully sequence-verified, full-length clonal ORFs were isolated (Supplementary Fig. 2a). Resulting pairs were then filtered based on updated literature information and annotations for binary PPIs (Supplementary Table 2, Supplementary Data 1). Finally, 60 PRS and 78 RRS pairs were assembled into a second-generation of reference sets: hsPRS-v2 and hsRRS-v2, respectively (see Methods for details and Supplementary Fig. 2b). After repeating four assays that were originally benchmarked against hsPRS-v1 and hsRRS-v1 with the second generation of reference sets, the cumulative PRS recovery rate reached 50 and 58% with -v1 and -v2 sets respectively, when none of the RRS pairs were scored positive (Supplementary Fig. 2c).

As the failure of 10 well-established assays to detect nearly all binary PPIs did not arise from problems with the original reference sets, we decided to evaluate whether expanding the number of experimental conditions, or assay versions (Fig. 1b), covered by any single technique could increase its coverage and ultimately reduce the number of assays required. To investigate this, we benchmarked several versions of Y2H, MAPPIT, KISS, and GPCA against the second-generation of reference sets, hsPRS-v2 and hsRRS-v2. These four assays and corresponding versions (10 in total) were implemented as summarized in Supplementary Table 1. In particular, we constructed Gateway-compatible vectors for GPCA to cover all four possibilities of tagging configurations with this assay, i.e. Tag1-X/Tag2-Y, X-Tag1/Tag2-Y, Tag1-X/Y-Tag2, and X-Tag1/Y-Tag2 or, simply, N1N2, C1N2, N1C2, and C1C2, respectively. Between 8 and 37% of hsPRS-v2 pairs were detected by the 10 versions of GPCA, MAPPIT, KISS and Y2H when none of the hsRRS-v2 pairs were scored positive (Fig. 2c), which represented an average PRS detection rate of 22.8 ± 8.8% (S.D.) (Fig. 2c, shaded blue area).

When we considered all protein pairs identified by the union of these 10 versions corresponding to four assays, the hsPRS-v2 recovery rate reached 63% (Fig. 2d and Supplementary Fig. 2d), similar to what was obtained by combining ten distinct assays, performed in only one or two version(s), as published in the literature (65%; Fig. 2b). This result demonstrates that the combination of four distinct assays exploring different experimental conditions can recover as many PPIs as 10 fundamentally different techniques limited to a few versions.

Toward developing a highly versatile binary PPI assayIn an attempt to further minimize the overall number of multi-version assays required to maximize binary PPI detection and, by extension, reduce the associated cloning efforts and lab resources, we decided to develop a highly versatile binary PPI detection system opening access to all assay versions defined by the three experimental parameters presented in Fig. 1b.Although after reanalyzing published data, 1.3-fold and 1.4-fold increases in PRS detection were observed when switching from one orientation to the other (i.e. Tag1-X/Tag2-Y to Tag1-Y/Tag2-X), or from one configuration to another (i.e. N1N2 to C1N2), respectively (Supplementary Fig. 3a, b), the impact of permuting expression systems remained unclear as no assay had been used to systematically explore this parameter. However, several high-quality PPI datasets have been obtained using heterologous expression of protein pairs in yeast12,34, in mammalian cells35,36 and via cell-free coupled transcription-translation systems in vitro37,38.After surveying existing binary interaction assays, it appeared that split-protein reconstitution was the most suitable technology to cover the three experimental parameters presented in Fig. 1b. Indeed, compared to other binary PPI methods involving cell-specific responses or antibody-based captures, split-protein-based assays enable detection of PPIs through the simple reassembly of a functional reporter protein (Fig. 1c). Among the different reporters available today, NanoLuc, a highly stable 19 kDa protein derived from the luciferase complex of the deep-sea shrimp O. gracilirostris, is gaining increasing attention for the development of binary PPI detection assays, as highlighted by recent publications31,39–41. Following the single-step addition of imidazopyrazinone substrates, NanoLuc instantly produces an intense and sustainable glow-type luminescence with a specific activity about 100-fold greater than that of other luciferases, allowing its detection in vitro as well as in living cells and animals31,42,43. Therefore, split-NanoLuc-based technology appears to be the ideal candidate for the high-throughput detection of binary PPIs under various experimental conditions.Among the recent studies reporting a split-NanoLuc binary PPI assay, only one extensively fragmented the enzyme to test different tag pairs in an unbiased way (Supplementary Table 3)39. After selecting a cleavage site between amino acids 156 and 157, the authors heavily modified the corresponding fragments to reduce their association affinity. The resulting moieties, 11S (156 amino acids) and 114 (11 amino acids), were then used to develop the NanoLuc Binary Technology (NanoBiT) assay39.Although NanoBiT was originally designed for expression of protein pairs in mammalian cells, we decided to evaluate whether fragments 11S and 114 could be used to further develop our new, highly versatile binary PPI detection system (Fig. 1c, right panel). To investigate this, we first adapted the available NanoBiT vectors with the reported original split site39 to achieve different tagging configurations with Gateway-cloning compatibility and then benchmarked all four pairing combinations, i.e. N1N2, C1N2, N1C2, and C1C2, against hsPRS-v2 and hsRRS-v2. The benchmarking of these four NanoBiT versions in conditions where none of the hsRRS-v2 pairs are scored positive showed that between 3 and 25% of hsPRS-v2 pairs could be recovered (Fig. 2c). These results revealed optimal performances when the larger NanoBiT fragment, 11S (156 amino acids), was fused to the C-terminal extremity of the tested proteins, as demonstrated by PRS recovery rates of 22% and 25% for the C1N2 and C1C2 configurations, respectively, similar to those from four other well-established assays (Fig. 2c). These observations were in agreement with the author’s original assumption that the C-terminus tagging of proteins with 11S might impose minimal steric constraints39.

Toward developing a highly versatile binary PPI assay

In an attempt to further minimize the overall number of multi-version assays required to maximize binary PPI detection and, by extension, reduce the associated cloning efforts and lab resources, we decided to develop a highly versatile binary PPI detection system opening access to all assay versions defined by the three experimental parameters presented in Fig. 1b.

Although after reanalyzing published data, 1.3-fold and 1.4-fold increases in PRS detection were observed when switching from one orientation to the other (i.e. Tag1-X/Tag2-Y to Tag1-Y/Tag2-X), or from one configuration to another (i.e. N1N2 to C1N2), respectively (Supplementary Fig. 3a, b), the impact of permuting expression systems remained unclear as no assay had been used to systematically explore this parameter. However, several high-quality PPI datasets have been obtained using heterologous expression of protein pairs in yeast12,34, in mammalian cells35,36 and via cell-free coupled transcription-translation systems in vitro37,38.

After surveying existing binary interaction assays, it appeared that split-protein reconstitution was the most suitable technology to cover the three experimental parameters presented in Fig. 1b. Indeed, compared to other binary PPI methods involving cell-specific responses or antibody-based captures, split-protein-based assays enable detection of PPIs through the simple reassembly of a functional reporter protein (Fig. 1c). Among the different reporters available today, NanoLuc, a highly stable 19 kDa protein derived from the luciferase complex of the deep-sea shrimp O. gracilirostris, is gaining increasing attention for the development of binary PPI detection assays, as highlighted by recent publications31,39–41. Following the single-step addition of imidazopyrazinone substrates, NanoLuc instantly produces an intense and sustainable glow-type luminescence with a specific activity about 100-fold greater than that of other luciferases, allowing its detection in vitro as well as in living cells and animals31,42,43. Therefore, split-NanoLuc-based technology appears to be the ideal candidate for the high-throughput detection of binary PPIs under various experimental conditions.

Among the recent studies reporting a split-NanoLuc binary PPI assay, only one extensively fragmented the enzyme to test different tag pairs in an unbiased way (Supplementary Table 3)39. After selecting a cleavage site between amino acids 156 and 157, the authors heavily modified the corresponding fragments to reduce their association affinity. The resulting moieties, 11S (156 amino acids) and 114 (11 amino acids), were then used to develop the NanoLuc Binary Technology (NanoBiT) assay39.

Although NanoBiT was originally designed for expression of protein pairs in mammalian cells, we decided to evaluate whether fragments 11S and 114 could be used to further develop our new, highly versatile binary PPI detection system (Fig. 1c, right panel). To investigate this, we first adapted the available NanoBiT vectors with the reported original split site39 to achieve different tagging configurations with Gateway-cloning compatibility and then benchmarked all four pairing combinations, i.e. N1N2, C1N2, N1C2, and C1C2, against hsPRS-v2 and hsRRS-v2. The benchmarking of these four NanoBiT versions in conditions where none of the hsRRS-v2 pairs are scored positive showed that between 3 and 25% of hsPRS-v2 pairs could be recovered (Fig. 2c). These results revealed optimal performances when the larger NanoBiT fragment, 11S (156 amino acids), was fused to the C-terminal extremity of the tested proteins, as demonstrated by PRS recovery rates of 22% and 25% for the C1N2 and C1C2 configurations, respectively, similar to those from four other well-established assays (Fig. 2c). These observations were in agreement with the author’s original assumption that the C-terminus tagging of proteins with 11S might impose minimal steric constraints39.

Nanoluc 2-Hybrid (N2H): a highly versatile binary PPI assayAlthough NanoBiT appears to be a powerful tool to study PPI kinetics in live cells, the comprehensive benchmarking of this assay revealed two underperforming versions, N1N2 and N1C2, that might be associated with the inherent properties of the selected tag pair39. Therefore, we decided to empirically define a new split site for NanoLuc in an unbiased way. To do this, we first extensively fragmented the full-length enzyme by transposon-based insertional mutagenesis (Supplementary Fig. 4a). Complementation of the resulting fragments, F1 and F2 (58 pairs in total), was then systematically tested using a proximity assay in vitro (Supplementary Fig. 4b), and a cleavage site between amino acids 65 and 66 was selected (Fig. 3a) to build the NanoLuc 2-hybrid (N2H) assay (Fig. 1c, right panel; Supplementary Fig. 4c).Fig. 3Development of the highly versatile N2H assay. a Determination of a new split site for NanoLuc. Dark blue bar and red arrow indicate the selected NanoLuc fragments. RLU relative luminescence units, NLuc NanoLuc. Blank control corresponds to reagents without any NanoLuc fragment. (b) Tripartite promoter and specific activities of selected N2H fragments, F1 and F2, normalized to full-length NanoLuc when expressed in mammalian cells (mN2H; orange), in yeast (yN2H; purple), or in vitro (vN2H; gray), from the same plasmid. For “empty”: no protein or fragment was expressed. CMV cytomegalovirus, HSV-TK herpes simplex virus-thymidine kinase, UAS upstream activating sequence, IRES internal ribosome entry site. Red arrow displayed on NanoLuc structure (PDB: 5IBO) indicates the cleavage area. c Reproducibility of N2H assay in three expression environments (mN2H, yN2H, and vN2H) for the N1N2 tagging configuration. Each triangle in the different graphs represents a single tested protein pair. d Normalized luminescence signals for 12 versions of N2H benchmarked against hsPRS-v2 and hsRRS-v2. Blue squares and purple circles indicate PRS and RRS pairs, respectively. The threshold of no RRS pair scoring positive is indicated (horizontal line) for each assay version. e Fraction of hsPRS-v2 pairs recovered by 12 N2H assay versions when none of the hsRRS-v2 pairs are scored positive, corresponding to the thresholds displayed in (d). Horizontal shaded blue area indicates the average recovery rate ± S.D. for all assay versions reported in Fig. 2c. f Cumulative hsPRS-v2 recovery rates when versions of N2H presented in (e) are combined (individual data points are displayed in Supplementary Fig. 5a). Error bars indicate standard deviations (b, f) or standard errors of proportions (e)In order to comprehensively cover the experimental parameters presented in Fig. 1b with the single N2H assay, we constructed four Gateway-compatible plasmids from which N2H fragments, F1 (amino acids 1–65) and F2 (amino acids 66–171), can be expressed as either N- or C-terminal fusions of any pair of proteins X and Y in yeast, in mammalian cells or in vitro (cell-free). To drive expression in all three environments with each individual plasmid, we engineered a tripartite promoter of ~1 kb that can easily be introduced into any vector of interest (Fig. 3b). The performance of this 3-in-1 promoter was evaluated by measuring the expression of full-length and fragmented NanoLuc in yeast (yN2H), in mammalian cells (mN2H) and in vitro (vN2H). We obtained intense signals ranging between 108 and 109 relative luminescence units (RLU) when the full-length NanoLuc was expressed in each individual environment. Furthermore, compared to the full-length enzyme, individually expressed NanoLuc fragments did not produce any significant signal. More importantly, the two fragments expressed together did not generate any strong self-assembly luminescence that could interfere with the accurate measurement of binary PPIs. Taken together, these results demonstrate the full functionality of both the tripartite promoter and the selected pair of NanoLuc fragments (Fig. 3b).By combining NanoLuc fragments with the tripartite promoter and the Gateway-cloning technology into four uniform expression vectors (pDEST-N2H-N1, -N2, -C1, and -C2), all 24 assay versions (4x3x2) presented in Fig. 1b can thus be implemented with the single N2H assay. The Gateway-compatibility of N2H destination plasmids also renders the entire human ORFeome collection of over 18,000 entry clones44 directly amenable to this assay.As N2H was the first binary PPI detection tool in which proteins could be expressed as either N- or C-terminal fusions of protein tags in yeast (yN2H), in mammalian cells (mN2H) or in vitro (vN2H) from the same Gateway-compatible destination vector (Figs. 1b, c, 3b), it was important to benchmark the robustness of PPI-mediated readouts in each of the three environments. Using hsPRS-v2 and hsRRS-v2 pairs, we compared bioluminescence signals between two independently replicated experiments in yN2H, mN2H, and vN2H. As highlighted by the R2 > 0.97, a robust reproducibility was observed in each case when all 138 pairs from hsPRS-v2 and hsRRS-v2 were tested. In addition, a large dynamic range of luminescence readout, between 102 and 107 RLU, was consistently observed in each environment (Fig. 3c).To compare the performance of N2H assay versions in detecting reference binary PPIs over random protein pairs, we normalized luminescence signals and separated hsPRS-v2 from hsRRS-v2. No major differences were observed in the overall luminescence ratios as a fraction of PRS pairs always generated higher signals than that from random pairs in each case (Fig. 3d). As observed for C1C2-mN2H or C1C2-yN2H, for example, the only differences in PRS detection rates between assay versions came from one or a few RRS pairs that generated higher luminescence signals than the average. These version-specific artifacts ultimately reduced the fraction of PRS pairs recovered as their corresponding thresholds of no RRS pair scoring positive increased (Fig. 3d). The same observation was also made for well-established assays (Fig. 2a, c). Thus, in conditions where none of the RRS pairs are scored positive (threshold displayed for each individual version in Fig. 3d), the 12 tested N2H assay versions detected between 7 and 32% of hsPRS-v2 pairs (Fig. 3e), demonstrating similar performances compared to five other PPI detection methods (Fig. 2c) that averaged 20.4 ± 9.9% (S.D.) PRS recovery (Fig. 3e, shaded blue area). When these 12 N2H versions were combined, a PRS detection of 60% was yielded (Fig. 3f and Supplementary Fig. 5a), similar to the 65% obtained by combining either 10 different, version-limited assays from the literature23–28 (Fig. 2a, b) or five non-N2H, multi-version assays (Fig. 2c and Supplementary Fig. 5b).These results highlight the advantages of using a single, highly versatile PPI detection tool, such as N2H, to reduce the overall cloning efforts and lab resources (Fig. 4a–c), as the same set of expression vectors are used to explore several assay versions at a time (Fig. 4d) whilst recovering as many PPIs as a combination of many different techniques (Fig. 4e). The published assays in Fig. 4 are presented in Fig. 2a and Supplementary Table 1.Fig. 4Invested resources for various combinations of binary PPI assays and PRS detection rates. Numbers of (a) assays, (b) plasmids, (c) clonings per pair, and (d) tested versions required to achieve the PRS recovery rates presented in (e). PRS recovery rates presented in Fig. 4e correspond to those found in Fig. 2b (published), Supplementary Fig. 5b (non-N2H) and Fig. 3f (N2H). Except for Y2H-N1C2, Y2H-C1C2, and Y2H-C1N2 for which unambiguous titrations could not be obtained from the original study28, all published assay versions benchmarked against hsPRS-v1 and hsRRS-v1 were used (Fig. 2a, Supplementary Table 1)

Nanoluc 2-Hybrid (N2H): a highly versatile binary PPI assay

Although NanoBiT appears to be a powerful tool to study PPI kinetics in live cells, the comprehensive benchmarking of this assay revealed two underperforming versions, N1N2 and N1C2, that might be associated with the inherent properties of the selected tag pair39. Therefore, we decided to empirically define a new split site for NanoLuc in an unbiased way. To do this, we first extensively fragmented the full-length enzyme by transposon-based insertional mutagenesis (Supplementary Fig. 4a). Complementation of the resulting fragments, F1 and F2 (58 pairs in total), was then systematically tested using a proximity assay in vitro (Supplementary Fig. 4b), and a cleavage site between amino acids 65 and 66 was selected (Fig. 3a) to build the NanoLuc 2-hybrid (N2H) assay (Fig. 1c, right panel; Supplementary Fig. 4c).Fig. 3Development of the highly versatile N2H assay. a Determination of a new split site for NanoLuc. Dark blue bar and red arrow indicate the selected NanoLuc fragments. RLU relative luminescence units, NLuc NanoLuc. Blank control corresponds to reagents without any NanoLuc fragment. (b) Tripartite promoter and specific activities of selected N2H fragments, F1 and F2, normalized to full-length NanoLuc when expressed in mammalian cells (mN2H; orange), in yeast (yN2H; purple), or in vitro (vN2H; gray), from the same plasmid. For “empty”: no protein or fragment was expressed. CMV cytomegalovirus, HSV-TK herpes simplex virus-thymidine kinase, UAS upstream activating sequence, IRES internal ribosome entry site. Red arrow displayed on NanoLuc structure (PDB: 5IBO) indicates the cleavage area. c Reproducibility of N2H assay in three expression environments (mN2H, yN2H, and vN2H) for the N1N2 tagging configuration. Each triangle in the different graphs represents a single tested protein pair. d Normalized luminescence signals for 12 versions of N2H benchmarked against hsPRS-v2 and hsRRS-v2. Blue squares and purple circles indicate PRS and RRS pairs, respectively. The threshold of no RRS pair scoring positive is indicated (horizontal line) for each assay version. e Fraction of hsPRS-v2 pairs recovered by 12 N2H assay versions when none of the hsRRS-v2 pairs are scored positive, corresponding to the thresholds displayed in (d). Horizontal shaded blue area indicates the average recovery rate ± S.D. for all assay versions reported in Fig. 2c. f Cumulative hsPRS-v2 recovery rates when versions of N2H presented in (e) are combined (individual data points are displayed in Supplementary Fig. 5a). Error bars indicate standard deviations (b, f) or standard errors of proportions (e)

Development of the highly versatile N2H assay. a Determination of a new split site for NanoLuc. Dark blue bar and red arrow indicate the selected NanoLuc fragments. RLU relative luminescence units, NLuc NanoLuc. Blank control corresponds to reagents without any NanoLuc fragment. (b) Tripartite promoter and specific activities of selected N2H fragments, F1 and F2, normalized to full-length NanoLuc when expressed in mammalian cells (mN2H; orange), in yeast (yN2H; purple), or in vitro (vN2H; gray), from the same plasmid. For “empty”: no protein or fragment was expressed. CMV cytomegalovirus, HSV-TK herpes simplex virus-thymidine kinase, UAS upstream activating sequence, IRES internal ribosome entry site. Red arrow displayed on NanoLuc structure (PDB: 5IBO) indicates the cleavage area. c Reproducibility of N2H assay in three expression environments (mN2H, yN2H, and vN2H) for the N1N2 tagging configuration. Each triangle in the different graphs represents a single tested protein pair. d Normalized luminescence signals for 12 versions of N2H benchmarked against hsPRS-v2 and hsRRS-v2. Blue squares and purple circles indicate PRS and RRS pairs, respectively. The threshold of no RRS pair scoring positive is indicated (horizontal line) for each assay version. e Fraction of hsPRS-v2 pairs recovered by 12 N2H assay versions when none of the hsRRS-v2 pairs are scored positive, corresponding to the thresholds displayed in (d). Horizontal shaded blue area indicates the average recovery rate ± S.D. for all assay versions reported in Fig. 2c. f Cumulative hsPRS-v2 recovery rates when versions of N2H presented in (e) are combined (individual data points are displayed in Supplementary Fig. 5a). Error bars indicate standard deviations (b, f) or standard errors of proportions (e)

In order to comprehensively cover the experimental parameters presented in Fig. 1b with the single N2H assay, we constructed four Gateway-compatible plasmids from which N2H fragments, F1 (amino acids 1–65) and F2 (amino acids 66–171), can be expressed as either N- or C-terminal fusions of any pair of proteins X and Y in yeast, in mammalian cells or in vitro (cell-free). To drive expression in all three environments with each individual plasmid, we engineered a tripartite promoter of ~1 kb that can easily be introduced into any vector of interest (Fig. 3b). The performance of this 3-in-1 promoter was evaluated by measuring the expression of full-length and fragmented NanoLuc in yeast (yN2H), in mammalian cells (mN2H) and in vitro (vN2H). We obtained intense signals ranging between 108 and 109 relative luminescence units (RLU) when the full-length NanoLuc was expressed in each individual environment. Furthermore, compared to the full-length enzyme, individually expressed NanoLuc fragments did not produce any significant signal. More importantly, the two fragments expressed together did not generate any strong self-assembly luminescence that could interfere with the accurate measurement of binary PPIs. Taken together, these results demonstrate the full functionality of both the tripartite promoter and the selected pair of NanoLuc fragments (Fig. 3b).

By combining NanoLuc fragments with the tripartite promoter and the Gateway-cloning technology into four uniform expression vectors (pDEST-N2H-N1, -N2, -C1, and -C2), all 24 assay versions (4x3x2) presented in Fig. 1b can thus be implemented with the single N2H assay. The Gateway-compatibility of N2H destination plasmids also renders the entire human ORFeome collection of over 18,000 entry clones44 directly amenable to this assay.

As N2H was the first binary PPI detection tool in which proteins could be expressed as either N- or C-terminal fusions of protein tags in yeast (yN2H), in mammalian cells (mN2H) or in vitro (vN2H) from the same Gateway-compatible destination vector (Figs. 1b, c, 3b), it was important to benchmark the robustness of PPI-mediated readouts in each of the three environments. Using hsPRS-v2 and hsRRS-v2 pairs, we compared bioluminescence signals between two independently replicated experiments in yN2H, mN2H, and vN2H. As highlighted by the R2 > 0.97, a robust reproducibility was observed in each case when all 138 pairs from hsPRS-v2 and hsRRS-v2 were tested. In addition, a large dynamic range of luminescence readout, between 102 and 107 RLU, was consistently observed in each environment (Fig. 3c).

To compare the performance of N2H assay versions in detecting reference binary PPIs over random protein pairs, we normalized luminescence signals and separated hsPRS-v2 from hsRRS-v2. No major differences were observed in the overall luminescence ratios as a fraction of PRS pairs always generated higher signals than that from random pairs in each case (Fig. 3d). As observed for C1C2-mN2H or C1C2-yN2H, for example, the only differences in PRS detection rates between assay versions came from one or a few RRS pairs that generated higher luminescence signals than the average. These version-specific artifacts ultimately reduced the fraction of PRS pairs recovered as their corresponding thresholds of no RRS pair scoring positive increased (Fig. 3d). The same observation was also made for well-established assays (Fig. 2a, c). Thus, in conditions where none of the RRS pairs are scored positive (threshold displayed for each individual version in Fig. 3d), the 12 tested N2H assay versions detected between 7 and 32% of hsPRS-v2 pairs (Fig. 3e), demonstrating similar performances compared to five other PPI detection methods (Fig. 2c) that averaged 20.4 ± 9.9% (S.D.) PRS recovery (Fig. 3e, shaded blue area). When these 12 N2H versions were combined, a PRS detection of 60% was yielded (Fig. 3f and Supplementary Fig. 5a), similar to the 65% obtained by combining either 10 different, version-limited assays from the literature23–28 (Fig. 2a, b) or five non-N2H, multi-version assays (Fig. 2c and Supplementary Fig. 5b).

These results highlight the advantages of using a single, highly versatile PPI detection tool, such as N2H, to reduce the overall cloning efforts and lab resources (Fig. 4a–c), as the same set of expression vectors are used to explore several assay versions at a time (Fig. 4d) whilst recovering as many PPIs as a combination of many different techniques (Fig. 4e). The published assays in Fig. 4 are presented in Fig. 2a and Supplementary Table 1.Fig. 4Invested resources for various combinations of binary PPI assays and PRS detection rates. Numbers of (a) assays, (b) plasmids, (c) clonings per pair, and (d) tested versions required to achieve the PRS recovery rates presented in (e). PRS recovery rates presented in Fig. 4e correspond to those found in Fig. 2b (published), Supplementary Fig. 5b (non-N2H) and Fig. 3f (N2H). Except for Y2H-N1C2, Y2H-C1C2, and Y2H-C1N2 for which unambiguous titrations could not be obtained from the original study28, all published assay versions benchmarked against hsPRS-v1 and hsRRS-v1 were used (Fig. 2a, Supplementary Table 1)

Invested resources for various combinations of binary PPI assays and PRS detection rates. Numbers of (a) assays, (b) plasmids, (c) clonings per pair, and (d) tested versions required to achieve the PRS recovery rates presented in (e). PRS recovery rates presented in Fig. 4e correspond to those found in Fig. 2b (published), Supplementary Fig. 5b (non-N2H) and Fig. 3f (N2H). Except for Y2H-N1C2, Y2H-C1C2, and Y2H-C1N2 for which unambiguous titrations could not be obtained from the original study28, all published assay versions benchmarked against hsPRS-v1 and hsRRS-v1 were used (Fig. 2a, Supplementary Table 1)

Combining multi-version assays to maximize PPI detectionWith all assays in hand to systematically evaluate the impact of the three variables presented in Fig. 1b, we decided to combine the different PPI assay versions previously tested against the improved reference sets (Figs. 2c, 3e) with respect to each individual parameter. All analyses were conducted in conditions where none of the random pairs are scored positive.As the impact of permuting proteins X and Y from one tag to the other (Tag1-X:Tag2-Y vs. Tag1-Y:Tag2-X) had already been comprehensively investigated by six different assays in the literature23,26,27 (Supplementary Table 1), we only repeated Y2H and MAPPIT to explore this parameter. While a single, randomly picked orientation yielded PRS recovery rates of 19% and 32%, switching to the second orientation increased detection to 25% and 40% for Y2H and MAPPIT, respectively. Importantly, cumulative PPI detection increased from 38% to 47% when a single or two orientations were combined for both assays, respectively (Fig. 5a). This 1.2-fold (47/38) increase was consistent with our analysis of published results23,26,27 (Supplementary Fig. 3a) and confirmed that permuting the orientation of tested proteins is a valuable option to increase PPI detection.Fig. 5Impact of permuting assay parameters on binary PPI detection. a–c PPIs detected by permuting (a) protein orientations in Y2H and MAPPIT, (b) tagging configurations in KISS (only two versions tested), NanoBiT, GPCA, mN2H, yN2H and vN2H, and (c) protein expression environments in N2H. Error bars in (a-c) indicate standard deviationsTo probe the influence of different tagging configurations on PPI detection, i.e. the N- or C-terminal positioning of the assay tags, we systematically evaluated the contribution of each pairing combination (N1N2, C1N2, N1C2, and C1C2) with KISS, NanoBiT, GPCA, mN2H, yN2H, and vN2H. For KISS, the analysis was restricted to the two published configurations: N1N2 and C1N225. Using mN2H as a first example, we observed an average PRS detection rate of 19% for a single, randomly picked tagging configuration (Fig. 5b). When two of the four possible tagging configurations were used, the PRS recovery rate was further increased to 28%. Finally, when three and all four tagging configurations were combined, 34 and 37% of hsPRS-v2 pairs were identified (Fig. 5b). Thus, using more than one tagging configuration substantially increased the detection of binary PPIs by mN2H. Similar observations were made for the other assays as the PRS recovery rates were increased from an average 26% to 42% for KISS, where only two configurations were tested, and from 14% to 28% for NanoBiT, 19% to 35% for GPCA, 21% to 45% for yN2H and 19% to 33% for vN2H when only one or all four configurations were tested, respectively. The same trend was observed when combining all tested versions and a 1.6-fold increase in PRS detection obtained by extending assays from a single to multiple configurations (Fig. 5b, union). These results illustrate the gain of sensitivity achieved when performing the same assay with different combinations of tagging configurations, and confirm the preliminary observations made after reanalyzing published data23,25 (Supplementary Fig. 3b).To evaluate the impact of different protein expression systems on binary PPI detection, we implemented different versions of the N2H assay in yeast, in mammalian cells and in vitro (cell-free). All four tagging configurations (Fig. 1b, c) were systematically tested in each environment. For the N1N2 version, an average PRS detection rate of 24% was reached when performing the assay in a single environment. This recovery was increased to 35% when switching to a second expression environment. Finally, after combining detected PRS pairs from all three environments, the cumulative recovery rate reached 42% for the N1N2 versions of N2H (Fig. 5c). When we expanded the analysis to the other tagging configurations, N1C2, C1C2, and C1N2, we consistently observed that performing the assay with different expression systems allowed the identification of a larger fraction of PPIs (Fig. 5c). Indeed, a robust growth of cumulative PRS detection was observed for N1C2 (from 18 to 40%), C1C2 (from 11 to 22%), and C1N2 (from 24 to 43%) when PPIs were tested in just one or all three different expression environments, respectively. Importantly, the cumulative PPI recovery rate for all 12 N2H assay versions tested here was increased from 38% to 60% when using one or all three expression systems, respectively, which represented an overall 1.6-fold increase in PPI detection (Fig. 5c, union). This result also highlights that the performance of the single multi-version N2H assay (60% PRS recovery) was significantly higher than that of any other method presented in this study (Fig. 2a, c).Taken together, these observations demonstrate the benefit of permuting experimental conditions in binary PPI assays to increase detection (Fig. 5a-c). What remained unclear was the overall complementarity of assay versions when they are all combined. To visualize the different subsets of interactions recovered by the six different assays analyzed in this study (Figs. 2c, 3e), we displayed the performance of their corresponding 26 assay versions in the form of a checkerboard plot. Under conditions where none of the hsRRS-v2 pairs are scored positive, the different versions detected distinct but partially overlapping subsets of PPIs, the union of which reached 78% (Fig. 6). Importantly, as already observed in Fig. 3f, the combination of 12 N2H versions detected 60% of PRS pairs, which means that this single assay recovered over three-quarters of all PPIs detected in this study (Fig. 6). Taken together, these results showed that a single assay exploring multiple dimensions of the parameter space (Fig. 1b, c) will be sufficient to map a large fraction of binary PPIs in a defined search space.Fig. 6Complementarity of different assay versions. Detection of individual hsPRS-v2 pairs (left panel) by the assay versions used in Fig. 5a–c, under conditions where none of the hsRRS-v2 pairs (right panel) are scored positive. Detected hsPRS-v2 pairs are indicated by blue squares: light blue corresponds to individual assay versions and dark blue to the union of assay versions into distinct assays or combinations of assays. Identity of pairs can be found in Supplementary Table 4

Combining multi-version assays to maximize PPI detection

With all assays in hand to systematically evaluate the impact of the three variables presented in Fig. 1b, we decided to combine the different PPI assay versions previously tested against the improved reference sets (Figs. 2c, 3e) with respect to each individual parameter. All analyses were conducted in conditions where none of the random pairs are scored positive.

As the impact of permuting proteins X and Y from one tag to the other (Tag1-X:Tag2-Y vs. Tag1-Y:Tag2-X) had already been comprehensively investigated by six different assays in the literature23,26,27 (Supplementary Table 1), we only repeated Y2H and MAPPIT to explore this parameter. While a single, randomly picked orientation yielded PRS recovery rates of 19% and 32%, switching to the second orientation increased detection to 25% and 40% for Y2H and MAPPIT, respectively. Importantly, cumulative PPI detection increased from 38% to 47% when a single or two orientations were combined for both assays, respectively (Fig. 5a). This 1.2-fold (47/38) increase was consistent with our analysis of published results23,26,27 (Supplementary Fig. 3a) and confirmed that permuting the orientation of tested proteins is a valuable option to increase PPI detection.Fig. 5Impact of permuting assay parameters on binary PPI detection. a–c PPIs detected by permuting (a) protein orientations in Y2H and MAPPIT, (b) tagging configurations in KISS (only two versions tested), NanoBiT, GPCA, mN2H, yN2H and vN2H, and (c) protein expression environments in N2H. Error bars in (a-c) indicate standard deviations

Impact of permuting assay parameters on binary PPI detection. a–c PPIs detected by permuting (a) protein orientations in Y2H and MAPPIT, (b) tagging configurations in KISS (only two versions tested), NanoBiT, GPCA, mN2H, yN2H and vN2H, and (c) protein expression environments in N2H. Error bars in (a-c) indicate standard deviations

To probe the influence of different tagging configurations on PPI detection, i.e. the N- or C-terminal positioning of the assay tags, we systematically evaluated the contribution of each pairing combination (N1N2, C1N2, N1C2, and C1C2) with KISS, NanoBiT, GPCA, mN2H, yN2H, and vN2H. For KISS, the analysis was restricted to the two published configurations: N1N2 and C1N225. Using mN2H as a first example, we observed an average PRS detection rate of 19% for a single, randomly picked tagging configuration (Fig. 5b). When two of the four possible tagging configurations were used, the PRS recovery rate was further increased to 28%. Finally, when three and all four tagging configurations were combined, 34 and 37% of hsPRS-v2 pairs were identified (Fig. 5b). Thus, using more than one tagging configuration substantially increased the detection of binary PPIs by mN2H. Similar observations were made for the other assays as the PRS recovery rates were increased from an average 26% to 42% for KISS, where only two configurations were tested, and from 14% to 28% for NanoBiT, 19% to 35% for GPCA, 21% to 45% for yN2H and 19% to 33% for vN2H when only one or all four configurations were tested, respectively. The same trend was observed when combining all tested versions and a 1.6-fold increase in PRS detection obtained by extending assays from a single to multiple configurations (Fig. 5b, union). These results illustrate the gain of sensitivity achieved when performing the same assay with different combinations of tagging configurations, and confirm the preliminary observations made after reanalyzing published data23,25 (Supplementary Fig. 3b).

To evaluate the impact of different protein expression systems on binary PPI detection, we implemented different versions of the N2H assay in yeast, in mammalian cells and in vitro (cell-free). All four tagging configurations (Fig. 1b, c) were systematically tested in each environment. For the N1N2 version, an average PRS detection rate of 24% was reached when performing the assay in a single environment. This recovery was increased to 35% when switching to a second expression environment. Finally, after combining detected PRS pairs from all three environments, the cumulative recovery rate reached 42% for the N1N2 versions of N2H (Fig. 5c). When we expanded the analysis to the other tagging configurations, N1C2, C1C2, and C1N2, we consistently observed that performing the assay with different expression systems allowed the identification of a larger fraction of PPIs (Fig. 5c). Indeed, a robust growth of cumulative PRS detection was observed for N1C2 (from 18 to 40%), C1C2 (from 11 to 22%), and C1N2 (from 24 to 43%) when PPIs were tested in just one or all three different expression environments, respectively. Importantly, the cumulative PPI recovery rate for all 12 N2H assay versions tested here was increased from 38% to 60% when using one or all three expression systems, respectively, which represented an overall 1.6-fold increase in PPI detection (Fig. 5c, union). This result also highlights that the performance of the single multi-version N2H assay (60% PRS recovery) was significantly higher than that of any other method presented in this study (Fig. 2a, c).

Taken together, these observations demonstrate the benefit of permuting experimental conditions in binary PPI assays to increase detection (Fig. 5a-c). What remained unclear was the overall complementarity of assay versions when they are all combined. To visualize the different subsets of interactions recovered by the six different assays analyzed in this study (Figs. 2c, 3e), we displayed the performance of their corresponding 26 assay versions in the form of a checkerboard plot. Under conditions where none of the hsRRS-v2 pairs are scored positive, the different versions detected distinct but partially overlapping subsets of PPIs, the union of which reached 78% (Fig. 6). Importantly, as already observed in Fig. 3f, the combination of 12 N2H versions detected 60% of PRS pairs, which means that this single assay recovered over three-quarters of all PPIs detected in this study (Fig. 6). Taken together, these results showed that a single assay exploring multiple dimensions of the parameter space (Fig. 1b, c) will be sufficient to map a large fraction of binary PPIs in a defined search space.Fig. 6Complementarity of different assay versions. Detection of individual hsPRS-v2 pairs (left panel) by the assay versions used in Fig. 5a–c, under conditions where none of the hsRRS-v2 pairs (right panel) are scored positive. Detected hsPRS-v2 pairs are indicated by blue squares: light blue corresponds to individual assay versions and dark blue to the union of assay versions into distinct assays or combinations of assays. Identity of pairs can be found in Supplementary Table 4

Complementarity of different assay versions. Detection of individual hsPRS-v2 pairs (left panel) by the assay versions used in Fig. 5a–c, under conditions where none of the hsRRS-v2 pairs (right panel) are scored positive. Detected hsPRS-v2 pairs are indicated by blue squares: light blue corresponds to individual assay versions and dark blue to the union of assay versions into distinct assays or combinations of assays. Identity of pairs can be found in Supplementary Table 4

DiscussionOver the past 10 years, the community has continuously released new binary PPI detection tools. The benchmarking of these assays, using a first generation of positive (hsPRS-v1) and random (hsRRS-v1) reference sets, revealed that they often identify distinct subsets of interactions23. Therefore, a combination of multiple techniques will be required to increase overall PPI detection and reach completeness (Fig. 1a). Under conditions of optimal specificity, we have shown that no single PPI assay is exceptionally superior to any other, including the most recently developed technologies. Furthermore, we have established that instead of continuously developing fundamentally different techniques, it should be possible to reach completeness by assembling a minimal number of binary PPI assays, as long as different experimental conditions, or assay versions, can be implemented for each selected assay (Fig. 1b, c).We eliminated the possibility that the failure of 10 combined assays to detect about a third of known interactions in the original hsPRSv122 came from ORF clone quality issues or problematic interaction annotations by benchmarking assays against an improved, unbiased second-generation of reference sets: hsPRS-v2 and hsRRS-v2. Although the improvement of reference sets was purely focused on pair quality in this study, the next generation of PRS and RRS should include larger fractions of interactions such as those involved in higher order complex structures as well as interactions from co-complex crystal structures where direct contacts are well defined. Protein pairs for which a variety of KD measurements are available27 will also be included to characterize assays and assay versions according to their capabilities of recovering interactions of different strengths.Using the improved standards under conditions that maximized specificity, we have shown that considering three experimental parameters, i.e. protein orientation, tagging configuration and expression system (Fig. 1b), for assay development is a valuable option to increase PPI detection. In particular, we demonstrated that combining a relatively small number of assays exploring these parameters could lead to a PPI detection rate of up to 78%. In this context, we showed that combining 12 different versions of N2H, a new, highly versatile split NanoLuc-based assay, could recover 60% of all positive control interactions, similar to the 65% recovery rates obtained by combining either 10 published version-limited assays or five distinct multi-version assays. This means that the highly versatile N2H assay identified 77% of all PPIs detected in this study. In short, our results highlight the advantage of N2H or similar versatile systems, capable of handling several experimental parameters in a single, multi-version toolkit, to improve PPI detection and ultimately reduce the number of binary PPI assays required. This also suggests that only a few versatile binary PPI assays will be sufficient to reach complete coverage of any defined search space.Surprisingly, 22% of hsPRS-v2 pairs remained undetected after combining the six assays tested in this study, even though all interactions had similar numbers of experimental evidence and publications (Supplementary Fig. 6). We examined whether there was any correlation between a tested PPI assay in this study, and the original detection methods45 of the PRS pairs, but we did not find any correlation between these two elements. In two cases, the ORFs we used corresponded to shorter isoforms that did not contain reported interaction domains (Supplementary Table 5). Upon removal of these two ORFs, the PRS recovery rate by the union of all 26 assay versions yields 81%. However, two thirds of undetected hsPRS-v2 pairs contained reported interaction domains. Therefore, to reach completeness, we suggest that future assay development should be focused on expanding the number of versions covered by already-existing techniques. In that regard, we have extended our finding to the recently developed LuTHy assay27 and validated our original conclusions as additional PRS detection was observed when running all 16 versions (2 environments × 2 orientations × 4 configurations) allowed by this technology (data not shown). In addition, exploring yet untouched experimental parameters such as additional modifications of tag-protein fusions, or alternative expression environments including sub-cellular localization should also help reaching full coverage.Given the difficulties of performing heterogeneous PPI assays with the necessity of preparing different clones for expression vectors and setting up different experiments, performing an equal number of N2H assay versions to acquire comparable PPI detection offers a valuable alternative to reduce these efforts (Fig. 4a–e).The fact that expression environment has a strong impact on PPI detection is particularly interesting as it can easily be tuned with N2H or similar versatile assays. For example, protein pairs could be expressed in multiple cell lines, including distinct cell types and tissue origins, while a range of pH or salt concentrations could easily be tested when performing the assay in vitro. From a single set of plasmids, this would open access to almost unlimited assay versions to generate near-complete, high-quality binary interactome maps in a more time-effective manner.

Over the past 10 years, the community has continuously released new binary PPI detection tools. The benchmarking of these assays, using a first generation of positive (hsPRS-v1) and random (hsRRS-v1) reference sets, revealed that they often identify distinct subsets of interactions23. Therefore, a combination of multiple techniques will be required to increase overall PPI detection and reach completeness (Fig. 1a). Under conditions of optimal specificity, we have shown that no single PPI assay is exceptionally superior to any other, including the most recently developed technologies. Furthermore, we have established that instead of continuously developing fundamentally different techniques, it should be possible to reach completeness by assembling a minimal number of binary PPI assays, as long as different experimental conditions, or assay versions, can be implemented for each selected assay (Fig. 1b, c).

We eliminated the possibility that the failure of 10 combined assays to detect about a third of known interactions in the original hsPRSv122 came from ORF clone quality issues or problematic interaction annotations by benchmarking assays against an improved, unbiased second-generation of reference sets: hsPRS-v2 and hsRRS-v2. Although the improvement of reference sets was purely focused on pair quality in this study, the next generation of PRS and RRS should include larger fractions of interactions such as those involved in higher order complex structures as well as interactions from co-complex crystal structures where direct contacts are well defined. Protein pairs for which a variety of KD measurements are available27 will also be included to characterize assays and assay versions according to their capabilities of recovering interactions of different strengths.

Using the improved standards under conditions that maximized specificity, we have shown that considering three experimental parameters, i.e. protein orientation, tagging configuration and expression system (Fig. 1b), for assay development is a valuable option to increase PPI detection. In particular, we demonstrated that combining a relatively small number of assays exploring these parameters could lead to a PPI detection rate of up to 78%. In this context, we showed that combining 12 different versions of N2H, a new, highly versatile split NanoLuc-based assay, could recover 60% of all positive control interactions, similar to the 65% recovery rates obtained by combining either 10 published version-limited assays or five distinct multi-version assays. This means that the highly versatile N2H assay identified 77% of all PPIs detected in this study. In short, our results highlight the advantage of N2H or similar versatile systems, capable of handling several experimental parameters in a single, multi-version toolkit, to improve PPI detection and ultimately reduce the number of binary PPI assays required. This also suggests that only a few versatile binary PPI assays will be sufficient to reach complete coverage of any defined search space.

Surprisingly, 22% of hsPRS-v2 pairs remained undetected after combining the six assays tested in this study, even though all interactions had similar numbers of experimental evidence and publications (Supplementary Fig. 6). We examined whether there was any correlation between a tested PPI assay in this study, and the original detection methods45 of the PRS pairs, but we did not find any correlation between these two elements. In two cases, the ORFs we used corresponded to shorter isoforms that did not contain reported interaction domains (Supplementary Table 5). Upon removal of these two ORFs, the PRS recovery rate by the union of all 26 assay versions yields 81%. However, two thirds of undetected hsPRS-v2 pairs contained reported interaction domains. Therefore, to reach completeness, we suggest that future assay development should be focused on expanding the number of versions covered by already-existing techniques. In that regard, we have extended our finding to the recently developed LuTHy assay27 and validated our original conclusions as additional PRS detection was observed when running all 16 versions (2 environments × 2 orientations × 4 configurations) allowed by this technology (data not shown). In addition, exploring yet untouched experimental parameters such as additional modifications of tag-protein fusions, or alternative expression environments including sub-cellular localization should also help reaching full coverage.

Given the difficulties of performing heterogeneous PPI assays with the necessity of preparing different clones for expression vectors and setting up different experiments, performing an equal number of N2H assay versions to acquire comparable PPI detection offers a valuable alternative to reduce these efforts (Fig. 4a–e).

The fact that expression environment has a strong impact on PPI detection is particularly interesting as it can easily be tuned with N2H or similar versatile assays. For example, protein pairs could be expressed in multiple cell lines, including distinct cell types and tissue origins, while a range of pH or salt concentrations could easily be tested when performing the assay in vitro. From a single set of plasmids, this would open access to almost unlimited assay versions to generate near-complete, high-quality binary interactome maps in a more time-effective manner.

MethodsProcessing publicly available hsPRS-v1 and hsRRS-v1 dataPublicly available protein interaction datasets used in this study came from the original publications23,24,26 or were provided by the authors upon request25,27. Analyses were restricted to datasets for which a straightforward titration at a threshold of no RRS pair scoring positive could be applied. To evaluate the recovery rates of hsPRS-v1 when none of the RRS pairs are scored positive, data were calibrated such that the provided score for a given PRS pair must be strictly higher than the highest score of any of the RRS pairs. When authors clearly identified a particular pair as not interacting (negative) in their experiments, this assignment was also considered to generate the final dataset at a threshold where no RRS pair was scored positive. The same criterion was applied to evaluate the fractions of pairs found with available assay permutations, by using the highest RRS score found in each independent version of the corresponding assay.Construction of hsPRS-v2 and hsRRS-v2 reference setsSecond-generation hsPRS-v2 (60 pairs) and hsRRS-v2 (78 pairs) were constructed from pre-existing hsPRS-v1 (92 pairs) and hsRRS-v1 (92 pairs)23. To update the v1 sets generated over 10 years ago, we first decided to obtain single colony isolates for each ORF clone, removing all possible mutations that could have appeared in the original mini-pool ORFeome collection11,46. Each entry clone passed through multiple rounds of individual single colony isolations, plasmid DNA purifications, and Sanger DNA sequencings, and only full-length, fully sequence-verified clones (next generation sequencing) were selected. By comparing the ORF sequences with genome annotations, we removed ORFs that did not match annotations in Gencode 2747, resulting in 76 PRS pairs and 78 RRS pairs (91.8% success rate at the ORF level, Supplementary Fig. 2a). We further inspected the remaining 76 PRS pairs by literature mining and informatics filtering. We revisited the supporting evidence for each PRS pair using an updated classification of binary PPI detection methods (Supplementary Data 1). For example, pull down and co-immunoprecipitation are no longer considered as binary PPI detection methods12. Ten PRS pairs not satisfying binary classification were removed. Three PRS pairs containing HLA proteins were also discarded, as no reference ORF could be defined. Finally, three PRS pairs between precursor ligands (FGF1, CXCL1, and TNFSF10) and their respective receptors were also removed due to the requirement of proteolytic cleavages of the ligands for maturation (Supplementary Table 2, Supplementary Fig. 2b). After applying these filters, at total of 60 PRS pairs and 78 RRS pairs were thus assembled into hsPRS-v2 and hsRRS-v2, respectively (Supplementary Table 4). Discarded pairs from hsPRS-v1 and hsRRS-v1 are presented in Supplementary Data 1, and PubMed IDs supporting remaining hsPRS-v2 pairs are presented in Supplementary Data 2. The performances of the updated reference sets were compared to the older sets by repeating four assays previously benchmarked with hsPRS-v1 and hsRRS-v1 (Supplementary Fig. 2c).Cloning ORFs into Gateway-compatible expression plasmidsHuman hsPRS-v2 and hsRRS-v2 ORFs are available in pDONR221 or pDONR223 vectors, which make them compatible with the Gateway-cloning technology. Each ORF was introduced into the different assay-specific expression vectors used in this study via an LR clonase-mediated Gateway reaction (Life Technologies). LR reaction products were subsequently transformed into DH5α competent bacterial cells and grown for 24 h in ampicillin-containing TFB liquid medium. Plasmid DNA was extracted using a NucleoSpin 96 Plasmid kit from Macherey-Nagel. After PCR-amplification using plasmid-specific primers, the size of each DNA amplicon was examined by agarose gel electrophoresis. For each batch of ready-to-go destination vectors, DNA sequencing was performed on a subset of the cloned samples to check the quality of cloning before running the PPI assays.Construction of the tripartite promoter and N2H plasmidsThe design of the 3-in-1 hybrid promoter was based on the recent description of highly potent, minimal synthetic promoters in yeast S. cerevisiae48. These promoters comprise a short synthetic upstream activating sequence (UAS) and generic core elements, including a TATA box and a Transcription Start Site (TSS). The average distance separating the UAS motif upstream of the TATA box in native yeast and mammalian cell promoters was sufficient to insert the GC boxes and the CCAAT box-binding transcription factor (CTF) binding site of the herpes simplex virus-thymidine kinase (HSV-TK) promoter49 between these two elements. To boost expression levels in mammalian cells, the cytomegalovirus (CMV) immediate early enhancer was inserted upstream of the yeast synthetic UAS. To drive expression in vitro using a coupled transcription/translation system, a T7 promoter was inserted 30 nucleotides downstream of the TATA box, immediately followed by a cricket paralysis virus internal ribosome entry site (CrPV-IRES), a versatile IRES used to initiate translation in both, mammalian and yeast cells50,51. All these regulatory elements were assembled using gene synthesis (GenScript) and 40 nucleotides of sequence homology to the insertion sites in the vector backbone (pESC-LEU, Agilent, cat #217452; pESC-TRP, Agilent, cat #217453) were added for gap repair recombination in yeast52,53. NanoLuc fragments, F1 (1–65) and F2 (66–171), were linked to either the N-terminal (pDEST-N2H-N1, -N2) or C-terminal (pDEST-N2H-C1, -C2) extremity of the tested protein, using a flexible hinge encoding a polypeptide of 20 amino acids along with Gateway recombinational cloning sites. Finally, expression levels in the different systems were assessed using full length or fragmented NanoLuc (Promega, cat #N1001) reporter genes (Fig. 3b).Systematic evaluation of NanoLuc fragments complementationNanoLuc fragments were systematically generated by transposon-based insertional mutagenesis randomly distributed between nucleotides 55 and 249 of the NanoLuc coding sequence, corresponding to amino acid residues 19 and 83, respectively (Supplementary Fig. 4a). Resulting moieties were characterized by Sanger DNA sequencing using pMOD forward and reverse primers according to the manufacturer’s protocol (EZ-Tn5, epicenter-Illumina, #TNP10622). Each fragment was then subcloned into a specifically designed SNAP-Tag in vitro expression vector, using Gateway recombination cloning (Life Technologies). They were expressed using a human in vitro transcription/translation (IVT) system and their complementation was assessed using a proximity-based assay.Design of the SNAP-Tag in vitro expression vectorDNA encoding the SNAP-Tag was PCR-amplified from the pSNAPf vector (NEBs, #N9183S) to be inserted in frame into the pT7CFE1-CHis plasmid provided in the human IVT kit (ThermoFisher, #88882). The Gateway-cloning cassette was introduced immediately after the SNAP-tag sequence using the Gateway vector conversion system (Invitrogen, #11828–029). All subsequent cloning steps were performed using Gateway BP and LR reactions according to the manufacturer’s protocol. DNA constructs were prepared using standard maxi- and mini-prep protocols.Human IVT in vitro expression systemTo assess complementation of NanoLuc fragments, a HeLa cell lysate was used for in vitro coupled transcription and translation (IVT system, ThermoFisher, #88882). All the reactions were performed according to the manufacturer’s protocol, except for DNA concentrations (150 ng for both expression vectors were used) and the use of a 5× dilution of the HeLa cell lysate in the DB buffer. Concentrated DB buffer (10×) consisted in 540 mM HEPES-KOH pH: 7.6; 800 mM K-acetate; 40 mM Mg-acetate; 50 mM DTT; 5 mM Spermidine; 12.5 mM ATP and 200 mM Creatine phosphate. The final mixture containing the diluted HeLa cell lysate with the accessory proteins and the manufacturer’s reaction buffer was supplemented with 0.8 mM UTP, CTP; 1.2 mM GTP, and 50 µM of each amino acid (Promega, #L446A). T7 RNA polymerase (NEBs, #MO251L) were added at a final concentration of 1 unit/µL. Reaction was performed for 90 min at 30 °C, directly in 96 or 384-well plates.In vitro proximity assay based on SNAP-tag captureNanoLuc fragments fused to SNAP-tags were expressed in the human IVT system and assayed for complementation by generating a Biotin–Streptavidin complex. After IVT expression, the reaction mixture was incubated for 30 min at 30 °C with 1 µL/well of a 250 µM SNAP-biotin substrate dilution in fresh DMSO. Streptavidin–biotin complexes were further generated by adding 1 µL/well of 1 mg/mL of streptavidin solution (Sigma-Aldrich, #85878). The reconstituted luciferase enzymatic activity was monitored by injecting 50 µL/well of the Nano-Glo reagent (Promega, #N1120).Mammalian cell-based version of N2H assay (mN2H)HEK293T cells were seeded at 6x104 cells per well in 96-well, flat-bottom, cell culture microplates (Greiner Bio-One, #655083), and cultured in Dulbeccoʼs modified Eagleʼs medium (DMEM) supplemented with 10% fetal calf serum at 37 °C and 5% CO2. Twenty-four hour later, cells were transfected with 100 ng of each N2H plasmid (pDEST-N2H-N1, -N2, -C1 or -C2) using linear polyethylenimine (PEI) to co-express the protein pairs fused with complementary NanoLuc fragments, F1 and F2. The DNA/PEI ratio used for transfection was 1:3 (mass:mass). Twenty-four hour after transfection, the culture medium was removed and 50 μL of 100× diluted NanoLuc substrate (Promega, #N1110) was added to each well of a 96-well microplate containing the transfected cells. Plates were incubated for 3 min at room temperature. Luciferase enzymatic activity was measured using a TriStar or CentroXS luminometer (Berthold; 2 s integration time).The stock solution of PEI HCl (PEI MAX 40000; Polysciences Inc; Cat# 24765) was prepared according to the manufacturer’s instructions. Briefly, 200 mg of PEI powder added to 170 mL of water, stirred until complete dissolution, and pH was adjusted to 7 with 1 M NaOH. Water was added to obtain a final concentration of 1 mg/mL, and the stock solution was filtered through a 0.22 µm membrane.Yeast cell-based version of N2H assay (yN2H)Using a high-efficiency LiAc/SS carrier DNA/PEG protocol54, haploid S. cerevisiae Y8800 (MATa, leu2-3,112 trp1-901, his3Δ200, ade2-101 ura3-52, gal4Δ, gal80Δ, chy2R, GAL2::ADE2, GAL7::LacZ@met2, GAL1::HIS3@LYS2) and Y8930 (MATα, leu2-3,112 trp1-901, his3Δ200, ade2-101 ura3-52, gal4Δ, gal80Δ, chy2R, GAL2::ADE2, GAL7::LacZ@met2, GAL1::HIS3@LYS2) yeast strains were transformed with pDEST-N2H plasmids (pDEST-N2H-N1, -N2, -C1, and –C2) containing a TRP1 (N2/C2/Fragment 2 vectors) or a LEU2 (N1/C1/Fragment 1 vectors) cassette, respectively. For each individual transformation, 0.3 to 1.5 µg of high-purity plasmid and 1 mL of log phase yeast cell culture (OD600 nm ~0.6–0.9) were used. The resulting transformants carrying N2/C2/Fragment2 vectors (TRP1 cassette) or N1/C1/Fragment1 vectors (LEU2 cassette) were spotted (8 µL per spot) onto solid synthetic complete media lacking tryptophan (SC-W) or leucine (SC-L), respectively. After 3 days of incubation at 30 °C, each transformant growing on solid SC-W or SC-L was picked by scratching the corresponding spot and inoculated into a 96-well microplate (Fisher Scientific/Corning, #07200720A) containing 160 µL of liquid SC-W or SC-L per well, respectively. Inoculated yeast cells were incubated overnight (15–18 h), at 30 °C and under shaking conditions (220 rpm). Haploid yeast were mated by inoculating 5 µL of transformed MATa Y8800 and 5 µL of transformed MATα Y8930 into a 96-well microplate containing 160 µL of YEPD medium (yeast extract, peptone, dextrose) per well. Mating was obtained by growing cells overnight (15–18 h), at 30 °C and under shaking conditions (220 rpm). A first diploid selection was conducted by inoculating 10 µL of mated yeast cells into a 96-well microplate containing 160 µL of SC-LW medium per well. Cells were incubated overnight (15–18 h), at 30 °C and under shaking conditions (220 rpm). A second diploid selection was conducted by inoculating 50 µL from the first diploid selection culture into a 96-well deep well plate (QIAGEN, #19579) containing 1.2 mL of liquid SC-LW medium per well. Diploids were grown by incubating cells for 24 h, at 30 °C and under shaking conditions (900 rpm). Cell density was determined by transferring 100 µL of each diploid cell culture into individual wells of a 96-well microplate and measuring OD600 nm with a microplate reader (optional). Deep well plates containing diploid yeast cells were centrifuged at 2,500 g for 15 min. After centrifugation, the supernatant was discarded and residual SC-LW medium removed by gently blotting deep well plates with absorbent paper. Each yeast cell pellet was individually resuspended into 100 µL of the NanoLuc Assay solution by gently pipetting up-and-down several times, until cell pellets were fully dispersed. Homogenized solutions were transferred into white flat-bottom 96-well plates (Greiner bio-one, cat# 655073) that were then sealed with aluminum foil and incubated for 1 h at room temperature. After incubation, plates were shaken for 30 s at 950 rpm to resuspend yeast cells. Using a luminometer (Berthold) set up for one-second orbital shaking before each measurement (speed: fast; diameter: 5 mm), luminescence was evaluated for each reaction. Integration time of the luminescence signal was set at 2 s per sample.In vitro version of N2H assay (vN2H)Synthesis of proteins was performed separately using the in vitro coupled transcription–translation system based on Insect Cell Extract (ICE) (TnT-T7-ICE, Promega, #L1102). In this ICE coupled transcription–translation system, transcription is driven by T7 RNA-Polymerase present in the ICE reagent. The necessary T7 promoter is included in the N2H synthetic promoter as described in Fig. 3b. Each reaction was performed in 25 µL final volume with 1 µg of pDEST-N2H plasmid (5 µL at 200 ng/µL) and 20 µL of diluted insect cell extract (containing 4 µL of ICE, 3.2 µL of 5× ICB dilution buffer and eight units of T7-RNAP-Promega#P4074, supplemented with H2O up to 20 µL). Incubation was at 30 °C for 4 h. The synthesis reaction was stopped by addition of 50 µL of 1× PBS (Gibco, #14190–094) containing 20% glycerol (for storage at −80 °C). The NanoLuc assay was carried out in 96-well format by mixing 5 µL of each in vitro synthesized protein with 15 µL of 1× PBS per well. After incubation at 30 °C for 90 min, 50 µL of 100× diluted NanoLuc substrate (Promega, #N1110) were added, and luciferase enzymatic activity was measured using a TriStar or CentroXS luminometer (Berthold; 5 s integration time).Concentrated insect cell buffer (ICB 5×) consists of 150 mM HEPES-KOH pH: 7.6; 750 mM K-acetate; 19.5 mM Mg-acetate; 12.5 mM DTT; 500 µM spermidine; 100 mM creatine phosphate; 0.5 mg/mL creatinine kinase; 8.75 mM ATP; 1.5 mM GTP, CTP, UTP and 500 µM amino acids mix. The stock solution of amino acids (3.4 mM) was made by dissolving the 20 powders into 5 M KOH, at concentrations ranging from 1.6 to 4 M. The resulting mixtures containing each of the 20 amino acids were diluted in water to achieve a final concentration of 238 mM. The final aqueous solution was adjusted to pH 7.5 with acetic acid, diluted to 3.4 mM and frozen in liquid nitrogen for storage at −80 °C.Yeast 2-Hybrid (Y2H) assayUsing a high-efficiency LiAc/SS carrier DNA/PEG protocol54, haploid S. cerevisiae Y8800 (MATa, leu2-3,112 trp1-901, his3Δ200, ade2-101 ura3-52, gal4Δ, gal80Δ, chy2R, GAL2::ADE2, GAL7::LacZ@met2, GAL1::HIS3@LYS2) and Y8930 (MATα, leu2-3,112 trp1-901, his3Δ200, ade2-101 ura3-52, gal4Δ, gal80Δ, chy2R, GAL2::ADE2, GAL7::LacZ@met2, GAL1::HIS3@LYS2) yeast strains were transformed with pDEST-AD and pDEST-DB plasmids16 containing a TRP1 or a LEU2 cassette, respectively. For auto-activation tests (DB-X/Y vs. AD-empty), Y8800 (MATa) yeast cells were also transformed with a pDEST-AD-empty plasmid. Six standard Y2H controls were also transformed into yeast cells: (1) pDEST-AD-empty/pDEST-DB-empty, (2) pDEST-AD-CBLB/pDEST-DB-GRB2, (3) pDEST-AD-XIAP/pDEST-DB-CASP9, (4) pDEST-AD-empty/pDEST-DB-scGal4, (5) pDEST-AD-empty/pDEST-DB-WDR62, and (6) pDEST-AD-scUme6 (a.a. 531-a.a. 836)/pDEST-DB. For each individual transformation, 0.3–1.5 µg of high-purity plasmid and 1 mL of a log phase-growth yeast cell culture (OD600 nm ~0.6–0.9) were used. The resulting transformants carrying pDEST-AD vectors (TRP1 cassette) or pDEST-DB vectors (LEU2 cassette) were spotted (8 µL per spot) onto solid synthetic complete media lacking tryptophan (SC-W) or leucine (SC-L), respectively. After 3 days of incubation at 30 °C, each transformant growing on solid SC-W or SC-L was picked by scratching the corresponding spot, and inoculated into a 96-well costar plate (Fisher Scientific/Corning, #07200720A) containing 160 µL of liquid SC-W or SC-L per well, respectively. Inoculated yeast cells were incubated overnight (15–18 h), at 30 °C and under shaking conditions (220 rpm). Haploid yeast were mated by inoculating 5 µL of transformed MATa Y8800 and 5 µL of transformed MATα Y8930 into a 96-well microplate containing 160 µL of YEPD medium (yeast extract, peptone, dextrose) per well. The exact same mating was done for the auto-activation tests (DB-X/Y-containing cells mated with AD-empty-containing cells). Mating was obtained by growing cells overnight (15–18 h), at 30 °C and under shaking conditions (220 rpm). The diploid selection was conducted by inoculating 10 µL of mated yeast cells into a 96-well microplate containing 160 µL of SC-LW medium per well. Cells were incubated overnight (15–18 h), at 30 °C and under shaking conditions (220 rpm). Using a liquid handling robot, each diploid cell culture was then spotted in quadruplicate (1 μL/spot) onto two different solid media plates: (1) SC-LW, and (2) SC-LW lacking histidine and containing 1 mM of 3-Amino-1,2,4-triazole (3-AT) (SC-LWH, 1 mM 3-AT). The six controls described above were also spotted on each solid plate to validate composition of selective media (based on their known phenotypes on these media) and compare their growth rates to the ones of tested protein pairs. Plates were incubated for 4 days at 30 °C and 2 days at room temperature before evaluating growth scores.Gaussia princeps complementation assay (GPCA)HEK293T cells were cultured as described for the mN2H assay version. For each protein, 100 ng of purified plasmid DNA was transfected into HEK293T cells in 96-well, flat-bottom, cell culture plates (Greiner Bio-One, #655083) supplemented with 10%FBS in DMEM using polyethylenimine24. The DNA/PEI ratio (mass:mass) was 1:3. GPCA vectors carry the human cytomegalovirus (CMV) promoter and are maintained as high copy number with the human virus SV40 replication origin in mammalian cells. 24 h after DNA transfection, the cell culture medium was removed, cells were gently washed with 150 µL pre-warmed 1x PBS, 40 µL of lysis buffer was then added per well, and cell lysis was performed under vigorous shaking of the plate for 20 min at 900 rpm. Luminescence was measured by auto-injecting 50 µL Renilla luciferase substrate (Renilla Luciferase Assay system, catalog No. E2820, Promega) per well and integrating light output for 4 s using a TriStar luminometer (Berthold).Construction of C-terminal plasmids for GPCAGPCA C1 and C2 vectors are based on two fragments of the humanized Gaussia princeps luciferase (herein referred as GLuc), similar to the GPCA N1 and N2 vectors24. Both GLuc fragments were linked to the C-terminal extremity of the tested proteins by a flexible hinge polypeptide of 20 amino acids, including the Gateway recombinational cloning sites. To normalize mRNA translation initiation, a consensus Kozak translation start sequence was added. Both constructs were carried by the same CMV driven mammalian expression vector (pCI-neo derived, Promega) and were maintained as multi-copies within the cells via the presence of the SV40 replication origin.NanoLuc® Binary Technology (NanoBiT) assayHEK293T cells were cultured as described for the mN2H assay version. For each protein, 100 ng of purified plasmid DNA was transfected into HEK293T cells grown in 96-well, flat-bottom, cell culture microplates (Greiner Bio-One, #655083). The DNA/PEI ratio (mass:mass) was 1:3 for the transfection. 24 h later, 100 µL of the NanoLuc substrate (Promega, #N1110) were added to each well, plates were incubated for 3 min at room temperature, and luminescence output was measured using a TriStar luminometer (Berthold; 1 s integration time).Construction of Gateway-compatible plasmids for NanoBiTThe Gateway reading frame cassette was amplified by PCR, and polished with a PCR polishing kit (Agilent, # 200409) to generate blunt-end DNA. NanoBiT vectors (Promega, # N2014) with N- and C-terminus tagging configurations (NB MCS1, NB MCS2, NB MCS3, NB MCS4) were obtained from Promega, each containing an HSV-TK promoter for protein expression in mammalian cells and an ampicillin resistance marker for bacterial selection of transformed cells. Each plasmid DNA for NanoBiT vectors (NB MCS1-4) was digested with a single-cut restriction enzyme (EcoRI for NB MCS1 and 2, and SacI for NB MCS3 and 4) at the multiple cloning sites (MCS). The linearized plasmid DNA was processed with the PCR polishing kit generating blunt-end DNA. Antarctic phosphatase (NEB, M0289S) was used to remove 5′-phosphates to reduce self-ligation background of the linearized vector. The ligation reaction was performed with T4 DNA ligase (NEB, M0202S) joining the linearized NanoBiT vectors (NB MCS1-4) with the polished Gateway reading frame cassette. The ligated, Gateway-compatible NanoBiT vectors (NanoBiT-GW-N1, -N2, -C1 and –C2) were introduced into One ShotTM ccdB survivalTM 2 T1R competent E. coli cells (Invitrogen, #A10460) via heat shock bacterial transformation, and selected on chloramphenicol-containing LB agar plates. The sequences of constructed NanoBiT-GW vectors were examined by Sanger DNA sequencing. The final vectors contained correct sequences of the promoter, NanoBiT fragments, junctions, and Gateway cassette.MAPPIT and KISS assaysHEK293T cells were cultured as described for the mN2H assay version. In both Mammalian protein–protein interaction trap (MAPPIT) and Kinase substrate sensor (KISS assays)12,25, HEK293T cells were transfected with bait, prey and reporter plasmids applying a standard calcium phosphate transfection method. Luciferase activity was measured 48 h after transfection using the Luciferase Assay System kit (Promega) on a Enspire luminometer (Perkin-Elmer). For MAPPIT, half of the cells were stimulated with erythropoietin (2 ng/mL; GenScript) 24 h after transfection. For each tested pair in MAPPIT, the fold-induction value (signal from two stimulated culture wells divided by the signal from two unstimulated culture wells) was calculated. For KISS the average of four culture wells was used.Scoring hsPRS-v2 pairsData for each assay version performed in this study are presented in Source Data. All binary PPI experiments were independently performed three times. For the different versions of N2H experiments, the average signal of the three replicates was used to determine a raw luminescence value for each protein pair X/Y. Control experiments were performed similarly where protein X or Y fused to the NanoLuc fragment F1 or F2 was co-expressed with the matching NanoLuc fragment alone (X/empty or empty/Y). For each protein pair X–Y, we calculated a normalized luminescence ratio (NLR) corresponding to the raw luminescence value of the tested pair (X–Y) divided by the maximum luminescence value from one of the two controls (X-empty or empty-Y), as summarized in Eq. (1) 25:1\documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$${\mathrm{NLR}}_{{\mathrm{X}} - {\mathrm{Y}}} = \frac{{{\mathrm{Luminescence}}_{{\mathrm{X}} - {\mathrm{Y}}}}}{{{\mathrm{Maximum}}\,{\mathrm{Luminescence}}_{({\mathrm{X}} - {\mathrm{empty}}){\mathrm{or}}({\mathrm{empty}} - {\mathrm{Y}})}}}.$$\end{document}NLRX-Y=LuminescenceX-YMaximumLuminescence(X-empty)or(empty-Y).To identify protein pairs that score positive at a threshold of no RRS detection, two criteria were considered: (1) NLRPRS > NLRHIGHEST RRS and (2) NLRPRS > 1.0. All PRS pairs that did not meet these two criteria were not scored and therefore defined as not detectable in the corresponding assay version. The same analysis and criteria were applied for both KISS and MAPPIT assays.For GPCA and NanoBiT experiments, a PRS pair was scored positive if its corresponding raw luminescence signal (from the average of three independent experiments) was strictly higher than the RRS pair having the highest raw luminescence signal (from the average of three independent experiments).For Y2H assay, diploid yeast cells containing both, DB-X/Y bait and AD-Y/X prey plasmids, were spotted in quadruplicate on selective medium supplemented with 3-AT (SC-LWH, 3-AT). After titration of 3-AT, a minimal concentration of 1 mM was selected as no RRS pairs were detected for this specific threshold12. In parallel, diploid yeast cells containing DB-X/Y bait and AD-empty prey vectors were also used to identify bait proteins that are auto-activators of the HIS3 selection marker. After 6 days of culture, yeast growth was visually determined and scored from “0” (no growth) to “4” (strong growth) for both the DB-X/Y:AD-Y/X and DB-X/Y:AD-empty combinations. If the growth score was >0 for the DB-X/Y:AD-Y/X combination in at least two of the three experiments performed and was superior by at least 2 units to the one of DB-X/Y:AD-empty combination, a protein pair was scored positive.Construction of cumulative recovery plotsTo plot the fraction of interaction recovered for n number of assays or assay versions, the recovery rate for each of the possible n combinations was calculated first (Supplementary Figs. 1c, 2d, 5a) and then averaged (Figs. 2b, d, 3f and Supplementary Fig. 5b). The error bars in Figs. 2b, d, 3f and Supplementary Fig. 5b indicate the standard deviation of the average.Reporting summaryFurther information on research design is available in the Nature Research Reporting Summary linked to this article.

Processing publicly available hsPRS-v1 and hsRRS-v1 dataPublicly available protein interaction datasets used in this study came from the original publications23,24,26 or were provided by the authors upon request25,27. Analyses were restricted to datasets for which a straightforward titration at a threshold of no RRS pair scoring positive could be applied. To evaluate the recovery rates of hsPRS-v1 when none of the RRS pairs are scored positive, data were calibrated such that the provided score for a given PRS pair must be strictly higher than the highest score of any of the RRS pairs. When authors clearly identified a particular pair as not interacting (negative) in their experiments, this assignment was also considered to generate the final dataset at a threshold where no RRS pair was scored positive. The same criterion was applied to evaluate the fractions of pairs found with available assay permutations, by using the highest RRS score found in each independent version of the corresponding assay.

Processing publicly available hsPRS-v1 and hsRRS-v1 data

Publicly available protein interaction datasets used in this study came from the original publications23,24,26 or were provided by the authors upon request25,27. Analyses were restricted to datasets for which a straightforward titration at a threshold of no RRS pair scoring positive could be applied. To evaluate the recovery rates of hsPRS-v1 when none of the RRS pairs are scored positive, data were calibrated such that the provided score for a given PRS pair must be strictly higher than the highest score of any of the RRS pairs. When authors clearly identified a particular pair as not interacting (negative) in their experiments, this assignment was also considered to generate the final dataset at a threshold where no RRS pair was scored positive. The same criterion was applied to evaluate the fractions of pairs found with available assay permutations, by using the highest RRS score found in each independent version of the corresponding assay.

Construction of hsPRS-v2 and hsRRS-v2 reference setsSecond-generation hsPRS-v2 (60 pairs) and hsRRS-v2 (78 pairs) were constructed from pre-existing hsPRS-v1 (92 pairs) and hsRRS-v1 (92 pairs)23. To update the v1 sets generated over 10 years ago, we first decided to obtain single colony isolates for each ORF clone, removing all possible mutations that could have appeared in the original mini-pool ORFeome collection11,46. Each entry clone passed through multiple rounds of individual single colony isolations, plasmid DNA purifications, and Sanger DNA sequencings, and only full-length, fully sequence-verified clones (next generation sequencing) were selected. By comparing the ORF sequences with genome annotations, we removed ORFs that did not match annotations in Gencode 2747, resulting in 76 PRS pairs and 78 RRS pairs (91.8% success rate at the ORF level, Supplementary Fig. 2a). We further inspected the remaining 76 PRS pairs by literature mining and informatics filtering. We revisited the supporting evidence for each PRS pair using an updated classification of binary PPI detection methods (Supplementary Data 1). For example, pull down and co-immunoprecipitation are no longer considered as binary PPI detection methods12. Ten PRS pairs not satisfying binary classification were removed. Three PRS pairs containing HLA proteins were also discarded, as no reference ORF could be defined. Finally, three PRS pairs between precursor ligands (FGF1, CXCL1, and TNFSF10) and their respective receptors were also removed due to the requirement of proteolytic cleavages of the ligands for maturation (Supplementary Table 2, Supplementary Fig. 2b). After applying these filters, at total of 60 PRS pairs and 78 RRS pairs were thus assembled into hsPRS-v2 and hsRRS-v2, respectively (Supplementary Table 4). Discarded pairs from hsPRS-v1 and hsRRS-v1 are presented in Supplementary Data 1, and PubMed IDs supporting remaining hsPRS-v2 pairs are presented in Supplementary Data 2. The performances of the updated reference sets were compared to the older sets by repeating four assays previously benchmarked with hsPRS-v1 and hsRRS-v1 (Supplementary Fig. 2c).

Construction of hsPRS-v2 and hsRRS-v2 reference sets

Second-generation hsPRS-v2 (60 pairs) and hsRRS-v2 (78 pairs) were constructed from pre-existing hsPRS-v1 (92 pairs) and hsRRS-v1 (92 pairs)23. To update the v1 sets generated over 10 years ago, we first decided to obtain single colony isolates for each ORF clone, removing all possible mutations that could have appeared in the original mini-pool ORFeome collection11,46. Each entry clone passed through multiple rounds of individual single colony isolations, plasmid DNA purifications, and Sanger DNA sequencings, and only full-length, fully sequence-verified clones (next generation sequencing) were selected. By comparing the ORF sequences with genome annotations, we removed ORFs that did not match annotations in Gencode 2747, resulting in 76 PRS pairs and 78 RRS pairs (91.8% success rate at the ORF level, Supplementary Fig. 2a). We further inspected the remaining 76 PRS pairs by literature mining and informatics filtering. We revisited the supporting evidence for each PRS pair using an updated classification of binary PPI detection methods (Supplementary Data 1). For example, pull down and co-immunoprecipitation are no longer considered as binary PPI detection methods12. Ten PRS pairs not satisfying binary classification were removed. Three PRS pairs containing HLA proteins were also discarded, as no reference ORF could be defined. Finally, three PRS pairs between precursor ligands (FGF1, CXCL1, and TNFSF10) and their respective receptors were also removed due to the requirement of proteolytic cleavages of the ligands for maturation (Supplementary Table 2, Supplementary Fig. 2b). After applying these filters, at total of 60 PRS pairs and 78 RRS pairs were thus assembled into hsPRS-v2 and hsRRS-v2, respectively (Supplementary Table 4). Discarded pairs from hsPRS-v1 and hsRRS-v1 are presented in Supplementary Data 1, and PubMed IDs supporting remaining hsPRS-v2 pairs are presented in Supplementary Data 2. The performances of the updated reference sets were compared to the older sets by repeating four assays previously benchmarked with hsPRS-v1 and hsRRS-v1 (Supplementary Fig. 2c).

Cloning ORFs into Gateway-compatible expression plasmidsHuman hsPRS-v2 and hsRRS-v2 ORFs are available in pDONR221 or pDONR223 vectors, which make them compatible with the Gateway-cloning technology. Each ORF was introduced into the different assay-specific expression vectors used in this study via an LR clonase-mediated Gateway reaction (Life Technologies). LR reaction products were subsequently transformed into DH5α competent bacterial cells and grown for 24 h in ampicillin-containing TFB liquid medium. Plasmid DNA was extracted using a NucleoSpin 96 Plasmid kit from Macherey-Nagel. After PCR-amplification using plasmid-specific primers, the size of each DNA amplicon was examined by agarose gel electrophoresis. For each batch of ready-to-go destination vectors, DNA sequencing was performed on a subset of the cloned samples to check the quality of cloning before running the PPI assays.

Cloning ORFs into Gateway-compatible expression plasmids

Human hsPRS-v2 and hsRRS-v2 ORFs are available in pDONR221 or pDONR223 vectors, which make them compatible with the Gateway-cloning technology. Each ORF was introduced into the different assay-specific expression vectors used in this study via an LR clonase-mediated Gateway reaction (Life Technologies). LR reaction products were subsequently transformed into DH5α competent bacterial cells and grown for 24 h in ampicillin-containing TFB liquid medium. Plasmid DNA was extracted using a NucleoSpin 96 Plasmid kit from Macherey-Nagel. After PCR-amplification using plasmid-specific primers, the size of each DNA amplicon was examined by agarose gel electrophoresis. For each batch of ready-to-go destination vectors, DNA sequencing was performed on a subset of the cloned samples to check the quality of cloning before running the PPI assays.

Construction of the tripartite promoter and N2H plasmidsThe design of the 3-in-1 hybrid promoter was based on the recent description of highly potent, minimal synthetic promoters in yeast S. cerevisiae48. These promoters comprise a short synthetic upstream activating sequence (UAS) and generic core elements, including a TATA box and a Transcription Start Site (TSS). The average distance separating the UAS motif upstream of the TATA box in native yeast and mammalian cell promoters was sufficient to insert the GC boxes and the CCAAT box-binding transcription factor (CTF) binding site of the herpes simplex virus-thymidine kinase (HSV-TK) promoter49 between these two elements. To boost expression levels in mammalian cells, the cytomegalovirus (CMV) immediate early enhancer was inserted upstream of the yeast synthetic UAS. To drive expression in vitro using a coupled transcription/translation system, a T7 promoter was inserted 30 nucleotides downstream of the TATA box, immediately followed by a cricket paralysis virus internal ribosome entry site (CrPV-IRES), a versatile IRES used to initiate translation in both, mammalian and yeast cells50,51. All these regulatory elements were assembled using gene synthesis (GenScript) and 40 nucleotides of sequence homology to the insertion sites in the vector backbone (pESC-LEU, Agilent, cat #217452; pESC-TRP, Agilent, cat #217453) were added for gap repair recombination in yeast52,53. NanoLuc fragments, F1 (1–65) and F2 (66–171), were linked to either the N-terminal (pDEST-N2H-N1, -N2) or C-terminal (pDEST-N2H-C1, -C2) extremity of the tested protein, using a flexible hinge encoding a polypeptide of 20 amino acids along with Gateway recombinational cloning sites. Finally, expression levels in the different systems were assessed using full length or fragmented NanoLuc (Promega, cat #N1001) reporter genes (Fig. 3b).

Construction of the tripartite promoter and N2H plasmids

The design of the 3-in-1 hybrid promoter was based on the recent description of highly potent, minimal synthetic promoters in yeast S. cerevisiae48. These promoters comprise a short synthetic upstream activating sequence (UAS) and generic core elements, including a TATA box and a Transcription Start Site (TSS). The average distance separating the UAS motif upstream of the TATA box in native yeast and mammalian cell promoters was sufficient to insert the GC boxes and the CCAAT box-binding transcription factor (CTF) binding site of the herpes simplex virus-thymidine kinase (HSV-TK) promoter49 between these two elements. To boost expression levels in mammalian cells, the cytomegalovirus (CMV) immediate early enhancer was inserted upstream of the yeast synthetic UAS. To drive expression in vitro using a coupled transcription/translation system, a T7 promoter was inserted 30 nucleotides downstream of the TATA box, immediately followed by a cricket paralysis virus internal ribosome entry site (CrPV-IRES), a versatile IRES used to initiate translation in both, mammalian and yeast cells50,51. All these regulatory elements were assembled using gene synthesis (GenScript) and 40 nucleotides of sequence homology to the insertion sites in the vector backbone (pESC-LEU, Agilent, cat #217452; pESC-TRP, Agilent, cat #217453) were added for gap repair recombination in yeast52,53. NanoLuc fragments, F1 (1–65) and F2 (66–171), were linked to either the N-terminal (pDEST-N2H-N1, -N2) or C-terminal (pDEST-N2H-C1, -C2) extremity of the tested protein, using a flexible hinge encoding a polypeptide of 20 amino acids along with Gateway recombinational cloning sites. Finally, expression levels in the different systems were assessed using full length or fragmented NanoLuc (Promega, cat #N1001) reporter genes (Fig. 3b).

Systematic evaluation of NanoLuc fragments complementationNanoLuc fragments were systematically generated by transposon-based insertional mutagenesis randomly distributed between nucleotides 55 and 249 of the NanoLuc coding sequence, corresponding to amino acid residues 19 and 83, respectively (Supplementary Fig. 4a). Resulting moieties were characterized by Sanger DNA sequencing using pMOD forward and reverse primers according to the manufacturer’s protocol (EZ-Tn5, epicenter-Illumina, #TNP10622). Each fragment was then subcloned into a specifically designed SNAP-Tag in vitro expression vector, using Gateway recombination cloning (Life Technologies). They were expressed using a human in vitro transcription/translation (IVT) system and their complementation was assessed using a proximity-based assay.

Systematic evaluation of NanoLuc fragments complementation

NanoLuc fragments were systematically generated by transposon-based insertional mutagenesis randomly distributed between nucleotides 55 and 249 of the NanoLuc coding sequence, corresponding to amino acid residues 19 and 83, respectively (Supplementary Fig. 4a). Resulting moieties were characterized by Sanger DNA sequencing using pMOD forward and reverse primers according to the manufacturer’s protocol (EZ-Tn5, epicenter-Illumina, #TNP10622). Each fragment was then subcloned into a specifically designed SNAP-Tag in vitro expression vector, using Gateway recombination cloning (Life Technologies). They were expressed using a human in vitro transcription/translation (IVT) system and their complementation was assessed using a proximity-based assay.

Design of the SNAP-Tag in vitro expression vectorDNA encoding the SNAP-Tag was PCR-amplified from the pSNAPf vector (NEBs, #N9183S) to be inserted in frame into the pT7CFE1-CHis plasmid provided in the human IVT kit (ThermoFisher, #88882). The Gateway-cloning cassette was introduced immediately after the SNAP-tag sequence using the Gateway vector conversion system (Invitrogen, #11828–029). All subsequent cloning steps were performed using Gateway BP and LR reactions according to the manufacturer’s protocol. DNA constructs were prepared using standard maxi- and mini-prep protocols.

Design of the SNAP-Tag in vitro expression vector

DNA encoding the SNAP-Tag was PCR-amplified from the pSNAPf vector (NEBs, #N9183S) to be inserted in frame into the pT7CFE1-CHis plasmid provided in the human IVT kit (ThermoFisher, #88882). The Gateway-cloning cassette was introduced immediately after the SNAP-tag sequence using the Gateway vector conversion system (Invitrogen, #11828–029). All subsequent cloning steps were performed using Gateway BP and LR reactions according to the manufacturer’s protocol. DNA constructs were prepared using standard maxi- and mini-prep protocols.

Human IVT in vitro expression systemTo assess complementation of NanoLuc fragments, a HeLa cell lysate was used for in vitro coupled transcription and translation (IVT system, ThermoFisher, #88882). All the reactions were performed according to the manufacturer’s protocol, except for DNA concentrations (150 ng for both expression vectors were used) and the use of a 5× dilution of the HeLa cell lysate in the DB buffer. Concentrated DB buffer (10×) consisted in 540 mM HEPES-KOH pH: 7.6; 800 mM K-acetate; 40 mM Mg-acetate; 50 mM DTT; 5 mM Spermidine; 12.5 mM ATP and 200 mM Creatine phosphate. The final mixture containing the diluted HeLa cell lysate with the accessory proteins and the manufacturer’s reaction buffer was supplemented with 0.8 mM UTP, CTP; 1.2 mM GTP, and 50 µM of each amino acid (Promega, #L446A). T7 RNA polymerase (NEBs, #MO251L) were added at a final concentration of 1 unit/µL. Reaction was performed for 90 min at 30 °C, directly in 96 or 384-well plates.

Human IVT in vitro expression system

To assess complementation of NanoLuc fragments, a HeLa cell lysate was used for in vitro coupled transcription and translation (IVT system, ThermoFisher, #88882). All the reactions were performed according to the manufacturer’s protocol, except for DNA concentrations (150 ng for both expression vectors were used) and the use of a 5× dilution of the HeLa cell lysate in the DB buffer. Concentrated DB buffer (10×) consisted in 540 mM HEPES-KOH pH: 7.6; 800 mM K-acetate; 40 mM Mg-acetate; 50 mM DTT; 5 mM Spermidine; 12.5 mM ATP and 200 mM Creatine phosphate. The final mixture containing the diluted HeLa cell lysate with the accessory proteins and the manufacturer’s reaction buffer was supplemented with 0.8 mM UTP, CTP; 1.2 mM GTP, and 50 µM of each amino acid (Promega, #L446A). T7 RNA polymerase (NEBs, #MO251L) were added at a final concentration of 1 unit/µL. Reaction was performed for 90 min at 30 °C, directly in 96 or 384-well plates.

In vitro proximity assay based on SNAP-tag captureNanoLuc fragments fused to SNAP-tags were expressed in the human IVT system and assayed for complementation by generating a Biotin–Streptavidin complex. After IVT expression, the reaction mixture was incubated for 30 min at 30 °C with 1 µL/well of a 250 µM SNAP-biotin substrate dilution in fresh DMSO. Streptavidin–biotin complexes were further generated by adding 1 µL/well of 1 mg/mL of streptavidin solution (Sigma-Aldrich, #85878). The reconstituted luciferase enzymatic activity was monitored by injecting 50 µL/well of the Nano-Glo reagent (Promega, #N1120).

In vitro proximity assay based on SNAP-tag capture

NanoLuc fragments fused to SNAP-tags were expressed in the human IVT system and assayed for complementation by generating a Biotin–Streptavidin complex. After IVT expression, the reaction mixture was incubated for 30 min at 30 °C with 1 µL/well of a 250 µM SNAP-biotin substrate dilution in fresh DMSO. Streptavidin–biotin complexes were further generated by adding 1 µL/well of 1 mg/mL of streptavidin solution (Sigma-Aldrich, #85878). The reconstituted luciferase enzymatic activity was monitored by injecting 50 µL/well of the Nano-Glo reagent (Promega, #N1120).

Mammalian cell-based version of N2H assay (mN2H)HEK293T cells were seeded at 6x104 cells per well in 96-well, flat-bottom, cell culture microplates (Greiner Bio-One, #655083), and cultured in Dulbeccoʼs modified Eagleʼs medium (DMEM) supplemented with 10% fetal calf serum at 37 °C and 5% CO2. Twenty-four hour later, cells were transfected with 100 ng of each N2H plasmid (pDEST-N2H-N1, -N2, -C1 or -C2) using linear polyethylenimine (PEI) to co-express the protein pairs fused with complementary NanoLuc fragments, F1 and F2. The DNA/PEI ratio used for transfection was 1:3 (mass:mass). Twenty-four hour after transfection, the culture medium was removed and 50 μL of 100× diluted NanoLuc substrate (Promega, #N1110) was added to each well of a 96-well microplate containing the transfected cells. Plates were incubated for 3 min at room temperature. Luciferase enzymatic activity was measured using a TriStar or CentroXS luminometer (Berthold; 2 s integration time).The stock solution of PEI HCl (PEI MAX 40000; Polysciences Inc; Cat# 24765) was prepared according to the manufacturer’s instructions. Briefly, 200 mg of PEI powder added to 170 mL of water, stirred until complete dissolution, and pH was adjusted to 7 with 1 M NaOH. Water was added to obtain a final concentration of 1 mg/mL, and the stock solution was filtered through a 0.22 µm membrane.

Mammalian cell-based version of N2H assay (mN2H)

HEK293T cells were seeded at 6x104 cells per well in 96-well, flat-bottom, cell culture microplates (Greiner Bio-One, #655083), and cultured in Dulbeccoʼs modified Eagleʼs medium (DMEM) supplemented with 10% fetal calf serum at 37 °C and 5% CO2. Twenty-four hour later, cells were transfected with 100 ng of each N2H plasmid (pDEST-N2H-N1, -N2, -C1 or -C2) using linear polyethylenimine (PEI) to co-express the protein pairs fused with complementary NanoLuc fragments, F1 and F2. The DNA/PEI ratio used for transfection was 1:3 (mass:mass). Twenty-four hour after transfection, the culture medium was removed and 50 μL of 100× diluted NanoLuc substrate (Promega, #N1110) was added to each well of a 96-well microplate containing the transfected cells. Plates were incubated for 3 min at room temperature. Luciferase enzymatic activity was measured using a TriStar or CentroXS luminometer (Berthold; 2 s integration time).

The stock solution of PEI HCl (PEI MAX 40000; Polysciences Inc; Cat# 24765) was prepared according to the manufacturer’s instructions. Briefly, 200 mg of PEI powder added to 170 mL of water, stirred until complete dissolution, and pH was adjusted to 7 with 1 M NaOH. Water was added to obtain a final concentration of 1 mg/mL, and the stock solution was filtered through a 0.22 µm membrane.

Yeast cell-based version of N2H assay (yN2H)Using a high-efficiency LiAc/SS carrier DNA/PEG protocol54, haploid S. cerevisiae Y8800 (MATa, leu2-3,112 trp1-901, his3Δ200, ade2-101 ura3-52, gal4Δ, gal80Δ, chy2R, GAL2::ADE2, GAL7::LacZ@met2, GAL1::HIS3@LYS2) and Y8930 (MATα, leu2-3,112 trp1-901, his3Δ200, ade2-101 ura3-52, gal4Δ, gal80Δ, chy2R, GAL2::ADE2, GAL7::LacZ@met2, GAL1::HIS3@LYS2) yeast strains were transformed with pDEST-N2H plasmids (pDEST-N2H-N1, -N2, -C1, and –C2) containing a TRP1 (N2/C2/Fragment 2 vectors) or a LEU2 (N1/C1/Fragment 1 vectors) cassette, respectively. For each individual transformation, 0.3 to 1.5 µg of high-purity plasmid and 1 mL of log phase yeast cell culture (OD600 nm ~0.6–0.9) were used. The resulting transformants carrying N2/C2/Fragment2 vectors (TRP1 cassette) or N1/C1/Fragment1 vectors (LEU2 cassette) were spotted (8 µL per spot) onto solid synthetic complete media lacking tryptophan (SC-W) or leucine (SC-L), respectively. After 3 days of incubation at 30 °C, each transformant growing on solid SC-W or SC-L was picked by scratching the corresponding spot and inoculated into a 96-well microplate (Fisher Scientific/Corning, #07200720A) containing 160 µL of liquid SC-W or SC-L per well, respectively. Inoculated yeast cells were incubated overnight (15–18 h), at 30 °C and under shaking conditions (220 rpm). Haploid yeast were mated by inoculating 5 µL of transformed MATa Y8800 and 5 µL of transformed MATα Y8930 into a 96-well microplate containing 160 µL of YEPD medium (yeast extract, peptone, dextrose) per well. Mating was obtained by growing cells overnight (15–18 h), at 30 °C and under shaking conditions (220 rpm). A first diploid selection was conducted by inoculating 10 µL of mated yeast cells into a 96-well microplate containing 160 µL of SC-LW medium per well. Cells were incubated overnight (15–18 h), at 30 °C and under shaking conditions (220 rpm). A second diploid selection was conducted by inoculating 50 µL from the first diploid selection culture into a 96-well deep well plate (QIAGEN, #19579) containing 1.2 mL of liquid SC-LW medium per well. Diploids were grown by incubating cells for 24 h, at 30 °C and under shaking conditions (900 rpm). Cell density was determined by transferring 100 µL of each diploid cell culture into individual wells of a 96-well microplate and measuring OD600 nm with a microplate reader (optional). Deep well plates containing diploid yeast cells were centrifuged at 2,500 g for 15 min. After centrifugation, the supernatant was discarded and residual SC-LW medium removed by gently blotting deep well plates with absorbent paper. Each yeast cell pellet was individually resuspended into 100 µL of the NanoLuc Assay solution by gently pipetting up-and-down several times, until cell pellets were fully dispersed. Homogenized solutions were transferred into white flat-bottom 96-well plates (Greiner bio-one, cat# 655073) that were then sealed with aluminum foil and incubated for 1 h at room temperature. After incubation, plates were shaken for 30 s at 950 rpm to resuspend yeast cells. Using a luminometer (Berthold) set up for one-second orbital shaking before each measurement (speed: fast; diameter: 5 mm), luminescence was evaluated for each reaction. Integration time of the luminescence signal was set at 2 s per sample.

Yeast cell-based version of N2H assay (yN2H)

Using a high-efficiency LiAc/SS carrier DNA/PEG protocol54, haploid S. cerevisiae Y8800 (MATa, leu2-3,112 trp1-901, his3Δ200, ade2-101 ura3-52, gal4Δ, gal80Δ, chy2R, GAL2::ADE2, GAL7::LacZ@met2, GAL1::HIS3@LYS2) and Y8930 (MATα, leu2-3,112 trp1-901, his3Δ200, ade2-101 ura3-52, gal4Δ, gal80Δ, chy2R, GAL2::ADE2, GAL7::LacZ@met2, GAL1::HIS3@LYS2) yeast strains were transformed with pDEST-N2H plasmids (pDEST-N2H-N1, -N2, -C1, and –C2) containing a TRP1 (N2/C2/Fragment 2 vectors) or a LEU2 (N1/C1/Fragment 1 vectors) cassette, respectively. For each individual transformation, 0.3 to 1.5 µg of high-purity plasmid and 1 mL of log phase yeast cell culture (OD600 nm ~0.6–0.9) were used. The resulting transformants carrying N2/C2/Fragment2 vectors (TRP1 cassette) or N1/C1/Fragment1 vectors (LEU2 cassette) were spotted (8 µL per spot) onto solid synthetic complete media lacking tryptophan (SC-W) or leucine (SC-L), respectively. After 3 days of incubation at 30 °C, each transformant growing on solid SC-W or SC-L was picked by scratching the corresponding spot and inoculated into a 96-well microplate (Fisher Scientific/Corning, #07200720A) containing 160 µL of liquid SC-W or SC-L per well, respectively. Inoculated yeast cells were incubated overnight (15–18 h), at 30 °C and under shaking conditions (220 rpm). Haploid yeast were mated by inoculating 5 µL of transformed MATa Y8800 and 5 µL of transformed MATα Y8930 into a 96-well microplate containing 160 µL of YEPD medium (yeast extract, peptone, dextrose) per well. Mating was obtained by growing cells overnight (15–18 h), at 30 °C and under shaking conditions (220 rpm). A first diploid selection was conducted by inoculating 10 µL of mated yeast cells into a 96-well microplate containing 160 µL of SC-LW medium per well. Cells were incubated overnight (15–18 h), at 30 °C and under shaking conditions (220 rpm). A second diploid selection was conducted by inoculating 50 µL from the first diploid selection culture into a 96-well deep well plate (QIAGEN, #19579) containing 1.2 mL of liquid SC-LW medium per well. Diploids were grown by incubating cells for 24 h, at 30 °C and under shaking conditions (900 rpm). Cell density was determined by transferring 100 µL of each diploid cell culture into individual wells of a 96-well microplate and measuring OD600 nm with a microplate reader (optional). Deep well plates containing diploid yeast cells were centrifuged at 2,500 g for 15 min. After centrifugation, the supernatant was discarded and residual SC-LW medium removed by gently blotting deep well plates with absorbent paper. Each yeast cell pellet was individually resuspended into 100 µL of the NanoLuc Assay solution by gently pipetting up-and-down several times, until cell pellets were fully dispersed. Homogenized solutions were transferred into white flat-bottom 96-well plates (Greiner bio-one, cat# 655073) that were then sealed with aluminum foil and incubated for 1 h at room temperature. After incubation, plates were shaken for 30 s at 950 rpm to resuspend yeast cells. Using a luminometer (Berthold) set up for one-second orbital shaking before each measurement (speed: fast; diameter: 5 mm), luminescence was evaluated for each reaction. Integration time of the luminescence signal was set at 2 s per sample.

In vitro version of N2H assay (vN2H)Synthesis of proteins was performed separately using the in vitro coupled transcription–translation system based on Insect Cell Extract (ICE) (TnT-T7-ICE, Promega, #L1102). In this ICE coupled transcription–translation system, transcription is driven by T7 RNA-Polymerase present in the ICE reagent. The necessary T7 promoter is included in the N2H synthetic promoter as described in Fig. 3b. Each reaction was performed in 25 µL final volume with 1 µg of pDEST-N2H plasmid (5 µL at 200 ng/µL) and 20 µL of diluted insect cell extract (containing 4 µL of ICE, 3.2 µL of 5× ICB dilution buffer and eight units of T7-RNAP-Promega#P4074, supplemented with H2O up to 20 µL). Incubation was at 30 °C for 4 h. The synthesis reaction was stopped by addition of 50 µL of 1× PBS (Gibco, #14190–094) containing 20% glycerol (for storage at −80 °C). The NanoLuc assay was carried out in 96-well format by mixing 5 µL of each in vitro synthesized protein with 15 µL of 1× PBS per well. After incubation at 30 °C for 90 min, 50 µL of 100× diluted NanoLuc substrate (Promega, #N1110) were added, and luciferase enzymatic activity was measured using a TriStar or CentroXS luminometer (Berthold; 5 s integration time).Concentrated insect cell buffer (ICB 5×) consists of 150 mM HEPES-KOH pH: 7.6; 750 mM K-acetate; 19.5 mM Mg-acetate; 12.5 mM DTT; 500 µM spermidine; 100 mM creatine phosphate; 0.5 mg/mL creatinine kinase; 8.75 mM ATP; 1.5 mM GTP, CTP, UTP and 500 µM amino acids mix. The stock solution of amino acids (3.4 mM) was made by dissolving the 20 powders into 5 M KOH, at concentrations ranging from 1.6 to 4 M. The resulting mixtures containing each of the 20 amino acids were diluted in water to achieve a final concentration of 238 mM. The final aqueous solution was adjusted to pH 7.5 with acetic acid, diluted to 3.4 mM and frozen in liquid nitrogen for storage at −80 °C.

In vitro version of N2H assay (vN2H)

Synthesis of proteins was performed separately using the in vitro coupled transcription–translation system based on Insect Cell Extract (ICE) (TnT-T7-ICE, Promega, #L1102). In this ICE coupled transcription–translation system, transcription is driven by T7 RNA-Polymerase present in the ICE reagent. The necessary T7 promoter is included in the N2H synthetic promoter as described in Fig. 3b. Each reaction was performed in 25 µL final volume with 1 µg of pDEST-N2H plasmid (5 µL at 200 ng/µL) and 20 µL of diluted insect cell extract (containing 4 µL of ICE, 3.2 µL of 5× ICB dilution buffer and eight units of T7-RNAP-Promega#P4074, supplemented with H2O up to 20 µL). Incubation was at 30 °C for 4 h. The synthesis reaction was stopped by addition of 50 µL of 1× PBS (Gibco, #14190–094) containing 20% glycerol (for storage at −80 °C). The NanoLuc assay was carried out in 96-well format by mixing 5 µL of each in vitro synthesized protein with 15 µL of 1× PBS per well. After incubation at 30 °C for 90 min, 50 µL of 100× diluted NanoLuc substrate (Promega, #N1110) were added, and luciferase enzymatic activity was measured using a TriStar or CentroXS luminometer (Berthold; 5 s integration time).

Concentrated insect cell buffer (ICB 5×) consists of 150 mM HEPES-KOH pH: 7.6; 750 mM K-acetate; 19.5 mM Mg-acetate; 12.5 mM DTT; 500 µM spermidine; 100 mM creatine phosphate; 0.5 mg/mL creatinine kinase; 8.75 mM ATP; 1.5 mM GTP, CTP, UTP and 500 µM amino acids mix. The stock solution of amino acids (3.4 mM) was made by dissolving the 20 powders into 5 M KOH, at concentrations ranging from 1.6 to 4 M. The resulting mixtures containing each of the 20 amino acids were diluted in water to achieve a final concentration of 238 mM. The final aqueous solution was adjusted to pH 7.5 with acetic acid, diluted to 3.4 mM and frozen in liquid nitrogen for storage at −80 °C.

Yeast 2-Hybrid (Y2H) assayUsing a high-efficiency LiAc/SS carrier DNA/PEG protocol54, haploid S. cerevisiae Y8800 (MATa, leu2-3,112 trp1-901, his3Δ200, ade2-101 ura3-52, gal4Δ, gal80Δ, chy2R, GAL2::ADE2, GAL7::LacZ@met2, GAL1::HIS3@LYS2) and Y8930 (MATα, leu2-3,112 trp1-901, his3Δ200, ade2-101 ura3-52, gal4Δ, gal80Δ, chy2R, GAL2::ADE2, GAL7::LacZ@met2, GAL1::HIS3@LYS2) yeast strains were transformed with pDEST-AD and pDEST-DB plasmids16 containing a TRP1 or a LEU2 cassette, respectively. For auto-activation tests (DB-X/Y vs. AD-empty), Y8800 (MATa) yeast cells were also transformed with a pDEST-AD-empty plasmid. Six standard Y2H controls were also transformed into yeast cells: (1) pDEST-AD-empty/pDEST-DB-empty, (2) pDEST-AD-CBLB/pDEST-DB-GRB2, (3) pDEST-AD-XIAP/pDEST-DB-CASP9, (4) pDEST-AD-empty/pDEST-DB-scGal4, (5) pDEST-AD-empty/pDEST-DB-WDR62, and (6) pDEST-AD-scUme6 (a.a. 531-a.a. 836)/pDEST-DB. For each individual transformation, 0.3–1.5 µg of high-purity plasmid and 1 mL of a log phase-growth yeast cell culture (OD600 nm ~0.6–0.9) were used. The resulting transformants carrying pDEST-AD vectors (TRP1 cassette) or pDEST-DB vectors (LEU2 cassette) were spotted (8 µL per spot) onto solid synthetic complete media lacking tryptophan (SC-W) or leucine (SC-L), respectively. After 3 days of incubation at 30 °C, each transformant growing on solid SC-W or SC-L was picked by scratching the corresponding spot, and inoculated into a 96-well costar plate (Fisher Scientific/Corning, #07200720A) containing 160 µL of liquid SC-W or SC-L per well, respectively. Inoculated yeast cells were incubated overnight (15–18 h), at 30 °C and under shaking conditions (220 rpm). Haploid yeast were mated by inoculating 5 µL of transformed MATa Y8800 and 5 µL of transformed MATα Y8930 into a 96-well microplate containing 160 µL of YEPD medium (yeast extract, peptone, dextrose) per well. The exact same mating was done for the auto-activation tests (DB-X/Y-containing cells mated with AD-empty-containing cells). Mating was obtained by growing cells overnight (15–18 h), at 30 °C and under shaking conditions (220 rpm). The diploid selection was conducted by inoculating 10 µL of mated yeast cells into a 96-well microplate containing 160 µL of SC-LW medium per well. Cells were incubated overnight (15–18 h), at 30 °C and under shaking conditions (220 rpm). Using a liquid handling robot, each diploid cell culture was then spotted in quadruplicate (1 μL/spot) onto two different solid media plates: (1) SC-LW, and (2) SC-LW lacking histidine and containing 1 mM of 3-Amino-1,2,4-triazole (3-AT) (SC-LWH, 1 mM 3-AT). The six controls described above were also spotted on each solid plate to validate composition of selective media (based on their known phenotypes on these media) and compare their growth rates to the ones of tested protein pairs. Plates were incubated for 4 days at 30 °C and 2 days at room temperature before evaluating growth scores.

Yeast 2-Hybrid (Y2H) assay

Using a high-efficiency LiAc/SS carrier DNA/PEG protocol54, haploid S. cerevisiae Y8800 (MATa, leu2-3,112 trp1-901, his3Δ200, ade2-101 ura3-52, gal4Δ, gal80Δ, chy2R, GAL2::ADE2, GAL7::LacZ@met2, GAL1::HIS3@LYS2) and Y8930 (MATα, leu2-3,112 trp1-901, his3Δ200, ade2-101 ura3-52, gal4Δ, gal80Δ, chy2R, GAL2::ADE2, GAL7::LacZ@met2, GAL1::HIS3@LYS2) yeast strains were transformed with pDEST-AD and pDEST-DB plasmids16 containing a TRP1 or a LEU2 cassette, respectively. For auto-activation tests (DB-X/Y vs. AD-empty), Y8800 (MATa) yeast cells were also transformed with a pDEST-AD-empty plasmid. Six standard Y2H controls were also transformed into yeast cells: (1) pDEST-AD-empty/pDEST-DB-empty, (2) pDEST-AD-CBLB/pDEST-DB-GRB2, (3) pDEST-AD-XIAP/pDEST-DB-CASP9, (4) pDEST-AD-empty/pDEST-DB-scGal4, (5) pDEST-AD-empty/pDEST-DB-WDR62, and (6) pDEST-AD-scUme6 (a.a. 531-a.a. 836)/pDEST-DB. For each individual transformation, 0.3–1.5 µg of high-purity plasmid and 1 mL of a log phase-growth yeast cell culture (OD600 nm ~0.6–0.9) were used. The resulting transformants carrying pDEST-AD vectors (TRP1 cassette) or pDEST-DB vectors (LEU2 cassette) were spotted (8 µL per spot) onto solid synthetic complete media lacking tryptophan (SC-W) or leucine (SC-L), respectively. After 3 days of incubation at 30 °C, each transformant growing on solid SC-W or SC-L was picked by scratching the corresponding spot, and inoculated into a 96-well costar plate (Fisher Scientific/Corning, #07200720A) containing 160 µL of liquid SC-W or SC-L per well, respectively. Inoculated yeast cells were incubated overnight (15–18 h), at 30 °C and under shaking conditions (220 rpm). Haploid yeast were mated by inoculating 5 µL of transformed MATa Y8800 and 5 µL of transformed MATα Y8930 into a 96-well microplate containing 160 µL of YEPD medium (yeast extract, peptone, dextrose) per well. The exact same mating was done for the auto-activation tests (DB-X/Y-containing cells mated with AD-empty-containing cells). Mating was obtained by growing cells overnight (15–18 h), at 30 °C and under shaking conditions (220 rpm). The diploid selection was conducted by inoculating 10 µL of mated yeast cells into a 96-well microplate containing 160 µL of SC-LW medium per well. Cells were incubated overnight (15–18 h), at 30 °C and under shaking conditions (220 rpm). Using a liquid handling robot, each diploid cell culture was then spotted in quadruplicate (1 μL/spot) onto two different solid media plates: (1) SC-LW, and (2) SC-LW lacking histidine and containing 1 mM of 3-Amino-1,2,4-triazole (3-AT) (SC-LWH, 1 mM 3-AT). The six controls described above were also spotted on each solid plate to validate composition of selective media (based on their known phenotypes on these media) and compare their growth rates to the ones of tested protein pairs. Plates were incubated for 4 days at 30 °C and 2 days at room temperature before evaluating growth scores.

Gaussia princeps complementation assay (GPCA)HEK293T cells were cultured as described for the mN2H assay version. For each protein, 100 ng of purified plasmid DNA was transfected into HEK293T cells in 96-well, flat-bottom, cell culture plates (Greiner Bio-One, #655083) supplemented with 10%FBS in DMEM using polyethylenimine24. The DNA/PEI ratio (mass:mass) was 1:3. GPCA vectors carry the human cytomegalovirus (CMV) promoter and are maintained as high copy number with the human virus SV40 replication origin in mammalian cells. 24 h after DNA transfection, the cell culture medium was removed, cells were gently washed with 150 µL pre-warmed 1x PBS, 40 µL of lysis buffer was then added per well, and cell lysis was performed under vigorous shaking of the plate for 20 min at 900 rpm. Luminescence was measured by auto-injecting 50 µL Renilla luciferase substrate (Renilla Luciferase Assay system, catalog No. E2820, Promega) per well and integrating light output for 4 s using a TriStar luminometer (Berthold).

Gaussia princeps complementation assay (GPCA)

HEK293T cells were cultured as described for the mN2H assay version. For each protein, 100 ng of purified plasmid DNA was transfected into HEK293T cells in 96-well, flat-bottom, cell culture plates (Greiner Bio-One, #655083) supplemented with 10%FBS in DMEM using polyethylenimine24. The DNA/PEI ratio (mass:mass) was 1:3. GPCA vectors carry the human cytomegalovirus (CMV) promoter and are maintained as high copy number with the human virus SV40 replication origin in mammalian cells. 24 h after DNA transfection, the cell culture medium was removed, cells were gently washed with 150 µL pre-warmed 1x PBS, 40 µL of lysis buffer was then added per well, and cell lysis was performed under vigorous shaking of the plate for 20 min at 900 rpm. Luminescence was measured by auto-injecting 50 µL Renilla luciferase substrate (Renilla Luciferase Assay system, catalog No. E2820, Promega) per well and integrating light output for 4 s using a TriStar luminometer (Berthold).

Construction of C-terminal plasmids for GPCAGPCA C1 and C2 vectors are based on two fragments of the humanized Gaussia princeps luciferase (herein referred as GLuc), similar to the GPCA N1 and N2 vectors24. Both GLuc fragments were linked to the C-terminal extremity of the tested proteins by a flexible hinge polypeptide of 20 amino acids, including the Gateway recombinational cloning sites. To normalize mRNA translation initiation, a consensus Kozak translation start sequence was added. Both constructs were carried by the same CMV driven mammalian expression vector (pCI-neo derived, Promega) and were maintained as multi-copies within the cells via the presence of the SV40 replication origin.

Construction of C-terminal plasmids for GPCA

GPCA C1 and C2 vectors are based on two fragments of the humanized Gaussia princeps luciferase (herein referred as GLuc), similar to the GPCA N1 and N2 vectors24. Both GLuc fragments were linked to the C-terminal extremity of the tested proteins by a flexible hinge polypeptide of 20 amino acids, including the Gateway recombinational cloning sites. To normalize mRNA translation initiation, a consensus Kozak translation start sequence was added. Both constructs were carried by the same CMV driven mammalian expression vector (pCI-neo derived, Promega) and were maintained as multi-copies within the cells via the presence of the SV40 replication origin.

NanoLuc® Binary Technology (NanoBiT) assayHEK293T cells were cultured as described for the mN2H assay version. For each protein, 100 ng of purified plasmid DNA was transfected into HEK293T cells grown in 96-well, flat-bottom, cell culture microplates (Greiner Bio-One, #655083). The DNA/PEI ratio (mass:mass) was 1:3 for the transfection. 24 h later, 100 µL of the NanoLuc substrate (Promega, #N1110) were added to each well, plates were incubated for 3 min at room temperature, and luminescence output was measured using a TriStar luminometer (Berthold; 1 s integration time).

NanoLuc® Binary Technology (NanoBiT) assay

HEK293T cells were cultured as described for the mN2H assay version. For each protein, 100 ng of purified plasmid DNA was transfected into HEK293T cells grown in 96-well, flat-bottom, cell culture microplates (Greiner Bio-One, #655083). The DNA/PEI ratio (mass:mass) was 1:3 for the transfection. 24 h later, 100 µL of the NanoLuc substrate (Promega, #N1110) were added to each well, plates were incubated for 3 min at room temperature, and luminescence output was measured using a TriStar luminometer (Berthold; 1 s integration time).

Construction of Gateway-compatible plasmids for NanoBiTThe Gateway reading frame cassette was amplified by PCR, and polished with a PCR polishing kit (Agilent, # 200409) to generate blunt-end DNA. NanoBiT vectors (Promega, # N2014) with N- and C-terminus tagging configurations (NB MCS1, NB MCS2, NB MCS3, NB MCS4) were obtained from Promega, each containing an HSV-TK promoter for protein expression in mammalian cells and an ampicillin resistance marker for bacterial selection of transformed cells. Each plasmid DNA for NanoBiT vectors (NB MCS1-4) was digested with a single-cut restriction enzyme (EcoRI for NB MCS1 and 2, and SacI for NB MCS3 and 4) at the multiple cloning sites (MCS). The linearized plasmid DNA was processed with the PCR polishing kit generating blunt-end DNA. Antarctic phosphatase (NEB, M0289S) was used to remove 5′-phosphates to reduce self-ligation background of the linearized vector. The ligation reaction was performed with T4 DNA ligase (NEB, M0202S) joining the linearized NanoBiT vectors (NB MCS1-4) with the polished Gateway reading frame cassette. The ligated, Gateway-compatible NanoBiT vectors (NanoBiT-GW-N1, -N2, -C1 and –C2) were introduced into One ShotTM ccdB survivalTM 2 T1R competent E. coli cells (Invitrogen, #A10460) via heat shock bacterial transformation, and selected on chloramphenicol-containing LB agar plates. The sequences of constructed NanoBiT-GW vectors were examined by Sanger DNA sequencing. The final vectors contained correct sequences of the promoter, NanoBiT fragments, junctions, and Gateway cassette.

Construction of Gateway-compatible plasmids for NanoBiT

The Gateway reading frame cassette was amplified by PCR, and polished with a PCR polishing kit (Agilent, # 200409) to generate blunt-end DNA. NanoBiT vectors (Promega, # N2014) with N- and C-terminus tagging configurations (NB MCS1, NB MCS2, NB MCS3, NB MCS4) were obtained from Promega, each containing an HSV-TK promoter for protein expression in mammalian cells and an ampicillin resistance marker for bacterial selection of transformed cells. Each plasmid DNA for NanoBiT vectors (NB MCS1-4) was digested with a single-cut restriction enzyme (EcoRI for NB MCS1 and 2, and SacI for NB MCS3 and 4) at the multiple cloning sites (MCS). The linearized plasmid DNA was processed with the PCR polishing kit generating blunt-end DNA. Antarctic phosphatase (NEB, M0289S) was used to remove 5′-phosphates to reduce self-ligation background of the linearized vector. The ligation reaction was performed with T4 DNA ligase (NEB, M0202S) joining the linearized NanoBiT vectors (NB MCS1-4) with the polished Gateway reading frame cassette. The ligated, Gateway-compatible NanoBiT vectors (NanoBiT-GW-N1, -N2, -C1 and –C2) were introduced into One ShotTM ccdB survivalTM 2 T1R competent E. coli cells (Invitrogen, #A10460) via heat shock bacterial transformation, and selected on chloramphenicol-containing LB agar plates. The sequences of constructed NanoBiT-GW vectors were examined by Sanger DNA sequencing. The final vectors contained correct sequences of the promoter, NanoBiT fragments, junctions, and Gateway cassette.

MAPPIT and KISS assaysHEK293T cells were cultured as described for the mN2H assay version. In both Mammalian protein–protein interaction trap (MAPPIT) and Kinase substrate sensor (KISS assays)12,25, HEK293T cells were transfected with bait, prey and reporter plasmids applying a standard calcium phosphate transfection method. Luciferase activity was measured 48 h after transfection using the Luciferase Assay System kit (Promega) on a Enspire luminometer (Perkin-Elmer). For MAPPIT, half of the cells were stimulated with erythropoietin (2 ng/mL; GenScript) 24 h after transfection. For each tested pair in MAPPIT, the fold-induction value (signal from two stimulated culture wells divided by the signal from two unstimulated culture wells) was calculated. For KISS the average of four culture wells was used.

MAPPIT and KISS assays

HEK293T cells were cultured as described for the mN2H assay version. In both Mammalian protein–protein interaction trap (MAPPIT) and Kinase substrate sensor (KISS assays)12,25, HEK293T cells were transfected with bait, prey and reporter plasmids applying a standard calcium phosphate transfection method. Luciferase activity was measured 48 h after transfection using the Luciferase Assay System kit (Promega) on a Enspire luminometer (Perkin-Elmer). For MAPPIT, half of the cells were stimulated with erythropoietin (2 ng/mL; GenScript) 24 h after transfection. For each tested pair in MAPPIT, the fold-induction value (signal from two stimulated culture wells divided by the signal from two unstimulated culture wells) was calculated. For KISS the average of four culture wells was used.

Scoring hsPRS-v2 pairsData for each assay version performed in this study are presented in Source Data. All binary PPI experiments were independently performed three times. For the different versions of N2H experiments, the average signal of the three replicates was used to determine a raw luminescence value for each protein pair X/Y. Control experiments were performed similarly where protein X or Y fused to the NanoLuc fragment F1 or F2 was co-expressed with the matching NanoLuc fragment alone (X/empty or empty/Y). For each protein pair X–Y, we calculated a normalized luminescence ratio (NLR) corresponding to the raw luminescence value of the tested pair (X–Y) divided by the maximum luminescence value from one of the two controls (X-empty or empty-Y), as summarized in Eq. (1) 25:1\documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$${\mathrm{NLR}}_{{\mathrm{X}} - {\mathrm{Y}}} = \frac{{{\mathrm{Luminescence}}_{{\mathrm{X}} - {\mathrm{Y}}}}}{{{\mathrm{Maximum}}\,{\mathrm{Luminescence}}_{({\mathrm{X}} - {\mathrm{empty}}){\mathrm{or}}({\mathrm{empty}} - {\mathrm{Y}})}}}.$$\end{document}NLRX-Y=LuminescenceX-YMaximumLuminescence(X-empty)or(empty-Y).To identify protein pairs that score positive at a threshold of no RRS detection, two criteria were considered: (1) NLRPRS > NLRHIGHEST RRS and (2) NLRPRS > 1.0. All PRS pairs that did not meet these two criteria were not scored and therefore defined as not detectable in the corresponding assay version. The same analysis and criteria were applied for both KISS and MAPPIT assays.For GPCA and NanoBiT experiments, a PRS pair was scored positive if its corresponding raw luminescence signal (from the average of three independent experiments) was strictly higher than the RRS pair having the highest raw luminescence signal (from the average of three independent experiments).For Y2H assay, diploid yeast cells containing both, DB-X/Y bait and AD-Y/X prey plasmids, were spotted in quadruplicate on selective medium supplemented with 3-AT (SC-LWH, 3-AT). After titration of 3-AT, a minimal concentration of 1 mM was selected as no RRS pairs were detected for this specific threshold12. In parallel, diploid yeast cells containing DB-X/Y bait and AD-empty prey vectors were also used to identify bait proteins that are auto-activators of the HIS3 selection marker. After 6 days of culture, yeast growth was visually determined and scored from “0” (no growth) to “4” (strong growth) for both the DB-X/Y:AD-Y/X and DB-X/Y:AD-empty combinations. If the growth score was >0 for the DB-X/Y:AD-Y/X combination in at least two of the three experiments performed and was superior by at least 2 units to the one of DB-X/Y:AD-empty combination, a protein pair was scored positive.

Scoring hsPRS-v2 pairs

Data for each assay version performed in this study are presented in Source Data. All binary PPI experiments were independently performed three times. For the different versions of N2H experiments, the average signal of the three replicates was used to determine a raw luminescence value for each protein pair X/Y. Control experiments were performed similarly where protein X or Y fused to the NanoLuc fragment F1 or F2 was co-expressed with the matching NanoLuc fragment alone (X/empty or empty/Y). For each protein pair X–Y, we calculated a normalized luminescence ratio (NLR) corresponding to the raw luminescence value of the tested pair (X–Y) divided by the maximum luminescence value from one of the two controls (X-empty or empty-Y), as summarized in Eq. (1) 25:1\documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$${\mathrm{NLR}}_{{\mathrm{X}} - {\mathrm{Y}}} = \frac{{{\mathrm{Luminescence}}_{{\mathrm{X}} - {\mathrm{Y}}}}}{{{\mathrm{Maximum}}\,{\mathrm{Luminescence}}_{({\mathrm{X}} - {\mathrm{empty}}){\mathrm{or}}({\mathrm{empty}} - {\mathrm{Y}})}}}.$$\end{document}NLRX-Y=LuminescenceX-YMaximumLuminescence(X-empty)or(empty-Y).

To identify protein pairs that score positive at a threshold of no RRS detection, two criteria were considered: (1) NLRPRS > NLRHIGHEST RRS and (2) NLRPRS > 1.0. All PRS pairs that did not meet these two criteria were not scored and therefore defined as not detectable in the corresponding assay version. The same analysis and criteria were applied for both KISS and MAPPIT assays.

For GPCA and NanoBiT experiments, a PRS pair was scored positive if its corresponding raw luminescence signal (from the average of three independent experiments) was strictly higher than the RRS pair having the highest raw luminescence signal (from the average of three independent experiments).

For Y2H assay, diploid yeast cells containing both, DB-X/Y bait and AD-Y/X prey plasmids, were spotted in quadruplicate on selective medium supplemented with 3-AT (SC-LWH, 3-AT). After titration of 3-AT, a minimal concentration of 1 mM was selected as no RRS pairs were detected for this specific threshold12. In parallel, diploid yeast cells containing DB-X/Y bait and AD-empty prey vectors were also used to identify bait proteins that are auto-activators of the HIS3 selection marker. After 6 days of culture, yeast growth was visually determined and scored from “0” (no growth) to “4” (strong growth) for both the DB-X/Y:AD-Y/X and DB-X/Y:AD-empty combinations. If the growth score was >0 for the DB-X/Y:AD-Y/X combination in at least two of the three experiments performed and was superior by at least 2 units to the one of DB-X/Y:AD-empty combination, a protein pair was scored positive.

Construction of cumulative recovery plotsTo plot the fraction of interaction recovered for n number of assays or assay versions, the recovery rate for each of the possible n combinations was calculated first (Supplementary Figs. 1c, 2d, 5a) and then averaged (Figs. 2b, d, 3f and Supplementary Fig. 5b). The error bars in Figs. 2b, d, 3f and Supplementary Fig. 5b indicate the standard deviation of the average.

Construction of cumulative recovery plots

To plot the fraction of interaction recovered for n number of assays or assay versions, the recovery rate for each of the possible n combinations was calculated first (Supplementary Figs. 1c, 2d, 5a) and then averaged (Figs. 2b, d, 3f and Supplementary Fig. 5b). The error bars in Figs. 2b, d, 3f and Supplementary Fig. 5b indicate the standard deviation of the average.

Reporting summaryFurther information on research design is available in the Nature Research Reporting Summary linked to this article.

Reporting summary

Further information on research design is available in the Nature Research Reporting Summary linked to this article.

Supplementary information
Description of Additional Supplementary Files
Supplementary Information
Supplementary Data 1
Supplementary Data 2
Reporting Summary

Source Data

Supplementary information

Description of Additional Supplementary Files
Supplementary Information
Supplementary Data 1
Supplementary Data 2
Reporting Summary

Source Data

Description of Additional Supplementary Files
Supplementary Information
Supplementary Data 1
Supplementary Data 2
Reporting Summary

Description of Additional Supplementary Files

Supplementary Information

Supplementary Data 1

Supplementary Data 2

Reporting Summary

Source Data

Source Data
