---
pmid: '17267404'
title: ATF2 is required for amino acid-regulated transcription by orchestrating specific
  histone acetylation.
authors:
- Bruhat A
- Chérasse Y
- Maurin AC
- Breitwieser W
- Parry L
- Deval C
- Jones N
- Jousse C
- Fafournoux P
journal: Nucleic Acids Res
year: '2007'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC1851658
doi: 10.1093/nar/gkm038
---

# ATF2 is required for amino acid-regulated transcription by orchestrating specific histone acetylation.
**Authors:** Bruhat A, Chérasse Y, Maurin AC, Breitwieser W, Parry L, Deval C, Jones N, Jousse C, Fafournoux P
**Journal:** Nucleic Acids Res (2007)
**DOI:** [10.1093/nar/gkm038](https://doi.org/10.1093/nar/gkm038)
**PMC:** [PMC1851658](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1851658/)

## Abstract

1. Nucleic Acids Res. 2007;35(4):1312-21. doi: 10.1093/nar/gkm038. Epub 2007 Jan 
31.

ATF2 is required for amino acid-regulated transcription by orchestrating 
specific histone acetylation.

Bruhat A(1), Chérasse Y, Maurin AC, Breitwieser W, Parry L, Deval C, Jones N, 
Jousse C, Fafournoux P.

Author information:
(1)UMR 1019, Unité de Nutrition Humaine, INRA de Theix, 63122 Saint Genès 
Champanelle, France. bruhat@clermont.inra.fr

The transcriptional activation of CHOP (a CCAAT/enhancer-binding protein-related 
gene) by amino acid deprivation involves the activating transcription factor 2 
(ATF2) and the activating transcription factor 4 (ATF4) binding the amino acid 
response element (AARE) within the promoter. Using a chromatin 
immunoprecipitation approach, we report that in vivo binding of phospho-ATF2 and 
ATF4 to CHOP AARE are associated with acetylation of histones H4 and H2B in 
response to amino acid starvation. A time course analysis reveals that ATF2 
phosphorylation precedes histone acetylation, ATF4 binding and the increase in 
CHOP mRNA. We also show that ATF4 binding and histone acetylation are two 
independent events that are required for the CHOP induction upon amino acid 
starvation. Using ATF2-deficient mouse embryonic fibroblasts, we demonstrate 
that ATF2 is essential in the acetylation of histone H4 and H2B in vivo. The 
role of ATF2 on histone H4 acetylation is dependent on its binding to the AARE 
and can be extended to other amino acid regulated genes. Thus, ATF2 is involved 
in promoting the modification of the chromatin structure to enhance the 
transcription of a number of amino acid-regulated genes.

DOI: 10.1093/nar/gkm038
PMCID: PMC1851658
PMID: 17267404 [Indexed for MEDLINE]

## Full Text

INTRODUCTIONAll mammalian cells regulate gene expression in response to changes in the external environment such as nutrient availability. An amino acid response pathway designed to detect and respond to amino acid deficiency has been described. Limiting any essential amino acid initiates this signalling cascade, which leads to increased translation of a ‘master regulator’, activating transcription factor 4 (ATF4) and ultimately, to regulation of a number of steps during protein expression (1,2). Although the understanding of how amino acids control gene expression in mammalian cells remains relatively limited, significant progress has been achieved during the past few years, in particular with regard to amino acid control of gene transcription. At the molecular level, most of the results have been obtained studying the transcriptional regulation of Asparagine synthetase (ASNS) and C/EBP homologous protein (CHOP) gene expression in response to amino acid deprivation (3,4).The CHOP gene encodes a ubiquitous transcription factor that heterodimerizes avidly with the other members of the C/EBP and jun/fos families (5). CHOP expression is tightly regulated by a wide variety of stresses and agents (6–8). The regulation of CHOP mRNA expression in response to amino acid deprivation is mainly regulated at the transcriptional level (9). Our earlier studies have identified an Amino Acid Response Element (AARE) located between −313 and −295 (10). This element is essential for amino acid regulation of CHOP transcription and can confer amino acid responsiveness to a heterologous promoter. The sequence of the CHOP AARE (5′-ATTGCATCA-3′) is related to C/EBP and ATF/CRE binding sites. Thereafter, functional sequences that share a high nucleotide sequence similarity with CHOP AARE have been identified in other amino acid-regulated genes such as ASNS (11), activating transcription factor 3 (ATF3) (12), system A amino acid transporter (SNAT2) (13) and arginine/lysine transporter cat-1 (cat-1) (14). The CHOP AARE was described to bind in vitro activating transcription factor 2 (ATF2) and ATF4 and the expression of these factors was shown to be essential for the transcriptional activation of CHOP by leucine starvation (4,10). However, while the role of ATF4 in the control of amino acid-regulated genes has been well established, the precise functions of ATF2 remain to be elucidated.ATF2 belongs to the basic region leucine zipper (bZIP) family of transcription factors and is an important member of activating protein 1 (AP-1) (15). ATF2 functions either as a homodimer or as a heterodimer with other members of the ATF family, as well as other bZIP proteins, to bind to specific DNA sequences and activate gene expression. One major role of ATF2 is to regulate the response of cells to stress signals (16–19). The transactivation capacity of the N-terminal domain of this transcription factor can be enhanced through phosphorylation of two N-terminal threonine residues Thr-69 and Thr-71 (16,20,21). Although the phosphorylation events are critical for the full transcriptional activity of ATF2, the underlying mechanism of this activation is poorly defined.In the context of gene regulation by amino acid starvation, the role of ATF2 has been principally studied using CHOP as a working model (4,10). It was shown that (i) in cells devoid of ATF2 expression, the induction of CHOP transcription upon amino acid starvation is completely lost, (ii) ATF2 binds in vitro to the CHOP AARE in starved and unstarved conditions and (iii) ATF2 phosphorylation is necessary for the activation of the CHOP AARE-dependent transcription. We have also documented that ATF2 plays a key role in the expression of a number of amino acid-regulated genes which are totally dependent on this transcription factor. It was further hypothesized that phosphorylation of ATF2 could activate gene transcription either by interactions with transcriptional machinery or by effects on a chromatin component.The present study was designed to identify the mechanisms by which ATF2 phosphorylation can lead to the onset of CHOP transcription in response to amino acid starvation. Using a chromatin immunoprecipitation (ChIP) approach, we show that binding of ATF2 and ATF4 to CHOP AARE are associated with acetylation of histones H4 and H2B in response to amino acid starvation. A time course analysis revealed that phosphorylation of ATF2 on Thr71 precedes histone acetylation, ATF4 binding and the increase in CHOP mRNA. We provide evidence that ATF2 bound to the AARE plays a crucial role in vivo in the acetylation of histone H4 and H2B in response to amino acid starvation and thereby may be involved in the modification of the chromatin structure to enhance transcription of a number of amino acid-regulated genes.

INTRODUCTION

All mammalian cells regulate gene expression in response to changes in the external environment such as nutrient availability. An amino acid response pathway designed to detect and respond to amino acid deficiency has been described. Limiting any essential amino acid initiates this signalling cascade, which leads to increased translation of a ‘master regulator’, activating transcription factor 4 (ATF4) and ultimately, to regulation of a number of steps during protein expression (1,2). Although the understanding of how amino acids control gene expression in mammalian cells remains relatively limited, significant progress has been achieved during the past few years, in particular with regard to amino acid control of gene transcription. At the molecular level, most of the results have been obtained studying the transcriptional regulation of Asparagine synthetase (ASNS) and C/EBP homologous protein (CHOP) gene expression in response to amino acid deprivation (3,4).

The CHOP gene encodes a ubiquitous transcription factor that heterodimerizes avidly with the other members of the C/EBP and jun/fos families (5). CHOP expression is tightly regulated by a wide variety of stresses and agents (6–8). The regulation of CHOP mRNA expression in response to amino acid deprivation is mainly regulated at the transcriptional level (9). Our earlier studies have identified an Amino Acid Response Element (AARE) located between −313 and −295 (10). This element is essential for amino acid regulation of CHOP transcription and can confer amino acid responsiveness to a heterologous promoter. The sequence of the CHOP AARE (5′-ATTGCATCA-3′) is related to C/EBP and ATF/CRE binding sites. Thereafter, functional sequences that share a high nucleotide sequence similarity with CHOP AARE have been identified in other amino acid-regulated genes such as ASNS (11), activating transcription factor 3 (ATF3) (12), system A amino acid transporter (SNAT2) (13) and arginine/lysine transporter cat-1 (cat-1) (14). The CHOP AARE was described to bind in vitro activating transcription factor 2 (ATF2) and ATF4 and the expression of these factors was shown to be essential for the transcriptional activation of CHOP by leucine starvation (4,10). However, while the role of ATF4 in the control of amino acid-regulated genes has been well established, the precise functions of ATF2 remain to be elucidated.

ATF2 belongs to the basic region leucine zipper (bZIP) family of transcription factors and is an important member of activating protein 1 (AP-1) (15). ATF2 functions either as a homodimer or as a heterodimer with other members of the ATF family, as well as other bZIP proteins, to bind to specific DNA sequences and activate gene expression. One major role of ATF2 is to regulate the response of cells to stress signals (16–19). The transactivation capacity of the N-terminal domain of this transcription factor can be enhanced through phosphorylation of two N-terminal threonine residues Thr-69 and Thr-71 (16,20,21). Although the phosphorylation events are critical for the full transcriptional activity of ATF2, the underlying mechanism of this activation is poorly defined.

In the context of gene regulation by amino acid starvation, the role of ATF2 has been principally studied using CHOP as a working model (4,10). It was shown that (i) in cells devoid of ATF2 expression, the induction of CHOP transcription upon amino acid starvation is completely lost, (ii) ATF2 binds in vitro to the CHOP AARE in starved and unstarved conditions and (iii) ATF2 phosphorylation is necessary for the activation of the CHOP AARE-dependent transcription. We have also documented that ATF2 plays a key role in the expression of a number of amino acid-regulated genes which are totally dependent on this transcription factor. It was further hypothesized that phosphorylation of ATF2 could activate gene transcription either by interactions with transcriptional machinery or by effects on a chromatin component.

The present study was designed to identify the mechanisms by which ATF2 phosphorylation can lead to the onset of CHOP transcription in response to amino acid starvation. Using a chromatin immunoprecipitation (ChIP) approach, we show that binding of ATF2 and ATF4 to CHOP AARE are associated with acetylation of histones H4 and H2B in response to amino acid starvation. A time course analysis revealed that phosphorylation of ATF2 on Thr71 precedes histone acetylation, ATF4 binding and the increase in CHOP mRNA. We provide evidence that ATF2 bound to the AARE plays a crucial role in vivo in the acetylation of histone H4 and H2B in response to amino acid starvation and thereby may be involved in the modification of the chromatin structure to enhance transcription of a number of amino acid-regulated genes.

MATERIALS AND METHODSCell culture and treatment conditionsCells were cultured at 37°C in Dulbecco's modified Eagle's medium F12 (DMEM F12) (Sigma) containing 10% fetal bovine serum. When indicated, DMEM F12 lacking leucine was used. In all experiments involving amino acid starvation, 10% dialyzed calf serum was used. Mouse embryonic fibroblasts (MEFs) deficient in ATF4 were kindly given by Dr D. Ron (Skirball Institute of Biomolecular Medicine, New York) (22). ATF2 mutant mice were generated by insertion of loxP sites into genomic sequences flanking exons 8 and 9 of the ATF2 gene (encoding the whole DNA binding and most of the leucine zipper domain) and induction of recombination by transiently expressing Cre recombinase in ES cells (W. Breitwieser, unpublished data). Primary MEF were generated from E13.5 embryos derived from crossing heterozygous animals carrying a germline allele of the mutant ATF2 gene.Analysis of gene expression using real-time RT-PCRTotal RNA was prepared using a RNeasy mini kit (Qiagen) and treated with DNase I, Amp Grade (InVitrogen) prior to cDNA synthesis. RNA integrity was electrophoretically verified by ethidium bromide staining. RNA (0.5 μg) was reverse transcribed with 100 U of Superscript II plus RNase H− Reverse Transcriptase (InVitrogen) using 100 μM random hexamer primers (Amersham Biosciences), according to the manufacturer's instructions. Primers for human sequences (used in HeLa cells): hCHOP (forward primer, 5′-cagaaccagcagaggtcaca-3′; reverse primer, 5′-agctgtgccactttcctttc-3′). Primers for mouse sequences (used in ATF4 +/+, ATF4 −/−, ATF2 +/+ and ATF2 −/− cells): mCHOP (forward primer, 5′-cctagcttggctgacagagg-3′; reverse primer, 5′-ctgctccttctccttcatgc-3′), mASNS (forward primer, 5′-tacaaccacaaggcgctaca-3′; reverse primer, 5′-aagggcctgactccataggt-3′) and mATF3 (forward primer, 5′-cgccatccagaataaacacc-3′; reverse primer, 5′-gcagccactctgtcttctcc-3′) were used and yielded PCR products of approximatively 200 bp in size. To control for RNA quality and cDNA synthesis, β-actin mRNA was also amplified with forward (5′-tacagcttcaccaccacagc-3′) and reverse primers (5′-aaggaaggctggaaaagagc-3′). Quantification involved the use of standard curves that had been prepared with plasmids containing specific sequences of each gene. We cloned the PCR products of CHOP, ASNS, ATF3 and β-actin into the pGEM-T easy vector (Promega) according to the manufacturer's instructions. For the construction of standard curves for CHOP, ASNS, ATF3 and β-actin, pGEM-T easy plasmids were prepared as 10-fold serial dilution in water, from 4 ng to 0.4 pg. PCR was carried out using a LightCycler™ System (Roche) as described previously (4). LightCycler quantification software (version 3.5) was used to compare amplification in experimental samples during the log-linear phase to the standard curve from the dilution series of control plasmids. Relative results were displayed in nanograms of CHOP, ASNS and ATF3 per 100 nanograms of β-actin. Each experiment was repeated three times to confirm the reproducibility of the results.Nuclear extract preparationNuclear extracts were prepared as described previously (10).Western blot analysisNuclear proteins were resolved by SDS-polyacrylamide gel electrophoresis and transferred onto a Hybond-P PVDF membrane (Amersham Biosciences). Membranes were blocked for 1 h at room temperature with a solution of 5% non-fat milk powder in TN (50 mM Tris-HCL, pH 8.0, 150 mM NaCl, 0.1% Tween-20). The blots were then incubated with antibody in blocking solution overnight at 4°C. Antibodies were diluted according to the manufacturer's instructions. The blots were washed three times in TN and incubated with horseradish peroxidase-conjugated goat anti-rabbit IgG (1:20 000) (Santa Cruz, CA) in blocking buffer for 1 h at room temperature. After three washes, the blots were developed using the enhanced chemiluminescence (ECL) detection system (Amersham Biosciences).AntibodiesThe following antibodies were purchased from Santa Cruz Biotechnology, Inc (Santa Cruz, CA): ATF2, sc-187; ATF4, sc-200. Anti-phospho-ATF2 (catalog no. 9221) was obtained from Cell Signaling Technology (Beverly, MA). Acetylated histone H3, 06-599 (recognizes acetylated H3 at Lys-9 and Lys-14) and acetylated histone H4, 06-866 (recognizes acetylated H4 at Lys-5, -8, -12 and -16) antibodies were purchased from Upstate Biotechnology (Charlottes-ville, VA). Acetylated histone H2B (recognizes acetylated H2B at Lys-12 and Lys-15) antibody was from Abcam (Cambridge, UK).Chromatin immunoprecipitation analysis (ChIP)ChIP analysis was performed according to the protocol of Upstate Biotechnology, Inc. (Charlottesville, VA) with minor modifications. Cells were seeded at 1 × 106/100-mm dish with DMEM F12 and grown for 24 h. Cells were transferred to fresh DMEM F12 12 h before transfer to either complete DMEM F12 or DMEM F12 lacking leucine for the time period indicated in each figure. Protein–DNA was cross-linked by adding formaldehyde directly to the culture medium to a final concentration of 1% and then stopped 8 min later by the addition of glycine to a final concentration of 0.125 M. Cross-linked chromatin was sonicated using a Vibra cell sonicator (Bioblock Scientific Technology) for 10 bursts of 30 s at power 2 with 1-min cooling on ice between each burst to obtain DNA fragments of an average of 400 bp. Extracts from 1 × 106 cells were incubated with 5 µg of antibody. A rabbit anti-chicken IgG was used as the non-specific antibody control. The antibody-bound complex was precipitated by protein A-Agarose beads (Upstate Biotechnology). The DNA fragments in the immunoprecipitated complex were released by reversing the cross-linking overnight at 65°C and purified using a phenol/chloroform extraction and ethanol precipitation. Real-time quantitative PCR was performed by using a LightCycler (Roche) and a SYBR-Green-I-containing PCR mix (Qiagen), following the recommendations of the manufacturer. The immunoprecipitated material was quantified relative to a standard curve of genomic DNA. Primers used for human sequences: hCHOP amplicon A, 5′-gcagcctaaccaaagacctg-3′ and 5′-ggaggcaacttgaccaaaag-3′; hCHOP amplicon B (AARE), 5′-aagaggctcacgaccgacta-3′ and 5′-atgatgcaatgtttggcaac-3′; hCHOP amplicon C, 5′-agtgccacggagaaagctaa-3′ and 5′-ccatacagcagcctgagtga-3′. Primers used for mouse sequences: mCHOP AARE, 5′-gggcagacaagttcaggaag-3′ and 5′-atgatgcaatgtttggcaac-3′; mASNS AARE, 5′-cagaacacctcctggctctc-3′ and 5′-ccgcttgccaccttagagtc-3′, mATF3 AARE, 5′- ggtctccacccaccttttg -3′ and 5′- ctcgctgagtgagactgtgg -3′. The reactions were incubated at 95°C for 15 min to activate the polymerase, followed by amplification at 95°C for 15 s, 55°C for 20 s and 72°C for 20 s for 45 cycles. After PCR, melting curves were acquired by stepwise increases in the temperature from 65 to 95°C to ensure that a single product was amplified in the reaction. The results are expressed as the percentage of antibody binding versus the amount of PCR product obtained using a standardized aliquot of input chromatin. Values are the means from at least three independent immunoprecipitations.

MATERIALS AND METHODS

Cell culture and treatment conditionsCells were cultured at 37°C in Dulbecco's modified Eagle's medium F12 (DMEM F12) (Sigma) containing 10% fetal bovine serum. When indicated, DMEM F12 lacking leucine was used. In all experiments involving amino acid starvation, 10% dialyzed calf serum was used. Mouse embryonic fibroblasts (MEFs) deficient in ATF4 were kindly given by Dr D. Ron (Skirball Institute of Biomolecular Medicine, New York) (22). ATF2 mutant mice were generated by insertion of loxP sites into genomic sequences flanking exons 8 and 9 of the ATF2 gene (encoding the whole DNA binding and most of the leucine zipper domain) and induction of recombination by transiently expressing Cre recombinase in ES cells (W. Breitwieser, unpublished data). Primary MEF were generated from E13.5 embryos derived from crossing heterozygous animals carrying a germline allele of the mutant ATF2 gene.

Cell culture and treatment conditions

Cells were cultured at 37°C in Dulbecco's modified Eagle's medium F12 (DMEM F12) (Sigma) containing 10% fetal bovine serum. When indicated, DMEM F12 lacking leucine was used. In all experiments involving amino acid starvation, 10% dialyzed calf serum was used. Mouse embryonic fibroblasts (MEFs) deficient in ATF4 were kindly given by Dr D. Ron (Skirball Institute of Biomolecular Medicine, New York) (22). ATF2 mutant mice were generated by insertion of loxP sites into genomic sequences flanking exons 8 and 9 of the ATF2 gene (encoding the whole DNA binding and most of the leucine zipper domain) and induction of recombination by transiently expressing Cre recombinase in ES cells (W. Breitwieser, unpublished data). Primary MEF were generated from E13.5 embryos derived from crossing heterozygous animals carrying a germline allele of the mutant ATF2 gene.

Analysis of gene expression using real-time RT-PCRTotal RNA was prepared using a RNeasy mini kit (Qiagen) and treated with DNase I, Amp Grade (InVitrogen) prior to cDNA synthesis. RNA integrity was electrophoretically verified by ethidium bromide staining. RNA (0.5 μg) was reverse transcribed with 100 U of Superscript II plus RNase H− Reverse Transcriptase (InVitrogen) using 100 μM random hexamer primers (Amersham Biosciences), according to the manufacturer's instructions. Primers for human sequences (used in HeLa cells): hCHOP (forward primer, 5′-cagaaccagcagaggtcaca-3′; reverse primer, 5′-agctgtgccactttcctttc-3′). Primers for mouse sequences (used in ATF4 +/+, ATF4 −/−, ATF2 +/+ and ATF2 −/− cells): mCHOP (forward primer, 5′-cctagcttggctgacagagg-3′; reverse primer, 5′-ctgctccttctccttcatgc-3′), mASNS (forward primer, 5′-tacaaccacaaggcgctaca-3′; reverse primer, 5′-aagggcctgactccataggt-3′) and mATF3 (forward primer, 5′-cgccatccagaataaacacc-3′; reverse primer, 5′-gcagccactctgtcttctcc-3′) were used and yielded PCR products of approximatively 200 bp in size. To control for RNA quality and cDNA synthesis, β-actin mRNA was also amplified with forward (5′-tacagcttcaccaccacagc-3′) and reverse primers (5′-aaggaaggctggaaaagagc-3′). Quantification involved the use of standard curves that had been prepared with plasmids containing specific sequences of each gene. We cloned the PCR products of CHOP, ASNS, ATF3 and β-actin into the pGEM-T easy vector (Promega) according to the manufacturer's instructions. For the construction of standard curves for CHOP, ASNS, ATF3 and β-actin, pGEM-T easy plasmids were prepared as 10-fold serial dilution in water, from 4 ng to 0.4 pg. PCR was carried out using a LightCycler™ System (Roche) as described previously (4). LightCycler quantification software (version 3.5) was used to compare amplification in experimental samples during the log-linear phase to the standard curve from the dilution series of control plasmids. Relative results were displayed in nanograms of CHOP, ASNS and ATF3 per 100 nanograms of β-actin. Each experiment was repeated three times to confirm the reproducibility of the results.

Analysis of gene expression using real-time RT-PCR

Total RNA was prepared using a RNeasy mini kit (Qiagen) and treated with DNase I, Amp Grade (InVitrogen) prior to cDNA synthesis. RNA integrity was electrophoretically verified by ethidium bromide staining. RNA (0.5 μg) was reverse transcribed with 100 U of Superscript II plus RNase H− Reverse Transcriptase (InVitrogen) using 100 μM random hexamer primers (Amersham Biosciences), according to the manufacturer's instructions. Primers for human sequences (used in HeLa cells): hCHOP (forward primer, 5′-cagaaccagcagaggtcaca-3′; reverse primer, 5′-agctgtgccactttcctttc-3′). Primers for mouse sequences (used in ATF4 +/+, ATF4 −/−, ATF2 +/+ and ATF2 −/− cells): mCHOP (forward primer, 5′-cctagcttggctgacagagg-3′; reverse primer, 5′-ctgctccttctccttcatgc-3′), mASNS (forward primer, 5′-tacaaccacaaggcgctaca-3′; reverse primer, 5′-aagggcctgactccataggt-3′) and mATF3 (forward primer, 5′-cgccatccagaataaacacc-3′; reverse primer, 5′-gcagccactctgtcttctcc-3′) were used and yielded PCR products of approximatively 200 bp in size. To control for RNA quality and cDNA synthesis, β-actin mRNA was also amplified with forward (5′-tacagcttcaccaccacagc-3′) and reverse primers (5′-aaggaaggctggaaaagagc-3′). Quantification involved the use of standard curves that had been prepared with plasmids containing specific sequences of each gene. We cloned the PCR products of CHOP, ASNS, ATF3 and β-actin into the pGEM-T easy vector (Promega) according to the manufacturer's instructions. For the construction of standard curves for CHOP, ASNS, ATF3 and β-actin, pGEM-T easy plasmids were prepared as 10-fold serial dilution in water, from 4 ng to 0.4 pg. PCR was carried out using a LightCycler™ System (Roche) as described previously (4). LightCycler quantification software (version 3.5) was used to compare amplification in experimental samples during the log-linear phase to the standard curve from the dilution series of control plasmids. Relative results were displayed in nanograms of CHOP, ASNS and ATF3 per 100 nanograms of β-actin. Each experiment was repeated three times to confirm the reproducibility of the results.

Nuclear extract preparationNuclear extracts were prepared as described previously (10).

Nuclear extract preparation

Nuclear extracts were prepared as described previously (10).

Western blot analysisNuclear proteins were resolved by SDS-polyacrylamide gel electrophoresis and transferred onto a Hybond-P PVDF membrane (Amersham Biosciences). Membranes were blocked for 1 h at room temperature with a solution of 5% non-fat milk powder in TN (50 mM Tris-HCL, pH 8.0, 150 mM NaCl, 0.1% Tween-20). The blots were then incubated with antibody in blocking solution overnight at 4°C. Antibodies were diluted according to the manufacturer's instructions. The blots were washed three times in TN and incubated with horseradish peroxidase-conjugated goat anti-rabbit IgG (1:20 000) (Santa Cruz, CA) in blocking buffer for 1 h at room temperature. After three washes, the blots were developed using the enhanced chemiluminescence (ECL) detection system (Amersham Biosciences).

Western blot analysis

Nuclear proteins were resolved by SDS-polyacrylamide gel electrophoresis and transferred onto a Hybond-P PVDF membrane (Amersham Biosciences). Membranes were blocked for 1 h at room temperature with a solution of 5% non-fat milk powder in TN (50 mM Tris-HCL, pH 8.0, 150 mM NaCl, 0.1% Tween-20). The blots were then incubated with antibody in blocking solution overnight at 4°C. Antibodies were diluted according to the manufacturer's instructions. The blots were washed three times in TN and incubated with horseradish peroxidase-conjugated goat anti-rabbit IgG (1:20 000) (Santa Cruz, CA) in blocking buffer for 1 h at room temperature. After three washes, the blots were developed using the enhanced chemiluminescence (ECL) detection system (Amersham Biosciences).

AntibodiesThe following antibodies were purchased from Santa Cruz Biotechnology, Inc (Santa Cruz, CA): ATF2, sc-187; ATF4, sc-200. Anti-phospho-ATF2 (catalog no. 9221) was obtained from Cell Signaling Technology (Beverly, MA). Acetylated histone H3, 06-599 (recognizes acetylated H3 at Lys-9 and Lys-14) and acetylated histone H4, 06-866 (recognizes acetylated H4 at Lys-5, -8, -12 and -16) antibodies were purchased from Upstate Biotechnology (Charlottes-ville, VA). Acetylated histone H2B (recognizes acetylated H2B at Lys-12 and Lys-15) antibody was from Abcam (Cambridge, UK).

The following antibodies were purchased from Santa Cruz Biotechnology, Inc (Santa Cruz, CA): ATF2, sc-187; ATF4, sc-200. Anti-phospho-ATF2 (catalog no. 9221) was obtained from Cell Signaling Technology (Beverly, MA). Acetylated histone H3, 06-599 (recognizes acetylated H3 at Lys-9 and Lys-14) and acetylated histone H4, 06-866 (recognizes acetylated H4 at Lys-5, -8, -12 and -16) antibodies were purchased from Upstate Biotechnology (Charlottes-ville, VA). Acetylated histone H2B (recognizes acetylated H2B at Lys-12 and Lys-15) antibody was from Abcam (Cambridge, UK).

Chromatin immunoprecipitation analysis (ChIP)ChIP analysis was performed according to the protocol of Upstate Biotechnology, Inc. (Charlottesville, VA) with minor modifications. Cells were seeded at 1 × 106/100-mm dish with DMEM F12 and grown for 24 h. Cells were transferred to fresh DMEM F12 12 h before transfer to either complete DMEM F12 or DMEM F12 lacking leucine for the time period indicated in each figure. Protein–DNA was cross-linked by adding formaldehyde directly to the culture medium to a final concentration of 1% and then stopped 8 min later by the addition of glycine to a final concentration of 0.125 M. Cross-linked chromatin was sonicated using a Vibra cell sonicator (Bioblock Scientific Technology) for 10 bursts of 30 s at power 2 with 1-min cooling on ice between each burst to obtain DNA fragments of an average of 400 bp. Extracts from 1 × 106 cells were incubated with 5 µg of antibody. A rabbit anti-chicken IgG was used as the non-specific antibody control. The antibody-bound complex was precipitated by protein A-Agarose beads (Upstate Biotechnology). The DNA fragments in the immunoprecipitated complex were released by reversing the cross-linking overnight at 65°C and purified using a phenol/chloroform extraction and ethanol precipitation. Real-time quantitative PCR was performed by using a LightCycler (Roche) and a SYBR-Green-I-containing PCR mix (Qiagen), following the recommendations of the manufacturer. The immunoprecipitated material was quantified relative to a standard curve of genomic DNA. Primers used for human sequences: hCHOP amplicon A, 5′-gcagcctaaccaaagacctg-3′ and 5′-ggaggcaacttgaccaaaag-3′; hCHOP amplicon B (AARE), 5′-aagaggctcacgaccgacta-3′ and 5′-atgatgcaatgtttggcaac-3′; hCHOP amplicon C, 5′-agtgccacggagaaagctaa-3′ and 5′-ccatacagcagcctgagtga-3′. Primers used for mouse sequences: mCHOP AARE, 5′-gggcagacaagttcaggaag-3′ and 5′-atgatgcaatgtttggcaac-3′; mASNS AARE, 5′-cagaacacctcctggctctc-3′ and 5′-ccgcttgccaccttagagtc-3′, mATF3 AARE, 5′- ggtctccacccaccttttg -3′ and 5′- ctcgctgagtgagactgtgg -3′. The reactions were incubated at 95°C for 15 min to activate the polymerase, followed by amplification at 95°C for 15 s, 55°C for 20 s and 72°C for 20 s for 45 cycles. After PCR, melting curves were acquired by stepwise increases in the temperature from 65 to 95°C to ensure that a single product was amplified in the reaction. The results are expressed as the percentage of antibody binding versus the amount of PCR product obtained using a standardized aliquot of input chromatin. Values are the means from at least three independent immunoprecipitations.

Chromatin immunoprecipitation analysis (ChIP)

ChIP analysis was performed according to the protocol of Upstate Biotechnology, Inc. (Charlottesville, VA) with minor modifications. Cells were seeded at 1 × 106/100-mm dish with DMEM F12 and grown for 24 h. Cells were transferred to fresh DMEM F12 12 h before transfer to either complete DMEM F12 or DMEM F12 lacking leucine for the time period indicated in each figure. Protein–DNA was cross-linked by adding formaldehyde directly to the culture medium to a final concentration of 1% and then stopped 8 min later by the addition of glycine to a final concentration of 0.125 M. Cross-linked chromatin was sonicated using a Vibra cell sonicator (Bioblock Scientific Technology) for 10 bursts of 30 s at power 2 with 1-min cooling on ice between each burst to obtain DNA fragments of an average of 400 bp. Extracts from 1 × 106 cells were incubated with 5 µg of antibody. A rabbit anti-chicken IgG was used as the non-specific antibody control. The antibody-bound complex was precipitated by protein A-Agarose beads (Upstate Biotechnology). The DNA fragments in the immunoprecipitated complex were released by reversing the cross-linking overnight at 65°C and purified using a phenol/chloroform extraction and ethanol precipitation. Real-time quantitative PCR was performed by using a LightCycler (Roche) and a SYBR-Green-I-containing PCR mix (Qiagen), following the recommendations of the manufacturer. The immunoprecipitated material was quantified relative to a standard curve of genomic DNA. Primers used for human sequences: hCHOP amplicon A, 5′-gcagcctaaccaaagacctg-3′ and 5′-ggaggcaacttgaccaaaag-3′; hCHOP amplicon B (AARE), 5′-aagaggctcacgaccgacta-3′ and 5′-atgatgcaatgtttggcaac-3′; hCHOP amplicon C, 5′-agtgccacggagaaagctaa-3′ and 5′-ccatacagcagcctgagtga-3′. Primers used for mouse sequences: mCHOP AARE, 5′-gggcagacaagttcaggaag-3′ and 5′-atgatgcaatgtttggcaac-3′; mASNS AARE, 5′-cagaacacctcctggctctc-3′ and 5′-ccgcttgccaccttagagtc-3′, mATF3 AARE, 5′- ggtctccacccaccttttg -3′ and 5′- ctcgctgagtgagactgtgg -3′. The reactions were incubated at 95°C for 15 min to activate the polymerase, followed by amplification at 95°C for 15 s, 55°C for 20 s and 72°C for 20 s for 45 cycles. After PCR, melting curves were acquired by stepwise increases in the temperature from 65 to 95°C to ensure that a single product was amplified in the reaction. The results are expressed as the percentage of antibody binding versus the amount of PCR product obtained using a standardized aliquot of input chromatin. Values are the means from at least three independent immunoprecipitations.

RESULTSATF2 and ATF4 are required in the early stages of amino acid starvation to activate CHOP transcriptionIn previous studies, ATF2 and ATF4 were shown to have critical roles for CHOP induction in response to 4–16 h of leucine starvation (4,10). To further dissect the early molecular events involved in the induction of CHOP transcription upon amino acid starvation, we first examined the kinetics of the increase in CHOP mRNA content in response to a short period of leucine starvation in human HeLa cells. CHOP mRNA was increased 2.5-fold after 1 h of leucine starvation and reached a maximum level (9–10-fold) after 4–6 h (Figure 1A).
Figure 1.Rapid induction of CHOP mRNA by leucine starvation. (A) HeLa cells were incubated either in control (+leu) or leucine-free medium (−leu) and harvested after the indicated incubation times. Total RNA was extracted and real time RT-PCR was performed as described in Materials and Methods. The mRNA induction level is defined as the ratio of the relative mRNA level of leucine-starved cells to that of non-starved cells. (B) Sequence of the CHOP AARE (−313 to −295). The minimum AARE core sequence is boxed in grey. (C) ATF4 +/+ or ATF4 −/− MEF or (D) ATF2 +/+ or ATF2 −/− MEF. In (C) and (D) cells were incubated either in control (+leu) or leucine-free medium (−leu) and harvested after the indicated incubation times. Total RNA was extracted and real time RT-PCR was performed as described in Materials and Methods. The mRNA induction level is defined as the ratio of the relative mRNA level of leucine starved cells to that of non-starved cells. (E) ATF4 +/+ or ATF4 −/− MEF (left panel) or ATF2 +/+ or ATF2 −/− MEF (right panel) were incubated either in control (+) or leucine-free medium (−) for 4 h and western blot analysis of ATF2, ATF4 and phospho-ATF2 (Thr-71) was performed from nuclear extracts.We previously identified a genomic cis-acting element (AARE) involved in the transcriptional activation of CHOP by leucine starvation and showed that it binds ATF2 and ATF4 (Figure 1B) (4,10). To determine whether ATF2 or ATF4 is required to mediate the CHOP mRNA induction during a short period of leucine starvation, we used MEFs deficient in ATF2 or ATF4 and their corresponding wild type cells. The time course of CHOP mRNA induction by leucine starvation in wild type MEFs was similar to that observed in HeLa cells (Figure 1C and D). In contrast, lack of either ATF4 or ATF2 resulted in a complete loss of CHOP mRNA inducibility. Protein analysis shows that the induction of ATF2 phosphorylation was still observed upon leucine starvation in MEFs deficient in ATF4 expression (Figure 1E). On the other hand, the lack of ATF2 did not affect the increase of ATF4 expression. Taken together, these results demonstrate that induction of CHOP gene expression occurs rapidly following leucine starvation and that both ATF2 and ATF4 are critical for this induction.Binding of ATF2 and ATF4 to the CHOP AARE in vivo is associated with acetylation of histones H4 and H2B in response to amino acid deprivationElectrophoresis mobility shift analysis has demonstrated that ATF2 and ATF4 are each capable of binding to the AARE sequence within the CHOP promoter in vitro (4,10). To investigate in vivo binding of these factors, HeLa cells were incubated in control or leucine-free medium for 2 h and ChIP assays were performed with primer sets covering either the 5′ region (amplicon A), the AARE (amplicon B) or the first intron (amplicon C) of the CHOP gene (Figure 2A). The results show an increase in ATF4 binding to the AARE following 2 h of leucine deprivation whereas the binding of ATF2 remained constitutive (Figure 2B). Furthermore, binding of ATF2 and ATF4 was not detected in the 5′ region and in the first intron of CHOP confirming that these factors bind specifically to the AARE.
Figure 2.Transcription factor binding and histone acetylation to CHOP AARE in response to leucine starvation. (A) Scheme of the human CHOP gene indicating the different amplicons produced for the ChIP analysis: A (−1678 to −1478), B (−472 to −301) and C (+1163 to +1372). The AARE is boxed in grey. (B) HeLa cells were incubated 2 h either in control (+leu) or leucine-free medium (−leu) and harvested. ChIP analysis was performed as described under Materials and Methods using antibodies specific for ATF2 and ATF4 and different sets of primers to produce amplicon A, B or C. Data were plotted as the percentage of antibody binding versus the amount of PCR product obtained using a standardized aliquot of input chromatin. Each point represents the mean value of three independent experiments, and the error bars represent the standard error of the means. (C) The experiment described in (B) was also performed using antibodies specific for acetylated H3, acetylated H4 and acetylated H2B.To test the influence of leucine deprivation on the modification of chromatin structure, acetylation of histones was analysed. The abundance of acetylated histones H4 and H2B significantly increased in leucine-deprived cells whereas the acetylation of histone H3 remained unchanged (Figure 2C). This increase was observed in the AARE as well as in the 5′ region and first intron of CHOP indicating that the change in histone acetylation is propagated in the vicinity of the AARE.Phosphorylation of ATF2 bound on the CHOP AARE precedes histones H4 and H2B acetylation and CHOP mRNA increaseWe next investigated the kinetics of ATF2 and ATF4 binding to the CHOP AARE. ATF4 levels increased after 1 h of leucine deprivation (Figure 3A) and rapidly reached a maximum within 2 h. By contrast, ATF2 binding to the CHOP AARE remained constitutive throughout the 4 h period investigated. ChIP analysis using an antibody that specifically recognize ATF2 phosphorylated on threonine 71 revealed that the phosphorylation of ATF2 was detectable 30 min after removal of leucine from the medium and reached a maximum level of phosphorylation within 2 h.
Figure 3.Time course of ATF2 and ATF4 binding and histone acetylation during leucine starvation. (A) HeLa cells were incubated either in control (+leu) or leucine-free medium (−leu) and harvested for 0–4 h. ChIP analysis was performed as described under Materials and Methods using antibodies specific for (A) ATF4, ATF2, phospho-ATF2 (Thr-71) and (B) acetylated H3, acetylated H4 and acetylated H2B and a set of primers to amplify amplicon B (see Figure 2A). Data were plotted as the percentage of antibody binding versus the amount of PCR product obtained using a standardized aliquot of input chromatin. Each point represents the mean value of three independent experiments and the error bars represent the standard error of the means. The dotted line represents the increase in CHOP mRNA induction level as shown in Figure 1A.We also investigated the kinetics of histone acetylation and found a significant increase in acetylated H4 and H2B at the CHOP AARE region within 1 h of removal of leucine from the medium and maintained for the 4 h period (Figure 3B). The acetylation of histone H3 remained unchanged. By plotting the mRNA content on the same graph, it is apparent that the binding of ATF4 and the acetylation of histones H4 and H2B closely paralleled the increase in CHOP mRNA in the first 2 h (Figure 3A and B). Most importantly, these results demonstrate that the phosphorylation of ATF2 on threonine 71 precedes histone acetylation and the induction of CHOP transcription.ATF2 is essential for histone H4 and H2B acetylation in response to amino acid deprivationThe results described above suggest that ATF2 could play a critical role in the acetylation of histone and thus in the onset of CHOP transcription upon amino acid starvation. To investigate the link between ATF2 and acetylation of histones, ChIP experiments were performed in MEFs deficient in ATF2 and in the corresponding wild type cells incubated either in control or in leucine-starved medium for 2 h. The ChIP results obtained with wild type MEFs (Figure 4A) are consistent with those described above with HeLa cells (Figure 2B and C). In ATF2-deficient cells, no ATF2 or phosphorylated ATF2 bound to the CHOP AARE is detected. However, the increase in the ATF4 binding remains (Figure 4A). Furthermore, in the absence of ATF2, the increase in histone H4 and H2B acetylation in response to amino acid starvation was lost. The same result was obtained with cells starved for 4 h with leucine (data not shown). ATF2 therefore is essential for the acetylation of histones H4 and H2B and thereby plays a crucial role in the modification of the chromatin structure associated with activation of CHOP transcription. By contrast, in cells lacking ATF4, the levels of ATF2 phosphorylation and of histone H4 and H2B acetylation are unchanged (Figure 4B). Taken together, these results demonstrate that histone acetylation and ATF4 binding are two independent events that are required for the CHOP induction upon amino acid starvation. It is noticeable that it remains an ATF2-independent level of acetylated histone. ChIP experiments were performed from ATF2 KO cells with primer sets covering much farther upstream or downstream from the CHOP gene (data not shown). The results indicated that the amount of histone acetylation in ATF2-deficient cells is due to the background observed for each histone antibody.
Figure 4.Role of ATF2 in histone acetylation in response to leucine deprivation. (A) ATF2 +/+ and ATF2 −/− MEF were incubated 2 h either in control (+leu) or leucine-free medium (−leu) and harvested. ChIP analysis was performed as described under Materials and Methods using antibodies specific for ATF4, ATF2, phospho-ATF2 (Thr-71), acetylated H3, acetylated H4 and acetylated H2B and a set of primers to produce amplicon B (Figure 2A). Data were plotted as the percentage of antibody binding versus the amount of PCR product obtained using a standardized aliquot of input chromatin. Each point represents the mean value of three independent experiments and the error bars represent the standard error of the means. (B) The same experiment as described in (A) was also performed with wild ATF4 +/+ and ATF4 −/− MEF.The role of ATF2 in histone acetylation can be extended to other amino acid-regulated genes and is linked to its binding to AARE sequencesWe have previously shown that the requirement for ATF2 in amino acid-regulated gene expression varies according to the tested genes (4). Although loss of ATF2 entirely abolished the leucine control of most of the tested genes, the amino acid control of certain genes remains partially or completely insensitive to such loss. Based on these data, we have chosen to study two other known functional AARE sequences for their ability to bind ATF2 differentially.To determine whether other amino-acid-regulated genes also require ATF2-mediated histone acetylation for their activation, we have tested the ATF3 gene. ATF3 has been shown to contain a functional AARE that is required to mediate the transcriptional activation in response to amino acid starvation (12). Sequence analysis revealed that the 9 bp core sequence of this element differs from that in the CHOP promoter by only one nucleotide (Figure 5A). Lack of ATF2 resulted in a loss of the ATF3 mRNA inducibility in response to leucine starvation (Figure 5B). ChIP assays were performed with a primer set covering the AARE of the ATF3 gene and using an anti-acetylated histone H4 antibody (Figure 5C). In wild type cells, the binding of ATF2 to the ATF3 AARE remained constitutive whereas ATF4 binding and histone H4 acetylation were increased following leucine deprivation. Lack of ATF2 resulted in a complete loss in the elevation of histone H4 acetylation. These data demonstrate that the role of ATF2 in histone H4 acetylation is not restricted to amino acid regulation of CHOP and can be extended to another amino acid-regulated gene.
Figure 5.Role of ATF2 in the transcriptional regulation of ATF3 and ASNS in response to amino acid starvation. (A) Sequence comparison of the human CHOP AARE (−313 to −298) with the human ATF3 AARE (−12 to −27) and the human ASNS AARE (−57 to −72). The minimum AARE core sequence is boxed. Identical nucleotides in the core are boxed in grey. ATF2 +/+ or ATF2 −/− MEF were incubated either in control (+leu) or leucine-free medium (−leu) and harvested after 4 h. Total RNA was extracted and analysed by real time RT-PCR for ATF3 (B) and ASNS (C) mRNA content as described in Materials and Methods. ChIP analysis was performed as described under Materials and Methods using antibodies specific for ATF4, ATF2 and acetylated histone H4 and two different sets of primers to amplify (D) ATF3 AARE or (E) ASNS AARE. Data were plotted as the percentage of antibody binding versus the amount of PCR product obtained using a standardized aliquot of input chromatin. Each point represents the mean value of three independent experiments, and the error bars represent the standard error of the means.To further investigate a link between the binding of ATF2 to AARE sequences and the increase in histone H4 acetylation, we have chosen to study the AARE sequences of ASNS which have been described to bind ATF2 poorly even though it shares nucleotide sequence similarities with the CHOP and ATF3 AAREs (23) (Figure 5A). According to these data, ATF2 is not essential in the specific amino acid pathway that leads to the induction of ASNS transcription (Figure 5D). ChIP experiments were performed in MEF deficient in ATF2 and in the corresponding wild type cells with a primer set covering the AARE of the ASNS gene (Figure 5E). As expected, the binding of ATF4 to this AARE was increased in response to amino acid starvation in both cell types. Most importantly, no significant binding of ATF2 to the ASNS AARE was detected in wild type cells while, in contrast to the results obtained with the CHOP and ATF3 AAREs the lack of ATF2 did not affect the increase in histone H4 acetylation in the vicinity of the AARE following leucine deprivation. These findings show that an ATF2-independent HAT activity is involved in the amino acid regulation of ASNS transcription.

ATF2 and ATF4 are required in the early stages of amino acid starvation to activate CHOP transcriptionIn previous studies, ATF2 and ATF4 were shown to have critical roles for CHOP induction in response to 4–16 h of leucine starvation (4,10). To further dissect the early molecular events involved in the induction of CHOP transcription upon amino acid starvation, we first examined the kinetics of the increase in CHOP mRNA content in response to a short period of leucine starvation in human HeLa cells. CHOP mRNA was increased 2.5-fold after 1 h of leucine starvation and reached a maximum level (9–10-fold) after 4–6 h (Figure 1A).
Figure 1.Rapid induction of CHOP mRNA by leucine starvation. (A) HeLa cells were incubated either in control (+leu) or leucine-free medium (−leu) and harvested after the indicated incubation times. Total RNA was extracted and real time RT-PCR was performed as described in Materials and Methods. The mRNA induction level is defined as the ratio of the relative mRNA level of leucine-starved cells to that of non-starved cells. (B) Sequence of the CHOP AARE (−313 to −295). The minimum AARE core sequence is boxed in grey. (C) ATF4 +/+ or ATF4 −/− MEF or (D) ATF2 +/+ or ATF2 −/− MEF. In (C) and (D) cells were incubated either in control (+leu) or leucine-free medium (−leu) and harvested after the indicated incubation times. Total RNA was extracted and real time RT-PCR was performed as described in Materials and Methods. The mRNA induction level is defined as the ratio of the relative mRNA level of leucine starved cells to that of non-starved cells. (E) ATF4 +/+ or ATF4 −/− MEF (left panel) or ATF2 +/+ or ATF2 −/− MEF (right panel) were incubated either in control (+) or leucine-free medium (−) for 4 h and western blot analysis of ATF2, ATF4 and phospho-ATF2 (Thr-71) was performed from nuclear extracts.We previously identified a genomic cis-acting element (AARE) involved in the transcriptional activation of CHOP by leucine starvation and showed that it binds ATF2 and ATF4 (Figure 1B) (4,10). To determine whether ATF2 or ATF4 is required to mediate the CHOP mRNA induction during a short period of leucine starvation, we used MEFs deficient in ATF2 or ATF4 and their corresponding wild type cells. The time course of CHOP mRNA induction by leucine starvation in wild type MEFs was similar to that observed in HeLa cells (Figure 1C and D). In contrast, lack of either ATF4 or ATF2 resulted in a complete loss of CHOP mRNA inducibility. Protein analysis shows that the induction of ATF2 phosphorylation was still observed upon leucine starvation in MEFs deficient in ATF4 expression (Figure 1E). On the other hand, the lack of ATF2 did not affect the increase of ATF4 expression. Taken together, these results demonstrate that induction of CHOP gene expression occurs rapidly following leucine starvation and that both ATF2 and ATF4 are critical for this induction.

ATF2 and ATF4 are required in the early stages of amino acid starvation to activate CHOP transcription

In previous studies, ATF2 and ATF4 were shown to have critical roles for CHOP induction in response to 4–16 h of leucine starvation (4,10). To further dissect the early molecular events involved in the induction of CHOP transcription upon amino acid starvation, we first examined the kinetics of the increase in CHOP mRNA content in response to a short period of leucine starvation in human HeLa cells. CHOP mRNA was increased 2.5-fold after 1 h of leucine starvation and reached a maximum level (9–10-fold) after 4–6 h (Figure 1A).
Figure 1.Rapid induction of CHOP mRNA by leucine starvation. (A) HeLa cells were incubated either in control (+leu) or leucine-free medium (−leu) and harvested after the indicated incubation times. Total RNA was extracted and real time RT-PCR was performed as described in Materials and Methods. The mRNA induction level is defined as the ratio of the relative mRNA level of leucine-starved cells to that of non-starved cells. (B) Sequence of the CHOP AARE (−313 to −295). The minimum AARE core sequence is boxed in grey. (C) ATF4 +/+ or ATF4 −/− MEF or (D) ATF2 +/+ or ATF2 −/− MEF. In (C) and (D) cells were incubated either in control (+leu) or leucine-free medium (−leu) and harvested after the indicated incubation times. Total RNA was extracted and real time RT-PCR was performed as described in Materials and Methods. The mRNA induction level is defined as the ratio of the relative mRNA level of leucine starved cells to that of non-starved cells. (E) ATF4 +/+ or ATF4 −/− MEF (left panel) or ATF2 +/+ or ATF2 −/− MEF (right panel) were incubated either in control (+) or leucine-free medium (−) for 4 h and western blot analysis of ATF2, ATF4 and phospho-ATF2 (Thr-71) was performed from nuclear extracts.

Rapid induction of CHOP mRNA by leucine starvation. (A) HeLa cells were incubated either in control (+leu) or leucine-free medium (−leu) and harvested after the indicated incubation times. Total RNA was extracted and real time RT-PCR was performed as described in Materials and Methods. The mRNA induction level is defined as the ratio of the relative mRNA level of leucine-starved cells to that of non-starved cells. (B) Sequence of the CHOP AARE (−313 to −295). The minimum AARE core sequence is boxed in grey. (C) ATF4 +/+ or ATF4 −/− MEF or (D) ATF2 +/+ or ATF2 −/− MEF. In (C) and (D) cells were incubated either in control (+leu) or leucine-free medium (−leu) and harvested after the indicated incubation times. Total RNA was extracted and real time RT-PCR was performed as described in Materials and Methods. The mRNA induction level is defined as the ratio of the relative mRNA level of leucine starved cells to that of non-starved cells. (E) ATF4 +/+ or ATF4 −/− MEF (left panel) or ATF2 +/+ or ATF2 −/− MEF (right panel) were incubated either in control (+) or leucine-free medium (−) for 4 h and western blot analysis of ATF2, ATF4 and phospho-ATF2 (Thr-71) was performed from nuclear extracts.

We previously identified a genomic cis-acting element (AARE) involved in the transcriptional activation of CHOP by leucine starvation and showed that it binds ATF2 and ATF4 (Figure 1B) (4,10). To determine whether ATF2 or ATF4 is required to mediate the CHOP mRNA induction during a short period of leucine starvation, we used MEFs deficient in ATF2 or ATF4 and their corresponding wild type cells. The time course of CHOP mRNA induction by leucine starvation in wild type MEFs was similar to that observed in HeLa cells (Figure 1C and D). In contrast, lack of either ATF4 or ATF2 resulted in a complete loss of CHOP mRNA inducibility. Protein analysis shows that the induction of ATF2 phosphorylation was still observed upon leucine starvation in MEFs deficient in ATF4 expression (Figure 1E). On the other hand, the lack of ATF2 did not affect the increase of ATF4 expression. Taken together, these results demonstrate that induction of CHOP gene expression occurs rapidly following leucine starvation and that both ATF2 and ATF4 are critical for this induction.

Binding of ATF2 and ATF4 to the CHOP AARE in vivo is associated with acetylation of histones H4 and H2B in response to amino acid deprivationElectrophoresis mobility shift analysis has demonstrated that ATF2 and ATF4 are each capable of binding to the AARE sequence within the CHOP promoter in vitro (4,10). To investigate in vivo binding of these factors, HeLa cells were incubated in control or leucine-free medium for 2 h and ChIP assays were performed with primer sets covering either the 5′ region (amplicon A), the AARE (amplicon B) or the first intron (amplicon C) of the CHOP gene (Figure 2A). The results show an increase in ATF4 binding to the AARE following 2 h of leucine deprivation whereas the binding of ATF2 remained constitutive (Figure 2B). Furthermore, binding of ATF2 and ATF4 was not detected in the 5′ region and in the first intron of CHOP confirming that these factors bind specifically to the AARE.
Figure 2.Transcription factor binding and histone acetylation to CHOP AARE in response to leucine starvation. (A) Scheme of the human CHOP gene indicating the different amplicons produced for the ChIP analysis: A (−1678 to −1478), B (−472 to −301) and C (+1163 to +1372). The AARE is boxed in grey. (B) HeLa cells were incubated 2 h either in control (+leu) or leucine-free medium (−leu) and harvested. ChIP analysis was performed as described under Materials and Methods using antibodies specific for ATF2 and ATF4 and different sets of primers to produce amplicon A, B or C. Data were plotted as the percentage of antibody binding versus the amount of PCR product obtained using a standardized aliquot of input chromatin. Each point represents the mean value of three independent experiments, and the error bars represent the standard error of the means. (C) The experiment described in (B) was also performed using antibodies specific for acetylated H3, acetylated H4 and acetylated H2B.To test the influence of leucine deprivation on the modification of chromatin structure, acetylation of histones was analysed. The abundance of acetylated histones H4 and H2B significantly increased in leucine-deprived cells whereas the acetylation of histone H3 remained unchanged (Figure 2C). This increase was observed in the AARE as well as in the 5′ region and first intron of CHOP indicating that the change in histone acetylation is propagated in the vicinity of the AARE.

Binding of ATF2 and ATF4 to the CHOP AARE in vivo is associated with acetylation of histones H4 and H2B in response to amino acid deprivation

Electrophoresis mobility shift analysis has demonstrated that ATF2 and ATF4 are each capable of binding to the AARE sequence within the CHOP promoter in vitro (4,10). To investigate in vivo binding of these factors, HeLa cells were incubated in control or leucine-free medium for 2 h and ChIP assays were performed with primer sets covering either the 5′ region (amplicon A), the AARE (amplicon B) or the first intron (amplicon C) of the CHOP gene (Figure 2A). The results show an increase in ATF4 binding to the AARE following 2 h of leucine deprivation whereas the binding of ATF2 remained constitutive (Figure 2B). Furthermore, binding of ATF2 and ATF4 was not detected in the 5′ region and in the first intron of CHOP confirming that these factors bind specifically to the AARE.
Figure 2.Transcription factor binding and histone acetylation to CHOP AARE in response to leucine starvation. (A) Scheme of the human CHOP gene indicating the different amplicons produced for the ChIP analysis: A (−1678 to −1478), B (−472 to −301) and C (+1163 to +1372). The AARE is boxed in grey. (B) HeLa cells were incubated 2 h either in control (+leu) or leucine-free medium (−leu) and harvested. ChIP analysis was performed as described under Materials and Methods using antibodies specific for ATF2 and ATF4 and different sets of primers to produce amplicon A, B or C. Data were plotted as the percentage of antibody binding versus the amount of PCR product obtained using a standardized aliquot of input chromatin. Each point represents the mean value of three independent experiments, and the error bars represent the standard error of the means. (C) The experiment described in (B) was also performed using antibodies specific for acetylated H3, acetylated H4 and acetylated H2B.

Transcription factor binding and histone acetylation to CHOP AARE in response to leucine starvation. (A) Scheme of the human CHOP gene indicating the different amplicons produced for the ChIP analysis: A (−1678 to −1478), B (−472 to −301) and C (+1163 to +1372). The AARE is boxed in grey. (B) HeLa cells were incubated 2 h either in control (+leu) or leucine-free medium (−leu) and harvested. ChIP analysis was performed as described under Materials and Methods using antibodies specific for ATF2 and ATF4 and different sets of primers to produce amplicon A, B or C. Data were plotted as the percentage of antibody binding versus the amount of PCR product obtained using a standardized aliquot of input chromatin. Each point represents the mean value of three independent experiments, and the error bars represent the standard error of the means. (C) The experiment described in (B) was also performed using antibodies specific for acetylated H3, acetylated H4 and acetylated H2B.

To test the influence of leucine deprivation on the modification of chromatin structure, acetylation of histones was analysed. The abundance of acetylated histones H4 and H2B significantly increased in leucine-deprived cells whereas the acetylation of histone H3 remained unchanged (Figure 2C). This increase was observed in the AARE as well as in the 5′ region and first intron of CHOP indicating that the change in histone acetylation is propagated in the vicinity of the AARE.

Phosphorylation of ATF2 bound on the CHOP AARE precedes histones H4 and H2B acetylation and CHOP mRNA increaseWe next investigated the kinetics of ATF2 and ATF4 binding to the CHOP AARE. ATF4 levels increased after 1 h of leucine deprivation (Figure 3A) and rapidly reached a maximum within 2 h. By contrast, ATF2 binding to the CHOP AARE remained constitutive throughout the 4 h period investigated. ChIP analysis using an antibody that specifically recognize ATF2 phosphorylated on threonine 71 revealed that the phosphorylation of ATF2 was detectable 30 min after removal of leucine from the medium and reached a maximum level of phosphorylation within 2 h.
Figure 3.Time course of ATF2 and ATF4 binding and histone acetylation during leucine starvation. (A) HeLa cells were incubated either in control (+leu) or leucine-free medium (−leu) and harvested for 0–4 h. ChIP analysis was performed as described under Materials and Methods using antibodies specific for (A) ATF4, ATF2, phospho-ATF2 (Thr-71) and (B) acetylated H3, acetylated H4 and acetylated H2B and a set of primers to amplify amplicon B (see Figure 2A). Data were plotted as the percentage of antibody binding versus the amount of PCR product obtained using a standardized aliquot of input chromatin. Each point represents the mean value of three independent experiments and the error bars represent the standard error of the means. The dotted line represents the increase in CHOP mRNA induction level as shown in Figure 1A.We also investigated the kinetics of histone acetylation and found a significant increase in acetylated H4 and H2B at the CHOP AARE region within 1 h of removal of leucine from the medium and maintained for the 4 h period (Figure 3B). The acetylation of histone H3 remained unchanged. By plotting the mRNA content on the same graph, it is apparent that the binding of ATF4 and the acetylation of histones H4 and H2B closely paralleled the increase in CHOP mRNA in the first 2 h (Figure 3A and B). Most importantly, these results demonstrate that the phosphorylation of ATF2 on threonine 71 precedes histone acetylation and the induction of CHOP transcription.

Phosphorylation of ATF2 bound on the CHOP AARE precedes histones H4 and H2B acetylation and CHOP mRNA increase

We next investigated the kinetics of ATF2 and ATF4 binding to the CHOP AARE. ATF4 levels increased after 1 h of leucine deprivation (Figure 3A) and rapidly reached a maximum within 2 h. By contrast, ATF2 binding to the CHOP AARE remained constitutive throughout the 4 h period investigated. ChIP analysis using an antibody that specifically recognize ATF2 phosphorylated on threonine 71 revealed that the phosphorylation of ATF2 was detectable 30 min after removal of leucine from the medium and reached a maximum level of phosphorylation within 2 h.
Figure 3.Time course of ATF2 and ATF4 binding and histone acetylation during leucine starvation. (A) HeLa cells were incubated either in control (+leu) or leucine-free medium (−leu) and harvested for 0–4 h. ChIP analysis was performed as described under Materials and Methods using antibodies specific for (A) ATF4, ATF2, phospho-ATF2 (Thr-71) and (B) acetylated H3, acetylated H4 and acetylated H2B and a set of primers to amplify amplicon B (see Figure 2A). Data were plotted as the percentage of antibody binding versus the amount of PCR product obtained using a standardized aliquot of input chromatin. Each point represents the mean value of three independent experiments and the error bars represent the standard error of the means. The dotted line represents the increase in CHOP mRNA induction level as shown in Figure 1A.

Time course of ATF2 and ATF4 binding and histone acetylation during leucine starvation. (A) HeLa cells were incubated either in control (+leu) or leucine-free medium (−leu) and harvested for 0–4 h. ChIP analysis was performed as described under Materials and Methods using antibodies specific for (A) ATF4, ATF2, phospho-ATF2 (Thr-71) and (B) acetylated H3, acetylated H4 and acetylated H2B and a set of primers to amplify amplicon B (see Figure 2A). Data were plotted as the percentage of antibody binding versus the amount of PCR product obtained using a standardized aliquot of input chromatin. Each point represents the mean value of three independent experiments and the error bars represent the standard error of the means. The dotted line represents the increase in CHOP mRNA induction level as shown in Figure 1A.

We also investigated the kinetics of histone acetylation and found a significant increase in acetylated H4 and H2B at the CHOP AARE region within 1 h of removal of leucine from the medium and maintained for the 4 h period (Figure 3B). The acetylation of histone H3 remained unchanged. By plotting the mRNA content on the same graph, it is apparent that the binding of ATF4 and the acetylation of histones H4 and H2B closely paralleled the increase in CHOP mRNA in the first 2 h (Figure 3A and B). Most importantly, these results demonstrate that the phosphorylation of ATF2 on threonine 71 precedes histone acetylation and the induction of CHOP transcription.

ATF2 is essential for histone H4 and H2B acetylation in response to amino acid deprivationThe results described above suggest that ATF2 could play a critical role in the acetylation of histone and thus in the onset of CHOP transcription upon amino acid starvation. To investigate the link between ATF2 and acetylation of histones, ChIP experiments were performed in MEFs deficient in ATF2 and in the corresponding wild type cells incubated either in control or in leucine-starved medium for 2 h. The ChIP results obtained with wild type MEFs (Figure 4A) are consistent with those described above with HeLa cells (Figure 2B and C). In ATF2-deficient cells, no ATF2 or phosphorylated ATF2 bound to the CHOP AARE is detected. However, the increase in the ATF4 binding remains (Figure 4A). Furthermore, in the absence of ATF2, the increase in histone H4 and H2B acetylation in response to amino acid starvation was lost. The same result was obtained with cells starved for 4 h with leucine (data not shown). ATF2 therefore is essential for the acetylation of histones H4 and H2B and thereby plays a crucial role in the modification of the chromatin structure associated with activation of CHOP transcription. By contrast, in cells lacking ATF4, the levels of ATF2 phosphorylation and of histone H4 and H2B acetylation are unchanged (Figure 4B). Taken together, these results demonstrate that histone acetylation and ATF4 binding are two independent events that are required for the CHOP induction upon amino acid starvation. It is noticeable that it remains an ATF2-independent level of acetylated histone. ChIP experiments were performed from ATF2 KO cells with primer sets covering much farther upstream or downstream from the CHOP gene (data not shown). The results indicated that the amount of histone acetylation in ATF2-deficient cells is due to the background observed for each histone antibody.
Figure 4.Role of ATF2 in histone acetylation in response to leucine deprivation. (A) ATF2 +/+ and ATF2 −/− MEF were incubated 2 h either in control (+leu) or leucine-free medium (−leu) and harvested. ChIP analysis was performed as described under Materials and Methods using antibodies specific for ATF4, ATF2, phospho-ATF2 (Thr-71), acetylated H3, acetylated H4 and acetylated H2B and a set of primers to produce amplicon B (Figure 2A). Data were plotted as the percentage of antibody binding versus the amount of PCR product obtained using a standardized aliquot of input chromatin. Each point represents the mean value of three independent experiments and the error bars represent the standard error of the means. (B) The same experiment as described in (A) was also performed with wild ATF4 +/+ and ATF4 −/− MEF.

ATF2 is essential for histone H4 and H2B acetylation in response to amino acid deprivation

The results described above suggest that ATF2 could play a critical role in the acetylation of histone and thus in the onset of CHOP transcription upon amino acid starvation. To investigate the link between ATF2 and acetylation of histones, ChIP experiments were performed in MEFs deficient in ATF2 and in the corresponding wild type cells incubated either in control or in leucine-starved medium for 2 h. The ChIP results obtained with wild type MEFs (Figure 4A) are consistent with those described above with HeLa cells (Figure 2B and C). In ATF2-deficient cells, no ATF2 or phosphorylated ATF2 bound to the CHOP AARE is detected. However, the increase in the ATF4 binding remains (Figure 4A). Furthermore, in the absence of ATF2, the increase in histone H4 and H2B acetylation in response to amino acid starvation was lost. The same result was obtained with cells starved for 4 h with leucine (data not shown). ATF2 therefore is essential for the acetylation of histones H4 and H2B and thereby plays a crucial role in the modification of the chromatin structure associated with activation of CHOP transcription. By contrast, in cells lacking ATF4, the levels of ATF2 phosphorylation and of histone H4 and H2B acetylation are unchanged (Figure 4B). Taken together, these results demonstrate that histone acetylation and ATF4 binding are two independent events that are required for the CHOP induction upon amino acid starvation. It is noticeable that it remains an ATF2-independent level of acetylated histone. ChIP experiments were performed from ATF2 KO cells with primer sets covering much farther upstream or downstream from the CHOP gene (data not shown). The results indicated that the amount of histone acetylation in ATF2-deficient cells is due to the background observed for each histone antibody.
Figure 4.Role of ATF2 in histone acetylation in response to leucine deprivation. (A) ATF2 +/+ and ATF2 −/− MEF were incubated 2 h either in control (+leu) or leucine-free medium (−leu) and harvested. ChIP analysis was performed as described under Materials and Methods using antibodies specific for ATF4, ATF2, phospho-ATF2 (Thr-71), acetylated H3, acetylated H4 and acetylated H2B and a set of primers to produce amplicon B (Figure 2A). Data were plotted as the percentage of antibody binding versus the amount of PCR product obtained using a standardized aliquot of input chromatin. Each point represents the mean value of three independent experiments and the error bars represent the standard error of the means. (B) The same experiment as described in (A) was also performed with wild ATF4 +/+ and ATF4 −/− MEF.

Role of ATF2 in histone acetylation in response to leucine deprivation. (A) ATF2 +/+ and ATF2 −/− MEF were incubated 2 h either in control (+leu) or leucine-free medium (−leu) and harvested. ChIP analysis was performed as described under Materials and Methods using antibodies specific for ATF4, ATF2, phospho-ATF2 (Thr-71), acetylated H3, acetylated H4 and acetylated H2B and a set of primers to produce amplicon B (Figure 2A). Data were plotted as the percentage of antibody binding versus the amount of PCR product obtained using a standardized aliquot of input chromatin. Each point represents the mean value of three independent experiments and the error bars represent the standard error of the means. (B) The same experiment as described in (A) was also performed with wild ATF4 +/+ and ATF4 −/− MEF.

The role of ATF2 in histone acetylation can be extended to other amino acid-regulated genes and is linked to its binding to AARE sequencesWe have previously shown that the requirement for ATF2 in amino acid-regulated gene expression varies according to the tested genes (4). Although loss of ATF2 entirely abolished the leucine control of most of the tested genes, the amino acid control of certain genes remains partially or completely insensitive to such loss. Based on these data, we have chosen to study two other known functional AARE sequences for their ability to bind ATF2 differentially.To determine whether other amino-acid-regulated genes also require ATF2-mediated histone acetylation for their activation, we have tested the ATF3 gene. ATF3 has been shown to contain a functional AARE that is required to mediate the transcriptional activation in response to amino acid starvation (12). Sequence analysis revealed that the 9 bp core sequence of this element differs from that in the CHOP promoter by only one nucleotide (Figure 5A). Lack of ATF2 resulted in a loss of the ATF3 mRNA inducibility in response to leucine starvation (Figure 5B). ChIP assays were performed with a primer set covering the AARE of the ATF3 gene and using an anti-acetylated histone H4 antibody (Figure 5C). In wild type cells, the binding of ATF2 to the ATF3 AARE remained constitutive whereas ATF4 binding and histone H4 acetylation were increased following leucine deprivation. Lack of ATF2 resulted in a complete loss in the elevation of histone H4 acetylation. These data demonstrate that the role of ATF2 in histone H4 acetylation is not restricted to amino acid regulation of CHOP and can be extended to another amino acid-regulated gene.
Figure 5.Role of ATF2 in the transcriptional regulation of ATF3 and ASNS in response to amino acid starvation. (A) Sequence comparison of the human CHOP AARE (−313 to −298) with the human ATF3 AARE (−12 to −27) and the human ASNS AARE (−57 to −72). The minimum AARE core sequence is boxed. Identical nucleotides in the core are boxed in grey. ATF2 +/+ or ATF2 −/− MEF were incubated either in control (+leu) or leucine-free medium (−leu) and harvested after 4 h. Total RNA was extracted and analysed by real time RT-PCR for ATF3 (B) and ASNS (C) mRNA content as described in Materials and Methods. ChIP analysis was performed as described under Materials and Methods using antibodies specific for ATF4, ATF2 and acetylated histone H4 and two different sets of primers to amplify (D) ATF3 AARE or (E) ASNS AARE. Data were plotted as the percentage of antibody binding versus the amount of PCR product obtained using a standardized aliquot of input chromatin. Each point represents the mean value of three independent experiments, and the error bars represent the standard error of the means.To further investigate a link between the binding of ATF2 to AARE sequences and the increase in histone H4 acetylation, we have chosen to study the AARE sequences of ASNS which have been described to bind ATF2 poorly even though it shares nucleotide sequence similarities with the CHOP and ATF3 AAREs (23) (Figure 5A). According to these data, ATF2 is not essential in the specific amino acid pathway that leads to the induction of ASNS transcription (Figure 5D). ChIP experiments were performed in MEF deficient in ATF2 and in the corresponding wild type cells with a primer set covering the AARE of the ASNS gene (Figure 5E). As expected, the binding of ATF4 to this AARE was increased in response to amino acid starvation in both cell types. Most importantly, no significant binding of ATF2 to the ASNS AARE was detected in wild type cells while, in contrast to the results obtained with the CHOP and ATF3 AAREs the lack of ATF2 did not affect the increase in histone H4 acetylation in the vicinity of the AARE following leucine deprivation. These findings show that an ATF2-independent HAT activity is involved in the amino acid regulation of ASNS transcription.

The role of ATF2 in histone acetylation can be extended to other amino acid-regulated genes and is linked to its binding to AARE sequences

We have previously shown that the requirement for ATF2 in amino acid-regulated gene expression varies according to the tested genes (4). Although loss of ATF2 entirely abolished the leucine control of most of the tested genes, the amino acid control of certain genes remains partially or completely insensitive to such loss. Based on these data, we have chosen to study two other known functional AARE sequences for their ability to bind ATF2 differentially.

To determine whether other amino-acid-regulated genes also require ATF2-mediated histone acetylation for their activation, we have tested the ATF3 gene. ATF3 has been shown to contain a functional AARE that is required to mediate the transcriptional activation in response to amino acid starvation (12). Sequence analysis revealed that the 9 bp core sequence of this element differs from that in the CHOP promoter by only one nucleotide (Figure 5A). Lack of ATF2 resulted in a loss of the ATF3 mRNA inducibility in response to leucine starvation (Figure 5B). ChIP assays were performed with a primer set covering the AARE of the ATF3 gene and using an anti-acetylated histone H4 antibody (Figure 5C). In wild type cells, the binding of ATF2 to the ATF3 AARE remained constitutive whereas ATF4 binding and histone H4 acetylation were increased following leucine deprivation. Lack of ATF2 resulted in a complete loss in the elevation of histone H4 acetylation. These data demonstrate that the role of ATF2 in histone H4 acetylation is not restricted to amino acid regulation of CHOP and can be extended to another amino acid-regulated gene.
Figure 5.Role of ATF2 in the transcriptional regulation of ATF3 and ASNS in response to amino acid starvation. (A) Sequence comparison of the human CHOP AARE (−313 to −298) with the human ATF3 AARE (−12 to −27) and the human ASNS AARE (−57 to −72). The minimum AARE core sequence is boxed. Identical nucleotides in the core are boxed in grey. ATF2 +/+ or ATF2 −/− MEF were incubated either in control (+leu) or leucine-free medium (−leu) and harvested after 4 h. Total RNA was extracted and analysed by real time RT-PCR for ATF3 (B) and ASNS (C) mRNA content as described in Materials and Methods. ChIP analysis was performed as described under Materials and Methods using antibodies specific for ATF4, ATF2 and acetylated histone H4 and two different sets of primers to amplify (D) ATF3 AARE or (E) ASNS AARE. Data were plotted as the percentage of antibody binding versus the amount of PCR product obtained using a standardized aliquot of input chromatin. Each point represents the mean value of three independent experiments, and the error bars represent the standard error of the means.

Role of ATF2 in the transcriptional regulation of ATF3 and ASNS in response to amino acid starvation. (A) Sequence comparison of the human CHOP AARE (−313 to −298) with the human ATF3 AARE (−12 to −27) and the human ASNS AARE (−57 to −72). The minimum AARE core sequence is boxed. Identical nucleotides in the core are boxed in grey. ATF2 +/+ or ATF2 −/− MEF were incubated either in control (+leu) or leucine-free medium (−leu) and harvested after 4 h. Total RNA was extracted and analysed by real time RT-PCR for ATF3 (B) and ASNS (C) mRNA content as described in Materials and Methods. ChIP analysis was performed as described under Materials and Methods using antibodies specific for ATF4, ATF2 and acetylated histone H4 and two different sets of primers to amplify (D) ATF3 AARE or (E) ASNS AARE. Data were plotted as the percentage of antibody binding versus the amount of PCR product obtained using a standardized aliquot of input chromatin. Each point represents the mean value of three independent experiments, and the error bars represent the standard error of the means.

To further investigate a link between the binding of ATF2 to AARE sequences and the increase in histone H4 acetylation, we have chosen to study the AARE sequences of ASNS which have been described to bind ATF2 poorly even though it shares nucleotide sequence similarities with the CHOP and ATF3 AAREs (23) (Figure 5A). According to these data, ATF2 is not essential in the specific amino acid pathway that leads to the induction of ASNS transcription (Figure 5D). ChIP experiments were performed in MEF deficient in ATF2 and in the corresponding wild type cells with a primer set covering the AARE of the ASNS gene (Figure 5E). As expected, the binding of ATF4 to this AARE was increased in response to amino acid starvation in both cell types. Most importantly, no significant binding of ATF2 to the ASNS AARE was detected in wild type cells while, in contrast to the results obtained with the CHOP and ATF3 AAREs the lack of ATF2 did not affect the increase in histone H4 acetylation in the vicinity of the AARE following leucine deprivation. These findings show that an ATF2-independent HAT activity is involved in the amino acid regulation of ASNS transcription.

DISCUSSIONCells respond to the stress of amino acid starvation by activating a gene expression program that either protects them from stress or leads to apoptosis (24,25). The regulation of CHOP gene expression represents a mechanistic model to investigate how changes in amino acid-regulated transcription are triggered and maintained. The data described in the present study show several novel observations regarding the mechanisms by which ATF2 activates CHOP transcription upon amino acid starvation. (1) We provide in vivo evidence that amino acid starvation induces a change in the chromatin environment in increasing the acetylation of histones H4 and H2B. (2) A time course analysis reveals that phosphorylation of ATF2 precedes acetylation of histone H4 and H2B tails, ATF4 binding and the increase in CHOP mRNA. (3) We demonstrate that ATF2 is essential for the acetylation of histones H4 and H2B within AARE sequences in response to amino acid starvation. (4) Our results also show that ATF4 binding and histone acetylation are two independent events that are required for the CHOP induction upon amino acid starvation.The data presented here, in addition to our previously published results (4) demonstrate the existence of a specific amino acid-regulated pathway leading to the phosphorylation of prebound ATF2 on the CHOP promoter. Three families of mitogen-activated protein kinases (MAPKs), namely extracellular signal-regulated kinase (ERK), c-Jun N-terminal kinase (JNK) and p38 (21,26–28) are known to phosphorylate and activate ATF2 on Thr-69 and Thr-71. Results from several laboratories indicate that ERK (29,30) and JNK1 (31) can be activated by amino acid starvation. However, the published kinetics of JNK1 activation cannot explain the rapid phosphorylation of ATF2 bound to the CHOP AARE. Therefore, the kinase involved in the phosphorylation of ATF2 in response to leucine starvation remains to be identified.In eukaryotic cells, a direct connection between acetylation of the N-terminal domains of histones and transcriptional activity has been established (32). The acetylation of histone tails is thought to facilitate transcription by altering accessibility of DNA to transcriptional activators or chromatin remodelling complexes (33,34). The present experiments demonstrate a significant increase in histone H4 and H2B acetylation at the CHOP promoter during amino acid deprivation. Acetylation of histone H4 on lysine 16 has been shown to modulate both the chromatin structure and functional interactions between proteins and the chromatin fibre (35). However, the role of histone H4 acetylation in the transcriptional control of CHOP by amino acids remains to be elucidated. In contrast, histone H3 remains hypoacetylated at the CHOP promoter in response to amino acid starvation. Similar findings about the differential regulation of histone acetylation were also reported for the induction of the stress-inductible gene Hsp70 by heat shock (36) and suggest that distinct mechanisms are responsible for the acetylation of H3 and H4 tails in the nucleosome.The acetylation of specific lysine residues on the N-terminal regions of the histone components of chromatin is directed by histone acetyltransferases (HAT) (37). A large number of studies led to the discovery of a large number of HAT enzymes, many of which such as p300/CBP (38), TAFII250 (39) or PCAF (40) were previously identified as transcriptional coactivators. Data presented here clearly demonstrate that ATF2 has a critical role in the acetylation of H4 and H2B at the CHOP promoter in response to amino acid starvation. We have previously demonstrated that an ATF2-dominant negative mutant (10) in which Thr69 and Thr71 cannot be phosphorylated inhibits the CHOP promoter activity enhanced by leucine starvation. Our present results show that phosphorylation of ATF2 on Thr71 within the CHOP AARE precedes histone acetylation and induction of CHOP mRNA. Therefore, these data reinforce the idea that phosphorylation of ATF2 has a key role in stimulating the HAT activity. It has been proposed that ATF2 may have an intrinsic HAT activity (41). ATF2 might also recruit HAT activity to the promoter and activate transcription. For example, ATF2 was shown to interact and cooperate with p300 to regulate the retinoic acid-mediated transcription of the c-jun gene in F9 cells (42). Therefore, the involvement of the HAT activity associated with p300 or with other transcriptional coactivators in the induction of CHOP transcription by amino acid starvation merit further investigation.We have previously shown that amino acid-regulated genes can be divided into two classes according to the degree of their ATF2-dependence in eukaryotic cells (4): (i) The first class is composed of genes which are totally ATF2-dependent such as CHOP, ATF3, SARS or 4EBP1; (ii) in the second class of genes, such as ASNS and YARS, the lack of ATF2 decreases but does not abolish the amino acid regulation. As described for the CHOP AARE, ATF2 binds in vivo specifically to the ATF3 AARE in starved and unstarved conditions and is essential for the acetylation of histones H4 in response to amino acid starvation. Therefore, we propose that the phosphorylation of ATF2, the stimulation of the associated HAT activity and the modification of the chromatin structure may be a general mechanism involved in the transcriptional regulation of the ATF2-dependent genes in response to amino acid starvation. By contrast, with AARE sequences that do not bind ATF2 (as shown in the ASNS promoter), it appears that the increase in histone H4 acetylation involves an ATF2-independent HAT mechanism. Chen et al. (3) have shown that the histone H3 acetylation is increased on the ASNS promoter in response to amino acid starvation. Here we show that this histone variant remains hypoacetylated at the CHOP promoter. Taken together, these data suggest that although most of the amino acid-responsive genes have AARE sites that are similar in sequence, distinct HATs could be involved in the acetylation of the different histone variants in response to amino acid starvation. Such differences in the mechanism of histone acetylation would permit flexibility between amino- acid-regulated genes with regard to the rapidity and the magnitude of the transcriptional response despite the same initial signal.It is well established that ATF4 is essential for the transcriptional activation of a number of mammalian genes and acts as a key regulator of the amino acid response pathway (1). Our results demonstrate that lack of ATF2 and hence subsequent histone acetylation do not affect the induction of ATF4 binding at the CHOP AARE in response to amino acid starvation. We therefore speculate that the modification of the chromatin structure could enhance the recruitment of coactivators or could inhibit the binding of repressors, leading to the increase in gene transcription. In addition, we show that in the CHOP promoter, ATF4 does not play a critical role in the histone acetylation during amino acid deprivation. Therefore, it is now clear that following amino acid limitation there is a highly coordinated time-dependent programme of molecular events leading to transcriptional activation. The identification of the sequence of events by which ATF2 phosphorylation and ATF4 can lead to the onset of gene transcription in response to amino acid starvation will be an important contribution to our understanding of nutrient regulation of gene expression in mammalian cells.

Cells respond to the stress of amino acid starvation by activating a gene expression program that either protects them from stress or leads to apoptosis (24,25). The regulation of CHOP gene expression represents a mechanistic model to investigate how changes in amino acid-regulated transcription are triggered and maintained. The data described in the present study show several novel observations regarding the mechanisms by which ATF2 activates CHOP transcription upon amino acid starvation. (1) We provide in vivo evidence that amino acid starvation induces a change in the chromatin environment in increasing the acetylation of histones H4 and H2B. (2) A time course analysis reveals that phosphorylation of ATF2 precedes acetylation of histone H4 and H2B tails, ATF4 binding and the increase in CHOP mRNA. (3) We demonstrate that ATF2 is essential for the acetylation of histones H4 and H2B within AARE sequences in response to amino acid starvation. (4) Our results also show that ATF4 binding and histone acetylation are two independent events that are required for the CHOP induction upon amino acid starvation.

The data presented here, in addition to our previously published results (4) demonstrate the existence of a specific amino acid-regulated pathway leading to the phosphorylation of prebound ATF2 on the CHOP promoter. Three families of mitogen-activated protein kinases (MAPKs), namely extracellular signal-regulated kinase (ERK), c-Jun N-terminal kinase (JNK) and p38 (21,26–28) are known to phosphorylate and activate ATF2 on Thr-69 and Thr-71. Results from several laboratories indicate that ERK (29,30) and JNK1 (31) can be activated by amino acid starvation. However, the published kinetics of JNK1 activation cannot explain the rapid phosphorylation of ATF2 bound to the CHOP AARE. Therefore, the kinase involved in the phosphorylation of ATF2 in response to leucine starvation remains to be identified.

In eukaryotic cells, a direct connection between acetylation of the N-terminal domains of histones and transcriptional activity has been established (32). The acetylation of histone tails is thought to facilitate transcription by altering accessibility of DNA to transcriptional activators or chromatin remodelling complexes (33,34). The present experiments demonstrate a significant increase in histone H4 and H2B acetylation at the CHOP promoter during amino acid deprivation. Acetylation of histone H4 on lysine 16 has been shown to modulate both the chromatin structure and functional interactions between proteins and the chromatin fibre (35). However, the role of histone H4 acetylation in the transcriptional control of CHOP by amino acids remains to be elucidated. In contrast, histone H3 remains hypoacetylated at the CHOP promoter in response to amino acid starvation. Similar findings about the differential regulation of histone acetylation were also reported for the induction of the stress-inductible gene Hsp70 by heat shock (36) and suggest that distinct mechanisms are responsible for the acetylation of H3 and H4 tails in the nucleosome.

The acetylation of specific lysine residues on the N-terminal regions of the histone components of chromatin is directed by histone acetyltransferases (HAT) (37). A large number of studies led to the discovery of a large number of HAT enzymes, many of which such as p300/CBP (38), TAFII250 (39) or PCAF (40) were previously identified as transcriptional coactivators. Data presented here clearly demonstrate that ATF2 has a critical role in the acetylation of H4 and H2B at the CHOP promoter in response to amino acid starvation. We have previously demonstrated that an ATF2-dominant negative mutant (10) in which Thr69 and Thr71 cannot be phosphorylated inhibits the CHOP promoter activity enhanced by leucine starvation. Our present results show that phosphorylation of ATF2 on Thr71 within the CHOP AARE precedes histone acetylation and induction of CHOP mRNA. Therefore, these data reinforce the idea that phosphorylation of ATF2 has a key role in stimulating the HAT activity. It has been proposed that ATF2 may have an intrinsic HAT activity (41). ATF2 might also recruit HAT activity to the promoter and activate transcription. For example, ATF2 was shown to interact and cooperate with p300 to regulate the retinoic acid-mediated transcription of the c-jun gene in F9 cells (42). Therefore, the involvement of the HAT activity associated with p300 or with other transcriptional coactivators in the induction of CHOP transcription by amino acid starvation merit further investigation.

We have previously shown that amino acid-regulated genes can be divided into two classes according to the degree of their ATF2-dependence in eukaryotic cells (4): (i) The first class is composed of genes which are totally ATF2-dependent such as CHOP, ATF3, SARS or 4EBP1; (ii) in the second class of genes, such as ASNS and YARS, the lack of ATF2 decreases but does not abolish the amino acid regulation. As described for the CHOP AARE, ATF2 binds in vivo specifically to the ATF3 AARE in starved and unstarved conditions and is essential for the acetylation of histones H4 in response to amino acid starvation. Therefore, we propose that the phosphorylation of ATF2, the stimulation of the associated HAT activity and the modification of the chromatin structure may be a general mechanism involved in the transcriptional regulation of the ATF2-dependent genes in response to amino acid starvation. By contrast, with AARE sequences that do not bind ATF2 (as shown in the ASNS promoter), it appears that the increase in histone H4 acetylation involves an ATF2-independent HAT mechanism. Chen et al. (3) have shown that the histone H3 acetylation is increased on the ASNS promoter in response to amino acid starvation. Here we show that this histone variant remains hypoacetylated at the CHOP promoter. Taken together, these data suggest that although most of the amino acid-responsive genes have AARE sites that are similar in sequence, distinct HATs could be involved in the acetylation of the different histone variants in response to amino acid starvation. Such differences in the mechanism of histone acetylation would permit flexibility between amino- acid-regulated genes with regard to the rapidity and the magnitude of the transcriptional response despite the same initial signal.

It is well established that ATF4 is essential for the transcriptional activation of a number of mammalian genes and acts as a key regulator of the amino acid response pathway (1). Our results demonstrate that lack of ATF2 and hence subsequent histone acetylation do not affect the induction of ATF4 binding at the CHOP AARE in response to amino acid starvation. We therefore speculate that the modification of the chromatin structure could enhance the recruitment of coactivators or could inhibit the binding of repressors, leading to the increase in gene transcription. In addition, we show that in the CHOP promoter, ATF4 does not play a critical role in the histone acetylation during amino acid deprivation. Therefore, it is now clear that following amino acid limitation there is a highly coordinated time-dependent programme of molecular events leading to transcriptional activation. The identification of the sequence of events by which ATF2 phosphorylation and ATF4 can lead to the onset of gene transcription in response to amino acid starvation will be an important contribution to our understanding of nutrient regulation of gene expression in mammalian cells.
