---
pmid: '28473536'
title: Impact of cytosine methylation on DNA binding specificities of human transcription
  factors.
authors:
- Yin Y
- Morgunova E
- Jolma A
- Kaasinen E
- Sahu B
- Khund-Sayeed S
- Das PK
- Kivioja T
- Dave K
- Zhong F
- Nitta KR
- Taipale M
- Popov A
- Ginno PA
- Domcke S
- Yan J
- Schübeler D
- Vinson C
- Taipale J
journal: Science
year: '2017'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC8009048
doi: 10.1126/science.aaj2239
---

# Impact of cytosine methylation on DNA binding specificities of human transcription factors.
**Authors:** Yin Y, Morgunova E, Jolma A, Kaasinen E, Sahu B, Khund-Sayeed S, Das PK, Kivioja T, Dave K, Zhong F, Nitta KR, Taipale M, Popov A, Ginno PA, Domcke S, Yan J, Schübeler D, Vinson C, Taipale J
**Journal:** Science (2017)
**DOI:** [10.1126/science.aaj2239](https://doi.org/10.1126/science.aaj2239)
**PMC:** [PMC8009048](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8009048/)

## Abstract

1. Science. 2017 May 5;356(6337):eaaj2239. doi: 10.1126/science.aaj2239.

Impact of cytosine methylation on DNA binding specificities of human 
transcription factors.

Yin Y(1), Morgunova E(1), Jolma A(1), Kaasinen E(1), Sahu B(2), Khund-Sayeed 
S(3), Das PK(2), Kivioja T(2), Dave K(1), Zhong F(1), Nitta KR(1), Taipale M(1), 
Popov A(4), Ginno PA(5), Domcke S(5)(6), Yan J(1), Schübeler D(5)(6), Vinson 
C(3), Taipale J(7)(2).

Author information:
(1)Division of Functional Genomics and Systems Biology, Department of Medical 
Biochemistry and Biophysics, Karolinska Institutet, SE 141 83 Stockholm, Sweden.
(2)Genome-Scale Biology Program, Post Office Box 63, FI-00014 University of 
Helsinki, Helsinki, Finland.
(3)Laboratory of Metabolism, National Cancer Institute, National Institutes of 
Health, Room 3128, Building 37, Bethesda, MD 20892, USA.
(4)European Synchrotron Radiation Facility, 38043 Grenoble, France.
(5)Friedrich-Miescher-Institute for Biomedical Research (FMI), Maulbeerstrasse 
66, 4058 Basel, Switzerland.
(6)Faculty of Science, University of Basel, Petersplatz 1, 4003 Basel, 
Switzerland.
(7)Division of Functional Genomics and Systems Biology, Department of Medical 
Biochemistry and Biophysics, Karolinska Institutet, SE 141 83 Stockholm, Sweden. 
jussi.taipale@ki.se.

Comment in
    Science. 2017 May 5;356(6337):489-490. doi: 10.1126/science.aan2927.

The majority of CpG dinucleotides in the human genome are methylated at cytosine 
bases. However, active gene regulatory elements are generally hypomethylated 
relative to their flanking regions, and the binding of some transcription 
factors (TFs) is diminished by methylation of their target sequences. By 
analysis of 542 human TFs with methylation-sensitive SELEX (systematic evolution 
of ligands by exponential enrichment), we found that there are also many TFs 
that prefer CpG-methylated sequences. Most of these are in the extended 
homeodomain family. Structural analysis showed that homeodomain specificity for 
methylcytosine depends on direct hydrophobic interactions with the 
methylcytosine 5-methyl group. This study provides a systematic examination of 
the effect of an epigenetic DNA modification on human TF binding specificity and 
reveals that many developmentally important proteins display preference for 
mCpG-containing sequences.

Copyright © 2017, American Association for the Advancement of Science.

DOI: 10.1126/science.aaj2239
PMCID: PMC8009048
PMID: 28473536 [Indexed for MEDLINE]

## Full Text

Abstract

The majority of CpG dinucleotides in the human genome are methylated at cytosine bases. However, active gene regulatory elements are generally hypomethylated relative to their flanking regions, and the binding of some transcription factors (TFs) is diminished by methylation of their target sequences. By analysis of 542 human TFs with methylation-sensitive SELEX (systematic evolution of ligands by exponential enrichment), we found that there are also many TFs that prefer CpG-methylated sequences. Most of these are in the extended homeodomain family. Structural analysis showed that homeodomain specificity for methylcytosine depends on direct hydrophobic interactions with the methylcytosine 5-methyl group. This study provides a systematic examination of the effect of an epigenetic DNA modification on human TF binding specificity and reveals that many developmentally important proteins display preference for mCpG-containing sequences.

Conclusion

Biochemical studies and experiments using model organisms, cell lines, and in vitro reconstituted systems have resulted in the discovery of the principal mechanisms that control gene expression. The language by which the genome imparts when and where genes are expressed is thus understood at a conceptual level. However, reading the genomic instructions also requires knowledge of binding specificities of all TFs, which constitute the words of the gene regulatory language. In this work, we performed systematic analysis of DNA binding specificities of full-length TFs and eDBDs using unmethylated and CpG-methylated DNA ligands. Compared with our earlier work ( 26 ), this analysis used an extended clone collection, and it resulted in identification of binding motifs for 222 TFs for which a HT-SELEX motif was not available. Moreover, an improved computational pipeline ( 25 ) allowed us to identify secondary binding profiles for 57 TFs. Taken together, the full data set containing 596 previously unknown motifs ( table S2 ) substantially expands the known lexicon of TF binding specificities.

The results also shed light on the mechanisms of diversification of TF binding specificity during evolution. We found that specific members of three different structural TF families used an AT-hook domain to recognize an AT-rich sequence that flanked the motifs recognized by the TF and its paralogs. This mechanism of diversification of TF specificity is similar to one reported previously ( 26 ), whereby an addition of an arginine residue close to the DBD results in preference for an AAAA or TTTT sequence adjacent to the core binding motif. As a whole, these results suggest that TF specificity commonly evolves through addition of simple amino acid features that diversify the sequences recognized by paralogous proteins.

Our work shows that the effect of the most prevalent human epigenetic DNA modification, CpG methylation, on TF binding specificity is more widespread than previously appreciated. In contrast to most previously reported cases [for example, ( 9 , 10 , 12 , 17 , 18 )], we found that methylation can affect TF binding positively in many cases [see also ( 16 , 62 )]. For example, earlier work has found that only 4% of plant TFs prefer cytosine-methylated sites. This difference from our results is explained by four mechanisms. First, some variance is caused by the fact that the sets of TFs for which data have been obtained in this and previous studies are not complete and do not represent a random sample of the entire TF compendium. Second, all the major families of TFs that prefer mCpG in humans are absent in plants, including canonical homeodomain, POU, and NFAT proteins. Third, plants methylate repeat elements but rarely regulatory sequences. Fourth, non-CpG methylation is very common in plants, and many TFs that O’Malley et al . ( 63 ) studied are affected by this modification. We find here that most TFs can bind to a site whose methylation alters the binding affinity of the TF. However, most of these TFs can also bind to sites that do not contain a CpG dinucleotide. Thus, two regulatory elements responding to the same factor can differ in their sensitivity to DNA methylation. In addition, some DNA sequences, such as some motifs containing a CGGAA subsequence, can be bound by TFs that are either negatively (ETS) or positively (NFAT) influenced by methylation. Thus, methylation can act to select the factor that binds to a target sequence. Such a selective effect can have a major impact on transcription, particularly in cases where one factor acts as an activator and the other as a repressor.

Our structural analysis of homeodomains reveals that mCpG is recognized through direct hydrophobic interactions between amino acids and the 5-methyl group of the methylcytosine. Given the structural similarity of thymine and methylcytosine and the identical position of their 5-methyl groups, it is difficult to determine why the preference of TFs for mCpG has evolved. It could represent a simple consequence of high-affinity recognition of thymine. However, several findings suggest that this may not be the whole explanation. For example, all canonical homeodomains prefer similar core sequences containing an AATT sequence. However, their preference for mCpG varies as a function of the hydrophobicity of amino acid residues at three positions. In addition, the strongest preference for mCpG is observed in the posterior homeodomain proteins, a family that has expanded in the vertebrate lineage, where genome-wide DNA methylation is obligatory. Regardless of how the specificity for mCpG has evolved, our results show that it influences binding of TFs to DNA both in vitro and in vivo, leading to changes in the occupancy and activity of TFs based on methylation of their recognition motifs.

In this work, we identified many developmentally important TFs from several TF families that prefer to bind to mCpG, and we also determined the molecular mechanism behind the differential preference of homeodomain proteins for mCpG. TFs that can bind to methylated sequences may be particularly important for reprogramming of cells because CpG methylation functions as a barrier for cellular reprogramming ( 14 , 15 ). In this regard, it is of particular interest that several factors that regulate ES cell self-renewal—including PRDM4, Nanog, and POU homeodomain factor POU5F1 (OCT4)—are capable of binding to mCpG sites. This may explain in part the ability of POU5F1 to reprogram differentiated cells toward a pluripotent fate ( 48 ). Our finding that many TFs prefer methylated CpGs, together with the genome-scale resource on TF binding specificities that we have generated, will be important for future analyses of epigenetic inheritance and transcriptional regulation.
