---
pmid: '25486861'
title: PERP, a host tetraspanning membrane protein, is required for Salmonella-induced
  inflammation.
authors:
- Hallstrom KN
- Srikanth CV
- Agbor TA
- Dumont CM
- Peters KN
- Paraoan L
- Casanova JE
- Boll EJ
- McCormick BA
journal: Cell Microbiol
year: '2015'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC4915744
doi: 10.1111/cmi.12406
---

# PERP, a host tetraspanning membrane protein, is required for Salmonella-induced inflammation.
**Authors:** Hallstrom KN, Srikanth CV, Agbor TA, Dumont CM, Peters KN, Paraoan L, Casanova JE, Boll EJ, McCormick BA
**Journal:** Cell Microbiol (2015)
**DOI:** [10.1111/cmi.12406](https://doi.org/10.1111/cmi.12406)
**PMC:** [PMC4915744](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4915744/)

## Abstract

1. Cell Microbiol. 2015 Jun;17(6):843-59. doi: 10.1111/cmi.12406. Epub 2015 Jan
24.

PERP, a host tetraspanning membrane protein, is required for Salmonella-induced 
inflammation.

Hallstrom KN(1), Srikanth CV(1), Agbor TA(1), Dumont CM(1), Peters KN(1), 
Paraoan L(2), Casanova JE(3), Boll EJ(1), McCormick BA(1).

Author information:
(1)Department of Microbiology and Physiological Systems, University of 
Massachusetts Medical School, Worcester, MA, USA.
(2)Eye and Vision Science Institute of Ageing and Chronic Disease, Faculty of 
Health and Life Sciences, University of Liverpool, Liverpool, UK.
(3)Department of Cell Biology, University of Virginia Health System, 
Charlottesville, VA, USA.

Salmonella enterica Typhimurium induces intestinal inflammation through the 
activity of type III secreted effector (T3SE) proteins. Our prior results 
indicate that the secretion of the T3SE SipA and the ability of SipA to induce 
epithelial cell responses that lead to induction of polymorphonuclear 
transepithelial migration are not coupled to its direct delivery into epithelial 
cells from Salmonella. We therefore tested the hypothesis that SipA interacts 
with a membrane protein located at the apical surface of intestinal epithelial 
cells. Employing a split ubiquitin yeast-two-hybrid screen, we identified the 
tetraspanning membrane protein, p53 effector related to PMP-22 (PERP), as a SipA 
binding partner. SipA and PERP appear to have intersecting activities as we 
found PERP to be involved in proinflammatory pathways shown to be regulated by 
SipA. In sum, our studies reveal a critical role for PERP in the pathogenesis of 
S. Typhimurium, and for the first time demonstrate that SipA, a T3SE protein, 
can engage a host protein at the epithelial surface.

© 2014 The Authors. Cellular Microbiology published by John Wiley & Sons Ltd.

DOI: 10.1111/cmi.12406
PMCID: PMC4915744
PMID: 25486861 [Indexed for MEDLINE]

## Full Text

Introduction
Salmonella enterica serovar Typhimurium is one of several S. enterica strains responsible for over 1 million cases of salmonellosis in the United States each year. The pathological hallmark of Salmonella‐elicited enteritis is extensive intestinal inflammation characterized by a substantial polymorphonuclear leukocyte (PMN) infiltrate to the site of infection. Although PMNs are integral to innate immunity, poorly controlled intestinal inflammation results in extensive tissue destruction and, in some instances, the formation of crypt abscesses. Such PMN recruitment is coordinated by the epithelial release of an array of proinflammatory cytokines, among which are two potent PMN chemoattractants: interleukin‐8 (IL‐8) and hepoxilin A3 (HXA3). IL‐8 is secreted basolaterally by epithelial cells in response not only to the bacterial product flagellin but also to a host of Salmonella type III secretion system (T3SS) effectors (e.g. SopE, SopB) that increase IL‐8 gene expression via nuclear factor kappa B (Hobbie et al., 1997; Hardt et al., 1998). The basolateral secretion of IL‐8 establishes a stable haptotactic gradient across the lamina propria. This gradient serves to guide PMNs from the lamina propria to the subepithelium but does not induce movement across the epithelium as observed in both model epithelia (McCormick et al., 1993; 1995) and a double transgenic mouse model with the ability to induce the expression of human IL‐8 (Kucharzik et al., 2005).Using an in vitro model of S. Typhimurium infection of human intestinal epithelial cells to study such inflammatory events occurring at the intestinal mucosa, we determined that subsequent PMN transit through the epithelial monolayer to the lumenal surface (defined as PMN transepithelial migration) is directed by the eicosanoid HXA3 (McCormick et al., 1998; Mrsny et al., 2004). HXA3 is a potent PMN chemoattractant that is secreted apically in response to the Salmonella T3SS effector protein SipA (McCormick et al., 1998; Lee et al., 2000; Silva et al., 2004). The key role that SipA plays in inducing epithelial responses resulting in the transepithelial migration of PMNs has also been substantiated using two distinct in vivo models of Salmonella‐induced enteritis (Zhang et al., 2002; Barthel et al., 2003; Wall et al., 2007). To date, the molecular mechanism underlying these cellular events has revealed that SipA activates a novel adenosine diphosphate ribosylation factor (ARF) 6‐ and phospholipase D‐dependent lipid signalling cascade (Criss et al., 2001) that in turn activates protein kinase C‐α and 12‐lipoxygenase (Lee et al., 2000; Mumy et al., 2008), events that ultimately lead to apical efflux of HXA3 (Mrsny et al., 2004; Mumy et al., 2008; Pazos et al., 2008). HXA3 is an arachidonic acid‐derived hydroxy epoxide that forms a chemotactic gradient across the epithelial tight junctional complex, which directs PMNs across the intestinal epithelium to the lumenal surface (Mrsny et al., 2004), the final step in PMN recruitment to the mucosal lumen.Although such studies have informed us of the nature of the signal transduction pathways induced by SipA that prompt PMN transepithelial migration, the way in which SipA initiates this complex cellular network remains undefined. Through both biochemical and genetic assessment, we have previously determined that host cellular translocation is not necessary for SipA to elicit inflammation (Lee et al., 2000) but that interaction of SipA at the apical surface of intestinal epithelial cells is sufficient to initiate the cellular events that lead to PMN transepithelial migration. Based on these observations, we hypothesize that SipA need not enter the epithelial cell cytosol to stimulate proinflammatory signal transduction pathways but rather may function extracellularly at the epithelial cell surface (Wall et al., 2007; Srikanth et al., 2010). This hypothesis is also consistent with the bi‐functional properties of SipA, which promotes gastroenteritis via two distinct functional domains that activate not only inflammation but also mechanisms of bacterial entry by exploiting discreet extracellular and intracellular locations respectively (Zhou et al., 1999; Higashide et al., 2002; Lilic et al., 2003; Wall et al., 2007).To test the hypothesis that SipA engages an apical surface receptor that triggers the induction of PMN transepithelial migration, we used a yeast‐two‐hybrid (Y2H) strategy to screen a human colonic cDNA library and identified the tetraspanning membrane protein p53 effector related to PMP‐22 (PERP) as a SipA binding partner. PERP was first identified as a p53 effector (Attardi et al., 2000) but has since been shown to play a role in development (Ihrie et al., 2005), caspase activation (Davies et al., 2009; Singaravelu et al., 2009), inflammation and cancer (Paraoan et al., 2006; Beaudry et al., 2010). To our knowledge, this is the first report that a T3SS effector protein engages an extracellular membrane binding partner. Herein, we describe the unappreciated role of PERP in promoting the SipA‐dependent inflammatory response to S. Typhimurium infection.

Introduction

Salmonella enterica serovar Typhimurium is one of several S. enterica strains responsible for over 1 million cases of salmonellosis in the United States each year. The pathological hallmark of Salmonella‐elicited enteritis is extensive intestinal inflammation characterized by a substantial polymorphonuclear leukocyte (PMN) infiltrate to the site of infection. Although PMNs are integral to innate immunity, poorly controlled intestinal inflammation results in extensive tissue destruction and, in some instances, the formation of crypt abscesses. Such PMN recruitment is coordinated by the epithelial release of an array of proinflammatory cytokines, among which are two potent PMN chemoattractants: interleukin‐8 (IL‐8) and hepoxilin A3 (HXA3). IL‐8 is secreted basolaterally by epithelial cells in response not only to the bacterial product flagellin but also to a host of Salmonella type III secretion system (T3SS) effectors (e.g. SopE, SopB) that increase IL‐8 gene expression via nuclear factor kappa B (Hobbie et al., 1997; Hardt et al., 1998). The basolateral secretion of IL‐8 establishes a stable haptotactic gradient across the lamina propria. This gradient serves to guide PMNs from the lamina propria to the subepithelium but does not induce movement across the epithelium as observed in both model epithelia (McCormick et al., 1993; 1995) and a double transgenic mouse model with the ability to induce the expression of human IL‐8 (Kucharzik et al., 2005).

Using an in vitro model of S. Typhimurium infection of human intestinal epithelial cells to study such inflammatory events occurring at the intestinal mucosa, we determined that subsequent PMN transit through the epithelial monolayer to the lumenal surface (defined as PMN transepithelial migration) is directed by the eicosanoid HXA3 (McCormick et al., 1998; Mrsny et al., 2004). HXA3 is a potent PMN chemoattractant that is secreted apically in response to the Salmonella T3SS effector protein SipA (McCormick et al., 1998; Lee et al., 2000; Silva et al., 2004). The key role that SipA plays in inducing epithelial responses resulting in the transepithelial migration of PMNs has also been substantiated using two distinct in vivo models of Salmonella‐induced enteritis (Zhang et al., 2002; Barthel et al., 2003; Wall et al., 2007). To date, the molecular mechanism underlying these cellular events has revealed that SipA activates a novel adenosine diphosphate ribosylation factor (ARF) 6‐ and phospholipase D‐dependent lipid signalling cascade (Criss et al., 2001) that in turn activates protein kinase C‐α and 12‐lipoxygenase (Lee et al., 2000; Mumy et al., 2008), events that ultimately lead to apical efflux of HXA3 (Mrsny et al., 2004; Mumy et al., 2008; Pazos et al., 2008). HXA3 is an arachidonic acid‐derived hydroxy epoxide that forms a chemotactic gradient across the epithelial tight junctional complex, which directs PMNs across the intestinal epithelium to the lumenal surface (Mrsny et al., 2004), the final step in PMN recruitment to the mucosal lumen.

Although such studies have informed us of the nature of the signal transduction pathways induced by SipA that prompt PMN transepithelial migration, the way in which SipA initiates this complex cellular network remains undefined. Through both biochemical and genetic assessment, we have previously determined that host cellular translocation is not necessary for SipA to elicit inflammation (Lee et al., 2000) but that interaction of SipA at the apical surface of intestinal epithelial cells is sufficient to initiate the cellular events that lead to PMN transepithelial migration. Based on these observations, we hypothesize that SipA need not enter the epithelial cell cytosol to stimulate proinflammatory signal transduction pathways but rather may function extracellularly at the epithelial cell surface (Wall et al., 2007; Srikanth et al., 2010). This hypothesis is also consistent with the bi‐functional properties of SipA, which promotes gastroenteritis via two distinct functional domains that activate not only inflammation but also mechanisms of bacterial entry by exploiting discreet extracellular and intracellular locations respectively (Zhou et al., 1999; Higashide et al., 2002; Lilic et al., 2003; Wall et al., 2007).

To test the hypothesis that SipA engages an apical surface receptor that triggers the induction of PMN transepithelial migration, we used a yeast‐two‐hybrid (Y2H) strategy to screen a human colonic cDNA library and identified the tetraspanning membrane protein p53 effector related to PMP‐22 (PERP) as a SipA binding partner. PERP was first identified as a p53 effector (Attardi et al., 2000) but has since been shown to play a role in development (Ihrie et al., 2005), caspase activation (Davies et al., 2009; Singaravelu et al., 2009), inflammation and cancer (Paraoan et al., 2006; Beaudry et al., 2010). To our knowledge, this is the first report that a T3SS effector protein engages an extracellular membrane binding partner. Herein, we describe the unappreciated role of PERP in promoting the SipA‐dependent inflammatory response to S. Typhimurium infection.

Results
PERP is a binding partner of SipA
Previous studies have identified the S. Typhimurium effector, SipA, as an important mediator of the immune inflammatory response that results in PMN influx. The fact that our prior studies found purified SipA to directly activate this response has prompted us to consider whether SipA might engage a surface receptor (Lee et al., 2000). Because we infer that this putative receptor represents the initiation site of the transcellular PMN signalling cascade, identification of a functional receptor will be crucial for understanding SipA's involvement in controlling intestinal inflammation. We used a split ubiquitin‐based Y2H (protein–protein interaction) analysis system (Dualsystems Biotech) (Stagljar et al., 1998; Dirnberger et al., 2008), with full‐length SipA as bait and a human colonic cDNA‐based library as prey. Approximately 4 × 106 transformants were screened and selected based on two growth reporters. Candidate interacting partners were then selected using a Lac‐Z based colorimetric reporter assay. The screen yielded seven positive clones out of which PERP was represented three times (Table 1).Table 1
SipA interacting partner candidatesSipA interacting partner candidates identified via Y2H screenCloneGene nameFunctionReference1.SERP1ER stress responseYamaguchi et al., 1999
2.DERP2Cell death regulationOka et al., 2008
3.TMEM87Unknown4.TMEM147Interacts with nicalin–NOMO complexDettmer et al., 2010

5.

PERP
a

p53 effector, regulates caspase‐3 activation
Attardi et al., 2000; Davies et al., 2009
aMultiple hits.Five potential SipA‐binding candidates were identified from our Y2H screen. Most candidates have been identified as membrane proteins with various functions pertaining to cell stress and death regulation. Out of these candidates, PERP was the only one to be pulled out multiple times from our screen.2014 John Wiley & Sons Ltd.Using the Lac‐Z reporter assay, we confirmed the PERP–SipA interaction in a ‘reverse’ Y2H assay in which SipA was sub‐cloned into the prey vector, and PERP of the initial screen was used as bait. Furthermore, we used a biochemical approach to demonstrate PERP–SipA interactions. Model human intestinal epithelial cells (HCT8) were infected with a wild‐type S. Typhimurium strain expressing an HA‐tagged SipA protein (AJK63). Immunoprecipitation of infected cell lysates with an anti‐HA antibody specifically resulted in the pull down of PERP, as immunoprecipitation with a control IgG antibody under similar conditions yielded neither HA‐SipA nor PERP (Fig. 1A). We also performed the pull down with another Salmonella T3SS effector, SifA, also tagged with HA to ensure our observation was not due to non‐specific recognition of the HA‐tag by the PERP antibody (Fig. 1B).Figure 1
SipA and PERP are binding partners.A. T84 lysates infected with an HA‐tagged SipA containing strain of S
almonella were pulled down with IgG or anti‐HA antibody, and then probed for PERP. Only the HA‐SipA pull down samples resulted in a PERP band.B. The specificity of the PERP–SipA interaction was confirmed by a pull down of HA‐tagged SipA and HA‐tagged SifA. Cells were infected with S
almonella expressing HA‐tagged SipA or HA‐tagged SifA, or left non‐infected as a non‐specific control [(−) control] and lysed. Lysates were pulled down with anti‐HA antibody and probed for PERP. The SipA‐HA lysates resulted in a PERP band, whereas the SifA‐HA lysates resulted only in a faint band of background intensity.C. Passage of HCT8 lysates through glutathione beads conjugated to the GST‐tagged C‐terminus of SipC (‘conjugated’) resulted in the specific pull down of PERP (‘eluted’), while passage of the lysates through non‐conjugated beads fails to result in the pull down of PERP.figure2014 John Wiley & Sons Ltd.Because these data support our contention that PERP is a SipA binding partner, we next examined the specificity of the PERP–SipA interaction by testing whether PERP binds to the Salmonella protein SipC, a component of the T3SS1 translocon. SipC is not only required for the translocation of Salmonella effectors into the host cell (Collazo and Galan, 1997) and for Salmonella invasion (Myeni and Zhou, 2010) but also SipC and SipA are known to have cooperative roles during invasion (McGhie et al., 2001). As shown in Fig. 1C, passage of HCT8 lysates across beads bound to the GST‐labeled C‐terminus of SipC (Nichols and Casanova, 2010) resulted in the specific pull down of PERP, suggesting that PERP is able to interact with two Salmonella proteins that function during early stages of Salmonella pathogenesis and that PERP may have a role mediating these events. However, the precise mechanism(s) remain unknown.Functional consequences of PERP in the promotion of the inflammatory response to Salmonella infectionPERP is a tetraspanning membrane protein that belongs to the PMP‐22(Gas3)/epithelial membrane protein (EMP) family (Attardi et al., 2000), which includes PMP‐22 and EMP 1, 2 and 3. Detection of PERP as an interacting partner with SipA piqued our interest given that PERP has been documented to impact inflammatory signalling pathways (Beaudry et al., 2010) as well as to regulate the activation of caspase‐3 (Davies et al., 2009; Singaravelu et al., 2009). Because we have shown the Salmonella effector SipA induces inflammatory pathways that lead to the recruitment of PMNs to the site of infection, we sought to determine the extent to which PERP might also be involved in governing these processes during infection with Salmonella using our in vitro PMN migration assay (Experimental procedures). Following infection, polarized intestinal cell monolayers were exposed to 25 μg ml−1 of anti‐PERP antibody, anti‐MTCO‐1 antibody (mitochondrial marker – used as an irrelevant isotype control) or IgG isotype control antibody prior to adding freshly isolated human peripheral blood PMNs. As shown in Fig. 2A, the presence of anti‐PERP antibody decreased the ability of Salmonella to induce PMN transepithelial migration by 90%. This result was specific to exposure with the PERP antibody, as the MTCO‐1 and IgG‐treated cells did not similarly inhibit Salmonella‐induced PMN transmigration. Addition of the PERP antibody in the absence of infection has no impact on PMN transmigration (Fig. S1).Figure 2
PERP promotes the inflammatory response to S
almonella infection.A. Polarized HCT8 cells were infected with wild‐type S. Typhimurium for 40 minutes and then exposed to 25 μg ml−1 of PERP, MTCO‐1 (mitochondrial marker, control) or IgG (control) antibodies at the basolateral surface or left in HBSS + buffer prior to addition of PMNs at the basolateral surface. Values are expressed as percent PMN migration compared with results from infected cells that were not treated with antibodies.B. Lines of stable PERP knockdown cells and control cells were generated via transfection of PERP siRNA (inset). The PERP knockdown cells showed no defect in barrier function compared with the vector control cells (data not shown). Knocking down PERP resulted in a statistically significant 40% reduction in PMN migration in response to infection compared with vector control cells. Error bars represent ± standard errors and P‐values less than 0.05 according to Student's t‐test were considered statistically significant.figure2014 John Wiley & Sons Ltd.As a complementary approach, we performed PMN transepithelial migration assays using PERP siRNA knockdown cells (p11) paired with a siRNA vector control (p24). Comparable with the PERP antibody blocking studies, PMN transepithelial migration across the PERP knockdown cells in response to Salmonella infection, where HXA3 is the major PMN chemoattractant gradient induced, was reduced by 40% as compared with the vector control cells (Fig. 2B). Although these studies suggest that PERP is involved in facilitating PMN transmigration in response to Salmonella infection, PERP might also play a role in other intestinal inflammatory conditions beyond that of Salmonella infection where PMN migration is a key pathological feature. We modeled such conditions in vitro via addition of formyl‐methionyl‐leucyl‐phenylalanine (fMLP), a PMN chemoattractant, to our polarized monolayers in the absence of infection. As shown in Fig. 3, PMN transepithelial migration in response to an imposed gradient of fMLP across cells treated with the PERP blocking antibody (Fig. 3A), or across the PERP knockdown cells (Fig. 3B) was reduced approximately 90% and 35% respectively. We also probed the function of PERP during PMN transmigration in response to chemoattractants other than HXA3 known to be secreted by intestinal epithelial cells, such as IL‐8 and leukotriene B4 (LTB4). We found that blocking PERP by pretreating HCT8 cell monolayers with 25 μg ml−1 of PERP antibody for 30 min prior to inducing imposed gradients of IL‐8 or LTB4 to the apical surface (see Experimental Procedures for details) resulted in a modest, although statistically significant inhibitory impact on IL‐8‐induced migration (Fig. 3C), but not on LTB4‐induced migration (Fig. 3D). Together, these results indicate PERP has a broad, although not universal, role in regulating PMN migration.Figure 3
PERP promotes PMN migration. Migration was stimulated by the addition of fMLP, IL‐8 or LTB4. Presence of the PERP antibody (A) specifically reduced fMLP‐induced PMN migration by about 90%. Similarly, migration across PERP knockdown monolayers (B) was reduced by about 35%. Presence of the PERP antibody also reduced IL‐8‐induced migration by about 20% (C), although had no impact on migration induced by LTB4 (D). Values are expressed as percent PMN migration normalized to chemoattractant treatment alone samples ± standard errors. P‐values less than 0.05 according to Student's t‐test were considered statistically significant.figure2014 John Wiley & Sons Ltd.The S. Typhimurium effector protein, SipA, promotes gastroenteritis via two distinct functional motifs that trigger not only inflammation but also mechanisms of bacterial entry (Wall et al., 2007). Moreover, we also recently found that during infection of intestinal epithelial cells, SipA is responsible for the early activation of caspase‐3 (Srikanth et al., 2010). This enzyme is essential for SipA cleavage at a specific recognition motif, dividing the protein into its two functional domains (Srikanth et al., 2010). Such studies further revealed that cleavage of the SipA caspase‐3 motif is central for promoting proinflammatory responses, and therefore we infer the involvement of caspase‐3 during pathogenesis given that Salmonella is less virulent in caspase‐3 knockout (caspase‐3−/−) mice (Srikanth et al., 2010). Because prior studies have reported that increased levels of PERP lead to caspase‐3 activation (Davies et al., 2009), we next sought to determine the extent to which PERP plays a role in Salmonella‐induced activation of caspase‐3.Using a colorimetric caspase‐3 bioactivity assay kit, we evaluated the extent to which PERP regulates caspase‐3 activation in an HCT8 line of transient PERP knockdown cells (Fig. 4A) in the absence and presence of Salmonella infection. We observed an increase of 86.7% ± 5.3 (standard error, P < 0.05) in the level of Salmonella‐induced activated caspase‐3 in the vector control cells compared with only a 57.8% ± 4.5% (standard error) increase in Salmonella‐induced activated caspase‐3 in the PERP knockdown cells (Fig. 4B). Because the partial knockdown of PERP resulted in about a 30% decrease in the ability to induce caspase‐3, these results suggest that PERP may be necessary but not sufficient for caspase‐3 activation during Salmonella infection.Figure 4
PERP promotes caspase‐3 activity during S
almonella infection.A. PERP was transiently knocked down in HCT8 cells. A no‐targeting control was used to confirm specificity. By this method, PERP was reproducibly knocked down by about 50%.B. Levels of activated caspase‐3 in response to S
almonella infection were reduced by about 30% in the PERP knockdown cells. Numbers are expressed as percent of activated caspase‐3 relative to activated caspase‐3 levels in non‐infected, vector control cells. Error bars show ± standard error. P‐values less than 0.05 according to Student's t‐test were considered statistically significant.figure2014 John Wiley & Sons Ltd.
PERP accumulates at the apical surface in a SipA‐dependent mannerThus far, our observations show that during infection with Salmonella, PERP not only plays a crucial role in governing PMN recruitment but is also involved with the activation of caspase‐3. PERP, as a tetraspanning protein, performs a wide range of functions and confers cell‐specific and tissue‐specific roles. For example, PERP has been shown to localize to desmosomes in mouse newborn skin (Ihrie et al., 2005). More recently, PERP was shown to localize to peri‐ and interdesmosomal regions termed ‘tessellate junctions’ in stratified epithelia as well as to desmosomes in bovine intestinal epithelium (Franke et al., 2013). Consistent with this, we also observed PERP expression on the mucosal surface of mouse proximal colon tissue (Fig. S2).Because aberrant localization of transmembrane proteins is linked to numerous human diseases, we examined whether PERP is redistributed during infection with Salmonella. The apical surface of polarized cell monolayers was selectively biotinylated following infection with wild‐type Salmonella or mock infection with buffer. This method permits us to identify changes in protein expression specifically at the apical surface in response to Salmonella infection. As shown in Fig. 5A, PERP accumulates at the apical surface of polarized intestinal epithelial cells in response to wild‐type infection. Moreover, the involvement of SipA in the accumulation of PERP at the apical surface is evidenced by infection with the isogenic SipA‐deficient strain, which results in significantly less PERP accumulation at this location compared with wild‐type infection (Fig. 5A). By contrast, infection with a SipA‐complemented strain correlated with greater PERP accumulation at the apical surface as compared with the wild‐type strain (Fig. 5B). These results indicate SipA is necessary, although not sufficient, to induce PERP accumulation at the apical surface during Salmonella infection. The accumulation of PERP at the apical surface also appears to be a directed cellular event since a similar assessment of PERP distribution to the basolateral surface resulted in only a modest increase (Fig. S3A).Figure 5
PERP accumulates at the apical cell surface in a SipA‐dependent manner.A–C. Polarized HCT8 monolayers were infected with wild‐type (WT), SipA‐deficient (ΔSipA), SipA‐complemented S
almonella (SipA +) or SipB‐deficient (ΔSipB), or left uninfected (−) in HBSS + for 1 h, and the apical cell surfaces were biotinylated, pulled down with streptavidin and Western blotted for PERP.D. Whole cell lysates from non‐infected and WT‐infected HCT8s were probed for overall PERP expression. Glyceraldehyde 3‐phosphate dehydrogenase (GAPDH) serves as a loading control. Densitometry confirms a significant, although minor, increase in PERP expression in response to wild‐type infection.E. Cells were treated with 150 μM of brefeldin A (BFA) for 1 h or left untreated in HBSS + for 1 h prior to infection. PERP expression at the apical surface was examined as explained for (A–C).F. Cells were treated with 2 μg ml−1 of cycloheximide (cyc) for 1 h or left untreated in HBSS + for 1 h prior to infection. PERP expression at the apical surface was examined as explained for (A–C). Although it is noted that the basal level of PERP in (E) is comparatively higher than the basal level of PERP in (A–C), we interpret this difference as normal variation seen when using different stocks of cultured cell lines. Regardless of this observed difference, we are able to consistently reproduce results showing a function for PERP during S
almonella pathogenesis. Densitometry analyses show the change in PERP expression induced by infection with mutant or complemented strains compared with the change in PERP expression induced by WT‐infected cells after normalizing to non‐infected values. Error bars show ± standard error. P‐values less than 0.05 according to Student's t‐test were considered statistically significant.figure2014 John Wiley & Sons Ltd.Our previous results demonstrated that purified S. Typhimurium SipA protein could trigger the PMN migration response in the absence of the type III secretion and translocation factors, such as SipB and SipC, suggesting that this effector may not need to be translocated to initiate the events that lead to PMN transepithelial migration (Lee et al., 2000). To examine whether PERP also accumulates at the apical surface in the absence of translocated SipA, we took a genetic approach using an isogenic ΔsipB non‐polar deletion mutant. This strain expresses native SipA from the chromosomal sipA locus and is capable of secreting effector proteins but cannot translocate them into the host cell cytosol (Wall et al., 2007). Moreover, we have previously reported on the secretion profile of SipA from the ΔsipB non‐polar deletion strain, confirming that this strain secretes identical amounts of SipA compared with the parent wild‐type S. Typhimurium strain (SL1344). As shown in Fig. 5C and consistent with our prior studies (Lee et al., 2000; Wall et al., 2007; Srikanth et al., 2010) infection with the ΔsipB non‐polar deletion mutant failed to disrupt PERP accumulation at the apical surface. Thus, these observations provide important genetic‐based evidence to further substantiate our contention that SipA does not need to be translocated into the epithelial cell cytosol but rather acts extracellularly to elicit PERP accumulation.Because Salmonella enters host cells by a ‘trigger’ mechanism characterized by membrane ruffling and actin cytoskeleton rearrangements at sites of invasion, we further confirmed that our observation of PERP accumulating at the apical surface was not simply due to leakage of biotin through the intercellular junctions. To control for this possibility, we evaluated the gap junction protein, E‐cadherin, during infection with Salmonella. As shown in Fig. S3B, we failed to detect apical biotin labeling of E‐cadherin in response to wild‐type Salmonella infection, again demonstrating the specific detection of apically located PERP.Lastly, to determine whether our observations were due to redistribution of PERP or the result of an overall increase in protein expression in response to Salmonella infection, we examined the total level of PERP expression in non‐infected compared with wild‐type infected polarized cells. As shown in Fig. 5D we detected a moderate increase in PERP protein expression in response to Salmonella infection. Although this result is statistically significant, such an increase is modest compared with the prominent increase in PERP protein expression found at the apical epithelial surface in response to Salmonella infection. To confirm the cellular increase of PERP expression does not completely explain its accumulation at the apical surface of Salmonella‐infected cells, we performed the cell surface biotinylation experiments with the addition of brefeldin A, a drug known to block the anterograde transport of proteins from the endoplasmic reticulum to the Golgi apparatus. If the apical increase of PERP during Salmonella infection is due to the transport of newly synthesized PERP, we would expect treatment with brefeldin A to block this response. As shown in Fig. 5E, treatment with brefeldin A reduces the amount of PERP at the apical surface compared with cells not treated with the drug; however, we still observed a considerable increase in PERP expression in response to Salmonella infection. This observation was further confirmed by the failure of treatment with cycloheximide, which prevents new protein synthesis, to block apical accumulation of PERP in response to infection (Fig. 5F). Taken together, these results are consistent with the hypothesis that the apical accumulation of PERP might be due to alterations in protein trafficking rather than to an increase in total cellular stores.Mechanism governing PERP localizationWe have begun to examine the molecular mechanism governing the apical accumulation of PERP by analysing the localization of PERP in response to Salmonella infection of polarized monolayers of intestinal epithelial cells. Using wide‐field fluorescent microscopy, we observed a distinct punctate PERP staining pattern in Salmonella‐infected cells, which is in contrast to a mostly diffuse staining pattern in cell monolayers not infected with Salmonella (Fig. 6A and B). The amount of PERP punctae was quantified with the FIJI software (Fig. S4) and found to be significantly increased in response to Salmonella infection compared with non‐infected cells. The PERP punctae were consistently found to be apically located (Fig. 7), providing further evidence that Salmonella‐induced PERP redistribution occurs at the apical surface (Fig. 5). Additionally, the formation of the punctate staining pattern appeared to be at least in part dependent on SipA, as cells infected with the isogenic SipA mutant strain showed a more diffuse PERP staining pattern similar to that seen in the non‐infected cells (Fig. 6C), whereas infection with the complemented strain rescued the punctate staining pattern (Fig. 6D). Exogenous addition of purified HA‐tagged SipA at concentrations previously shown to trigger PMN migration to the same degree as wild‐type Salmonella infection (Lee et al., 2000) also induced a punctate staining pattern similar to that of infected cells (Fig. 6E and Fig. S4), further indicating that extracellular SipA is capable of triggering PERP redistribution.Figure 6
PERP reorganizes in response to S
almonella infection. T84 monolayers were treated with (A) buffer only (−) or infected with (B) wild‐type S
almonella (WT), (C) SipA‐deficient S
almonella (ΔSipA), (D) SipA‐complemented S
almonella (pSipA), (E) treated apically with 20 μg ml−1 of HA‐tagged SipA (Lee et al., 2000) (+ HA‐SipA) or (F) S
almonella expressing a caspase‐3 site mutant SipA (CSM). Cells were stained with an antibody against PERP followed by secondary conjugated to Alexa Fluor 488 (green), and with phalloidin conjugated to Alexa Fluor 647 (projected blue). The volume plots imaged at 60× magnification show PERP located at the apical surface, and the presence of punctate staining patterns in response to infection. The level of punctate staining was quantified via FIJI (Fig. S4). There is more punctate staining with WT infection and infection with the CSM strain compared with buffer‐only treated cells. There is less punctate staining with the SipA‐deficient infection, which is rescued by infection with the SipA‐complemented strain. Treatment with HA‐tagged SipA results in a punctate PERP staining pattern comparable with that seen with WT S
almonella infection. Bar represents 10 μm.figure2014 John Wiley & Sons Ltd.Figure 7
PERP punctae are apically located. T84 cells were treated as indicated in Fig. 6. The side view of the monolayer volume plots show the punctate staining is mostly apical (above the bisecting Z‐plane line). The μm values at the top left of the images indicate the thickness of the respective monolayer. The location of punctae was found to be apical across all samples.figure2014 John Wiley & Sons Ltd.It has been documented that increased levels of PERP lead to caspase‐3 activation (Davies et al., 2009). Because of the finding that SipA may play a role in the redistribution of PERP during S. Typhimurium infection, and because we previously showed the proinflammatory function of SipA requires cleavage by caspase‐3, we next investigated the extent to which SipA processing by caspase‐3 is necessary to induce the redistribution and accumulation of PERP to the apical surface. We therefore infected polarized intestinal epithelial cell monolayers with an isogenic S. Typhimurium strain in which the caspase‐3 recognition motif was changed in the key aspartic acid at position four to alanine (DEVD → DEVA; termed caspase site mutant: csm‐SipA), rendering SipA insensitive to caspase‐3 cleavage (Srikanth et al., 2010). As shown in Fig. 6F, we found that the csm‐SipA strain induced a PERP punctate staining pattern comparable to that of wild‐type Salmonella infection. This result suggests that SipA does not depend on caspase‐3 cleavage to alter PERP localization, but rather SipA is able to promote PERP intracellular trafficking prior to being cleaved by caspase‐3. These observations build on our initial report of the role of caspase‐3 activity during Salmonella infection providing new insight into the point at which specific events in Salmonella infection are required to promote pathogenesis.

PERP is a binding partner of SipA
Previous studies have identified the S. Typhimurium effector, SipA, as an important mediator of the immune inflammatory response that results in PMN influx. The fact that our prior studies found purified SipA to directly activate this response has prompted us to consider whether SipA might engage a surface receptor (Lee et al., 2000). Because we infer that this putative receptor represents the initiation site of the transcellular PMN signalling cascade, identification of a functional receptor will be crucial for understanding SipA's involvement in controlling intestinal inflammation. We used a split ubiquitin‐based Y2H (protein–protein interaction) analysis system (Dualsystems Biotech) (Stagljar et al., 1998; Dirnberger et al., 2008), with full‐length SipA as bait and a human colonic cDNA‐based library as prey. Approximately 4 × 106 transformants were screened and selected based on two growth reporters. Candidate interacting partners were then selected using a Lac‐Z based colorimetric reporter assay. The screen yielded seven positive clones out of which PERP was represented three times (Table 1).Table 1
SipA interacting partner candidatesSipA interacting partner candidates identified via Y2H screenCloneGene nameFunctionReference1.SERP1ER stress responseYamaguchi et al., 1999
2.DERP2Cell death regulationOka et al., 2008
3.TMEM87Unknown4.TMEM147Interacts with nicalin–NOMO complexDettmer et al., 2010

5.

PERP
a

p53 effector, regulates caspase‐3 activation
Attardi et al., 2000; Davies et al., 2009
aMultiple hits.Five potential SipA‐binding candidates were identified from our Y2H screen. Most candidates have been identified as membrane proteins with various functions pertaining to cell stress and death regulation. Out of these candidates, PERP was the only one to be pulled out multiple times from our screen.2014 John Wiley & Sons Ltd.Using the Lac‐Z reporter assay, we confirmed the PERP–SipA interaction in a ‘reverse’ Y2H assay in which SipA was sub‐cloned into the prey vector, and PERP of the initial screen was used as bait. Furthermore, we used a biochemical approach to demonstrate PERP–SipA interactions. Model human intestinal epithelial cells (HCT8) were infected with a wild‐type S. Typhimurium strain expressing an HA‐tagged SipA protein (AJK63). Immunoprecipitation of infected cell lysates with an anti‐HA antibody specifically resulted in the pull down of PERP, as immunoprecipitation with a control IgG antibody under similar conditions yielded neither HA‐SipA nor PERP (Fig. 1A). We also performed the pull down with another Salmonella T3SS effector, SifA, also tagged with HA to ensure our observation was not due to non‐specific recognition of the HA‐tag by the PERP antibody (Fig. 1B).Figure 1
SipA and PERP are binding partners.A. T84 lysates infected with an HA‐tagged SipA containing strain of S
almonella were pulled down with IgG or anti‐HA antibody, and then probed for PERP. Only the HA‐SipA pull down samples resulted in a PERP band.B. The specificity of the PERP–SipA interaction was confirmed by a pull down of HA‐tagged SipA and HA‐tagged SifA. Cells were infected with S
almonella expressing HA‐tagged SipA or HA‐tagged SifA, or left non‐infected as a non‐specific control [(−) control] and lysed. Lysates were pulled down with anti‐HA antibody and probed for PERP. The SipA‐HA lysates resulted in a PERP band, whereas the SifA‐HA lysates resulted only in a faint band of background intensity.C. Passage of HCT8 lysates through glutathione beads conjugated to the GST‐tagged C‐terminus of SipC (‘conjugated’) resulted in the specific pull down of PERP (‘eluted’), while passage of the lysates through non‐conjugated beads fails to result in the pull down of PERP.figure2014 John Wiley & Sons Ltd.Because these data support our contention that PERP is a SipA binding partner, we next examined the specificity of the PERP–SipA interaction by testing whether PERP binds to the Salmonella protein SipC, a component of the T3SS1 translocon. SipC is not only required for the translocation of Salmonella effectors into the host cell (Collazo and Galan, 1997) and for Salmonella invasion (Myeni and Zhou, 2010) but also SipC and SipA are known to have cooperative roles during invasion (McGhie et al., 2001). As shown in Fig. 1C, passage of HCT8 lysates across beads bound to the GST‐labeled C‐terminus of SipC (Nichols and Casanova, 2010) resulted in the specific pull down of PERP, suggesting that PERP is able to interact with two Salmonella proteins that function during early stages of Salmonella pathogenesis and that PERP may have a role mediating these events. However, the precise mechanism(s) remain unknown.

PERP is a binding partner of SipA

Previous studies have identified the S. Typhimurium effector, SipA, as an important mediator of the immune inflammatory response that results in PMN influx. The fact that our prior studies found purified SipA to directly activate this response has prompted us to consider whether SipA might engage a surface receptor (Lee et al., 2000). Because we infer that this putative receptor represents the initiation site of the transcellular PMN signalling cascade, identification of a functional receptor will be crucial for understanding SipA's involvement in controlling intestinal inflammation. We used a split ubiquitin‐based Y2H (protein–protein interaction) analysis system (Dualsystems Biotech) (Stagljar et al., 1998; Dirnberger et al., 2008), with full‐length SipA as bait and a human colonic cDNA‐based library as prey. Approximately 4 × 106 transformants were screened and selected based on two growth reporters. Candidate interacting partners were then selected using a Lac‐Z based colorimetric reporter assay. The screen yielded seven positive clones out of which PERP was represented three times (Table 1).

SipA interacting partner candidates

Multiple hits.

Five potential SipA‐binding candidates were identified from our Y2H screen. Most candidates have been identified as membrane proteins with various functions pertaining to cell stress and death regulation. Out of these candidates, PERP was the only one to be pulled out multiple times from our screen.

Using the Lac‐Z reporter assay, we confirmed the PERP–SipA interaction in a ‘reverse’ Y2H assay in which SipA was sub‐cloned into the prey vector, and PERP of the initial screen was used as bait. Furthermore, we used a biochemical approach to demonstrate PERP–SipA interactions. Model human intestinal epithelial cells (HCT8) were infected with a wild‐type S. Typhimurium strain expressing an HA‐tagged SipA protein (AJK63). Immunoprecipitation of infected cell lysates with an anti‐HA antibody specifically resulted in the pull down of PERP, as immunoprecipitation with a control IgG antibody under similar conditions yielded neither HA‐SipA nor PERP (Fig. 1A). We also performed the pull down with another Salmonella T3SS effector, SifA, also tagged with HA to ensure our observation was not due to non‐specific recognition of the HA‐tag by the PERP antibody (Fig. 1B).

SipA and PERP are binding partners.

A. T84 lysates infected with an HA‐tagged SipA containing strain of S
almonella were pulled down with IgG or anti‐HA antibody, and then probed for PERP. Only the HA‐SipA pull down samples resulted in a PERP band.

B. The specificity of the PERP–SipA interaction was confirmed by a pull down of HA‐tagged SipA and HA‐tagged SifA. Cells were infected with S
almonella expressing HA‐tagged SipA or HA‐tagged SifA, or left non‐infected as a non‐specific control [(−) control] and lysed. Lysates were pulled down with anti‐HA antibody and probed for PERP. The SipA‐HA lysates resulted in a PERP band, whereas the SifA‐HA lysates resulted only in a faint band of background intensity.

C. Passage of HCT8 lysates through glutathione beads conjugated to the GST‐tagged C‐terminus of SipC (‘conjugated’) resulted in the specific pull down of PERP (‘eluted’), while passage of the lysates through non‐conjugated beads fails to result in the pull down of PERP.

Because these data support our contention that PERP is a SipA binding partner, we next examined the specificity of the PERP–SipA interaction by testing whether PERP binds to the Salmonella protein SipC, a component of the T3SS1 translocon. SipC is not only required for the translocation of Salmonella effectors into the host cell (Collazo and Galan, 1997) and for Salmonella invasion (Myeni and Zhou, 2010) but also SipC and SipA are known to have cooperative roles during invasion (McGhie et al., 2001). As shown in Fig. 1C, passage of HCT8 lysates across beads bound to the GST‐labeled C‐terminus of SipC (Nichols and Casanova, 2010) resulted in the specific pull down of PERP, suggesting that PERP is able to interact with two Salmonella proteins that function during early stages of Salmonella pathogenesis and that PERP may have a role mediating these events. However, the precise mechanism(s) remain unknown.

Functional consequences of PERP in the promotion of the inflammatory response to Salmonella infectionPERP is a tetraspanning membrane protein that belongs to the PMP‐22(Gas3)/epithelial membrane protein (EMP) family (Attardi et al., 2000), which includes PMP‐22 and EMP 1, 2 and 3. Detection of PERP as an interacting partner with SipA piqued our interest given that PERP has been documented to impact inflammatory signalling pathways (Beaudry et al., 2010) as well as to regulate the activation of caspase‐3 (Davies et al., 2009; Singaravelu et al., 2009). Because we have shown the Salmonella effector SipA induces inflammatory pathways that lead to the recruitment of PMNs to the site of infection, we sought to determine the extent to which PERP might also be involved in governing these processes during infection with Salmonella using our in vitro PMN migration assay (Experimental procedures). Following infection, polarized intestinal cell monolayers were exposed to 25 μg ml−1 of anti‐PERP antibody, anti‐MTCO‐1 antibody (mitochondrial marker – used as an irrelevant isotype control) or IgG isotype control antibody prior to adding freshly isolated human peripheral blood PMNs. As shown in Fig. 2A, the presence of anti‐PERP antibody decreased the ability of Salmonella to induce PMN transepithelial migration by 90%. This result was specific to exposure with the PERP antibody, as the MTCO‐1 and IgG‐treated cells did not similarly inhibit Salmonella‐induced PMN transmigration. Addition of the PERP antibody in the absence of infection has no impact on PMN transmigration (Fig. S1).Figure 2
PERP promotes the inflammatory response to S
almonella infection.A. Polarized HCT8 cells were infected with wild‐type S. Typhimurium for 40 minutes and then exposed to 25 μg ml−1 of PERP, MTCO‐1 (mitochondrial marker, control) or IgG (control) antibodies at the basolateral surface or left in HBSS + buffer prior to addition of PMNs at the basolateral surface. Values are expressed as percent PMN migration compared with results from infected cells that were not treated with antibodies.B. Lines of stable PERP knockdown cells and control cells were generated via transfection of PERP siRNA (inset). The PERP knockdown cells showed no defect in barrier function compared with the vector control cells (data not shown). Knocking down PERP resulted in a statistically significant 40% reduction in PMN migration in response to infection compared with vector control cells. Error bars represent ± standard errors and P‐values less than 0.05 according to Student's t‐test were considered statistically significant.figure2014 John Wiley & Sons Ltd.As a complementary approach, we performed PMN transepithelial migration assays using PERP siRNA knockdown cells (p11) paired with a siRNA vector control (p24). Comparable with the PERP antibody blocking studies, PMN transepithelial migration across the PERP knockdown cells in response to Salmonella infection, where HXA3 is the major PMN chemoattractant gradient induced, was reduced by 40% as compared with the vector control cells (Fig. 2B). Although these studies suggest that PERP is involved in facilitating PMN transmigration in response to Salmonella infection, PERP might also play a role in other intestinal inflammatory conditions beyond that of Salmonella infection where PMN migration is a key pathological feature. We modeled such conditions in vitro via addition of formyl‐methionyl‐leucyl‐phenylalanine (fMLP), a PMN chemoattractant, to our polarized monolayers in the absence of infection. As shown in Fig. 3, PMN transepithelial migration in response to an imposed gradient of fMLP across cells treated with the PERP blocking antibody (Fig. 3A), or across the PERP knockdown cells (Fig. 3B) was reduced approximately 90% and 35% respectively. We also probed the function of PERP during PMN transmigration in response to chemoattractants other than HXA3 known to be secreted by intestinal epithelial cells, such as IL‐8 and leukotriene B4 (LTB4). We found that blocking PERP by pretreating HCT8 cell monolayers with 25 μg ml−1 of PERP antibody for 30 min prior to inducing imposed gradients of IL‐8 or LTB4 to the apical surface (see Experimental Procedures for details) resulted in a modest, although statistically significant inhibitory impact on IL‐8‐induced migration (Fig. 3C), but not on LTB4‐induced migration (Fig. 3D). Together, these results indicate PERP has a broad, although not universal, role in regulating PMN migration.Figure 3
PERP promotes PMN migration. Migration was stimulated by the addition of fMLP, IL‐8 or LTB4. Presence of the PERP antibody (A) specifically reduced fMLP‐induced PMN migration by about 90%. Similarly, migration across PERP knockdown monolayers (B) was reduced by about 35%. Presence of the PERP antibody also reduced IL‐8‐induced migration by about 20% (C), although had no impact on migration induced by LTB4 (D). Values are expressed as percent PMN migration normalized to chemoattractant treatment alone samples ± standard errors. P‐values less than 0.05 according to Student's t‐test were considered statistically significant.figure2014 John Wiley & Sons Ltd.The S. Typhimurium effector protein, SipA, promotes gastroenteritis via two distinct functional motifs that trigger not only inflammation but also mechanisms of bacterial entry (Wall et al., 2007). Moreover, we also recently found that during infection of intestinal epithelial cells, SipA is responsible for the early activation of caspase‐3 (Srikanth et al., 2010). This enzyme is essential for SipA cleavage at a specific recognition motif, dividing the protein into its two functional domains (Srikanth et al., 2010). Such studies further revealed that cleavage of the SipA caspase‐3 motif is central for promoting proinflammatory responses, and therefore we infer the involvement of caspase‐3 during pathogenesis given that Salmonella is less virulent in caspase‐3 knockout (caspase‐3−/−) mice (Srikanth et al., 2010). Because prior studies have reported that increased levels of PERP lead to caspase‐3 activation (Davies et al., 2009), we next sought to determine the extent to which PERP plays a role in Salmonella‐induced activation of caspase‐3.Using a colorimetric caspase‐3 bioactivity assay kit, we evaluated the extent to which PERP regulates caspase‐3 activation in an HCT8 line of transient PERP knockdown cells (Fig. 4A) in the absence and presence of Salmonella infection. We observed an increase of 86.7% ± 5.3 (standard error, P < 0.05) in the level of Salmonella‐induced activated caspase‐3 in the vector control cells compared with only a 57.8% ± 4.5% (standard error) increase in Salmonella‐induced activated caspase‐3 in the PERP knockdown cells (Fig. 4B). Because the partial knockdown of PERP resulted in about a 30% decrease in the ability to induce caspase‐3, these results suggest that PERP may be necessary but not sufficient for caspase‐3 activation during Salmonella infection.Figure 4
PERP promotes caspase‐3 activity during S
almonella infection.A. PERP was transiently knocked down in HCT8 cells. A no‐targeting control was used to confirm specificity. By this method, PERP was reproducibly knocked down by about 50%.B. Levels of activated caspase‐3 in response to S
almonella infection were reduced by about 30% in the PERP knockdown cells. Numbers are expressed as percent of activated caspase‐3 relative to activated caspase‐3 levels in non‐infected, vector control cells. Error bars show ± standard error. P‐values less than 0.05 according to Student's t‐test were considered statistically significant.figure2014 John Wiley & Sons Ltd.

Functional consequences of PERP in the promotion of the inflammatory response to Salmonella infection

PERP is a tetraspanning membrane protein that belongs to the PMP‐22(Gas3)/epithelial membrane protein (EMP) family (Attardi et al., 2000), which includes PMP‐22 and EMP 1, 2 and 3. Detection of PERP as an interacting partner with SipA piqued our interest given that PERP has been documented to impact inflammatory signalling pathways (Beaudry et al., 2010) as well as to regulate the activation of caspase‐3 (Davies et al., 2009; Singaravelu et al., 2009). Because we have shown the Salmonella effector SipA induces inflammatory pathways that lead to the recruitment of PMNs to the site of infection, we sought to determine the extent to which PERP might also be involved in governing these processes during infection with Salmonella using our in vitro PMN migration assay (Experimental procedures). Following infection, polarized intestinal cell monolayers were exposed to 25 μg ml−1 of anti‐PERP antibody, anti‐MTCO‐1 antibody (mitochondrial marker – used as an irrelevant isotype control) or IgG isotype control antibody prior to adding freshly isolated human peripheral blood PMNs. As shown in Fig. 2A, the presence of anti‐PERP antibody decreased the ability of Salmonella to induce PMN transepithelial migration by 90%. This result was specific to exposure with the PERP antibody, as the MTCO‐1 and IgG‐treated cells did not similarly inhibit Salmonella‐induced PMN transmigration. Addition of the PERP antibody in the absence of infection has no impact on PMN transmigration (Fig. S1).

PERP promotes the inflammatory response to S
almonella infection.

A. Polarized HCT8 cells were infected with wild‐type S. Typhimurium for 40 minutes and then exposed to 25 μg ml−1 of PERP, MTCO‐1 (mitochondrial marker, control) or IgG (control) antibodies at the basolateral surface or left in HBSS + buffer prior to addition of PMNs at the basolateral surface. Values are expressed as percent PMN migration compared with results from infected cells that were not treated with antibodies.

B. Lines of stable PERP knockdown cells and control cells were generated via transfection of PERP siRNA (inset). The PERP knockdown cells showed no defect in barrier function compared with the vector control cells (data not shown). Knocking down PERP resulted in a statistically significant 40% reduction in PMN migration in response to infection compared with vector control cells. Error bars represent ± standard errors and P‐values less than 0.05 according to Student's t‐test were considered statistically significant.

As a complementary approach, we performed PMN transepithelial migration assays using PERP siRNA knockdown cells (p11) paired with a siRNA vector control (p24). Comparable with the PERP antibody blocking studies, PMN transepithelial migration across the PERP knockdown cells in response to Salmonella infection, where HXA3 is the major PMN chemoattractant gradient induced, was reduced by 40% as compared with the vector control cells (Fig. 2B). Although these studies suggest that PERP is involved in facilitating PMN transmigration in response to Salmonella infection, PERP might also play a role in other intestinal inflammatory conditions beyond that of Salmonella infection where PMN migration is a key pathological feature. We modeled such conditions in vitro via addition of formyl‐methionyl‐leucyl‐phenylalanine (fMLP), a PMN chemoattractant, to our polarized monolayers in the absence of infection. As shown in Fig. 3, PMN transepithelial migration in response to an imposed gradient of fMLP across cells treated with the PERP blocking antibody (Fig. 3A), or across the PERP knockdown cells (Fig. 3B) was reduced approximately 90% and 35% respectively. We also probed the function of PERP during PMN transmigration in response to chemoattractants other than HXA3 known to be secreted by intestinal epithelial cells, such as IL‐8 and leukotriene B4 (LTB4). We found that blocking PERP by pretreating HCT8 cell monolayers with 25 μg ml−1 of PERP antibody for 30 min prior to inducing imposed gradients of IL‐8 or LTB4 to the apical surface (see Experimental Procedures for details) resulted in a modest, although statistically significant inhibitory impact on IL‐8‐induced migration (Fig. 3C), but not on LTB4‐induced migration (Fig. 3D). Together, these results indicate PERP has a broad, although not universal, role in regulating PMN migration.

PERP promotes PMN migration. Migration was stimulated by the addition of fMLP, IL‐8 or LTB4. Presence of the PERP antibody (A) specifically reduced fMLP‐induced PMN migration by about 90%. Similarly, migration across PERP knockdown monolayers (B) was reduced by about 35%. Presence of the PERP antibody also reduced IL‐8‐induced migration by about 20% (C), although had no impact on migration induced by LTB4 (D). Values are expressed as percent PMN migration normalized to chemoattractant treatment alone samples ± standard errors. P‐values less than 0.05 according to Student's t‐test were considered statistically significant.

The S. Typhimurium effector protein, SipA, promotes gastroenteritis via two distinct functional motifs that trigger not only inflammation but also mechanisms of bacterial entry (Wall et al., 2007). Moreover, we also recently found that during infection of intestinal epithelial cells, SipA is responsible for the early activation of caspase‐3 (Srikanth et al., 2010). This enzyme is essential for SipA cleavage at a specific recognition motif, dividing the protein into its two functional domains (Srikanth et al., 2010). Such studies further revealed that cleavage of the SipA caspase‐3 motif is central for promoting proinflammatory responses, and therefore we infer the involvement of caspase‐3 during pathogenesis given that Salmonella is less virulent in caspase‐3 knockout (caspase‐3−/−) mice (Srikanth et al., 2010). Because prior studies have reported that increased levels of PERP lead to caspase‐3 activation (Davies et al., 2009), we next sought to determine the extent to which PERP plays a role in Salmonella‐induced activation of caspase‐3.

Using a colorimetric caspase‐3 bioactivity assay kit, we evaluated the extent to which PERP regulates caspase‐3 activation in an HCT8 line of transient PERP knockdown cells (Fig. 4A) in the absence and presence of Salmonella infection. We observed an increase of 86.7% ± 5.3 (standard error, P < 0.05) in the level of Salmonella‐induced activated caspase‐3 in the vector control cells compared with only a 57.8% ± 4.5% (standard error) increase in Salmonella‐induced activated caspase‐3 in the PERP knockdown cells (Fig. 4B). Because the partial knockdown of PERP resulted in about a 30% decrease in the ability to induce caspase‐3, these results suggest that PERP may be necessary but not sufficient for caspase‐3 activation during Salmonella infection.

PERP promotes caspase‐3 activity during S
almonella infection.

A. PERP was transiently knocked down in HCT8 cells. A no‐targeting control was used to confirm specificity. By this method, PERP was reproducibly knocked down by about 50%.

B. Levels of activated caspase‐3 in response to S
almonella infection were reduced by about 30% in the PERP knockdown cells. Numbers are expressed as percent of activated caspase‐3 relative to activated caspase‐3 levels in non‐infected, vector control cells. Error bars show ± standard error. P‐values less than 0.05 according to Student's t‐test were considered statistically significant.

PERP accumulates at the apical surface in a SipA‐dependent mannerThus far, our observations show that during infection with Salmonella, PERP not only plays a crucial role in governing PMN recruitment but is also involved with the activation of caspase‐3. PERP, as a tetraspanning protein, performs a wide range of functions and confers cell‐specific and tissue‐specific roles. For example, PERP has been shown to localize to desmosomes in mouse newborn skin (Ihrie et al., 2005). More recently, PERP was shown to localize to peri‐ and interdesmosomal regions termed ‘tessellate junctions’ in stratified epithelia as well as to desmosomes in bovine intestinal epithelium (Franke et al., 2013). Consistent with this, we also observed PERP expression on the mucosal surface of mouse proximal colon tissue (Fig. S2).Because aberrant localization of transmembrane proteins is linked to numerous human diseases, we examined whether PERP is redistributed during infection with Salmonella. The apical surface of polarized cell monolayers was selectively biotinylated following infection with wild‐type Salmonella or mock infection with buffer. This method permits us to identify changes in protein expression specifically at the apical surface in response to Salmonella infection. As shown in Fig. 5A, PERP accumulates at the apical surface of polarized intestinal epithelial cells in response to wild‐type infection. Moreover, the involvement of SipA in the accumulation of PERP at the apical surface is evidenced by infection with the isogenic SipA‐deficient strain, which results in significantly less PERP accumulation at this location compared with wild‐type infection (Fig. 5A). By contrast, infection with a SipA‐complemented strain correlated with greater PERP accumulation at the apical surface as compared with the wild‐type strain (Fig. 5B). These results indicate SipA is necessary, although not sufficient, to induce PERP accumulation at the apical surface during Salmonella infection. The accumulation of PERP at the apical surface also appears to be a directed cellular event since a similar assessment of PERP distribution to the basolateral surface resulted in only a modest increase (Fig. S3A).Figure 5
PERP accumulates at the apical cell surface in a SipA‐dependent manner.A–C. Polarized HCT8 monolayers were infected with wild‐type (WT), SipA‐deficient (ΔSipA), SipA‐complemented S
almonella (SipA +) or SipB‐deficient (ΔSipB), or left uninfected (−) in HBSS + for 1 h, and the apical cell surfaces were biotinylated, pulled down with streptavidin and Western blotted for PERP.D. Whole cell lysates from non‐infected and WT‐infected HCT8s were probed for overall PERP expression. Glyceraldehyde 3‐phosphate dehydrogenase (GAPDH) serves as a loading control. Densitometry confirms a significant, although minor, increase in PERP expression in response to wild‐type infection.E. Cells were treated with 150 μM of brefeldin A (BFA) for 1 h or left untreated in HBSS + for 1 h prior to infection. PERP expression at the apical surface was examined as explained for (A–C).F. Cells were treated with 2 μg ml−1 of cycloheximide (cyc) for 1 h or left untreated in HBSS + for 1 h prior to infection. PERP expression at the apical surface was examined as explained for (A–C). Although it is noted that the basal level of PERP in (E) is comparatively higher than the basal level of PERP in (A–C), we interpret this difference as normal variation seen when using different stocks of cultured cell lines. Regardless of this observed difference, we are able to consistently reproduce results showing a function for PERP during S
almonella pathogenesis. Densitometry analyses show the change in PERP expression induced by infection with mutant or complemented strains compared with the change in PERP expression induced by WT‐infected cells after normalizing to non‐infected values. Error bars show ± standard error. P‐values less than 0.05 according to Student's t‐test were considered statistically significant.figure2014 John Wiley & Sons Ltd.Our previous results demonstrated that purified S. Typhimurium SipA protein could trigger the PMN migration response in the absence of the type III secretion and translocation factors, such as SipB and SipC, suggesting that this effector may not need to be translocated to initiate the events that lead to PMN transepithelial migration (Lee et al., 2000). To examine whether PERP also accumulates at the apical surface in the absence of translocated SipA, we took a genetic approach using an isogenic ΔsipB non‐polar deletion mutant. This strain expresses native SipA from the chromosomal sipA locus and is capable of secreting effector proteins but cannot translocate them into the host cell cytosol (Wall et al., 2007). Moreover, we have previously reported on the secretion profile of SipA from the ΔsipB non‐polar deletion strain, confirming that this strain secretes identical amounts of SipA compared with the parent wild‐type S. Typhimurium strain (SL1344). As shown in Fig. 5C and consistent with our prior studies (Lee et al., 2000; Wall et al., 2007; Srikanth et al., 2010) infection with the ΔsipB non‐polar deletion mutant failed to disrupt PERP accumulation at the apical surface. Thus, these observations provide important genetic‐based evidence to further substantiate our contention that SipA does not need to be translocated into the epithelial cell cytosol but rather acts extracellularly to elicit PERP accumulation.Because Salmonella enters host cells by a ‘trigger’ mechanism characterized by membrane ruffling and actin cytoskeleton rearrangements at sites of invasion, we further confirmed that our observation of PERP accumulating at the apical surface was not simply due to leakage of biotin through the intercellular junctions. To control for this possibility, we evaluated the gap junction protein, E‐cadherin, during infection with Salmonella. As shown in Fig. S3B, we failed to detect apical biotin labeling of E‐cadherin in response to wild‐type Salmonella infection, again demonstrating the specific detection of apically located PERP.Lastly, to determine whether our observations were due to redistribution of PERP or the result of an overall increase in protein expression in response to Salmonella infection, we examined the total level of PERP expression in non‐infected compared with wild‐type infected polarized cells. As shown in Fig. 5D we detected a moderate increase in PERP protein expression in response to Salmonella infection. Although this result is statistically significant, such an increase is modest compared with the prominent increase in PERP protein expression found at the apical epithelial surface in response to Salmonella infection. To confirm the cellular increase of PERP expression does not completely explain its accumulation at the apical surface of Salmonella‐infected cells, we performed the cell surface biotinylation experiments with the addition of brefeldin A, a drug known to block the anterograde transport of proteins from the endoplasmic reticulum to the Golgi apparatus. If the apical increase of PERP during Salmonella infection is due to the transport of newly synthesized PERP, we would expect treatment with brefeldin A to block this response. As shown in Fig. 5E, treatment with brefeldin A reduces the amount of PERP at the apical surface compared with cells not treated with the drug; however, we still observed a considerable increase in PERP expression in response to Salmonella infection. This observation was further confirmed by the failure of treatment with cycloheximide, which prevents new protein synthesis, to block apical accumulation of PERP in response to infection (Fig. 5F). Taken together, these results are consistent with the hypothesis that the apical accumulation of PERP might be due to alterations in protein trafficking rather than to an increase in total cellular stores.

PERP accumulates at the apical surface in a SipA‐dependent manner

Thus far, our observations show that during infection with Salmonella, PERP not only plays a crucial role in governing PMN recruitment but is also involved with the activation of caspase‐3. PERP, as a tetraspanning protein, performs a wide range of functions and confers cell‐specific and tissue‐specific roles. For example, PERP has been shown to localize to desmosomes in mouse newborn skin (Ihrie et al., 2005). More recently, PERP was shown to localize to peri‐ and interdesmosomal regions termed ‘tessellate junctions’ in stratified epithelia as well as to desmosomes in bovine intestinal epithelium (Franke et al., 2013). Consistent with this, we also observed PERP expression on the mucosal surface of mouse proximal colon tissue (Fig. S2).

Because aberrant localization of transmembrane proteins is linked to numerous human diseases, we examined whether PERP is redistributed during infection with Salmonella. The apical surface of polarized cell monolayers was selectively biotinylated following infection with wild‐type Salmonella or mock infection with buffer. This method permits us to identify changes in protein expression specifically at the apical surface in response to Salmonella infection. As shown in Fig. 5A, PERP accumulates at the apical surface of polarized intestinal epithelial cells in response to wild‐type infection. Moreover, the involvement of SipA in the accumulation of PERP at the apical surface is evidenced by infection with the isogenic SipA‐deficient strain, which results in significantly less PERP accumulation at this location compared with wild‐type infection (Fig. 5A). By contrast, infection with a SipA‐complemented strain correlated with greater PERP accumulation at the apical surface as compared with the wild‐type strain (Fig. 5B). These results indicate SipA is necessary, although not sufficient, to induce PERP accumulation at the apical surface during Salmonella infection. The accumulation of PERP at the apical surface also appears to be a directed cellular event since a similar assessment of PERP distribution to the basolateral surface resulted in only a modest increase (Fig. S3A).

PERP accumulates at the apical cell surface in a SipA‐dependent manner.

A–C. Polarized HCT8 monolayers were infected with wild‐type (WT), SipA‐deficient (ΔSipA), SipA‐complemented S
almonella (SipA +) or SipB‐deficient (ΔSipB), or left uninfected (−) in HBSS + for 1 h, and the apical cell surfaces were biotinylated, pulled down with streptavidin and Western blotted for PERP.

D. Whole cell lysates from non‐infected and WT‐infected HCT8s were probed for overall PERP expression. Glyceraldehyde 3‐phosphate dehydrogenase (GAPDH) serves as a loading control. Densitometry confirms a significant, although minor, increase in PERP expression in response to wild‐type infection.

E. Cells were treated with 150 μM of brefeldin A (BFA) for 1 h or left untreated in HBSS + for 1 h prior to infection. PERP expression at the apical surface was examined as explained for (A–C).

F. Cells were treated with 2 μg ml−1 of cycloheximide (cyc) for 1 h or left untreated in HBSS + for 1 h prior to infection. PERP expression at the apical surface was examined as explained for (A–C). Although it is noted that the basal level of PERP in (E) is comparatively higher than the basal level of PERP in (A–C), we interpret this difference as normal variation seen when using different stocks of cultured cell lines. Regardless of this observed difference, we are able to consistently reproduce results showing a function for PERP during S
almonella pathogenesis. Densitometry analyses show the change in PERP expression induced by infection with mutant or complemented strains compared with the change in PERP expression induced by WT‐infected cells after normalizing to non‐infected values. Error bars show ± standard error. P‐values less than 0.05 according to Student's t‐test were considered statistically significant.

Our previous results demonstrated that purified S. Typhimurium SipA protein could trigger the PMN migration response in the absence of the type III secretion and translocation factors, such as SipB and SipC, suggesting that this effector may not need to be translocated to initiate the events that lead to PMN transepithelial migration (Lee et al., 2000). To examine whether PERP also accumulates at the apical surface in the absence of translocated SipA, we took a genetic approach using an isogenic ΔsipB non‐polar deletion mutant. This strain expresses native SipA from the chromosomal sipA locus and is capable of secreting effector proteins but cannot translocate them into the host cell cytosol (Wall et al., 2007). Moreover, we have previously reported on the secretion profile of SipA from the ΔsipB non‐polar deletion strain, confirming that this strain secretes identical amounts of SipA compared with the parent wild‐type S. Typhimurium strain (SL1344). As shown in Fig. 5C and consistent with our prior studies (Lee et al., 2000; Wall et al., 2007; Srikanth et al., 2010) infection with the ΔsipB non‐polar deletion mutant failed to disrupt PERP accumulation at the apical surface. Thus, these observations provide important genetic‐based evidence to further substantiate our contention that SipA does not need to be translocated into the epithelial cell cytosol but rather acts extracellularly to elicit PERP accumulation.

Because Salmonella enters host cells by a ‘trigger’ mechanism characterized by membrane ruffling and actin cytoskeleton rearrangements at sites of invasion, we further confirmed that our observation of PERP accumulating at the apical surface was not simply due to leakage of biotin through the intercellular junctions. To control for this possibility, we evaluated the gap junction protein, E‐cadherin, during infection with Salmonella. As shown in Fig. S3B, we failed to detect apical biotin labeling of E‐cadherin in response to wild‐type Salmonella infection, again demonstrating the specific detection of apically located PERP.

Lastly, to determine whether our observations were due to redistribution of PERP or the result of an overall increase in protein expression in response to Salmonella infection, we examined the total level of PERP expression in non‐infected compared with wild‐type infected polarized cells. As shown in Fig. 5D we detected a moderate increase in PERP protein expression in response to Salmonella infection. Although this result is statistically significant, such an increase is modest compared with the prominent increase in PERP protein expression found at the apical epithelial surface in response to Salmonella infection. To confirm the cellular increase of PERP expression does not completely explain its accumulation at the apical surface of Salmonella‐infected cells, we performed the cell surface biotinylation experiments with the addition of brefeldin A, a drug known to block the anterograde transport of proteins from the endoplasmic reticulum to the Golgi apparatus. If the apical increase of PERP during Salmonella infection is due to the transport of newly synthesized PERP, we would expect treatment with brefeldin A to block this response. As shown in Fig. 5E, treatment with brefeldin A reduces the amount of PERP at the apical surface compared with cells not treated with the drug; however, we still observed a considerable increase in PERP expression in response to Salmonella infection. This observation was further confirmed by the failure of treatment with cycloheximide, which prevents new protein synthesis, to block apical accumulation of PERP in response to infection (Fig. 5F). Taken together, these results are consistent with the hypothesis that the apical accumulation of PERP might be due to alterations in protein trafficking rather than to an increase in total cellular stores.

Mechanism governing PERP localizationWe have begun to examine the molecular mechanism governing the apical accumulation of PERP by analysing the localization of PERP in response to Salmonella infection of polarized monolayers of intestinal epithelial cells. Using wide‐field fluorescent microscopy, we observed a distinct punctate PERP staining pattern in Salmonella‐infected cells, which is in contrast to a mostly diffuse staining pattern in cell monolayers not infected with Salmonella (Fig. 6A and B). The amount of PERP punctae was quantified with the FIJI software (Fig. S4) and found to be significantly increased in response to Salmonella infection compared with non‐infected cells. The PERP punctae were consistently found to be apically located (Fig. 7), providing further evidence that Salmonella‐induced PERP redistribution occurs at the apical surface (Fig. 5). Additionally, the formation of the punctate staining pattern appeared to be at least in part dependent on SipA, as cells infected with the isogenic SipA mutant strain showed a more diffuse PERP staining pattern similar to that seen in the non‐infected cells (Fig. 6C), whereas infection with the complemented strain rescued the punctate staining pattern (Fig. 6D). Exogenous addition of purified HA‐tagged SipA at concentrations previously shown to trigger PMN migration to the same degree as wild‐type Salmonella infection (Lee et al., 2000) also induced a punctate staining pattern similar to that of infected cells (Fig. 6E and Fig. S4), further indicating that extracellular SipA is capable of triggering PERP redistribution.Figure 6
PERP reorganizes in response to S
almonella infection. T84 monolayers were treated with (A) buffer only (−) or infected with (B) wild‐type S
almonella (WT), (C) SipA‐deficient S
almonella (ΔSipA), (D) SipA‐complemented S
almonella (pSipA), (E) treated apically with 20 μg ml−1 of HA‐tagged SipA (Lee et al., 2000) (+ HA‐SipA) or (F) S
almonella expressing a caspase‐3 site mutant SipA (CSM). Cells were stained with an antibody against PERP followed by secondary conjugated to Alexa Fluor 488 (green), and with phalloidin conjugated to Alexa Fluor 647 (projected blue). The volume plots imaged at 60× magnification show PERP located at the apical surface, and the presence of punctate staining patterns in response to infection. The level of punctate staining was quantified via FIJI (Fig. S4). There is more punctate staining with WT infection and infection with the CSM strain compared with buffer‐only treated cells. There is less punctate staining with the SipA‐deficient infection, which is rescued by infection with the SipA‐complemented strain. Treatment with HA‐tagged SipA results in a punctate PERP staining pattern comparable with that seen with WT S
almonella infection. Bar represents 10 μm.figure2014 John Wiley & Sons Ltd.Figure 7
PERP punctae are apically located. T84 cells were treated as indicated in Fig. 6. The side view of the monolayer volume plots show the punctate staining is mostly apical (above the bisecting Z‐plane line). The μm values at the top left of the images indicate the thickness of the respective monolayer. The location of punctae was found to be apical across all samples.figure2014 John Wiley & Sons Ltd.It has been documented that increased levels of PERP lead to caspase‐3 activation (Davies et al., 2009). Because of the finding that SipA may play a role in the redistribution of PERP during S. Typhimurium infection, and because we previously showed the proinflammatory function of SipA requires cleavage by caspase‐3, we next investigated the extent to which SipA processing by caspase‐3 is necessary to induce the redistribution and accumulation of PERP to the apical surface. We therefore infected polarized intestinal epithelial cell monolayers with an isogenic S. Typhimurium strain in which the caspase‐3 recognition motif was changed in the key aspartic acid at position four to alanine (DEVD → DEVA; termed caspase site mutant: csm‐SipA), rendering SipA insensitive to caspase‐3 cleavage (Srikanth et al., 2010). As shown in Fig. 6F, we found that the csm‐SipA strain induced a PERP punctate staining pattern comparable to that of wild‐type Salmonella infection. This result suggests that SipA does not depend on caspase‐3 cleavage to alter PERP localization, but rather SipA is able to promote PERP intracellular trafficking prior to being cleaved by caspase‐3. These observations build on our initial report of the role of caspase‐3 activity during Salmonella infection providing new insight into the point at which specific events in Salmonella infection are required to promote pathogenesis.

Mechanism governing PERP localization

We have begun to examine the molecular mechanism governing the apical accumulation of PERP by analysing the localization of PERP in response to Salmonella infection of polarized monolayers of intestinal epithelial cells. Using wide‐field fluorescent microscopy, we observed a distinct punctate PERP staining pattern in Salmonella‐infected cells, which is in contrast to a mostly diffuse staining pattern in cell monolayers not infected with Salmonella (Fig. 6A and B). The amount of PERP punctae was quantified with the FIJI software (Fig. S4) and found to be significantly increased in response to Salmonella infection compared with non‐infected cells. The PERP punctae were consistently found to be apically located (Fig. 7), providing further evidence that Salmonella‐induced PERP redistribution occurs at the apical surface (Fig. 5). Additionally, the formation of the punctate staining pattern appeared to be at least in part dependent on SipA, as cells infected with the isogenic SipA mutant strain showed a more diffuse PERP staining pattern similar to that seen in the non‐infected cells (Fig. 6C), whereas infection with the complemented strain rescued the punctate staining pattern (Fig. 6D). Exogenous addition of purified HA‐tagged SipA at concentrations previously shown to trigger PMN migration to the same degree as wild‐type Salmonella infection (Lee et al., 2000) also induced a punctate staining pattern similar to that of infected cells (Fig. 6E and Fig. S4), further indicating that extracellular SipA is capable of triggering PERP redistribution.

PERP reorganizes in response to S
almonella infection. T84 monolayers were treated with (A) buffer only (−) or infected with (B) wild‐type S
almonella (WT), (C) SipA‐deficient S
almonella (ΔSipA), (D) SipA‐complemented S
almonella (pSipA), (E) treated apically with 20 μg ml−1 of HA‐tagged SipA (Lee et al., 2000) (+ HA‐SipA) or (F) S
almonella expressing a caspase‐3 site mutant SipA (CSM). Cells were stained with an antibody against PERP followed by secondary conjugated to Alexa Fluor 488 (green), and with phalloidin conjugated to Alexa Fluor 647 (projected blue). The volume plots imaged at 60× magnification show PERP located at the apical surface, and the presence of punctate staining patterns in response to infection. The level of punctate staining was quantified via FIJI (Fig. S4). There is more punctate staining with WT infection and infection with the CSM strain compared with buffer‐only treated cells. There is less punctate staining with the SipA‐deficient infection, which is rescued by infection with the SipA‐complemented strain. Treatment with HA‐tagged SipA results in a punctate PERP staining pattern comparable with that seen with WT S
almonella infection. Bar represents 10 μm.

PERP punctae are apically located. T84 cells were treated as indicated in Fig. 6. The side view of the monolayer volume plots show the punctate staining is mostly apical (above the bisecting Z‐plane line). The μm values at the top left of the images indicate the thickness of the respective monolayer. The location of punctae was found to be apical across all samples.

It has been documented that increased levels of PERP lead to caspase‐3 activation (Davies et al., 2009). Because of the finding that SipA may play a role in the redistribution of PERP during S. Typhimurium infection, and because we previously showed the proinflammatory function of SipA requires cleavage by caspase‐3, we next investigated the extent to which SipA processing by caspase‐3 is necessary to induce the redistribution and accumulation of PERP to the apical surface. We therefore infected polarized intestinal epithelial cell monolayers with an isogenic S. Typhimurium strain in which the caspase‐3 recognition motif was changed in the key aspartic acid at position four to alanine (DEVD → DEVA; termed caspase site mutant: csm‐SipA), rendering SipA insensitive to caspase‐3 cleavage (Srikanth et al., 2010). As shown in Fig. 6F, we found that the csm‐SipA strain induced a PERP punctate staining pattern comparable to that of wild‐type Salmonella infection. This result suggests that SipA does not depend on caspase‐3 cleavage to alter PERP localization, but rather SipA is able to promote PERP intracellular trafficking prior to being cleaved by caspase‐3. These observations build on our initial report of the role of caspase‐3 activity during Salmonella infection providing new insight into the point at which specific events in Salmonella infection are required to promote pathogenesis.

DiscussionPERP is a tetraspanning membrane protein that belongs to the PMP‐22(Gas3)/EMP family (Attardi et al., 2000). Although PERP was first reported to be a downstream effector of p53 (Attardi et al., 2000), more recent studies have found PERP to play a critical role not only in maintaining epithelial barrier integrity (Ihrie et al., 2005) but also in regulating genes involved in inflammation (Beaudry et al., 2010). In the current study, we now identify a new role for PERP in the pathogenesis of the enteric pathogen S. Typhimurium. Of particular interest, we show that PERP associates with the S. Typhimurium type III secreted effector (T3SE) SipA and regulates PMN transmigration during infection.Precisely how SipA initially interacts with PERP remains to be determined and our current efforts are focused on understanding the biochemistry of the SipA–PERP interactions including the domains responsible. Nevertheless, some inferences can be made based on our findings. One possibility is that PERP is part of a membrane complex. We reason this to be the case since in addition to PERP, we also identified four other potential SipA binding partners that were less represented in the Y2H screen (Table 1). Tetraspanning proteins are well documented to complex with other tetraspanins, integrins, immunoglobulin proteins, signaling enzymes or co‐receptors to impart a variety of functions (reviewed in Maecker et al., 1997; Hemler, 2001). Thus, it is perhaps not surprising that we have identified a role for PERP in intestinal inflammation. What is striking, however, is that many of the properties of PERP function appear to be consistent with the reported activities of SipA in triggering intestinal inflammation characterized by PMN transepithelial migration, raising the question of whether SipA subverts PERP functional activities.The fact that PERP is involved in facilitating PMN transmigration in response to Salmonella infection is moreover consistent with previous studies showing that PERP regulates the expression of various inflammation‐associated gene products (Beaudry et al., 2010). Among these is Chi3L1, which is expressed in inflamed mucosa, particularly in Crohn's disease and ulcerative colitis patients, and appears to promote bacterial adhesion to colonic epithelial cells (Mizoguchi, 2006). PERP was also found to regulate Ccl20 (or MIP‐3‐alpha), which is expressed in intestinal epithelia associated with Peyer's patches and aids in the attraction of natural killer cells, memory T cells and immature dendritic cells to the site of inflammation (Hoover et al., 2002). Moreover, we have also found PERP to be increased in both a murine model of Salmonella colitis as well as in a dextran sodium sulfate chemically induced colitis model; in the former infection with the SipA mutant strain resulted in PERP expression levels that were similar to background control levels (Hallstrom and McCormick, unpubl. obs.).The molecular mechanism by which PERP supports PMN transmigration is still under investigation. We are exploring the possibilities that either PERP interacts with a ligand or receptor on the surface of PMNs in order to enable their transmigration to the apical surface or activates (or de‐activates) signaling pathways that promote PMN transmigration (Chin et al., 2008). Unpublished observations from our laboratory have also shown that PERP is able to bind to itself and may be expressed on PMNs. Because PERP is known to localize to desmosomes, this raises the interesting possibility that PERP could facilitate PMN migration by promoting PMN interactions with junctional proteins expressed by intestinal epithelial cells. Such activity, if confirmed, would indicate PERP could have a significant role in other intestinal inflammatory conditions beyond that of Salmonella infection where PMN migration is a key pathological feature.Our data also support the notion that PERP regulates caspase‐3 activation during Salmonella infection (Fig. 4). This observation is consistent with our previous studies where we identified that caspase‐3‐dependent processing of type III secreted effectors plays an important role in Salmonella pathogenesis (Srikanth et al., 2010). Of note, the SipA effector itself was found to be necessary and sufficient to promote activation of caspase‐3 (Srikanth et al., 2010) in a process independent of the apoptotic cascades. Given that we identified PERP to be a SipA interacting partner and prior studies have shown that PERP is linked to the activation of caspase‐3 (Davies et al., 2009), it is tempting to speculate that SipA‐induced caspase‐3 activity occurs through a PERP‐dependent pathway. Although we favour this hypothesis, we do not present evidence supporting this direct relationship, and therefore it remains possible that SipA could also trigger a PERP‐independent pathway to activate caspase‐3. Regardless, our data do suggest that the SipA caspase‐3 cleavage site is dispensable for PERP redistribution at the apical surface (Fig. 6F), indicating that caspase‐3 cleavage of SipA and the subsequent inflammatory events mediated by cleaved SipA (Srikanth et al., 2010) occur after PERP redistributes to the apical surface. Whether this indicates a direct role for PERP in mediating caspase‐3 cleavage of SipA remains to be determined.It is evident that infection with Salmonella prompts the accumulation of PERP at the apical surface and one mechanism that may account for the redistribution in PERP trafficking is subversion of the endosomal recycling pathway. The endosome recycling pathway has long been known to facilitate the shuttling of proteins, including junctional proteins (Lock and Stow, 2005), back and forth from intracellular to membrane locations, and plays a fundamental role in maintaining cellular polarity (reviewed in Perret et al., 2005; Golachowska et al., 2010). Endosomal pathways are well known to be involved in the response to Salmonella infection (Dukes et al., 2006; Bakowski et al., 2007; Brawn et al., 2007), and we found the staining patterns of Rab25, an apical recycling endosome marker and PERP both change with infection. We also observed PERP to co‐localize with Rab25 (Fig. S5), inviting speculation that Salmonella perturbs the cellular trafficking of PERP through a pathway involving the endosome recycling system. This hypothesis is supported by our previous studies, which demonstrate a requirement for ARF6 in S. Typhimurium‐induced PMN transepithelial migration and localization of this small GTPase to the apical site of bacterial entry (Criss et al., 2001). The nexus between these observations is that ARF6 is involved in the endocytosis and membrane recycling of a subset of membrane proteins, as well as in remodelling of the cortical actin cytoskeleton (D'Souza‐Schorey et al., 1995; Radhakrishna and Donaldson, 1997; Frank et al., 1998; Radhakrishna et al., 1999; Boshans et al., 2000). ARF6 is also highly expressed in polarized epithelial cells, where it localizes primarily to the apical brush border and apical early endosomes (Altschuler et al., 1999; Londono et al., 1999). Furthermore, Salmonella infection promotes exocyst formation at sites of invasion (Nichols and Casanova, 2010), which is thought to cause exocyst‐mediated docking of vesicles at this cellular location. Of particular interest to our current findings are studies that have previously documented the early endosomal marker Rab11 binds to components of the exocyst (reviewed in Heider and Munson, 2012), and that its localization is affected by the exocyst member Sec15 (Wu et al., 2005). As Rab25 is a Rab11 family member, we hypothesize that Salmonella‐mediated exocyst formation may induce the distorted trafficking of Rab25‐containing, and hence PERP‐containing, vesicles to the apical surface. However, further investigations targeting the trafficking pathways directing PERP accumulation to the apical surface will be required to validate this supposition.Our studies reveal a critical role for PERP in the pathogenesis of S. Typhimurium, and for the first time demonstrate that SipA, a T3SE protein, can engage a host protein at the epithelial surface. Therefore, more detailed investigations are required to further the understanding of the regulation underlying the SipA–PERP signaling mechanism including whether this interaction is direct or indirect. Examples of feedback and feed forward signaling, as evidenced by epidermal growth factor receptor tyrosine kinase activation and NOTCH activation, respectively, highlight the potential complexity involved in these cascades (Welsh et al., 1991; Caolo et al., 2010). Nevertheless, we propose a model (Fig. 8) that describes our observations for how PERP functions during Salmonella infection. As shown in path A in Fig. 8, we envisage S. Typhimurium infection induces increased expression of PERP. We propose this is due in part to Salmonella‐induced perturbation of endosome trafficking, which consequentially prevents PERP degradation. Next, increased PERP expression leads to an increase in cellular stores of activated caspase‐3. We have previously shown that intracellular caspase‐3 activates the iPLA2‐dependent cascade that leads to HXA3 synthesis (Mumy et al., 2008), thus linking our observations of PERP‐regulating caspase‐3 levels during infection to the inflammatory functions of PERP. Via path B in Fig. 8, SipA, likely in conjunction with other effectors, acts at the apical surface to trigger the redistribution of PERP, which we suspect to be via perturbation of endosomal networks in response to Salmonella infection. As shown in panel 2, at the apical surface, PERP facilitates the organization of a protein complex that binds to SipA as well as SipC. As shown in the third panel, we further propose that the protein complex functions to stabilize SipA at the apical surface such that it can be cleaved by caspase‐3 into its functional domains (Srikanth et al., 2010). The proinflammatory domain triggers the apical translocation of the ABC transporter, MRP2, which we have shown facilitates the apical secretion of HXA3 (Silva et al., 2004; Pazos et al., 2008; Agbor et al., 2011). Although there is still much to be learned about the role PERP plays in inflammatory conditions, we have now taken the first steps to show that this tetraspanning membrane protein plays a pivotal role in the pathogenesis of Salmonella infection.Figure 8Model for PERP's role in the inflammatory response to S
almonella infection. Left box: S
almonella invades the intestine and secretes SipA into the extracellular space. Infection induces a series of separate, although intertwined, PERP‐mediated events. Path A: Panel 1: WT. S
almonella infection induces increased expression of PERP. We propose this is due in part to the perturbation of endosome trafficking previously observed during S
almonella infection, thus preventing PERP degradation. Panel 2: Increased PERP expression leads to an increase in cellular stores of activated caspase‐3. Path B: Panel 1. SipA acts, likely in concert with other effectors, at the apical surface to trigger the redistribution of PERP. As described above, endosomal networks have been shown to be disrupted during S
almonella infection. Panel 2. At the apical surface, PERP facilitates the organization of a protein complex that binds to SipA and to SipC. Panel 3. We propose that the protein complex functions to stabilize SipA at the apical surface such that it can be cleaved by caspase‐3 into its functional domains.figure2014 John Wiley & Sons Ltd.

PERP is a tetraspanning membrane protein that belongs to the PMP‐22(Gas3)/EMP family (Attardi et al., 2000). Although PERP was first reported to be a downstream effector of p53 (Attardi et al., 2000), more recent studies have found PERP to play a critical role not only in maintaining epithelial barrier integrity (Ihrie et al., 2005) but also in regulating genes involved in inflammation (Beaudry et al., 2010). In the current study, we now identify a new role for PERP in the pathogenesis of the enteric pathogen S. Typhimurium. Of particular interest, we show that PERP associates with the S. Typhimurium type III secreted effector (T3SE) SipA and regulates PMN transmigration during infection.

Precisely how SipA initially interacts with PERP remains to be determined and our current efforts are focused on understanding the biochemistry of the SipA–PERP interactions including the domains responsible. Nevertheless, some inferences can be made based on our findings. One possibility is that PERP is part of a membrane complex. We reason this to be the case since in addition to PERP, we also identified four other potential SipA binding partners that were less represented in the Y2H screen (Table 1). Tetraspanning proteins are well documented to complex with other tetraspanins, integrins, immunoglobulin proteins, signaling enzymes or co‐receptors to impart a variety of functions (reviewed in Maecker et al., 1997; Hemler, 2001). Thus, it is perhaps not surprising that we have identified a role for PERP in intestinal inflammation. What is striking, however, is that many of the properties of PERP function appear to be consistent with the reported activities of SipA in triggering intestinal inflammation characterized by PMN transepithelial migration, raising the question of whether SipA subverts PERP functional activities.

The fact that PERP is involved in facilitating PMN transmigration in response to Salmonella infection is moreover consistent with previous studies showing that PERP regulates the expression of various inflammation‐associated gene products (Beaudry et al., 2010). Among these is Chi3L1, which is expressed in inflamed mucosa, particularly in Crohn's disease and ulcerative colitis patients, and appears to promote bacterial adhesion to colonic epithelial cells (Mizoguchi, 2006). PERP was also found to regulate Ccl20 (or MIP‐3‐alpha), which is expressed in intestinal epithelia associated with Peyer's patches and aids in the attraction of natural killer cells, memory T cells and immature dendritic cells to the site of inflammation (Hoover et al., 2002). Moreover, we have also found PERP to be increased in both a murine model of Salmonella colitis as well as in a dextran sodium sulfate chemically induced colitis model; in the former infection with the SipA mutant strain resulted in PERP expression levels that were similar to background control levels (Hallstrom and McCormick, unpubl. obs.).

The molecular mechanism by which PERP supports PMN transmigration is still under investigation. We are exploring the possibilities that either PERP interacts with a ligand or receptor on the surface of PMNs in order to enable their transmigration to the apical surface or activates (or de‐activates) signaling pathways that promote PMN transmigration (Chin et al., 2008). Unpublished observations from our laboratory have also shown that PERP is able to bind to itself and may be expressed on PMNs. Because PERP is known to localize to desmosomes, this raises the interesting possibility that PERP could facilitate PMN migration by promoting PMN interactions with junctional proteins expressed by intestinal epithelial cells. Such activity, if confirmed, would indicate PERP could have a significant role in other intestinal inflammatory conditions beyond that of Salmonella infection where PMN migration is a key pathological feature.

Our data also support the notion that PERP regulates caspase‐3 activation during Salmonella infection (Fig. 4). This observation is consistent with our previous studies where we identified that caspase‐3‐dependent processing of type III secreted effectors plays an important role in Salmonella pathogenesis (Srikanth et al., 2010). Of note, the SipA effector itself was found to be necessary and sufficient to promote activation of caspase‐3 (Srikanth et al., 2010) in a process independent of the apoptotic cascades. Given that we identified PERP to be a SipA interacting partner and prior studies have shown that PERP is linked to the activation of caspase‐3 (Davies et al., 2009), it is tempting to speculate that SipA‐induced caspase‐3 activity occurs through a PERP‐dependent pathway. Although we favour this hypothesis, we do not present evidence supporting this direct relationship, and therefore it remains possible that SipA could also trigger a PERP‐independent pathway to activate caspase‐3. Regardless, our data do suggest that the SipA caspase‐3 cleavage site is dispensable for PERP redistribution at the apical surface (Fig. 6F), indicating that caspase‐3 cleavage of SipA and the subsequent inflammatory events mediated by cleaved SipA (Srikanth et al., 2010) occur after PERP redistributes to the apical surface. Whether this indicates a direct role for PERP in mediating caspase‐3 cleavage of SipA remains to be determined.

It is evident that infection with Salmonella prompts the accumulation of PERP at the apical surface and one mechanism that may account for the redistribution in PERP trafficking is subversion of the endosomal recycling pathway. The endosome recycling pathway has long been known to facilitate the shuttling of proteins, including junctional proteins (Lock and Stow, 2005), back and forth from intracellular to membrane locations, and plays a fundamental role in maintaining cellular polarity (reviewed in Perret et al., 2005; Golachowska et al., 2010). Endosomal pathways are well known to be involved in the response to Salmonella infection (Dukes et al., 2006; Bakowski et al., 2007; Brawn et al., 2007), and we found the staining patterns of Rab25, an apical recycling endosome marker and PERP both change with infection. We also observed PERP to co‐localize with Rab25 (Fig. S5), inviting speculation that Salmonella perturbs the cellular trafficking of PERP through a pathway involving the endosome recycling system. This hypothesis is supported by our previous studies, which demonstrate a requirement for ARF6 in S. Typhimurium‐induced PMN transepithelial migration and localization of this small GTPase to the apical site of bacterial entry (Criss et al., 2001). The nexus between these observations is that ARF6 is involved in the endocytosis and membrane recycling of a subset of membrane proteins, as well as in remodelling of the cortical actin cytoskeleton (D'Souza‐Schorey et al., 1995; Radhakrishna and Donaldson, 1997; Frank et al., 1998; Radhakrishna et al., 1999; Boshans et al., 2000). ARF6 is also highly expressed in polarized epithelial cells, where it localizes primarily to the apical brush border and apical early endosomes (Altschuler et al., 1999; Londono et al., 1999). Furthermore, Salmonella infection promotes exocyst formation at sites of invasion (Nichols and Casanova, 2010), which is thought to cause exocyst‐mediated docking of vesicles at this cellular location. Of particular interest to our current findings are studies that have previously documented the early endosomal marker Rab11 binds to components of the exocyst (reviewed in Heider and Munson, 2012), and that its localization is affected by the exocyst member Sec15 (Wu et al., 2005). As Rab25 is a Rab11 family member, we hypothesize that Salmonella‐mediated exocyst formation may induce the distorted trafficking of Rab25‐containing, and hence PERP‐containing, vesicles to the apical surface. However, further investigations targeting the trafficking pathways directing PERP accumulation to the apical surface will be required to validate this supposition.

Our studies reveal a critical role for PERP in the pathogenesis of S. Typhimurium, and for the first time demonstrate that SipA, a T3SE protein, can engage a host protein at the epithelial surface. Therefore, more detailed investigations are required to further the understanding of the regulation underlying the SipA–PERP signaling mechanism including whether this interaction is direct or indirect. Examples of feedback and feed forward signaling, as evidenced by epidermal growth factor receptor tyrosine kinase activation and NOTCH activation, respectively, highlight the potential complexity involved in these cascades (Welsh et al., 1991; Caolo et al., 2010). Nevertheless, we propose a model (Fig. 8) that describes our observations for how PERP functions during Salmonella infection. As shown in path A in Fig. 8, we envisage S. Typhimurium infection induces increased expression of PERP. We propose this is due in part to Salmonella‐induced perturbation of endosome trafficking, which consequentially prevents PERP degradation. Next, increased PERP expression leads to an increase in cellular stores of activated caspase‐3. We have previously shown that intracellular caspase‐3 activates the iPLA2‐dependent cascade that leads to HXA3 synthesis (Mumy et al., 2008), thus linking our observations of PERP‐regulating caspase‐3 levels during infection to the inflammatory functions of PERP. Via path B in Fig. 8, SipA, likely in conjunction with other effectors, acts at the apical surface to trigger the redistribution of PERP, which we suspect to be via perturbation of endosomal networks in response to Salmonella infection. As shown in panel 2, at the apical surface, PERP facilitates the organization of a protein complex that binds to SipA as well as SipC. As shown in the third panel, we further propose that the protein complex functions to stabilize SipA at the apical surface such that it can be cleaved by caspase‐3 into its functional domains (Srikanth et al., 2010). The proinflammatory domain triggers the apical translocation of the ABC transporter, MRP2, which we have shown facilitates the apical secretion of HXA3 (Silva et al., 2004; Pazos et al., 2008; Agbor et al., 2011). Although there is still much to be learned about the role PERP plays in inflammatory conditions, we have now taken the first steps to show that this tetraspanning membrane protein plays a pivotal role in the pathogenesis of Salmonella infection.

Model for PERP's role in the inflammatory response to S
almonella infection. Left box: S
almonella invades the intestine and secretes SipA into the extracellular space. Infection induces a series of separate, although intertwined, PERP‐mediated events. Path A: Panel 1: WT. S
almonella infection induces increased expression of PERP. We propose this is due in part to the perturbation of endosome trafficking previously observed during S
almonella infection, thus preventing PERP degradation. Panel 2: Increased PERP expression leads to an increase in cellular stores of activated caspase‐3. Path B: Panel 1. SipA acts, likely in concert with other effectors, at the apical surface to trigger the redistribution of PERP. As described above, endosomal networks have been shown to be disrupted during S
almonella infection. Panel 2. At the apical surface, PERP facilitates the organization of a protein complex that binds to SipA and to SipC. Panel 3. We propose that the protein complex functions to stabilize SipA at the apical surface such that it can be cleaved by caspase‐3 into its functional domains.

Experimental proceduresTissue cultureT84‐ or HCT8‐polarized monolayers were grown on polycarbonate filters and used 6–8 days after plating. Inverted monolayers (Costar 3421) were used for PMN transmigration assays. Non‐inverted monolayers (Costar 3421) were used for microscopy. For biotinylation, cells were seeded on transwells in 100 mm tissue culture dishes (Costar 3419). For co‐immunoprecipitations and time course assays, cells were seeded on transwells in 6‐well plates (Costar 3412).Use of bacterial strains
S. Typhimurium strains (SL1344, wild type; EE633, SipA deficient) were grown as previously described (Lee et al., 2000). SipB‐deficient (ΔSipB) S. Typhimurium was grown in the same manner as the SipA‐deficient strain (Wall et al., 2007). pSipA (SipA complemented), AJK63 (expresses HA‐tagged SipA) and CSM (SipA caspase‐3 site mutant) were grown in the presence of 50 μg ml−1 ampicillin. Unless otherwise indicated, cells were infected at an MOI of 100:1 for 1 h. The pET3a‐GST plasmid containing the GST‐tagged C‐terminus of SipC (Nichols and Casanova, 2010) was transformed into BL21 cells and maintained in the presence of 50 μg ml−1 ampicillin.
Western blottingLysates were prepared in whole cell lysis buffer [150 mM NaCl; 25 mM Tris, pH 8; 1 mM ethylenediaminetetraacetic acid; 1% NP‐40; 5 mM Na3VO4, 20 mM NaF, 0.8 mM phenylmethanesulphonyl fluoride (PMSF) and protease inhibitor cocktail]. Homogenized supernatants were normalized for protein concentration, boiled in loading dye supplemented with β‐mercaptoethanol, separated by sodium dodecyl sulfate‐polyacrylamide gel electrophoresis and immunoblotted for the desired proteins. β‐Actin (Sigma) and glyceraldehyde 3‐phosphate dehydrogenase (Millipore) were used as loading controls.
PERP siRNA construct designThe pSUPER vector (Oligoengine) was used to generate a PERP siRNA construct as in Brummelkamp et al. (2002). Oligonucleotides contained a specific human PERP sequence (GI: 222080101: 184–765), its reverse complement (in italics) separated by a short spacer region and BglII or HindIII restriction sites. PERPKO_F1GATCCCC AAGATGACCTTCTGGGCAA TTCAAGAGA TTGCCCAGAAGGTCATCTT TTTTTGGAAA and PERPKO_R1 AGCTTTTCCAAAAA AAGATGACCTTCTGGGCAA TCTCTTGAA TTGCCCAGAAGGTCATCTT GGG and for a random control sequence, 5′‐GATCCCCCCGACAAGCTTGAATTTATTTCAAGAGAATAAATTCAAGCTTGTCGGTTTTTGGAAA‐3′ and 5′ AGCTTTTCCAAAAACCGACAAGCTTGAATTTATTCTCTTGAAATAAATTCAAGCTTGTCGGGGG‐3′.Transfection of HCT8 intestinal epithelial cellsFor stable PERP knockdowns, cells were transfected with the modified pSUPER using Lipofectamine 2000 (Invitrogen) per manufacturer's instructions in Roswell Park Memorial Institute (RPMI) 1640 without serum (Invitrogen), incubated in RPMI with 8% v/v fetal bovine serum then passaged into fresh media with neomycin‐G418 (Sigma‐Aldrich). Cells underwent two additional cycles of growth/passage in G418 prior to use. For transient PERP knockdowns, siRNA against human PERP and a non‐targeting pool were obtained from Dharmacon. HCT8s were transfected with 20 nM siRNA using Lipofectamine 2000 in OptiMem serum‐free media for 24 h.Split ubiquitin Y2H screenThe dual‐hunter split ubiquitin Y2H kit was used (Dualsystems Biotech AG). Coding DNA for SipA was cloned into the BAIT plasmid (pDHB1) and transformed into yeast reporter strain NMY51. A human colonic cDNA library (Dualsystems Biotech AG) was transformed into the bait‐expressing yeast per manufacturer's protocols. For the reverse Y2H assay, the coding DNA of PERP was cloned into pDHB1 while SipA was cloned into the PREY plasmid (pPR3‐N).
HA‐tagged SipA isolationAn overnight culture of AJK63 (S. Typhimurium SL1344 expressing HA‐tagged SipA) was back‐diluted then centrifuged at 6000 r.p.m. The supernatant was passed through an Amicon Centrifugal Filter Unit (Millipore UFC900324). We kept the volume left in the top chamber and added one tablet of Protease Inhibitor Complete Mini (Roche). We then prepped the HA column by adding 0.5 ml of HA‐Affinity matrix (Roche) and equilibrated per manufacturer instructions. The sample was then run through the column, followed by a washing step. Finally, the HA‐tagged protein was eluted with 1 mg ml−1 of HA peptide. Samples were analysed for concentration and stored at −80°C.ImmunoprecipitationsNormalized lysates from T84 cells infected with AJK63 or ΔSifA/SifA‐pBH (Salmonella Typhimurium SL1344 expressing HA‐tagged SifA) were immunoprecipitated for HA‐SipA or HA‐SifA, respectfully, using protein A/G agarose plus beads (Santa Cruz) and anti‐HA or IgG control antibodies (Abcam). The presence of PERP was determined via Western blot.The SipC‐PERP pull downs were performed in accordance with Nichols and Casanova (2010) with minor modifications. An overnight culture of BL21 expressing the pET3a‐GST plasmid containing the GST‐tagged C‐terminus of SipC was centrifuged at 6000 r.p.m. The pellet was resuspended in lysis buffer [25 mM Tris, 3 mM dithiothreitol, 1 mM PMSF], sonicated and centrifuged at 14 000 r.p.m. at 4°C for 1 h. The pre‐cleared supernatants were then incubated with glutathione sepharose 4B affinity matrix beads (GE Healthcare) prepared according to manufacturer instructions for 2 h at room temperature. Whole cell lysates from HCT8 cells were then incubated with the SipC‐GST‐bound beads over night at 4°C with end‐over‐end rotation. After washing steps with 1× phosphate buffered saline (PBS), the GST‐SipC protein complexes were eluted with reduced glutathione. The eluates were then diluted in 4× tricine loading dye, boiled and examined via Western blot for the presence of SipC‐GST (not shown) and PERP.BiotinylationApical cell surface biotinylation was performed using the protocol described by Agbor et al. (2011). Following infection, the apical surface of HCT8 monolayers was labeled with biotin at 4°C. Labeling of the basolateral surface was blocked with acetate. The cells were then lysed, normalized and incubated with streptavidin beads in order to pull down apically labeled proteins. The apically enriched lysates were then immunoblotted for PERP (Santa Cruz SC‐67184) or E‐cadherin (Santa Cruz SC‐7870). For basolateral surface biotinylation, the same protocol was followed with reversal of the biotin and acetate application. For brefeldin A experiments, cells were exposed to 150 μM brefeldin A in Hank's buffered salt solution (HBSS) + for 1 h prior to infection. The brefeldin A was then removed, and the cells were washed once prior to infection. For cycloheximide experiments, cells were exposed to 2 μg ml−1 of cycloheximide in HBSS + for 1 h prior to infection. The cycloheximide was then removed, and the cells were washed once prior to infection.
PMN transepithelial migration assaysPMN migration assays were carried out as described (McCormick et al., 1993) using p11 (PERP knockdown) and p24 (vector control) monolayers. PERP blocking was performed according to Zen et al. (2004) with some modifications. HCT8 cells were infected at the apical surface with SL1344 for 40 min. After washing, 25 μg ml−1 of antibodies for PERP (Santa Cruz), IgG control (Abcam) or the mitochondrial marker MTCO‐1 (Abcam) were added to the basolateral surface for 30 min prior to addition of PMNs and maintained during migration. Where indicated, fMLP, IL‐8 or LTB4 was used to induce migration in the absence of infection. Values were normalized to infected, untreated samples or to chemoattractant, untreated samples (set to 100%).Activated caspase‐3 assayFollowing transient PERP knockdown, cells were infected for 2 h then lysed and analysed for caspase‐3 activity via the BioVision colorimetric caspase‐3 activity assay per manufacturer's instructions (BioVision).Fluorescent wide‐field microscopyFor examination of PERP accumulation, T84 monolayers were grown on permeable filters and infected with SL1344, EE633, CSM, pSipA, treated with HA‐tagged SipA or left in HBSS + buffer (non‐infected) for 1 h. The filters were washed in 1% PBS, fixed with 1% paraformaldehyde (PFA) in PBS for 15 min, quenched with NH4Cl in PBS for 15 min, then permeabilized in 0.1% Triton in PBS for 5 min, with washing steps in between. The filters were then blocked with 5% NGS in PBS for 1 h, followed by staining with primary antibody against PERP (Abcam 5986) overnight at 4°C. The filters were stained the next day with secondary Alexa Fluor 488 (Life Technologies), Alexa Fluor 568 (Life Technologies) and phalloidin Alexa Fluor 647 (Life Technologies) for 1 h at room temperature in the dark. Filters were then mounted using SlowFade Gold with 4′, 6‐diamidino‐2‐phenylindole (fluorescent stain for nuclei) (DAPI) and maintained in the dark at 4°C. Immunofluorescence samples were imaged using a Nikon Ti‐E wide‐field fluorescent microscope (Nikon Instruments) with a 60× objective using a Photometrics QuantEM wide‐field camera at room temperature. Widefield Z‐stacks were taken with 0.4 μm Z slices using the filter pores to differentiate the basolateral from the apical surface of the monolayer. The entire monolayer was imaged in this manner, with at least five random distinct areas imaged for each sample. Images were acquired with the Nikon Elements SW version 4.13 software. Quantification of the PERP staining pattern was performed with Z volume projections processed using the Nikon Elements SW version 4.13 software, encompassing the whole monolayer. The level of punctate staining was quantified using FIJI to count puncta in each image. Four areas for each condition per experiment were quantified in this manner. The fold change over baseline for each condition was averaged across three experiments. To better determine apical or basolateral location of the punctae, a line was added to the Z projections during processing to bisect the Z volume of the monolayer.Confocal microscopyMouse colon tissueSections of the proximal colons from 6‐week‐old C57BL/6 mice were removed and snap frozen in OCT media, then cut into 5 μm sections on glass slides. Sections were fixed in 4% PFA, quenched with 50 mM ammonium chloride, then permeabilized with 0.5% Triton X in PBS. Sections were then blocked with blocking buffer (5% normal goat serum in PBS) for 1 h at room temperature, followed by overnight incubation with anti‐PERP antibody (Abcam 5986) in blocking buffer at 4°C. The next day, the sections were washed with blocking buffer, then incubated with secondary Alexa Fluor 488 (Life Technologies) at room temperature for 1 h. Sections were then washed in blocking buffer, mounted with SlowFade Gold with DAPI and viewed under a Leica TCS SP‐5 confocal microscope (Leica Microsystems) using a 40× oil objective with 1× digital zoom (Leica LASAF Software, Leica Microsystems). All samples were imaged as 0.2 μm Z‐stacks. Images were processed using FIJI (NIH). Animals were treated in accordance with institutional IACUC protocols.
PERP and Rab25 co‐localizationPolarized T84 monolayers were infected with SL1344 for 1 h and stained as described above for PERP (Abcam 5986) and Rab25 (Abcam 32004). Determination of PERP co‐localization with Rab25 was performed using a Leica TCS SP‐5 confocal microscope (Leica Microsystems) using a 63× oil objective with 6× digital zoom (Leica LASAF Software, Leica Microsystems). For increased resolution of PERP localization, pinhole was decreased to 0.5 airy units for all imaging and all samples were imaged as 0.2 μm Z‐stacks. Images shown are representative of three images taken from random fields per sample. Post‐imaging, images were processed using FIJI (NIH) with single 0.2 μm slices selected from the quarter most apical sections to show co‐localization of Rab25 (AlexaFluor 568, red pseudocolour) and PERP (AlexaFluor 488, green pseudocolour), along with F‐actin (phalloidin AlexaFluor 647, blue pseudocolour) to show cellular structure. The level of co‐localization was determined with Manders' coefficient analyses in FIJI.Data presentationImages are presented as one representative of at least three experiments showing reproducible trends. P‐values were calculated using the Student's t‐test, and values of < 0.05 were considered statistically significant. In cases where datasets contained more than two groups, one‐way analyses of variance were performed first, followed by individual Student's t‐test analyses to determine which treatment groups differed from the control. Error bars represent standard error.

Experimental procedures

Tissue cultureT84‐ or HCT8‐polarized monolayers were grown on polycarbonate filters and used 6–8 days after plating. Inverted monolayers (Costar 3421) were used for PMN transmigration assays. Non‐inverted monolayers (Costar 3421) were used for microscopy. For biotinylation, cells were seeded on transwells in 100 mm tissue culture dishes (Costar 3419). For co‐immunoprecipitations and time course assays, cells were seeded on transwells in 6‐well plates (Costar 3412).

Tissue culture

T84‐ or HCT8‐polarized monolayers were grown on polycarbonate filters and used 6–8 days after plating. Inverted monolayers (Costar 3421) were used for PMN transmigration assays. Non‐inverted monolayers (Costar 3421) were used for microscopy. For biotinylation, cells were seeded on transwells in 100 mm tissue culture dishes (Costar 3419). For co‐immunoprecipitations and time course assays, cells were seeded on transwells in 6‐well plates (Costar 3412).

Use of bacterial strains
S. Typhimurium strains (SL1344, wild type; EE633, SipA deficient) were grown as previously described (Lee et al., 2000). SipB‐deficient (ΔSipB) S. Typhimurium was grown in the same manner as the SipA‐deficient strain (Wall et al., 2007). pSipA (SipA complemented), AJK63 (expresses HA‐tagged SipA) and CSM (SipA caspase‐3 site mutant) were grown in the presence of 50 μg ml−1 ampicillin. Unless otherwise indicated, cells were infected at an MOI of 100:1 for 1 h. The pET3a‐GST plasmid containing the GST‐tagged C‐terminus of SipC (Nichols and Casanova, 2010) was transformed into BL21 cells and maintained in the presence of 50 μg ml−1 ampicillin.

Use of bacterial strains

S. Typhimurium strains (SL1344, wild type; EE633, SipA deficient) were grown as previously described (Lee et al., 2000). SipB‐deficient (ΔSipB) S. Typhimurium was grown in the same manner as the SipA‐deficient strain (Wall et al., 2007). pSipA (SipA complemented), AJK63 (expresses HA‐tagged SipA) and CSM (SipA caspase‐3 site mutant) were grown in the presence of 50 μg ml−1 ampicillin. Unless otherwise indicated, cells were infected at an MOI of 100:1 for 1 h. The pET3a‐GST plasmid containing the GST‐tagged C‐terminus of SipC (Nichols and Casanova, 2010) was transformed into BL21 cells and maintained in the presence of 50 μg ml−1 ampicillin.

Western blottingLysates were prepared in whole cell lysis buffer [150 mM NaCl; 25 mM Tris, pH 8; 1 mM ethylenediaminetetraacetic acid; 1% NP‐40; 5 mM Na3VO4, 20 mM NaF, 0.8 mM phenylmethanesulphonyl fluoride (PMSF) and protease inhibitor cocktail]. Homogenized supernatants were normalized for protein concentration, boiled in loading dye supplemented with β‐mercaptoethanol, separated by sodium dodecyl sulfate‐polyacrylamide gel electrophoresis and immunoblotted for the desired proteins. β‐Actin (Sigma) and glyceraldehyde 3‐phosphate dehydrogenase (Millipore) were used as loading controls.

Western blotting

Lysates were prepared in whole cell lysis buffer [150 mM NaCl; 25 mM Tris, pH 8; 1 mM ethylenediaminetetraacetic acid; 1% NP‐40; 5 mM Na3VO4, 20 mM NaF, 0.8 mM phenylmethanesulphonyl fluoride (PMSF) and protease inhibitor cocktail]. Homogenized supernatants were normalized for protein concentration, boiled in loading dye supplemented with β‐mercaptoethanol, separated by sodium dodecyl sulfate‐polyacrylamide gel electrophoresis and immunoblotted for the desired proteins. β‐Actin (Sigma) and glyceraldehyde 3‐phosphate dehydrogenase (Millipore) were used as loading controls.

PERP siRNA construct designThe pSUPER vector (Oligoengine) was used to generate a PERP siRNA construct as in Brummelkamp et al. (2002). Oligonucleotides contained a specific human PERP sequence (GI: 222080101: 184–765), its reverse complement (in italics) separated by a short spacer region and BglII or HindIII restriction sites. PERPKO_F1GATCCCC AAGATGACCTTCTGGGCAA TTCAAGAGA TTGCCCAGAAGGTCATCTT TTTTTGGAAA and PERPKO_R1 AGCTTTTCCAAAAA AAGATGACCTTCTGGGCAA TCTCTTGAA TTGCCCAGAAGGTCATCTT GGG and for a random control sequence, 5′‐GATCCCCCCGACAAGCTTGAATTTATTTCAAGAGAATAAATTCAAGCTTGTCGGTTTTTGGAAA‐3′ and 5′ AGCTTTTCCAAAAACCGACAAGCTTGAATTTATTCTCTTGAAATAAATTCAAGCTTGTCGGGGG‐3′.

PERP siRNA construct design

The pSUPER vector (Oligoengine) was used to generate a PERP siRNA construct as in Brummelkamp et al. (2002). Oligonucleotides contained a specific human PERP sequence (GI: 222080101: 184–765), its reverse complement (in italics) separated by a short spacer region and BglII or HindIII restriction sites. PERPKO_F1GATCCCC AAGATGACCTTCTGGGCAA TTCAAGAGA TTGCCCAGAAGGTCATCTT TTTTTGGAAA and PERPKO_R1 AGCTTTTCCAAAAA AAGATGACCTTCTGGGCAA TCTCTTGAA TTGCCCAGAAGGTCATCTT GGG and for a random control sequence, 5′‐GATCCCCCCGACAAGCTTGAATTTATTTCAAGAGAATAAATTCAAGCTTGTCGGTTTTTGGAAA‐3′ and 5′ AGCTTTTCCAAAAACCGACAAGCTTGAATTTATTCTCTTGAAATAAATTCAAGCTTGTCGGGGG‐3′.

Transfection of HCT8 intestinal epithelial cellsFor stable PERP knockdowns, cells were transfected with the modified pSUPER using Lipofectamine 2000 (Invitrogen) per manufacturer's instructions in Roswell Park Memorial Institute (RPMI) 1640 without serum (Invitrogen), incubated in RPMI with 8% v/v fetal bovine serum then passaged into fresh media with neomycin‐G418 (Sigma‐Aldrich). Cells underwent two additional cycles of growth/passage in G418 prior to use. For transient PERP knockdowns, siRNA against human PERP and a non‐targeting pool were obtained from Dharmacon. HCT8s were transfected with 20 nM siRNA using Lipofectamine 2000 in OptiMem serum‐free media for 24 h.

Transfection of HCT8 intestinal epithelial cells

For stable PERP knockdowns, cells were transfected with the modified pSUPER using Lipofectamine 2000 (Invitrogen) per manufacturer's instructions in Roswell Park Memorial Institute (RPMI) 1640 without serum (Invitrogen), incubated in RPMI with 8% v/v fetal bovine serum then passaged into fresh media with neomycin‐G418 (Sigma‐Aldrich). Cells underwent two additional cycles of growth/passage in G418 prior to use. For transient PERP knockdowns, siRNA against human PERP and a non‐targeting pool were obtained from Dharmacon. HCT8s were transfected with 20 nM siRNA using Lipofectamine 2000 in OptiMem serum‐free media for 24 h.

Split ubiquitin Y2H screenThe dual‐hunter split ubiquitin Y2H kit was used (Dualsystems Biotech AG). Coding DNA for SipA was cloned into the BAIT plasmid (pDHB1) and transformed into yeast reporter strain NMY51. A human colonic cDNA library (Dualsystems Biotech AG) was transformed into the bait‐expressing yeast per manufacturer's protocols. For the reverse Y2H assay, the coding DNA of PERP was cloned into pDHB1 while SipA was cloned into the PREY plasmid (pPR3‐N).

Split ubiquitin Y2H screen

The dual‐hunter split ubiquitin Y2H kit was used (Dualsystems Biotech AG). Coding DNA for SipA was cloned into the BAIT plasmid (pDHB1) and transformed into yeast reporter strain NMY51. A human colonic cDNA library (Dualsystems Biotech AG) was transformed into the bait‐expressing yeast per manufacturer's protocols. For the reverse Y2H assay, the coding DNA of PERP was cloned into pDHB1 while SipA was cloned into the PREY plasmid (pPR3‐N).

HA‐tagged SipA isolationAn overnight culture of AJK63 (S. Typhimurium SL1344 expressing HA‐tagged SipA) was back‐diluted then centrifuged at 6000 r.p.m. The supernatant was passed through an Amicon Centrifugal Filter Unit (Millipore UFC900324). We kept the volume left in the top chamber and added one tablet of Protease Inhibitor Complete Mini (Roche). We then prepped the HA column by adding 0.5 ml of HA‐Affinity matrix (Roche) and equilibrated per manufacturer instructions. The sample was then run through the column, followed by a washing step. Finally, the HA‐tagged protein was eluted with 1 mg ml−1 of HA peptide. Samples were analysed for concentration and stored at −80°C.

HA‐tagged SipA isolation

An overnight culture of AJK63 (S. Typhimurium SL1344 expressing HA‐tagged SipA) was back‐diluted then centrifuged at 6000 r.p.m. The supernatant was passed through an Amicon Centrifugal Filter Unit (Millipore UFC900324). We kept the volume left in the top chamber and added one tablet of Protease Inhibitor Complete Mini (Roche). We then prepped the HA column by adding 0.5 ml of HA‐Affinity matrix (Roche) and equilibrated per manufacturer instructions. The sample was then run through the column, followed by a washing step. Finally, the HA‐tagged protein was eluted with 1 mg ml−1 of HA peptide. Samples were analysed for concentration and stored at −80°C.

ImmunoprecipitationsNormalized lysates from T84 cells infected with AJK63 or ΔSifA/SifA‐pBH (Salmonella Typhimurium SL1344 expressing HA‐tagged SifA) were immunoprecipitated for HA‐SipA or HA‐SifA, respectfully, using protein A/G agarose plus beads (Santa Cruz) and anti‐HA or IgG control antibodies (Abcam). The presence of PERP was determined via Western blot.The SipC‐PERP pull downs were performed in accordance with Nichols and Casanova (2010) with minor modifications. An overnight culture of BL21 expressing the pET3a‐GST plasmid containing the GST‐tagged C‐terminus of SipC was centrifuged at 6000 r.p.m. The pellet was resuspended in lysis buffer [25 mM Tris, 3 mM dithiothreitol, 1 mM PMSF], sonicated and centrifuged at 14 000 r.p.m. at 4°C for 1 h. The pre‐cleared supernatants were then incubated with glutathione sepharose 4B affinity matrix beads (GE Healthcare) prepared according to manufacturer instructions for 2 h at room temperature. Whole cell lysates from HCT8 cells were then incubated with the SipC‐GST‐bound beads over night at 4°C with end‐over‐end rotation. After washing steps with 1× phosphate buffered saline (PBS), the GST‐SipC protein complexes were eluted with reduced glutathione. The eluates were then diluted in 4× tricine loading dye, boiled and examined via Western blot for the presence of SipC‐GST (not shown) and PERP.

Immunoprecipitations

Normalized lysates from T84 cells infected with AJK63 or ΔSifA/SifA‐pBH (Salmonella Typhimurium SL1344 expressing HA‐tagged SifA) were immunoprecipitated for HA‐SipA or HA‐SifA, respectfully, using protein A/G agarose plus beads (Santa Cruz) and anti‐HA or IgG control antibodies (Abcam). The presence of PERP was determined via Western blot.

The SipC‐PERP pull downs were performed in accordance with Nichols and Casanova (2010) with minor modifications. An overnight culture of BL21 expressing the pET3a‐GST plasmid containing the GST‐tagged C‐terminus of SipC was centrifuged at 6000 r.p.m. The pellet was resuspended in lysis buffer [25 mM Tris, 3 mM dithiothreitol, 1 mM PMSF], sonicated and centrifuged at 14 000 r.p.m. at 4°C for 1 h. The pre‐cleared supernatants were then incubated with glutathione sepharose 4B affinity matrix beads (GE Healthcare) prepared according to manufacturer instructions for 2 h at room temperature. Whole cell lysates from HCT8 cells were then incubated with the SipC‐GST‐bound beads over night at 4°C with end‐over‐end rotation. After washing steps with 1× phosphate buffered saline (PBS), the GST‐SipC protein complexes were eluted with reduced glutathione. The eluates were then diluted in 4× tricine loading dye, boiled and examined via Western blot for the presence of SipC‐GST (not shown) and PERP.

BiotinylationApical cell surface biotinylation was performed using the protocol described by Agbor et al. (2011). Following infection, the apical surface of HCT8 monolayers was labeled with biotin at 4°C. Labeling of the basolateral surface was blocked with acetate. The cells were then lysed, normalized and incubated with streptavidin beads in order to pull down apically labeled proteins. The apically enriched lysates were then immunoblotted for PERP (Santa Cruz SC‐67184) or E‐cadherin (Santa Cruz SC‐7870). For basolateral surface biotinylation, the same protocol was followed with reversal of the biotin and acetate application. For brefeldin A experiments, cells were exposed to 150 μM brefeldin A in Hank's buffered salt solution (HBSS) + for 1 h prior to infection. The brefeldin A was then removed, and the cells were washed once prior to infection. For cycloheximide experiments, cells were exposed to 2 μg ml−1 of cycloheximide in HBSS + for 1 h prior to infection. The cycloheximide was then removed, and the cells were washed once prior to infection.

Biotinylation

Apical cell surface biotinylation was performed using the protocol described by Agbor et al. (2011). Following infection, the apical surface of HCT8 monolayers was labeled with biotin at 4°C. Labeling of the basolateral surface was blocked with acetate. The cells were then lysed, normalized and incubated with streptavidin beads in order to pull down apically labeled proteins. The apically enriched lysates were then immunoblotted for PERP (Santa Cruz SC‐67184) or E‐cadherin (Santa Cruz SC‐7870). For basolateral surface biotinylation, the same protocol was followed with reversal of the biotin and acetate application. For brefeldin A experiments, cells were exposed to 150 μM brefeldin A in Hank's buffered salt solution (HBSS) + for 1 h prior to infection. The brefeldin A was then removed, and the cells were washed once prior to infection. For cycloheximide experiments, cells were exposed to 2 μg ml−1 of cycloheximide in HBSS + for 1 h prior to infection. The cycloheximide was then removed, and the cells were washed once prior to infection.

PMN transepithelial migration assaysPMN migration assays were carried out as described (McCormick et al., 1993) using p11 (PERP knockdown) and p24 (vector control) monolayers. PERP blocking was performed according to Zen et al. (2004) with some modifications. HCT8 cells were infected at the apical surface with SL1344 for 40 min. After washing, 25 μg ml−1 of antibodies for PERP (Santa Cruz), IgG control (Abcam) or the mitochondrial marker MTCO‐1 (Abcam) were added to the basolateral surface for 30 min prior to addition of PMNs and maintained during migration. Where indicated, fMLP, IL‐8 or LTB4 was used to induce migration in the absence of infection. Values were normalized to infected, untreated samples or to chemoattractant, untreated samples (set to 100%).

PMN transepithelial migration assays

PMN migration assays were carried out as described (McCormick et al., 1993) using p11 (PERP knockdown) and p24 (vector control) monolayers. PERP blocking was performed according to Zen et al. (2004) with some modifications. HCT8 cells were infected at the apical surface with SL1344 for 40 min. After washing, 25 μg ml−1 of antibodies for PERP (Santa Cruz), IgG control (Abcam) or the mitochondrial marker MTCO‐1 (Abcam) were added to the basolateral surface for 30 min prior to addition of PMNs and maintained during migration. Where indicated, fMLP, IL‐8 or LTB4 was used to induce migration in the absence of infection. Values were normalized to infected, untreated samples or to chemoattractant, untreated samples (set to 100%).

Activated caspase‐3 assayFollowing transient PERP knockdown, cells were infected for 2 h then lysed and analysed for caspase‐3 activity via the BioVision colorimetric caspase‐3 activity assay per manufacturer's instructions (BioVision).

Activated caspase‐3 assay

Following transient PERP knockdown, cells were infected for 2 h then lysed and analysed for caspase‐3 activity via the BioVision colorimetric caspase‐3 activity assay per manufacturer's instructions (BioVision).

Fluorescent wide‐field microscopyFor examination of PERP accumulation, T84 monolayers were grown on permeable filters and infected with SL1344, EE633, CSM, pSipA, treated with HA‐tagged SipA or left in HBSS + buffer (non‐infected) for 1 h. The filters were washed in 1% PBS, fixed with 1% paraformaldehyde (PFA) in PBS for 15 min, quenched with NH4Cl in PBS for 15 min, then permeabilized in 0.1% Triton in PBS for 5 min, with washing steps in between. The filters were then blocked with 5% NGS in PBS for 1 h, followed by staining with primary antibody against PERP (Abcam 5986) overnight at 4°C. The filters were stained the next day with secondary Alexa Fluor 488 (Life Technologies), Alexa Fluor 568 (Life Technologies) and phalloidin Alexa Fluor 647 (Life Technologies) for 1 h at room temperature in the dark. Filters were then mounted using SlowFade Gold with 4′, 6‐diamidino‐2‐phenylindole (fluorescent stain for nuclei) (DAPI) and maintained in the dark at 4°C. Immunofluorescence samples were imaged using a Nikon Ti‐E wide‐field fluorescent microscope (Nikon Instruments) with a 60× objective using a Photometrics QuantEM wide‐field camera at room temperature. Widefield Z‐stacks were taken with 0.4 μm Z slices using the filter pores to differentiate the basolateral from the apical surface of the monolayer. The entire monolayer was imaged in this manner, with at least five random distinct areas imaged for each sample. Images were acquired with the Nikon Elements SW version 4.13 software. Quantification of the PERP staining pattern was performed with Z volume projections processed using the Nikon Elements SW version 4.13 software, encompassing the whole monolayer. The level of punctate staining was quantified using FIJI to count puncta in each image. Four areas for each condition per experiment were quantified in this manner. The fold change over baseline for each condition was averaged across three experiments. To better determine apical or basolateral location of the punctae, a line was added to the Z projections during processing to bisect the Z volume of the monolayer.

Fluorescent wide‐field microscopy

For examination of PERP accumulation, T84 monolayers were grown on permeable filters and infected with SL1344, EE633, CSM, pSipA, treated with HA‐tagged SipA or left in HBSS + buffer (non‐infected) for 1 h. The filters were washed in 1% PBS, fixed with 1% paraformaldehyde (PFA) in PBS for 15 min, quenched with NH4Cl in PBS for 15 min, then permeabilized in 0.1% Triton in PBS for 5 min, with washing steps in between. The filters were then blocked with 5% NGS in PBS for 1 h, followed by staining with primary antibody against PERP (Abcam 5986) overnight at 4°C. The filters were stained the next day with secondary Alexa Fluor 488 (Life Technologies), Alexa Fluor 568 (Life Technologies) and phalloidin Alexa Fluor 647 (Life Technologies) for 1 h at room temperature in the dark. Filters were then mounted using SlowFade Gold with 4′, 6‐diamidino‐2‐phenylindole (fluorescent stain for nuclei) (DAPI) and maintained in the dark at 4°C. Immunofluorescence samples were imaged using a Nikon Ti‐E wide‐field fluorescent microscope (Nikon Instruments) with a 60× objective using a Photometrics QuantEM wide‐field camera at room temperature. Widefield Z‐stacks were taken with 0.4 μm Z slices using the filter pores to differentiate the basolateral from the apical surface of the monolayer. The entire monolayer was imaged in this manner, with at least five random distinct areas imaged for each sample. Images were acquired with the Nikon Elements SW version 4.13 software. Quantification of the PERP staining pattern was performed with Z volume projections processed using the Nikon Elements SW version 4.13 software, encompassing the whole monolayer. The level of punctate staining was quantified using FIJI to count puncta in each image. Four areas for each condition per experiment were quantified in this manner. The fold change over baseline for each condition was averaged across three experiments. To better determine apical or basolateral location of the punctae, a line was added to the Z projections during processing to bisect the Z volume of the monolayer.

Confocal microscopyMouse colon tissueSections of the proximal colons from 6‐week‐old C57BL/6 mice were removed and snap frozen in OCT media, then cut into 5 μm sections on glass slides. Sections were fixed in 4% PFA, quenched with 50 mM ammonium chloride, then permeabilized with 0.5% Triton X in PBS. Sections were then blocked with blocking buffer (5% normal goat serum in PBS) for 1 h at room temperature, followed by overnight incubation with anti‐PERP antibody (Abcam 5986) in blocking buffer at 4°C. The next day, the sections were washed with blocking buffer, then incubated with secondary Alexa Fluor 488 (Life Technologies) at room temperature for 1 h. Sections were then washed in blocking buffer, mounted with SlowFade Gold with DAPI and viewed under a Leica TCS SP‐5 confocal microscope (Leica Microsystems) using a 40× oil objective with 1× digital zoom (Leica LASAF Software, Leica Microsystems). All samples were imaged as 0.2 μm Z‐stacks. Images were processed using FIJI (NIH). Animals were treated in accordance with institutional IACUC protocols.
PERP and Rab25 co‐localizationPolarized T84 monolayers were infected with SL1344 for 1 h and stained as described above for PERP (Abcam 5986) and Rab25 (Abcam 32004). Determination of PERP co‐localization with Rab25 was performed using a Leica TCS SP‐5 confocal microscope (Leica Microsystems) using a 63× oil objective with 6× digital zoom (Leica LASAF Software, Leica Microsystems). For increased resolution of PERP localization, pinhole was decreased to 0.5 airy units for all imaging and all samples were imaged as 0.2 μm Z‐stacks. Images shown are representative of three images taken from random fields per sample. Post‐imaging, images were processed using FIJI (NIH) with single 0.2 μm slices selected from the quarter most apical sections to show co‐localization of Rab25 (AlexaFluor 568, red pseudocolour) and PERP (AlexaFluor 488, green pseudocolour), along with F‐actin (phalloidin AlexaFluor 647, blue pseudocolour) to show cellular structure. The level of co‐localization was determined with Manders' coefficient analyses in FIJI.

Confocal microscopy

Mouse colon tissueSections of the proximal colons from 6‐week‐old C57BL/6 mice were removed and snap frozen in OCT media, then cut into 5 μm sections on glass slides. Sections were fixed in 4% PFA, quenched with 50 mM ammonium chloride, then permeabilized with 0.5% Triton X in PBS. Sections were then blocked with blocking buffer (5% normal goat serum in PBS) for 1 h at room temperature, followed by overnight incubation with anti‐PERP antibody (Abcam 5986) in blocking buffer at 4°C. The next day, the sections were washed with blocking buffer, then incubated with secondary Alexa Fluor 488 (Life Technologies) at room temperature for 1 h. Sections were then washed in blocking buffer, mounted with SlowFade Gold with DAPI and viewed under a Leica TCS SP‐5 confocal microscope (Leica Microsystems) using a 40× oil objective with 1× digital zoom (Leica LASAF Software, Leica Microsystems). All samples were imaged as 0.2 μm Z‐stacks. Images were processed using FIJI (NIH). Animals were treated in accordance with institutional IACUC protocols.

Mouse colon tissue

Sections of the proximal colons from 6‐week‐old C57BL/6 mice were removed and snap frozen in OCT media, then cut into 5 μm sections on glass slides. Sections were fixed in 4% PFA, quenched with 50 mM ammonium chloride, then permeabilized with 0.5% Triton X in PBS. Sections were then blocked with blocking buffer (5% normal goat serum in PBS) for 1 h at room temperature, followed by overnight incubation with anti‐PERP antibody (Abcam 5986) in blocking buffer at 4°C. The next day, the sections were washed with blocking buffer, then incubated with secondary Alexa Fluor 488 (Life Technologies) at room temperature for 1 h. Sections were then washed in blocking buffer, mounted with SlowFade Gold with DAPI and viewed under a Leica TCS SP‐5 confocal microscope (Leica Microsystems) using a 40× oil objective with 1× digital zoom (Leica LASAF Software, Leica Microsystems). All samples were imaged as 0.2 μm Z‐stacks. Images were processed using FIJI (NIH). Animals were treated in accordance with institutional IACUC protocols.

PERP and Rab25 co‐localizationPolarized T84 monolayers were infected with SL1344 for 1 h and stained as described above for PERP (Abcam 5986) and Rab25 (Abcam 32004). Determination of PERP co‐localization with Rab25 was performed using a Leica TCS SP‐5 confocal microscope (Leica Microsystems) using a 63× oil objective with 6× digital zoom (Leica LASAF Software, Leica Microsystems). For increased resolution of PERP localization, pinhole was decreased to 0.5 airy units for all imaging and all samples were imaged as 0.2 μm Z‐stacks. Images shown are representative of three images taken from random fields per sample. Post‐imaging, images were processed using FIJI (NIH) with single 0.2 μm slices selected from the quarter most apical sections to show co‐localization of Rab25 (AlexaFluor 568, red pseudocolour) and PERP (AlexaFluor 488, green pseudocolour), along with F‐actin (phalloidin AlexaFluor 647, blue pseudocolour) to show cellular structure. The level of co‐localization was determined with Manders' coefficient analyses in FIJI.

PERP and Rab25 co‐localization

Polarized T84 monolayers were infected with SL1344 for 1 h and stained as described above for PERP (Abcam 5986) and Rab25 (Abcam 32004). Determination of PERP co‐localization with Rab25 was performed using a Leica TCS SP‐5 confocal microscope (Leica Microsystems) using a 63× oil objective with 6× digital zoom (Leica LASAF Software, Leica Microsystems). For increased resolution of PERP localization, pinhole was decreased to 0.5 airy units for all imaging and all samples were imaged as 0.2 μm Z‐stacks. Images shown are representative of three images taken from random fields per sample. Post‐imaging, images were processed using FIJI (NIH) with single 0.2 μm slices selected from the quarter most apical sections to show co‐localization of Rab25 (AlexaFluor 568, red pseudocolour) and PERP (AlexaFluor 488, green pseudocolour), along with F‐actin (phalloidin AlexaFluor 647, blue pseudocolour) to show cellular structure. The level of co‐localization was determined with Manders' coefficient analyses in FIJI.

Data presentationImages are presented as one representative of at least three experiments showing reproducible trends. P‐values were calculated using the Student's t‐test, and values of < 0.05 were considered statistically significant. In cases where datasets contained more than two groups, one‐way analyses of variance were performed first, followed by individual Student's t‐test analyses to determine which treatment groups differed from the control. Error bars represent standard error.

Data presentation

Images are presented as one representative of at least three experiments showing reproducible trends. P‐values were calculated using the Student's t‐test, and values of < 0.05 were considered statistically significant. In cases where datasets contained more than two groups, one‐way analyses of variance were performed first, followed by individual Student's t‐test analyses to determine which treatment groups differed from the control. Error bars represent standard error.

Supporting information
Fig. S1. PERP antibody migration control. Polarized HCT8 monolayers were treated with the PMN chemoattractant fMLP or left untreated and exposed to 25 μg ml−1 of anti‐PERP antibody or incubated in HBSS +. Cells incubated with anti‐PERP antibody only (no fMLP or infection) did not induce PMN transmigration, indicating treatment with anti‐PERP antibody alone does not impact PMN transmigration.Click here for additional data file.
Fig. S2. Expression of PERP in mouse colon. Five micron cryosections of mouse proximal colon were stained for PERP (green) or DAPI (blue). We detect the presence of PERP throughout the tissue, including at the mucosal surface where epithelial cells would be exposed to Salmonella infection.Click here for additional data file.
Fig. S3. Controls for apical surface biotinylation.A. Polarized HCT8 monolayers were infected (WT) or left uninfected (−) in HBSS + for 1 h, and the basolateral surface was biotinylated, pulled down via streptavidin and analysed for PERP via Western blot.B. Polarized HCT8 monolayers were infected (WT) or left uninfected (−) in HBSS + for 1 h, and the apical surface was biotinylated, pulled down via streptavidin and analysed for PERP or E‐cadherin. ‘WCL’ refers to the input used for pull down.Click here for additional data file.
Fig. S4. Quantification of punctate PERP staining in response to Salmonella infection. The level of PERP punctate staining was quantified with FIJI using the same size filter for the punctae across all samples. Data show values averaged across three experiments. Data from each treatment group were normalized to data from non‐treated cells. Student's t‐test analyses were used to statistically evaluate data from WT samples to the remaining groups. WT infection induced more PERP punctae than cells left untreated or cells infected with the SipA‐deficient strain (ΔSipA). The levels of PERP punctae from cells infected with the SipA‐complemented strain (pSipA), the caspase‐3 site mutant strain (CSM) or cells treated exogenously with HA‐SipA were not significantly different from the level of PERP punctae resulting from WT infection. Error bars indicate standard error. P‐values less than 0.05 were considered statistically significant. N.S. indicates no statistical difference between the indicated group and WT.Click here for additional data file.
Fig. S5. PERP co‐localizes with apical recycling endosomes. T84 monolayers were treated with HBSS + only (−) or infected with wild‐type Salmonella (WT). The cells were stained with an antibody against PERP followed by secondary conjugated to Alexa Fluor 488 (green pseudocolour), and with an antibody against Rab25 followed by secondary conjugated to Alexa Fluor 568 (red pseudocolour). PERP and Rab25 co‐localize in non‐infected (C) and infected (F) cells (white arrows), indicating the apical recycling endosome pathway is at least partly responsible for PERP localization. The level of co‐localization was determined via Manders' coefficient analyses, where M1 refers to PERP in comparison with Rab25 staining, and M2 refers to Rab25 in comparison with PERP staining.Click here for additional data file.

Supporting information

Fig. S1. PERP antibody migration control. Polarized HCT8 monolayers were treated with the PMN chemoattractant fMLP or left untreated and exposed to 25 μg ml−1 of anti‐PERP antibody or incubated in HBSS +. Cells incubated with anti‐PERP antibody only (no fMLP or infection) did not induce PMN transmigration, indicating treatment with anti‐PERP antibody alone does not impact PMN transmigration.

Click here for additional data file.

Fig. S2. Expression of PERP in mouse colon. Five micron cryosections of mouse proximal colon were stained for PERP (green) or DAPI (blue). We detect the presence of PERP throughout the tissue, including at the mucosal surface where epithelial cells would be exposed to Salmonella infection.

Click here for additional data file.

Fig. S3. Controls for apical surface biotinylation.

A. Polarized HCT8 monolayers were infected (WT) or left uninfected (−) in HBSS + for 1 h, and the basolateral surface was biotinylated, pulled down via streptavidin and analysed for PERP via Western blot.

B. Polarized HCT8 monolayers were infected (WT) or left uninfected (−) in HBSS + for 1 h, and the apical surface was biotinylated, pulled down via streptavidin and analysed for PERP or E‐cadherin. ‘WCL’ refers to the input used for pull down.

Click here for additional data file.

Fig. S4. Quantification of punctate PERP staining in response to Salmonella infection. The level of PERP punctate staining was quantified with FIJI using the same size filter for the punctae across all samples. Data show values averaged across three experiments. Data from each treatment group were normalized to data from non‐treated cells. Student's t‐test analyses were used to statistically evaluate data from WT samples to the remaining groups. WT infection induced more PERP punctae than cells left untreated or cells infected with the SipA‐deficient strain (ΔSipA). The levels of PERP punctae from cells infected with the SipA‐complemented strain (pSipA), the caspase‐3 site mutant strain (CSM) or cells treated exogenously with HA‐SipA were not significantly different from the level of PERP punctae resulting from WT infection. Error bars indicate standard error. P‐values less than 0.05 were considered statistically significant. N.S. indicates no statistical difference between the indicated group and WT.

Click here for additional data file.

Fig. S5. PERP co‐localizes with apical recycling endosomes. T84 monolayers were treated with HBSS + only (−) or infected with wild‐type Salmonella (WT). The cells were stained with an antibody against PERP followed by secondary conjugated to Alexa Fluor 488 (green pseudocolour), and with an antibody against Rab25 followed by secondary conjugated to Alexa Fluor 568 (red pseudocolour). PERP and Rab25 co‐localize in non‐infected (C) and infected (F) cells (white arrows), indicating the apical recycling endosome pathway is at least partly responsible for PERP localization. The level of co‐localization was determined via Manders' coefficient analyses, where M1 refers to PERP in comparison with Rab25 staining, and M2 refers to Rab25 in comparison with PERP staining.

Click here for additional data file.
