---
pmid: '25751282'
title: Mutation in CPT1C Associated With Pure Autosomal Dominant Spastic Paraplegia.
authors:
- Rinaldi C
- Schmidt T
- Situ AJ
- Johnson JO
- Lee PR
- Chen KL
- Bott LC
- Fadó R
- Harmison GH
- Parodi S
- Grunseich C
- Renvoisé B
- Biesecker LG
- De Michele G
- Santorelli FM
- Filla A
- Stevanin G
- Dürr A
- Brice A
- Casals N
- Traynor BJ
- Blackstone C
- Ulmer TS
- Fischbeck KH
journal: JAMA Neurol
year: '2015'
full_text_available: false
pmcid: PMC5612424
doi: 10.1001/jamaneurol.2014.4769
---

# Mutation in CPT1C Associated With Pure Autosomal Dominant Spastic Paraplegia.
**Authors:** Rinaldi C, Schmidt T, Situ AJ, Johnson JO, Lee PR, Chen KL, Bott LC, Fadó R, Harmison GH, Parodi S, Grunseich C, Renvoisé B, Biesecker LG, De Michele G, Santorelli FM, Filla A, Stevanin G, Dürr A, Brice A, Casals N, Traynor BJ, Blackstone C, Ulmer TS, Fischbeck KH
**Journal:** JAMA Neurol (2015)
**DOI:** [10.1001/jamaneurol.2014.4769](https://doi.org/10.1001/jamaneurol.2014.4769)
**PMC:** [PMC5612424](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5612424/)

## Abstract

1. JAMA Neurol. 2015 May;72(5):561-70. doi: 10.1001/jamaneurol.2014.4769.

Mutation in CPT1C Associated With Pure Autosomal Dominant Spastic Paraplegia.

Rinaldi C(1), Schmidt T(2), Situ AJ(2), Johnson JO(3), Lee PR(4), Chen KL(1), 
Bott LC(5), Fadó R(6), Harmison GH(1), Parodi S(7), Grunseich C(1), Renvoisé 
B(1), Biesecker LG(8), De Michele G(9), Santorelli FM(10), Filla A(9), Stevanin 
G(11), Dürr A(12), Brice A(12), Casals N(6), Traynor BJ(3), Blackstone C(1), 
Ulmer TS(2), Fischbeck KH(1).

Author information:
(1)Neurogenetics Branch, National Institute of Neurological Disorders and 
Stroke, National Institutes of Health, Bethesda, Maryland.
(2)Department of Biochemistry and Molecular Biology, Zilkha Neurogenetic 
Institute, Keck School of Medicine, University of Southern California, Los 
Angeles.
(3)Neuromuscular Diseases Research Section, Laboratory of Neurogenetics, 
National Institute on Aging, National Institutes of Health, Bethesda, Maryland.
(4)Section on Nervous System Development and Plasticity, The Eunice Kennedy 
Shriver National Institute of Child and Human Development, National Institutes 
of Health, Bethesda, Maryland.
(5)Neurogenetics Branch, National Institute of Neurological Disorders and 
Stroke, National Institutes of Health, Bethesda, Maryland5Department of Cell and 
Molecular Biology, Karolinska Institutet, Stockholm, Sweden.
(6)Basic Sciences Department, Facultat de Medicina i Ciències de la Salut, 
Universitat Internacional de Catalunya, and CIBER Fisiopatología de la Obesidad 
y la Nutrición (CIBERobn), Sant Cugat del Vallés, Spain.
(7)Neurogenetics Branch, National Institute of Neurological Disorders and 
Stroke, National Institutes of Health, Bethesda, Maryland7Department of 
Neuroscience and Brain Technologies, Istituto Italiano di Tecnologia, Genoa, 
Italy.
(8)Genetic Disease Research Branch, National Human Genome Research Institute, 
and the National Institutes of Health Intramural Sequencing Center, National 
Institutes of Health, Bethesda, Maryland.
(9)Department of Neurosciences, Reproductive Sciences, and Odontostomatology, 
University of Naples Federico II, Naples, Italy.
(10)Neurogenetics Istituto di Ricovero e Cura a Carattere Scientifico, Stella 
Maris, Pisa, Italy.
(11)Institut du Cerveau et de la Moelle Épinière, Paris, France12Laboratoire de 
Neurogénétique, École Pratique des Hautes Études-héSam Université, Institut du 
Cerveau et de la Moelle Épinière, Groupe Hospitalier Pitié-Salpêtrière, Paris, 
France13Sorbonne Univ.
(12)Institut du Cerveau et de la Moelle Épinière, Paris, France13Sorbonne 
Universités, Université Pierre et Marie Curie, Institut du Cerveau et de la 
Moelle Épinière, Paris, France14Department of Genetics, Assistance Publique 
Hopitaux de Paris, Groupe Hospita.

Erratum in
    JAMA Neurol. 2015 May;72(5):608. doi: 10.1001/jamaneurol.2015.0528.

IMPORTANCE: The family of genes implicated in hereditary spastic paraplegias 
(HSPs) is quickly expanding, mostly owing to the widespread availability of 
next-generation DNA sequencing methods. Nevertheless, a genetic diagnosis 
remains unavailable for many patients.
OBJECTIVE: To identify the genetic cause for a novel form of pure autosomal 
dominant HSP.
DESIGN, SETTING, AND PARTICIPANTS: We examined and followed up with a family 
presenting to a tertiary referral center for evaluation of HSP for a decade 
until August 2014. Whole-exome sequencing was performed in 4 patients from the 
same family and was integrated with linkage analysis. Sanger sequencing was used 
to confirm the presence of the candidate variant in the remaining affected and 
unaffected members of the family and screen the additional patients with HSP. 
Five affected and 6 unaffected participants from a 3-generation family with pure 
adult-onset autosomal dominant HSP of unknown genetic origin were included. 
Additionally, 163 unrelated participants with pure HSP of unknown genetic cause 
were screened.
MAIN OUTCOME AND MEASURE: Mutation in the neuronal isoform of carnitine 
palmitoyl-transferase (CPT1C) gene.
RESULTS: We identified the nucleotide substitution c.109C>T in exon 3 of CPT1C, 
which determined the base substitution of an evolutionarily conserved Cys 
residue for an Arg in the gene product. This variant strictly cosegregated with 
the disease phenotype and was absent in online single-nucleotide polymorphism 
databases and in 712 additional exomes of control participants. We showed that 
CPT1C, which localizes to the endoplasmic reticulum, is expressed in motor 
neurons and interacts with atlastin-1, an endoplasmic reticulum protein encoded 
by the ATL1 gene known to be mutated in pure HSPs. The mutation, as indicated by 
nuclear magnetic resonance spectroscopy studies, alters the protein conformation 
and reduces the mean (SD) number (213.0 [46.99] vs 81.9 [14.2]; P < .01) and 
size (0.29 [0.01] vs 0.26 [0.01]; P < .05) of lipid droplets on overexpression 
in cells. We also observed a reduction of mean (SD) lipid droplets in primary 
cortical neurons isolated from Cpt1c-/- mice as compared with wild-type mice 
(1.0 [0.12] vs 0.44 [0.05]; P < .001), suggesting a dominant negative mechanism 
for the mutation.
CONCLUSIONS AND RELEVANCE: This study expands the genetics of autosomal dominant 
HSP and is the first, to our knowledge, to link mutation in CPT1C with a human 
disease. The association of the CPT1C mutation with changes in lipid droplet 
biogenesis supports a role for altered lipid-mediated signal transduction in HSP 
pathogenesis.

DOI: 10.1001/jamaneurol.2014.4769
PMCID: PMC5612424
PMID: 25751282 [Indexed for MEDLINE]

Conflict of interest statement: Conflict of Interest Disclosures: Dr Biesecker 
receives royalties from Amgen and Genentech and is an uncompensated advisor to 
the Illumina Corporation. No other disclosures were reported.
