---
pmid: '25848931'
title: siRNA screen identifies QPCT as a druggable target for Huntington's disease.
authors:
- Jimenez-Sanchez M
- Lam W
- Hannus M
- Sönnichsen B
- Imarisio S
- Fleming A
- Tarditi A
- Menzies F
- Dami TE
- Xu C
- Gonzalez-Couto E
- Lazzeroni G
- Heitz F
- Diamanti D
- Massai L
- Satagopam VP
- Marconi G
- Caramelli C
- Nencini A
- Andreini M
- Sardone GL
- Caradonna NP
- Porcari V
- Scali C
- Schneider R
- Pollio G
- O'Kane CJ
- Caricasole A
- Rubinsztein DC
journal: Nat Chem Biol
year: '2015'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC4696152
doi: 10.1038/nchembio.1790
---

# siRNA screen identifies QPCT as a druggable target for Huntington's disease.
**Authors:** Jimenez-Sanchez M, Lam W, Hannus M, Sönnichsen B, Imarisio S, Fleming A, Tarditi A, Menzies F, Dami TE, Xu C, Gonzalez-Couto E, Lazzeroni G, Heitz F, Diamanti D, Massai L, Satagopam VP, Marconi G, Caramelli C, Nencini A, Andreini M, Sardone GL, Caradonna NP, Porcari V, Scali C, Schneider R, Pollio G, O'Kane CJ, Caricasole A, Rubinsztein DC
**Journal:** Nat Chem Biol (2015)
**DOI:** [10.1038/nchembio.1790](https://doi.org/10.1038/nchembio.1790)
**PMC:** [PMC4696152](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4696152/)

## Abstract

1. Nat Chem Biol. 2015 May;11(5):347-354. doi: 10.1038/nchembio.1790. Epub 2015
Apr  6.

siRNA screen identifies QPCT as a druggable target for Huntington's disease.

Jimenez-Sanchez M(1), Lam W(1)(2), Hannus M(#)(3), Sönnichsen B(#)(3), Imarisio 
S(#)(1)(2), Fleming A(#)(1)(4), Tarditi A(5), Menzies F(1), Dami TE(1)(4)(6), Xu 
C(1)(4), Gonzalez-Couto E(5), Lazzeroni G(5), Heitz F(5), Diamanti D(5), Massai 
L(5), Satagopam VP(7)(8), Marconi G(5), Caramelli C(5), Nencini A(5), Andreini 
M(5), Sardone GL(5), Caradonna NP(5), Porcari V(5), Scali C(5), Schneider 
R(7)(8), Pollio G(5), O'Kane CJ(2), Caricasole A(5), Rubinsztein DC(1).

Author information:
(1)Department of Medical Genetics, University of Cambridge, Cambridge Institute 
for Medical Research, Addenbrooke's Hospital, Hills Road, Cambridge CB2 0XY, UK.
(2)Department of Genetics, University of Cambridge, Cambridge CB2 3EH, UK.
(3)Cenix BioScience GmbH, Tatzberg 47, 01307 Dresden, Germany.
(4)Department of Physiology, Development and Neuroscience, University of 
Cambridge, Downing Street, Cambridge, UK, CB2 3EG.
(5)Siena Biotech. Strada del Petriccio e Belriguardo, 35 53100 Siena, Italy.
(6)Department of Neuroscience, Psychology, Drug Research and Child Health, 
Division of Pharmacology and Toxicology, University of Florence, Florence, 
Italy.
(7)Structural and Computational Biology, EMBL, Meyerhofstr.1, 69117, Heidelberg, 
Germany.
(8)Luxembourg Centre for Systems Biomedicine (LCSB), University of Luxembourg, 
Campus Belval, House of Biomedicine, 7 avenue des Hauts-Fourneaux, L-4362 
Esch-sur-Alzette, Luxembourg.
(#)Contributed equally

Huntington's disease (HD) is a currently incurable neurodegenerative condition 
caused by an abnormally expanded polyglutamine tract in huntingtin (HTT). We 
identified new modifiers of mutant HTT toxicity by performing a large-scale 
'druggable genome' siRNA screen in human cultured cells, followed by hit 
validation in Drosophila. We focused on glutaminyl cyclase (QPCT), which had one 
of the strongest effects on mutant HTT-induced toxicity and aggregation in the 
cell-based siRNA screen and also rescued these phenotypes in Drosophila. We 
found that QPCT inhibition induced the levels of the molecular chaperone 
αB-crystallin and reduced the aggregation of diverse proteins. We generated new 
QPCT inhibitors using in silico methods followed by in vitro screening, which 
rescued the HD-related phenotypes in cell, Drosophila and zebrafish HD models. 
Our data reveal a new HD druggable target affecting mutant HTT aggregation and 
provide proof of principle for a discovery pipeline from druggable genome screen 
to drug development.

DOI: 10.1038/nchembio.1790
PMCID: PMC4696152
PMID: 25848931 [Indexed for MEDLINE]

## Full Text

IntroductionHuntington’s disease (HD) is a fatal, currently incurable, late-onset neurodegenerative disorder. The disease signs include involuntary and repetitive choreic movements, psychological dysfunction and cognitive impairment, which result from progressive degeneration of cortical and striatal neurons 12.HD is caused by the expansion of a CAG repeat tract in exon 1 of the gene encoding huntingtin (HTT), which results in an abnormally long polyglutamine stretch in the N-terminus of the protein 3. Although the mechanisms are not fully understood, it is believed that the disease arises from a toxic-gain-of function of the mutant protein 45. A hallmark of HD is the presence of intracellular aggregates, which is also a characteristic of the other ten polyglutamine-expansion disorders, as well as other neurodegenerative conditions such as Parkinson’s or Alzheimer’s disease 6. The role of these aggregates in the disease is not clear, although an increasing importance of the oligomeric forms in toxicity is emerging 78 and reducing mutant HTT aggregation with strategies such as pharmacological upregulation of chaperone function has been pursued as a therapeutic strategy in HD 9. Mutant HTT toxicity is believed to be accentuated, or possibly induced, after cleavage events resulting in the formation of short N-terminal polyglutamine containing fragments, which can also be produced by aberrant splicing 10. Hence, exon 1 models have been frequently used for disease modeling.Here, we combined two approaches to identify modifiers of mutant HTT toxicity by first performing a cell-based screen to identify genes that when knocked down could suppress mutant HTT-induced toxicity, using a library of 5,623 siRNAs selected according to the potential druggability of their targets with small molecules 11. We performed this screen in two different HD models. Initially, we screened the effects of siRNAs in a mammalian cell line inducibly expressing HTT with an abnormal polyglutamine expansion. In a secondary analysis, we validated primary hits in a Drosophila model of HD.One of the strongest suppressors of mutant HTT toxicity in both mammalian cells and Drosophila was an enzyme responsible for the modification of N-terminal residues of glutamine or glutamate into an N-terminal 5-oxoproline or pyroglutamate (pE), named glutaminyl cyclase (QPCT).. QPCT not only suppressed mutant HTT induced toxicity but also greatly reduced the number of aggregates. This effect is not HTT-specific, since QPCT exerted a general effect on aggregation of different aggregate-prone proteins, including other proteins containing an expanded polyglutamine or polyalanine tract, which could be attributed to increased levels of the chaperone alpha B-crystallin upon QPCT inhibition. Furthermore, we designed small molecule modulators of QPCT activity, which effectively suppressed mutant HTT aggregation and toxicity in cells, neurons, fly and zebrafish models of the disease.

Introduction

Huntington’s disease (HD) is a fatal, currently incurable, late-onset neurodegenerative disorder. The disease signs include involuntary and repetitive choreic movements, psychological dysfunction and cognitive impairment, which result from progressive degeneration of cortical and striatal neurons 12.

HD is caused by the expansion of a CAG repeat tract in exon 1 of the gene encoding huntingtin (HTT), which results in an abnormally long polyglutamine stretch in the N-terminus of the protein 3. Although the mechanisms are not fully understood, it is believed that the disease arises from a toxic-gain-of function of the mutant protein 45. A hallmark of HD is the presence of intracellular aggregates, which is also a characteristic of the other ten polyglutamine-expansion disorders, as well as other neurodegenerative conditions such as Parkinson’s or Alzheimer’s disease 6. The role of these aggregates in the disease is not clear, although an increasing importance of the oligomeric forms in toxicity is emerging 78 and reducing mutant HTT aggregation with strategies such as pharmacological upregulation of chaperone function has been pursued as a therapeutic strategy in HD 9. Mutant HTT toxicity is believed to be accentuated, or possibly induced, after cleavage events resulting in the formation of short N-terminal polyglutamine containing fragments, which can also be produced by aberrant splicing 10. Hence, exon 1 models have been frequently used for disease modeling.

Here, we combined two approaches to identify modifiers of mutant HTT toxicity by first performing a cell-based screen to identify genes that when knocked down could suppress mutant HTT-induced toxicity, using a library of 5,623 siRNAs selected according to the potential druggability of their targets with small molecules 11. We performed this screen in two different HD models. Initially, we screened the effects of siRNAs in a mammalian cell line inducibly expressing HTT with an abnormal polyglutamine expansion. In a secondary analysis, we validated primary hits in a Drosophila model of HD.

One of the strongest suppressors of mutant HTT toxicity in both mammalian cells and Drosophila was an enzyme responsible for the modification of N-terminal residues of glutamine or glutamate into an N-terminal 5-oxoproline or pyroglutamate (pE), named glutaminyl cyclase (QPCT).. QPCT not only suppressed mutant HTT induced toxicity but also greatly reduced the number of aggregates. This effect is not HTT-specific, since QPCT exerted a general effect on aggregation of different aggregate-prone proteins, including other proteins containing an expanded polyglutamine or polyalanine tract, which could be attributed to increased levels of the chaperone alpha B-crystallin upon QPCT inhibition. Furthermore, we designed small molecule modulators of QPCT activity, which effectively suppressed mutant HTT aggregation and toxicity in cells, neurons, fly and zebrafish models of the disease.

ResultsPrimary cell screen for suppressors of mutant Htt toxicityWe performed the primary screen using a stable HEK293/T Rex cell line expressing full-length human HTT bearing 138 polyglutamines (Q138) under the control of a tetracycline-inducible promoter. We confirmed the expression of HTT(Q138) after inducing the cells with doxycycline using antibodies recognizing the N-terminus of human HTT (Supplementary Results, Supplementary Fig. 1a and Supplementary Note 1), and quantitative RT-PCR using primers spanning different areas of the human HTT cDNA (Supplementary Fig. 1b). This cell line had reduced cell viability after expression of mutant HTT, which was reverted by treatment with a known reference compound (Y27632) 12 (Supplementary Fig. 1c), suggesting that this model could be used to identify potential modulators of mutant HTT cellular toxicity in a large-scale screen.For our high-throughput screen, we utilised a strategy consisting of an iterative siRNA screen where positive genes were selected after three consecutive rounds to compensate for the variability of the assay. We eliminated non-positive siRNAs and added new siRNAs targeting the selected genes in consecutive passes. We assessed rescue of cellular toxicity by each siRNA by fluorescence microscopy and automated image analysis using three independent readouts: 1) number of cell nuclei (#nuclei), 2) apoptotic index and 3) aberrant nuclei index, and used rescue indices to express the effect of each individual siRNA for each parameter analysed. In an initial screen, we tested 3 independent siRNAs for each of the 5,623 genes (a total of 16,869 siRNAs), from which we selected 670 primary genes (see Supplementary Note 1 for screen assay and criteria selection). As shown in supplementary figure 2a, the three readouts were partially redundant, as more than 50% of the 1,000 top scoring siRNAs of one rescue index also ranked amongst the top 1,000 siRNAs of at least one of the other rescue indices. In supplementary figure 1b, a representation of rescue indices obtained in pass 1 shows the relatively large variability of the assay, with non-targeting negative control siRNAs, negQ and negF, showing a #nuclei rescue indices of 14% and 3% respectively, while using siRNA targeting HTT as a positive control rendered a mean #nuclei rescue index of 81%.After 3 consecutive rounds of screening, we selected and 257 genes and ranked these based on all three rescue indices, using #nuclei rescue index as a primary criterion (Supplementary Data Set 1).Secondary RNAi screening in a Drosophila model of HDTo validate the hits obtained in mammalian cells and to focus on targets with potential relevance in vivo, we performed a secondary screen in a Drosophila model that expressed a construct containing 48 polyglutamines, Q48, that causes eye degeneration when expressed using a GMR-GAL4 driver 13. For most genes selected, we studied two UAS-RNAi constructs from the Vienna Drosophila RNAi Center (VDRC): a P-element (GD) and a phiC31 (KK) construct, the latter of which carries more GAL4-binding sites and should therefore express the RNAi more strongly 14. Of the 257 mammalian genes previously selected, we detected 133 that had one or more gene orthologs in flies (Supplementary Data Set 1 and 2). Of these 133 mammalian genes with fly orthologs, 74 Drosophila genes (corresponding to 66 mammalian genes) rescued the Q48-induced eye degeneration with at least one RNAi line, while the others showed no obvious or significant effect (Supplementary Fig. 3a and 3b and Supplementary Data Sets 1 and 2). We crossed suppressor RNAi lines to transgenic flies that expressed EGFP, also driven by the same GMR-GAL4 driver. We used EGFP to test whether modifiers affected transgene protein synthesis, since Q48 levels can be modified by aggregation or autophagic degradation, which do not impact EGFP levels. Two of these fly RNAi lines, targeting orthologs to human CTSF and to human ADAM8, ADAM11 and ADAM33, reduced EGFP levels on western blots (Supplementary Data Set 2), suggesting a general effect of these genes in protein expression, while suppression exerted by the other RNAi lines seemed to be polyglutamine-specific.Functional categorization of mutant HTT modifiersTo gain further insight into the biological relevance of the data generated, we categorized the different sets of HD toxicity modulators according to their molecular function. Suppressors were enriched for certain classes of proteins such as GPCRs or transporters compared to the initial library, while the number of positive kinases in the screen was reduced and no cytokines, growth factors or translational regulators were represented. We observed similar functional categorizations after selection from the cell and Drosophila screen (Supplementary Fig. 4a). An Ingenuity Pathway Analysis (IPA) of the hits obtained in the primary screen in cells (Supplementary Table 1a) revealed that the majority of these proteins participate in general processes such as GPCR- or cAMP-mediated signalling, but also in canonical pathways related to neurodegeneration, such as apoptosis, mitochondrial dysfunction, amyloid processing or protein ubiquitination. Importantly, 10 of these proteins have been previously related to HD signalling, including subunits of the succinate dehydrogenase complex and huntingtin-associated protein 1 (HAP1) (Supplementary Table 1a). Many of the genes validated in Drosophila (Supplementary Fig. 4b and Supplementary Table 1b) are also involved in processes related to neurodegeneration but were enriched in mitochondrial metabolic pathways, especially those associated with fatty acid biosynthesis and metabolism.Validation of QPCT in DrosophilaWe focused our attention on a gene that had one of the strongest and most consistent effects in rescuing mutant HTT-induced toxicity in the cell-based siRNA screen. The gene product has glutaminyl cyclase activity and is named QPCT. Two orthologs have been reported in fly 15, Glutaminyl cyclase (QC) and iso Glutaminyl cyclase (isoQC), which show about 39% amino acid identity; a third fly ortholog, CG6168, shows expression restricted to male accessory glands (www.flyatlas.org) and is not considered further here. RNAi lines targeting either QC or isoQC partially rescued eye depigmentation and mediated a significant decrease in the number of black spots in flies expressing Q48 (Fig. 1a, 1b and Supplementary Fig. 5a) (Data are shown for GD- and KK-RNAi lines in the case of QC, but only a KK line was available for isoQC). These effects are likely independent of transcription/translation of the Q48, since no change in EGFP protein levels were seen when we crossed transgenic flies expressing EGFP driven by the same GMR-GAL4 driver as Q48 with QC or isoQC RNAi lines (Supplementary Fig. 5b). Thus, QPCT represents an interesting candidate for studying in HD.To further evaluate the benefits of downregulating QPCT on HD, we took advantage of an additional Drosophila model of neurodegeneration, HD flies that express exon 1 of HTT with 120 polyglutamines, GMR-HTT.Q120 in eye photoreceptors 16. Drosophila melanogaster has a compound eye consisting of many ommatidia, each of which is composed of 8 photoreceptors, seven of which can be visualized by light microscopy using the pseudopupil technique 17. Neurodegeneration results in the loss of visible rhabdomeres of each photoreceptor and can be rescued or enhanced by genetic or chemical approaches 18. Consistent with our data using the Q48 flies, the loss of visible photoreceptors in transgenic flies expressing GMR-HTT.Q120 was partially rescued when they were crossed with RNAi lines for either of the two QPCT fly orthologues, QC and isoQC (Fig. 1c). We observed no effect on the number of rhabdomeres in QPCT RNAi lines in the absence of GMR-HTT.Q120. The effects of QPCT knockdown on toxicity correlated with a reduction in HTT aggregation, which we assessed in flies expressing GFP-tagged expanded huntingtin exon 1, HTTEx1-Q46-eGFP in the eye 19 (Fig. 1d).QPCT modulates mutant HTT aggregationTo further validate QPCT, we first confirmed the protective effect of its knockdown against toxicity and aggregation in HEK293 cells expressing the exon 1 of HTT (from residue 8) with a 74 polyglutamine expansion fused at its N-terminal to EGFP (EGFP-HTT(Q74)) 20 (Fig. 2a, Supplementary fig. 6a and 6b). The QPCT siRNAs used in these experiments as well as in the screen do not target QPCT-like, which encodes a paralogous protein that catalyzes a similar reaction and shows 51% of sequence identity to QPCT (Supplementary Fig. 6b and 6c). We also validated the effect of QPCT knockdown on aggregation in HeLa cells (Supplementary Fig. 6d) which, like HEK293 cells, express QPCT 21. We also confirmed a decrease in protein aggregation of a construct which expresses full-length HTT carrying 138 polyglutamines (similar to the one used in the initial screen) (Supplementary Fig. 6e). QPCT siRNA did not have a general anti-apoptotic effect as it did not affect caspase 3 activity in response to staurosporine treatment (Supplementary Fig. 6f). Consistent with these data, QPCT shRNA reduced EGFP-Q80 (80 glutamines fused to EGFP) aggregation in primary cortical neurons (Fig. 2b and Supplementary fig. 6g). We could not assess the effect of QPCT knockdown on polyglutamine-mediated toxicity in these neurons, where the levels of cell death obtained in this assay were very low, as can be appreciated in fig. 2b. While knocking down QPCT was protective, overexpression of QPCT in HeLa and HEK293 cells increased the numbers of apoptotic nuclei and also led to a large accumulation of HTT(Q74) aggregates (Fig. 2c and Supplementary Fig. 7a), while QPCT did not increase caspase activity upon staurosporine treatment (Supplementary Fig. 7b). The effects of QPCT were activity-dependent, since the catalytically inactive E201Q mutant did not increase the percentage of cells with HTT(Q74) aggregates (Fig. 2d and Supplementary Fig. 7c and 7d).We measured mRNA levels of QPCT in HD mice and found that its expression was reduced when compared to their wild-type littermates, suggesting that QPCT expression may be downregulated as a compensatory mechanism (Supplementary Fig. 8) and that raised QPCT activity may not be a prerequisite for aggregation.QPCT catalyzes the modification of N-terminal glutamines or glutamates into a pyroglutamate (pE) residue. Although the presence of an extended polyglutamine tract makes HTT a potential substrate for QPCT, this enzyme only modifies N-terminal residues, suggesting that any modification on mutant HTT would require an N-terminal cleavage to reveal a glutamine at the N-terminal that could be cyclated. The formation of a pE residue may then affect its stability and propensity to aggregate, a hypothesis that was previously suggested 22. This cleavage model in either the polyglutamine tract or HTT exon 1 or GFP is unlikely, as QPCT modulated the aggregation of constructs consisting only of isolated polyglutamine expansions (Q57 and Q81) fused C-terminal to EGFP (Fig. 2e and 2f), or HTT exon 1 with 74 glutamines fused to HA 23 (Supplementary Fig. 9a and 9b), and QPCT siRNA also reduced the aggregation of an expansion of 37 alanines 24 (Fig. 2f). QPCT appeared to modulate the early stages of mutant HTT oligomerisation, since QPCT overexpression increased the amounts of Flag-tagged monomeric mutant HTT that were co-immunoprecipitated by GFP-tagged mutant HTT (Fig. 2g) 25. Since QPCT did not interact with HTT directly by immunoprecipitation (e.g. Fig. 2g), its effect on HTT oligomer formation is likely mediated via intermediaries.Design and characterization of compounds that inhibit QPCTTo target QPCT pharmacologically, we tested a previously described QPCT inhibitor 26, which did not rescue the HD phenotype in mammalian cells (Supplementary Fig. 10a and 10b). While this compound has been effective in Alzheimer disease (AD) models by reducing the formation of extracellular pE-Aβ, this may be due to extracellular QPCT inhibition 2127. Thus, we reasoned that the failure of this compound was likely due to poor cell permeability. In order to generate novel QPCT inhibitors, we employed existing data on its structure and known inhibitors to generate three 3D pharmacophore models, two ligand-based and one structure-based (using the human QPCT X-Ray structure (PDB id: 2AFW)). We used these models, along with stringently applied CNS filters and a solubility model developed in-house, to select 10,000 compounds from both commercially available screening compounds and the SienaBiotech compound library. We screened these molecules in a functional assay assessing the conversion of the H-Glu-AMC fluorogenic substrate into pyroGlu-AMC, as previously described 28, and selected hits associated with predicted robust binding for the hit-to-lead phase. The optimization strategy was based on physicochemical properties and ensemble docking model-driven approaches. The ensemble docking methodology 2930 was chosen to take into account the flexibility of human QPCT catalytic site and was constructed using both X-Ray structures and protein conformations coming from a 100 ns molecular dynamic study of the human QPCT 2AFW X-Ray structure. The ensemble docking model was evolved during the project development. Initially, only 4 X-Ray structures were used (PDB ID: 2AFW, 2AFX, 2AFZ31), then a set of 16 protein conformations, selected by clustering of molecular dynamic simulations, were added to improve model accuracy. Recently, two more X-Ray structures were added to the model (3PBB32 an 3SI033). All the docking calculations were performed using CCDC Gold (versions 4 and 5)343536 along with an ad-hoc developed program to rank and select the best scored ligand docking pose from the pool of QPCT conformations. Along with the biochemical readouts used during this optimization, we included a range of in vitro ADME assays, including solubility measurements, a CNS membrane permeability assay (PAMPA-BBB) 37 and stability in the presence of human CYP3A4, a member of the cytochrome P450 mixed-function oxidase system, and a key enzyme involved in the metabolism of xenobiotics in humans.We selected a series of compounds on the basis of these properties and validated their effects on mutant HTT aggregation and toxicity in cells expressing HTT(Q74)GFP, which led to the selection of three of them, SEN177 (1), SEN817 (2) and SEN180 (3) (Supplementary note 2, Fig. 3a, Supplementary Fig. 11a). Non-toxic concentrations of these compounds caused a dose-dependent reduction in the percentage of cells with aggregates, which correlated with a suppression of mutant HTT-induced apoptosis (Fig. 3b, 3c, 3d, 3d and Supplementary Fig. 11b). As seen with genetic knockdown experiments, pharmacologic inhibition of QPCT using these compounds also reduced aggregation of polyalanines (Fig. 3d) and did not affect protein levels, as assessed by measuring GFP levels by western blotting (Supplementary Fig. 11c) or by metabolic labeling of wild type HTT followed by detection of newly synthesized protein in the presence of SEN177 (Supplementary Fig. 11d). Importantly, the effect of these compounds was blocked when QPCT expression was suppressed by shRNA, confirming that they protect by a mechanism that requires QPCT inhibition (Fig 3e, Supplementary Fig. 11e and 11f). Thus, even though these compounds also inhibited QPCT-like (Supplementary Fig. 11a) and we cannot exclude the possibility that at least some of the effects observed may be mediated by this QPCT isoenzyme, their effects on aggregation were QPCT-dependent, as the shRNA used did not target QPCT-like. Consistent with these data, SEN177 greatly reduced the early stages of mutant HTT oligomerisation, as it decreased the amounts of GFP-tagged monomeric HTT that were co-immunoprecipitated by Flag-tagged HTT (Fig. 3f). The protective effect of these compounds was also confirmed in primary cortical neurons (Fig. 3g), with SEN177 and SEN817 significantly reducing the percentage of neurons with Q80 aggregates.QPCT modulates the levels of alpha B-crystallinThe effects of QPCT inhibition on HTT aggregation appeared to be independent of effects on protein clearance pathways targeting mutant huntingtin (autophagy and the ubiquitin-proteasome system) (Supplementary Fig. 12), changes in mRNA or protein levels (Supplementary Fig. 13a and 13b), or secretion of the enzyme into the medium (Supplementary Fig. 13c). QPCT is localized in the ER and secretory pathway and its knockdown, overexpression or inhibition seemed to have inconsistent and rather modest effects on different readouts of the ER stress response, measured by GRP78/BIP levels or phosphorylation of eIF2α, which did not correlate with its effect on aggregation (Supplementary Fig. 14). Our data also suggested that CREB (c-AMP response element binding protein) or ERK (extracellular signal-regulated kinase) signaling, recently reported to be activated upon QPCT inhibition 38 (Supplementary Fig. 15a and 15b), or JNK signaling (Supplementary Fig. 15c) were unlikely contributors to the effects we have observed.QPCT overexpression or knockdown did not modulate levels of HSP70, the main inducible stress response chaperone (Supplementary Fig. 15d). We performed transcriptional profiling to assess changes in alternative molecular chaperones induced by SEN177 in the presence of mutant HTT, and observed upregulation of several small heat shock proteins (sHSPs) (HSPB6 with 1.6 fold-change; HSPB3 with 1.5 fold-change; HSPB7 with 1.5 fold-change; and notably, alpha B-crystallin which had >2.5 fold increase in transcript levels) (Supplementary Fig. 16a and Supplementary Data Set 3). We confirmed this induction at the protein level as well as with other QPCT inhibitors (Fig. 4a). Genetic inhibition of QPCT dramatically increased alpha B-crystallin protein and mRNA levels in the presence of HTT(Q74) (Fig. 4b and 4c and Supplementary Fig. 16b), while QPCT overexpression, which increased mutant HTT aggregation and toxicity (Fig. 2c and Supplementary Fig. 7a), reduced alpha B-crystallin levels (Supplementary Fig. 16c). QPCT also modestly modulated alpha B-crystallin levels in the absence of mutant HTT or in the presence of the non-pathogenic Q23 (Supplementary Fig. 16b and 16c).As a sHSP, alpha B-crystallin acts as a molecular chaperone and is a suppressor of polyglutamine toxicity in cells and in Drosophila
394041. As expected, overexpression of alpha B-crystallin lowered the number of HTT(Q74) aggregates, while QPCT inhibitors failed to reduce aggregation further (Fig. 4d and Supplementary Fig. 16d), suggesting that this increment in alpha B-crystallin was a major contributor to the protection afforded by QPCT inhibition.QPCT inhibition protects fly and zebrafish HD modelsWe tested QPCT inhibitors in flies expressing Httex1Q46 in the eye and found a reduction in the number of aggregates (Fig. 5a). The compound with a greatest effect, SEN177, was able to also rescue the number of visible rhabdomeres and prevent neurodegeneration (Fig. 5b).A transgenic zebrafish expressing Htt exon 1 with 71Q fused to EGFP in the rod photoreceptors using the rhodopsin promoter has been established and validated as a model to study mutant huntingtin aggregation in vivo 42. Zebrafish have two homologs with putative glutaminyl-peptide cyclotransferase activity, QPCT and QPCTLA with 51% and 47% protein identity with QPCT and QPCT-like respectively. In order to test the effect of pharmacologic inhibition of QPCT in this model, we first determined the maximum tolerated concentration for each of the three compounds tested in mammalian cells and subsequently treated HD larvae. SEN817 and SEN180 reduced total number of EGFP-aggregates in the retina (Fig. 6a), which correlated with a marked decrease in toxicity similar to the positive control, clonidine 42, assessed by a rescue in the total area of eye photoreceptors (Fig. 6b).Although the three compounds were protective, their effectiveness varied between these models, which might be due to intrinsic properties of each system, SEN180 only mildly reduced aggregation in neurons and the effect of SEN817 was not significant in Drosophila. Although SEN177 had the highest in vitro activity and was able to efficiently reduce aggregates in mammalian cells, primary neurons and Drosophila, we found that this compound was tolerated at much higher concentrations than its analogs in zebrafish and therefore the bioavailability in this model is much lower, which could explain the lack of effect in this system. All together, we have identified a number of small molecules that through QPCT inhibition have beneficial effects on the treatment of HD in a variety of in vivo models.

Primary cell screen for suppressors of mutant Htt toxicityWe performed the primary screen using a stable HEK293/T Rex cell line expressing full-length human HTT bearing 138 polyglutamines (Q138) under the control of a tetracycline-inducible promoter. We confirmed the expression of HTT(Q138) after inducing the cells with doxycycline using antibodies recognizing the N-terminus of human HTT (Supplementary Results, Supplementary Fig. 1a and Supplementary Note 1), and quantitative RT-PCR using primers spanning different areas of the human HTT cDNA (Supplementary Fig. 1b). This cell line had reduced cell viability after expression of mutant HTT, which was reverted by treatment with a known reference compound (Y27632) 12 (Supplementary Fig. 1c), suggesting that this model could be used to identify potential modulators of mutant HTT cellular toxicity in a large-scale screen.For our high-throughput screen, we utilised a strategy consisting of an iterative siRNA screen where positive genes were selected after three consecutive rounds to compensate for the variability of the assay. We eliminated non-positive siRNAs and added new siRNAs targeting the selected genes in consecutive passes. We assessed rescue of cellular toxicity by each siRNA by fluorescence microscopy and automated image analysis using three independent readouts: 1) number of cell nuclei (#nuclei), 2) apoptotic index and 3) aberrant nuclei index, and used rescue indices to express the effect of each individual siRNA for each parameter analysed. In an initial screen, we tested 3 independent siRNAs for each of the 5,623 genes (a total of 16,869 siRNAs), from which we selected 670 primary genes (see Supplementary Note 1 for screen assay and criteria selection). As shown in supplementary figure 2a, the three readouts were partially redundant, as more than 50% of the 1,000 top scoring siRNAs of one rescue index also ranked amongst the top 1,000 siRNAs of at least one of the other rescue indices. In supplementary figure 1b, a representation of rescue indices obtained in pass 1 shows the relatively large variability of the assay, with non-targeting negative control siRNAs, negQ and negF, showing a #nuclei rescue indices of 14% and 3% respectively, while using siRNA targeting HTT as a positive control rendered a mean #nuclei rescue index of 81%.After 3 consecutive rounds of screening, we selected and 257 genes and ranked these based on all three rescue indices, using #nuclei rescue index as a primary criterion (Supplementary Data Set 1).

Primary cell screen for suppressors of mutant Htt toxicity

We performed the primary screen using a stable HEK293/T Rex cell line expressing full-length human HTT bearing 138 polyglutamines (Q138) under the control of a tetracycline-inducible promoter. We confirmed the expression of HTT(Q138) after inducing the cells with doxycycline using antibodies recognizing the N-terminus of human HTT (Supplementary Results, Supplementary Fig. 1a and Supplementary Note 1), and quantitative RT-PCR using primers spanning different areas of the human HTT cDNA (Supplementary Fig. 1b). This cell line had reduced cell viability after expression of mutant HTT, which was reverted by treatment with a known reference compound (Y27632) 12 (Supplementary Fig. 1c), suggesting that this model could be used to identify potential modulators of mutant HTT cellular toxicity in a large-scale screen.

For our high-throughput screen, we utilised a strategy consisting of an iterative siRNA screen where positive genes were selected after three consecutive rounds to compensate for the variability of the assay. We eliminated non-positive siRNAs and added new siRNAs targeting the selected genes in consecutive passes. We assessed rescue of cellular toxicity by each siRNA by fluorescence microscopy and automated image analysis using three independent readouts: 1) number of cell nuclei (#nuclei), 2) apoptotic index and 3) aberrant nuclei index, and used rescue indices to express the effect of each individual siRNA for each parameter analysed. In an initial screen, we tested 3 independent siRNAs for each of the 5,623 genes (a total of 16,869 siRNAs), from which we selected 670 primary genes (see Supplementary Note 1 for screen assay and criteria selection). As shown in supplementary figure 2a, the three readouts were partially redundant, as more than 50% of the 1,000 top scoring siRNAs of one rescue index also ranked amongst the top 1,000 siRNAs of at least one of the other rescue indices. In supplementary figure 1b, a representation of rescue indices obtained in pass 1 shows the relatively large variability of the assay, with non-targeting negative control siRNAs, negQ and negF, showing a #nuclei rescue indices of 14% and 3% respectively, while using siRNA targeting HTT as a positive control rendered a mean #nuclei rescue index of 81%.

After 3 consecutive rounds of screening, we selected and 257 genes and ranked these based on all three rescue indices, using #nuclei rescue index as a primary criterion (Supplementary Data Set 1).

Secondary RNAi screening in a Drosophila model of HDTo validate the hits obtained in mammalian cells and to focus on targets with potential relevance in vivo, we performed a secondary screen in a Drosophila model that expressed a construct containing 48 polyglutamines, Q48, that causes eye degeneration when expressed using a GMR-GAL4 driver 13. For most genes selected, we studied two UAS-RNAi constructs from the Vienna Drosophila RNAi Center (VDRC): a P-element (GD) and a phiC31 (KK) construct, the latter of which carries more GAL4-binding sites and should therefore express the RNAi more strongly 14. Of the 257 mammalian genes previously selected, we detected 133 that had one or more gene orthologs in flies (Supplementary Data Set 1 and 2). Of these 133 mammalian genes with fly orthologs, 74 Drosophila genes (corresponding to 66 mammalian genes) rescued the Q48-induced eye degeneration with at least one RNAi line, while the others showed no obvious or significant effect (Supplementary Fig. 3a and 3b and Supplementary Data Sets 1 and 2). We crossed suppressor RNAi lines to transgenic flies that expressed EGFP, also driven by the same GMR-GAL4 driver. We used EGFP to test whether modifiers affected transgene protein synthesis, since Q48 levels can be modified by aggregation or autophagic degradation, which do not impact EGFP levels. Two of these fly RNAi lines, targeting orthologs to human CTSF and to human ADAM8, ADAM11 and ADAM33, reduced EGFP levels on western blots (Supplementary Data Set 2), suggesting a general effect of these genes in protein expression, while suppression exerted by the other RNAi lines seemed to be polyglutamine-specific.

Secondary RNAi screening in a Drosophila model of HD

To validate the hits obtained in mammalian cells and to focus on targets with potential relevance in vivo, we performed a secondary screen in a Drosophila model that expressed a construct containing 48 polyglutamines, Q48, that causes eye degeneration when expressed using a GMR-GAL4 driver 13. For most genes selected, we studied two UAS-RNAi constructs from the Vienna Drosophila RNAi Center (VDRC): a P-element (GD) and a phiC31 (KK) construct, the latter of which carries more GAL4-binding sites and should therefore express the RNAi more strongly 14. Of the 257 mammalian genes previously selected, we detected 133 that had one or more gene orthologs in flies (Supplementary Data Set 1 and 2). Of these 133 mammalian genes with fly orthologs, 74 Drosophila genes (corresponding to 66 mammalian genes) rescued the Q48-induced eye degeneration with at least one RNAi line, while the others showed no obvious or significant effect (Supplementary Fig. 3a and 3b and Supplementary Data Sets 1 and 2). We crossed suppressor RNAi lines to transgenic flies that expressed EGFP, also driven by the same GMR-GAL4 driver. We used EGFP to test whether modifiers affected transgene protein synthesis, since Q48 levels can be modified by aggregation or autophagic degradation, which do not impact EGFP levels. Two of these fly RNAi lines, targeting orthologs to human CTSF and to human ADAM8, ADAM11 and ADAM33, reduced EGFP levels on western blots (Supplementary Data Set 2), suggesting a general effect of these genes in protein expression, while suppression exerted by the other RNAi lines seemed to be polyglutamine-specific.

Functional categorization of mutant HTT modifiersTo gain further insight into the biological relevance of the data generated, we categorized the different sets of HD toxicity modulators according to their molecular function. Suppressors were enriched for certain classes of proteins such as GPCRs or transporters compared to the initial library, while the number of positive kinases in the screen was reduced and no cytokines, growth factors or translational regulators were represented. We observed similar functional categorizations after selection from the cell and Drosophila screen (Supplementary Fig. 4a). An Ingenuity Pathway Analysis (IPA) of the hits obtained in the primary screen in cells (Supplementary Table 1a) revealed that the majority of these proteins participate in general processes such as GPCR- or cAMP-mediated signalling, but also in canonical pathways related to neurodegeneration, such as apoptosis, mitochondrial dysfunction, amyloid processing or protein ubiquitination. Importantly, 10 of these proteins have been previously related to HD signalling, including subunits of the succinate dehydrogenase complex and huntingtin-associated protein 1 (HAP1) (Supplementary Table 1a). Many of the genes validated in Drosophila (Supplementary Fig. 4b and Supplementary Table 1b) are also involved in processes related to neurodegeneration but were enriched in mitochondrial metabolic pathways, especially those associated with fatty acid biosynthesis and metabolism.

Functional categorization of mutant HTT modifiers

To gain further insight into the biological relevance of the data generated, we categorized the different sets of HD toxicity modulators according to their molecular function. Suppressors were enriched for certain classes of proteins such as GPCRs or transporters compared to the initial library, while the number of positive kinases in the screen was reduced and no cytokines, growth factors or translational regulators were represented. We observed similar functional categorizations after selection from the cell and Drosophila screen (Supplementary Fig. 4a). An Ingenuity Pathway Analysis (IPA) of the hits obtained in the primary screen in cells (Supplementary Table 1a) revealed that the majority of these proteins participate in general processes such as GPCR- or cAMP-mediated signalling, but also in canonical pathways related to neurodegeneration, such as apoptosis, mitochondrial dysfunction, amyloid processing or protein ubiquitination. Importantly, 10 of these proteins have been previously related to HD signalling, including subunits of the succinate dehydrogenase complex and huntingtin-associated protein 1 (HAP1) (Supplementary Table 1a). Many of the genes validated in Drosophila (Supplementary Fig. 4b and Supplementary Table 1b) are also involved in processes related to neurodegeneration but were enriched in mitochondrial metabolic pathways, especially those associated with fatty acid biosynthesis and metabolism.

Validation of QPCT in DrosophilaWe focused our attention on a gene that had one of the strongest and most consistent effects in rescuing mutant HTT-induced toxicity in the cell-based siRNA screen. The gene product has glutaminyl cyclase activity and is named QPCT. Two orthologs have been reported in fly 15, Glutaminyl cyclase (QC) and iso Glutaminyl cyclase (isoQC), which show about 39% amino acid identity; a third fly ortholog, CG6168, shows expression restricted to male accessory glands (www.flyatlas.org) and is not considered further here. RNAi lines targeting either QC or isoQC partially rescued eye depigmentation and mediated a significant decrease in the number of black spots in flies expressing Q48 (Fig. 1a, 1b and Supplementary Fig. 5a) (Data are shown for GD- and KK-RNAi lines in the case of QC, but only a KK line was available for isoQC). These effects are likely independent of transcription/translation of the Q48, since no change in EGFP protein levels were seen when we crossed transgenic flies expressing EGFP driven by the same GMR-GAL4 driver as Q48 with QC or isoQC RNAi lines (Supplementary Fig. 5b). Thus, QPCT represents an interesting candidate for studying in HD.To further evaluate the benefits of downregulating QPCT on HD, we took advantage of an additional Drosophila model of neurodegeneration, HD flies that express exon 1 of HTT with 120 polyglutamines, GMR-HTT.Q120 in eye photoreceptors 16. Drosophila melanogaster has a compound eye consisting of many ommatidia, each of which is composed of 8 photoreceptors, seven of which can be visualized by light microscopy using the pseudopupil technique 17. Neurodegeneration results in the loss of visible rhabdomeres of each photoreceptor and can be rescued or enhanced by genetic or chemical approaches 18. Consistent with our data using the Q48 flies, the loss of visible photoreceptors in transgenic flies expressing GMR-HTT.Q120 was partially rescued when they were crossed with RNAi lines for either of the two QPCT fly orthologues, QC and isoQC (Fig. 1c). We observed no effect on the number of rhabdomeres in QPCT RNAi lines in the absence of GMR-HTT.Q120. The effects of QPCT knockdown on toxicity correlated with a reduction in HTT aggregation, which we assessed in flies expressing GFP-tagged expanded huntingtin exon 1, HTTEx1-Q46-eGFP in the eye 19 (Fig. 1d).

Validation of QPCT in Drosophila

We focused our attention on a gene that had one of the strongest and most consistent effects in rescuing mutant HTT-induced toxicity in the cell-based siRNA screen. The gene product has glutaminyl cyclase activity and is named QPCT. Two orthologs have been reported in fly 15, Glutaminyl cyclase (QC) and iso Glutaminyl cyclase (isoQC), which show about 39% amino acid identity; a third fly ortholog, CG6168, shows expression restricted to male accessory glands (www.flyatlas.org) and is not considered further here. RNAi lines targeting either QC or isoQC partially rescued eye depigmentation and mediated a significant decrease in the number of black spots in flies expressing Q48 (Fig. 1a, 1b and Supplementary Fig. 5a) (Data are shown for GD- and KK-RNAi lines in the case of QC, but only a KK line was available for isoQC). These effects are likely independent of transcription/translation of the Q48, since no change in EGFP protein levels were seen when we crossed transgenic flies expressing EGFP driven by the same GMR-GAL4 driver as Q48 with QC or isoQC RNAi lines (Supplementary Fig. 5b). Thus, QPCT represents an interesting candidate for studying in HD.

To further evaluate the benefits of downregulating QPCT on HD, we took advantage of an additional Drosophila model of neurodegeneration, HD flies that express exon 1 of HTT with 120 polyglutamines, GMR-HTT.Q120 in eye photoreceptors 16. Drosophila melanogaster has a compound eye consisting of many ommatidia, each of which is composed of 8 photoreceptors, seven of which can be visualized by light microscopy using the pseudopupil technique 17. Neurodegeneration results in the loss of visible rhabdomeres of each photoreceptor and can be rescued or enhanced by genetic or chemical approaches 18. Consistent with our data using the Q48 flies, the loss of visible photoreceptors in transgenic flies expressing GMR-HTT.Q120 was partially rescued when they were crossed with RNAi lines for either of the two QPCT fly orthologues, QC and isoQC (Fig. 1c). We observed no effect on the number of rhabdomeres in QPCT RNAi lines in the absence of GMR-HTT.Q120. The effects of QPCT knockdown on toxicity correlated with a reduction in HTT aggregation, which we assessed in flies expressing GFP-tagged expanded huntingtin exon 1, HTTEx1-Q46-eGFP in the eye 19 (Fig. 1d).

QPCT modulates mutant HTT aggregationTo further validate QPCT, we first confirmed the protective effect of its knockdown against toxicity and aggregation in HEK293 cells expressing the exon 1 of HTT (from residue 8) with a 74 polyglutamine expansion fused at its N-terminal to EGFP (EGFP-HTT(Q74)) 20 (Fig. 2a, Supplementary fig. 6a and 6b). The QPCT siRNAs used in these experiments as well as in the screen do not target QPCT-like, which encodes a paralogous protein that catalyzes a similar reaction and shows 51% of sequence identity to QPCT (Supplementary Fig. 6b and 6c). We also validated the effect of QPCT knockdown on aggregation in HeLa cells (Supplementary Fig. 6d) which, like HEK293 cells, express QPCT 21. We also confirmed a decrease in protein aggregation of a construct which expresses full-length HTT carrying 138 polyglutamines (similar to the one used in the initial screen) (Supplementary Fig. 6e). QPCT siRNA did not have a general anti-apoptotic effect as it did not affect caspase 3 activity in response to staurosporine treatment (Supplementary Fig. 6f). Consistent with these data, QPCT shRNA reduced EGFP-Q80 (80 glutamines fused to EGFP) aggregation in primary cortical neurons (Fig. 2b and Supplementary fig. 6g). We could not assess the effect of QPCT knockdown on polyglutamine-mediated toxicity in these neurons, where the levels of cell death obtained in this assay were very low, as can be appreciated in fig. 2b. While knocking down QPCT was protective, overexpression of QPCT in HeLa and HEK293 cells increased the numbers of apoptotic nuclei and also led to a large accumulation of HTT(Q74) aggregates (Fig. 2c and Supplementary Fig. 7a), while QPCT did not increase caspase activity upon staurosporine treatment (Supplementary Fig. 7b). The effects of QPCT were activity-dependent, since the catalytically inactive E201Q mutant did not increase the percentage of cells with HTT(Q74) aggregates (Fig. 2d and Supplementary Fig. 7c and 7d).We measured mRNA levels of QPCT in HD mice and found that its expression was reduced when compared to their wild-type littermates, suggesting that QPCT expression may be downregulated as a compensatory mechanism (Supplementary Fig. 8) and that raised QPCT activity may not be a prerequisite for aggregation.QPCT catalyzes the modification of N-terminal glutamines or glutamates into a pyroglutamate (pE) residue. Although the presence of an extended polyglutamine tract makes HTT a potential substrate for QPCT, this enzyme only modifies N-terminal residues, suggesting that any modification on mutant HTT would require an N-terminal cleavage to reveal a glutamine at the N-terminal that could be cyclated. The formation of a pE residue may then affect its stability and propensity to aggregate, a hypothesis that was previously suggested 22. This cleavage model in either the polyglutamine tract or HTT exon 1 or GFP is unlikely, as QPCT modulated the aggregation of constructs consisting only of isolated polyglutamine expansions (Q57 and Q81) fused C-terminal to EGFP (Fig. 2e and 2f), or HTT exon 1 with 74 glutamines fused to HA 23 (Supplementary Fig. 9a and 9b), and QPCT siRNA also reduced the aggregation of an expansion of 37 alanines 24 (Fig. 2f). QPCT appeared to modulate the early stages of mutant HTT oligomerisation, since QPCT overexpression increased the amounts of Flag-tagged monomeric mutant HTT that were co-immunoprecipitated by GFP-tagged mutant HTT (Fig. 2g) 25. Since QPCT did not interact with HTT directly by immunoprecipitation (e.g. Fig. 2g), its effect on HTT oligomer formation is likely mediated via intermediaries.

QPCT modulates mutant HTT aggregation

To further validate QPCT, we first confirmed the protective effect of its knockdown against toxicity and aggregation in HEK293 cells expressing the exon 1 of HTT (from residue 8) with a 74 polyglutamine expansion fused at its N-terminal to EGFP (EGFP-HTT(Q74)) 20 (Fig. 2a, Supplementary fig. 6a and 6b). The QPCT siRNAs used in these experiments as well as in the screen do not target QPCT-like, which encodes a paralogous protein that catalyzes a similar reaction and shows 51% of sequence identity to QPCT (Supplementary Fig. 6b and 6c). We also validated the effect of QPCT knockdown on aggregation in HeLa cells (Supplementary Fig. 6d) which, like HEK293 cells, express QPCT 21. We also confirmed a decrease in protein aggregation of a construct which expresses full-length HTT carrying 138 polyglutamines (similar to the one used in the initial screen) (Supplementary Fig. 6e). QPCT siRNA did not have a general anti-apoptotic effect as it did not affect caspase 3 activity in response to staurosporine treatment (Supplementary Fig. 6f). Consistent with these data, QPCT shRNA reduced EGFP-Q80 (80 glutamines fused to EGFP) aggregation in primary cortical neurons (Fig. 2b and Supplementary fig. 6g). We could not assess the effect of QPCT knockdown on polyglutamine-mediated toxicity in these neurons, where the levels of cell death obtained in this assay were very low, as can be appreciated in fig. 2b. While knocking down QPCT was protective, overexpression of QPCT in HeLa and HEK293 cells increased the numbers of apoptotic nuclei and also led to a large accumulation of HTT(Q74) aggregates (Fig. 2c and Supplementary Fig. 7a), while QPCT did not increase caspase activity upon staurosporine treatment (Supplementary Fig. 7b). The effects of QPCT were activity-dependent, since the catalytically inactive E201Q mutant did not increase the percentage of cells with HTT(Q74) aggregates (Fig. 2d and Supplementary Fig. 7c and 7d).

We measured mRNA levels of QPCT in HD mice and found that its expression was reduced when compared to their wild-type littermates, suggesting that QPCT expression may be downregulated as a compensatory mechanism (Supplementary Fig. 8) and that raised QPCT activity may not be a prerequisite for aggregation.

QPCT catalyzes the modification of N-terminal glutamines or glutamates into a pyroglutamate (pE) residue. Although the presence of an extended polyglutamine tract makes HTT a potential substrate for QPCT, this enzyme only modifies N-terminal residues, suggesting that any modification on mutant HTT would require an N-terminal cleavage to reveal a glutamine at the N-terminal that could be cyclated. The formation of a pE residue may then affect its stability and propensity to aggregate, a hypothesis that was previously suggested 22. This cleavage model in either the polyglutamine tract or HTT exon 1 or GFP is unlikely, as QPCT modulated the aggregation of constructs consisting only of isolated polyglutamine expansions (Q57 and Q81) fused C-terminal to EGFP (Fig. 2e and 2f), or HTT exon 1 with 74 glutamines fused to HA 23 (Supplementary Fig. 9a and 9b), and QPCT siRNA also reduced the aggregation of an expansion of 37 alanines 24 (Fig. 2f). QPCT appeared to modulate the early stages of mutant HTT oligomerisation, since QPCT overexpression increased the amounts of Flag-tagged monomeric mutant HTT that were co-immunoprecipitated by GFP-tagged mutant HTT (Fig. 2g) 25. Since QPCT did not interact with HTT directly by immunoprecipitation (e.g. Fig. 2g), its effect on HTT oligomer formation is likely mediated via intermediaries.

Design and characterization of compounds that inhibit QPCTTo target QPCT pharmacologically, we tested a previously described QPCT inhibitor 26, which did not rescue the HD phenotype in mammalian cells (Supplementary Fig. 10a and 10b). While this compound has been effective in Alzheimer disease (AD) models by reducing the formation of extracellular pE-Aβ, this may be due to extracellular QPCT inhibition 2127. Thus, we reasoned that the failure of this compound was likely due to poor cell permeability. In order to generate novel QPCT inhibitors, we employed existing data on its structure and known inhibitors to generate three 3D pharmacophore models, two ligand-based and one structure-based (using the human QPCT X-Ray structure (PDB id: 2AFW)). We used these models, along with stringently applied CNS filters and a solubility model developed in-house, to select 10,000 compounds from both commercially available screening compounds and the SienaBiotech compound library. We screened these molecules in a functional assay assessing the conversion of the H-Glu-AMC fluorogenic substrate into pyroGlu-AMC, as previously described 28, and selected hits associated with predicted robust binding for the hit-to-lead phase. The optimization strategy was based on physicochemical properties and ensemble docking model-driven approaches. The ensemble docking methodology 2930 was chosen to take into account the flexibility of human QPCT catalytic site and was constructed using both X-Ray structures and protein conformations coming from a 100 ns molecular dynamic study of the human QPCT 2AFW X-Ray structure. The ensemble docking model was evolved during the project development. Initially, only 4 X-Ray structures were used (PDB ID: 2AFW, 2AFX, 2AFZ31), then a set of 16 protein conformations, selected by clustering of molecular dynamic simulations, were added to improve model accuracy. Recently, two more X-Ray structures were added to the model (3PBB32 an 3SI033). All the docking calculations were performed using CCDC Gold (versions 4 and 5)343536 along with an ad-hoc developed program to rank and select the best scored ligand docking pose from the pool of QPCT conformations. Along with the biochemical readouts used during this optimization, we included a range of in vitro ADME assays, including solubility measurements, a CNS membrane permeability assay (PAMPA-BBB) 37 and stability in the presence of human CYP3A4, a member of the cytochrome P450 mixed-function oxidase system, and a key enzyme involved in the metabolism of xenobiotics in humans.We selected a series of compounds on the basis of these properties and validated their effects on mutant HTT aggregation and toxicity in cells expressing HTT(Q74)GFP, which led to the selection of three of them, SEN177 (1), SEN817 (2) and SEN180 (3) (Supplementary note 2, Fig. 3a, Supplementary Fig. 11a). Non-toxic concentrations of these compounds caused a dose-dependent reduction in the percentage of cells with aggregates, which correlated with a suppression of mutant HTT-induced apoptosis (Fig. 3b, 3c, 3d, 3d and Supplementary Fig. 11b). As seen with genetic knockdown experiments, pharmacologic inhibition of QPCT using these compounds also reduced aggregation of polyalanines (Fig. 3d) and did not affect protein levels, as assessed by measuring GFP levels by western blotting (Supplementary Fig. 11c) or by metabolic labeling of wild type HTT followed by detection of newly synthesized protein in the presence of SEN177 (Supplementary Fig. 11d). Importantly, the effect of these compounds was blocked when QPCT expression was suppressed by shRNA, confirming that they protect by a mechanism that requires QPCT inhibition (Fig 3e, Supplementary Fig. 11e and 11f). Thus, even though these compounds also inhibited QPCT-like (Supplementary Fig. 11a) and we cannot exclude the possibility that at least some of the effects observed may be mediated by this QPCT isoenzyme, their effects on aggregation were QPCT-dependent, as the shRNA used did not target QPCT-like. Consistent with these data, SEN177 greatly reduced the early stages of mutant HTT oligomerisation, as it decreased the amounts of GFP-tagged monomeric HTT that were co-immunoprecipitated by Flag-tagged HTT (Fig. 3f). The protective effect of these compounds was also confirmed in primary cortical neurons (Fig. 3g), with SEN177 and SEN817 significantly reducing the percentage of neurons with Q80 aggregates.

Design and characterization of compounds that inhibit QPCT

To target QPCT pharmacologically, we tested a previously described QPCT inhibitor 26, which did not rescue the HD phenotype in mammalian cells (Supplementary Fig. 10a and 10b). While this compound has been effective in Alzheimer disease (AD) models by reducing the formation of extracellular pE-Aβ, this may be due to extracellular QPCT inhibition 2127. Thus, we reasoned that the failure of this compound was likely due to poor cell permeability. In order to generate novel QPCT inhibitors, we employed existing data on its structure and known inhibitors to generate three 3D pharmacophore models, two ligand-based and one structure-based (using the human QPCT X-Ray structure (PDB id: 2AFW)). We used these models, along with stringently applied CNS filters and a solubility model developed in-house, to select 10,000 compounds from both commercially available screening compounds and the SienaBiotech compound library. We screened these molecules in a functional assay assessing the conversion of the H-Glu-AMC fluorogenic substrate into pyroGlu-AMC, as previously described 28, and selected hits associated with predicted robust binding for the hit-to-lead phase. The optimization strategy was based on physicochemical properties and ensemble docking model-driven approaches. The ensemble docking methodology 2930 was chosen to take into account the flexibility of human QPCT catalytic site and was constructed using both X-Ray structures and protein conformations coming from a 100 ns molecular dynamic study of the human QPCT 2AFW X-Ray structure. The ensemble docking model was evolved during the project development. Initially, only 4 X-Ray structures were used (PDB ID: 2AFW, 2AFX, 2AFZ31), then a set of 16 protein conformations, selected by clustering of molecular dynamic simulations, were added to improve model accuracy. Recently, two more X-Ray structures were added to the model (3PBB32 an 3SI033). All the docking calculations were performed using CCDC Gold (versions 4 and 5)343536 along with an ad-hoc developed program to rank and select the best scored ligand docking pose from the pool of QPCT conformations. Along with the biochemical readouts used during this optimization, we included a range of in vitro ADME assays, including solubility measurements, a CNS membrane permeability assay (PAMPA-BBB) 37 and stability in the presence of human CYP3A4, a member of the cytochrome P450 mixed-function oxidase system, and a key enzyme involved in the metabolism of xenobiotics in humans.

We selected a series of compounds on the basis of these properties and validated their effects on mutant HTT aggregation and toxicity in cells expressing HTT(Q74)GFP, which led to the selection of three of them, SEN177 (1), SEN817 (2) and SEN180 (3) (Supplementary note 2, Fig. 3a, Supplementary Fig. 11a). Non-toxic concentrations of these compounds caused a dose-dependent reduction in the percentage of cells with aggregates, which correlated with a suppression of mutant HTT-induced apoptosis (Fig. 3b, 3c, 3d, 3d and Supplementary Fig. 11b). As seen with genetic knockdown experiments, pharmacologic inhibition of QPCT using these compounds also reduced aggregation of polyalanines (Fig. 3d) and did not affect protein levels, as assessed by measuring GFP levels by western blotting (Supplementary Fig. 11c) or by metabolic labeling of wild type HTT followed by detection of newly synthesized protein in the presence of SEN177 (Supplementary Fig. 11d). Importantly, the effect of these compounds was blocked when QPCT expression was suppressed by shRNA, confirming that they protect by a mechanism that requires QPCT inhibition (Fig 3e, Supplementary Fig. 11e and 11f). Thus, even though these compounds also inhibited QPCT-like (Supplementary Fig. 11a) and we cannot exclude the possibility that at least some of the effects observed may be mediated by this QPCT isoenzyme, their effects on aggregation were QPCT-dependent, as the shRNA used did not target QPCT-like. Consistent with these data, SEN177 greatly reduced the early stages of mutant HTT oligomerisation, as it decreased the amounts of GFP-tagged monomeric HTT that were co-immunoprecipitated by Flag-tagged HTT (Fig. 3f). The protective effect of these compounds was also confirmed in primary cortical neurons (Fig. 3g), with SEN177 and SEN817 significantly reducing the percentage of neurons with Q80 aggregates.

QPCT modulates the levels of alpha B-crystallinThe effects of QPCT inhibition on HTT aggregation appeared to be independent of effects on protein clearance pathways targeting mutant huntingtin (autophagy and the ubiquitin-proteasome system) (Supplementary Fig. 12), changes in mRNA or protein levels (Supplementary Fig. 13a and 13b), or secretion of the enzyme into the medium (Supplementary Fig. 13c). QPCT is localized in the ER and secretory pathway and its knockdown, overexpression or inhibition seemed to have inconsistent and rather modest effects on different readouts of the ER stress response, measured by GRP78/BIP levels or phosphorylation of eIF2α, which did not correlate with its effect on aggregation (Supplementary Fig. 14). Our data also suggested that CREB (c-AMP response element binding protein) or ERK (extracellular signal-regulated kinase) signaling, recently reported to be activated upon QPCT inhibition 38 (Supplementary Fig. 15a and 15b), or JNK signaling (Supplementary Fig. 15c) were unlikely contributors to the effects we have observed.QPCT overexpression or knockdown did not modulate levels of HSP70, the main inducible stress response chaperone (Supplementary Fig. 15d). We performed transcriptional profiling to assess changes in alternative molecular chaperones induced by SEN177 in the presence of mutant HTT, and observed upregulation of several small heat shock proteins (sHSPs) (HSPB6 with 1.6 fold-change; HSPB3 with 1.5 fold-change; HSPB7 with 1.5 fold-change; and notably, alpha B-crystallin which had >2.5 fold increase in transcript levels) (Supplementary Fig. 16a and Supplementary Data Set 3). We confirmed this induction at the protein level as well as with other QPCT inhibitors (Fig. 4a). Genetic inhibition of QPCT dramatically increased alpha B-crystallin protein and mRNA levels in the presence of HTT(Q74) (Fig. 4b and 4c and Supplementary Fig. 16b), while QPCT overexpression, which increased mutant HTT aggregation and toxicity (Fig. 2c and Supplementary Fig. 7a), reduced alpha B-crystallin levels (Supplementary Fig. 16c). QPCT also modestly modulated alpha B-crystallin levels in the absence of mutant HTT or in the presence of the non-pathogenic Q23 (Supplementary Fig. 16b and 16c).As a sHSP, alpha B-crystallin acts as a molecular chaperone and is a suppressor of polyglutamine toxicity in cells and in Drosophila
394041. As expected, overexpression of alpha B-crystallin lowered the number of HTT(Q74) aggregates, while QPCT inhibitors failed to reduce aggregation further (Fig. 4d and Supplementary Fig. 16d), suggesting that this increment in alpha B-crystallin was a major contributor to the protection afforded by QPCT inhibition.

QPCT modulates the levels of alpha B-crystallin

The effects of QPCT inhibition on HTT aggregation appeared to be independent of effects on protein clearance pathways targeting mutant huntingtin (autophagy and the ubiquitin-proteasome system) (Supplementary Fig. 12), changes in mRNA or protein levels (Supplementary Fig. 13a and 13b), or secretion of the enzyme into the medium (Supplementary Fig. 13c). QPCT is localized in the ER and secretory pathway and its knockdown, overexpression or inhibition seemed to have inconsistent and rather modest effects on different readouts of the ER stress response, measured by GRP78/BIP levels or phosphorylation of eIF2α, which did not correlate with its effect on aggregation (Supplementary Fig. 14). Our data also suggested that CREB (c-AMP response element binding protein) or ERK (extracellular signal-regulated kinase) signaling, recently reported to be activated upon QPCT inhibition 38 (Supplementary Fig. 15a and 15b), or JNK signaling (Supplementary Fig. 15c) were unlikely contributors to the effects we have observed.

QPCT overexpression or knockdown did not modulate levels of HSP70, the main inducible stress response chaperone (Supplementary Fig. 15d). We performed transcriptional profiling to assess changes in alternative molecular chaperones induced by SEN177 in the presence of mutant HTT, and observed upregulation of several small heat shock proteins (sHSPs) (HSPB6 with 1.6 fold-change; HSPB3 with 1.5 fold-change; HSPB7 with 1.5 fold-change; and notably, alpha B-crystallin which had >2.5 fold increase in transcript levels) (Supplementary Fig. 16a and Supplementary Data Set 3). We confirmed this induction at the protein level as well as with other QPCT inhibitors (Fig. 4a). Genetic inhibition of QPCT dramatically increased alpha B-crystallin protein and mRNA levels in the presence of HTT(Q74) (Fig. 4b and 4c and Supplementary Fig. 16b), while QPCT overexpression, which increased mutant HTT aggregation and toxicity (Fig. 2c and Supplementary Fig. 7a), reduced alpha B-crystallin levels (Supplementary Fig. 16c). QPCT also modestly modulated alpha B-crystallin levels in the absence of mutant HTT or in the presence of the non-pathogenic Q23 (Supplementary Fig. 16b and 16c).

As a sHSP, alpha B-crystallin acts as a molecular chaperone and is a suppressor of polyglutamine toxicity in cells and in Drosophila
394041. As expected, overexpression of alpha B-crystallin lowered the number of HTT(Q74) aggregates, while QPCT inhibitors failed to reduce aggregation further (Fig. 4d and Supplementary Fig. 16d), suggesting that this increment in alpha B-crystallin was a major contributor to the protection afforded by QPCT inhibition.

QPCT inhibition protects fly and zebrafish HD modelsWe tested QPCT inhibitors in flies expressing Httex1Q46 in the eye and found a reduction in the number of aggregates (Fig. 5a). The compound with a greatest effect, SEN177, was able to also rescue the number of visible rhabdomeres and prevent neurodegeneration (Fig. 5b).A transgenic zebrafish expressing Htt exon 1 with 71Q fused to EGFP in the rod photoreceptors using the rhodopsin promoter has been established and validated as a model to study mutant huntingtin aggregation in vivo 42. Zebrafish have two homologs with putative glutaminyl-peptide cyclotransferase activity, QPCT and QPCTLA with 51% and 47% protein identity with QPCT and QPCT-like respectively. In order to test the effect of pharmacologic inhibition of QPCT in this model, we first determined the maximum tolerated concentration for each of the three compounds tested in mammalian cells and subsequently treated HD larvae. SEN817 and SEN180 reduced total number of EGFP-aggregates in the retina (Fig. 6a), which correlated with a marked decrease in toxicity similar to the positive control, clonidine 42, assessed by a rescue in the total area of eye photoreceptors (Fig. 6b).Although the three compounds were protective, their effectiveness varied between these models, which might be due to intrinsic properties of each system, SEN180 only mildly reduced aggregation in neurons and the effect of SEN817 was not significant in Drosophila. Although SEN177 had the highest in vitro activity and was able to efficiently reduce aggregates in mammalian cells, primary neurons and Drosophila, we found that this compound was tolerated at much higher concentrations than its analogs in zebrafish and therefore the bioavailability in this model is much lower, which could explain the lack of effect in this system. All together, we have identified a number of small molecules that through QPCT inhibition have beneficial effects on the treatment of HD in a variety of in vivo models.

QPCT inhibition protects fly and zebrafish HD models

We tested QPCT inhibitors in flies expressing Httex1Q46 in the eye and found a reduction in the number of aggregates (Fig. 5a). The compound with a greatest effect, SEN177, was able to also rescue the number of visible rhabdomeres and prevent neurodegeneration (Fig. 5b).

A transgenic zebrafish expressing Htt exon 1 with 71Q fused to EGFP in the rod photoreceptors using the rhodopsin promoter has been established and validated as a model to study mutant huntingtin aggregation in vivo 42. Zebrafish have two homologs with putative glutaminyl-peptide cyclotransferase activity, QPCT and QPCTLA with 51% and 47% protein identity with QPCT and QPCT-like respectively. In order to test the effect of pharmacologic inhibition of QPCT in this model, we first determined the maximum tolerated concentration for each of the three compounds tested in mammalian cells and subsequently treated HD larvae. SEN817 and SEN180 reduced total number of EGFP-aggregates in the retina (Fig. 6a), which correlated with a marked decrease in toxicity similar to the positive control, clonidine 42, assessed by a rescue in the total area of eye photoreceptors (Fig. 6b).

Although the three compounds were protective, their effectiveness varied between these models, which might be due to intrinsic properties of each system, SEN180 only mildly reduced aggregation in neurons and the effect of SEN817 was not significant in Drosophila. Although SEN177 had the highest in vitro activity and was able to efficiently reduce aggregates in mammalian cells, primary neurons and Drosophila, we found that this compound was tolerated at much higher concentrations than its analogs in zebrafish and therefore the bioavailability in this model is much lower, which could explain the lack of effect in this system. All together, we have identified a number of small molecules that through QPCT inhibition have beneficial effects on the treatment of HD in a variety of in vivo models.

DiscussionOur approach using a two-step screen, starting with an initial large-scale analysis in human cell models followed by a validation in Drosophila, has yielded a number of potentially druggable targets which may be suitable for HD. A variety of HT-RNAi screens have identified genetic suppressors of phenotypes mediated by mutant HTT N-terminal fragments in Drosophila, C. elegans and mammalian (mouse and human) cells 44454647. In most cases, aggregation was the primary readout, often measured with C-terminal GFP fusions. Differences in the nature of the previous screens (species, cellular context, huntingtin fragment length, length of the polyglutamine expansion, primary readout and differences in siRNA/shRNA sequences) complicates cross-screen comparisons. Also, virtually no screens in this area have examined their false negative rates due to inefficient knockdown. Additionally, the screen presented here was biased towards the druggable component of the human genome, and a further selection was made in the course of triaging towards specific protein target classes. This likely contributes to the relatively poor overlap of hits in the present and previous screens. A comparison with a screen performed in HEK293T cells to identify genetic suppressors of inducibly expressed mutant HTT exon 1 toxicity 46 revealed an overlap of only 4 genes (CPA1, GRIN2A, NR3C2 and USP21) when considering the top 257 hits (Supplementary Data Set 1). However, matrix metalloproteases, identified in HEK293T cells as modulators of fragmentation and toxicity of N-terminal portions of mutant HTT 45 were also identified in our dataset, as well as PAK1, which we previously identified as a kinase promoting mutant HTT self-association and toxicity 25, thus validating the effectiveness of the screen.Based on the reproducible and clear rescue that QPCT inhibition exerts on mutant HTT toxicity in cells and in Drosophila, we focused on this target. A catalytically inactive QPCT was not able to increase the number of aggregates, suggesting that pE modifications modulate the levels of aggregates in HD models. Although one obvious mechanism would involve cleavage of the polyglutamine tract followed by cyclation of an N-terminal pE residue that may change properties such as stability or hydrophobicity, which would account for its change in aggregation 22, our data suggest that the effect of QPCT on HTT may be indirect. We found that modulation of aggregation by QPCT was not restricted to mutant HTT but it also affected aggregation of other aggregate-prone proteins and that QPCT influences the formation of mutant HTT oligomeric species. We observed an induction in several sHSPs, mostly alpha B-crystallin, suggesting that QPCT inhibition caused a stress response distinct from classical Hsp70 induction, which might be mediated by indirect substrates for pE modification. This molecular chaperone reduces aggregation of polyglutamine containing proteins 3941, alpha-synuclein 4841or amyloid-β peptide 4950, underscoring QPCT inhibition as an effective target for misfolded protein disorders. Since alpha B-crystallin is regulated at the transcriptional level while QPCT resides in the secretory pathway, inhibition of QPCT may activate a signalling response that enhance alpha B-crystallin transcription. Our data suggest that this is likely independent of an ER stress response or the involvement of ERK and CREB, which have been recently found phosphorylated upon QPCT inhibition 38, as well as other stress signalling pathways such as JNK. Further work will need to clarify the QPCT substrate mediating this effect. It is important to stress that the benefits of QPCT downregulation may not be restricted to alpha B-crystallin as an effector, as the upregulation of other related sHSPs may also contribute beneficially.We identified and characterised a series of compounds that efficiently reduce mutant HTT aggregation in mammalian cell lines and also in primary mouse neurons, fly eye and in zebrafish. While the levels of rescue and significance obtained varied between compounds depending on the model used, this may be as a result of differences in absorption routes and bioavailability. Nevertheless, our data showed that pharmacologic inhibition of QPCT using this compound series can rescue HD phenotypes and provides proof-of-principle for QPCT as a potential therapeutic target for HD and possibly other related intracellular proteinopathies by modulating the formation of oligomeric forms, which have been proposed as the most toxic species in these diseases 78. Clearly, further work is required before considering that this will be clinically relevant, including likely additional drug development. Nevertheless, in a broader perspective, our data suggest that a discovery pipeline from druggable genome screen to drug development may be tractable for neurodegenerative diseases.

Our approach using a two-step screen, starting with an initial large-scale analysis in human cell models followed by a validation in Drosophila, has yielded a number of potentially druggable targets which may be suitable for HD. A variety of HT-RNAi screens have identified genetic suppressors of phenotypes mediated by mutant HTT N-terminal fragments in Drosophila, C. elegans and mammalian (mouse and human) cells 44454647. In most cases, aggregation was the primary readout, often measured with C-terminal GFP fusions. Differences in the nature of the previous screens (species, cellular context, huntingtin fragment length, length of the polyglutamine expansion, primary readout and differences in siRNA/shRNA sequences) complicates cross-screen comparisons. Also, virtually no screens in this area have examined their false negative rates due to inefficient knockdown. Additionally, the screen presented here was biased towards the druggable component of the human genome, and a further selection was made in the course of triaging towards specific protein target classes. This likely contributes to the relatively poor overlap of hits in the present and previous screens. A comparison with a screen performed in HEK293T cells to identify genetic suppressors of inducibly expressed mutant HTT exon 1 toxicity 46 revealed an overlap of only 4 genes (CPA1, GRIN2A, NR3C2 and USP21) when considering the top 257 hits (Supplementary Data Set 1). However, matrix metalloproteases, identified in HEK293T cells as modulators of fragmentation and toxicity of N-terminal portions of mutant HTT 45 were also identified in our dataset, as well as PAK1, which we previously identified as a kinase promoting mutant HTT self-association and toxicity 25, thus validating the effectiveness of the screen.

Based on the reproducible and clear rescue that QPCT inhibition exerts on mutant HTT toxicity in cells and in Drosophila, we focused on this target. A catalytically inactive QPCT was not able to increase the number of aggregates, suggesting that pE modifications modulate the levels of aggregates in HD models. Although one obvious mechanism would involve cleavage of the polyglutamine tract followed by cyclation of an N-terminal pE residue that may change properties such as stability or hydrophobicity, which would account for its change in aggregation 22, our data suggest that the effect of QPCT on HTT may be indirect. We found that modulation of aggregation by QPCT was not restricted to mutant HTT but it also affected aggregation of other aggregate-prone proteins and that QPCT influences the formation of mutant HTT oligomeric species. We observed an induction in several sHSPs, mostly alpha B-crystallin, suggesting that QPCT inhibition caused a stress response distinct from classical Hsp70 induction, which might be mediated by indirect substrates for pE modification. This molecular chaperone reduces aggregation of polyglutamine containing proteins 3941, alpha-synuclein 4841or amyloid-β peptide 4950, underscoring QPCT inhibition as an effective target for misfolded protein disorders. Since alpha B-crystallin is regulated at the transcriptional level while QPCT resides in the secretory pathway, inhibition of QPCT may activate a signalling response that enhance alpha B-crystallin transcription. Our data suggest that this is likely independent of an ER stress response or the involvement of ERK and CREB, which have been recently found phosphorylated upon QPCT inhibition 38, as well as other stress signalling pathways such as JNK. Further work will need to clarify the QPCT substrate mediating this effect. It is important to stress that the benefits of QPCT downregulation may not be restricted to alpha B-crystallin as an effector, as the upregulation of other related sHSPs may also contribute beneficially.

We identified and characterised a series of compounds that efficiently reduce mutant HTT aggregation in mammalian cell lines and also in primary mouse neurons, fly eye and in zebrafish. While the levels of rescue and significance obtained varied between compounds depending on the model used, this may be as a result of differences in absorption routes and bioavailability. Nevertheless, our data showed that pharmacologic inhibition of QPCT using this compound series can rescue HD phenotypes and provides proof-of-principle for QPCT as a potential therapeutic target for HD and possibly other related intracellular proteinopathies by modulating the formation of oligomeric forms, which have been proposed as the most toxic species in these diseases 78. Clearly, further work is required before considering that this will be clinically relevant, including likely additional drug development. Nevertheless, in a broader perspective, our data suggest that a discovery pipeline from druggable genome screen to drug development may be tractable for neurodegenerative diseases.

Online MethodsAssays for validation polyglutamine toxicity modifiers in DrosophilaDrosophila fly stocksAs a model of polyglutamine toxicity, flies that expressed a protein with 48 glutamines encoded by P{UAS-Q48.myc/flag}31
13 in eyes under control of the GMR-Gal4 driver P{GAL4-ninaE.GMR}12
51 (Q48) were used. Fly orthologs to the genes identified in the cell screen were selected by performing reciprocal BLASTP and cross checking with databases including http://www.ncbi.nlm.nih.gov/homologene, http://www.genecards.org/, http://www.ensembl.org/index.html. The RNAi lines corresponding to the identified genes were obtained from Vienna Drosophila RNAi Center (VDRC, http://stockcenter.vdrc.at/control/main).The following stocks were generous gifts: UAS-Q48.myc/flag from J.L. Marsh 13, UAS-Httex1-Q46-eGFP from N. Perrimon 19. Fly lines that are not referenced here are documented in FlyBase (www.flybase.org).All fly crosses and experiments were performed at 25°C.Drosophila RNAi screenFive virgins of genotype w; GMR-GAL4; UAS-Q48.myc/flag (Q48) were crossed to males carrying each UAS-RNAi (GD- and KK-RNAi collections, VDRC, http://stockcenter.vdrc.at/control/main). Genetic background was controlled by crossing w; GMR-GAL4; UAS-Q48.myc/flag females to w1118 males that share the same genetic background (VDRC stock number 60000 for the GD-RNAi lines and 60100 for the KK-RNAi lines). For Glutaminyl cyclase (CG32412) the GD-RNAi line 38277 and the KK-RNAi line 106341 were used. For isoGlutaminyl cyclase (CG5976), the KK-RNAi line 101533 was used. For GD-RNAi lines, degeneration was determined by scoring the eye depigmentation in the progeny of the above crosses 4 days after eclosion, assessing modification of polyglutamine loss-of-pigmentation and black necrotic-like spots. For KK-RNAi lines, as their background leads to dark eye pigmentation (http://www.vdrc.at/rnai-library/rnai-protocols), toxicity was assessed by scoring the presence or abscence of black necrotic-like spots in the eyes of 10-day old flies. Fisher’s exact test was performed to compare the numbers of necrotic-spot-containing flies in the KK-RNAi crosses with controls using an arbitrary p<0.005 as a statistical cut-off for significance. Eyes were imaged using a Nikon CoolPix 990 digital camera attached to a dissecting microscope.EGFP expression levels assessed in Drosophila RNAi linesWestern blot analysis was performed using progeny of crosses between virgins of the genotype w; GMR-GAL4; UASEGFP and males of each VDRC-RNAi line used or background control (VDRC stock number 60100). Fly heads were homogenized in Laemmli sample buffer. Rabbit polyclonal anti-GFP at 1:1000 (AbCam, Ab6556) and monoclonal anti-beta tubulin at 1:10000 (Developmental Studies Hybridoma Bank) were used. Blots were scanned using Odyssey Fc Imaging System (LI-COR Biosciences). This validation was initially performed once on each suppressor, and subsequently RNAi lines showing an apparent reduction in EGFP levels were re-tested using the progeny of three independent crosses. Statistical analysis was performed by two-tailed paired t-test between the RNAi lines and the control line.Pseudopupil assayAnalysis was performed as previously described 17. Virgins of genotype elav-GAL4C155; {GMR-HD.Q120}4.62/TM3 (elav-Gal4; GMR-HTT.Q120) 16 were crossed with males carrying the RNAi construct for Glutaminyl cyclase (lines QCGD38277 or QCKK106341) or isoGlutaminyl cyclase (line isoQCKK101533) and compared to background control line.To evaluate the effect of QPCT inhibitors, virgins of genotype yw; {GMRHD.Q120}2.4 (GMR-HTT.Q120) were allowed to mate with w1118 control males for 48 hours on standard cornmeal food and then transferred on fly food containing the compounds.The number of rhabdomeres per ommatidium was scored in progeny of the above crosses at 3 (GMR-Q120) or 4 (elav-Gal4; GMR-HTT.Q120) days post-eclosion. Statistical analysis was performed using one-tailed t-test on data from 3 or 4 independent experiments, each based on approximately 10 individuals for each genotype, scoring 15 ommatidia per eye. When compounds were tested, the analysis was done on females and males of each treatment separately.Aggregate countingVirgins of genotype w; GMR-GAL4; UAS-Httex1-Q46-eGFP
19 were crossed with males of QPCT UAS-RNAi lines or from the background KK-RNAi control line, since all the background controls show similar aggregate scoring. Eye pictures of 18-day old progeny were taken using a Leica MZ16F microscope connected to a Leica DFC340FX digital camera. For each genotype, GFP punctae indicating aggregate formation was counted using ImageJ “Cell Counter” plugin in the eyes of 20 males, a pool of 5 males from four independent crosses. For compound testing, virgins of genotype w; GMR-GAL4; UAS-Httex1-Q46-eGFP were crossed with w1118 control males, and females of the progeny scored 15 days post-eclosion. The experiment was repeated at least three times and for each experiment at least 4 female eyes were scored. An unpaired one-tailed t-test was used to determine statistical significance for single comparisons between two groups using GraphPad Prism.Compound treatmentFlies were reared on food (Instant Fly Food, Philip Harris, Ashby de la Zouch, UK) containing either QPCT inhibitor (50 μM) dissolved in DMSO or DMSO alone. The progeny were flipped every 2 days on fresh food containing the specific inhibitor or DMSO.Bioinformatics AnalysisIngenuity Pathways Analysis (Ingenuity® Systems, www.ingenuity.com) was used to analyze the distribution of siRNAs tested among the different protein classes as well as to determine the canonical pathways associated to the confirmed primary actives.Assays for validation of polyglutamine toxicity and aggregation modifiers in human cell linesCell cultureHEK293 (Human Embryonic Kidney), HeLa (Human cervical carcinoma) cells and Atg5-deficient (Atg5-/-) mouse embryonic fibrolasts (MEFs) (gift from N. Mizushima) were grown in Dulbecco’s modified eagle medium supplemented with 10% FBS, 100 U/ml penicillin/streptomycin and 2 mM l-glutamine at 37°C in 5% CO2. UbG76V-GFP-expressing stable HeLa cell line (kind gift from N.P. Dantuma) was maintained in medium containing 0.5 mg/ml G418.Isolation and culture of mouse primary cortical neuronsPrimary cortical neurons were isolated from C57BL/6 mice (Jackson Laboratories) embryos at E16.5. Briefly, brains were harvested and placed in ice-cold PBS/glucose where the meninges were removed and the cerebral cortices were dissected. After mechanical dissociation using sterile micropipette tips, dissociated neurons were resuspended in PBS/glucose and collected by centrifugation. Viable cells were seeded on poly-ornithine-coated 12-multiwell plates. Cells were cultured in Neurobasal medium supplemented with 2 mM glutamine, 200 mM B27 supplement, and 1% Penicillin-Streptomycin at 37°C in a humidified incubator with 5% CO2. One half of the culture medium was changed every two days until treatment. After 5 days of culturing in vitro, differentiated cortical neurons were infected with lentiviral particles bearing EGFP-Q80 and scramble or QPCT-directed shRNAs. Compounds were added 3 days after EGFP-Q80 viral infection and left for another 24h. When EGFP-Q80 was expressed together with shRNA, 5-6 days were needed before cultures were fixed in a 2% PFA-7.5% glucose solution.DNA constructsHuman QPCT (NM_012413) plasmid was purchased from Origene (pCMV6-XL5-QPCT). A C-terminal Flag-tagged QPCT construct was generated by PCR amplification of QPCT cDNA from pCMV6-XL5-QPCT using primers overhanging HindIII and BgIII sites and insertion into the pCMV5-FLAG in HindIII and BamHI restriction sites, using standard restriction enzyme digestion and ligation procedures. QPCT(E201Q)-Flag was generated using QuickChange II Agilent Site-Directed mutagenesis kit with the following primers Fw 5′-CTTCTTTGATGGTCAAGAGGCTTTTCTTCACTGG-3′ and Rev 5′-CCAGTGAAGAAAA GCCTCTTGACCATCAAAGAAG-3'. pcDNA or pCMV5-Flag empty vectors were used as mock controls for pCMV6-XL5-QPCT or QPCT-Flag respectively.Constructs expressing the first exon of the Htt gene carrying 74 polyglutamines expressed from pEGFP-C1 (Clontech) (EGFP-HTTQ74) or pHM6 (Roche Diagnostics) (HAHTTQ74), or with only 23 polyglutamines (EGFP-HTTQ23), were described previously 52. pEGFP-N1-Q57 and pEGFP-N1-Q81
23 and pEGFP-C1-A37
24 have been previously described. Mutant HTT(1-588)-Flag was provided by MR Hayden and mutant HTT(1-548)GFP generated by S. Luo 25. 3xFlag-CRYAB construct has been previously described 53. The pGL3-BIP/GRP78-luciferase construct was kindly provided by M. Renna 54ReagentsChemical compounds used in cell culture were the autophagy inhibitors Bafilomycin A1 (400nM, DMSO; 4 hours; Millipore) and 3MA (10 mM, 16 hours; SIGMA), staurosporine (3 μM) and the proteasome inhibitor MG132 (10 μM). PBD150 was synthesized as described in 26.TransfectionCells were transfected in 6-well plates with 0.5-1.5 μg of DNA and 5 μl of Lipofectamine (Invitrogen) or TransIT-2020 (Mirus) per well for 4 hours in Optimem (GIBCO-BRL) and then incubated in full media for 48 hours. Gene knockdown experiments were performed using ON-TARGETplus SMARTpool siRNA (Dharmacon) for human QPCT, consisting on 4 siRNAs with the following sequences: CUAUGGGUCUCGACACUUA; GUACCGGUCUUUCUCAAAU; CCUUAAAGACUGUUUCAGA; GGAACUUGCUCGUGCCUUA, and which do not target the QPCT like sequence. For siRNA treatment, a single transfection protocol using 50nM siRNA for 48 h or a double transfection protocol which consisted on a first 50 nM siRNA transfection followed by a second 50 nM siRNA transfection after 48 hours.Western blottingCells were washed once in PBS and harvested on lysis buffer (20 mM Tris-HCl pH 6.8, 137nM NaCl, 1 mM EGTA, 1% Triton X-100, 10% glycerol, 1x Roche complete mini protease inhibitor). Equal loading was obtained by protein concentration determination using a Bio-Rad assay followed by resuspension and boiling in Laemli buffer. Samples were subjected to 12% SDS-PAGE and transferred to PVDF membrane (Immobilion-P, GE Healthcare). Blots were proved with primary antibody: anti-LC3 (1:2000; Novus Biologicals, NB100-2220), anti-Hsp70 (1:1000; Enzo SPA810), anti-CRYAB (1:1000; Cell signalling 8851), anti-actin (1:2000; Sigma, A2066), anti-α-tubulin (1:4000; T9026, Sigma), anti-Flag epitope (1:2000; SIGMA, F7425), anti-GFP (1:1000; Clontech, Living colours, polyclonal), eIF2α (1:1000, Abcam 5369) and phospho-S51-eIF2α_(1:1000, Abcam 32157), GRP78 (1:1000, Abcam 21685), anti-phospho-ERK (1:1000, Cell signalling, 9101), anti-ERK (1:1000, Cell signalling, 9102), anti-phospho-CREB (S133) (1:1000, Cell signalling 9191), anti-CREB 86B10 (1:1000, Cell signalling, 9104), anti-phospho-JNK (1:1000, Cell signalling, 9255), anti-JNK (1:1000, Cell signalling, 9252). The appropriate anti-mouse or anti-rabbit secondary antibodies were used and visualized using an ECL detection kit (Amersham) or LI-COR Biosciences infrared imager (Odyssey).Caspase 3/7 activity assayCells were seeded in a 96-well plate 24h prior to the assay and 1 μM staurosporine or DMSO was added for the last 8h. Caspase 3/7 activity was measured by using a luminogenic caspase 3/7 substrate (Caspase 3/7-Glo Assay, Promega) following manufacturer protocols in a Glomax luminometer (Promega). Protein concentration was determined in each cell lysate and caspase 3/7 activity was normalized to protein levels.Co-inmunoprecipitation assaysAssays were performed as previously described 25, where HTT(1-588)Flag(Q138) and HTT(1-548)GFP(Q138) were expressed in HeLa cells together with QPCT plasmid for 48h, or treated with 25 μM SEN177 for 24h. Cells were lysed in buffer B containing 10 mM Tris pH 7.4, 150 mM NaCl, 1 mM EDTA pH8, 1% triton and 1x Roche complete mini protease inhibitor for 20 min on ice, followed by centrifugation at 13000 rpm for 10 min. Five hundred micrograms total protein were incubated with primary anti-Flag M2 (Sigma) or anti-GFP (Clontech, Living colours, polyclonal) at 5 μg/ml overnight at 4°C. Protein G Dynabeads (LifeTechnologies) were added and incubated for further 2 h. Beads were washed 3 times with buffer B and eluted using 0.1 M glycine pH 2.5M followed by boiling in laemli buffer. Samples were subjected to western blot and visualized using LICOR. A fraction of the total lysates was run simultaneously.Reverse-transcriptase PCR analysisTotal RNA was isolated from cell pellets using Trizol Reagent (Invitrogen), treated with DNase I, and cDNA synthesis was performed by SuperScript III First-Strand Synthesis System (Invitrogen). Standard conditions were used for cDNA amplification and PCR products were analyzed by agarose gel electrophoresis and ethidium bromide staining or quantitated with real-time PCR. For real-time PCR analysis, the reaction mixture containing cDNA template, primers, and SYBR Green PCR Master Mix (Invitrogen) was run in a 7900 Fast Real-time PCR System (Applied Bio-systems, Carlsbad, CA). Fold changes on mRNA levels were determined by standard curve and after normalization to internal control β-actin RNA levels. Primer sequences used in this study are: QPCT, 5′-CATGGCATGGATTTATTGG-3′ and 5′- GACGGTATCAGATCAAAC-3′; QPCT-like, 5′- CAGCGTCTCTGGAGCACTTA-3′ and 5′- GCCTCCAGGAACTTTCTGACT-3; GFP 5′- ACGTAAACGGCCACAAGTTC-3′ and 5′- TTCAGGGTCAGCTTGCCGTA-3′; actin, 5′-AGAAAATCTGGCCCACACC-3′ and 5′-GGGGTGTTGAAGGTCTCAAA-3′; CRYAB, 5′- TCTTGAGCTCAGTGAGTACTGG-3′ and 5′-AGCTCACCAGCAGTTCATGG-3′; and mouse QPCT, 5′-CGACTTGAGCCAATTGCTGA-3′ and 5′-CTTCCGGGTTAAGAGTGCTG-3′.mRNA isolation from mouse brainAll mouse experiments were performed under appropriate UK Home Office licences and following institutional procedures. We analyzed samples from N171 mutant HD mice and wild-type littermate controls at 20 week. mRNA was extracted from brains homogenized in Trizol (Invitrogen) using an Ultra torax homogenizer.Lentivirus infectionshRNA containing pLKO.1 vectors targeting both mouse and human QPCT (TRCN032432) were obtained from The RNAi Consortium (TRC) and scramble shRNA vector was generated in D. Sabatinit’s laboratory (Addgene, plasmid 1864). Lentiviral plasmids to express Q80-GFP were kindly provided by J. Uney 55. Lentiviral particles were produced and transduced following The RNAi Consortium protocols.Cell toxicity and aggregation assaysCells were fixed for 7 minutes in 4% paraformaldehyde (PFA). For EGFP-tagged constructs, slides were mounted in Citifluor (Citifluor, Ltd.) containing 4′,6-diamidino-2-phenylindole (DAPI; 3 μg/ml; Sigma) and visualized using an Eclipse E600 fluorescence microscope (plan-apo 60x/1.4 oil immersion lens) (Nikon). For detection of HA-tagged constructs, inmunofluorescence with an anti-HA (Covance laboratories 1:500) and anti-mouse Alexa488 secondary antibody (Invitrogen, 1:1000) was performed followed by mounting in Citiflour-DAPI. We assessed the percentage of transfected cells (EGFP- or HA-positive cells) with at least one aggregate per cell. Apoptotic cell death was determined by assessing the nuclear morphology (nuclei fragmented or condensed) in transfected cells. Slides were blinded and at least 200 transfected cells per slide were scored; each individual experiment was performed in triplicate.Detection of nascent protein synthesisProtein synthesis was assessed by metabolic incorporation of AHA (L-azidohomoalanine) into cells transfected with EGFP-HTT(Q23). Briefly, 12 hours after HeLa cells transfection, media was washed and replaced with L-methionine/L-cysteine free medium and treated with DMSO or SEN177 (50 uM) for 1h prior to addition of AHA (L-azidohomoalanine) to the media and collection of cells every 2 hours. Labelled protein was detected by western blot after performing Click-IT protein detection assay (Life Technologies) using biotin, following manufacturer protocols.Luciferase reporter assayCells were transfected with 1 μg of GRP78-luciferase (firefly) reporter construct and 50 ng of renilla-luciferase (pRL-TK) as an internal transfection efficiency control. Cells were collected in Passive lysis buffer and luciferase activity was measured using the Dual-luciferase Reporter Assay System (Promega) following manufacturer’s protocol in a Glomax Luminometer (Promega). GRP78-luciferase relative activity was calculated relative to the renilla-luciferase transfection efficiency control activity for each sample; experiments were performed in triplicate.Statistical analysisQuantification of immunoblots was performed by densitometric analysis using the Image J software or the LI-COR Biosciences infrared imager software and normalized to loading control (actin or tubulin, as indicated). The p-values were determined by two-tailed Student’s t-test.Aggregates were counted in at least 200 cells per slide (with the observer blinded to their identity), and percentage was calculated relative to control conditions. p-values were determined by unpaired two-tailed Student’s t-test.All experiments were done at least three times in triplicate and a representative blot or graph from a triplicate experiment is shown unless indicated.Heat shock proteins and chaperones PCR arrayThe Human Heat Shock Proteins and Chaperones RT2 Profiler PCR Array (SABiosciences, Frederick, MD) was used to study the expression profile of 84 heat shock proteins according to the manufacturer’s procedure. Briefly, total RNA was extracted from cells transfected with HTT(Q74)GFP treated with DMSO or 25 uM of SEN177 inhibitor for 24h, using Trizol (Invitrogen) and further purified using RNeasy mini kit with oncolumn DNAse digest (Qiagen), cDNA was then synthesized using an RT2 First strand kit (SABiosciences) and real-time PCR was performed using 7900HT fast real time PCR system (Applied iosciences). Data were analysed with RT2 profiler PCR array data analysis software version 3.5.Assays for validation of polyglutamine aggregation modifiers in zebrafishMaintenance of zebrafish stocks and collection of embryosAll zebrafish husbandry and experiments were performed in accordance with UK legislation under a licence granted by the Home Office and with local ethical approval. Zebrafish were reared under standard conditions (Westerfield et al, 2005) on a 14 h light/10 h dark cycle. Embryos were collected from natural spawnings, staged according to the established criteria 56 and reared in embryo medium (5 mM NaCl, 0.17 mM KCl, 0.33 mM CaCl2, 0.33 mM Mg2SO4, 5 mM HEPES).Determination of the maximum-tolerated concentration of compounds in larval zebrafishCompound exposure experiments were performed on wild-type larvae (TL strain) from 2 to 3 days post-fertilization (d.p.f.). Concentration response assays were performed over log intervals, from 100 nM to 1 mM, to determine the maximum non-toxic concentration (MTC) for subsequent aggregate analysis assays (n=10 larvae per concentration). Compound exposure experiments were performed in the dark at 28.5 °C.Measuring aggregate number and rhodopsin protein levels in transgenic HD zebrafishAggregate counting and analysis of rod photoreceptor degeneration (photoreceptor number) was performed using heterozygous larvae from Tg (rho:EGFP-HTT71Q)cu5 zebrafish 42 (hereafter referred to as transgenic HD zebrafish). Embryos from outcrossed transgenic HD zebrafish were raised in 0.2 mM 1-phenyl-2-thiourea (PTU) from 1 to 3 d.p.f. to inhibit pigment formation, screened for transgene expression using EGFP fluorescence, and then washed twice in the embryo medium to remove PTU. From 3 to 9 d.p.f., transgenic HD zebrafish larvae were dark-reared in embryo medium alone or embryo medium contain containing either DMSO, 1mM SEN177, 100 μM SEN180 or 100 μM SEN817. Embryo medium and compounds were replenished daily. Larvae were anaesthetized by immersion in 0.2 mg/ml 3-amino benzoic acid ethyl ester (MS222), then fixed for aggregate counting at 7 d.p.f. or for photoreceptor analysis at 9 d.p.f. Anaesthetised larvae were fixed using 4% paraformaldehyde (PFA) in PBS at 4 °C. Larvae were washed briefly in PBS, allowed to equilibrate in 30% sucrose in PBS then embedded in OCT medium (Tissue-Tek) and frozen on dry ice for subsequent cryosectioning. Sections were cut at 10 μm thickness using a cryostat (Bright Instruments). For aggregate counting, sections were mounted in 50% glycerol in PBS and the total number of GFP-positive aggregates were counted over 100 μm of the central retina, either side of the optic nerve head and mean values were calculated (n = 5 fish (10 eyes)) for each treatment group. For quantification of photoreceptor number, the GFP-positive area of the central retina was quantified using image thresholding and area analysis in ImageJ (n ≥ 5 fish (10 eyes) for each treatment group). To demonstrate that loss of GFP corresponds to loss of photoreceptors, sections were stained with anti-rhodopsin (1D1) antibody (a kind gift from Paul Linser, University of Florida, FL 57 and mounted using VectaShield hard set mounting medium (Vector Laboratories). Sections were viewed and representative images acquired using a GX Optical LED fluorescent microscope, GXCAM3.3 digital camera and GX Capture software.

Online Methods

Assays for validation polyglutamine toxicity modifiers in DrosophilaDrosophila fly stocksAs a model of polyglutamine toxicity, flies that expressed a protein with 48 glutamines encoded by P{UAS-Q48.myc/flag}31
13 in eyes under control of the GMR-Gal4 driver P{GAL4-ninaE.GMR}12
51 (Q48) were used. Fly orthologs to the genes identified in the cell screen were selected by performing reciprocal BLASTP and cross checking with databases including http://www.ncbi.nlm.nih.gov/homologene, http://www.genecards.org/, http://www.ensembl.org/index.html. The RNAi lines corresponding to the identified genes were obtained from Vienna Drosophila RNAi Center (VDRC, http://stockcenter.vdrc.at/control/main).The following stocks were generous gifts: UAS-Q48.myc/flag from J.L. Marsh 13, UAS-Httex1-Q46-eGFP from N. Perrimon 19. Fly lines that are not referenced here are documented in FlyBase (www.flybase.org).All fly crosses and experiments were performed at 25°C.Drosophila RNAi screenFive virgins of genotype w; GMR-GAL4; UAS-Q48.myc/flag (Q48) were crossed to males carrying each UAS-RNAi (GD- and KK-RNAi collections, VDRC, http://stockcenter.vdrc.at/control/main). Genetic background was controlled by crossing w; GMR-GAL4; UAS-Q48.myc/flag females to w1118 males that share the same genetic background (VDRC stock number 60000 for the GD-RNAi lines and 60100 for the KK-RNAi lines). For Glutaminyl cyclase (CG32412) the GD-RNAi line 38277 and the KK-RNAi line 106341 were used. For isoGlutaminyl cyclase (CG5976), the KK-RNAi line 101533 was used. For GD-RNAi lines, degeneration was determined by scoring the eye depigmentation in the progeny of the above crosses 4 days after eclosion, assessing modification of polyglutamine loss-of-pigmentation and black necrotic-like spots. For KK-RNAi lines, as their background leads to dark eye pigmentation (http://www.vdrc.at/rnai-library/rnai-protocols), toxicity was assessed by scoring the presence or abscence of black necrotic-like spots in the eyes of 10-day old flies. Fisher’s exact test was performed to compare the numbers of necrotic-spot-containing flies in the KK-RNAi crosses with controls using an arbitrary p<0.005 as a statistical cut-off for significance. Eyes were imaged using a Nikon CoolPix 990 digital camera attached to a dissecting microscope.EGFP expression levels assessed in Drosophila RNAi linesWestern blot analysis was performed using progeny of crosses between virgins of the genotype w; GMR-GAL4; UASEGFP and males of each VDRC-RNAi line used or background control (VDRC stock number 60100). Fly heads were homogenized in Laemmli sample buffer. Rabbit polyclonal anti-GFP at 1:1000 (AbCam, Ab6556) and monoclonal anti-beta tubulin at 1:10000 (Developmental Studies Hybridoma Bank) were used. Blots were scanned using Odyssey Fc Imaging System (LI-COR Biosciences). This validation was initially performed once on each suppressor, and subsequently RNAi lines showing an apparent reduction in EGFP levels were re-tested using the progeny of three independent crosses. Statistical analysis was performed by two-tailed paired t-test between the RNAi lines and the control line.Pseudopupil assayAnalysis was performed as previously described 17. Virgins of genotype elav-GAL4C155; {GMR-HD.Q120}4.62/TM3 (elav-Gal4; GMR-HTT.Q120) 16 were crossed with males carrying the RNAi construct for Glutaminyl cyclase (lines QCGD38277 or QCKK106341) or isoGlutaminyl cyclase (line isoQCKK101533) and compared to background control line.To evaluate the effect of QPCT inhibitors, virgins of genotype yw; {GMRHD.Q120}2.4 (GMR-HTT.Q120) were allowed to mate with w1118 control males for 48 hours on standard cornmeal food and then transferred on fly food containing the compounds.The number of rhabdomeres per ommatidium was scored in progeny of the above crosses at 3 (GMR-Q120) or 4 (elav-Gal4; GMR-HTT.Q120) days post-eclosion. Statistical analysis was performed using one-tailed t-test on data from 3 or 4 independent experiments, each based on approximately 10 individuals for each genotype, scoring 15 ommatidia per eye. When compounds were tested, the analysis was done on females and males of each treatment separately.Aggregate countingVirgins of genotype w; GMR-GAL4; UAS-Httex1-Q46-eGFP
19 were crossed with males of QPCT UAS-RNAi lines or from the background KK-RNAi control line, since all the background controls show similar aggregate scoring. Eye pictures of 18-day old progeny were taken using a Leica MZ16F microscope connected to a Leica DFC340FX digital camera. For each genotype, GFP punctae indicating aggregate formation was counted using ImageJ “Cell Counter” plugin in the eyes of 20 males, a pool of 5 males from four independent crosses. For compound testing, virgins of genotype w; GMR-GAL4; UAS-Httex1-Q46-eGFP were crossed with w1118 control males, and females of the progeny scored 15 days post-eclosion. The experiment was repeated at least three times and for each experiment at least 4 female eyes were scored. An unpaired one-tailed t-test was used to determine statistical significance for single comparisons between two groups using GraphPad Prism.Compound treatmentFlies were reared on food (Instant Fly Food, Philip Harris, Ashby de la Zouch, UK) containing either QPCT inhibitor (50 μM) dissolved in DMSO or DMSO alone. The progeny were flipped every 2 days on fresh food containing the specific inhibitor or DMSO.

Assays for validation polyglutamine toxicity modifiers in Drosophila

Drosophila fly stocksAs a model of polyglutamine toxicity, flies that expressed a protein with 48 glutamines encoded by P{UAS-Q48.myc/flag}31
13 in eyes under control of the GMR-Gal4 driver P{GAL4-ninaE.GMR}12
51 (Q48) were used. Fly orthologs to the genes identified in the cell screen were selected by performing reciprocal BLASTP and cross checking with databases including http://www.ncbi.nlm.nih.gov/homologene, http://www.genecards.org/, http://www.ensembl.org/index.html. The RNAi lines corresponding to the identified genes were obtained from Vienna Drosophila RNAi Center (VDRC, http://stockcenter.vdrc.at/control/main).The following stocks were generous gifts: UAS-Q48.myc/flag from J.L. Marsh 13, UAS-Httex1-Q46-eGFP from N. Perrimon 19. Fly lines that are not referenced here are documented in FlyBase (www.flybase.org).All fly crosses and experiments were performed at 25°C.

Drosophila fly stocks

As a model of polyglutamine toxicity, flies that expressed a protein with 48 glutamines encoded by P{UAS-Q48.myc/flag}31
13 in eyes under control of the GMR-Gal4 driver P{GAL4-ninaE.GMR}12
51 (Q48) were used. Fly orthologs to the genes identified in the cell screen were selected by performing reciprocal BLASTP and cross checking with databases including http://www.ncbi.nlm.nih.gov/homologene, http://www.genecards.org/, http://www.ensembl.org/index.html. The RNAi lines corresponding to the identified genes were obtained from Vienna Drosophila RNAi Center (VDRC, http://stockcenter.vdrc.at/control/main).

The following stocks were generous gifts: UAS-Q48.myc/flag from J.L. Marsh 13, UAS-Httex1-Q46-eGFP from N. Perrimon 19. Fly lines that are not referenced here are documented in FlyBase (www.flybase.org).

All fly crosses and experiments were performed at 25°C.

Drosophila RNAi screenFive virgins of genotype w; GMR-GAL4; UAS-Q48.myc/flag (Q48) were crossed to males carrying each UAS-RNAi (GD- and KK-RNAi collections, VDRC, http://stockcenter.vdrc.at/control/main). Genetic background was controlled by crossing w; GMR-GAL4; UAS-Q48.myc/flag females to w1118 males that share the same genetic background (VDRC stock number 60000 for the GD-RNAi lines and 60100 for the KK-RNAi lines). For Glutaminyl cyclase (CG32412) the GD-RNAi line 38277 and the KK-RNAi line 106341 were used. For isoGlutaminyl cyclase (CG5976), the KK-RNAi line 101533 was used. For GD-RNAi lines, degeneration was determined by scoring the eye depigmentation in the progeny of the above crosses 4 days after eclosion, assessing modification of polyglutamine loss-of-pigmentation and black necrotic-like spots. For KK-RNAi lines, as their background leads to dark eye pigmentation (http://www.vdrc.at/rnai-library/rnai-protocols), toxicity was assessed by scoring the presence or abscence of black necrotic-like spots in the eyes of 10-day old flies. Fisher’s exact test was performed to compare the numbers of necrotic-spot-containing flies in the KK-RNAi crosses with controls using an arbitrary p<0.005 as a statistical cut-off for significance. Eyes were imaged using a Nikon CoolPix 990 digital camera attached to a dissecting microscope.

Drosophila RNAi screen

Five virgins of genotype w; GMR-GAL4; UAS-Q48.myc/flag (Q48) were crossed to males carrying each UAS-RNAi (GD- and KK-RNAi collections, VDRC, http://stockcenter.vdrc.at/control/main). Genetic background was controlled by crossing w; GMR-GAL4; UAS-Q48.myc/flag females to w1118 males that share the same genetic background (VDRC stock number 60000 for the GD-RNAi lines and 60100 for the KK-RNAi lines). For Glutaminyl cyclase (CG32412) the GD-RNAi line 38277 and the KK-RNAi line 106341 were used. For isoGlutaminyl cyclase (CG5976), the KK-RNAi line 101533 was used. For GD-RNAi lines, degeneration was determined by scoring the eye depigmentation in the progeny of the above crosses 4 days after eclosion, assessing modification of polyglutamine loss-of-pigmentation and black necrotic-like spots. For KK-RNAi lines, as their background leads to dark eye pigmentation (http://www.vdrc.at/rnai-library/rnai-protocols), toxicity was assessed by scoring the presence or abscence of black necrotic-like spots in the eyes of 10-day old flies. Fisher’s exact test was performed to compare the numbers of necrotic-spot-containing flies in the KK-RNAi crosses with controls using an arbitrary p<0.005 as a statistical cut-off for significance. Eyes were imaged using a Nikon CoolPix 990 digital camera attached to a dissecting microscope.

EGFP expression levels assessed in Drosophila RNAi linesWestern blot analysis was performed using progeny of crosses between virgins of the genotype w; GMR-GAL4; UASEGFP and males of each VDRC-RNAi line used or background control (VDRC stock number 60100). Fly heads were homogenized in Laemmli sample buffer. Rabbit polyclonal anti-GFP at 1:1000 (AbCam, Ab6556) and monoclonal anti-beta tubulin at 1:10000 (Developmental Studies Hybridoma Bank) were used. Blots were scanned using Odyssey Fc Imaging System (LI-COR Biosciences). This validation was initially performed once on each suppressor, and subsequently RNAi lines showing an apparent reduction in EGFP levels were re-tested using the progeny of three independent crosses. Statistical analysis was performed by two-tailed paired t-test between the RNAi lines and the control line.

EGFP expression levels assessed in Drosophila RNAi lines

Western blot analysis was performed using progeny of crosses between virgins of the genotype w; GMR-GAL4; UASEGFP and males of each VDRC-RNAi line used or background control (VDRC stock number 60100). Fly heads were homogenized in Laemmli sample buffer. Rabbit polyclonal anti-GFP at 1:1000 (AbCam, Ab6556) and monoclonal anti-beta tubulin at 1:10000 (Developmental Studies Hybridoma Bank) were used. Blots were scanned using Odyssey Fc Imaging System (LI-COR Biosciences). This validation was initially performed once on each suppressor, and subsequently RNAi lines showing an apparent reduction in EGFP levels were re-tested using the progeny of three independent crosses. Statistical analysis was performed by two-tailed paired t-test between the RNAi lines and the control line.

Pseudopupil assayAnalysis was performed as previously described 17. Virgins of genotype elav-GAL4C155; {GMR-HD.Q120}4.62/TM3 (elav-Gal4; GMR-HTT.Q120) 16 were crossed with males carrying the RNAi construct for Glutaminyl cyclase (lines QCGD38277 or QCKK106341) or isoGlutaminyl cyclase (line isoQCKK101533) and compared to background control line.To evaluate the effect of QPCT inhibitors, virgins of genotype yw; {GMRHD.Q120}2.4 (GMR-HTT.Q120) were allowed to mate with w1118 control males for 48 hours on standard cornmeal food and then transferred on fly food containing the compounds.The number of rhabdomeres per ommatidium was scored in progeny of the above crosses at 3 (GMR-Q120) or 4 (elav-Gal4; GMR-HTT.Q120) days post-eclosion. Statistical analysis was performed using one-tailed t-test on data from 3 or 4 independent experiments, each based on approximately 10 individuals for each genotype, scoring 15 ommatidia per eye. When compounds were tested, the analysis was done on females and males of each treatment separately.

Pseudopupil assay

Analysis was performed as previously described 17. Virgins of genotype elav-GAL4C155; {GMR-HD.Q120}4.62/TM3 (elav-Gal4; GMR-HTT.Q120) 16 were crossed with males carrying the RNAi construct for Glutaminyl cyclase (lines QCGD38277 or QCKK106341) or isoGlutaminyl cyclase (line isoQCKK101533) and compared to background control line.

To evaluate the effect of QPCT inhibitors, virgins of genotype yw; {GMRHD.Q120}2.4 (GMR-HTT.Q120) were allowed to mate with w1118 control males for 48 hours on standard cornmeal food and then transferred on fly food containing the compounds.

The number of rhabdomeres per ommatidium was scored in progeny of the above crosses at 3 (GMR-Q120) or 4 (elav-Gal4; GMR-HTT.Q120) days post-eclosion. Statistical analysis was performed using one-tailed t-test on data from 3 or 4 independent experiments, each based on approximately 10 individuals for each genotype, scoring 15 ommatidia per eye. When compounds were tested, the analysis was done on females and males of each treatment separately.

Aggregate countingVirgins of genotype w; GMR-GAL4; UAS-Httex1-Q46-eGFP
19 were crossed with males of QPCT UAS-RNAi lines or from the background KK-RNAi control line, since all the background controls show similar aggregate scoring. Eye pictures of 18-day old progeny were taken using a Leica MZ16F microscope connected to a Leica DFC340FX digital camera. For each genotype, GFP punctae indicating aggregate formation was counted using ImageJ “Cell Counter” plugin in the eyes of 20 males, a pool of 5 males from four independent crosses. For compound testing, virgins of genotype w; GMR-GAL4; UAS-Httex1-Q46-eGFP were crossed with w1118 control males, and females of the progeny scored 15 days post-eclosion. The experiment was repeated at least three times and for each experiment at least 4 female eyes were scored. An unpaired one-tailed t-test was used to determine statistical significance for single comparisons between two groups using GraphPad Prism.

Aggregate counting

Virgins of genotype w; GMR-GAL4; UAS-Httex1-Q46-eGFP
19 were crossed with males of QPCT UAS-RNAi lines or from the background KK-RNAi control line, since all the background controls show similar aggregate scoring. Eye pictures of 18-day old progeny were taken using a Leica MZ16F microscope connected to a Leica DFC340FX digital camera. For each genotype, GFP punctae indicating aggregate formation was counted using ImageJ “Cell Counter” plugin in the eyes of 20 males, a pool of 5 males from four independent crosses. For compound testing, virgins of genotype w; GMR-GAL4; UAS-Httex1-Q46-eGFP were crossed with w1118 control males, and females of the progeny scored 15 days post-eclosion. The experiment was repeated at least three times and for each experiment at least 4 female eyes were scored. An unpaired one-tailed t-test was used to determine statistical significance for single comparisons between two groups using GraphPad Prism.

Compound treatmentFlies were reared on food (Instant Fly Food, Philip Harris, Ashby de la Zouch, UK) containing either QPCT inhibitor (50 μM) dissolved in DMSO or DMSO alone. The progeny were flipped every 2 days on fresh food containing the specific inhibitor or DMSO.

Compound treatment

Flies were reared on food (Instant Fly Food, Philip Harris, Ashby de la Zouch, UK) containing either QPCT inhibitor (50 μM) dissolved in DMSO or DMSO alone. The progeny were flipped every 2 days on fresh food containing the specific inhibitor or DMSO.

Bioinformatics AnalysisIngenuity Pathways Analysis (Ingenuity® Systems, www.ingenuity.com) was used to analyze the distribution of siRNAs tested among the different protein classes as well as to determine the canonical pathways associated to the confirmed primary actives.

Bioinformatics Analysis

Ingenuity Pathways Analysis (Ingenuity® Systems, www.ingenuity.com) was used to analyze the distribution of siRNAs tested among the different protein classes as well as to determine the canonical pathways associated to the confirmed primary actives.

Assays for validation of polyglutamine toxicity and aggregation modifiers in human cell linesCell cultureHEK293 (Human Embryonic Kidney), HeLa (Human cervical carcinoma) cells and Atg5-deficient (Atg5-/-) mouse embryonic fibrolasts (MEFs) (gift from N. Mizushima) were grown in Dulbecco’s modified eagle medium supplemented with 10% FBS, 100 U/ml penicillin/streptomycin and 2 mM l-glutamine at 37°C in 5% CO2. UbG76V-GFP-expressing stable HeLa cell line (kind gift from N.P. Dantuma) was maintained in medium containing 0.5 mg/ml G418.Isolation and culture of mouse primary cortical neuronsPrimary cortical neurons were isolated from C57BL/6 mice (Jackson Laboratories) embryos at E16.5. Briefly, brains were harvested and placed in ice-cold PBS/glucose where the meninges were removed and the cerebral cortices were dissected. After mechanical dissociation using sterile micropipette tips, dissociated neurons were resuspended in PBS/glucose and collected by centrifugation. Viable cells were seeded on poly-ornithine-coated 12-multiwell plates. Cells were cultured in Neurobasal medium supplemented with 2 mM glutamine, 200 mM B27 supplement, and 1% Penicillin-Streptomycin at 37°C in a humidified incubator with 5% CO2. One half of the culture medium was changed every two days until treatment. After 5 days of culturing in vitro, differentiated cortical neurons were infected with lentiviral particles bearing EGFP-Q80 and scramble or QPCT-directed shRNAs. Compounds were added 3 days after EGFP-Q80 viral infection and left for another 24h. When EGFP-Q80 was expressed together with shRNA, 5-6 days were needed before cultures were fixed in a 2% PFA-7.5% glucose solution.DNA constructsHuman QPCT (NM_012413) plasmid was purchased from Origene (pCMV6-XL5-QPCT). A C-terminal Flag-tagged QPCT construct was generated by PCR amplification of QPCT cDNA from pCMV6-XL5-QPCT using primers overhanging HindIII and BgIII sites and insertion into the pCMV5-FLAG in HindIII and BamHI restriction sites, using standard restriction enzyme digestion and ligation procedures. QPCT(E201Q)-Flag was generated using QuickChange II Agilent Site-Directed mutagenesis kit with the following primers Fw 5′-CTTCTTTGATGGTCAAGAGGCTTTTCTTCACTGG-3′ and Rev 5′-CCAGTGAAGAAAA GCCTCTTGACCATCAAAGAAG-3'. pcDNA or pCMV5-Flag empty vectors were used as mock controls for pCMV6-XL5-QPCT or QPCT-Flag respectively.Constructs expressing the first exon of the Htt gene carrying 74 polyglutamines expressed from pEGFP-C1 (Clontech) (EGFP-HTTQ74) or pHM6 (Roche Diagnostics) (HAHTTQ74), or with only 23 polyglutamines (EGFP-HTTQ23), were described previously 52. pEGFP-N1-Q57 and pEGFP-N1-Q81
23 and pEGFP-C1-A37
24 have been previously described. Mutant HTT(1-588)-Flag was provided by MR Hayden and mutant HTT(1-548)GFP generated by S. Luo 25. 3xFlag-CRYAB construct has been previously described 53. The pGL3-BIP/GRP78-luciferase construct was kindly provided by M. Renna 54ReagentsChemical compounds used in cell culture were the autophagy inhibitors Bafilomycin A1 (400nM, DMSO; 4 hours; Millipore) and 3MA (10 mM, 16 hours; SIGMA), staurosporine (3 μM) and the proteasome inhibitor MG132 (10 μM). PBD150 was synthesized as described in 26.TransfectionCells were transfected in 6-well plates with 0.5-1.5 μg of DNA and 5 μl of Lipofectamine (Invitrogen) or TransIT-2020 (Mirus) per well for 4 hours in Optimem (GIBCO-BRL) and then incubated in full media for 48 hours. Gene knockdown experiments were performed using ON-TARGETplus SMARTpool siRNA (Dharmacon) for human QPCT, consisting on 4 siRNAs with the following sequences: CUAUGGGUCUCGACACUUA; GUACCGGUCUUUCUCAAAU; CCUUAAAGACUGUUUCAGA; GGAACUUGCUCGUGCCUUA, and which do not target the QPCT like sequence. For siRNA treatment, a single transfection protocol using 50nM siRNA for 48 h or a double transfection protocol which consisted on a first 50 nM siRNA transfection followed by a second 50 nM siRNA transfection after 48 hours.Western blottingCells were washed once in PBS and harvested on lysis buffer (20 mM Tris-HCl pH 6.8, 137nM NaCl, 1 mM EGTA, 1% Triton X-100, 10% glycerol, 1x Roche complete mini protease inhibitor). Equal loading was obtained by protein concentration determination using a Bio-Rad assay followed by resuspension and boiling in Laemli buffer. Samples were subjected to 12% SDS-PAGE and transferred to PVDF membrane (Immobilion-P, GE Healthcare). Blots were proved with primary antibody: anti-LC3 (1:2000; Novus Biologicals, NB100-2220), anti-Hsp70 (1:1000; Enzo SPA810), anti-CRYAB (1:1000; Cell signalling 8851), anti-actin (1:2000; Sigma, A2066), anti-α-tubulin (1:4000; T9026, Sigma), anti-Flag epitope (1:2000; SIGMA, F7425), anti-GFP (1:1000; Clontech, Living colours, polyclonal), eIF2α (1:1000, Abcam 5369) and phospho-S51-eIF2α_(1:1000, Abcam 32157), GRP78 (1:1000, Abcam 21685), anti-phospho-ERK (1:1000, Cell signalling, 9101), anti-ERK (1:1000, Cell signalling, 9102), anti-phospho-CREB (S133) (1:1000, Cell signalling 9191), anti-CREB 86B10 (1:1000, Cell signalling, 9104), anti-phospho-JNK (1:1000, Cell signalling, 9255), anti-JNK (1:1000, Cell signalling, 9252). The appropriate anti-mouse or anti-rabbit secondary antibodies were used and visualized using an ECL detection kit (Amersham) or LI-COR Biosciences infrared imager (Odyssey).Caspase 3/7 activity assayCells were seeded in a 96-well plate 24h prior to the assay and 1 μM staurosporine or DMSO was added for the last 8h. Caspase 3/7 activity was measured by using a luminogenic caspase 3/7 substrate (Caspase 3/7-Glo Assay, Promega) following manufacturer protocols in a Glomax luminometer (Promega). Protein concentration was determined in each cell lysate and caspase 3/7 activity was normalized to protein levels.Co-inmunoprecipitation assaysAssays were performed as previously described 25, where HTT(1-588)Flag(Q138) and HTT(1-548)GFP(Q138) were expressed in HeLa cells together with QPCT plasmid for 48h, or treated with 25 μM SEN177 for 24h. Cells were lysed in buffer B containing 10 mM Tris pH 7.4, 150 mM NaCl, 1 mM EDTA pH8, 1% triton and 1x Roche complete mini protease inhibitor for 20 min on ice, followed by centrifugation at 13000 rpm for 10 min. Five hundred micrograms total protein were incubated with primary anti-Flag M2 (Sigma) or anti-GFP (Clontech, Living colours, polyclonal) at 5 μg/ml overnight at 4°C. Protein G Dynabeads (LifeTechnologies) were added and incubated for further 2 h. Beads were washed 3 times with buffer B and eluted using 0.1 M glycine pH 2.5M followed by boiling in laemli buffer. Samples were subjected to western blot and visualized using LICOR. A fraction of the total lysates was run simultaneously.Reverse-transcriptase PCR analysisTotal RNA was isolated from cell pellets using Trizol Reagent (Invitrogen), treated with DNase I, and cDNA synthesis was performed by SuperScript III First-Strand Synthesis System (Invitrogen). Standard conditions were used for cDNA amplification and PCR products were analyzed by agarose gel electrophoresis and ethidium bromide staining or quantitated with real-time PCR. For real-time PCR analysis, the reaction mixture containing cDNA template, primers, and SYBR Green PCR Master Mix (Invitrogen) was run in a 7900 Fast Real-time PCR System (Applied Bio-systems, Carlsbad, CA). Fold changes on mRNA levels were determined by standard curve and after normalization to internal control β-actin RNA levels. Primer sequences used in this study are: QPCT, 5′-CATGGCATGGATTTATTGG-3′ and 5′- GACGGTATCAGATCAAAC-3′; QPCT-like, 5′- CAGCGTCTCTGGAGCACTTA-3′ and 5′- GCCTCCAGGAACTTTCTGACT-3; GFP 5′- ACGTAAACGGCCACAAGTTC-3′ and 5′- TTCAGGGTCAGCTTGCCGTA-3′; actin, 5′-AGAAAATCTGGCCCACACC-3′ and 5′-GGGGTGTTGAAGGTCTCAAA-3′; CRYAB, 5′- TCTTGAGCTCAGTGAGTACTGG-3′ and 5′-AGCTCACCAGCAGTTCATGG-3′; and mouse QPCT, 5′-CGACTTGAGCCAATTGCTGA-3′ and 5′-CTTCCGGGTTAAGAGTGCTG-3′.mRNA isolation from mouse brainAll mouse experiments were performed under appropriate UK Home Office licences and following institutional procedures. We analyzed samples from N171 mutant HD mice and wild-type littermate controls at 20 week. mRNA was extracted from brains homogenized in Trizol (Invitrogen) using an Ultra torax homogenizer.Lentivirus infectionshRNA containing pLKO.1 vectors targeting both mouse and human QPCT (TRCN032432) were obtained from The RNAi Consortium (TRC) and scramble shRNA vector was generated in D. Sabatinit’s laboratory (Addgene, plasmid 1864). Lentiviral plasmids to express Q80-GFP were kindly provided by J. Uney 55. Lentiviral particles were produced and transduced following The RNAi Consortium protocols.Cell toxicity and aggregation assaysCells were fixed for 7 minutes in 4% paraformaldehyde (PFA). For EGFP-tagged constructs, slides were mounted in Citifluor (Citifluor, Ltd.) containing 4′,6-diamidino-2-phenylindole (DAPI; 3 μg/ml; Sigma) and visualized using an Eclipse E600 fluorescence microscope (plan-apo 60x/1.4 oil immersion lens) (Nikon). For detection of HA-tagged constructs, inmunofluorescence with an anti-HA (Covance laboratories 1:500) and anti-mouse Alexa488 secondary antibody (Invitrogen, 1:1000) was performed followed by mounting in Citiflour-DAPI. We assessed the percentage of transfected cells (EGFP- or HA-positive cells) with at least one aggregate per cell. Apoptotic cell death was determined by assessing the nuclear morphology (nuclei fragmented or condensed) in transfected cells. Slides were blinded and at least 200 transfected cells per slide were scored; each individual experiment was performed in triplicate.Detection of nascent protein synthesisProtein synthesis was assessed by metabolic incorporation of AHA (L-azidohomoalanine) into cells transfected with EGFP-HTT(Q23). Briefly, 12 hours after HeLa cells transfection, media was washed and replaced with L-methionine/L-cysteine free medium and treated with DMSO or SEN177 (50 uM) for 1h prior to addition of AHA (L-azidohomoalanine) to the media and collection of cells every 2 hours. Labelled protein was detected by western blot after performing Click-IT protein detection assay (Life Technologies) using biotin, following manufacturer protocols.Luciferase reporter assayCells were transfected with 1 μg of GRP78-luciferase (firefly) reporter construct and 50 ng of renilla-luciferase (pRL-TK) as an internal transfection efficiency control. Cells were collected in Passive lysis buffer and luciferase activity was measured using the Dual-luciferase Reporter Assay System (Promega) following manufacturer’s protocol in a Glomax Luminometer (Promega). GRP78-luciferase relative activity was calculated relative to the renilla-luciferase transfection efficiency control activity for each sample; experiments were performed in triplicate.Statistical analysisQuantification of immunoblots was performed by densitometric analysis using the Image J software or the LI-COR Biosciences infrared imager software and normalized to loading control (actin or tubulin, as indicated). The p-values were determined by two-tailed Student’s t-test.Aggregates were counted in at least 200 cells per slide (with the observer blinded to their identity), and percentage was calculated relative to control conditions. p-values were determined by unpaired two-tailed Student’s t-test.All experiments were done at least three times in triplicate and a representative blot or graph from a triplicate experiment is shown unless indicated.Heat shock proteins and chaperones PCR arrayThe Human Heat Shock Proteins and Chaperones RT2 Profiler PCR Array (SABiosciences, Frederick, MD) was used to study the expression profile of 84 heat shock proteins according to the manufacturer’s procedure. Briefly, total RNA was extracted from cells transfected with HTT(Q74)GFP treated with DMSO or 25 uM of SEN177 inhibitor for 24h, using Trizol (Invitrogen) and further purified using RNeasy mini kit with oncolumn DNAse digest (Qiagen), cDNA was then synthesized using an RT2 First strand kit (SABiosciences) and real-time PCR was performed using 7900HT fast real time PCR system (Applied iosciences). Data were analysed with RT2 profiler PCR array data analysis software version 3.5.

Assays for validation of polyglutamine toxicity and aggregation modifiers in human cell lines

Cell cultureHEK293 (Human Embryonic Kidney), HeLa (Human cervical carcinoma) cells and Atg5-deficient (Atg5-/-) mouse embryonic fibrolasts (MEFs) (gift from N. Mizushima) were grown in Dulbecco’s modified eagle medium supplemented with 10% FBS, 100 U/ml penicillin/streptomycin and 2 mM l-glutamine at 37°C in 5% CO2. UbG76V-GFP-expressing stable HeLa cell line (kind gift from N.P. Dantuma) was maintained in medium containing 0.5 mg/ml G418.

Cell culture

HEK293 (Human Embryonic Kidney), HeLa (Human cervical carcinoma) cells and Atg5-deficient (Atg5-/-) mouse embryonic fibrolasts (MEFs) (gift from N. Mizushima) were grown in Dulbecco’s modified eagle medium supplemented with 10% FBS, 100 U/ml penicillin/streptomycin and 2 mM l-glutamine at 37°C in 5% CO2. UbG76V-GFP-expressing stable HeLa cell line (kind gift from N.P. Dantuma) was maintained in medium containing 0.5 mg/ml G418.

Isolation and culture of mouse primary cortical neuronsPrimary cortical neurons were isolated from C57BL/6 mice (Jackson Laboratories) embryos at E16.5. Briefly, brains were harvested and placed in ice-cold PBS/glucose where the meninges were removed and the cerebral cortices were dissected. After mechanical dissociation using sterile micropipette tips, dissociated neurons were resuspended in PBS/glucose and collected by centrifugation. Viable cells were seeded on poly-ornithine-coated 12-multiwell plates. Cells were cultured in Neurobasal medium supplemented with 2 mM glutamine, 200 mM B27 supplement, and 1% Penicillin-Streptomycin at 37°C in a humidified incubator with 5% CO2. One half of the culture medium was changed every two days until treatment. After 5 days of culturing in vitro, differentiated cortical neurons were infected with lentiviral particles bearing EGFP-Q80 and scramble or QPCT-directed shRNAs. Compounds were added 3 days after EGFP-Q80 viral infection and left for another 24h. When EGFP-Q80 was expressed together with shRNA, 5-6 days were needed before cultures were fixed in a 2% PFA-7.5% glucose solution.

Isolation and culture of mouse primary cortical neurons

Primary cortical neurons were isolated from C57BL/6 mice (Jackson Laboratories) embryos at E16.5. Briefly, brains were harvested and placed in ice-cold PBS/glucose where the meninges were removed and the cerebral cortices were dissected. After mechanical dissociation using sterile micropipette tips, dissociated neurons were resuspended in PBS/glucose and collected by centrifugation. Viable cells were seeded on poly-ornithine-coated 12-multiwell plates. Cells were cultured in Neurobasal medium supplemented with 2 mM glutamine, 200 mM B27 supplement, and 1% Penicillin-Streptomycin at 37°C in a humidified incubator with 5% CO2. One half of the culture medium was changed every two days until treatment. After 5 days of culturing in vitro, differentiated cortical neurons were infected with lentiviral particles bearing EGFP-Q80 and scramble or QPCT-directed shRNAs. Compounds were added 3 days after EGFP-Q80 viral infection and left for another 24h. When EGFP-Q80 was expressed together with shRNA, 5-6 days were needed before cultures were fixed in a 2% PFA-7.5% glucose solution.

DNA constructsHuman QPCT (NM_012413) plasmid was purchased from Origene (pCMV6-XL5-QPCT). A C-terminal Flag-tagged QPCT construct was generated by PCR amplification of QPCT cDNA from pCMV6-XL5-QPCT using primers overhanging HindIII and BgIII sites and insertion into the pCMV5-FLAG in HindIII and BamHI restriction sites, using standard restriction enzyme digestion and ligation procedures. QPCT(E201Q)-Flag was generated using QuickChange II Agilent Site-Directed mutagenesis kit with the following primers Fw 5′-CTTCTTTGATGGTCAAGAGGCTTTTCTTCACTGG-3′ and Rev 5′-CCAGTGAAGAAAA GCCTCTTGACCATCAAAGAAG-3'. pcDNA or pCMV5-Flag empty vectors were used as mock controls for pCMV6-XL5-QPCT or QPCT-Flag respectively.Constructs expressing the first exon of the Htt gene carrying 74 polyglutamines expressed from pEGFP-C1 (Clontech) (EGFP-HTTQ74) or pHM6 (Roche Diagnostics) (HAHTTQ74), or with only 23 polyglutamines (EGFP-HTTQ23), were described previously 52. pEGFP-N1-Q57 and pEGFP-N1-Q81
23 and pEGFP-C1-A37
24 have been previously described. Mutant HTT(1-588)-Flag was provided by MR Hayden and mutant HTT(1-548)GFP generated by S. Luo 25. 3xFlag-CRYAB construct has been previously described 53. The pGL3-BIP/GRP78-luciferase construct was kindly provided by M. Renna 54

DNA constructs

Human QPCT (NM_012413) plasmid was purchased from Origene (pCMV6-XL5-QPCT). A C-terminal Flag-tagged QPCT construct was generated by PCR amplification of QPCT cDNA from pCMV6-XL5-QPCT using primers overhanging HindIII and BgIII sites and insertion into the pCMV5-FLAG in HindIII and BamHI restriction sites, using standard restriction enzyme digestion and ligation procedures. QPCT(E201Q)-Flag was generated using QuickChange II Agilent Site-Directed mutagenesis kit with the following primers Fw 5′-CTTCTTTGATGGTCAAGAGGCTTTTCTTCACTGG-3′ and Rev 5′-CCAGTGAAGAAAA GCCTCTTGACCATCAAAGAAG-3'. pcDNA or pCMV5-Flag empty vectors were used as mock controls for pCMV6-XL5-QPCT or QPCT-Flag respectively.

Constructs expressing the first exon of the Htt gene carrying 74 polyglutamines expressed from pEGFP-C1 (Clontech) (EGFP-HTTQ74) or pHM6 (Roche Diagnostics) (HAHTTQ74), or with only 23 polyglutamines (EGFP-HTTQ23), were described previously 52. pEGFP-N1-Q57 and pEGFP-N1-Q81
23 and pEGFP-C1-A37
24 have been previously described. Mutant HTT(1-588)-Flag was provided by MR Hayden and mutant HTT(1-548)GFP generated by S. Luo 25. 3xFlag-CRYAB construct has been previously described 53. The pGL3-BIP/GRP78-luciferase construct was kindly provided by M. Renna 54

ReagentsChemical compounds used in cell culture were the autophagy inhibitors Bafilomycin A1 (400nM, DMSO; 4 hours; Millipore) and 3MA (10 mM, 16 hours; SIGMA), staurosporine (3 μM) and the proteasome inhibitor MG132 (10 μM). PBD150 was synthesized as described in 26.

Chemical compounds used in cell culture were the autophagy inhibitors Bafilomycin A1 (400nM, DMSO; 4 hours; Millipore) and 3MA (10 mM, 16 hours; SIGMA), staurosporine (3 μM) and the proteasome inhibitor MG132 (10 μM). PBD150 was synthesized as described in 26.

TransfectionCells were transfected in 6-well plates with 0.5-1.5 μg of DNA and 5 μl of Lipofectamine (Invitrogen) or TransIT-2020 (Mirus) per well for 4 hours in Optimem (GIBCO-BRL) and then incubated in full media for 48 hours. Gene knockdown experiments were performed using ON-TARGETplus SMARTpool siRNA (Dharmacon) for human QPCT, consisting on 4 siRNAs with the following sequences: CUAUGGGUCUCGACACUUA; GUACCGGUCUUUCUCAAAU; CCUUAAAGACUGUUUCAGA; GGAACUUGCUCGUGCCUUA, and which do not target the QPCT like sequence. For siRNA treatment, a single transfection protocol using 50nM siRNA for 48 h or a double transfection protocol which consisted on a first 50 nM siRNA transfection followed by a second 50 nM siRNA transfection after 48 hours.

Transfection

Cells were transfected in 6-well plates with 0.5-1.5 μg of DNA and 5 μl of Lipofectamine (Invitrogen) or TransIT-2020 (Mirus) per well for 4 hours in Optimem (GIBCO-BRL) and then incubated in full media for 48 hours. Gene knockdown experiments were performed using ON-TARGETplus SMARTpool siRNA (Dharmacon) for human QPCT, consisting on 4 siRNAs with the following sequences: CUAUGGGUCUCGACACUUA; GUACCGGUCUUUCUCAAAU; CCUUAAAGACUGUUUCAGA; GGAACUUGCUCGUGCCUUA, and which do not target the QPCT like sequence. For siRNA treatment, a single transfection protocol using 50nM siRNA for 48 h or a double transfection protocol which consisted on a first 50 nM siRNA transfection followed by a second 50 nM siRNA transfection after 48 hours.

Western blottingCells were washed once in PBS and harvested on lysis buffer (20 mM Tris-HCl pH 6.8, 137nM NaCl, 1 mM EGTA, 1% Triton X-100, 10% glycerol, 1x Roche complete mini protease inhibitor). Equal loading was obtained by protein concentration determination using a Bio-Rad assay followed by resuspension and boiling in Laemli buffer. Samples were subjected to 12% SDS-PAGE and transferred to PVDF membrane (Immobilion-P, GE Healthcare). Blots were proved with primary antibody: anti-LC3 (1:2000; Novus Biologicals, NB100-2220), anti-Hsp70 (1:1000; Enzo SPA810), anti-CRYAB (1:1000; Cell signalling 8851), anti-actin (1:2000; Sigma, A2066), anti-α-tubulin (1:4000; T9026, Sigma), anti-Flag epitope (1:2000; SIGMA, F7425), anti-GFP (1:1000; Clontech, Living colours, polyclonal), eIF2α (1:1000, Abcam 5369) and phospho-S51-eIF2α_(1:1000, Abcam 32157), GRP78 (1:1000, Abcam 21685), anti-phospho-ERK (1:1000, Cell signalling, 9101), anti-ERK (1:1000, Cell signalling, 9102), anti-phospho-CREB (S133) (1:1000, Cell signalling 9191), anti-CREB 86B10 (1:1000, Cell signalling, 9104), anti-phospho-JNK (1:1000, Cell signalling, 9255), anti-JNK (1:1000, Cell signalling, 9252). The appropriate anti-mouse or anti-rabbit secondary antibodies were used and visualized using an ECL detection kit (Amersham) or LI-COR Biosciences infrared imager (Odyssey).

Western blotting

Cells were washed once in PBS and harvested on lysis buffer (20 mM Tris-HCl pH 6.8, 137nM NaCl, 1 mM EGTA, 1% Triton X-100, 10% glycerol, 1x Roche complete mini protease inhibitor). Equal loading was obtained by protein concentration determination using a Bio-Rad assay followed by resuspension and boiling in Laemli buffer. Samples were subjected to 12% SDS-PAGE and transferred to PVDF membrane (Immobilion-P, GE Healthcare). Blots were proved with primary antibody: anti-LC3 (1:2000; Novus Biologicals, NB100-2220), anti-Hsp70 (1:1000; Enzo SPA810), anti-CRYAB (1:1000; Cell signalling 8851), anti-actin (1:2000; Sigma, A2066), anti-α-tubulin (1:4000; T9026, Sigma), anti-Flag epitope (1:2000; SIGMA, F7425), anti-GFP (1:1000; Clontech, Living colours, polyclonal), eIF2α (1:1000, Abcam 5369) and phospho-S51-eIF2α_(1:1000, Abcam 32157), GRP78 (1:1000, Abcam 21685), anti-phospho-ERK (1:1000, Cell signalling, 9101), anti-ERK (1:1000, Cell signalling, 9102), anti-phospho-CREB (S133) (1:1000, Cell signalling 9191), anti-CREB 86B10 (1:1000, Cell signalling, 9104), anti-phospho-JNK (1:1000, Cell signalling, 9255), anti-JNK (1:1000, Cell signalling, 9252). The appropriate anti-mouse or anti-rabbit secondary antibodies were used and visualized using an ECL detection kit (Amersham) or LI-COR Biosciences infrared imager (Odyssey).

Caspase 3/7 activity assayCells were seeded in a 96-well plate 24h prior to the assay and 1 μM staurosporine or DMSO was added for the last 8h. Caspase 3/7 activity was measured by using a luminogenic caspase 3/7 substrate (Caspase 3/7-Glo Assay, Promega) following manufacturer protocols in a Glomax luminometer (Promega). Protein concentration was determined in each cell lysate and caspase 3/7 activity was normalized to protein levels.

Caspase 3/7 activity assay

Cells were seeded in a 96-well plate 24h prior to the assay and 1 μM staurosporine or DMSO was added for the last 8h. Caspase 3/7 activity was measured by using a luminogenic caspase 3/7 substrate (Caspase 3/7-Glo Assay, Promega) following manufacturer protocols in a Glomax luminometer (Promega). Protein concentration was determined in each cell lysate and caspase 3/7 activity was normalized to protein levels.

Co-inmunoprecipitation assaysAssays were performed as previously described 25, where HTT(1-588)Flag(Q138) and HTT(1-548)GFP(Q138) were expressed in HeLa cells together with QPCT plasmid for 48h, or treated with 25 μM SEN177 for 24h. Cells were lysed in buffer B containing 10 mM Tris pH 7.4, 150 mM NaCl, 1 mM EDTA pH8, 1% triton and 1x Roche complete mini protease inhibitor for 20 min on ice, followed by centrifugation at 13000 rpm for 10 min. Five hundred micrograms total protein were incubated with primary anti-Flag M2 (Sigma) or anti-GFP (Clontech, Living colours, polyclonal) at 5 μg/ml overnight at 4°C. Protein G Dynabeads (LifeTechnologies) were added and incubated for further 2 h. Beads were washed 3 times with buffer B and eluted using 0.1 M glycine pH 2.5M followed by boiling in laemli buffer. Samples were subjected to western blot and visualized using LICOR. A fraction of the total lysates was run simultaneously.

Co-inmunoprecipitation assays

Assays were performed as previously described 25, where HTT(1-588)Flag(Q138) and HTT(1-548)GFP(Q138) were expressed in HeLa cells together with QPCT plasmid for 48h, or treated with 25 μM SEN177 for 24h. Cells were lysed in buffer B containing 10 mM Tris pH 7.4, 150 mM NaCl, 1 mM EDTA pH8, 1% triton and 1x Roche complete mini protease inhibitor for 20 min on ice, followed by centrifugation at 13000 rpm for 10 min. Five hundred micrograms total protein were incubated with primary anti-Flag M2 (Sigma) or anti-GFP (Clontech, Living colours, polyclonal) at 5 μg/ml overnight at 4°C. Protein G Dynabeads (LifeTechnologies) were added and incubated for further 2 h. Beads were washed 3 times with buffer B and eluted using 0.1 M glycine pH 2.5M followed by boiling in laemli buffer. Samples were subjected to western blot and visualized using LICOR. A fraction of the total lysates was run simultaneously.

Reverse-transcriptase PCR analysisTotal RNA was isolated from cell pellets using Trizol Reagent (Invitrogen), treated with DNase I, and cDNA synthesis was performed by SuperScript III First-Strand Synthesis System (Invitrogen). Standard conditions were used for cDNA amplification and PCR products were analyzed by agarose gel electrophoresis and ethidium bromide staining or quantitated with real-time PCR. For real-time PCR analysis, the reaction mixture containing cDNA template, primers, and SYBR Green PCR Master Mix (Invitrogen) was run in a 7900 Fast Real-time PCR System (Applied Bio-systems, Carlsbad, CA). Fold changes on mRNA levels were determined by standard curve and after normalization to internal control β-actin RNA levels. Primer sequences used in this study are: QPCT, 5′-CATGGCATGGATTTATTGG-3′ and 5′- GACGGTATCAGATCAAAC-3′; QPCT-like, 5′- CAGCGTCTCTGGAGCACTTA-3′ and 5′- GCCTCCAGGAACTTTCTGACT-3; GFP 5′- ACGTAAACGGCCACAAGTTC-3′ and 5′- TTCAGGGTCAGCTTGCCGTA-3′; actin, 5′-AGAAAATCTGGCCCACACC-3′ and 5′-GGGGTGTTGAAGGTCTCAAA-3′; CRYAB, 5′- TCTTGAGCTCAGTGAGTACTGG-3′ and 5′-AGCTCACCAGCAGTTCATGG-3′; and mouse QPCT, 5′-CGACTTGAGCCAATTGCTGA-3′ and 5′-CTTCCGGGTTAAGAGTGCTG-3′.

Reverse-transcriptase PCR analysis

Total RNA was isolated from cell pellets using Trizol Reagent (Invitrogen), treated with DNase I, and cDNA synthesis was performed by SuperScript III First-Strand Synthesis System (Invitrogen). Standard conditions were used for cDNA amplification and PCR products were analyzed by agarose gel electrophoresis and ethidium bromide staining or quantitated with real-time PCR. For real-time PCR analysis, the reaction mixture containing cDNA template, primers, and SYBR Green PCR Master Mix (Invitrogen) was run in a 7900 Fast Real-time PCR System (Applied Bio-systems, Carlsbad, CA). Fold changes on mRNA levels were determined by standard curve and after normalization to internal control β-actin RNA levels. Primer sequences used in this study are: QPCT, 5′-CATGGCATGGATTTATTGG-3′ and 5′- GACGGTATCAGATCAAAC-3′; QPCT-like, 5′- CAGCGTCTCTGGAGCACTTA-3′ and 5′- GCCTCCAGGAACTTTCTGACT-3; GFP 5′- ACGTAAACGGCCACAAGTTC-3′ and 5′- TTCAGGGTCAGCTTGCCGTA-3′; actin, 5′-AGAAAATCTGGCCCACACC-3′ and 5′-GGGGTGTTGAAGGTCTCAAA-3′; CRYAB, 5′- TCTTGAGCTCAGTGAGTACTGG-3′ and 5′-AGCTCACCAGCAGTTCATGG-3′; and mouse QPCT, 5′-CGACTTGAGCCAATTGCTGA-3′ and 5′-CTTCCGGGTTAAGAGTGCTG-3′.

mRNA isolation from mouse brainAll mouse experiments were performed under appropriate UK Home Office licences and following institutional procedures. We analyzed samples from N171 mutant HD mice and wild-type littermate controls at 20 week. mRNA was extracted from brains homogenized in Trizol (Invitrogen) using an Ultra torax homogenizer.

mRNA isolation from mouse brain

All mouse experiments were performed under appropriate UK Home Office licences and following institutional procedures. We analyzed samples from N171 mutant HD mice and wild-type littermate controls at 20 week. mRNA was extracted from brains homogenized in Trizol (Invitrogen) using an Ultra torax homogenizer.

Lentivirus infectionshRNA containing pLKO.1 vectors targeting both mouse and human QPCT (TRCN032432) were obtained from The RNAi Consortium (TRC) and scramble shRNA vector was generated in D. Sabatinit’s laboratory (Addgene, plasmid 1864). Lentiviral plasmids to express Q80-GFP were kindly provided by J. Uney 55. Lentiviral particles were produced and transduced following The RNAi Consortium protocols.

Lentivirus infection

shRNA containing pLKO.1 vectors targeting both mouse and human QPCT (TRCN032432) were obtained from The RNAi Consortium (TRC) and scramble shRNA vector was generated in D. Sabatinit’s laboratory (Addgene, plasmid 1864). Lentiviral plasmids to express Q80-GFP were kindly provided by J. Uney 55. Lentiviral particles were produced and transduced following The RNAi Consortium protocols.

Cell toxicity and aggregation assaysCells were fixed for 7 minutes in 4% paraformaldehyde (PFA). For EGFP-tagged constructs, slides were mounted in Citifluor (Citifluor, Ltd.) containing 4′,6-diamidino-2-phenylindole (DAPI; 3 μg/ml; Sigma) and visualized using an Eclipse E600 fluorescence microscope (plan-apo 60x/1.4 oil immersion lens) (Nikon). For detection of HA-tagged constructs, inmunofluorescence with an anti-HA (Covance laboratories 1:500) and anti-mouse Alexa488 secondary antibody (Invitrogen, 1:1000) was performed followed by mounting in Citiflour-DAPI. We assessed the percentage of transfected cells (EGFP- or HA-positive cells) with at least one aggregate per cell. Apoptotic cell death was determined by assessing the nuclear morphology (nuclei fragmented or condensed) in transfected cells. Slides were blinded and at least 200 transfected cells per slide were scored; each individual experiment was performed in triplicate.

Cell toxicity and aggregation assays

Cells were fixed for 7 minutes in 4% paraformaldehyde (PFA). For EGFP-tagged constructs, slides were mounted in Citifluor (Citifluor, Ltd.) containing 4′,6-diamidino-2-phenylindole (DAPI; 3 μg/ml; Sigma) and visualized using an Eclipse E600 fluorescence microscope (plan-apo 60x/1.4 oil immersion lens) (Nikon). For detection of HA-tagged constructs, inmunofluorescence with an anti-HA (Covance laboratories 1:500) and anti-mouse Alexa488 secondary antibody (Invitrogen, 1:1000) was performed followed by mounting in Citiflour-DAPI. We assessed the percentage of transfected cells (EGFP- or HA-positive cells) with at least one aggregate per cell. Apoptotic cell death was determined by assessing the nuclear morphology (nuclei fragmented or condensed) in transfected cells. Slides were blinded and at least 200 transfected cells per slide were scored; each individual experiment was performed in triplicate.

Detection of nascent protein synthesisProtein synthesis was assessed by metabolic incorporation of AHA (L-azidohomoalanine) into cells transfected with EGFP-HTT(Q23). Briefly, 12 hours after HeLa cells transfection, media was washed and replaced with L-methionine/L-cysteine free medium and treated with DMSO or SEN177 (50 uM) for 1h prior to addition of AHA (L-azidohomoalanine) to the media and collection of cells every 2 hours. Labelled protein was detected by western blot after performing Click-IT protein detection assay (Life Technologies) using biotin, following manufacturer protocols.

Detection of nascent protein synthesis

Protein synthesis was assessed by metabolic incorporation of AHA (L-azidohomoalanine) into cells transfected with EGFP-HTT(Q23). Briefly, 12 hours after HeLa cells transfection, media was washed and replaced with L-methionine/L-cysteine free medium and treated with DMSO or SEN177 (50 uM) for 1h prior to addition of AHA (L-azidohomoalanine) to the media and collection of cells every 2 hours. Labelled protein was detected by western blot after performing Click-IT protein detection assay (Life Technologies) using biotin, following manufacturer protocols.

Luciferase reporter assayCells were transfected with 1 μg of GRP78-luciferase (firefly) reporter construct and 50 ng of renilla-luciferase (pRL-TK) as an internal transfection efficiency control. Cells were collected in Passive lysis buffer and luciferase activity was measured using the Dual-luciferase Reporter Assay System (Promega) following manufacturer’s protocol in a Glomax Luminometer (Promega). GRP78-luciferase relative activity was calculated relative to the renilla-luciferase transfection efficiency control activity for each sample; experiments were performed in triplicate.

Luciferase reporter assay

Cells were transfected with 1 μg of GRP78-luciferase (firefly) reporter construct and 50 ng of renilla-luciferase (pRL-TK) as an internal transfection efficiency control. Cells were collected in Passive lysis buffer and luciferase activity was measured using the Dual-luciferase Reporter Assay System (Promega) following manufacturer’s protocol in a Glomax Luminometer (Promega). GRP78-luciferase relative activity was calculated relative to the renilla-luciferase transfection efficiency control activity for each sample; experiments were performed in triplicate.

Statistical analysisQuantification of immunoblots was performed by densitometric analysis using the Image J software or the LI-COR Biosciences infrared imager software and normalized to loading control (actin or tubulin, as indicated). The p-values were determined by two-tailed Student’s t-test.Aggregates were counted in at least 200 cells per slide (with the observer blinded to their identity), and percentage was calculated relative to control conditions. p-values were determined by unpaired two-tailed Student’s t-test.All experiments were done at least three times in triplicate and a representative blot or graph from a triplicate experiment is shown unless indicated.

Statistical analysis

Quantification of immunoblots was performed by densitometric analysis using the Image J software or the LI-COR Biosciences infrared imager software and normalized to loading control (actin or tubulin, as indicated). The p-values were determined by two-tailed Student’s t-test.

Aggregates were counted in at least 200 cells per slide (with the observer blinded to their identity), and percentage was calculated relative to control conditions. p-values were determined by unpaired two-tailed Student’s t-test.

All experiments were done at least three times in triplicate and a representative blot or graph from a triplicate experiment is shown unless indicated.

Heat shock proteins and chaperones PCR arrayThe Human Heat Shock Proteins and Chaperones RT2 Profiler PCR Array (SABiosciences, Frederick, MD) was used to study the expression profile of 84 heat shock proteins according to the manufacturer’s procedure. Briefly, total RNA was extracted from cells transfected with HTT(Q74)GFP treated with DMSO or 25 uM of SEN177 inhibitor for 24h, using Trizol (Invitrogen) and further purified using RNeasy mini kit with oncolumn DNAse digest (Qiagen), cDNA was then synthesized using an RT2 First strand kit (SABiosciences) and real-time PCR was performed using 7900HT fast real time PCR system (Applied iosciences). Data were analysed with RT2 profiler PCR array data analysis software version 3.5.

Heat shock proteins and chaperones PCR array

The Human Heat Shock Proteins and Chaperones RT2 Profiler PCR Array (SABiosciences, Frederick, MD) was used to study the expression profile of 84 heat shock proteins according to the manufacturer’s procedure. Briefly, total RNA was extracted from cells transfected with HTT(Q74)GFP treated with DMSO or 25 uM of SEN177 inhibitor for 24h, using Trizol (Invitrogen) and further purified using RNeasy mini kit with oncolumn DNAse digest (Qiagen), cDNA was then synthesized using an RT2 First strand kit (SABiosciences) and real-time PCR was performed using 7900HT fast real time PCR system (Applied iosciences). Data were analysed with RT2 profiler PCR array data analysis software version 3.5.

Assays for validation of polyglutamine aggregation modifiers in zebrafishMaintenance of zebrafish stocks and collection of embryosAll zebrafish husbandry and experiments were performed in accordance with UK legislation under a licence granted by the Home Office and with local ethical approval. Zebrafish were reared under standard conditions (Westerfield et al, 2005) on a 14 h light/10 h dark cycle. Embryos were collected from natural spawnings, staged according to the established criteria 56 and reared in embryo medium (5 mM NaCl, 0.17 mM KCl, 0.33 mM CaCl2, 0.33 mM Mg2SO4, 5 mM HEPES).Determination of the maximum-tolerated concentration of compounds in larval zebrafishCompound exposure experiments were performed on wild-type larvae (TL strain) from 2 to 3 days post-fertilization (d.p.f.). Concentration response assays were performed over log intervals, from 100 nM to 1 mM, to determine the maximum non-toxic concentration (MTC) for subsequent aggregate analysis assays (n=10 larvae per concentration). Compound exposure experiments were performed in the dark at 28.5 °C.Measuring aggregate number and rhodopsin protein levels in transgenic HD zebrafishAggregate counting and analysis of rod photoreceptor degeneration (photoreceptor number) was performed using heterozygous larvae from Tg (rho:EGFP-HTT71Q)cu5 zebrafish 42 (hereafter referred to as transgenic HD zebrafish). Embryos from outcrossed transgenic HD zebrafish were raised in 0.2 mM 1-phenyl-2-thiourea (PTU) from 1 to 3 d.p.f. to inhibit pigment formation, screened for transgene expression using EGFP fluorescence, and then washed twice in the embryo medium to remove PTU. From 3 to 9 d.p.f., transgenic HD zebrafish larvae were dark-reared in embryo medium alone or embryo medium contain containing either DMSO, 1mM SEN177, 100 μM SEN180 or 100 μM SEN817. Embryo medium and compounds were replenished daily. Larvae were anaesthetized by immersion in 0.2 mg/ml 3-amino benzoic acid ethyl ester (MS222), then fixed for aggregate counting at 7 d.p.f. or for photoreceptor analysis at 9 d.p.f. Anaesthetised larvae were fixed using 4% paraformaldehyde (PFA) in PBS at 4 °C. Larvae were washed briefly in PBS, allowed to equilibrate in 30% sucrose in PBS then embedded in OCT medium (Tissue-Tek) and frozen on dry ice for subsequent cryosectioning. Sections were cut at 10 μm thickness using a cryostat (Bright Instruments). For aggregate counting, sections were mounted in 50% glycerol in PBS and the total number of GFP-positive aggregates were counted over 100 μm of the central retina, either side of the optic nerve head and mean values were calculated (n = 5 fish (10 eyes)) for each treatment group. For quantification of photoreceptor number, the GFP-positive area of the central retina was quantified using image thresholding and area analysis in ImageJ (n ≥ 5 fish (10 eyes) for each treatment group). To demonstrate that loss of GFP corresponds to loss of photoreceptors, sections were stained with anti-rhodopsin (1D1) antibody (a kind gift from Paul Linser, University of Florida, FL 57 and mounted using VectaShield hard set mounting medium (Vector Laboratories). Sections were viewed and representative images acquired using a GX Optical LED fluorescent microscope, GXCAM3.3 digital camera and GX Capture software.

Assays for validation of polyglutamine aggregation modifiers in zebrafish

Maintenance of zebrafish stocks and collection of embryosAll zebrafish husbandry and experiments were performed in accordance with UK legislation under a licence granted by the Home Office and with local ethical approval. Zebrafish were reared under standard conditions (Westerfield et al, 2005) on a 14 h light/10 h dark cycle. Embryos were collected from natural spawnings, staged according to the established criteria 56 and reared in embryo medium (5 mM NaCl, 0.17 mM KCl, 0.33 mM CaCl2, 0.33 mM Mg2SO4, 5 mM HEPES).

Maintenance of zebrafish stocks and collection of embryos

All zebrafish husbandry and experiments were performed in accordance with UK legislation under a licence granted by the Home Office and with local ethical approval. Zebrafish were reared under standard conditions (Westerfield et al, 2005) on a 14 h light/10 h dark cycle. Embryos were collected from natural spawnings, staged according to the established criteria 56 and reared in embryo medium (5 mM NaCl, 0.17 mM KCl, 0.33 mM CaCl2, 0.33 mM Mg2SO4, 5 mM HEPES).

Determination of the maximum-tolerated concentration of compounds in larval zebrafishCompound exposure experiments were performed on wild-type larvae (TL strain) from 2 to 3 days post-fertilization (d.p.f.). Concentration response assays were performed over log intervals, from 100 nM to 1 mM, to determine the maximum non-toxic concentration (MTC) for subsequent aggregate analysis assays (n=10 larvae per concentration). Compound exposure experiments were performed in the dark at 28.5 °C.

Determination of the maximum-tolerated concentration of compounds in larval zebrafish

Compound exposure experiments were performed on wild-type larvae (TL strain) from 2 to 3 days post-fertilization (d.p.f.). Concentration response assays were performed over log intervals, from 100 nM to 1 mM, to determine the maximum non-toxic concentration (MTC) for subsequent aggregate analysis assays (n=10 larvae per concentration). Compound exposure experiments were performed in the dark at 28.5 °C.

Measuring aggregate number and rhodopsin protein levels in transgenic HD zebrafishAggregate counting and analysis of rod photoreceptor degeneration (photoreceptor number) was performed using heterozygous larvae from Tg (rho:EGFP-HTT71Q)cu5 zebrafish 42 (hereafter referred to as transgenic HD zebrafish). Embryos from outcrossed transgenic HD zebrafish were raised in 0.2 mM 1-phenyl-2-thiourea (PTU) from 1 to 3 d.p.f. to inhibit pigment formation, screened for transgene expression using EGFP fluorescence, and then washed twice in the embryo medium to remove PTU. From 3 to 9 d.p.f., transgenic HD zebrafish larvae were dark-reared in embryo medium alone or embryo medium contain containing either DMSO, 1mM SEN177, 100 μM SEN180 or 100 μM SEN817. Embryo medium and compounds were replenished daily. Larvae were anaesthetized by immersion in 0.2 mg/ml 3-amino benzoic acid ethyl ester (MS222), then fixed for aggregate counting at 7 d.p.f. or for photoreceptor analysis at 9 d.p.f. Anaesthetised larvae were fixed using 4% paraformaldehyde (PFA) in PBS at 4 °C. Larvae were washed briefly in PBS, allowed to equilibrate in 30% sucrose in PBS then embedded in OCT medium (Tissue-Tek) and frozen on dry ice for subsequent cryosectioning. Sections were cut at 10 μm thickness using a cryostat (Bright Instruments). For aggregate counting, sections were mounted in 50% glycerol in PBS and the total number of GFP-positive aggregates were counted over 100 μm of the central retina, either side of the optic nerve head and mean values were calculated (n = 5 fish (10 eyes)) for each treatment group. For quantification of photoreceptor number, the GFP-positive area of the central retina was quantified using image thresholding and area analysis in ImageJ (n ≥ 5 fish (10 eyes) for each treatment group). To demonstrate that loss of GFP corresponds to loss of photoreceptors, sections were stained with anti-rhodopsin (1D1) antibody (a kind gift from Paul Linser, University of Florida, FL 57 and mounted using VectaShield hard set mounting medium (Vector Laboratories). Sections were viewed and representative images acquired using a GX Optical LED fluorescent microscope, GXCAM3.3 digital camera and GX Capture software.

Measuring aggregate number and rhodopsin protein levels in transgenic HD zebrafish

Aggregate counting and analysis of rod photoreceptor degeneration (photoreceptor number) was performed using heterozygous larvae from Tg (rho:EGFP-HTT71Q)cu5 zebrafish 42 (hereafter referred to as transgenic HD zebrafish). Embryos from outcrossed transgenic HD zebrafish were raised in 0.2 mM 1-phenyl-2-thiourea (PTU) from 1 to 3 d.p.f. to inhibit pigment formation, screened for transgene expression using EGFP fluorescence, and then washed twice in the embryo medium to remove PTU. From 3 to 9 d.p.f., transgenic HD zebrafish larvae were dark-reared in embryo medium alone or embryo medium contain containing either DMSO, 1mM SEN177, 100 μM SEN180 or 100 μM SEN817. Embryo medium and compounds were replenished daily. Larvae were anaesthetized by immersion in 0.2 mg/ml 3-amino benzoic acid ethyl ester (MS222), then fixed for aggregate counting at 7 d.p.f. or for photoreceptor analysis at 9 d.p.f. Anaesthetised larvae were fixed using 4% paraformaldehyde (PFA) in PBS at 4 °C. Larvae were washed briefly in PBS, allowed to equilibrate in 30% sucrose in PBS then embedded in OCT medium (Tissue-Tek) and frozen on dry ice for subsequent cryosectioning. Sections were cut at 10 μm thickness using a cryostat (Bright Instruments). For aggregate counting, sections were mounted in 50% glycerol in PBS and the total number of GFP-positive aggregates were counted over 100 μm of the central retina, either side of the optic nerve head and mean values were calculated (n = 5 fish (10 eyes)) for each treatment group. For quantification of photoreceptor number, the GFP-positive area of the central retina was quantified using image thresholding and area analysis in ImageJ (n ≥ 5 fish (10 eyes) for each treatment group). To demonstrate that loss of GFP corresponds to loss of photoreceptors, sections were stained with anti-rhodopsin (1D1) antibody (a kind gift from Paul Linser, University of Florida, FL 57 and mounted using VectaShield hard set mounting medium (Vector Laboratories). Sections were viewed and representative images acquired using a GX Optical LED fluorescent microscope, GXCAM3.3 digital camera and GX Capture software.

Supplementary MaterialDataSet1DataSet2DataSet3Supplementary InformationSupplementary Note 1Supplementary Note 2

Supplementary Material
