---
pmid: '12925853'
title: SIGIRR, a negative regulator of Toll-like receptor-interleukin 1 receptor signaling.
authors:
- Wald D
- Qin J
- Zhao Z
- Qian Y
- Naramura M
- Tian L
- Towne J
- Sims JE
- Stark GR
- Li X
journal: Nat Immunol
year: '2003'
full_text_available: true
pmcid: PMC12005339
doi: 10.1038/ni968
---

# SIGIRR, a negative regulator of Toll-like receptor-interleukin 1 receptor signaling.
**Authors:** Wald D, Qin J, Zhao Z, Qian Y, Naramura M, Tian L, Towne J, Sims JE, Stark GR, Li X
**Journal:** Nat Immunol (2003)
**DOI:** [10.1038/ni968](https://doi.org/10.1038/ni968)
**PMC:** [PMC12005339](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC12005339/)

## Abstract

1. Nat Immunol. 2003 Sep;4(9):920-7. doi: 10.1038/ni968. Epub 2003 Aug 17.

SIGIRR, a negative regulator of Toll-like receptor-interleukin 1 receptor 
signaling.

Wald D(1), Qin J, Zhao Z, Qian Y, Naramura M, Tian L, Towne J, Sims JE, Stark 
GR, Li X.

Author information:
(1)Cleveland Clinic Foundation, Department of Immunology, Cleveland, Ohio 44195, 
USA.

Comment in
    Nat Immunol. 2003 Sep;4(9):823-4. doi: 10.1038/ni0903-823.

The Toll-like receptor-interleukin 1 receptor signaling (TLR-IL-1R) receptor 
superfamily is important in differentially recognizing pathogen products and 
eliciting appropriate immune responses. These receptors alter gene expression, 
mainly through the activation of nuclear factor-kappaB and activating protein 1. 
SIGIRR (single immunoglobulin IL-1R-related molecule), a member of this family 
that does not activate these factors, instead negatively modulates immune 
responses. Inflammation is enhanced in SIGIRR-deficient mice, as shown by their 
enhanced chemokine induction after IL-1 injection and reduced threshold for 
lethal endotoxin challenge. Cells from SIGIRR-deficient mice showed enhanced 
activation in response to either IL-1 or certain Toll ligands. Finally, 
biochemical analysis indicated that SIGIRR binds to the TLR-IL-1R signaling 
components in a ligand-dependent way. Our data show that SIGIRR functions as a 
biologically important modulator of TLR-IL-1R signaling.

DOI: 10.1038/ni968
PMID: 12925853 [Indexed for MEDLINE]

## Full Text

IntroductionThe gut microbiome is critical for the development and maturation of the intestinal tract and imprints immune cell and brain development.1,2,3 Several studies have profiled the progression of microbial community and diversity from the time of birth till the establishment of adult-like stable gut microbiome in humans.4,5,6,7,8,9 These studies have identified a distinct pattern of intestinal colonization starting with a neonatal phase, followed sequentially by developmental (3–14 months), transitional (15–31 months), and stable phases (>31 months).6,7,8 Breast milk intake, mode of birth, and geographical factors are the key factors programming development of the gut microbiome in humans during infancy.6,8,9Mice are ubiquitously used to investigate host-microbiota interactions that underpin intestinal homeostasis as well as gut-brain and gut-immune system crosstalk.10 The sequential progression of the gut microbiome community structure and diversity in the human equivalent of the neonatal and infancy period remains understudied in mice. Further, the majority of studies in humans and mice have investigated the stool microbiome as a surrogate for the intestinal microbiome.11,12 Considering the marked differences in the function of ileum and colon, this convenient approach masks the regional heterogeneity of the ileal and the colonic microbiome that likely influences intestinal health and disease.11,12 Positing that early gut colonization will exhibit distinct patterns of evolution over time and space in mice, we studied the ileal and colonic microbiome after birth to post-weaning. The functional profiling of the metagenome was predicted using Phylogenetic Investigation of Communities by Reconstruction of Unobserved States (PICRUSt2),13 with subsequent analysis of MetaCyc and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathways.During the phase of postnatal adaptation to rapid intestinal colonization neonates are vulnerable to diseases arising from deviant host-microbiota interactions, the most devastating of which is necrotizing enterocolitis (NEC) in premature infants.14,15,16,17 Developmental dysmaturity of preterm intestine, formula milk feeding, ischemia, oxygen toxicity, and genetics are considered risk factors for NEC.18,19,20,21,22 While the pathogenesis of NEC is complex, intestinal dysbiosis occurring in the setting of the immature preterm intestine is believed to trigger bacterial invasion and unregulated Toll like receptor (TLR)-mediated inflammation and intestinal injury.15,16,18,19,22 Human studies have shown that NEC is associated with dysbiosis characterized by enrichment of Gammaproteobacteria spp, decreased bacterial diversity, and modest reductions in anerobic species such as Bacteroides spp.14,15,19 Mice are frequently used to model several pediatric and adult diseases and is the most common model used to study the pathogenesis of NEC.10,23,24,25 Whether formula milk feeding induces intestinal dysbiosis that predisposes to NEC similar to that described in preterm infants fed formula milk was evaluated in this study in order to understand if microbiota changes associated with human NEC are recapitulated in the mouse model.Several epidemiological and genetic studies suggest that interindividual variability to NEC susceptibility exists.21,26 Our laboratory has identified single immunoglobulin interleukin-1 related receptor (SIGIRR) as a potential locus for NEC susceptibility in premature infants.27 SIGIRR is a major inhibitor of TLR and interleukin-1 mediated inflammatory signaling and is highly expressed in the adult intestine.28 Previous work from our lab and others has shown that the developmental immaturity of neonatal intestinal SIGIRR expression in mice and human corresponds to the period of increased NEC vulnerability in mice.18,22,29,30 The SIGIRR p.Y168X mutation identified in human NEC exaggerates inflammatory responses to TLR ligands in human intestinal epithelial cells and induces intestinal TLR hypersensitivity in mice that disrupts postnatal gut adaptation to bacterial colonization.22,27 Intrigued by studies in humans that noted a modest effect of host genetics on the gut microbiome, we queried whether SIGIRR genotypes alter the microbiota composition in the developing intestine.31 To study this, we temporally investigated the microbiota composition during postnatal gut adaptation in mice (SigirrMut) encoding the SIGIRR p.Y168X mutation that we previously generated.22 We hypothesized that the SIGIRR p.Y168X identified in human NEC will alter the gut microbiome either at baseline or after intestinal injury when compared to Sigirr sufficient wild-type (WT) mice. Our results show dynamic changes in the microbiota community structure and function across time, site of sampling and with injury in the developing intestine. We also reveal that the SIGIRR p.Y168X mutation exaggerates dysbiosis induced by formula milk in neonatal mice that mimics the dysbiosis reported in preterm infants fed formula milk.5,14,15

Introduction

The gut microbiome is critical for the development and maturation of the intestinal tract and imprints immune cell and brain development.1,2,3 Several studies have profiled the progression of microbial community and diversity from the time of birth till the establishment of adult-like stable gut microbiome in humans.4,5,6,7,8,9 These studies have identified a distinct pattern of intestinal colonization starting with a neonatal phase, followed sequentially by developmental (3–14 months), transitional (15–31 months), and stable phases (>31 months).6,7,8 Breast milk intake, mode of birth, and geographical factors are the key factors programming development of the gut microbiome in humans during infancy.6,8,9

Mice are ubiquitously used to investigate host-microbiota interactions that underpin intestinal homeostasis as well as gut-brain and gut-immune system crosstalk.10 The sequential progression of the gut microbiome community structure and diversity in the human equivalent of the neonatal and infancy period remains understudied in mice. Further, the majority of studies in humans and mice have investigated the stool microbiome as a surrogate for the intestinal microbiome.11,12 Considering the marked differences in the function of ileum and colon, this convenient approach masks the regional heterogeneity of the ileal and the colonic microbiome that likely influences intestinal health and disease.11,12 Positing that early gut colonization will exhibit distinct patterns of evolution over time and space in mice, we studied the ileal and colonic microbiome after birth to post-weaning. The functional profiling of the metagenome was predicted using Phylogenetic Investigation of Communities by Reconstruction of Unobserved States (PICRUSt2),13 with subsequent analysis of MetaCyc and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathways.

During the phase of postnatal adaptation to rapid intestinal colonization neonates are vulnerable to diseases arising from deviant host-microbiota interactions, the most devastating of which is necrotizing enterocolitis (NEC) in premature infants.14,15,16,17 Developmental dysmaturity of preterm intestine, formula milk feeding, ischemia, oxygen toxicity, and genetics are considered risk factors for NEC.18,19,20,21,22 While the pathogenesis of NEC is complex, intestinal dysbiosis occurring in the setting of the immature preterm intestine is believed to trigger bacterial invasion and unregulated Toll like receptor (TLR)-mediated inflammation and intestinal injury.15,16,18,19,22 Human studies have shown that NEC is associated with dysbiosis characterized by enrichment of Gammaproteobacteria spp, decreased bacterial diversity, and modest reductions in anerobic species such as Bacteroides spp.14,15,19 Mice are frequently used to model several pediatric and adult diseases and is the most common model used to study the pathogenesis of NEC.10,23,24,25 Whether formula milk feeding induces intestinal dysbiosis that predisposes to NEC similar to that described in preterm infants fed formula milk was evaluated in this study in order to understand if microbiota changes associated with human NEC are recapitulated in the mouse model.

Several epidemiological and genetic studies suggest that interindividual variability to NEC susceptibility exists.21,26 Our laboratory has identified single immunoglobulin interleukin-1 related receptor (SIGIRR) as a potential locus for NEC susceptibility in premature infants.27 SIGIRR is a major inhibitor of TLR and interleukin-1 mediated inflammatory signaling and is highly expressed in the adult intestine.28 Previous work from our lab and others has shown that the developmental immaturity of neonatal intestinal SIGIRR expression in mice and human corresponds to the period of increased NEC vulnerability in mice.18,22,29,30 The SIGIRR p.Y168X mutation identified in human NEC exaggerates inflammatory responses to TLR ligands in human intestinal epithelial cells and induces intestinal TLR hypersensitivity in mice that disrupts postnatal gut adaptation to bacterial colonization.22,27 Intrigued by studies in humans that noted a modest effect of host genetics on the gut microbiome, we queried whether SIGIRR genotypes alter the microbiota composition in the developing intestine.31 To study this, we temporally investigated the microbiota composition during postnatal gut adaptation in mice (SigirrMut) encoding the SIGIRR p.Y168X mutation that we previously generated.22 We hypothesized that the SIGIRR p.Y168X identified in human NEC will alter the gut microbiome either at baseline or after intestinal injury when compared to Sigirr sufficient wild-type (WT) mice. Our results show dynamic changes in the microbiota community structure and function across time, site of sampling and with injury in the developing intestine. We also reveal that the SIGIRR p.Y168X mutation exaggerates dysbiosis induced by formula milk in neonatal mice that mimics the dysbiosis reported in preterm infants fed formula milk.5,14,15

ResultsTemporal changes in the terminal ileum and colon microbiome during postnatal gut developmentTo determine changes in the small intestinal microbiota community structure and function with gut maturity, we collected terminal ileum (TI) contents from wild-type mouse pups at postnatal day 11 (P11), 14 (P14), and 28 (P28) and subjected them to V3-V4 sequencing. Earlier time points were less amenable to interrogation as luminal contents were scarce in the ileum. Figure S1A is a boxplot of the read counts at indicated days of life. We also tested mice of both sexes but did not disaggregate the data as the indices were not significant (Figure S1B). Analysis of the sequencing data revealed amplicon sequence variants that were plotted as pie charts to reveal dominant phyla. The core microbiota was composed of Firmicutes (71.3%, 77.2%, and 84.8%) at P11, P14, and P28 and Bacteroidetes (0.3%, 3.4%, and 14.2%) at P14 and P28, respectively (Figure 1A). Proteobacteria on the other hand, was observed at 28.3% at P11 and at 18.7% at P14 but the abundance diminished at P28 (Figure 1A). At the genus level, Lactobacillus (67.1%, 67.8%, and 72.4% dominated at P11, P14, and P28 while genus level changes in Muribacter (22.1% and 5.5%) at P11 and P14 and Escherichia_Shigella at P14 were less notable (Figure S1C). When we looked at alpha diversity (i.e., within-sample diversity) at different time points using the Chao1 and Shannon indices of richness and evenness, we discovered that both Chao1 and Shannon indices were higher at P11 with subsequent decline in diversity at P14 and P28, respectively. These changes were, however, not significant (Figures 1B and 1C). In contrast, β-diversity indices including Bray-Curtis and Jensen-Shannon were significantly different particularly at postnatal day 11 and 28, respectively (Figures 1D and 1E). This was captured by force-directed plotting of a distance threshold network with Fruchterman-Reingold network graph on counts of ileum specimens across days of life. Bray distance showed the progression in dissimilarity from day 11 to day 28 and co-located clusters suggesting possible strong correlations (Figure 1F). Specifically, the day 14 and day 28 samples looked very similar despite being from different mice; however, the day 11 samples showed a heterogeneity as the force-directed layout pushed them apart. Principal-component analysis (PCA) revealed that the P11 and P28 were significantly separated, with 63.3 and 6.3% of the variation explained by principal component 1 (PC1) and PC3, respectively, while PC2 revealed 18.0% variation at the two time points (Figure 1G). Employing linear discriminant analysis effect size (LEfSe) analysis to obtain dominant bacterial phyla and genera at indicated time points, we discovered that Proteobacteria at P11 and Firmicutes and Bacteroidetes at P28 were the dominant phyla while Actinobacteria also contributed to a linear discriminant analysis (LDA) score of >4 (Figure 1H). This correlated with the abundance of 9 key bacteria in the ileum including g_Muribacter, g_Candidatus_Arthromitus, g_Escherichia_Shigella, g_Streptococcus, g_Pseudomonas, and g_Staphylococcus among others (Figure 1I; LDA >4, p < 0.05).Figure 1Microbial profile of terminal ileum (TI) content across time(A) Pie charts representing the abundance of major phyla at postnatal day 11 (P11), P14, and P28.(B and C) Alpha diversity indices of indicated samples showing Chao 1 (B) and Shannon (C) indices, respectively (p values as indicated).(D and E) β-diversity visualization with non-metric multi-dimensional scaling (NMDS) based on Bray-Curtis (D) and distance and Jensen-Shannon divergence metric (E; p values as indicated).(F) Fruchterman-Reingold network graph on counts of ileum specimens across days of life. Bray distance showing the progression in dissimilarity from day 11 to day 28.(G) The principal-component (PC) analysis plots showing significant separation between P11 and P28 with 63.3% (PC1) and 6.3% (PC3) of the variation while PC2 revealed 18.0% variation at the two time points.(H and I) The linear discriminant analysis (LDA) value distribution histogram. Taxa meeting a linear discriminant analysis significant threshold >4 are shown. t test or PERMANOVA were used as the statistical methods to assess differences among the groups. n = 7–11 pups in each group.We next profiled the microbiome changes in the colon of WT mice on postnatal days 8, 11, 14, and 28, respectively. Figure S1A is a boxplot of the read counts at indicated days of life. The core microbiota was composed of taxa belonging to the phylum Firmicutes (66.8%, 91.2%, 57%, and 62.3%) and Bacteroidetes (0.1%, 2.5%, 30.8%, and 35.8%). Interestingly, Proteobacteria was detected at 32.5% at P8; however, the levels did not sustain at P11 and P14 (6.3% and 11.8%) and Proteobacteria were not detected at P28 (Figure 2A). At the genus level, Lactobacillus (63.5% and 80.2%) dominated over Streptococcus (2.3% and 5.4%) at P8 and P11 (Figure S1). At P14 and P28, while Lactobacillus levels were maintained at 40% and 40.9%, we did not detect Streptococcus at these time points (Figure S1D). To further delve into the Proteobacteria genera, we looked at Escherichia_Shigella and noted significant presence at P8 (Figure S1). At P14 however, the levels were reduced to 11.1% and Escherichia_Shigella was not detected at P28 (Figure S1C) consistent with phylum level decline at this time point (see Figure 2A). When we looked at alpha diversity, we discovered significant variation in alpha diversity in the colon at different time points wherein, Chao1 indices were higher at P8 with a drop at P11 followed by gradual increases at P14 and P28 (Figure 2B). Shannon index at P8, however, was significantly lower with gradual increase at P11 and P28, respectively (Figure 2C). The non-metric multidimensional scaling (NMDS) along with Permutational Multivariate Analysis of Variance (PERMANOVA) evaluation based on Bray-Curtis dissimilarity measures and Jensen-Shannon divergence discovered statistically significant differences in community composition, particularly at P8 and P28 time points by 16S rRNA profiles (Figures 2D and 2E). This was elegantly captured by force-directed plotting of a distance threshold network with Fruchterman-Reingold that demonstrated the graph on counts of colon specimens across days of life. Bray distance showed the progression in dissimilarity from day 8 to day 28, and the co-occurrence of bacterial networks as they group together with less variation in day 11 samples and more clustering across all days (Figure 2F).Figure 2Microbial profile of colon content across time(A) Pie charts representing the abundance of major phyla at postnatal day 11 (P11), P14, and P28.(B and C) Alpha diversity indices of indicated samples showing Chao 1 (B) and Shannon (C) indices, respectively (p values as indicated).(D and E) β-diversity visualization with Non-metric Multi-dimensional Scaling (NMDS) based on Bray-Curtis distance (D) and Jensen-Shannon divergence metric (E; p values as indicated).(F) Fruchterman-Reingold network graph on counts of ileum specimens across days of life. Bray distance showing the progression in dissimilarity from day 11 to day 28.(G) The principal-component (PC) analysis plots showing significant separation between P8 and P28 with 52.3 (PC1) and 7.9% (PC3) of the variation while PC2 revealed 32.7% variation at the two time points.(H and I) The linear discriminant analysis (LDA) value distribution histogram. Taxa meeting a linear discriminant analysis significant threshold >4 are shown. t test or PERMANOVA were used as the statistical methods to assess differences among the groups. n = 7–11 pups in each group.PCA revealed that the P8 and P28 were significantly separated, with 52.3 and 7.9% of the variation explained by PC1 and PC3, respectively, while PC2 revealed 32.7% variation at the two time points (Figure 2G). Employing LEfSe analysis to obtain dominant bacterial phyla and genera at indicated time points, we discovered that Proteobacteria at P8, Firmicutes at P11, and Bacteroidetes at P28 were the dominant phyla. This correlated with the abundance of 15 key bacteria in the colon including g_Escherichia_Shigella, g_Lactobacillus, g_Lachnospiraceae_NK4A136_group, g_Bacteroides, and g_Streptococcus among others (Figure 2I; LDA >4, p < 0.05).Predicted bacterial function of microbiome based on analysis of metabolic pathwaysThe biosynthesis and metabolism of gut bacteria influence host physiology.32 To elucidate the metabolic pathways activated at different days of life in the terminal ileum and the colon, we performed PICRUSt2 using MetaCyc database as a reference. The abundance of biosynthetic processes identified using STAMP, was higher when compared to other metabolic pathways. The major biological pathways in the terminal ileum included amino acid biosynthesis, urea cycle, purine and pyrimidine, and peptidoglycan biosynthesis, respectively (Figure S2A). The kinetics of the three pathways related to lysine biosynthesis, urea cycle, and teichoic acid biosynthesis were similar and increased progressively from P11 to P28 (Figure S2A). The progressive increase in Pathway-4984 (PWY-4984) related to urea cycle at P28, maybe related to absence of bacteria with ureolytic functions such as Bifidobacteria and over-abundance of anaerobes such as Lactobacilli (see Figure 1A).33PICRUSt2 analysis of the colon revealed that the non-oxidative arm of the pentose phosphate pathway (PPP) was significantly enriched at P28, which correlated with aromatic amino acid biosynthesis (Figure S2B). On the other hand, the pathway associated with de-nitrification (DENITRIFICATION-PWY) was significantly reduced at P28 thus indicating availability of nitrogen for amino acid and other nitrogen compound biosynthesis (Figure S2B).34,35Stratifying differences between ileal and colonic microbiota based on composition, diversity, and pathway enrichmentLimited human studies have clearly outlined the compositional variation in gut bacteria along the gastrointestinal (GI) tract with small intestine inhabiting fast-growing, primary fermenting bacteria that utilize simple carbohydrates in contrast to the colonic ecosystem, which is driven by the conversion of complex carbohydrates.36,37 A similar study detailing whether the microbiome changes along the GI tract influence developmental processes in the mouse gut, however, is lacking. We therefore investigated differences in the gut microbiota between the terminal ileum and the colon across postnatal days 11, 14, and 28, respectively. When assessing alpha diversity, we observed significant differences in both Chao1 and Shannon indices wherein, both indices posited a declining trend in the colon versus the terminal ileum at P11 (Figures 3A and 3B). At P14, similar trend was noted in the Chao1 index although this was not significant. On the other hand, Shannon index was significantly higher for colon at P14 (Figures 3A and 3B). At P28, we observed a reverse trend wherein, both indices were significantly lower for terminal ileum when compared to colon suggesting a shift in the bacterial landscape at these sites at later time point (Figures 3A and 3B) consistent with alpha diversity changes reported for terminal ileum and colon in Figures 1 and 2. When analyzing β-diversity to confirm compositional shifts at these sites, we observed both Bray-Curtis dissimilarity and Jensen-Shannon divergence to be significantly different between terminal ileum and colon (Figures 3C and 3D). Employing LEfSe analysis, we discovered significant differences in microbial species between terminal ileum and colon. At P11, p_Proteobacteria, g_Muribacter, and g_Pseudomonas were enriched in the terminal ileum compared to g_Lachnospiraceae_NK4A136_group, g_Enterococcus, and g_Anaerotruncus, all belonging to the phylum Firmicutes, in the colon (Figures 3E and 3F; LDA >4, p < 0.05). At P14, we observed p_Firmicutes and p_Bacteroidetes to be enriched in the terminal ileum and colon, respectively. This correlated with enrichment of g_Bacteroides, g_Rikenellaceae_RC9_gut_group, and g_Allistipes belonging to the Bacteroidetes phyla and enrichment of g_Lachnospiraceae_NK4A136_group in the Firmicutes phyla in the colon (Figures 3E and 3F). In the terminal ileum, g_Lactobacillus, g_Muribacter, and g_Pseudomonas were enriched (Figures 3E and 3F; LDA >4, p < 0.05). At p28, p_Bacteroidetes, p_Tenericutes, and p_Deferribacteres were significantly enriched in the colon while p_Firmicutes was the dominant phylum in the terminal ileum (Figures 3E and 3F). This correlated with enrichment of g_Allistipes, g_Bacteroides, and g_Prevotellaceae_UCG_001, g_Lachnospiraceae_NK4A136_group, and g_Roseburia belonging to the phylum Firmicutes along with others in the colon (Figures 3E and 3F) and enrichment of g_Lactobacillus in the ileum (Figures 3E and 3F; LDA >4, p < 0.05). It can be posited from these findings that differences in ileum and colon microbiota underlie developmental processes at these sites.Figure 3Comparing terminal ileum and colon bacterial profile across timeTerminal ileum (TI) and colon (Colon) contents collected from mouse pups at postnatal days 11, 14, and 28 were analyzed systematically for alpha (Chao1 and Shannon) and β-diversities (Bray-Curtis, Jensen-Shannon) indices as indicated. t test or PERMANOVA were used as the statistical methods to assess differences between groups. p values as indicated. n = 7–11 pups in each group.The results of metabolic pathways disclosed that Pathway-5183 (superpathway of aerobic toluene degradation) was highly active in the terminal ileum at P11 correlating with significant abundance of g_Pseudomonas (Figure S3), implicated in toluene degradation.38 At P14, Pathway-3781 [aerobic respiration I (cytochrome c)] and Pathway-7094 (fatty acid salvage) which could be due to predominance of Lactobacillus, were active (Figure S3) Intriguingly, none of these pathways were active in the colon (Figure S3). At P28, however, we noted significant activation of Pathway-7371 (1,4-dihydroxy-6-naphthoate biosynthesis II), Pathway-7374 (1,4-dihroxy-6-naphthoate biosynthesis I), and Pathway-Heme-Biosynthesis-II (heme biosynthesis II (aerobic), respectively (Figure S3) in the colon. Both PWY-7371 and PWY-7374 are active in most bacteria irrespective of whether they are gram positive or negative. These findings clearly illustrate the differences in activation of metabolic pathways in the terminal ileum and the colon consistent with presence of a different ecosystem in the TI and the colon.39Effect of formula feeding on colonic bacterial abundance, diversity, and metabolismThe gut microbiome of infants who receive formula milk differs from those that are fed breast milk.40 To investigate how formula milk feeding, a known risk factor for NEC, impacts microbiota composition in WT mice, we performed a systematic study that looked at changes in bacterial abundance, diversity and metabolism. Studies were performed in postnatal day 8 (P8) mice that were either dam-fed or given formula for three days. Formula feeding led to dramatic increase in p_Proteobacteria (70.2%) concomitant with significant decline in p_Firmicutes (24.2%) compared to 27.2% p_Proteobacteria and 71.9% p_Firmicutes in control (Figures 4A–4D). These changes correlated with increases in g_Escherichia-Shigella (63.7%) with decline in g_Lactobacillus (18.8%) in formula-fed mice compared to 68.1% of g_Lactobacillus and 25.7% of g_Escherichia-Shigella in control (Figures 4E and 4F). When assessing alpha diversity, we observed significant differences in both Chao1 and Shannon indices wherein both indices were significantly higher in formula-fed mice compared to control (Figures 4G and 4H). When analyzing β-diversity to confirm compositional shifts in the two groups, we observed Bray-Curtis dissimilarity to correlate with significant differences in the two groups (Figure 4I). Changes in Jensen-Shannon divergence, however, were not significant (Figure 4J). We next performed the PCA wherein the first three principal coordinate analysis were plotted against each other with regression planes and ellipsoids of concentration. As depicted in Figure 4K and A clear separation could be seen between the control and formula-fed group. Employing LEfSe analysis, we discovered significant differences in microbial species between control and formula-fed group. As is depicted in Figure 4L, p_Firmicutes dominated the control group while p_Proteobacteria and p_Bacteroidetes were abundant in the formula-fed group. At the genus level, g_Lactobacillus was abundant in the control group while several bacterial populations including g_Lachnospiraceae_NK4A136_group, g_Phyllobacterium, and intriguingly, g_Staphylococcus belonging to the phylum Firmicutes, were abundant in the formula-fed group (Figure 4M).Figure 4Effect of formula feeding on bacterial profile in wild-type mice(A and B) Mouse pups at postnatal day 8 were either dam-fed (Control, C) or received formula milk (F) for 3 days before euthanasia. Colon contents were collected and subjected to V3-V4 sequencing. Boxplots for bacterial abundance (A, B) are shown.(C–F) Pie charts representing the abundance of major phyla (C and D) and genus (E and F) in C and formula-fed (F) group.(G and H) Alpha diversity indices of indicated samples showing Chao 1 (G) and Shannon (H) indices, respectively (p values as indicated).(I and J) β-diversity visualization with non-metric multi-dimensional scaling (NMDS) based on Bray-Curtis distance (I) and Jensen-Shannon divergence metric (J; p values as indicated).(K) The principal coordinate (PC) analysis. The first three principal coordinates were plotted against each other with regression planes and ellipsoids of concentration to show a clear separation.(L and M) The linear discriminant analysis (LDA) value distribution histogram. Taxa meeting a linear discriminant analysis significant threshold >4 are shown.(N and O) Heatmaps showing KEGG top 30 features and MetaCyc pathways most active in the C or F group. Boxed areas represent enrichment of iron complex transport system permease protein in C (green) or F (blue) fed group. n = 7–11 pups in each group.To determine if formula feeding promoted functional changes in the P8 mice, we next performed PICRUSt2 analysis on the control and the formula-fed group. In the KEGG ortholog database, we screened 4,471 differential orthologs in the control vs. 6,555 differential orthologs in the formula-fed group (p < 0.05). Of the 30 differentially metabolic orthologs, the ABC.FEV.P; iron complex transport system permease protein was highly expressed in the formula-fed group compared to control. Iron acquisition, a critical physiological need for most bacteria, is integrated into a variety of proteins that function in processes including photosynthesis, respiration, and nitrogen metabolism.41 ABC transporters, made up of a substrate-binding protein (SBP), help facilitate transport of iron containing compounds (Fe2+, Fe3+, Fe3+-siderophore complex or heme) across the cytoplasmic membrane.42 The upregulation of this particular transporter in response to formula feeding therefore represents a step geared toward higher iron transport needed to accommodate the need for a diverse group of bacteria to survive and bloom. The results of metabolic pathways as inferred from the genomics data, demonstrated that in formula-fed group, Glycolysis III from Glucose was significantly upregulated compared to control group (Figure 4O). Yet, pyruvate, the end product of glycolysis, did not get fermented into acetate and lactate as the levels of these metabolites were significantly reduced in the formula group suggesting entry of pyruvate into the TCA cycle (Figure 4O). This is consistent with increases in p_Proteobacteria in the formula-fed group as a primarily aerobic luminal environment promotes dysbiosis.43Effect of SIGIRR genotype on gut microbiome development in the postnatal intestineTLR hypersensitivity in neonatal mice disrupts postnatal gut adaptation to bacterial colonization resulting in spontaneous inflammation.22,27 Our lab has previously shown that SIGIRR mutations might contribute to NEC vulnerability in premature infants by inducing aberrant intestinal TLR activation. Intrigued by studies in humans that noted a modest effect of host genetics on the gut microbiome, we queried whether SIGIRR genotypes alter the microbiota composition in the developing intestine.31 To study this, we temporally investigated the microbiota composition during postnatal gut adaptation at 11, 14, and 28 days of life in mice encoding the SIGIRR p.Y168X mutation identified in NEC that we previously generated.22 Profiling of bacteria in the terminal ileum at P11-P28 revealed three major phyla, p_Firmicutes, p_Bacteroidetes, and p_Proteobacteria (Figure 5A). Shown in Figure 5B are the top bacterial features (Firmicutes, Bacteroidetes, and Proteobacteria) based on their LDA scores and p values, which reflect significant changes in their actual abundance within each genotype group (WT, Het, Homo). Despite these shifts, we observed no significant differences in the abundance of these features between genotypes (Het/Homo vs. WT) across different time points, indicating that genotype does not significantly influence the microbial composition during postnatal development. The kinetics of phyla level changes in p_Bacteroidetes however, were significantly different between P11 to P28 with higher level recorded at P28 (LDA score: 5.72–5.92; Figures 5B and 5C).Figure 5Effect of genotypes on terminal ileum or colon bacterial profile across time(A) Terminal ileum (TI) or colon (Colon) contents from wild-type (WT) mice or mice heterozygous (het) or homozygous (homo) for Sigirr gene were collected and subjected to V3-V4 sequencing. Relative abundances are shown.(B) Boxplots showing relative abundance at postnatal days 11, 14, or 28 in the TI or Colon across genotypes. p values and LDA scores are as indicated. A Kruskal-Wallis rank-sum test was performed on each identified phylum using MicrobiomeAnalyst’s Kruskal test module.(C) Pie charts representing major phyla distribution in the TI or Colon across genotypes. n = 7–11 pups in each group.To systematically analyze changes in bacterial profile in the colons of either WT or mice heterozygous/homozygous for SIGIRR mutation, we collected the colon content at postnatal days 8, 11, 14, and 28 and subjected them to 16S rRNA gene sequencing. Profiling of bacteria in the colon at P8-P28 revealed three major phyla, p_Firmicutes, p_Bacteroidetes, and p_Proteobacteria (Figure 5A). The kinetics of p_Firmicutes changes were very similar to those recorded for TI (Figures 5B and 5C). The changes in p_Bacteroidetes at P14 in particular, were quite different compared to TI as the abundance increased significantly at this time point and at P28, paralleled those recorded in the colon (Figures 5B and 5C). Similar to TI, we did not observe differences in bacterial population either at the phyla or genus level in the colon based on the genotypes (Figure S4).Effect of SIGIRR genotype on formula milk-induced alterations in gut microbiomeWe and others have shown that formula feeding disrupts the gut microbiome in early life and is an important risk factor for NEC.44,45,46,47 Hypothesizing that presence of the SIGIRR mutant allele will exaggerate dysbiosis and gut injury induced by formula milk feeding, we evaluated alterations in microbial community composition and diversity in the colon in SIGIRR wild type and transgenic mice after formula feeding experiments as aforementioned. We found that differences in host genotype and gut microbiome programming increased dysbiosis after formula feeding (Figures 6A–6F). At the phylum level, while a relatively higher level of p_Firmicutes was seen in Het/Homo group compared to WT in the control with a positive LDA score, the changes were not significant. In response to formula feeding however, mice that carried the Sigirr mutation (either heterozygous or homozygous) displayed significant decrease in p_Firmicutes compared to WT with a negative LDA score indicating a lower abundance of p_Firmicutes in the formula group (Figure 6A). At the same time, an increase in p_Proteobacteria was recorded in the formula-fed group (Figure 6E). These changes correlated with genus level decrease in g_Lactobacillus with higher levels of g_Alphaproteobacteria (Figures 6B and 6F). Interestingly, formula feeding led to relatively higher levels of p_Bacteroidetes and g_Bacteroidia compared to WT (Figures 6C and 6D). Bacteroides species are normally commensals in the gut but can be opportunistic pathogens responsible for infections with significant morbidity and mortality.48Figure 6Effect of formula feeding on bacterial profile across genotypes(A–F) Sigirr wild-type (WT) or Het/Homo mouse pups at postnatal day 8 (P8) were either dam-fed or given formula milk for 3 days before euthanasia. Colon contents were collected and subjected to V3-V4 sequencing. Boxplots represent relative abundance at the phylum and genus level. Linear discriminant analysis effect size (LEfSe) was performed, and taxa were considered significant if they had p values < 0.05 and LDA values ≥2. n = 7–11 pups in each group.(G) Alpha diversity measurement between control and SIGIRR mutant (Het+Homo) mice after formula feeding. Chao1 and Shannon indices are shown.(H and I) D3 Interactive network at a max Bray distance of either 0.15 showing clear separation between the formula and the controls (H) or 0.2 showing dissimilarity between the formula and the controls, with formula WT more similar to the controls (I).(J) Principal coordinate (PC) analysis of the dam-fed or formula-fed mouse pups. First 3 principal coordinates were plotted in 3D space, on the day 8 formula colon experiment showing clear separation between the controls and formula, with Formula WT closer (more similar) to controls (all genotype).(K) Heatmap showing MetaCyc top 30 pathways in the C or F group. Boxed areas represent enrichment of glycolysis III from glucose or pyruvate fermentation to acetate and lactate II in C (green) or F (blue) fed group. n = 7–11 pups in each group.When assessing alpha diversity, we observed significant differences between control and formula feeding wherein, SIGIRR mutant (Het+Homo) mice receiving formula had higher Chao1 and Shannon indices in the order Homo>Het>WT (Figure 6G). This was further verified through regression analysis for genotypes (Table S1). To measure β-diversity, we employed sophisticated D3 Interactive network at a max Bray distance of 0.15. The output from the D3 interactive interface wherein nodes represent stool samples and edges represent relationships, different subnetworks attempt to avoid overlapping one another via a slow repulsion-like algorithm. As is depicted in Figure 6H, with a maximum distance of 0.15, we found clear separation between formula and controls. When less stringent cut-off was applied (0.2, Figure 6I), expanding the network to include moderately similar samples, the similarity relationship between control and formula occurred via WT mice, further lending to the hypothesis that Het/Homo and formula account for additive variation. These are connectivity insights, and the color coding informs on inter-mouse variability. During principal coordinates analysis, plotting the day 8 formula colon experiment in 3D space showed clear separation between the control and the formula with formula WT closer (more similar) to controls (all genotype) (Figure 6J). This was further validated through (1) regression analysis wherein, the SIGIRR mutation (het or homo) on the formula status was the most significantly associated with variation in the microbiome (p value: 1.01e-07; Table S2) and (2) machine learning exercise (random forest) (Table S3) demonstrating significance (0.003) and accuracy (0.6). Lastly, the amplicon sequence variant (ASV) most highly contributing toward these changes belonged to Proteobact_371 representing c_Gammaproteobacteria (Figure S5).Next, upon PICRUSt2 analysis on the control and the formula-fed group, we found that Glycolysis III from glucose was significantly upregulated in the formula-fed group compared to control group (across genotypes) (Figure 6K). Thus, while glycolysis increased in the formula group, the fermentation of pyruvate into acetate and lactate was minimal in agreement with findings described earlier (see Figure 4O). We posit from these findings that formula feeding intrinsically necessitates entry of the pyruvate into the TCA cycle irrespective of the genotypes to allow proliferation and bloom of p_Proteobacteria. Indeed, the transition from commensals to dysbiotic microbial community requires an environment which is predominantly aerobic to support the growth of pathogenic bacteria.43,49Effect of host genetics and formula feeding on intestinal injuryWe and others have shown that SIGIRR is developmentally regulated in the neonatal mouse and human intestine24,29 and the SIGIRR genetic variants are enriched in neonates with NEC. Noting the increased dysbiosis seen in SigirrMut mice after formula milk, we investigated whether formula milk-induced gut injury would be influenced by Sigirr mutant allele genotype. For formula feeding induced intestinal injury in mice, we used a standardized 4-point scale, to grade intestinal injury in the ileum based on histologic slides stained with hematoxylin and eosin.24,50 As is depicted in Figure 7A, evaluation of ileal histology revealed that intestinal injury after formula feeding showed a dose-dependent increase with presence of the Sigirr mutant allele (Figures 7A and 7B). We also noted minor increase in intestinal injury scores even at baseline in Sigirr transgenic mice homozygous for the stop mutation (Figures 7A and 7B). Next, we examined changes in the expression of inflammatory cytokines in the ileum after formula feeding (Figure 7C). RNA expression of Cxcl1, intercellular adhesion molecule 1 (Icam1) and Il1b cytokines showed a dose-dependent increase in expression after formula milk with the Sigirr mutant allele (Figure 7C). Increased inflammation and intestinal injury in Sigirr transgenic mice paralleled increased nitrotyrosine staining (Figure 7D), a marker of inflammatory oxidative stress. Finally, we performed the TUNEL assay to evaluate intestinal apoptosis, which showed increased apoptosis in Sigirr transgenic mice after formula feeding (Figures 7E and 7F). These data demonstrate that interactions between the Sigirr host genotype and formula milk induces worse intestinal injury in Sigirr transgenic mice.Figure 7Effect of formula feeding on intestinal injury in mouse pups(A) Representative histology of the terminal ileum from wild-type (WT) mice or mice heterozygous (het) or homozygous (homo) for Sigirr gene, either dam-fed or given formula. Scale bar, 100μm.(B) Histologic intestinal injury scores.(C) Relative gene expression of Cxcl1, Icam1, and IL-1β in controls vs. formula-fed pups.(D) Representative images showing Nitrotyrosine staining. Scale bar, 100 μm.(E) TUNEL staining of terminal ileum of control and formula-fed groups. Green indicates TUNEL positive cells; blue, DAPI cells. Scale bar, 120 μm.(F) Graph showing percentage of apoptotic cells in control vs. the formula-fed group.All data are presented as mean ± SD; ∗p < 0.05, ∗∗p < 0.01, by ANOVA. n = 7–11 pups in each group.Effect of Sigirr genotype on formula milk-induced alterations in gut microbiomeAs we noted more severe intestinal injury with formula milk in SigirrMut mice that were homozygous for SIGIRR mutant allele, we compared microbiota changes between the WT and SigirrMut mice at postnatal days 11, 14, and 28 (terminal ileum, TI) and P8, 11, 14, and 28 (colon). Following 2D PCA, we found that while the days (P11-P28) overlapped, P11 and P28 in the TI clustered on different axes in the WT when compared to SigirrMut mice indicating development differences in microbiome progression (Figures 8Ai and 8Aii). In the colon, while P8 and P28 separated, we observed differences in how the microbiome clustered in WT and SigirrMut mice further supporting the notion that subtle developmental cues can influence microbiome progression (Figures 8Bi and 8Bii). Since we have previously established that formula feeding promotes bacterial dysbiosis,44 we next evaluated the effect of formula feeding to SigirrMut mice. A PCA wherein the first three principal coordinate analyses were plotted against each other with regression planes and ellipsoids of concentration, we found a clear separation between the control and formula-fed group (Figure 8C). PICRUSt2 analysis revealed significant differences between the control and formula feeding group, irrespective of the genotype. In response to formula feeding however, we did see enrichment of KEGG orthologs K07483 (transposase), K07497 (putative transposase), and K07484 (transposase) in SigirrMut vs. the WT (Figure 8D). During pathway analysis, Glycolysis I and II from glucose- and fructose-6-phosphate, was higher in the formula-fed group compared to control group (across genotypes) (Figure 8E). Whether this promotes formula feeding-driven entry of the glycolytic end products into the TCA cycle as seen elsewhere (see Figure 4O) is under investigation. It can be gleaned from these findings that Sigirr homozygosity influences microbiota composition and function in the developing intestine.Figure 8Impact of Sigirr homozygous mutation (SigirrMut) on bacterial profile across timeTerminal ileum (TI) and colon (Colon) contents from WT or SigirrMut mice were collected at postnatal days 11, 14, and 28 (TI) or 8, 11, 14, and 28 (Colon) and subjected to V3-V4 sequencing.(A and B) 2D principal component (PC) analysis showing clear separation at P11 and P28 (TI; Ai, Aii) and P8 and P28 (Colon; Bi, Bii) that indicates microbiome progression.(C) A 3D principal component (PC) analysis showing a clear separation between the control and formula-fed group across genotypes. n = 7–11 pups in each group.(D and E) PICRUSt2 analysis showing top 30 features of KEGG (D) and MetaCyc pathways (E). Boxed areas represent differences in KEGG pathways K07483, K07497, and K-07484 belonging to transposase in WT and SigirrMut mice. Boxed areas are also shown for differences in MetaCyc pathways between WT and SigirrMut mice.

Temporal changes in the terminal ileum and colon microbiome during postnatal gut developmentTo determine changes in the small intestinal microbiota community structure and function with gut maturity, we collected terminal ileum (TI) contents from wild-type mouse pups at postnatal day 11 (P11), 14 (P14), and 28 (P28) and subjected them to V3-V4 sequencing. Earlier time points were less amenable to interrogation as luminal contents were scarce in the ileum. Figure S1A is a boxplot of the read counts at indicated days of life. We also tested mice of both sexes but did not disaggregate the data as the indices were not significant (Figure S1B). Analysis of the sequencing data revealed amplicon sequence variants that were plotted as pie charts to reveal dominant phyla. The core microbiota was composed of Firmicutes (71.3%, 77.2%, and 84.8%) at P11, P14, and P28 and Bacteroidetes (0.3%, 3.4%, and 14.2%) at P14 and P28, respectively (Figure 1A). Proteobacteria on the other hand, was observed at 28.3% at P11 and at 18.7% at P14 but the abundance diminished at P28 (Figure 1A). At the genus level, Lactobacillus (67.1%, 67.8%, and 72.4% dominated at P11, P14, and P28 while genus level changes in Muribacter (22.1% and 5.5%) at P11 and P14 and Escherichia_Shigella at P14 were less notable (Figure S1C). When we looked at alpha diversity (i.e., within-sample diversity) at different time points using the Chao1 and Shannon indices of richness and evenness, we discovered that both Chao1 and Shannon indices were higher at P11 with subsequent decline in diversity at P14 and P28, respectively. These changes were, however, not significant (Figures 1B and 1C). In contrast, β-diversity indices including Bray-Curtis and Jensen-Shannon were significantly different particularly at postnatal day 11 and 28, respectively (Figures 1D and 1E). This was captured by force-directed plotting of a distance threshold network with Fruchterman-Reingold network graph on counts of ileum specimens across days of life. Bray distance showed the progression in dissimilarity from day 11 to day 28 and co-located clusters suggesting possible strong correlations (Figure 1F). Specifically, the day 14 and day 28 samples looked very similar despite being from different mice; however, the day 11 samples showed a heterogeneity as the force-directed layout pushed them apart. Principal-component analysis (PCA) revealed that the P11 and P28 were significantly separated, with 63.3 and 6.3% of the variation explained by principal component 1 (PC1) and PC3, respectively, while PC2 revealed 18.0% variation at the two time points (Figure 1G). Employing linear discriminant analysis effect size (LEfSe) analysis to obtain dominant bacterial phyla and genera at indicated time points, we discovered that Proteobacteria at P11 and Firmicutes and Bacteroidetes at P28 were the dominant phyla while Actinobacteria also contributed to a linear discriminant analysis (LDA) score of >4 (Figure 1H). This correlated with the abundance of 9 key bacteria in the ileum including g_Muribacter, g_Candidatus_Arthromitus, g_Escherichia_Shigella, g_Streptococcus, g_Pseudomonas, and g_Staphylococcus among others (Figure 1I; LDA >4, p < 0.05).Figure 1Microbial profile of terminal ileum (TI) content across time(A) Pie charts representing the abundance of major phyla at postnatal day 11 (P11), P14, and P28.(B and C) Alpha diversity indices of indicated samples showing Chao 1 (B) and Shannon (C) indices, respectively (p values as indicated).(D and E) β-diversity visualization with non-metric multi-dimensional scaling (NMDS) based on Bray-Curtis (D) and distance and Jensen-Shannon divergence metric (E; p values as indicated).(F) Fruchterman-Reingold network graph on counts of ileum specimens across days of life. Bray distance showing the progression in dissimilarity from day 11 to day 28.(G) The principal-component (PC) analysis plots showing significant separation between P11 and P28 with 63.3% (PC1) and 6.3% (PC3) of the variation while PC2 revealed 18.0% variation at the two time points.(H and I) The linear discriminant analysis (LDA) value distribution histogram. Taxa meeting a linear discriminant analysis significant threshold >4 are shown. t test or PERMANOVA were used as the statistical methods to assess differences among the groups. n = 7–11 pups in each group.We next profiled the microbiome changes in the colon of WT mice on postnatal days 8, 11, 14, and 28, respectively. Figure S1A is a boxplot of the read counts at indicated days of life. The core microbiota was composed of taxa belonging to the phylum Firmicutes (66.8%, 91.2%, 57%, and 62.3%) and Bacteroidetes (0.1%, 2.5%, 30.8%, and 35.8%). Interestingly, Proteobacteria was detected at 32.5% at P8; however, the levels did not sustain at P11 and P14 (6.3% and 11.8%) and Proteobacteria were not detected at P28 (Figure 2A). At the genus level, Lactobacillus (63.5% and 80.2%) dominated over Streptococcus (2.3% and 5.4%) at P8 and P11 (Figure S1). At P14 and P28, while Lactobacillus levels were maintained at 40% and 40.9%, we did not detect Streptococcus at these time points (Figure S1D). To further delve into the Proteobacteria genera, we looked at Escherichia_Shigella and noted significant presence at P8 (Figure S1). At P14 however, the levels were reduced to 11.1% and Escherichia_Shigella was not detected at P28 (Figure S1C) consistent with phylum level decline at this time point (see Figure 2A). When we looked at alpha diversity, we discovered significant variation in alpha diversity in the colon at different time points wherein, Chao1 indices were higher at P8 with a drop at P11 followed by gradual increases at P14 and P28 (Figure 2B). Shannon index at P8, however, was significantly lower with gradual increase at P11 and P28, respectively (Figure 2C). The non-metric multidimensional scaling (NMDS) along with Permutational Multivariate Analysis of Variance (PERMANOVA) evaluation based on Bray-Curtis dissimilarity measures and Jensen-Shannon divergence discovered statistically significant differences in community composition, particularly at P8 and P28 time points by 16S rRNA profiles (Figures 2D and 2E). This was elegantly captured by force-directed plotting of a distance threshold network with Fruchterman-Reingold that demonstrated the graph on counts of colon specimens across days of life. Bray distance showed the progression in dissimilarity from day 8 to day 28, and the co-occurrence of bacterial networks as they group together with less variation in day 11 samples and more clustering across all days (Figure 2F).Figure 2Microbial profile of colon content across time(A) Pie charts representing the abundance of major phyla at postnatal day 11 (P11), P14, and P28.(B and C) Alpha diversity indices of indicated samples showing Chao 1 (B) and Shannon (C) indices, respectively (p values as indicated).(D and E) β-diversity visualization with Non-metric Multi-dimensional Scaling (NMDS) based on Bray-Curtis distance (D) and Jensen-Shannon divergence metric (E; p values as indicated).(F) Fruchterman-Reingold network graph on counts of ileum specimens across days of life. Bray distance showing the progression in dissimilarity from day 11 to day 28.(G) The principal-component (PC) analysis plots showing significant separation between P8 and P28 with 52.3 (PC1) and 7.9% (PC3) of the variation while PC2 revealed 32.7% variation at the two time points.(H and I) The linear discriminant analysis (LDA) value distribution histogram. Taxa meeting a linear discriminant analysis significant threshold >4 are shown. t test or PERMANOVA were used as the statistical methods to assess differences among the groups. n = 7–11 pups in each group.PCA revealed that the P8 and P28 were significantly separated, with 52.3 and 7.9% of the variation explained by PC1 and PC3, respectively, while PC2 revealed 32.7% variation at the two time points (Figure 2G). Employing LEfSe analysis to obtain dominant bacterial phyla and genera at indicated time points, we discovered that Proteobacteria at P8, Firmicutes at P11, and Bacteroidetes at P28 were the dominant phyla. This correlated with the abundance of 15 key bacteria in the colon including g_Escherichia_Shigella, g_Lactobacillus, g_Lachnospiraceae_NK4A136_group, g_Bacteroides, and g_Streptococcus among others (Figure 2I; LDA >4, p < 0.05).

Temporal changes in the terminal ileum and colon microbiome during postnatal gut development

To determine changes in the small intestinal microbiota community structure and function with gut maturity, we collected terminal ileum (TI) contents from wild-type mouse pups at postnatal day 11 (P11), 14 (P14), and 28 (P28) and subjected them to V3-V4 sequencing. Earlier time points were less amenable to interrogation as luminal contents were scarce in the ileum. Figure S1A is a boxplot of the read counts at indicated days of life. We also tested mice of both sexes but did not disaggregate the data as the indices were not significant (Figure S1B). Analysis of the sequencing data revealed amplicon sequence variants that were plotted as pie charts to reveal dominant phyla. The core microbiota was composed of Firmicutes (71.3%, 77.2%, and 84.8%) at P11, P14, and P28 and Bacteroidetes (0.3%, 3.4%, and 14.2%) at P14 and P28, respectively (Figure 1A). Proteobacteria on the other hand, was observed at 28.3% at P11 and at 18.7% at P14 but the abundance diminished at P28 (Figure 1A). At the genus level, Lactobacillus (67.1%, 67.8%, and 72.4% dominated at P11, P14, and P28 while genus level changes in Muribacter (22.1% and 5.5%) at P11 and P14 and Escherichia_Shigella at P14 were less notable (Figure S1C). When we looked at alpha diversity (i.e., within-sample diversity) at different time points using the Chao1 and Shannon indices of richness and evenness, we discovered that both Chao1 and Shannon indices were higher at P11 with subsequent decline in diversity at P14 and P28, respectively. These changes were, however, not significant (Figures 1B and 1C). In contrast, β-diversity indices including Bray-Curtis and Jensen-Shannon were significantly different particularly at postnatal day 11 and 28, respectively (Figures 1D and 1E). This was captured by force-directed plotting of a distance threshold network with Fruchterman-Reingold network graph on counts of ileum specimens across days of life. Bray distance showed the progression in dissimilarity from day 11 to day 28 and co-located clusters suggesting possible strong correlations (Figure 1F). Specifically, the day 14 and day 28 samples looked very similar despite being from different mice; however, the day 11 samples showed a heterogeneity as the force-directed layout pushed them apart. Principal-component analysis (PCA) revealed that the P11 and P28 were significantly separated, with 63.3 and 6.3% of the variation explained by principal component 1 (PC1) and PC3, respectively, while PC2 revealed 18.0% variation at the two time points (Figure 1G). Employing linear discriminant analysis effect size (LEfSe) analysis to obtain dominant bacterial phyla and genera at indicated time points, we discovered that Proteobacteria at P11 and Firmicutes and Bacteroidetes at P28 were the dominant phyla while Actinobacteria also contributed to a linear discriminant analysis (LDA) score of >4 (Figure 1H). This correlated with the abundance of 9 key bacteria in the ileum including g_Muribacter, g_Candidatus_Arthromitus, g_Escherichia_Shigella, g_Streptococcus, g_Pseudomonas, and g_Staphylococcus among others (Figure 1I; LDA >4, p < 0.05).Figure 1Microbial profile of terminal ileum (TI) content across time(A) Pie charts representing the abundance of major phyla at postnatal day 11 (P11), P14, and P28.(B and C) Alpha diversity indices of indicated samples showing Chao 1 (B) and Shannon (C) indices, respectively (p values as indicated).(D and E) β-diversity visualization with non-metric multi-dimensional scaling (NMDS) based on Bray-Curtis (D) and distance and Jensen-Shannon divergence metric (E; p values as indicated).(F) Fruchterman-Reingold network graph on counts of ileum specimens across days of life. Bray distance showing the progression in dissimilarity from day 11 to day 28.(G) The principal-component (PC) analysis plots showing significant separation between P11 and P28 with 63.3% (PC1) and 6.3% (PC3) of the variation while PC2 revealed 18.0% variation at the two time points.(H and I) The linear discriminant analysis (LDA) value distribution histogram. Taxa meeting a linear discriminant analysis significant threshold >4 are shown. t test or PERMANOVA were used as the statistical methods to assess differences among the groups. n = 7–11 pups in each group.

Microbial profile of terminal ileum (TI) content across time

(A) Pie charts representing the abundance of major phyla at postnatal day 11 (P11), P14, and P28.

(B and C) Alpha diversity indices of indicated samples showing Chao 1 (B) and Shannon (C) indices, respectively (p values as indicated).

(D and E) β-diversity visualization with non-metric multi-dimensional scaling (NMDS) based on Bray-Curtis (D) and distance and Jensen-Shannon divergence metric (E; p values as indicated).

(F) Fruchterman-Reingold network graph on counts of ileum specimens across days of life. Bray distance showing the progression in dissimilarity from day 11 to day 28.

(G) The principal-component (PC) analysis plots showing significant separation between P11 and P28 with 63.3% (PC1) and 6.3% (PC3) of the variation while PC2 revealed 18.0% variation at the two time points.

(H and I) The linear discriminant analysis (LDA) value distribution histogram. Taxa meeting a linear discriminant analysis significant threshold >4 are shown. t test or PERMANOVA were used as the statistical methods to assess differences among the groups. n = 7–11 pups in each group.

We next profiled the microbiome changes in the colon of WT mice on postnatal days 8, 11, 14, and 28, respectively. Figure S1A is a boxplot of the read counts at indicated days of life. The core microbiota was composed of taxa belonging to the phylum Firmicutes (66.8%, 91.2%, 57%, and 62.3%) and Bacteroidetes (0.1%, 2.5%, 30.8%, and 35.8%). Interestingly, Proteobacteria was detected at 32.5% at P8; however, the levels did not sustain at P11 and P14 (6.3% and 11.8%) and Proteobacteria were not detected at P28 (Figure 2A). At the genus level, Lactobacillus (63.5% and 80.2%) dominated over Streptococcus (2.3% and 5.4%) at P8 and P11 (Figure S1). At P14 and P28, while Lactobacillus levels were maintained at 40% and 40.9%, we did not detect Streptococcus at these time points (Figure S1D). To further delve into the Proteobacteria genera, we looked at Escherichia_Shigella and noted significant presence at P8 (Figure S1). At P14 however, the levels were reduced to 11.1% and Escherichia_Shigella was not detected at P28 (Figure S1C) consistent with phylum level decline at this time point (see Figure 2A). When we looked at alpha diversity, we discovered significant variation in alpha diversity in the colon at different time points wherein, Chao1 indices were higher at P8 with a drop at P11 followed by gradual increases at P14 and P28 (Figure 2B). Shannon index at P8, however, was significantly lower with gradual increase at P11 and P28, respectively (Figure 2C). The non-metric multidimensional scaling (NMDS) along with Permutational Multivariate Analysis of Variance (PERMANOVA) evaluation based on Bray-Curtis dissimilarity measures and Jensen-Shannon divergence discovered statistically significant differences in community composition, particularly at P8 and P28 time points by 16S rRNA profiles (Figures 2D and 2E). This was elegantly captured by force-directed plotting of a distance threshold network with Fruchterman-Reingold that demonstrated the graph on counts of colon specimens across days of life. Bray distance showed the progression in dissimilarity from day 8 to day 28, and the co-occurrence of bacterial networks as they group together with less variation in day 11 samples and more clustering across all days (Figure 2F).Figure 2Microbial profile of colon content across time(A) Pie charts representing the abundance of major phyla at postnatal day 11 (P11), P14, and P28.(B and C) Alpha diversity indices of indicated samples showing Chao 1 (B) and Shannon (C) indices, respectively (p values as indicated).(D and E) β-diversity visualization with Non-metric Multi-dimensional Scaling (NMDS) based on Bray-Curtis distance (D) and Jensen-Shannon divergence metric (E; p values as indicated).(F) Fruchterman-Reingold network graph on counts of ileum specimens across days of life. Bray distance showing the progression in dissimilarity from day 11 to day 28.(G) The principal-component (PC) analysis plots showing significant separation between P8 and P28 with 52.3 (PC1) and 7.9% (PC3) of the variation while PC2 revealed 32.7% variation at the two time points.(H and I) The linear discriminant analysis (LDA) value distribution histogram. Taxa meeting a linear discriminant analysis significant threshold >4 are shown. t test or PERMANOVA were used as the statistical methods to assess differences among the groups. n = 7–11 pups in each group.

Microbial profile of colon content across time

(A) Pie charts representing the abundance of major phyla at postnatal day 11 (P11), P14, and P28.

(B and C) Alpha diversity indices of indicated samples showing Chao 1 (B) and Shannon (C) indices, respectively (p values as indicated).

(D and E) β-diversity visualization with Non-metric Multi-dimensional Scaling (NMDS) based on Bray-Curtis distance (D) and Jensen-Shannon divergence metric (E; p values as indicated).

(F) Fruchterman-Reingold network graph on counts of ileum specimens across days of life. Bray distance showing the progression in dissimilarity from day 11 to day 28.

(G) The principal-component (PC) analysis plots showing significant separation between P8 and P28 with 52.3 (PC1) and 7.9% (PC3) of the variation while PC2 revealed 32.7% variation at the two time points.

(H and I) The linear discriminant analysis (LDA) value distribution histogram. Taxa meeting a linear discriminant analysis significant threshold >4 are shown. t test or PERMANOVA were used as the statistical methods to assess differences among the groups. n = 7–11 pups in each group.

PCA revealed that the P8 and P28 were significantly separated, with 52.3 and 7.9% of the variation explained by PC1 and PC3, respectively, while PC2 revealed 32.7% variation at the two time points (Figure 2G). Employing LEfSe analysis to obtain dominant bacterial phyla and genera at indicated time points, we discovered that Proteobacteria at P8, Firmicutes at P11, and Bacteroidetes at P28 were the dominant phyla. This correlated with the abundance of 15 key bacteria in the colon including g_Escherichia_Shigella, g_Lactobacillus, g_Lachnospiraceae_NK4A136_group, g_Bacteroides, and g_Streptococcus among others (Figure 2I; LDA >4, p < 0.05).

Predicted bacterial function of microbiome based on analysis of metabolic pathwaysThe biosynthesis and metabolism of gut bacteria influence host physiology.32 To elucidate the metabolic pathways activated at different days of life in the terminal ileum and the colon, we performed PICRUSt2 using MetaCyc database as a reference. The abundance of biosynthetic processes identified using STAMP, was higher when compared to other metabolic pathways. The major biological pathways in the terminal ileum included amino acid biosynthesis, urea cycle, purine and pyrimidine, and peptidoglycan biosynthesis, respectively (Figure S2A). The kinetics of the three pathways related to lysine biosynthesis, urea cycle, and teichoic acid biosynthesis were similar and increased progressively from P11 to P28 (Figure S2A). The progressive increase in Pathway-4984 (PWY-4984) related to urea cycle at P28, maybe related to absence of bacteria with ureolytic functions such as Bifidobacteria and over-abundance of anaerobes such as Lactobacilli (see Figure 1A).33PICRUSt2 analysis of the colon revealed that the non-oxidative arm of the pentose phosphate pathway (PPP) was significantly enriched at P28, which correlated with aromatic amino acid biosynthesis (Figure S2B). On the other hand, the pathway associated with de-nitrification (DENITRIFICATION-PWY) was significantly reduced at P28 thus indicating availability of nitrogen for amino acid and other nitrogen compound biosynthesis (Figure S2B).34,35

Predicted bacterial function of microbiome based on analysis of metabolic pathways

The biosynthesis and metabolism of gut bacteria influence host physiology.32 To elucidate the metabolic pathways activated at different days of life in the terminal ileum and the colon, we performed PICRUSt2 using MetaCyc database as a reference. The abundance of biosynthetic processes identified using STAMP, was higher when compared to other metabolic pathways. The major biological pathways in the terminal ileum included amino acid biosynthesis, urea cycle, purine and pyrimidine, and peptidoglycan biosynthesis, respectively (Figure S2A). The kinetics of the three pathways related to lysine biosynthesis, urea cycle, and teichoic acid biosynthesis were similar and increased progressively from P11 to P28 (Figure S2A). The progressive increase in Pathway-4984 (PWY-4984) related to urea cycle at P28, maybe related to absence of bacteria with ureolytic functions such as Bifidobacteria and over-abundance of anaerobes such as Lactobacilli (see Figure 1A).33

PICRUSt2 analysis of the colon revealed that the non-oxidative arm of the pentose phosphate pathway (PPP) was significantly enriched at P28, which correlated with aromatic amino acid biosynthesis (Figure S2B). On the other hand, the pathway associated with de-nitrification (DENITRIFICATION-PWY) was significantly reduced at P28 thus indicating availability of nitrogen for amino acid and other nitrogen compound biosynthesis (Figure S2B).34,35

Stratifying differences between ileal and colonic microbiota based on composition, diversity, and pathway enrichmentLimited human studies have clearly outlined the compositional variation in gut bacteria along the gastrointestinal (GI) tract with small intestine inhabiting fast-growing, primary fermenting bacteria that utilize simple carbohydrates in contrast to the colonic ecosystem, which is driven by the conversion of complex carbohydrates.36,37 A similar study detailing whether the microbiome changes along the GI tract influence developmental processes in the mouse gut, however, is lacking. We therefore investigated differences in the gut microbiota between the terminal ileum and the colon across postnatal days 11, 14, and 28, respectively. When assessing alpha diversity, we observed significant differences in both Chao1 and Shannon indices wherein, both indices posited a declining trend in the colon versus the terminal ileum at P11 (Figures 3A and 3B). At P14, similar trend was noted in the Chao1 index although this was not significant. On the other hand, Shannon index was significantly higher for colon at P14 (Figures 3A and 3B). At P28, we observed a reverse trend wherein, both indices were significantly lower for terminal ileum when compared to colon suggesting a shift in the bacterial landscape at these sites at later time point (Figures 3A and 3B) consistent with alpha diversity changes reported for terminal ileum and colon in Figures 1 and 2. When analyzing β-diversity to confirm compositional shifts at these sites, we observed both Bray-Curtis dissimilarity and Jensen-Shannon divergence to be significantly different between terminal ileum and colon (Figures 3C and 3D). Employing LEfSe analysis, we discovered significant differences in microbial species between terminal ileum and colon. At P11, p_Proteobacteria, g_Muribacter, and g_Pseudomonas were enriched in the terminal ileum compared to g_Lachnospiraceae_NK4A136_group, g_Enterococcus, and g_Anaerotruncus, all belonging to the phylum Firmicutes, in the colon (Figures 3E and 3F; LDA >4, p < 0.05). At P14, we observed p_Firmicutes and p_Bacteroidetes to be enriched in the terminal ileum and colon, respectively. This correlated with enrichment of g_Bacteroides, g_Rikenellaceae_RC9_gut_group, and g_Allistipes belonging to the Bacteroidetes phyla and enrichment of g_Lachnospiraceae_NK4A136_group in the Firmicutes phyla in the colon (Figures 3E and 3F). In the terminal ileum, g_Lactobacillus, g_Muribacter, and g_Pseudomonas were enriched (Figures 3E and 3F; LDA >4, p < 0.05). At p28, p_Bacteroidetes, p_Tenericutes, and p_Deferribacteres were significantly enriched in the colon while p_Firmicutes was the dominant phylum in the terminal ileum (Figures 3E and 3F). This correlated with enrichment of g_Allistipes, g_Bacteroides, and g_Prevotellaceae_UCG_001, g_Lachnospiraceae_NK4A136_group, and g_Roseburia belonging to the phylum Firmicutes along with others in the colon (Figures 3E and 3F) and enrichment of g_Lactobacillus in the ileum (Figures 3E and 3F; LDA >4, p < 0.05). It can be posited from these findings that differences in ileum and colon microbiota underlie developmental processes at these sites.Figure 3Comparing terminal ileum and colon bacterial profile across timeTerminal ileum (TI) and colon (Colon) contents collected from mouse pups at postnatal days 11, 14, and 28 were analyzed systematically for alpha (Chao1 and Shannon) and β-diversities (Bray-Curtis, Jensen-Shannon) indices as indicated. t test or PERMANOVA were used as the statistical methods to assess differences between groups. p values as indicated. n = 7–11 pups in each group.The results of metabolic pathways disclosed that Pathway-5183 (superpathway of aerobic toluene degradation) was highly active in the terminal ileum at P11 correlating with significant abundance of g_Pseudomonas (Figure S3), implicated in toluene degradation.38 At P14, Pathway-3781 [aerobic respiration I (cytochrome c)] and Pathway-7094 (fatty acid salvage) which could be due to predominance of Lactobacillus, were active (Figure S3) Intriguingly, none of these pathways were active in the colon (Figure S3). At P28, however, we noted significant activation of Pathway-7371 (1,4-dihydroxy-6-naphthoate biosynthesis II), Pathway-7374 (1,4-dihroxy-6-naphthoate biosynthesis I), and Pathway-Heme-Biosynthesis-II (heme biosynthesis II (aerobic), respectively (Figure S3) in the colon. Both PWY-7371 and PWY-7374 are active in most bacteria irrespective of whether they are gram positive or negative. These findings clearly illustrate the differences in activation of metabolic pathways in the terminal ileum and the colon consistent with presence of a different ecosystem in the TI and the colon.39

Stratifying differences between ileal and colonic microbiota based on composition, diversity, and pathway enrichment

Limited human studies have clearly outlined the compositional variation in gut bacteria along the gastrointestinal (GI) tract with small intestine inhabiting fast-growing, primary fermenting bacteria that utilize simple carbohydrates in contrast to the colonic ecosystem, which is driven by the conversion of complex carbohydrates.36,37 A similar study detailing whether the microbiome changes along the GI tract influence developmental processes in the mouse gut, however, is lacking. We therefore investigated differences in the gut microbiota between the terminal ileum and the colon across postnatal days 11, 14, and 28, respectively. When assessing alpha diversity, we observed significant differences in both Chao1 and Shannon indices wherein, both indices posited a declining trend in the colon versus the terminal ileum at P11 (Figures 3A and 3B). At P14, similar trend was noted in the Chao1 index although this was not significant. On the other hand, Shannon index was significantly higher for colon at P14 (Figures 3A and 3B). At P28, we observed a reverse trend wherein, both indices were significantly lower for terminal ileum when compared to colon suggesting a shift in the bacterial landscape at these sites at later time point (Figures 3A and 3B) consistent with alpha diversity changes reported for terminal ileum and colon in Figures 1 and 2. When analyzing β-diversity to confirm compositional shifts at these sites, we observed both Bray-Curtis dissimilarity and Jensen-Shannon divergence to be significantly different between terminal ileum and colon (Figures 3C and 3D). Employing LEfSe analysis, we discovered significant differences in microbial species between terminal ileum and colon. At P11, p_Proteobacteria, g_Muribacter, and g_Pseudomonas were enriched in the terminal ileum compared to g_Lachnospiraceae_NK4A136_group, g_Enterococcus, and g_Anaerotruncus, all belonging to the phylum Firmicutes, in the colon (Figures 3E and 3F; LDA >4, p < 0.05). At P14, we observed p_Firmicutes and p_Bacteroidetes to be enriched in the terminal ileum and colon, respectively. This correlated with enrichment of g_Bacteroides, g_Rikenellaceae_RC9_gut_group, and g_Allistipes belonging to the Bacteroidetes phyla and enrichment of g_Lachnospiraceae_NK4A136_group in the Firmicutes phyla in the colon (Figures 3E and 3F). In the terminal ileum, g_Lactobacillus, g_Muribacter, and g_Pseudomonas were enriched (Figures 3E and 3F; LDA >4, p < 0.05). At p28, p_Bacteroidetes, p_Tenericutes, and p_Deferribacteres were significantly enriched in the colon while p_Firmicutes was the dominant phylum in the terminal ileum (Figures 3E and 3F). This correlated with enrichment of g_Allistipes, g_Bacteroides, and g_Prevotellaceae_UCG_001, g_Lachnospiraceae_NK4A136_group, and g_Roseburia belonging to the phylum Firmicutes along with others in the colon (Figures 3E and 3F) and enrichment of g_Lactobacillus in the ileum (Figures 3E and 3F; LDA >4, p < 0.05). It can be posited from these findings that differences in ileum and colon microbiota underlie developmental processes at these sites.Figure 3Comparing terminal ileum and colon bacterial profile across timeTerminal ileum (TI) and colon (Colon) contents collected from mouse pups at postnatal days 11, 14, and 28 were analyzed systematically for alpha (Chao1 and Shannon) and β-diversities (Bray-Curtis, Jensen-Shannon) indices as indicated. t test or PERMANOVA were used as the statistical methods to assess differences between groups. p values as indicated. n = 7–11 pups in each group.

Comparing terminal ileum and colon bacterial profile across time

Terminal ileum (TI) and colon (Colon) contents collected from mouse pups at postnatal days 11, 14, and 28 were analyzed systematically for alpha (Chao1 and Shannon) and β-diversities (Bray-Curtis, Jensen-Shannon) indices as indicated. t test or PERMANOVA were used as the statistical methods to assess differences between groups. p values as indicated. n = 7–11 pups in each group.

The results of metabolic pathways disclosed that Pathway-5183 (superpathway of aerobic toluene degradation) was highly active in the terminal ileum at P11 correlating with significant abundance of g_Pseudomonas (Figure S3), implicated in toluene degradation.38 At P14, Pathway-3781 [aerobic respiration I (cytochrome c)] and Pathway-7094 (fatty acid salvage) which could be due to predominance of Lactobacillus, were active (Figure S3) Intriguingly, none of these pathways were active in the colon (Figure S3). At P28, however, we noted significant activation of Pathway-7371 (1,4-dihydroxy-6-naphthoate biosynthesis II), Pathway-7374 (1,4-dihroxy-6-naphthoate biosynthesis I), and Pathway-Heme-Biosynthesis-II (heme biosynthesis II (aerobic), respectively (Figure S3) in the colon. Both PWY-7371 and PWY-7374 are active in most bacteria irrespective of whether they are gram positive or negative. These findings clearly illustrate the differences in activation of metabolic pathways in the terminal ileum and the colon consistent with presence of a different ecosystem in the TI and the colon.39

Effect of formula feeding on colonic bacterial abundance, diversity, and metabolismThe gut microbiome of infants who receive formula milk differs from those that are fed breast milk.40 To investigate how formula milk feeding, a known risk factor for NEC, impacts microbiota composition in WT mice, we performed a systematic study that looked at changes in bacterial abundance, diversity and metabolism. Studies were performed in postnatal day 8 (P8) mice that were either dam-fed or given formula for three days. Formula feeding led to dramatic increase in p_Proteobacteria (70.2%) concomitant with significant decline in p_Firmicutes (24.2%) compared to 27.2% p_Proteobacteria and 71.9% p_Firmicutes in control (Figures 4A–4D). These changes correlated with increases in g_Escherichia-Shigella (63.7%) with decline in g_Lactobacillus (18.8%) in formula-fed mice compared to 68.1% of g_Lactobacillus and 25.7% of g_Escherichia-Shigella in control (Figures 4E and 4F). When assessing alpha diversity, we observed significant differences in both Chao1 and Shannon indices wherein both indices were significantly higher in formula-fed mice compared to control (Figures 4G and 4H). When analyzing β-diversity to confirm compositional shifts in the two groups, we observed Bray-Curtis dissimilarity to correlate with significant differences in the two groups (Figure 4I). Changes in Jensen-Shannon divergence, however, were not significant (Figure 4J). We next performed the PCA wherein the first three principal coordinate analysis were plotted against each other with regression planes and ellipsoids of concentration. As depicted in Figure 4K and A clear separation could be seen between the control and formula-fed group. Employing LEfSe analysis, we discovered significant differences in microbial species between control and formula-fed group. As is depicted in Figure 4L, p_Firmicutes dominated the control group while p_Proteobacteria and p_Bacteroidetes were abundant in the formula-fed group. At the genus level, g_Lactobacillus was abundant in the control group while several bacterial populations including g_Lachnospiraceae_NK4A136_group, g_Phyllobacterium, and intriguingly, g_Staphylococcus belonging to the phylum Firmicutes, were abundant in the formula-fed group (Figure 4M).Figure 4Effect of formula feeding on bacterial profile in wild-type mice(A and B) Mouse pups at postnatal day 8 were either dam-fed (Control, C) or received formula milk (F) for 3 days before euthanasia. Colon contents were collected and subjected to V3-V4 sequencing. Boxplots for bacterial abundance (A, B) are shown.(C–F) Pie charts representing the abundance of major phyla (C and D) and genus (E and F) in C and formula-fed (F) group.(G and H) Alpha diversity indices of indicated samples showing Chao 1 (G) and Shannon (H) indices, respectively (p values as indicated).(I and J) β-diversity visualization with non-metric multi-dimensional scaling (NMDS) based on Bray-Curtis distance (I) and Jensen-Shannon divergence metric (J; p values as indicated).(K) The principal coordinate (PC) analysis. The first three principal coordinates were plotted against each other with regression planes and ellipsoids of concentration to show a clear separation.(L and M) The linear discriminant analysis (LDA) value distribution histogram. Taxa meeting a linear discriminant analysis significant threshold >4 are shown.(N and O) Heatmaps showing KEGG top 30 features and MetaCyc pathways most active in the C or F group. Boxed areas represent enrichment of iron complex transport system permease protein in C (green) or F (blue) fed group. n = 7–11 pups in each group.To determine if formula feeding promoted functional changes in the P8 mice, we next performed PICRUSt2 analysis on the control and the formula-fed group. In the KEGG ortholog database, we screened 4,471 differential orthologs in the control vs. 6,555 differential orthologs in the formula-fed group (p < 0.05). Of the 30 differentially metabolic orthologs, the ABC.FEV.P; iron complex transport system permease protein was highly expressed in the formula-fed group compared to control. Iron acquisition, a critical physiological need for most bacteria, is integrated into a variety of proteins that function in processes including photosynthesis, respiration, and nitrogen metabolism.41 ABC transporters, made up of a substrate-binding protein (SBP), help facilitate transport of iron containing compounds (Fe2+, Fe3+, Fe3+-siderophore complex or heme) across the cytoplasmic membrane.42 The upregulation of this particular transporter in response to formula feeding therefore represents a step geared toward higher iron transport needed to accommodate the need for a diverse group of bacteria to survive and bloom. The results of metabolic pathways as inferred from the genomics data, demonstrated that in formula-fed group, Glycolysis III from Glucose was significantly upregulated compared to control group (Figure 4O). Yet, pyruvate, the end product of glycolysis, did not get fermented into acetate and lactate as the levels of these metabolites were significantly reduced in the formula group suggesting entry of pyruvate into the TCA cycle (Figure 4O). This is consistent with increases in p_Proteobacteria in the formula-fed group as a primarily aerobic luminal environment promotes dysbiosis.43

Effect of formula feeding on colonic bacterial abundance, diversity, and metabolism

The gut microbiome of infants who receive formula milk differs from those that are fed breast milk.40 To investigate how formula milk feeding, a known risk factor for NEC, impacts microbiota composition in WT mice, we performed a systematic study that looked at changes in bacterial abundance, diversity and metabolism. Studies were performed in postnatal day 8 (P8) mice that were either dam-fed or given formula for three days. Formula feeding led to dramatic increase in p_Proteobacteria (70.2%) concomitant with significant decline in p_Firmicutes (24.2%) compared to 27.2% p_Proteobacteria and 71.9% p_Firmicutes in control (Figures 4A–4D). These changes correlated with increases in g_Escherichia-Shigella (63.7%) with decline in g_Lactobacillus (18.8%) in formula-fed mice compared to 68.1% of g_Lactobacillus and 25.7% of g_Escherichia-Shigella in control (Figures 4E and 4F). When assessing alpha diversity, we observed significant differences in both Chao1 and Shannon indices wherein both indices were significantly higher in formula-fed mice compared to control (Figures 4G and 4H). When analyzing β-diversity to confirm compositional shifts in the two groups, we observed Bray-Curtis dissimilarity to correlate with significant differences in the two groups (Figure 4I). Changes in Jensen-Shannon divergence, however, were not significant (Figure 4J). We next performed the PCA wherein the first three principal coordinate analysis were plotted against each other with regression planes and ellipsoids of concentration. As depicted in Figure 4K and A clear separation could be seen between the control and formula-fed group. Employing LEfSe analysis, we discovered significant differences in microbial species between control and formula-fed group. As is depicted in Figure 4L, p_Firmicutes dominated the control group while p_Proteobacteria and p_Bacteroidetes were abundant in the formula-fed group. At the genus level, g_Lactobacillus was abundant in the control group while several bacterial populations including g_Lachnospiraceae_NK4A136_group, g_Phyllobacterium, and intriguingly, g_Staphylococcus belonging to the phylum Firmicutes, were abundant in the formula-fed group (Figure 4M).Figure 4Effect of formula feeding on bacterial profile in wild-type mice(A and B) Mouse pups at postnatal day 8 were either dam-fed (Control, C) or received formula milk (F) for 3 days before euthanasia. Colon contents were collected and subjected to V3-V4 sequencing. Boxplots for bacterial abundance (A, B) are shown.(C–F) Pie charts representing the abundance of major phyla (C and D) and genus (E and F) in C and formula-fed (F) group.(G and H) Alpha diversity indices of indicated samples showing Chao 1 (G) and Shannon (H) indices, respectively (p values as indicated).(I and J) β-diversity visualization with non-metric multi-dimensional scaling (NMDS) based on Bray-Curtis distance (I) and Jensen-Shannon divergence metric (J; p values as indicated).(K) The principal coordinate (PC) analysis. The first three principal coordinates were plotted against each other with regression planes and ellipsoids of concentration to show a clear separation.(L and M) The linear discriminant analysis (LDA) value distribution histogram. Taxa meeting a linear discriminant analysis significant threshold >4 are shown.(N and O) Heatmaps showing KEGG top 30 features and MetaCyc pathways most active in the C or F group. Boxed areas represent enrichment of iron complex transport system permease protein in C (green) or F (blue) fed group. n = 7–11 pups in each group.

Effect of formula feeding on bacterial profile in wild-type mice

(A and B) Mouse pups at postnatal day 8 were either dam-fed (Control, C) or received formula milk (F) for 3 days before euthanasia. Colon contents were collected and subjected to V3-V4 sequencing. Boxplots for bacterial abundance (A, B) are shown.

(C–F) Pie charts representing the abundance of major phyla (C and D) and genus (E and F) in C and formula-fed (F) group.

(G and H) Alpha diversity indices of indicated samples showing Chao 1 (G) and Shannon (H) indices, respectively (p values as indicated).

(I and J) β-diversity visualization with non-metric multi-dimensional scaling (NMDS) based on Bray-Curtis distance (I) and Jensen-Shannon divergence metric (J; p values as indicated).

(K) The principal coordinate (PC) analysis. The first three principal coordinates were plotted against each other with regression planes and ellipsoids of concentration to show a clear separation.

(L and M) The linear discriminant analysis (LDA) value distribution histogram. Taxa meeting a linear discriminant analysis significant threshold >4 are shown.

(N and O) Heatmaps showing KEGG top 30 features and MetaCyc pathways most active in the C or F group. Boxed areas represent enrichment of iron complex transport system permease protein in C (green) or F (blue) fed group. n = 7–11 pups in each group.

To determine if formula feeding promoted functional changes in the P8 mice, we next performed PICRUSt2 analysis on the control and the formula-fed group. In the KEGG ortholog database, we screened 4,471 differential orthologs in the control vs. 6,555 differential orthologs in the formula-fed group (p < 0.05). Of the 30 differentially metabolic orthologs, the ABC.FEV.P; iron complex transport system permease protein was highly expressed in the formula-fed group compared to control. Iron acquisition, a critical physiological need for most bacteria, is integrated into a variety of proteins that function in processes including photosynthesis, respiration, and nitrogen metabolism.41 ABC transporters, made up of a substrate-binding protein (SBP), help facilitate transport of iron containing compounds (Fe2+, Fe3+, Fe3+-siderophore complex or heme) across the cytoplasmic membrane.42 The upregulation of this particular transporter in response to formula feeding therefore represents a step geared toward higher iron transport needed to accommodate the need for a diverse group of bacteria to survive and bloom. The results of metabolic pathways as inferred from the genomics data, demonstrated that in formula-fed group, Glycolysis III from Glucose was significantly upregulated compared to control group (Figure 4O). Yet, pyruvate, the end product of glycolysis, did not get fermented into acetate and lactate as the levels of these metabolites were significantly reduced in the formula group suggesting entry of pyruvate into the TCA cycle (Figure 4O). This is consistent with increases in p_Proteobacteria in the formula-fed group as a primarily aerobic luminal environment promotes dysbiosis.43

Effect of SIGIRR genotype on gut microbiome development in the postnatal intestineTLR hypersensitivity in neonatal mice disrupts postnatal gut adaptation to bacterial colonization resulting in spontaneous inflammation.22,27 Our lab has previously shown that SIGIRR mutations might contribute to NEC vulnerability in premature infants by inducing aberrant intestinal TLR activation. Intrigued by studies in humans that noted a modest effect of host genetics on the gut microbiome, we queried whether SIGIRR genotypes alter the microbiota composition in the developing intestine.31 To study this, we temporally investigated the microbiota composition during postnatal gut adaptation at 11, 14, and 28 days of life in mice encoding the SIGIRR p.Y168X mutation identified in NEC that we previously generated.22 Profiling of bacteria in the terminal ileum at P11-P28 revealed three major phyla, p_Firmicutes, p_Bacteroidetes, and p_Proteobacteria (Figure 5A). Shown in Figure 5B are the top bacterial features (Firmicutes, Bacteroidetes, and Proteobacteria) based on their LDA scores and p values, which reflect significant changes in their actual abundance within each genotype group (WT, Het, Homo). Despite these shifts, we observed no significant differences in the abundance of these features between genotypes (Het/Homo vs. WT) across different time points, indicating that genotype does not significantly influence the microbial composition during postnatal development. The kinetics of phyla level changes in p_Bacteroidetes however, were significantly different between P11 to P28 with higher level recorded at P28 (LDA score: 5.72–5.92; Figures 5B and 5C).Figure 5Effect of genotypes on terminal ileum or colon bacterial profile across time(A) Terminal ileum (TI) or colon (Colon) contents from wild-type (WT) mice or mice heterozygous (het) or homozygous (homo) for Sigirr gene were collected and subjected to V3-V4 sequencing. Relative abundances are shown.(B) Boxplots showing relative abundance at postnatal days 11, 14, or 28 in the TI or Colon across genotypes. p values and LDA scores are as indicated. A Kruskal-Wallis rank-sum test was performed on each identified phylum using MicrobiomeAnalyst’s Kruskal test module.(C) Pie charts representing major phyla distribution in the TI or Colon across genotypes. n = 7–11 pups in each group.To systematically analyze changes in bacterial profile in the colons of either WT or mice heterozygous/homozygous for SIGIRR mutation, we collected the colon content at postnatal days 8, 11, 14, and 28 and subjected them to 16S rRNA gene sequencing. Profiling of bacteria in the colon at P8-P28 revealed three major phyla, p_Firmicutes, p_Bacteroidetes, and p_Proteobacteria (Figure 5A). The kinetics of p_Firmicutes changes were very similar to those recorded for TI (Figures 5B and 5C). The changes in p_Bacteroidetes at P14 in particular, were quite different compared to TI as the abundance increased significantly at this time point and at P28, paralleled those recorded in the colon (Figures 5B and 5C). Similar to TI, we did not observe differences in bacterial population either at the phyla or genus level in the colon based on the genotypes (Figure S4).

Effect of SIGIRR genotype on gut microbiome development in the postnatal intestine

TLR hypersensitivity in neonatal mice disrupts postnatal gut adaptation to bacterial colonization resulting in spontaneous inflammation.22,27 Our lab has previously shown that SIGIRR mutations might contribute to NEC vulnerability in premature infants by inducing aberrant intestinal TLR activation. Intrigued by studies in humans that noted a modest effect of host genetics on the gut microbiome, we queried whether SIGIRR genotypes alter the microbiota composition in the developing intestine.31 To study this, we temporally investigated the microbiota composition during postnatal gut adaptation at 11, 14, and 28 days of life in mice encoding the SIGIRR p.Y168X mutation identified in NEC that we previously generated.22 Profiling of bacteria in the terminal ileum at P11-P28 revealed three major phyla, p_Firmicutes, p_Bacteroidetes, and p_Proteobacteria (Figure 5A). Shown in Figure 5B are the top bacterial features (Firmicutes, Bacteroidetes, and Proteobacteria) based on their LDA scores and p values, which reflect significant changes in their actual abundance within each genotype group (WT, Het, Homo). Despite these shifts, we observed no significant differences in the abundance of these features between genotypes (Het/Homo vs. WT) across different time points, indicating that genotype does not significantly influence the microbial composition during postnatal development. The kinetics of phyla level changes in p_Bacteroidetes however, were significantly different between P11 to P28 with higher level recorded at P28 (LDA score: 5.72–5.92; Figures 5B and 5C).Figure 5Effect of genotypes on terminal ileum or colon bacterial profile across time(A) Terminal ileum (TI) or colon (Colon) contents from wild-type (WT) mice or mice heterozygous (het) or homozygous (homo) for Sigirr gene were collected and subjected to V3-V4 sequencing. Relative abundances are shown.(B) Boxplots showing relative abundance at postnatal days 11, 14, or 28 in the TI or Colon across genotypes. p values and LDA scores are as indicated. A Kruskal-Wallis rank-sum test was performed on each identified phylum using MicrobiomeAnalyst’s Kruskal test module.(C) Pie charts representing major phyla distribution in the TI or Colon across genotypes. n = 7–11 pups in each group.

Effect of genotypes on terminal ileum or colon bacterial profile across time

(A) Terminal ileum (TI) or colon (Colon) contents from wild-type (WT) mice or mice heterozygous (het) or homozygous (homo) for Sigirr gene were collected and subjected to V3-V4 sequencing. Relative abundances are shown.

(B) Boxplots showing relative abundance at postnatal days 11, 14, or 28 in the TI or Colon across genotypes. p values and LDA scores are as indicated. A Kruskal-Wallis rank-sum test was performed on each identified phylum using MicrobiomeAnalyst’s Kruskal test module.

(C) Pie charts representing major phyla distribution in the TI or Colon across genotypes. n = 7–11 pups in each group.

To systematically analyze changes in bacterial profile in the colons of either WT or mice heterozygous/homozygous for SIGIRR mutation, we collected the colon content at postnatal days 8, 11, 14, and 28 and subjected them to 16S rRNA gene sequencing. Profiling of bacteria in the colon at P8-P28 revealed three major phyla, p_Firmicutes, p_Bacteroidetes, and p_Proteobacteria (Figure 5A). The kinetics of p_Firmicutes changes were very similar to those recorded for TI (Figures 5B and 5C). The changes in p_Bacteroidetes at P14 in particular, were quite different compared to TI as the abundance increased significantly at this time point and at P28, paralleled those recorded in the colon (Figures 5B and 5C). Similar to TI, we did not observe differences in bacterial population either at the phyla or genus level in the colon based on the genotypes (Figure S4).

Effect of SIGIRR genotype on formula milk-induced alterations in gut microbiomeWe and others have shown that formula feeding disrupts the gut microbiome in early life and is an important risk factor for NEC.44,45,46,47 Hypothesizing that presence of the SIGIRR mutant allele will exaggerate dysbiosis and gut injury induced by formula milk feeding, we evaluated alterations in microbial community composition and diversity in the colon in SIGIRR wild type and transgenic mice after formula feeding experiments as aforementioned. We found that differences in host genotype and gut microbiome programming increased dysbiosis after formula feeding (Figures 6A–6F). At the phylum level, while a relatively higher level of p_Firmicutes was seen in Het/Homo group compared to WT in the control with a positive LDA score, the changes were not significant. In response to formula feeding however, mice that carried the Sigirr mutation (either heterozygous or homozygous) displayed significant decrease in p_Firmicutes compared to WT with a negative LDA score indicating a lower abundance of p_Firmicutes in the formula group (Figure 6A). At the same time, an increase in p_Proteobacteria was recorded in the formula-fed group (Figure 6E). These changes correlated with genus level decrease in g_Lactobacillus with higher levels of g_Alphaproteobacteria (Figures 6B and 6F). Interestingly, formula feeding led to relatively higher levels of p_Bacteroidetes and g_Bacteroidia compared to WT (Figures 6C and 6D). Bacteroides species are normally commensals in the gut but can be opportunistic pathogens responsible for infections with significant morbidity and mortality.48Figure 6Effect of formula feeding on bacterial profile across genotypes(A–F) Sigirr wild-type (WT) or Het/Homo mouse pups at postnatal day 8 (P8) were either dam-fed or given formula milk for 3 days before euthanasia. Colon contents were collected and subjected to V3-V4 sequencing. Boxplots represent relative abundance at the phylum and genus level. Linear discriminant analysis effect size (LEfSe) was performed, and taxa were considered significant if they had p values < 0.05 and LDA values ≥2. n = 7–11 pups in each group.(G) Alpha diversity measurement between control and SIGIRR mutant (Het+Homo) mice after formula feeding. Chao1 and Shannon indices are shown.(H and I) D3 Interactive network at a max Bray distance of either 0.15 showing clear separation between the formula and the controls (H) or 0.2 showing dissimilarity between the formula and the controls, with formula WT more similar to the controls (I).(J) Principal coordinate (PC) analysis of the dam-fed or formula-fed mouse pups. First 3 principal coordinates were plotted in 3D space, on the day 8 formula colon experiment showing clear separation between the controls and formula, with Formula WT closer (more similar) to controls (all genotype).(K) Heatmap showing MetaCyc top 30 pathways in the C or F group. Boxed areas represent enrichment of glycolysis III from glucose or pyruvate fermentation to acetate and lactate II in C (green) or F (blue) fed group. n = 7–11 pups in each group.When assessing alpha diversity, we observed significant differences between control and formula feeding wherein, SIGIRR mutant (Het+Homo) mice receiving formula had higher Chao1 and Shannon indices in the order Homo>Het>WT (Figure 6G). This was further verified through regression analysis for genotypes (Table S1). To measure β-diversity, we employed sophisticated D3 Interactive network at a max Bray distance of 0.15. The output from the D3 interactive interface wherein nodes represent stool samples and edges represent relationships, different subnetworks attempt to avoid overlapping one another via a slow repulsion-like algorithm. As is depicted in Figure 6H, with a maximum distance of 0.15, we found clear separation between formula and controls. When less stringent cut-off was applied (0.2, Figure 6I), expanding the network to include moderately similar samples, the similarity relationship between control and formula occurred via WT mice, further lending to the hypothesis that Het/Homo and formula account for additive variation. These are connectivity insights, and the color coding informs on inter-mouse variability. During principal coordinates analysis, plotting the day 8 formula colon experiment in 3D space showed clear separation between the control and the formula with formula WT closer (more similar) to controls (all genotype) (Figure 6J). This was further validated through (1) regression analysis wherein, the SIGIRR mutation (het or homo) on the formula status was the most significantly associated with variation in the microbiome (p value: 1.01e-07; Table S2) and (2) machine learning exercise (random forest) (Table S3) demonstrating significance (0.003) and accuracy (0.6). Lastly, the amplicon sequence variant (ASV) most highly contributing toward these changes belonged to Proteobact_371 representing c_Gammaproteobacteria (Figure S5).Next, upon PICRUSt2 analysis on the control and the formula-fed group, we found that Glycolysis III from glucose was significantly upregulated in the formula-fed group compared to control group (across genotypes) (Figure 6K). Thus, while glycolysis increased in the formula group, the fermentation of pyruvate into acetate and lactate was minimal in agreement with findings described earlier (see Figure 4O). We posit from these findings that formula feeding intrinsically necessitates entry of the pyruvate into the TCA cycle irrespective of the genotypes to allow proliferation and bloom of p_Proteobacteria. Indeed, the transition from commensals to dysbiotic microbial community requires an environment which is predominantly aerobic to support the growth of pathogenic bacteria.43,49

Effect of SIGIRR genotype on formula milk-induced alterations in gut microbiome

We and others have shown that formula feeding disrupts the gut microbiome in early life and is an important risk factor for NEC.44,45,46,47 Hypothesizing that presence of the SIGIRR mutant allele will exaggerate dysbiosis and gut injury induced by formula milk feeding, we evaluated alterations in microbial community composition and diversity in the colon in SIGIRR wild type and transgenic mice after formula feeding experiments as aforementioned. We found that differences in host genotype and gut microbiome programming increased dysbiosis after formula feeding (Figures 6A–6F). At the phylum level, while a relatively higher level of p_Firmicutes was seen in Het/Homo group compared to WT in the control with a positive LDA score, the changes were not significant. In response to formula feeding however, mice that carried the Sigirr mutation (either heterozygous or homozygous) displayed significant decrease in p_Firmicutes compared to WT with a negative LDA score indicating a lower abundance of p_Firmicutes in the formula group (Figure 6A). At the same time, an increase in p_Proteobacteria was recorded in the formula-fed group (Figure 6E). These changes correlated with genus level decrease in g_Lactobacillus with higher levels of g_Alphaproteobacteria (Figures 6B and 6F). Interestingly, formula feeding led to relatively higher levels of p_Bacteroidetes and g_Bacteroidia compared to WT (Figures 6C and 6D). Bacteroides species are normally commensals in the gut but can be opportunistic pathogens responsible for infections with significant morbidity and mortality.48Figure 6Effect of formula feeding on bacterial profile across genotypes(A–F) Sigirr wild-type (WT) or Het/Homo mouse pups at postnatal day 8 (P8) were either dam-fed or given formula milk for 3 days before euthanasia. Colon contents were collected and subjected to V3-V4 sequencing. Boxplots represent relative abundance at the phylum and genus level. Linear discriminant analysis effect size (LEfSe) was performed, and taxa were considered significant if they had p values < 0.05 and LDA values ≥2. n = 7–11 pups in each group.(G) Alpha diversity measurement between control and SIGIRR mutant (Het+Homo) mice after formula feeding. Chao1 and Shannon indices are shown.(H and I) D3 Interactive network at a max Bray distance of either 0.15 showing clear separation between the formula and the controls (H) or 0.2 showing dissimilarity between the formula and the controls, with formula WT more similar to the controls (I).(J) Principal coordinate (PC) analysis of the dam-fed or formula-fed mouse pups. First 3 principal coordinates were plotted in 3D space, on the day 8 formula colon experiment showing clear separation between the controls and formula, with Formula WT closer (more similar) to controls (all genotype).(K) Heatmap showing MetaCyc top 30 pathways in the C or F group. Boxed areas represent enrichment of glycolysis III from glucose or pyruvate fermentation to acetate and lactate II in C (green) or F (blue) fed group. n = 7–11 pups in each group.

Effect of formula feeding on bacterial profile across genotypes

(A–F) Sigirr wild-type (WT) or Het/Homo mouse pups at postnatal day 8 (P8) were either dam-fed or given formula milk for 3 days before euthanasia. Colon contents were collected and subjected to V3-V4 sequencing. Boxplots represent relative abundance at the phylum and genus level. Linear discriminant analysis effect size (LEfSe) was performed, and taxa were considered significant if they had p values < 0.05 and LDA values ≥2. n = 7–11 pups in each group.

(G) Alpha diversity measurement between control and SIGIRR mutant (Het+Homo) mice after formula feeding. Chao1 and Shannon indices are shown.

(H and I) D3 Interactive network at a max Bray distance of either 0.15 showing clear separation between the formula and the controls (H) or 0.2 showing dissimilarity between the formula and the controls, with formula WT more similar to the controls (I).

(J) Principal coordinate (PC) analysis of the dam-fed or formula-fed mouse pups. First 3 principal coordinates were plotted in 3D space, on the day 8 formula colon experiment showing clear separation between the controls and formula, with Formula WT closer (more similar) to controls (all genotype).

(K) Heatmap showing MetaCyc top 30 pathways in the C or F group. Boxed areas represent enrichment of glycolysis III from glucose or pyruvate fermentation to acetate and lactate II in C (green) or F (blue) fed group. n = 7–11 pups in each group.

When assessing alpha diversity, we observed significant differences between control and formula feeding wherein, SIGIRR mutant (Het+Homo) mice receiving formula had higher Chao1 and Shannon indices in the order Homo>Het>WT (Figure 6G). This was further verified through regression analysis for genotypes (Table S1). To measure β-diversity, we employed sophisticated D3 Interactive network at a max Bray distance of 0.15. The output from the D3 interactive interface wherein nodes represent stool samples and edges represent relationships, different subnetworks attempt to avoid overlapping one another via a slow repulsion-like algorithm. As is depicted in Figure 6H, with a maximum distance of 0.15, we found clear separation between formula and controls. When less stringent cut-off was applied (0.2, Figure 6I), expanding the network to include moderately similar samples, the similarity relationship between control and formula occurred via WT mice, further lending to the hypothesis that Het/Homo and formula account for additive variation. These are connectivity insights, and the color coding informs on inter-mouse variability. During principal coordinates analysis, plotting the day 8 formula colon experiment in 3D space showed clear separation between the control and the formula with formula WT closer (more similar) to controls (all genotype) (Figure 6J). This was further validated through (1) regression analysis wherein, the SIGIRR mutation (het or homo) on the formula status was the most significantly associated with variation in the microbiome (p value: 1.01e-07; Table S2) and (2) machine learning exercise (random forest) (Table S3) demonstrating significance (0.003) and accuracy (0.6). Lastly, the amplicon sequence variant (ASV) most highly contributing toward these changes belonged to Proteobact_371 representing c_Gammaproteobacteria (Figure S5).

Next, upon PICRUSt2 analysis on the control and the formula-fed group, we found that Glycolysis III from glucose was significantly upregulated in the formula-fed group compared to control group (across genotypes) (Figure 6K). Thus, while glycolysis increased in the formula group, the fermentation of pyruvate into acetate and lactate was minimal in agreement with findings described earlier (see Figure 4O). We posit from these findings that formula feeding intrinsically necessitates entry of the pyruvate into the TCA cycle irrespective of the genotypes to allow proliferation and bloom of p_Proteobacteria. Indeed, the transition from commensals to dysbiotic microbial community requires an environment which is predominantly aerobic to support the growth of pathogenic bacteria.43,49

Effect of host genetics and formula feeding on intestinal injuryWe and others have shown that SIGIRR is developmentally regulated in the neonatal mouse and human intestine24,29 and the SIGIRR genetic variants are enriched in neonates with NEC. Noting the increased dysbiosis seen in SigirrMut mice after formula milk, we investigated whether formula milk-induced gut injury would be influenced by Sigirr mutant allele genotype. For formula feeding induced intestinal injury in mice, we used a standardized 4-point scale, to grade intestinal injury in the ileum based on histologic slides stained with hematoxylin and eosin.24,50 As is depicted in Figure 7A, evaluation of ileal histology revealed that intestinal injury after formula feeding showed a dose-dependent increase with presence of the Sigirr mutant allele (Figures 7A and 7B). We also noted minor increase in intestinal injury scores even at baseline in Sigirr transgenic mice homozygous for the stop mutation (Figures 7A and 7B). Next, we examined changes in the expression of inflammatory cytokines in the ileum after formula feeding (Figure 7C). RNA expression of Cxcl1, intercellular adhesion molecule 1 (Icam1) and Il1b cytokines showed a dose-dependent increase in expression after formula milk with the Sigirr mutant allele (Figure 7C). Increased inflammation and intestinal injury in Sigirr transgenic mice paralleled increased nitrotyrosine staining (Figure 7D), a marker of inflammatory oxidative stress. Finally, we performed the TUNEL assay to evaluate intestinal apoptosis, which showed increased apoptosis in Sigirr transgenic mice after formula feeding (Figures 7E and 7F). These data demonstrate that interactions between the Sigirr host genotype and formula milk induces worse intestinal injury in Sigirr transgenic mice.Figure 7Effect of formula feeding on intestinal injury in mouse pups(A) Representative histology of the terminal ileum from wild-type (WT) mice or mice heterozygous (het) or homozygous (homo) for Sigirr gene, either dam-fed or given formula. Scale bar, 100μm.(B) Histologic intestinal injury scores.(C) Relative gene expression of Cxcl1, Icam1, and IL-1β in controls vs. formula-fed pups.(D) Representative images showing Nitrotyrosine staining. Scale bar, 100 μm.(E) TUNEL staining of terminal ileum of control and formula-fed groups. Green indicates TUNEL positive cells; blue, DAPI cells. Scale bar, 120 μm.(F) Graph showing percentage of apoptotic cells in control vs. the formula-fed group.All data are presented as mean ± SD; ∗p < 0.05, ∗∗p < 0.01, by ANOVA. n = 7–11 pups in each group.

Effect of host genetics and formula feeding on intestinal injury

We and others have shown that SIGIRR is developmentally regulated in the neonatal mouse and human intestine24,29 and the SIGIRR genetic variants are enriched in neonates with NEC. Noting the increased dysbiosis seen in SigirrMut mice after formula milk, we investigated whether formula milk-induced gut injury would be influenced by Sigirr mutant allele genotype. For formula feeding induced intestinal injury in mice, we used a standardized 4-point scale, to grade intestinal injury in the ileum based on histologic slides stained with hematoxylin and eosin.24,50 As is depicted in Figure 7A, evaluation of ileal histology revealed that intestinal injury after formula feeding showed a dose-dependent increase with presence of the Sigirr mutant allele (Figures 7A and 7B). We also noted minor increase in intestinal injury scores even at baseline in Sigirr transgenic mice homozygous for the stop mutation (Figures 7A and 7B). Next, we examined changes in the expression of inflammatory cytokines in the ileum after formula feeding (Figure 7C). RNA expression of Cxcl1, intercellular adhesion molecule 1 (Icam1) and Il1b cytokines showed a dose-dependent increase in expression after formula milk with the Sigirr mutant allele (Figure 7C). Increased inflammation and intestinal injury in Sigirr transgenic mice paralleled increased nitrotyrosine staining (Figure 7D), a marker of inflammatory oxidative stress. Finally, we performed the TUNEL assay to evaluate intestinal apoptosis, which showed increased apoptosis in Sigirr transgenic mice after formula feeding (Figures 7E and 7F). These data demonstrate that interactions between the Sigirr host genotype and formula milk induces worse intestinal injury in Sigirr transgenic mice.Figure 7Effect of formula feeding on intestinal injury in mouse pups(A) Representative histology of the terminal ileum from wild-type (WT) mice or mice heterozygous (het) or homozygous (homo) for Sigirr gene, either dam-fed or given formula. Scale bar, 100μm.(B) Histologic intestinal injury scores.(C) Relative gene expression of Cxcl1, Icam1, and IL-1β in controls vs. formula-fed pups.(D) Representative images showing Nitrotyrosine staining. Scale bar, 100 μm.(E) TUNEL staining of terminal ileum of control and formula-fed groups. Green indicates TUNEL positive cells; blue, DAPI cells. Scale bar, 120 μm.(F) Graph showing percentage of apoptotic cells in control vs. the formula-fed group.All data are presented as mean ± SD; ∗p < 0.05, ∗∗p < 0.01, by ANOVA. n = 7–11 pups in each group.

Effect of formula feeding on intestinal injury in mouse pups

(A) Representative histology of the terminal ileum from wild-type (WT) mice or mice heterozygous (het) or homozygous (homo) for Sigirr gene, either dam-fed or given formula. Scale bar, 100μm.

(B) Histologic intestinal injury scores.

(C) Relative gene expression of Cxcl1, Icam1, and IL-1β in controls vs. formula-fed pups.

(D) Representative images showing Nitrotyrosine staining. Scale bar, 100 μm.

(E) TUNEL staining of terminal ileum of control and formula-fed groups. Green indicates TUNEL positive cells; blue, DAPI cells. Scale bar, 120 μm.

(F) Graph showing percentage of apoptotic cells in control vs. the formula-fed group.

All data are presented as mean ± SD; ∗p < 0.05, ∗∗p < 0.01, by ANOVA. n = 7–11 pups in each group.

Effect of Sigirr genotype on formula milk-induced alterations in gut microbiomeAs we noted more severe intestinal injury with formula milk in SigirrMut mice that were homozygous for SIGIRR mutant allele, we compared microbiota changes between the WT and SigirrMut mice at postnatal days 11, 14, and 28 (terminal ileum, TI) and P8, 11, 14, and 28 (colon). Following 2D PCA, we found that while the days (P11-P28) overlapped, P11 and P28 in the TI clustered on different axes in the WT when compared to SigirrMut mice indicating development differences in microbiome progression (Figures 8Ai and 8Aii). In the colon, while P8 and P28 separated, we observed differences in how the microbiome clustered in WT and SigirrMut mice further supporting the notion that subtle developmental cues can influence microbiome progression (Figures 8Bi and 8Bii). Since we have previously established that formula feeding promotes bacterial dysbiosis,44 we next evaluated the effect of formula feeding to SigirrMut mice. A PCA wherein the first three principal coordinate analyses were plotted against each other with regression planes and ellipsoids of concentration, we found a clear separation between the control and formula-fed group (Figure 8C). PICRUSt2 analysis revealed significant differences between the control and formula feeding group, irrespective of the genotype. In response to formula feeding however, we did see enrichment of KEGG orthologs K07483 (transposase), K07497 (putative transposase), and K07484 (transposase) in SigirrMut vs. the WT (Figure 8D). During pathway analysis, Glycolysis I and II from glucose- and fructose-6-phosphate, was higher in the formula-fed group compared to control group (across genotypes) (Figure 8E). Whether this promotes formula feeding-driven entry of the glycolytic end products into the TCA cycle as seen elsewhere (see Figure 4O) is under investigation. It can be gleaned from these findings that Sigirr homozygosity influences microbiota composition and function in the developing intestine.Figure 8Impact of Sigirr homozygous mutation (SigirrMut) on bacterial profile across timeTerminal ileum (TI) and colon (Colon) contents from WT or SigirrMut mice were collected at postnatal days 11, 14, and 28 (TI) or 8, 11, 14, and 28 (Colon) and subjected to V3-V4 sequencing.(A and B) 2D principal component (PC) analysis showing clear separation at P11 and P28 (TI; Ai, Aii) and P8 and P28 (Colon; Bi, Bii) that indicates microbiome progression.(C) A 3D principal component (PC) analysis showing a clear separation between the control and formula-fed group across genotypes. n = 7–11 pups in each group.(D and E) PICRUSt2 analysis showing top 30 features of KEGG (D) and MetaCyc pathways (E). Boxed areas represent differences in KEGG pathways K07483, K07497, and K-07484 belonging to transposase in WT and SigirrMut mice. Boxed areas are also shown for differences in MetaCyc pathways between WT and SigirrMut mice.

Effect of Sigirr genotype on formula milk-induced alterations in gut microbiome

As we noted more severe intestinal injury with formula milk in SigirrMut mice that were homozygous for SIGIRR mutant allele, we compared microbiota changes between the WT and SigirrMut mice at postnatal days 11, 14, and 28 (terminal ileum, TI) and P8, 11, 14, and 28 (colon). Following 2D PCA, we found that while the days (P11-P28) overlapped, P11 and P28 in the TI clustered on different axes in the WT when compared to SigirrMut mice indicating development differences in microbiome progression (Figures 8Ai and 8Aii). In the colon, while P8 and P28 separated, we observed differences in how the microbiome clustered in WT and SigirrMut mice further supporting the notion that subtle developmental cues can influence microbiome progression (Figures 8Bi and 8Bii). Since we have previously established that formula feeding promotes bacterial dysbiosis,44 we next evaluated the effect of formula feeding to SigirrMut mice. A PCA wherein the first three principal coordinate analyses were plotted against each other with regression planes and ellipsoids of concentration, we found a clear separation between the control and formula-fed group (Figure 8C). PICRUSt2 analysis revealed significant differences between the control and formula feeding group, irrespective of the genotype. In response to formula feeding however, we did see enrichment of KEGG orthologs K07483 (transposase), K07497 (putative transposase), and K07484 (transposase) in SigirrMut vs. the WT (Figure 8D). During pathway analysis, Glycolysis I and II from glucose- and fructose-6-phosphate, was higher in the formula-fed group compared to control group (across genotypes) (Figure 8E). Whether this promotes formula feeding-driven entry of the glycolytic end products into the TCA cycle as seen elsewhere (see Figure 4O) is under investigation. It can be gleaned from these findings that Sigirr homozygosity influences microbiota composition and function in the developing intestine.Figure 8Impact of Sigirr homozygous mutation (SigirrMut) on bacterial profile across timeTerminal ileum (TI) and colon (Colon) contents from WT or SigirrMut mice were collected at postnatal days 11, 14, and 28 (TI) or 8, 11, 14, and 28 (Colon) and subjected to V3-V4 sequencing.(A and B) 2D principal component (PC) analysis showing clear separation at P11 and P28 (TI; Ai, Aii) and P8 and P28 (Colon; Bi, Bii) that indicates microbiome progression.(C) A 3D principal component (PC) analysis showing a clear separation between the control and formula-fed group across genotypes. n = 7–11 pups in each group.(D and E) PICRUSt2 analysis showing top 30 features of KEGG (D) and MetaCyc pathways (E). Boxed areas represent differences in KEGG pathways K07483, K07497, and K-07484 belonging to transposase in WT and SigirrMut mice. Boxed areas are also shown for differences in MetaCyc pathways between WT and SigirrMut mice.

Impact of Sigirr homozygous mutation (SigirrMut) on bacterial profile across time

Terminal ileum (TI) and colon (Colon) contents from WT or SigirrMut mice were collected at postnatal days 11, 14, and 28 (TI) or 8, 11, 14, and 28 (Colon) and subjected to V3-V4 sequencing.

(A and B) 2D principal component (PC) analysis showing clear separation at P11 and P28 (TI; Ai, Aii) and P8 and P28 (Colon; Bi, Bii) that indicates microbiome progression.

(C) A 3D principal component (PC) analysis showing a clear separation between the control and formula-fed group across genotypes. n = 7–11 pups in each group.

(D and E) PICRUSt2 analysis showing top 30 features of KEGG (D) and MetaCyc pathways (E). Boxed areas represent differences in KEGG pathways K07483, K07497, and K-07484 belonging to transposase in WT and SigirrMut mice. Boxed areas are also shown for differences in MetaCyc pathways between WT and SigirrMut mice.

DiscussionEstablishment of a symbiotic intestinal holobiont after birth is important for gut, brain, and lung development with several studies linking alterations in early life gut microbiota to neonatal, childhood, and adult diseases affecting several organ systems.51,52 While the mouse is commonly used to study human biology and model disease, evolution of the microbiota during postnatal intestinal development, and how it is shaped by intestinal site, injury and host genetics remains poorly characterized. In this study we show a clear progression of the gut microbiota wherein a less diverse Proteobacteria (Escherichia_Shigella) rich community gives way to a more diverse, Fimicutes/Bacteroidetes dominant community immediately after weaning. While this transitional pattern was observed both across the ileum and colon, there were remarkable differences in the bacterial communities and the metabolic pathways encoded in these communities between the Ileum and colon. Formula milk induced a composition shift toward more Proteobacteria and less Firmicutes, remarkably similar to patterns described in neonates given formula feeding. SigirrMut mice that encode a mutation identified in an infant with NEC displayed minor changes in microbiota composition at baseline but significantly differed from WT mice after injury was induced with formula milk. Our results highlight a patterned developmental progression of intestinal microbiome that is influenced by time, site of sampling, host genetics, and injury.Our data showing early enrichment of the gut microbiota with p_Firmicutes (g. Lactobacillus) and p_Proteobacteria (g_ Escherichia/Shigella) followed by transition to a more stable pattern with decreased p_Proteobacteria, increased p_Bacteroidetes, and stable p_Firmicutes is consistent with what is reported in full-term human.52 However, postnatal mouse gut development differs from human intestinal development in that the mouse intestinal morphogenesis and villus maturation happens postnatally in the first 4 weeks after birth compared to a near fully mature villus in the human newborn.53,54 We did note some parallels to the progression in bacterial classes from Bacilli to Gammaprotebacteria to Clostridia observed in preterm infants.5,7,8,9 The composition of the intestinal microbiota in preterm infants is correlated to an increased risk of NEC due potentially to elevated levels of Enterobacteriaceae, Enterococcus, and other pathobionts compared to full-term neonates.55,56,57,58,59,60,61,62,63 During our investigation, profiling microbiota succession longitudinally till immediately after weaning revealed a distinct pattern of gut microbiome evolution in the postnatal mouse intestine, which was shaped by developmental maturity, site of sampling, formula milk feeding, and SIGIRR genotype.It has been established that the neonatal gut environment is more aerobic than adult wherein, the gut is predominantly hypoxic.64 The presence of oxygen or lack thereof, governs the types of bacteria that colonize the intestine. In our investigation, we discovered a significant presence of facultative anaerobe g_Lactobacillus that correlated with progressive increase in PWY-4894 related to urea cycle. The urea cycle plays an essential role in metabolism in the liver and to a lesser extent in the intestine. It is closely linked to the citric acid cycle and derives its nitrogen (N2) through transamination of oxaloacetate to form aspartate returning fumarate to the cycle in the process.65 The biosynthesis of urea demands energy and can be impacted by compositional shift wherein, a decline in g_Bifidobacteria and increased abundance of g_Lactobacillus can lead to activated urea metabolism in the intestine33 which corroborates with our findings.In the colon, the metabolism of carbohydrates by the gut microbiota is a key process supplying nutrients and energy to the host with Lachnospiraceae, Lactobacillaceae, Ruminococcaceae, and Roseburia species hydrolyzing starch and other sugars to produce butyrate and other short-chain fatty acids (SCFAs).66,67,68 In addition, nitrogen (N2) is a fundamental need for bacteria to survive in any environment. Indeed, several non-protein N2 sources including creatine, creatinine, polyamines, and free amino acids are equally essential besides oligosaccharides for infant health.69,70 Our studies provide data that suggests that reduction of denitrification pathway at P28 and not necessarily bifidogenic processes, is most likely associated with N2-dependent increases in aromatic amino acid and other nitrogen compound biosynthesis in the colon and that these processes are contributing to a healthy neonatal gut and commensal microbiota.Studies in humans have consistently demonstrated existence of unique microbial communities in the adult ileum and colon, and yet microbial colonization differences between the ileum and colon are not well understood during postnatal gut development. Longitudinal profiling of mouse pups at postnatal days 11, 14, and 28 (TI) and P8, 11, 14, and 28 (colon), identified significant alterations in microbial composition with corresponding changes in metabolic pathways. We opted to obtain luminal contents for both the ileum and colon to avoid issues related to contamination, and because profiling stool may not reflect regional diversity between distal colon and proximal intestine. At the phylum level, the predominant bacterial groups in the TI were p_Firmicutes and p_Bacteroidetes while p_Proteobacteria declined from P11 to P28. At the genus level, constant presence of g_Lactobacillus with declining trend in g_Escherichia-Shigella correlated with significant differences in α- and β-diversities across time. In the colon, while the trend paralleled changes recorded in the TI, there were significant differences in the proportion/types of bacteria that progressively colonized at different time points. From p_Proteobacteria (LDA score >6) and g_Escherichia-Shigella (LDA score >6) at P8 to p_Patescibacteria (LDA score >4) and g_Lachnospiraceae_NK4A136_group (LDA score >5), f_Prevotellaceae (LDA score >5), and g_Roseburia (LDA score >4) at P28. These data align with full term human neonates wherein, p_Proteobacteria and p_Firmicutes are the main phyla represented during the first days of life.71The physiological variations along the length of the small intestine and colon that harbors a variety of distinct microbial habitats include chemical and nutrient gradients, as well as compartmentalized host immune activity, which are known to influence bacterial community composition. The neonatal gut is initially dominated by Bifidobacterium, Veillonella, Streptococcus, Citrobacter, Escherichia, Bacteroides and Clostridium, which are also abundant in the adults but progressively populated by strict anaerobic bacteria belonging to the Firmicutes and Bacteroidetes phyla.52,72 During a comparative analysis of the bacterial constituents of the TI and colon, we discovered significant differences in the types of bacteria that inhabited the two sites at each postnatal life that correlated with dynamic changes in pathways ranging from toluene degradation and fatty acid salvage in the TI to pathways belonging to naphthoate and heme biosynthesis in the colon. We infer from these findings that similar to humans, the composition, function, and plasticity of the gut microbiota in the neonates may underlie key differences in both ecology and physiology between the TI and colon.Studies in humans have suggested a modest effect of host genetics, especially in combination with diet on microbiota composition both at baseline and disease.73,74,75,76 SIGIRR is a prominent negative regulator of TLR signaling in the gut, and we have shown that SIGIRR variants that promote exaggerated TLR4 signaling are associated with increased NEC risk in premature infants potentially through alterations in gut microbiota in the developing intestine.27,28 Positing that mice encoding the p.Y168X stop mutation might alter microbiota composition, we profiled the microbiome during postnatal gut development at baseline or after changes in the diet (formula milk). Our investigations revealed that among the three major phyla, p_Firmicutes, p_Bacteroidetes and p_Proteobacteria, changes in p_Bacteroidetes at P14 between TI and colon were more dramatic despite lack of a genotype effect. These ecological differences between TI and colon clearly suggest that increases in relative abundance of colon p_Bacteroidetes on its own may be sufficient for the maintenance of biodiversity and infant health while both p_Bacteroidetes and p_Firmicutes may be required at P28 in the TI for the control of metabolism, function, and colonization resistance to maintain homeostasis.Formula milk feeding is one of the major risk factors for NEC in premature infants,17,18 and is commonly used to induce experimental NEC in animal models.77,78 We queried whether formula feeding induced dysbiosis in the mouse pups similar to that reported in preterm infants. Both α- and β-diversities were strikingly different between the control and the formula-fed group due to significant decreases in p_Firmicutes with concurrent increase in p_Proteobacteria in the colonic stool of formula fed pups. Formula feeding induced intestinal injury in WT mice that resembles milder forms of NEC described in preterm neonates. Strikingly, increases in c_Gammaproteobacteria (p_Proteobacteria) with relative decreases in bacteria belonging to p_Firmicutes have been noted in infants who develop NEC.14,15,19 Although the pathogenesis of human NEC is complex and is related to bovine-derived formula milk, it is interesting that feeding canine formula in neonatal mice induces a dysbiotic microbial signature associated with NEC in humans. We speculate whether this arises from the effect of fortification that increases the carbohydrate content in milk, which favors growth of facultative anaerobes such as c_Gammaproteobacteria over anaerobes or microaerophilic bacteria such as g_Lactobacillus.43,49 Our PICRUSt2 analysis revealing an enrichment in glycolysis pathway with relative decrease in pyruvate fermentation to acetate and lactate is consistent with our inference.We next investigated the interactions between Sigirr genotype and formula milk in altering microbiota composition. Predictably, formula milk induced decreases in p_Firmicutes and g_Lactobacillus were exaggerated in SigirrMut mice, while there were trends toward increases in p_Proteobacteria, p_Bacteroidetes, and g_Alphaproteobacteria. Further, Sigirr mutant mice had greater increases in alpha diversity after formula milk feeding. Using PCA and AI-based analysis showed distinct separation in the microbiota in formula-fed mice vs. dam-fed mice, with SigirrMut mice fed formula milk showing the most separation from dam-fed mice. Further confirmation that Sigirr genotype influenced injury-driven dysbiosis was noted in regression and random-forest plots that confirmed that variation in the microbiota composition after formula feeding was significantly affected by presence of the SIGIRR mutation identified in NEC. Studies in adults have identified interactions between the lactase non-persistence phenotype and Bifidobacterium abundance as well as the combination of ABO blood group antigens and fucosyl transferase secretor status with Bacteroides and Faecalibacterium spp.73,74,76 Interestingly, the impact of lactase non-secretor phenotype on Bifidobacterium abundance is dependent on milk consumption, similar to our results wherein adding formula milk reveals the effect of the SIGIRR genotype on dysbiosis.74,79 Noting that the SIGIRR genotype programmed shifts in the microbiota composition that favored relative abundance of bacteria implicated in NEC (p_Proteobacteria, g_Escherichia coli/Shigella), we posited that formula feeding would induce more severe injury in SigirrMut mice. We observed dose-dependent increase in intestinal inflammation, cell death, oxidative stress, and injury with formula feeding in SigirrMut mice. While our data indicate that interactions between host genetics and clinical risk factors can impact NEC vulnerability,21 the precise mechanisms underlying this remain to be determined. Whether mild inflammation noted even at baseline in SigirrMut mice programs an altered metabolic or oxidative milieu conducive of microbiota composition shifts and more severe injury needs to be further investigated.80,81Limitations of the studyThe host-bacterial ecosystem in the gut is frequently considered a holobiont with symbiotic relationships shaping gut health and beyond. Our study reveals a patterned progression of gut microbiota from an aerobic, p_Proteobacteria/p_Firmicutes-rich community to an anaerobic, p_Bacteroidetes/p_Firmicutes rich community that resembles human infants. These temporal changes in the bacterial community structure are strongly impacted by site of sampling and diet (formula milk feeding and weaning). Strikingly, we noted that formula milk induced dysbiosis in neonatal mice resembles that described in neonates, especially preterm infants. These results lend confidence to the use of mice to model neonatal human gut-brain/gut-lung axis and disease. Although, we noted only very modest changes in the gut microbiota composition at baseline with the SIGIRR mutation identified in human NEC, formula-milk induced dysbiosis and severity of intestinal injury was clearly influenced by Sigirr genotypes. Sequentially profiling the intestinal metabolome along with deeper metagenomics analysis of intestinal microbiome would have added further strength to our results. Future research will focus on the mechanisms by which host genetics and neonatal gut inflammation shape microbiome development and function in mice.

Establishment of a symbiotic intestinal holobiont after birth is important for gut, brain, and lung development with several studies linking alterations in early life gut microbiota to neonatal, childhood, and adult diseases affecting several organ systems.51,52 While the mouse is commonly used to study human biology and model disease, evolution of the microbiota during postnatal intestinal development, and how it is shaped by intestinal site, injury and host genetics remains poorly characterized. In this study we show a clear progression of the gut microbiota wherein a less diverse Proteobacteria (Escherichia_Shigella) rich community gives way to a more diverse, Fimicutes/Bacteroidetes dominant community immediately after weaning. While this transitional pattern was observed both across the ileum and colon, there were remarkable differences in the bacterial communities and the metabolic pathways encoded in these communities between the Ileum and colon. Formula milk induced a composition shift toward more Proteobacteria and less Firmicutes, remarkably similar to patterns described in neonates given formula feeding. SigirrMut mice that encode a mutation identified in an infant with NEC displayed minor changes in microbiota composition at baseline but significantly differed from WT mice after injury was induced with formula milk. Our results highlight a patterned developmental progression of intestinal microbiome that is influenced by time, site of sampling, host genetics, and injury.

Our data showing early enrichment of the gut microbiota with p_Firmicutes (g. Lactobacillus) and p_Proteobacteria (g_ Escherichia/Shigella) followed by transition to a more stable pattern with decreased p_Proteobacteria, increased p_Bacteroidetes, and stable p_Firmicutes is consistent with what is reported in full-term human.52 However, postnatal mouse gut development differs from human intestinal development in that the mouse intestinal morphogenesis and villus maturation happens postnatally in the first 4 weeks after birth compared to a near fully mature villus in the human newborn.53,54 We did note some parallels to the progression in bacterial classes from Bacilli to Gammaprotebacteria to Clostridia observed in preterm infants.5,7,8,9 The composition of the intestinal microbiota in preterm infants is correlated to an increased risk of NEC due potentially to elevated levels of Enterobacteriaceae, Enterococcus, and other pathobionts compared to full-term neonates.55,56,57,58,59,60,61,62,63 During our investigation, profiling microbiota succession longitudinally till immediately after weaning revealed a distinct pattern of gut microbiome evolution in the postnatal mouse intestine, which was shaped by developmental maturity, site of sampling, formula milk feeding, and SIGIRR genotype.

It has been established that the neonatal gut environment is more aerobic than adult wherein, the gut is predominantly hypoxic.64 The presence of oxygen or lack thereof, governs the types of bacteria that colonize the intestine. In our investigation, we discovered a significant presence of facultative anaerobe g_Lactobacillus that correlated with progressive increase in PWY-4894 related to urea cycle. The urea cycle plays an essential role in metabolism in the liver and to a lesser extent in the intestine. It is closely linked to the citric acid cycle and derives its nitrogen (N2) through transamination of oxaloacetate to form aspartate returning fumarate to the cycle in the process.65 The biosynthesis of urea demands energy and can be impacted by compositional shift wherein, a decline in g_Bifidobacteria and increased abundance of g_Lactobacillus can lead to activated urea metabolism in the intestine33 which corroborates with our findings.

In the colon, the metabolism of carbohydrates by the gut microbiota is a key process supplying nutrients and energy to the host with Lachnospiraceae, Lactobacillaceae, Ruminococcaceae, and Roseburia species hydrolyzing starch and other sugars to produce butyrate and other short-chain fatty acids (SCFAs).66,67,68 In addition, nitrogen (N2) is a fundamental need for bacteria to survive in any environment. Indeed, several non-protein N2 sources including creatine, creatinine, polyamines, and free amino acids are equally essential besides oligosaccharides for infant health.69,70 Our studies provide data that suggests that reduction of denitrification pathway at P28 and not necessarily bifidogenic processes, is most likely associated with N2-dependent increases in aromatic amino acid and other nitrogen compound biosynthesis in the colon and that these processes are contributing to a healthy neonatal gut and commensal microbiota.

Studies in humans have consistently demonstrated existence of unique microbial communities in the adult ileum and colon, and yet microbial colonization differences between the ileum and colon are not well understood during postnatal gut development. Longitudinal profiling of mouse pups at postnatal days 11, 14, and 28 (TI) and P8, 11, 14, and 28 (colon), identified significant alterations in microbial composition with corresponding changes in metabolic pathways. We opted to obtain luminal contents for both the ileum and colon to avoid issues related to contamination, and because profiling stool may not reflect regional diversity between distal colon and proximal intestine. At the phylum level, the predominant bacterial groups in the TI were p_Firmicutes and p_Bacteroidetes while p_Proteobacteria declined from P11 to P28. At the genus level, constant presence of g_Lactobacillus with declining trend in g_Escherichia-Shigella correlated with significant differences in α- and β-diversities across time. In the colon, while the trend paralleled changes recorded in the TI, there were significant differences in the proportion/types of bacteria that progressively colonized at different time points. From p_Proteobacteria (LDA score >6) and g_Escherichia-Shigella (LDA score >6) at P8 to p_Patescibacteria (LDA score >4) and g_Lachnospiraceae_NK4A136_group (LDA score >5), f_Prevotellaceae (LDA score >5), and g_Roseburia (LDA score >4) at P28. These data align with full term human neonates wherein, p_Proteobacteria and p_Firmicutes are the main phyla represented during the first days of life.71

The physiological variations along the length of the small intestine and colon that harbors a variety of distinct microbial habitats include chemical and nutrient gradients, as well as compartmentalized host immune activity, which are known to influence bacterial community composition. The neonatal gut is initially dominated by Bifidobacterium, Veillonella, Streptococcus, Citrobacter, Escherichia, Bacteroides and Clostridium, which are also abundant in the adults but progressively populated by strict anaerobic bacteria belonging to the Firmicutes and Bacteroidetes phyla.52,72 During a comparative analysis of the bacterial constituents of the TI and colon, we discovered significant differences in the types of bacteria that inhabited the two sites at each postnatal life that correlated with dynamic changes in pathways ranging from toluene degradation and fatty acid salvage in the TI to pathways belonging to naphthoate and heme biosynthesis in the colon. We infer from these findings that similar to humans, the composition, function, and plasticity of the gut microbiota in the neonates may underlie key differences in both ecology and physiology between the TI and colon.

Studies in humans have suggested a modest effect of host genetics, especially in combination with diet on microbiota composition both at baseline and disease.73,74,75,76 SIGIRR is a prominent negative regulator of TLR signaling in the gut, and we have shown that SIGIRR variants that promote exaggerated TLR4 signaling are associated with increased NEC risk in premature infants potentially through alterations in gut microbiota in the developing intestine.27,28 Positing that mice encoding the p.Y168X stop mutation might alter microbiota composition, we profiled the microbiome during postnatal gut development at baseline or after changes in the diet (formula milk). Our investigations revealed that among the three major phyla, p_Firmicutes, p_Bacteroidetes and p_Proteobacteria, changes in p_Bacteroidetes at P14 between TI and colon were more dramatic despite lack of a genotype effect. These ecological differences between TI and colon clearly suggest that increases in relative abundance of colon p_Bacteroidetes on its own may be sufficient for the maintenance of biodiversity and infant health while both p_Bacteroidetes and p_Firmicutes may be required at P28 in the TI for the control of metabolism, function, and colonization resistance to maintain homeostasis.

Formula milk feeding is one of the major risk factors for NEC in premature infants,17,18 and is commonly used to induce experimental NEC in animal models.77,78 We queried whether formula feeding induced dysbiosis in the mouse pups similar to that reported in preterm infants. Both α- and β-diversities were strikingly different between the control and the formula-fed group due to significant decreases in p_Firmicutes with concurrent increase in p_Proteobacteria in the colonic stool of formula fed pups. Formula feeding induced intestinal injury in WT mice that resembles milder forms of NEC described in preterm neonates. Strikingly, increases in c_Gammaproteobacteria (p_Proteobacteria) with relative decreases in bacteria belonging to p_Firmicutes have been noted in infants who develop NEC.14,15,19 Although the pathogenesis of human NEC is complex and is related to bovine-derived formula milk, it is interesting that feeding canine formula in neonatal mice induces a dysbiotic microbial signature associated with NEC in humans. We speculate whether this arises from the effect of fortification that increases the carbohydrate content in milk, which favors growth of facultative anaerobes such as c_Gammaproteobacteria over anaerobes or microaerophilic bacteria such as g_Lactobacillus.43,49 Our PICRUSt2 analysis revealing an enrichment in glycolysis pathway with relative decrease in pyruvate fermentation to acetate and lactate is consistent with our inference.

We next investigated the interactions between Sigirr genotype and formula milk in altering microbiota composition. Predictably, formula milk induced decreases in p_Firmicutes and g_Lactobacillus were exaggerated in SigirrMut mice, while there were trends toward increases in p_Proteobacteria, p_Bacteroidetes, and g_Alphaproteobacteria. Further, Sigirr mutant mice had greater increases in alpha diversity after formula milk feeding. Using PCA and AI-based analysis showed distinct separation in the microbiota in formula-fed mice vs. dam-fed mice, with SigirrMut mice fed formula milk showing the most separation from dam-fed mice. Further confirmation that Sigirr genotype influenced injury-driven dysbiosis was noted in regression and random-forest plots that confirmed that variation in the microbiota composition after formula feeding was significantly affected by presence of the SIGIRR mutation identified in NEC. Studies in adults have identified interactions between the lactase non-persistence phenotype and Bifidobacterium abundance as well as the combination of ABO blood group antigens and fucosyl transferase secretor status with Bacteroides and Faecalibacterium spp.73,74,76 Interestingly, the impact of lactase non-secretor phenotype on Bifidobacterium abundance is dependent on milk consumption, similar to our results wherein adding formula milk reveals the effect of the SIGIRR genotype on dysbiosis.74,79 Noting that the SIGIRR genotype programmed shifts in the microbiota composition that favored relative abundance of bacteria implicated in NEC (p_Proteobacteria, g_Escherichia coli/Shigella), we posited that formula feeding would induce more severe injury in SigirrMut mice. We observed dose-dependent increase in intestinal inflammation, cell death, oxidative stress, and injury with formula feeding in SigirrMut mice. While our data indicate that interactions between host genetics and clinical risk factors can impact NEC vulnerability,21 the precise mechanisms underlying this remain to be determined. Whether mild inflammation noted even at baseline in SigirrMut mice programs an altered metabolic or oxidative milieu conducive of microbiota composition shifts and more severe injury needs to be further investigated.80,81

Limitations of the studyThe host-bacterial ecosystem in the gut is frequently considered a holobiont with symbiotic relationships shaping gut health and beyond. Our study reveals a patterned progression of gut microbiota from an aerobic, p_Proteobacteria/p_Firmicutes-rich community to an anaerobic, p_Bacteroidetes/p_Firmicutes rich community that resembles human infants. These temporal changes in the bacterial community structure are strongly impacted by site of sampling and diet (formula milk feeding and weaning). Strikingly, we noted that formula milk induced dysbiosis in neonatal mice resembles that described in neonates, especially preterm infants. These results lend confidence to the use of mice to model neonatal human gut-brain/gut-lung axis and disease. Although, we noted only very modest changes in the gut microbiota composition at baseline with the SIGIRR mutation identified in human NEC, formula-milk induced dysbiosis and severity of intestinal injury was clearly influenced by Sigirr genotypes. Sequentially profiling the intestinal metabolome along with deeper metagenomics analysis of intestinal microbiome would have added further strength to our results. Future research will focus on the mechanisms by which host genetics and neonatal gut inflammation shape microbiome development and function in mice.

Limitations of the study

The host-bacterial ecosystem in the gut is frequently considered a holobiont with symbiotic relationships shaping gut health and beyond. Our study reveals a patterned progression of gut microbiota from an aerobic, p_Proteobacteria/p_Firmicutes-rich community to an anaerobic, p_Bacteroidetes/p_Firmicutes rich community that resembles human infants. These temporal changes in the bacterial community structure are strongly impacted by site of sampling and diet (formula milk feeding and weaning). Strikingly, we noted that formula milk induced dysbiosis in neonatal mice resembles that described in neonates, especially preterm infants. These results lend confidence to the use of mice to model neonatal human gut-brain/gut-lung axis and disease. Although, we noted only very modest changes in the gut microbiota composition at baseline with the SIGIRR mutation identified in human NEC, formula-milk induced dysbiosis and severity of intestinal injury was clearly influenced by Sigirr genotypes. Sequentially profiling the intestinal metabolome along with deeper metagenomics analysis of intestinal microbiome would have added further strength to our results. Future research will focus on the mechanisms by which host genetics and neonatal gut inflammation shape microbiome development and function in mice.

Resource availabilityLead contactRequests for further information and resources should be directed to and will be fulfilled by the lead contact, Shahid Umar (sumar@kumc.edu).Materials availabilityThis study did not generate new unique reagents.Data and code availability
•Raw data used for analysis has been uploaded to the Figshare, DOI: https://doi.org/10.6084/m9.figshare.28570697.v2.•Source codes have been deposited on Git repositories and are publicly available which are listed in the key resources table.•Any additional information required for the data reported in this paper is available from the lead contact upon request.

Resource availability

Lead contactRequests for further information and resources should be directed to and will be fulfilled by the lead contact, Shahid Umar (sumar@kumc.edu).

Lead contact

Requests for further information and resources should be directed to and will be fulfilled by the lead contact, Shahid Umar (sumar@kumc.edu).

Materials availabilityThis study did not generate new unique reagents.

Materials availability

This study did not generate new unique reagents.

Data and code availability
•Raw data used for analysis has been uploaded to the Figshare, DOI: https://doi.org/10.6084/m9.figshare.28570697.v2.•Source codes have been deposited on Git repositories and are publicly available which are listed in the key resources table.•Any additional information required for the data reported in this paper is available from the lead contact upon request.

Data and code availability

•Raw data used for analysis has been uploaded to the Figshare, DOI: https://doi.org/10.6084/m9.figshare.28570697.v2.•Source codes have been deposited on Git repositories and are publicly available which are listed in the key resources table.•Any additional information required for the data reported in this paper is available from the lead contact upon request.

Raw data used for analysis has been uploaded to the Figshare, DOI: https://doi.org/10.6084/m9.figshare.28570697.v2.

Source codes have been deposited on Git repositories and are publicly available which are listed in the key resources table.

Any additional information required for the data reported in this paper is available from the lead contact upon request.

The work presented here was supported by 1R01 DK117296-05 (S.U. and V.S.) and 1R01DK139636 (V.S.) from the 10.13039/100000002National Institutes of Health. X.H. and C.Z. are funded by the 10.13039/100000001National Science Foundation CAREER award [DBI-1943291].

S.U. conceptualized the project along with VS, analyzed microbiome data, wrote and critically reviewed the manuscript for intellectual content; H.X. processed and analyzed microbiota data, prepared data visualization, and revised the manuscript for critical intellectual content; C.Z. processed and analyzed microbiota data, performed data visualization; W.Y. conducted mouse experiments, performed expression assays and analysis, prepared histological images and analyzed them; M.M. provided intellectual input; M.B.R. prepared data visualization and revised the manuscript for critical intellectual content; M.I.A. performed all the machine learning work and critically analyzed the data; V.S. conceptualized the study, supervised key portions of the project, wrote and critically revised the manuscript for intellectual content; All authors read and approved the final manuscript.

All authors declare no financial or non-financial competing interests.

STAR★MethodsKey resources table
REAGENT or RESOURCESOURCEIDENTIFIERAntibodiesNitrotyrosineSanta Cruzsc-32757; RRID: AB_628022Chemicals, peptides, and recombinant proteinsEsbilac canine milk replacerPetAgN/ATRiZolThermoFisher15596018cDNA reverse transcription kitNEBM3010FLUORO-GELElectron Microscopy Science1798510Critical commercial assaysQIAamp PowerFecal Pro DNA KitQiagen51804DeadEnd™ Fluorometric TUNEL SystemPromegaG3250Deposited data16S rRNA Amplicon SequencingThis paperFigShare, https://doi.org/10.6084/m9.figshare.28570697.v2Experimental models: Organisms/strainsSigirr mutant miceSampath labSoftware and algorithmsRR Development Core Teamhttps://cran.r-project.org/PythonPython Software Foundationhttps://www.python.org/DADA2DADA2 Toolhttps://benjjneb.github.io/dada2/PICRUSt 2.0PICRUSt 2 Toolhttps://huttenhower.sph.harvard.edu/picrust/STAMPSTAMP Softwarehttps://beikolab.cs.dal.ca/software/STAMPSource codeThis studyhttps://github.com/xuan13hao/Postnatal_gut_microbiota_SIGIRR
Experimental model and study participant detailsThe Sigirr mutant mice mimicking human truncated SIGIRR variant (pY168x) were generated in the genetic background of C57BL/6 using CRISPR-Cas9 technology as previously described.22 Mice were housed under pathogen-free conditions (temperature, 25°C; 12-h light/dark cycle) with food and water available ad libitum. Littermate neonatal mice from heterozygous X heterozygous breeding were randomly allocated to each experimental group. Sex of all mice used were noted and the experimental groups were equally represented with male and female mouse pups. In the results sections and the figures (including supplemental) data from sex-based comparisons is shown. Care of mice before and during experimental procedures was conducted in accordance with the policies at the University of Missouri–Kansas City Laboratory Animal Resource Center and the National Institutes of Health Guidelines for the Care and Use of Laboratory Animals. Protocols had prior approval from the University of Missouri–Kansas City Institutional Animal Care and Use Committee.Method detailsFormula feedingFor formula feeding mouse model, litter mate wild type and Sigirr mutant pups were gavage-fed 0.1mL of Esbilac canine milk replacer (PetAg, Inc., Hampshire, IL) five times daily starting at day of life (DOL) 5 to DOL 7. At DOL8, mice were euthanized, and terminal ileum and colon were collected and snap frozen for further microbiome and histological studies.16S rRNA amplicon sequencingTerminal ileum and colon stool contents were collected and immediately frozen and stored at −80°C. Total bacterial genomic DNA was extracted using the QIAmp DNA stool kit (Qiagen) following their instructions. The DNA integrity was assessed by agarose gel electrophoresis; concentration and quality were determined by absorption at A260, and A260/A280 ratio, respectively, using a Nanodrop2000 spectrophotometer (ThermoFisher Scientific). The 16S V4 region was amplified using 515F/806R primers and sequenced using amplicon sequencing on IonS5TMXL to generate raw reads. Paired-end reads were assigned to samples based on their unique barcode and truncated by cutting off the barcode and primer sequences.Analysis of 16S rRNA amplicon sequencing dataThe 16S rRNA amplicon sequencing data underwent processing using DADA2 v1.1,82 generating an Amplicon Sequence Variant (ASV) table. This processing involved quality trimming, filtering, dereplication, error rate estimation, denoising, merging of paired-end reads, screening for mismatches, and chimeric sequence removal. Taxonomic classification was performed using the SILVA reference database (SILVA 132 release).83 Then preprocessing steps were applied using the MicrobiomeAnalyst platform.84 Firstly, we filtered out low-abundance features based on a 20% prevalence filter and set a minimum count threshold of 4. Features with low variance were removed using the interquartile range, employing a cutoff range of 10%. To ensure uniformity across samples, we rarified all samples to the sequencing depth of the sample with the lowest sequence count. Subsequently, we normalized the remaining features using rarefaction to the minimum library size and the total sum scaling (TSS) method. Alpha diversity indices were calculated, and statistical analysis was conducted to assess differences between groups. Beta diversity was explored using Nonmetric Multidimensional Scaling (NMDS) ordination, and differentially abundant taxa were identified using Kruskal-Wallis rank sum tests and Linear Discriminant Analysis Effect Size (LEfSe). Additionally, functional profiling of the metagenome was predicted using Phylogenetic Investigation of Communities by Reconstruction of Unobserved States (PICRUSt2),13 with subsequent analysis of MetaCyc and KEGG pathways using STAMP,85 employing appropriate statistical tests to identify differences in pathway abundances between groups.TUNEL assay for apoptosisThe Terminal deoxynucleotidyl transferase dUTP nick end labeling (TUNEL) assay was conducted on terminal ileum sections according to the guidelines provided by the manufacturer (Promega, Madison, WI). High-resolution images were obtained at using a Nikon W1 confocal microscope equipped with a 20X objective lens. The quantification process involved counting the number of total cells by DAPI staining and those that were TUNEL positive (indicating green fluorescence) in each field. For accuracy, this was performed across a minimum of 3 HPFs for each specimen. Subsequently, the apoptotic index was determined by calculating the ratio of TUNEL-positive cells to the total DAPI-stained cell population.Quantitative real-time PCRTotal RNA was isolated using TRiZol (ThermoFisher). cDNA was synthesized using a cDNA reverse transcription kit (NEB). For the quantification of gene amplification, Real-time PCR was performed using a ViiA7 (Applied Biosystems). The sequences of gene-specific primers are given in supplemental information. RPLP0 or GAPDH were used as endogenous normalization control.ImmunostainingParaffin sections of terminal ileum were cut in 4μm thickness. Sections were deparaffinized with xylene and graded series of alcohol. Antigen retrieval was performed using citrate buffer for 20mins at 95 C. After several rinses with PBS, sections were blocked with Power Block™ Universal Blocking Reagent for one hour and then stained overnight at 4°C with primary Nitrotyrosine Antibody (Santa Cruz 1: 100, sc-32757). After washing with PBS, slides were stained with secondary antibody conjugated with Alexa488 (Invitrogen,1:100) for one hour at room temperature. After treating with secondary antibodies slides were washed several times with PBS, counterstained with 4′6′-diamidino-2-phenylindole (DAPI) and mounted with FLUORO-GEL (Electron Microscopy Science 1798510). Images were acquired using X25/oil objective with a Zeiss Inverted LSM 510 meta laser scanning confocal microscope.Quantification and statistical analysisAnalysis of variance with Bonferroni correction for multiple testing was used for analysis. Statistical analysis was done using GraphPad Prism (San Diego, CA) with statistical significance set at p < 0.05. For microbiome analysis, alpha diversity was assessed using indices, such as Chao1 and Shannon, calculated across samples. Then, differences between groups were evaluated using either a t-test or ANOVA. Beta diversity was explored using NMDS ordination with Bray-Curtis index and Jensen-Shannon divergence at both phylum and genus levels. To test for significance, permutational multivariate analysis of variance (PERMANOVA) was employed. Furthermore, Kruskal-Wallis rank sum tests were conducted on identified genera and phyla to identify differentially abundant taxa between groups. Taxa with raw p-values <0.05 were retained for further analysis, consistent with the methodology applied in our LEfSe analysis. Additionally, LEfSe was utilized to gain insight into differential abundance at each taxonomic level, considering taxa significant if they had p-values <0.05 and LDA values ≥2. For studies on formula milk-induced intestinal inflammation and injury, ANOVA was used with a p < 0.05 considered significant. We used “phyloseq” library in version 1.44 in Rstudio 2024.04.2 Build 764 to contain the microbiome data in a phyloseq object. The R library “shiny” version 1.7.5 was used, in conjunction with “phyloseq” in an interactive web application to visualize the data.86 In this interactive D3 network visualization shown in Figures 6H and 6I, nodes represent stool samples and edges represent relationships. The Bray-Curtis distance of 0.15 and 0.2 is a measure of dissimilarity with measure of 0.2 being more dissimilar than 0.15. The 0.15 threshold will therefore show more separation as the more dissimilar relationships are excluded than the slightly less stringent 0.2 threshold. The tighter the clusters, the more homogenous the samples within a group. The R function “glm” from the “stats” library version 4.3.1 was used for regression analysis of genotypes. The “train” function from the “caret” library version 6.0.94 was used to generate random forest machine learning analysis.87

STAR★Methods

Key resources table
REAGENT or RESOURCESOURCEIDENTIFIERAntibodiesNitrotyrosineSanta Cruzsc-32757; RRID: AB_628022Chemicals, peptides, and recombinant proteinsEsbilac canine milk replacerPetAgN/ATRiZolThermoFisher15596018cDNA reverse transcription kitNEBM3010FLUORO-GELElectron Microscopy Science1798510Critical commercial assaysQIAamp PowerFecal Pro DNA KitQiagen51804DeadEnd™ Fluorometric TUNEL SystemPromegaG3250Deposited data16S rRNA Amplicon SequencingThis paperFigShare, https://doi.org/10.6084/m9.figshare.28570697.v2Experimental models: Organisms/strainsSigirr mutant miceSampath labSoftware and algorithmsRR Development Core Teamhttps://cran.r-project.org/PythonPython Software Foundationhttps://www.python.org/DADA2DADA2 Toolhttps://benjjneb.github.io/dada2/PICRUSt 2.0PICRUSt 2 Toolhttps://huttenhower.sph.harvard.edu/picrust/STAMPSTAMP Softwarehttps://beikolab.cs.dal.ca/software/STAMPSource codeThis studyhttps://github.com/xuan13hao/Postnatal_gut_microbiota_SIGIRR

Key resources table

REAGENT or RESOURCESOURCEIDENTIFIERAntibodiesNitrotyrosineSanta Cruzsc-32757; RRID: AB_628022Chemicals, peptides, and recombinant proteinsEsbilac canine milk replacerPetAgN/ATRiZolThermoFisher15596018cDNA reverse transcription kitNEBM3010FLUORO-GELElectron Microscopy Science1798510Critical commercial assaysQIAamp PowerFecal Pro DNA KitQiagen51804DeadEnd™ Fluorometric TUNEL SystemPromegaG3250Deposited data16S rRNA Amplicon SequencingThis paperFigShare, https://doi.org/10.6084/m9.figshare.28570697.v2Experimental models: Organisms/strainsSigirr mutant miceSampath labSoftware and algorithmsRR Development Core Teamhttps://cran.r-project.org/PythonPython Software Foundationhttps://www.python.org/DADA2DADA2 Toolhttps://benjjneb.github.io/dada2/PICRUSt 2.0PICRUSt 2 Toolhttps://huttenhower.sph.harvard.edu/picrust/STAMPSTAMP Softwarehttps://beikolab.cs.dal.ca/software/STAMPSource codeThis studyhttps://github.com/xuan13hao/Postnatal_gut_microbiota_SIGIRR

Experimental model and study participant detailsThe Sigirr mutant mice mimicking human truncated SIGIRR variant (pY168x) were generated in the genetic background of C57BL/6 using CRISPR-Cas9 technology as previously described.22 Mice were housed under pathogen-free conditions (temperature, 25°C; 12-h light/dark cycle) with food and water available ad libitum. Littermate neonatal mice from heterozygous X heterozygous breeding were randomly allocated to each experimental group. Sex of all mice used were noted and the experimental groups were equally represented with male and female mouse pups. In the results sections and the figures (including supplemental) data from sex-based comparisons is shown. Care of mice before and during experimental procedures was conducted in accordance with the policies at the University of Missouri–Kansas City Laboratory Animal Resource Center and the National Institutes of Health Guidelines for the Care and Use of Laboratory Animals. Protocols had prior approval from the University of Missouri–Kansas City Institutional Animal Care and Use Committee.

Experimental model and study participant details

The Sigirr mutant mice mimicking human truncated SIGIRR variant (pY168x) were generated in the genetic background of C57BL/6 using CRISPR-Cas9 technology as previously described.22 Mice were housed under pathogen-free conditions (temperature, 25°C; 12-h light/dark cycle) with food and water available ad libitum. Littermate neonatal mice from heterozygous X heterozygous breeding were randomly allocated to each experimental group. Sex of all mice used were noted and the experimental groups were equally represented with male and female mouse pups. In the results sections and the figures (including supplemental) data from sex-based comparisons is shown. Care of mice before and during experimental procedures was conducted in accordance with the policies at the University of Missouri–Kansas City Laboratory Animal Resource Center and the National Institutes of Health Guidelines for the Care and Use of Laboratory Animals. Protocols had prior approval from the University of Missouri–Kansas City Institutional Animal Care and Use Committee.

Method detailsFormula feedingFor formula feeding mouse model, litter mate wild type and Sigirr mutant pups were gavage-fed 0.1mL of Esbilac canine milk replacer (PetAg, Inc., Hampshire, IL) five times daily starting at day of life (DOL) 5 to DOL 7. At DOL8, mice were euthanized, and terminal ileum and colon were collected and snap frozen for further microbiome and histological studies.16S rRNA amplicon sequencingTerminal ileum and colon stool contents were collected and immediately frozen and stored at −80°C. Total bacterial genomic DNA was extracted using the QIAmp DNA stool kit (Qiagen) following their instructions. The DNA integrity was assessed by agarose gel electrophoresis; concentration and quality were determined by absorption at A260, and A260/A280 ratio, respectively, using a Nanodrop2000 spectrophotometer (ThermoFisher Scientific). The 16S V4 region was amplified using 515F/806R primers and sequenced using amplicon sequencing on IonS5TMXL to generate raw reads. Paired-end reads were assigned to samples based on their unique barcode and truncated by cutting off the barcode and primer sequences.Analysis of 16S rRNA amplicon sequencing dataThe 16S rRNA amplicon sequencing data underwent processing using DADA2 v1.1,82 generating an Amplicon Sequence Variant (ASV) table. This processing involved quality trimming, filtering, dereplication, error rate estimation, denoising, merging of paired-end reads, screening for mismatches, and chimeric sequence removal. Taxonomic classification was performed using the SILVA reference database (SILVA 132 release).83 Then preprocessing steps were applied using the MicrobiomeAnalyst platform.84 Firstly, we filtered out low-abundance features based on a 20% prevalence filter and set a minimum count threshold of 4. Features with low variance were removed using the interquartile range, employing a cutoff range of 10%. To ensure uniformity across samples, we rarified all samples to the sequencing depth of the sample with the lowest sequence count. Subsequently, we normalized the remaining features using rarefaction to the minimum library size and the total sum scaling (TSS) method. Alpha diversity indices were calculated, and statistical analysis was conducted to assess differences between groups. Beta diversity was explored using Nonmetric Multidimensional Scaling (NMDS) ordination, and differentially abundant taxa were identified using Kruskal-Wallis rank sum tests and Linear Discriminant Analysis Effect Size (LEfSe). Additionally, functional profiling of the metagenome was predicted using Phylogenetic Investigation of Communities by Reconstruction of Unobserved States (PICRUSt2),13 with subsequent analysis of MetaCyc and KEGG pathways using STAMP,85 employing appropriate statistical tests to identify differences in pathway abundances between groups.TUNEL assay for apoptosisThe Terminal deoxynucleotidyl transferase dUTP nick end labeling (TUNEL) assay was conducted on terminal ileum sections according to the guidelines provided by the manufacturer (Promega, Madison, WI). High-resolution images were obtained at using a Nikon W1 confocal microscope equipped with a 20X objective lens. The quantification process involved counting the number of total cells by DAPI staining and those that were TUNEL positive (indicating green fluorescence) in each field. For accuracy, this was performed across a minimum of 3 HPFs for each specimen. Subsequently, the apoptotic index was determined by calculating the ratio of TUNEL-positive cells to the total DAPI-stained cell population.Quantitative real-time PCRTotal RNA was isolated using TRiZol (ThermoFisher). cDNA was synthesized using a cDNA reverse transcription kit (NEB). For the quantification of gene amplification, Real-time PCR was performed using a ViiA7 (Applied Biosystems). The sequences of gene-specific primers are given in supplemental information. RPLP0 or GAPDH were used as endogenous normalization control.ImmunostainingParaffin sections of terminal ileum were cut in 4μm thickness. Sections were deparaffinized with xylene and graded series of alcohol. Antigen retrieval was performed using citrate buffer for 20mins at 95 C. After several rinses with PBS, sections were blocked with Power Block™ Universal Blocking Reagent for one hour and then stained overnight at 4°C with primary Nitrotyrosine Antibody (Santa Cruz 1: 100, sc-32757). After washing with PBS, slides were stained with secondary antibody conjugated with Alexa488 (Invitrogen,1:100) for one hour at room temperature. After treating with secondary antibodies slides were washed several times with PBS, counterstained with 4′6′-diamidino-2-phenylindole (DAPI) and mounted with FLUORO-GEL (Electron Microscopy Science 1798510). Images were acquired using X25/oil objective with a Zeiss Inverted LSM 510 meta laser scanning confocal microscope.

Method details

Formula feedingFor formula feeding mouse model, litter mate wild type and Sigirr mutant pups were gavage-fed 0.1mL of Esbilac canine milk replacer (PetAg, Inc., Hampshire, IL) five times daily starting at day of life (DOL) 5 to DOL 7. At DOL8, mice were euthanized, and terminal ileum and colon were collected and snap frozen for further microbiome and histological studies.

Formula feeding

For formula feeding mouse model, litter mate wild type and Sigirr mutant pups were gavage-fed 0.1mL of Esbilac canine milk replacer (PetAg, Inc., Hampshire, IL) five times daily starting at day of life (DOL) 5 to DOL 7. At DOL8, mice were euthanized, and terminal ileum and colon were collected and snap frozen for further microbiome and histological studies.

16S rRNA amplicon sequencingTerminal ileum and colon stool contents were collected and immediately frozen and stored at −80°C. Total bacterial genomic DNA was extracted using the QIAmp DNA stool kit (Qiagen) following their instructions. The DNA integrity was assessed by agarose gel electrophoresis; concentration and quality were determined by absorption at A260, and A260/A280 ratio, respectively, using a Nanodrop2000 spectrophotometer (ThermoFisher Scientific). The 16S V4 region was amplified using 515F/806R primers and sequenced using amplicon sequencing on IonS5TMXL to generate raw reads. Paired-end reads were assigned to samples based on their unique barcode and truncated by cutting off the barcode and primer sequences.

16S rRNA amplicon sequencing

Terminal ileum and colon stool contents were collected and immediately frozen and stored at −80°C. Total bacterial genomic DNA was extracted using the QIAmp DNA stool kit (Qiagen) following their instructions. The DNA integrity was assessed by agarose gel electrophoresis; concentration and quality were determined by absorption at A260, and A260/A280 ratio, respectively, using a Nanodrop2000 spectrophotometer (ThermoFisher Scientific). The 16S V4 region was amplified using 515F/806R primers and sequenced using amplicon sequencing on IonS5TMXL to generate raw reads. Paired-end reads were assigned to samples based on their unique barcode and truncated by cutting off the barcode and primer sequences.

Analysis of 16S rRNA amplicon sequencing dataThe 16S rRNA amplicon sequencing data underwent processing using DADA2 v1.1,82 generating an Amplicon Sequence Variant (ASV) table. This processing involved quality trimming, filtering, dereplication, error rate estimation, denoising, merging of paired-end reads, screening for mismatches, and chimeric sequence removal. Taxonomic classification was performed using the SILVA reference database (SILVA 132 release).83 Then preprocessing steps were applied using the MicrobiomeAnalyst platform.84 Firstly, we filtered out low-abundance features based on a 20% prevalence filter and set a minimum count threshold of 4. Features with low variance were removed using the interquartile range, employing a cutoff range of 10%. To ensure uniformity across samples, we rarified all samples to the sequencing depth of the sample with the lowest sequence count. Subsequently, we normalized the remaining features using rarefaction to the minimum library size and the total sum scaling (TSS) method. Alpha diversity indices were calculated, and statistical analysis was conducted to assess differences between groups. Beta diversity was explored using Nonmetric Multidimensional Scaling (NMDS) ordination, and differentially abundant taxa were identified using Kruskal-Wallis rank sum tests and Linear Discriminant Analysis Effect Size (LEfSe). Additionally, functional profiling of the metagenome was predicted using Phylogenetic Investigation of Communities by Reconstruction of Unobserved States (PICRUSt2),13 with subsequent analysis of MetaCyc and KEGG pathways using STAMP,85 employing appropriate statistical tests to identify differences in pathway abundances between groups.

Analysis of 16S rRNA amplicon sequencing data

The 16S rRNA amplicon sequencing data underwent processing using DADA2 v1.1,82 generating an Amplicon Sequence Variant (ASV) table. This processing involved quality trimming, filtering, dereplication, error rate estimation, denoising, merging of paired-end reads, screening for mismatches, and chimeric sequence removal. Taxonomic classification was performed using the SILVA reference database (SILVA 132 release).83 Then preprocessing steps were applied using the MicrobiomeAnalyst platform.84 Firstly, we filtered out low-abundance features based on a 20% prevalence filter and set a minimum count threshold of 4. Features with low variance were removed using the interquartile range, employing a cutoff range of 10%. To ensure uniformity across samples, we rarified all samples to the sequencing depth of the sample with the lowest sequence count. Subsequently, we normalized the remaining features using rarefaction to the minimum library size and the total sum scaling (TSS) method. Alpha diversity indices were calculated, and statistical analysis was conducted to assess differences between groups. Beta diversity was explored using Nonmetric Multidimensional Scaling (NMDS) ordination, and differentially abundant taxa were identified using Kruskal-Wallis rank sum tests and Linear Discriminant Analysis Effect Size (LEfSe). Additionally, functional profiling of the metagenome was predicted using Phylogenetic Investigation of Communities by Reconstruction of Unobserved States (PICRUSt2),13 with subsequent analysis of MetaCyc and KEGG pathways using STAMP,85 employing appropriate statistical tests to identify differences in pathway abundances between groups.

TUNEL assay for apoptosisThe Terminal deoxynucleotidyl transferase dUTP nick end labeling (TUNEL) assay was conducted on terminal ileum sections according to the guidelines provided by the manufacturer (Promega, Madison, WI). High-resolution images were obtained at using a Nikon W1 confocal microscope equipped with a 20X objective lens. The quantification process involved counting the number of total cells by DAPI staining and those that were TUNEL positive (indicating green fluorescence) in each field. For accuracy, this was performed across a minimum of 3 HPFs for each specimen. Subsequently, the apoptotic index was determined by calculating the ratio of TUNEL-positive cells to the total DAPI-stained cell population.

TUNEL assay for apoptosis

The Terminal deoxynucleotidyl transferase dUTP nick end labeling (TUNEL) assay was conducted on terminal ileum sections according to the guidelines provided by the manufacturer (Promega, Madison, WI). High-resolution images were obtained at using a Nikon W1 confocal microscope equipped with a 20X objective lens. The quantification process involved counting the number of total cells by DAPI staining and those that were TUNEL positive (indicating green fluorescence) in each field. For accuracy, this was performed across a minimum of 3 HPFs for each specimen. Subsequently, the apoptotic index was determined by calculating the ratio of TUNEL-positive cells to the total DAPI-stained cell population.

Quantitative real-time PCRTotal RNA was isolated using TRiZol (ThermoFisher). cDNA was synthesized using a cDNA reverse transcription kit (NEB). For the quantification of gene amplification, Real-time PCR was performed using a ViiA7 (Applied Biosystems). The sequences of gene-specific primers are given in supplemental information. RPLP0 or GAPDH were used as endogenous normalization control.

Quantitative real-time PCR

Total RNA was isolated using TRiZol (ThermoFisher). cDNA was synthesized using a cDNA reverse transcription kit (NEB). For the quantification of gene amplification, Real-time PCR was performed using a ViiA7 (Applied Biosystems). The sequences of gene-specific primers are given in supplemental information. RPLP0 or GAPDH were used as endogenous normalization control.

ImmunostainingParaffin sections of terminal ileum were cut in 4μm thickness. Sections were deparaffinized with xylene and graded series of alcohol. Antigen retrieval was performed using citrate buffer for 20mins at 95 C. After several rinses with PBS, sections were blocked with Power Block™ Universal Blocking Reagent for one hour and then stained overnight at 4°C with primary Nitrotyrosine Antibody (Santa Cruz 1: 100, sc-32757). After washing with PBS, slides were stained with secondary antibody conjugated with Alexa488 (Invitrogen,1:100) for one hour at room temperature. After treating with secondary antibodies slides were washed several times with PBS, counterstained with 4′6′-diamidino-2-phenylindole (DAPI) and mounted with FLUORO-GEL (Electron Microscopy Science 1798510). Images were acquired using X25/oil objective with a Zeiss Inverted LSM 510 meta laser scanning confocal microscope.

Immunostaining

Paraffin sections of terminal ileum were cut in 4μm thickness. Sections were deparaffinized with xylene and graded series of alcohol. Antigen retrieval was performed using citrate buffer for 20mins at 95 C. After several rinses with PBS, sections were blocked with Power Block™ Universal Blocking Reagent for one hour and then stained overnight at 4°C with primary Nitrotyrosine Antibody (Santa Cruz 1: 100, sc-32757). After washing with PBS, slides were stained with secondary antibody conjugated with Alexa488 (Invitrogen,1:100) for one hour at room temperature. After treating with secondary antibodies slides were washed several times with PBS, counterstained with 4′6′-diamidino-2-phenylindole (DAPI) and mounted with FLUORO-GEL (Electron Microscopy Science 1798510). Images were acquired using X25/oil objective with a Zeiss Inverted LSM 510 meta laser scanning confocal microscope.

Quantification and statistical analysisAnalysis of variance with Bonferroni correction for multiple testing was used for analysis. Statistical analysis was done using GraphPad Prism (San Diego, CA) with statistical significance set at p < 0.05. For microbiome analysis, alpha diversity was assessed using indices, such as Chao1 and Shannon, calculated across samples. Then, differences between groups were evaluated using either a t-test or ANOVA. Beta diversity was explored using NMDS ordination with Bray-Curtis index and Jensen-Shannon divergence at both phylum and genus levels. To test for significance, permutational multivariate analysis of variance (PERMANOVA) was employed. Furthermore, Kruskal-Wallis rank sum tests were conducted on identified genera and phyla to identify differentially abundant taxa between groups. Taxa with raw p-values <0.05 were retained for further analysis, consistent with the methodology applied in our LEfSe analysis. Additionally, LEfSe was utilized to gain insight into differential abundance at each taxonomic level, considering taxa significant if they had p-values <0.05 and LDA values ≥2. For studies on formula milk-induced intestinal inflammation and injury, ANOVA was used with a p < 0.05 considered significant. We used “phyloseq” library in version 1.44 in Rstudio 2024.04.2 Build 764 to contain the microbiome data in a phyloseq object. The R library “shiny” version 1.7.5 was used, in conjunction with “phyloseq” in an interactive web application to visualize the data.86 In this interactive D3 network visualization shown in Figures 6H and 6I, nodes represent stool samples and edges represent relationships. The Bray-Curtis distance of 0.15 and 0.2 is a measure of dissimilarity with measure of 0.2 being more dissimilar than 0.15. The 0.15 threshold will therefore show more separation as the more dissimilar relationships are excluded than the slightly less stringent 0.2 threshold. The tighter the clusters, the more homogenous the samples within a group. The R function “glm” from the “stats” library version 4.3.1 was used for regression analysis of genotypes. The “train” function from the “caret” library version 6.0.94 was used to generate random forest machine learning analysis.87

Quantification and statistical analysis

Analysis of variance with Bonferroni correction for multiple testing was used for analysis. Statistical analysis was done using GraphPad Prism (San Diego, CA) with statistical significance set at p < 0.05. For microbiome analysis, alpha diversity was assessed using indices, such as Chao1 and Shannon, calculated across samples. Then, differences between groups were evaluated using either a t-test or ANOVA. Beta diversity was explored using NMDS ordination with Bray-Curtis index and Jensen-Shannon divergence at both phylum and genus levels. To test for significance, permutational multivariate analysis of variance (PERMANOVA) was employed. Furthermore, Kruskal-Wallis rank sum tests were conducted on identified genera and phyla to identify differentially abundant taxa between groups. Taxa with raw p-values <0.05 were retained for further analysis, consistent with the methodology applied in our LEfSe analysis. Additionally, LEfSe was utilized to gain insight into differential abundance at each taxonomic level, considering taxa significant if they had p-values <0.05 and LDA values ≥2. For studies on formula milk-induced intestinal inflammation and injury, ANOVA was used with a p < 0.05 considered significant. We used “phyloseq” library in version 1.44 in Rstudio 2024.04.2 Build 764 to contain the microbiome data in a phyloseq object. The R library “shiny” version 1.7.5 was used, in conjunction with “phyloseq” in an interactive web application to visualize the data.86 In this interactive D3 network visualization shown in Figures 6H and 6I, nodes represent stool samples and edges represent relationships. The Bray-Curtis distance of 0.15 and 0.2 is a measure of dissimilarity with measure of 0.2 being more dissimilar than 0.15. The 0.15 threshold will therefore show more separation as the more dissimilar relationships are excluded than the slightly less stringent 0.2 threshold. The tighter the clusters, the more homogenous the samples within a group. The R function “glm” from the “stats” library version 4.3.1 was used for regression analysis of genotypes. The “train” function from the “caret” library version 6.0.94 was used to generate random forest machine learning analysis.87
