---
pmid: '23073694'
title: Metabolic and transcriptomic changes induced in Arabidopsis by the rhizobacterium
  Pseudomonas fluorescens SS101.
authors:
- van de Mortel JE
- de Vos RC
- Dekkers E
- Pineda A
- Guillod L
- Bouwmeester K
- van Loon JJ
- Dicke M
- Raaijmakers JM
journal: Plant Physiol
year: '2012'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC3510139
doi: 10.1104/pp.112.207324
---

# Metabolic and transcriptomic changes induced in Arabidopsis by the rhizobacterium Pseudomonas fluorescens SS101.
**Authors:** van de Mortel JE, de Vos RC, Dekkers E, Pineda A, Guillod L, Bouwmeester K, van Loon JJ, Dicke M, Raaijmakers JM
**Journal:** Plant Physiol (2012)
**DOI:** [10.1104/pp.112.207324](https://doi.org/10.1104/pp.112.207324)
**PMC:** [PMC3510139](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3510139/)

## Abstract

1. Plant Physiol. 2012 Dec;160(4):2173-88. doi: 10.1104/pp.112.207324. Epub 2012 
Oct 16.

Metabolic and transcriptomic changes induced in Arabidopsis by the 
rhizobacterium Pseudomonas fluorescens SS101.

van de Mortel JE(1), de Vos RC, Dekkers E, Pineda A, Guillod L, Bouwmeester K, 
van Loon JJ, Dicke M, Raaijmakers JM.

Author information:
(1)Laboratory of Phytopathology , 6708 PB Wageningen, The Netherlands.

Systemic resistance induced in plants by nonpathogenic rhizobacteria is 
typically effective against multiple pathogens. Here, we show that 
root-colonizing Pseudomonas fluorescens strain SS101 (Pf.SS101) enhanced 
resistance in Arabidopsis (Arabidopsis thaliana) against several bacterial 
pathogens, including Pseudomonas syringae pv tomato (Pst) and the insect pest 
Spodoptera exigua. Transcriptomic analysis and bioassays with specific 
Arabidopsis mutants revealed that, unlike many other rhizobacteria, the 
Pf.SS101-induced resistance response to Pst is dependent on salicylic acid 
signaling and not on jasmonic acid and ethylene signaling. Genome-wide 
transcriptomic and untargeted metabolomic analyses showed that in roots and 
leaves of Arabidopsis plants treated with Pf.SS101, approximately 1,910 genes 
and 50 metabolites were differentially regulated relative to untreated plants. 
Integration of both sets of "omics" data pointed to a prominent role of 
camalexin and glucosinolates in the Pf.SS101-induced resistance response. 
Subsequent bioassays with seven Arabidopsis mutants (myb51, cyp79B2cyp79B3, 
cyp81F2, pen2, cyp71A12, cyp71A13, and myb28myb29) disrupted in the biosynthesis 
pathways for these plant secondary metabolites showed that camalexin and 
glucosinolates are indeed required for the induction of Pst resistance by 
Pf.SS101. Also for the insect S. exigua, the indolic glucosinolates appeared to 
play a role in the Pf.SS101-induced resistance response. This study provides, to 
our knowledge for the first time, insight into the substantial biochemical and 
temporal transcriptional changes in Arabidopsis associated with the salicylic 
acid-dependent resistance response induced by specific rhizobacteria.

DOI: 10.1104/pp.112.207324
PMCID: PMC3510139
PMID: 23073694 [Indexed for MEDLINE]

## Full Text

Abstract

Systemic resistance induced in plants by nonpathogenic rhizobacteria is typically effective against multiple pathogens. Here, we show that root-colonizing Pseudomonas fluorescens strain SS101 ( Pf .SS101 ) enhanced resistance in Arabidopsis ( Arabidopsis thaliana ) against several bacterial pathogens, including Pseudomonas syringae pv tomato ( Pst ) and the insect pest Spodoptera exigua . Transcriptomic analysis and bioassays with specific Arabidopsis mutants revealed that, unlike many other rhizobacteria, the Pf .SS101 -induced resistance response to Pst is dependent on salicylic acid signaling and not on jasmonic acid and ethylene signaling. Genome-wide transcriptomic and untargeted metabolomic analyses showed that in roots and leaves of Arabidopsis plants treated with Pf .SS101 , approximately 1,910 genes and 50 metabolites were differentially regulated relative to untreated plants. Integration of both sets of “omics” data pointed to a prominent role of camalexin and glucosinolates in the Pf .SS101 -induced resistance response. Subsequent bioassays with seven Arabidopsis mutants ( myb51 , cyp79B2cyp79B3 , cyp81F2 , pen2 , cyp71A12 , cyp71A13 , and myb28myb29 ) disrupted in the biosynthesis pathways for these plant secondary metabolites showed that camalexin and glucosinolates are indeed required for the induction of Pst resistance by Pf .SS101 . Also for the insect S. exigua , the indolic glucosinolates appeared to play a role in the Pf .SS101 -induced resistance response. This study provides, to our knowledge for the first time, insight into the substantial biochemical and temporal transcriptional changes in Arabidopsis associated with the salicylic acid-dependent resistance response induced by specific rhizobacteria.

DISCUSSION

In this study, we showed that colonization of Arabidopsis roots by the rhizobacterium Pf .SS101 leads to an enhanced level of resistance against various bacterial pathogens and the lepidopteran insect herbivore S. exigua . Our metabolomics and transcriptomics data showed that this specific rhizobacterial strain induces extensive metabolic and transcriptional changes in roots and leaves of Arabidopsis. Integration of the two omics data sets showed that, next to SA signaling, glucosinolates and camalexin play important roles in the resistance response of Arabidopsis induced by Pf .SS101 .

Rhizobacteria-mediated ISR has frequently been shown to be dependent on JA / ET and independent of SA signaling ( Pieterse et al., 1998 ; Ton et al., 2001 ; Yan et al., 2002 ; Van Wees et al., 2008 ; Van der Ent et al., 2009 ; Zamioudis and Pieterse, 2012 ). In contrast to these studies, we showed that the rhizobacterium Pf .SS101 induced resistance in Arabidopsis against Pst via the SA signaling pathway and not via the JA / ET pathways. Van der Ent and colleagues (2008) showed that colonization of Arabidopsis roots by nonpathogenic Pf .WCS417r triggered JA / ET -dependent ISR that is effective against a broad range of pathogens. Their microarray analysis revealed that the R2R3-MYB-like transcription factor gene MYB72 was specifically activated in the roots upon colonization by Pf .WCS417r ( Verhagen et al., 2004 ). They further showed that the T-DNA knockout mutants myb72-1 and myb72-2 were incapable of mounting ISR against the pathogens Pst , H. parasitica , Alternaria brassicicola , and Botrytis cinerea , indicating that MYB72 is essential to establish broad-spectrum ISR ( Van der Ent et al., 2008 ). Here, we showed that Pf .SS101 did not affect the expression of MYB72 and provided protection against Pst in the myb72 knockout mutants to a level similar to that in wild-type Col-0 ( Fig. 2C ). Furthermore, Pf .SS101 was unable to induce resistance against Pst in mutants npr1-1 and npr1-3 ( Fig. 2B ). Mutant npr1-1 is impaired in JA / ET / SA signaling, while mutant npr1-3 is only implicated in SA signaling ( Cao et al., 1994 , 1997 ). Our analyses further showed significantly higher SA (free and bound) concentrations in roots of Pf .SS101 -treated plants. In leaves, however, no differences in SA concentrations were detected between Pf .SS101 -treated plants and nontreated plants ( Fig. 2D ). It is not yet known if and how the elevated SA concentrations in the roots of Pf .SS101 -treated plants trigger the enhanced resistance to Pst in the leaves. Recently, Dempsey and Klessig (2012) suggested that a diverse group of compounds, including methylsalicylate, azelaic acid, the abietane diterpenoid dehydroabietinal, JA , and the amino acid derivative pipecolic acid, may act as mobile signals. Whether these signals play a role in Pf .SS101 -induced resistance remains to be investigated. Altogether, we conclude that Pf .SS101 induces resistance dependent on SA signaling, thus differing from the signaling pathways induced by Pf .WCS417r and various other rhizobacterial strains.

Recently, Weston et al. (2012) used an integrated transcriptome and metabolome analysis to study the response of Arabidopsis to P. fluorescens strains GM30 and Pf-5. They showed a strain-specific response, with GM30-treated plants showing overrepresentation in genes for glucosinolate synthesis, abscisic acid, and gibberellin metabolism. Plants treated with Pf-5 showed enhanced expression of genes contributing to Trp degradation, auxin metabolism, and JA metabolism. When we compare the transcriptional changes induced by GM30 and Pf-5 with those induced by Pf .SS101 , 442 genes were in common. In contrast, only 16 to 64 of the genes were shared between Pf .SS101 and other rhizobacteria ( Supplemental Fig. S1 ; Supplemental Table S9 ; Cartieaux et al., 2003 , 2008 ; Verhagen et al., 2004 ; Wang et al., 2005 ; Weston et al., 2012 ). From these data, we conclude that the signal transduction pathways and metabolic changes induced in Arabidopsis by Pf .SS101 do not resemble those induced by many other rhizobacteria studied to date.

Our results further showed that many of the defense-related genes that were up-regulated at 10 and 14 DPI in roots and leaves of Pf .SS101 -treated Arabidopsis plants ( Supplemental Tables S1 and S2 ) have also been described previously for pathogen-associated molecular pattern ( PAMP )-triggered immunity ( Thilmony et al., 2006 ). Expression analysis by Thilmony et al. (2006) of Arabidopsis plants treated with only avirulent Pst resulted in the identification of 735 PAMP -responsive genes, with 389 genes repressed and 346 genes induced. Of the 389 genes that were repressed upon PAMP perception, only 25 genes were repressed in the leaves of Arabidopsis plants treated with Pf .SS101 . These results indicate that there is little overlap between the genes repressed by avirulent pathogens and by Pf .SS101 . However, of the 346 genes that were induced upon PAMP perception by avirulent Pst , 195 genes were also induced in Arabidopsis leaves by Pf .SS101 ( Supplemental Fig. S2A ; Supplemental Table S10 ). When comparing the genes differentially expressed in Arabidopsis leaves by Pf .SS101 and virulent Pst , 134 genes are in common ( Supplemental Fig. S2B ; Supplemental Table S10 ). Of these 134 genes, 87 genes were induced and 47 genes were repressed by virulent Pst ( Supplemental Fig. S2B ; Supplemental Table S10 ). Thilmony et al. (2006) found that a total of 201 of the 346 PAMP genes induced by avirulent Pst were either repressed or not differentially regulated in response to virulent Pst . Based on these results, they postulated that these 201 genes may play a role in the basal defense response of Arabidopsis. More than one-half (108) of these 201 genes were significantly up-regulated in Arabidopsis leaves by Pf .SS101 at 10 or 14 DPI . These 108 genes may be considered as potential candidates to further unravel the genetic basis of the defense response induced in Arabidopsis by Pf .SS101 .

Many of the metabolic changes induced in Arabidopsis by Pf .SS101 are also similar to those induced in Arabidopsis leaves by avirulent Pst-AvrRPM1 : more SA , camalexin, indole-3-carboxylic acid β- d -glucopyranosyl, and scopoletin accumulated in Pst-AvrRPM1- challenged leaves than in uninfected adjacent areas ( Hagemeier et al., 2001 ; Simon et al., 2010 ). Additionally, Pf .SS101 induced the biosynthesis of indolic and aliphatic glucosinolates. Glucosinolates are well-studied plant metabolites with a key role in plant defense against microbial pathogens ( Clay et al., 2009 ) and generalist herbivores from different feeding guilds (i.e. leaf chewers and phloem feeders; Mewis et al., 2006 ; Gols et al., 2008 ; Kim et al., 2008 ; Müller et al., 2010 ). Here, we show that glucosinolates are also an important component of the plant response to nonpathogenic, root-colonizing bacteria, which in turn affects resistance against pathogens and generalist herbivores. Furthermore, Pf .SS101 also induced the biosynthesis of various other metabolites not reported previously for Arabidopsis ( Tohge et al., 2005 ; Malitsky et al., 2008; Albinsky et al., 2010 ;). Structural analysis of these unknown compounds and analysis of their activity spectrum have been initiated to resolve their functions in ISR .

The bacterial determinants involved in Pf .SS101 -induced resistance in Arabidopsis remain unknown. Preliminary analyses indicated that massetolide A, the lipopeptide surfactant that contributes to induced systemic resistance in tomato against P. infestans ( Tran et al., 2007 ), is not involved in the induced resistance response in Arabidopsis against Pst . This preliminary conclusion was based on assays with the massA mutant and purified massetolide A. Zamioudis and Pieterse (2012) indicated that beneficial microbes may be initially perceived by plants as potential invaders, whereas at later stages of the interaction the beneficial microbes “short-circuit plant defense responses to enable successful colonization of host roots.” A possible scenario is that microbe-associated molecular patterns of Pf .SS101 and PAMPs of pathogens share specific structural features or are recognized in a similar manner, ultimately resulting in an enhanced defensive capacity of the plant. One potential candidate in this context is flagellin. Flg22 is a 22-amino acid polypeptide corresponding to a highly conserved epitope of the Pseudomonas aeruginosa flagellin protein ( Felix et al., 1999 ) and is widely used as a proxy of flagellin-mediated signaling in Arabidopsis. In the genome of Pf .SS101 ( Loper et al., 2012 ), we found a protein encoding a flagellin domain protein (PflSS101_3776; TRLSSGLKINSAKDDAAGMQIA) with an amino acid sequence similar to Flg22 in the N terminus (residues 30–51). This sequence has 95% identity to the Pst Flg22 sequence, with a difference of only one amino acid, and 80% identity to the P. aeruginosa Flg22 sequence, with a difference of four amino acids ( Supplemental Fig. S3 ). Flagellin perception in Arabidopsis functions via the receptor-like kinase Flagellin-Sensing2 (FLS2; Zipfel et al., 2004 ). Hence, we investigated if FLS2 and also the EF-Tu Receptor (EFR) are involved in the resistance response induced by Pf .SS101 . Stimulation of these receptors through their cognate ligands (i.e. bacterial flagellin or elongation factor Tu) leads to a defense response and increased resistance. The results of our bioassays showed that treatment of the mutants fls2 and efr with Pf .SS101 led to a significant ( P < 0.05) reduction in disease incidence to a similar level as observed for wild-type Arabidopsis Col-0 ( Supplemental Fig. S4A ). These results suggest that Pf .SS101 does not require FLS2 and EFR to induce resistance against Pst in Arabidopsis. To further support these results, we also studied callose deposition in Pf .SS101 -treated plants. Flg22-elicited callose deposition in Arabidopsis cotyledons is dependent on the biosynthesis of indol-3-ylmethylglucosinolate, which in turn is dependent on the transcription factor MYB51 ( Clay et al., 2009 ). In our bioassays, we observed that root tip inoculation with Pf .SS101 did not cause callose deposition in Arabidopsis leaves ( Supplemental Fig. S4B ). Furthermore, callose deposition was also not induced in leaves when Pf .SS101 -treated Arabidopsis plants were challenged with Pst ( Supplemental Fig. S4B ) or induced in roots of Arabidopsis treated with Pf .SS101 (data not shown).

In silico analysis of the genome sequence of Pf .SS101 showed the presence of a type III secretion system ( TTSS ; Loper et al., 2012 ). TTSS is known to function in the delivery of effector molecules into the plant and plays an important role in the pathogenicity/virulence of bacterial pathogens. For P. fluorescens strain SBW25, the tts cluster harbors several open reading frames that share a significant degree of similarity with the hrc/hrp genes of pathogenic bacteria ( Preston et al., 2001 ). Also, the TTSS of Pf .SS101 resembles the hrc/hrp genes of P. syringae and carries a rhizosphere-expressed secretion protein and an rsp-conserved ( rsp / rsc ) gene cluster ( Loper et al., 2012 ). Furthermore, the Pf .SS101 genome harbors 12 additional genes preceded by putative Hrp(Rsp)L-dependent promoters and encoding proteins with compositions typical of type III effectors. The potential role of these putative novel effectors in induced systemic resistance by Pf .SS101 is the subject of future studies.
