---
pmid: '16484192'
title: Global analysis of heat shock response in Desulfovibrio vulgaris Hildenborough.
authors:
- Chhabra SR
- He Q
- Huang KH
- Gaucher SP
- Alm EJ
- He Z
- Hadi MZ
- Hazen TC
- Wall JD
- Zhou J
- Arkin AP
- Singh AK
journal: J Bacteriol
year: '2006'
full_text_available: false
pmcid: PMC1426554
doi: 10.1128/JB.188.5.1817-1828.2006
---

# Global analysis of heat shock response in Desulfovibrio vulgaris Hildenborough.
**Authors:** Chhabra SR, He Q, Huang KH, Gaucher SP, Alm EJ, He Z, Hadi MZ, Hazen TC, Wall JD, Zhou J, Arkin AP, Singh AK
**Journal:** J Bacteriol (2006)
**DOI:** [10.1128/JB.188.5.1817-1828.2006](https://doi.org/10.1128/JB.188.5.1817-1828.2006)
**PMC:** [PMC1426554](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1426554/)

## Abstract

1. J Bacteriol. 2006 Mar;188(5):1817-28. doi: 10.1128/JB.188.5.1817-1828.2006.

Global analysis of heat shock response in Desulfovibrio vulgaris Hildenborough.

Chhabra SR(1), He Q, Huang KH, Gaucher SP, Alm EJ, He Z, Hadi MZ, Hazen TC, Wall 
JD, Zhou J, Arkin AP, Singh AK.

Author information:
(1)Biosystems Research Department, Mailstop 9292, Sandia National Laboratory, 
7011 East Ave., Livermore, CA 94550, USA.

Desulfovibrio vulgaris Hildenborough belongs to a class of sulfate-reducing 
bacteria (SRB) and is found ubiquitously in nature. Given the importance of 
SRB-mediated reduction for bioremediation of metal ion contaminants, ongoing 
research on D. vulgaris has been in the direction of elucidating regulatory 
mechanisms for this organism under a variety of stress conditions. This work 
presents a global view of this organism's response to elevated growth 
temperature using whole-cell transcriptomics and proteomics tools. 
Transcriptional response (1.7-fold change or greater; Z >/= 1.5) ranged from 
1,135 genes at 15 min to 1,463 genes at 120 min for a temperature up-shift of 13 
degrees C from a growth temperature of 37 degrees C for this organism and 
suggested both direct and indirect modes of heat sensing. Clusters of 
orthologous group categories that were significantly affected included 
posttranslational modifications; protein turnover and chaperones (up-regulated); 
energy production and conversion (down-regulated), nucleotide transport, 
metabolism (down-regulated), and translation; ribosomal structure; and 
biogenesis (down-regulated). Analysis of the genome sequence revealed the 
presence of features of both negative and positive regulation which included the 
CIRCE element and promoter sequences corresponding to the alternate sigma 
factors sigma(32) and sigma(54). While mechanisms of heat shock control for some 
genes appeared to coincide with those established for Escherichia coli and 
Bacillus subtilis, the presence of unique control schemes for several other 
genes was also evident. Analysis of protein expression levels using differential 
in-gel electrophoresis suggested good agreement with transcriptional profiles of 
several heat shock proteins, including DnaK (DVU0811), HtpG (DVU2643), HtrA 
(DVU1468), and AhpC (DVU2247). The proteomics study also suggested the 
possibility of posttranslational modifications in the chaperones DnaK, AhpC, 
GroES (DVU1977), and GroEL (DVU1976) and also several periplasmic ABC 
transporters.

DOI: 10.1128/JB.188.5.1817-1828.2006
PMCID: PMC1426554
PMID: 16484192 [Indexed for MEDLINE]
