---
pmid: '23135403'
title: Mitochondrial Atpif1 regulates haem synthesis in developing erythroblasts.
authors:
- Shah DI
- Takahashi-Makise N
- Cooney JD
- Li L
- Schultz IJ
- Pierce EL
- Narla A
- Seguin A
- Hattangadi SM
- Medlock AE
- Langer NB
- Dailey TA
- Hurst SN
- Faccenda D
- Wiwczar JM
- Heggers SK
- Vogin G
- Chen W
- Chen C
- Campagna DR
- Brugnara C
- Zhou Y
- Ebert BL
- Danial NN
- Fleming MD
- Ward DM
- Campanella M
- Dailey HA
- Kaplan J
- Paw BH
journal: Nature
year: '2012'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC3504625
doi: 10.1038/nature11536
---

# Mitochondrial Atpif1 regulates haem synthesis in developing erythroblasts.
**Authors:** Shah DI, Takahashi-Makise N, Cooney JD, Li L, Schultz IJ, Pierce EL, Narla A, Seguin A, Hattangadi SM, Medlock AE, Langer NB, Dailey TA, Hurst SN, Faccenda D, Wiwczar JM, Heggers SK, Vogin G, Chen W, Chen C, Campagna DR, Brugnara C, Zhou Y, Ebert BL, Danial NN, Fleming MD, Ward DM, Campanella M, Dailey HA, Kaplan J, Paw BH
**Journal:** Nature (2012)
**DOI:** [10.1038/nature11536](https://doi.org/10.1038/nature11536)
**PMC:** [PMC3504625](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3504625/)

## Abstract

1. Nature. 2012 Nov 22;491(7425):608-12. doi: 10.1038/nature11536. Epub 2012 Nov
7.

Mitochondrial Atpif1 regulates haem synthesis in developing erythroblasts.

Shah DI(1), Takahashi-Makise N, Cooney JD, Li L, Schultz IJ, Pierce EL, Narla A, 
Seguin A, Hattangadi SM, Medlock AE, Langer NB, Dailey TA, Hurst SN, Faccenda D, 
Wiwczar JM, Heggers SK, Vogin G, Chen W, Chen C, Campagna DR, Brugnara C, Zhou 
Y, Ebert BL, Danial NN, Fleming MD, Ward DM, Campanella M, Dailey HA, Kaplan J, 
Paw BH.

Author information:
(1)Department of Medicine, Division of Hematology, Brigham and Women's Hospital, 
Harvard Medical School, Boston, Massachusetts 02115, USA.

Erratum in
    Nature. 2013 Apr 18;496(7445):386.

Defects in the availability of haem substrates or the catalytic activity of the 
terminal enzyme in haem biosynthesis, ferrochelatase (Fech), impair haem 
synthesis and thus cause human congenital anaemias. The interdependent functions 
of regulators of mitochondrial homeostasis and enzymes responsible for haem 
synthesis are largely unknown. To investigate this we used zebrafish genetic 
screens and cloned mitochondrial ATPase inhibitory factor 1 (atpif1) from a 
zebrafish mutant with profound anaemia, pinotage (pnt (tq209)). Here we describe 
a direct mechanism establishing that Atpif1 regulates the catalytic efficiency 
of vertebrate Fech to synthesize haem. The loss of Atpif1 impairs haemoglobin 
synthesis in zebrafish, mouse and human haematopoietic models as a consequence 
of diminished Fech activity and elevated mitochondrial pH. To understand the 
relationship between mitochondrial pH, redox potential, [2Fe-2S] clusters and 
Fech activity, we used genetic complementation studies of Fech constructs with 
or without [2Fe-2S] clusters in pnt, as well as pharmacological agents 
modulating mitochondrial pH and redox potential. The presence of [2Fe-2S] 
cluster renders vertebrate Fech vulnerable to perturbations in Atpif1-regulated 
mitochondrial pH and redox potential. Therefore, Atpif1 deficiency reduces the 
efficiency of vertebrate Fech to synthesize haem, resulting in anaemia. The 
identification of mitochondrial Atpif1 as a regulator of haem synthesis advances 
our understanding of the mechanisms regulating mitochondrial haem homeostasis 
and red blood cell development. An ATPIF1 deficiency may contribute to important 
human diseases, such as congenital sideroblastic anaemias and 
mitochondriopathies.

DOI: 10.1038/nature11536
PMCID: PMC3504625
PMID: 23135403 [Indexed for MEDLINE]

Conflict of interest statement: Author InformationThe authors declare no 
competing financial interests

## Full Text

A deficiency in heme, which is used in a wide variety of metabolic and regulatory pathways in cells3, results in pathological conditions that range from mild anemia to early in utero death4. As an essential component of hemoglobin, the individual enzymes and substrates of heme biosynthesis have been well studied2; however, key gaps remain in our knowledge of genes that regulate iron and heme trafficking and homeostasis. This incomplete understanding prevents researchers from developing targeted therapies for a broad range of disorders, including congenital anemias and porphyrias, as well as metabolic and neurological disorders.

We recovered pnt, a zebrafish non-lethal recessive mutant, from an unbiased ethyl nitrosourea (ENU) mutagenesis screen5 for defects in circulating erythroid cells6. pnt embryos were anemic (Fig. 1a) despite normal expression of erythroid cell markers, β-globin and band-3 (data not shown). Based on red cell indices, the erythrocytes from pnt embryos that survive to adult stage exhibited hypochromic, microcytic anemia (Supplementary Fig. 1a). Histological analysis of adult pnt hematopoietic tissues, showed no gross morphological defects (Supplementary Fig. 1b).

The positional cloning and chromosomal walk identified the most telomeric gene, the mitochondrial ATPase inhibitory factor 1 (atpif1a; zgc:162207), as the most likely candidate for the pnt locus (Fig. 1b). Phylogenetic analysis showed that an ATPase inhibitory factor 1-like protein (atpif1b; zgc: 153321) on Chr. 17 is 71% similar to atpif1a at the amino acid level (Fig. 1d), and is likely the result of gene duplication in teleosts7. Peptide alignments further displayed human (ATPIF1, chromosome 1) and mouse (Atpif1, chromosome 4) homologs for atpif1a and atpif1b (Fig. 1c). Quantitative reverse transcriptase-polymerase chain reaction (qRT-PCR) showed reduced levels of atpif1a mRNA in pnt embryos (Fig. 1d) and pnt adult kidney marrow compared to respective wild-type (WT) controls (Supplementary Fig. 1c). The levels of atpif1b mRNA were, however, unchanged in pnt embryos (Fig. 1d) and elevated 2 to 3-fold in pnt adult kidney marrow (Supplementary Fig. 1c). Thus, atpif1a is likely the gene disrupted in the pnt locus. Previous studies have shown that mitochondrial Atpif1 regulates the proton motive force via mitochondrial influx of H+ ions, mitochondrial structure, and ATP synthesis, indicating that Atpif1 is required in a wide range of metabolically active tissues8. The broad requirement for Atpif1 is reinforced by the ubiquitous expression of both atpif1a and atpif1b in zebrafish embryos (Supplementary Fig. 1d), and Atpif1 in various mouse adult and fetal organs (Supplementary Fig. 1e).

To verify the loss-of-function phenotype for atpif1a, we injected two atpif1a antisense morpholinos (MO), a splice-blocking (Fig. 2a) and a translational-blocking (data not shown), to knock down atpif1a expression in zebrafish embryos. The atpif1a-silenced embryos (morphants) lacked hemoglobinized cells, as detected by o-dianisidine staining, thereby phenocopying the anemia in pnt embryos (Fig. 2a). The anemic phenotype in the morphant embryos correlates with a reduction of atpif1a mRNA levels, verifying that the splice-blocking MO accurately targeted atpif1a (Fig. 2b, Supplementary Discussion 1, Supplementary Figs. 2a–2d).

To further validate that atpif1a is the gene disrupted in pnt, we over-expressed atpif1a cRNA in pnt embryos and subsequently evaluated their hemoglobinization. To assess the specificity of our complementation assay, we also injected pnt embryos with non-functional atpif1a cRNA harboring a mis-sense mutation (E26A) in the regulatory inhibitory domain9. Only functional atpif1a complemented the anemia in pnt embryos (Fig. 2c), thereby confirming that atpif1a is the gene disrupted in pnt embryos. Consistent with 2 to 3-fold increase in atpif1b expression in surviving pnt adults (Supplementary Fig. 1c), the ectopic-expression of atpif1b cRNA also complemented pnt anemia (Fig. 2c), a result of their redundant function. These data suggest that the hypomorphic and viable phenotype of pnt could be attributed to the ability of atpif1b to partially compensate for the loss of function of atpif1a (Supplementary Fig. 1c). This is in contrast to other situations with paralogous genes such as the mitoferrin (mfrn) transporters10.

To identify a genetic mutation in the pnt locus, we first sequenced the open reading frame (ORF) and splice junctions of the atpif1a gene and found no mutation. Instead, a polymorphism in the 3′ untranslated region (UTR) of the atpif1a gene was discovered (Fig. 2d). This mutation was linked to the pnt locus following the distinct segregation of WT and mutant (MT) conformations on single-strand conformational polymorphism (SSCP) (Fig. 2e). To establish the functional consequences of the 3′UTR polymorphism on the stability of atpif1a mRNA, we designed constructs containing the zebrafish atpif1a ORF fused with either the WT or polymorphic 3′UTR sequences, and transfected mouse erythroleukemia (MEL) cells with each construct individually for stable selection (Supplementary Fig. 2e). qRT-PCR analysis showed a decrease in steady state atpif1a mRNA levels in cells harboring polymorphic 3′UTR constructs as compared to the WT construct (Fig. 2f, Supplementary Discussion 2). These data functionally demonstrate that the polymorphism in the 3′UTR of the atpif1a gene destabilizes its steady state mRNA. The observed decrease in the atpif1a mRNA level in pnt embryos (Fig. 1d) is consistent with the ascribed function for the 3′UTR polymorphism, which destabilizes atpif1a steady state mRNA and leads to its degradation 11.

In mammalian cells, Atpif1 is located in the inner mitochondrial membrane and primarily regulates the function of the F1F0-ATP synthase12. There is, however, no direct evidence suggesting a role for Atpif1 in heme synthesis or erythropoiesis. To ascertain the role of Atpif1 in mammalian heme synthesis, we utilized short hairpin RNAs (shRNAs) to stably silence Atpif1 in differentiating mammalian erythroid cells: human primary CD34+ (hCD34+) cells, mouse primary fetal liver (MPFL), and MEL cells. The reduced number of hemoglobinized cells in Atpif1-silenced hCD34+ and MEL cells (Fig. 3a, Supplementary Fig. 3a) and the reduced hemoglobin content in MPFL cells transduced with Atpif1-shRNAs (Supplementary Figs. 3b–3c), demonstrate conserved heme-specific function of Atpif1. Consistent with a high turnover of heme in developing erythroblasts13, we observed that the hemoglobinization defect was specific to Atpif1-silenced differentiating erythroid cells. Similarly, we observed that the expression of Atpif1 protein increases along with other proteins required for heme synthesis14, such as Fech and mitoferrin1 (Mfrn1, Slc25a37), during terminal erythroid maturation (Supplementary Fig. 3d). Analogous Atpif1-silencing experiments performed in non-erythroid (NIH3T3) and undifferentiated MEL cells showed no measurable heme defects (Supplementary Fig. 4). This indicates that Atpif1 is essential for terminal erythroid maturation and is not critical for heme synthesis in non-erythroid cells and early erythroid progenitors.

Under physiological conditions, the F1F0-ATP synthase generates a proton motive force and ATP by transporting H+ ions into the mitochondria8, 15. Under oxidative stress, the F1F0-ATP synthase reverses the direction of H+ transport, thus reducing ATP and proton motive force8, 15. Atpif1 inhibits the reversal of F1F0-ATP synthase to reduce ATP hydrolysis, thereby maintaining the mitochondrial membrane potential and proton motive force8. To determine the causal association between the roles of Atpif1 in mitochondrial homeostasis and mammalian heme synthesis, we used stable Atpif1-shRNA-silenced MEL cells. Consistent with the silencing of Atpif1, the Atpif1 protein levels were reduced (Fig. 3b). While excluding the possibility of global mitochondrial structural dysfunction: (1) We found normal protein expression of integral mitochondrial proteins, such as the β-subunit of ATP synthase (AtpB), complex IV (Cox IV), voltage-dependent anionic-selective channel protein 1 (Vdac1), and heat shock protein 60 (Hsp60) (Fig. 3b). (2) Electron microscopic and biochemical analysis revealed normal integrity of the mitochondria; indicating that the gross structural architecture of mitochondria in Atpif1-silenced MEL cells was preserved (Fig. 3b, Supplementary Fig. 5). We, however, found that the loss of Atpif1 in differentiating MEL cells caused a depletion of cellular ATP levels (Supplementary Figs. 6a–6b), an increase in mitochondrial membrane potential (ΔΨm) (Fig. 3c, Supplementary Fig. 6c), and an alkalinization of the mitochondria (Supplementary Fig. 6d).

Heme synthesis requires the incorporation of ferrous ion (Fe2+) into protoporphyrin IX (PPIX), which is catalyzed by Fech on the matrix side of the inner mitochondrial membrane16. Since the loss of Atpif1 results primarily in a defect of heme synthesis in differentiating erythroid cells, we analyzed the mitochondrial levels of heme substrates, heme, and their enzymes. We have previously shown that Mfrn1, an inner mitochondrial membrane transporter, facilitates the import of Fe into erythroid mitochondria10; which is dependent upon the mitochondrial ΔΨm17,18. Consistent with the increased mitochondrial ΔΨm in Atpif1-silenced MEL cells (Fig. 3c, Supplementary Fig. 6c), the uptake of radio-labeled iron (59Fe) into the mitochondria was increased (Fig. 3d), indicating that the loss of Atpif1 increases mitochondrial iron load. Iron transported into the mitochondrial matrix is either: 1) utilized to make [2Fe-2S] clusters, or 2) incorporated into PPIX by Fech to generate heme1, 2, 3, 13. Atpif1-silenced MEL cells had normal levels of mitochondrial PPIX (Fig. 3e), suggesting that enzymes and substrates necessary to make this heme precursor are not rate-limiting. The incorporation of 59Fe into PPIX to generate heme, however, was reduced (Fig. 3f), despite having sufficient levels of Fech substrates, iron and PPIX. We predicted that the catalytic efficiency of Fech might have been compromised, and thus, we measured the protein expression levels of Fech and Fech activity. Although normal levels of Fech protein were present, Fech activity was reduced (Fig. 4a), suggesting that the defect in heme synthesis in pnt could be attributed to a reduction in Fech activity. We found that the basal mitochondrial pH of control cells was pH 7.4, and for Atpif1-silenced MEL cells was pH 8.6 (Fig. 4b, Supplementary Fig. 6d). Since Atpif1-silenced MEL cells have higher basal mitochondrial pH, we predicted that (1) elevated mitochondrial pH might influence the efficiency of Fech to make heme, and (2) the Fech isolated from these cells is already harmfully influenced by the elevated pH. To experimentally support our hypothesis, we pre-incubated the purified human Fech at pH 7.4, 8.0, 8.5, and 9.0 for 1 hr, and then run Fech assay at pH 7.4. We found that pre-incubation at elevated pH have harmful impact on human Fech, and thus reduces its capacity to make heme (Supplementary Fig. 6e). In addition, we found that increasing mitochondrial pH to 8.5 and pH 9.0 would further lower Fech activity in both control and Atpif1-silenced MEL cells (Fig. 4c), indicating the Fech activity is a function of mitochondrial pH (Supplementary Discussion 3). Altogether, these data suggest that the ability of Fech to generate heme was compromised due to the increase in mitochondrial pH that occurs in Atpif1-silenced differentiating MEL cells.

We further found that the lowering mitochondrial membrane potential and thus mitochondrial pH, using pharmacological agents carbonyl cyanide 4-(trifluoromethoxy)phenylhydrazone (FCCP; 1 mM; 1hr) and 2, 4-dintro-phenol (2,4-DNP; 500 μM; 1hr)19, could rescue anemic phenotype in Atpif1-silencd MEL cells (Fig. 4d). These data reinforce our observation that Atpif1 regulates heme synthesis via its modulation of the mitochondrial Ψm and mitochondrial pH.

Atpif1-silenced MEL cells have normal cytosolic and mitochondrial [2Fe-2S] cluster biogenesis as they have normal activities of proteins dependent on [2Fe-2S] cluster synthesis, such as cytosolic xanthine oxidase (Supplementary Fig. 7a), and mitochondrial aconitase (Fig. 4e). The post-translational stability of Fech apoprotein depends on iron availability and an intact [2Fe-2S] cluster assembly machinery20, 21. However, the functional role of the [2Fe-2S] cluster in Fech activity remains unknown (Supplementary Discussion 4). All vertebrate Fech enzymes possess [2Fe-2S] clusters, whereas Fech from the yeast Saccharomyces cerevisiae does not have a cluster16, 21. Based on these data and our current observations, we hypothesized that the [2Fe-2S] cluster of Fech could influence Fech activity in situ by rendering Fech responsive to changes in the mitochondrial proton motive force, and thus mitochondrial pH and redox potential.

To experimentally test whether [2Fe-2S] cluster-bound Fech responded to the capacity of Atpif1 to regulate mitochondrial proton motive force and thus mitochondrial pH, we designed constructs containing either zebrafish Fech (with [2Fe-2S]) or yeast Fech (without [2Fe-2S]) with the vertebrate mitochondrial targeting pro-peptide for cRNA injections. Both constructs, irrespective of the presence of [2Fe-2S] clusters, functionally complemented a genetic deficiency for zebrafish Fech in freixenet6, 22 (frxtu271,
Supplementary Fig. 7b). When either zebrafish Fech or yeast Fech cRNA was injected into pnt embryos, the yeast Fech rescued pnt anemia, while zebrafish Fech did not (Fig. 4f). These data suggest that it is the [2Fe-2S] cluster of the zebrafish Fech that makes this enzyme vulnerable to pH perturbations of the mitochondrial matrix in atpif1-deficient pnt embryos. Therefore, Atpif1 plays an essential role in stabilizing the activity of [2Fe-2S] cluster-containing Fech by modulating changes in mitochondrial matrix pH. Based upon our results, we infer that yeast Fech, lacking the [2Fe-2S] cluster, is resistant to mitochondrial pH changes in pnt embryos. Consistent with our observation that the activity of yeast Fech would be unaffected by mitochondrial pH (Fig. 4f), we found that the deficiency of Inh1 did not produce a defect in heme synthesis (data not shown). These data show that Atpif1 regulates heme synthesis in higher vertebrate erythroid tissues, while its counterpart in baker’s yeast, Inh1, does not regulate heme synthesis.

The redox potential of mitochondrial matrix at pH 7.4 is -280 mV, and it is reduced to -370 mV at pH 8.423. Since the mitochondrial pH is inversely related to mitochondrial redox potential23, we further analyzed the susceptibility of the [2Fe-2S] of Fech to reduction in mitochondrial redox potential. When we treated human Fech (with [2Fe-2S]) and yeast Fech (without [2Fe-2S]) with a strong reducing agent, sodium dithionate (DTN, 10 mM), we found that the DTN treatment could only reduce the activity of human Fech and not the activity of yeast Fech (Fig. 4g). This further suggests that [2Fe-2S] of vertebrate Fech is sensitive to the reduction in the redox potential. Thus, the Fech activity is reduced in Atpif1-silenced MEL cells as its [2Fe-2S] cluster is sensitive to an elevation of pH (Fig. 4b, 4c, 4d, 4f) and a reduction in redox potential (Fig. 4g), and not due to a defect in [2Fe-2S] cluster biosynthetic assembly (Fig. 4e, Supplementary Fig. 7a).

To exclude an anemic phenotype in pnt as a causal effect of reactive oxygen species (ROS) induced heme degradation, we treated pnt mutant embryos with N-acetyl cysteine (NAC, 10 μM, 48 hr), a potent antioxidant employed for ROS studies in zebrafish embryos24, 25, and found that NAC could not rescue pnt anemia (Fig. 4h). This demonstrates that anemic phenotype in pnt is not due to increased ROS generation. The heme-oxygenase-1 enzyme (Hmox1) is a ROS-responsive, cellular enzyme responsible for heme degradation. We found normal transcript levels of Hmox1 in control and Atpif1-silenced MEL cells (Supplementary Fig. 7c). Consistent with our model, it has been demonstrated that 1) only overexpression of Atpif1, not loss of Atpif1 activity, induces ROS production, independent of the Hif1α pathway26, and 2) the overexpression of Atpif1 did not affect cellular hydrogen peroxide levels, indicating that the Atpif1-mediated superoxide signal is of mild-intensity and localized in mitochondria26. Altogether, these data suggest that ROS and heme degradation are not responsible for the heme defect we observed in erythroid cells.

In summary, Atpif1 modulates the mitochondrial pH and redox potential, and thus allows Fech to efficiently catalyze the incorporation of iron into PPIX to produce heme. Loss of Atpif1 allows the mitochondrial pH and, consequently, the redox potential to change to a level that reduces [2Fe-2S] cluster-containing Fech activity, thereby reducing heme synthesis, resulting in hypochromic anemia in pnt (Fig. 4i). Previous human studies have identified ALAS2, FXN, ABCB7, and SLC25A38 as the genetic causes of congenital sideroblastic anemias27, 28. Data presented herein demonstrate that Atpif1 should now be added to the list of proteins whose native function is required for normal erythropoiesis. This makes Atpif1 a candidate gene for the understanding and treatment of human sideroblastic anemias and mitochondrial disorders.

METHODS SUMMARYAll procedures were approved by the Animal Care and Use Committee of Boston Children’s Hospital. We performed complete blood count and Wright-Geimsa analyses on peripheral blood recovered from adult pnt. Genetic mapping and positional cloning were utilized to identify zgc: 162207 (atpif1a) as the candidate gene for the pnt locus on zebrafish Chr. 19. We employed qRT-PCR using TaqMan gene expression assays (Applied Biosystems, Carlsbad, CA) to measure levels of atpif1a and atpif1b mRNA. Morpholinos (Gene Tools, Philomath, OR) against splice-site of atpif1a and atpif1b were designed and injected in WT embryos to verify loss-of-function phenotype. The cRNA for atpif1a, atpif1a-E26A, and atpif1b were injected in pnt embryos for complementation. The cDNA prepared from WT and pnt embryos was sequenced, and the polymorphism in the 3′UTR of the atpif1a sequence was verified using SSCP gels.We silenced Atpif1 in hCD34+, MPFL and MEL using shRNAs. The Atpif1-silenced, differentiated hCD34+ and MEL cells were stained with o-dianisidine to measure hemoglobinized cells, while MPFL cells were treated with Drabkin’s reagent to measure relative hemoglobin content. The loss of Atpif1 protein and the state of mitochondrial structural proteins in MEL cells were verified using western and electron microscopic analyses. We analyzed fluorescent intensities of TMRE as a function of mitochondrial membrane potential, Mg green as a function of ATP levels, and ratio of carboxy SNARF-1 to Mitotracker green as a function of the mitochondrial matrix pH8, 12.We prepared 59Fe-saturated transferrin, and measured 59Fe incorporated in mitochondria and complexed in heme using a gamma counter. We examined PPIX levels and the catalytic efficiency of Fech in MEL cells using spectrophotometric analysis. The MEL cells were treated with FCCP and 2,4-DNP, followed by analysis for hemoglobinized cells. Human and yeast Fech were treated with DTN, and subsequently their catalytic efficiency were measured. Aconitase activity was determined as a measure of [2Fe-2S] cluster levels29. The cRNA for zebrafish Fech or yeast Fech was injected in pnt embryos, and their efficiency to rescue the anemia in pnt was measured using o-dianisidine staining and verified by using SSCP analysis10. Statistical analyses were performed by paired or un-paired t-test. Significance was set at p<0.05.

METHODS SUMMARY

All procedures were approved by the Animal Care and Use Committee of Boston Children’s Hospital. We performed complete blood count and Wright-Geimsa analyses on peripheral blood recovered from adult pnt. Genetic mapping and positional cloning were utilized to identify zgc: 162207 (atpif1a) as the candidate gene for the pnt locus on zebrafish Chr. 19. We employed qRT-PCR using TaqMan gene expression assays (Applied Biosystems, Carlsbad, CA) to measure levels of atpif1a and atpif1b mRNA. Morpholinos (Gene Tools, Philomath, OR) against splice-site of atpif1a and atpif1b were designed and injected in WT embryos to verify loss-of-function phenotype. The cRNA for atpif1a, atpif1a-E26A, and atpif1b were injected in pnt embryos for complementation. The cDNA prepared from WT and pnt embryos was sequenced, and the polymorphism in the 3′UTR of the atpif1a sequence was verified using SSCP gels.

We silenced Atpif1 in hCD34+, MPFL and MEL using shRNAs. The Atpif1-silenced, differentiated hCD34+ and MEL cells were stained with o-dianisidine to measure hemoglobinized cells, while MPFL cells were treated with Drabkin’s reagent to measure relative hemoglobin content. The loss of Atpif1 protein and the state of mitochondrial structural proteins in MEL cells were verified using western and electron microscopic analyses. We analyzed fluorescent intensities of TMRE as a function of mitochondrial membrane potential, Mg green as a function of ATP levels, and ratio of carboxy SNARF-1 to Mitotracker green as a function of the mitochondrial matrix pH8, 12.

We prepared 59Fe-saturated transferrin, and measured 59Fe incorporated in mitochondria and complexed in heme using a gamma counter. We examined PPIX levels and the catalytic efficiency of Fech in MEL cells using spectrophotometric analysis. The MEL cells were treated with FCCP and 2,4-DNP, followed by analysis for hemoglobinized cells. Human and yeast Fech were treated with DTN, and subsequently their catalytic efficiency were measured. Aconitase activity was determined as a measure of [2Fe-2S] cluster levels29. The cRNA for zebrafish Fech or yeast Fech was injected in pnt embryos, and their efficiency to rescue the anemia in pnt was measured using o-dianisidine staining and verified by using SSCP analysis10. Statistical analyses were performed by paired or un-paired t-test. Significance was set at p<0.05.

Supplementary Material1

Supplementary Material
