---
pmid: '27525505'
title: SEDS proteins are a widespread family of bacterial cell wall polymerases.
authors:
- Meeske AJ
- Riley EP
- Robins WP
- Uehara T
- Mekalanos JJ
- Kahne D
- Walker S
- Kruse AC
- Bernhardt TG
- Rudner DZ
journal: Nature
year: '2016'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC5161649
doi: 10.1038/nature19331
---

# SEDS proteins are a widespread family of bacterial cell wall polymerases.
**Authors:** Meeske AJ, Riley EP, Robins WP, Uehara T, Mekalanos JJ, Kahne D, Walker S, Kruse AC, Bernhardt TG, Rudner DZ
**Journal:** Nature (2016)
**DOI:** [10.1038/nature19331](https://doi.org/10.1038/nature19331)
**PMC:** [PMC5161649](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5161649/)

## Abstract

1. Nature. 2016 Sep 29;537(7622):634-638. doi: 10.1038/nature19331. Epub 2016 Aug
 15.

SEDS proteins are a widespread family of bacterial cell wall polymerases.

Meeske AJ(1), Riley EP(1), Robins WP(1), Uehara T(1), Mekalanos JJ(1), Kahne 
D(2)(3), Walker S(1)(2)(3), Kruse AC(2), Bernhardt TG(1), Rudner DZ(1).

Author information:
(1)Department of Microbiology and Immunobiology, Harvard Medical School, 77 
Avenue Louis Pasteur, Boston, Massachusetts 02115, USA.
(2)Department of Biological Chemistry and Molecular Pharmacology, Harvard 
Medical School, 250 Longwood Ave, Boston, Massachusetts 02115, USA.
(3)Department of Chemistry and Chemical Biology, Harvard University, Cambridge, 
Massachusetts 02138, USA.

Comment in
    Nature. 2016 Sep 28;537(7622):622-4. doi: 10.1038/537622a.
    Curr Biol. 2016 Nov 7;26(21):R1158-R1160. doi: 10.1016/j.cub.2016.09.028.

Elongation of rod-shaped bacteria is mediated by a dynamic 
peptidoglycan-synthetizing machinery called the Rod complex. Here we report 
that, in Bacillus subtilis, this complex is functional in the absence of all 
known peptidoglycan polymerases. Cells lacking these enzymes survive by inducing 
an envelope stress response that increases the expression of RodA, a widely 
conserved core component of the Rod complex. RodA is a member of the SEDS 
(shape, elongation, division and sporulation) family of proteins, which have 
essential but ill-defined roles in cell wall biogenesis during growth, division 
and sporulation. Our genetic and biochemical analyses indicate that SEDS 
proteins constitute a family of peptidoglycan polymerases. Thus, B. subtilis and 
probably most bacteria use two distinct classes of polymerase to synthesize 
their exoskeleton. Our findings indicate that SEDS family proteins are core cell 
wall synthases of the cell elongation and division machinery, and represent 
attractive targets for antibiotic development.

DOI: 10.1038/nature19331
PMCID: PMC5161649
PMID: 27525505 [Indexed for MEDLINE]

Conflict of interest statement: The authors declare no competing financial 
interests.

## Full Text

TextThe peptidoglycan (PG) cell wall is the bacterial exoskeleton. It specifies cell shape, protects the cell from osmotic lysis, and its biogenesis is among the most enduring targets for therapeutic intervention. PG is composed of polysaccharide chains crosslinked to each other by short peptides. This meshwork is synthesized by peptidoglycan glycosyltransferases (PGT) that polymerize the glycan strands from a lipid-linked precursor called lipid II. The nascent strands are incorporated into the existing meshwork by the transpeptidase (TP) activity of penicillin binding proteins (PBPs), which crosslink stem peptides attached to the polysaccharide chains. Glycan strand synthesis is carried out by a subset of PBPs called Class A PBPs (for simplicity, referred to as aPBPs) that are bi-functional enzymes possessing a glycosyltransferase 51 (GT51) domain and a TP domain1,2. aPBPs are thought to polymerize glycan strands in the context of two essential and broadly conserved cell wall synthetic machines. The Rod complex (also called the elongasome) polymerizes PG along the cell cylinder to drive cell elongation, while the divisome synthesizes PG during cytokinesis generating the incipient cell poles3.Intriguingly, a small subset of bacteria that possess PG lack aPBPs and do not encode other proteins with GT51 domains4. Furthermore, the model Gram-positive bacterium Bacillus subtilis encodes four aPBPs, yet Popham and co-workers reported over a decade ago that cells lacking all four enzymes are viable, make PG, and are rod-shaped (Fig. 1a)5. Similar findings have been made in two species of Enterococcus6,7. Under these conditions, cross-linking of adjacent glycan strands is likely carried out by the essential Class B PBPs (for simplicity, bPBPs), which have TP domains but lack PGT activity1. However, the enzyme(s) responsible for polymerizing the glycan strands in these cells is completely unknown. It is also unclear whether these unidentified PGTs are central players in cell wall synthesis or moonlighting enzymes that compensate for the loss of aPBP function5.

The peptidoglycan (PG) cell wall is the bacterial exoskeleton. It specifies cell shape, protects the cell from osmotic lysis, and its biogenesis is among the most enduring targets for therapeutic intervention. PG is composed of polysaccharide chains crosslinked to each other by short peptides. This meshwork is synthesized by peptidoglycan glycosyltransferases (PGT) that polymerize the glycan strands from a lipid-linked precursor called lipid II. The nascent strands are incorporated into the existing meshwork by the transpeptidase (TP) activity of penicillin binding proteins (PBPs), which crosslink stem peptides attached to the polysaccharide chains. Glycan strand synthesis is carried out by a subset of PBPs called Class A PBPs (for simplicity, referred to as aPBPs) that are bi-functional enzymes possessing a glycosyltransferase 51 (GT51) domain and a TP domain1,2. aPBPs are thought to polymerize glycan strands in the context of two essential and broadly conserved cell wall synthetic machines. The Rod complex (also called the elongasome) polymerizes PG along the cell cylinder to drive cell elongation, while the divisome synthesizes PG during cytokinesis generating the incipient cell poles3.

Intriguingly, a small subset of bacteria that possess PG lack aPBPs and do not encode other proteins with GT51 domains4. Furthermore, the model Gram-positive bacterium Bacillus subtilis encodes four aPBPs, yet Popham and co-workers reported over a decade ago that cells lacking all four enzymes are viable, make PG, and are rod-shaped (Fig. 1a)5. Similar findings have been made in two species of Enterococcus6,7. Under these conditions, cross-linking of adjacent glycan strands is likely carried out by the essential Class B PBPs (for simplicity, bPBPs), which have TP domains but lack PGT activity1. However, the enzyme(s) responsible for polymerizing the glycan strands in these cells is completely unknown. It is also unclear whether these unidentified PGTs are central players in cell wall synthesis or moonlighting enzymes that compensate for the loss of aPBP function5.

The Rod complex functions without aPBPsTo investigate whether the cell wall elongation machinery is functional in the absence of the aPBPs, we monitored a GFP-labeled component (Mbl) of the Rod complex by time-lapse fluorescence microscopy. Recent work indicates that the Rod complex moves in a directed and circumferential manner around the long axis of the cell8–10. This movement depends on active cell wall synthesis, and thus reflects Rod complex-dependent PG polymerization. Strikingly, cells lacking all four aPBPs exhibited GFP-Mbl dynamics that were qualitatively and quantitatively similar to wild-type (Fig. 1b–e and Supplementary Videos 1 and 2). Furthermore, the directed movement was sensitive to the cell wall synthesis inhibitors vancomycin and ampicillin (Fig. 1f, Extended Data Fig. 1a–b, Supplementary Videos 3–4). Thus, Rod complex-dependent PG synthesis is largely unaffected in the absence of all known PGTs. Consistent with these findings, GFP-Mbl dynamics in wild-type cells were unperturbed by moenomycin, an inhibitor of the glycosyltransferase activity of aPBPs (Extended Data Fig. 1c–d, Supplementary Video 5). We conclude that an unidentified PGT provides the critical polymerase activity needed for Rod complex function. Furthermore, since cells lacking the aPBPs are capable of dividing5 (Fig. 1a), the same or a related PGT activity is also likely to function in the divisome.

The Rod complex functions without aPBPs

To investigate whether the cell wall elongation machinery is functional in the absence of the aPBPs, we monitored a GFP-labeled component (Mbl) of the Rod complex by time-lapse fluorescence microscopy. Recent work indicates that the Rod complex moves in a directed and circumferential manner around the long axis of the cell8–10. This movement depends on active cell wall synthesis, and thus reflects Rod complex-dependent PG polymerization. Strikingly, cells lacking all four aPBPs exhibited GFP-Mbl dynamics that were qualitatively and quantitatively similar to wild-type (Fig. 1b–e and Supplementary Videos 1 and 2). Furthermore, the directed movement was sensitive to the cell wall synthesis inhibitors vancomycin and ampicillin (Fig. 1f, Extended Data Fig. 1a–b, Supplementary Videos 3–4). Thus, Rod complex-dependent PG synthesis is largely unaffected in the absence of all known PGTs. Consistent with these findings, GFP-Mbl dynamics in wild-type cells were unperturbed by moenomycin, an inhibitor of the glycosyltransferase activity of aPBPs (Extended Data Fig. 1c–d, Supplementary Video 5). We conclude that an unidentified PGT provides the critical polymerase activity needed for Rod complex function. Furthermore, since cells lacking the aPBPs are capable of dividing5 (Fig. 1a), the same or a related PGT activity is also likely to function in the divisome.

SEDS proteins bear similarity to known GTsWe reasoned that either a single unidentified synthase acts in both the Rod complex and the divisome, or there are two homologous enzymes, one for each complex. In line with the latter hypothesis, two protein families are represented in both synthetic machines: the bPBPs which crosslink the stem peptides on the glycan strands and the SEDS (Shape, Elongation, Division and Sporulation) family of enigmatic function11. The bPBPs and SEDS proteins are thought to work together as a subcomplex12–15, and are genetically linked in many bacteria (Extended Data Fig. 2). In B. subtilis, the bPBP-SEDS pair present in the Rod complex is PBP2A (or PBPH) and RodA, respectively16,17, while PBP2B and FtsW are components of the divisome18,19. Intriguingly, three decades ago Ishino and co-workers provided evidence that members of these two families stimulate PGT activity in Escherichia coli20 but these findings were never extended or corroborated.Using the remote homology detection algorithm HHpred21, we discovered that the SEDS proteins exhibit weak similarity to O-antigen ligases involved in synthesis of lipopolysaccharide (LPS) in Gram-negative bacteria. Although the homology did not extend to conserved sequence motifs, the secondary structure and positional amino acid distribution produced a high confidence match (Probability 98.4%, E-value 0.0071). O-antigen ligase is a polytopic membrane protein glycosyltransferase that carries out the en bloc transfer of undecaprenyl-pyrophosphate-linked O-antigen polymers to a Lipid A-core glycolipid acceptor (Fig. 2)22. Like the SEDS proteins, O-antigen ligases contain 10–12 transmembrane segments and a large extracytoplasmic loop that is required for activity. Moreover, peptidoglycan precursors are similarly linked to an undecaprenyl-pyrophosphate carrier (Fig. 2). O-antigen ligase represents one of many multipass membrane protein glycosyltransferases that use lipid-linked precursor substrates, including proteins involved in the synthesis of LPS and other surface polymers, as well as O- and N-linked protein glycosylation (Fig. 2)22–26. On the basis of these observations, we hypothesized that the SEDS proteins RodA and FtsW are the unidentified PGTs in the Rod complex and the divisome, respectively.RodA expression suppresses the aPBP mutantTo investigate whether RodA can catalyze PGT activity, we sought to purify RodA from the B. subtilis strain lacking all four aPBPs (Δ4) to avoid contaminating activities. A functional rodA-his10 fusion under the control of a strong IPTG-inducible promoter was introduced into the quadruple aPBP mutant. Strikingly, we discovered that ~10-fold over-production of RodA-His10 significantly alleviated the growth defect of the quadruple mutant (Fig. 3a and Extended Data Fig. 3a,d). Furthermore, examination of these cells by fluorescence microscopy revealed nearly complete suppression of the cell width and elevated lysis phenotypes observed in the Δ4 mutant (Fig. 3b–c, and Extended Data Fig. 3b–c).Isolated membranes from wild-type cells, the Δ4 mutant, and the Δ4 mutant overexpressing RodA-His10 were assayed for PGT activity using radiolabeled synthetic lipid II27. As expected, membranes lacking the aPBPs had less activity than WT (Fig. 3d). However, PGT activity was significantly higher in the membranes in which RodA was over-produced. Furthermore, RodA and RodA-dependent PGT activity could be solubilized by the zwitterionic detergent CHAPS (Fig. 3d and Extended Data Fig. 3e). These experiments suggest that RodA is a PGT or stimulates an unidentified PGT. However, despite substantial effort, we were unable to affinity purify RodA-His10 under these conditions.RodA has PGT activity in vitroTo improve RodA purification, we constructed an E. coli expression system modeled after those used to purify G protein-coupled receptors, in which a SUMO-FLAG-RodA fusion and the SUMO protease Ulp1 were co-expressed. SUMO cleavage generates an amino-terminal aspartic acid in the FLAG tag that is recognized by the M1 anti-FLAG monoclonal antibody allowing for rapid and specific immuno-affinity purification. To reduce possible contamination from E. coli aPBPs and other proteins containing GT51 domains, we deleted three (ponB, pbpC, mtgA) of the four genes that encode them from our expression strain. The only remaining family member was PBP1A, whose presence during RodA purification could be tracked by immunoblot. FLAG-RodA was immunopurified from detergent-solubilized membranes and eluted with EDTA and FLAG peptide. The protein, estimated to be 60% pure (Fig. 4a), catalyzed the conversion of lipid II into glycan chains in vitro (Fig. 4b–c and Extended Data Fig. 4). Importantly, the PGT activity was resistant to moenomcyin at concentrations that inhibit PBP1A28 as well as SgtB, a PGT from Staphylococcus aureus (Fig. 4b–c), suggesting the activity was not due to aPBP contamination. To test whether RodA was responsible for glycan strand polymerization, we sought to assay non-functional mutants. We screened for essential residues in RodA by mutagenesis followed by high-throughput sequencing (MutSeq)29. Among the residues identified in our screen (Supplementary Table 1 and Extended Data Fig. 5a), we chose two (W105 and D280) predicted to be in the second and fourth extracellular loops of RodA (Extended Data Fig. 5). Alanine substitutions at either position abolished RodA function in vivo without affecting protein levels (Fig. 3a and Extended Data Fig. 6). Purified FLAG-RodA(W105A) and separately (D280A) (Fig. 4a) failed to polymerize glycan strands (Fig. 4b–c). Importantly, immunoblot and mass spectrometry analyses indicate that the levels of PBP1A and other E. coli contaminating proteins were similar in all three purifications (Fig. 4a and Supplementary Table 2). Although we cannot formally rule out the possibility that a novel moenomycin-resistant E. coli PGT that is present in our purified preparation is stimulated by wild-type RodA but not by either point mutant, our data are most consistent with the conclusion that RodA is a PG polymerase.RodA induction confers moenomycin resistanceB. subtilis is intrinsically resistant to moenomycin, and this resistance is dependent upon the extracytoplasmic function (ECF) sigma factor SigM (σM)30 (Fig. 4d). Consistent with the idea that aPBPs are specifically targeted by moenomycin, we found that cells lacking all four aPBPs require sigM for viability (Fig. 4e). Among the genes that are induced by σM in response to envelope stress is rodA31. In light of our data that RodA is a moenomycin-resistant PGT (Fig. 4b–c), we investigated whether rodA upregulation was responsible for this natural resistance. Indeed, overexpression of rodA restored moenomycin resistance to a sigM null mutant (Fig. 4d). Furthermore, it suppressed the synthetic lethality of the ΔsigM Δ4 aPBP mutant (Extended Data Fig. 7a). To directly test whether σM-dependent expression of rodA provides moenomycin resistance, we mutated the σM recognition sequences (both −10 and −35 elements) in the rodA promoter. In the absence of drug, the promoter mutant (PΔsigM) was indistinguishable from wild-type with respect to growth rate and morphology (Extended Data Fig. 7b–c). However, the PΔsigM mutant was sensitive to moenomycin and synthetically lethal with the quadruple aPBP deletion (Fig. 4d–e). Thus, increased expression of rodA mediated by σM is both necessary and sufficient to confer intrinsic resistance to moenomycin and viability to cells lacking the aPBPs.

SEDS proteins bear similarity to known GTs

We reasoned that either a single unidentified synthase acts in both the Rod complex and the divisome, or there are two homologous enzymes, one for each complex. In line with the latter hypothesis, two protein families are represented in both synthetic machines: the bPBPs which crosslink the stem peptides on the glycan strands and the SEDS (Shape, Elongation, Division and Sporulation) family of enigmatic function11. The bPBPs and SEDS proteins are thought to work together as a subcomplex12–15, and are genetically linked in many bacteria (Extended Data Fig. 2). In B. subtilis, the bPBP-SEDS pair present in the Rod complex is PBP2A (or PBPH) and RodA, respectively16,17, while PBP2B and FtsW are components of the divisome18,19. Intriguingly, three decades ago Ishino and co-workers provided evidence that members of these two families stimulate PGT activity in Escherichia coli20 but these findings were never extended or corroborated.

Using the remote homology detection algorithm HHpred21, we discovered that the SEDS proteins exhibit weak similarity to O-antigen ligases involved in synthesis of lipopolysaccharide (LPS) in Gram-negative bacteria. Although the homology did not extend to conserved sequence motifs, the secondary structure and positional amino acid distribution produced a high confidence match (Probability 98.4%, E-value 0.0071). O-antigen ligase is a polytopic membrane protein glycosyltransferase that carries out the en bloc transfer of undecaprenyl-pyrophosphate-linked O-antigen polymers to a Lipid A-core glycolipid acceptor (Fig. 2)22. Like the SEDS proteins, O-antigen ligases contain 10–12 transmembrane segments and a large extracytoplasmic loop that is required for activity. Moreover, peptidoglycan precursors are similarly linked to an undecaprenyl-pyrophosphate carrier (Fig. 2). O-antigen ligase represents one of many multipass membrane protein glycosyltransferases that use lipid-linked precursor substrates, including proteins involved in the synthesis of LPS and other surface polymers, as well as O- and N-linked protein glycosylation (Fig. 2)22–26. On the basis of these observations, we hypothesized that the SEDS proteins RodA and FtsW are the unidentified PGTs in the Rod complex and the divisome, respectively.

RodA expression suppresses the aPBP mutantTo investigate whether RodA can catalyze PGT activity, we sought to purify RodA from the B. subtilis strain lacking all four aPBPs (Δ4) to avoid contaminating activities. A functional rodA-his10 fusion under the control of a strong IPTG-inducible promoter was introduced into the quadruple aPBP mutant. Strikingly, we discovered that ~10-fold over-production of RodA-His10 significantly alleviated the growth defect of the quadruple mutant (Fig. 3a and Extended Data Fig. 3a,d). Furthermore, examination of these cells by fluorescence microscopy revealed nearly complete suppression of the cell width and elevated lysis phenotypes observed in the Δ4 mutant (Fig. 3b–c, and Extended Data Fig. 3b–c).Isolated membranes from wild-type cells, the Δ4 mutant, and the Δ4 mutant overexpressing RodA-His10 were assayed for PGT activity using radiolabeled synthetic lipid II27. As expected, membranes lacking the aPBPs had less activity than WT (Fig. 3d). However, PGT activity was significantly higher in the membranes in which RodA was over-produced. Furthermore, RodA and RodA-dependent PGT activity could be solubilized by the zwitterionic detergent CHAPS (Fig. 3d and Extended Data Fig. 3e). These experiments suggest that RodA is a PGT or stimulates an unidentified PGT. However, despite substantial effort, we were unable to affinity purify RodA-His10 under these conditions.

RodA expression suppresses the aPBP mutant

To investigate whether RodA can catalyze PGT activity, we sought to purify RodA from the B. subtilis strain lacking all four aPBPs (Δ4) to avoid contaminating activities. A functional rodA-his10 fusion under the control of a strong IPTG-inducible promoter was introduced into the quadruple aPBP mutant. Strikingly, we discovered that ~10-fold over-production of RodA-His10 significantly alleviated the growth defect of the quadruple mutant (Fig. 3a and Extended Data Fig. 3a,d). Furthermore, examination of these cells by fluorescence microscopy revealed nearly complete suppression of the cell width and elevated lysis phenotypes observed in the Δ4 mutant (Fig. 3b–c, and Extended Data Fig. 3b–c).

Isolated membranes from wild-type cells, the Δ4 mutant, and the Δ4 mutant overexpressing RodA-His10 were assayed for PGT activity using radiolabeled synthetic lipid II27. As expected, membranes lacking the aPBPs had less activity than WT (Fig. 3d). However, PGT activity was significantly higher in the membranes in which RodA was over-produced. Furthermore, RodA and RodA-dependent PGT activity could be solubilized by the zwitterionic detergent CHAPS (Fig. 3d and Extended Data Fig. 3e). These experiments suggest that RodA is a PGT or stimulates an unidentified PGT. However, despite substantial effort, we were unable to affinity purify RodA-His10 under these conditions.

RodA has PGT activity in vitroTo improve RodA purification, we constructed an E. coli expression system modeled after those used to purify G protein-coupled receptors, in which a SUMO-FLAG-RodA fusion and the SUMO protease Ulp1 were co-expressed. SUMO cleavage generates an amino-terminal aspartic acid in the FLAG tag that is recognized by the M1 anti-FLAG monoclonal antibody allowing for rapid and specific immuno-affinity purification. To reduce possible contamination from E. coli aPBPs and other proteins containing GT51 domains, we deleted three (ponB, pbpC, mtgA) of the four genes that encode them from our expression strain. The only remaining family member was PBP1A, whose presence during RodA purification could be tracked by immunoblot. FLAG-RodA was immunopurified from detergent-solubilized membranes and eluted with EDTA and FLAG peptide. The protein, estimated to be 60% pure (Fig. 4a), catalyzed the conversion of lipid II into glycan chains in vitro (Fig. 4b–c and Extended Data Fig. 4). Importantly, the PGT activity was resistant to moenomcyin at concentrations that inhibit PBP1A28 as well as SgtB, a PGT from Staphylococcus aureus (Fig. 4b–c), suggesting the activity was not due to aPBP contamination. To test whether RodA was responsible for glycan strand polymerization, we sought to assay non-functional mutants. We screened for essential residues in RodA by mutagenesis followed by high-throughput sequencing (MutSeq)29. Among the residues identified in our screen (Supplementary Table 1 and Extended Data Fig. 5a), we chose two (W105 and D280) predicted to be in the second and fourth extracellular loops of RodA (Extended Data Fig. 5). Alanine substitutions at either position abolished RodA function in vivo without affecting protein levels (Fig. 3a and Extended Data Fig. 6). Purified FLAG-RodA(W105A) and separately (D280A) (Fig. 4a) failed to polymerize glycan strands (Fig. 4b–c). Importantly, immunoblot and mass spectrometry analyses indicate that the levels of PBP1A and other E. coli contaminating proteins were similar in all three purifications (Fig. 4a and Supplementary Table 2). Although we cannot formally rule out the possibility that a novel moenomycin-resistant E. coli PGT that is present in our purified preparation is stimulated by wild-type RodA but not by either point mutant, our data are most consistent with the conclusion that RodA is a PG polymerase.

RodA has PGT activity in vitro

To improve RodA purification, we constructed an E. coli expression system modeled after those used to purify G protein-coupled receptors, in which a SUMO-FLAG-RodA fusion and the SUMO protease Ulp1 were co-expressed. SUMO cleavage generates an amino-terminal aspartic acid in the FLAG tag that is recognized by the M1 anti-FLAG monoclonal antibody allowing for rapid and specific immuno-affinity purification. To reduce possible contamination from E. coli aPBPs and other proteins containing GT51 domains, we deleted three (ponB, pbpC, mtgA) of the four genes that encode them from our expression strain. The only remaining family member was PBP1A, whose presence during RodA purification could be tracked by immunoblot. FLAG-RodA was immunopurified from detergent-solubilized membranes and eluted with EDTA and FLAG peptide. The protein, estimated to be 60% pure (Fig. 4a), catalyzed the conversion of lipid II into glycan chains in vitro (Fig. 4b–c and Extended Data Fig. 4). Importantly, the PGT activity was resistant to moenomcyin at concentrations that inhibit PBP1A28 as well as SgtB, a PGT from Staphylococcus aureus (Fig. 4b–c), suggesting the activity was not due to aPBP contamination. To test whether RodA was responsible for glycan strand polymerization, we sought to assay non-functional mutants. We screened for essential residues in RodA by mutagenesis followed by high-throughput sequencing (MutSeq)29. Among the residues identified in our screen (Supplementary Table 1 and Extended Data Fig. 5a), we chose two (W105 and D280) predicted to be in the second and fourth extracellular loops of RodA (Extended Data Fig. 5). Alanine substitutions at either position abolished RodA function in vivo without affecting protein levels (Fig. 3a and Extended Data Fig. 6). Purified FLAG-RodA(W105A) and separately (D280A) (Fig. 4a) failed to polymerize glycan strands (Fig. 4b–c). Importantly, immunoblot and mass spectrometry analyses indicate that the levels of PBP1A and other E. coli contaminating proteins were similar in all three purifications (Fig. 4a and Supplementary Table 2). Although we cannot formally rule out the possibility that a novel moenomycin-resistant E. coli PGT that is present in our purified preparation is stimulated by wild-type RodA but not by either point mutant, our data are most consistent with the conclusion that RodA is a PG polymerase.

RodA induction confers moenomycin resistanceB. subtilis is intrinsically resistant to moenomycin, and this resistance is dependent upon the extracytoplasmic function (ECF) sigma factor SigM (σM)30 (Fig. 4d). Consistent with the idea that aPBPs are specifically targeted by moenomycin, we found that cells lacking all four aPBPs require sigM for viability (Fig. 4e). Among the genes that are induced by σM in response to envelope stress is rodA31. In light of our data that RodA is a moenomycin-resistant PGT (Fig. 4b–c), we investigated whether rodA upregulation was responsible for this natural resistance. Indeed, overexpression of rodA restored moenomycin resistance to a sigM null mutant (Fig. 4d). Furthermore, it suppressed the synthetic lethality of the ΔsigM Δ4 aPBP mutant (Extended Data Fig. 7a). To directly test whether σM-dependent expression of rodA provides moenomycin resistance, we mutated the σM recognition sequences (both −10 and −35 elements) in the rodA promoter. In the absence of drug, the promoter mutant (PΔsigM) was indistinguishable from wild-type with respect to growth rate and morphology (Extended Data Fig. 7b–c). However, the PΔsigM mutant was sensitive to moenomycin and synthetically lethal with the quadruple aPBP deletion (Fig. 4d–e). Thus, increased expression of rodA mediated by σM is both necessary and sufficient to confer intrinsic resistance to moenomycin and viability to cells lacking the aPBPs.

RodA induction confers moenomycin resistance

B. subtilis is intrinsically resistant to moenomycin, and this resistance is dependent upon the extracytoplasmic function (ECF) sigma factor SigM (σM)30 (Fig. 4d). Consistent with the idea that aPBPs are specifically targeted by moenomycin, we found that cells lacking all four aPBPs require sigM for viability (Fig. 4e). Among the genes that are induced by σM in response to envelope stress is rodA31. In light of our data that RodA is a moenomycin-resistant PGT (Fig. 4b–c), we investigated whether rodA upregulation was responsible for this natural resistance. Indeed, overexpression of rodA restored moenomycin resistance to a sigM null mutant (Fig. 4d). Furthermore, it suppressed the synthetic lethality of the ΔsigM Δ4 aPBP mutant (Extended Data Fig. 7a). To directly test whether σM-dependent expression of rodA provides moenomycin resistance, we mutated the σM recognition sequences (both −10 and −35 elements) in the rodA promoter. In the absence of drug, the promoter mutant (PΔsigM) was indistinguishable from wild-type with respect to growth rate and morphology (Extended Data Fig. 7b–c). However, the PΔsigM mutant was sensitive to moenomycin and synthetically lethal with the quadruple aPBP deletion (Fig. 4d–e). Thus, increased expression of rodA mediated by σM is both necessary and sufficient to confer intrinsic resistance to moenomycin and viability to cells lacking the aPBPs.

DiscussionHere, we have presented evidence that the SEDS family protein RodA is the principal PGT in the Rod complex and is likely to be responsible for its dynamic movement. Since B. subtilis cells lacking all aPBPs undergo cytokinesis, by extension, we propose that the SEDS family protein FtsW plays a similar role in the divisome. The requirement of SEDS proteins for cell wall biogenesis; the extensive genetic linkage between the genes encoding SEDS proteins and those involved in PG synthesis (Extended Data Fig. 2); and the nearly universal conservation of the residues essential for the catalytic activity of RodA (Extended Data Fig. 5b) collectively argue that SEDS proteins constitute a widespread family of peptidoglycan glycosyltransferases. Furthermore, given that SEDS and bPBPs are known to interact, these factors likely function together within their designated complexes as cognate enzyme pairs that polymerize and crosslink nascent PG into the existing meshwork.Phylogenetic analysis indicates that SEDS family members and bPBPs are more broadly conserved than aPBPs (Extended Data Fig. 8). In particular, a subset of bacteria including the pathogens Francisella tularensis and Chlamydia trachomatis, possess a cell wall but do not encode aPBPs. Among these taxa are organisms recently discovered to make PG32–35 or encode an essential lipid II biosynthetic pathway36. In all cases, these organisms contain at least one SEDS family member and a bPBP, and we hypothesize that the assembly of their cell wall is mediated by these enzymes. Based on these and previous findings20, we propose that SEDS-bPBP pairs are the core PG synthases in the cell wall elongation and division machineries. The specific role(s) of the aPBPs and whether or not they function in the context of these complexes remain important questions for future investigation. SEDS family members have previously been proposed to transport PG precursors (lipid II) across the cytoplasmic membrane, however, recent evidence suggests that they are not required for this process and that MurJ family members likely fulfill this role37,38. While it is formally possible that SEDS proteins are both PG polymerases and lipid II flippases, we favor the idea that these activities are catalyzed by distinct families of proteins. This view is informed by spore formation in B. subtilis, where a thick layer of spore-specific cell wall called the cortex is deposited around the developing spore. No aPBP has been found to be required for cortex biogenesis39. However, there are three sporulation-induced proteins that are absolutely essential: a MurJ homolog (SpoVB); a SEDS family member (SpoVE); and a bPBP (SpoVD)40. These three protein families possess the three activities (flippase, PGT, TP) required to build a cell wall, and thus may constitute a minimal PG assembly machine. Finally, drugs that target cell wall synthesis are among the most effective in combatting bacterial infection. The discovery that SEDS family proteins are PGTs with extracytoplasmic catalytic centers provides an attractive new avenue to target this pathway for antibiotic development.

Here, we have presented evidence that the SEDS family protein RodA is the principal PGT in the Rod complex and is likely to be responsible for its dynamic movement. Since B. subtilis cells lacking all aPBPs undergo cytokinesis, by extension, we propose that the SEDS family protein FtsW plays a similar role in the divisome. The requirement of SEDS proteins for cell wall biogenesis; the extensive genetic linkage between the genes encoding SEDS proteins and those involved in PG synthesis (Extended Data Fig. 2); and the nearly universal conservation of the residues essential for the catalytic activity of RodA (Extended Data Fig. 5b) collectively argue that SEDS proteins constitute a widespread family of peptidoglycan glycosyltransferases. Furthermore, given that SEDS and bPBPs are known to interact, these factors likely function together within their designated complexes as cognate enzyme pairs that polymerize and crosslink nascent PG into the existing meshwork.

Phylogenetic analysis indicates that SEDS family members and bPBPs are more broadly conserved than aPBPs (Extended Data Fig. 8). In particular, a subset of bacteria including the pathogens Francisella tularensis and Chlamydia trachomatis, possess a cell wall but do not encode aPBPs. Among these taxa are organisms recently discovered to make PG32–35 or encode an essential lipid II biosynthetic pathway36. In all cases, these organisms contain at least one SEDS family member and a bPBP, and we hypothesize that the assembly of their cell wall is mediated by these enzymes. Based on these and previous findings20, we propose that SEDS-bPBP pairs are the core PG synthases in the cell wall elongation and division machineries. The specific role(s) of the aPBPs and whether or not they function in the context of these complexes remain important questions for future investigation. SEDS family members have previously been proposed to transport PG precursors (lipid II) across the cytoplasmic membrane, however, recent evidence suggests that they are not required for this process and that MurJ family members likely fulfill this role37,38. While it is formally possible that SEDS proteins are both PG polymerases and lipid II flippases, we favor the idea that these activities are catalyzed by distinct families of proteins. This view is informed by spore formation in B. subtilis, where a thick layer of spore-specific cell wall called the cortex is deposited around the developing spore. No aPBP has been found to be required for cortex biogenesis39. However, there are three sporulation-induced proteins that are absolutely essential: a MurJ homolog (SpoVB); a SEDS family member (SpoVE); and a bPBP (SpoVD)40. These three protein families possess the three activities (flippase, PGT, TP) required to build a cell wall, and thus may constitute a minimal PG assembly machine. Finally, drugs that target cell wall synthesis are among the most effective in combatting bacterial infection. The discovery that SEDS family proteins are PGTs with extracytoplasmic catalytic centers provides an attractive new avenue to target this pathway for antibiotic development.

MethodsGeneral methodsB. subtilis strains were derived from the prototrophic strain PY79. Unless otherwise indicated, cells were grown in LB or CH medium at 37° C. Insertion-deletion mutants came from the B. subtilis knock out collection (back-crossed to PY79) or were generated by isothermal assembly41 of PCR products followed by direct transformation into B. subtilis PY79. Antibiotic cassette removal was performed using a temperature-sensitive plasmid encoding Cre recombinase as described previously38. Moenomycin was purified from flavomycin feed stock as previously described42. For RodA-mediated suppression of the quadruple aPBP strain, rodA was induced with 50 μM IPTG. For all other induction experiments IPTG was used at 500 μM. Lists of strains (Supplementary Table 3), plasmids (Supplementary Table 4) and oligonucleotide primers (Supplementary Table 5) can be found online as supplementary information. A description of strain and plasmid construction can be found below. Source data for all SDS-PAGE experiments can be found in Supplementary Fig. 1.ImmunoblottingSDS-PAGE lysates were prepared by brief lysozyme treatment and resuspension in an equal volume of 2X Laemmli buffer. Generally, 2–10 mOD units were loaded for analysis on 15% SDS-PAGE gels. For analysis of membrane proteins, samples were not heated prior to electrophoretic separation. Proteins were transferred to a methanol-activated PVDF membrane, blocked with 5% milk, probed with anti-His (Genscript, 1:4,000), anti-σA (1:10,000)43, anti-ParB (1:5,000)44 primary antibody, then horseradish peroxidase-coupled anti-rabbit (Bio-Rad) or anti-mouse (Bio-Rad) secondary antibody. Proteins were detected using Western Lightning chemiluminescence reagent.Fluorescence microscopyExponentially growing cells were concentrated by centrifugation at 8,000 rpm for 1 min and immobilized on 1% agarose pads containing growth medium. Fluorescence microscopy was performed using an Olympus BX61 microscope equipped with a UplanF1 100X phase contrast objective lens and a CoolSnapHQ digital camera (Photometrics) or a Nikon Ti-E inverted microscope with a Nikon CFI Plan Apo VC 100X objective lens. Images were acquired using Metamorph or Nikon Elements software. Membranes were stained with TMA-DPH (50 μM) (Molecular Probes). Live-dead staining was performed with propidium iodide (5 μM) (Invitrogen). GFP-Mbl images were acquired using an ND8 neutral density filter to limit phototoxicity and photobleaching. For time-lapse imaging, images were acquired every 2 sec for 4 min. For vancomycin or ampicilllin addition experiments, drugs were added directly to the agarose pad containing the cells prior to imaging. Exposure times were 400 ms for TMA-DPH and propidium iodide, 500ms for GFP-Mbl. Image analysis and processing were performed in Fiji.B. subtilis membrane preparation and detergent solubilization of membrane proteinsMembranes were isolated from 50 mL B. subtilis cultures grown in LB at 37°C. At OD600 = 0.4, IPTG was added to 500 μM to induce RodA-His10. After a 1 h induction, cells were pelleted by centrifugation and frozen at −80°C. The cell pellets were resuspended in 20 ml lysis buffer (50 mM HEPES pH 7.5, 0.3 M NaCl) and lysed by two passes at 20,000 psi in a French Pressure cell. Unbroken cells were removed by centrifugation at 4,000g for 10 min at 4°C. Membranes were collected by ultracentrifugation at 100,000g for 1 h at 4°C. The membrane pellet was dispersed in 1 ml 50 mM HEPES pH 7.5, 0.5 M NaCl, 20% glycerol at a protein concentration of 1 mg/ml. For detergent solubilization, CHAPS (Anatrace) was added to a final concentration of 2% and incubated overnight at 4°C. Solubilized membrane proteins were isolated by ultracentrifugation at 100,000g for 1 h at 4°C.Protein purificationProtein expression was carried out in an E. coli C43 derivative of BL21 (DE3) harboring deletions in ponB, pbpC, mtgA (strain CAM333) carrying a plasmid-borne arabinose-inducible Ulp1(403–621) protease (pAM174). The SUMO-FLAG-RodA expression plasmid and mutant derivatives (pAM172, pAM172b, pAM172c) were transformed into the expression strain under selection for both plasmids. 5 fresh transformants from each were inoculated into 5 ml LB supplemented with ampicillin (50 μg/ml) and grown to exponential phase at 37°C and then diluted into 1 L LB + 50μg/ml ampicillin. Cultures were grown at 37°C to OD600 = 0.3 and shifted to 30°C. At OD600 = 0.6, IPTG was added (1 mM final) to induce RodA and arabinose was added (0.2% final) to induce Ulp1. After a 5 h induction, cells were pelleted and frozen at −80°C. Cell pellets were resuspended in lysis buffer (50mM HEPES pH 7.5, 0.3 M NaCl), lysed, and membranes isolated as described above. Membranes were mechanically homogenized using a teflon dounce, and solubilized in a mixed detergent system containing CHAPS (0.5%) and n-dodecyl-β-D-maltoside (DDM) (0.5%) for 90 min at 4°C. Insoluble material was removed by ultracentrifugation at 100,000g for 1 h at 4°C. The soluble fraction was supplemented with 2 mM CaCl2 and applied to 1 mL M1 anti-FLAG antibody resin45. The resin was washed with 80 ml wash buffer (50 mM HEPES pH 7.5, 0.5 M NaCl, 20% glycerol, 2 mM CaCl2, 0.05% DDM, 0.5% CHAPS). FLAG-RodA and mutants were eluted with 4 ml wash buffer supplemented with FLAG peptide (0.2 mg/mL) and 5 mM EDTA.PGT activity assayPGT activity in B. subtilis membranes, detergent-solubilized membrane proteins, and immuno-purified proteins from E. coli was assayed as previously described27. Briefly, membranes or protein samples were incubated with 4 μM 14C-labeled synthetic lipid II with a heptaprenyl alkyl chain for 1 h unless otherwise indicated. The reaction buffer consisted of 20% DMSO, 50 mM HEPES pH 7.0, 20 mM MgCl2, 20 mM CaCl2. For B. subtilis membranes and detergent-solubilized membrane proteins, the final protein concentration was 0.05 mg/ml. Purified proteins were used at a final concentration of 0.2 μM. Moenomycin and mutanolysin (Sigma-Aldrich) were used at 0.6 μM and 100 μg/ml, respectively. Synthetic lipid II was synthesized as previously described27. Reactions were quenched by the addition of an equal volume of 10% Triton-X-100. Substrate and products were resolved by paper chromatography. The reactions were spotted onto 20 cm paper strips and chromatography was carried out for 8 h using a mobile phase of isobutyric acid : 1M NH4OH (5:3). Strips were cut 2.5 cm from the origin and each section was analyzed by scintillation counting (Ecolite scintillation cocktail). Counts at the origin-proximal section represent polymeric glycan products, while counts on the distal part of the strip represent unpolymerized lipid II (or short oligomers). For SDS-PAGE-based analysis, reaction products were quenched by boiling for 2 min, dried by speedvac and resuspended in 10 μl Laemmli buffer and resolved on a 20×20cm 12% polyacrylamide gel. The gel was dried and exposed to a phosphor screen for 2 weeks.Mut-Seq library preparation and data analysisMut-seq was performed on two independently generated libraries, as previously described29. In brief, PCR mutagenesis of the rodA open reading frame was performed using GeneMorph II Random Mutagenesis Kit (Stratagene) under conditions yielding ~1.4 mut/kb with primers oDR1067 and oDR1068 and PY79 genomic DNA. A mutant library was constructed using isothermal assembly of PCR-mutagenized rodA with upstream and downstream homology regions and a spectinomycin resistance cassette generated by high-fidelity PCR (upstream region: primers oDR1069 and oDR1070 and PY79 genomic DNA; downstream region and spec(R) gene: oDR1071 and oDR1072 and BDR2702 genomic DNA). The assembled products were transformed directly into competent Bacillus subtilis PY79 selecting for Spec(R) and RodA function. >50,000 transformants were pooled, aliquoted, and frozen. Genomic DNA was extracted from each replicate and the rodA locus amplified by PCR (primers oDR1081 and oDR1082). PCR products were column purified and fragmented by sonication to an average size of 200 bp. Libraries were constructed using the NEBNext DNA Library Mastermix kit and amplified with primers for Illumina sequencing. 50 bp single-end sequencing was carried out on the Illumina HiSeq 2000 platform. Reads were mapped to the B. subtilis PY79 genome reference sequence (GenBank accession: CP006881) and filtered using CLC-Genomics Workbench software. Reads with a single mismatch were quality-filtered by applying a CASAVA 1.8 quality score filter of 38 or higher. On average, we detected 143,000 quality-filtered SNPs, corresponding to approximately 120 SNPs per nucleotide in the RodA coding sequence. The number of mutations at each nucleotide position and their corresponding amino acid changes were tabulated (Supplementary Table 1). Both G->T and C->A transversions were overrepresented when compared to all other classes of mutations and the average counts did not substantially differ between synonymous and nonsense mutations. These are among the most predominant rare mutations in deep-sequence DNA libraries and are thought to be generated by oxidative damage to DNA generated before and after DNA isolation by processes such as sonication-based shearing. We excluded these two mutation types from our analysis. The complete raw dataset is available on sheets 2–4 of Supplementary Table 1. For all other classes of nucleotide changes, the mean number of synonymous and nonsense mutations (when applicable) were used as thresholds to assess whether or not a missense mutation was permissive.Strain constructionAll deletion mutants except ΔsigM and rodA PΔsigM were generated by isothermal assembly followed by direct transformation into B. subtilis. All assembly reactions had three PCR products: an antibiotic resistance gene and upstream and downstream fragments that flank the locus to be deleted. A loxP-flanked kanamycin resistance cassette was amplified using oligonucleotide primers oJM28 and oJM29 and pWX470 as template. Upstream and downstream PCR products were amplified from PY79 genomic DNA using the following oligonucleotide primers: pbpF (oAM38, oAM39, oAM42, oAM43), pbpG (oAM44, oAM45, oAM48, oAM49), pbpD (oAM50, oAM51, oAM54, oAM55), ponA (oJM001, oJM002, oJM003, oJM004), rodA (oDR1069, oDR1124, oDR1125, oDR1072).The ΔsigM deletion was derived from the B. subtilis knockout collection and was confirmed by PCR using oKO000 (within the erythromycin resistance cassette) and an upstream primer oKO82.The rodA PΔsigM mutant was generated by allelic replacement using the pminiMAD-derived plasmid pAM199. This plasmid, harboring mutations in the −10 and −35 elements recognized by SigM46 was transformed into PY79 selecting for resistance to erythromycin. Transformants were grown without selection at 22°C and plated on LB agar at 37°C. Single colonies were screened for loss of the integrated plasmid and sensitivity to erythromycin. Genomic DNA was prepared from four independent Erm(S) colonies and the rodA promoter region was PCR amplified and sequenced (using primer oAM201) to identify strains harboring the SigM promoter mutations.E. coli expression strain CAM333 [C43 ΔponB ΔpbpC ΔmtgA pPBAD-Ulp1(403–621)] was constructed by sequential P1 transduction and antibiotic cassette removal of the FRT-flanked ponB::kan, pbpC::kan, and mtgA::kan deletions from the Keio collection47. The kanamycin resistance cassettes were excised by expression of FLP recombinase from a temperature-sensitive plasmid at 30°C, and the plasmid was subsequently lost by nonselective growth at 37°C. All three deletions were confirmed by PCR using primers flanking the deleted gene (ponB: primers TB1290 and TB1291, pbpC: primers TB633 and TB644, mtgA: primers TB635 and TB636). The strain was then transformed with pAM174 [PBAD-Ulp1(403–621) (cat) p15A ori].Plasmid constructionspAM84 [amyE::Phyperspank-rodA-his10 (spec)(amp)] was generated in a two-way ligation of a HindIII-NheI rodA-his10 PCR product (amplified in sequential PCR steps with oligonucleotide primers oAM231 and oAM114 followed by oAM231 and oAM233 from PY79 genomic DNA) and pDR111 (amyE::Phyperspank spec amp) cut with HindIII and NheI.pAM84b [amyE::Phyperspank-rodA(W105A)-his10 (spec)(amp)] was generated by site-directed mutagenesis from pAM84 using oDR1107pAM84c [amyE::Phyperspank-rodA(D280A)-his10 (spec)(amp)] was generated by site-directed mutagenesis from pAM84 using oDR1104pAM199 [rodA PΔsigM pminiMAD (erm) (amp)] was generated in a three-way isothermal assembly reaction containing: rodA promoter fragment 1 (amplified from PY79 genomic DNA with oligonucleotide primers oAM517 and oAM518); rodA promoter fragment 2 (amplified from PY79 genomic DNA with oligonucleotide primers oAM519 and oAM520) and pminiMAD (erm, amp) digested with BamHI and HindIII. The σM-dependent rodA promoter (with −35 and −10 elements in bold) was changed from:5′ TCAATCGAAACATTTCGGTTTATGATACGTCATATTTCGT 3′ to5′ TCAATCTGCGCATTTCGGTTTATGATATACTATATTTCGT 3′pAM14 [ycgO::Pspank*-ponA (cat) (amp)] was generated in a two-way ligation reaction with a PCR product containing ponA (amplified from PY79 genomic DNA using oligonucleotide primers oAM62 and oAM64) and pAM12 (ycgO::Pspank* cat amp) digested with XmaI and SpeI.pER135 [ycgO::Pspank-rodA (erm) (amp)] was generated in a two-way ligation reaction with a PCR product containing rodA (amplified from PY79 genomic DNA using oligonucleotide primers oDR1122 and oDR1123) and pER62 (ycgO::Pspank erm amp) digested with HindIII and BamHI.pER133 [amyE::PxylA-rodA (spec) (amp)] was generated in a two-way ligation reaction with a PCR product containing rodA (amplified from PY79 genomic DNA using oligonucleotide primers oDR1122 and oDR1123) and pDR150 (amyE::PxylA spec amp) digested with HindIII and BamHI.pER155 [amyE::PxylA-rodA (E288A) (spec) (amp)] was generated by site-directed mutagenesis of pER133 using oDR1119.pER146 [amyE::Pxyl-rodA (D280A) (spec) (amp)] was generated by site-directed mutagenesis of pER133 using oDR1104.pER160 [amyE::Pxyl-rodA (W105A) (spec) (amp)] was generated by site-directed mutagenesis of pER133 using oDR1107.pER157 [amyE::PxylA-rodA (E117A) (spec) (amp)] was generated by site-directed mutagenesis of pER133 using oDR1128.pAM174 [PBAD-Ulp1(403–621)
(cat) p15A ori] was generated in a two-piece isothermal assembly reaction with a His6-Ulp1(403–621) PCR product (amplified from pTB145 (PT7-His6-Ulp1(403–621)) using oligonucleotide primers oAM453 and oAM454) and pMT30 (PBAD-sfgfp p15A cat) digested with NdeI and HindIII.pAM172 [PT7-His6-SUMO-FLAG-RodA (amp)] was generated in a three-piece isothermal assembly reaction with FLAG-RodA fragment 1 (amplified from FLAG-RodA gBlock (IDT) using oligonucleotide primers oAM449 and oAM450), FLAG-RodA fragment 2 (amplified from PY79 genomic DNA using oAM446 and oAM447) and pTB146 (PT7-His6-SUMO amp) digested with SapI and XhoI.pAM172b [PT7-His6-SUMO-FLAG-RodA(W105A) (amp)] was generated by site-directed mutagenesis of pAM172 using oDR1107.pAM172c [PT7-His6-SUMO-FLAG-RodA(D280A) (amp)] was generated by site-directed mutagenesis of pAM172 using oDR1104.The sequence of the FLAG-RodA gBlock was: ATGGAGAATCTGTATTTTCAGGACTACAAAGACGACGACGACCTGGAAGTTCTGTTCCAGGGTCCGGGTGGTTCTTCTAGTCGATATAAGAAACAGCAAAGCCCCTTTTACCAGGGGGATTTGATTTTTATATTTGGTGTGTTTTTTATTATAAGCGTAGTATCAATATATGCTGCCGGTCAGTTTGGGCAGTATGGGAACACGGATTGGATTCAGCAAATTGTGTTTTATCTTTTGGGGGCTGTAGCCATTACAGTTCTTTTGTACTTTGATTTAGAACAGCTTGAAAAGTTAAGTTTATATATTTTTATCATTGGTATTTTATCTTTGATTATTCTTAAAATCAGTCCCGAGTCTATTGCACCTGTTATTAAAGGGGCAAAAAGTTGGTTTAGGATCGGTAGAATAACAATACAGCCGTCAGAGTTTATGAAGGTTGGTTTGATTATGATGCTTGCTTCAGTTATTGGAAAAGCAAATCCTAAAGGAGTTCGGACGC

General methodsB. subtilis strains were derived from the prototrophic strain PY79. Unless otherwise indicated, cells were grown in LB or CH medium at 37° C. Insertion-deletion mutants came from the B. subtilis knock out collection (back-crossed to PY79) or were generated by isothermal assembly41 of PCR products followed by direct transformation into B. subtilis PY79. Antibiotic cassette removal was performed using a temperature-sensitive plasmid encoding Cre recombinase as described previously38. Moenomycin was purified from flavomycin feed stock as previously described42. For RodA-mediated suppression of the quadruple aPBP strain, rodA was induced with 50 μM IPTG. For all other induction experiments IPTG was used at 500 μM. Lists of strains (Supplementary Table 3), plasmids (Supplementary Table 4) and oligonucleotide primers (Supplementary Table 5) can be found online as supplementary information. A description of strain and plasmid construction can be found below. Source data for all SDS-PAGE experiments can be found in Supplementary Fig. 1.

General methods

B. subtilis strains were derived from the prototrophic strain PY79. Unless otherwise indicated, cells were grown in LB or CH medium at 37° C. Insertion-deletion mutants came from the B. subtilis knock out collection (back-crossed to PY79) or were generated by isothermal assembly41 of PCR products followed by direct transformation into B. subtilis PY79. Antibiotic cassette removal was performed using a temperature-sensitive plasmid encoding Cre recombinase as described previously38. Moenomycin was purified from flavomycin feed stock as previously described42. For RodA-mediated suppression of the quadruple aPBP strain, rodA was induced with 50 μM IPTG. For all other induction experiments IPTG was used at 500 μM. Lists of strains (Supplementary Table 3), plasmids (Supplementary Table 4) and oligonucleotide primers (Supplementary Table 5) can be found online as supplementary information. A description of strain and plasmid construction can be found below. Source data for all SDS-PAGE experiments can be found in Supplementary Fig. 1.

ImmunoblottingSDS-PAGE lysates were prepared by brief lysozyme treatment and resuspension in an equal volume of 2X Laemmli buffer. Generally, 2–10 mOD units were loaded for analysis on 15% SDS-PAGE gels. For analysis of membrane proteins, samples were not heated prior to electrophoretic separation. Proteins were transferred to a methanol-activated PVDF membrane, blocked with 5% milk, probed with anti-His (Genscript, 1:4,000), anti-σA (1:10,000)43, anti-ParB (1:5,000)44 primary antibody, then horseradish peroxidase-coupled anti-rabbit (Bio-Rad) or anti-mouse (Bio-Rad) secondary antibody. Proteins were detected using Western Lightning chemiluminescence reagent.

Immunoblotting

SDS-PAGE lysates were prepared by brief lysozyme treatment and resuspension in an equal volume of 2X Laemmli buffer. Generally, 2–10 mOD units were loaded for analysis on 15% SDS-PAGE gels. For analysis of membrane proteins, samples were not heated prior to electrophoretic separation. Proteins were transferred to a methanol-activated PVDF membrane, blocked with 5% milk, probed with anti-His (Genscript, 1:4,000), anti-σA (1:10,000)43, anti-ParB (1:5,000)44 primary antibody, then horseradish peroxidase-coupled anti-rabbit (Bio-Rad) or anti-mouse (Bio-Rad) secondary antibody. Proteins were detected using Western Lightning chemiluminescence reagent.

Fluorescence microscopyExponentially growing cells were concentrated by centrifugation at 8,000 rpm for 1 min and immobilized on 1% agarose pads containing growth medium. Fluorescence microscopy was performed using an Olympus BX61 microscope equipped with a UplanF1 100X phase contrast objective lens and a CoolSnapHQ digital camera (Photometrics) or a Nikon Ti-E inverted microscope with a Nikon CFI Plan Apo VC 100X objective lens. Images were acquired using Metamorph or Nikon Elements software. Membranes were stained with TMA-DPH (50 μM) (Molecular Probes). Live-dead staining was performed with propidium iodide (5 μM) (Invitrogen). GFP-Mbl images were acquired using an ND8 neutral density filter to limit phototoxicity and photobleaching. For time-lapse imaging, images were acquired every 2 sec for 4 min. For vancomycin or ampicilllin addition experiments, drugs were added directly to the agarose pad containing the cells prior to imaging. Exposure times were 400 ms for TMA-DPH and propidium iodide, 500ms for GFP-Mbl. Image analysis and processing were performed in Fiji.

Fluorescence microscopy

Exponentially growing cells were concentrated by centrifugation at 8,000 rpm for 1 min and immobilized on 1% agarose pads containing growth medium. Fluorescence microscopy was performed using an Olympus BX61 microscope equipped with a UplanF1 100X phase contrast objective lens and a CoolSnapHQ digital camera (Photometrics) or a Nikon Ti-E inverted microscope with a Nikon CFI Plan Apo VC 100X objective lens. Images were acquired using Metamorph or Nikon Elements software. Membranes were stained with TMA-DPH (50 μM) (Molecular Probes). Live-dead staining was performed with propidium iodide (5 μM) (Invitrogen). GFP-Mbl images were acquired using an ND8 neutral density filter to limit phototoxicity and photobleaching. For time-lapse imaging, images were acquired every 2 sec for 4 min. For vancomycin or ampicilllin addition experiments, drugs were added directly to the agarose pad containing the cells prior to imaging. Exposure times were 400 ms for TMA-DPH and propidium iodide, 500ms for GFP-Mbl. Image analysis and processing were performed in Fiji.

B. subtilis membrane preparation and detergent solubilization of membrane proteinsMembranes were isolated from 50 mL B. subtilis cultures grown in LB at 37°C. At OD600 = 0.4, IPTG was added to 500 μM to induce RodA-His10. After a 1 h induction, cells were pelleted by centrifugation and frozen at −80°C. The cell pellets were resuspended in 20 ml lysis buffer (50 mM HEPES pH 7.5, 0.3 M NaCl) and lysed by two passes at 20,000 psi in a French Pressure cell. Unbroken cells were removed by centrifugation at 4,000g for 10 min at 4°C. Membranes were collected by ultracentrifugation at 100,000g for 1 h at 4°C. The membrane pellet was dispersed in 1 ml 50 mM HEPES pH 7.5, 0.5 M NaCl, 20% glycerol at a protein concentration of 1 mg/ml. For detergent solubilization, CHAPS (Anatrace) was added to a final concentration of 2% and incubated overnight at 4°C. Solubilized membrane proteins were isolated by ultracentrifugation at 100,000g for 1 h at 4°C.

B. subtilis membrane preparation and detergent solubilization of membrane proteins

Membranes were isolated from 50 mL B. subtilis cultures grown in LB at 37°C. At OD600 = 0.4, IPTG was added to 500 μM to induce RodA-His10. After a 1 h induction, cells were pelleted by centrifugation and frozen at −80°C. The cell pellets were resuspended in 20 ml lysis buffer (50 mM HEPES pH 7.5, 0.3 M NaCl) and lysed by two passes at 20,000 psi in a French Pressure cell. Unbroken cells were removed by centrifugation at 4,000g for 10 min at 4°C. Membranes were collected by ultracentrifugation at 100,000g for 1 h at 4°C. The membrane pellet was dispersed in 1 ml 50 mM HEPES pH 7.5, 0.5 M NaCl, 20% glycerol at a protein concentration of 1 mg/ml. For detergent solubilization, CHAPS (Anatrace) was added to a final concentration of 2% and incubated overnight at 4°C. Solubilized membrane proteins were isolated by ultracentrifugation at 100,000g for 1 h at 4°C.

Protein purificationProtein expression was carried out in an E. coli C43 derivative of BL21 (DE3) harboring deletions in ponB, pbpC, mtgA (strain CAM333) carrying a plasmid-borne arabinose-inducible Ulp1(403–621) protease (pAM174). The SUMO-FLAG-RodA expression plasmid and mutant derivatives (pAM172, pAM172b, pAM172c) were transformed into the expression strain under selection for both plasmids. 5 fresh transformants from each were inoculated into 5 ml LB supplemented with ampicillin (50 μg/ml) and grown to exponential phase at 37°C and then diluted into 1 L LB + 50μg/ml ampicillin. Cultures were grown at 37°C to OD600 = 0.3 and shifted to 30°C. At OD600 = 0.6, IPTG was added (1 mM final) to induce RodA and arabinose was added (0.2% final) to induce Ulp1. After a 5 h induction, cells were pelleted and frozen at −80°C. Cell pellets were resuspended in lysis buffer (50mM HEPES pH 7.5, 0.3 M NaCl), lysed, and membranes isolated as described above. Membranes were mechanically homogenized using a teflon dounce, and solubilized in a mixed detergent system containing CHAPS (0.5%) and n-dodecyl-β-D-maltoside (DDM) (0.5%) for 90 min at 4°C. Insoluble material was removed by ultracentrifugation at 100,000g for 1 h at 4°C. The soluble fraction was supplemented with 2 mM CaCl2 and applied to 1 mL M1 anti-FLAG antibody resin45. The resin was washed with 80 ml wash buffer (50 mM HEPES pH 7.5, 0.5 M NaCl, 20% glycerol, 2 mM CaCl2, 0.05% DDM, 0.5% CHAPS). FLAG-RodA and mutants were eluted with 4 ml wash buffer supplemented with FLAG peptide (0.2 mg/mL) and 5 mM EDTA.

Protein purification

Protein expression was carried out in an E. coli C43 derivative of BL21 (DE3) harboring deletions in ponB, pbpC, mtgA (strain CAM333) carrying a plasmid-borne arabinose-inducible Ulp1(403–621) protease (pAM174). The SUMO-FLAG-RodA expression plasmid and mutant derivatives (pAM172, pAM172b, pAM172c) were transformed into the expression strain under selection for both plasmids. 5 fresh transformants from each were inoculated into 5 ml LB supplemented with ampicillin (50 μg/ml) and grown to exponential phase at 37°C and then diluted into 1 L LB + 50μg/ml ampicillin. Cultures were grown at 37°C to OD600 = 0.3 and shifted to 30°C. At OD600 = 0.6, IPTG was added (1 mM final) to induce RodA and arabinose was added (0.2% final) to induce Ulp1. After a 5 h induction, cells were pelleted and frozen at −80°C. Cell pellets were resuspended in lysis buffer (50mM HEPES pH 7.5, 0.3 M NaCl), lysed, and membranes isolated as described above. Membranes were mechanically homogenized using a teflon dounce, and solubilized in a mixed detergent system containing CHAPS (0.5%) and n-dodecyl-β-D-maltoside (DDM) (0.5%) for 90 min at 4°C. Insoluble material was removed by ultracentrifugation at 100,000g for 1 h at 4°C. The soluble fraction was supplemented with 2 mM CaCl2 and applied to 1 mL M1 anti-FLAG antibody resin45. The resin was washed with 80 ml wash buffer (50 mM HEPES pH 7.5, 0.5 M NaCl, 20% glycerol, 2 mM CaCl2, 0.05% DDM, 0.5% CHAPS). FLAG-RodA and mutants were eluted with 4 ml wash buffer supplemented with FLAG peptide (0.2 mg/mL) and 5 mM EDTA.

PGT activity assayPGT activity in B. subtilis membranes, detergent-solubilized membrane proteins, and immuno-purified proteins from E. coli was assayed as previously described27. Briefly, membranes or protein samples were incubated with 4 μM 14C-labeled synthetic lipid II with a heptaprenyl alkyl chain for 1 h unless otherwise indicated. The reaction buffer consisted of 20% DMSO, 50 mM HEPES pH 7.0, 20 mM MgCl2, 20 mM CaCl2. For B. subtilis membranes and detergent-solubilized membrane proteins, the final protein concentration was 0.05 mg/ml. Purified proteins were used at a final concentration of 0.2 μM. Moenomycin and mutanolysin (Sigma-Aldrich) were used at 0.6 μM and 100 μg/ml, respectively. Synthetic lipid II was synthesized as previously described27. Reactions were quenched by the addition of an equal volume of 10% Triton-X-100. Substrate and products were resolved by paper chromatography. The reactions were spotted onto 20 cm paper strips and chromatography was carried out for 8 h using a mobile phase of isobutyric acid : 1M NH4OH (5:3). Strips were cut 2.5 cm from the origin and each section was analyzed by scintillation counting (Ecolite scintillation cocktail). Counts at the origin-proximal section represent polymeric glycan products, while counts on the distal part of the strip represent unpolymerized lipid II (or short oligomers). For SDS-PAGE-based analysis, reaction products were quenched by boiling for 2 min, dried by speedvac and resuspended in 10 μl Laemmli buffer and resolved on a 20×20cm 12% polyacrylamide gel. The gel was dried and exposed to a phosphor screen for 2 weeks.

PGT activity assay

PGT activity in B. subtilis membranes, detergent-solubilized membrane proteins, and immuno-purified proteins from E. coli was assayed as previously described27. Briefly, membranes or protein samples were incubated with 4 μM 14C-labeled synthetic lipid II with a heptaprenyl alkyl chain for 1 h unless otherwise indicated. The reaction buffer consisted of 20% DMSO, 50 mM HEPES pH 7.0, 20 mM MgCl2, 20 mM CaCl2. For B. subtilis membranes and detergent-solubilized membrane proteins, the final protein concentration was 0.05 mg/ml. Purified proteins were used at a final concentration of 0.2 μM. Moenomycin and mutanolysin (Sigma-Aldrich) were used at 0.6 μM and 100 μg/ml, respectively. Synthetic lipid II was synthesized as previously described27. Reactions were quenched by the addition of an equal volume of 10% Triton-X-100. Substrate and products were resolved by paper chromatography. The reactions were spotted onto 20 cm paper strips and chromatography was carried out for 8 h using a mobile phase of isobutyric acid : 1M NH4OH (5:3). Strips were cut 2.5 cm from the origin and each section was analyzed by scintillation counting (Ecolite scintillation cocktail). Counts at the origin-proximal section represent polymeric glycan products, while counts on the distal part of the strip represent unpolymerized lipid II (or short oligomers). For SDS-PAGE-based analysis, reaction products were quenched by boiling for 2 min, dried by speedvac and resuspended in 10 μl Laemmli buffer and resolved on a 20×20cm 12% polyacrylamide gel. The gel was dried and exposed to a phosphor screen for 2 weeks.

Mut-Seq library preparation and data analysisMut-seq was performed on two independently generated libraries, as previously described29. In brief, PCR mutagenesis of the rodA open reading frame was performed using GeneMorph II Random Mutagenesis Kit (Stratagene) under conditions yielding ~1.4 mut/kb with primers oDR1067 and oDR1068 and PY79 genomic DNA. A mutant library was constructed using isothermal assembly of PCR-mutagenized rodA with upstream and downstream homology regions and a spectinomycin resistance cassette generated by high-fidelity PCR (upstream region: primers oDR1069 and oDR1070 and PY79 genomic DNA; downstream region and spec(R) gene: oDR1071 and oDR1072 and BDR2702 genomic DNA). The assembled products were transformed directly into competent Bacillus subtilis PY79 selecting for Spec(R) and RodA function. >50,000 transformants were pooled, aliquoted, and frozen. Genomic DNA was extracted from each replicate and the rodA locus amplified by PCR (primers oDR1081 and oDR1082). PCR products were column purified and fragmented by sonication to an average size of 200 bp. Libraries were constructed using the NEBNext DNA Library Mastermix kit and amplified with primers for Illumina sequencing. 50 bp single-end sequencing was carried out on the Illumina HiSeq 2000 platform. Reads were mapped to the B. subtilis PY79 genome reference sequence (GenBank accession: CP006881) and filtered using CLC-Genomics Workbench software. Reads with a single mismatch were quality-filtered by applying a CASAVA 1.8 quality score filter of 38 or higher. On average, we detected 143,000 quality-filtered SNPs, corresponding to approximately 120 SNPs per nucleotide in the RodA coding sequence. The number of mutations at each nucleotide position and their corresponding amino acid changes were tabulated (Supplementary Table 1). Both G->T and C->A transversions were overrepresented when compared to all other classes of mutations and the average counts did not substantially differ between synonymous and nonsense mutations. These are among the most predominant rare mutations in deep-sequence DNA libraries and are thought to be generated by oxidative damage to DNA generated before and after DNA isolation by processes such as sonication-based shearing. We excluded these two mutation types from our analysis. The complete raw dataset is available on sheets 2–4 of Supplementary Table 1. For all other classes of nucleotide changes, the mean number of synonymous and nonsense mutations (when applicable) were used as thresholds to assess whether or not a missense mutation was permissive.

Mut-Seq library preparation and data analysis

Mut-seq was performed on two independently generated libraries, as previously described29. In brief, PCR mutagenesis of the rodA open reading frame was performed using GeneMorph II Random Mutagenesis Kit (Stratagene) under conditions yielding ~1.4 mut/kb with primers oDR1067 and oDR1068 and PY79 genomic DNA. A mutant library was constructed using isothermal assembly of PCR-mutagenized rodA with upstream and downstream homology regions and a spectinomycin resistance cassette generated by high-fidelity PCR (upstream region: primers oDR1069 and oDR1070 and PY79 genomic DNA; downstream region and spec(R) gene: oDR1071 and oDR1072 and BDR2702 genomic DNA). The assembled products were transformed directly into competent Bacillus subtilis PY79 selecting for Spec(R) and RodA function. >50,000 transformants were pooled, aliquoted, and frozen. Genomic DNA was extracted from each replicate and the rodA locus amplified by PCR (primers oDR1081 and oDR1082). PCR products were column purified and fragmented by sonication to an average size of 200 bp. Libraries were constructed using the NEBNext DNA Library Mastermix kit and amplified with primers for Illumina sequencing. 50 bp single-end sequencing was carried out on the Illumina HiSeq 2000 platform. Reads were mapped to the B. subtilis PY79 genome reference sequence (GenBank accession: CP006881) and filtered using CLC-Genomics Workbench software. Reads with a single mismatch were quality-filtered by applying a CASAVA 1.8 quality score filter of 38 or higher. On average, we detected 143,000 quality-filtered SNPs, corresponding to approximately 120 SNPs per nucleotide in the RodA coding sequence. The number of mutations at each nucleotide position and their corresponding amino acid changes were tabulated (Supplementary Table 1). Both G->T and C->A transversions were overrepresented when compared to all other classes of mutations and the average counts did not substantially differ between synonymous and nonsense mutations. These are among the most predominant rare mutations in deep-sequence DNA libraries and are thought to be generated by oxidative damage to DNA generated before and after DNA isolation by processes such as sonication-based shearing. We excluded these two mutation types from our analysis. The complete raw dataset is available on sheets 2–4 of Supplementary Table 1. For all other classes of nucleotide changes, the mean number of synonymous and nonsense mutations (when applicable) were used as thresholds to assess whether or not a missense mutation was permissive.

Strain constructionAll deletion mutants except ΔsigM and rodA PΔsigM were generated by isothermal assembly followed by direct transformation into B. subtilis. All assembly reactions had three PCR products: an antibiotic resistance gene and upstream and downstream fragments that flank the locus to be deleted. A loxP-flanked kanamycin resistance cassette was amplified using oligonucleotide primers oJM28 and oJM29 and pWX470 as template. Upstream and downstream PCR products were amplified from PY79 genomic DNA using the following oligonucleotide primers: pbpF (oAM38, oAM39, oAM42, oAM43), pbpG (oAM44, oAM45, oAM48, oAM49), pbpD (oAM50, oAM51, oAM54, oAM55), ponA (oJM001, oJM002, oJM003, oJM004), rodA (oDR1069, oDR1124, oDR1125, oDR1072).The ΔsigM deletion was derived from the B. subtilis knockout collection and was confirmed by PCR using oKO000 (within the erythromycin resistance cassette) and an upstream primer oKO82.The rodA PΔsigM mutant was generated by allelic replacement using the pminiMAD-derived plasmid pAM199. This plasmid, harboring mutations in the −10 and −35 elements recognized by SigM46 was transformed into PY79 selecting for resistance to erythromycin. Transformants were grown without selection at 22°C and plated on LB agar at 37°C. Single colonies were screened for loss of the integrated plasmid and sensitivity to erythromycin. Genomic DNA was prepared from four independent Erm(S) colonies and the rodA promoter region was PCR amplified and sequenced (using primer oAM201) to identify strains harboring the SigM promoter mutations.E. coli expression strain CAM333 [C43 ΔponB ΔpbpC ΔmtgA pPBAD-Ulp1(403–621)] was constructed by sequential P1 transduction and antibiotic cassette removal of the FRT-flanked ponB::kan, pbpC::kan, and mtgA::kan deletions from the Keio collection47. The kanamycin resistance cassettes were excised by expression of FLP recombinase from a temperature-sensitive plasmid at 30°C, and the plasmid was subsequently lost by nonselective growth at 37°C. All three deletions were confirmed by PCR using primers flanking the deleted gene (ponB: primers TB1290 and TB1291, pbpC: primers TB633 and TB644, mtgA: primers TB635 and TB636). The strain was then transformed with pAM174 [PBAD-Ulp1(403–621) (cat) p15A ori].

Strain construction

All deletion mutants except ΔsigM and rodA PΔsigM were generated by isothermal assembly followed by direct transformation into B. subtilis. All assembly reactions had three PCR products: an antibiotic resistance gene and upstream and downstream fragments that flank the locus to be deleted. A loxP-flanked kanamycin resistance cassette was amplified using oligonucleotide primers oJM28 and oJM29 and pWX470 as template. Upstream and downstream PCR products were amplified from PY79 genomic DNA using the following oligonucleotide primers: pbpF (oAM38, oAM39, oAM42, oAM43), pbpG (oAM44, oAM45, oAM48, oAM49), pbpD (oAM50, oAM51, oAM54, oAM55), ponA (oJM001, oJM002, oJM003, oJM004), rodA (oDR1069, oDR1124, oDR1125, oDR1072).

The ΔsigM deletion was derived from the B. subtilis knockout collection and was confirmed by PCR using oKO000 (within the erythromycin resistance cassette) and an upstream primer oKO82.

The rodA PΔsigM mutant was generated by allelic replacement using the pminiMAD-derived plasmid pAM199. This plasmid, harboring mutations in the −10 and −35 elements recognized by SigM46 was transformed into PY79 selecting for resistance to erythromycin. Transformants were grown without selection at 22°C and plated on LB agar at 37°C. Single colonies were screened for loss of the integrated plasmid and sensitivity to erythromycin. Genomic DNA was prepared from four independent Erm(S) colonies and the rodA promoter region was PCR amplified and sequenced (using primer oAM201) to identify strains harboring the SigM promoter mutations.

E. coli expression strain CAM333 [C43 ΔponB ΔpbpC ΔmtgA pPBAD-Ulp1(403–621)] was constructed by sequential P1 transduction and antibiotic cassette removal of the FRT-flanked ponB::kan, pbpC::kan, and mtgA::kan deletions from the Keio collection47. The kanamycin resistance cassettes were excised by expression of FLP recombinase from a temperature-sensitive plasmid at 30°C, and the plasmid was subsequently lost by nonselective growth at 37°C. All three deletions were confirmed by PCR using primers flanking the deleted gene (ponB: primers TB1290 and TB1291, pbpC: primers TB633 and TB644, mtgA: primers TB635 and TB636). The strain was then transformed with pAM174 [PBAD-Ulp1(403–621) (cat) p15A ori].

Plasmid constructionspAM84 [amyE::Phyperspank-rodA-his10 (spec)(amp)] was generated in a two-way ligation of a HindIII-NheI rodA-his10 PCR product (amplified in sequential PCR steps with oligonucleotide primers oAM231 and oAM114 followed by oAM231 and oAM233 from PY79 genomic DNA) and pDR111 (amyE::Phyperspank spec amp) cut with HindIII and NheI.pAM84b [amyE::Phyperspank-rodA(W105A)-his10 (spec)(amp)] was generated by site-directed mutagenesis from pAM84 using oDR1107pAM84c [amyE::Phyperspank-rodA(D280A)-his10 (spec)(amp)] was generated by site-directed mutagenesis from pAM84 using oDR1104pAM199 [rodA PΔsigM pminiMAD (erm) (amp)] was generated in a three-way isothermal assembly reaction containing: rodA promoter fragment 1 (amplified from PY79 genomic DNA with oligonucleotide primers oAM517 and oAM518); rodA promoter fragment 2 (amplified from PY79 genomic DNA with oligonucleotide primers oAM519 and oAM520) and pminiMAD (erm, amp) digested with BamHI and HindIII. The σM-dependent rodA promoter (with −35 and −10 elements in bold) was changed from:5′ TCAATCGAAACATTTCGGTTTATGATACGTCATATTTCGT 3′ to5′ TCAATCTGCGCATTTCGGTTTATGATATACTATATTTCGT 3′pAM14 [ycgO::Pspank*-ponA (cat) (amp)] was generated in a two-way ligation reaction with a PCR product containing ponA (amplified from PY79 genomic DNA using oligonucleotide primers oAM62 and oAM64) and pAM12 (ycgO::Pspank* cat amp) digested with XmaI and SpeI.pER135 [ycgO::Pspank-rodA (erm) (amp)] was generated in a two-way ligation reaction with a PCR product containing rodA (amplified from PY79 genomic DNA using oligonucleotide primers oDR1122 and oDR1123) and pER62 (ycgO::Pspank erm amp) digested with HindIII and BamHI.pER133 [amyE::PxylA-rodA (spec) (amp)] was generated in a two-way ligation reaction with a PCR product containing rodA (amplified from PY79 genomic DNA using oligonucleotide primers oDR1122 and oDR1123) and pDR150 (amyE::PxylA spec amp) digested with HindIII and BamHI.pER155 [amyE::PxylA-rodA (E288A) (spec) (amp)] was generated by site-directed mutagenesis of pER133 using oDR1119.pER146 [amyE::Pxyl-rodA (D280A) (spec) (amp)] was generated by site-directed mutagenesis of pER133 using oDR1104.pER160 [amyE::Pxyl-rodA (W105A) (spec) (amp)] was generated by site-directed mutagenesis of pER133 using oDR1107.pER157 [amyE::PxylA-rodA (E117A) (spec) (amp)] was generated by site-directed mutagenesis of pER133 using oDR1128.pAM174 [PBAD-Ulp1(403–621)
(cat) p15A ori] was generated in a two-piece isothermal assembly reaction with a His6-Ulp1(403–621) PCR product (amplified from pTB145 (PT7-His6-Ulp1(403–621)) using oligonucleotide primers oAM453 and oAM454) and pMT30 (PBAD-sfgfp p15A cat) digested with NdeI and HindIII.pAM172 [PT7-His6-SUMO-FLAG-RodA (amp)] was generated in a three-piece isothermal assembly reaction with FLAG-RodA fragment 1 (amplified from FLAG-RodA gBlock (IDT) using oligonucleotide primers oAM449 and oAM450), FLAG-RodA fragment 2 (amplified from PY79 genomic DNA using oAM446 and oAM447) and pTB146 (PT7-His6-SUMO amp) digested with SapI and XhoI.pAM172b [PT7-His6-SUMO-FLAG-RodA(W105A) (amp)] was generated by site-directed mutagenesis of pAM172 using oDR1107.pAM172c [PT7-His6-SUMO-FLAG-RodA(D280A) (amp)] was generated by site-directed mutagenesis of pAM172 using oDR1104.The sequence of the FLAG-RodA gBlock was: ATGGAGAATCTGTATTTTCAGGACTACAAAGACGACGACGACCTGGAAGTTCTGTTCCAGGGTCCGGGTGGTTCTTCTAGTCGATATAAGAAACAGCAAAGCCCCTTTTACCAGGGGGATTTGATTTTTATATTTGGTGTGTTTTTTATTATAAGCGTAGTATCAATATATGCTGCCGGTCAGTTTGGGCAGTATGGGAACACGGATTGGATTCAGCAAATTGTGTTTTATCTTTTGGGGGCTGTAGCCATTACAGTTCTTTTGTACTTTGATTTAGAACAGCTTGAAAAGTTAAGTTTATATATTTTTATCATTGGTATTTTATCTTTGATTATTCTTAAAATCAGTCCCGAGTCTATTGCACCTGTTATTAAAGGGGCAAAAAGTTGGTTTAGGATCGGTAGAATAACAATACAGCCGTCAGAGTTTATGAAGGTTGGTTTGATTATGATGCTTGCTTCAGTTATTGGAAAAGCAAATCCTAAAGGAGTTCGGACGC

Plasmid constructions

pAM84 [amyE::Phyperspank-rodA-his10 (spec)(amp)] was generated in a two-way ligation of a HindIII-NheI rodA-his10 PCR product (amplified in sequential PCR steps with oligonucleotide primers oAM231 and oAM114 followed by oAM231 and oAM233 from PY79 genomic DNA) and pDR111 (amyE::Phyperspank spec amp) cut with HindIII and NheI.

pAM84b [amyE::Phyperspank-rodA(W105A)-his10 (spec)(amp)] was generated by site-directed mutagenesis from pAM84 using oDR1107

pAM84c [amyE::Phyperspank-rodA(D280A)-his10 (spec)(amp)] was generated by site-directed mutagenesis from pAM84 using oDR1104

pAM199 [rodA PΔsigM pminiMAD (erm) (amp)] was generated in a three-way isothermal assembly reaction containing: rodA promoter fragment 1 (amplified from PY79 genomic DNA with oligonucleotide primers oAM517 and oAM518); rodA promoter fragment 2 (amplified from PY79 genomic DNA with oligonucleotide primers oAM519 and oAM520) and pminiMAD (erm, amp) digested with BamHI and HindIII. The σM-dependent rodA promoter (with −35 and −10 elements in bold) was changed from:

5′ TCAATCGAAACATTTCGGTTTATGATACGTCATATTTCGT 3′ to

5′ TCAATCTGCGCATTTCGGTTTATGATATACTATATTTCGT 3′

pAM14 [ycgO::Pspank*-ponA (cat) (amp)] was generated in a two-way ligation reaction with a PCR product containing ponA (amplified from PY79 genomic DNA using oligonucleotide primers oAM62 and oAM64) and pAM12 (ycgO::Pspank* cat amp) digested with XmaI and SpeI.

pER135 [ycgO::Pspank-rodA (erm) (amp)] was generated in a two-way ligation reaction with a PCR product containing rodA (amplified from PY79 genomic DNA using oligonucleotide primers oDR1122 and oDR1123) and pER62 (ycgO::Pspank erm amp) digested with HindIII and BamHI.

pER133 [amyE::PxylA-rodA (spec) (amp)] was generated in a two-way ligation reaction with a PCR product containing rodA (amplified from PY79 genomic DNA using oligonucleotide primers oDR1122 and oDR1123) and pDR150 (amyE::PxylA spec amp) digested with HindIII and BamHI.

pER155 [amyE::PxylA-rodA (E288A) (spec) (amp)] was generated by site-directed mutagenesis of pER133 using oDR1119.

pER146 [amyE::Pxyl-rodA (D280A) (spec) (amp)] was generated by site-directed mutagenesis of pER133 using oDR1104.

pER160 [amyE::Pxyl-rodA (W105A) (spec) (amp)] was generated by site-directed mutagenesis of pER133 using oDR1107.

pER157 [amyE::PxylA-rodA (E117A) (spec) (amp)] was generated by site-directed mutagenesis of pER133 using oDR1128.

pAM174 [PBAD-Ulp1(403–621)
(cat) p15A ori] was generated in a two-piece isothermal assembly reaction with a His6-Ulp1(403–621) PCR product (amplified from pTB145 (PT7-His6-Ulp1(403–621)) using oligonucleotide primers oAM453 and oAM454) and pMT30 (PBAD-sfgfp p15A cat) digested with NdeI and HindIII.

pAM172 [PT7-His6-SUMO-FLAG-RodA (amp)] was generated in a three-piece isothermal assembly reaction with FLAG-RodA fragment 1 (amplified from FLAG-RodA gBlock (IDT) using oligonucleotide primers oAM449 and oAM450), FLAG-RodA fragment 2 (amplified from PY79 genomic DNA using oAM446 and oAM447) and pTB146 (PT7-His6-SUMO amp) digested with SapI and XhoI.

pAM172b [PT7-His6-SUMO-FLAG-RodA(W105A) (amp)] was generated by site-directed mutagenesis of pAM172 using oDR1107.

pAM172c [PT7-His6-SUMO-FLAG-RodA(D280A) (amp)] was generated by site-directed mutagenesis of pAM172 using oDR1104.

The sequence of the FLAG-RodA gBlock was: ATGGAGAATCTGTATTTTCAGGACTACAAAGACGACGACGACCTGGAAGTTCTGTTCCAGGGTCCGGGTGGTTCTTCTAGTCGATATAAGAAACAGCAAAGCCCCTTTTACCAGGGGGATTTGATTTTTATATTTGGTGTGTTTTTTATTATAAGCGTAGTATCAATATATGCTGCCGGTCAGTTTGGGCAGTATGGGAACACGGATTGGATTCAGCAAATTGTGTTTTATCTTTTGGGGGCTGTAGCCATTACAGTTCTTTTGTACTTTGATTTAGAACAGCTTGAAAAGTTAAGTTTATATATTTTTATCATTGGTATTTTATCTTTGATTATTCTTAAAATCAGTCCCGAGTCTATTGCACCTGTTATTAAAGGGGCAAAAAGTTGGTTTAGGATCGGTAGAATAACAATACAGCCGTCAGAGTTTATGAAGGTTGGTTTGATTATGATGCTTGCTTCAGTTATTGGAAAAGCAAATCCTAAAGGAGTTCGGACGC

Extended DataExtended Data Fig. 1Rod complex dynamics in the presence of different antibioticsRepresentative kymographs showing cessation of GFP-Mbl particle movement in a, wild-type and b, the quadruple (Δ4) aPBP mutant upon treatment with 10 μg/mL ampicillin. c, Rod complex motion is unaffected by moenomycin treatment. Cells harboring GFP-Mbl were pre-treated with 1 μg/mL moenomycin for 15 min then imaged in the presence of moenomycin. Representative kymographs of individual GFP-Mbl particles are shown. (d) Maximum intensity projection of the timelapse in Supplementary Video 5. Scale bar indicates 1 μm. Data are representative of 3 biological replicates.Extended Data Fig. 2Conserved neighborhood architecture for loci encoding SEDS proteins and bPBPsa, Diagrams depicting the genomic context of genes encoding SEDS proteins (red) in a diverse set of bacterial taxa. Genes encoding bPBPs are depicted in blue and are frequently located adjacent to SEDS loci. These SEDS-bPBP pairs are often found in the context of the mreBCD operon (faded pink), suggesting that these orthologs function in cell elongation. SEDS and bPBP loci are also frequently present in the cluster of cell wall synthesis and cell division genes exemplified by the E. coli dcw cluster (faded green) and these orthologs likely function in cell division. Unrelated genes are shown as white triangles. Phylogenetic tree was constructed in PhyLoT (phylot.biobyte.de) and visualized in iToL (itol.embl.de). b, Histogram showing the genetic distance (on log10 scale) between 2,958 SEDS loci (red) and the nearest bPBP locus (blue). Two commonly observed SEDS-bPBP neighborhood architectures are depicted. Distances between SEDS and the nearest recA gene are shown in yellow as a negative control. SEDS and bPBP loci were identified using tblastn with five diverse members of each family used as the query.Extended Data Fig. 3RodA overexpression partially suppresses the phenotypes of the quadruple aPBP mutanta, Growth curves of wild-type (WT), the quadruple (Δ4) aPBP mutant, and the Δ4 mutant overexpressing rodA-his10, representative of three biological replicates. b, Quantification of indicated cytological phenotypes, n=500. Error bars denote s.e.m. c, Live-dead (propidium iodide) staining of strains analyzed in a. Dead cells or cells with membrane integrity defects were visualized by fluorescence microscopy. Images representative of 3 biological replicates. Scale bar indicates 5 μm. d, Immunoblot analysis of RodA-His10 levels for the three strains in a as well as the Δ4 strain overexpressing nonfunctional mutants W105A and D280A. A fusion of his10 to rodA at its native locus was used to assess wild-type RodA levels (lane 2). Sigma A (σA) levels are shown to control for loading. e, Detergent solubilization of RodA-His10 from B. subtilis membranes using CHAPS. Anti-His immunoblot showing the relative amounts of solubilized RodA-His10 after overnight incubation with 2% CHAPS and ultracentrifugation at 100,000g.Extended Data Fig. 4Polymers synthesized by RodA in vitro are susceptible to muramidase digestiona, To determine whether the products of RodA activity are glycan strands, their susceptibility to cleavage by the muramidase mutanolysin was investigated. Mutanolysin specifically cleaves the β(1,4) linkage between N-acetylmuramic acid and N-acetylglucosamine in PG glycan chains. 0.2 μM FLAG-RodA was incubated with 4 μM synthetic lipid II for 1 h, then quenched by boiling for 2 min. The products were subjected to overnight digestion with mutanolysin (0.1 mg/ml) at 37°C, and analyzed by SDS-PAGE. Lipid II, and undigested RodA products are shown for comparison. Data representative of two technical replicates. b, The reaction catalyzed by RodA can be inhibited by vancomycin (50 μg/mL), which binds and sequesters the lipid II substrate. Graph denotes the mean from 3 technical replicates, error bars show s.e.m.Extended Data Fig. 5Critical amino acid residues in RodA identified by MutSeqa, Topological map of the RodA protein. The extent to which each amino acid residue tolerated mutations based on the MutSeq screen are indicated. Residues that tolerated a spectrum of amino acid changes are shown in grey. Residues that did not tolerate any mutations are shown in red. Residues that only tolerated conservative changes (conservation of charge, hydrophobicity, or functional groups) are in purple. Residues that had limited mutability but tolerated a nonconservative substitution are shown in pink. The complete dataset can be found in Supplementary Table 1. b, Multiple sequence alignment (created using ESPRIPT: http://espript.ibcp.fr/) of 14 diverse SEDS proteins with W105 and D280 residues highlighted.Extended Data Fig. 6Validation of critical amino acids identified in the MutSeq screena, Schematic of the strain used to test a subset of critical amino acid residues in rodA identified by MutSeq. A wild-type copy of rodA was placed under IPTG-inducible control at an ectopic chromosomal locus (ycgO) and the native copy of rodA was deleted. The mutant alleles to be tested were placed under xylose-inducible control at a second ectopic locus (amyE). As a positive control, a wild-type allele under xylose control was integrated at the second locus. The empty vector was used as a negative control. b, Immunoblot analysis of the RodA mutants expressed as His10-tagged fusions under xylose-inducible control, representative of two biological replicates. ParB levels are shown to control for loading. c, Growth curves of strains expressing mutant rodA alleles. Each strain was grown at 37°C in CH medium in the presence of 500 μM IPTG to maintain expression of wild-type rodA. Cultures were then washed three times in medium lacking inducer, diluted to OD600 = 0.02 in CH medium with 10mM xylose, and growth was monitored. Growth curves are representative of 2 biological replicates. d, Morphological phenotypes of the strains analyzed in (c) were examined by fluorescence microscopy at the indicated time points (below the images) after resuspension in xylose-containing medium. Fluorescent images of cell membranes stained with TMA-DPH and phase contrast images are shown. A mutation in the highly conserved residue E288 that was found to be mutable by Mutseq was included as a negative control. Consistent with the Mutseq analysis, substitution to alanine (E288A) supported wild-type growth rates and had no impact on cell morphology.Extended Data Fig. 7RodA overexpression suppresses the synthetic lethality of ΔsigM and the quadruple aPBP mutanta, LB agar plates onto which a ΔsigM Δ4 aPBP strain harboring an IPTG-inducible allele of rodA was streaked in the presence and absence of 15 μM IPTG and incubated at 37°C overnight. b, The rodA allele containing mutations in its SigM-dependent promoter (rodA ΔPsigM) grows in a manner indistinguishable from wild-type in the absence of moenomycin. Wild-type (WT) and the rodA ΔP sigM mutant were grown in LB and OD600 was monitored continuously. c, The rodA ΔP sigM mutant has a normal rod-shaped morphology. Phase contrast image of cells harboring the rodA ΔPsigM promoter mutant grown to mid-exponential phase in LB medium. All data representative of two biological replicates. Scale bar indicates 1 μm.Extended Data Fig. 8SEDS proteins and bPBPs are more widely conserved than aPBPsPhylogenetic tree showing distribution of SEDS proteins, bPBPs, and aPBPs in a diverse set of 1773 bacterial taxa. The amino acid sequences of five members of each family were used as queries in a BLASTp search against the NCBI nr database with an E-value cutoff of 10−4. The phylogenetic tree was constructed using PhyloT (phylot.bio-byte.de) and BLASTp results were plotted against the tree. The occurrence of a SEDS protein is indicated in red, a bPBP in blue, and an aPBP in green. The tree was visualized and annotated using iToL (itol.embl.de). Clades whose genomes contain a SEDS protein and bPBP but lack aPBPs are indicated. The PG-less Mycoplasma lack all three proteins.

Extended Data

Rod complex dynamics in the presence of different antibiotics

Representative kymographs showing cessation of GFP-Mbl particle movement in a, wild-type and b, the quadruple (Δ4) aPBP mutant upon treatment with 10 μg/mL ampicillin. c, Rod complex motion is unaffected by moenomycin treatment. Cells harboring GFP-Mbl were pre-treated with 1 μg/mL moenomycin for 15 min then imaged in the presence of moenomycin. Representative kymographs of individual GFP-Mbl particles are shown. (d) Maximum intensity projection of the timelapse in Supplementary Video 5. Scale bar indicates 1 μm. Data are representative of 3 biological replicates.

Conserved neighborhood architecture for loci encoding SEDS proteins and bPBPs

a, Diagrams depicting the genomic context of genes encoding SEDS proteins (red) in a diverse set of bacterial taxa. Genes encoding bPBPs are depicted in blue and are frequently located adjacent to SEDS loci. These SEDS-bPBP pairs are often found in the context of the mreBCD operon (faded pink), suggesting that these orthologs function in cell elongation. SEDS and bPBP loci are also frequently present in the cluster of cell wall synthesis and cell division genes exemplified by the E. coli dcw cluster (faded green) and these orthologs likely function in cell division. Unrelated genes are shown as white triangles. Phylogenetic tree was constructed in PhyLoT (phylot.biobyte.de) and visualized in iToL (itol.embl.de). b, Histogram showing the genetic distance (on log10 scale) between 2,958 SEDS loci (red) and the nearest bPBP locus (blue). Two commonly observed SEDS-bPBP neighborhood architectures are depicted. Distances between SEDS and the nearest recA gene are shown in yellow as a negative control. SEDS and bPBP loci were identified using tblastn with five diverse members of each family used as the query.

RodA overexpression partially suppresses the phenotypes of the quadruple aPBP mutant

a, Growth curves of wild-type (WT), the quadruple (Δ4) aPBP mutant, and the Δ4 mutant overexpressing rodA-his10, representative of three biological replicates. b, Quantification of indicated cytological phenotypes, n=500. Error bars denote s.e.m. c, Live-dead (propidium iodide) staining of strains analyzed in a. Dead cells or cells with membrane integrity defects were visualized by fluorescence microscopy. Images representative of 3 biological replicates. Scale bar indicates 5 μm. d, Immunoblot analysis of RodA-His10 levels for the three strains in a as well as the Δ4 strain overexpressing nonfunctional mutants W105A and D280A. A fusion of his10 to rodA at its native locus was used to assess wild-type RodA levels (lane 2). Sigma A (σA) levels are shown to control for loading. e, Detergent solubilization of RodA-His10 from B. subtilis membranes using CHAPS. Anti-His immunoblot showing the relative amounts of solubilized RodA-His10 after overnight incubation with 2% CHAPS and ultracentrifugation at 100,000g.

Polymers synthesized by RodA in vitro are susceptible to muramidase digestion

a, To determine whether the products of RodA activity are glycan strands, their susceptibility to cleavage by the muramidase mutanolysin was investigated. Mutanolysin specifically cleaves the β(1,4) linkage between N-acetylmuramic acid and N-acetylglucosamine in PG glycan chains. 0.2 μM FLAG-RodA was incubated with 4 μM synthetic lipid II for 1 h, then quenched by boiling for 2 min. The products were subjected to overnight digestion with mutanolysin (0.1 mg/ml) at 37°C, and analyzed by SDS-PAGE. Lipid II, and undigested RodA products are shown for comparison. Data representative of two technical replicates. b, The reaction catalyzed by RodA can be inhibited by vancomycin (50 μg/mL), which binds and sequesters the lipid II substrate. Graph denotes the mean from 3 technical replicates, error bars show s.e.m.

Critical amino acid residues in RodA identified by MutSeq

a, Topological map of the RodA protein. The extent to which each amino acid residue tolerated mutations based on the MutSeq screen are indicated. Residues that tolerated a spectrum of amino acid changes are shown in grey. Residues that did not tolerate any mutations are shown in red. Residues that only tolerated conservative changes (conservation of charge, hydrophobicity, or functional groups) are in purple. Residues that had limited mutability but tolerated a nonconservative substitution are shown in pink. The complete dataset can be found in Supplementary Table 1. b, Multiple sequence alignment (created using ESPRIPT: http://espript.ibcp.fr/) of 14 diverse SEDS proteins with W105 and D280 residues highlighted.

Validation of critical amino acids identified in the MutSeq screen

a, Schematic of the strain used to test a subset of critical amino acid residues in rodA identified by MutSeq. A wild-type copy of rodA was placed under IPTG-inducible control at an ectopic chromosomal locus (ycgO) and the native copy of rodA was deleted. The mutant alleles to be tested were placed under xylose-inducible control at a second ectopic locus (amyE). As a positive control, a wild-type allele under xylose control was integrated at the second locus. The empty vector was used as a negative control. b, Immunoblot analysis of the RodA mutants expressed as His10-tagged fusions under xylose-inducible control, representative of two biological replicates. ParB levels are shown to control for loading. c, Growth curves of strains expressing mutant rodA alleles. Each strain was grown at 37°C in CH medium in the presence of 500 μM IPTG to maintain expression of wild-type rodA. Cultures were then washed three times in medium lacking inducer, diluted to OD600 = 0.02 in CH medium with 10mM xylose, and growth was monitored. Growth curves are representative of 2 biological replicates. d, Morphological phenotypes of the strains analyzed in (c) were examined by fluorescence microscopy at the indicated time points (below the images) after resuspension in xylose-containing medium. Fluorescent images of cell membranes stained with TMA-DPH and phase contrast images are shown. A mutation in the highly conserved residue E288 that was found to be mutable by Mutseq was included as a negative control. Consistent with the Mutseq analysis, substitution to alanine (E288A) supported wild-type growth rates and had no impact on cell morphology.

RodA overexpression suppresses the synthetic lethality of ΔsigM and the quadruple aPBP mutant

a, LB agar plates onto which a ΔsigM Δ4 aPBP strain harboring an IPTG-inducible allele of rodA was streaked in the presence and absence of 15 μM IPTG and incubated at 37°C overnight. b, The rodA allele containing mutations in its SigM-dependent promoter (rodA ΔPsigM) grows in a manner indistinguishable from wild-type in the absence of moenomycin. Wild-type (WT) and the rodA ΔP sigM mutant were grown in LB and OD600 was monitored continuously. c, The rodA ΔP sigM mutant has a normal rod-shaped morphology. Phase contrast image of cells harboring the rodA ΔPsigM promoter mutant grown to mid-exponential phase in LB medium. All data representative of two biological replicates. Scale bar indicates 1 μm.

SEDS proteins and bPBPs are more widely conserved than aPBPs

Phylogenetic tree showing distribution of SEDS proteins, bPBPs, and aPBPs in a diverse set of 1773 bacterial taxa. The amino acid sequences of five members of each family were used as queries in a BLASTp search against the NCBI nr database with an E-value cutoff of 10−4. The phylogenetic tree was constructed using PhyloT (phylot.bio-byte.de) and BLASTp results were plotted against the tree. The occurrence of a SEDS protein is indicated in red, a bPBP in blue, and an aPBP in green. The tree was visualized and annotated using iToL (itol.embl.de). Clades whose genomes contain a SEDS protein and bPBP but lack aPBPs are indicated. The PG-less Mycoplasma lack all three proteins.

Supplementary Materialsupp_fig1supp_guidesupp_table1supp_table2supp_table3supp_table4supp_table5vid1vid2vid3vid4vid5

Supplementary Material
