---
pmid: '24930953'
title: Biochemical and cellular analysis of human variants of the DYT1 dystonia protein,
  TorsinA/TOR1A.
authors:
- Hettich J
- Ryan SD
- de Souza ON
- Saraiva Macedo Timmers LF
- Tsai S
- Atai NA
- da Hora CC
- Zhang X
- Kothary R
- Snapp E
- Ericsson M
- Grundmann K
- Breakefield XO
- Nery FC
journal: Hum Mutat
year: '2014'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC4134760
doi: 10.1002/humu.22602
---

# Biochemical and cellular analysis of human variants of the DYT1 dystonia protein, TorsinA/TOR1A.
**Authors:** Hettich J, Ryan SD, de Souza ON, Saraiva Macedo Timmers LF, Tsai S, Atai NA, da Hora CC, Zhang X, Kothary R, Snapp E, Ericsson M, Grundmann K, Breakefield XO, Nery FC
**Journal:** Hum Mutat (2014)
**DOI:** [10.1002/humu.22602](https://doi.org/10.1002/humu.22602)
**PMC:** [PMC4134760](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4134760/)

## Abstract

1. Hum Mutat. 2014 Sep;35(9):1101-13. doi: 10.1002/humu.22602. Epub 2014 Jul 17.

Biochemical and cellular analysis of human variants of the DYT1 dystonia 
protein, TorsinA/TOR1A.

Hettich J(1), Ryan SD, de Souza ON, Saraiva Macedo Timmers LF, Tsai S, Atai NA, 
da Hora CC, Zhang X, Kothary R, Snapp E, Ericsson M, Grundmann K, Breakefield 
XO, Nery FC.

Author information:
(1)Molecular Neurogenetics Unit, Department of Neurology and Center for 
Molecular Imaging Research, Department of Radiology, Massachusetts General 
Hospital and Program in Neuroscience, Harvard Medical School, Boston, 
Massachusetts; Department of Medical Genetics and Applied Genomics, University 
of Tuebingen, Tübingen, Germany.

Early-onset dystonia is associated with the deletion of one of a pair of 
glutamic acid residues (c.904_906delGAG/c.907_909delGAG; p.Glu302del/Glu303del; 
ΔE 302/303) near the carboxyl-terminus of torsinA, a member of the AAA(+) 
protein family that localizes to the endoplasmic reticulum lumen and nuclear 
envelope. This deletion commonly underlies early-onset DYT1 dystonia. While the 
role of the disease-causing mutation, torsinAΔE, has been established through 
genetic association studies, it is much less clear whether other rare human 
variants of torsinA are pathogenic. Two missense variations have been described 
in single patients: R288Q (c.863G>A; p.Arg288Gln; R288Q) identified in a patient 
with onset of severe generalized dystonia and myoclonus since infancy and F205I 
(c.613T>A, p.Phe205Ile; F205I) in a psychiatric patient with late-onset focal 
dystonia. In this study, we have undertaken a series of analyses comparing the 
biochemical and cellular effects of these rare variants to torsinAΔE and 
wild-type (wt) torsinA to reveal whether there are common dysfunctional 
features. The results revealed that the variants, R288Q and F205I, are more 
similar in their properties to torsinAΔE protein than to torsinAwt. These 
findings provide functional evidence for the potential pathogenic nature of 
these rare sequence variants in the TOR1A gene, thus implicating these 
pathologies in the development of dystonia.

© 2014 WILEY PERIODICALS, INC.

DOI: 10.1002/humu.22602
PMCID: PMC4134760
PMID: 24930953 [Indexed for MEDLINE]

Conflict of interest statement: Conflict of Interest The authors declare no 
conflict of interest.

## Full Text

IntroductionDystonia is characterized by involuntary muscle contractions, which cause sustained twisting and repetitive movements, as well as abnormal postures [Fahn, 1988]. Early onset DYT1 generalized dystonia is the most common and severe form of the hereditary dystonias with autosomal-dominant inheritance and a reduced penetrance of 30–40% [Bressman et al., 1989]. In most cases, DYT1 is caused by a heterozygous 3 bp deletion (ΔGAG) in exon 5 of the TOR1A gene (MIM# 605204) corresponding to loss of one of the two adjacent glutamic acid residues at positions 302 or 303 in torsinA, torsinAΔE (c.904_906delGAG/c.907_909delGAG; p.Glu302del/Glu303del) [Ozelius et al., 1997]. Additional sequence variants which change an amino acid (a.a.) in torsinA have been found in the heterozygous state in other patients with dystonia, but their role in causing the disease has not yet been determined. These include an 18 bp deletion (c.966_983del18; p.Val322_Tyr328delinsVal) in exon 5, encoding a.a. 323 – 328, identified in a patient presenting with atypical dystonia with myoclonic features [Leung et al., 2001]. The potential pathogenic nature of this mutation is in doubt, however, as this patient also had a mutation in the epsilon sarcoglycan gene responsible for myoclonus dystonia [Doheny et al., 2002]. An aspartic acid (D) 216 to histidine (H) (c.646G>C; p.Asp216His) change has been found to influence penetrance of the ΔGAG mutation, but is not independently associated with dystonia onset [Kock et al., 2006; Risch et al., 2007; Kamm et al., 2008]. A 4 bp deletion (c.934_937delAGAG; p.Arg312_Val313delinsPhefs) causing a frameshift truncation was identified in an anonymous control sample for whom no neurologic evaluation was available [Kabakci et al., 2004] and later also found in a patient with late onset myoclonic and dystonic features with signs of Parkinson’s disease [Ritz et al., 2009]. Other missense variants reported include an arginine (R) 288 to glutamine (Q) substitution (c.863G>A; p.Arg288Gln) in a patient with severe generalized dystonia [Zirn et al., 2008], and a phenylalanine (F) 205 to isoleucine (I) substitution (c.613T>A, p.Phe205Ile) in a psychiatric patient with late-onset focal dystonia [Calakos et al., 2010].The TOR1A gene belongs to the family of AAA+ proteins (ATPase associated with variety of cellular activities) [Ozelius et al., 1997; Neuwald et al., 1999], most of which form hexameric rings in their active conformation, often assembled from inactive dimers [Beuron et al., 2003; DeLaBarre and Brunger, 2003; Krzywda et al., 2002]. The nucleotide binding sites of AAA+ proteins lie at the interface between subunits permitting determinants from adjacent subunits to contribute to nucleotide hydrolysis. Like other AAA+ proteins torsinA may be functional as multimers with torsinAΔE inhibiting the activity of wild-type (wt) torsinA [Kustedjo et al., 2000; Gordon and Gonzalez-Alegre, 2008; Breakefield et al., 2001; Torres et al., 2004; Goodchild et al., 2005; Hewett et al., 2007; Hewett et al., 2008; Chen et al., 2010a; Nery et al., 2011].TorsinA is localized in the lumen of the endoplasmic reticulum (ER) and contiguous nuclear envelope (NE) of most cells, with torsinAΔE being more concentrated in the NE and, when overexpressed, producing whorled membrane inclusions derived from NE/ER membranes [Hewett et al., 2000; Hewett et al., 2003; Kustedjo et al., 2000; Gonzalez-Alegre and Paulson, 2004; Goodchild and Dauer, 2004], as does overexpression of a torsinB variant [Rose et al., 2014]. Although the function of torsinA is still unclear, a variety of properties in cultured cells change upon expression of torsinAΔE and/or down-regulation of torsinA levels consistent with a role in membrane-cytoskeleton interactions. In the ER, torsinA has been associated with processing of proteins through the secretory pathway [Torres et al., 2004; Hewett et al., 2007; Hewett et al., 2008], degradation of proteins by ER associated degradation (ERAD) [Esapa et al., 2007; Nery et al., 2011], the ER stress response [Chen et al., 2010a; Nery et al., 2011] and extension of neurites [Hewett et al., 2006; Nery et al., 2008; Atai et al., 2012]. In the NE, torsinA is involved in nuclear polarization during cell migration [Nery et al., 2008], morphology of NE membranes [Naismith et al., 2004; Goodchild et al., 2005], and nuclear egress of large ribonuclear protein (RNP) complexes [Jokhi et al., 2013] and herpes simplex virus (HSV) [Maric et al., 2011].To better understand the impact of torsinA variants associated with dystonia (torsinAΔE, torsinAF205I and torsinAR288Q) on cellular phenotype, and the mechanism of action of torsinA, we used homology modeling approaches as well as biochemical and cell based assays to compare activities of torsinAwt with these variants. This study confirms that these torsinA variants have features similar to torsinAE and hence are implicated in causing dystonia.

Introduction

Dystonia is characterized by involuntary muscle contractions, which cause sustained twisting and repetitive movements, as well as abnormal postures [Fahn, 1988]. Early onset DYT1 generalized dystonia is the most common and severe form of the hereditary dystonias with autosomal-dominant inheritance and a reduced penetrance of 30–40% [Bressman et al., 1989]. In most cases, DYT1 is caused by a heterozygous 3 bp deletion (ΔGAG) in exon 5 of the TOR1A gene (MIM# 605204) corresponding to loss of one of the two adjacent glutamic acid residues at positions 302 or 303 in torsinA, torsinAΔE (c.904_906delGAG/c.907_909delGAG; p.Glu302del/Glu303del) [Ozelius et al., 1997]. Additional sequence variants which change an amino acid (a.a.) in torsinA have been found in the heterozygous state in other patients with dystonia, but their role in causing the disease has not yet been determined. These include an 18 bp deletion (c.966_983del18; p.Val322_Tyr328delinsVal) in exon 5, encoding a.a. 323 – 328, identified in a patient presenting with atypical dystonia with myoclonic features [Leung et al., 2001]. The potential pathogenic nature of this mutation is in doubt, however, as this patient also had a mutation in the epsilon sarcoglycan gene responsible for myoclonus dystonia [Doheny et al., 2002]. An aspartic acid (D) 216 to histidine (H) (c.646G>C; p.Asp216His) change has been found to influence penetrance of the ΔGAG mutation, but is not independently associated with dystonia onset [Kock et al., 2006; Risch et al., 2007; Kamm et al., 2008]. A 4 bp deletion (c.934_937delAGAG; p.Arg312_Val313delinsPhefs) causing a frameshift truncation was identified in an anonymous control sample for whom no neurologic evaluation was available [Kabakci et al., 2004] and later also found in a patient with late onset myoclonic and dystonic features with signs of Parkinson’s disease [Ritz et al., 2009]. Other missense variants reported include an arginine (R) 288 to glutamine (Q) substitution (c.863G>A; p.Arg288Gln) in a patient with severe generalized dystonia [Zirn et al., 2008], and a phenylalanine (F) 205 to isoleucine (I) substitution (c.613T>A, p.Phe205Ile) in a psychiatric patient with late-onset focal dystonia [Calakos et al., 2010].

The TOR1A gene belongs to the family of AAA+ proteins (ATPase associated with variety of cellular activities) [Ozelius et al., 1997; Neuwald et al., 1999], most of which form hexameric rings in their active conformation, often assembled from inactive dimers [Beuron et al., 2003; DeLaBarre and Brunger, 2003; Krzywda et al., 2002]. The nucleotide binding sites of AAA+ proteins lie at the interface between subunits permitting determinants from adjacent subunits to contribute to nucleotide hydrolysis. Like other AAA+ proteins torsinA may be functional as multimers with torsinAΔE inhibiting the activity of wild-type (wt) torsinA [Kustedjo et al., 2000; Gordon and Gonzalez-Alegre, 2008; Breakefield et al., 2001; Torres et al., 2004; Goodchild et al., 2005; Hewett et al., 2007; Hewett et al., 2008; Chen et al., 2010a; Nery et al., 2011].

TorsinA is localized in the lumen of the endoplasmic reticulum (ER) and contiguous nuclear envelope (NE) of most cells, with torsinAΔE being more concentrated in the NE and, when overexpressed, producing whorled membrane inclusions derived from NE/ER membranes [Hewett et al., 2000; Hewett et al., 2003; Kustedjo et al., 2000; Gonzalez-Alegre and Paulson, 2004; Goodchild and Dauer, 2004], as does overexpression of a torsinB variant [Rose et al., 2014]. Although the function of torsinA is still unclear, a variety of properties in cultured cells change upon expression of torsinAΔE and/or down-regulation of torsinA levels consistent with a role in membrane-cytoskeleton interactions. In the ER, torsinA has been associated with processing of proteins through the secretory pathway [Torres et al., 2004; Hewett et al., 2007; Hewett et al., 2008], degradation of proteins by ER associated degradation (ERAD) [Esapa et al., 2007; Nery et al., 2011], the ER stress response [Chen et al., 2010a; Nery et al., 2011] and extension of neurites [Hewett et al., 2006; Nery et al., 2008; Atai et al., 2012]. In the NE, torsinA is involved in nuclear polarization during cell migration [Nery et al., 2008], morphology of NE membranes [Naismith et al., 2004; Goodchild et al., 2005], and nuclear egress of large ribonuclear protein (RNP) complexes [Jokhi et al., 2013] and herpes simplex virus (HSV) [Maric et al., 2011].

To better understand the impact of torsinA variants associated with dystonia (torsinAΔE, torsinAF205I and torsinAR288Q) on cellular phenotype, and the mechanism of action of torsinA, we used homology modeling approaches as well as biochemical and cell based assays to compare activities of torsinAwt with these variants. This study confirms that these torsinA variants have features similar to torsinAE and hence are implicated in causing dystonia.

Materials and MethodsPredicted Structural Features of TorsinA VariantsBioinformatics sequence analysis with SignalP 4.1 [Petersen et al., 2011] revealed the nature of the first 60 a.a. residues of torsinA. Residues 1 to 20 were identified as a signal sequence (SS) and residues 21 to 60 were predicted by TMHMM [Moller et al., 2001] to reside in a membrane-associated domain. Hence, these residues were left out of the modeling procedure. We used the homology modeling approach, implemented in the MODELLER [Sali and Blundell, 1993] 9v10 program, to build models of torsinAwt, delE302, delE303, F205I and R288Q based on the 3D structure of the ClpB second AAA+ domain of Thermus thermophilus (T. thermophilus) [Lee et al., 2003]. The protocol used to perform the molecular modeling experiments was: generation of 10 models, from which one model for each torsinA sequence was selected. All models were submitted to the DOPE energy scoring function [Shen and Sali, 2006] implemented in the MODELLER 9v10 aiming to select the best structures. The MOLPROBITY webserver [Chen et al., 2010b] and PROCHECK [Laskowski et al., 1993] were used to verify and validate the stereochemical quality of the models. Intermolecular hydrogen bonds (HB) were calculated and displayed with the program LIGPLOT [Wallace et al., 1995]. All images were generated with the PyMOL program (The PyMOL Molecular Graphics System, Version 1.5.0.4 Schrödinger, LLC).Molecular Dynamics SimulationsFive independent molecular dynamic (MD) simulations and trajectory analyses were carried out with the GROMACS 4.5.5 package [Hess et al., 2008] for wt and variants of torsinA at room temperature or 298.15 K. Each dimeric model contained approximately 180,000 atoms, including ATP, explicit water molecules, and counter ions (Na+ and Cl−) to neutralize the system. The CHARMM27 force field [MacKerell et al., 1997] was used for proteins. Parameters for ATP and TIP3P water molecules were taken from the AMBER force field [Duan et al., 2003 and Jorgensen et al., 1983, respectively]. The LINCS algorithm [Hess et al., 1997] was used to constrain all covalent bonds allowing an integration time step of 2 fs. The systems’ temperature was slowly increased in five steps (50 K, 100 K, 200 K, 250 K, and 298.15 K), each lasting 200 ps. An equilibration phase (5 ns) was obtained without any position restraint within an NPT ensemble at 298.15 K and 1 atm. Each MD simulation lasted for 20.0 ns and average values were calculated for the last 12.0 ns of the MD trajectory during which the system is considered equilibrated.DNA ConstructsHuman torsinA cDNAs (NM_000113.2), 999 base pairs (bp), encoding torsinAwt and ΔE were generated from controls and DYT1 patients and introduced into EcoRI and NotI restriction sites of pcDNA3.1+ (Invitrogen Life Technologies, Darmstadt, GERMANY), generating pcDNA3-torsinAwt, pcDNA3-torsinAΔE (302/303) [Ozelius et al., 1997]. To generate the pcDNA3-torsinAR288Q construct, the Expand Long Template PCR System (Roche Applied Science, Mannheim, GERMANY) was used to introduce the R288Q mutation into the pcDNA3-torsinAwt construct using the primer, 5‘-GTGGAAATGCAGTCCCAAGGCTATGAAATTGATG-3‘ and 5‘-CATCAATTTCATAGCCTTGGGACTGCATTTCCAC-3‘ with a subsequent DpnI digestion. To generate the pcDNA-torsinAF205I construct, the Quickchange Site-Directed Mutagenesis Kit (Stratagene, La Jolla, CA, USA) was used to introduce the F205I mutation into the pcDNAwt construct using the primers, 5′-CCAGAAAGCCATGTTCATAATTCTCAGCAATGCTGGAGC-3′ and 5′-GCTCCAGCATTGCTGAGAATTATGAACATGGCTTTCTGG-3′, as described previously [Calakos et al., 2010].All cDNAs encoding human torsinA variants were also cloned into the NheI and XhoI sites of the CSCW2-IRES lentivirus vector plasmid [Shu et al., 2006]. CSCW2 is a self-inactivating lentiviral vector, which has a cytomegalovirus (CMV) immediate early promoter controlling transgene expression with the monomeric Cherry (mCherry) fluorescent protein encoded after the internal ribosomal entry site (IRES) element [Sena-Esteves et al., 2004]. All final vectors were sequenced to confirm sequence identities. Lentiviral constructs expressing secreted Gaussia luciferase (Gluc) and firefly luciferase (Fluc), CSCW2-Gluc-IRES-GFP [Badr et al., 2007] and CSCW2-Fluc-IRES-mCherry [Maguire et al., 2008] were prepared, as described [Sena-Esteves et al., 2004].Statistical AnalysisData were analyzed using two-tailed Student’s t test (Excel, Microsoft, Redmond, WA, USA) or factorial analysis of variance (ANOVA) as applicable using InStat v3.0 (GraphPad Software, La Jolla, CA, USA). After detection of a statistically significant difference in a given series of treatments by ANOVA, post-hoc Dunnett’s t-tests were performed when appropriate. The p-values <0.05 were considered statistically significant (shown as a single asterisk in the figures); p-values <0.01 were considered highly statistically significant (shown as a double asterisk in the figures), and p-values <0.001 (shown as triple asterisk in the figures).

Materials and Methods

Predicted Structural Features of TorsinA VariantsBioinformatics sequence analysis with SignalP 4.1 [Petersen et al., 2011] revealed the nature of the first 60 a.a. residues of torsinA. Residues 1 to 20 were identified as a signal sequence (SS) and residues 21 to 60 were predicted by TMHMM [Moller et al., 2001] to reside in a membrane-associated domain. Hence, these residues were left out of the modeling procedure. We used the homology modeling approach, implemented in the MODELLER [Sali and Blundell, 1993] 9v10 program, to build models of torsinAwt, delE302, delE303, F205I and R288Q based on the 3D structure of the ClpB second AAA+ domain of Thermus thermophilus (T. thermophilus) [Lee et al., 2003]. The protocol used to perform the molecular modeling experiments was: generation of 10 models, from which one model for each torsinA sequence was selected. All models were submitted to the DOPE energy scoring function [Shen and Sali, 2006] implemented in the MODELLER 9v10 aiming to select the best structures. The MOLPROBITY webserver [Chen et al., 2010b] and PROCHECK [Laskowski et al., 1993] were used to verify and validate the stereochemical quality of the models. Intermolecular hydrogen bonds (HB) were calculated and displayed with the program LIGPLOT [Wallace et al., 1995]. All images were generated with the PyMOL program (The PyMOL Molecular Graphics System, Version 1.5.0.4 Schrödinger, LLC).

Predicted Structural Features of TorsinA Variants

Bioinformatics sequence analysis with SignalP 4.1 [Petersen et al., 2011] revealed the nature of the first 60 a.a. residues of torsinA. Residues 1 to 20 were identified as a signal sequence (SS) and residues 21 to 60 were predicted by TMHMM [Moller et al., 2001] to reside in a membrane-associated domain. Hence, these residues were left out of the modeling procedure. We used the homology modeling approach, implemented in the MODELLER [Sali and Blundell, 1993] 9v10 program, to build models of torsinAwt, delE302, delE303, F205I and R288Q based on the 3D structure of the ClpB second AAA+ domain of Thermus thermophilus (T. thermophilus) [Lee et al., 2003]. The protocol used to perform the molecular modeling experiments was: generation of 10 models, from which one model for each torsinA sequence was selected. All models were submitted to the DOPE energy scoring function [Shen and Sali, 2006] implemented in the MODELLER 9v10 aiming to select the best structures. The MOLPROBITY webserver [Chen et al., 2010b] and PROCHECK [Laskowski et al., 1993] were used to verify and validate the stereochemical quality of the models. Intermolecular hydrogen bonds (HB) were calculated and displayed with the program LIGPLOT [Wallace et al., 1995]. All images were generated with the PyMOL program (The PyMOL Molecular Graphics System, Version 1.5.0.4 Schrödinger, LLC).

Molecular Dynamics SimulationsFive independent molecular dynamic (MD) simulations and trajectory analyses were carried out with the GROMACS 4.5.5 package [Hess et al., 2008] for wt and variants of torsinA at room temperature or 298.15 K. Each dimeric model contained approximately 180,000 atoms, including ATP, explicit water molecules, and counter ions (Na+ and Cl−) to neutralize the system. The CHARMM27 force field [MacKerell et al., 1997] was used for proteins. Parameters for ATP and TIP3P water molecules were taken from the AMBER force field [Duan et al., 2003 and Jorgensen et al., 1983, respectively]. The LINCS algorithm [Hess et al., 1997] was used to constrain all covalent bonds allowing an integration time step of 2 fs. The systems’ temperature was slowly increased in five steps (50 K, 100 K, 200 K, 250 K, and 298.15 K), each lasting 200 ps. An equilibration phase (5 ns) was obtained without any position restraint within an NPT ensemble at 298.15 K and 1 atm. Each MD simulation lasted for 20.0 ns and average values were calculated for the last 12.0 ns of the MD trajectory during which the system is considered equilibrated.

Molecular Dynamics Simulations

Five independent molecular dynamic (MD) simulations and trajectory analyses were carried out with the GROMACS 4.5.5 package [Hess et al., 2008] for wt and variants of torsinA at room temperature or 298.15 K. Each dimeric model contained approximately 180,000 atoms, including ATP, explicit water molecules, and counter ions (Na+ and Cl−) to neutralize the system. The CHARMM27 force field [MacKerell et al., 1997] was used for proteins. Parameters for ATP and TIP3P water molecules were taken from the AMBER force field [Duan et al., 2003 and Jorgensen et al., 1983, respectively]. The LINCS algorithm [Hess et al., 1997] was used to constrain all covalent bonds allowing an integration time step of 2 fs. The systems’ temperature was slowly increased in five steps (50 K, 100 K, 200 K, 250 K, and 298.15 K), each lasting 200 ps. An equilibration phase (5 ns) was obtained without any position restraint within an NPT ensemble at 298.15 K and 1 atm. Each MD simulation lasted for 20.0 ns and average values were calculated for the last 12.0 ns of the MD trajectory during which the system is considered equilibrated.

DNA ConstructsHuman torsinA cDNAs (NM_000113.2), 999 base pairs (bp), encoding torsinAwt and ΔE were generated from controls and DYT1 patients and introduced into EcoRI and NotI restriction sites of pcDNA3.1+ (Invitrogen Life Technologies, Darmstadt, GERMANY), generating pcDNA3-torsinAwt, pcDNA3-torsinAΔE (302/303) [Ozelius et al., 1997]. To generate the pcDNA3-torsinAR288Q construct, the Expand Long Template PCR System (Roche Applied Science, Mannheim, GERMANY) was used to introduce the R288Q mutation into the pcDNA3-torsinAwt construct using the primer, 5‘-GTGGAAATGCAGTCCCAAGGCTATGAAATTGATG-3‘ and 5‘-CATCAATTTCATAGCCTTGGGACTGCATTTCCAC-3‘ with a subsequent DpnI digestion. To generate the pcDNA-torsinAF205I construct, the Quickchange Site-Directed Mutagenesis Kit (Stratagene, La Jolla, CA, USA) was used to introduce the F205I mutation into the pcDNAwt construct using the primers, 5′-CCAGAAAGCCATGTTCATAATTCTCAGCAATGCTGGAGC-3′ and 5′-GCTCCAGCATTGCTGAGAATTATGAACATGGCTTTCTGG-3′, as described previously [Calakos et al., 2010].All cDNAs encoding human torsinA variants were also cloned into the NheI and XhoI sites of the CSCW2-IRES lentivirus vector plasmid [Shu et al., 2006]. CSCW2 is a self-inactivating lentiviral vector, which has a cytomegalovirus (CMV) immediate early promoter controlling transgene expression with the monomeric Cherry (mCherry) fluorescent protein encoded after the internal ribosomal entry site (IRES) element [Sena-Esteves et al., 2004]. All final vectors were sequenced to confirm sequence identities. Lentiviral constructs expressing secreted Gaussia luciferase (Gluc) and firefly luciferase (Fluc), CSCW2-Gluc-IRES-GFP [Badr et al., 2007] and CSCW2-Fluc-IRES-mCherry [Maguire et al., 2008] were prepared, as described [Sena-Esteves et al., 2004].

DNA Constructs

Human torsinA cDNAs (NM_000113.2), 999 base pairs (bp), encoding torsinAwt and ΔE were generated from controls and DYT1 patients and introduced into EcoRI and NotI restriction sites of pcDNA3.1+ (Invitrogen Life Technologies, Darmstadt, GERMANY), generating pcDNA3-torsinAwt, pcDNA3-torsinAΔE (302/303) [Ozelius et al., 1997]. To generate the pcDNA3-torsinAR288Q construct, the Expand Long Template PCR System (Roche Applied Science, Mannheim, GERMANY) was used to introduce the R288Q mutation into the pcDNA3-torsinAwt construct using the primer, 5‘-GTGGAAATGCAGTCCCAAGGCTATGAAATTGATG-3‘ and 5‘-CATCAATTTCATAGCCTTGGGACTGCATTTCCAC-3‘ with a subsequent DpnI digestion. To generate the pcDNA-torsinAF205I construct, the Quickchange Site-Directed Mutagenesis Kit (Stratagene, La Jolla, CA, USA) was used to introduce the F205I mutation into the pcDNAwt construct using the primers, 5′-CCAGAAAGCCATGTTCATAATTCTCAGCAATGCTGGAGC-3′ and 5′-GCTCCAGCATTGCTGAGAATTATGAACATGGCTTTCTGG-3′, as described previously [Calakos et al., 2010].

All cDNAs encoding human torsinA variants were also cloned into the NheI and XhoI sites of the CSCW2-IRES lentivirus vector plasmid [Shu et al., 2006]. CSCW2 is a self-inactivating lentiviral vector, which has a cytomegalovirus (CMV) immediate early promoter controlling transgene expression with the monomeric Cherry (mCherry) fluorescent protein encoded after the internal ribosomal entry site (IRES) element [Sena-Esteves et al., 2004]. All final vectors were sequenced to confirm sequence identities. Lentiviral constructs expressing secreted Gaussia luciferase (Gluc) and firefly luciferase (Fluc), CSCW2-Gluc-IRES-GFP [Badr et al., 2007] and CSCW2-Fluc-IRES-mCherry [Maguire et al., 2008] were prepared, as described [Sena-Esteves et al., 2004].

Statistical AnalysisData were analyzed using two-tailed Student’s t test (Excel, Microsoft, Redmond, WA, USA) or factorial analysis of variance (ANOVA) as applicable using InStat v3.0 (GraphPad Software, La Jolla, CA, USA). After detection of a statistically significant difference in a given series of treatments by ANOVA, post-hoc Dunnett’s t-tests were performed when appropriate. The p-values <0.05 were considered statistically significant (shown as a single asterisk in the figures); p-values <0.01 were considered highly statistically significant (shown as a double asterisk in the figures), and p-values <0.001 (shown as triple asterisk in the figures).

Statistical Analysis

Data were analyzed using two-tailed Student’s t test (Excel, Microsoft, Redmond, WA, USA) or factorial analysis of variance (ANOVA) as applicable using InStat v3.0 (GraphPad Software, La Jolla, CA, USA). After detection of a statistically significant difference in a given series of treatments by ANOVA, post-hoc Dunnett’s t-tests were performed when appropriate. The p-values <0.05 were considered statistically significant (shown as a single asterisk in the figures); p-values <0.01 were considered highly statistically significant (shown as a double asterisk in the figures), and p-values <0.001 (shown as triple asterisk in the figures).

ResultsPredicted Structural Features of Variant Forms of TorsinAThe crystal structure of T. thermophilus ClpB has been described previously [Lee et al., 2003] and was used as a template to generate a structural framework for torsinA [Kock et al., 2006]. Human torsinA presents 332 a.a., which can be divided into four regions. The first 20 a.a. are predicted to be a SS. Residues 21 to 60 are mostly hydrophobic and predicted to be a membrane-associated domain (Fig. 1A). Therefore, no acceptable Protein Data Bank (PDB) templates were found for these regions of the protein. The other two additional domains, the AAA+ domain (L61 to N246) and the C-terminal domain (N247 to D332), were fully modeled in this work. The positions of variant sequences found in torsinA in dystonia patients are indicated in Figure 1A. In the case of the torsinAΔE deletion, it is not clear whether a.a. E302 or E303 is deleted. Other sequence variations in patients with dystonic symptoms include missense variations in exon 3 (F205I; [Calakos et al., 2010]) and exon 5 (R288Q; [Zirn et al., 2008]).The ClpB and torsinA alignment, previously proposed [Kock et al., 2006] was used as a reference for modeling torsinAwt (Fig. 1B), and to analyze the consequences of four sequence variants (delE302, delE303, F205I and R288Q) in the torsinA structure (Fig. 1C). The models for torsinAwt, torsinAF205I, and torsinAR288Q results in a 273 a.a. chain while the torsinAΔE models (delE302 and delE303) have 272 a.a. each. As proposed in Kock et al. [2006], the human torsinA topology is similar to that of the C-terminal region of ClpB proteins. The human torsinA models proposed herein depict fourteen α-helices, α1 (Q62-N68), α2 (H73-V83), α3 (K108-Y120), α4 (F132-A134), α5 (F138-I144), α6 (Q152-L153), α7 (N158-C162), α8 (G178-L187), α9 (A209-L219), α10 (L240-V242), α11 (N246-N249), α12 (K275-Y290), α13 (E303-V313) and α14 (T321-TY329) and five β-strands, β1 (L98-H101), β2 (N127-L129), β3 (I166-N170), β4 (M202-S207) and β5 (P268-L270). With regard to the organization of these structural elements, this protein has a large domain composed of a five strand β-sheet surrounded by eleven α-helices, and a smaller domain made of three α-helices [Zhu et al., 2008]. Figure 1B shows a three-dimensional (3-D) model of human torsinA.It has been shown that proteins having the same or very similar sequences may have significantly different structures (Kosloff and Kolodny, 2008). Our studies show that the overall structures of the AAA+ and C-terminal domains of torsinAwt compared to torsinAΔE (delE302 and delE303), torsinAF205I and torsinAR288Q do not indicate major changes in conformation resulting from the sequence variation (Fig. 1B). All models possess both well-characterized domains. However, we observed the present of some differences in a number of secondary structural elements, notably the helices. All torsinA secondary structures have a different number of α-helices and in the small region linking the three helices of the C-terminal domain. Mutation F205I is located in the AAA+ domain, specifically at strand β4. Based on our homology modeling, this mutation does not result in any major or local structural changes when compared with wt. The other mutations are all located in the C-terminal domain. The R288Q mutation lies within subdomain α5. This subdomain is adjacent to subdomain α6 that is affected by the ΔE deletion. Further, removing E302 may not have a large impact on secondary structure because it is at the border between Helix 15 (Supp. Figure S1). Conversely, when we remove E303, we remove a residue found within Helix 15 (these measurements use torsinAwt as a reference). Insertions and deletions are usually well accommodated when they occur at turns and loops connecting secondary structural elements. The different structures with identical sequences are used as starting conformations for the MD simulations. As observed from our homology modeling we have found some secondary structural changes in the new torsinA variants, and they may play roles in oligomer assembly and/or interaction with torsinA’s partners compromising its function. We further investigated these possibilities by subjecting the models to simulation using the MDs method, as described below.Analyses of TorsinA Interactions (putative dimers)Similar to other AAA+ proteins, torsinA is predicted to assemble into oligomers [Breakefield et al., 2001]. Self-association is a general and conserved feature of members of the AAA+ family and changes in oligomerization have been correlated with changes in function [Mogk et al., 2003]. A 75 kDa torsinA immunoreactive band, in addition to the predicted monomeric 37 kDa torsinA band, has been observed by SDS-PAGE under non-reducing conditions for torsinAΔE [Torres et al., 2004; Gordon and Gonzalez-Alegre, 2008], consistent with the formation of dimer complexes. We asked whether this oligomerization of torsinA is also affected by the other two sequence variants found in dystonia patients, F205I and R288Q. Lysates from BE(2)C cells transduced with lentivirus vectors expressing torsinAwt, torsinAΔE, torsinAR288Q, or torsinAF205I were exposed to non-reducing and reducing conditions, and proteins resolved by SDS gel electrophoresis with immunoblotting for torsinA. While all three torsinA sequence variants resulted in both the 37 kDa form, as well as a stable higher molecular weight (MW) species of 75 kDa under non-reducing conditions, torsinAwt was primarily present as the 37 kDa species with lower tendency to form the 75 kDa species. The higher MW form is predicted to be torsinA dimers and disappears under reducing conditions (Fig. 1D). Although the intensity of the 75 kDa band varied between experiments, it was consistently found with torsinA variants and only rarely and weakly with torsinAwt (Fig. 1E), demonstrating a clear biochemical difference associated with the torsinA variants. These results suggest that dimerization in cells is enhanced by these torsinA variants.Molecular Dynamics Simulations for TorsinAComputational MDs simulations have a growing role in the understanding of the behavior of a variety of macromolecular systems and their interactions [van Gunsteren and Mark, 1998]. These include analysis of molecular complex formation and stability [Monecke et al., 2013; Norberto de Souza and Ornstein, 1999], predictions of enzyme-substrate activity [Chien et al., 2012], effects of mutations on drug resistance [Schroeder et al., 2005], and inhibitor binding [Costa et al., 2012].To relate the putative dimerization results with torsinA expression in neuroblastoma cells, MD simulations (in aqueous solution in the presence of counter ions (Na+ and Cl−) to simulate a physiologic buffer) were performed to monitor the stability of dimers of torsinAwt and torsinA variants (delE302 and delE303, F205I and R288Q), as well as the affinity of these models for ATP. Figure 1F summarizes the results of radius of gyration (RG) and HB pattern with the ATP ligand. RG is a measure of protein structure compactness. The lower the RG the tighter the packing of the protein while higher RG values indicate loosely packed proteins (Flory, 1953). As depicted in Figure 1F, torsinAwt and the delE302 present higher deviations in solution when compared with the variants delE303, R288Q, F205I, and the reference RG value (19.0 ± 0.1 Å) for proteins of the same class and size as torsinA [Lobanov et al., 2008]. Since RG is a measure of the compactness of torsinA dimers, these results suggest that torsinAwt and delE302 may form looser dimers, relative to the other variants. This is in agreement with our SDS-PAGE/western blot analysis, which showed that the torsinA variants have an increased tendency to form dimers (Fig. 1D–E).The HB pattern of the ATP-torsinA interaction along the MD simulation trajectory was also determined. The average values of HB formation for each model are shown in Figure 1F. Modeling revealed that the mutants have fewer HBs coupled to ATP relative to torsinAwt. Kock et al. [2006] also proposed that torsinA variants could be interfering with ATP association. F205I and delE303 models have the lowest number of HBs, compared to other variants and torsinAwt. Overall, the modeling and MD simulation results support and, at least in part, explain some of the experimental data in this work, as well as implicating E303 as the mutation underlying DYT1 dystonia.Furthermore, lentiviral vectors encoding Fluc (for normalization) and secreted Gluc were used to transduce BE(2)-C neuroblastoma cells previously transduced with lentiviral vectors expressing either torsinAwt or torsinA variants, in the configuration CMV-torsinA-IRES-mCherry (Fig. 2A). Three days after transfection, neuroblastoma BE(2)-C cells expressing torsinAwt or torsinA disease variants and mCherry were selected by fluorescence-activated cell sorting (FACS) (Supp. Figure S2, see Supp. Materials and Methods). These sorted populations expressed mCherry at similar levels. The recovered cell lines maintained stable expression over 10 passages in culture.These cells show the typical ER association of torsinA as indicated by co-localization with the ER chaperone protein, PDI, as well as the tendency of torsinAΔE to form ER-derived inclusions [Hewett et al., 2003]. Cell lysates resolved by SDS-PAGE with immunostaining for torsinA and β-actin showed similar levels of expression of torsinA in transduced cells regardless of the variant expressed (Fig. 2B). Non-infected cells have low endogenous expression of torsinA, which was not detected here due to the high dilution of antibody used to monitor overexpression.Using these cells, we assessed Gluc flux through the secretory pathway by measuring luciferase activity in the medium over time normalized to the Fluc activity in cells (Fig. 2C). Neuroblastoma cells expressing torsinA variants showed decreased levels of Gluc activity in the medium relative to those expressing torsinAwt (Fig. 2C), indicating a common impairment of protein secretion in cells expressing these torsinA variants, as previously found for torsinAΔE in other cell types [Hewett et al., 2007; Hewett et al., 2008].In order to determine whether expression of torsinAR288Q and torsinAF205I altered neurite length, we differentiated transduced BE(2)-C human neuroblastoma cells and compared these with non-transduced cells. The cell morphology (Fig. 2D) and length of neurites (Fig. 2E) were analyzed three days post-differentiation. While cells transduced with lentiviral vectors expressing torsinAwt and torsinAR288Q showed similar morphologies, expression of torsinAΔE and F205I resulted in gross abnormalities in both ER and microtubule structure (Fig. 2D). This is consistent with previous reports that torsinAΔE forms inclusions derived from the ER and suggests that torsinAF205I may lead to ER vacuolization. Moreover, no difference in the neurite number was observed between cells expressing torsinA variants and torsinAwt. While no changes in the final neurite length was observed between non-transduced BE(2)-C cells or cells expressing either torsinAwt or torsinAR288Q, corresponding to an increase in neurite length of 40 ± 7 µm in 3 days. BE(2)-C cells expressing torsinAΔE and torsinAF205I showed significantly shorter neurites of 28 ± 6 µm after the same time period (Fig. 2E).Characterization of VSVG Transport Under Conditions of TorsinA Knock-downTo further elucidate a potential role for torsinA in protein processing, we traced the movement of YFP-tagged vesicular stomatitis virus glycoprotein (VSVG)ts045 through the secretory pathway. By regulating environmental temperature, unfolded VSVG can be loaded into the ER and then visualized as it traffics to the Golgi complex, where the protein temporarily accumulates while undergoing N-glycosylation. Trafficking continues onto the plasma membrane (PM), where the protein ultimately localizes. To determine the role of endogenous torsinAwt in this process, an adenovirus vector expressing YFP-VSVGts045 [Keller et al., 2001] was used to transduce 293T cells with and without silencing of torsinA using si1958 or si1963 (Fig. 3A–B; see Supp. Materials and Methods). VSVG trafficking from the ER to the Golgi complex was visualized by confocal microscopy after torsinA depletion relative to scrambled siRNA transduced cells (Fig. 3A; see Supp. Materials and Methods). Quantification of the kinetics of VSVG trafficking from the ER to the Golgi complex over a 10 min period showed a delay in transport to the Golgi when torsinAwt levels were reduced (Fig. 3C). Blinded image analysis was used to quantitate the percent of VSVG in the ER (Fig. 3D) and Golgi complex (Fig. 3E) over 20 min. Down-regulation of torsinA significantly increased the amount of VSVG retained in the ER by 3.2-fold at 10 min and 10.9-fold at 20 min, with a corresponding decrease of about 50% in the Golgi complex at 20 min.In addition, we used an antibody specific to the folded form of VSVG [Lefrancois and Lyles, 1982] to evaluate whether this protein was being folded properly in the ER under conditions of torsinA depletion (Supp. Figure S3). The appearance of folded VSVG protein was evident in 74% of cells treated with scrambled siRNA within 1 min of allowing cells to recover from temperature induced blockade of protein processing (Supp. Figure S3A). Under conditions of torsinA depletion (cells treated with siRNAs 1958 and 1963; Supp. Figure S3B, C, and D), however, VSVG folded protein was largely absent until 5 min post-recovery, at which time only 29% of cells were immuno-positive for folded VSVG (Supp. Figure S3E). This supports the role of torsinA as a chaperone in the processing of this secreted protein.VSVG Transport in Presence of Wild-type and Variant Forms of TorsinAVSVG trafficking was also evaluated in 293T cells stably transduced with lentivirus vectors expressing equivalent amounts of torsinAwt, torsinAΔE, torsinAR288Q, or torsinAF205I. VSVG localization in cells was monitored over 20 min by immunocytochemistry and fluorescence microscopy following a shift to the permissive temperature (Fig. 3F). Image analysis revealed that exit from the ER was almost twice as high in cells expressing torsinAwt, as compared to expression of all the other variants at 10 min (Fig. 3G). This trend continued out to 20 min with torsinAR288Q apparently being the most competent among the torsinA variants in facilitating the trafficking of VSVG cargo (Fig. 3H). The differences in the VSVG trafficking were not related to differential loading of VSGV in the ER, as we observed comparable levels of VSVG in the ER after 16 hr at 40.5°C, the restrictive temperature, in all cells independent of the type of torsinA expressed (Supp. Figure S4).Protein Exit from the ER is Not Affected by the Presence of Wild-type and Variant Forms of TorsinATo test the role of torsinA in the exit of proteins from the ER, 293T cells expressing torsinAwt and variant forms: torsinAΔE, torsinAR288Q and torsinAF205I were transfected with an expression cassette encoding a sec-sfGFP as a soluble cargo [Aronson et al., 2011; Pédelacq et al., 2006]. Sec-sfGFP is known to possess robust folding properties that allow it to exit the ER through the secretory pathway without assistance from ER chaperone proteins. To evaluate the extent of GFP secreted in the medium, we normalized expression to the amount retained in cells. In order to inhibit expression of torsinA, siRNA oligonucleotides were used to knock-down torsinAwt mRNA. siRNAs were transfected into 293T cells and torsinA and actin levels were measured 72 hr later by western blot. siRNA 1963 was able to selectively decrease torsinA levels, but did not change the amount of sec-sfGFP secreted or retained (Fig. 4A). Quantitative densitometry of sec-sfGFP showed no difference in the levels of sec-sfGFP secretion or retention in torsinA-silenced versus siRNA scrambled controls when normalized to α-actin levels (Fig. 4B). In addition, we measured sec-sfGFP secretion in cells transduced with lentivirus vectors encoding torsinAwt and torsinA variants (torsinAΔE, torsinAR288Q, and torsinAF205I) (Fig. 4C). Neither overexpression of torsinAwt or torsinA mutant/variants had any effect on the extent of sec-sfGFP secretion (Fig. 4D), indicating that torsinA is not involved in the regulation of ER exit sites.Morphological Changes Caused by TorsinA Variants in FibroblastsHuman fibroblasts were used to further evaluate ER morphology as their large cytoplasmic volume allows for ease of assessment of ER structure. Fibroblasts from a control individual were transduced with lentiviral vectors expressing torsinAwt or torsinA variants and then immunostained 72 hr later for torsinA and the ER protein, PDI (Fig. 5A). Western blot analysis confirmed similar expression levels of torsinA among lines (data not shown). Consistent with our findings in differentiated BE(2)-C cells, expression of torsinA R288Q in fibroblasts did not result in notable morphological changes in the ER, relative to cells expressing torsinAwt. A notable change was observed, however, in cells expressing either torsinAΔE or torsinAF205I, with torsinAE causing inclusions and the torsinAF205I form causing apparent ER vacuolization (Fig. 5A, lower panel in left). These findings are consistent with those found in BE(2)-C cells, with the apparent vacuolization also seen upon expression of a truncated form of torsinA (313–332) in 293T or fibroblasts [Nery et al., 2011]. The majority of the cells expressing the F205I construct showed evidence of ER vacuolization (Supp. Figure S5).We next examined the ultrastructure of the NE by electron microscopy (EM; see Supp. Materials and Methods). To confirm that the abnormalities in nuclear membrane structure were not a result of torsinA staining, we did not label torsinA in our EM studies. In fibroblasts transduced with the torsinAwt construct, EM analysis showed a uniform spacing between the outer and inner nuclear membranes typical of NE ultrastructure (Fig. 2B). By contrast, focally enlarged perinuclear spaces, and ballooning into large bubbles were found in the NE of cells overexpressing torsinAΔE, torsinAR288Q or torsinAF205I (compare Fig. 5B upper panel with three bottom panels). Expression of torsinAR288Q and torsinAF20I showed sites where the inner nuclear membrane was separated from the nuclear lamina, producing membrane-delimited herniations into the perinuclear space (Fig. 5C–D).

Predicted Structural Features of Variant Forms of TorsinAThe crystal structure of T. thermophilus ClpB has been described previously [Lee et al., 2003] and was used as a template to generate a structural framework for torsinA [Kock et al., 2006]. Human torsinA presents 332 a.a., which can be divided into four regions. The first 20 a.a. are predicted to be a SS. Residues 21 to 60 are mostly hydrophobic and predicted to be a membrane-associated domain (Fig. 1A). Therefore, no acceptable Protein Data Bank (PDB) templates were found for these regions of the protein. The other two additional domains, the AAA+ domain (L61 to N246) and the C-terminal domain (N247 to D332), were fully modeled in this work. The positions of variant sequences found in torsinA in dystonia patients are indicated in Figure 1A. In the case of the torsinAΔE deletion, it is not clear whether a.a. E302 or E303 is deleted. Other sequence variations in patients with dystonic symptoms include missense variations in exon 3 (F205I; [Calakos et al., 2010]) and exon 5 (R288Q; [Zirn et al., 2008]).The ClpB and torsinA alignment, previously proposed [Kock et al., 2006] was used as a reference for modeling torsinAwt (Fig. 1B), and to analyze the consequences of four sequence variants (delE302, delE303, F205I and R288Q) in the torsinA structure (Fig. 1C). The models for torsinAwt, torsinAF205I, and torsinAR288Q results in a 273 a.a. chain while the torsinAΔE models (delE302 and delE303) have 272 a.a. each. As proposed in Kock et al. [2006], the human torsinA topology is similar to that of the C-terminal region of ClpB proteins. The human torsinA models proposed herein depict fourteen α-helices, α1 (Q62-N68), α2 (H73-V83), α3 (K108-Y120), α4 (F132-A134), α5 (F138-I144), α6 (Q152-L153), α7 (N158-C162), α8 (G178-L187), α9 (A209-L219), α10 (L240-V242), α11 (N246-N249), α12 (K275-Y290), α13 (E303-V313) and α14 (T321-TY329) and five β-strands, β1 (L98-H101), β2 (N127-L129), β3 (I166-N170), β4 (M202-S207) and β5 (P268-L270). With regard to the organization of these structural elements, this protein has a large domain composed of a five strand β-sheet surrounded by eleven α-helices, and a smaller domain made of three α-helices [Zhu et al., 2008]. Figure 1B shows a three-dimensional (3-D) model of human torsinA.It has been shown that proteins having the same or very similar sequences may have significantly different structures (Kosloff and Kolodny, 2008). Our studies show that the overall structures of the AAA+ and C-terminal domains of torsinAwt compared to torsinAΔE (delE302 and delE303), torsinAF205I and torsinAR288Q do not indicate major changes in conformation resulting from the sequence variation (Fig. 1B). All models possess both well-characterized domains. However, we observed the present of some differences in a number of secondary structural elements, notably the helices. All torsinA secondary structures have a different number of α-helices and in the small region linking the three helices of the C-terminal domain. Mutation F205I is located in the AAA+ domain, specifically at strand β4. Based on our homology modeling, this mutation does not result in any major or local structural changes when compared with wt. The other mutations are all located in the C-terminal domain. The R288Q mutation lies within subdomain α5. This subdomain is adjacent to subdomain α6 that is affected by the ΔE deletion. Further, removing E302 may not have a large impact on secondary structure because it is at the border between Helix 15 (Supp. Figure S1). Conversely, when we remove E303, we remove a residue found within Helix 15 (these measurements use torsinAwt as a reference). Insertions and deletions are usually well accommodated when they occur at turns and loops connecting secondary structural elements. The different structures with identical sequences are used as starting conformations for the MD simulations. As observed from our homology modeling we have found some secondary structural changes in the new torsinA variants, and they may play roles in oligomer assembly and/or interaction with torsinA’s partners compromising its function. We further investigated these possibilities by subjecting the models to simulation using the MDs method, as described below.

Predicted Structural Features of Variant Forms of TorsinA

The crystal structure of T. thermophilus ClpB has been described previously [Lee et al., 2003] and was used as a template to generate a structural framework for torsinA [Kock et al., 2006]. Human torsinA presents 332 a.a., which can be divided into four regions. The first 20 a.a. are predicted to be a SS. Residues 21 to 60 are mostly hydrophobic and predicted to be a membrane-associated domain (Fig. 1A). Therefore, no acceptable Protein Data Bank (PDB) templates were found for these regions of the protein. The other two additional domains, the AAA+ domain (L61 to N246) and the C-terminal domain (N247 to D332), were fully modeled in this work. The positions of variant sequences found in torsinA in dystonia patients are indicated in Figure 1A. In the case of the torsinAΔE deletion, it is not clear whether a.a. E302 or E303 is deleted. Other sequence variations in patients with dystonic symptoms include missense variations in exon 3 (F205I; [Calakos et al., 2010]) and exon 5 (R288Q; [Zirn et al., 2008]).

The ClpB and torsinA alignment, previously proposed [Kock et al., 2006] was used as a reference for modeling torsinAwt (Fig. 1B), and to analyze the consequences of four sequence variants (delE302, delE303, F205I and R288Q) in the torsinA structure (Fig. 1C). The models for torsinAwt, torsinAF205I, and torsinAR288Q results in a 273 a.a. chain while the torsinAΔE models (delE302 and delE303) have 272 a.a. each. As proposed in Kock et al. [2006], the human torsinA topology is similar to that of the C-terminal region of ClpB proteins. The human torsinA models proposed herein depict fourteen α-helices, α1 (Q62-N68), α2 (H73-V83), α3 (K108-Y120), α4 (F132-A134), α5 (F138-I144), α6 (Q152-L153), α7 (N158-C162), α8 (G178-L187), α9 (A209-L219), α10 (L240-V242), α11 (N246-N249), α12 (K275-Y290), α13 (E303-V313) and α14 (T321-TY329) and five β-strands, β1 (L98-H101), β2 (N127-L129), β3 (I166-N170), β4 (M202-S207) and β5 (P268-L270). With regard to the organization of these structural elements, this protein has a large domain composed of a five strand β-sheet surrounded by eleven α-helices, and a smaller domain made of three α-helices [Zhu et al., 2008]. Figure 1B shows a three-dimensional (3-D) model of human torsinA.

It has been shown that proteins having the same or very similar sequences may have significantly different structures (Kosloff and Kolodny, 2008). Our studies show that the overall structures of the AAA+ and C-terminal domains of torsinAwt compared to torsinAΔE (delE302 and delE303), torsinAF205I and torsinAR288Q do not indicate major changes in conformation resulting from the sequence variation (Fig. 1B). All models possess both well-characterized domains. However, we observed the present of some differences in a number of secondary structural elements, notably the helices. All torsinA secondary structures have a different number of α-helices and in the small region linking the three helices of the C-terminal domain. Mutation F205I is located in the AAA+ domain, specifically at strand β4. Based on our homology modeling, this mutation does not result in any major or local structural changes when compared with wt. The other mutations are all located in the C-terminal domain. The R288Q mutation lies within subdomain α5. This subdomain is adjacent to subdomain α6 that is affected by the ΔE deletion. Further, removing E302 may not have a large impact on secondary structure because it is at the border between Helix 15 (Supp. Figure S1). Conversely, when we remove E303, we remove a residue found within Helix 15 (these measurements use torsinAwt as a reference). Insertions and deletions are usually well accommodated when they occur at turns and loops connecting secondary structural elements. The different structures with identical sequences are used as starting conformations for the MD simulations. As observed from our homology modeling we have found some secondary structural changes in the new torsinA variants, and they may play roles in oligomer assembly and/or interaction with torsinA’s partners compromising its function. We further investigated these possibilities by subjecting the models to simulation using the MDs method, as described below.

Analyses of TorsinA Interactions (putative dimers)Similar to other AAA+ proteins, torsinA is predicted to assemble into oligomers [Breakefield et al., 2001]. Self-association is a general and conserved feature of members of the AAA+ family and changes in oligomerization have been correlated with changes in function [Mogk et al., 2003]. A 75 kDa torsinA immunoreactive band, in addition to the predicted monomeric 37 kDa torsinA band, has been observed by SDS-PAGE under non-reducing conditions for torsinAΔE [Torres et al., 2004; Gordon and Gonzalez-Alegre, 2008], consistent with the formation of dimer complexes. We asked whether this oligomerization of torsinA is also affected by the other two sequence variants found in dystonia patients, F205I and R288Q. Lysates from BE(2)C cells transduced with lentivirus vectors expressing torsinAwt, torsinAΔE, torsinAR288Q, or torsinAF205I were exposed to non-reducing and reducing conditions, and proteins resolved by SDS gel electrophoresis with immunoblotting for torsinA. While all three torsinA sequence variants resulted in both the 37 kDa form, as well as a stable higher molecular weight (MW) species of 75 kDa under non-reducing conditions, torsinAwt was primarily present as the 37 kDa species with lower tendency to form the 75 kDa species. The higher MW form is predicted to be torsinA dimers and disappears under reducing conditions (Fig. 1D). Although the intensity of the 75 kDa band varied between experiments, it was consistently found with torsinA variants and only rarely and weakly with torsinAwt (Fig. 1E), demonstrating a clear biochemical difference associated with the torsinA variants. These results suggest that dimerization in cells is enhanced by these torsinA variants.

Analyses of TorsinA Interactions (putative dimers)

Similar to other AAA+ proteins, torsinA is predicted to assemble into oligomers [Breakefield et al., 2001]. Self-association is a general and conserved feature of members of the AAA+ family and changes in oligomerization have been correlated with changes in function [Mogk et al., 2003]. A 75 kDa torsinA immunoreactive band, in addition to the predicted monomeric 37 kDa torsinA band, has been observed by SDS-PAGE under non-reducing conditions for torsinAΔE [Torres et al., 2004; Gordon and Gonzalez-Alegre, 2008], consistent with the formation of dimer complexes. We asked whether this oligomerization of torsinA is also affected by the other two sequence variants found in dystonia patients, F205I and R288Q. Lysates from BE(2)C cells transduced with lentivirus vectors expressing torsinAwt, torsinAΔE, torsinAR288Q, or torsinAF205I were exposed to non-reducing and reducing conditions, and proteins resolved by SDS gel electrophoresis with immunoblotting for torsinA. While all three torsinA sequence variants resulted in both the 37 kDa form, as well as a stable higher molecular weight (MW) species of 75 kDa under non-reducing conditions, torsinAwt was primarily present as the 37 kDa species with lower tendency to form the 75 kDa species. The higher MW form is predicted to be torsinA dimers and disappears under reducing conditions (Fig. 1D). Although the intensity of the 75 kDa band varied between experiments, it was consistently found with torsinA variants and only rarely and weakly with torsinAwt (Fig. 1E), demonstrating a clear biochemical difference associated with the torsinA variants. These results suggest that dimerization in cells is enhanced by these torsinA variants.

Molecular Dynamics Simulations for TorsinAComputational MDs simulations have a growing role in the understanding of the behavior of a variety of macromolecular systems and their interactions [van Gunsteren and Mark, 1998]. These include analysis of molecular complex formation and stability [Monecke et al., 2013; Norberto de Souza and Ornstein, 1999], predictions of enzyme-substrate activity [Chien et al., 2012], effects of mutations on drug resistance [Schroeder et al., 2005], and inhibitor binding [Costa et al., 2012].To relate the putative dimerization results with torsinA expression in neuroblastoma cells, MD simulations (in aqueous solution in the presence of counter ions (Na+ and Cl−) to simulate a physiologic buffer) were performed to monitor the stability of dimers of torsinAwt and torsinA variants (delE302 and delE303, F205I and R288Q), as well as the affinity of these models for ATP. Figure 1F summarizes the results of radius of gyration (RG) and HB pattern with the ATP ligand. RG is a measure of protein structure compactness. The lower the RG the tighter the packing of the protein while higher RG values indicate loosely packed proteins (Flory, 1953). As depicted in Figure 1F, torsinAwt and the delE302 present higher deviations in solution when compared with the variants delE303, R288Q, F205I, and the reference RG value (19.0 ± 0.1 Å) for proteins of the same class and size as torsinA [Lobanov et al., 2008]. Since RG is a measure of the compactness of torsinA dimers, these results suggest that torsinAwt and delE302 may form looser dimers, relative to the other variants. This is in agreement with our SDS-PAGE/western blot analysis, which showed that the torsinA variants have an increased tendency to form dimers (Fig. 1D–E).The HB pattern of the ATP-torsinA interaction along the MD simulation trajectory was also determined. The average values of HB formation for each model are shown in Figure 1F. Modeling revealed that the mutants have fewer HBs coupled to ATP relative to torsinAwt. Kock et al. [2006] also proposed that torsinA variants could be interfering with ATP association. F205I and delE303 models have the lowest number of HBs, compared to other variants and torsinAwt. Overall, the modeling and MD simulation results support and, at least in part, explain some of the experimental data in this work, as well as implicating E303 as the mutation underlying DYT1 dystonia.Furthermore, lentiviral vectors encoding Fluc (for normalization) and secreted Gluc were used to transduce BE(2)-C neuroblastoma cells previously transduced with lentiviral vectors expressing either torsinAwt or torsinA variants, in the configuration CMV-torsinA-IRES-mCherry (Fig. 2A). Three days after transfection, neuroblastoma BE(2)-C cells expressing torsinAwt or torsinA disease variants and mCherry were selected by fluorescence-activated cell sorting (FACS) (Supp. Figure S2, see Supp. Materials and Methods). These sorted populations expressed mCherry at similar levels. The recovered cell lines maintained stable expression over 10 passages in culture.These cells show the typical ER association of torsinA as indicated by co-localization with the ER chaperone protein, PDI, as well as the tendency of torsinAΔE to form ER-derived inclusions [Hewett et al., 2003]. Cell lysates resolved by SDS-PAGE with immunostaining for torsinA and β-actin showed similar levels of expression of torsinA in transduced cells regardless of the variant expressed (Fig. 2B). Non-infected cells have low endogenous expression of torsinA, which was not detected here due to the high dilution of antibody used to monitor overexpression.Using these cells, we assessed Gluc flux through the secretory pathway by measuring luciferase activity in the medium over time normalized to the Fluc activity in cells (Fig. 2C). Neuroblastoma cells expressing torsinA variants showed decreased levels of Gluc activity in the medium relative to those expressing torsinAwt (Fig. 2C), indicating a common impairment of protein secretion in cells expressing these torsinA variants, as previously found for torsinAΔE in other cell types [Hewett et al., 2007; Hewett et al., 2008].In order to determine whether expression of torsinAR288Q and torsinAF205I altered neurite length, we differentiated transduced BE(2)-C human neuroblastoma cells and compared these with non-transduced cells. The cell morphology (Fig. 2D) and length of neurites (Fig. 2E) were analyzed three days post-differentiation. While cells transduced with lentiviral vectors expressing torsinAwt and torsinAR288Q showed similar morphologies, expression of torsinAΔE and F205I resulted in gross abnormalities in both ER and microtubule structure (Fig. 2D). This is consistent with previous reports that torsinAΔE forms inclusions derived from the ER and suggests that torsinAF205I may lead to ER vacuolization. Moreover, no difference in the neurite number was observed between cells expressing torsinA variants and torsinAwt. While no changes in the final neurite length was observed between non-transduced BE(2)-C cells or cells expressing either torsinAwt or torsinAR288Q, corresponding to an increase in neurite length of 40 ± 7 µm in 3 days. BE(2)-C cells expressing torsinAΔE and torsinAF205I showed significantly shorter neurites of 28 ± 6 µm after the same time period (Fig. 2E).

Molecular Dynamics Simulations for TorsinA

Computational MDs simulations have a growing role in the understanding of the behavior of a variety of macromolecular systems and their interactions [van Gunsteren and Mark, 1998]. These include analysis of molecular complex formation and stability [Monecke et al., 2013; Norberto de Souza and Ornstein, 1999], predictions of enzyme-substrate activity [Chien et al., 2012], effects of mutations on drug resistance [Schroeder et al., 2005], and inhibitor binding [Costa et al., 2012].

To relate the putative dimerization results with torsinA expression in neuroblastoma cells, MD simulations (in aqueous solution in the presence of counter ions (Na+ and Cl−) to simulate a physiologic buffer) were performed to monitor the stability of dimers of torsinAwt and torsinA variants (delE302 and delE303, F205I and R288Q), as well as the affinity of these models for ATP. Figure 1F summarizes the results of radius of gyration (RG) and HB pattern with the ATP ligand. RG is a measure of protein structure compactness. The lower the RG the tighter the packing of the protein while higher RG values indicate loosely packed proteins (Flory, 1953). As depicted in Figure 1F, torsinAwt and the delE302 present higher deviations in solution when compared with the variants delE303, R288Q, F205I, and the reference RG value (19.0 ± 0.1 Å) for proteins of the same class and size as torsinA [Lobanov et al., 2008]. Since RG is a measure of the compactness of torsinA dimers, these results suggest that torsinAwt and delE302 may form looser dimers, relative to the other variants. This is in agreement with our SDS-PAGE/western blot analysis, which showed that the torsinA variants have an increased tendency to form dimers (Fig. 1D–E).

The HB pattern of the ATP-torsinA interaction along the MD simulation trajectory was also determined. The average values of HB formation for each model are shown in Figure 1F. Modeling revealed that the mutants have fewer HBs coupled to ATP relative to torsinAwt. Kock et al. [2006] also proposed that torsinA variants could be interfering with ATP association. F205I and delE303 models have the lowest number of HBs, compared to other variants and torsinAwt. Overall, the modeling and MD simulation results support and, at least in part, explain some of the experimental data in this work, as well as implicating E303 as the mutation underlying DYT1 dystonia.

Furthermore, lentiviral vectors encoding Fluc (for normalization) and secreted Gluc were used to transduce BE(2)-C neuroblastoma cells previously transduced with lentiviral vectors expressing either torsinAwt or torsinA variants, in the configuration CMV-torsinA-IRES-mCherry (Fig. 2A). Three days after transfection, neuroblastoma BE(2)-C cells expressing torsinAwt or torsinA disease variants and mCherry were selected by fluorescence-activated cell sorting (FACS) (Supp. Figure S2, see Supp. Materials and Methods). These sorted populations expressed mCherry at similar levels. The recovered cell lines maintained stable expression over 10 passages in culture.

These cells show the typical ER association of torsinA as indicated by co-localization with the ER chaperone protein, PDI, as well as the tendency of torsinAΔE to form ER-derived inclusions [Hewett et al., 2003]. Cell lysates resolved by SDS-PAGE with immunostaining for torsinA and β-actin showed similar levels of expression of torsinA in transduced cells regardless of the variant expressed (Fig. 2B). Non-infected cells have low endogenous expression of torsinA, which was not detected here due to the high dilution of antibody used to monitor overexpression.

Using these cells, we assessed Gluc flux through the secretory pathway by measuring luciferase activity in the medium over time normalized to the Fluc activity in cells (Fig. 2C). Neuroblastoma cells expressing torsinA variants showed decreased levels of Gluc activity in the medium relative to those expressing torsinAwt (Fig. 2C), indicating a common impairment of protein secretion in cells expressing these torsinA variants, as previously found for torsinAΔE in other cell types [Hewett et al., 2007; Hewett et al., 2008].

In order to determine whether expression of torsinAR288Q and torsinAF205I altered neurite length, we differentiated transduced BE(2)-C human neuroblastoma cells and compared these with non-transduced cells. The cell morphology (Fig. 2D) and length of neurites (Fig. 2E) were analyzed three days post-differentiation. While cells transduced with lentiviral vectors expressing torsinAwt and torsinAR288Q showed similar morphologies, expression of torsinAΔE and F205I resulted in gross abnormalities in both ER and microtubule structure (Fig. 2D). This is consistent with previous reports that torsinAΔE forms inclusions derived from the ER and suggests that torsinAF205I may lead to ER vacuolization. Moreover, no difference in the neurite number was observed between cells expressing torsinA variants and torsinAwt. While no changes in the final neurite length was observed between non-transduced BE(2)-C cells or cells expressing either torsinAwt or torsinAR288Q, corresponding to an increase in neurite length of 40 ± 7 µm in 3 days. BE(2)-C cells expressing torsinAΔE and torsinAF205I showed significantly shorter neurites of 28 ± 6 µm after the same time period (Fig. 2E).

Characterization of VSVG Transport Under Conditions of TorsinA Knock-downTo further elucidate a potential role for torsinA in protein processing, we traced the movement of YFP-tagged vesicular stomatitis virus glycoprotein (VSVG)ts045 through the secretory pathway. By regulating environmental temperature, unfolded VSVG can be loaded into the ER and then visualized as it traffics to the Golgi complex, where the protein temporarily accumulates while undergoing N-glycosylation. Trafficking continues onto the plasma membrane (PM), where the protein ultimately localizes. To determine the role of endogenous torsinAwt in this process, an adenovirus vector expressing YFP-VSVGts045 [Keller et al., 2001] was used to transduce 293T cells with and without silencing of torsinA using si1958 or si1963 (Fig. 3A–B; see Supp. Materials and Methods). VSVG trafficking from the ER to the Golgi complex was visualized by confocal microscopy after torsinA depletion relative to scrambled siRNA transduced cells (Fig. 3A; see Supp. Materials and Methods). Quantification of the kinetics of VSVG trafficking from the ER to the Golgi complex over a 10 min period showed a delay in transport to the Golgi when torsinAwt levels were reduced (Fig. 3C). Blinded image analysis was used to quantitate the percent of VSVG in the ER (Fig. 3D) and Golgi complex (Fig. 3E) over 20 min. Down-regulation of torsinA significantly increased the amount of VSVG retained in the ER by 3.2-fold at 10 min and 10.9-fold at 20 min, with a corresponding decrease of about 50% in the Golgi complex at 20 min.In addition, we used an antibody specific to the folded form of VSVG [Lefrancois and Lyles, 1982] to evaluate whether this protein was being folded properly in the ER under conditions of torsinA depletion (Supp. Figure S3). The appearance of folded VSVG protein was evident in 74% of cells treated with scrambled siRNA within 1 min of allowing cells to recover from temperature induced blockade of protein processing (Supp. Figure S3A). Under conditions of torsinA depletion (cells treated with siRNAs 1958 and 1963; Supp. Figure S3B, C, and D), however, VSVG folded protein was largely absent until 5 min post-recovery, at which time only 29% of cells were immuno-positive for folded VSVG (Supp. Figure S3E). This supports the role of torsinA as a chaperone in the processing of this secreted protein.

Characterization of VSVG Transport Under Conditions of TorsinA Knock-down

To further elucidate a potential role for torsinA in protein processing, we traced the movement of YFP-tagged vesicular stomatitis virus glycoprotein (VSVG)ts045 through the secretory pathway. By regulating environmental temperature, unfolded VSVG can be loaded into the ER and then visualized as it traffics to the Golgi complex, where the protein temporarily accumulates while undergoing N-glycosylation. Trafficking continues onto the plasma membrane (PM), where the protein ultimately localizes. To determine the role of endogenous torsinAwt in this process, an adenovirus vector expressing YFP-VSVGts045 [Keller et al., 2001] was used to transduce 293T cells with and without silencing of torsinA using si1958 or si1963 (Fig. 3A–B; see Supp. Materials and Methods). VSVG trafficking from the ER to the Golgi complex was visualized by confocal microscopy after torsinA depletion relative to scrambled siRNA transduced cells (Fig. 3A; see Supp. Materials and Methods). Quantification of the kinetics of VSVG trafficking from the ER to the Golgi complex over a 10 min period showed a delay in transport to the Golgi when torsinAwt levels were reduced (Fig. 3C). Blinded image analysis was used to quantitate the percent of VSVG in the ER (Fig. 3D) and Golgi complex (Fig. 3E) over 20 min. Down-regulation of torsinA significantly increased the amount of VSVG retained in the ER by 3.2-fold at 10 min and 10.9-fold at 20 min, with a corresponding decrease of about 50% in the Golgi complex at 20 min.

In addition, we used an antibody specific to the folded form of VSVG [Lefrancois and Lyles, 1982] to evaluate whether this protein was being folded properly in the ER under conditions of torsinA depletion (Supp. Figure S3). The appearance of folded VSVG protein was evident in 74% of cells treated with scrambled siRNA within 1 min of allowing cells to recover from temperature induced blockade of protein processing (Supp. Figure S3A). Under conditions of torsinA depletion (cells treated with siRNAs 1958 and 1963; Supp. Figure S3B, C, and D), however, VSVG folded protein was largely absent until 5 min post-recovery, at which time only 29% of cells were immuno-positive for folded VSVG (Supp. Figure S3E). This supports the role of torsinA as a chaperone in the processing of this secreted protein.

VSVG Transport in Presence of Wild-type and Variant Forms of TorsinAVSVG trafficking was also evaluated in 293T cells stably transduced with lentivirus vectors expressing equivalent amounts of torsinAwt, torsinAΔE, torsinAR288Q, or torsinAF205I. VSVG localization in cells was monitored over 20 min by immunocytochemistry and fluorescence microscopy following a shift to the permissive temperature (Fig. 3F). Image analysis revealed that exit from the ER was almost twice as high in cells expressing torsinAwt, as compared to expression of all the other variants at 10 min (Fig. 3G). This trend continued out to 20 min with torsinAR288Q apparently being the most competent among the torsinA variants in facilitating the trafficking of VSVG cargo (Fig. 3H). The differences in the VSVG trafficking were not related to differential loading of VSGV in the ER, as we observed comparable levels of VSVG in the ER after 16 hr at 40.5°C, the restrictive temperature, in all cells independent of the type of torsinA expressed (Supp. Figure S4).

VSVG Transport in Presence of Wild-type and Variant Forms of TorsinA

VSVG trafficking was also evaluated in 293T cells stably transduced with lentivirus vectors expressing equivalent amounts of torsinAwt, torsinAΔE, torsinAR288Q, or torsinAF205I. VSVG localization in cells was monitored over 20 min by immunocytochemistry and fluorescence microscopy following a shift to the permissive temperature (Fig. 3F). Image analysis revealed that exit from the ER was almost twice as high in cells expressing torsinAwt, as compared to expression of all the other variants at 10 min (Fig. 3G). This trend continued out to 20 min with torsinAR288Q apparently being the most competent among the torsinA variants in facilitating the trafficking of VSVG cargo (Fig. 3H). The differences in the VSVG trafficking were not related to differential loading of VSGV in the ER, as we observed comparable levels of VSVG in the ER after 16 hr at 40.5°C, the restrictive temperature, in all cells independent of the type of torsinA expressed (Supp. Figure S4).

Protein Exit from the ER is Not Affected by the Presence of Wild-type and Variant Forms of TorsinATo test the role of torsinA in the exit of proteins from the ER, 293T cells expressing torsinAwt and variant forms: torsinAΔE, torsinAR288Q and torsinAF205I were transfected with an expression cassette encoding a sec-sfGFP as a soluble cargo [Aronson et al., 2011; Pédelacq et al., 2006]. Sec-sfGFP is known to possess robust folding properties that allow it to exit the ER through the secretory pathway without assistance from ER chaperone proteins. To evaluate the extent of GFP secreted in the medium, we normalized expression to the amount retained in cells. In order to inhibit expression of torsinA, siRNA oligonucleotides were used to knock-down torsinAwt mRNA. siRNAs were transfected into 293T cells and torsinA and actin levels were measured 72 hr later by western blot. siRNA 1963 was able to selectively decrease torsinA levels, but did not change the amount of sec-sfGFP secreted or retained (Fig. 4A). Quantitative densitometry of sec-sfGFP showed no difference in the levels of sec-sfGFP secretion or retention in torsinA-silenced versus siRNA scrambled controls when normalized to α-actin levels (Fig. 4B). In addition, we measured sec-sfGFP secretion in cells transduced with lentivirus vectors encoding torsinAwt and torsinA variants (torsinAΔE, torsinAR288Q, and torsinAF205I) (Fig. 4C). Neither overexpression of torsinAwt or torsinA mutant/variants had any effect on the extent of sec-sfGFP secretion (Fig. 4D), indicating that torsinA is not involved in the regulation of ER exit sites.

Protein Exit from the ER is Not Affected by the Presence of Wild-type and Variant Forms of TorsinA

To test the role of torsinA in the exit of proteins from the ER, 293T cells expressing torsinAwt and variant forms: torsinAΔE, torsinAR288Q and torsinAF205I were transfected with an expression cassette encoding a sec-sfGFP as a soluble cargo [Aronson et al., 2011; Pédelacq et al., 2006]. Sec-sfGFP is known to possess robust folding properties that allow it to exit the ER through the secretory pathway without assistance from ER chaperone proteins. To evaluate the extent of GFP secreted in the medium, we normalized expression to the amount retained in cells. In order to inhibit expression of torsinA, siRNA oligonucleotides were used to knock-down torsinAwt mRNA. siRNAs were transfected into 293T cells and torsinA and actin levels were measured 72 hr later by western blot. siRNA 1963 was able to selectively decrease torsinA levels, but did not change the amount of sec-sfGFP secreted or retained (Fig. 4A). Quantitative densitometry of sec-sfGFP showed no difference in the levels of sec-sfGFP secretion or retention in torsinA-silenced versus siRNA scrambled controls when normalized to α-actin levels (Fig. 4B). In addition, we measured sec-sfGFP secretion in cells transduced with lentivirus vectors encoding torsinAwt and torsinA variants (torsinAΔE, torsinAR288Q, and torsinAF205I) (Fig. 4C). Neither overexpression of torsinAwt or torsinA mutant/variants had any effect on the extent of sec-sfGFP secretion (Fig. 4D), indicating that torsinA is not involved in the regulation of ER exit sites.

Morphological Changes Caused by TorsinA Variants in FibroblastsHuman fibroblasts were used to further evaluate ER morphology as their large cytoplasmic volume allows for ease of assessment of ER structure. Fibroblasts from a control individual were transduced with lentiviral vectors expressing torsinAwt or torsinA variants and then immunostained 72 hr later for torsinA and the ER protein, PDI (Fig. 5A). Western blot analysis confirmed similar expression levels of torsinA among lines (data not shown). Consistent with our findings in differentiated BE(2)-C cells, expression of torsinA R288Q in fibroblasts did not result in notable morphological changes in the ER, relative to cells expressing torsinAwt. A notable change was observed, however, in cells expressing either torsinAΔE or torsinAF205I, with torsinAE causing inclusions and the torsinAF205I form causing apparent ER vacuolization (Fig. 5A, lower panel in left). These findings are consistent with those found in BE(2)-C cells, with the apparent vacuolization also seen upon expression of a truncated form of torsinA (313–332) in 293T or fibroblasts [Nery et al., 2011]. The majority of the cells expressing the F205I construct showed evidence of ER vacuolization (Supp. Figure S5).We next examined the ultrastructure of the NE by electron microscopy (EM; see Supp. Materials and Methods). To confirm that the abnormalities in nuclear membrane structure were not a result of torsinA staining, we did not label torsinA in our EM studies. In fibroblasts transduced with the torsinAwt construct, EM analysis showed a uniform spacing between the outer and inner nuclear membranes typical of NE ultrastructure (Fig. 2B). By contrast, focally enlarged perinuclear spaces, and ballooning into large bubbles were found in the NE of cells overexpressing torsinAΔE, torsinAR288Q or torsinAF205I (compare Fig. 5B upper panel with three bottom panels). Expression of torsinAR288Q and torsinAF20I showed sites where the inner nuclear membrane was separated from the nuclear lamina, producing membrane-delimited herniations into the perinuclear space (Fig. 5C–D).

Morphological Changes Caused by TorsinA Variants in Fibroblasts

Human fibroblasts were used to further evaluate ER morphology as their large cytoplasmic volume allows for ease of assessment of ER structure. Fibroblasts from a control individual were transduced with lentiviral vectors expressing torsinAwt or torsinA variants and then immunostained 72 hr later for torsinA and the ER protein, PDI (Fig. 5A). Western blot analysis confirmed similar expression levels of torsinA among lines (data not shown). Consistent with our findings in differentiated BE(2)-C cells, expression of torsinA R288Q in fibroblasts did not result in notable morphological changes in the ER, relative to cells expressing torsinAwt. A notable change was observed, however, in cells expressing either torsinAΔE or torsinAF205I, with torsinAE causing inclusions and the torsinAF205I form causing apparent ER vacuolization (Fig. 5A, lower panel in left). These findings are consistent with those found in BE(2)-C cells, with the apparent vacuolization also seen upon expression of a truncated form of torsinA (313–332) in 293T or fibroblasts [Nery et al., 2011]. The majority of the cells expressing the F205I construct showed evidence of ER vacuolization (Supp. Figure S5).

We next examined the ultrastructure of the NE by electron microscopy (EM; see Supp. Materials and Methods). To confirm that the abnormalities in nuclear membrane structure were not a result of torsinA staining, we did not label torsinA in our EM studies. In fibroblasts transduced with the torsinAwt construct, EM analysis showed a uniform spacing between the outer and inner nuclear membranes typical of NE ultrastructure (Fig. 2B). By contrast, focally enlarged perinuclear spaces, and ballooning into large bubbles were found in the NE of cells overexpressing torsinAΔE, torsinAR288Q or torsinAF205I (compare Fig. 5B upper panel with three bottom panels). Expression of torsinAR288Q and torsinAF20I showed sites where the inner nuclear membrane was separated from the nuclear lamina, producing membrane-delimited herniations into the perinuclear space (Fig. 5C–D).

DiscussionThese studies employ a number of assays which have previously demonstrated marked differences between torsinAwt and torsinAΔE when overexpressed in cultured cells, including a tendency to dimerize in the absence of reducing agents, reduced processing of Gluc and VSVG through the secretory pathway, decreased neurite extension and alterations in ER/NE morphology. These assays were used to evaluate the consequences of defects in torsinA function in two missense variant forms of torsinA, torsinAR288Q and torsinAF205I, found in patients with dystonia. As with torsinAΔE, both these torsinA variants showed an increased tendency to dimerize, reduced processing of Gluc and VSVG, and altered NE morphology. The R288Q mutation appeared to have less impact on normal torsinA function relative to the F205I mutation, as only the latter showed compromised neurite extension and grossly altered ER morphology. Since the R288Q mutation is associated with severe generalized early onset dystonia [Zirn et al., 2008], and the F205I mutation with late onset, focal dystonia [Calakos et al., 2010], this suggests that torsinA properties of dimerization, protein processing and NE morphology may contribute to the pathophysiology of dystonia. Collectively, these data provides a means of assessing the impact of coding sequence variations in torsinA on cellular pathogenicity, as well as providing a series of tests to measure potential efficacy of novel therapeutic agents.AAA+ proteins, such as torsinA, typically form six-membered homo-oligomeric rings that control both assembly and disassembly of numerous protein complexes involved in protein processing, membrane trafficking, cytoskeletal dynamics and organelle biogenesis [Vale, 2000; Ogura and Wilkinson, 2001; Hanson and Whiteheart, 2005]. The closest orthologue of torsinA with known 3D structure is the second AAA+ domain of the bacterial heat-shock protein ClpB, derived from T. thermophilus [Lee et al., 2003; Iyer et al., 2004]. To assess the effects of missense mutations or deletions within the AAA+ or C-terminal domain of torsinA on its function, we built structural homology models for torsinAwt, F205I, R288Q, delE302, and delE303 (Fig. 1B and 1C) using the 3D structure of ClpB as a template. Given the close topological relationship between torsinA and the ClpB template [Kock et al., 2006], we did not expect much structural variation from these static models. We therefore performed MD simulations for the dimers of all models generated in an explicit aqueous solution. In MD simulations macromolecular systems may relax over the period of simulation time, allowing them to accommodate to intrinsic and environmental alterations. The changes we observed in RG during MD simulation suggests that the dimer models of torsinAwt and delE302 are less compact relative to those for F205I, R288Q, and delE303. The looser dimers observed in models of torsinAwt and delE302 may be due to a weaker association between their respective monomers, while the higher compactness predicted in models for F205I, R288Q, and delE303 may by contrast indicate stronger associations. This hypothesis is supported by western blotting experiments that indicated a greater tendency of torsinA variants to form dimers under non-reducing conditions. Interestingly, HB analysis of torsinA-ATP interaction models showed that torsinAwt has a higher affinity for ATP than torsinA disease variants. While R288Q is located in the C-terminal domain, F205I is found in strand β4 closer to the ATP binding site in the AAA+ domain (Fig. 1C). The proximity of F205I seems to result in long range structural changes that may affect its ability to bind ATP. The differences in HB pattern (Fig. 1F) and the relative positions of R288Q and F205I mutations may explain why F205I has a more prominent impact on torsinA function in biological assays. Deletion of E303 predicts a severe decrease in torsinA binding to ATP (6 HBs) when compared to either wt (13 HBs) or deletion of E302 (10 HBs). These results are consistent with those of Kock et al. [2006] and Zhao et al. [2013] in predicting which torsinA variants interfere with ATP binding affinity. In addition, the B-factor behavior showed that deletion E303 is more structurally deleterious than E302. We can rationalize this finding based on the changes in helix α6 of the C-terminal domain. According to the torsinA/ClpB pairwise sequence alignment (Fig. 1B of 8) residues E302 and E303 are the first two residues of helix α6. Residue E302 is at the boundary with the loop connecting helix α5 to α6. It is highly flexible and its conformation may transition between coil and helical conformations. Residue E303 is located towards the interior of helix α6, and therefore its deletion may cause local structural disorders that can propagate towards both the N- and C-terminus of the helix. Hence, deletion E303 is more disturbing to the structure. The spatial long range effects of these changes results in poor packing of the C-terminal three-helix bundle domain. Although this structural disorder enhances the probability of delE303 dimerization, as observed through the RG parameter behavior, it is not compensated by the very low affinity for ATP.Previous studies have suggested that differences in oligomerization of torsinA can influence its rate of degradation [Torres et al., 2004; Gordon and Gonzalez-Alegre, 2008]. Self-association is a general and conserved feature of members of the AAA+ family of ATPases and can influence AAA+ protein function [Mogk et al., 2003; Oguchi et al., 2012]. Thus, we examined whether torsinA self-associates and whether this property was altered in any of the torsinA variants. All mutants showed an increased tendency to dimerize in cell lysates under non-reducing conditions with torsinAΔE>R228Q>F205I>wt. This was consistent with structural predictions and may indicate a more stable structure of mutant oligomers relative to wt oligomers. This may suggest a dominant-negative effect of heterozygotic variants in patient cells with mutant forms presumably binding to the wt forms and thereby reducing their activity.Our previous studies demonstrated that trafficking of proteins through the secretory pathway can be monitored using a naturally secreted, and highly sensitive luciferase, Gluc [Tannous et al., 2005] and that DYT1 patient cells have a marked decrease in the rate of Gluc secretion as compared with control cells [Hewett et al., 2007]. This appeared to be due to reduced function of torsinA as an ER chaperone protein, as mouse embryonic fibroblasts (MEFs) from homozygous torsinA knock-out mice also showed reduced Gluc secretion, relative to wt MEFs [Hewett et al., 2007]. In addition, knock-down experiments using siRNAs that reduced levels of wt torsinA decreased Gluc secretion, while reduced levels of torsinAΔE increased secretion in DYT1 fibroblasts [Hewett et al., 2008]. Here, we expanded these findings by monitoring trafficking of YFP-VSVGts045 from the ER to the Golgi complex. Decreased levels of torsinA following siRNA knock-down of torsinA markedly reduced this trafficking (by 80%), again implicating torsinA in protein processing in the ER. Since sec-sfGFP exit from the ER was not impaired, delayed trafficking of YFP-VSVGts045 from the ER to the Golgi complex is likely due to impaired protein processing in the ER. This is consistent with our finding that torsinA depletion impairs normal folding of YFP-VSVGts045 in the ER and with studies showing that impaired processing of proteins in cells expressing endogenous or elevated levels of torsinAΔE show ER stress, presumably due to protein overload [Hewett et al., 2003; Nery et al., 2011; Chen et al., 2010a].Further, in previous studies using human neuroblastoma SH-SY5Y cells we showed the overexpression of torsinAΔE delays the kinetics of neurite outgrowth relative to torsinAwt [Hewett et al., 2006]. In the present study we find that overexpression of the torsinA variant F205I, similarly compromises neurite extension in human neuroblastoma cells. While F205I results are consistent with those of torsinAΔE, a more detailed morphological analysis is needed to ascertain the basis of these defects in neurite growth. Abnormalities in brain structure have been reported in mouse models with altered torsinA expression, including delayed interkinetic nuclear migration during neurogenesis and impaired migration of neurons in torsinA homozygous knock-out embryos [McCarthy et al., 2012] and changes in size and dendrite morphology of neurons in heterozygous torsinA knock-in mice [Song et al., 2013]. Reduction in neuronal fiber tracts have also been observed in the brains of heterozygous DYT1 knock-in mice [Uluğ et al., 2011].Overexpression of all three torsinA mutant variants described herein resulted in abnormal spacing between the inner and outer nuclear membranes, including presence of intraluminal vesicles. These findings are consistent with those reported following torsinAΔE overexpression in a variety of cell types, as well as absence torsinA in mouse homozygous knock-out embryos [Gonzalez-Alegre and Paulson, 2004; Naismith et al., 2004; Goodchild et al., 2005]. This is also consistent with the defined role of torsinA in modulating nuclear membrane dynamics. This function is dependent on torsinA's interaction with nesprins, which span the outer nuclear membrane and anchor the NE to the cytoskeleton [Ketema and Sonnenberg, 2011]. TorsinA also appears to mediate association between SUN proteins in the inner NE and nesprins in the outer NE, as a component of the LINC complex [Vander Heyden et al., 2011; Jungwirth et al., 2011]. The consequences of mutations in or down-regulation of torsinA in the NE include restricted movement of the nucleus, manifesting as a delay in cell migration [Hewett et al., 2008], and the nuclear export of both large ribonuclear particles [Jokhi et al., 2013] and HSV-1 virions [Maric et al., 2011].Mutation or depletion of torsinA impacts a variety of cellular processes, including nuclear polarization, cell migration, export of ribonuclear particles from the nucleus, protein folding, neurite extension and the maintenance of NE and ER morphology. The ability of torsinA to simultaneously impact so many discrete cellular compartments may be due to association with multiple proteins that span the ER and NE membranes and their interaction with cytoskeletal elements. Our results showing that torsinA variants form more stable dimers may suggest differences in their ability association numerous proteins, as compared to torsinAwt, thus impacting on a multitude of cellular functions. It may thus seem surprising that so few mutations have been identified in torsinA that result in dystonia. Mutations in torsinA that result in a null allele in combination with a wt allele are fully compatible with normal motor function, while complete loss of expression, as seen in homozygous knock-out mice, results in neonatal death [Goodchild et al., 2005; Dang et al., 2005]. The mutations causing dystonia in the heterozygous state must therefore be unique and exert a dominant-negative effect that blocks normal torsinA activity. Structural analysis of the disease-causing torsinA mutants reported in this paper appears to compromise the region of the protein predicted to bind to ATP. This region also appears to affect dynamic aspects of oligomerization with the mutants forming more stable complexes with each other and possibly torsinAwt. Alternatively, they may impede binding to other proteins, such as LULL1 associated with ATPase activity [Zhao et al., 2013]. Collectively, the assays used in the present study provide a means both to evaluate the role of specific torsinA mutants in the disease process, as well as to validate efficacy of therapeutics in correcting or overriding the dominant-negative effects of mutant torsinA. These assays implicate both torsinAR228Q and torsinAF205I as additional mutations causing autosomal dominant dystonia in humans.

These studies employ a number of assays which have previously demonstrated marked differences between torsinAwt and torsinAΔE when overexpressed in cultured cells, including a tendency to dimerize in the absence of reducing agents, reduced processing of Gluc and VSVG through the secretory pathway, decreased neurite extension and alterations in ER/NE morphology. These assays were used to evaluate the consequences of defects in torsinA function in two missense variant forms of torsinA, torsinAR288Q and torsinAF205I, found in patients with dystonia. As with torsinAΔE, both these torsinA variants showed an increased tendency to dimerize, reduced processing of Gluc and VSVG, and altered NE morphology. The R288Q mutation appeared to have less impact on normal torsinA function relative to the F205I mutation, as only the latter showed compromised neurite extension and grossly altered ER morphology. Since the R288Q mutation is associated with severe generalized early onset dystonia [Zirn et al., 2008], and the F205I mutation with late onset, focal dystonia [Calakos et al., 2010], this suggests that torsinA properties of dimerization, protein processing and NE morphology may contribute to the pathophysiology of dystonia. Collectively, these data provides a means of assessing the impact of coding sequence variations in torsinA on cellular pathogenicity, as well as providing a series of tests to measure potential efficacy of novel therapeutic agents.

AAA+ proteins, such as torsinA, typically form six-membered homo-oligomeric rings that control both assembly and disassembly of numerous protein complexes involved in protein processing, membrane trafficking, cytoskeletal dynamics and organelle biogenesis [Vale, 2000; Ogura and Wilkinson, 2001; Hanson and Whiteheart, 2005]. The closest orthologue of torsinA with known 3D structure is the second AAA+ domain of the bacterial heat-shock protein ClpB, derived from T. thermophilus [Lee et al., 2003; Iyer et al., 2004]. To assess the effects of missense mutations or deletions within the AAA+ or C-terminal domain of torsinA on its function, we built structural homology models for torsinAwt, F205I, R288Q, delE302, and delE303 (Fig. 1B and 1C) using the 3D structure of ClpB as a template. Given the close topological relationship between torsinA and the ClpB template [Kock et al., 2006], we did not expect much structural variation from these static models. We therefore performed MD simulations for the dimers of all models generated in an explicit aqueous solution. In MD simulations macromolecular systems may relax over the period of simulation time, allowing them to accommodate to intrinsic and environmental alterations. The changes we observed in RG during MD simulation suggests that the dimer models of torsinAwt and delE302 are less compact relative to those for F205I, R288Q, and delE303. The looser dimers observed in models of torsinAwt and delE302 may be due to a weaker association between their respective monomers, while the higher compactness predicted in models for F205I, R288Q, and delE303 may by contrast indicate stronger associations. This hypothesis is supported by western blotting experiments that indicated a greater tendency of torsinA variants to form dimers under non-reducing conditions. Interestingly, HB analysis of torsinA-ATP interaction models showed that torsinAwt has a higher affinity for ATP than torsinA disease variants. While R288Q is located in the C-terminal domain, F205I is found in strand β4 closer to the ATP binding site in the AAA+ domain (Fig. 1C). The proximity of F205I seems to result in long range structural changes that may affect its ability to bind ATP. The differences in HB pattern (Fig. 1F) and the relative positions of R288Q and F205I mutations may explain why F205I has a more prominent impact on torsinA function in biological assays. Deletion of E303 predicts a severe decrease in torsinA binding to ATP (6 HBs) when compared to either wt (13 HBs) or deletion of E302 (10 HBs). These results are consistent with those of Kock et al. [2006] and Zhao et al. [2013] in predicting which torsinA variants interfere with ATP binding affinity. In addition, the B-factor behavior showed that deletion E303 is more structurally deleterious than E302. We can rationalize this finding based on the changes in helix α6 of the C-terminal domain. According to the torsinA/ClpB pairwise sequence alignment (Fig. 1B of 8) residues E302 and E303 are the first two residues of helix α6. Residue E302 is at the boundary with the loop connecting helix α5 to α6. It is highly flexible and its conformation may transition between coil and helical conformations. Residue E303 is located towards the interior of helix α6, and therefore its deletion may cause local structural disorders that can propagate towards both the N- and C-terminus of the helix. Hence, deletion E303 is more disturbing to the structure. The spatial long range effects of these changes results in poor packing of the C-terminal three-helix bundle domain. Although this structural disorder enhances the probability of delE303 dimerization, as observed through the RG parameter behavior, it is not compensated by the very low affinity for ATP.

Previous studies have suggested that differences in oligomerization of torsinA can influence its rate of degradation [Torres et al., 2004; Gordon and Gonzalez-Alegre, 2008]. Self-association is a general and conserved feature of members of the AAA+ family of ATPases and can influence AAA+ protein function [Mogk et al., 2003; Oguchi et al., 2012]. Thus, we examined whether torsinA self-associates and whether this property was altered in any of the torsinA variants. All mutants showed an increased tendency to dimerize in cell lysates under non-reducing conditions with torsinAΔE>R228Q>F205I>wt. This was consistent with structural predictions and may indicate a more stable structure of mutant oligomers relative to wt oligomers. This may suggest a dominant-negative effect of heterozygotic variants in patient cells with mutant forms presumably binding to the wt forms and thereby reducing their activity.

Our previous studies demonstrated that trafficking of proteins through the secretory pathway can be monitored using a naturally secreted, and highly sensitive luciferase, Gluc [Tannous et al., 2005] and that DYT1 patient cells have a marked decrease in the rate of Gluc secretion as compared with control cells [Hewett et al., 2007]. This appeared to be due to reduced function of torsinA as an ER chaperone protein, as mouse embryonic fibroblasts (MEFs) from homozygous torsinA knock-out mice also showed reduced Gluc secretion, relative to wt MEFs [Hewett et al., 2007]. In addition, knock-down experiments using siRNAs that reduced levels of wt torsinA decreased Gluc secretion, while reduced levels of torsinAΔE increased secretion in DYT1 fibroblasts [Hewett et al., 2008]. Here, we expanded these findings by monitoring trafficking of YFP-VSVGts045 from the ER to the Golgi complex. Decreased levels of torsinA following siRNA knock-down of torsinA markedly reduced this trafficking (by 80%), again implicating torsinA in protein processing in the ER. Since sec-sfGFP exit from the ER was not impaired, delayed trafficking of YFP-VSVGts045 from the ER to the Golgi complex is likely due to impaired protein processing in the ER. This is consistent with our finding that torsinA depletion impairs normal folding of YFP-VSVGts045 in the ER and with studies showing that impaired processing of proteins in cells expressing endogenous or elevated levels of torsinAΔE show ER stress, presumably due to protein overload [Hewett et al., 2003; Nery et al., 2011; Chen et al., 2010a].

Further, in previous studies using human neuroblastoma SH-SY5Y cells we showed the overexpression of torsinAΔE delays the kinetics of neurite outgrowth relative to torsinAwt [Hewett et al., 2006]. In the present study we find that overexpression of the torsinA variant F205I, similarly compromises neurite extension in human neuroblastoma cells. While F205I results are consistent with those of torsinAΔE, a more detailed morphological analysis is needed to ascertain the basis of these defects in neurite growth. Abnormalities in brain structure have been reported in mouse models with altered torsinA expression, including delayed interkinetic nuclear migration during neurogenesis and impaired migration of neurons in torsinA homozygous knock-out embryos [McCarthy et al., 2012] and changes in size and dendrite morphology of neurons in heterozygous torsinA knock-in mice [Song et al., 2013]. Reduction in neuronal fiber tracts have also been observed in the brains of heterozygous DYT1 knock-in mice [Uluğ et al., 2011].

Overexpression of all three torsinA mutant variants described herein resulted in abnormal spacing between the inner and outer nuclear membranes, including presence of intraluminal vesicles. These findings are consistent with those reported following torsinAΔE overexpression in a variety of cell types, as well as absence torsinA in mouse homozygous knock-out embryos [Gonzalez-Alegre and Paulson, 2004; Naismith et al., 2004; Goodchild et al., 2005]. This is also consistent with the defined role of torsinA in modulating nuclear membrane dynamics. This function is dependent on torsinA's interaction with nesprins, which span the outer nuclear membrane and anchor the NE to the cytoskeleton [Ketema and Sonnenberg, 2011]. TorsinA also appears to mediate association between SUN proteins in the inner NE and nesprins in the outer NE, as a component of the LINC complex [Vander Heyden et al., 2011; Jungwirth et al., 2011]. The consequences of mutations in or down-regulation of torsinA in the NE include restricted movement of the nucleus, manifesting as a delay in cell migration [Hewett et al., 2008], and the nuclear export of both large ribonuclear particles [Jokhi et al., 2013] and HSV-1 virions [Maric et al., 2011].

Mutation or depletion of torsinA impacts a variety of cellular processes, including nuclear polarization, cell migration, export of ribonuclear particles from the nucleus, protein folding, neurite extension and the maintenance of NE and ER morphology. The ability of torsinA to simultaneously impact so many discrete cellular compartments may be due to association with multiple proteins that span the ER and NE membranes and their interaction with cytoskeletal elements. Our results showing that torsinA variants form more stable dimers may suggest differences in their ability association numerous proteins, as compared to torsinAwt, thus impacting on a multitude of cellular functions. It may thus seem surprising that so few mutations have been identified in torsinA that result in dystonia. Mutations in torsinA that result in a null allele in combination with a wt allele are fully compatible with normal motor function, while complete loss of expression, as seen in homozygous knock-out mice, results in neonatal death [Goodchild et al., 2005; Dang et al., 2005]. The mutations causing dystonia in the heterozygous state must therefore be unique and exert a dominant-negative effect that blocks normal torsinA activity. Structural analysis of the disease-causing torsinA mutants reported in this paper appears to compromise the region of the protein predicted to bind to ATP. This region also appears to affect dynamic aspects of oligomerization with the mutants forming more stable complexes with each other and possibly torsinAwt. Alternatively, they may impede binding to other proteins, such as LULL1 associated with ATPase activity [Zhao et al., 2013]. Collectively, the assays used in the present study provide a means both to evaluate the role of specific torsinA mutants in the disease process, as well as to validate efficacy of therapeutics in correcting or overriding the dominant-negative effects of mutant torsinA. These assays implicate both torsinAR228Q and torsinAF205I as additional mutations causing autosomal dominant dystonia in humans.

Supplementary MaterialSupp Material

Supplementary Material
