---
pmid: '18599299'
title: Structure of IL-22 bound to its high-affinity IL-22R1 chain.
authors:
- Jones BC
- Logsdon NJ
- Walter MR
journal: Structure
year: '2008'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC2637415
doi: 10.1016/j.str.2008.06.005
---

# Structure of IL-22 bound to its high-affinity IL-22R1 chain.
**Authors:** Jones BC, Logsdon NJ, Walter MR
**Journal:** Structure (2008)
**DOI:** [10.1016/j.str.2008.06.005](https://doi.org/10.1016/j.str.2008.06.005)
**PMC:** [PMC2637415](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2637415/)

## Abstract

1. Structure. 2008 Sep 10;16(9):1333-44. doi: 10.1016/j.str.2008.06.005. Epub
2008  Jul 3.

Structure of IL-22 bound to its high-affinity IL-22R1 chain.

Jones BC(1), Logsdon NJ, Walter MR.

Author information:
(1)Center for Biophysical Sciences and Engineering, University of Alabama at 
Birmingham, Birmingham, AL 35294, USA.

IL-22 is an IL-10 family cytokine that initiates innate immune responses against 
bacterial pathogens and contributes to immune disease. IL-22 biological activity 
is initiated by binding to a cell-surface complex composed of IL-22R1 and 
IL-10R2 receptor chains and further regulated by interactions with a soluble 
binding protein, IL-22BP, which shares sequence similarity with an extracellular 
region of IL-22R1 (sIL-22R1). IL-22R1 also pairs with the IL-20R2 chain to 
induce IL-20 and IL-24 signaling. To define the molecular basis of these diverse 
interactions, we have determined the structure of the IL-22/sIL-22R1 complex. 
The structure, combined with homology modeling and surface plasmon resonance 
studies, defines the molecular basis for the distinct affinities and 
specificities of IL-22 and IL-10 receptor chains that regulate cellular 
targeting and signal transduction to elicit effective immune responses.

DOI: 10.1016/j.str.2008.06.005
PMCID: PMC2637415
PMID: 18599299 [Indexed for MEDLINE]

## Full Text

Introduction

Interleukin-22 (IL-22) is an α-helical cytokine produced by activated DC and T cells, including the recently identified Th17 lineage ( Dumoutier et al., 2000a ; Liang et al., 2006 ). Despite its production by immune cells, IL-22 biological activity is specifically targeted to cells of epithelial origins where it induces pro-inflammatory activities that include the production of early systemically circulated defense proteins (acute phase proteins) ( Dumoutier et al., 2000b ). More recently, IL-22 has been shown to induce anti-microbial proteins, including S100s and defensins, in colonic and lung epithelial cells, which are critical for host defense against enteric and pulmonary bacterial pathogens ( Aujla et al., 2008 ; Liang et al., 2006 ; Wolk et al., 2004 ; Zheng et al., 2008 ). This suggests IL22 allows coordination between the adaptive and innate immune systems to fight invading pathogens.

In addition to its pro-inflammatory activities, IL-22 confers a protective effect on the liver by promoting hepatocyte survival ( Radaeva et al., 2004 ; Zenewicz et al., 2007 ). IL-22 also enhances lung epithelial cell proliferation and the maintenance of trans-epithelial resistance to injury ( Aujla et al., 2008 ). IL-22-mediated protective effects have also been observed in the gut, where IL-22 knockout mice, subjected to C. rodentium infection, exhibit increased epithelial cell damage and increased mortality ( Zheng et al., 2008 ). Thus, IL-22 exhibits paradoxical pro- and anti- inflammatory properties on epithelial cells, which are strikingly similar to the activity profile of IL-10 on immune cells ( Moore et al., 2001 ).

IL-22 and IL-10 induce cellular responses by engaging heterodimeric cell surface receptor complexes that contain IL-22R1 and IL-10R1 chains, respectively, and the shared signal transducing receptor, IL-10R2 ( Figure 1 , ( Kotenko et al., 2001a ; Xie et al., 2000 ). Formation of the IL-22/IL-22R1/IL-10R2 complex activates intracellular kinases (JAK1, Tyk2, and MAP kinases) and transcription factors, especially STAT3, leading to IL-22 cellular responses ( Lejeune et al. , 2002 ). IL-10/IL-10R1/IL-10R2 assembly induces overlapping kinases and also predominantly activates STAT3 ( Moore et al., 2001 ).

IL-22R1 is selectively expressed on skin and epithelial cells, while IL-10R1 is predominantly expressed on DC, monocytes, and macrophages ( Nagalakshmi et al., 2004 ). In contrast, IL-10R2 is expressed on all cell types, which reflects its role as a shared receptor in IL-22, IL-10, IL-26, IL-28, and IL-29 receptor complexes. Surface plasmon resonance (SPR) studies are consistent with the targeting function of IL-22R1 and IL-10R1 ( Logsdon et al., 2004 ; Logsdon et al., 2002 ; Yoon et al., 2005 ; Yoon et al., 2006 ). sIL-22R1 and sIL-10R1 both exhibit high affinity (~nM) for their respective ligands. In contrast, IL-10R2 interactions are very weak (~µM-mM), which may facilitate promiscuous, but essential, interactions in multiple cytokine complexes resulting in cellular signaling responses.

IL-22 and IL-10 are structurally similar to each other and the rest of the IL-10 cytokine family, which includes IL-19, IL-20, IL-24, and IL-26 ( Pestka et al., 2004 ). In addition to specific high affinity interactions with IL-22, IL-22R1 also pairs with IL-20R2 to induce IL-20 and IL-24 signaling ( Dumoutier et al., 2001a )( Figure 1 ). Like IL-22, IL-20 and IL-24 have been implicated skin homeostasis and pathology ( Blumberg et al., 2001 ; Kunz et al., 2006 ; Sa et al., 2007 ). IL-20 induces keratinocyte proliferation and over expression in mice leads to a psoriasis-like phenotype ( Blumberg et al., 2001 ). In contrast to IL-22, IL-20 and IL-24 are expressed predominantly by monocytes and keratinocytes. Under Th1 conditions, IL-22R1 and IL-20R2 chains are also expressed on keratinocytes ( Kunz et al., 2006 ). This suggests IL-20 and IL-24, via the IL-22R1/IL-20R2 complex, may provide a communication link between monocytes and keratinocytes, as well as a feedback loop on keratinocytes themselves.

The high level of IL-22 in human psoratic plaques and its pathogenic role in a mouse model of psoriasis, suggests antagonists that block the IL-22/sIL-22R1 interaction may have therapeutic potential ( Boniface et al., 2007 ; Ma et al., 2008 ; Wolk et al., 2006 ). Furthermore, cells tightly regulate IL-22 activities by expressing an IL-22 binding protein (IL-22BP), which shares 34% sequence identity with the extracellular region of IL-22R1 (sIL-22R1) ( Dumoutier et al., 2001b ; Kotenko et al., 2001b ; Wei et al., 2003 ; Xu et al., 2001 ). Three splice forms of human IL-22BP have been identified in human cells ( Kotenko et al., 2001b ). Mature IL-22BP isoform-2 (IL-22BPI2, swiss prot id Q969J5-2) is 210 amino acids long and predicted to adopt a structure similar to sIL-22R1, while isoform-3 (IL-22BPI3, swiss prot id Q969J5-3) is 109 amino acids long and should adopt a structure similar to the sIL-22R1 D1 domain ( Figure 1 ). The biochemical properties and functional roles of the IL-22BP isoforms have not been elucidated.

The fidelity of intracellular signaling between innate and adaptive immune systems is critical to balance pathogen elimination versus damage to host tissues. IL-22 and IL-22R1 play critical roles in this process. Not only by activation of the IL-22/IL-22R1/IL-10R2 complex, but through additional diverse interactions between IL-22 / IL-22BP and the participation of IL-22R1 in IL-20/IL-22R1/IL-20R2 and IL-24/IL-22R1/IL-20R2 complexes. To understand the molecular basis of these protein-protein interactions, we have determined the structure of the IL-22/sIL-22R1 complex. Comparison of IL-22/sIL-22R1 and IL-10/sIL-10R1 structures, combined with homology modeling and SPR analysis of IL-22/IL-22BP and IL-22/IL-22R1/IL-10R2 complexes, provides a molecular understanding of how these molecules form specific and promiscuous complexes with affinities that differ by ~6 orders of magnitude. ( Table 1 , Figure 1 ).

Discussion

IL-22 folds into a compact monomeric structure, in contrast to IL-10, which assembles into domain-swapped dimers ( Nagem et al., 2002 ; Walter and Nagabhushan, 1995 ; Xu et al., 2005 ; Zdanov et al., 1995 ). Interestingly, recent X-ray scattering data reveals IL-22 forms non-covalent dimers, and tetramers, in solution that adopt “V-shaped” structures similar to the IL-10 dimer ( de Oliveira Neto et al., 2007 ). Although we do not observe an IL-10-like dimer complex in the crystal lattice, it is clear that IL-22 and IL-10 form receptor complexes with very similar architectures. This suggests the overall receptor complex structure, with the exception of oligomerization (.e.g. monomer-dimer transition), is not responsible for differences in IL-22 and IL-10 signaling.

FRET data has revealed IL-10R1 and IL-10R2 associate with one another on the cell surface in the absence of IL-10 ( Krause et al., 2006 ). Since IL-22R1 pairs with IL-10R2 to induce IL-22 signaling, IL-22R1 and IL-10R2 are also predicted to form cell surface heterodimers ( Krause et al., 2006 ). Based on mechanistic studies of growth hormone and erythropoietin signaling, binding of IL-22 and IL-10 to their respective R1/R2 heterodimers is expected to induce rotational and/or conformational changes in the receptors leading to cell signaling ( Brown et al., 2005 ; Remy et al., 1999 ). Although many of the general features of the IL-22 and IL-10 signaling complexes are conserved, IL-22 and IL-10 exhibit unique specificities and receptor binding kinetics, which are dependent on the structural details of the site 1 and site 2 interfaces. IL-22/IL-22R1 and IL-10/IL-10R1 site 1 contacts exhibit K d values (nM) that are at least 4 orders of magnitude higher than site 2 contacts made by the common IL-10R2 chain (µM).

Because of the large differences in the site 1 and site 2 binding affinity/kinetics, IL-22 and IL-10 TC activation are “R1 dominant”. This suggests that despite the pre-association of R1 and R2 chains on the cell surface, cell specificity/targeting, and receptor occupancy time are controlled by the binding kinetics of the R1 chains, which are selectively expressed on different cell types ( Nagalakshmi et al., 2004 ). In contrast to IL-22R1 and IL-10R1, IL-10R2 is ubiquitously expressed on all cell types ( Nagalakshmi et al., 2004 ). The weak binding of IL-10R2, via site 2, suggests it functions as a “sensor chain”, which activates signaling based on the kinetics of the R1 chain, via site 1. Weak site 2 binding also promotes promiscuous IL-10R2 interactions with IL-22, IL-10, and other cytokines (IL-26, IL-28, and IL-29), and reduces the number of unique chains required for IL-10 family cytokine signaling. Our conclusions, based on SPR data, agree with IL-10 cell surface binding studies that reveal IL-10 binds to cells expressing only the IL-10R1 chain with an identical affinity as cells expressing both IL-10R1 and IL-10R2 ( Ding et al., 2001 ).

Additional regulation of IL-22 cell targeting/signaling is suggested by the structure and binding properties of IL-22BP isoforms. Our analysis suggests IL-22/IL-22BPI2 forms an extremely tight (K d ~1pM) interaction with IL-22, which differs considerably from the 1nM K d obtained by another recent SPR analysis of the IL-22/IL-22BPI2 complex ( Wolk et al., 2007 ). Possible reasons for these discrepancies are the use of amine coupled IL-22 by Wolk et al., which can lead to reduced on-rates and the use of short dissociation times, which can under estimate slow off-rates. Also in contrast to Wolk et al., our capture experiments suggest sIL-22R1 and IL-22BPI2 exhibit essentially identical on-rates. However, the off-rates differ dramatically resulting in T 1/2 values of ~7 minutes for IL-22/sIL-22R1 compared to ~4.7 days for IL-22/IL-22BP. The impact of these findings on IL-22 biology depends on IL-22, IL-22R1 and IL-22BP levels in various tissues, which are now being explored in more detail.

SPR analysis of IL-22BPI3 reveals it displays binding kinetics similar to the IL-22/sIL-22R1 complex. Analysis of IL-22BPI3 confirms the importance of site 1a for IL-22 binding and the small role of site 1b in IL-22/sIL-22R1 interactions. The functional role of the IL-22BPI3 isoforms remains unclear. Relative to IL-22BPI2, IL-22BPI3 is essentially inactive due to a ~2800 fold reduction in IL-22 affinity, which suggests splicing may be a mechanism to inactivate IL-22BPI2. However, since IL-22BPI3 does retain “sIL-22R1-like” affinity for IL-22, it is formally possible that IL-22/IL-22BPI3 complexes have a yet unknown role in IL-22 biology.

In addition to providing a structural model that explains observed IL-10R2 affinities and promiscuity, the IL-22 TC model has also revealed a possible role for N-linked glycosylation in IL-22R1/IL-10R2 site 2c interactions. This is interesting since carbohydrate attached to IL-22 (Asn54 IL-22 ) was previous shown to be important for IL-10R2 binding ( Logsdon et al., 2004 ). Docking studies were performed in the presence of Asn-54 IL-22 -linked carbohydrate obtained from the crystal structure of glycosylated IL-22 ( Xu et al., 2005 ). Thus, the IL-22 TC contains sufficient space to accommodate N-linked carbohydrate between sIL-22R1 and sIL-10R2, but an explanation for how mutation of Asn-54 IL-22 to glutamine disrupts sIL-10R2 binding and cell signaling could not be identified ( Logsdon et al., 2004 ). This may be due to errors in the modeled sIL-10R2 loops, or Asn-54 IL-22 may play an indirect role (e.g. mediate conformational changes) in activating the IL-22 TC.

In addition to IL-22 signaling complexes, IL-22R1 along with IL-20R2, also form promiscuous IL-20 and IL-24 TCs. However, IL-20/sIL-22R1 interactions are weak, based on our inability to isolate a stable IL-20/sIL-22R1 complex by size exclusion chromatography (N.J. Logsdon and M.R Walter, unpublished results). Thus, in contrast to the “R1 dominant” IL-22 and IL-10 signaling complexes, sIL-22R1 appears to play a different role in IL-20 and IL-24 complexes. We have identified two critical recognition elements that facilitate promiscuous binding by IL-22R1. First, IL-20 and IL-24 conserve two of three IL-22 AB loop residues (Thr-70 IL-22 , Asp-71 IL-22 , and Arg-73 IL-22 ) that form extensive contacts with sIL-22R1. Second, the low affinity of the IL-20/sIL-22R1 is consistent with the small site 1b interface observed in IL-22/sIL-22R1, which in our specificity model, is critical for promiscuous contacts. The necessity of these affinity and specificity differences may allow fine tuning of IL-19, IL-20, and IL-24 signaling via IL-22R1/IL-20R1 versus IL-20R1/IL-20R2 complexes in the skin compared to other tissues. The detailed structural basis that regulates these distinct signaling mechanisms will be the focus of future investigations.
