---
pmid: '12242332'
title: Insig-2, a second endoplasmic reticulum protein that binds SCAP and blocks
  export of sterol regulatory element-binding proteins.
authors:
- Yabe D
- Brown MS
- Goldstein JL
journal: Proc Natl Acad Sci U S A
year: '2002'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC130532
doi: 10.1073/pnas.162488899
---

# Insig-2, a second endoplasmic reticulum protein that binds SCAP and blocks export of sterol regulatory element-binding proteins.
**Authors:** Yabe D, Brown MS, Goldstein JL
**Journal:** Proc Natl Acad Sci U S A (2002)
**DOI:** [10.1073/pnas.162488899](https://doi.org/10.1073/pnas.162488899)
**PMC:** [PMC130532](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC130532/)

## Abstract

1. Proc Natl Acad Sci U S A. 2002 Oct 1;99(20):12753-8. doi: 
10.1073/pnas.162488899. Epub 2002 Sep 19.

Insig-2, a second endoplasmic reticulum protein that binds SCAP and blocks 
export of sterol regulatory element-binding proteins.

Yabe D(1), Brown MS, Goldstein JL.

Author information:
(1)Department of Molecular Genetics, University of Texas Southwestern Medical 
Center, Dallas, TX 75390-9046, USA.

This paper describes insig-2, a second protein of the endoplasmic reticulum that 
blocks the processing of sterol regulatory element-binding proteins (SREBPs) by 
binding to SCAP (SREBP cleavage-activating protein) in a sterol-regulated 
fashion, thus preventing it from escorting SREBPs to the Golgi. By blocking this 
movement, insig-2, like the previously described insig-1, prevents the 
proteolytic processing of SREBPs by Golgi enzymes, thereby blocking cholesterol 
synthesis. The sequences of human insig-1 and -2 are 59% identical. Both 
proteins are predicted to contain six transmembrane helices. The proteins differ 
functionally in two respects: (i) production of insig-1, but not insig-2, in 
cultured mammalian cells requires nuclear SREBPs; and (ii) at high levels of 
expression, insig-1, but not insig-2, can block SCAP movement in the absence of 
exogenous sterols. The combined actions of insig-1 and -2 permit feedback 
regulation of cholesterol synthesis over a wide range of sterol concentrations.

DOI: 10.1073/pnas.162488899
PMCID: PMC130532
PMID: 12242332 [Indexed for MEDLINE]

## Full Text

Abstract

This paper describes insig-2, a second protein of the endoplasmic reticulum that blocks the processing of sterol regulatory element-binding proteins (SREBPs) by binding to SCAP (SREBP cleavage-activating protein) in a sterol-regulated fashion, thus preventing it from escorting SREBPs to the Golgi. By blocking this movement, insig-2, like the previously described insig-1, prevents the proteolytic processing of SREBPs by Golgi enzymes, thereby blocking cholesterol synthesis. The sequences of human insig-1 and -2 are 59% identical. Both proteins are predicted to contain six transmembrane helices. The proteins differ functionally in two respects: ( i ) production of insig-1, but not insig-2, in cultured mammalian cells requires nuclear SREBPs; and ( ii ) at high levels of expression, insig-1, but not insig-2, can block SCAP movement in the absence of exogenous sterols. The combined actions of insig-1 and -2 permit feedback regulation of cholesterol synthesis over a wide range of sterol concentrations.

Materials and Methods

Materials used in this study and methods (cDNA cloning of insig-1 and -2, construction of expression plasmids, and Northern blot analysis) may be found in Supporting Materials and Methods , which is published as supporting information on the PNAS web site, www.pnas.org .

Results

Fig. 1 A compares the amino acid sequences of human insig-1 ( 7 ) and insig-2, the latter compiled from multiple EST clones (see Supporting Materials and Methods ). Insig-2 differs from insig-1 primarily in the NH 2 -terminal sequence, which is much shorter in insig-2. Both proteins are hydrophobic with six postulated membrane-spanning helices (Fig. 1 B ). Insig-1 and -2 mRNAs are both expressed in multiple organs of the mouse, with especially high expression in the liver (Fig. 1 C ). Immunofluorescence studies of cells overexpressing epitope-tagged versions of both proteins show an ER distribution with no apparent change in the presence of sterols (Fig. 1 D ).

Insig-2 is more highly conserved than insig-1 among various vertebrate species (Fig. 2 ). The amino acid sequences of human and hamster insig-1 and -2 are 80% and 95% identical, respectively. Zebrafish and human insig-2 are 80% identical. Most of the variation occurs in the hydrophilic NH 2 - and COOH-terminal sequences.

To test the function of insig-2, we used SRD-13A cells, a line of CHO cells that is deficient in SCAP, owing to mutations in both alleles ( 15 ). We transfected these cells transiently with a cDNA encoding SCAP under control of the strong CMV promoter, which overproduces SCAP (Fig. 3 A Upper ). The cells were also transfected with a cDNA encoding epitope-tagged SREBP-2, one of the two predominant isoforms of SREBP expressed in cultured cells ( 17 ), and either insig-1 or -2. The cells were incubated either in sterol-depleted conditions or sterol-overloaded conditions. Total cell extracts were subjected to SDS/PAGE and blotted with various antibodies. When SCAP was overexpressed, we observed the nuclear form of SREBP-2, and it did not decrease when sterols were added (lanes 1 and 2), indicating that the retention process was saturated by the excess SCAP. Expression of a small amount of insig-1 had no effect on nuclear SREBP-2 when sterols were absent, but it permitted strong inhibition when sterols were added (lanes 3 and 4). When higher amounts of insig-1 were expressed, we observed a reduction of nuclear SREBP-2 even in the absence of sterols (lanes 5–8). A different result was observed with insig-2. Expression of a low level of insig-2 restored sterol-dependent suppression of SREBP-2 processing (lanes 9 and 10). Even at high levels of expression, insig-2 continued to block SREBP-2 processing only in the presence of sterols (lanes 11–14). Throughout these experiments, we consistently observed that the insig-1 cDNA produced about 10-fold more protein than the insig-2 cDNA, even though both used the same CMV-driven expression vector and the same myc epitope tag. To achieve comparable protein expression levels, we transfected 10-fold more cDNA encoding insig-2 (see insig immunoblot in Fig. 3 A ). Insig-1 and -2 had the same effect on the processing of SREBP-1a that they had on SREBP-2 (Fig. 3 A Lower ).

The coimmunoprecipitation experiment of Fig. 3 B confirms that insig-2 binds to the SCAP/SREBP-2 complex only in the presence of sterols. Extracts of transfected cells were subjected to immunoprecipitation with an antibody against the myc epitope on insig-1 or -2. Pellets and supernatants were subjected to SDS/PAGE and blotted with various antibodies (Fig. 3 B ). When 0.1 μg of insig-1 plasmid was transfected, we observed coimmunoprecipitation of SCAP and the SREBP-2 precursor in the absence of sterols (lane 5), and this was increased by sterols (lane 6). With insig-2 there was no precipitation of SCAP or SREBP-2 in the absence of sterols (lane 7), but both proteins were coprecipitated when sterols were present (lane 8). The amounts of insig-1 and -2 expressed in these cells were similar, as determined by immunoblotting against their common myc epitopes. Nevertheless, insig-1 brought down more SCAP and SREBP-2 as compared with insig-2, suggesting that the affinity of insig-1 for SCAP is higher than that of insig-2.

As reported ( 7 ), expression of insig-1 in the presence of transfected SCAP rendered SREBP processing extremely sensitive to low concentrations of 25-hydroxycholesterol (Fig. 4 A ). Insig-2 also shifted the sterol inhibition curve (50% inhibition at 5.7 ng/ml or 14 nM), but the shift was less pronounced than with insig-1 (50% inhibition at 0.8 ng/ml or 2 nM; Fig. 4 B ).

Fig. 5 shows that expression of insig-1 or -2 restored the ability of sterols to block the exit of overexpressed SCAP from the ER as determined by an in vitro vesicle-formation assay. In previous studies using this assay, we found that sterol-mediated regulation of SCAP budding could only be demonstrated in membranes from nontransfected cells ( 11 ), presumably because the endogenous insig proteins are saturated when SCAP is overexpressed ( 7 ). The availability of the insig cDNAs allowed us to study this process in transiently transfected cells. Membranes were prepared from cells that were transfected with cDNAs encoding SREBP-2, SCAP, the G protein of VSV, and insig-1 or -2. The cells were incubated in the absence or presence of sterols, after which membranes were isolated. The membranes were incubated with cytosol in vitro under conditions that permit the budding of vesicles from the ER ( 11 ). The donor membrane fraction and the budding vesicles were separated by centrifugation and subjected to SDS/PAGE and immunoblotting. The G protein of VSV, a transmembrane protein that is exported from the ER to the Golgi, was used as a control for nonspecific effects on ER budding. In the absence of cytosol, no SCAP or the G protein of VSV was incorporated into the vesicle fraction (lane 1). In the presence of cytosol, SCAP and the G protein of VSV were both incorporated into vesicles (lane 2), and sterols had no effect on either protein (lane 3). Coexpression of insig-1 or -2 did not affect budding in the absence of sterols (lanes 4 and 6), but it permitted sterols to specifically block SCAP budding without affecting the budding of the G protein of VSV (lanes 5 and 7).

When SCAP bears a substitution of a tyrosine for a cysteine at position 298 in the sterol-sensing domain (Y298C), the protein is no longer retained in the ER on sterol addition ( 18 ). This mutant SCAP also fails to bind to insig-1 ( 7 ). To determine whether SCAP(Y298C) fails to interact with insig-2, we transfected SRD-13A cells with cDNAs encoding epitope-tagged SREBP-2, wild-type or mutant SCAP, and insig-2 (Fig. 6 A ). Insig-2 restored sterol regulation of SREBP-2 cleavage when cells expressed wild-type SCAP (lanes 5 and 6), but not when they expressed SCAP(Y298C) (lanes 9 and 10). An antibody against epitope-tagged insig-2 brought down wild-type SCAP and SREBP-2 in the presence of sterols (Fig. 6 B , lane 6), but there was no coprecipitation when the cells expressed SCAP(Y298C) (lane 8).

Earlier studies in our laboratory ( 19 ) showed that the insig-1 mRNA is increased by nuclear SREBPs ( 7 ). To determine whether insig-2 is regulated similarly, we used N-BP cells. These genetically engineered CHO cells are unable to process their own SREBPs, owing to an absence of site-2 protease, but they express truncated nuclear forms of either SREBP-1a or SREBP-2 under control of a promoter that is induced by ponasterone A ( 16 ). Ponasterone A-mediated induction of nuclear SREBP-1a or SREBP-2 in these cells led to a marked increase in insig-1 mRNA (Fig. 7 A , lanes 2 and 4). On the other hand, the insig-2 mRNA was expressed constitutively and was not affected by induction of either SREBP. To test this SREBP dependence in another way, we measured the mRNAs encoding insig-1 and -2 in a variety of mutant CHO cells with defects in processing of SREBPs. Insig-1 mRNA was present in wild-type CHO-K1 cells and CHO/pS2P cells, a parental wild-type line for mutant SRD-12B and SRD-13A cells, when they were incubated in the absence of sterols (Fig. 7 B , lanes 1 and 5), and it declined when SREBP processing was inhibited by sterols (lanes 2 and 6). Insig-1 mRNA was not expressed in M19 cells, which lack site-2 protease (Fig. 7 B , lanes 3 and 4). Insig-1 was also not expressed in SRD-12B and SRD-13A cells that lack the site-1 protease and SCAP, respectively (lanes 7–10). In striking contrast, all of these cell lines expressed similar amounts of insig-2, and there was no regulation by sterols.

Discussion

The current results indicate that cells contain a second isoform of insig that binds to SCAP in a sterol-regulated fashion and is capable of retaining the SCAP/SREBP complex in the ER. Insig-1 and -2 are both polytopic membrane proteins of the ER. They differ in two important respects: ( i ) at high levels of expression, insig-1, but not insig-2, can block SREBP processing in the absence of added exogenous sterols; and ( ii ) insig-1, but not insig-2, requires nuclear SREBPs for its expression.

The existence of a constitutively expressed form of insig resolves a paradox inherent in previous data ( 7 ). If all insig depended on nuclear SREBPs for expression, then sterols might have only a transient ability to block SREBP processing. When cells are depleted of sterols, SCAP would dissociate from insig-1 and enter budding vesicles, thereby carrying SREBPs to the Golgi for proteolytic processing. The resultant nuclear SREBPs would turn on synthesis of sterols and also insig-1. The combination of sterols and insig-1 would lead eventually to a decline in SREBP processing. When SREBPs disappear from the nucleus, insig-1 would no longer be produced, but new SCAP would be synthesized. Eventually the amount of SCAP would exceed the amount of insig-1. This would permit SCAP to escape from the ER, carrying SREBPs to the Golgi even though cellular sterols were still high. Insig-2 should prevent this escape. Inasmuch as insig-2 is constitutively expressed, its levels would not fall when SREBPs disappear from the nucleus, and this would allow continued blocking of SCAP transport, but only as long as sterols remain high.

The 25-hydroxycholesterol concentration curve of Fig. 4 suggests that insig-1 acts at lower sterol concentration than insig-2. If conditions exist in which insig-1 is selectively reduced, then cells would be expected to require higher levels of sterols to suppress SREBP processing. Indeed, cells may modulate their response to sterols by regulating the ratio of insig-1 to insig-2.

The existence of two forms of insig may have implications for the differential regulation of the SREBP-1 and -2 isoforms. Under the usual conditions of cell culture, the processing of these two isoforms is activated in parallel by sterol deprivation and repressed by sterol addition ( 1 ). However, when cells are depleted of fatty acids, the processing of SREBP-1a, but not SREBP-2, becomes resistant to sterol inhibition ( 6 ). Moreover, in livers of hamsters ( 20 ) and mice (J. Horton, I. Shimomura, M.S.B., and J.L.G., unpublished work), there is a divergence in the regulation of processing of SREBP-2 and -1c. The latter is derived from an alternate transcript from the SREBP-1 gene that predominates over the SREBP-1a transcript in liver. When animals are treated with a combination of a cholesterol synthesis inhibitor (lovastatin) and a bile acid binding resin (cholestyramine), hepatic cholesterol is depleted, and this elicits the expected increase in the processing of SREBP-2. Paradoxically, the amount of nuclear SREBP-1c decreases. In part, this is due to diminished transcription of the SREBP-1c gene, owing to a deficiency of an endogenous oxysterol ligand for the liver X receptor, which is required for SREBP-1c production ( 21 , 22 ). In addition, there appears to be an inhibition of SREBP-1c processing in these sterol-depleted livers. Further studies are necessary to determine whether the two different forms of insig, both of which are expressed in liver, may account for the two different patterns of regulation of SREBP processing in this organ.

A major unresolved question relates to the mechanism by which binding of SCAP to insig-1 or -2 leads to its retention in the ER. Inasmuch as neither insig-1 nor -2 has an obvious ER retention signal, it seems likely that these proteins must bind to other proteins that tether the insig/SCAP/SREBP complex in the ER. Present efforts are underway to identify other members of this postulated ER retention apparatus.
