---
pmid: '19369943'
title: Prdx1 inhibits tumorigenesis via regulating PTEN/AKT activity.
authors:
- Cao J
- Schulte J
- Knight A
- Leslie NR
- Zagozdzon A
- Bronson R
- Manevich Y
- Beeson C
- Neumann CA
journal: EMBO J
year: '2009'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC2688529
doi: 10.1038/emboj.2009.101
---

# Prdx1 inhibits tumorigenesis via regulating PTEN/AKT activity.
**Authors:** Cao J, Schulte J, Knight A, Leslie NR, Zagozdzon A, Bronson R, Manevich Y, Beeson C, Neumann CA
**Journal:** EMBO J (2009)
**DOI:** [10.1038/emboj.2009.101](https://doi.org/10.1038/emboj.2009.101)
**PMC:** [PMC2688529](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2688529/)

## Abstract

1. EMBO J. 2009 May 20;28(10):1505-17. doi: 10.1038/emboj.2009.101. Epub 2009 Apr
 16.

Prdx1 inhibits tumorigenesis via regulating PTEN/AKT activity.

Cao J(1), Schulte J, Knight A, Leslie NR, Zagozdzon A, Bronson R, Manevich Y, 
Beeson C, Neumann CA.

Author information:
(1)Department of Cell and Molecular Pharmacology and Experimental Therapeutics 
Medical Medical University of South Carolina, Charleston, SC 29425, USA.

It is widely accepted that reactive oxygen species (ROS) promote tumorigenesis. 
However, the exact mechanisms are still unclear. As mice lacking the peroxidase 
peroxiredoxin1 (Prdx1) produce more cellular ROS and die prematurely of cancer, 
they offer an ideal model system to study ROS-induced tumorigenesis. Prdx1 
ablation increased the susceptibility to Ras-induced breast cancer. We, 
therefore, investigated the role of Prdx1 in regulating oncogenic Ras effector 
pathways. We found Akt hyperactive in fibroblasts and mammary epithelial cells 
lacking Prdx1. Investigating the nature of such elevated Akt activation 
established a novel role for Prdx1 as a safeguard for the lipid phosphatase 
activity of PTEN, which is essential for its tumour suppressive function. We 
found binding of the peroxidase Prdx1 to PTEN essential for protecting PTEN from 
oxidation-induced inactivation. Along those lines, Prdx1 tumour suppression of 
Ras- or ErbB-2-induced transformation was mediated mainly via PTEN.

DOI: 10.1038/emboj.2009.101
PMCID: PMC2688529
PMID: 19369943 [Indexed for MEDLINE]

## Full Text

IntroductionPeroxiredoxins (Prdxs) are a superfamily of small non-seleno peroxidases (22–27 kDa) currently known to possess six mammalian isoforms. Although their individual roles in cellular redox regulation and antioxidant protection are quite distinct, they all catalyse peroxide reduction to balance cellular H2O2 levels essential for signalling and metabolism (Rhee, 2006). Prdxs I–IV share the common CxxC motif and use thioredoxin (Trx) as an electron donor (Chae et al, 1994a). On reaction with peroxides, the Prdxs' ‘peroxidatic' cysteine (Cys51 in Prdx1) is oxidized into a sulfenic acid intermediate, which then forms a disulfide bond with the ‘resolving' cysteine (Cys172 in Prdx1) from the other homodimer's subunit. Catalytic reduction of peroxides results in different oxidation states of the catalytic cysteine. Lower oxidation states (sulfenic acid) are reduced back to thiol by the Trx system. Higher oxidation states such as sulfinic acid can be reduced back to thiol by sulfiredoxin, whereas the oxidation to sulfonic acid is irreversible and results in Prdx degradation (Neumann and Fang, 2007). Loss of Prdx1 shortens the life span of mice due to the development of hemolytic anemia and cancer. Analysis of cells and mice lacking Prdx1 suggested that it not only regulated H2O2 levels, but also that it possessed the properties of a tumour suppressor (Neumann et al, 2003), which was further supported by the finding that Prdx1 interacts with c-Myc thereby selectively inhibiting c-Myc transcriptional activity (Egler et al, 2005).Peroxides are known to modify protein tyrosine phosphatases (PTPs) by oxidation. PTP catalytic property depends on a thiolate anion of a low pKa cysteine residue (pKa 4.7–5.4) located in the conserved motif of its active site (Zhang and Dixon, 1993; Peters et al, 1998). This highly nucleophilic group makes the initial attack on the phosphate group of the substrate, but renders the PTP extremely susceptible to oxidation resulting in its inactivation. The PTP and tumour suppressor PTEN exhibits phosphatase activity towards phosphoinositides and tyrosine phosphates and is known to be inactivated through H2O2-mediated oxidation or through growth factor signalling, including epithelial growth factor (EGF) or platelet-derived growth factor (PDGF) (Leslie et al, 2003; Kwon et al, 2004). Analysis of human recombinant PTEN revealed that two of the five cysteines in its N-terminal phosphatase domain (PTD) (Cys71 and Cys124) form a disulfide bond after oxidation, which resulted in the transient inhibition of its phosphatase activity and allowed oxidized PTEN to migrate faster on a non-reducing SDS–PAGE (Lee et al, 2002; Kwon et al, 2004). In stimulated macrophages, PTEN oxidation led to the temporary inhibition of its phosphatase activity and in the phosphorylation of Akt on Serine 473 (Leslie et al, 2003).PTEN deficiency is a hallmark of many human tumours (Keniry and Parsons, 2008) and is accompanied by enhanced cell proliferation, decreased cell apoptosis and increased Akt activity (Stambolic et al, 1998). Akt is regulated by PI3K and PTEN, thus playing a central role particularly in Ras- and ErbB-2-induced transformation. Akt activity has been proven crucial for the initiation of this transformation, both in vitro and in vivo (Sheng et al, 2001; Hutchinson et al, 2004; Lim and Counter, 2005) and phosphatase inactive PTEN (PTEN Cys124Ser) is incapable of suppressing Ras-induced transformation either in vitro or in vivo (Tolkacheva and Chan, 2000; Koul et al, 2002).Although it is known that PTEN activity is negatively regulated through oxidation, the existence of a protective mechanism inhibiting such inactivation has not been proposed. On the basis of our findings, we propose here that Prdx1 promotes PTEN tumour suppressive function by binding PTEN and protecting its lipid phosphatase activity from H2O2-induced inactivation. This way, Prdx1 controls excessive cellular Akt activity and reduces the susceptibility to H-Ras and ErbB-2-induced transformation.

Introduction

Peroxiredoxins (Prdxs) are a superfamily of small non-seleno peroxidases (22–27 kDa) currently known to possess six mammalian isoforms. Although their individual roles in cellular redox regulation and antioxidant protection are quite distinct, they all catalyse peroxide reduction to balance cellular H2O2 levels essential for signalling and metabolism (Rhee, 2006). Prdxs I–IV share the common CxxC motif and use thioredoxin (Trx) as an electron donor (Chae et al, 1994a). On reaction with peroxides, the Prdxs' ‘peroxidatic' cysteine (Cys51 in Prdx1) is oxidized into a sulfenic acid intermediate, which then forms a disulfide bond with the ‘resolving' cysteine (Cys172 in Prdx1) from the other homodimer's subunit. Catalytic reduction of peroxides results in different oxidation states of the catalytic cysteine. Lower oxidation states (sulfenic acid) are reduced back to thiol by the Trx system. Higher oxidation states such as sulfinic acid can be reduced back to thiol by sulfiredoxin, whereas the oxidation to sulfonic acid is irreversible and results in Prdx degradation (Neumann and Fang, 2007). Loss of Prdx1 shortens the life span of mice due to the development of hemolytic anemia and cancer. Analysis of cells and mice lacking Prdx1 suggested that it not only regulated H2O2 levels, but also that it possessed the properties of a tumour suppressor (Neumann et al, 2003), which was further supported by the finding that Prdx1 interacts with c-Myc thereby selectively inhibiting c-Myc transcriptional activity (Egler et al, 2005).

Peroxides are known to modify protein tyrosine phosphatases (PTPs) by oxidation. PTP catalytic property depends on a thiolate anion of a low pKa cysteine residue (pKa 4.7–5.4) located in the conserved motif of its active site (Zhang and Dixon, 1993; Peters et al, 1998). This highly nucleophilic group makes the initial attack on the phosphate group of the substrate, but renders the PTP extremely susceptible to oxidation resulting in its inactivation. The PTP and tumour suppressor PTEN exhibits phosphatase activity towards phosphoinositides and tyrosine phosphates and is known to be inactivated through H2O2-mediated oxidation or through growth factor signalling, including epithelial growth factor (EGF) or platelet-derived growth factor (PDGF) (Leslie et al, 2003; Kwon et al, 2004). Analysis of human recombinant PTEN revealed that two of the five cysteines in its N-terminal phosphatase domain (PTD) (Cys71 and Cys124) form a disulfide bond after oxidation, which resulted in the transient inhibition of its phosphatase activity and allowed oxidized PTEN to migrate faster on a non-reducing SDS–PAGE (Lee et al, 2002; Kwon et al, 2004). In stimulated macrophages, PTEN oxidation led to the temporary inhibition of its phosphatase activity and in the phosphorylation of Akt on Serine 473 (Leslie et al, 2003).

PTEN deficiency is a hallmark of many human tumours (Keniry and Parsons, 2008) and is accompanied by enhanced cell proliferation, decreased cell apoptosis and increased Akt activity (Stambolic et al, 1998). Akt is regulated by PI3K and PTEN, thus playing a central role particularly in Ras- and ErbB-2-induced transformation. Akt activity has been proven crucial for the initiation of this transformation, both in vitro and in vivo (Sheng et al, 2001; Hutchinson et al, 2004; Lim and Counter, 2005) and phosphatase inactive PTEN (PTEN Cys124Ser) is incapable of suppressing Ras-induced transformation either in vitro or in vivo (Tolkacheva and Chan, 2000; Koul et al, 2002).

Although it is known that PTEN activity is negatively regulated through oxidation, the existence of a protective mechanism inhibiting such inactivation has not been proposed. On the basis of our findings, we propose here that Prdx1 promotes PTEN tumour suppressive function by binding PTEN and protecting its lipid phosphatase activity from H2O2-induced inactivation. This way, Prdx1 controls excessive cellular Akt activity and reduces the susceptibility to H-Ras and ErbB-2-induced transformation.

ResultsPrdx1 inhibits Ras-induced transformation through its peroxidase activityWe tested first Prdx1 peroxidase activity under physiological conditions and measured endogenous H2O2 release in growing cells over time. Extracellular H2O2 buildup from Prdx1−/−;Prdx1WTMEFs was less compared with Prdx1−/−MEFs during the 240 min measured. In contrast, Prdx1−/−;Prdx1C51/172SMEFs resulted in an excess of extracellular H2O2 buildup (Supplementary Figure S1A). Rates calculated for extracellular H2O2 buildup (pmol/min) confirmed that Prdx1−/−;Prdx1WTMEFs released H2O2 at a 30% lower rate than Prdx1−/−MEFs, whereas Prdx1−/−;Cys51/172SMEFs released H2O2-release 1.5-fold more than Prdx1−/−MEFs (Figure 1A, left panel). Western blotting analysis further confirmed that Prdx1 was functioning as a peroxidase, as Prdx1WT protein formed mostly DTT-reducible (data not shown) H2O2-scavenging dimeric structures (Figure 1A, right panel), whereas Prdx1Cys51/172S did not, as dimer formation depends on Cys51 and Cys172 disulfides (Chae et al, 1994b). The remaining Prdx1 monomers were not over-oxidized and therefore active (unlike Prdx1 monomers in H2O2-treated Prdx1−/−;Prdx1WTMEFs). Prdx1 peroxidase activity was required to suppress H-Ras-induced tumour suppression, as Prdx1−/−MEFs transduced with retrovirus expressing H-RasV12 formed two-fold more colonies in soft agar compared with Prdx1+/+;H−RasV12MEFs (Figure 1B, left panel). Expression of exogenous Prdx1WT reduced colony formation by 50% in Prdx1−/−;H−RasV12MEFs and by 20% in Prdx1+/+;H−RasV12MEFs, whereas exogenously expressed Prdx1C51/172S modestly increased colony formation in Prdx1−/−;H−RasV12MEFs and Prdx1+/+;H−RasV12MEFs. Western blot analysis ensured that re-expression of Prdx1 proteins in Prdx1−/−MEFs did not exceed those Prdx1 levels found in Prdx1+/+MEFs (Figure 1B, right panel). Prdx1−/−;H−RasV12MEFs appeared also more spindle-like than Prdx1+/+;H−RasV12MEFs, whereas expression of Prdx1WT in Prdx1−/−;H−RasV12MEFs reversed the spindle-like morphology. Expression of exogenous Prdx1C51/172S, however, promoted spindle-like cell morphology in both Prdx1−/−;H−RasV12MEFs and Prdx1+/+;H−RasV12MEFs (Supplementary Figure S1B). Analysis of the Prdx1 peroxidase activity in H-RasV12-transformed MEFs showed that Prdx1−/−;Prdx1WT;H−RasV12MEFs released H2O2 at a 26% lesser rate than Prdx1−/−;H−RasV12MEFs, which was comparable to the rate found Prdx1−/−EVMEFs. In contrast, Prdx1−/−;C51/172S;H−RasV12MEFs had a 1.5-fold increased rate of H2O2 release compared with Prdx1−/−;Prdx1WT;H−RasV12MEFs (Figure 1C).Loss of Prdx1 promotes PTEN oxidation and Akt activationWe investigated now whether Prdx1 regulates H-Ras effector pathways known to promote transformation, including PI3K/Akt, RalGEF or MAPK pathways (Li et al, 2004; Lim and Counter, 2005). Treatment of MEFs with H2O2 (50 μM) for increasing periods of time enhanced Akt phosphorylation on Ser473 (pAktSer473) and Thr308 (pAktThr308) at a higher level in Prdx1−/−MEFs compared with Prdx1+/+MEFs (Figure 2A). This finding correlated with a steady increase of PTEN oxidation over time in Prdx1−/−MEFs, whereas in Prdx1+/+MEFs PTEN oxidation plateaued after 6–8 min (Figure 2B and C). Phosphorylation of Akt on serine 473 (pAktSer473) and PTEN oxidation increased also in Prdx1−/− MEFs compared with Prdx1+/+MEFs when exposed to increasing amounts of H2O2 (Supplementary Figure S2A and B). Similar data were obtained by inducing H2O2 endogenously through treating cells with PDGF, as it led to a larger increase in (1) pAktSer473 and pAktThr308 (Figure 2D) and (2) PTEN oxidation in Prdx1−/−MEFs when compared with Prdx1+/+MEFs (Figure 2E). Although PTEN oxidation steadily increased in Prdx1−/−MEFs, it decreased in Prdx1+/+MEFs over time (Figure 2E and F). Levels of PAktSer473 and PTEN oxidation was also increased in Prdx1−/−MEFs compared with Prdx1+/+MEFs when exposed to increasing amounts of H2O2 or PDGF (Supplementary Figure S2A–D). Lastly, we confirmed that lack of Prdx1 enhanced basal and H2O2-induced phosphorylation of Akt substrates in Prdx1−/−MEFs more than in Prdx1+/+MEFs (Figure 2G).Prdx1 interacts with PTENPTEN forms a complex with endogenous and epitope-tagged HA–Prdx1 (Figure 3A). This complex was regulated by oxidative stress, as co-expressed epitope-tagged Myc–PTEN and HA–Prdx1 dissociated under increasing concentrations of oxidative stress (Figure 3B, upper panel). In contrast, increasing dosages of H2O2 did not disrupt binding of Prdx1Cys51/72Ser with PTEN (Figure 3C). Immunoblotting confirmed equal expression of proteins (Supplementary Figure S3A and B). Peroxidatic Cys51 may regulate the H2O2-induced Prdx1:PTEN complex disruption as Prdx1C51S and Prdx1C51/172S proteins bind more to wild-type PTEN than Prdx1C172S (Figure 3D). We further confirmed a physical association between Prdx1 and PTEN by using GST-pull down of GST–Prdx1 and His–PTEN (Figure 3E). Mutational analysis suggested that Prdx1 interacts within the C2 domain of PTEN (amino acids 186–274) (Figure 3F; Supplementary Figure S3C and D) and PTEN with the N terminus of Prdx1 (amino acids 1–40) and the C terminus of Prdx1 (amino acids 157–199) (Figure 3F, lower schematic and Supplementary Figure S3G). Computational analysis allowed us to narrow those interaction surfaces (Figure 3G). The images of Prdx1 and PTEN were generated from coordinates from the protein database 1d5r (Lee et al, 1999) and 2z9s (Matsumura et al, 2008), respectively. As shown for PTEN, the region of 186–251 consists of two double-stranded anti-parallel sheets connected through two flexible loops. Similarly, the 1–40 region of Prdx1 consists of a single double-stranded anti-parallel sheet and another, surface exposed single sheet strand. In Figure 3G, the Prdx-1 protein is shown only as the monomer and the helical region that lies at the dimer interface, residues 183–199, are also positioned in a surface exposed position that is obviously well poised to mediate a protein–protein interaction. Deleting the identified interaction sites in PTEN and Prdx1 altered enzyme activity of both enzymes (data not shown), which did not allow us to study the role of a physical interaction in protecting PTEN lipid phosphatase activity.Prdx1 interaction with PTEN protects and promotes PTEN lipid phosphates activity under oxidative stressPTEN lipid phosphatase activity was fully protected by Prdx1 in cells under mild oxidative stress (25 μM H2O2) (Figure 4A), where Prdx1 was found to bind PTEN (Figure 3B). However, under higher oxidative stress treatment (500 μM H2O2), which resulted in decreased binding of PTEN and Prdx1 (Figure 3B), impaired PTEN's lipid phosphatase activity. Western blotting confirmed equal amounts of PTEN protein in the lipid phosphatase assay (Figure 4A, lower panel). To complement those results, we exposed recombinant purified His-tagged PTEN to H2O2 in the absence or presence of increasing amounts of purified recombinant Prdx1 and assayed PTEN for PI(3,4,5)P3 utilization. As expected (Figure 4B), in the presence of H2O2, we found that low molar amounts of Prdx1 protein weakly protected PTEN lipid phosphatase activity, whereas equimolar Prdx1 fully restored and further increases in Prdx1 protein amount did not enhance PTEN lipid phosphatase activity any more. Pre-incubation of PTEN in this assay with N-and C-terminal Prdx1 peptide replica (P1 and P2, respectively; peptide sequences were based on Prdx1 interaction mapping as shown in Supplementary Figure S3G and discussed for Figure 3G) decreased PTEN lipid phosphatase activity efficiently to 53%, which was comparable to PTEN activity observed with H2O2 treatment in the absence of Prdx1 (50%) (Figure 4C). Addition of P1 decreased PTEN lipid phosphatase activity only to 65%, whereas P2 lowered it to 57%. However, as shown in Figure 4D, the addition of His-purified PTEN to recombinant Prdx1 inhibited Prdx1 peroxidase activity. Under our experimental conditions, the kinetics of NADPH oxidation to NADP+ were linear and the Prdx1 inhibition was proportional to molar amount of PTEN and reached saturation (completed inhibition) at PTEN/Prdx1=1:1 (mol/mol) ratio. Moreover, an increase in PTEN ratio lowered the velocity of Prdx1 peroxidase activity (Figure 4E).Prdx1 suppresses H-Ras and ErbB-2-induced transformation mainly via promoting PTEN activityNext, we examined in PTEN−/−MEFs whether Prdx1 has an important function in PTEN-induced tumour suppression. Although Prdx1 knock down in PTEN−/−MEFs decreased pAktSer473 levels, H2O2 treatment of MEFs increased it equally in PTEN−/−MEFs and PTEN+/+MEFs (Figure 5A). Prdx1 is neither over-oxidized nor does it form less peroxidase active dimers in PTEN−/−MEFs compared with PTEN−/−MEFs reconstituted with GFP–PTEN, suggesting that in PTEN-null cells, Prdx1 activity is not compromised after H2O2 treatment (Figure 5B). PI3K/Akt signalling is also essential for oncogenic ErbB-2-induced transformation (Maroulakou et al, 2007), and Akt1 deficiency sufficiently suppresses tumour development in PTEN+/−mice (Chen et al, 2006). As shown for H-Ras (Figure 1B), Prdx1WT also prevented ErbB-2/neuT-induced transformation depending on its peroxidase activity (Supplementary Figure S5A). By using small hairpin (sh) RNA targeting Prdx1 mRNA (shPrdx1) in PTEN−/−MEFs, transformed by either H-RasV12 or ErbB-2/neuT, we showed that Prdx1 suppressed transformation mainly by regulating PTEN (Figure 5C–E). Although shPrdx1 expressed in PTEN−/−;H−RasV12MEFs or PTEN−/−;ErbB−2/neuT MEFs did not increase colony formation compared with PTEN−/−;H−Ras;pMKO1MEFs or PTEN−/−ErbB−2/neuT;pMKO1MEFs, in PTEN−/−;shPrdx1;GFP−PTEN,H−RasV12MEFs showed 1.5-fold increase in colony formation compared with PTEN−/−;pMKO1;GFP−PTEN,H−RasV12MEFs and PTEN−/−;shPrdx1;GFP−PTEN,ErbB−2/neuTMEFs and 1.35-fold increase in colony formation compared with PTEN−/−;pKMO1;GFP−PTEN;ErbB−2/neuTMEFs. Furthermore, a slight increase of Prdx1 oxidation was noted in PTEN−/−;pMKO1;H−RasV12MEFs compared with PTEN−/−;GFP−PTEN;H−RasV12MEFs and in PTEN−/−;pMKO1;ErbB−2/neuTMEFs compared with PTEN−/−;GFP−PTEN;ErbB−2/neuTMEFs. By using minimum dosages of two well-known PI3K inhibitors, Wortmannin (WM) and Ly294002 (Ly) that inhibited H2O2-induced Akt activity in Prdx1−/−MEFs less efficiently than in Prdx1+/+MEFs (Supplementary Figure S5B), we found that partial inhibition of PI3K/Akt activity translates into less efficient inhibition of H-Ras or ErbB-2-induced transformation in Prdx1−/−MEFs compared with Prdx1+/+MEFs (Table I): 5 nM WM reduced Prdx1−/−;H−RasV12MEFs colony number by <4%, 10 nM WM by 2.3%. However, 5 nM WM decreased Prdx1+/+;H−RasV12MEFs colony numbers by 46% and (P=<0.001) 10 nM WM by 67%. Similarly, 10 μM Ly decreased Prdx1−/−;H−RasV21MEFs colony number by 61% and in Prdx1+/+;H−RasV12MEFs by 73%. However, 10 μM Ly decreased colony number of Prdx1−/−;ErbB−2/neuTMEFs by only 53%, whereas in Prdx1+/+;ErbB−2/neuTMEFs by 75%. Moreover, 25 μM Ly decreased colony formation equally in Prdx1−/−;ErbB−2/neuTMEFs and Prdx1+/+;ErbB−2/neuTMEFs to 4–5%.Loss of Prdx1 promotes H-Ras-induced mammary tumours and PTEN oxidation in mammary epithelial cellsMMTV/v-H-Ras mice transgenic mice are highly susceptible to mammary carcinomas due to the targeted over-expression of the activated Ras oncogene in their mammary glands (Sinn et al, 1987) and Akt1 ablation lowers the incidence of tumours in these mice (Maroulakou et al, 2007). In our experiment, we confirmed that Prdx1 plays a role in Ras-induced tumorigenesis in vivo (Figure 6A). In all, 60% of the Ras/Prdx1+/+ mice remained tumour free during the entire observation period of 36 weeks. In contrast, only 40% of the Ras/Prdx1+/− mice remained tumour free during that time with a median survival (T50) of 28 weeks, which indicated that the loss of one copy of Prdx1 accelerated Ras-induced breast cancer. Moreover, eliminating both copies of Prdx1 in MMTV/v-H-Ras mice (Ras/Prdx1−/−), the T50 for breast cancer was only 20 weeks. We then evaluated basal PTEN oxidation and pAktSer473 in mammary epithelial cells (MECs) from 16-week-old mice that had not developed tumours. Lack of Prdx1 increased basal PTEN oxidation (Figure 6B) and pAktSer473 (Figure 6C).

Prdx1 inhibits Ras-induced transformation through its peroxidase activityWe tested first Prdx1 peroxidase activity under physiological conditions and measured endogenous H2O2 release in growing cells over time. Extracellular H2O2 buildup from Prdx1−/−;Prdx1WTMEFs was less compared with Prdx1−/−MEFs during the 240 min measured. In contrast, Prdx1−/−;Prdx1C51/172SMEFs resulted in an excess of extracellular H2O2 buildup (Supplementary Figure S1A). Rates calculated for extracellular H2O2 buildup (pmol/min) confirmed that Prdx1−/−;Prdx1WTMEFs released H2O2 at a 30% lower rate than Prdx1−/−MEFs, whereas Prdx1−/−;Cys51/172SMEFs released H2O2-release 1.5-fold more than Prdx1−/−MEFs (Figure 1A, left panel). Western blotting analysis further confirmed that Prdx1 was functioning as a peroxidase, as Prdx1WT protein formed mostly DTT-reducible (data not shown) H2O2-scavenging dimeric structures (Figure 1A, right panel), whereas Prdx1Cys51/172S did not, as dimer formation depends on Cys51 and Cys172 disulfides (Chae et al, 1994b). The remaining Prdx1 monomers were not over-oxidized and therefore active (unlike Prdx1 monomers in H2O2-treated Prdx1−/−;Prdx1WTMEFs). Prdx1 peroxidase activity was required to suppress H-Ras-induced tumour suppression, as Prdx1−/−MEFs transduced with retrovirus expressing H-RasV12 formed two-fold more colonies in soft agar compared with Prdx1+/+;H−RasV12MEFs (Figure 1B, left panel). Expression of exogenous Prdx1WT reduced colony formation by 50% in Prdx1−/−;H−RasV12MEFs and by 20% in Prdx1+/+;H−RasV12MEFs, whereas exogenously expressed Prdx1C51/172S modestly increased colony formation in Prdx1−/−;H−RasV12MEFs and Prdx1+/+;H−RasV12MEFs. Western blot analysis ensured that re-expression of Prdx1 proteins in Prdx1−/−MEFs did not exceed those Prdx1 levels found in Prdx1+/+MEFs (Figure 1B, right panel). Prdx1−/−;H−RasV12MEFs appeared also more spindle-like than Prdx1+/+;H−RasV12MEFs, whereas expression of Prdx1WT in Prdx1−/−;H−RasV12MEFs reversed the spindle-like morphology. Expression of exogenous Prdx1C51/172S, however, promoted spindle-like cell morphology in both Prdx1−/−;H−RasV12MEFs and Prdx1+/+;H−RasV12MEFs (Supplementary Figure S1B). Analysis of the Prdx1 peroxidase activity in H-RasV12-transformed MEFs showed that Prdx1−/−;Prdx1WT;H−RasV12MEFs released H2O2 at a 26% lesser rate than Prdx1−/−;H−RasV12MEFs, which was comparable to the rate found Prdx1−/−EVMEFs. In contrast, Prdx1−/−;C51/172S;H−RasV12MEFs had a 1.5-fold increased rate of H2O2 release compared with Prdx1−/−;Prdx1WT;H−RasV12MEFs (Figure 1C).

Prdx1 inhibits Ras-induced transformation through its peroxidase activity

We tested first Prdx1 peroxidase activity under physiological conditions and measured endogenous H2O2 release in growing cells over time. Extracellular H2O2 buildup from Prdx1−/−;Prdx1WTMEFs was less compared with Prdx1−/−MEFs during the 240 min measured. In contrast, Prdx1−/−;Prdx1C51/172SMEFs resulted in an excess of extracellular H2O2 buildup (Supplementary Figure S1A). Rates calculated for extracellular H2O2 buildup (pmol/min) confirmed that Prdx1−/−;Prdx1WTMEFs released H2O2 at a 30% lower rate than Prdx1−/−MEFs, whereas Prdx1−/−;Cys51/172SMEFs released H2O2-release 1.5-fold more than Prdx1−/−MEFs (Figure 1A, left panel). Western blotting analysis further confirmed that Prdx1 was functioning as a peroxidase, as Prdx1WT protein formed mostly DTT-reducible (data not shown) H2O2-scavenging dimeric structures (Figure 1A, right panel), whereas Prdx1Cys51/172S did not, as dimer formation depends on Cys51 and Cys172 disulfides (Chae et al, 1994b). The remaining Prdx1 monomers were not over-oxidized and therefore active (unlike Prdx1 monomers in H2O2-treated Prdx1−/−;Prdx1WTMEFs). Prdx1 peroxidase activity was required to suppress H-Ras-induced tumour suppression, as Prdx1−/−MEFs transduced with retrovirus expressing H-RasV12 formed two-fold more colonies in soft agar compared with Prdx1+/+;H−RasV12MEFs (Figure 1B, left panel). Expression of exogenous Prdx1WT reduced colony formation by 50% in Prdx1−/−;H−RasV12MEFs and by 20% in Prdx1+/+;H−RasV12MEFs, whereas exogenously expressed Prdx1C51/172S modestly increased colony formation in Prdx1−/−;H−RasV12MEFs and Prdx1+/+;H−RasV12MEFs. Western blot analysis ensured that re-expression of Prdx1 proteins in Prdx1−/−MEFs did not exceed those Prdx1 levels found in Prdx1+/+MEFs (Figure 1B, right panel). Prdx1−/−;H−RasV12MEFs appeared also more spindle-like than Prdx1+/+;H−RasV12MEFs, whereas expression of Prdx1WT in Prdx1−/−;H−RasV12MEFs reversed the spindle-like morphology. Expression of exogenous Prdx1C51/172S, however, promoted spindle-like cell morphology in both Prdx1−/−;H−RasV12MEFs and Prdx1+/+;H−RasV12MEFs (Supplementary Figure S1B). Analysis of the Prdx1 peroxidase activity in H-RasV12-transformed MEFs showed that Prdx1−/−;Prdx1WT;H−RasV12MEFs released H2O2 at a 26% lesser rate than Prdx1−/−;H−RasV12MEFs, which was comparable to the rate found Prdx1−/−EVMEFs. In contrast, Prdx1−/−;C51/172S;H−RasV12MEFs had a 1.5-fold increased rate of H2O2 release compared with Prdx1−/−;Prdx1WT;H−RasV12MEFs (Figure 1C).

Loss of Prdx1 promotes PTEN oxidation and Akt activationWe investigated now whether Prdx1 regulates H-Ras effector pathways known to promote transformation, including PI3K/Akt, RalGEF or MAPK pathways (Li et al, 2004; Lim and Counter, 2005). Treatment of MEFs with H2O2 (50 μM) for increasing periods of time enhanced Akt phosphorylation on Ser473 (pAktSer473) and Thr308 (pAktThr308) at a higher level in Prdx1−/−MEFs compared with Prdx1+/+MEFs (Figure 2A). This finding correlated with a steady increase of PTEN oxidation over time in Prdx1−/−MEFs, whereas in Prdx1+/+MEFs PTEN oxidation plateaued after 6–8 min (Figure 2B and C). Phosphorylation of Akt on serine 473 (pAktSer473) and PTEN oxidation increased also in Prdx1−/− MEFs compared with Prdx1+/+MEFs when exposed to increasing amounts of H2O2 (Supplementary Figure S2A and B). Similar data were obtained by inducing H2O2 endogenously through treating cells with PDGF, as it led to a larger increase in (1) pAktSer473 and pAktThr308 (Figure 2D) and (2) PTEN oxidation in Prdx1−/−MEFs when compared with Prdx1+/+MEFs (Figure 2E). Although PTEN oxidation steadily increased in Prdx1−/−MEFs, it decreased in Prdx1+/+MEFs over time (Figure 2E and F). Levels of PAktSer473 and PTEN oxidation was also increased in Prdx1−/−MEFs compared with Prdx1+/+MEFs when exposed to increasing amounts of H2O2 or PDGF (Supplementary Figure S2A–D). Lastly, we confirmed that lack of Prdx1 enhanced basal and H2O2-induced phosphorylation of Akt substrates in Prdx1−/−MEFs more than in Prdx1+/+MEFs (Figure 2G).

Loss of Prdx1 promotes PTEN oxidation and Akt activation

We investigated now whether Prdx1 regulates H-Ras effector pathways known to promote transformation, including PI3K/Akt, RalGEF or MAPK pathways (Li et al, 2004; Lim and Counter, 2005). Treatment of MEFs with H2O2 (50 μM) for increasing periods of time enhanced Akt phosphorylation on Ser473 (pAktSer473) and Thr308 (pAktThr308) at a higher level in Prdx1−/−MEFs compared with Prdx1+/+MEFs (Figure 2A). This finding correlated with a steady increase of PTEN oxidation over time in Prdx1−/−MEFs, whereas in Prdx1+/+MEFs PTEN oxidation plateaued after 6–8 min (Figure 2B and C). Phosphorylation of Akt on serine 473 (pAktSer473) and PTEN oxidation increased also in Prdx1−/− MEFs compared with Prdx1+/+MEFs when exposed to increasing amounts of H2O2 (Supplementary Figure S2A and B). Similar data were obtained by inducing H2O2 endogenously through treating cells with PDGF, as it led to a larger increase in (1) pAktSer473 and pAktThr308 (Figure 2D) and (2) PTEN oxidation in Prdx1−/−MEFs when compared with Prdx1+/+MEFs (Figure 2E). Although PTEN oxidation steadily increased in Prdx1−/−MEFs, it decreased in Prdx1+/+MEFs over time (Figure 2E and F). Levels of PAktSer473 and PTEN oxidation was also increased in Prdx1−/−MEFs compared with Prdx1+/+MEFs when exposed to increasing amounts of H2O2 or PDGF (Supplementary Figure S2A–D). Lastly, we confirmed that lack of Prdx1 enhanced basal and H2O2-induced phosphorylation of Akt substrates in Prdx1−/−MEFs more than in Prdx1+/+MEFs (Figure 2G).

Prdx1 interacts with PTENPTEN forms a complex with endogenous and epitope-tagged HA–Prdx1 (Figure 3A). This complex was regulated by oxidative stress, as co-expressed epitope-tagged Myc–PTEN and HA–Prdx1 dissociated under increasing concentrations of oxidative stress (Figure 3B, upper panel). In contrast, increasing dosages of H2O2 did not disrupt binding of Prdx1Cys51/72Ser with PTEN (Figure 3C). Immunoblotting confirmed equal expression of proteins (Supplementary Figure S3A and B). Peroxidatic Cys51 may regulate the H2O2-induced Prdx1:PTEN complex disruption as Prdx1C51S and Prdx1C51/172S proteins bind more to wild-type PTEN than Prdx1C172S (Figure 3D). We further confirmed a physical association between Prdx1 and PTEN by using GST-pull down of GST–Prdx1 and His–PTEN (Figure 3E). Mutational analysis suggested that Prdx1 interacts within the C2 domain of PTEN (amino acids 186–274) (Figure 3F; Supplementary Figure S3C and D) and PTEN with the N terminus of Prdx1 (amino acids 1–40) and the C terminus of Prdx1 (amino acids 157–199) (Figure 3F, lower schematic and Supplementary Figure S3G). Computational analysis allowed us to narrow those interaction surfaces (Figure 3G). The images of Prdx1 and PTEN were generated from coordinates from the protein database 1d5r (Lee et al, 1999) and 2z9s (Matsumura et al, 2008), respectively. As shown for PTEN, the region of 186–251 consists of two double-stranded anti-parallel sheets connected through two flexible loops. Similarly, the 1–40 region of Prdx1 consists of a single double-stranded anti-parallel sheet and another, surface exposed single sheet strand. In Figure 3G, the Prdx-1 protein is shown only as the monomer and the helical region that lies at the dimer interface, residues 183–199, are also positioned in a surface exposed position that is obviously well poised to mediate a protein–protein interaction. Deleting the identified interaction sites in PTEN and Prdx1 altered enzyme activity of both enzymes (data not shown), which did not allow us to study the role of a physical interaction in protecting PTEN lipid phosphatase activity.

Prdx1 interacts with PTEN

PTEN forms a complex with endogenous and epitope-tagged HA–Prdx1 (Figure 3A). This complex was regulated by oxidative stress, as co-expressed epitope-tagged Myc–PTEN and HA–Prdx1 dissociated under increasing concentrations of oxidative stress (Figure 3B, upper panel). In contrast, increasing dosages of H2O2 did not disrupt binding of Prdx1Cys51/72Ser with PTEN (Figure 3C). Immunoblotting confirmed equal expression of proteins (Supplementary Figure S3A and B). Peroxidatic Cys51 may regulate the H2O2-induced Prdx1:PTEN complex disruption as Prdx1C51S and Prdx1C51/172S proteins bind more to wild-type PTEN than Prdx1C172S (Figure 3D). We further confirmed a physical association between Prdx1 and PTEN by using GST-pull down of GST–Prdx1 and His–PTEN (Figure 3E). Mutational analysis suggested that Prdx1 interacts within the C2 domain of PTEN (amino acids 186–274) (Figure 3F; Supplementary Figure S3C and D) and PTEN with the N terminus of Prdx1 (amino acids 1–40) and the C terminus of Prdx1 (amino acids 157–199) (Figure 3F, lower schematic and Supplementary Figure S3G). Computational analysis allowed us to narrow those interaction surfaces (Figure 3G). The images of Prdx1 and PTEN were generated from coordinates from the protein database 1d5r (Lee et al, 1999) and 2z9s (Matsumura et al, 2008), respectively. As shown for PTEN, the region of 186–251 consists of two double-stranded anti-parallel sheets connected through two flexible loops. Similarly, the 1–40 region of Prdx1 consists of a single double-stranded anti-parallel sheet and another, surface exposed single sheet strand. In Figure 3G, the Prdx-1 protein is shown only as the monomer and the helical region that lies at the dimer interface, residues 183–199, are also positioned in a surface exposed position that is obviously well poised to mediate a protein–protein interaction. Deleting the identified interaction sites in PTEN and Prdx1 altered enzyme activity of both enzymes (data not shown), which did not allow us to study the role of a physical interaction in protecting PTEN lipid phosphatase activity.

Prdx1 interaction with PTEN protects and promotes PTEN lipid phosphates activity under oxidative stressPTEN lipid phosphatase activity was fully protected by Prdx1 in cells under mild oxidative stress (25 μM H2O2) (Figure 4A), where Prdx1 was found to bind PTEN (Figure 3B). However, under higher oxidative stress treatment (500 μM H2O2), which resulted in decreased binding of PTEN and Prdx1 (Figure 3B), impaired PTEN's lipid phosphatase activity. Western blotting confirmed equal amounts of PTEN protein in the lipid phosphatase assay (Figure 4A, lower panel). To complement those results, we exposed recombinant purified His-tagged PTEN to H2O2 in the absence or presence of increasing amounts of purified recombinant Prdx1 and assayed PTEN for PI(3,4,5)P3 utilization. As expected (Figure 4B), in the presence of H2O2, we found that low molar amounts of Prdx1 protein weakly protected PTEN lipid phosphatase activity, whereas equimolar Prdx1 fully restored and further increases in Prdx1 protein amount did not enhance PTEN lipid phosphatase activity any more. Pre-incubation of PTEN in this assay with N-and C-terminal Prdx1 peptide replica (P1 and P2, respectively; peptide sequences were based on Prdx1 interaction mapping as shown in Supplementary Figure S3G and discussed for Figure 3G) decreased PTEN lipid phosphatase activity efficiently to 53%, which was comparable to PTEN activity observed with H2O2 treatment in the absence of Prdx1 (50%) (Figure 4C). Addition of P1 decreased PTEN lipid phosphatase activity only to 65%, whereas P2 lowered it to 57%. However, as shown in Figure 4D, the addition of His-purified PTEN to recombinant Prdx1 inhibited Prdx1 peroxidase activity. Under our experimental conditions, the kinetics of NADPH oxidation to NADP+ were linear and the Prdx1 inhibition was proportional to molar amount of PTEN and reached saturation (completed inhibition) at PTEN/Prdx1=1:1 (mol/mol) ratio. Moreover, an increase in PTEN ratio lowered the velocity of Prdx1 peroxidase activity (Figure 4E).

Prdx1 interaction with PTEN protects and promotes PTEN lipid phosphates activity under oxidative stress

PTEN lipid phosphatase activity was fully protected by Prdx1 in cells under mild oxidative stress (25 μM H2O2) (Figure 4A), where Prdx1 was found to bind PTEN (Figure 3B). However, under higher oxidative stress treatment (500 μM H2O2), which resulted in decreased binding of PTEN and Prdx1 (Figure 3B), impaired PTEN's lipid phosphatase activity. Western blotting confirmed equal amounts of PTEN protein in the lipid phosphatase assay (Figure 4A, lower panel). To complement those results, we exposed recombinant purified His-tagged PTEN to H2O2 in the absence or presence of increasing amounts of purified recombinant Prdx1 and assayed PTEN for PI(3,4,5)P3 utilization. As expected (Figure 4B), in the presence of H2O2, we found that low molar amounts of Prdx1 protein weakly protected PTEN lipid phosphatase activity, whereas equimolar Prdx1 fully restored and further increases in Prdx1 protein amount did not enhance PTEN lipid phosphatase activity any more. Pre-incubation of PTEN in this assay with N-and C-terminal Prdx1 peptide replica (P1 and P2, respectively; peptide sequences were based on Prdx1 interaction mapping as shown in Supplementary Figure S3G and discussed for Figure 3G) decreased PTEN lipid phosphatase activity efficiently to 53%, which was comparable to PTEN activity observed with H2O2 treatment in the absence of Prdx1 (50%) (Figure 4C). Addition of P1 decreased PTEN lipid phosphatase activity only to 65%, whereas P2 lowered it to 57%. However, as shown in Figure 4D, the addition of His-purified PTEN to recombinant Prdx1 inhibited Prdx1 peroxidase activity. Under our experimental conditions, the kinetics of NADPH oxidation to NADP+ were linear and the Prdx1 inhibition was proportional to molar amount of PTEN and reached saturation (completed inhibition) at PTEN/Prdx1=1:1 (mol/mol) ratio. Moreover, an increase in PTEN ratio lowered the velocity of Prdx1 peroxidase activity (Figure 4E).

Prdx1 suppresses H-Ras and ErbB-2-induced transformation mainly via promoting PTEN activityNext, we examined in PTEN−/−MEFs whether Prdx1 has an important function in PTEN-induced tumour suppression. Although Prdx1 knock down in PTEN−/−MEFs decreased pAktSer473 levels, H2O2 treatment of MEFs increased it equally in PTEN−/−MEFs and PTEN+/+MEFs (Figure 5A). Prdx1 is neither over-oxidized nor does it form less peroxidase active dimers in PTEN−/−MEFs compared with PTEN−/−MEFs reconstituted with GFP–PTEN, suggesting that in PTEN-null cells, Prdx1 activity is not compromised after H2O2 treatment (Figure 5B). PI3K/Akt signalling is also essential for oncogenic ErbB-2-induced transformation (Maroulakou et al, 2007), and Akt1 deficiency sufficiently suppresses tumour development in PTEN+/−mice (Chen et al, 2006). As shown for H-Ras (Figure 1B), Prdx1WT also prevented ErbB-2/neuT-induced transformation depending on its peroxidase activity (Supplementary Figure S5A). By using small hairpin (sh) RNA targeting Prdx1 mRNA (shPrdx1) in PTEN−/−MEFs, transformed by either H-RasV12 or ErbB-2/neuT, we showed that Prdx1 suppressed transformation mainly by regulating PTEN (Figure 5C–E). Although shPrdx1 expressed in PTEN−/−;H−RasV12MEFs or PTEN−/−;ErbB−2/neuT MEFs did not increase colony formation compared with PTEN−/−;H−Ras;pMKO1MEFs or PTEN−/−ErbB−2/neuT;pMKO1MEFs, in PTEN−/−;shPrdx1;GFP−PTEN,H−RasV12MEFs showed 1.5-fold increase in colony formation compared with PTEN−/−;pMKO1;GFP−PTEN,H−RasV12MEFs and PTEN−/−;shPrdx1;GFP−PTEN,ErbB−2/neuTMEFs and 1.35-fold increase in colony formation compared with PTEN−/−;pKMO1;GFP−PTEN;ErbB−2/neuTMEFs. Furthermore, a slight increase of Prdx1 oxidation was noted in PTEN−/−;pMKO1;H−RasV12MEFs compared with PTEN−/−;GFP−PTEN;H−RasV12MEFs and in PTEN−/−;pMKO1;ErbB−2/neuTMEFs compared with PTEN−/−;GFP−PTEN;ErbB−2/neuTMEFs. By using minimum dosages of two well-known PI3K inhibitors, Wortmannin (WM) and Ly294002 (Ly) that inhibited H2O2-induced Akt activity in Prdx1−/−MEFs less efficiently than in Prdx1+/+MEFs (Supplementary Figure S5B), we found that partial inhibition of PI3K/Akt activity translates into less efficient inhibition of H-Ras or ErbB-2-induced transformation in Prdx1−/−MEFs compared with Prdx1+/+MEFs (Table I): 5 nM WM reduced Prdx1−/−;H−RasV12MEFs colony number by <4%, 10 nM WM by 2.3%. However, 5 nM WM decreased Prdx1+/+;H−RasV12MEFs colony numbers by 46% and (P=<0.001) 10 nM WM by 67%. Similarly, 10 μM Ly decreased Prdx1−/−;H−RasV21MEFs colony number by 61% and in Prdx1+/+;H−RasV12MEFs by 73%. However, 10 μM Ly decreased colony number of Prdx1−/−;ErbB−2/neuTMEFs by only 53%, whereas in Prdx1+/+;ErbB−2/neuTMEFs by 75%. Moreover, 25 μM Ly decreased colony formation equally in Prdx1−/−;ErbB−2/neuTMEFs and Prdx1+/+;ErbB−2/neuTMEFs to 4–5%.

Prdx1 suppresses H-Ras and ErbB-2-induced transformation mainly via promoting PTEN activity

Next, we examined in PTEN−/−MEFs whether Prdx1 has an important function in PTEN-induced tumour suppression. Although Prdx1 knock down in PTEN−/−MEFs decreased pAktSer473 levels, H2O2 treatment of MEFs increased it equally in PTEN−/−MEFs and PTEN+/+MEFs (Figure 5A). Prdx1 is neither over-oxidized nor does it form less peroxidase active dimers in PTEN−/−MEFs compared with PTEN−/−MEFs reconstituted with GFP–PTEN, suggesting that in PTEN-null cells, Prdx1 activity is not compromised after H2O2 treatment (Figure 5B). PI3K/Akt signalling is also essential for oncogenic ErbB-2-induced transformation (Maroulakou et al, 2007), and Akt1 deficiency sufficiently suppresses tumour development in PTEN+/−mice (Chen et al, 2006). As shown for H-Ras (Figure 1B), Prdx1WT also prevented ErbB-2/neuT-induced transformation depending on its peroxidase activity (Supplementary Figure S5A). By using small hairpin (sh) RNA targeting Prdx1 mRNA (shPrdx1) in PTEN−/−MEFs, transformed by either H-RasV12 or ErbB-2/neuT, we showed that Prdx1 suppressed transformation mainly by regulating PTEN (Figure 5C–E). Although shPrdx1 expressed in PTEN−/−;H−RasV12MEFs or PTEN−/−;ErbB−2/neuT MEFs did not increase colony formation compared with PTEN−/−;H−Ras;pMKO1MEFs or PTEN−/−ErbB−2/neuT;pMKO1MEFs, in PTEN−/−;shPrdx1;GFP−PTEN,H−RasV12MEFs showed 1.5-fold increase in colony formation compared with PTEN−/−;pMKO1;GFP−PTEN,H−RasV12MEFs and PTEN−/−;shPrdx1;GFP−PTEN,ErbB−2/neuTMEFs and 1.35-fold increase in colony formation compared with PTEN−/−;pKMO1;GFP−PTEN;ErbB−2/neuTMEFs. Furthermore, a slight increase of Prdx1 oxidation was noted in PTEN−/−;pMKO1;H−RasV12MEFs compared with PTEN−/−;GFP−PTEN;H−RasV12MEFs and in PTEN−/−;pMKO1;ErbB−2/neuTMEFs compared with PTEN−/−;GFP−PTEN;ErbB−2/neuTMEFs. By using minimum dosages of two well-known PI3K inhibitors, Wortmannin (WM) and Ly294002 (Ly) that inhibited H2O2-induced Akt activity in Prdx1−/−MEFs less efficiently than in Prdx1+/+MEFs (Supplementary Figure S5B), we found that partial inhibition of PI3K/Akt activity translates into less efficient inhibition of H-Ras or ErbB-2-induced transformation in Prdx1−/−MEFs compared with Prdx1+/+MEFs (Table I): 5 nM WM reduced Prdx1−/−;H−RasV12MEFs colony number by <4%, 10 nM WM by 2.3%. However, 5 nM WM decreased Prdx1+/+;H−RasV12MEFs colony numbers by 46% and (P=<0.001) 10 nM WM by 67%. Similarly, 10 μM Ly decreased Prdx1−/−;H−RasV21MEFs colony number by 61% and in Prdx1+/+;H−RasV12MEFs by 73%. However, 10 μM Ly decreased colony number of Prdx1−/−;ErbB−2/neuTMEFs by only 53%, whereas in Prdx1+/+;ErbB−2/neuTMEFs by 75%. Moreover, 25 μM Ly decreased colony formation equally in Prdx1−/−;ErbB−2/neuTMEFs and Prdx1+/+;ErbB−2/neuTMEFs to 4–5%.

Loss of Prdx1 promotes H-Ras-induced mammary tumours and PTEN oxidation in mammary epithelial cellsMMTV/v-H-Ras mice transgenic mice are highly susceptible to mammary carcinomas due to the targeted over-expression of the activated Ras oncogene in their mammary glands (Sinn et al, 1987) and Akt1 ablation lowers the incidence of tumours in these mice (Maroulakou et al, 2007). In our experiment, we confirmed that Prdx1 plays a role in Ras-induced tumorigenesis in vivo (Figure 6A). In all, 60% of the Ras/Prdx1+/+ mice remained tumour free during the entire observation period of 36 weeks. In contrast, only 40% of the Ras/Prdx1+/− mice remained tumour free during that time with a median survival (T50) of 28 weeks, which indicated that the loss of one copy of Prdx1 accelerated Ras-induced breast cancer. Moreover, eliminating both copies of Prdx1 in MMTV/v-H-Ras mice (Ras/Prdx1−/−), the T50 for breast cancer was only 20 weeks. We then evaluated basal PTEN oxidation and pAktSer473 in mammary epithelial cells (MECs) from 16-week-old mice that had not developed tumours. Lack of Prdx1 increased basal PTEN oxidation (Figure 6B) and pAktSer473 (Figure 6C).

Loss of Prdx1 promotes H-Ras-induced mammary tumours and PTEN oxidation in mammary epithelial cells

MMTV/v-H-Ras mice transgenic mice are highly susceptible to mammary carcinomas due to the targeted over-expression of the activated Ras oncogene in their mammary glands (Sinn et al, 1987) and Akt1 ablation lowers the incidence of tumours in these mice (Maroulakou et al, 2007). In our experiment, we confirmed that Prdx1 plays a role in Ras-induced tumorigenesis in vivo (Figure 6A). In all, 60% of the Ras/Prdx1+/+ mice remained tumour free during the entire observation period of 36 weeks. In contrast, only 40% of the Ras/Prdx1+/− mice remained tumour free during that time with a median survival (T50) of 28 weeks, which indicated that the loss of one copy of Prdx1 accelerated Ras-induced breast cancer. Moreover, eliminating both copies of Prdx1 in MMTV/v-H-Ras mice (Ras/Prdx1−/−), the T50 for breast cancer was only 20 weeks. We then evaluated basal PTEN oxidation and pAktSer473 in mammary epithelial cells (MECs) from 16-week-old mice that had not developed tumours. Lack of Prdx1 increased basal PTEN oxidation (Figure 6B) and pAktSer473 (Figure 6C).

DiscussionCompared with Prdxs, catalase peroxidase activity decomposes H2O2 with a >1000-fold higher KM (Chae et al, 1994a; Loewen et al, 2004) and therefore eliminates H2O2 with much higher efficiency. Thus, we cannot expect that Prdx1 scavenges H2O2 in considerable amounts, as shown in Figure 1A and Supplementary Figure S1A. Yet, Prdx1's peroxidase activity is important in preventing H-RasV12 (Figure 1B) or ErbB-2/neuT (Supplementary Figure S5A) induced transformation. Similar to the catalase knockout mice, all other currently published Prdxs knockout mice, show phenotypically no increased susceptibility to transformation (Ho et al, 1997). Therefore, Prdx1 may have a unique function amongst cellular peroxidases in tumour suppression, by affecting signalling directly through physical interaction with target enzymes. This was shown for c-Abl tyrosine kinase (Wen and VanEtten, 1997) and c-Jun terminal kinase (Kim et al, 2006). As described here for PTEN (Figure 3B), oxidative stress induced Prdx1 to dissociate, which in case of c-Abl (Neumann et al, 1998) and JNK (Kim et al, 2006) resulted in kinase activation. The Prdx1:PTEN heterodimer is most likely formed in a 1:1 molar ratio, as (1) the Prdx1 preservation of PTEN lipid phosphatase activity under oxidative stress is achieved by a 1:1 (mol:mol) ratio of Prdx1 and PTEN and could not be further increased by excess of Prdx1 (Figure 4B) and (2) Prdx1C51/172S, which is unable to form dimeric structures (Chae et al, 1994b), bound to PTEN and did not dissociate under H2O2 treatment (Figure 3C). Therefore, these data provide compelling evidence that Prdx1 may bind PTEN as a monomer.We further propose that the Prdx1:PTEN interaction is essential for protecting PTEN from oxidation-induced inactivation (Figure 4C), as Prdx1 C- and N-terminal peptide replica, which were designed based on our mapping studies and computer modelling to block PTEN docking sites on Prdx1 (Figure 3F and G; Supplementary Figure S3E), resulted after H2O2 treatment in a 47% reduction of PTEN activity. This was comparable to PTEN activity after H2O2 treatment in the absence of Prdx1 (50%) (Figure 4C). The Prdx1 N terminus, which includes Cys51, may have an important function in regulating the H2O2-induced dissociation of the Prdx1:PTEN complex, as Cys51 when replaced by a serine enhances Prdx1 binding to PTEN (Figure 3D). X-ray crystallography of Prdxs suggested significant conformational changes, such as unwinding of the active site N-terminal helix, to form disulfides, as in reduced Prdxs, the sulfur atoms of the N- and C-terminal conserved cysteine residues are too far apart to react with each other (Schroder et al, 2000; Matsumura et al, 2008). Therefore, in the Prdx1:PTEN complex, increased H2O2 may oxidize Cys51 (Prdx1), thus, promoting unwinding of the PTEN-binding conformation of Prdx1 consequently inducing dissociation.Over-oxidation of Prdx1 Cys51 occurs during catalysis and correlates with increasing amounts of Trx (Yang et al, 2002). The reduction of Cys51 by Trx inhibits the folding back of Cys51 into its pocket thereby exposing it to further oxidation by H2O2. The binding of PTEN to Prdx1 could have similar effects on Prdx1, thereby inhibiting sufficient reduction of Cys51 by Trx and decreasing Prdx1 peroxidase activity (Figure 4C and D). Alternatively, as it has been shown that Trx binds PTEN and inactivates its lipid phosphates activity, it could be that under our experimental conditions Trx is depleted from the Prdx1 scavenging system by binding PTEN (Meuillet et al, 2004). This seems unlikely though, as Trx was added in approximately 300-fold molar excess, which should provide a large Trx pool to reduce oxidized Prdx1 proteins.We defined a novel interaction site of Prdx1 with PTEN in the C2 domain of PTEN (Figure 3F; Supplementary Figure S3D and E), which together with the N-terminal PTD, are both required for enzyme activity. Surface plasmon resonance analysis revealed that the C2 domain is essential for high-affinity membrane binding of PTEN (Das et al, 2003). Our data support this finding, as it has been shown recently through chemical proteomic strategy using cleavable lipid baits, that Prdx1 is a potential novel phosphoinositides-binding protein (Pasquali et al, 2007). Prdx1 may, therefore, stabilize PTEN binding to the plasma membrane, which in a recent analysis using single-molecule TIRF microscopy, lasts only for a few hundred milliseconds (Vazquez et al, 2006). Such short time of membrane binding could explain why we (Figure 2B, C, E and F; Supplementary Figure S2B and D) and others see only small amounts of PTEN oxidized after cell exposure to H2O2 or PDGF (Kwon et al, 2004). PDGF-induced H2O2 stems either from metabolized superoxide released by NADPH oxidases (NOXs) or the mitochondria and is considered a signalling molecule (Rhee, 2006). As discussed earlier (Leslie et al, 2003), such finding may be attributed to the proximity of NOXs complexes and PIP3. Over-expression of Nox1, in PDGF-stimulated NIH 3T3 cells rapidly increases PIP3 levels, in contrast to PI3K activity, suggesting that Nox1-induced superoxide enhances PIP3 levels by inactivating PTEN rather than activating PI3K. Expression of a peroxidase inactive Prdx2 leads to a rapid increase of PIP3 levels in growth factor stimulated cells, which is reversed by Prdx2WT (Kwon et al, 2004). In our hands however, Prdx2 did not bind PTEN, as we were unable to detect an interaction of Prdx2 and PTEN by immunoprecipitation (data not shown).PTEN lipid phosphatase activity has an important function in tumour suppression (Tolkacheva and Chan, 2000; Koul et al, 2002) and Akt1 ablation protects (MMTV)-ErbB2/neu, MMTV-polyoma middle T and MMTV-v-H-Ras mice from breast cancer initiation (Skeen et al, 2006; Maroulakou et al, 2007). Here, we show that loss of Prdx1 increased basal, H2O2 and PDGF-induced Akt activity (Figure 2A, D and G) and Prdx1 peroxidase activity was essential in suppressing H-Rasv12 and ErbB-2/neuT-induced transformation (Figure 1B; Supplementary Figure S5A). Along those lines, Prdx1 ablation accelerated mammary tumorigenesis in MMTV-v-H-Ras mice and increased levels of pAktSer473 and PTEN oxidation in MECs from these mice (Figure 6A–C). Prdx1 tumour suppression seemed to occur largely via regulation of the PTEN and Akt. Knock down of Prdx1 in PTEN−/−MEFs did not significantly enhance H-Ras or ErbB-2-induced transformation (Figure 5C and D), whereas in transformed PTEN-containing MEFs, decrease in Prdx1 levels resulted in a gain of transformation: 1.5-fold for H-RasV12 and only 1.35-fold for ErbB-2/neuT, which we contributed to decreased Prdx1 knock down in PTEN−/−;shPrdx1;GFP−PTEN;ErbB−2/neuTMEFs compared with PTEN−/−;shPrdx1;GFP−PTEN;H−RasV12MEFs). Lack of PTEN also did not result in Prdx1 over-oxidation nor reduced expression (Figure 5B and E), suggesting that in PTEN-deficient cells, Prdx1 is acting as peroxidase.However, we have to consider the regulation of other phosphatases or kinases by Prdx1 or H2O2 in Akt-driven tumorigenesis. SHIP-1 and 2 are known to dephosphorylate PIP3 to PtdIns (3,4)P2 (Vivanco and Sawyers, 2002). However, SHIP2 is not inactivated by oxidative stress (Leslie et al, 2003). PTEN−/−;shPrdx1MEFs showed decreased pAKTSer473 levels compared with PTEN−/−MEFs (Figure 5D), perhaps due to the activation of SHIP2 by small amounts of H2O2. However, H2O2 treatment of PTEN−/−;shPrdx1MEFs increased pAKTSer473 levels comparable to PTEN−/−MEFs, suggesting maybe a H2O2-induced inactivation of PP2A (Rao and Clayton, 2002), as loss of Prdx1 also inactivated PP2A activity (Supplementary Figure 2E). We also cannot exclude a regulation of PI3K by Prdx1, as H-Ras and ErbB-2/neuT transformed Prdx1−/−MEFs display a lesser sensitivity towards PI3K inhibitors (Table I; Supplementary Figure 5B), compared with Prdx1+/+MEFs.In summary, oxidative stress and carcinogenesis are closely linked, however, molecular details, especially how oxidative stress promotes pro-oncogenic signalling pathways are scarce. Our studies shed some light on how H2O2 can promote oncogenic Ras and ErbB-2 signalling by providing evidence for the first time that the tumour suppressive function of PTEN and Prdx1 are closely related. The mechanistic details of the Prdx1 and PTEN interaction and its regulation in terms of Prdx1 peroxidase activity and PTEN oxidation will be important in future work.

Compared with Prdxs, catalase peroxidase activity decomposes H2O2 with a >1000-fold higher KM (Chae et al, 1994a; Loewen et al, 2004) and therefore eliminates H2O2 with much higher efficiency. Thus, we cannot expect that Prdx1 scavenges H2O2 in considerable amounts, as shown in Figure 1A and Supplementary Figure S1A. Yet, Prdx1's peroxidase activity is important in preventing H-RasV12 (Figure 1B) or ErbB-2/neuT (Supplementary Figure S5A) induced transformation. Similar to the catalase knockout mice, all other currently published Prdxs knockout mice, show phenotypically no increased susceptibility to transformation (Ho et al, 1997). Therefore, Prdx1 may have a unique function amongst cellular peroxidases in tumour suppression, by affecting signalling directly through physical interaction with target enzymes. This was shown for c-Abl tyrosine kinase (Wen and VanEtten, 1997) and c-Jun terminal kinase (Kim et al, 2006). As described here for PTEN (Figure 3B), oxidative stress induced Prdx1 to dissociate, which in case of c-Abl (Neumann et al, 1998) and JNK (Kim et al, 2006) resulted in kinase activation. The Prdx1:PTEN heterodimer is most likely formed in a 1:1 molar ratio, as (1) the Prdx1 preservation of PTEN lipid phosphatase activity under oxidative stress is achieved by a 1:1 (mol:mol) ratio of Prdx1 and PTEN and could not be further increased by excess of Prdx1 (Figure 4B) and (2) Prdx1C51/172S, which is unable to form dimeric structures (Chae et al, 1994b), bound to PTEN and did not dissociate under H2O2 treatment (Figure 3C). Therefore, these data provide compelling evidence that Prdx1 may bind PTEN as a monomer.

We further propose that the Prdx1:PTEN interaction is essential for protecting PTEN from oxidation-induced inactivation (Figure 4C), as Prdx1 C- and N-terminal peptide replica, which were designed based on our mapping studies and computer modelling to block PTEN docking sites on Prdx1 (Figure 3F and G; Supplementary Figure S3E), resulted after H2O2 treatment in a 47% reduction of PTEN activity. This was comparable to PTEN activity after H2O2 treatment in the absence of Prdx1 (50%) (Figure 4C). The Prdx1 N terminus, which includes Cys51, may have an important function in regulating the H2O2-induced dissociation of the Prdx1:PTEN complex, as Cys51 when replaced by a serine enhances Prdx1 binding to PTEN (Figure 3D). X-ray crystallography of Prdxs suggested significant conformational changes, such as unwinding of the active site N-terminal helix, to form disulfides, as in reduced Prdxs, the sulfur atoms of the N- and C-terminal conserved cysteine residues are too far apart to react with each other (Schroder et al, 2000; Matsumura et al, 2008). Therefore, in the Prdx1:PTEN complex, increased H2O2 may oxidize Cys51 (Prdx1), thus, promoting unwinding of the PTEN-binding conformation of Prdx1 consequently inducing dissociation.

Over-oxidation of Prdx1 Cys51 occurs during catalysis and correlates with increasing amounts of Trx (Yang et al, 2002). The reduction of Cys51 by Trx inhibits the folding back of Cys51 into its pocket thereby exposing it to further oxidation by H2O2. The binding of PTEN to Prdx1 could have similar effects on Prdx1, thereby inhibiting sufficient reduction of Cys51 by Trx and decreasing Prdx1 peroxidase activity (Figure 4C and D). Alternatively, as it has been shown that Trx binds PTEN and inactivates its lipid phosphates activity, it could be that under our experimental conditions Trx is depleted from the Prdx1 scavenging system by binding PTEN (Meuillet et al, 2004). This seems unlikely though, as Trx was added in approximately 300-fold molar excess, which should provide a large Trx pool to reduce oxidized Prdx1 proteins.

We defined a novel interaction site of Prdx1 with PTEN in the C2 domain of PTEN (Figure 3F; Supplementary Figure S3D and E), which together with the N-terminal PTD, are both required for enzyme activity. Surface plasmon resonance analysis revealed that the C2 domain is essential for high-affinity membrane binding of PTEN (Das et al, 2003). Our data support this finding, as it has been shown recently through chemical proteomic strategy using cleavable lipid baits, that Prdx1 is a potential novel phosphoinositides-binding protein (Pasquali et al, 2007). Prdx1 may, therefore, stabilize PTEN binding to the plasma membrane, which in a recent analysis using single-molecule TIRF microscopy, lasts only for a few hundred milliseconds (Vazquez et al, 2006). Such short time of membrane binding could explain why we (Figure 2B, C, E and F; Supplementary Figure S2B and D) and others see only small amounts of PTEN oxidized after cell exposure to H2O2 or PDGF (Kwon et al, 2004). PDGF-induced H2O2 stems either from metabolized superoxide released by NADPH oxidases (NOXs) or the mitochondria and is considered a signalling molecule (Rhee, 2006). As discussed earlier (Leslie et al, 2003), such finding may be attributed to the proximity of NOXs complexes and PIP3. Over-expression of Nox1, in PDGF-stimulated NIH 3T3 cells rapidly increases PIP3 levels, in contrast to PI3K activity, suggesting that Nox1-induced superoxide enhances PIP3 levels by inactivating PTEN rather than activating PI3K. Expression of a peroxidase inactive Prdx2 leads to a rapid increase of PIP3 levels in growth factor stimulated cells, which is reversed by Prdx2WT (Kwon et al, 2004). In our hands however, Prdx2 did not bind PTEN, as we were unable to detect an interaction of Prdx2 and PTEN by immunoprecipitation (data not shown).

PTEN lipid phosphatase activity has an important function in tumour suppression (Tolkacheva and Chan, 2000; Koul et al, 2002) and Akt1 ablation protects (MMTV)-ErbB2/neu, MMTV-polyoma middle T and MMTV-v-H-Ras mice from breast cancer initiation (Skeen et al, 2006; Maroulakou et al, 2007). Here, we show that loss of Prdx1 increased basal, H2O2 and PDGF-induced Akt activity (Figure 2A, D and G) and Prdx1 peroxidase activity was essential in suppressing H-Rasv12 and ErbB-2/neuT-induced transformation (Figure 1B; Supplementary Figure S5A). Along those lines, Prdx1 ablation accelerated mammary tumorigenesis in MMTV-v-H-Ras mice and increased levels of pAktSer473 and PTEN oxidation in MECs from these mice (Figure 6A–C). Prdx1 tumour suppression seemed to occur largely via regulation of the PTEN and Akt. Knock down of Prdx1 in PTEN−/−MEFs did not significantly enhance H-Ras or ErbB-2-induced transformation (Figure 5C and D), whereas in transformed PTEN-containing MEFs, decrease in Prdx1 levels resulted in a gain of transformation: 1.5-fold for H-RasV12 and only 1.35-fold for ErbB-2/neuT, which we contributed to decreased Prdx1 knock down in PTEN−/−;shPrdx1;GFP−PTEN;ErbB−2/neuTMEFs compared with PTEN−/−;shPrdx1;GFP−PTEN;H−RasV12MEFs). Lack of PTEN also did not result in Prdx1 over-oxidation nor reduced expression (Figure 5B and E), suggesting that in PTEN-deficient cells, Prdx1 is acting as peroxidase.

However, we have to consider the regulation of other phosphatases or kinases by Prdx1 or H2O2 in Akt-driven tumorigenesis. SHIP-1 and 2 are known to dephosphorylate PIP3 to PtdIns (3,4)P2 (Vivanco and Sawyers, 2002). However, SHIP2 is not inactivated by oxidative stress (Leslie et al, 2003). PTEN−/−;shPrdx1MEFs showed decreased pAKTSer473 levels compared with PTEN−/−MEFs (Figure 5D), perhaps due to the activation of SHIP2 by small amounts of H2O2. However, H2O2 treatment of PTEN−/−;shPrdx1MEFs increased pAKTSer473 levels comparable to PTEN−/−MEFs, suggesting maybe a H2O2-induced inactivation of PP2A (Rao and Clayton, 2002), as loss of Prdx1 also inactivated PP2A activity (Supplementary Figure 2E). We also cannot exclude a regulation of PI3K by Prdx1, as H-Ras and ErbB-2/neuT transformed Prdx1−/−MEFs display a lesser sensitivity towards PI3K inhibitors (Table I; Supplementary Figure 5B), compared with Prdx1+/+MEFs.

In summary, oxidative stress and carcinogenesis are closely linked, however, molecular details, especially how oxidative stress promotes pro-oncogenic signalling pathways are scarce. Our studies shed some light on how H2O2 can promote oncogenic Ras and ErbB-2 signalling by providing evidence for the first time that the tumour suppressive function of PTEN and Prdx1 are closely related. The mechanistic details of the Prdx1 and PTEN interaction and its regulation in terms of Prdx1 peroxidase activity and PTEN oxidation will be important in future work.

Materials and methodsReagentsAll chemicals were purchased from Sigma Aldrich unless otherwise indicated. Ly294002, WM, antibodies against PTEN (138G6), Akt (11-E7), Akt-phosphoserine 473 (193H12), Akt-phosphothreonine 308 (244F9), phospho Akt substrate (110B7E) and HA-tag were purchased from Cell Signaling; Antibodies recognizing PTEN (A2B1), PTEN (N-19) and Actin (C-11) were purchased from Santa Cruz; antibodies against Prdx1 and Prdx1-SO2/-SO3 were purchased from Abcam; Ras antibody from Oncogene; ErbB-2 antibody from Biosource. HA-conjugated agarose beads were purchased from Roche. Amplex Red reagent: Molecular Probes. DMEM, FBS, Glutamax, NEAA, Pen/Strep, Sodium Pyruvate, DMEM without Phenol Red, PBS and PDGF stem from Invitrogen; GST-sepharose from Pharmacia. 6His–PTEN was expressed in Escherichia coli BL21 from the fusion vector pProEx HTb PTEN and purified by standard protocols.RetrovirusThe retrovirus was generated after conducting a transient transfection of retroviral constructs into a 293T/17 (ATCC) ecotropic-packaging cell line. Retroviral infection of MEFs was carried out for 4–6 h in the presence of 8 μg/ml polybrene. An antibiotic selection was carried out 24 h later until all uninfected cells were eliminated and stable polyclonal cell lines were generated.Generation of MEFsMEFs were generated from Prdx1+/− littermate matings in a C57BL/6J/129Sv background. Primary MEFs were harvested from E13.5 embryos as described earlier (Neumann et al, 2003). Primary Prdx1 and PTEN MEFs (passages 3–4) were immortalized by retroviral infection with pBabe-hygro DD p53 expressing dominant-negative p53 (Hahn et al, 2002) and selected with 150 μg/ml hygromycin for 14 days.Amplex red assayIn total, 35 000 MEFs were plated (n=6) in 24-well plates (Costar 24) coated with fibronectin. On the day of the assay, cells were washed with PBS and repleted with serum-free and phenol red-free DMEM medium. Amplex Red reagent and horse radish peroxidase (HRP) were prepared as recommended by the manufacturer (HRP 0.2 U/ml). MEFs were incubated with Amplex Red/HRP solution at 37°C before the first reading (T0) at 540 nm on a plate reader (Molecular Devices SpectraMax M5). Plates were kept in an incubator between readings. The resulting fluorescence (FLU) was compared with FLU from H2O2 standard curves and analysed.Soft agar assaysP53DD-immortalized MEFs (Prdx1−/−, Prdx1+/+) were infected with retrovirus containing either Prdx1WT or Prdx1C172/51S, both subcloned into pQCXI-puro (Clontech) and selected 10 days in puromycin 2 μg/ml. Some clones were further infected with a retrovirus containing either H-RasV12 (pBabe-puro H-RasV12) (Yu et al, 2001) or wild-type ErbB-2 (pBabe-puro ErbB-2) (Yu et al, 2001) and selected in 5 μg/ml puromycin for another 5 days. MEFs were then resuspended in DMEM containing 0.53% agarose (Difco) and plated onto a layer of 0.7% agarose-containing medium in 3.5-cm plates in duplicate (10 000 cells per plate). Colonies were counted after 14–21 days.Akt activity and PTEN oxidationMEFs (1.8 × 105) were plated on 10-cm plates and serum-starved (0.25% FBS) for 48 h. Plates were then incubated with H2O2 (±Ly294002 or WM) as indicated. Akt and PTEN protein analysis were done as described earlier (Leslie et al, 2003) with slight modifications. Briefly, lysis buffer (50 mM Tris pH 7.4; 1% Triton 100; 0.5 mM EDTA; 0.5 mM EGTA; 150 mM NaCl; 10% Glycerol; 50 mM NaF; 100 mM NaVO4; 40 mM β-Glycerophosphate) was degassed with Argon for 20 min before adding 1% TritonX-100; 2 mM PMSF; 5 mg/ml Aprotinin; N-ethylmaleimide 40 mM; and bovine catalase 100 μg/ml. Plates were transferred into an anaerobic/argon cabinet and were washed twice with cold degassed PBS. Cells were then scraped into lysis buffer and lysed on ice for 20 min before conducting quantitative analysis as described above. A measure of 65 μg of protein were analysed under non-reducing conditions with a 7.5% SDS–PAGE.Co-immunoprecipitationPTENWT (Addgene) was subcloned into a pQCXI vector (Clonetech) and N-terminally tagged with Myc epitope. Prdx1WT and Prdx1C51/172S, Prdx1C51S, Prdx1C172S (generated by site-specific mutagenesis) were subcloned into a pCGN vector and N-terminally tagged with an HA epitope. All constructs were transfected into 293T/17 (ATCC) cells using Fugene transfection reagent (Roche). Cells were harvested 72 h later for protein lysis under anaerobic conditions (as described above). A measure of 1000 μg protein was immunoprecipitated using anti-HA affinity matrix overnight at 4°C before washing four times with degassed lysis buffer before analysis on 4–12% gradient SDS–PAGE. Proteins were detected as described above.GST-pull downBL21(DE3) were transformed and cultured overnight with pGEX-4T-2 (Amersham) ligated earlier with Prdx1. At absorbance of 0.6–0.8 at 600 nm, GST–Prdx1 expression was induced by the addition of IPTG (200 μM). Bacterial pellets were re-suspended using fusion protein buffer containing 140 mM NaCl, 10 mM Na2HPO4, 1.8 mM KH2PO4, 2.7 mM KCl, 1 mM DTT, 50 μM PMSF, 5 mM benzamidine hydrochloride hydrate and 3 μM aprotinin. Lysate was then sonicated and proteins were solubilized by the addition of Triton X-100 (2%). After incubation of supernatant with glutathione affinity matrix, the matrix was washed consecutively with 500 mM NaCl, 50 mM NaCl and the GST-tagged fusion protein was eluted from the matrix by incubation with 30 mM reduced glutathione. The fusion protein was then desalted into 20 mM Tris (pH 7.4) and concentrated by centrifugation using a Centricon 30 kD MWC filter (Millipore, Bedford, MA). Incubations of equimolar concentrations (100 nM) of GST–Prdx1 plus His–PTEN or GST–EV (empty vector) plus His–PTEN were performed in a buffer containing 20 mM Hepes (pH 8.0), 150 mM NaCl, 2 mM MgCl2, 1 mM DTT, 50 μM PMSF, 5 mM benzamidine hydrochloride, 3 μM aprotinin and 1%Triton X-100. Mixtures were allowed to incubate overnight at 4°C before pre-washed glutathione agarose beads were added. The retained proteins were eluted from the resin by addition of 35 μl of 4 × loading buffer and analysed by 10% SDS gels.PTEN PlasmidsDifferent lengths of cDNA fragments encoding human PTEN amino acids from 1–403 were subcloned from pSG5L HA–PTENWT (Addgene#10750), pSG5L–HA–PTENC124S (#10745), pSG5L–HA–PTEN1−274 (#10740), pSG5L–HA–PTEN1−373 (#10766), pSG5L–HA–PTENΔ89–138 (#10924), pGEX2T–PTENΔ274–342 (#10739) (Ramaswamy et al, 1999) at restriction sites BamHI/EcoRI. A double-stranded oligonucleotide encoding the Myc-tag sequence was inserted 5′ of the PTEN to generate Myc–PTEN. Inserts tagged with Myc tag were cloned into pQCXIP to generate the corresponding pQCXIP–Myc–PTEN plasmids; pQCXIP–MYC–PTEN1−185 or PTEN1−351 or PTEN186−351 or PTENΔ244−251 were generated by PCR mutagenesis and confirmed by DNA sequencing. Prdx1 N- and C-terminal truncation mutants were generated by Bal31 nuclease digest, fill-in with T4-polymerase and subcloned into pCGN-HA.Recombinant Prdx1 activity assayPrdx1 peroxidase activity was studied by standard Trx/Trx reductase (TrxR)/NADPH-coupled spectrophotometric assay as described (Chae et al, 1994b). Briefly, NADPH oxidation coupled to the reduction of H2O2 was monitored at 37°C as a decrease in A340 using a SpectraMax M5 plate reading spectrophotometer. The reaction was initiated by addition of the H2O2 (500 μM) as a substrate into a reaction mixture containing 50 mM Hepes (pH 7.0), 0.15 U TrxR, 18 μM Trx, 25 μM NADPH and 680 nM Prdx1 (Sigma). The effect of PTEN on Prdx1 activity was studied by titration of reaction mixture with indicated increasing amount of PTEN (0.25–1.25 PTEN/Prdx1 mol/mol ratio). All experimental data were normalized to initial (before addition of substrate) absorbency and fitted with linear regression (Sigma Plot 10, Systat Software, Inc., San Jose, CA). Quality of fit was controlled by regression coefficient (R2⩾0.9).PTEN inositol phosphatase activityMyc–PTEN was expressed in 293T/17 cells in the presence or absence of HA–Prdx1. Cells were lysed as described above in phosphatase inhibitors free lysis buffer. PTEN proteins were immunoprecipitated from lysates with anti-PTEN (Santa Cruz) o/n as described above. IPs were then washed twice as described (Schwartzbauer and Robbins, 2001), followed by a final wash in PTEN enzyme reaction buffer (10 mM Hepes, 150 mM NaCl, 10 mM DTT pH 7.2). The phosphatase reactions were done following manufacturer's instructions (Echelon Biosciences). Briefly, the immunoprecipitated PTEN proteins were incubated with PTEN enzyme reaction buffer for an appropriate amount of time at 37°C in a 96-well plate coated with PI(3,4,5)P3. After removal of PTEN proteins from plates, PI(4,5)P2 was determined according to the manufacturer's instructions. Absorbance was measured at 450 nm in a microplate reader, and PI(4,5)P2 produced was calculated from a standard curve.Recombinant PTEN inositol phosphatase activityA measure of 20 pmol purified recombinant human His–PTEN was incubated with different amounts of Prdx1 (Sigma Aldrich) and 500 μM H2O2 in 20 mM Tris–Cl (pH 7.4) buffer at room temperature for 10 min. H2O2 was removed by Bio-Gel P-6 chromatography columns (Bio-Rad), following the manufacture's instructions. The protein mixture was then loaded onto PTEN activity assay plates as described above (Echelon Biosciences). Equal amount of PTEN was confirmed by western blot with the PTEN antibody. For Prdx1 peptide interference, 20 pmoles purified recombinant human His–PTEN was pre-incubated with 20-fold excess amount of peptides on ice in 20 mM Tris–Cl (pH 7.4) (for Peptide 1 or scramble peptide 1) or 0.2 mM Tris–Cl (pH 7.4) (peptide 2 or scramble peptide 2) for 1 h, the unbound peptides were removed by Microcon Ultracel YM-10 filters following the manufacture's instructions. The recovered PTEN–peptide mixtures were then incubated with 20 pmoles Prdx1 (Sigma Aldrich) and 500 μM H2O2 in 20 mM Tris–Cl (pH 7.4) buffer at room temperature for 10 min and further processed as described above. Peptide 1: MSSGNAKIGHPAPNFKATAVM; scrambled peptide 1: KTHPMPSIAAKFNAGSNAMVG; Peptide 2: PAGWKPGSDTIKPDVQKSKEYFSKQK; scrambled peptide 2: IKKSWKFGPGQPTSAPKQSKEVKYDD; all peptides had purity >75% and were purchased from Genscript, NJ.PTEN−/−MEFs experimentsAn optimal sequence for knock down of Prdx1 by sh (5′-GCTCAGGATTATGGAGTCTTA-3′) was identified from the TRC consortium (http://www.broad.mit.edu) and was subcloned into pMKO1 retroviral vector by AgeI/EcoRI digest. PTEN−/−MEFs were infected with retroviral supernatant containing either pMKO1–Prdx1shRNA or pMKO1 EV, and selected in puromycin 2 μg/ml for 1 week. GFP–PTEN was subcloned from pEGFP–C2–PTEN (Leslie et al, 2000) into pQCXIP at restriction sites of AgeI-EcoRI. PTEN−/−;pMKO1MEFs or PTEN−/−;shPrdx1MEFs were then transduced with GFP–PTEN retrovirus and selected in 6 μg/ml puromycin. Infection and selection with retrovirus expressing either H-RasV12 or ErbB-2/neuT was then challenged by 10 μg/ml puromycin.MMTV-v-H-Ras mice lacking Prdx1MMTV-v-H-Ras mice (Sinn et al, 1987) were purchased from Charles River. Prdx1+/−/MMTV-v-H-Ras mice were intercrossed to generate Prdx1−/−/MMTV-v-H-Ras and Prdx1+/+/MMTV-v-H-Ras mice. Mice were monitored twice a week for tumour appearance. Mouse mammary glands were isolated from age-matched 16-week-old female MMTV-v-H-ras/Prdx1−/− and MMTV-v-H-ras/Prdx1+/+ (n=3) mice following earlier described procedures (Fujiwara et al, 2005). Proteins were processed as described above.

Materials and methods

ReagentsAll chemicals were purchased from Sigma Aldrich unless otherwise indicated. Ly294002, WM, antibodies against PTEN (138G6), Akt (11-E7), Akt-phosphoserine 473 (193H12), Akt-phosphothreonine 308 (244F9), phospho Akt substrate (110B7E) and HA-tag were purchased from Cell Signaling; Antibodies recognizing PTEN (A2B1), PTEN (N-19) and Actin (C-11) were purchased from Santa Cruz; antibodies against Prdx1 and Prdx1-SO2/-SO3 were purchased from Abcam; Ras antibody from Oncogene; ErbB-2 antibody from Biosource. HA-conjugated agarose beads were purchased from Roche. Amplex Red reagent: Molecular Probes. DMEM, FBS, Glutamax, NEAA, Pen/Strep, Sodium Pyruvate, DMEM without Phenol Red, PBS and PDGF stem from Invitrogen; GST-sepharose from Pharmacia. 6His–PTEN was expressed in Escherichia coli BL21 from the fusion vector pProEx HTb PTEN and purified by standard protocols.

All chemicals were purchased from Sigma Aldrich unless otherwise indicated. Ly294002, WM, antibodies against PTEN (138G6), Akt (11-E7), Akt-phosphoserine 473 (193H12), Akt-phosphothreonine 308 (244F9), phospho Akt substrate (110B7E) and HA-tag were purchased from Cell Signaling; Antibodies recognizing PTEN (A2B1), PTEN (N-19) and Actin (C-11) were purchased from Santa Cruz; antibodies against Prdx1 and Prdx1-SO2/-SO3 were purchased from Abcam; Ras antibody from Oncogene; ErbB-2 antibody from Biosource. HA-conjugated agarose beads were purchased from Roche. Amplex Red reagent: Molecular Probes. DMEM, FBS, Glutamax, NEAA, Pen/Strep, Sodium Pyruvate, DMEM without Phenol Red, PBS and PDGF stem from Invitrogen; GST-sepharose from Pharmacia. 6His–PTEN was expressed in Escherichia coli BL21 from the fusion vector pProEx HTb PTEN and purified by standard protocols.

RetrovirusThe retrovirus was generated after conducting a transient transfection of retroviral constructs into a 293T/17 (ATCC) ecotropic-packaging cell line. Retroviral infection of MEFs was carried out for 4–6 h in the presence of 8 μg/ml polybrene. An antibiotic selection was carried out 24 h later until all uninfected cells were eliminated and stable polyclonal cell lines were generated.

The retrovirus was generated after conducting a transient transfection of retroviral constructs into a 293T/17 (ATCC) ecotropic-packaging cell line. Retroviral infection of MEFs was carried out for 4–6 h in the presence of 8 μg/ml polybrene. An antibiotic selection was carried out 24 h later until all uninfected cells were eliminated and stable polyclonal cell lines were generated.

Generation of MEFsMEFs were generated from Prdx1+/− littermate matings in a C57BL/6J/129Sv background. Primary MEFs were harvested from E13.5 embryos as described earlier (Neumann et al, 2003). Primary Prdx1 and PTEN MEFs (passages 3–4) were immortalized by retroviral infection with pBabe-hygro DD p53 expressing dominant-negative p53 (Hahn et al, 2002) and selected with 150 μg/ml hygromycin for 14 days.

Generation of MEFs

MEFs were generated from Prdx1+/− littermate matings in a C57BL/6J/129Sv background. Primary MEFs were harvested from E13.5 embryos as described earlier (Neumann et al, 2003). Primary Prdx1 and PTEN MEFs (passages 3–4) were immortalized by retroviral infection with pBabe-hygro DD p53 expressing dominant-negative p53 (Hahn et al, 2002) and selected with 150 μg/ml hygromycin for 14 days.

Amplex red assayIn total, 35 000 MEFs were plated (n=6) in 24-well plates (Costar 24) coated with fibronectin. On the day of the assay, cells were washed with PBS and repleted with serum-free and phenol red-free DMEM medium. Amplex Red reagent and horse radish peroxidase (HRP) were prepared as recommended by the manufacturer (HRP 0.2 U/ml). MEFs were incubated with Amplex Red/HRP solution at 37°C before the first reading (T0) at 540 nm on a plate reader (Molecular Devices SpectraMax M5). Plates were kept in an incubator between readings. The resulting fluorescence (FLU) was compared with FLU from H2O2 standard curves and analysed.

Amplex red assay

In total, 35 000 MEFs were plated (n=6) in 24-well plates (Costar 24) coated with fibronectin. On the day of the assay, cells were washed with PBS and repleted with serum-free and phenol red-free DMEM medium. Amplex Red reagent and horse radish peroxidase (HRP) were prepared as recommended by the manufacturer (HRP 0.2 U/ml). MEFs were incubated with Amplex Red/HRP solution at 37°C before the first reading (T0) at 540 nm on a plate reader (Molecular Devices SpectraMax M5). Plates were kept in an incubator between readings. The resulting fluorescence (FLU) was compared with FLU from H2O2 standard curves and analysed.

Soft agar assaysP53DD-immortalized MEFs (Prdx1−/−, Prdx1+/+) were infected with retrovirus containing either Prdx1WT or Prdx1C172/51S, both subcloned into pQCXI-puro (Clontech) and selected 10 days in puromycin 2 μg/ml. Some clones were further infected with a retrovirus containing either H-RasV12 (pBabe-puro H-RasV12) (Yu et al, 2001) or wild-type ErbB-2 (pBabe-puro ErbB-2) (Yu et al, 2001) and selected in 5 μg/ml puromycin for another 5 days. MEFs were then resuspended in DMEM containing 0.53% agarose (Difco) and plated onto a layer of 0.7% agarose-containing medium in 3.5-cm plates in duplicate (10 000 cells per plate). Colonies were counted after 14–21 days.

Soft agar assays

P53DD-immortalized MEFs (Prdx1−/−, Prdx1+/+) were infected with retrovirus containing either Prdx1WT or Prdx1C172/51S, both subcloned into pQCXI-puro (Clontech) and selected 10 days in puromycin 2 μg/ml. Some clones were further infected with a retrovirus containing either H-RasV12 (pBabe-puro H-RasV12) (Yu et al, 2001) or wild-type ErbB-2 (pBabe-puro ErbB-2) (Yu et al, 2001) and selected in 5 μg/ml puromycin for another 5 days. MEFs were then resuspended in DMEM containing 0.53% agarose (Difco) and plated onto a layer of 0.7% agarose-containing medium in 3.5-cm plates in duplicate (10 000 cells per plate). Colonies were counted after 14–21 days.

Akt activity and PTEN oxidationMEFs (1.8 × 105) were plated on 10-cm plates and serum-starved (0.25% FBS) for 48 h. Plates were then incubated with H2O2 (±Ly294002 or WM) as indicated. Akt and PTEN protein analysis were done as described earlier (Leslie et al, 2003) with slight modifications. Briefly, lysis buffer (50 mM Tris pH 7.4; 1% Triton 100; 0.5 mM EDTA; 0.5 mM EGTA; 150 mM NaCl; 10% Glycerol; 50 mM NaF; 100 mM NaVO4; 40 mM β-Glycerophosphate) was degassed with Argon for 20 min before adding 1% TritonX-100; 2 mM PMSF; 5 mg/ml Aprotinin; N-ethylmaleimide 40 mM; and bovine catalase 100 μg/ml. Plates were transferred into an anaerobic/argon cabinet and were washed twice with cold degassed PBS. Cells were then scraped into lysis buffer and lysed on ice for 20 min before conducting quantitative analysis as described above. A measure of 65 μg of protein were analysed under non-reducing conditions with a 7.5% SDS–PAGE.

Akt activity and PTEN oxidation

MEFs (1.8 × 105) were plated on 10-cm plates and serum-starved (0.25% FBS) for 48 h. Plates were then incubated with H2O2 (±Ly294002 or WM) as indicated. Akt and PTEN protein analysis were done as described earlier (Leslie et al, 2003) with slight modifications. Briefly, lysis buffer (50 mM Tris pH 7.4; 1% Triton 100; 0.5 mM EDTA; 0.5 mM EGTA; 150 mM NaCl; 10% Glycerol; 50 mM NaF; 100 mM NaVO4; 40 mM β-Glycerophosphate) was degassed with Argon for 20 min before adding 1% TritonX-100; 2 mM PMSF; 5 mg/ml Aprotinin; N-ethylmaleimide 40 mM; and bovine catalase 100 μg/ml. Plates were transferred into an anaerobic/argon cabinet and were washed twice with cold degassed PBS. Cells were then scraped into lysis buffer and lysed on ice for 20 min before conducting quantitative analysis as described above. A measure of 65 μg of protein were analysed under non-reducing conditions with a 7.5% SDS–PAGE.

Co-immunoprecipitationPTENWT (Addgene) was subcloned into a pQCXI vector (Clonetech) and N-terminally tagged with Myc epitope. Prdx1WT and Prdx1C51/172S, Prdx1C51S, Prdx1C172S (generated by site-specific mutagenesis) were subcloned into a pCGN vector and N-terminally tagged with an HA epitope. All constructs were transfected into 293T/17 (ATCC) cells using Fugene transfection reagent (Roche). Cells were harvested 72 h later for protein lysis under anaerobic conditions (as described above). A measure of 1000 μg protein was immunoprecipitated using anti-HA affinity matrix overnight at 4°C before washing four times with degassed lysis buffer before analysis on 4–12% gradient SDS–PAGE. Proteins were detected as described above.

Co-immunoprecipitation

PTENWT (Addgene) was subcloned into a pQCXI vector (Clonetech) and N-terminally tagged with Myc epitope. Prdx1WT and Prdx1C51/172S, Prdx1C51S, Prdx1C172S (generated by site-specific mutagenesis) were subcloned into a pCGN vector and N-terminally tagged with an HA epitope. All constructs were transfected into 293T/17 (ATCC) cells using Fugene transfection reagent (Roche). Cells were harvested 72 h later for protein lysis under anaerobic conditions (as described above). A measure of 1000 μg protein was immunoprecipitated using anti-HA affinity matrix overnight at 4°C before washing four times with degassed lysis buffer before analysis on 4–12% gradient SDS–PAGE. Proteins were detected as described above.

GST-pull downBL21(DE3) were transformed and cultured overnight with pGEX-4T-2 (Amersham) ligated earlier with Prdx1. At absorbance of 0.6–0.8 at 600 nm, GST–Prdx1 expression was induced by the addition of IPTG (200 μM). Bacterial pellets were re-suspended using fusion protein buffer containing 140 mM NaCl, 10 mM Na2HPO4, 1.8 mM KH2PO4, 2.7 mM KCl, 1 mM DTT, 50 μM PMSF, 5 mM benzamidine hydrochloride hydrate and 3 μM aprotinin. Lysate was then sonicated and proteins were solubilized by the addition of Triton X-100 (2%). After incubation of supernatant with glutathione affinity matrix, the matrix was washed consecutively with 500 mM NaCl, 50 mM NaCl and the GST-tagged fusion protein was eluted from the matrix by incubation with 30 mM reduced glutathione. The fusion protein was then desalted into 20 mM Tris (pH 7.4) and concentrated by centrifugation using a Centricon 30 kD MWC filter (Millipore, Bedford, MA). Incubations of equimolar concentrations (100 nM) of GST–Prdx1 plus His–PTEN or GST–EV (empty vector) plus His–PTEN were performed in a buffer containing 20 mM Hepes (pH 8.0), 150 mM NaCl, 2 mM MgCl2, 1 mM DTT, 50 μM PMSF, 5 mM benzamidine hydrochloride, 3 μM aprotinin and 1%Triton X-100. Mixtures were allowed to incubate overnight at 4°C before pre-washed glutathione agarose beads were added. The retained proteins were eluted from the resin by addition of 35 μl of 4 × loading buffer and analysed by 10% SDS gels.

GST-pull down

BL21(DE3) were transformed and cultured overnight with pGEX-4T-2 (Amersham) ligated earlier with Prdx1. At absorbance of 0.6–0.8 at 600 nm, GST–Prdx1 expression was induced by the addition of IPTG (200 μM). Bacterial pellets were re-suspended using fusion protein buffer containing 140 mM NaCl, 10 mM Na2HPO4, 1.8 mM KH2PO4, 2.7 mM KCl, 1 mM DTT, 50 μM PMSF, 5 mM benzamidine hydrochloride hydrate and 3 μM aprotinin. Lysate was then sonicated and proteins were solubilized by the addition of Triton X-100 (2%). After incubation of supernatant with glutathione affinity matrix, the matrix was washed consecutively with 500 mM NaCl, 50 mM NaCl and the GST-tagged fusion protein was eluted from the matrix by incubation with 30 mM reduced glutathione. The fusion protein was then desalted into 20 mM Tris (pH 7.4) and concentrated by centrifugation using a Centricon 30 kD MWC filter (Millipore, Bedford, MA). Incubations of equimolar concentrations (100 nM) of GST–Prdx1 plus His–PTEN or GST–EV (empty vector) plus His–PTEN were performed in a buffer containing 20 mM Hepes (pH 8.0), 150 mM NaCl, 2 mM MgCl2, 1 mM DTT, 50 μM PMSF, 5 mM benzamidine hydrochloride, 3 μM aprotinin and 1%Triton X-100. Mixtures were allowed to incubate overnight at 4°C before pre-washed glutathione agarose beads were added. The retained proteins were eluted from the resin by addition of 35 μl of 4 × loading buffer and analysed by 10% SDS gels.

PTEN PlasmidsDifferent lengths of cDNA fragments encoding human PTEN amino acids from 1–403 were subcloned from pSG5L HA–PTENWT (Addgene#10750), pSG5L–HA–PTENC124S (#10745), pSG5L–HA–PTEN1−274 (#10740), pSG5L–HA–PTEN1−373 (#10766), pSG5L–HA–PTENΔ89–138 (#10924), pGEX2T–PTENΔ274–342 (#10739) (Ramaswamy et al, 1999) at restriction sites BamHI/EcoRI. A double-stranded oligonucleotide encoding the Myc-tag sequence was inserted 5′ of the PTEN to generate Myc–PTEN. Inserts tagged with Myc tag were cloned into pQCXIP to generate the corresponding pQCXIP–Myc–PTEN plasmids; pQCXIP–MYC–PTEN1−185 or PTEN1−351 or PTEN186−351 or PTENΔ244−251 were generated by PCR mutagenesis and confirmed by DNA sequencing. Prdx1 N- and C-terminal truncation mutants were generated by Bal31 nuclease digest, fill-in with T4-polymerase and subcloned into pCGN-HA.

PTEN Plasmids

Different lengths of cDNA fragments encoding human PTEN amino acids from 1–403 were subcloned from pSG5L HA–PTENWT (Addgene#10750), pSG5L–HA–PTENC124S (#10745), pSG5L–HA–PTEN1−274 (#10740), pSG5L–HA–PTEN1−373 (#10766), pSG5L–HA–PTENΔ89–138 (#10924), pGEX2T–PTENΔ274–342 (#10739) (Ramaswamy et al, 1999) at restriction sites BamHI/EcoRI. A double-stranded oligonucleotide encoding the Myc-tag sequence was inserted 5′ of the PTEN to generate Myc–PTEN. Inserts tagged with Myc tag were cloned into pQCXIP to generate the corresponding pQCXIP–Myc–PTEN plasmids; pQCXIP–MYC–PTEN1−185 or PTEN1−351 or PTEN186−351 or PTENΔ244−251 were generated by PCR mutagenesis and confirmed by DNA sequencing. Prdx1 N- and C-terminal truncation mutants were generated by Bal31 nuclease digest, fill-in with T4-polymerase and subcloned into pCGN-HA.

Recombinant Prdx1 activity assayPrdx1 peroxidase activity was studied by standard Trx/Trx reductase (TrxR)/NADPH-coupled spectrophotometric assay as described (Chae et al, 1994b). Briefly, NADPH oxidation coupled to the reduction of H2O2 was monitored at 37°C as a decrease in A340 using a SpectraMax M5 plate reading spectrophotometer. The reaction was initiated by addition of the H2O2 (500 μM) as a substrate into a reaction mixture containing 50 mM Hepes (pH 7.0), 0.15 U TrxR, 18 μM Trx, 25 μM NADPH and 680 nM Prdx1 (Sigma). The effect of PTEN on Prdx1 activity was studied by titration of reaction mixture with indicated increasing amount of PTEN (0.25–1.25 PTEN/Prdx1 mol/mol ratio). All experimental data were normalized to initial (before addition of substrate) absorbency and fitted with linear regression (Sigma Plot 10, Systat Software, Inc., San Jose, CA). Quality of fit was controlled by regression coefficient (R2⩾0.9).

Recombinant Prdx1 activity assay

Prdx1 peroxidase activity was studied by standard Trx/Trx reductase (TrxR)/NADPH-coupled spectrophotometric assay as described (Chae et al, 1994b). Briefly, NADPH oxidation coupled to the reduction of H2O2 was monitored at 37°C as a decrease in A340 using a SpectraMax M5 plate reading spectrophotometer. The reaction was initiated by addition of the H2O2 (500 μM) as a substrate into a reaction mixture containing 50 mM Hepes (pH 7.0), 0.15 U TrxR, 18 μM Trx, 25 μM NADPH and 680 nM Prdx1 (Sigma). The effect of PTEN on Prdx1 activity was studied by titration of reaction mixture with indicated increasing amount of PTEN (0.25–1.25 PTEN/Prdx1 mol/mol ratio). All experimental data were normalized to initial (before addition of substrate) absorbency and fitted with linear regression (Sigma Plot 10, Systat Software, Inc., San Jose, CA). Quality of fit was controlled by regression coefficient (R2⩾0.9).

PTEN inositol phosphatase activityMyc–PTEN was expressed in 293T/17 cells in the presence or absence of HA–Prdx1. Cells were lysed as described above in phosphatase inhibitors free lysis buffer. PTEN proteins were immunoprecipitated from lysates with anti-PTEN (Santa Cruz) o/n as described above. IPs were then washed twice as described (Schwartzbauer and Robbins, 2001), followed by a final wash in PTEN enzyme reaction buffer (10 mM Hepes, 150 mM NaCl, 10 mM DTT pH 7.2). The phosphatase reactions were done following manufacturer's instructions (Echelon Biosciences). Briefly, the immunoprecipitated PTEN proteins were incubated with PTEN enzyme reaction buffer for an appropriate amount of time at 37°C in a 96-well plate coated with PI(3,4,5)P3. After removal of PTEN proteins from plates, PI(4,5)P2 was determined according to the manufacturer's instructions. Absorbance was measured at 450 nm in a microplate reader, and PI(4,5)P2 produced was calculated from a standard curve.

PTEN inositol phosphatase activity

Myc–PTEN was expressed in 293T/17 cells in the presence or absence of HA–Prdx1. Cells were lysed as described above in phosphatase inhibitors free lysis buffer. PTEN proteins were immunoprecipitated from lysates with anti-PTEN (Santa Cruz) o/n as described above. IPs were then washed twice as described (Schwartzbauer and Robbins, 2001), followed by a final wash in PTEN enzyme reaction buffer (10 mM Hepes, 150 mM NaCl, 10 mM DTT pH 7.2). The phosphatase reactions were done following manufacturer's instructions (Echelon Biosciences). Briefly, the immunoprecipitated PTEN proteins were incubated with PTEN enzyme reaction buffer for an appropriate amount of time at 37°C in a 96-well plate coated with PI(3,4,5)P3. After removal of PTEN proteins from plates, PI(4,5)P2 was determined according to the manufacturer's instructions. Absorbance was measured at 450 nm in a microplate reader, and PI(4,5)P2 produced was calculated from a standard curve.

Recombinant PTEN inositol phosphatase activityA measure of 20 pmol purified recombinant human His–PTEN was incubated with different amounts of Prdx1 (Sigma Aldrich) and 500 μM H2O2 in 20 mM Tris–Cl (pH 7.4) buffer at room temperature for 10 min. H2O2 was removed by Bio-Gel P-6 chromatography columns (Bio-Rad), following the manufacture's instructions. The protein mixture was then loaded onto PTEN activity assay plates as described above (Echelon Biosciences). Equal amount of PTEN was confirmed by western blot with the PTEN antibody. For Prdx1 peptide interference, 20 pmoles purified recombinant human His–PTEN was pre-incubated with 20-fold excess amount of peptides on ice in 20 mM Tris–Cl (pH 7.4) (for Peptide 1 or scramble peptide 1) or 0.2 mM Tris–Cl (pH 7.4) (peptide 2 or scramble peptide 2) for 1 h, the unbound peptides were removed by Microcon Ultracel YM-10 filters following the manufacture's instructions. The recovered PTEN–peptide mixtures were then incubated with 20 pmoles Prdx1 (Sigma Aldrich) and 500 μM H2O2 in 20 mM Tris–Cl (pH 7.4) buffer at room temperature for 10 min and further processed as described above. Peptide 1: MSSGNAKIGHPAPNFKATAVM; scrambled peptide 1: KTHPMPSIAAKFNAGSNAMVG; Peptide 2: PAGWKPGSDTIKPDVQKSKEYFSKQK; scrambled peptide 2: IKKSWKFGPGQPTSAPKQSKEVKYDD; all peptides had purity >75% and were purchased from Genscript, NJ.

Recombinant PTEN inositol phosphatase activity

A measure of 20 pmol purified recombinant human His–PTEN was incubated with different amounts of Prdx1 (Sigma Aldrich) and 500 μM H2O2 in 20 mM Tris–Cl (pH 7.4) buffer at room temperature for 10 min. H2O2 was removed by Bio-Gel P-6 chromatography columns (Bio-Rad), following the manufacture's instructions. The protein mixture was then loaded onto PTEN activity assay plates as described above (Echelon Biosciences). Equal amount of PTEN was confirmed by western blot with the PTEN antibody. For Prdx1 peptide interference, 20 pmoles purified recombinant human His–PTEN was pre-incubated with 20-fold excess amount of peptides on ice in 20 mM Tris–Cl (pH 7.4) (for Peptide 1 or scramble peptide 1) or 0.2 mM Tris–Cl (pH 7.4) (peptide 2 or scramble peptide 2) for 1 h, the unbound peptides were removed by Microcon Ultracel YM-10 filters following the manufacture's instructions. The recovered PTEN–peptide mixtures were then incubated with 20 pmoles Prdx1 (Sigma Aldrich) and 500 μM H2O2 in 20 mM Tris–Cl (pH 7.4) buffer at room temperature for 10 min and further processed as described above. Peptide 1: MSSGNAKIGHPAPNFKATAVM; scrambled peptide 1: KTHPMPSIAAKFNAGSNAMVG; Peptide 2: PAGWKPGSDTIKPDVQKSKEYFSKQK; scrambled peptide 2: IKKSWKFGPGQPTSAPKQSKEVKYDD; all peptides had purity >75% and were purchased from Genscript, NJ.

PTEN−/−MEFs experimentsAn optimal sequence for knock down of Prdx1 by sh (5′-GCTCAGGATTATGGAGTCTTA-3′) was identified from the TRC consortium (http://www.broad.mit.edu) and was subcloned into pMKO1 retroviral vector by AgeI/EcoRI digest. PTEN−/−MEFs were infected with retroviral supernatant containing either pMKO1–Prdx1shRNA or pMKO1 EV, and selected in puromycin 2 μg/ml for 1 week. GFP–PTEN was subcloned from pEGFP–C2–PTEN (Leslie et al, 2000) into pQCXIP at restriction sites of AgeI-EcoRI. PTEN−/−;pMKO1MEFs or PTEN−/−;shPrdx1MEFs were then transduced with GFP–PTEN retrovirus and selected in 6 μg/ml puromycin. Infection and selection with retrovirus expressing either H-RasV12 or ErbB-2/neuT was then challenged by 10 μg/ml puromycin.

PTEN−/−MEFs experiments

An optimal sequence for knock down of Prdx1 by sh (5′-GCTCAGGATTATGGAGTCTTA-3′) was identified from the TRC consortium (http://www.broad.mit.edu) and was subcloned into pMKO1 retroviral vector by AgeI/EcoRI digest. PTEN−/−MEFs were infected with retroviral supernatant containing either pMKO1–Prdx1shRNA or pMKO1 EV, and selected in puromycin 2 μg/ml for 1 week. GFP–PTEN was subcloned from pEGFP–C2–PTEN (Leslie et al, 2000) into pQCXIP at restriction sites of AgeI-EcoRI. PTEN−/−;pMKO1MEFs or PTEN−/−;shPrdx1MEFs were then transduced with GFP–PTEN retrovirus and selected in 6 μg/ml puromycin. Infection and selection with retrovirus expressing either H-RasV12 or ErbB-2/neuT was then challenged by 10 μg/ml puromycin.

MMTV-v-H-Ras mice lacking Prdx1MMTV-v-H-Ras mice (Sinn et al, 1987) were purchased from Charles River. Prdx1+/−/MMTV-v-H-Ras mice were intercrossed to generate Prdx1−/−/MMTV-v-H-Ras and Prdx1+/+/MMTV-v-H-Ras mice. Mice were monitored twice a week for tumour appearance. Mouse mammary glands were isolated from age-matched 16-week-old female MMTV-v-H-ras/Prdx1−/− and MMTV-v-H-ras/Prdx1+/+ (n=3) mice following earlier described procedures (Fujiwara et al, 2005). Proteins were processed as described above.

MMTV-v-H-Ras mice lacking Prdx1

MMTV-v-H-Ras mice (Sinn et al, 1987) were purchased from Charles River. Prdx1+/−/MMTV-v-H-Ras mice were intercrossed to generate Prdx1−/−/MMTV-v-H-Ras and Prdx1+/+/MMTV-v-H-Ras mice. Mice were monitored twice a week for tumour appearance. Mouse mammary glands were isolated from age-matched 16-week-old female MMTV-v-H-ras/Prdx1−/− and MMTV-v-H-ras/Prdx1+/+ (n=3) mice following earlier described procedures (Fujiwara et al, 2005). Proteins were processed as described above.

Supplementary MaterialSupplementary Figures S1–S5

Supplementary Material

Supplementary Figures S1–S5
