---
pmid: '19483087'
title: Identification and characterization of two novel isoforms of Pirh2 ubiquitin
  ligase that negatively regulate p53 independent of RING finger domains.
authors:
- Corcoran CA
- Montalbano J
- Sun H
- He Q
- Huang Y
- Sheikh MS
journal: J Biol Chem
year: '2009'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC2755920
doi: 10.1074/jbc.M109.024232
---

# Identification and characterization of two novel isoforms of Pirh2 ubiquitin ligase that negatively regulate p53 independent of RING finger domains.
**Authors:** Corcoran CA, Montalbano J, Sun H, He Q, Huang Y, Sheikh MS
**Journal:** J Biol Chem (2009)
**DOI:** [10.1074/jbc.M109.024232](https://doi.org/10.1074/jbc.M109.024232)
**PMC:** [PMC2755920](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2755920/)

## Abstract

1. J Biol Chem. 2009 Aug 14;284(33):21955-21970. doi: 10.1074/jbc.M109.024232.
Epub  2009 May 29.

Identification and characterization of two novel isoforms of Pirh2 ubiquitin 
ligase that negatively regulate p53 independent of RING finger domains.

Corcoran CA(1), Montalbano J(1), Sun H(1), He Q(1), Huang Y(1), Sheikh MS(1).

Author information:
(1)Department of Pharmacology, State University of New York, Upstate Medical 
University, Syracuse, New York 13210.

Pirh2 is a newly identified E3 ubiquitin ligase known to inhibit tumor 
suppressor p53 function via ubiquitination and proteasomal degradation. We have 
identified two novel Pirh2 splice variants that encode different Pirh2 isoforms 
and named these Pirh2B and Pirh2C. Accordingly, the full-length protein is now 
classified as isoform Pirh2A. The central region of Pirh2 harbors a RING finger 
domain that is critical for its ubiquitin ligase function. The Pirh2B isoform 
lacks amino acids 171-179, whereas Pirh2C is missing C-terminal amino acids 
180-261, which for each isoform results in a RING domain deletion and the 
abrogation of ubiquitin ligase activity. Our findings further indicate that the 
Pirh2B isoform but not the Pirh2C isoform is capable of binding to Pirh2A, 
suggesting that the C-terminal region absent in Pirh2C is critical for 
Pirh2-Pirh2 interactions. Similar to Pirh2A, both Pirh2B and Pirh2C interact 
with p53; however, interactions between p53 and Pirh2B appear stronger than 
those between p53 and Pirh2C. Interestingly, although both Pirh2B and Pirh2C are 
not able to promote in vitro p53 ubiquitination, both are capable of negatively 
regulating p53 protein stability and promoting the intracellular ubiquitination 
of p53. Furthermore, like Pirh2A, both isoforms are able to inhibit p53 
transcriptional activity. We have also for the first time demonstrated that 
Pirh2A as well as the novel isoforms also interact directly with MDM2 within a 
region encompassing MDM2 acidic and zinc finger domains. It is therefore 
possible that Pirh2A and the novel Pirh2 isoforms identified in this study may 
also modulate p53 function by engaging MDM2.

DOI: 10.1074/jbc.M109.024232
PMCID: PMC2755920
PMID: 19483087 [Indexed for MEDLINE]

## Full Text

Abstract

Pirh2 is a newly identified E3 ubiquitin ligase known to inhibit tumor suppressor p53 function via ubiquitination and proteasomal degradation. We have identified two novel Pirh2 splice variants that encode different Pirh2 isoforms and named these Pirh2B and Pirh2C. Accordingly, the full-length protein is now classified as isoform Pirh2A. The central region of Pirh2 harbors a RING finger domain that is critical for its ubiquitin ligase function. The Pirh2B isoform lacks amino acids 171–179, whereas Pirh2C is missing C-terminal amino acids 180–261, which for each isoform results in a RING domain deletion and the abrogation of ubiquitin ligase activity. Our findings further indicate that the Pirh2B isoform but not the Pirh2C isoform is capable of binding to Pirh2A, suggesting that the C-terminal region absent in Pirh2C is critical for Pirh2-Pirh2 interactions. Similar to Pirh2A, both Pirh2B and Pirh2C interact with p53; however, interactions between p53 and Pirh2B appear stronger than those between p53 and Pirh2C. Interestingly, although both Pirh2B and Pirh2C are not able to promote in vitro p53 ubiquitination, both are capable of negatively regulating p53 protein stability and promoting the intracellular ubiquitination of p53. Furthermore, like Pirh2A, both isoforms are able to inhibit p53 transcriptional activity. We have also for the first time demonstrated that Pirh2A as well as the novel isoforms also interact directly with MDM2 within a region encompassing MDM2 acidic and zinc finger domains. It is therefore possible that Pirh2A and the novel Pirh2 isoforms identified in this study may also modulate p53 function by engaging MDM2.

DISCUSSION

A large body of evidence indicates that the deregulation of p53 by proteins and pathways implicated in p53 ubiquitination appears be a principal means by which p53 is inactivated in human cancers. Pirh2 is one candidate protein that is known to negatively regulate p53 through ubiquitination and is overexpressed in primary tumors ( 23 , 24 ). Despite initial information implicating it in negative regulation of p53, much remains to be elucidated as to how Pirh2 functions to inhibit p53 and how it may contribute to tumorigenesis.

To this end, we report here the identification of two novel Pirh2 isoforms, Pirh2B and Pirh2C, that each lack a portion of their central RING domain as a result of differential mRNA splicing. Expression of Pirh2B and Pirh2C isoforms is not restricted to a single cell or tissue type, since our results indicate that the Pirh2B and Pirh2C isoforms are expressed in various cells representing multiple tissues. Furthermore, ESTs corresponding to these novel isoforms are present in several cDNA libraries generated from various tissues. Moreover, we have found that Pirh2B expression appears to be conserved across species, since sequences highly homologous and predicted to encode Pirh2B are expressed in tissues derived from the Rhesus monkey, chimpanzee, and dog. Pirh2C may also be similarly conserved; however, we did not find homologous sequences encoding Pirh2C in other species. Because only two ESTs representing the human Pirh2C homologue were identified ( BQ429858 and BX459245 ), it may be possible that sequences representing Pirh2C from other species have not yet been reported in the data bases.

The splicing mechanisms generating these isoforms remain unclear; however, the alternate splicing of an E3 ligase targeting p53 is not without precedence. For example, MDM2 variants are known to arise from the alternate usage of either a 5′-untranslated region or a p53-responsive region within MDM2 intron 1 ( 44 – 47 ). Pirh2B is generated by the alternative splicing of exon 7. Although the consensus p53 response element (p53RE) for Pirh2 has been mapped to Pirh2 intron 3 ( 11 ), very little is known regarding regulation of Pirh2 gene expression. Further studies are therefore needed to elucidate the exact mechanisms controlling the splicing events responsible for the generation of this isoform. The splicing of Pirh2C, unlike Pirh2B, appears to occur at a second, previously undefined 5′ donor site (donor site 2) 28 base pairs into Pirh2 intron 7. By Northern blot analysis, Pirh2A (full-length Pirh2) has been demonstrated to be the predominantly expressed isoform ( 11 ), and therefore donor site 1 is likely to be the stronger donor site. However, our results clearly demonstrate that Pirh2C exists naturally, perhaps through “leaky” utilization of donor site 2. A number of factors could influence the differential splicing of Pirh2C, such as lack of definition between the two canonical donor sites, or it could be due to comparatively weak elements within Pirh2 exons or introns that provide information to the mRNA splicing machinery ( 48 ). Further in-depth studies are also needed to elucidate the molecular mechanisms underlying the generation of this isoform. Future studies are also needed to explore the relative expression of these isoforms in matched normal and cancerous tissue specimens and would prove helpful in further elucidating their function in human malignancies.

Although Pirh2A, Pirh2B, and Pirh2C showed similarity in their subcellular distributions, each protein differed in terms of protein turnover, as demonstrated by protein stability assays and Pirh2 stable cell lines. This is due, in part, to their inability to autoubiquitinate, which we have confirmed in vitro . Ubiquitination of Pirh2C by Pirh2A also probably does not occur due to lack of physical interactions between the two proteins. Of note, as this manuscript was in preparation, a study reported that artificial deletion of the C-terminal end of the Pirh2A isoform abolished Pirh2-Pirh2 interactions ( 49 ), a finding that validates our results with the naturally occurring Pirh2C variant. Furthermore, our results for Pirh2 are unlike those established for MDM2, which is understood to form MDM2 dimer complexes through RING-RING interactions ( 29 – 33 ). An alternative yet unverified possibility could also be that enhanced Pirh2B and Pirh2C stability is the result of lysine deletions to which ubiquitin can be covalently attached. Of the 14 lysines present on Pirh2A, Pirh2B lacks Lys 178 , whereas Pirh2C does not harbor Lys 239 and Lys 241 . Hence, Pirh2B and Pirh2C stability could be enhanced due to the lack of critical lysine residues targeted for ubiquitination.

Unexpectedly, we discovered that Pirh2B and Pirh2C both negatively regulate p53 protein stability possibly by facilitating the intracellular ( in vivo ) ubiquitination of p53, despite our findings that each lacked intrinsic ubiquitination activity. The means by which this occurs is currently unclear; however, our findings appear to rule out the possibility that either functions through Pirh2A, since knockdown of Pirh2A by RNA interference had no apparent effect on their ability to negatively regulate p53 stability. The fact that Pirh2C does not interact with Pirh2A yet still facilitates p53 ubiquitination intracellularly only further supports the notion that these isoforms may mediate p53 ubiquitination inside the cell independent of full-length Pirh2. Still, because Pirh2 knockdown is less than 100%, it remains a possibility that the remaining Pirh2A expressed is sufficient for Pirh2B or Pirh2C-mediated p53 ubiquitination.

Our results indicate that expression of Pirh2B and Pirh2C, as well as Pirh2A (full-length Pirh2), enhance the levels of endogenous MDM2. In addition, our results have, for the first time, demonstrated that endogenous MDM2 and each of the various Pirh2 isoforms are capable of mutual interactions. Our findings show that interactions between MDM2 and full-length Pirh2, Pirh2B, and Pirh2C are direct, which is another novel finding of our study. We have delineated the domains on MDM2 responsible for interactions with Pirh2 to be both the central acidic domain (amino acids 223–274) and the N-terminal zinc finger domain (amino acids 305–322). Pirh2 and its isoforms can be grouped with a growing number of proteins that associate with the MDM2 acidic and zinc finger regions, including YY1 ( 50 ), pRb ( 51 ), ARF ( 52 ), and ribosomal proteins L5, L11, and L23 ( 53 ).

Interestingly, loss of the MDM2 RING finger in MDM2 variant 1–350 decreased Pirh2 binding; however, the association remained stronger than binding between Pirh2 and full-length MDM2. MDM2 variant 351–491, which contains only the RING finger domain of MDM2, possessed no Pirh2 binding capability. Reciprocally, the truncated RING domains of Pirh2B and Pirh2C did not have any apparent effect on their association with MDM2. Collectively, these results therefore suggest that unlike interactions between numerous other RING domain-containing proteins, interactions between Pirh2 and MDM2 are not mediated through their RING domains ( 54 – 56 ). Notably, no association was seen for MDM2 variant 1–150, which contains the MDM2 p53 binding domain, indicating that Pirh2 and p53 bind MDM2 in disparate regions of MDM2.

Importantly, our findings demonstrate that p53 does not facilitate the association between MDM2 and Pirh2, based on the fact that MDM2 and the Pirh2 isoforms exhibit direct interactions in the absence of p53. In addition to their mutual cross-talk in the context of p53, our results therefore also raise the notion that Pirh2 and MDM2 may fulfill coordinate p53-independent functions within the cell, and future investigations are needed to explore this possibility. Collectively, these data strongly suggest that Pirh2A, -B, and -C may promote p53 ubiquitination indirectly through an MDM2-mediated mechanism. Surprisingly, however, in HCT116 MDM2 shRNA-infected knockdown cells, Pirh2A, -B, and -C were each capable of negatively regulating p53 protein stability despite loss of MDM2 expression (data not shown). Whether this is due to the fact that a certain amount of MDM2 is still expressed is an issue that warrants additional study. With the very recent discovery of several additional p53-specific E3 ligases, the potential for Pirh2A, -B, or -C to regulate still other ubiquitin ligases through these mechanisms also requires examination.

In addition to our novel findings characterizing a relationship between Pirh2 and MDM2, the notion that Pirh2 and its isoforms function at least in part through MDM2 is also supported by a report showing that co-expression of full-length Pirh2 and MDM2 has an additive effect on p53 ubiquitination ( 11 ). Given that the two proteins bind disparate regions of p53 ( 11 , 35 ), this outcome could be the result of paralleled ubiquitination of p53, and this could be true for Pirh2A that harbors intrinsic E3 ligase activity. However, our results indicate that both Pirh2B and Pirh2C lack ubiquitin ligase activity and Pirh2C has a diminished ability to bind p53. It is therefore possible that these isoforms (as well as Pirh2A) could positively regulate MDM2 ligase activity toward p53 through a mechanism that involves Pirh2-MDM2 interactions. A hypothetical model showing this possibility is illustrated in Fig. 13 . Whether Pirh2A, -B, or -C increases MDM2 protein levels or its p53-specific E3 ligase function via their mutual interactions with MDM2 could be an intriguing possibility. Although speculative, each Pirh2 isoform could potentially promote MDM2 ligase activity by enhancing MDM2 protein stability post-translationally. Such a possibility would provide a reasonable explanation for increased MDM2 despite Pirh2-mediated inhibition of p53 transcriptional activity. Still further, Pirh2A, -B, or -C-mediated alterations in a signaling pathway upstream to MDM2 represent yet another potential mechanism by which Pirh2 may regulate MDM2 protein levels. Future in-depth studies examining the interactions between these two important p53-specific E3 ligases and the means by which they appear to cooperate to promote p53 ubiquitination will help to shed more light on this intriguing issue. Likewise, exploration into possible interactions between Pirh2 and other more recently discovered E3 ligases is also needed.

Our results also indicate that p53 transcriptional activity is inhibited by both Pirh2B and Pirh2C, in addition to Pirh2A. Of note, our report is the first to demonstrate that two naturally occurring Pirh2 isoforms with disrupted RING domains are capable of promoting p53 ubiquitination and inhibiting p53 transcriptional function. Our results are in agreement with those of a previous study ( 11 ) showing that a truncated Pirh2 protein designed to lack its entire RING finger domain also inhibits p53-mediated transcription. Because it is well established that p53 ubiquitination interferes with its transcriptional activity, it is likely that Pirh2B- and Pirh2C-mediated increases in p53 ubiquitination/degradation are at least in part responsible for the observed inhibition. However, in the case of MDM2, in addition to ubiquitination/degradation, it is well accepted that p53 inhibition also involves a physical interaction at the N terminus of p53 to block its association with the basal transcription machinery ( 36 – 38 ). Because Pirh2 is known to bind to the central DNA binding and tetramerization domains of p53 ( 11 , 35 ), it is possible that the full-length Pirh2A and its novel isoforms may interfere with the ability of p53 to bind to DNA ( 4 ) and hence regulate p53 function independent of their ubiquitin ligase activities. Further in depth studies will help to better elucidate how Pirh2A and the two novel isoforms elicit their ubiquitin-dependent and/or independent effects on the tumor suppressor p53.
