---
pmid: '23267104'
title: Proteome-wide protein interaction measurements of bacterial proteins of unknown
  function.
authors:
- Meier M
- Sit RV
- Quake SR
journal: Proc Natl Acad Sci U S A
year: '2013'
pmcid: PMC3545810
doi: 10.1073/pnas.1210634110
---

# Proteome-wide protein interaction measurements of bacterial proteins of unknown function.
**Authors:** Meier M, Sit RV, Quake SR
**Journal:** Proc Natl Acad Sci U S A (2013)
**DOI:** [10.1073/pnas.1210634110](https://doi.org/10.1073/pnas.1210634110)
**PMC:** [PMC3545810](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3545810/)

## Abstract

1. Proc Natl Acad Sci U S A. 2013 Jan 8;110(2):477-82. doi: 
10.1073/pnas.1210634110. Epub 2012 Dec 24.

Proteome-wide protein interaction measurements of bacterial proteins of unknown 
function.

Meier M(1), Sit RV, Quake SR.

Author information:
(1)Department of Applied Physics and Bioengineering, Stanford University, 
Stanford, CA 94305, USA.

Despite the enormous proliferation of bacterial genome data, surprisingly 
persistent collections of bacterial proteins have resisted functional 
annotation. In a typical genome, roughly 30% of genes have no assigned function. 
Many of these proteins are conserved across a large number of bacterial genomes. 
To assign a putative function to these conserved proteins of unknown function, 
we created a physical interaction map by measuring biophysical interaction of 
these proteins. Binary protein--protein interactions in the model organism 
Streptococcus pneumoniae (TIGR4) are measured with a microfluidic 
high-throughput assay technology. In some cases, informatic analysis was used to 
restrict the space of potential binding partners. In other cases, we performed 
in vitro proteome-wide interaction screens. We were able to assign putative 
functions to 50 conserved proteins of unknown function that we studied with this 
approach.

DOI: 10.1073/pnas.1210634110
PMCID: PMC3545810
PMID: 23267104 [Indexed for MEDLINE]

Conflict of interest statement: The authors declare no conflict of interest.

## Full Text

Edited by David A. Weitz, Harvard University, Cambridge, MA, and approved November 27, 2012 (received for review June 22, 2012)

Author contributions: M.M. and S.R.Q. designed research; M.M. and R.V.S. performed research; M.M. and S.R.Q. contributed new reagents/analytic tools; M.M. and S.R.Q. analyzed data; and M.M., R.V.S., and S.R.Q. wrote the paper.

1Present address: Centre for Signalling Studies and Department of Microsystems Engineering, University of Freiburg 79110, Germany.

The authors declare no conflict of interest.

Despite the enormous proliferation of bacterial genome data, surprisingly persistent collections of bacterial proteins have resisted functional annotation. In a typical genome, roughly 30% of genes have no assigned function. Many of these proteins are conserved across a large number of bacterial genomes. To assign a putative function to these conserved proteins of unknown function, we created a physical interaction map by measuring biophysical interaction of these proteins. Binary protein-â€“protein interactions in the model organism Streptococcus pneumoniae (TIGR4) are measured with a microfluidic high-throughput assay technology. In some cases, informatic analysis was used to restrict the space of potential binding partners. In other cases, we performed in vitro proteome-wide interaction screens. We were able to assign putative functions to 50 conserved proteins of unknown function that we studied with this approach.
