---
pmid: '28675934'
title: Characterization of the Extracellular Matrix of Normal and Diseased Tissues
  Using Proteomics.
authors:
- Naba A
- Pearce OMT
- Del Rosario A
- Ma D
- Ding H
- Rajeeve V
- Cutillas PR
- Balkwill FR
- Hynes RO
journal: J Proteome Res
year: '2017'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC8078728
doi: 10.1021/acs.jproteome.7b00191
---

# Characterization of the Extracellular Matrix of Normal and Diseased Tissues Using Proteomics.
**Authors:** Naba A, Pearce OMT, Del Rosario A, Ma D, Ding H, Rajeeve V, Cutillas PR, Balkwill FR, Hynes RO
**Journal:** J Proteome Res (2017)
**DOI:** [10.1021/acs.jproteome.7b00191](https://doi.org/10.1021/acs.jproteome.7b00191)
**PMC:** [PMC8078728](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8078728/)

## Abstract

1. J Proteome Res. 2017 Aug 4;16(8):3083-3091. doi:
10.1021/acs.jproteome.7b00191.  Epub 2017 Jul 19.

Characterization of the Extracellular Matrix of Normal and Diseased Tissues 
Using Proteomics.

Naba A(1), Pearce OMT(2), Del Rosario A(3), Ma D(4), Ding H(4), Rajeeve V(2), 
Cutillas PR(2), Balkwill FR(2), Hynes RO(1)(5).

Author information:
(1)Koch Institute for Integrative Cancer Research, Massachusetts Institute of 
Technology , Cambridge, Massachusetts 02139, United States.
(2)Barts Cancer Institute, Queen Mary University of London , London EC1M 6BQ, 
United Kingdom.
(3)Proteomics Core Facility, Koch Institute for Integrative Cancer Research, 
Massachusetts Institute of Technology , Cambridge, Massachusetts 02139, United 
States.
(4)Bioinformatics and Computing Facility, Koch Institute for Integrative Cancer 
Research, Massachusetts Institute of Technology , Cambridge, Massachusetts 
02139, United States.
(5)Howard Hughes Medical Institute, Massachusetts Institute of Technology , 
Cambridge, Massachusetts 02139, United States.

The extracellular matrix (ECM) is a complex meshwork of insoluble fibrillar 
proteins and signaling factors interacting together to provide architectural and 
instructional cues to the surrounding cells. Alterations in ECM organization or 
composition and excessive ECM deposition have been observed in diseases such as 
fibrosis, cardiovascular diseases, and cancer. We provide here optimized 
protocols to solubilize ECM proteins from normal or tumor tissues, digest the 
proteins into peptides, analyze ECM peptides by mass spectrometry, and interpret 
the mass spectrometric data. In addition, we present here two novel 
R-script-based web tools allowing rapid annotation and relative quantification 
of ECM proteins, peptides, and intensity/abundance in mass spectrometric data 
output files. We illustrate this protocol with ECMs obtained from two pairs of 
tissues, which differ in ECM content and cellularity: triple-negative breast 
cancer and adjacent mammary tissue, and omental metastasis from high-grade 
serous ovarian cancer and normal omentum. The complete proteomics data set 
generated in this study has been deposited to the public repository 
ProteomeXchange with the data set identifier: PXD005554.

DOI: 10.1021/acs.jproteome.7b00191
PMCID: PMC8078728
PMID: 28675934 [Indexed for MEDLINE]

## Full Text

Abstract

The extracellular matrix (ECM) is a complex meshwork of insoluble fibrillar proteins and signaling factors interacting together to provide architectural and instructional cues to the surrounding cells. Alterations in ECM organization or composition and excessive ECM deposition have been observed in diseases such as fibrosis, cardiovascular diseases, and cancer. We provide here optimized protocols to solubilize ECM proteins from normal or tumor tissues, digest the proteins into peptides, analyze ECM peptides by mass spectrometry, and interpret the mass spectrometric data. In addition, we present here two novel R-script-based web tools allowing rapid annotation and relative quantification of ECM proteins, peptides, and intensity/abundance in mass spectrometric data output files. We illustrate this protocol with ECMs obtained from two pairs of tissues, which differ in ECM content and cellularity: triple-negative breast cancer and adjacent mammary tissue, and omental metastasis from high-grade serous ovarian cancer and normal omentum. The complete proteomics data set generated in this study has been deposited to the public repository ProteomeXchange with the data set identifier: PXD005554.

■ INTRODUCTION

The extracellular matrix (ECM), a complex and dynamic meshwork of cross-linked proteins, is a fundamental component of multicellular organisms. 1 The ECM provides architectural, mechanical, and biochemical signals interpreted by cell-surface receptors and orchestrates cellular processes such as adhesion, migration, proliferation, survival, and differentiation. 2 , 3 Alterations in the composition and organization of the ECM cause or accompany the development of diseases including fibroses, cardiovascular diseases, and cancers. 4 − 6

Because of its complexity and insolubility, the ECM has been difficult to analyze biochemically, and, until recently, it was not possible to obtain a detailed understanding of its composition. 7 We have previously developed a proteomic pipeline to characterize the composition of in vivo ECMs and have shown that any given tissue or tumor includes well over 100 ECM and ECM-associated proteins and that characteristic differences can be detected between different tissues or tumors. 8 − 10 One critical step in this proteomic approach was the development of a method to enrich ECM proteins from tissues and tumors. To do so, we and others devised decellularization methods that deplete intracellular proteins and permit the enrichment of ECM proteins. 8 , 11 − 20 A second critical step of the approach was the identification and annotation of the ECM components, or “matrisome”, in large proteomic data sets. To this end, we devised a computational approach and defined the in silico matrisome as the ensemble of genes encoding core ECM proteins (ECM glycoproteins, collagens, and proteoglycans) and ECM-associated proteins (these are proteins structurally resembling ECM proteins, ECM remodeling enzymes, and secreted factors including growth factors). 8 , 21

Because the details of the methods applied in the various analyses of ECM proteins using mass-spectrometry-based proteomics in our laboratories and others vary somewhat, we have now explored the impact of variations in the protocols. We provide here updated protocols for (1) digestion of ECM proteins into peptides, (2) analysis of ECM peptides by mass spectrometry, and (3) interpretation of mass spectrometric data. We illustrate the protocols with results obtained on the ECM from two sets of tissues that differ significantly in their ECM content and cellularity: human triple-negative breast cancer samples and adjacent breast tissue, and omental metastases from high-grade serous ovarian cancer (HGSOC) and normal omentum ( Figure 1A ). The optimized protocols presented here make the comprehensive characterization of the composition of the ECM accessible to a broad range of scientists and should help enhance the identification of disease biomarkers and novel therapeutic targets.

Optimization of Methods To Solubilize and Digest ECM Proteins

We initially devised a method that consisted of enrichment of tissue samples for ECM proteins, partial solubilization in 8 M urea, and digestion of this “crude” ECM extract into peptides. Using this protocol, the most abundant proteins identified were fibrillar collagens. 8 We hypothesized that the abundance of fibrillar collagens could prevent the identification of other ECM or ECM-associated proteins present in lower abundance. To test this, we compared the original method with a method consisting of digesting only the urea-soluble fraction of ECM-enriched samples ( Figure 1B ). We observed that for the four tissues analyzed (breast cancer and adjacent breast tissue samples and omentum and ovarian-cancer-derived omental metastasis samples; n = 2 for each sample type) the abundance of ECM peptides ( Figures 2A , B , upper panels) and the numbers of identified spectra ( Figures 2A , B , middle panels) were somewhat greater in most of the crude ECM preparations as compared with the corresponding urea-soluble extracts ( Supplementary Table S1B , C ). However, the numbers of ECM proteins confidently identified were mostly similar ( Figures 2A , B , lower panels, and Supplementary Table S1B , C ), although urea-soluble preparations of normal and diseased omentum generally gave a higher number of ECM proteins compared with crude ECM preparations. This might be explained by the low content of fibrillar ECM in this tissue ( Figure 1A ).

We next sought to analyze the composition of the urea-insoluble protein fraction ( Supplementary Figure S1A and Supplementary Table S1B ) and observed that only four proteins were detected solely in the pellet and not in either urea-soluble or crude ECM preparations (namely, AMBP and LMAN1L, which is, in fact, a transmembrane protein in TNBC-adjacent breast tissue and COL6A5 and MFAP2 in TNBC samples) ( Supplementary Figure S1B and Supplementary Table S1B ). However, the total number of matrisome proteins detected in the pellet was less than either the crude or urea-soluble preparations. It is possible that with more extensive analysis, including peptide fractionation (see below), these proteins would also be detected in the urea-soluble or crude ECM preparations. Therefore, in general, both crude and urea-soluble extractions result in similar numbers of ECM proteins detected, although one method may be superior to the other depending on the tissue and, in particular, depending on the collagen content of the tissue.

Optimization of Methods To Fractionate ECM-Protein-Derived Peptides

We have previously reported that off-line fractionation of peptide mixtures by off-gel electrophoresis increased by a factor of 3 the number of matrisome peptides confidently identified, which resulted in the identification of twice as many core ECM components and over 10 times the number of matrisome-associated proteins. 8 Off-gel electrophoresis is a lengthy process and is now being replaced in proteomic pipelines by fractionation using high-pH liquid chromatography (also termed basic-reversed-phase or bRP-LC), orthogonal to the LC separation in-line with the tandem mass spectrometry. We sought to determine whether fractionation by bRP-LC was beneficial to identify otherwise undetected ECM proteins from both urea-soluble and crude ECM preparations. Independent of the tissue type or the method employed to digest ECM-enriched samples into peptides, fractionation prior to LC−MS/MS resulted in the identification of a greater number of spectra and proteins ( Figure 3 and Supplementary Table S1D ). In the case of omental samples, the urea-soluble preparation, when coupled to bRP-LC fractionation, consistently resulted in higher numbers of ECM proteins detected when compared with fractionated crude preparations ( Figure 3A ). This is presumably a consequence of the reduced amount of collagens in the urea-soluble fraction ( Supplementary Table S1D ). Furthermore, peptide fractionation was particularly beneficial to identify lower-abundance matrisome-associated proteins such as ECM-affiliated proteins and ECM regulators (including ECM cross-linking enzymes or proteases) ( Figure 3B , C and Supplementary Table S1D ).

■ DISCUSSION

We provide here a detailed description of mass-spectrometry-based methods allowing in-depth and comprehensive characterization of the composition of ECM of normal and diseased tissues. The major advantage of these methods is that each step, from the solubilization of ECM proteins to the interpretation of mass spectrometric data, has been optimized to take into account biochemical characteristics of ECM and ECM-associated proteins. For example, we recommend allowing for proline and lysine hydroxylation when searching databases, as this significantly increases ECM protein detection and identification. The preferred method to solubilize ECM-enriched protein samples for digestion to peptides depends on the tissue and its ECM content, and, in some cases, restricting the analysis to urea-soluble ECM proteins permits detection of additional proteins. This is because significant proportions of the collagens remain insoluble. Few proteins are missed by ignoring the urea-insoluble fraction, although, of course, the relative amounts of proteins detected are altered, which compromises semi-quantitative conclusions. Furthermore, the inclusion of a pre-fractionation step by bRP-LC chromatography provides an alternative means, allowing the detection of additional proteins. It is worth noting that high-pH reversed-phase peptide fractionation can be achieved using commercially available spin columns requiring only a bench centrifuge. In addition, any number of fractions can be generated depending on the complexity of the samples and the desired coverage.

Although not discussed in the manuscript, these methods can be coupled to label-based quantitative proteomics, 29 which allows quantification of the abundance of proteins and peptides in different samples and also allows multiplexing, thus reducing mass-spectrometer time.

■ CONCLUSIONS

Alterations in the composition and organization of the ECM cause or accompany the development of diseases such as fibrosis, cardiovascular diseases, musculo-skeletal diseases, and cancers.

The application of the proposed methods to profile the composition of the ECM of, for example, pairs of normal versus diseased tissues, treated versus untreated tissues, poorly versus highly metastatic tumors, or cell or organoid cultures can lead to the identification of novel ECM proteins that could serve as prognostic or diagnostic biomarkers or novel therapeutic targets.
