---
pmid: '23262492'
title: Interaction between FIP200 and ATG16L1 distinguishes ULK1 complex-dependent
  and -independent autophagy.
authors:
- Gammoh N
- Florey O
- Overholtzer M
- Jiang X
journal: Nat Struct Mol Biol
year: '2013'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC3565010
doi: 10.1038/nsmb.2475
---

# Interaction between FIP200 and ATG16L1 distinguishes ULK1 complex-dependent and -independent autophagy.
**Authors:** Gammoh N, Florey O, Overholtzer M, Jiang X
**Journal:** Nat Struct Mol Biol (2013)
**DOI:** [10.1038/nsmb.2475](https://doi.org/10.1038/nsmb.2475)
**PMC:** [PMC3565010](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3565010/)

## Abstract

1. Nat Struct Mol Biol. 2013 Feb;20(2):144-9. doi: 10.1038/nsmb.2475. Epub 2012
Dec  23.

Interaction between FIP200 and ATG16L1 distinguishes ULK1 complex-dependent and 
-independent autophagy.

Gammoh N(1), Florey O, Overholtzer M, Jiang X.

Author information:
(1)Cell Biology Department, Memorial Sloan Kettering Cancer Center, New York, 
New York, USA.

Autophagy is a finely orchestrated cellular catabolic process that requires 
multiple autophagy-related gene products (ATG proteins). The ULK1 complex 
functions to integrate upstream signals to downstream ATG proteins through an 
unknown mechanism. Here we have identified an interaction between mammalian 
FIP200 and ATG16L1, essential components of the ULK1 and ATG5 complexes, 
respectively. Further analyses show this is a direct interaction mediated by a 
short domain of ATG16L1 that we term the FIP200-binding domain (FBD). The FBD is 
not required for ATG16L1 self-dimerization or interaction with ATG5. Notably, an 
FBD-deleted ATG16L1 mutant is defective in mediating amino acid 
starvation-induced autophagy, which requires the ULK1 complex. However, this 
mutant retains its function in supporting glucose deprivation-induced autophagy, 
a ULK1 complex-independent process. This study therefore identifies a previously 
uncharacterized interaction between the ULK1 and ATG5 complexes that can 
distinguish ULK1-dependent and -independent autophagy processes.

DOI: 10.1038/nsmb.2475
PMCID: PMC3565010
PMID: 23262492 [Indexed for MEDLINE]

Conflict of interest statement: COMPETING FINANCIAL INTERESTS The authors 
declare no competing interests.

## Full Text

Degradation of cellular contents can occur via the proteasome or lysosome systems1. A number of pathways exist that deliver substrates for degradation to either systems. Macroautophagy (commonly referred to as autophagy) is one pathway that facilitates the degradation of long lived proteins, damaged organelles and infectious pathogens, resulting in the clearance of toxic materials and increased nutrients availability in the cell2. Because of these substrates, autophagy has been implicated in a number of physiological and pathological processes including development, pathogen infection, neurodegeneration and cancer3.

During autophagy, a number of protein complexes orchestrate the formation of a lipid bilayer (termed phagophore or pre-autophagosome) which upon maturation, engulfs cytoplasmic materials and forms the autophagosome. The eventual fusion of autophagosomes with lysosomes results in the degradation of the autophagosome content and the recycling of nutrients back into the cytoplasm. Key players in these events include a family of ubiquitin-like proteins, such as LC3, which are involved in the phagophore maturation and possibly cargo selection4–7. Membrane targeting of LC3 is essential for autophagy and requires a series of ubiquitin-like conjugation events that lead to the conjugation of cytosolic LC3 (LC3–I) to membrane bound, phosphatidylethanolamine (PE)-conjugated form (LC3–II)4. These events are catalyzed by autophagy-related proteins (or ATG proteins) with E1, E2 and E3-like enzymatic activities termed ATG7, ATG3 and ATG5–ATG12, respectively. The ATG5–ATG12 conjugate (herein referred to as ATG5–12) formation is also catalyzed by similar conjugation events but requires a distinct E2-like enzyme termed ATG108. In addition to its possible role as an E3-like enzyme during LC3–II formation, the ATG5–12 conjugate was shown to form a large protein complex with ATG16L1 which is thought to be required to specify the site of LC3 conjugation during autophagy9. Depletion of any of ATG5, ATG16L1 or ATG7 completely abolishes autophagosome formation10–12. Meanwhile, depletion of ATG3 results in defective maturation of the phagophore structure13. LC3 can also be recruited to single membrane structures, for example during phagocytosis or entotic cell clearance14–16. In these cases, the core machinery utilized during LC3–II formation is also required.

A number of upstream signaling complexes can regulate autophagy including the Vps34 and ULK1 complexes. The Vps34 complex is comprised of a number of proteins, including the Vps34 lipid kinase, p150, Beclin and ATG14, and is essential for phagophore formation and proper recruitment of ATG proteins to the phagophore17,18. On the other hand, the ULK1 complex, comprised of the protein kinase ULK1 and several regulatory components including ATG13 and FIP200, is suppressed by the mTORC1 kinase activity19,20. Once mTORC1 activity is inhibited, for example by amino acid deprivation or cytotoxic response, the ULK1 complex becomes activated and stimulates autophagy. Mechanistically, how the ULK1 complex coordinates with other ATG complexes during autophagy is not clear, and the protein substrate(s) of its kinase activity required for autophagy induction have not yet been identified.

In this study we sought to identify novel players in autophagy that are particularly required during the initial phagophore formation stage. To do so, we developed a system to isolate membrane-localized ATG proteins. Using this system, we conducted tandem affinity purification and identified FIP200 as a direct binding partner of ATG16L1. We further provide evidence to demonstrate that this novel interaction between the ULK1 complex and the ATG5 complex is required for ULK1 complex-dependent but not ULK1 complex-independent autophagy processes.

ResultsPre-autophagosomes accumulate in the absence of ATG3In order to study the molecular events during pre-autophagosome structures formation, we separated membrane-bound proteins from cytosolic proteins by subcellular fractionation. As can be seen in figure 1a, cytosolic LC3 (LC3–I) was separated from membrane-bound LC3 (LC3–II) in wild type mouse embryonic fibroblasts (MEFs). However, using this method, we hardly detected the accumulation of upstream ATG proteins, such as ATG16L1, in the membrane fraction of wild type MEFs even when autophagy was induced by amino acid deprivation (Fig. 1b). This suggests that the recruitment of upstream ATG proteins to pre-autophagosome structures may occur in a transient manner. Previous studies suggest that in the absence of ATG3, pre-autophagosome structures are unable to mature into autophagosomes and therefore accumulate in the cell likely leading to stabilization of upstream autophagy events which are transient in ATG3-expressing cells13,21. Therefore, we tested the subcellular localization of ATG16L1 in Atg3−/−. Interestingly, ATG16L1 accumulated on the membrane fraction of these cells (Fig. 1b). We observed no detectable increase of ATG16L1 recruitment to the membrane fraction after 2 hrs of amino acid starvation in Atg3−/−, in agreement with previous studies21. Similarly, we show by immunofluorescence analyses that ATG16L1 positive structures, which co-localize with endogenous ATG5, were detectable even under nutrient rich conditions in the absence of ATG3 (Fig. 1c).Identification of FIP200 as an ATG16L1-interacting partnerWe further sought to identify novel protein-protein interactions of membrane bound ATG proteins in Atg3−/− using ATG14 and ATG16L1 as baits. To do so, we used tandem affinity purification to purify ATG14 and ATG16L1 complexes followed by SDS–PAGE analysis and silver staining (Fig. 2a). We observed prominent bands in the ATG14 sample that correspond to known ATG14 interacting partners, including Vps34 and p150, thereby confirming that this method is able to purify functional autophagy complexes. In the case of ATG16L1 purification, we detected a prominent distinct band of a molecular weight greater than 170 kDa in addition to the ATG5–12 conjugate, the known binding partner of ATG16L1. Mass spectrometric analysis revealed the identity of this band as FIP200, an essential component of the ULK1 complex. Previously, the ULK1 complex was proposed to be functionally linked to the ATG5 complex, but how these two complexes cross-talk is unclear.To verify the interaction between ATG16L1 and FIP200, we expressed ATG16L1 in 293T cells and found that ATG16L1, but not ATG14, can pull down FIP200 as well as ATG13 (Fig. 2b). In addition, stably expressed ATG16L1 in Atg3−/− can also pull down endogenous FIP200, ATG13 and to a lower extent ULK1 (Supplementary Fig. 1a). Reciprocal pull down of FIP200 expressed in 293T cells can co-precipitate ATG16L1 as well as the ATG5–12 conjugate (Fig. 2c). Furthermore, we tested the potential interaction of endogenous FIP200 with ATG16L1. Because their interaction is most likely transient and stabilized by ATG3 deletion, we used Atg3−/− for this purpose. As shown in figure 2d, immunoprecipitation of endogenous FIP200 can co-precipitate endogenous ATG16L1 in these cells. Using immunofluorescence analysis we show that ATG16L1 punctate structures almost completely co-localized with FIP200 in both wild type MEFs (Fig. 2e) and Atg3−/− (Supplementary Fig. 1b) as was described previously21,22.Interaction between FIP200 and ATG16L1 is directHaving shown that the ATG16L1-containing ATG5 complex and the FIP200-containing ULK1 complex can co-precipitate, we further sought to determine which proteins directly mediate the interaction between these two complexes. First, we found that ATG16L1 and FIP200 can interact in MEF cells lacking the expression of ATG5 (Fig. 3a and Supplementary Fig. 2a) or ATG13 (Fig. 3b), indicating that ATG5 and ATG13 are dispensable for the interaction between ATG16L1 and FIP200. Furthermore, ATG5 and its ATG12 conjugation mutant (ATG5 K130R) were unable to pull down FIP200 (Supplementary Fig. 2b). In addition, we show that purified recombinant ATG16L1 can interact with recombinant FIP200 (Fig. 3c, left panel) but not with recombinant ATG13 (Fig. 3c, right panel), ATG3 or ATG7 (Supplementary Fig. 2c). Overall, these results indicate that ATG16L1 and FIP200 directly interact with each other independently of other complex components.Identification of ATG16L1 domain required for FIP200 bindingIn order to determine the function of the ATG16L1–FIP200 interaction in autophagy, we first sought to map the ATG16L1 domain required to mediate its binding to FIP200. To do so, we generated a series of ATG16L1 truncation fragments (depicted in Fig. 4a) based on previous structural analysis of yeast ATG1623–25. These include truncations of ATG16L1 that lack ATG5-binding region (mediated through the N-terminal region), coiled-coil domain (CCD, required for self-dimerization), and seven WD40 repeats (located at the C-terminal half). We expressed the truncation fragments of ATG16L1 in 293T cells and tested their interaction with endogenous FIP200 and ATG5. As shown in figure 4b, ATG16L1 fragments that were defective in ATG5 binding can still bind to FIP200, indicating that FIP200 and ATG5 interact with ATG16L1 through distinct domains. The ATG16L1 fragment lacking a region between the WD40 and coiled-coil domains (Δ4 fragment, residues 1–335 deleted) were defective in binding to FIP200. These results suggest that the interaction between ATG16L1 and FIP200 is mediated through residues 206–335 of ATG16L1. This region of ATG16L1 was previously shown to be required for Rab33B binding26.The Δ4 fragment includes a large deletion of the N-terminal half of ATG16L1 and is therefore unable to bind ATG5 or self-dimerize in addition to its inability to bind FIP200. Therefore, we sought to identify a smaller region of ATG16L1 that is required for FIP200 binding. By further mutational analyses we identified a shorter region of ATG16L1 within residues 206–335 to be required for FIP200 binding. As shown in figure 4c, ATG16L1 mutant deleted of residues number 229–242 (denoted as Δ229–242) as well as an additional mutant harboring a larger deletion (Δ182–242) were defective in binding to FIP200 when compared to the full length protein. Importantly, Δ229–242 mutant of ATG16L1 still retained its ATG5 binding ability (Fig. 4c) as well as its ability to dimerize with full length protein (Fig. 4d). Interestingly, residues 229–242 were previously shown dispensable for Rab33B binding26. In addition, these residues are not conserved in ATG16L2, which does not support autophagy despite its ability to self-dimerize and bind to ATG526, but is conserved in ATG16L1 from various vertebrate species (Supplementary Fig. 3). Consistently, we found that FIP200 was unable to bind ATG16L2 (Fig. 4e). These results demonstrate that residues 229–242 can differentiate the ability of ATG16L1 to bind to FIP200 from binding to ATG5 or self-dimerization. We name this region as the FIP200-binding domain (FBD) of ATG16L1.ΔFBD mutant is defective in ULK1-dependent autophagyTo test whether the FIP200–ATG16L1 interaction is required for the function of ATG16L1 in autophagy, we reconstituted Atg16l1−/− with either full length or FIP200-binding domain truncated mutant (ΔFBD). In the absence of ATG16L1 expression, autophagy was completely disrupted10. Meanwhile, when we expressed full length ATG16L1 or ATG16L1 FBD basal levels of autophagy was restored, as measured by LC3–I to LC3–II conversion (Fig. 5a, compare lanes 3 and 5). However, when we induced autophagy by depriving cells of amino acids, ATG16L1ΔFBD-expressing cells showed greatly reduced autophagy compared to cells expressing full length ATG16L1 (Fig. 5a, compare lanes 4 and 6). We obtained similar results when we induced autophagy by treating cells with the selective mTOR inhibitor, Torin 1 (Fig. 5b). The defect of ATG16L1ΔFBD in mediating amino acid starvation-induced autophagy is further supported by impeded p62 degradation (a well establish autophagosome substrate) in the ATG16L1ΔFBD-reconstituted cells (Fig. 5c). On the other hand, we found that the ΔWD40 mutant, which retains its ability to bind to FIP200 (Fig. 4b), can restore amino acid starvation-induced autophagy to similar levels to full length protein (Supplementary Fig. 4). Consistently, we show that ATG16L1ΔFBD exhibits reduced ability to localize to punctate structures corresponding to pre-autophagosomes (Fig. 5d). The formation of GFP–LC3 punctate structures, a marker of autophagosomes, was also markedly reduced in cells reconstituted with ATG16L1ΔFBD compared to cells reconstituted with full length ATG16L1 (Fig. 5e). Therefore, the FIP200–ATG16L1 interaction is crucial for the function of ATG16L1 during amino acid starvation-induced autophagy, a process which requires the FIP200-containing ULK1 complex.ΔFBD mutant is fully active during ULK1-independent autophagyThe inability of ATG16L1ΔFBD to fully restore autophagy can be potentially due to an overall misfolding of the protein caused by the introduced truncation rather than its deficiency in binding FIP200. If so, our results in figure 5 would not be able to support a functional relevance of the FIP200–ATG16L1 interaction. Therefore, we sought to assess whether ATG16L1ΔFBD possesses any biological function that is independent of FIP200 and thereby the ULK1 complex. Recently, glucose starvation was suggested to induce autophagy in a ULK1 complex-independent manner which correlates with an increase in cellular ammonia levels and lack of mTORC1 inhibition27,28. We confirmed this by using ULK1 and ULK2 double knockout MEFs (ULK DKO) where glucose starvation induced LC3–II formation which is susceptible to lysosomal degradation (Fig. 6a). Meanwhile, we observed that glucose starvation did not induce ULK1 dephosphorylation on residues that are dephosphorylated during amino acid starvation29 (Fig. 6b). Interestingly, we when reconstituted Atg16l1−/− with full length ATG16L1 or ATG16L1ΔFBD comparable LC3–I to LC3–II conversion upon glucose starvation were exhibited whereas the absence of ATG16L1 expression completely abolished LC3 conjugation (Fig. 6c) indicating that ATG16L1ΔFBD is fully functional during such ULK1 complex-independent autophagy. Similarly, we observed comparable GFP–LC3 punctate structure formation between full length ATG16L1 and ATG16L1ΔFBD expressing cells (Fig. 6d). Overall, these analyses suggest that, unlike ULK1 complex-dependent autophagy induced by amino acid starvation, ULK1 complex-independent autophagy induced by glucose starvation does not require FIP200 binding to ATG16L1.

Pre-autophagosomes accumulate in the absence of ATG3In order to study the molecular events during pre-autophagosome structures formation, we separated membrane-bound proteins from cytosolic proteins by subcellular fractionation. As can be seen in figure 1a, cytosolic LC3 (LC3–I) was separated from membrane-bound LC3 (LC3–II) in wild type mouse embryonic fibroblasts (MEFs). However, using this method, we hardly detected the accumulation of upstream ATG proteins, such as ATG16L1, in the membrane fraction of wild type MEFs even when autophagy was induced by amino acid deprivation (Fig. 1b). This suggests that the recruitment of upstream ATG proteins to pre-autophagosome structures may occur in a transient manner. Previous studies suggest that in the absence of ATG3, pre-autophagosome structures are unable to mature into autophagosomes and therefore accumulate in the cell likely leading to stabilization of upstream autophagy events which are transient in ATG3-expressing cells13,21. Therefore, we tested the subcellular localization of ATG16L1 in Atg3−/−. Interestingly, ATG16L1 accumulated on the membrane fraction of these cells (Fig. 1b). We observed no detectable increase of ATG16L1 recruitment to the membrane fraction after 2 hrs of amino acid starvation in Atg3−/−, in agreement with previous studies21. Similarly, we show by immunofluorescence analyses that ATG16L1 positive structures, which co-localize with endogenous ATG5, were detectable even under nutrient rich conditions in the absence of ATG3 (Fig. 1c).

Pre-autophagosomes accumulate in the absence of ATG3

In order to study the molecular events during pre-autophagosome structures formation, we separated membrane-bound proteins from cytosolic proteins by subcellular fractionation. As can be seen in figure 1a, cytosolic LC3 (LC3–I) was separated from membrane-bound LC3 (LC3–II) in wild type mouse embryonic fibroblasts (MEFs). However, using this method, we hardly detected the accumulation of upstream ATG proteins, such as ATG16L1, in the membrane fraction of wild type MEFs even when autophagy was induced by amino acid deprivation (Fig. 1b). This suggests that the recruitment of upstream ATG proteins to pre-autophagosome structures may occur in a transient manner. Previous studies suggest that in the absence of ATG3, pre-autophagosome structures are unable to mature into autophagosomes and therefore accumulate in the cell likely leading to stabilization of upstream autophagy events which are transient in ATG3-expressing cells13,21. Therefore, we tested the subcellular localization of ATG16L1 in Atg3−/−. Interestingly, ATG16L1 accumulated on the membrane fraction of these cells (Fig. 1b). We observed no detectable increase of ATG16L1 recruitment to the membrane fraction after 2 hrs of amino acid starvation in Atg3−/−, in agreement with previous studies21. Similarly, we show by immunofluorescence analyses that ATG16L1 positive structures, which co-localize with endogenous ATG5, were detectable even under nutrient rich conditions in the absence of ATG3 (Fig. 1c).

Identification of FIP200 as an ATG16L1-interacting partnerWe further sought to identify novel protein-protein interactions of membrane bound ATG proteins in Atg3−/− using ATG14 and ATG16L1 as baits. To do so, we used tandem affinity purification to purify ATG14 and ATG16L1 complexes followed by SDS–PAGE analysis and silver staining (Fig. 2a). We observed prominent bands in the ATG14 sample that correspond to known ATG14 interacting partners, including Vps34 and p150, thereby confirming that this method is able to purify functional autophagy complexes. In the case of ATG16L1 purification, we detected a prominent distinct band of a molecular weight greater than 170 kDa in addition to the ATG5–12 conjugate, the known binding partner of ATG16L1. Mass spectrometric analysis revealed the identity of this band as FIP200, an essential component of the ULK1 complex. Previously, the ULK1 complex was proposed to be functionally linked to the ATG5 complex, but how these two complexes cross-talk is unclear.To verify the interaction between ATG16L1 and FIP200, we expressed ATG16L1 in 293T cells and found that ATG16L1, but not ATG14, can pull down FIP200 as well as ATG13 (Fig. 2b). In addition, stably expressed ATG16L1 in Atg3−/− can also pull down endogenous FIP200, ATG13 and to a lower extent ULK1 (Supplementary Fig. 1a). Reciprocal pull down of FIP200 expressed in 293T cells can co-precipitate ATG16L1 as well as the ATG5–12 conjugate (Fig. 2c). Furthermore, we tested the potential interaction of endogenous FIP200 with ATG16L1. Because their interaction is most likely transient and stabilized by ATG3 deletion, we used Atg3−/− for this purpose. As shown in figure 2d, immunoprecipitation of endogenous FIP200 can co-precipitate endogenous ATG16L1 in these cells. Using immunofluorescence analysis we show that ATG16L1 punctate structures almost completely co-localized with FIP200 in both wild type MEFs (Fig. 2e) and Atg3−/− (Supplementary Fig. 1b) as was described previously21,22.

Identification of FIP200 as an ATG16L1-interacting partner

We further sought to identify novel protein-protein interactions of membrane bound ATG proteins in Atg3−/− using ATG14 and ATG16L1 as baits. To do so, we used tandem affinity purification to purify ATG14 and ATG16L1 complexes followed by SDS–PAGE analysis and silver staining (Fig. 2a). We observed prominent bands in the ATG14 sample that correspond to known ATG14 interacting partners, including Vps34 and p150, thereby confirming that this method is able to purify functional autophagy complexes. In the case of ATG16L1 purification, we detected a prominent distinct band of a molecular weight greater than 170 kDa in addition to the ATG5–12 conjugate, the known binding partner of ATG16L1. Mass spectrometric analysis revealed the identity of this band as FIP200, an essential component of the ULK1 complex. Previously, the ULK1 complex was proposed to be functionally linked to the ATG5 complex, but how these two complexes cross-talk is unclear.

To verify the interaction between ATG16L1 and FIP200, we expressed ATG16L1 in 293T cells and found that ATG16L1, but not ATG14, can pull down FIP200 as well as ATG13 (Fig. 2b). In addition, stably expressed ATG16L1 in Atg3−/− can also pull down endogenous FIP200, ATG13 and to a lower extent ULK1 (Supplementary Fig. 1a). Reciprocal pull down of FIP200 expressed in 293T cells can co-precipitate ATG16L1 as well as the ATG5–12 conjugate (Fig. 2c). Furthermore, we tested the potential interaction of endogenous FIP200 with ATG16L1. Because their interaction is most likely transient and stabilized by ATG3 deletion, we used Atg3−/− for this purpose. As shown in figure 2d, immunoprecipitation of endogenous FIP200 can co-precipitate endogenous ATG16L1 in these cells. Using immunofluorescence analysis we show that ATG16L1 punctate structures almost completely co-localized with FIP200 in both wild type MEFs (Fig. 2e) and Atg3−/− (Supplementary Fig. 1b) as was described previously21,22.

Interaction between FIP200 and ATG16L1 is directHaving shown that the ATG16L1-containing ATG5 complex and the FIP200-containing ULK1 complex can co-precipitate, we further sought to determine which proteins directly mediate the interaction between these two complexes. First, we found that ATG16L1 and FIP200 can interact in MEF cells lacking the expression of ATG5 (Fig. 3a and Supplementary Fig. 2a) or ATG13 (Fig. 3b), indicating that ATG5 and ATG13 are dispensable for the interaction between ATG16L1 and FIP200. Furthermore, ATG5 and its ATG12 conjugation mutant (ATG5 K130R) were unable to pull down FIP200 (Supplementary Fig. 2b). In addition, we show that purified recombinant ATG16L1 can interact with recombinant FIP200 (Fig. 3c, left panel) but not with recombinant ATG13 (Fig. 3c, right panel), ATG3 or ATG7 (Supplementary Fig. 2c). Overall, these results indicate that ATG16L1 and FIP200 directly interact with each other independently of other complex components.

Interaction between FIP200 and ATG16L1 is direct

Having shown that the ATG16L1-containing ATG5 complex and the FIP200-containing ULK1 complex can co-precipitate, we further sought to determine which proteins directly mediate the interaction between these two complexes. First, we found that ATG16L1 and FIP200 can interact in MEF cells lacking the expression of ATG5 (Fig. 3a and Supplementary Fig. 2a) or ATG13 (Fig. 3b), indicating that ATG5 and ATG13 are dispensable for the interaction between ATG16L1 and FIP200. Furthermore, ATG5 and its ATG12 conjugation mutant (ATG5 K130R) were unable to pull down FIP200 (Supplementary Fig. 2b). In addition, we show that purified recombinant ATG16L1 can interact with recombinant FIP200 (Fig. 3c, left panel) but not with recombinant ATG13 (Fig. 3c, right panel), ATG3 or ATG7 (Supplementary Fig. 2c). Overall, these results indicate that ATG16L1 and FIP200 directly interact with each other independently of other complex components.

Identification of ATG16L1 domain required for FIP200 bindingIn order to determine the function of the ATG16L1–FIP200 interaction in autophagy, we first sought to map the ATG16L1 domain required to mediate its binding to FIP200. To do so, we generated a series of ATG16L1 truncation fragments (depicted in Fig. 4a) based on previous structural analysis of yeast ATG1623–25. These include truncations of ATG16L1 that lack ATG5-binding region (mediated through the N-terminal region), coiled-coil domain (CCD, required for self-dimerization), and seven WD40 repeats (located at the C-terminal half). We expressed the truncation fragments of ATG16L1 in 293T cells and tested their interaction with endogenous FIP200 and ATG5. As shown in figure 4b, ATG16L1 fragments that were defective in ATG5 binding can still bind to FIP200, indicating that FIP200 and ATG5 interact with ATG16L1 through distinct domains. The ATG16L1 fragment lacking a region between the WD40 and coiled-coil domains (Δ4 fragment, residues 1–335 deleted) were defective in binding to FIP200. These results suggest that the interaction between ATG16L1 and FIP200 is mediated through residues 206–335 of ATG16L1. This region of ATG16L1 was previously shown to be required for Rab33B binding26.The Δ4 fragment includes a large deletion of the N-terminal half of ATG16L1 and is therefore unable to bind ATG5 or self-dimerize in addition to its inability to bind FIP200. Therefore, we sought to identify a smaller region of ATG16L1 that is required for FIP200 binding. By further mutational analyses we identified a shorter region of ATG16L1 within residues 206–335 to be required for FIP200 binding. As shown in figure 4c, ATG16L1 mutant deleted of residues number 229–242 (denoted as Δ229–242) as well as an additional mutant harboring a larger deletion (Δ182–242) were defective in binding to FIP200 when compared to the full length protein. Importantly, Δ229–242 mutant of ATG16L1 still retained its ATG5 binding ability (Fig. 4c) as well as its ability to dimerize with full length protein (Fig. 4d). Interestingly, residues 229–242 were previously shown dispensable for Rab33B binding26. In addition, these residues are not conserved in ATG16L2, which does not support autophagy despite its ability to self-dimerize and bind to ATG526, but is conserved in ATG16L1 from various vertebrate species (Supplementary Fig. 3). Consistently, we found that FIP200 was unable to bind ATG16L2 (Fig. 4e). These results demonstrate that residues 229–242 can differentiate the ability of ATG16L1 to bind to FIP200 from binding to ATG5 or self-dimerization. We name this region as the FIP200-binding domain (FBD) of ATG16L1.

Identification of ATG16L1 domain required for FIP200 binding

In order to determine the function of the ATG16L1–FIP200 interaction in autophagy, we first sought to map the ATG16L1 domain required to mediate its binding to FIP200. To do so, we generated a series of ATG16L1 truncation fragments (depicted in Fig. 4a) based on previous structural analysis of yeast ATG1623–25. These include truncations of ATG16L1 that lack ATG5-binding region (mediated through the N-terminal region), coiled-coil domain (CCD, required for self-dimerization), and seven WD40 repeats (located at the C-terminal half). We expressed the truncation fragments of ATG16L1 in 293T cells and tested their interaction with endogenous FIP200 and ATG5. As shown in figure 4b, ATG16L1 fragments that were defective in ATG5 binding can still bind to FIP200, indicating that FIP200 and ATG5 interact with ATG16L1 through distinct domains. The ATG16L1 fragment lacking a region between the WD40 and coiled-coil domains (Δ4 fragment, residues 1–335 deleted) were defective in binding to FIP200. These results suggest that the interaction between ATG16L1 and FIP200 is mediated through residues 206–335 of ATG16L1. This region of ATG16L1 was previously shown to be required for Rab33B binding26.

The Δ4 fragment includes a large deletion of the N-terminal half of ATG16L1 and is therefore unable to bind ATG5 or self-dimerize in addition to its inability to bind FIP200. Therefore, we sought to identify a smaller region of ATG16L1 that is required for FIP200 binding. By further mutational analyses we identified a shorter region of ATG16L1 within residues 206–335 to be required for FIP200 binding. As shown in figure 4c, ATG16L1 mutant deleted of residues number 229–242 (denoted as Δ229–242) as well as an additional mutant harboring a larger deletion (Δ182–242) were defective in binding to FIP200 when compared to the full length protein. Importantly, Δ229–242 mutant of ATG16L1 still retained its ATG5 binding ability (Fig. 4c) as well as its ability to dimerize with full length protein (Fig. 4d). Interestingly, residues 229–242 were previously shown dispensable for Rab33B binding26. In addition, these residues are not conserved in ATG16L2, which does not support autophagy despite its ability to self-dimerize and bind to ATG526, but is conserved in ATG16L1 from various vertebrate species (Supplementary Fig. 3). Consistently, we found that FIP200 was unable to bind ATG16L2 (Fig. 4e). These results demonstrate that residues 229–242 can differentiate the ability of ATG16L1 to bind to FIP200 from binding to ATG5 or self-dimerization. We name this region as the FIP200-binding domain (FBD) of ATG16L1.

ΔFBD mutant is defective in ULK1-dependent autophagyTo test whether the FIP200–ATG16L1 interaction is required for the function of ATG16L1 in autophagy, we reconstituted Atg16l1−/− with either full length or FIP200-binding domain truncated mutant (ΔFBD). In the absence of ATG16L1 expression, autophagy was completely disrupted10. Meanwhile, when we expressed full length ATG16L1 or ATG16L1 FBD basal levels of autophagy was restored, as measured by LC3–I to LC3–II conversion (Fig. 5a, compare lanes 3 and 5). However, when we induced autophagy by depriving cells of amino acids, ATG16L1ΔFBD-expressing cells showed greatly reduced autophagy compared to cells expressing full length ATG16L1 (Fig. 5a, compare lanes 4 and 6). We obtained similar results when we induced autophagy by treating cells with the selective mTOR inhibitor, Torin 1 (Fig. 5b). The defect of ATG16L1ΔFBD in mediating amino acid starvation-induced autophagy is further supported by impeded p62 degradation (a well establish autophagosome substrate) in the ATG16L1ΔFBD-reconstituted cells (Fig. 5c). On the other hand, we found that the ΔWD40 mutant, which retains its ability to bind to FIP200 (Fig. 4b), can restore amino acid starvation-induced autophagy to similar levels to full length protein (Supplementary Fig. 4). Consistently, we show that ATG16L1ΔFBD exhibits reduced ability to localize to punctate structures corresponding to pre-autophagosomes (Fig. 5d). The formation of GFP–LC3 punctate structures, a marker of autophagosomes, was also markedly reduced in cells reconstituted with ATG16L1ΔFBD compared to cells reconstituted with full length ATG16L1 (Fig. 5e). Therefore, the FIP200–ATG16L1 interaction is crucial for the function of ATG16L1 during amino acid starvation-induced autophagy, a process which requires the FIP200-containing ULK1 complex.

ΔFBD mutant is defective in ULK1-dependent autophagy

To test whether the FIP200–ATG16L1 interaction is required for the function of ATG16L1 in autophagy, we reconstituted Atg16l1−/− with either full length or FIP200-binding domain truncated mutant (ΔFBD). In the absence of ATG16L1 expression, autophagy was completely disrupted10. Meanwhile, when we expressed full length ATG16L1 or ATG16L1 FBD basal levels of autophagy was restored, as measured by LC3–I to LC3–II conversion (Fig. 5a, compare lanes 3 and 5). However, when we induced autophagy by depriving cells of amino acids, ATG16L1ΔFBD-expressing cells showed greatly reduced autophagy compared to cells expressing full length ATG16L1 (Fig. 5a, compare lanes 4 and 6). We obtained similar results when we induced autophagy by treating cells with the selective mTOR inhibitor, Torin 1 (Fig. 5b). The defect of ATG16L1ΔFBD in mediating amino acid starvation-induced autophagy is further supported by impeded p62 degradation (a well establish autophagosome substrate) in the ATG16L1ΔFBD-reconstituted cells (Fig. 5c). On the other hand, we found that the ΔWD40 mutant, which retains its ability to bind to FIP200 (Fig. 4b), can restore amino acid starvation-induced autophagy to similar levels to full length protein (Supplementary Fig. 4). Consistently, we show that ATG16L1ΔFBD exhibits reduced ability to localize to punctate structures corresponding to pre-autophagosomes (Fig. 5d). The formation of GFP–LC3 punctate structures, a marker of autophagosomes, was also markedly reduced in cells reconstituted with ATG16L1ΔFBD compared to cells reconstituted with full length ATG16L1 (Fig. 5e). Therefore, the FIP200–ATG16L1 interaction is crucial for the function of ATG16L1 during amino acid starvation-induced autophagy, a process which requires the FIP200-containing ULK1 complex.

ΔFBD mutant is fully active during ULK1-independent autophagyThe inability of ATG16L1ΔFBD to fully restore autophagy can be potentially due to an overall misfolding of the protein caused by the introduced truncation rather than its deficiency in binding FIP200. If so, our results in figure 5 would not be able to support a functional relevance of the FIP200–ATG16L1 interaction. Therefore, we sought to assess whether ATG16L1ΔFBD possesses any biological function that is independent of FIP200 and thereby the ULK1 complex. Recently, glucose starvation was suggested to induce autophagy in a ULK1 complex-independent manner which correlates with an increase in cellular ammonia levels and lack of mTORC1 inhibition27,28. We confirmed this by using ULK1 and ULK2 double knockout MEFs (ULK DKO) where glucose starvation induced LC3–II formation which is susceptible to lysosomal degradation (Fig. 6a). Meanwhile, we observed that glucose starvation did not induce ULK1 dephosphorylation on residues that are dephosphorylated during amino acid starvation29 (Fig. 6b). Interestingly, we when reconstituted Atg16l1−/− with full length ATG16L1 or ATG16L1ΔFBD comparable LC3–I to LC3–II conversion upon glucose starvation were exhibited whereas the absence of ATG16L1 expression completely abolished LC3 conjugation (Fig. 6c) indicating that ATG16L1ΔFBD is fully functional during such ULK1 complex-independent autophagy. Similarly, we observed comparable GFP–LC3 punctate structure formation between full length ATG16L1 and ATG16L1ΔFBD expressing cells (Fig. 6d). Overall, these analyses suggest that, unlike ULK1 complex-dependent autophagy induced by amino acid starvation, ULK1 complex-independent autophagy induced by glucose starvation does not require FIP200 binding to ATG16L1.

ΔFBD mutant is fully active during ULK1-independent autophagy

The inability of ATG16L1ΔFBD to fully restore autophagy can be potentially due to an overall misfolding of the protein caused by the introduced truncation rather than its deficiency in binding FIP200. If so, our results in figure 5 would not be able to support a functional relevance of the FIP200–ATG16L1 interaction. Therefore, we sought to assess whether ATG16L1ΔFBD possesses any biological function that is independent of FIP200 and thereby the ULK1 complex. Recently, glucose starvation was suggested to induce autophagy in a ULK1 complex-independent manner which correlates with an increase in cellular ammonia levels and lack of mTORC1 inhibition27,28. We confirmed this by using ULK1 and ULK2 double knockout MEFs (ULK DKO) where glucose starvation induced LC3–II formation which is susceptible to lysosomal degradation (Fig. 6a). Meanwhile, we observed that glucose starvation did not induce ULK1 dephosphorylation on residues that are dephosphorylated during amino acid starvation29 (Fig. 6b). Interestingly, we when reconstituted Atg16l1−/− with full length ATG16L1 or ATG16L1ΔFBD comparable LC3–I to LC3–II conversion upon glucose starvation were exhibited whereas the absence of ATG16L1 expression completely abolished LC3 conjugation (Fig. 6c) indicating that ATG16L1ΔFBD is fully functional during such ULK1 complex-independent autophagy. Similarly, we observed comparable GFP–LC3 punctate structure formation between full length ATG16L1 and ATG16L1ΔFBD expressing cells (Fig. 6d). Overall, these analyses suggest that, unlike ULK1 complex-dependent autophagy induced by amino acid starvation, ULK1 complex-independent autophagy induced by glucose starvation does not require FIP200 binding to ATG16L1.

DISCUSSIONIn this study we provide evidence for a direct interaction between the ULK1 complex component, FIP200, and the ATG5 complex component, ATG16L1. This interaction is specifically required for ULK1 complex-dependent autophagy, thus providing mechanistic insights into how the ULK1 complex communicates with other ATG complexes, such as the ATG5 complex. Meanwhile the ATG5 complex (including its component ATG16L1) belongs to the essential core autophagy machinery, the ULK1 complex appears to only mediate autophagy induced by certain specific triggers. The biochemical LC3 conjugation reaction is intact in cells with genetic deletion of ULK130,31 and LC3–II formation takes place at basal levels in cells depleted of the ULK1 complex components22,32. Indeed, the ULK1 complex, but not the ATG5 complex, is dispensable for glucose starvation-induced autophagy27.Could ATG16L1 be a converging point mediating other upstream autophagy signals in addition to those through ULK1? This would be an interesting hypothesis to consider. It is clear, however, that the FBD domain of ATG16L1 is not necessarily responsible for all other signals, because ATG16L1ΔFBD is functionally intact during glucose starvation-induced autophagy. Another recognizable structure in ATG16L1 is its C-terminal WD40 repeats, which is a versatile protein-protein interaction domain and is not present in yeast ATG16. Although not required for autophagy triggered by amino acid starvation, it is possible that this region mediates certain other autophagy signals. If so, then the C-terminal region of ATG16L1 enables the mammalian autophagy pathway to sense more diverse and complex signals compared to its yeast counterpart. On the other hand, it is also possible that the WD40 repeats in mammalian ATG16L1 might only be relevant to non-autophagy processes10,33. In this sense, the potential pathological role of the Crohn’s disease-associated ATG16L1 mutation (T300A), which lies within the WD40 repeats, during both autophagy-related and non-autophagy-related processes should be explored34,35.A detailed comparison of ATG16L1 with its homolog ATG16L2 should also shed light on the function of ATG16L1 during autophagy. Unlike ATG16L1, ATG16L2 is unable to support autophagy or localize to the phagophore structures despite its ability to bind ATG5, self-oligomerize and form a large protein complex with the ATG5–12 complex26. ATG16L1 from various vertebrate species all possess highly conserved FBD regions, whereas ATG16L2 lacks this domain. Consistently, we found ATG16L2 failed to interact with FIP200. However, whether the lack of FBD renders ATG16L2 inactive in autophagy is unclear, as ATG16L1 with its FBD deleted can still mediate ULK1 complex-independent autophagy. In addition, neither yeast nor C. elegans ATG16 have this domain (there is no true FIP200 homolog in these organisms either). Furthermore, previous in vitro biochemical studies suggest that yeast ATG16 is not required for the E3-like activity of the ATG5–12 during the conjugation of ATG8 (the yeast homologue of LC3) to PE36. Thus in mammalian cells, the difference between ATG16L1 and ATG16L2 in autophagy does not appear to be due to their differential influence on the E3-like enzymatic activity of the ATG5 complex. As such, the exact structural and biochemical basis that renders ATG16L1 but not ATG16L2 to be an essential functional component in autophagy has yet to be defined.In conclusion, this study has uncovered a novel functional interaction between two upstream ATG complexes, and demonstrated that ATG16L1 is not only an essential structural component of the ATG5 complex but also a signaling protein that can mediate specific upstream signals during autophagy such as those transduced by the ULK1 complex.

In this study we provide evidence for a direct interaction between the ULK1 complex component, FIP200, and the ATG5 complex component, ATG16L1. This interaction is specifically required for ULK1 complex-dependent autophagy, thus providing mechanistic insights into how the ULK1 complex communicates with other ATG complexes, such as the ATG5 complex. Meanwhile the ATG5 complex (including its component ATG16L1) belongs to the essential core autophagy machinery, the ULK1 complex appears to only mediate autophagy induced by certain specific triggers. The biochemical LC3 conjugation reaction is intact in cells with genetic deletion of ULK130,31 and LC3–II formation takes place at basal levels in cells depleted of the ULK1 complex components22,32. Indeed, the ULK1 complex, but not the ATG5 complex, is dispensable for glucose starvation-induced autophagy27.

Could ATG16L1 be a converging point mediating other upstream autophagy signals in addition to those through ULK1? This would be an interesting hypothesis to consider. It is clear, however, that the FBD domain of ATG16L1 is not necessarily responsible for all other signals, because ATG16L1ΔFBD is functionally intact during glucose starvation-induced autophagy. Another recognizable structure in ATG16L1 is its C-terminal WD40 repeats, which is a versatile protein-protein interaction domain and is not present in yeast ATG16. Although not required for autophagy triggered by amino acid starvation, it is possible that this region mediates certain other autophagy signals. If so, then the C-terminal region of ATG16L1 enables the mammalian autophagy pathway to sense more diverse and complex signals compared to its yeast counterpart. On the other hand, it is also possible that the WD40 repeats in mammalian ATG16L1 might only be relevant to non-autophagy processes10,33. In this sense, the potential pathological role of the Crohn’s disease-associated ATG16L1 mutation (T300A), which lies within the WD40 repeats, during both autophagy-related and non-autophagy-related processes should be explored34,35.

A detailed comparison of ATG16L1 with its homolog ATG16L2 should also shed light on the function of ATG16L1 during autophagy. Unlike ATG16L1, ATG16L2 is unable to support autophagy or localize to the phagophore structures despite its ability to bind ATG5, self-oligomerize and form a large protein complex with the ATG5–12 complex26. ATG16L1 from various vertebrate species all possess highly conserved FBD regions, whereas ATG16L2 lacks this domain. Consistently, we found ATG16L2 failed to interact with FIP200. However, whether the lack of FBD renders ATG16L2 inactive in autophagy is unclear, as ATG16L1 with its FBD deleted can still mediate ULK1 complex-independent autophagy. In addition, neither yeast nor C. elegans ATG16 have this domain (there is no true FIP200 homolog in these organisms either). Furthermore, previous in vitro biochemical studies suggest that yeast ATG16 is not required for the E3-like activity of the ATG5–12 during the conjugation of ATG8 (the yeast homologue of LC3) to PE36. Thus in mammalian cells, the difference between ATG16L1 and ATG16L2 in autophagy does not appear to be due to their differential influence on the E3-like enzymatic activity of the ATG5 complex. As such, the exact structural and biochemical basis that renders ATG16L1 but not ATG16L2 to be an essential functional component in autophagy has yet to be defined.

In conclusion, this study has uncovered a novel functional interaction between two upstream ATG complexes, and demonstrated that ATG16L1 is not only an essential structural component of the ATG5 complex but also a signaling protein that can mediate specific upstream signals during autophagy such as those transduced by the ULK1 complex.

ONLINE METHODSCell culture and treatmentsWild type Mouse Embryonic Fibroblasts (MEFs), Atg3 knockout MEFs (Atg3−/−), Atg16l1−/−, Atg5−/−, Atg13−/−, ULK1 and ULK2 Double knockout MEFs (ULK DKO) and 293T cells were cultured in DMEM supplemented with 10% FBS, L-Glutamine (2 mM), Pencillin (10 Units/mL) and Streptomycin (0.1 mg/ml). Stable overexpression of GFP-LC3 or F-S-ATG proteins was obtained by retroviral infection using pBabe expression plasmid followed by blasticidin or puromycin selection. For transient expression, Lipofectamine 2000 (Invitrogen) was used according to the manufacturer’s instructions.For amino acid starvation experiments, cells were grown in DMEM lacking amino acids and serum typically for 2 hrs prior to harvesting. Glucose starvation was performed by culturing cells in DMEM lacking glucose and sodium pyruvate, supplemented with 10% dialyzed FBS and glutamine for 20 hrs. Control cells were also grown in 10% dialyzed FBS and glutamine but in the presence of glucose and sodium pyruvate.Antibodies and reagentsFor Western blot analysis, the following antibodies were used: anti-LC3 (1:2000, Sigma, #L7543); anti-Actin (1:5000, Sigma, #A5316); anti-ULK1 (1:2000, Sigma, #A7481); anti-ATG13 (1:3000, Sigma, SAB4200100); anti-ATG7 (1:2000, Santa Cruz, clone H300); anti-p62/SQSTM1 (1:10,000, MBL, #PM045); anti-ATG16L1 (1:3000, MBL, clone 1F12); anti-α-tubulin (1:1000, Calbiochem, #CP06); anti-S tag (1:1000, Novagen); anti-ATG5 (1:1000, Sigma, #A0731); anti-FIP200 (1:1000, ProteinTech, #10043-2-AP); anti-myc tag (1:1000, Santa Cruz, clone 9E10); anti-T7 tag HRP (1:5000, Novagen); anti-ATG16L2 (1:1000, Abgent, #K292); anti-phospho-p70 S6 Kinase (1:1000, p-p70S6K, Cell Signaling, #9205); control rabbit IgG (Millipore, 12–370). The anti-β-integrin antibody (1:1000) is a generous gift from Dr Filippo Giancotti; phospho-specific antibodies against ULK1 (p-757 and p-637, 1:1000) were described previously29.Bafilomycin A1 (Baf A1, inhibitor of lysosome degradation) was purchased from Sigma and used at a final concentration of 20 nM. Torin 1 (an mTOR specific inhibitor) was purchase from Tocris Bioscience and used at a final concentration of 1 μM.PlasmidsFlag-S-tagged proteins (F-S-) were cloned into pBabe-F-S retroviral vector using SalI cloning sites. ATG16L1 full length, fragments and mutants were generated based on mouse ATG16L1 using the following forward primers: full length (amino acids 1–623): 5′ GCAGCAGTCGACATGTCGTCGGGCCTGCGCGC; Δ1 (amino acids 39–623): 5′ GCAGCAGTCGACATGTATACCAAGTTGCTGGAAAAG; Δ2 (amino acids 120–623): 5′ GCAGCAGTCGACATGAAGGACAAGGAGATACAGATG; Δ3 (amino acids 206–623): 5′ GCAGCAGTCGACATGAATGCAGAGAATGAGAAGGAC; Δ4 (amino acids 336–623): 5′ GCAGCAGTCGACATGGCGCATGACGGAGAGGTCAAC. The above constructs used the following common reverse primer: 5′ CTTAAGTCGACTCAAGGCTGTGCCCACAGCAC. The ΔWD40 (amino acids 1–335): reverse primer: 5 ′ CCGAAGTCGACTCAATCGAAGACATACGAGGCAGTAG. The following deletion mutants were obtained by two-step PCR using the following forward primers: Δ229–242: 5′ AAGGAGCTTGCAGAAGCAGCAATTGTGGATGAGACCTCA; Δ182–242: GCCCTAGAAGAGAAACTGAGGATTGTGGATGAGACCTCA. C-terminally tagged GFP-ATG16L1 was cloned into pBabe vector using MfeI and SalI sites. The myc-tagged FIP200 was expressed from pCMV vector as described previously19. ATG14 was cloned into pBabe-F-S vector using SalI/MfeI restriction sites. ATG5 and the ATG5 K130R mutant (unable to conjugate to ATG12) were expressed from pCDNA3.1 with HA-FLAG-S-tags cloned using BamHI/XbaI restriction sites.Tandem affinity purificationCytosolic and membrane fractions were obtained using the subcellular fractionation kit (Thermo Scientific). Membrane fractions were obtained from 18 X 15 cm plates of Atg3−/− stably expressing Flag-S-tagged ATG16L1 or ATG14 (F-S-ATG16L1 or F-S-ATG14, respectively) starved for 2 hrs in amino acid deficient media. Membrane fractions were then spun at 60,000g for 2 hrs. Pre-cleared lysates were incubated overnight with Flag-beads (Sigma) at 4°C followed 3 washes in NP-40 buffer (100mM NaCl, 25mM Hepes pH7.5, 1.5 mM MgCl2, 1 mM EDTA, 1.5 mM β-ME and 0.5% NP-40, supplemented with protease inhibitor cocktail and proteasome inhibitor MG132, Sigma). Bound complexes were then eluted in NP-40 buffer containing 200 μg/ml 3XFLAG peptide (Sigma) for 30 min at 4 °C with rotation. Eluates were further incubated with S protein agarose (Novagen) for 6 hrs at 4 °C with rotation. Following four washes in NP-40 buffer, samples were eluted in SDS-sample buffer and protein complexes were analyzed by SDS-PAGE, silver staining and mass spectrometry.Binding assaysCell lysates were obtained from 293T or MEF cells grown in 10 cm plates and harvested by direct lysing in NP-40 buffer (except that NaCl concentration was increased to 150 mM). Cell lysates were cleared by spinning at 12,000g for 5 min at 4 °C, and incubated with either S protein agarose (Novagen) for 5 hrs at 4 °C or with anti-myc antibodies (Santa Cruz, clone 9E10) for 3 hrs followed by protein G agarose for an additional 2 hrs. Beads were then washed 3 times with NP-40 buffer and bound proteins were analyzed by SDS-PAGE and Western blot analyses. For endogenous co-immunoprecipitation assays, Atg3−/− were harvest as above and cleared cell lysates were incubated with 0.3 μg FIP200 antibodies or control rabbit IgG overnight at 4 °C followed by protein G agarose for an additional 3 hrs. Beads were washed 4 times with NP-40 buffer and protein complexes were analyzed by Western blot analysis.Recombinant proteins purification was performed as described previously19,37,38. Briefly, T7-tagged recombinant ATG16L1 and ATG3 were purified from BL21 E. coli cells, whereas, recombinant FIP200, ATG13 and ATG7 were purified from SF-9 cells19,37,38. For recombinant protein binding, recombinant ATG16L1 was incubated with anti-ATG16L1 antibodies and protein G for 3 hrs at 4 °C with rotation followed by 2 washes in NP-40 buffer. Beads were then incubated with recombinant ATG13, ATG7 or ATG3 for 1 hr at 4 °C followed by 3 washes in NP-40 buffer. As negative controls, recombinant proteins were incubated with protein G pre-incubated with anti-ATG16L1 antibody but in the absence of recombinant ATG16L1. Bound proteins were analyzed by SDS-PAGE and Western blot analysis using anti-T7-HRP (to detect ATG3), anti-ATG16L1, anti-ATG13 or anti-ATG7 antibodies. Alternatively, S-tagged ATG16L1 expressed in wild type or Atg5−/− was precipitated using S protein agarose, followed by extensive washings and incubation with recombinant FIP200 as above.MicroscopyFor fluorescence analysis, cells stably were grown on glass coverslips in a 6-well plate. 24 hrs later, cells were either left untreated or treated as indicated. Coverslips were then fixed with 3.7% Paraformaldehyde in 20 mM HEPES pH7.5 for 30 min at room temperature followed by permeabilization in 0.1% triton in PBS for 5 min. Slides were then incubated in primary antibodies in blocking buffer (PBS supplemented with 1% BSA) at 37 °C for 2 hrs followed by incubation with Alexa Fluor secondary antibodies (Invitrogen) for 30 min at room temperature. Following extensive washings, coverslips were then mounted on microscope slides and visualized under Nikon Eclipse Ti-U Confocal Microscope using a 60X magnification objective. Images were acquired using Nikon EZ-C1 image acquisition software and edited using Photoshop.

ONLINE METHODS

Cell culture and treatmentsWild type Mouse Embryonic Fibroblasts (MEFs), Atg3 knockout MEFs (Atg3−/−), Atg16l1−/−, Atg5−/−, Atg13−/−, ULK1 and ULK2 Double knockout MEFs (ULK DKO) and 293T cells were cultured in DMEM supplemented with 10% FBS, L-Glutamine (2 mM), Pencillin (10 Units/mL) and Streptomycin (0.1 mg/ml). Stable overexpression of GFP-LC3 or F-S-ATG proteins was obtained by retroviral infection using pBabe expression plasmid followed by blasticidin or puromycin selection. For transient expression, Lipofectamine 2000 (Invitrogen) was used according to the manufacturer’s instructions.For amino acid starvation experiments, cells were grown in DMEM lacking amino acids and serum typically for 2 hrs prior to harvesting. Glucose starvation was performed by culturing cells in DMEM lacking glucose and sodium pyruvate, supplemented with 10% dialyzed FBS and glutamine for 20 hrs. Control cells were also grown in 10% dialyzed FBS and glutamine but in the presence of glucose and sodium pyruvate.

Cell culture and treatments

Wild type Mouse Embryonic Fibroblasts (MEFs), Atg3 knockout MEFs (Atg3−/−), Atg16l1−/−, Atg5−/−, Atg13−/−, ULK1 and ULK2 Double knockout MEFs (ULK DKO) and 293T cells were cultured in DMEM supplemented with 10% FBS, L-Glutamine (2 mM), Pencillin (10 Units/mL) and Streptomycin (0.1 mg/ml). Stable overexpression of GFP-LC3 or F-S-ATG proteins was obtained by retroviral infection using pBabe expression plasmid followed by blasticidin or puromycin selection. For transient expression, Lipofectamine 2000 (Invitrogen) was used according to the manufacturer’s instructions.

For amino acid starvation experiments, cells were grown in DMEM lacking amino acids and serum typically for 2 hrs prior to harvesting. Glucose starvation was performed by culturing cells in DMEM lacking glucose and sodium pyruvate, supplemented with 10% dialyzed FBS and glutamine for 20 hrs. Control cells were also grown in 10% dialyzed FBS and glutamine but in the presence of glucose and sodium pyruvate.

Antibodies and reagentsFor Western blot analysis, the following antibodies were used: anti-LC3 (1:2000, Sigma, #L7543); anti-Actin (1:5000, Sigma, #A5316); anti-ULK1 (1:2000, Sigma, #A7481); anti-ATG13 (1:3000, Sigma, SAB4200100); anti-ATG7 (1:2000, Santa Cruz, clone H300); anti-p62/SQSTM1 (1:10,000, MBL, #PM045); anti-ATG16L1 (1:3000, MBL, clone 1F12); anti-α-tubulin (1:1000, Calbiochem, #CP06); anti-S tag (1:1000, Novagen); anti-ATG5 (1:1000, Sigma, #A0731); anti-FIP200 (1:1000, ProteinTech, #10043-2-AP); anti-myc tag (1:1000, Santa Cruz, clone 9E10); anti-T7 tag HRP (1:5000, Novagen); anti-ATG16L2 (1:1000, Abgent, #K292); anti-phospho-p70 S6 Kinase (1:1000, p-p70S6K, Cell Signaling, #9205); control rabbit IgG (Millipore, 12–370). The anti-β-integrin antibody (1:1000) is a generous gift from Dr Filippo Giancotti; phospho-specific antibodies against ULK1 (p-757 and p-637, 1:1000) were described previously29.Bafilomycin A1 (Baf A1, inhibitor of lysosome degradation) was purchased from Sigma and used at a final concentration of 20 nM. Torin 1 (an mTOR specific inhibitor) was purchase from Tocris Bioscience and used at a final concentration of 1 μM.

Antibodies and reagents

For Western blot analysis, the following antibodies were used: anti-LC3 (1:2000, Sigma, #L7543); anti-Actin (1:5000, Sigma, #A5316); anti-ULK1 (1:2000, Sigma, #A7481); anti-ATG13 (1:3000, Sigma, SAB4200100); anti-ATG7 (1:2000, Santa Cruz, clone H300); anti-p62/SQSTM1 (1:10,000, MBL, #PM045); anti-ATG16L1 (1:3000, MBL, clone 1F12); anti-α-tubulin (1:1000, Calbiochem, #CP06); anti-S tag (1:1000, Novagen); anti-ATG5 (1:1000, Sigma, #A0731); anti-FIP200 (1:1000, ProteinTech, #10043-2-AP); anti-myc tag (1:1000, Santa Cruz, clone 9E10); anti-T7 tag HRP (1:5000, Novagen); anti-ATG16L2 (1:1000, Abgent, #K292); anti-phospho-p70 S6 Kinase (1:1000, p-p70S6K, Cell Signaling, #9205); control rabbit IgG (Millipore, 12–370). The anti-β-integrin antibody (1:1000) is a generous gift from Dr Filippo Giancotti; phospho-specific antibodies against ULK1 (p-757 and p-637, 1:1000) were described previously29.

Bafilomycin A1 (Baf A1, inhibitor of lysosome degradation) was purchased from Sigma and used at a final concentration of 20 nM. Torin 1 (an mTOR specific inhibitor) was purchase from Tocris Bioscience and used at a final concentration of 1 μM.

PlasmidsFlag-S-tagged proteins (F-S-) were cloned into pBabe-F-S retroviral vector using SalI cloning sites. ATG16L1 full length, fragments and mutants were generated based on mouse ATG16L1 using the following forward primers: full length (amino acids 1–623): 5′ GCAGCAGTCGACATGTCGTCGGGCCTGCGCGC; Δ1 (amino acids 39–623): 5′ GCAGCAGTCGACATGTATACCAAGTTGCTGGAAAAG; Δ2 (amino acids 120–623): 5′ GCAGCAGTCGACATGAAGGACAAGGAGATACAGATG; Δ3 (amino acids 206–623): 5′ GCAGCAGTCGACATGAATGCAGAGAATGAGAAGGAC; Δ4 (amino acids 336–623): 5′ GCAGCAGTCGACATGGCGCATGACGGAGAGGTCAAC. The above constructs used the following common reverse primer: 5′ CTTAAGTCGACTCAAGGCTGTGCCCACAGCAC. The ΔWD40 (amino acids 1–335): reverse primer: 5 ′ CCGAAGTCGACTCAATCGAAGACATACGAGGCAGTAG. The following deletion mutants were obtained by two-step PCR using the following forward primers: Δ229–242: 5′ AAGGAGCTTGCAGAAGCAGCAATTGTGGATGAGACCTCA; Δ182–242: GCCCTAGAAGAGAAACTGAGGATTGTGGATGAGACCTCA. C-terminally tagged GFP-ATG16L1 was cloned into pBabe vector using MfeI and SalI sites. The myc-tagged FIP200 was expressed from pCMV vector as described previously19. ATG14 was cloned into pBabe-F-S vector using SalI/MfeI restriction sites. ATG5 and the ATG5 K130R mutant (unable to conjugate to ATG12) were expressed from pCDNA3.1 with HA-FLAG-S-tags cloned using BamHI/XbaI restriction sites.

Flag-S-tagged proteins (F-S-) were cloned into pBabe-F-S retroviral vector using SalI cloning sites. ATG16L1 full length, fragments and mutants were generated based on mouse ATG16L1 using the following forward primers: full length (amino acids 1–623): 5′ GCAGCAGTCGACATGTCGTCGGGCCTGCGCGC; Δ1 (amino acids 39–623): 5′ GCAGCAGTCGACATGTATACCAAGTTGCTGGAAAAG; Δ2 (amino acids 120–623): 5′ GCAGCAGTCGACATGAAGGACAAGGAGATACAGATG; Δ3 (amino acids 206–623): 5′ GCAGCAGTCGACATGAATGCAGAGAATGAGAAGGAC; Δ4 (amino acids 336–623): 5′ GCAGCAGTCGACATGGCGCATGACGGAGAGGTCAAC. The above constructs used the following common reverse primer: 5′ CTTAAGTCGACTCAAGGCTGTGCCCACAGCAC. The ΔWD40 (amino acids 1–335): reverse primer: 5 ′ CCGAAGTCGACTCAATCGAAGACATACGAGGCAGTAG. The following deletion mutants were obtained by two-step PCR using the following forward primers: Δ229–242: 5′ AAGGAGCTTGCAGAAGCAGCAATTGTGGATGAGACCTCA; Δ182–242: GCCCTAGAAGAGAAACTGAGGATTGTGGATGAGACCTCA. C-terminally tagged GFP-ATG16L1 was cloned into pBabe vector using MfeI and SalI sites. The myc-tagged FIP200 was expressed from pCMV vector as described previously19. ATG14 was cloned into pBabe-F-S vector using SalI/MfeI restriction sites. ATG5 and the ATG5 K130R mutant (unable to conjugate to ATG12) were expressed from pCDNA3.1 with HA-FLAG-S-tags cloned using BamHI/XbaI restriction sites.

Tandem affinity purificationCytosolic and membrane fractions were obtained using the subcellular fractionation kit (Thermo Scientific). Membrane fractions were obtained from 18 X 15 cm plates of Atg3−/− stably expressing Flag-S-tagged ATG16L1 or ATG14 (F-S-ATG16L1 or F-S-ATG14, respectively) starved for 2 hrs in amino acid deficient media. Membrane fractions were then spun at 60,000g for 2 hrs. Pre-cleared lysates were incubated overnight with Flag-beads (Sigma) at 4°C followed 3 washes in NP-40 buffer (100mM NaCl, 25mM Hepes pH7.5, 1.5 mM MgCl2, 1 mM EDTA, 1.5 mM β-ME and 0.5% NP-40, supplemented with protease inhibitor cocktail and proteasome inhibitor MG132, Sigma). Bound complexes were then eluted in NP-40 buffer containing 200 μg/ml 3XFLAG peptide (Sigma) for 30 min at 4 °C with rotation. Eluates were further incubated with S protein agarose (Novagen) for 6 hrs at 4 °C with rotation. Following four washes in NP-40 buffer, samples were eluted in SDS-sample buffer and protein complexes were analyzed by SDS-PAGE, silver staining and mass spectrometry.

Tandem affinity purification

Cytosolic and membrane fractions were obtained using the subcellular fractionation kit (Thermo Scientific). Membrane fractions were obtained from 18 X 15 cm plates of Atg3−/− stably expressing Flag-S-tagged ATG16L1 or ATG14 (F-S-ATG16L1 or F-S-ATG14, respectively) starved for 2 hrs in amino acid deficient media. Membrane fractions were then spun at 60,000g for 2 hrs. Pre-cleared lysates were incubated overnight with Flag-beads (Sigma) at 4°C followed 3 washes in NP-40 buffer (100mM NaCl, 25mM Hepes pH7.5, 1.5 mM MgCl2, 1 mM EDTA, 1.5 mM β-ME and 0.5% NP-40, supplemented with protease inhibitor cocktail and proteasome inhibitor MG132, Sigma). Bound complexes were then eluted in NP-40 buffer containing 200 μg/ml 3XFLAG peptide (Sigma) for 30 min at 4 °C with rotation. Eluates were further incubated with S protein agarose (Novagen) for 6 hrs at 4 °C with rotation. Following four washes in NP-40 buffer, samples were eluted in SDS-sample buffer and protein complexes were analyzed by SDS-PAGE, silver staining and mass spectrometry.

Binding assaysCell lysates were obtained from 293T or MEF cells grown in 10 cm plates and harvested by direct lysing in NP-40 buffer (except that NaCl concentration was increased to 150 mM). Cell lysates were cleared by spinning at 12,000g for 5 min at 4 °C, and incubated with either S protein agarose (Novagen) for 5 hrs at 4 °C or with anti-myc antibodies (Santa Cruz, clone 9E10) for 3 hrs followed by protein G agarose for an additional 2 hrs. Beads were then washed 3 times with NP-40 buffer and bound proteins were analyzed by SDS-PAGE and Western blot analyses. For endogenous co-immunoprecipitation assays, Atg3−/− were harvest as above and cleared cell lysates were incubated with 0.3 μg FIP200 antibodies or control rabbit IgG overnight at 4 °C followed by protein G agarose for an additional 3 hrs. Beads were washed 4 times with NP-40 buffer and protein complexes were analyzed by Western blot analysis.Recombinant proteins purification was performed as described previously19,37,38. Briefly, T7-tagged recombinant ATG16L1 and ATG3 were purified from BL21 E. coli cells, whereas, recombinant FIP200, ATG13 and ATG7 were purified from SF-9 cells19,37,38. For recombinant protein binding, recombinant ATG16L1 was incubated with anti-ATG16L1 antibodies and protein G for 3 hrs at 4 °C with rotation followed by 2 washes in NP-40 buffer. Beads were then incubated with recombinant ATG13, ATG7 or ATG3 for 1 hr at 4 °C followed by 3 washes in NP-40 buffer. As negative controls, recombinant proteins were incubated with protein G pre-incubated with anti-ATG16L1 antibody but in the absence of recombinant ATG16L1. Bound proteins were analyzed by SDS-PAGE and Western blot analysis using anti-T7-HRP (to detect ATG3), anti-ATG16L1, anti-ATG13 or anti-ATG7 antibodies. Alternatively, S-tagged ATG16L1 expressed in wild type or Atg5−/− was precipitated using S protein agarose, followed by extensive washings and incubation with recombinant FIP200 as above.

Binding assays

Cell lysates were obtained from 293T or MEF cells grown in 10 cm plates and harvested by direct lysing in NP-40 buffer (except that NaCl concentration was increased to 150 mM). Cell lysates were cleared by spinning at 12,000g for 5 min at 4 °C, and incubated with either S protein agarose (Novagen) for 5 hrs at 4 °C or with anti-myc antibodies (Santa Cruz, clone 9E10) for 3 hrs followed by protein G agarose for an additional 2 hrs. Beads were then washed 3 times with NP-40 buffer and bound proteins were analyzed by SDS-PAGE and Western blot analyses. For endogenous co-immunoprecipitation assays, Atg3−/− were harvest as above and cleared cell lysates were incubated with 0.3 μg FIP200 antibodies or control rabbit IgG overnight at 4 °C followed by protein G agarose for an additional 3 hrs. Beads were washed 4 times with NP-40 buffer and protein complexes were analyzed by Western blot analysis.

Recombinant proteins purification was performed as described previously19,37,38. Briefly, T7-tagged recombinant ATG16L1 and ATG3 were purified from BL21 E. coli cells, whereas, recombinant FIP200, ATG13 and ATG7 were purified from SF-9 cells19,37,38. For recombinant protein binding, recombinant ATG16L1 was incubated with anti-ATG16L1 antibodies and protein G for 3 hrs at 4 °C with rotation followed by 2 washes in NP-40 buffer. Beads were then incubated with recombinant ATG13, ATG7 or ATG3 for 1 hr at 4 °C followed by 3 washes in NP-40 buffer. As negative controls, recombinant proteins were incubated with protein G pre-incubated with anti-ATG16L1 antibody but in the absence of recombinant ATG16L1. Bound proteins were analyzed by SDS-PAGE and Western blot analysis using anti-T7-HRP (to detect ATG3), anti-ATG16L1, anti-ATG13 or anti-ATG7 antibodies. Alternatively, S-tagged ATG16L1 expressed in wild type or Atg5−/− was precipitated using S protein agarose, followed by extensive washings and incubation with recombinant FIP200 as above.

MicroscopyFor fluorescence analysis, cells stably were grown on glass coverslips in a 6-well plate. 24 hrs later, cells were either left untreated or treated as indicated. Coverslips were then fixed with 3.7% Paraformaldehyde in 20 mM HEPES pH7.5 for 30 min at room temperature followed by permeabilization in 0.1% triton in PBS for 5 min. Slides were then incubated in primary antibodies in blocking buffer (PBS supplemented with 1% BSA) at 37 °C for 2 hrs followed by incubation with Alexa Fluor secondary antibodies (Invitrogen) for 30 min at room temperature. Following extensive washings, coverslips were then mounted on microscope slides and visualized under Nikon Eclipse Ti-U Confocal Microscope using a 60X magnification objective. Images were acquired using Nikon EZ-C1 image acquisition software and edited using Photoshop.

For fluorescence analysis, cells stably were grown on glass coverslips in a 6-well plate. 24 hrs later, cells were either left untreated or treated as indicated. Coverslips were then fixed with 3.7% Paraformaldehyde in 20 mM HEPES pH7.5 for 30 min at room temperature followed by permeabilization in 0.1% triton in PBS for 5 min. Slides were then incubated in primary antibodies in blocking buffer (PBS supplemented with 1% BSA) at 37 °C for 2 hrs followed by incubation with Alexa Fluor secondary antibodies (Invitrogen) for 30 min at room temperature. Following extensive washings, coverslips were then mounted on microscope slides and visualized under Nikon Eclipse Ti-U Confocal Microscope using a 60X magnification objective. Images were acquired using Nikon EZ-C1 image acquisition software and edited using Photoshop.

Supplementary Material1

Supplementary Material
