---
pmid: '28666008'
title: The whole set of the constitutive promoters recognized by four minor sigma
  subunits of Escherichia coli RNA polymerase.
authors:
- Shimada T
- Tanaka K
- Ishihama A
journal: PLoS One
year: '2017'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC5493296
doi: 10.1371/journal.pone.0179181
---

# The whole set of the constitutive promoters recognized by four minor sigma subunits of Escherichia coli RNA polymerase.
**Authors:** Shimada T, Tanaka K, Ishihama A
**Journal:** PLoS One (2017)
**DOI:** [10.1371/journal.pone.0179181](https://doi.org/10.1371/journal.pone.0179181)
**PMC:** [PMC5493296](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5493296/)

## Abstract

1. PLoS One. 2017 Jun 30;12(6):e0179181. doi: 10.1371/journal.pone.0179181. 
eCollection 2017.

The whole set of the constitutive promoters recognized by four minor sigma 
subunits of Escherichia coli RNA polymerase.

Shimada T(1)(2), Tanaka K(2), Ishihama A(1).

Author information:
(1)Research Center for Micro-Nano Technology, Hosei University, Koganei, Tokyo, 
Japan.
(2)Laboratory for Chemistry and Life Science, Institute of Innovative Research, 
Tokyo Institute of Technology, Nagatsuda, Yokohama, Japan.

The promoter selectivity of Escherichia coli RNA polymerase (RNAP) is determined 
by the sigma subunit. The model prokaryote Escherichia coli K-12 contains seven 
species of the sigma subunit, each recognizing a specific set of promoters. For 
identification of the "constitutive promoters" that are recognized by each RNAP 
holoenzyme alone in the absence of other supporting factors, we have performed 
the genomic SELEX screening in vitro for their binding sites along the E. coli 
K-12 W3110 genome using each of the reconstituted RNAP holoenzymes and a 
collection of genome DNA segments of E. coli K-12. The whole set of constitutive 
promoters for each RNAP holoenzyme was then estimated based on the location of 
RNAP-binding sites. The first successful screening of the constitutive promoters 
was achieved for RpoD (σ70), the principal sigma for transcription of 
growth-related genes. As an extension, we performed in this study the screening 
of constitutive promoters for four minor sigma subunits, stationary-phase 
specific RpoS (σ38), heat-shock specific RpoH (σ32), flagellar-chemotaxis 
specific RpoF (σ28) and extra-cytoplasmic stress-response RpoE (σ24). The total 
number of constitutive promoters were: 129~179 for RpoS; 101~142 for RpoH; 34~41 
for RpoF; and 77~106 for RpoE. The list of constitutive promoters were compared 
with that of known promoters identified in vivo under various conditions and 
using varieties of E. coli strains, altogether allowing the estimation of 
"inducible promoters" in the presence of additional supporting factors.

DOI: 10.1371/journal.pone.0179181
PMCID: PMC5493296
PMID: 28666008 [Indexed for MEDLINE]

Conflict of interest statement: Competing Interests: The authors have declared 
that no competing interests exist.

## Full Text

IntroductionThe genome of Escherichia coli K-12, the most well-characterized model prokaryote, contains a total of more than 4,500 genes, which are transcribed by a single species of the RNA polymerase (RNAP). The intracellular concentration of RNAP is, however, approximately 2,000 molecules per genome, which is less than the total number of genes or operons [1–3]. The pattern of genome expression is therefore determined by the selective distribution of a limited number of RNAP within the genome [4,5]. For adaptation to stressful environments, the pattern of genome transcription is, however, altered by modulating the promoter selectivity of RNAP through two-step interaction with two groups of the regulatory factor, i.e., 7 species of the sigma factor with promoter recognition activity at the first step [5,6] and then approximately 300 species of the transcription factor (TF) including both protein and nucleotide factors at the second step [4,5,7,8]. For understanding the genome regulation at molecular level, therefore, three kinds of the basic knowledge are absolutely needed for both all the sigma and TF factors [8,9]: (1) the whole set of regulatory target promoters, genes or operons under the control of each regulatory factor; (2) the binding affinity of the test regulatory protein to target DNA; and (3) the intracellular concentrations of the functional forms of each regulatory protein [note that the activity of TF is often controlled by effector ligands or protein modification such as phosphorylation]. Once we get these three lines of knowledge, we will be able to predict the pattern of genome transcription.After the complete genome sequencing of E. coli K-12, its transcription pattern or transcriptome in vivo has been analyzed for various E. coli wild-type and mutant strains growing under various stress conditions, including niches within host animals, using modern technologies such as the microarray system [10,11]. The localization of RNAP and TFs on the genome was also analyzed by using ChIP-chip system [12–14]. More recently microarray was replaced by direct sequencing of RNAs [15–17] or mapping of transcription start sites [18,19]. These data are assembled in the databases such as RegulonDB [20,21] and EcoCyc [22,23]. The huge accumulation of background knowledge is absolutely needed for understanding the regulation mechanism of genome transcription as a whole in a single organism, and thus at this stage, E. coli is reassessed as the model organism. The binding sites of RNAP and TF identified in vivo using these modern techniques, however, do not represent the whole set of their binding sites because: i) their binding to regulatory sites is often interfered by other DNA-binding proteins, thereby masking their binding target sequences by antagonistic inhibitory proteins [8,9,24]; and ii) in the case of activator-dependent transcription, their binding to targets depends on the simultaneous presence of supporting factors [8,9,25]. Under the in vivo situations, therefore, it is in principle impossible to obtain the whole set of binding sites for both RNAP and TFs. In addition, the transcription-related data listed in the databases include different levels of accuracy. For instance, a number of TF-binding sites are estimated in silico relying on the consensus sequences that often include the inaccurate prediction. Another serious problem is originated from the use of various E. coli strains with different genetic background and of different culture conditions used in each experiment (for details see Discussion).In order to avoid the problems associated with these in vivo experiments, we then decided to employ the in vitro approaches. For identification of the binding sites of RNAP and TFs, we developed the Genomic SELEX system [26] and successfully employed for search of regulatory targets for a number of TFs [8,9]. We also employed the Genomic SELEX for mapping of promoters. As described in the previous report [27], we identified a total of 2,071 sites on the E. coli K-12 genome of binding of RNAP holoenzyme containing RpoD (σ70), the major sigma for transcription of most of the growth-related genes, and mapped the location of “constitutive promoters” that are recognized by RpoD holoenzyme alone in the absence of other DNA-binding proteins [Note that the “constitutive promoter” is defined as the promoter that is recognized by RNAP alone in the absence of supporting factors while the promoters that are detected only in vivo are defined as the “inducible promoters”, supposedly under the support of accessory regulatory factors].Besides this major house-keeping RpoD sigma (σ70), E. coli K-12 contains six alternative minor sigma factors, i.e., nitrogen-regulated gene-specific RpoN (σ54), stationary-phase nutrient-starvation specific RpoS (σ38), heat-shock response-specific RpoH (σ32), flagellar-chemotaxis specific RpoF (σ28), extra-cytoplasmic stress-response RpoE (σ24), and iron-starvation specific FecI (σ28) [4–6]. In this study, we identified the list of constitutive promoters for four minor sigma factors, RpoS, RpoH, RpoF and RpoE. Since RpoN sigma requires an additional TF such as NtrC for promoter binding, the set of promoters recognized by RpoN sigma differs depending on the species of collaborative TF. The list of promoters recognized by RpoN will be described elsewhere. On the other hand, FecI sigma is rather a unique sigma that recognizes only a specific target of the gene for fecA encoding transport of ferric citrate [28]. Thus, these two sigma factors, RpoN and FecI, are not included in this report. The list of constitutive promoters herein described provides the fundamental catalogs for the promoters recognized by the four minor sigma factors alone. The data described in this report will be deposited into TEC (Transcription Profile of Escherchia coli) database (https://shigen.nig.ac.jp/ecoli/tec/) [9]. The data of each minor sigma will be shown by ordering the sigma name (RpoS, RpoH, RpoF or RpoE) [https://shigen.nig.ac.jp/ecoli/tec/tfmap]. For details follow the instruction in TEC [9].

Introduction

The genome of Escherichia coli K-12, the most well-characterized model prokaryote, contains a total of more than 4,500 genes, which are transcribed by a single species of the RNA polymerase (RNAP). The intracellular concentration of RNAP is, however, approximately 2,000 molecules per genome, which is less than the total number of genes or operons [1–3]. The pattern of genome expression is therefore determined by the selective distribution of a limited number of RNAP within the genome [4,5]. For adaptation to stressful environments, the pattern of genome transcription is, however, altered by modulating the promoter selectivity of RNAP through two-step interaction with two groups of the regulatory factor, i.e., 7 species of the sigma factor with promoter recognition activity at the first step [5,6] and then approximately 300 species of the transcription factor (TF) including both protein and nucleotide factors at the second step [4,5,7,8]. For understanding the genome regulation at molecular level, therefore, three kinds of the basic knowledge are absolutely needed for both all the sigma and TF factors [8,9]: (1) the whole set of regulatory target promoters, genes or operons under the control of each regulatory factor; (2) the binding affinity of the test regulatory protein to target DNA; and (3) the intracellular concentrations of the functional forms of each regulatory protein [note that the activity of TF is often controlled by effector ligands or protein modification such as phosphorylation]. Once we get these three lines of knowledge, we will be able to predict the pattern of genome transcription.

After the complete genome sequencing of E. coli K-12, its transcription pattern or transcriptome in vivo has been analyzed for various E. coli wild-type and mutant strains growing under various stress conditions, including niches within host animals, using modern technologies such as the microarray system [10,11]. The localization of RNAP and TFs on the genome was also analyzed by using ChIP-chip system [12–14]. More recently microarray was replaced by direct sequencing of RNAs [15–17] or mapping of transcription start sites [18,19]. These data are assembled in the databases such as RegulonDB [20,21] and EcoCyc [22,23]. The huge accumulation of background knowledge is absolutely needed for understanding the regulation mechanism of genome transcription as a whole in a single organism, and thus at this stage, E. coli is reassessed as the model organism. The binding sites of RNAP and TF identified in vivo using these modern techniques, however, do not represent the whole set of their binding sites because: i) their binding to regulatory sites is often interfered by other DNA-binding proteins, thereby masking their binding target sequences by antagonistic inhibitory proteins [8,9,24]; and ii) in the case of activator-dependent transcription, their binding to targets depends on the simultaneous presence of supporting factors [8,9,25]. Under the in vivo situations, therefore, it is in principle impossible to obtain the whole set of binding sites for both RNAP and TFs. In addition, the transcription-related data listed in the databases include different levels of accuracy. For instance, a number of TF-binding sites are estimated in silico relying on the consensus sequences that often include the inaccurate prediction. Another serious problem is originated from the use of various E. coli strains with different genetic background and of different culture conditions used in each experiment (for details see Discussion).

In order to avoid the problems associated with these in vivo experiments, we then decided to employ the in vitro approaches. For identification of the binding sites of RNAP and TFs, we developed the Genomic SELEX system [26] and successfully employed for search of regulatory targets for a number of TFs [8,9]. We also employed the Genomic SELEX for mapping of promoters. As described in the previous report [27], we identified a total of 2,071 sites on the E. coli K-12 genome of binding of RNAP holoenzyme containing RpoD (σ70), the major sigma for transcription of most of the growth-related genes, and mapped the location of “constitutive promoters” that are recognized by RpoD holoenzyme alone in the absence of other DNA-binding proteins [Note that the “constitutive promoter” is defined as the promoter that is recognized by RNAP alone in the absence of supporting factors while the promoters that are detected only in vivo are defined as the “inducible promoters”, supposedly under the support of accessory regulatory factors].

Besides this major house-keeping RpoD sigma (σ70), E. coli K-12 contains six alternative minor sigma factors, i.e., nitrogen-regulated gene-specific RpoN (σ54), stationary-phase nutrient-starvation specific RpoS (σ38), heat-shock response-specific RpoH (σ32), flagellar-chemotaxis specific RpoF (σ28), extra-cytoplasmic stress-response RpoE (σ24), and iron-starvation specific FecI (σ28) [4–6]. In this study, we identified the list of constitutive promoters for four minor sigma factors, RpoS, RpoH, RpoF and RpoE. Since RpoN sigma requires an additional TF such as NtrC for promoter binding, the set of promoters recognized by RpoN sigma differs depending on the species of collaborative TF. The list of promoters recognized by RpoN will be described elsewhere. On the other hand, FecI sigma is rather a unique sigma that recognizes only a specific target of the gene for fecA encoding transport of ferric citrate [28]. Thus, these two sigma factors, RpoN and FecI, are not included in this report. The list of constitutive promoters herein described provides the fundamental catalogs for the promoters recognized by the four minor sigma factors alone. The data described in this report will be deposited into TEC (Transcription Profile of Escherchia coli) database (https://shigen.nig.ac.jp/ecoli/tec/) [9]. The data of each minor sigma will be shown by ordering the sigma name (RpoS, RpoH, RpoF or RpoE) [https://shigen.nig.ac.jp/ecoli/tec/tfmap]. For details follow the instruction in TEC [9].

ResultsGenomic SELEX screening for the constitutive promotersThe constitutive promoters are transcribed in vitro by the RNA polymerase holoenzyme alone in the absence of supporting factors. In order to identify the whole set of constitutive promoters on the entire genome of E. coli K-12 W3110, we performed a mass-screening in vitro of the whole set of sequences that are recognized by the reconstituted holoenzymes, each containing only one specific minor sigma factor. The sigma-free core enzyme was prepared by passing the purified RNA polymerase three times through phosphocellulose column chromatography in the presence of 5% glycerol, the stabilizer of holoenzyme complexes in the storage buffer [29]. The level of remaining sigma subunits was less than 0.1%, if any, as detected by both protein staining and immuno-staining against each of all seven species of sigma factors (RpoD, RpoN, RpoS, RpoH, RpoF, RpoE and FecI). The stoichiometry between core enzyme subunits was also checked by immune-staining with antibodies against the core subunits, RpoA, RpoB, RpoC and RpoZ. The holoenzymes fully saturated with each sigma subunit were reconstituted by mixing this sigma-free core enzyme and 4-fold molar excess of purified sigma factors, RpoS, RpoH, RpoF and RpoE. Since these sigma subunits alone are unable to bind to promoter DNA, the presence of excess sigma does not interfere with the function of RNAP holoenzymes. For the identification of DNA sequences that are recognized by each holoenzyme, we employed the Genomic SELEX screening system [26], in which a library of E. coli genome DNA fragments of 200–300 bp in length was used instead of synthetic oligonucleotides with all possible sequences used in the original SELEX method [30–32].The multi-copy plasmid library of 200–300 bp-long random DNA fragments was constructed from the E. coli K-12 W3110 genome [26]. The library used in this study contained 6.5-fold molar excess of the entire genome, and thus a single and the same sequence might be included in 6 different overlapping segments on average, thereby increasing the resolution of mapping of SELEX fragments. In each experiment of Genomic SELEX screening, the mixture of genome DNA fragments, which was regenerated by PCR from the genome DNA library, was mixed with 2-fold molar excess of the reconstituted each RNAP holoenzyme, and subjected to Genomic SELEX screening. DNA-holoenzyme complexes formed were recovered using the anti-RpoC antibody, which gave the highest level of RNAP recovery among all the anti-core subunit antibodies. RNA polymerase-associated DNA was isolated from the antibody precipitates, amplified by PCR, and subjected to next cycles of SELEX. After repeated SELEX screening, the final products of holoenzyme-bound DNA fragments were subjected to mapping on the genome using a DNA tilling microarray (Oxford Gene Technology, Oxford, UK) [14]. The binding intensity was measured as the ratio of holoenzyme-bound DNA labeled by Cy3 against original library DNA labeled by Cy5 on an array and plotted along E. coli genome about each holoenzyme. On the DNA tilling array used, the 60 b-long probes are aligned along the E. coli genome at 105 bp-intervals, and therefore approximately 300 bp-long SELEX fragments should bind to two or more consecutive probes. This criterion was employed to avoid the background noise of non-specific binding of holoenzyme-bound DNA fragments to the tilling array [note that peaks showing hybridization to only a single probe was judged as a false-positive noise].The binding sites were classified into two groups, one ‘within spacers’ and another ‘inside genes’. The binding sites on ‘within spacers’ were further classified into 3 types; type-A spacer located between bidirectional transcription units, type-B spacer located upstream of one transcription unit but downstream of another transcription unit, and type-C spacer located downstream of both transcription units. Based on the transcription direction of flanking genes, the total number of the constitutive promoters was predicted to range between the minimum [number of type-A spacer plus number of type-B spacer] and the maximum [number of type-A spacer x 2 plus number of type-B spacer]. The intragenic binding site was referred to type-D site. The height of binding intensity identified by SELEX-chip system is generally in good agreement with the number of clones identified by SELEX-clos (cloning-sequencing) system, indicating that these two parameters correlate with the binding affinity of test TF to DNA [4,5,8,9].The whole set of constitutive promoters for the stationary-phase sigma RpoSIn laboratory culture of E. coli, cell growth enters into the stationary phase mainly due to the limited availability of nutrients. Upon entry into the stationary phase, the pattern of genome expression is markedly altered by turning down the growth-related genes and instead up-regulation of the stress-response genes. In switching the transcription pattern, the stationary-phase specific minor sigma RrpoS is involved [33,34]. The rpoS gene is not essential for growth under non-stress conditions, but strains carrying mutations affecting rpoS activity are extremely sensitive to environmental stresses. As in the case of other sigma factors, RpoS interacts with RNAP core enzyme and modulates its promoter recognition specificity so as to recognize a specific but large set of genes.As noted above, the set of genes identified in vivo include a number of genes under the indirect control of RpoS. On the other hand, some target promoters of RpoS are masked in vivo due to competitive interference by other DNA-binding proteins. In order to identify the constitutive promoters directly recognized by RpoS in the absence of other DNA-binding proteins, the Genomic SELEX screening in vitro was performed using the reconstituted RNAP RpoS holoenzyme. The sequences with binding affinity to the RpoS holoenzyme formed a number of peaks along the entire E. coli genome (Fig 1). Location of peaks was aligned along the map of E. coli K-12 genome (Table 1). By setting the cut-off level of 3.0 fold-higher intensity over the background of original library DNA, a total of 218 peaks were identified, of which 125 (67%) are located within intergenic spacers and 73 (33%) are inside of open reading frames (Table 2). Since the majority of hitherto identified promoters are located within spacers and generally upstream of open reading frames, detailed search for the constitutive promoters was focused on these 125 spacer peaks. These spacers can be classified into three types: 50 peaks are located within type-A spacer between bidirectional transcription units; 79 peaks are located within type-B spacers located upstream of one transcription unit but downstream of another transcription unit; and 16 peaks are located within type-C located spacers downstream of both transcription units. Based on the transcription direction of flanking genes, the total number of RpoS constitutive promoters was predicted to range between minimum 129 (50 type-A plus 79 type-B) and maximum 179 (50x2 type-A plus 79 type-B) (Table 2). Type-A spacers should contain two promoters for bidirectional transcription, at least one of which should be RpoS-dependent promoter. The RpoS holoenzyme-binding sites identified in a total of 50 type-A spacers should represent promoters for one or both of bidirectional transcription.10.1371/journal.pone.0179181.g001Fig 1SELEX-chip search for RNAP RpoS holoenzyme-binding sequences on the E. coli K-12 genome.The y-axis represents the relative number of RpoS holoenzyme-bound DNA fragments whereas x-axis represents the position on the E. coli K-12 genome, in base pair. The adjacent gene on E. coli K-12 genome of peak position was indicated for high intensity peaks. The peaks located within spacer regions are shown in green color, while peaks located within open reading frames are shown in orange color. The list of RpoS holoenzyme-binding sites is described in Table 1.10.1371/journal.pone.0179181.t001Table 1RpoS holoenzyme-binding sites on the E. coli K-12 genome.NoTypeMapGene FunctionLeftDRpoSDRightGene FunctionIntensity1D17732sokC>nhaA>nhaRtranscriptional activator5.32A41946predicted transportercaiT<>fixAelectron transfer flavoprotein3.63B63256RNAP-associated helicasehepA<<polB3.84D238050aspV>yafT<ykfM19.05B251970dinB>>yafNYafO-YafN toxin-antitoxin system45.16B289870ornithine carbamoyltransferase 2argF<<insB25.07D296234CP4-6 prophage proteinyagN<intF>ptwFXaa tRNA70.48A296336CP4-6 prophage integraseintF<>ptwFXaa tRNA45.59A328672transcriptional repressorbetI<>betTcholine transporter3.810B379186frmRAB operon regulatorfrmR<<yaiO50.811A383868conserved proteinyaiS<>tauAtaurine transporter24.112D451740cytochrome o ubiquinol oxidasecyoA<ampG<yajG5.913B455746tig>>clpPATP-dependent serine protease7.414B467530cof>>ybaODNA-binding transcriptional regulator11.515D479868modulator of gene expression with H-NShha<tomB<acrB45.116B480454predicted proteintomB<<acrB22.617A515034membrane anchored proteaseqmcA<>ybbLABC superfamily transporter70.318D532860allA>allR>gclglyoxylate carboligase3.519D562966sfmH>sfmF<fimZ45.320D569366emrE>ybcK>ybcLDLP12 prophage kinase inhibitor6.821B570042ybcK>>ybcLDLP12 prophage kinase inhibitor19.722A576032IS5 transposase and trans-activatorinsH<>essDDLP12 prophage lysis protein34.023B582672nohB>>*appYDLP12 prophage transcriptional activator12.924D583166nohB>appY<ompT11.425D606570predicted inner membrane proteinybdJ<ybdK>hokEtoxic polypeptide17.326A651466citrate lyase synthetasecitC<>dpiBCitBA TCS sensory histidine kinase8.327D654932dpiA>dcuC>pagPpalmitoyl transferase for Lipid A60.728D659134tatE>lipA<ybeF5.929B692668nucleoside triphosphate hydrolaseybeZ<<miaB19.930C720060ybfK><kdpE3.631B770146mngB>>cydAcytochrome d terminal oxidase, subunit I127.532D799356pgl>ybhD>ybhHconserved protein4.833A812472hypothetical proteinybhU<>uvrBnucleotide excision repair nuclease12.934B837732conserved protein*ybiI<<ybiX19.635A849568threonine and homoserine efflux systemrhtA<>ompXouter membrane protein5.436A852256ncRNArybA<>mntRDNA-binding transcriptional regulator26.037A877270SAM-dependent methyltransferaseyliG<>bssRconserved protein5.338B891170nfsA>>rimKribosomal protein S6 modification protein4.139B903170arginine transporter*artP<<ybjP154.240A931668thioredoxin reductasetrxB<>lrpDNA-binding transcriptional dual regulator5.941D943972dmsB>dmsC<ycaC24.342B953940formate transporterfocA<<ycaO3.443B959450aroA>>ycaLpeptidase with chaperone function6.644B962934rpsA>>*ihfBintegration host factor (IHF)3.645D1028042hemimethhylated DNA-binding proteinhspQ<yccW>yccXpredicted acylphosphatase40.546C1049830insB><cspH167.247B1084156efeB>>phoHconserved protein with NTPase domain62.748B1120372predicted proteinbssS<<dinI6.649B1120772DNA damage-inducible protein IdinI<<pyrC3.750D1144848yceQ>rluC<yceF23.751D1147330rpmF>plsX>fabH3-oxoacyl-[acyl-carrier-protein] synthase III31.252A1168238DNA-binding transcriptional regulatorycfQ<>bhsApredicted protein17.653D1187848conserved proteinycfD<phoQ<phoP5.554D12110305-methyl-C-specific restriction nucleasemcrA>elbA<ycgX4.755D1212068predicted proteinelbA<ycgX<ycgE27.856A1214962predicted FAD-binding phosphodiesteraseycgF<>*ycgZpredicted protein14.257B1215830ariR>>ymgCpredicted protein6.758C1218960ymgF><ymgD15.159A1250254dihydroxyacetone kinase*dhaK<>*dhaRDNA-binding transcriptional regulator15.760A1257834peptidyl-tRNA hydrolasepth<>ychHpredicted inner membrane protein72.561D1318064indole-3-glycerol-P synthetasetrpC<trpD<trpE73.662B1341438lipoprotein*osmB<<yciT3.463D1342872hypothetical proteinyciZ<gmr<rnb48.264A1359040gamma-Glu-putrescine synthase*puuA<>*puuDgamma-Glu-GABA hydrolase21.865D1393856mppA>ynaI>insHIS5 transposase and trans-activator4.066B1432638DNA-binding transcriptional regulatorynaE<<uspF10.267A1438872pyruvate-flavodoxin oxidoreductaseydbK<>ydbJpredicted protein5.168D1447730feaB>tynA<maoC5.069C1463278paaY><insD6.270A1489640ncRNArydC<>ydcApredicted protein14.971B1500460tehB>>ydcLpredicted lipoprotein7.472A1515332hypothetical proteinyncL<>ydcXpredicted inner membrane protein28.573B1518146yncB>>mcbRDNA-binding transcriptional regulator4.474D1528272yncH>ydcD>ydcCconserved protein9.275B1543272predicted proteinyddJ<<yddG3.576D1553260dehydrogenase/acetaldehyde reductaseadhP<maeA<sra4.977D1566238predicted diguanylate cyclaseyddV<yddW<gadC14.578B1570272glutamate decarboxylase B*gadB<<pqqL14.979B1580646conserved proteinydeN<<ydeO8.080B1590548DNA-binding transcriptional regulatorhipB<<ydeU94.481B1596530predicted lipoproteinydeK<<lsrK12.982D1613844yneJ>yneK>ydeApredicted arabinose transporter4.383B1618030marA>>marBpredicted protein3.984B1627238ydfH>>ydfZconserved protein5.585A1630740predicted mannonate dehydrogenaseydfI<>ydfKQin prophage transcriptional regulator10.886A1655452predicted proteinynfC<>ynfDpredicted protein32.387B1682244rstB>>tusinhibitor of replication at Ter77.288B1687868manA>>ydgAconserved protein132.489B1745130ydhR>>ydhSprotein with FAD/NAD(P)-binding domain3.890A1753168predicted proteinydhZ<>*pykFpyruvate kinase I4.291D1769372conserved proteinydiL>ydiM>ydiNpredicted transporter3.292C1793764integration host factor (IHF)ihfA<pheT<pheS8.193D1811050ydjN>ydjO<cedA29.294C1841754gdhA><ynjI79.595B1894766nudL>>sdaAL-serine deaminase I55.496B1905768predicted proteinyobF<<yebO11.297D1921260ncRNAryeA>ryeB<yebY11.598B1927030protease IIptrB<<yebE6.199A1928846conserved protein regulated by LexAyebG<>purTP-ribosylglycinamide formyltransferase 263.2100D1928972conserved protein regulated by LexAyebG<purT<eda390.6101A1944202RuvABC resolvasomeruvA<>yebBpredicted protein11.3102B1956162TMAO reductase III (TorYZ)torY<<cutC17.4103C1966932CheAB TCS chemotaxis regulatorcheB<cheR<tap8.7104B1994970DNA-binding transcriptional activatorsdiA<<yecC21.7105A2023030predicted protein*dsrB<>yodDpredicted protein88.4106B2031954rseX>>*hchAHsp31 molecular chaperone7.7107D2049968asnT>yeeJ>shiAshikimate transporter150.8108D2057732Asn tRNAasnW<yeeO>asnUAsn tRNA3.2109B2061434L,D-transpeptidase linking Lpp to mureinerfK<<cobT44.9110D2103732predicted acyl transferasewbbJ<wbbI<rfc4.3111D2104956conserved proteinwbbI<rfc<glf14.6112D2106934UDP-galactopyranose mutaseglf<rfbX<rfbC54.2113B2134130protein-tyrosine phosphatasewzb<<wza53.9114B2226932NAD(P)-binding oxidoreductaseyohF<<dusC3.3115A2247636DNA-binding transcriptional regulatoryeiE<>yeiHconserved inner membrane protein12.7116D2255672predicted nucleoside transporterpsuT<psuG<psuKpseudouridine kinase7.7117B2284170yejM>>proLPro tRNA9.5118A2311066outer membrane porin protein CompC<>micFncRNA16.1119B2311354micF>>rcsDRcsBC TCS phosphotransfer protein13.8120D2389234yfbP>nuoN<nuoM13.0121B2454170predicted fimbrial-like adhesin proteinyfcV<<sixA25.4122D2467360yfdH>yfdI<yfdK38.3123C2468764yfdI><yfdK26.5124D2480972predicted multidrug efflux systememrY<emrK>*evgAEvgAS TCS response regulator126.6125A2510860manganese/divalent cation transportermntH<>nupCnucleoside (except guanosine) transporter13.5126D2532356ptsH>ptsI>*crrglucose-specific PTS enzyme IIA9.2127B2574144carboxysome structural proteineutS<<maeB11.3128D2587966narQ>acrD<ypfM18.4129B2651536sseA>>ryfAncRNA40.6130B2689548ncRNAglmY<<purL24.9131B2753630smpB>>ssrAtmRNA60.7132B2763338CP4-57 prophage; predicted proteinyfjL<<yfjM13.8133B2765760yfjO>>yfjPCP4-57 prophage GTP-binding protein5.2134D2771468yfjT>yfjW>yfjXCP4-57 prophage antirestriction protein32.6135D2772262yfjT>yfjW>yfjXCP4-57 prophage antirestriction protein26.4136B2773166yfjW>>yfjXCP4-57 prophage antirestriction protein41.0137D2779640psaA>ypjA<ileY28.8138B2783272adhesin-like autotransporterypjA<<ileY58.9139A2784466Ile tRNAileY<>*csiDpredicted protein3.7140A2786358Ile tRNAileY<>*csiDpredicted protein31.8141A2795168predicted membrane proteinyqaE<>ygaVDNA-binding transcriptional regulator53.2142B2882250predicted proteinygcL<<ygcB6.3143A2903434conserved proteinygcF<>ygcGpredicted protein57.0144B2985568yqeG>>yqeHprotein with bipartite regulator domain32.6145C2987942yqeJ><yqeK9.1146D2990738ygeG>ygeH>ygeIpredicted protein6.8147C2992070ygeI><insD6.4148C2992950ygeI><insD6.9149C2993358ygeI><insD13.2150B3134436phosphate transporterpitB<<gsp12.6151A3145934predicted proteinyghW<>yghZaldo-keto reductase3.2152B3166762predicted cyanide hydratasemqsR<<ygiV65.1153C3181642zupT><ribB10.0154B3183246yqiC>>ygiLpredicted fimbrial-like adhesin protein3.7155D3189250yqiH>yqiI<glgS68.5156D3210472rpsU>dnaG>rpoDRNA polymerase sigma 7021.3157D3259830L-PSP (mRNA) endoribonucleasetdcF<tdcE<tdcD3.6158D3266232tdcR>yhaB>yhaCpredicted protein9.6159C3326238yhbY><greA3.6160B3335948ABC-type organic solvent transporteryrbC<<yrbD14.0161D3345766ptsN>yhbJ>nprN-regulated PTS system (Npr)75.4162D3348770ncRNAryhA>arcB<yhcC5.9163D3395840ribonuclease Grng<yhdE<mreD4.3164D3453834general secretory pathway componentgspA<gspC>gspDgeneral secretory pathway component9.3165B3467930periplasmic endochitinasechiA<<tufA5.0166B3497370cysG>>yhfLconserved secreted peptide13.0167D3533142pck>envZ<ompR41.3168D3542562yhgA>bioH>gntXgluconate periplasmic binding protein15.9169B3576742DNA-binding transcriptional repressorgntR<<yhhW12.2170B3582734yrhD>>yrhBpredicted protein4.3171A3584864gamma-glutamyltranspeptidaseggt<>yhhAconserved protein12.9172D3604756yhhN>zntA<sirA8.8173B3621930yhhH>>yhhIpredicted transposase3.3174A3632156predicted proteinyhiJ<>*yhiMconserved inner membrane protein32.2175A3632742predicted proteinyhiJ<>*yhiMconserved inner membrane protein13.3176A3637868universal stress protein B*uspB<>uspAuniversal stress global response regulator22.7177D3647644arsR>arsB>arsCarsenate reductase31.8178C3648872arsC><insH153.6179B3656130hdeD>>gadEDNA-binding transcriptional activator9.1180B3708672predicted metal dependent hydrolaseeptB<<yhjX48.9181B3717944yiaG>>cspAmajor cold shock protein6.1182B3720058insK>>sokAncRNA31.7183A3749938predicted proteinyiaT<>yiaUDNA-binding transcriptional regulator22.4184B3794944rfaC>>rfaLO-antigen ligase3.8185D3796942rfaL>waaU<rfaZ46.2186D3798472lipopolysaccharide core synthesis proteinrfaZ<rfaY<rfaJ145.8187D3800958UDP-D-glucose:LPS glucosyltransferaserfaJ<rfaI<rfaB15.8188D3802448UDP-galactose:LPS galactosyltransferaserfaB<rfaS<rfaP21.5189B3834856Sec tRNAselC>>setCpredicted sugar efflux system22.6190A3851272ncRNAistR<>tisBlexA-regulated toxic peptide57.3191B3886640tnaC>>tnaAtryptophanase/L-cysteine desulfhydrase29.5192D4001054conserved inner membrane proteinyigF<yigG<rarD5.5193D4002164predicted inner membrane proteinyigG<rarD<yigI13.0194B4076572yiiD>>yiiEpredicted transcriptional regulator4.5195D4110740conserved proteinyiiQ<yiiR>yiiSconserved protein10.8196A4116232glycerol facilitatorglpF<>yiiUconserved protein15.6197B4120330HslUV proteasehslV<<ftsN3.6198D4220334aceK>arpA<iclR5.4199B4281230conserved proteinyjcF<<actP83.6200A4380530fumarate reductasefrdA<>poxApredicted lysyl-tRNA synthetase4.7201A4417636L-ascorbate 6-phosphate lactonaseulaG<>ulaAL-ascorbate-specific PTS enzyme IIC4.9202A4432132NAD(P)H:quinone oxidoreductaseytfG<>ytfHpredicted transcriptional regulator90.2203A44345622':3'-cyclic-nucleotide 2'-phosphodiesterasecpdB<>cysQPAPS 3'(2'),5'-bisphosphate nucleotidase4.3204B4472740yjgJ>>yjgKconserved protein15.2205C4475268yjgL><argI80.9206A4477770predicted acetyltransferaseyjgM<>yjgNconserved inner membrane protein4.1207A4492648L-idonate 5-dehydrogenase, NAD-bindingidnD<>idnKD-gluconate kinase, thermosensitive6.0208B4504448yjhC>>ythAexpressed protein8.4209C4505142ythA><insI5.3210A4538166N-acetylnuraminic acid OM channel proteinnanC<>fimBTyr recombinase/fimA inversion regulator5.5211A4538758N-acetylnuraminic acid OM channel proteinnanC<>fimBTyr recombinase/fimA inversion regulator127.7212D4543668fimC>fimD>fimFminor component of type 1 fimbriae5.0213C4570642yjiS><mcrC41.4214D4575670yjiS>mcrC<mcrB12.1215D45764685-methylcytosine-specific restriction enzymemcrC<mcrB<symE26.3216D4588370conserved proteinyjiX<*yjiY>tsrmethyl-accepting chemotaxis protein I10.5217A4601330predicted inner membrane proteinyjjP<>yjjQDNA-binding transcriptional regulator22.9218B4609336prfC>>osmYperiplasmic protein3.7Genomic SELEX was performed for search of the binding sites of RNAP RpoS holoenzyme. By setting the cut-off level of 3.0, a total of 218 binding sites were identified (see Fig 1 for SELEX pattern), which are aligned along the map of E. coli K12 genome. A total of 125 sites are located within intergenic spacers: 50 within type-A spacers (shown under orange background); and 79 within type-B spacers (shown under green background). The constitutive promoters of RpoS were predicted based on the adjacent genes [note that only the genes next to the RpoS holoenzyme-binding sites are shown] and the gene orientation (shown by arrows in the column of transcription direction). A total of 73 RpoS holoenzyme-binding sites are located inside open reading frames as indicated by the gene symbols shown in RpoS column.*The genes listed in RegulonDB as the regulated targets of RpoS.10.1371/journal.pone.0179181.t002Table 2Distribution of the binding sites of each RNAP holoenzyme.SigmaTotal no. holoenzyme binding sitesWithin SpacersInside GenesConstitutive promotersType-AType-BType-CType-A spacerType-B spacerTotalRpoD132017731749777 (60%)177~354317494~671543 (40%)RpoS21850791673 (33%)50~10079129~179125 (67%)RpoH1334160626 (20%)41~8260101~142107 (80%)RpoF105727368 (65%)7~142734~4137 (35%)RpoE1262948742 (33%)29~584877~10684 (67%)RNAP holoenzyme was reconstituted from the sigma-free core enzyme and 4-fold molar excess of each sigma subunit. The binding site of each holoenzyme on the genome of E. coli K-12 W3110 was determined in vitro using the improved Genomic SELEX screening system. Details of the experimental procedures are described previously [26]. The number of constitutive promoters were estimated based on the location of holoenzyme-binding sites. The number of constitutive promoters recognized by RpoD holooenzyme were described in the previous report [27].Up to the present, two general approaches have been employed to define the RpoS regulon: the proteome analysis using two-dimensional gels of whole cell lysates [35]; and the transcriptome analysis using ChIP-chip or ChIP-Seq systems [37–39]. These studies altogether indicated that RpoS regulates, directly or indirectly, 10% (approximately 500 genes) of the E. coli genes, of which only about 140 genes were predicted to be under the direct control in vivo of RpoS [38]. The total number of RpoS promoters (or the transcription initiation sites) listed in the current RegulonDB database is as many as 164 [note that all these promoters were detected in vivo]. Of which 21 were identified by setting the cut-off level at 3.0 (Table 1, marked by asterisk), indicating that only these promoters represent the constitutive promoters and the majority of other known RpoS promoters represent the inducible promoters that are expressed only under the support of regulatory factors. Genomic SELEX analysis identified minimum 129 and maximum 179 RpoS constitutive promoters including 21 known RpoS-dependent promoters (Table 2). The highest peak (390-fold higher than the background of original library alone) was located at the 5’-proximal region of the purT gene (Fig 1), which encodes a bifunctional enzyme with both phosphoribosylglycinamide formyltransferase using formate (the third-step reaction of purine nucleotide synthesis) and acetate kinase for the synthesis of acetylphosphate (AcP). AcP might be utilized as the general phosphate donor for phosphorylation of most of the stress-response TCS (two-component system) response regulators under stressful conditions. A high-level peak (154-fold higher than the background of DNA library) was detected upstream of the artPIQM operon encoding L-arginine ABC transporter. This promoter was also identified in vivo to be RpoS dependent [36] (Table 1). High-level binding of the RpoS holoenzyme was also identified upstream of the cydAB operon encoding cytochrome oxidase for anaerobic respiration, and the hipBA operon encoding anti-toxin-toxin pair for control the persistence (Fig 1 and Table 1). RpoS-dependent constitutive promoter(s) also exists upstream of the nanCMS operon (N-acetylneuraminic acid transport and utilization) and/or the fimB gene (regulator for fimA encoding fimbrin, the major type-1 pili) (Fig 1 and Table 1). Noteworthy is that most of the RpoS-dependent promoters listed in the current databases might be those under the indirect control of RpoS [8,9,27]. Otherwise a set of RpoS-dependent promoters, designated as the inducible promoters, might be activated in the presence of additional supporting factors.Using the newly constructed collection of E. coli promoters expressing two-fluorescent reporters, one attached to the test promoter and another to the reference promoter, we performed a systematic quantitative search in vivo for E. coli promoters that are activated in the stationary phase [39]. The activity of RpoS-dependent promoters was measured at various growth phases under various growth conditions. The results indicated that the constitutive promoters exhibited low but steady-state activity while the inducible promoters generally showed high activity during the transition from exponential growth to stationary phase.The RpoS regulon is involved in not only cell survival in the stationary phase, but also in cross protection against various stresses, including nutrient starvation, osmotic stress, acid shock, cold shock, heat shock, and oxidative DNA damage [33,34]. Beyond entry into stationary phase, E. coli forms aggregates or biofilms that are morphologically and physiologically distinct from cells of planktonic growth. This requires coordinated production of an extracellular matrix of polysaccharide polymers and protein fibers that facilitate cell aggregation and adhesion to solid surface. The genes involved in biofilm formation and transformation into persister cells were included in the list of RpoS constitutive promoters [40,41].The whole sets of constitutive promoters for heat-shock response sigma RpoHWhen E. coli cells are exposed to higher temperature, a set of heat-shock proteins (HSPs) is markedly and transiently induced. Heat shock-induced proteins (HSPs) play major roles in controlling the structure and function of various proteins, including protein folding, assembly, transport, repair and degradation during normal growth as well as under stress conditions [42,43]. The heat-shock response is a cellular protective system for maintenance of protein homeostasis. The set of HSPs include the GroEL (HSP60) and DnaK (HSP70) chaperones and the Lon and the Clp proteases. RpoH is specifically required for expression of the genes encoding a set of HSPs as identified by proteome [44,45] and also by transcriptome analyses [46]. Genome-wide transcription profiling of the regulatory targets of RpoH was identified under the moderate induction of a plasmid-borne rpoH gene under defined, steady state growth conditions [47]. A total of 126 genes were influenced in the absence or in the over-expression of RpoH, which are organized in 85 operons. The set of genes identified in vivo by changing the level of RpoH include a large number of indirect targets, which are affected in response to the changes in the level of direct target [8,9,27]. The total number of RpoH promoters (or the transcription initiation sites) listed in the current RegulonDB database is as many as 322, but the majority of RpoH targets are predicted by the computational analysis using the consensus sequence that was predicted based on a few experimentally identified RpoH promoters.We isolated RpoH protein for the first time and confirmed its recognition in vitro of the known HSP gene promoters [48]. Since then no serious examination in vitro has been performed to identify the RpoH function and it regulatory targets. To get insights into the regulatory role of RpoH sigma, we then performed in this study the Genomic SELEX screening using the reconstituted RNAP RpoH holoenzyme. By setting the cut-off level of 3.0 fold higher than the background of original library DNA, a total of 133 RpoH holoenzyme-binding peaks were identified (Fig 2 and Table 3), of which 107 (80%) are located within intergenic spacers and 26 (20%) are inside of open reading frames (Table 2). Since the majority of hitherto identified promoters are located within spacers, detailed search for the constitutive promoters was focused on the total of 107 peaks within spacers. The spacers containing RpoH holoenzyme-binding sites were also classified into three types (Tables 2 and 3 for the whole list): 41 peaks are located within type-A spacer; 60 peaks are located within type-B spacers; and 6 peaks are located within type-C spacers. Based on the transcription direction of flanking genes, the total number of RpoH constitutive promoters was predicted to range between minimum 101 (41 type-A plus 60 type-B) and maximum 142 (41x2 type-A plus 60 type-B) (Table 2).10.1371/journal.pone.0179181.g002Fig 2SELEX-chip search for RNAP RpoH holoenzyme-binding sequences on the E. coli K-12 genome.The y-axis represents the relative number of RpoH holoenzyme-bound DNA fragments whereas x-axis represents the position on the E.coli K-12 genome, in base pair. The adjacent gene on E. coli K-12 genome of peak position was indicated for high intensity peaks. The peaks located within spacer regions are shown in green color, while peaks located within open reading frames are shown in orange color. The list of RpoH holoenzyme-binding sites is described in Table 3.10.1371/journal.pone.0179181.t003Table 3RpoH holoenzyme-binding sites on the E. coli K-12 genome.NoTypeMapGene FunctionLeftDRpoHDRightGene FunctionIntensity1A12044predicted proteinyaaI<>*dnaKchaperone Hsp703.22A142736carbonic anhydrase*can<>yadGABC superfamily transporter3.93B155442outer membrane usher proteinhtrE<<ecpD3.04B164658hrpB>>mrcBglycosyl transferase and transpeptidase3.85B202068lpxD>>fabZhydroxymyristol acyl carrier dehydratase4.06B229134aspU>>dkgB2,5-diketo-D-gluconate reductase B3.07C262270thrW><ykfI3.68B292170CP4-6 prophage proteinyagK<<yagL4.39B331456betT>>yahADNA-binding transcriptional regulator3.510D343660yahK>yahL>yahMpredicted protein5.611B379186frmRAB operon regulatorfrmR<<yaiO3.012B406536yaiA>>aroMconserved protein3.613B477848conserved inner membrane proteinylaB<<ylaC3.314D543470conserved proteinylbA<allC<allD10.115B557960sfmA>>sfmCpilin chaperone3.516D559456sfmC>sfmD>sfmHfimbrial-like adhesin protein3.117B581644nohB>>appYDLP12 transcriptional activator3.818B592452phage N4 receptor IM proteinnfrB<<cusS3.219B629042ybdB>>cstAcarbon starvation protein6.020D655854anaerobic C4-dicarboxylate transportdcuC<pagP>cspEDNA-binding transcriptional repressor3.321B661936conserved protein*ybeD<<dacA20.122D732870ybfA>*rhsC>ybfBpredicted inner membrane protein3.223D747240predicted regulatorabrB<ybgO<ybgP3.524B784068zinc efflux systemzitB<<ybgS3.525A784656conserved proteinybgS<>aroGD-arabino-heptulosonate-7P synthase3.026B837732conserved proteinybiI<<ybiX3.127B913136anaerobic terminal reductaseshcp<<ybjE3.828A918368conserved proteinybjX<>macAmacrolide transporter6.329B959450aroA>>ycaLpeptidase with chaperone function3.230B985134aspartate aminotransferaseaspC<<ompF3.431A1050632CspA-family stress protein*cspH<>cspGDNA-binding transcriptional regulator3.632A1102554DNA-binding transcriptional activatorcsgD<>csgBcurlin nucleator protein4.033B1120230predicted protein*bssS<<dinI5.434C1195868icd><ymfD6.235D1198640e14 prophage inner membrane proteinymfE<lit<intE3.836D1200062e14 prophage integraseintE<xisE>ymfIe14 prophage; predicted protein3.237B1218154ycgG>>ymgFpredicted protein3.438C1219948ymgF><ymgD3.939A1233950sodium:proton antiporternhaB<>fadRDNA-binding transcriptional regulator3.140B1255834predicted adhesinycgV<<ychF3.241B1308330voltage-gated potassium channelkch<<yciI3.742B1349272enoyl-[acyl-carrier-protein] reductasefabI<<ycjD4.343A1389946predicted hydrolase*ycjY<>*ycjZDNA-binding transcriptional regulator4.444B1432738Rac prophage transcriptional regulatorynaE<<uspF4.245B1486246ydcF>>aldAaldehyde dehydrogenase A3.746B1565470predicted diguanylate cyclaseyddV<<yddW5.247B1568560glutamate:aminobutyric acid antiportergadC<<gadB2.348B1580550conserved proteinydeN<<ydeO3.349B1585730fimbrial-like adhesin proteinydeQ<<ydeR3.750B1613766yneJ>>*yneKpredicted protein3.051A1630638mannonate dehydrogenaseydfI<>ydfKQin prophage transcriptional regulator4.052B1639072Qin prophage S lysis proteinessQ<<cspB3.353A1639660Qin prophage cold shock proteincspB<>cspFQin prophage cold shock protein3.754B1646444dicA>>*ydfAQin prophage protein3.255B1669358ynfM>>asracid-shock periplasmic protein3.756B1751846predicted oxidoreductaseydhV<<ydhY3.157B1762570Fe-S cluster assembly proteinsufA<<rydB5.158A1801256threonyl-tRNA synthetasethrS<>yniDpredicted protein5.359A1801758threonyl-tRNA synthetasethrS<>yniDpredicted protein3.460B1990832PG phosphate synthasepgsA<<uvrC4.261D2055332amn>yeeN<asnW3.062A2060070DNA-binding transcriptionall regulatornac<>asnVAsn tRNA3.263B2096360LPS O-antigen length regulator*cld<<ugd3.264D2104956conserved proteinwbbI<*rfc<glf3.465A2261530fructose-specific PTS enzyme IIAfruB<>setBlactose/glucose efflux system3.366B2276452predicted proteinyejG<<bcr3.167A2342846hypothetical proteinypaB<>nrdAribonucleoside diphosphate reductase3.468D2355640glpC>yfaD>ypaApredicted protein3.469D2363146nudI>ais>arnBuridine aminotransferase3.870B2403570NADH:ubiquinone oxidoreductasenuoA<<lrhA5.171A2405036DNA-binding transcriptional regulatorlrhA<>yfbQpredicted aminotransferase3.272D2476546DNA-binding transcriptional regulatordsdC<dsdX>dsdAD-serine ammonia-lyase3.673D2489940predicted transporteryfdV<oxc<frc3.674D2490262predicted oxalyl-CoA decarboxylaseoxc<frc<yfdX3.675A2492430predicted proteinyfdX<>ypdIlipoprotein for colanic acid biosynthesis3.076A2493362predicted inner membrane proteinyfdY<>lpxPpalmitoleoyl-ACP acyltransferase3.677D2532356ptsH>ptsI>crrglucose-specific PTS enzyme IIA3.278A2597838dihydrodipicolinate synthasedapA<>gcvRDNA-binding transcriptional repressor3.179B2599140bcp>>hyfAhydrogenase 4, 4Fe-4S subunit3.180D2626032ppx>yfgF>yfgGpredicted protein3.681A2696642conserved proteinyfhB<>yfhHDNA-binding transcriptional regulator3.382A2714742pyruvate formate lyase subunityfiD<>unguracil-DNA-glycosylase4.883A2739338D-arabino-heptulosonate-7P synthasearoF<>yfiLpredicted protein3.584A2823870lytic murein transglycosylase BmltB<>srlAglucitol/sorbitol-specific PTS IIC3.185A2837534DNA-binding transcriptional repressorascG<>ascFcellobiose/arbutin/salicin PTS IIB-IIC3.586D2884946predicted proteinygcL<ygcB<cysH3.487A2898372deoxygluconate dehydrogenaseygcW<>yqcEpredicted transporter4.688A2932264L-fuculose-1-phosphate aldolasefucA<>fucPL-fucose transporter3.389B2967056nucleotide hydrolaserppH<<ygdT4.590A2976930diaminopimelate decarboxylaselysA<>lysRDNA-binding transcriptional regulator3.491B2985164yqeG>>yqeHprotein with bipartite regulator domain3.392D2985970yqeG>yqeH>yqeItranscriptional regulator3.293D2991152ygeG>ygeH>ygeIpredicted protein3.794C2991858ygeI><insD3.595B3067930mechanosensitive channelmscS<<fbaA3.096B3166268DNA-binding transcriptional regulatorygiT<<mqsR3.097B3237654alx>>sstTsodium:serine/threonine symporter6.998D3266066tdcR>yhaB>yhaCpredicted protein3.099A3276944DNA-binding transcriptional regulatoragaR<>kbaZtagatose 6-phosphate aldolase 13.6100B3319952predicted hydrolase, inner membraneyhbX<<leuU3.2101A332583223S rRNA methyltransferaserrmJ<>yhbYpredicted RNA-binding protein14.1102D3360232gltF>*yhcA>yhcDpredicted outer membrane protein3.2103B3375554stringent starvation protein AsspA<<rpsI3.7104B3387142p-hydroxybenzoic acid efflux system*aaeA<<aaeX3.1105A3559934thiosulfate:cyanide sulfurtransferaseglpE<>glpDsn-glycerol-3-phosphate dehydrogenase3.5106D3629568predicted HlyD family secretion proteinyhiI<yhiJ>yhiMconserved inner membrane protein3.6107C3634072yhiM><yhiN3.8108B3655654hdeD>>gadEDNA-binding transcriptional activator4.1109C3661646mdtF><gadW3.7110B3681552C4-dicarboxylic acid citrate transporterdctA<<yhjK3.0111A3717248conserved proteinyiaF<>yiaGtranscriptional regulator3.0112B3717858yiaG>>cspAmajor cold shock protein3.0113B3720058insK>>sokAncRNA3.0114A3826772glutamate transportergltS<>yicEpredicted transporter4.4115B3834954selC>>setCpredicted sugar efflux system3.4116A3865668heat shock chaperone*ibpA<>yidQconserved outer membrane protein4.6117A3939432predicted transcriptional regulatoryieP<>*rrsC16S ribosomal RNA of rrnC operon3.6118A4002730conserved proteinyigI<>*pldAouter membrane phospholipase A3.7119B4068432predicted aldose-1-epimeraseyihR<<yihS7.5120B4360430DNA-binding transcriptional activatorcadC<<pheU4.0121B4364770C4-dicarboxylate antiporterdcuA<<aspA4.6122A4366568aspartate ammonia-lyaseaspA<>*fxsAinner membrane protein19.2123D4371850yjeI>*yjeJ<yjeK4.9124D4486746yjgQ>yjgR<idnR4.1125B4494638leuX>>insCKpLE2 phage IS2 element repressor3.1126B4518430KpLE2 transcriptional regulatoryjhU<<yjhF3.1127B4530030KpLE2 phage endoglucanasesgcX<<yjhP3.3128A4538050N-acetylnuraminic acid channel proteinnanC<>fimBtyrosine recombinase, fimA regulator3.5129A4538964N-acetylnuraminic acid channel proteinnanC<>fimBtyrosine recombinase, fimA regulator3.4130B4540968fimE>>fimAmajor type 1 subunit fimbrin (pilin)3.4131A4549330fructuronate transportergntP<>uxuAmannonate hydrolase5.0132D45764685-methylcytosine restriction enzymemcrC<mcrB<symE3.5133A4633370DNA-binding transcriptional activatorrob<>*creAconserved protein3.3Genomic SELEX was performed for search of the binding sites of RNAP RpoH holoenzyme. By setting the cut-off level of 3.0, a total of 133 binding sites were identified (see Fig 2 for SELEX pattern), which are aligned along the map of E. coli K12 genome. A total of 107 sites are located within intergenic spacers: 41 wihin type-A spacers (shown under orange background); and 60 within type-B spacers (shown under green background). The constitutive promoters of RpoH were predicted based on the adjacent genes [note that only the genes next to the RpoH holoenzyme-binding sites are shown] and the gene orientation (shown by arrows in the column of transcription direction). A total of 26 RpoH holoenzyme-binding sites are located inside open reading frames as indicated by the gene symbols shown in RpoH column.* The genes listed in RegulonDB as the regulated targets of RpoH.Among a total of 322 RpoH promoters (or the transcription initiation sites) listed in RegulonDB database, 20 were identified setting the cut-off level at 3.0 (Table 3, marked by asterisk). The majority of RpoH promoters in the database were suggested to belong to the inducible promoters that are expressed only under the support of other positive regulatory factors. Otherwise these RpoH promoters might represent the inaccurate prediction as note above. Genomic SELEX analysis identified minimum 100 and maximum 140 RpoH constitutive promoters including 18 known RpoH dependent promoters (Table 2). The highest peak was 20-fold intensity that was detected on promoter region of the ybeD gene, which encodes a conserved protein of unknown function under regulation of RpoH (Fig 2) [49], followed by high-level peaks at the aspA-fxsA and the rlmJ-yhbY intergenic regions. The fxsA gene encodes an inner membrane protein, which is involved in sensitivity control to bacteriophage T7 [49]. The rlmE gene encodes 23S rRNA 2’-O-ribose U2552 metyltransferase, and has been proposed to carry RpoH-dependent promoter [50]. The regulatory target of RpoH sigma identified by Genomic SELEX expands to a set of genes related to varieties of stress-response genes beyond the HSP genes. In fact, the genes for response to environmental insults such as ethanol, alkaline pH, and hyperosmotic shock and the genes for proteolysis and cell division have been indicated under the control of RpoH. The set of RpoH-regulon genes thus identified in vivo, however, vary depending on the culture conditions.The whole sets of constitutive promoters for the flagella-chemotaxis sigma RpoFThe bacterial flagellum is a complex organelle consisting of three distinctive structural parts, the basal body, the hook and the filament [51]. The synthesis, assembly and function of the flagellar and chemotaxis system require the expression of more than 50 genes, which are divided into three temporally regulated transcriptional classes based on the hierarchy of expression order: class-I (early), class-II (middle), and class-III (late) [52,53]. The class-1 (early) consists of a single operon including two genes, flhD and flhC, each encoding transcription factor FlhD and FlhC, respectively, which together form a complex, FlhD2-FlhC2 or FlhD4-FlhC2, that activates transcription of a set of class-2 (middle) genes, including both the rpoF sigma gene (renamed fliA) and the flgM gene encoding the anti-RpoF factor [51,52]. RpoF is the sigma factor for flagellar chemotaxis, which recognizes the promoters of motility and flagellar synthesis genes. The regulatory target of RpoF in Salmonella was identified to include a set of genes that were classified into the class-3 operons of the flagella regulon [54,55]. More than 30 genes have been proposed to carry promoters that are under the control of RpoF sigma, including a set of the structural genes for flagella formation, and the chemotaxis genes encoding sensor of environmental signals affecting the motility control [54,56]. With use the combination of ChIP-chip, ChIP-seq and RNA-seq systems, a more comprehensive screening was recently performed for identification of the regulatory targets of RpoF sigma in E. coli [57]. A total of 52 RpoF-binding sites were identified in vivo on the genome of exponentially growing E. coli K-12 MG1655 cells in a rich LB medium, with a considerable level of over-lapping with the hitherto identified target genes of the RpoF regulon. The total number of RpoF promoters (or the transcription initiation sites) listed in the current RegulonDB database is as many as 144, which have been identified in vivo using ChIP-chip and ChIP-RNA Seq analyses. Most of the targets predicted by the in vivo data, however, represent those indirectly affected upon knock-out of the rpoF gene or over-production of RpoF.We then performed the Genomic SELEX screening in vitro for search of the direct target promoters, genes and operons under the control of RpoF using the reconstituted RNAP RpoF holoenzyme. By setting the cut-off level of 4.0 fold higher than the background of original library DNA, a total of 105 RpoF holoenzyme-binding peaks were identified (Fig 3 and Table 4), of which 37 (35%) are located within intergenic spacers and 68 (65%) are inside of genes (Table 2). One unique feature of RpoF holoenzyme is its high-level (65%) binding to inside of open reading frames of a number of genes. A high-level (60%) of RNAP binding was also identified for RpoD holoenzyme [27]. The identification of the promoter-like sequences inside these genes awaits further analysis. The spacers containing RpoF holoenzyme-binding sites were also classified into three types (Tables 2 and 4 for the whole list): 7 peaks are located within type-A spacer; 27 peaks are located within type-B spacers; and 3 peaks are located within type-C spacers. Based on the transcription direction of flanking genes, the total number of RpoF constitutive promoters was predicted to range between minimum 34 (7 type-A plus 27 type-B) and maximum 41 (7x2 type-A plus 27 type-B) (Table 2). The total number of RpoF promoters (or the transcription initiation sites) listed in the current RegulonDB database is as many as 144. Of which 14 were identified setting cut-off level at 4.0 (Table 4, marked by asterisk), indicating that these promoters are constitutive promoters and the majority of known promoters represent the inducible promoters that are expressed only under the support of positive regulatory factors.10.1371/journal.pone.0179181.g003Fig 3SELEX-chip search for RNAP RpoF holoenzyme-binding sequences on the E. coli K-12 genome.The y-axis represents the relative number of RpoF holoenzyme-bound DNA fragments whereas x-axis represents the position on the E. coli K-12 genome, in base pair. The adjacent gene on E. coli K-12 genome of peak position was indicated for high intensity peaks. The peaks located within spacer regions are shown in green color, while peaks located within open reading frames are shown in orange color. The list of RpoF holoenzyme-binding sites is described in Table 4.10.1371/journal.pone.0179181.t004Table 4RpoF holoenzyme-binding sites on the E. coli K-12 genome.NoTypeMapGene FunctionLeftDRpoFDRightGene FunctionIntensity1D25672ileS>lspA>fkpBpeptidyl-prolyl cis-trans isomerase8.12B68342L-arabinose isomerasearaA<<araB4.33D109556secM>secA>mutTNTP pyrophosphohydrolase5.54A131366predicted proteinyacH<>acnBaconitate hydratase4.65B164658hrpB>>mrcBglycosyl transferase and transpeptidase6.46D198960rseP>bamA>skpperiplasmic chaperone4.77B202068lpxD>>fabZhydroxymyristol acyl carrier dehydratase7.18D298660conserved proteinyagQ<yagR<yagS4.49D343660yahK>yahL>yahMpredicted protein27.110D353230prpD>prpE>codBcytosine transporter4.711B463170ppiD>>ybaVconserved protein5.412D472938glnK>amtB<tesB4.013D477460ybaA>ylaB<ylaC7.814D482072predicted proteintomB<acrB<acrA4.515D543470conserved proteinylbA<allC<allD12.316D633930predicted oxidoreductaseybdH<ybdL<ybdM5.417D652632citrate lyase synthetasecitC<dpiB>dpiACitAB TCS response regulator5.018D747240predicted regulatorabrB<ybgO<ybgP13.119D761630sucA>sucB>sucCsuccinyl-CoA synthetase4.720B794336ybhT>>*modAmolybdate transporter subunit9.221B822944cardiolipin synthase 2ybhO<<ybhP4.922B837732conserved proteinybiI<<ybiX5.723D844530rlmF>ybiO<glnQ9.024B874568yliE>>yliFpredicted diguanylate cyclase4.325D878444bssR>yliI<yliJ5.326D893136ybjN>potF>potGABC superfamily putrescine transporter4.127B956734ycaP>>serC3-phosphoserine aminotransferase6.228D1039338appC>appB>yccBhypothetical protein5.029D1048250conserved proteingfcC<gfcB<gfcA4.030B1062060modulator of CbpA co-chaperonecbpM<<cbpA5.231B1129438anti-sigma factor for FliA (sigma 28)*flgM<<flgA15.032B1133868flgF>>flgGflagellar component6.733B1137530flgJ>>*flgKflagellar hook-filament junction protein 127.234D1158658ptsG>fhuE>hinTpurine nucleoside phosphoramidase6.535B1193064tRNA methyltransferasemnmA<<nudJ5.336D1236068fadR>ycgB>dadAD-amino acid dehydrogenase6.937A1243852protein involved in flagellar function*ycgR<>ymgEpredicted inner membrane protein4.638B1349272enoyl-[acyl-carrier-protein] reductasefabI<<ycjD5.639D1356550predicted proteinymjA<puuP<puuA5.040D1392064ycjZ>mppA<ynaI6.541A1434934outer membrane pore protein NompN<>micCncRNA5.942B1488236aldA>>cybBcytochrome b56113.943D1504230predicted benzoate transporterydcO<ydcN>ydcPpredicted peptidase16.144B1644248ydfV>>*flxAQin prophage; predicted protein23.245D1661452ynfF>ynfG>ynfHoxidoreductase, membrane subunit5.546D1734472sodB>ydhP<ynhF6.147D1794968integration host factor (IHF)ihfA<pheT<pheS12.348A1801758threonyl-tRNA synthetasethrS<>yniDpredicted protein5.349B1815172conserved protein*chbG<<chbF4.250D1822736cho>*ves<spy10.851D1861032methionine sulfoxide reductase BmsrB<gapA>yeaDconserved protein4.252B1887966acyl-CoA synthetasefadD<<yeaY9.953A1906860predicted proteinmgrB<>yobHpredicted protein8.354D1914836methionine-(R)-sulfoxide reductaseyebR<yebS>yebTconserved protein14.455D1928332conserved proteinyebE<yebF<yebG4.356D1938762myristoyl-ACP-dependent acyltransferaselpxM<yebA<znuA11.957D1971862purine-binding chemotaxis proteincheW<cheA<motB4.358D1988066predicted protein*yecH<tyrP<yecA7.959B1999848RNA polymerase sigma 28*fliA<<fliC46.260D2233032yeiS>yeiT>yeiADihydropyrimidine dehydrogenase4.861D2262454fructose-specific PTS enzymes IIAfruB<setB<yeiW6.262D2379830N-acyltransferaseelaA<rbn>elaDpredicted enzyme12.563B2403570NADH:ubiquinone oxidoreductasenuoA<<lrhA3.564D2412454conserved inner membrane proteinyfbV<ackA>ptaphosphate acetyltransferase6.265D2456468phosphohistidine phosphatasesixA<fadJ<fadI4.566D2609062hyfH>hyfI>hyfJprocessing element hydrogenase 47.167D2684772serine hydroxymethyltransferaseglyA<hmp<glnB5.868D274556630S ribosomal subunit protein S16rpsP<ffh>ypjDpredicted inner membrane protein5.969D2868142L-isoaspartate carboxymethyltransfeerasepcm<surE<truD5.870D2916840barA>gudD<gudX5.471D2952958exonuclease V (RecBCD complex),recD<recB<ptrA8.072D2965066diacylglyceryl transferaselgt<ptsP<rppH6.773D3087058metK>galP>yggIconserved protein6.474D3124360glycolate oxidase iron-sulfur subunitglcF<glcE<glcD4.075C3146946yghZ><yqhA5.276B3197666deadenylyltransferase/adenylyltransferaseglnE<<ygiF8.877B3237654alx>>sstTsodium:serine/threonine symporter9.978D3338440ABC superfamily toluene transporteryrbF<yrbG>kdsDD-arabinose 5-phosphate isomerase6.279D3344772hpf>ptsN>yhbJprotein with NTP hydrolase domain5.980C3388568aaeR><tldD4.781D3452166type II secretion divergongspB<gspA>gspCgeneral secretory pathway component11.882D3454168general secretory pathway componentgspA<gspC>gspDgeneral secretory pathway component14.783D3478072predicted proteinyheV<kefB<kefG6.184D3515848predicted proteindamX<aroB<aroK4.585C3528652hslO><yhgE4.086D3534454TCS sensory histidine kinaseenvZ<ompR>*greBtranscription elongation factor4.387D3585866yhhA>ugpQ<ugpC18.188D3633746predicted proteinyhiJ<*yhiM<yhiN4.889D3689064endo-1,4-D-glucanasebcsZ<bcsB<bcsA4.690B3779166lldD>>yibKpredicted rRNA methylase9.291D3789732threonine 3-dehydrogenasetdh<kbl<htrL4.992A3826672glutamate transportergltS<>yicEpredicted transporter5.493D3829430yicE>yicH<yicI4.394B3906432phosphate transporterpstB<<pstA4.195D3945232trpT>*hdfR>yifEconserved protein4.796B3951430ilvE>>ilvDdihydroxyacid dehydratase8.697D4003446conserved proteinyigI<pldA>recQATP-dependent DNA helicase4.598D4032158yigZ>trkH>hemGprotoporphyrin oxidase, flavoprotein6.099B4068432predicted aldose-1-epimeraseyihR<<yihS9.6100D4178758rplJ>rplL>rpoBRNA polymerase, beta subunit6.7101D4286472acetyl-CoA synthetaseacs<nrfA>nrfBnitrite reductase5.8102D4327160conserved proteinyjdM<*yjdA>yjcZconserved protein12.4103D4338862biodegradative arginine decarboxylaseadiA<melR>melAalpha-galactosidase, NAD(P)-binding4.5104D4535430conserved proteinyjhX<yjhS<nanM4.7105A4589632predicted inner membrane proteinyjiY<>*tsrmethyl-accepting chemotaxis protein I5.3Genomic SELEX was performed for search of the binding sites of RNAP RpoF holoenzyme. By setting the cut-off level of 3.0, a total of 105 binding sites were identified (see Fig 3 for SELEX pattern), which are aligned along the map of E. coli K12 genome. A total of 37 sites are located within intergenic spacers: 7 wihin type-A spacers (shown under orange background); and 27 within type-B spacers (shown under green background). The constitutive promoters of RpoF were predicted based on the adjacent genes [note that only the genes next to the RpoF holoenzyme-binding sites are shown] and the gene orientation (shown by arrows in the column of transcription direction). A total of as many as 68 RpoF holoenzyme-binding sites are located inside open reading frames as indicated by the gene symbols shown in RpoF column.*The genes listed in RegulonDB as the regulated targets of RpoF.The highest peak was 46-fold intensity detected on promoter region of rpoF itself (Fig 3), indicating the autoregulation as already suggested [58]. A high-level peak was also identified upstream of the flgK gene, which encodes flagellar hook-filament junction protein that connects the filament to the hook, and its transcription has been shown in vitro under the direct control of RpoF [59]. The flgM gene encodes the anti-sigma factor for RpoF [55]. FlgM forms a complex with RpoF, thereby inactivating its sigma function but protects its degradation by the Lon protease for preservation [60].The whole set of constitutive promoters for extra-cytoplasmic stress response sigma RpoEThe bacterial cell envelope is a dynamic compartment, changing its structure and function in response to environmental conditions. Accordingly, the integrity of envelope is maintained through frequent modulation of its composition and components. The minor sigma factor RpoE plays a central role in this process, by controlling the selective expression of envelope components [61]. The regulatory targets have been estimated after proteome and transcriptome analyses in vivo [62–64]. The activity of RpoE is negatively regulated by a membrane-bound anti-sigma factor RseA, which sequesters RpoE under unstressed conditions. Within membrane, RseA is associated at its C-terminal domain with a periplasmic protein RseB, which senses misfolded proteins for release and activation of RpoE from RpoE-RseA complexes [62,65]. The total number of RpoE promoters (or the transcription initiation sites) listed in the current RegulonDB database is as many as 518, of which most are identified by computational analyses based on the consensus sequence of RpoE promoters without experimental analysis. After SELEX screening as noted below, most of these RpoE promoters must be inaccurate estimation due to the error in the consensus sequence.The Genomic SELEX screening system was employed as a short-cut approach for identification of the RpoE regulon. Previously we purified RpoE and examined its promoter selectivity using an in vitro transcription system [66]. Using this purified RpoE, we performed SELEX screening. By setting the cut-off level of 4.0 fold against original library DNA, a total of 126 RpoE holoenzyme-binding peaks were identified (Fig 4 and Table 5), of which 84 (67%) are located within intergenic spacers and 42 (33%) are inside of open reading frames (Tables 2 and 5 for the whole set). Since the majority of hitherto identified promoters are located within spacers, detailed search for the constitutive promoters was focused on the total of 84 peaks within spacers. The spacers containing RpoE holoenzyme-binding sites were also classified into three types (Table 2): 29 peaks are located within type-A spacer; 48 peaks are located within type-B spacers; and 7 peaks are located within type-C spacers. Based on the transcription direction of flanking genes, the total number of RpoE constitutive promoters was predicted to range between minimum 77 (29 type-A plus 48 type-B) and maximum 106 (29x2 type-A plus 48 type-B) (Table 2). Within the set of constitutive promoters identified by setting the cut-off level at 4.0, a total of 19 known RpoE promoters were identified (Table 5, marked by asterisk). The majority of known promoters represent the inducible promoters that are expressed only under the support of positive regulatory factors.10.1371/journal.pone.0179181.g004Fig 4SELEX-chip search for RNAP RpoE holoenzyme-binding sequences on the E. coli K-12 genome.The y-axis represents the relative number of RpoE holoenzyme-bound DNA fragments whereas x-axis represents the position on the E.coli genome, in base pair. The adjacent gene on E. coli K-12 genome of peak position was indicated for high intensity peaks. The peaks located within spacer regions are shown in green color, while peaks located within open reading frames are shown in orange color. The list of RpoE holoenzyme-binding sites is described in Table 5.10.1371/journal.pone.0179181.t005Table 5RpoE holoenzyme-binding sites on the E. coli K-12 genome.No.TypeMapLeft Gene FunctionLeftDRpoEDRightRight Gene FunctionIntensity1D154236fimbrial-like adhesin proteinyadM<htrE<ecpD5.02B164658hrpB>>mrcBglycosyl transferase and transpeptidase4.33B202068lpxD>>fabZhydroxymyristol acyl carrier dehydratase30.94C262270thrW><ykfI4.65D270562CP4-6 prophage transcriptional regulatorperR<insI<insH4.66B292358CP4-6 prophage; conserved proteinyagK<<yagL4.77D302738xanthine dehydrogenase 2Fe-2S subunityagT<yagU<ykgJ5.38A319448oxidoreductase with FAD-binding domain*ykgC<>ykgDDNA-binding transcriptional regulator4.09C323832ykgG><ykgH5.610B331456betT>>yahADNA-binding transcriptional regulator4.911d343660yahK>yahL>*yahMpredicted protein17.812A345542neutral amino-acid efflux systemyahN<>yahOpredicted protein4.013B477848conserved inner membrane proteinylaB<<ylaC4.514D543560conserved protein*ylbA<allC<allD9.615B581644nohB>>appYDLP12 DNA-binding transcriptional activator11.516B629042ybdB>>cstAcarbon starvation protein4.317B728638ybfA>>rhsCrhsC element core protein RshC5.618D732870ybfA>rhsC>ybfBpredicted inner membrane protein4.119D747240predicted regulatorabrB<*ybgO<ybgP9.320A753948citrate synthasegltA<>sdhCsuccinate dehydrogenase4.721B773834ybgE>>ybgCpredicted acyl-CoA thioesterase4.222B837732conserved proteinybiI<<ybiX4.123D838958conserved proteinybiX<fiu<mcbA4.324D844530rlmF>ybiO<glnQ6.025A862768pyruvate formate lyase activating enzymeybiY<>fsaAfructose-6-phosphate aldolase 14.426B887366ncRNA*rybB<<ybjL55.527B986332outer membrane porin 1a (Ia;b;F)ompF<<asnS5.028D992744pepN>ssuB<ssuC5.329D1045134exopolysaccharide export proteingfcE<gfcD<gfcC5.130C1049830insB><cspH6.331A1050632CspA-family stress protein*cspH<>cspGDNA-binding transcriptional regulator4.732D1088072predicted glycosyl transferasepgaC<pgaB<pgaA4.833B1166772ndh>>ycfJpredicted protein4.034C1195868icd><ymfD6.435B1218154ycgG>>*ymgFpredicted protein5.436D1259856ychH>ychM<prs5.337D1342036lipoproteinosmB<*yciT<yciZ4.338B1349272enoyl-[acyl-carrier-protein] reductasetfabI<<ycjD5.639B1384744ycjF>>tyrRDNA-binding transcriptional regulator5.040A1389946predicted hydrolaseycjY<>ycjZDNA-binding transcriptional regulator17.241B1432738Rac prophage transcriptional regulatorynaE<<uspF5.642D1468142KpLE2 phage-like IS repressor InsAinsC<insI>ydbCNAD(P)-binding oxidoreductase5.443D1500658tehB>*ydcL>ydcMpredicted transposase4.844B1518232yncB>>mcbRDNA-binding transcriptional regulator4.545B1542352nitrate/nitrite transporternarU<<yddJ4.946B1585730fimbrial-like adhesin proteinydeQ<<ydeR4.447B1588350fimbrial-like adhesin proteinydeS<<hipA4.548A1630638predicted mannonate dehydrogenaseydfI<>*ydfKQin prophage transcriptional regulator5.149B1638930Qin prophage; predicted S lysis proteinessQ<<cspB7.050A1639660Qin prophage; cold shock proteincspB<>*cspFQin prophage; cold shock protein7.351A1676254pyridine nucleotide transhydrogenasepntA<>ydgHpredicted protein5.052B1751846predicted oxidoreductaseydhV<<ydhY6.653A1801758threonyl-tRNA synthetasethrS<>yniDpredicted protein10.154D1852370predicted transporter*ydjE<*ydjF<ydjG4.455B1887966acyl-CoA synthetasefadD<<yeaY4.556A1923044conserved proteinyobA<>holEDNA polymerase III4.557D1938762myristoyl-acyl carrier ACP acyltransferaselpxM<*yebA<znuA7.158A2060070DNA-binding transcriptional dual regulatornac<>asnVAsn tRNA4.459D2103368predicted acyl transferasewbbJ<wbbI<rfc4.260D2104870conserved proteinwbbI<rfc<glf5.761D2144358uridine/cytidine kinaseudk<yegE<alkA5.862B2196338metG>>yehIconserved protein8.663D2210570conserved proteinyehS<yehT<yehU5.264D2214150yohO>yehW<yehX4.065D2302258yojO>eco<mqo7.266D2369270arnD>arnT>arnEconserved protein5.967A2405036DNA-binding transcriptional repressorlrhA<>yfbQpredicted aminotransferase4.068D24411383-oxoacyl-[acyl-carrier-protein] synthase IfabB<mnmC<yfcL5.769A2459264conserved proteinyfcZ<>fadLlong-chain fatty acid OM transporter5.370D2484966evgA>evgS<yfdE5.271D2490262predicted oxalyl-CoA decarboxylase*oxc<frc<yfdX4.972D2492854predicted proteinyfdX<*ypdI<yfdY4.173A2507446glucokinaseglk<>yfeOpredicted ion channel protein4.374A2535342pyridoxal-pyridoxamine kinasepdxK<>*yfeKpredicted protein7.575D2649866fused transglycosylase/transpeptidasepbpC<yfhM>sseA3-mercaptopyruvate sulfurtransferase6.176A2696642conserved proteinyfhB<>yfhHDNA-binding transcriptional regulator4.777A2708134RNA polymerase sigma 24rpoE<>nadBquinolinate synthase6.378B2761570CP4-57 prophage proteinyfjK<<yfjL5.679B2763272CP4-57 prophage proteinyfjL<<yfjM3.280D2764730CP4-57 prophage proteinyfjM<rnlA>yfjOCP4-57 prophage protein6.181B2773454yfjW>>yfjXCP4-57 prophage antirestriction protein6.182C2796040ygaP><stpA4.383A2812736S-ribosylhomocysteinaseluxS<>*micAncRNA40.984A2874372sulfate adenylyltransferasecysD<>*iapaminopeptidase4.285D2881556predicted proteinygcK<ygcL<ygcB10.986D2894072predicted flavoproteinygcQ<ygcR<ygcS4.087D2903952conserved proteinygcF<*ygcG<eno4.588B2985164yqeG>>yqeHprotein with bipartite regulator domain4.089A3004270DNA-binding transcriptional regulatorygeV<>*ygeWconserved protein4.590D3036652ssDNA exonucleaserecJ<*dsbC<xerD5.191B3237758alx>>sstTsodium:serine/threonine symporter23.492D3246540yqjA>yqjB>yqjCconserved protein4.493A3276944DNA-binding transcriptional dual regulatoragaR<>kbaZtagatose 6-phosphate aldolase 14.294B3375554stringent starvation protein AsspA<<rpsI4.795B3387142p-hydroxybenzoic acid efflux systemaaeA<<aaeX4.996C3528652hslO><yhgE4.197A3559934thiosulfate:cyanide sulfurtransferaseglpE<>glpDsn-glycerol-3-phosphate dehydrogenase4.098B3576850DNA-binding transcriptional repressorgntR<<yhhW4.999A3579162ncRNAryhB<>yhhYpredicted acetyltransferase6.5100B3599044RNA polymerase sigma 32*rpoH<<ftsX37.4101C3661646mdtF><gadW6.7102D3666862glutamate decarboxylase AgadA<yhjA>treFcytoplasmic trehalase6.6103B3720058insK>>sokAncRNA7.3104B37396604Fe-4S ferredoxin-type hydrogenaseysaA<<yiaJ4.2105D3764054predicted glutathione S-transferaseyibF<rhsA>yibAlyase containing HEAT-repeat4.0106B3779166lldD>>yibKpredicted rRNA methylase10.8107B3828470yicE>>yicHconserved protein4.2108D3829430yicE>yicH<yicI8.2109D3862372yidL>yidP<yidE4.3110B3906432phosphate transporter subunitpstB<<pstA5.2111D3955270ilvA>ilvY>ilvCketol-acid reductoisomerase4.1112A3958552peptidyl-prolyl cis-trans isomerase CppiC<>repDNA helicase/ssDNA-dependent ATPase4.8113B3978772rffM>>yifKpredicted transporter4.0114B4068538predicted aldose-1-epimeraseyihR<<yihS10.7115B4173964thrT>>tufBprotein chain elongation factor EF-Tu5.7116B4187734rpoC>>yjaZheat shock protein4.0117A4212172predicted acetyltransferaseyjaB<>metAhomoserine O-transsuccinylase4.3118B4249366malM>>ubiCchorismate pyruvate lyase5.7119B4364770C4-dicarboxylate antiporterdcuA<<aspA5.2120A4368630predicted transporteryjeH<>groSCpn10 chaperonin GroES4.3121A4380530anaerobic fumarate reductasefrdA<>poxApredicted lysyl-tRNA synthetase4.0122B4427854fklB>>cycAD-alanine/D-serine/glycine transporter8.0123D4486746yjgQ>yjgR<idnR9.4124B4530030KpLE2 phage endoglucanasesgcX<<yjhP5.2125B4533162conserved proteinyjhX<<yjhS4.0126A4538964N-acetylnuraminic acid outer membrane channel proteinnanC<>fimBtyrosine recombinase/fimA regulator6.4127B4638610yjjY>>*yjtDpredicted rRNA methyltransferase4.6Genomic SELEX was performed for search of the binding sites of RNAP RpoE holoenzyme. By setting the cut-off level of 4.0, a total of 126 binding sites were identified (see Fig 4 for SELEX pattern), which are aligned along the map of E. coli K12 genome. A total of 84 sites are located within intergenic spacers: 29 wihin type-A spacers (shown under orange background); and 48 within type-B spacers (shown under green background). The constitutive promoters of RpoE were predicted based on the adjacent genes [note that only the genes next to the RpoE holoenzyme-binding sites are shown] and the gene orientation (shown by arrows in the column of transcription direction). A total of 42 RpoE holoenzyme-binding sites are located inside open reading frames as indicated by the gene symbols shown in RpoE column.*The genes listed in RegulonDB as the regulated targets of RpoE.Genomic SELEX analysis identified minimum 77 and maximum 106 for the RpoE constitutive promoters. The highest peak (55-fold intensity) was detected in the promoter region of rybB, which encodes a small regulatory RNA for expression control of some outer membrane proteins. The rybB promoter is known to be regulated by RpoE (Fig 4) [67]. The second highest peak was located on the luxS-micA intergenic region. The micA gene again encodes a small regulatory RNA that regulates expression of many genes including outer membrane proteins [68,69]. The micA promoter is also established under the control of RpoE. These sRNAs control the repair of damages in the outer membrane that took place in response to envelope stress [70,71]. High-intensity peaks were detected on some other known RpoE-dependent promoters such as rpoH, pgrR and ycjY [72,73]. Among the total of 136 binding sites of RpoE holoenzyme, 36 overlaps with that of RpoH holoenzyme. Most of these overlapping sites are related to the genes that are expressed under envelope stresss or heat-shock stress.The intracellular levels of all seven sigma factors in E. coli K-12 W3110In this study, we determined the constitutive promoters for the four minor sigma factors, RpoS, RpoH, RpoF and RpoE, from E. coli K-12 W3110. These promoters are recognized by the RNAP holoenzyme containing each sigma in the absence of other supporting factors. Using the mixed reconstitution in vitro of RNAP holoenzyme in the presence of all seven sigma factors, we estimated the binding affinity of each sigma to the common core enzyme, the order being RpoD (highest) > RpoN > RpoF > RpoH > FecI > RpoE > RpoS (lowest) [74]. Once we get the knowledge of intracellular concentrations of these sigma factors, it should be possible to predict the expression levels of the regulatory target genes and operons under the control of the constitutive promoters of each sigma factor. Including these four minor sigma factors, we then determined the intracellular concentrations of all seven sigma subunits. For this purpose, antibodies were made against each of the purified sigma factors that were also used for SELEX screening.E. coli K-12 W3110 type-A was cultivated with shaking at 37°C in LB medium, and the whole cell lysates were prepared in both exponential growing phase and the stationary phase. By using the quantitative immuno-blotting method and the purified sigma proteins as the reference controls, we measured the concentrations of all seven sigma subunits. The measurement was carried out for two independent cultures, and the immuno-blot analysis was repeated for all the samples. The intracellular concentration of RpoD sigma is maintained at a constant level (on average, 160 fmol/μg total protein) throughout the transition from the exponential growth phase to the stationary phase (Fig 5) in good agreement with the previous estimation [1,75]. Based on the relative level of RNAP core enzyme subunits, the number of RpoD sigma was estimated to be 800 molecules per genome [note that the number of RNAP core enzyme is 2,000 molecules per genome]. The level of RpoD sigma in E. coli K-12 MG1655 was about half the level of W3110 (data not shown). The second abundant sigma was RpoF (70 fmol/μg protein at log phase and 80 fmol/μg protein at stationary phase) (Fig 5), but the unneccesary RpoF is stored in an inactive form by forming complexes with the anti-RpoF sigma, FlgM [76]. RpoN is also always present at a constant level (approximately 40 fmol/μg protein) in both log and stationary phases (Fig 5). The levels of other four sigma, RpoS, RpoH, RpoE and FecI, are very low during the steady-state growth (Fig 5A), but upon entry into the stationary phase, the level of RpoS increased markedly up to the level (about 100 fmol/μg protein) higher than other five minor sigma factors (Fig 5B). Taking the intracellular concentrations and the binding affinity of sigma to RNAP core enzyme as noted above, we are now able to estimate the level of each RNAP holoenzyme. Noteworthy is that the total number of all seven sigma factors is approximately as many as that of the core enzyme, but the RNAP involved in the transcription cycle or the elongation of RNA chains is considered to lack sigma subunit, the RNAP not involved in transcription should be stored as the holoenzyme forms.10.1371/journal.pone.0179181.g005Fig 5Intracellular concentrations of seven sigma factors in E. coli K-12 W3110 type-A strain.E. coli K-12 W3110 type-A strain was grown in LB medium at 37°C with shaking. Cells were harvested at various times and cell lysates were subjected to the quantitative immuno-blot analysis of all seven sigma factors as described in Materials and Methods. [A] The sigma levels at exponential growth phase; [B] the sigma levels in the stationary phase.

Genomic SELEX screening for the constitutive promotersThe constitutive promoters are transcribed in vitro by the RNA polymerase holoenzyme alone in the absence of supporting factors. In order to identify the whole set of constitutive promoters on the entire genome of E. coli K-12 W3110, we performed a mass-screening in vitro of the whole set of sequences that are recognized by the reconstituted holoenzymes, each containing only one specific minor sigma factor. The sigma-free core enzyme was prepared by passing the purified RNA polymerase three times through phosphocellulose column chromatography in the presence of 5% glycerol, the stabilizer of holoenzyme complexes in the storage buffer [29]. The level of remaining sigma subunits was less than 0.1%, if any, as detected by both protein staining and immuno-staining against each of all seven species of sigma factors (RpoD, RpoN, RpoS, RpoH, RpoF, RpoE and FecI). The stoichiometry between core enzyme subunits was also checked by immune-staining with antibodies against the core subunits, RpoA, RpoB, RpoC and RpoZ. The holoenzymes fully saturated with each sigma subunit were reconstituted by mixing this sigma-free core enzyme and 4-fold molar excess of purified sigma factors, RpoS, RpoH, RpoF and RpoE. Since these sigma subunits alone are unable to bind to promoter DNA, the presence of excess sigma does not interfere with the function of RNAP holoenzymes. For the identification of DNA sequences that are recognized by each holoenzyme, we employed the Genomic SELEX screening system [26], in which a library of E. coli genome DNA fragments of 200–300 bp in length was used instead of synthetic oligonucleotides with all possible sequences used in the original SELEX method [30–32].The multi-copy plasmid library of 200–300 bp-long random DNA fragments was constructed from the E. coli K-12 W3110 genome [26]. The library used in this study contained 6.5-fold molar excess of the entire genome, and thus a single and the same sequence might be included in 6 different overlapping segments on average, thereby increasing the resolution of mapping of SELEX fragments. In each experiment of Genomic SELEX screening, the mixture of genome DNA fragments, which was regenerated by PCR from the genome DNA library, was mixed with 2-fold molar excess of the reconstituted each RNAP holoenzyme, and subjected to Genomic SELEX screening. DNA-holoenzyme complexes formed were recovered using the anti-RpoC antibody, which gave the highest level of RNAP recovery among all the anti-core subunit antibodies. RNA polymerase-associated DNA was isolated from the antibody precipitates, amplified by PCR, and subjected to next cycles of SELEX. After repeated SELEX screening, the final products of holoenzyme-bound DNA fragments were subjected to mapping on the genome using a DNA tilling microarray (Oxford Gene Technology, Oxford, UK) [14]. The binding intensity was measured as the ratio of holoenzyme-bound DNA labeled by Cy3 against original library DNA labeled by Cy5 on an array and plotted along E. coli genome about each holoenzyme. On the DNA tilling array used, the 60 b-long probes are aligned along the E. coli genome at 105 bp-intervals, and therefore approximately 300 bp-long SELEX fragments should bind to two or more consecutive probes. This criterion was employed to avoid the background noise of non-specific binding of holoenzyme-bound DNA fragments to the tilling array [note that peaks showing hybridization to only a single probe was judged as a false-positive noise].The binding sites were classified into two groups, one ‘within spacers’ and another ‘inside genes’. The binding sites on ‘within spacers’ were further classified into 3 types; type-A spacer located between bidirectional transcription units, type-B spacer located upstream of one transcription unit but downstream of another transcription unit, and type-C spacer located downstream of both transcription units. Based on the transcription direction of flanking genes, the total number of the constitutive promoters was predicted to range between the minimum [number of type-A spacer plus number of type-B spacer] and the maximum [number of type-A spacer x 2 plus number of type-B spacer]. The intragenic binding site was referred to type-D site. The height of binding intensity identified by SELEX-chip system is generally in good agreement with the number of clones identified by SELEX-clos (cloning-sequencing) system, indicating that these two parameters correlate with the binding affinity of test TF to DNA [4,5,8,9].

Genomic SELEX screening for the constitutive promoters

The constitutive promoters are transcribed in vitro by the RNA polymerase holoenzyme alone in the absence of supporting factors. In order to identify the whole set of constitutive promoters on the entire genome of E. coli K-12 W3110, we performed a mass-screening in vitro of the whole set of sequences that are recognized by the reconstituted holoenzymes, each containing only one specific minor sigma factor. The sigma-free core enzyme was prepared by passing the purified RNA polymerase three times through phosphocellulose column chromatography in the presence of 5% glycerol, the stabilizer of holoenzyme complexes in the storage buffer [29]. The level of remaining sigma subunits was less than 0.1%, if any, as detected by both protein staining and immuno-staining against each of all seven species of sigma factors (RpoD, RpoN, RpoS, RpoH, RpoF, RpoE and FecI). The stoichiometry between core enzyme subunits was also checked by immune-staining with antibodies against the core subunits, RpoA, RpoB, RpoC and RpoZ. The holoenzymes fully saturated with each sigma subunit were reconstituted by mixing this sigma-free core enzyme and 4-fold molar excess of purified sigma factors, RpoS, RpoH, RpoF and RpoE. Since these sigma subunits alone are unable to bind to promoter DNA, the presence of excess sigma does not interfere with the function of RNAP holoenzymes. For the identification of DNA sequences that are recognized by each holoenzyme, we employed the Genomic SELEX screening system [26], in which a library of E. coli genome DNA fragments of 200–300 bp in length was used instead of synthetic oligonucleotides with all possible sequences used in the original SELEX method [30–32].

The multi-copy plasmid library of 200–300 bp-long random DNA fragments was constructed from the E. coli K-12 W3110 genome [26]. The library used in this study contained 6.5-fold molar excess of the entire genome, and thus a single and the same sequence might be included in 6 different overlapping segments on average, thereby increasing the resolution of mapping of SELEX fragments. In each experiment of Genomic SELEX screening, the mixture of genome DNA fragments, which was regenerated by PCR from the genome DNA library, was mixed with 2-fold molar excess of the reconstituted each RNAP holoenzyme, and subjected to Genomic SELEX screening. DNA-holoenzyme complexes formed were recovered using the anti-RpoC antibody, which gave the highest level of RNAP recovery among all the anti-core subunit antibodies. RNA polymerase-associated DNA was isolated from the antibody precipitates, amplified by PCR, and subjected to next cycles of SELEX. After repeated SELEX screening, the final products of holoenzyme-bound DNA fragments were subjected to mapping on the genome using a DNA tilling microarray (Oxford Gene Technology, Oxford, UK) [14]. The binding intensity was measured as the ratio of holoenzyme-bound DNA labeled by Cy3 against original library DNA labeled by Cy5 on an array and plotted along E. coli genome about each holoenzyme. On the DNA tilling array used, the 60 b-long probes are aligned along the E. coli genome at 105 bp-intervals, and therefore approximately 300 bp-long SELEX fragments should bind to two or more consecutive probes. This criterion was employed to avoid the background noise of non-specific binding of holoenzyme-bound DNA fragments to the tilling array [note that peaks showing hybridization to only a single probe was judged as a false-positive noise].

The binding sites were classified into two groups, one ‘within spacers’ and another ‘inside genes’. The binding sites on ‘within spacers’ were further classified into 3 types; type-A spacer located between bidirectional transcription units, type-B spacer located upstream of one transcription unit but downstream of another transcription unit, and type-C spacer located downstream of both transcription units. Based on the transcription direction of flanking genes, the total number of the constitutive promoters was predicted to range between the minimum [number of type-A spacer plus number of type-B spacer] and the maximum [number of type-A spacer x 2 plus number of type-B spacer]. The intragenic binding site was referred to type-D site. The height of binding intensity identified by SELEX-chip system is generally in good agreement with the number of clones identified by SELEX-clos (cloning-sequencing) system, indicating that these two parameters correlate with the binding affinity of test TF to DNA [4,5,8,9].

The whole set of constitutive promoters for the stationary-phase sigma RpoSIn laboratory culture of E. coli, cell growth enters into the stationary phase mainly due to the limited availability of nutrients. Upon entry into the stationary phase, the pattern of genome expression is markedly altered by turning down the growth-related genes and instead up-regulation of the stress-response genes. In switching the transcription pattern, the stationary-phase specific minor sigma RrpoS is involved [33,34]. The rpoS gene is not essential for growth under non-stress conditions, but strains carrying mutations affecting rpoS activity are extremely sensitive to environmental stresses. As in the case of other sigma factors, RpoS interacts with RNAP core enzyme and modulates its promoter recognition specificity so as to recognize a specific but large set of genes.As noted above, the set of genes identified in vivo include a number of genes under the indirect control of RpoS. On the other hand, some target promoters of RpoS are masked in vivo due to competitive interference by other DNA-binding proteins. In order to identify the constitutive promoters directly recognized by RpoS in the absence of other DNA-binding proteins, the Genomic SELEX screening in vitro was performed using the reconstituted RNAP RpoS holoenzyme. The sequences with binding affinity to the RpoS holoenzyme formed a number of peaks along the entire E. coli genome (Fig 1). Location of peaks was aligned along the map of E. coli K-12 genome (Table 1). By setting the cut-off level of 3.0 fold-higher intensity over the background of original library DNA, a total of 218 peaks were identified, of which 125 (67%) are located within intergenic spacers and 73 (33%) are inside of open reading frames (Table 2). Since the majority of hitherto identified promoters are located within spacers and generally upstream of open reading frames, detailed search for the constitutive promoters was focused on these 125 spacer peaks. These spacers can be classified into three types: 50 peaks are located within type-A spacer between bidirectional transcription units; 79 peaks are located within type-B spacers located upstream of one transcription unit but downstream of another transcription unit; and 16 peaks are located within type-C located spacers downstream of both transcription units. Based on the transcription direction of flanking genes, the total number of RpoS constitutive promoters was predicted to range between minimum 129 (50 type-A plus 79 type-B) and maximum 179 (50x2 type-A plus 79 type-B) (Table 2). Type-A spacers should contain two promoters for bidirectional transcription, at least one of which should be RpoS-dependent promoter. The RpoS holoenzyme-binding sites identified in a total of 50 type-A spacers should represent promoters for one or both of bidirectional transcription.10.1371/journal.pone.0179181.g001Fig 1SELEX-chip search for RNAP RpoS holoenzyme-binding sequences on the E. coli K-12 genome.The y-axis represents the relative number of RpoS holoenzyme-bound DNA fragments whereas x-axis represents the position on the E. coli K-12 genome, in base pair. The adjacent gene on E. coli K-12 genome of peak position was indicated for high intensity peaks. The peaks located within spacer regions are shown in green color, while peaks located within open reading frames are shown in orange color. The list of RpoS holoenzyme-binding sites is described in Table 1.10.1371/journal.pone.0179181.t001Table 1RpoS holoenzyme-binding sites on the E. coli K-12 genome.NoTypeMapGene FunctionLeftDRpoSDRightGene FunctionIntensity1D17732sokC>nhaA>nhaRtranscriptional activator5.32A41946predicted transportercaiT<>fixAelectron transfer flavoprotein3.63B63256RNAP-associated helicasehepA<<polB3.84D238050aspV>yafT<ykfM19.05B251970dinB>>yafNYafO-YafN toxin-antitoxin system45.16B289870ornithine carbamoyltransferase 2argF<<insB25.07D296234CP4-6 prophage proteinyagN<intF>ptwFXaa tRNA70.48A296336CP4-6 prophage integraseintF<>ptwFXaa tRNA45.59A328672transcriptional repressorbetI<>betTcholine transporter3.810B379186frmRAB operon regulatorfrmR<<yaiO50.811A383868conserved proteinyaiS<>tauAtaurine transporter24.112D451740cytochrome o ubiquinol oxidasecyoA<ampG<yajG5.913B455746tig>>clpPATP-dependent serine protease7.414B467530cof>>ybaODNA-binding transcriptional regulator11.515D479868modulator of gene expression with H-NShha<tomB<acrB45.116B480454predicted proteintomB<<acrB22.617A515034membrane anchored proteaseqmcA<>ybbLABC superfamily transporter70.318D532860allA>allR>gclglyoxylate carboligase3.519D562966sfmH>sfmF<fimZ45.320D569366emrE>ybcK>ybcLDLP12 prophage kinase inhibitor6.821B570042ybcK>>ybcLDLP12 prophage kinase inhibitor19.722A576032IS5 transposase and trans-activatorinsH<>essDDLP12 prophage lysis protein34.023B582672nohB>>*appYDLP12 prophage transcriptional activator12.924D583166nohB>appY<ompT11.425D606570predicted inner membrane proteinybdJ<ybdK>hokEtoxic polypeptide17.326A651466citrate lyase synthetasecitC<>dpiBCitBA TCS sensory histidine kinase8.327D654932dpiA>dcuC>pagPpalmitoyl transferase for Lipid A60.728D659134tatE>lipA<ybeF5.929B692668nucleoside triphosphate hydrolaseybeZ<<miaB19.930C720060ybfK><kdpE3.631B770146mngB>>cydAcytochrome d terminal oxidase, subunit I127.532D799356pgl>ybhD>ybhHconserved protein4.833A812472hypothetical proteinybhU<>uvrBnucleotide excision repair nuclease12.934B837732conserved protein*ybiI<<ybiX19.635A849568threonine and homoserine efflux systemrhtA<>ompXouter membrane protein5.436A852256ncRNArybA<>mntRDNA-binding transcriptional regulator26.037A877270SAM-dependent methyltransferaseyliG<>bssRconserved protein5.338B891170nfsA>>rimKribosomal protein S6 modification protein4.139B903170arginine transporter*artP<<ybjP154.240A931668thioredoxin reductasetrxB<>lrpDNA-binding transcriptional dual regulator5.941D943972dmsB>dmsC<ycaC24.342B953940formate transporterfocA<<ycaO3.443B959450aroA>>ycaLpeptidase with chaperone function6.644B962934rpsA>>*ihfBintegration host factor (IHF)3.645D1028042hemimethhylated DNA-binding proteinhspQ<yccW>yccXpredicted acylphosphatase40.546C1049830insB><cspH167.247B1084156efeB>>phoHconserved protein with NTPase domain62.748B1120372predicted proteinbssS<<dinI6.649B1120772DNA damage-inducible protein IdinI<<pyrC3.750D1144848yceQ>rluC<yceF23.751D1147330rpmF>plsX>fabH3-oxoacyl-[acyl-carrier-protein] synthase III31.252A1168238DNA-binding transcriptional regulatorycfQ<>bhsApredicted protein17.653D1187848conserved proteinycfD<phoQ<phoP5.554D12110305-methyl-C-specific restriction nucleasemcrA>elbA<ycgX4.755D1212068predicted proteinelbA<ycgX<ycgE27.856A1214962predicted FAD-binding phosphodiesteraseycgF<>*ycgZpredicted protein14.257B1215830ariR>>ymgCpredicted protein6.758C1218960ymgF><ymgD15.159A1250254dihydroxyacetone kinase*dhaK<>*dhaRDNA-binding transcriptional regulator15.760A1257834peptidyl-tRNA hydrolasepth<>ychHpredicted inner membrane protein72.561D1318064indole-3-glycerol-P synthetasetrpC<trpD<trpE73.662B1341438lipoprotein*osmB<<yciT3.463D1342872hypothetical proteinyciZ<gmr<rnb48.264A1359040gamma-Glu-putrescine synthase*puuA<>*puuDgamma-Glu-GABA hydrolase21.865D1393856mppA>ynaI>insHIS5 transposase and trans-activator4.066B1432638DNA-binding transcriptional regulatorynaE<<uspF10.267A1438872pyruvate-flavodoxin oxidoreductaseydbK<>ydbJpredicted protein5.168D1447730feaB>tynA<maoC5.069C1463278paaY><insD6.270A1489640ncRNArydC<>ydcApredicted protein14.971B1500460tehB>>ydcLpredicted lipoprotein7.472A1515332hypothetical proteinyncL<>ydcXpredicted inner membrane protein28.573B1518146yncB>>mcbRDNA-binding transcriptional regulator4.474D1528272yncH>ydcD>ydcCconserved protein9.275B1543272predicted proteinyddJ<<yddG3.576D1553260dehydrogenase/acetaldehyde reductaseadhP<maeA<sra4.977D1566238predicted diguanylate cyclaseyddV<yddW<gadC14.578B1570272glutamate decarboxylase B*gadB<<pqqL14.979B1580646conserved proteinydeN<<ydeO8.080B1590548DNA-binding transcriptional regulatorhipB<<ydeU94.481B1596530predicted lipoproteinydeK<<lsrK12.982D1613844yneJ>yneK>ydeApredicted arabinose transporter4.383B1618030marA>>marBpredicted protein3.984B1627238ydfH>>ydfZconserved protein5.585A1630740predicted mannonate dehydrogenaseydfI<>ydfKQin prophage transcriptional regulator10.886A1655452predicted proteinynfC<>ynfDpredicted protein32.387B1682244rstB>>tusinhibitor of replication at Ter77.288B1687868manA>>ydgAconserved protein132.489B1745130ydhR>>ydhSprotein with FAD/NAD(P)-binding domain3.890A1753168predicted proteinydhZ<>*pykFpyruvate kinase I4.291D1769372conserved proteinydiL>ydiM>ydiNpredicted transporter3.292C1793764integration host factor (IHF)ihfA<pheT<pheS8.193D1811050ydjN>ydjO<cedA29.294C1841754gdhA><ynjI79.595B1894766nudL>>sdaAL-serine deaminase I55.496B1905768predicted proteinyobF<<yebO11.297D1921260ncRNAryeA>ryeB<yebY11.598B1927030protease IIptrB<<yebE6.199A1928846conserved protein regulated by LexAyebG<>purTP-ribosylglycinamide formyltransferase 263.2100D1928972conserved protein regulated by LexAyebG<purT<eda390.6101A1944202RuvABC resolvasomeruvA<>yebBpredicted protein11.3102B1956162TMAO reductase III (TorYZ)torY<<cutC17.4103C1966932CheAB TCS chemotaxis regulatorcheB<cheR<tap8.7104B1994970DNA-binding transcriptional activatorsdiA<<yecC21.7105A2023030predicted protein*dsrB<>yodDpredicted protein88.4106B2031954rseX>>*hchAHsp31 molecular chaperone7.7107D2049968asnT>yeeJ>shiAshikimate transporter150.8108D2057732Asn tRNAasnW<yeeO>asnUAsn tRNA3.2109B2061434L,D-transpeptidase linking Lpp to mureinerfK<<cobT44.9110D2103732predicted acyl transferasewbbJ<wbbI<rfc4.3111D2104956conserved proteinwbbI<rfc<glf14.6112D2106934UDP-galactopyranose mutaseglf<rfbX<rfbC54.2113B2134130protein-tyrosine phosphatasewzb<<wza53.9114B2226932NAD(P)-binding oxidoreductaseyohF<<dusC3.3115A2247636DNA-binding transcriptional regulatoryeiE<>yeiHconserved inner membrane protein12.7116D2255672predicted nucleoside transporterpsuT<psuG<psuKpseudouridine kinase7.7117B2284170yejM>>proLPro tRNA9.5118A2311066outer membrane porin protein CompC<>micFncRNA16.1119B2311354micF>>rcsDRcsBC TCS phosphotransfer protein13.8120D2389234yfbP>nuoN<nuoM13.0121B2454170predicted fimbrial-like adhesin proteinyfcV<<sixA25.4122D2467360yfdH>yfdI<yfdK38.3123C2468764yfdI><yfdK26.5124D2480972predicted multidrug efflux systememrY<emrK>*evgAEvgAS TCS response regulator126.6125A2510860manganese/divalent cation transportermntH<>nupCnucleoside (except guanosine) transporter13.5126D2532356ptsH>ptsI>*crrglucose-specific PTS enzyme IIA9.2127B2574144carboxysome structural proteineutS<<maeB11.3128D2587966narQ>acrD<ypfM18.4129B2651536sseA>>ryfAncRNA40.6130B2689548ncRNAglmY<<purL24.9131B2753630smpB>>ssrAtmRNA60.7132B2763338CP4-57 prophage; predicted proteinyfjL<<yfjM13.8133B2765760yfjO>>yfjPCP4-57 prophage GTP-binding protein5.2134D2771468yfjT>yfjW>yfjXCP4-57 prophage antirestriction protein32.6135D2772262yfjT>yfjW>yfjXCP4-57 prophage antirestriction protein26.4136B2773166yfjW>>yfjXCP4-57 prophage antirestriction protein41.0137D2779640psaA>ypjA<ileY28.8138B2783272adhesin-like autotransporterypjA<<ileY58.9139A2784466Ile tRNAileY<>*csiDpredicted protein3.7140A2786358Ile tRNAileY<>*csiDpredicted protein31.8141A2795168predicted membrane proteinyqaE<>ygaVDNA-binding transcriptional regulator53.2142B2882250predicted proteinygcL<<ygcB6.3143A2903434conserved proteinygcF<>ygcGpredicted protein57.0144B2985568yqeG>>yqeHprotein with bipartite regulator domain32.6145C2987942yqeJ><yqeK9.1146D2990738ygeG>ygeH>ygeIpredicted protein6.8147C2992070ygeI><insD6.4148C2992950ygeI><insD6.9149C2993358ygeI><insD13.2150B3134436phosphate transporterpitB<<gsp12.6151A3145934predicted proteinyghW<>yghZaldo-keto reductase3.2152B3166762predicted cyanide hydratasemqsR<<ygiV65.1153C3181642zupT><ribB10.0154B3183246yqiC>>ygiLpredicted fimbrial-like adhesin protein3.7155D3189250yqiH>yqiI<glgS68.5156D3210472rpsU>dnaG>rpoDRNA polymerase sigma 7021.3157D3259830L-PSP (mRNA) endoribonucleasetdcF<tdcE<tdcD3.6158D3266232tdcR>yhaB>yhaCpredicted protein9.6159C3326238yhbY><greA3.6160B3335948ABC-type organic solvent transporteryrbC<<yrbD14.0161D3345766ptsN>yhbJ>nprN-regulated PTS system (Npr)75.4162D3348770ncRNAryhA>arcB<yhcC5.9163D3395840ribonuclease Grng<yhdE<mreD4.3164D3453834general secretory pathway componentgspA<gspC>gspDgeneral secretory pathway component9.3165B3467930periplasmic endochitinasechiA<<tufA5.0166B3497370cysG>>yhfLconserved secreted peptide13.0167D3533142pck>envZ<ompR41.3168D3542562yhgA>bioH>gntXgluconate periplasmic binding protein15.9169B3576742DNA-binding transcriptional repressorgntR<<yhhW12.2170B3582734yrhD>>yrhBpredicted protein4.3171A3584864gamma-glutamyltranspeptidaseggt<>yhhAconserved protein12.9172D3604756yhhN>zntA<sirA8.8173B3621930yhhH>>yhhIpredicted transposase3.3174A3632156predicted proteinyhiJ<>*yhiMconserved inner membrane protein32.2175A3632742predicted proteinyhiJ<>*yhiMconserved inner membrane protein13.3176A3637868universal stress protein B*uspB<>uspAuniversal stress global response regulator22.7177D3647644arsR>arsB>arsCarsenate reductase31.8178C3648872arsC><insH153.6179B3656130hdeD>>gadEDNA-binding transcriptional activator9.1180B3708672predicted metal dependent hydrolaseeptB<<yhjX48.9181B3717944yiaG>>cspAmajor cold shock protein6.1182B3720058insK>>sokAncRNA31.7183A3749938predicted proteinyiaT<>yiaUDNA-binding transcriptional regulator22.4184B3794944rfaC>>rfaLO-antigen ligase3.8185D3796942rfaL>waaU<rfaZ46.2186D3798472lipopolysaccharide core synthesis proteinrfaZ<rfaY<rfaJ145.8187D3800958UDP-D-glucose:LPS glucosyltransferaserfaJ<rfaI<rfaB15.8188D3802448UDP-galactose:LPS galactosyltransferaserfaB<rfaS<rfaP21.5189B3834856Sec tRNAselC>>setCpredicted sugar efflux system22.6190A3851272ncRNAistR<>tisBlexA-regulated toxic peptide57.3191B3886640tnaC>>tnaAtryptophanase/L-cysteine desulfhydrase29.5192D4001054conserved inner membrane proteinyigF<yigG<rarD5.5193D4002164predicted inner membrane proteinyigG<rarD<yigI13.0194B4076572yiiD>>yiiEpredicted transcriptional regulator4.5195D4110740conserved proteinyiiQ<yiiR>yiiSconserved protein10.8196A4116232glycerol facilitatorglpF<>yiiUconserved protein15.6197B4120330HslUV proteasehslV<<ftsN3.6198D4220334aceK>arpA<iclR5.4199B4281230conserved proteinyjcF<<actP83.6200A4380530fumarate reductasefrdA<>poxApredicted lysyl-tRNA synthetase4.7201A4417636L-ascorbate 6-phosphate lactonaseulaG<>ulaAL-ascorbate-specific PTS enzyme IIC4.9202A4432132NAD(P)H:quinone oxidoreductaseytfG<>ytfHpredicted transcriptional regulator90.2203A44345622':3'-cyclic-nucleotide 2'-phosphodiesterasecpdB<>cysQPAPS 3'(2'),5'-bisphosphate nucleotidase4.3204B4472740yjgJ>>yjgKconserved protein15.2205C4475268yjgL><argI80.9206A4477770predicted acetyltransferaseyjgM<>yjgNconserved inner membrane protein4.1207A4492648L-idonate 5-dehydrogenase, NAD-bindingidnD<>idnKD-gluconate kinase, thermosensitive6.0208B4504448yjhC>>ythAexpressed protein8.4209C4505142ythA><insI5.3210A4538166N-acetylnuraminic acid OM channel proteinnanC<>fimBTyr recombinase/fimA inversion regulator5.5211A4538758N-acetylnuraminic acid OM channel proteinnanC<>fimBTyr recombinase/fimA inversion regulator127.7212D4543668fimC>fimD>fimFminor component of type 1 fimbriae5.0213C4570642yjiS><mcrC41.4214D4575670yjiS>mcrC<mcrB12.1215D45764685-methylcytosine-specific restriction enzymemcrC<mcrB<symE26.3216D4588370conserved proteinyjiX<*yjiY>tsrmethyl-accepting chemotaxis protein I10.5217A4601330predicted inner membrane proteinyjjP<>yjjQDNA-binding transcriptional regulator22.9218B4609336prfC>>osmYperiplasmic protein3.7Genomic SELEX was performed for search of the binding sites of RNAP RpoS holoenzyme. By setting the cut-off level of 3.0, a total of 218 binding sites were identified (see Fig 1 for SELEX pattern), which are aligned along the map of E. coli K12 genome. A total of 125 sites are located within intergenic spacers: 50 within type-A spacers (shown under orange background); and 79 within type-B spacers (shown under green background). The constitutive promoters of RpoS were predicted based on the adjacent genes [note that only the genes next to the RpoS holoenzyme-binding sites are shown] and the gene orientation (shown by arrows in the column of transcription direction). A total of 73 RpoS holoenzyme-binding sites are located inside open reading frames as indicated by the gene symbols shown in RpoS column.*The genes listed in RegulonDB as the regulated targets of RpoS.10.1371/journal.pone.0179181.t002Table 2Distribution of the binding sites of each RNAP holoenzyme.SigmaTotal no. holoenzyme binding sitesWithin SpacersInside GenesConstitutive promotersType-AType-BType-CType-A spacerType-B spacerTotalRpoD132017731749777 (60%)177~354317494~671543 (40%)RpoS21850791673 (33%)50~10079129~179125 (67%)RpoH1334160626 (20%)41~8260101~142107 (80%)RpoF105727368 (65%)7~142734~4137 (35%)RpoE1262948742 (33%)29~584877~10684 (67%)RNAP holoenzyme was reconstituted from the sigma-free core enzyme and 4-fold molar excess of each sigma subunit. The binding site of each holoenzyme on the genome of E. coli K-12 W3110 was determined in vitro using the improved Genomic SELEX screening system. Details of the experimental procedures are described previously [26]. The number of constitutive promoters were estimated based on the location of holoenzyme-binding sites. The number of constitutive promoters recognized by RpoD holooenzyme were described in the previous report [27].Up to the present, two general approaches have been employed to define the RpoS regulon: the proteome analysis using two-dimensional gels of whole cell lysates [35]; and the transcriptome analysis using ChIP-chip or ChIP-Seq systems [37–39]. These studies altogether indicated that RpoS regulates, directly or indirectly, 10% (approximately 500 genes) of the E. coli genes, of which only about 140 genes were predicted to be under the direct control in vivo of RpoS [38]. The total number of RpoS promoters (or the transcription initiation sites) listed in the current RegulonDB database is as many as 164 [note that all these promoters were detected in vivo]. Of which 21 were identified by setting the cut-off level at 3.0 (Table 1, marked by asterisk), indicating that only these promoters represent the constitutive promoters and the majority of other known RpoS promoters represent the inducible promoters that are expressed only under the support of regulatory factors. Genomic SELEX analysis identified minimum 129 and maximum 179 RpoS constitutive promoters including 21 known RpoS-dependent promoters (Table 2). The highest peak (390-fold higher than the background of original library alone) was located at the 5’-proximal region of the purT gene (Fig 1), which encodes a bifunctional enzyme with both phosphoribosylglycinamide formyltransferase using formate (the third-step reaction of purine nucleotide synthesis) and acetate kinase for the synthesis of acetylphosphate (AcP). AcP might be utilized as the general phosphate donor for phosphorylation of most of the stress-response TCS (two-component system) response regulators under stressful conditions. A high-level peak (154-fold higher than the background of DNA library) was detected upstream of the artPIQM operon encoding L-arginine ABC transporter. This promoter was also identified in vivo to be RpoS dependent [36] (Table 1). High-level binding of the RpoS holoenzyme was also identified upstream of the cydAB operon encoding cytochrome oxidase for anaerobic respiration, and the hipBA operon encoding anti-toxin-toxin pair for control the persistence (Fig 1 and Table 1). RpoS-dependent constitutive promoter(s) also exists upstream of the nanCMS operon (N-acetylneuraminic acid transport and utilization) and/or the fimB gene (regulator for fimA encoding fimbrin, the major type-1 pili) (Fig 1 and Table 1). Noteworthy is that most of the RpoS-dependent promoters listed in the current databases might be those under the indirect control of RpoS [8,9,27]. Otherwise a set of RpoS-dependent promoters, designated as the inducible promoters, might be activated in the presence of additional supporting factors.Using the newly constructed collection of E. coli promoters expressing two-fluorescent reporters, one attached to the test promoter and another to the reference promoter, we performed a systematic quantitative search in vivo for E. coli promoters that are activated in the stationary phase [39]. The activity of RpoS-dependent promoters was measured at various growth phases under various growth conditions. The results indicated that the constitutive promoters exhibited low but steady-state activity while the inducible promoters generally showed high activity during the transition from exponential growth to stationary phase.The RpoS regulon is involved in not only cell survival in the stationary phase, but also in cross protection against various stresses, including nutrient starvation, osmotic stress, acid shock, cold shock, heat shock, and oxidative DNA damage [33,34]. Beyond entry into stationary phase, E. coli forms aggregates or biofilms that are morphologically and physiologically distinct from cells of planktonic growth. This requires coordinated production of an extracellular matrix of polysaccharide polymers and protein fibers that facilitate cell aggregation and adhesion to solid surface. The genes involved in biofilm formation and transformation into persister cells were included in the list of RpoS constitutive promoters [40,41].

The whole set of constitutive promoters for the stationary-phase sigma RpoS

In laboratory culture of E. coli, cell growth enters into the stationary phase mainly due to the limited availability of nutrients. Upon entry into the stationary phase, the pattern of genome expression is markedly altered by turning down the growth-related genes and instead up-regulation of the stress-response genes. In switching the transcription pattern, the stationary-phase specific minor sigma RrpoS is involved [33,34]. The rpoS gene is not essential for growth under non-stress conditions, but strains carrying mutations affecting rpoS activity are extremely sensitive to environmental stresses. As in the case of other sigma factors, RpoS interacts with RNAP core enzyme and modulates its promoter recognition specificity so as to recognize a specific but large set of genes.

As noted above, the set of genes identified in vivo include a number of genes under the indirect control of RpoS. On the other hand, some target promoters of RpoS are masked in vivo due to competitive interference by other DNA-binding proteins. In order to identify the constitutive promoters directly recognized by RpoS in the absence of other DNA-binding proteins, the Genomic SELEX screening in vitro was performed using the reconstituted RNAP RpoS holoenzyme. The sequences with binding affinity to the RpoS holoenzyme formed a number of peaks along the entire E. coli genome (Fig 1). Location of peaks was aligned along the map of E. coli K-12 genome (Table 1). By setting the cut-off level of 3.0 fold-higher intensity over the background of original library DNA, a total of 218 peaks were identified, of which 125 (67%) are located within intergenic spacers and 73 (33%) are inside of open reading frames (Table 2). Since the majority of hitherto identified promoters are located within spacers and generally upstream of open reading frames, detailed search for the constitutive promoters was focused on these 125 spacer peaks. These spacers can be classified into three types: 50 peaks are located within type-A spacer between bidirectional transcription units; 79 peaks are located within type-B spacers located upstream of one transcription unit but downstream of another transcription unit; and 16 peaks are located within type-C located spacers downstream of both transcription units. Based on the transcription direction of flanking genes, the total number of RpoS constitutive promoters was predicted to range between minimum 129 (50 type-A plus 79 type-B) and maximum 179 (50x2 type-A plus 79 type-B) (Table 2). Type-A spacers should contain two promoters for bidirectional transcription, at least one of which should be RpoS-dependent promoter. The RpoS holoenzyme-binding sites identified in a total of 50 type-A spacers should represent promoters for one or both of bidirectional transcription.

SELEX-chip search for RNAP RpoS holoenzyme-binding sequences on the E. coli K-12 genome.

The y-axis represents the relative number of RpoS holoenzyme-bound DNA fragments whereas x-axis represents the position on the E. coli K-12 genome, in base pair. The adjacent gene on E. coli K-12 genome of peak position was indicated for high intensity peaks. The peaks located within spacer regions are shown in green color, while peaks located within open reading frames are shown in orange color. The list of RpoS holoenzyme-binding sites is described in Table 1.

RpoS holoenzyme-binding sites on the E. coli K-12 genome.

Genomic SELEX was performed for search of the binding sites of RNAP RpoS holoenzyme. By setting the cut-off level of 3.0, a total of 218 binding sites were identified (see Fig 1 for SELEX pattern), which are aligned along the map of E. coli K12 genome. A total of 125 sites are located within intergenic spacers: 50 within type-A spacers (shown under orange background); and 79 within type-B spacers (shown under green background). The constitutive promoters of RpoS were predicted based on the adjacent genes [note that only the genes next to the RpoS holoenzyme-binding sites are shown] and the gene orientation (shown by arrows in the column of transcription direction). A total of 73 RpoS holoenzyme-binding sites are located inside open reading frames as indicated by the gene symbols shown in RpoS column.

*The genes listed in RegulonDB as the regulated targets of RpoS.

Distribution of the binding sites of each RNAP holoenzyme.

RNAP holoenzyme was reconstituted from the sigma-free core enzyme and 4-fold molar excess of each sigma subunit. The binding site of each holoenzyme on the genome of E. coli K-12 W3110 was determined in vitro using the improved Genomic SELEX screening system. Details of the experimental procedures are described previously [26]. The number of constitutive promoters were estimated based on the location of holoenzyme-binding sites. The number of constitutive promoters recognized by RpoD holooenzyme were described in the previous report [27].

Up to the present, two general approaches have been employed to define the RpoS regulon: the proteome analysis using two-dimensional gels of whole cell lysates [35]; and the transcriptome analysis using ChIP-chip or ChIP-Seq systems [37–39]. These studies altogether indicated that RpoS regulates, directly or indirectly, 10% (approximately 500 genes) of the E. coli genes, of which only about 140 genes were predicted to be under the direct control in vivo of RpoS [38]. The total number of RpoS promoters (or the transcription initiation sites) listed in the current RegulonDB database is as many as 164 [note that all these promoters were detected in vivo]. Of which 21 were identified by setting the cut-off level at 3.0 (Table 1, marked by asterisk), indicating that only these promoters represent the constitutive promoters and the majority of other known RpoS promoters represent the inducible promoters that are expressed only under the support of regulatory factors. Genomic SELEX analysis identified minimum 129 and maximum 179 RpoS constitutive promoters including 21 known RpoS-dependent promoters (Table 2). The highest peak (390-fold higher than the background of original library alone) was located at the 5’-proximal region of the purT gene (Fig 1), which encodes a bifunctional enzyme with both phosphoribosylglycinamide formyltransferase using formate (the third-step reaction of purine nucleotide synthesis) and acetate kinase for the synthesis of acetylphosphate (AcP). AcP might be utilized as the general phosphate donor for phosphorylation of most of the stress-response TCS (two-component system) response regulators under stressful conditions. A high-level peak (154-fold higher than the background of DNA library) was detected upstream of the artPIQM operon encoding L-arginine ABC transporter. This promoter was also identified in vivo to be RpoS dependent [36] (Table 1). High-level binding of the RpoS holoenzyme was also identified upstream of the cydAB operon encoding cytochrome oxidase for anaerobic respiration, and the hipBA operon encoding anti-toxin-toxin pair for control the persistence (Fig 1 and Table 1). RpoS-dependent constitutive promoter(s) also exists upstream of the nanCMS operon (N-acetylneuraminic acid transport and utilization) and/or the fimB gene (regulator for fimA encoding fimbrin, the major type-1 pili) (Fig 1 and Table 1). Noteworthy is that most of the RpoS-dependent promoters listed in the current databases might be those under the indirect control of RpoS [8,9,27]. Otherwise a set of RpoS-dependent promoters, designated as the inducible promoters, might be activated in the presence of additional supporting factors.

Using the newly constructed collection of E. coli promoters expressing two-fluorescent reporters, one attached to the test promoter and another to the reference promoter, we performed a systematic quantitative search in vivo for E. coli promoters that are activated in the stationary phase [39]. The activity of RpoS-dependent promoters was measured at various growth phases under various growth conditions. The results indicated that the constitutive promoters exhibited low but steady-state activity while the inducible promoters generally showed high activity during the transition from exponential growth to stationary phase.

The RpoS regulon is involved in not only cell survival in the stationary phase, but also in cross protection against various stresses, including nutrient starvation, osmotic stress, acid shock, cold shock, heat shock, and oxidative DNA damage [33,34]. Beyond entry into stationary phase, E. coli forms aggregates or biofilms that are morphologically and physiologically distinct from cells of planktonic growth. This requires coordinated production of an extracellular matrix of polysaccharide polymers and protein fibers that facilitate cell aggregation and adhesion to solid surface. The genes involved in biofilm formation and transformation into persister cells were included in the list of RpoS constitutive promoters [40,41].

The whole sets of constitutive promoters for heat-shock response sigma RpoHWhen E. coli cells are exposed to higher temperature, a set of heat-shock proteins (HSPs) is markedly and transiently induced. Heat shock-induced proteins (HSPs) play major roles in controlling the structure and function of various proteins, including protein folding, assembly, transport, repair and degradation during normal growth as well as under stress conditions [42,43]. The heat-shock response is a cellular protective system for maintenance of protein homeostasis. The set of HSPs include the GroEL (HSP60) and DnaK (HSP70) chaperones and the Lon and the Clp proteases. RpoH is specifically required for expression of the genes encoding a set of HSPs as identified by proteome [44,45] and also by transcriptome analyses [46]. Genome-wide transcription profiling of the regulatory targets of RpoH was identified under the moderate induction of a plasmid-borne rpoH gene under defined, steady state growth conditions [47]. A total of 126 genes were influenced in the absence or in the over-expression of RpoH, which are organized in 85 operons. The set of genes identified in vivo by changing the level of RpoH include a large number of indirect targets, which are affected in response to the changes in the level of direct target [8,9,27]. The total number of RpoH promoters (or the transcription initiation sites) listed in the current RegulonDB database is as many as 322, but the majority of RpoH targets are predicted by the computational analysis using the consensus sequence that was predicted based on a few experimentally identified RpoH promoters.We isolated RpoH protein for the first time and confirmed its recognition in vitro of the known HSP gene promoters [48]. Since then no serious examination in vitro has been performed to identify the RpoH function and it regulatory targets. To get insights into the regulatory role of RpoH sigma, we then performed in this study the Genomic SELEX screening using the reconstituted RNAP RpoH holoenzyme. By setting the cut-off level of 3.0 fold higher than the background of original library DNA, a total of 133 RpoH holoenzyme-binding peaks were identified (Fig 2 and Table 3), of which 107 (80%) are located within intergenic spacers and 26 (20%) are inside of open reading frames (Table 2). Since the majority of hitherto identified promoters are located within spacers, detailed search for the constitutive promoters was focused on the total of 107 peaks within spacers. The spacers containing RpoH holoenzyme-binding sites were also classified into three types (Tables 2 and 3 for the whole list): 41 peaks are located within type-A spacer; 60 peaks are located within type-B spacers; and 6 peaks are located within type-C spacers. Based on the transcription direction of flanking genes, the total number of RpoH constitutive promoters was predicted to range between minimum 101 (41 type-A plus 60 type-B) and maximum 142 (41x2 type-A plus 60 type-B) (Table 2).10.1371/journal.pone.0179181.g002Fig 2SELEX-chip search for RNAP RpoH holoenzyme-binding sequences on the E. coli K-12 genome.The y-axis represents the relative number of RpoH holoenzyme-bound DNA fragments whereas x-axis represents the position on the E.coli K-12 genome, in base pair. The adjacent gene on E. coli K-12 genome of peak position was indicated for high intensity peaks. The peaks located within spacer regions are shown in green color, while peaks located within open reading frames are shown in orange color. The list of RpoH holoenzyme-binding sites is described in Table 3.10.1371/journal.pone.0179181.t003Table 3RpoH holoenzyme-binding sites on the E. coli K-12 genome.NoTypeMapGene FunctionLeftDRpoHDRightGene FunctionIntensity1A12044predicted proteinyaaI<>*dnaKchaperone Hsp703.22A142736carbonic anhydrase*can<>yadGABC superfamily transporter3.93B155442outer membrane usher proteinhtrE<<ecpD3.04B164658hrpB>>mrcBglycosyl transferase and transpeptidase3.85B202068lpxD>>fabZhydroxymyristol acyl carrier dehydratase4.06B229134aspU>>dkgB2,5-diketo-D-gluconate reductase B3.07C262270thrW><ykfI3.68B292170CP4-6 prophage proteinyagK<<yagL4.39B331456betT>>yahADNA-binding transcriptional regulator3.510D343660yahK>yahL>yahMpredicted protein5.611B379186frmRAB operon regulatorfrmR<<yaiO3.012B406536yaiA>>aroMconserved protein3.613B477848conserved inner membrane proteinylaB<<ylaC3.314D543470conserved proteinylbA<allC<allD10.115B557960sfmA>>sfmCpilin chaperone3.516D559456sfmC>sfmD>sfmHfimbrial-like adhesin protein3.117B581644nohB>>appYDLP12 transcriptional activator3.818B592452phage N4 receptor IM proteinnfrB<<cusS3.219B629042ybdB>>cstAcarbon starvation protein6.020D655854anaerobic C4-dicarboxylate transportdcuC<pagP>cspEDNA-binding transcriptional repressor3.321B661936conserved protein*ybeD<<dacA20.122D732870ybfA>*rhsC>ybfBpredicted inner membrane protein3.223D747240predicted regulatorabrB<ybgO<ybgP3.524B784068zinc efflux systemzitB<<ybgS3.525A784656conserved proteinybgS<>aroGD-arabino-heptulosonate-7P synthase3.026B837732conserved proteinybiI<<ybiX3.127B913136anaerobic terminal reductaseshcp<<ybjE3.828A918368conserved proteinybjX<>macAmacrolide transporter6.329B959450aroA>>ycaLpeptidase with chaperone function3.230B985134aspartate aminotransferaseaspC<<ompF3.431A1050632CspA-family stress protein*cspH<>cspGDNA-binding transcriptional regulator3.632A1102554DNA-binding transcriptional activatorcsgD<>csgBcurlin nucleator protein4.033B1120230predicted protein*bssS<<dinI5.434C1195868icd><ymfD6.235D1198640e14 prophage inner membrane proteinymfE<lit<intE3.836D1200062e14 prophage integraseintE<xisE>ymfIe14 prophage; predicted protein3.237B1218154ycgG>>ymgFpredicted protein3.438C1219948ymgF><ymgD3.939A1233950sodium:proton antiporternhaB<>fadRDNA-binding transcriptional regulator3.140B1255834predicted adhesinycgV<<ychF3.241B1308330voltage-gated potassium channelkch<<yciI3.742B1349272enoyl-[acyl-carrier-protein] reductasefabI<<ycjD4.343A1389946predicted hydrolase*ycjY<>*ycjZDNA-binding transcriptional regulator4.444B1432738Rac prophage transcriptional regulatorynaE<<uspF4.245B1486246ydcF>>aldAaldehyde dehydrogenase A3.746B1565470predicted diguanylate cyclaseyddV<<yddW5.247B1568560glutamate:aminobutyric acid antiportergadC<<gadB2.348B1580550conserved proteinydeN<<ydeO3.349B1585730fimbrial-like adhesin proteinydeQ<<ydeR3.750B1613766yneJ>>*yneKpredicted protein3.051A1630638mannonate dehydrogenaseydfI<>ydfKQin prophage transcriptional regulator4.052B1639072Qin prophage S lysis proteinessQ<<cspB3.353A1639660Qin prophage cold shock proteincspB<>cspFQin prophage cold shock protein3.754B1646444dicA>>*ydfAQin prophage protein3.255B1669358ynfM>>asracid-shock periplasmic protein3.756B1751846predicted oxidoreductaseydhV<<ydhY3.157B1762570Fe-S cluster assembly proteinsufA<<rydB5.158A1801256threonyl-tRNA synthetasethrS<>yniDpredicted protein5.359A1801758threonyl-tRNA synthetasethrS<>yniDpredicted protein3.460B1990832PG phosphate synthasepgsA<<uvrC4.261D2055332amn>yeeN<asnW3.062A2060070DNA-binding transcriptionall regulatornac<>asnVAsn tRNA3.263B2096360LPS O-antigen length regulator*cld<<ugd3.264D2104956conserved proteinwbbI<*rfc<glf3.465A2261530fructose-specific PTS enzyme IIAfruB<>setBlactose/glucose efflux system3.366B2276452predicted proteinyejG<<bcr3.167A2342846hypothetical proteinypaB<>nrdAribonucleoside diphosphate reductase3.468D2355640glpC>yfaD>ypaApredicted protein3.469D2363146nudI>ais>arnBuridine aminotransferase3.870B2403570NADH:ubiquinone oxidoreductasenuoA<<lrhA5.171A2405036DNA-binding transcriptional regulatorlrhA<>yfbQpredicted aminotransferase3.272D2476546DNA-binding transcriptional regulatordsdC<dsdX>dsdAD-serine ammonia-lyase3.673D2489940predicted transporteryfdV<oxc<frc3.674D2490262predicted oxalyl-CoA decarboxylaseoxc<frc<yfdX3.675A2492430predicted proteinyfdX<>ypdIlipoprotein for colanic acid biosynthesis3.076A2493362predicted inner membrane proteinyfdY<>lpxPpalmitoleoyl-ACP acyltransferase3.677D2532356ptsH>ptsI>crrglucose-specific PTS enzyme IIA3.278A2597838dihydrodipicolinate synthasedapA<>gcvRDNA-binding transcriptional repressor3.179B2599140bcp>>hyfAhydrogenase 4, 4Fe-4S subunit3.180D2626032ppx>yfgF>yfgGpredicted protein3.681A2696642conserved proteinyfhB<>yfhHDNA-binding transcriptional regulator3.382A2714742pyruvate formate lyase subunityfiD<>unguracil-DNA-glycosylase4.883A2739338D-arabino-heptulosonate-7P synthasearoF<>yfiLpredicted protein3.584A2823870lytic murein transglycosylase BmltB<>srlAglucitol/sorbitol-specific PTS IIC3.185A2837534DNA-binding transcriptional repressorascG<>ascFcellobiose/arbutin/salicin PTS IIB-IIC3.586D2884946predicted proteinygcL<ygcB<cysH3.487A2898372deoxygluconate dehydrogenaseygcW<>yqcEpredicted transporter4.688A2932264L-fuculose-1-phosphate aldolasefucA<>fucPL-fucose transporter3.389B2967056nucleotide hydrolaserppH<<ygdT4.590A2976930diaminopimelate decarboxylaselysA<>lysRDNA-binding transcriptional regulator3.491B2985164yqeG>>yqeHprotein with bipartite regulator domain3.392D2985970yqeG>yqeH>yqeItranscriptional regulator3.293D2991152ygeG>ygeH>ygeIpredicted protein3.794C2991858ygeI><insD3.595B3067930mechanosensitive channelmscS<<fbaA3.096B3166268DNA-binding transcriptional regulatorygiT<<mqsR3.097B3237654alx>>sstTsodium:serine/threonine symporter6.998D3266066tdcR>yhaB>yhaCpredicted protein3.099A3276944DNA-binding transcriptional regulatoragaR<>kbaZtagatose 6-phosphate aldolase 13.6100B3319952predicted hydrolase, inner membraneyhbX<<leuU3.2101A332583223S rRNA methyltransferaserrmJ<>yhbYpredicted RNA-binding protein14.1102D3360232gltF>*yhcA>yhcDpredicted outer membrane protein3.2103B3375554stringent starvation protein AsspA<<rpsI3.7104B3387142p-hydroxybenzoic acid efflux system*aaeA<<aaeX3.1105A3559934thiosulfate:cyanide sulfurtransferaseglpE<>glpDsn-glycerol-3-phosphate dehydrogenase3.5106D3629568predicted HlyD family secretion proteinyhiI<yhiJ>yhiMconserved inner membrane protein3.6107C3634072yhiM><yhiN3.8108B3655654hdeD>>gadEDNA-binding transcriptional activator4.1109C3661646mdtF><gadW3.7110B3681552C4-dicarboxylic acid citrate transporterdctA<<yhjK3.0111A3717248conserved proteinyiaF<>yiaGtranscriptional regulator3.0112B3717858yiaG>>cspAmajor cold shock protein3.0113B3720058insK>>sokAncRNA3.0114A3826772glutamate transportergltS<>yicEpredicted transporter4.4115B3834954selC>>setCpredicted sugar efflux system3.4116A3865668heat shock chaperone*ibpA<>yidQconserved outer membrane protein4.6117A3939432predicted transcriptional regulatoryieP<>*rrsC16S ribosomal RNA of rrnC operon3.6118A4002730conserved proteinyigI<>*pldAouter membrane phospholipase A3.7119B4068432predicted aldose-1-epimeraseyihR<<yihS7.5120B4360430DNA-binding transcriptional activatorcadC<<pheU4.0121B4364770C4-dicarboxylate antiporterdcuA<<aspA4.6122A4366568aspartate ammonia-lyaseaspA<>*fxsAinner membrane protein19.2123D4371850yjeI>*yjeJ<yjeK4.9124D4486746yjgQ>yjgR<idnR4.1125B4494638leuX>>insCKpLE2 phage IS2 element repressor3.1126B4518430KpLE2 transcriptional regulatoryjhU<<yjhF3.1127B4530030KpLE2 phage endoglucanasesgcX<<yjhP3.3128A4538050N-acetylnuraminic acid channel proteinnanC<>fimBtyrosine recombinase, fimA regulator3.5129A4538964N-acetylnuraminic acid channel proteinnanC<>fimBtyrosine recombinase, fimA regulator3.4130B4540968fimE>>fimAmajor type 1 subunit fimbrin (pilin)3.4131A4549330fructuronate transportergntP<>uxuAmannonate hydrolase5.0132D45764685-methylcytosine restriction enzymemcrC<mcrB<symE3.5133A4633370DNA-binding transcriptional activatorrob<>*creAconserved protein3.3Genomic SELEX was performed for search of the binding sites of RNAP RpoH holoenzyme. By setting the cut-off level of 3.0, a total of 133 binding sites were identified (see Fig 2 for SELEX pattern), which are aligned along the map of E. coli K12 genome. A total of 107 sites are located within intergenic spacers: 41 wihin type-A spacers (shown under orange background); and 60 within type-B spacers (shown under green background). The constitutive promoters of RpoH were predicted based on the adjacent genes [note that only the genes next to the RpoH holoenzyme-binding sites are shown] and the gene orientation (shown by arrows in the column of transcription direction). A total of 26 RpoH holoenzyme-binding sites are located inside open reading frames as indicated by the gene symbols shown in RpoH column.* The genes listed in RegulonDB as the regulated targets of RpoH.Among a total of 322 RpoH promoters (or the transcription initiation sites) listed in RegulonDB database, 20 were identified setting the cut-off level at 3.0 (Table 3, marked by asterisk). The majority of RpoH promoters in the database were suggested to belong to the inducible promoters that are expressed only under the support of other positive regulatory factors. Otherwise these RpoH promoters might represent the inaccurate prediction as note above. Genomic SELEX analysis identified minimum 100 and maximum 140 RpoH constitutive promoters including 18 known RpoH dependent promoters (Table 2). The highest peak was 20-fold intensity that was detected on promoter region of the ybeD gene, which encodes a conserved protein of unknown function under regulation of RpoH (Fig 2) [49], followed by high-level peaks at the aspA-fxsA and the rlmJ-yhbY intergenic regions. The fxsA gene encodes an inner membrane protein, which is involved in sensitivity control to bacteriophage T7 [49]. The rlmE gene encodes 23S rRNA 2’-O-ribose U2552 metyltransferase, and has been proposed to carry RpoH-dependent promoter [50]. The regulatory target of RpoH sigma identified by Genomic SELEX expands to a set of genes related to varieties of stress-response genes beyond the HSP genes. In fact, the genes for response to environmental insults such as ethanol, alkaline pH, and hyperosmotic shock and the genes for proteolysis and cell division have been indicated under the control of RpoH. The set of RpoH-regulon genes thus identified in vivo, however, vary depending on the culture conditions.

The whole sets of constitutive promoters for heat-shock response sigma RpoH

When E. coli cells are exposed to higher temperature, a set of heat-shock proteins (HSPs) is markedly and transiently induced. Heat shock-induced proteins (HSPs) play major roles in controlling the structure and function of various proteins, including protein folding, assembly, transport, repair and degradation during normal growth as well as under stress conditions [42,43]. The heat-shock response is a cellular protective system for maintenance of protein homeostasis. The set of HSPs include the GroEL (HSP60) and DnaK (HSP70) chaperones and the Lon and the Clp proteases. RpoH is specifically required for expression of the genes encoding a set of HSPs as identified by proteome [44,45] and also by transcriptome analyses [46]. Genome-wide transcription profiling of the regulatory targets of RpoH was identified under the moderate induction of a plasmid-borne rpoH gene under defined, steady state growth conditions [47]. A total of 126 genes were influenced in the absence or in the over-expression of RpoH, which are organized in 85 operons. The set of genes identified in vivo by changing the level of RpoH include a large number of indirect targets, which are affected in response to the changes in the level of direct target [8,9,27]. The total number of RpoH promoters (or the transcription initiation sites) listed in the current RegulonDB database is as many as 322, but the majority of RpoH targets are predicted by the computational analysis using the consensus sequence that was predicted based on a few experimentally identified RpoH promoters.

We isolated RpoH protein for the first time and confirmed its recognition in vitro of the known HSP gene promoters [48]. Since then no serious examination in vitro has been performed to identify the RpoH function and it regulatory targets. To get insights into the regulatory role of RpoH sigma, we then performed in this study the Genomic SELEX screening using the reconstituted RNAP RpoH holoenzyme. By setting the cut-off level of 3.0 fold higher than the background of original library DNA, a total of 133 RpoH holoenzyme-binding peaks were identified (Fig 2 and Table 3), of which 107 (80%) are located within intergenic spacers and 26 (20%) are inside of open reading frames (Table 2). Since the majority of hitherto identified promoters are located within spacers, detailed search for the constitutive promoters was focused on the total of 107 peaks within spacers. The spacers containing RpoH holoenzyme-binding sites were also classified into three types (Tables 2 and 3 for the whole list): 41 peaks are located within type-A spacer; 60 peaks are located within type-B spacers; and 6 peaks are located within type-C spacers. Based on the transcription direction of flanking genes, the total number of RpoH constitutive promoters was predicted to range between minimum 101 (41 type-A plus 60 type-B) and maximum 142 (41x2 type-A plus 60 type-B) (Table 2).

SELEX-chip search for RNAP RpoH holoenzyme-binding sequences on the E. coli K-12 genome.

The y-axis represents the relative number of RpoH holoenzyme-bound DNA fragments whereas x-axis represents the position on the E.coli K-12 genome, in base pair. The adjacent gene on E. coli K-12 genome of peak position was indicated for high intensity peaks. The peaks located within spacer regions are shown in green color, while peaks located within open reading frames are shown in orange color. The list of RpoH holoenzyme-binding sites is described in Table 3.

RpoH holoenzyme-binding sites on the E. coli K-12 genome.

Genomic SELEX was performed for search of the binding sites of RNAP RpoH holoenzyme. By setting the cut-off level of 3.0, a total of 133 binding sites were identified (see Fig 2 for SELEX pattern), which are aligned along the map of E. coli K12 genome. A total of 107 sites are located within intergenic spacers: 41 wihin type-A spacers (shown under orange background); and 60 within type-B spacers (shown under green background). The constitutive promoters of RpoH were predicted based on the adjacent genes [note that only the genes next to the RpoH holoenzyme-binding sites are shown] and the gene orientation (shown by arrows in the column of transcription direction). A total of 26 RpoH holoenzyme-binding sites are located inside open reading frames as indicated by the gene symbols shown in RpoH column.

* The genes listed in RegulonDB as the regulated targets of RpoH.

Among a total of 322 RpoH promoters (or the transcription initiation sites) listed in RegulonDB database, 20 were identified setting the cut-off level at 3.0 (Table 3, marked by asterisk). The majority of RpoH promoters in the database were suggested to belong to the inducible promoters that are expressed only under the support of other positive regulatory factors. Otherwise these RpoH promoters might represent the inaccurate prediction as note above. Genomic SELEX analysis identified minimum 100 and maximum 140 RpoH constitutive promoters including 18 known RpoH dependent promoters (Table 2). The highest peak was 20-fold intensity that was detected on promoter region of the ybeD gene, which encodes a conserved protein of unknown function under regulation of RpoH (Fig 2) [49], followed by high-level peaks at the aspA-fxsA and the rlmJ-yhbY intergenic regions. The fxsA gene encodes an inner membrane protein, which is involved in sensitivity control to bacteriophage T7 [49]. The rlmE gene encodes 23S rRNA 2’-O-ribose U2552 metyltransferase, and has been proposed to carry RpoH-dependent promoter [50]. The regulatory target of RpoH sigma identified by Genomic SELEX expands to a set of genes related to varieties of stress-response genes beyond the HSP genes. In fact, the genes for response to environmental insults such as ethanol, alkaline pH, and hyperosmotic shock and the genes for proteolysis and cell division have been indicated under the control of RpoH. The set of RpoH-regulon genes thus identified in vivo, however, vary depending on the culture conditions.

The whole sets of constitutive promoters for the flagella-chemotaxis sigma RpoFThe bacterial flagellum is a complex organelle consisting of three distinctive structural parts, the basal body, the hook and the filament [51]. The synthesis, assembly and function of the flagellar and chemotaxis system require the expression of more than 50 genes, which are divided into three temporally regulated transcriptional classes based on the hierarchy of expression order: class-I (early), class-II (middle), and class-III (late) [52,53]. The class-1 (early) consists of a single operon including two genes, flhD and flhC, each encoding transcription factor FlhD and FlhC, respectively, which together form a complex, FlhD2-FlhC2 or FlhD4-FlhC2, that activates transcription of a set of class-2 (middle) genes, including both the rpoF sigma gene (renamed fliA) and the flgM gene encoding the anti-RpoF factor [51,52]. RpoF is the sigma factor for flagellar chemotaxis, which recognizes the promoters of motility and flagellar synthesis genes. The regulatory target of RpoF in Salmonella was identified to include a set of genes that were classified into the class-3 operons of the flagella regulon [54,55]. More than 30 genes have been proposed to carry promoters that are under the control of RpoF sigma, including a set of the structural genes for flagella formation, and the chemotaxis genes encoding sensor of environmental signals affecting the motility control [54,56]. With use the combination of ChIP-chip, ChIP-seq and RNA-seq systems, a more comprehensive screening was recently performed for identification of the regulatory targets of RpoF sigma in E. coli [57]. A total of 52 RpoF-binding sites were identified in vivo on the genome of exponentially growing E. coli K-12 MG1655 cells in a rich LB medium, with a considerable level of over-lapping with the hitherto identified target genes of the RpoF regulon. The total number of RpoF promoters (or the transcription initiation sites) listed in the current RegulonDB database is as many as 144, which have been identified in vivo using ChIP-chip and ChIP-RNA Seq analyses. Most of the targets predicted by the in vivo data, however, represent those indirectly affected upon knock-out of the rpoF gene or over-production of RpoF.We then performed the Genomic SELEX screening in vitro for search of the direct target promoters, genes and operons under the control of RpoF using the reconstituted RNAP RpoF holoenzyme. By setting the cut-off level of 4.0 fold higher than the background of original library DNA, a total of 105 RpoF holoenzyme-binding peaks were identified (Fig 3 and Table 4), of which 37 (35%) are located within intergenic spacers and 68 (65%) are inside of genes (Table 2). One unique feature of RpoF holoenzyme is its high-level (65%) binding to inside of open reading frames of a number of genes. A high-level (60%) of RNAP binding was also identified for RpoD holoenzyme [27]. The identification of the promoter-like sequences inside these genes awaits further analysis. The spacers containing RpoF holoenzyme-binding sites were also classified into three types (Tables 2 and 4 for the whole list): 7 peaks are located within type-A spacer; 27 peaks are located within type-B spacers; and 3 peaks are located within type-C spacers. Based on the transcription direction of flanking genes, the total number of RpoF constitutive promoters was predicted to range between minimum 34 (7 type-A plus 27 type-B) and maximum 41 (7x2 type-A plus 27 type-B) (Table 2). The total number of RpoF promoters (or the transcription initiation sites) listed in the current RegulonDB database is as many as 144. Of which 14 were identified setting cut-off level at 4.0 (Table 4, marked by asterisk), indicating that these promoters are constitutive promoters and the majority of known promoters represent the inducible promoters that are expressed only under the support of positive regulatory factors.10.1371/journal.pone.0179181.g003Fig 3SELEX-chip search for RNAP RpoF holoenzyme-binding sequences on the E. coli K-12 genome.The y-axis represents the relative number of RpoF holoenzyme-bound DNA fragments whereas x-axis represents the position on the E. coli K-12 genome, in base pair. The adjacent gene on E. coli K-12 genome of peak position was indicated for high intensity peaks. The peaks located within spacer regions are shown in green color, while peaks located within open reading frames are shown in orange color. The list of RpoF holoenzyme-binding sites is described in Table 4.10.1371/journal.pone.0179181.t004Table 4RpoF holoenzyme-binding sites on the E. coli K-12 genome.NoTypeMapGene FunctionLeftDRpoFDRightGene FunctionIntensity1D25672ileS>lspA>fkpBpeptidyl-prolyl cis-trans isomerase8.12B68342L-arabinose isomerasearaA<<araB4.33D109556secM>secA>mutTNTP pyrophosphohydrolase5.54A131366predicted proteinyacH<>acnBaconitate hydratase4.65B164658hrpB>>mrcBglycosyl transferase and transpeptidase6.46D198960rseP>bamA>skpperiplasmic chaperone4.77B202068lpxD>>fabZhydroxymyristol acyl carrier dehydratase7.18D298660conserved proteinyagQ<yagR<yagS4.49D343660yahK>yahL>yahMpredicted protein27.110D353230prpD>prpE>codBcytosine transporter4.711B463170ppiD>>ybaVconserved protein5.412D472938glnK>amtB<tesB4.013D477460ybaA>ylaB<ylaC7.814D482072predicted proteintomB<acrB<acrA4.515D543470conserved proteinylbA<allC<allD12.316D633930predicted oxidoreductaseybdH<ybdL<ybdM5.417D652632citrate lyase synthetasecitC<dpiB>dpiACitAB TCS response regulator5.018D747240predicted regulatorabrB<ybgO<ybgP13.119D761630sucA>sucB>sucCsuccinyl-CoA synthetase4.720B794336ybhT>>*modAmolybdate transporter subunit9.221B822944cardiolipin synthase 2ybhO<<ybhP4.922B837732conserved proteinybiI<<ybiX5.723D844530rlmF>ybiO<glnQ9.024B874568yliE>>yliFpredicted diguanylate cyclase4.325D878444bssR>yliI<yliJ5.326D893136ybjN>potF>potGABC superfamily putrescine transporter4.127B956734ycaP>>serC3-phosphoserine aminotransferase6.228D1039338appC>appB>yccBhypothetical protein5.029D1048250conserved proteingfcC<gfcB<gfcA4.030B1062060modulator of CbpA co-chaperonecbpM<<cbpA5.231B1129438anti-sigma factor for FliA (sigma 28)*flgM<<flgA15.032B1133868flgF>>flgGflagellar component6.733B1137530flgJ>>*flgKflagellar hook-filament junction protein 127.234D1158658ptsG>fhuE>hinTpurine nucleoside phosphoramidase6.535B1193064tRNA methyltransferasemnmA<<nudJ5.336D1236068fadR>ycgB>dadAD-amino acid dehydrogenase6.937A1243852protein involved in flagellar function*ycgR<>ymgEpredicted inner membrane protein4.638B1349272enoyl-[acyl-carrier-protein] reductasefabI<<ycjD5.639D1356550predicted proteinymjA<puuP<puuA5.040D1392064ycjZ>mppA<ynaI6.541A1434934outer membrane pore protein NompN<>micCncRNA5.942B1488236aldA>>cybBcytochrome b56113.943D1504230predicted benzoate transporterydcO<ydcN>ydcPpredicted peptidase16.144B1644248ydfV>>*flxAQin prophage; predicted protein23.245D1661452ynfF>ynfG>ynfHoxidoreductase, membrane subunit5.546D1734472sodB>ydhP<ynhF6.147D1794968integration host factor (IHF)ihfA<pheT<pheS12.348A1801758threonyl-tRNA synthetasethrS<>yniDpredicted protein5.349B1815172conserved protein*chbG<<chbF4.250D1822736cho>*ves<spy10.851D1861032methionine sulfoxide reductase BmsrB<gapA>yeaDconserved protein4.252B1887966acyl-CoA synthetasefadD<<yeaY9.953A1906860predicted proteinmgrB<>yobHpredicted protein8.354D1914836methionine-(R)-sulfoxide reductaseyebR<yebS>yebTconserved protein14.455D1928332conserved proteinyebE<yebF<yebG4.356D1938762myristoyl-ACP-dependent acyltransferaselpxM<yebA<znuA11.957D1971862purine-binding chemotaxis proteincheW<cheA<motB4.358D1988066predicted protein*yecH<tyrP<yecA7.959B1999848RNA polymerase sigma 28*fliA<<fliC46.260D2233032yeiS>yeiT>yeiADihydropyrimidine dehydrogenase4.861D2262454fructose-specific PTS enzymes IIAfruB<setB<yeiW6.262D2379830N-acyltransferaseelaA<rbn>elaDpredicted enzyme12.563B2403570NADH:ubiquinone oxidoreductasenuoA<<lrhA3.564D2412454conserved inner membrane proteinyfbV<ackA>ptaphosphate acetyltransferase6.265D2456468phosphohistidine phosphatasesixA<fadJ<fadI4.566D2609062hyfH>hyfI>hyfJprocessing element hydrogenase 47.167D2684772serine hydroxymethyltransferaseglyA<hmp<glnB5.868D274556630S ribosomal subunit protein S16rpsP<ffh>ypjDpredicted inner membrane protein5.969D2868142L-isoaspartate carboxymethyltransfeerasepcm<surE<truD5.870D2916840barA>gudD<gudX5.471D2952958exonuclease V (RecBCD complex),recD<recB<ptrA8.072D2965066diacylglyceryl transferaselgt<ptsP<rppH6.773D3087058metK>galP>yggIconserved protein6.474D3124360glycolate oxidase iron-sulfur subunitglcF<glcE<glcD4.075C3146946yghZ><yqhA5.276B3197666deadenylyltransferase/adenylyltransferaseglnE<<ygiF8.877B3237654alx>>sstTsodium:serine/threonine symporter9.978D3338440ABC superfamily toluene transporteryrbF<yrbG>kdsDD-arabinose 5-phosphate isomerase6.279D3344772hpf>ptsN>yhbJprotein with NTP hydrolase domain5.980C3388568aaeR><tldD4.781D3452166type II secretion divergongspB<gspA>gspCgeneral secretory pathway component11.882D3454168general secretory pathway componentgspA<gspC>gspDgeneral secretory pathway component14.783D3478072predicted proteinyheV<kefB<kefG6.184D3515848predicted proteindamX<aroB<aroK4.585C3528652hslO><yhgE4.086D3534454TCS sensory histidine kinaseenvZ<ompR>*greBtranscription elongation factor4.387D3585866yhhA>ugpQ<ugpC18.188D3633746predicted proteinyhiJ<*yhiM<yhiN4.889D3689064endo-1,4-D-glucanasebcsZ<bcsB<bcsA4.690B3779166lldD>>yibKpredicted rRNA methylase9.291D3789732threonine 3-dehydrogenasetdh<kbl<htrL4.992A3826672glutamate transportergltS<>yicEpredicted transporter5.493D3829430yicE>yicH<yicI4.394B3906432phosphate transporterpstB<<pstA4.195D3945232trpT>*hdfR>yifEconserved protein4.796B3951430ilvE>>ilvDdihydroxyacid dehydratase8.697D4003446conserved proteinyigI<pldA>recQATP-dependent DNA helicase4.598D4032158yigZ>trkH>hemGprotoporphyrin oxidase, flavoprotein6.099B4068432predicted aldose-1-epimeraseyihR<<yihS9.6100D4178758rplJ>rplL>rpoBRNA polymerase, beta subunit6.7101D4286472acetyl-CoA synthetaseacs<nrfA>nrfBnitrite reductase5.8102D4327160conserved proteinyjdM<*yjdA>yjcZconserved protein12.4103D4338862biodegradative arginine decarboxylaseadiA<melR>melAalpha-galactosidase, NAD(P)-binding4.5104D4535430conserved proteinyjhX<yjhS<nanM4.7105A4589632predicted inner membrane proteinyjiY<>*tsrmethyl-accepting chemotaxis protein I5.3Genomic SELEX was performed for search of the binding sites of RNAP RpoF holoenzyme. By setting the cut-off level of 3.0, a total of 105 binding sites were identified (see Fig 3 for SELEX pattern), which are aligned along the map of E. coli K12 genome. A total of 37 sites are located within intergenic spacers: 7 wihin type-A spacers (shown under orange background); and 27 within type-B spacers (shown under green background). The constitutive promoters of RpoF were predicted based on the adjacent genes [note that only the genes next to the RpoF holoenzyme-binding sites are shown] and the gene orientation (shown by arrows in the column of transcription direction). A total of as many as 68 RpoF holoenzyme-binding sites are located inside open reading frames as indicated by the gene symbols shown in RpoF column.*The genes listed in RegulonDB as the regulated targets of RpoF.The highest peak was 46-fold intensity detected on promoter region of rpoF itself (Fig 3), indicating the autoregulation as already suggested [58]. A high-level peak was also identified upstream of the flgK gene, which encodes flagellar hook-filament junction protein that connects the filament to the hook, and its transcription has been shown in vitro under the direct control of RpoF [59]. The flgM gene encodes the anti-sigma factor for RpoF [55]. FlgM forms a complex with RpoF, thereby inactivating its sigma function but protects its degradation by the Lon protease for preservation [60].

The whole sets of constitutive promoters for the flagella-chemotaxis sigma RpoF

The bacterial flagellum is a complex organelle consisting of three distinctive structural parts, the basal body, the hook and the filament [51]. The synthesis, assembly and function of the flagellar and chemotaxis system require the expression of more than 50 genes, which are divided into three temporally regulated transcriptional classes based on the hierarchy of expression order: class-I (early), class-II (middle), and class-III (late) [52,53]. The class-1 (early) consists of a single operon including two genes, flhD and flhC, each encoding transcription factor FlhD and FlhC, respectively, which together form a complex, FlhD2-FlhC2 or FlhD4-FlhC2, that activates transcription of a set of class-2 (middle) genes, including both the rpoF sigma gene (renamed fliA) and the flgM gene encoding the anti-RpoF factor [51,52]. RpoF is the sigma factor for flagellar chemotaxis, which recognizes the promoters of motility and flagellar synthesis genes. The regulatory target of RpoF in Salmonella was identified to include a set of genes that were classified into the class-3 operons of the flagella regulon [54,55]. More than 30 genes have been proposed to carry promoters that are under the control of RpoF sigma, including a set of the structural genes for flagella formation, and the chemotaxis genes encoding sensor of environmental signals affecting the motility control [54,56]. With use the combination of ChIP-chip, ChIP-seq and RNA-seq systems, a more comprehensive screening was recently performed for identification of the regulatory targets of RpoF sigma in E. coli [57]. A total of 52 RpoF-binding sites were identified in vivo on the genome of exponentially growing E. coli K-12 MG1655 cells in a rich LB medium, with a considerable level of over-lapping with the hitherto identified target genes of the RpoF regulon. The total number of RpoF promoters (or the transcription initiation sites) listed in the current RegulonDB database is as many as 144, which have been identified in vivo using ChIP-chip and ChIP-RNA Seq analyses. Most of the targets predicted by the in vivo data, however, represent those indirectly affected upon knock-out of the rpoF gene or over-production of RpoF.

We then performed the Genomic SELEX screening in vitro for search of the direct target promoters, genes and operons under the control of RpoF using the reconstituted RNAP RpoF holoenzyme. By setting the cut-off level of 4.0 fold higher than the background of original library DNA, a total of 105 RpoF holoenzyme-binding peaks were identified (Fig 3 and Table 4), of which 37 (35%) are located within intergenic spacers and 68 (65%) are inside of genes (Table 2). One unique feature of RpoF holoenzyme is its high-level (65%) binding to inside of open reading frames of a number of genes. A high-level (60%) of RNAP binding was also identified for RpoD holoenzyme [27]. The identification of the promoter-like sequences inside these genes awaits further analysis. The spacers containing RpoF holoenzyme-binding sites were also classified into three types (Tables 2 and 4 for the whole list): 7 peaks are located within type-A spacer; 27 peaks are located within type-B spacers; and 3 peaks are located within type-C spacers. Based on the transcription direction of flanking genes, the total number of RpoF constitutive promoters was predicted to range between minimum 34 (7 type-A plus 27 type-B) and maximum 41 (7x2 type-A plus 27 type-B) (Table 2). The total number of RpoF promoters (or the transcription initiation sites) listed in the current RegulonDB database is as many as 144. Of which 14 were identified setting cut-off level at 4.0 (Table 4, marked by asterisk), indicating that these promoters are constitutive promoters and the majority of known promoters represent the inducible promoters that are expressed only under the support of positive regulatory factors.

SELEX-chip search for RNAP RpoF holoenzyme-binding sequences on the E. coli K-12 genome.

The y-axis represents the relative number of RpoF holoenzyme-bound DNA fragments whereas x-axis represents the position on the E. coli K-12 genome, in base pair. The adjacent gene on E. coli K-12 genome of peak position was indicated for high intensity peaks. The peaks located within spacer regions are shown in green color, while peaks located within open reading frames are shown in orange color. The list of RpoF holoenzyme-binding sites is described in Table 4.

RpoF holoenzyme-binding sites on the E. coli K-12 genome.

Genomic SELEX was performed for search of the binding sites of RNAP RpoF holoenzyme. By setting the cut-off level of 3.0, a total of 105 binding sites were identified (see Fig 3 for SELEX pattern), which are aligned along the map of E. coli K12 genome. A total of 37 sites are located within intergenic spacers: 7 wihin type-A spacers (shown under orange background); and 27 within type-B spacers (shown under green background). The constitutive promoters of RpoF were predicted based on the adjacent genes [note that only the genes next to the RpoF holoenzyme-binding sites are shown] and the gene orientation (shown by arrows in the column of transcription direction). A total of as many as 68 RpoF holoenzyme-binding sites are located inside open reading frames as indicated by the gene symbols shown in RpoF column.

*The genes listed in RegulonDB as the regulated targets of RpoF.

The highest peak was 46-fold intensity detected on promoter region of rpoF itself (Fig 3), indicating the autoregulation as already suggested [58]. A high-level peak was also identified upstream of the flgK gene, which encodes flagellar hook-filament junction protein that connects the filament to the hook, and its transcription has been shown in vitro under the direct control of RpoF [59]. The flgM gene encodes the anti-sigma factor for RpoF [55]. FlgM forms a complex with RpoF, thereby inactivating its sigma function but protects its degradation by the Lon protease for preservation [60].

The whole set of constitutive promoters for extra-cytoplasmic stress response sigma RpoEThe bacterial cell envelope is a dynamic compartment, changing its structure and function in response to environmental conditions. Accordingly, the integrity of envelope is maintained through frequent modulation of its composition and components. The minor sigma factor RpoE plays a central role in this process, by controlling the selective expression of envelope components [61]. The regulatory targets have been estimated after proteome and transcriptome analyses in vivo [62–64]. The activity of RpoE is negatively regulated by a membrane-bound anti-sigma factor RseA, which sequesters RpoE under unstressed conditions. Within membrane, RseA is associated at its C-terminal domain with a periplasmic protein RseB, which senses misfolded proteins for release and activation of RpoE from RpoE-RseA complexes [62,65]. The total number of RpoE promoters (or the transcription initiation sites) listed in the current RegulonDB database is as many as 518, of which most are identified by computational analyses based on the consensus sequence of RpoE promoters without experimental analysis. After SELEX screening as noted below, most of these RpoE promoters must be inaccurate estimation due to the error in the consensus sequence.The Genomic SELEX screening system was employed as a short-cut approach for identification of the RpoE regulon. Previously we purified RpoE and examined its promoter selectivity using an in vitro transcription system [66]. Using this purified RpoE, we performed SELEX screening. By setting the cut-off level of 4.0 fold against original library DNA, a total of 126 RpoE holoenzyme-binding peaks were identified (Fig 4 and Table 5), of which 84 (67%) are located within intergenic spacers and 42 (33%) are inside of open reading frames (Tables 2 and 5 for the whole set). Since the majority of hitherto identified promoters are located within spacers, detailed search for the constitutive promoters was focused on the total of 84 peaks within spacers. The spacers containing RpoE holoenzyme-binding sites were also classified into three types (Table 2): 29 peaks are located within type-A spacer; 48 peaks are located within type-B spacers; and 7 peaks are located within type-C spacers. Based on the transcription direction of flanking genes, the total number of RpoE constitutive promoters was predicted to range between minimum 77 (29 type-A plus 48 type-B) and maximum 106 (29x2 type-A plus 48 type-B) (Table 2). Within the set of constitutive promoters identified by setting the cut-off level at 4.0, a total of 19 known RpoE promoters were identified (Table 5, marked by asterisk). The majority of known promoters represent the inducible promoters that are expressed only under the support of positive regulatory factors.10.1371/journal.pone.0179181.g004Fig 4SELEX-chip search for RNAP RpoE holoenzyme-binding sequences on the E. coli K-12 genome.The y-axis represents the relative number of RpoE holoenzyme-bound DNA fragments whereas x-axis represents the position on the E.coli genome, in base pair. The adjacent gene on E. coli K-12 genome of peak position was indicated for high intensity peaks. The peaks located within spacer regions are shown in green color, while peaks located within open reading frames are shown in orange color. The list of RpoE holoenzyme-binding sites is described in Table 5.10.1371/journal.pone.0179181.t005Table 5RpoE holoenzyme-binding sites on the E. coli K-12 genome.No.TypeMapLeft Gene FunctionLeftDRpoEDRightRight Gene FunctionIntensity1D154236fimbrial-like adhesin proteinyadM<htrE<ecpD5.02B164658hrpB>>mrcBglycosyl transferase and transpeptidase4.33B202068lpxD>>fabZhydroxymyristol acyl carrier dehydratase30.94C262270thrW><ykfI4.65D270562CP4-6 prophage transcriptional regulatorperR<insI<insH4.66B292358CP4-6 prophage; conserved proteinyagK<<yagL4.77D302738xanthine dehydrogenase 2Fe-2S subunityagT<yagU<ykgJ5.38A319448oxidoreductase with FAD-binding domain*ykgC<>ykgDDNA-binding transcriptional regulator4.09C323832ykgG><ykgH5.610B331456betT>>yahADNA-binding transcriptional regulator4.911d343660yahK>yahL>*yahMpredicted protein17.812A345542neutral amino-acid efflux systemyahN<>yahOpredicted protein4.013B477848conserved inner membrane proteinylaB<<ylaC4.514D543560conserved protein*ylbA<allC<allD9.615B581644nohB>>appYDLP12 DNA-binding transcriptional activator11.516B629042ybdB>>cstAcarbon starvation protein4.317B728638ybfA>>rhsCrhsC element core protein RshC5.618D732870ybfA>rhsC>ybfBpredicted inner membrane protein4.119D747240predicted regulatorabrB<*ybgO<ybgP9.320A753948citrate synthasegltA<>sdhCsuccinate dehydrogenase4.721B773834ybgE>>ybgCpredicted acyl-CoA thioesterase4.222B837732conserved proteinybiI<<ybiX4.123D838958conserved proteinybiX<fiu<mcbA4.324D844530rlmF>ybiO<glnQ6.025A862768pyruvate formate lyase activating enzymeybiY<>fsaAfructose-6-phosphate aldolase 14.426B887366ncRNA*rybB<<ybjL55.527B986332outer membrane porin 1a (Ia;b;F)ompF<<asnS5.028D992744pepN>ssuB<ssuC5.329D1045134exopolysaccharide export proteingfcE<gfcD<gfcC5.130C1049830insB><cspH6.331A1050632CspA-family stress protein*cspH<>cspGDNA-binding transcriptional regulator4.732D1088072predicted glycosyl transferasepgaC<pgaB<pgaA4.833B1166772ndh>>ycfJpredicted protein4.034C1195868icd><ymfD6.435B1218154ycgG>>*ymgFpredicted protein5.436D1259856ychH>ychM<prs5.337D1342036lipoproteinosmB<*yciT<yciZ4.338B1349272enoyl-[acyl-carrier-protein] reductasetfabI<<ycjD5.639B1384744ycjF>>tyrRDNA-binding transcriptional regulator5.040A1389946predicted hydrolaseycjY<>ycjZDNA-binding transcriptional regulator17.241B1432738Rac prophage transcriptional regulatorynaE<<uspF5.642D1468142KpLE2 phage-like IS repressor InsAinsC<insI>ydbCNAD(P)-binding oxidoreductase5.443D1500658tehB>*ydcL>ydcMpredicted transposase4.844B1518232yncB>>mcbRDNA-binding transcriptional regulator4.545B1542352nitrate/nitrite transporternarU<<yddJ4.946B1585730fimbrial-like adhesin proteinydeQ<<ydeR4.447B1588350fimbrial-like adhesin proteinydeS<<hipA4.548A1630638predicted mannonate dehydrogenaseydfI<>*ydfKQin prophage transcriptional regulator5.149B1638930Qin prophage; predicted S lysis proteinessQ<<cspB7.050A1639660Qin prophage; cold shock proteincspB<>*cspFQin prophage; cold shock protein7.351A1676254pyridine nucleotide transhydrogenasepntA<>ydgHpredicted protein5.052B1751846predicted oxidoreductaseydhV<<ydhY6.653A1801758threonyl-tRNA synthetasethrS<>yniDpredicted protein10.154D1852370predicted transporter*ydjE<*ydjF<ydjG4.455B1887966acyl-CoA synthetasefadD<<yeaY4.556A1923044conserved proteinyobA<>holEDNA polymerase III4.557D1938762myristoyl-acyl carrier ACP acyltransferaselpxM<*yebA<znuA7.158A2060070DNA-binding transcriptional dual regulatornac<>asnVAsn tRNA4.459D2103368predicted acyl transferasewbbJ<wbbI<rfc4.260D2104870conserved proteinwbbI<rfc<glf5.761D2144358uridine/cytidine kinaseudk<yegE<alkA5.862B2196338metG>>yehIconserved protein8.663D2210570conserved proteinyehS<yehT<yehU5.264D2214150yohO>yehW<yehX4.065D2302258yojO>eco<mqo7.266D2369270arnD>arnT>arnEconserved protein5.967A2405036DNA-binding transcriptional repressorlrhA<>yfbQpredicted aminotransferase4.068D24411383-oxoacyl-[acyl-carrier-protein] synthase IfabB<mnmC<yfcL5.769A2459264conserved proteinyfcZ<>fadLlong-chain fatty acid OM transporter5.370D2484966evgA>evgS<yfdE5.271D2490262predicted oxalyl-CoA decarboxylase*oxc<frc<yfdX4.972D2492854predicted proteinyfdX<*ypdI<yfdY4.173A2507446glucokinaseglk<>yfeOpredicted ion channel protein4.374A2535342pyridoxal-pyridoxamine kinasepdxK<>*yfeKpredicted protein7.575D2649866fused transglycosylase/transpeptidasepbpC<yfhM>sseA3-mercaptopyruvate sulfurtransferase6.176A2696642conserved proteinyfhB<>yfhHDNA-binding transcriptional regulator4.777A2708134RNA polymerase sigma 24rpoE<>nadBquinolinate synthase6.378B2761570CP4-57 prophage proteinyfjK<<yfjL5.679B2763272CP4-57 prophage proteinyfjL<<yfjM3.280D2764730CP4-57 prophage proteinyfjM<rnlA>yfjOCP4-57 prophage protein6.181B2773454yfjW>>yfjXCP4-57 prophage antirestriction protein6.182C2796040ygaP><stpA4.383A2812736S-ribosylhomocysteinaseluxS<>*micAncRNA40.984A2874372sulfate adenylyltransferasecysD<>*iapaminopeptidase4.285D2881556predicted proteinygcK<ygcL<ygcB10.986D2894072predicted flavoproteinygcQ<ygcR<ygcS4.087D2903952conserved proteinygcF<*ygcG<eno4.588B2985164yqeG>>yqeHprotein with bipartite regulator domain4.089A3004270DNA-binding transcriptional regulatorygeV<>*ygeWconserved protein4.590D3036652ssDNA exonucleaserecJ<*dsbC<xerD5.191B3237758alx>>sstTsodium:serine/threonine symporter23.492D3246540yqjA>yqjB>yqjCconserved protein4.493A3276944DNA-binding transcriptional dual regulatoragaR<>kbaZtagatose 6-phosphate aldolase 14.294B3375554stringent starvation protein AsspA<<rpsI4.795B3387142p-hydroxybenzoic acid efflux systemaaeA<<aaeX4.996C3528652hslO><yhgE4.197A3559934thiosulfate:cyanide sulfurtransferaseglpE<>glpDsn-glycerol-3-phosphate dehydrogenase4.098B3576850DNA-binding transcriptional repressorgntR<<yhhW4.999A3579162ncRNAryhB<>yhhYpredicted acetyltransferase6.5100B3599044RNA polymerase sigma 32*rpoH<<ftsX37.4101C3661646mdtF><gadW6.7102D3666862glutamate decarboxylase AgadA<yhjA>treFcytoplasmic trehalase6.6103B3720058insK>>sokAncRNA7.3104B37396604Fe-4S ferredoxin-type hydrogenaseysaA<<yiaJ4.2105D3764054predicted glutathione S-transferaseyibF<rhsA>yibAlyase containing HEAT-repeat4.0106B3779166lldD>>yibKpredicted rRNA methylase10.8107B3828470yicE>>yicHconserved protein4.2108D3829430yicE>yicH<yicI8.2109D3862372yidL>yidP<yidE4.3110B3906432phosphate transporter subunitpstB<<pstA5.2111D3955270ilvA>ilvY>ilvCketol-acid reductoisomerase4.1112A3958552peptidyl-prolyl cis-trans isomerase CppiC<>repDNA helicase/ssDNA-dependent ATPase4.8113B3978772rffM>>yifKpredicted transporter4.0114B4068538predicted aldose-1-epimeraseyihR<<yihS10.7115B4173964thrT>>tufBprotein chain elongation factor EF-Tu5.7116B4187734rpoC>>yjaZheat shock protein4.0117A4212172predicted acetyltransferaseyjaB<>metAhomoserine O-transsuccinylase4.3118B4249366malM>>ubiCchorismate pyruvate lyase5.7119B4364770C4-dicarboxylate antiporterdcuA<<aspA5.2120A4368630predicted transporteryjeH<>groSCpn10 chaperonin GroES4.3121A4380530anaerobic fumarate reductasefrdA<>poxApredicted lysyl-tRNA synthetase4.0122B4427854fklB>>cycAD-alanine/D-serine/glycine transporter8.0123D4486746yjgQ>yjgR<idnR9.4124B4530030KpLE2 phage endoglucanasesgcX<<yjhP5.2125B4533162conserved proteinyjhX<<yjhS4.0126A4538964N-acetylnuraminic acid outer membrane channel proteinnanC<>fimBtyrosine recombinase/fimA regulator6.4127B4638610yjjY>>*yjtDpredicted rRNA methyltransferase4.6Genomic SELEX was performed for search of the binding sites of RNAP RpoE holoenzyme. By setting the cut-off level of 4.0, a total of 126 binding sites were identified (see Fig 4 for SELEX pattern), which are aligned along the map of E. coli K12 genome. A total of 84 sites are located within intergenic spacers: 29 wihin type-A spacers (shown under orange background); and 48 within type-B spacers (shown under green background). The constitutive promoters of RpoE were predicted based on the adjacent genes [note that only the genes next to the RpoE holoenzyme-binding sites are shown] and the gene orientation (shown by arrows in the column of transcription direction). A total of 42 RpoE holoenzyme-binding sites are located inside open reading frames as indicated by the gene symbols shown in RpoE column.*The genes listed in RegulonDB as the regulated targets of RpoE.Genomic SELEX analysis identified minimum 77 and maximum 106 for the RpoE constitutive promoters. The highest peak (55-fold intensity) was detected in the promoter region of rybB, which encodes a small regulatory RNA for expression control of some outer membrane proteins. The rybB promoter is known to be regulated by RpoE (Fig 4) [67]. The second highest peak was located on the luxS-micA intergenic region. The micA gene again encodes a small regulatory RNA that regulates expression of many genes including outer membrane proteins [68,69]. The micA promoter is also established under the control of RpoE. These sRNAs control the repair of damages in the outer membrane that took place in response to envelope stress [70,71]. High-intensity peaks were detected on some other known RpoE-dependent promoters such as rpoH, pgrR and ycjY [72,73]. Among the total of 136 binding sites of RpoE holoenzyme, 36 overlaps with that of RpoH holoenzyme. Most of these overlapping sites are related to the genes that are expressed under envelope stresss or heat-shock stress.

The whole set of constitutive promoters for extra-cytoplasmic stress response sigma RpoE

The bacterial cell envelope is a dynamic compartment, changing its structure and function in response to environmental conditions. Accordingly, the integrity of envelope is maintained through frequent modulation of its composition and components. The minor sigma factor RpoE plays a central role in this process, by controlling the selective expression of envelope components [61]. The regulatory targets have been estimated after proteome and transcriptome analyses in vivo [62–64]. The activity of RpoE is negatively regulated by a membrane-bound anti-sigma factor RseA, which sequesters RpoE under unstressed conditions. Within membrane, RseA is associated at its C-terminal domain with a periplasmic protein RseB, which senses misfolded proteins for release and activation of RpoE from RpoE-RseA complexes [62,65]. The total number of RpoE promoters (or the transcription initiation sites) listed in the current RegulonDB database is as many as 518, of which most are identified by computational analyses based on the consensus sequence of RpoE promoters without experimental analysis. After SELEX screening as noted below, most of these RpoE promoters must be inaccurate estimation due to the error in the consensus sequence.

The Genomic SELEX screening system was employed as a short-cut approach for identification of the RpoE regulon. Previously we purified RpoE and examined its promoter selectivity using an in vitro transcription system [66]. Using this purified RpoE, we performed SELEX screening. By setting the cut-off level of 4.0 fold against original library DNA, a total of 126 RpoE holoenzyme-binding peaks were identified (Fig 4 and Table 5), of which 84 (67%) are located within intergenic spacers and 42 (33%) are inside of open reading frames (Tables 2 and 5 for the whole set). Since the majority of hitherto identified promoters are located within spacers, detailed search for the constitutive promoters was focused on the total of 84 peaks within spacers. The spacers containing RpoE holoenzyme-binding sites were also classified into three types (Table 2): 29 peaks are located within type-A spacer; 48 peaks are located within type-B spacers; and 7 peaks are located within type-C spacers. Based on the transcription direction of flanking genes, the total number of RpoE constitutive promoters was predicted to range between minimum 77 (29 type-A plus 48 type-B) and maximum 106 (29x2 type-A plus 48 type-B) (Table 2). Within the set of constitutive promoters identified by setting the cut-off level at 4.0, a total of 19 known RpoE promoters were identified (Table 5, marked by asterisk). The majority of known promoters represent the inducible promoters that are expressed only under the support of positive regulatory factors.

SELEX-chip search for RNAP RpoE holoenzyme-binding sequences on the E. coli K-12 genome.

The y-axis represents the relative number of RpoE holoenzyme-bound DNA fragments whereas x-axis represents the position on the E.coli genome, in base pair. The adjacent gene on E. coli K-12 genome of peak position was indicated for high intensity peaks. The peaks located within spacer regions are shown in green color, while peaks located within open reading frames are shown in orange color. The list of RpoE holoenzyme-binding sites is described in Table 5.

RpoE holoenzyme-binding sites on the E. coli K-12 genome.

Genomic SELEX was performed for search of the binding sites of RNAP RpoE holoenzyme. By setting the cut-off level of 4.0, a total of 126 binding sites were identified (see Fig 4 for SELEX pattern), which are aligned along the map of E. coli K12 genome. A total of 84 sites are located within intergenic spacers: 29 wihin type-A spacers (shown under orange background); and 48 within type-B spacers (shown under green background). The constitutive promoters of RpoE were predicted based on the adjacent genes [note that only the genes next to the RpoE holoenzyme-binding sites are shown] and the gene orientation (shown by arrows in the column of transcription direction). A total of 42 RpoE holoenzyme-binding sites are located inside open reading frames as indicated by the gene symbols shown in RpoE column.

*The genes listed in RegulonDB as the regulated targets of RpoE.

Genomic SELEX analysis identified minimum 77 and maximum 106 for the RpoE constitutive promoters. The highest peak (55-fold intensity) was detected in the promoter region of rybB, which encodes a small regulatory RNA for expression control of some outer membrane proteins. The rybB promoter is known to be regulated by RpoE (Fig 4) [67]. The second highest peak was located on the luxS-micA intergenic region. The micA gene again encodes a small regulatory RNA that regulates expression of many genes including outer membrane proteins [68,69]. The micA promoter is also established under the control of RpoE. These sRNAs control the repair of damages in the outer membrane that took place in response to envelope stress [70,71]. High-intensity peaks were detected on some other known RpoE-dependent promoters such as rpoH, pgrR and ycjY [72,73]. Among the total of 136 binding sites of RpoE holoenzyme, 36 overlaps with that of RpoH holoenzyme. Most of these overlapping sites are related to the genes that are expressed under envelope stresss or heat-shock stress.

The intracellular levels of all seven sigma factors in E. coli K-12 W3110In this study, we determined the constitutive promoters for the four minor sigma factors, RpoS, RpoH, RpoF and RpoE, from E. coli K-12 W3110. These promoters are recognized by the RNAP holoenzyme containing each sigma in the absence of other supporting factors. Using the mixed reconstitution in vitro of RNAP holoenzyme in the presence of all seven sigma factors, we estimated the binding affinity of each sigma to the common core enzyme, the order being RpoD (highest) > RpoN > RpoF > RpoH > FecI > RpoE > RpoS (lowest) [74]. Once we get the knowledge of intracellular concentrations of these sigma factors, it should be possible to predict the expression levels of the regulatory target genes and operons under the control of the constitutive promoters of each sigma factor. Including these four minor sigma factors, we then determined the intracellular concentrations of all seven sigma subunits. For this purpose, antibodies were made against each of the purified sigma factors that were also used for SELEX screening.E. coli K-12 W3110 type-A was cultivated with shaking at 37°C in LB medium, and the whole cell lysates were prepared in both exponential growing phase and the stationary phase. By using the quantitative immuno-blotting method and the purified sigma proteins as the reference controls, we measured the concentrations of all seven sigma subunits. The measurement was carried out for two independent cultures, and the immuno-blot analysis was repeated for all the samples. The intracellular concentration of RpoD sigma is maintained at a constant level (on average, 160 fmol/μg total protein) throughout the transition from the exponential growth phase to the stationary phase (Fig 5) in good agreement with the previous estimation [1,75]. Based on the relative level of RNAP core enzyme subunits, the number of RpoD sigma was estimated to be 800 molecules per genome [note that the number of RNAP core enzyme is 2,000 molecules per genome]. The level of RpoD sigma in E. coli K-12 MG1655 was about half the level of W3110 (data not shown). The second abundant sigma was RpoF (70 fmol/μg protein at log phase and 80 fmol/μg protein at stationary phase) (Fig 5), but the unneccesary RpoF is stored in an inactive form by forming complexes with the anti-RpoF sigma, FlgM [76]. RpoN is also always present at a constant level (approximately 40 fmol/μg protein) in both log and stationary phases (Fig 5). The levels of other four sigma, RpoS, RpoH, RpoE and FecI, are very low during the steady-state growth (Fig 5A), but upon entry into the stationary phase, the level of RpoS increased markedly up to the level (about 100 fmol/μg protein) higher than other five minor sigma factors (Fig 5B). Taking the intracellular concentrations and the binding affinity of sigma to RNAP core enzyme as noted above, we are now able to estimate the level of each RNAP holoenzyme. Noteworthy is that the total number of all seven sigma factors is approximately as many as that of the core enzyme, but the RNAP involved in the transcription cycle or the elongation of RNA chains is considered to lack sigma subunit, the RNAP not involved in transcription should be stored as the holoenzyme forms.10.1371/journal.pone.0179181.g005Fig 5Intracellular concentrations of seven sigma factors in E. coli K-12 W3110 type-A strain.E. coli K-12 W3110 type-A strain was grown in LB medium at 37°C with shaking. Cells were harvested at various times and cell lysates were subjected to the quantitative immuno-blot analysis of all seven sigma factors as described in Materials and Methods. [A] The sigma levels at exponential growth phase; [B] the sigma levels in the stationary phase.

The intracellular levels of all seven sigma factors in E. coli K-12 W3110

In this study, we determined the constitutive promoters for the four minor sigma factors, RpoS, RpoH, RpoF and RpoE, from E. coli K-12 W3110. These promoters are recognized by the RNAP holoenzyme containing each sigma in the absence of other supporting factors. Using the mixed reconstitution in vitro of RNAP holoenzyme in the presence of all seven sigma factors, we estimated the binding affinity of each sigma to the common core enzyme, the order being RpoD (highest) > RpoN > RpoF > RpoH > FecI > RpoE > RpoS (lowest) [74]. Once we get the knowledge of intracellular concentrations of these sigma factors, it should be possible to predict the expression levels of the regulatory target genes and operons under the control of the constitutive promoters of each sigma factor. Including these four minor sigma factors, we then determined the intracellular concentrations of all seven sigma subunits. For this purpose, antibodies were made against each of the purified sigma factors that were also used for SELEX screening.

E. coli K-12 W3110 type-A was cultivated with shaking at 37°C in LB medium, and the whole cell lysates were prepared in both exponential growing phase and the stationary phase. By using the quantitative immuno-blotting method and the purified sigma proteins as the reference controls, we measured the concentrations of all seven sigma subunits. The measurement was carried out for two independent cultures, and the immuno-blot analysis was repeated for all the samples. The intracellular concentration of RpoD sigma is maintained at a constant level (on average, 160 fmol/μg total protein) throughout the transition from the exponential growth phase to the stationary phase (Fig 5) in good agreement with the previous estimation [1,75]. Based on the relative level of RNAP core enzyme subunits, the number of RpoD sigma was estimated to be 800 molecules per genome [note that the number of RNAP core enzyme is 2,000 molecules per genome]. The level of RpoD sigma in E. coli K-12 MG1655 was about half the level of W3110 (data not shown). The second abundant sigma was RpoF (70 fmol/μg protein at log phase and 80 fmol/μg protein at stationary phase) (Fig 5), but the unneccesary RpoF is stored in an inactive form by forming complexes with the anti-RpoF sigma, FlgM [76]. RpoN is also always present at a constant level (approximately 40 fmol/μg protein) in both log and stationary phases (Fig 5). The levels of other four sigma, RpoS, RpoH, RpoE and FecI, are very low during the steady-state growth (Fig 5A), but upon entry into the stationary phase, the level of RpoS increased markedly up to the level (about 100 fmol/μg protein) higher than other five minor sigma factors (Fig 5B). Taking the intracellular concentrations and the binding affinity of sigma to RNAP core enzyme as noted above, we are now able to estimate the level of each RNAP holoenzyme. Noteworthy is that the total number of all seven sigma factors is approximately as many as that of the core enzyme, but the RNAP involved in the transcription cycle or the elongation of RNA chains is considered to lack sigma subunit, the RNAP not involved in transcription should be stored as the holoenzyme forms.

Intracellular concentrations of seven sigma factors in E. coli K-12 W3110 type-A strain.

E. coli K-12 W3110 type-A strain was grown in LB medium at 37°C with shaking. Cells were harvested at various times and cell lysates were subjected to the quantitative immuno-blot analysis of all seven sigma factors as described in Materials and Methods. [A] The sigma levels at exponential growth phase; [B] the sigma levels in the stationary phase.

DiscussionSeven species of the sigma subunit exist in E. coli K-12, the widely used model E. coli strain. Here we identified the whole set of constitutive promoters for four minor sigma factors, RpoS, RpoH, RpoF and RpoE, by using the Genomic SELEX system. Up to the present time, the binding sites of RNAP and TF have been identified in vivo using the high-throughput systems such as ChIP-chip, ChIP-seq and RNA-seq systems. Even using these modern techniques, however, it is in principle impossible to obtain the whole set of binding sites for both RNAP and TFs because of the competition with other DNA-binding proteins in binding to DNA targets [8,9,24] [note that E. coli contains more than 500 DNA-binding proteins [77], and because the binding of RNAP and TFs often depends on the supporting factors for binding to targets [8,9,25].The computational approaches in silico have also been employed to identify the target binding sequences of RNAP and TFs, relying on the consensus sequences predicted based on the known target sequences listed in the databases such as RegulonDB [20,21] and EcoCyc [22,23] (Table 2) (Details of the promoter list and the evidence are in Supplemental Information: S1 Table for RpoS; S2 Table for RpoH; S3 Table for RpoF; and S4 Table for RpoE). In particular, more than 80% of RpoE-dependent promoters were predicted in silico (Table 5; and S4 Table). The consensus sequences, however, often include the inaccurate non-target sequences due to the lack of experiments for confirmation or some regulators recognize wide-varieties of the binding sequences [8,9,27,78]. Another serious problem associated with in vivo approaches is the difference in genetic background of E. coli strains used. Up to the present, the complete genome sequence has been determined for more than 1,000 different E. coli strains, allowing the prediction of about 3,000 core genes for all strains but at least one third of the total genes on the E. coli genome are different among the hitherto sequenced E. coli genome [79]. The difference in genetic background exists even in the RNAP and TF genes and between not only different strains but also different stocks of the same E. coli strain. For instance, the difference in the gene encoding the stationary-phase sigma RpoS was first identified between laboratory stocks of a single and the same E. coli K-12 W3110 strain [80]. The widely used databases such as RegulonDB [20,21] and EcoCyc [22,23] include huge collections of useful data of transcription in vivo, but care should be taken to use these data for theoretical prediction of transcription regulation, in particular, as to the bacterial strains and culture conditions used in each experiment.In this study, we performed the SELEX screening for the constitutive promoters that are recognized in vitro by four minor sigma factors, RpoS, RpoH, RpoF and RpoE, but in the absence of repressors, activators and other DNA-binding proteins. It should be noted that all the proteins and promoters used in this study are prepared from a single and the same E. coli K-12 W3110. Here we also determined the intracellular concentrations of all seven sigma factors in both growing and stationary-phase cells of E. coli K-12 W3110. These data altogether will be used for our ultimate purpose of the prediction of genome expression under a given condition. The list of constitutive promoters for the minor sigma factors will be deposited into TEC database (Transcriptional profile of Escherichia coli database: https://shigen.nig.ac.jp/ecoli/tec/top/) [9].

Seven species of the sigma subunit exist in E. coli K-12, the widely used model E. coli strain. Here we identified the whole set of constitutive promoters for four minor sigma factors, RpoS, RpoH, RpoF and RpoE, by using the Genomic SELEX system. Up to the present time, the binding sites of RNAP and TF have been identified in vivo using the high-throughput systems such as ChIP-chip, ChIP-seq and RNA-seq systems. Even using these modern techniques, however, it is in principle impossible to obtain the whole set of binding sites for both RNAP and TFs because of the competition with other DNA-binding proteins in binding to DNA targets [8,9,24] [note that E. coli contains more than 500 DNA-binding proteins [77], and because the binding of RNAP and TFs often depends on the supporting factors for binding to targets [8,9,25].

The computational approaches in silico have also been employed to identify the target binding sequences of RNAP and TFs, relying on the consensus sequences predicted based on the known target sequences listed in the databases such as RegulonDB [20,21] and EcoCyc [22,23] (Table 2) (Details of the promoter list and the evidence are in Supplemental Information: S1 Table for RpoS; S2 Table for RpoH; S3 Table for RpoF; and S4 Table for RpoE). In particular, more than 80% of RpoE-dependent promoters were predicted in silico (Table 5; and S4 Table). The consensus sequences, however, often include the inaccurate non-target sequences due to the lack of experiments for confirmation or some regulators recognize wide-varieties of the binding sequences [8,9,27,78]. Another serious problem associated with in vivo approaches is the difference in genetic background of E. coli strains used. Up to the present, the complete genome sequence has been determined for more than 1,000 different E. coli strains, allowing the prediction of about 3,000 core genes for all strains but at least one third of the total genes on the E. coli genome are different among the hitherto sequenced E. coli genome [79]. The difference in genetic background exists even in the RNAP and TF genes and between not only different strains but also different stocks of the same E. coli strain. For instance, the difference in the gene encoding the stationary-phase sigma RpoS was first identified between laboratory stocks of a single and the same E. coli K-12 W3110 strain [80]. The widely used databases such as RegulonDB [20,21] and EcoCyc [22,23] include huge collections of useful data of transcription in vivo, but care should be taken to use these data for theoretical prediction of transcription regulation, in particular, as to the bacterial strains and culture conditions used in each experiment.

In this study, we performed the SELEX screening for the constitutive promoters that are recognized in vitro by four minor sigma factors, RpoS, RpoH, RpoF and RpoE, but in the absence of repressors, activators and other DNA-binding proteins. It should be noted that all the proteins and promoters used in this study are prepared from a single and the same E. coli K-12 W3110. Here we also determined the intracellular concentrations of all seven sigma factors in both growing and stationary-phase cells of E. coli K-12 W3110. These data altogether will be used for our ultimate purpose of the prediction of genome expression under a given condition. The list of constitutive promoters for the minor sigma factors will be deposited into TEC database (Transcriptional profile of Escherichia coli database: https://shigen.nig.ac.jp/ecoli/tec/top/) [9].

Materials and methodsBacterial strains and plasmidsE. coli K12 W3350 type-A containing the full-set of seven sigma factors [80] was used for purification of RNA polymerase and the template DNA for Genomic SELEX screening of RpoS, RpoH, RpoF and RpoE promoters. E. coli BL21(DE3) was used for the expression and purification of sigma and core enzyme subunit proteins. Expression plasmids for the core enzyme subunits (pRpoA, pRpoB and pRpoC) and all seven sigma subunits (pRpoD, pRpoN, pRpoS, pRpoH, pRpoF, pRpoE and pFecI) were constructed by ligating the respective coding sequences, which were prepared by PCR amplification of the E. coli K12 W3350 type-A genome DNA as template, into pET21 expression vector essentially according to the standard procedure used for expression of all sigma and all transcription factors in this laboratory [74,81].Purification of core RNA polymeraseRNAP was purified from log-phase cells of E. coli K-12 W3350 by the standard procedure [29]. Separation of the native core from holoenzymes was performed by passing the purified RNAP through P11-phosphocellulose column in the presence of 50% glycerol. To remove trace amounts of the core enzyme-associated sigma factors, the purified RNAP in the storage buffer containing 50% glycerol was dialyzed against the same buffer but containing 5% glycerol and fractionated by P11-phosphocellulose column chromatography in the presence of 5% glycerol. The level of remaining sigma factors was less than 0.1%, if any, as checked of SDS-PAGE gels by both protein-staining with a silver reagent and immuno-staining with antibodies against each of seven sigma factors.Purification of core and sigma subunitsThe core enzyme subunits (RpoA, RpoB, RpoC and RpoZ) were expressed using the respective expression plasmids and purified by two cycles of column chromatography through DEAE (DE52) and P11-phosphocellulose [29]. Sigma subunits were expressed and purified by ion-exchange column chromatography through DE52 and P11 followed by Sephacryl S-300 gel filtration column. The purified sigma and core subunit proteins were more than 99% pure as judged by both protein-staining and immuno-staining of SDS-PAGE gels.Preparation of antibodiesAntibodies against sigma factors and core enzyme subunits were produced in rabbits by injecting purified sigma proteins [75,76]. Antibodies against each RNA polymerase protein were produced in two rabbits, and after examination of antibody activity using immune-blot analysis, the batch of higher activity was used in this study. Anti-RpoD, anti-RpoS, anti-RpoN, anti-RpoH, anti-RpoF, anti-RpoE, anti-FecI and anti-RpoC used in this study did not cross-react with each other. These antibodies were produced in the Nippon Institute for Biological Science (One, Tokyo) and the Animal Laboratory of Mitsubishi Chemical Medience Co. (Uto, Kumamoto, Japan).Genomic SELEX screening of RNA polymerase holoenzyme binding sequencesThe Genomic SELEX screening was carried out under the standard procedure [26,27]. This method was developed by improvement of the original SELEX methods [30–32]. A mixture of DNA fragments of the E. coli K-12 W3110 genome was prepared after sonication of purified genome DNA, and cloned into a multi-copy plasmid pBR322 at EcoRV site. In each SELEX screening, the DNA mixture was regenerated by PCR using a pair of primers with the flanking sequences of pBR322 EcoRV. For SELEX screening, 5 pmol of the mixture of DNA fragments and 10 pmol of RNA polymerase holoenzyme were mixed in a binding buffer (10 mM Tris-HCl, pH 7.8 at 4°C, 3 mM magnesium acetate, 150 mM NaCl, and 1.25 mg/ml bovine serum albumin) and incubated for 30 min at 37°C. The DNA-RNA polymerase mixture was treated with anti-RpoC antibody and DNA fragments recovered from the complexes were PCR-amplified and subjected to next cycle of SELEX for enrichment of RNA polymerase-bound DNA fragments.For SELEX-chip analysis, DNA samples were isolated from the DNA-protein complexes at the final state of SELEX, PCR-amplified and labeled with Cy5 while the original DNA library was labeled with Cy3. The fluorescent labeled DNA mixtures were hybridized to a DNA microarray consisting of 43,450 species of 60 b-long DNA probe, which are designed to cover the entire E. coli K-12 MG1655 genome at 105 bp interval (Oxford Gene Technology, Oxford, UK) [14]. The fluorescent intensity of test sample at each probe was normalized with that of the corresponding peak of original library. After normalization of each pattern, the Cy5/Cy3 ratio was measured and plotted along the E. coli K-12 MG1655 genome. The gene organization is almost identical between two well-characterized E. coli K-12 strains except for a long-range inversion between the rrnD and rrnE operons.Immuno-blot analysis for determination of sigma levelsFor the measurement of sigma factors in E. coli K-12 W3110, a quantitative Western blot analysis was employed with the anti-sigma antibodies as employed in the previous studies [66,75,76]. In brief, cell lysates were treated with a SDS (sodium dodecyl sulfate) sample buffer (50 mM Tris-HCl, pH 6.8, 2% SDS, 1% 2-mercaptoethanol, 10% glycerol, and 0.025% bromophenol blue) and separated on SDS–7.5 or 10% polyacrylamide gels. Proteins in gels were directly electro-blotted onto polyvinylidene difluoride membranes (Nippon Genetics). Blots were blocked overnight at 48C in 3% BSA in PBS (phosphate-buffered saline), probed with the polyclonal antibodies against each sigma factor, washed with 0.5% Tween 20 in PBS, and incubated with goat anti-rabbit immunoglobulin G conjugated with hydroxyperoxidase (Cappel). The blots were developed with 3,3’-diaminobenzidine tetrahydrochloride (Dojindo). Staining intensity was measured with a PDI image analyzer system equipped with a white light scanner. The standard curve for the calculation of each sigma level was prepared from the immuno-blot patterns of increasing concentrations of each sigma factor. Under the standard Western-blot conditions herein employed, the linearity was detected over a 10-fold range at least between 2 and 20 ng sigma proteins. The determination of test sigma proteins subunits was first performed using several different volumes of the cell lysates. Using the optimum volumes of cell lysates to give the sigma concentrations within the linear range of standard curves, we finally repeated the determination of individual sigma factors.

Materials and methods

Bacterial strains and plasmidsE. coli K12 W3350 type-A containing the full-set of seven sigma factors [80] was used for purification of RNA polymerase and the template DNA for Genomic SELEX screening of RpoS, RpoH, RpoF and RpoE promoters. E. coli BL21(DE3) was used for the expression and purification of sigma and core enzyme subunit proteins. Expression plasmids for the core enzyme subunits (pRpoA, pRpoB and pRpoC) and all seven sigma subunits (pRpoD, pRpoN, pRpoS, pRpoH, pRpoF, pRpoE and pFecI) were constructed by ligating the respective coding sequences, which were prepared by PCR amplification of the E. coli K12 W3350 type-A genome DNA as template, into pET21 expression vector essentially according to the standard procedure used for expression of all sigma and all transcription factors in this laboratory [74,81].

Bacterial strains and plasmids

E. coli K12 W3350 type-A containing the full-set of seven sigma factors [80] was used for purification of RNA polymerase and the template DNA for Genomic SELEX screening of RpoS, RpoH, RpoF and RpoE promoters. E. coli BL21(DE3) was used for the expression and purification of sigma and core enzyme subunit proteins. Expression plasmids for the core enzyme subunits (pRpoA, pRpoB and pRpoC) and all seven sigma subunits (pRpoD, pRpoN, pRpoS, pRpoH, pRpoF, pRpoE and pFecI) were constructed by ligating the respective coding sequences, which were prepared by PCR amplification of the E. coli K12 W3350 type-A genome DNA as template, into pET21 expression vector essentially according to the standard procedure used for expression of all sigma and all transcription factors in this laboratory [74,81].

Purification of core RNA polymeraseRNAP was purified from log-phase cells of E. coli K-12 W3350 by the standard procedure [29]. Separation of the native core from holoenzymes was performed by passing the purified RNAP through P11-phosphocellulose column in the presence of 50% glycerol. To remove trace amounts of the core enzyme-associated sigma factors, the purified RNAP in the storage buffer containing 50% glycerol was dialyzed against the same buffer but containing 5% glycerol and fractionated by P11-phosphocellulose column chromatography in the presence of 5% glycerol. The level of remaining sigma factors was less than 0.1%, if any, as checked of SDS-PAGE gels by both protein-staining with a silver reagent and immuno-staining with antibodies against each of seven sigma factors.

Purification of core RNA polymerase

RNAP was purified from log-phase cells of E. coli K-12 W3350 by the standard procedure [29]. Separation of the native core from holoenzymes was performed by passing the purified RNAP through P11-phosphocellulose column in the presence of 50% glycerol. To remove trace amounts of the core enzyme-associated sigma factors, the purified RNAP in the storage buffer containing 50% glycerol was dialyzed against the same buffer but containing 5% glycerol and fractionated by P11-phosphocellulose column chromatography in the presence of 5% glycerol. The level of remaining sigma factors was less than 0.1%, if any, as checked of SDS-PAGE gels by both protein-staining with a silver reagent and immuno-staining with antibodies against each of seven sigma factors.

Purification of core and sigma subunitsThe core enzyme subunits (RpoA, RpoB, RpoC and RpoZ) were expressed using the respective expression plasmids and purified by two cycles of column chromatography through DEAE (DE52) and P11-phosphocellulose [29]. Sigma subunits were expressed and purified by ion-exchange column chromatography through DE52 and P11 followed by Sephacryl S-300 gel filtration column. The purified sigma and core subunit proteins were more than 99% pure as judged by both protein-staining and immuno-staining of SDS-PAGE gels.

Purification of core and sigma subunits

The core enzyme subunits (RpoA, RpoB, RpoC and RpoZ) were expressed using the respective expression plasmids and purified by two cycles of column chromatography through DEAE (DE52) and P11-phosphocellulose [29]. Sigma subunits were expressed and purified by ion-exchange column chromatography through DE52 and P11 followed by Sephacryl S-300 gel filtration column. The purified sigma and core subunit proteins were more than 99% pure as judged by both protein-staining and immuno-staining of SDS-PAGE gels.

Preparation of antibodiesAntibodies against sigma factors and core enzyme subunits were produced in rabbits by injecting purified sigma proteins [75,76]. Antibodies against each RNA polymerase protein were produced in two rabbits, and after examination of antibody activity using immune-blot analysis, the batch of higher activity was used in this study. Anti-RpoD, anti-RpoS, anti-RpoN, anti-RpoH, anti-RpoF, anti-RpoE, anti-FecI and anti-RpoC used in this study did not cross-react with each other. These antibodies were produced in the Nippon Institute for Biological Science (One, Tokyo) and the Animal Laboratory of Mitsubishi Chemical Medience Co. (Uto, Kumamoto, Japan).

Preparation of antibodies

Antibodies against sigma factors and core enzyme subunits were produced in rabbits by injecting purified sigma proteins [75,76]. Antibodies against each RNA polymerase protein were produced in two rabbits, and after examination of antibody activity using immune-blot analysis, the batch of higher activity was used in this study. Anti-RpoD, anti-RpoS, anti-RpoN, anti-RpoH, anti-RpoF, anti-RpoE, anti-FecI and anti-RpoC used in this study did not cross-react with each other. These antibodies were produced in the Nippon Institute for Biological Science (One, Tokyo) and the Animal Laboratory of Mitsubishi Chemical Medience Co. (Uto, Kumamoto, Japan).

Genomic SELEX screening of RNA polymerase holoenzyme binding sequencesThe Genomic SELEX screening was carried out under the standard procedure [26,27]. This method was developed by improvement of the original SELEX methods [30–32]. A mixture of DNA fragments of the E. coli K-12 W3110 genome was prepared after sonication of purified genome DNA, and cloned into a multi-copy plasmid pBR322 at EcoRV site. In each SELEX screening, the DNA mixture was regenerated by PCR using a pair of primers with the flanking sequences of pBR322 EcoRV. For SELEX screening, 5 pmol of the mixture of DNA fragments and 10 pmol of RNA polymerase holoenzyme were mixed in a binding buffer (10 mM Tris-HCl, pH 7.8 at 4°C, 3 mM magnesium acetate, 150 mM NaCl, and 1.25 mg/ml bovine serum albumin) and incubated for 30 min at 37°C. The DNA-RNA polymerase mixture was treated with anti-RpoC antibody and DNA fragments recovered from the complexes were PCR-amplified and subjected to next cycle of SELEX for enrichment of RNA polymerase-bound DNA fragments.For SELEX-chip analysis, DNA samples were isolated from the DNA-protein complexes at the final state of SELEX, PCR-amplified and labeled with Cy5 while the original DNA library was labeled with Cy3. The fluorescent labeled DNA mixtures were hybridized to a DNA microarray consisting of 43,450 species of 60 b-long DNA probe, which are designed to cover the entire E. coli K-12 MG1655 genome at 105 bp interval (Oxford Gene Technology, Oxford, UK) [14]. The fluorescent intensity of test sample at each probe was normalized with that of the corresponding peak of original library. After normalization of each pattern, the Cy5/Cy3 ratio was measured and plotted along the E. coli K-12 MG1655 genome. The gene organization is almost identical between two well-characterized E. coli K-12 strains except for a long-range inversion between the rrnD and rrnE operons.

Genomic SELEX screening of RNA polymerase holoenzyme binding sequences

The Genomic SELEX screening was carried out under the standard procedure [26,27]. This method was developed by improvement of the original SELEX methods [30–32]. A mixture of DNA fragments of the E. coli K-12 W3110 genome was prepared after sonication of purified genome DNA, and cloned into a multi-copy plasmid pBR322 at EcoRV site. In each SELEX screening, the DNA mixture was regenerated by PCR using a pair of primers with the flanking sequences of pBR322 EcoRV. For SELEX screening, 5 pmol of the mixture of DNA fragments and 10 pmol of RNA polymerase holoenzyme were mixed in a binding buffer (10 mM Tris-HCl, pH 7.8 at 4°C, 3 mM magnesium acetate, 150 mM NaCl, and 1.25 mg/ml bovine serum albumin) and incubated for 30 min at 37°C. The DNA-RNA polymerase mixture was treated with anti-RpoC antibody and DNA fragments recovered from the complexes were PCR-amplified and subjected to next cycle of SELEX for enrichment of RNA polymerase-bound DNA fragments.

For SELEX-chip analysis, DNA samples were isolated from the DNA-protein complexes at the final state of SELEX, PCR-amplified and labeled with Cy5 while the original DNA library was labeled with Cy3. The fluorescent labeled DNA mixtures were hybridized to a DNA microarray consisting of 43,450 species of 60 b-long DNA probe, which are designed to cover the entire E. coli K-12 MG1655 genome at 105 bp interval (Oxford Gene Technology, Oxford, UK) [14]. The fluorescent intensity of test sample at each probe was normalized with that of the corresponding peak of original library. After normalization of each pattern, the Cy5/Cy3 ratio was measured and plotted along the E. coli K-12 MG1655 genome. The gene organization is almost identical between two well-characterized E. coli K-12 strains except for a long-range inversion between the rrnD and rrnE operons.

Immuno-blot analysis for determination of sigma levelsFor the measurement of sigma factors in E. coli K-12 W3110, a quantitative Western blot analysis was employed with the anti-sigma antibodies as employed in the previous studies [66,75,76]. In brief, cell lysates were treated with a SDS (sodium dodecyl sulfate) sample buffer (50 mM Tris-HCl, pH 6.8, 2% SDS, 1% 2-mercaptoethanol, 10% glycerol, and 0.025% bromophenol blue) and separated on SDS–7.5 or 10% polyacrylamide gels. Proteins in gels were directly electro-blotted onto polyvinylidene difluoride membranes (Nippon Genetics). Blots were blocked overnight at 48C in 3% BSA in PBS (phosphate-buffered saline), probed with the polyclonal antibodies against each sigma factor, washed with 0.5% Tween 20 in PBS, and incubated with goat anti-rabbit immunoglobulin G conjugated with hydroxyperoxidase (Cappel). The blots were developed with 3,3’-diaminobenzidine tetrahydrochloride (Dojindo). Staining intensity was measured with a PDI image analyzer system equipped with a white light scanner. The standard curve for the calculation of each sigma level was prepared from the immuno-blot patterns of increasing concentrations of each sigma factor. Under the standard Western-blot conditions herein employed, the linearity was detected over a 10-fold range at least between 2 and 20 ng sigma proteins. The determination of test sigma proteins subunits was first performed using several different volumes of the cell lysates. Using the optimum volumes of cell lysates to give the sigma concentrations within the linear range of standard curves, we finally repeated the determination of individual sigma factors.

Immuno-blot analysis for determination of sigma levels

For the measurement of sigma factors in E. coli K-12 W3110, a quantitative Western blot analysis was employed with the anti-sigma antibodies as employed in the previous studies [66,75,76]. In brief, cell lysates were treated with a SDS (sodium dodecyl sulfate) sample buffer (50 mM Tris-HCl, pH 6.8, 2% SDS, 1% 2-mercaptoethanol, 10% glycerol, and 0.025% bromophenol blue) and separated on SDS–7.5 or 10% polyacrylamide gels. Proteins in gels were directly electro-blotted onto polyvinylidene difluoride membranes (Nippon Genetics). Blots were blocked overnight at 48C in 3% BSA in PBS (phosphate-buffered saline), probed with the polyclonal antibodies against each sigma factor, washed with 0.5% Tween 20 in PBS, and incubated with goat anti-rabbit immunoglobulin G conjugated with hydroxyperoxidase (Cappel). The blots were developed with 3,3’-diaminobenzidine tetrahydrochloride (Dojindo). Staining intensity was measured with a PDI image analyzer system equipped with a white light scanner. The standard curve for the calculation of each sigma level was prepared from the immuno-blot patterns of increasing concentrations of each sigma factor. Under the standard Western-blot conditions herein employed, the linearity was detected over a 10-fold range at least between 2 and 20 ng sigma proteins. The determination of test sigma proteins subunits was first performed using several different volumes of the cell lysates. Using the optimum volumes of cell lysates to give the sigma concentrations within the linear range of standard curves, we finally repeated the determination of individual sigma factors.

Supporting informationS1 TableRpoS-dependent promoters listed in RegulonDB.Promoters listed in RegulonDB are classified into those not identified as the constitutive promoters (A) and the constitutive promoters identified by SELEX screening (B). Evidence for each promoter are as described in RegulonDB (see Table 5): (Group-A) Promoters were experimentally identified by using HTTIM (high-throughput transcription initiation mapping), TIM (transcription initiation mapping), FP (footprinting), or IDA (inferred by direct promoter assay; (Class-B) Promoters were predicted based on AIPP (automated inference of promoter position), ICWHO (inferred computationally without human oversight), HIPP (human inference of promoter position), NTAS (non-traceable author statement), TASES (traceable author statement to experimental support), TAS (traceable author statement), IMP (inferred from mutant) or IEP (inferred from expression pattern).(PDF)Click here for additional data file.S2 TableRpoH-dependent promoters listed in RegulonDB.Promoters listed in RegulonDB are classified into those not identified as the constitutive promoters (A) and the constitutive promoters identified by SELEX screening (B). Evidence for each promoter are as described in S1 Table.(PDF)Click here for additional data file.S3 TableRpoF-dependent promoters listed in RegulonDB.Promoters listed in RegulonDB are classified into those not identified as the constitutive promoters (A) and the constitutive promoters identified by SELEX screening (B). Evidence for each promoter are as described in S1 Table.(PDF)Click here for additional data file.S4 TableRpoS-dependent promoters listed in RegulonDB.Promoters listed in RegulonDB are classified into those not identified as the constitutive promoters (A) and the constitutive promoters identified by SELEX screening (B). Evidence for each promoter are as described in S1 Table.(PDF)Click here for additional data file.

Supporting information

RpoS-dependent promoters listed in RegulonDB.

Promoters listed in RegulonDB are classified into those not identified as the constitutive promoters (A) and the constitutive promoters identified by SELEX screening (B). Evidence for each promoter are as described in RegulonDB (see Table 5): (Group-A) Promoters were experimentally identified by using HTTIM (high-throughput transcription initiation mapping), TIM (transcription initiation mapping), FP (footprinting), or IDA (inferred by direct promoter assay; (Class-B) Promoters were predicted based on AIPP (automated inference of promoter position), ICWHO (inferred computationally without human oversight), HIPP (human inference of promoter position), NTAS (non-traceable author statement), TASES (traceable author statement to experimental support), TAS (traceable author statement), IMP (inferred from mutant) or IEP (inferred from expression pattern).

Click here for additional data file.

RpoH-dependent promoters listed in RegulonDB.

Promoters listed in RegulonDB are classified into those not identified as the constitutive promoters (A) and the constitutive promoters identified by SELEX screening (B). Evidence for each promoter are as described in S1 Table.

Click here for additional data file.

RpoF-dependent promoters listed in RegulonDB.

Promoters listed in RegulonDB are classified into those not identified as the constitutive promoters (A) and the constitutive promoters identified by SELEX screening (B). Evidence for each promoter are as described in S1 Table.

Click here for additional data file.

RpoS-dependent promoters listed in RegulonDB.

Promoters listed in RegulonDB are classified into those not identified as the constitutive promoters (A) and the constitutive promoters identified by SELEX screening (B). Evidence for each promoter are as described in S1 Table.

Click here for additional data file.
