---
pmid: '25843628'
title: Systematic discovery of Xist RNA binding proteins.
authors:
- Chu C
- Zhang QC
- da Rocha ST
- Flynn RA
- Bharadwaj M
- Calabrese JM
- Magnuson T
- Heard E
- Chang HY
journal: Cell
year: '2015'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC4425988
doi: 10.1016/j.cell.2015.03.025
---

# Systematic discovery of Xist RNA binding proteins.
**Authors:** Chu C, Zhang QC, da Rocha ST, Flynn RA, Bharadwaj M, Calabrese JM, Magnuson T, Heard E, Chang HY
**Journal:** Cell (2015)
**DOI:** [10.1016/j.cell.2015.03.025](https://doi.org/10.1016/j.cell.2015.03.025)
**PMC:** [PMC4425988](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4425988/)

## Abstract

1. Cell. 2015 Apr 9;161(2):404-16. doi: 10.1016/j.cell.2015.03.025. Epub 2015 Apr
 2.

Systematic discovery of Xist RNA binding proteins.

Chu C(1), Zhang QC(2), da Rocha ST(3), Flynn RA(2), Bharadwaj M(2), Calabrese 
JM(4), Magnuson T(5), Heard E(3), Chang HY(6).

Author information:
(1)Howard Hughes Medical Institute and Program in Epithelial Biology, Stanford 
University School of Medicine, Stanford, CA 94305, USA; Genome Institute of 
Singapore, 60 Biopolis Street, Singapore 138672, Singapore.
(2)Howard Hughes Medical Institute and Program in Epithelial Biology, Stanford 
University School of Medicine, Stanford, CA 94305, USA.
(3)Institut Curie, CNRS UMR3215, INSERM U934, 26 rue d'Ulm, Paris 75248, France.
(4)Department of Pharmacology and Lineberger Comprehensive Cancer Center, 
University of North Carolina, Chapel Hill, NC 27599, USA.
(5)Department of Genetics and Lineberger Comprehensive Cancer Center, University 
of North Carolina, Chapel Hill, NC 27599, USA.
(6)Howard Hughes Medical Institute and Program in Epithelial Biology, Stanford 
University School of Medicine, Stanford, CA 94305, USA. Electronic address: 
howchang@stanford.edu.

Noncoding RNAs (ncRNAs) function with associated proteins to effect complex 
structural and regulatory outcomes. To reveal the composition and dynamics of 
specific noncoding RNA-protein complexes (RNPs) in vivo, we developed 
comprehensive identification of RNA binding proteins by mass spectrometry 
(ChIRP-MS). ChIRP-MS analysis of four ncRNAs captures key protein interactors, 
including a U1-specific link to the 3' RNA processing machinery. Xist, an 
essential lncRNA for X chromosome inactivation (XCI), interacts with 81 proteins 
from chromatin modification, nuclear matrix, and RNA remodeling pathways. The 
Xist RNA-protein particle assembles in two steps coupled with the transition 
from pluripotency to differentiation. Specific interactors include HnrnpK, which 
participates in Xist-mediated gene silencing and histone modifications but not 
Xist localization, and Drosophila Split ends homolog Spen, which interacts via 
the A-repeat domain of Xist and is required for gene silencing. Thus, Xist 
lncRNA engages with proteins in a modular and developmentally controlled manner 
to coordinate chromatin spreading and silencing.

Copyright © 2015 Elsevier Inc. All rights reserved.

DOI: 10.1016/j.cell.2015.03.025
PMCID: PMC4425988
PMID: 25843628 [Indexed for MEDLINE]

## Full Text

Introduction

Many lncRNAs are recently recognized as functional regulators of gene expression ( Rinn and Chang, 2012 ), but their mechanisms of action are largely unknown. RNA-binding proteins (RBPs) play key roles in lncRNA-mediated gene regulation, and obtaining the full interaction map of proteins bound to a lncRNA of interest is critical to our understanding of its function. Many tools have been developed to describe RNA-protein interaction from a protein-centric view, typically by immunoprecipitating a protein and analyzing the associated RNAs with a microarray or high-throughput sequencing (reviewed by Riley and Steitz, 2013 ). In contrast, fewer methods are available from the perspective of a particular RNA. This is usually achieved by 1) tagging the RNA with affinity-aptamers, which involves complicated genetic engineering; 2) using in-vitro transcribed RNA to retrieve proteins from native cell lysates (RNA chromatography), which is prone to the formation of non-physiological RNA-protein interactions; 3) using immobilized oligonucleotides to capture RNA:protein complex under native conditions, which suffers from both post-lysis re-associations and unpredictable specificity of target RNA retrieval (reviewed by Chu et al., 2015 ). The ideal strategy should capture in vivo lncRNA-protein interactions, achieve high yield and specificity without genetic tagging, and provide comprehensive portraits of lncRNP in diverse biological states.

Xist is a lncRNA (17Kb long in the mouse) required for X-chromosome inactivation (XCI) of one of the two X-chromosomes in female cells, thus enabling dosage compensation between XX females and XY males ( Gendrel and Heard, 2011 ). XCI takes place early in embryonic development, and is thought to occur in multiple steps: counting and choosing the X chromosome to silence, spreading of Xist over the target X chromosome, and silencing of most of its active genes ( Payer and Lee, 2008 ). The latter two steps are believed to be mediated by specific Xist-associated protein factors, which remain largely mysterious. Xist expression marks the future inactive X chromosome (Xi) and is sufficient to recruit silencing chromatin modifications complex such as the Polycomb proteins ( Gendrel and Heard, 2011 ). It has been debated whether Xist RNA physically recruits one or more silencing factors, or whether Xist indirectly promotes transcriptional silencing via reinforcement of repressive chromatin. XCI is also developmentally regulated in several important ways. In the mouse, XCI can proceed by random inactivation of either paternal or maternal chromosome in somatic cells, or by always inactivating the paternally derived X in extra-embryonic cells, a process called imprinted XCI ( Takagi and Sasaki, 1975 ). During random XCI, Xist is not expressed in pluripotent embryonic stem cells (ESCs), and is up-regulated during differentiation ( Wutz and Jaenisch, 2000 ). Ectopic Xist RNA–coating can induce gene silencing in ESCs, although this is reversible during an early differentiation time window, becoming irreversible at later stages ( Wutz and Jaenisch, 2000 ). Knowledge of the Xist lncRNP in these diverse states may provide insights into this classic and intricate epigenetic system.

Here we introduce c ompre h ensive i dentification of R NA-binding p roteins by m ass s pectrometry (ChIRP-MS), an optimized method for the identification of lncRNA-bound proteome. Applying ChIRP-MS to three noncoding RNAs, we found known and validated novel functional interactors. By performing Xist ChIRP-MS in different cell states, lineages and cell types, and with mutant Xist alleles, we uncover mechanisms of dynamic and coordinate assembly of Xist binding partners, suggesting an organizing principle for lncRNPs.

ChIRP-MS method

Extending on ChIRP-seq, a method using DNA oligonucleotides to capture lncRNAs and their genomic DNA binding sites ( Chu et al., 2011 ), we optimized ChIRP-MS to identify lncRNA-associated proteins ( Figure 1A ). We crosslink cells extensively with formaldehyde, retrieve target RNA with oligonucleotide hybridization, and use a gentle biotin-elution to liberate associated proteins. The enriched proteins were identified by liquid chromatography-tandem mass spectrometry (LC-MS/MS). We conducted negative controls by use of non-interacting control probes, RNase treatment of chromatin prior to ChIRP, or genetic removal of the target RNA.

As a proof-of-principle, we performed ChIRP-MS of human U1 and U2 snRNAs in HeLa S3 cells. The snRNAs are ideal for validating ChIRP-MS because they are abundant (∼1 million copies of U1 per cell)( Gesteland and Atkins, 1993 ) and the spliceosome composition is well known ( Pomeranz Krummel et al., 2009 ; Stark et al., 2001 ; Zhou et al., 2002 ). Furthermore, non-canonical roles of U1 in preventing premature mRNA cleavage and polyadenylation have been recently reported ( Almada et al., 2013 ; Berg et al., 2012 ; Kaida et al., 2010 ), implying potential novel interactors that ChIRP-MS may discover. We designed antisense DNA oligonucleotides targeting U1 and U2 snRNAs respectively in regions previously found to be accessible for morpholino binding, and as a negative control, we chose a non-targeting probe that does not bind any human RNA ( Berg et al., 2012 ; Kaida et al., 2010 ). While the input RNA spread over a large size range (due to shearing by sonication) with distinct tRNA peaks, after ChIRP-enrichment the two snRNAs predominated ( Figure 1B ). U1 probe retrieved the known direct binding protein U1A, while the control probe did not. U2 probe also enriched for U1A, although the indirect interaction resulted in reduced enrichment. U2 probe also retrieved known U2-binding protein U2B, which crossreacts with U1A antibody due to their close homology (arrow, Figure 1C ). ChIRP of U3, an abundant small nucleolar RNA not involved in splicing, specifically retrieved the nucleolar protein fibrillarin but not U1A ( Figure 1C ). Beta-Actin (ACTB) was not enriched by any probe, serving as another negative control. These results indicate that ChIRP is specific even for very abundant RBPs.
