---
pmid: '23698369'
title: BAF complexes facilitate decatenation of DNA by topoisomerase IIα.
authors:
- Dykhuizen EC
- Hargreaves DC
- Miller EL
- Cui K
- Korshunov A
- Kool M
- Pfister S
- Cho YJ
- Zhao K
- Crabtree GR
journal: Nature
year: '2013'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC3668793
doi: 10.1038/nature12146
---

# BAF complexes facilitate decatenation of DNA by topoisomerase IIα.
**Authors:** Dykhuizen EC, Hargreaves DC, Miller EL, Cui K, Korshunov A, Kool M, Pfister S, Cho YJ, Zhao K, Crabtree GR
**Journal:** Nature (2013)
**DOI:** [10.1038/nature12146](https://doi.org/10.1038/nature12146)
**PMC:** [PMC3668793](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3668793/)

## Abstract

1. Nature. 2013 May 30;497(7451):624-7. doi: 10.1038/nature12146. Epub 2013 May
22.

BAF complexes facilitate decatenation of DNA by topoisomerase IIα.

Dykhuizen EC(1), Hargreaves DC, Miller EL, Cui K, Korshunov A, Kool M, Pfister 
S, Cho YJ, Zhao K, Crabtree GR.

Author information:
(1)Howard Hughes Medical Institute, Stanford University School of Medicine, 
Stanford, California 94305, USA.

Recent exon-sequencing studies of human tumours have revealed that subunits of 
BAF (mammalian SWI/SNF) complexes are mutated in more than 20% of all human 
malignancies, but the mechanisms involved in tumour suppression are unclear. BAF 
chromatin-remodelling complexes are polymorphic assemblies that use energy 
provided by ATP hydrolysis to regulate transcription through the control of 
chromatin structure and the placement of Polycomb repressive complex 2 (PRC2) 
across the genome. Several proteins dedicated to this multisubunit complex, 
including BRG1 (also known as SMARCA4) and BAF250a (also known as ARID1A), are 
mutated at frequencies similar to those of recognized tumour suppressors. In 
particular, the core ATPase BRG1 is mutated in 5-10% of childhood 
medulloblastomas and more than 15% of Burkitt's lymphomas. Here we show a 
previously unknown function of BAF complexes in decatenating newly replicated 
sister chromatids, a requirement for proper chromosome segregation during 
mitosis. We find that deletion of Brg1 in mouse cells, as well as the expression 
of BRG1 point mutants identified in human tumours, leads to anaphase bridge 
formation (in which sister chromatids are linked by catenated strands of DNA) 
and a G2/M-phase block characteristic of the decatenation checkpoint. Endogenous 
BAF complexes interact directly with endogenous topoisomerase IIα (TOP2A) 
through BAF250a and are required for the binding of TOP2A to approximately 
12,000 sites across the genome. Our results demonstrate that TOP2A chromatin 
binding is dependent on the ATPase activity of BRG1, which is compromised in 
oncogenic BRG1 mutants. These studies indicate that the ability of TOP2A to 
prevent DNA entanglement at mitosis requires BAF complexes and suggest that this 
activity contributes to the role of BAF subunits as tumour suppressors.

DOI: 10.1038/nature12146
PMCID: PMC3668793
PMID: 23698369 [Indexed for MEDLINE]

Conflict of interest statement: The authors declare no competing interests

## Full Text

Using Brg1flox/flox;actin-CreER embryonic stem (ES) cells (Brgf/f), we observed that tamoxifen-induced deletion of Brg1 (Brgf/fERΔ) results in the appearance of DNA bridges during anaphase (Fig. 1a). This phenotype is characteristic of a deficiency in TopoIIα function, which normally resolves DNA catenanes that develop during transcription and replication12. We determined the frequency of anaphase bridges in Brgf/fERΔ cells to be similar to that of cells deficient in other putative tumor suppressors that regulate TopoIIα function, including BRCA1, RanBP2, and RECQL513-15 (Fig. 1a).

In previous studies, we recovered peptides from TopoIIα in mass spectrometric analysis of endogenous BAF complexes16. Immunoprecipitation (IP) of BAF complexes with antibodies to Brg1 recovered TopoIIα and, conversely, IP of TopoIIα revealed Brg1 (Fig. 1b). This association is not dependent on DNA (Supplementary Fig.1a, b). We detected this association in several additional cell types, including mouse embryonic fibroblasts (MEFs) and HEK293Ts (Supplementary Fig. 1c).

Failure of TopoIIα to resolve catenated DNA leads to slow progression through the G2/M phase of the cell cycle17. To better understand the mitotic defect in Brg1-deficient cells, we synchronized Brgf/f and Brgf/fERΔ cells in G1/S using double-thymidine block. Following release, Brgf/f and Brgf/fERΔ cells transited through the cell cycle at the same rate until reaching G2/M, where the Brgf/fERΔ cells exhibited a significant delay (Fig. 1c). In asynchronously dividing cells, this delay resulted in a 1.5- to 2-fold increase in Brgf/fERΔ cells in G2/M (Fig. 1d), similar to the treatment of cells with the TopoII inhibitor ICRF-19318. Caffeine, an inhibitor of ATM/ATR, forces cells through an ICRF-193-induced decatenation checkpoint18 and similarly overrides the G2/M arrest in Brgf/fERΔ cells (Fig. 1d). Furthermore, expression of TopoIIαS1524A, which fails to recruit MDC1 to chromatin upon initiation of the decatenation checkpoint19, alleviated the cell cycle delay, suggesting that Brgf/fERΔ cells arrest due to activation of the decatenation checkpoint (Fig. 1e, supplementary Fig. 1d). Finally, chromosomes from Brgf/fERΔ cells are significantly longer than chromosomes from Brgf/f cells (Fig. 1f, supplementary Fig. 1e),a defect observed in TopoIIα-deficient cells due to its role in chromosome condensation12,20. These data indicate that Brg1 deletion resembles the mitotic defects observed with chemical inhibition and/or knockdown of TopoIIα12,17,18,20.

We investigated oncogenic point mutants of Brg1 found in medulloblastoma and Burkitt's lymphoma (Brg1G1232D (BrgGD) and Brg1T910M (BrgTM)6-11) by expressing FLAG-tagged versions in Brgf/f cells (Fig. 2a). The Brg1 mutants were incorporated into the BAF complex normally and did not alter the composition of the complex (supplementary Fig 2a). Although T910M is located in the ATP binding pocket of Brg1, the G1232D mutation is downstream of HELICc domain and thus not obviously involved in ATP turnover (Fig. 2a). Subjecting BAF complexes containing BrgGD, BrgTM, BrgWT, or Brg1K798R (BrgKR)21, the ATPase-dead point mutant of Brg1, to an assay for ATPase activity revealed that both cancer mutants are significantly compromised in ATPase activity, though not as profoundly as BrgKR (Fig. 2b). BrgTM is more severely compromised than BrgGD, which correlated with the viability of the respective cell line (supplementary Fig 2b).

Cells expressing BrgGD and BrgTM, but not BrgWT, display increases in the percentage of G2/M cells and the incidence of anaphase bridges similar to that of Brgf/fERΔ cells (Fig. 2c, d). Importantly, expression of the mutants in the presence of endogenous Brg1 gives similar increases, although less severe, in G2/M percentage and anaphase bridge incidence compared to vector cells (Fig. 2c, d). The dominant nature of these mutants on cell cycle and anaphase bridge formation suggests that medulloblastomas with both heterozygous and homozygous mutations in Brg1 have these mitotic defects.

To explore whether the increase in anaphase bridges contributes to increased chromosome instability as it does in TopoIIα-deficient cells12,17, we analyzed ploidy in the Brg1 mutant cell lines. Expression of BrgGD or BrgTM results in a significant increase in cells with >4N DNA content in both ethanol- and tamoxifen-treated cells (Fig. 2c, supplementary Fig 2c). We also observed a significant increase in the number of BrgGD and BrgTM expressing cells with abnormal chromosome number in metaphase spreads from both ethanol- and tamoxifen-treated samples (Fig. 2e). These data suggest that G1232D and T910M mutations in Brg1 can contribute to chromosome instability as a result of deficiencies in TopoIIα function.

We collected several BRG1 mutant medulloblastomas to determine whether the effects of TopoIIα deficiency can be observed in primary tumors. We observed an increased proportion of anaphase bridges in each of five histologic samples from BRG1 mutant tumors relative to controls, suggesting these tumors have decatenation defects (Fig. 2f, supplementary Fig 2d). Aneuploidy is common in medulloblastoma and ranges from the partial gain or loss of single chromosomes to full tetraploidy8,22,23. However, a recent study showed that the relative rate of tetraploidy of 5 BRG1 mutant tumors was similar to that of 15 BRG1 WT tumors8. Additional sample characterization will be necessary to definitively assess whether BRG1 mutation causes mitotic defects through insufficient TopoIIα function in medulloblastoma.

Microarray analysis of Brgf/f and Brgf/fERΔ ES cells indicated that Brg1-dependent genes are not enriched for GO terms related to DNA damage or repair24. Additionally, we found no alterations in the abundance, post-translational modifications, cellular localization, or in vitro enzymatic activity of TopoIIα in Brgf/fERΔ cells (supplementary Fig. 3a-f, Supplementary 4). To test whether purified BAF complexes could enhance the enzymatic activity of recombinant TopoIIα, we used the standard in vitro kinetoplast DNA-based decatenation assay. Immobilized BAF complexes increased TopoIIα's enzymatic activity (Supplementary Fig 5a); however, Brg1 mutant BAF complexes also enhanced TopoIIα's activity, as did PRC2, indicating a nonspecific activity on a bare DNA template25 that does not reflect our in vivo observations. The Brg1 mutants did however reduce TopoIIα's association with chromatin, such that more TopoIIα remained associated with chromatin after high salt wash in BrgWT cells than in BrgTM, BrgGD, and vector cells (Fig. 3a, Supplementary Fig 5b, c). Reduced binding of TopoIIα to chromatin would be expected to compromise TopoIIα function and could represent an inability of TopoIIα to associate with substrate DNA during decatenation.

To identify defined regions of TopoIIα binding across the genome, we performed a TopoIIα ChIP-seq in Brgf/f and Brgf/fERΔ cells. We recovered very few peaks using traditional ChIP methods, so we employed etoposide, a small molecule that freezes TopoIIα in a covalent complex with DNA during the enzymatic process, thereby identifying sites of active TopoIIα cleavage26. We recovered 16591 TopoIIα peaks in Brgf/f cells and 4623 TopoIIα peaks in Brgf/fERΔ cells, demonstrating the contribution of Brg1 to TopoIIα binding (Fig. 3b). Almost two thirds of the TopoIIα Brgf/f peaks are DNase I hypersensitive, consistent with TopoIIα's preference for nucleosome-free DNA27. An example reflecting these trends is shown in Figure 3c. We confirmed TopoIIα binding by ChIP-qPCR at 14 Brg1-dependent and 10 Brg1-independent sites in Brgf/f and Brgf/fERΔ cells (Fig. 3d). Additionally, we determined that TopoIIα binding is mitigated in BrgTM and BrgGD mutant Brgf/fERΔ cells at Brg1-dependent sites (Fig. 3e). This is not the result of reduced binding of the Brg1 mutants to chromatin, as BrgTM and BrgGD bind similarly to BrgWT at these sites (Fig. 3f). Given that the BrgTM and BrgGD mutants display reduced ATPase activity, these data implicate a role for the ATP-dependent accessibility activity of BAF complexes in TopoIIα binding and function across the genome, a function previously identified for yeast Snf5 in transcription28.

Due to the dedicated nature of subunits within BAF complexes, TopoIIα could be interacting with any BAF subunit. Indeed, we precipitated TopoIIα with antibodies to several dedicated subunits as determined by glycerol gradient centrifugation analysis (Fig. 4a, Supplementary Fig 6a). Quantitation of the precipitated TopoIIα revealed that little TopoIIα was recovered after IP with antibodies raised against BAF250a (aa1236-1325) and BAF250b (aa1300-1350), while other antibodies immunoprecipitated TopoIIα well (Fig 4a). We reasoned that the BAF250a/b antibody might disrupt the interaction between TopoIIα and the BAF complex if TopoIIα bound directly to BAF250a/b. Indeed, TopoIIα associated with full-length BAF250a and BAF250a (aa1-1758), but not BAF250a (aa1759-2285) in a heterologous expression system (Fig. 4b). This interaction is independent of Brg1 because we were unable to detect Brg1 in co-precipitates of BAF250a (aa1-1758) and TopoIIα. Furthermore, the association between TopoIIα and Brg1 was lost upon knockdown of BAF250a, with the most severe knockdown resulting in the most severe loss of association (Fig. 4c, Supplementary Fig 6b). To determine whether the interaction between TopoIIα and BAF250a was physiologically relevant, we knocked down BAF250a in MEFs and observed frequencies of anaphase bridges and G2/M delay similar to knockdown of Brg1 or TopoIIα (Fig. 4d, e, Supplementary Fig. 6c, d). These data indicate that TopoIIα associates with Brg1 via a direct interaction with BAF250a.

Our studies point to a new role for ATP-dependent chromatin remodeling in decatenating DNA. Reduced decatenation in vivo is revealed by the frequency of anaphase bridges and an increase in the number of cells in G2/M upon deletion of Brg1 or expression of the tumor-associated T910M and G1232D Brg1 mutants (Fig. 1a, d, Fig. 2c, d). Although mitotic defects have been noted in cells lacking Brg1, the cause of these defects was unclear29. In addition to Brg1, loss of BAF250a also results in decatenation defects (Fig. 4d, e), which could reflect the high incidence of mutations in BRG1 and BAF250A (ARID1A) in human tumors1,23. Our in vivo observations are reinforced by the requirement of BAF for TopoIIα binding at DNase I hypersensitive Brg1-binding sites (Fig. 3b-e).

The dependence of TopoIIα on BAF function offers a possible explanation for the frequency with which BAF subunit mutations are detected in screens for driving mutations in human cancers. Anaphase bridges are often forcibly severed during cytokinesis30, resulting in partial or complete chromosome gains or losses as well as polyploidy if the cell fails to undergo mitosis12,17. At present, the number of BRG1 mutant medulloblastomas analyzed for ploidy status is insufficient to determine whether BRG1 mutation results in aneuploidy in human tumors. In the case of medulloblastoma, mutations in Brg1 are often accompanied by activating mutations in the WNT signaling pathways and/or MYC amplification23. Further studies highlighting these pairings will help define the contribution of reduced TopoIIα function as a result of BRG1 mutation to tumorigenesis.

MethodsBrg1 deletion from Brgf/fCreER ES cells and MEFs was performed as previously described4. Lentiviruses were produced in HEK293T cells using PEI transfection. Cells were synchronized using double thymidine block. Cell cycle analysis was performed according to manufacturer instructions (BD Biosciences). TopoIIα ChIP-seq was performed following etoposide fixation26. Real-time PCR, immunofluorescence, immunoprecipitation, and western blotting were done using standard protocols. The chromatin fraction from nuclei in varying concentrations of NaCl was analyzed by western blot.ImmunofluorescenceTo quantitate anaphase bridges, cells were fixed with 4% paraformaldehyde for 20 minutes, washed, and stained with DAPI (Sigma). The number of anaphases/telophases with bridges over the total number of anaphases (between 56-187 total anaphases per 25mm slide) was recorded from each slide for more than four independent experiments. Brgf/f and Brgf/fERΔ ES cells were visualized with DAPI 72 hours after tamoxifen treatment. WT MEFs infected with lentiviruses containing hairpins to Brg1, BAF250a or TopoIIα and analyzed 48-96 hours after infection.To stain for TopoIIα and centromeres/microtubules, cells were blocked with 5% BSA/1% goat serum in PBST for 1 hour following fixation and incubated with anticentromere (Antibodies Inc 15-235) or anti-γtubulin (Sigma T5326) and anti-TopoIIα (Santa Cruz sc-365916) for 2 hours. Following several washes, anti-human AlexFluor488 and anti-rabbit AlexaFluor568 were added for 1 hour. The cells were then stained with DAPI for 10 minutes and washed 3×PBS for 10 minutes each. The coverslips were mounted on slides with Vectashield Hardmount (Vector Labs, Burlingame, CA).To quantitate anaphase bridges from paraffin-embedded human tumor samples, slides were incubated 2×25 minutes in xylenes, then rehydrated in 100% EtOH, then 95% EtOH, then water for 2 minutes each. The slides were boiled in Citrate Buffer (pH 6.0) (Vector Labs, Burlingame, CA) for 20 minutes and washed 2×5 minutes in PBS-Tween. The slides were then stained with DAPI for 10 minutes and washed 3×5 minutes with PBS before mounting with Vectashield Hardmount.Cell synchronizationES cells were incubated with 2mM thymidine for 7-8 hours, released into fresh media for 7 hours, and then incubated with thymidine again for 7 hours. The cells were washed several times with PBS, released into fresh media, and harvested at time points thereafter.Cell Cycle analysisThe cell cycle analysis was performed using BD Biosciences BrdU-FITC FACS kit. ES cells were incubated with BrdU for 1 hour and MEFs were incubated with BrdU for 4 hours. Brgf/f and Brgf/fERΔ ES cells were analyzed 72 hours after tamoxifen treatment. Caffeine was added to media 2 hours before BrdU incubation. To determine the percent of cells in G2/M, DNA was stained with 7AAD and analyzed by FACS.H3(S10)P cell cycle analysisBrgf/f ES cells were infected with RNAi-resistant wild-type hTopoIIα or hTopoIIαS1524A and shRNAs to mouse TopoIIα. Cells were stained with anti-H3(S10)P and analyzed by flow cytometry 72 hours after treatment with or without tamoxifen.Metaphase Spread PreparationMEFs were grown to 85% confluence and incubated for 4 hours with colcemid. Cells were harvested and swelled by dropwise addition of 1:1 0.4% KCl/0.4% Sodium Citrate for 7 minutes at 37°C. Cells were then fixed by dropwise addition of 3:1 MeOH/Acetic Acid for 20 minutes, spun down, and fixed for another 30 minutes. Metaphases were dropped onto slides, dried on wet paper towels and stained with DAPI for visualization. Chromosomes were then measured and counted using ImageJ software. To analyze polyloidy, only cells with greater than 35 chromosomes were counted to eliminate artifacts due to partial spreads.Gene Expression Profiling and AnalysisRNA was isolated using TRIzol (Invitrogen) and reverse transcribed into cDNA using SuperScript III reverse transcriptase (Invitrogen). Real-time PCR was performed on the StepOnePlus (ABI) machine using FastStart Universal SYBR Green Master with ROX (Roche).ImmunoprecipitationNuclei were isolated from cells with Buffer A (25 mM Hepes, pH7.6, 5 mM MgCl2, 25 mM KCl, 0.05 mM EDTA, 10% glycerol, 0.1% NP-40) and lysed for 30 min in IP buffer (50 mM Tris-HCl, pH 8.0, 150 mM NaCl, 0.1% NP-40). The chromatin was removed using centrifugation and the lysates were precleared with 20 μL protein A or protein G dynabeads for 30 min. The protein concentration was quantitated using the BCA assay and adjusted to a final volume of 200 μL at a final concentration of 1.5 mg/mL with IP buffer. Each IP was incubated with 3 μg of anti-Brg1 (Santa Cruz sc-17796), anti-TopoIIα (Abcam ab52934), anti-BAF45d (Crabtree Lab), anti-BAF47 (Santa Cruz sc-166165), anti-BAF57 (Bethyl A300-810A), anti-BAF155 (Crabtree Lab), anti-BAF60a (BD Transduction Laboratories 611728), anti-BAF250a (Santa Cruz sc-32761x, Santa Cruz sc-98441X), anti-BAF180 (Bethyl A301-590A), anti-BAF250b (Santa Cruz sc-32762X, Bethyl A301-046A), anti-SS18 (Santa Cruz sc-28698) BAF200 (Santa Cruz sc-98299X) or anti-IgG (Santa Cruz sc-2025) overnight at 4°C and then for 2h with 20 μL protein A/G dynabeads. The beads were washed four times with 1 mL IP buffer and resuspended in 10 μL gel loading buffer (4× LDS Buffer; Invitrogen).Glycerol Gradient Centrifugation AnalysisES cells were lysed in Buffer A (10 mM Hepes (pH 7.6), 25 mM KCl, 1 mM EDTA, 10% glycerol, 1 mM DTT, and protease inhibitors (complete mini tablets (Roche) supplemented with 1 mM PMSF) on ice. Nuclei were sedimented by centrifugation (1,000×g), resuspended in Buffer C (10 mM Hepes (pH 7.6), 3 mM MgCl2, 100 mM KCl, 0.1 mM EDTA, 10% glycerol, 1 mM DTT, and protease inhibitors), and lysed by the addition of ammonium sulfate to a final concentration of 0.3 M. Soluble nuclear proteins were separated by insoluble chromatin fraction by ultracentrifugation (100,000×g) and precipitated with 0.3 mg/ml ammonium sulfate for 20 min on ice. Protein precipitate was isolated by ultracentrifugation (100,000×g), and resuspended in HEMG-0 buffer (25mM HEPES (pH 7.9), 0.1mM EDTA, 12.5mM MgCl2, 100mM KCl) for glycerol gradient analyses. 800 ug of protein was overlaid on to a 10ml 10-30% glycerol (in HEMG buffer) gradient prepared in a 14×89 mm polyallomer centrifuge tube (Beckman, part# 331327). Tubes were placed in a SW-40 swing bucket rotor and centrifuged at 4 degrees for 16 hours at 40,000 RPM. 0.5 ml fractions were harvested and used in gel electrophoresis and subsequent western blotting analyses.Western BlotsNuclei were isolated from cells with Buffer A (25 mM Hepes, pH 7.6, 5 mM MgCl2, 25 mM KCl, 0.05 mM EDTA, 10% glycerol, 0.1% NP-40) and lysed for 30 min in RIPA buffer (10mM Tris pH 7.4, 150mM NaCl, 0.1% SDS, 0.5% sodium deoxycholate, 1% Triton X-100, 1mM EDTA). Chromatin was either spun out or samples were sonicated prior to BCA analysis. Equal amounts of protein were boiled in gel loading buffer and loaded onto 4-10% BisTris NuPage gels. After transfer, blots were blocked in 5% BSA and incubated with anti-Brg, anti-TopoIIα, anti-pTyr (Millipore clone 4g10 05-321), anti-pSer (Millipore clone 4A4 05-1000), or anti-Ubiquitin (Santa Cruz clone P4D1 sc-8017). Proteins were detected using the LICOR detection system or ECL/autoradiography for SUMO-TopoIIα.Chromatin Association AssayNuclei were isolated using Buffer A and re-suspended in 20 mM Tris-HCl pH 7.6, 3 mM EDTA at 60 mill cells/mL. Samples of 25 μL were alloquotted into tubes and NaCl concentrations were adjusted to a final volume of 50 μL. Samples were gently mixed and incubated on ice for 20 min and centrifuged at high speed for 20 min to isolate chromatin. The lysate was removed and the chromatin pellet was re-suspended in 120 μL gel loading dye. The pellet was solubilized using sonication and the association of TopoIIα to chromatin was analyzed using western blotting.ATPase AssayATPase assay was adapted from the literature31. Immunoprecipitations were performed as described above with anti-Brg antibody. IPs were washed a final time with 10 mM TrisHCl pH 7.5, 50 mM NaCl, 5 mM MgCl2, 1 mM DTT and resuspended in 20 μL assay buffer (10 mM Tris-HCl pH 7.5, 50 mM NaCl, 5mM MgCl2, 20% glycerol, 1 mg/ml BSA, 0.5 mM ATP, 20 nM plasmid DNA, 50 μCi/mL γ-32P ATP, 1 mM DTT, protease inhibitors). The reaction was agitated at 37 °C for 1h (or when approximately 50% of ATP was converted to inorganic phosphate). Reaction mixture (0.5 uL) was spotted onto PEI cellulose plates and thin layer chromatography was performed in 0.5M LiCl, 1M formic acid. The plates were dried and imaged using phosphorimaging. The enzymatic activity was quantitated as a ratio of product (32P-Pi) to starting material (γ-32P ATP). Values were normalized to the activity of BrgWT (100%) and vector control (0%) cellsChromatin ImmunoprecipitationFor the Brg1 ChIP, 40 mill ES cells were fixed for 12 minutes in 1% formaldehyde at room temperature. Nuclei were sonicated in 1 mL ChIP Lysis Buffer (50 mM HEPES pH 7.5, 150 mM NaCl, 2 mM EDTA, 1% Triton X-100, 0.1% SDS) to yield fragments between 200-500 bp. 500 μl of lysate was incubated with 5 μg of anti-Brg1 (Crabtree Lab) or 5 μg anti-rabbit IgG and rotated overnight at 4° C and then for 2h with 20 μl Protein A/G Dynabeads. After five washes with ChIP Lysis Buffer and one wash in TE, DNA was eluted by boiling in 10% Chelex slurry.The etoposide ChIP of TopoIIα was adapted from the literature26. Specifically, 20 million ES cells were treated with 100 μM etoposide for 10 minutes. Cells were washed once with PBS and lysed with 1 ml of a buffer containing 1% Sarkosyl, 10 mM Tris-HCl (pH 7.5), 10 mM EDTA, and protease inhibitor. A solution of 7 M CsCl (7 M) was added to a final concentration of 0.5 M and the lysate was sonicated to yield fragments between 200-500 bp. ChIP buffer (300 μL) was added to 300 μl of lysate for a final concentration of 50 mM HEPES pH 7.5, 300 mM NaCl, 1 mM EDTA, 1 % Triton X-100, 0.1 % DOC, and 0.1% SDS and 3 μg Anti-TopoIIα (sc-365916) prebound to 20 μl Protein G Dynabeads was added. The lysate was rotated overnight at 4° C and washed four times with ChIP lysis buffer, one time with LiCl buffer (10 mM Tris pH 8.0, 0.25 M LiCl, 0.5 % NP-40, 0.5% DOC, 1 mM EDTA) and one time with TE. The DNA was eluted with 300 μl of 1% SDS, 0.1 M NaHCO3 for 20 minutes and removed from the beads. The solution was adjusted to 200mM NaCl, 10mM EDTA, 40mM Tris pH 6.5 and 0.2 mg/mL RNase A was added for 30 min at 37° C. Proteinase K was added to 0.03 mg/ml and digested overnight at 55° C. The DNA was extracted with phenol/chloroform and precipitated with ethanol for analysis by qPCR. Primers used for ChIP-qPCR are available upon request.ChIP-seq and AnalysisThe library preparation and sequencing was performed as previously described32. Raw ChIP-seq reads were mapped to the Mus musculus genome (build mm9/NCBI37) using the short-read aligner Bowtie (version 0.12.7)33. Peaks were then called using Model-base Analysis of ChIP-seq (MACS) (version 1.4.1)34. Further analysis was aided by the Bedtools suite (version 2.16.2) 35. Genome annotations were acquired from the UCSC Genome Browser (http://genome.ucsc.edu/)36,37. We also uploaded our data to the genome browser, which was used to produce screenshots of chromatin binding/modification profiles at individual loci.Topoisomerase Activity AssayReactions contain: 150 ng kinetoplast DNA (Topogen), 50 mM Tris-HCl, pH 8.0, 150 mM NaCl, 10 mM MgCl2, 2 mM ATP, a standard TopoIIα IP or varying amounts of recombinant TopoIIα (Topogen).Lentiviral Infection293XTs were transfected with lentiviruses containing vector alone, wild-type Brg1, Brg1 point mutants, wild-type hTopoIIα, or hTopoIIαS1524A or with vectors containing hairpins to brg, (TRCN0000071386), arid1a (TRCN0000071395, Origene TG517733), or top2a (V2LMM_11295). 48 hours later, supernatants were collected and centrifuged at 20,000 rpm for 2 hours. Viral pellets were resuspended in PBS and used to infect ES cells in suspension or MEFs by spinfection. Cells were selected with puromycin and collected 48-96 hours postinfection for analysis.

Brg1 deletion from Brgf/fCreER ES cells and MEFs was performed as previously described4. Lentiviruses were produced in HEK293T cells using PEI transfection. Cells were synchronized using double thymidine block. Cell cycle analysis was performed according to manufacturer instructions (BD Biosciences). TopoIIα ChIP-seq was performed following etoposide fixation26. Real-time PCR, immunofluorescence, immunoprecipitation, and western blotting were done using standard protocols. The chromatin fraction from nuclei in varying concentrations of NaCl was analyzed by western blot.

ImmunofluorescenceTo quantitate anaphase bridges, cells were fixed with 4% paraformaldehyde for 20 minutes, washed, and stained with DAPI (Sigma). The number of anaphases/telophases with bridges over the total number of anaphases (between 56-187 total anaphases per 25mm slide) was recorded from each slide for more than four independent experiments. Brgf/f and Brgf/fERΔ ES cells were visualized with DAPI 72 hours after tamoxifen treatment. WT MEFs infected with lentiviruses containing hairpins to Brg1, BAF250a or TopoIIα and analyzed 48-96 hours after infection.To stain for TopoIIα and centromeres/microtubules, cells were blocked with 5% BSA/1% goat serum in PBST for 1 hour following fixation and incubated with anticentromere (Antibodies Inc 15-235) or anti-γtubulin (Sigma T5326) and anti-TopoIIα (Santa Cruz sc-365916) for 2 hours. Following several washes, anti-human AlexFluor488 and anti-rabbit AlexaFluor568 were added for 1 hour. The cells were then stained with DAPI for 10 minutes and washed 3×PBS for 10 minutes each. The coverslips were mounted on slides with Vectashield Hardmount (Vector Labs, Burlingame, CA).To quantitate anaphase bridges from paraffin-embedded human tumor samples, slides were incubated 2×25 minutes in xylenes, then rehydrated in 100% EtOH, then 95% EtOH, then water for 2 minutes each. The slides were boiled in Citrate Buffer (pH 6.0) (Vector Labs, Burlingame, CA) for 20 minutes and washed 2×5 minutes in PBS-Tween. The slides were then stained with DAPI for 10 minutes and washed 3×5 minutes with PBS before mounting with Vectashield Hardmount.

Immunofluorescence

To quantitate anaphase bridges, cells were fixed with 4% paraformaldehyde for 20 minutes, washed, and stained with DAPI (Sigma). The number of anaphases/telophases with bridges over the total number of anaphases (between 56-187 total anaphases per 25mm slide) was recorded from each slide for more than four independent experiments. Brgf/f and Brgf/fERΔ ES cells were visualized with DAPI 72 hours after tamoxifen treatment. WT MEFs infected with lentiviruses containing hairpins to Brg1, BAF250a or TopoIIα and analyzed 48-96 hours after infection.

To stain for TopoIIα and centromeres/microtubules, cells were blocked with 5% BSA/1% goat serum in PBST for 1 hour following fixation and incubated with anticentromere (Antibodies Inc 15-235) or anti-γtubulin (Sigma T5326) and anti-TopoIIα (Santa Cruz sc-365916) for 2 hours. Following several washes, anti-human AlexFluor488 and anti-rabbit AlexaFluor568 were added for 1 hour. The cells were then stained with DAPI for 10 minutes and washed 3×PBS for 10 minutes each. The coverslips were mounted on slides with Vectashield Hardmount (Vector Labs, Burlingame, CA).

To quantitate anaphase bridges from paraffin-embedded human tumor samples, slides were incubated 2×25 minutes in xylenes, then rehydrated in 100% EtOH, then 95% EtOH, then water for 2 minutes each. The slides were boiled in Citrate Buffer (pH 6.0) (Vector Labs, Burlingame, CA) for 20 minutes and washed 2×5 minutes in PBS-Tween. The slides were then stained with DAPI for 10 minutes and washed 3×5 minutes with PBS before mounting with Vectashield Hardmount.

Cell synchronizationES cells were incubated with 2mM thymidine for 7-8 hours, released into fresh media for 7 hours, and then incubated with thymidine again for 7 hours. The cells were washed several times with PBS, released into fresh media, and harvested at time points thereafter.

Cell synchronization

ES cells were incubated with 2mM thymidine for 7-8 hours, released into fresh media for 7 hours, and then incubated with thymidine again for 7 hours. The cells were washed several times with PBS, released into fresh media, and harvested at time points thereafter.

Cell Cycle analysisThe cell cycle analysis was performed using BD Biosciences BrdU-FITC FACS kit. ES cells were incubated with BrdU for 1 hour and MEFs were incubated with BrdU for 4 hours. Brgf/f and Brgf/fERΔ ES cells were analyzed 72 hours after tamoxifen treatment. Caffeine was added to media 2 hours before BrdU incubation. To determine the percent of cells in G2/M, DNA was stained with 7AAD and analyzed by FACS.

Cell Cycle analysis

The cell cycle analysis was performed using BD Biosciences BrdU-FITC FACS kit. ES cells were incubated with BrdU for 1 hour and MEFs were incubated with BrdU for 4 hours. Brgf/f and Brgf/fERΔ ES cells were analyzed 72 hours after tamoxifen treatment. Caffeine was added to media 2 hours before BrdU incubation. To determine the percent of cells in G2/M, DNA was stained with 7AAD and analyzed by FACS.

H3(S10)P cell cycle analysisBrgf/f ES cells were infected with RNAi-resistant wild-type hTopoIIα or hTopoIIαS1524A and shRNAs to mouse TopoIIα. Cells were stained with anti-H3(S10)P and analyzed by flow cytometry 72 hours after treatment with or without tamoxifen.

H3(S10)P cell cycle analysis

Brgf/f ES cells were infected with RNAi-resistant wild-type hTopoIIα or hTopoIIαS1524A and shRNAs to mouse TopoIIα. Cells were stained with anti-H3(S10)P and analyzed by flow cytometry 72 hours after treatment with or without tamoxifen.

Metaphase Spread PreparationMEFs were grown to 85% confluence and incubated for 4 hours with colcemid. Cells were harvested and swelled by dropwise addition of 1:1 0.4% KCl/0.4% Sodium Citrate for 7 minutes at 37°C. Cells were then fixed by dropwise addition of 3:1 MeOH/Acetic Acid for 20 minutes, spun down, and fixed for another 30 minutes. Metaphases were dropped onto slides, dried on wet paper towels and stained with DAPI for visualization. Chromosomes were then measured and counted using ImageJ software. To analyze polyloidy, only cells with greater than 35 chromosomes were counted to eliminate artifacts due to partial spreads.

Metaphase Spread Preparation

MEFs were grown to 85% confluence and incubated for 4 hours with colcemid. Cells were harvested and swelled by dropwise addition of 1:1 0.4% KCl/0.4% Sodium Citrate for 7 minutes at 37°C. Cells were then fixed by dropwise addition of 3:1 MeOH/Acetic Acid for 20 minutes, spun down, and fixed for another 30 minutes. Metaphases were dropped onto slides, dried on wet paper towels and stained with DAPI for visualization. Chromosomes were then measured and counted using ImageJ software. To analyze polyloidy, only cells with greater than 35 chromosomes were counted to eliminate artifacts due to partial spreads.

Gene Expression Profiling and AnalysisRNA was isolated using TRIzol (Invitrogen) and reverse transcribed into cDNA using SuperScript III reverse transcriptase (Invitrogen). Real-time PCR was performed on the StepOnePlus (ABI) machine using FastStart Universal SYBR Green Master with ROX (Roche).

Gene Expression Profiling and Analysis

RNA was isolated using TRIzol (Invitrogen) and reverse transcribed into cDNA using SuperScript III reverse transcriptase (Invitrogen). Real-time PCR was performed on the StepOnePlus (ABI) machine using FastStart Universal SYBR Green Master with ROX (Roche).

ImmunoprecipitationNuclei were isolated from cells with Buffer A (25 mM Hepes, pH7.6, 5 mM MgCl2, 25 mM KCl, 0.05 mM EDTA, 10% glycerol, 0.1% NP-40) and lysed for 30 min in IP buffer (50 mM Tris-HCl, pH 8.0, 150 mM NaCl, 0.1% NP-40). The chromatin was removed using centrifugation and the lysates were precleared with 20 μL protein A or protein G dynabeads for 30 min. The protein concentration was quantitated using the BCA assay and adjusted to a final volume of 200 μL at a final concentration of 1.5 mg/mL with IP buffer. Each IP was incubated with 3 μg of anti-Brg1 (Santa Cruz sc-17796), anti-TopoIIα (Abcam ab52934), anti-BAF45d (Crabtree Lab), anti-BAF47 (Santa Cruz sc-166165), anti-BAF57 (Bethyl A300-810A), anti-BAF155 (Crabtree Lab), anti-BAF60a (BD Transduction Laboratories 611728), anti-BAF250a (Santa Cruz sc-32761x, Santa Cruz sc-98441X), anti-BAF180 (Bethyl A301-590A), anti-BAF250b (Santa Cruz sc-32762X, Bethyl A301-046A), anti-SS18 (Santa Cruz sc-28698) BAF200 (Santa Cruz sc-98299X) or anti-IgG (Santa Cruz sc-2025) overnight at 4°C and then for 2h with 20 μL protein A/G dynabeads. The beads were washed four times with 1 mL IP buffer and resuspended in 10 μL gel loading buffer (4× LDS Buffer; Invitrogen).

Immunoprecipitation

Nuclei were isolated from cells with Buffer A (25 mM Hepes, pH7.6, 5 mM MgCl2, 25 mM KCl, 0.05 mM EDTA, 10% glycerol, 0.1% NP-40) and lysed for 30 min in IP buffer (50 mM Tris-HCl, pH 8.0, 150 mM NaCl, 0.1% NP-40). The chromatin was removed using centrifugation and the lysates were precleared with 20 μL protein A or protein G dynabeads for 30 min. The protein concentration was quantitated using the BCA assay and adjusted to a final volume of 200 μL at a final concentration of 1.5 mg/mL with IP buffer. Each IP was incubated with 3 μg of anti-Brg1 (Santa Cruz sc-17796), anti-TopoIIα (Abcam ab52934), anti-BAF45d (Crabtree Lab), anti-BAF47 (Santa Cruz sc-166165), anti-BAF57 (Bethyl A300-810A), anti-BAF155 (Crabtree Lab), anti-BAF60a (BD Transduction Laboratories 611728), anti-BAF250a (Santa Cruz sc-32761x, Santa Cruz sc-98441X), anti-BAF180 (Bethyl A301-590A), anti-BAF250b (Santa Cruz sc-32762X, Bethyl A301-046A), anti-SS18 (Santa Cruz sc-28698) BAF200 (Santa Cruz sc-98299X) or anti-IgG (Santa Cruz sc-2025) overnight at 4°C and then for 2h with 20 μL protein A/G dynabeads. The beads were washed four times with 1 mL IP buffer and resuspended in 10 μL gel loading buffer (4× LDS Buffer; Invitrogen).

Glycerol Gradient Centrifugation AnalysisES cells were lysed in Buffer A (10 mM Hepes (pH 7.6), 25 mM KCl, 1 mM EDTA, 10% glycerol, 1 mM DTT, and protease inhibitors (complete mini tablets (Roche) supplemented with 1 mM PMSF) on ice. Nuclei were sedimented by centrifugation (1,000×g), resuspended in Buffer C (10 mM Hepes (pH 7.6), 3 mM MgCl2, 100 mM KCl, 0.1 mM EDTA, 10% glycerol, 1 mM DTT, and protease inhibitors), and lysed by the addition of ammonium sulfate to a final concentration of 0.3 M. Soluble nuclear proteins were separated by insoluble chromatin fraction by ultracentrifugation (100,000×g) and precipitated with 0.3 mg/ml ammonium sulfate for 20 min on ice. Protein precipitate was isolated by ultracentrifugation (100,000×g), and resuspended in HEMG-0 buffer (25mM HEPES (pH 7.9), 0.1mM EDTA, 12.5mM MgCl2, 100mM KCl) for glycerol gradient analyses. 800 ug of protein was overlaid on to a 10ml 10-30% glycerol (in HEMG buffer) gradient prepared in a 14×89 mm polyallomer centrifuge tube (Beckman, part# 331327). Tubes were placed in a SW-40 swing bucket rotor and centrifuged at 4 degrees for 16 hours at 40,000 RPM. 0.5 ml fractions were harvested and used in gel electrophoresis and subsequent western blotting analyses.

Glycerol Gradient Centrifugation Analysis

ES cells were lysed in Buffer A (10 mM Hepes (pH 7.6), 25 mM KCl, 1 mM EDTA, 10% glycerol, 1 mM DTT, and protease inhibitors (complete mini tablets (Roche) supplemented with 1 mM PMSF) on ice. Nuclei were sedimented by centrifugation (1,000×g), resuspended in Buffer C (10 mM Hepes (pH 7.6), 3 mM MgCl2, 100 mM KCl, 0.1 mM EDTA, 10% glycerol, 1 mM DTT, and protease inhibitors), and lysed by the addition of ammonium sulfate to a final concentration of 0.3 M. Soluble nuclear proteins were separated by insoluble chromatin fraction by ultracentrifugation (100,000×g) and precipitated with 0.3 mg/ml ammonium sulfate for 20 min on ice. Protein precipitate was isolated by ultracentrifugation (100,000×g), and resuspended in HEMG-0 buffer (25mM HEPES (pH 7.9), 0.1mM EDTA, 12.5mM MgCl2, 100mM KCl) for glycerol gradient analyses. 800 ug of protein was overlaid on to a 10ml 10-30% glycerol (in HEMG buffer) gradient prepared in a 14×89 mm polyallomer centrifuge tube (Beckman, part# 331327). Tubes were placed in a SW-40 swing bucket rotor and centrifuged at 4 degrees for 16 hours at 40,000 RPM. 0.5 ml fractions were harvested and used in gel electrophoresis and subsequent western blotting analyses.

Western BlotsNuclei were isolated from cells with Buffer A (25 mM Hepes, pH 7.6, 5 mM MgCl2, 25 mM KCl, 0.05 mM EDTA, 10% glycerol, 0.1% NP-40) and lysed for 30 min in RIPA buffer (10mM Tris pH 7.4, 150mM NaCl, 0.1% SDS, 0.5% sodium deoxycholate, 1% Triton X-100, 1mM EDTA). Chromatin was either spun out or samples were sonicated prior to BCA analysis. Equal amounts of protein were boiled in gel loading buffer and loaded onto 4-10% BisTris NuPage gels. After transfer, blots were blocked in 5% BSA and incubated with anti-Brg, anti-TopoIIα, anti-pTyr (Millipore clone 4g10 05-321), anti-pSer (Millipore clone 4A4 05-1000), or anti-Ubiquitin (Santa Cruz clone P4D1 sc-8017). Proteins were detected using the LICOR detection system or ECL/autoradiography for SUMO-TopoIIα.

Western Blots

Nuclei were isolated from cells with Buffer A (25 mM Hepes, pH 7.6, 5 mM MgCl2, 25 mM KCl, 0.05 mM EDTA, 10% glycerol, 0.1% NP-40) and lysed for 30 min in RIPA buffer (10mM Tris pH 7.4, 150mM NaCl, 0.1% SDS, 0.5% sodium deoxycholate, 1% Triton X-100, 1mM EDTA). Chromatin was either spun out or samples were sonicated prior to BCA analysis. Equal amounts of protein were boiled in gel loading buffer and loaded onto 4-10% BisTris NuPage gels. After transfer, blots were blocked in 5% BSA and incubated with anti-Brg, anti-TopoIIα, anti-pTyr (Millipore clone 4g10 05-321), anti-pSer (Millipore clone 4A4 05-1000), or anti-Ubiquitin (Santa Cruz clone P4D1 sc-8017). Proteins were detected using the LICOR detection system or ECL/autoradiography for SUMO-TopoIIα.

Chromatin Association AssayNuclei were isolated using Buffer A and re-suspended in 20 mM Tris-HCl pH 7.6, 3 mM EDTA at 60 mill cells/mL. Samples of 25 μL were alloquotted into tubes and NaCl concentrations were adjusted to a final volume of 50 μL. Samples were gently mixed and incubated on ice for 20 min and centrifuged at high speed for 20 min to isolate chromatin. The lysate was removed and the chromatin pellet was re-suspended in 120 μL gel loading dye. The pellet was solubilized using sonication and the association of TopoIIα to chromatin was analyzed using western blotting.

Chromatin Association Assay

Nuclei were isolated using Buffer A and re-suspended in 20 mM Tris-HCl pH 7.6, 3 mM EDTA at 60 mill cells/mL. Samples of 25 μL were alloquotted into tubes and NaCl concentrations were adjusted to a final volume of 50 μL. Samples were gently mixed and incubated on ice for 20 min and centrifuged at high speed for 20 min to isolate chromatin. The lysate was removed and the chromatin pellet was re-suspended in 120 μL gel loading dye. The pellet was solubilized using sonication and the association of TopoIIα to chromatin was analyzed using western blotting.

ATPase AssayATPase assay was adapted from the literature31. Immunoprecipitations were performed as described above with anti-Brg antibody. IPs were washed a final time with 10 mM TrisHCl pH 7.5, 50 mM NaCl, 5 mM MgCl2, 1 mM DTT and resuspended in 20 μL assay buffer (10 mM Tris-HCl pH 7.5, 50 mM NaCl, 5mM MgCl2, 20% glycerol, 1 mg/ml BSA, 0.5 mM ATP, 20 nM plasmid DNA, 50 μCi/mL γ-32P ATP, 1 mM DTT, protease inhibitors). The reaction was agitated at 37 °C for 1h (or when approximately 50% of ATP was converted to inorganic phosphate). Reaction mixture (0.5 uL) was spotted onto PEI cellulose plates and thin layer chromatography was performed in 0.5M LiCl, 1M formic acid. The plates were dried and imaged using phosphorimaging. The enzymatic activity was quantitated as a ratio of product (32P-Pi) to starting material (γ-32P ATP). Values were normalized to the activity of BrgWT (100%) and vector control (0%) cells

ATPase Assay

ATPase assay was adapted from the literature31. Immunoprecipitations were performed as described above with anti-Brg antibody. IPs were washed a final time with 10 mM TrisHCl pH 7.5, 50 mM NaCl, 5 mM MgCl2, 1 mM DTT and resuspended in 20 μL assay buffer (10 mM Tris-HCl pH 7.5, 50 mM NaCl, 5mM MgCl2, 20% glycerol, 1 mg/ml BSA, 0.5 mM ATP, 20 nM plasmid DNA, 50 μCi/mL γ-32P ATP, 1 mM DTT, protease inhibitors). The reaction was agitated at 37 °C for 1h (or when approximately 50% of ATP was converted to inorganic phosphate). Reaction mixture (0.5 uL) was spotted onto PEI cellulose plates and thin layer chromatography was performed in 0.5M LiCl, 1M formic acid. The plates were dried and imaged using phosphorimaging. The enzymatic activity was quantitated as a ratio of product (32P-Pi) to starting material (γ-32P ATP). Values were normalized to the activity of BrgWT (100%) and vector control (0%) cells

Chromatin ImmunoprecipitationFor the Brg1 ChIP, 40 mill ES cells were fixed for 12 minutes in 1% formaldehyde at room temperature. Nuclei were sonicated in 1 mL ChIP Lysis Buffer (50 mM HEPES pH 7.5, 150 mM NaCl, 2 mM EDTA, 1% Triton X-100, 0.1% SDS) to yield fragments between 200-500 bp. 500 μl of lysate was incubated with 5 μg of anti-Brg1 (Crabtree Lab) or 5 μg anti-rabbit IgG and rotated overnight at 4° C and then for 2h with 20 μl Protein A/G Dynabeads. After five washes with ChIP Lysis Buffer and one wash in TE, DNA was eluted by boiling in 10% Chelex slurry.The etoposide ChIP of TopoIIα was adapted from the literature26. Specifically, 20 million ES cells were treated with 100 μM etoposide for 10 minutes. Cells were washed once with PBS and lysed with 1 ml of a buffer containing 1% Sarkosyl, 10 mM Tris-HCl (pH 7.5), 10 mM EDTA, and protease inhibitor. A solution of 7 M CsCl (7 M) was added to a final concentration of 0.5 M and the lysate was sonicated to yield fragments between 200-500 bp. ChIP buffer (300 μL) was added to 300 μl of lysate for a final concentration of 50 mM HEPES pH 7.5, 300 mM NaCl, 1 mM EDTA, 1 % Triton X-100, 0.1 % DOC, and 0.1% SDS and 3 μg Anti-TopoIIα (sc-365916) prebound to 20 μl Protein G Dynabeads was added. The lysate was rotated overnight at 4° C and washed four times with ChIP lysis buffer, one time with LiCl buffer (10 mM Tris pH 8.0, 0.25 M LiCl, 0.5 % NP-40, 0.5% DOC, 1 mM EDTA) and one time with TE. The DNA was eluted with 300 μl of 1% SDS, 0.1 M NaHCO3 for 20 minutes and removed from the beads. The solution was adjusted to 200mM NaCl, 10mM EDTA, 40mM Tris pH 6.5 and 0.2 mg/mL RNase A was added for 30 min at 37° C. Proteinase K was added to 0.03 mg/ml and digested overnight at 55° C. The DNA was extracted with phenol/chloroform and precipitated with ethanol for analysis by qPCR. Primers used for ChIP-qPCR are available upon request.

Chromatin Immunoprecipitation

For the Brg1 ChIP, 40 mill ES cells were fixed for 12 minutes in 1% formaldehyde at room temperature. Nuclei were sonicated in 1 mL ChIP Lysis Buffer (50 mM HEPES pH 7.5, 150 mM NaCl, 2 mM EDTA, 1% Triton X-100, 0.1% SDS) to yield fragments between 200-500 bp. 500 μl of lysate was incubated with 5 μg of anti-Brg1 (Crabtree Lab) or 5 μg anti-rabbit IgG and rotated overnight at 4° C and then for 2h with 20 μl Protein A/G Dynabeads. After five washes with ChIP Lysis Buffer and one wash in TE, DNA was eluted by boiling in 10% Chelex slurry.

The etoposide ChIP of TopoIIα was adapted from the literature26. Specifically, 20 million ES cells were treated with 100 μM etoposide for 10 minutes. Cells were washed once with PBS and lysed with 1 ml of a buffer containing 1% Sarkosyl, 10 mM Tris-HCl (pH 7.5), 10 mM EDTA, and protease inhibitor. A solution of 7 M CsCl (7 M) was added to a final concentration of 0.5 M and the lysate was sonicated to yield fragments between 200-500 bp. ChIP buffer (300 μL) was added to 300 μl of lysate for a final concentration of 50 mM HEPES pH 7.5, 300 mM NaCl, 1 mM EDTA, 1 % Triton X-100, 0.1 % DOC, and 0.1% SDS and 3 μg Anti-TopoIIα (sc-365916) prebound to 20 μl Protein G Dynabeads was added. The lysate was rotated overnight at 4° C and washed four times with ChIP lysis buffer, one time with LiCl buffer (10 mM Tris pH 8.0, 0.25 M LiCl, 0.5 % NP-40, 0.5% DOC, 1 mM EDTA) and one time with TE. The DNA was eluted with 300 μl of 1% SDS, 0.1 M NaHCO3 for 20 minutes and removed from the beads. The solution was adjusted to 200mM NaCl, 10mM EDTA, 40mM Tris pH 6.5 and 0.2 mg/mL RNase A was added for 30 min at 37° C. Proteinase K was added to 0.03 mg/ml and digested overnight at 55° C. The DNA was extracted with phenol/chloroform and precipitated with ethanol for analysis by qPCR. Primers used for ChIP-qPCR are available upon request.

ChIP-seq and AnalysisThe library preparation and sequencing was performed as previously described32. Raw ChIP-seq reads were mapped to the Mus musculus genome (build mm9/NCBI37) using the short-read aligner Bowtie (version 0.12.7)33. Peaks were then called using Model-base Analysis of ChIP-seq (MACS) (version 1.4.1)34. Further analysis was aided by the Bedtools suite (version 2.16.2) 35. Genome annotations were acquired from the UCSC Genome Browser (http://genome.ucsc.edu/)36,37. We also uploaded our data to the genome browser, which was used to produce screenshots of chromatin binding/modification profiles at individual loci.

ChIP-seq and Analysis

The library preparation and sequencing was performed as previously described32. Raw ChIP-seq reads were mapped to the Mus musculus genome (build mm9/NCBI37) using the short-read aligner Bowtie (version 0.12.7)33. Peaks were then called using Model-base Analysis of ChIP-seq (MACS) (version 1.4.1)34. Further analysis was aided by the Bedtools suite (version 2.16.2) 35. Genome annotations were acquired from the UCSC Genome Browser (http://genome.ucsc.edu/)36,37. We also uploaded our data to the genome browser, which was used to produce screenshots of chromatin binding/modification profiles at individual loci.

Topoisomerase Activity AssayReactions contain: 150 ng kinetoplast DNA (Topogen), 50 mM Tris-HCl, pH 8.0, 150 mM NaCl, 10 mM MgCl2, 2 mM ATP, a standard TopoIIα IP or varying amounts of recombinant TopoIIα (Topogen).

Topoisomerase Activity Assay

Reactions contain: 150 ng kinetoplast DNA (Topogen), 50 mM Tris-HCl, pH 8.0, 150 mM NaCl, 10 mM MgCl2, 2 mM ATP, a standard TopoIIα IP or varying amounts of recombinant TopoIIα (Topogen).

Lentiviral Infection293XTs were transfected with lentiviruses containing vector alone, wild-type Brg1, Brg1 point mutants, wild-type hTopoIIα, or hTopoIIαS1524A or with vectors containing hairpins to brg, (TRCN0000071386), arid1a (TRCN0000071395, Origene TG517733), or top2a (V2LMM_11295). 48 hours later, supernatants were collected and centrifuged at 20,000 rpm for 2 hours. Viral pellets were resuspended in PBS and used to infect ES cells in suspension or MEFs by spinfection. Cells were selected with puromycin and collected 48-96 hours postinfection for analysis.

Lentiviral Infection

293XTs were transfected with lentiviruses containing vector alone, wild-type Brg1, Brg1 point mutants, wild-type hTopoIIα, or hTopoIIαS1524A or with vectors containing hairpins to brg, (TRCN0000071386), arid1a (TRCN0000071395, Origene TG517733), or top2a (V2LMM_11295). 48 hours later, supernatants were collected and centrifuged at 20,000 rpm for 2 hours. Viral pellets were resuspended in PBS and used to infect ES cells in suspension or MEFs by spinfection. Cells were selected with puromycin and collected 48-96 hours postinfection for analysis.

Supplementary Material1

Supplementary Material
