---
pmid: '20622854'
title: PHF8 mediates histone H4 lysine 20 demethylation events involved in cell cycle
  progression.
authors:
- Liu W
- Tanasa B
- Tyurina OV
- Zhou TY
- Gassmann R
- Liu WT
- Ohgi KA
- Benner C
- Garcia-Bassets I
- Aggarwal AK
- Desai A
- Dorrestein PC
- Glass CK
- Rosenfeld MG
journal: Nature
year: '2010'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC3059551
doi: 10.1038/nature09272
---

# PHF8 mediates histone H4 lysine 20 demethylation events involved in cell cycle progression.
**Authors:** Liu W, Tanasa B, Tyurina OV, Zhou TY, Gassmann R, Liu WT, Ohgi KA, Benner C, Garcia-Bassets I, Aggarwal AK, Desai A, Dorrestein PC, Glass CK, Rosenfeld MG
**Journal:** Nature (2010)
**DOI:** [10.1038/nature09272](https://doi.org/10.1038/nature09272)
**PMC:** [PMC3059551](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3059551/)

## Abstract

1. Nature. 2010 Jul 22;466(7305):508-12. doi: 10.1038/nature09272. Epub 2010 Jul 
11.

PHF8 mediates histone H4 lysine 20 demethylation events involved in cell cycle 
progression.

Liu W(1), Tanasa B, Tyurina OV, Zhou TY, Gassmann R, Liu WT, Ohgi KA, Benner C, 
Garcia-Bassets I, Aggarwal AK, Desai A, Dorrestein PC, Glass CK, Rosenfeld MG.

Author information:
(1)Howard Hughes Medical Institute, School of Medicine, University of California 
at San Diego, 9500 Gilman Drive, La Jolla, California 92093, USA.

While reversible histone modifications are linked to an ever-expanding range of 
biological functions, the demethylases for histone H4 lysine 20 and their 
potential regulatory roles remain unknown. Here we report that the PHD and 
Jumonji C (JmjC) domain-containing protein, PHF8, while using multiple 
substrates, including H3K9me1/2 and H3K27me2, also functions as an H4K20me1 
demethylase. PHF8 is recruited to promoters by its PHD domain based on 
interaction with H3K4me2/3 and controls G1-S transition in conjunction with 
E2F1, HCF-1 (also known as HCFC1) and SET1A (also known as SETD1A), at least in 
part, by removing the repressive H4K20me1 mark from a subset of E2F1-regulated 
gene promoters. Phosphorylation-dependent PHF8 dismissal from chromatin in 
prophase is apparently required for the accumulation of H4K20me1 during early 
mitosis, which might represent a component of the condensin II loading process. 
Accordingly, the HEAT repeat clusters in two non-structural maintenance of 
chromosomes (SMC) condensin II subunits, N-CAPD3 and N-CAPG2 (also known as 
NCAPD3 and NCAPG2, respectively), are capable of recognizing H4K20me1, and 
ChIP-Seq analysis demonstrates a significant overlap of condensin II and 
H4K20me1 sites in mitotic HeLa cells. Thus, the identification and 
characterization of an H4K20me1 demethylase, PHF8, has revealed an intimate link 
between this enzyme and two distinct events in cell cycle progression.

DOI: 10.1038/nature09272
PMCID: PMC3059551
PMID: 20622854 [Indexed for MEDLINE]

## Full Text

We evaluated potential substrates for the putative histone demethylase, PHF8, on mononucleosomes. Flag-tagged PHF8 immunoprecipitated from HEK293T cells was capable of demethylating H4K20me1, H3K9me1/2 and H3K27me2, but had no effects on other histone methylation marks tested. Mutation of histidine 247, predicted to be part of the Fe(II) binding site5, impaired these activities (Fig. 1a), which were similarly confirmed using bacterially-expressed PHF8 (Fig. 1b, left panels) and in a dose- and time-dependent manner (Fig. S2a). Surprisingly, when using core histones as substrates, while the activities towards H3K9me2 and H3K27me2 were preserved, only minimal activities were detected towards H4K20me1 and H3K9me1 (Fig. 1b, right panels), as similarly found for PHF8 immunoprecipitated from HEK293T cell lysates (Fig. S2b).

To gain insight into biological substrates of PHF8, we performed ChIP-Seq. in HeLa cells, finding that ~ 72% of PHF8 peaks localized on promoters, with the most statistically-significant predicted binding sites being those for Ets and E2F1 (Fig. 1c). The great majority of PHF8-bound promoters overlapped with H3K4me3 (6) and H3K4me2 positive promoters, while very few harbored H4K20me1, H3K9me1/2, or H3K27me2 marks (Fig. 1d and Fig. S2c). The PHF8 tags distribution relative to transcription start site corresponded to that of the H3K4me2 mark (Fig. 1e), suggesting that PHF8 might be recruited to promoters by H3K4me2/3 based on interaction with its PHD finger, a known methyl lysine binding motif7. In vitro peptide pull-down revealed that PHF8 specifically bound to H3K4me2 and H3K4me3 histone tails, but not to others tested (Fig.1f). Direct interaction of PHD finger of PHF8 with H3K4me2/3 histone tails was confirmed using bacterially-expressed PHF8 PHD finger (Fig. 1g), while PHF8 ΔPHD failed to interact (Fig. S2d). Consistent with these data, PHF8ΔPHD retained its activities on mononucleosomes towards H3K9me2 and H3K27me2, but not towards H4K20me1 or H3K9me1 (Fig. 1h). These results indicate a required PHD finger-mediated targeting of PHF8 to H3K4me2/3-containing nucleosomes to efficiently demethylate H4K20me1 or H3K9me1.

To evaluate PHF8 enzymatic activities in vivo, a PHF8-specific siRNA was transfected into HeLa cells, which led to significantly increased H4K20me1 levels and slightly increased H4K20me2 and H3K9me1, without changes in other histone marks (Fig. 2a). This H4K20me1 increase was dependent on PHF8 enzymatic activity (Fig. S2e), perhaps also partially attributed to PHF8 depletion interference with cell cycle progression (vide infra). Other known H3K9/K27 demethylases might compensate the loss of PHF8, therefore minimizing PHF8 siRNA effects. Immuno-fluorescence microscopy analysis in HeLa cells over-expressing Flag-tagged full-length PHF8 revealed that ~25% of cells exhibited significantly decreased H4K20me1 signal intensity (n=200), while PHF8 ΔPHD or PHF8 (H247A) had no effects (Fig. 2b and Fig. S2f). PHF8 over-expression caused no significant differences for other substrates (Fig. S2g–n). Similar results were obtained with PHF8 (1-447) (Fig. S3a), consistent with recent studies8,9, except that we additionally observed PHF8 activity towards H4K20me1. U2OS cells over-expressing Flag-tagged full-length PHF8 showed decrease in H4K20me1 (~20%) as well as H3K9me2 (~60%), but not H3K9me1 or H3K27me2 signal intensity (n=200) (Fig. S3b). Thus, PHF8 activity may be regulated both in a cell type-dependent manner and by specific modifications10–12. To further evaluate PHF8 enzymatic activities in HeLa cells, ChIP-Seq. analysis of H4K20me1, H3K9me1, H3K9me2 and H3K27me2 was performed in control or PHF8 siRNA-transfected HeLa cells, finding that H4K20me1 and H3K9me1, but not H3K9me2 or H3K27me2, increased significantly on PHF8-bound promoter regions upon PHF8 depletion (Fig. 2c, d and Fig. S3c, d). The difference in H4K20me1 and H3K9me1 levels between PHF8 siRNA-transfected and control cells correlated with the number of PHF8 ChIP-Seq. tags on promoters (Fig. S3e, f).

We next focused on investigating the PHF8 activity towards H4K20me1, a histone mark that has been implicated in cell cycle regulation13–17. Interestingly, gene ontology analysis of PHF8-bound promoters identified by ChIP-Seq. (Fig. 1c and Fig. S4a) revealed that one of the most statistically-significant terms was “cell cycle” (Fig. S4b), as similarly found in RNA profiling analysis for genes positively regulated by PHF8 (Fig. S4c, d), with ~72% of which proving to be PHF8 ChIP-Seq. targets. To gain further insight into PHF8 function, its associated proteins were purified, which included critical G1/S transition regulators, E2F1, HCF-1 and Set1A18,19. Interactions between PHF8 and these proteins were confirmed by immunoprecipitation (Fig. 2e). Gel filtration chromatography revealed co-fractionation of native PHF8, E2F1, HCF-1 and Set1A (Fig. S4e). Furthermore, E2F1 ChIP-Seq. analysis in HeLa cells indicated that >79% of E2F1-bound promoters corresponded to those binding PHF8 (Fig. S5a–c), suggesting a role of PHF8 in regulating E2F1 target genes.

Consistent with potential roles of PHF8 in cell cycle regulation, PHF8 siRNA-treated HeLa cells exhibited a strikingly decreased cell proliferation (Fig. S6a). Flow-cytometry analysis suggested a delay in G1/S transition and revealed a slight increase in M phase population upon PHF8 knock-down (Fig. 2f). In addition, there was a decrease in cell size after PHF8 siRNA treatment (Fig. S6b). BrdU incorporation in control or PHF8 siRNA-treated HeLa cells released from mitotic arrest confirmed that PHF8 depletion led to S phase entry delay (Fig. S6c, d). To further assess PHF8 enzymatic activity at different stages during cell cycle, chromatin-bound fractions were isolated from control or PHF8 siRNA-treated HeLa cells that were synchronized following the protocols in Fig. S6e, finding that PHF8 siRNA treatment led to a significant and distinct increase in H4K20me1 at G1/S and S phases, but exhibited minimal effects at G2/M (Fig. 2g and Fig. S6f, g), which was confirmed by immuno-fluorescence microscopy analysis (Fig. S6h). Minimal effects at G2/M might be due to PHF8 dissociation from chromatin at this stage (vide infra). ChIP analysis on G1/S transition-regulated promoters bound by PHF8 and E2F1, including RBL1 (p107), CDC25A, CCNE1 and E2F1, revealed an increase in H4K20me1 levels and decrease in PHF8 binding upon PHF8 knock-down (Fig. 2h and Fig. S7a). The levels of H4K20me3, H3K9me1/2/3 or H3K27me1/2/3 were not significantly altered, except for an increase of H4K20me3 on CCNE1 promoter (Fig. S7b–h). L3MBTL1, which has been shown to associate with H4K20me1 and function in gene repression20, exhibited increased binding (Fig. 2i), while H3K4me3 decreased upon PHF8 knock-down (Fig. S7i). ChIP analysis on promoters exhibiting changes in H3K9me1 and/or H4K20me1 in ChIP-Seq experiments confirmed subsets exhibiting an increase of only H4K20me1 or H3K9me1, or both marks upon PHF8 knock-down (Fig. S8). Similarly, a change in H3K9me2 on rDNA gene promoter regions in PHF8 siRNA-treated HEK293T cells has been reported12. Therefore, PHF8 uses distinct substrates on different subsets of genes to exert its function.

To further elucidate the role of PHF8 in G1/S transition, we performed PHF8 ChIP-Seq. in synchronized G1 and G1/S HeLa cells, revealing ~77% of binding sites at G1 localizing on promoters, as did 86% of PHF8 binding sites shared in both G1 and G1/S border. For all those shared PHF8 binding sites, ~85% showed increased PHF8 tags when cells reached G1/S border (Fig. 3a and Fig. S9), when H4K20me1 reaches its lowest levels15 (Fig. 2g). Moreover, ~13,000 new PHF8 binding sites were detected, mostly at intra- or extra-genic loci (>70%), which likely mitigate inappropriate appearance of H4K20me1 (Fig. 3a). ChIP analysis confirmed PHF8 binding increased on promoter regions of selected G1/S transition-regulated cell cycle genes (Fig. 3b). In concert with association between PHF8 and HCF-1, HCF-1 was similarly recruited (Fig. S10a). There was also a decrease in H4K20me1 levels (Fig. 3c) and L3MBTL1 binding (Fig. 3d), but an increase in H3K4me3 (Fig. S10b) during G1/S transition. Both PHF8 and HCF-1 siRNAs blocked induction of selected G1/S transition-regulated genes (Fig. S11). PHF8 and HCF-1 co-immunoprecipitated (Fig. 3e) and the interacting region was mapped to the N-terminus of HCF-1 (Fig. 3f), which has been suggested to be essential for its function as G1/S transition regulator21, further supporting their functional interactions. Furthermore, PHF8 knock-down impaired HCF-1 and Set1A recruitment (Fig. 3g, h), while HCF-1 knock-down did not impair PHF8 recruitment (Fig. S12).

We examined chromatin fractions isolated from different phases in cell cycle, confirming PHF8 disappearance from chromatin when cells enter mitosis, which correlates with an increase in H4K20me1 and Pr-Set7 (Fig. 2g and Fig. 4a), while minimal or no increase in other marks was observed (Fig. 13a). Immunostaining of PHF8 without pre-extraction revealed no significant signal intensity change between interphase and prophase cells (Fig. S13b), while with pre-extraction revealed its dissociation from chromatin in prophase and re-association in telophase (Fig. 4b), as similarly found for GFP-tagged PHF8 (Fig. S13c). In contrast, H4K20me1 levels increased dramatically in prophase (Fig. S13c, d). PHF8 dissociation became more evident as chromosomes condensed (Fig. S13e). These results suggest that PHF8 dissociation in prophase might be involved in H4K2ome1 increase at this stage, as similarly proposed due to the increase of methyltransferase, Pr-Set715. Interestingly, the five-subunit Condensin II complex begins loading onto chromosomes starting in prophase22–25. Cellular fractionation revealed that PHF8 dissociation from chromatin paralleled chromosomal loading of Condensin II during M-phase (Fig. 13f). We therefore wanted to test whether PHF8 dissociation from chromatin permits accumulation of H4K20me1 and, potentially, Condensin II loading.

To test these hypotheses, we first investigated the mechanisms underlying PHF8 dissociation from chromatin. CDK1/cyclin B1 activity is known to fluctuate during cell cycle26, with its activity being inverse to PHF8 association with chromatin. In vitro kinase assays revealed CDK1 phosphorylates PHF8 (Fig. 4c). Two putative phosphorylation sites at serine 33 and 84 were identified by mass spectrometry (Fig. S14a, b), and mutation at either site proved to be a poor substrate, with double mutation further diminishing the phosphorylation (Fig. S14c). Inhibition of CDK activity impaired PHF8 dissociation from chromatin in mitosis (Fig. S14d). Similarly, mutation of the phosphorylation sites disrupted PHF8 dissociation from chromatin and increase in H4K20me1 levels that occurred in colcemid-treated cells (Fig. 4d). Thus, PHF8 phosphorylation at both serine 33 and 84 appears to be required for triggering its dissociation from chromatin and accumulation of H4K20me1 levels in prophase. In accord with the possibility that PHF8 dissociation from chromatin could potentially participate in Condensin II loading, M phase-synchronized cells transfected with an empty vector or wild type PHF8, but not mutant PHF8, exhibited significant increase of chromatin-associated Condensin II components compared with asynchronized cells (Fig. 4e).

The Condensin II complex from HeLa mitotic extracts was selectively pulled down by H4K20me1 and weakly by H4K20me2, but not by other histone tails tested (Fig. 4f and Fig. S15a). No interaction was detected between H4K20me1 and the non-SMC subunits in the Condensin I complex or one component in the Cohesin complex, Rad21, as similarly found for Condensin II from asynchronized cells (Fig. S15b). Furthermore, purified Condensin II bound preferentially to mononucleosomes mono-methylated at H4K20 (Fig. S15c). We then asked which subunit in Condensin II could mediate these interactions. As shown in Fig. 4g, N-CAPD3 strongly and N-CAPG2, to a less extent, interacted with H4K20me1 histone tails, but not the other three subunits. Little, if any, interaction was detected with H4K20me3 (Fig. S16a). Since none of the subunit in Condensin II contains known motifs associating with H4K20me1, we hypothesized that a distinct motif harbored in N-CAPD3 and N-CAPG2, the HEAT repeat27–30, might be responsible for these events (Fig. S16b). Indeed, the three predicted HEAT repeat clusters in N-CAPD3 and N-CAPG2 exhibited interactions with H4K20me1 histone tails, with the ones from N-CAPD3 showing stronger affinity than that from N-CAPG2. In contrast, minimal, if any, interaction with H4K20me3 was detected (Fig. 4h, lane 2, 3 and 4). As control, H4K20me1 histone tails failed to pull down N-CAPD3 C-terminus (Fig. 4h, lane 1). Furthermore, ChIP-Seq. analysis found SMC4 and H4K2ome1 co-localized predominately at intra- and extra-genic regions (Fig. 4i), with ~55% of SMC4-bound regions occupied by H4K20me1 (Fig. 4j). The specificity of a number of SMC4 binding sites was validated by ChIP-qPCR (Fig. S17b, c). Given that SMC4 is present in both Condensin I and II complexes, which have different distributions along chromosomes27, the ~55% overlap of SMC4 and H4K20me1 loci supports the specific association between H4K20me1 and Condensin II.

In conclusion, our study has revealed that PHF8 acts as a cell cycle regulator, at least partially based on its H4K20me1 demethylase activity (Supplementary Fig. 1). The absence of PHF8 leads to a delay in G1/S transition and its dissociation from chromatin in early mitosis, which in conjunction with increased expression of Pr-Set7, leads to a surge of the H4K20me1 mark capable of interacting with the Condensin II complex through the HEAT repeat clusters harbored in two non-SMC subunits, N-CAPD3 and N-CAPG2.

MethodsPHF8 protein purification and Demethylation assayFlag-PHF8 proteins were expressed in HEK293T cells and cells were lysed in lysis buffer containing 50mM Tris HCl, pH 7.4, 450mM NaCl, 1mM EDTA, 1% TRITON X-100 followed by sonication. High salt concentration and sonication were applied here due to PHF8 is mainly and tightly associated with chromatin. Flag-PHF8 proteins were then affinity purified by using Anti-flag M2-agarose as described in the technical bulletin (Sigma A2220) and washed extensively. Before elution with 3X Flag peptides (Sigma), the affinity resins were washed with demethylation buffer (20 mM Tris-HCl, pH 7.5, 150 mM NaCl, 50 mM [NH4]2Fe[SO4]2, 1 mM α-ketoglutarate, and 2 mM ascorbic acid) twice. His-tagged PHF8 proteins were expressed in BL21 (DE3) bacterial cells (Stratagene) and purified by using Ni-NTA Agarose (Qiagen). For demethylation reactions, PHF8 proteins were incubated with 5μg of bulk histones (Sigma H9250) or mononucleosomes prepared from HeLa cells in demethylation buffer at 37 °C.Constructs, antibodies and other methods are described in the Supplementary Methods.

PHF8 protein purification and Demethylation assayFlag-PHF8 proteins were expressed in HEK293T cells and cells were lysed in lysis buffer containing 50mM Tris HCl, pH 7.4, 450mM NaCl, 1mM EDTA, 1% TRITON X-100 followed by sonication. High salt concentration and sonication were applied here due to PHF8 is mainly and tightly associated with chromatin. Flag-PHF8 proteins were then affinity purified by using Anti-flag M2-agarose as described in the technical bulletin (Sigma A2220) and washed extensively. Before elution with 3X Flag peptides (Sigma), the affinity resins were washed with demethylation buffer (20 mM Tris-HCl, pH 7.5, 150 mM NaCl, 50 mM [NH4]2Fe[SO4]2, 1 mM α-ketoglutarate, and 2 mM ascorbic acid) twice. His-tagged PHF8 proteins were expressed in BL21 (DE3) bacterial cells (Stratagene) and purified by using Ni-NTA Agarose (Qiagen). For demethylation reactions, PHF8 proteins were incubated with 5μg of bulk histones (Sigma H9250) or mononucleosomes prepared from HeLa cells in demethylation buffer at 37 °C.Constructs, antibodies and other methods are described in the Supplementary Methods.

PHF8 protein purification and Demethylation assay

Flag-PHF8 proteins were expressed in HEK293T cells and cells were lysed in lysis buffer containing 50mM Tris HCl, pH 7.4, 450mM NaCl, 1mM EDTA, 1% TRITON X-100 followed by sonication. High salt concentration and sonication were applied here due to PHF8 is mainly and tightly associated with chromatin. Flag-PHF8 proteins were then affinity purified by using Anti-flag M2-agarose as described in the technical bulletin (Sigma A2220) and washed extensively. Before elution with 3X Flag peptides (Sigma), the affinity resins were washed with demethylation buffer (20 mM Tris-HCl, pH 7.5, 150 mM NaCl, 50 mM [NH4]2Fe[SO4]2, 1 mM α-ketoglutarate, and 2 mM ascorbic acid) twice. His-tagged PHF8 proteins were expressed in BL21 (DE3) bacterial cells (Stratagene) and purified by using Ni-NTA Agarose (Qiagen). For demethylation reactions, PHF8 proteins were incubated with 5μg of bulk histones (Sigma H9250) or mononucleosomes prepared from HeLa cells in demethylation buffer at 37 °C.

Constructs, antibodies and other methods are described in the Supplementary Methods.

Supplementary Material123Supplementary methods and referencessupplementary figures

Supplementary Material
