---
pmid: '27881680'
title: Structures and Short Linear Motif of Disordered Transcription Factor Regions
  Provide Clues to the Interactome of the Cellular Hub Protein Radical-induced Cell
  Death1.
authors:
- O'Shea C
- Staby L
- Bendsen SK
- Tidemand FG
- Redsted A
- Willemoës M
- Kragelund BB
- Skriver K
journal: J Biol Chem
year: '2017'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC5241728
doi: 10.1074/jbc.M116.753426
---

# Structures and Short Linear Motif of Disordered Transcription Factor Regions Provide Clues to the Interactome of the Cellular Hub Protein Radical-induced Cell Death1.
**Authors:** O'Shea C, Staby L, Bendsen SK, Tidemand FG, Redsted A, Willemoës M, Kragelund BB, Skriver K
**Journal:** J Biol Chem (2017)
**DOI:** [10.1074/jbc.M116.753426](https://doi.org/10.1074/jbc.M116.753426)
**PMC:** [PMC5241728](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5241728/)

## Abstract

1. J Biol Chem. 2017 Jan 13;292(2):512-527. doi: 10.1074/jbc.M116.753426. Epub
2016  Nov 23.

Structures and Short Linear Motif of Disordered Transcription Factor Regions 
Provide Clues to the Interactome of the Cellular Hub Protein Radical-induced 
Cell Death1.

O'Shea C(1), Staby L(1), Bendsen SK(1), Tidemand FG(1), Redsted A(1), Willemoës 
M(1), Kragelund BB(1), Skriver K(2).

Author information:
(1)From the Linderstrøm-Lang Centre for Protein Science, Department of Biology, 
University of Copenhagen, 5 Ole Maaloes Vej, Copenhagen DK-2200, Denmark.
(2)From the Linderstrøm-Lang Centre for Protein Science, Department of Biology, 
University of Copenhagen, 5 Ole Maaloes Vej, Copenhagen DK-2200, Denmark 
kskriver@bio.ku.dk.

Intrinsically disordered protein regions (IDRs) lack a well defined 
three-dimensional structure but often facilitate key protein functions. Some 
interactions between IDRs and folded protein domains rely on short linear motifs 
(SLiMs). These motifs are challenging to identify, but once found they can point 
to larger networks of interactions, such as with proteins that serve as hubs for 
essential cellular functions. The stress-associated plant protein 
radical-induced cell death1 (RCD1) is one such hub, interacting with many 
transcription factors via their flexible IDRs. To identify the SLiM bound by 
RCD1, we analyzed the IDRs in three protein partners, DREB2A 
(dehydration-responsive element-binding protein 2A), ANAC013, and ANAC046, 
considering parameters such as disorder, context, charges, and pI. Using a 
combined bioinformatics and experimental approach, we have identified the 
bipartite RCD1-binding SLiM as (DE)X(1,2)(YF)X(1,4)(DE)L, with essential 
contributions from conserved aromatic, acidic, and leucine residues. Detailed 
thermodynamic analysis revealed both favorable and unfavorable contributions 
from the IDRs surrounding the SLiM to the interactions with RCD1, and the SLiM 
affinities ranged from low nanomolar to 50 times higher Kd values. Specifically, 
although the SLiM was surrounded by IDRs, individual intrinsic α-helix 
propensities varied as shown by CD spectroscopy. NMR spectroscopy further 
demonstrated that DREB2A underwent coupled folding and binding with α-helix 
formation upon interaction with RCD1, whereas peptides from ANAC013 and ANAC046 
formed different structures or were fuzzy in the complexes. These findings allow 
us to present a model of the stress-associated RCD1-transcription factor 
interactome and to contribute to the emerging understanding of the interactions 
between folded hubs and their intrinsically disordered partners.

© 2017 by The American Society for Biochemistry and Molecular Biology, Inc.

DOI: 10.1074/jbc.M116.753426
PMCID: PMC5241728
PMID: 27881680 [Indexed for MEDLINE]

## Full Text

Abstract

Intrinsically disordered protein regions (IDRs) lack a well defined three-dimensional structure but often facilitate key protein functions. Some interactions between IDRs and folded protein domains rely on short linear motifs (SLiMs). These motifs are challenging to identify, but once found they can point to larger networks of interactions, such as with proteins that serve as hubs for essential cellular functions. The stress-associated plant protein r adical-induced c ell d eath1 (RCD1) is one such hub, interacting with many transcription factors via their flexible IDRs. To identify the SLiM bound by RCD1, we analyzed the IDRs in three protein partners, DREB2A (dehydration-responsive element-binding protein 2A), ANAC013, and ANAC046, considering parameters such as disorder, context, charges, and pI. Using a combined bioinformatics and experimental approach, we have identified the bipartite RCD1-binding SLiM as (DE) X (1,2)(YF) X (1,4)(DE)L, with essential contributions from conserved aromatic, acidic, and leucine residues. Detailed thermodynamic analysis revealed both favorable and unfavorable contributions from the IDRs surrounding the SLiM to the interactions with RCD1, and the SLiM affinities ranged from low nanomolar to 50 times higher K d values. Specifically, although the SLiM was surrounded by IDRs, individual intrinsic α-helix propensities varied as shown by CD spectroscopy. NMR spectroscopy further demonstrated that DREB2A underwent coupled folding and binding with α-helix formation upon interaction with RCD1, whereas peptides from ANAC013 and ANAC046 formed different structures or were fuzzy in the complexes. These findings allow us to present a model of the stress-associated RCD1-transcription factor interactome and to contribute to the emerging understanding of the interactions between folded hubs and their intrinsically disordered partners.

Introduction

Intrinsically disordered protein regions (IDRs), 3 which lack a well defined three-dimensional structure, are being recognized as key facilitators of protein function ( 1 – 3 ). This means that the understanding of protein-protein interactions is changing. Short linear motifs (SLiMs) are central to IDRs. They are responsible for interactions with globular protein domains and mediate a wide range of important cellular tasks ( 4 ). They are short, typically between 3 and 11 residues ( 5 , 6 ), and are most often found in IDRs or surface-accessible regions ( 7 , 8 ). Their limited binding surface may result in low affinity, often in the low micromolar K d range, with transient and promiscuous interactions ( 5 , 9 , 10 ), which are well suited for dynamic binding typical of intracellular signaling. Molecular recognition features (MoRFs) represent another type of ID-associated interaction element. They are also short, usually less than 20 residues, and are located within longer IDRs ( 11 ). Unlike SLiMs, MoRFs are not defined on the basis of sequence, but as interaction-prone disordered segments that may form secondary structures upon binding. MoRFs may themselves contain SLiMs. More than 2000 domain-interacting SLiMs have already been identified ( 12 ), but because the human proteome was estimated to encode more than 100,000 binding motifs ( 4 ), the identification of many more motifs may be expected. However, motif prediction is difficult mostly due to the challenge of obtaining robust statistical assessments ( 13 ), which means that experimental approaches are still needed for de novo SLiM identification.

SLiMs are especially relevant in signaling networks in which the proteins referred to as hubs have high connectivity. Hub proteins are essential for interaction network functionality, and their disruption is therefore frequently associated with diseases. Knock-out of the cellular plant hub protein r adical induced c ell d eath1 (RCD1) also resulted in severe phenotypic changes, explained by the pleiotropic roles of the RCD1 gene during both stress responses and plant development ( 14 , 15 ). RCD1 interacts with at least 30 proteins, including 21 transcription factors representing several different transcription factor families ( 14 ), using its small C-terminal α-helical R CD1- S RO- T AF4 (RST) domain ( 14 , 16 ). The rcd1 knock-out mutant showed altered expression of more than 500 genes, of which several are target genes of the RCD1-interaction partners ( 14 ). However, recent studies suggest that RCD1 does not transcriptionally regulate genes encoding its interaction partners ( 17 ). Instead, RCD1 was suggested to negatively regulate the stability of stress-associated d ehydration- r esponsive e lement- b inding protein 2A (DREB2A), belonging to the AP2/ERF plant-specific transcription factor family. Upon stress, RCD1 was rapidly degraded, promoting the proper DREB2A function ( 18 ).

RCD1 and its interaction partners represent an obvious model system for studies of interactions between folded hubs and intrinsically disordered proteins (IDPs). Several of the transcription factors interacting with RCD1 play significant roles in plant biology. For example, DREB2A modulates gene expression in response to various abiotic stress exposures ( 19 ), and the membrane-bound n o apical meristem, A TAF, c up-shaped cotyledon (NAC) transcription factor Arabidopsis thaliana (A)NAC013 is involved in mitochondrial retrograde regulation in response to oxidative stress ( 20 ). Furthermore, the A. thaliana plant system allows transformation of in vitro results to the organismal level ( 21 ). The ability of RCD1 to interact with many proteins was explained by intrinsic disorder (ID)-associated flexibility ( 21 ). For ANAC046, the RCD1-interacting site coincides with the only MoRF within its large transcription regulatory domain (TRD) ( 16 ). However, most RCD1-interacting transcription factors, e.g. ANAC013 and DREB2A, have complex order-disorder profiles making prediction of protein interaction sites difficult ( 16 ). Using yeast two-hybrid analysis, the RCD1-interacting motif FD XX ELL XX LN was identified for DREB2A. Although the RCD1-interacting regions of DREB2A and ANAC046 show some compositional similarities ( 16 ), the DREB2A RCD1-interacting motif could not be identified in ANAC046 or in the other RCD1 partners ( 18 ). In this study, we combined bioinformatics and experimental approaches to identify a loose SLiM present in several RCD1 interaction partners. Based on detailed thermodynamic and biophysical characterizations, a model for interactions between RCD1 and its target transcription factors is presented. This study also provides a novel framework for experimental determination of new SLiMs that, together with the rest of the work, will contribute to the emerging understanding of the complex interactions between folded hubs and their disordered partners.

Discussion

Only 2% of the ELM database entries are derived from plant proteins ( 37 ) making efforts for motif identification in plants urgent. In this study, we defined the SLiM (DE) X (1,2)(YF) X (1,4)(DE)L for transcription factor binding to RCD1. The functional importance of this SLiM is supported by its conservation in IDRs with low degrees of sequence similarities ( 22 ) and by its in vivo role for DREB2A·RCD1 interactions ( 18 ). A high risk of false-positives exists for motif identification ( 38 ), and not all of the regions shown in Fig. 3 , A and B, may be biochemically and physiologically relevant. Six transcription factors from the RCD1 interactome do not contain the RCD1 SLiM. They may either contain a variant or a different binding sequence ( 10 ).

Several known SLiMs are compositionally similar to the RCD1 SLiM. For example, the region DFDLDMLGD from the herpes simplex virus VP16 and several additional transcriptional regulators are implicated in essential interactions, and these regulatory proteins all interact with the general co-activator TAF9 ( 39 ). Interestingly, another general co-activator, Arabidopsis TAF4, contains an RST domain ( 40 ), making the RCD1 SLiM of relevance not only to gene-specific transcription factors but also to the general transcriptional apparatus. The RCD1 SLiM also shows similarity to the acidic class EDLL activation motif found in a sub-group of AP2/ERF transcription factors ( 41 , 42 ).

Substitution analyses of the RCD1 SLiM ( Tables 1 and 3 ) ( 16 ) support the observation that substitution of a single residue can abolish SLiM binding due to the short length of SLiMs ( 10 ). The results are also in accordance with additional typical features of SLiMs in IDRs. Thus, hydrophobic amino acid residues are over-represented in SLiMs ( 11 ); polycation-π interactions involving an aromatic residue can be significant components of interactions involving IDPs ( 43 ), and electrostatic interactions involving charged residues are key components of interactions involving IDPs ( 44 ). Furthermore, truncation of side chains in a SLiM almost always impairs binding, because SLiM contacts are nearly optimal ( 24 ). In conclusion, the RCD1 SLiM characteristics are typical of SLiMs in IDRs and of transcriptional activation motifs.

The affinity of the interactions between the RCD1-RST domain and the transcription factor peptides varied ∼70-fold ( Tables 1 – 3 ) ( 16 ). Although the differences in affinity may have biological consequences, our thermodynamic characterization demonstrated that peptide length and context also significantly affect affinity ( Tables 1 and 3 ). Curiously, removal of the PEPEPT sequence from ANAC013(254–274) resulted in a 66-fold decrease in affinity due to a marked decrease in binding enthalpy. Computational studies suggested that the core of a SLiM contributes about 80% of the binding energy ( 24 ). This study revealed a more complex pattern. Whereas the results suggested a negative allosteric effect on binding by the contexts of ANAC013(254–274), the contexts of DREB2A(255–272) had an opposite positive effect. For ANAC013, truncation reversed the entropic contribution to binding from negative to positive, whereas the opposite was observed for the DREB2A truncation series. This is in accordance with the recent finding that the sequences of IDRs are more than passive scaffolds for motifs and contribute to regulation of functions ( 45 ). It is generally assumed that IDRs pay an entropic cost upon binding because of conformational restriction ( 46 ). However, IDRs may also use entropy for binding ( Tables 1 – 3 ) through solvent-mediated entropic interactions or by release of charge-charge interactions in the unbound state. It was recently suggested that interactions may increase conformational flexibility to induce favorable entropic changes ( 47 ) or lead to less compaction of the surrounding IDR compared with the unbound state ( 45 ). This could explain the thermodynamic profiles of interactions between RCD1 and the COL10 and STO peptides ( Table 2 ). If, and how, these complex affinity and thermodynamics patterns relate to the in vivo concentrations of the transcription factors is currently not known.

Although similarities were observed for binding of the different transcription factor peptides to RCD1, differences were also observed. Thus, the importance of α-helix structure for binding differs. Studies of protein binding to the hub proteins Keap1 ( 48 ) and calmodulin ( 49 ) suggested that a reduction in the conformational freedom of the free state results in an affinity increase. A similar observation was not apparent for the peptides binding to RCD1. Supportive of this, a pre-formed α-helix in different disordered peptides had a small effect in enhancing the binding affinity for the target proteins ( 50 ). For the DREB2A peptide, strong α-helix formation in the complex was demonstrated by NMR spectroscopic analysis ( Fig. 7 ). In contrast, the ANAC013 and ANAC046 peptides may form non-helical or even fuzzy interfaces in the complexes ( Figs. 5 A and 7 , B and C ) ( 11 , 51 ). To this end, kinetic analysis is needed to determine whether binding involves conformational selection, induced fit, or a continuum of the two ( 52 ).

Based on primary and secondary structure analyses ( Fig. 6 A ) ( 40 ), the RST domain of RCD1 is likely to form a fold similar to that of the human TAF4 TAFH domain. This domain adopts an α-helical fold with a large hydrophobic groove that forms the binding surface for TAF4-interacting transcription factors. In addition to hydrophobic/aromatic residues, the TAFH groove also contains a lysine residue, which plays an important role in ligand recognition. The TAFH-binding transcription factor regions contain conserved acidic and hydrophobic residues, similar to the RCD1 SLiM ( 53 ). Fig. 8 shows a model of RST-transcription factor interactions. Our data suggest that the RST domain of RCD1 can accommodate structural diversity in its partners, from fuzziness to folded α-helical structure. The observation that the spacing between the core residues of the RCD1-binding SLiM differs most likely reflects the potential to adapt to different structures in the bound state. Fuzziness in the interaction could minimize the entropic loss upon binding, and it is even conceivable that the ligands may interchange between anchoring points within the potential hydrophobic groove of the RST domain. The bipartite architecture of the SLiM may also allow for elasticity thus making variation in structural context possible. Because no current data can determine whether the interactions are carried out through conformer selection or induced fit, Fig. 8 allows for both possibilities, as indicated in the middle of the model by the equilibrium between different conformations of the free ligand. This type of interaction surface is also observed in the complex between the TAZ1 scaffolding domain of the transcriptional co-activator CBP/p300 and RelA of the NF-κB family ( 54 ). Part of the RelA activation domain folds upon binding and spirals through the exposed hydrophobic pockets of TAZ1. In this way, it anchors itself at a number of points. However, distinct peptide-binding pockets in the RST domain of RCD1 cannot be excluded at present. To summarize, although a single SLiM describes several RCD1-interacting peptides, the peptides vary structurally from fuzzy to α-helical in the complexes. The RCD1 SLiM is bipartite, and the transcription factors are likely to use at least two anchoring points for RCD1 interactions ( Fig. 8 ).

The physiological relevance of most of the RCD1-transcription factor interactions remains elusive. However, in vivo evidence for the RCD1·DREB2A interaction exists ( 18 ), and both large scale and specific studies are suggestive of functional RCD1-transcription factor interactions in specific networks. RCD1 plays multiple roles in stress responses, including reactive oxygen species responses, and in the senescence process ( 14 , 15 , 18 ). Accordingly, Gene Ontology analysis of the RCD1-interacting transcription factors revealed enrichment for both abiotic stress responses and developmental processes ( Table 4 ). Recently, the expression profiles of DREB2A , ANAC013, and ANAC046 were shown to be similar in response to various hormone and stress treatments ( 17 ), in accordance with previous focused studies demonstrating a role of these transcription factors in regulation of stress responses and senescence ( 16 , 19 , 20 , 55 ). In this study, two new RCD1 interaction partners, ANAC016 and ANAC017, were identified. During drought stress, ANAC016 functions as a regulator of leaf senescence through cross-talk with salt and oxidative stress-responsive pathways ( 56 ), and ANAC017 mediates H 2 O 2 stress signaling ( 25 ). This means that the transcription factors of the RCD1 interactome are involved in stress-related signaling and senescence similarly to RCD1. Future experiments will show whether the transcription factors compete for RCD1 binding under stress conditions.

Many questions remain to be answered for the RCD1-transcription factor interactome. In addition to relating function to evolutionary conservation of SLiMs, the relevance of their occurrence in proteins needs to be characterized in the appropriate model organisms using a combination of high and low throughput methods to assess their functionality. This work has laid the foundation for such studies and has shown that by combining multiple techniques addressing different protein features, important novel SLiMs in IDRs can be discovered.
