---
pmid: '32673164'
title: A novel mechanism for the retention of Golgi membrane proteins mediated by
  the Bre5p/Ubp3p deubiquitinase complex.
authors:
- Wang P
- Ye Z
- Banfield DK
journal: Mol Biol Cell
year: '2020'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC7530903
doi: 10.1091/mbc.E20-03-0168
---

# A novel mechanism for the retention of Golgi membrane proteins mediated by the Bre5p/Ubp3p deubiquitinase complex.
**Authors:** Wang P, Ye Z, Banfield DK
**Journal:** Mol Biol Cell (2020)
**DOI:** [10.1091/mbc.E20-03-0168](https://doi.org/10.1091/mbc.E20-03-0168)
**PMC:** [PMC7530903](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7530903/)

## Abstract

1. Mol Biol Cell. 2020 Sep 1;31(19):2139-2155. doi: 10.1091/mbc.E20-03-0168. Epub
 2020 Jul 16.

A novel mechanism for the retention of Golgi membrane proteins mediated by the 
Bre5p/Ubp3p deubiquitinase complex.

Wang P(1), Ye Z(1), Banfield DK(1).

Author information:
(1)Division of Life Science, The Hong Kong University of Science and Technology, 
Clear Water Bay, Kowloon, Hong Kong, China.

The mechanisms employed in the retention of Golgi resident membrane proteins are 
diverse and include features such as the composition and length of the protein's 
transmembrane domain and motifs that mediate direct or indirect associations 
with COPI-coatomer. However, in sum the current compendium of mechanisms cannot 
account for the localization of all Golgi membrane proteins, and this is 
particularly the case for proteins such as the glycosyltransferases. Here we 
describe a novel mechanism that mediates the steady-state retention of a subset 
of glycosyltransferases in the Golgi of budding yeast cells. This mechanism is 
mediated by a deubiquitinase complex composed of Bre5p and Ubp3p. We show that 
in the absence of this deubiquitinase certain glycosyltransferases are 
mislocalized to the vacuole, where they are degraded. We also show that 
Bre5p/Ubp3p clients bind to COPI-coatomer via a series of positively charged 
amino acids in their cytoplasmically exposed N-termini. Furthermore, we identify 
two proteins (Ktr3p and Mnn4p) that show a requirement for both Bre5p/Ubp3p as 
well as the COPI-coatomer-affiliated sorting receptor Vps74p. We also establish 
that some proteins show a nutrient-dependent role for Vps74p in their Golgi 
retention. This study expands the repertoire of mechanisms mediating the 
retention of Golgi membrane proteins.

DOI: 10.1091/mbc.E20-03-0168
PMCID: PMC7530903
PMID: 32673164 [Indexed for MEDLINE]

## Full Text

INTRODUCTIONThe Golgi serves as the protein sorting hub of the cell. It is estimated that up to 20% of the proteins produced by a typical eukaryotic cell transit this organelle—many of them substrates for posttranslational modifications carried out by Golgi resident glycosyltransferases (GTs) and glycosidases. Golgi resident enzymes display gradient distributions across subcompartments of this organelle—termed cisternae—and the steady-state localization of a particular enzyme denotes its position in the sequential modification of its substrate. Given the substantial flux of proteins through this organelle, cells have evolved a means to ensure that its Golgi resident proteins are retained in the cisternae in which they function. Thus, ascertaining the mechanism(s) by which Golgi resident proteins are localized is instrumental to informing our understanding of the biogenesis and maintenance of this organelle.Initial studies on the localization of Golgi resident membrane proteins focused on the GTs and glycosidases, many of which are type II membrane proteins that share a similar modular organization consisting of an N-terminal cytoplasmically exposed region (NTD), a single membrane spanning segment (TMD), a spacer/stalk region, followed by the enzymatic domain (Tu and Banfield, 2010; Banfield, 2011). Such studies established that various features of an enzyme could account for its retention in the Golgi and moreover that these features varied from one protein to another (Tu and Banfield, 2010; Banfield, 2011).More recently a protein termed Vps74p (in yeast)/GOLPH3 (in mammal cells) was proposed to function as a retention factor for certain Golgi resident enzymes (Schmitz et al., 2008; Tu et al., 2008, 2012; Ali et al., 2012; Eckert et al., 2014; Pereira et al., 2014). In yeast cells Vps74p was shown to bind to a motif in the NTD of several enzymes. Additionally, Vps74p also contains a COPI-coatomer binding motif-amino acid substitutions in which result in loss of Golgi enzyme retention (Tu et al., 2012). Therefore, at least in yeast cells, Vps74p appears to function as a sorting/retention receptor for several—but not all—Golgi resident enzymes (Schmitz et al., 2008; Tu et al., 2008). To identify additional retention mechanisms for GTs, we carried out a genetic screen using a temperature-sensitive allele of VPS74. From this screen we identified BRE5 and UBP3. BRE5 and UBP3 encode a cofactor and protease, respectively, and the Bre5p/Ubp3p deubiquitination complex is known to play a role in trafficking between the endoplasmic reticulum (ER) and Golgi by deubiquitinating COPII and COPI vesicle coat constituents (Cohen et al., 2003a,b). Here we report that cells lacking BRE5 or UBP3 show a selective loss of at least four Golgi-resident proteins—two GTs (Och1p, Ktr3p), Mnn4p (regulator of the GT Mnn6p), and Mnn11p (a subunit of a Golgi resident mannosyltransferase complex). Moreover, Ktr3p and Mnn4p employ a dual retention mechanism that requires both Bre5p/Ubp3p as well as Vps74p for their robust Golgi retention. The mechanism by which Bre5p/Ubp3p influences Golgi membrane protein retention appears to be mediated by interactions with the COPI coat as both Ktr3p and Mnn4p bind to COPI-coatomer via basic motifs in their respective NTDs. Our findings reveal an unexpected level of complexity in the modes of retention of Golgi resident membrane proteins.

INTRODUCTION

The Golgi serves as the protein sorting hub of the cell. It is estimated that up to 20% of the proteins produced by a typical eukaryotic cell transit this organelle—many of them substrates for posttranslational modifications carried out by Golgi resident glycosyltransferases (GTs) and glycosidases. Golgi resident enzymes display gradient distributions across subcompartments of this organelle—termed cisternae—and the steady-state localization of a particular enzyme denotes its position in the sequential modification of its substrate. Given the substantial flux of proteins through this organelle, cells have evolved a means to ensure that its Golgi resident proteins are retained in the cisternae in which they function. Thus, ascertaining the mechanism(s) by which Golgi resident proteins are localized is instrumental to informing our understanding of the biogenesis and maintenance of this organelle.

Initial studies on the localization of Golgi resident membrane proteins focused on the GTs and glycosidases, many of which are type II membrane proteins that share a similar modular organization consisting of an N-terminal cytoplasmically exposed region (NTD), a single membrane spanning segment (TMD), a spacer/stalk region, followed by the enzymatic domain (Tu and Banfield, 2010; Banfield, 2011). Such studies established that various features of an enzyme could account for its retention in the Golgi and moreover that these features varied from one protein to another (Tu and Banfield, 2010; Banfield, 2011).

More recently a protein termed Vps74p (in yeast)/GOLPH3 (in mammal cells) was proposed to function as a retention factor for certain Golgi resident enzymes (Schmitz et al., 2008; Tu et al., 2008, 2012; Ali et al., 2012; Eckert et al., 2014; Pereira et al., 2014). In yeast cells Vps74p was shown to bind to a motif in the NTD of several enzymes. Additionally, Vps74p also contains a COPI-coatomer binding motif-amino acid substitutions in which result in loss of Golgi enzyme retention (Tu et al., 2012). Therefore, at least in yeast cells, Vps74p appears to function as a sorting/retention receptor for several—but not all—Golgi resident enzymes (Schmitz et al., 2008; Tu et al., 2008). To identify additional retention mechanisms for GTs, we carried out a genetic screen using a temperature-sensitive allele of VPS74. From this screen we identified BRE5 and UBP3. BRE5 and UBP3 encode a cofactor and protease, respectively, and the Bre5p/Ubp3p deubiquitination complex is known to play a role in trafficking between the endoplasmic reticulum (ER) and Golgi by deubiquitinating COPII and COPI vesicle coat constituents (Cohen et al., 2003a,b). Here we report that cells lacking BRE5 or UBP3 show a selective loss of at least four Golgi-resident proteins—two GTs (Och1p, Ktr3p), Mnn4p (regulator of the GT Mnn6p), and Mnn11p (a subunit of a Golgi resident mannosyltransferase complex). Moreover, Ktr3p and Mnn4p employ a dual retention mechanism that requires both Bre5p/Ubp3p as well as Vps74p for their robust Golgi retention. The mechanism by which Bre5p/Ubp3p influences Golgi membrane protein retention appears to be mediated by interactions with the COPI coat as both Ktr3p and Mnn4p bind to COPI-coatomer via basic motifs in their respective NTDs. Our findings reveal an unexpected level of complexity in the modes of retention of Golgi resident membrane proteins.

RESULTSCells lacking BRE5 or UBP3 show genetic interactions with VPS74 as well as synthetic cell wall defectsThe majority of GTs that reside in the Golgi at steady state are type II membrane proteins composed of a short cytoplasmically oriented N-terminus (NTD), a single transmembrane spanning sequence (TMD) followed by a stem region and enzymatic domain (Tu and Banfield, 2010). Our studies in yeast have revealed that some GTs contain a motif in their NTDs that mediates binding to Vps74p—a soluble protein that binds PI4P (phosphatidylinositol-4-phosphate) and COPI-coatomer (Tu et al., 2012; Wood et al., 2012). However, the Vps74p-dependent mechanism does not account for the retention of all yeast GTs (see Table 1). To identify additional factors required for the retention of GTs in the Golgi, we isolated dosage suppressors of a temperature-sensitive allele of VPS74 (SARY3739; for a list of yeast strains used in this study see Table 2). From this screen we isolated BRE5, UBP3, and DCR2 (Figure 1A) as well as a number of genes with known roles in protein trafficking (TLG1, GOT1, YIP3, SNF7, VPS21, and VMA5) and genes encoding membrane protein residents of the Golgi (VRG4 and KEX1). These genetic data are consistent with the role of Vps74p in the Golgi (Schmitz et al., 2008; Tu et al., 2008) (for a list of plasmids used in this study see Table 3). DCR2 encodes presumptive protein phosphatase (Guo and Polymenis, 2006; Pathak et al., 2007). For reasons that will be explained in detail elsewhere (unpublished data), vps74Δ ted1Δ cells are not viable but cells expressing a mutant of vps74 designated vps74-1 in which the altered protein is defective in binding to coatomer (vps74p(R6A,R7A,R8A); Tu et al., 2012) are temperature-sensitive for growth in ted1Δ cells (SARY3739; see Table 2). The relationship between vps74-1 ted1Δ and DCR2 will also be described elsewhere (unpublished data). BRE5 and UBP3 encode components of a nonessential deubiquitination complex that was previously shown to be involved in both ER–Golgi as well as Golgi–ER trafficking in yeast (Cohen et al., 2003a,b). Cells lacking VPS74 and UBP3 or VPS74 and BRE5 (hereafter referred to as vps74Δ ubp3Δ cells or vps74Δ bre5Δ cells, respectively) are temperature-sensitive for growth, and the temperature-sensitive phenotype is osmoremedial, suggesting that these double mutants may have defective cell walls—a phenotype that correlates with glycosylation deficiencies (Figure 1B). Consistent with these data, we also found that vps74Δ and ubp3Δ cells are sensitive to calcofluor white and, importantly, that vps74Δ ubp3Δ cells displayed a synthetic sensitivity to calcofluor white—another hallmark of mutants with additive defects in glycosylation (Figure 1C).TABLE 1:Amino acid sequence of the N-terminal cytoplasmic region (NTD) of some yeast Golgi type II membrane proteins.Red font denotes Vps74p-binding motifs. N- and O- denote N- and O-glycosylation, respectively.aMunro, 2001; b
Lussier et al., 1996; c
Yoko-o et al., 2003; d
Lussier et al., 1997; eSaccharomyces genome database (SGD, yeastgenome.org); f
Lussier et al., 1999; g
Romero et al., 1999.TABLE 2:Yeast strains used in this study.NameGenotypeSourceBY4741MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0Lab collectionSARY3739MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 vps74::kanMX4 ted1::LEU2 vps74R6A,7A,8A-pRS416Lab collectionY04208MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 vps74::kanMX4EUROSCARFSARY5644MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 bre5::loxPThis studySARY5646MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 ubp3::loxPThis studySARY5275MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 vps74::kanMX4 bre5::loxPThis studySARY5277MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 vps74::kanMX4 ubp3::loxPThis studySARY5624MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 ktr3::loxP-HIS3-loxPThis studyY01420MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 kre2::kanMX4EUROSCARFSARY5794MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 kre2::kanMX4 ktr3::loxP-HIS3-loxPThis studySARY5871MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 ktr3::KTR3-mNeon-loxP-HIS3-loxPThis studySARY5877MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 vps74::kanMX4 ktr3::KTR3-mNeon-loxP-HIS3-loxPThis studySARY5873MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 bre5::loxP ktr3::KTR3-mNeon-loxP-HIS3-loxPThis studySARY5875MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 ubp3::loxP ktr3::KTR3-mNeon-loxP-HIS3-loxPThis studySARY5770MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 ktr3::KTR3-9myc-loxP-HIS3-loxPThis studySARY5880MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 vps74::kanMX4 ktr3::KTR3-9myc-loxP-HIS3-loxPThis studySARY5772MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 bre5::loxP ktr3::KTR3-9myc-loxP-HIS3-loxPThis studySARY5774MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 ubp3::loxP ktr3::KTR3-9myc-loxP-HIS3-loxPThis studySARY5925MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 vps74::kanMX4 bre5::loxP ktr3::KTR3-9myc-loxP-HIS3-loxPThis studySARY6093MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 och1::OCH1-mNeon-loxP-HIS3-loxPThis studySARY6095MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 vps74::kanMX4 och1::OCH1-mNeon-loxP-HIS3-loxPThis studySARY6096MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 bre5::loxP och1::OCH1-mNeon-loxP-HIS3-loxPThis studySARY6098MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 ubp3::loxP och1::OCH1-mNeon-loxP-HIS3-loxPThis studySARY6253MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 och1::OCH1-9myc-loxP-HIS3-loxPThis studySARY6255MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 vps74::kanMX4 och1::OCH1-9myc-loxP-HIS3-loxPThis studySARY6257MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 bre5::loxP och1::OCH1-9myc-loxP-HIS3-loxPThis studySARY6259MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 ubp3::loxP och1::OCH1-9myc-loxP-HIS3-loxPThis studySARY6028MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 mnn11::MNN11-mNeon-loxP-HIS3-loxPThis studySARY6030MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 vps74::kanMX4 mnn11::MNN11-mNeon-loxP-HIS3-loxPThis studySARY6031MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 bre5::loxP mnn11::MNN11-mNeon-loxP-HIS3-loxPThis studySARY6033MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 ubp3::loxP mnn11::MNN11-mNeon-loxP-HIS3-loxPThis studySARY6243MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 mnn11::MNN11-9myc-loxP-HIS3-loxPThis studySARY6245MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 vps74::kanMX4 mnn11::MNN11-9myc-loxP-HIS3-loxPThis studySARY6247MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 bre5::loxP mnn11::MNN11-9myc-loxP-HIS3-loxPThis studySARY6249MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 ubp3::loxP mnn11::MNN11-9myc-loxP-HIS3-loxPThis studySARY6994MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 mnn4::MNN4-9myc-loxP-HIS3-loxPThis studySARY6996MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 vps74::kanMX4 mnn4::MNN4-9myc-loxP-HIS3-loxPThis studySARY6998MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 bre5::loxP mnn4::MNN4-9myc-loxP-HIS3-loxPThis studySARY7000MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 ubp3::loxP mnn4::MNN4-9myc-loxP-HIS3-loxPThis studySARY7002MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 vps74::kanMX4 bre5::loxP mnn4::MNN4-9myc-loxP-HIS3-loxPThis studySARY6490MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 anp1::ANP1-mNeon-loxP-HIS3-loxPThis studySARY6560MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 vps74::kanMX4 anp1::ANP1-mNeon-loxP-HIS3-loxPThis studySARY6562MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 bre5::loxP anp1::ANP1-mNeon-loxP-HIS3-loxPThis studySARY6051MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 mnn2::MNN2-mNeon-loxP-HIS3-loxPThis studySARY6053MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 vps74::kanMX4 mnn2::MNN2-mNeon-loxP-HIS3-loxPThis studySARY6055MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 bre5::loxP mnn2::MNN2-mNeon-loxP-HIS3-loxPThis studySARY6057MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 ubp3::loxP mnn2::MNN2-mNeon-loxP-HIS3-loxPThis studySARY5984MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 hoc1::HOC1-mNeon-loxP-HIS3-loxPThis studySARY5986MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 vps74::kanMX4 hoc1::HOC1-mNeon-loxP-HIS3-loxPThis studySARY5988MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 bre5::loxP hoc1::HOC1-mNeon-loxP-HIS3-loxPThis studySARY5990MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 ubp3::loxP hoc1::HOC1-mNeon-loxP-HIS3-loxPThis studySARY6021MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 mnn5::MNN5-mNeon-loxP-HIS3-loxPThis studySARY6023MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 vps74::kanMX4 mnn5::MNN5-mNeon-loxP-HIS3-loxPThis studySARY6024MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 bre5::loxP mnn5::MNN5-mNeon-loxP-HIS3-loxPThis studySARY6026MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 ubp3::loxP mnn5::MNN5-mNeon-loxP-HIS3-loxPThis studySARY5947MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 mnn9::MNN9-mNeon-loxP-HIS3-loxPThis studySARY5949MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 vps74::kanMX4 mnn9::MNN9-mNeon-loxP-HIS3-loxPThis studySARY5951MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 bre5::loxP mnn9::MNN9-mNeon-loxP-HIS3-loxPThis studySARY5953MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 ubp3::loxP mnn9::MNN9-mNeon-loxP-HIS3-loxPThis studySARY6086MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 kex1::KEX1-mNeon-loxP-HIS3-loxPThis studySARY6088MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 vps74::kanMX4 kex1::KEX1-mNeon-loxP-HIS3-loxPThis studySARY6090MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 bre5::loxP kex1::KEX1-mNeon-loxP-HIS3-loxPThis studySARY6092MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 ubp3::loxP kex1::KEX1-mNeon-loxP-HIS3-loxPThis studySARY7092MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 hoc1::HOC1-mNeon-loxP-HIS3-loxP sec27::sec27-1This studySARY7093MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 hoc1::HOC1-mNeon-loxP-HIS3-loxP sec21::sec21-1This studySARY7094MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 hoc1::HOC1-mNeon-loxP-HIS3-loxP ret1::ret1-1This studyFIGURE 1:BRE5 and UBP3 show synthetic genetic and phenotypic interactions with VPS74. (A) BRE5 and UBP3 are high-copy-number dosage suppressors of the temperature-sensitive growth phenotype of vps74-1 ted1Δ cells. vps74-1 ted1Δ cells (SARY3739; see Table 2) were transformed with the indicated plasmids, and transformants were grown to early–log phase before being subjected to 10-fold serial dilutions and aliquoting onto nutrient agar plates. Plates were incubated at the indicated temperatures for 36 h prior to being photographed. (B) vps74∆ bre5∆ cells and vps74∆ ubp3∆ cells show synthetic growth phenotypes. The indicated yeast strains were treated as described in A. (C) vps74∆ ubp3∆ cells show synthetic increased sensitivity to calcofluor white. The indicated yeast strains were treated as described in A.TABLE 3:Plasmids used in this study.PlasmidDescriptionSourcepRS413ARS/CEN plasmid with Saccharomyces cerevisiae HIS3 markerEUROSCARFpRS415ARS/CEN plasmid with S. cerevisiae LEU2 markerEUROSCARFpRS416ARS/CEN plasmid with S. cerevisiae URA3 markerEUROSCARFpRS4252µ plasmid with S. cerevisiae LEU2 markerEUROSCARFpRS4262µ plasmid with S. cerevisiae URA3 markerEUROSCARFgVPS74-pRS416The DNA fragment encoding VPS74 under the control of its own promoter was cloned as an XhoI-BamHI fragment into pRS416Lab collectiongvps74-6-8A-pRS416The DNA fragment encoding vps74 6-8A under the control of its own promoter was cloned as an XhoI-BamHI fragment into pRS416Lab collectionBRE5-MoBY2µ, LEU2 plasmid containing BRE5 ORF from Molecular Barcoded Yeast (MoBY) ORF libraryLab collectionUBP3-MoBY2µ, LEU2 plasmid containing UBP3 ORF from MoBY ORF libraryLab collectionGLO3-MoBY2µ, LEU2 plasmid containing GLO3 ORF from MoBY ORF libraryLab collectionSLY1-MoBY2µ, LEU2 plasmid containing SLY1 ORF from MoBY ORF libraryLab collectionYPT1-MoBY2µ, LEU2 plasmid containing YPT1 ORF from MoBY ORF libraryLab collectionYPT6-MoBY2µ, LEU2 plasmid containing YPT6 ORF from MoBY ORF libraryLab collectionRUD3-MoBY2µ, LEU2 plasmid containing RUD3 ORF from MoBY ORF libraryLab collectionGCS1-pRS426The DNA fragment encoding GCS1 under the control of its own promoter was cloned as an XhoI-SacII fragment into pRS426Lab collectiongcs1-R54A-pRS426The DNA fragment encoding gcs1 R54A under the control of its own promoter was cloned as an XhoI-SacII fragment into pRS426Lab collectiongcs1-H59A-pRS426The DNA fragment encoding gcs1 H59A under the control of its own promoter was cloned as an XhoI-SacII fragment into pRS426Lab collectionGLO3-pRS426The DNA fragment encoding GLO3 under the control of its own promoter was cloned as an XhoI-SacII fragment into pRS426Lab collectionglo3-R59A pRS426The DNA fragment encoding glo3 R59A under the control of its own promoter was cloned as an XhoI-SacII fragment into pRS426Lab collectionglo3-1-375 pRS426The DNA fragment encoding glo3 1-375 (amino acids 1–375) under the control of its own promoter was cloned as an XhoI-SacII fragment into pRS426Lab collectionBRE5-pRS416The DNA fragment encoding BRE5 under the control of its own promoter was cloned as a BamHI-SacII fragment into pRS416This studybre5-146-515 pRS416The DNA fragment encoding bre5 146-515 (amino acids 146–515) under the control of its own promoter was cloned as an XhoI-SacII fragment into pRS416This studyUBP3-pRS416The DNA fragment encoding UBP3 under the control of its own promoter was cloned as a BamHI-SacII fragment into pRS416This studyubp3-C469A-pRS416The DNA fragment encoding ubp3 C469A under the control of its own promoter was cloned as a BamHI-SacII fragment into pRS416This studyTPI-MNN4CT-­mNeon-pRS416The DNA fragment encoding MNN4CT (amino acids 1–49) was cloned as an EcoRI-BamHI fragment C-tagged with mNeonGreen into pRS416; TPI1 promoterThis studyTPI-KTR7CT-mNeon-pRS416The DNA fragment encoding KTR7CT (amino acids 1–44) was cloned as an EcoRI-BamHI fragment C-tagged with mNeonGreen into pRS416; TPI1 promoterThis studyTPI-ALG5CT-mNeon-pRS416The DNA fragment encoding ALG5CT (amino acids 1–34) was cloned as an EcoRI-BamHI fragment C-tagged with mNeonGreen into pRS416; TPI1 promoterThis studymNeon-Sed5 pRS416The DNA fragment encoding mNeonGreen-SED5 under the control of SED5 ­promoter was cloned as a BamHI-NotI fragment into pRS416Lab collectionpGEX-2TpTAC-driven N-terminal GST fusion expression vectorPharmaciaKTR3c-pGEX-2TThe DNA fragment encoding KTR3 cytoplasmic tail (amino acids 1–25) was cloned as a BamHI-EcoRI fragment into pGEX-2TThis studyKTR3c 6-8Q-pGEX-2TThe DNA fragment encoding KTR3 cytoplasmic tail (amino acids 1–25 with 6–8 substituted with glutamines) was cloned as a BamHI-EcoRI fragment into pGEX-2TThis studyKTR3c 18-19Q-pGEX-2TThe DNA fragment encoding KTR3 cytoplasmic tail (amino acids 1–25 with 18 and 19 substituted with glutamines) was cloned as a BamHI-EcoRI fragment into pGEX-2TThis studyMNN4c-pGEX-2TThe DNA fragment encoding MNN4 cytoplasmic tail (amino acids 1–27) was cloned as a BamHI-EcoRI fragment into pGEX-2TThis studyMNN4c 13-15A-pGEX-2TThe DNA fragment encoding MNN4 cytoplasmic tail (amino acids 1–27 with 13–15 substituted with alanines) was cloned as a BamHI-EcoRI fragment into pGEX-2TThis studyMNN4c 8,10-12Q-pGEX-2TThe DNA fragment encoding MNN4 cytoplasmic tail (amino acids 1–27 with 8 and 10–12 substituted with glutamines) was cloned as a BamHI-EcoRI fragment into pGEX-2TThis studyMNN4c 19,21-22Q-pGEX-2TThe DNA fragment encoding MNN4 cytoplasmic tail (amino acids 1–27 with 19, 21, and 22 substituted with alanines) was cloned as a BamHI-EcoRI fragment into pGEX-2TThis studyMNN4c 8,13-15,19,21-22Q-pGEX-2TThe DNA fragment encoding MNN4 cytoplasmic tail (amino acids 1–27 with 8, 13–15, 19, 21, and 22 substituted with alanines) was cloned as a BamHI-EcoRI fragment into pGEX-2TThis studyOCH1c-pGEX-2TThe DNA fragment encoding OCH1 cytoplasmic tail (amino acids 1–15) was cloned as a BamHI-EcoRI fragment into pGEX-2TThis studyMNN11c-pGEX-2TThe DNA fragment encoding MNN11 cytoplasmic tail (amino acids 1–32) was cloned as a BamHI-EcoRI fragment into pGEX-2TThis studyGST-KKXXThe DNA fragment encoding WBP1 cytoplasmic tail (amino acids 421–430) was cloned as a BamHI-EcoRI fragment into pGEX-2TLab collectionpET-HBacterial vector for expressing thioredoxin fusion proteins with an enterokinase siteLab collectionGST-pET-HDNA fragment encoding GST was cloned as a BamHI-EcoRI fragment into pET-HLab collectionKTR3c-GST-pET-HThe DNA fragment encoding KTR3 cytoplasmic tail (amino acids 1–25) was cloned as a BamHI-EcoRI fragment into GST-pET-HThis studyMNN4c-GST-pET-HThe DNA fragment encoding MNN4 cytoplasmic tail (amino acids 1–27) was cloned as a BamHI-EcoRI fragment into GST-pET-HThis studyOCH1c-GST-pET-HThe DNA fragment encoding OCH1 cytoplasmic tail (amino acids 1–15) was cloned as a BamHI-EcoRI fragment into GST-pET-HThis studyMNN11c-GST-pET-HKTR3-mNeon-pRS416KTR3 6-8Q-mNeon-pRS416KTR3 18-19Q-mNeon-pRS416The DNA fragment encoding MNN11 cytoplasmic tail (amino acids 1–32) was cloned as a BamHI-EcoRI fragment into GST-pET-HThe DNA fragment encoding KTR3-mNeon was Gibson assembled into pRS416The DNA fragment encoding KTR3 6-8Q-mNeon was Gibson assembled into pRS416The DNA fragment encoding KTR3 18-19Q-mNeon was Gibson assembled into pRS416This studyThis studyThis studyThis studyvps74Δ bre5Δ and vps74Δ ubp3Δ cells show synthetic glycosylation defects of Gas1p and CPYTo investigate the implications of the defects displayed in vps74Δ ubp3Δ and vps74Δ bre5Δ cells, we examined the processing of two glycoproteins—a glycosylphosphatidylinositol (GPI)-anchored protein termed Gas1p and the vacuolar protease carboxypeptidase Y (CPY). While CPY is exclusively N-glycosylated (Winther et al., 1991; Munro, 2001), Gas1p is subject to both N- and O-glycosylation; however, the preponderance of glycosylation events (O-­glycosylation) occur within a 36-amino-acid serine-rich region (Gatti et al., 1994; Popolo and Vai, 1999; Neubert and Strahl, 2016). Thus, CPY and Gas1p serve as useful reporter proteins of N- and O-­glycosylation anomalies in yeast cells. Both Gas1p and CPY are translocated into the lumen of the ER, where they undergo “core” glycosylation. The ER-modified forms of these proteins are referred to as pGas1p and p1CPY. Following their export from the ER, both pGas1p and p1CPY are extensively glycosylated in the Golgi (resulting in an alteration of their apparent molecular weights), after which the so-called mature form of Gas1p (mGas1p) is delivered to the cell surface while Golgi-modified CPY (designated p2CPY) is delivered to the vacuole, where limited proteolysis converts p2CPY to its mature form (mCPY). Defects in the transportation and glycosylation of either protein can be readily assessed by SDS–PAGE and immunoblotting. Consistent with a previous report, vps74Δ cells showed hypoglycosylation of mGas1p (Tu et al., 2008; Figure 2A). Although bre5Δ and ubp3Δ cells did not show any perceivable Gas1p glycosylation defect, vps74Δ bre5Δ and vps74Δ ubp3Δ cells showed what appears to be synthetic hypoglycosylation of the post-ER form of Gas1p as judged by the altered gel mobility of the protein on SDS–PAGE gels (mGas1p; Figure 2A). Similarly, presumptive glycosylation defects were also observed when the processing of CPY was examined. The electrophoretic mobility of the mature form of CPY (mCPY) seen from bre5Δ and ubp3Δ cell extracts was similar to that observed in vps74Δ cells, suggesting that, compared with wild-type (WT) cells, CPY had an altered glycosylation profile in these mutants. The accumulation of p1CPY evident in bre5Δ, ubp3Δ, vps74Δ bre5Δ, and vps74Δ ubp3Δ cells is consistent with earlier reports that described a role for Bre5p/Ubp3p in ER–Golgi trafficking (Cohen et al., 2003a). While we cannot rule out the possibility that Bre5p/Ubp3p affects the trafficking of CPY and Gas1p, given the apparent differences in the gel mobilities and glycosylation profiles of both proteins in vps74Δ bre5Δ and vps74Δ ubp3Δ cells, and in particular the absence of any substantial synthetic accumulation of p2CPY, we considered it likely that together with Vps74p, Bre5p/Ubp3p in some way affects the glycosylation of Gas1p and CPY.FIGURE 2:BRE5 and UBP3 are required for the retention of Ktr3p in the Golgi. (A) vps74Δ bre5Δ cells and vps74Δ ubp3Δ cells show synthetic glycosylation defects. The indicated yeast strains were grown to early–log phase, and proteins from WCEs were resolved by SDS–PAGE before being immunoblotted with antibodies against Gas1p, proteinase C (CPY), and 3-phosphoglycerate kinase (Pgk1p). Pgk1p serves as a WCE loading control. pGas1p and mGas1p denote the ER and post-ER forms of Gas1p, respectively. p1CPY, p2CPY, and mCPY denote the ER form, Golgi form, and vacuolar form of CPY, respectively. MW denotes the relative positions of molecular weight standards. IB denotes primary antibody used for immunoblotting. (B) Depiction of the sequential order of action in the Golgi of yeast mannosyltransferases on Gas1p (see also Table 2). (C) kre2∆ ktr3∆ and vps74∆ ktr3∆ cells phenocopy the Gas1p glycosylation deficiency observed in vps74∆ bre5∆ and vps74∆ ubp3∆ cells. The indicated yeast strains were grown to early–log phase, and WCEs were resolved by SDS–PAGE before being immunoblotted with antibodies against Gas1p, proteinase C (CPY), and 3-phosphoglycerate kinase (Pgk1p). Pgk1p serves as a WCE loading control. (D) Ktr3p is mislocalized to the vacuole in bre5Δ or ubp3Δ cells. The DNA sequence encoding mNeon was integrated in frame at the 3′end of the coding sequence for KTR3 in each of the indicated strains. Yeast cells were grown in YEPD medium to early–log phase, stained with FM4-64, and photographed using an epifluorescence microscope (see Materials and Methods). Scale bar 5 µm. (E) The steady-state level of Ktr3p is reduced in vps74∆ bre5∆ cells. Proteins (derived from WCEs of the indicated strains) were resolved by SDS–PAGE and immunoblotted for Ktr3-9myc (see Materials and Methods) using a monoclonal antibody against the myc epitope. Quantification of the relative amounts of Ktr3-9myc in each strain (% relative to WT) is indicated below the immunoblot. Pgk1p serves as a loading control. (F) Ktr3-mNeon is not mislocalized to the vacuole in vps74Δ cells grown in SD medium and exclusively mislocalized to the vacuole in bre5∆ and ubp3∆ cells. (G) Truncation and amino acid substitutions that disrupt the activity of the Bre5p/Ubp3p deubiquitinase result in mislocalization of Ktr3-mNeon to the vacuole. (H) Truncation and amino acid substitutions that disrupt the activity of the Bre5p/Ubp3p deubiquitinase result in a reduction of the steady-state levels of Ktr3p in cells. The percentage of Ktr3p remaining in cell lysates (relative to WT cells) is indicated below the immunoblot.If Bre5p/Ubp3p could contribute to GT retention, for example, we surmised that the deubiquitinase would likely impact enzymes that were not Vps74p clients (see Table 1), as this could explain the synthetic gel mobility phenotypes observed. We previously established that the hypoglycosylation of Gas1p observed in vps74Δ cells was due to the loss of retention of the mannosyltransferase Kre2p from the Golgi (Figure 2B) (Tu et al., 2008). On the basis of the observation that Gas1p from vps74Δ bre5Δ and vps74Δ ubp3Δ cells migrates more quickly than Gas1p from vps74Δ cells, and the fact that Gas1p is extensively O-glycosylated (rather than N-glycosylated; Gatti et al., 1994), we next considered the prospect that the synthetic Gas1p glycosylation defect evident in vps74Δ bre5Δ and vps74Δ ubp3Δ cells (Figure 2A) might be due to loss of retention of enzymes that function concomitantly with Kre2p in the O-glycosylation of Gas1p (Figure 2B) as no further reduction in Kre2p levels was evident in vps74Δ bre5Δ cells compared with vps74Δ cells (Supplemental Figure S1A). As Ktr1p and Ktr3p are not known to be Vps74p clients (see Table 1), we examined the processing of Gas1p and CPY in ktr1Δ, ktr3Δ, vps74Δ ktr3Δ, and vps74Δ ktr1Δ cells (Figure 2C and Supplemental Figure S1B). While the processing of Gas1p and CPY in ktr3Δ or ktr1Δ cells was indistinguishable from that in WT cells, vps74Δ ktr3Δ and kre2Δ ktr3Δ cells showed a Gas1p glycosylation defect very much like that observed in vps74Δ bre5Δ cells and vps74Δ ubp3Δ cells, whereas vps74Δ ktr1Δ cells did not (Figure 2C and Supplemental Figure S1B). On the basis of these data, we surmised that Ktr3p might be mislocalized in bre5Δ or ubp3Δ cells, rather than Ktr1p or Kre2p (Figure 2C).To establish whether Ktr3p was mislocalized and degraded in the vacuole in bre5Δ and ubp3Δ cells, we examined the fate of Ktr3-mNeon in these mutants (see Table 2). As expected, Ktr3-mNeon was localized to Golgi-like puncta in WT cells and this was also the case—to some degree—in vps74Δ cells; however, in bre5Δ and ubp3Δ cells the protein was exclusively localized to the lumen of the vacuole (Figure 2D). Consistent with these observations, we found that the steady-state levels of Ktr3-9myc (see Table 2) were reduced in bre5Δ cells (85% of WT levels) (Figure 2E). The findings for vps74 cells were rather surprising as the NTD of Ktr3p does not contain a canonical Vps74p-binding motif (see Table 1). Rather puzzlingly, the steady-state levels of Ktr3-9myc were increased in vps74Δ cells (111% of WT levels), which is a finding that is inconsistent with observations made with Ktr3-mNeon in living cells. However, consistent with the data presented in Figure 2, A and C, vps74Δ bre5Δ cells showed a synthetic reduction in the steady-state levels of Ktr3-9myc (46% of WT levels; Figure 2E).For technical reasons the cells used for immunoblotting were grown in synthetic defined (SD) medium while cells for fluorescence microscopy were grown in yeast extract peptone dextrose (YEPD) medium (see Materials and Methods). To ascertain whether the differences in the distribution and steady-state levels of Ktr3p could be accounted for based on the nutrient composition of the growth media used, we examined the localization of Ktr3-mNeon in cells grown in SD medium. Like WT cells, Ktr3-mNeon was found exclusively in puncta when vps74Δ cells were grown in SD medium (Figure 2F) rather than in puncta and the lumen of the vacuole, as was the case for cells grown in YEPD medium (Figure 2D), whereas as expected Ktr3-mNeon was found exclusively in the vacuole in bre5Δ and ubp3Δ cells. These data suggested that the role of Vps74p in the Golgi retention of Ktr3p was affected by the nutrient composition of the medium in which cells were grown. Moreover, the data presented thus far are consistent with a dual Golgi retention mechanism for Ktr3p—one involving Vps74p and the other Bre5p/Ubp3p.To verify that the deubiquitinase activity of Bre5p/Ubp3p was required for the retention of Ktr3p in the Golgi, we generated a form of Bre5p that was previously shown to no longer bind to Ubp3p (bre5p (146-515)) (Cohen et al., 2003a,b; Li et al., 2005) as well as a form of Ubp3p that is catalytically inactive (ubp3p(C469A)) (Cohen et al., 2003a). Cells exclusively expressing either bre5p(145-515) or ubp3p(C469A) mislocalized Ktr3-mNeon to the vacuole (Figure 2G), and the steady levels of Ktr3-9myc in cells expressing bre5p(146-515) and ubp3p(C469A) were indistinguishable from those in bre5Δ and ubp3Δ cells, respectively (Figure 2H). We therefore concluded that it is the deubiquitinase activity of Bre5p/Ubp3p that is required for the retention of Ktr3p in the Golgi.There is an apparent inconsistency in the relative amounts of Ktr3-9myc observed in bre5Δ cells in Figure 2, E and H. This discrepancy may be due to subtle differences in the growth dynamics of the cell populations used in these experiments, to differences in the amounts of protein assessed by immunoblotting, or to variations in immunoblot exposure times—as detection of chemiluminescent signals is nonlinear. In any case, the data presented in Figure 2, E and H, show a consistent reduction in the levels of Ktr3-9myc in line with our hypothesis that Ktr3p’s Golgi retention is dependent in part on Bre5p/Ubp3p.Vps74p and Bre5p/Ubp3p show both distinct and additive roles in the retention of Golgi membrane proteinsThe synthetic glycosylation phenotypes of CPY observed in vps74Δ bre5Δ and vps74Δ ubp3Δ cells (Figure 2, A and C) do not precisely phenocopy those observed for vps74Δ ktr3Δ cells. This observation suggests that in addition to Ktr3p, Bre5p/Ubp3p might also be required for the retention of other GTs and in addition Vps74p and Bre5p/Ubp3p might function independently of each other in this capacity (Figure 2, D and F). To address this proposition, we asked whether other Golgi membrane proteins that lacked canonical Vps74p-binding motifs (Och1p and Mnn11p; see Table 1) were mislocalized in bre5Δ or ubp3Δ cells but not in vps74Δ cells. As anticipated when we examined the fate of Och1-mNeon in WT and vps74Δ cells, the protein was localized to Golgi-like puncta when cells were grown in YEPD medium (Figure 3A) although—as was the case for Ktr3-mNeon in vps74Δ cells (Figure 2, D and F) —some vacuolar luminal fluorescence was also apparent for Och1-mNeon in vps74Δ cells (Figure 3A). A similar localization profile was also evident for Och1-mNeon in vps74Δ cells grown in SD medium (Figure 3A). In contrast to vps74Δ cells, in bre5Δ and ubp3Δ cells, Och1-mNeon was predominantly localized to the vacuole (Figure 3A) and the steady-state levels of Och1-9myc were reduced to 46% (bre5Δ) and 44% (ubp3Δ) of WT levels when cells were grown in SD medium (Figure 3B). In cells grown in YEPD medium the steady-state levels of Och1-9myc were 70% (bre5Δ) and 74% (ubp3Δ) of that in WT cells (Figure 3B). By contrast, the steady-state levels of Och1-9myc in vps74Δ cells were indistinguishable from those of WT cells when the cells were grown in either SD or YEPD medium (Figure 3B). The vacuolar localization observed for Och1-mNeon in vps74Δ cells grown in both YEPD and SD media is likely due to the addition of the mNeon tag to the C-terminus of Och1p as the steady-state levels of Och1-9myc did not reveal any reduction in the protein levels for this GT (Figure 3, A and B). This is in contrast with the data obtained for Ktr3p (Figure 2, D–F) and particularly for the steady-state levels of Ktr3-9myc, which were increased relative to those seen in WT cells. We speculate that the addition of the mNeon tag can subtly affect the distributions of both Och1p and Ktr3p in various genetic backgrounds and under different growth conditions. In addition, it is also possible that some GTs are more sensitive to the addition of mNeon to their C-termini and as a result are more susceptible to mislocalization. Nevertheless, the Golgi retention of Och1p appears to be largely independent of Vps74p, but rather to require Bre5p/Ubp3p, as unlike Ktr3-9myc there is no synthetic loss of Och1-9myc in vps74Δ bre5Δ cells (Figure 3B).FIGURE 3:Golgi membrane proteins show diverse retention requirements for Vps74p and Bre5p/Ubp3p. (A) Och1p is mislocalized in bre5Δ and ubp3 cells. Scale bar 5 µm. (B) The steady-state levels of Och1p are reduced in bre5Δ and ubp3 cells. Proteins from WCEs from the indicated strains were resolved by SDS–PAGE and immunostained for Och1p and Pgk1p. Pgk1p serves as a gel loading control. (C) Mnn11p is mislocalized in bre5Δ and ubp3 cells when grown in SD medium and partially mislocalized in bre5Δ cells when grown in YEPD medium, whereas in vps74Δ cells Mnn11p is mislocalized to the vacuole. Scale bar 5 µm. (D) The differential effects of growth media on the steady-state levels of Mnn11p in vps74Δ, bre5Δ, and ubp3 cells. Proteins from WCEs from the indicated strains were resolved by SDS–PAGE and immunostained for Mnn11p and Pgk1p. Pgk1p serves as a gel loading control. (E) Mnn4p is mislocalized in vps74Δ, bre5Δ, and ubp3 cells. Scale bar 5 µm. (F) vps74Δ and bre5Δ cells show a synthetic reduction in the steady-state levels of Mnn4p. Proteins from WCEs from the indicated strains were resolved by SDS–PAGE and immunostained for Mnn4p and Pgk1p. Pgk1p serves as a gel loading control. (G–L) Anp1p, Mnn2p, Mnn5p, Mnn9p, Ktr7p, and Kre2p are not mislocalized in bre5Δ or ubp3 cells but are mislocalized to the vacuole in vps74Δ cells. In panels G–L, cells were grown in YEPD medium. Scale bar 5 µm.In contrast to Och1-mNeon, Mnn11-mNeon was localized to the vacuole in vps74Δ cells and to puncta and the vacuole in bre5Δ or ubp3Δ cells when the cells were grown in YEPD medium (Figure 3C). However, when cells were grown in SD medium, Mnn11-mNeon was localized principally to puncta in vps74Δ cells and primarily to the vacuole in bre5Δ or ubp3Δ cells (Figure 3C). Complementary observations were also made when the steady-state levels of Mnn11-9myc were examined by immunoblotting (Figure 3D). Like Och1-mNeon (see above), Mnn11-mNeon also appears to be more sensitive to alterations in genetic backgrounds as well as to growth medium variation—as Mnn11-mNeon localized to puncta and the vacuole in bre5Δ and ubp3Δ cells grown in YEPD medium. Thus even though Mnn11p lacks a canonical Vps74p-binding motif (see Table 1), it nonetheless displays a nutrient-dependent requirement for Vps74p (as was also the case for Ktr3p; Figure 2), while Bre5p/Ubp3p was required for Mnn11p’s Golgi retention irrespective of the nutrient conditions under which the cells were grown (Figure 3, C and D). Interestingly, the requirement for Bre5p/Ubp3p in the retention of Och1p and Mnn11p in the Golgi is most evident when bre5Δ or ubp3Δ cells were grown in SD medium (Figure 3, B and D).We next asked whether a type II membrane protein that contained a canonical Vps74p-binding motif might exhibit both Vps74p- and Bre5p/Ubp3p-dependent Golgi retention mechanisms. To address this prospect, we selected Mnn4p (see Table 1). Unlike Mnn11p and Och1p, the addition of mNeon to the C-terminus of Mnn4p was difficult to image due to the low expression levels of the fusion protein (unpublished data). However, our previous study revealed that an in vivo reporter for the Vps74p client Kre2p—which lacked the stem and enzymatic domains of this GT (termed Kre2CT-GFP, where CT is cytoplasmic and transmembrane domain)—was localized to the ER when cells were grown in SD medium but to Golgi-like puncta when cells were grown in YEPD medium. But, crucially, Kre2CT-GFP was mislocalized to the vacuole in SD and YEPD media in vps74Δ cells (Tu et al., 2008). We therefore used a version of Mnn4p in which the N-terminal cytoplasmic domain and the transmembrane domain were expressed as a mNeon fusion protein under the control of the triose phosphate isomerase (TPI) promoter (Mnn4CT-mNeon). Similar to Kre2CT-GFP, when WT cells were grown in YEPD medium Mnn4CT-mNeon localized exclusively to puncta and to the ER as well as to Golgi-like puncta when cells were grown in SD medium (Figure 3E). However, in vps74Δ, bre5Δ, or ubp3Δ cells Mnn4CT-mNeon was found in puncta as well as in the lumen of the vacuole regardless of the nutrient conditions (Figure 3E). The steady-state levels of Mnn4-9myc were 55%, 70%, and 64% of those found in WT cells in YEPD medium for vps74Δ, bre5Δ, or ubp3Δ cells, respectively (Figure 3F). In vps74Δ bre5Δ cells, the steady-state levels of Mnn4-9myc were further reduced to 28% of that in WT cells (Figure 3F). Moreover, the steady-state levels of Mnn4-9myc were even lower in cells grown in SD medium (Figure 3F). Taken together, the data presented in Figure 3, E and F, suggest that Mnn4p can use a dual mechanism for its Golgi retention and that Vps74p and Bre5p/Ubp3p play additive roles in the retention of Mnn4p in cells.The apparent capacity of Mnn4p to be a client for either Vps74p or Bre5p/Ubp3p does not appear to be a universal feature of GTs or of type II membrane proteins that contain canonical Vps74p-binding motifs, however (see Table 1). For example, the Golgi localization of Anp1p, Mnn2p, Mnn5p, and Mnn9p was unaffected in bre5Δ and ubp3Δ cells—as judged by fluorescence microscopy (Figure 3, G–J). The same was also true for Ktr7CT-mNeon and Kre2CT-mNeon (Figure 3, K and L). Similarly, a requirement for Bre5p/Ubp3p does not extend to Golgi/ER membrane proteins more generally as the steady-state localizations of Hoc1p, Sed5p, Kex1p, and Alg5p were unaffected in bre5Δ and ubp3Δ cells (Supplemental Figure S2). Although Hoc1p does not appear to be a client of Vps74p or Bre5p/Ubp3p, its Golgi retention is nonetheless dependent on COPI function as Hoc1-mNeon was mislocalized to the vacuole in temperature-sensitive COPI mutants (Supplemental Figure S3).GTs that require Bre5p/Ubp3p for their Golgi retention bind to COPI-coatomer via their NTDsWhat is the basis for the requirement for Bre5p/Ubp3p in the retention of Golgi membrane proteins? To address this question, we took a genetic approach. As vps74Δ bre5Δ cells are temperature-­sensitive for growth (Figure 1B), we sought to identify genes that could function as dosage suppressors of this phenotype. Given the relationship between COPI and Vps74p (Tu et al., 2008, 2012), we surmised that genes whose products function to modulate the function of the Golgi vesicle coat complex—COPI-coatomer—might be likely candidates. Glo3p and Gcs1p are GTPase-activating proteins (GAPs) for COPI-coatomer’s GTPase Arf1p and as such are required for the function of the COPI coat (Poon et al., 1996; Lewis et al., 2004). As anticipated, GLO3 and GCS1 were dosage suppressors of the temperature-sensitive growth phenotype of vps74Δ bre5Δ cells (Figure 4, A and B). Moreover, it is the GAP activity that is required for suppression, as amino acid substitutions to Glo3p or Gcs1p that render these proteins catalytically inactive no longer functioned as suppressors (Figure 4B). These findings are consistent with those of Cohen et al. (2003b), who showed that cells lacking both BRE5 and GLO3 were inviable. It may therefore be the case that the activity or steady-state levels of Arf1-GAPs are compromised in vps74Δ bre5Δ cells or that the activity of COPI-coatomer itself is compromised in some manner.FIGURE 4:COPI plays a direct role in the retention of Bre5p/Ubp3p clients in the Golgi. (A) The Arf1 GAP Glo3p is a dosage suppressor of the temperature-sensitive growth phenotype of vps74Δ bre5Δ cells. (B) The GAP activity of Glo3p and Gcs1p is required to suppress the temperature-sensitive growth phenotype of vps74Δ bre5Δ cells. For A and B, 10-fold serial dilutions of the indicated yeast strains were spotted on nutrient agar plates, which were then placed in a 25°C or 37°C incubator for 48 h prior to being photographed. (C, D) The N-terminal cytoplasmic domains (NTD) of some Golgi resident type II membrane proteins bind to COPI-coatomer in in vitro mixing assays. (E) The interaction between COPI-coatomer and the NTDs of Golgi resident type II membrane proteins does not require Vps74p. (F, G) Positively charged amino acid motifs mediate the interaction between the NTDs of Golgi resident type II membrane proteins with COPI-coatomer. Glutamine amino acid substitutions were introduced at the indicated positions in the NTDs of Ktr3p and Mnn4p, as indicated. (C–G) WCEs from WT (C, D, F, and G) or vps74Δ (E) cells were incubated with the indicated bacterially expressed GST-NTDs of Golgi resident type II membrane proteins immobilized on glutathione beads. Bound proteins were resolved by SDS–PAGE and immunoblotted with anti-coatomer and anti-Vps74p antibodies. Prior to immunoblotting, the membranes were stained with Ponceau S to reveal the relative amount of GST fusion proteins (bottom panel). (H) Fate of amino acid substitutions in the NTD of Ktr3p. Amino acid substitutions (K or R to Q) were introduced into the NTD of Ktr3p at the indicated positions. The fate of the amino acid substitutions on Ktr3p localization was examined in YEPD and SD media by epifluorescence microscopy. Cells were stained with FM4-64 to visualize the vacuole. Scale bar 5 µmTo identify the relationship between Bre5p/Ubp3p, Golgi membrane proteins, and COPI, we noted that a recent study had established that some mammalian GTs could bind directly to COPI via their NTDs (Liu et al., 2018). We therefore next asked whether the NTDs of Bre5p/Ubp3p-dependent Golgi membrane proteins (see Table 1) could bind to COPI-coatomer in an in vitro mixing assay. When purified bacterially expressed glutathione S-transferase (GST) fusion proteins containing the NTDs from Ktr3p, Och1p, Mnn1p, Mnn4p, or Mnn11p were mixed with whole cell extracts (WCEs) from WT yeast cells, all five NTDs bound to COPI-coatomer to various degrees (Figure 4, C and D). In addition to COPI-coatomer, we also immunoblotted NTD-bound proteins for the presence of Vps74p—and only GST-Mnn4NTD and GST-Mnn1NTD showed any detectable binding to Vps74p (Figure 4D). The association of the GST-NTDs with COPI-coatomer was not dependent on Vps74p, however, as binding to COPI-coatomer was still observed with all four NTDs when WCEs from vps74Δ cells were used (Figure 4E).Positively charged amino acids are common features of COPI-binding proteins; for example, an N-terminal positively charged motif is required for Sed5p/syntaxin 5 and for Vps74p/GOLPH3 as well as for some mammalian GTs to bind COPI (Tu et al., 2012; Liu et al., 2018; Gao and Banfield, 2020). To identify the amino acid residues required for COPI-coatomer binding, we therefore generated amino acid–substituted forms of the NTDs of Ktr3p and Mnn4p in which we specifically targeted positively charged amino acids for substitution with glutamine—which has a similar side chain length but is uncharged. These experiments established that the positively charged amino acids at positions 6–8, 18, and 19 were required for COPI-coatomer binding to the NTD of Ktr3p (Figure 4F), whereas the positively charged amino acids at positions 8, 10–12, 19, 21, and 22 were required for COPI-coatomer binding to the NTD of Mnn4p (Figure 4G). As expected, amino acid substitutions to Mnn4p’s Vps74p-binding motif (13FLS15) resulted in the loss of binding to Vps74p, as did amino acid substitutions at residues 8 and 10–12—most likely because these residues were directly adjacent to the FLS motif (Figure 4G). In addition, amino acid substitutions at positions 19, 21, and 22 of Mnn4p also resulted in reduced binding to Vps74p (56% of WT levels); however it is not obvious why this should be the case.When the fates of the Ktr3p variants were examined in cells, both ktr3(K6Q, K7Q, K8Q)-mNeon and ktr3(R18Q, K19Q)-mNeon were localized to the vacuole as well as to puncta in otherwise WT cells (Figure 4H). These findings are consistent with the results from our in vitro mixing experiments as well as with our other data that suggest a dual mechanism for the retention of Ktr3p, and imply that in vivo one or the other basic region is at least partially functional for COPI-coatomer binding. However, as we did not use purified coatomer in our in vitro mixing assays, we cannot conclude that the NTDs of Mnn4p, Och1p, Ktr3p, and Mnn11p bind directly to coatomer or whether the binding is mediated by some as yet to be identified intermediary protein(s).The prevalence of lysine residues in the NTDs of Bre5p/Ubp3p clients raises the possibility that ubiquitination of the NTD accounts for the loss of Golgi retention observed for these clients in bre5Δ and ubp3Δ cells. At present Ktr3p is the only GT for which a ubiquitinated lysine has been identified in the NTD (K12; see www.yeastgenome.org/locus/S000000409/protein). There is, however, no evidence of a physiological role for this modification. To address the prospect that the failure to deubiquitinate the NTD of Ktr3p might account for the protein’s loss of Golgi retention, we generated amino acid substitutions in all of the lysine residues in the NTD of Ktr3p to arginine and examined the localization of these variants in WT as well as bre5Δ cells. To facilitate these experiments, we used a form of Ktr3p in which the luminal portion of the protein was removed (Ktr3CT). Like the Kre2p and Mnn4p counterparts (Kre2CT and Mnn4CT; Tu et al., 2008, 2012; Figure 3E), Ktr3CT-mNeon localized to the ER and puncta in WT cells and to the vacuole and a few puncta in bre5Δ cells when grown in SD medium (a requirement for plasmid selection) (Supplemental Figure S4). Similar results were also obtained when we examined the localization of ktr3(K6R, K7R, K8R)CT-mNeon, ktr3(K12R)CT-mNeon, and ktr3(K19R, K22R)CT-mNeon in WT and bre5Δ cells (Supplemental Figure S4). These findings suggest that failure to deubiquitinate the NTD of Ktr3p may not account for the loss of Golgi retention observed for Ktr3p in bre5Δ and ubp3Δ cells.

Cells lacking BRE5 or UBP3 show genetic interactions with VPS74 as well as synthetic cell wall defectsThe majority of GTs that reside in the Golgi at steady state are type II membrane proteins composed of a short cytoplasmically oriented N-terminus (NTD), a single transmembrane spanning sequence (TMD) followed by a stem region and enzymatic domain (Tu and Banfield, 2010). Our studies in yeast have revealed that some GTs contain a motif in their NTDs that mediates binding to Vps74p—a soluble protein that binds PI4P (phosphatidylinositol-4-phosphate) and COPI-coatomer (Tu et al., 2012; Wood et al., 2012). However, the Vps74p-dependent mechanism does not account for the retention of all yeast GTs (see Table 1). To identify additional factors required for the retention of GTs in the Golgi, we isolated dosage suppressors of a temperature-sensitive allele of VPS74 (SARY3739; for a list of yeast strains used in this study see Table 2). From this screen we isolated BRE5, UBP3, and DCR2 (Figure 1A) as well as a number of genes with known roles in protein trafficking (TLG1, GOT1, YIP3, SNF7, VPS21, and VMA5) and genes encoding membrane protein residents of the Golgi (VRG4 and KEX1). These genetic data are consistent with the role of Vps74p in the Golgi (Schmitz et al., 2008; Tu et al., 2008) (for a list of plasmids used in this study see Table 3). DCR2 encodes presumptive protein phosphatase (Guo and Polymenis, 2006; Pathak et al., 2007). For reasons that will be explained in detail elsewhere (unpublished data), vps74Δ ted1Δ cells are not viable but cells expressing a mutant of vps74 designated vps74-1 in which the altered protein is defective in binding to coatomer (vps74p(R6A,R7A,R8A); Tu et al., 2012) are temperature-sensitive for growth in ted1Δ cells (SARY3739; see Table 2). The relationship between vps74-1 ted1Δ and DCR2 will also be described elsewhere (unpublished data). BRE5 and UBP3 encode components of a nonessential deubiquitination complex that was previously shown to be involved in both ER–Golgi as well as Golgi–ER trafficking in yeast (Cohen et al., 2003a,b). Cells lacking VPS74 and UBP3 or VPS74 and BRE5 (hereafter referred to as vps74Δ ubp3Δ cells or vps74Δ bre5Δ cells, respectively) are temperature-sensitive for growth, and the temperature-sensitive phenotype is osmoremedial, suggesting that these double mutants may have defective cell walls—a phenotype that correlates with glycosylation deficiencies (Figure 1B). Consistent with these data, we also found that vps74Δ and ubp3Δ cells are sensitive to calcofluor white and, importantly, that vps74Δ ubp3Δ cells displayed a synthetic sensitivity to calcofluor white—another hallmark of mutants with additive defects in glycosylation (Figure 1C).TABLE 1:Amino acid sequence of the N-terminal cytoplasmic region (NTD) of some yeast Golgi type II membrane proteins.Red font denotes Vps74p-binding motifs. N- and O- denote N- and O-glycosylation, respectively.aMunro, 2001; b
Lussier et al., 1996; c
Yoko-o et al., 2003; d
Lussier et al., 1997; eSaccharomyces genome database (SGD, yeastgenome.org); f
Lussier et al., 1999; g
Romero et al., 1999.TABLE 2:Yeast strains used in this study.NameGenotypeSourceBY4741MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0Lab collectionSARY3739MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 vps74::kanMX4 ted1::LEU2 vps74R6A,7A,8A-pRS416Lab collectionY04208MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 vps74::kanMX4EUROSCARFSARY5644MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 bre5::loxPThis studySARY5646MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 ubp3::loxPThis studySARY5275MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 vps74::kanMX4 bre5::loxPThis studySARY5277MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 vps74::kanMX4 ubp3::loxPThis studySARY5624MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 ktr3::loxP-HIS3-loxPThis studyY01420MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 kre2::kanMX4EUROSCARFSARY5794MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 kre2::kanMX4 ktr3::loxP-HIS3-loxPThis studySARY5871MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 ktr3::KTR3-mNeon-loxP-HIS3-loxPThis studySARY5877MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 vps74::kanMX4 ktr3::KTR3-mNeon-loxP-HIS3-loxPThis studySARY5873MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 bre5::loxP ktr3::KTR3-mNeon-loxP-HIS3-loxPThis studySARY5875MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 ubp3::loxP ktr3::KTR3-mNeon-loxP-HIS3-loxPThis studySARY5770MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 ktr3::KTR3-9myc-loxP-HIS3-loxPThis studySARY5880MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 vps74::kanMX4 ktr3::KTR3-9myc-loxP-HIS3-loxPThis studySARY5772MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 bre5::loxP ktr3::KTR3-9myc-loxP-HIS3-loxPThis studySARY5774MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 ubp3::loxP ktr3::KTR3-9myc-loxP-HIS3-loxPThis studySARY5925MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 vps74::kanMX4 bre5::loxP ktr3::KTR3-9myc-loxP-HIS3-loxPThis studySARY6093MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 och1::OCH1-mNeon-loxP-HIS3-loxPThis studySARY6095MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 vps74::kanMX4 och1::OCH1-mNeon-loxP-HIS3-loxPThis studySARY6096MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 bre5::loxP och1::OCH1-mNeon-loxP-HIS3-loxPThis studySARY6098MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 ubp3::loxP och1::OCH1-mNeon-loxP-HIS3-loxPThis studySARY6253MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 och1::OCH1-9myc-loxP-HIS3-loxPThis studySARY6255MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 vps74::kanMX4 och1::OCH1-9myc-loxP-HIS3-loxPThis studySARY6257MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 bre5::loxP och1::OCH1-9myc-loxP-HIS3-loxPThis studySARY6259MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 ubp3::loxP och1::OCH1-9myc-loxP-HIS3-loxPThis studySARY6028MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 mnn11::MNN11-mNeon-loxP-HIS3-loxPThis studySARY6030MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 vps74::kanMX4 mnn11::MNN11-mNeon-loxP-HIS3-loxPThis studySARY6031MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 bre5::loxP mnn11::MNN11-mNeon-loxP-HIS3-loxPThis studySARY6033MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 ubp3::loxP mnn11::MNN11-mNeon-loxP-HIS3-loxPThis studySARY6243MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 mnn11::MNN11-9myc-loxP-HIS3-loxPThis studySARY6245MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 vps74::kanMX4 mnn11::MNN11-9myc-loxP-HIS3-loxPThis studySARY6247MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 bre5::loxP mnn11::MNN11-9myc-loxP-HIS3-loxPThis studySARY6249MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 ubp3::loxP mnn11::MNN11-9myc-loxP-HIS3-loxPThis studySARY6994MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 mnn4::MNN4-9myc-loxP-HIS3-loxPThis studySARY6996MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 vps74::kanMX4 mnn4::MNN4-9myc-loxP-HIS3-loxPThis studySARY6998MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 bre5::loxP mnn4::MNN4-9myc-loxP-HIS3-loxPThis studySARY7000MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 ubp3::loxP mnn4::MNN4-9myc-loxP-HIS3-loxPThis studySARY7002MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 vps74::kanMX4 bre5::loxP mnn4::MNN4-9myc-loxP-HIS3-loxPThis studySARY6490MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 anp1::ANP1-mNeon-loxP-HIS3-loxPThis studySARY6560MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 vps74::kanMX4 anp1::ANP1-mNeon-loxP-HIS3-loxPThis studySARY6562MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 bre5::loxP anp1::ANP1-mNeon-loxP-HIS3-loxPThis studySARY6051MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 mnn2::MNN2-mNeon-loxP-HIS3-loxPThis studySARY6053MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 vps74::kanMX4 mnn2::MNN2-mNeon-loxP-HIS3-loxPThis studySARY6055MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 bre5::loxP mnn2::MNN2-mNeon-loxP-HIS3-loxPThis studySARY6057MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 ubp3::loxP mnn2::MNN2-mNeon-loxP-HIS3-loxPThis studySARY5984MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 hoc1::HOC1-mNeon-loxP-HIS3-loxPThis studySARY5986MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 vps74::kanMX4 hoc1::HOC1-mNeon-loxP-HIS3-loxPThis studySARY5988MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 bre5::loxP hoc1::HOC1-mNeon-loxP-HIS3-loxPThis studySARY5990MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 ubp3::loxP hoc1::HOC1-mNeon-loxP-HIS3-loxPThis studySARY6021MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 mnn5::MNN5-mNeon-loxP-HIS3-loxPThis studySARY6023MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 vps74::kanMX4 mnn5::MNN5-mNeon-loxP-HIS3-loxPThis studySARY6024MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 bre5::loxP mnn5::MNN5-mNeon-loxP-HIS3-loxPThis studySARY6026MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 ubp3::loxP mnn5::MNN5-mNeon-loxP-HIS3-loxPThis studySARY5947MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 mnn9::MNN9-mNeon-loxP-HIS3-loxPThis studySARY5949MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 vps74::kanMX4 mnn9::MNN9-mNeon-loxP-HIS3-loxPThis studySARY5951MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 bre5::loxP mnn9::MNN9-mNeon-loxP-HIS3-loxPThis studySARY5953MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 ubp3::loxP mnn9::MNN9-mNeon-loxP-HIS3-loxPThis studySARY6086MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 kex1::KEX1-mNeon-loxP-HIS3-loxPThis studySARY6088MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 vps74::kanMX4 kex1::KEX1-mNeon-loxP-HIS3-loxPThis studySARY6090MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 bre5::loxP kex1::KEX1-mNeon-loxP-HIS3-loxPThis studySARY6092MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 ubp3::loxP kex1::KEX1-mNeon-loxP-HIS3-loxPThis studySARY7092MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 hoc1::HOC1-mNeon-loxP-HIS3-loxP sec27::sec27-1This studySARY7093MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 hoc1::HOC1-mNeon-loxP-HIS3-loxP sec21::sec21-1This studySARY7094MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 hoc1::HOC1-mNeon-loxP-HIS3-loxP ret1::ret1-1This studyFIGURE 1:BRE5 and UBP3 show synthetic genetic and phenotypic interactions with VPS74. (A) BRE5 and UBP3 are high-copy-number dosage suppressors of the temperature-sensitive growth phenotype of vps74-1 ted1Δ cells. vps74-1 ted1Δ cells (SARY3739; see Table 2) were transformed with the indicated plasmids, and transformants were grown to early–log phase before being subjected to 10-fold serial dilutions and aliquoting onto nutrient agar plates. Plates were incubated at the indicated temperatures for 36 h prior to being photographed. (B) vps74∆ bre5∆ cells and vps74∆ ubp3∆ cells show synthetic growth phenotypes. The indicated yeast strains were treated as described in A. (C) vps74∆ ubp3∆ cells show synthetic increased sensitivity to calcofluor white. The indicated yeast strains were treated as described in A.TABLE 3:Plasmids used in this study.PlasmidDescriptionSourcepRS413ARS/CEN plasmid with Saccharomyces cerevisiae HIS3 markerEUROSCARFpRS415ARS/CEN plasmid with S. cerevisiae LEU2 markerEUROSCARFpRS416ARS/CEN plasmid with S. cerevisiae URA3 markerEUROSCARFpRS4252µ plasmid with S. cerevisiae LEU2 markerEUROSCARFpRS4262µ plasmid with S. cerevisiae URA3 markerEUROSCARFgVPS74-pRS416The DNA fragment encoding VPS74 under the control of its own promoter was cloned as an XhoI-BamHI fragment into pRS416Lab collectiongvps74-6-8A-pRS416The DNA fragment encoding vps74 6-8A under the control of its own promoter was cloned as an XhoI-BamHI fragment into pRS416Lab collectionBRE5-MoBY2µ, LEU2 plasmid containing BRE5 ORF from Molecular Barcoded Yeast (MoBY) ORF libraryLab collectionUBP3-MoBY2µ, LEU2 plasmid containing UBP3 ORF from MoBY ORF libraryLab collectionGLO3-MoBY2µ, LEU2 plasmid containing GLO3 ORF from MoBY ORF libraryLab collectionSLY1-MoBY2µ, LEU2 plasmid containing SLY1 ORF from MoBY ORF libraryLab collectionYPT1-MoBY2µ, LEU2 plasmid containing YPT1 ORF from MoBY ORF libraryLab collectionYPT6-MoBY2µ, LEU2 plasmid containing YPT6 ORF from MoBY ORF libraryLab collectionRUD3-MoBY2µ, LEU2 plasmid containing RUD3 ORF from MoBY ORF libraryLab collectionGCS1-pRS426The DNA fragment encoding GCS1 under the control of its own promoter was cloned as an XhoI-SacII fragment into pRS426Lab collectiongcs1-R54A-pRS426The DNA fragment encoding gcs1 R54A under the control of its own promoter was cloned as an XhoI-SacII fragment into pRS426Lab collectiongcs1-H59A-pRS426The DNA fragment encoding gcs1 H59A under the control of its own promoter was cloned as an XhoI-SacII fragment into pRS426Lab collectionGLO3-pRS426The DNA fragment encoding GLO3 under the control of its own promoter was cloned as an XhoI-SacII fragment into pRS426Lab collectionglo3-R59A pRS426The DNA fragment encoding glo3 R59A under the control of its own promoter was cloned as an XhoI-SacII fragment into pRS426Lab collectionglo3-1-375 pRS426The DNA fragment encoding glo3 1-375 (amino acids 1–375) under the control of its own promoter was cloned as an XhoI-SacII fragment into pRS426Lab collectionBRE5-pRS416The DNA fragment encoding BRE5 under the control of its own promoter was cloned as a BamHI-SacII fragment into pRS416This studybre5-146-515 pRS416The DNA fragment encoding bre5 146-515 (amino acids 146–515) under the control of its own promoter was cloned as an XhoI-SacII fragment into pRS416This studyUBP3-pRS416The DNA fragment encoding UBP3 under the control of its own promoter was cloned as a BamHI-SacII fragment into pRS416This studyubp3-C469A-pRS416The DNA fragment encoding ubp3 C469A under the control of its own promoter was cloned as a BamHI-SacII fragment into pRS416This studyTPI-MNN4CT-­mNeon-pRS416The DNA fragment encoding MNN4CT (amino acids 1–49) was cloned as an EcoRI-BamHI fragment C-tagged with mNeonGreen into pRS416; TPI1 promoterThis studyTPI-KTR7CT-mNeon-pRS416The DNA fragment encoding KTR7CT (amino acids 1–44) was cloned as an EcoRI-BamHI fragment C-tagged with mNeonGreen into pRS416; TPI1 promoterThis studyTPI-ALG5CT-mNeon-pRS416The DNA fragment encoding ALG5CT (amino acids 1–34) was cloned as an EcoRI-BamHI fragment C-tagged with mNeonGreen into pRS416; TPI1 promoterThis studymNeon-Sed5 pRS416The DNA fragment encoding mNeonGreen-SED5 under the control of SED5 ­promoter was cloned as a BamHI-NotI fragment into pRS416Lab collectionpGEX-2TpTAC-driven N-terminal GST fusion expression vectorPharmaciaKTR3c-pGEX-2TThe DNA fragment encoding KTR3 cytoplasmic tail (amino acids 1–25) was cloned as a BamHI-EcoRI fragment into pGEX-2TThis studyKTR3c 6-8Q-pGEX-2TThe DNA fragment encoding KTR3 cytoplasmic tail (amino acids 1–25 with 6–8 substituted with glutamines) was cloned as a BamHI-EcoRI fragment into pGEX-2TThis studyKTR3c 18-19Q-pGEX-2TThe DNA fragment encoding KTR3 cytoplasmic tail (amino acids 1–25 with 18 and 19 substituted with glutamines) was cloned as a BamHI-EcoRI fragment into pGEX-2TThis studyMNN4c-pGEX-2TThe DNA fragment encoding MNN4 cytoplasmic tail (amino acids 1–27) was cloned as a BamHI-EcoRI fragment into pGEX-2TThis studyMNN4c 13-15A-pGEX-2TThe DNA fragment encoding MNN4 cytoplasmic tail (amino acids 1–27 with 13–15 substituted with alanines) was cloned as a BamHI-EcoRI fragment into pGEX-2TThis studyMNN4c 8,10-12Q-pGEX-2TThe DNA fragment encoding MNN4 cytoplasmic tail (amino acids 1–27 with 8 and 10–12 substituted with glutamines) was cloned as a BamHI-EcoRI fragment into pGEX-2TThis studyMNN4c 19,21-22Q-pGEX-2TThe DNA fragment encoding MNN4 cytoplasmic tail (amino acids 1–27 with 19, 21, and 22 substituted with alanines) was cloned as a BamHI-EcoRI fragment into pGEX-2TThis studyMNN4c 8,13-15,19,21-22Q-pGEX-2TThe DNA fragment encoding MNN4 cytoplasmic tail (amino acids 1–27 with 8, 13–15, 19, 21, and 22 substituted with alanines) was cloned as a BamHI-EcoRI fragment into pGEX-2TThis studyOCH1c-pGEX-2TThe DNA fragment encoding OCH1 cytoplasmic tail (amino acids 1–15) was cloned as a BamHI-EcoRI fragment into pGEX-2TThis studyMNN11c-pGEX-2TThe DNA fragment encoding MNN11 cytoplasmic tail (amino acids 1–32) was cloned as a BamHI-EcoRI fragment into pGEX-2TThis studyGST-KKXXThe DNA fragment encoding WBP1 cytoplasmic tail (amino acids 421–430) was cloned as a BamHI-EcoRI fragment into pGEX-2TLab collectionpET-HBacterial vector for expressing thioredoxin fusion proteins with an enterokinase siteLab collectionGST-pET-HDNA fragment encoding GST was cloned as a BamHI-EcoRI fragment into pET-HLab collectionKTR3c-GST-pET-HThe DNA fragment encoding KTR3 cytoplasmic tail (amino acids 1–25) was cloned as a BamHI-EcoRI fragment into GST-pET-HThis studyMNN4c-GST-pET-HThe DNA fragment encoding MNN4 cytoplasmic tail (amino acids 1–27) was cloned as a BamHI-EcoRI fragment into GST-pET-HThis studyOCH1c-GST-pET-HThe DNA fragment encoding OCH1 cytoplasmic tail (amino acids 1–15) was cloned as a BamHI-EcoRI fragment into GST-pET-HThis studyMNN11c-GST-pET-HKTR3-mNeon-pRS416KTR3 6-8Q-mNeon-pRS416KTR3 18-19Q-mNeon-pRS416The DNA fragment encoding MNN11 cytoplasmic tail (amino acids 1–32) was cloned as a BamHI-EcoRI fragment into GST-pET-HThe DNA fragment encoding KTR3-mNeon was Gibson assembled into pRS416The DNA fragment encoding KTR3 6-8Q-mNeon was Gibson assembled into pRS416The DNA fragment encoding KTR3 18-19Q-mNeon was Gibson assembled into pRS416This studyThis studyThis studyThis study

Cells lacking BRE5 or UBP3 show genetic interactions with VPS74 as well as synthetic cell wall defects

The majority of GTs that reside in the Golgi at steady state are type II membrane proteins composed of a short cytoplasmically oriented N-terminus (NTD), a single transmembrane spanning sequence (TMD) followed by a stem region and enzymatic domain (Tu and Banfield, 2010). Our studies in yeast have revealed that some GTs contain a motif in their NTDs that mediates binding to Vps74p—a soluble protein that binds PI4P (phosphatidylinositol-4-phosphate) and COPI-coatomer (Tu et al., 2012; Wood et al., 2012). However, the Vps74p-dependent mechanism does not account for the retention of all yeast GTs (see Table 1). To identify additional factors required for the retention of GTs in the Golgi, we isolated dosage suppressors of a temperature-sensitive allele of VPS74 (SARY3739; for a list of yeast strains used in this study see Table 2). From this screen we isolated BRE5, UBP3, and DCR2 (Figure 1A) as well as a number of genes with known roles in protein trafficking (TLG1, GOT1, YIP3, SNF7, VPS21, and VMA5) and genes encoding membrane protein residents of the Golgi (VRG4 and KEX1). These genetic data are consistent with the role of Vps74p in the Golgi (Schmitz et al., 2008; Tu et al., 2008) (for a list of plasmids used in this study see Table 3). DCR2 encodes presumptive protein phosphatase (Guo and Polymenis, 2006; Pathak et al., 2007). For reasons that will be explained in detail elsewhere (unpublished data), vps74Δ ted1Δ cells are not viable but cells expressing a mutant of vps74 designated vps74-1 in which the altered protein is defective in binding to coatomer (vps74p(R6A,R7A,R8A); Tu et al., 2012) are temperature-sensitive for growth in ted1Δ cells (SARY3739; see Table 2). The relationship between vps74-1 ted1Δ and DCR2 will also be described elsewhere (unpublished data). BRE5 and UBP3 encode components of a nonessential deubiquitination complex that was previously shown to be involved in both ER–Golgi as well as Golgi–ER trafficking in yeast (Cohen et al., 2003a,b). Cells lacking VPS74 and UBP3 or VPS74 and BRE5 (hereafter referred to as vps74Δ ubp3Δ cells or vps74Δ bre5Δ cells, respectively) are temperature-sensitive for growth, and the temperature-sensitive phenotype is osmoremedial, suggesting that these double mutants may have defective cell walls—a phenotype that correlates with glycosylation deficiencies (Figure 1B). Consistent with these data, we also found that vps74Δ and ubp3Δ cells are sensitive to calcofluor white and, importantly, that vps74Δ ubp3Δ cells displayed a synthetic sensitivity to calcofluor white—another hallmark of mutants with additive defects in glycosylation (Figure 1C).

Amino acid sequence of the N-terminal cytoplasmic region (NTD) of some yeast Golgi type II membrane proteins.

Red font denotes Vps74p-binding motifs. N- and O- denote N- and O-glycosylation, respectively.

aMunro, 2001; b
Lussier et al., 1996; c
Yoko-o et al., 2003; d
Lussier et al., 1997; eSaccharomyces genome database (SGD, yeastgenome.org); f
Lussier et al., 1999; g
Romero et al., 1999.

Yeast strains used in this study.

BRE5 and UBP3 show synthetic genetic and phenotypic interactions with VPS74. (A) BRE5 and UBP3 are high-copy-number dosage suppressors of the temperature-sensitive growth phenotype of vps74-1 ted1Δ cells. vps74-1 ted1Δ cells (SARY3739; see Table 2) were transformed with the indicated plasmids, and transformants were grown to early–log phase before being subjected to 10-fold serial dilutions and aliquoting onto nutrient agar plates. Plates were incubated at the indicated temperatures for 36 h prior to being photographed. (B) vps74∆ bre5∆ cells and vps74∆ ubp3∆ cells show synthetic growth phenotypes. The indicated yeast strains were treated as described in A. (C) vps74∆ ubp3∆ cells show synthetic increased sensitivity to calcofluor white. The indicated yeast strains were treated as described in A.

Plasmids used in this study.

vps74Δ bre5Δ and vps74Δ ubp3Δ cells show synthetic glycosylation defects of Gas1p and CPYTo investigate the implications of the defects displayed in vps74Δ ubp3Δ and vps74Δ bre5Δ cells, we examined the processing of two glycoproteins—a glycosylphosphatidylinositol (GPI)-anchored protein termed Gas1p and the vacuolar protease carboxypeptidase Y (CPY). While CPY is exclusively N-glycosylated (Winther et al., 1991; Munro, 2001), Gas1p is subject to both N- and O-glycosylation; however, the preponderance of glycosylation events (O-­glycosylation) occur within a 36-amino-acid serine-rich region (Gatti et al., 1994; Popolo and Vai, 1999; Neubert and Strahl, 2016). Thus, CPY and Gas1p serve as useful reporter proteins of N- and O-­glycosylation anomalies in yeast cells. Both Gas1p and CPY are translocated into the lumen of the ER, where they undergo “core” glycosylation. The ER-modified forms of these proteins are referred to as pGas1p and p1CPY. Following their export from the ER, both pGas1p and p1CPY are extensively glycosylated in the Golgi (resulting in an alteration of their apparent molecular weights), after which the so-called mature form of Gas1p (mGas1p) is delivered to the cell surface while Golgi-modified CPY (designated p2CPY) is delivered to the vacuole, where limited proteolysis converts p2CPY to its mature form (mCPY). Defects in the transportation and glycosylation of either protein can be readily assessed by SDS–PAGE and immunoblotting. Consistent with a previous report, vps74Δ cells showed hypoglycosylation of mGas1p (Tu et al., 2008; Figure 2A). Although bre5Δ and ubp3Δ cells did not show any perceivable Gas1p glycosylation defect, vps74Δ bre5Δ and vps74Δ ubp3Δ cells showed what appears to be synthetic hypoglycosylation of the post-ER form of Gas1p as judged by the altered gel mobility of the protein on SDS–PAGE gels (mGas1p; Figure 2A). Similarly, presumptive glycosylation defects were also observed when the processing of CPY was examined. The electrophoretic mobility of the mature form of CPY (mCPY) seen from bre5Δ and ubp3Δ cell extracts was similar to that observed in vps74Δ cells, suggesting that, compared with wild-type (WT) cells, CPY had an altered glycosylation profile in these mutants. The accumulation of p1CPY evident in bre5Δ, ubp3Δ, vps74Δ bre5Δ, and vps74Δ ubp3Δ cells is consistent with earlier reports that described a role for Bre5p/Ubp3p in ER–Golgi trafficking (Cohen et al., 2003a). While we cannot rule out the possibility that Bre5p/Ubp3p affects the trafficking of CPY and Gas1p, given the apparent differences in the gel mobilities and glycosylation profiles of both proteins in vps74Δ bre5Δ and vps74Δ ubp3Δ cells, and in particular the absence of any substantial synthetic accumulation of p2CPY, we considered it likely that together with Vps74p, Bre5p/Ubp3p in some way affects the glycosylation of Gas1p and CPY.FIGURE 2:BRE5 and UBP3 are required for the retention of Ktr3p in the Golgi. (A) vps74Δ bre5Δ cells and vps74Δ ubp3Δ cells show synthetic glycosylation defects. The indicated yeast strains were grown to early–log phase, and proteins from WCEs were resolved by SDS–PAGE before being immunoblotted with antibodies against Gas1p, proteinase C (CPY), and 3-phosphoglycerate kinase (Pgk1p). Pgk1p serves as a WCE loading control. pGas1p and mGas1p denote the ER and post-ER forms of Gas1p, respectively. p1CPY, p2CPY, and mCPY denote the ER form, Golgi form, and vacuolar form of CPY, respectively. MW denotes the relative positions of molecular weight standards. IB denotes primary antibody used for immunoblotting. (B) Depiction of the sequential order of action in the Golgi of yeast mannosyltransferases on Gas1p (see also Table 2). (C) kre2∆ ktr3∆ and vps74∆ ktr3∆ cells phenocopy the Gas1p glycosylation deficiency observed in vps74∆ bre5∆ and vps74∆ ubp3∆ cells. The indicated yeast strains were grown to early–log phase, and WCEs were resolved by SDS–PAGE before being immunoblotted with antibodies against Gas1p, proteinase C (CPY), and 3-phosphoglycerate kinase (Pgk1p). Pgk1p serves as a WCE loading control. (D) Ktr3p is mislocalized to the vacuole in bre5Δ or ubp3Δ cells. The DNA sequence encoding mNeon was integrated in frame at the 3′end of the coding sequence for KTR3 in each of the indicated strains. Yeast cells were grown in YEPD medium to early–log phase, stained with FM4-64, and photographed using an epifluorescence microscope (see Materials and Methods). Scale bar 5 µm. (E) The steady-state level of Ktr3p is reduced in vps74∆ bre5∆ cells. Proteins (derived from WCEs of the indicated strains) were resolved by SDS–PAGE and immunoblotted for Ktr3-9myc (see Materials and Methods) using a monoclonal antibody against the myc epitope. Quantification of the relative amounts of Ktr3-9myc in each strain (% relative to WT) is indicated below the immunoblot. Pgk1p serves as a loading control. (F) Ktr3-mNeon is not mislocalized to the vacuole in vps74Δ cells grown in SD medium and exclusively mislocalized to the vacuole in bre5∆ and ubp3∆ cells. (G) Truncation and amino acid substitutions that disrupt the activity of the Bre5p/Ubp3p deubiquitinase result in mislocalization of Ktr3-mNeon to the vacuole. (H) Truncation and amino acid substitutions that disrupt the activity of the Bre5p/Ubp3p deubiquitinase result in a reduction of the steady-state levels of Ktr3p in cells. The percentage of Ktr3p remaining in cell lysates (relative to WT cells) is indicated below the immunoblot.If Bre5p/Ubp3p could contribute to GT retention, for example, we surmised that the deubiquitinase would likely impact enzymes that were not Vps74p clients (see Table 1), as this could explain the synthetic gel mobility phenotypes observed. We previously established that the hypoglycosylation of Gas1p observed in vps74Δ cells was due to the loss of retention of the mannosyltransferase Kre2p from the Golgi (Figure 2B) (Tu et al., 2008). On the basis of the observation that Gas1p from vps74Δ bre5Δ and vps74Δ ubp3Δ cells migrates more quickly than Gas1p from vps74Δ cells, and the fact that Gas1p is extensively O-glycosylated (rather than N-glycosylated; Gatti et al., 1994), we next considered the prospect that the synthetic Gas1p glycosylation defect evident in vps74Δ bre5Δ and vps74Δ ubp3Δ cells (Figure 2A) might be due to loss of retention of enzymes that function concomitantly with Kre2p in the O-glycosylation of Gas1p (Figure 2B) as no further reduction in Kre2p levels was evident in vps74Δ bre5Δ cells compared with vps74Δ cells (Supplemental Figure S1A). As Ktr1p and Ktr3p are not known to be Vps74p clients (see Table 1), we examined the processing of Gas1p and CPY in ktr1Δ, ktr3Δ, vps74Δ ktr3Δ, and vps74Δ ktr1Δ cells (Figure 2C and Supplemental Figure S1B). While the processing of Gas1p and CPY in ktr3Δ or ktr1Δ cells was indistinguishable from that in WT cells, vps74Δ ktr3Δ and kre2Δ ktr3Δ cells showed a Gas1p glycosylation defect very much like that observed in vps74Δ bre5Δ cells and vps74Δ ubp3Δ cells, whereas vps74Δ ktr1Δ cells did not (Figure 2C and Supplemental Figure S1B). On the basis of these data, we surmised that Ktr3p might be mislocalized in bre5Δ or ubp3Δ cells, rather than Ktr1p or Kre2p (Figure 2C).To establish whether Ktr3p was mislocalized and degraded in the vacuole in bre5Δ and ubp3Δ cells, we examined the fate of Ktr3-mNeon in these mutants (see Table 2). As expected, Ktr3-mNeon was localized to Golgi-like puncta in WT cells and this was also the case—to some degree—in vps74Δ cells; however, in bre5Δ and ubp3Δ cells the protein was exclusively localized to the lumen of the vacuole (Figure 2D). Consistent with these observations, we found that the steady-state levels of Ktr3-9myc (see Table 2) were reduced in bre5Δ cells (85% of WT levels) (Figure 2E). The findings for vps74 cells were rather surprising as the NTD of Ktr3p does not contain a canonical Vps74p-binding motif (see Table 1). Rather puzzlingly, the steady-state levels of Ktr3-9myc were increased in vps74Δ cells (111% of WT levels), which is a finding that is inconsistent with observations made with Ktr3-mNeon in living cells. However, consistent with the data presented in Figure 2, A and C, vps74Δ bre5Δ cells showed a synthetic reduction in the steady-state levels of Ktr3-9myc (46% of WT levels; Figure 2E).For technical reasons the cells used for immunoblotting were grown in synthetic defined (SD) medium while cells for fluorescence microscopy were grown in yeast extract peptone dextrose (YEPD) medium (see Materials and Methods). To ascertain whether the differences in the distribution and steady-state levels of Ktr3p could be accounted for based on the nutrient composition of the growth media used, we examined the localization of Ktr3-mNeon in cells grown in SD medium. Like WT cells, Ktr3-mNeon was found exclusively in puncta when vps74Δ cells were grown in SD medium (Figure 2F) rather than in puncta and the lumen of the vacuole, as was the case for cells grown in YEPD medium (Figure 2D), whereas as expected Ktr3-mNeon was found exclusively in the vacuole in bre5Δ and ubp3Δ cells. These data suggested that the role of Vps74p in the Golgi retention of Ktr3p was affected by the nutrient composition of the medium in which cells were grown. Moreover, the data presented thus far are consistent with a dual Golgi retention mechanism for Ktr3p—one involving Vps74p and the other Bre5p/Ubp3p.To verify that the deubiquitinase activity of Bre5p/Ubp3p was required for the retention of Ktr3p in the Golgi, we generated a form of Bre5p that was previously shown to no longer bind to Ubp3p (bre5p (146-515)) (Cohen et al., 2003a,b; Li et al., 2005) as well as a form of Ubp3p that is catalytically inactive (ubp3p(C469A)) (Cohen et al., 2003a). Cells exclusively expressing either bre5p(145-515) or ubp3p(C469A) mislocalized Ktr3-mNeon to the vacuole (Figure 2G), and the steady levels of Ktr3-9myc in cells expressing bre5p(146-515) and ubp3p(C469A) were indistinguishable from those in bre5Δ and ubp3Δ cells, respectively (Figure 2H). We therefore concluded that it is the deubiquitinase activity of Bre5p/Ubp3p that is required for the retention of Ktr3p in the Golgi.There is an apparent inconsistency in the relative amounts of Ktr3-9myc observed in bre5Δ cells in Figure 2, E and H. This discrepancy may be due to subtle differences in the growth dynamics of the cell populations used in these experiments, to differences in the amounts of protein assessed by immunoblotting, or to variations in immunoblot exposure times—as detection of chemiluminescent signals is nonlinear. In any case, the data presented in Figure 2, E and H, show a consistent reduction in the levels of Ktr3-9myc in line with our hypothesis that Ktr3p’s Golgi retention is dependent in part on Bre5p/Ubp3p.

vps74Δ bre5Δ and vps74Δ ubp3Δ cells show synthetic glycosylation defects of Gas1p and CPY

To investigate the implications of the defects displayed in vps74Δ ubp3Δ and vps74Δ bre5Δ cells, we examined the processing of two glycoproteins—a glycosylphosphatidylinositol (GPI)-anchored protein termed Gas1p and the vacuolar protease carboxypeptidase Y (CPY). While CPY is exclusively N-glycosylated (Winther et al., 1991; Munro, 2001), Gas1p is subject to both N- and O-glycosylation; however, the preponderance of glycosylation events (O-­glycosylation) occur within a 36-amino-acid serine-rich region (Gatti et al., 1994; Popolo and Vai, 1999; Neubert and Strahl, 2016). Thus, CPY and Gas1p serve as useful reporter proteins of N- and O-­glycosylation anomalies in yeast cells. Both Gas1p and CPY are translocated into the lumen of the ER, where they undergo “core” glycosylation. The ER-modified forms of these proteins are referred to as pGas1p and p1CPY. Following their export from the ER, both pGas1p and p1CPY are extensively glycosylated in the Golgi (resulting in an alteration of their apparent molecular weights), after which the so-called mature form of Gas1p (mGas1p) is delivered to the cell surface while Golgi-modified CPY (designated p2CPY) is delivered to the vacuole, where limited proteolysis converts p2CPY to its mature form (mCPY). Defects in the transportation and glycosylation of either protein can be readily assessed by SDS–PAGE and immunoblotting. Consistent with a previous report, vps74Δ cells showed hypoglycosylation of mGas1p (Tu et al., 2008; Figure 2A). Although bre5Δ and ubp3Δ cells did not show any perceivable Gas1p glycosylation defect, vps74Δ bre5Δ and vps74Δ ubp3Δ cells showed what appears to be synthetic hypoglycosylation of the post-ER form of Gas1p as judged by the altered gel mobility of the protein on SDS–PAGE gels (mGas1p; Figure 2A). Similarly, presumptive glycosylation defects were also observed when the processing of CPY was examined. The electrophoretic mobility of the mature form of CPY (mCPY) seen from bre5Δ and ubp3Δ cell extracts was similar to that observed in vps74Δ cells, suggesting that, compared with wild-type (WT) cells, CPY had an altered glycosylation profile in these mutants. The accumulation of p1CPY evident in bre5Δ, ubp3Δ, vps74Δ bre5Δ, and vps74Δ ubp3Δ cells is consistent with earlier reports that described a role for Bre5p/Ubp3p in ER–Golgi trafficking (Cohen et al., 2003a). While we cannot rule out the possibility that Bre5p/Ubp3p affects the trafficking of CPY and Gas1p, given the apparent differences in the gel mobilities and glycosylation profiles of both proteins in vps74Δ bre5Δ and vps74Δ ubp3Δ cells, and in particular the absence of any substantial synthetic accumulation of p2CPY, we considered it likely that together with Vps74p, Bre5p/Ubp3p in some way affects the glycosylation of Gas1p and CPY.

BRE5 and UBP3 are required for the retention of Ktr3p in the Golgi. (A) vps74Δ bre5Δ cells and vps74Δ ubp3Δ cells show synthetic glycosylation defects. The indicated yeast strains were grown to early–log phase, and proteins from WCEs were resolved by SDS–PAGE before being immunoblotted with antibodies against Gas1p, proteinase C (CPY), and 3-phosphoglycerate kinase (Pgk1p). Pgk1p serves as a WCE loading control. pGas1p and mGas1p denote the ER and post-ER forms of Gas1p, respectively. p1CPY, p2CPY, and mCPY denote the ER form, Golgi form, and vacuolar form of CPY, respectively. MW denotes the relative positions of molecular weight standards. IB denotes primary antibody used for immunoblotting. (B) Depiction of the sequential order of action in the Golgi of yeast mannosyltransferases on Gas1p (see also Table 2). (C) kre2∆ ktr3∆ and vps74∆ ktr3∆ cells phenocopy the Gas1p glycosylation deficiency observed in vps74∆ bre5∆ and vps74∆ ubp3∆ cells. The indicated yeast strains were grown to early–log phase, and WCEs were resolved by SDS–PAGE before being immunoblotted with antibodies against Gas1p, proteinase C (CPY), and 3-phosphoglycerate kinase (Pgk1p). Pgk1p serves as a WCE loading control. (D) Ktr3p is mislocalized to the vacuole in bre5Δ or ubp3Δ cells. The DNA sequence encoding mNeon was integrated in frame at the 3′end of the coding sequence for KTR3 in each of the indicated strains. Yeast cells were grown in YEPD medium to early–log phase, stained with FM4-64, and photographed using an epifluorescence microscope (see Materials and Methods). Scale bar 5 µm. (E) The steady-state level of Ktr3p is reduced in vps74∆ bre5∆ cells. Proteins (derived from WCEs of the indicated strains) were resolved by SDS–PAGE and immunoblotted for Ktr3-9myc (see Materials and Methods) using a monoclonal antibody against the myc epitope. Quantification of the relative amounts of Ktr3-9myc in each strain (% relative to WT) is indicated below the immunoblot. Pgk1p serves as a loading control. (F) Ktr3-mNeon is not mislocalized to the vacuole in vps74Δ cells grown in SD medium and exclusively mislocalized to the vacuole in bre5∆ and ubp3∆ cells. (G) Truncation and amino acid substitutions that disrupt the activity of the Bre5p/Ubp3p deubiquitinase result in mislocalization of Ktr3-mNeon to the vacuole. (H) Truncation and amino acid substitutions that disrupt the activity of the Bre5p/Ubp3p deubiquitinase result in a reduction of the steady-state levels of Ktr3p in cells. The percentage of Ktr3p remaining in cell lysates (relative to WT cells) is indicated below the immunoblot.

If Bre5p/Ubp3p could contribute to GT retention, for example, we surmised that the deubiquitinase would likely impact enzymes that were not Vps74p clients (see Table 1), as this could explain the synthetic gel mobility phenotypes observed. We previously established that the hypoglycosylation of Gas1p observed in vps74Δ cells was due to the loss of retention of the mannosyltransferase Kre2p from the Golgi (Figure 2B) (Tu et al., 2008). On the basis of the observation that Gas1p from vps74Δ bre5Δ and vps74Δ ubp3Δ cells migrates more quickly than Gas1p from vps74Δ cells, and the fact that Gas1p is extensively O-glycosylated (rather than N-glycosylated; Gatti et al., 1994), we next considered the prospect that the synthetic Gas1p glycosylation defect evident in vps74Δ bre5Δ and vps74Δ ubp3Δ cells (Figure 2A) might be due to loss of retention of enzymes that function concomitantly with Kre2p in the O-glycosylation of Gas1p (Figure 2B) as no further reduction in Kre2p levels was evident in vps74Δ bre5Δ cells compared with vps74Δ cells (Supplemental Figure S1A). As Ktr1p and Ktr3p are not known to be Vps74p clients (see Table 1), we examined the processing of Gas1p and CPY in ktr1Δ, ktr3Δ, vps74Δ ktr3Δ, and vps74Δ ktr1Δ cells (Figure 2C and Supplemental Figure S1B). While the processing of Gas1p and CPY in ktr3Δ or ktr1Δ cells was indistinguishable from that in WT cells, vps74Δ ktr3Δ and kre2Δ ktr3Δ cells showed a Gas1p glycosylation defect very much like that observed in vps74Δ bre5Δ cells and vps74Δ ubp3Δ cells, whereas vps74Δ ktr1Δ cells did not (Figure 2C and Supplemental Figure S1B). On the basis of these data, we surmised that Ktr3p might be mislocalized in bre5Δ or ubp3Δ cells, rather than Ktr1p or Kre2p (Figure 2C).

To establish whether Ktr3p was mislocalized and degraded in the vacuole in bre5Δ and ubp3Δ cells, we examined the fate of Ktr3-mNeon in these mutants (see Table 2). As expected, Ktr3-mNeon was localized to Golgi-like puncta in WT cells and this was also the case—to some degree—in vps74Δ cells; however, in bre5Δ and ubp3Δ cells the protein was exclusively localized to the lumen of the vacuole (Figure 2D). Consistent with these observations, we found that the steady-state levels of Ktr3-9myc (see Table 2) were reduced in bre5Δ cells (85% of WT levels) (Figure 2E). The findings for vps74 cells were rather surprising as the NTD of Ktr3p does not contain a canonical Vps74p-binding motif (see Table 1). Rather puzzlingly, the steady-state levels of Ktr3-9myc were increased in vps74Δ cells (111% of WT levels), which is a finding that is inconsistent with observations made with Ktr3-mNeon in living cells. However, consistent with the data presented in Figure 2, A and C, vps74Δ bre5Δ cells showed a synthetic reduction in the steady-state levels of Ktr3-9myc (46% of WT levels; Figure 2E).

For technical reasons the cells used for immunoblotting were grown in synthetic defined (SD) medium while cells for fluorescence microscopy were grown in yeast extract peptone dextrose (YEPD) medium (see Materials and Methods). To ascertain whether the differences in the distribution and steady-state levels of Ktr3p could be accounted for based on the nutrient composition of the growth media used, we examined the localization of Ktr3-mNeon in cells grown in SD medium. Like WT cells, Ktr3-mNeon was found exclusively in puncta when vps74Δ cells were grown in SD medium (Figure 2F) rather than in puncta and the lumen of the vacuole, as was the case for cells grown in YEPD medium (Figure 2D), whereas as expected Ktr3-mNeon was found exclusively in the vacuole in bre5Δ and ubp3Δ cells. These data suggested that the role of Vps74p in the Golgi retention of Ktr3p was affected by the nutrient composition of the medium in which cells were grown. Moreover, the data presented thus far are consistent with a dual Golgi retention mechanism for Ktr3p—one involving Vps74p and the other Bre5p/Ubp3p.

To verify that the deubiquitinase activity of Bre5p/Ubp3p was required for the retention of Ktr3p in the Golgi, we generated a form of Bre5p that was previously shown to no longer bind to Ubp3p (bre5p (146-515)) (Cohen et al., 2003a,b; Li et al., 2005) as well as a form of Ubp3p that is catalytically inactive (ubp3p(C469A)) (Cohen et al., 2003a). Cells exclusively expressing either bre5p(145-515) or ubp3p(C469A) mislocalized Ktr3-mNeon to the vacuole (Figure 2G), and the steady levels of Ktr3-9myc in cells expressing bre5p(146-515) and ubp3p(C469A) were indistinguishable from those in bre5Δ and ubp3Δ cells, respectively (Figure 2H). We therefore concluded that it is the deubiquitinase activity of Bre5p/Ubp3p that is required for the retention of Ktr3p in the Golgi.

There is an apparent inconsistency in the relative amounts of Ktr3-9myc observed in bre5Δ cells in Figure 2, E and H. This discrepancy may be due to subtle differences in the growth dynamics of the cell populations used in these experiments, to differences in the amounts of protein assessed by immunoblotting, or to variations in immunoblot exposure times—as detection of chemiluminescent signals is nonlinear. In any case, the data presented in Figure 2, E and H, show a consistent reduction in the levels of Ktr3-9myc in line with our hypothesis that Ktr3p’s Golgi retention is dependent in part on Bre5p/Ubp3p.

Vps74p and Bre5p/Ubp3p show both distinct and additive roles in the retention of Golgi membrane proteinsThe synthetic glycosylation phenotypes of CPY observed in vps74Δ bre5Δ and vps74Δ ubp3Δ cells (Figure 2, A and C) do not precisely phenocopy those observed for vps74Δ ktr3Δ cells. This observation suggests that in addition to Ktr3p, Bre5p/Ubp3p might also be required for the retention of other GTs and in addition Vps74p and Bre5p/Ubp3p might function independently of each other in this capacity (Figure 2, D and F). To address this proposition, we asked whether other Golgi membrane proteins that lacked canonical Vps74p-binding motifs (Och1p and Mnn11p; see Table 1) were mislocalized in bre5Δ or ubp3Δ cells but not in vps74Δ cells. As anticipated when we examined the fate of Och1-mNeon in WT and vps74Δ cells, the protein was localized to Golgi-like puncta when cells were grown in YEPD medium (Figure 3A) although—as was the case for Ktr3-mNeon in vps74Δ cells (Figure 2, D and F) —some vacuolar luminal fluorescence was also apparent for Och1-mNeon in vps74Δ cells (Figure 3A). A similar localization profile was also evident for Och1-mNeon in vps74Δ cells grown in SD medium (Figure 3A). In contrast to vps74Δ cells, in bre5Δ and ubp3Δ cells, Och1-mNeon was predominantly localized to the vacuole (Figure 3A) and the steady-state levels of Och1-9myc were reduced to 46% (bre5Δ) and 44% (ubp3Δ) of WT levels when cells were grown in SD medium (Figure 3B). In cells grown in YEPD medium the steady-state levels of Och1-9myc were 70% (bre5Δ) and 74% (ubp3Δ) of that in WT cells (Figure 3B). By contrast, the steady-state levels of Och1-9myc in vps74Δ cells were indistinguishable from those of WT cells when the cells were grown in either SD or YEPD medium (Figure 3B). The vacuolar localization observed for Och1-mNeon in vps74Δ cells grown in both YEPD and SD media is likely due to the addition of the mNeon tag to the C-terminus of Och1p as the steady-state levels of Och1-9myc did not reveal any reduction in the protein levels for this GT (Figure 3, A and B). This is in contrast with the data obtained for Ktr3p (Figure 2, D–F) and particularly for the steady-state levels of Ktr3-9myc, which were increased relative to those seen in WT cells. We speculate that the addition of the mNeon tag can subtly affect the distributions of both Och1p and Ktr3p in various genetic backgrounds and under different growth conditions. In addition, it is also possible that some GTs are more sensitive to the addition of mNeon to their C-termini and as a result are more susceptible to mislocalization. Nevertheless, the Golgi retention of Och1p appears to be largely independent of Vps74p, but rather to require Bre5p/Ubp3p, as unlike Ktr3-9myc there is no synthetic loss of Och1-9myc in vps74Δ bre5Δ cells (Figure 3B).FIGURE 3:Golgi membrane proteins show diverse retention requirements for Vps74p and Bre5p/Ubp3p. (A) Och1p is mislocalized in bre5Δ and ubp3 cells. Scale bar 5 µm. (B) The steady-state levels of Och1p are reduced in bre5Δ and ubp3 cells. Proteins from WCEs from the indicated strains were resolved by SDS–PAGE and immunostained for Och1p and Pgk1p. Pgk1p serves as a gel loading control. (C) Mnn11p is mislocalized in bre5Δ and ubp3 cells when grown in SD medium and partially mislocalized in bre5Δ cells when grown in YEPD medium, whereas in vps74Δ cells Mnn11p is mislocalized to the vacuole. Scale bar 5 µm. (D) The differential effects of growth media on the steady-state levels of Mnn11p in vps74Δ, bre5Δ, and ubp3 cells. Proteins from WCEs from the indicated strains were resolved by SDS–PAGE and immunostained for Mnn11p and Pgk1p. Pgk1p serves as a gel loading control. (E) Mnn4p is mislocalized in vps74Δ, bre5Δ, and ubp3 cells. Scale bar 5 µm. (F) vps74Δ and bre5Δ cells show a synthetic reduction in the steady-state levels of Mnn4p. Proteins from WCEs from the indicated strains were resolved by SDS–PAGE and immunostained for Mnn4p and Pgk1p. Pgk1p serves as a gel loading control. (G–L) Anp1p, Mnn2p, Mnn5p, Mnn9p, Ktr7p, and Kre2p are not mislocalized in bre5Δ or ubp3 cells but are mislocalized to the vacuole in vps74Δ cells. In panels G–L, cells were grown in YEPD medium. Scale bar 5 µm.In contrast to Och1-mNeon, Mnn11-mNeon was localized to the vacuole in vps74Δ cells and to puncta and the vacuole in bre5Δ or ubp3Δ cells when the cells were grown in YEPD medium (Figure 3C). However, when cells were grown in SD medium, Mnn11-mNeon was localized principally to puncta in vps74Δ cells and primarily to the vacuole in bre5Δ or ubp3Δ cells (Figure 3C). Complementary observations were also made when the steady-state levels of Mnn11-9myc were examined by immunoblotting (Figure 3D). Like Och1-mNeon (see above), Mnn11-mNeon also appears to be more sensitive to alterations in genetic backgrounds as well as to growth medium variation—as Mnn11-mNeon localized to puncta and the vacuole in bre5Δ and ubp3Δ cells grown in YEPD medium. Thus even though Mnn11p lacks a canonical Vps74p-binding motif (see Table 1), it nonetheless displays a nutrient-dependent requirement for Vps74p (as was also the case for Ktr3p; Figure 2), while Bre5p/Ubp3p was required for Mnn11p’s Golgi retention irrespective of the nutrient conditions under which the cells were grown (Figure 3, C and D). Interestingly, the requirement for Bre5p/Ubp3p in the retention of Och1p and Mnn11p in the Golgi is most evident when bre5Δ or ubp3Δ cells were grown in SD medium (Figure 3, B and D).We next asked whether a type II membrane protein that contained a canonical Vps74p-binding motif might exhibit both Vps74p- and Bre5p/Ubp3p-dependent Golgi retention mechanisms. To address this prospect, we selected Mnn4p (see Table 1). Unlike Mnn11p and Och1p, the addition of mNeon to the C-terminus of Mnn4p was difficult to image due to the low expression levels of the fusion protein (unpublished data). However, our previous study revealed that an in vivo reporter for the Vps74p client Kre2p—which lacked the stem and enzymatic domains of this GT (termed Kre2CT-GFP, where CT is cytoplasmic and transmembrane domain)—was localized to the ER when cells were grown in SD medium but to Golgi-like puncta when cells were grown in YEPD medium. But, crucially, Kre2CT-GFP was mislocalized to the vacuole in SD and YEPD media in vps74Δ cells (Tu et al., 2008). We therefore used a version of Mnn4p in which the N-terminal cytoplasmic domain and the transmembrane domain were expressed as a mNeon fusion protein under the control of the triose phosphate isomerase (TPI) promoter (Mnn4CT-mNeon). Similar to Kre2CT-GFP, when WT cells were grown in YEPD medium Mnn4CT-mNeon localized exclusively to puncta and to the ER as well as to Golgi-like puncta when cells were grown in SD medium (Figure 3E). However, in vps74Δ, bre5Δ, or ubp3Δ cells Mnn4CT-mNeon was found in puncta as well as in the lumen of the vacuole regardless of the nutrient conditions (Figure 3E). The steady-state levels of Mnn4-9myc were 55%, 70%, and 64% of those found in WT cells in YEPD medium for vps74Δ, bre5Δ, or ubp3Δ cells, respectively (Figure 3F). In vps74Δ bre5Δ cells, the steady-state levels of Mnn4-9myc were further reduced to 28% of that in WT cells (Figure 3F). Moreover, the steady-state levels of Mnn4-9myc were even lower in cells grown in SD medium (Figure 3F). Taken together, the data presented in Figure 3, E and F, suggest that Mnn4p can use a dual mechanism for its Golgi retention and that Vps74p and Bre5p/Ubp3p play additive roles in the retention of Mnn4p in cells.The apparent capacity of Mnn4p to be a client for either Vps74p or Bre5p/Ubp3p does not appear to be a universal feature of GTs or of type II membrane proteins that contain canonical Vps74p-binding motifs, however (see Table 1). For example, the Golgi localization of Anp1p, Mnn2p, Mnn5p, and Mnn9p was unaffected in bre5Δ and ubp3Δ cells—as judged by fluorescence microscopy (Figure 3, G–J). The same was also true for Ktr7CT-mNeon and Kre2CT-mNeon (Figure 3, K and L). Similarly, a requirement for Bre5p/Ubp3p does not extend to Golgi/ER membrane proteins more generally as the steady-state localizations of Hoc1p, Sed5p, Kex1p, and Alg5p were unaffected in bre5Δ and ubp3Δ cells (Supplemental Figure S2). Although Hoc1p does not appear to be a client of Vps74p or Bre5p/Ubp3p, its Golgi retention is nonetheless dependent on COPI function as Hoc1-mNeon was mislocalized to the vacuole in temperature-sensitive COPI mutants (Supplemental Figure S3).

Vps74p and Bre5p/Ubp3p show both distinct and additive roles in the retention of Golgi membrane proteins

The synthetic glycosylation phenotypes of CPY observed in vps74Δ bre5Δ and vps74Δ ubp3Δ cells (Figure 2, A and C) do not precisely phenocopy those observed for vps74Δ ktr3Δ cells. This observation suggests that in addition to Ktr3p, Bre5p/Ubp3p might also be required for the retention of other GTs and in addition Vps74p and Bre5p/Ubp3p might function independently of each other in this capacity (Figure 2, D and F). To address this proposition, we asked whether other Golgi membrane proteins that lacked canonical Vps74p-binding motifs (Och1p and Mnn11p; see Table 1) were mislocalized in bre5Δ or ubp3Δ cells but not in vps74Δ cells. As anticipated when we examined the fate of Och1-mNeon in WT and vps74Δ cells, the protein was localized to Golgi-like puncta when cells were grown in YEPD medium (Figure 3A) although—as was the case for Ktr3-mNeon in vps74Δ cells (Figure 2, D and F) —some vacuolar luminal fluorescence was also apparent for Och1-mNeon in vps74Δ cells (Figure 3A). A similar localization profile was also evident for Och1-mNeon in vps74Δ cells grown in SD medium (Figure 3A). In contrast to vps74Δ cells, in bre5Δ and ubp3Δ cells, Och1-mNeon was predominantly localized to the vacuole (Figure 3A) and the steady-state levels of Och1-9myc were reduced to 46% (bre5Δ) and 44% (ubp3Δ) of WT levels when cells were grown in SD medium (Figure 3B). In cells grown in YEPD medium the steady-state levels of Och1-9myc were 70% (bre5Δ) and 74% (ubp3Δ) of that in WT cells (Figure 3B). By contrast, the steady-state levels of Och1-9myc in vps74Δ cells were indistinguishable from those of WT cells when the cells were grown in either SD or YEPD medium (Figure 3B). The vacuolar localization observed for Och1-mNeon in vps74Δ cells grown in both YEPD and SD media is likely due to the addition of the mNeon tag to the C-terminus of Och1p as the steady-state levels of Och1-9myc did not reveal any reduction in the protein levels for this GT (Figure 3, A and B). This is in contrast with the data obtained for Ktr3p (Figure 2, D–F) and particularly for the steady-state levels of Ktr3-9myc, which were increased relative to those seen in WT cells. We speculate that the addition of the mNeon tag can subtly affect the distributions of both Och1p and Ktr3p in various genetic backgrounds and under different growth conditions. In addition, it is also possible that some GTs are more sensitive to the addition of mNeon to their C-termini and as a result are more susceptible to mislocalization. Nevertheless, the Golgi retention of Och1p appears to be largely independent of Vps74p, but rather to require Bre5p/Ubp3p, as unlike Ktr3-9myc there is no synthetic loss of Och1-9myc in vps74Δ bre5Δ cells (Figure 3B).

Golgi membrane proteins show diverse retention requirements for Vps74p and Bre5p/Ubp3p. (A) Och1p is mislocalized in bre5Δ and ubp3 cells. Scale bar 5 µm. (B) The steady-state levels of Och1p are reduced in bre5Δ and ubp3 cells. Proteins from WCEs from the indicated strains were resolved by SDS–PAGE and immunostained for Och1p and Pgk1p. Pgk1p serves as a gel loading control. (C) Mnn11p is mislocalized in bre5Δ and ubp3 cells when grown in SD medium and partially mislocalized in bre5Δ cells when grown in YEPD medium, whereas in vps74Δ cells Mnn11p is mislocalized to the vacuole. Scale bar 5 µm. (D) The differential effects of growth media on the steady-state levels of Mnn11p in vps74Δ, bre5Δ, and ubp3 cells. Proteins from WCEs from the indicated strains were resolved by SDS–PAGE and immunostained for Mnn11p and Pgk1p. Pgk1p serves as a gel loading control. (E) Mnn4p is mislocalized in vps74Δ, bre5Δ, and ubp3 cells. Scale bar 5 µm. (F) vps74Δ and bre5Δ cells show a synthetic reduction in the steady-state levels of Mnn4p. Proteins from WCEs from the indicated strains were resolved by SDS–PAGE and immunostained for Mnn4p and Pgk1p. Pgk1p serves as a gel loading control. (G–L) Anp1p, Mnn2p, Mnn5p, Mnn9p, Ktr7p, and Kre2p are not mislocalized in bre5Δ or ubp3 cells but are mislocalized to the vacuole in vps74Δ cells. In panels G–L, cells were grown in YEPD medium. Scale bar 5 µm.

In contrast to Och1-mNeon, Mnn11-mNeon was localized to the vacuole in vps74Δ cells and to puncta and the vacuole in bre5Δ or ubp3Δ cells when the cells were grown in YEPD medium (Figure 3C). However, when cells were grown in SD medium, Mnn11-mNeon was localized principally to puncta in vps74Δ cells and primarily to the vacuole in bre5Δ or ubp3Δ cells (Figure 3C). Complementary observations were also made when the steady-state levels of Mnn11-9myc were examined by immunoblotting (Figure 3D). Like Och1-mNeon (see above), Mnn11-mNeon also appears to be more sensitive to alterations in genetic backgrounds as well as to growth medium variation—as Mnn11-mNeon localized to puncta and the vacuole in bre5Δ and ubp3Δ cells grown in YEPD medium. Thus even though Mnn11p lacks a canonical Vps74p-binding motif (see Table 1), it nonetheless displays a nutrient-dependent requirement for Vps74p (as was also the case for Ktr3p; Figure 2), while Bre5p/Ubp3p was required for Mnn11p’s Golgi retention irrespective of the nutrient conditions under which the cells were grown (Figure 3, C and D). Interestingly, the requirement for Bre5p/Ubp3p in the retention of Och1p and Mnn11p in the Golgi is most evident when bre5Δ or ubp3Δ cells were grown in SD medium (Figure 3, B and D).

We next asked whether a type II membrane protein that contained a canonical Vps74p-binding motif might exhibit both Vps74p- and Bre5p/Ubp3p-dependent Golgi retention mechanisms. To address this prospect, we selected Mnn4p (see Table 1). Unlike Mnn11p and Och1p, the addition of mNeon to the C-terminus of Mnn4p was difficult to image due to the low expression levels of the fusion protein (unpublished data). However, our previous study revealed that an in vivo reporter for the Vps74p client Kre2p—which lacked the stem and enzymatic domains of this GT (termed Kre2CT-GFP, where CT is cytoplasmic and transmembrane domain)—was localized to the ER when cells were grown in SD medium but to Golgi-like puncta when cells were grown in YEPD medium. But, crucially, Kre2CT-GFP was mislocalized to the vacuole in SD and YEPD media in vps74Δ cells (Tu et al., 2008). We therefore used a version of Mnn4p in which the N-terminal cytoplasmic domain and the transmembrane domain were expressed as a mNeon fusion protein under the control of the triose phosphate isomerase (TPI) promoter (Mnn4CT-mNeon). Similar to Kre2CT-GFP, when WT cells were grown in YEPD medium Mnn4CT-mNeon localized exclusively to puncta and to the ER as well as to Golgi-like puncta when cells were grown in SD medium (Figure 3E). However, in vps74Δ, bre5Δ, or ubp3Δ cells Mnn4CT-mNeon was found in puncta as well as in the lumen of the vacuole regardless of the nutrient conditions (Figure 3E). The steady-state levels of Mnn4-9myc were 55%, 70%, and 64% of those found in WT cells in YEPD medium for vps74Δ, bre5Δ, or ubp3Δ cells, respectively (Figure 3F). In vps74Δ bre5Δ cells, the steady-state levels of Mnn4-9myc were further reduced to 28% of that in WT cells (Figure 3F). Moreover, the steady-state levels of Mnn4-9myc were even lower in cells grown in SD medium (Figure 3F). Taken together, the data presented in Figure 3, E and F, suggest that Mnn4p can use a dual mechanism for its Golgi retention and that Vps74p and Bre5p/Ubp3p play additive roles in the retention of Mnn4p in cells.

The apparent capacity of Mnn4p to be a client for either Vps74p or Bre5p/Ubp3p does not appear to be a universal feature of GTs or of type II membrane proteins that contain canonical Vps74p-binding motifs, however (see Table 1). For example, the Golgi localization of Anp1p, Mnn2p, Mnn5p, and Mnn9p was unaffected in bre5Δ and ubp3Δ cells—as judged by fluorescence microscopy (Figure 3, G–J). The same was also true for Ktr7CT-mNeon and Kre2CT-mNeon (Figure 3, K and L). Similarly, a requirement for Bre5p/Ubp3p does not extend to Golgi/ER membrane proteins more generally as the steady-state localizations of Hoc1p, Sed5p, Kex1p, and Alg5p were unaffected in bre5Δ and ubp3Δ cells (Supplemental Figure S2). Although Hoc1p does not appear to be a client of Vps74p or Bre5p/Ubp3p, its Golgi retention is nonetheless dependent on COPI function as Hoc1-mNeon was mislocalized to the vacuole in temperature-sensitive COPI mutants (Supplemental Figure S3).

GTs that require Bre5p/Ubp3p for their Golgi retention bind to COPI-coatomer via their NTDsWhat is the basis for the requirement for Bre5p/Ubp3p in the retention of Golgi membrane proteins? To address this question, we took a genetic approach. As vps74Δ bre5Δ cells are temperature-­sensitive for growth (Figure 1B), we sought to identify genes that could function as dosage suppressors of this phenotype. Given the relationship between COPI and Vps74p (Tu et al., 2008, 2012), we surmised that genes whose products function to modulate the function of the Golgi vesicle coat complex—COPI-coatomer—might be likely candidates. Glo3p and Gcs1p are GTPase-activating proteins (GAPs) for COPI-coatomer’s GTPase Arf1p and as such are required for the function of the COPI coat (Poon et al., 1996; Lewis et al., 2004). As anticipated, GLO3 and GCS1 were dosage suppressors of the temperature-sensitive growth phenotype of vps74Δ bre5Δ cells (Figure 4, A and B). Moreover, it is the GAP activity that is required for suppression, as amino acid substitutions to Glo3p or Gcs1p that render these proteins catalytically inactive no longer functioned as suppressors (Figure 4B). These findings are consistent with those of Cohen et al. (2003b), who showed that cells lacking both BRE5 and GLO3 were inviable. It may therefore be the case that the activity or steady-state levels of Arf1-GAPs are compromised in vps74Δ bre5Δ cells or that the activity of COPI-coatomer itself is compromised in some manner.FIGURE 4:COPI plays a direct role in the retention of Bre5p/Ubp3p clients in the Golgi. (A) The Arf1 GAP Glo3p is a dosage suppressor of the temperature-sensitive growth phenotype of vps74Δ bre5Δ cells. (B) The GAP activity of Glo3p and Gcs1p is required to suppress the temperature-sensitive growth phenotype of vps74Δ bre5Δ cells. For A and B, 10-fold serial dilutions of the indicated yeast strains were spotted on nutrient agar plates, which were then placed in a 25°C or 37°C incubator for 48 h prior to being photographed. (C, D) The N-terminal cytoplasmic domains (NTD) of some Golgi resident type II membrane proteins bind to COPI-coatomer in in vitro mixing assays. (E) The interaction between COPI-coatomer and the NTDs of Golgi resident type II membrane proteins does not require Vps74p. (F, G) Positively charged amino acid motifs mediate the interaction between the NTDs of Golgi resident type II membrane proteins with COPI-coatomer. Glutamine amino acid substitutions were introduced at the indicated positions in the NTDs of Ktr3p and Mnn4p, as indicated. (C–G) WCEs from WT (C, D, F, and G) or vps74Δ (E) cells were incubated with the indicated bacterially expressed GST-NTDs of Golgi resident type II membrane proteins immobilized on glutathione beads. Bound proteins were resolved by SDS–PAGE and immunoblotted with anti-coatomer and anti-Vps74p antibodies. Prior to immunoblotting, the membranes were stained with Ponceau S to reveal the relative amount of GST fusion proteins (bottom panel). (H) Fate of amino acid substitutions in the NTD of Ktr3p. Amino acid substitutions (K or R to Q) were introduced into the NTD of Ktr3p at the indicated positions. The fate of the amino acid substitutions on Ktr3p localization was examined in YEPD and SD media by epifluorescence microscopy. Cells were stained with FM4-64 to visualize the vacuole. Scale bar 5 µmTo identify the relationship between Bre5p/Ubp3p, Golgi membrane proteins, and COPI, we noted that a recent study had established that some mammalian GTs could bind directly to COPI via their NTDs (Liu et al., 2018). We therefore next asked whether the NTDs of Bre5p/Ubp3p-dependent Golgi membrane proteins (see Table 1) could bind to COPI-coatomer in an in vitro mixing assay. When purified bacterially expressed glutathione S-transferase (GST) fusion proteins containing the NTDs from Ktr3p, Och1p, Mnn1p, Mnn4p, or Mnn11p were mixed with whole cell extracts (WCEs) from WT yeast cells, all five NTDs bound to COPI-coatomer to various degrees (Figure 4, C and D). In addition to COPI-coatomer, we also immunoblotted NTD-bound proteins for the presence of Vps74p—and only GST-Mnn4NTD and GST-Mnn1NTD showed any detectable binding to Vps74p (Figure 4D). The association of the GST-NTDs with COPI-coatomer was not dependent on Vps74p, however, as binding to COPI-coatomer was still observed with all four NTDs when WCEs from vps74Δ cells were used (Figure 4E).Positively charged amino acids are common features of COPI-binding proteins; for example, an N-terminal positively charged motif is required for Sed5p/syntaxin 5 and for Vps74p/GOLPH3 as well as for some mammalian GTs to bind COPI (Tu et al., 2012; Liu et al., 2018; Gao and Banfield, 2020). To identify the amino acid residues required for COPI-coatomer binding, we therefore generated amino acid–substituted forms of the NTDs of Ktr3p and Mnn4p in which we specifically targeted positively charged amino acids for substitution with glutamine—which has a similar side chain length but is uncharged. These experiments established that the positively charged amino acids at positions 6–8, 18, and 19 were required for COPI-coatomer binding to the NTD of Ktr3p (Figure 4F), whereas the positively charged amino acids at positions 8, 10–12, 19, 21, and 22 were required for COPI-coatomer binding to the NTD of Mnn4p (Figure 4G). As expected, amino acid substitutions to Mnn4p’s Vps74p-binding motif (13FLS15) resulted in the loss of binding to Vps74p, as did amino acid substitutions at residues 8 and 10–12—most likely because these residues were directly adjacent to the FLS motif (Figure 4G). In addition, amino acid substitutions at positions 19, 21, and 22 of Mnn4p also resulted in reduced binding to Vps74p (56% of WT levels); however it is not obvious why this should be the case.When the fates of the Ktr3p variants were examined in cells, both ktr3(K6Q, K7Q, K8Q)-mNeon and ktr3(R18Q, K19Q)-mNeon were localized to the vacuole as well as to puncta in otherwise WT cells (Figure 4H). These findings are consistent with the results from our in vitro mixing experiments as well as with our other data that suggest a dual mechanism for the retention of Ktr3p, and imply that in vivo one or the other basic region is at least partially functional for COPI-coatomer binding. However, as we did not use purified coatomer in our in vitro mixing assays, we cannot conclude that the NTDs of Mnn4p, Och1p, Ktr3p, and Mnn11p bind directly to coatomer or whether the binding is mediated by some as yet to be identified intermediary protein(s).The prevalence of lysine residues in the NTDs of Bre5p/Ubp3p clients raises the possibility that ubiquitination of the NTD accounts for the loss of Golgi retention observed for these clients in bre5Δ and ubp3Δ cells. At present Ktr3p is the only GT for which a ubiquitinated lysine has been identified in the NTD (K12; see www.yeastgenome.org/locus/S000000409/protein). There is, however, no evidence of a physiological role for this modification. To address the prospect that the failure to deubiquitinate the NTD of Ktr3p might account for the protein’s loss of Golgi retention, we generated amino acid substitutions in all of the lysine residues in the NTD of Ktr3p to arginine and examined the localization of these variants in WT as well as bre5Δ cells. To facilitate these experiments, we used a form of Ktr3p in which the luminal portion of the protein was removed (Ktr3CT). Like the Kre2p and Mnn4p counterparts (Kre2CT and Mnn4CT; Tu et al., 2008, 2012; Figure 3E), Ktr3CT-mNeon localized to the ER and puncta in WT cells and to the vacuole and a few puncta in bre5Δ cells when grown in SD medium (a requirement for plasmid selection) (Supplemental Figure S4). Similar results were also obtained when we examined the localization of ktr3(K6R, K7R, K8R)CT-mNeon, ktr3(K12R)CT-mNeon, and ktr3(K19R, K22R)CT-mNeon in WT and bre5Δ cells (Supplemental Figure S4). These findings suggest that failure to deubiquitinate the NTD of Ktr3p may not account for the loss of Golgi retention observed for Ktr3p in bre5Δ and ubp3Δ cells.

GTs that require Bre5p/Ubp3p for their Golgi retention bind to COPI-coatomer via their NTDs

What is the basis for the requirement for Bre5p/Ubp3p in the retention of Golgi membrane proteins? To address this question, we took a genetic approach. As vps74Δ bre5Δ cells are temperature-­sensitive for growth (Figure 1B), we sought to identify genes that could function as dosage suppressors of this phenotype. Given the relationship between COPI and Vps74p (Tu et al., 2008, 2012), we surmised that genes whose products function to modulate the function of the Golgi vesicle coat complex—COPI-coatomer—might be likely candidates. Glo3p and Gcs1p are GTPase-activating proteins (GAPs) for COPI-coatomer’s GTPase Arf1p and as such are required for the function of the COPI coat (Poon et al., 1996; Lewis et al., 2004). As anticipated, GLO3 and GCS1 were dosage suppressors of the temperature-sensitive growth phenotype of vps74Δ bre5Δ cells (Figure 4, A and B). Moreover, it is the GAP activity that is required for suppression, as amino acid substitutions to Glo3p or Gcs1p that render these proteins catalytically inactive no longer functioned as suppressors (Figure 4B). These findings are consistent with those of Cohen et al. (2003b), who showed that cells lacking both BRE5 and GLO3 were inviable. It may therefore be the case that the activity or steady-state levels of Arf1-GAPs are compromised in vps74Δ bre5Δ cells or that the activity of COPI-coatomer itself is compromised in some manner.

COPI plays a direct role in the retention of Bre5p/Ubp3p clients in the Golgi. (A) The Arf1 GAP Glo3p is a dosage suppressor of the temperature-sensitive growth phenotype of vps74Δ bre5Δ cells. (B) The GAP activity of Glo3p and Gcs1p is required to suppress the temperature-sensitive growth phenotype of vps74Δ bre5Δ cells. For A and B, 10-fold serial dilutions of the indicated yeast strains were spotted on nutrient agar plates, which were then placed in a 25°C or 37°C incubator for 48 h prior to being photographed. (C, D) The N-terminal cytoplasmic domains (NTD) of some Golgi resident type II membrane proteins bind to COPI-coatomer in in vitro mixing assays. (E) The interaction between COPI-coatomer and the NTDs of Golgi resident type II membrane proteins does not require Vps74p. (F, G) Positively charged amino acid motifs mediate the interaction between the NTDs of Golgi resident type II membrane proteins with COPI-coatomer. Glutamine amino acid substitutions were introduced at the indicated positions in the NTDs of Ktr3p and Mnn4p, as indicated. (C–G) WCEs from WT (C, D, F, and G) or vps74Δ (E) cells were incubated with the indicated bacterially expressed GST-NTDs of Golgi resident type II membrane proteins immobilized on glutathione beads. Bound proteins were resolved by SDS–PAGE and immunoblotted with anti-coatomer and anti-Vps74p antibodies. Prior to immunoblotting, the membranes were stained with Ponceau S to reveal the relative amount of GST fusion proteins (bottom panel). (H) Fate of amino acid substitutions in the NTD of Ktr3p. Amino acid substitutions (K or R to Q) were introduced into the NTD of Ktr3p at the indicated positions. The fate of the amino acid substitutions on Ktr3p localization was examined in YEPD and SD media by epifluorescence microscopy. Cells were stained with FM4-64 to visualize the vacuole. Scale bar 5 µm

To identify the relationship between Bre5p/Ubp3p, Golgi membrane proteins, and COPI, we noted that a recent study had established that some mammalian GTs could bind directly to COPI via their NTDs (Liu et al., 2018). We therefore next asked whether the NTDs of Bre5p/Ubp3p-dependent Golgi membrane proteins (see Table 1) could bind to COPI-coatomer in an in vitro mixing assay. When purified bacterially expressed glutathione S-transferase (GST) fusion proteins containing the NTDs from Ktr3p, Och1p, Mnn1p, Mnn4p, or Mnn11p were mixed with whole cell extracts (WCEs) from WT yeast cells, all five NTDs bound to COPI-coatomer to various degrees (Figure 4, C and D). In addition to COPI-coatomer, we also immunoblotted NTD-bound proteins for the presence of Vps74p—and only GST-Mnn4NTD and GST-Mnn1NTD showed any detectable binding to Vps74p (Figure 4D). The association of the GST-NTDs with COPI-coatomer was not dependent on Vps74p, however, as binding to COPI-coatomer was still observed with all four NTDs when WCEs from vps74Δ cells were used (Figure 4E).

Positively charged amino acids are common features of COPI-binding proteins; for example, an N-terminal positively charged motif is required for Sed5p/syntaxin 5 and for Vps74p/GOLPH3 as well as for some mammalian GTs to bind COPI (Tu et al., 2012; Liu et al., 2018; Gao and Banfield, 2020). To identify the amino acid residues required for COPI-coatomer binding, we therefore generated amino acid–substituted forms of the NTDs of Ktr3p and Mnn4p in which we specifically targeted positively charged amino acids for substitution with glutamine—which has a similar side chain length but is uncharged. These experiments established that the positively charged amino acids at positions 6–8, 18, and 19 were required for COPI-coatomer binding to the NTD of Ktr3p (Figure 4F), whereas the positively charged amino acids at positions 8, 10–12, 19, 21, and 22 were required for COPI-coatomer binding to the NTD of Mnn4p (Figure 4G). As expected, amino acid substitutions to Mnn4p’s Vps74p-binding motif (13FLS15) resulted in the loss of binding to Vps74p, as did amino acid substitutions at residues 8 and 10–12—most likely because these residues were directly adjacent to the FLS motif (Figure 4G). In addition, amino acid substitutions at positions 19, 21, and 22 of Mnn4p also resulted in reduced binding to Vps74p (56% of WT levels); however it is not obvious why this should be the case.

When the fates of the Ktr3p variants were examined in cells, both ktr3(K6Q, K7Q, K8Q)-mNeon and ktr3(R18Q, K19Q)-mNeon were localized to the vacuole as well as to puncta in otherwise WT cells (Figure 4H). These findings are consistent with the results from our in vitro mixing experiments as well as with our other data that suggest a dual mechanism for the retention of Ktr3p, and imply that in vivo one or the other basic region is at least partially functional for COPI-coatomer binding. However, as we did not use purified coatomer in our in vitro mixing assays, we cannot conclude that the NTDs of Mnn4p, Och1p, Ktr3p, and Mnn11p bind directly to coatomer or whether the binding is mediated by some as yet to be identified intermediary protein(s).

The prevalence of lysine residues in the NTDs of Bre5p/Ubp3p clients raises the possibility that ubiquitination of the NTD accounts for the loss of Golgi retention observed for these clients in bre5Δ and ubp3Δ cells. At present Ktr3p is the only GT for which a ubiquitinated lysine has been identified in the NTD (K12; see www.yeastgenome.org/locus/S000000409/protein). There is, however, no evidence of a physiological role for this modification. To address the prospect that the failure to deubiquitinate the NTD of Ktr3p might account for the protein’s loss of Golgi retention, we generated amino acid substitutions in all of the lysine residues in the NTD of Ktr3p to arginine and examined the localization of these variants in WT as well as bre5Δ cells. To facilitate these experiments, we used a form of Ktr3p in which the luminal portion of the protein was removed (Ktr3CT). Like the Kre2p and Mnn4p counterparts (Kre2CT and Mnn4CT; Tu et al., 2008, 2012; Figure 3E), Ktr3CT-mNeon localized to the ER and puncta in WT cells and to the vacuole and a few puncta in bre5Δ cells when grown in SD medium (a requirement for plasmid selection) (Supplemental Figure S4). Similar results were also obtained when we examined the localization of ktr3(K6R, K7R, K8R)CT-mNeon, ktr3(K12R)CT-mNeon, and ktr3(K19R, K22R)CT-mNeon in WT and bre5Δ cells (Supplemental Figure S4). These findings suggest that failure to deubiquitinate the NTD of Ktr3p may not account for the loss of Golgi retention observed for Ktr3p in bre5Δ and ubp3Δ cells.

DISCUSSIONIn this study we report a novel mechanism for the retention of Golgi resident type II membrane proteins (GTs and Mnn4p) mediated by the deubiquitinase Bre5p/Ubp3p. This finding expands our understanding of the mechanism(s) that govern the retention of Golgi resident integral membrane proteins. Moreover, our findings reveal a heretofore unexpected level of mechanistic complexity. For example, some proteins appear to be exclusive clients of Bre5p/Ubp3p (Och1p) or Vps74p (Kre2p, Mnn2p, Mnn5p, Mnn9p, and Anp1p) while at least three proteins (Ktr3p, Mnn11p, and Mnn4p) are clients of both Vps74p and Bre5p/Ubp3p. It will be important to ascertain whether Vps74p and Bre5p/Ubp3p function on the same Golgi cisterna or in distinct locations. Vps74p, for example, binds to Golgi membranes in a PI4P-dependent manner, and PI4P is predominantly found on the trans-Golgi (Wood et al., 2009). In contrast, although Hoc1p’s mechanism of Golgi retention requires COPI function, it appears to be independent of Vps74p and Bre5p/Ubp3p (Supplemental Figures S2A and S3). The findings for Hoc1p and Mnn11p are particularly intriguing as these GTs reportedly are subunits of a Golgi mannosyltransferase complex together with Anp1p, Mnn9p, and Mnn10p (Jungmann and Munro, 1998). While Anp1p and Mnn9p are both Vps74p clients, Hoc1p is not—our findings therefore raise questions regarding the robustness of the associations of the mannosyltransferases within this complex in cells.Surprisingly, two GTs—Ktr3p and Mnn11p—showed a requirement for Vps74p that was evident only when cells were grown in YEPD medium. However, neither Ktr3p nor Mnn11p appears to contain a canonical Vps74p-dependent binding motif in its NTDs (see Table 1), and in in vitro binding studies the NTDs of Ktr3p and Mnn11p did not bind to Vps74p. As cells were grown only to early log phase (in either YEPD or SD medium), this cannot be a starvation-induced phenomenon but rather suggests that COPI-mediated trafficking can be influenced by the composition of nutrients made available to the cell, which in turn may impact the growth dynamics of the cels. Additionally, compelling evidence for nutrient-based modulation of the activity of COPI-coatomer comes from the observation that at least two of the Golgi protein reporters—Kre2CT-mNeon and Mnn4CT-mNeon—are localized to the ER when cells are grown in SD medium and to puncta when cells are grown in YEPD medium. This medium-dependent differential localization is dependent on Vps74p (and indirectly COPI) as Kre2CT-mNeon/Kre2CT-GFP and Mnn4CT-mNeon were mislocalized to the vacuole in vps74Δ cells (Figure 3, E and L) (Tu et al., 2008).Notably, type II proteins that showed a requirement for Bre5p/Ubp3p in their Golgi retention (Ktr3p, Mnn4p, Mnn11p, and Och1p) bound to COPI-coatomer via their NTDs (in biochemical assays), suggesting a direct role for COPI-coatomer in the Bre5p/Ubp3p-dependent retention mechanism. This was also the case for the NTD of Mnn1p—although we were not able to determine whether Mnn1p was a client of Bre5p/Ubp3p as both Mnn1-mNeon and Mnn1CT- mNeon were mislocalized to the vacuole even in WT cells (unpublished data). For two of these Golgi type II proteins (Mnn4p and Ktr3p), the motif that mediates COPI-coatomer binding is composed of a bipartite basic motif. The motifs identified in this study are distinct from the consensus sequence identified in mammalian enzymes “φ-(K/R)-X-L-X(K/R)” and which binds to the delta and gamma subunits of COPI-coatomer (Liu et al., 2018). Moreover, the Ktr3p and Mnn4p motifs bear no obvious similarity to one another—either compositionally or in their “punctuation” within the NTD. Nevertheless, it appears that there is a correlation for a Golgi membrane protein being a client for Bre5p/Ubp3p-dependent retention as each of the four GT (Ktr3p, Mnn4p, Mnn11p, and Och1p) NTDs can mediate COPI-coatomer binding in vitro.How might ubiquitination/deubiquitination affect the localization of Golgi resident membrane proteins? Amino acid substitutions of the lysine residues in Ktr3p suggest that ubiquitination/deubiquitination of the NTD may not play a direct role in the Bre5p/Ubp3p-dependent retention of this GT in the Golgi (at least for Ktr3p). Rather the data presented herein are suggestive of a direct role for COPI-coatomer. In this regard it is noteworthy that in bre5Δ and ubp3Δ cells evidently only β′-COP (yeast Sec27p) was ubiquitinated (Cohen et al., 2003b)—a finding that suggests a possible role for Sec27p in the Bre5p/Ubp3p-dependent localization of GTs. Glo3p and Gcs1p could be potential substrates too—based on our genetic data—but given that bre5Δ and ubp3Δ cells did not result in more general deficiencies in COPI function (Figure 3, G–L, and Supplemental Figure S2), it is unlikely that either protein is degraded in the absence of the deubiquitinase. Given the limited number of clients involved, protein degradation in the absence of the deubiquitinase is also unlikely in the case of β′-COP (yeast Sec27p) or indeed any other coatomer subunits. We currently favor a model in which the ubiquitination and deubiquitination of COPI-coatomer functions to directly facilitate selection of Golgi membrane proteins such as GTs into nascent vesicles. Given the phenotype observed in bre5Δ and ubp3Δ cells, we postulate that ubiquitination functions to limit or inhibit binding of GTs (as well as other Golgi protein clients) to COPI. Such a mechanism could occur on multiple subunits of coatomer or indeed on Glo3p and or Gcs1p—as all of these proteins are reportedly ubiquitinated (yeastgenome.org). Indeed, basic motifs such as the ones identified here appear to bind to multiple sites on COPI-coatomer (Liu et al., 2018), and given that the motifs identified in Ktr3p and Mnn4p are not particularly similar, it is also possible that they bind to different regions/subunits of COPI. Moreover, evidently the action of Bre5p/Ubp3p is specific to clients bearing basic motifs such as those found in Ktr3p and Mnn4p, rather than the basic motif found in Vps74p—as Vps74p’s clients were unaffected in bre5Δ and ubp3Δ cells (Figure 3, G –L). That said, the relatively short length of the NTDs will clearly rule out some coatomer subunits based on the geometry of coatomer and hence the proximity of potential ubiquitination sites to the Golgi membrane surface.What is the physiological significance of reversibly controlling access of protein clients to COPI? One intriguing hypothesis is that such a mechanism may provide the cell with a means by which to exclude proteins from incorporation into COPI vesicles in some cisternae, while allowing their selection in others. It has been recognized for some time that functionally distinct pools of COPI function on the cis and medial/trans-Golgi in yeast cells (Wooding and Pelham, 1998). For example, not all Golgi membrane proteins appear to cycle between the Golgi and the ER even though they are mislocalized in COPI/SEC mutants. Significantly those proteins that do cycle between the Golgi and the ER are typically steady-state residents of the cis-Golgi—such as Sed5p, Rer1p, and Emp46p (Sato et al., 1995; Sato and Nakano, 2002; Wooding and Pelham, 1998; Gao and Banfield, 2020). By contrast, the GTs Anp1p, Och1p, and Mnn1p reportedly do not cycle through the ER (Harris and Waters, 1996; Wooding and Pelham, 1998), and of these we have shown that Och1p is a Bre5p/Ubp3p client.A recent study revealed a role for ubiquitination in the sorting of the R-SNARE Snc1p mediated by the COPI coat (Xu et al., 2017). The authors demonstrated that the ubiquitination of Snc1p was a prerequisite for binding to a region of β′COP (Sec27p) that binds polyubiquitin (Xu et al., 2017). While it remains a possibility that the NTDs of some Bre5p/Ubp3p clients are themselves ubiquitinated, lysine to arginine substitutions in the NTD of Ktr3p had no effect on the retention of Ktr3CT-mNeon in WT or bre5Δ cells (Supplemental Figure S4). Therefore, a more detailed investigation of the role of reversible ubiquitination of COPI-coatomer and the sorting/retention of GTs is certainly merited (Cohen et al., 2003b). In any case, the retention of Bre5p/Ubp3p clients is mechanistically distinct from that of Snc1p as blocking deubiquitination results in the mislocalization of GTs rather than being required for their Golgi retention as is the case for Snc1p.

In this study we report a novel mechanism for the retention of Golgi resident type II membrane proteins (GTs and Mnn4p) mediated by the deubiquitinase Bre5p/Ubp3p. This finding expands our understanding of the mechanism(s) that govern the retention of Golgi resident integral membrane proteins. Moreover, our findings reveal a heretofore unexpected level of mechanistic complexity. For example, some proteins appear to be exclusive clients of Bre5p/Ubp3p (Och1p) or Vps74p (Kre2p, Mnn2p, Mnn5p, Mnn9p, and Anp1p) while at least three proteins (Ktr3p, Mnn11p, and Mnn4p) are clients of both Vps74p and Bre5p/Ubp3p. It will be important to ascertain whether Vps74p and Bre5p/Ubp3p function on the same Golgi cisterna or in distinct locations. Vps74p, for example, binds to Golgi membranes in a PI4P-dependent manner, and PI4P is predominantly found on the trans-Golgi (Wood et al., 2009). In contrast, although Hoc1p’s mechanism of Golgi retention requires COPI function, it appears to be independent of Vps74p and Bre5p/Ubp3p (Supplemental Figures S2A and S3). The findings for Hoc1p and Mnn11p are particularly intriguing as these GTs reportedly are subunits of a Golgi mannosyltransferase complex together with Anp1p, Mnn9p, and Mnn10p (Jungmann and Munro, 1998). While Anp1p and Mnn9p are both Vps74p clients, Hoc1p is not—our findings therefore raise questions regarding the robustness of the associations of the mannosyltransferases within this complex in cells.

Surprisingly, two GTs—Ktr3p and Mnn11p—showed a requirement for Vps74p that was evident only when cells were grown in YEPD medium. However, neither Ktr3p nor Mnn11p appears to contain a canonical Vps74p-dependent binding motif in its NTDs (see Table 1), and in in vitro binding studies the NTDs of Ktr3p and Mnn11p did not bind to Vps74p. As cells were grown only to early log phase (in either YEPD or SD medium), this cannot be a starvation-induced phenomenon but rather suggests that COPI-mediated trafficking can be influenced by the composition of nutrients made available to the cell, which in turn may impact the growth dynamics of the cels. Additionally, compelling evidence for nutrient-based modulation of the activity of COPI-coatomer comes from the observation that at least two of the Golgi protein reporters—Kre2CT-mNeon and Mnn4CT-mNeon—are localized to the ER when cells are grown in SD medium and to puncta when cells are grown in YEPD medium. This medium-dependent differential localization is dependent on Vps74p (and indirectly COPI) as Kre2CT-mNeon/Kre2CT-GFP and Mnn4CT-mNeon were mislocalized to the vacuole in vps74Δ cells (Figure 3, E and L) (Tu et al., 2008).

Notably, type II proteins that showed a requirement for Bre5p/Ubp3p in their Golgi retention (Ktr3p, Mnn4p, Mnn11p, and Och1p) bound to COPI-coatomer via their NTDs (in biochemical assays), suggesting a direct role for COPI-coatomer in the Bre5p/Ubp3p-dependent retention mechanism. This was also the case for the NTD of Mnn1p—although we were not able to determine whether Mnn1p was a client of Bre5p/Ubp3p as both Mnn1-mNeon and Mnn1CT- mNeon were mislocalized to the vacuole even in WT cells (unpublished data). For two of these Golgi type II proteins (Mnn4p and Ktr3p), the motif that mediates COPI-coatomer binding is composed of a bipartite basic motif. The motifs identified in this study are distinct from the consensus sequence identified in mammalian enzymes “φ-(K/R)-X-L-X(K/R)” and which binds to the delta and gamma subunits of COPI-coatomer (Liu et al., 2018). Moreover, the Ktr3p and Mnn4p motifs bear no obvious similarity to one another—either compositionally or in their “punctuation” within the NTD. Nevertheless, it appears that there is a correlation for a Golgi membrane protein being a client for Bre5p/Ubp3p-dependent retention as each of the four GT (Ktr3p, Mnn4p, Mnn11p, and Och1p) NTDs can mediate COPI-coatomer binding in vitro.

How might ubiquitination/deubiquitination affect the localization of Golgi resident membrane proteins? Amino acid substitutions of the lysine residues in Ktr3p suggest that ubiquitination/deubiquitination of the NTD may not play a direct role in the Bre5p/Ubp3p-dependent retention of this GT in the Golgi (at least for Ktr3p). Rather the data presented herein are suggestive of a direct role for COPI-coatomer. In this regard it is noteworthy that in bre5Δ and ubp3Δ cells evidently only β′-COP (yeast Sec27p) was ubiquitinated (Cohen et al., 2003b)—a finding that suggests a possible role for Sec27p in the Bre5p/Ubp3p-dependent localization of GTs. Glo3p and Gcs1p could be potential substrates too—based on our genetic data—but given that bre5Δ and ubp3Δ cells did not result in more general deficiencies in COPI function (Figure 3, G–L, and Supplemental Figure S2), it is unlikely that either protein is degraded in the absence of the deubiquitinase. Given the limited number of clients involved, protein degradation in the absence of the deubiquitinase is also unlikely in the case of β′-COP (yeast Sec27p) or indeed any other coatomer subunits. We currently favor a model in which the ubiquitination and deubiquitination of COPI-coatomer functions to directly facilitate selection of Golgi membrane proteins such as GTs into nascent vesicles. Given the phenotype observed in bre5Δ and ubp3Δ cells, we postulate that ubiquitination functions to limit or inhibit binding of GTs (as well as other Golgi protein clients) to COPI. Such a mechanism could occur on multiple subunits of coatomer or indeed on Glo3p and or Gcs1p—as all of these proteins are reportedly ubiquitinated (yeastgenome.org). Indeed, basic motifs such as the ones identified here appear to bind to multiple sites on COPI-coatomer (Liu et al., 2018), and given that the motifs identified in Ktr3p and Mnn4p are not particularly similar, it is also possible that they bind to different regions/subunits of COPI. Moreover, evidently the action of Bre5p/Ubp3p is specific to clients bearing basic motifs such as those found in Ktr3p and Mnn4p, rather than the basic motif found in Vps74p—as Vps74p’s clients were unaffected in bre5Δ and ubp3Δ cells (Figure 3, G –L). That said, the relatively short length of the NTDs will clearly rule out some coatomer subunits based on the geometry of coatomer and hence the proximity of potential ubiquitination sites to the Golgi membrane surface.

What is the physiological significance of reversibly controlling access of protein clients to COPI? One intriguing hypothesis is that such a mechanism may provide the cell with a means by which to exclude proteins from incorporation into COPI vesicles in some cisternae, while allowing their selection in others. It has been recognized for some time that functionally distinct pools of COPI function on the cis and medial/trans-Golgi in yeast cells (Wooding and Pelham, 1998). For example, not all Golgi membrane proteins appear to cycle between the Golgi and the ER even though they are mislocalized in COPI/SEC mutants. Significantly those proteins that do cycle between the Golgi and the ER are typically steady-state residents of the cis-Golgi—such as Sed5p, Rer1p, and Emp46p (Sato et al., 1995; Sato and Nakano, 2002; Wooding and Pelham, 1998; Gao and Banfield, 2020). By contrast, the GTs Anp1p, Och1p, and Mnn1p reportedly do not cycle through the ER (Harris and Waters, 1996; Wooding and Pelham, 1998), and of these we have shown that Och1p is a Bre5p/Ubp3p client.

A recent study revealed a role for ubiquitination in the sorting of the R-SNARE Snc1p mediated by the COPI coat (Xu et al., 2017). The authors demonstrated that the ubiquitination of Snc1p was a prerequisite for binding to a region of β′COP (Sec27p) that binds polyubiquitin (Xu et al., 2017). While it remains a possibility that the NTDs of some Bre5p/Ubp3p clients are themselves ubiquitinated, lysine to arginine substitutions in the NTD of Ktr3p had no effect on the retention of Ktr3CT-mNeon in WT or bre5Δ cells (Supplemental Figure S4). Therefore, a more detailed investigation of the role of reversible ubiquitination of COPI-coatomer and the sorting/retention of GTs is certainly merited (Cohen et al., 2003b). In any case, the retention of Bre5p/Ubp3p clients is mechanistically distinct from that of Snc1p as blocking deubiquitination results in the mislocalization of GTs rather than being required for their Golgi retention as is the case for Snc1p.

MATERIALS AND METHODSMaterialsFM 4-64 Dye (N-(3-triethylammoniumpropyl)-4-(6-(4-(diethylamino) phenyl) hexatrienyl) pyridinium dibromide) and IPTG (isopropylthio-β-galactoside) were purchased from Invitrogen. EDTA-free protease inhibitor cocktail, Pefabloc SC (AEBSF), endoglycosidase H, and anti–c-myc antibody were purchased from Roche. DTT (dithiothreitol) was purchased from USB. The anti-Pgk1p and anti-CPY antibodies were purchased from Molecular Probes. The anti-mouse immunoglobulin G (IgG) horseradish peroxidase–conjugated antibody, concanavalin A (C7275), lysozyme, and calcofluor white were purchased from Sigma. The anti-rabbit IgG horseradish peroxidase–linked antibody, Amersham ECL Western Blotting Detection Reagent, and glutathione Sepharose 4B were purchased from GE Healthcare. The anti-Vps74p antibody was from the Banfield lab collection. The anti-coatomer antibody was kindly provided by Anne Spang (University of Basel). The anti-Gas1p antibody was kindly provided by Howard Riezman (University of Geneva, Switzerland). The mNeonGreen coding sequence was kindly provided by Chris Fromme (Cornell University).Yeast protein extraction for CPY and Gas1p detectionYeast cells were grown to mid–log phase, and 107 cells were harvested and resuspended in 100 µl SDS sample buffer supplemented with 1× EDTA-free protease inhibitor cocktail, 1 mM Pefabloc SC, and 5 mM DTT. Cells were lysed by the addition of acid-washed glass beads and treatment with a MINI-BEADBEATER (Biospec Products, USA) for 5 min at 4°C. The lysate supernatants were incubated at 95°C for 10 min, and 10 µl samples (equivalent to 106 cells) were loaded onto 10% SDS–polyacrylamide gels. Following SDS–PAGE, the proteins were transferred to nitrocellulose membranes, which were then immunoblotted with anti-Gas1p, anti-CPY, and anti-Pgk1p antibodies.Staining of the vacuole limiting membraneYeast cells were grown to early–log phase, and cells from a 0.5 ml culture were harvested and resuspended in 50 µl YEPD medium containing 20 µM FM 4-64 Dye (FM4-64). The cell suspensions were incubated at 30°C for 20 min. After the incubation, cells were washed (to remove excess dye) by centrifugation and cell pellets were resuspended in 1 ml YEPD medium. The culture was then incubated (with aeration) for 90 min, after which the cells were mounted on glass slides and examined by fluorescence microscopy.Fluorescence microscopyYeast knock-in strains or transformants were grown to mid–log phase, and cells were collected by centrifugation and washed once with ddH2O before being resuspended in ddH2O to a final concentration of 105 cells per µl. Yeast cell suspension (0.7 µl) was applied to a glass slide precoated with Concanavalin A and a cover slide, and nail polish was used to seal the sample. Samples were immediately visualized by a Nikon Eclipse 80i (Nikon, Japan), and the photos were obtained by a Spot RT3 monochrome digital camera (Diagnostic Instruments, Sterling Heights, MI). Images were processed using Adobe Photoshop (Adobe Photoshop CC 2019).Measurement of the steady-state level of GTsYeast cells expressing 9xmyc-tagged GTs (see Table 2) were grown to mid–log phase, and 107 cells were harvested by centrifugation. Cell pellets were subjected to alkaline lysis, whereby cells were resuspended in 200 µl 0.1 M NaOH and incubated for 5 min. Excess NaOH was removed by centrifugation, and cell pellets were resuspended in 100 µl SDS sample buffer supplemented with 1× EDTA-free protease inhibitor cocktail, 1 mM Pefabloc SC, and 5 mM DTT. Cells were lysed by heating to 95°C for 10 min, and 10 µl lysates (equivalent to 106 lysates) were loaded onto 10% SDS–polyacrylamide gels. Following SDS–PAGE, the proteins were transferred to nitrocellulose membranes, which were then immunoblotted with anti-myc and anti-Pgk1p antibodies. The steady-state levels of GTs were normalized with corresponding Pgk1p levels by ImageJ software (National Institutes of Health).Endoglycosidase H treatmentYeast cells (107 mid–log phase) were harvested, and the pellets were treated with 200 µl 0.1 M NaOH for 5 min. After NaOH was removed by centrifugation, 100 µl SDS sample buffer supplemented with 1× EDTA-free protease inhibitor, 1 mM Pefabloc SC, and 5 mM DTT was used to resuspend the pellets, and the samples were incubated at 95°C for 10 min. The heat-treated samples (20 µl) were supplemented with 80 mM potassium acetate (pH 5.56) and 0.001 U endoglycosidase H and incubated at 37°C for 2 h. Samples (10 µl) were loaded onto 10% SDS –polyacrylamide gels for subsequent immunoblotting.ImmunoblottingThe nitrocellulose membranes were incubated in PBST buffer (phosphate-buffered saline [PBS] containing 0.1% Tween-20) with 5% nonfat milk powder for 1 h, after which the membranes were incubated with corresponding primary antibodies in PBST buffer for 1 h followed by three washes with PBST. After being washed, the membranes were incubated with secondary antibodies in PBST buffer for 1 h and then washed with PBST three times. Amersham ECL Western Blotting Detection Reagent and autoradiography or the Chemidoc Imaging System (Bio-Rad, USA) were used to detect the signals.Protein expression and preparation of bacterial lysatesGST fusion protein constructs (see Table 3) were transformed into Escherichia coli BL21(DE3) cells. Transformed BL21 cells were grown to OD600 0.6–0.8 at 37°C in 200 ml Lysogeny broth (LB) before protein expression induction. Protein expression was induced by the addition of 0.2 mM IPTG at 37°C for 4 h. E. coli cells from 200 ml cultures were harvested by centrifugation, washed with 5 ml ddH2O, and resuspended in 5 ml buffer A (25 mM Tris-HCl, pH 7.5, 400 mM KCl, 10% glycerol, 4% Triton X-100) supplemented with 1× EDTA-free protease inhibitor cocktail, 1 mM Pefabloc SC, 5 mM DTT, and 250 µg/ml lysozyme. Cells were lysed using a Q125 Sonicator (QSONICA, USA) at 4°C for 8 min (2 s on, 2 s off, 40% amplitude). Bacterial lysates were cleared by centrifugation at 13,000 × g at 4°C for 10 min.Preparation of yeast WCEsYeast cells were grown to mid–log phase in 100 ml YEPD medium and harvested by centrifugation. Cell pellets were washed with 5 ml ddH2O and resuspended with 5 ml PBS supplemented with 0.1% Triton X-100, 1× EDTA-free protease inhibitor cocktail, 1 mM Pefabloc SC, and 5 mM DTT. The cells were mechanically disrupted with glass beads using the BEAD-BEATER (Model 1107900; BioSpec Products, USA) for 3 min followed by 3 min cooling down and another 3 min beating. The extracts were cleared by centrifugation at 13,000 × g for 10 min at 4°C.GST pull-down assaysGlutathione Sepharose 4B beads (50 µl) were equilibrated with buffer A and then mixed with E.coli lysate containing GST fusion proteins from 108 cells (see Protein expression and preparation of bacterial lysates). The mixture was incubated with continuous rotation at 4°C for 1 h. Beads were sedimented by centrifugation and washed by resuspension in 500 µl buffer A twice and then in PBS 500 µl buffer containing 0.1% Triton X-100 twice. Yeast WCEs from 108 cells prepared by the steps mentioned above were mixed with washed beads, and the mixture was incubated with continuous rotation at 4°C for 2 h. After the incubation, the beads were washed with 500 µl PBS containing 0.1% Triton X-100 four times and resuspended with 100 µl SDS sample buffer. Samples were heated to 95°C for 10 min, and 10 µl of the heat-treated samples was loaded onto 10% SDS–polyacrylamide gels. After the proteins were transferred to nitrocellulose, the membranes were immunoblotted with anti-coatomer and anti-Vps74p antibodies.

MATERIALS AND METHODS

MaterialsFM 4-64 Dye (N-(3-triethylammoniumpropyl)-4-(6-(4-(diethylamino) phenyl) hexatrienyl) pyridinium dibromide) and IPTG (isopropylthio-β-galactoside) were purchased from Invitrogen. EDTA-free protease inhibitor cocktail, Pefabloc SC (AEBSF), endoglycosidase H, and anti–c-myc antibody were purchased from Roche. DTT (dithiothreitol) was purchased from USB. The anti-Pgk1p and anti-CPY antibodies were purchased from Molecular Probes. The anti-mouse immunoglobulin G (IgG) horseradish peroxidase–conjugated antibody, concanavalin A (C7275), lysozyme, and calcofluor white were purchased from Sigma. The anti-rabbit IgG horseradish peroxidase–linked antibody, Amersham ECL Western Blotting Detection Reagent, and glutathione Sepharose 4B were purchased from GE Healthcare. The anti-Vps74p antibody was from the Banfield lab collection. The anti-coatomer antibody was kindly provided by Anne Spang (University of Basel). The anti-Gas1p antibody was kindly provided by Howard Riezman (University of Geneva, Switzerland). The mNeonGreen coding sequence was kindly provided by Chris Fromme (Cornell University).

FM 4-64 Dye (N-(3-triethylammoniumpropyl)-4-(6-(4-(diethylamino) phenyl) hexatrienyl) pyridinium dibromide) and IPTG (isopropylthio-β-galactoside) were purchased from Invitrogen. EDTA-free protease inhibitor cocktail, Pefabloc SC (AEBSF), endoglycosidase H, and anti–c-myc antibody were purchased from Roche. DTT (dithiothreitol) was purchased from USB. The anti-Pgk1p and anti-CPY antibodies were purchased from Molecular Probes. The anti-mouse immunoglobulin G (IgG) horseradish peroxidase–conjugated antibody, concanavalin A (C7275), lysozyme, and calcofluor white were purchased from Sigma. The anti-rabbit IgG horseradish peroxidase–linked antibody, Amersham ECL Western Blotting Detection Reagent, and glutathione Sepharose 4B were purchased from GE Healthcare. The anti-Vps74p antibody was from the Banfield lab collection. The anti-coatomer antibody was kindly provided by Anne Spang (University of Basel). The anti-Gas1p antibody was kindly provided by Howard Riezman (University of Geneva, Switzerland). The mNeonGreen coding sequence was kindly provided by Chris Fromme (Cornell University).

Yeast protein extraction for CPY and Gas1p detectionYeast cells were grown to mid–log phase, and 107 cells were harvested and resuspended in 100 µl SDS sample buffer supplemented with 1× EDTA-free protease inhibitor cocktail, 1 mM Pefabloc SC, and 5 mM DTT. Cells were lysed by the addition of acid-washed glass beads and treatment with a MINI-BEADBEATER (Biospec Products, USA) for 5 min at 4°C. The lysate supernatants were incubated at 95°C for 10 min, and 10 µl samples (equivalent to 106 cells) were loaded onto 10% SDS–polyacrylamide gels. Following SDS–PAGE, the proteins were transferred to nitrocellulose membranes, which were then immunoblotted with anti-Gas1p, anti-CPY, and anti-Pgk1p antibodies.

Yeast protein extraction for CPY and Gas1p detection

Yeast cells were grown to mid–log phase, and 107 cells were harvested and resuspended in 100 µl SDS sample buffer supplemented with 1× EDTA-free protease inhibitor cocktail, 1 mM Pefabloc SC, and 5 mM DTT. Cells were lysed by the addition of acid-washed glass beads and treatment with a MINI-BEADBEATER (Biospec Products, USA) for 5 min at 4°C. The lysate supernatants were incubated at 95°C for 10 min, and 10 µl samples (equivalent to 106 cells) were loaded onto 10% SDS–polyacrylamide gels. Following SDS–PAGE, the proteins were transferred to nitrocellulose membranes, which were then immunoblotted with anti-Gas1p, anti-CPY, and anti-Pgk1p antibodies.

Staining of the vacuole limiting membraneYeast cells were grown to early–log phase, and cells from a 0.5 ml culture were harvested and resuspended in 50 µl YEPD medium containing 20 µM FM 4-64 Dye (FM4-64). The cell suspensions were incubated at 30°C for 20 min. After the incubation, cells were washed (to remove excess dye) by centrifugation and cell pellets were resuspended in 1 ml YEPD medium. The culture was then incubated (with aeration) for 90 min, after which the cells were mounted on glass slides and examined by fluorescence microscopy.

Staining of the vacuole limiting membrane

Yeast cells were grown to early–log phase, and cells from a 0.5 ml culture were harvested and resuspended in 50 µl YEPD medium containing 20 µM FM 4-64 Dye (FM4-64). The cell suspensions were incubated at 30°C for 20 min. After the incubation, cells were washed (to remove excess dye) by centrifugation and cell pellets were resuspended in 1 ml YEPD medium. The culture was then incubated (with aeration) for 90 min, after which the cells were mounted on glass slides and examined by fluorescence microscopy.

Fluorescence microscopyYeast knock-in strains or transformants were grown to mid–log phase, and cells were collected by centrifugation and washed once with ddH2O before being resuspended in ddH2O to a final concentration of 105 cells per µl. Yeast cell suspension (0.7 µl) was applied to a glass slide precoated with Concanavalin A and a cover slide, and nail polish was used to seal the sample. Samples were immediately visualized by a Nikon Eclipse 80i (Nikon, Japan), and the photos were obtained by a Spot RT3 monochrome digital camera (Diagnostic Instruments, Sterling Heights, MI). Images were processed using Adobe Photoshop (Adobe Photoshop CC 2019).

Fluorescence microscopy

Yeast knock-in strains or transformants were grown to mid–log phase, and cells were collected by centrifugation and washed once with ddH2O before being resuspended in ddH2O to a final concentration of 105 cells per µl. Yeast cell suspension (0.7 µl) was applied to a glass slide precoated with Concanavalin A and a cover slide, and nail polish was used to seal the sample. Samples were immediately visualized by a Nikon Eclipse 80i (Nikon, Japan), and the photos were obtained by a Spot RT3 monochrome digital camera (Diagnostic Instruments, Sterling Heights, MI). Images were processed using Adobe Photoshop (Adobe Photoshop CC 2019).

Measurement of the steady-state level of GTsYeast cells expressing 9xmyc-tagged GTs (see Table 2) were grown to mid–log phase, and 107 cells were harvested by centrifugation. Cell pellets were subjected to alkaline lysis, whereby cells were resuspended in 200 µl 0.1 M NaOH and incubated for 5 min. Excess NaOH was removed by centrifugation, and cell pellets were resuspended in 100 µl SDS sample buffer supplemented with 1× EDTA-free protease inhibitor cocktail, 1 mM Pefabloc SC, and 5 mM DTT. Cells were lysed by heating to 95°C for 10 min, and 10 µl lysates (equivalent to 106 lysates) were loaded onto 10% SDS–polyacrylamide gels. Following SDS–PAGE, the proteins were transferred to nitrocellulose membranes, which were then immunoblotted with anti-myc and anti-Pgk1p antibodies. The steady-state levels of GTs were normalized with corresponding Pgk1p levels by ImageJ software (National Institutes of Health).

Measurement of the steady-state level of GTs

Yeast cells expressing 9xmyc-tagged GTs (see Table 2) were grown to mid–log phase, and 107 cells were harvested by centrifugation. Cell pellets were subjected to alkaline lysis, whereby cells were resuspended in 200 µl 0.1 M NaOH and incubated for 5 min. Excess NaOH was removed by centrifugation, and cell pellets were resuspended in 100 µl SDS sample buffer supplemented with 1× EDTA-free protease inhibitor cocktail, 1 mM Pefabloc SC, and 5 mM DTT. Cells were lysed by heating to 95°C for 10 min, and 10 µl lysates (equivalent to 106 lysates) were loaded onto 10% SDS–polyacrylamide gels. Following SDS–PAGE, the proteins were transferred to nitrocellulose membranes, which were then immunoblotted with anti-myc and anti-Pgk1p antibodies. The steady-state levels of GTs were normalized with corresponding Pgk1p levels by ImageJ software (National Institutes of Health).

Endoglycosidase H treatmentYeast cells (107 mid–log phase) were harvested, and the pellets were treated with 200 µl 0.1 M NaOH for 5 min. After NaOH was removed by centrifugation, 100 µl SDS sample buffer supplemented with 1× EDTA-free protease inhibitor, 1 mM Pefabloc SC, and 5 mM DTT was used to resuspend the pellets, and the samples were incubated at 95°C for 10 min. The heat-treated samples (20 µl) were supplemented with 80 mM potassium acetate (pH 5.56) and 0.001 U endoglycosidase H and incubated at 37°C for 2 h. Samples (10 µl) were loaded onto 10% SDS –polyacrylamide gels for subsequent immunoblotting.

Endoglycosidase H treatment

Yeast cells (107 mid–log phase) were harvested, and the pellets were treated with 200 µl 0.1 M NaOH for 5 min. After NaOH was removed by centrifugation, 100 µl SDS sample buffer supplemented with 1× EDTA-free protease inhibitor, 1 mM Pefabloc SC, and 5 mM DTT was used to resuspend the pellets, and the samples were incubated at 95°C for 10 min. The heat-treated samples (20 µl) were supplemented with 80 mM potassium acetate (pH 5.56) and 0.001 U endoglycosidase H and incubated at 37°C for 2 h. Samples (10 µl) were loaded onto 10% SDS –polyacrylamide gels for subsequent immunoblotting.

ImmunoblottingThe nitrocellulose membranes were incubated in PBST buffer (phosphate-buffered saline [PBS] containing 0.1% Tween-20) with 5% nonfat milk powder for 1 h, after which the membranes were incubated with corresponding primary antibodies in PBST buffer for 1 h followed by three washes with PBST. After being washed, the membranes were incubated with secondary antibodies in PBST buffer for 1 h and then washed with PBST three times. Amersham ECL Western Blotting Detection Reagent and autoradiography or the Chemidoc Imaging System (Bio-Rad, USA) were used to detect the signals.

Immunoblotting

The nitrocellulose membranes were incubated in PBST buffer (phosphate-buffered saline [PBS] containing 0.1% Tween-20) with 5% nonfat milk powder for 1 h, after which the membranes were incubated with corresponding primary antibodies in PBST buffer for 1 h followed by three washes with PBST. After being washed, the membranes were incubated with secondary antibodies in PBST buffer for 1 h and then washed with PBST three times. Amersham ECL Western Blotting Detection Reagent and autoradiography or the Chemidoc Imaging System (Bio-Rad, USA) were used to detect the signals.

Protein expression and preparation of bacterial lysatesGST fusion protein constructs (see Table 3) were transformed into Escherichia coli BL21(DE3) cells. Transformed BL21 cells were grown to OD600 0.6–0.8 at 37°C in 200 ml Lysogeny broth (LB) before protein expression induction. Protein expression was induced by the addition of 0.2 mM IPTG at 37°C for 4 h. E. coli cells from 200 ml cultures were harvested by centrifugation, washed with 5 ml ddH2O, and resuspended in 5 ml buffer A (25 mM Tris-HCl, pH 7.5, 400 mM KCl, 10% glycerol, 4% Triton X-100) supplemented with 1× EDTA-free protease inhibitor cocktail, 1 mM Pefabloc SC, 5 mM DTT, and 250 µg/ml lysozyme. Cells were lysed using a Q125 Sonicator (QSONICA, USA) at 4°C for 8 min (2 s on, 2 s off, 40% amplitude). Bacterial lysates were cleared by centrifugation at 13,000 × g at 4°C for 10 min.

Protein expression and preparation of bacterial lysates

GST fusion protein constructs (see Table 3) were transformed into Escherichia coli BL21(DE3) cells. Transformed BL21 cells were grown to OD600 0.6–0.8 at 37°C in 200 ml Lysogeny broth (LB) before protein expression induction. Protein expression was induced by the addition of 0.2 mM IPTG at 37°C for 4 h. E. coli cells from 200 ml cultures were harvested by centrifugation, washed with 5 ml ddH2O, and resuspended in 5 ml buffer A (25 mM Tris-HCl, pH 7.5, 400 mM KCl, 10% glycerol, 4% Triton X-100) supplemented with 1× EDTA-free protease inhibitor cocktail, 1 mM Pefabloc SC, 5 mM DTT, and 250 µg/ml lysozyme. Cells were lysed using a Q125 Sonicator (QSONICA, USA) at 4°C for 8 min (2 s on, 2 s off, 40% amplitude). Bacterial lysates were cleared by centrifugation at 13,000 × g at 4°C for 10 min.

Preparation of yeast WCEsYeast cells were grown to mid–log phase in 100 ml YEPD medium and harvested by centrifugation. Cell pellets were washed with 5 ml ddH2O and resuspended with 5 ml PBS supplemented with 0.1% Triton X-100, 1× EDTA-free protease inhibitor cocktail, 1 mM Pefabloc SC, and 5 mM DTT. The cells were mechanically disrupted with glass beads using the BEAD-BEATER (Model 1107900; BioSpec Products, USA) for 3 min followed by 3 min cooling down and another 3 min beating. The extracts were cleared by centrifugation at 13,000 × g for 10 min at 4°C.

Preparation of yeast WCEs

Yeast cells were grown to mid–log phase in 100 ml YEPD medium and harvested by centrifugation. Cell pellets were washed with 5 ml ddH2O and resuspended with 5 ml PBS supplemented with 0.1% Triton X-100, 1× EDTA-free protease inhibitor cocktail, 1 mM Pefabloc SC, and 5 mM DTT. The cells were mechanically disrupted with glass beads using the BEAD-BEATER (Model 1107900; BioSpec Products, USA) for 3 min followed by 3 min cooling down and another 3 min beating. The extracts were cleared by centrifugation at 13,000 × g for 10 min at 4°C.

GST pull-down assaysGlutathione Sepharose 4B beads (50 µl) were equilibrated with buffer A and then mixed with E.coli lysate containing GST fusion proteins from 108 cells (see Protein expression and preparation of bacterial lysates). The mixture was incubated with continuous rotation at 4°C for 1 h. Beads were sedimented by centrifugation and washed by resuspension in 500 µl buffer A twice and then in PBS 500 µl buffer containing 0.1% Triton X-100 twice. Yeast WCEs from 108 cells prepared by the steps mentioned above were mixed with washed beads, and the mixture was incubated with continuous rotation at 4°C for 2 h. After the incubation, the beads were washed with 500 µl PBS containing 0.1% Triton X-100 four times and resuspended with 100 µl SDS sample buffer. Samples were heated to 95°C for 10 min, and 10 µl of the heat-treated samples was loaded onto 10% SDS–polyacrylamide gels. After the proteins were transferred to nitrocellulose, the membranes were immunoblotted with anti-coatomer and anti-Vps74p antibodies.

GST pull-down assays

Glutathione Sepharose 4B beads (50 µl) were equilibrated with buffer A and then mixed with E.coli lysate containing GST fusion proteins from 108 cells (see Protein expression and preparation of bacterial lysates). The mixture was incubated with continuous rotation at 4°C for 1 h. Beads were sedimented by centrifugation and washed by resuspension in 500 µl buffer A twice and then in PBS 500 µl buffer containing 0.1% Triton X-100 twice. Yeast WCEs from 108 cells prepared by the steps mentioned above were mixed with washed beads, and the mixture was incubated with continuous rotation at 4°C for 2 h. After the incubation, the beads were washed with 500 µl PBS containing 0.1% Triton X-100 four times and resuspended with 100 µl SDS sample buffer. Samples were heated to 95°C for 10 min, and 10 µl of the heat-treated samples was loaded onto 10% SDS–polyacrylamide gels. After the proteins were transferred to nitrocellulose, the membranes were immunoblotted with anti-coatomer and anti-Vps74p antibodies.

Supplementary MaterialClick here for additional data file.Click here for additional data file.

Supplementary Material

Click here for additional data file.

Click here for additional data file.
