---
pmid: '21170087'
title: Ribosomal protein S27-like and S27 interplay with p53-MDM2 axis as a target,
  a substrate and a regulator.
authors:
- Xiong X
- Zhao Y
- He H
- Sun Y
journal: Oncogene
year: '2011'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC3077453
doi: 10.1038/onc.2010.569
---

# Ribosomal protein S27-like and S27 interplay with p53-MDM2 axis as a target, a substrate and a regulator.
**Authors:** Xiong X, Zhao Y, He H, Sun Y
**Journal:** Oncogene (2011)
**DOI:** [10.1038/onc.2010.569](https://doi.org/10.1038/onc.2010.569)
**PMC:** [PMC3077453](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3077453/)

## Abstract

1. Oncogene. 2011 Apr 14;30(15):1798-811. doi: 10.1038/onc.2010.569. Epub 2010
Dec  20.

Ribosomal protein S27-like and S27 interplay with p53-MDM2 axis as a target, a 
substrate and a regulator.

Xiong X(1), Zhao Y, He H, Sun Y.

Author information:
(1)Division of Radiation and Cancer Biology, Department of Radiation Oncology, 
University of Michigan, Ann Arbor, MI 48109, USA.

Several ribosomal proteins regulate p53 function by modulating MDM2. We recently 
found that RPS27L, a RPS27-like protein, is a direct p53-inducible target. Here 
we showed that RPS27 itself is a p53-repressible target. Furthermore, the 
N-terminal region of either RPS27L or RPS27 binds to MDM2 on the central acidic 
domain of MDM2. RPS27L or RPS27 forms an in vivo triplex with MDM2-p53 and 
competes with p53 for MDM2 binding. Similar to p53, RPS27L, but not RPS27, is a 
short-lived protein and a novel MDM2 substrate. Degradation of RPS27L requires 
the RING or acidic domain of MDM2. Ectopic expression of RPS27L or RPS27 
inhibits MDM-2-mediated p53 ubiquitination and increases p53 levels by extending 
p53 protein half-life, whereas siRNA silencing of RPS27L decreases p53 levels by 
shortening p53 half-life, with a corresponding reduction in p53 transcription 
activity. RPS27L is mainly localized in the cytoplasm, but upon p53-activating 
signals, a portion of RPS27L shuttled to the nucleoplasm where it colocalizes 
with MDM2. Both the cytoplasmic and the nuclear p53, induced by ribosomal 
stress, were reduced upon RPS27L silencing. Our study reveals a multilevel 
interplay between RPS27L/S27 and p53-MDM2 axis, with RPS27L functioning as a p53 
target, a MDM2 substrate and a p53 regulator.

DOI: 10.1038/onc.2010.569
PMCID: PMC3077453
PMID: 21170087 [Indexed for MEDLINE]

Conflict of interest statement: Conflict of interest The authors declare no 
conflict of interest.

## Full Text

IntroductionThe primary role of ribosomal proteins is to assemble with rRNA for new protein synthesis. The entire ribosome, consisting of 4 rRNA species (5S, 5.8S, 18S, 28S) and approximately 75 ribosomal proteins, is first assembled in the nucleolus as two precursor subunits (40S small subunit and 60S large subunit). The ribosomal precursors then move to the cytoplasm for complete processing into ribosomes and polysomes for protein synthesis. The mature 40S ribosomal subunit contains the 18S rRNA and ~32 ribosomal proteins, whereas the 60S ribosomal subunit consists of the 5S, 5.8S, and 28S rRNA and ~45 ribosomal proteins (Kressler et al 1999, Venema and Tollervey 1999). Recently, some ribosomal proteins, such as L5, L11, L23, and S7, were found to regulate cell cycle progression and apoptosis through binding to and inhibiting MDM2, leading to activation of the p53 tumor suppressor (Bhat et al 2004, Chen et al 2007, Dai and Lu 2004, Dai et al 2004, Jin et al 2004, Lindstrom et al 2007, Lohrum et al 2003, Zhang et al 2003, Zhang and Lu 2009, Zhu et al 2009). Thus, some ribosomal proteins have an extended role in addition to protein synthesis.RPS27L (NM_015920) is a uncharacterized novel protein, which was recently identified as a p53 inducible target by our group (He and Sun 2007) and confirmed by another group (Li et al 2007). RPS27L differs from its family member RPS27 (Ribosomal Protein S27) (NM_001030) by only three amino acids (R5K, L12P, K17R) at the N-terminus. Human RPS27L and RPS27 were mapped onto chromosome 15q22.1 and 1q21, respectively, whereas mouse RPS27L and RPS27 were mapped onto chromosome 9 and 3, respectively. The similarity of these two RPS27L and RPS27 loci at the amino acid level suggests that either RPS27 or RPS27L arose via duplication of the other locus (Balasubramanian et al 2009).Although little is known about RPS27L, a number of studies have revealed the cancer relevance of its family member, RPS27. RPS27, also called metallopanstimulin-1 (MPS-1), was first identified as a growth factor inducible gene which encodes a 9.5 kDa protein of 84 amino acids with a zinc finger-like motif (Chan et al 1993, Fernandez-Pol et al 1993, Wong et al 1993). RPS27/MPS-1 is overexpressed in large quantities in various proliferating tissues (Fernandez-Pol et al 1994) and in multiple human cancers, including carcinomas of colon (Ganger et al 1997), prostate (Fernandez-Pol et al 1997), breast (Atsuta et al 2002), liver (Ganger et al 2001), stomach (Wang et al 2006) and head and neck (Lee et al 2004). A recent study showed that RPS27/MPS-1 was overexpressed in 86% of gastric cancer tissues, and that its overexpression strongly correlated with tumor-node metastasis (Wang et al 2006). SiRNA silencing of RPS27/MPS-1 in a gastric cancer cell line induced spontaneous apoptosis, and inhibited cell growth in monolayer cultures and in nude mouse xenograft tumors (Wang et al 2006). Furthermore, when overexpressed in cancer tissues, RPS27/MPS-1 is released by secretion or gradient diffusion into the extracellular space which allows for its measurement in serum (Lee et al 2004). Upon release, RPS27/MPS-1 binds to albumin in serum and can be detected by radio-immuno assay or western blotting as a 75 kDa band (Fernandez-Pol 1996, Lee et al 2004). The level of RPS27/MPS-1 was found to be elevated in the sera of patients with various types of common cancers (Fernandez-Pol 1996, Lee et al 2004, Stack et al 2004). It was, therefore, proposed that RPS27/MPS-1 can serve as a tumor marker, particularly for head and neck squamous cell carcinoma (Fernandez-Pol 1996, Lee et al 2004, Stack et al 2004). However, the molecular mechanisms of RPS27 and RPS27L action remain largely elusive.Here we report an interesting interplay among p53, MDM2, and RPS27L or RPS27. It is well-established that p53-MDM2 forms a negative auto-regulatory feedback loop in which p53 transactivates MDM2, whereas induced MDM2 binds to p53 and promotes p53 degradation (Wu et al 1993). RPS27L and RPS27 join this axis as a target, a substrate or a regulator of p53-MDM2. First, in contrast to RPS27L, which is subjected to p53 induction (He and Sun 2007, Li et al 2007), RPS27 is a p53 repressed gene, indicating that like MDM2, both ribosomal proteins are p53 downstream targets. Second, either RPS27L or RPS27 binds to MDM2, and subjected, particularly RPS27L, to Mdm-2 mediated ubiquitination and degradation. Third, either RPS27L or RPS27 competes with p53 for MDM2 binding, and consequently, inhibiting MDM2-mediated p53 ubiquitination and degradation. Fourth, while ectopic expression of RPS27L or RPS27 increases p53 levels, siRNA silencing of RPS27L, but not RPS27 destabilizes p53. Finally, RPS27L is mainly localized in cytoplasm under unstressed condition, but a portion of RPS27L is shuttled to nucleus, upon DNA damage and ribosomal stress, where it co-localized with MDM2. Both cytoplasmic and nucleoplasmic p53, induced by actinomycin D, were attenuated upon RPS27L silencing. Taken together, our study showed that although several ribosomal proteins have been previously shown to bind and inhibit MDM2 for p53 activation (Lindstrom et al 2007, Zhang and Lu 2009) with RPS7 and RPL26 being MDM2 substrates (Ofir-Rosenfeld et al 2008, Zhu et al 2009), RPS27 and RPS27L are the first class of ribosomal or ribosomal-like proteins that are direct p53 targets as well as MDM2 substrate (in case of RPS27L) and that regulate p53 via MDM2.

Introduction

The primary role of ribosomal proteins is to assemble with rRNA for new protein synthesis. The entire ribosome, consisting of 4 rRNA species (5S, 5.8S, 18S, 28S) and approximately 75 ribosomal proteins, is first assembled in the nucleolus as two precursor subunits (40S small subunit and 60S large subunit). The ribosomal precursors then move to the cytoplasm for complete processing into ribosomes and polysomes for protein synthesis. The mature 40S ribosomal subunit contains the 18S rRNA and ~32 ribosomal proteins, whereas the 60S ribosomal subunit consists of the 5S, 5.8S, and 28S rRNA and ~45 ribosomal proteins (Kressler et al 1999, Venema and Tollervey 1999). Recently, some ribosomal proteins, such as L5, L11, L23, and S7, were found to regulate cell cycle progression and apoptosis through binding to and inhibiting MDM2, leading to activation of the p53 tumor suppressor (Bhat et al 2004, Chen et al 2007, Dai and Lu 2004, Dai et al 2004, Jin et al 2004, Lindstrom et al 2007, Lohrum et al 2003, Zhang et al 2003, Zhang and Lu 2009, Zhu et al 2009). Thus, some ribosomal proteins have an extended role in addition to protein synthesis.

RPS27L (NM_015920) is a uncharacterized novel protein, which was recently identified as a p53 inducible target by our group (He and Sun 2007) and confirmed by another group (Li et al 2007). RPS27L differs from its family member RPS27 (Ribosomal Protein S27) (NM_001030) by only three amino acids (R5K, L12P, K17R) at the N-terminus. Human RPS27L and RPS27 were mapped onto chromosome 15q22.1 and 1q21, respectively, whereas mouse RPS27L and RPS27 were mapped onto chromosome 9 and 3, respectively. The similarity of these two RPS27L and RPS27 loci at the amino acid level suggests that either RPS27 or RPS27L arose via duplication of the other locus (Balasubramanian et al 2009).

Although little is known about RPS27L, a number of studies have revealed the cancer relevance of its family member, RPS27. RPS27, also called metallopanstimulin-1 (MPS-1), was first identified as a growth factor inducible gene which encodes a 9.5 kDa protein of 84 amino acids with a zinc finger-like motif (Chan et al 1993, Fernandez-Pol et al 1993, Wong et al 1993). RPS27/MPS-1 is overexpressed in large quantities in various proliferating tissues (Fernandez-Pol et al 1994) and in multiple human cancers, including carcinomas of colon (Ganger et al 1997), prostate (Fernandez-Pol et al 1997), breast (Atsuta et al 2002), liver (Ganger et al 2001), stomach (Wang et al 2006) and head and neck (Lee et al 2004). A recent study showed that RPS27/MPS-1 was overexpressed in 86% of gastric cancer tissues, and that its overexpression strongly correlated with tumor-node metastasis (Wang et al 2006). SiRNA silencing of RPS27/MPS-1 in a gastric cancer cell line induced spontaneous apoptosis, and inhibited cell growth in monolayer cultures and in nude mouse xenograft tumors (Wang et al 2006). Furthermore, when overexpressed in cancer tissues, RPS27/MPS-1 is released by secretion or gradient diffusion into the extracellular space which allows for its measurement in serum (Lee et al 2004). Upon release, RPS27/MPS-1 binds to albumin in serum and can be detected by radio-immuno assay or western blotting as a 75 kDa band (Fernandez-Pol 1996, Lee et al 2004). The level of RPS27/MPS-1 was found to be elevated in the sera of patients with various types of common cancers (Fernandez-Pol 1996, Lee et al 2004, Stack et al 2004). It was, therefore, proposed that RPS27/MPS-1 can serve as a tumor marker, particularly for head and neck squamous cell carcinoma (Fernandez-Pol 1996, Lee et al 2004, Stack et al 2004). However, the molecular mechanisms of RPS27 and RPS27L action remain largely elusive.

Here we report an interesting interplay among p53, MDM2, and RPS27L or RPS27. It is well-established that p53-MDM2 forms a negative auto-regulatory feedback loop in which p53 transactivates MDM2, whereas induced MDM2 binds to p53 and promotes p53 degradation (Wu et al 1993). RPS27L and RPS27 join this axis as a target, a substrate or a regulator of p53-MDM2. First, in contrast to RPS27L, which is subjected to p53 induction (He and Sun 2007, Li et al 2007), RPS27 is a p53 repressed gene, indicating that like MDM2, both ribosomal proteins are p53 downstream targets. Second, either RPS27L or RPS27 binds to MDM2, and subjected, particularly RPS27L, to Mdm-2 mediated ubiquitination and degradation. Third, either RPS27L or RPS27 competes with p53 for MDM2 binding, and consequently, inhibiting MDM2-mediated p53 ubiquitination and degradation. Fourth, while ectopic expression of RPS27L or RPS27 increases p53 levels, siRNA silencing of RPS27L, but not RPS27 destabilizes p53. Finally, RPS27L is mainly localized in cytoplasm under unstressed condition, but a portion of RPS27L is shuttled to nucleus, upon DNA damage and ribosomal stress, where it co-localized with MDM2. Both cytoplasmic and nucleoplasmic p53, induced by actinomycin D, were attenuated upon RPS27L silencing. Taken together, our study showed that although several ribosomal proteins have been previously shown to bind and inhibit MDM2 for p53 activation (Lindstrom et al 2007, Zhang and Lu 2009) with RPS7 and RPL26 being MDM2 substrates (Ofir-Rosenfeld et al 2008, Zhu et al 2009), RPS27 and RPS27L are the first class of ribosomal or ribosomal-like proteins that are direct p53 targets as well as MDM2 substrate (in case of RPS27L) and that regulate p53 via MDM2.

ResultsRPS27 and RPS27L are evolutionarily conserved proteinsRPS27 and its family member, RPS27L are 84 amino acids-containing proteins with a zinc-finger-like domain. Two family members differ from each other by only three amino acids (R5K, L12P, K17R) at the N-terminus (Fig 1A). We compared the protein sequences of RPS27 and RPS27L in more than 10 species. As shown in Figure 1B, both family members are highly conserved during evolution, suggesting their functional significance. In addition to human, the R5K/L12P/K17R conservation of two family members is also found in mouse, rat, and dog. Like in humans, there are two members of RPS27/S27L family in yeast S. cerevisiae, RPS27A (YKL156W) and RPS27B (YHR021C), differing by only one amino acid at codon 62 (I62V). The yeast with systematic deletion of either RPS27A or RPS27B is still viable, although deletion of RPS27B showed some growth defect (Giaever et al 2002) (also see www.yeastgenome.org). In C-elegans, however, there is only one family member (RPS27). SiRNA silencing lead to embryonic lethality (Sonnichsen et al 2005) (also see www.wormgenome.org). In Drosophila, one family member is identified. Its molecular function is described as structural constituent of ribosome, nucleic acid binding (see http://flybase.bio.indiana.edu/reports/FBgn0039300.html) and it is involved in the biological process of translation.To facilitate the detection of endogenous RPS27L and RPS27 selectively, we generated, affinity purified, and characterized two antibodies specifically against either one of the family members. As shown in Figure 1C, the RPS27L antibody detected both endogenous and HA- tagged RPS27L (S27L-HA), but not HA-tagged RPS27 (S27-HA) (top panel), whereas the RPS27 antibody detected both endogenous and S27-HA, but not S27L-HA (second panel). As expected, anti-HA tag antibody detected both S27L-HA and S27-HA (third panel). Thus, two antibodies are specific for their corresponding proteins, and do not cross-react with each other.RPS27 is a p53 repressible geneWe previously showed that RPS27L is a p53 target, subjecting to p53 induction (He and Sun 2007), which was later confirmed by the others (Li et al 2007). During our parallel study of p53 regulation of RPS27L and RPS27, we found that in contrast to RPS27L, RPS27 was actually repressed by p53. As shown in Figure 2A, treatment with etoposide, a DNA damaging agent, which induced p53 and its downstream targets p21 and RPS27L, significantly reduced the level of RPS27 in wt p53-containing HCT116 cells (lanes 1 vs. 2), but only a slight reduction of RPS27 was seen in p53-null HCT116 cells (lanes 3&4). The p53-dependent repression of RPS27 was further demonstrated in wt p53-containing A549 cells. As shown in Figure 2B, in A549 cells infected with control siRNA, the basal level of RPS27 was very high, but remarkably reduced upon p53 activation by MI-219, a small molecule that disrupts the MDM2-p53 binding (Shangary 2008), or etoposide (lanes 1–3). The p53-induced RPS27 repression was abrogated if p53 was silenced via siRNA (lanes 4–6). We further confirmed that p53-induced RPS27 repression occurred at the transcriptional level, as demonstrated by RT-qPCR analysis (Fig 2C). Finally, we used the H1299-p53 temperature sensitive model (Peng et al 2003, Robinson 2003) and showed that while MDM2 and RPS27L were induced when cells were grown at the permissive 32°C (wt p53 conformation), the RPS27 expression was repressed. No difference was seen among these proteins at 37°C with a mutant p53 confirmation (Fig 2D, top). Neither RPS27 reduction, nor RPS27L induction was due to cold shock, since the same treatment in p53-null H1299-neo control cells even slightly increased the RPS27 levels, but had no effect on RPS27L (Fig 2D, bottom). Our results demonstrate that RPS27 is a p53 repressible gene.RPS27L or RPS27 binds to MDM2 in vitro: mapping of the binding domainsA few ribosomal proteins (L11, L23, L5, L26 or S7) were previously found to bind to MDM2 and activate p53 (Bhat et al 2004, Dai and Lu 2004, Dai et al 2004, Jin et al 2004, Lohrum et al 2003, Ofir-Rosenfeld et al 2008, Zhang et al 2003, Zhu et al 2009). We, therefore, took this candidate approach and determined if RPS27L or RPS27 would directly bind to MDM2, and if so, defined the binding motif on respective protein. GST-fusion proteins were expressed and purified for full-length RPS27L 1–84, the N-terminal portion, 1–36 (where RPS27L differs from RPS27 at three amino acids, codons, 5, 12, 17), the C-terminal portion 37–84 (exactly same as RPS27), the full-length RPS27 1–84, and the N-terminal portion of RPS27 1–36, along with GST control (Fig 3B). Approximately equal amount of proteins were used for the pull-down assay with His-tagged full length MDM2, along with the empty His-tag vector control. As shown in Figure 3A, the full-length RPS27L or RPS27 bound to MDM2, as well as the N-terminal portion, but not the C-terminal zinc-finger containing portion, although the binding was much weaker for RPS27. Thus, both family members bind to MDM2 and the binding domain is mapped to the N-terminus (AA 1–36).To further confirm the RPS27L-MDM2 binding at the cellular level, and to define the potential RPS27L binding region on MDM2, we performed a GST-pull-down experiment, using full-length MDM2, and its deletion mutants. GST-fused full length MDM2 (1–491) and a deletion mutant (1–301), but not the mutant 1–150, nor other deletion mutants, bind to endogenous RPS27L in MEF cells derived from p53/Mdm2 double null mice (to avoid potential interference from p53) (Fig 3C). Expression and purity of GST-fused MDM2 and its deletion mutants are shown in Figure 3D. Finally, we confirmed the binding of MDM2-RPS27L and MDM2-RPS27 using GST-MDM2 to pull-down RPS27L or RPS27 produced by in vitro transcription and translation (data not shown). These results clearly demonstrated that MDM2 binds to endogenous RPS27L in the absence of p53 and that the binding region on MDM2 is from codons 151–293 (Fig 3C, bottom panel), an acidic domain-containing region, previously shown as the second p53 binding domain (Kulikov et al 2006), implying a potential competition between RPS27L and p53 for MDM2 binding.The in vivo binding of endogenous RPS27L-MDM2 and RPS27-MDM2 and competition between RPS27L or RPS27 and p53 for MDM2 bindingWe further determined if endogenous RPS27L or RPS27 binds to endogenous MDM2 under physiological condition. Normally growing p53-null H1299 cells at the subconfluency were harvested and subjected to immunoprecipitation (IP) using antibodies against RPS27L, RPS27, or IgG control. As shown in Figure 4A, MDM2 can be detected in the immunoprecipitates pulled down by either RPS27L or RPS27 antibody, respectively, but not by normal IgG control. In a reciprocal experiment, RPS27L or RPS27 can be pulled down by antibody against MDM2, but not by normal IgG control (Fig 4B). Thus, RPS27L or RPS27 forms a complex in vivo independent of p53.It is well-established that the major Mdm2-p53 binding occurs between their N-terminal domains (Kussie et al 1996). A second binding site at the center portion of each protein (the DNA binding domain of p53 and the acidic domain of Mdm2) was also reported (Burch et al 2000, Kulikov et al 2006, Shimizu et al 2002). Since RPS27L or RPS27 binds to Mdm2 on the acidic region, where it also binds to p53, we determined if RPS27L or RPS27 competes with p53 for Mdm2 binding, thus interfering the MDM2-p53 binding. We used a FLAG-tagged ligase-dead MDM2 mutant, MDM2-446/475 (Swaroop and Sun 2003), avoiding potential degradation of p53 or RPS27L (see below). After cotransfection with RPS27L or RPS27, FLAG-MDM2 was immunoprecipitated, along with its binding partners. As shown in Figure 4C, the p53 level in MDM2 complex was significantly reduced, if either RPS27L or RPS27 was cotransfected. Likewise, in a reciprocal experiment, we found that siRNA silencing of endogenous p53 significantly enhanced the binding of RPS27L or RPS27 to MDM2-446/475 (Fig. 4D). Taken together, these results suggested a competition of RPS27L or RPS27 with p53 for MDM2 binding.MDM2 regulation of RPS27L and RPS27 under overexpressed or physiological conditionsTwo MDM2 binding ribosomal proteins L26 (Ofir-Rosenfeld et al 2008) and S7 (Zhu et al 2009) was found to be the substrates of MDM2. We determined if RPS27L or RPS27 is subjected to MDM2-mediated degradation. As shown in Figure 5A, when co-transfected into p53-null H1299 cells, ectopic expression of MDM2 dramatically reduced the steady-state level of exogenous expressed RPS27L (HA-S27L, lanes 2 vs. 1). Reduction of RPS27L requires MDM2 acidic domain where two proteins bind and the RING E3 ligase domain, since the deletion of either domain abolishes MDM2 activity (lanes 3&4). Similar results were found when MDM2 was cotransfected with RPS27 (Fig 5B). We further determined the potential effect of MDM2 on the protein half-life of RPS27L or RPS27. Transfected RPS27L has a half-life of ~60 min which is shortened to ~30 min by MDM2-cotransfection. In contrast, RPS27 has a much longer protein half-life up to 180 min and was only moderately affected, if any, by MDM2 within the 3-hrs testing periods (Fig 5C). Finally, we showed, using an immunoprecipitation assay, that MDM2 remarkably promotes polyubiquitination of RPS27L, but with a minor effect on RPS27 (Fig 5D). The results strongly suggested that although both family members bind to MDM2 and are subjected to MDM2 degradation under overexpressed conditions, RPS27L with a much shorter protein half-life is more likely regulated by MDM2 under physiological condition.We tested this hypothesis by determining if endogenous RPS27L or RPS27 is subject to regulation by endogenous MDM2. We compared the levels of the two family members in p53-null and p53/Mdm2-double null MEFs. As shown in Figure 5E, the levels of RPS27L, but not of RPS27, were higher in p53/Mdm2-double null MEFs than that in p53-null MEFs, suggesting that only RPS27L is subjected to Mdm2 regulation. We followed up this finding by measuring the protein half-life of two proteins in these two types of MEFs. As shown in Figure 5F, the protein half-life of endogenous RPS27L is about 8 hrs in p53-null cells, but was significantly extended up to 24 hrs in p53/Mdm2 double null cells. In contrast, the half-life of RPS27 is much longer, up to 24 hrs, regardless of the presence of Mdm2. Taken together, these results clearly suggested that RPS27L is a physiological substrate of MDM2, whereas RPS27, with a much longer protein half-life, can be ubiquitinated and degraded by MDM2 under overexpressed conditions.Overexpression of RPS27L or RPS27 stabilizes p53 by inhibiting MDM2-induced p53 ubiquitinationSince RPS27L/S27 competes with p53 for MDM2 binding (Fig 4), we reasoned that their overexpression would free up p53 and reduces its ubiquitination by MDM2. Indeed, in a typical in vivo ubiquitination assay, using p53-null H1299 cells, we found that p53 ubiquitination was remarkably promoted by MDM2 (Figure 6A, lanes 4 vs. 3), but significantly reduced by co-transfection with RPS27L (lane 5) or RPS27 (lane 6). Likewise, overexpression of either RPS27L or RPS27 caused the accumulation of endogenous wild type p53 in A549 cells (Fig 6B). Furthermore, we found that p53, when transfected alone into p53-null MEF cells, had a half-life of less than 2 hrs. However, when p53 was cotransfected with RPS27L, the steady-state level of p53 was much higher, and p53 half-life was extended to ~ 6 hrs (Fig 6C). Importantly, RPS27L-induced p53 increase and half-life extension was indeed mediated by MDM2, since a similar steady-state p53 level and p53 half-life was observed when tested in p53-Mdm2 double null MEF cells, regardless of RPS27L co-transfection (Fig 6D). Taken together, these results strongly suggest that via binding to MDM2, thus reducing MDM2-p53 binding, RPS27L decreases MDM2-induced p53 ubiquitination, leading to increased p53 levels by extending p53 protein half-life.SiRNA silencing of RPS27L reduces p53 levels, shortens p53 protein half-life, and inhibits p53 transcription activityWe further determined if siRNA silencing of RPS27L or RPS27 would change the levels of endogenous p53. As shown in Figure 7A, lentivirus-based siRNA silencing caused a complete elimination of endogenous RPS27L and a nearly complete elimination RPS27 in wt p53-containing A549 cells, respectively. Accompanying the silencing of RPS27L, but not RPS27, the basal level of p53 was significantly decreased (Fig 7A), which appeared to occur at the protein levels, since no change at the p53 mRNA level was observed (Fig 7B). We, therefore, focused our study on RPS27L silencing and found that p53 reduction, upon RPS27L silencing, was MDM2 dependent, since simultaneous silencing RPS27L and MDM2 restored the p53 level (Fig 7C, lanes 3 vs. 4). Furthermore, in wild-type p53-containing A549 cells, siRNA silencing of RPS27L shortened the p53 half-life from ~1 hr to 30 min (Fig 7D). Finally, using a p53-temperature sensitive H1299-p53ts line (Peng et al 2003, Robinson 2003), we found that p53 induction of p21, at the permissive temperature of 32°C where p53 is active, was significantly reduced upon RPS27L silencing (Fig 7E). Likewise, p53 transcription activity, as reflected by luciferase activity in a p53-driven luciferase reporter assay, was also significantly decreased upon RPS27L silencing (Fig 7F). Taken together, RPS27L, a p53 inducible protein and an MDM2 substrate, can positively regulate p53 activity through MDM2 binding, establishing a three-way interplay among p53, MDM2, and RPS27L.Co-localization of endogenous RPS27L with MDM2 upon DNA damage or ribosomal stressSeveral MDM2 binding ribosomal proteins were found to be localized in nucleolus, such as L11 (Bhat et al 2004, Lohrum et al 2003) and S7 (Zhu et al 2009) or in nucleoplasm, such as L23 (Dai et al 2004, Jin et al 2004). We determined the subcellular localization of RPS27L and its co-localization with MDM2 under unstressed or stressed conditions by immune-fluorescent staining. As shown in Figure 8A, under unstressed condition, endogenous RPS27L as well as RPS27 (not shown) are mainly localized in the cytoplasm in A549 cells or in SJSA cells (data not shown), whereas Mdm2 was mainly localized in nucleoplasm with a very low, but detectable level in cytoplasm. Upon exposure of cells to different types of stress inducers, including etoposide, MI-219 (Shangary et al 2008), or low dose of actionmycin D that triggers ribosomal stress, a portion of RPS27L was translocated into nucleoplasm, where it co-localized with Mdm2. These stress inducers also increased the cytoplasmic Mdm2 level. Western blotting analysis confirmed the induction by these stress inducers of p53 and its downstream targets, Mdm2, p21 and to a less extent, RPS27L (except actinomycin D which failed to induce RPS27L) (Fig 8B). Cell fractionation experiment showed that upon exposure to p53-activating signals, both cytoplasmic and nuclear p53, Mdm2 and RPS27L (except by actinomycin D) were induced, whereas p21 induction was only detectable in the cytoplasm (Fig 8C). These results indicate that RSP27L is mainly a cytoplasmic protein and can be translocated into nucleoplasm in response to p53 inducing signals where it co-localized with Mdm2, and likely competed with p53 for Mdm2 binding.SiRNA silencing of RPS27L reduces the levels of both cytoplasmic and nuclear p53Finally, we determined if RPS27L silencing would reduce p53 levels induced by these p53-activating signals. Indeed, RPS27L silencing remarkably inhibited the basal and induced levels of p53 and p21 upon exposure to actinomycin D (Fig 9A) or radiation (Fig 9B). Induction of p53 and its downstream targets, p21 and MDM2 by etoposide was also remarkably inhibited (Fig 9C). Blockage of induction of p53 and its downstream targets was not due to an overall inhibition of protein expression upon RPS27L silencing, since the expression of many other short-lived proteins, including p27, c-JUN, WEE1, IκBα, and β-catenin was not changed under the same experimental conditions (data not shown). We also observed in a cell fractionation study that RPS27L silencing reduced the levels of p53 and MDM2 in both cytoplasm and nucleus upon actinomycin D treatment (Fig 9D). Thus, the presence of RPS27L ensures the full activation of p53, consistent with our previous observation that RPS27L, upon induced by p53, amplifies p53 signals (He and Sun 2007). This is likely achieved by competition of RPS27L with p53 for Mdm2 binding, thus counteracting the Mdm2-p53 negative feedback loop.

RPS27 and RPS27L are evolutionarily conserved proteinsRPS27 and its family member, RPS27L are 84 amino acids-containing proteins with a zinc-finger-like domain. Two family members differ from each other by only three amino acids (R5K, L12P, K17R) at the N-terminus (Fig 1A). We compared the protein sequences of RPS27 and RPS27L in more than 10 species. As shown in Figure 1B, both family members are highly conserved during evolution, suggesting their functional significance. In addition to human, the R5K/L12P/K17R conservation of two family members is also found in mouse, rat, and dog. Like in humans, there are two members of RPS27/S27L family in yeast S. cerevisiae, RPS27A (YKL156W) and RPS27B (YHR021C), differing by only one amino acid at codon 62 (I62V). The yeast with systematic deletion of either RPS27A or RPS27B is still viable, although deletion of RPS27B showed some growth defect (Giaever et al 2002) (also see www.yeastgenome.org). In C-elegans, however, there is only one family member (RPS27). SiRNA silencing lead to embryonic lethality (Sonnichsen et al 2005) (also see www.wormgenome.org). In Drosophila, one family member is identified. Its molecular function is described as structural constituent of ribosome, nucleic acid binding (see http://flybase.bio.indiana.edu/reports/FBgn0039300.html) and it is involved in the biological process of translation.To facilitate the detection of endogenous RPS27L and RPS27 selectively, we generated, affinity purified, and characterized two antibodies specifically against either one of the family members. As shown in Figure 1C, the RPS27L antibody detected both endogenous and HA- tagged RPS27L (S27L-HA), but not HA-tagged RPS27 (S27-HA) (top panel), whereas the RPS27 antibody detected both endogenous and S27-HA, but not S27L-HA (second panel). As expected, anti-HA tag antibody detected both S27L-HA and S27-HA (third panel). Thus, two antibodies are specific for their corresponding proteins, and do not cross-react with each other.

RPS27 and RPS27L are evolutionarily conserved proteins

RPS27 and its family member, RPS27L are 84 amino acids-containing proteins with a zinc-finger-like domain. Two family members differ from each other by only three amino acids (R5K, L12P, K17R) at the N-terminus (Fig 1A). We compared the protein sequences of RPS27 and RPS27L in more than 10 species. As shown in Figure 1B, both family members are highly conserved during evolution, suggesting their functional significance. In addition to human, the R5K/L12P/K17R conservation of two family members is also found in mouse, rat, and dog. Like in humans, there are two members of RPS27/S27L family in yeast S. cerevisiae, RPS27A (YKL156W) and RPS27B (YHR021C), differing by only one amino acid at codon 62 (I62V). The yeast with systematic deletion of either RPS27A or RPS27B is still viable, although deletion of RPS27B showed some growth defect (Giaever et al 2002) (also see www.yeastgenome.org). In C-elegans, however, there is only one family member (RPS27). SiRNA silencing lead to embryonic lethality (Sonnichsen et al 2005) (also see www.wormgenome.org). In Drosophila, one family member is identified. Its molecular function is described as structural constituent of ribosome, nucleic acid binding (see http://flybase.bio.indiana.edu/reports/FBgn0039300.html) and it is involved in the biological process of translation.

To facilitate the detection of endogenous RPS27L and RPS27 selectively, we generated, affinity purified, and characterized two antibodies specifically against either one of the family members. As shown in Figure 1C, the RPS27L antibody detected both endogenous and HA- tagged RPS27L (S27L-HA), but not HA-tagged RPS27 (S27-HA) (top panel), whereas the RPS27 antibody detected both endogenous and S27-HA, but not S27L-HA (second panel). As expected, anti-HA tag antibody detected both S27L-HA and S27-HA (third panel). Thus, two antibodies are specific for their corresponding proteins, and do not cross-react with each other.

RPS27 is a p53 repressible geneWe previously showed that RPS27L is a p53 target, subjecting to p53 induction (He and Sun 2007), which was later confirmed by the others (Li et al 2007). During our parallel study of p53 regulation of RPS27L and RPS27, we found that in contrast to RPS27L, RPS27 was actually repressed by p53. As shown in Figure 2A, treatment with etoposide, a DNA damaging agent, which induced p53 and its downstream targets p21 and RPS27L, significantly reduced the level of RPS27 in wt p53-containing HCT116 cells (lanes 1 vs. 2), but only a slight reduction of RPS27 was seen in p53-null HCT116 cells (lanes 3&4). The p53-dependent repression of RPS27 was further demonstrated in wt p53-containing A549 cells. As shown in Figure 2B, in A549 cells infected with control siRNA, the basal level of RPS27 was very high, but remarkably reduced upon p53 activation by MI-219, a small molecule that disrupts the MDM2-p53 binding (Shangary 2008), or etoposide (lanes 1–3). The p53-induced RPS27 repression was abrogated if p53 was silenced via siRNA (lanes 4–6). We further confirmed that p53-induced RPS27 repression occurred at the transcriptional level, as demonstrated by RT-qPCR analysis (Fig 2C). Finally, we used the H1299-p53 temperature sensitive model (Peng et al 2003, Robinson 2003) and showed that while MDM2 and RPS27L were induced when cells were grown at the permissive 32°C (wt p53 conformation), the RPS27 expression was repressed. No difference was seen among these proteins at 37°C with a mutant p53 confirmation (Fig 2D, top). Neither RPS27 reduction, nor RPS27L induction was due to cold shock, since the same treatment in p53-null H1299-neo control cells even slightly increased the RPS27 levels, but had no effect on RPS27L (Fig 2D, bottom). Our results demonstrate that RPS27 is a p53 repressible gene.

RPS27 is a p53 repressible gene

We previously showed that RPS27L is a p53 target, subjecting to p53 induction (He and Sun 2007), which was later confirmed by the others (Li et al 2007). During our parallel study of p53 regulation of RPS27L and RPS27, we found that in contrast to RPS27L, RPS27 was actually repressed by p53. As shown in Figure 2A, treatment with etoposide, a DNA damaging agent, which induced p53 and its downstream targets p21 and RPS27L, significantly reduced the level of RPS27 in wt p53-containing HCT116 cells (lanes 1 vs. 2), but only a slight reduction of RPS27 was seen in p53-null HCT116 cells (lanes 3&4). The p53-dependent repression of RPS27 was further demonstrated in wt p53-containing A549 cells. As shown in Figure 2B, in A549 cells infected with control siRNA, the basal level of RPS27 was very high, but remarkably reduced upon p53 activation by MI-219, a small molecule that disrupts the MDM2-p53 binding (Shangary 2008), or etoposide (lanes 1–3). The p53-induced RPS27 repression was abrogated if p53 was silenced via siRNA (lanes 4–6). We further confirmed that p53-induced RPS27 repression occurred at the transcriptional level, as demonstrated by RT-qPCR analysis (Fig 2C). Finally, we used the H1299-p53 temperature sensitive model (Peng et al 2003, Robinson 2003) and showed that while MDM2 and RPS27L were induced when cells were grown at the permissive 32°C (wt p53 conformation), the RPS27 expression was repressed. No difference was seen among these proteins at 37°C with a mutant p53 confirmation (Fig 2D, top). Neither RPS27 reduction, nor RPS27L induction was due to cold shock, since the same treatment in p53-null H1299-neo control cells even slightly increased the RPS27 levels, but had no effect on RPS27L (Fig 2D, bottom). Our results demonstrate that RPS27 is a p53 repressible gene.

RPS27L or RPS27 binds to MDM2 in vitro: mapping of the binding domainsA few ribosomal proteins (L11, L23, L5, L26 or S7) were previously found to bind to MDM2 and activate p53 (Bhat et al 2004, Dai and Lu 2004, Dai et al 2004, Jin et al 2004, Lohrum et al 2003, Ofir-Rosenfeld et al 2008, Zhang et al 2003, Zhu et al 2009). We, therefore, took this candidate approach and determined if RPS27L or RPS27 would directly bind to MDM2, and if so, defined the binding motif on respective protein. GST-fusion proteins were expressed and purified for full-length RPS27L 1–84, the N-terminal portion, 1–36 (where RPS27L differs from RPS27 at three amino acids, codons, 5, 12, 17), the C-terminal portion 37–84 (exactly same as RPS27), the full-length RPS27 1–84, and the N-terminal portion of RPS27 1–36, along with GST control (Fig 3B). Approximately equal amount of proteins were used for the pull-down assay with His-tagged full length MDM2, along with the empty His-tag vector control. As shown in Figure 3A, the full-length RPS27L or RPS27 bound to MDM2, as well as the N-terminal portion, but not the C-terminal zinc-finger containing portion, although the binding was much weaker for RPS27. Thus, both family members bind to MDM2 and the binding domain is mapped to the N-terminus (AA 1–36).To further confirm the RPS27L-MDM2 binding at the cellular level, and to define the potential RPS27L binding region on MDM2, we performed a GST-pull-down experiment, using full-length MDM2, and its deletion mutants. GST-fused full length MDM2 (1–491) and a deletion mutant (1–301), but not the mutant 1–150, nor other deletion mutants, bind to endogenous RPS27L in MEF cells derived from p53/Mdm2 double null mice (to avoid potential interference from p53) (Fig 3C). Expression and purity of GST-fused MDM2 and its deletion mutants are shown in Figure 3D. Finally, we confirmed the binding of MDM2-RPS27L and MDM2-RPS27 using GST-MDM2 to pull-down RPS27L or RPS27 produced by in vitro transcription and translation (data not shown). These results clearly demonstrated that MDM2 binds to endogenous RPS27L in the absence of p53 and that the binding region on MDM2 is from codons 151–293 (Fig 3C, bottom panel), an acidic domain-containing region, previously shown as the second p53 binding domain (Kulikov et al 2006), implying a potential competition between RPS27L and p53 for MDM2 binding.

RPS27L or RPS27 binds to MDM2 in vitro: mapping of the binding domains

A few ribosomal proteins (L11, L23, L5, L26 or S7) were previously found to bind to MDM2 and activate p53 (Bhat et al 2004, Dai and Lu 2004, Dai et al 2004, Jin et al 2004, Lohrum et al 2003, Ofir-Rosenfeld et al 2008, Zhang et al 2003, Zhu et al 2009). We, therefore, took this candidate approach and determined if RPS27L or RPS27 would directly bind to MDM2, and if so, defined the binding motif on respective protein. GST-fusion proteins were expressed and purified for full-length RPS27L 1–84, the N-terminal portion, 1–36 (where RPS27L differs from RPS27 at three amino acids, codons, 5, 12, 17), the C-terminal portion 37–84 (exactly same as RPS27), the full-length RPS27 1–84, and the N-terminal portion of RPS27 1–36, along with GST control (Fig 3B). Approximately equal amount of proteins were used for the pull-down assay with His-tagged full length MDM2, along with the empty His-tag vector control. As shown in Figure 3A, the full-length RPS27L or RPS27 bound to MDM2, as well as the N-terminal portion, but not the C-terminal zinc-finger containing portion, although the binding was much weaker for RPS27. Thus, both family members bind to MDM2 and the binding domain is mapped to the N-terminus (AA 1–36).

To further confirm the RPS27L-MDM2 binding at the cellular level, and to define the potential RPS27L binding region on MDM2, we performed a GST-pull-down experiment, using full-length MDM2, and its deletion mutants. GST-fused full length MDM2 (1–491) and a deletion mutant (1–301), but not the mutant 1–150, nor other deletion mutants, bind to endogenous RPS27L in MEF cells derived from p53/Mdm2 double null mice (to avoid potential interference from p53) (Fig 3C). Expression and purity of GST-fused MDM2 and its deletion mutants are shown in Figure 3D. Finally, we confirmed the binding of MDM2-RPS27L and MDM2-RPS27 using GST-MDM2 to pull-down RPS27L or RPS27 produced by in vitro transcription and translation (data not shown). These results clearly demonstrated that MDM2 binds to endogenous RPS27L in the absence of p53 and that the binding region on MDM2 is from codons 151–293 (Fig 3C, bottom panel), an acidic domain-containing region, previously shown as the second p53 binding domain (Kulikov et al 2006), implying a potential competition between RPS27L and p53 for MDM2 binding.

The in vivo binding of endogenous RPS27L-MDM2 and RPS27-MDM2 and competition between RPS27L or RPS27 and p53 for MDM2 bindingWe further determined if endogenous RPS27L or RPS27 binds to endogenous MDM2 under physiological condition. Normally growing p53-null H1299 cells at the subconfluency were harvested and subjected to immunoprecipitation (IP) using antibodies against RPS27L, RPS27, or IgG control. As shown in Figure 4A, MDM2 can be detected in the immunoprecipitates pulled down by either RPS27L or RPS27 antibody, respectively, but not by normal IgG control. In a reciprocal experiment, RPS27L or RPS27 can be pulled down by antibody against MDM2, but not by normal IgG control (Fig 4B). Thus, RPS27L or RPS27 forms a complex in vivo independent of p53.It is well-established that the major Mdm2-p53 binding occurs between their N-terminal domains (Kussie et al 1996). A second binding site at the center portion of each protein (the DNA binding domain of p53 and the acidic domain of Mdm2) was also reported (Burch et al 2000, Kulikov et al 2006, Shimizu et al 2002). Since RPS27L or RPS27 binds to Mdm2 on the acidic region, where it also binds to p53, we determined if RPS27L or RPS27 competes with p53 for Mdm2 binding, thus interfering the MDM2-p53 binding. We used a FLAG-tagged ligase-dead MDM2 mutant, MDM2-446/475 (Swaroop and Sun 2003), avoiding potential degradation of p53 or RPS27L (see below). After cotransfection with RPS27L or RPS27, FLAG-MDM2 was immunoprecipitated, along with its binding partners. As shown in Figure 4C, the p53 level in MDM2 complex was significantly reduced, if either RPS27L or RPS27 was cotransfected. Likewise, in a reciprocal experiment, we found that siRNA silencing of endogenous p53 significantly enhanced the binding of RPS27L or RPS27 to MDM2-446/475 (Fig. 4D). Taken together, these results suggested a competition of RPS27L or RPS27 with p53 for MDM2 binding.

The in vivo binding of endogenous RPS27L-MDM2 and RPS27-MDM2 and competition between RPS27L or RPS27 and p53 for MDM2 binding

We further determined if endogenous RPS27L or RPS27 binds to endogenous MDM2 under physiological condition. Normally growing p53-null H1299 cells at the subconfluency were harvested and subjected to immunoprecipitation (IP) using antibodies against RPS27L, RPS27, or IgG control. As shown in Figure 4A, MDM2 can be detected in the immunoprecipitates pulled down by either RPS27L or RPS27 antibody, respectively, but not by normal IgG control. In a reciprocal experiment, RPS27L or RPS27 can be pulled down by antibody against MDM2, but not by normal IgG control (Fig 4B). Thus, RPS27L or RPS27 forms a complex in vivo independent of p53.

It is well-established that the major Mdm2-p53 binding occurs between their N-terminal domains (Kussie et al 1996). A second binding site at the center portion of each protein (the DNA binding domain of p53 and the acidic domain of Mdm2) was also reported (Burch et al 2000, Kulikov et al 2006, Shimizu et al 2002). Since RPS27L or RPS27 binds to Mdm2 on the acidic region, where it also binds to p53, we determined if RPS27L or RPS27 competes with p53 for Mdm2 binding, thus interfering the MDM2-p53 binding. We used a FLAG-tagged ligase-dead MDM2 mutant, MDM2-446/475 (Swaroop and Sun 2003), avoiding potential degradation of p53 or RPS27L (see below). After cotransfection with RPS27L or RPS27, FLAG-MDM2 was immunoprecipitated, along with its binding partners. As shown in Figure 4C, the p53 level in MDM2 complex was significantly reduced, if either RPS27L or RPS27 was cotransfected. Likewise, in a reciprocal experiment, we found that siRNA silencing of endogenous p53 significantly enhanced the binding of RPS27L or RPS27 to MDM2-446/475 (Fig. 4D). Taken together, these results suggested a competition of RPS27L or RPS27 with p53 for MDM2 binding.

MDM2 regulation of RPS27L and RPS27 under overexpressed or physiological conditionsTwo MDM2 binding ribosomal proteins L26 (Ofir-Rosenfeld et al 2008) and S7 (Zhu et al 2009) was found to be the substrates of MDM2. We determined if RPS27L or RPS27 is subjected to MDM2-mediated degradation. As shown in Figure 5A, when co-transfected into p53-null H1299 cells, ectopic expression of MDM2 dramatically reduced the steady-state level of exogenous expressed RPS27L (HA-S27L, lanes 2 vs. 1). Reduction of RPS27L requires MDM2 acidic domain where two proteins bind and the RING E3 ligase domain, since the deletion of either domain abolishes MDM2 activity (lanes 3&4). Similar results were found when MDM2 was cotransfected with RPS27 (Fig 5B). We further determined the potential effect of MDM2 on the protein half-life of RPS27L or RPS27. Transfected RPS27L has a half-life of ~60 min which is shortened to ~30 min by MDM2-cotransfection. In contrast, RPS27 has a much longer protein half-life up to 180 min and was only moderately affected, if any, by MDM2 within the 3-hrs testing periods (Fig 5C). Finally, we showed, using an immunoprecipitation assay, that MDM2 remarkably promotes polyubiquitination of RPS27L, but with a minor effect on RPS27 (Fig 5D). The results strongly suggested that although both family members bind to MDM2 and are subjected to MDM2 degradation under overexpressed conditions, RPS27L with a much shorter protein half-life is more likely regulated by MDM2 under physiological condition.We tested this hypothesis by determining if endogenous RPS27L or RPS27 is subject to regulation by endogenous MDM2. We compared the levels of the two family members in p53-null and p53/Mdm2-double null MEFs. As shown in Figure 5E, the levels of RPS27L, but not of RPS27, were higher in p53/Mdm2-double null MEFs than that in p53-null MEFs, suggesting that only RPS27L is subjected to Mdm2 regulation. We followed up this finding by measuring the protein half-life of two proteins in these two types of MEFs. As shown in Figure 5F, the protein half-life of endogenous RPS27L is about 8 hrs in p53-null cells, but was significantly extended up to 24 hrs in p53/Mdm2 double null cells. In contrast, the half-life of RPS27 is much longer, up to 24 hrs, regardless of the presence of Mdm2. Taken together, these results clearly suggested that RPS27L is a physiological substrate of MDM2, whereas RPS27, with a much longer protein half-life, can be ubiquitinated and degraded by MDM2 under overexpressed conditions.

MDM2 regulation of RPS27L and RPS27 under overexpressed or physiological conditions

Two MDM2 binding ribosomal proteins L26 (Ofir-Rosenfeld et al 2008) and S7 (Zhu et al 2009) was found to be the substrates of MDM2. We determined if RPS27L or RPS27 is subjected to MDM2-mediated degradation. As shown in Figure 5A, when co-transfected into p53-null H1299 cells, ectopic expression of MDM2 dramatically reduced the steady-state level of exogenous expressed RPS27L (HA-S27L, lanes 2 vs. 1). Reduction of RPS27L requires MDM2 acidic domain where two proteins bind and the RING E3 ligase domain, since the deletion of either domain abolishes MDM2 activity (lanes 3&4). Similar results were found when MDM2 was cotransfected with RPS27 (Fig 5B). We further determined the potential effect of MDM2 on the protein half-life of RPS27L or RPS27. Transfected RPS27L has a half-life of ~60 min which is shortened to ~30 min by MDM2-cotransfection. In contrast, RPS27 has a much longer protein half-life up to 180 min and was only moderately affected, if any, by MDM2 within the 3-hrs testing periods (Fig 5C). Finally, we showed, using an immunoprecipitation assay, that MDM2 remarkably promotes polyubiquitination of RPS27L, but with a minor effect on RPS27 (Fig 5D). The results strongly suggested that although both family members bind to MDM2 and are subjected to MDM2 degradation under overexpressed conditions, RPS27L with a much shorter protein half-life is more likely regulated by MDM2 under physiological condition.

We tested this hypothesis by determining if endogenous RPS27L or RPS27 is subject to regulation by endogenous MDM2. We compared the levels of the two family members in p53-null and p53/Mdm2-double null MEFs. As shown in Figure 5E, the levels of RPS27L, but not of RPS27, were higher in p53/Mdm2-double null MEFs than that in p53-null MEFs, suggesting that only RPS27L is subjected to Mdm2 regulation. We followed up this finding by measuring the protein half-life of two proteins in these two types of MEFs. As shown in Figure 5F, the protein half-life of endogenous RPS27L is about 8 hrs in p53-null cells, but was significantly extended up to 24 hrs in p53/Mdm2 double null cells. In contrast, the half-life of RPS27 is much longer, up to 24 hrs, regardless of the presence of Mdm2. Taken together, these results clearly suggested that RPS27L is a physiological substrate of MDM2, whereas RPS27, with a much longer protein half-life, can be ubiquitinated and degraded by MDM2 under overexpressed conditions.

Overexpression of RPS27L or RPS27 stabilizes p53 by inhibiting MDM2-induced p53 ubiquitinationSince RPS27L/S27 competes with p53 for MDM2 binding (Fig 4), we reasoned that their overexpression would free up p53 and reduces its ubiquitination by MDM2. Indeed, in a typical in vivo ubiquitination assay, using p53-null H1299 cells, we found that p53 ubiquitination was remarkably promoted by MDM2 (Figure 6A, lanes 4 vs. 3), but significantly reduced by co-transfection with RPS27L (lane 5) or RPS27 (lane 6). Likewise, overexpression of either RPS27L or RPS27 caused the accumulation of endogenous wild type p53 in A549 cells (Fig 6B). Furthermore, we found that p53, when transfected alone into p53-null MEF cells, had a half-life of less than 2 hrs. However, when p53 was cotransfected with RPS27L, the steady-state level of p53 was much higher, and p53 half-life was extended to ~ 6 hrs (Fig 6C). Importantly, RPS27L-induced p53 increase and half-life extension was indeed mediated by MDM2, since a similar steady-state p53 level and p53 half-life was observed when tested in p53-Mdm2 double null MEF cells, regardless of RPS27L co-transfection (Fig 6D). Taken together, these results strongly suggest that via binding to MDM2, thus reducing MDM2-p53 binding, RPS27L decreases MDM2-induced p53 ubiquitination, leading to increased p53 levels by extending p53 protein half-life.

Overexpression of RPS27L or RPS27 stabilizes p53 by inhibiting MDM2-induced p53 ubiquitination

Since RPS27L/S27 competes with p53 for MDM2 binding (Fig 4), we reasoned that their overexpression would free up p53 and reduces its ubiquitination by MDM2. Indeed, in a typical in vivo ubiquitination assay, using p53-null H1299 cells, we found that p53 ubiquitination was remarkably promoted by MDM2 (Figure 6A, lanes 4 vs. 3), but significantly reduced by co-transfection with RPS27L (lane 5) or RPS27 (lane 6). Likewise, overexpression of either RPS27L or RPS27 caused the accumulation of endogenous wild type p53 in A549 cells (Fig 6B). Furthermore, we found that p53, when transfected alone into p53-null MEF cells, had a half-life of less than 2 hrs. However, when p53 was cotransfected with RPS27L, the steady-state level of p53 was much higher, and p53 half-life was extended to ~ 6 hrs (Fig 6C). Importantly, RPS27L-induced p53 increase and half-life extension was indeed mediated by MDM2, since a similar steady-state p53 level and p53 half-life was observed when tested in p53-Mdm2 double null MEF cells, regardless of RPS27L co-transfection (Fig 6D). Taken together, these results strongly suggest that via binding to MDM2, thus reducing MDM2-p53 binding, RPS27L decreases MDM2-induced p53 ubiquitination, leading to increased p53 levels by extending p53 protein half-life.

SiRNA silencing of RPS27L reduces p53 levels, shortens p53 protein half-life, and inhibits p53 transcription activityWe further determined if siRNA silencing of RPS27L or RPS27 would change the levels of endogenous p53. As shown in Figure 7A, lentivirus-based siRNA silencing caused a complete elimination of endogenous RPS27L and a nearly complete elimination RPS27 in wt p53-containing A549 cells, respectively. Accompanying the silencing of RPS27L, but not RPS27, the basal level of p53 was significantly decreased (Fig 7A), which appeared to occur at the protein levels, since no change at the p53 mRNA level was observed (Fig 7B). We, therefore, focused our study on RPS27L silencing and found that p53 reduction, upon RPS27L silencing, was MDM2 dependent, since simultaneous silencing RPS27L and MDM2 restored the p53 level (Fig 7C, lanes 3 vs. 4). Furthermore, in wild-type p53-containing A549 cells, siRNA silencing of RPS27L shortened the p53 half-life from ~1 hr to 30 min (Fig 7D). Finally, using a p53-temperature sensitive H1299-p53ts line (Peng et al 2003, Robinson 2003), we found that p53 induction of p21, at the permissive temperature of 32°C where p53 is active, was significantly reduced upon RPS27L silencing (Fig 7E). Likewise, p53 transcription activity, as reflected by luciferase activity in a p53-driven luciferase reporter assay, was also significantly decreased upon RPS27L silencing (Fig 7F). Taken together, RPS27L, a p53 inducible protein and an MDM2 substrate, can positively regulate p53 activity through MDM2 binding, establishing a three-way interplay among p53, MDM2, and RPS27L.

SiRNA silencing of RPS27L reduces p53 levels, shortens p53 protein half-life, and inhibits p53 transcription activity

We further determined if siRNA silencing of RPS27L or RPS27 would change the levels of endogenous p53. As shown in Figure 7A, lentivirus-based siRNA silencing caused a complete elimination of endogenous RPS27L and a nearly complete elimination RPS27 in wt p53-containing A549 cells, respectively. Accompanying the silencing of RPS27L, but not RPS27, the basal level of p53 was significantly decreased (Fig 7A), which appeared to occur at the protein levels, since no change at the p53 mRNA level was observed (Fig 7B). We, therefore, focused our study on RPS27L silencing and found that p53 reduction, upon RPS27L silencing, was MDM2 dependent, since simultaneous silencing RPS27L and MDM2 restored the p53 level (Fig 7C, lanes 3 vs. 4). Furthermore, in wild-type p53-containing A549 cells, siRNA silencing of RPS27L shortened the p53 half-life from ~1 hr to 30 min (Fig 7D). Finally, using a p53-temperature sensitive H1299-p53ts line (Peng et al 2003, Robinson 2003), we found that p53 induction of p21, at the permissive temperature of 32°C where p53 is active, was significantly reduced upon RPS27L silencing (Fig 7E). Likewise, p53 transcription activity, as reflected by luciferase activity in a p53-driven luciferase reporter assay, was also significantly decreased upon RPS27L silencing (Fig 7F). Taken together, RPS27L, a p53 inducible protein and an MDM2 substrate, can positively regulate p53 activity through MDM2 binding, establishing a three-way interplay among p53, MDM2, and RPS27L.

Co-localization of endogenous RPS27L with MDM2 upon DNA damage or ribosomal stressSeveral MDM2 binding ribosomal proteins were found to be localized in nucleolus, such as L11 (Bhat et al 2004, Lohrum et al 2003) and S7 (Zhu et al 2009) or in nucleoplasm, such as L23 (Dai et al 2004, Jin et al 2004). We determined the subcellular localization of RPS27L and its co-localization with MDM2 under unstressed or stressed conditions by immune-fluorescent staining. As shown in Figure 8A, under unstressed condition, endogenous RPS27L as well as RPS27 (not shown) are mainly localized in the cytoplasm in A549 cells or in SJSA cells (data not shown), whereas Mdm2 was mainly localized in nucleoplasm with a very low, but detectable level in cytoplasm. Upon exposure of cells to different types of stress inducers, including etoposide, MI-219 (Shangary et al 2008), or low dose of actionmycin D that triggers ribosomal stress, a portion of RPS27L was translocated into nucleoplasm, where it co-localized with Mdm2. These stress inducers also increased the cytoplasmic Mdm2 level. Western blotting analysis confirmed the induction by these stress inducers of p53 and its downstream targets, Mdm2, p21 and to a less extent, RPS27L (except actinomycin D which failed to induce RPS27L) (Fig 8B). Cell fractionation experiment showed that upon exposure to p53-activating signals, both cytoplasmic and nuclear p53, Mdm2 and RPS27L (except by actinomycin D) were induced, whereas p21 induction was only detectable in the cytoplasm (Fig 8C). These results indicate that RSP27L is mainly a cytoplasmic protein and can be translocated into nucleoplasm in response to p53 inducing signals where it co-localized with Mdm2, and likely competed with p53 for Mdm2 binding.

Co-localization of endogenous RPS27L with MDM2 upon DNA damage or ribosomal stress

Several MDM2 binding ribosomal proteins were found to be localized in nucleolus, such as L11 (Bhat et al 2004, Lohrum et al 2003) and S7 (Zhu et al 2009) or in nucleoplasm, such as L23 (Dai et al 2004, Jin et al 2004). We determined the subcellular localization of RPS27L and its co-localization with MDM2 under unstressed or stressed conditions by immune-fluorescent staining. As shown in Figure 8A, under unstressed condition, endogenous RPS27L as well as RPS27 (not shown) are mainly localized in the cytoplasm in A549 cells or in SJSA cells (data not shown), whereas Mdm2 was mainly localized in nucleoplasm with a very low, but detectable level in cytoplasm. Upon exposure of cells to different types of stress inducers, including etoposide, MI-219 (Shangary et al 2008), or low dose of actionmycin D that triggers ribosomal stress, a portion of RPS27L was translocated into nucleoplasm, where it co-localized with Mdm2. These stress inducers also increased the cytoplasmic Mdm2 level. Western blotting analysis confirmed the induction by these stress inducers of p53 and its downstream targets, Mdm2, p21 and to a less extent, RPS27L (except actinomycin D which failed to induce RPS27L) (Fig 8B). Cell fractionation experiment showed that upon exposure to p53-activating signals, both cytoplasmic and nuclear p53, Mdm2 and RPS27L (except by actinomycin D) were induced, whereas p21 induction was only detectable in the cytoplasm (Fig 8C). These results indicate that RSP27L is mainly a cytoplasmic protein and can be translocated into nucleoplasm in response to p53 inducing signals where it co-localized with Mdm2, and likely competed with p53 for Mdm2 binding.

SiRNA silencing of RPS27L reduces the levels of both cytoplasmic and nuclear p53Finally, we determined if RPS27L silencing would reduce p53 levels induced by these p53-activating signals. Indeed, RPS27L silencing remarkably inhibited the basal and induced levels of p53 and p21 upon exposure to actinomycin D (Fig 9A) or radiation (Fig 9B). Induction of p53 and its downstream targets, p21 and MDM2 by etoposide was also remarkably inhibited (Fig 9C). Blockage of induction of p53 and its downstream targets was not due to an overall inhibition of protein expression upon RPS27L silencing, since the expression of many other short-lived proteins, including p27, c-JUN, WEE1, IκBα, and β-catenin was not changed under the same experimental conditions (data not shown). We also observed in a cell fractionation study that RPS27L silencing reduced the levels of p53 and MDM2 in both cytoplasm and nucleus upon actinomycin D treatment (Fig 9D). Thus, the presence of RPS27L ensures the full activation of p53, consistent with our previous observation that RPS27L, upon induced by p53, amplifies p53 signals (He and Sun 2007). This is likely achieved by competition of RPS27L with p53 for Mdm2 binding, thus counteracting the Mdm2-p53 negative feedback loop.

SiRNA silencing of RPS27L reduces the levels of both cytoplasmic and nuclear p53

Finally, we determined if RPS27L silencing would reduce p53 levels induced by these p53-activating signals. Indeed, RPS27L silencing remarkably inhibited the basal and induced levels of p53 and p21 upon exposure to actinomycin D (Fig 9A) or radiation (Fig 9B). Induction of p53 and its downstream targets, p21 and MDM2 by etoposide was also remarkably inhibited (Fig 9C). Blockage of induction of p53 and its downstream targets was not due to an overall inhibition of protein expression upon RPS27L silencing, since the expression of many other short-lived proteins, including p27, c-JUN, WEE1, IκBα, and β-catenin was not changed under the same experimental conditions (data not shown). We also observed in a cell fractionation study that RPS27L silencing reduced the levels of p53 and MDM2 in both cytoplasm and nucleus upon actinomycin D treatment (Fig 9D). Thus, the presence of RPS27L ensures the full activation of p53, consistent with our previous observation that RPS27L, upon induced by p53, amplifies p53 signals (He and Sun 2007). This is likely achieved by competition of RPS27L with p53 for Mdm2 binding, thus counteracting the Mdm2-p53 negative feedback loop.

DISCUSSIONWe characterized here the functions of an evolutionarily conserved family of ribosomal proteins, RPS27 and RPS27L, which differ in only three amino acids at the N-terminus. Similarities and differences between two family members can be summarized as follows. First, RPS27 appears to be an oncogenic protein with overexpression in multiple human cancer (Atsuta et al 2002, Fernandez-Pol et al 1997, Ganger et al 1997, Ganger et al 2001, Lee et al 2004, Wang et al 2006) or may serve as a tumor biomarker (Fernandez-Pol 1996, Lee et al 2004, Stack et al 2004), whereas RPS27L is much less studied protein with a known function in mediating p53-induced apoptosis (He and Sun 2007). Second, two proteins are subjected to p53 regulation but in an opposite direction. While RPS27L is a direct p53 inducible gene (He and Sun 2007, Li et al 2007), RPS27 is transcriptionally repressed by p53. Third, both family members bind to MDM2 and are subjected to MDM2 degradation in RING and acidic domain dependent manner when co-transfected with MDM2, but only RPS27L appears to be a physiological substrate of MDM2. Fourth, both proteins inhibit MDM2-mediated p53 ubiquitination and increase endogenous p53 levels under overexpressed condition, but only RPS27L, upon siRNA silencing, causes p53 destabilization. Taken together, our results suggest that upon p53 activation, RPS27L is induced which in turn competes with p53 for MDM2 binding, thus reducing MDM2-mediated p53 degradation and extending p53-activating signals. On the other hand, MDM2, upon induction by p53, degrades both p53 and RPS27L to terminate p53 activating signals. Repression of RPS27 by activated p53 makes MDM2 degradation of RPS27 unnecessary under stressed conditions.Our finding that RPS27L or RPS27 binds to MDM2 via its central acidic domain add them into the list of other ribosomal proteins, including L5, L11, L23, and S7 (Dai and Lu 2004, Dai et al 2004, Zhu et al 2009), known to regulate p53 activity via binding to MDM2 at the same acidic domain or adjacent zinc finger region. However, among all MDM2-binding ribosomal proteins that inhibit MDM2-induced p53 polyubiquitination for p53 stabilization, RPS27L and RPS27 have several unique features. First, they are the only known MDM2-binding ribosomal or ribosomal-like proteins directly subjecting to p53 transcriptional regulation, thus are directly involved in feedback regulation of p53/MDM2 axis. Second, unlike other MDM2-binding ribosomal proteins which are localized either in nucleolus (e.g. L11 or S7) (Bhat et al 2004, Lohrum et al 2003, Zhu et al 2009) or in nucleoplasm, (e.g. L23) (Dai et al 2004, Jin et al 2004), RPS27L (or RPS27) is mainly localized in cytoplasm under physiological unstressed condition where very low level of cytoplasmic MDM2 exists. Under stressed conditions that activate p53, a portion of RPS27L is shuttled to nucleoplasm where it co-localized with induced MDM2. Since RPS27L and MDM2 in both cytoplasm and nucleus are induced upon p53 activating signals (Fig 8C), induced RPS27L could compete with p53 for Mdm2 binding, thus releasing/protecting p53 from MDM2-mediated degradation in both subcellular locations, as evidenced by a reduction of p53 levels at both cytoplasm and nucleus upon actinomycin D treatment, if RPS27L is silenced (Fig 9D). Finally, our study showing that RPS27L is a physiological substrate of MDM2 added RPS27L into the list of ribosomal proteins, including L26 (Ofir-Rosenfeld et al 2008) and S7 (Zhu et al 2009), that directly bind to MDM2 and serve as the substrates of MDM2. It is very likely that MDM2 in different sub-cellular compartments is responsible for the degradation of these ribosomal or ribosomal-like proteins under physiological condition or after stress stimuli.RPS27 was found to be overexpressed in many human cancers (Atsuta et al 2002, Fernandez-Pol et al 1997, Ganger et al 1997, Ganger et al 2001, Lee et al 2004, Wang et al 2006). However, the underlying mechanism for its overexpression is unknown. Our study report here that RPS27 is subjected to transcriptional repression by wild type p53, but not mutant p53 (Fig 2) may provide one logical explanation. It is well known that p53 is mutated in ~50% of human cancer (Greenblatt et al 1994). Human cancers with p53 mutation would, therefore, likely have a higher RPS27 expression due to the loss of repression by wild type p53. On the other hand, overexpressed RPS27 may in turn stabilize mutant p53, since ectopic expression of RPS27 could increase p53 levels (Fig. 6B). This feed-forward regulation could confer growth advantage in these cancer cells. Future study will be directed to address this interesting correlation.Our siRNA silencing study revealed that knockdown of RPS27L, but not RPS27, caused destabilization of basal p53 in A549 and SJSA cells (Fig. 7A & Fig. 9), suggesting under normal physiological condition, RPS27L is the one involving in the interplay with MDM2/p53 axis. It is noteworthy, however, that in some human cancer lines with wild type p53 (e.g. HCT116 and U2OS), RPS27L knockdown did not destabilize p53 (data not shown). This is likely due to a higher sensitivity of these lines to unknown stresses triggered by RPS27L knockdown to cause p53 induction, which counteracts direct modulation of MDM2/p53 axis by RPS27L. On the other hand, it is unclear why RPS27 silencing did not destabilize p53 in A549 cells, although it binds to MDM2 and inhibits MDM2-induced p53 degradation under overexpressed condition. One possibility is that RPS27 is a house-keeping ribosomal protein whose knockdown may trigger ribosomal stress which could induce p53. It is conceivable that, compared to RPS27L, house-keeping RPS27 has a long protein half-life and is resistant to MDM2-mediated degradation under physiological or even stressed conditions. This RPS27 unique feature could confer more flexibility for cells to deal with harsh environmental conditions without stopping essential protein synthesis.In summary, our current work, along with our previous finding (He and Sun 2007), revealed a multi-level interplay between RPS27L and RPS27 family members and p53/MDM2 axis. p53 is activated upon DNA damage to transactivate MDM2 and RPS27L, and to transrepress RPS27 as well. On one hand, induced MDM2 binds to p53 and RPS27L and promotes their ubiquitination and degradation. On the other hand, induced RPS27L competes with p53 for MDM2 binding, thus releasing p53 from MDM2-mediated degradation, resulting in p53 stabilization. Through interacting with MDM2-p53 axis, RPS27L/S27 could regulate cell growth and survival (Fig 10).

We characterized here the functions of an evolutionarily conserved family of ribosomal proteins, RPS27 and RPS27L, which differ in only three amino acids at the N-terminus. Similarities and differences between two family members can be summarized as follows. First, RPS27 appears to be an oncogenic protein with overexpression in multiple human cancer (Atsuta et al 2002, Fernandez-Pol et al 1997, Ganger et al 1997, Ganger et al 2001, Lee et al 2004, Wang et al 2006) or may serve as a tumor biomarker (Fernandez-Pol 1996, Lee et al 2004, Stack et al 2004), whereas RPS27L is much less studied protein with a known function in mediating p53-induced apoptosis (He and Sun 2007). Second, two proteins are subjected to p53 regulation but in an opposite direction. While RPS27L is a direct p53 inducible gene (He and Sun 2007, Li et al 2007), RPS27 is transcriptionally repressed by p53. Third, both family members bind to MDM2 and are subjected to MDM2 degradation in RING and acidic domain dependent manner when co-transfected with MDM2, but only RPS27L appears to be a physiological substrate of MDM2. Fourth, both proteins inhibit MDM2-mediated p53 ubiquitination and increase endogenous p53 levels under overexpressed condition, but only RPS27L, upon siRNA silencing, causes p53 destabilization. Taken together, our results suggest that upon p53 activation, RPS27L is induced which in turn competes with p53 for MDM2 binding, thus reducing MDM2-mediated p53 degradation and extending p53-activating signals. On the other hand, MDM2, upon induction by p53, degrades both p53 and RPS27L to terminate p53 activating signals. Repression of RPS27 by activated p53 makes MDM2 degradation of RPS27 unnecessary under stressed conditions.

Our finding that RPS27L or RPS27 binds to MDM2 via its central acidic domain add them into the list of other ribosomal proteins, including L5, L11, L23, and S7 (Dai and Lu 2004, Dai et al 2004, Zhu et al 2009), known to regulate p53 activity via binding to MDM2 at the same acidic domain or adjacent zinc finger region. However, among all MDM2-binding ribosomal proteins that inhibit MDM2-induced p53 polyubiquitination for p53 stabilization, RPS27L and RPS27 have several unique features. First, they are the only known MDM2-binding ribosomal or ribosomal-like proteins directly subjecting to p53 transcriptional regulation, thus are directly involved in feedback regulation of p53/MDM2 axis. Second, unlike other MDM2-binding ribosomal proteins which are localized either in nucleolus (e.g. L11 or S7) (Bhat et al 2004, Lohrum et al 2003, Zhu et al 2009) or in nucleoplasm, (e.g. L23) (Dai et al 2004, Jin et al 2004), RPS27L (or RPS27) is mainly localized in cytoplasm under physiological unstressed condition where very low level of cytoplasmic MDM2 exists. Under stressed conditions that activate p53, a portion of RPS27L is shuttled to nucleoplasm where it co-localized with induced MDM2. Since RPS27L and MDM2 in both cytoplasm and nucleus are induced upon p53 activating signals (Fig 8C), induced RPS27L could compete with p53 for Mdm2 binding, thus releasing/protecting p53 from MDM2-mediated degradation in both subcellular locations, as evidenced by a reduction of p53 levels at both cytoplasm and nucleus upon actinomycin D treatment, if RPS27L is silenced (Fig 9D). Finally, our study showing that RPS27L is a physiological substrate of MDM2 added RPS27L into the list of ribosomal proteins, including L26 (Ofir-Rosenfeld et al 2008) and S7 (Zhu et al 2009), that directly bind to MDM2 and serve as the substrates of MDM2. It is very likely that MDM2 in different sub-cellular compartments is responsible for the degradation of these ribosomal or ribosomal-like proteins under physiological condition or after stress stimuli.

RPS27 was found to be overexpressed in many human cancers (Atsuta et al 2002, Fernandez-Pol et al 1997, Ganger et al 1997, Ganger et al 2001, Lee et al 2004, Wang et al 2006). However, the underlying mechanism for its overexpression is unknown. Our study report here that RPS27 is subjected to transcriptional repression by wild type p53, but not mutant p53 (Fig 2) may provide one logical explanation. It is well known that p53 is mutated in ~50% of human cancer (Greenblatt et al 1994). Human cancers with p53 mutation would, therefore, likely have a higher RPS27 expression due to the loss of repression by wild type p53. On the other hand, overexpressed RPS27 may in turn stabilize mutant p53, since ectopic expression of RPS27 could increase p53 levels (Fig. 6B). This feed-forward regulation could confer growth advantage in these cancer cells. Future study will be directed to address this interesting correlation.

Our siRNA silencing study revealed that knockdown of RPS27L, but not RPS27, caused destabilization of basal p53 in A549 and SJSA cells (Fig. 7A & Fig. 9), suggesting under normal physiological condition, RPS27L is the one involving in the interplay with MDM2/p53 axis. It is noteworthy, however, that in some human cancer lines with wild type p53 (e.g. HCT116 and U2OS), RPS27L knockdown did not destabilize p53 (data not shown). This is likely due to a higher sensitivity of these lines to unknown stresses triggered by RPS27L knockdown to cause p53 induction, which counteracts direct modulation of MDM2/p53 axis by RPS27L. On the other hand, it is unclear why RPS27 silencing did not destabilize p53 in A549 cells, although it binds to MDM2 and inhibits MDM2-induced p53 degradation under overexpressed condition. One possibility is that RPS27 is a house-keeping ribosomal protein whose knockdown may trigger ribosomal stress which could induce p53. It is conceivable that, compared to RPS27L, house-keeping RPS27 has a long protein half-life and is resistant to MDM2-mediated degradation under physiological or even stressed conditions. This RPS27 unique feature could confer more flexibility for cells to deal with harsh environmental conditions without stopping essential protein synthesis.

In summary, our current work, along with our previous finding (He and Sun 2007), revealed a multi-level interplay between RPS27L and RPS27 family members and p53/MDM2 axis. p53 is activated upon DNA damage to transactivate MDM2 and RPS27L, and to transrepress RPS27 as well. On one hand, induced MDM2 binds to p53 and RPS27L and promotes their ubiquitination and degradation. On the other hand, induced RPS27L competes with p53 for MDM2 binding, thus releasing p53 from MDM2-mediated degradation, resulting in p53 stabilization. Through interacting with MDM2-p53 axis, RPS27L/S27 could regulate cell growth and survival (Fig 10).

Materials and MethodsCell culture and drug treatmentAll cell lines used in this study, except for MRC5, SJSA, and HCT116, were maintained in Dulbecco’s modified Eagle’s medium (DMEM) supplemented with 10% (v/v) fetal bovine serum (FBS). MRC5 and SJSA cells were maintained in RPMI 1640 Medium with 10% FBS. HCT116 cells were maintained in McCoy’s 5A medium containing 10% FBS. To change p53 conformation, the culture temperature for H1299-p53V138A cells (H1299-ts) was either 37°C (nonpermissive for wild-type p53) or 32°C (permissive for wild type). For drug treatment, subconfluent cells were incubated with actinomycin D, etoposide (Sigma, St Louis, MO, USA) and MI-219 (Shangary et al 2008) (a gift from Dr. Shaomeng Wang) for 24 hrs. The control cells were treated with dimethyl sulphoxide (DMSO). Subconfluent cells were also exposed to ionizing radiation at 6 Gy and harvested at 6 hrs or 24 hrs thereafter.DNA transfection, Western blotting analysis and immunoprecipitationCells, except for MEFs, were plated into six-well plate at 2 ×105 cells per well and transfected the following day with a variety of plasmid using Lipofectamine 2000 (Invitrogen, Carlsbad, CA, USA). MEF cells were plated into six-well plate at 1 ×106 cells per well and transfected with indicated plasmids using PolyJet DNA transfection reagent (Signagen Laboratories, Ijamsville, MD, USA). Cells were harvested, lysed and subjected to Western blotting analysis and immunoprecipitation, as described (Tan et al 2008), using various antibodies as follows: p53 (Ab-6, Calbiochem, San Diego, CA, USA), HA (Roche, IN, USA), FLAG and β-Actin, (Sigma, St Louis, MO, USA), MDM2 (Ab-1, Calbiochem; 2A9, gift from Jiandong Chen; N-20 and SMP14 from Santa Cruz Biotechnology, Santa Cruz, CA, USA), and p21 (BD Pharmingen, San Jose, CA, USA). RPS27L or RPS27 polyclonal rabbit antibody was raised and purified as described (He and Sun 2007). The antigenic peptides used to generate these two antibodies are RL-S27L: MPLARDLLHPSLEEC (for RPS27L) and KP-S27: MPLAKDLLHPSPEEC (for RPS27), respectively, differing in two amino acids from each other. After two runs of affinity purification and affinity absorption, the specificity of two antibodies were tested and confirmed in cells over-expressing each protein (see Fig. 1C).Pull-down assayTwo types of pull-down assays were performed. First, full length of RPS27L, RPS27, their N-terminal fragments (AA1–36), and C-terminal fragment (AA37–84) were expressed as GST-fusion proteins. The fusion proteins were purified with GSH-beads and eluted with GSH. Purified GST-tagged RPS27L/S27 proteins were then incubated with His-tagged-MDM2 immobilized on beads, along with the His-Tag control, followed by Western blotting using anti-GST antibody. Second, GST-fused MDM2 (1–491) and its deletion mutants were expressed in bacteria and purified using GSH-coated beads. About 500 ng of purified GST or GST-fusion proteins immobilized on GSH beads were used to incubate with cell lysates (2 mg), prepared from p53/MDM2-double null MEFs cells in a pull-down assay. After extensive washing, the beads were boiled, and bound RPS27L was detected by Western blotting analysis.Lentivirus-based siRNA and lentivirus infectionThe sequences of RPS27L/RPS27 siRNA oligonucleotides used for construction of lentivirus silencing vector are as follows: LT-S27L-01: 5’- AATGATTCAAACAGCTTCCTGTTCAAGAGACAGGAAGCTGTTTGAATCATTTTTT TTGT -3’, and LT-S27L-02: 5’- CTAGACAAAAA AAATGATTCAAACAGCTTCCTGTCTC TTGAACAGGAAGCTGTTTGAATCATT-3’; LT-S27-01: 5'-AAGCACTCTGAGTCAAGAT GATTCAAGAGATCATCTTGACTCAGAGTGCTTTTTTTTGT-3', and LT-S27-02: 5'-CTAGACAAAAAAAAGCACTCTGAGTCAAGATGATCTCTTGAATCATCTTGACTCAG AGTGCTT-3'. The sequences of scrambled control siRNA are LT-CONT-01 5’-ATTGTATGC GATCGCAGACTTTTCAAGAGAAAGTCTGCGATCGCATACAATTTTTTGT-3’; and LT-CONT-02 5’-CTAGACAAAAAATTGTATGCGATCGCAGACTTTCTCTTGAAAAGTCTGC GATCGCATACAAT-3’. Cells were infected and split, followed by assays as described (Jia et al 2009). To silence endogenous wt p53, cells were infected with a lenti-virus based scrambled control siRNA, or LT-p53-siRNA, as described (Sun et al 2008). The pooled siRNA oligos targeting MDM2 (SMART pool) was purchased from Dharmacon, Lafayette, CO.In vivo ubiquitination assayHuman lung H1299 cells were transiently transfected with various plasmids and harvested 24 hrs post transfection and split into two aliquots with one for direct Western blotting analysis and the other for in vivo ubiquitination assay as described (Gu et al 2007). Briefly, cell pellets were lysed in buffer A [6 mol/L guanidinium-HCl, 0.1 mol/L Na2HPO4/NaH2PO4, 10 mmol/L Tris-HCl (pH 8.0), 10 mmol/L β-mecaptoethanol] and incubated with Ni-NTA beads (Qiagen, Valencia, CA) at room temperature for 4 hrs. Beads were washed once with each of buffer A, buffer B [8 mol/L urea, 0.1 mol/L Na2HPO4/NaH2PO4, 10 mmol/L Tris-HCl (pH 8.0), 10 mmol/L β-mecaptoethanol], and buffer C [8 mol/L urea, 0.1 mol/L Na2HPO4/NaH2PO4, 10 mmol/L Tris-HCl (pH 6.3), 10 mmol/L β -mecaptoethanol]. Proteins were eluted from beads with buffer D [200 mmol/L imidazole, 0.15 mol/L Tris-HCl (pH 6.7), 30% glycerol, 0.72 M β -mecaptoethanol, 5% SDS]. The eluted proteins were analyzed by Western blotting for polyubiquitination of p53 with anti-p53 antibody.Quantitative RT-PCRTotal RNA was isolated from cells using a Trizol kit (Invitrogen, Carlsbad, CA, USA), and subjected to quantitative RT-PCR analysis, according to the manufacturer’s instruction of QuantiTect SYBR green RT-PCR kit (Qiagen, Valencia, CA). Cycling program was set as the following: 50 °C 30 min for RT, 95 °C 15 min for the PCR initial activation and 40 cycles of denaturation at 94 °C for 15 sec, annealing at 55 °C for 30 sec and extension at 72 °C for 30 sec. The sequences of RPS27, p53 and GAPDH are as follows: hu-p53 F1: TCTGTGACTTGCACGTAC, hu-p53 R1: ATTTCCTTCCACTCGGAT; RPS27 F1: GAGAACATGCCTCTCGCAAA, RPS27 R1: CTCCTGAAGGAACATCCTTCT; GAPDH-F1: GTTGCCATCAATGACCCCTT, GAPDH -R1: AGAGGCAGGGATGATGTTCT.Luciferase reporter assayH1299-ts cells (Robinson 2003) stably transfected with a p53-responsive BP100-luc reporter (Peng et al 2003) were infected with control siRNA or siRNA targeting RPS27L, and then split and grown at 32°C for 18 hrs, followed by luciferase reporter assay (Promega, Madison, WI, USA). The results are presented as the fold change with the value of siCont-transfected cells, arbitrarily setting as 1.Immunofluorescent stainingA549 cells were left untreated or treated for 24 hrs with ribosomal stress inducer, actinomycin D (5 nM), DNA damaging agent, etoposide (25 µM), or MI-219 (10 µM), a small molecule compound that disrupts MDM2-p53 binding to activate p53, followed by MDM2 accumulation (Shangary et al 2008). Cells were then immunostained with indicated antibodies and photographed (Jia et al 2009).Cellular FractionationAfter various drug treatments, cells were harvested and rinsed with Buffer A (10 mM HEPES-K+ pH7.5, 10 mM KCl, 1.5 mM MgCl2, 0.5 mM DTT, and protease inhibitor cocktail). The cells were then lysed in 0.5% NP-40 containing Buffer A for 10 min. The supernatant was harvested as cytoplasm protein by spinning at 3,000 rpm for 2 min at 4°C. The nuclear pellets were washed with Buffer A, followed by resuspending in Buffer B (20 mM HEPES-K+ pH7.9, 420 mM NaCl, 0.2 mM EDTA, 1.5 mM MgCl2, 0.5 mM DTT, 25% glycerol, and protease inhibitor cocktail) and incubated on ice for 30 min. Supernatant containing nuclear protein were collected by spinning at 14,500 rpm for 10 min at 4°C.Statistical analysisThe paired Student t test was used with SAS software.

Materials and Methods

Cell culture and drug treatmentAll cell lines used in this study, except for MRC5, SJSA, and HCT116, were maintained in Dulbecco’s modified Eagle’s medium (DMEM) supplemented with 10% (v/v) fetal bovine serum (FBS). MRC5 and SJSA cells were maintained in RPMI 1640 Medium with 10% FBS. HCT116 cells were maintained in McCoy’s 5A medium containing 10% FBS. To change p53 conformation, the culture temperature for H1299-p53V138A cells (H1299-ts) was either 37°C (nonpermissive for wild-type p53) or 32°C (permissive for wild type). For drug treatment, subconfluent cells were incubated with actinomycin D, etoposide (Sigma, St Louis, MO, USA) and MI-219 (Shangary et al 2008) (a gift from Dr. Shaomeng Wang) for 24 hrs. The control cells were treated with dimethyl sulphoxide (DMSO). Subconfluent cells were also exposed to ionizing radiation at 6 Gy and harvested at 6 hrs or 24 hrs thereafter.

Cell culture and drug treatment

All cell lines used in this study, except for MRC5, SJSA, and HCT116, were maintained in Dulbecco’s modified Eagle’s medium (DMEM) supplemented with 10% (v/v) fetal bovine serum (FBS). MRC5 and SJSA cells were maintained in RPMI 1640 Medium with 10% FBS. HCT116 cells were maintained in McCoy’s 5A medium containing 10% FBS. To change p53 conformation, the culture temperature for H1299-p53V138A cells (H1299-ts) was either 37°C (nonpermissive for wild-type p53) or 32°C (permissive for wild type). For drug treatment, subconfluent cells were incubated with actinomycin D, etoposide (Sigma, St Louis, MO, USA) and MI-219 (Shangary et al 2008) (a gift from Dr. Shaomeng Wang) for 24 hrs. The control cells were treated with dimethyl sulphoxide (DMSO). Subconfluent cells were also exposed to ionizing radiation at 6 Gy and harvested at 6 hrs or 24 hrs thereafter.

DNA transfection, Western blotting analysis and immunoprecipitationCells, except for MEFs, were plated into six-well plate at 2 ×105 cells per well and transfected the following day with a variety of plasmid using Lipofectamine 2000 (Invitrogen, Carlsbad, CA, USA). MEF cells were plated into six-well plate at 1 ×106 cells per well and transfected with indicated plasmids using PolyJet DNA transfection reagent (Signagen Laboratories, Ijamsville, MD, USA). Cells were harvested, lysed and subjected to Western blotting analysis and immunoprecipitation, as described (Tan et al 2008), using various antibodies as follows: p53 (Ab-6, Calbiochem, San Diego, CA, USA), HA (Roche, IN, USA), FLAG and β-Actin, (Sigma, St Louis, MO, USA), MDM2 (Ab-1, Calbiochem; 2A9, gift from Jiandong Chen; N-20 and SMP14 from Santa Cruz Biotechnology, Santa Cruz, CA, USA), and p21 (BD Pharmingen, San Jose, CA, USA). RPS27L or RPS27 polyclonal rabbit antibody was raised and purified as described (He and Sun 2007). The antigenic peptides used to generate these two antibodies are RL-S27L: MPLARDLLHPSLEEC (for RPS27L) and KP-S27: MPLAKDLLHPSPEEC (for RPS27), respectively, differing in two amino acids from each other. After two runs of affinity purification and affinity absorption, the specificity of two antibodies were tested and confirmed in cells over-expressing each protein (see Fig. 1C).

DNA transfection, Western blotting analysis and immunoprecipitation

Cells, except for MEFs, were plated into six-well plate at 2 ×105 cells per well and transfected the following day with a variety of plasmid using Lipofectamine 2000 (Invitrogen, Carlsbad, CA, USA). MEF cells were plated into six-well plate at 1 ×106 cells per well and transfected with indicated plasmids using PolyJet DNA transfection reagent (Signagen Laboratories, Ijamsville, MD, USA). Cells were harvested, lysed and subjected to Western blotting analysis and immunoprecipitation, as described (Tan et al 2008), using various antibodies as follows: p53 (Ab-6, Calbiochem, San Diego, CA, USA), HA (Roche, IN, USA), FLAG and β-Actin, (Sigma, St Louis, MO, USA), MDM2 (Ab-1, Calbiochem; 2A9, gift from Jiandong Chen; N-20 and SMP14 from Santa Cruz Biotechnology, Santa Cruz, CA, USA), and p21 (BD Pharmingen, San Jose, CA, USA). RPS27L or RPS27 polyclonal rabbit antibody was raised and purified as described (He and Sun 2007). The antigenic peptides used to generate these two antibodies are RL-S27L: MPLARDLLHPSLEEC (for RPS27L) and KP-S27: MPLAKDLLHPSPEEC (for RPS27), respectively, differing in two amino acids from each other. After two runs of affinity purification and affinity absorption, the specificity of two antibodies were tested and confirmed in cells over-expressing each protein (see Fig. 1C).

Pull-down assayTwo types of pull-down assays were performed. First, full length of RPS27L, RPS27, their N-terminal fragments (AA1–36), and C-terminal fragment (AA37–84) were expressed as GST-fusion proteins. The fusion proteins were purified with GSH-beads and eluted with GSH. Purified GST-tagged RPS27L/S27 proteins were then incubated with His-tagged-MDM2 immobilized on beads, along with the His-Tag control, followed by Western blotting using anti-GST antibody. Second, GST-fused MDM2 (1–491) and its deletion mutants were expressed in bacteria and purified using GSH-coated beads. About 500 ng of purified GST or GST-fusion proteins immobilized on GSH beads were used to incubate with cell lysates (2 mg), prepared from p53/MDM2-double null MEFs cells in a pull-down assay. After extensive washing, the beads were boiled, and bound RPS27L was detected by Western blotting analysis.

Pull-down assay

Two types of pull-down assays were performed. First, full length of RPS27L, RPS27, their N-terminal fragments (AA1–36), and C-terminal fragment (AA37–84) were expressed as GST-fusion proteins. The fusion proteins were purified with GSH-beads and eluted with GSH. Purified GST-tagged RPS27L/S27 proteins were then incubated with His-tagged-MDM2 immobilized on beads, along with the His-Tag control, followed by Western blotting using anti-GST antibody. Second, GST-fused MDM2 (1–491) and its deletion mutants were expressed in bacteria and purified using GSH-coated beads. About 500 ng of purified GST or GST-fusion proteins immobilized on GSH beads were used to incubate with cell lysates (2 mg), prepared from p53/MDM2-double null MEFs cells in a pull-down assay. After extensive washing, the beads were boiled, and bound RPS27L was detected by Western blotting analysis.

Lentivirus-based siRNA and lentivirus infectionThe sequences of RPS27L/RPS27 siRNA oligonucleotides used for construction of lentivirus silencing vector are as follows: LT-S27L-01: 5’- AATGATTCAAACAGCTTCCTGTTCAAGAGACAGGAAGCTGTTTGAATCATTTTTT TTGT -3’, and LT-S27L-02: 5’- CTAGACAAAAA AAATGATTCAAACAGCTTCCTGTCTC TTGAACAGGAAGCTGTTTGAATCATT-3’; LT-S27-01: 5'-AAGCACTCTGAGTCAAGAT GATTCAAGAGATCATCTTGACTCAGAGTGCTTTTTTTTGT-3', and LT-S27-02: 5'-CTAGACAAAAAAAAGCACTCTGAGTCAAGATGATCTCTTGAATCATCTTGACTCAG AGTGCTT-3'. The sequences of scrambled control siRNA are LT-CONT-01 5’-ATTGTATGC GATCGCAGACTTTTCAAGAGAAAGTCTGCGATCGCATACAATTTTTTGT-3’; and LT-CONT-02 5’-CTAGACAAAAAATTGTATGCGATCGCAGACTTTCTCTTGAAAAGTCTGC GATCGCATACAAT-3’. Cells were infected and split, followed by assays as described (Jia et al 2009). To silence endogenous wt p53, cells were infected with a lenti-virus based scrambled control siRNA, or LT-p53-siRNA, as described (Sun et al 2008). The pooled siRNA oligos targeting MDM2 (SMART pool) was purchased from Dharmacon, Lafayette, CO.

Lentivirus-based siRNA and lentivirus infection

The sequences of RPS27L/RPS27 siRNA oligonucleotides used for construction of lentivirus silencing vector are as follows: LT-S27L-01: 5’- AATGATTCAAACAGCTTCCTGTTCAAGAGACAGGAAGCTGTTTGAATCATTTTTT TTGT -3’, and LT-S27L-02: 5’- CTAGACAAAAA AAATGATTCAAACAGCTTCCTGTCTC TTGAACAGGAAGCTGTTTGAATCATT-3’; LT-S27-01: 5'-AAGCACTCTGAGTCAAGAT GATTCAAGAGATCATCTTGACTCAGAGTGCTTTTTTTTGT-3', and LT-S27-02: 5'-CTAGACAAAAAAAAGCACTCTGAGTCAAGATGATCTCTTGAATCATCTTGACTCAG AGTGCTT-3'. The sequences of scrambled control siRNA are LT-CONT-01 5’-ATTGTATGC GATCGCAGACTTTTCAAGAGAAAGTCTGCGATCGCATACAATTTTTTGT-3’; and LT-CONT-02 5’-CTAGACAAAAAATTGTATGCGATCGCAGACTTTCTCTTGAAAAGTCTGC GATCGCATACAAT-3’. Cells were infected and split, followed by assays as described (Jia et al 2009). To silence endogenous wt p53, cells were infected with a lenti-virus based scrambled control siRNA, or LT-p53-siRNA, as described (Sun et al 2008). The pooled siRNA oligos targeting MDM2 (SMART pool) was purchased from Dharmacon, Lafayette, CO.

In vivo ubiquitination assayHuman lung H1299 cells were transiently transfected with various plasmids and harvested 24 hrs post transfection and split into two aliquots with one for direct Western blotting analysis and the other for in vivo ubiquitination assay as described (Gu et al 2007). Briefly, cell pellets were lysed in buffer A [6 mol/L guanidinium-HCl, 0.1 mol/L Na2HPO4/NaH2PO4, 10 mmol/L Tris-HCl (pH 8.0), 10 mmol/L β-mecaptoethanol] and incubated with Ni-NTA beads (Qiagen, Valencia, CA) at room temperature for 4 hrs. Beads were washed once with each of buffer A, buffer B [8 mol/L urea, 0.1 mol/L Na2HPO4/NaH2PO4, 10 mmol/L Tris-HCl (pH 8.0), 10 mmol/L β-mecaptoethanol], and buffer C [8 mol/L urea, 0.1 mol/L Na2HPO4/NaH2PO4, 10 mmol/L Tris-HCl (pH 6.3), 10 mmol/L β -mecaptoethanol]. Proteins were eluted from beads with buffer D [200 mmol/L imidazole, 0.15 mol/L Tris-HCl (pH 6.7), 30% glycerol, 0.72 M β -mecaptoethanol, 5% SDS]. The eluted proteins were analyzed by Western blotting for polyubiquitination of p53 with anti-p53 antibody.

In vivo ubiquitination assay

Human lung H1299 cells were transiently transfected with various plasmids and harvested 24 hrs post transfection and split into two aliquots with one for direct Western blotting analysis and the other for in vivo ubiquitination assay as described (Gu et al 2007). Briefly, cell pellets were lysed in buffer A [6 mol/L guanidinium-HCl, 0.1 mol/L Na2HPO4/NaH2PO4, 10 mmol/L Tris-HCl (pH 8.0), 10 mmol/L β-mecaptoethanol] and incubated with Ni-NTA beads (Qiagen, Valencia, CA) at room temperature for 4 hrs. Beads were washed once with each of buffer A, buffer B [8 mol/L urea, 0.1 mol/L Na2HPO4/NaH2PO4, 10 mmol/L Tris-HCl (pH 8.0), 10 mmol/L β-mecaptoethanol], and buffer C [8 mol/L urea, 0.1 mol/L Na2HPO4/NaH2PO4, 10 mmol/L Tris-HCl (pH 6.3), 10 mmol/L β -mecaptoethanol]. Proteins were eluted from beads with buffer D [200 mmol/L imidazole, 0.15 mol/L Tris-HCl (pH 6.7), 30% glycerol, 0.72 M β -mecaptoethanol, 5% SDS]. The eluted proteins were analyzed by Western blotting for polyubiquitination of p53 with anti-p53 antibody.

Quantitative RT-PCRTotal RNA was isolated from cells using a Trizol kit (Invitrogen, Carlsbad, CA, USA), and subjected to quantitative RT-PCR analysis, according to the manufacturer’s instruction of QuantiTect SYBR green RT-PCR kit (Qiagen, Valencia, CA). Cycling program was set as the following: 50 °C 30 min for RT, 95 °C 15 min for the PCR initial activation and 40 cycles of denaturation at 94 °C for 15 sec, annealing at 55 °C for 30 sec and extension at 72 °C for 30 sec. The sequences of RPS27, p53 and GAPDH are as follows: hu-p53 F1: TCTGTGACTTGCACGTAC, hu-p53 R1: ATTTCCTTCCACTCGGAT; RPS27 F1: GAGAACATGCCTCTCGCAAA, RPS27 R1: CTCCTGAAGGAACATCCTTCT; GAPDH-F1: GTTGCCATCAATGACCCCTT, GAPDH -R1: AGAGGCAGGGATGATGTTCT.

Quantitative RT-PCR

Total RNA was isolated from cells using a Trizol kit (Invitrogen, Carlsbad, CA, USA), and subjected to quantitative RT-PCR analysis, according to the manufacturer’s instruction of QuantiTect SYBR green RT-PCR kit (Qiagen, Valencia, CA). Cycling program was set as the following: 50 °C 30 min for RT, 95 °C 15 min for the PCR initial activation and 40 cycles of denaturation at 94 °C for 15 sec, annealing at 55 °C for 30 sec and extension at 72 °C for 30 sec. The sequences of RPS27, p53 and GAPDH are as follows: hu-p53 F1: TCTGTGACTTGCACGTAC, hu-p53 R1: ATTTCCTTCCACTCGGAT; RPS27 F1: GAGAACATGCCTCTCGCAAA, RPS27 R1: CTCCTGAAGGAACATCCTTCT; GAPDH-F1: GTTGCCATCAATGACCCCTT, GAPDH -R1: AGAGGCAGGGATGATGTTCT.

Luciferase reporter assayH1299-ts cells (Robinson 2003) stably transfected with a p53-responsive BP100-luc reporter (Peng et al 2003) were infected with control siRNA or siRNA targeting RPS27L, and then split and grown at 32°C for 18 hrs, followed by luciferase reporter assay (Promega, Madison, WI, USA). The results are presented as the fold change with the value of siCont-transfected cells, arbitrarily setting as 1.

Luciferase reporter assay

H1299-ts cells (Robinson 2003) stably transfected with a p53-responsive BP100-luc reporter (Peng et al 2003) were infected with control siRNA or siRNA targeting RPS27L, and then split and grown at 32°C for 18 hrs, followed by luciferase reporter assay (Promega, Madison, WI, USA). The results are presented as the fold change with the value of siCont-transfected cells, arbitrarily setting as 1.

Immunofluorescent stainingA549 cells were left untreated or treated for 24 hrs with ribosomal stress inducer, actinomycin D (5 nM), DNA damaging agent, etoposide (25 µM), or MI-219 (10 µM), a small molecule compound that disrupts MDM2-p53 binding to activate p53, followed by MDM2 accumulation (Shangary et al 2008). Cells were then immunostained with indicated antibodies and photographed (Jia et al 2009).

Immunofluorescent staining

A549 cells were left untreated or treated for 24 hrs with ribosomal stress inducer, actinomycin D (5 nM), DNA damaging agent, etoposide (25 µM), or MI-219 (10 µM), a small molecule compound that disrupts MDM2-p53 binding to activate p53, followed by MDM2 accumulation (Shangary et al 2008). Cells were then immunostained with indicated antibodies and photographed (Jia et al 2009).

Cellular FractionationAfter various drug treatments, cells were harvested and rinsed with Buffer A (10 mM HEPES-K+ pH7.5, 10 mM KCl, 1.5 mM MgCl2, 0.5 mM DTT, and protease inhibitor cocktail). The cells were then lysed in 0.5% NP-40 containing Buffer A for 10 min. The supernatant was harvested as cytoplasm protein by spinning at 3,000 rpm for 2 min at 4°C. The nuclear pellets were washed with Buffer A, followed by resuspending in Buffer B (20 mM HEPES-K+ pH7.9, 420 mM NaCl, 0.2 mM EDTA, 1.5 mM MgCl2, 0.5 mM DTT, 25% glycerol, and protease inhibitor cocktail) and incubated on ice for 30 min. Supernatant containing nuclear protein were collected by spinning at 14,500 rpm for 10 min at 4°C.

Cellular Fractionation

After various drug treatments, cells were harvested and rinsed with Buffer A (10 mM HEPES-K+ pH7.5, 10 mM KCl, 1.5 mM MgCl2, 0.5 mM DTT, and protease inhibitor cocktail). The cells were then lysed in 0.5% NP-40 containing Buffer A for 10 min. The supernatant was harvested as cytoplasm protein by spinning at 3,000 rpm for 2 min at 4°C. The nuclear pellets were washed with Buffer A, followed by resuspending in Buffer B (20 mM HEPES-K+ pH7.9, 420 mM NaCl, 0.2 mM EDTA, 1.5 mM MgCl2, 0.5 mM DTT, 25% glycerol, and protease inhibitor cocktail) and incubated on ice for 30 min. Supernatant containing nuclear protein were collected by spinning at 14,500 rpm for 10 min at 4°C.

Statistical analysisThe paired Student t test was used with SAS software.

Statistical analysis

The paired Student t test was used with SAS software.
