---
pmid: '19455133'
title: Targeted tandem affinity purification of PSD-95 recovers core postsynaptic
  complexes and schizophrenia susceptibility proteins.
authors:
- Fernández E
- Collins MO
- Uren RT
- Kopanitsa MV
- Komiyama NH
- Croning MD
- Zografos L
- Armstrong JD
- Choudhary JS
- Grant SG
journal: Mol Syst Biol
year: '2009'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC2694677
doi: 10.1038/msb.2009.27
---

# Targeted tandem affinity purification of PSD-95 recovers core postsynaptic complexes and schizophrenia susceptibility proteins.
**Authors:** Fernández E, Collins MO, Uren RT, Kopanitsa MV, Komiyama NH, Croning MD, Zografos L, Armstrong JD, Choudhary JS, Grant SG
**Journal:** Mol Syst Biol (2009)
**DOI:** [10.1038/msb.2009.27](https://doi.org/10.1038/msb.2009.27)
**PMC:** [PMC2694677](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2694677/)

## Abstract

1. Mol Syst Biol. 2009;5:269. doi: 10.1038/msb.2009.27. Epub 2009 May 19.

Targeted tandem affinity purification of PSD-95 recovers core postsynaptic 
complexes and schizophrenia susceptibility proteins.

Fernández E(1), Collins MO, Uren RT, Kopanitsa MV, Komiyama NH, Croning MD, 
Zografos L, Armstrong JD, Choudhary JS, Grant SG.

Author information:
(1)Genes to Cognition Programme, The Wellcome Trust Sanger Institute, Cambridge, 
UK.

The molecular complexity of mammalian proteomes demands new methods for mapping 
the organization of multiprotein complexes. Here, we combine mouse genetics and 
proteomics to characterize synapse protein complexes and interaction networks. 
New tandem affinity purification (TAP) tags were fused to the carboxyl terminus 
of PSD-95 using gene targeting in mice. Homozygous mice showed no detectable 
abnormalities in PSD-95 expression, subcellular localization or synaptic 
electrophysiological function. Analysis of multiprotein complexes purified under 
native conditions by mass spectrometry defined known and new interactors: 118 
proteins comprising crucial functional components of synapses, including 
glutamate receptors, K+ channels, scaffolding and signaling proteins, were 
recovered. Network clustering of protein interactions generated five connected 
clusters, with two clusters containing all the major ionotropic glutamate 
receptors and one cluster with voltage-dependent K+ channels. Annotation of 
clusters with human disease associations revealed that multiple disorders map to 
the network, with a significant correlation of schizophrenia within the 
glutamate receptor clusters. This targeted TAP tagging strategy is generally 
applicable to mammalian proteomics and systems biology approaches to disease.

DOI: 10.1038/msb.2009.27
PMCID: PMC2694677
PMID: 19455133 [Indexed for MEDLINE]

Conflict of interest statement: The authors declare that they have no conflict 
of interest.

## Full Text

IntroductionSynapses are fundamental structural and functional units of the nervous system responsible for information processing. Their principal role is the transmission of electrical activity by the release of neurotransmitter from the presynaptic terminal onto postsynaptic receptors and ion channels. Postsynaptic ionotropic receptors initiate the postsynaptic depolarization that elicits action potential generation in the postsynaptic neuron. The second major role is the detection and processing of information contained in the patterns of electrical activity. This is achieved by the coupling of neurotransmitter receptors to second-messenger signaling pathways that modulate downstream effectors, ranging from modulation of ion channels themselves to structural changes and gene expression.In recent years, proteomic studies have revealed that mammalian synapses comprise up to 2000 proteins in the presynaptic and postsynaptic terminals (Husi et al, 2000; Walikonis et al, 2000; Husi and Grant, 2001; Sheng and Kim, 2002; Peng et al, 2004; Takamori et al, 2006; Trinidad et al, 2008). To understand the macromolecular organization of complexes and substructures, isolation of complexes by antibody, peptide and ligand affinity methods was used to recover smaller sets of proteins (Husi et al, 2000; Farr et al, 2004; Collins et al, 2006; Dosemeci et al, 2007; Klemmer et al, 2009; Paulo et al, 2009). These methods generally involve a single purification step, which is limited by the specificity of the affinity reagent and potentially recovers more contaminants than those with multiple steps. Furthermore, these protocols are not generally suitable for recovery of native complexes in solution, which could be used for enzymatic and structural studies.A potential solution to this major problem has been achieved in yeast through genetic modification of the endogenous protein by fusion with a Tandem Affinity Purification (TAP) tag into the C- or N-terminus of the protein of interest (Rigaut et al, 1999). This tagged protein can be isolated (with its associated proteins) in a tandem procedure, overcoming many of the inherent specificity and sensitivity limitations of traditional fractionation methods, as well as antibody, ligand and peptide affinity purification methods. In mammalian tissues, where developmental and cell-type control of regulation is more complex, the targeting of the TAP tag into the endogenous gene provides advantages over transgenic random integration or cDNA overexpression systems (Knuesel et al, 2003; Bouwmeester et al, 2004; Brajenovic et al, 2004; Drakas et al, 2005; Wang et al, 2005, 2006; Angrand et al, 2006; Burckstummer et al, 2006). For those reasons we chose to explore TAP tagging in mammals using gene targeting in mouse.Our first aim was to test TAP tagging using a gene-targeting approach in mice, with the specific objective of purifying signaling complexes from the synapse. We generated knockin mice in which TAP tags were inserted into the endogenous locus of post synaptic density-95 (PSD-95), which is one of the most abundant scaffold proteins at excitatory brain synapses (Nourry et al, 2003; Peng et al, 2004). PSD-95 is localized to the postsynaptic compartment in which it interacts with neurotransmitter receptors and ion channels to assemble signaling complexes (Kornau et al, 1995; Hunt et al, 1996; Tu et al, 1999; Husi et al, 2000; Nehring et al, 2000; Dosemeci et al, 2007) controlling neuronal plasticity (Migaud et al, 1998; Carlisle et al, 2008; Cuthbert et al, 2007) underlying learning and memory (Migaud et al, 1998), pain (Garry et al, 2003) and drug addiction (Yao et al, 2004). Our second aim was to integrate TAP tag proteomic data with systems biology approaches to analyze the organization and function of complexes.We show the first example of gene-targeted TAP tagging in mice and show that the tagging did not introduce a mutation or alter the expression or localization of the protein. Clear advantages of two-step purification methods over the existing single-step methods were found. Mass spectrometry analysis of four replicates of the purification revealed that PSD-95-associated complexes comprise the principal ionotropic glutamate receptors and K+ channels in addition to important signaling proteins. Text mining and systematic annotation together with clustering of proteins using protein interaction data revealed the network substructure with a core subnetwork involved in schizophrenia.

Introduction

Synapses are fundamental structural and functional units of the nervous system responsible for information processing. Their principal role is the transmission of electrical activity by the release of neurotransmitter from the presynaptic terminal onto postsynaptic receptors and ion channels. Postsynaptic ionotropic receptors initiate the postsynaptic depolarization that elicits action potential generation in the postsynaptic neuron. The second major role is the detection and processing of information contained in the patterns of electrical activity. This is achieved by the coupling of neurotransmitter receptors to second-messenger signaling pathways that modulate downstream effectors, ranging from modulation of ion channels themselves to structural changes and gene expression.

In recent years, proteomic studies have revealed that mammalian synapses comprise up to 2000 proteins in the presynaptic and postsynaptic terminals (Husi et al, 2000; Walikonis et al, 2000; Husi and Grant, 2001; Sheng and Kim, 2002; Peng et al, 2004; Takamori et al, 2006; Trinidad et al, 2008). To understand the macromolecular organization of complexes and substructures, isolation of complexes by antibody, peptide and ligand affinity methods was used to recover smaller sets of proteins (Husi et al, 2000; Farr et al, 2004; Collins et al, 2006; Dosemeci et al, 2007; Klemmer et al, 2009; Paulo et al, 2009). These methods generally involve a single purification step, which is limited by the specificity of the affinity reagent and potentially recovers more contaminants than those with multiple steps. Furthermore, these protocols are not generally suitable for recovery of native complexes in solution, which could be used for enzymatic and structural studies.

A potential solution to this major problem has been achieved in yeast through genetic modification of the endogenous protein by fusion with a Tandem Affinity Purification (TAP) tag into the C- or N-terminus of the protein of interest (Rigaut et al, 1999). This tagged protein can be isolated (with its associated proteins) in a tandem procedure, overcoming many of the inherent specificity and sensitivity limitations of traditional fractionation methods, as well as antibody, ligand and peptide affinity purification methods. In mammalian tissues, where developmental and cell-type control of regulation is more complex, the targeting of the TAP tag into the endogenous gene provides advantages over transgenic random integration or cDNA overexpression systems (Knuesel et al, 2003; Bouwmeester et al, 2004; Brajenovic et al, 2004; Drakas et al, 2005; Wang et al, 2005, 2006; Angrand et al, 2006; Burckstummer et al, 2006). For those reasons we chose to explore TAP tagging in mammals using gene targeting in mouse.

Our first aim was to test TAP tagging using a gene-targeting approach in mice, with the specific objective of purifying signaling complexes from the synapse. We generated knockin mice in which TAP tags were inserted into the endogenous locus of post synaptic density-95 (PSD-95), which is one of the most abundant scaffold proteins at excitatory brain synapses (Nourry et al, 2003; Peng et al, 2004). PSD-95 is localized to the postsynaptic compartment in which it interacts with neurotransmitter receptors and ion channels to assemble signaling complexes (Kornau et al, 1995; Hunt et al, 1996; Tu et al, 1999; Husi et al, 2000; Nehring et al, 2000; Dosemeci et al, 2007) controlling neuronal plasticity (Migaud et al, 1998; Carlisle et al, 2008; Cuthbert et al, 2007) underlying learning and memory (Migaud et al, 1998), pain (Garry et al, 2003) and drug addiction (Yao et al, 2004). Our second aim was to integrate TAP tag proteomic data with systems biology approaches to analyze the organization and function of complexes.

We show the first example of gene-targeted TAP tagging in mice and show that the tagging did not introduce a mutation or alter the expression or localization of the protein. Clear advantages of two-step purification methods over the existing single-step methods were found. Mass spectrometry analysis of four replicates of the purification revealed that PSD-95-associated complexes comprise the principal ionotropic glutamate receptors and K+ channels in addition to important signaling proteins. Text mining and systematic annotation together with clustering of proteins using protein interaction data revealed the network substructure with a core subnetwork involved in schizophrenia.

ResultsA strategy for purification of in vivo multiprotein complexesWe used a TAP tag consisting of a poly-histidine affinity tag (HAT) and a triple FLAG tag (Terpe, 2003) in tandem, separated by a unique TEV-protease cleavage site (Figure 1A). This 5-kDa tag is considerably smaller than the tag first applied in yeast (20 kDa) (Rigaut et al, 1999) and exploits the specificity of both FLAG and HAT-tag binding. Targeting the endogenous gene allows a thorough testing of the potential mutant phenotype by breeding to homozygosity and comparing with the existing mutant mice.Generation of a TAP-tagged PSD-95 knockin mouse lineWe chose to test TAP tagging in mice, with a focus on PSD-95 as a first model gene for the following reasons: (i) PSD-95 has discrete expression in the postsynaptic compartment of excitatory synapses of the brain, (ii) PSD-95 mutant mice are well characterized and show robust phenotypes in electrophysiological studies of synapses and behavior (Migaud et al, 1998; El-Husseini et al, 2000; Yao et al, 2004; Beique et al, 2006), and (iii) this protein has been extensively studied using methods that identify binary interaction partners (Kim and Sheng, 2004).PSD-95 is a scaffold protein with three PDZ domains, an SH3 and a guanylate kinase domain that mediate protein interactions (Figure 1A). As mouse PSD-95 is known to have multiple isoforms generated by multiple promoters and all forms utilize a common C-terminus (Bence et al, 2005), the TAP tag was inserted into the open reading frame in the 3′-end before the stop codon of exon 19, using Escherichia coli recombineering-based methods (Zhou et al, 2004) (Figure 1B). The final targeting vector, containing a 5′-end homology arm of 6.3 kb and a 3′-end homology arm of 2.9 kb, was transfected into ES cells and integration was detected using standard methods. PCR of neomycin-resistant ES-cell DNA confirmed the expected 3388 bp band in 16 clones (targeting efficiency was 5.6%), and germline transmission of the TAP-tag insertion was established (Figure 1C). This line of mice is referred to herein as PSD-95TAP.Normal expression and synaptic localization of TAP-tagged PSD-95We first intercrossed PSD-95TAP heterozygous mice (PSD-95TAP/+) and found no distortion of transmission frequency in the offspring of PSD-95TAP/+ intercrosses (data not shown). We next examined protein expression of TAP-tagged PSD-95 to ensure that introduction of the tag into the gene did not affect the expression and localization of the tagged protein. The solubilization conditions used here have been reported as the best conditions to mostly purify N-methyl-D-aspartate (NMDA) receptors and PSD-95 from adult mouse brain (Husi and Grant, 2001). The forebrain tissue was solubilized from heterozygous (PSD-95TAP/+) mice and PSD-95 was immunoprecipitated and immunoblotted using an anti-PSD-95 antibody (Figure 1D). Two bands of similar intensity were observed, where the upper band corresponded to the TAP-tagged PSD-95 (confirmed by immunoprecipitation using anti-FLAG antibody, right panel) and the lower band to the endogenous PSD-95. Comparison of extracts (5, 10, 15 μg) from wild-type (wt) and homozygous (PSD-95TAP/TAP) mice showed similar amounts of PSD-95 on immunoblots compared with an internal control immunoblot using anti-tubulin antibody (Figure 1E).We next carried out immunohistochemistry with an anti-PSD-95 antibody on sagittal brain sections to examine the expression pattern of TAP-tagged PSD-95. As shown in Figure 2A, the expression pattern of PSD-95 in PSD-95TAP/TAP brain was the same as in the brains of wt animals, with the highest expression in the CA1 area, dentate gyrus, cortex, cerebellum and lower expression in striatum and brainstem (Figure 2A). There was no detectable abnormality of brain morphology in the PSD-95TAP/TAP mice. As shown in Figure 2B, the expression of PSD-95 in the hippocampal subfields was unaffected by the genetic manipulation and particularly in the stratum radiatum, where the electrophysiological experiments were carried out (described below), was normal.To examine the synaptic localization of TAP-tagged PSD-95, we cultured embryonic hippocampal neurons from PSD-95TAP/TAP and wt mice. The specific subcellular localization of PSD-95 to the postsynaptic compartment of synapses (dendritic spines) was monitored using postsynaptic markers for glutamate neurotransmitter receptors (GluR1 or NR1), presynaptic marker (synaptophysin) and dendritic markers (MAP2) (Figure 2B and Supplementary Figure 1). Similar to PSD-95 staining in wt neurons, TAP-tagged PSD-95 was localized to punctate structures along the length of dendrites in PSD-95TAP/TAP neurons (Figure 2C, top panels). FLAG staining also shows the punctate structures in PSD-95TAP/TAP neurons (Figure 2C, bottom panels). Synaptophysin staining shows typical juxtaposition, indicating that TAP-tagged PSD-95 is found at synapses in PSD-95TAP/TAP neurons (Supplementary Figure 1, top panels). Furthermore, the co-localization of GluR1 and NR1 subunits with TAP-tagged PSD-95 (Supplementary Figure 1, middle and bottom panels, respectively) confirms its postsynaptic localization in the excitatory synapses, just as occurs for wt PSD-95.The TAP tag does not affect the synaptic electrophysiologyKnockout mutations or overexpression of PSD-95 results in striking changes in synaptic physiology (Migaud et al, 1998; El-Husseini et al, 2000; Beique et al, 2006). In particular, long-term potentiation (LTP) of the excitatory synaptic transmission is greatly enhanced in PSD-95 knockout mice (Migaud et al, 1998; Komiyama et al, 2002; Beique et al, 2006). To determine whether TAP tagging of PSD-95 also altered the synaptic physiology, we studied short- and long-term plasticity in hippocampal slices of PSD-95TAP/TAP mice (Figure 2D). A short episode of theta-burst stimulation was used to induce LTP of field extracellular post-synaptic potentials (fEPSPs) in the CA1 area of the hippocampus. In the period of 60–65 min after theta-burst stimulation, amplitudes of fEPSPs in the test pathway normalized relative to control pathway were not different between PSD95TAP/TAP and wt mice (194±15% versus 176±8%; P=0.295) (Figure 2D). Likewise, paired-pulse facilitation, an established measure of short-term plasticity, was similar in wt and PSD-95TAP/TAP animals (Supplementary Figure 2), whereas it is known that this parameter is significantly enhanced in PSD-95 knockout mice (Migaud et al, 1998; Beique et al, 2006). Therefore, we conclude that the engineering of the TAP tag into PSD-95 using the knockin strategy has not altered the synaptic physiological function of PSD-95. Together, the physiological, biochemical, tissue and subcellular localization studies indicate that the presence of the TAP tag did not significantly alter the expression or function of PSD-95.Optimization of single-step and tandem affinity purification of PSD-95-associated complexesThe following four-stage protocol for isolation of PSD-95 complexes was used (Figure 3A). First, TAP-tagged PSD-95 (from homozygous PSD-95TAP/TAP mice) was captured from brain extracts with an anti-FLAG antibody covalently coupled to Dynal beads. Second, the complex was eluted by cleavage with TEV protease completing the single step of purification. In the third stage, the complex was recovered from solution by Ni2+–NTA–agarose column that binds the HAT-tagged PSD-95. The fourth and final stage was the release of the PSD-95 complex from the column using imidazole, completing the tandem purification.We examined the efficiency of the different steps in this protocol by monitoring PSD-95. All the solubilized PSD-95 was captured by FLAG-Dynal beads and >90% was cleaved using TEV protease (Supplementary Figure 3A). In the absence of TEV, there was no spontaneous release of TAP-tagged PSD-95 during incubations (Figure 3B, lane 4, 5 Non-TEV, El lane). TEV incubation released 50–70% of cleaved PSD-95 (Figure 3B, lane 2, El) and 30–50% of cleaved PSD-95 remained on the beads (Figure 3B, lane 3, BB). TEV had efficiently cleaved the retained protein as the size of PSD-95 in the BB lane corresponded to cleaved TAP-tagged PSD-95, and moreover, was not recognized by anti-FLAG antibodies on immunoblotting (Supplementary Figure 3A, comparison of BB lanes). This partial retention of cleaved protein seems to be a protein-specific phenomenon as we have observed this with other proteins studied in a similar manner (data not shown). For other controls, as expected, the FLAG antibody did not precipitate PSD-95 from wt mouse forebrain (Figure 3B, right panel). The recovery of PSD-95 using a Ni2+–NTA–agarose column that binds the HAT-tagged PSD-95 was very efficient (>95%) (Figure 3C, lanes 2 and 3, TEV El, SN). Subsequent elution using imidazole was also highly efficient (>95%) (no detectable retained-PSD-95 in BB lane, Figure 3C). Overall, we estimate that the yield of the protein recovery was 50–60% of the total PSD-95 present in the brain lysate.Characterization of TAP-tagged PSD-95-associated complexesWe next examined the components of PSD-95 complexes and compared the single-step purification with the tandem purification. Complexes from PSD-95TAP/TAP and wt mice were subjected to SDS–PAGE and stained with colloidal Coomassie for band visualization before gel lanes were cut into slices (Figure 3D). These gels show a strong 95–100 kDa band in the PSD-95TAP/TAP lanes that corresponds to TAP-tagged PSD-95 and was absent in purifications from wt mice. A total of 301 different proteins were identified by LC-MS/MS from PSD-95TAP/TAP in all single-step and tandem purifications (Supplementary Tables 1 and 2).Toward identifying ‘core' complexes, we found 118 (39% of 301) proteins in three of four independent tandem purifications (a total of 158 proteins were found in all four tandem purification experiments) (Figure 3E and Tables I and II). The tandem purification has a significant technical advantage because abundant proteins (present in single step) ‘mask' less abundant proteins that appear after further purification (Wang et al, 2006). We found 71 (45%) out of the 158 proteins masked in the single purification. This is shown in the Venn diagram (Figure 3F). We created a Web source that provides access to this data and links to physiological and behavioral data from knockout mice for the respective genes (www.genes2cognition.org/TAP-PSD-95).To further explore the advantages of the TAP method, we examined the specific proteins to identify new PSD-95 interactors and also compared the types of proteins remaining in the core set after tandem purification. From the core complex of 118 proteins, 26 (22%) were reported as primary interactors (data present in HPRD, Biogrid, BIND and HOMOMINT databases) and included membrane-associated guanylate kinases (MAGUK or disc large homolog, Dlg family), NMDA receptor subunits, potassium channels and cytoskeletal proteins (Supplementary Table 3). Using immunoblotting we confirmed the presence of 13 PSD-95 interactors (Supplementary Figure 3B). Ten of those interactors were examined by reverse immunoprecipitation and all were validated (Supplementary Figure 3C). We also examined four new interactors using co-immunoprecipitation experiments: Arc/Arg3.1, Rac1, Nsf and Ablim1 (Figure 4 and Supplementary Figure 3B). These proteins are involved in cytoskeletal, vesicular-trafficking and G-protein-mediated signaling pathways.We used two methods to examine the types of proteins enriched by the tandem procedure. In the first approach, we grouped all proteins from both single-step and tandem purifications into ten functional categories and graphed the numbers of proteins in each category as a percentage of the dataset (Supplementary Figure 4). The tandem purification was enriched in channels/receptors, cytoskeletal/structural/adhesion and adaptors/regulatory proteins. There was a striking depletion of enzymes in the tandem purification compared with the single-step purification, consistent with the fact that many metabolic enzymes are abundant and can contaminate such purifications (Chen and Gingras, 2007). In the second approach, we considered the emPAI values (a semiquantitative measure of protein abundance based on MS data) for the 301 proteins and divided the dataset into two groups: tandem enriched and tandem depleted (Supplementary Table 4 and Supplementary Figure 5). Analysis of Gene Ontology (GO) terms showed that the tandem depleted set was significantly over-represented with the following GO terms: metabolism (P=1.67e−3), cytoplasm (P=2.10e−5) and mitochondrion (P=7.96e−4). In contrast, the tandem enriched set was significantly enriched with the following GO terms: signal transducer activity (P=6.16e−6), synapse (P=3.52e−7), postsynaptic membrane (P=3.22e−5) and cell communication (P=2.33e−5). Therefore we conclude that the TAP strategy can be used to recover a smaller and more specific subset of proteins than a single immunoaffinity (FLAG antibody) procedure.We compared the lists of proteins identified in the TAP experiments with earlier studies of synapse proteomes (Supplementary Tables 1 and 5). An earlier report using a single immunoprecipitation with a PSD-95 antibody identified 276 proteins from PSD fractions extracted in the absence of detergent (Dosemeci et al, 2007). The comparison of this list with the PSD-95 core complexes of 118 proteins reported here shows 49 proteins in common. A peptide affinity method for binding PDZ domains of MAGUK proteins (Husi et al, 2000; Husi and Grant, 2001; Collins et al, 2005; Emes et al, 2008) was used in the same extraction conditions reported here and recovered 105 proteins (Collins et al, 2005). This peptide affinity method was not specific to PSD-95 as the peptides are known to bind PSD-93 and SAP102 (Lim et al, 2002; Chung et al, 2004). These 105 proteins and the proteins found by NMDA-receptor immunopurification were used to generate a list of 186 MASC proteins (Collins et al, 2006). Comparison of our 118 PSD-95 TAP list with the 186 proteins from the MASC complex shows 48 proteins in common (Supplementary Tables 1 and 5). An important set of proteins that was recovered using the TAP method consisted of the AMPA (α-amino-3-hydroxy-5-methyl-4-isoxazolepropionic acid) receptors and K+ channels, which is discussed below. Overall we conclude that the targeted TAP tagging allowed for the enrichment of crucial synaptic proteins.Composition and organization of PSD-95 interaction networksTo explore the functional organization of the PSD-95 complexes, we reconstructed a network using protein–protein interaction data from high-quality manually-curated interaction data (Pocklington et al, 2006) and the UniHi database (http://www.mdc-berlin.de/unihi). After manual curation, we identified 119 interactions between 50 proteins (excluding self-interactions) from the 118 proteins in the PSD-95 core complex (Table I). No binary interactions were found for the remaining 68 proteins.Network clustering of the interacting proteins showed 40 out of the 50 proteins formed a large connected component (major connected component, MCC) and a modular structure that was segregated into five clusters (see Materials and methods), referred to as cluster a (Cla) to cluster e (Cle) (Figure 5A). In addition to the five MCC clusters, two further disconnected clusters (‘Clf' and ‘Clg') were found (see Table I for details).It is interesting to note the location and proximity of the receptors and channels responsible for the postsynaptic depolarization and subsequent action potential generation. All NMDA, AMPA and kainate glutamate receptors were restricted to Cla and Clb and the voltage-dependent K+ channels were found in Cla and Clc (entirely comprised of K+ channels). These channels are known to couple to plasticity mechanisms (Watanabe et al, 2002; Chen et al, 2006a; Kim et al, 2007), and we noted that Cla contains important signaling enzymes involved in plasticity, including CamKII (Frankland et al, 2001) and SynGAP (Komiyama et al, 2002). It therefore seems that Cla, Clb and Clc are enriched with membrane proteins responsible for the electrical properties of the postsynaptic terminal.As PSD-95/Dlg4 was the bait for the biochemical isolation of the complexes, we examined the distribution of its primary interactors (proteins that directly bind PSD-95) and secondary interactors (proteins that do not bind PSD-95 directly, but bind one of its primary interactors) (Figure 5A). Of the 39 MCC proteins (excluding PSD-95), 26 (67%) were primary interactors (blue symbols in Figure 5A) and 12 (31%) were secondary interactors (yellow symbols in Figure 5A) and only one protein, the AMPA receptor subunit Gria3, was a tertiary interactor. The majority of each cluster of the MCC comprised primary interactors: Cla (74%), Clb (43%), Clc (67%), Cld (50%) and Cle (50%). To examine the centrality of each protein in the network the shortest path from each protein to every other protein was counted, and the average shortest path (ASP) calculated. For all proteins, the mean ASP was 2.25. Ranking the ASP of each protein (Supplementary Table 6) showed PSD-95 had the lowest ASP (1.3), consistent with its central role in these networks.It was of interest to compare the PSD-95 network (MCC of 40 proteins) with the previously published MASC network (MCC of 90 proteins) that was built from proteins co-purified with the NMDA receptor complex and with the PDZ peptide affinity method described in the previous section (Pocklington et al, 2006). The MASC MCC had 13 clusters, with cluster 1 containing PSD-95 (Supplementary Figure 6). A total of 16 proteins were common to PSD-95 MCC and MASC MCC (P<10e−7) and with overlap centered (10/16 proteins, P<10e−3) on Cla and MASC cluster 1. In contrast to MASC MCC, the PSD-95 MCC had Clb (AMPA receptors) and Clc (K+ channels).Psychiatric disorders and PSD-95 complexesTo explore the potential medical importance of the 118 core PSD-95 interactome we asked (i) which diseases are these proteins involved with, and (ii) is there any relationship between the network clusters and disease types. For each of the proteins in the 118 core set, we manually curated information on their disease involvement from the literature. A total of 49 genes were implicated in multiple diseases: schizophrenia (28), mental retardation (6), bipolar disorder (13) Alzheimer's disease (6) and others (29) (Table III).We next analyzed the pair-wise correlation between functional categories (Tables I and II) and disease type. The ‘receptors/channels/transporters' category and ionotropic glutamate receptors were significantly correlated with schizophrenia (P=0.0024 and P<10e−6, respectively). Out of 28 schizophrenia-implicated proteins, 20 were mapped into the network model (orange in Figure 5B). Of those 20 proteins, 70% fell into Cla, which was significantly enriched in schizophrenia-related proteins (P=0.0089). All but one of the remaining schizophrenia-related proteins were found in cluster Clb.

A strategy for purification of in vivo multiprotein complexesWe used a TAP tag consisting of a poly-histidine affinity tag (HAT) and a triple FLAG tag (Terpe, 2003) in tandem, separated by a unique TEV-protease cleavage site (Figure 1A). This 5-kDa tag is considerably smaller than the tag first applied in yeast (20 kDa) (Rigaut et al, 1999) and exploits the specificity of both FLAG and HAT-tag binding. Targeting the endogenous gene allows a thorough testing of the potential mutant phenotype by breeding to homozygosity and comparing with the existing mutant mice.

A strategy for purification of in vivo multiprotein complexes

We used a TAP tag consisting of a poly-histidine affinity tag (HAT) and a triple FLAG tag (Terpe, 2003) in tandem, separated by a unique TEV-protease cleavage site (Figure 1A). This 5-kDa tag is considerably smaller than the tag first applied in yeast (20 kDa) (Rigaut et al, 1999) and exploits the specificity of both FLAG and HAT-tag binding. Targeting the endogenous gene allows a thorough testing of the potential mutant phenotype by breeding to homozygosity and comparing with the existing mutant mice.

Generation of a TAP-tagged PSD-95 knockin mouse lineWe chose to test TAP tagging in mice, with a focus on PSD-95 as a first model gene for the following reasons: (i) PSD-95 has discrete expression in the postsynaptic compartment of excitatory synapses of the brain, (ii) PSD-95 mutant mice are well characterized and show robust phenotypes in electrophysiological studies of synapses and behavior (Migaud et al, 1998; El-Husseini et al, 2000; Yao et al, 2004; Beique et al, 2006), and (iii) this protein has been extensively studied using methods that identify binary interaction partners (Kim and Sheng, 2004).PSD-95 is a scaffold protein with three PDZ domains, an SH3 and a guanylate kinase domain that mediate protein interactions (Figure 1A). As mouse PSD-95 is known to have multiple isoforms generated by multiple promoters and all forms utilize a common C-terminus (Bence et al, 2005), the TAP tag was inserted into the open reading frame in the 3′-end before the stop codon of exon 19, using Escherichia coli recombineering-based methods (Zhou et al, 2004) (Figure 1B). The final targeting vector, containing a 5′-end homology arm of 6.3 kb and a 3′-end homology arm of 2.9 kb, was transfected into ES cells and integration was detected using standard methods. PCR of neomycin-resistant ES-cell DNA confirmed the expected 3388 bp band in 16 clones (targeting efficiency was 5.6%), and germline transmission of the TAP-tag insertion was established (Figure 1C). This line of mice is referred to herein as PSD-95TAP.

Generation of a TAP-tagged PSD-95 knockin mouse line

We chose to test TAP tagging in mice, with a focus on PSD-95 as a first model gene for the following reasons: (i) PSD-95 has discrete expression in the postsynaptic compartment of excitatory synapses of the brain, (ii) PSD-95 mutant mice are well characterized and show robust phenotypes in electrophysiological studies of synapses and behavior (Migaud et al, 1998; El-Husseini et al, 2000; Yao et al, 2004; Beique et al, 2006), and (iii) this protein has been extensively studied using methods that identify binary interaction partners (Kim and Sheng, 2004).

PSD-95 is a scaffold protein with three PDZ domains, an SH3 and a guanylate kinase domain that mediate protein interactions (Figure 1A). As mouse PSD-95 is known to have multiple isoforms generated by multiple promoters and all forms utilize a common C-terminus (Bence et al, 2005), the TAP tag was inserted into the open reading frame in the 3′-end before the stop codon of exon 19, using Escherichia coli recombineering-based methods (Zhou et al, 2004) (Figure 1B). The final targeting vector, containing a 5′-end homology arm of 6.3 kb and a 3′-end homology arm of 2.9 kb, was transfected into ES cells and integration was detected using standard methods. PCR of neomycin-resistant ES-cell DNA confirmed the expected 3388 bp band in 16 clones (targeting efficiency was 5.6%), and germline transmission of the TAP-tag insertion was established (Figure 1C). This line of mice is referred to herein as PSD-95TAP.

Normal expression and synaptic localization of TAP-tagged PSD-95We first intercrossed PSD-95TAP heterozygous mice (PSD-95TAP/+) and found no distortion of transmission frequency in the offspring of PSD-95TAP/+ intercrosses (data not shown). We next examined protein expression of TAP-tagged PSD-95 to ensure that introduction of the tag into the gene did not affect the expression and localization of the tagged protein. The solubilization conditions used here have been reported as the best conditions to mostly purify N-methyl-D-aspartate (NMDA) receptors and PSD-95 from adult mouse brain (Husi and Grant, 2001). The forebrain tissue was solubilized from heterozygous (PSD-95TAP/+) mice and PSD-95 was immunoprecipitated and immunoblotted using an anti-PSD-95 antibody (Figure 1D). Two bands of similar intensity were observed, where the upper band corresponded to the TAP-tagged PSD-95 (confirmed by immunoprecipitation using anti-FLAG antibody, right panel) and the lower band to the endogenous PSD-95. Comparison of extracts (5, 10, 15 μg) from wild-type (wt) and homozygous (PSD-95TAP/TAP) mice showed similar amounts of PSD-95 on immunoblots compared with an internal control immunoblot using anti-tubulin antibody (Figure 1E).We next carried out immunohistochemistry with an anti-PSD-95 antibody on sagittal brain sections to examine the expression pattern of TAP-tagged PSD-95. As shown in Figure 2A, the expression pattern of PSD-95 in PSD-95TAP/TAP brain was the same as in the brains of wt animals, with the highest expression in the CA1 area, dentate gyrus, cortex, cerebellum and lower expression in striatum and brainstem (Figure 2A). There was no detectable abnormality of brain morphology in the PSD-95TAP/TAP mice. As shown in Figure 2B, the expression of PSD-95 in the hippocampal subfields was unaffected by the genetic manipulation and particularly in the stratum radiatum, where the electrophysiological experiments were carried out (described below), was normal.To examine the synaptic localization of TAP-tagged PSD-95, we cultured embryonic hippocampal neurons from PSD-95TAP/TAP and wt mice. The specific subcellular localization of PSD-95 to the postsynaptic compartment of synapses (dendritic spines) was monitored using postsynaptic markers for glutamate neurotransmitter receptors (GluR1 or NR1), presynaptic marker (synaptophysin) and dendritic markers (MAP2) (Figure 2B and Supplementary Figure 1). Similar to PSD-95 staining in wt neurons, TAP-tagged PSD-95 was localized to punctate structures along the length of dendrites in PSD-95TAP/TAP neurons (Figure 2C, top panels). FLAG staining also shows the punctate structures in PSD-95TAP/TAP neurons (Figure 2C, bottom panels). Synaptophysin staining shows typical juxtaposition, indicating that TAP-tagged PSD-95 is found at synapses in PSD-95TAP/TAP neurons (Supplementary Figure 1, top panels). Furthermore, the co-localization of GluR1 and NR1 subunits with TAP-tagged PSD-95 (Supplementary Figure 1, middle and bottom panels, respectively) confirms its postsynaptic localization in the excitatory synapses, just as occurs for wt PSD-95.

Normal expression and synaptic localization of TAP-tagged PSD-95

We first intercrossed PSD-95TAP heterozygous mice (PSD-95TAP/+) and found no distortion of transmission frequency in the offspring of PSD-95TAP/+ intercrosses (data not shown). We next examined protein expression of TAP-tagged PSD-95 to ensure that introduction of the tag into the gene did not affect the expression and localization of the tagged protein. The solubilization conditions used here have been reported as the best conditions to mostly purify N-methyl-D-aspartate (NMDA) receptors and PSD-95 from adult mouse brain (Husi and Grant, 2001). The forebrain tissue was solubilized from heterozygous (PSD-95TAP/+) mice and PSD-95 was immunoprecipitated and immunoblotted using an anti-PSD-95 antibody (Figure 1D). Two bands of similar intensity were observed, where the upper band corresponded to the TAP-tagged PSD-95 (confirmed by immunoprecipitation using anti-FLAG antibody, right panel) and the lower band to the endogenous PSD-95. Comparison of extracts (5, 10, 15 μg) from wild-type (wt) and homozygous (PSD-95TAP/TAP) mice showed similar amounts of PSD-95 on immunoblots compared with an internal control immunoblot using anti-tubulin antibody (Figure 1E).

We next carried out immunohistochemistry with an anti-PSD-95 antibody on sagittal brain sections to examine the expression pattern of TAP-tagged PSD-95. As shown in Figure 2A, the expression pattern of PSD-95 in PSD-95TAP/TAP brain was the same as in the brains of wt animals, with the highest expression in the CA1 area, dentate gyrus, cortex, cerebellum and lower expression in striatum and brainstem (Figure 2A). There was no detectable abnormality of brain morphology in the PSD-95TAP/TAP mice. As shown in Figure 2B, the expression of PSD-95 in the hippocampal subfields was unaffected by the genetic manipulation and particularly in the stratum radiatum, where the electrophysiological experiments were carried out (described below), was normal.

To examine the synaptic localization of TAP-tagged PSD-95, we cultured embryonic hippocampal neurons from PSD-95TAP/TAP and wt mice. The specific subcellular localization of PSD-95 to the postsynaptic compartment of synapses (dendritic spines) was monitored using postsynaptic markers for glutamate neurotransmitter receptors (GluR1 or NR1), presynaptic marker (synaptophysin) and dendritic markers (MAP2) (Figure 2B and Supplementary Figure 1). Similar to PSD-95 staining in wt neurons, TAP-tagged PSD-95 was localized to punctate structures along the length of dendrites in PSD-95TAP/TAP neurons (Figure 2C, top panels). FLAG staining also shows the punctate structures in PSD-95TAP/TAP neurons (Figure 2C, bottom panels). Synaptophysin staining shows typical juxtaposition, indicating that TAP-tagged PSD-95 is found at synapses in PSD-95TAP/TAP neurons (Supplementary Figure 1, top panels). Furthermore, the co-localization of GluR1 and NR1 subunits with TAP-tagged PSD-95 (Supplementary Figure 1, middle and bottom panels, respectively) confirms its postsynaptic localization in the excitatory synapses, just as occurs for wt PSD-95.

The TAP tag does not affect the synaptic electrophysiologyKnockout mutations or overexpression of PSD-95 results in striking changes in synaptic physiology (Migaud et al, 1998; El-Husseini et al, 2000; Beique et al, 2006). In particular, long-term potentiation (LTP) of the excitatory synaptic transmission is greatly enhanced in PSD-95 knockout mice (Migaud et al, 1998; Komiyama et al, 2002; Beique et al, 2006). To determine whether TAP tagging of PSD-95 also altered the synaptic physiology, we studied short- and long-term plasticity in hippocampal slices of PSD-95TAP/TAP mice (Figure 2D). A short episode of theta-burst stimulation was used to induce LTP of field extracellular post-synaptic potentials (fEPSPs) in the CA1 area of the hippocampus. In the period of 60–65 min after theta-burst stimulation, amplitudes of fEPSPs in the test pathway normalized relative to control pathway were not different between PSD95TAP/TAP and wt mice (194±15% versus 176±8%; P=0.295) (Figure 2D). Likewise, paired-pulse facilitation, an established measure of short-term plasticity, was similar in wt and PSD-95TAP/TAP animals (Supplementary Figure 2), whereas it is known that this parameter is significantly enhanced in PSD-95 knockout mice (Migaud et al, 1998; Beique et al, 2006). Therefore, we conclude that the engineering of the TAP tag into PSD-95 using the knockin strategy has not altered the synaptic physiological function of PSD-95. Together, the physiological, biochemical, tissue and subcellular localization studies indicate that the presence of the TAP tag did not significantly alter the expression or function of PSD-95.

The TAP tag does not affect the synaptic electrophysiology

Knockout mutations or overexpression of PSD-95 results in striking changes in synaptic physiology (Migaud et al, 1998; El-Husseini et al, 2000; Beique et al, 2006). In particular, long-term potentiation (LTP) of the excitatory synaptic transmission is greatly enhanced in PSD-95 knockout mice (Migaud et al, 1998; Komiyama et al, 2002; Beique et al, 2006). To determine whether TAP tagging of PSD-95 also altered the synaptic physiology, we studied short- and long-term plasticity in hippocampal slices of PSD-95TAP/TAP mice (Figure 2D). A short episode of theta-burst stimulation was used to induce LTP of field extracellular post-synaptic potentials (fEPSPs) in the CA1 area of the hippocampus. In the period of 60–65 min after theta-burst stimulation, amplitudes of fEPSPs in the test pathway normalized relative to control pathway were not different between PSD95TAP/TAP and wt mice (194±15% versus 176±8%; P=0.295) (Figure 2D). Likewise, paired-pulse facilitation, an established measure of short-term plasticity, was similar in wt and PSD-95TAP/TAP animals (Supplementary Figure 2), whereas it is known that this parameter is significantly enhanced in PSD-95 knockout mice (Migaud et al, 1998; Beique et al, 2006). Therefore, we conclude that the engineering of the TAP tag into PSD-95 using the knockin strategy has not altered the synaptic physiological function of PSD-95. Together, the physiological, biochemical, tissue and subcellular localization studies indicate that the presence of the TAP tag did not significantly alter the expression or function of PSD-95.

Optimization of single-step and tandem affinity purification of PSD-95-associated complexesThe following four-stage protocol for isolation of PSD-95 complexes was used (Figure 3A). First, TAP-tagged PSD-95 (from homozygous PSD-95TAP/TAP mice) was captured from brain extracts with an anti-FLAG antibody covalently coupled to Dynal beads. Second, the complex was eluted by cleavage with TEV protease completing the single step of purification. In the third stage, the complex was recovered from solution by Ni2+–NTA–agarose column that binds the HAT-tagged PSD-95. The fourth and final stage was the release of the PSD-95 complex from the column using imidazole, completing the tandem purification.We examined the efficiency of the different steps in this protocol by monitoring PSD-95. All the solubilized PSD-95 was captured by FLAG-Dynal beads and >90% was cleaved using TEV protease (Supplementary Figure 3A). In the absence of TEV, there was no spontaneous release of TAP-tagged PSD-95 during incubations (Figure 3B, lane 4, 5 Non-TEV, El lane). TEV incubation released 50–70% of cleaved PSD-95 (Figure 3B, lane 2, El) and 30–50% of cleaved PSD-95 remained on the beads (Figure 3B, lane 3, BB). TEV had efficiently cleaved the retained protein as the size of PSD-95 in the BB lane corresponded to cleaved TAP-tagged PSD-95, and moreover, was not recognized by anti-FLAG antibodies on immunoblotting (Supplementary Figure 3A, comparison of BB lanes). This partial retention of cleaved protein seems to be a protein-specific phenomenon as we have observed this with other proteins studied in a similar manner (data not shown). For other controls, as expected, the FLAG antibody did not precipitate PSD-95 from wt mouse forebrain (Figure 3B, right panel). The recovery of PSD-95 using a Ni2+–NTA–agarose column that binds the HAT-tagged PSD-95 was very efficient (>95%) (Figure 3C, lanes 2 and 3, TEV El, SN). Subsequent elution using imidazole was also highly efficient (>95%) (no detectable retained-PSD-95 in BB lane, Figure 3C). Overall, we estimate that the yield of the protein recovery was 50–60% of the total PSD-95 present in the brain lysate.

Optimization of single-step and tandem affinity purification of PSD-95-associated complexes

The following four-stage protocol for isolation of PSD-95 complexes was used (Figure 3A). First, TAP-tagged PSD-95 (from homozygous PSD-95TAP/TAP mice) was captured from brain extracts with an anti-FLAG antibody covalently coupled to Dynal beads. Second, the complex was eluted by cleavage with TEV protease completing the single step of purification. In the third stage, the complex was recovered from solution by Ni2+–NTA–agarose column that binds the HAT-tagged PSD-95. The fourth and final stage was the release of the PSD-95 complex from the column using imidazole, completing the tandem purification.

We examined the efficiency of the different steps in this protocol by monitoring PSD-95. All the solubilized PSD-95 was captured by FLAG-Dynal beads and >90% was cleaved using TEV protease (Supplementary Figure 3A). In the absence of TEV, there was no spontaneous release of TAP-tagged PSD-95 during incubations (Figure 3B, lane 4, 5 Non-TEV, El lane). TEV incubation released 50–70% of cleaved PSD-95 (Figure 3B, lane 2, El) and 30–50% of cleaved PSD-95 remained on the beads (Figure 3B, lane 3, BB). TEV had efficiently cleaved the retained protein as the size of PSD-95 in the BB lane corresponded to cleaved TAP-tagged PSD-95, and moreover, was not recognized by anti-FLAG antibodies on immunoblotting (Supplementary Figure 3A, comparison of BB lanes). This partial retention of cleaved protein seems to be a protein-specific phenomenon as we have observed this with other proteins studied in a similar manner (data not shown). For other controls, as expected, the FLAG antibody did not precipitate PSD-95 from wt mouse forebrain (Figure 3B, right panel). The recovery of PSD-95 using a Ni2+–NTA–agarose column that binds the HAT-tagged PSD-95 was very efficient (>95%) (Figure 3C, lanes 2 and 3, TEV El, SN). Subsequent elution using imidazole was also highly efficient (>95%) (no detectable retained-PSD-95 in BB lane, Figure 3C). Overall, we estimate that the yield of the protein recovery was 50–60% of the total PSD-95 present in the brain lysate.

Characterization of TAP-tagged PSD-95-associated complexesWe next examined the components of PSD-95 complexes and compared the single-step purification with the tandem purification. Complexes from PSD-95TAP/TAP and wt mice were subjected to SDS–PAGE and stained with colloidal Coomassie for band visualization before gel lanes were cut into slices (Figure 3D). These gels show a strong 95–100 kDa band in the PSD-95TAP/TAP lanes that corresponds to TAP-tagged PSD-95 and was absent in purifications from wt mice. A total of 301 different proteins were identified by LC-MS/MS from PSD-95TAP/TAP in all single-step and tandem purifications (Supplementary Tables 1 and 2).Toward identifying ‘core' complexes, we found 118 (39% of 301) proteins in three of four independent tandem purifications (a total of 158 proteins were found in all four tandem purification experiments) (Figure 3E and Tables I and II). The tandem purification has a significant technical advantage because abundant proteins (present in single step) ‘mask' less abundant proteins that appear after further purification (Wang et al, 2006). We found 71 (45%) out of the 158 proteins masked in the single purification. This is shown in the Venn diagram (Figure 3F). We created a Web source that provides access to this data and links to physiological and behavioral data from knockout mice for the respective genes (www.genes2cognition.org/TAP-PSD-95).To further explore the advantages of the TAP method, we examined the specific proteins to identify new PSD-95 interactors and also compared the types of proteins remaining in the core set after tandem purification. From the core complex of 118 proteins, 26 (22%) were reported as primary interactors (data present in HPRD, Biogrid, BIND and HOMOMINT databases) and included membrane-associated guanylate kinases (MAGUK or disc large homolog, Dlg family), NMDA receptor subunits, potassium channels and cytoskeletal proteins (Supplementary Table 3). Using immunoblotting we confirmed the presence of 13 PSD-95 interactors (Supplementary Figure 3B). Ten of those interactors were examined by reverse immunoprecipitation and all were validated (Supplementary Figure 3C). We also examined four new interactors using co-immunoprecipitation experiments: Arc/Arg3.1, Rac1, Nsf and Ablim1 (Figure 4 and Supplementary Figure 3B). These proteins are involved in cytoskeletal, vesicular-trafficking and G-protein-mediated signaling pathways.We used two methods to examine the types of proteins enriched by the tandem procedure. In the first approach, we grouped all proteins from both single-step and tandem purifications into ten functional categories and graphed the numbers of proteins in each category as a percentage of the dataset (Supplementary Figure 4). The tandem purification was enriched in channels/receptors, cytoskeletal/structural/adhesion and adaptors/regulatory proteins. There was a striking depletion of enzymes in the tandem purification compared with the single-step purification, consistent with the fact that many metabolic enzymes are abundant and can contaminate such purifications (Chen and Gingras, 2007). In the second approach, we considered the emPAI values (a semiquantitative measure of protein abundance based on MS data) for the 301 proteins and divided the dataset into two groups: tandem enriched and tandem depleted (Supplementary Table 4 and Supplementary Figure 5). Analysis of Gene Ontology (GO) terms showed that the tandem depleted set was significantly over-represented with the following GO terms: metabolism (P=1.67e−3), cytoplasm (P=2.10e−5) and mitochondrion (P=7.96e−4). In contrast, the tandem enriched set was significantly enriched with the following GO terms: signal transducer activity (P=6.16e−6), synapse (P=3.52e−7), postsynaptic membrane (P=3.22e−5) and cell communication (P=2.33e−5). Therefore we conclude that the TAP strategy can be used to recover a smaller and more specific subset of proteins than a single immunoaffinity (FLAG antibody) procedure.We compared the lists of proteins identified in the TAP experiments with earlier studies of synapse proteomes (Supplementary Tables 1 and 5). An earlier report using a single immunoprecipitation with a PSD-95 antibody identified 276 proteins from PSD fractions extracted in the absence of detergent (Dosemeci et al, 2007). The comparison of this list with the PSD-95 core complexes of 118 proteins reported here shows 49 proteins in common. A peptide affinity method for binding PDZ domains of MAGUK proteins (Husi et al, 2000; Husi and Grant, 2001; Collins et al, 2005; Emes et al, 2008) was used in the same extraction conditions reported here and recovered 105 proteins (Collins et al, 2005). This peptide affinity method was not specific to PSD-95 as the peptides are known to bind PSD-93 and SAP102 (Lim et al, 2002; Chung et al, 2004). These 105 proteins and the proteins found by NMDA-receptor immunopurification were used to generate a list of 186 MASC proteins (Collins et al, 2006). Comparison of our 118 PSD-95 TAP list with the 186 proteins from the MASC complex shows 48 proteins in common (Supplementary Tables 1 and 5). An important set of proteins that was recovered using the TAP method consisted of the AMPA (α-amino-3-hydroxy-5-methyl-4-isoxazolepropionic acid) receptors and K+ channels, which is discussed below. Overall we conclude that the targeted TAP tagging allowed for the enrichment of crucial synaptic proteins.

Characterization of TAP-tagged PSD-95-associated complexes

We next examined the components of PSD-95 complexes and compared the single-step purification with the tandem purification. Complexes from PSD-95TAP/TAP and wt mice were subjected to SDS–PAGE and stained with colloidal Coomassie for band visualization before gel lanes were cut into slices (Figure 3D). These gels show a strong 95–100 kDa band in the PSD-95TAP/TAP lanes that corresponds to TAP-tagged PSD-95 and was absent in purifications from wt mice. A total of 301 different proteins were identified by LC-MS/MS from PSD-95TAP/TAP in all single-step and tandem purifications (Supplementary Tables 1 and 2).

Toward identifying ‘core' complexes, we found 118 (39% of 301) proteins in three of four independent tandem purifications (a total of 158 proteins were found in all four tandem purification experiments) (Figure 3E and Tables I and II). The tandem purification has a significant technical advantage because abundant proteins (present in single step) ‘mask' less abundant proteins that appear after further purification (Wang et al, 2006). We found 71 (45%) out of the 158 proteins masked in the single purification. This is shown in the Venn diagram (Figure 3F). We created a Web source that provides access to this data and links to physiological and behavioral data from knockout mice for the respective genes (www.genes2cognition.org/TAP-PSD-95).

To further explore the advantages of the TAP method, we examined the specific proteins to identify new PSD-95 interactors and also compared the types of proteins remaining in the core set after tandem purification. From the core complex of 118 proteins, 26 (22%) were reported as primary interactors (data present in HPRD, Biogrid, BIND and HOMOMINT databases) and included membrane-associated guanylate kinases (MAGUK or disc large homolog, Dlg family), NMDA receptor subunits, potassium channels and cytoskeletal proteins (Supplementary Table 3). Using immunoblotting we confirmed the presence of 13 PSD-95 interactors (Supplementary Figure 3B). Ten of those interactors were examined by reverse immunoprecipitation and all were validated (Supplementary Figure 3C). We also examined four new interactors using co-immunoprecipitation experiments: Arc/Arg3.1, Rac1, Nsf and Ablim1 (Figure 4 and Supplementary Figure 3B). These proteins are involved in cytoskeletal, vesicular-trafficking and G-protein-mediated signaling pathways.

We used two methods to examine the types of proteins enriched by the tandem procedure. In the first approach, we grouped all proteins from both single-step and tandem purifications into ten functional categories and graphed the numbers of proteins in each category as a percentage of the dataset (Supplementary Figure 4). The tandem purification was enriched in channels/receptors, cytoskeletal/structural/adhesion and adaptors/regulatory proteins. There was a striking depletion of enzymes in the tandem purification compared with the single-step purification, consistent with the fact that many metabolic enzymes are abundant and can contaminate such purifications (Chen and Gingras, 2007). In the second approach, we considered the emPAI values (a semiquantitative measure of protein abundance based on MS data) for the 301 proteins and divided the dataset into two groups: tandem enriched and tandem depleted (Supplementary Table 4 and Supplementary Figure 5). Analysis of Gene Ontology (GO) terms showed that the tandem depleted set was significantly over-represented with the following GO terms: metabolism (P=1.67e−3), cytoplasm (P=2.10e−5) and mitochondrion (P=7.96e−4). In contrast, the tandem enriched set was significantly enriched with the following GO terms: signal transducer activity (P=6.16e−6), synapse (P=3.52e−7), postsynaptic membrane (P=3.22e−5) and cell communication (P=2.33e−5). Therefore we conclude that the TAP strategy can be used to recover a smaller and more specific subset of proteins than a single immunoaffinity (FLAG antibody) procedure.

We compared the lists of proteins identified in the TAP experiments with earlier studies of synapse proteomes (Supplementary Tables 1 and 5). An earlier report using a single immunoprecipitation with a PSD-95 antibody identified 276 proteins from PSD fractions extracted in the absence of detergent (Dosemeci et al, 2007). The comparison of this list with the PSD-95 core complexes of 118 proteins reported here shows 49 proteins in common. A peptide affinity method for binding PDZ domains of MAGUK proteins (Husi et al, 2000; Husi and Grant, 2001; Collins et al, 2005; Emes et al, 2008) was used in the same extraction conditions reported here and recovered 105 proteins (Collins et al, 2005). This peptide affinity method was not specific to PSD-95 as the peptides are known to bind PSD-93 and SAP102 (Lim et al, 2002; Chung et al, 2004). These 105 proteins and the proteins found by NMDA-receptor immunopurification were used to generate a list of 186 MASC proteins (Collins et al, 2006). Comparison of our 118 PSD-95 TAP list with the 186 proteins from the MASC complex shows 48 proteins in common (Supplementary Tables 1 and 5). An important set of proteins that was recovered using the TAP method consisted of the AMPA (α-amino-3-hydroxy-5-methyl-4-isoxazolepropionic acid) receptors and K+ channels, which is discussed below. Overall we conclude that the targeted TAP tagging allowed for the enrichment of crucial synaptic proteins.

Composition and organization of PSD-95 interaction networksTo explore the functional organization of the PSD-95 complexes, we reconstructed a network using protein–protein interaction data from high-quality manually-curated interaction data (Pocklington et al, 2006) and the UniHi database (http://www.mdc-berlin.de/unihi). After manual curation, we identified 119 interactions between 50 proteins (excluding self-interactions) from the 118 proteins in the PSD-95 core complex (Table I). No binary interactions were found for the remaining 68 proteins.Network clustering of the interacting proteins showed 40 out of the 50 proteins formed a large connected component (major connected component, MCC) and a modular structure that was segregated into five clusters (see Materials and methods), referred to as cluster a (Cla) to cluster e (Cle) (Figure 5A). In addition to the five MCC clusters, two further disconnected clusters (‘Clf' and ‘Clg') were found (see Table I for details).It is interesting to note the location and proximity of the receptors and channels responsible for the postsynaptic depolarization and subsequent action potential generation. All NMDA, AMPA and kainate glutamate receptors were restricted to Cla and Clb and the voltage-dependent K+ channels were found in Cla and Clc (entirely comprised of K+ channels). These channels are known to couple to plasticity mechanisms (Watanabe et al, 2002; Chen et al, 2006a; Kim et al, 2007), and we noted that Cla contains important signaling enzymes involved in plasticity, including CamKII (Frankland et al, 2001) and SynGAP (Komiyama et al, 2002). It therefore seems that Cla, Clb and Clc are enriched with membrane proteins responsible for the electrical properties of the postsynaptic terminal.As PSD-95/Dlg4 was the bait for the biochemical isolation of the complexes, we examined the distribution of its primary interactors (proteins that directly bind PSD-95) and secondary interactors (proteins that do not bind PSD-95 directly, but bind one of its primary interactors) (Figure 5A). Of the 39 MCC proteins (excluding PSD-95), 26 (67%) were primary interactors (blue symbols in Figure 5A) and 12 (31%) were secondary interactors (yellow symbols in Figure 5A) and only one protein, the AMPA receptor subunit Gria3, was a tertiary interactor. The majority of each cluster of the MCC comprised primary interactors: Cla (74%), Clb (43%), Clc (67%), Cld (50%) and Cle (50%). To examine the centrality of each protein in the network the shortest path from each protein to every other protein was counted, and the average shortest path (ASP) calculated. For all proteins, the mean ASP was 2.25. Ranking the ASP of each protein (Supplementary Table 6) showed PSD-95 had the lowest ASP (1.3), consistent with its central role in these networks.It was of interest to compare the PSD-95 network (MCC of 40 proteins) with the previously published MASC network (MCC of 90 proteins) that was built from proteins co-purified with the NMDA receptor complex and with the PDZ peptide affinity method described in the previous section (Pocklington et al, 2006). The MASC MCC had 13 clusters, with cluster 1 containing PSD-95 (Supplementary Figure 6). A total of 16 proteins were common to PSD-95 MCC and MASC MCC (P<10e−7) and with overlap centered (10/16 proteins, P<10e−3) on Cla and MASC cluster 1. In contrast to MASC MCC, the PSD-95 MCC had Clb (AMPA receptors) and Clc (K+ channels).

Composition and organization of PSD-95 interaction networks

To explore the functional organization of the PSD-95 complexes, we reconstructed a network using protein–protein interaction data from high-quality manually-curated interaction data (Pocklington et al, 2006) and the UniHi database (http://www.mdc-berlin.de/unihi). After manual curation, we identified 119 interactions between 50 proteins (excluding self-interactions) from the 118 proteins in the PSD-95 core complex (Table I). No binary interactions were found for the remaining 68 proteins.

Network clustering of the interacting proteins showed 40 out of the 50 proteins formed a large connected component (major connected component, MCC) and a modular structure that was segregated into five clusters (see Materials and methods), referred to as cluster a (Cla) to cluster e (Cle) (Figure 5A). In addition to the five MCC clusters, two further disconnected clusters (‘Clf' and ‘Clg') were found (see Table I for details).

It is interesting to note the location and proximity of the receptors and channels responsible for the postsynaptic depolarization and subsequent action potential generation. All NMDA, AMPA and kainate glutamate receptors were restricted to Cla and Clb and the voltage-dependent K+ channels were found in Cla and Clc (entirely comprised of K+ channels). These channels are known to couple to plasticity mechanisms (Watanabe et al, 2002; Chen et al, 2006a; Kim et al, 2007), and we noted that Cla contains important signaling enzymes involved in plasticity, including CamKII (Frankland et al, 2001) and SynGAP (Komiyama et al, 2002). It therefore seems that Cla, Clb and Clc are enriched with membrane proteins responsible for the electrical properties of the postsynaptic terminal.

As PSD-95/Dlg4 was the bait for the biochemical isolation of the complexes, we examined the distribution of its primary interactors (proteins that directly bind PSD-95) and secondary interactors (proteins that do not bind PSD-95 directly, but bind one of its primary interactors) (Figure 5A). Of the 39 MCC proteins (excluding PSD-95), 26 (67%) were primary interactors (blue symbols in Figure 5A) and 12 (31%) were secondary interactors (yellow symbols in Figure 5A) and only one protein, the AMPA receptor subunit Gria3, was a tertiary interactor. The majority of each cluster of the MCC comprised primary interactors: Cla (74%), Clb (43%), Clc (67%), Cld (50%) and Cle (50%). To examine the centrality of each protein in the network the shortest path from each protein to every other protein was counted, and the average shortest path (ASP) calculated. For all proteins, the mean ASP was 2.25. Ranking the ASP of each protein (Supplementary Table 6) showed PSD-95 had the lowest ASP (1.3), consistent with its central role in these networks.

It was of interest to compare the PSD-95 network (MCC of 40 proteins) with the previously published MASC network (MCC of 90 proteins) that was built from proteins co-purified with the NMDA receptor complex and with the PDZ peptide affinity method described in the previous section (Pocklington et al, 2006). The MASC MCC had 13 clusters, with cluster 1 containing PSD-95 (Supplementary Figure 6). A total of 16 proteins were common to PSD-95 MCC and MASC MCC (P<10e−7) and with overlap centered (10/16 proteins, P<10e−3) on Cla and MASC cluster 1. In contrast to MASC MCC, the PSD-95 MCC had Clb (AMPA receptors) and Clc (K+ channels).

Psychiatric disorders and PSD-95 complexesTo explore the potential medical importance of the 118 core PSD-95 interactome we asked (i) which diseases are these proteins involved with, and (ii) is there any relationship between the network clusters and disease types. For each of the proteins in the 118 core set, we manually curated information on their disease involvement from the literature. A total of 49 genes were implicated in multiple diseases: schizophrenia (28), mental retardation (6), bipolar disorder (13) Alzheimer's disease (6) and others (29) (Table III).We next analyzed the pair-wise correlation between functional categories (Tables I and II) and disease type. The ‘receptors/channels/transporters' category and ionotropic glutamate receptors were significantly correlated with schizophrenia (P=0.0024 and P<10e−6, respectively). Out of 28 schizophrenia-implicated proteins, 20 were mapped into the network model (orange in Figure 5B). Of those 20 proteins, 70% fell into Cla, which was significantly enriched in schizophrenia-related proteins (P=0.0089). All but one of the remaining schizophrenia-related proteins were found in cluster Clb.

Psychiatric disorders and PSD-95 complexes

To explore the potential medical importance of the 118 core PSD-95 interactome we asked (i) which diseases are these proteins involved with, and (ii) is there any relationship between the network clusters and disease types. For each of the proteins in the 118 core set, we manually curated information on their disease involvement from the literature. A total of 49 genes were implicated in multiple diseases: schizophrenia (28), mental retardation (6), bipolar disorder (13) Alzheimer's disease (6) and others (29) (Table III).

We next analyzed the pair-wise correlation between functional categories (Tables I and II) and disease type. The ‘receptors/channels/transporters' category and ionotropic glutamate receptors were significantly correlated with schizophrenia (P=0.0024 and P<10e−6, respectively). Out of 28 schizophrenia-implicated proteins, 20 were mapped into the network model (orange in Figure 5B). Of those 20 proteins, 70% fell into Cla, which was significantly enriched in schizophrenia-related proteins (P=0.0089). All but one of the remaining schizophrenia-related proteins were found in cluster Clb.

DiscussionHere we report the first isolation of multiprotein complexes from mice using a knockin of a TAP tag fused with the endogenous protein. This allowed expression of the tagged protein to be controlled by its endogenous regulatory elements. A stable and specific complex of 118 proteins associated with PSD-95, containing a range of important synaptic receptors, channels and signaling molecules, including new interactors was isolated.Previous use of TAP tagging in mammalian cells and tissues was limited to expression of exogenous tagged cDNAs (Knuesel et al, 2003; Bouwmeester et al, 2004; Brajenovic et al, 2004; Drakas et al, 2005; Wang et al, 2005, 2006; Angrand et al, 2006; Burckstummer et al, 2006), which do not recapitulate the natural expression of the protein. A TAP insertion using homologous recombination was published by Chen et al, however, purification of complexes was not reported from mouse tissue (Chen et al, 2006b). In addition to the advantage of recapitulating the natural expression of the protein, and thereby limiting artefactual interactions, the targeting of the endogenous gene allows the breeding of the mice to homozygosity. This permits testing the possibility that the insertion created a mutation. We found no evidence of a mutant phenotype as neither the level, tissue expression pattern, subcellular localization or synaptic physiology of PSD-95 was found in homozygous mice.The two consecutive steps of purification in the TAP protocol offer advantages over single-step methods such as immunoprecipitation, which is the most commonly used approach. Immunoprecipitation is limited by (i) availability of suitable antibodies and their cross reaction with other proteins, (ii) the possibility that the antibody–protein interaction might be affected by either post-translational modifications or by the binding with other proteins, (iii) the antibody might disrupt interacting partners, (iv) the harsh conditions for the complex elution might result in protein degradation. In addition to overcoming these limitations, the TAP procedure offers an efficient method for isolation of native complexes. Also, we observed that the two-step procedure unmasked core interacting proteins that were not detected by mass spectrometry in the single-step purification: Ten known PSD-95 interactors, Begain, Cit, Grik2, Grik5, Grin2c, Kcna4, Lrp1, Nlgn2, Nlgn3 and Shank1, were present only after the tandem purification. Furthermore, we found 21 new proteins in the PSD-95 core complexes that were not reported in earlier PSP proteomic analysis (Collins et al, 2006 [19]Dosemeci et al, 2007) (Supplementary Table 1), again suggesting that the targeted TAP-tagging strategy produces greater depth and quality of interactors.The fact that PSD-95-associated complexes contain ionotropic glutamate receptors of the NMDA, AMPA and kainate subtypes as well as major K+ channels is of considerable technical and biological significance. These proteins are the major postsynaptic constituents responsible for synaptic transmission and shaping the postsynaptic electrophysiological response to presynaptic input. We also believe that this is the first method that allows the robust co-purification of these proteins and that the PSD-95TAP mice will be a valuable tool for studying the postsynaptic terminal in vivo. These applications will extend to physiological and behavioral studies of many regions of the brain and disease models.Although PSD-95 is a known direct binding partner of NMDA receptors, there is conflicting data about physical interactions between PSD-95 and AMPA receptor subunits. PSD-95 expression affects AMPA receptor-mediated excitatory synaptic transmission (Migaud et al, 1998; Beique et al, 2006; Carlisle et al, 2008) and is thought to involve indirect interactions through stargazin, SAP-97, Adam22, Lgi1 and Nsf (Leonard et al, 1998; Osten et al, 1998; Fukata et al, 2006). As we show using reciprocal co-precipitation, the interaction of PSD-95 with N-ethylmaleimide sensitive fusion protein (NSF), a cytosolic ATPase, was required for intracellular membrane fusion, and this reinforces the idea of PSD-95 involvement in synaptic vesicle trafficking and AMPA surface-expression modulation (Luthi et al, 1999; Noel et al, 1999). Other proteins involved in the trafficking and clustering of AMPA receptor are Arc/Arg3.1 (Chowdhury et al, 2006; Shepherd et al, 2006) and Rac1 (Wiens et al, 2005), and these were found within the complexes. The isolation of multiple AMPA-receptor modulators in the PSD-95 complexes underlines the importance of this complex in mediating synaptic plasticity.We annotated the disease involvement of the proteins in the PSD-95 complexes as a step toward using this proteomic data to drive human genetic studies. We identified 49 of the proteins as involved with human mental disorders, of which there was a high representation of cognitive disorders. Nineteen genes involved in schizophrenia were significantly associated with the clusters Cla and Clb that contain all the glutamate receptors and MAGUK/Dlg proteins. Mapping the primary interactors of these schizophrenia proteins recruited many other proteins found in the other modules of the network. This suggests that the overall network and its various clusters might play a role in schizophrenia, and not simply the glutamate receptors, as was generally considered in the ‘glutamate hypothesis' of schizophrenia (Greene, 2001; Coyle, 2006; Lisman et al, 2008). Proteomic studies are likely to be useful for driving high-throughput sequencing in human diseases and aid in medical systems biology.

Here we report the first isolation of multiprotein complexes from mice using a knockin of a TAP tag fused with the endogenous protein. This allowed expression of the tagged protein to be controlled by its endogenous regulatory elements. A stable and specific complex of 118 proteins associated with PSD-95, containing a range of important synaptic receptors, channels and signaling molecules, including new interactors was isolated.

Previous use of TAP tagging in mammalian cells and tissues was limited to expression of exogenous tagged cDNAs (Knuesel et al, 2003; Bouwmeester et al, 2004; Brajenovic et al, 2004; Drakas et al, 2005; Wang et al, 2005, 2006; Angrand et al, 2006; Burckstummer et al, 2006), which do not recapitulate the natural expression of the protein. A TAP insertion using homologous recombination was published by Chen et al, however, purification of complexes was not reported from mouse tissue (Chen et al, 2006b). In addition to the advantage of recapitulating the natural expression of the protein, and thereby limiting artefactual interactions, the targeting of the endogenous gene allows the breeding of the mice to homozygosity. This permits testing the possibility that the insertion created a mutation. We found no evidence of a mutant phenotype as neither the level, tissue expression pattern, subcellular localization or synaptic physiology of PSD-95 was found in homozygous mice.

The two consecutive steps of purification in the TAP protocol offer advantages over single-step methods such as immunoprecipitation, which is the most commonly used approach. Immunoprecipitation is limited by (i) availability of suitable antibodies and their cross reaction with other proteins, (ii) the possibility that the antibody–protein interaction might be affected by either post-translational modifications or by the binding with other proteins, (iii) the antibody might disrupt interacting partners, (iv) the harsh conditions for the complex elution might result in protein degradation. In addition to overcoming these limitations, the TAP procedure offers an efficient method for isolation of native complexes. Also, we observed that the two-step procedure unmasked core interacting proteins that were not detected by mass spectrometry in the single-step purification: Ten known PSD-95 interactors, Begain, Cit, Grik2, Grik5, Grin2c, Kcna4, Lrp1, Nlgn2, Nlgn3 and Shank1, were present only after the tandem purification. Furthermore, we found 21 new proteins in the PSD-95 core complexes that were not reported in earlier PSP proteomic analysis (Collins et al, 2006 [19]Dosemeci et al, 2007) (Supplementary Table 1), again suggesting that the targeted TAP-tagging strategy produces greater depth and quality of interactors.

The fact that PSD-95-associated complexes contain ionotropic glutamate receptors of the NMDA, AMPA and kainate subtypes as well as major K+ channels is of considerable technical and biological significance. These proteins are the major postsynaptic constituents responsible for synaptic transmission and shaping the postsynaptic electrophysiological response to presynaptic input. We also believe that this is the first method that allows the robust co-purification of these proteins and that the PSD-95TAP mice will be a valuable tool for studying the postsynaptic terminal in vivo. These applications will extend to physiological and behavioral studies of many regions of the brain and disease models.

Although PSD-95 is a known direct binding partner of NMDA receptors, there is conflicting data about physical interactions between PSD-95 and AMPA receptor subunits. PSD-95 expression affects AMPA receptor-mediated excitatory synaptic transmission (Migaud et al, 1998; Beique et al, 2006; Carlisle et al, 2008) and is thought to involve indirect interactions through stargazin, SAP-97, Adam22, Lgi1 and Nsf (Leonard et al, 1998; Osten et al, 1998; Fukata et al, 2006). As we show using reciprocal co-precipitation, the interaction of PSD-95 with N-ethylmaleimide sensitive fusion protein (NSF), a cytosolic ATPase, was required for intracellular membrane fusion, and this reinforces the idea of PSD-95 involvement in synaptic vesicle trafficking and AMPA surface-expression modulation (Luthi et al, 1999; Noel et al, 1999). Other proteins involved in the trafficking and clustering of AMPA receptor are Arc/Arg3.1 (Chowdhury et al, 2006; Shepherd et al, 2006) and Rac1 (Wiens et al, 2005), and these were found within the complexes. The isolation of multiple AMPA-receptor modulators in the PSD-95 complexes underlines the importance of this complex in mediating synaptic plasticity.

We annotated the disease involvement of the proteins in the PSD-95 complexes as a step toward using this proteomic data to drive human genetic studies. We identified 49 of the proteins as involved with human mental disorders, of which there was a high representation of cognitive disorders. Nineteen genes involved in schizophrenia were significantly associated with the clusters Cla and Clb that contain all the glutamate receptors and MAGUK/Dlg proteins. Mapping the primary interactors of these schizophrenia proteins recruited many other proteins found in the other modules of the network. This suggests that the overall network and its various clusters might play a role in schizophrenia, and not simply the glutamate receptors, as was generally considered in the ‘glutamate hypothesis' of schizophrenia (Greene, 2001; Coyle, 2006; Lisman et al, 2008). Proteomic studies are likely to be useful for driving high-throughput sequencing in human diseases and aid in medical systems biology.

Materials and methodsVector generation and gene targetingThe TAP tag was constructed by assembling two PCR fragments containing histidine affinity tag (HAT), TEV protease and FLAG sequences. The HAT tag was amplified by PCR (PCR1) using 1 ng of the pHAT20 vector (Clontech) as a template with the forward XbaIHATF1 and reverse HATR1 oligos as primers. The 5′-end tail of the forward primer had an XbaI restriction site and the reverse primer had a tail containing the six-amino-acid linker in the 5′-end and the TEV protease sequences. The FLAG tag was amplified by PCR (PCR2) with the forward FLAGF1 and reverse BclIFLAGR1 oligos using as a template 1 ng of the C-terminal p3xFLAG–CMV™14 vector (Sigma). The 5′-end of the forward primer contains the TEV sequence and the six-amino-acid linker sequence, whereas the reverse primer contains a BclI restriction site. A third PCR was then carried out with XbaIHATF1 and BclIFLAGR1 oligos and a mix of PCR1 and PCR2 products as a template. The 3′-end of the PCR1 and the 5′-end of the PCR2 shared the six-amino-acid linker and the TEV sequences, allowing both the fragments to anneal during the third PCR. This fragment (the TAP tag sequence) was cloned into pneoflox vector between the XbaI and BclI restriction sites (TAP tag pneoflox vector). Two homology arms of the genomic PSD-95 sequence were amplified with forward Psd95HAXhoIF and reverse Psd95HAXbaIR primers and forward PSD95HAAcc65IF and reverse Psd95HABglIIR primers using the BAC bMQ239c12 (Adams et al, 2005) as a template. Both homology arms were cloned into the TAP tag pneoflox vector, leaving in between the TAP tag sequence, 2loxP sites, PGK and EM7 promoters, the G418r gene and a SV40 polyadenylation site. The cassette flanked by two homology arms was removed and transformed into EL350 E. coli cells containing a pTargeter vector with the genomic PSD-95 sequence cr11 69851809 to cr11 69861137 (Ensembl release 47). The cassette was inserted into the pTargeter vector by recombination (Knuesel et al, 2003). The final vector containing a 5′-end homolog PSD-95 sequence of 6384 bp and a 3′-end homology arm of 2946 bp (ENSMUSG00000020886) was linearized with PvuI enzyme and electroporated into E14 ES cells. Sixteen neomycin-resistant colonies from 252 were cloned, expanded and frozen. Genomic DNA was extracted from all of them and PCR was carried out using pneoF3 and Psd95R3 to identify TAP-tagged PSD-95 homologous recombinants. One of the ES-cells-positives clones was microinjected into C57BL/6 blastocysts and this generated nine germline chimeras containing 30–70% of targeted cells. These chimeras were crossed to an MF1 genetic background. Tail DNA from the litters was extracted and analyzed by PCR with a 5′ Psd95F5 primer and two 3′ pneoR4 and Psd95R6 primers to distinguish the PSD-95 TAP (+/−) and wild-type alleles (+/+).Tandem affinity purificationFor each independent purification, two forebrains were homogenized in DOC buffer (50 mM Tris pH 9.0, 1% sodium deoxycholate, 50 mM NaF, 20 μM ZnCl, 1 mM Na3VO4, 2 mM Pefabloc SC (Roche) and 1 tablet/10 ml protease inhibitor cocktail tablets (Roche)) and clarified as described earlier (Husi and Grant, 2001). A total of 25 mg of protein was incubated Dynal beads coupled with FLAG antibody for 2 h at 4°C. The resin was washed with three cycles of 15 resin volumes of DOC buffer and twice with TEV-protease cleavage buffer (Invitrogen). The tagged protein was cut from the beads by addition of TEV protease and the protein eluate was collected. The eluate was dialyzed against 2 L of dialysis buffer (50 mM sodium phosphate pH 8.0, 50 mM NaCl) at 4°C with constant agitation. After dialysis the supernatant was collected and added to Ni2+–NTA–agarose resin (Qiagen) pre-washed thrice with the dialysis buffer. The coupling was carried out for 40 min at 4°C with constant agitation in batch, then collected with supernatant and packed into 5 ml plastic columns (Clontech). After sedimentation, the supernatant was collected by gravity flow and the resin was washed thrice with wash buffer containing 0.1% sodium deoxycholate and 1 mM of imidazole. The elution was carried out with 750 μl of elution buffer and fractions were recovered.All imidazole eluted fractions from the tandem purification that contained PSD-95 were pooled, concentrated in a Vivaspin concentrator (Vivascience, GE), reduced with DTT, alkylated with iodoacetamide and separated by one-dimensional SDS-electrophoresis 4–12% (NUPAGE, Invitrogen, CA). The gel was fixed and stained with colloidal Coomassie and entire gel lanes corresponding to the single-step and tandem purifications from PSD-95TAP/TAP, and wt forebrains were cut into slices and each slice was destained and digested overnight with trypsin (Roche, Trypsin modified, sequencing grade). A solution digest was carried out on the same quantity of starting material as the gel analyzed PSD-95TAP−TAP and wt purifications. Solution digests were carried out using sequencing grade, modified trypsin (Promega) for 4.5 h at 37°C.LC-MS/MS analysisPeptides extracted from gel slices were separated on a 40 min RP gradient and solution digests were separated on a 120-min gradient using a PepMap C18 column (75-μm inner diameter × 15 cm; LC Packings). LC-MS/MS analysis was performed on an LTQ-FT (Thermo) mass spectrometer in which the top five most intense ions in a given chromatographic window were subjected to MS/MS sequencing. A total of 102 LC-MS/MS analyses were performed and 59 885 MS/MS spectra were acquired. All data were processed using BioWorks V3.2 (Thermo) and searched using Mascot V2.1 (Matrix science) against mouse IPI sequence database (June, 2007). False discovery rates determined by reverse database searches and empirical analyses of the distributions of mass deviation and Mascot Ion Scores were used to establish score and mass accuracy filters. Only proteins with two or more approved unique peptides were accepted for gel-analysed samples, whereas proteins with only one approved peptide found in a solution digest experiment were required to be present in another replicate analysis of the same solution digest with a different unique peptide. Application of these filters to the PSD-95TAP/TAP datasets resulted in a <1% false discovery rate as assessed by reverse database searching. In general, proteins that were identified in both PSD-95TAP/TAP and control experiments were not accepted. In cases, however, when a protein was found specifically in the tandem TAP-PSD-95 purification but not in the control tandem purification, but was present in both the one-step TAP-PSD-95 purification and in the control single-step purification, then the ratio of approved peptides in the one-step TAP-PSD-95 purification/single-step control purification had to be >3 in order for the protein to be accepted in the tandem purification. Protein hits from all datasets were blast-clustered using a threshold of 95% sequence homology over at least 50% of sequence length.The data for this manuscript are open access according to the Science Commons CC0 license and can be downloaded from the Tranche network (http://tranche.proteomecommons.org) using the following hash: (J9KSi8FHLDgHFyl2zz1LRq332aRrhVZl/cgPIJAO5WG8tzhAhlrwxvHJOjnre8hIAKLF RTY11dRkXdIEtnkrlqUbg7gAAAAAAAA8fw==). In addition, the raw data are available in Peptide Atlas (http://peptideatlas.org/repository). The protein interaction data have been submitted to the IMEx consortium through the IntAct molecular interaction database (http://www.ebi.ac.uk/intact) and assigned the identifier IM-11694.emPAI calculationEmPAI values were calculated for each protein as described (Ishihama et al, 2005) using the following formula: emPAI=10PAI-1, where PAI=number of observed peptides/number of observable peptides. The ratio of protein emPAI vales for proteins in the tandem purification/single-step purification was calculated using emPAI values that were normalized to the total protein emPAI in each set and then normalized to the emPAI value of the bait protein (PSD-95). The tandem-enriched set contained proteins specifically identified in the tandem plus proteins identified in both tandem and single-step purifications with an emPAI ratio (tandem/single step) of >0.5, 126 proteins, and the tandem-depleted set contained proteins specifically identified in the single step plus proteins identified in both tandem and single with an emPAI ratio (tandem/single step) of <0.5, 175 proteins.Network building and analysisProteins appearing in three or more of the replicate experiments were used for the model. Out of these 118, five had gene ambiguity (Cpne4, Tuba1a, Tubb2b, Uba52, and Vamp2) and were removed. A final set of 113 proteins was used. Protein interaction data were sourced from the UniHi database (November 2008) (Chaurasia et al, 2007), with existing curated interactions in the NRC/MASC complex described by Pocklington et al, 2006 and the additional interactions obtained from literature. All interactions were manually re-curated. Interactions sourced from UniHi were traced back to their original database entries and the supporting reference was curated. No high-throughput yeast 2 hybrid data were used unless confirmed by other techniques.Clustering was performed with the Newman & Girvan algorithm (Newman and Girvan, 2004), using edge betweenness. The modularity coefficient (Q) for the clustering configuration used was 0.37. We found clustering configurations with higher values (up to 0.42); however, these configurations did not reflect the functional organization of the network as well as the one used. With the latter in mind, and also on the basis of the observation that 0.37 was over the average of examined configurations, we decided to use that. A shortest path is defined as a path between two nodes such that the number of its constituent edges is minimized. The average shortest path of a node was calculated as the average of the shortest paths between the node and all the other nodes in the network. Graphical representation of the network was produced by Cytoscape.Annotation overlap significance was performed according to Pocklington et al (2006). The significance of the overlap between a pair of annotations (e.g. ‘Glutamate Receptors' and ‘Schizophrenia') was evaluated by calculating its probability of occurrence under a random distribution. If within a set of N proteins, na and nb possess the annotations a and b, respectively, and both annotations are distributed randomly in the set the probability of nab proteins possessing both annotations is given by Given the actual number of proteins possessing the both annotations, mab, we can estimate the significance by calculating the probability P(mab) of an overlap as or less likely under the random distribution. Additional methodsDetailed description of additional methods is available in the Supplementary information. We described the electrophysiology analysis, immunoprecipitation, immunoblotting, immunocytochemistry and immunohistochemistry. Primers used in this work are summarized in Supplementary Table 7.

Materials and methods

Vector generation and gene targetingThe TAP tag was constructed by assembling two PCR fragments containing histidine affinity tag (HAT), TEV protease and FLAG sequences. The HAT tag was amplified by PCR (PCR1) using 1 ng of the pHAT20 vector (Clontech) as a template with the forward XbaIHATF1 and reverse HATR1 oligos as primers. The 5′-end tail of the forward primer had an XbaI restriction site and the reverse primer had a tail containing the six-amino-acid linker in the 5′-end and the TEV protease sequences. The FLAG tag was amplified by PCR (PCR2) with the forward FLAGF1 and reverse BclIFLAGR1 oligos using as a template 1 ng of the C-terminal p3xFLAG–CMV™14 vector (Sigma). The 5′-end of the forward primer contains the TEV sequence and the six-amino-acid linker sequence, whereas the reverse primer contains a BclI restriction site. A third PCR was then carried out with XbaIHATF1 and BclIFLAGR1 oligos and a mix of PCR1 and PCR2 products as a template. The 3′-end of the PCR1 and the 5′-end of the PCR2 shared the six-amino-acid linker and the TEV sequences, allowing both the fragments to anneal during the third PCR. This fragment (the TAP tag sequence) was cloned into pneoflox vector between the XbaI and BclI restriction sites (TAP tag pneoflox vector). Two homology arms of the genomic PSD-95 sequence were amplified with forward Psd95HAXhoIF and reverse Psd95HAXbaIR primers and forward PSD95HAAcc65IF and reverse Psd95HABglIIR primers using the BAC bMQ239c12 (Adams et al, 2005) as a template. Both homology arms were cloned into the TAP tag pneoflox vector, leaving in between the TAP tag sequence, 2loxP sites, PGK and EM7 promoters, the G418r gene and a SV40 polyadenylation site. The cassette flanked by two homology arms was removed and transformed into EL350 E. coli cells containing a pTargeter vector with the genomic PSD-95 sequence cr11 69851809 to cr11 69861137 (Ensembl release 47). The cassette was inserted into the pTargeter vector by recombination (Knuesel et al, 2003). The final vector containing a 5′-end homolog PSD-95 sequence of 6384 bp and a 3′-end homology arm of 2946 bp (ENSMUSG00000020886) was linearized with PvuI enzyme and electroporated into E14 ES cells. Sixteen neomycin-resistant colonies from 252 were cloned, expanded and frozen. Genomic DNA was extracted from all of them and PCR was carried out using pneoF3 and Psd95R3 to identify TAP-tagged PSD-95 homologous recombinants. One of the ES-cells-positives clones was microinjected into C57BL/6 blastocysts and this generated nine germline chimeras containing 30–70% of targeted cells. These chimeras were crossed to an MF1 genetic background. Tail DNA from the litters was extracted and analyzed by PCR with a 5′ Psd95F5 primer and two 3′ pneoR4 and Psd95R6 primers to distinguish the PSD-95 TAP (+/−) and wild-type alleles (+/+).

Vector generation and gene targeting

The TAP tag was constructed by assembling two PCR fragments containing histidine affinity tag (HAT), TEV protease and FLAG sequences. The HAT tag was amplified by PCR (PCR1) using 1 ng of the pHAT20 vector (Clontech) as a template with the forward XbaIHATF1 and reverse HATR1 oligos as primers. The 5′-end tail of the forward primer had an XbaI restriction site and the reverse primer had a tail containing the six-amino-acid linker in the 5′-end and the TEV protease sequences. The FLAG tag was amplified by PCR (PCR2) with the forward FLAGF1 and reverse BclIFLAGR1 oligos using as a template 1 ng of the C-terminal p3xFLAG–CMV™14 vector (Sigma). The 5′-end of the forward primer contains the TEV sequence and the six-amino-acid linker sequence, whereas the reverse primer contains a BclI restriction site. A third PCR was then carried out with XbaIHATF1 and BclIFLAGR1 oligos and a mix of PCR1 and PCR2 products as a template. The 3′-end of the PCR1 and the 5′-end of the PCR2 shared the six-amino-acid linker and the TEV sequences, allowing both the fragments to anneal during the third PCR. This fragment (the TAP tag sequence) was cloned into pneoflox vector between the XbaI and BclI restriction sites (TAP tag pneoflox vector). Two homology arms of the genomic PSD-95 sequence were amplified with forward Psd95HAXhoIF and reverse Psd95HAXbaIR primers and forward PSD95HAAcc65IF and reverse Psd95HABglIIR primers using the BAC bMQ239c12 (Adams et al, 2005) as a template. Both homology arms were cloned into the TAP tag pneoflox vector, leaving in between the TAP tag sequence, 2loxP sites, PGK and EM7 promoters, the G418r gene and a SV40 polyadenylation site. The cassette flanked by two homology arms was removed and transformed into EL350 E. coli cells containing a pTargeter vector with the genomic PSD-95 sequence cr11 69851809 to cr11 69861137 (Ensembl release 47). The cassette was inserted into the pTargeter vector by recombination (Knuesel et al, 2003). The final vector containing a 5′-end homolog PSD-95 sequence of 6384 bp and a 3′-end homology arm of 2946 bp (ENSMUSG00000020886) was linearized with PvuI enzyme and electroporated into E14 ES cells. Sixteen neomycin-resistant colonies from 252 were cloned, expanded and frozen. Genomic DNA was extracted from all of them and PCR was carried out using pneoF3 and Psd95R3 to identify TAP-tagged PSD-95 homologous recombinants. One of the ES-cells-positives clones was microinjected into C57BL/6 blastocysts and this generated nine germline chimeras containing 30–70% of targeted cells. These chimeras were crossed to an MF1 genetic background. Tail DNA from the litters was extracted and analyzed by PCR with a 5′ Psd95F5 primer and two 3′ pneoR4 and Psd95R6 primers to distinguish the PSD-95 TAP (+/−) and wild-type alleles (+/+).

Tandem affinity purificationFor each independent purification, two forebrains were homogenized in DOC buffer (50 mM Tris pH 9.0, 1% sodium deoxycholate, 50 mM NaF, 20 μM ZnCl, 1 mM Na3VO4, 2 mM Pefabloc SC (Roche) and 1 tablet/10 ml protease inhibitor cocktail tablets (Roche)) and clarified as described earlier (Husi and Grant, 2001). A total of 25 mg of protein was incubated Dynal beads coupled with FLAG antibody for 2 h at 4°C. The resin was washed with three cycles of 15 resin volumes of DOC buffer and twice with TEV-protease cleavage buffer (Invitrogen). The tagged protein was cut from the beads by addition of TEV protease and the protein eluate was collected. The eluate was dialyzed against 2 L of dialysis buffer (50 mM sodium phosphate pH 8.0, 50 mM NaCl) at 4°C with constant agitation. After dialysis the supernatant was collected and added to Ni2+–NTA–agarose resin (Qiagen) pre-washed thrice with the dialysis buffer. The coupling was carried out for 40 min at 4°C with constant agitation in batch, then collected with supernatant and packed into 5 ml plastic columns (Clontech). After sedimentation, the supernatant was collected by gravity flow and the resin was washed thrice with wash buffer containing 0.1% sodium deoxycholate and 1 mM of imidazole. The elution was carried out with 750 μl of elution buffer and fractions were recovered.All imidazole eluted fractions from the tandem purification that contained PSD-95 were pooled, concentrated in a Vivaspin concentrator (Vivascience, GE), reduced with DTT, alkylated with iodoacetamide and separated by one-dimensional SDS-electrophoresis 4–12% (NUPAGE, Invitrogen, CA). The gel was fixed and stained with colloidal Coomassie and entire gel lanes corresponding to the single-step and tandem purifications from PSD-95TAP/TAP, and wt forebrains were cut into slices and each slice was destained and digested overnight with trypsin (Roche, Trypsin modified, sequencing grade). A solution digest was carried out on the same quantity of starting material as the gel analyzed PSD-95TAP−TAP and wt purifications. Solution digests were carried out using sequencing grade, modified trypsin (Promega) for 4.5 h at 37°C.

Tandem affinity purification

For each independent purification, two forebrains were homogenized in DOC buffer (50 mM Tris pH 9.0, 1% sodium deoxycholate, 50 mM NaF, 20 μM ZnCl, 1 mM Na3VO4, 2 mM Pefabloc SC (Roche) and 1 tablet/10 ml protease inhibitor cocktail tablets (Roche)) and clarified as described earlier (Husi and Grant, 2001). A total of 25 mg of protein was incubated Dynal beads coupled with FLAG antibody for 2 h at 4°C. The resin was washed with three cycles of 15 resin volumes of DOC buffer and twice with TEV-protease cleavage buffer (Invitrogen). The tagged protein was cut from the beads by addition of TEV protease and the protein eluate was collected. The eluate was dialyzed against 2 L of dialysis buffer (50 mM sodium phosphate pH 8.0, 50 mM NaCl) at 4°C with constant agitation. After dialysis the supernatant was collected and added to Ni2+–NTA–agarose resin (Qiagen) pre-washed thrice with the dialysis buffer. The coupling was carried out for 40 min at 4°C with constant agitation in batch, then collected with supernatant and packed into 5 ml plastic columns (Clontech). After sedimentation, the supernatant was collected by gravity flow and the resin was washed thrice with wash buffer containing 0.1% sodium deoxycholate and 1 mM of imidazole. The elution was carried out with 750 μl of elution buffer and fractions were recovered.

All imidazole eluted fractions from the tandem purification that contained PSD-95 were pooled, concentrated in a Vivaspin concentrator (Vivascience, GE), reduced with DTT, alkylated with iodoacetamide and separated by one-dimensional SDS-electrophoresis 4–12% (NUPAGE, Invitrogen, CA). The gel was fixed and stained with colloidal Coomassie and entire gel lanes corresponding to the single-step and tandem purifications from PSD-95TAP/TAP, and wt forebrains were cut into slices and each slice was destained and digested overnight with trypsin (Roche, Trypsin modified, sequencing grade). A solution digest was carried out on the same quantity of starting material as the gel analyzed PSD-95TAP−TAP and wt purifications. Solution digests were carried out using sequencing grade, modified trypsin (Promega) for 4.5 h at 37°C.

LC-MS/MS analysisPeptides extracted from gel slices were separated on a 40 min RP gradient and solution digests were separated on a 120-min gradient using a PepMap C18 column (75-μm inner diameter × 15 cm; LC Packings). LC-MS/MS analysis was performed on an LTQ-FT (Thermo) mass spectrometer in which the top five most intense ions in a given chromatographic window were subjected to MS/MS sequencing. A total of 102 LC-MS/MS analyses were performed and 59 885 MS/MS spectra were acquired. All data were processed using BioWorks V3.2 (Thermo) and searched using Mascot V2.1 (Matrix science) against mouse IPI sequence database (June, 2007). False discovery rates determined by reverse database searches and empirical analyses of the distributions of mass deviation and Mascot Ion Scores were used to establish score and mass accuracy filters. Only proteins with two or more approved unique peptides were accepted for gel-analysed samples, whereas proteins with only one approved peptide found in a solution digest experiment were required to be present in another replicate analysis of the same solution digest with a different unique peptide. Application of these filters to the PSD-95TAP/TAP datasets resulted in a <1% false discovery rate as assessed by reverse database searching. In general, proteins that were identified in both PSD-95TAP/TAP and control experiments were not accepted. In cases, however, when a protein was found specifically in the tandem TAP-PSD-95 purification but not in the control tandem purification, but was present in both the one-step TAP-PSD-95 purification and in the control single-step purification, then the ratio of approved peptides in the one-step TAP-PSD-95 purification/single-step control purification had to be >3 in order for the protein to be accepted in the tandem purification. Protein hits from all datasets were blast-clustered using a threshold of 95% sequence homology over at least 50% of sequence length.The data for this manuscript are open access according to the Science Commons CC0 license and can be downloaded from the Tranche network (http://tranche.proteomecommons.org) using the following hash: (J9KSi8FHLDgHFyl2zz1LRq332aRrhVZl/cgPIJAO5WG8tzhAhlrwxvHJOjnre8hIAKLF RTY11dRkXdIEtnkrlqUbg7gAAAAAAAA8fw==). In addition, the raw data are available in Peptide Atlas (http://peptideatlas.org/repository). The protein interaction data have been submitted to the IMEx consortium through the IntAct molecular interaction database (http://www.ebi.ac.uk/intact) and assigned the identifier IM-11694.

LC-MS/MS analysis

Peptides extracted from gel slices were separated on a 40 min RP gradient and solution digests were separated on a 120-min gradient using a PepMap C18 column (75-μm inner diameter × 15 cm; LC Packings). LC-MS/MS analysis was performed on an LTQ-FT (Thermo) mass spectrometer in which the top five most intense ions in a given chromatographic window were subjected to MS/MS sequencing. A total of 102 LC-MS/MS analyses were performed and 59 885 MS/MS spectra were acquired. All data were processed using BioWorks V3.2 (Thermo) and searched using Mascot V2.1 (Matrix science) against mouse IPI sequence database (June, 2007). False discovery rates determined by reverse database searches and empirical analyses of the distributions of mass deviation and Mascot Ion Scores were used to establish score and mass accuracy filters. Only proteins with two or more approved unique peptides were accepted for gel-analysed samples, whereas proteins with only one approved peptide found in a solution digest experiment were required to be present in another replicate analysis of the same solution digest with a different unique peptide. Application of these filters to the PSD-95TAP/TAP datasets resulted in a <1% false discovery rate as assessed by reverse database searching. In general, proteins that were identified in both PSD-95TAP/TAP and control experiments were not accepted. In cases, however, when a protein was found specifically in the tandem TAP-PSD-95 purification but not in the control tandem purification, but was present in both the one-step TAP-PSD-95 purification and in the control single-step purification, then the ratio of approved peptides in the one-step TAP-PSD-95 purification/single-step control purification had to be >3 in order for the protein to be accepted in the tandem purification. Protein hits from all datasets were blast-clustered using a threshold of 95% sequence homology over at least 50% of sequence length.

The data for this manuscript are open access according to the Science Commons CC0 license and can be downloaded from the Tranche network (http://tranche.proteomecommons.org) using the following hash: (J9KSi8FHLDgHFyl2zz1LRq332aRrhVZl/cgPIJAO5WG8tzhAhlrwxvHJOjnre8hIAKLF RTY11dRkXdIEtnkrlqUbg7gAAAAAAAA8fw==). In addition, the raw data are available in Peptide Atlas (http://peptideatlas.org/repository). The protein interaction data have been submitted to the IMEx consortium through the IntAct molecular interaction database (http://www.ebi.ac.uk/intact) and assigned the identifier IM-11694.

emPAI calculationEmPAI values were calculated for each protein as described (Ishihama et al, 2005) using the following formula: emPAI=10PAI-1, where PAI=number of observed peptides/number of observable peptides. The ratio of protein emPAI vales for proteins in the tandem purification/single-step purification was calculated using emPAI values that were normalized to the total protein emPAI in each set and then normalized to the emPAI value of the bait protein (PSD-95). The tandem-enriched set contained proteins specifically identified in the tandem plus proteins identified in both tandem and single-step purifications with an emPAI ratio (tandem/single step) of >0.5, 126 proteins, and the tandem-depleted set contained proteins specifically identified in the single step plus proteins identified in both tandem and single with an emPAI ratio (tandem/single step) of <0.5, 175 proteins.

emPAI calculation

EmPAI values were calculated for each protein as described (Ishihama et al, 2005) using the following formula: emPAI=10PAI-1, where PAI=number of observed peptides/number of observable peptides. The ratio of protein emPAI vales for proteins in the tandem purification/single-step purification was calculated using emPAI values that were normalized to the total protein emPAI in each set and then normalized to the emPAI value of the bait protein (PSD-95). The tandem-enriched set contained proteins specifically identified in the tandem plus proteins identified in both tandem and single-step purifications with an emPAI ratio (tandem/single step) of >0.5, 126 proteins, and the tandem-depleted set contained proteins specifically identified in the single step plus proteins identified in both tandem and single with an emPAI ratio (tandem/single step) of <0.5, 175 proteins.

Network building and analysisProteins appearing in three or more of the replicate experiments were used for the model. Out of these 118, five had gene ambiguity (Cpne4, Tuba1a, Tubb2b, Uba52, and Vamp2) and were removed. A final set of 113 proteins was used. Protein interaction data were sourced from the UniHi database (November 2008) (Chaurasia et al, 2007), with existing curated interactions in the NRC/MASC complex described by Pocklington et al, 2006 and the additional interactions obtained from literature. All interactions were manually re-curated. Interactions sourced from UniHi were traced back to their original database entries and the supporting reference was curated. No high-throughput yeast 2 hybrid data were used unless confirmed by other techniques.Clustering was performed with the Newman & Girvan algorithm (Newman and Girvan, 2004), using edge betweenness. The modularity coefficient (Q) for the clustering configuration used was 0.37. We found clustering configurations with higher values (up to 0.42); however, these configurations did not reflect the functional organization of the network as well as the one used. With the latter in mind, and also on the basis of the observation that 0.37 was over the average of examined configurations, we decided to use that. A shortest path is defined as a path between two nodes such that the number of its constituent edges is minimized. The average shortest path of a node was calculated as the average of the shortest paths between the node and all the other nodes in the network. Graphical representation of the network was produced by Cytoscape.Annotation overlap significance was performed according to Pocklington et al (2006). The significance of the overlap between a pair of annotations (e.g. ‘Glutamate Receptors' and ‘Schizophrenia') was evaluated by calculating its probability of occurrence under a random distribution. If within a set of N proteins, na and nb possess the annotations a and b, respectively, and both annotations are distributed randomly in the set the probability of nab proteins possessing both annotations is given by Given the actual number of proteins possessing the both annotations, mab, we can estimate the significance by calculating the probability P(mab) of an overlap as or less likely under the random distribution.

Network building and analysis

Proteins appearing in three or more of the replicate experiments were used for the model. Out of these 118, five had gene ambiguity (Cpne4, Tuba1a, Tubb2b, Uba52, and Vamp2) and were removed. A final set of 113 proteins was used. Protein interaction data were sourced from the UniHi database (November 2008) (Chaurasia et al, 2007), with existing curated interactions in the NRC/MASC complex described by Pocklington et al, 2006 and the additional interactions obtained from literature. All interactions were manually re-curated. Interactions sourced from UniHi were traced back to their original database entries and the supporting reference was curated. No high-throughput yeast 2 hybrid data were used unless confirmed by other techniques.

Clustering was performed with the Newman & Girvan algorithm (Newman and Girvan, 2004), using edge betweenness. The modularity coefficient (Q) for the clustering configuration used was 0.37. We found clustering configurations with higher values (up to 0.42); however, these configurations did not reflect the functional organization of the network as well as the one used. With the latter in mind, and also on the basis of the observation that 0.37 was over the average of examined configurations, we decided to use that. A shortest path is defined as a path between two nodes such that the number of its constituent edges is minimized. The average shortest path of a node was calculated as the average of the shortest paths between the node and all the other nodes in the network. Graphical representation of the network was produced by Cytoscape.

Annotation overlap significance was performed according to Pocklington et al (2006). The significance of the overlap between a pair of annotations (e.g. ‘Glutamate Receptors' and ‘Schizophrenia') was evaluated by calculating its probability of occurrence under a random distribution. If within a set of N proteins, na and nb possess the annotations a and b, respectively, and both annotations are distributed randomly in the set the probability of nab proteins possessing both annotations is given by

Given the actual number of proteins possessing the both annotations, mab, we can estimate the significance by calculating the probability P(mab) of an overlap as or less likely under the random distribution.

Additional methodsDetailed description of additional methods is available in the Supplementary information. We described the electrophysiology analysis, immunoprecipitation, immunoblotting, immunocytochemistry and immunohistochemistry. Primers used in this work are summarized in Supplementary Table 7.

Additional methods

Detailed description of additional methods is available in the Supplementary information. We described the electrophysiology analysis, immunoprecipitation, immunoblotting, immunocytochemistry and immunohistochemistry. Primers used in this work are summarized in Supplementary Table 7.

Supplementary MaterialSupplementary Materials #1Supplementary figures S1-6, Supplementary table legends for tables S1-7, Materials & MethodsSupplementary Materials #2 Supplementary table S1Supplementary Materials #3 Supplementary table S2Supplementary Materials #4Supplementary table S3Supplementary Materials #5Supplementary table S4Supplementary Materials #6Supplementary table S5Supplementary Materials #7Supplementary table S6Supplementary Materials #8Supplementary table S7References for Table III

Supplementary Material

Supplementary figures S1-6, Supplementary table legends for tables S1-7, Materials & Methods

Supplementary table S1

Supplementary table S2

Supplementary table S3

Supplementary table S4

Supplementary table S5

Supplementary table S6

Supplementary table S7
