---
pmid: '18552202'
title: Arabidopsis DREB2A-interacting proteins function as RING E3 ligases and negatively
  regulate plant drought stress-responsive gene expression.
authors:
- Qin F
- Sakuma Y
- Tran LS
- Maruyama K
- Kidokoro S
- Fujita Y
- Fujita M
- Umezawa T
- Sawano Y
- Miyazono K
- Tanokura M
- Shinozaki K
- Yamaguchi-Shinozaki K
journal: Plant Cell
year: '2008'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC2483357
doi: 10.1105/tpc.107.057380
---

# Arabidopsis DREB2A-interacting proteins function as RING E3 ligases and negatively regulate plant drought stress-responsive gene expression.
**Authors:** Qin F, Sakuma Y, Tran LS, Maruyama K, Kidokoro S, Fujita Y, Fujita M, Umezawa T, Sawano Y, Miyazono K, Tanokura M, Shinozaki K, Yamaguchi-Shinozaki K
**Journal:** Plant Cell (2008)
**DOI:** [10.1105/tpc.107.057380](https://doi.org/10.1105/tpc.107.057380)
**PMC:** [PMC2483357](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2483357/)

## Abstract

1. Plant Cell. 2008 Jun;20(6):1693-707. doi: 10.1105/tpc.107.057380. Epub 2008
Jun  13.

Arabidopsis DREB2A-interacting proteins function as RING E3 ligases and 
negatively regulate plant drought stress-responsive gene expression.

Qin F(1), Sakuma Y, Tran LS, Maruyama K, Kidokoro S, Fujita Y, Fujita M, Umezawa 
T, Sawano Y, Miyazono K, Tanokura M, Shinozaki K, Yamaguchi-Shinozaki K.

Author information:
(1)Biological Resources Division, Japan International Research Center for 
Agricultural Sciences, Tsukuba, Ibaraki 305-8686, Japan.

Comment in
    Plant Cell. 2008 Jun;20(6):1424. doi: 10.1105/tpc.108.200611.

The DEHYDRATION-RESPONSIVE ELEMENT BINDING PROTEIN2A (DREB2A) transcription 
factor controls water deficit-inducible gene expression and requires 
posttranslational modification for its activation. The activation mechanism is 
not well understood; however, the stability of this protein in the nucleus was 
recently found to be important for its activation. Here, we report the isolation 
of Arabidopsis thaliana DREB2A-INTERACTING PROTEIN1 (DRIP1) and DRIP2, C3HC4 
RING domain-containing proteins that interact with the DREB2A protein in the 
nucleus. An in vitro ubiquitination assay showed that they function as E3 
ubiquitin ligases and are capable of mediating DREB2A ubiquitination. 
Overexpression of DRIP1 in Arabidopsis delayed the expression of 
DREB2A-regulated drought-responsive genes. Drought-inducible gene expression was 
slightly enhanced in the single T-DNA mutants of drip1-1 and drip2-1. By 
contrast, significantly enhanced gene expression was revealed in the drip1 drip2 
double mutant under dehydration stress. Collectively, these data imply that 
DRIP1 and DRIP2 function negatively in the response of plants to drought stress. 
Moreover, overexpression of full-length DREB2A protein was more stable in 
drip1-1 than in the wild-type background. These results suggest that DRIP1 and 
DRIP2 act as novel negative regulators in drought-responsive gene expression by 
targeting DREB2A to 26S proteasome proteolysis.

DOI: 10.1105/tpc.107.057380
PMCID: PMC2483357
PMID: 18552202 [Indexed for MEDLINE]

## Full Text

Abstract

The DEHYDRATION-RESPONSIVE ELEMENT BINDING PROTEIN2A (DREB2A) transcription factor controls water deficit–inducible gene expression and requires posttranslational modification for its activation. The activation mechanism is not well understood; however, the stability of this protein in the nucleus was recently found to be important for its activation. Here, we report the isolation of Arabidopsis thaliana DREB2A-INTERACTING PROTEIN1 (DRIP1) and DRIP2, C3HC4 RING domain–containing proteins that interact with the DREB2A protein in the nucleus. An in vitro ubiquitination assay showed that they function as E3 ubiquitin ligases and are capable of mediating DREB2A ubiquitination. Overexpression of DRIP1 in Arabidopsis delayed the expression of DREB2A-regulated drought-responsive genes. Drought-inducible gene expression was slightly enhanced in the single T-DNA mutants of drip1-1 and drip2-1 . By contrast, significantly enhanced gene expression was revealed in the drip1 drip2 double mutant under dehydration stress. Collectively, these data imply that DRIP1 and DRIP2 function negatively in the response of plants to drought stress. Moreover, overexpression of full-length DREB2A protein was more stable in drip1-1 than in the wild-type background. These results suggest that DRIP1 and DRIP2 act as novel negative regulators in drought-responsive gene expression by targeting DREB2A to 26S proteasome proteolysis.

INTRODUCTION

Plants have evolved a number of mechanisms to adapt to environmental stresses such as low temperature, drought, and salinity. Studies of stress-responsive gene expression suggest that transcriptional modulation is one of the most important steps that plants undergo to adapt to stress conditions ( Thomashow, 1999 ; Zhu, 2002 ; Chinnusamy et al., 2004 ; Bartels and Sunkar, 2005 ; Yamaguchi-Shinozaki and Shinozaki, 2006 ). Many stress-inducible genes have been characterized in Arabidopsis thaliana and serve as indicators for the perception and transmission of stress signals in plant cells. Promoter analysis has confirmed that several kinds of cis -acting elements play fundamental roles in controlling gene expression under stress conditions ( Yamaguchi-Shinozaki and Shinozaki, 2005 ). The dehydration-responsive element (DRE), having the same core motif as the C-repeat and the low-temperature-responsive element, was identified from a subset of stress-inducible gene promoters ( Baker et al., 1994 ; Yamaguchi-Shinozaki and Shinozaki, 1994 ; Jiang et al., 1996 ; Thomashow, 1999 ).

DEHYDRATION-RESPONSIVE ELEMENT BINDING PROTEIN2A (DREB2A) was isolated using a yeast one-hybrid screen for proteins that bind to the DRE sequence. Also isolated in this screen was DREB1A/CBF3, an AP2/ERF-type transcription factor that binds to the DRE sequence ( Stockinger et al., 1997 ; Liu et al., 1998 ). The function of DREB1/CBF genes ( DREB1A/CBF3 , DREB1B/CBF1 , and DREB1C/CBF2 ) has been well documented in the past decade. However, the function of DREB2A was only recently identified by generating a constitutively active form of DREB2A ( DREB2A-CA ) ( Sakuma et al., 2006a ). Overexpression of the full-length DREB2A gene did not result in any remarkable alterations in plant phenotype or the expression of downstream genes ( Liu et al., 1998 ). It was hypothesized that a posttranslational modification was required for the activation of DREB2A. Deletion of a Ser- and Thr-rich 30–amino acid region adjacent to the ERF/AP2 DNA binding domain of DREB2A transformed DREB2A into DREB2-CA. As a result, this region was named the DREB2A negative regulatory domain (DREB2A-NRD) ( Sakuma et al., 2006a ). Importantly, the overproduced DREB2A protein was not stable under normal growth conditions. By contrast, the DREB2A-CA protein could stably accumulate in plant cells ( Sakuma et al., 2006a ). Transgenic plants overexpressing DREB2A-CA were more tolerant to drought as well as high-temperature stress, and specific target genes of DREB2A were identified ( Sakuma et al., 2006a , 2006b ). Although the generation of DREB2A-CA enabled the identification of an in planta functional role for DREB2A, the mechanism controlling its activation and degradation remained to be elucidated.

Ubiquitination and proteolysis by the 26S proteasome pathway is involved in a wide rage of biological processes, as is protein phosphorylation (reviewed in Hare et al., 2003 ; Vierstra, 2003 ; Moon et al., 2004 ). The general ubiquitin pathway conjugates either single or multiple ubiquitin molecules to the target protein, thus enabling the ubiquitin-labeled protein to be recognized by the 26S proteasome and targeted for degradation. Three enzymes catalyze this reaction and ultimately covalently attach a 76–amino acid ubiquitin protein to the target protein on the Lys residue. The three enzymes are E1 (ubiquitin-activating enzyme), E2 (ubiquitin-conjugating enzyme), and E3 (ubiquitin ligase). For the E3 category, Arabidopsis is predicted to contain ∼700 F-box proteins and ∼400 RING domain–containing proteins, which may function as multiple-subunit or single-subunit E3 ligases, respectively ( Gagne et al., 2002 ; Kosarev et al., 2002 ; Kuroda et al., 2002 ; Risseeuw et al., 2003 ; Stone et al., 2005 ). The RING domain is rich in Cys, and the eight metal ligand residues coordinate zinc ions in a cross-brace structure ( Barlow et al., 1994 ; Borden and Freemont 1996 ; Borden 2000 ). Depending on whether the metal ligand at position 5 is a His or a Cys, RING domain proteins are divided into C3HC4 and C3H2C3 types ( Freemont, 1993 ; Lovering et al., 1993 ).

Some single-subunit RING E3 ligases target transcription factors for degradation and thereby regulate gene expression in response to environmental or hormone signals. COP1 (for Constitutive Photomorphogenesis1) protein is a well-studied example that functions as a negative regulator of the light response ( Deng et al., 1991 ; von Arnim and Deng, 1994 ). COP1 represses photomorphogenesis by targeting the activators of light-responsive genes for degradation, such as HY5, LAF1, and HFR1 ( Hardtke et al., 2000 ; Osterlund et al., 2000 ; Seo et al., 2003 ; Yang et al., 2005 ). SINAT5 (for SINA of Arabidopsis 5) is another single-subunit RING E3 ligase that interacts with NAC1 (for NAM/CUC-like protein1) and directs it to degradation for the attenuation of auxin signals ( Xie et al., 2002 ). AIP2 (for ABI3 Interaction Protein2) is another single-subunit RING E3 ligase that targets a B3 domain transcription factor (ABI3) for proteolysis. This targeted degradation relieves the abscisic acid effect for germination after stratification ( Zhang et al., 2005 ). HOS1 (for High Expression of Osmotically Responsive Gene1), another RING E3 ligase, was recently shown to target a MYC-like basic helix-loop-helix (ICE1) transcriptional activator to 26S proteasome degradation for the attenuation of cold stress in Arabidopsis ( Dong et al., 2006 ). Therefore, it is interesting to speculate whether a ubiquitin E3 ligase might function in plant water deficit stress response through an interaction with key transcription factors signaling in such an important biological process.

In this study, we identified two C3HC4 RING–domain containing proteins, DREB2A-INTERACTING PROTEIN1 (DRIP1) and DRIP2, that interact with the key transcription factor DREB2A. These proteins were shown to function as E3 ubiquitin ligases and mediate DREB2A degradation as a mechanism to tightly control DREB2A protein abundance. The drip1 drip2 double mutant showed increased expression of stress-responsive genes and also a delay in plant development and growth. We hypothesize that DRIP1 and DRIP2 negatively regulate the response to water stress.

Overexpression ofDREB2Ain thedrip1-1Mutant Results in a Dwarf Phenotype That Mimics theDREB2A-CAOverexpressor

Since DREB2A is expressed at very low levels under nonstressed conditions, in order to understand DRIP1 function on DREB2A protein stability under normal conditions, we transformed the 35S:GFP-DREB2A construct into the drip1-1 mutant and obtained eight independent transgenic lines. Quantitative RT-PCR analysis revealed a relatively high level of transgene expression in four lines ( Figure 7A ). Interestingly, all four lines showed growth retardation and an overall dwarfed phenotype, resembling the DREB2A-CA overexpressor phenotype ( Figure 7B ). However, when the 35S:GFP-DREB2A construct was transformed into the wild-type Columbia plants, with a comparable level of transgene expression, normal growth was observed in transgenic plants ( Figure 7B ). Furthermore, stronger green fluorescent signals were observed in the roots of the 35S:GFP-DREB2A/drip1-1 plants relative to the 35S:GFP-DREB2A/Col plants. An anti-GFP immunoblot detection of GFP or GFP fusion proteins in 35S:GFP/Col , 35S:GFP-DREB2A/Col , and 35S:GFP-DREB2A/drip1-1 plants was performed. The predicted GFP-DREB2A molecular mass is ∼64.3 kD. With protein gel blot analysis, a protein band was detected in two independent 35S:GFP-DREB2A/drip1-1 transgenic lines (B and D) in the corresponding size range. However, no band was detected in 35S:GFP/Col or two 35S:GFP-DREB2A/Col transgenic lines, G and R ( Figure 7C ). The expression of two DREB2A-regulated genes ( RD29A and RD29B ) under normal conditions was compared in two separate lines from both 35S:GFP-DREB2A/drip1-1 and 35S:GFP-DREB2A/Col with quantitative RT-PCR analysis. As expected, even under nonstressed conditions, these genes were higher in the 35S:GFP-DREB2A/drip1-1 plants than in 35S:GFP-DREB2A/Col plants ( Figure 7D ). These results suggest that DRIP1 could mediate DREB2A protein degradation under nonstressed conditions and consequently may limit the amount of DREB2A to minimize its potential negative effects on plant growth.

DISCUSSION

Signal transduction networks and the regulation of transcription factors affecting stress-responsive gene expression play a central role in stress responses. Among these transcription factors, DREB2A is one of the most important transcription factors controlling drought-responsive gene expression for several reasons: its expression is significantly induced by dehydration and high salt stresses; many stress-inducible genes containing DRE sequence(s) in their promoters are downstream target genes of DREB2A; and overexpression of DREB2A-CA confers strong drought stress tolerance to transgenic Arabidopsis ( Sakuma et al., 2006a ). Although the protein stability of DREB2A is known to be important for the regulation of DREB2A activity, the mechanism of its degradation has not been elucidated.

The single-subunit E3 ligase simultaneously docks ubi-E2 and intermediate and target substrate proteins for ubiquitination in a single protein ( Moon et al., 2004 ; Stone et al., 2005 ). These two biological functions were hypothesized to be mediated by the RING domain and another protein–protein interactive domain. In this research, yeast two-hybrid analysis demonstrated that DRIP1 and DRIP2 proteins, both containing a C3H4 RING domain, interact with the DREB2A protein. Use of in vitro pull-down assays and the in vivo BiFC system confirmed the protein interaction of DRIP1 with DREB2A ( Figures 2A and 2C ). Moreover, the RING domains of DRIP1 and DRIP2 were found to be essential for the E3 ligase activity, and their C-terminal conserved regions were responsible for the protein interaction with DREB2A ( Figures 1C, 1D , and 3 ; see Supplemental Figures 1 and 3 online). However, in yeast two-hybrid assays, the reporter gene activity of yeast cells transformed by the full-length DRIP1 construct and DREB2A bait was quite low ( Figure 1C ). One explanation is that in yeast cells, once the full-length DRIP protein associates with the DREB2A bait protein, it may also lead to the destabilization of the target protein. It is also possible that the full-length DRIP1 protein interaction with DREB2A is actually lower than that of the truncated protein. This would be reasonable to consider, since many protein–protein interaction studies suggest the possibility that the protein–protein interaction might be masked by some domain existing in the full-length protein. Thus, the truncated protein may be more interactive than its counterpart. For example, SPA1 interacts with COP1 through the coiled-coil domains that are contained in both proteins, and the interaction between the two coiled-coil domains was considerably higher than the interaction observed between the two full-length proteins ( Saijo et al., 2003 ). The coiled-coil domain of the COP1 protein, rather than the full-length protein, also displayed a higher interaction with the HFR1 protein ( Yang et al., 2005 ).

In vitro ubiquitination assays in the presence of ubiquitin, rabbit E1, and human UBCH5c (E2) showed that DRIP1 and DRIP2 proteins are clearly autoubiquitinated and display E3 ligase activity ( Figures 3A and 3B ). Deletion of the RING domain of DRIP1 abolished its autoubiquitination activity ( Figure 3C ). Four kinds of E2 enzymes from human, UBCH5c, UBCH10, UBCH12, and UBCH13, and one from Arabidopsis , UBC8, were tested with DRIP1- or DRIP2-GST fusion proteins in the in vitro ubiquitination assays. Only UBCH5c was capable of catalyzing DRIP1 and DRIP2 autoubiquitination (data not shown). Although DRIP1 and DRIP2 protein autoubiquitination was verified in the presence of UBCH5c as an E2 protein, the real interactive E2 protein for DRIP proteins in Arabidopsis remains unknown. DRIP1 and DRIP2 showed low activity mediating the monoubiquitination of the DREB2A in our in vitro ubiquitination assays ( Figure 3D ). However, future studies are necessary to study the in vivo DREB2A ubiquitination status in planta. It is likely that the functional Arabidopsis E2 enzyme may better enable DRIP1 to mediate DREB2A ubiquitination.

Overexpression of DRIP1 in Arabidopsis delayed the expression of some stress-inducible genes in response to dehydration stress. These data indicated that DRIP1 likely plays a negative role in drought stress signaling ( Figure 5 ). Moreover, disruption of DRIP1 expression resulted in enhanced expression of drought-responsive genes in the drip1-1 mutant ( Figure 6C ). Similarly, disruption of DRIP2 also resulted in mildly elevated expression of the genes responsive to dehydration stress in the drip2-1 mutant ( Figure 6C ). Collectively, their sequence homology, similar tissue-specific expression patterns, and equivalent E3 ubiquitin ligase activities indicated that they might share a functional overlap in planta. A double mutant was created ( drip1 drip2 ) and resulted in delay of growth and bolting, resembling the phenotype of the 35S:DREB2A-CA plants ( Figure 6B ). Moreover, microarray and RNA gel blot analyses revealed that disruption of both DRIP1 and DRIP2 significantly increased the expression of a large number of stress-responsive genes induced by dehydration stress, especially genes regulated by DREB2A ( Figure 6C ; see Supplemental Data Set 3 online). DRIP1 and DRIP2 , therefore, negatively function in drought-responsive gene expression.

Tight control of the levels of regulatory proteins is critical for the homeostasis of plants. Overproduction of either DREB1A or DREB2A-CA proteins in plants resulted in deleterious effects on plant growth and development; transgenics in both cases showed overall dwarfed phenotypes ( Liu et al., 1998 ; Sakuma et al., 2006a ). In nature, DREB1A and DREB2A exhibit stress-inducible expression patterns in response to cold and water deficit, respectively. However, expression of the DREB1A gene was always absent before stress, whereas DREB2A transcription could be found in plant roots and weakly in leaves even under normal growth conditions ( Liu et al., 1998 ; Sakuma et al., 2002 ). Although DREB2A mRNA was present, the protein was undetectable and may undergo proteolysis by the 26S proteasome under nonstressed conditions ( Figure 4B ). DRIP1 and DRIP2 are expressed at constant levels in various plant tissues ( Figure 4A ; see Supplemental Figure 2 online). Therefore, it is possible that they function to restrict DREB2A protein amounts prior to stress stimuli. DREB2A protein degradation mediated by DRIP1 and DRIP2 may minimize its negative effects on plant growth and development, since disruption of DRIP1 and DRIP2 expression in the double mutant significantly affected plant germination and bolting ( Figure 6B ). Although most of the drip1 drip2 mutant seeds are viable on germination medium, some seeds failed in germination or were completely arrested in development. It is possible that these observations might be due to seed immaturity. At present, it is also unclear whether the seed maturation program is affected in the double mutant. The ectopic expression of DREB2A in the drip1-1 mutant also resulted in a severe dwarfism in the transgenics, resembling the DREB2A-CA overexpressor ( Figure 7 ).

Since overexpression of intact DREB2A in plants did not produce an evident phenotype, DREB2A protein was predicted to require stress signal–dependent activation for it to be active ( Liu et al., 1998 ; Sakuma et al., 2006a ). Here, we demonstrated that overexpressed, the full-length DREB2A protein is more stable in drip1-1 than in the wild-type background, indicating that elimination of DRIP1 protein results in the stability of DREB2A protein. Moreover, enhanced expression of downstream target genes was also detected in the 35S:GFP-DREB2A/drip1-1 plants, suggesting that stabilization of the protein contributed to the activity of DREB2A in plants. Due to the unavailability of an effective antibody, despite repeated efforts at antibody generation, we are currently unable to dissect the DREB2A ubiquitination status in planta. On the other hand, the possible activation mechanism of DREB2A, in addition to the stabilization of DREB2A under stress conditions, should not be ruled out. In the 2-h dehydrated drip1 drip2 plants, expression of DREB2A-regulated genes was significantly higher than in wild-type plants ( Figure 6C ; see Supplemental Data Set 3 online), indicating that the DREB2A response can be induced in the absence of DRIP1 and DRIP2. The enhanced expression of DREB2A-regulated genes may be attributed to the increased availability of DREB2A protein in plant cells. Additionally, DREB2A may be more accessible for activation upon dehydration stress in the double mutant. Moreover, in the 35S:GFP-DREB2A/drip1-1 plants, although expression of RD29A and RD29B was clearly elevated under normal conditions, higher expression of these genes could be achieved after stress treatment. These data indicate that the stress-dependent activation mechanism might also be necessary for the modulation of DREB2A activity ( Figure 7C ).

Some RING E3 ligases function to attenuate hormone or abiotic signals by modulating the abundance of transcription factors in cells, such as SINAT5 mediating NAC1 ubiquitination, AIP2 targeting ABI3, and HOS1 degrading ICE1 ( Xie et al., 2002 ; Dong et al., 2006 ; Zhang et al., 2005 ). However, some results suggested that DRIP1 and DRIP2 did not function to attenuate stress signals in the late stage of drought stress, although DRIP1 and DRIP2 function negatively in stress-responsive gene expression. First, unlike SINAT5 , AIP2 , or HOS1 , DRIP1 expression was not stress-inducible; instead, DRIP1 was constitutively expressed under stress and nonstressed conditions (see Supplemental Figure 3A online). These data cannot fit into an attenuation hypothesis. Second, overexpression of DRIP1 in Arabidopsis delayed the expression of drought-responsive genes at an early phase of the dehydration response but did not reduce the expression of these genes at a late phase of the stress. Third, we could not detect higher expression of stress-responsive genes in the drip1 drip2 double mutant at a later phase of drought stress (data not shown). DRIP1 may not function to attenuate drought stress signals but instead restricts DREB2A transactivation activity under nonstressed conditions.

In conclusion, two novel proteins, DRIP1 and DRIP2, were identified and confirmed to interact with DREB2A. DRIP1 and DRIP2 function as RING E3 ligases and play negative roles in plant drought stress signaling, probably by mediating DREB2A proteolysis through 26S proteasome protein degradation. We hypothesized that under normal growth conditions, plants maintained low or leaky expression of DREB2A . However, the translated DREB2A protein was recognized and ubiquitinated by the constitutively expressed DRIP1 and DRIP2 and subjected to 26S proteasome proteolysis. Once stress signaling occurred, the degradation process might be blocked or inhibited directly by either stresses or a competition of a stress signal-dependent modification. Consequently, plant cells could quickly acquire a sufficient amount of the effective DREB2A protein to activate the expression of downstream stress-responsive genes ( Figure 8 ).
