---
pmid: '10710437'
title: RNA-binding analyses of HuC and HuD with the VEGF and c-myc 3'-untranslated
  regions using a novel ELISA-based assay.
authors:
- King PH
journal: Nucleic Acids Res
year: '2000'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC102803
doi: 10.1093/nar/28.7.e20
---

# RNA-binding analyses of HuC and HuD with the VEGF and c-myc 3'-untranslated regions using a novel ELISA-based assay.
**Authors:** King PH
**Journal:** Nucleic Acids Res (2000)
**DOI:** [10.1093/nar/28.7.e20](https://doi.org/10.1093/nar/28.7.e20)
**PMC:** [PMC102803](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC102803/)

## Abstract

1. Nucleic Acids Res. 2000 Apr 1;28(7):E20. doi: 10.1093/nar/28.7.e20.

RNA-binding analyses of HuC and HuD with the VEGF and c-myc 3'-untranslated 
regions using a novel ELISA-based assay.

King PH(1).

Author information:
(1)Department of Neurology, University of Alabama, 1235 Jefferson Tower, 625 
South 19th Street, Birmingham, AL 35294-0007, USA. pking@uab.edu

Human members of the ELAV family, referred to as ELAV-like proteins (ELPs), 
include HuC, HuD, Hel-N1 and HuR. These proteins bind to AU-rich elements in the 
3'-untranslated regions (3'-UTRs) of many growth-related mRNAs, including c-myc 
and VEGF, and may participate in regulating the stability of these transcripts. 
Here, I have developed an enzyme-linked immunosorbent assay (ELISA) which can 
rapidly assess the RNA-protein-binding properties of ELPs. With this assay, I 
demonstrate that HuC and HuD bind to the VEGF 3'-UTR regulatory segment (VRS) 
and to the c- myc 3'-UTR in a specific and concentration-dependent pattern, with 
both proteins showing a greater affinity for the VRS. Further analysis of the 
VRS indicated that the binding affinity was greater for the 3'-end where the 
majority of AU motifs reside. Binding to the VRS could be competed by both 
proteins as well as a poly(U) ribohomopolymer. The binding could not be competed 
by other ribohomopolymers or serum from patients with high titer anti-HuD 
antibodies. In summary, this assay provides a rapid analysis of ELP-RNA binding 
which can be utilized for further characterization of RNA-binding properties and 
for identification of competitor molecules for in vivo functional analysis of 
ELPs.

DOI: 10.1093/nar/28.7.e20
PMCID: PMC102803
PMID: 10710437 [Indexed for MEDLINE]

## Full Text

Abstract

Human members of the ELAV family, referred to as ELAV-like proteins (ELPs), include HuC, HuD, Hel-N1 and HuR. These proteins bind to AU-rich elements in the 3′-untranslated regions (3′-UTRs) of many growth-related mRNAs, including c- myc and VEGF , and may participate in regulating the stability of these transcripts. Here, I have developed an enzyme-linked immunosorbent assay (ELISA) which can rapidly assess the RNA–protein-binding properties of ELPs. With this assay, I demonstrate that HuC and HuD bind to the VEGF 3′-UTR regulatory segment (VRS) and to the c- myc 3′-UTR in a specific and concentration-dependent pattern, with both proteins showing a greater affinity for the VRS. Further analysis of the VRS indicated that the binding affinity was greater for the 3′-end where the majority of AU motifs reside. Binding to the VRS could be competed by both proteins as well as a poly(U) ribohomopolymer. The binding could not be competed by other ribohomopolymers or serum from patients with high titer anti-HuD antibodies. In summary, this assay provides a rapid analysis of ELP–RNA binding which can be utilized for further characterization of RNA-binding properties and for identification of competitor molecules for in vivo functional analysis of ELPs.

INTRODUCTION

AU-rich elements (ARE) are present within the 3′-untranslated regions (3′-UTRs) of many protooncogene, cytokine and lymphokine mRNAs and has been linked to the stability of these transcripts ( 1 , 2 ). The ELAV family of RNA-binding proteins is a prime candidate for participation in this level of gene regulation because of its avid and specific binding to the ARE ( 3 – 6 ). The four human members, HuR, Hel-N1, HuC and HuD, referred to as ELAV-like proteins (ELP), have been shown by immunoprecipitation or gel mobility shift assay to bind c- myc , c- fos , Id , N- myc , GM-CSF , VEGF and GAP -43 mRNAs ( 3 , 4 , 6 – 10 ). Although the biological consequence of this binding remains to be elucidated, recent evidence suggests that HuR may participate in the stabilization of VEGF and c- fos mRNAs ( 11 – 13 ). While the basic requirement of an ARE for ELAV protein binding is clear, many questions remain unanswered. Different members, for example, have been shown to bind to the same 3′-UTRs, such as c- fos and c- myc ( 3 , 4 , 6 ), but do some family members bind preferentially to certain 3′-UTRs? Are there other requirements for binding in addition to the AU motifs within the ARE? Are there competitor molecules that can disrupt the RNA–protein binding and thus be utilized in vivo to investigate the biological function of ELPs? Here I describe an enzyme-linked immunosorbent assay (ELISA)-based RNA-binding assay which can rapidly delineate the RNA-binding properties of ELPs. Using the 3′-UTRs of VEGF and c- myc as model RNA targets, I demonstrate several novel binding characteristics of two neural-specific ELPs, HuC and HuD. This assay will facilitate future efforts to characterize the ELP–RNA interaction and to identify competitor molecules which can inhibit this interaction.

DISCUSSION

In this paper I have demonstrated a novel, ELISA-based assay to analyze the RNA-binding properties of HuC and HuD using the c- myc 3′-UTR and the VRS as model RNA targets. The biological relevance of these RNA–protein interactions has recently come to light when HuR, another ELP, was shown to participate in stabilization of the VEGF transcript under hypoxic conditions ( 11 ). The ELPs, however, have been shown to bind the 3′-UTRs of many other growth-related genes, almost interchangeably. This assay will expedite both the analysis and comparison of binding by individual ELPs to these many possible target RNAs. Furthermore, it will help define additional cis elements which participate in the specificity of binding among ELPs.

Previous RNA-binding analyses with ELPs, including the initial one linking them to AU-rich RNA sequences, have centered on the standard techniques of immunoprecipitation, gel mobility shift, UV crosslinking or filter binding ( 3 , 4 , 6 , 8 , 9 , 11 ). In each of these assays, both the protein and target RNA remain in solution during the binding phase. The assay described here, however, indicates that the RNA-binding capacity of ELPs is preserved following adsorption to an ELISA well. The ELISA assay permits a rapid assessment (requiring <2 h) of RNA-binding properties as shown with the c- myc 3′-UTR and the VRS (Fig. 2 ). For each protein, a concentration-dependent binding curve is generated, from which an apparent K d can be calculated. The lack of any signal with control proteins or RNA, moreover, indicated that the RNA–protein binding within the ELISA well was highly specific. The assay also allowed an accurate assessment of binding kinetics, indicating that near maximal binding of HuD to the VRS occurred within 60 min. In addition to a shortened time of analysis, the advantage of this assay is that a large number of individual RNA-binding experiments can be performed simultaneously. With the use of a multi-pipettor and washer, for example, 40–60 binding reactions can be initiated and stopped within 1 min. This number of samples would be prohibitive for the standard techniques of gel shift analysis or immunoprecipitation. Likewise, the assay utilizes biotinylated probes which obviates the need for radioisotopes.

With this assay, the first observation with HuC and HuD was the substantially higher affinity of both proteins for the VRS compared to the c- myc 3′-UTR (Fig. 2 ). The apparent K d , for example, was from 4- to 10-fold higher for the VRS. To explore potential explanations for this discrepancy, both sequences were analyzed with respect to motifs linked to ELP binding. In the original RNA-binding study with ELPs, Hel-N1 was shown by immunoprecipitation to bind three U-rich motifs, including AUUUG, GUUUUU and the canonical motif for RNA stability, AUUUA ( 1 – 3 ). Since then, two other high affinity binding sites have been identified for HuD and HuR (in addition to the AUUUA motif): AUUUUA and AUUUUUA ( 4 , 5 ). The higher affinity of binding to the VRS probe observed here may relate to the greater number of AU motifs within this sequence. Notably, the VRS probe contains a higher number of nearly all the motifs which have been linked to ELP binding (Table 1 ). The c- myc probe also lacks one of the two high affinity sites identified for HuR by RNase T1 mapping ( 4 ). The importance of these motifs was demonstrated by analyzing the truncated segments of the VRS (Fig. 3 ). When these motifs are not present, as with the VRS 5′ construct, binding of both HuD and HuC was markedly reduced (Fig. 3 ). This finding is similar to that for HuD and c- fos ARE binding, where deletion of two uridine stretches of 4–7 nt (with flanking adenosines) completely abrogated binding ( 5 ). Interestingly, both the VRS 5′ and 3′ constructs had a similar number of AUUUA motifs and the 5′ construct had a higher number of GUUUUs, which further emphasizes the importance of longer uridine stretches in binding affinity. The effective inhibition of ELP–VRS binding by the poly(U) homoribopolymer further supports this observation. The absolute number of uridine residues, however, does not predict ELP binding as the VRS 3′ construct contained 59% fewer residues than the 5′ construct. Interestingly, the 3′ construct contained the nonamer motif UUAUUUAUU, which has been linked to mRNA degradation ( 22 , 23 ). The importance of this motif in ELP binding will require additional mutational analyses.

This assay also effectively demonstrates the capacity of ELPs to compete with each other for RNA binding. The competition for binding to the VRS occurred with similar concentrations of either protein (10-fold excess), which is consistent with the similar binding affinities observed in Figure 2 . This finding suggests that the ELPs could potentially be used as inhibitors to further analyze the biological roles of individual family members in certain cell systems. Overexpression of HuD in gliomas, for example, could effectively compete with HuR for VEGF binding and block the adaptive responses to hypoxia ( 11 ). Polyclonal anti-ELP antibodies, on the other hand, did not compete for VRS binding over a range of titers. This finding raises the possibility that the dominant epitopes for the polyclonal antibodies are away from the RNP motifs where RNA binding is thought to occur ( 24 , 25 ). The antibodies, however, may participate in the pathogenesis of paraneoplastic encephalomyelitis by other mechanisms, such as inhibiting ELP interactions with other proteins.

In summary, the ELISA-based RNA-binding assay provides a rapid assessment of ELPs and their capacity to bind AREs. Since a large number of growth-related genes, including cytokine, lymphokine and other protooncogene genes, contain AREs in the 3′-UTR ( 2 ), the assay will permit a rapid analysis and ranking of the ELPs and their capacity to bind these mRNAs. Furthermore, this assay will provide an efficient screening process to identify potential competitor molecules (peptides, nucleic acids or antibodies) that can disrupt RNA binding. The in vivo inhibition of HuR binding to the VRS, for example, may destabilize the mRNA and down-regulate its expression ( 11 – 13 , 26 ). Such an inhibition could have therapeutic benefits in cancers where neovascularization has been linked to the stabilization of VEGF mRNA ( 27 – 29 ).
