---
pmid: '18776048'
title: Comparisons of tyrosine phosphorylated proteins in cells expressing lung cancer-specific
  alleles of EGFR and KRAS.
authors:
- Guha U
- Chaerkady R
- Marimuthu A
- Patterson AS
- Kashyap MK
- Harsha HC
- Sato M
- Bader JS
- Lash AE
- Minna JD
- Pandey A
- Varmus HE
journal: Proc Natl Acad Sci U S A
year: '2008'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC2531065
doi: 10.1073/pnas.0806158105
---

# Comparisons of tyrosine phosphorylated proteins in cells expressing lung cancer-specific alleles of EGFR and KRAS.
**Authors:** Guha U, Chaerkady R, Marimuthu A, Patterson AS, Kashyap MK, Harsha HC, Sato M, Bader JS, Lash AE, Minna JD, Pandey A, Varmus HE
**Journal:** Proc Natl Acad Sci U S A (2008)
**DOI:** [10.1073/pnas.0806158105](https://doi.org/10.1073/pnas.0806158105)
**PMC:** [PMC2531065](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2531065/)

## Abstract

1. Proc Natl Acad Sci U S A. 2008 Sep 16;105(37):14112-7. doi: 
10.1073/pnas.0806158105. Epub 2008 Sep 5.

Comparisons of tyrosine phosphorylated proteins in cells expressing lung 
cancer-specific alleles of EGFR and KRAS.

Guha U(1), Chaerkady R, Marimuthu A, Patterson AS, Kashyap MK, Harsha HC, Sato 
M, Bader JS, Lash AE, Minna JD, Pandey A, Varmus HE.

Author information:
(1)Program in Cancer Biology and Genetics, Memorial Sloan-Kettering Cancer 
Center, New York, NY 10065, USA. guhau@mskcc.org

We have used unbiased phosphoproteomic approaches, based on quantitative mass 
spectrometry using stable isotope labeling with amino acids in cell culture 
(SILAC), to identify tyrosine phosphorylated proteins in isogenic human 
bronchial epithelial cells (HBECs) and human lung adenocarcinoma cell lines, 
expressing either of the two mutant alleles of EGFR (L858R and Del E746-A750), 
or a mutant KRAS allele, which are common in human lung adenocarcinomas. 
Tyrosine phosphorylation of signaling molecules was greater in HBECs expressing 
the mutant EGFRs than in cells expressing WT EGFR or mutant KRAS. Receptor 
tyrosine kinases (such as EGFR, ERBB2, MET, and IGF1R), and Mig-6, an inhibitor 
of EGFR signaling, were more phosphorylated in HBECs expressing mutant EGFR than 
in cells expressing WT EGFR or mutant RAS. Phosphorylation of some proteins 
differed in the two EGFR mutant-expressing cells; for example, some cell 
junction proteins (beta-catenin, plakoglobin, and E-cadherin) were more 
phosphorylated in HBECs expressing L858R EGFR than in cells expressing Del EGFR. 
There were also differences in degree of phosphorylation at individual tyrosine 
sites within a protein; for example, a previously uncharacterized 
phosphorylation site in the nucleotide-binding loop of the kinase domains of 
EGFR (Y727), ERBB2 (Y735), or ERBB4 (Y733), is phosphorylated significantly more 
in HBECs expressing the deletion mutant than in cells expressing the wild type 
or L858R EGFR. Signaling molecules not previously implicated in ERBB signaling, 
such as polymerase transcript release factor (PTRF), were also phosphorylated in 
cells expressing mutant EGFR. Bayesian network analysis of these and other 
datasets revealed that PTRF might be a potentially important component of the 
ERBB signaling network.

DOI: 10.1073/pnas.0806158105
PMCID: PMC2531065
PMID: 18776048 [Indexed for MEDLINE]

Conflict of interest statement: The authors declare no conflict of interest.

## Full Text

Abstract

We have used unbiased phosphoproteomic approaches, based on quantitative mass spectrometry using stable isotope labeling with amino acids in cell culture (SILAC), to identify tyrosine phosphorylated proteins in isogenic human bronchial epithelial cells (HBECs) and human lung adenocarcinoma cell lines, expressing either of the two mutant alleles of EGFR ( L858R and Del E746-A750 ), or a mutant KRAS allele, which are common in human lung adenocarcinomas. Tyrosine phosphorylation of signaling molecules was greater in HBECs expressing the mutant EGFRs than in cells expressing WT EGFR or mutant KRAS. Receptor tyrosine kinases (such as EGFR, ERBB2, MET, and IGF1R), and Mig-6, an inhibitor of EGFR signaling, were more phosphorylated in HBECs expressing mutant EGFR than in cells expressing WT EGFR or mutant RAS. Phosphorylation of some proteins differed in the two EGFR mutant-expressing cells; for example, some cell junction proteins (β-catenin, plakoglobin, and E-cadherin) were more phosphorylated in HBECs expressing L858R EGFR than in cells expressing Del EGFR. There were also differences in degree of phosphorylation at individual tyrosine sites within a protein; for example, a previously uncharacterized phosphorylation site in the nucleotide-binding loop of the kinase domains of EGFR (Y727), ERBB2 (Y735), or ERBB4 (Y733), is phosphorylated significantly more in HBECs expressing the deletion mutant than in cells expressing the wild type or L858R EGFR. Signaling molecules not previously implicated in ERBB signaling, such as polymerase transcript release factor (PTRF), were also phosphorylated in cells expressing mutant EGFR. Bayesian network analysis of these and other datasets revealed that PTRF might be a potentially important component of the ERBB signaling network.

Results

As a preliminary survey of phosphotyrosine-based signaling activity stimulated by oncogenic alleles of KRAS and EGFR , we used immunoprecipitation and Western blot analysis with anti-phosphotyrosine antibodies to compare proteins in isogenic HBECs expressing WT EGFR, KRAS G12V, L858R EGFR, or DelE746-A750 EGFR, and in several nonisogenic human adenocarcinoma cell lines with KRAS and EGFR mutations [ supporting information (SI) Fig. S1 ]. The HBECs expressing EGFR mutants and the adenocarcinoma cells with an EGFR mutation constitutively displayed more tyrosine phosphorylated proteins than did the other cell lines, and there was minimal further stimulation of tyrosine phosphorylation upon EGF administration in these cells. However, AKT and ERK were activated, suggesting that overexpression of the mutant receptors did not alter canonical EGFR signaling in these cells ( Fig. S1 C ).

Discussion

We used mass spectrometry to identify proteins in phosphotyrosine IPs from whole cell extracts, an approach that has been used extensively to probe the dynamics of tyrosine phosphorylation and tyrosine phosphorylation-dependent interaction after growth factor stimulation ( 19 , 26 ). In addition, we enriched tyrosine phosphorylated peptides by immunoprecipitation to quantify differences in degree of tyrosine phosphorylation, at specific sites, within a protein.

Our survey of tyrosine phosphorylated proteins showed that either of the common forms of oncogenic EGFR causes increased tyrosine phosphorylation of numerous signaling molecules, either directly or indirectly, compared with WT EGFR or mutant KRAS. In addition, some proteins enriched in antiphosphotyrosine immunoprecipitates may not themselves be phosphorylated, but may be more likely to associate with proteins that are more efficiently tyrosine phosphorylated in cells expressing mutant EGFR than in cells expressing WT EGFR or mutant KRAS.

In some systems, activated RAS proteins appear to augment RTK signaling, for example, by increasing the production of EGFR ligands, such as TGFα ( 27 ). We did not find evidence for this in HBECs expressing oncogenic KRAS. However, 52 proteins were more abundant in phosphotyrosine IPs from the adenocarcinoma cell line H2030 (harboring mutant KRAS), than in the cell line H1650 (with mutant EGFR). We contend that the likely explanation for these unexpected findings is the difference in concentrations of such proteins in the two adenocarcinoma cell lines, as exemplified by the lower levels of RIN1 or the p66 isoform of SHC in the H1650 cell line. These findings underscore the importance of studying differences in the status of signaling proteins by using isogenic cells (such as the HBEC lines) that express different oncogenes or by comparing a single cell line under different conditions (e.g., with and without growth factors or in the presence and absence of inhibitors).

We identified 13 proteins that were hyperphosphorylated in HBECs expressing mutant EGFR and in the adenocarcinoma cell line H1650. Among these were RTKs, such as EGFR, ERBB2, and MET, suggesting a role for lateral signaling or cross talk between various RTKs, with subsequent signaling through multiple receptors. In support of this possibility, multiple RTKs have been shown to be active in glioblastoma cell lines, and combinations of RTK inhibitors can reduce cell survival and downstream signaling in these cells ( 28 ). MET has been shown to interact with EGFR ( 14 ), and combination therapy with EGFR and MET inhibitors is better than EGFR inhibitors alone in glioblastoma cells expressing the oncogenic EGFR-VIII mutant ( 21 ).

Cell junction proteins that have been implicated in ERBB signaling, such as β-catenin, E-cadherin, δ-catenin, and junctional plakoglobin, were also more abundant in phosphotyrosine IPs from cells expressing mutant EGFR. Others have reported that β-catenin-ERBB complexes are present in breast tumors in MMTV-Wnt1 and MMTV-neu transgenic mice, and in infiltrating ductal breast carcinomas in women ( 29 ). Recently, AKT-mediated phosphorylation of β-catenin downstream of EGFR signaling has been linked to increased transcriptional activity of β-catenin and tumor cell invasion ( 30 ), suggesting that increased tyrosine phosphorylation of β-catenin and E-cadherin may be important in transducing signals from the oncogenic EGFRs.

We also identified proteins that were hyperphosphorylated in mutant EGFR-expressing cell lines, but not previously implicated in EGFR signaling (see Table 1 ). One such protein, PTRF, co-localizes with caveolin 1 in caveolae in adipocytes and has been implicated in the formation of caveolae ( 31 , 32 , 39 ). Bayesian network modeling of proteins, based on their degree of tyrosine phosphorylation in various ERBB signaling and mass spectrometry-based studies, including ours, reveals that PTRF may be an integral member of ERBB signaling network ( Fig. 5 B ).

The overall level of tyrosine phosphorylation in EGFR or ERBB2 was similar in HBECs expressing L858R EGFR and Del EGFR. However, there were interesting differences at individual sites of phosphorylation within EGFR. The most significant of these sites was Y727 of EGFR, the site corresponding to Y735 of ERBB2 or Y733 of ERBB4. This site was phosphorylated in the Del-EGFR-expressing cells but not in L858R or WT EGFR-expressing HBEC cells. Although this site was found to be phosphorylated in adenocarcinoma cell lines that harbor the L858R mutation (H3255) and a line that carries the Del EGFR allele (HCC827) ( 14 ), the degree of phosphorylation at this site was not compared between the cell lines expressing the two EGFR mutants. Ward et al. have shown that a synthetic peptide of EGFR, phosphorylated at Y727, binds to purified SHC SH2 domain in vitro ( 33 ). In addition, phosphorylated peptides containing Y727 of EGFR, Y735 of ERBB2, or Y733 of ERBB4 have been shown to bind SHC in lysates of HeLa cells ( 34 ). We speculate that phosphorylation at the above site in EGFR, ERBB2, or ERBB4 occurs in a specific signaling context, and phosphorylation may recruit certain signaling molecules having SH2 domains. Phosphorylation at this site may also alter the structure of the catalytic site in the ERBBs to affect kinase activity.

A few proteins involved in ERBB receptor recycling, such as Caveolins, Rab7, Rab8, and the protein Mig-6, a known inhibitor of ERBB signaling had increased phosphorylation in cells expressing either of the mutant EGFRs. Mig-6 has been shown to act as a tumor suppressor, and germ-line disruption of Mig-6 results in adenoma or adenocarcinomas in the lung ( 35 ). The crystal structure of a fragment of Mig-6 and the EGFR kinase domain has shown that Mig-6 inhibits the formation of an activating dimer interface in EGFR. The C-terminal fragment of Mig-6 containing the Y394 site increases the potency of inhibition of the L858R EGFR kinase activity in solution ( 36 ). From these studies, we speculate that Y394 of Mig-6 is a direct substrate of EGFR. The significance of increased phosphorylation at this site by the mutant EGFRs remains to be determined.
