---
pmid: '16891401'
title: Functional profiling reveals that only a small number of phytochrome-regulated
  early-response genes in Arabidopsis are necessary for optimal deetiolation.
authors:
- Khanna R
- Shen Y
- Toledo-Ortiz G
- Kikis EA
- Johannesson H
- Hwang YS
- Quail PH
journal: Plant Cell
year: '2006'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC1560915
doi: 10.1105/tpc.106.042200
---

# Functional profiling reveals that only a small number of phytochrome-regulated early-response genes in Arabidopsis are necessary for optimal deetiolation.
**Authors:** Khanna R, Shen Y, Toledo-Ortiz G, Kikis EA, Johannesson H, Hwang YS, Quail PH
**Journal:** Plant Cell (2006)
**DOI:** [10.1105/tpc.106.042200](https://doi.org/10.1105/tpc.106.042200)
**PMC:** [PMC1560915](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1560915/)

## Abstract

1. Plant Cell. 2006 Sep;18(9):2157-71. doi: 10.1105/tpc.106.042200. Epub 2006 Aug
 4.

Functional profiling reveals that only a small number of phytochrome-regulated 
early-response genes in Arabidopsis are necessary for optimal deetiolation.

Khanna R(1), Shen Y, Toledo-Ortiz G, Kikis EA, Johannesson H, Hwang YS, Quail 
PH.

Author information:
(1)Department of Plant and Microbial Biology, University of California, 
Berkeley, 94720, USA.

In previous time-resolved microarray-based expression profiling, we identified 
32 genes encoding putative transcription factors, signaling components, and 
unknown proteins that are rapidly and robustly induced by phytochrome 
(phy)-mediated light signals. Postulating that they are the most likely to be 
direct targets of phy signaling and to function in the primary phy regulatory 
circuitry, we examined the impact of targeted mutations in these genes on the 
phy-induced seedling deetiolation process in Arabidopsis thaliana. Using 
light-imposed concomitant inhibition of hypocotyl and stimulation of cotyledon 
growth as diagnostic criteria for normal deetiolation, we identified three major 
mutant response categories. Seven (22%) lines displayed statistically 
significant, reciprocal, aberrant photoresponsiveness in the two organs, 
suggesting disruption of normal deetiolation; 13 (41%) lines displayed 
significant defects either unidirectionally in both organs or in hypocotyls 
only, suggesting global effects not directly related to photomorphogenic 
signaling; and 12 (37%) lines displayed no significant difference in 
photoresponsiveness from the wild type. Potential reasons for the high 
proportion of rapidly light-responsive genes apparently unnecessary for the 
deetiolation phenotype are discussed. One of the seven disrupted genes 
displaying a significant mutant phenotype, the basic helix-loop-helix 
factor-encoding PHYTOCHROME-INTERACTING FACTOR3-LIKE1 gene, was found to be 
necessary for rapid light-induced expression of the photomorphogenesis- and 
circadian-related PSEUDO-RESPONSE REGULATOR9 gene, indicating a regulatory 
function in the early phy-induced transcriptional network.

DOI: 10.1105/tpc.106.042200
PMCID: PMC1560915
PMID: 16891401 [Indexed for MEDLINE]

## Full Text

Abstract

In previous time-resolved microarray-based expression profiling, we identified 32 genes encoding putative transcription factors, signaling components, and unknown proteins that are rapidly and robustly induced by phytochrome (phy)-mediated light signals. Postulating that they are the most likely to be direct targets of phy signaling and to function in the primary phy regulatory circuitry, we examined the impact of targeted mutations in these genes on the phy-induced seedling deetiolation process in Arabidopsis thaliana . Using light-imposed concomitant inhibition of hypocotyl and stimulation of cotyledon growth as diagnostic criteria for normal deetiolation, we identified three major mutant response categories. Seven (22%) lines displayed statistically significant, reciprocal, aberrant photoresponsiveness in the two organs, suggesting disruption of normal deetiolation; 13 (41%) lines displayed significant defects either unidirectionally in both organs or in hypocotyls only, suggesting global effects not directly related to photomorphogenic signaling; and 12 (37%) lines displayed no significant difference in photoresponsiveness from the wild type. Potential reasons for the high proportion of rapidly light-responsive genes apparently unnecessary for the deetiolation phenotype are discussed. One of the seven disrupted genes displaying a significant mutant phenotype, the basic helix-loop-helix factor–encoding PHYTOCHROME-INTERACTING FACTOR3-LIKE1 gene, was found to be necessary for rapid light-induced expression of the photomorphogenesis- and circadian-related PSEUDO-RESPONSE REGULATOR9 gene, indicating a regulatory function in the early phy-induced transcriptional network.

INTRODUCTION

Reverse genetics disruption of target genes provides a centrally important tool for examining the consequences of the loss of gene function. This strategy has been used in multiple model eukaryotes in a range of contexts, from investigation of specific individual genes to global systematic assessment of all genes in the genome ( Krysan et al., 1999 ; Giaever et al., 2002 ; Kamath and Ahringer, 2003 ; Carpenter and Sabatani, 2004 ; Kuepfer et al., 2005 ; Sönnichsen et al., 2005 ). The majority of genome-scale studies reported to date have focused on probing the functional roles of multiple sequence-related members of selected gene families of interest. In yeast, however, genes identified by microarray-based, genome-wide expression profiling as exhibiting significant increases in expression in response to different imposed stress conditions have been assessed in parallel for their functional necessity to the cell's capacity to respond and grow optimally under those conditions ( Giaever et al., 2002 ). This was assessed by coculturing all members of a genome-wide set of gene-disruption mutants under each stress condition and monitoring the fitness of each mutant for growth and survival using gene bar code tags. Strikingly, only 0.3 to 7% of the genes that displayed stress-induced expression changes were functionally necessary for optimal growth under the relevant stress imposed ( Giaever et al., 2002 ). Genome-wide RNA interference screens have been conducted in Caenorhabditis elegans to identify genes whose silencing causes observable growth phenotypes, but no correlation with the expression of these genes was examined ( Kamath and Ahringer, 2003 ; Sönnichsen et al., 2005 ). In Arabidopsis thaliana , the available disruptional mutant collections (Syngenta Arabidopsis Insertion Library [SAIL] and SIGnAL) have been used to assess the functional relevance of numerous individual genes to a variety of cellular and developmental processes as well as to explore the potential functions of the multiple sequence-related members of some of the various gene families present in plants ( Kranz et al., 1998 ; Meissner et al., 1999 ; Okushima et al., 2005 ). However, except for a limited study of four ethylene-response genes ( Alonso et al., 2003 ) and three salicylic acid–induced genes ( Wang et al., 2005 ), there have been no reports of the systematic global analysis of genes identified by expression profiling as coordinately induced by specific environmental or hormonal signals for functional relevance to the plant's response to these signals. Only very recently, one expression profiling study examined 16 genes in Arabidopsis that respond to developmental cues associated with secondary cell wall formation and found that 7 of these genes are apparently functionally involved in secondary cell wall biosynthesis by reverse genetics ( Brown et al., 2005 ).

The Arabidopsis phytochrome (phy) family (designated phyA to phyE) of red (R) and far-red (FR) light sensory photoreceptors regulates growth and developmental transitions in response to light signals throughout the plant life cycle ( Quail, 2002a , 2002b ). Studies with mutants lacking functional photoreceptor proteins have shown that the different phy family members exhibit differential, albeit partially overlapping, photosensory and/or physiological functions in controlling plant light responses ( Whitelam and Devlin, 1997 ; Devlin et al., 1998 ; Quail, 1998 ; Tepperman et al., 2004 , 2006 ). Although phyA is exclusively responsible for photoresponsiveness to continuous monochromatic far-red (FRc) light, phyB has been considered to be predominantly responsible for photoresponsiveness to continuous monochromatic red (Rc) light ( Quail et al., 1995 ; Fankhauser and Chory, 1997 ; Whitelam and Devlin, 1997 ; Quail, 1998 ; Whitelam et al., 1998 ; Neff et al., 2000 ; Chen et al., 2004 ; Franklin and Whitelam, 2004 ). Strikingly, however, recent microarray expression profiling has revealed that phyA, and not phyB, dominates in transducing Rc signals to rapidly responding genes at the initiation of seedling deetiolation ( Monte et al., 2004 ; Tepperman et al., 2006 )

Seedling deetiolation is marked by concomitant light-induced inhibition of hypocotyl cell elongation and reciprocal promotion of cotyledon cell expansion ( Quail, 2002a ). Genetic screens for mutant seedlings exhibiting hyposensitivity or hypersensitivity to light inhibition of hypocotyl cell elongation in response to FRc and Rc light have identified numerous loci putatively involved in phy signaling pathways ( Quail, 2002a , 2002b ; Gyula et al., 2003 ; Kaczorowski and Quail, 2003 ; Wang and Deng, 2003 ; Chen et al., 2004 ; Franklin and Whitelam, 2004 ; Huq and Quail, 2005 ; Zhou et al., 2005 ). These forward genetics screens have been very useful but appear to have been carried to near saturation, perhaps because in practice they are limited to identifying mutations that cause relatively large phenotypic differences that are easily observable when comparing hypocotyl length. However, mutations in the Arabidopsis genome representing all of the genes necessary to the optimal functioning of the phy signaling pathway may be far from saturated. Disruptions of some genes may result in obscured phenotypes or may exhibit only weak phenotypes as a result of possible partial functional redundancy with other closely related gene family members ( Zhou et al., 2005 ). Reverse genetics can help overcome this problem by targeting genes of interest and studying the effects of loss of function independently or in combination with the loss of function of one or more other family members ( Ostergaard and Yanofsky, 2004 ).

Using Affymetrix ATH1 microarray genome-scale expression profiling (22,000 genes), we have identified a set of genes regulated by the phy system in response to light signals during seedling deetiolation ( Monte et al., 2004 ; Tepperman et al., 2006 ). These data confirm and extend the list of genes identified as robustly light-responsive in earlier studies using the more limited 8000-gene Affymetrix microarray ( Tepperman et al., 2001 , 2004 ). Based on the rationale that genes responding most rapidly to the light signals are those most likely to be direct targets of phy signaling and to have a functionally significant role in regulating downstream responses, we have focused on genes that respond within 1 h of initial exposure of etiolated seedlings to light (defined as early-response genes) for further analysis. Of the large number of genes identified as responding rapidly to light in a statistically significant manner by limma analysis at a false discovery rate P value of 0.05 ( Gentleman et al., 2004 ; Smyth, 2004 ; see http://www.bioconductor.org/ ), we have chosen to focus on the subset exhibiting the quantitatively greatest response to the light signal (defined as greater than twofold induction or repression relative to unirradiated controls). Sequential application of these dual statistical and response-magnitude criteria, coupled with RNA gel blot validation of multiple selected examples, has resulted in the identification of ∼250 early-response genes defined as robustly light-regulated ( Monte et al., 2004 ; Tepperman et al., 2006 ). The data show that phyA dominates in eliciting the early-light-induced responsiveness of these genes ( Tepperman et al., 2006 ).

To begin to determine whether these genes have functionally relevant roles in transducing phy signals in the overall, global process of seedling deetiolation, we have selected here an initial subset of 32 genes categorized as having potential transcriptional-regulatory or signaling activity, as well as several unannotated sequences, for systematic analysis by reverse-genetic disruption. For this purpose, we have isolated and/or characterized for photomorphogenic defects insertional-disruption or other known mutants in these genes. These include five previously reported mutants characterized to date only for hypocotyl length, and not cotyledon expansion, and therefore lacking definitive phenotypic evidence of involvement at early steps in the complete seedling deetiolation process.

DISCUSSION

The primary goal of this analysis was to determine whether and to what extent genes that respond rapidly to, and are therefore potential primary targets of, the light-induced phy signaling system exhibit overt functional necessity to the visibly monitorable seedling deetiolation process, as determined by targeted reverse genetics disruption of the gene. Genes identified as phy-regulated, early-response genes and assigned to transcriptional, signaling, or unknown functional categories in our microarray analyses ( Tepperman et al., 2001 , 2004 , 2006 ; Monte et al., 2004 ) were selected as those most likely to have regulatory functions in phy signaling and transcriptional networks, and all available T-DNA insertional mutants for these loci were obtained. The well-established, coordinated reciprocal growth responses of hypocotyl and cotyledon cells induced by light signals ( Quail, 2002a ) were stringently applied as diagnostic criteria in defining loci most likely to be involved in early steps in the overall deetiolation process. Only mutants displaying statistically significant, light-induced, reciprocal deviations of hypocotyl and cotyledon cell expansion rates from those of the wild type (class I mutants, Table 1 ) are considered to provide evidence of potential defects in early steps of the phy signaling or transcriptional networks. Conversely, mutants displaying parallel, unidirectional perturbations of cell expansion responses in the two organs (class II mutants, Table 1 ; here, exhibiting significant inhibition [six mutants] or enhancement [one mutant] of both hypocotyl and cotyledon growth in light compared with the wild type) are considered more likely to be affected in general cell growth processes than in the upstream events specific to phy signaling. Mutants displaying statistically significant deviations in light-regulated growth of only one of the two organs (class III mutants, Table 1 ; here, all exhibiting stronger inhibition of hypocotyl extension than the wild type but no difference in cotyledon expansion) could represent organ-specific components involved in either normal phy signaling or more general cell growth processes. Distinguishing between these and other possible alternatives will require more extensive investigation. These mutants, therefore, were not included as representing likely contributors to phy signaling in this study.

The data indicate that only four of the genes newly mutagenized in this study exhibit significant perturbation of the synchronized parameters of the light-induced deetiolation process (class I, Table 1 ). One of these, PKS1 , had been implicated previously in phy signaling by Fankhauser et al. (1999) based on overexpression, a notion supported more recently by pks1 mutant data ( Lariguet et al., 2003 ). Two more of these genes, ELF4 ( Khanna et al., 2003 ) and PIL1 ( Salter et al., 2003 ), both of which were initially annotated as hypothetical/unknown, were separately characterized while this study was in progress and evidence was presented for each being involved in phy-regulated responses (see below). The fourth gene, At2g37970 (designated SOUL1 here for convenience), also initially unannotated, encodes a SOUL family putative heme binding protein not previously implicated in phy signaling. Interestingly, the elf4 and pil1 mutants exhibit reduced sensitivity to light signals, whereas pks1 and soul1 mutants show enhanced sensitivity. In addition to these four genes, two of the other early-response genes identified by microarray ( Monte et al., 2004 ; Tepperman et al., 2006 ), HY5 and SPA1 , had been identified even earlier in forward genetics screens as having important roles in phy-regulated seedling deetiolation ( Koornneef et al., 1980 ; Hoecker et al., 1998 , 1999 ). In addition, another early-response gene identified by microarray ( Tepperman et al., 2001 , 2004 ; Monte et al., 2004 ), PRR9 , was more recently identified as important for optimal sensitivity of responses to red and blue light ( Eriksson et al., 2003 ). It is notable that the phenotypes caused by disruption of the four new loci, elf4 , pil1 , pks1 , and soul1 , are relatively moderate compared with the most extreme hyposensitive and hypersensitive phenotypes observed for phyB -null mutants and phyB overexpressors, respectively ( Figure 3 ). This observation indicates that although these loci appear to have a level of functional importance in light-induced deetiolation, none is singularly essential sufficiently early in the phy-induced signaling cascade to pleiotropically affect the full development of the deetiolation response.

The subset of six new mutant loci, lhy-101 , sth ( zf3 ), dof , cbf3 , At2g42870 , and elip1 , that exhibit statistically significant hypersensitivity in hypocotyl inhibition in Rc light but no significant difference in cotyledon expansion (class III, Table 1 ), do not provide evidence of functional importance in early, central regulatory events in phy signaling. On the other hand, the possibility of organ-specific activity in hypocotyl cells cannot be ruled out. It is noteworthy that the data for the lhy-101 allele isolated in this work document a seedling-deetiolation phenotype for a monogenic loss-of function mutant at this locus, in contrast with previous studies that used an overexpressor allele ( Schaffer et al., 1998 ). This latter report showed that LHY overexpression results in hypocotyl hyposensitivity to Rc light ( Schaffer et al., 1998 ), the converse of what was observed for lhy-101 , strengthening the notion that LHY may indeed function in mediating phy signaling in these cells. A more recent study with a cca1 lhy double mutant under light/dark cycling conditions supports the role of LHY as a component of the central circadian oscillator ( Mizoguchi et al., 2005 ).

One of the seven class II mutants, cca1-like , displays enhanced cell expansion in both hypocotyl and cotyledon, whereas the other six class II mutants, ring-zfn , b-box type zf , sigE , put.ep1 , β -exp , and mate , display hypersensitive inhibition of cell expansion in both hypocotyl and cotyledon. These loci are either important to the normal cell expansion process in general or defects in these genes make the seedlings sensitive to the deleterious effects of light, such as through photooxidative damage to cellular components (at least for the six hypersensitive mutants). Care is needed in studies of potential photomorphogenic mutants to avoid interpreting short hypocotyls alone (without cotyledon measurements) in light-grown seedlings as evidence of a specific defect in the photoreceptor perception and signaling systems ( Quail, 2002a ).

Why do only a limited proportion of genes that respond rapidly to light signals through the phy system display clear seedling deetiolation phenotypes when disrupted ( Figure 5 )? The data show that only 7 genes (22%; including previously evaluated HY5 and SPA1 ) of the 32 genes identified by microarray analysis as phy-regulated early-response genes ( Tepperman et al., 2001 , 2004 , 2006 ; Monte et al., 2004 ) and investigated here significantly perturb global deetiolation when mutated ( Table 1 , Figure 5 ). These results with a temporally selected, relatively small subset of photoresponsive genes are consistent with the genome-wide functional profiling in yeast, in which only 7% or less of responsive genes were identified as functionally necessary for optimal growth under the relevant imposed environment ( Giaever et al., 2002 ). Possible explanations for these findings in our study include the following.

First, it is possible that the microarray-based expression measurements were not sufficiently reliable or that the induced changes are quantitatively functionally insignificant to the organism. Inspection of microarray expression data makes it clear that changes declared significantly different by statistical methods identify large numbers of genes for which these changes are quantitatively minor ( Monte et al., 2004 ; Tepperman et al., 2006 ). Although the initial selection criteria used here for the early-response genes imposed, in addition, a minimal quantitative change threshold (induction or repression) of twofold within 1 h of the start of the light signal, it is possible that these criteria are insufficiently stringent to identify genes with biologically meaningful light-induced changes in expression.

Second, it is possible that the rapid changes in expression measured after the initial transfer of seedlings from dark to light are transient and are only functionally important to an early transitional process that has no observable longer term impact on the overall deetiolation process under our conditions. This possibility might be consistent with the observations of Spalding and coworkers ( Parks and Spalding, 1999 ; Parks et al., 2001 ; Parks, 2003 ), who observed that phyA acts only transiently to inhibit hypocotyl growth upon initial exposure of seedlings to light and that this has little or no measurable impact on hypocotyl expansion determined in end point analyses after prolonged (4 to 5 d) growth under light conditions. On the other hand, defects in some such genes could have negative selective impact for seedlings germinated under natural conditions that are not apparent for seedlings germinated on agar plates in the laboratory.

Third, because we only examined the phenotypes at one fluence rate for each wavelength, we cannot rule out the possibility that a phenotype might be observed at another fluence rate. However, the fluence rates selected (Rc light, 7 μmol·m −2 ·s −1 ; FRc light, 2 μmol·m −2 ·s −1 ) were similar to those used for the microarray analyses and in our hands approach, but are not at, maximum effectiveness for the wild type; therefore, they are expected both to maximize the opportunity for the display of hyposensitive mutants and to leave sufficient latitude for the detection of hypersensitivity.

Fourth, it cannot be excluded that the light-induced changes for some genes are functionally irrelevant to the deetiolation process altogether, perhaps being an inevitable consequence of selection pressures on light-regulated events at other stages of the life cycle.

Finally, the commonly offered explanation for the lack of apparent phenotypes from reverse genetics disruption of target loci is functional redundancy ( Ostergaard and Yanofsky, 2004 ; Cutler and McCourt, 2005 ; Kuepfer et al., 2005 ). The assessment of this possibility can in principle be approached through systematic construction of higher order combinations of candidate mutant loci. However, in this case, it is not immediately apparent which of the coordinately light-regulated loci that define the molecular phenotype are likely to function redundantly in determining the visible deetiolation phenotype. Although some of the 78% of early-response genes that do not exhibit a coordinated seedling-deetiolation phenotype when mutated are potentially sequence-related (e.g., CCA1 and LHY ; ATHB4 and HAT4 ), the majority do not appear to be so related and therefore are more difficult to reconcile with any postulated redundant molecular or biochemical activity. One possibility, in the case of the transcriptional regulators, is that multiple different classes of these factors might mediate the phy-regulated expression of the same single target genes through the presence of multiple cognate cis elements for these factors arrayed in the individual promoters of each of these genes. Thus, compared with studies focused on multigene families in which sequence-relatedness may be a more reliable predictor of functional redundancy, identifying targets for mutant combinations here offers additional challenges. On the other hand, it is possible that broadening or refinement of the parameters used to assess the visible deetiolation phenotype might reveal previously undetected defects for some of these loci. Furthermore, a systematic assessment of light-regulated gene expression in the mutants might reveal defects not visible at the phenotypic level.

Despite the relatively low proportion of genes targeted here that were found to display insertional mutant phenotypes, some new loci not identified previously in forward genetics screens as involved in phy-regulated deetiolation were found. Although mutations in most of these loci resulted in relatively modest phenotypes, we nevertheless consider that the results indicate that the strategy has merit, particularly in the quest to define the full complement of genes involved in light-regulated development.

This point is exemplified by ELF4 and PIL1 . Each was targeted for potential functional involvement in deetiolation at the outset of this analysis, and each was also subsequently identified in separate studies as involved in other phy-regulated processes, namely, flowering ( Doyle et al., 2002 ) and shade avoidance ( Salter et al., 2003 ). The additional pil1 mutant analysis here provides evidence of further functional roles for PIL1 in the phyB-imposed suppression of petiole extension during vegetative development and the induction of floral development. Importantly, in addition, the data also suggest that PIL1 may have a regulatory role in the phy-induced expression of the PRR9 gene. The evidence shows that PIL1 is necessary for normal maximum PRR9 expression upon first exposure of dark-grown seedlings to light. Together with a comparison of the temporal patterns of rapid light-induced changes in PIL1 and PRR9 expression ( Figure 4C ), these data suggest the possibility of transcriptional regulation of PRR9 by PIL1 in a light-modulated manner. PIL1 expression is high in dark-grown seedlings but is rapidly repressed within 60 min to a new steady state basal level upon light exposure. By contrast, PRR9 is rapidly induced in the light, but only transiently, with a sharp maximum 30 to 45 min after initial exposure ( Figures 4B and 4C ). A potential model consistent with these data would suggest that PIL1 is necessary for light-induced PRR9 expression. According to this model, the light-induced expression of PRR9 would be supported by the initially high levels of PIL1 at the dark-to-light transition, but because PIL1 itself is rapidly negatively regulated by the light signal, PIL1 protein abundance would decrease rapidly, thereby reversing the initial increase in PRR9 expression and rendering the overall temporal pattern of induction transient. Because PRR9 is proposed to have a function in circadian regulation ( Eriksson et al., 2003 ) and PIL1 is proposed to interact with the circadian clock in regulating shade avoidance ( Salter et al., 2003 ), these data may provide insight into the possible signaling mechanisms involved. It is important to note that the absence of significant effects of pil1 mutations on the light-induced initiation of oscillations of the central oscillator components, CCA1 , LHY , and TOC1 (see Supplemental Figure 10 online), which occurs upon first exposure of etiolated seedlings to light ( Kikis et al., 2005 ), provides evidence against the possibility that PRR9 is regulated indirectly by PIL1 via the circadian clock ( Harmer and Kay, 2005 ).

In summary, the data presented here document the functional profiling of a plant signaling system responsive to an external environmental stimulus, based on genome-scale identification of responsive genes in a transcriptional network by microarray expression analysis coupled with targeted reverse genetics assessment of the functional necessity of each gene to the induced morphogenic response. The evidence from these monogenic mutants shows that only a low proportion of genes that respond rapidly to phy-mediated light signals during the initiation of seedling deetiolation appear to be functionally necessary for the normal development of the visibly measurable deetiolation phenotype, consistent with analogous genome-wide studies in yeast. Although functional redundancy may provide an explanation for the majority of the rapidly light-responsive genes that appear to be functionally unnecessary for deetiolation, this possibility is difficult to reconcile with the absence of any obvious sequence-relatedness among most of these genes. Alternative possible explanations include a lack of dependence of full deetiolation on initial light-induced expression changes of some genes, organ- or cell-specific functional activity of some genes, and statistically defined expression changes that are either quantitatively too small to be important to the phenotype or simply functionally irrelevant to the organism. These findings may be relevant to other plant studies using reverse genetics strategies to determine gene function in Arabidopsis .
