---
pmid: '30531922'
title: The molecular landscape of glioma in patients with Neurofibromatosis 1.
authors:
- D'Angelo F
- Ceccarelli M
- Tala
- Garofano L
- Zhang J
- Frattini V
- Caruso FP
- Lewis G
- Alfaro KD
- Bauchet L
- Berzero G
- Cachia D
- Cangiano M
- Capelle L
- de Groot J
- DiMeco F
- Ducray F
- Farah W
- Finocchiaro G
- Goutagny S
- Kamiya-Matsuoka C
- Lavarino C
- Loiseau H
- Lorgis V
- Marras CE
- McCutcheon I
- Nam DH
- Ronchi S
- Saletti V
- Seizeur R
- Slopis J
- Suñol M
- Vandenbos F
- Varlet P
- Vidaud D
- Watts C
- Tabar V
- Reuss DE
- Kim SK
- Meyronet D
- Mokhtari K
- Salvador H
- Bhat KP
- Eoli M
- Sanson M
- Lasorella A
- Iavarone A
journal: Nat Med
year: '2019'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC6857804
doi: 10.1038/s41591-018-0263-8
---

# The molecular landscape of glioma in patients with Neurofibromatosis 1.
**Authors:** D'Angelo F, Ceccarelli M, Tala, Garofano L, Zhang J, Frattini V, Caruso FP, Lewis G, Alfaro KD, Bauchet L, Berzero G, Cachia D, Cangiano M, Capelle L, de Groot J, DiMeco F, Ducray F, Farah W, Finocchiaro G, Goutagny S, Kamiya-Matsuoka C, Lavarino C, Loiseau H, Lorgis V, Marras CE, McCutcheon I, Nam DH, Ronchi S, Saletti V, Seizeur R, Slopis J, Suñol M, Vandenbos F, Varlet P, Vidaud D, Watts C, Tabar V, Reuss DE, Kim SK, Meyronet D, Mokhtari K, Salvador H, Bhat KP, Eoli M, Sanson M, Lasorella A, Iavarone A
**Journal:** Nat Med (2019)
**DOI:** [10.1038/s41591-018-0263-8](https://doi.org/10.1038/s41591-018-0263-8)
**PMC:** [PMC6857804](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6857804/)

## Abstract

1. Nat Med. 2019 Jan;25(1):176-187. doi: 10.1038/s41591-018-0263-8. Epub 2018 Dec
 10.

The molecular landscape of glioma in patients with Neurofibromatosis 1.

D'Angelo F(#)(1)(2), Ceccarelli M(#)(2)(3), Tala(1), Garofano L(1)(2), Zhang 
J(1), Frattini V(1), Caruso FP(2)(3), Lewis G(1), Alfaro KD(4), Bauchet L(5), 
Berzero G(6), Cachia D(7)(8), Cangiano M(2), Capelle L(9), de Groot J(10), 
DiMeco F(11)(12)(13), Ducray F(14), Farah W(15), Finocchiaro G(16), Goutagny 
S(17), Kamiya-Matsuoka C(10), Lavarino C(18), Loiseau H(19), Lorgis V(20), 
Marras CE(21), McCutcheon I(10), Nam DH(22)(23), Ronchi S(6), Saletti V(24), 
Seizeur R(25), Slopis J(10), Suñol M(26), Vandenbos F(27), Varlet P(28)(29), 
Vidaud D(30), Watts C(31), Tabar V(32), Reuss DE(33)(34), Kim SK(35), Meyronet 
D(36), Mokhtari K(6), Salvador H(37), Bhat KP(10), Eoli M(16), Sanson M(6), 
Lasorella A(38)(39)(40), Iavarone A(41)(42)(43).

Author information:
(1)Institute for Cancer Genetics, Columbia University Medical Center, New York, 
NY, USA.
(2)BIOGEM Istituto di Ricerche Genetiche 'G. Salvatore', Ariano Irpino, Italy.
(3)Department of Science and Technology, Università degli Studi del Sannio, 
Benevento, Italy.
(4)The University of Texas M.D. Anderson Cancer Center John Mendelsohn Faculty 
Center (FC7.3025) - Neuro-Oncology - Unit 0431, Houston, TX, USA.
(5)Department of Neurosurgery, Gui de Chauliac Hospital, Montpellier University 
Medical Center, Montpellier, France.
(6)Sorbonne Universités UPMC Université Paris 06, UMR S 1127, Inserm U 1127, 
CNRS UMR 7225, ICM, APHP, Paris, France.
(7)Department of Neuro-Oncology, Medical University of South Carolina, 
Charleston, SC, USA.
(8)Department of Neurosurgery, Medical University of South Carolina, Charleston, 
SC, USA.
(9)AP-HP, Hôpital de la Pitié-Salpêtrière, Service de Neurochirurgie, Paris, 
France.
(10)The University of Texas M.D. Anderson Cancer Center, Houston, TX, USA.
(11)Department of Neurological Surgery, Carlo Besta Neurological Institute, 
Milan, Italy.
(12)Department of Pathophysiology and Transplantation, University of Milan, 
Milan, Italy.
(13)Hunterian Brain Tumor Research Laboratory CRB2 2M41, Baltimore, MD, USA.
(14)Service de Neuro-Oncologie, Hospices Civils de Lyon, Université Claude 
Bernard Lyon 1, Department of Cancer Cell Plasticity, Cancer Research Center of 
Lyon, INSERM U1052, CNRS UMR5286, Lyon, France.
(15)Department of Neurosurgery, CHU, Dijon, France.
(16)Unit of Molecular Neuro-Oncology, IRCCS Foundation, Carlo Besta Neurological 
Institute, Milan, Italy.
(17)Service de Neurochirurgie, Hôpital Beaujon, Assistance Publique-Hôpitaux de 
Paris, Clichy, France.
(18)Developmental Tumor Laboratory, Fundación Sant Joan de Déu, Barcelona, 
Spain.
(19)Department of Neurosurgery, Bordeaux University Hospital. Labex TRAIL 
(ANR-10-LABX-57). EA 7435 - IMOTION Bordeaux University, Bordeaux, France.
(20)Department of Medical Oncology, Centre GF Leclerc, Dijon, France.
(21)Pediatric Neurosurgery Unit, Department of Neuroscience and 
Neurorehabilitation, Bambino Gesù Children's Hospital, Rome, Italy.
(22)Department of Neurosurgery, Samsung Medical Center, Sungkyunkwan University 
School of Medicine, Seoul, Republic of Korea.
(23)Department of Health Sciences and Technology, SAIHST, Sungkyunkwan 
University, Seoul, Republic of Korea.
(24)Developmental Neurology Unit, IRCCS Foundation, Carlo Besta Neurological 
Institute, Milan, Italy.
(25)Service de Neurochirurgie, Hôpital de la Cavale Blanche, CHRU de Brest, 
Université de Brest, Brest, France.
(26)Department of Pathology, Hospital Sant Joan de Déu, Barcelona, Spain.
(27)Central Laboratory of Pathology, Pasteur I University Hospital, Nice, 
France.
(28)Department of Neuropathology, Sainte-Anne Hospital, Paris, France.
(29)IMA-Brain, Inserm U894, Institute of Psychiatry and Neuroscience of Paris, 
Paris, France.
(30)EA7331, Université Paris Descartes, France; Service de Génétique et Biologie 
Moléculaires, Hôpital Cochin, AP-HP, Paris, France.
(31)Institute of Cancer and Genomic Sciences University of Birmingham Edgbaston, 
Birmingham, United Kingdom.
(32)Department of Neurosurgery, Memorial Sloan Kettering Cancer Center, New 
York, NY, USA.
(33)Clinical Cooperation Unit Neuropathology, German Cancer Research Center 
(DKFZ), Heidelberg, Germany.
(34)Department of Neuropathology, Institute of Pathology, Heidelberg University 
Hospital, Heidelberg, Germany.
(35)Division of Pediatric Neurosurgery, Seoul National University Children's 
Hospital, Seoul National University College of Medicine, Seoul, Republic of 
Korea.
(36)Centre de Pathologie Et Neuropathologie Est Hospices Civils de Lyon, Lyon, 
France.
(37)Pediatric Oncology Unit, Hospital Sant Joan de Déu, Esplugues, Barcelona, 
Spain.
(38)Institute for Cancer Genetics, Columbia University Medical Center, New York, 
NY, USA. al2179@cumc.columbia.edu.
(39)Department of Pediatrics, Columbia University Medical Center, New York, NY, 
USA. al2179@cumc.columbia.edu.
(40)Department of Pathology and Cell Biology, Columbia University Medical 
Center, New York, NY, USA. al2179@cumc.columbia.edu.
(41)Institute for Cancer Genetics, Columbia University Medical Center, New York, 
NY, USA. ai2102@cumc.columbia.edu.
(42)Department of Pathology and Cell Biology, Columbia University Medical 
Center, New York, NY, USA. ai2102@cumc.columbia.edu.
(43)Department of Neurology, Columbia University Medical Center, New York, NY, 
USA. ai2102@cumc.columbia.edu.
(#)Contributed equally

Neurofibromatosis type 1 (NF1) is a common tumor predisposition syndrome in 
which glioma is one of the prevalent tumors. Gliomagenesis in NF1 results in a 
heterogeneous spectrum of low- to high-grade neoplasms occurring during the 
entire lifespan of patients. The pattern of genetic and epigenetic alterations 
of glioma that develops in NF1 patients and the similarities with sporadic 
glioma remain unknown. Here, we present the molecular landscape of low- and 
high-grade gliomas in patients affected by NF1 (NF1-glioma). We found that the 
predisposing germline mutation of the NF1 gene was frequently converted to 
homozygosity and the somatic mutational load of NF1-glioma was influenced by age 
and grade. High-grade tumors harbored genetic alterations of TP53 and CDKN2A, 
frequent mutations of ATRX associated with Alternative Lengthening of Telomere, 
and were enriched in genetic alterations of transcription/chromatin regulation 
and PI3 kinase pathways. Low-grade tumors exhibited fewer mutations that were 
over-represented in genes of the MAP kinase pathway. Approximately 50% of 
low-grade NF1-gliomas displayed an immune signature, T lymphocyte infiltrates, 
and increased neo-antigen load. DNA methylation assigned NF1-glioma to LGm6, a 
poorly defined Isocitrate Dehydrogenase 1 wild-type subgroup enriched with ATRX 
mutations. Thus, the profiling of NF1-glioma defined a distinct landscape that 
recapitulates a subset of sporadic tumors.

DOI: 10.1038/s41591-018-0263-8
PMCID: PMC6857804
PMID: 30531922 [Indexed for MEDLINE]

Conflict of interest statement: Competing interests The authors declare no 
competing interests.

## Full Text

Abstract

Neurofibromatosis type 1 (NF1) is a common tumor predisposition syndrome in which glioma is one of the prevalent tumors. Gliomagenesis in NF1 results in a heterogeneous spectrum of low- to high-grade neoplasms occurring during the entire lifespan of patients. The pattern of genetic and epigenetic alterations of glioma that develops in NF1 patients and the similarities with sporadic glioma remain unknown. Here, we present the molecular landscape of low- and high-grade gliomas in patients affected by NF1 (NF1-glioma). We found that the predisposing germline mutation of the NF1 gene was frequently converted to homozygosity and the somatic mutational load of NF1-glioma was influenced by age and grade. High-grade tumors harbored genetic alterations of TP53 and CDKN2A, frequent mutations of ATRX associated with Alternative Lengthening of Telomere, and were enriched in genetic alterations of transcription/chromatin regulation and PI3 kinase pathways. Low-grade tumors exhibited fewer mutations that were over-represented in genes of the MAP kinase pathway. Approximately 50% of low-grade NF1-gliomas displayed an immune signature, T lymphocyte infiltrates, and increased neo-antigen load. DNA methylation assigned NF1-glioma to LGm6, a poorly defined Isocitrate Dehydrogenase 1 wild-type subgroup enriched with ATRX mutations. Thus, the profiling of NF1-glioma defined a distinct landscape that recapitulates a subset of sporadic tumors.

Discussion

Here, we reported the landscape of genetic and epigenetic alterations of gliomas occurring in NF1 patients. It is important to consider that the inaccessible brain tumor location, the relatively benign behavior of brain lesions, the comorbidities, and the neurological deterioration associated with surgical intervention most frequently argue against surgery as choice of treatment for glioma patients with NF1 13 . Therefore, the collection of glioma samples analyzed in this study represents the selected subset of tumor lesions that undergo surgical resection in NF1 patients. Nevertheless, a comprehensive analysis of the complete spectrum of glioma grades throughout the lifespan of NF1 patients has allowed us to follow NF1 gliomagenesis and identify the genetic modules and the expression signatures that distinguish low-from high-grade tumors. We found that abundant infiltrates of activated T lymphocytes and mutation-derived neoantigens characterize a subset of low-grade gliomas, whereas high-grade tumors exhibit frequent mutations of ATRX typically co-occurring with alterations of TP53 and CDKN2A. We also classified gliomas occurring in the context of the NF1 syndrome within a particular methylation sub-group of sporadic gliomas, the LGm6, that recapitulates mutational and epigenetic profiles of NF1-glioma. The discovery that ATRX mutations drive aggressiveness in NF1-glioma prompted re-evaluation of the mutational and clinical features of the sporadic glioma counterpart (LGm6), leading to a more accurate classification of the sporadic tumors that cluster into this group.

The NF1 tumor-predisposing syndrome is associated with a heterogeneous pattern of glioma in children and adults 57 . Therefore, the dissection of the molecular landscape of glioma in NF1 patients required a comprehensive molecular study of brain tumors that could not be limited to a particular age and/or glioma grade. We observed several features that differentiate low-grade gliomas— which are relatively more common in children—from the high-grade tumors that instead predominate in adults with NF1. Pediatric low-grade NF1-gliomas exhibit a very low overall mutation rate in comparison to high-grade tumors, with few recurrent somatic mutations. The only set of recurrently mutated genes in low-grade NF1-glioma are genes involved in the MAPK pathway, thus recapitulating the genetic features of sporadic pilocytic astrocytoma. Conversely, the mutation burden of high-grade glioma in NF1 patients was higher and was characterized by recurrent alterations composing a genetic module that includes loss-of-function events targeting ATRX, TP53, and CDKN2A. Loss of ATRX in high-grade NF1-glioma is unique when considered within the genetic contexts associated with ATRX mutations in sporadic gliomas, in which they are typically associated with mutations of H3.3 in children or IDH1 in adults. The inactivating mutations of the ATRX gene result in loss of a functional ATRX protein with at least two important mechanistic consequences: development of the ALT phenotype and activation of a transcriptional/chromatin remodeling gene expression signature in ATRX mutant NF1-glioma.

Previous work reported that mouse models of low-grade glioma sustained by loss of NF1 manifest specific alterations of the immune microenvironment 58 . The analysis of human NF1-glioma confirms and extends this notion. We found that, compared with high-grade tumors, the gene expression signature more strongly enriched in low-grade NF1-glioma is an immune signature including several effector lymphocyte categories. This has been confirmed by the finding that the high-immune group of low-grade NF1-glioma contains both rich infiltrates of T lymphocytes, some of which are endowed with cytolytic activity, and mutation-derived neoantigens that exhibited enhanced HLA binding. The reduced DNA methylation of immune genes expressed in low-grade/high-immune NF1-glioma is consistent with previous studies in which reduced methylation and increased expression of immune genes in human tumors was linked to tumor infiltration by lymphocytes characterized by demethylated and transcriptionally active genes involved in T lymphocyte functions that, on the contrary, were highly methylated and transcriptionally repressed in cancer cells 51 , 52 . Taken together, our findings suggest that the long indolent course of low-grade NF1-gliomas that rarely progress to high-grade disease 9 , 13 , 57 may be preserved by the checks imposed by the adaptive immunity acquired by some low-grade tumors.

DNA methylation profiling classified NF1-gliomas within the LGm6 IDH wild-type cluster of sporadic gliomas from our recent panglioma TCGA study 10 . Prompted by the discovery that high-grade IDH wild-type gliomas in NF1 patients harbor frequent mutations of ATRX, we re-analyzed the LGm6 subgroup of sporadic tumors. We found that the original definition of pilocytic astrocytoma-like, which combined histological grade II and grade III tumors in the LGm6 cluster, is not an accurate representation of the biology of these tumors. Indeed, mutation pattern and clinical outcome of grade II-LGm6 gliomas diverge markedly from those of grade III-LGm6 tumors harboring ATRX mutations, which are more similar to LGm6-GBM. This finding calls for a re-evaluation of the guidelines for the classification of IDH wild-type gliomas, whereby the LGm6 group exhibits a larger heterogeneity than previously described and requires analysis of the ATRX status for the accurate qualification of the clinical features. As previous studies have shown that loss of ATRX increases sensitivity to DNA-damaging agents 59 , 60 , ATRX mutations may represent a point of therapeutic intervention for high-grade NF1-gliomas and LGm6 sporadic gliomas. Our study also argues that, similar to NF1-glioma, the comparative genomic analysis of other NF1-associated neoplasms with the sporadic counterpart may reveal distinct pathogenic mechanisms that could have therapeutic implications.
