---
pmid: '21753751'
title: The ELF4-ELF3-LUX complex links the circadian clock to diurnal control of hypocotyl
  growth.
authors:
- Nusinow DA
- Helfer A
- Hamilton EE
- King JJ
- Imaizumi T
- Schultz TF
- Farré EM
- Kay SA
journal: Nature
year: '2011'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC3155984
doi: 10.1038/nature10182
---

# The ELF4-ELF3-LUX complex links the circadian clock to diurnal control of hypocotyl growth.
**Authors:** Nusinow DA, Helfer A, Hamilton EE, King JJ, Imaizumi T, Schultz TF, Farré EM, Kay SA
**Journal:** Nature (2011)
**DOI:** [10.1038/nature10182](https://doi.org/10.1038/nature10182)
**PMC:** [PMC3155984](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3155984/)

## Abstract

1. Nature. 2011 Jul 13;475(7356):398-402. doi: 10.1038/nature10182.

The ELF4-ELF3-LUX complex links the circadian clock to diurnal control of 
hypocotyl growth.

Nusinow DA(1), Helfer A, Hamilton EE, King JJ, Imaizumi T, Schultz TF, Farré EM, 
Kay SA.

Author information:
(1)Section of Cell & Developmental Biology, Division of Biological Sciences, 
University of California San Diego 9500 Gilman Drive, La Jolla, California 
92093-0130, USA.

The circadian clock is required for adaptive responses to daily and seasonal 
changes in environmental conditions. Light and the circadian clock interact to 
consolidate the phase of hypocotyl cell elongation to peak at dawn under diurnal 
cycles in Arabidopsis thaliana. Here we identify a protein complex (called the 
evening complex)--composed of the proteins encoded by EARLY FLOWERING 3 (ELF3), 
ELF4 and the transcription-factor-encoding gene LUX ARRHYTHMO (LUX; also known 
as PHYTOCLOCK 1)--that directly regulates plant growth. ELF3 is both necessary 
and sufficient to form a complex between ELF4 and LUX, and the complex is 
diurnally regulated, peaking at dusk. ELF3, ELF4 and LUX are required for the 
proper expression of the growth-promoting transcription factors encoded by 
PHYTOCHROME INTERACTING FACTOR 4 (PIF4) and PIF5 (also known as PHYTOCHROME 
INTERACTING FACTOR 3-LIKE 6) under diurnal conditions. LUX targets the complex 
to the promoters of PIF4 and PIF5 in vivo. Mutations in PIF4 and/or PIF5 are 
epistatic to the loss of the ELF4-ELF3-LUX complex, suggesting that regulation 
of PIF4 and PIF5 is a crucial function of the complex. Therefore, the evening 
complex underlies the molecular basis for circadian gating of hypocotyl growth 
in the early evening.

DOI: 10.1038/nature10182
PMCID: PMC3155984
PMID: 21753751 [Indexed for MEDLINE]

## Full Text

The circadian clock is an endogenous molecular oscillator with a period of ∼24 hours that is present nearly ubiquitously1. In plants, multiple interlocking transcriptional feedback loops contribute to the robust architecture of this oscillator network3. The clock functions to enable anticipation of diurnal rhythmic environmental changes, allowing for the optimal phasing of molecular, physiological and behavioral responses to specific times of day2. Plant growth is a physiological response that is controlled by both the clock and changes in light conditions, and under diurnal growth conditions, maximal plant growth occurs at the end of night4-7.

EARLY FLOWERING 3 and EARLY FLOWERING 4 were first identified in genetic screens for photoperiodism mutants and were found to regulate circadian rhythms8-10,14. ELF3 and ELF4 encode nuclear, plant-specific proteins with no known functional domains9,10,15,16. LUX ARRHYTHMO (also PHYTOCLOCK 1) is a single-MYB domain SHAQYF-type GARP transcription factor identified in a genetic screen for long hypocotyl mutants and aberrant circadian-regulated gene expression11,12. elf3, elf4 and lux share multiple phenotypes, including an arrhythmic circadian oscillator, abnormal hypocotyl growth under diurnal cycles and early flowering4,8-12,14,15,17. ELF3, ELF4 and LUX showed similar expression profiles in microarray experiments (Figure S1; http://diurnal.cgrb.oregonstate.edu/18,19), and these expression profiles were confirmed by quantitative RT-PCR analysis in diurnal and circadian conditions (Figure 1a).

The similarities in expression patterns and phenotypes prompted us to test whether these proteins could interact. Using a yeast two-hybrid assay, we found that ELF4 interacted with ELF3 (Figure 1b). Also, when LUX-fragments were used as baits (full-length LUX showed auto-activation, data not shown), ELF3 showed an interaction with LUX-C (a.a. 144-324), containing the DNA-binding domain of LUX11,12,20, but not with LUX-N (a.a. 1-143) (Figure 1c). ELF4 and LUX did not interact in any combination (Figure 1b and c). As ELF3 could interact independently with either ELF4 or LUX, we hypothesized that ELF3 might form a complex between these two proteins. To test this, ELF3 was used in a yeast three-hybrid system with ELF4-GAL4-DBD and/or LUX-GAL4-AD. Activation of the reporter was observed only when all three proteins were present, suggesting that ELF3 was sufficient to bridge an interaction between ELF4 and LUX (Figure 1d).

Next, we tested whether ELF4, ELF3 and LUX interact in vivo. Antibodies were developed against ELF3 and LUX and an ELF4∷ELF4-HA construct was introduced into the elf4-2 mutant21. The ELF4-HA protein is likely functional, as we identified transformants that rescued the hypocotyl length (Figure S2a) and circadian CHLOROPHYLL A/B BINDING PROTEIN∷LUCIFERASE (CAB2∷LUC) rhythmicity, albeit with a shorter period (Figure S2b-d). We then asked whether ELF4-HA could co-immunoprecipitate endogenous ELF3 and/or LUX at Zeitgeber time (ZT) 12 (Figure 2a). We found that ELF4-HA could co-immunoprecipitate both ELF3 and LUX (Figure 2a and S2f). The experiments in yeast suggested that ELF3 bridges an interaction between ELF4 and LUX, and that ELF3 would be necessary for the co-immunoprecipitation of LUX by ELF4-HA. To test this, we introduced elf3-1 into the ELF4∷ELF4-HA elf4-2 and immunoprecipitated ELF4-HA. Although similar amounts of ELF4 and LUX were present in the extracts, LUX did not co-immunoprecipitate with ELF4-HA (Figure 2a). These results show that ELF3 is necessary for tripartite complex formation with ELF4 and LUX in vivo. Furthermore, hypocotyl length in elf3-1 elf4-3 and elf3-1 lux-4 double mutants grown under 12 h light: 12 h dark (12L:12D) did not exhibit additive effects over elf3 (Figure S3). These results are consistent with the hypothesis that ELF3, ELF4 and LUX function together as a complex to regulate common pathways.

Since ELF4, ELF3 and LUX mRNA levels oscillate and peak with a similar phase, we analyzed the dynamics of the protein levels under diurnal cycles. ELF4∷ELF4-HA tissue was harvested every 4 hours, starting at ZT12 under 12L:12D cycles and then after transferring to constant light at ZT0 the following day. ELF3, LUX and ELF4-HA peaked at ZT12, declined during the night, reached a trough between ZT0-ZT4, and then increased again (Figure 2b,S4). The levels of all three proteins remained elevated into the subjective dark period relative to their respective time points in the dark and the protein peak was now shifted to the middle of the subjective night (Figure 2b). Comparable results were observed for ELF3 and LUX levels in wild-type seedlings (Figure S5). To assay for time-dependent formation of the ELF4-ELF3-LUX “Evening Complex” (EC), ELF4-HA was immunoprecipitated from the diurnal samples. Formation of the EC followed the same pattern as that of its composite parts, suggesting that they would associate when present (Figure 2b).

Photoperiodic control of flowering and growth is compromised in elf3, elf4 and lux mutants4,8-12,14,15,17,22. To determine how ELF4, ELF3 and LUX respond to altered photoperiods, we analyzed the levels and formation of the EC in plants grown under short (8L:16D) and long days (16L:8D). Peak levels of ELF4, ELF3 and LUX followed their respective RNA profiles under different photoperiods (Figure 2c and d, S4), similar to previous reports9,10,15,22. Complex formation was also sensitive to photoperiod, peaking earlier in short days compared to long days (Figure 2c and d).

To investigate the molecular role of the EC, we focused on the diurnal hypocotyl growth phenotype shared by all mutants5,8-11,20. Previous work demonstrated that the bHLH transcription factors PHYTOCHROME-INTERACTING FACTOR 4 (PIF4) and PIF5 (also known as PHYTOCHROME-INTERACTING FACTOR LIKE 6) are critical components that determine the hypocotyl elongation rate in seedlings, and that both genes act downstream of light- and clock-signaling pathways4,6,7,13,23. Expression of PIF4 and PIF5 was nearly anti-phasic to the EC under different photocycles (Figure S6). This raised the possibility that the EC may be repressing the transcription of PIF4 and PIF5, consistent with recent reports that ELF3 and LUX act as transcriptional repressors in the circadian clock20,24. The levels of PIF4 and PIF5 are elevated in elf3-1, elf4-2 and lux-4, particularly during the early evening (Figure 3a). Recent work demonstrated that the addition of an activation domain to LUX (LUX-VP64) induced a neomorphic hypocotyl elongation phenotype20 and we found that PIF4/5 expression levels were increased in this background (Figure S7). These results, as well as the presence of full consensus LUX binding sites (LBS)20 in the 5′-UTR of both PIF4 and PIF5 (Figure 3b), suggested that LUX may participate directly in the modulation of PIF4/5 expression. Indeed, LUX could directly bind the PIF4 and PIF5 promoters in yeast, and LUX binding to the PIF5 promoter (-481/+13 bp) was lost when the consensus LBS was mutated (Figure 3c).

To determine if members of the EC were bound to the PIF4 and PIF5 promoters in vivo, chromatin immunoprecipitations (ChIP) was performed in LUX∷LUX-GFP transgenic lines followed by amplification of PIF4 and PIF5 promoter sequences. These experiments revealed in vivo binding to the LBS in the PIF4 and PIF5 promoters, but not to control sequences in their coding regions or in the UBIQUITIN10 (UBQ10) promoter (Figure 3d). The formation of the EC (Figure 2) suggested that all members might participate in the regulation of PIF4/5 expression; therefore, we performed similar ChIP experiments for ELF3 and ELF4-HA. We found that ELF3 and ELF4-HA showed specific enrichment at PIF4 and PIF5 promoter sequences that were also bound by LUX (Figure 3e-f). Additionally, ELF3 ChIP experiments performed at the trough of EC levels (ZT2) showed a lower specific enrichment relative to ZT14 (Figure S8).

The localization pattern of the EC members on the PIF4/5 promoters suggested that the transcription factor LUX might be responsible for recruitment. ELF3 ChIP experiments in lux-4 seedlings demonstrated a lower, but incomplete loss of recruitment of ELF3 to the PIF4/5 promoters (Figure S9). Previous work identified a MYB transcription factor highly similar to LUX, named NOX11,12,20. NOX binds to similar sequences as LUX in yeast20 and could also form a complex with ELF4 and ELF3 (Figure S10a). We generated an artificial microRNA transgenic line that would simultaneously reduce the levels of both NOX and LUX (LUX/NOX ami), designed using a web-based amiRNA algorithm (http://wmd3.weigelworld.org/cgi-bin/webapp.cgi)25,26. LUX protein and NOX expression levels are reduced in this line (Figure S10b and c), which showed similar defects in circadian rhythms to lux-4 (Figure S10e and f); however, we observed an increase in hypocotyl length and PIF4/5 expression level compared to lux-4 (Figure S10d and g). When ELF3 ChIP was performed in the LUX/NOX ami line, we observed loss of ELF3 signal at the PIF4 and PIF5 promoters (Figure 3g). ELF3 was still present in extracts from these plants (Figure S10c), suggesting that recruitment of ELF3 (and therefore the EC) is mediated by both LUX and NOX.

Previous reports showed that ectopic overexpression of the MYB transcription factors CIRCADIAN CLOCK ASSOCIATED 1 (in CCA1-OX) or LATE ELONGATED HYPOCOTYL (in lhy-1) resulted in phenotypes similar to elf3, elf4 or lux (Figure 3a and S11)4-6. As CCA1 and LHY forms a complex that controls the expression of evening element containing genes27, such as ELF4 and LUX11, 28, the misexpression of PIF4 and PIF5 seen in CCA1-OX or lhy-1 could be a result of EC misregulation. Therefore, we analyzed the expression of ELF4, ELF3 and LUX in the lhy-1 background using the DIURNAL database18, 19. We found that ELF4 is clamped low, while ELF3 and LUX are shifted 4 and 12 hours later, respectively (Figure S11). These results are consistent with the circadian clock playing a critical role in the proper expression and phasing of the EC proteins.

If improper regulation of PIF4 and PIF5 underlies the hypocotyl growth defects observed in the EC mutants, then loss of PIF4 and PIF5 should be epistatic to loss of the EC. To test this, we introduced pif4 and pif5 mutant alleles into the elf3-2 mutant background, since mutating ELF3 caused dissolution of the EC (Figure 2a). Loss of PIF4 or PIF5 additively mitigated the hypocotyl length defect elf3-2 (Figure 4a and b), indicating that the hypocotyl phenotypes of EC mutants are mainly caused by misexpression of PIF4 and PIF5. In addition, loss of PIF4 or/and PIF5 did not restore circadian rhythms in an elf3 background (Figure S12), consistent with PIF4 and PIF5 being clock outputs that do not feedback into the oscillator4.

To summarize, we have identified a novel multi-protein complex that directly links the circadian clock to diurnal regulation of hypocotyl growth. The ELF4-ELF3-LUX complex is regulated by the clock and light (Figure 1a, 2b-d) and represses the expression of PIF4 and PIF5 in the early evening (Figure 4c). This is combined with light-regulated turnover of PIF4/5 to permit maximal hypocotyl growth at dawn in diurnal conditions4,13 (Figure 4c). ELF3 is necessary and sufficient to bring together ELF4 and LUX to form a complex (Figure 1d and 2a), providing a mechanistic framework to understand their shared phenotypes in regulating circadian rhythms, growth and flowering. The role of ELF3 as an adaptor protein is similar to its previously described capacity to modulate GIGANTEA levels through association with CONSTITUTIVELY PHOTOMORPHOGENIC 1 to regulate flowering and circadian rhythms22. The EC is composed of multiple proteins known to regulate signaling from the environment5,9-12,14-17,20,22,24,28; therefore, elucidating Evening Complex function will ultimately contribute to understanding how the clock gates biochemical, physiological and developmental outputs.

Methods SummaryAll wild-type, mutant and transgenic lines were in the Arabidopsis thaliana ecotype Columbia-0 (Col-0). All transgenic and mutant lines were brought to homozygosity before use. The procedures for Arabidopsis husbandry; yeast one-hybrid, two-hybrid, and three-hybrid analyses; biolumenscent imaging; immunoprecipitations; chromatin immunoprecipitations; and hypocotyl measurements were as described previously20,29,30, with modifications detailed in Methods. In all growth chambers, light was supplied at 80 μmol m-2 sec-1 by cool-white fluorescent bulbs at 22 °C. For yeast two-hybrid analyses, SD-WL medium selects for the presence of both bait and prey vectors, and SD-WLHA medium selects for an interaction between bait and prey proteins. IPP2, APX3 and At1g11910 were used to normalize real-time PCR expression analyses, and all primers for quantitative PCR are listed in Supplemental Table 1. ELF4∷ELF4-HA construct includes 580 bp of promoter sequence cloned from Col-0 DNA amplified using primers listed in Supplemental Table 1. The sequence TATGATATCCTTGCGTACCCA is the target of the LUX/NOX amiRNA. Antibodies were generated in rabbits (Sigma Genosys) against either an ELF3 specific peptide (CSIQEERKRYDSSKP), or a full-length LUX protein fused to glutathione-S-transferase (GST). Antibodies were affinity purified against the same ELF3 peptide using a SulfoLink Immobilization kit (Thermo Scientific) or using a GST-LUX affinity column. All immunoprecipitations were performed with Protein G Dynabeads (Invitrogen). For westerns, ACTIN served as a loading control. Blots for ELF4 represent 20% of the total IP sample, as ELF4 must be run on a separate 15% gel, identified by (*), due to its low molecular weight. The dot (·) denotes a background signal arising from the cross-linked HA beads (data not shown). LUX runs as high and low molecular weight isoforms, denoted by (–). Hypocotyl measurements were performed on evenly spaced seedlings grown under 12L:12D, measured on day 10.

Methods Summary

All wild-type, mutant and transgenic lines were in the Arabidopsis thaliana ecotype Columbia-0 (Col-0). All transgenic and mutant lines were brought to homozygosity before use. The procedures for Arabidopsis husbandry; yeast one-hybrid, two-hybrid, and three-hybrid analyses; biolumenscent imaging; immunoprecipitations; chromatin immunoprecipitations; and hypocotyl measurements were as described previously20,29,30, with modifications detailed in Methods. In all growth chambers, light was supplied at 80 μmol m-2 sec-1 by cool-white fluorescent bulbs at 22 °C. For yeast two-hybrid analyses, SD-WL medium selects for the presence of both bait and prey vectors, and SD-WLHA medium selects for an interaction between bait and prey proteins. IPP2, APX3 and At1g11910 were used to normalize real-time PCR expression analyses, and all primers for quantitative PCR are listed in Supplemental Table 1. ELF4∷ELF4-HA construct includes 580 bp of promoter sequence cloned from Col-0 DNA amplified using primers listed in Supplemental Table 1. The sequence TATGATATCCTTGCGTACCCA is the target of the LUX/NOX amiRNA. Antibodies were generated in rabbits (Sigma Genosys) against either an ELF3 specific peptide (CSIQEERKRYDSSKP), or a full-length LUX protein fused to glutathione-S-transferase (GST). Antibodies were affinity purified against the same ELF3 peptide using a SulfoLink Immobilization kit (Thermo Scientific) or using a GST-LUX affinity column. All immunoprecipitations were performed with Protein G Dynabeads (Invitrogen). For westerns, ACTIN served as a loading control. Blots for ELF4 represent 20% of the total IP sample, as ELF4 must be run on a separate 15% gel, identified by (*), due to its low molecular weight. The dot (·) denotes a background signal arising from the cross-linked HA beads (data not shown). LUX runs as high and low molecular weight isoforms, denoted by (–). Hypocotyl measurements were performed on evenly spaced seedlings grown under 12L:12D, measured on day 10.

Supplementary Material1

Supplementary Material
