---
pmid: '19116412'
title: Splicing factor SFRS1 recognizes a functionally diverse landscape of RNA transcripts.
authors:
- Sanford JR
- Wang X
- Mort M
- Vanduyn N
- Cooper DN
- Mooney SD
- Edenberg HJ
- Liu Y
journal: Genome Res
year: '2009'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC2661799
doi: 10.1101/gr.082503.108
---

# Splicing factor SFRS1 recognizes a functionally diverse landscape of RNA transcripts.
**Authors:** Sanford JR, Wang X, Mort M, Vanduyn N, Cooper DN, Mooney SD, Edenberg HJ, Liu Y
**Journal:** Genome Res (2009)
**DOI:** [10.1101/gr.082503.108](https://doi.org/10.1101/gr.082503.108)
**PMC:** [PMC2661799](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2661799/)

## Abstract

1. Genome Res. 2009 Mar;19(3):381-94. doi: 10.1101/gr.082503.108. Epub 2008 Dec
30.

Splicing factor SFRS1 recognizes a functionally diverse landscape of RNA 
transcripts.

Sanford JR(1), Wang X, Mort M, Vanduyn N, Cooper DN, Mooney SD, Edenberg HJ, Liu 
Y.

Author information:
(1)Department of Molecular, Cellular, and Developmental Biology, University of 
California Santa Cruz, Santa Cruz, California 95064, USA. 
sanford@biology.ucsc.edu

Metazoan genes are encrypted with at least two superimposed codes: the genetic 
code to specify the primary structure of proteins and the splicing code to 
expand their proteomic output via alternative splicing. Here, we define the 
specificity of a central regulator of pre-mRNA splicing, the conserved, 
essential splicing factor SFRS1. Cross-linking immunoprecipitation and 
high-throughput sequencing (CLIP-seq) identified 23,632 binding sites for SFRS1 
in the transcriptome of cultured human embryonic kidney cells. SFRS1 was found 
to engage many different classes of functionally distinct transcripts including 
mRNA, miRNA, snoRNAs, ncRNAs, and conserved intergenic transcripts of unknown 
function. The majority of these diverse transcripts share a purine-rich 
consensus motif corresponding to the canonical SFRS1 binding site. The consensus 
site was not only enriched in exons cross-linked to SFRS1 in vivo, but was also 
enriched in close proximity to splice sites. mRNAs encoding RNA processing 
factors were significantly overrepresented, suggesting that SFRS1 may broadly 
influence the post-transcriptional control of gene expression in vivo. Finally, 
a search for the SFRS1 consensus motif within the Human Gene Mutation Database 
identified 181 mutations in 82 different genes that disrupt predicted SFRS1 
binding sites. This comprehensive analysis substantially expands the known roles 
of human SR proteins in the regulation of a diverse array of RNA transcripts.

DOI: 10.1101/gr.082503.108
PMCID: PMC2661799
PMID: 19116412 [Indexed for MEDLINE]

## Full Text

Abstract

Metazoan genes are encrypted with at least two superimposed codes: the genetic code to specify the primary structure of proteins and the splicing code to expand their proteomic output via alternative splicing. Here, we define the specificity of a central regulator of pre-mRNA splicing, the conserved, essential splicing factor SFRS1. Cross-linking immunoprecipitation and high-throughput sequencing (CLIP-seq) identified 23,632 binding sites for SFRS1 in the transcriptome of cultured human embryonic kidney cells. SFRS1 was found to engage many different classes of functionally distinct transcripts including mRNA, miRNA, snoRNAs, ncRNAs, and conserved intergenic transcripts of unknown function. The majority of these diverse transcripts share a purine-rich consensus motif corresponding to the canonical SFRS1 binding site. The consensus site was not only enriched in exons cross-linked to SFRS1 in vivo, but was also enriched in close proximity to splice sites. mRNAs encoding RNA processing factors were significantly overrepresented, suggesting that SFRS1 may broadly influence the post-transcriptional control of gene expression in vivo. Finally, a search for the SFRS1 consensus motif within the Human Gene Mutation Database identified 181 mutations in 82 different genes that disrupt predicted SFRS1 binding sites. This comprehensive analysis substantially expands the known roles of human SR proteins in the regulation of a diverse array of RNA transcripts.

Discussion

High-throughput DNA sequencing is rapidly changing the landscape of genomic research ( Wold and Myers 2008 ). Our study is perhaps the first to utilize high-throughput sequencing to analyze protein–RNA interactions. We used the 454 FLX system (Roche) for amplicon sequencing based upon several considerations including read length and the well-validated platform. The read length of the 454 FLX system, using a short read kit, allows for 120–150 bp reads and is ideal for completely sequencing the RNA fragment and linkers produced by CLIP, ensuring that all sequence information is preserved. The longer read length provided by the 454 platform facilitated mapping of sequences to the genome and allowed for detection of many exon–intron and exon–exon junctions in the amplicon library. The primary consideration for future CLIP-seq experiments is clearly an increased sequencing throughput. Issues arising from the natural abundance of different RNA transcripts and the preferential PCR amplification during library preparation have the potential to introduce many redundant reads. The work presented here cannot be viewed as comprehensive, because significant new sequences were discovered in each of the four CLIP-seq experiments analyzed. However, the majority of the novel sequences share at least one statistically significant match to the consensus SFRS1 binding site. Therefore, for the specific application of CLIP-seq, data generated by the 454 platform are akin to a large-scale sampling or snapshot. Systems such as the SOLiD platform from ABI, which promise significantly increased throughput, have the potential to deliver truly comprehensive CLIP-seq analyses.

The work presented above represents a significant step toward elucidating the roles of SFRS1 in post-transcriptional gene expression. Perhaps more importantly, these experiments demonstrate the potential of CLIP-seq to illuminate the recognition code of RNA-binding proteins and their in situ binding sites. A comprehensive evaluation of protein–RNA interactions is critical for understanding how RNA-binding proteins positively or negatively regulate post-transcriptional processes such as alternative splicing. As future CLIP-seq experiments increase the catalog of known protein–RNA interactions, efforts to integrate binding-site data with functional genomic approaches have the potential to reveal the global organization of post-transcriptional regulatory networks in mammalian cells ( Ule et al. 2006 ; Wang and Burge 2008 ).

A significant advantage of CLIP-seq is the large amount of raw data generated by the high-throughput sequencing of amplicons. These data facilitate the elucidation of consensus sites using motif-finding algorithms such as MEME. The motif presented in Figure 2 was the only statistically significant sequence identified by MEME. The robust nature of the binding-site model allowed for high-resolution mapping of SFRS1 binding sites within the amplicon data. This is important because future in silico analyses should focus on these positionally restricted windows for identification of SFRS1-regulated exons. Such an approach is exemplified by the search for SFRS1 binding sites abolished by inherited mutations causing human genetic disease. We identified 181 exonic mutations in 82 different disease genes that abolish putative SFRS1 binding sites. Nearly 87% of these mutations were located within 50 bp of the nearest splice site, a region already demonstrated by CLIP-seq to be enriched in SFRS1 binding sites. It is quite possible that mutations falling outside of the preferred zone of SFRS1 binding will have little impact on RNA processing. However, at least five of the mutations identified here have already been correlated with changes in alternative splicing. Given that none of the mutations we identified were apparent in previously published reports identifying large numbers of splicing-relevant disease mutations, the pathological impact of exonic mutations upon splicing could turn out to be quite significant. Our findings argue that defective RNA processing, typically considered unusual in cases of nonsynonymous disease mutations, could actually be the rule rather than the exception.

The CLIP method, developed in the Darnell laboratory at Rockefeller University, was first used to identify RNA targets of the splicing regulator NOVA. NOVA and SFRS1 are very different types of splicing factors and these differences are clearly reflected in their in situ RNA-binding specificities elucidated by CLIP. NOVA and SFRS1 engage RNA through structurally distinct RNA-binding domains. The K-homology RNA-binding domain of NOVA recognizes a pyrimidine-rich YCAY motif that is nearly threefold more abundant in RNA fragments bound by NOVA relative to nontargeted sequences. In contrast, our study shows that SFRS1 binds a purine-rich octamer with a GAAGAA core. This motif is highly enriched in exons bound by SFRS1 relative to randomly selected exon sequences ( Fig. 6 ). The binding sites for both proteins within pre-mRNA are restricted to specific positions. Intronic and 3′ UTR binding sites are most prevalent in the NOVA targets, whereas internal exonic binding sites are strongly preferred by SFRS1 ( Fig. 7 ). The positional binding specificities of both proteins can provide insight into their mechanisms of action. By comparing alternative splicing patterns of target transcripts in NOVA knockout and wild-type mice, Ule and coworkers were able to deduce how different positions of NOVA binding sites influenced splice site selection ( Ule et al. 2006 ). Although we have not yet established the functionality of each binding site for SFRS1 identified by CLIP-seq, based upon the well established roles of SFRS1 in pre-mRNA splicing, we speculate that exonic binding sites are likely to function as splicing enhancers. However, because the majority of blocks identified by CLIP-seq are classified as exonic, it is not possible to determine whether these binding events occur during spliceosome assembly or instead at a later stage of mRNA processing. In contrast, blocks spanning exon–intron boundaries clearly represent interactions with pre-mRNA, whereas those spanning exon junctions are derived from spliced mRNA. Fewer than 5% of all blocks (1065) mapped to exon–intron boundaries and only 2245 mapped to exon junctions. Given the proximity of these blocks to splice sites, it is possible that the same RNA elements recognized by SFRS1 influence pre-mRNA splicing and subsequent cytoplasmic steps of post-transcriptional gene expression. Recent work from our laboratory identified binding sites in several mRNAs that are engaged by SFRS1 in both nuclear and cytoplasmic/polysomal mRNA fractions of the cell ( Sanford et al. 2008 ). Clearly, further functional studies are required to elucidate the functions of SFRS1 binding sites identified by CLIP-seq.

Another interesting finding from our study is that binding sites for SFRS1 are enriched in exons that are adjacent to alternative cassette exons ( Table 2 ). A previous study demonstrated that SFRS1 binding sites in a constitutive exon regulated the skipping of an upstream alternative cassette exon in the receptor tyrosine kinase RON gene ( Ghigna et al. 2005 ). We propose that SFRS1 may play a prominent role in regulating this mode of competitive exon skipping by activating downstream splice sites. Finally, there are also significant differences between the functions of proteins encoded by mRNAs targeted by NOVA and SFRS1. NOVA mRNA targets tend to encode proteins involved in pre- and postsynaptic function as well as neuronal inhibition ( Ule et al. 2003 , 2005b ). In contrast, mRNAs encoding other RNA-binding proteins are overrepresented in the collection of SFRS1 targets. These include a statistically significant enrichment of other splicing factors ( Fig. 4 ). We are confident that the enrichment of RNA-binding protein messages is biologically significant for several reasons. First, comparisons of SFRS1 targets with mRNAs present in the nonselected input RNA samples demonstrate that transcript abundance alone does not account for the enrichment of RBP mRNAs in the CLIP-seq data set. Secondly, recent experiments describe auto- and trans -regulatory post-transcriptional networks involved in homeostatic control of RNA-binding protein expression ( Lareau et al. 2007 ; Ni et al. 2007 ; Barberan-Soler and Zahler 2008 ; Saltzman et al. 2008 ). Two hallmarks of this mechanism include alternative splicing-coupled nonsense-mediated decay (AS-NMD) and ultraconserved cis -acting regulatory elements within coding exons of RBP mRNAs ( Bejerano et al. 2004 ). In many cases, the ultraconserved regions of RBP genes overlap alternative exons with the potential to induce NMD ( Bejerano et al. 2004 ; Lareau et al. 2007 ; Ni et al. 2007 ). In total, we identified eight out of 111 known ultraconserved regions within exonic sequences. Included in these ultraconserved binding sites are several other genes encoding RNA-binding proteins such as SFRS1 itself, SFRS6 , HNRNPM , PCBP2 , and CLK4 encoding SR protein kinase (Supplemental Fig. 4). We suggest that SFRS1 may be involved in controlling RBP homeostasis.
