---
pmid: '19124653'
title: Htm1 protein generates the N-glycan signal for glycoprotein degradation in
  the endoplasmic reticulum.
authors:
- Clerc S
- Hirsch C
- Oggier DM
- Deprez P
- Jakob C
- Sommer T
- Aebi M
journal: J Cell Biol
year: '2009'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC2615083
doi: 10.1083/jcb.200809198
---

# Htm1 protein generates the N-glycan signal for glycoprotein degradation in the endoplasmic reticulum.
**Authors:** Clerc S, Hirsch C, Oggier DM, Deprez P, Jakob C, Sommer T, Aebi M
**Journal:** J Cell Biol (2009)
**DOI:** [10.1083/jcb.200809198](https://doi.org/10.1083/jcb.200809198)
**PMC:** [PMC2615083](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2615083/)

## Abstract

1. J Cell Biol. 2009 Jan 12;184(1):159-72. doi: 10.1083/jcb.200809198. Epub 2009 
Jan 5.

Htm1 protein generates the N-glycan signal for glycoprotein degradation in the 
endoplasmic reticulum.

Clerc S(1), Hirsch C, Oggier DM, Deprez P, Jakob C, Sommer T, Aebi M.

Author information:
(1)Department of Biology, Institute of Microbiology, Eidgenössische Technische 
Hochschule Zürich, Zürich, Switzerland.

To maintain protein homeostasis in secretory compartments, eukaryotic cells 
harbor a quality control system that monitors protein folding and protein 
complex assembly in the endoplasmic reticulum (ER). Proteins that do not fold 
properly or integrate into cognate complexes are degraded by ER-associated 
degradation (ERAD) involving retrotranslocation to the cytoplasm and proteasomal 
peptide hydrolysis. N-linked glycans are essential in glycoprotein ERAD; the 
covalent oligosaccharide structure is used as a signal to display the folding 
status of the host protein. In this study, we define the function of the Htm1 
protein as an alpha1,2-specific exomannosidase that generates the 
Man(7)GlcNAc(2) oligosaccharide with a terminal alpha1,6-linked mannosyl residue 
on degradation substrates. This oligosaccharide signal is decoded by the 
ER-localized lectin Yos9p that in conjunction with Hrd3p triggers the 
ubiquitin-proteasome-dependent hydrolysis of these glycoproteins. The Htm1p 
exomannosidase activity requires processing of the N-glycan by glucosidase I, 
glucosidase II, and mannosidase I, resulting in a sequential order of specific 
N-glycan structures that reflect the folding status of the glycoprotein.

DOI: 10.1083/jcb.200809198
PMCID: PMC2615083
PMID: 19124653 [Indexed for MEDLINE]

## Full Text

IntroductionIn eukaryotic cells, the synthesis of asparagine-linked glycoproteins occurs in the ER. As nascent protein chains enter the ER lumen through the Sec61 translocon complex, the oligosaccharyltransferase transfers the Glc3Man9GlcNAc2 glycan to asparagines in the consensus motif N-X-S/T, generating an N-linked glycan. Soon after the transfer, the N-linked glycan is processed by glucosidase I, glucosidase II, and ER mannosidase I. N-glycans increase the hydrophilicity of the protein and thereby affect protein folding directly. Importantly, these glycans also serve as ligands for proteins that modulate the folding (Helenius and Aebi, 2004). The Glc1Man9GlcNAc2 structure is a signal for the interaction with the molecular chaperones calnexin or calreticulin in higher eukaryotes or Cne1p, respectively, as proposed in Saccharomyces cerevisiae (Xu et al., 2004). In mammalian cells, after the removal of the mannose-linked glucose by glucosidase II, the folding sensor UDP-glucose/glycoprotein glucosyltransferase can readd a terminal glucose and thereby allows for another cycle of reassociation with calnexin/calreticulin. In contrast, in S. cerevisiae, no reglucosylation of misfolded proteins has been found, which implies the absence of the calnexin/calreticulin cycle in this system (Fernandez et al., 1994).Proteins that achieve the correct fold and integrate correctly into their cognate complexes can continue along the secretory pathway to their site of action. In contrast, proteins that do not achieve the proper fold in due time, as well as excess subunits of protein complexes, are retained in the ER and ultimately are degraded by the cytoplasmically located 26S proteasome. The process of recognition, retrotranslocation, and degradation of terminally misfolded proteins is referred to as ER-associated degradation (ERAD).Although the N-glycans play a crucial role in the folding of N-glycoproteins, their remodeling by glucosidase I, glucosidase II, and ER mannosidase I is also crucial for N-glycoprotein ERAD (Knop et al., 1996; Jakob et al., 1998a; Hitt and Wolf, 2004). The finding that ER mannosidase I is a relatively slow-acting enzyme (compared with glucosidase I and II) supported the mannose timer hypothesis (Su et al., 1993; Helenius et al., 1997) and suggested that the ER mannosidase I product glycan Man8GlcNAc2, when present on misfolded proteins, serves as the ERAD targeting signal in mammalian cells and in S. cerevisiae. This led to the proposal that the ER contains a lectin or a lectinlike protein that recognizes Man8GlcNAc2 glycans on misfolded glycoproteins and by this initiates glycoprotein ERAD (Liu et al., 1997; Jakob et al., 1998a; Yang et al., 1998).A group of proteins found throughout the eukaryotic kingdom with homology to class I mannosidases (family 47 glycosylhydrolases), termed Htm1p/ER degradation–enhancing α-mannosidase–like protein (EDEM), was proposed to act as such degradation lectins in ERAD. Sequence comparison of ER mannosidase I, EDEM1/2/3, and Htm1p reveals a mannosidase homology region with significant sequence identity among the homologues. However, the proteins differ in their remaining domains. One particular characteristic is the presence of a C-terminal extension that varies among the EDEM/Htm1p homologues (Mast et al., 2005; Kanehara et al., 2007). In Htm1p, this domain constitutes almost one third of the whole protein. Although the essential residues of the processing mannosidases are conserved in Htm1p/EDEM1/2/3, initially neither in vivo (Jakob et al., 2001) nor in vitro (Hosokawa et al., 2001; Mast et al., 2005) mannosidase activity was reported, supporting their potential role as lectins. More recently, another protein besides Htm1p, Yos9p, has been suggested to act as ER degradation lectin. Yos9p harbors a mannose 6-phosphate receptor homology domain essential for its function in N-glycoprotein ERAD. It was shown to be required for the efficient degradation of ERAD substrates that carry lesions in the ER lumen (Buschhorn et al., 2004; Bhamidipati et al., 2005; Kim et al., 2005; Szathmary et al., 2005).In S. cerevisiae, three pathways, ERAD–lumen (ERAD-L), ERAD–cytosol, and ERAD–membrane, have been described, engaging different factors for the degradation of proteins with misfolded lesions in the corresponding domains (Bonifacino et al., 1990; Taxis et al., 2003; Vashist and Ng, 2004). ERAD-L engages the Hrd1p E3 ubiquitin ligase, which is in a complex with the Ubc7p/Cue1p E2 ubiquitin–conjugating enzyme and AAA-ATPase (ATPase associated with a variety of cellular activities) Cdc48p with the cofactors Npl4p, Ufd1p, and Ubx2p. Moreover, this complex includes Hrd3p, Yos9p, Kar2p, Der1p, and Usa1p (Gardner et al., 2000; Carvalho et al., 2006; Denic et al., 2006; Gauss et al., 2006a,b). Hrd3p/Kar2p/Yos9p is believed to specifically recognize and target misfolded proteins carrying Man8GlcNAc2 glycans of the ERAD-L pathway to the degradation. Interestingly, Htm1p is also acting in the ERAD-L pathway via the Hrd1p complex (Vashist and Ng, 2004); however, it was not found to be a member of the complex but proposed to act upstream of Yos9p (Denic et al., 2006; Gauss et al., 2006a; Kanehara et al., 2007).Although in S. cerevisiae the Man8GlcNAc2 glycan as N-glycan ERAD determinant remained, it became evident that in mammalian cells, N-glycans of folding-deficient proteins are further trimmed to Man6GlcNAc2 and Man5GlcNAc2 structures (Ermonval et al., 2001; Foulquier et al., 2002, 2004; Frenkel et al., 2003; Hosokawa et al., 2003; Kitzmuller et al., 2003). By this extensive demannosylation, the terminal mannose of the A branch, which is the acceptor mannose of UDP-glucuronosyltransferase, is removed. Thus, the proteins can no longer associate with the chaperones calnexin and calreticulin and can be directly deviated into the disposal machinery (Molinari, 2007). Recently, EDEM3 was shown to have mannosidase activity in vivo, as its overexpression leads to mannose trimming in vivo to Man7GlcNAc2 and Man6GlcNAc2 (Hirao et al., 2006), and interestingly, EDEM1 was also proposed to act as processing mannosidase in the extraction of misfolded glycoproteins from the calnexin/calreticulin cycle rather than being a lectin (Olivari et al., 2006).Reconsidering glycoprotein quality control and ERAD in S. cerevisiae in which the calnexin/calreticulin cycle is absent, no apparent need for enhanced N-glycan processing has emerged to date. Nevertheless, the essential residues of the processing mannosidases are conserved in Htm1p. Therefore, we reevaluated the function of Htm1p in the yeast ERAD system.

Introduction

In eukaryotic cells, the synthesis of asparagine-linked glycoproteins occurs in the ER. As nascent protein chains enter the ER lumen through the Sec61 translocon complex, the oligosaccharyltransferase transfers the Glc3Man9GlcNAc2 glycan to asparagines in the consensus motif N-X-S/T, generating an N-linked glycan. Soon after the transfer, the N-linked glycan is processed by glucosidase I, glucosidase II, and ER mannosidase I. N-glycans increase the hydrophilicity of the protein and thereby affect protein folding directly. Importantly, these glycans also serve as ligands for proteins that modulate the folding (Helenius and Aebi, 2004). The Glc1Man9GlcNAc2 structure is a signal for the interaction with the molecular chaperones calnexin or calreticulin in higher eukaryotes or Cne1p, respectively, as proposed in Saccharomyces cerevisiae (Xu et al., 2004). In mammalian cells, after the removal of the mannose-linked glucose by glucosidase II, the folding sensor UDP-glucose/glycoprotein glucosyltransferase can readd a terminal glucose and thereby allows for another cycle of reassociation with calnexin/calreticulin. In contrast, in S. cerevisiae, no reglucosylation of misfolded proteins has been found, which implies the absence of the calnexin/calreticulin cycle in this system (Fernandez et al., 1994).

Proteins that achieve the correct fold and integrate correctly into their cognate complexes can continue along the secretory pathway to their site of action. In contrast, proteins that do not achieve the proper fold in due time, as well as excess subunits of protein complexes, are retained in the ER and ultimately are degraded by the cytoplasmically located 26S proteasome. The process of recognition, retrotranslocation, and degradation of terminally misfolded proteins is referred to as ER-associated degradation (ERAD).

Although the N-glycans play a crucial role in the folding of N-glycoproteins, their remodeling by glucosidase I, glucosidase II, and ER mannosidase I is also crucial for N-glycoprotein ERAD (Knop et al., 1996; Jakob et al., 1998a; Hitt and Wolf, 2004). The finding that ER mannosidase I is a relatively slow-acting enzyme (compared with glucosidase I and II) supported the mannose timer hypothesis (Su et al., 1993; Helenius et al., 1997) and suggested that the ER mannosidase I product glycan Man8GlcNAc2, when present on misfolded proteins, serves as the ERAD targeting signal in mammalian cells and in S. cerevisiae. This led to the proposal that the ER contains a lectin or a lectinlike protein that recognizes Man8GlcNAc2 glycans on misfolded glycoproteins and by this initiates glycoprotein ERAD (Liu et al., 1997; Jakob et al., 1998a; Yang et al., 1998).

A group of proteins found throughout the eukaryotic kingdom with homology to class I mannosidases (family 47 glycosylhydrolases), termed Htm1p/ER degradation–enhancing α-mannosidase–like protein (EDEM), was proposed to act as such degradation lectins in ERAD. Sequence comparison of ER mannosidase I, EDEM1/2/3, and Htm1p reveals a mannosidase homology region with significant sequence identity among the homologues. However, the proteins differ in their remaining domains. One particular characteristic is the presence of a C-terminal extension that varies among the EDEM/Htm1p homologues (Mast et al., 2005; Kanehara et al., 2007). In Htm1p, this domain constitutes almost one third of the whole protein. Although the essential residues of the processing mannosidases are conserved in Htm1p/EDEM1/2/3, initially neither in vivo (Jakob et al., 2001) nor in vitro (Hosokawa et al., 2001; Mast et al., 2005) mannosidase activity was reported, supporting their potential role as lectins. More recently, another protein besides Htm1p, Yos9p, has been suggested to act as ER degradation lectin. Yos9p harbors a mannose 6-phosphate receptor homology domain essential for its function in N-glycoprotein ERAD. It was shown to be required for the efficient degradation of ERAD substrates that carry lesions in the ER lumen (Buschhorn et al., 2004; Bhamidipati et al., 2005; Kim et al., 2005; Szathmary et al., 2005).

In S. cerevisiae, three pathways, ERAD–lumen (ERAD-L), ERAD–cytosol, and ERAD–membrane, have been described, engaging different factors for the degradation of proteins with misfolded lesions in the corresponding domains (Bonifacino et al., 1990; Taxis et al., 2003; Vashist and Ng, 2004). ERAD-L engages the Hrd1p E3 ubiquitin ligase, which is in a complex with the Ubc7p/Cue1p E2 ubiquitin–conjugating enzyme and AAA-ATPase (ATPase associated with a variety of cellular activities) Cdc48p with the cofactors Npl4p, Ufd1p, and Ubx2p. Moreover, this complex includes Hrd3p, Yos9p, Kar2p, Der1p, and Usa1p (Gardner et al., 2000; Carvalho et al., 2006; Denic et al., 2006; Gauss et al., 2006a,b). Hrd3p/Kar2p/Yos9p is believed to specifically recognize and target misfolded proteins carrying Man8GlcNAc2 glycans of the ERAD-L pathway to the degradation. Interestingly, Htm1p is also acting in the ERAD-L pathway via the Hrd1p complex (Vashist and Ng, 2004); however, it was not found to be a member of the complex but proposed to act upstream of Yos9p (Denic et al., 2006; Gauss et al., 2006a; Kanehara et al., 2007).

Although in S. cerevisiae the Man8GlcNAc2 glycan as N-glycan ERAD determinant remained, it became evident that in mammalian cells, N-glycans of folding-deficient proteins are further trimmed to Man6GlcNAc2 and Man5GlcNAc2 structures (Ermonval et al., 2001; Foulquier et al., 2002, 2004; Frenkel et al., 2003; Hosokawa et al., 2003; Kitzmuller et al., 2003). By this extensive demannosylation, the terminal mannose of the A branch, which is the acceptor mannose of UDP-glucuronosyltransferase, is removed. Thus, the proteins can no longer associate with the chaperones calnexin and calreticulin and can be directly deviated into the disposal machinery (Molinari, 2007). Recently, EDEM3 was shown to have mannosidase activity in vivo, as its overexpression leads to mannose trimming in vivo to Man7GlcNAc2 and Man6GlcNAc2 (Hirao et al., 2006), and interestingly, EDEM1 was also proposed to act as processing mannosidase in the extraction of misfolded glycoproteins from the calnexin/calreticulin cycle rather than being a lectin (Olivari et al., 2006).

Reconsidering glycoprotein quality control and ERAD in S. cerevisiae in which the calnexin/calreticulin cycle is absent, no apparent need for enhanced N-glycan processing has emerged to date. Nevertheless, the essential residues of the processing mannosidases are conserved in Htm1p. Therefore, we reevaluated the function of Htm1p in the yeast ERAD system.

ResultsHTM1 overexpression leads to altered N-glycan processingTo readdress the question of whether Htm1p is an active mannosidase in vivo, the N-glycan structures of overall proteins from wild-type, Δhtm1, and wild-type cells overexpressing HTM1 were compared. N-linked sugars were metabolically labeled in vivo for 20 min with [3H]mannose, and whole cell protein extracts were prepared. N-glycans were released by peptide N-glycosidase F treatment and analyzed by HPLC. As previously reported, the most abundant N-glycan structure in the ER of wild-type cells was Man8GlcNAc2 (Byrd et al., 1982; Jakob et al., 1998a). In addition, small amounts of oligosaccharides migrating at the positions of Man5GlcNAc2 and Man7GlcNAc2 glycans were also detected (Fig. 1 A). The conversion of the N-linked glycans to Man8GlcNAc2 was not impaired in Δhtm1 cells, and the N-glycan profile appeared similar in both wild-type and Δhtm1 cells (Fig. 1 A; Jakob et al., 2001). However, when HTM1 was overexpressed, we were able to detect an increase of a putative Man7GlcNAc2 glycan, indicating Htm1p-induced N-glycan processing (Fig. 1, A and B).Figure 1.Overexpression of HTM1 leads to altered N-glycan processing. (A) [3H]mannose-labeled N-linked glycan profiles from strain YG618 transformed with YEp352 (wild type [wt] + empty vector) or YEp352-HTM1 (wild type + HTM1) and strain YCJ1 carrying YEp352 (Δhtm1 + empty vector). M8, M7, and M5 mark the elution positions of Man8GlcNAc2, Man7GlcNAc2, and Man5GlcNAc2, respectively. Labeling of the glycans was performed for 20 min. N-glycans were released after the preparation of whole cell protein extracts with peptide N-glycosidase F, purified, and analyzed by HPLC analysis. (B) Relative intensity of the Man8GlcNAc2 (M8), Man7GlcNAc2 (M7), and Man5GlcNAc2 (M5) peaks in the cells from A. Deviations represent the experimental error of three measurements of the experiment depicted in A. (C) CPY* degradation analysis in strain YG618 transformed with YEp352 (wild type + empty vector) or YEp352-HTM1 (wild type + HTM1) and strain YCJ1 carrying YEp352 (Δhtm1 + empty vector). The graph represents the amount of remaining CPY* at the indicated chase time points. Each time point represents the mean ± standard deviation of three independent experiments.We also tested the effect of HTM1 deletion and overexpression on the degradation of the ERAD model substrate carboxypeptidase Y* (CPY*; Wolf and Fink, 1975; Finger et al., 1993). As previously reported, HTM1 deletion reduced the degradation rate of CPY*, but overexpression of the same protein had no effect on the stability of this ERAD substrate (Fig. 1 C), suggesting that Htm1p function is not the rate-limiting step in the degradation process.HTM1 overexpression generates a Man7GlcNAc2 glycanNext, we analyzed the kinetics of N-linked glycan processing by pulse-chase experiments. In wild-type cells, a 10-min pulse predominantly revealed the Man8GlcNAc2 structure. Small amounts of a Man9GlcNAc2 oligosaccharide were also present. Upon the chase, oligosaccharides with higher molecular weight became prominent, reflecting the processing of N-linked glycans by Golgi-localized mannosyltransferases (Byrd et al., 1982). In contrast, in HTM1-overexpressing cells Man7GlcNAc2 was detected directly after the pulse, and the relative amount of this oligosaccharide increased during the chase period (Fig. 2 A). In comparison with the empty vector control, the amount of the Man7GlcNAc2 glycan was 4-fold or 4.5-fold increased at 0 min or 30 min of chase, respectively. Concomitantly, Man8GlcNAc2 was substantially reduced, indicating that it had served as physiological substrate for the trimming (Fig. 2 B). We concluded that overexpression of Htm1p results in an amplified level of the Man7GlcNAc2 N-linked glycan. This suggested a novel exomannosidase function of this protein in vivo.Figure 2.HTM1 overexpression generates a Man7GlcNAc2 glycan. (A) Pulse-chase analysis of the N-glycan processing in YG618 transformed with YEp352 (empty vector) or YEp352-HTM1 (HTM1 overexpression). Cells were pulsed for 10 min with [3H]mannose and chased for the different times indicated on the left. M9, M8, and M7 mark the elution positions of Man9GlcNAc2, Man8GlcNAc2, and Man7GlcNAc2, respectively. (B) Quantification of the relative glycan intensities at 0 and 30 min of chase. The values represent the mean of seven independent experiments + standard deviation. *, P < 0.05; **, P < 0.005; ***, P < 0.001, determined by using paired Student's t test. HEX, hexose.Htm1p mannosidase acts on Man8GlcNAc2, yielding Man7GlcNAc2 isomer CThe aforementioned results indicated that the Man7GlcNAc2 oligosaccharide was generated by trimming of the Man8GlcNAc2 oligosaccharide. Therefore, we first tested whether the Man8GlcNAc2 oligosaccharide structure was required for the Htm1p-dependent processing. For this, we took advantage of processing-deficient mutant strains and of the relaxed substrate specificity of the yeast oligosaccharyltransferase to genetically tailor the structure of the glycans present on proteins (Jakob et al., 1998b). In wild-type cells, the activities of glucosidase I and II as well as ER mannosidase I result in the generation of the Man8GlcNAc2 structure (Kornfeld and Kornfeld, 1985; Moremen et al., 1994), and overexpression of Htm1p yielded the Man7GlcNAc2 glycan. The Glc2Man9GlcNAc2 oligosaccharide in a glucosidase II–deficient strain is still processed by ER mannosidase I to the Glc2Man8GlcNAc2 oligosaccharide (Jakob et al., 1998a), but HTM1-dependent processing was strongly impaired (Fig. 3, A and B). In contrast, the Glc1Man9GlcNAc2 oligosaccharide transferred in Δalg8Δgls2 cells was processed by ER mannosidase I to the Glc1Man8GlcNAc2 oligosaccharide, and this glycan was also a target for Htm1p-dependent processing to Glc1Man7GlcNAc2. The deletion of the MNS1 locus resulted in the generation of Man9GlcNAc2 glycans, and the absence of ER mannosidase I activity almost completely abolished Htm1p-dependent processing. From this result, we concluded that the sequential trimming of the N-glycan by glucosidase I, glucosidase II, and ER mannosidase I, generating the Man8GlcNAc2 oligosaccharide in vivo, is a prerequisite for Htm1p-dependent processing. Moreover, because extension of the A branch by one glucose did not prevent HTM1-dependent processing, this analysis suggested that the trimming involved either the B or C branch of the N-linked glycan target.Figure 3.Processing of N-glycans by glucosidase I, glucosidase II, and mannosidase I is required for efficient trimming by Htm1p. (A) N-glycan pulse-chase analysis over two time points (10-min pulse; 0- and 30-min chase). Graphs display N-glycan profiles obtained in YG618 (wild type [wt]), YG695 (Δgls2), YG624 (Δalg8Δgls2), and YG777 (Δmns1) transformed each with YEp352 (empty vector) or YEp352-HTM1 (HTM1). The symbolized N-glycans shown at the left represent the structure of the main N-glycan, which is produced in the ER of the corresponding cells. M7, Man7GlcNAc2; M7G1, Glc1Man7GlcNAc2; M8, Man8GlcNAc2; M8G2, Glc2Man8GlcNAc2; M8G1, Glc1Man8GlcNAc2; M9, Man9GlcNAc2. (B) Quantification of HTM1-dependent processing products in the four different strains shown in A. The values represent the relative level + standard deviation of the HTM1-dependent processing products from three independent measurements after a 30-min chase.To elucidate whether the B or C branch was processed by Htm1p, we isolated the radioactively labeled Man7GlcNAc2 product from HTM1-overexpressing cells and wild-type (vector control) cells by preparative HPLC. The glycans were digested in vitro with α1,2-exomannosidase from Trichoderma reesei (Maras et al., 2000), and the products were reassessed by HPLC analysis to determine whether two or three α1,2-exomannosidase–sensitive linkages were present yet (Fig. 4 A). The chromatograms showed a 3.5-fold increase of Man5GlcNAc2 over the Man4GlcNAc2 glycan in cells that had overexpressed HTM1 in comparison with the vector control (Fig. 4, B and C). From this, we deduced that the Htm1p mannosidase acted on an α1,2-linked mannose residue of the Man8GlcNAc2 oligosaccharide. As the presence of a glucosyl residue on the A branch of the oligosaccharide still allowed Htm1p-dependent processing, we concluded that overexpression of Htm1p results in the selective hydrolysis of the α1,2-linked mannose of the C branch.Figure 4.Analysis of the Man7GlcNAc2 oligosaccharide by α1,2-exomannosidase digestion. (A) Schematic representation of the three possible Man7GlcNAc2 product isoforms A, B, and C (M7A, M7B, and M7C) upon removal of one mannose unit from Man8GlcNAc2 (M8). Depending on whether an α1,2-linked mannose of branch A or C or the α1,3-linked mannose of branch B had been trimmed, two or three α1,2-exomannosidase–sensitive mannoses remain. The products A and C can thus be distinguished from glycan B. M, mannose. (B) HPLC profiles of the radiolabeled isolated Man7GlcNAc2 (M7) peaks from YG618 containing YEp352 (empty vector) or YEp352-HTM1 (HTM1 overexpression). Isolated glycans were left untreated (−) or treated (+) with the α1,2-exomannosidase from T. reesei and analyzed by HPLC. M5 represents the elution peak of a Man5GlcNAc2 that was used as a standard (third trace). (C) Quantification of the Man5GlcNAc2 (M5) oligosaccharide obtained from the Man7GlcNAc2 oligosaccharide after digestion with α1,2-exomannosidase. The Man4GlcNAc2 (M4) and Man5GlcNAc2 oligosaccharide levels in the elution profiles depicted in B were quantified, and the level of the Man4GlcNAc2 oligosaccharide was set to 1. The value represents one measurement of oligosaccharides obtained from strain YG618 transformed with YEp352 (empty vector) and the mean of two independent experiments for YG618 transformed with YEp352-HTM1 (HTM1 overexpression).Htm1p is an active mannosidase in vivo, and the essential residues are required for CPY* degradationTo determine whether the observed N-glycan processing activity was accomplished directly by Htm1p, mutant forms of Htm1p in which the conserved amino acid residues E222 and D279 were replaced by glutamine or asparagine, respectively, were overexpressed in yeast. The orthologous residues of E222 and D279 in ER mannosidase I are essential for the hydrolase function (Lipari and Herscovics, 1999). To assess the stability of the mutants, the overexpressed proteins were marked with a TPH7 tag (Knop et al., 1999) at the C terminus. The mutations did not affect the stability of the proteins, as was confirmed by immunoblot analysis of the TPH7-tagged proteins (Fig. 5 A). However, the N-glycan pulse-chase analysis with the overexpressed Htm1-E222Q-TPH7 as well as Htm1-D279N-TPH7 revealed that both mutations abolished the N-glycan processing (Fig. 5, B and C). The same result was obtained with the untagged mutants Htm1-E222Q and Htm1-D279N (unpublished data). This indicated that the mannosidase activity is a direct function of Htm1p.Figure 5.Structure function analysis of Htm1p. (A) Strain SS328 was transformed with plasmids expressing the TPH7-tagged Htm1p (Htm1-TPH7; lane 1) and the mutant forms of Htm1p given at the right (lanes 2–4) or empty vector DNA (lane 5). Extracts were prepared, separated by SDS-PAGE, and transferred to nitrocellulose. Blots were probed with peroxidase–antiperoxidase-specific antibodies (top) and HXK-specific serum (bottom) to verify equal loading of the gel. The position of molecular-size markers is given at the left. The blot shows equivalent expression of wild-type and mutant forms of Htm1p fusions. (B) N-glycan analysis after a pulse of 10 min and a chase of 0 and 30 min. Graphs display N-glycan HPLC profiles obtained in YG618 transformed with empty vector or plasmid DNA expressing the different mutant forms of tagged Htm1p given at the left. Only the overexpression of tagged wild-type Htm1p resulted in enhanced production of Man7GlcNAc2 (M7) oligosaccharide. M8, Man8GlcNAc2. (C) Quantification of the Man7GlcNAc2 glycan levels obtained from the different strains given in B. N-glycan profiles of a 30-min chase as shown in B were quantified, and the relative level of Man7GlcNAc2 is given. Relative intensities of the peaks depicted in B are plotted + standard deviations of two independent experiments. (D) Mutations in the catalytic domain of Htm1p and truncation of the C terminus abolishes Htm1p function in CPY* degradation. Strain YCJ1 was transformed with empty vector (Δhtm1 + empty vector) and the different mutant constructs indicated above the lanes. The cells were chased with cycloheximide for the time indicated, extracts were prepared, and the proteins were separated by SDS-PAGE. After transfer to nitrocellulose, CPY* was detected with serum directed against CPY. The position of the band representing CPY* is given at the right. The secondary antibody used in this experiment also reacted with protein A in the TPH7 tag of the Htm1 proteins (*, position of the band representing full-length protein; **, band representing truncated Htm1 protein). Detection of HXK levels made a quantification of the CPY* degradation in these experiments possible. (E) Values given represent the mean value of remaining CPY* levels from three independent experiments. The standard deviation is given. MW, molecular weight.To determine whether Htm1p ERAD function relies on its mannosidase activity, we assessed the functionality of the mutant Htm1 proteins in the ERAD process. Therefore, CPY* degradation rates of Δhtm1 cells transformed with HTM1-TPH7, Htm1-E222Q-TPH7, Htm1-D279N-TPH7, or empty vector were analyzed. The Htm1-TPH7 construct complemented the CPY* degradation deficiency in Δhtm1 cells. In contrast, the point mutants Htm1-E222Q-TPH7 and Htm1-D279N-TPH7 did not rescue this phenotype (Fig. 5, D and E). The same result was obtained with the untagged point mutants (unpublished data). Therefore, the acidic residues E222 and D279 are essential for Htm1p ERAD function as well as for the observed mannosidase activity.As noted in the Introduction, both Mns1p and Htm1p contain a mannosidase homology region, but Htm1p is characterized by a C-terminal extension that accounts for almost one third of the whole protein. We generated a truncated version of the HTM1 locus encoding the mannosidase domain only (Htm1-Δ517-TPH7). Immunoblot analysis confirmed that Htm1-Δ517-TPH7 was expressed (Fig. 5 A) but did not result in N-glycan processing (Fig. 5, B and C) and did not complement the Δhtm1 CPY* degradation phenotype (Fig. 5, D and E). Therefore, the mannosidase homology region was necessary but not sufficient for Htm1p function. We propose that Htm1p functions as an α1,2-mannosidase on the C branch of the N-linked glycan. This mannosidase activity is essential for efficient degradation of the CPY* ERAD substrate and requires the C-terminal domain of the protein.Htm1p physically interacts with Pdi1p via its C-terminal domainTo address the function of the C-terminal domain more closely, Htm1p-interacting proteins were identified. We generated a functional C-terminally Myc–tandem affinity purification (TAP)–tagged Htm1p construct (unpublished data). Microsomes were prepared, and solubilized Htm1–Myc-TAP protein was precipitated from the supernatant with IgG beads. The purified complexes were resolved by SDS-PAGE, and proteins were stained (Fig. 6 A). The two prominent bands that were visualized around 62 kD were isolated from the gel and subjected to mass spectrometry. Both bands were identified as protein disulfide isomerase 1 protein (Pdi1p; Table S1, available at http://www.jcb.org/cgi/content/full/jcb.200809198/DC1). We wondered whether the C terminus of Htm1p might be required for this interaction. Therefore, Myc-tagged constructs of Htm1p were generated of different truncates within the C terminus, namely at the positions 749, 683, 647, 582, and 547. We also generated Pdi1-HA-HDEL and a C-terminal truncate of Pdi1p at position 459, Pdi1-Δ459–HA-HDEL. Immunopurification of Pdi1-HA-HDEL or Pdi1-Δ459–HA-HDEL was performed, and the presence or absence of full-length or truncated Htm1p-Myc was determined by immunoblot analysis. The results confirmed the interaction of the full-length Pdi1p with Htm1p. However, no interaction could be detected between Pdi1-Δ459–HA-HDEL and Htm1-Myc (Fig. 6 B). When we pulled down full-length Pdi1p–HA-HDEL and assessed the binding of the Htm1p truncates, the interaction was readily detectable with the Htm1-Δ749–, Htm1-Δ683–, and Htm1-Δ647–Myc truncates but not with Htm1-Δ582 and -Δ547 (Fig. 6 C). From this, we deduced that the C terminus is involved in the interaction with Pdi1p.Figure 6.Htm1p physically interacts with Pdi1p. (A) Microsomes were prepared from control cells or cells expressing Htm1–Myc-TAP. After solubilization, both lysates were incubated with IgG beads. The bound material was subsequently liberated by digestion with tobacco etch virus protease and analyzed by SDS-PAGE. Visualization of the protein content by silver staining revealed Htm1-Myc and two bands around 62 kD that were clearly absent from the control lane (compare lane 2 with lane 3). Proteins were excised from the gel, trypsinized, and subjected to mass spectrometry. Both bands at 62 kD were identified as Pdi1p. The identity of the Htm1-Myc band was also confirmed by mass spectrometry. For size comparison, a molecular weight (MW) standard was loaded in lane 1. (B) Detergent-solubilized microsomes from yeast strains expressing full-length Pdi1-HA-HDEL (lanes 1 and 2) or Pdi1-Δ459–HA-HDEL (lane 3) together with Htm1-Myc (lanes 2 and 3) were analyzed by Western blot (WB) to confirm the presence of the tagged proteins (left). HA-tagged proteins were immunoprecipitated, and analysis of the anti-HA precipitates by Western blotting (right) revealed that Htm1-Myc interacted with full-length Pdi1-HA-HDEL (lane 5) but not with Pdi1-Δ459–HA-HDEL (lane 6). (C) Microsomes were isolated from yeast strains expressing either full-length Htm1-Myc or the indicated truncated variants of Htm1p together with Pdi1-HA-HDEL (lanes 2–7). Immunoblotting of the total lysates confirmed the expression of the tagged proteins (left, top and middle), and immunoprecipitation of Pdi1-HA-HDEL was confirmed (top right). Probing for the Myc-tagged Htm1p constructs revealed a readily detectable interaction between Pdi1-HA-HDEL and Htm1-, Htm1-Δ749–, Htm1-Δ683–, and -Δ647–Myc truncates (lanes 9–12) but not with shorter variants of Htm1p (lanes 13 and 14). Probing of the supernatants after immunopurification with serum against HA confirmed the quantitative depletion of Pdi1-HA-HDEL from the lysates (bottom left). The secondary antibody used in the Western blot also reacted with the heavy chain (HC) of the HA antibody used in the immunopurification (IP).A terminal α1,6-linked mannose as a glycan determinant directing misfolded glycoproteins to degradationHaving identified the main product of the Htm1p activity as a specific Man7GlcNAc2 isomer, we were wondering whether this glycan, rather than the postulated Man8GlcNAc2 oligosaccharide, might function as the putative glycoprotein degradation signal in S. cerevisiae. Generation of this isomer on a misfolded protein might target its host protein for degradation. In contrast to Man8GlcNAc2, the Man7GlcNAc2 C structure is characterized by a terminal α1,6-linked mannose of the C branch (Fig. 7 A). Within the framework of our hypothesis, we assumed specifically that a terminal α1,6-linked mannosyl residue functioned as a signal required for degradation. Yos9p, the putative ER-degradation lectin with a mannose 6-phosphate receptor homology domain essential for glycoprotein ERAD, might act as the receptor for this structure (Buschhorn et al., 2004; Bhamidipati et al., 2005; Kim et al., 2005; Szathmary et al., 2005). To address this hypothesis directly, we took advantage of the possibility to alter N-glycan structures by manipulating the biosynthetic pathway of lipid-linked oligosaccharide assembly (Jakob et al., 1998b). It is noteworthy that the putative product of Htm1p, the Man7GlcNAc2 oligosaccharide, is not a biosynthetic intermediate in the pathway of N-glycoprotein biosynthesis. However, Δalg3 mutant cells produce a Man5GlcNAc2 glycan that also carries a terminal α1,6-linked mannose linked to an α1,3-substituted mannose, as is the case in the Man7GlcNAc2 C isomer. In contrast, Δalg9 and Δalg12 cells do not produce a terminal α1,6-linked mannose (Fig. 7 A). In support of our hypothesis, wild-type and Δalg3 cells degrade CPY* efficiently, whereas Δalg9 and Δalg12 cells show a strongly reduced degradation rate (Fig. 7, B and C; Jakob et al., 1998a). Moreover, the rapid degradation in wild-type and Δalg3 cells was dependent on the presence of the putative lectin Yos9p (Fig. 7, F and G), but Htm1p was only required in wild-type but not in Δalg3 cells (Fig. 7, D and E). This reinforced our hypothesis that Htm1p is essential in wild-type cells for the generation of a specific glycan signal, a terminal α1,6-linked mannose. This signal is always present in the Δalg3 cell, making degradation of malfolded proteins independent of Htm1p function but still dependent on the Yos9p lectin.Figure 7.HTM1-independent but YOS9-dependent degradation of CPY* in Δalg3 cells. (A) N-glycan structures of the wild-type Man8GlcNAc2 (Mns1p product), Htm1p product Man7GlcNAc2 C, and the N-glycans prevailing in Δalg3, Δalg9, or Δalg12 cells. Man7GlcNAc2 C and Man5GlcNAc2 in Δalg3 both expose an α1,6-linked terminal mannose (circles). (B–G) Analysis of CPY* degradation in different mutant strains. The cells were chased with cycloheximide for the time indicated, extracts were prepared, and the proteins were separated by SDS-PAGE. After transfer to nitrocellulose, CPY* was detected with serum directed against CPY. The position of the band representing CPY* is given at the right. Detection of HXK levels made a quantification of the CPY* degradation in these experiments possible. (B and C) Analysis of wild-type (wt), Δalg3, and Δalg9 cells. (D and E) Analysis of wild-type, Δhtm1, Δalg3, and Δalg3Δhtm1 cells. (F and G) Analysis of wild-type, Δyos9, Δalg3, and Δalg3Δyos9 cells. The values given in the relative quantification experiments are the mean of three independent experiments (B and C are two independent experiments). Standard deviations are indicated.

HTM1 overexpression leads to altered N-glycan processingTo readdress the question of whether Htm1p is an active mannosidase in vivo, the N-glycan structures of overall proteins from wild-type, Δhtm1, and wild-type cells overexpressing HTM1 were compared. N-linked sugars were metabolically labeled in vivo for 20 min with [3H]mannose, and whole cell protein extracts were prepared. N-glycans were released by peptide N-glycosidase F treatment and analyzed by HPLC. As previously reported, the most abundant N-glycan structure in the ER of wild-type cells was Man8GlcNAc2 (Byrd et al., 1982; Jakob et al., 1998a). In addition, small amounts of oligosaccharides migrating at the positions of Man5GlcNAc2 and Man7GlcNAc2 glycans were also detected (Fig. 1 A). The conversion of the N-linked glycans to Man8GlcNAc2 was not impaired in Δhtm1 cells, and the N-glycan profile appeared similar in both wild-type and Δhtm1 cells (Fig. 1 A; Jakob et al., 2001). However, when HTM1 was overexpressed, we were able to detect an increase of a putative Man7GlcNAc2 glycan, indicating Htm1p-induced N-glycan processing (Fig. 1, A and B).Figure 1.Overexpression of HTM1 leads to altered N-glycan processing. (A) [3H]mannose-labeled N-linked glycan profiles from strain YG618 transformed with YEp352 (wild type [wt] + empty vector) or YEp352-HTM1 (wild type + HTM1) and strain YCJ1 carrying YEp352 (Δhtm1 + empty vector). M8, M7, and M5 mark the elution positions of Man8GlcNAc2, Man7GlcNAc2, and Man5GlcNAc2, respectively. Labeling of the glycans was performed for 20 min. N-glycans were released after the preparation of whole cell protein extracts with peptide N-glycosidase F, purified, and analyzed by HPLC analysis. (B) Relative intensity of the Man8GlcNAc2 (M8), Man7GlcNAc2 (M7), and Man5GlcNAc2 (M5) peaks in the cells from A. Deviations represent the experimental error of three measurements of the experiment depicted in A. (C) CPY* degradation analysis in strain YG618 transformed with YEp352 (wild type + empty vector) or YEp352-HTM1 (wild type + HTM1) and strain YCJ1 carrying YEp352 (Δhtm1 + empty vector). The graph represents the amount of remaining CPY* at the indicated chase time points. Each time point represents the mean ± standard deviation of three independent experiments.We also tested the effect of HTM1 deletion and overexpression on the degradation of the ERAD model substrate carboxypeptidase Y* (CPY*; Wolf and Fink, 1975; Finger et al., 1993). As previously reported, HTM1 deletion reduced the degradation rate of CPY*, but overexpression of the same protein had no effect on the stability of this ERAD substrate (Fig. 1 C), suggesting that Htm1p function is not the rate-limiting step in the degradation process.

HTM1 overexpression leads to altered N-glycan processing

To readdress the question of whether Htm1p is an active mannosidase in vivo, the N-glycan structures of overall proteins from wild-type, Δhtm1, and wild-type cells overexpressing HTM1 were compared. N-linked sugars were metabolically labeled in vivo for 20 min with [3H]mannose, and whole cell protein extracts were prepared. N-glycans were released by peptide N-glycosidase F treatment and analyzed by HPLC. As previously reported, the most abundant N-glycan structure in the ER of wild-type cells was Man8GlcNAc2 (Byrd et al., 1982; Jakob et al., 1998a). In addition, small amounts of oligosaccharides migrating at the positions of Man5GlcNAc2 and Man7GlcNAc2 glycans were also detected (Fig. 1 A). The conversion of the N-linked glycans to Man8GlcNAc2 was not impaired in Δhtm1 cells, and the N-glycan profile appeared similar in both wild-type and Δhtm1 cells (Fig. 1 A; Jakob et al., 2001). However, when HTM1 was overexpressed, we were able to detect an increase of a putative Man7GlcNAc2 glycan, indicating Htm1p-induced N-glycan processing (Fig. 1, A and B).

Overexpression of HTM1 leads to altered N-glycan processing. (A) [3H]mannose-labeled N-linked glycan profiles from strain YG618 transformed with YEp352 (wild type [wt] + empty vector) or YEp352-HTM1 (wild type + HTM1) and strain YCJ1 carrying YEp352 (Δhtm1 + empty vector). M8, M7, and M5 mark the elution positions of Man8GlcNAc2, Man7GlcNAc2, and Man5GlcNAc2, respectively. Labeling of the glycans was performed for 20 min. N-glycans were released after the preparation of whole cell protein extracts with peptide N-glycosidase F, purified, and analyzed by HPLC analysis. (B) Relative intensity of the Man8GlcNAc2 (M8), Man7GlcNAc2 (M7), and Man5GlcNAc2 (M5) peaks in the cells from A. Deviations represent the experimental error of three measurements of the experiment depicted in A. (C) CPY* degradation analysis in strain YG618 transformed with YEp352 (wild type + empty vector) or YEp352-HTM1 (wild type + HTM1) and strain YCJ1 carrying YEp352 (Δhtm1 + empty vector). The graph represents the amount of remaining CPY* at the indicated chase time points. Each time point represents the mean ± standard deviation of three independent experiments.

We also tested the effect of HTM1 deletion and overexpression on the degradation of the ERAD model substrate carboxypeptidase Y* (CPY*; Wolf and Fink, 1975; Finger et al., 1993). As previously reported, HTM1 deletion reduced the degradation rate of CPY*, but overexpression of the same protein had no effect on the stability of this ERAD substrate (Fig. 1 C), suggesting that Htm1p function is not the rate-limiting step in the degradation process.

HTM1 overexpression generates a Man7GlcNAc2 glycanNext, we analyzed the kinetics of N-linked glycan processing by pulse-chase experiments. In wild-type cells, a 10-min pulse predominantly revealed the Man8GlcNAc2 structure. Small amounts of a Man9GlcNAc2 oligosaccharide were also present. Upon the chase, oligosaccharides with higher molecular weight became prominent, reflecting the processing of N-linked glycans by Golgi-localized mannosyltransferases (Byrd et al., 1982). In contrast, in HTM1-overexpressing cells Man7GlcNAc2 was detected directly after the pulse, and the relative amount of this oligosaccharide increased during the chase period (Fig. 2 A). In comparison with the empty vector control, the amount of the Man7GlcNAc2 glycan was 4-fold or 4.5-fold increased at 0 min or 30 min of chase, respectively. Concomitantly, Man8GlcNAc2 was substantially reduced, indicating that it had served as physiological substrate for the trimming (Fig. 2 B). We concluded that overexpression of Htm1p results in an amplified level of the Man7GlcNAc2 N-linked glycan. This suggested a novel exomannosidase function of this protein in vivo.Figure 2.HTM1 overexpression generates a Man7GlcNAc2 glycan. (A) Pulse-chase analysis of the N-glycan processing in YG618 transformed with YEp352 (empty vector) or YEp352-HTM1 (HTM1 overexpression). Cells were pulsed for 10 min with [3H]mannose and chased for the different times indicated on the left. M9, M8, and M7 mark the elution positions of Man9GlcNAc2, Man8GlcNAc2, and Man7GlcNAc2, respectively. (B) Quantification of the relative glycan intensities at 0 and 30 min of chase. The values represent the mean of seven independent experiments + standard deviation. *, P < 0.05; **, P < 0.005; ***, P < 0.001, determined by using paired Student's t test. HEX, hexose.

HTM1 overexpression generates a Man7GlcNAc2 glycan

Next, we analyzed the kinetics of N-linked glycan processing by pulse-chase experiments. In wild-type cells, a 10-min pulse predominantly revealed the Man8GlcNAc2 structure. Small amounts of a Man9GlcNAc2 oligosaccharide were also present. Upon the chase, oligosaccharides with higher molecular weight became prominent, reflecting the processing of N-linked glycans by Golgi-localized mannosyltransferases (Byrd et al., 1982). In contrast, in HTM1-overexpressing cells Man7GlcNAc2 was detected directly after the pulse, and the relative amount of this oligosaccharide increased during the chase period (Fig. 2 A). In comparison with the empty vector control, the amount of the Man7GlcNAc2 glycan was 4-fold or 4.5-fold increased at 0 min or 30 min of chase, respectively. Concomitantly, Man8GlcNAc2 was substantially reduced, indicating that it had served as physiological substrate for the trimming (Fig. 2 B). We concluded that overexpression of Htm1p results in an amplified level of the Man7GlcNAc2 N-linked glycan. This suggested a novel exomannosidase function of this protein in vivo.

HTM1 overexpression generates a Man7GlcNAc2 glycan. (A) Pulse-chase analysis of the N-glycan processing in YG618 transformed with YEp352 (empty vector) or YEp352-HTM1 (HTM1 overexpression). Cells were pulsed for 10 min with [3H]mannose and chased for the different times indicated on the left. M9, M8, and M7 mark the elution positions of Man9GlcNAc2, Man8GlcNAc2, and Man7GlcNAc2, respectively. (B) Quantification of the relative glycan intensities at 0 and 30 min of chase. The values represent the mean of seven independent experiments + standard deviation. *, P < 0.05; **, P < 0.005; ***, P < 0.001, determined by using paired Student's t test. HEX, hexose.

Htm1p mannosidase acts on Man8GlcNAc2, yielding Man7GlcNAc2 isomer CThe aforementioned results indicated that the Man7GlcNAc2 oligosaccharide was generated by trimming of the Man8GlcNAc2 oligosaccharide. Therefore, we first tested whether the Man8GlcNAc2 oligosaccharide structure was required for the Htm1p-dependent processing. For this, we took advantage of processing-deficient mutant strains and of the relaxed substrate specificity of the yeast oligosaccharyltransferase to genetically tailor the structure of the glycans present on proteins (Jakob et al., 1998b). In wild-type cells, the activities of glucosidase I and II as well as ER mannosidase I result in the generation of the Man8GlcNAc2 structure (Kornfeld and Kornfeld, 1985; Moremen et al., 1994), and overexpression of Htm1p yielded the Man7GlcNAc2 glycan. The Glc2Man9GlcNAc2 oligosaccharide in a glucosidase II–deficient strain is still processed by ER mannosidase I to the Glc2Man8GlcNAc2 oligosaccharide (Jakob et al., 1998a), but HTM1-dependent processing was strongly impaired (Fig. 3, A and B). In contrast, the Glc1Man9GlcNAc2 oligosaccharide transferred in Δalg8Δgls2 cells was processed by ER mannosidase I to the Glc1Man8GlcNAc2 oligosaccharide, and this glycan was also a target for Htm1p-dependent processing to Glc1Man7GlcNAc2. The deletion of the MNS1 locus resulted in the generation of Man9GlcNAc2 glycans, and the absence of ER mannosidase I activity almost completely abolished Htm1p-dependent processing. From this result, we concluded that the sequential trimming of the N-glycan by glucosidase I, glucosidase II, and ER mannosidase I, generating the Man8GlcNAc2 oligosaccharide in vivo, is a prerequisite for Htm1p-dependent processing. Moreover, because extension of the A branch by one glucose did not prevent HTM1-dependent processing, this analysis suggested that the trimming involved either the B or C branch of the N-linked glycan target.Figure 3.Processing of N-glycans by glucosidase I, glucosidase II, and mannosidase I is required for efficient trimming by Htm1p. (A) N-glycan pulse-chase analysis over two time points (10-min pulse; 0- and 30-min chase). Graphs display N-glycan profiles obtained in YG618 (wild type [wt]), YG695 (Δgls2), YG624 (Δalg8Δgls2), and YG777 (Δmns1) transformed each with YEp352 (empty vector) or YEp352-HTM1 (HTM1). The symbolized N-glycans shown at the left represent the structure of the main N-glycan, which is produced in the ER of the corresponding cells. M7, Man7GlcNAc2; M7G1, Glc1Man7GlcNAc2; M8, Man8GlcNAc2; M8G2, Glc2Man8GlcNAc2; M8G1, Glc1Man8GlcNAc2; M9, Man9GlcNAc2. (B) Quantification of HTM1-dependent processing products in the four different strains shown in A. The values represent the relative level + standard deviation of the HTM1-dependent processing products from three independent measurements after a 30-min chase.To elucidate whether the B or C branch was processed by Htm1p, we isolated the radioactively labeled Man7GlcNAc2 product from HTM1-overexpressing cells and wild-type (vector control) cells by preparative HPLC. The glycans were digested in vitro with α1,2-exomannosidase from Trichoderma reesei (Maras et al., 2000), and the products were reassessed by HPLC analysis to determine whether two or three α1,2-exomannosidase–sensitive linkages were present yet (Fig. 4 A). The chromatograms showed a 3.5-fold increase of Man5GlcNAc2 over the Man4GlcNAc2 glycan in cells that had overexpressed HTM1 in comparison with the vector control (Fig. 4, B and C). From this, we deduced that the Htm1p mannosidase acted on an α1,2-linked mannose residue of the Man8GlcNAc2 oligosaccharide. As the presence of a glucosyl residue on the A branch of the oligosaccharide still allowed Htm1p-dependent processing, we concluded that overexpression of Htm1p results in the selective hydrolysis of the α1,2-linked mannose of the C branch.Figure 4.Analysis of the Man7GlcNAc2 oligosaccharide by α1,2-exomannosidase digestion. (A) Schematic representation of the three possible Man7GlcNAc2 product isoforms A, B, and C (M7A, M7B, and M7C) upon removal of one mannose unit from Man8GlcNAc2 (M8). Depending on whether an α1,2-linked mannose of branch A or C or the α1,3-linked mannose of branch B had been trimmed, two or three α1,2-exomannosidase–sensitive mannoses remain. The products A and C can thus be distinguished from glycan B. M, mannose. (B) HPLC profiles of the radiolabeled isolated Man7GlcNAc2 (M7) peaks from YG618 containing YEp352 (empty vector) or YEp352-HTM1 (HTM1 overexpression). Isolated glycans were left untreated (−) or treated (+) with the α1,2-exomannosidase from T. reesei and analyzed by HPLC. M5 represents the elution peak of a Man5GlcNAc2 that was used as a standard (third trace). (C) Quantification of the Man5GlcNAc2 (M5) oligosaccharide obtained from the Man7GlcNAc2 oligosaccharide after digestion with α1,2-exomannosidase. The Man4GlcNAc2 (M4) and Man5GlcNAc2 oligosaccharide levels in the elution profiles depicted in B were quantified, and the level of the Man4GlcNAc2 oligosaccharide was set to 1. The value represents one measurement of oligosaccharides obtained from strain YG618 transformed with YEp352 (empty vector) and the mean of two independent experiments for YG618 transformed with YEp352-HTM1 (HTM1 overexpression).

Htm1p mannosidase acts on Man8GlcNAc2, yielding Man7GlcNAc2 isomer C

The aforementioned results indicated that the Man7GlcNAc2 oligosaccharide was generated by trimming of the Man8GlcNAc2 oligosaccharide. Therefore, we first tested whether the Man8GlcNAc2 oligosaccharide structure was required for the Htm1p-dependent processing. For this, we took advantage of processing-deficient mutant strains and of the relaxed substrate specificity of the yeast oligosaccharyltransferase to genetically tailor the structure of the glycans present on proteins (Jakob et al., 1998b). In wild-type cells, the activities of glucosidase I and II as well as ER mannosidase I result in the generation of the Man8GlcNAc2 structure (Kornfeld and Kornfeld, 1985; Moremen et al., 1994), and overexpression of Htm1p yielded the Man7GlcNAc2 glycan. The Glc2Man9GlcNAc2 oligosaccharide in a glucosidase II–deficient strain is still processed by ER mannosidase I to the Glc2Man8GlcNAc2 oligosaccharide (Jakob et al., 1998a), but HTM1-dependent processing was strongly impaired (Fig. 3, A and B). In contrast, the Glc1Man9GlcNAc2 oligosaccharide transferred in Δalg8Δgls2 cells was processed by ER mannosidase I to the Glc1Man8GlcNAc2 oligosaccharide, and this glycan was also a target for Htm1p-dependent processing to Glc1Man7GlcNAc2. The deletion of the MNS1 locus resulted in the generation of Man9GlcNAc2 glycans, and the absence of ER mannosidase I activity almost completely abolished Htm1p-dependent processing. From this result, we concluded that the sequential trimming of the N-glycan by glucosidase I, glucosidase II, and ER mannosidase I, generating the Man8GlcNAc2 oligosaccharide in vivo, is a prerequisite for Htm1p-dependent processing. Moreover, because extension of the A branch by one glucose did not prevent HTM1-dependent processing, this analysis suggested that the trimming involved either the B or C branch of the N-linked glycan target.

Processing of N-glycans by glucosidase I, glucosidase II, and mannosidase I is required for efficient trimming by Htm1p. (A) N-glycan pulse-chase analysis over two time points (10-min pulse; 0- and 30-min chase). Graphs display N-glycan profiles obtained in YG618 (wild type [wt]), YG695 (Δgls2), YG624 (Δalg8Δgls2), and YG777 (Δmns1) transformed each with YEp352 (empty vector) or YEp352-HTM1 (HTM1). The symbolized N-glycans shown at the left represent the structure of the main N-glycan, which is produced in the ER of the corresponding cells. M7, Man7GlcNAc2; M7G1, Glc1Man7GlcNAc2; M8, Man8GlcNAc2; M8G2, Glc2Man8GlcNAc2; M8G1, Glc1Man8GlcNAc2; M9, Man9GlcNAc2. (B) Quantification of HTM1-dependent processing products in the four different strains shown in A. The values represent the relative level + standard deviation of the HTM1-dependent processing products from three independent measurements after a 30-min chase.

To elucidate whether the B or C branch was processed by Htm1p, we isolated the radioactively labeled Man7GlcNAc2 product from HTM1-overexpressing cells and wild-type (vector control) cells by preparative HPLC. The glycans were digested in vitro with α1,2-exomannosidase from Trichoderma reesei (Maras et al., 2000), and the products were reassessed by HPLC analysis to determine whether two or three α1,2-exomannosidase–sensitive linkages were present yet (Fig. 4 A). The chromatograms showed a 3.5-fold increase of Man5GlcNAc2 over the Man4GlcNAc2 glycan in cells that had overexpressed HTM1 in comparison with the vector control (Fig. 4, B and C). From this, we deduced that the Htm1p mannosidase acted on an α1,2-linked mannose residue of the Man8GlcNAc2 oligosaccharide. As the presence of a glucosyl residue on the A branch of the oligosaccharide still allowed Htm1p-dependent processing, we concluded that overexpression of Htm1p results in the selective hydrolysis of the α1,2-linked mannose of the C branch.

Analysis of the Man7GlcNAc2 oligosaccharide by α1,2-exomannosidase digestion. (A) Schematic representation of the three possible Man7GlcNAc2 product isoforms A, B, and C (M7A, M7B, and M7C) upon removal of one mannose unit from Man8GlcNAc2 (M8). Depending on whether an α1,2-linked mannose of branch A or C or the α1,3-linked mannose of branch B had been trimmed, two or three α1,2-exomannosidase–sensitive mannoses remain. The products A and C can thus be distinguished from glycan B. M, mannose. (B) HPLC profiles of the radiolabeled isolated Man7GlcNAc2 (M7) peaks from YG618 containing YEp352 (empty vector) or YEp352-HTM1 (HTM1 overexpression). Isolated glycans were left untreated (−) or treated (+) with the α1,2-exomannosidase from T. reesei and analyzed by HPLC. M5 represents the elution peak of a Man5GlcNAc2 that was used as a standard (third trace). (C) Quantification of the Man5GlcNAc2 (M5) oligosaccharide obtained from the Man7GlcNAc2 oligosaccharide after digestion with α1,2-exomannosidase. The Man4GlcNAc2 (M4) and Man5GlcNAc2 oligosaccharide levels in the elution profiles depicted in B were quantified, and the level of the Man4GlcNAc2 oligosaccharide was set to 1. The value represents one measurement of oligosaccharides obtained from strain YG618 transformed with YEp352 (empty vector) and the mean of two independent experiments for YG618 transformed with YEp352-HTM1 (HTM1 overexpression).

Htm1p is an active mannosidase in vivo, and the essential residues are required for CPY* degradationTo determine whether the observed N-glycan processing activity was accomplished directly by Htm1p, mutant forms of Htm1p in which the conserved amino acid residues E222 and D279 were replaced by glutamine or asparagine, respectively, were overexpressed in yeast. The orthologous residues of E222 and D279 in ER mannosidase I are essential for the hydrolase function (Lipari and Herscovics, 1999). To assess the stability of the mutants, the overexpressed proteins were marked with a TPH7 tag (Knop et al., 1999) at the C terminus. The mutations did not affect the stability of the proteins, as was confirmed by immunoblot analysis of the TPH7-tagged proteins (Fig. 5 A). However, the N-glycan pulse-chase analysis with the overexpressed Htm1-E222Q-TPH7 as well as Htm1-D279N-TPH7 revealed that both mutations abolished the N-glycan processing (Fig. 5, B and C). The same result was obtained with the untagged mutants Htm1-E222Q and Htm1-D279N (unpublished data). This indicated that the mannosidase activity is a direct function of Htm1p.Figure 5.Structure function analysis of Htm1p. (A) Strain SS328 was transformed with plasmids expressing the TPH7-tagged Htm1p (Htm1-TPH7; lane 1) and the mutant forms of Htm1p given at the right (lanes 2–4) or empty vector DNA (lane 5). Extracts were prepared, separated by SDS-PAGE, and transferred to nitrocellulose. Blots were probed with peroxidase–antiperoxidase-specific antibodies (top) and HXK-specific serum (bottom) to verify equal loading of the gel. The position of molecular-size markers is given at the left. The blot shows equivalent expression of wild-type and mutant forms of Htm1p fusions. (B) N-glycan analysis after a pulse of 10 min and a chase of 0 and 30 min. Graphs display N-glycan HPLC profiles obtained in YG618 transformed with empty vector or plasmid DNA expressing the different mutant forms of tagged Htm1p given at the left. Only the overexpression of tagged wild-type Htm1p resulted in enhanced production of Man7GlcNAc2 (M7) oligosaccharide. M8, Man8GlcNAc2. (C) Quantification of the Man7GlcNAc2 glycan levels obtained from the different strains given in B. N-glycan profiles of a 30-min chase as shown in B were quantified, and the relative level of Man7GlcNAc2 is given. Relative intensities of the peaks depicted in B are plotted + standard deviations of two independent experiments. (D) Mutations in the catalytic domain of Htm1p and truncation of the C terminus abolishes Htm1p function in CPY* degradation. Strain YCJ1 was transformed with empty vector (Δhtm1 + empty vector) and the different mutant constructs indicated above the lanes. The cells were chased with cycloheximide for the time indicated, extracts were prepared, and the proteins were separated by SDS-PAGE. After transfer to nitrocellulose, CPY* was detected with serum directed against CPY. The position of the band representing CPY* is given at the right. The secondary antibody used in this experiment also reacted with protein A in the TPH7 tag of the Htm1 proteins (*, position of the band representing full-length protein; **, band representing truncated Htm1 protein). Detection of HXK levels made a quantification of the CPY* degradation in these experiments possible. (E) Values given represent the mean value of remaining CPY* levels from three independent experiments. The standard deviation is given. MW, molecular weight.To determine whether Htm1p ERAD function relies on its mannosidase activity, we assessed the functionality of the mutant Htm1 proteins in the ERAD process. Therefore, CPY* degradation rates of Δhtm1 cells transformed with HTM1-TPH7, Htm1-E222Q-TPH7, Htm1-D279N-TPH7, or empty vector were analyzed. The Htm1-TPH7 construct complemented the CPY* degradation deficiency in Δhtm1 cells. In contrast, the point mutants Htm1-E222Q-TPH7 and Htm1-D279N-TPH7 did not rescue this phenotype (Fig. 5, D and E). The same result was obtained with the untagged point mutants (unpublished data). Therefore, the acidic residues E222 and D279 are essential for Htm1p ERAD function as well as for the observed mannosidase activity.As noted in the Introduction, both Mns1p and Htm1p contain a mannosidase homology region, but Htm1p is characterized by a C-terminal extension that accounts for almost one third of the whole protein. We generated a truncated version of the HTM1 locus encoding the mannosidase domain only (Htm1-Δ517-TPH7). Immunoblot analysis confirmed that Htm1-Δ517-TPH7 was expressed (Fig. 5 A) but did not result in N-glycan processing (Fig. 5, B and C) and did not complement the Δhtm1 CPY* degradation phenotype (Fig. 5, D and E). Therefore, the mannosidase homology region was necessary but not sufficient for Htm1p function. We propose that Htm1p functions as an α1,2-mannosidase on the C branch of the N-linked glycan. This mannosidase activity is essential for efficient degradation of the CPY* ERAD substrate and requires the C-terminal domain of the protein.

Htm1p is an active mannosidase in vivo, and the essential residues are required for CPY* degradation

To determine whether the observed N-glycan processing activity was accomplished directly by Htm1p, mutant forms of Htm1p in which the conserved amino acid residues E222 and D279 were replaced by glutamine or asparagine, respectively, were overexpressed in yeast. The orthologous residues of E222 and D279 in ER mannosidase I are essential for the hydrolase function (Lipari and Herscovics, 1999). To assess the stability of the mutants, the overexpressed proteins were marked with a TPH7 tag (Knop et al., 1999) at the C terminus. The mutations did not affect the stability of the proteins, as was confirmed by immunoblot analysis of the TPH7-tagged proteins (Fig. 5 A). However, the N-glycan pulse-chase analysis with the overexpressed Htm1-E222Q-TPH7 as well as Htm1-D279N-TPH7 revealed that both mutations abolished the N-glycan processing (Fig. 5, B and C). The same result was obtained with the untagged mutants Htm1-E222Q and Htm1-D279N (unpublished data). This indicated that the mannosidase activity is a direct function of Htm1p.

Structure function analysis of Htm1p. (A) Strain SS328 was transformed with plasmids expressing the TPH7-tagged Htm1p (Htm1-TPH7; lane 1) and the mutant forms of Htm1p given at the right (lanes 2–4) or empty vector DNA (lane 5). Extracts were prepared, separated by SDS-PAGE, and transferred to nitrocellulose. Blots were probed with peroxidase–antiperoxidase-specific antibodies (top) and HXK-specific serum (bottom) to verify equal loading of the gel. The position of molecular-size markers is given at the left. The blot shows equivalent expression of wild-type and mutant forms of Htm1p fusions. (B) N-glycan analysis after a pulse of 10 min and a chase of 0 and 30 min. Graphs display N-glycan HPLC profiles obtained in YG618 transformed with empty vector or plasmid DNA expressing the different mutant forms of tagged Htm1p given at the left. Only the overexpression of tagged wild-type Htm1p resulted in enhanced production of Man7GlcNAc2 (M7) oligosaccharide. M8, Man8GlcNAc2. (C) Quantification of the Man7GlcNAc2 glycan levels obtained from the different strains given in B. N-glycan profiles of a 30-min chase as shown in B were quantified, and the relative level of Man7GlcNAc2 is given. Relative intensities of the peaks depicted in B are plotted + standard deviations of two independent experiments. (D) Mutations in the catalytic domain of Htm1p and truncation of the C terminus abolishes Htm1p function in CPY* degradation. Strain YCJ1 was transformed with empty vector (Δhtm1 + empty vector) and the different mutant constructs indicated above the lanes. The cells were chased with cycloheximide for the time indicated, extracts were prepared, and the proteins were separated by SDS-PAGE. After transfer to nitrocellulose, CPY* was detected with serum directed against CPY. The position of the band representing CPY* is given at the right. The secondary antibody used in this experiment also reacted with protein A in the TPH7 tag of the Htm1 proteins (*, position of the band representing full-length protein; **, band representing truncated Htm1 protein). Detection of HXK levels made a quantification of the CPY* degradation in these experiments possible. (E) Values given represent the mean value of remaining CPY* levels from three independent experiments. The standard deviation is given. MW, molecular weight.

To determine whether Htm1p ERAD function relies on its mannosidase activity, we assessed the functionality of the mutant Htm1 proteins in the ERAD process. Therefore, CPY* degradation rates of Δhtm1 cells transformed with HTM1-TPH7, Htm1-E222Q-TPH7, Htm1-D279N-TPH7, or empty vector were analyzed. The Htm1-TPH7 construct complemented the CPY* degradation deficiency in Δhtm1 cells. In contrast, the point mutants Htm1-E222Q-TPH7 and Htm1-D279N-TPH7 did not rescue this phenotype (Fig. 5, D and E). The same result was obtained with the untagged point mutants (unpublished data). Therefore, the acidic residues E222 and D279 are essential for Htm1p ERAD function as well as for the observed mannosidase activity.

As noted in the Introduction, both Mns1p and Htm1p contain a mannosidase homology region, but Htm1p is characterized by a C-terminal extension that accounts for almost one third of the whole protein. We generated a truncated version of the HTM1 locus encoding the mannosidase domain only (Htm1-Δ517-TPH7). Immunoblot analysis confirmed that Htm1-Δ517-TPH7 was expressed (Fig. 5 A) but did not result in N-glycan processing (Fig. 5, B and C) and did not complement the Δhtm1 CPY* degradation phenotype (Fig. 5, D and E). Therefore, the mannosidase homology region was necessary but not sufficient for Htm1p function. We propose that Htm1p functions as an α1,2-mannosidase on the C branch of the N-linked glycan. This mannosidase activity is essential for efficient degradation of the CPY* ERAD substrate and requires the C-terminal domain of the protein.

Htm1p physically interacts with Pdi1p via its C-terminal domainTo address the function of the C-terminal domain more closely, Htm1p-interacting proteins were identified. We generated a functional C-terminally Myc–tandem affinity purification (TAP)–tagged Htm1p construct (unpublished data). Microsomes were prepared, and solubilized Htm1–Myc-TAP protein was precipitated from the supernatant with IgG beads. The purified complexes were resolved by SDS-PAGE, and proteins were stained (Fig. 6 A). The two prominent bands that were visualized around 62 kD were isolated from the gel and subjected to mass spectrometry. Both bands were identified as protein disulfide isomerase 1 protein (Pdi1p; Table S1, available at http://www.jcb.org/cgi/content/full/jcb.200809198/DC1). We wondered whether the C terminus of Htm1p might be required for this interaction. Therefore, Myc-tagged constructs of Htm1p were generated of different truncates within the C terminus, namely at the positions 749, 683, 647, 582, and 547. We also generated Pdi1-HA-HDEL and a C-terminal truncate of Pdi1p at position 459, Pdi1-Δ459–HA-HDEL. Immunopurification of Pdi1-HA-HDEL or Pdi1-Δ459–HA-HDEL was performed, and the presence or absence of full-length or truncated Htm1p-Myc was determined by immunoblot analysis. The results confirmed the interaction of the full-length Pdi1p with Htm1p. However, no interaction could be detected between Pdi1-Δ459–HA-HDEL and Htm1-Myc (Fig. 6 B). When we pulled down full-length Pdi1p–HA-HDEL and assessed the binding of the Htm1p truncates, the interaction was readily detectable with the Htm1-Δ749–, Htm1-Δ683–, and Htm1-Δ647–Myc truncates but not with Htm1-Δ582 and -Δ547 (Fig. 6 C). From this, we deduced that the C terminus is involved in the interaction with Pdi1p.Figure 6.Htm1p physically interacts with Pdi1p. (A) Microsomes were prepared from control cells or cells expressing Htm1–Myc-TAP. After solubilization, both lysates were incubated with IgG beads. The bound material was subsequently liberated by digestion with tobacco etch virus protease and analyzed by SDS-PAGE. Visualization of the protein content by silver staining revealed Htm1-Myc and two bands around 62 kD that were clearly absent from the control lane (compare lane 2 with lane 3). Proteins were excised from the gel, trypsinized, and subjected to mass spectrometry. Both bands at 62 kD were identified as Pdi1p. The identity of the Htm1-Myc band was also confirmed by mass spectrometry. For size comparison, a molecular weight (MW) standard was loaded in lane 1. (B) Detergent-solubilized microsomes from yeast strains expressing full-length Pdi1-HA-HDEL (lanes 1 and 2) or Pdi1-Δ459–HA-HDEL (lane 3) together with Htm1-Myc (lanes 2 and 3) were analyzed by Western blot (WB) to confirm the presence of the tagged proteins (left). HA-tagged proteins were immunoprecipitated, and analysis of the anti-HA precipitates by Western blotting (right) revealed that Htm1-Myc interacted with full-length Pdi1-HA-HDEL (lane 5) but not with Pdi1-Δ459–HA-HDEL (lane 6). (C) Microsomes were isolated from yeast strains expressing either full-length Htm1-Myc or the indicated truncated variants of Htm1p together with Pdi1-HA-HDEL (lanes 2–7). Immunoblotting of the total lysates confirmed the expression of the tagged proteins (left, top and middle), and immunoprecipitation of Pdi1-HA-HDEL was confirmed (top right). Probing for the Myc-tagged Htm1p constructs revealed a readily detectable interaction between Pdi1-HA-HDEL and Htm1-, Htm1-Δ749–, Htm1-Δ683–, and -Δ647–Myc truncates (lanes 9–12) but not with shorter variants of Htm1p (lanes 13 and 14). Probing of the supernatants after immunopurification with serum against HA confirmed the quantitative depletion of Pdi1-HA-HDEL from the lysates (bottom left). The secondary antibody used in the Western blot also reacted with the heavy chain (HC) of the HA antibody used in the immunopurification (IP).

Htm1p physically interacts with Pdi1p via its C-terminal domain

To address the function of the C-terminal domain more closely, Htm1p-interacting proteins were identified. We generated a functional C-terminally Myc–tandem affinity purification (TAP)–tagged Htm1p construct (unpublished data). Microsomes were prepared, and solubilized Htm1–Myc-TAP protein was precipitated from the supernatant with IgG beads. The purified complexes were resolved by SDS-PAGE, and proteins were stained (Fig. 6 A). The two prominent bands that were visualized around 62 kD were isolated from the gel and subjected to mass spectrometry. Both bands were identified as protein disulfide isomerase 1 protein (Pdi1p; Table S1, available at http://www.jcb.org/cgi/content/full/jcb.200809198/DC1). We wondered whether the C terminus of Htm1p might be required for this interaction. Therefore, Myc-tagged constructs of Htm1p were generated of different truncates within the C terminus, namely at the positions 749, 683, 647, 582, and 547. We also generated Pdi1-HA-HDEL and a C-terminal truncate of Pdi1p at position 459, Pdi1-Δ459–HA-HDEL. Immunopurification of Pdi1-HA-HDEL or Pdi1-Δ459–HA-HDEL was performed, and the presence or absence of full-length or truncated Htm1p-Myc was determined by immunoblot analysis. The results confirmed the interaction of the full-length Pdi1p with Htm1p. However, no interaction could be detected between Pdi1-Δ459–HA-HDEL and Htm1-Myc (Fig. 6 B). When we pulled down full-length Pdi1p–HA-HDEL and assessed the binding of the Htm1p truncates, the interaction was readily detectable with the Htm1-Δ749–, Htm1-Δ683–, and Htm1-Δ647–Myc truncates but not with Htm1-Δ582 and -Δ547 (Fig. 6 C). From this, we deduced that the C terminus is involved in the interaction with Pdi1p.

Htm1p physically interacts with Pdi1p. (A) Microsomes were prepared from control cells or cells expressing Htm1–Myc-TAP. After solubilization, both lysates were incubated with IgG beads. The bound material was subsequently liberated by digestion with tobacco etch virus protease and analyzed by SDS-PAGE. Visualization of the protein content by silver staining revealed Htm1-Myc and two bands around 62 kD that were clearly absent from the control lane (compare lane 2 with lane 3). Proteins were excised from the gel, trypsinized, and subjected to mass spectrometry. Both bands at 62 kD were identified as Pdi1p. The identity of the Htm1-Myc band was also confirmed by mass spectrometry. For size comparison, a molecular weight (MW) standard was loaded in lane 1. (B) Detergent-solubilized microsomes from yeast strains expressing full-length Pdi1-HA-HDEL (lanes 1 and 2) or Pdi1-Δ459–HA-HDEL (lane 3) together with Htm1-Myc (lanes 2 and 3) were analyzed by Western blot (WB) to confirm the presence of the tagged proteins (left). HA-tagged proteins were immunoprecipitated, and analysis of the anti-HA precipitates by Western blotting (right) revealed that Htm1-Myc interacted with full-length Pdi1-HA-HDEL (lane 5) but not with Pdi1-Δ459–HA-HDEL (lane 6). (C) Microsomes were isolated from yeast strains expressing either full-length Htm1-Myc or the indicated truncated variants of Htm1p together with Pdi1-HA-HDEL (lanes 2–7). Immunoblotting of the total lysates confirmed the expression of the tagged proteins (left, top and middle), and immunoprecipitation of Pdi1-HA-HDEL was confirmed (top right). Probing for the Myc-tagged Htm1p constructs revealed a readily detectable interaction between Pdi1-HA-HDEL and Htm1-, Htm1-Δ749–, Htm1-Δ683–, and -Δ647–Myc truncates (lanes 9–12) but not with shorter variants of Htm1p (lanes 13 and 14). Probing of the supernatants after immunopurification with serum against HA confirmed the quantitative depletion of Pdi1-HA-HDEL from the lysates (bottom left). The secondary antibody used in the Western blot also reacted with the heavy chain (HC) of the HA antibody used in the immunopurification (IP).

A terminal α1,6-linked mannose as a glycan determinant directing misfolded glycoproteins to degradationHaving identified the main product of the Htm1p activity as a specific Man7GlcNAc2 isomer, we were wondering whether this glycan, rather than the postulated Man8GlcNAc2 oligosaccharide, might function as the putative glycoprotein degradation signal in S. cerevisiae. Generation of this isomer on a misfolded protein might target its host protein for degradation. In contrast to Man8GlcNAc2, the Man7GlcNAc2 C structure is characterized by a terminal α1,6-linked mannose of the C branch (Fig. 7 A). Within the framework of our hypothesis, we assumed specifically that a terminal α1,6-linked mannosyl residue functioned as a signal required for degradation. Yos9p, the putative ER-degradation lectin with a mannose 6-phosphate receptor homology domain essential for glycoprotein ERAD, might act as the receptor for this structure (Buschhorn et al., 2004; Bhamidipati et al., 2005; Kim et al., 2005; Szathmary et al., 2005). To address this hypothesis directly, we took advantage of the possibility to alter N-glycan structures by manipulating the biosynthetic pathway of lipid-linked oligosaccharide assembly (Jakob et al., 1998b). It is noteworthy that the putative product of Htm1p, the Man7GlcNAc2 oligosaccharide, is not a biosynthetic intermediate in the pathway of N-glycoprotein biosynthesis. However, Δalg3 mutant cells produce a Man5GlcNAc2 glycan that also carries a terminal α1,6-linked mannose linked to an α1,3-substituted mannose, as is the case in the Man7GlcNAc2 C isomer. In contrast, Δalg9 and Δalg12 cells do not produce a terminal α1,6-linked mannose (Fig. 7 A). In support of our hypothesis, wild-type and Δalg3 cells degrade CPY* efficiently, whereas Δalg9 and Δalg12 cells show a strongly reduced degradation rate (Fig. 7, B and C; Jakob et al., 1998a). Moreover, the rapid degradation in wild-type and Δalg3 cells was dependent on the presence of the putative lectin Yos9p (Fig. 7, F and G), but Htm1p was only required in wild-type but not in Δalg3 cells (Fig. 7, D and E). This reinforced our hypothesis that Htm1p is essential in wild-type cells for the generation of a specific glycan signal, a terminal α1,6-linked mannose. This signal is always present in the Δalg3 cell, making degradation of malfolded proteins independent of Htm1p function but still dependent on the Yos9p lectin.Figure 7.HTM1-independent but YOS9-dependent degradation of CPY* in Δalg3 cells. (A) N-glycan structures of the wild-type Man8GlcNAc2 (Mns1p product), Htm1p product Man7GlcNAc2 C, and the N-glycans prevailing in Δalg3, Δalg9, or Δalg12 cells. Man7GlcNAc2 C and Man5GlcNAc2 in Δalg3 both expose an α1,6-linked terminal mannose (circles). (B–G) Analysis of CPY* degradation in different mutant strains. The cells were chased with cycloheximide for the time indicated, extracts were prepared, and the proteins were separated by SDS-PAGE. After transfer to nitrocellulose, CPY* was detected with serum directed against CPY. The position of the band representing CPY* is given at the right. Detection of HXK levels made a quantification of the CPY* degradation in these experiments possible. (B and C) Analysis of wild-type (wt), Δalg3, and Δalg9 cells. (D and E) Analysis of wild-type, Δhtm1, Δalg3, and Δalg3Δhtm1 cells. (F and G) Analysis of wild-type, Δyos9, Δalg3, and Δalg3Δyos9 cells. The values given in the relative quantification experiments are the mean of three independent experiments (B and C are two independent experiments). Standard deviations are indicated.

A terminal α1,6-linked mannose as a glycan determinant directing misfolded glycoproteins to degradation

Having identified the main product of the Htm1p activity as a specific Man7GlcNAc2 isomer, we were wondering whether this glycan, rather than the postulated Man8GlcNAc2 oligosaccharide, might function as the putative glycoprotein degradation signal in S. cerevisiae. Generation of this isomer on a misfolded protein might target its host protein for degradation. In contrast to Man8GlcNAc2, the Man7GlcNAc2 C structure is characterized by a terminal α1,6-linked mannose of the C branch (Fig. 7 A). Within the framework of our hypothesis, we assumed specifically that a terminal α1,6-linked mannosyl residue functioned as a signal required for degradation. Yos9p, the putative ER-degradation lectin with a mannose 6-phosphate receptor homology domain essential for glycoprotein ERAD, might act as the receptor for this structure (Buschhorn et al., 2004; Bhamidipati et al., 2005; Kim et al., 2005; Szathmary et al., 2005). To address this hypothesis directly, we took advantage of the possibility to alter N-glycan structures by manipulating the biosynthetic pathway of lipid-linked oligosaccharide assembly (Jakob et al., 1998b). It is noteworthy that the putative product of Htm1p, the Man7GlcNAc2 oligosaccharide, is not a biosynthetic intermediate in the pathway of N-glycoprotein biosynthesis. However, Δalg3 mutant cells produce a Man5GlcNAc2 glycan that also carries a terminal α1,6-linked mannose linked to an α1,3-substituted mannose, as is the case in the Man7GlcNAc2 C isomer. In contrast, Δalg9 and Δalg12 cells do not produce a terminal α1,6-linked mannose (Fig. 7 A). In support of our hypothesis, wild-type and Δalg3 cells degrade CPY* efficiently, whereas Δalg9 and Δalg12 cells show a strongly reduced degradation rate (Fig. 7, B and C; Jakob et al., 1998a). Moreover, the rapid degradation in wild-type and Δalg3 cells was dependent on the presence of the putative lectin Yos9p (Fig. 7, F and G), but Htm1p was only required in wild-type but not in Δalg3 cells (Fig. 7, D and E). This reinforced our hypothesis that Htm1p is essential in wild-type cells for the generation of a specific glycan signal, a terminal α1,6-linked mannose. This signal is always present in the Δalg3 cell, making degradation of malfolded proteins independent of Htm1p function but still dependent on the Yos9p lectin.

HTM1-independent but YOS9-dependent degradation of CPY* in Δalg3 cells. (A) N-glycan structures of the wild-type Man8GlcNAc2 (Mns1p product), Htm1p product Man7GlcNAc2 C, and the N-glycans prevailing in Δalg3, Δalg9, or Δalg12 cells. Man7GlcNAc2 C and Man5GlcNAc2 in Δalg3 both expose an α1,6-linked terminal mannose (circles). (B–G) Analysis of CPY* degradation in different mutant strains. The cells were chased with cycloheximide for the time indicated, extracts were prepared, and the proteins were separated by SDS-PAGE. After transfer to nitrocellulose, CPY* was detected with serum directed against CPY. The position of the band representing CPY* is given at the right. Detection of HXK levels made a quantification of the CPY* degradation in these experiments possible. (B and C) Analysis of wild-type (wt), Δalg3, and Δalg9 cells. (D and E) Analysis of wild-type, Δhtm1, Δalg3, and Δalg3Δhtm1 cells. (F and G) Analysis of wild-type, Δyos9, Δalg3, and Δalg3Δyos9 cells. The values given in the relative quantification experiments are the mean of three independent experiments (B and C are two independent experiments). Standard deviations are indicated.

DiscussionInitial findings suggested that the Man8GlcNAc2 B-glycan may serve as a glycan signal for degradation in S. cerevisiae and mammalian cells. Htm1p/EDEM1 and more recently Yos9p were proposed to act as lectins or lectinlike proteins that specifically recognize misfolded glycoproteins carrying Man8GlcNAc2 glycans and by this initiate glycoprotein ERAD. In this study, we analyzed the function of Htm1p in more detail.Htm1p is an exomannosidase involved in ERADWe detected the removal of one mannose upon HTM1 overexpression yielding the Man7GlcNAc2 C-glycan, but this did not increase the degradation rate of the model protein CPY*. This suggests that Htm1p is an α1,2-exomannosidase acting on the C branch of the N-linked glycan of glycoproteins. Importantly, overexpression of mutant Htm1p did not result in an increase in the overall Man7GlcNAc2 glycan level, and these mutant forms were inactive in ERAD function. From this, we propose that the Man7GlcNAc2 glycan generated by Htm1p on ERAD substrates serves as the N-glycan degradation signal, but Htm1p activity is not limiting in the degradation pathway. In this study, we addressed several aspects of this model: (a) what is the nature of the glycan signal; (b) how is the signal specifically generated on an ERAD substrate; and (c) what is the receptor of the glycan signal in the ERAD pathway?A terminal α1,6-linked mannosyl residue is the product of Htm1p activityThe combined actions of ER mannosidase I and Htm1p result in a unique protein-bound high mannose structure not found during the biosynthesis of the lipid-linked oligosaccharide substrate for N-linked protein glycosylation (Burda and Aebi, 1999). ER mannosidase I produces a terminal α1,3-linked mannose on the B branch, whereas the action of Htm1p removes the capping α1,2-mannose of the C branch, resulting in a terminal α1,6-linked mannose. Interestingly, a similar structure is present on the Man5GlcNAc2 oligosaccharide found in Δalg3 mutant cells. Both contain a terminal α1,6-linked mannose (Fig. 7 A). As reported earlier (Jakob et al., 1998a) and in this study, degradation of CPY* is independent of glycan processing in Δalg3 mutant cells, suggesting that the signal for degradation is present on the unprocessed Δalg3 glycan. Therefore, we postulate that Htm1p-dependent processing unmasks the α1,6-linked mannose residue, and this structure is an essential functionality in the recognition of misfolded glycoproteins.It has been proposed that mammalian homologues of Htm1p (EDEM1 and EDEM3) also act on the α1,2-linked mannose residues of the A branch (Hirao et al., 2006; Olivari et al., 2006). Our data do not allow us to formally exclude the possibility that this is also the case for Htm1p. However, analysis of the free N-linked glycans, the putative products of ERAD present in the cytoplasm of yeast, reveal that Man8GlcNAc2 and Man7GlcNAc2 are the predominant forms (Chantret et al., 2003; Suzuki and Funakoshi, 2006; unpublished data).Substrate specificity of Htm1pOur model suggests that Htm1p has a dual substrate specificity; it requires a well-defined oligosaccharide structure present on an unfolded protein. Our data showed that the processing of the N-glycan by ER mannosidase I was essential for efficient generation of the α1,6-linked mannose by Htm1p (Fig. 3). We propose that substrate recognition by Htm1p involves a terminal α1,3-mannosyl residue generated by ER mannosidase I. This explains the observation that glycoprotein ERAD requires trimming by ER mannosidase I both in yeast and in mammalian cell culture systems (Liu et al., 1997; Jakob et al., 1998a; Yang et al., 1998).Our analysis did not allow us to address the recognition of the substrate protein domains by Htm1p directly. Detailed work by Ng and others revealed that glycans localized on specific sites of ERAD substrates are required for degradation, whereas others are dispensable (Kostova and Wolf, 2005; Spear and Ng, 2005; unpublished data; D. Ng, personal communication). Therefore, only a minor portion of N-linked glycans might be processed by Htm1p. Our observation that overexpression of this protein was required to visualize small amounts of processing products is in line with this hypothesis. It is tempting to speculate that the C-terminal domain of Htm1p is required for the recognition of the protein part of the substrate. An appealing explanation could be that this part of the protein directly or indirectly binds misfolded peptides or hydrophobic patches and thus mediates substrate selection. In support of this proposal is our finding that the C terminus was required for the interaction with Pdi1p (Fig. 6 C). This interaction had previously been reported from a high throughput proteome-wide purification of protein complexes from S. cerevisiae (Krogan et al., 2006; Collins et al., 2007). Interestingly, Pdi1p has been shown to recognize terminally misfolded secretory proteins and target them to the retrotranslocation (Gillece et al., 1999).Yos9p recognizes the α1,6-linked terminal mannoseThe specifically modified N-glycan as a signal for ERAD requires decoding by a lectin. Several groups have reported on the role of the putative lectin Yos9p in the process of ERAD (Buschhorn et al., 2004; Bhamidipati et al., 2005; Kim et al., 2005; Szathmary et al., 2005; Carvalho et al., 2006; Denic et al., 2006; Gauss et al., 2006a). Yos9p is part of a large complex that, among other factors, comprises the Hrd1p/Hrd3p E3 ligase. The interaction of a nonfolded protein with Hrd3p is believed to provide the first signal for degradation, and Yos9p further inspects the glycans of the protein. If this glycan signals terminal misfolding, the dual recognition triggers the ubiquitination and proteasomal degradation of the substrate protein (Denic et al., 2006; Gauss et al., 2006a). Our genetic data support the hypothesis that Yos9p specifically binds to the terminal α1,6-linked mannosyl residue generated by the Htm1p activity. Work by the group of J.S. Weissman corroborates this proposal by showing that the Yos9p lectin indeed prefers glycan structures that contain a terminal α1,6-linked mannose (Quan et al., 2008). In addition, the in vivo immunoprecipitation experiments performed with CPY*-HA and Yos9-ProA-His7 protein revealed physical interaction of these proteins in wild-type cells in which the signal is generated by Htm1p and in the Δalg3 mutant cells that expose the signal as a result of incomplete N-glycan biosynthesis (Szathmary et al., 2005).Comparison of Htm1p activity and EDEM function in mammalian cellsRecent studies suggest that EDEM1 as well as EDEM3 act as enzymes rather than as lectins and enhance the demannosylation of ERAD substrates in vivo (Hirao et al., 2006; Olivari et al., 2006). In view of the conservation of essential residues of the processing mannosidases, our finding that Htm1p has mannosidase activity in vivo is in accordance with the described activities of the mammalian orthologues.Although direct structural analysis of processing products are missing, indirect evidence suggests that mammalian EDEM proteins have a dual function; they process the terminal mannose residues of the A branch and thereby remove the UDP-glucose/glycoprotein glucosyltransferase acceptor, thus extracting the proteins from the calnexin/calreticulin cycle. However, N-glycans of misfolded glycoproteins are trimmed further down to Man6GlcNAc2 and Man5GlcNAc2 in mammalian cells. Besides the EDEM family, other processing mannosidases of the glycosyl-hydrolase 47 family, like Golgi mannosidases and ER mannosidase I, have been shown to contribute to the enhanced N-glycan processing of misfolded glycoproteins (Moremen and Molinari, 2006; Molinari, 2007).We extend the model of glycan signaling in ERAD of mammalian cells and propose that the extensive mannose trimming serves two distinct goals: the removal of the terminal α1,2-linked mannose of the A branch ensures the exit from the calnexin/calreticulin cycle, whereas the additional trimming of the C branch generates the terminal misfolding signal and targets the N-glycoprotein for degradation. This dual activity explains the biphasic mode of glycoprotein degradation in mammalian cells, a lag phase in which the protein remains in the calnexin/calreticulin cycle and is protected from degradation followed by the actual degradation phase (Molinari, 2007). Mannose trimming of the A branch by EDEM mannosidases releases the protein from the protective calnexin/calreticulin cycle and allows degradation. Indeed, overexpression of these enzymes was shown to result in an earlier onset of degradation (Molinari et al., 2003; Molinari, 2007). In view of our results, we propose that degradation itself requires the processing of the C branch to generate the α1,6-linked mannose as a degradation signal. It is possible that EDEM proteins are able to trim both the A and the C branch of the N-linked glycan. The absence of a calnexin/calreticulin cycle in S. cerevisiae made it possible to analyze the generation of this degradation signal directly.It is a unique feature of the eukaryotic N-linked protein glycosylation that a complex oligosaccharide of a defined structure can be transferred to multiple sites characterized by a short consensus sequence, N-X-S/T, of very diverse secretory proteins. Therefore, this posttranslational modification tags many different proteins with a unique structure that is processed in the ER by a set of hydrolases and glycosyltransferases. They act in a sequential order, and some of these enzymes modify only N-glycans of unfolded proteins. This orchestrated processing results in a set of defined glycan signals that can be interpreted by a general folding and degradation machinery (Helenius and Aebi, 2004). Our work reveals the identity of one of these signals, the terminal α1,6-linked mannose residue generated by the exomannosidase activity of Htm1p. Our data suggest that this signal is interpreted by the lectin Yos9p, and we propose that this interaction communicates to the multicomponent degradation platform (Carvalho et al., 2006; Denic et al., 2006; Gauss et al., 2006a) that an unfolded bound protein is destined for destruction. The complete bipartite signal leads to the retrotranslocation, ubiquitination, and proteasomal degradation of the substrate protein. However, we do not know how Htm1p activity is regulated by the presence of an unfolded protein domain or how the folding status of many different glycoproteins can modulate this mannosidase activity. A detailed biochemical analysis of the EDEM/Htm1 protein family will clarify this issue.

Initial findings suggested that the Man8GlcNAc2 B-glycan may serve as a glycan signal for degradation in S. cerevisiae and mammalian cells. Htm1p/EDEM1 and more recently Yos9p were proposed to act as lectins or lectinlike proteins that specifically recognize misfolded glycoproteins carrying Man8GlcNAc2 glycans and by this initiate glycoprotein ERAD. In this study, we analyzed the function of Htm1p in more detail.

Htm1p is an exomannosidase involved in ERADWe detected the removal of one mannose upon HTM1 overexpression yielding the Man7GlcNAc2 C-glycan, but this did not increase the degradation rate of the model protein CPY*. This suggests that Htm1p is an α1,2-exomannosidase acting on the C branch of the N-linked glycan of glycoproteins. Importantly, overexpression of mutant Htm1p did not result in an increase in the overall Man7GlcNAc2 glycan level, and these mutant forms were inactive in ERAD function. From this, we propose that the Man7GlcNAc2 glycan generated by Htm1p on ERAD substrates serves as the N-glycan degradation signal, but Htm1p activity is not limiting in the degradation pathway. In this study, we addressed several aspects of this model: (a) what is the nature of the glycan signal; (b) how is the signal specifically generated on an ERAD substrate; and (c) what is the receptor of the glycan signal in the ERAD pathway?

Htm1p is an exomannosidase involved in ERAD

We detected the removal of one mannose upon HTM1 overexpression yielding the Man7GlcNAc2 C-glycan, but this did not increase the degradation rate of the model protein CPY*. This suggests that Htm1p is an α1,2-exomannosidase acting on the C branch of the N-linked glycan of glycoproteins. Importantly, overexpression of mutant Htm1p did not result in an increase in the overall Man7GlcNAc2 glycan level, and these mutant forms were inactive in ERAD function. From this, we propose that the Man7GlcNAc2 glycan generated by Htm1p on ERAD substrates serves as the N-glycan degradation signal, but Htm1p activity is not limiting in the degradation pathway. In this study, we addressed several aspects of this model: (a) what is the nature of the glycan signal; (b) how is the signal specifically generated on an ERAD substrate; and (c) what is the receptor of the glycan signal in the ERAD pathway?

A terminal α1,6-linked mannosyl residue is the product of Htm1p activityThe combined actions of ER mannosidase I and Htm1p result in a unique protein-bound high mannose structure not found during the biosynthesis of the lipid-linked oligosaccharide substrate for N-linked protein glycosylation (Burda and Aebi, 1999). ER mannosidase I produces a terminal α1,3-linked mannose on the B branch, whereas the action of Htm1p removes the capping α1,2-mannose of the C branch, resulting in a terminal α1,6-linked mannose. Interestingly, a similar structure is present on the Man5GlcNAc2 oligosaccharide found in Δalg3 mutant cells. Both contain a terminal α1,6-linked mannose (Fig. 7 A). As reported earlier (Jakob et al., 1998a) and in this study, degradation of CPY* is independent of glycan processing in Δalg3 mutant cells, suggesting that the signal for degradation is present on the unprocessed Δalg3 glycan. Therefore, we postulate that Htm1p-dependent processing unmasks the α1,6-linked mannose residue, and this structure is an essential functionality in the recognition of misfolded glycoproteins.It has been proposed that mammalian homologues of Htm1p (EDEM1 and EDEM3) also act on the α1,2-linked mannose residues of the A branch (Hirao et al., 2006; Olivari et al., 2006). Our data do not allow us to formally exclude the possibility that this is also the case for Htm1p. However, analysis of the free N-linked glycans, the putative products of ERAD present in the cytoplasm of yeast, reveal that Man8GlcNAc2 and Man7GlcNAc2 are the predominant forms (Chantret et al., 2003; Suzuki and Funakoshi, 2006; unpublished data).

A terminal α1,6-linked mannosyl residue is the product of Htm1p activity

The combined actions of ER mannosidase I and Htm1p result in a unique protein-bound high mannose structure not found during the biosynthesis of the lipid-linked oligosaccharide substrate for N-linked protein glycosylation (Burda and Aebi, 1999). ER mannosidase I produces a terminal α1,3-linked mannose on the B branch, whereas the action of Htm1p removes the capping α1,2-mannose of the C branch, resulting in a terminal α1,6-linked mannose. Interestingly, a similar structure is present on the Man5GlcNAc2 oligosaccharide found in Δalg3 mutant cells. Both contain a terminal α1,6-linked mannose (Fig. 7 A). As reported earlier (Jakob et al., 1998a) and in this study, degradation of CPY* is independent of glycan processing in Δalg3 mutant cells, suggesting that the signal for degradation is present on the unprocessed Δalg3 glycan. Therefore, we postulate that Htm1p-dependent processing unmasks the α1,6-linked mannose residue, and this structure is an essential functionality in the recognition of misfolded glycoproteins.

It has been proposed that mammalian homologues of Htm1p (EDEM1 and EDEM3) also act on the α1,2-linked mannose residues of the A branch (Hirao et al., 2006; Olivari et al., 2006). Our data do not allow us to formally exclude the possibility that this is also the case for Htm1p. However, analysis of the free N-linked glycans, the putative products of ERAD present in the cytoplasm of yeast, reveal that Man8GlcNAc2 and Man7GlcNAc2 are the predominant forms (Chantret et al., 2003; Suzuki and Funakoshi, 2006; unpublished data).

Substrate specificity of Htm1pOur model suggests that Htm1p has a dual substrate specificity; it requires a well-defined oligosaccharide structure present on an unfolded protein. Our data showed that the processing of the N-glycan by ER mannosidase I was essential for efficient generation of the α1,6-linked mannose by Htm1p (Fig. 3). We propose that substrate recognition by Htm1p involves a terminal α1,3-mannosyl residue generated by ER mannosidase I. This explains the observation that glycoprotein ERAD requires trimming by ER mannosidase I both in yeast and in mammalian cell culture systems (Liu et al., 1997; Jakob et al., 1998a; Yang et al., 1998).Our analysis did not allow us to address the recognition of the substrate protein domains by Htm1p directly. Detailed work by Ng and others revealed that glycans localized on specific sites of ERAD substrates are required for degradation, whereas others are dispensable (Kostova and Wolf, 2005; Spear and Ng, 2005; unpublished data; D. Ng, personal communication). Therefore, only a minor portion of N-linked glycans might be processed by Htm1p. Our observation that overexpression of this protein was required to visualize small amounts of processing products is in line with this hypothesis. It is tempting to speculate that the C-terminal domain of Htm1p is required for the recognition of the protein part of the substrate. An appealing explanation could be that this part of the protein directly or indirectly binds misfolded peptides or hydrophobic patches and thus mediates substrate selection. In support of this proposal is our finding that the C terminus was required for the interaction with Pdi1p (Fig. 6 C). This interaction had previously been reported from a high throughput proteome-wide purification of protein complexes from S. cerevisiae (Krogan et al., 2006; Collins et al., 2007). Interestingly, Pdi1p has been shown to recognize terminally misfolded secretory proteins and target them to the retrotranslocation (Gillece et al., 1999).

Substrate specificity of Htm1p

Our model suggests that Htm1p has a dual substrate specificity; it requires a well-defined oligosaccharide structure present on an unfolded protein. Our data showed that the processing of the N-glycan by ER mannosidase I was essential for efficient generation of the α1,6-linked mannose by Htm1p (Fig. 3). We propose that substrate recognition by Htm1p involves a terminal α1,3-mannosyl residue generated by ER mannosidase I. This explains the observation that glycoprotein ERAD requires trimming by ER mannosidase I both in yeast and in mammalian cell culture systems (Liu et al., 1997; Jakob et al., 1998a; Yang et al., 1998).

Our analysis did not allow us to address the recognition of the substrate protein domains by Htm1p directly. Detailed work by Ng and others revealed that glycans localized on specific sites of ERAD substrates are required for degradation, whereas others are dispensable (Kostova and Wolf, 2005; Spear and Ng, 2005; unpublished data; D. Ng, personal communication). Therefore, only a minor portion of N-linked glycans might be processed by Htm1p. Our observation that overexpression of this protein was required to visualize small amounts of processing products is in line with this hypothesis. It is tempting to speculate that the C-terminal domain of Htm1p is required for the recognition of the protein part of the substrate. An appealing explanation could be that this part of the protein directly or indirectly binds misfolded peptides or hydrophobic patches and thus mediates substrate selection. In support of this proposal is our finding that the C terminus was required for the interaction with Pdi1p (Fig. 6 C). This interaction had previously been reported from a high throughput proteome-wide purification of protein complexes from S. cerevisiae (Krogan et al., 2006; Collins et al., 2007). Interestingly, Pdi1p has been shown to recognize terminally misfolded secretory proteins and target them to the retrotranslocation (Gillece et al., 1999).

Yos9p recognizes the α1,6-linked terminal mannoseThe specifically modified N-glycan as a signal for ERAD requires decoding by a lectin. Several groups have reported on the role of the putative lectin Yos9p in the process of ERAD (Buschhorn et al., 2004; Bhamidipati et al., 2005; Kim et al., 2005; Szathmary et al., 2005; Carvalho et al., 2006; Denic et al., 2006; Gauss et al., 2006a). Yos9p is part of a large complex that, among other factors, comprises the Hrd1p/Hrd3p E3 ligase. The interaction of a nonfolded protein with Hrd3p is believed to provide the first signal for degradation, and Yos9p further inspects the glycans of the protein. If this glycan signals terminal misfolding, the dual recognition triggers the ubiquitination and proteasomal degradation of the substrate protein (Denic et al., 2006; Gauss et al., 2006a). Our genetic data support the hypothesis that Yos9p specifically binds to the terminal α1,6-linked mannosyl residue generated by the Htm1p activity. Work by the group of J.S. Weissman corroborates this proposal by showing that the Yos9p lectin indeed prefers glycan structures that contain a terminal α1,6-linked mannose (Quan et al., 2008). In addition, the in vivo immunoprecipitation experiments performed with CPY*-HA and Yos9-ProA-His7 protein revealed physical interaction of these proteins in wild-type cells in which the signal is generated by Htm1p and in the Δalg3 mutant cells that expose the signal as a result of incomplete N-glycan biosynthesis (Szathmary et al., 2005).

Yos9p recognizes the α1,6-linked terminal mannose

The specifically modified N-glycan as a signal for ERAD requires decoding by a lectin. Several groups have reported on the role of the putative lectin Yos9p in the process of ERAD (Buschhorn et al., 2004; Bhamidipati et al., 2005; Kim et al., 2005; Szathmary et al., 2005; Carvalho et al., 2006; Denic et al., 2006; Gauss et al., 2006a). Yos9p is part of a large complex that, among other factors, comprises the Hrd1p/Hrd3p E3 ligase. The interaction of a nonfolded protein with Hrd3p is believed to provide the first signal for degradation, and Yos9p further inspects the glycans of the protein. If this glycan signals terminal misfolding, the dual recognition triggers the ubiquitination and proteasomal degradation of the substrate protein (Denic et al., 2006; Gauss et al., 2006a). Our genetic data support the hypothesis that Yos9p specifically binds to the terminal α1,6-linked mannosyl residue generated by the Htm1p activity. Work by the group of J.S. Weissman corroborates this proposal by showing that the Yos9p lectin indeed prefers glycan structures that contain a terminal α1,6-linked mannose (Quan et al., 2008). In addition, the in vivo immunoprecipitation experiments performed with CPY*-HA and Yos9-ProA-His7 protein revealed physical interaction of these proteins in wild-type cells in which the signal is generated by Htm1p and in the Δalg3 mutant cells that expose the signal as a result of incomplete N-glycan biosynthesis (Szathmary et al., 2005).

Comparison of Htm1p activity and EDEM function in mammalian cellsRecent studies suggest that EDEM1 as well as EDEM3 act as enzymes rather than as lectins and enhance the demannosylation of ERAD substrates in vivo (Hirao et al., 2006; Olivari et al., 2006). In view of the conservation of essential residues of the processing mannosidases, our finding that Htm1p has mannosidase activity in vivo is in accordance with the described activities of the mammalian orthologues.Although direct structural analysis of processing products are missing, indirect evidence suggests that mammalian EDEM proteins have a dual function; they process the terminal mannose residues of the A branch and thereby remove the UDP-glucose/glycoprotein glucosyltransferase acceptor, thus extracting the proteins from the calnexin/calreticulin cycle. However, N-glycans of misfolded glycoproteins are trimmed further down to Man6GlcNAc2 and Man5GlcNAc2 in mammalian cells. Besides the EDEM family, other processing mannosidases of the glycosyl-hydrolase 47 family, like Golgi mannosidases and ER mannosidase I, have been shown to contribute to the enhanced N-glycan processing of misfolded glycoproteins (Moremen and Molinari, 2006; Molinari, 2007).We extend the model of glycan signaling in ERAD of mammalian cells and propose that the extensive mannose trimming serves two distinct goals: the removal of the terminal α1,2-linked mannose of the A branch ensures the exit from the calnexin/calreticulin cycle, whereas the additional trimming of the C branch generates the terminal misfolding signal and targets the N-glycoprotein for degradation. This dual activity explains the biphasic mode of glycoprotein degradation in mammalian cells, a lag phase in which the protein remains in the calnexin/calreticulin cycle and is protected from degradation followed by the actual degradation phase (Molinari, 2007). Mannose trimming of the A branch by EDEM mannosidases releases the protein from the protective calnexin/calreticulin cycle and allows degradation. Indeed, overexpression of these enzymes was shown to result in an earlier onset of degradation (Molinari et al., 2003; Molinari, 2007). In view of our results, we propose that degradation itself requires the processing of the C branch to generate the α1,6-linked mannose as a degradation signal. It is possible that EDEM proteins are able to trim both the A and the C branch of the N-linked glycan. The absence of a calnexin/calreticulin cycle in S. cerevisiae made it possible to analyze the generation of this degradation signal directly.It is a unique feature of the eukaryotic N-linked protein glycosylation that a complex oligosaccharide of a defined structure can be transferred to multiple sites characterized by a short consensus sequence, N-X-S/T, of very diverse secretory proteins. Therefore, this posttranslational modification tags many different proteins with a unique structure that is processed in the ER by a set of hydrolases and glycosyltransferases. They act in a sequential order, and some of these enzymes modify only N-glycans of unfolded proteins. This orchestrated processing results in a set of defined glycan signals that can be interpreted by a general folding and degradation machinery (Helenius and Aebi, 2004). Our work reveals the identity of one of these signals, the terminal α1,6-linked mannose residue generated by the exomannosidase activity of Htm1p. Our data suggest that this signal is interpreted by the lectin Yos9p, and we propose that this interaction communicates to the multicomponent degradation platform (Carvalho et al., 2006; Denic et al., 2006; Gauss et al., 2006a) that an unfolded bound protein is destined for destruction. The complete bipartite signal leads to the retrotranslocation, ubiquitination, and proteasomal degradation of the substrate protein. However, we do not know how Htm1p activity is regulated by the presence of an unfolded protein domain or how the folding status of many different glycoproteins can modulate this mannosidase activity. A detailed biochemical analysis of the EDEM/Htm1 protein family will clarify this issue.

Comparison of Htm1p activity and EDEM function in mammalian cells

Recent studies suggest that EDEM1 as well as EDEM3 act as enzymes rather than as lectins and enhance the demannosylation of ERAD substrates in vivo (Hirao et al., 2006; Olivari et al., 2006). In view of the conservation of essential residues of the processing mannosidases, our finding that Htm1p has mannosidase activity in vivo is in accordance with the described activities of the mammalian orthologues.

Although direct structural analysis of processing products are missing, indirect evidence suggests that mammalian EDEM proteins have a dual function; they process the terminal mannose residues of the A branch and thereby remove the UDP-glucose/glycoprotein glucosyltransferase acceptor, thus extracting the proteins from the calnexin/calreticulin cycle. However, N-glycans of misfolded glycoproteins are trimmed further down to Man6GlcNAc2 and Man5GlcNAc2 in mammalian cells. Besides the EDEM family, other processing mannosidases of the glycosyl-hydrolase 47 family, like Golgi mannosidases and ER mannosidase I, have been shown to contribute to the enhanced N-glycan processing of misfolded glycoproteins (Moremen and Molinari, 2006; Molinari, 2007).

We extend the model of glycan signaling in ERAD of mammalian cells and propose that the extensive mannose trimming serves two distinct goals: the removal of the terminal α1,2-linked mannose of the A branch ensures the exit from the calnexin/calreticulin cycle, whereas the additional trimming of the C branch generates the terminal misfolding signal and targets the N-glycoprotein for degradation. This dual activity explains the biphasic mode of glycoprotein degradation in mammalian cells, a lag phase in which the protein remains in the calnexin/calreticulin cycle and is protected from degradation followed by the actual degradation phase (Molinari, 2007). Mannose trimming of the A branch by EDEM mannosidases releases the protein from the protective calnexin/calreticulin cycle and allows degradation. Indeed, overexpression of these enzymes was shown to result in an earlier onset of degradation (Molinari et al., 2003; Molinari, 2007). In view of our results, we propose that degradation itself requires the processing of the C branch to generate the α1,6-linked mannose as a degradation signal. It is possible that EDEM proteins are able to trim both the A and the C branch of the N-linked glycan. The absence of a calnexin/calreticulin cycle in S. cerevisiae made it possible to analyze the generation of this degradation signal directly.

It is a unique feature of the eukaryotic N-linked protein glycosylation that a complex oligosaccharide of a defined structure can be transferred to multiple sites characterized by a short consensus sequence, N-X-S/T, of very diverse secretory proteins. Therefore, this posttranslational modification tags many different proteins with a unique structure that is processed in the ER by a set of hydrolases and glycosyltransferases. They act in a sequential order, and some of these enzymes modify only N-glycans of unfolded proteins. This orchestrated processing results in a set of defined glycan signals that can be interpreted by a general folding and degradation machinery (Helenius and Aebi, 2004). Our work reveals the identity of one of these signals, the terminal α1,6-linked mannose residue generated by the exomannosidase activity of Htm1p. Our data suggest that this signal is interpreted by the lectin Yos9p, and we propose that this interaction communicates to the multicomponent degradation platform (Carvalho et al., 2006; Denic et al., 2006; Gauss et al., 2006a) that an unfolded bound protein is destined for destruction. The complete bipartite signal leads to the retrotranslocation, ubiquitination, and proteasomal degradation of the substrate protein. However, we do not know how Htm1p activity is regulated by the presence of an unfolded protein domain or how the folding status of many different glycoproteins can modulate this mannosidase activity. A detailed biochemical analysis of the EDEM/Htm1 protein family will clarify this issue.

Materials and methodsYeast strains and plasmidsYeast strains used in this study are listed in Table S2 (available at http://www.jcb.org/cgi/content/full/jcb.200809198/DC1; Vijayraghavan et al., 1989; Biederer et al., 1997; Jakob et al., 1998a, 2001). Standard yeast media and genetic techniques were used (Guthrie and Fink, 1991). YG695, YSC8, YSC17, YSC24, YG1330, and YDO1 are haploid descendants of the wild-type strains SS328/SS330. YG1330 and YDO1 were generated using the pYM C-terminal epitope-tagging method (Knop et al., 1999). In brief, the TPH7 tag together with the KanMX6 cassette was amplified by PCR. The PCR fragment was inserted into the genome of YG618 by homologous recombination at the 3′ end of the HTM1 locus or at the 3′ end of the mannosidase homology region between Asn517 and Asn518, respectively. YTX140, YCH030, YCH036, YCH046, YCH133, YCH135, YCH137, YCH175, YCH176, YCH177, and YCH178 are haploid descendants of DF5. PCR-based strategies were used to introduce the epitope tags to Htm1p and Pdi1p in these strains. All epitope-tagged proteins were expressed from their natural chromosomal locus under control of the endogenous promoter.The YEp352-HTM1 (pHTM1u-1) plasmid was constructed by transferring the HTM1 gene from pRS316-HTM1 (pHTM1-7) to the high copy number vector YEp352 plasmid using XhoI–SacI restriction endonucleases. YEp352-HTM1-TPH7 (pDOG3) and YEp352-htm1Δ518-TPH7 (pDOG4) were generated by PCR amplification of the modified HTM1 loci from YG1330 and YDO1 and subcloning into YEp352. Mutations in full-length YEp352-HTM1-TPH7 were introduced by the QuikChange method (Agilent Technologies), yielding YEp352-htm1-E222Q-TPH7 (pDOG5) and YEp352-htm1-D279N-TPH7 (pDOG6). Plasmids were confirmed by DNA sequence analysis.Analysis of CPY* degradationThe cycloheximide chase was performed as described previously (Jakob et al., 1998a). In brief, yeast cells were grown at 30°C in appropriate medium to mid-log phase corresponding to an OD600nm of 0.8–1.2. 3 × 108 cells were harvested and resuspended in medium. The chase was initiated by addition of cycloheximide (final concentration of 100 μg/ml) and was performed at 30°C. 108 cells were removed at each time point and transferred into NaN3 (final concentration of 0.1% [wt/vol]) on ice, immediately pelleted, and flash frozen in liquid nitrogen. Whole cell protein extracts were prepared using glass beads, 1% (wt/vol) SDS, 50 mM Tris-HCl, pH 7.5, and 2 mM PMSF. Proteins were subjected to reducing 7% SDS-PAGE and electroblotted to nitrocellulose membranes. CPY* was immunologically detected with anti-CPY antibody (rabbit) at a dilution of 1:1,000 (Zufferey et al., 1995) and goat anti–rabbit IgG–horseradish peroxidase at 1:3,000 (Santa Cruz Biotechnology, Inc.). Visualization was performed with ECL detection (GE Healthcare). Reprobing of membranes with anti-hexokinase (HXK) antibody (rabbit) was performed accordingly. The x-ray films (super RX; Fujifilm) were scanned, and the CPY* protein amounts were determined densitometrically with a molecular imager (FX; Bio-Rad Laboratories) using the Quantity One program (Bio-Rad Laboratories).[3H]mannose labeling and analysis of N-linked glycansCells were grown at 30°C in appropriate medium to mid-log phase corresponding to an OD600nm of 0.8–1.2. n × 5 × 108 cells (n = number of analyzed time points) were harvested and washed in YPD (1% yeast extract, 2% peptone, and 2% glucose) containing 0.1% glucose (YP0.1%D). Labeling was performed in n × 200 μl YP0.1%D containing n × 100 μCi d-2-[3H]mannose (500 GBq/mmol; Hartmann Analytic) and incubated for 10 (Figs. 2–5) or 20 min (Fig. 1) at 30°C. The radioactivity was chased by the addition of d-mannose to a 2% (weight per volume) final concentration at 30°C incubation. The chase was stopped by removing 5 × 108 cells into TCA on ice (final concentration of 10% [vol/vol]). TCA precipitates were washed twice with cold acetone and air dried for 20 min. The pellets were resuspended in buffer S (20 mM NaP, pH 7.5, 0.5% SDS, and 40 mM DTT) and vortexed with glass beads at 50°C for 1 h. Lysates were cleared by centrifugation, and iodoacetamide was added to a 50-mM final concentration. Samples were incubated for 30 min at 37°C. Oligosaccharides were cleaved from proteins by digestion with peptide N-glycosidase F according to the recommendations of the manufacturer (BioConcept). Cleanup of oligosaccharides was modified from Grubenmann et al. (2004). Prepacked C18 Sep Pak columns (Waters) were connected to columns (extended-volume empty reservoirs; Socochim SA) that were packed with Supelclean ENVI-Carb 120/400 (Sigma-Aldrich). The combined columns were equilibrated with methanol, acetonitrile, acetonitrile/H2O (25:75; vol/vol), and acetonitrile/H2O (2:98; vol/vol). Samples were loaded onto the columns after supplementation with acetonitrile/H2O (2:98; vol/vol). Columns were washed with acetonitrile/H2O (2:98; vol/vol), and glycans were eluted from the ENVI-Carb column with acetonitrile/H2O (25:75; vol/vol). The solvent was evaporated in a speed vacuum.For HPLC analysis of the oligosaccharides, a liquid chromatography (LC)-NH2 column (250 × 4.6 mm; Sigma-Aldrich; Cacan et al., 1993) including an LC-NH2 guard column was used. Oligosaccharide samples in acetonitrile/H2O (70:30; vol/vol) were filtered through a 0.45-μm filter (Millipore) and injected on the equilibrated (acetonitrile/H2O [70:30; vol/vol]; 30 min) system using an autosampling device (Merck/Hitachi AS-2000; Sigma-Aldrich). The gradient used was acetonitrile/H2O (70:30; vol/vol) to acetonitrile/H2O (45:50; vol/vol) for >90 min, 5 min at acetonitrile/H2O (45:50; vol/vol), returning to acetonitrile/H2O (70:30; vol/vol) for >5 min, and washing for 20 min at acetonitrile/H2O (70:30; vol/vol) before injection of the following sample using a pump (Merck/Hitachi L-2600A; Sigma-Aldrich). The eluate from the column was mixed continuously with scintillation fluid (FLO-Scint A; PerkinElmer) in a ratio of 1:1.5 (eluate/scintillation mix; vol/vol), and radioactivity was monitored with a flow monitor (FLO-ONE A-525; PerkinElmer) as described previously (Zufferey et al., 1995). Quantification of the peaks was performed using the PeakFit 4.06 program (SPSS) using the second derivative method.Preparative purification of [3H]mannose-labeled N-linked glycans[3H]mannose labeling was performed as described in the previous section with the following modifications: 4 × 109 cells were labeled with 800 μCi d-2-[3H]mannose in 2 ml YP0.1%D. The labeling was stopped by addition of TCA to the final concentration of 10% (vol/vol). The samples were worked up each in two batches and after the elution from the ENVI-Carb column were pooled again. An analytical HPLC run was performed with the N-glycans from 5 × 108 cells. For isolation of individual N-glycans, the LC-NH2 column was disconnected from the flow monitor, and elution fractions of 500 μl were manually collected. The radioactivity in the fractions was determined using a β-counter (Tri-Carb 2800TR; PerkinElmer) with the QuantaSmart program (PerkinElmer). The counts were displayed in Excel (Microsoft), and the reconstituted profiles were compared with the analytical run for identification of the peaks. Fractions constituting one peak were pooled, and the solvent was evaporated by speed vacuum. The oligosaccharides were resuspended in 50 mM sodium acetate buffer, pH 5, and digested with 1 μl of α1,2-exomannosidase from T. reesei (provided by N. Callewaert, Ghent University, Ghent, Belgium; Maras et al., 2000) or mock at 37°C overnight. The products were reassessed by HPLC analysis (see previous section).Determination of Htm1p-TPH7 protein expressionYeast cells were grown at 30°C in appropriate medium to mid-log phase corresponding to an OD600nm of 0.8–1.2. 4 × 107 cells were harvested and lysed with glass beads in 1% (wt/vol) SDS, 50 mM Tris-HCl, pH 7.5, 2 mM PMSF, and 1× protease inhibitor complete cocktail (Roche) by vortexing at 4°C for 10 min. Samples were heated at 95°C for 5 min. Supernatants were cleared by centrifugation and subjected to endoglycosidase H treatment according to the recommendations of the manufacturer (BioConcept). Proteins were subjected to reducing 7% SDS-PAGE and electroblotted to nitrocellulose membranes. TPH7-tagged proteins were immunologically detected with peroxidase–antiperoxidase antibody at a dilution of 1:2,000 (Sigma-Aldrich). Visualization and reprobing of membranes with anti-HXK antibody (rabbit) were performed as stated in the Analysis of CPY* degradation section.Purification of Htm1p1,500 OD of yeast cells expressing C-terminally Myc-TAP–tagged Htm1p were washed with water + 1 mM PMSF and disrupted in 3 ml IP32 (50 mM Hepes-NaOH, pH 7.2, 50 mM NaCl, 125 mM KOAc, 2 mM MgCl2, 1 mM EDTA, and 3% glycerol) using glass beads. Upon addition of 45 ml IP32 + 1 mM PMSF, the lysate was centrifuged (1,000 g for 8 min at 4°C). The resulting supernatant was centrifuged (32,000 g for 1 h at 4°C), and the pellet was lysed in 50 ml IP32 + 0.25% NP-40. After clearance of the insoluble material by centrifugation (32,000 g for 30 min at 4°C), Htm1p was precipitated from the supernatant by the addition of 50 μl IgG beads (GE Healthcare) overnight. Beads were washed three times with IP32 + 0.25% NP-40. Bound proteins were liberated by the addition of tobacco etch virus protease (Invitrogen) in a volume of 100 μl IP32 + 0.25% NP-40 followed by incubation for 3 h at 16°C. Samples were lyophylized and analyzed by SDS-PAGE followed by Coomassie staining and mass spectrometry.Mass spectrometryProteins were separated on 10% polyacrylamide gels and stained with Coomassie brilliant blue R-250. Protein bands were excised and in-gel digested with sequencing-grade trypsin (Promega). The peptide mixture was separated on a reverse-phase column (75-μm PepMap C18; Dionex) connected to a capillary liquid chromatography system delivering a gradient of 5–50% acetonitrile. Eluting peptides were ionized by electrospray ionization on a mass spectrometer (Q-TOF1; Micromass). Mass spectrometry/mass spectrometry analyses were conducted by using collision energy profiles chosen on the basis of the mass to charge ratio value and the charge state of the parent ion. The generated mass data were processed into peak lists containing mass to charge value, charge state of the parent ion, fragment ion masses, and intensities using the MassLynx 4.1 software (Micromass) and were correlated with protein databases using the Mascot 2.2 software (Matrix Science; Perkins et al., 1999). Nonredundant protein databases National Center for Biotechnology Information and SwissProt (entry version 92) were searched without applying any constraints on Mr or species. Precursor ion accuracy was ±0.1 D, and fragment ion accuracy was ±0.2 D. The results were manually validated.CoimmunoprecipitationTo precipitate HA- and Myc-tagged proteins, logarithmically growing cells were harvested and washed with water + 1 mM PMSF. 50 OD cells were disrupted with glass beads in 400 μl IP32 (50 mM Hepes-NaOH, pH 7.2, 50 mM NaCl, 125 mM KOAc, 2 mM MgCl2, 1 mM EDTA, and 3% glycerol) + 1 mM PMSF. After lysis, 1 ml of IP32 buffer was added, and a low speed centrifugation (1,000 g for 5 min at 4°C) was performed. From the supernatant, a microsomal fraction was generated by high speed centrifugation (20,000 g for 20 min at 4°C). After solubilization in IP32 + 0.5% NP-40 and 1 mM PMSF, insoluble material was cleared from the lysate by centrifugation (20,000 g for 15 min at 4°C). Tagged proteins were precipitated from the supernatant by addition of 1 μl anti-HA (Sigma-Aldrich) or anti-Myc (Sigma-Aldrich) antibodies and 10 μl protein A–Sepharose beads (GE Healthcare) at 4°C overnight. Beads were washed three times with IP32 + 0.5% NP-40. Bound proteins were eluted by the addition of 100 μl of SDS sample buffer and analyzed by Western blotting using the indicated antibodies.Online supplemental materialTable S1 shows the peptide analysis of Htm1p-binding protein. Table S2 details the strains used in this study. Online supplemental material is available at http://www.jcb.org/cgi/content/full/jcb.200809198/DC1.

Materials and methods

Yeast strains and plasmidsYeast strains used in this study are listed in Table S2 (available at http://www.jcb.org/cgi/content/full/jcb.200809198/DC1; Vijayraghavan et al., 1989; Biederer et al., 1997; Jakob et al., 1998a, 2001). Standard yeast media and genetic techniques were used (Guthrie and Fink, 1991). YG695, YSC8, YSC17, YSC24, YG1330, and YDO1 are haploid descendants of the wild-type strains SS328/SS330. YG1330 and YDO1 were generated using the pYM C-terminal epitope-tagging method (Knop et al., 1999). In brief, the TPH7 tag together with the KanMX6 cassette was amplified by PCR. The PCR fragment was inserted into the genome of YG618 by homologous recombination at the 3′ end of the HTM1 locus or at the 3′ end of the mannosidase homology region between Asn517 and Asn518, respectively. YTX140, YCH030, YCH036, YCH046, YCH133, YCH135, YCH137, YCH175, YCH176, YCH177, and YCH178 are haploid descendants of DF5. PCR-based strategies were used to introduce the epitope tags to Htm1p and Pdi1p in these strains. All epitope-tagged proteins were expressed from their natural chromosomal locus under control of the endogenous promoter.The YEp352-HTM1 (pHTM1u-1) plasmid was constructed by transferring the HTM1 gene from pRS316-HTM1 (pHTM1-7) to the high copy number vector YEp352 plasmid using XhoI–SacI restriction endonucleases. YEp352-HTM1-TPH7 (pDOG3) and YEp352-htm1Δ518-TPH7 (pDOG4) were generated by PCR amplification of the modified HTM1 loci from YG1330 and YDO1 and subcloning into YEp352. Mutations in full-length YEp352-HTM1-TPH7 were introduced by the QuikChange method (Agilent Technologies), yielding YEp352-htm1-E222Q-TPH7 (pDOG5) and YEp352-htm1-D279N-TPH7 (pDOG6). Plasmids were confirmed by DNA sequence analysis.

Yeast strains and plasmids

Yeast strains used in this study are listed in Table S2 (available at http://www.jcb.org/cgi/content/full/jcb.200809198/DC1; Vijayraghavan et al., 1989; Biederer et al., 1997; Jakob et al., 1998a, 2001). Standard yeast media and genetic techniques were used (Guthrie and Fink, 1991). YG695, YSC8, YSC17, YSC24, YG1330, and YDO1 are haploid descendants of the wild-type strains SS328/SS330. YG1330 and YDO1 were generated using the pYM C-terminal epitope-tagging method (Knop et al., 1999). In brief, the TPH7 tag together with the KanMX6 cassette was amplified by PCR. The PCR fragment was inserted into the genome of YG618 by homologous recombination at the 3′ end of the HTM1 locus or at the 3′ end of the mannosidase homology region between Asn517 and Asn518, respectively. YTX140, YCH030, YCH036, YCH046, YCH133, YCH135, YCH137, YCH175, YCH176, YCH177, and YCH178 are haploid descendants of DF5. PCR-based strategies were used to introduce the epitope tags to Htm1p and Pdi1p in these strains. All epitope-tagged proteins were expressed from their natural chromosomal locus under control of the endogenous promoter.

The YEp352-HTM1 (pHTM1u-1) plasmid was constructed by transferring the HTM1 gene from pRS316-HTM1 (pHTM1-7) to the high copy number vector YEp352 plasmid using XhoI–SacI restriction endonucleases. YEp352-HTM1-TPH7 (pDOG3) and YEp352-htm1Δ518-TPH7 (pDOG4) were generated by PCR amplification of the modified HTM1 loci from YG1330 and YDO1 and subcloning into YEp352. Mutations in full-length YEp352-HTM1-TPH7 were introduced by the QuikChange method (Agilent Technologies), yielding YEp352-htm1-E222Q-TPH7 (pDOG5) and YEp352-htm1-D279N-TPH7 (pDOG6). Plasmids were confirmed by DNA sequence analysis.

Analysis of CPY* degradationThe cycloheximide chase was performed as described previously (Jakob et al., 1998a). In brief, yeast cells were grown at 30°C in appropriate medium to mid-log phase corresponding to an OD600nm of 0.8–1.2. 3 × 108 cells were harvested and resuspended in medium. The chase was initiated by addition of cycloheximide (final concentration of 100 μg/ml) and was performed at 30°C. 108 cells were removed at each time point and transferred into NaN3 (final concentration of 0.1% [wt/vol]) on ice, immediately pelleted, and flash frozen in liquid nitrogen. Whole cell protein extracts were prepared using glass beads, 1% (wt/vol) SDS, 50 mM Tris-HCl, pH 7.5, and 2 mM PMSF. Proteins were subjected to reducing 7% SDS-PAGE and electroblotted to nitrocellulose membranes. CPY* was immunologically detected with anti-CPY antibody (rabbit) at a dilution of 1:1,000 (Zufferey et al., 1995) and goat anti–rabbit IgG–horseradish peroxidase at 1:3,000 (Santa Cruz Biotechnology, Inc.). Visualization was performed with ECL detection (GE Healthcare). Reprobing of membranes with anti-hexokinase (HXK) antibody (rabbit) was performed accordingly. The x-ray films (super RX; Fujifilm) were scanned, and the CPY* protein amounts were determined densitometrically with a molecular imager (FX; Bio-Rad Laboratories) using the Quantity One program (Bio-Rad Laboratories).

Analysis of CPY* degradation

The cycloheximide chase was performed as described previously (Jakob et al., 1998a). In brief, yeast cells were grown at 30°C in appropriate medium to mid-log phase corresponding to an OD600nm of 0.8–1.2. 3 × 108 cells were harvested and resuspended in medium. The chase was initiated by addition of cycloheximide (final concentration of 100 μg/ml) and was performed at 30°C. 108 cells were removed at each time point and transferred into NaN3 (final concentration of 0.1% [wt/vol]) on ice, immediately pelleted, and flash frozen in liquid nitrogen. Whole cell protein extracts were prepared using glass beads, 1% (wt/vol) SDS, 50 mM Tris-HCl, pH 7.5, and 2 mM PMSF. Proteins were subjected to reducing 7% SDS-PAGE and electroblotted to nitrocellulose membranes. CPY* was immunologically detected with anti-CPY antibody (rabbit) at a dilution of 1:1,000 (Zufferey et al., 1995) and goat anti–rabbit IgG–horseradish peroxidase at 1:3,000 (Santa Cruz Biotechnology, Inc.). Visualization was performed with ECL detection (GE Healthcare). Reprobing of membranes with anti-hexokinase (HXK) antibody (rabbit) was performed accordingly. The x-ray films (super RX; Fujifilm) were scanned, and the CPY* protein amounts were determined densitometrically with a molecular imager (FX; Bio-Rad Laboratories) using the Quantity One program (Bio-Rad Laboratories).

[3H]mannose labeling and analysis of N-linked glycansCells were grown at 30°C in appropriate medium to mid-log phase corresponding to an OD600nm of 0.8–1.2. n × 5 × 108 cells (n = number of analyzed time points) were harvested and washed in YPD (1% yeast extract, 2% peptone, and 2% glucose) containing 0.1% glucose (YP0.1%D). Labeling was performed in n × 200 μl YP0.1%D containing n × 100 μCi d-2-[3H]mannose (500 GBq/mmol; Hartmann Analytic) and incubated for 10 (Figs. 2–5) or 20 min (Fig. 1) at 30°C. The radioactivity was chased by the addition of d-mannose to a 2% (weight per volume) final concentration at 30°C incubation. The chase was stopped by removing 5 × 108 cells into TCA on ice (final concentration of 10% [vol/vol]). TCA precipitates were washed twice with cold acetone and air dried for 20 min. The pellets were resuspended in buffer S (20 mM NaP, pH 7.5, 0.5% SDS, and 40 mM DTT) and vortexed with glass beads at 50°C for 1 h. Lysates were cleared by centrifugation, and iodoacetamide was added to a 50-mM final concentration. Samples were incubated for 30 min at 37°C. Oligosaccharides were cleaved from proteins by digestion with peptide N-glycosidase F according to the recommendations of the manufacturer (BioConcept). Cleanup of oligosaccharides was modified from Grubenmann et al. (2004). Prepacked C18 Sep Pak columns (Waters) were connected to columns (extended-volume empty reservoirs; Socochim SA) that were packed with Supelclean ENVI-Carb 120/400 (Sigma-Aldrich). The combined columns were equilibrated with methanol, acetonitrile, acetonitrile/H2O (25:75; vol/vol), and acetonitrile/H2O (2:98; vol/vol). Samples were loaded onto the columns after supplementation with acetonitrile/H2O (2:98; vol/vol). Columns were washed with acetonitrile/H2O (2:98; vol/vol), and glycans were eluted from the ENVI-Carb column with acetonitrile/H2O (25:75; vol/vol). The solvent was evaporated in a speed vacuum.For HPLC analysis of the oligosaccharides, a liquid chromatography (LC)-NH2 column (250 × 4.6 mm; Sigma-Aldrich; Cacan et al., 1993) including an LC-NH2 guard column was used. Oligosaccharide samples in acetonitrile/H2O (70:30; vol/vol) were filtered through a 0.45-μm filter (Millipore) and injected on the equilibrated (acetonitrile/H2O [70:30; vol/vol]; 30 min) system using an autosampling device (Merck/Hitachi AS-2000; Sigma-Aldrich). The gradient used was acetonitrile/H2O (70:30; vol/vol) to acetonitrile/H2O (45:50; vol/vol) for >90 min, 5 min at acetonitrile/H2O (45:50; vol/vol), returning to acetonitrile/H2O (70:30; vol/vol) for >5 min, and washing for 20 min at acetonitrile/H2O (70:30; vol/vol) before injection of the following sample using a pump (Merck/Hitachi L-2600A; Sigma-Aldrich). The eluate from the column was mixed continuously with scintillation fluid (FLO-Scint A; PerkinElmer) in a ratio of 1:1.5 (eluate/scintillation mix; vol/vol), and radioactivity was monitored with a flow monitor (FLO-ONE A-525; PerkinElmer) as described previously (Zufferey et al., 1995). Quantification of the peaks was performed using the PeakFit 4.06 program (SPSS) using the second derivative method.

[3H]mannose labeling and analysis of N-linked glycans

Cells were grown at 30°C in appropriate medium to mid-log phase corresponding to an OD600nm of 0.8–1.2. n × 5 × 108 cells (n = number of analyzed time points) were harvested and washed in YPD (1% yeast extract, 2% peptone, and 2% glucose) containing 0.1% glucose (YP0.1%D). Labeling was performed in n × 200 μl YP0.1%D containing n × 100 μCi d-2-[3H]mannose (500 GBq/mmol; Hartmann Analytic) and incubated for 10 (Figs. 2–5) or 20 min (Fig. 1) at 30°C. The radioactivity was chased by the addition of d-mannose to a 2% (weight per volume) final concentration at 30°C incubation. The chase was stopped by removing 5 × 108 cells into TCA on ice (final concentration of 10% [vol/vol]). TCA precipitates were washed twice with cold acetone and air dried for 20 min. The pellets were resuspended in buffer S (20 mM NaP, pH 7.5, 0.5% SDS, and 40 mM DTT) and vortexed with glass beads at 50°C for 1 h. Lysates were cleared by centrifugation, and iodoacetamide was added to a 50-mM final concentration. Samples were incubated for 30 min at 37°C. Oligosaccharides were cleaved from proteins by digestion with peptide N-glycosidase F according to the recommendations of the manufacturer (BioConcept). Cleanup of oligosaccharides was modified from Grubenmann et al. (2004). Prepacked C18 Sep Pak columns (Waters) were connected to columns (extended-volume empty reservoirs; Socochim SA) that were packed with Supelclean ENVI-Carb 120/400 (Sigma-Aldrich). The combined columns were equilibrated with methanol, acetonitrile, acetonitrile/H2O (25:75; vol/vol), and acetonitrile/H2O (2:98; vol/vol). Samples were loaded onto the columns after supplementation with acetonitrile/H2O (2:98; vol/vol). Columns were washed with acetonitrile/H2O (2:98; vol/vol), and glycans were eluted from the ENVI-Carb column with acetonitrile/H2O (25:75; vol/vol). The solvent was evaporated in a speed vacuum.

For HPLC analysis of the oligosaccharides, a liquid chromatography (LC)-NH2 column (250 × 4.6 mm; Sigma-Aldrich; Cacan et al., 1993) including an LC-NH2 guard column was used. Oligosaccharide samples in acetonitrile/H2O (70:30; vol/vol) were filtered through a 0.45-μm filter (Millipore) and injected on the equilibrated (acetonitrile/H2O [70:30; vol/vol]; 30 min) system using an autosampling device (Merck/Hitachi AS-2000; Sigma-Aldrich). The gradient used was acetonitrile/H2O (70:30; vol/vol) to acetonitrile/H2O (45:50; vol/vol) for >90 min, 5 min at acetonitrile/H2O (45:50; vol/vol), returning to acetonitrile/H2O (70:30; vol/vol) for >5 min, and washing for 20 min at acetonitrile/H2O (70:30; vol/vol) before injection of the following sample using a pump (Merck/Hitachi L-2600A; Sigma-Aldrich). The eluate from the column was mixed continuously with scintillation fluid (FLO-Scint A; PerkinElmer) in a ratio of 1:1.5 (eluate/scintillation mix; vol/vol), and radioactivity was monitored with a flow monitor (FLO-ONE A-525; PerkinElmer) as described previously (Zufferey et al., 1995). Quantification of the peaks was performed using the PeakFit 4.06 program (SPSS) using the second derivative method.

Preparative purification of [3H]mannose-labeled N-linked glycans[3H]mannose labeling was performed as described in the previous section with the following modifications: 4 × 109 cells were labeled with 800 μCi d-2-[3H]mannose in 2 ml YP0.1%D. The labeling was stopped by addition of TCA to the final concentration of 10% (vol/vol). The samples were worked up each in two batches and after the elution from the ENVI-Carb column were pooled again. An analytical HPLC run was performed with the N-glycans from 5 × 108 cells. For isolation of individual N-glycans, the LC-NH2 column was disconnected from the flow monitor, and elution fractions of 500 μl were manually collected. The radioactivity in the fractions was determined using a β-counter (Tri-Carb 2800TR; PerkinElmer) with the QuantaSmart program (PerkinElmer). The counts were displayed in Excel (Microsoft), and the reconstituted profiles were compared with the analytical run for identification of the peaks. Fractions constituting one peak were pooled, and the solvent was evaporated by speed vacuum. The oligosaccharides were resuspended in 50 mM sodium acetate buffer, pH 5, and digested with 1 μl of α1,2-exomannosidase from T. reesei (provided by N. Callewaert, Ghent University, Ghent, Belgium; Maras et al., 2000) or mock at 37°C overnight. The products were reassessed by HPLC analysis (see previous section).

Preparative purification of [3H]mannose-labeled N-linked glycans

[3H]mannose labeling was performed as described in the previous section with the following modifications: 4 × 109 cells were labeled with 800 μCi d-2-[3H]mannose in 2 ml YP0.1%D. The labeling was stopped by addition of TCA to the final concentration of 10% (vol/vol). The samples were worked up each in two batches and after the elution from the ENVI-Carb column were pooled again. An analytical HPLC run was performed with the N-glycans from 5 × 108 cells. For isolation of individual N-glycans, the LC-NH2 column was disconnected from the flow monitor, and elution fractions of 500 μl were manually collected. The radioactivity in the fractions was determined using a β-counter (Tri-Carb 2800TR; PerkinElmer) with the QuantaSmart program (PerkinElmer). The counts were displayed in Excel (Microsoft), and the reconstituted profiles were compared with the analytical run for identification of the peaks. Fractions constituting one peak were pooled, and the solvent was evaporated by speed vacuum. The oligosaccharides were resuspended in 50 mM sodium acetate buffer, pH 5, and digested with 1 μl of α1,2-exomannosidase from T. reesei (provided by N. Callewaert, Ghent University, Ghent, Belgium; Maras et al., 2000) or mock at 37°C overnight. The products were reassessed by HPLC analysis (see previous section).

Determination of Htm1p-TPH7 protein expressionYeast cells were grown at 30°C in appropriate medium to mid-log phase corresponding to an OD600nm of 0.8–1.2. 4 × 107 cells were harvested and lysed with glass beads in 1% (wt/vol) SDS, 50 mM Tris-HCl, pH 7.5, 2 mM PMSF, and 1× protease inhibitor complete cocktail (Roche) by vortexing at 4°C for 10 min. Samples were heated at 95°C for 5 min. Supernatants were cleared by centrifugation and subjected to endoglycosidase H treatment according to the recommendations of the manufacturer (BioConcept). Proteins were subjected to reducing 7% SDS-PAGE and electroblotted to nitrocellulose membranes. TPH7-tagged proteins were immunologically detected with peroxidase–antiperoxidase antibody at a dilution of 1:2,000 (Sigma-Aldrich). Visualization and reprobing of membranes with anti-HXK antibody (rabbit) were performed as stated in the Analysis of CPY* degradation section.

Determination of Htm1p-TPH7 protein expression

Yeast cells were grown at 30°C in appropriate medium to mid-log phase corresponding to an OD600nm of 0.8–1.2. 4 × 107 cells were harvested and lysed with glass beads in 1% (wt/vol) SDS, 50 mM Tris-HCl, pH 7.5, 2 mM PMSF, and 1× protease inhibitor complete cocktail (Roche) by vortexing at 4°C for 10 min. Samples were heated at 95°C for 5 min. Supernatants were cleared by centrifugation and subjected to endoglycosidase H treatment according to the recommendations of the manufacturer (BioConcept). Proteins were subjected to reducing 7% SDS-PAGE and electroblotted to nitrocellulose membranes. TPH7-tagged proteins were immunologically detected with peroxidase–antiperoxidase antibody at a dilution of 1:2,000 (Sigma-Aldrich). Visualization and reprobing of membranes with anti-HXK antibody (rabbit) were performed as stated in the Analysis of CPY* degradation section.

Purification of Htm1p1,500 OD of yeast cells expressing C-terminally Myc-TAP–tagged Htm1p were washed with water + 1 mM PMSF and disrupted in 3 ml IP32 (50 mM Hepes-NaOH, pH 7.2, 50 mM NaCl, 125 mM KOAc, 2 mM MgCl2, 1 mM EDTA, and 3% glycerol) using glass beads. Upon addition of 45 ml IP32 + 1 mM PMSF, the lysate was centrifuged (1,000 g for 8 min at 4°C). The resulting supernatant was centrifuged (32,000 g for 1 h at 4°C), and the pellet was lysed in 50 ml IP32 + 0.25% NP-40. After clearance of the insoluble material by centrifugation (32,000 g for 30 min at 4°C), Htm1p was precipitated from the supernatant by the addition of 50 μl IgG beads (GE Healthcare) overnight. Beads were washed three times with IP32 + 0.25% NP-40. Bound proteins were liberated by the addition of tobacco etch virus protease (Invitrogen) in a volume of 100 μl IP32 + 0.25% NP-40 followed by incubation for 3 h at 16°C. Samples were lyophylized and analyzed by SDS-PAGE followed by Coomassie staining and mass spectrometry.

Purification of Htm1p

1,500 OD of yeast cells expressing C-terminally Myc-TAP–tagged Htm1p were washed with water + 1 mM PMSF and disrupted in 3 ml IP32 (50 mM Hepes-NaOH, pH 7.2, 50 mM NaCl, 125 mM KOAc, 2 mM MgCl2, 1 mM EDTA, and 3% glycerol) using glass beads. Upon addition of 45 ml IP32 + 1 mM PMSF, the lysate was centrifuged (1,000 g for 8 min at 4°C). The resulting supernatant was centrifuged (32,000 g for 1 h at 4°C), and the pellet was lysed in 50 ml IP32 + 0.25% NP-40. After clearance of the insoluble material by centrifugation (32,000 g for 30 min at 4°C), Htm1p was precipitated from the supernatant by the addition of 50 μl IgG beads (GE Healthcare) overnight. Beads were washed three times with IP32 + 0.25% NP-40. Bound proteins were liberated by the addition of tobacco etch virus protease (Invitrogen) in a volume of 100 μl IP32 + 0.25% NP-40 followed by incubation for 3 h at 16°C. Samples were lyophylized and analyzed by SDS-PAGE followed by Coomassie staining and mass spectrometry.

Mass spectrometryProteins were separated on 10% polyacrylamide gels and stained with Coomassie brilliant blue R-250. Protein bands were excised and in-gel digested with sequencing-grade trypsin (Promega). The peptide mixture was separated on a reverse-phase column (75-μm PepMap C18; Dionex) connected to a capillary liquid chromatography system delivering a gradient of 5–50% acetonitrile. Eluting peptides were ionized by electrospray ionization on a mass spectrometer (Q-TOF1; Micromass). Mass spectrometry/mass spectrometry analyses were conducted by using collision energy profiles chosen on the basis of the mass to charge ratio value and the charge state of the parent ion. The generated mass data were processed into peak lists containing mass to charge value, charge state of the parent ion, fragment ion masses, and intensities using the MassLynx 4.1 software (Micromass) and were correlated with protein databases using the Mascot 2.2 software (Matrix Science; Perkins et al., 1999). Nonredundant protein databases National Center for Biotechnology Information and SwissProt (entry version 92) were searched without applying any constraints on Mr or species. Precursor ion accuracy was ±0.1 D, and fragment ion accuracy was ±0.2 D. The results were manually validated.

Mass spectrometry

Proteins were separated on 10% polyacrylamide gels and stained with Coomassie brilliant blue R-250. Protein bands were excised and in-gel digested with sequencing-grade trypsin (Promega). The peptide mixture was separated on a reverse-phase column (75-μm PepMap C18; Dionex) connected to a capillary liquid chromatography system delivering a gradient of 5–50% acetonitrile. Eluting peptides were ionized by electrospray ionization on a mass spectrometer (Q-TOF1; Micromass). Mass spectrometry/mass spectrometry analyses were conducted by using collision energy profiles chosen on the basis of the mass to charge ratio value and the charge state of the parent ion. The generated mass data were processed into peak lists containing mass to charge value, charge state of the parent ion, fragment ion masses, and intensities using the MassLynx 4.1 software (Micromass) and were correlated with protein databases using the Mascot 2.2 software (Matrix Science; Perkins et al., 1999). Nonredundant protein databases National Center for Biotechnology Information and SwissProt (entry version 92) were searched without applying any constraints on Mr or species. Precursor ion accuracy was ±0.1 D, and fragment ion accuracy was ±0.2 D. The results were manually validated.

CoimmunoprecipitationTo precipitate HA- and Myc-tagged proteins, logarithmically growing cells were harvested and washed with water + 1 mM PMSF. 50 OD cells were disrupted with glass beads in 400 μl IP32 (50 mM Hepes-NaOH, pH 7.2, 50 mM NaCl, 125 mM KOAc, 2 mM MgCl2, 1 mM EDTA, and 3% glycerol) + 1 mM PMSF. After lysis, 1 ml of IP32 buffer was added, and a low speed centrifugation (1,000 g for 5 min at 4°C) was performed. From the supernatant, a microsomal fraction was generated by high speed centrifugation (20,000 g for 20 min at 4°C). After solubilization in IP32 + 0.5% NP-40 and 1 mM PMSF, insoluble material was cleared from the lysate by centrifugation (20,000 g for 15 min at 4°C). Tagged proteins were precipitated from the supernatant by addition of 1 μl anti-HA (Sigma-Aldrich) or anti-Myc (Sigma-Aldrich) antibodies and 10 μl protein A–Sepharose beads (GE Healthcare) at 4°C overnight. Beads were washed three times with IP32 + 0.5% NP-40. Bound proteins were eluted by the addition of 100 μl of SDS sample buffer and analyzed by Western blotting using the indicated antibodies.

Coimmunoprecipitation

To precipitate HA- and Myc-tagged proteins, logarithmically growing cells were harvested and washed with water + 1 mM PMSF. 50 OD cells were disrupted with glass beads in 400 μl IP32 (50 mM Hepes-NaOH, pH 7.2, 50 mM NaCl, 125 mM KOAc, 2 mM MgCl2, 1 mM EDTA, and 3% glycerol) + 1 mM PMSF. After lysis, 1 ml of IP32 buffer was added, and a low speed centrifugation (1,000 g for 5 min at 4°C) was performed. From the supernatant, a microsomal fraction was generated by high speed centrifugation (20,000 g for 20 min at 4°C). After solubilization in IP32 + 0.5% NP-40 and 1 mM PMSF, insoluble material was cleared from the lysate by centrifugation (20,000 g for 15 min at 4°C). Tagged proteins were precipitated from the supernatant by addition of 1 μl anti-HA (Sigma-Aldrich) or anti-Myc (Sigma-Aldrich) antibodies and 10 μl protein A–Sepharose beads (GE Healthcare) at 4°C overnight. Beads were washed three times with IP32 + 0.5% NP-40. Bound proteins were eluted by the addition of 100 μl of SDS sample buffer and analyzed by Western blotting using the indicated antibodies.

Online supplemental materialTable S1 shows the peptide analysis of Htm1p-binding protein. Table S2 details the strains used in this study. Online supplemental material is available at http://www.jcb.org/cgi/content/full/jcb.200809198/DC1.

Online supplemental material

Table S1 shows the peptide analysis of Htm1p-binding protein. Table S2 details the strains used in this study. Online supplemental material is available at http://www.jcb.org/cgi/content/full/jcb.200809198/DC1.

Supplementary Material[Supplemental Material Index]

Supplementary Material

[Supplemental Material Index]
