---
pmid: '31167142'
title: Yeast Sirtuin Family Members Maintain Transcription Homeostasis to Ensure Genome
  Stability.
authors:
- Feldman JL
- Peterson CL
journal: Cell Rep
year: '2019'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC6640630
doi: 10.1016/j.celrep.2019.05.009
---

# Yeast Sirtuin Family Members Maintain Transcription Homeostasis to Ensure Genome Stability.
**Authors:** Feldman JL, Peterson CL
**Journal:** Cell Rep (2019)
**DOI:** [10.1016/j.celrep.2019.05.009](https://doi.org/10.1016/j.celrep.2019.05.009)
**PMC:** [PMC6640630](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6640630/)

## Abstract

1. Cell Rep. 2019 Jun 4;27(10):2978-2989.e5. doi: 10.1016/j.celrep.2019.05.009.

Yeast Sirtuin Family Members Maintain Transcription Homeostasis to Ensure Genome 
Stability.

Feldman JL(1), Peterson CL(2).

Author information:
(1)Program in Molecular Medicine, University of Massachusetts Medical School, 
Worcester, MA 01605, USA.
(2)Program in Molecular Medicine, University of Massachusetts Medical School, 
Worcester, MA 01605, USA. Electronic address: craig.peterson@umassmed.edu.

The mammalian sirtuin, SIRT6, is a key tumor suppressor that maintains genome 
stability and regulates transcription, though how SIRT6 family members control 
genome stability is unclear. Here, we use multiple genome-wide approaches to 
demonstrate that the yeast SIRT6 homologs, Hst3 and Hst4, prevent genome 
instability by tuning levels of both coding and noncoding transcription. While 
nascent RNAs are elevated in the absence of Hst3 and Hst4, a global impact on 
steady-state mRNAs is masked by the nuclear exosome, indicating that sirtuins 
and the exosome provide two levels of regulation to maintain transcription 
homeostasis. We find that, in the absence of Hst3 and Hst4, increased 
transcription is associated with excessive DNA-RNA hybrids (R-loops) that appear 
to lead to new DNA double-strand breaks. Importantly, dissolution of R-loops 
suppresses the genome instability phenotypes of hst3 hst4 mutants, suggesting 
that the sirtuins maintain genome stability by acting as a rheostat to prevent 
promiscuous transcription.

Copyright © 2019 The Author(s). Published by Elsevier Inc. All rights reserved.

DOI: 10.1016/j.celrep.2019.05.009
PMCID: PMC6640630
PMID: 31167142 [Indexed for MEDLINE]

Conflict of interest statement: DECLARATION OF INTERESTS The authors declare no 
competing interests.

## Full Text

INTRODUCTIONThe epigenetic control of gene expression and genome stability plays a central role in ensuring normal cellular function. Dysregulation has been associated with numerous human malignancies, and chromatin factors have emerged as some of the most frequently affected proteins in cancer (Morgan and Shilatifard, 2015; Shah et al., 2014). SIRT6 is a mammalian member of the Sirtuin family of nicotinamide adenine dinucleotide (NAD+) dependent lysine deacetylases that are conserved across all species (Frye, 2000). SIRT6 functions primarily as a lysine 56 (H3-K56Ac) (Michishita et al., 2009; Yang et al., 2009) and lysine 9 (H3-K9Ac) (Michishita et al., 2008) histone H3 deacetylase at promoters to regulate the expression of genes involved in various pathways, including metabolism, pluripotency, inflammation, and ribosome biogenesis (Etchegaray et al., 2015; Kugel and Mostoslavsky, 2014; Kugel et al., 2016). Global changes in transcription in the absence of SIRT6 have not been reported. Deletion of Sirt6 causes major genomic and metabolic instability (Mostoslavsky et al., 2006), and loss of SIRT6 is sufficient to drive tumorigenesis in mice independent of oncogene activation (Sebastián et al., 2012). Mutations of Sirt6 that affect activity have been identified in human cancers (Kugel et al., 2015), and, strikingly, Sirt6 is deleted in ~60% and ~30% of pancreatic and colorectal cancer cell lines, respectively (Sebastián et al., 2012). Together, the results point to an important role for SIRT6 as a tumor suppressor that regulates transcription and maintains genome stability.Hst3 and Hst4 are the two SIRT6 homologs in yeast that regulate H3-K56Ac levels (Celic et al., 2006; Maas et al., 2006), with the highest deacetylation activity observed during the S/G2 phase transition (Celic et al., 2006; Maas et al., 2006). Similar to Sirt6, deletion of HST3 and HST4 induces a host of genome instability phenotypes, including spontaneous DNA double-strand breaks, replication fork collapse, increased chromosomal loss, impairment of break-induced replication, and heightened susceptibility to genotoxic agents (Brachmann et al., 1995; Celic et al., 2006; Che et al., 2015). Notably, these phenotypes are alleviated by inactivation of the Asf1 subunit of the Rtt109 histone acetyltransferase (HAT) complex or by a non-acetylatable H3-K56R mutant, suggesting that persistent H3-K56 hyperacetylation promotes genomic instability (Celic et al., 2006, 2008; Maas et al., 2006). A prevailing model proposes that DNA damage is caused by the presence of hyperacetylated nucleosomes due to the lack of Hst3 and Hst4 that either impede replication fork progression or destabilize stalled forks (Celic et al., 2006).Here, we provide evidence of a functional link between increased transcription driven by loss of the sirtuins and the genomic instability phenotype observed in a hst3 hst4 mutant. Using a combination of native elongating transcript sequencing (NET-seq) (Churchman and Weissman, 2011) and RNA sequencing (RNA-seq) analyses, we show that Hst3 and Hst4 are required to repress transcription of coding and non-coding RNAs. Nascent RNAs are increased throughout coding regions in the absence of Hst3 and Hst4, and we also observe a shift in RNA polymerase II (Pol II) occupancy toward transcription start sites (TSSs). In addition, divergent antisense transcription is increased around the −1 nucleosome, similar to what was observed previously at several promoters (Marquardt et al., 2014). Together, the results are consistent with increased transcription initiation at divergent promoters in a hst3 hst4 mutant, providing an additional mechanism utilized by cells to limit divergent ncRNA abundance. Interestingly, we find that increased nascent RNA is not reflected in the steady-state mRNA pool due to activity of the nuclear exosome. This impact of the exosome was also seen previously in a rtt109Δ mutant that lacked H3-K56Ac. We further use DNA-RNA immunoprecipitation with deep sequencing (DRIP-seq) (Ginno et al., 2012) analyses to identify loci with increased R-loop levels in the absence of Hst3 and Hst4. We show that a subset of regions with increased R-loops are also prone to the formation of DNA double-strand breaks, and we find that overexpression of human RNase-H1 suppresses the sensitivity of a hst3 hst4 mutant to genotoxic stress. Together, the results indicate that the sirtuins function to regulate transcription in order to prevent pervasive R-loop formation and subsequent genomic instability.

INTRODUCTION

The epigenetic control of gene expression and genome stability plays a central role in ensuring normal cellular function. Dysregulation has been associated with numerous human malignancies, and chromatin factors have emerged as some of the most frequently affected proteins in cancer (Morgan and Shilatifard, 2015; Shah et al., 2014). SIRT6 is a mammalian member of the Sirtuin family of nicotinamide adenine dinucleotide (NAD+) dependent lysine deacetylases that are conserved across all species (Frye, 2000). SIRT6 functions primarily as a lysine 56 (H3-K56Ac) (Michishita et al., 2009; Yang et al., 2009) and lysine 9 (H3-K9Ac) (Michishita et al., 2008) histone H3 deacetylase at promoters to regulate the expression of genes involved in various pathways, including metabolism, pluripotency, inflammation, and ribosome biogenesis (Etchegaray et al., 2015; Kugel and Mostoslavsky, 2014; Kugel et al., 2016). Global changes in transcription in the absence of SIRT6 have not been reported. Deletion of Sirt6 causes major genomic and metabolic instability (Mostoslavsky et al., 2006), and loss of SIRT6 is sufficient to drive tumorigenesis in mice independent of oncogene activation (Sebastián et al., 2012). Mutations of Sirt6 that affect activity have been identified in human cancers (Kugel et al., 2015), and, strikingly, Sirt6 is deleted in ~60% and ~30% of pancreatic and colorectal cancer cell lines, respectively (Sebastián et al., 2012). Together, the results point to an important role for SIRT6 as a tumor suppressor that regulates transcription and maintains genome stability.

Hst3 and Hst4 are the two SIRT6 homologs in yeast that regulate H3-K56Ac levels (Celic et al., 2006; Maas et al., 2006), with the highest deacetylation activity observed during the S/G2 phase transition (Celic et al., 2006; Maas et al., 2006). Similar to Sirt6, deletion of HST3 and HST4 induces a host of genome instability phenotypes, including spontaneous DNA double-strand breaks, replication fork collapse, increased chromosomal loss, impairment of break-induced replication, and heightened susceptibility to genotoxic agents (Brachmann et al., 1995; Celic et al., 2006; Che et al., 2015). Notably, these phenotypes are alleviated by inactivation of the Asf1 subunit of the Rtt109 histone acetyltransferase (HAT) complex or by a non-acetylatable H3-K56R mutant, suggesting that persistent H3-K56 hyperacetylation promotes genomic instability (Celic et al., 2006, 2008; Maas et al., 2006). A prevailing model proposes that DNA damage is caused by the presence of hyperacetylated nucleosomes due to the lack of Hst3 and Hst4 that either impede replication fork progression or destabilize stalled forks (Celic et al., 2006).

Here, we provide evidence of a functional link between increased transcription driven by loss of the sirtuins and the genomic instability phenotype observed in a hst3 hst4 mutant. Using a combination of native elongating transcript sequencing (NET-seq) (Churchman and Weissman, 2011) and RNA sequencing (RNA-seq) analyses, we show that Hst3 and Hst4 are required to repress transcription of coding and non-coding RNAs. Nascent RNAs are increased throughout coding regions in the absence of Hst3 and Hst4, and we also observe a shift in RNA polymerase II (Pol II) occupancy toward transcription start sites (TSSs). In addition, divergent antisense transcription is increased around the −1 nucleosome, similar to what was observed previously at several promoters (Marquardt et al., 2014). Together, the results are consistent with increased transcription initiation at divergent promoters in a hst3 hst4 mutant, providing an additional mechanism utilized by cells to limit divergent ncRNA abundance. Interestingly, we find that increased nascent RNA is not reflected in the steady-state mRNA pool due to activity of the nuclear exosome. This impact of the exosome was also seen previously in a rtt109Δ mutant that lacked H3-K56Ac. We further use DNA-RNA immunoprecipitation with deep sequencing (DRIP-seq) (Ginno et al., 2012) analyses to identify loci with increased R-loop levels in the absence of Hst3 and Hst4. We show that a subset of regions with increased R-loops are also prone to the formation of DNA double-strand breaks, and we find that overexpression of human RNase-H1 suppresses the sensitivity of a hst3 hst4 mutant to genotoxic stress. Together, the results indicate that the sirtuins function to regulate transcription in order to prevent pervasive R-loop formation and subsequent genomic instability.

RESULTSHst3 and Hst4 Repress Nascent RNA TranscriptionDeletion of HST3 and HST4 leads to genomic instability (Brachmann et al., 1995; Celic et al., 2006), making such strains susceptible to second-site suppressor mutations. Notably, previous studies showed that the loss of both Hst3 and Hst4 is needed in order to observe measurable phenotypes (Celic et al., 2006), indicating that they perform redundant roles. Therefore, we wanted to establish an alternative approach to characterize the impact of Hst3 and Hst4 on transcription. To this end, the anchor away system (Haruki et al., 2008) was used to conditionally deplete Hst3 from the nucleus in a hst4Δ strain by tagging the C terminus of the HST3 locus with the FKBP12-rapamycin-binding (FRB) domain (hst4Δ HST3-FRB). The parent strain harbors a FK506 binding protein (FKBP12) fused to the C terminus of RPL13A, which is a highly abundant ribosomal protein that shuttles from the nucleus to the cytoplasm during ribosome assembly. A ternary complex between the FRB and FKBP12 domains is formed in the presence of rapamycin and, thus, rapidly depletes Hst3 from the nucleus. In addition, anchor away strains contain a rapamycin resistant tor1-1 allele to ensure rapamycin is not toxic to the wild-type (WT) strain (Haruki et al., 2008).We first confirmed that depletion of Hst3 in a hst4Δ displays a similar phenotype to a hst3Δ hst4Δ strain by spot dilution assay. The hst4Δ HST3-FRB strain is sensitive to 0.01% methyl metha-nesulfonate (MMS) and 0.1 M hydroxyurea (HU) only in the presence of rapamycin, similar to the hst3Δ hst4Δ strain in the presence of MMS and HU on DMSO (Figure S1A). Consistent with redundant roles for Hst3 and Hst4, the individual HST3-FRB mutant is not sensitive to genotoxic agents in the presence of rapamycin (Figure S1A).Given that mammalian Sirt6 plays key roles in transcription, we sought to determine the impact of yeast Hst3 and Hst4 on nascent RNA production. We performed NET-seq (Churchman and Weissman, 2011) in WT and the hst4Δ HST3-FRB mutant, using asynchronous cells treated with rapamycin for 3 h. Since we anticipated a potential for global changes in transcription, S. pombe cells were used as a spike-in control for library normalization. Loss of Hst3 and Hst4 led to a global shift in the nascent RNA transcriptome, with an average fold increase of ~1.4 (p < 2 × 10−16, Mann-Whitney U test) (Figure 1A), and approximately a quarter of the genome (1,092 genes) increased by 1.5-fold or greater (false discovery rate [FDR] ≤ 0.1) in the hst4Δ HST3-FRB mutant. Loss of Hst3 and Hst4 has a somewhat larger impact on poorly expressed genes, as the log2 fold change (LFC) between mutant and WT cells is greater for the bottom 25% and 50% of genes transcribed in WT, compared to the top quatriles (p < 2.2 × 10−16, Mann-Whitney U test) (Figure 1B).Metagene plots of mean nascent transcript levels, representative genome browser views of NET-seq data, and a heatmap of the log2-fold change between the hst4Δ HST3-FRB mutant and WT confirmed higher levels of transcription throughout genic regions (Figures 1C, S1B, and S1C) in the mutant compared to the WT, especially for genes within the lowest quartile of expression levels (Figure 1C). Genes within the top 25% of the WT expression level also showed increases in nascent RNA, though these increases were greater near the TSS compared to the gene body and transcription termination site (TTS) (Figure 1C, bottom; Figure S1C). In addition, there was an increase in the 5’ to 3’ ratio of RNA transcripts genome wide (p < 2.2 × 10−16, Mann-Whitney U test) (Figure S1D). As further evidence that loss of Hst3 and Hst4 causes a shift in the Pol II distribution toward the TSS, we analyzed the distribution of Pol II after normalizing for differences in overall transcription. In agreement with the 5’ to 3’ ratios (Figure S1D), we observed a shift in Pol II distribution toward the TSS and a corresponding decrease near the TTS (Figure 1D). Taken together, our analyses indicate that Hst3 and Hst4 repress transcription initiation and, furthermore, that the absence of these sirtuins leads to the accumulation of Pol II near the TSS, which may be indicative of increased Pol II pausing.Hst3 and Hst4 Repress Transcription of Many Non-coding RNAsStudies in recent years have demonstrated that many eukaryotic promoters are inherently bidirectional (Scruggs et al., 2015; Wei et al., 2011), and transcription termination sequences, RNA degradation complexes, and chromatin modifying factors function to limit the abundance of divergent non-coding RNAs (ncRNAs) relative to mRNAs (Hainer et al., 2015; Huang and Workman, 2013; Marquardt et al., 2014; Wei et al., 2011; Whitehouse et al., 2007). We compared the abundance of divergent antisense nascent transcripts by NET-seq in WT and hst4Δ HST3-FRB mutant cells at tandem genes (2,716) by analyzing antisense reads in the region from −600 to −100 bp from the TSS. We observed a global increase in nascent transcripts up-stream of genes in the absence of Hst3 and Hst4, with an average fold increase of ~1.6 (p < 2 × 10−16, Mann-Whitney U test) (1,051 LFC ≥ 0.59, FDR ≤ 0.1) (Figure 2A). The increase in transcription maps around the −1 and −2 nucleosomes, with little change in the nucleosome depleted region (NDR; Figure 2B). These data are in agreement with previous results that showed increased divergent transcription by northern blot in the absence of Hst3 and Hst4 at several promoters (Marquardt et al., 2014).In addition to divergent antisense transcripts, we investigated the role of Hst3 and Hst4 on cryptic unstable transcript (CUT) levels. CUTs are 5’ capped and polyadenylated ~400 bp transcripts that are rapidly degraded due a high abundance of binding motifs for the Nrd1-Nab1-Sen1 (NNS) termination machinery and subsequent targeting by the nuclear exosome (Arigo et al., 2006; Schulz et al., 2013; Thiebaut et al., 2006). Nascent CUT RNAs are also increased in the hst4 HST3-FRB mutant compared to WT (1.6-fold, p = 6 × 10–13, Mann-Whitney U test) (245 LFC ≥ 0.59, FDR ≤ 0.1) (Figures 2C and 2D). Taken together, our analyses point to an important role for Hst3 and Hst4 in limiting non-coding RNA production.Steady-State RNA Pool Minimally Affected by Loss of Hst3 and Hst4Our NET-seq analyses indicated that Hst3 and Hst4 repress the transcription initiation of genes. Therefore, we investigated whether the increase in nascent transcription translated to increased steady-state mRNA levels by analyzing RNA profiles by stranded RNA-seq. Similar to NET-seq analyses, S. pombe cells were used as a spike-in control for library normalization. Unexpectedly, and in contrast to the global increase in nascent transcription, steady-state mRNA levels remained relatively unchanged in the absence of Hst3 and Hst4 (0.981-fold, p = 0.02, Mann-Whitney U test) (Figure 3A). Hst3 and Hst4 negatively regulate the steady-state RNA level of 225 genes (FDR ≤ 0.1, LFC ≥ 0.59, edgeR) and positively regulate 85 genes (FDR ≤ 0.1, LFC ≤ −0.59, edgeR) (Figure S2A). Consistent with what was observed at the nascent RNA level, only very poorly transcribed genes in WT were increased to a greater extent in the mutant compared to the highly transcribed genes (p < 2.2 × 10−16, Mann-Whitney U test) (Figure 3B). Taken together, these results are similar to our previous analyses of the H3-K56 acetyltransferase Rtt109, in which we observed little change in the steady-state mRNA pool despite a global decrease in Pol II occupancy in a rtt109Δ strain (Rege et al., 2015).The observation that the steady-state mRNA pool remains relatively unchanged even though there is a global increase in nascent RNA production led us to investigate the similarities and differences between the NET-seq and RNA-seq datasets. A k-means clustering approach was used to identify subsets of genes that are differentially regulated at the nascent and steady-state levels (Figure 3C; Table S1; see also Figures S2B and S2C for genome browser views). Group A genes, which are highly transcribed in WT cells (Figure 3D), show small increases at the nascent RNA level, whereas many show an opposite, decreased level in the steady-state RNA pool (Figure 3C). Nascent RNA transcription of the other three groups of genes (groups B–D) was increased in the absence of Hst3 and Hst4, but variable effects are observed at the steady-state level (Figure 3C). With the exception of group B genes, which are the most poorly transcribed genes in WT cells (Figure 3D) and are upregulated in the hst4Δ HST3-FRB mutant by RNA-seq analyses, steady-state mRNA levels are minimally affected (group D) or are decreased (group C) in the hst4Δ HST3-FRB mutant (Figure 3C). The results reveal that many of the increased transcripts observed at the nascent RNA level in the absence of Hst3 and Hst4 are post-transcriptionally regulated and, thus, are not observed in the steady-state RNA pool.Increased Transcription in the Absence of Hst3 and Hst4 Is Masked by the Nuclear ExosomeIn addition to a role in regulating ncRNA transcription (Schneider et al., 2012) and processing small nuclear and nucleolar RNAs (snRNAs and snoRNAs) (Gudipati et al., 2012), the nuclear exosome plays a more general role in the surveillance of nuclear mRNAs (Rege et al., 2015; Schmid et al., 2012). To investigate whether the nuclear exosome might be responsible for masking the impact of Hst3 and Hst4 loss on the steady-state RNA pool, the anchor away system was used to deplete the 3’ to 5’ exonuclease subunit, Rrp6, from the nucleus for 3 h, alone or in combination with the hst4Δ HST3-FRB mutant. Interestingly, growth assays revealed an additive effect of depleting Rrp6 in the absence of both Hst3 and Hst4, as the cells become more sensitive to HU compared to either the RRP6-FRB single mutant or the hst4Δ HST3-FRB double mutant (Figure S3A).RNA-seq was performed in the hst4Δ HST3-FRB RRP6-FRB triple mutant and RRP6-FRB single mutant, and these datasets were compared to the NET-seq and RNA-seq datasets from the hst4Δ HST3-FRB double mutant, using the same gene groups identified in Figure 3C (Figure 4A; Table S2). Inactivation of the nuclear exosome increased steady-state RNA levels to those more closely resembling what was observed by NET-seq in the absence of Hst3 and Hst4 (Figure 4A). Remarkably, many of the RNAs that increased due to depletion of Hst3 and Hst4 were also increased by the single depletion of the RNA exosome (Figures 4A–C, S3B, and S3C), indicating that Hst3 and Hst4 and the nuclear exosome regulate many of the same target genes. However, there are many genes in groups B–D whose expression is increased to a greater extent in the hst4Δ HST3-FRB RRP6-FRB triple mutant compared to the RRP6-FRB single mutant (Figures 4A, 4C, S3B, and S3C), and there are an additional ~800 genes that are increased ≥ 1.5-fold over WT (FDR ≤ 0.1, edgeR) only in the hst4Δ HST3-FRB RRP6-FRB triple mutant (Figures 4C and S3D). Taken together, the RNA-seq analyses in the absence of the nuclear exosome confirm the observations made by NET-seq. Transcription is elevated in the absence of Hst3 and Hst4, and at many loci the nuclear exosome functions to degrade the increased nascent transcripts. In addition, there are genes that are not targeted by Rrp6 for degradation despite increased NET-seq reads (Figure S3E). In particular, at group C genes, which encode for proteins involved in lipid, sterol, and fatty acid metabolism (Table S1), loss of Rrp6 has little to no effect on steady-state mRNA levels. Thus, indicating there are additional co-transcriptional or post-transcriptional mechanisms for regulating steady-state RNA levels in the hst4Δ HST3-FRB mutant.Sirtuins Prevent the Accumulation of R Loops That Cause Genomic InstabilityThe hst3 hst4 mutant is sensitive to genotoxic stress, and loss of Hst3 and Hst4 induces many genomic instability phenotypes (Celic et al., 2006). We hypothesized that increased transcription in the hst4Δ HST3-FRB mutant might be an underlying cause of genomic instability by increasing the prevalence of transcription-associated R-loops. We performed DNA-RNA immunoprecipitation sequencing (DRIP-seq) on WT and hst4Δ HST3-FRB asynchronous cells treated with rapamycin for 3 h to deplete Hst3. We analyzed DRIP-seq reads over open reading frames (ORFs) and compared the signals to RNase-H treated controls. Higher DRIP-seq signals were observed in the mutant compared to WT (214 ≥ 0.59 versus 58 ≤ −0.59, FDR ≤ 0.25) (Figures 5A and 5B), indicating that there is an increased abundance of R-loops in the absence of Hst3 and Hst4. Many of the R-loops identified overlapped with previously mapped R-loops in a rnh1Δ rnh201Δ double mutant (Wahba et al., 2016), and we confirmed the increased presence of R-loops at several genes by DRIP-qPCR (Figure S4A).Metagene analysis was performed on genes that had a 1.3-fold or greater DRIP-seq signal in the WT or mutant relative to their respective RNase-H controls (1,105 genes), and as expected, R-loop levels were higher in the hst4Δ HST3-FRB mutant compared to WT (Figure 5B). The increased DRIP-seq signals were found over coding regions, consistent with transcription-associated R-loop formation (Figure 5C). Indeed, nascent transcription levels are increased in the absence of Hst3 and Hst4 at these loci (Figures 5A and 5C). Together, the data suggest that R loops form at a subset of genes with increased nascent transcription in the absence of Hst3 and Hst4.As a secondary method of identifying R loops genome wide, MACS (model-based analysis of chromatin immunoprecipitation sequencing [ChIP-seq]) (Zhang et al., 2008) was used to identify genomic regions that were significantly enriched for DRIP-seq reads (667 peaks, p < 1 × 10−5) in the hst4Δ HST3-FRB mutant (Figure S4B). Of these, 588 peaks overlapped genes, and there was ~40% overlap with R-loop-enriched genes identified in Figure 5B( Figure S4C). In addition to ORFs, many of the peaks overlap with tRNA, transposable elements, long terminal repeats, and snRNAs and snoRNAs (Figures S4B and S4D), suggesting a possible role for Hst3 and Hst4 in regulating other genomic loci in addition to coding regions.Recently, it was reported that a small subset of R loops in a sen1Δ rnh1Δ rnh201Δ mutant are hotspots for irreparable DNA damage (Costantino and Koshland, 2018). These persistent sites for R loops hinder DNA repair pathways, leading to large stretches of single-stranded DNA (Costantino and Koshland, 2018), which can function as precursors for gross chromosomal rearrangements (GCRs). To investigate if there are DNA damage sites proximal to DRIP-seq peaks found in the hst4Δ HST3-FRB mutant, we performed Break-seq to identify DNA double-strand breaks (DSBs) (Hoffman et al., 2015) in hst4Δ HST3-FRB and WT cells treated with rapamycin for 3 h. MACS (Zhang et al., 2008) was used to identify genomic regions that were significantly enriched for end-labeled, DSB signals (239 peaks, p < 1 × 10−5) in the hst4Δ HST3-FRB mutant (Figures 6A, 6B, and S5A). Of these, 71 peaks overlapped within a 4-kb region centered around genomic regions that had DRIP-seq peaks in the hst4Δ HST3-FRB mutant (Figures 6A, 6B, and S5A). The large regions of DSBs around R loops (Figure 6B) are similar to what was observed previously for Rad52 ChIP-seq peaks identified in a rnh1Δ rnh201Δ SEN1-AID, consistent with impeded DNA repair. Together, the results identify new sites of DNA damage that are proximal to R loops in the hst4Δ HST3-FRB mutant.To provide functional support for the hypothesis that R loops cause genomic instability in the absence of Hst3 and Hst4, we overexpressed human RNase-H1 in WT and hst4Δ HST3-FRB cells and monitored their sensitivity to genotoxic agents by spot dilution assay. Expression of human RNase-H1 was previously shown to reduce R-loop levels in vivo in yeast (Wahba et al., 2011). Overexpression of RNase-H1 did not affect the growth of WT cells on DMSO or rapamycin, nor in the presence of 0.1 M HU, 0.005% MMS, or 5 μg/mL camptothecin (CPT) (Figures 6C and S5B). Strikingly, RNase-H1 overexpression partially suppressed the genomic instability of the hst4Δ HST3-FRB mutant grown on rapamycin in the presence of HU, MMS, and CPT (Figures 6C and S5B), providing direct evidence that increased R loops, in the absence of Hst3 and Hst4, cause genomic instability.

Hst3 and Hst4 Repress Nascent RNA TranscriptionDeletion of HST3 and HST4 leads to genomic instability (Brachmann et al., 1995; Celic et al., 2006), making such strains susceptible to second-site suppressor mutations. Notably, previous studies showed that the loss of both Hst3 and Hst4 is needed in order to observe measurable phenotypes (Celic et al., 2006), indicating that they perform redundant roles. Therefore, we wanted to establish an alternative approach to characterize the impact of Hst3 and Hst4 on transcription. To this end, the anchor away system (Haruki et al., 2008) was used to conditionally deplete Hst3 from the nucleus in a hst4Δ strain by tagging the C terminus of the HST3 locus with the FKBP12-rapamycin-binding (FRB) domain (hst4Δ HST3-FRB). The parent strain harbors a FK506 binding protein (FKBP12) fused to the C terminus of RPL13A, which is a highly abundant ribosomal protein that shuttles from the nucleus to the cytoplasm during ribosome assembly. A ternary complex between the FRB and FKBP12 domains is formed in the presence of rapamycin and, thus, rapidly depletes Hst3 from the nucleus. In addition, anchor away strains contain a rapamycin resistant tor1-1 allele to ensure rapamycin is not toxic to the wild-type (WT) strain (Haruki et al., 2008).We first confirmed that depletion of Hst3 in a hst4Δ displays a similar phenotype to a hst3Δ hst4Δ strain by spot dilution assay. The hst4Δ HST3-FRB strain is sensitive to 0.01% methyl metha-nesulfonate (MMS) and 0.1 M hydroxyurea (HU) only in the presence of rapamycin, similar to the hst3Δ hst4Δ strain in the presence of MMS and HU on DMSO (Figure S1A). Consistent with redundant roles for Hst3 and Hst4, the individual HST3-FRB mutant is not sensitive to genotoxic agents in the presence of rapamycin (Figure S1A).Given that mammalian Sirt6 plays key roles in transcription, we sought to determine the impact of yeast Hst3 and Hst4 on nascent RNA production. We performed NET-seq (Churchman and Weissman, 2011) in WT and the hst4Δ HST3-FRB mutant, using asynchronous cells treated with rapamycin for 3 h. Since we anticipated a potential for global changes in transcription, S. pombe cells were used as a spike-in control for library normalization. Loss of Hst3 and Hst4 led to a global shift in the nascent RNA transcriptome, with an average fold increase of ~1.4 (p < 2 × 10−16, Mann-Whitney U test) (Figure 1A), and approximately a quarter of the genome (1,092 genes) increased by 1.5-fold or greater (false discovery rate [FDR] ≤ 0.1) in the hst4Δ HST3-FRB mutant. Loss of Hst3 and Hst4 has a somewhat larger impact on poorly expressed genes, as the log2 fold change (LFC) between mutant and WT cells is greater for the bottom 25% and 50% of genes transcribed in WT, compared to the top quatriles (p < 2.2 × 10−16, Mann-Whitney U test) (Figure 1B).Metagene plots of mean nascent transcript levels, representative genome browser views of NET-seq data, and a heatmap of the log2-fold change between the hst4Δ HST3-FRB mutant and WT confirmed higher levels of transcription throughout genic regions (Figures 1C, S1B, and S1C) in the mutant compared to the WT, especially for genes within the lowest quartile of expression levels (Figure 1C). Genes within the top 25% of the WT expression level also showed increases in nascent RNA, though these increases were greater near the TSS compared to the gene body and transcription termination site (TTS) (Figure 1C, bottom; Figure S1C). In addition, there was an increase in the 5’ to 3’ ratio of RNA transcripts genome wide (p < 2.2 × 10−16, Mann-Whitney U test) (Figure S1D). As further evidence that loss of Hst3 and Hst4 causes a shift in the Pol II distribution toward the TSS, we analyzed the distribution of Pol II after normalizing for differences in overall transcription. In agreement with the 5’ to 3’ ratios (Figure S1D), we observed a shift in Pol II distribution toward the TSS and a corresponding decrease near the TTS (Figure 1D). Taken together, our analyses indicate that Hst3 and Hst4 repress transcription initiation and, furthermore, that the absence of these sirtuins leads to the accumulation of Pol II near the TSS, which may be indicative of increased Pol II pausing.

Hst3 and Hst4 Repress Nascent RNA Transcription

Deletion of HST3 and HST4 leads to genomic instability (Brachmann et al., 1995; Celic et al., 2006), making such strains susceptible to second-site suppressor mutations. Notably, previous studies showed that the loss of both Hst3 and Hst4 is needed in order to observe measurable phenotypes (Celic et al., 2006), indicating that they perform redundant roles. Therefore, we wanted to establish an alternative approach to characterize the impact of Hst3 and Hst4 on transcription. To this end, the anchor away system (Haruki et al., 2008) was used to conditionally deplete Hst3 from the nucleus in a hst4Δ strain by tagging the C terminus of the HST3 locus with the FKBP12-rapamycin-binding (FRB) domain (hst4Δ HST3-FRB). The parent strain harbors a FK506 binding protein (FKBP12) fused to the C terminus of RPL13A, which is a highly abundant ribosomal protein that shuttles from the nucleus to the cytoplasm during ribosome assembly. A ternary complex between the FRB and FKBP12 domains is formed in the presence of rapamycin and, thus, rapidly depletes Hst3 from the nucleus. In addition, anchor away strains contain a rapamycin resistant tor1-1 allele to ensure rapamycin is not toxic to the wild-type (WT) strain (Haruki et al., 2008).

We first confirmed that depletion of Hst3 in a hst4Δ displays a similar phenotype to a hst3Δ hst4Δ strain by spot dilution assay. The hst4Δ HST3-FRB strain is sensitive to 0.01% methyl metha-nesulfonate (MMS) and 0.1 M hydroxyurea (HU) only in the presence of rapamycin, similar to the hst3Δ hst4Δ strain in the presence of MMS and HU on DMSO (Figure S1A). Consistent with redundant roles for Hst3 and Hst4, the individual HST3-FRB mutant is not sensitive to genotoxic agents in the presence of rapamycin (Figure S1A).

Given that mammalian Sirt6 plays key roles in transcription, we sought to determine the impact of yeast Hst3 and Hst4 on nascent RNA production. We performed NET-seq (Churchman and Weissman, 2011) in WT and the hst4Δ HST3-FRB mutant, using asynchronous cells treated with rapamycin for 3 h. Since we anticipated a potential for global changes in transcription, S. pombe cells were used as a spike-in control for library normalization. Loss of Hst3 and Hst4 led to a global shift in the nascent RNA transcriptome, with an average fold increase of ~1.4 (p < 2 × 10−16, Mann-Whitney U test) (Figure 1A), and approximately a quarter of the genome (1,092 genes) increased by 1.5-fold or greater (false discovery rate [FDR] ≤ 0.1) in the hst4Δ HST3-FRB mutant. Loss of Hst3 and Hst4 has a somewhat larger impact on poorly expressed genes, as the log2 fold change (LFC) between mutant and WT cells is greater for the bottom 25% and 50% of genes transcribed in WT, compared to the top quatriles (p < 2.2 × 10−16, Mann-Whitney U test) (Figure 1B).

Metagene plots of mean nascent transcript levels, representative genome browser views of NET-seq data, and a heatmap of the log2-fold change between the hst4Δ HST3-FRB mutant and WT confirmed higher levels of transcription throughout genic regions (Figures 1C, S1B, and S1C) in the mutant compared to the WT, especially for genes within the lowest quartile of expression levels (Figure 1C). Genes within the top 25% of the WT expression level also showed increases in nascent RNA, though these increases were greater near the TSS compared to the gene body and transcription termination site (TTS) (Figure 1C, bottom; Figure S1C). In addition, there was an increase in the 5’ to 3’ ratio of RNA transcripts genome wide (p < 2.2 × 10−16, Mann-Whitney U test) (Figure S1D). As further evidence that loss of Hst3 and Hst4 causes a shift in the Pol II distribution toward the TSS, we analyzed the distribution of Pol II after normalizing for differences in overall transcription. In agreement with the 5’ to 3’ ratios (Figure S1D), we observed a shift in Pol II distribution toward the TSS and a corresponding decrease near the TTS (Figure 1D). Taken together, our analyses indicate that Hst3 and Hst4 repress transcription initiation and, furthermore, that the absence of these sirtuins leads to the accumulation of Pol II near the TSS, which may be indicative of increased Pol II pausing.

Hst3 and Hst4 Repress Transcription of Many Non-coding RNAsStudies in recent years have demonstrated that many eukaryotic promoters are inherently bidirectional (Scruggs et al., 2015; Wei et al., 2011), and transcription termination sequences, RNA degradation complexes, and chromatin modifying factors function to limit the abundance of divergent non-coding RNAs (ncRNAs) relative to mRNAs (Hainer et al., 2015; Huang and Workman, 2013; Marquardt et al., 2014; Wei et al., 2011; Whitehouse et al., 2007). We compared the abundance of divergent antisense nascent transcripts by NET-seq in WT and hst4Δ HST3-FRB mutant cells at tandem genes (2,716) by analyzing antisense reads in the region from −600 to −100 bp from the TSS. We observed a global increase in nascent transcripts up-stream of genes in the absence of Hst3 and Hst4, with an average fold increase of ~1.6 (p < 2 × 10−16, Mann-Whitney U test) (1,051 LFC ≥ 0.59, FDR ≤ 0.1) (Figure 2A). The increase in transcription maps around the −1 and −2 nucleosomes, with little change in the nucleosome depleted region (NDR; Figure 2B). These data are in agreement with previous results that showed increased divergent transcription by northern blot in the absence of Hst3 and Hst4 at several promoters (Marquardt et al., 2014).In addition to divergent antisense transcripts, we investigated the role of Hst3 and Hst4 on cryptic unstable transcript (CUT) levels. CUTs are 5’ capped and polyadenylated ~400 bp transcripts that are rapidly degraded due a high abundance of binding motifs for the Nrd1-Nab1-Sen1 (NNS) termination machinery and subsequent targeting by the nuclear exosome (Arigo et al., 2006; Schulz et al., 2013; Thiebaut et al., 2006). Nascent CUT RNAs are also increased in the hst4 HST3-FRB mutant compared to WT (1.6-fold, p = 6 × 10–13, Mann-Whitney U test) (245 LFC ≥ 0.59, FDR ≤ 0.1) (Figures 2C and 2D). Taken together, our analyses point to an important role for Hst3 and Hst4 in limiting non-coding RNA production.

Hst3 and Hst4 Repress Transcription of Many Non-coding RNAs

Studies in recent years have demonstrated that many eukaryotic promoters are inherently bidirectional (Scruggs et al., 2015; Wei et al., 2011), and transcription termination sequences, RNA degradation complexes, and chromatin modifying factors function to limit the abundance of divergent non-coding RNAs (ncRNAs) relative to mRNAs (Hainer et al., 2015; Huang and Workman, 2013; Marquardt et al., 2014; Wei et al., 2011; Whitehouse et al., 2007). We compared the abundance of divergent antisense nascent transcripts by NET-seq in WT and hst4Δ HST3-FRB mutant cells at tandem genes (2,716) by analyzing antisense reads in the region from −600 to −100 bp from the TSS. We observed a global increase in nascent transcripts up-stream of genes in the absence of Hst3 and Hst4, with an average fold increase of ~1.6 (p < 2 × 10−16, Mann-Whitney U test) (1,051 LFC ≥ 0.59, FDR ≤ 0.1) (Figure 2A). The increase in transcription maps around the −1 and −2 nucleosomes, with little change in the nucleosome depleted region (NDR; Figure 2B). These data are in agreement with previous results that showed increased divergent transcription by northern blot in the absence of Hst3 and Hst4 at several promoters (Marquardt et al., 2014).

In addition to divergent antisense transcripts, we investigated the role of Hst3 and Hst4 on cryptic unstable transcript (CUT) levels. CUTs are 5’ capped and polyadenylated ~400 bp transcripts that are rapidly degraded due a high abundance of binding motifs for the Nrd1-Nab1-Sen1 (NNS) termination machinery and subsequent targeting by the nuclear exosome (Arigo et al., 2006; Schulz et al., 2013; Thiebaut et al., 2006). Nascent CUT RNAs are also increased in the hst4 HST3-FRB mutant compared to WT (1.6-fold, p = 6 × 10–13, Mann-Whitney U test) (245 LFC ≥ 0.59, FDR ≤ 0.1) (Figures 2C and 2D). Taken together, our analyses point to an important role for Hst3 and Hst4 in limiting non-coding RNA production.

Steady-State RNA Pool Minimally Affected by Loss of Hst3 and Hst4Our NET-seq analyses indicated that Hst3 and Hst4 repress the transcription initiation of genes. Therefore, we investigated whether the increase in nascent transcription translated to increased steady-state mRNA levels by analyzing RNA profiles by stranded RNA-seq. Similar to NET-seq analyses, S. pombe cells were used as a spike-in control for library normalization. Unexpectedly, and in contrast to the global increase in nascent transcription, steady-state mRNA levels remained relatively unchanged in the absence of Hst3 and Hst4 (0.981-fold, p = 0.02, Mann-Whitney U test) (Figure 3A). Hst3 and Hst4 negatively regulate the steady-state RNA level of 225 genes (FDR ≤ 0.1, LFC ≥ 0.59, edgeR) and positively regulate 85 genes (FDR ≤ 0.1, LFC ≤ −0.59, edgeR) (Figure S2A). Consistent with what was observed at the nascent RNA level, only very poorly transcribed genes in WT were increased to a greater extent in the mutant compared to the highly transcribed genes (p < 2.2 × 10−16, Mann-Whitney U test) (Figure 3B). Taken together, these results are similar to our previous analyses of the H3-K56 acetyltransferase Rtt109, in which we observed little change in the steady-state mRNA pool despite a global decrease in Pol II occupancy in a rtt109Δ strain (Rege et al., 2015).The observation that the steady-state mRNA pool remains relatively unchanged even though there is a global increase in nascent RNA production led us to investigate the similarities and differences between the NET-seq and RNA-seq datasets. A k-means clustering approach was used to identify subsets of genes that are differentially regulated at the nascent and steady-state levels (Figure 3C; Table S1; see also Figures S2B and S2C for genome browser views). Group A genes, which are highly transcribed in WT cells (Figure 3D), show small increases at the nascent RNA level, whereas many show an opposite, decreased level in the steady-state RNA pool (Figure 3C). Nascent RNA transcription of the other three groups of genes (groups B–D) was increased in the absence of Hst3 and Hst4, but variable effects are observed at the steady-state level (Figure 3C). With the exception of group B genes, which are the most poorly transcribed genes in WT cells (Figure 3D) and are upregulated in the hst4Δ HST3-FRB mutant by RNA-seq analyses, steady-state mRNA levels are minimally affected (group D) or are decreased (group C) in the hst4Δ HST3-FRB mutant (Figure 3C). The results reveal that many of the increased transcripts observed at the nascent RNA level in the absence of Hst3 and Hst4 are post-transcriptionally regulated and, thus, are not observed in the steady-state RNA pool.

Steady-State RNA Pool Minimally Affected by Loss of Hst3 and Hst4

Our NET-seq analyses indicated that Hst3 and Hst4 repress the transcription initiation of genes. Therefore, we investigated whether the increase in nascent transcription translated to increased steady-state mRNA levels by analyzing RNA profiles by stranded RNA-seq. Similar to NET-seq analyses, S. pombe cells were used as a spike-in control for library normalization. Unexpectedly, and in contrast to the global increase in nascent transcription, steady-state mRNA levels remained relatively unchanged in the absence of Hst3 and Hst4 (0.981-fold, p = 0.02, Mann-Whitney U test) (Figure 3A). Hst3 and Hst4 negatively regulate the steady-state RNA level of 225 genes (FDR ≤ 0.1, LFC ≥ 0.59, edgeR) and positively regulate 85 genes (FDR ≤ 0.1, LFC ≤ −0.59, edgeR) (Figure S2A). Consistent with what was observed at the nascent RNA level, only very poorly transcribed genes in WT were increased to a greater extent in the mutant compared to the highly transcribed genes (p < 2.2 × 10−16, Mann-Whitney U test) (Figure 3B). Taken together, these results are similar to our previous analyses of the H3-K56 acetyltransferase Rtt109, in which we observed little change in the steady-state mRNA pool despite a global decrease in Pol II occupancy in a rtt109Δ strain (Rege et al., 2015).

The observation that the steady-state mRNA pool remains relatively unchanged even though there is a global increase in nascent RNA production led us to investigate the similarities and differences between the NET-seq and RNA-seq datasets. A k-means clustering approach was used to identify subsets of genes that are differentially regulated at the nascent and steady-state levels (Figure 3C; Table S1; see also Figures S2B and S2C for genome browser views). Group A genes, which are highly transcribed in WT cells (Figure 3D), show small increases at the nascent RNA level, whereas many show an opposite, decreased level in the steady-state RNA pool (Figure 3C). Nascent RNA transcription of the other three groups of genes (groups B–D) was increased in the absence of Hst3 and Hst4, but variable effects are observed at the steady-state level (Figure 3C). With the exception of group B genes, which are the most poorly transcribed genes in WT cells (Figure 3D) and are upregulated in the hst4Δ HST3-FRB mutant by RNA-seq analyses, steady-state mRNA levels are minimally affected (group D) or are decreased (group C) in the hst4Δ HST3-FRB mutant (Figure 3C). The results reveal that many of the increased transcripts observed at the nascent RNA level in the absence of Hst3 and Hst4 are post-transcriptionally regulated and, thus, are not observed in the steady-state RNA pool.

Increased Transcription in the Absence of Hst3 and Hst4 Is Masked by the Nuclear ExosomeIn addition to a role in regulating ncRNA transcription (Schneider et al., 2012) and processing small nuclear and nucleolar RNAs (snRNAs and snoRNAs) (Gudipati et al., 2012), the nuclear exosome plays a more general role in the surveillance of nuclear mRNAs (Rege et al., 2015; Schmid et al., 2012). To investigate whether the nuclear exosome might be responsible for masking the impact of Hst3 and Hst4 loss on the steady-state RNA pool, the anchor away system was used to deplete the 3’ to 5’ exonuclease subunit, Rrp6, from the nucleus for 3 h, alone or in combination with the hst4Δ HST3-FRB mutant. Interestingly, growth assays revealed an additive effect of depleting Rrp6 in the absence of both Hst3 and Hst4, as the cells become more sensitive to HU compared to either the RRP6-FRB single mutant or the hst4Δ HST3-FRB double mutant (Figure S3A).RNA-seq was performed in the hst4Δ HST3-FRB RRP6-FRB triple mutant and RRP6-FRB single mutant, and these datasets were compared to the NET-seq and RNA-seq datasets from the hst4Δ HST3-FRB double mutant, using the same gene groups identified in Figure 3C (Figure 4A; Table S2). Inactivation of the nuclear exosome increased steady-state RNA levels to those more closely resembling what was observed by NET-seq in the absence of Hst3 and Hst4 (Figure 4A). Remarkably, many of the RNAs that increased due to depletion of Hst3 and Hst4 were also increased by the single depletion of the RNA exosome (Figures 4A–C, S3B, and S3C), indicating that Hst3 and Hst4 and the nuclear exosome regulate many of the same target genes. However, there are many genes in groups B–D whose expression is increased to a greater extent in the hst4Δ HST3-FRB RRP6-FRB triple mutant compared to the RRP6-FRB single mutant (Figures 4A, 4C, S3B, and S3C), and there are an additional ~800 genes that are increased ≥ 1.5-fold over WT (FDR ≤ 0.1, edgeR) only in the hst4Δ HST3-FRB RRP6-FRB triple mutant (Figures 4C and S3D). Taken together, the RNA-seq analyses in the absence of the nuclear exosome confirm the observations made by NET-seq. Transcription is elevated in the absence of Hst3 and Hst4, and at many loci the nuclear exosome functions to degrade the increased nascent transcripts. In addition, there are genes that are not targeted by Rrp6 for degradation despite increased NET-seq reads (Figure S3E). In particular, at group C genes, which encode for proteins involved in lipid, sterol, and fatty acid metabolism (Table S1), loss of Rrp6 has little to no effect on steady-state mRNA levels. Thus, indicating there are additional co-transcriptional or post-transcriptional mechanisms for regulating steady-state RNA levels in the hst4Δ HST3-FRB mutant.

Increased Transcription in the Absence of Hst3 and Hst4 Is Masked by the Nuclear Exosome

In addition to a role in regulating ncRNA transcription (Schneider et al., 2012) and processing small nuclear and nucleolar RNAs (snRNAs and snoRNAs) (Gudipati et al., 2012), the nuclear exosome plays a more general role in the surveillance of nuclear mRNAs (Rege et al., 2015; Schmid et al., 2012). To investigate whether the nuclear exosome might be responsible for masking the impact of Hst3 and Hst4 loss on the steady-state RNA pool, the anchor away system was used to deplete the 3’ to 5’ exonuclease subunit, Rrp6, from the nucleus for 3 h, alone or in combination with the hst4Δ HST3-FRB mutant. Interestingly, growth assays revealed an additive effect of depleting Rrp6 in the absence of both Hst3 and Hst4, as the cells become more sensitive to HU compared to either the RRP6-FRB single mutant or the hst4Δ HST3-FRB double mutant (Figure S3A).

RNA-seq was performed in the hst4Δ HST3-FRB RRP6-FRB triple mutant and RRP6-FRB single mutant, and these datasets were compared to the NET-seq and RNA-seq datasets from the hst4Δ HST3-FRB double mutant, using the same gene groups identified in Figure 3C (Figure 4A; Table S2). Inactivation of the nuclear exosome increased steady-state RNA levels to those more closely resembling what was observed by NET-seq in the absence of Hst3 and Hst4 (Figure 4A). Remarkably, many of the RNAs that increased due to depletion of Hst3 and Hst4 were also increased by the single depletion of the RNA exosome (Figures 4A–C, S3B, and S3C), indicating that Hst3 and Hst4 and the nuclear exosome regulate many of the same target genes. However, there are many genes in groups B–D whose expression is increased to a greater extent in the hst4Δ HST3-FRB RRP6-FRB triple mutant compared to the RRP6-FRB single mutant (Figures 4A, 4C, S3B, and S3C), and there are an additional ~800 genes that are increased ≥ 1.5-fold over WT (FDR ≤ 0.1, edgeR) only in the hst4Δ HST3-FRB RRP6-FRB triple mutant (Figures 4C and S3D). Taken together, the RNA-seq analyses in the absence of the nuclear exosome confirm the observations made by NET-seq. Transcription is elevated in the absence of Hst3 and Hst4, and at many loci the nuclear exosome functions to degrade the increased nascent transcripts. In addition, there are genes that are not targeted by Rrp6 for degradation despite increased NET-seq reads (Figure S3E). In particular, at group C genes, which encode for proteins involved in lipid, sterol, and fatty acid metabolism (Table S1), loss of Rrp6 has little to no effect on steady-state mRNA levels. Thus, indicating there are additional co-transcriptional or post-transcriptional mechanisms for regulating steady-state RNA levels in the hst4Δ HST3-FRB mutant.

Sirtuins Prevent the Accumulation of R Loops That Cause Genomic InstabilityThe hst3 hst4 mutant is sensitive to genotoxic stress, and loss of Hst3 and Hst4 induces many genomic instability phenotypes (Celic et al., 2006). We hypothesized that increased transcription in the hst4Δ HST3-FRB mutant might be an underlying cause of genomic instability by increasing the prevalence of transcription-associated R-loops. We performed DNA-RNA immunoprecipitation sequencing (DRIP-seq) on WT and hst4Δ HST3-FRB asynchronous cells treated with rapamycin for 3 h to deplete Hst3. We analyzed DRIP-seq reads over open reading frames (ORFs) and compared the signals to RNase-H treated controls. Higher DRIP-seq signals were observed in the mutant compared to WT (214 ≥ 0.59 versus 58 ≤ −0.59, FDR ≤ 0.25) (Figures 5A and 5B), indicating that there is an increased abundance of R-loops in the absence of Hst3 and Hst4. Many of the R-loops identified overlapped with previously mapped R-loops in a rnh1Δ rnh201Δ double mutant (Wahba et al., 2016), and we confirmed the increased presence of R-loops at several genes by DRIP-qPCR (Figure S4A).Metagene analysis was performed on genes that had a 1.3-fold or greater DRIP-seq signal in the WT or mutant relative to their respective RNase-H controls (1,105 genes), and as expected, R-loop levels were higher in the hst4Δ HST3-FRB mutant compared to WT (Figure 5B). The increased DRIP-seq signals were found over coding regions, consistent with transcription-associated R-loop formation (Figure 5C). Indeed, nascent transcription levels are increased in the absence of Hst3 and Hst4 at these loci (Figures 5A and 5C). Together, the data suggest that R loops form at a subset of genes with increased nascent transcription in the absence of Hst3 and Hst4.As a secondary method of identifying R loops genome wide, MACS (model-based analysis of chromatin immunoprecipitation sequencing [ChIP-seq]) (Zhang et al., 2008) was used to identify genomic regions that were significantly enriched for DRIP-seq reads (667 peaks, p < 1 × 10−5) in the hst4Δ HST3-FRB mutant (Figure S4B). Of these, 588 peaks overlapped genes, and there was ~40% overlap with R-loop-enriched genes identified in Figure 5B( Figure S4C). In addition to ORFs, many of the peaks overlap with tRNA, transposable elements, long terminal repeats, and snRNAs and snoRNAs (Figures S4B and S4D), suggesting a possible role for Hst3 and Hst4 in regulating other genomic loci in addition to coding regions.Recently, it was reported that a small subset of R loops in a sen1Δ rnh1Δ rnh201Δ mutant are hotspots for irreparable DNA damage (Costantino and Koshland, 2018). These persistent sites for R loops hinder DNA repair pathways, leading to large stretches of single-stranded DNA (Costantino and Koshland, 2018), which can function as precursors for gross chromosomal rearrangements (GCRs). To investigate if there are DNA damage sites proximal to DRIP-seq peaks found in the hst4Δ HST3-FRB mutant, we performed Break-seq to identify DNA double-strand breaks (DSBs) (Hoffman et al., 2015) in hst4Δ HST3-FRB and WT cells treated with rapamycin for 3 h. MACS (Zhang et al., 2008) was used to identify genomic regions that were significantly enriched for end-labeled, DSB signals (239 peaks, p < 1 × 10−5) in the hst4Δ HST3-FRB mutant (Figures 6A, 6B, and S5A). Of these, 71 peaks overlapped within a 4-kb region centered around genomic regions that had DRIP-seq peaks in the hst4Δ HST3-FRB mutant (Figures 6A, 6B, and S5A). The large regions of DSBs around R loops (Figure 6B) are similar to what was observed previously for Rad52 ChIP-seq peaks identified in a rnh1Δ rnh201Δ SEN1-AID, consistent with impeded DNA repair. Together, the results identify new sites of DNA damage that are proximal to R loops in the hst4Δ HST3-FRB mutant.To provide functional support for the hypothesis that R loops cause genomic instability in the absence of Hst3 and Hst4, we overexpressed human RNase-H1 in WT and hst4Δ HST3-FRB cells and monitored their sensitivity to genotoxic agents by spot dilution assay. Expression of human RNase-H1 was previously shown to reduce R-loop levels in vivo in yeast (Wahba et al., 2011). Overexpression of RNase-H1 did not affect the growth of WT cells on DMSO or rapamycin, nor in the presence of 0.1 M HU, 0.005% MMS, or 5 μg/mL camptothecin (CPT) (Figures 6C and S5B). Strikingly, RNase-H1 overexpression partially suppressed the genomic instability of the hst4Δ HST3-FRB mutant grown on rapamycin in the presence of HU, MMS, and CPT (Figures 6C and S5B), providing direct evidence that increased R loops, in the absence of Hst3 and Hst4, cause genomic instability.

Sirtuins Prevent the Accumulation of R Loops That Cause Genomic Instability

The hst3 hst4 mutant is sensitive to genotoxic stress, and loss of Hst3 and Hst4 induces many genomic instability phenotypes (Celic et al., 2006). We hypothesized that increased transcription in the hst4Δ HST3-FRB mutant might be an underlying cause of genomic instability by increasing the prevalence of transcription-associated R-loops. We performed DNA-RNA immunoprecipitation sequencing (DRIP-seq) on WT and hst4Δ HST3-FRB asynchronous cells treated with rapamycin for 3 h to deplete Hst3. We analyzed DRIP-seq reads over open reading frames (ORFs) and compared the signals to RNase-H treated controls. Higher DRIP-seq signals were observed in the mutant compared to WT (214 ≥ 0.59 versus 58 ≤ −0.59, FDR ≤ 0.25) (Figures 5A and 5B), indicating that there is an increased abundance of R-loops in the absence of Hst3 and Hst4. Many of the R-loops identified overlapped with previously mapped R-loops in a rnh1Δ rnh201Δ double mutant (Wahba et al., 2016), and we confirmed the increased presence of R-loops at several genes by DRIP-qPCR (Figure S4A).

Metagene analysis was performed on genes that had a 1.3-fold or greater DRIP-seq signal in the WT or mutant relative to their respective RNase-H controls (1,105 genes), and as expected, R-loop levels were higher in the hst4Δ HST3-FRB mutant compared to WT (Figure 5B). The increased DRIP-seq signals were found over coding regions, consistent with transcription-associated R-loop formation (Figure 5C). Indeed, nascent transcription levels are increased in the absence of Hst3 and Hst4 at these loci (Figures 5A and 5C). Together, the data suggest that R loops form at a subset of genes with increased nascent transcription in the absence of Hst3 and Hst4.

As a secondary method of identifying R loops genome wide, MACS (model-based analysis of chromatin immunoprecipitation sequencing [ChIP-seq]) (Zhang et al., 2008) was used to identify genomic regions that were significantly enriched for DRIP-seq reads (667 peaks, p < 1 × 10−5) in the hst4Δ HST3-FRB mutant (Figure S4B). Of these, 588 peaks overlapped genes, and there was ~40% overlap with R-loop-enriched genes identified in Figure 5B( Figure S4C). In addition to ORFs, many of the peaks overlap with tRNA, transposable elements, long terminal repeats, and snRNAs and snoRNAs (Figures S4B and S4D), suggesting a possible role for Hst3 and Hst4 in regulating other genomic loci in addition to coding regions.

Recently, it was reported that a small subset of R loops in a sen1Δ rnh1Δ rnh201Δ mutant are hotspots for irreparable DNA damage (Costantino and Koshland, 2018). These persistent sites for R loops hinder DNA repair pathways, leading to large stretches of single-stranded DNA (Costantino and Koshland, 2018), which can function as precursors for gross chromosomal rearrangements (GCRs). To investigate if there are DNA damage sites proximal to DRIP-seq peaks found in the hst4Δ HST3-FRB mutant, we performed Break-seq to identify DNA double-strand breaks (DSBs) (Hoffman et al., 2015) in hst4Δ HST3-FRB and WT cells treated with rapamycin for 3 h. MACS (Zhang et al., 2008) was used to identify genomic regions that were significantly enriched for end-labeled, DSB signals (239 peaks, p < 1 × 10−5) in the hst4Δ HST3-FRB mutant (Figures 6A, 6B, and S5A). Of these, 71 peaks overlapped within a 4-kb region centered around genomic regions that had DRIP-seq peaks in the hst4Δ HST3-FRB mutant (Figures 6A, 6B, and S5A). The large regions of DSBs around R loops (Figure 6B) are similar to what was observed previously for Rad52 ChIP-seq peaks identified in a rnh1Δ rnh201Δ SEN1-AID, consistent with impeded DNA repair. Together, the results identify new sites of DNA damage that are proximal to R loops in the hst4Δ HST3-FRB mutant.

To provide functional support for the hypothesis that R loops cause genomic instability in the absence of Hst3 and Hst4, we overexpressed human RNase-H1 in WT and hst4Δ HST3-FRB cells and monitored their sensitivity to genotoxic agents by spot dilution assay. Expression of human RNase-H1 was previously shown to reduce R-loop levels in vivo in yeast (Wahba et al., 2011). Overexpression of RNase-H1 did not affect the growth of WT cells on DMSO or rapamycin, nor in the presence of 0.1 M HU, 0.005% MMS, or 5 μg/mL camptothecin (CPT) (Figures 6C and S5B). Strikingly, RNase-H1 overexpression partially suppressed the genomic instability of the hst4Δ HST3-FRB mutant grown on rapamycin in the presence of HU, MMS, and CPT (Figures 6C and S5B), providing direct evidence that increased R loops, in the absence of Hst3 and Hst4, cause genomic instability.

DISCUSSIONChanges in transcription are tightly regulated to ensure transcription homeostasis, and mis-regulation can have widespread effects on cellular function. The mammalian tumor suppressor SIRT6 functions primarily as a H3-K56 and H3-K9 deacetylase at specific genes (Kugel and Mostoslavsky, 2014), and loss of SIRT6 is sufficient to drive tumorigenesis (Sebastián et al., 2012). The data presented here reveal a link between pervasive, unregulated transcription and the genomic instability phenotypes observed in the absence of the yeast SIRT6 homologs: Hst3 and Hst4. By using a combination of RNA sequencing methodologies (NET-seq and RNA-seq), yeast genetics, DRIP-seq, and Break-seq, we demonstrate that Hst3 and Hst4 are globally required to repress nascent RNA transcription, and transcription-associated R loops and DNA double-strand breaks are elevated in a hst3 hst4 mutant. Furthermore, increased R loops appear to be causative for genomic instability, at least in part, as we observe co-localization of both R loops and DSBs, and overexpression of human RNase H1 can partially alleviate the sensitivity of an hst3 hst4 double mutant to genotoxic stress. Given this partial suppression phenotype, it may be that other pathways are regulated by Hst3 and Hst4 independent of transcriptional regulation. For instance, mammalian Sirt6 also deacetylates nonhistone substrates that impact DNA repair pathways, and it also functions in telomere protection (reviewed in Kugel and Mostoslavsky, 2014). Likewise, several studies have found that increases in transcription are sometimes not sufficient for the formation of R loops (Bayona-Feliu et al., 2017; Salas-Armenteros et al., 2017).Promoter-proximal nucleosomes flanking active genes are highly acetylated (Rufiange et al., 2007; Yang et al., 2016) and display rapid, replication-independent nucleosome turnover (Dion et al., 2007; Rufiange et al., 2007; Yang et al., 2016). These results suggest that enhanced nucleosome dynamics and histone acetylation are generally beneficial for transcription. Consistent with this view, we find that nascent transcription increases globally in the absence of Hst3 and Hst4, likely due to increased histone acetylation levels, such as H3-K56Ac and/or H3-K9Ac. The increase in nascent transcription is predominantly observed around gene promoters, consistent with enhanced promoter nucleosome dynamics and the creation of a more favorable environment for transcription initiation. While promoter-proximal pausing of Pol II has not been observed in yeast, NET-seq profiles in WT strains show a general accumulation of nascent, coding transcripts near the 5’ end of genes. This position correlates with the location of the +2 nucleosome and the transition point from Pol II transcription initiation to productive elongation, which has been reported to function as a “kinetic” checkpoint (Buratowski, 2009; Rodríguez-Molina et al., 2016). After depletion of Hst3 and Hst4, we observed a greater number of RNA molecules near the 5’ end compared to the 3’ end and the gene body, indicating that many of the polymerases may not transition to productive elongation. This could be a direct effect of increased histone acetylation or indicate that the transition to transcription elongation is influenced by the density of Pol II. Consistent with the latter model, high-density Pol II genes are targets for the nuclear exosome (Rege et al., 2015). Even though termination factors, Nrd1 and Nab3, have been shown to preferentially bind ncRNAs (Schulz et al., 2013), transcription attenuation has been observed for protein coding genes (Colin et al., 2011; Mischo and Proudfoot, 2013). Furthermore, Nrd1 was reported to cross-link to the 5’ end of highly expressed genes (Creamer et al., 2011), providing additional support for the second model.The significant increase in nascent RNA, combined with minimal changes in steady-state RNA, indicated that post-transcriptional processes are functioning to regulate the increase in nascent transcription. Rapid depletion of the 3’ to 5’ exonuclease subunit of the nuclear RNA exosome, Rrp6, increased steady-state RNA levels, such that expression in the hst4Δ HST3-FRB RRP6-FRB strain more closely resembled nascent transcript abundance observed in the absence of Hst3 and Hst4. Thus, the nuclear exosome appears to repress the increased nascent transcription observed in the hst4Δ HST3-FRB mutant in order to maintain similar steady-state levels of mRNA in the absence of Hst3 and Hst4. We found that rapid depletion of the nuclear exosome, in an otherwise WT strain, increased expression levels of nearly 800 protein-coding genes, and the majority of these genes are also regulated by Hst3 and Hst4. This suggests that histone deacetylation and post-transcriptional exosome activity provides two distinct levels of regulation to maintain transcription homeostasis of a common set of genes. The results are similar to previous studies showing that mRNA synthesis and degradation are dynamically balanced to buffer against changes in either process (Timmers and Tora, 2018). Our results presented here, as well as our previous data for cells lacking the H3-K56 acetyltransferase, Rtt109 (Rege et al., 2015), indicate that a functional relationship might exist between H3-K56Ac, Pol II abundance, and nuclear exosome activity, which may affect the number of polymerases that proceed to productive elongation.Deletion of HST3 and HST4 induces many genome instability phenotypes (Celic et al., 2006; Che et al., 2015), which can be suppressed by an H3-K56R substitution derivative or by inactivation of the Asf1 subunit of the Rtt109 HAT complex (Celic et al., 2006, 2008; Maas et al., 2006), indicating that H3-K56 hyperacetylation is the cause of the genomic instability. Hst3 and Hst4 remove H3-K56Ac throughout the cell cycle, with the highest activity observed during S phase (Celic et al., 2006; Maas et al., 2006). Chronic γ-H2A phosphorylation is observed in a hst3Δ hst4Δ strain (Celic et al., 2006), and overexpression of a clamp loader protein, RFC1, suppresses hst3 hst4 phenotypes (Celic et al., 2008), indicating that Hst3 and Hst4 prevent DNA damage during replication. The current prevailing model is that prolonged hyperacetylation of H3-K56 in the absence of Hst3 and Hst4 leads to DNA damage by causing the replication machinery to interact with parental nucleosomes ahead of the fork that are H3-K56 acetylated, for example by creating a roadblock that impedes fork progression or affects fork stability (Celic et al., 2006). However, H3-K56Ac nucleosomes are more dynamic than non-H3-K56Ac nucleosomes (Kaplan et al., 2008; Rufiange et al., 2007; Yang et al., 2016), which suggests that they should not impede the replication fork.We investigated an alternative hypothesis that pervasive, unregulated transcription in a hst3 hst4 mutant poses a risk to the cell by increasing the propensity for transcription-associated R-loop formation. Many of the phenotypes observed in hst3 hst4 mutants are similar to what is observed in cells with unregulated R loops. Previous studies indicate that unregulated R-loops are a source of genomic instability by hindering DNA repair pathways and causing DNA damage during replication (Aguilera and Gómez-González, 2017; Costantino and Koshland, 2018). R-loop abundance, measured by DRIP-seq, is increased over genes that have higher nascent transcription in the absence of Hst3 and Hst4. Furthermore, Break-seq reads are higher in several regions that had DRIP-seq peaks in the hst4Δ HST3-FRB mutant, pointing to the presence of R-loop-induced DNA damage at these sites. Strikingly, overexpression of human RNase-H1 partially suppresses the sensitivity of the hst4Δ HST3-FRB mutant to genotoxic agents, providing direct evidence that R loops contribute in part to the genomic instability of hst3 hst4 mutants. Unregulated R loops in the absence of Hst3 and Hst4 might directly impact replication fork stability, providing an alternative model for how the absence of Sirtuins impacts the replisome.Our findings suggest an additional chromatin-based mechanism used by cells to regulate transcription and demonstrate that Hst3 and Hst4 function to maintain genome stability by repressing transcription of both coding and non-coding RNAs genome wide. Increased transcription in a hst3 hst4 mutant leads to genomic instability by increasing the abundance of R loops, providing a functional link between pervasive transcription and genomic instability. Given the phenotypic similarities between yeast and mammalian Sirt6 mutants, it seems likely that this functional relationship is evolutionarily conserved.

Changes in transcription are tightly regulated to ensure transcription homeostasis, and mis-regulation can have widespread effects on cellular function. The mammalian tumor suppressor SIRT6 functions primarily as a H3-K56 and H3-K9 deacetylase at specific genes (Kugel and Mostoslavsky, 2014), and loss of SIRT6 is sufficient to drive tumorigenesis (Sebastián et al., 2012). The data presented here reveal a link between pervasive, unregulated transcription and the genomic instability phenotypes observed in the absence of the yeast SIRT6 homologs: Hst3 and Hst4. By using a combination of RNA sequencing methodologies (NET-seq and RNA-seq), yeast genetics, DRIP-seq, and Break-seq, we demonstrate that Hst3 and Hst4 are globally required to repress nascent RNA transcription, and transcription-associated R loops and DNA double-strand breaks are elevated in a hst3 hst4 mutant. Furthermore, increased R loops appear to be causative for genomic instability, at least in part, as we observe co-localization of both R loops and DSBs, and overexpression of human RNase H1 can partially alleviate the sensitivity of an hst3 hst4 double mutant to genotoxic stress. Given this partial suppression phenotype, it may be that other pathways are regulated by Hst3 and Hst4 independent of transcriptional regulation. For instance, mammalian Sirt6 also deacetylates nonhistone substrates that impact DNA repair pathways, and it also functions in telomere protection (reviewed in Kugel and Mostoslavsky, 2014). Likewise, several studies have found that increases in transcription are sometimes not sufficient for the formation of R loops (Bayona-Feliu et al., 2017; Salas-Armenteros et al., 2017).

Promoter-proximal nucleosomes flanking active genes are highly acetylated (Rufiange et al., 2007; Yang et al., 2016) and display rapid, replication-independent nucleosome turnover (Dion et al., 2007; Rufiange et al., 2007; Yang et al., 2016). These results suggest that enhanced nucleosome dynamics and histone acetylation are generally beneficial for transcription. Consistent with this view, we find that nascent transcription increases globally in the absence of Hst3 and Hst4, likely due to increased histone acetylation levels, such as H3-K56Ac and/or H3-K9Ac. The increase in nascent transcription is predominantly observed around gene promoters, consistent with enhanced promoter nucleosome dynamics and the creation of a more favorable environment for transcription initiation. While promoter-proximal pausing of Pol II has not been observed in yeast, NET-seq profiles in WT strains show a general accumulation of nascent, coding transcripts near the 5’ end of genes. This position correlates with the location of the +2 nucleosome and the transition point from Pol II transcription initiation to productive elongation, which has been reported to function as a “kinetic” checkpoint (Buratowski, 2009; Rodríguez-Molina et al., 2016). After depletion of Hst3 and Hst4, we observed a greater number of RNA molecules near the 5’ end compared to the 3’ end and the gene body, indicating that many of the polymerases may not transition to productive elongation. This could be a direct effect of increased histone acetylation or indicate that the transition to transcription elongation is influenced by the density of Pol II. Consistent with the latter model, high-density Pol II genes are targets for the nuclear exosome (Rege et al., 2015). Even though termination factors, Nrd1 and Nab3, have been shown to preferentially bind ncRNAs (Schulz et al., 2013), transcription attenuation has been observed for protein coding genes (Colin et al., 2011; Mischo and Proudfoot, 2013). Furthermore, Nrd1 was reported to cross-link to the 5’ end of highly expressed genes (Creamer et al., 2011), providing additional support for the second model.

The significant increase in nascent RNA, combined with minimal changes in steady-state RNA, indicated that post-transcriptional processes are functioning to regulate the increase in nascent transcription. Rapid depletion of the 3’ to 5’ exonuclease subunit of the nuclear RNA exosome, Rrp6, increased steady-state RNA levels, such that expression in the hst4Δ HST3-FRB RRP6-FRB strain more closely resembled nascent transcript abundance observed in the absence of Hst3 and Hst4. Thus, the nuclear exosome appears to repress the increased nascent transcription observed in the hst4Δ HST3-FRB mutant in order to maintain similar steady-state levels of mRNA in the absence of Hst3 and Hst4. We found that rapid depletion of the nuclear exosome, in an otherwise WT strain, increased expression levels of nearly 800 protein-coding genes, and the majority of these genes are also regulated by Hst3 and Hst4. This suggests that histone deacetylation and post-transcriptional exosome activity provides two distinct levels of regulation to maintain transcription homeostasis of a common set of genes. The results are similar to previous studies showing that mRNA synthesis and degradation are dynamically balanced to buffer against changes in either process (Timmers and Tora, 2018). Our results presented here, as well as our previous data for cells lacking the H3-K56 acetyltransferase, Rtt109 (Rege et al., 2015), indicate that a functional relationship might exist between H3-K56Ac, Pol II abundance, and nuclear exosome activity, which may affect the number of polymerases that proceed to productive elongation.

Deletion of HST3 and HST4 induces many genome instability phenotypes (Celic et al., 2006; Che et al., 2015), which can be suppressed by an H3-K56R substitution derivative or by inactivation of the Asf1 subunit of the Rtt109 HAT complex (Celic et al., 2006, 2008; Maas et al., 2006), indicating that H3-K56 hyperacetylation is the cause of the genomic instability. Hst3 and Hst4 remove H3-K56Ac throughout the cell cycle, with the highest activity observed during S phase (Celic et al., 2006; Maas et al., 2006). Chronic γ-H2A phosphorylation is observed in a hst3Δ hst4Δ strain (Celic et al., 2006), and overexpression of a clamp loader protein, RFC1, suppresses hst3 hst4 phenotypes (Celic et al., 2008), indicating that Hst3 and Hst4 prevent DNA damage during replication. The current prevailing model is that prolonged hyperacetylation of H3-K56 in the absence of Hst3 and Hst4 leads to DNA damage by causing the replication machinery to interact with parental nucleosomes ahead of the fork that are H3-K56 acetylated, for example by creating a roadblock that impedes fork progression or affects fork stability (Celic et al., 2006). However, H3-K56Ac nucleosomes are more dynamic than non-H3-K56Ac nucleosomes (Kaplan et al., 2008; Rufiange et al., 2007; Yang et al., 2016), which suggests that they should not impede the replication fork.

We investigated an alternative hypothesis that pervasive, unregulated transcription in a hst3 hst4 mutant poses a risk to the cell by increasing the propensity for transcription-associated R-loop formation. Many of the phenotypes observed in hst3 hst4 mutants are similar to what is observed in cells with unregulated R loops. Previous studies indicate that unregulated R-loops are a source of genomic instability by hindering DNA repair pathways and causing DNA damage during replication (Aguilera and Gómez-González, 2017; Costantino and Koshland, 2018). R-loop abundance, measured by DRIP-seq, is increased over genes that have higher nascent transcription in the absence of Hst3 and Hst4. Furthermore, Break-seq reads are higher in several regions that had DRIP-seq peaks in the hst4Δ HST3-FRB mutant, pointing to the presence of R-loop-induced DNA damage at these sites. Strikingly, overexpression of human RNase-H1 partially suppresses the sensitivity of the hst4Δ HST3-FRB mutant to genotoxic agents, providing direct evidence that R loops contribute in part to the genomic instability of hst3 hst4 mutants. Unregulated R loops in the absence of Hst3 and Hst4 might directly impact replication fork stability, providing an alternative model for how the absence of Sirtuins impacts the replisome.

Our findings suggest an additional chromatin-based mechanism used by cells to regulate transcription and demonstrate that Hst3 and Hst4 function to maintain genome stability by repressing transcription of both coding and non-coding RNAs genome wide. Increased transcription in a hst3 hst4 mutant leads to genomic instability by increasing the abundance of R loops, providing a functional link between pervasive transcription and genomic instability. Given the phenotypic similarities between yeast and mammalian Sirt6 mutants, it seems likely that this functional relationship is evolutionarily conserved.

STAR★METHODSCONTACT FOR REAGENT AND RESOURCE SHARINGFurther information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Craig Peterson (craig.peterson@umassmed.edu).EXPERIMENTAL MODEL AND SUBJECT DETAILSThe S. cerevisiae strains used here are derived from HHY168 (MATα tor1-1 fpr1::NAT RPL13A-2X FKBP12::TRP1 ade 2-1 trp1-1 can1-100 leu2-3,112 his3-11,15 ura3 GAL psi+) and HHY221 (MATa tor1-1 fpr1::loxP-LEU2-loxP RPL13A-2 × FKBP12::loxP ade 2-1 trp1-1 can1-100 leu2-3,112 his3-11,15 ura3 GAL psi+). Growth conditions are detailed in the Method Details section. The full strain and plasmid list is shown in the Key Resources Table.METHOD DETAILSSpot Dilution Plate AssaysFor serial dilution spot plate assays, cells were cultured to saturation in 5 mL YPD or SD (-Ieu). Yeast was diluted to an OD600 of 1.0 in sterile dH2O, serially diluted 10-fold four times, and 5 μL of each dilution was spotted onto plates of indicated media. Where used, DMSO was 0.1% vol/vol, rapamycin was 8 μg/mL, methyl methanesulfonate was 0.01% or 0.005% wt/vol, hydroxyurea was 0.1 M, and camptothecin was 5 μg/mL.NET-seqLibrary ConstructionNET-seq libraries were produced as described in (Churchman and Weissman, 2011) for 4 WT replicates and 3 hstΔ HST3-FRB replicates. Briefly, overnight cultures from single yeast colonies were diluted to an OD600 = 0.05 in 1 L of YPD. Cells were grown at 30°C until OD600 = 0.25. Rapamycin was added at a final concentration of 8 μg/mL and cells were grown for 3 h (OD600 = 0.7-0.8). We utilized a S. pombe spike-in, which contained a flag-tagged Rpb3 subunit (JY741), to normalize the sequencing libraries. S. pombe cells were mixed with S. cerevisiae cells at a 1:10 ratio, and the cells were harvested as described in (Churchman and Weissman 2011). RNA Pol II IP, RNA purification, and library construction was carried out as described previously (Churchman and Weissman, 2011; Mayer et al., 2015). 3’ end sequencing was performed on an Illumina NextSeq 500 with a read length of 75 bp.Data AnalysisNET-seq reads were processed and aligned as follows using the Galaxy web platform (Afgan et al., 2018). The adaptor sequence was (ATCTCGTATGCCGTCTTCTGCTTG) removed and the random hexamer sequence was removed from the 5’ end. The 3’ ends of the reads were then trimmed for quality using FASTQ Quality Timmer by sliding window (Blankenberg et al., 2010) with a window size of 10 and a step size of 5. The reads were trimmed until the aggregate score was ≤ 21. Reads were first aligned using Bowtie2 (Langmead and Salzberg, 2012; Langmeadet al., 2009) to a combined FASTA file of S. cerevisiae and S. pombe rRNA, tRNA, and RDN sequences to remove contaminating reads. Reads were then aligned to a combined version of the S. cerevisiae genome (SacCer3, SGD) and the S. pombe genome (ASM294v.2, PomBase) with TopHat2 (Kim et al., 2013), allowing up to three mismatches. The reads were separated by their respective genomes with SAMtools (Li et al., 2009), and only uniquely mapped reads were used for further analyses. Libraries were normalized by scaling the uniquely mapped S. pombe reads to 100,000 reads. This scaling factor was then used to scale the uniquely mapped S. cerevisiae reads. To account for differences between sequencing run depth for various NextSeq runs, the pombe-scaled WT S. cerevisiae read counts were then scaled to 1 M reads, and this additional scaling factor was included to scale the sample reads. Finally, only the 5’ end of the sequencing read, which corresponds to the 3’ end of the nascent RNA was recorded and used for downstream analyses. TSS and TTS annotation was obtained from Xu et al. (2009). Read counts for genes and non-coding regions were obtained by summing normalized base pair reads over the region of interest. For average profiles, BAM files of biological replicates were merged and processed as above, and only genes longer than 500 bp were analyzed. Genes were scaled to 500 bp, and samples were scored in 1 bp bins using the deepTools program (Ramírez et al., 2016). Reads were analyzed as in Harlen et al. (2016). To calculate 5’ to 3’ ratios, the sum of reads from 1-250 bp from the TSS were divided by the sum of reads 250 bp upstream of the TTS to the TTS.RNA-seqLibrary ConstructionStrand-specific RNA-seq libraries without polyA selection were prepared similarly to Zhang et al. (2012) for 2 biological replicates. Briefly, overnight cultures from single yeast colonies were diluted to an OD600 = 0.05 in 50 mL of YPD. Cells were grown at 30°C until OD600 = 0.25. Rapamycin was added at a final concentration of 8 μg/mL and cells were grown for 3 h (OD600 = 0.7). We utilized a S. pombe spike-in to normalize the sequencing libraries. S. pombe cells were mixed with S. cerevisiae cells at a 1:6 ratio, and cells were harvested by centrifugation. RNA was purified by hot-phenol extraction and ethanol precipitation. RNA was further purified by RNeasy Miniprep kit and the DNA was digested. RiboZero magnetic beads (Illumina) were then used to remove rRNAs from 3 μg of RNA. Libraries were then prepared as in Zhang et al. (2012), and paired-end sequencing was performed on an Illumina NextSeq 500 with a read length of 75 bp.Data AnalysisFASTQ files from paired end libraries were collapsed by barcode and the Illumina adaptor sequence was trimmed from the 3’ end. Files were uploaded and analyzed using the Galaxy web platform (Afgan et al., 2018). Reads were first aligned using Bowtie2 (Langmead and Salzberg, 2012; Langmead et al., 2009) to a combined FASTA file of S. cerevisiae and S. pombe rRNA, tRNA, and RDN sequences to remove contaminating reads. Reads were then aligned to a combined version of the S. cerevisiae genome (SacCer3, SGD) and the S. pombe genome (ASM294v.2, PomBase) with TopHat2 (Kim et al., 2013) allowing up to two mismatches. The reads were separated by their respective genomes with SAMtools (Li et al., 2009), and only uniquely mapped reads were used for further analyses. For visualization in USCS genome browser, libraries were normalized by scaling the uniquely mapped S. pombe reads to 100,000 reads. To compare RNA expression between samples, HTseq 0.9.1 (Anders et al., 2015) was used to count the number of reads that aligned to each annotated gene. The annotation file for the S. cerevisiae genome was generated from the Xu et al. dataset (Xu et al., 2009), and the S. pombe annotation file was obtained from PomBase. Normalized S. cerevisiae read counts were generated by using a linear regression model to scale the S. pombe read counts relative to WT biological replicate 1 to determine scaling factors for S. cerevisiae read counts. Differential expression analysis was performed using edgeR (Robinson et al., 2010) or in excel using the R qvalue package.RT-qPCRDNase-treated RNA (1 μg) was reverse transcribed using SuperScript III (Invitrogen) and oligo-dT according to the manufacturer protocol. qPCR reactions were performed with PowerUp Syber Green Master Mix (Applied Biosystems). Primer sequences are listed in Supplemental Information. Relative mRNA levels were quantified in duplicate for biological replicates using the highly expressed PDC1 gene for normalization. Significance was determined by Student’s t test.DRIP-seqLibrary ConstructionDNA-RNA hybrid immunoprecipitation sequencing (DRIP-seq) was performed using the DNA-RNA hybrid specific S9.6 antibody similarly to Bonnet et al. (2017) for 2 biological replicates. Briefly, overnight cultures from single yeast colonies were diluted to an OD600 = 0.1 in 200 mL of YPD. Cells were grown at 30°C until OD600 ≅ 0.3. Rapamycin was added at a final concentration of 8 mg/mL and cells were grown for 3 h (OD600 ≅ 0.8). Genomic DNA was extracted by spheroplasting and EtOH precipitation, excluding a RNase-A digestion. Purified DNA (50 μg) was digested by a cocktail of restriction enzymes (50 U HindIII, EcorI, XbaI, SspI, BsrGI, NEB) in the presence or absence of 30 U of RNase-H (NEB) overnight at 37°C in a final volume of 110 μL. Digested DNA was further diluted to 500 uL with FA-lysis buffer (50 mM HEPES, pH 7.5, 140 mM NaCl, 1 mM EDTA, 1% Triton X-100, 0.1% sodium deoxycholate) and the samples were pre-cleared with Dynabeads M-280 Sheep Anti-Mouse IgG (ThermoFisher) for 3 h. DNA was then incubated overnight in the presence of 3 μg of S9.6 antibody (Millipore). Immunoprecipitated DNA fragments were captured using Dynabeads M-280 Sheep Anti-Mouse IgG (ThermoFisher), washed. and eluted according standard ChIP procedures (Bennett et al., 2013). Prior to phenol:chloroform extraction and precipiateion, the eluted DNA was incubated with 200 μg of Proteinase Kfor 1 h at 42°C while shaking at 1000 rpm to ensure complete antibody removal. Libraries were prepared by adaptor ligation and PCR amplification for paired-end sequencing on an Illumina NextSeq 500 with a read length of 75 bp.Data AnalysisFASTQ files from paired end libraries were collapsed by barcode and the Illumina adaptor sequence was trimmed from the 3’ end. Files were uploaded and analyzed using the Galaxy web platform (Afgan etal., 2018). Reads were aligned to the S. cerevisiae genome (SacCer3, SGD) using Bowtie2 with a maximum fragment length for valid paired-end alignments set to 500 bp. Aligned reads were then filtered for quality and only uniquely mapped paired reads were used for future analys3s. For visualization in USCS genome browser and metagene plots, aligned libraries were merged and normalized by scaling to 1 million reads. RNase-H control IP signal was subtracted from the matched sample. To calculate DRIP-seq signals over genes, BEDtools was used to calculate normalized bp counts across the genome for each replicate individually. Subsequent analysis was performed on median read coverage across 100-bp windows with a 50-bp sliding window. The median read counts were then summed over genes. DRIP-seq enriched genes were determined similarly to Zeller et al. (2016) by comparing DRIP-seq signals to RNase-H treated controls. The fold change in the signal versus a matched RNase-H control was calculated, and a 1.3-fold cutoff in the average of the two replicates for either WT or the hst4Δ HST3-FRB mutant was used to identify genes enriched for R-loops. Differential analysis between WT and the hst4Δ HST3-FRB mutant was performed using the R qvalue package. MACS Peak Caller was used to identify new peaks in a non-biased manner in the hst4Δ HST3-FRB biological replicates using the mean WT file as the input (p value 1×10−5, 500 bp fragment size).DRIP-qPCRCells were grown, genomic DNA was purified, and immunoprecipitated as done for DIP-seq. qPCR reactions were performed with PowerUp Syber Green Master Mix (Applied Biosystems). Primer sequences are listed in Supplemental Information. R-loop levels were determined relative to a matched RNase-H-treated sample for 3 biological replicates. Significance was determined by Student’s t test.Break-seqLibrary ConstructionBreak-seq libraries were prepared similarly to Hoffman et al. (2015) for 2 biological replicates. Briefly, overnight cultures from single yeast colonies were diluted to an OD600 = 0.1 in 100 mL of YPD. Cells were grown at 30°C until OD600 ≅ 0.32. Rapamycin was added at a final concentration of 8 mg/mLand cells were grown for 3 h (OD600 = 1.0). Cells were harvested and resuspended in 1 mL of 50 mM EDTA. Cells (50 μL) were combined with 50 μL of 1% low melting temperature agarose (Lonza). Following in gel labeling and sonication, libraries were prepared as in Hoffman et al. (2015). Paired-end sequencing was performed on an Illumina NextSeq 500 with a read length of 75 bp.Data AnalysisFASTQ files from paired end libraries were collapsed by barcode and the Illumina adaptor sequence was trimmed from the 3’ end. Files were uploaded and analyzed using the Galaxy web platform (Afgan et al., 2018). Reads were aligned to the S. cerevisiae genome (SacCer3, SGD) using Bowtie2 with a maximum fragment length for valid paired-end alignments set to 500 bp. Aligned reads were then filtered for quality and only uniquely mapped paired reads were used for future analyses. BAM files were uploaded to SeqMonk and normalized by scaling to 1 million reads. MACS Peak Caller was used to identify new peaks in hst4Δ HST3-FRB biological replicates using the mean WT file as the input (p value 1×10−5, 500 bp fragment size). Genome browser views were obtained by generating probes using a running window with a probe size of 1000 bp and a step size of 500 bp.QUANTIFICATION AND STATISTICAL ANALYSISSoftware and statistical analysis details can be found in the Method Details section of the STAR Methods, as well as the Key Resources Table.DATA AND SOFTWARE AVAILABILITYAll data from this study have been deposited in the Gene Expression Omnibus (GEO) under accession GEO: GSE124132.

STAR★METHODS

CONTACT FOR REAGENT AND RESOURCE SHARINGFurther information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Craig Peterson (craig.peterson@umassmed.edu).

CONTACT FOR REAGENT AND RESOURCE SHARING

Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Craig Peterson (craig.peterson@umassmed.edu).

EXPERIMENTAL MODEL AND SUBJECT DETAILSThe S. cerevisiae strains used here are derived from HHY168 (MATα tor1-1 fpr1::NAT RPL13A-2X FKBP12::TRP1 ade 2-1 trp1-1 can1-100 leu2-3,112 his3-11,15 ura3 GAL psi+) and HHY221 (MATa tor1-1 fpr1::loxP-LEU2-loxP RPL13A-2 × FKBP12::loxP ade 2-1 trp1-1 can1-100 leu2-3,112 his3-11,15 ura3 GAL psi+). Growth conditions are detailed in the Method Details section. The full strain and plasmid list is shown in the Key Resources Table.

EXPERIMENTAL MODEL AND SUBJECT DETAILS

The S. cerevisiae strains used here are derived from HHY168 (MATα tor1-1 fpr1::NAT RPL13A-2X FKBP12::TRP1 ade 2-1 trp1-1 can1-100 leu2-3,112 his3-11,15 ura3 GAL psi+) and HHY221 (MATa tor1-1 fpr1::loxP-LEU2-loxP RPL13A-2 × FKBP12::loxP ade 2-1 trp1-1 can1-100 leu2-3,112 his3-11,15 ura3 GAL psi+). Growth conditions are detailed in the Method Details section. The full strain and plasmid list is shown in the Key Resources Table.

METHOD DETAILSSpot Dilution Plate AssaysFor serial dilution spot plate assays, cells were cultured to saturation in 5 mL YPD or SD (-Ieu). Yeast was diluted to an OD600 of 1.0 in sterile dH2O, serially diluted 10-fold four times, and 5 μL of each dilution was spotted onto plates of indicated media. Where used, DMSO was 0.1% vol/vol, rapamycin was 8 μg/mL, methyl methanesulfonate was 0.01% or 0.005% wt/vol, hydroxyurea was 0.1 M, and camptothecin was 5 μg/mL.NET-seqLibrary ConstructionNET-seq libraries were produced as described in (Churchman and Weissman, 2011) for 4 WT replicates and 3 hstΔ HST3-FRB replicates. Briefly, overnight cultures from single yeast colonies were diluted to an OD600 = 0.05 in 1 L of YPD. Cells were grown at 30°C until OD600 = 0.25. Rapamycin was added at a final concentration of 8 μg/mL and cells were grown for 3 h (OD600 = 0.7-0.8). We utilized a S. pombe spike-in, which contained a flag-tagged Rpb3 subunit (JY741), to normalize the sequencing libraries. S. pombe cells were mixed with S. cerevisiae cells at a 1:10 ratio, and the cells were harvested as described in (Churchman and Weissman 2011). RNA Pol II IP, RNA purification, and library construction was carried out as described previously (Churchman and Weissman, 2011; Mayer et al., 2015). 3’ end sequencing was performed on an Illumina NextSeq 500 with a read length of 75 bp.Data AnalysisNET-seq reads were processed and aligned as follows using the Galaxy web platform (Afgan et al., 2018). The adaptor sequence was (ATCTCGTATGCCGTCTTCTGCTTG) removed and the random hexamer sequence was removed from the 5’ end. The 3’ ends of the reads were then trimmed for quality using FASTQ Quality Timmer by sliding window (Blankenberg et al., 2010) with a window size of 10 and a step size of 5. The reads were trimmed until the aggregate score was ≤ 21. Reads were first aligned using Bowtie2 (Langmead and Salzberg, 2012; Langmeadet al., 2009) to a combined FASTA file of S. cerevisiae and S. pombe rRNA, tRNA, and RDN sequences to remove contaminating reads. Reads were then aligned to a combined version of the S. cerevisiae genome (SacCer3, SGD) and the S. pombe genome (ASM294v.2, PomBase) with TopHat2 (Kim et al., 2013), allowing up to three mismatches. The reads were separated by their respective genomes with SAMtools (Li et al., 2009), and only uniquely mapped reads were used for further analyses. Libraries were normalized by scaling the uniquely mapped S. pombe reads to 100,000 reads. This scaling factor was then used to scale the uniquely mapped S. cerevisiae reads. To account for differences between sequencing run depth for various NextSeq runs, the pombe-scaled WT S. cerevisiae read counts were then scaled to 1 M reads, and this additional scaling factor was included to scale the sample reads. Finally, only the 5’ end of the sequencing read, which corresponds to the 3’ end of the nascent RNA was recorded and used for downstream analyses. TSS and TTS annotation was obtained from Xu et al. (2009). Read counts for genes and non-coding regions were obtained by summing normalized base pair reads over the region of interest. For average profiles, BAM files of biological replicates were merged and processed as above, and only genes longer than 500 bp were analyzed. Genes were scaled to 500 bp, and samples were scored in 1 bp bins using the deepTools program (Ramírez et al., 2016). Reads were analyzed as in Harlen et al. (2016). To calculate 5’ to 3’ ratios, the sum of reads from 1-250 bp from the TSS were divided by the sum of reads 250 bp upstream of the TTS to the TTS.RNA-seqLibrary ConstructionStrand-specific RNA-seq libraries without polyA selection were prepared similarly to Zhang et al. (2012) for 2 biological replicates. Briefly, overnight cultures from single yeast colonies were diluted to an OD600 = 0.05 in 50 mL of YPD. Cells were grown at 30°C until OD600 = 0.25. Rapamycin was added at a final concentration of 8 μg/mL and cells were grown for 3 h (OD600 = 0.7). We utilized a S. pombe spike-in to normalize the sequencing libraries. S. pombe cells were mixed with S. cerevisiae cells at a 1:6 ratio, and cells were harvested by centrifugation. RNA was purified by hot-phenol extraction and ethanol precipitation. RNA was further purified by RNeasy Miniprep kit and the DNA was digested. RiboZero magnetic beads (Illumina) were then used to remove rRNAs from 3 μg of RNA. Libraries were then prepared as in Zhang et al. (2012), and paired-end sequencing was performed on an Illumina NextSeq 500 with a read length of 75 bp.Data AnalysisFASTQ files from paired end libraries were collapsed by barcode and the Illumina adaptor sequence was trimmed from the 3’ end. Files were uploaded and analyzed using the Galaxy web platform (Afgan et al., 2018). Reads were first aligned using Bowtie2 (Langmead and Salzberg, 2012; Langmead et al., 2009) to a combined FASTA file of S. cerevisiae and S. pombe rRNA, tRNA, and RDN sequences to remove contaminating reads. Reads were then aligned to a combined version of the S. cerevisiae genome (SacCer3, SGD) and the S. pombe genome (ASM294v.2, PomBase) with TopHat2 (Kim et al., 2013) allowing up to two mismatches. The reads were separated by their respective genomes with SAMtools (Li et al., 2009), and only uniquely mapped reads were used for further analyses. For visualization in USCS genome browser, libraries were normalized by scaling the uniquely mapped S. pombe reads to 100,000 reads. To compare RNA expression between samples, HTseq 0.9.1 (Anders et al., 2015) was used to count the number of reads that aligned to each annotated gene. The annotation file for the S. cerevisiae genome was generated from the Xu et al. dataset (Xu et al., 2009), and the S. pombe annotation file was obtained from PomBase. Normalized S. cerevisiae read counts were generated by using a linear regression model to scale the S. pombe read counts relative to WT biological replicate 1 to determine scaling factors for S. cerevisiae read counts. Differential expression analysis was performed using edgeR (Robinson et al., 2010) or in excel using the R qvalue package.RT-qPCRDNase-treated RNA (1 μg) was reverse transcribed using SuperScript III (Invitrogen) and oligo-dT according to the manufacturer protocol. qPCR reactions were performed with PowerUp Syber Green Master Mix (Applied Biosystems). Primer sequences are listed in Supplemental Information. Relative mRNA levels were quantified in duplicate for biological replicates using the highly expressed PDC1 gene for normalization. Significance was determined by Student’s t test.DRIP-seqLibrary ConstructionDNA-RNA hybrid immunoprecipitation sequencing (DRIP-seq) was performed using the DNA-RNA hybrid specific S9.6 antibody similarly to Bonnet et al. (2017) for 2 biological replicates. Briefly, overnight cultures from single yeast colonies were diluted to an OD600 = 0.1 in 200 mL of YPD. Cells were grown at 30°C until OD600 ≅ 0.3. Rapamycin was added at a final concentration of 8 mg/mL and cells were grown for 3 h (OD600 ≅ 0.8). Genomic DNA was extracted by spheroplasting and EtOH precipitation, excluding a RNase-A digestion. Purified DNA (50 μg) was digested by a cocktail of restriction enzymes (50 U HindIII, EcorI, XbaI, SspI, BsrGI, NEB) in the presence or absence of 30 U of RNase-H (NEB) overnight at 37°C in a final volume of 110 μL. Digested DNA was further diluted to 500 uL with FA-lysis buffer (50 mM HEPES, pH 7.5, 140 mM NaCl, 1 mM EDTA, 1% Triton X-100, 0.1% sodium deoxycholate) and the samples were pre-cleared with Dynabeads M-280 Sheep Anti-Mouse IgG (ThermoFisher) for 3 h. DNA was then incubated overnight in the presence of 3 μg of S9.6 antibody (Millipore). Immunoprecipitated DNA fragments were captured using Dynabeads M-280 Sheep Anti-Mouse IgG (ThermoFisher), washed. and eluted according standard ChIP procedures (Bennett et al., 2013). Prior to phenol:chloroform extraction and precipiateion, the eluted DNA was incubated with 200 μg of Proteinase Kfor 1 h at 42°C while shaking at 1000 rpm to ensure complete antibody removal. Libraries were prepared by adaptor ligation and PCR amplification for paired-end sequencing on an Illumina NextSeq 500 with a read length of 75 bp.Data AnalysisFASTQ files from paired end libraries were collapsed by barcode and the Illumina adaptor sequence was trimmed from the 3’ end. Files were uploaded and analyzed using the Galaxy web platform (Afgan etal., 2018). Reads were aligned to the S. cerevisiae genome (SacCer3, SGD) using Bowtie2 with a maximum fragment length for valid paired-end alignments set to 500 bp. Aligned reads were then filtered for quality and only uniquely mapped paired reads were used for future analys3s. For visualization in USCS genome browser and metagene plots, aligned libraries were merged and normalized by scaling to 1 million reads. RNase-H control IP signal was subtracted from the matched sample. To calculate DRIP-seq signals over genes, BEDtools was used to calculate normalized bp counts across the genome for each replicate individually. Subsequent analysis was performed on median read coverage across 100-bp windows with a 50-bp sliding window. The median read counts were then summed over genes. DRIP-seq enriched genes were determined similarly to Zeller et al. (2016) by comparing DRIP-seq signals to RNase-H treated controls. The fold change in the signal versus a matched RNase-H control was calculated, and a 1.3-fold cutoff in the average of the two replicates for either WT or the hst4Δ HST3-FRB mutant was used to identify genes enriched for R-loops. Differential analysis between WT and the hst4Δ HST3-FRB mutant was performed using the R qvalue package. MACS Peak Caller was used to identify new peaks in a non-biased manner in the hst4Δ HST3-FRB biological replicates using the mean WT file as the input (p value 1×10−5, 500 bp fragment size).DRIP-qPCRCells were grown, genomic DNA was purified, and immunoprecipitated as done for DIP-seq. qPCR reactions were performed with PowerUp Syber Green Master Mix (Applied Biosystems). Primer sequences are listed in Supplemental Information. R-loop levels were determined relative to a matched RNase-H-treated sample for 3 biological replicates. Significance was determined by Student’s t test.Break-seqLibrary ConstructionBreak-seq libraries were prepared similarly to Hoffman et al. (2015) for 2 biological replicates. Briefly, overnight cultures from single yeast colonies were diluted to an OD600 = 0.1 in 100 mL of YPD. Cells were grown at 30°C until OD600 ≅ 0.32. Rapamycin was added at a final concentration of 8 mg/mLand cells were grown for 3 h (OD600 = 1.0). Cells were harvested and resuspended in 1 mL of 50 mM EDTA. Cells (50 μL) were combined with 50 μL of 1% low melting temperature agarose (Lonza). Following in gel labeling and sonication, libraries were prepared as in Hoffman et al. (2015). Paired-end sequencing was performed on an Illumina NextSeq 500 with a read length of 75 bp.Data AnalysisFASTQ files from paired end libraries were collapsed by barcode and the Illumina adaptor sequence was trimmed from the 3’ end. Files were uploaded and analyzed using the Galaxy web platform (Afgan et al., 2018). Reads were aligned to the S. cerevisiae genome (SacCer3, SGD) using Bowtie2 with a maximum fragment length for valid paired-end alignments set to 500 bp. Aligned reads were then filtered for quality and only uniquely mapped paired reads were used for future analyses. BAM files were uploaded to SeqMonk and normalized by scaling to 1 million reads. MACS Peak Caller was used to identify new peaks in hst4Δ HST3-FRB biological replicates using the mean WT file as the input (p value 1×10−5, 500 bp fragment size). Genome browser views were obtained by generating probes using a running window with a probe size of 1000 bp and a step size of 500 bp.

METHOD DETAILS

Spot Dilution Plate AssaysFor serial dilution spot plate assays, cells were cultured to saturation in 5 mL YPD or SD (-Ieu). Yeast was diluted to an OD600 of 1.0 in sterile dH2O, serially diluted 10-fold four times, and 5 μL of each dilution was spotted onto plates of indicated media. Where used, DMSO was 0.1% vol/vol, rapamycin was 8 μg/mL, methyl methanesulfonate was 0.01% or 0.005% wt/vol, hydroxyurea was 0.1 M, and camptothecin was 5 μg/mL.

Spot Dilution Plate Assays

For serial dilution spot plate assays, cells were cultured to saturation in 5 mL YPD or SD (-Ieu). Yeast was diluted to an OD600 of 1.0 in sterile dH2O, serially diluted 10-fold four times, and 5 μL of each dilution was spotted onto plates of indicated media. Where used, DMSO was 0.1% vol/vol, rapamycin was 8 μg/mL, methyl methanesulfonate was 0.01% or 0.005% wt/vol, hydroxyurea was 0.1 M, and camptothecin was 5 μg/mL.

NET-seqLibrary ConstructionNET-seq libraries were produced as described in (Churchman and Weissman, 2011) for 4 WT replicates and 3 hstΔ HST3-FRB replicates. Briefly, overnight cultures from single yeast colonies were diluted to an OD600 = 0.05 in 1 L of YPD. Cells were grown at 30°C until OD600 = 0.25. Rapamycin was added at a final concentration of 8 μg/mL and cells were grown for 3 h (OD600 = 0.7-0.8). We utilized a S. pombe spike-in, which contained a flag-tagged Rpb3 subunit (JY741), to normalize the sequencing libraries. S. pombe cells were mixed with S. cerevisiae cells at a 1:10 ratio, and the cells were harvested as described in (Churchman and Weissman 2011). RNA Pol II IP, RNA purification, and library construction was carried out as described previously (Churchman and Weissman, 2011; Mayer et al., 2015). 3’ end sequencing was performed on an Illumina NextSeq 500 with a read length of 75 bp.Data AnalysisNET-seq reads were processed and aligned as follows using the Galaxy web platform (Afgan et al., 2018). The adaptor sequence was (ATCTCGTATGCCGTCTTCTGCTTG) removed and the random hexamer sequence was removed from the 5’ end. The 3’ ends of the reads were then trimmed for quality using FASTQ Quality Timmer by sliding window (Blankenberg et al., 2010) with a window size of 10 and a step size of 5. The reads were trimmed until the aggregate score was ≤ 21. Reads were first aligned using Bowtie2 (Langmead and Salzberg, 2012; Langmeadet al., 2009) to a combined FASTA file of S. cerevisiae and S. pombe rRNA, tRNA, and RDN sequences to remove contaminating reads. Reads were then aligned to a combined version of the S. cerevisiae genome (SacCer3, SGD) and the S. pombe genome (ASM294v.2, PomBase) with TopHat2 (Kim et al., 2013), allowing up to three mismatches. The reads were separated by their respective genomes with SAMtools (Li et al., 2009), and only uniquely mapped reads were used for further analyses. Libraries were normalized by scaling the uniquely mapped S. pombe reads to 100,000 reads. This scaling factor was then used to scale the uniquely mapped S. cerevisiae reads. To account for differences between sequencing run depth for various NextSeq runs, the pombe-scaled WT S. cerevisiae read counts were then scaled to 1 M reads, and this additional scaling factor was included to scale the sample reads. Finally, only the 5’ end of the sequencing read, which corresponds to the 3’ end of the nascent RNA was recorded and used for downstream analyses. TSS and TTS annotation was obtained from Xu et al. (2009). Read counts for genes and non-coding regions were obtained by summing normalized base pair reads over the region of interest. For average profiles, BAM files of biological replicates were merged and processed as above, and only genes longer than 500 bp were analyzed. Genes were scaled to 500 bp, and samples were scored in 1 bp bins using the deepTools program (Ramírez et al., 2016). Reads were analyzed as in Harlen et al. (2016). To calculate 5’ to 3’ ratios, the sum of reads from 1-250 bp from the TSS were divided by the sum of reads 250 bp upstream of the TTS to the TTS.

Library ConstructionNET-seq libraries were produced as described in (Churchman and Weissman, 2011) for 4 WT replicates and 3 hstΔ HST3-FRB replicates. Briefly, overnight cultures from single yeast colonies were diluted to an OD600 = 0.05 in 1 L of YPD. Cells were grown at 30°C until OD600 = 0.25. Rapamycin was added at a final concentration of 8 μg/mL and cells were grown for 3 h (OD600 = 0.7-0.8). We utilized a S. pombe spike-in, which contained a flag-tagged Rpb3 subunit (JY741), to normalize the sequencing libraries. S. pombe cells were mixed with S. cerevisiae cells at a 1:10 ratio, and the cells were harvested as described in (Churchman and Weissman 2011). RNA Pol II IP, RNA purification, and library construction was carried out as described previously (Churchman and Weissman, 2011; Mayer et al., 2015). 3’ end sequencing was performed on an Illumina NextSeq 500 with a read length of 75 bp.

Library Construction

NET-seq libraries were produced as described in (Churchman and Weissman, 2011) for 4 WT replicates and 3 hstΔ HST3-FRB replicates. Briefly, overnight cultures from single yeast colonies were diluted to an OD600 = 0.05 in 1 L of YPD. Cells were grown at 30°C until OD600 = 0.25. Rapamycin was added at a final concentration of 8 μg/mL and cells were grown for 3 h (OD600 = 0.7-0.8). We utilized a S. pombe spike-in, which contained a flag-tagged Rpb3 subunit (JY741), to normalize the sequencing libraries. S. pombe cells were mixed with S. cerevisiae cells at a 1:10 ratio, and the cells were harvested as described in (Churchman and Weissman 2011). RNA Pol II IP, RNA purification, and library construction was carried out as described previously (Churchman and Weissman, 2011; Mayer et al., 2015). 3’ end sequencing was performed on an Illumina NextSeq 500 with a read length of 75 bp.

Data AnalysisNET-seq reads were processed and aligned as follows using the Galaxy web platform (Afgan et al., 2018). The adaptor sequence was (ATCTCGTATGCCGTCTTCTGCTTG) removed and the random hexamer sequence was removed from the 5’ end. The 3’ ends of the reads were then trimmed for quality using FASTQ Quality Timmer by sliding window (Blankenberg et al., 2010) with a window size of 10 and a step size of 5. The reads were trimmed until the aggregate score was ≤ 21. Reads were first aligned using Bowtie2 (Langmead and Salzberg, 2012; Langmeadet al., 2009) to a combined FASTA file of S. cerevisiae and S. pombe rRNA, tRNA, and RDN sequences to remove contaminating reads. Reads were then aligned to a combined version of the S. cerevisiae genome (SacCer3, SGD) and the S. pombe genome (ASM294v.2, PomBase) with TopHat2 (Kim et al., 2013), allowing up to three mismatches. The reads were separated by their respective genomes with SAMtools (Li et al., 2009), and only uniquely mapped reads were used for further analyses. Libraries were normalized by scaling the uniquely mapped S. pombe reads to 100,000 reads. This scaling factor was then used to scale the uniquely mapped S. cerevisiae reads. To account for differences between sequencing run depth for various NextSeq runs, the pombe-scaled WT S. cerevisiae read counts were then scaled to 1 M reads, and this additional scaling factor was included to scale the sample reads. Finally, only the 5’ end of the sequencing read, which corresponds to the 3’ end of the nascent RNA was recorded and used for downstream analyses. TSS and TTS annotation was obtained from Xu et al. (2009). Read counts for genes and non-coding regions were obtained by summing normalized base pair reads over the region of interest. For average profiles, BAM files of biological replicates were merged and processed as above, and only genes longer than 500 bp were analyzed. Genes were scaled to 500 bp, and samples were scored in 1 bp bins using the deepTools program (Ramírez et al., 2016). Reads were analyzed as in Harlen et al. (2016). To calculate 5’ to 3’ ratios, the sum of reads from 1-250 bp from the TSS were divided by the sum of reads 250 bp upstream of the TTS to the TTS.

Data Analysis

NET-seq reads were processed and aligned as follows using the Galaxy web platform (Afgan et al., 2018). The adaptor sequence was (ATCTCGTATGCCGTCTTCTGCTTG) removed and the random hexamer sequence was removed from the 5’ end. The 3’ ends of the reads were then trimmed for quality using FASTQ Quality Timmer by sliding window (Blankenberg et al., 2010) with a window size of 10 and a step size of 5. The reads were trimmed until the aggregate score was ≤ 21. Reads were first aligned using Bowtie2 (Langmead and Salzberg, 2012; Langmeadet al., 2009) to a combined FASTA file of S. cerevisiae and S. pombe rRNA, tRNA, and RDN sequences to remove contaminating reads. Reads were then aligned to a combined version of the S. cerevisiae genome (SacCer3, SGD) and the S. pombe genome (ASM294v.2, PomBase) with TopHat2 (Kim et al., 2013), allowing up to three mismatches. The reads were separated by their respective genomes with SAMtools (Li et al., 2009), and only uniquely mapped reads were used for further analyses. Libraries were normalized by scaling the uniquely mapped S. pombe reads to 100,000 reads. This scaling factor was then used to scale the uniquely mapped S. cerevisiae reads. To account for differences between sequencing run depth for various NextSeq runs, the pombe-scaled WT S. cerevisiae read counts were then scaled to 1 M reads, and this additional scaling factor was included to scale the sample reads. Finally, only the 5’ end of the sequencing read, which corresponds to the 3’ end of the nascent RNA was recorded and used for downstream analyses. TSS and TTS annotation was obtained from Xu et al. (2009). Read counts for genes and non-coding regions were obtained by summing normalized base pair reads over the region of interest. For average profiles, BAM files of biological replicates were merged and processed as above, and only genes longer than 500 bp were analyzed. Genes were scaled to 500 bp, and samples were scored in 1 bp bins using the deepTools program (Ramírez et al., 2016). Reads were analyzed as in Harlen et al. (2016). To calculate 5’ to 3’ ratios, the sum of reads from 1-250 bp from the TSS were divided by the sum of reads 250 bp upstream of the TTS to the TTS.

RNA-seqLibrary ConstructionStrand-specific RNA-seq libraries without polyA selection were prepared similarly to Zhang et al. (2012) for 2 biological replicates. Briefly, overnight cultures from single yeast colonies were diluted to an OD600 = 0.05 in 50 mL of YPD. Cells were grown at 30°C until OD600 = 0.25. Rapamycin was added at a final concentration of 8 μg/mL and cells were grown for 3 h (OD600 = 0.7). We utilized a S. pombe spike-in to normalize the sequencing libraries. S. pombe cells were mixed with S. cerevisiae cells at a 1:6 ratio, and cells were harvested by centrifugation. RNA was purified by hot-phenol extraction and ethanol precipitation. RNA was further purified by RNeasy Miniprep kit and the DNA was digested. RiboZero magnetic beads (Illumina) were then used to remove rRNAs from 3 μg of RNA. Libraries were then prepared as in Zhang et al. (2012), and paired-end sequencing was performed on an Illumina NextSeq 500 with a read length of 75 bp.Data AnalysisFASTQ files from paired end libraries were collapsed by barcode and the Illumina adaptor sequence was trimmed from the 3’ end. Files were uploaded and analyzed using the Galaxy web platform (Afgan et al., 2018). Reads were first aligned using Bowtie2 (Langmead and Salzberg, 2012; Langmead et al., 2009) to a combined FASTA file of S. cerevisiae and S. pombe rRNA, tRNA, and RDN sequences to remove contaminating reads. Reads were then aligned to a combined version of the S. cerevisiae genome (SacCer3, SGD) and the S. pombe genome (ASM294v.2, PomBase) with TopHat2 (Kim et al., 2013) allowing up to two mismatches. The reads were separated by their respective genomes with SAMtools (Li et al., 2009), and only uniquely mapped reads were used for further analyses. For visualization in USCS genome browser, libraries were normalized by scaling the uniquely mapped S. pombe reads to 100,000 reads. To compare RNA expression between samples, HTseq 0.9.1 (Anders et al., 2015) was used to count the number of reads that aligned to each annotated gene. The annotation file for the S. cerevisiae genome was generated from the Xu et al. dataset (Xu et al., 2009), and the S. pombe annotation file was obtained from PomBase. Normalized S. cerevisiae read counts were generated by using a linear regression model to scale the S. pombe read counts relative to WT biological replicate 1 to determine scaling factors for S. cerevisiae read counts. Differential expression analysis was performed using edgeR (Robinson et al., 2010) or in excel using the R qvalue package.

Library ConstructionStrand-specific RNA-seq libraries without polyA selection were prepared similarly to Zhang et al. (2012) for 2 biological replicates. Briefly, overnight cultures from single yeast colonies were diluted to an OD600 = 0.05 in 50 mL of YPD. Cells were grown at 30°C until OD600 = 0.25. Rapamycin was added at a final concentration of 8 μg/mL and cells were grown for 3 h (OD600 = 0.7). We utilized a S. pombe spike-in to normalize the sequencing libraries. S. pombe cells were mixed with S. cerevisiae cells at a 1:6 ratio, and cells were harvested by centrifugation. RNA was purified by hot-phenol extraction and ethanol precipitation. RNA was further purified by RNeasy Miniprep kit and the DNA was digested. RiboZero magnetic beads (Illumina) were then used to remove rRNAs from 3 μg of RNA. Libraries were then prepared as in Zhang et al. (2012), and paired-end sequencing was performed on an Illumina NextSeq 500 with a read length of 75 bp.

Library Construction

Strand-specific RNA-seq libraries without polyA selection were prepared similarly to Zhang et al. (2012) for 2 biological replicates. Briefly, overnight cultures from single yeast colonies were diluted to an OD600 = 0.05 in 50 mL of YPD. Cells were grown at 30°C until OD600 = 0.25. Rapamycin was added at a final concentration of 8 μg/mL and cells were grown for 3 h (OD600 = 0.7). We utilized a S. pombe spike-in to normalize the sequencing libraries. S. pombe cells were mixed with S. cerevisiae cells at a 1:6 ratio, and cells were harvested by centrifugation. RNA was purified by hot-phenol extraction and ethanol precipitation. RNA was further purified by RNeasy Miniprep kit and the DNA was digested. RiboZero magnetic beads (Illumina) were then used to remove rRNAs from 3 μg of RNA. Libraries were then prepared as in Zhang et al. (2012), and paired-end sequencing was performed on an Illumina NextSeq 500 with a read length of 75 bp.

Data AnalysisFASTQ files from paired end libraries were collapsed by barcode and the Illumina adaptor sequence was trimmed from the 3’ end. Files were uploaded and analyzed using the Galaxy web platform (Afgan et al., 2018). Reads were first aligned using Bowtie2 (Langmead and Salzberg, 2012; Langmead et al., 2009) to a combined FASTA file of S. cerevisiae and S. pombe rRNA, tRNA, and RDN sequences to remove contaminating reads. Reads were then aligned to a combined version of the S. cerevisiae genome (SacCer3, SGD) and the S. pombe genome (ASM294v.2, PomBase) with TopHat2 (Kim et al., 2013) allowing up to two mismatches. The reads were separated by their respective genomes with SAMtools (Li et al., 2009), and only uniquely mapped reads were used for further analyses. For visualization in USCS genome browser, libraries were normalized by scaling the uniquely mapped S. pombe reads to 100,000 reads. To compare RNA expression between samples, HTseq 0.9.1 (Anders et al., 2015) was used to count the number of reads that aligned to each annotated gene. The annotation file for the S. cerevisiae genome was generated from the Xu et al. dataset (Xu et al., 2009), and the S. pombe annotation file was obtained from PomBase. Normalized S. cerevisiae read counts were generated by using a linear regression model to scale the S. pombe read counts relative to WT biological replicate 1 to determine scaling factors for S. cerevisiae read counts. Differential expression analysis was performed using edgeR (Robinson et al., 2010) or in excel using the R qvalue package.

Data Analysis

FASTQ files from paired end libraries were collapsed by barcode and the Illumina adaptor sequence was trimmed from the 3’ end. Files were uploaded and analyzed using the Galaxy web platform (Afgan et al., 2018). Reads were first aligned using Bowtie2 (Langmead and Salzberg, 2012; Langmead et al., 2009) to a combined FASTA file of S. cerevisiae and S. pombe rRNA, tRNA, and RDN sequences to remove contaminating reads. Reads were then aligned to a combined version of the S. cerevisiae genome (SacCer3, SGD) and the S. pombe genome (ASM294v.2, PomBase) with TopHat2 (Kim et al., 2013) allowing up to two mismatches. The reads were separated by their respective genomes with SAMtools (Li et al., 2009), and only uniquely mapped reads were used for further analyses. For visualization in USCS genome browser, libraries were normalized by scaling the uniquely mapped S. pombe reads to 100,000 reads. To compare RNA expression between samples, HTseq 0.9.1 (Anders et al., 2015) was used to count the number of reads that aligned to each annotated gene. The annotation file for the S. cerevisiae genome was generated from the Xu et al. dataset (Xu et al., 2009), and the S. pombe annotation file was obtained from PomBase. Normalized S. cerevisiae read counts were generated by using a linear regression model to scale the S. pombe read counts relative to WT biological replicate 1 to determine scaling factors for S. cerevisiae read counts. Differential expression analysis was performed using edgeR (Robinson et al., 2010) or in excel using the R qvalue package.

RT-qPCRDNase-treated RNA (1 μg) was reverse transcribed using SuperScript III (Invitrogen) and oligo-dT according to the manufacturer protocol. qPCR reactions were performed with PowerUp Syber Green Master Mix (Applied Biosystems). Primer sequences are listed in Supplemental Information. Relative mRNA levels were quantified in duplicate for biological replicates using the highly expressed PDC1 gene for normalization. Significance was determined by Student’s t test.

DNase-treated RNA (1 μg) was reverse transcribed using SuperScript III (Invitrogen) and oligo-dT according to the manufacturer protocol. qPCR reactions were performed with PowerUp Syber Green Master Mix (Applied Biosystems). Primer sequences are listed in Supplemental Information. Relative mRNA levels were quantified in duplicate for biological replicates using the highly expressed PDC1 gene for normalization. Significance was determined by Student’s t test.

DRIP-seqLibrary ConstructionDNA-RNA hybrid immunoprecipitation sequencing (DRIP-seq) was performed using the DNA-RNA hybrid specific S9.6 antibody similarly to Bonnet et al. (2017) for 2 biological replicates. Briefly, overnight cultures from single yeast colonies were diluted to an OD600 = 0.1 in 200 mL of YPD. Cells were grown at 30°C until OD600 ≅ 0.3. Rapamycin was added at a final concentration of 8 mg/mL and cells were grown for 3 h (OD600 ≅ 0.8). Genomic DNA was extracted by spheroplasting and EtOH precipitation, excluding a RNase-A digestion. Purified DNA (50 μg) was digested by a cocktail of restriction enzymes (50 U HindIII, EcorI, XbaI, SspI, BsrGI, NEB) in the presence or absence of 30 U of RNase-H (NEB) overnight at 37°C in a final volume of 110 μL. Digested DNA was further diluted to 500 uL with FA-lysis buffer (50 mM HEPES, pH 7.5, 140 mM NaCl, 1 mM EDTA, 1% Triton X-100, 0.1% sodium deoxycholate) and the samples were pre-cleared with Dynabeads M-280 Sheep Anti-Mouse IgG (ThermoFisher) for 3 h. DNA was then incubated overnight in the presence of 3 μg of S9.6 antibody (Millipore). Immunoprecipitated DNA fragments were captured using Dynabeads M-280 Sheep Anti-Mouse IgG (ThermoFisher), washed. and eluted according standard ChIP procedures (Bennett et al., 2013). Prior to phenol:chloroform extraction and precipiateion, the eluted DNA was incubated with 200 μg of Proteinase Kfor 1 h at 42°C while shaking at 1000 rpm to ensure complete antibody removal. Libraries were prepared by adaptor ligation and PCR amplification for paired-end sequencing on an Illumina NextSeq 500 with a read length of 75 bp.Data AnalysisFASTQ files from paired end libraries were collapsed by barcode and the Illumina adaptor sequence was trimmed from the 3’ end. Files were uploaded and analyzed using the Galaxy web platform (Afgan etal., 2018). Reads were aligned to the S. cerevisiae genome (SacCer3, SGD) using Bowtie2 with a maximum fragment length for valid paired-end alignments set to 500 bp. Aligned reads were then filtered for quality and only uniquely mapped paired reads were used for future analys3s. For visualization in USCS genome browser and metagene plots, aligned libraries were merged and normalized by scaling to 1 million reads. RNase-H control IP signal was subtracted from the matched sample. To calculate DRIP-seq signals over genes, BEDtools was used to calculate normalized bp counts across the genome for each replicate individually. Subsequent analysis was performed on median read coverage across 100-bp windows with a 50-bp sliding window. The median read counts were then summed over genes. DRIP-seq enriched genes were determined similarly to Zeller et al. (2016) by comparing DRIP-seq signals to RNase-H treated controls. The fold change in the signal versus a matched RNase-H control was calculated, and a 1.3-fold cutoff in the average of the two replicates for either WT or the hst4Δ HST3-FRB mutant was used to identify genes enriched for R-loops. Differential analysis between WT and the hst4Δ HST3-FRB mutant was performed using the R qvalue package. MACS Peak Caller was used to identify new peaks in a non-biased manner in the hst4Δ HST3-FRB biological replicates using the mean WT file as the input (p value 1×10−5, 500 bp fragment size).

Library ConstructionDNA-RNA hybrid immunoprecipitation sequencing (DRIP-seq) was performed using the DNA-RNA hybrid specific S9.6 antibody similarly to Bonnet et al. (2017) for 2 biological replicates. Briefly, overnight cultures from single yeast colonies were diluted to an OD600 = 0.1 in 200 mL of YPD. Cells were grown at 30°C until OD600 ≅ 0.3. Rapamycin was added at a final concentration of 8 mg/mL and cells were grown for 3 h (OD600 ≅ 0.8). Genomic DNA was extracted by spheroplasting and EtOH precipitation, excluding a RNase-A digestion. Purified DNA (50 μg) was digested by a cocktail of restriction enzymes (50 U HindIII, EcorI, XbaI, SspI, BsrGI, NEB) in the presence or absence of 30 U of RNase-H (NEB) overnight at 37°C in a final volume of 110 μL. Digested DNA was further diluted to 500 uL with FA-lysis buffer (50 mM HEPES, pH 7.5, 140 mM NaCl, 1 mM EDTA, 1% Triton X-100, 0.1% sodium deoxycholate) and the samples were pre-cleared with Dynabeads M-280 Sheep Anti-Mouse IgG (ThermoFisher) for 3 h. DNA was then incubated overnight in the presence of 3 μg of S9.6 antibody (Millipore). Immunoprecipitated DNA fragments were captured using Dynabeads M-280 Sheep Anti-Mouse IgG (ThermoFisher), washed. and eluted according standard ChIP procedures (Bennett et al., 2013). Prior to phenol:chloroform extraction and precipiateion, the eluted DNA was incubated with 200 μg of Proteinase Kfor 1 h at 42°C while shaking at 1000 rpm to ensure complete antibody removal. Libraries were prepared by adaptor ligation and PCR amplification for paired-end sequencing on an Illumina NextSeq 500 with a read length of 75 bp.

Library Construction

DNA-RNA hybrid immunoprecipitation sequencing (DRIP-seq) was performed using the DNA-RNA hybrid specific S9.6 antibody similarly to Bonnet et al. (2017) for 2 biological replicates. Briefly, overnight cultures from single yeast colonies were diluted to an OD600 = 0.1 in 200 mL of YPD. Cells were grown at 30°C until OD600 ≅ 0.3. Rapamycin was added at a final concentration of 8 mg/mL and cells were grown for 3 h (OD600 ≅ 0.8). Genomic DNA was extracted by spheroplasting and EtOH precipitation, excluding a RNase-A digestion. Purified DNA (50 μg) was digested by a cocktail of restriction enzymes (50 U HindIII, EcorI, XbaI, SspI, BsrGI, NEB) in the presence or absence of 30 U of RNase-H (NEB) overnight at 37°C in a final volume of 110 μL. Digested DNA was further diluted to 500 uL with FA-lysis buffer (50 mM HEPES, pH 7.5, 140 mM NaCl, 1 mM EDTA, 1% Triton X-100, 0.1% sodium deoxycholate) and the samples were pre-cleared with Dynabeads M-280 Sheep Anti-Mouse IgG (ThermoFisher) for 3 h. DNA was then incubated overnight in the presence of 3 μg of S9.6 antibody (Millipore). Immunoprecipitated DNA fragments were captured using Dynabeads M-280 Sheep Anti-Mouse IgG (ThermoFisher), washed. and eluted according standard ChIP procedures (Bennett et al., 2013). Prior to phenol:chloroform extraction and precipiateion, the eluted DNA was incubated with 200 μg of Proteinase Kfor 1 h at 42°C while shaking at 1000 rpm to ensure complete antibody removal. Libraries were prepared by adaptor ligation and PCR amplification for paired-end sequencing on an Illumina NextSeq 500 with a read length of 75 bp.

Data AnalysisFASTQ files from paired end libraries were collapsed by barcode and the Illumina adaptor sequence was trimmed from the 3’ end. Files were uploaded and analyzed using the Galaxy web platform (Afgan etal., 2018). Reads were aligned to the S. cerevisiae genome (SacCer3, SGD) using Bowtie2 with a maximum fragment length for valid paired-end alignments set to 500 bp. Aligned reads were then filtered for quality and only uniquely mapped paired reads were used for future analys3s. For visualization in USCS genome browser and metagene plots, aligned libraries were merged and normalized by scaling to 1 million reads. RNase-H control IP signal was subtracted from the matched sample. To calculate DRIP-seq signals over genes, BEDtools was used to calculate normalized bp counts across the genome for each replicate individually. Subsequent analysis was performed on median read coverage across 100-bp windows with a 50-bp sliding window. The median read counts were then summed over genes. DRIP-seq enriched genes were determined similarly to Zeller et al. (2016) by comparing DRIP-seq signals to RNase-H treated controls. The fold change in the signal versus a matched RNase-H control was calculated, and a 1.3-fold cutoff in the average of the two replicates for either WT or the hst4Δ HST3-FRB mutant was used to identify genes enriched for R-loops. Differential analysis between WT and the hst4Δ HST3-FRB mutant was performed using the R qvalue package. MACS Peak Caller was used to identify new peaks in a non-biased manner in the hst4Δ HST3-FRB biological replicates using the mean WT file as the input (p value 1×10−5, 500 bp fragment size).

Data Analysis

FASTQ files from paired end libraries were collapsed by barcode and the Illumina adaptor sequence was trimmed from the 3’ end. Files were uploaded and analyzed using the Galaxy web platform (Afgan etal., 2018). Reads were aligned to the S. cerevisiae genome (SacCer3, SGD) using Bowtie2 with a maximum fragment length for valid paired-end alignments set to 500 bp. Aligned reads were then filtered for quality and only uniquely mapped paired reads were used for future analys3s. For visualization in USCS genome browser and metagene plots, aligned libraries were merged and normalized by scaling to 1 million reads. RNase-H control IP signal was subtracted from the matched sample. To calculate DRIP-seq signals over genes, BEDtools was used to calculate normalized bp counts across the genome for each replicate individually. Subsequent analysis was performed on median read coverage across 100-bp windows with a 50-bp sliding window. The median read counts were then summed over genes. DRIP-seq enriched genes were determined similarly to Zeller et al. (2016) by comparing DRIP-seq signals to RNase-H treated controls. The fold change in the signal versus a matched RNase-H control was calculated, and a 1.3-fold cutoff in the average of the two replicates for either WT or the hst4Δ HST3-FRB mutant was used to identify genes enriched for R-loops. Differential analysis between WT and the hst4Δ HST3-FRB mutant was performed using the R qvalue package. MACS Peak Caller was used to identify new peaks in a non-biased manner in the hst4Δ HST3-FRB biological replicates using the mean WT file as the input (p value 1×10−5, 500 bp fragment size).

DRIP-qPCRCells were grown, genomic DNA was purified, and immunoprecipitated as done for DIP-seq. qPCR reactions were performed with PowerUp Syber Green Master Mix (Applied Biosystems). Primer sequences are listed in Supplemental Information. R-loop levels were determined relative to a matched RNase-H-treated sample for 3 biological replicates. Significance was determined by Student’s t test.

Cells were grown, genomic DNA was purified, and immunoprecipitated as done for DIP-seq. qPCR reactions were performed with PowerUp Syber Green Master Mix (Applied Biosystems). Primer sequences are listed in Supplemental Information. R-loop levels were determined relative to a matched RNase-H-treated sample for 3 biological replicates. Significance was determined by Student’s t test.

Break-seqLibrary ConstructionBreak-seq libraries were prepared similarly to Hoffman et al. (2015) for 2 biological replicates. Briefly, overnight cultures from single yeast colonies were diluted to an OD600 = 0.1 in 100 mL of YPD. Cells were grown at 30°C until OD600 ≅ 0.32. Rapamycin was added at a final concentration of 8 mg/mLand cells were grown for 3 h (OD600 = 1.0). Cells were harvested and resuspended in 1 mL of 50 mM EDTA. Cells (50 μL) were combined with 50 μL of 1% low melting temperature agarose (Lonza). Following in gel labeling and sonication, libraries were prepared as in Hoffman et al. (2015). Paired-end sequencing was performed on an Illumina NextSeq 500 with a read length of 75 bp.Data AnalysisFASTQ files from paired end libraries were collapsed by barcode and the Illumina adaptor sequence was trimmed from the 3’ end. Files were uploaded and analyzed using the Galaxy web platform (Afgan et al., 2018). Reads were aligned to the S. cerevisiae genome (SacCer3, SGD) using Bowtie2 with a maximum fragment length for valid paired-end alignments set to 500 bp. Aligned reads were then filtered for quality and only uniquely mapped paired reads were used for future analyses. BAM files were uploaded to SeqMonk and normalized by scaling to 1 million reads. MACS Peak Caller was used to identify new peaks in hst4Δ HST3-FRB biological replicates using the mean WT file as the input (p value 1×10−5, 500 bp fragment size). Genome browser views were obtained by generating probes using a running window with a probe size of 1000 bp and a step size of 500 bp.

Library ConstructionBreak-seq libraries were prepared similarly to Hoffman et al. (2015) for 2 biological replicates. Briefly, overnight cultures from single yeast colonies were diluted to an OD600 = 0.1 in 100 mL of YPD. Cells were grown at 30°C until OD600 ≅ 0.32. Rapamycin was added at a final concentration of 8 mg/mLand cells were grown for 3 h (OD600 = 1.0). Cells were harvested and resuspended in 1 mL of 50 mM EDTA. Cells (50 μL) were combined with 50 μL of 1% low melting temperature agarose (Lonza). Following in gel labeling and sonication, libraries were prepared as in Hoffman et al. (2015). Paired-end sequencing was performed on an Illumina NextSeq 500 with a read length of 75 bp.

Library Construction

Break-seq libraries were prepared similarly to Hoffman et al. (2015) for 2 biological replicates. Briefly, overnight cultures from single yeast colonies were diluted to an OD600 = 0.1 in 100 mL of YPD. Cells were grown at 30°C until OD600 ≅ 0.32. Rapamycin was added at a final concentration of 8 mg/mLand cells were grown for 3 h (OD600 = 1.0). Cells were harvested and resuspended in 1 mL of 50 mM EDTA. Cells (50 μL) were combined with 50 μL of 1% low melting temperature agarose (Lonza). Following in gel labeling and sonication, libraries were prepared as in Hoffman et al. (2015). Paired-end sequencing was performed on an Illumina NextSeq 500 with a read length of 75 bp.

Data AnalysisFASTQ files from paired end libraries were collapsed by barcode and the Illumina adaptor sequence was trimmed from the 3’ end. Files were uploaded and analyzed using the Galaxy web platform (Afgan et al., 2018). Reads were aligned to the S. cerevisiae genome (SacCer3, SGD) using Bowtie2 with a maximum fragment length for valid paired-end alignments set to 500 bp. Aligned reads were then filtered for quality and only uniquely mapped paired reads were used for future analyses. BAM files were uploaded to SeqMonk and normalized by scaling to 1 million reads. MACS Peak Caller was used to identify new peaks in hst4Δ HST3-FRB biological replicates using the mean WT file as the input (p value 1×10−5, 500 bp fragment size). Genome browser views were obtained by generating probes using a running window with a probe size of 1000 bp and a step size of 500 bp.

Data Analysis

FASTQ files from paired end libraries were collapsed by barcode and the Illumina adaptor sequence was trimmed from the 3’ end. Files were uploaded and analyzed using the Galaxy web platform (Afgan et al., 2018). Reads were aligned to the S. cerevisiae genome (SacCer3, SGD) using Bowtie2 with a maximum fragment length for valid paired-end alignments set to 500 bp. Aligned reads were then filtered for quality and only uniquely mapped paired reads were used for future analyses. BAM files were uploaded to SeqMonk and normalized by scaling to 1 million reads. MACS Peak Caller was used to identify new peaks in hst4Δ HST3-FRB biological replicates using the mean WT file as the input (p value 1×10−5, 500 bp fragment size). Genome browser views were obtained by generating probes using a running window with a probe size of 1000 bp and a step size of 500 bp.

QUANTIFICATION AND STATISTICAL ANALYSISSoftware and statistical analysis details can be found in the Method Details section of the STAR Methods, as well as the Key Resources Table.

QUANTIFICATION AND STATISTICAL ANALYSIS

Software and statistical analysis details can be found in the Method Details section of the STAR Methods, as well as the Key Resources Table.

DATA AND SOFTWARE AVAILABILITYAll data from this study have been deposited in the Gene Expression Omnibus (GEO) under accession GEO: GSE124132.

DATA AND SOFTWARE AVAILABILITY

All data from this study have been deposited in the Gene Expression Omnibus (GEO) under accession GEO: GSE124132.

Supplementary Material123

Supplementary Material
