---
pmid: '31262713'
title: FAM129B, an antioxidative protein, reduces chemosensitivity by competing with
  Nrf2 for Keap1 binding.
authors:
- Cheng KC
- Lin RJ
- Cheng JY
- Wang SH
- Yu JC
- Wu JC
- Liang YJ
- Hsu HM
- Yu J
- Yu AL
journal: EBioMedicine
year: '2019'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC6642435
doi: 10.1016/j.ebiom.2019.06.022
---

# FAM129B, an antioxidative protein, reduces chemosensitivity by competing with Nrf2 for Keap1 binding.
**Authors:** Cheng KC, Lin RJ, Cheng JY, Wang SH, Yu JC, Wu JC, Liang YJ, Hsu HM, Yu J, Yu AL
**Journal:** EBioMedicine (2019)
**DOI:** [10.1016/j.ebiom.2019.06.022](https://doi.org/10.1016/j.ebiom.2019.06.022)
**PMC:** [PMC6642435](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6642435/)

## Abstract

1. EBioMedicine. 2019 Jul;45:25-38. doi: 10.1016/j.ebiom.2019.06.022. Epub 2019
Jun  28.

FAM129B, an antioxidative protein, reduces chemosensitivity by competing with 
Nrf2 for Keap1 binding.

Cheng KC(1), Lin RJ(2), Cheng JY(2), Wang SH(2), Yu JC(3), Wu JC(2), Liang 
YJ(4), Hsu HM(3), Yu J(2), Yu AL(5).

Author information:
(1)Institute of Stem Cell & Translational Cancer Research, Chang Gung Memorial 
Hospital at Linkou, Chang Gung University, Taoyuan, Taiwan; Institute of 
Biochemistry and Molecular Biology, National Yang-Ming University, Taipei, 
Taiwan.
(2)Institute of Stem Cell & Translational Cancer Research, Chang Gung Memorial 
Hospital at Linkou, Chang Gung University, Taoyuan, Taiwan.
(3)General Surgery, Department of Surgery, Tri-Service General Hospital, 
National Defense Medical Center, Taipei, Taiwan.
(4)Translational Research Division, Medical Research Department, Taipei Veterans 
General Hospital, Taipei, Taiwan.
(5)Institute of Stem Cell & Translational Cancer Research, Chang Gung Memorial 
Hospital at Linkou, Chang Gung University, Taoyuan, Taiwan; Department of 
Pediatrics, University of California in San Diego, San Diego, CA, USA. 
Electronic address: a1yu@ucsd.edu.

BACKGROUND: The transcription factor Nrf2 is a master regulator of antioxidant 
response. While Nrf2 activation may counter increasing oxidative stress in 
aging, its activation in cancer can promote cancer progression and metastasis, 
and confer resistance to chemotherapy and radiotherapy. Thus, Nrf2 has been 
considered as a key pharmacological target. Unfortunately, there are no specific 
Nrf2 inhibitors for therapeutic application. Moreover, high Nrf2 activity in 
many tumors without Keap1 or Nrf2 mutations suggests that alternative mechanisms 
of Nrf2 regulation exist.
METHODS: Interaction of FAM129B with Keap1 is demonstrated by 
immunofluorescence, colocalization, co-immunoprecipitation and mammalian 
two-hybrid assay. Antioxidative function of FAM129B is analyzed by measuring ROS 
levels with DCF/flow cytometry, Nrf2 activation using luciferase reporter assay 
and determination of downstream gene expression by qPCR and wester blotting. 
Impact of FAM129B on in vivo chemosensitivity is examined in mice bearing breast 
and colon cancer xenografts. The clinical relevance of FAM129B is assessed by 
qPCR in breast cancer samples and data mining of publicly available databases.
FINDINGS: We have demonstrated that FAM129B in cancer promotes Nrf2 activity by 
reducing its ubiquitination through competition with Nrf2 for Keap1 binding via 
its DLG and ETGE motifs. In addition, FAM129B reduces chemosensitivity by 
augmenting Nrf2 antioxidative signaling and confers poor prognosis in breast and 
lung cancer.
INTERPRETATION: These findings demonstrate the important role of FAM129B in Nrf2 
activation and antioxidative response, and identify FMA129B as a potential 
therapeutic target. FUND: The Chang Gung Medical Foundation (Taiwan) and the 
Ministry of Science and Technology (Taiwan).

Copyright © 2019 The Authors. Published by Elsevier B.V. All rights reserved.

DOI: 10.1016/j.ebiom.2019.06.022
PMCID: PMC6642435
PMID: 31262713 [Indexed for MEDLINE]

## Full Text

Research in contextEvidence before this studyNrf2-Keap1 system is a key cellular defense mechanism against oxidative stress. Somatic mutations of Nrf2 or Keap1 contributing to Nrf2 hyperactivation have been reported in many cancers. However, few studies address the mechanisms of Nrf2 activation without such genetic mutations. Thus, it is imperative to explore alternative regulatory mechanisms that govern the Nrf2 activation, which may offer novel strategies for cancer treatment.Added value of this studyFAM129B competes with Nrf2 for binding to Kelch domain of Keap1 via its DLG and ETGE motifs. FAM129B can stabilize Nrf2 to drive downstream antioxidant genes, confer resistance to oxidant injury and chemotherapeutics. Clinically, higher expression of FAM129B correlates with poorer outcome in cancer by data mining. Examination of breast cancer samples shows high FAM129B expression to be an independent predictor of tumor recurrence.Implications of all available evidenceThis study shows that expression of FAM129B in cancer promotes Nrf2 activity by reducing its degradation through binding to Keap1. It provides an alternative mechanism that regulates the Nrf2 activation and anti-oxidative response. These findings elucidate the mechanistic underpinnings on how FAM129B reduces chemosensitivity, and identify FAM129B as a new antioxidant molecule, a potential cancer therapeutic target and a poor prognosis factor for cancer.Alt-text: Unlabelled Box

Research in context

Evidence before this studyNrf2-Keap1 system is a key cellular defense mechanism against oxidative stress. Somatic mutations of Nrf2 or Keap1 contributing to Nrf2 hyperactivation have been reported in many cancers. However, few studies address the mechanisms of Nrf2 activation without such genetic mutations. Thus, it is imperative to explore alternative regulatory mechanisms that govern the Nrf2 activation, which may offer novel strategies for cancer treatment.

Evidence before this study

Nrf2-Keap1 system is a key cellular defense mechanism against oxidative stress. Somatic mutations of Nrf2 or Keap1 contributing to Nrf2 hyperactivation have been reported in many cancers. However, few studies address the mechanisms of Nrf2 activation without such genetic mutations. Thus, it is imperative to explore alternative regulatory mechanisms that govern the Nrf2 activation, which may offer novel strategies for cancer treatment.

Added value of this studyFAM129B competes with Nrf2 for binding to Kelch domain of Keap1 via its DLG and ETGE motifs. FAM129B can stabilize Nrf2 to drive downstream antioxidant genes, confer resistance to oxidant injury and chemotherapeutics. Clinically, higher expression of FAM129B correlates with poorer outcome in cancer by data mining. Examination of breast cancer samples shows high FAM129B expression to be an independent predictor of tumor recurrence.

Added value of this study

FAM129B competes with Nrf2 for binding to Kelch domain of Keap1 via its DLG and ETGE motifs. FAM129B can stabilize Nrf2 to drive downstream antioxidant genes, confer resistance to oxidant injury and chemotherapeutics. Clinically, higher expression of FAM129B correlates with poorer outcome in cancer by data mining. Examination of breast cancer samples shows high FAM129B expression to be an independent predictor of tumor recurrence.

Implications of all available evidenceThis study shows that expression of FAM129B in cancer promotes Nrf2 activity by reducing its degradation through binding to Keap1. It provides an alternative mechanism that regulates the Nrf2 activation and anti-oxidative response. These findings elucidate the mechanistic underpinnings on how FAM129B reduces chemosensitivity, and identify FAM129B as a new antioxidant molecule, a potential cancer therapeutic target and a poor prognosis factor for cancer.

Implications of all available evidence

This study shows that expression of FAM129B in cancer promotes Nrf2 activity by reducing its degradation through binding to Keap1. It provides an alternative mechanism that regulates the Nrf2 activation and anti-oxidative response. These findings elucidate the mechanistic underpinnings on how FAM129B reduces chemosensitivity, and identify FAM129B as a new antioxidant molecule, a potential cancer therapeutic target and a poor prognosis factor for cancer.

Alt-text: Unlabelled Box

1IntroductionThe Nrf2 (Nuclear factor-erythroid 2-related factor 2)-Keap1 (Kelch-like ECH-associated protein 1) system is a key cellular defense mechanism against oxidative stress. The key function of the transcription factor Nrf2 is to govern the cellular antioxidant response by transcriptionally activate several cytoprotective genes to protect cell from the effect of oxidative stress [1]. Under basal conditions, Nrf2 is constitutively degraded through the ubiquitin-proteasome pathway via interaction with Keap1, a substrate scaffold for Cul3-containing E3 ubiquitin ligase [2,3]. Under conditions of oxidative stress, Nrf2-Keap1 interaction is disrupted by modification of cysteine residues of Keap1, such as C151, C273, or C288, causing a conformational change that may affect its ideal binding with Cul3 or Nrf2 and resulting in diminished Nrf2 ubiquitination [4]. The consequent stabilization of Nrf2 allows its translocates to the nucleus, where it induces the transcription of numerous genes involved in cell defense, including antioxidants, drug-metabolizing enzymes, and drug efflux transporters by binding to the antioxidant response elements (AREs) in their regulatory regions [5,6].The activation of Nrf2 helps normal cells to endure oxidative stresses and maintain the redox homeostasis. Under physiological conditions, Nrf2 signaling is turned on by the presence of stressors and is rapidly deactivated when the insult subsides. However, under pathological conditions, the tight regulation of Nrf2 may be lost, resulting in constitutive activation of Nrf2 which confers a survival advantage to cancer cells under adverse conditions [7]. Several studies have indicated that aberrant activation of Nrf2 is beneficial to cancer cells because Nrf2 downstream genes play crucial roles in cell survival, and tumor promotion [[8], [9], [10]]. Recent cancer genomic studies have revealed somatic mutations of Nrf2 or Keap1 that disrupt Keap1-Nrf2 interaction in many cancers [[11], [12], [13]], and are associated with resistance to chemotherapies. In addition to mutation, hypermethylation of the Keap1 promoter and amplifications of Nrf2 copy number can also promote Nrf2 activity in cancer [14,15]. These findings indicate that Nrf2 hyperactivation protects cancer cells from excessive oxidative stress, chemotherapeutic agents, or radiotherapy and therby promote their survival [7]. Interestingly, Nrf2 is hyperactivated in many tumors lacking genetic alterations in Nrf2 or Keap1, implying that alternative mechanisms of Nrf2 regulation exist. In this study, we have identified FAM129B (Family with sequence similarity 129, member B) as an alternative regulator for Nrf2 activation, and may serve as a novel target for cancer treatment.FAM129B, also known as Niban-like protein 1 or MINERVA, belongs to a family of Niban proteins that includes FAM129A (also known as Niban) and FAM129C (also known as Niban-like protein 2 or B-cell novel protein 1). Although three members of this protein family share some amino acid sequence identity, little is known about the function or regulation of these proteins in any organism [16]. Interesting, FAM129B is up-regulated in several type of human cancer, such as breast, lung, colon, renal and endometrial cancers as well as hematopoietic and central nervous system tumors [17,18]. The protein structure of FAM129B contains a pleckstrin homology (pH) domain near the amino-terminus and a proline-rich domain near the carboxyl-terminus, which has six serine residues that can be phosphorylated [16]. Recent studies indicate that phosphorylation of FAM129B by EGFR at the Y593 residue mediates Ras activation and that FAM129B phosphorylation by ERK1/2 promotes tumor cell invasion [16,19]. In addition, FAM129B knockdown not only inhibits Wnt3A/β-catenin signaling-mediated expression of AXIN2 but also accelerates TNFα-induced apoptosis, which can be reversed by co-transfection with recombinant FAM129B [20,21]. However, the detailed molecular mechanisms linking these findings on FAM129B are largely unknown. In this report, we present molecular evidence supporting the notion that FAM129B-dependent cell survival under oxidative stress is mediated through stabilization of Nrf2 protein by a mechanism involving competition between FAM129B and Nrf2 for binding to Keap1. Importantly, we also demonstrate that the expression of FAM129B correlates with adverse clinical outcome.

Introduction

The Nrf2 (Nuclear factor-erythroid 2-related factor 2)-Keap1 (Kelch-like ECH-associated protein 1) system is a key cellular defense mechanism against oxidative stress. The key function of the transcription factor Nrf2 is to govern the cellular antioxidant response by transcriptionally activate several cytoprotective genes to protect cell from the effect of oxidative stress [1]. Under basal conditions, Nrf2 is constitutively degraded through the ubiquitin-proteasome pathway via interaction with Keap1, a substrate scaffold for Cul3-containing E3 ubiquitin ligase [2,3]. Under conditions of oxidative stress, Nrf2-Keap1 interaction is disrupted by modification of cysteine residues of Keap1, such as C151, C273, or C288, causing a conformational change that may affect its ideal binding with Cul3 or Nrf2 and resulting in diminished Nrf2 ubiquitination [4]. The consequent stabilization of Nrf2 allows its translocates to the nucleus, where it induces the transcription of numerous genes involved in cell defense, including antioxidants, drug-metabolizing enzymes, and drug efflux transporters by binding to the antioxidant response elements (AREs) in their regulatory regions [5,6].

The activation of Nrf2 helps normal cells to endure oxidative stresses and maintain the redox homeostasis. Under physiological conditions, Nrf2 signaling is turned on by the presence of stressors and is rapidly deactivated when the insult subsides. However, under pathological conditions, the tight regulation of Nrf2 may be lost, resulting in constitutive activation of Nrf2 which confers a survival advantage to cancer cells under adverse conditions [7]. Several studies have indicated that aberrant activation of Nrf2 is beneficial to cancer cells because Nrf2 downstream genes play crucial roles in cell survival, and tumor promotion [[8], [9], [10]]. Recent cancer genomic studies have revealed somatic mutations of Nrf2 or Keap1 that disrupt Keap1-Nrf2 interaction in many cancers [[11], [12], [13]], and are associated with resistance to chemotherapies. In addition to mutation, hypermethylation of the Keap1 promoter and amplifications of Nrf2 copy number can also promote Nrf2 activity in cancer [14,15]. These findings indicate that Nrf2 hyperactivation protects cancer cells from excessive oxidative stress, chemotherapeutic agents, or radiotherapy and therby promote their survival [7]. Interestingly, Nrf2 is hyperactivated in many tumors lacking genetic alterations in Nrf2 or Keap1, implying that alternative mechanisms of Nrf2 regulation exist. In this study, we have identified FAM129B (Family with sequence similarity 129, member B) as an alternative regulator for Nrf2 activation, and may serve as a novel target for cancer treatment.

FAM129B, also known as Niban-like protein 1 or MINERVA, belongs to a family of Niban proteins that includes FAM129A (also known as Niban) and FAM129C (also known as Niban-like protein 2 or B-cell novel protein 1). Although three members of this protein family share some amino acid sequence identity, little is known about the function or regulation of these proteins in any organism [16]. Interesting, FAM129B is up-regulated in several type of human cancer, such as breast, lung, colon, renal and endometrial cancers as well as hematopoietic and central nervous system tumors [17,18]. The protein structure of FAM129B contains a pleckstrin homology (pH) domain near the amino-terminus and a proline-rich domain near the carboxyl-terminus, which has six serine residues that can be phosphorylated [16]. Recent studies indicate that phosphorylation of FAM129B by EGFR at the Y593 residue mediates Ras activation and that FAM129B phosphorylation by ERK1/2 promotes tumor cell invasion [16,19]. In addition, FAM129B knockdown not only inhibits Wnt3A/β-catenin signaling-mediated expression of AXIN2 but also accelerates TNFα-induced apoptosis, which can be reversed by co-transfection with recombinant FAM129B [20,21]. However, the detailed molecular mechanisms linking these findings on FAM129B are largely unknown. In this report, we present molecular evidence supporting the notion that FAM129B-dependent cell survival under oxidative stress is mediated through stabilization of Nrf2 protein by a mechanism involving competition between FAM129B and Nrf2 for binding to Keap1. Importantly, we also demonstrate that the expression of FAM129B correlates with adverse clinical outcome.

2Materials and methods2.1Cell culture and chemicalsMDA-MB-231, Hs578T, and HCT116 were obtained from the Bioresource Collection and Research Center (BCRC, Taiwan). H1299 and 293T cells were obtained from the ATCC. Cells were grown in Dulbecco's modified Eagle's medium (DMEM) containing 10% FBS. All cells were cultured at 37 °C with 5% CO2. Cell lines were routinely tested to exclude mycoplasma contamination.2.2In silico prediction of binding of human Keap1 protein bind to FAM129B through both DLG and ETGE motifsIn order to model the complex structures comprising the Kelch domain of human Keap1 and the peptide segments of FAM129B, the complex models which were searched from Protein Data Bank (http://www.rcsb.org) reveal the molecular interactions between the mouse Keap1 and Nrf2 peptide segments containing DLG or ETGE motifs (PDB: 3WN7 and 1X2R, respectively) were provided as structural templates for homology modeling. The sequence alignments among human Kelch domain and its mouse homologs were performed using BLAST [22]. The 3D structure of human Kelch domain and the complex structures binding with FAM129B segments were simulated using Modeller v9.12 [23], with the functions of the AUTOMODEL class in python scripts. After energy optimization and refinement, the Discrete Optimized Protein Energy (DOPE) method [23] was used to select the best model from the initially-generated models. The molecular interactions between Kelch domain and FAM129B peptide were analyzed and scored by the HotLig [24]. The high-quality images of protein-peptide models were rendered using UCSF Chimera [25]. The 2D schematic diagrams for illustrating molecular interactions were generated using Ligplot [26].2.3In vivo studies in xenograft mouse modelsMale severe combined immune-deficient (NOD/SCID) mice between 4 and 5 weeks of age were purchased from the National Laboratory Animal Center (NLAC), NARLabs. FAM129B expression in HCT116 was suppressed by transfection with the pooled three siRNAs targeting FAM129B (HCT116/si-FAM129B); cells transfected with non-specific control oligos served as a control (HCT116/si-Control). The si-FAM129B and si-Control HCT116 cells were inoculated subcutaneously 1.5 × 106 cells) into the flank of the NOD/SCID mice. When the tumor sizes reached about 100 mm3, the mice were randomly divided into two groups and subjected to treatment with PBS or oxaliplatin (7.5 mg/kg, intraperitoneally, every 4 days for 3 weeks) (n = 5 for each group). The tumor volumes, calculated as length × width2 × 0.5, were determined twice a week. At day 20 after injection of tumor cells, the tumors were carefully removed, photographed and weighed.Female severe combined immune-deficient (NSG) mice between 4 and 5 weeks of age were purchased from The Jackson Laboratory. FAM129B expression in MDA-MB-231 cells was suppressed by transfection with the pooled three siRNAs targeting FAM129B (MB-231/si-FAM129B); cells transfected with non-specific control oligos served as a control (MB-231/si-Control). The si-FAM129B and si-Control MB-231 cells (1.5 × 106 cells) were injected into mammary fat pad of the NSG mice. After 2 days, the mice were randomly divided into two groups and subjected to treatment with PBS or oxaliplatin (20 mg/kg, intraperitoneally, once) (n = 4 for each group). The tumor volumes, calculated as length × width2 × 0.5, were determined twice a week. At day 35 after injection of tumor cells, the tumors were carefully removed, photographed and weighed. All procedures were performed in compliance with the regulations of the Institutional Animal Care and Use Committee of Chang-Gung University.2.4Breast cancer patient samplesOne hundred and twenty-six fresh primary breast cancer tumor and adjacent normal tissue specimens were collected during surgical resections performed at the Tri-Service General Hospital (Taipei, Taiwan). Informed consent was obtained from all subjects before their tissues were deposited. The samples were fully encoded and used under a protocol approved by the Institutional Review Board of the Human Subjects Research Ethics Committee of the Tri-Service General Hospital and Chang Gung Memorial Hospital (Taoyuan, Taiwan). The clinicopathologic information is listed in Supplemental Table S1.2.5Small interfering RNAs (siRNAs) and plasmid transfectionsiRNAs specifically targeting FAM129B and non-specific si-Control RNAs were synthesized by MDBio. The sequences are shown in the Key Resources Table. Transfection with the pooled three FAM129B siRNAs listed in the table was performed with RNAiMAX (ThermoFisher) according to the manufacturer's instructions.Full length Keap1 and the five truncated Keap1 mutants shown in the Key Resources Table were cloned into pACT vectors. Full length FAM129B, Nrf2, and Keap1 were cloned into pBIND, pcDNA3.1 and pFLAG-CMV2 vectors, respectively, using appropriate restriction enzyme digests. The numbered (or indicated) amino acids in DLG708 and E718T719GE in wild-type FAM129B were replaced with alanine, to generate DLA708- and A718A719GE-mutant FAM129B, respectively, using the QuikChange Lightning Site-Directed Mutagenesis Kit (Agilent) with full length FAM129B-expressing plasmids as templates. Transfections of plasmid DNA were performed with TransIT-2020 reagent (Mirus) according to the manufacturer's instructions.2.6ROS detectionCells were treated with 10 μM 2′,7′-dichlorofluorescein diacetate (DCF-DA) for 30 min, following 2 washes with PBS. Reduced DCF-DA can be oxidized and converted to fluorescent 2′,7′-dichlorofluorescein (DCF) by intracellular ROS. Fluorescence signals were detected by flow cytometry (Sony EC800 Analyzer). Totally, 10,000 cells were analyzed per sample.2.7Western blot (WB) assayCells were washed with PBS and lysed in RIPA buffer containing NP-40 and protease inhibitors (Roche). Total protein extracts (10–30 μg) were separated on 4–12% NuPAGE (Invitrogen), and transferred to PVDF membranes (Millipore, Bedford, MA, USA). The membranes were incubated with primary antibodies at 4 °C overnight, followed by alkaline phosphatase-conjugated secondary antibodies (Jackson Immunoresearch Laboratories) for 1 h at room temperature. Then, the membrane was scanned with a Typhoon9400 Variable Mode Imager (GE Healthcare Life Sciences) to detect the fluorescent signals released from catalyzed ECF substrate (GE Healthcare Life Sciences). The details for the antibodies used in this study are provided in the Key Resources Table. The results of western blots were quantified using ImageQuant 5.2 software (GE Healthcare Life Sciences).2.8Real-time PCR (RT-PCR)Total RNA of breast cancer tissue was isolated using TRIzol reagent (Invitrogen), and cDNA was generated from 1000 ng of total RNA, using a High Capacity cDNA Reverse Transcription Kit (Applied BioSystems/ABI). RT-PCR assays were performed on a QuantStudio™ 7 Flex Real-Time PCR system (Applied Biosystems) using SYBR Green MasterMix (ABI). For accurate normalization of quantitative data, multiple housekeeping genes, including GAPDH, GUSB (glucuronidase-beta) and UBC (polyubiquitin) were assayed. The primer sequences used in RT-PCR assays are listed in the Key Resources Table.2.9ImmunoprecipitationCells were washed with PBS and lysed in RIPA buffer containing NP-40 and protease inhibitors (Roche). The lysate was cleared by centrifugation at 13,000 Rcf for 30 min at 4 °C. Immunoprecipitation was performed by incubation of cell lysates with anti-Flag or anti-HA antibody, and capture on Dynabeads® Protein G (10003D, Thermo Fisher Scientific) for 4 h at 4 °C in a rocking incubator. After six washes with washing buffer, immunoprecipitated complexes were eluted in sample buffer by boiling in water for 4 min, electrophoresed through 4–12% NuPAGE (Invitrogen) gel, and subjected to immunoblot analysis.2.10Determination of cell viability by MTS assayCell viabilities were assessed by adding MTS reagent,3-(4,5-Dimethylthiazol-2-yl)-5-(3-carboxymethoxyphenyl)-2-(4-sulfophenyl)-2H-tetrazolium (Biovision), to the cells. Optical density was measured on a SpectraMax M2 microplate reader (Molecular Devices) at 490 nm. Each experiment was conducted in triplicate and repeated independently three times.2.11Luciferase reporter assaySix copies of the antioxidant response element (ARE) (5′-GTGACAAAGCACCCGTGACAAAGCACCCGTGACAAAGCACCCGTGACAAAGCACCCGTGACAAAGCACCCGTGACAAAGCA-3′) were cloned into the pGL3-basic luciferase reporter plasmid [27]. The indicated cells were transfected with ARE-luciferase reporter and Renilla luciferase together with either FAM129B plasmids or si-FAM129B. At 48–72 h after transfection, the luciferase activities of cell lysates were measured with the Dual Luciferase assay system (Promega).2.12GSH/GSSG detectionThe ratio of reduced to oxidized glutathione (GSH/GSSG) in cells was measured using the GSH/GSSG-Glo™ Assay according to the manufacturer's instructions (Promega, V6611).2.13FLIM-FRET measurementFLIM-FRET measurements were made as previously described [28], with modifications. A Leica TCS SP5 equipped with multiphoton fluorescence lifetime imaging microscopy (Leica TCS-SP5-AOBS-MP) system was used for confocal imaging and to measure fluorescence lifetime. A water immersion objective (Leica, 63×/0.9 APO) was employed both for focusing laser light onto the samples and for collecting fluorescence emissions from the samples. The fluorescence lifetime for each image pixel was recorded using the time-correlated single photon counting technique (Becker & Hickl SPEC-830 TCSPC modules).2.14In situ proximity ligation assay (PLA)Interaction between Keap1 and FAM129Bwere assessed using an in situ PLA kit (Duolink) according to the manufacturer's instructions. In brief, cells (2 × 104) grown on 8-well slides (ibidi) were fixed in 4% paraformaldehyde in PBS for 15 min at room temperature, blocked with Duolink blocking solution for 30 min at 37 °C, washed with PBS, incubated overnight at 4 °C with primary antibodies (Rabbit anti-FAM129B and mouse anti-Keap1, as described above), washed with PBS, incubated for 1 h at 37 °C with secondary antibody (anti-mouse PLA-plus probe or anti-rabbit PLA-minus probe; Duolink; dilution 1:50), washed twice (5 min each time) with Duolink Wash buffer A, combined with Duolink ligation mixture, incubated for 30 min at 37 °C, washed twice with Wash buffer A, combined with Duolink amplification mixture and Polymerase, subjected to amplification reaction for 100 min at 37 °C, washed twice with Wash buffer B and once with 0.1× Wash buffer B, and mounted with Duolink In Situ Mounting Medium with DAPI [29]. Fluorescence dot images were obtained by confocal immunofluorescence microscopy as above.2.15Mammalian Two-Hybrid SystemInteraction between Keap1 and FAM129B were assessed using a CheckMate™ Mammalian Two-Hybrid System (Promega, E2440) according to the manufacturer's instructions.2.16Statistical analysesThe prognostic performance of genes was calculated as mean values. Relapse-free survival (RFS) values were estimated by the Kaplan–Meier method and were compared by the log-rank test. The Cox regression model was used for analysis of factors potentially related to RFS. The statistical analyses were performed with GraphPad Prism software (version 5.0, GraphPad Software) and MedCalc statistical software (version 14.8, MedCalc Software). Data are presented as means ±standard deviation (SD). Student's t-test was applied to assess the statistical significance. p-Values <.05 were considered significant.2.17Ubiquitination assayCells were exposed to 10 μM MG132 (Sigma) for 4 h. Cells were lysed by boiling in a buffer containing 2% SDS, 150 mM NaCl, 10 mM Tris-HCl with 2 mM sodium orthovanadate, 50 mM sodium fluoride, and protease inhibitors. The lysates were incubated with an Nrf2 antibody and subjected to immunoblot analysis.2.18Protein half-life measurementFifty micromolar cycloheximide (Sigma) was added in MDA-MB-231 cells. Total cell lysates were collected at different time points and subjected to immunoblot analysis with an anti-Nrf2 antibody. The relative intensities of the bands were quantified by using the ImageQuant 5.2 software.

Materials and methods

2.1Cell culture and chemicalsMDA-MB-231, Hs578T, and HCT116 were obtained from the Bioresource Collection and Research Center (BCRC, Taiwan). H1299 and 293T cells were obtained from the ATCC. Cells were grown in Dulbecco's modified Eagle's medium (DMEM) containing 10% FBS. All cells were cultured at 37 °C with 5% CO2. Cell lines were routinely tested to exclude mycoplasma contamination.

Cell culture and chemicals

MDA-MB-231, Hs578T, and HCT116 were obtained from the Bioresource Collection and Research Center (BCRC, Taiwan). H1299 and 293T cells were obtained from the ATCC. Cells were grown in Dulbecco's modified Eagle's medium (DMEM) containing 10% FBS. All cells were cultured at 37 °C with 5% CO2. Cell lines were routinely tested to exclude mycoplasma contamination.

2.2In silico prediction of binding of human Keap1 protein bind to FAM129B through both DLG and ETGE motifsIn order to model the complex structures comprising the Kelch domain of human Keap1 and the peptide segments of FAM129B, the complex models which were searched from Protein Data Bank (http://www.rcsb.org) reveal the molecular interactions between the mouse Keap1 and Nrf2 peptide segments containing DLG or ETGE motifs (PDB: 3WN7 and 1X2R, respectively) were provided as structural templates for homology modeling. The sequence alignments among human Kelch domain and its mouse homologs were performed using BLAST [22]. The 3D structure of human Kelch domain and the complex structures binding with FAM129B segments were simulated using Modeller v9.12 [23], with the functions of the AUTOMODEL class in python scripts. After energy optimization and refinement, the Discrete Optimized Protein Energy (DOPE) method [23] was used to select the best model from the initially-generated models. The molecular interactions between Kelch domain and FAM129B peptide were analyzed and scored by the HotLig [24]. The high-quality images of protein-peptide models were rendered using UCSF Chimera [25]. The 2D schematic diagrams for illustrating molecular interactions were generated using Ligplot [26].

In silico prediction of binding of human Keap1 protein bind to FAM129B through both DLG and ETGE motifs

In order to model the complex structures comprising the Kelch domain of human Keap1 and the peptide segments of FAM129B, the complex models which were searched from Protein Data Bank (http://www.rcsb.org) reveal the molecular interactions between the mouse Keap1 and Nrf2 peptide segments containing DLG or ETGE motifs (PDB: 3WN7 and 1X2R, respectively) were provided as structural templates for homology modeling. The sequence alignments among human Kelch domain and its mouse homologs were performed using BLAST [22]. The 3D structure of human Kelch domain and the complex structures binding with FAM129B segments were simulated using Modeller v9.12 [23], with the functions of the AUTOMODEL class in python scripts. After energy optimization and refinement, the Discrete Optimized Protein Energy (DOPE) method [23] was used to select the best model from the initially-generated models. The molecular interactions between Kelch domain and FAM129B peptide were analyzed and scored by the HotLig [24]. The high-quality images of protein-peptide models were rendered using UCSF Chimera [25]. The 2D schematic diagrams for illustrating molecular interactions were generated using Ligplot [26].

2.3In vivo studies in xenograft mouse modelsMale severe combined immune-deficient (NOD/SCID) mice between 4 and 5 weeks of age were purchased from the National Laboratory Animal Center (NLAC), NARLabs. FAM129B expression in HCT116 was suppressed by transfection with the pooled three siRNAs targeting FAM129B (HCT116/si-FAM129B); cells transfected with non-specific control oligos served as a control (HCT116/si-Control). The si-FAM129B and si-Control HCT116 cells were inoculated subcutaneously 1.5 × 106 cells) into the flank of the NOD/SCID mice. When the tumor sizes reached about 100 mm3, the mice were randomly divided into two groups and subjected to treatment with PBS or oxaliplatin (7.5 mg/kg, intraperitoneally, every 4 days for 3 weeks) (n = 5 for each group). The tumor volumes, calculated as length × width2 × 0.5, were determined twice a week. At day 20 after injection of tumor cells, the tumors were carefully removed, photographed and weighed.Female severe combined immune-deficient (NSG) mice between 4 and 5 weeks of age were purchased from The Jackson Laboratory. FAM129B expression in MDA-MB-231 cells was suppressed by transfection with the pooled three siRNAs targeting FAM129B (MB-231/si-FAM129B); cells transfected with non-specific control oligos served as a control (MB-231/si-Control). The si-FAM129B and si-Control MB-231 cells (1.5 × 106 cells) were injected into mammary fat pad of the NSG mice. After 2 days, the mice were randomly divided into two groups and subjected to treatment with PBS or oxaliplatin (20 mg/kg, intraperitoneally, once) (n = 4 for each group). The tumor volumes, calculated as length × width2 × 0.5, were determined twice a week. At day 35 after injection of tumor cells, the tumors were carefully removed, photographed and weighed. All procedures were performed in compliance with the regulations of the Institutional Animal Care and Use Committee of Chang-Gung University.

In vivo studies in xenograft mouse models

Male severe combined immune-deficient (NOD/SCID) mice between 4 and 5 weeks of age were purchased from the National Laboratory Animal Center (NLAC), NARLabs. FAM129B expression in HCT116 was suppressed by transfection with the pooled three siRNAs targeting FAM129B (HCT116/si-FAM129B); cells transfected with non-specific control oligos served as a control (HCT116/si-Control). The si-FAM129B and si-Control HCT116 cells were inoculated subcutaneously 1.5 × 106 cells) into the flank of the NOD/SCID mice. When the tumor sizes reached about 100 mm3, the mice were randomly divided into two groups and subjected to treatment with PBS or oxaliplatin (7.5 mg/kg, intraperitoneally, every 4 days for 3 weeks) (n = 5 for each group). The tumor volumes, calculated as length × width2 × 0.5, were determined twice a week. At day 20 after injection of tumor cells, the tumors were carefully removed, photographed and weighed.

Female severe combined immune-deficient (NSG) mice between 4 and 5 weeks of age were purchased from The Jackson Laboratory. FAM129B expression in MDA-MB-231 cells was suppressed by transfection with the pooled three siRNAs targeting FAM129B (MB-231/si-FAM129B); cells transfected with non-specific control oligos served as a control (MB-231/si-Control). The si-FAM129B and si-Control MB-231 cells (1.5 × 106 cells) were injected into mammary fat pad of the NSG mice. After 2 days, the mice were randomly divided into two groups and subjected to treatment with PBS or oxaliplatin (20 mg/kg, intraperitoneally, once) (n = 4 for each group). The tumor volumes, calculated as length × width2 × 0.5, were determined twice a week. At day 35 after injection of tumor cells, the tumors were carefully removed, photographed and weighed. All procedures were performed in compliance with the regulations of the Institutional Animal Care and Use Committee of Chang-Gung University.

2.4Breast cancer patient samplesOne hundred and twenty-six fresh primary breast cancer tumor and adjacent normal tissue specimens were collected during surgical resections performed at the Tri-Service General Hospital (Taipei, Taiwan). Informed consent was obtained from all subjects before their tissues were deposited. The samples were fully encoded and used under a protocol approved by the Institutional Review Board of the Human Subjects Research Ethics Committee of the Tri-Service General Hospital and Chang Gung Memorial Hospital (Taoyuan, Taiwan). The clinicopathologic information is listed in Supplemental Table S1.

Breast cancer patient samples

One hundred and twenty-six fresh primary breast cancer tumor and adjacent normal tissue specimens were collected during surgical resections performed at the Tri-Service General Hospital (Taipei, Taiwan). Informed consent was obtained from all subjects before their tissues were deposited. The samples were fully encoded and used under a protocol approved by the Institutional Review Board of the Human Subjects Research Ethics Committee of the Tri-Service General Hospital and Chang Gung Memorial Hospital (Taoyuan, Taiwan). The clinicopathologic information is listed in Supplemental Table S1.

2.5Small interfering RNAs (siRNAs) and plasmid transfectionsiRNAs specifically targeting FAM129B and non-specific si-Control RNAs were synthesized by MDBio. The sequences are shown in the Key Resources Table. Transfection with the pooled three FAM129B siRNAs listed in the table was performed with RNAiMAX (ThermoFisher) according to the manufacturer's instructions.Full length Keap1 and the five truncated Keap1 mutants shown in the Key Resources Table were cloned into pACT vectors. Full length FAM129B, Nrf2, and Keap1 were cloned into pBIND, pcDNA3.1 and pFLAG-CMV2 vectors, respectively, using appropriate restriction enzyme digests. The numbered (or indicated) amino acids in DLG708 and E718T719GE in wild-type FAM129B were replaced with alanine, to generate DLA708- and A718A719GE-mutant FAM129B, respectively, using the QuikChange Lightning Site-Directed Mutagenesis Kit (Agilent) with full length FAM129B-expressing plasmids as templates. Transfections of plasmid DNA were performed with TransIT-2020 reagent (Mirus) according to the manufacturer's instructions.

Small interfering RNAs (siRNAs) and plasmid transfection

siRNAs specifically targeting FAM129B and non-specific si-Control RNAs were synthesized by MDBio. The sequences are shown in the Key Resources Table. Transfection with the pooled three FAM129B siRNAs listed in the table was performed with RNAiMAX (ThermoFisher) according to the manufacturer's instructions.

Full length Keap1 and the five truncated Keap1 mutants shown in the Key Resources Table were cloned into pACT vectors. Full length FAM129B, Nrf2, and Keap1 were cloned into pBIND, pcDNA3.1 and pFLAG-CMV2 vectors, respectively, using appropriate restriction enzyme digests. The numbered (or indicated) amino acids in DLG708 and E718T719GE in wild-type FAM129B were replaced with alanine, to generate DLA708- and A718A719GE-mutant FAM129B, respectively, using the QuikChange Lightning Site-Directed Mutagenesis Kit (Agilent) with full length FAM129B-expressing plasmids as templates. Transfections of plasmid DNA were performed with TransIT-2020 reagent (Mirus) according to the manufacturer's instructions.

2.6ROS detectionCells were treated with 10 μM 2′,7′-dichlorofluorescein diacetate (DCF-DA) for 30 min, following 2 washes with PBS. Reduced DCF-DA can be oxidized and converted to fluorescent 2′,7′-dichlorofluorescein (DCF) by intracellular ROS. Fluorescence signals were detected by flow cytometry (Sony EC800 Analyzer). Totally, 10,000 cells were analyzed per sample.

ROS detection

Cells were treated with 10 μM 2′,7′-dichlorofluorescein diacetate (DCF-DA) for 30 min, following 2 washes with PBS. Reduced DCF-DA can be oxidized and converted to fluorescent 2′,7′-dichlorofluorescein (DCF) by intracellular ROS. Fluorescence signals were detected by flow cytometry (Sony EC800 Analyzer). Totally, 10,000 cells were analyzed per sample.

2.7Western blot (WB) assayCells were washed with PBS and lysed in RIPA buffer containing NP-40 and protease inhibitors (Roche). Total protein extracts (10–30 μg) were separated on 4–12% NuPAGE (Invitrogen), and transferred to PVDF membranes (Millipore, Bedford, MA, USA). The membranes were incubated with primary antibodies at 4 °C overnight, followed by alkaline phosphatase-conjugated secondary antibodies (Jackson Immunoresearch Laboratories) for 1 h at room temperature. Then, the membrane was scanned with a Typhoon9400 Variable Mode Imager (GE Healthcare Life Sciences) to detect the fluorescent signals released from catalyzed ECF substrate (GE Healthcare Life Sciences). The details for the antibodies used in this study are provided in the Key Resources Table. The results of western blots were quantified using ImageQuant 5.2 software (GE Healthcare Life Sciences).

Western blot (WB) assay

Cells were washed with PBS and lysed in RIPA buffer containing NP-40 and protease inhibitors (Roche). Total protein extracts (10–30 μg) were separated on 4–12% NuPAGE (Invitrogen), and transferred to PVDF membranes (Millipore, Bedford, MA, USA). The membranes were incubated with primary antibodies at 4 °C overnight, followed by alkaline phosphatase-conjugated secondary antibodies (Jackson Immunoresearch Laboratories) for 1 h at room temperature. Then, the membrane was scanned with a Typhoon9400 Variable Mode Imager (GE Healthcare Life Sciences) to detect the fluorescent signals released from catalyzed ECF substrate (GE Healthcare Life Sciences). The details for the antibodies used in this study are provided in the Key Resources Table. The results of western blots were quantified using ImageQuant 5.2 software (GE Healthcare Life Sciences).

2.8Real-time PCR (RT-PCR)Total RNA of breast cancer tissue was isolated using TRIzol reagent (Invitrogen), and cDNA was generated from 1000 ng of total RNA, using a High Capacity cDNA Reverse Transcription Kit (Applied BioSystems/ABI). RT-PCR assays were performed on a QuantStudio™ 7 Flex Real-Time PCR system (Applied Biosystems) using SYBR Green MasterMix (ABI). For accurate normalization of quantitative data, multiple housekeeping genes, including GAPDH, GUSB (glucuronidase-beta) and UBC (polyubiquitin) were assayed. The primer sequences used in RT-PCR assays are listed in the Key Resources Table.

Real-time PCR (RT-PCR)

Total RNA of breast cancer tissue was isolated using TRIzol reagent (Invitrogen), and cDNA was generated from 1000 ng of total RNA, using a High Capacity cDNA Reverse Transcription Kit (Applied BioSystems/ABI). RT-PCR assays were performed on a QuantStudio™ 7 Flex Real-Time PCR system (Applied Biosystems) using SYBR Green MasterMix (ABI). For accurate normalization of quantitative data, multiple housekeeping genes, including GAPDH, GUSB (glucuronidase-beta) and UBC (polyubiquitin) were assayed. The primer sequences used in RT-PCR assays are listed in the Key Resources Table.

2.9ImmunoprecipitationCells were washed with PBS and lysed in RIPA buffer containing NP-40 and protease inhibitors (Roche). The lysate was cleared by centrifugation at 13,000 Rcf for 30 min at 4 °C. Immunoprecipitation was performed by incubation of cell lysates with anti-Flag or anti-HA antibody, and capture on Dynabeads® Protein G (10003D, Thermo Fisher Scientific) for 4 h at 4 °C in a rocking incubator. After six washes with washing buffer, immunoprecipitated complexes were eluted in sample buffer by boiling in water for 4 min, electrophoresed through 4–12% NuPAGE (Invitrogen) gel, and subjected to immunoblot analysis.

Immunoprecipitation

Cells were washed with PBS and lysed in RIPA buffer containing NP-40 and protease inhibitors (Roche). The lysate was cleared by centrifugation at 13,000 Rcf for 30 min at 4 °C. Immunoprecipitation was performed by incubation of cell lysates with anti-Flag or anti-HA antibody, and capture on Dynabeads® Protein G (10003D, Thermo Fisher Scientific) for 4 h at 4 °C in a rocking incubator. After six washes with washing buffer, immunoprecipitated complexes were eluted in sample buffer by boiling in water for 4 min, electrophoresed through 4–12% NuPAGE (Invitrogen) gel, and subjected to immunoblot analysis.

2.10Determination of cell viability by MTS assayCell viabilities were assessed by adding MTS reagent,3-(4,5-Dimethylthiazol-2-yl)-5-(3-carboxymethoxyphenyl)-2-(4-sulfophenyl)-2H-tetrazolium (Biovision), to the cells. Optical density was measured on a SpectraMax M2 microplate reader (Molecular Devices) at 490 nm. Each experiment was conducted in triplicate and repeated independently three times.

Determination of cell viability by MTS assay

Cell viabilities were assessed by adding MTS reagent,

3-(4,5-Dimethylthiazol-2-yl)-5-(3-carboxymethoxyphenyl)-2-(4-sulfophenyl)-2H-tetrazolium (Biovision), to the cells. Optical density was measured on a SpectraMax M2 microplate reader (Molecular Devices) at 490 nm. Each experiment was conducted in triplicate and repeated independently three times.

2.11Luciferase reporter assaySix copies of the antioxidant response element (ARE) (5′-GTGACAAAGCACCCGTGACAAAGCACCCGTGACAAAGCACCCGTGACAAAGCACCCGTGACAAAGCACCCGTGACAAAGCA-3′) were cloned into the pGL3-basic luciferase reporter plasmid [27]. The indicated cells were transfected with ARE-luciferase reporter and Renilla luciferase together with either FAM129B plasmids or si-FAM129B. At 48–72 h after transfection, the luciferase activities of cell lysates were measured with the Dual Luciferase assay system (Promega).

Luciferase reporter assay

Six copies of the antioxidant response element (ARE) (5′-GTGACAAAGCACCCGTGACAAAGCACCCGTGACAAAGCACCCGTGACAAAGCACCCGTGACAAAGCACCCGTGACAAAGCA-3′) were cloned into the pGL3-basic luciferase reporter plasmid [27]. The indicated cells were transfected with ARE-luciferase reporter and Renilla luciferase together with either FAM129B plasmids or si-FAM129B. At 48–72 h after transfection, the luciferase activities of cell lysates were measured with the Dual Luciferase assay system (Promega).

2.12GSH/GSSG detectionThe ratio of reduced to oxidized glutathione (GSH/GSSG) in cells was measured using the GSH/GSSG-Glo™ Assay according to the manufacturer's instructions (Promega, V6611).

GSH/GSSG detection

The ratio of reduced to oxidized glutathione (GSH/GSSG) in cells was measured using the GSH/GSSG-Glo™ Assay according to the manufacturer's instructions (Promega, V6611).

2.13FLIM-FRET measurementFLIM-FRET measurements were made as previously described [28], with modifications. A Leica TCS SP5 equipped with multiphoton fluorescence lifetime imaging microscopy (Leica TCS-SP5-AOBS-MP) system was used for confocal imaging and to measure fluorescence lifetime. A water immersion objective (Leica, 63×/0.9 APO) was employed both for focusing laser light onto the samples and for collecting fluorescence emissions from the samples. The fluorescence lifetime for each image pixel was recorded using the time-correlated single photon counting technique (Becker & Hickl SPEC-830 TCSPC modules).

FLIM-FRET measurement

FLIM-FRET measurements were made as previously described [28], with modifications. A Leica TCS SP5 equipped with multiphoton fluorescence lifetime imaging microscopy (Leica TCS-SP5-AOBS-MP) system was used for confocal imaging and to measure fluorescence lifetime. A water immersion objective (Leica, 63×/0.9 APO) was employed both for focusing laser light onto the samples and for collecting fluorescence emissions from the samples. The fluorescence lifetime for each image pixel was recorded using the time-correlated single photon counting technique (Becker & Hickl SPEC-830 TCSPC modules).

2.14In situ proximity ligation assay (PLA)Interaction between Keap1 and FAM129Bwere assessed using an in situ PLA kit (Duolink) according to the manufacturer's instructions. In brief, cells (2 × 104) grown on 8-well slides (ibidi) were fixed in 4% paraformaldehyde in PBS for 15 min at room temperature, blocked with Duolink blocking solution for 30 min at 37 °C, washed with PBS, incubated overnight at 4 °C with primary antibodies (Rabbit anti-FAM129B and mouse anti-Keap1, as described above), washed with PBS, incubated for 1 h at 37 °C with secondary antibody (anti-mouse PLA-plus probe or anti-rabbit PLA-minus probe; Duolink; dilution 1:50), washed twice (5 min each time) with Duolink Wash buffer A, combined with Duolink ligation mixture, incubated for 30 min at 37 °C, washed twice with Wash buffer A, combined with Duolink amplification mixture and Polymerase, subjected to amplification reaction for 100 min at 37 °C, washed twice with Wash buffer B and once with 0.1× Wash buffer B, and mounted with Duolink In Situ Mounting Medium with DAPI [29]. Fluorescence dot images were obtained by confocal immunofluorescence microscopy as above.

In situ proximity ligation assay (PLA)

Interaction between Keap1 and FAM129Bwere assessed using an in situ PLA kit (Duolink) according to the manufacturer's instructions. In brief, cells (2 × 104) grown on 8-well slides (ibidi) were fixed in 4% paraformaldehyde in PBS for 15 min at room temperature, blocked with Duolink blocking solution for 30 min at 37 °C, washed with PBS, incubated overnight at 4 °C with primary antibodies (Rabbit anti-FAM129B and mouse anti-Keap1, as described above), washed with PBS, incubated for 1 h at 37 °C with secondary antibody (anti-mouse PLA-plus probe or anti-rabbit PLA-minus probe; Duolink; dilution 1:50), washed twice (5 min each time) with Duolink Wash buffer A, combined with Duolink ligation mixture, incubated for 30 min at 37 °C, washed twice with Wash buffer A, combined with Duolink amplification mixture and Polymerase, subjected to amplification reaction for 100 min at 37 °C, washed twice with Wash buffer B and once with 0.1× Wash buffer B, and mounted with Duolink In Situ Mounting Medium with DAPI [29]. Fluorescence dot images were obtained by confocal immunofluorescence microscopy as above.

2.15Mammalian Two-Hybrid SystemInteraction between Keap1 and FAM129B were assessed using a CheckMate™ Mammalian Two-Hybrid System (Promega, E2440) according to the manufacturer's instructions.

Mammalian Two-Hybrid System

Interaction between Keap1 and FAM129B were assessed using a CheckMate™ Mammalian Two-Hybrid System (Promega, E2440) according to the manufacturer's instructions.

2.16Statistical analysesThe prognostic performance of genes was calculated as mean values. Relapse-free survival (RFS) values were estimated by the Kaplan–Meier method and were compared by the log-rank test. The Cox regression model was used for analysis of factors potentially related to RFS. The statistical analyses were performed with GraphPad Prism software (version 5.0, GraphPad Software) and MedCalc statistical software (version 14.8, MedCalc Software). Data are presented as means ±standard deviation (SD). Student's t-test was applied to assess the statistical significance. p-Values <.05 were considered significant.

Statistical analyses

The prognostic performance of genes was calculated as mean values. Relapse-free survival (RFS) values were estimated by the Kaplan–Meier method and were compared by the log-rank test. The Cox regression model was used for analysis of factors potentially related to RFS. The statistical analyses were performed with GraphPad Prism software (version 5.0, GraphPad Software) and MedCalc statistical software (version 14.8, MedCalc Software). Data are presented as means ±standard deviation (SD). Student's t-test was applied to assess the statistical significance. p-Values <.05 were considered significant.

2.17Ubiquitination assayCells were exposed to 10 μM MG132 (Sigma) for 4 h. Cells were lysed by boiling in a buffer containing 2% SDS, 150 mM NaCl, 10 mM Tris-HCl with 2 mM sodium orthovanadate, 50 mM sodium fluoride, and protease inhibitors. The lysates were incubated with an Nrf2 antibody and subjected to immunoblot analysis.

Ubiquitination assay

Cells were exposed to 10 μM MG132 (Sigma) for 4 h. Cells were lysed by boiling in a buffer containing 2% SDS, 150 mM NaCl, 10 mM Tris-HCl with 2 mM sodium orthovanadate, 50 mM sodium fluoride, and protease inhibitors. The lysates were incubated with an Nrf2 antibody and subjected to immunoblot analysis.

2.18Protein half-life measurementFifty micromolar cycloheximide (Sigma) was added in MDA-MB-231 cells. Total cell lysates were collected at different time points and subjected to immunoblot analysis with an anti-Nrf2 antibody. The relative intensities of the bands were quantified by using the ImageQuant 5.2 software.

Protein half-life measurement

Fifty micromolar cycloheximide (Sigma) was added in MDA-MB-231 cells. Total cell lysates were collected at different time points and subjected to immunoblot analysis with an anti-Nrf2 antibody. The relative intensities of the bands were quantified by using the ImageQuant 5.2 software.

3Results3.1Identification of FAM129B as a Keap1-interacting proteinIn a global proteomic analysis of 75 deubiquitinating enzymes [30] and a proteomic analysis of Keap1 associated proteins [31], FAM129B was listed as a potential candidate interacting protein for Keap1 by mass spectrometry. In this study, we performed a series of experiments to verify this association. First, 293T cells were co-transfected with Flag-FAM129B and HA-Keap1, followed by reciprocal immunoprecipitation experiments. As shown in Fig. 1a, HA-Keap1 was present in the Flag immunoprecipitates and Flag-FAM129B was detected in the HA immunoprecipitates, indicating that FAM129B and Keap1 were both present in the same complex. Next, fluorescence microscopy was used to determine whether FAM129B and Keap1 co-localized in mammalian cells. FAM129B and Keap1 were individually tagged with AcGFP1 and DsRed2, respectively. AcGFP1-FAM129B and DsRed2-Keap1 proteins were co-expressed in 293T cells, and their cellular localization was visualized by real-time live imaging system under a fluorescent microscope. As shown in Fig. 1b, ectopically expressed AcGFP1-FAM129B and DsRED2-Keap1 co-localized mainly in the cytoplasm (Pearson's Correlation = 0.9126). To further validate their interactions, we measured Förster resonance energy transfer (FRET) of AcGFP1 using multiphoton fluorescence lifetime imaging microscopy (FLIM). The fluorescence lifetime (t1/2) of cells expressing AcGFP1-FAM129B alone was distributed around 2.72 ns. Co-expression of DsRed2-Keap1 with AcGFP1-FAM129B produced a substantial shift in the distribution of AcGFP1t1/2 to 2.57 ns (Fig. 1c). The shorter t1/2 is indicative of FRET interaction between AcGFP1-FAM129B and DsRed2-Keap1. This finding is further supported by the demonstration of in situ interactions between FAM129B and Keap1 using the proximity ligation assay (PLA). With this method, when a pair of PLA probes binds two molecules in close proximity (< 16 nm), complementary DNA strands conjugated to PLA probes are ligated, amplified, and visualized as distinct points using a fluorescent probe [29]. In both Hs578T and HCT116 cells, strong PLA signals were observed for the FAM129B-Keap1 association (Fig. 1d). Moreover, a mammalian two-hybrid analysis was carried out in 293T cells. FAM129B was fused to the DNA-binding domain (DBD) of GAL4, while Keap1 was fused to the activation domain (AD) of VP16. Activation of the GAL4/Luc reporter (pG5) was observed in cells co-transfected with DBD-FAM129B and AD-Keap1 compared to cells transfected with the DBD construct alone (Fig. 1e). Taken together, these results validate that FAM129B and Keap1 can form a complex in cells.Fig. 1Identification of FAM129B as a Keap1-Interacting Protein.(a) Determination of interaction between FAM129B and Keap1 by co-immunoprecipitation. Lysates of 293T cells co-expressing Flag-FAM129B and HA-Keap1 were immunoprecipitated with anti-HA or anti-Flag antibodies. The total lysates and the immunoprecipitates were subjected to immunoblotting with anti-Flag and anti-HA antibodies for detecting FAM129B and Keap1, respectively.(b) Examination of co-localization of FAM129B and Keap1. 293T cells were co-transfected with AcGFP1-FAM129B and DsRed2-Keap1 expression constructs, and observed under a confocal microscope. Overlay images demonstrate co-localization of green FAM129B and red Keap1 molecules by a shift to orange.(c) FLIM-FRET measurements of the FAM129B-Keap1 interaction by determination of fluorescence lifetime of FRET between AcGFP1-FAM129B constructs and DsRed2-Keap1. Fluorescence lifetime of AcGFP1 was determined in 293T cells 48 h after transfection with either AcGFP1-FAM129B alone or in combination with DsRed2-Keap1. (left panels) Pictorial representations of the AcGFP1 lifetime; the color of the cell corresponds with the lifetime, which ranged from 2.1 to 2.8 ns. (right panels) Lifetime data from each pixel of the image, plotted on a graph.(d) Demonstration of interaction between FAM129B and Keap1 by proximity ligation assay (PLA) in Hs578T and HCT116 cells as determined by in situ PLA using anti-FAM129B and anti-Keap1 antibodies. Co-localization of FAM129B and Keap1 is reflected by red fluorescence in the images. Scale bar in (b-d), 20 μm.(e) Examination of the FAM129B-Keap1 interaction using a mammalian two-hybrid assay. 293T cells were co-transfected with a construct expressing Keap1 fused with AD of VP16, along with a construct expressing full-length FAM129B fused with the DBD of GAL4 and the pG5 luciferase reporter, followed by performance of the mammalian two-hybrid assay at 24 h post transfection. The firefly luciferase activities were normalized to Renilla luciferase activities. The data are presented as the mean values of triplicates ± SD. (For interpretation of the references to color in this figure legend, the reader is referred to the web version of this article.)Fig. 13.2FAM129B interacts with the Kelch domain of Keap1 via its DLG and ETGE motifsTo delineate which Keap1 domains contribute to the interaction with FAM129B, we used mammalian two-hybrid assays to examine the abilities of truncation mutants of different Keap1 domains, including N, BTB, BACK, Kelch and Kelch/C, to bind FAM129B. We found that FAM129B could only interact with Keap1 truncation mutants containing the Kelch domain (Fig. 2a). Previously, the Kelch domain of Keap1 was reported to interact with the Neh2 domain of Nrf2 through its DLG and ETGE motifs [32]. Interestingly, C-terminus of FAM129B also contains DLG and ETGE motifs which were highly conserved in different species (Fig. 2b). Although tertiary structure of FAM129B protein remains unknown; we used computer modeling to predict binding of human Kelch domain of Keap1 to a peptide fragment (K705–S725) of FAM129B which covered the two motifs. Based on the structural information of Kelch-Neh2 complex available from Protein Data Bank (see EXPERIMENTEL MODEL), the DLG and the ETGE motifs of Nrf2 bound separately to the same binding pocket located at Kelch domain of Keap1via different orientation and molecular interactions. As shown in Fig. 2c, our computer modeling predicted two orientations of FAM129B K705–S725 peptide which could interact with the same area of Kelch domain of human Keap1 through DLG motif (blue color) and ETGE motifs (red color). Moreover, the Hydrogen-bond analysis revealed interactions between the Keap1-Kelch domain and FAM129B K705–S725 peptide. The DLG motif of FAM129B peptide sequence forms 5 hydrogen bonds (green-dashed lines in Fig. 2d) with the amino acids on Keap1-Kelch domain. On the other hand, as many as 12 hydrogen bonds might occur near the ETGE motif of FAM129B peptide sequence (Fig. 2e). We further assessed the binding energies of the molecular interactions for these two binding modes using the HotLig scoring program [24]. The estimated energy scores for the binding interactions through the DLG and the ETGE motifs were − 21.7 and − 24.6, respectively. To further characterize the FAM129B-Keap1 interaction biochemically, three FAM129B mutants were created by alanine replacement of amino-acid residues of its DLG and/or ETGE motifs. We used mammalian two-hybrid assays and co-immunoprecipitation to examine the binding abilities of the different FAM129B mutants to Keap1. The mammalian two-hybrid assays showed that mutation of either the DLG or ETGE motif abolished binding to Keap1 (Fig. 2f). However, immunoprecipitation analysis revealed that mutation of the DLG motif had a negligible effect on binding to Keap1, while mutation of the ETGE motif abolished binding (Fig. 2g). The latter finding is consistent with the reported lower binding affinity of the DLG motif ofNrf2 (1 × 106 M−1) for Keap1compared to the ETGE motif (2 × 108 M−1) [33]. These findings are also in agreement with our demonstration of higher binding affinity of the ETGE motif of FAM129B than the DLG motif for Keap1.Fig. 2FAM129B Interacts with the Kelch domain of Keap1 via its DLG and ETGE motifs.(a) Examination of FAM129B-Keap1 domain interactions by mammalian two-hybrid assay. Five different domains of Keap1, including an N-terminal region (amino acids 1–76), a BTB domain (amino acids 77–179), a BACK domain (amino acids 180–326), a Kelch domain (amino acids 327–611) and a Kelch/C domain (amino acids 327–624) were cloned and fused with the AD of VP16. 293T cells were co-transfected with a construct expressing Keap1 fused with AD of VP16, along with a construct expressing full-length FAM129B fused with the DBD of GAL4 and the pG5 luciferase reporter, followed by performance of the mammalian two-hybrid assay at 24 h post transfection. The firefly luciferase activities were normalized to Renilla luciferase activities. The data are presented as the mean values of triplicates ± SD.(b) Alignment of the Keap1 interacting motifs (DLG and ETGE motifs; black lines) of FAM129B homologs in various species. Black and grey boxes indicate identical amino acid residues with complete and partial conservation, respectively.(c) Predicted interactions of the Kelch domain with the DLG or the ETGE motif of the FAM129B peptide segment (705-KAVDLGPPKPSDQETGEQVSS-725). Both motifs can access the same binding site on Kelch domain.(d) Predicted hydrogen bonds (green dashed lines) and hydrophobic interactions (radiating spokes) near the DLG motif of the FAM129B for binding with the Kelch domain. The radiating spokes represent the atoms or amino acid residues that have hydrophobic interactions.(e) Molecular interactions near the ETGE motif involve more hydrogen bonds than DLG motif as shown in (d).(f) Examination of FAM129B-Keap1interaction through the FAM129B DLG motif or ETGE motif by mammalian two-hybrid assay. Three FAM129B mutants, M1, M2 and M3, were generated, in which the indicated amino acids in the DLG and/or ETGE motif were replaced with alanine residues. The data are presented as the mean values of triplicates ± SD.(g) Effects of mutations of the DLG and ETGE motifs of FAM129B on its interaction with Keap1. 293T cell lysates co-expressing wild type (WT) or mutant Flag-FAM129B and HA-Keap1 were immunoprecipitated with anti-Flag antibody. The total lysates and the immunoprecipitates were subjected to immunoblot analysis with both anti-Flag and anti-HA antibodies for detection of FAM129B and Keap1.Fig. 23.3FAM129B Dampens the Keap1-dependent ubiquitination of Nrf2 by competing with Nrf2 for Keap1 bindingThe“hinge and latch”model has been proposed for Nrf2 stabilization which involves high-affinity binding of ETGE motif (hinge) to fix Nrf2 to Keap1 and the low-affinity binding of DLG motif (latch) to lock down the Neh2 domain of Nrf2, thereby facilitating the ubiquitylation of lysine residues and constant degradation of Nrf2 [32,33]. Since FAM129B interacts with Keap1 through the same motif sequences (DLG and ETGE) as Nrf2, the Keap1 dimer might also recruit FAM129B through binding to its DLG and ETGE motifs, similar to the binding mode involved in the Keap1-Nrf2 complex (Fig. 3a, right). In support of this notion, in vitro binding experiments revealed that the amount of recombinant Flag-Nrf2 protein forming a complex with HA-Keap1 decreased in proportion to the recombinant Flag-FAM129B protein level (Fig. 3b). At the cellular level, FAM129B overexpression in 293T cells led to a marked decrease in Nrf2 ubiquititylation (Fig. 3c). As expected, the half-life of Nrf2 was shorter in FAM129B-siRNA cells than in control-siRNA cells (15.4 min versus 43.0 min) (Fig. 3d). In several cancer cell lines, FAM129B silencing by siRNA reduced the expression of endogenous FAM129B protein with a concomitant decrease in Nrf2 protein levels (Fig. 3e). In contrast, overexpression of FAM129B increased Nrf2 protein levels (Fig. 3f). These results indicate that FAM129B interfered with the Keap1-dependent ubiquitination of Nrf2 by competing with Nrf2 for Keap1 binding.Fig. 3FAM129B Dampens the Keap1-Dependent Ubiquitination of Nrf2 by Competing with Nrf2 for Keap1 Binding.(a) Keap1 proteins form a homodimer through the N-terminal BTB domains, while the C-terminal Kelch domains interact with Nrf2 protein. One Kelch domain recognizes the DLG motif and the other binds the ETGE motif of Nrf2. Similarly, Keap1 dimer might recruit FAM129B through binding to its DLG and ETGE motifs.(b) Competition between Nrf2 and FAM129B for binding to Keap1. Cell lysates from HA-Keap1 expressing 293T cells were immunoprecipitated with anti-HA antibody. The resulting immunoprecipitates were co-incubated with Nrf2 recombinant protein and increasing amounts of FAM129B recombinant protein, and then immunoblotted with the indicated antibodies.(c) Effects of FAM129B overexpression on the ubiquitination of Nrf2. 293T cells were transfected with the indicated plasmids followed by treatment with MG132 (10 μM) for 4 h. Cell lysates were subjected to an in vivo ubiquitination assay to detect the ubiquitin-conjugated Nrf2 protein. Lysates were denatured and immunoprecipitated with anti-Nrf2 antibody and blotted with anti-HA antibody. An aliquot of total lysate was analyzed using the indicated antibodies.(d) Effects of FAM129B silencing on the stability of Nrf2 protein. MDA-MB-231 cells were transfected with either control-siRNA or pooled FAM129B-siRNAs. At 72 h, cells were treated with cycloheximide (CHX), and cell lysates were harvested at the indicated time points for immunoblotting using the indicated antibodies. Nrf2 protein expression was normalized to β-actin, and the control group was set as 1.(e) Effects of FAM129B silencing on the expression of Nrf2 protein. Cells were transfected with either control-siRNA (C) or pooled FAM129B-siRNAs (F). Cell lysates were collected at 72 h post transfection and subjected to immunoblot analyses using the indicated antibodies.(f) Effects of FAM129B overexpression on the expression of Nrf2 protein. Cells were transfected with either empty vector or Flag-FAM129B. Cell lysates were collected at 48 h post transfection and subjected to immunoblot analyses using the indicated antibodies.(g) Effects of FAM129B silencing on the protein levels of Nrf2 downstream genes. Cells were transfected with either control-siRNA (C) or pooled FAM129B-siRNAs (F). Cell lysates were collected at 72 h post transfection and subjected to immunoblot analyses using the indicated antibodies.Fig. 33.4Nrf2 antioxidative signaling is required for FAM129B to inhibit ROSAs the master antioxidant regulator, Nrf2 governs the intracellular ROS level by regulating antioxidant gene products. Our findings that FAM129B modulates the level of Nrf2 protein suggest the possibility of involvement of FAM129B in regulating intracellular ROS. To address this possibility, we measured ROS levels in control and FAM129B knockdown cells, using flow cytometry after staining with dichlorofluorescein diacetate (DCF-DA). As shown in Fig. 4b, the expression of FAM129B protein was effectively silenced by pooled FAM129B-siRNAs in MDA-MB-231, Hs578T, HCT116 and H1299 cell lines. Interestingly, FAM129B silencing was associated with elevated basal intracellular ROS levels in these cells (Fig. 4a). As alterations in ROS levels can affect the intracellular redox state, we measured the ratio of reduced to oxidized glutathione (GSH/GSSG), which is a major indicator for oxidative stress in cells [34]. As expected, the GSH/GSSG ratio was markedly lowered in the FAM129B-siRNA-transfected cells (Fig. 4c). These results suggest that FAM129B may ameliorate intracellular oxidative stress in cancer cells.Fig. 4Nrf2 Antioxidative Signaling is Required for FAM129B to Inhibit ROS.(a) Effects of FAM129B silencing on ROS. At 72 h after transfection with the indicated siRNAs, intracellular ROS levels were measured by using a DCF/flow cytometric method.(b) Efficiency of FAM129B silencing by pooled FAM129B-siRNAs. Cells were transfected with control-siRNA (C) or pooled FAM129B-siRNAs (F). Cell lysates were collected at 72 h post transfection and subjected to immunoblot analyses using the indicated antibodies.(c) Effects of FAM129B silencing on the GSH/GSSG ratio. Cellular oxidative status was determined by measuring the GSH/GSSG ratio after siRNA-mediated FAM129B knockdown. Fold changes in GSH/GSSG ratio are presented as the mean values of triplicates ± SD.(d, e) Regulation of the ARE-reporter gene activity of Nrf2 by FAM129B. HCT116 cells were transfected with the indicated siRNAs (d) or constructs (e), along with the expression vector for 6 repeats of the ARE element of the firefly luciferase promoter and Renilla luciferase. The firefly luciferase activities were normalized to Renilla luciferase activities. The data are presented as the mean values of triplicates ± SD. ⁎p < .05, ⁎⁎⁎p < .001 (Student's t-test).(f) Effects of FAM129B silencing on the mRNA expression of Nrf2 and its downstream genes. Cells were transfected with the indicated siRNAs. After 72 h, mRNA was extracted for determination of the expression levels of Nrf2 and its downstream genes by real-time RT-PCR. Values were normalized to GAPDH. The data are presented as the mean values of triplicates ± SD.(g) Effects of FAM129B overexpression on the mRNA expression of Nrf2 and its downstream genes. Cells were transfected with either empty vector or flag-FAM129B. At 48 h, mRNA was extracted for determination of the expression levels of Nrf2 and its downstream genes by real-time RT-PCR. Values were normalized to GAPDH. The data are presented as the mean values of triplicates ± SD.Fig. 4To investigate the mechanisms underlying FAM129B-mediated ROS inhibition, the relationship between FAM129B and Nrf2 signaling was explored further. Nrf2 fulfills its function mainly by binding to the ARE (antioxidant response element) region of antioxidant gene promoters [35]. To investigate the effect of FAM129B on Nrf2 transcriptional activity, we used a luciferase reporter pGL3-ARE-luc to monitor the transcriptional activity of Nrf2 in HCT116 cells. Under FAM129B knockdown or overexpression conditions, the pGL3-ARE-luc activity was significantly decreased by FAM129B knockdown to about 70% of control values (Fig. 4d), but increased by about 3-fold by FAM129B overexpression (Fig. 4e). To confirm above findings, we also determined the mRNA levels of several Nrf2 downstream genes by real-time RT-PCR, including AKR1B10 AKR1C1, AKR1C2, AKR1C3 and NQO1 [36]. In general, FAM129B knockdown decreased the expression of most of these Nrf2 downstream genes, whereas FAM129B overexpression increased their levels in the two cell lines examined (Fig. 4f and g). The protein levels of Nrf2 downstream genes, NQO1 and HO-1, were also reduced in FAN129B silenced cells (Fig. 3g). Furthermore, ectopic expression of Nrf2 reversed the increase in ROS induced by FAM129B knockdown (Fig. S1a) and also rescued the reduction in the expression Nrf2 downstream genes upon FAM129B silencing (Fig. S1b). Collectively, these results further demonstrate that FAM129B inhibits ROS production by enhancing the transcriptional activity of Nrf2 with the consequent Nrf2-dependent antioxidant response.3.5FAM129B silencing enhances sensitivity of cancer cells to oxaliplatinOur finding of the ability of FAM129B to stabilize Nrf2 protein implies that FAM129B may contribute to antioxidant response. To decipher the antioxidant function mediated by FAM129B, the effects of tert-butyl hydroperoxide (tBHP), an organic peroxide, on ROS levels and cell survival were determined. Treatment of FAM129B-siRNA-transfected cells with tBHP further augmented ROS elevation as compared to control-siRNA cells in each of the four cancer cell lines examined (Fig. 5a). FAM129B knockdown significantly increased the susceptibility of these cell lines to tBHP-induced cell death (Fig. 5b). These findings demonstrate that FAM129B has an antioxidant effect that is required for cellular protection from oxidative stress.Fig. 5FAM129B Silencing Enhances Sensitivity of Cancer Cells to Oxaliplatin.(a,b) Sensitivity of FAM129B-silenced cells to oxidative stress. Cells were transfected with control siRNA or pooled FAM129B-siRNAs. At 72 h, cells were treated with the indicated concentrations of tBHP for 24 h. Intracellular ROS levels were measured by using a DCF/flow cytometric method (a). Cell viability was analyzed by MTS assay (b). The data are presented as the mean values of triplicates ± SD. ⁎p < .05, ⁎⁎p < .01, ⁎⁎⁎p < .001 (Student's t-test).(c) Sensitivity of FAM129B silenced cells to oxaliplatin. Cells were transfected with control siRNA or pooled FAM129B-siRNAs. At 72 h, cells were treated with the indicated concentrations of oxaliplatin for 48 h, and cell viability was analyzed by MTS assay. The data are presented as the mean values of triplicates ± SD.⁎⁎p < .01, ⁎⁎⁎p < .001 (Student's t-test).(d–g) FAM129B silencing significantly enhanced the suppressant effect of oxaliplatin on tumor growth. NSG mice (n = 4 each group) were injected into mammary fat pad with si-Control or si-FAM129B MDA-MB-231 cells. After 2 days, the mice were treated with PBS only or oxaliplatin. Tumor volumes at the indicated time (d), and tumor images and weights at day 35 after injection of tumor cells are shown (e). NOD/SCID mice (n = 5 each group) were injected subcutaneously with si-Control or si-FAM129B HCT116 cells. When the tumor sizes reached about 100 mm3 the mice were treated with PBS only or oxaliplatin. Tumor volumes at the indicated time (f), and tumor images and weights at day 20 after injection of tumor cells are shown (g). The data are presented as means ± SD. ⁎p < .05, ⁎⁎p < .01, ⁎⁎⁎p < .001 (one-way ANOVA with Tukey's multiple comparison test).Fig. 5Induction of ROS-mediated damage in cancer cells by pharmacological agents such as platinum coordination compounds and anthracyclines that either promote ROS generation or disable the cellular antioxidant system has been considered as an effective therapeutic strategy to preferentially kill cancer cells [37]. In light of the antioxidative activity of FAM129B, we further examined whether FAM129B silencing would improve the response of cancer cells to the widely used chemotherapeutic drug oxaliplatin. As expected, siRNA-mediated FAM129B silencing significantly increased the sensitivity of MDA-MB-231, Hs578T, and HCT116 cells to oxaliplatin (Fig. 5c). These findings indicate that FAM129B depletion is a good strategy to augment the efficacy of oxaliplatin in cancer therapy.The in vivo effect of FAM129B on chemosensitivity was further examined using the MDA-MB-231 human breast cancer xenograft model in severe combined immune-deficient NSG mice. As shown in Fig. 5d, FAM129B knockdown did not suppress the tumor growth, in comparison to si-Controls (Fig. 5d and e). However, treatment with low dose oxaliplatin had a significantly greater inhibitory effect, as assessed by decreases in tumor volume, on si-FAM129B tumors than on si-Controls (p < .01 [one-way ANOVA with Tukey's multiple comparison test], Fig. 5d). Although oxaliplatin treatment had negligible effects on tumor weight in si-Controls, it significantly reduced tumor weight in the FAM129B knockdown group to 56% (p < .01 [one-way ANOVA with Tukey's multiple comparison test]) of si-Controls (Fig. 5e). To further confirm the role of FAM129B in chemosensitivity, a human colon cancer xenograft model of HCT116 cells in severe combined immune-deficient (NOD/SCID) mice was used. In contrast to results obtained with the MDA-MB-231 xenograft model, FAM129B knockdown significantly suppressed tumor growth, as compared to si-Controls (p < .05 [one-way ANOVA with Tukey's multiple comparison test], Fig. 5f). Treatment of si-Controls with oxaliplatin also decreased tumor growth. The combined inhibitory effect of FAM129B silencing and oxaliplatin treatment on tumor growth was greater than the effect of oxaliplatin alone (p < .05 [one-way ANOVA with Tukey's multiple comparison test]) or FAM129B silencing alone (p < .01 [one-way ANOVA with Tukey's multiple comparison test]). Furthermore, the average tumor weight in the FAM129B knockdown group was 76% of the si-Control group (p < .05 [one-way ANOVA with Tukey's multiple comparison test], Fig. 5g) and treatment of the si-FAM129B group with oxaliplatin further reduced tumor weight to 27% (p < .001 [one-way ANOVA with Tukey's multiple comparison test]) of si-Controls. In comparison, oxaliplatin treatment of the si-Control group decreased tumor weight tumor to 49% (p < .001 [one-way ANOVA with Tukey's multiple comparison test]) of the untreated si-Controls (Fig. 5g). Collectively, these results indicate that FAM129B silencing rendered xenografts more susceptible to oxaliplatin, showing a critical role of FAM129B in chemosensitivity in vivo.3.6Elevated FAM129B expression correlates with expression of Nrf2 target genes and confers poor clinical outcomeTo investigate the clinical relevance of FAM129B in human cancers, we determined FAM129B expression by RT-PCR in127 breast cancer specimens and their adjacent normal tissues. The clinicopathologic characteristics of these patients are shown in Supplemental Table S1. FAM129B expression was significantly higher in tumor than the adjacent normal tissue (p < .0001 [Paired t-test], Fig. 6a). More importantly, Kaplan–Meier survival analysis showed that patients with greater than mean expression levels of FAM129B had significantly worse relapse-free survival (RFS) than those with lower FAM129B expression (p = .0206 [Log rank test], Fig. 6b). Univariate Cox proportional hazard regression analyses were conducted. RFS correlated with age, ER negative status, advanced stage (stage III and IV), triple negative molecular type and high FAM129B expression (Table 1). The statistically significant factors for RFS that were identified using multivariate analyses are also presented in Table1. Advanced stage (stage III and IV), triple negative molecular type, and high FAM129B expression were identified as independent predictors of tumor recurrence (Table 1). These results suggest that in addition to the well-known negative impact of “clinical stage” and “triple-negative molecular type”, high FAM129B gene expression can serve as a prognostic marker for tumor recurrence (hazard ratio 1.967, 95% CI, 1.065–3.632). To further validate the prognostic value of FAM129B, we applied GEPIA (Gene Expression Profiling Interactive Analysis; http://gepia.cancer-pku.cn), for survival analysis of web-based TCGA (The Cancer Genome Atlas) cancer gene expression profiles [38]. Consistent with our findings, elevated FAM129 was found to correlate with poorer RFS in breast cancer (p < .001 [Log rank test], Fig. 6c). In addition, FAM129B overexpression had significant adverse impacts on overall survival in breast cancer (p < .001 [Log rank test], Fig. 6d), as well as RFS (p < .01 [Log rank test], Fig. 6e) and overall survival (p < .06 [Log rank test], Fig. 6f) in lung cancer. Thus, FAM129B may serve as a prognostic marker for clinical outcome of breast and lung cancer.Finally, to establish the physiological role of FAM129B in controlling Nrf2 transcriptional activity in breast and lung cancer, we examined whether the expression of FAM129B correlated to Nrf2 downstream genes. Using GEPIA analysis of TCGA database, we found FAM129B expression to be positively correlated with several Nrf2 downstream genes including ABCC1, ACOT7, MAFG and PTGR1 [39,40] in breast and lung cancer (Fig. 6g). These data support the notion that high FAM129B expression in tumors can upregulate their Nrf2 activities.Fig. 6Elevated FAM129B expression correlates with expression of Nrf2 target genes and confers poor clinical outcome.(a) FAM129B mRNA levels in paired human breast tumor and adjacent normal tissues as determined by real-time RT-PCR and expressed in -Δ Ct, after subtracting the average of 3 reference genes, GAPDH, GUSB and UBC. (Paired t-test).(b) Kaplan–Meier relapse-free survival (RFS) curves of 125 breast cancer patients stratified by mean FAM129B mRNA levels in their tumor samples. (Log-rank test).(c, d) Kaplan–Meier plots of relapse-free survival (RFS) and overall survival (OS) stratified by median FAM129B expression in TCGA-BRAC (breast invasive carcinoma). (Log-rank test).(e, f) Kaplan–Meier plots of relapse-free survival (RFS) and overall survival (OS) stratified by quartile FAM129B expression in TCGA-LUSC (lung squamous cell carcinoma). (Log-rank test).(g) Correlation analysis of the expression of FAM129B and Nrf2 downstream genes in TCGA-BRAC and TCGA-LUSC expression profiles. (Spearman rank correlation test).Fig. 6Table 1Univariate and multivariate analyses of factors associated with relapse-free survival.Table 1Clinical variableUnivariate analysisaMultivariate analysisbHazard ratio (95% CI)cp valueHazard ratio (95% CI)p valueAge (years)1.0220 (1.0007–1.0438)0.0435ER (negative vs positive)0.4787 (0.2618–0.8753)0.0173PR (negative vs positive)0.5903 (0.3229–1.0789)0.0883HER-2 (negative vs positive)0.6637 (0.3631–1.2130)0.185Stage (I + II vs III + IV)2.5809 (1.3829–4.8617)0.0032.8646 (1.5162–5.4125)0.0013Subtype Luminal A vs Luminal B0.8782 (0.4194–1.8388)0.7318 Luminal A vs HER2 positive0.7136 (0.2381–2.1386)0.5489 Luminal A vs Triple negative2.8099 (1.2908–6.1166)0.00963.2317 (1.5868–6.5816)0.0013FAM129B (low vs high)2.0030 (1.1026–3.6388)0.02331.9674 (1.0655–3.6327)0.0314The bold indicate the numbers p values that are statistically significant.aUnivariate analysis, Cox proportional hazards regression.bMultivariate analysis, Cox proportional hazards regression.c95%CI: 95% confidence interval.

3.1Identification of FAM129B as a Keap1-interacting proteinIn a global proteomic analysis of 75 deubiquitinating enzymes [30] and a proteomic analysis of Keap1 associated proteins [31], FAM129B was listed as a potential candidate interacting protein for Keap1 by mass spectrometry. In this study, we performed a series of experiments to verify this association. First, 293T cells were co-transfected with Flag-FAM129B and HA-Keap1, followed by reciprocal immunoprecipitation experiments. As shown in Fig. 1a, HA-Keap1 was present in the Flag immunoprecipitates and Flag-FAM129B was detected in the HA immunoprecipitates, indicating that FAM129B and Keap1 were both present in the same complex. Next, fluorescence microscopy was used to determine whether FAM129B and Keap1 co-localized in mammalian cells. FAM129B and Keap1 were individually tagged with AcGFP1 and DsRed2, respectively. AcGFP1-FAM129B and DsRed2-Keap1 proteins were co-expressed in 293T cells, and their cellular localization was visualized by real-time live imaging system under a fluorescent microscope. As shown in Fig. 1b, ectopically expressed AcGFP1-FAM129B and DsRED2-Keap1 co-localized mainly in the cytoplasm (Pearson's Correlation = 0.9126). To further validate their interactions, we measured Förster resonance energy transfer (FRET) of AcGFP1 using multiphoton fluorescence lifetime imaging microscopy (FLIM). The fluorescence lifetime (t1/2) of cells expressing AcGFP1-FAM129B alone was distributed around 2.72 ns. Co-expression of DsRed2-Keap1 with AcGFP1-FAM129B produced a substantial shift in the distribution of AcGFP1t1/2 to 2.57 ns (Fig. 1c). The shorter t1/2 is indicative of FRET interaction between AcGFP1-FAM129B and DsRed2-Keap1. This finding is further supported by the demonstration of in situ interactions between FAM129B and Keap1 using the proximity ligation assay (PLA). With this method, when a pair of PLA probes binds two molecules in close proximity (< 16 nm), complementary DNA strands conjugated to PLA probes are ligated, amplified, and visualized as distinct points using a fluorescent probe [29]. In both Hs578T and HCT116 cells, strong PLA signals were observed for the FAM129B-Keap1 association (Fig. 1d). Moreover, a mammalian two-hybrid analysis was carried out in 293T cells. FAM129B was fused to the DNA-binding domain (DBD) of GAL4, while Keap1 was fused to the activation domain (AD) of VP16. Activation of the GAL4/Luc reporter (pG5) was observed in cells co-transfected with DBD-FAM129B and AD-Keap1 compared to cells transfected with the DBD construct alone (Fig. 1e). Taken together, these results validate that FAM129B and Keap1 can form a complex in cells.Fig. 1Identification of FAM129B as a Keap1-Interacting Protein.(a) Determination of interaction between FAM129B and Keap1 by co-immunoprecipitation. Lysates of 293T cells co-expressing Flag-FAM129B and HA-Keap1 were immunoprecipitated with anti-HA or anti-Flag antibodies. The total lysates and the immunoprecipitates were subjected to immunoblotting with anti-Flag and anti-HA antibodies for detecting FAM129B and Keap1, respectively.(b) Examination of co-localization of FAM129B and Keap1. 293T cells were co-transfected with AcGFP1-FAM129B and DsRed2-Keap1 expression constructs, and observed under a confocal microscope. Overlay images demonstrate co-localization of green FAM129B and red Keap1 molecules by a shift to orange.(c) FLIM-FRET measurements of the FAM129B-Keap1 interaction by determination of fluorescence lifetime of FRET between AcGFP1-FAM129B constructs and DsRed2-Keap1. Fluorescence lifetime of AcGFP1 was determined in 293T cells 48 h after transfection with either AcGFP1-FAM129B alone or in combination with DsRed2-Keap1. (left panels) Pictorial representations of the AcGFP1 lifetime; the color of the cell corresponds with the lifetime, which ranged from 2.1 to 2.8 ns. (right panels) Lifetime data from each pixel of the image, plotted on a graph.(d) Demonstration of interaction between FAM129B and Keap1 by proximity ligation assay (PLA) in Hs578T and HCT116 cells as determined by in situ PLA using anti-FAM129B and anti-Keap1 antibodies. Co-localization of FAM129B and Keap1 is reflected by red fluorescence in the images. Scale bar in (b-d), 20 μm.(e) Examination of the FAM129B-Keap1 interaction using a mammalian two-hybrid assay. 293T cells were co-transfected with a construct expressing Keap1 fused with AD of VP16, along with a construct expressing full-length FAM129B fused with the DBD of GAL4 and the pG5 luciferase reporter, followed by performance of the mammalian two-hybrid assay at 24 h post transfection. The firefly luciferase activities were normalized to Renilla luciferase activities. The data are presented as the mean values of triplicates ± SD. (For interpretation of the references to color in this figure legend, the reader is referred to the web version of this article.)Fig. 1

Identification of FAM129B as a Keap1-interacting protein

In a global proteomic analysis of 75 deubiquitinating enzymes [30] and a proteomic analysis of Keap1 associated proteins [31], FAM129B was listed as a potential candidate interacting protein for Keap1 by mass spectrometry. In this study, we performed a series of experiments to verify this association. First, 293T cells were co-transfected with Flag-FAM129B and HA-Keap1, followed by reciprocal immunoprecipitation experiments. As shown in Fig. 1a, HA-Keap1 was present in the Flag immunoprecipitates and Flag-FAM129B was detected in the HA immunoprecipitates, indicating that FAM129B and Keap1 were both present in the same complex. Next, fluorescence microscopy was used to determine whether FAM129B and Keap1 co-localized in mammalian cells. FAM129B and Keap1 were individually tagged with AcGFP1 and DsRed2, respectively. AcGFP1-FAM129B and DsRed2-Keap1 proteins were co-expressed in 293T cells, and their cellular localization was visualized by real-time live imaging system under a fluorescent microscope. As shown in Fig. 1b, ectopically expressed AcGFP1-FAM129B and DsRED2-Keap1 co-localized mainly in the cytoplasm (Pearson's Correlation = 0.9126). To further validate their interactions, we measured Förster resonance energy transfer (FRET) of AcGFP1 using multiphoton fluorescence lifetime imaging microscopy (FLIM). The fluorescence lifetime (t1/2) of cells expressing AcGFP1-FAM129B alone was distributed around 2.72 ns. Co-expression of DsRed2-Keap1 with AcGFP1-FAM129B produced a substantial shift in the distribution of AcGFP1t1/2 to 2.57 ns (Fig. 1c). The shorter t1/2 is indicative of FRET interaction between AcGFP1-FAM129B and DsRed2-Keap1. This finding is further supported by the demonstration of in situ interactions between FAM129B and Keap1 using the proximity ligation assay (PLA). With this method, when a pair of PLA probes binds two molecules in close proximity (< 16 nm), complementary DNA strands conjugated to PLA probes are ligated, amplified, and visualized as distinct points using a fluorescent probe [29]. In both Hs578T and HCT116 cells, strong PLA signals were observed for the FAM129B-Keap1 association (Fig. 1d). Moreover, a mammalian two-hybrid analysis was carried out in 293T cells. FAM129B was fused to the DNA-binding domain (DBD) of GAL4, while Keap1 was fused to the activation domain (AD) of VP16. Activation of the GAL4/Luc reporter (pG5) was observed in cells co-transfected with DBD-FAM129B and AD-Keap1 compared to cells transfected with the DBD construct alone (Fig. 1e). Taken together, these results validate that FAM129B and Keap1 can form a complex in cells.Fig. 1Identification of FAM129B as a Keap1-Interacting Protein.(a) Determination of interaction between FAM129B and Keap1 by co-immunoprecipitation. Lysates of 293T cells co-expressing Flag-FAM129B and HA-Keap1 were immunoprecipitated with anti-HA or anti-Flag antibodies. The total lysates and the immunoprecipitates were subjected to immunoblotting with anti-Flag and anti-HA antibodies for detecting FAM129B and Keap1, respectively.(b) Examination of co-localization of FAM129B and Keap1. 293T cells were co-transfected with AcGFP1-FAM129B and DsRed2-Keap1 expression constructs, and observed under a confocal microscope. Overlay images demonstrate co-localization of green FAM129B and red Keap1 molecules by a shift to orange.(c) FLIM-FRET measurements of the FAM129B-Keap1 interaction by determination of fluorescence lifetime of FRET between AcGFP1-FAM129B constructs and DsRed2-Keap1. Fluorescence lifetime of AcGFP1 was determined in 293T cells 48 h after transfection with either AcGFP1-FAM129B alone or in combination with DsRed2-Keap1. (left panels) Pictorial representations of the AcGFP1 lifetime; the color of the cell corresponds with the lifetime, which ranged from 2.1 to 2.8 ns. (right panels) Lifetime data from each pixel of the image, plotted on a graph.(d) Demonstration of interaction between FAM129B and Keap1 by proximity ligation assay (PLA) in Hs578T and HCT116 cells as determined by in situ PLA using anti-FAM129B and anti-Keap1 antibodies. Co-localization of FAM129B and Keap1 is reflected by red fluorescence in the images. Scale bar in (b-d), 20 μm.(e) Examination of the FAM129B-Keap1 interaction using a mammalian two-hybrid assay. 293T cells were co-transfected with a construct expressing Keap1 fused with AD of VP16, along with a construct expressing full-length FAM129B fused with the DBD of GAL4 and the pG5 luciferase reporter, followed by performance of the mammalian two-hybrid assay at 24 h post transfection. The firefly luciferase activities were normalized to Renilla luciferase activities. The data are presented as the mean values of triplicates ± SD. (For interpretation of the references to color in this figure legend, the reader is referred to the web version of this article.)Fig. 1

Identification of FAM129B as a Keap1-Interacting Protein.

(a) Determination of interaction between FAM129B and Keap1 by co-immunoprecipitation. Lysates of 293T cells co-expressing Flag-FAM129B and HA-Keap1 were immunoprecipitated with anti-HA or anti-Flag antibodies. The total lysates and the immunoprecipitates were subjected to immunoblotting with anti-Flag and anti-HA antibodies for detecting FAM129B and Keap1, respectively.

(b) Examination of co-localization of FAM129B and Keap1. 293T cells were co-transfected with AcGFP1-FAM129B and DsRed2-Keap1 expression constructs, and observed under a confocal microscope. Overlay images demonstrate co-localization of green FAM129B and red Keap1 molecules by a shift to orange.

(c) FLIM-FRET measurements of the FAM129B-Keap1 interaction by determination of fluorescence lifetime of FRET between AcGFP1-FAM129B constructs and DsRed2-Keap1. Fluorescence lifetime of AcGFP1 was determined in 293T cells 48 h after transfection with either AcGFP1-FAM129B alone or in combination with DsRed2-Keap1. (left panels) Pictorial representations of the AcGFP1 lifetime; the color of the cell corresponds with the lifetime, which ranged from 2.1 to 2.8 ns. (right panels) Lifetime data from each pixel of the image, plotted on a graph.

(d) Demonstration of interaction between FAM129B and Keap1 by proximity ligation assay (PLA) in Hs578T and HCT116 cells as determined by in situ PLA using anti-FAM129B and anti-Keap1 antibodies. Co-localization of FAM129B and Keap1 is reflected by red fluorescence in the images. Scale bar in (b-d), 20 μm.

(e) Examination of the FAM129B-Keap1 interaction using a mammalian two-hybrid assay. 293T cells were co-transfected with a construct expressing Keap1 fused with AD of VP16, along with a construct expressing full-length FAM129B fused with the DBD of GAL4 and the pG5 luciferase reporter, followed by performance of the mammalian two-hybrid assay at 24 h post transfection. The firefly luciferase activities were normalized to Renilla luciferase activities. The data are presented as the mean values of triplicates ± SD. (For interpretation of the references to color in this figure legend, the reader is referred to the web version of this article.)

3.2FAM129B interacts with the Kelch domain of Keap1 via its DLG and ETGE motifsTo delineate which Keap1 domains contribute to the interaction with FAM129B, we used mammalian two-hybrid assays to examine the abilities of truncation mutants of different Keap1 domains, including N, BTB, BACK, Kelch and Kelch/C, to bind FAM129B. We found that FAM129B could only interact with Keap1 truncation mutants containing the Kelch domain (Fig. 2a). Previously, the Kelch domain of Keap1 was reported to interact with the Neh2 domain of Nrf2 through its DLG and ETGE motifs [32]. Interestingly, C-terminus of FAM129B also contains DLG and ETGE motifs which were highly conserved in different species (Fig. 2b). Although tertiary structure of FAM129B protein remains unknown; we used computer modeling to predict binding of human Kelch domain of Keap1 to a peptide fragment (K705–S725) of FAM129B which covered the two motifs. Based on the structural information of Kelch-Neh2 complex available from Protein Data Bank (see EXPERIMENTEL MODEL), the DLG and the ETGE motifs of Nrf2 bound separately to the same binding pocket located at Kelch domain of Keap1via different orientation and molecular interactions. As shown in Fig. 2c, our computer modeling predicted two orientations of FAM129B K705–S725 peptide which could interact with the same area of Kelch domain of human Keap1 through DLG motif (blue color) and ETGE motifs (red color). Moreover, the Hydrogen-bond analysis revealed interactions between the Keap1-Kelch domain and FAM129B K705–S725 peptide. The DLG motif of FAM129B peptide sequence forms 5 hydrogen bonds (green-dashed lines in Fig. 2d) with the amino acids on Keap1-Kelch domain. On the other hand, as many as 12 hydrogen bonds might occur near the ETGE motif of FAM129B peptide sequence (Fig. 2e). We further assessed the binding energies of the molecular interactions for these two binding modes using the HotLig scoring program [24]. The estimated energy scores for the binding interactions through the DLG and the ETGE motifs were − 21.7 and − 24.6, respectively. To further characterize the FAM129B-Keap1 interaction biochemically, three FAM129B mutants were created by alanine replacement of amino-acid residues of its DLG and/or ETGE motifs. We used mammalian two-hybrid assays and co-immunoprecipitation to examine the binding abilities of the different FAM129B mutants to Keap1. The mammalian two-hybrid assays showed that mutation of either the DLG or ETGE motif abolished binding to Keap1 (Fig. 2f). However, immunoprecipitation analysis revealed that mutation of the DLG motif had a negligible effect on binding to Keap1, while mutation of the ETGE motif abolished binding (Fig. 2g). The latter finding is consistent with the reported lower binding affinity of the DLG motif ofNrf2 (1 × 106 M−1) for Keap1compared to the ETGE motif (2 × 108 M−1) [33]. These findings are also in agreement with our demonstration of higher binding affinity of the ETGE motif of FAM129B than the DLG motif for Keap1.Fig. 2FAM129B Interacts with the Kelch domain of Keap1 via its DLG and ETGE motifs.(a) Examination of FAM129B-Keap1 domain interactions by mammalian two-hybrid assay. Five different domains of Keap1, including an N-terminal region (amino acids 1–76), a BTB domain (amino acids 77–179), a BACK domain (amino acids 180–326), a Kelch domain (amino acids 327–611) and a Kelch/C domain (amino acids 327–624) were cloned and fused with the AD of VP16. 293T cells were co-transfected with a construct expressing Keap1 fused with AD of VP16, along with a construct expressing full-length FAM129B fused with the DBD of GAL4 and the pG5 luciferase reporter, followed by performance of the mammalian two-hybrid assay at 24 h post transfection. The firefly luciferase activities were normalized to Renilla luciferase activities. The data are presented as the mean values of triplicates ± SD.(b) Alignment of the Keap1 interacting motifs (DLG and ETGE motifs; black lines) of FAM129B homologs in various species. Black and grey boxes indicate identical amino acid residues with complete and partial conservation, respectively.(c) Predicted interactions of the Kelch domain with the DLG or the ETGE motif of the FAM129B peptide segment (705-KAVDLGPPKPSDQETGEQVSS-725). Both motifs can access the same binding site on Kelch domain.(d) Predicted hydrogen bonds (green dashed lines) and hydrophobic interactions (radiating spokes) near the DLG motif of the FAM129B for binding with the Kelch domain. The radiating spokes represent the atoms or amino acid residues that have hydrophobic interactions.(e) Molecular interactions near the ETGE motif involve more hydrogen bonds than DLG motif as shown in (d).(f) Examination of FAM129B-Keap1interaction through the FAM129B DLG motif or ETGE motif by mammalian two-hybrid assay. Three FAM129B mutants, M1, M2 and M3, were generated, in which the indicated amino acids in the DLG and/or ETGE motif were replaced with alanine residues. The data are presented as the mean values of triplicates ± SD.(g) Effects of mutations of the DLG and ETGE motifs of FAM129B on its interaction with Keap1. 293T cell lysates co-expressing wild type (WT) or mutant Flag-FAM129B and HA-Keap1 were immunoprecipitated with anti-Flag antibody. The total lysates and the immunoprecipitates were subjected to immunoblot analysis with both anti-Flag and anti-HA antibodies for detection of FAM129B and Keap1.Fig. 2

FAM129B interacts with the Kelch domain of Keap1 via its DLG and ETGE motifs

To delineate which Keap1 domains contribute to the interaction with FAM129B, we used mammalian two-hybrid assays to examine the abilities of truncation mutants of different Keap1 domains, including N, BTB, BACK, Kelch and Kelch/C, to bind FAM129B. We found that FAM129B could only interact with Keap1 truncation mutants containing the Kelch domain (Fig. 2a). Previously, the Kelch domain of Keap1 was reported to interact with the Neh2 domain of Nrf2 through its DLG and ETGE motifs [32]. Interestingly, C-terminus of FAM129B also contains DLG and ETGE motifs which were highly conserved in different species (Fig. 2b). Although tertiary structure of FAM129B protein remains unknown; we used computer modeling to predict binding of human Kelch domain of Keap1 to a peptide fragment (K705–S725) of FAM129B which covered the two motifs. Based on the structural information of Kelch-Neh2 complex available from Protein Data Bank (see EXPERIMENTEL MODEL), the DLG and the ETGE motifs of Nrf2 bound separately to the same binding pocket located at Kelch domain of Keap1via different orientation and molecular interactions. As shown in Fig. 2c, our computer modeling predicted two orientations of FAM129B K705–S725 peptide which could interact with the same area of Kelch domain of human Keap1 through DLG motif (blue color) and ETGE motifs (red color). Moreover, the Hydrogen-bond analysis revealed interactions between the Keap1-Kelch domain and FAM129B K705–S725 peptide. The DLG motif of FAM129B peptide sequence forms 5 hydrogen bonds (green-dashed lines in Fig. 2d) with the amino acids on Keap1-Kelch domain. On the other hand, as many as 12 hydrogen bonds might occur near the ETGE motif of FAM129B peptide sequence (Fig. 2e). We further assessed the binding energies of the molecular interactions for these two binding modes using the HotLig scoring program [24]. The estimated energy scores for the binding interactions through the DLG and the ETGE motifs were − 21.7 and − 24.6, respectively. To further characterize the FAM129B-Keap1 interaction biochemically, three FAM129B mutants were created by alanine replacement of amino-acid residues of its DLG and/or ETGE motifs. We used mammalian two-hybrid assays and co-immunoprecipitation to examine the binding abilities of the different FAM129B mutants to Keap1. The mammalian two-hybrid assays showed that mutation of either the DLG or ETGE motif abolished binding to Keap1 (Fig. 2f). However, immunoprecipitation analysis revealed that mutation of the DLG motif had a negligible effect on binding to Keap1, while mutation of the ETGE motif abolished binding (Fig. 2g). The latter finding is consistent with the reported lower binding affinity of the DLG motif ofNrf2 (1 × 106 M−1) for Keap1compared to the ETGE motif (2 × 108 M−1) [33]. These findings are also in agreement with our demonstration of higher binding affinity of the ETGE motif of FAM129B than the DLG motif for Keap1.Fig. 2FAM129B Interacts with the Kelch domain of Keap1 via its DLG and ETGE motifs.(a) Examination of FAM129B-Keap1 domain interactions by mammalian two-hybrid assay. Five different domains of Keap1, including an N-terminal region (amino acids 1–76), a BTB domain (amino acids 77–179), a BACK domain (amino acids 180–326), a Kelch domain (amino acids 327–611) and a Kelch/C domain (amino acids 327–624) were cloned and fused with the AD of VP16. 293T cells were co-transfected with a construct expressing Keap1 fused with AD of VP16, along with a construct expressing full-length FAM129B fused with the DBD of GAL4 and the pG5 luciferase reporter, followed by performance of the mammalian two-hybrid assay at 24 h post transfection. The firefly luciferase activities were normalized to Renilla luciferase activities. The data are presented as the mean values of triplicates ± SD.(b) Alignment of the Keap1 interacting motifs (DLG and ETGE motifs; black lines) of FAM129B homologs in various species. Black and grey boxes indicate identical amino acid residues with complete and partial conservation, respectively.(c) Predicted interactions of the Kelch domain with the DLG or the ETGE motif of the FAM129B peptide segment (705-KAVDLGPPKPSDQETGEQVSS-725). Both motifs can access the same binding site on Kelch domain.(d) Predicted hydrogen bonds (green dashed lines) and hydrophobic interactions (radiating spokes) near the DLG motif of the FAM129B for binding with the Kelch domain. The radiating spokes represent the atoms or amino acid residues that have hydrophobic interactions.(e) Molecular interactions near the ETGE motif involve more hydrogen bonds than DLG motif as shown in (d).(f) Examination of FAM129B-Keap1interaction through the FAM129B DLG motif or ETGE motif by mammalian two-hybrid assay. Three FAM129B mutants, M1, M2 and M3, were generated, in which the indicated amino acids in the DLG and/or ETGE motif were replaced with alanine residues. The data are presented as the mean values of triplicates ± SD.(g) Effects of mutations of the DLG and ETGE motifs of FAM129B on its interaction with Keap1. 293T cell lysates co-expressing wild type (WT) or mutant Flag-FAM129B and HA-Keap1 were immunoprecipitated with anti-Flag antibody. The total lysates and the immunoprecipitates were subjected to immunoblot analysis with both anti-Flag and anti-HA antibodies for detection of FAM129B and Keap1.Fig. 2

FAM129B Interacts with the Kelch domain of Keap1 via its DLG and ETGE motifs.

(a) Examination of FAM129B-Keap1 domain interactions by mammalian two-hybrid assay. Five different domains of Keap1, including an N-terminal region (amino acids 1–76), a BTB domain (amino acids 77–179), a BACK domain (amino acids 180–326), a Kelch domain (amino acids 327–611) and a Kelch/C domain (amino acids 327–624) were cloned and fused with the AD of VP16. 293T cells were co-transfected with a construct expressing Keap1 fused with AD of VP16, along with a construct expressing full-length FAM129B fused with the DBD of GAL4 and the pG5 luciferase reporter, followed by performance of the mammalian two-hybrid assay at 24 h post transfection. The firefly luciferase activities were normalized to Renilla luciferase activities. The data are presented as the mean values of triplicates ± SD.

(b) Alignment of the Keap1 interacting motifs (DLG and ETGE motifs; black lines) of FAM129B homologs in various species. Black and grey boxes indicate identical amino acid residues with complete and partial conservation, respectively.

(c) Predicted interactions of the Kelch domain with the DLG or the ETGE motif of the FAM129B peptide segment (705-KAVDLGPPKPSDQETGEQVSS-725). Both motifs can access the same binding site on Kelch domain.

(d) Predicted hydrogen bonds (green dashed lines) and hydrophobic interactions (radiating spokes) near the DLG motif of the FAM129B for binding with the Kelch domain. The radiating spokes represent the atoms or amino acid residues that have hydrophobic interactions.

(e) Molecular interactions near the ETGE motif involve more hydrogen bonds than DLG motif as shown in (d).

(f) Examination of FAM129B-Keap1interaction through the FAM129B DLG motif or ETGE motif by mammalian two-hybrid assay. Three FAM129B mutants, M1, M2 and M3, were generated, in which the indicated amino acids in the DLG and/or ETGE motif were replaced with alanine residues. The data are presented as the mean values of triplicates ± SD.

(g) Effects of mutations of the DLG and ETGE motifs of FAM129B on its interaction with Keap1. 293T cell lysates co-expressing wild type (WT) or mutant Flag-FAM129B and HA-Keap1 were immunoprecipitated with anti-Flag antibody. The total lysates and the immunoprecipitates were subjected to immunoblot analysis with both anti-Flag and anti-HA antibodies for detection of FAM129B and Keap1.

3.3FAM129B Dampens the Keap1-dependent ubiquitination of Nrf2 by competing with Nrf2 for Keap1 bindingThe“hinge and latch”model has been proposed for Nrf2 stabilization which involves high-affinity binding of ETGE motif (hinge) to fix Nrf2 to Keap1 and the low-affinity binding of DLG motif (latch) to lock down the Neh2 domain of Nrf2, thereby facilitating the ubiquitylation of lysine residues and constant degradation of Nrf2 [32,33]. Since FAM129B interacts with Keap1 through the same motif sequences (DLG and ETGE) as Nrf2, the Keap1 dimer might also recruit FAM129B through binding to its DLG and ETGE motifs, similar to the binding mode involved in the Keap1-Nrf2 complex (Fig. 3a, right). In support of this notion, in vitro binding experiments revealed that the amount of recombinant Flag-Nrf2 protein forming a complex with HA-Keap1 decreased in proportion to the recombinant Flag-FAM129B protein level (Fig. 3b). At the cellular level, FAM129B overexpression in 293T cells led to a marked decrease in Nrf2 ubiquititylation (Fig. 3c). As expected, the half-life of Nrf2 was shorter in FAM129B-siRNA cells than in control-siRNA cells (15.4 min versus 43.0 min) (Fig. 3d). In several cancer cell lines, FAM129B silencing by siRNA reduced the expression of endogenous FAM129B protein with a concomitant decrease in Nrf2 protein levels (Fig. 3e). In contrast, overexpression of FAM129B increased Nrf2 protein levels (Fig. 3f). These results indicate that FAM129B interfered with the Keap1-dependent ubiquitination of Nrf2 by competing with Nrf2 for Keap1 binding.Fig. 3FAM129B Dampens the Keap1-Dependent Ubiquitination of Nrf2 by Competing with Nrf2 for Keap1 Binding.(a) Keap1 proteins form a homodimer through the N-terminal BTB domains, while the C-terminal Kelch domains interact with Nrf2 protein. One Kelch domain recognizes the DLG motif and the other binds the ETGE motif of Nrf2. Similarly, Keap1 dimer might recruit FAM129B through binding to its DLG and ETGE motifs.(b) Competition between Nrf2 and FAM129B for binding to Keap1. Cell lysates from HA-Keap1 expressing 293T cells were immunoprecipitated with anti-HA antibody. The resulting immunoprecipitates were co-incubated with Nrf2 recombinant protein and increasing amounts of FAM129B recombinant protein, and then immunoblotted with the indicated antibodies.(c) Effects of FAM129B overexpression on the ubiquitination of Nrf2. 293T cells were transfected with the indicated plasmids followed by treatment with MG132 (10 μM) for 4 h. Cell lysates were subjected to an in vivo ubiquitination assay to detect the ubiquitin-conjugated Nrf2 protein. Lysates were denatured and immunoprecipitated with anti-Nrf2 antibody and blotted with anti-HA antibody. An aliquot of total lysate was analyzed using the indicated antibodies.(d) Effects of FAM129B silencing on the stability of Nrf2 protein. MDA-MB-231 cells were transfected with either control-siRNA or pooled FAM129B-siRNAs. At 72 h, cells were treated with cycloheximide (CHX), and cell lysates were harvested at the indicated time points for immunoblotting using the indicated antibodies. Nrf2 protein expression was normalized to β-actin, and the control group was set as 1.(e) Effects of FAM129B silencing on the expression of Nrf2 protein. Cells were transfected with either control-siRNA (C) or pooled FAM129B-siRNAs (F). Cell lysates were collected at 72 h post transfection and subjected to immunoblot analyses using the indicated antibodies.(f) Effects of FAM129B overexpression on the expression of Nrf2 protein. Cells were transfected with either empty vector or Flag-FAM129B. Cell lysates were collected at 48 h post transfection and subjected to immunoblot analyses using the indicated antibodies.(g) Effects of FAM129B silencing on the protein levels of Nrf2 downstream genes. Cells were transfected with either control-siRNA (C) or pooled FAM129B-siRNAs (F). Cell lysates were collected at 72 h post transfection and subjected to immunoblot analyses using the indicated antibodies.Fig. 3

FAM129B Dampens the Keap1-dependent ubiquitination of Nrf2 by competing with Nrf2 for Keap1 binding

The“hinge and latch”model has been proposed for Nrf2 stabilization which involves high-affinity binding of ETGE motif (hinge) to fix Nrf2 to Keap1 and the low-affinity binding of DLG motif (latch) to lock down the Neh2 domain of Nrf2, thereby facilitating the ubiquitylation of lysine residues and constant degradation of Nrf2 [32,33]. Since FAM129B interacts with Keap1 through the same motif sequences (DLG and ETGE) as Nrf2, the Keap1 dimer might also recruit FAM129B through binding to its DLG and ETGE motifs, similar to the binding mode involved in the Keap1-Nrf2 complex (Fig. 3a, right). In support of this notion, in vitro binding experiments revealed that the amount of recombinant Flag-Nrf2 protein forming a complex with HA-Keap1 decreased in proportion to the recombinant Flag-FAM129B protein level (Fig. 3b). At the cellular level, FAM129B overexpression in 293T cells led to a marked decrease in Nrf2 ubiquititylation (Fig. 3c). As expected, the half-life of Nrf2 was shorter in FAM129B-siRNA cells than in control-siRNA cells (15.4 min versus 43.0 min) (Fig. 3d). In several cancer cell lines, FAM129B silencing by siRNA reduced the expression of endogenous FAM129B protein with a concomitant decrease in Nrf2 protein levels (Fig. 3e). In contrast, overexpression of FAM129B increased Nrf2 protein levels (Fig. 3f). These results indicate that FAM129B interfered with the Keap1-dependent ubiquitination of Nrf2 by competing with Nrf2 for Keap1 binding.Fig. 3FAM129B Dampens the Keap1-Dependent Ubiquitination of Nrf2 by Competing with Nrf2 for Keap1 Binding.(a) Keap1 proteins form a homodimer through the N-terminal BTB domains, while the C-terminal Kelch domains interact with Nrf2 protein. One Kelch domain recognizes the DLG motif and the other binds the ETGE motif of Nrf2. Similarly, Keap1 dimer might recruit FAM129B through binding to its DLG and ETGE motifs.(b) Competition between Nrf2 and FAM129B for binding to Keap1. Cell lysates from HA-Keap1 expressing 293T cells were immunoprecipitated with anti-HA antibody. The resulting immunoprecipitates were co-incubated with Nrf2 recombinant protein and increasing amounts of FAM129B recombinant protein, and then immunoblotted with the indicated antibodies.(c) Effects of FAM129B overexpression on the ubiquitination of Nrf2. 293T cells were transfected with the indicated plasmids followed by treatment with MG132 (10 μM) for 4 h. Cell lysates were subjected to an in vivo ubiquitination assay to detect the ubiquitin-conjugated Nrf2 protein. Lysates were denatured and immunoprecipitated with anti-Nrf2 antibody and blotted with anti-HA antibody. An aliquot of total lysate was analyzed using the indicated antibodies.(d) Effects of FAM129B silencing on the stability of Nrf2 protein. MDA-MB-231 cells were transfected with either control-siRNA or pooled FAM129B-siRNAs. At 72 h, cells were treated with cycloheximide (CHX), and cell lysates were harvested at the indicated time points for immunoblotting using the indicated antibodies. Nrf2 protein expression was normalized to β-actin, and the control group was set as 1.(e) Effects of FAM129B silencing on the expression of Nrf2 protein. Cells were transfected with either control-siRNA (C) or pooled FAM129B-siRNAs (F). Cell lysates were collected at 72 h post transfection and subjected to immunoblot analyses using the indicated antibodies.(f) Effects of FAM129B overexpression on the expression of Nrf2 protein. Cells were transfected with either empty vector or Flag-FAM129B. Cell lysates were collected at 48 h post transfection and subjected to immunoblot analyses using the indicated antibodies.(g) Effects of FAM129B silencing on the protein levels of Nrf2 downstream genes. Cells were transfected with either control-siRNA (C) or pooled FAM129B-siRNAs (F). Cell lysates were collected at 72 h post transfection and subjected to immunoblot analyses using the indicated antibodies.Fig. 3

FAM129B Dampens the Keap1-Dependent Ubiquitination of Nrf2 by Competing with Nrf2 for Keap1 Binding.

(a) Keap1 proteins form a homodimer through the N-terminal BTB domains, while the C-terminal Kelch domains interact with Nrf2 protein. One Kelch domain recognizes the DLG motif and the other binds the ETGE motif of Nrf2. Similarly, Keap1 dimer might recruit FAM129B through binding to its DLG and ETGE motifs.

(b) Competition between Nrf2 and FAM129B for binding to Keap1. Cell lysates from HA-Keap1 expressing 293T cells were immunoprecipitated with anti-HA antibody. The resulting immunoprecipitates were co-incubated with Nrf2 recombinant protein and increasing amounts of FAM129B recombinant protein, and then immunoblotted with the indicated antibodies.

(c) Effects of FAM129B overexpression on the ubiquitination of Nrf2. 293T cells were transfected with the indicated plasmids followed by treatment with MG132 (10 μM) for 4 h. Cell lysates were subjected to an in vivo ubiquitination assay to detect the ubiquitin-conjugated Nrf2 protein. Lysates were denatured and immunoprecipitated with anti-Nrf2 antibody and blotted with anti-HA antibody. An aliquot of total lysate was analyzed using the indicated antibodies.

(d) Effects of FAM129B silencing on the stability of Nrf2 protein. MDA-MB-231 cells were transfected with either control-siRNA or pooled FAM129B-siRNAs. At 72 h, cells were treated with cycloheximide (CHX), and cell lysates were harvested at the indicated time points for immunoblotting using the indicated antibodies. Nrf2 protein expression was normalized to β-actin, and the control group was set as 1.

(e) Effects of FAM129B silencing on the expression of Nrf2 protein. Cells were transfected with either control-siRNA (C) or pooled FAM129B-siRNAs (F). Cell lysates were collected at 72 h post transfection and subjected to immunoblot analyses using the indicated antibodies.

(f) Effects of FAM129B overexpression on the expression of Nrf2 protein. Cells were transfected with either empty vector or Flag-FAM129B. Cell lysates were collected at 48 h post transfection and subjected to immunoblot analyses using the indicated antibodies.

(g) Effects of FAM129B silencing on the protein levels of Nrf2 downstream genes. Cells were transfected with either control-siRNA (C) or pooled FAM129B-siRNAs (F). Cell lysates were collected at 72 h post transfection and subjected to immunoblot analyses using the indicated antibodies.

3.4Nrf2 antioxidative signaling is required for FAM129B to inhibit ROSAs the master antioxidant regulator, Nrf2 governs the intracellular ROS level by regulating antioxidant gene products. Our findings that FAM129B modulates the level of Nrf2 protein suggest the possibility of involvement of FAM129B in regulating intracellular ROS. To address this possibility, we measured ROS levels in control and FAM129B knockdown cells, using flow cytometry after staining with dichlorofluorescein diacetate (DCF-DA). As shown in Fig. 4b, the expression of FAM129B protein was effectively silenced by pooled FAM129B-siRNAs in MDA-MB-231, Hs578T, HCT116 and H1299 cell lines. Interestingly, FAM129B silencing was associated with elevated basal intracellular ROS levels in these cells (Fig. 4a). As alterations in ROS levels can affect the intracellular redox state, we measured the ratio of reduced to oxidized glutathione (GSH/GSSG), which is a major indicator for oxidative stress in cells [34]. As expected, the GSH/GSSG ratio was markedly lowered in the FAM129B-siRNA-transfected cells (Fig. 4c). These results suggest that FAM129B may ameliorate intracellular oxidative stress in cancer cells.Fig. 4Nrf2 Antioxidative Signaling is Required for FAM129B to Inhibit ROS.(a) Effects of FAM129B silencing on ROS. At 72 h after transfection with the indicated siRNAs, intracellular ROS levels were measured by using a DCF/flow cytometric method.(b) Efficiency of FAM129B silencing by pooled FAM129B-siRNAs. Cells were transfected with control-siRNA (C) or pooled FAM129B-siRNAs (F). Cell lysates were collected at 72 h post transfection and subjected to immunoblot analyses using the indicated antibodies.(c) Effects of FAM129B silencing on the GSH/GSSG ratio. Cellular oxidative status was determined by measuring the GSH/GSSG ratio after siRNA-mediated FAM129B knockdown. Fold changes in GSH/GSSG ratio are presented as the mean values of triplicates ± SD.(d, e) Regulation of the ARE-reporter gene activity of Nrf2 by FAM129B. HCT116 cells were transfected with the indicated siRNAs (d) or constructs (e), along with the expression vector for 6 repeats of the ARE element of the firefly luciferase promoter and Renilla luciferase. The firefly luciferase activities were normalized to Renilla luciferase activities. The data are presented as the mean values of triplicates ± SD. ⁎p < .05, ⁎⁎⁎p < .001 (Student's t-test).(f) Effects of FAM129B silencing on the mRNA expression of Nrf2 and its downstream genes. Cells were transfected with the indicated siRNAs. After 72 h, mRNA was extracted for determination of the expression levels of Nrf2 and its downstream genes by real-time RT-PCR. Values were normalized to GAPDH. The data are presented as the mean values of triplicates ± SD.(g) Effects of FAM129B overexpression on the mRNA expression of Nrf2 and its downstream genes. Cells were transfected with either empty vector or flag-FAM129B. At 48 h, mRNA was extracted for determination of the expression levels of Nrf2 and its downstream genes by real-time RT-PCR. Values were normalized to GAPDH. The data are presented as the mean values of triplicates ± SD.Fig. 4To investigate the mechanisms underlying FAM129B-mediated ROS inhibition, the relationship between FAM129B and Nrf2 signaling was explored further. Nrf2 fulfills its function mainly by binding to the ARE (antioxidant response element) region of antioxidant gene promoters [35]. To investigate the effect of FAM129B on Nrf2 transcriptional activity, we used a luciferase reporter pGL3-ARE-luc to monitor the transcriptional activity of Nrf2 in HCT116 cells. Under FAM129B knockdown or overexpression conditions, the pGL3-ARE-luc activity was significantly decreased by FAM129B knockdown to about 70% of control values (Fig. 4d), but increased by about 3-fold by FAM129B overexpression (Fig. 4e). To confirm above findings, we also determined the mRNA levels of several Nrf2 downstream genes by real-time RT-PCR, including AKR1B10 AKR1C1, AKR1C2, AKR1C3 and NQO1 [36]. In general, FAM129B knockdown decreased the expression of most of these Nrf2 downstream genes, whereas FAM129B overexpression increased their levels in the two cell lines examined (Fig. 4f and g). The protein levels of Nrf2 downstream genes, NQO1 and HO-1, were also reduced in FAN129B silenced cells (Fig. 3g). Furthermore, ectopic expression of Nrf2 reversed the increase in ROS induced by FAM129B knockdown (Fig. S1a) and also rescued the reduction in the expression Nrf2 downstream genes upon FAM129B silencing (Fig. S1b). Collectively, these results further demonstrate that FAM129B inhibits ROS production by enhancing the transcriptional activity of Nrf2 with the consequent Nrf2-dependent antioxidant response.

Nrf2 antioxidative signaling is required for FAM129B to inhibit ROS

As the master antioxidant regulator, Nrf2 governs the intracellular ROS level by regulating antioxidant gene products. Our findings that FAM129B modulates the level of Nrf2 protein suggest the possibility of involvement of FAM129B in regulating intracellular ROS. To address this possibility, we measured ROS levels in control and FAM129B knockdown cells, using flow cytometry after staining with dichlorofluorescein diacetate (DCF-DA). As shown in Fig. 4b, the expression of FAM129B protein was effectively silenced by pooled FAM129B-siRNAs in MDA-MB-231, Hs578T, HCT116 and H1299 cell lines. Interestingly, FAM129B silencing was associated with elevated basal intracellular ROS levels in these cells (Fig. 4a). As alterations in ROS levels can affect the intracellular redox state, we measured the ratio of reduced to oxidized glutathione (GSH/GSSG), which is a major indicator for oxidative stress in cells [34]. As expected, the GSH/GSSG ratio was markedly lowered in the FAM129B-siRNA-transfected cells (Fig. 4c). These results suggest that FAM129B may ameliorate intracellular oxidative stress in cancer cells.Fig. 4Nrf2 Antioxidative Signaling is Required for FAM129B to Inhibit ROS.(a) Effects of FAM129B silencing on ROS. At 72 h after transfection with the indicated siRNAs, intracellular ROS levels were measured by using a DCF/flow cytometric method.(b) Efficiency of FAM129B silencing by pooled FAM129B-siRNAs. Cells were transfected with control-siRNA (C) or pooled FAM129B-siRNAs (F). Cell lysates were collected at 72 h post transfection and subjected to immunoblot analyses using the indicated antibodies.(c) Effects of FAM129B silencing on the GSH/GSSG ratio. Cellular oxidative status was determined by measuring the GSH/GSSG ratio after siRNA-mediated FAM129B knockdown. Fold changes in GSH/GSSG ratio are presented as the mean values of triplicates ± SD.(d, e) Regulation of the ARE-reporter gene activity of Nrf2 by FAM129B. HCT116 cells were transfected with the indicated siRNAs (d) or constructs (e), along with the expression vector for 6 repeats of the ARE element of the firefly luciferase promoter and Renilla luciferase. The firefly luciferase activities were normalized to Renilla luciferase activities. The data are presented as the mean values of triplicates ± SD. ⁎p < .05, ⁎⁎⁎p < .001 (Student's t-test).(f) Effects of FAM129B silencing on the mRNA expression of Nrf2 and its downstream genes. Cells were transfected with the indicated siRNAs. After 72 h, mRNA was extracted for determination of the expression levels of Nrf2 and its downstream genes by real-time RT-PCR. Values were normalized to GAPDH. The data are presented as the mean values of triplicates ± SD.(g) Effects of FAM129B overexpression on the mRNA expression of Nrf2 and its downstream genes. Cells were transfected with either empty vector or flag-FAM129B. At 48 h, mRNA was extracted for determination of the expression levels of Nrf2 and its downstream genes by real-time RT-PCR. Values were normalized to GAPDH. The data are presented as the mean values of triplicates ± SD.Fig. 4

Nrf2 Antioxidative Signaling is Required for FAM129B to Inhibit ROS.

(a) Effects of FAM129B silencing on ROS. At 72 h after transfection with the indicated siRNAs, intracellular ROS levels were measured by using a DCF/flow cytometric method.

(b) Efficiency of FAM129B silencing by pooled FAM129B-siRNAs. Cells were transfected with control-siRNA (C) or pooled FAM129B-siRNAs (F). Cell lysates were collected at 72 h post transfection and subjected to immunoblot analyses using the indicated antibodies.

(c) Effects of FAM129B silencing on the GSH/GSSG ratio. Cellular oxidative status was determined by measuring the GSH/GSSG ratio after siRNA-mediated FAM129B knockdown. Fold changes in GSH/GSSG ratio are presented as the mean values of triplicates ± SD.

(d, e) Regulation of the ARE-reporter gene activity of Nrf2 by FAM129B. HCT116 cells were transfected with the indicated siRNAs (d) or constructs (e), along with the expression vector for 6 repeats of the ARE element of the firefly luciferase promoter and Renilla luciferase. The firefly luciferase activities were normalized to Renilla luciferase activities. The data are presented as the mean values of triplicates ± SD. ⁎p < .05, ⁎⁎⁎p < .001 (Student's t-test).

(f) Effects of FAM129B silencing on the mRNA expression of Nrf2 and its downstream genes. Cells were transfected with the indicated siRNAs. After 72 h, mRNA was extracted for determination of the expression levels of Nrf2 and its downstream genes by real-time RT-PCR. Values were normalized to GAPDH. The data are presented as the mean values of triplicates ± SD.

(g) Effects of FAM129B overexpression on the mRNA expression of Nrf2 and its downstream genes. Cells were transfected with either empty vector or flag-FAM129B. At 48 h, mRNA was extracted for determination of the expression levels of Nrf2 and its downstream genes by real-time RT-PCR. Values were normalized to GAPDH. The data are presented as the mean values of triplicates ± SD.

To investigate the mechanisms underlying FAM129B-mediated ROS inhibition, the relationship between FAM129B and Nrf2 signaling was explored further. Nrf2 fulfills its function mainly by binding to the ARE (antioxidant response element) region of antioxidant gene promoters [35]. To investigate the effect of FAM129B on Nrf2 transcriptional activity, we used a luciferase reporter pGL3-ARE-luc to monitor the transcriptional activity of Nrf2 in HCT116 cells. Under FAM129B knockdown or overexpression conditions, the pGL3-ARE-luc activity was significantly decreased by FAM129B knockdown to about 70% of control values (Fig. 4d), but increased by about 3-fold by FAM129B overexpression (Fig. 4e). To confirm above findings, we also determined the mRNA levels of several Nrf2 downstream genes by real-time RT-PCR, including AKR1B10 AKR1C1, AKR1C2, AKR1C3 and NQO1 [36]. In general, FAM129B knockdown decreased the expression of most of these Nrf2 downstream genes, whereas FAM129B overexpression increased their levels in the two cell lines examined (Fig. 4f and g). The protein levels of Nrf2 downstream genes, NQO1 and HO-1, were also reduced in FAN129B silenced cells (Fig. 3g). Furthermore, ectopic expression of Nrf2 reversed the increase in ROS induced by FAM129B knockdown (Fig. S1a) and also rescued the reduction in the expression Nrf2 downstream genes upon FAM129B silencing (Fig. S1b). Collectively, these results further demonstrate that FAM129B inhibits ROS production by enhancing the transcriptional activity of Nrf2 with the consequent Nrf2-dependent antioxidant response.

3.5FAM129B silencing enhances sensitivity of cancer cells to oxaliplatinOur finding of the ability of FAM129B to stabilize Nrf2 protein implies that FAM129B may contribute to antioxidant response. To decipher the antioxidant function mediated by FAM129B, the effects of tert-butyl hydroperoxide (tBHP), an organic peroxide, on ROS levels and cell survival were determined. Treatment of FAM129B-siRNA-transfected cells with tBHP further augmented ROS elevation as compared to control-siRNA cells in each of the four cancer cell lines examined (Fig. 5a). FAM129B knockdown significantly increased the susceptibility of these cell lines to tBHP-induced cell death (Fig. 5b). These findings demonstrate that FAM129B has an antioxidant effect that is required for cellular protection from oxidative stress.Fig. 5FAM129B Silencing Enhances Sensitivity of Cancer Cells to Oxaliplatin.(a,b) Sensitivity of FAM129B-silenced cells to oxidative stress. Cells were transfected with control siRNA or pooled FAM129B-siRNAs. At 72 h, cells were treated with the indicated concentrations of tBHP for 24 h. Intracellular ROS levels were measured by using a DCF/flow cytometric method (a). Cell viability was analyzed by MTS assay (b). The data are presented as the mean values of triplicates ± SD. ⁎p < .05, ⁎⁎p < .01, ⁎⁎⁎p < .001 (Student's t-test).(c) Sensitivity of FAM129B silenced cells to oxaliplatin. Cells were transfected with control siRNA or pooled FAM129B-siRNAs. At 72 h, cells were treated with the indicated concentrations of oxaliplatin for 48 h, and cell viability was analyzed by MTS assay. The data are presented as the mean values of triplicates ± SD.⁎⁎p < .01, ⁎⁎⁎p < .001 (Student's t-test).(d–g) FAM129B silencing significantly enhanced the suppressant effect of oxaliplatin on tumor growth. NSG mice (n = 4 each group) were injected into mammary fat pad with si-Control or si-FAM129B MDA-MB-231 cells. After 2 days, the mice were treated with PBS only or oxaliplatin. Tumor volumes at the indicated time (d), and tumor images and weights at day 35 after injection of tumor cells are shown (e). NOD/SCID mice (n = 5 each group) were injected subcutaneously with si-Control or si-FAM129B HCT116 cells. When the tumor sizes reached about 100 mm3 the mice were treated with PBS only or oxaliplatin. Tumor volumes at the indicated time (f), and tumor images and weights at day 20 after injection of tumor cells are shown (g). The data are presented as means ± SD. ⁎p < .05, ⁎⁎p < .01, ⁎⁎⁎p < .001 (one-way ANOVA with Tukey's multiple comparison test).Fig. 5Induction of ROS-mediated damage in cancer cells by pharmacological agents such as platinum coordination compounds and anthracyclines that either promote ROS generation or disable the cellular antioxidant system has been considered as an effective therapeutic strategy to preferentially kill cancer cells [37]. In light of the antioxidative activity of FAM129B, we further examined whether FAM129B silencing would improve the response of cancer cells to the widely used chemotherapeutic drug oxaliplatin. As expected, siRNA-mediated FAM129B silencing significantly increased the sensitivity of MDA-MB-231, Hs578T, and HCT116 cells to oxaliplatin (Fig. 5c). These findings indicate that FAM129B depletion is a good strategy to augment the efficacy of oxaliplatin in cancer therapy.The in vivo effect of FAM129B on chemosensitivity was further examined using the MDA-MB-231 human breast cancer xenograft model in severe combined immune-deficient NSG mice. As shown in Fig. 5d, FAM129B knockdown did not suppress the tumor growth, in comparison to si-Controls (Fig. 5d and e). However, treatment with low dose oxaliplatin had a significantly greater inhibitory effect, as assessed by decreases in tumor volume, on si-FAM129B tumors than on si-Controls (p < .01 [one-way ANOVA with Tukey's multiple comparison test], Fig. 5d). Although oxaliplatin treatment had negligible effects on tumor weight in si-Controls, it significantly reduced tumor weight in the FAM129B knockdown group to 56% (p < .01 [one-way ANOVA with Tukey's multiple comparison test]) of si-Controls (Fig. 5e). To further confirm the role of FAM129B in chemosensitivity, a human colon cancer xenograft model of HCT116 cells in severe combined immune-deficient (NOD/SCID) mice was used. In contrast to results obtained with the MDA-MB-231 xenograft model, FAM129B knockdown significantly suppressed tumor growth, as compared to si-Controls (p < .05 [one-way ANOVA with Tukey's multiple comparison test], Fig. 5f). Treatment of si-Controls with oxaliplatin also decreased tumor growth. The combined inhibitory effect of FAM129B silencing and oxaliplatin treatment on tumor growth was greater than the effect of oxaliplatin alone (p < .05 [one-way ANOVA with Tukey's multiple comparison test]) or FAM129B silencing alone (p < .01 [one-way ANOVA with Tukey's multiple comparison test]). Furthermore, the average tumor weight in the FAM129B knockdown group was 76% of the si-Control group (p < .05 [one-way ANOVA with Tukey's multiple comparison test], Fig. 5g) and treatment of the si-FAM129B group with oxaliplatin further reduced tumor weight to 27% (p < .001 [one-way ANOVA with Tukey's multiple comparison test]) of si-Controls. In comparison, oxaliplatin treatment of the si-Control group decreased tumor weight tumor to 49% (p < .001 [one-way ANOVA with Tukey's multiple comparison test]) of the untreated si-Controls (Fig. 5g). Collectively, these results indicate that FAM129B silencing rendered xenografts more susceptible to oxaliplatin, showing a critical role of FAM129B in chemosensitivity in vivo.

FAM129B silencing enhances sensitivity of cancer cells to oxaliplatin

Our finding of the ability of FAM129B to stabilize Nrf2 protein implies that FAM129B may contribute to antioxidant response. To decipher the antioxidant function mediated by FAM129B, the effects of tert-butyl hydroperoxide (tBHP), an organic peroxide, on ROS levels and cell survival were determined. Treatment of FAM129B-siRNA-transfected cells with tBHP further augmented ROS elevation as compared to control-siRNA cells in each of the four cancer cell lines examined (Fig. 5a). FAM129B knockdown significantly increased the susceptibility of these cell lines to tBHP-induced cell death (Fig. 5b). These findings demonstrate that FAM129B has an antioxidant effect that is required for cellular protection from oxidative stress.Fig. 5FAM129B Silencing Enhances Sensitivity of Cancer Cells to Oxaliplatin.(a,b) Sensitivity of FAM129B-silenced cells to oxidative stress. Cells were transfected with control siRNA or pooled FAM129B-siRNAs. At 72 h, cells were treated with the indicated concentrations of tBHP for 24 h. Intracellular ROS levels were measured by using a DCF/flow cytometric method (a). Cell viability was analyzed by MTS assay (b). The data are presented as the mean values of triplicates ± SD. ⁎p < .05, ⁎⁎p < .01, ⁎⁎⁎p < .001 (Student's t-test).(c) Sensitivity of FAM129B silenced cells to oxaliplatin. Cells were transfected with control siRNA or pooled FAM129B-siRNAs. At 72 h, cells were treated with the indicated concentrations of oxaliplatin for 48 h, and cell viability was analyzed by MTS assay. The data are presented as the mean values of triplicates ± SD.⁎⁎p < .01, ⁎⁎⁎p < .001 (Student's t-test).(d–g) FAM129B silencing significantly enhanced the suppressant effect of oxaliplatin on tumor growth. NSG mice (n = 4 each group) were injected into mammary fat pad with si-Control or si-FAM129B MDA-MB-231 cells. After 2 days, the mice were treated with PBS only or oxaliplatin. Tumor volumes at the indicated time (d), and tumor images and weights at day 35 after injection of tumor cells are shown (e). NOD/SCID mice (n = 5 each group) were injected subcutaneously with si-Control or si-FAM129B HCT116 cells. When the tumor sizes reached about 100 mm3 the mice were treated with PBS only or oxaliplatin. Tumor volumes at the indicated time (f), and tumor images and weights at day 20 after injection of tumor cells are shown (g). The data are presented as means ± SD. ⁎p < .05, ⁎⁎p < .01, ⁎⁎⁎p < .001 (one-way ANOVA with Tukey's multiple comparison test).Fig. 5

FAM129B Silencing Enhances Sensitivity of Cancer Cells to Oxaliplatin.

(a,b) Sensitivity of FAM129B-silenced cells to oxidative stress. Cells were transfected with control siRNA or pooled FAM129B-siRNAs. At 72 h, cells were treated with the indicated concentrations of tBHP for 24 h. Intracellular ROS levels were measured by using a DCF/flow cytometric method (a). Cell viability was analyzed by MTS assay (b). The data are presented as the mean values of triplicates ± SD. ⁎p < .05, ⁎⁎p < .01, ⁎⁎⁎p < .001 (Student's t-test).

(c) Sensitivity of FAM129B silenced cells to oxaliplatin. Cells were transfected with control siRNA or pooled FAM129B-siRNAs. At 72 h, cells were treated with the indicated concentrations of oxaliplatin for 48 h, and cell viability was analyzed by MTS assay. The data are presented as the mean values of triplicates ± SD.⁎⁎p < .01, ⁎⁎⁎p < .001 (Student's t-test).

(d–g) FAM129B silencing significantly enhanced the suppressant effect of oxaliplatin on tumor growth. NSG mice (n = 4 each group) were injected into mammary fat pad with si-Control or si-FAM129B MDA-MB-231 cells. After 2 days, the mice were treated with PBS only or oxaliplatin. Tumor volumes at the indicated time (d), and tumor images and weights at day 35 after injection of tumor cells are shown (e). NOD/SCID mice (n = 5 each group) were injected subcutaneously with si-Control or si-FAM129B HCT116 cells. When the tumor sizes reached about 100 mm3 the mice were treated with PBS only or oxaliplatin. Tumor volumes at the indicated time (f), and tumor images and weights at day 20 after injection of tumor cells are shown (g). The data are presented as means ± SD. ⁎p < .05, ⁎⁎p < .01, ⁎⁎⁎p < .001 (one-way ANOVA with Tukey's multiple comparison test).

Induction of ROS-mediated damage in cancer cells by pharmacological agents such as platinum coordination compounds and anthracyclines that either promote ROS generation or disable the cellular antioxidant system has been considered as an effective therapeutic strategy to preferentially kill cancer cells [37]. In light of the antioxidative activity of FAM129B, we further examined whether FAM129B silencing would improve the response of cancer cells to the widely used chemotherapeutic drug oxaliplatin. As expected, siRNA-mediated FAM129B silencing significantly increased the sensitivity of MDA-MB-231, Hs578T, and HCT116 cells to oxaliplatin (Fig. 5c). These findings indicate that FAM129B depletion is a good strategy to augment the efficacy of oxaliplatin in cancer therapy.

The in vivo effect of FAM129B on chemosensitivity was further examined using the MDA-MB-231 human breast cancer xenograft model in severe combined immune-deficient NSG mice. As shown in Fig. 5d, FAM129B knockdown did not suppress the tumor growth, in comparison to si-Controls (Fig. 5d and e). However, treatment with low dose oxaliplatin had a significantly greater inhibitory effect, as assessed by decreases in tumor volume, on si-FAM129B tumors than on si-Controls (p < .01 [one-way ANOVA with Tukey's multiple comparison test], Fig. 5d). Although oxaliplatin treatment had negligible effects on tumor weight in si-Controls, it significantly reduced tumor weight in the FAM129B knockdown group to 56% (p < .01 [one-way ANOVA with Tukey's multiple comparison test]) of si-Controls (Fig. 5e). To further confirm the role of FAM129B in chemosensitivity, a human colon cancer xenograft model of HCT116 cells in severe combined immune-deficient (NOD/SCID) mice was used. In contrast to results obtained with the MDA-MB-231 xenograft model, FAM129B knockdown significantly suppressed tumor growth, as compared to si-Controls (p < .05 [one-way ANOVA with Tukey's multiple comparison test], Fig. 5f). Treatment of si-Controls with oxaliplatin also decreased tumor growth. The combined inhibitory effect of FAM129B silencing and oxaliplatin treatment on tumor growth was greater than the effect of oxaliplatin alone (p < .05 [one-way ANOVA with Tukey's multiple comparison test]) or FAM129B silencing alone (p < .01 [one-way ANOVA with Tukey's multiple comparison test]). Furthermore, the average tumor weight in the FAM129B knockdown group was 76% of the si-Control group (p < .05 [one-way ANOVA with Tukey's multiple comparison test], Fig. 5g) and treatment of the si-FAM129B group with oxaliplatin further reduced tumor weight to 27% (p < .001 [one-way ANOVA with Tukey's multiple comparison test]) of si-Controls. In comparison, oxaliplatin treatment of the si-Control group decreased tumor weight tumor to 49% (p < .001 [one-way ANOVA with Tukey's multiple comparison test]) of the untreated si-Controls (Fig. 5g). Collectively, these results indicate that FAM129B silencing rendered xenografts more susceptible to oxaliplatin, showing a critical role of FAM129B in chemosensitivity in vivo.

3.6Elevated FAM129B expression correlates with expression of Nrf2 target genes and confers poor clinical outcomeTo investigate the clinical relevance of FAM129B in human cancers, we determined FAM129B expression by RT-PCR in127 breast cancer specimens and their adjacent normal tissues. The clinicopathologic characteristics of these patients are shown in Supplemental Table S1. FAM129B expression was significantly higher in tumor than the adjacent normal tissue (p < .0001 [Paired t-test], Fig. 6a). More importantly, Kaplan–Meier survival analysis showed that patients with greater than mean expression levels of FAM129B had significantly worse relapse-free survival (RFS) than those with lower FAM129B expression (p = .0206 [Log rank test], Fig. 6b). Univariate Cox proportional hazard regression analyses were conducted. RFS correlated with age, ER negative status, advanced stage (stage III and IV), triple negative molecular type and high FAM129B expression (Table 1). The statistically significant factors for RFS that were identified using multivariate analyses are also presented in Table1. Advanced stage (stage III and IV), triple negative molecular type, and high FAM129B expression were identified as independent predictors of tumor recurrence (Table 1). These results suggest that in addition to the well-known negative impact of “clinical stage” and “triple-negative molecular type”, high FAM129B gene expression can serve as a prognostic marker for tumor recurrence (hazard ratio 1.967, 95% CI, 1.065–3.632). To further validate the prognostic value of FAM129B, we applied GEPIA (Gene Expression Profiling Interactive Analysis; http://gepia.cancer-pku.cn), for survival analysis of web-based TCGA (The Cancer Genome Atlas) cancer gene expression profiles [38]. Consistent with our findings, elevated FAM129 was found to correlate with poorer RFS in breast cancer (p < .001 [Log rank test], Fig. 6c). In addition, FAM129B overexpression had significant adverse impacts on overall survival in breast cancer (p < .001 [Log rank test], Fig. 6d), as well as RFS (p < .01 [Log rank test], Fig. 6e) and overall survival (p < .06 [Log rank test], Fig. 6f) in lung cancer. Thus, FAM129B may serve as a prognostic marker for clinical outcome of breast and lung cancer.Finally, to establish the physiological role of FAM129B in controlling Nrf2 transcriptional activity in breast and lung cancer, we examined whether the expression of FAM129B correlated to Nrf2 downstream genes. Using GEPIA analysis of TCGA database, we found FAM129B expression to be positively correlated with several Nrf2 downstream genes including ABCC1, ACOT7, MAFG and PTGR1 [39,40] in breast and lung cancer (Fig. 6g). These data support the notion that high FAM129B expression in tumors can upregulate their Nrf2 activities.Fig. 6Elevated FAM129B expression correlates with expression of Nrf2 target genes and confers poor clinical outcome.(a) FAM129B mRNA levels in paired human breast tumor and adjacent normal tissues as determined by real-time RT-PCR and expressed in -Δ Ct, after subtracting the average of 3 reference genes, GAPDH, GUSB and UBC. (Paired t-test).(b) Kaplan–Meier relapse-free survival (RFS) curves of 125 breast cancer patients stratified by mean FAM129B mRNA levels in their tumor samples. (Log-rank test).(c, d) Kaplan–Meier plots of relapse-free survival (RFS) and overall survival (OS) stratified by median FAM129B expression in TCGA-BRAC (breast invasive carcinoma). (Log-rank test).(e, f) Kaplan–Meier plots of relapse-free survival (RFS) and overall survival (OS) stratified by quartile FAM129B expression in TCGA-LUSC (lung squamous cell carcinoma). (Log-rank test).(g) Correlation analysis of the expression of FAM129B and Nrf2 downstream genes in TCGA-BRAC and TCGA-LUSC expression profiles. (Spearman rank correlation test).Fig. 6Table 1Univariate and multivariate analyses of factors associated with relapse-free survival.Table 1Clinical variableUnivariate analysisaMultivariate analysisbHazard ratio (95% CI)cp valueHazard ratio (95% CI)p valueAge (years)1.0220 (1.0007–1.0438)0.0435ER (negative vs positive)0.4787 (0.2618–0.8753)0.0173PR (negative vs positive)0.5903 (0.3229–1.0789)0.0883HER-2 (negative vs positive)0.6637 (0.3631–1.2130)0.185Stage (I + II vs III + IV)2.5809 (1.3829–4.8617)0.0032.8646 (1.5162–5.4125)0.0013Subtype Luminal A vs Luminal B0.8782 (0.4194–1.8388)0.7318 Luminal A vs HER2 positive0.7136 (0.2381–2.1386)0.5489 Luminal A vs Triple negative2.8099 (1.2908–6.1166)0.00963.2317 (1.5868–6.5816)0.0013FAM129B (low vs high)2.0030 (1.1026–3.6388)0.02331.9674 (1.0655–3.6327)0.0314The bold indicate the numbers p values that are statistically significant.aUnivariate analysis, Cox proportional hazards regression.bMultivariate analysis, Cox proportional hazards regression.c95%CI: 95% confidence interval.

Elevated FAM129B expression correlates with expression of Nrf2 target genes and confers poor clinical outcome

To investigate the clinical relevance of FAM129B in human cancers, we determined FAM129B expression by RT-PCR in127 breast cancer specimens and their adjacent normal tissues. The clinicopathologic characteristics of these patients are shown in Supplemental Table S1. FAM129B expression was significantly higher in tumor than the adjacent normal tissue (p < .0001 [Paired t-test], Fig. 6a). More importantly, Kaplan–Meier survival analysis showed that patients with greater than mean expression levels of FAM129B had significantly worse relapse-free survival (RFS) than those with lower FAM129B expression (p = .0206 [Log rank test], Fig. 6b). Univariate Cox proportional hazard regression analyses were conducted. RFS correlated with age, ER negative status, advanced stage (stage III and IV), triple negative molecular type and high FAM129B expression (Table 1). The statistically significant factors for RFS that were identified using multivariate analyses are also presented in Table1. Advanced stage (stage III and IV), triple negative molecular type, and high FAM129B expression were identified as independent predictors of tumor recurrence (Table 1). These results suggest that in addition to the well-known negative impact of “clinical stage” and “triple-negative molecular type”, high FAM129B gene expression can serve as a prognostic marker for tumor recurrence (hazard ratio 1.967, 95% CI, 1.065–3.632). To further validate the prognostic value of FAM129B, we applied GEPIA (Gene Expression Profiling Interactive Analysis; http://gepia.cancer-pku.cn), for survival analysis of web-based TCGA (The Cancer Genome Atlas) cancer gene expression profiles [38]. Consistent with our findings, elevated FAM129 was found to correlate with poorer RFS in breast cancer (p < .001 [Log rank test], Fig. 6c). In addition, FAM129B overexpression had significant adverse impacts on overall survival in breast cancer (p < .001 [Log rank test], Fig. 6d), as well as RFS (p < .01 [Log rank test], Fig. 6e) and overall survival (p < .06 [Log rank test], Fig. 6f) in lung cancer. Thus, FAM129B may serve as a prognostic marker for clinical outcome of breast and lung cancer.

Finally, to establish the physiological role of FAM129B in controlling Nrf2 transcriptional activity in breast and lung cancer, we examined whether the expression of FAM129B correlated to Nrf2 downstream genes. Using GEPIA analysis of TCGA database, we found FAM129B expression to be positively correlated with several Nrf2 downstream genes including ABCC1, ACOT7, MAFG and PTGR1 [39,40] in breast and lung cancer (Fig. 6g). These data support the notion that high FAM129B expression in tumors can upregulate their Nrf2 activities.Fig. 6Elevated FAM129B expression correlates with expression of Nrf2 target genes and confers poor clinical outcome.(a) FAM129B mRNA levels in paired human breast tumor and adjacent normal tissues as determined by real-time RT-PCR and expressed in -Δ Ct, after subtracting the average of 3 reference genes, GAPDH, GUSB and UBC. (Paired t-test).(b) Kaplan–Meier relapse-free survival (RFS) curves of 125 breast cancer patients stratified by mean FAM129B mRNA levels in their tumor samples. (Log-rank test).(c, d) Kaplan–Meier plots of relapse-free survival (RFS) and overall survival (OS) stratified by median FAM129B expression in TCGA-BRAC (breast invasive carcinoma). (Log-rank test).(e, f) Kaplan–Meier plots of relapse-free survival (RFS) and overall survival (OS) stratified by quartile FAM129B expression in TCGA-LUSC (lung squamous cell carcinoma). (Log-rank test).(g) Correlation analysis of the expression of FAM129B and Nrf2 downstream genes in TCGA-BRAC and TCGA-LUSC expression profiles. (Spearman rank correlation test).Fig. 6Table 1Univariate and multivariate analyses of factors associated with relapse-free survival.Table 1Clinical variableUnivariate analysisaMultivariate analysisbHazard ratio (95% CI)cp valueHazard ratio (95% CI)p valueAge (years)1.0220 (1.0007–1.0438)0.0435ER (negative vs positive)0.4787 (0.2618–0.8753)0.0173PR (negative vs positive)0.5903 (0.3229–1.0789)0.0883HER-2 (negative vs positive)0.6637 (0.3631–1.2130)0.185Stage (I + II vs III + IV)2.5809 (1.3829–4.8617)0.0032.8646 (1.5162–5.4125)0.0013Subtype Luminal A vs Luminal B0.8782 (0.4194–1.8388)0.7318 Luminal A vs HER2 positive0.7136 (0.2381–2.1386)0.5489 Luminal A vs Triple negative2.8099 (1.2908–6.1166)0.00963.2317 (1.5868–6.5816)0.0013FAM129B (low vs high)2.0030 (1.1026–3.6388)0.02331.9674 (1.0655–3.6327)0.0314The bold indicate the numbers p values that are statistically significant.aUnivariate analysis, Cox proportional hazards regression.bMultivariate analysis, Cox proportional hazards regression.c95%CI: 95% confidence interval.

Elevated FAM129B expression correlates with expression of Nrf2 target genes and confers poor clinical outcome.

(a) FAM129B mRNA levels in paired human breast tumor and adjacent normal tissues as determined by real-time RT-PCR and expressed in -Δ Ct, after subtracting the average of 3 reference genes, GAPDH, GUSB and UBC. (Paired t-test).

(b) Kaplan–Meier relapse-free survival (RFS) curves of 125 breast cancer patients stratified by mean FAM129B mRNA levels in their tumor samples. (Log-rank test).

(c, d) Kaplan–Meier plots of relapse-free survival (RFS) and overall survival (OS) stratified by median FAM129B expression in TCGA-BRAC (breast invasive carcinoma). (Log-rank test).

(e, f) Kaplan–Meier plots of relapse-free survival (RFS) and overall survival (OS) stratified by quartile FAM129B expression in TCGA-LUSC (lung squamous cell carcinoma). (Log-rank test).

(g) Correlation analysis of the expression of FAM129B and Nrf2 downstream genes in TCGA-BRAC and TCGA-LUSC expression profiles. (Spearman rank correlation test).

Univariate and multivariate analyses of factors associated with relapse-free survival.

The bold indicate the numbers p values that are statistically significant.

Univariate analysis, Cox proportional hazards regression.

Multivariate analysis, Cox proportional hazards regression.

95%CI: 95% confidence interval.

4DiscussionIn this report, we have provided the first evidence that FAM129B plays an important role in regulating the antioxidant capacity of cancer cells via stabilization of Nrf2 protein by competing with Nrf2 for Keap1 binding through both the DLG and ETGE motifs of FAM129B as shown in the schematic diagram (Fig. 7). We also demonstrate that high expression of FAM129B is associated with chemosensitivity and adverse clinical outcome of breast cancer and possibly other types of cancer.Fig. 7A schematic diagram of regulation of Nrf2 activity by FAM129B in cancer cells.The schema illustrates how FAM129B modulates ROS and chemosensitivity. Overexpression of FAM129B in cancer promotes Nrf2 activity by reducing its ubiquitination through competition with Nrf2 for Keap1 binding, allowing nuclear translocation and transcriptional activation of Nrf2 to drive downstream antioxidant genes, which ultimately suppresses ROS production and reduces chemosensitivity.Fig. 7It has been shown that the unique association mechanism between Nrf2 and Keap1 involves two Keap1 molecules interacting with one Nrf2 molecule through its DLG and ETGE motifs. This two-site binding facilitates the ubiquitylation and degradation of Nrf2 [32,33]. A hinge and latch model was proposed as the stress-sensing mechanism, in which the low-affinity DLG motif acts as a latch for turning the ubiquitylation of Nrf2 on or off [32]. Furthermore, a recent study proposed that the Keap1-Nrf2 complex is in dynamic flux. In cells, the open” conformation, in which Nrf2 binds with Keap1 via the DLG motif only, coexists with the “closed” conformation, in which Nrf2 binds with Keap1 via both the DLG and ETGE motifs. Cycling from an open to a closed conformation allows ubiquitination and degradation of Nrf2 with subsequent regeneration of free Keap1 [41,42]. As discussed above, FAM129B may interfere with the dynamics of the Keap1-Nrf2 complex, leading to insufficient free Keap1 for binding Nrf2. Thus, de novo-synthesized Nrf2 will be free to translocate to the nucleus for activation of downstream genes. In addition, recent studies have identified six proteins, p21, p62, WTX, PALB2, DPP3, and iASPP that bind Keap1 or Nrf2 and, thereby, inhibit Nrf2 ubiquitination [31,[42], [43], [44], [45], [46]]. Of these, WTX, PALB2, and DPP3 employ an ETGE motif to bind Keap1 directly, thus displacing and stabilizing Nrf2. Unlike most of the Keap1-Nrf2 complex disruptor proteins, FAM129B contains both ETGE and DLG motifs. Therefore, FAM129B may interfere with the binding of Nrf2 to Keap1 more efficiently than proteins containing an ETGE motif only.The Nrf2-Keap1 antioxidant response pathway plays a crucial role in chemoprevention and cancer therapy. High levels of Nrf2 protein in cancer cells confer resistance to chemotherapeutic drugs such as cisplatin, doxorubicin and etoposide [47,48].The effector function of Nrf2 may contribute to chemotherapy resistance by several mechanisms: 1) suppression of oxidative stress, which is an important aspect of the cytotoxicity of chemotherapy; 2) drug detoxification by glutathione and other conjugating mechanisms; 3) transcriptional up-regulation of the multidrug resistance genes, which can lower effective drug concentrations [49]. In view of the high frequency of tumors displaying Nrf2 hyperactivation, Nrf2 has been considered as a potential pharmacological target. Unfortunately, it has been difficult to develop specific and effective Nrf2 inhibitors since Nrf2 belongs to a big family of basic leucine zipper transcription factors, which are involved in the regulation of diverse and critical biological functions [42,50]. On the other hand, focusing on uncovering Nrf2 regulatory mechanisms via protein-protein interactions may provide an alternative to inhibit Nrf2 directly. Our finding that FAM129B-silencing enhanced the sensitivity of cancer cells to oxaliplatin provided an additional mechanism of Nrf2-associated drug resistance through competition between FAM129B and Nrf2 for Keap1 binding. Thus, FAM129B is an attractive target, especially in view of our findings of higher FAM129B levels in many cancers, as compared to their normal tissue counterparts. These findings suggest that FAM129B depletion may be considered as a novel strategy for improving treatment outcome in cancers.In our xenograft models, FAM129B silencing rendered xenografts more susceptible to oxaliplatin, leading to reduction of tumor weight and volume. We should point out that the gene silencing effect via siRNA will dwindle by 1 week. Since HCT116 and MDA-MB-231 tumors were harvested on day 20 and 35 respectively, in our xenograft models, we found no difference in the expression of level of Nrf2 and its downstream genes in HCT116 xenograft tumors between si-Control and si-FAM129B group via real-time PCR (Fig. S2). This suggests that the impacts of FAM129B on chemosensitivity must have occurred during the initial week after silencing of FAM129B by siRNA approach.Given the above findings that FAM129B contributes to Nrf2 stability and reduces chemosensitivity, and the previous reports of pro-tumorigenic activity of FAM129B, it is likely that FAM129B expression level may influence patient outcome. Indeed, we demonstrated that high expression of FAM129B was a poor prognostic factor for breast cancer and lung cancer by examination of clinical specimens of breast cancer and data mining. To our knowledge, this is the first report on the prognostic significance of high expression of FAM129B. Furthermore, FAM129B expression positively associated with the expression of NRF2 downstream genes, in line with our in vitro finding of stabilization of Nrf2 by FAM129B. Since several cancer types have been found to exhibit constitutive Nrf2 activation without genomic alterations in Nrf2 and Keap1, our data collectively support the notion that FAM129B may not only provide a novel mechanism for promoting Nrf2 activation, but also serve as a potential therapeutic target and prognosis factor for multiple different cancer types.The following are the supplementary data related to this articleSupplementaryFigure legendSupplementary Table S1Supplementary FigureImage 1Supplementary Table S1 Clinical and pathological characteristics of 126 breast cancerImage 2

In this report, we have provided the first evidence that FAM129B plays an important role in regulating the antioxidant capacity of cancer cells via stabilization of Nrf2 protein by competing with Nrf2 for Keap1 binding through both the DLG and ETGE motifs of FAM129B as shown in the schematic diagram (Fig. 7). We also demonstrate that high expression of FAM129B is associated with chemosensitivity and adverse clinical outcome of breast cancer and possibly other types of cancer.Fig. 7A schematic diagram of regulation of Nrf2 activity by FAM129B in cancer cells.The schema illustrates how FAM129B modulates ROS and chemosensitivity. Overexpression of FAM129B in cancer promotes Nrf2 activity by reducing its ubiquitination through competition with Nrf2 for Keap1 binding, allowing nuclear translocation and transcriptional activation of Nrf2 to drive downstream antioxidant genes, which ultimately suppresses ROS production and reduces chemosensitivity.Fig. 7

A schematic diagram of regulation of Nrf2 activity by FAM129B in cancer cells.

The schema illustrates how FAM129B modulates ROS and chemosensitivity. Overexpression of FAM129B in cancer promotes Nrf2 activity by reducing its ubiquitination through competition with Nrf2 for Keap1 binding, allowing nuclear translocation and transcriptional activation of Nrf2 to drive downstream antioxidant genes, which ultimately suppresses ROS production and reduces chemosensitivity.

It has been shown that the unique association mechanism between Nrf2 and Keap1 involves two Keap1 molecules interacting with one Nrf2 molecule through its DLG and ETGE motifs. This two-site binding facilitates the ubiquitylation and degradation of Nrf2 [32,33]. A hinge and latch model was proposed as the stress-sensing mechanism, in which the low-affinity DLG motif acts as a latch for turning the ubiquitylation of Nrf2 on or off [32]. Furthermore, a recent study proposed that the Keap1-Nrf2 complex is in dynamic flux. In cells, the open” conformation, in which Nrf2 binds with Keap1 via the DLG motif only, coexists with the “closed” conformation, in which Nrf2 binds with Keap1 via both the DLG and ETGE motifs. Cycling from an open to a closed conformation allows ubiquitination and degradation of Nrf2 with subsequent regeneration of free Keap1 [41,42]. As discussed above, FAM129B may interfere with the dynamics of the Keap1-Nrf2 complex, leading to insufficient free Keap1 for binding Nrf2. Thus, de novo-synthesized Nrf2 will be free to translocate to the nucleus for activation of downstream genes. In addition, recent studies have identified six proteins, p21, p62, WTX, PALB2, DPP3, and iASPP that bind Keap1 or Nrf2 and, thereby, inhibit Nrf2 ubiquitination [31,[42], [43], [44], [45], [46]]. Of these, WTX, PALB2, and DPP3 employ an ETGE motif to bind Keap1 directly, thus displacing and stabilizing Nrf2. Unlike most of the Keap1-Nrf2 complex disruptor proteins, FAM129B contains both ETGE and DLG motifs. Therefore, FAM129B may interfere with the binding of Nrf2 to Keap1 more efficiently than proteins containing an ETGE motif only.

The Nrf2-Keap1 antioxidant response pathway plays a crucial role in chemoprevention and cancer therapy. High levels of Nrf2 protein in cancer cells confer resistance to chemotherapeutic drugs such as cisplatin, doxorubicin and etoposide [47,48].The effector function of Nrf2 may contribute to chemotherapy resistance by several mechanisms: 1) suppression of oxidative stress, which is an important aspect of the cytotoxicity of chemotherapy; 2) drug detoxification by glutathione and other conjugating mechanisms; 3) transcriptional up-regulation of the multidrug resistance genes, which can lower effective drug concentrations [49]. In view of the high frequency of tumors displaying Nrf2 hyperactivation, Nrf2 has been considered as a potential pharmacological target. Unfortunately, it has been difficult to develop specific and effective Nrf2 inhibitors since Nrf2 belongs to a big family of basic leucine zipper transcription factors, which are involved in the regulation of diverse and critical biological functions [42,50]. On the other hand, focusing on uncovering Nrf2 regulatory mechanisms via protein-protein interactions may provide an alternative to inhibit Nrf2 directly. Our finding that FAM129B-silencing enhanced the sensitivity of cancer cells to oxaliplatin provided an additional mechanism of Nrf2-associated drug resistance through competition between FAM129B and Nrf2 for Keap1 binding. Thus, FAM129B is an attractive target, especially in view of our findings of higher FAM129B levels in many cancers, as compared to their normal tissue counterparts. These findings suggest that FAM129B depletion may be considered as a novel strategy for improving treatment outcome in cancers.

In our xenograft models, FAM129B silencing rendered xenografts more susceptible to oxaliplatin, leading to reduction of tumor weight and volume. We should point out that the gene silencing effect via siRNA will dwindle by 1 week. Since HCT116 and MDA-MB-231 tumors were harvested on day 20 and 35 respectively, in our xenograft models, we found no difference in the expression of level of Nrf2 and its downstream genes in HCT116 xenograft tumors between si-Control and si-FAM129B group via real-time PCR (Fig. S2). This suggests that the impacts of FAM129B on chemosensitivity must have occurred during the initial week after silencing of FAM129B by siRNA approach.

Given the above findings that FAM129B contributes to Nrf2 stability and reduces chemosensitivity, and the previous reports of pro-tumorigenic activity of FAM129B, it is likely that FAM129B expression level may influence patient outcome. Indeed, we demonstrated that high expression of FAM129B was a poor prognostic factor for breast cancer and lung cancer by examination of clinical specimens of breast cancer and data mining. To our knowledge, this is the first report on the prognostic significance of high expression of FAM129B. Furthermore, FAM129B expression positively associated with the expression of NRF2 downstream genes, in line with our in vitro finding of stabilization of Nrf2 by FAM129B. Since several cancer types have been found to exhibit constitutive Nrf2 activation without genomic alterations in Nrf2 and Keap1, our data collectively support the notion that FAM129B may not only provide a novel mechanism for promoting Nrf2 activation, but also serve as a potential therapeutic target and prognosis factor for multiple different cancer types.

The following are the supplementary data related to this articleSupplementaryFigure legendSupplementary Table S1Supplementary FigureImage 1Supplementary Table S1 Clinical and pathological characteristics of 126 breast cancerImage 2

Supplementary

Figure legend

Supplementary Figure

Supplementary Table S1 Clinical and pathological characteristics of 126 breast cancer
