---
pmid: '21722948'
title: Thymine DNA glycosylase is essential for active DNA demethylation by linked
  deamination-base excision repair.
authors:
- Cortellino S
- Xu J
- Sannai M
- Moore R
- Caretti E
- Cigliano A
- Le Coz M
- Devarajan K
- Wessels A
- Soprano D
- Abramowitz LK
- Bartolomei MS
- Rambow F
- Bassi MR
- Bruno T
- Fanciulli M
- Renner C
- Klein-Szanto AJ
- Matsumoto Y
- Kobi D
- Davidson I
- Alberti C
- Larue L
- Bellacosa A
journal: Cell
year: '2011'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC3230223
doi: 10.1016/j.cell.2011.06.020
---

# Thymine DNA glycosylase is essential for active DNA demethylation by linked deamination-base excision repair.
**Authors:** Cortellino S, Xu J, Sannai M, Moore R, Caretti E, Cigliano A, Le Coz M, Devarajan K, Wessels A, Soprano D, Abramowitz LK, Bartolomei MS, Rambow F, Bassi MR, Bruno T, Fanciulli M, Renner C, Klein-Szanto AJ, Matsumoto Y, Kobi D, Davidson I, Alberti C, Larue L, Bellacosa A
**Journal:** Cell (2011)
**DOI:** [10.1016/j.cell.2011.06.020](https://doi.org/10.1016/j.cell.2011.06.020)
**PMC:** [PMC3230223](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3230223/)

## Abstract

1. Cell. 2011 Jul 8;146(1):67-79. doi: 10.1016/j.cell.2011.06.020. Epub 2011 Jun 
30.

Thymine DNA glycosylase is essential for active DNA demethylation by linked 
deamination-base excision repair.

Cortellino S(1), Xu J, Sannai M, Moore R, Caretti E, Cigliano A, Le Coz M, 
Devarajan K, Wessels A, Soprano D, Abramowitz LK, Bartolomei MS, Rambow F, Bassi 
MR, Bruno T, Fanciulli M, Renner C, Klein-Szanto AJ, Matsumoto Y, Kobi D, 
Davidson I, Alberti C, Larue L, Bellacosa A.

Author information:
(1)Cancer Biology Program and Epigenetics and Progenitor Cells Keystone Program, 
Fox Chase Cancer Center, Philadelphia, PA 19111, USA.

DNA methylation is a major epigenetic mechanism for gene silencing. Whereas 
methyltransferases mediate cytosine methylation, it is less clear how 
unmethylated regions in mammalian genomes are protected from de novo methylation 
and whether an active demethylating activity is involved. Here, we show that 
either knockout or catalytic inactivation of the DNA repair enzyme thymine DNA 
glycosylase (TDG) leads to embryonic lethality in mice. TDG is necessary for 
recruiting p300 to retinoic acid (RA)-regulated promoters, protection of CpG 
islands from hypermethylation, and active demethylation of tissue-specific 
developmentally and hormonally regulated promoters and enhancers. TDG interacts 
with the deaminase AID and the damage response protein GADD45a. These findings 
highlight a dual role for TDG in promoting proper epigenetic states during 
development and suggest a two-step mechanism for DNA demethylation in mammals, 
whereby 5-methylcytosine and 5-hydroxymethylcytosine are first deaminated by AID 
to thymine and 5-hydroxymethyluracil, respectively, followed by TDG-mediated 
thymine and 5-hydroxymethyluracil excision repair.

Copyright © 2011 Elsevier Inc. All rights reserved.

DOI: 10.1016/j.cell.2011.06.020
PMCID: PMC3230223
PMID: 21722948 [Indexed for MEDLINE]

## Full Text

Introduction

Cytosine methylation - the formation of 5-methylcytosine (5mC) at CpG sites - is an important epigenetic modification used by mammals to mediate transcriptional regulation, including transcriptional repression, X-chromosome inactivation, imprinting, and suppression of parasitic sequences ( Bird, 1992 ; Kass et al., 1997 ; Siegfried and Cedar, 1997 ). The establishment and maintenance of the correct DNA methylation patterns at CpG sites is essential in mammals during development, gametogenesis and differentiation of somatic tissues. Indeed alterations in DNA methylation patterns, with the associated chromatin changes, have profound consequences, as demonstrated by embryonic lethality in the absence of DNA methylation ( Li et al., 1992 ; Okano et al., 1999 ), developmental defects and accelerated aging in cloned mammals ( Rideout et al., 2001 ), and characteristic epigenetic changes in cancer, such as global genome hypomethylation and tumor suppressor gene hypermethylation ( Feinberg and Tycko, 2004 ; Jones and Laird, 1999 ).

While DNA methylation is mediated by de novo DNA methyltransferases (DNMT3a and DNMT3b) that act on unmethylated DNA and maintenance DNA methyltransferases (DNMT1) that act on newly replicated, transiently hemimethylated DNA, the demethylating activities or processes that remove methylation marks in mammals are largely unknown. Indeed, it has been controversial as to whether demethylation is an active process in mammals ( Ooi and Bestor, 2008 ) and which mechanisms are involved ( Wu and Zhang, 2010 ).

Demethylation can occur passively due to replication in the absence of re-methylation, with consequent dilution of this modification. However, there is evidence supporting the occurrence of active demethylation in mammals, including demethylation of the paternal genome shortly after fertilization ( Mayer et al., 2000 ; Oswald et al., 2000 ), demethylation to erase and reset imprinting in primordial germ cells ( Reik et al., 2001 ; Surani et al., 2007 ), demethylation during somatic differentiation of the developing embryo to establish tissue-specific gene expression patterns ( Kress et al., 2006 ; Niehrs, 2009 ) and during gene activation in adult kidney ( Kim et al., 2009 ) and brain ( Ma et al., 2009 ). In addition, it is generally thought that active transcription contributes to the maintenance of the unmethylated state of promoter-associated CpG-rich sequences known as CpG islands, but the molecular details of protection from hypermethylation and the potential involvement of an active demethylation process are unknown ( Illingworth and Bird, 2009 ).

Accumulating evidence in non-mammalian model organisms point to the involvement of DNA repair mechanisms in active demethylation ( Gehring et al., 2009 ; Niehrs, 2009 ). In Arabidopsis, the base excision repair (BER) proteins Demeter and ROS1 affect demethylation by directly removing 5mC through their glycosylase activities ( Gehring et al., 2006 ; Morales-Ruiz et al., 2006 ). In Xenopus, demethylation has been reported to be initiated by the genome stability protein Gadd45a (growth arrest and DNA damage-inducible protein 45 alpha) in a process dependent on the nucleotide excision repair protein XPG ( Barreto et al., 2007 ); however the role of mammalian GADD45 in demethylation ( Barreto et al., 2007 ; Schmitz et al., 2009 ) has been challenged ( Jin et al., 2008 ). In zebrafish embryos, rapid demethylation of exogenous and genomic DNA occurs in two coupled steps: enzymatic 5mC deamination to thymine by Activation Induced deaminase (AID) or Apolipoprotein B RNA-editing catalytic component 2b and 2a (Apobec2b, 2a), followed by removal of the mismatched thymine by the zebrafish thymine glycosylase MBD4, with Gadd45 promoting the reaction ( Rai et al., 2008 ). Recently, 5-hydroxymethylcytosine (5hmC), an oxidative product of 5mC generated by the Tet hydroxylases ( Kriaucionis and Heintz, 2009 ; Tahiliani et al., 2009 ), has been proposed as a demethylation intermediate ( Globisch et al., 2010 ; Wu and Zhang, 2010 ). During gene activation in the adult mouse brain, demethylation by TET1-mediated hydroxylation of 5meC to 5hmC was promoted by AID/Apobec deaminases, in a process that generates 5-hydroxymethyluracil (5hmU) and ultimately requires BER, although the specific glycosylases involved were not identified ( Guo et al., 2011 ).

Numerous in vitro studies have documented a potential role of the BER enzyme TDG (thymine DNA glycosylase) in transcriptional regulation and demethylation. Indeed, TDG interacts with several transcription factors, including retinoic acid receptor (RAR), retinoid X receptor (RXR) ( Um et al., 1998 ), estrogen receptor α (ERα) ( Chen et al., 2003 ), thyroid transcription factor 1 (TTF1) ( Missero et al., 2001 ) and histone acetyl-transferases p300 and CBP ( Tini et al., 2002 ). It has been proposed that TDG may be responsible for demethylation, either through a direct 5mC glycosylase activity ( Zhu et al., 2000 ), or indirectly, by acting on G:T mismatches originated by a controlled deaminase activity of DNMT3a and DNMT3b ( Metivier et al., 2008 ). Very recently, TDG was shown to be involved in maintaining active and bivalent chromatin marks in mouse embryo fibroblasts and ES cells undergoing neuronal differentiation, respectively, but the mechanism for such epigenetic effects and the requirement of its catalytic activity were not clarified ( Cortazar et al., 2011 ). To investigate the functional role of TDG in epigenetic regulation, DNA demethylation and mammalian development, we generated mice with targeted inactivation of the TDG gene. Tdg -null embryos die in midgestation and exhibit a complex developmental phenotype that appears to derive from the failure to establish and maintain proper DNA methylation patterns at promoters and enhancers. A knock-in mutation that inactivates the glycosylase function of TDG is also embryonically lethal, and TDG is found in a complex with AID and GADD45a. These findings suggest a two-step catalytic mechanism for DNA demethylation that is essential for mammalian development.

Discussion

Our results demonstrate that TDG is required for normal mammalian development and the establishment of proper promoter/enhancer DNA methylation patterns that are conducive to transcription during embryogenesis. Failure to establish and maintain correct DNA methylation patterns is likely the cause of lethality and the observed complex developmental phenotype, as it is known that even small changes in DNA methylation cause abnormalities or lethality ( Gaudet et al., 2003 ).

In maintaining the proper epigenetic states, TDG apparently has a dual role: protection from aberrant hypermethylation (examples include the CpG islands of Efs, HoxA5 and Crabp2 in MEFs and the maternal alleles of H19 and Igf2 DMR2 in PGCs) and promotion of demethylation (exemplified by the Alb1 and Tat enhancers in hepatoblasts and the Oct4∷EGFP reporter in P19 cells).

Our combined biochemical and developmental data suggest that demethylation is an active process requiring TDG catalytic activity immediately downstream of the deaminase-catalyzed conversion of 5mC into thymine and/or of 5hmC into 5hmU. Thus, both the 5mC deamination and hydroxylation-deamination pathways may converge on TDG ( Figure 7B ) and perhaps this convergence may explain the absolute requirement of TDG during embryogenesis, although the relative prevalence of each pathway in different developmental contexts is currently unknown. The extent of possible partial compensation by activities of MBD4 and SMUG1 is also unknown, although the observed lethality of the Tdg knockout suggests that it is not sufficient to maintain embryogenesis.

Similarity of the lethal phenotype of the TDG knock-in and knock-out embryos, and in turn their resemblance to the p300/CBP and RAR/RXR mutant embryos, strongly suggests the possibility that protection from aberrant de novo methylation may also involve an active process, requiring TDG glycosylase activity to constantly antagonize methylation. However, a non-catalytic role of TDG cannot be ruled out completely and it is possible that protection from hypermethylation may occur at least in part via TDG inhibition of de novo DNA methyltransferases ( Li et al., 2007 ).

Our observation that TDG is required for the interaction of RAR/RXR with p300 both on and off the DNA suggests a model in which transcription factor binding (which is TDG-independent, Fig 2D,E ) may be responsible for tethering TDG to the promoter/enhancer ( Figure S7 ). Thus, it is possible that the TDG-provoked demethylation of differentiation-associated promoters/enhancers depends on TDG tethering by tissue-specific transcription factors. Of note, GADD45 is known to bind to nuclear hormone receptors ( Yi et al., 2000 ) and preferentially to hyperacetylated nucleosomes ( Carrier et al., 1999 ), suggesting that GADD45 may also be involved in targeting promoters for demethylation. In this model, proper targeting of AID may in turn depend on its interaction with GADD45 and TDG. Future binding studies, conducted on a genomic scale in different tissues and at various developmental stages, will further define the promoters/enhancers that are direct targets of the TDG-AID-GADD45a complex.

The identification of a ternary complex containing TDG, AID and GADD45a is consistent with the recent recognition of AID as a factor required for DNA demethylation during reprogramming of somatic cells ( Bhutani et al., 2010 ) and erasure of DNA methylation at imprinted and other loci in PGCs ( Popp et al., 2010 ), as well as with the role of GADD45a ( Barreto et al., 2007 ; Schmitz et al., 2009 ) and the related GADD45b ( Ma et al., 2009 ) in demethylation of specific promoters. It is also consistent with the role of BER in genome-wide active DNA demethylation in PGCs ( Hajkova et al., 2010 ). However, AID-deficient mice are viable and fertile ( Muramatsu et al., 2000 ; Revy et al., 2000 ), which suggests that TDG may also function downstream of Apobec deaminases and possibly engage different GADD45 proteins. In fact, various TDG-deaminase-GADD45 complexes, and possibly different TET proteins, may be utilized for demethylation associated with distinct developmental processes and reprogramming events.

Our observation of the lack of deamination-induced transition mutations in Tdg -mutant MEFs and Tdg -knockdown cells at promoters that undergo TDG-dependent protection from hypermethylation and demethylation suggests that TDG has a role not only in repairing deamination products, but also in initiating the DNA demethylation process, thus controlling the potentially mutagenic deaminase activity of AID. It is possible that initiation is regulated by targeting of the AID-TDG-GADD45a complex to relevant promoter/enhancers or by the optimal reciprocal amounts of these proteins in the complex, as TDG affects AID levels and/or stability ( Figure 6G ). Given the frequent CpG transition mutations and hypermethylation of tumor suppressor gene promoters in human cancer, inactivation of TDG and its demethylating complex or altered relative amounts of TDG, AID and GADD45a may play a role in tumor formation.
