---
pmid: '27694622'
title: The β-isoform of BCCIP promotes ADP release from the RAD51 presynaptic filament
  and enhances homologous DNA pairing.
authors:
- Kelso AA
- Goodson SD
- Watts LE
- Ledford LL
- Waldvogel SM
- Diehl JN
- Shah SB
- Say AF
- White JD
- Sehorn MG
journal: Nucleic Acids Res
year: '2017'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC5314795
doi: 10.1093/nar/gkw877
---

# The β-isoform of BCCIP promotes ADP release from the RAD51 presynaptic filament and enhances homologous DNA pairing.
**Authors:** Kelso AA, Goodson SD, Watts LE, Ledford LL, Waldvogel SM, Diehl JN, Shah SB, Say AF, White JD, Sehorn MG
**Journal:** Nucleic Acids Res (2017)
**DOI:** [10.1093/nar/gkw877](https://doi.org/10.1093/nar/gkw877)
**PMC:** [PMC5314795](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5314795/)

## Abstract

1. Nucleic Acids Res. 2017 Jan 25;45(2):711-725. doi: 10.1093/nar/gkw877. Epub
2016  Sep 30.

The β-isoform of BCCIP promotes ADP release from the RAD51 presynaptic filament 
and enhances homologous DNA pairing.

Kelso AA(1), Goodson SD(1), Watts LE(1), Ledford LL(1), Waldvogel SM(1), Diehl 
JN(1), Shah SB(1), Say AF(1), White JD(1), Sehorn MG(2)(3)(4).

Author information:
(1)Department of Genetics and Biochemistry, Clemson University, Clemson, SC 
29634, USA.
(2)Department of Genetics and Biochemistry, Clemson University, Clemson, SC 
29634, USA msehorn@clemson.edu.
(3)Center for Optical Materials Science and Engineering Technologies, Clemson 
University, Clemson, SC 29634, USA.
(4)Clemson University School of Health Research, Clemson, SC 29634, USA.

Homologous recombination (HR) is a template-driven repair pathway that mends DNA 
double-stranded breaks (DSBs), and thus helps to maintain genome stability. The 
RAD51 recombinase facilitates DNA joint formation during HR, but to accomplish 
this task, RAD51 must be loaded onto the single-stranded DNA. DSS1, a candidate 
gene for split hand/split foot syndrome, provides the ability to recognize 
RPA-coated ssDNA to the tumor suppressor BRCA2, which is complexed with RAD51. 
Together BRCA2-DSS1 displace RPA and load RAD51 onto the ssDNA. In addition, the 
BRCA2 interacting protein BCCIP normally colocalizes with chromatin bound BRCA2, 
and upon DSB induction, RAD51 colocalizes with BRCA2-BCCIP foci. Down-regulation 
of BCCIP reduces DSB repair and disrupts BRCA2 and RAD51 foci formation. While 
BCCIP is known to interact with BRCA2, the relationship between BCCIP and RAD51 
is not known. In this study, we investigated the biochemical role of the 
β-isoform of BCCIP in relation to the RAD51 recombinase. We demonstrate that 
BCCIPβ binds DNA and physically and functionally interacts with RAD51 to 
stimulate its homologous DNA pairing activity. Notably, this stimulatory effect 
is not the result of RAD51 nucleoprotein filament stabilization; rather, we 
demonstrate that BCCIPβ induces a conformational change within the RAD51 
filament that promotes release of ADP to help maintain an active presynaptic 
filament. Our findings reveal a functional role for BCCIPβ as a RAD51 accessory 
factor in HR.

© The Author(s) 2016. Published by Oxford University Press on behalf of Nucleic 
Acids Research.

DOI: 10.1093/nar/gkw877
PMCID: PMC5314795
PMID: 27694622 [Indexed for MEDLINE]

## Full Text

INTRODUCTIONHomologous recombination (HR) is an indispensable repair pathway involved in both genome maintenance through the repair of chromosomal lesions such as DNA double-stranded breaks (DSBs) and in creating genetic diversity among progeny. DSBs can arise from reactive oxygen species generated after exposure to exogenous agents, such as ionizing radiation or radiomimetic chemicals, as well as from endogenous stress, such as damaged replication forks and metabolic processes (1). Defects in the HR machinery may manifest as erroneously repaired DSBs that cause chromosomal aberrations and cancer (2–5).The repair of DSBs by HR is a carefully regulated, multi-step process. The ends of the DSB are nucleolytically processed to expose 3′ single-stranded DNA (ssDNA) overhangs that serve as the nucleation sites for the HR machinery. One key component of the HR machinery is RAD51, the eukaryotic ortholog of the Escherichia coli RecA recombinase, which binds the ssDNA tail to form a nucleoprotein filament known as a presynaptic filament. The ATP-bound active form of the RAD51 presynaptic filament searches for homology within the sister chromatid. When homology is located, the presynaptic filament base-pairs the ssDNA to its complementary strand, displacing the homologous strand to form a displacement loop (D-loop) structure. RAD51 extends the D-loop via DNA strand exchange.There are several well-characterized accessory proteins that assist RAD51 in the HR pathway including replication protein A (RPA) and BRCA2. RPA is a heterotrimeric ssDNA binding protein that is necessary to promote DNA strand exchange by removing secondary structure (6). Paradoxically, RPA also interferes with RAD51-mediated DNA strand exchange by competing for the same binding sites as RAD51 on the 3′ ssDNA overhangs. To overcome this inhibitory effect, protein factors known as recombination mediators help to displace RPA and facilitate the loading of RAD51 on the ssDNA nucleation site. The tumor suppressor BRCA2 is a recombination mediator in humans (7–10) that has an accessory factor of its own. DSS1, associated with split hand/foot syndrome (11,12), is a small polypeptide that interacts with the oligonucleotide binding domain (OB1) within the DNA binding domain of BRCA2. The interaction of DSS1 with BRCA2 facilitates the loading of RAD51 onto RPA-coated ssDNA because DSS1 functions as a DNA mimic to reduce the affinity of RPA for ssDNA, aiding in the function of BRCA2 (10).In addition to DSS1 and RAD51, there are other BRCA2-interacting partners (13–15), one of which is the BRCA2 and CDKN1A Interacting Protein, BCCIP (16). BCCIP is an essential gene, and two major splice variant isoforms are present in humans: BCCIPα and BCCIPβ (17). Reduced expression of BCCIP is associated with ovarian cancer, renal cell carcinoma, colorectal cancer (17,18) and astrocytic brain tumors (19). BCCIP was identified as a BRCA2-interacting protein from a yeast two-hybrid screen that used the highly conserved DNA binding domain of BRCA2 (exons 14–24) as bait (16). Subsequently, BCCIP was shown to co-localize with RAD51 foci and BRCA2 foci in the nucleus. RNA interference of BCCIP yielded a reduction in RAD51 and BRCA2 focus formation after irradiation (20,21) and reduction in the subsequent repair of DSBs by HR in vivo (20). The significance of reduced levels of BCCIP, the loss of RAD51 and BRCA2 foci, and unrepaired DSBs is not known. Although previous studies demonstrated that RAD51 co-immunoprecipitated with BCCIP (20), it is not known whether BCCIP interacts and functions directly with RAD51 or indirectly through its interaction with BRCA2.In the majority of human cells, BCCIPβ is the foremost expressed isoform (16). Therefore, the goal of this study was to interrogate the biochemical role of BCCIPβ in HR. To do this, we expressed and purified BCCIPβ-(HIS)6 to near homogeneity. Our data using purified BCCIPβ-(HIS)6 revealed that BCCIPβ-(HIS)6 harbors DNA binding activity and physically interacts with RAD51. Importantly, we demonstrate a functional interaction of BCCIPβ-(HIS)6 with RAD51 through its ability to enhance RAD51-mediated homologous DNA pairing. We provide evidence that the enhancement by BCCIPβ-(HIS)6 is not due to the stabilization of the RAD51 nucleoprotein filament, but rather to the ability of BCCIPβ-(HIS)6 to stimulate RAD51 ATP hydrolysis and promote the release of ADP from the inactive RAD51-ADP-ssDNA filament. We demonstrate the enhanced release of ADP is likely due to a conformational change induced in RAD51 upon interaction with BCCIPβ-(HIS)6. These results are the first to provide direct evidence that BCCIPβ-(HIS)6 regulates the activity of RAD51 and suggest that BCCIPβ is a critical player in maintaining chromosome stability.

INTRODUCTION

Homologous recombination (HR) is an indispensable repair pathway involved in both genome maintenance through the repair of chromosomal lesions such as DNA double-stranded breaks (DSBs) and in creating genetic diversity among progeny. DSBs can arise from reactive oxygen species generated after exposure to exogenous agents, such as ionizing radiation or radiomimetic chemicals, as well as from endogenous stress, such as damaged replication forks and metabolic processes (1). Defects in the HR machinery may manifest as erroneously repaired DSBs that cause chromosomal aberrations and cancer (2–5).

The repair of DSBs by HR is a carefully regulated, multi-step process. The ends of the DSB are nucleolytically processed to expose 3′ single-stranded DNA (ssDNA) overhangs that serve as the nucleation sites for the HR machinery. One key component of the HR machinery is RAD51, the eukaryotic ortholog of the Escherichia coli RecA recombinase, which binds the ssDNA tail to form a nucleoprotein filament known as a presynaptic filament. The ATP-bound active form of the RAD51 presynaptic filament searches for homology within the sister chromatid. When homology is located, the presynaptic filament base-pairs the ssDNA to its complementary strand, displacing the homologous strand to form a displacement loop (D-loop) structure. RAD51 extends the D-loop via DNA strand exchange.

There are several well-characterized accessory proteins that assist RAD51 in the HR pathway including replication protein A (RPA) and BRCA2. RPA is a heterotrimeric ssDNA binding protein that is necessary to promote DNA strand exchange by removing secondary structure (6). Paradoxically, RPA also interferes with RAD51-mediated DNA strand exchange by competing for the same binding sites as RAD51 on the 3′ ssDNA overhangs. To overcome this inhibitory effect, protein factors known as recombination mediators help to displace RPA and facilitate the loading of RAD51 on the ssDNA nucleation site. The tumor suppressor BRCA2 is a recombination mediator in humans (7–10) that has an accessory factor of its own. DSS1, associated with split hand/foot syndrome (11,12), is a small polypeptide that interacts with the oligonucleotide binding domain (OB1) within the DNA binding domain of BRCA2. The interaction of DSS1 with BRCA2 facilitates the loading of RAD51 onto RPA-coated ssDNA because DSS1 functions as a DNA mimic to reduce the affinity of RPA for ssDNA, aiding in the function of BRCA2 (10).

In addition to DSS1 and RAD51, there are other BRCA2-interacting partners (13–15), one of which is the BRCA2 and CDKN1A Interacting Protein, BCCIP (16). BCCIP is an essential gene, and two major splice variant isoforms are present in humans: BCCIPα and BCCIPβ (17). Reduced expression of BCCIP is associated with ovarian cancer, renal cell carcinoma, colorectal cancer (17,18) and astrocytic brain tumors (19). BCCIP was identified as a BRCA2-interacting protein from a yeast two-hybrid screen that used the highly conserved DNA binding domain of BRCA2 (exons 14–24) as bait (16). Subsequently, BCCIP was shown to co-localize with RAD51 foci and BRCA2 foci in the nucleus. RNA interference of BCCIP yielded a reduction in RAD51 and BRCA2 focus formation after irradiation (20,21) and reduction in the subsequent repair of DSBs by HR in vivo (20). The significance of reduced levels of BCCIP, the loss of RAD51 and BRCA2 foci, and unrepaired DSBs is not known. Although previous studies demonstrated that RAD51 co-immunoprecipitated with BCCIP (20), it is not known whether BCCIP interacts and functions directly with RAD51 or indirectly through its interaction with BRCA2.

In the majority of human cells, BCCIPβ is the foremost expressed isoform (16). Therefore, the goal of this study was to interrogate the biochemical role of BCCIPβ in HR. To do this, we expressed and purified BCCIPβ-(HIS)6 to near homogeneity. Our data using purified BCCIPβ-(HIS)6 revealed that BCCIPβ-(HIS)6 harbors DNA binding activity and physically interacts with RAD51. Importantly, we demonstrate a functional interaction of BCCIPβ-(HIS)6 with RAD51 through its ability to enhance RAD51-mediated homologous DNA pairing. We provide evidence that the enhancement by BCCIPβ-(HIS)6 is not due to the stabilization of the RAD51 nucleoprotein filament, but rather to the ability of BCCIPβ-(HIS)6 to stimulate RAD51 ATP hydrolysis and promote the release of ADP from the inactive RAD51-ADP-ssDNA filament. We demonstrate the enhanced release of ADP is likely due to a conformational change induced in RAD51 upon interaction with BCCIPβ-(HIS)6. These results are the first to provide direct evidence that BCCIPβ-(HIS)6 regulates the activity of RAD51 and suggest that BCCIPβ is a critical player in maintaining chromosome stability.

MATERIALS AND METHODSPlasmids and oligonucleotidesThe human BCCIPβ cDNA was purchased from Open Biosystems. A (HIS)6 tag was added to the 3′ end of BCCIPβ via PCR using the forward primer 5′-GGGAATCCCATATGGCGTCCAGGTCTAAGCGGCGTG and reverse primer 5′-CCCATATGGAATTCTTAATGATGATGATGATGATGAGGACCACCGACAGATAGATATTCTTTCAGTTTATCCATG. The amplified product was inserted into the bacterial expression plasmid pET11c (Novagen), and sequenced to ensure no undesired mutations occurred. The oligonucleotide OL90 (5′-AAATCAATCTAAAGTATATATGAGTAAACTTGGTCTGACAGTTACCAATGCTTAATCAGTGAGGCACCTATCTCAGCGATCTGTCTATTT) was radiolabeled using T4 polynucleotide kinase and [32P-γ]-ATP as described (22). All oligonucleotides were purchased from Integrated DNA Technologies. pBluescript was purified from E. coli using a Giga Kit (Qiagen). ϕX174 (+) virion ssDNA and ϕX174 replicative form I double-stranded DNA (dsDNA) were purchased from New England BioLabs—ϕX174 dsDNA was linearized with ApaLI (New England BioLabs).Cell growth, expression and purification of BCCIPβThe BCCIPβ-(HIS)6 pET11c expression plasmid was transformed into the E. coli strain BL21(DE3). Cells were grown at 37°C to an OD600 of 1.0, and protein expression was induced with 0.4 mM IPTG for 20 hr at 16°C. The cells were harvested by centrifugation. All purification steps were performed at 4°C. The E. coli cell paste (26 g) was resuspended in 130 ml of Buffer A (50 mM Tris-HCl pH 7.5, 1 mM ethylenediaminetetraacetic acid (EDTA), 10% sucrose, 1 mM β-mercaptoethanol, 0.01% Igepal, 1 mM benzamide, 10 μg/ml lysozyme, 1 mM phenylmethylsulfonyl fluoride and protease inhibitors: aprotinin, leupeptin, chymostatin and pepstatin A at final concentrations of 5 μg/ml) containing 500 mM KCl. The cells were sonicated with six 30 s cycles at 80% amplitude using a sonicator (Qsonica Q125). The lysate was clarified by ultracentrifugation at 40 000 rpm for 90 min in a Beckman Type Ti45 rotor. The clarified lysate was incubated with 3 ml Ni-NTA agarose beads (GE Healthcare) followed a wash with 30 ml of Buffer B (20 mM KH2PO4 pH 7.5, 10% glycerol, 0.5 mM EDTA) supplemented with 0.01% Igepal, 1 mM β-mercaptoethanol, 1 M KCl and 50 mM imidazole. The BCCIPβ-(HIS)6 protein was eluted using Buffer B containing 100 mM KCl and 500 mM imidazole. Fractions containing BCCIPβ-(HIS)6 were pooled and loaded onto a 2.5 ml Macro hydroxyapatite column (Bio-Rad). The protein was eluted with a 50 ml gradient of Buffer B containing 0–400 mM KH2PO4 and 1 mM dithiothreitol. The peak fractions (∼290 mM KH2PO4) were pooled and loaded onto an 8 ml Source 15Q column (GE Healthcare), which was fractionated using a 120 ml gradient of Buffer B containing 100 mM–1 M KCl. BCCIPβ-(HIS)6 eluted from the column at approximately 330 mM KCl, and the peak fractions were pooled and concentrated to 3.5 mg/ml using an Amicon Ultra 30 000 MWCO (Millipore). Approximately 0.8 mg of purified BCCIPβ-(HIS)6 was frozen in small aliquots and stored at −80°C.Other protein purificationsThe purifications of untagged human RAD51, human RAD51K133A, Saccharomyces cerevisiae Rad51 (ScRad51) and S. cerevisiae Rad54 (ScRad54) were performed as previously described (23–26).ϕx174 DNA mobility shift assayIncreasing concentrations of BCCIPβ-(HIS)6 (0.24 μM, 0.47 μM, 0.96 μM, 1.8 μM, 2.8 μM and 4.7 μM) were incubated with ϕx174 circular ssDNA (30 μM nucleotides) or linearized ϕx174 dsDNA (30 μM base pairs) in Buffer C (25 mM Tris-HCl pH 7.4, 0.1 μg/μl bovine serum albumin (BSA), 40 mM KCl) at 37°C for 10 min. DNA loading dye (10 mM Tris-HCl pH 7.5, 0.5 mM EDTA, 50% glycerol, 0.1% (w/v) Orange G) was added to each reaction, and the samples were resolved on a 1% agarose gel at 4°C. Gels were stained with ethidium bromide and imaged using Image Lab software (BioRad). A control reaction, containing the highest concentration of BCCIPβ-(HIS)6 (4.7 μM), was deproteinized by addition of SDS (0.5%) and Proteinase K (0.5 μg/ml) followed by an incubation at 37°C for 20 min. Three independent experiments for each reaction yielded the same results.Pull-down assayBCCIPβ-(HIS)6 (5 μg) was incubated with human RAD51 (5 μg) or ScRad51 (5 μg) in the presence of Ni-NTA agarose beads (GE Healthcare) with agitation at 4°C for 60 min in Buffer B (20 mM KH2PO4 pH 7.5, 10% glycerol, 0.5 mM EDTA) containing 120 mM KCl. The supernatant was removed, and the beads were washed three times with Buffer B containing 120 mM KCl. An equal volume of sodium dodecyl sulphate (SDS) loading dye (160 mM Tris-HCl pH 6.8, 60% glycerol, 4% SDS (w/v)) was added to the supernatant and the wash, while 30 μl of SDS loading dye was added to the beads to elute any bound proteins. The supernatant, wash and eluate were separated on a 12% SDS-polyacrylamide gel and stained with Coomassie blue. As a control, RAD51 (5 μg) or ScRad51 (5 μg) was incubated with Ni-NTA agarose beads in the absence of BCCIPβ-(HIS)6, under the same experimental conditions as above.D-loop assay32P-labeled oligonucleotide OL90 (4.5 μM) was incubated with RAD51 (1.5 μM) in Buffer D (25 mM Tris-HCl pH 7.4, 0.1 μg/μl BSA, 1 mM DTT, 2 mM ATP, 1.4 mM MgCl2, 16 mM creatine phosphate, 36 μg/ml creatine kinase and 30 mM KCl) for 8 min at 37°C. CaCl2 (1.8 mM final) was added where indicated, followed by an additional 2 min incubation at 37°C. Increasing concentrations of BCCIPβ-(HIS)6 (0.75 μM, 1.5 μM, 3 μM, 4.5 μM) were added to the reactions followed by an 8 min incubation at 37°C. The addition of supercoiled pBluescript (35 μM base pairs) initiated the reaction. After a 6 min incubation at 37°C, SDS (0.5%) and Proteinase K (5 μg/ml) were added to deproteinize the reactions, followed by a 20 min incubation at 37°C. All reaction products were resolved on a 0.9% agarose gel, dried on DE81 anion exchange paper (GE Healthcare), visualized using a phosphorimager (Typhoon FLA 7000, GE Healthcare) and quantified using ImageQuant TL (GE Healthcare). The reactions monitoring RAD51-mediated D-loop formation as a function of time were carried out in the same manner as above, except each reaction was stopped at the indicated times. To test the species specificity, ScRad51 (1.5 μM) was incubated with 32P-OL90 (4.5 μM) in Buffer E (25 mM Tris-HCl pH 7.4, 0.1 μg/μl BSA, 1 mM DTT, 2 mM ATP, 2.4 mM MgCl2, 16 mM creatine phosphate, 36 μg/ml creatine kinase, and 30 mM KCl) for 10 min in a total reaction volume of 12.5 μl. ScRad54 (0.2 μM) or increasing concentrations of BCCIPβ-(HIS)6 (1.5 μM, 3 μM, 4.5 μM) were added to the reactions (as indicated) followed by an 8 min incubation at 37°C prior to the addition of pBluescript. After 8 min, the reaction products were deproteinized and analyzed as described above. Three independent experiments were tested for each D-loop reaction, and the average percent D-loop was graphed. Error bars represent s.e.m. Significance was determined by comparing the results of RAD51 to each of the conditions with BCCIPβ-(HIS)6 using a two-sample t-test.BCCIPβ gel filtrationGel filtration was facilitated with the ÄKTA pure chromatography system (GE Healthcare). Purified BCCIPβ-(HIS)6 (100 μg) was loaded onto a 35 ml Sephacryl S-400 (GE Healthcare) size exclusion chromatography column (1 cm diameter), pre-equilibrated in Buffer B (20 mM KH2PO4 pH 7.5, 10% glycerol, 0.5 mM EDTA) with 100 mM KCl. Each fraction (40 fractions total, 1 ml each, 0.5 ml/min flow rate) was precipitated by the addition of deoxycholate (0.02%) and trichloroacetic acid (0.1%), resuspended in 20 μl of SDS loading dye, and resolved using 12% SDS-polyacrylamide gel electrophoresis. The gel was transferred to a nitrocellulose membrane (Whatman) and visualized using HRP-conjugated anti-poly-histidine antibodies (Sigma-Aldrich, A7058; 1:2000) in Buffer F (0.12% 4-chloro-1-naphthol (w/v), 0.12% H2O2 (v/v), in PBS). Molecular weight standards (Bio-Rad) were processed in the same manner as above. To establish molecular sizing for the fractions, molecular weight standards were plotted based on the log of their molecular weight (y-axis) and fraction number (x-axis) using KaleidaGraph 4.1.3, and the linear equation was derived using the linear curve fit tool.Nuclease protection assay32P-labeled OL90 (3 μM nucleotides) was incubated with RAD51 (0.4 μM) in Buffer D (25 mM Tris-HCl pH 7.4, 0.1 μg/μl BSA, 1 mM DTT, 2 mM ATP, 1.4 mM MgCl2, 16 mM creatine phosphate, 36 μg/ml creatine kinase and 30 mM KCl) for 10 min at 37°C. Increasing concentrations of BCCIPβ-(HIS)6 (0.4 μM, 0.8 μM, 1.2 μM) were added to the reaction and incubated for 8 min. Subsequently, 2 units of DNase I (Promega) were added followed by a 15 min incubation at 37°C. The reactions were stopped by the addition of equal parts SDS (0.8%) and Proteinase K (1.6 μg/ml). The reaction products were resolved using 10% non-denaturing polyacrylamide gel electrophoresis. The gels were then dried on Whatman cellulose chromatography paper (Sigma-Aldrich) and analyzed using a phosphorimager. Error bars represent s.e.m. The mean percent protection from three separate experiments was graphed.Trypsin treatment of RAD51 complexesThe trypsin digest of RAD51 complexes was performed similar to a previously described procedure (27). Briefly, RAD51 (5 μM) incubated in 10 μl of Buffer G (25 mM Tris-HCl pH 7.4, 1 mM DTT, 1.4 mM MgCl2, 30 mM KCl) in the presence or absence of the indicated combinations of ATP (2 mM), ϕX174 circular ssDNA (30 μM nucleotides), CaCl2 (1.8 mM) and/or BCCIPβ-(HIS)6 (10 μM) for 10 min at 37°C. Trypsin (20 μg/ml) was added to each reaction and incubated for an additional 30 min. The reactions were inactivated by the addition of SDS loading dye (7.5 μl) and a 10 min incubation at 80°C. The reaction products were resolved using 16% SDS-polyacrylamide gel electrophoresis, followed by transfer to a nitrocellulose membrane (Whatman). The membrane was incubated with anti-RAD51 as the primary antibody (1:2000; ab63801, Abcam) and HRP-conjugated anti-rabbit IgG as the secondary antibody (1:2000, Sigma-Aldrich). The membrane was developed using Buffer F (0.12% 4-chloro-1-naphthol (w/v), 0.12% H2O2 (v/v), in phosphate buffered saline (PBS)), and imaged using Image Lab software (BioRad). The amount of undigested RAD51 fragments was plotted, and the amount of Fragments A, B, C and D was plotted based on relative intensities of each band. Results were derived from three separate experiments, and error bars represent s.e.m.ATP hydrolysisThe ATP hydrolysis assay was performed with RAD51 as previously described (28). Briefly, RAD51 (2 μM) was incubated with ϕx174 ssDNA (60 μM nucleotides) in Buffer G (25 mM Tris-HCl pH 7.4, 1 mM DTT, 1.4 mM MgCl2, 30 mM KCl) with 0.1 μg/μl BSA and 0.5 mM ATP for 5 min at 37°C for filament formation. BCCIPβ-(HIS)6 (4 μM) was added to the reaction where indicated. After an additional 5 min incubation at 37°C, 0.3 μCi of [32P-γ]-ATP was added to the reactions followed by the addition of an equal volume of 0.5 mM EDTA at the indicated time points to stop each reaction. The reaction products were resolved using thin-layer chromatography on polyethyleneimine-cellulose plates (Sigma-Aldrich). The plates were analyzed using a phosphorimager. The mean percent hydrolysis was graphed from three independent experiments. Error bars represent s.e.m, and significance was determined using a two-sample t-test and comparing RAD51 to RAD51+BCCIPβ-(HIS)6.RAD51 MANT-ADP releaseA Photon Technologies International Spectrometer Model 814 was used to monitor the fluorescence changes of unbound free MANT-ADP (ThermoFisher Scientific) and MANT-ADP bound by RAD51 in the presence of ssDNA in several experiments, similar to previous studies on DnaC and ClpB (29,30). (i) To obtain fluorescence emission spectra of MANT-ADP and the MANT-ADP-RAD51-ssDNA complex, MANT-ADP (0.5 μM) was preincubated in Buffer H (25 mM Tris-HCl, pH 7.4, 1.4 mM MgCl2, 32 mM KCl) with ϕX174 ssDNA (2.0 μM nucleotides) in the absence or presence of RAD51 (0.5 μM) or RAD51K133A (0.5 μM) for 30 min at 25°C with constant stirring in a quartz cuvette (1 cm path length). Samples were excited at 350 nm (5 nm bandwidth) and emission spectra were taken from 400–500 nm (5 nm bandwidth) at 1 nm steps. (ii) The dissociation of MANT-ADP from a pre-formed MANT-ADP-RAD51-ssDNA in the complex was observed as a decrease in the fluorescence signal. RAD51 (0.5 μM), ssDNA (2.0 μM nucleotides) and MANT-ADP (0.5 μM) in either Buffer H alone or Buffer H supplemented with CaCl2 (1.8 mM) were maintained in a quartz cuvette (1-cm path length) with constant stirring. To initiate ADP release in the presence of magnesium, either BCCIPβ-(HIS)6 (1.0 μM) or Buffer H was added to the cuvette. ADP release in the presence of calcium was initiated by the addition of either BCCIPβ-(HIS)6 (1.0 μM) with or without ATP (5 μM) or Buffer H with or without ATP (5 μM) to the cuvette. The resulting decrease in fluorescence was monitored in time base mode with an excitation wavelength of 350 nm (5 nm bandwidth) and an emission wavelength of 444 nm (5 nm bandwidth). Significance was determined using a two-sample t-test, comparing the 600 s time point for RAD51+Buffer to RAD51+BCCIPβ-(HIS)6 in the presence of magnesium and RAD51+Buffer+ATP to RAD51+BCCIPβ-(HIS)6+ATP in the presence of calcium.

MATERIALS AND METHODS

Plasmids and oligonucleotidesThe human BCCIPβ cDNA was purchased from Open Biosystems. A (HIS)6 tag was added to the 3′ end of BCCIPβ via PCR using the forward primer 5′-GGGAATCCCATATGGCGTCCAGGTCTAAGCGGCGTG and reverse primer 5′-CCCATATGGAATTCTTAATGATGATGATGATGATGAGGACCACCGACAGATAGATATTCTTTCAGTTTATCCATG. The amplified product was inserted into the bacterial expression plasmid pET11c (Novagen), and sequenced to ensure no undesired mutations occurred. The oligonucleotide OL90 (5′-AAATCAATCTAAAGTATATATGAGTAAACTTGGTCTGACAGTTACCAATGCTTAATCAGTGAGGCACCTATCTCAGCGATCTGTCTATTT) was radiolabeled using T4 polynucleotide kinase and [32P-γ]-ATP as described (22). All oligonucleotides were purchased from Integrated DNA Technologies. pBluescript was purified from E. coli using a Giga Kit (Qiagen). ϕX174 (+) virion ssDNA and ϕX174 replicative form I double-stranded DNA (dsDNA) were purchased from New England BioLabs—ϕX174 dsDNA was linearized with ApaLI (New England BioLabs).

Plasmids and oligonucleotides

The human BCCIPβ cDNA was purchased from Open Biosystems. A (HIS)6 tag was added to the 3′ end of BCCIPβ via PCR using the forward primer 5′-GGGAATCCCATATGGCGTCCAGGTCTAAGCGGCGTG and reverse primer 5′-CCCATATGGAATTCTTAATGATGATGATGATGATGAGGACCACCGACAGATAGATATTCTTTCAGTTTATCCATG. The amplified product was inserted into the bacterial expression plasmid pET11c (Novagen), and sequenced to ensure no undesired mutations occurred. The oligonucleotide OL90 (5′-AAATCAATCTAAAGTATATATGAGTAAACTTGGTCTGACAGTTACCAATGCTTAATCAGTGAGGCACCTATCTCAGCGATCTGTCTATTT) was radiolabeled using T4 polynucleotide kinase and [32P-γ]-ATP as described (22). All oligonucleotides were purchased from Integrated DNA Technologies. pBluescript was purified from E. coli using a Giga Kit (Qiagen). ϕX174 (+) virion ssDNA and ϕX174 replicative form I double-stranded DNA (dsDNA) were purchased from New England BioLabs—ϕX174 dsDNA was linearized with ApaLI (New England BioLabs).

Cell growth, expression and purification of BCCIPβThe BCCIPβ-(HIS)6 pET11c expression plasmid was transformed into the E. coli strain BL21(DE3). Cells were grown at 37°C to an OD600 of 1.0, and protein expression was induced with 0.4 mM IPTG for 20 hr at 16°C. The cells were harvested by centrifugation. All purification steps were performed at 4°C. The E. coli cell paste (26 g) was resuspended in 130 ml of Buffer A (50 mM Tris-HCl pH 7.5, 1 mM ethylenediaminetetraacetic acid (EDTA), 10% sucrose, 1 mM β-mercaptoethanol, 0.01% Igepal, 1 mM benzamide, 10 μg/ml lysozyme, 1 mM phenylmethylsulfonyl fluoride and protease inhibitors: aprotinin, leupeptin, chymostatin and pepstatin A at final concentrations of 5 μg/ml) containing 500 mM KCl. The cells were sonicated with six 30 s cycles at 80% amplitude using a sonicator (Qsonica Q125). The lysate was clarified by ultracentrifugation at 40 000 rpm for 90 min in a Beckman Type Ti45 rotor. The clarified lysate was incubated with 3 ml Ni-NTA agarose beads (GE Healthcare) followed a wash with 30 ml of Buffer B (20 mM KH2PO4 pH 7.5, 10% glycerol, 0.5 mM EDTA) supplemented with 0.01% Igepal, 1 mM β-mercaptoethanol, 1 M KCl and 50 mM imidazole. The BCCIPβ-(HIS)6 protein was eluted using Buffer B containing 100 mM KCl and 500 mM imidazole. Fractions containing BCCIPβ-(HIS)6 were pooled and loaded onto a 2.5 ml Macro hydroxyapatite column (Bio-Rad). The protein was eluted with a 50 ml gradient of Buffer B containing 0–400 mM KH2PO4 and 1 mM dithiothreitol. The peak fractions (∼290 mM KH2PO4) were pooled and loaded onto an 8 ml Source 15Q column (GE Healthcare), which was fractionated using a 120 ml gradient of Buffer B containing 100 mM–1 M KCl. BCCIPβ-(HIS)6 eluted from the column at approximately 330 mM KCl, and the peak fractions were pooled and concentrated to 3.5 mg/ml using an Amicon Ultra 30 000 MWCO (Millipore). Approximately 0.8 mg of purified BCCIPβ-(HIS)6 was frozen in small aliquots and stored at −80°C.

Cell growth, expression and purification of BCCIPβ

The BCCIPβ-(HIS)6 pET11c expression plasmid was transformed into the E. coli strain BL21(DE3). Cells were grown at 37°C to an OD600 of 1.0, and protein expression was induced with 0.4 mM IPTG for 20 hr at 16°C. The cells were harvested by centrifugation. All purification steps were performed at 4°C. The E. coli cell paste (26 g) was resuspended in 130 ml of Buffer A (50 mM Tris-HCl pH 7.5, 1 mM ethylenediaminetetraacetic acid (EDTA), 10% sucrose, 1 mM β-mercaptoethanol, 0.01% Igepal, 1 mM benzamide, 10 μg/ml lysozyme, 1 mM phenylmethylsulfonyl fluoride and protease inhibitors: aprotinin, leupeptin, chymostatin and pepstatin A at final concentrations of 5 μg/ml) containing 500 mM KCl. The cells were sonicated with six 30 s cycles at 80% amplitude using a sonicator (Qsonica Q125). The lysate was clarified by ultracentrifugation at 40 000 rpm for 90 min in a Beckman Type Ti45 rotor. The clarified lysate was incubated with 3 ml Ni-NTA agarose beads (GE Healthcare) followed a wash with 30 ml of Buffer B (20 mM KH2PO4 pH 7.5, 10% glycerol, 0.5 mM EDTA) supplemented with 0.01% Igepal, 1 mM β-mercaptoethanol, 1 M KCl and 50 mM imidazole. The BCCIPβ-(HIS)6 protein was eluted using Buffer B containing 100 mM KCl and 500 mM imidazole. Fractions containing BCCIPβ-(HIS)6 were pooled and loaded onto a 2.5 ml Macro hydroxyapatite column (Bio-Rad). The protein was eluted with a 50 ml gradient of Buffer B containing 0–400 mM KH2PO4 and 1 mM dithiothreitol. The peak fractions (∼290 mM KH2PO4) were pooled and loaded onto an 8 ml Source 15Q column (GE Healthcare), which was fractionated using a 120 ml gradient of Buffer B containing 100 mM–1 M KCl. BCCIPβ-(HIS)6 eluted from the column at approximately 330 mM KCl, and the peak fractions were pooled and concentrated to 3.5 mg/ml using an Amicon Ultra 30 000 MWCO (Millipore). Approximately 0.8 mg of purified BCCIPβ-(HIS)6 was frozen in small aliquots and stored at −80°C.

Other protein purificationsThe purifications of untagged human RAD51, human RAD51K133A, Saccharomyces cerevisiae Rad51 (ScRad51) and S. cerevisiae Rad54 (ScRad54) were performed as previously described (23–26).

Other protein purifications

The purifications of untagged human RAD51, human RAD51K133A, Saccharomyces cerevisiae Rad51 (ScRad51) and S. cerevisiae Rad54 (ScRad54) were performed as previously described (23–26).

ϕx174 DNA mobility shift assayIncreasing concentrations of BCCIPβ-(HIS)6 (0.24 μM, 0.47 μM, 0.96 μM, 1.8 μM, 2.8 μM and 4.7 μM) were incubated with ϕx174 circular ssDNA (30 μM nucleotides) or linearized ϕx174 dsDNA (30 μM base pairs) in Buffer C (25 mM Tris-HCl pH 7.4, 0.1 μg/μl bovine serum albumin (BSA), 40 mM KCl) at 37°C for 10 min. DNA loading dye (10 mM Tris-HCl pH 7.5, 0.5 mM EDTA, 50% glycerol, 0.1% (w/v) Orange G) was added to each reaction, and the samples were resolved on a 1% agarose gel at 4°C. Gels were stained with ethidium bromide and imaged using Image Lab software (BioRad). A control reaction, containing the highest concentration of BCCIPβ-(HIS)6 (4.7 μM), was deproteinized by addition of SDS (0.5%) and Proteinase K (0.5 μg/ml) followed by an incubation at 37°C for 20 min. Three independent experiments for each reaction yielded the same results.

ϕx174 DNA mobility shift assay

Increasing concentrations of BCCIPβ-(HIS)6 (0.24 μM, 0.47 μM, 0.96 μM, 1.8 μM, 2.8 μM and 4.7 μM) were incubated with ϕx174 circular ssDNA (30 μM nucleotides) or linearized ϕx174 dsDNA (30 μM base pairs) in Buffer C (25 mM Tris-HCl pH 7.4, 0.1 μg/μl bovine serum albumin (BSA), 40 mM KCl) at 37°C for 10 min. DNA loading dye (10 mM Tris-HCl pH 7.5, 0.5 mM EDTA, 50% glycerol, 0.1% (w/v) Orange G) was added to each reaction, and the samples were resolved on a 1% agarose gel at 4°C. Gels were stained with ethidium bromide and imaged using Image Lab software (BioRad). A control reaction, containing the highest concentration of BCCIPβ-(HIS)6 (4.7 μM), was deproteinized by addition of SDS (0.5%) and Proteinase K (0.5 μg/ml) followed by an incubation at 37°C for 20 min. Three independent experiments for each reaction yielded the same results.

Pull-down assayBCCIPβ-(HIS)6 (5 μg) was incubated with human RAD51 (5 μg) or ScRad51 (5 μg) in the presence of Ni-NTA agarose beads (GE Healthcare) with agitation at 4°C for 60 min in Buffer B (20 mM KH2PO4 pH 7.5, 10% glycerol, 0.5 mM EDTA) containing 120 mM KCl. The supernatant was removed, and the beads were washed three times with Buffer B containing 120 mM KCl. An equal volume of sodium dodecyl sulphate (SDS) loading dye (160 mM Tris-HCl pH 6.8, 60% glycerol, 4% SDS (w/v)) was added to the supernatant and the wash, while 30 μl of SDS loading dye was added to the beads to elute any bound proteins. The supernatant, wash and eluate were separated on a 12% SDS-polyacrylamide gel and stained with Coomassie blue. As a control, RAD51 (5 μg) or ScRad51 (5 μg) was incubated with Ni-NTA agarose beads in the absence of BCCIPβ-(HIS)6, under the same experimental conditions as above.

Pull-down assay

BCCIPβ-(HIS)6 (5 μg) was incubated with human RAD51 (5 μg) or ScRad51 (5 μg) in the presence of Ni-NTA agarose beads (GE Healthcare) with agitation at 4°C for 60 min in Buffer B (20 mM KH2PO4 pH 7.5, 10% glycerol, 0.5 mM EDTA) containing 120 mM KCl. The supernatant was removed, and the beads were washed three times with Buffer B containing 120 mM KCl. An equal volume of sodium dodecyl sulphate (SDS) loading dye (160 mM Tris-HCl pH 6.8, 60% glycerol, 4% SDS (w/v)) was added to the supernatant and the wash, while 30 μl of SDS loading dye was added to the beads to elute any bound proteins. The supernatant, wash and eluate were separated on a 12% SDS-polyacrylamide gel and stained with Coomassie blue. As a control, RAD51 (5 μg) or ScRad51 (5 μg) was incubated with Ni-NTA agarose beads in the absence of BCCIPβ-(HIS)6, under the same experimental conditions as above.

D-loop assay32P-labeled oligonucleotide OL90 (4.5 μM) was incubated with RAD51 (1.5 μM) in Buffer D (25 mM Tris-HCl pH 7.4, 0.1 μg/μl BSA, 1 mM DTT, 2 mM ATP, 1.4 mM MgCl2, 16 mM creatine phosphate, 36 μg/ml creatine kinase and 30 mM KCl) for 8 min at 37°C. CaCl2 (1.8 mM final) was added where indicated, followed by an additional 2 min incubation at 37°C. Increasing concentrations of BCCIPβ-(HIS)6 (0.75 μM, 1.5 μM, 3 μM, 4.5 μM) were added to the reactions followed by an 8 min incubation at 37°C. The addition of supercoiled pBluescript (35 μM base pairs) initiated the reaction. After a 6 min incubation at 37°C, SDS (0.5%) and Proteinase K (5 μg/ml) were added to deproteinize the reactions, followed by a 20 min incubation at 37°C. All reaction products were resolved on a 0.9% agarose gel, dried on DE81 anion exchange paper (GE Healthcare), visualized using a phosphorimager (Typhoon FLA 7000, GE Healthcare) and quantified using ImageQuant TL (GE Healthcare). The reactions monitoring RAD51-mediated D-loop formation as a function of time were carried out in the same manner as above, except each reaction was stopped at the indicated times. To test the species specificity, ScRad51 (1.5 μM) was incubated with 32P-OL90 (4.5 μM) in Buffer E (25 mM Tris-HCl pH 7.4, 0.1 μg/μl BSA, 1 mM DTT, 2 mM ATP, 2.4 mM MgCl2, 16 mM creatine phosphate, 36 μg/ml creatine kinase, and 30 mM KCl) for 10 min in a total reaction volume of 12.5 μl. ScRad54 (0.2 μM) or increasing concentrations of BCCIPβ-(HIS)6 (1.5 μM, 3 μM, 4.5 μM) were added to the reactions (as indicated) followed by an 8 min incubation at 37°C prior to the addition of pBluescript. After 8 min, the reaction products were deproteinized and analyzed as described above. Three independent experiments were tested for each D-loop reaction, and the average percent D-loop was graphed. Error bars represent s.e.m. Significance was determined by comparing the results of RAD51 to each of the conditions with BCCIPβ-(HIS)6 using a two-sample t-test.

D-loop assay

32P-labeled oligonucleotide OL90 (4.5 μM) was incubated with RAD51 (1.5 μM) in Buffer D (25 mM Tris-HCl pH 7.4, 0.1 μg/μl BSA, 1 mM DTT, 2 mM ATP, 1.4 mM MgCl2, 16 mM creatine phosphate, 36 μg/ml creatine kinase and 30 mM KCl) for 8 min at 37°C. CaCl2 (1.8 mM final) was added where indicated, followed by an additional 2 min incubation at 37°C. Increasing concentrations of BCCIPβ-(HIS)6 (0.75 μM, 1.5 μM, 3 μM, 4.5 μM) were added to the reactions followed by an 8 min incubation at 37°C. The addition of supercoiled pBluescript (35 μM base pairs) initiated the reaction. After a 6 min incubation at 37°C, SDS (0.5%) and Proteinase K (5 μg/ml) were added to deproteinize the reactions, followed by a 20 min incubation at 37°C. All reaction products were resolved on a 0.9% agarose gel, dried on DE81 anion exchange paper (GE Healthcare), visualized using a phosphorimager (Typhoon FLA 7000, GE Healthcare) and quantified using ImageQuant TL (GE Healthcare). The reactions monitoring RAD51-mediated D-loop formation as a function of time were carried out in the same manner as above, except each reaction was stopped at the indicated times. To test the species specificity, ScRad51 (1.5 μM) was incubated with 32P-OL90 (4.5 μM) in Buffer E (25 mM Tris-HCl pH 7.4, 0.1 μg/μl BSA, 1 mM DTT, 2 mM ATP, 2.4 mM MgCl2, 16 mM creatine phosphate, 36 μg/ml creatine kinase, and 30 mM KCl) for 10 min in a total reaction volume of 12.5 μl. ScRad54 (0.2 μM) or increasing concentrations of BCCIPβ-(HIS)6 (1.5 μM, 3 μM, 4.5 μM) were added to the reactions (as indicated) followed by an 8 min incubation at 37°C prior to the addition of pBluescript. After 8 min, the reaction products were deproteinized and analyzed as described above. Three independent experiments were tested for each D-loop reaction, and the average percent D-loop was graphed. Error bars represent s.e.m. Significance was determined by comparing the results of RAD51 to each of the conditions with BCCIPβ-(HIS)6 using a two-sample t-test.

BCCIPβ gel filtrationGel filtration was facilitated with the ÄKTA pure chromatography system (GE Healthcare). Purified BCCIPβ-(HIS)6 (100 μg) was loaded onto a 35 ml Sephacryl S-400 (GE Healthcare) size exclusion chromatography column (1 cm diameter), pre-equilibrated in Buffer B (20 mM KH2PO4 pH 7.5, 10% glycerol, 0.5 mM EDTA) with 100 mM KCl. Each fraction (40 fractions total, 1 ml each, 0.5 ml/min flow rate) was precipitated by the addition of deoxycholate (0.02%) and trichloroacetic acid (0.1%), resuspended in 20 μl of SDS loading dye, and resolved using 12% SDS-polyacrylamide gel electrophoresis. The gel was transferred to a nitrocellulose membrane (Whatman) and visualized using HRP-conjugated anti-poly-histidine antibodies (Sigma-Aldrich, A7058; 1:2000) in Buffer F (0.12% 4-chloro-1-naphthol (w/v), 0.12% H2O2 (v/v), in PBS). Molecular weight standards (Bio-Rad) were processed in the same manner as above. To establish molecular sizing for the fractions, molecular weight standards were plotted based on the log of their molecular weight (y-axis) and fraction number (x-axis) using KaleidaGraph 4.1.3, and the linear equation was derived using the linear curve fit tool.

BCCIPβ gel filtration

Gel filtration was facilitated with the ÄKTA pure chromatography system (GE Healthcare). Purified BCCIPβ-(HIS)6 (100 μg) was loaded onto a 35 ml Sephacryl S-400 (GE Healthcare) size exclusion chromatography column (1 cm diameter), pre-equilibrated in Buffer B (20 mM KH2PO4 pH 7.5, 10% glycerol, 0.5 mM EDTA) with 100 mM KCl. Each fraction (40 fractions total, 1 ml each, 0.5 ml/min flow rate) was precipitated by the addition of deoxycholate (0.02%) and trichloroacetic acid (0.1%), resuspended in 20 μl of SDS loading dye, and resolved using 12% SDS-polyacrylamide gel electrophoresis. The gel was transferred to a nitrocellulose membrane (Whatman) and visualized using HRP-conjugated anti-poly-histidine antibodies (Sigma-Aldrich, A7058; 1:2000) in Buffer F (0.12% 4-chloro-1-naphthol (w/v), 0.12% H2O2 (v/v), in PBS). Molecular weight standards (Bio-Rad) were processed in the same manner as above. To establish molecular sizing for the fractions, molecular weight standards were plotted based on the log of their molecular weight (y-axis) and fraction number (x-axis) using KaleidaGraph 4.1.3, and the linear equation was derived using the linear curve fit tool.

Nuclease protection assay32P-labeled OL90 (3 μM nucleotides) was incubated with RAD51 (0.4 μM) in Buffer D (25 mM Tris-HCl pH 7.4, 0.1 μg/μl BSA, 1 mM DTT, 2 mM ATP, 1.4 mM MgCl2, 16 mM creatine phosphate, 36 μg/ml creatine kinase and 30 mM KCl) for 10 min at 37°C. Increasing concentrations of BCCIPβ-(HIS)6 (0.4 μM, 0.8 μM, 1.2 μM) were added to the reaction and incubated for 8 min. Subsequently, 2 units of DNase I (Promega) were added followed by a 15 min incubation at 37°C. The reactions were stopped by the addition of equal parts SDS (0.8%) and Proteinase K (1.6 μg/ml). The reaction products were resolved using 10% non-denaturing polyacrylamide gel electrophoresis. The gels were then dried on Whatman cellulose chromatography paper (Sigma-Aldrich) and analyzed using a phosphorimager. Error bars represent s.e.m. The mean percent protection from three separate experiments was graphed.

Nuclease protection assay

32P-labeled OL90 (3 μM nucleotides) was incubated with RAD51 (0.4 μM) in Buffer D (25 mM Tris-HCl pH 7.4, 0.1 μg/μl BSA, 1 mM DTT, 2 mM ATP, 1.4 mM MgCl2, 16 mM creatine phosphate, 36 μg/ml creatine kinase and 30 mM KCl) for 10 min at 37°C. Increasing concentrations of BCCIPβ-(HIS)6 (0.4 μM, 0.8 μM, 1.2 μM) were added to the reaction and incubated for 8 min. Subsequently, 2 units of DNase I (Promega) were added followed by a 15 min incubation at 37°C. The reactions were stopped by the addition of equal parts SDS (0.8%) and Proteinase K (1.6 μg/ml). The reaction products were resolved using 10% non-denaturing polyacrylamide gel electrophoresis. The gels were then dried on Whatman cellulose chromatography paper (Sigma-Aldrich) and analyzed using a phosphorimager. Error bars represent s.e.m. The mean percent protection from three separate experiments was graphed.

Trypsin treatment of RAD51 complexesThe trypsin digest of RAD51 complexes was performed similar to a previously described procedure (27). Briefly, RAD51 (5 μM) incubated in 10 μl of Buffer G (25 mM Tris-HCl pH 7.4, 1 mM DTT, 1.4 mM MgCl2, 30 mM KCl) in the presence or absence of the indicated combinations of ATP (2 mM), ϕX174 circular ssDNA (30 μM nucleotides), CaCl2 (1.8 mM) and/or BCCIPβ-(HIS)6 (10 μM) for 10 min at 37°C. Trypsin (20 μg/ml) was added to each reaction and incubated for an additional 30 min. The reactions were inactivated by the addition of SDS loading dye (7.5 μl) and a 10 min incubation at 80°C. The reaction products were resolved using 16% SDS-polyacrylamide gel electrophoresis, followed by transfer to a nitrocellulose membrane (Whatman). The membrane was incubated with anti-RAD51 as the primary antibody (1:2000; ab63801, Abcam) and HRP-conjugated anti-rabbit IgG as the secondary antibody (1:2000, Sigma-Aldrich). The membrane was developed using Buffer F (0.12% 4-chloro-1-naphthol (w/v), 0.12% H2O2 (v/v), in phosphate buffered saline (PBS)), and imaged using Image Lab software (BioRad). The amount of undigested RAD51 fragments was plotted, and the amount of Fragments A, B, C and D was plotted based on relative intensities of each band. Results were derived from three separate experiments, and error bars represent s.e.m.

Trypsin treatment of RAD51 complexes

The trypsin digest of RAD51 complexes was performed similar to a previously described procedure (27). Briefly, RAD51 (5 μM) incubated in 10 μl of Buffer G (25 mM Tris-HCl pH 7.4, 1 mM DTT, 1.4 mM MgCl2, 30 mM KCl) in the presence or absence of the indicated combinations of ATP (2 mM), ϕX174 circular ssDNA (30 μM nucleotides), CaCl2 (1.8 mM) and/or BCCIPβ-(HIS)6 (10 μM) for 10 min at 37°C. Trypsin (20 μg/ml) was added to each reaction and incubated for an additional 30 min. The reactions were inactivated by the addition of SDS loading dye (7.5 μl) and a 10 min incubation at 80°C. The reaction products were resolved using 16% SDS-polyacrylamide gel electrophoresis, followed by transfer to a nitrocellulose membrane (Whatman). The membrane was incubated with anti-RAD51 as the primary antibody (1:2000; ab63801, Abcam) and HRP-conjugated anti-rabbit IgG as the secondary antibody (1:2000, Sigma-Aldrich). The membrane was developed using Buffer F (0.12% 4-chloro-1-naphthol (w/v), 0.12% H2O2 (v/v), in phosphate buffered saline (PBS)), and imaged using Image Lab software (BioRad). The amount of undigested RAD51 fragments was plotted, and the amount of Fragments A, B, C and D was plotted based on relative intensities of each band. Results were derived from three separate experiments, and error bars represent s.e.m.

ATP hydrolysisThe ATP hydrolysis assay was performed with RAD51 as previously described (28). Briefly, RAD51 (2 μM) was incubated with ϕx174 ssDNA (60 μM nucleotides) in Buffer G (25 mM Tris-HCl pH 7.4, 1 mM DTT, 1.4 mM MgCl2, 30 mM KCl) with 0.1 μg/μl BSA and 0.5 mM ATP for 5 min at 37°C for filament formation. BCCIPβ-(HIS)6 (4 μM) was added to the reaction where indicated. After an additional 5 min incubation at 37°C, 0.3 μCi of [32P-γ]-ATP was added to the reactions followed by the addition of an equal volume of 0.5 mM EDTA at the indicated time points to stop each reaction. The reaction products were resolved using thin-layer chromatography on polyethyleneimine-cellulose plates (Sigma-Aldrich). The plates were analyzed using a phosphorimager. The mean percent hydrolysis was graphed from three independent experiments. Error bars represent s.e.m, and significance was determined using a two-sample t-test and comparing RAD51 to RAD51+BCCIPβ-(HIS)6.

ATP hydrolysis

The ATP hydrolysis assay was performed with RAD51 as previously described (28). Briefly, RAD51 (2 μM) was incubated with ϕx174 ssDNA (60 μM nucleotides) in Buffer G (25 mM Tris-HCl pH 7.4, 1 mM DTT, 1.4 mM MgCl2, 30 mM KCl) with 0.1 μg/μl BSA and 0.5 mM ATP for 5 min at 37°C for filament formation. BCCIPβ-(HIS)6 (4 μM) was added to the reaction where indicated. After an additional 5 min incubation at 37°C, 0.3 μCi of [32P-γ]-ATP was added to the reactions followed by the addition of an equal volume of 0.5 mM EDTA at the indicated time points to stop each reaction. The reaction products were resolved using thin-layer chromatography on polyethyleneimine-cellulose plates (Sigma-Aldrich). The plates were analyzed using a phosphorimager. The mean percent hydrolysis was graphed from three independent experiments. Error bars represent s.e.m, and significance was determined using a two-sample t-test and comparing RAD51 to RAD51+BCCIPβ-(HIS)6.

RAD51 MANT-ADP releaseA Photon Technologies International Spectrometer Model 814 was used to monitor the fluorescence changes of unbound free MANT-ADP (ThermoFisher Scientific) and MANT-ADP bound by RAD51 in the presence of ssDNA in several experiments, similar to previous studies on DnaC and ClpB (29,30). (i) To obtain fluorescence emission spectra of MANT-ADP and the MANT-ADP-RAD51-ssDNA complex, MANT-ADP (0.5 μM) was preincubated in Buffer H (25 mM Tris-HCl, pH 7.4, 1.4 mM MgCl2, 32 mM KCl) with ϕX174 ssDNA (2.0 μM nucleotides) in the absence or presence of RAD51 (0.5 μM) or RAD51K133A (0.5 μM) for 30 min at 25°C with constant stirring in a quartz cuvette (1 cm path length). Samples were excited at 350 nm (5 nm bandwidth) and emission spectra were taken from 400–500 nm (5 nm bandwidth) at 1 nm steps. (ii) The dissociation of MANT-ADP from a pre-formed MANT-ADP-RAD51-ssDNA in the complex was observed as a decrease in the fluorescence signal. RAD51 (0.5 μM), ssDNA (2.0 μM nucleotides) and MANT-ADP (0.5 μM) in either Buffer H alone or Buffer H supplemented with CaCl2 (1.8 mM) were maintained in a quartz cuvette (1-cm path length) with constant stirring. To initiate ADP release in the presence of magnesium, either BCCIPβ-(HIS)6 (1.0 μM) or Buffer H was added to the cuvette. ADP release in the presence of calcium was initiated by the addition of either BCCIPβ-(HIS)6 (1.0 μM) with or without ATP (5 μM) or Buffer H with or without ATP (5 μM) to the cuvette. The resulting decrease in fluorescence was monitored in time base mode with an excitation wavelength of 350 nm (5 nm bandwidth) and an emission wavelength of 444 nm (5 nm bandwidth). Significance was determined using a two-sample t-test, comparing the 600 s time point for RAD51+Buffer to RAD51+BCCIPβ-(HIS)6 in the presence of magnesium and RAD51+Buffer+ATP to RAD51+BCCIPβ-(HIS)6+ATP in the presence of calcium.

RAD51 MANT-ADP release

A Photon Technologies International Spectrometer Model 814 was used to monitor the fluorescence changes of unbound free MANT-ADP (ThermoFisher Scientific) and MANT-ADP bound by RAD51 in the presence of ssDNA in several experiments, similar to previous studies on DnaC and ClpB (29,30). (i) To obtain fluorescence emission spectra of MANT-ADP and the MANT-ADP-RAD51-ssDNA complex, MANT-ADP (0.5 μM) was preincubated in Buffer H (25 mM Tris-HCl, pH 7.4, 1.4 mM MgCl2, 32 mM KCl) with ϕX174 ssDNA (2.0 μM nucleotides) in the absence or presence of RAD51 (0.5 μM) or RAD51K133A (0.5 μM) for 30 min at 25°C with constant stirring in a quartz cuvette (1 cm path length). Samples were excited at 350 nm (5 nm bandwidth) and emission spectra were taken from 400–500 nm (5 nm bandwidth) at 1 nm steps. (ii) The dissociation of MANT-ADP from a pre-formed MANT-ADP-RAD51-ssDNA in the complex was observed as a decrease in the fluorescence signal. RAD51 (0.5 μM), ssDNA (2.0 μM nucleotides) and MANT-ADP (0.5 μM) in either Buffer H alone or Buffer H supplemented with CaCl2 (1.8 mM) were maintained in a quartz cuvette (1-cm path length) with constant stirring. To initiate ADP release in the presence of magnesium, either BCCIPβ-(HIS)6 (1.0 μM) or Buffer H was added to the cuvette. ADP release in the presence of calcium was initiated by the addition of either BCCIPβ-(HIS)6 (1.0 μM) with or without ATP (5 μM) or Buffer H with or without ATP (5 μM) to the cuvette. The resulting decrease in fluorescence was monitored in time base mode with an excitation wavelength of 350 nm (5 nm bandwidth) and an emission wavelength of 444 nm (5 nm bandwidth). Significance was determined using a two-sample t-test, comparing the 600 s time point for RAD51+Buffer to RAD51+BCCIPβ-(HIS)6 in the presence of magnesium and RAD51+Buffer+ATP to RAD51+BCCIPβ-(HIS)6+ATP in the presence of calcium.

RESULTSBCCIPβ purificationA (HIS)6 tag was fused to the C-terminus of BCCIPβ cDNA by PCR to generate the BCCIPβ-(HIS)6 bacterial expression plasmid. A procedure for protein expression and purification was devised (Figure 1A) that utilized affinity and ion exchange chromatography to purify BCCIPβ-(HIS)6 to near homogeneity (Figure 1B). The sole form of BCCIPβ used in this study was BCCIPβ-(HIS)6, which will be referred to as BCCIPβ hereafter. Four separate protein preparations yielded equivalent biochemical results. While the predicted molecular weight of BCCIPβ is ∼36 kDa, the purified recombinant BCCIPβ migrates aberrantly on a SDS polyacrylamide gel at ∼47 kDa (Figure 1B) similar to a previous report (16).Figure 1.Purification of BCCIPβ. (A) Schematic for the chromatography procedure to purify BCCIPβ. S100 is separation by centrifugation at 100 000 ×g. (B) Purified BCCIPβ (0.5 μg) was resolved by gel electrophoresis on a 12% sodium dodecyl sulphate (SDS)-polyacrylamide gel and stained with Coomassie Blue.DNA binding activity of BCCIPβBCCIP was reported to be present in chromatin-bound nuclear foci (20) suggesting that BCCIP may possess the ability to bind DNA. To test this idea, a DNA electrophoretic mobility shift assay with plasmid length ϕX174 ssDNA and linearized ϕX174 dsDNA was employed. In this assay, increasing concentrations of BCCIPβ were incubated with ssDNA or dsDNA, and the reaction products were resolved on an agarose gel. BCCIPβ bound both ssDNA (Figure 2A) and dsDNA (Figure 2B). The ssDNA started to shift at a lower concentration of BCCIPβ (0.47 μM; Figure 2A, lane 3) than was seen with dsDNA (1.8 μM; Figure 2B, lane 5). At saturating concentrations, the ssDNA–BCCIPβ complex mostly entered the gel, whereas the dsDNA–BCCIPβ complex was trapped in the well of the agarose gel. To determine if there was a preference for binding ssDNA or dsDNA by BCCIPβ, a competition experiment was performed. Co-incubation of BCCIPβ with both ssDNA and dsDNA resulted in no significant difference in the ability of BCCIPβ to bind ssDNA or dsDNA (data not shown). These results indicate BCCIPβ harbors DNA binding activity with no apparent preference for ssDNA or dsDNA.Figure 2.BCCIPβ binds DNA. (A) BCCIPβ (0.24 μM, 0.47 μM, 0.96 μM, 1.8 μM, 2.8 μM and 4.7 μM; lanes 2–7, respectively) incubated with ϕX174 (+) ssDNA (ss; 30 μM nucleotides). (B) BCCIPβ (0.24 μM, 0.47 μM, 0.96 μM, 1.8 μM, 2.8 μM and 4.7 μM; lanes 2–7, respectively) was incubated with ϕX174 RF (I) dsDNA (ds; 30 μM base pairs). The reaction products were separated on a 1.0% agarose gel, and were stained with ethidium bromide. Lane 1 contained no protein, and lane 8 was deproteinized with SDS and Proteinase K (S/P) prior to loading.BCCIPβ interacts with RAD51Previously, BCCIP was reported to co-localize with RAD51 foci in HT1080 cells (20). Additionally, BCCIPβ was shown to co-immunoprecipitate with RAD51 from HeLa cells (20). These results suggest that BCCIPβ interacts with RAD51. To determine if there is a direct, physical interaction between purified BCCIPβ and RAD51, an affinity pull-down assay was performed. In this assay, purified BCCIPβ and RAD51 were incubated to allow a potential complex to form. The addition of Ni-NTA agarose beads allowed for the capture of BCCIPβ through its six-histidine epitope tag. The supernatant was removed, and the beads were washed to remove non-specifically bound protein. BCCIPβ and RAD51 were both present in the elution (Figure 3A) indicating that RAD51 formed a stable complex with BCCIPβ. This interaction was not due to non-specific interactions of RAD51 with the Ni-NTA beads, since RAD51 was present only in the supernatant when incubated with Ni-NTA in the absence of BCCIPβ (Figure 3A). To determine if the interaction between RAD51 and BCCIPβ was evolutionarily conserved, the same pull-down assay was performed by incubation of BCCIPβ with purified Saccharomyces cerevisiae Rad51 (24) instead of human RAD51. As shown in Figure 3B, ScRad51 was only present in the supernatant, indicating that ScRad51 failed to interact with BCCIPβ. These data suggest BCCIPβ interacts specifically with human RAD51.Figure 3.BCCIPβ interacts with human RAD51. (A) Human RAD51 (5 μg) was incubated with Ni-NTA beads in the presence (lanes 1–3) or absence of BCCIPβ-(HIS)6 (5 μg; lanes 4–6). (B) ScRad51 (5 μg) was incubated with Ni-NTA in the absence (lanes 1–3) or presence of BCCIPβ-(HIS)6 (5 μg; lanes 4–6). The supernatant was removed, the beads were washed, and the bound proteins were eluted with SDS. The supernatant (S), wash (W) and eluate (E), were resolved using SDS-PAGE and stained with Coomassie blue.Stimulation of RAD51-mediated D-loop formation by BCCIPβDuring HR, a ssDNA overhang from the processed DSB is used by RAD51 to invade the sister chromatid in search of homology. Several HR accessory factors are known to interact with RAD51 to enhance homologous DNA pairing activity (31–36). To determine if BCCIPβ enhanced the RAD51-mediated homologous DNA pairing activity, we used an in vitro D-loop formation assay (22). In this assay, a single-stranded radiolabeled oligonucleotide is incubated with RAD51 to form a presynaptic filament that invades supercoiled duplex DNA to search for the homologous sequence. Once homology is located, the complementary strand is displaced, forming a D-loop (Figure 4A). Using this assay, our initial attempts showed that RAD51-mediated D-loop formation was not affected by the presence of BCCIPβ in the presence of magnesium (Figure 4B). Previous reports demonstrated that the presence of calcium enhanced RAD51 D-loop formation by inhibiting ATP hydrolysis, which results in a more stable presynaptic filament (37). With this in mind, we repeated the experiment with magnesium and calcium present. Our results show that BCCIPβ stimulated RAD51-mediated D-loop formation nearly 2-fold (P-value *< 0.05, **< 0.01) in a concentration-dependent manner (Figure 4C) in the presence of magnesium and calcium. This D-loop formation was dependent upon ATP and RAD51 (Figure 4C). To test whether BCCIPβ functions with RAD51 in a species-specific manner, we performed the D-loop assay with ScRad51. The D-loop formation activity of ScRad51 is barely detectable without the addition of the DNA translocase, ScRad54 (38). Our results show BCCIPβ was unable to enhance ScRad51 D-loop formation in the presence of magnesium regardless of the absence (Figure 4D) or presence of calcium (data not shown), which was expected since BCCIPβ did not interact with ScRad51.Figure 4.BCCIPβ stimulates human RAD51-mediated D-loop formation in the presence of calcium. (A) Schematic of the D-loop assay. (B) RAD51 (1.5 μM) was incubated with 32P-OL90 (ssDNA; 4.5 μM) in the absence (lane 2) or presence of increasing concentrations of BCCIPβ (lanes 3–6; 0.75 μM, 1.5 μM, 3 μM, 4.5 μM, respectively). The supercoiled plasmid pBluescript (sc; 35 μM base pairs) was introduced to the reaction. After 6 min, the reactions were deproteinized with SDS and Proteinase K. Lane 1 was devoid of protein, lane 7 contained RAD51 and BCCIPβ (4.5 μM) in the absence of ATP, and lane 8 contained BCCIPβ (4.5 μM) alone. (C) The reactions in B were tested in the presence of (1.8 mM) calcium. (D) The D-loop assay was performed with ScRad51 (1.5 μM) in presence of BCCIPβ (lanes 4–6; 1.5 μM, 3 μM, 4.5 μM, respectively) or ScRad54 (0.2 μM; lane 3). All reaction products were subjected to 0.9% agarose gel electrophoresis and visualized using a phosphorimager. Lane 1 contained no protein, and lane 2 contained only ScRad51. S.e.m. (n = 3) are plotted as error bars; P-value *< 0.05, **< 0.01.The affinity pull-down assay suggested that BCCIPβ did not completely bind equimolar amounts of RAD51 (Figure 3A, lane 1). In addition, BCCIPβ stimulated RAD51 D-loop formation in a concentration-dependent manner, further suggesting that a specific stoichiometric relationship may be necessary for BCCIPβ to stimulate RAD51 (Figure 4C). To further explore this idea, we examined the ability of BCCIPβ to stimulate RAD51-mediated D-loop formation as a function of time with RAD51 alone and in combination with BCCIPβ in stoichiometric molar ratios of 1:1, 1:2 and 1:3 (Figure 5A). RAD51 alone produced ∼13% D-loop product. When equal molar (1:1) amounts of RAD51 and BCCIPβ were used, there was not a significant increase in D-loop formation. A molar ratio of 1:2 or 1:3 (RAD51:BCCIPβ) resulted in a 2-fold increase (∼27%, P-value *< 0.05, **< 0.01) in the amount of D-loop formation (Figure 5A). Since a 1:3 molar ratio (RAD51:BCCIPβ) yielded the same amount of D-loop product as a 1:2 molar ratio (RAD51:BCCIPβ), we propose the preferred stoichiometric ratio for BCCIPβ stimulation of RAD51-mediated D-loop formation is 1:2 (RAD51:BCCIPβ).Figure 5.BCCIPβ is a homodimer that stimulates of RAD51-mediated D-loop formation. (A) RAD51 (1.5 μM) was incubated with ssDNA (4.5 μM) in the absence (lanes 2–6) or presence of increasing concentrations of BCCIPβ (1.5 μM, lanes 8–12; 3 μM, lanes 14–18; 4.5 μM, lanes 20–24). Lane 1 was devoid of any protein, and each concentration of BCCIPβ (1.5 μM, 3 μM, 4.5 μM; lanes 7, 13 and 19, respectively) was tested in the absence of RAD51. All reactions took place in the presence of calcium. The reactions were initiated by the addition of supercoiled DNA (35 μM base pairs) and deproteinized with SDS and Proteinase K at the indicated times. Reaction products were separated on a 0.9% agarose gel and visualized using a phosphorimager. (B) Gel filtration standards (bovine thyroglobulin 670 kDa, bovine γ-globulin 158 kDa, chicken ovalbumin 44 kDa and horse myoglobin 17 kDa) were used to calibrate a 35 ml S-400 gel filtration column. The eluate fraction (x-axis) and the log of the molecular weights were plotted (y-axis). The linear fit yielded: y = 6.8799 - 0.071211x and R = 0.99479. The expected values for a BCCIPβ monomer and a BCCIPβ dimer are indicated. (C) BCCIPβ (100 μg) was sized using the 35 ml S-400 gel filtration column. A western blot was performed, and HRP-conjugated anti-poly-histidine was used to develop the membrane. Error bars represent s.e.m. (n = 3); P-value *< 0.05, **< 0.01.BCCIPβ is a homodimerThe observation that there is a preferred stoichiometric molar ratio of two BCCIPβ molecules to one RAD51 molecule for optimal enhancement of RAD51 homologous DNA pairing suggested the potential for BCCIPβ to form a homodimer. To test this notion, we utilized size exclusion chromatography to monitor the migration of BCCIPβ through a size exclusion column. The column was calibrated with bovine thyroglobulin (670 kDa), bovine γ-globulin (158 kDa), chicken ovalbumin (44 kDa) and horse myoglobin (17 kDa), and their elution positions are as indicated (Figure 5B and C). Based on the elution positions of the molecular weight standards, we plotted the log of the molecular weight (y-axis) of the protein standards and their elution fraction (x-axis) to yield the fitted equation y = 6.8799 – 0.071211x with R = 0.99479 (Figure 5B). Based on this equation, BCCIPβ was expected to elute in fraction 32 (∼36 500 Da). However, the peak fraction of BCCIPβ eluted in fraction 28 (Figure 5C), which correlates with a globular protein having a molecular weight of ∼73 000 Da. These results are consistent with BCCIPβ forming a homodimer in solution.BCCIPβ is unable to stabilize RAD51 filamentA number of HR accessory factors enhance the recombination activity of RAD51 through the stabilization of the RAD51 nucleoprotein filament including RAD54, BRCA2, HOP2-MND1, HED1, RAD55/RAD57 and SWI5-SFR1 (39–44). To determine whether BCCIPβ stabilized the RAD51 filament, a nuclease protection assay (22,26,45,46) was employed. In this assay, RAD51 was incubated with a radiolabeled ssDNA oligonucleotide in the presence of ATP to allow for the formation of a presynaptic filament. The resulting nucleoprotein filament was then subjected to DNase I treatment. If BCCIPβ possessed the ability to stabilize the RAD51 filament, then nucleolytic degradation of the oligonucleotide ssDNA would decrease in the presence of BCCIPβ (Figure 6A). Our data show that increasing concentrations of BCCIPβ were unable to stabilize the RAD51 filament regardless of the presence of magnesium or calcium (Figure 6B and data not shown, respectively). These results and the observation that enhancement of RAD51 D-loop formation activity was dependent on a presynaptic filament stabilized by the presence of calcium, indicate BCCIPβ stimulation of RAD51-mediated D-loop formation occurs through a different mechanism.Figure 6.RAD51 nucleoprotein filament is not stabilized by BCCIPβ. (A) Schematic of the nuclease protection assay (adapted from Chi et al. 2007 (41)). (B) RAD51 (0.4 μM) was incubated with radiolabeled ssDNA (3 μM nucleotides) alone (lane 3) or with increasing concentrations of BCCIPβ (lanes 4–6; 0.4 μM, 0.8 μM, 1.2 μM, respectively). The reaction was initiated by the addition of DNase I (2 units). Reaction products were then deproteinized and separated using 10% polyacrylamide gel electrophoresis. Lane 1 contained radiolabeled ssDNA alone, lane 2 contained only radiolabeled ssDNA in the presence of DNase I, and lane 7 contained BCCIPβ (1.6 μM) in the absence of RAD51 but in the presence of DNase I. deg, degraded radiolabeled ssDNA.BCCIPβ induces conformational changes in RAD51The observation that BCCIPβ stimulated the D-loop formation activity of RAD51 without stabilizing the presynaptic filament led us to hypothesize that physical interaction with BCCIPβ may induce a change within the RAD51–ssDNA presynaptic filament to an active conformation capable of catalyzing homologous DNA pairing activity. To investigate this possibility, a limited-trypsin proteolytic digestion of RAD51 either with or without BCCIPβ was performed. Limited-proteolysis often provides information about conformational changes that result from the interaction of a protein with effector molecules such as DNA and/or other proteins (27,47–49). If the interaction of an effector molecule causes conformational changes within the protein of interest, then protease-sensitive regions may become resistant to proteolytic cleavage while new cleavage sites are potentially exposed, yielding an altered proteolytic fingerprint of the protein. Here, we incubated RAD51 or RAD51 and BCCIPβ with or without calcium under reaction conditions conducive to nucleoprotein filament formation, followed by the addition of trypsin. Changes in the protein fragmentation were monitored by western blot analysis using antibodies raised against RAD51. Our results, performed in the presence of magnesium (all reactions contain magnesium) and calcium (where indicated), show RAD51 is highly susceptible to trypsin proteolysis with only 10% of RAD51 remaining uncleaved (Figure 7A and B, lane 2) in the absence of effector molecules. The presence of ATP alone resulted in a slight increase in protection with 25% uncleaved RAD51 present (Figure 7A and B, lane 3). Approximately 45% of the RAD51 was protected from cleavage by trypsin in the presence of ATP and ssDNA under conditions that promote presynaptic filament formation (Figure 7A and B, lane 4). When BCCIPβ was present with a RAD51 presynaptic filament, the amount of uncleaved RAD51 increased to 70% (Figure 7A and B, compare lanes 4 and 6). In addition, there was a 5-fold decrease in the amount of Fragment B and a 16-fold decrease in Fragment D (Figure 7A and B, compare lanes 4 to 6). When calcium was included with a RAD51 presynaptic filament, 79% of RAD51 remained uncleaved (Figure 7A and B, lane 5). The addition of both calcium and BCCIPβ under the same conditions led to approximately the same level of uncleaved RAD51 (80%; Figure 7A and B, lane 7). However, there was an 8-fold decrease in Fragment B and a 10-fold decrease in the amount of Fragment D (Figure 7A and B, compare lanes 5 and 7). Interestingly, Fragments E and F were only observed when the RAD51 presynaptic filament was proteolytically cleaved in the presence of calcium (Figure 7A and B, lane 5). The presence of BCCIPβ altered this cleavage pattern since Fragment E was no longer observed (Figure 7A and B, compare lanes 5 and 7). Notably, Fragment G was only present when RAD51 was in the presence of ATP or in the presence of BCCIPβ alone (Figure 7A and B, lanes 3 and 9). We also observed a 9-fold increase in the amount of Fragment C and 45-fold increase in Fragment D when compared to trypsin digestion of RAD51 alone (Figure 7A and B, compare lanes 2 and 9). As expected, when RAD51 was not included, there was no signal, demonstrating the specificity of the antibodies used for RAD51 (Figure 7A and B, lane 8). Taken together, BCCIPβ has a significant effect on the proteolytic digestion fingerprint of RAD51 and the RAD51 presynaptic filament, suggesting that interaction with BCCIPβ results in conformational changes within RAD51.Figure 7.Interaction with BCCIPβ induces conformational changes in RAD51. (A) RAD51 (5 μM) was incubated with trypsin (20 μg/ml) in the presence and absence of ATP (2 mM), ϕX174 ssDNA (30 μM nucleotides), calcium (1.8 mM) and BCCIPβ (10 μM), as indicated. The reactions were stopped with SDS and heat. The reaction products were resolved using SDS-PAGE followed by western blot analysis. Antibodies against RAD51 were used to develop the membrane. (B) The amounts of each band from undigested RAD51 and Fragments A, B, C and D were graphed based on the relative intensity of each band. Quantitation of the proteolytic fragmentation of RAD51 was determined from three independent experiments.ATP hydrolysis by RAD51 is stimulated by BCCIPβThe enhanced D-loop formation attributed to RAD51 by the presence of calcium is through the inhibition of ATP hydrolysis (37). Given that BCCIPβ failed to stabilize the RAD51 presynaptic filament, yet induced a conformational change in the RAD51 filament, we asked whether BCCIPβ had an effect on the ATP hydrolysis activity of RAD51. To monitor RAD51 ATP hydrolysis, RAD51 was incubated with [32P-γ]-ATP in the presence of magnesium and in the absence and presence of ssDNA under conditions that favor presynaptic filament formation with or without BCCIPβ. At the given times, each reaction was stopped, and the amount of hydrolyzed [32P-γ]-ATP was analyzed. Our results indicate that BCCIPβ enhances the ATP hydrolysis activity of RAD51 in the presence of ssDNA ∼1.7-fold (P-value *< 0.05, Figure 8A and B). Upon further investigation, we found that there was no enhancement of ATP hydrolysis by RAD51 in the absence of ssDNA, (Figure 8A, compare lanes 2 and 4). BCCIPβ stimulated the ATP hydrolysis activity of RAD51 in the presence of ssDNA. These results are in agreement with our finding that BCCIPβ was only able to stimulate RAD51 D-loop formation when a stable RAD51 filament was present. As expected, no ATP hydrolysis activity was observed with BCCIPβ alone or in combination with ssDNA (Figure 8A, lane 3 and 6, respectively). These results demonstrate the ability of BCCIPβ to enhance ATP hydrolysis of RAD51 when bound to ssDNA, suggesting that enhanced D-loop activity was not due to reduced ATP hydrolysis activity by RAD51.Figure 8.BCCIPβ stimulates RAD51 ATP hydrolysis and promotes ADP release. (A) RAD51 (0.5 μM) ATP hydrolysis assay in the presence or absence of ϕX174 ssDNA (60 μM nucleotides) and BCCIPβ (1 μM). (B) Time course analysis of RAD51 (0.5 μM) ATP hydrolysis in the presence of ϕX174 ssDNA (60 μM nucleotides), with or without BCCIPβ (1 μM). Error bars represent s.e.m. (n = 3); P-value *< 0.05. (C) Fluorescence spectra (400–500 nm) of MANT-ADP (0.5 μM) and ϕX174 ssDNA (2.0 μM nucleotides) in the absence or presence of RAD51 or RAD51K133A (0.5 μM). (D) Time course analysis of MANT-ADP dissociation from a RAD51-MANT-ADP-ssDNA complex in the presence of magnesium with or without BCCIPβ (1 μM). (E) Time course analysis of MANT-ADP release from the RAD51-MANT-ADP-ssDNA filament in the presence of calcium with or without BCCIPβ (1 μM) and/or ATP (5 μM), as indicated. P-value *< 0.05 and **< 0.01 is indicative of the 600 s time point.BCCIPβ mediates ADP releaseA potential mechanism for the stimulation of RAD51 homologous DNA pairing activity by BCCIPβ is through the enhanced release of ADP from the RAD51 filament. In support of this idea, Shim et al. and Su et al. previously reported that XRCC2 and SWI5-SFR1, respectively, promote the release of ADP from a RAD51 filament (28,50). To monitor the effect of BCCIPβ on the RAD51-ADP-ssDNA filament, we needed to monitor the ability of RAD51 to form a filament in the presence of ADP. To do this, we incubated fluorescently labeled ADP (MANT-ADP) and ssDNA in the presence or absence of RAD51. After allowing ample time for filament formation, a fluorescence scan from 400 to 500 nm was performed. We found a ∼33% increase in the relative fluorescence and a 4 nm red shift in peak fluorescence (445–441 nm) when RAD51 was in the reaction, indicating the binding of MANT-ADP by RAD51–ssDNA (Figure 8C). To confirm that the increase in fluorescence and red-shift in peak fluorescence were due to RAD51 binding MANT-ADP, we substituted RAD51 for the RAD51K133A variant, known to have low affinity for nucleotides (26,27,51). As expected, there was no difference in the fluorescence spectra of MANT-ADP in buffer regardless of the presence of RAD51K133A indicating the increase in fluorescence and red-shift in peak fluorescence was due to RAD51 binding MANT-ADP (Figure 8C). Since there was an increase in fluorescence and a shift in the peak of the fluorescent signal upon RAD51 binding MANT-ADP, we performed a time course experiment that monitored the dissociation of MANT-ADP as a decrease in fluorescent signal. Here, we incubated RAD51 with MANT-ADP and ssDNA in the presence of magnesium to allow for the formation of the RAD51-ADP-ssDNA filament. Upon the addition of BCCIPβ or reaction buffer containing magnesium, the decrease in fluorescence was monitored at 445 nm. As shown in Figure 8D, the addition of BCCIPβ in reaction buffer containing magnesium resulted in a dramatic decrease (t1/2 = 32.0 ± 2.8 s) in the relative fluorescence compared to the addition of buffer alone (t1/2 = 52.6 ± 4.3 s; P-value **< 0.01). Next, we incubated RAD51 with MANT-ADP and ssDNA in the presence of calcium during the formation of the RAD51-ADP-ssDNA filament. The addition of reaction buffer containing calcium yielded a slow decrease in fluorescence (Figure 8E). Surprisingly, the addition of BCCIPβ in reaction buffer containing calcium resulted in no significant change in the level of fluorescence (Figure 8E). We found that the addition of BCCIPβ in combination with 10-fold excess ATP resulted in a significant decrease in fluorescence (t1/2 = 30.0 ± 3.1 s) compared to the addition of reaction buffer containing 10-fold excess ATP alone (t1/2 = 82.4 ± 4.9 s P-value *< 0.05) (Figure 8E). Our results suggest that BCCIPβ promotes the release of ADP from the RAD51-ADP-ssDNA filament in the presence of magnesium. In the presence of calcium, the release of ADP from RAD51–ssDNA is stimulated by BCCIPβ when ATP is present.

BCCIPβ purificationA (HIS)6 tag was fused to the C-terminus of BCCIPβ cDNA by PCR to generate the BCCIPβ-(HIS)6 bacterial expression plasmid. A procedure for protein expression and purification was devised (Figure 1A) that utilized affinity and ion exchange chromatography to purify BCCIPβ-(HIS)6 to near homogeneity (Figure 1B). The sole form of BCCIPβ used in this study was BCCIPβ-(HIS)6, which will be referred to as BCCIPβ hereafter. Four separate protein preparations yielded equivalent biochemical results. While the predicted molecular weight of BCCIPβ is ∼36 kDa, the purified recombinant BCCIPβ migrates aberrantly on a SDS polyacrylamide gel at ∼47 kDa (Figure 1B) similar to a previous report (16).Figure 1.Purification of BCCIPβ. (A) Schematic for the chromatography procedure to purify BCCIPβ. S100 is separation by centrifugation at 100 000 ×g. (B) Purified BCCIPβ (0.5 μg) was resolved by gel electrophoresis on a 12% sodium dodecyl sulphate (SDS)-polyacrylamide gel and stained with Coomassie Blue.

BCCIPβ purification

A (HIS)6 tag was fused to the C-terminus of BCCIPβ cDNA by PCR to generate the BCCIPβ-(HIS)6 bacterial expression plasmid. A procedure for protein expression and purification was devised (Figure 1A) that utilized affinity and ion exchange chromatography to purify BCCIPβ-(HIS)6 to near homogeneity (Figure 1B). The sole form of BCCIPβ used in this study was BCCIPβ-(HIS)6, which will be referred to as BCCIPβ hereafter. Four separate protein preparations yielded equivalent biochemical results. While the predicted molecular weight of BCCIPβ is ∼36 kDa, the purified recombinant BCCIPβ migrates aberrantly on a SDS polyacrylamide gel at ∼47 kDa (Figure 1B) similar to a previous report (16).

Purification of BCCIPβ. (A) Schematic for the chromatography procedure to purify BCCIPβ. S100 is separation by centrifugation at 100 000 ×g. (B) Purified BCCIPβ (0.5 μg) was resolved by gel electrophoresis on a 12% sodium dodecyl sulphate (SDS)-polyacrylamide gel and stained with Coomassie Blue.

DNA binding activity of BCCIPβBCCIP was reported to be present in chromatin-bound nuclear foci (20) suggesting that BCCIP may possess the ability to bind DNA. To test this idea, a DNA electrophoretic mobility shift assay with plasmid length ϕX174 ssDNA and linearized ϕX174 dsDNA was employed. In this assay, increasing concentrations of BCCIPβ were incubated with ssDNA or dsDNA, and the reaction products were resolved on an agarose gel. BCCIPβ bound both ssDNA (Figure 2A) and dsDNA (Figure 2B). The ssDNA started to shift at a lower concentration of BCCIPβ (0.47 μM; Figure 2A, lane 3) than was seen with dsDNA (1.8 μM; Figure 2B, lane 5). At saturating concentrations, the ssDNA–BCCIPβ complex mostly entered the gel, whereas the dsDNA–BCCIPβ complex was trapped in the well of the agarose gel. To determine if there was a preference for binding ssDNA or dsDNA by BCCIPβ, a competition experiment was performed. Co-incubation of BCCIPβ with both ssDNA and dsDNA resulted in no significant difference in the ability of BCCIPβ to bind ssDNA or dsDNA (data not shown). These results indicate BCCIPβ harbors DNA binding activity with no apparent preference for ssDNA or dsDNA.Figure 2.BCCIPβ binds DNA. (A) BCCIPβ (0.24 μM, 0.47 μM, 0.96 μM, 1.8 μM, 2.8 μM and 4.7 μM; lanes 2–7, respectively) incubated with ϕX174 (+) ssDNA (ss; 30 μM nucleotides). (B) BCCIPβ (0.24 μM, 0.47 μM, 0.96 μM, 1.8 μM, 2.8 μM and 4.7 μM; lanes 2–7, respectively) was incubated with ϕX174 RF (I) dsDNA (ds; 30 μM base pairs). The reaction products were separated on a 1.0% agarose gel, and were stained with ethidium bromide. Lane 1 contained no protein, and lane 8 was deproteinized with SDS and Proteinase K (S/P) prior to loading.

DNA binding activity of BCCIPβ

BCCIP was reported to be present in chromatin-bound nuclear foci (20) suggesting that BCCIP may possess the ability to bind DNA. To test this idea, a DNA electrophoretic mobility shift assay with plasmid length ϕX174 ssDNA and linearized ϕX174 dsDNA was employed. In this assay, increasing concentrations of BCCIPβ were incubated with ssDNA or dsDNA, and the reaction products were resolved on an agarose gel. BCCIPβ bound both ssDNA (Figure 2A) and dsDNA (Figure 2B). The ssDNA started to shift at a lower concentration of BCCIPβ (0.47 μM; Figure 2A, lane 3) than was seen with dsDNA (1.8 μM; Figure 2B, lane 5). At saturating concentrations, the ssDNA–BCCIPβ complex mostly entered the gel, whereas the dsDNA–BCCIPβ complex was trapped in the well of the agarose gel. To determine if there was a preference for binding ssDNA or dsDNA by BCCIPβ, a competition experiment was performed. Co-incubation of BCCIPβ with both ssDNA and dsDNA resulted in no significant difference in the ability of BCCIPβ to bind ssDNA or dsDNA (data not shown). These results indicate BCCIPβ harbors DNA binding activity with no apparent preference for ssDNA or dsDNA.

BCCIPβ binds DNA. (A) BCCIPβ (0.24 μM, 0.47 μM, 0.96 μM, 1.8 μM, 2.8 μM and 4.7 μM; lanes 2–7, respectively) incubated with ϕX174 (+) ssDNA (ss; 30 μM nucleotides). (B) BCCIPβ (0.24 μM, 0.47 μM, 0.96 μM, 1.8 μM, 2.8 μM and 4.7 μM; lanes 2–7, respectively) was incubated with ϕX174 RF (I) dsDNA (ds; 30 μM base pairs). The reaction products were separated on a 1.0% agarose gel, and were stained with ethidium bromide. Lane 1 contained no protein, and lane 8 was deproteinized with SDS and Proteinase K (S/P) prior to loading.

BCCIPβ interacts with RAD51Previously, BCCIP was reported to co-localize with RAD51 foci in HT1080 cells (20). Additionally, BCCIPβ was shown to co-immunoprecipitate with RAD51 from HeLa cells (20). These results suggest that BCCIPβ interacts with RAD51. To determine if there is a direct, physical interaction between purified BCCIPβ and RAD51, an affinity pull-down assay was performed. In this assay, purified BCCIPβ and RAD51 were incubated to allow a potential complex to form. The addition of Ni-NTA agarose beads allowed for the capture of BCCIPβ through its six-histidine epitope tag. The supernatant was removed, and the beads were washed to remove non-specifically bound protein. BCCIPβ and RAD51 were both present in the elution (Figure 3A) indicating that RAD51 formed a stable complex with BCCIPβ. This interaction was not due to non-specific interactions of RAD51 with the Ni-NTA beads, since RAD51 was present only in the supernatant when incubated with Ni-NTA in the absence of BCCIPβ (Figure 3A). To determine if the interaction between RAD51 and BCCIPβ was evolutionarily conserved, the same pull-down assay was performed by incubation of BCCIPβ with purified Saccharomyces cerevisiae Rad51 (24) instead of human RAD51. As shown in Figure 3B, ScRad51 was only present in the supernatant, indicating that ScRad51 failed to interact with BCCIPβ. These data suggest BCCIPβ interacts specifically with human RAD51.Figure 3.BCCIPβ interacts with human RAD51. (A) Human RAD51 (5 μg) was incubated with Ni-NTA beads in the presence (lanes 1–3) or absence of BCCIPβ-(HIS)6 (5 μg; lanes 4–6). (B) ScRad51 (5 μg) was incubated with Ni-NTA in the absence (lanes 1–3) or presence of BCCIPβ-(HIS)6 (5 μg; lanes 4–6). The supernatant was removed, the beads were washed, and the bound proteins were eluted with SDS. The supernatant (S), wash (W) and eluate (E), were resolved using SDS-PAGE and stained with Coomassie blue.

BCCIPβ interacts with RAD51

Previously, BCCIP was reported to co-localize with RAD51 foci in HT1080 cells (20). Additionally, BCCIPβ was shown to co-immunoprecipitate with RAD51 from HeLa cells (20). These results suggest that BCCIPβ interacts with RAD51. To determine if there is a direct, physical interaction between purified BCCIPβ and RAD51, an affinity pull-down assay was performed. In this assay, purified BCCIPβ and RAD51 were incubated to allow a potential complex to form. The addition of Ni-NTA agarose beads allowed for the capture of BCCIPβ through its six-histidine epitope tag. The supernatant was removed, and the beads were washed to remove non-specifically bound protein. BCCIPβ and RAD51 were both present in the elution (Figure 3A) indicating that RAD51 formed a stable complex with BCCIPβ. This interaction was not due to non-specific interactions of RAD51 with the Ni-NTA beads, since RAD51 was present only in the supernatant when incubated with Ni-NTA in the absence of BCCIPβ (Figure 3A). To determine if the interaction between RAD51 and BCCIPβ was evolutionarily conserved, the same pull-down assay was performed by incubation of BCCIPβ with purified Saccharomyces cerevisiae Rad51 (24) instead of human RAD51. As shown in Figure 3B, ScRad51 was only present in the supernatant, indicating that ScRad51 failed to interact with BCCIPβ. These data suggest BCCIPβ interacts specifically with human RAD51.

BCCIPβ interacts with human RAD51. (A) Human RAD51 (5 μg) was incubated with Ni-NTA beads in the presence (lanes 1–3) or absence of BCCIPβ-(HIS)6 (5 μg; lanes 4–6). (B) ScRad51 (5 μg) was incubated with Ni-NTA in the absence (lanes 1–3) or presence of BCCIPβ-(HIS)6 (5 μg; lanes 4–6). The supernatant was removed, the beads were washed, and the bound proteins were eluted with SDS. The supernatant (S), wash (W) and eluate (E), were resolved using SDS-PAGE and stained with Coomassie blue.

Stimulation of RAD51-mediated D-loop formation by BCCIPβDuring HR, a ssDNA overhang from the processed DSB is used by RAD51 to invade the sister chromatid in search of homology. Several HR accessory factors are known to interact with RAD51 to enhance homologous DNA pairing activity (31–36). To determine if BCCIPβ enhanced the RAD51-mediated homologous DNA pairing activity, we used an in vitro D-loop formation assay (22). In this assay, a single-stranded radiolabeled oligonucleotide is incubated with RAD51 to form a presynaptic filament that invades supercoiled duplex DNA to search for the homologous sequence. Once homology is located, the complementary strand is displaced, forming a D-loop (Figure 4A). Using this assay, our initial attempts showed that RAD51-mediated D-loop formation was not affected by the presence of BCCIPβ in the presence of magnesium (Figure 4B). Previous reports demonstrated that the presence of calcium enhanced RAD51 D-loop formation by inhibiting ATP hydrolysis, which results in a more stable presynaptic filament (37). With this in mind, we repeated the experiment with magnesium and calcium present. Our results show that BCCIPβ stimulated RAD51-mediated D-loop formation nearly 2-fold (P-value *< 0.05, **< 0.01) in a concentration-dependent manner (Figure 4C) in the presence of magnesium and calcium. This D-loop formation was dependent upon ATP and RAD51 (Figure 4C). To test whether BCCIPβ functions with RAD51 in a species-specific manner, we performed the D-loop assay with ScRad51. The D-loop formation activity of ScRad51 is barely detectable without the addition of the DNA translocase, ScRad54 (38). Our results show BCCIPβ was unable to enhance ScRad51 D-loop formation in the presence of magnesium regardless of the absence (Figure 4D) or presence of calcium (data not shown), which was expected since BCCIPβ did not interact with ScRad51.Figure 4.BCCIPβ stimulates human RAD51-mediated D-loop formation in the presence of calcium. (A) Schematic of the D-loop assay. (B) RAD51 (1.5 μM) was incubated with 32P-OL90 (ssDNA; 4.5 μM) in the absence (lane 2) or presence of increasing concentrations of BCCIPβ (lanes 3–6; 0.75 μM, 1.5 μM, 3 μM, 4.5 μM, respectively). The supercoiled plasmid pBluescript (sc; 35 μM base pairs) was introduced to the reaction. After 6 min, the reactions were deproteinized with SDS and Proteinase K. Lane 1 was devoid of protein, lane 7 contained RAD51 and BCCIPβ (4.5 μM) in the absence of ATP, and lane 8 contained BCCIPβ (4.5 μM) alone. (C) The reactions in B were tested in the presence of (1.8 mM) calcium. (D) The D-loop assay was performed with ScRad51 (1.5 μM) in presence of BCCIPβ (lanes 4–6; 1.5 μM, 3 μM, 4.5 μM, respectively) or ScRad54 (0.2 μM; lane 3). All reaction products were subjected to 0.9% agarose gel electrophoresis and visualized using a phosphorimager. Lane 1 contained no protein, and lane 2 contained only ScRad51. S.e.m. (n = 3) are plotted as error bars; P-value *< 0.05, **< 0.01.The affinity pull-down assay suggested that BCCIPβ did not completely bind equimolar amounts of RAD51 (Figure 3A, lane 1). In addition, BCCIPβ stimulated RAD51 D-loop formation in a concentration-dependent manner, further suggesting that a specific stoichiometric relationship may be necessary for BCCIPβ to stimulate RAD51 (Figure 4C). To further explore this idea, we examined the ability of BCCIPβ to stimulate RAD51-mediated D-loop formation as a function of time with RAD51 alone and in combination with BCCIPβ in stoichiometric molar ratios of 1:1, 1:2 and 1:3 (Figure 5A). RAD51 alone produced ∼13% D-loop product. When equal molar (1:1) amounts of RAD51 and BCCIPβ were used, there was not a significant increase in D-loop formation. A molar ratio of 1:2 or 1:3 (RAD51:BCCIPβ) resulted in a 2-fold increase (∼27%, P-value *< 0.05, **< 0.01) in the amount of D-loop formation (Figure 5A). Since a 1:3 molar ratio (RAD51:BCCIPβ) yielded the same amount of D-loop product as a 1:2 molar ratio (RAD51:BCCIPβ), we propose the preferred stoichiometric ratio for BCCIPβ stimulation of RAD51-mediated D-loop formation is 1:2 (RAD51:BCCIPβ).Figure 5.BCCIPβ is a homodimer that stimulates of RAD51-mediated D-loop formation. (A) RAD51 (1.5 μM) was incubated with ssDNA (4.5 μM) in the absence (lanes 2–6) or presence of increasing concentrations of BCCIPβ (1.5 μM, lanes 8–12; 3 μM, lanes 14–18; 4.5 μM, lanes 20–24). Lane 1 was devoid of any protein, and each concentration of BCCIPβ (1.5 μM, 3 μM, 4.5 μM; lanes 7, 13 and 19, respectively) was tested in the absence of RAD51. All reactions took place in the presence of calcium. The reactions were initiated by the addition of supercoiled DNA (35 μM base pairs) and deproteinized with SDS and Proteinase K at the indicated times. Reaction products were separated on a 0.9% agarose gel and visualized using a phosphorimager. (B) Gel filtration standards (bovine thyroglobulin 670 kDa, bovine γ-globulin 158 kDa, chicken ovalbumin 44 kDa and horse myoglobin 17 kDa) were used to calibrate a 35 ml S-400 gel filtration column. The eluate fraction (x-axis) and the log of the molecular weights were plotted (y-axis). The linear fit yielded: y = 6.8799 - 0.071211x and R = 0.99479. The expected values for a BCCIPβ monomer and a BCCIPβ dimer are indicated. (C) BCCIPβ (100 μg) was sized using the 35 ml S-400 gel filtration column. A western blot was performed, and HRP-conjugated anti-poly-histidine was used to develop the membrane. Error bars represent s.e.m. (n = 3); P-value *< 0.05, **< 0.01.

Stimulation of RAD51-mediated D-loop formation by BCCIPβ

During HR, a ssDNA overhang from the processed DSB is used by RAD51 to invade the sister chromatid in search of homology. Several HR accessory factors are known to interact with RAD51 to enhance homologous DNA pairing activity (31–36). To determine if BCCIPβ enhanced the RAD51-mediated homologous DNA pairing activity, we used an in vitro D-loop formation assay (22). In this assay, a single-stranded radiolabeled oligonucleotide is incubated with RAD51 to form a presynaptic filament that invades supercoiled duplex DNA to search for the homologous sequence. Once homology is located, the complementary strand is displaced, forming a D-loop (Figure 4A). Using this assay, our initial attempts showed that RAD51-mediated D-loop formation was not affected by the presence of BCCIPβ in the presence of magnesium (Figure 4B). Previous reports demonstrated that the presence of calcium enhanced RAD51 D-loop formation by inhibiting ATP hydrolysis, which results in a more stable presynaptic filament (37). With this in mind, we repeated the experiment with magnesium and calcium present. Our results show that BCCIPβ stimulated RAD51-mediated D-loop formation nearly 2-fold (P-value *< 0.05, **< 0.01) in a concentration-dependent manner (Figure 4C) in the presence of magnesium and calcium. This D-loop formation was dependent upon ATP and RAD51 (Figure 4C). To test whether BCCIPβ functions with RAD51 in a species-specific manner, we performed the D-loop assay with ScRad51. The D-loop formation activity of ScRad51 is barely detectable without the addition of the DNA translocase, ScRad54 (38). Our results show BCCIPβ was unable to enhance ScRad51 D-loop formation in the presence of magnesium regardless of the absence (Figure 4D) or presence of calcium (data not shown), which was expected since BCCIPβ did not interact with ScRad51.

BCCIPβ stimulates human RAD51-mediated D-loop formation in the presence of calcium. (A) Schematic of the D-loop assay. (B) RAD51 (1.5 μM) was incubated with 32P-OL90 (ssDNA; 4.5 μM) in the absence (lane 2) or presence of increasing concentrations of BCCIPβ (lanes 3–6; 0.75 μM, 1.5 μM, 3 μM, 4.5 μM, respectively). The supercoiled plasmid pBluescript (sc; 35 μM base pairs) was introduced to the reaction. After 6 min, the reactions were deproteinized with SDS and Proteinase K. Lane 1 was devoid of protein, lane 7 contained RAD51 and BCCIPβ (4.5 μM) in the absence of ATP, and lane 8 contained BCCIPβ (4.5 μM) alone. (C) The reactions in B were tested in the presence of (1.8 mM) calcium. (D) The D-loop assay was performed with ScRad51 (1.5 μM) in presence of BCCIPβ (lanes 4–6; 1.5 μM, 3 μM, 4.5 μM, respectively) or ScRad54 (0.2 μM; lane 3). All reaction products were subjected to 0.9% agarose gel electrophoresis and visualized using a phosphorimager. Lane 1 contained no protein, and lane 2 contained only ScRad51. S.e.m. (n = 3) are plotted as error bars; P-value *< 0.05, **< 0.01.

The affinity pull-down assay suggested that BCCIPβ did not completely bind equimolar amounts of RAD51 (Figure 3A, lane 1). In addition, BCCIPβ stimulated RAD51 D-loop formation in a concentration-dependent manner, further suggesting that a specific stoichiometric relationship may be necessary for BCCIPβ to stimulate RAD51 (Figure 4C). To further explore this idea, we examined the ability of BCCIPβ to stimulate RAD51-mediated D-loop formation as a function of time with RAD51 alone and in combination with BCCIPβ in stoichiometric molar ratios of 1:1, 1:2 and 1:3 (Figure 5A). RAD51 alone produced ∼13% D-loop product. When equal molar (1:1) amounts of RAD51 and BCCIPβ were used, there was not a significant increase in D-loop formation. A molar ratio of 1:2 or 1:3 (RAD51:BCCIPβ) resulted in a 2-fold increase (∼27%, P-value *< 0.05, **< 0.01) in the amount of D-loop formation (Figure 5A). Since a 1:3 molar ratio (RAD51:BCCIPβ) yielded the same amount of D-loop product as a 1:2 molar ratio (RAD51:BCCIPβ), we propose the preferred stoichiometric ratio for BCCIPβ stimulation of RAD51-mediated D-loop formation is 1:2 (RAD51:BCCIPβ).

BCCIPβ is a homodimer that stimulates of RAD51-mediated D-loop formation. (A) RAD51 (1.5 μM) was incubated with ssDNA (4.5 μM) in the absence (lanes 2–6) or presence of increasing concentrations of BCCIPβ (1.5 μM, lanes 8–12; 3 μM, lanes 14–18; 4.5 μM, lanes 20–24). Lane 1 was devoid of any protein, and each concentration of BCCIPβ (1.5 μM, 3 μM, 4.5 μM; lanes 7, 13 and 19, respectively) was tested in the absence of RAD51. All reactions took place in the presence of calcium. The reactions were initiated by the addition of supercoiled DNA (35 μM base pairs) and deproteinized with SDS and Proteinase K at the indicated times. Reaction products were separated on a 0.9% agarose gel and visualized using a phosphorimager. (B) Gel filtration standards (bovine thyroglobulin 670 kDa, bovine γ-globulin 158 kDa, chicken ovalbumin 44 kDa and horse myoglobin 17 kDa) were used to calibrate a 35 ml S-400 gel filtration column. The eluate fraction (x-axis) and the log of the molecular weights were plotted (y-axis). The linear fit yielded: y = 6.8799 - 0.071211x and R = 0.99479. The expected values for a BCCIPβ monomer and a BCCIPβ dimer are indicated. (C) BCCIPβ (100 μg) was sized using the 35 ml S-400 gel filtration column. A western blot was performed, and HRP-conjugated anti-poly-histidine was used to develop the membrane. Error bars represent s.e.m. (n = 3); P-value *< 0.05, **< 0.01.

BCCIPβ is a homodimerThe observation that there is a preferred stoichiometric molar ratio of two BCCIPβ molecules to one RAD51 molecule for optimal enhancement of RAD51 homologous DNA pairing suggested the potential for BCCIPβ to form a homodimer. To test this notion, we utilized size exclusion chromatography to monitor the migration of BCCIPβ through a size exclusion column. The column was calibrated with bovine thyroglobulin (670 kDa), bovine γ-globulin (158 kDa), chicken ovalbumin (44 kDa) and horse myoglobin (17 kDa), and their elution positions are as indicated (Figure 5B and C). Based on the elution positions of the molecular weight standards, we plotted the log of the molecular weight (y-axis) of the protein standards and their elution fraction (x-axis) to yield the fitted equation y = 6.8799 – 0.071211x with R = 0.99479 (Figure 5B). Based on this equation, BCCIPβ was expected to elute in fraction 32 (∼36 500 Da). However, the peak fraction of BCCIPβ eluted in fraction 28 (Figure 5C), which correlates with a globular protein having a molecular weight of ∼73 000 Da. These results are consistent with BCCIPβ forming a homodimer in solution.

BCCIPβ is a homodimer

The observation that there is a preferred stoichiometric molar ratio of two BCCIPβ molecules to one RAD51 molecule for optimal enhancement of RAD51 homologous DNA pairing suggested the potential for BCCIPβ to form a homodimer. To test this notion, we utilized size exclusion chromatography to monitor the migration of BCCIPβ through a size exclusion column. The column was calibrated with bovine thyroglobulin (670 kDa), bovine γ-globulin (158 kDa), chicken ovalbumin (44 kDa) and horse myoglobin (17 kDa), and their elution positions are as indicated (Figure 5B and C). Based on the elution positions of the molecular weight standards, we plotted the log of the molecular weight (y-axis) of the protein standards and their elution fraction (x-axis) to yield the fitted equation y = 6.8799 – 0.071211x with R = 0.99479 (Figure 5B). Based on this equation, BCCIPβ was expected to elute in fraction 32 (∼36 500 Da). However, the peak fraction of BCCIPβ eluted in fraction 28 (Figure 5C), which correlates with a globular protein having a molecular weight of ∼73 000 Da. These results are consistent with BCCIPβ forming a homodimer in solution.

BCCIPβ is unable to stabilize RAD51 filamentA number of HR accessory factors enhance the recombination activity of RAD51 through the stabilization of the RAD51 nucleoprotein filament including RAD54, BRCA2, HOP2-MND1, HED1, RAD55/RAD57 and SWI5-SFR1 (39–44). To determine whether BCCIPβ stabilized the RAD51 filament, a nuclease protection assay (22,26,45,46) was employed. In this assay, RAD51 was incubated with a radiolabeled ssDNA oligonucleotide in the presence of ATP to allow for the formation of a presynaptic filament. The resulting nucleoprotein filament was then subjected to DNase I treatment. If BCCIPβ possessed the ability to stabilize the RAD51 filament, then nucleolytic degradation of the oligonucleotide ssDNA would decrease in the presence of BCCIPβ (Figure 6A). Our data show that increasing concentrations of BCCIPβ were unable to stabilize the RAD51 filament regardless of the presence of magnesium or calcium (Figure 6B and data not shown, respectively). These results and the observation that enhancement of RAD51 D-loop formation activity was dependent on a presynaptic filament stabilized by the presence of calcium, indicate BCCIPβ stimulation of RAD51-mediated D-loop formation occurs through a different mechanism.Figure 6.RAD51 nucleoprotein filament is not stabilized by BCCIPβ. (A) Schematic of the nuclease protection assay (adapted from Chi et al. 2007 (41)). (B) RAD51 (0.4 μM) was incubated with radiolabeled ssDNA (3 μM nucleotides) alone (lane 3) or with increasing concentrations of BCCIPβ (lanes 4–6; 0.4 μM, 0.8 μM, 1.2 μM, respectively). The reaction was initiated by the addition of DNase I (2 units). Reaction products were then deproteinized and separated using 10% polyacrylamide gel electrophoresis. Lane 1 contained radiolabeled ssDNA alone, lane 2 contained only radiolabeled ssDNA in the presence of DNase I, and lane 7 contained BCCIPβ (1.6 μM) in the absence of RAD51 but in the presence of DNase I. deg, degraded radiolabeled ssDNA.

BCCIPβ is unable to stabilize RAD51 filament

A number of HR accessory factors enhance the recombination activity of RAD51 through the stabilization of the RAD51 nucleoprotein filament including RAD54, BRCA2, HOP2-MND1, HED1, RAD55/RAD57 and SWI5-SFR1 (39–44). To determine whether BCCIPβ stabilized the RAD51 filament, a nuclease protection assay (22,26,45,46) was employed. In this assay, RAD51 was incubated with a radiolabeled ssDNA oligonucleotide in the presence of ATP to allow for the formation of a presynaptic filament. The resulting nucleoprotein filament was then subjected to DNase I treatment. If BCCIPβ possessed the ability to stabilize the RAD51 filament, then nucleolytic degradation of the oligonucleotide ssDNA would decrease in the presence of BCCIPβ (Figure 6A). Our data show that increasing concentrations of BCCIPβ were unable to stabilize the RAD51 filament regardless of the presence of magnesium or calcium (Figure 6B and data not shown, respectively). These results and the observation that enhancement of RAD51 D-loop formation activity was dependent on a presynaptic filament stabilized by the presence of calcium, indicate BCCIPβ stimulation of RAD51-mediated D-loop formation occurs through a different mechanism.

RAD51 nucleoprotein filament is not stabilized by BCCIPβ. (A) Schematic of the nuclease protection assay (adapted from Chi et al. 2007 (41)). (B) RAD51 (0.4 μM) was incubated with radiolabeled ssDNA (3 μM nucleotides) alone (lane 3) or with increasing concentrations of BCCIPβ (lanes 4–6; 0.4 μM, 0.8 μM, 1.2 μM, respectively). The reaction was initiated by the addition of DNase I (2 units). Reaction products were then deproteinized and separated using 10% polyacrylamide gel electrophoresis. Lane 1 contained radiolabeled ssDNA alone, lane 2 contained only radiolabeled ssDNA in the presence of DNase I, and lane 7 contained BCCIPβ (1.6 μM) in the absence of RAD51 but in the presence of DNase I. deg, degraded radiolabeled ssDNA.

BCCIPβ induces conformational changes in RAD51The observation that BCCIPβ stimulated the D-loop formation activity of RAD51 without stabilizing the presynaptic filament led us to hypothesize that physical interaction with BCCIPβ may induce a change within the RAD51–ssDNA presynaptic filament to an active conformation capable of catalyzing homologous DNA pairing activity. To investigate this possibility, a limited-trypsin proteolytic digestion of RAD51 either with or without BCCIPβ was performed. Limited-proteolysis often provides information about conformational changes that result from the interaction of a protein with effector molecules such as DNA and/or other proteins (27,47–49). If the interaction of an effector molecule causes conformational changes within the protein of interest, then protease-sensitive regions may become resistant to proteolytic cleavage while new cleavage sites are potentially exposed, yielding an altered proteolytic fingerprint of the protein. Here, we incubated RAD51 or RAD51 and BCCIPβ with or without calcium under reaction conditions conducive to nucleoprotein filament formation, followed by the addition of trypsin. Changes in the protein fragmentation were monitored by western blot analysis using antibodies raised against RAD51. Our results, performed in the presence of magnesium (all reactions contain magnesium) and calcium (where indicated), show RAD51 is highly susceptible to trypsin proteolysis with only 10% of RAD51 remaining uncleaved (Figure 7A and B, lane 2) in the absence of effector molecules. The presence of ATP alone resulted in a slight increase in protection with 25% uncleaved RAD51 present (Figure 7A and B, lane 3). Approximately 45% of the RAD51 was protected from cleavage by trypsin in the presence of ATP and ssDNA under conditions that promote presynaptic filament formation (Figure 7A and B, lane 4). When BCCIPβ was present with a RAD51 presynaptic filament, the amount of uncleaved RAD51 increased to 70% (Figure 7A and B, compare lanes 4 and 6). In addition, there was a 5-fold decrease in the amount of Fragment B and a 16-fold decrease in Fragment D (Figure 7A and B, compare lanes 4 to 6). When calcium was included with a RAD51 presynaptic filament, 79% of RAD51 remained uncleaved (Figure 7A and B, lane 5). The addition of both calcium and BCCIPβ under the same conditions led to approximately the same level of uncleaved RAD51 (80%; Figure 7A and B, lane 7). However, there was an 8-fold decrease in Fragment B and a 10-fold decrease in the amount of Fragment D (Figure 7A and B, compare lanes 5 and 7). Interestingly, Fragments E and F were only observed when the RAD51 presynaptic filament was proteolytically cleaved in the presence of calcium (Figure 7A and B, lane 5). The presence of BCCIPβ altered this cleavage pattern since Fragment E was no longer observed (Figure 7A and B, compare lanes 5 and 7). Notably, Fragment G was only present when RAD51 was in the presence of ATP or in the presence of BCCIPβ alone (Figure 7A and B, lanes 3 and 9). We also observed a 9-fold increase in the amount of Fragment C and 45-fold increase in Fragment D when compared to trypsin digestion of RAD51 alone (Figure 7A and B, compare lanes 2 and 9). As expected, when RAD51 was not included, there was no signal, demonstrating the specificity of the antibodies used for RAD51 (Figure 7A and B, lane 8). Taken together, BCCIPβ has a significant effect on the proteolytic digestion fingerprint of RAD51 and the RAD51 presynaptic filament, suggesting that interaction with BCCIPβ results in conformational changes within RAD51.Figure 7.Interaction with BCCIPβ induces conformational changes in RAD51. (A) RAD51 (5 μM) was incubated with trypsin (20 μg/ml) in the presence and absence of ATP (2 mM), ϕX174 ssDNA (30 μM nucleotides), calcium (1.8 mM) and BCCIPβ (10 μM), as indicated. The reactions were stopped with SDS and heat. The reaction products were resolved using SDS-PAGE followed by western blot analysis. Antibodies against RAD51 were used to develop the membrane. (B) The amounts of each band from undigested RAD51 and Fragments A, B, C and D were graphed based on the relative intensity of each band. Quantitation of the proteolytic fragmentation of RAD51 was determined from three independent experiments.

BCCIPβ induces conformational changes in RAD51

The observation that BCCIPβ stimulated the D-loop formation activity of RAD51 without stabilizing the presynaptic filament led us to hypothesize that physical interaction with BCCIPβ may induce a change within the RAD51–ssDNA presynaptic filament to an active conformation capable of catalyzing homologous DNA pairing activity. To investigate this possibility, a limited-trypsin proteolytic digestion of RAD51 either with or without BCCIPβ was performed. Limited-proteolysis often provides information about conformational changes that result from the interaction of a protein with effector molecules such as DNA and/or other proteins (27,47–49). If the interaction of an effector molecule causes conformational changes within the protein of interest, then protease-sensitive regions may become resistant to proteolytic cleavage while new cleavage sites are potentially exposed, yielding an altered proteolytic fingerprint of the protein. Here, we incubated RAD51 or RAD51 and BCCIPβ with or without calcium under reaction conditions conducive to nucleoprotein filament formation, followed by the addition of trypsin. Changes in the protein fragmentation were monitored by western blot analysis using antibodies raised against RAD51. Our results, performed in the presence of magnesium (all reactions contain magnesium) and calcium (where indicated), show RAD51 is highly susceptible to trypsin proteolysis with only 10% of RAD51 remaining uncleaved (Figure 7A and B, lane 2) in the absence of effector molecules. The presence of ATP alone resulted in a slight increase in protection with 25% uncleaved RAD51 present (Figure 7A and B, lane 3). Approximately 45% of the RAD51 was protected from cleavage by trypsin in the presence of ATP and ssDNA under conditions that promote presynaptic filament formation (Figure 7A and B, lane 4). When BCCIPβ was present with a RAD51 presynaptic filament, the amount of uncleaved RAD51 increased to 70% (Figure 7A and B, compare lanes 4 and 6). In addition, there was a 5-fold decrease in the amount of Fragment B and a 16-fold decrease in Fragment D (Figure 7A and B, compare lanes 4 to 6). When calcium was included with a RAD51 presynaptic filament, 79% of RAD51 remained uncleaved (Figure 7A and B, lane 5). The addition of both calcium and BCCIPβ under the same conditions led to approximately the same level of uncleaved RAD51 (80%; Figure 7A and B, lane 7). However, there was an 8-fold decrease in Fragment B and a 10-fold decrease in the amount of Fragment D (Figure 7A and B, compare lanes 5 and 7). Interestingly, Fragments E and F were only observed when the RAD51 presynaptic filament was proteolytically cleaved in the presence of calcium (Figure 7A and B, lane 5). The presence of BCCIPβ altered this cleavage pattern since Fragment E was no longer observed (Figure 7A and B, compare lanes 5 and 7). Notably, Fragment G was only present when RAD51 was in the presence of ATP or in the presence of BCCIPβ alone (Figure 7A and B, lanes 3 and 9). We also observed a 9-fold increase in the amount of Fragment C and 45-fold increase in Fragment D when compared to trypsin digestion of RAD51 alone (Figure 7A and B, compare lanes 2 and 9). As expected, when RAD51 was not included, there was no signal, demonstrating the specificity of the antibodies used for RAD51 (Figure 7A and B, lane 8). Taken together, BCCIPβ has a significant effect on the proteolytic digestion fingerprint of RAD51 and the RAD51 presynaptic filament, suggesting that interaction with BCCIPβ results in conformational changes within RAD51.

Interaction with BCCIPβ induces conformational changes in RAD51. (A) RAD51 (5 μM) was incubated with trypsin (20 μg/ml) in the presence and absence of ATP (2 mM), ϕX174 ssDNA (30 μM nucleotides), calcium (1.8 mM) and BCCIPβ (10 μM), as indicated. The reactions were stopped with SDS and heat. The reaction products were resolved using SDS-PAGE followed by western blot analysis. Antibodies against RAD51 were used to develop the membrane. (B) The amounts of each band from undigested RAD51 and Fragments A, B, C and D were graphed based on the relative intensity of each band. Quantitation of the proteolytic fragmentation of RAD51 was determined from three independent experiments.

ATP hydrolysis by RAD51 is stimulated by BCCIPβThe enhanced D-loop formation attributed to RAD51 by the presence of calcium is through the inhibition of ATP hydrolysis (37). Given that BCCIPβ failed to stabilize the RAD51 presynaptic filament, yet induced a conformational change in the RAD51 filament, we asked whether BCCIPβ had an effect on the ATP hydrolysis activity of RAD51. To monitor RAD51 ATP hydrolysis, RAD51 was incubated with [32P-γ]-ATP in the presence of magnesium and in the absence and presence of ssDNA under conditions that favor presynaptic filament formation with or without BCCIPβ. At the given times, each reaction was stopped, and the amount of hydrolyzed [32P-γ]-ATP was analyzed. Our results indicate that BCCIPβ enhances the ATP hydrolysis activity of RAD51 in the presence of ssDNA ∼1.7-fold (P-value *< 0.05, Figure 8A and B). Upon further investigation, we found that there was no enhancement of ATP hydrolysis by RAD51 in the absence of ssDNA, (Figure 8A, compare lanes 2 and 4). BCCIPβ stimulated the ATP hydrolysis activity of RAD51 in the presence of ssDNA. These results are in agreement with our finding that BCCIPβ was only able to stimulate RAD51 D-loop formation when a stable RAD51 filament was present. As expected, no ATP hydrolysis activity was observed with BCCIPβ alone or in combination with ssDNA (Figure 8A, lane 3 and 6, respectively). These results demonstrate the ability of BCCIPβ to enhance ATP hydrolysis of RAD51 when bound to ssDNA, suggesting that enhanced D-loop activity was not due to reduced ATP hydrolysis activity by RAD51.Figure 8.BCCIPβ stimulates RAD51 ATP hydrolysis and promotes ADP release. (A) RAD51 (0.5 μM) ATP hydrolysis assay in the presence or absence of ϕX174 ssDNA (60 μM nucleotides) and BCCIPβ (1 μM). (B) Time course analysis of RAD51 (0.5 μM) ATP hydrolysis in the presence of ϕX174 ssDNA (60 μM nucleotides), with or without BCCIPβ (1 μM). Error bars represent s.e.m. (n = 3); P-value *< 0.05. (C) Fluorescence spectra (400–500 nm) of MANT-ADP (0.5 μM) and ϕX174 ssDNA (2.0 μM nucleotides) in the absence or presence of RAD51 or RAD51K133A (0.5 μM). (D) Time course analysis of MANT-ADP dissociation from a RAD51-MANT-ADP-ssDNA complex in the presence of magnesium with or without BCCIPβ (1 μM). (E) Time course analysis of MANT-ADP release from the RAD51-MANT-ADP-ssDNA filament in the presence of calcium with or without BCCIPβ (1 μM) and/or ATP (5 μM), as indicated. P-value *< 0.05 and **< 0.01 is indicative of the 600 s time point.

ATP hydrolysis by RAD51 is stimulated by BCCIPβ

The enhanced D-loop formation attributed to RAD51 by the presence of calcium is through the inhibition of ATP hydrolysis (37). Given that BCCIPβ failed to stabilize the RAD51 presynaptic filament, yet induced a conformational change in the RAD51 filament, we asked whether BCCIPβ had an effect on the ATP hydrolysis activity of RAD51. To monitor RAD51 ATP hydrolysis, RAD51 was incubated with [32P-γ]-ATP in the presence of magnesium and in the absence and presence of ssDNA under conditions that favor presynaptic filament formation with or without BCCIPβ. At the given times, each reaction was stopped, and the amount of hydrolyzed [32P-γ]-ATP was analyzed. Our results indicate that BCCIPβ enhances the ATP hydrolysis activity of RAD51 in the presence of ssDNA ∼1.7-fold (P-value *< 0.05, Figure 8A and B). Upon further investigation, we found that there was no enhancement of ATP hydrolysis by RAD51 in the absence of ssDNA, (Figure 8A, compare lanes 2 and 4). BCCIPβ stimulated the ATP hydrolysis activity of RAD51 in the presence of ssDNA. These results are in agreement with our finding that BCCIPβ was only able to stimulate RAD51 D-loop formation when a stable RAD51 filament was present. As expected, no ATP hydrolysis activity was observed with BCCIPβ alone or in combination with ssDNA (Figure 8A, lane 3 and 6, respectively). These results demonstrate the ability of BCCIPβ to enhance ATP hydrolysis of RAD51 when bound to ssDNA, suggesting that enhanced D-loop activity was not due to reduced ATP hydrolysis activity by RAD51.

BCCIPβ stimulates RAD51 ATP hydrolysis and promotes ADP release. (A) RAD51 (0.5 μM) ATP hydrolysis assay in the presence or absence of ϕX174 ssDNA (60 μM nucleotides) and BCCIPβ (1 μM). (B) Time course analysis of RAD51 (0.5 μM) ATP hydrolysis in the presence of ϕX174 ssDNA (60 μM nucleotides), with or without BCCIPβ (1 μM). Error bars represent s.e.m. (n = 3); P-value *< 0.05. (C) Fluorescence spectra (400–500 nm) of MANT-ADP (0.5 μM) and ϕX174 ssDNA (2.0 μM nucleotides) in the absence or presence of RAD51 or RAD51K133A (0.5 μM). (D) Time course analysis of MANT-ADP dissociation from a RAD51-MANT-ADP-ssDNA complex in the presence of magnesium with or without BCCIPβ (1 μM). (E) Time course analysis of MANT-ADP release from the RAD51-MANT-ADP-ssDNA filament in the presence of calcium with or without BCCIPβ (1 μM) and/or ATP (5 μM), as indicated. P-value *< 0.05 and **< 0.01 is indicative of the 600 s time point.

BCCIPβ mediates ADP releaseA potential mechanism for the stimulation of RAD51 homologous DNA pairing activity by BCCIPβ is through the enhanced release of ADP from the RAD51 filament. In support of this idea, Shim et al. and Su et al. previously reported that XRCC2 and SWI5-SFR1, respectively, promote the release of ADP from a RAD51 filament (28,50). To monitor the effect of BCCIPβ on the RAD51-ADP-ssDNA filament, we needed to monitor the ability of RAD51 to form a filament in the presence of ADP. To do this, we incubated fluorescently labeled ADP (MANT-ADP) and ssDNA in the presence or absence of RAD51. After allowing ample time for filament formation, a fluorescence scan from 400 to 500 nm was performed. We found a ∼33% increase in the relative fluorescence and a 4 nm red shift in peak fluorescence (445–441 nm) when RAD51 was in the reaction, indicating the binding of MANT-ADP by RAD51–ssDNA (Figure 8C). To confirm that the increase in fluorescence and red-shift in peak fluorescence were due to RAD51 binding MANT-ADP, we substituted RAD51 for the RAD51K133A variant, known to have low affinity for nucleotides (26,27,51). As expected, there was no difference in the fluorescence spectra of MANT-ADP in buffer regardless of the presence of RAD51K133A indicating the increase in fluorescence and red-shift in peak fluorescence was due to RAD51 binding MANT-ADP (Figure 8C). Since there was an increase in fluorescence and a shift in the peak of the fluorescent signal upon RAD51 binding MANT-ADP, we performed a time course experiment that monitored the dissociation of MANT-ADP as a decrease in fluorescent signal. Here, we incubated RAD51 with MANT-ADP and ssDNA in the presence of magnesium to allow for the formation of the RAD51-ADP-ssDNA filament. Upon the addition of BCCIPβ or reaction buffer containing magnesium, the decrease in fluorescence was monitored at 445 nm. As shown in Figure 8D, the addition of BCCIPβ in reaction buffer containing magnesium resulted in a dramatic decrease (t1/2 = 32.0 ± 2.8 s) in the relative fluorescence compared to the addition of buffer alone (t1/2 = 52.6 ± 4.3 s; P-value **< 0.01). Next, we incubated RAD51 with MANT-ADP and ssDNA in the presence of calcium during the formation of the RAD51-ADP-ssDNA filament. The addition of reaction buffer containing calcium yielded a slow decrease in fluorescence (Figure 8E). Surprisingly, the addition of BCCIPβ in reaction buffer containing calcium resulted in no significant change in the level of fluorescence (Figure 8E). We found that the addition of BCCIPβ in combination with 10-fold excess ATP resulted in a significant decrease in fluorescence (t1/2 = 30.0 ± 3.1 s) compared to the addition of reaction buffer containing 10-fold excess ATP alone (t1/2 = 82.4 ± 4.9 s P-value *< 0.05) (Figure 8E). Our results suggest that BCCIPβ promotes the release of ADP from the RAD51-ADP-ssDNA filament in the presence of magnesium. In the presence of calcium, the release of ADP from RAD51–ssDNA is stimulated by BCCIPβ when ATP is present.

BCCIPβ mediates ADP release

A potential mechanism for the stimulation of RAD51 homologous DNA pairing activity by BCCIPβ is through the enhanced release of ADP from the RAD51 filament. In support of this idea, Shim et al. and Su et al. previously reported that XRCC2 and SWI5-SFR1, respectively, promote the release of ADP from a RAD51 filament (28,50). To monitor the effect of BCCIPβ on the RAD51-ADP-ssDNA filament, we needed to monitor the ability of RAD51 to form a filament in the presence of ADP. To do this, we incubated fluorescently labeled ADP (MANT-ADP) and ssDNA in the presence or absence of RAD51. After allowing ample time for filament formation, a fluorescence scan from 400 to 500 nm was performed. We found a ∼33% increase in the relative fluorescence and a 4 nm red shift in peak fluorescence (445–441 nm) when RAD51 was in the reaction, indicating the binding of MANT-ADP by RAD51–ssDNA (Figure 8C). To confirm that the increase in fluorescence and red-shift in peak fluorescence were due to RAD51 binding MANT-ADP, we substituted RAD51 for the RAD51K133A variant, known to have low affinity for nucleotides (26,27,51). As expected, there was no difference in the fluorescence spectra of MANT-ADP in buffer regardless of the presence of RAD51K133A indicating the increase in fluorescence and red-shift in peak fluorescence was due to RAD51 binding MANT-ADP (Figure 8C). Since there was an increase in fluorescence and a shift in the peak of the fluorescent signal upon RAD51 binding MANT-ADP, we performed a time course experiment that monitored the dissociation of MANT-ADP as a decrease in fluorescent signal. Here, we incubated RAD51 with MANT-ADP and ssDNA in the presence of magnesium to allow for the formation of the RAD51-ADP-ssDNA filament. Upon the addition of BCCIPβ or reaction buffer containing magnesium, the decrease in fluorescence was monitored at 445 nm. As shown in Figure 8D, the addition of BCCIPβ in reaction buffer containing magnesium resulted in a dramatic decrease (t1/2 = 32.0 ± 2.8 s) in the relative fluorescence compared to the addition of buffer alone (t1/2 = 52.6 ± 4.3 s; P-value **< 0.01). Next, we incubated RAD51 with MANT-ADP and ssDNA in the presence of calcium during the formation of the RAD51-ADP-ssDNA filament. The addition of reaction buffer containing calcium yielded a slow decrease in fluorescence (Figure 8E). Surprisingly, the addition of BCCIPβ in reaction buffer containing calcium resulted in no significant change in the level of fluorescence (Figure 8E). We found that the addition of BCCIPβ in combination with 10-fold excess ATP resulted in a significant decrease in fluorescence (t1/2 = 30.0 ± 3.1 s) compared to the addition of reaction buffer containing 10-fold excess ATP alone (t1/2 = 82.4 ± 4.9 s P-value *< 0.05) (Figure 8E). Our results suggest that BCCIPβ promotes the release of ADP from the RAD51-ADP-ssDNA filament in the presence of magnesium. In the presence of calcium, the release of ADP from RAD51–ssDNA is stimulated by BCCIPβ when ATP is present.

DISCUSSIONIn this study, we sought to determine if BCCIPβ had a direct effect on the activity of RAD51. We show that BCCIPβ interacts with RAD51 directly, and the interaction between BCCIPβ and RAD51 is species-specific as human BCCIPβ failed to interact with S. cerevisiae Rad51, despite ScRad51 having 67% identity and 81% similarity to human RAD51. We conclude that the architecture of the interaction interface between RAD51 and BCCIPβ is likely to be sufficiently different or not present between ScRad51 and human BCCIPβ, preventing interaction.The ability of BCCIPβ to interact with RAD51 suggested that BCCIPβ might have an effect on RAD51 recombination activities. Our initial attempts to detect an effect of BCCIPβ on RAD51-mediated D-loop formation were unsuccessful until calcium was included in the reaction. To understand the requirement for calcium, we first ruled out the possibility that calcium was required for interaction between BCCIPβ and RAD51 because of the absence of calcium in the affinity pull-down experiments. The results from our nuclease protection assay in the absence of calcium suggested BCCIPβ does not function to stimulate RAD51-mediated D-loop formation by stabilizing the RAD51 presynaptic filament (22,26,45,46) as reported for other HR factors (39–44). Since calcium is known to prevent ATP hydrolysis by RAD51, thereby stabilizing the RAD51 nucleoprotein filament in an active conformation and leading to a large increase in the amount of D-loop formed by RAD51 (37), we reasoned that BCCIPβ might only function with the calcium-stabilized, active form of the RAD51 presynaptic filament.Based on the results of our ADP release and proteolysis experiments, we propose that the presence of different cofactors induce unique conformational changes in RAD51 that modulate the release of ADP. BCCIPβ induced a conformational change in RAD51 that favors the release of ADP in the absence of calcium, while the calcium-induced conformation of RAD51 prevented the BCCIPβ-induced conformation from releasing ADP. However, the addition of ATP in the presence of both calcium and BCCIPβ induced a third change in the conformation of RAD51 that led to the release of ADP. Support for the existence of these multiple conformations in RAD51 lies in our proteolytic analysis. Our data show that interaction of RAD51 with BCCIPβ, ATP or both BCCIPβ and ATP results in proteolytic fingerprints of RAD51 that are distinct from those in the absence of BCCIPβ or ATP. Furthermore, the proteolytic fingerprint of a RAD51 presynaptic filament is different from one in which BCCIPβ is added in either the presence or absence of calcium. As a result, the RAD51 presynaptic filament appears to be dynamic allowing for subtle changes in conformation that modulate its activity. The ability to induce a conformational change in RAD51 is not unique to BCCIPβ. The HOP2-MND1 complex, an accessory factor that stabilizes the RAD51 filament and strongly enhances homologous DNA pairing, induces conformational changes in RAD51 that modulates the binding of nucleotides (27). Alternatively, an interaction between calcium and BCCIPβ could cause the observed activities. A region within BCCIPβ is reported to share homology with the calcium binding sites of M-calpain (29% identity and 58% similarity) and calmodulin (26% identity and 43% similarity), both of which are calcium-binding proteins (16). It is possible that calcium not only stabilizes and promotes conformational changes in the RAD51 presynaptic filament but also promotes a conformational change in BCCIPβ that allows BCCIPβ to exert its effect on RAD51. Moreover, we cannot rule out the possibility that calcium is acting as a substitute for BRCA2, which interacts with BCCIPβ and stabilizes the RAD51 presynaptic filament in vitro (40).A previous report showed that a 50% reduction in the BCCIP protein levels resulted in a striking ∼100-fold reduction in HR repair (20). This result led Lu et al. (2005) to propose that BCCIPβ is required at a specific stoichiometry for proper BRCA2-RAD51 function during HR. In support of this idea, we found that the stimulation of RAD51-mediated D-loop formation was maximal at a ratio of two BCCIPβ proteins to one RAD51. Furthermore, we provide evidence that BCCIPβ forms a homodimer in the absence of RAD51. These findings suggest that a dimer of BCCIPβ interacts with RAD51 to promote D-loop formation. Expression analysis revealed the presence of a second isoform of BCCIP, BCCIPα, with primary sequence that is 80% identical to BCCIPβ (17). While the expression of BCCIPβ was relatively constant regardless of the tumor cell analyzed, the expression of BCCIPα varied from being absent to overexpressed depending on the tumor cell line (16). The demonstration that both isoforms of BCCIP are co-expressed leaves open the possibility that BCCIPβ can form a heterodimer with BCCIPα.The active form of the RAD51 presynaptic filament is bound to ATP. Once the bound ATP is hydrolyzed, the presynaptic filament converts into an inactive form with ADP bound. The dissociation of ADP from the inactive presynaptic filament is slow (37). We show that BCCIPβ increases the rate of ADP release from the inactive RAD51 presynaptic filament. This may allow RAD51 to bind ATP and return to the active state. In support of this notion, our results show the rate of ATP hydrolysis by the RAD51 presynaptic filament is elevated in the presence of BCCIPβ. The combination of calcium, to prevent ATP hydrolysis, and BCCIP, to promote the release of ADP, likely results in the RAD51 presynaptic filament remaining in the active state.BCCIP is not the first HR factor shown to stimulate RAD51 activities and promote the release of ADP. XRCC2 enhances RAD51-mediated DNA strand exchange. XRCC2 reduces the affinity of RAD51 for ADP, which leads to an increase in the rate of ADP release and thus increases the ATP hydrolysis activity of RAD51 (50), much like we report for BCCIPβ. More recently, the murine SWI5-SFR1 complex was shown to be involved in HR by enhancing RAD51-mediated homologous DNA pairing (28,44). In addition, the murine SWI5-SFR1 complex stimulates the ATP hydrolysis activity of the RAD51–ssDNA presynaptic filament and increases the rate of ADP release (28). While these observations are similar to our results with BCCIPβ, the ability of SWI5-SFR1 to stabilize the RAD51 presynaptic filament (36,44) is an activity not shared with BCCIPβ. Furthermore, the stimulatory effect of SWI5-SFR1 on RAD51 homologous DNA pairing was not detected in the presence of a stabilized presynaptic filament (44) suggesting that SWI5-SFR1 likely enhanced RAD51 recombinase activity via stabilization of the RAD51 presynaptic filament.BCCIP is a BRCA2-interacting protein (16) that co-localizes with RAD51 foci and BRCA2 foci in the nucleus. Based on the work of others and our results, we suggest that BRCA2-DSS1-BCCIP interact with RAD51 after the generation of DSBs. DSS1 targets BRCA2-BCCIP-RAD51 to RPA-coated ssDNA at the DSB site and aids in the removal of RPA. BRCA2 loads RAD51 onto the ssDNA and stabilizes the newly formed RAD51 presynaptic filament. As RAD51 binds ssDNA, its ATP hydrolysis activity is stimulated, leading to inactive RAD51-ADP filaments. BCCIPβ stimulates the release of ADP from RAD51, which helps maintain RAD51 in an active conformation capable of D-loop formation (Figure 9). Given the activities we report for BCCIPβ, it is understandable that the loss of BCCIP expression is frequently observed in brain, ovarian, kidney and colorectal cancers (18,19) as it is important for the repair of DSBs in vivo (20). Our study provides insight into the molecular functions of BCCIPβ enhancement of RAD51 in the HR pathway. Based on results of our study, the association of BCCIPβ with BRCA2 in vivo (16), and the role BRCA2 has as an accessory factor in HR (7–10), it will be important to investigate how BCCIP functions with BRCA2 in the HR pathway.Figure 9.Model for the role of BCCIP in BRCA2-DSS1-RAD51-mediated HR. DSS1 targets the BRCA2-DSS1-RAD51-BCCIPβ complex to RPA-coated ssDNA. DSS1 helps BRCA2 displace RPA from the ssDNA to facilitate nucleation of RAD51 onto the ssDNA forming the presynaptic filament. The active RAD51 filament hydrolyzes bound ATP (T) leading to a condensed and inactive ADP-bound RAD51 filament (D). BCCIPβ promotes the release of ADP from the inactive RAD51 filament, allowing ATP to bind reactivating the RAD51 presynaptic filament. Calcium inhibits ATP hydrolysis by RAD51 to help maintain the RAD51 filament in an active form.

In this study, we sought to determine if BCCIPβ had a direct effect on the activity of RAD51. We show that BCCIPβ interacts with RAD51 directly, and the interaction between BCCIPβ and RAD51 is species-specific as human BCCIPβ failed to interact with S. cerevisiae Rad51, despite ScRad51 having 67% identity and 81% similarity to human RAD51. We conclude that the architecture of the interaction interface between RAD51 and BCCIPβ is likely to be sufficiently different or not present between ScRad51 and human BCCIPβ, preventing interaction.

The ability of BCCIPβ to interact with RAD51 suggested that BCCIPβ might have an effect on RAD51 recombination activities. Our initial attempts to detect an effect of BCCIPβ on RAD51-mediated D-loop formation were unsuccessful until calcium was included in the reaction. To understand the requirement for calcium, we first ruled out the possibility that calcium was required for interaction between BCCIPβ and RAD51 because of the absence of calcium in the affinity pull-down experiments. The results from our nuclease protection assay in the absence of calcium suggested BCCIPβ does not function to stimulate RAD51-mediated D-loop formation by stabilizing the RAD51 presynaptic filament (22,26,45,46) as reported for other HR factors (39–44). Since calcium is known to prevent ATP hydrolysis by RAD51, thereby stabilizing the RAD51 nucleoprotein filament in an active conformation and leading to a large increase in the amount of D-loop formed by RAD51 (37), we reasoned that BCCIPβ might only function with the calcium-stabilized, active form of the RAD51 presynaptic filament.

Based on the results of our ADP release and proteolysis experiments, we propose that the presence of different cofactors induce unique conformational changes in RAD51 that modulate the release of ADP. BCCIPβ induced a conformational change in RAD51 that favors the release of ADP in the absence of calcium, while the calcium-induced conformation of RAD51 prevented the BCCIPβ-induced conformation from releasing ADP. However, the addition of ATP in the presence of both calcium and BCCIPβ induced a third change in the conformation of RAD51 that led to the release of ADP. Support for the existence of these multiple conformations in RAD51 lies in our proteolytic analysis. Our data show that interaction of RAD51 with BCCIPβ, ATP or both BCCIPβ and ATP results in proteolytic fingerprints of RAD51 that are distinct from those in the absence of BCCIPβ or ATP. Furthermore, the proteolytic fingerprint of a RAD51 presynaptic filament is different from one in which BCCIPβ is added in either the presence or absence of calcium. As a result, the RAD51 presynaptic filament appears to be dynamic allowing for subtle changes in conformation that modulate its activity. The ability to induce a conformational change in RAD51 is not unique to BCCIPβ. The HOP2-MND1 complex, an accessory factor that stabilizes the RAD51 filament and strongly enhances homologous DNA pairing, induces conformational changes in RAD51 that modulates the binding of nucleotides (27). Alternatively, an interaction between calcium and BCCIPβ could cause the observed activities. A region within BCCIPβ is reported to share homology with the calcium binding sites of M-calpain (29% identity and 58% similarity) and calmodulin (26% identity and 43% similarity), both of which are calcium-binding proteins (16). It is possible that calcium not only stabilizes and promotes conformational changes in the RAD51 presynaptic filament but also promotes a conformational change in BCCIPβ that allows BCCIPβ to exert its effect on RAD51. Moreover, we cannot rule out the possibility that calcium is acting as a substitute for BRCA2, which interacts with BCCIPβ and stabilizes the RAD51 presynaptic filament in vitro (40).

A previous report showed that a 50% reduction in the BCCIP protein levels resulted in a striking ∼100-fold reduction in HR repair (20). This result led Lu et al. (2005) to propose that BCCIPβ is required at a specific stoichiometry for proper BRCA2-RAD51 function during HR. In support of this idea, we found that the stimulation of RAD51-mediated D-loop formation was maximal at a ratio of two BCCIPβ proteins to one RAD51. Furthermore, we provide evidence that BCCIPβ forms a homodimer in the absence of RAD51. These findings suggest that a dimer of BCCIPβ interacts with RAD51 to promote D-loop formation. Expression analysis revealed the presence of a second isoform of BCCIP, BCCIPα, with primary sequence that is 80% identical to BCCIPβ (17). While the expression of BCCIPβ was relatively constant regardless of the tumor cell analyzed, the expression of BCCIPα varied from being absent to overexpressed depending on the tumor cell line (16). The demonstration that both isoforms of BCCIP are co-expressed leaves open the possibility that BCCIPβ can form a heterodimer with BCCIPα.

The active form of the RAD51 presynaptic filament is bound to ATP. Once the bound ATP is hydrolyzed, the presynaptic filament converts into an inactive form with ADP bound. The dissociation of ADP from the inactive presynaptic filament is slow (37). We show that BCCIPβ increases the rate of ADP release from the inactive RAD51 presynaptic filament. This may allow RAD51 to bind ATP and return to the active state. In support of this notion, our results show the rate of ATP hydrolysis by the RAD51 presynaptic filament is elevated in the presence of BCCIPβ. The combination of calcium, to prevent ATP hydrolysis, and BCCIP, to promote the release of ADP, likely results in the RAD51 presynaptic filament remaining in the active state.

BCCIP is not the first HR factor shown to stimulate RAD51 activities and promote the release of ADP. XRCC2 enhances RAD51-mediated DNA strand exchange. XRCC2 reduces the affinity of RAD51 for ADP, which leads to an increase in the rate of ADP release and thus increases the ATP hydrolysis activity of RAD51 (50), much like we report for BCCIPβ. More recently, the murine SWI5-SFR1 complex was shown to be involved in HR by enhancing RAD51-mediated homologous DNA pairing (28,44). In addition, the murine SWI5-SFR1 complex stimulates the ATP hydrolysis activity of the RAD51–ssDNA presynaptic filament and increases the rate of ADP release (28). While these observations are similar to our results with BCCIPβ, the ability of SWI5-SFR1 to stabilize the RAD51 presynaptic filament (36,44) is an activity not shared with BCCIPβ. Furthermore, the stimulatory effect of SWI5-SFR1 on RAD51 homologous DNA pairing was not detected in the presence of a stabilized presynaptic filament (44) suggesting that SWI5-SFR1 likely enhanced RAD51 recombinase activity via stabilization of the RAD51 presynaptic filament.

BCCIP is a BRCA2-interacting protein (16) that co-localizes with RAD51 foci and BRCA2 foci in the nucleus. Based on the work of others and our results, we suggest that BRCA2-DSS1-BCCIP interact with RAD51 after the generation of DSBs. DSS1 targets BRCA2-BCCIP-RAD51 to RPA-coated ssDNA at the DSB site and aids in the removal of RPA. BRCA2 loads RAD51 onto the ssDNA and stabilizes the newly formed RAD51 presynaptic filament. As RAD51 binds ssDNA, its ATP hydrolysis activity is stimulated, leading to inactive RAD51-ADP filaments. BCCIPβ stimulates the release of ADP from RAD51, which helps maintain RAD51 in an active conformation capable of D-loop formation (Figure 9). Given the activities we report for BCCIPβ, it is understandable that the loss of BCCIP expression is frequently observed in brain, ovarian, kidney and colorectal cancers (18,19) as it is important for the repair of DSBs in vivo (20). Our study provides insight into the molecular functions of BCCIPβ enhancement of RAD51 in the HR pathway. Based on results of our study, the association of BCCIPβ with BRCA2 in vivo (16), and the role BRCA2 has as an accessory factor in HR (7–10), it will be important to investigate how BCCIP functions with BRCA2 in the HR pathway.

Model for the role of BCCIP in BRCA2-DSS1-RAD51-mediated HR. DSS1 targets the BRCA2-DSS1-RAD51-BCCIPβ complex to RPA-coated ssDNA. DSS1 helps BRCA2 displace RPA from the ssDNA to facilitate nucleation of RAD51 onto the ssDNA forming the presynaptic filament. The active RAD51 filament hydrolyzes bound ATP (T) leading to a condensed and inactive ADP-bound RAD51 filament (D). BCCIPβ promotes the release of ADP from the inactive RAD51 filament, allowing ATP to bind reactivating the RAD51 presynaptic filament. Calcium inhibits ATP hydrolysis by RAD51 to help maintain the RAD51 filament in an active form.
