---
pmid: '22080955'
title: Structure of full-length Drosophila cryptochrome.
authors:
- Zoltowski BD
- Vaidya AT
- Top D
- Widom J
- Young MW
- Crane BR
journal: Nature
year: '2011'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC3240699
doi: 10.1038/nature10618
---

# Structure of full-length Drosophila cryptochrome.
**Authors:** Zoltowski BD, Vaidya AT, Top D, Widom J, Young MW, Crane BR
**Journal:** Nature (2011)
**DOI:** [10.1038/nature10618](https://doi.org/10.1038/nature10618)
**PMC:** [PMC3240699](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3240699/)

## Abstract

1. Nature. 2011 Nov 13;480(7377):396-9. doi: 10.1038/nature10618.

Structure of full-length Drosophila cryptochrome.

Zoltowski BD(1), Vaidya AT, Top D, Widom J, Young MW, Crane BR.

Author information:
(1)Department of Chemistry and Chemical Biology, Cornell University, Ithaca, New 
York 14853, USA.

Erratum in
    Nature. 2013 Apr 11;496(7444):252.

The cryptochrome/photolyase (CRY/PL) family of photoreceptors mediates adaptive 
responses to ultraviolet and blue light exposure in all kingdoms of life. 
Whereas PLs function predominantly in DNA repair of cyclobutane pyrimidine 
dimers (CPDs) and 6-4 photolesions caused by ultraviolet radiation, CRYs 
transduce signals important for growth, development, magnetosensitivity and 
circadian clocks. Despite these diverse functions, PLs/CRYs preserve a common 
structural fold, a dependence on flavin adenine dinucleotide (FAD) and an 
internal photoactivation mechanism. However, members of the CRY/PL family differ 
in the substrates recognized (protein or DNA), photochemical reactions catalysed 
and involvement of an antenna cofactor. It is largely unknown how the animal 
CRYs that regulate circadian rhythms act on their substrates. CRYs contain a 
variable carboxy-terminal tail that appends the conserved PL homology domain 
(PHD) and is important for function. Here, we report a 2.3-Å resolution crystal 
structure of Drosophila CRY with an intact C terminus. The C-terminal helix 
docks in the analogous groove that binds DNA substrates in PLs. Conserved 
Trp 536 juts into the CRY catalytic centre to mimic PL recognition of DNA 
photolesions. The FAD anionic semiquinone found in the crystals assumes a 
conformation to facilitate restructuring of the tail helix. These results help 
reconcile the diverse functions of the CRY/PL family by demonstrating how 
conserved protein architecture and photochemistry can be elaborated into a range 
of light-driven functions.

DOI: 10.1038/nature10618
PMCID: PMC3240699
PMID: 22080955 [Indexed for MEDLINE]

Conflict of interest statement: The authors declare no competing financial 
interests. Readers are welcome to comment on the online version of this article 
at www.nature.com/nature.

## Full Text

Drosophila CRY (dCRY), a Type I Cryptochrome13; 14, is the primary photoreceptor for entrainment of the fly circadian clock 1; 3 and also elicits magnetosensitivity 4; 5. Type II animal CRYs are also key to circadian rhythms in mammals but primarily participate in light-independent transcriptional repression3; 15. dCRY usually regulates circadian function by targeting the Timeless protein (TIM) for ubiquitin-mediated degradation in the presence of light16; 17. Light also causes dCRY degradation, but on a longer timescale7; 16. Both processes involve the E3-ubiquitin ligase jetlag (JET)16; 17; 18 and the associated action spectra are similar7; 18; 19; 20. The dCRY active photopigment is either FAD in the oxidized (FADox) or anionic semiquinone state (FAD•−)7; 13; 18; 20; 21. Light promotes dCRY binding to TIM; but TIM degradation may require an additional light response7; 8; 22. The light-dependent interaction of dCRY with TIM involves the C-terminal tail of dCRY7; 8; 9; 10. dCRY with the 19 or 20 C-terminal residues removed binds TIM both in the dark and light, but can still entrain the clock under low levels of light7; 8.

The structure of dCRY was determined to 2.3 Å resolution from crystals with an unusual form of non-merohedral twinning (see Supplementary Methods and Figs. S1–S4). The asymmetric unit comprises a dCRY dimer, but the contacts between subunits are not extensive (Supplementary Fig. S5). The dCRY structure most resembles that of the Drososphila 6-4 PL (6-4 dPL23) (Fig. 1a), but displays notable differences in loop structures, antenna cofactor binding site, FAD center and importantly, the variable C-terminal extension. The additional 23 C-terminal residues form a short connection (residues 516–529) from the PHD to a 10 residue helical tail (residues 530–539) that inserts into what would be the active center of the 6-4 photolyase (Fig. 1). The C-terminal tail (CTT) mimics the DNA substrates of PLs (6-4 and CPD lesions) and positions Trp536 (of the FFW motif invariant in Type I CRYs) analogously to the 6-4 photolesion (Fig. 1,2). The electronic state of FAD likely influence the extensive interactions between the binding pocket in dCRY and C-terminal residues 525–539 (Fig. 1d) and thus, FAD photochemistry may promote conformational changes at the CTT.

In cell culture, dCRY is less stable in light than dark and much less stable in both without the CTT (Δ)7; 10 or the CTT substituted with alanine residues (A10; Supplementary Fig. S6). Alanine substitution of the FFW motif reduced dCRY stability to a lesser extent in both the light and dark, whereas the W536A substitution alone had little effect (Supplementary Fig. S6a). For the Δ and A10 variants, where the CTT-PHD interaction is hence disrupted, TIM stabilizes the PHD equally in both light and dark (Supplementary Fig. S6b). In contrast, the FFW variant has a partially functional CTT interaction in dark and the stabilizing effect of TIM occurs mostly in the light (Supplementary Fig. S6b). Thus, TIM provides interactions to the PHD in light that compensate for those of the CTT in the dark. Because only the wt and W536A variant show enhanced dark vs. light stabilities with TIM (Supplementary Fig. S6b) a fully engaged CTT appears to stabilize the PHD to a greater extent than does TIM(Supplementary Fig. S6b). Phosphorylation may also regulate the CTT conformation and thereby dCRY stability. Mass-spectrometry and crystallography studies detected a phosphorylation site at Thr518, which lies at the junction between the PHD and the C-terminal helix (Supplementary Fig. S7).

Structural differences among CRYs and 6-4 PL reflect recognition of a CTT instead of a DNA substrate and different modes of regulation. Sequestration of the dCRY tail helix within the PHD DNA binding groove is coupled to positioning of three loop regions specific to Type I CRYs (Fig. 1,2; Supplementary Fig. S8): 1) the phosphate-binding loop (residues 249–263), which coordinates a phosphate anion close to FAD in the structure of A. thalania (At) PL6; 2) the protrusion motif (residues 288–306), an extended surface loop adjacent to the phosphate binding loop and 3) the C-terminal lid (residues 420–446), which packs against the dCRY C-terminal helix (Figs 2,3). Compared to 6-4 PL, the dCRY phosphate-binding loop is altered by a two turn extension of α8 that replaces the ~90° kink between the α8/α9 junction (Fig. 2, Supplementary Fig. S8). The dCRY α8 extension causes a 20 Å deviation from the PL structure that prevents approach of the phosphate binding loop to FAD (Fig. 2). This removes a hydrogen bond from Lys244 (at PL numbering) to the adenosine N7 that may stabilize a conserved site of Ser phosphorylation in Type II CRYs6. Although dCRY conserves this phosphorylation site (Ser261) it is remote from the active center and residues that would stabilize Ser phosphorylation in Type II CRYs are not found in Type I CRYs6(Fig. 2, Supplementary Fig. S9). In addition, several positively charged residues that bind the DNA phosphate backbone in 6-4 dPL (Lys154, Lys161, Lsy449, Arg502, Arg505)23 are absent in dCRY.

The Ser-rich “C-terminal lid” is conserved in sequence and structure by Type I CRYs but comprises a different motif in Type II CRYs and PLs (Fig. 2). In dCRY, the linker to the CTT binds between the C-terminal lid (residues 420–446) and the 154–160 loop, which extends down from the α/β domain (Supplementary Fig. S9). Given the close proximity of the C-terminal lid to where the CTT binds, this region may play a role in the recognition of targets, such as TIM or JET. Type II CRYs differ from Type I CRYs and 6-4 PLs in the C-terminal lid by the introduction of three cysteine residues (Fig. 2). The dCRY structure predicts that this cysteine rich sequence binds near the active site of Type II CRYs and may thereby serve a redox-related function (Fig. 2).

PLs utilize a pteridine-derived cofactor (either methenyl-trihydrofolate (MTHF), or a 8-hydroxy-deazaflavin (HDF) as a blue light antenna to sensitize the FADH•− cofactor and thereby faciliate charge injection into pyrimidine dimers2. dCRY has been reported to copurify with small amounts of MTHF21. In contrast, we found no cofactors additional to FAD in dCRY samples purified from insect cells through chemical, spectral or crystallographic analyses. Isothermal calorimetry also indicated no binding of MTHF nor the HDF-analog riboflavin to dCRY (Supplementary Fig. S10). Furthermore, residues that recognize the MTHF pterin moiety have very different character in dCRY than in E. coli PL (Fig. 3a, b) and Arabidopsis CRY3 (Supplementary Fig. S11). Compared to 6-4 dPl, dCRY conserves a pocket, loop flexibility, and some HDF-interacting residues (e.g. Phe40, Leu60), but others, particularly in the antenna recognition loop (residues 42–54) differ in dCRY (Fig. 3b and Supplementary Fig. S10). Thus, dCRY does not bind MTHF and is unlikely to bind HDF, unless a unique mode of recognition is employed. Notably, most CRYs assume the FADox or the FAD•− state, but not the activated FADH•− state required for DNA repair by PLs24. In the absence of an antenna cofactor dCRY can readily form FAD•− (Supplementary Fig. S12); which is likely important as either the ground state, or photoproduct for light-regulation of the CTT13; 18; 20; 21.

Upon light exposure dCRY rapidly converts to an anionic semiquinone FAD•−, which then decays back to the FADox state in a manner dependent on oxygen 13; 21; 24. In contrast, PLs typically form the neutral semiquinone (FADH•)24. In PLs, an Asn residue is poised to hydrogen bond with the protonated flavin N5, but does not interact with the unprotonated flavin (Fig. 3c). In dCRY, Cys416 at this same position is within hydrogen bonding distance of both unprotonated N5 and O4 (Fig. 3c). Thus, negative charge that localizes on O4 and N5 in FAD•− is stabilized by the neutral thiol. These interactions may prevent further proton transfer to FAD•− and thereby disfavor conversion to the 2-electron reduced state (FADH•−). Photoreduction of FADox to FAD•− depends upon stepwise electron transfer from the CRY/PL conserved Trp triad (Trp342 → Trp397 → Trp420 → FAD*)13; 21; 24. For Type I CRYs recombinantly expressed in insect cells, the Trp triad is required for photoreduction of FADox to FAD•−, but not for light-induced degradation of CRY13. However, unlike other CRY/PLs, Trp536 of the CTT provides an indole moiety as close to the flavin ring as Trp420 (Fig. 4a). Thus, photo-oxidation of Trp536 by FAD* or (FAD•−)*in Type I CRYs and could serve a role in photo-or magneto-sensing13; 25.

The dCRY flavin shows distortions indicative of reduction (Fig. 3c), as is characteristic of flavoprotein crystals exposed to high x-ray doses26; 27. Reduction increases sp3-character of N10 that leads to butterfly bending of the flavin and an expanded angle between the isoalloxazine ring and ribityl side chain26; 27. FAD•− likely forms in the crystal because low potential reductants cannot reduce dCRY beyond FAD•−18 and there is no further photoreduction, even on ps time scales 24. Importantly, the reduced flavin conformation influences neighboring residues that directly contact the tail helix (Fig. 3c). dCRY also conserves an active site His residue with PLs (His378) that contacts Trp536 (Fig. 3c, 4b). In PL, the His378 analog protonates in the flavin semiquinone state28; a similar process in dCRY would further destabilize interactions between the CTT and active center. Although crystal packing prevents CTT dissociation in the structure, the two crystallographically unique dCRY molecules display variation in tail helix binding mode, which includes displacement of Asp539 from a buried pocket in only one molecule (Fig. 4b). These conformers suggest that the CTT is not tightly bound when the flavin is reduced. A recent study revealed that light excitation of dCRY in the FAD•− state exposed two tryptic proteolysis sites at the dCRY C-terminus, one at the end of the PHD domain and the other within the CTT18. Given the substantial structural alterations in FAD upon reduction, it is somewhat surprising that more modest and short-lived changes induced by photo- excitation of FAD•− would be required to release the CTT. Nonetheless, the close association between the CTT and the flavin center provides the proximity to mechanistically link tail restructuring to flavin electronic state. Notably, light still drives rhythms in flies containing dCRY with the CTT removed7; 8. Thus, although flavin-dependent change in CTT conformation gate formation of the proper target complex with TIM or JET, additional dCRY photochemistry may be required to send appropriate signals in these and other processes independent of TIM4; 25; 29; 30.

Methods SummarydCRY amino acids 1–539 was cloned into pFastBac-HTa (Invitrogen) for Sf9 insect cell expression. Virus was selected and expanded (Kemp Biotechnologies Inc) and protein was at 27 C for 72 hours post infection. The 6xHis tagged protein was purified from 2L dCRY cultures via Ni-NTA (Qiagen) affinity chromatography. dCRY was eluted in buffer containing 100 mM imidazole, 50 mM Hepes pH 8, 150 mM NaCl, 2 mM DTT and 10% glycerol. The eluate was concentrated and run on size exclusion chromatography (Superdex 200 GE Healthcare Life Sciences) in buffer containing 50 mM Hepes pH 8 and 150 mM NaCl and 10% glycerol.dCRY crystals were obtained using the hanging drop method with equal volumes of well solution (18% Peg 4K, 150 mM Mg acetate, and 100 mM Tris pH 8.5) and dCRY (5 mg/ml). 2.3 Å resolution diffraction data were collected at the microbeam line ID-24E of the Advance Photon Light Source (APS) and corrected for nonmerohedral twinning as described in the Supplementary Methods. Molecular replacement with PDB file 1TEZ gave initial phases. The structure was built in XtalView and refined in CNS.dCRY cell culture studies were conducted with N-terminally 3xMyc tagged dCRY and 3xHA C-terminally tagged ls-TIM (dTIM) cloned into pAc5.1/V5 (Invitrogen). S2 cells in Sneider’s medium supplemented with 10% FBS were transfected with dCRY and either empty vector or dTIM plasmids. 24 hours post-transfection the cell medium was refreshed and the cells divided into two populations. After 24-hours cells were lysed under red light or after 1 hour white-light treatment (~600 lux). Relative abundance of dCRY was then assayed via 6% SDS-PAGE immunostained with myc (SIGMA) and tubulin (SIGMA).Supplementary Information is linked to the online version of the paper at http://www.nature.com/nature

Methods Summary

dCRY amino acids 1–539 was cloned into pFastBac-HTa (Invitrogen) for Sf9 insect cell expression. Virus was selected and expanded (Kemp Biotechnologies Inc) and protein was at 27 C for 72 hours post infection. The 6xHis tagged protein was purified from 2L dCRY cultures via Ni-NTA (Qiagen) affinity chromatography. dCRY was eluted in buffer containing 100 mM imidazole, 50 mM Hepes pH 8, 150 mM NaCl, 2 mM DTT and 10% glycerol. The eluate was concentrated and run on size exclusion chromatography (Superdex 200 GE Healthcare Life Sciences) in buffer containing 50 mM Hepes pH 8 and 150 mM NaCl and 10% glycerol.

dCRY crystals were obtained using the hanging drop method with equal volumes of well solution (18% Peg 4K, 150 mM Mg acetate, and 100 mM Tris pH 8.5) and dCRY (5 mg/ml). 2.3 Å resolution diffraction data were collected at the microbeam line ID-24E of the Advance Photon Light Source (APS) and corrected for nonmerohedral twinning as described in the Supplementary Methods. Molecular replacement with PDB file 1TEZ gave initial phases. The structure was built in XtalView and refined in CNS.

dCRY cell culture studies were conducted with N-terminally 3xMyc tagged dCRY and 3xHA C-terminally tagged ls-TIM (dTIM) cloned into pAc5.1/V5 (Invitrogen). S2 cells in Sneider’s medium supplemented with 10% FBS were transfected with dCRY and either empty vector or dTIM plasmids. 24 hours post-transfection the cell medium was refreshed and the cells divided into two populations. After 24-hours cells were lysed under red light or after 1 hour white-light treatment (~600 lux). Relative abundance of dCRY was then assayed via 6% SDS-PAGE immunostained with myc (SIGMA) and tubulin (SIGMA).

Supplementary Information is linked to the online version of the paper at http://www.nature.com/nature

Supplementary Material12

Supplementary Material
