---
pmid: '22190034'
title: Global landscape of HIV-human protein complexes.
authors:
- Jäger S
- Cimermancic P
- Gulbahce N
- Johnson JR
- McGovern KE
- Clarke SC
- Shales M
- Mercenne G
- Pache L
- Li K
- Hernandez H
- Jang GM
- Roth SL
- Akiva E
- Marlett J
- Stephens M
- D'Orso I
- Fernandes J
- Fahey M
- Mahon C
- O'Donoghue AJ
- Todorovic A
- Morris JH
- Maltby DA
- Alber T
- Cagney G
- Bushman FD
- Young JA
- Chanda SK
- Sundquist WI
- Kortemme T
- Hernandez RD
- Craik CS
- Burlingame A
- Sali A
- Frankel AD
- Krogan NJ
journal: Nature
year: '2011'
full_text_available: false
full_text_extraction_method: html_abstract_only
pmcid: PMC3310911
doi: 10.1038/nature10719
---

# Global landscape of HIV-human protein complexes.
**Authors:** Jäger S, Cimermancic P, Gulbahce N, Johnson JR, McGovern KE, Clarke SC, Shales M, Mercenne G, Pache L, Li K, Hernandez H, Jang GM, Roth SL, Akiva E, Marlett J, Stephens M, D'Orso I, Fernandes J, Fahey M, Mahon C, O'Donoghue AJ, Todorovic A, Morris JH, Maltby DA, Alber T, Cagney G, Bushman FD, Young JA, Chanda SK, Sundquist WI, Kortemme T, Hernandez RD, Craik CS, Burlingame A, Sali A, Frankel AD, Krogan NJ
**Journal:** Nature (2011)
**DOI:** [10.1038/nature10719](https://doi.org/10.1038/nature10719)
**PMC:** [PMC3310911](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3310911/)

## Abstract

1. Nature. 2011 Dec 21;481(7381):365-70. doi: 10.1038/nature10719.

Global landscape of HIV-human protein complexes.

Jäger S(1), Cimermancic P, Gulbahce N, Johnson JR, McGovern KE, Clarke SC, 
Shales M, Mercenne G, Pache L, Li K, Hernandez H, Jang GM, Roth SL, Akiva E, 
Marlett J, Stephens M, D'Orso I, Fernandes J, Fahey M, Mahon C, O'Donoghue AJ, 
Todorovic A, Morris JH, Maltby DA, Alber T, Cagney G, Bushman FD, Young JA, 
Chanda SK, Sundquist WI, Kortemme T, Hernandez RD, Craik CS, Burlingame A, Sali 
A, Frankel AD, Krogan NJ.

Author information:
(1)Department of Cellular and Molecular Pharmacology, University of California, 
San Francisco, California 94158, USA.

Comment in
    Nat Rev Microbiol. 2012 Jan 16;10(2):81. doi: 10.1038/nrmicro2739.

Human immunodeficiency virus (HIV) has a small genome and therefore relies 
heavily on the host cellular machinery to replicate. Identifying which host 
proteins and complexes come into physical contact with the viral proteins is 
crucial for a comprehensive understanding of how HIV rewires the host's cellular 
machinery during the course of infection. Here we report the use of affinity 
tagging and purification mass spectrometry to determine systematically the 
physical interactions of all 18 HIV-1 proteins and polyproteins with host 
proteins in two different human cell lines (HEK293 and Jurkat). Using a 
quantitative scoring system that we call MiST, we identified with high 
confidence 497 HIV-human protein-protein interactions involving 435 individual 
human proteins, with ∼40% of the interactions being identified in both cell 
types. We found that the host proteins hijacked by HIV, especially those found 
interacting in both cell types, are highly conserved across primates. We 
uncovered a number of host complexes targeted by viral proteins, including the 
finding that HIV protease cleaves eIF3d, a subunit of eukaryotic translation 
initiation factor 3. This host protein is one of eleven identified in this 
analysis that act to inhibit HIV replication. This data set facilitates a more 
comprehensive and detailed understanding of how the host machinery is 
manipulated during the course of HIV infection.

DOI: 10.1038/nature10719
PMCID: PMC3310911
PMID: 22190034 [Indexed for MEDLINE]

Conflict of interest statement: The authors declare no competing financial 
interests.

## Full Text

Abstract

Human immunodeficiency virus (HIV) has a small genome and therefore relies heavily on the host cellular machinery to replicate. Identifying which host proteins and complexes come into physical contact with the viral proteins is crucial for a comprehensive understanding of how HIV rewires the host’s cellular machinery during the course of infection. Here we report the use of affinity tagging and purification mass spectrometry 1 - 3 to determine systematically the physical interactions of all 18 HIV-1 proteins and polyproteins with host proteins in two different human cell lines (HEK293 and Jurkat). Using a quantitative scoring system that we call MiST, we identified with high confidence 497 HIV–human protein–protein interactions involving 435 individual human proteins, with ~40% of the interactions being identified in both cell types. We found that the host proteins hijacked by HIV, especially those found interacting in both cell types, are highly conserved across primates. We uncovered a number of host complexes targeted by viral proteins, including the finding that HIV protease cleaves eIF3d, a subunit of eukaryotic translation initiation factor 3. This host protein is one of eleven identified in this analysis that act to inhibit HIV replication. This data set facilitates a more comprehensive and detailed understanding of how the host machinery is manipulated during the course of HIV infection.

METHODS SUMMARY

More details on experimental assays, plasmid constructs, sequences, cell lines, antibodies and computational analysis are provided in Supplementary Methods . Briefly, affinity tagging and purification was carried out as previously described 2 and the protein samples were analysed on a Thermo Scientific LTQ Orbitrap XL mass spectrometer. For the evolutionary analysis, genome-wide alignments to rhesus macaque were downloaded from the University of California, Santa Cruz genome browser ( http://genome.ucsc.edu/ ) and evolutionary rates for each group of genes considered were measured using the synonymous and non-synonymous rates of evolution. For the in vitro protease assay, maltose binding protein (MBP)-tagged PR was expressed in BL21 (Gold) DE3 cells in the presence of 100 μM Saquinavir and purified on an MBP trap column. Purified eIF3 was obtained from J. Cate (University of California, Berkeley). For the infection assays, HeLa P4.R5 cells were transfected with short interfering RNAs and after 48 h infected with pNL4-3 or a pNL4-3-derived VSV-G-pseudotyped reporter virus. Infection levels were determined by luminescence read-out.
