---
pmid: '16189514'
title: Towards a proteome-scale map of the human protein-protein interaction network.
authors:
- Rual JF
- Venkatesan K
- Hao T
- Hirozane-Kishikawa T
- Dricot A
- Li N
- Berriz GF
- Gibbons FD
- Dreze M
- Ayivi-Guedehoussou N
- Klitgord N
- Simon C
- Boxem M
- Milstein S
- Rosenberg J
- Goldberg DS
- Zhang LV
- Wong SL
- Franklin G
- Li S
- Albala JS
- Lim J
- Fraughton C
- Llamosas E
- Cevik S
- Bex C
- Lamesch P
- Sikorski RS
- Vandenhaute J
- Zoghbi HY
- Smolyar A
- Bosak S
- Sequerra R
- Doucette-Stamm L
- Cusick ME
- Hill DE
- Roth FP
- Vidal M
journal: Nature
year: '2005'
full_text_available: true
pmcid: PMC12340668
doi: 10.1038/nature04209
---

# Towards a proteome-scale map of the human protein-protein interaction network.
**Authors:** Rual JF, Venkatesan K, Hao T, Hirozane-Kishikawa T, Dricot A, Li N, Berriz GF, Gibbons FD, Dreze M, Ayivi-Guedehoussou N, Klitgord N, Simon C, Boxem M, Milstein S, Rosenberg J, Goldberg DS, Zhang LV, Wong SL, Franklin G, Li S, Albala JS, Lim J, Fraughton C, Llamosas E, Cevik S, Bex C, Lamesch P, Sikorski RS, Vandenhaute J, Zoghbi HY, Smolyar A, Bosak S, Sequerra R, Doucette-Stamm L, Cusick ME, Hill DE, Roth FP, Vidal M
**Journal:** Nature (2005)
**DOI:** [10.1038/nature04209](https://doi.org/10.1038/nature04209)
**PMC:** [PMC12340668](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC12340668/)

## Abstract

1. Nature. 2005 Oct 20;437(7062):1173-8. doi: 10.1038/nature04209. Epub 2005 Sep 
28.

Towards a proteome-scale map of the human protein-protein interaction network.

Rual JF(1), Venkatesan K, Hao T, Hirozane-Kishikawa T, Dricot A, Li N, Berriz 
GF, Gibbons FD, Dreze M, Ayivi-Guedehoussou N, Klitgord N, Simon C, Boxem M, 
Milstein S, Rosenberg J, Goldberg DS, Zhang LV, Wong SL, Franklin G, Li S, 
Albala JS, Lim J, Fraughton C, Llamosas E, Cevik S, Bex C, Lamesch P, Sikorski 
RS, Vandenhaute J, Zoghbi HY, Smolyar A, Bosak S, Sequerra R, Doucette-Stamm L, 
Cusick ME, Hill DE, Roth FP, Vidal M.

Author information:
(1)Center for Cancer Systems Biology and Department of Cancer Biology, 
Dana-Farber Cancer Institute, Harvard Medical School, 44 Binney Street, Boston, 
Massachusetts 02115, USA.

Systematic mapping of protein-protein interactions, or 'interactome' mapping, 
was initiated in model organisms, starting with defined biological processes and 
then expanding to the scale of the proteome. Although far from complete, such 
maps have revealed global topological and dynamic features of interactome 
networks that relate to known biological properties, suggesting that a human 
interactome map will provide insight into development and disease mechanisms at 
a systems level. Here we describe an initial version of a proteome-scale map of 
human binary protein-protein interactions. Using a stringent, high-throughput 
yeast two-hybrid system, we tested pairwise interactions among the products of 
approximately 8,100 currently available Gateway-cloned open reading frames and 
detected approximately 2,800 interactions. This data set, called CCSB-HI1, has a 
verification rate of approximately 78% as revealed by an independent co-affinity 
purification assay, and correlates significantly with other biological 
attributes. The CCSB-HI1 data set increases by approximately 70% the set of 
available binary interactions within the tested space and reveals more than 300 
new connections to over 100 disease-associated proteins. This work represents an 
important step towards a systematic and comprehensive human interactome project.

DOI: 10.1038/nature04209
PMID: 16189514 [Indexed for MEDLINE]

## Full Text

1.IntroductionCharcot-Marie-Tooth disease (CMT) is the eponym for inherited neuropathies that are not part of a larger syndrome (Fridman et al., 2015). CMT affects ~1 in 2500 individuals and is caused by mutations in more than 100 different genes. In addition to the conventionally recognized forms of CMT, there are hundreds of genetic syndromes that include neuropathy, and for some of these, neuropathy can be the initial clinical manifestation (Rossor et al., 2017). Inherited neuropathies are typically named by their pattern of inheritance and by whether demyelination or axonal loss is the primary event in the pathogenesis, inferred from the conduction velocity of peripheral nerves. Recessive Trim2 mutations cause “CMT2R” (OMIM 615490), a rare, recessively inherited neuropathy (Pehlivan et al., 2015; Ylikallio et al., 2013) that is likely misnamed as CMT as it may be part of larger syndrome as we show here. One CMTR patient was a compound heterozygote - a missense mutation (p.Glu227Val) that destabilized the protein and a 1-bp deletion (c.1699delA) leading to a frameshift with premature termination and destabilized mRNA (Ylikallio et al., 2013). In the other reported case, exome sequencing revealed a missense mutation p.D667A; neither RNA or protein levels were examined in this individual (Pehlivan et al., 2015).The human genome encodes at least 80 TRIM proteins, which are characterized by an N-terminal RING-B-box-coiled-coil domain that is involved in protein–protein interactions (van Gent et al., 2018). The C-terminal domains of TRIM proteins are more variable, with approximately 10 different motifs that are present in the various family members. The C-terminus of the 4 mammalian subgroup VII TRIM proteins (TRIM2, TRIM3, TRIM32, and TRIM71) contain filamin domains as well as NCL-1, HT2A, and Lin-41 (NHL) repeats (Tocchini and Ciosk, 2015). TRIM3 has been implicated in the transport of cellular cargo (Labonte et al., 2013) and Trim3-null mice are fully viable (Cheung et al., 2010). TRIM32 is thought to play a role in muscle filaments and to ubiquitinate actin (Kudryashova et al., 2005); recessive mutations in TRIM32 cause a myopathy in humans (Frosk et al., 2002) and mice (Kudryashova et al., 2009). TRIM71 is involved in the biology of microRNAs (Chang et al., 2012; Loedige et al., 2013), and recessive mutations are embryonic lethal in mice (Schulman et al., 2008). TRIM2 is thought to be a ubiquitin ligase, with the neurofilament light subunit (NFL) as one of its substrates (Balastik et al., 2008). As part of our studies on the role of TRIM2 in infection of new world arenaviruses (NWA) (Lavanya et al., 2013), we created a series of mutant mice with deletions of different domains of Trim2, using CRISPR-Cas9 (Sarute et al., 2019). Here, we analyze the two mouse strains shown in Supplemental Fig. 1 – "strain A", which deleted sequences between the two guide RNAs and potentially expresses only the RING domain because of a stop codon introduced by the deletion (and produced an unstable protein), and "strain C", which deleted 30 amino acids, including the C′ terminal portion of the RING domain and retains the rest of the protein. Strain A mice and cells were more susceptible to infection with NWA, while strain C mice were infected to the same extent as wild type (WT) mice (Sarute et al., 2019).Here we provide the first description of the neurological phenotype in Trim2A/A mice and demonstrate that neurological disease is not strictly associated with the ubiquitin activity of TRIM2. Trim2A/A mice, but not Trim2C/C mice, develop an axonal neuropathy and Purkinje cell loss, and these mice are valuable models of CMT2R.

Introduction

Charcot-Marie-Tooth disease (CMT) is the eponym for inherited neuropathies that are not part of a larger syndrome (Fridman et al., 2015). CMT affects ~1 in 2500 individuals and is caused by mutations in more than 100 different genes. In addition to the conventionally recognized forms of CMT, there are hundreds of genetic syndromes that include neuropathy, and for some of these, neuropathy can be the initial clinical manifestation (Rossor et al., 2017). Inherited neuropathies are typically named by their pattern of inheritance and by whether demyelination or axonal loss is the primary event in the pathogenesis, inferred from the conduction velocity of peripheral nerves. Recessive Trim2 mutations cause “CMT2R” (OMIM 615490), a rare, recessively inherited neuropathy (Pehlivan et al., 2015; Ylikallio et al., 2013) that is likely misnamed as CMT as it may be part of larger syndrome as we show here. One CMTR patient was a compound heterozygote - a missense mutation (p.Glu227Val) that destabilized the protein and a 1-bp deletion (c.1699delA) leading to a frameshift with premature termination and destabilized mRNA (Ylikallio et al., 2013). In the other reported case, exome sequencing revealed a missense mutation p.D667A; neither RNA or protein levels were examined in this individual (Pehlivan et al., 2015).

The human genome encodes at least 80 TRIM proteins, which are characterized by an N-terminal RING-B-box-coiled-coil domain that is involved in protein–protein interactions (van Gent et al., 2018). The C-terminal domains of TRIM proteins are more variable, with approximately 10 different motifs that are present in the various family members. The C-terminus of the 4 mammalian subgroup VII TRIM proteins (TRIM2, TRIM3, TRIM32, and TRIM71) contain filamin domains as well as NCL-1, HT2A, and Lin-41 (NHL) repeats (Tocchini and Ciosk, 2015). TRIM3 has been implicated in the transport of cellular cargo (Labonte et al., 2013) and Trim3-null mice are fully viable (Cheung et al., 2010). TRIM32 is thought to play a role in muscle filaments and to ubiquitinate actin (Kudryashova et al., 2005); recessive mutations in TRIM32 cause a myopathy in humans (Frosk et al., 2002) and mice (Kudryashova et al., 2009). TRIM71 is involved in the biology of microRNAs (Chang et al., 2012; Loedige et al., 2013), and recessive mutations are embryonic lethal in mice (Schulman et al., 2008). TRIM2 is thought to be a ubiquitin ligase, with the neurofilament light subunit (NFL) as one of its substrates (Balastik et al., 2008). As part of our studies on the role of TRIM2 in infection of new world arenaviruses (NWA) (Lavanya et al., 2013), we created a series of mutant mice with deletions of different domains of Trim2, using CRISPR-Cas9 (Sarute et al., 2019). Here, we analyze the two mouse strains shown in Supplemental Fig. 1 – "strain A", which deleted sequences between the two guide RNAs and potentially expresses only the RING domain because of a stop codon introduced by the deletion (and produced an unstable protein), and "strain C", which deleted 30 amino acids, including the C′ terminal portion of the RING domain and retains the rest of the protein. Strain A mice and cells were more susceptible to infection with NWA, while strain C mice were infected to the same extent as wild type (WT) mice (Sarute et al., 2019).

Here we provide the first description of the neurological phenotype in Trim2A/A mice and demonstrate that neurological disease is not strictly associated with the ubiquitin activity of TRIM2. Trim2A/A mice, but not Trim2C/C mice, develop an axonal neuropathy and Purkinje cell loss, and these mice are valuable models of CMT2R.

2.Materials and methods2.1.MiceWe followed the NIH guide for the care and use of laboratory animals (NIH publications No. 8023). Briefly, Trim2-targeting sgRNAs and CRISPR RNAs were microinjected into C57BL/6 N zygotes. The ensuing mice born from this injection were analyzed by PCR of genomic DNA using primers that flanked the targeted region and T7 endonuclease cleavage assays. The mutant mice were crossed with C57BL/6 N mice and the heterozygous offspring of these crosses were both inter-crossed to generate mice with deletions on both alleles and crossed with additional C57BL/6 N mice to expand the colony. Both strains were crossed onto wild type BL/6 N mice for at least 2 generations. Sequencing of both genomic DNA and RNA from these mice was carried out to precisely map the deletion breakpoints and can be found in our previous publication (Sarute et al., 2019). The largest deletion of Trim2, here called Trim2A, truncates TRIM2 after the RING domain, but this protein was not stably expressed, so the mice are functionally null for TRIM2 (Sarute et al., 2019). A second mutant allele was generated in the same CRISPR-Cas9 targeted mutagenesis, Trim2C, which deleted part of the C-terminal RING domain that is needed for ubiquitin ligase activity (Sarute et al., 2019). The deletion in the strains is shown schematically in Supplemental Fig. 1. We subcloned Trim2A and Trim2C cDNAs derived from tissues harvested from Trim2A/A and Trim2C/C mice (Sarute et al., 2019). Because male Trim2A/A mice were unable to breed because of their ataxia, homozygous Trim2A/A and Trim2+/+/WT mice were obtained by crossing heterozygous Trim2A/+ parents, and were genotyped as previously described (Sarute et al., 2019). Because Trim2C/C strain mice did not have ataxia, homozygous mutants could breed.2.2.Quantification of cerebellar ataxiaAtaxia was scored by a blinded observer, using the ledge test, hindlimb clasping, gait and kyphosis (Guyenet et al., 2010). Scores from all four tests were averaged to give a total score per mouse.2.3.Nerve conductionsA portable machine (Nicolet) was used to record the compound action potentials (CAPs) from the caudal nerves. Mice were anesthetized with ketamine: xylazine (10:100 mg/kg, IP). After full anesthesia was verified by toe pinch, mice were placed supine with the tail fully extended. Electrodes were placed on the tail following the method of Maia et al. (2010). Wire loop electrodes were used for recording and placed with the reference loop 2 mm from the tail base and the active electrode 5 mm more distal along the tail. Two pin electrodes were used as ground electrodes (separated by 6 mm). More distally along the tail, a pair of loop electrodes was used for stimulation (separated by 5 mm). The distance between the recording electrode and stimulating cathode electrode is 30 mm. The intensity of electrical stimulation for mice is equivalent to the intensity used in clinical nerve conduction studies in awake humans. A series of 5 responses were averaged to reduce noise and variability. Tail surface temperature was maintained at 30–32 °C using a heating pad and monitored using infrared non-contact ther-mometer during recording. Upon completion of recording, mice were sacrificed and processed for electron microscopy (EM) as described below.2.4.ImmunohistochemistryMouse brains and cervical spinals were fixed with 10% neutral buffered formalin (NFB), embedded in paraffin, and sectioned at 5 μm. Sections were deparaffinized and stained on Bond RX autostainer (Leica Biosystems) following a preset protocol. Dorsal root ganglia (DRG) were fixed in cold 10% NBF for 20 min, embedded in OCT, sectioned at 10 μm, and stained on BondRX following the same protocol except that deparaffinization step was omitted. Sections were subjected to citric acid-based (Bond ER1 solution, pH = 6) antigen retrieval for 40 min at 99 °C, treated for 15 min with hydrogen peroxide, washed (Bond Wash Solution), blocked for 15 min with Background Sniper protein block (#BS966, Biocare Medical), labeled with an anti-TRIM2 rabbit antiserum (1:100; Millipore Sigma, #HPA035853) for 30 min, and washed. The detection was performed using Bond Polymer Refine Detection kit (Leica Biosystems, DS9800), which uses a peroxidase-conjugated goat anti-rabbit antiserum and diaminobenzidine as the chromogen. Slides were counterstained with hematoxylin for 10 min and mounted with Surgipath Micromount Media (Leica Biosystems), and were scanned on a Leica Aperio ScanScope at 20× magnification. Some sections were labeled with secondary antibody only.To quantify the number of Purkinje cells, cerebellum sections were labeled with a rabbit antiserum against calbindin-D-28K (1:2000, Millipore Sigma #C2724), and visualized with goat-anti-rabbit Alexa-594 (1:250, Thermo A-11012). The slides were incubated with DAPI (Invitrogen, #D3571) for 10 min and mounted with ProLong Diamond Antifade mounting media (Life Technologies, #P36961). Purkinje cells were counted by a blinded observer in an average of 3 sections from 8 to 12 weeks old Trim2A/A and WT mice (3 mice of each genotype) and the averages of the 3 sections for each mouse were plotted.To co-label for NFL and SIRPA, slides were treated as described above, and sequentially labeled with rabbit anti-TRIM2 antiserum (1:50), chicken anti-NFL antiserum (1:50; Neuromics, #CH22105), and a rat monoclonal anti-CD172a/SIRPA (1:50; Biolegend, #144001) on the BondRX platform using species-specific, goat secondary antibodies that were conjugated to Alexa-488, −555, and −647 (Molecular Probes), respectively, and counterstained with DAPI. Slides were mounted with ProLong Diamond antifade mounting media, scanned on a Vectra®3 (Akoya Biosciences, MA) at 4×. The 4× images were open in Phenochart (Akoya Biosciences, MA), where regions of interest were selected and scanned at 20× magnification. A spectral library acquired for each fluorophore and DAPI, and an auto-fluorescence slide was created with Nuance (Akoya Biosciences, MA). Images were imported into inForm v2.4.6 for spectral unmixing and visualization of each colour.2.5.Transmission EM and g-ratio measurementsAnesthetized mice were transcardially perfused with 2% paraformaldehyde and 2% glutaraldehyde in 0.1 M phosphate buffer (pH 7.4). The cerebellum, DRG and attached nerve roots, cervical spinal cord, and femoral nerves were dissected, fixed for at least 4 more hours at 4 °C, then were osmicated, dehydrated, infiltrated, and embedded in Embed 812 mixture (Electron Microscopy Sciences) as previously described (Potter et al., 2011). For light microscopy, cross-sections were cut at a thickness of 1 μm and stained with alkaline toluidine blue. The g-ratio was calculated by the square root of the ratio of inner to outer axonal area using Image J software. Two 100× images of a femoral motor nerve from 26-week-old WT (N = 3) and Trim2A/A (N = 3) mice were analyzed. For EM, cross-sections were cut at a thickness of 90 nm, and stained with lead citrate and uranyl acetate. The ultra-thin sections were imaged using a JEOL 1010 electron microscope.2.6.Western blot and ubiquitination assayProtein lysates were prepared from 8 to 12-week-old Trim2A/A, Trim2C/C and WT/Trim2+/+ mice. Equal amounts of protein extracts (50 μg) were resolved by 10% SDS-PAGE and transferred to polyvinylidene difluoride membranes. TRIM2, NFL, SIRPA, and GAPDH were detected with rabbit antisera as previously described (Sarute et al., 2019). S100b, P2RY12, NeuN, and GFAP were detected with rabbit antisera (anti-S100β and -P2RY12 from Proteintech; anti-NeuN and-GFAP from Cell Signaling Technologies); a mouse monoclonal antibody was used for IBA1 (Millipore).For the ubiquitination assays, full-length Trim2 and Trim2C constructs were co-transfected with NFL and hemagglutinin (HA)-ubiquitin constructs in 293 T cells and subjected to ubiquitination assay (Sarute et al., 2019). In brief, MG-132-treated cells were lysed and subjected to immunoprecipitation with a rabbit antiserum against HA (Abcam) and analyzed by western blot using rabbit antisera against TRIM2 and NFL.2.7.Statistical analysisData shown is the average of at least 3 independent experiments, or as indicated in the Figure legends. Statistical analysis was performed using the GraphPad 8.1/PRISM software.

Materials and methods

2.1.MiceWe followed the NIH guide for the care and use of laboratory animals (NIH publications No. 8023). Briefly, Trim2-targeting sgRNAs and CRISPR RNAs were microinjected into C57BL/6 N zygotes. The ensuing mice born from this injection were analyzed by PCR of genomic DNA using primers that flanked the targeted region and T7 endonuclease cleavage assays. The mutant mice were crossed with C57BL/6 N mice and the heterozygous offspring of these crosses were both inter-crossed to generate mice with deletions on both alleles and crossed with additional C57BL/6 N mice to expand the colony. Both strains were crossed onto wild type BL/6 N mice for at least 2 generations. Sequencing of both genomic DNA and RNA from these mice was carried out to precisely map the deletion breakpoints and can be found in our previous publication (Sarute et al., 2019). The largest deletion of Trim2, here called Trim2A, truncates TRIM2 after the RING domain, but this protein was not stably expressed, so the mice are functionally null for TRIM2 (Sarute et al., 2019). A second mutant allele was generated in the same CRISPR-Cas9 targeted mutagenesis, Trim2C, which deleted part of the C-terminal RING domain that is needed for ubiquitin ligase activity (Sarute et al., 2019). The deletion in the strains is shown schematically in Supplemental Fig. 1. We subcloned Trim2A and Trim2C cDNAs derived from tissues harvested from Trim2A/A and Trim2C/C mice (Sarute et al., 2019). Because male Trim2A/A mice were unable to breed because of their ataxia, homozygous Trim2A/A and Trim2+/+/WT mice were obtained by crossing heterozygous Trim2A/+ parents, and were genotyped as previously described (Sarute et al., 2019). Because Trim2C/C strain mice did not have ataxia, homozygous mutants could breed.

We followed the NIH guide for the care and use of laboratory animals (NIH publications No. 8023). Briefly, Trim2-targeting sgRNAs and CRISPR RNAs were microinjected into C57BL/6 N zygotes. The ensuing mice born from this injection were analyzed by PCR of genomic DNA using primers that flanked the targeted region and T7 endonuclease cleavage assays. The mutant mice were crossed with C57BL/6 N mice and the heterozygous offspring of these crosses were both inter-crossed to generate mice with deletions on both alleles and crossed with additional C57BL/6 N mice to expand the colony. Both strains were crossed onto wild type BL/6 N mice for at least 2 generations. Sequencing of both genomic DNA and RNA from these mice was carried out to precisely map the deletion breakpoints and can be found in our previous publication (Sarute et al., 2019). The largest deletion of Trim2, here called Trim2A, truncates TRIM2 after the RING domain, but this protein was not stably expressed, so the mice are functionally null for TRIM2 (Sarute et al., 2019). A second mutant allele was generated in the same CRISPR-Cas9 targeted mutagenesis, Trim2C, which deleted part of the C-terminal RING domain that is needed for ubiquitin ligase activity (Sarute et al., 2019). The deletion in the strains is shown schematically in Supplemental Fig. 1. We subcloned Trim2A and Trim2C cDNAs derived from tissues harvested from Trim2A/A and Trim2C/C mice (Sarute et al., 2019). Because male Trim2A/A mice were unable to breed because of their ataxia, homozygous Trim2A/A and Trim2+/+/WT mice were obtained by crossing heterozygous Trim2A/+ parents, and were genotyped as previously described (Sarute et al., 2019). Because Trim2C/C strain mice did not have ataxia, homozygous mutants could breed.

2.2.Quantification of cerebellar ataxiaAtaxia was scored by a blinded observer, using the ledge test, hindlimb clasping, gait and kyphosis (Guyenet et al., 2010). Scores from all four tests were averaged to give a total score per mouse.

Quantification of cerebellar ataxia

Ataxia was scored by a blinded observer, using the ledge test, hindlimb clasping, gait and kyphosis (Guyenet et al., 2010). Scores from all four tests were averaged to give a total score per mouse.

2.3.Nerve conductionsA portable machine (Nicolet) was used to record the compound action potentials (CAPs) from the caudal nerves. Mice were anesthetized with ketamine: xylazine (10:100 mg/kg, IP). After full anesthesia was verified by toe pinch, mice were placed supine with the tail fully extended. Electrodes were placed on the tail following the method of Maia et al. (2010). Wire loop electrodes were used for recording and placed with the reference loop 2 mm from the tail base and the active electrode 5 mm more distal along the tail. Two pin electrodes were used as ground electrodes (separated by 6 mm). More distally along the tail, a pair of loop electrodes was used for stimulation (separated by 5 mm). The distance between the recording electrode and stimulating cathode electrode is 30 mm. The intensity of electrical stimulation for mice is equivalent to the intensity used in clinical nerve conduction studies in awake humans. A series of 5 responses were averaged to reduce noise and variability. Tail surface temperature was maintained at 30–32 °C using a heating pad and monitored using infrared non-contact ther-mometer during recording. Upon completion of recording, mice were sacrificed and processed for electron microscopy (EM) as described below.

Nerve conductions

A portable machine (Nicolet) was used to record the compound action potentials (CAPs) from the caudal nerves. Mice were anesthetized with ketamine: xylazine (10:100 mg/kg, IP). After full anesthesia was verified by toe pinch, mice were placed supine with the tail fully extended. Electrodes were placed on the tail following the method of Maia et al. (2010). Wire loop electrodes were used for recording and placed with the reference loop 2 mm from the tail base and the active electrode 5 mm more distal along the tail. Two pin electrodes were used as ground electrodes (separated by 6 mm). More distally along the tail, a pair of loop electrodes was used for stimulation (separated by 5 mm). The distance between the recording electrode and stimulating cathode electrode is 30 mm. The intensity of electrical stimulation for mice is equivalent to the intensity used in clinical nerve conduction studies in awake humans. A series of 5 responses were averaged to reduce noise and variability. Tail surface temperature was maintained at 30–32 °C using a heating pad and monitored using infrared non-contact ther-mometer during recording. Upon completion of recording, mice were sacrificed and processed for electron microscopy (EM) as described below.

2.4.ImmunohistochemistryMouse brains and cervical spinals were fixed with 10% neutral buffered formalin (NFB), embedded in paraffin, and sectioned at 5 μm. Sections were deparaffinized and stained on Bond RX autostainer (Leica Biosystems) following a preset protocol. Dorsal root ganglia (DRG) were fixed in cold 10% NBF for 20 min, embedded in OCT, sectioned at 10 μm, and stained on BondRX following the same protocol except that deparaffinization step was omitted. Sections were subjected to citric acid-based (Bond ER1 solution, pH = 6) antigen retrieval for 40 min at 99 °C, treated for 15 min with hydrogen peroxide, washed (Bond Wash Solution), blocked for 15 min with Background Sniper protein block (#BS966, Biocare Medical), labeled with an anti-TRIM2 rabbit antiserum (1:100; Millipore Sigma, #HPA035853) for 30 min, and washed. The detection was performed using Bond Polymer Refine Detection kit (Leica Biosystems, DS9800), which uses a peroxidase-conjugated goat anti-rabbit antiserum and diaminobenzidine as the chromogen. Slides were counterstained with hematoxylin for 10 min and mounted with Surgipath Micromount Media (Leica Biosystems), and were scanned on a Leica Aperio ScanScope at 20× magnification. Some sections were labeled with secondary antibody only.To quantify the number of Purkinje cells, cerebellum sections were labeled with a rabbit antiserum against calbindin-D-28K (1:2000, Millipore Sigma #C2724), and visualized with goat-anti-rabbit Alexa-594 (1:250, Thermo A-11012). The slides were incubated with DAPI (Invitrogen, #D3571) for 10 min and mounted with ProLong Diamond Antifade mounting media (Life Technologies, #P36961). Purkinje cells were counted by a blinded observer in an average of 3 sections from 8 to 12 weeks old Trim2A/A and WT mice (3 mice of each genotype) and the averages of the 3 sections for each mouse were plotted.To co-label for NFL and SIRPA, slides were treated as described above, and sequentially labeled with rabbit anti-TRIM2 antiserum (1:50), chicken anti-NFL antiserum (1:50; Neuromics, #CH22105), and a rat monoclonal anti-CD172a/SIRPA (1:50; Biolegend, #144001) on the BondRX platform using species-specific, goat secondary antibodies that were conjugated to Alexa-488, −555, and −647 (Molecular Probes), respectively, and counterstained with DAPI. Slides were mounted with ProLong Diamond antifade mounting media, scanned on a Vectra®3 (Akoya Biosciences, MA) at 4×. The 4× images were open in Phenochart (Akoya Biosciences, MA), where regions of interest were selected and scanned at 20× magnification. A spectral library acquired for each fluorophore and DAPI, and an auto-fluorescence slide was created with Nuance (Akoya Biosciences, MA). Images were imported into inForm v2.4.6 for spectral unmixing and visualization of each colour.

Immunohistochemistry

Mouse brains and cervical spinals were fixed with 10% neutral buffered formalin (NFB), embedded in paraffin, and sectioned at 5 μm. Sections were deparaffinized and stained on Bond RX autostainer (Leica Biosystems) following a preset protocol. Dorsal root ganglia (DRG) were fixed in cold 10% NBF for 20 min, embedded in OCT, sectioned at 10 μm, and stained on BondRX following the same protocol except that deparaffinization step was omitted. Sections were subjected to citric acid-based (Bond ER1 solution, pH = 6) antigen retrieval for 40 min at 99 °C, treated for 15 min with hydrogen peroxide, washed (Bond Wash Solution), blocked for 15 min with Background Sniper protein block (#BS966, Biocare Medical), labeled with an anti-TRIM2 rabbit antiserum (1:100; Millipore Sigma, #HPA035853) for 30 min, and washed. The detection was performed using Bond Polymer Refine Detection kit (Leica Biosystems, DS9800), which uses a peroxidase-conjugated goat anti-rabbit antiserum and diaminobenzidine as the chromogen. Slides were counterstained with hematoxylin for 10 min and mounted with Surgipath Micromount Media (Leica Biosystems), and were scanned on a Leica Aperio ScanScope at 20× magnification. Some sections were labeled with secondary antibody only.

To quantify the number of Purkinje cells, cerebellum sections were labeled with a rabbit antiserum against calbindin-D-28K (1:2000, Millipore Sigma #C2724), and visualized with goat-anti-rabbit Alexa-594 (1:250, Thermo A-11012). The slides were incubated with DAPI (Invitrogen, #D3571) for 10 min and mounted with ProLong Diamond Antifade mounting media (Life Technologies, #P36961). Purkinje cells were counted by a blinded observer in an average of 3 sections from 8 to 12 weeks old Trim2A/A and WT mice (3 mice of each genotype) and the averages of the 3 sections for each mouse were plotted.

To co-label for NFL and SIRPA, slides were treated as described above, and sequentially labeled with rabbit anti-TRIM2 antiserum (1:50), chicken anti-NFL antiserum (1:50; Neuromics, #CH22105), and a rat monoclonal anti-CD172a/SIRPA (1:50; Biolegend, #144001) on the BondRX platform using species-specific, goat secondary antibodies that were conjugated to Alexa-488, −555, and −647 (Molecular Probes), respectively, and counterstained with DAPI. Slides were mounted with ProLong Diamond antifade mounting media, scanned on a Vectra®3 (Akoya Biosciences, MA) at 4×. The 4× images were open in Phenochart (Akoya Biosciences, MA), where regions of interest were selected and scanned at 20× magnification. A spectral library acquired for each fluorophore and DAPI, and an auto-fluorescence slide was created with Nuance (Akoya Biosciences, MA). Images were imported into inForm v2.4.6 for spectral unmixing and visualization of each colour.

2.5.Transmission EM and g-ratio measurementsAnesthetized mice were transcardially perfused with 2% paraformaldehyde and 2% glutaraldehyde in 0.1 M phosphate buffer (pH 7.4). The cerebellum, DRG and attached nerve roots, cervical spinal cord, and femoral nerves were dissected, fixed for at least 4 more hours at 4 °C, then were osmicated, dehydrated, infiltrated, and embedded in Embed 812 mixture (Electron Microscopy Sciences) as previously described (Potter et al., 2011). For light microscopy, cross-sections were cut at a thickness of 1 μm and stained with alkaline toluidine blue. The g-ratio was calculated by the square root of the ratio of inner to outer axonal area using Image J software. Two 100× images of a femoral motor nerve from 26-week-old WT (N = 3) and Trim2A/A (N = 3) mice were analyzed. For EM, cross-sections were cut at a thickness of 90 nm, and stained with lead citrate and uranyl acetate. The ultra-thin sections were imaged using a JEOL 1010 electron microscope.

Transmission EM and g-ratio measurements

Anesthetized mice were transcardially perfused with 2% paraformaldehyde and 2% glutaraldehyde in 0.1 M phosphate buffer (pH 7.4). The cerebellum, DRG and attached nerve roots, cervical spinal cord, and femoral nerves were dissected, fixed for at least 4 more hours at 4 °C, then were osmicated, dehydrated, infiltrated, and embedded in Embed 812 mixture (Electron Microscopy Sciences) as previously described (Potter et al., 2011). For light microscopy, cross-sections were cut at a thickness of 1 μm and stained with alkaline toluidine blue. The g-ratio was calculated by the square root of the ratio of inner to outer axonal area using Image J software. Two 100× images of a femoral motor nerve from 26-week-old WT (N = 3) and Trim2A/A (N = 3) mice were analyzed. For EM, cross-sections were cut at a thickness of 90 nm, and stained with lead citrate and uranyl acetate. The ultra-thin sections were imaged using a JEOL 1010 electron microscope.

2.6.Western blot and ubiquitination assayProtein lysates were prepared from 8 to 12-week-old Trim2A/A, Trim2C/C and WT/Trim2+/+ mice. Equal amounts of protein extracts (50 μg) were resolved by 10% SDS-PAGE and transferred to polyvinylidene difluoride membranes. TRIM2, NFL, SIRPA, and GAPDH were detected with rabbit antisera as previously described (Sarute et al., 2019). S100b, P2RY12, NeuN, and GFAP were detected with rabbit antisera (anti-S100β and -P2RY12 from Proteintech; anti-NeuN and-GFAP from Cell Signaling Technologies); a mouse monoclonal antibody was used for IBA1 (Millipore).For the ubiquitination assays, full-length Trim2 and Trim2C constructs were co-transfected with NFL and hemagglutinin (HA)-ubiquitin constructs in 293 T cells and subjected to ubiquitination assay (Sarute et al., 2019). In brief, MG-132-treated cells were lysed and subjected to immunoprecipitation with a rabbit antiserum against HA (Abcam) and analyzed by western blot using rabbit antisera against TRIM2 and NFL.

Western blot and ubiquitination assay

Protein lysates were prepared from 8 to 12-week-old Trim2A/A, Trim2C/C and WT/Trim2+/+ mice. Equal amounts of protein extracts (50 μg) were resolved by 10% SDS-PAGE and transferred to polyvinylidene difluoride membranes. TRIM2, NFL, SIRPA, and GAPDH were detected with rabbit antisera as previously described (Sarute et al., 2019). S100b, P2RY12, NeuN, and GFAP were detected with rabbit antisera (anti-S100β and -P2RY12 from Proteintech; anti-NeuN and-GFAP from Cell Signaling Technologies); a mouse monoclonal antibody was used for IBA1 (Millipore).

For the ubiquitination assays, full-length Trim2 and Trim2C constructs were co-transfected with NFL and hemagglutinin (HA)-ubiquitin constructs in 293 T cells and subjected to ubiquitination assay (Sarute et al., 2019). In brief, MG-132-treated cells were lysed and subjected to immunoprecipitation with a rabbit antiserum against HA (Abcam) and analyzed by western blot using rabbit antisera against TRIM2 and NFL.

2.7.Statistical analysisData shown is the average of at least 3 independent experiments, or as indicated in the Figure legends. Statistical analysis was performed using the GraphPad 8.1/PRISM software.

Statistical analysis

Data shown is the average of at least 3 independent experiments, or as indicated in the Figure legends. Statistical analysis was performed using the GraphPad 8.1/PRISM software.

3.Results3.1.Generating Trim2 mutant miceWe previously described and partially characterized mice with targeted deletions in Trim2 using CRISPR-Cas9 and guide RNAs to exons 3 and 9, generating an allele, Trim2A, that is predicted to truncate TRIM2 after the RING domain (Supplemental Fig. 1); this allele does not produce a stable protein in transfected cells (Sarute et al., 2019). This targeting also produced a second strain of mice, Trim2C, with a deletion of the 3′ of exon 3, resulting in a stable protein that lacked 30 amino acids, including the C-terminal portion of the RING domain that is required for ubiquitin ligase activity (Sarute et al., 2019).3.2.Behavioral observationsTrim2A/A mice were distinguishable from their heterozygous (Trim2A/+) and WT (Trim2+/+) littermates within a few weeks after birth, because they were ataxic (Supplementary Movie 1). We analyzed 6 to 10 mice per group for ataxia at 2, 4 and 6 months of age. Trim2A/A mice showed significantly higher scores on an ataxia test that includes ledge tests, gait, kyphosis, and hindlimb clasping (Guyenet et al., 2010), and their scores worsened with age (Fig. 1). Female Trim2A/A mice were fertile, but males were unable to sire offspring, likely due to ataxia, so that heterozygous males and females were used for breeding. In contrast, Trim2C/C mice, like WT mice, had no visible phenotype (Supplementary Movies 2 and 3).3.3.Pathological findings in the CNSBalastik et al. (2008) previously reported a gene trap line in which lacZ replaces Trim2. Trim2lacZ/lacZ mice develop ataxia and have axonal spheroids in multiple sites in the CNS, including the deep cerebellar nuclei, as well as a profound loss of Purkinje cells. In paraffin sections, we found reduced numbers of Purkinje cells in the Trim2A/A mice by immunostaining for TRIM2 or calbindin, a Purkinje cell maker, and quantified Purkinje cells (Fig. 2). To further investigate these findings, we examined semi-thin and thin sections of the cerebellum and cervical spinal cord in Trim2A/A mice. As shown in Fig. 2C-F, Purkinje cells were largely gone by 8 weeks of age. In addition, there were numerous axonal spheroids in the cerebellar nuclei and throughout the cervical spinal cord, especially in the gray matter (Fig. 2A&B). Each spheroid was surrounded by a myelin sheath, demonstrating that they were focal swellings within myelinated axons. EM showed that the spheroids contained numerous mitochondria and aggregates of neurofilaments (Fig. 3C&D), similar to what was described in the cerebellum of Trim2lacZ/lacZ mice (Balastik et al., 2008). In the cerebella of Trim2C/C mice, in contrast, there was no loss of Purkinje cells (Fig. 4) and axonal spheroids were not found (data not shown).3.4.Trim2A/A mice have an axonal neuropathyBecause humans with recessive TRIM2 mutations have peripheral neuropathy (Pehlivan et al., 2015; Ylikallio et al., 2013), we wished to determine whether Trim2A/A and Trim2C/C mice have a peripheral neuropathy. We measured the CAPs of the tail, a mixed nerve response from the bilateral dorsal and ventral caudal nerves, which are mainly comprised of myelinated and unmyelinated sensory axons (Maia et al., 2010; Miyoshi and Goto, 1973; Schmelzer and Low, 1987). Compared to WT/Trim2+/+ littermates, the CAP amplitudes for Trim2A/A mice were much smaller at 3, 8, and 26 weeks, with a moderately decreased conduction velocity (Fig. 5A). The CAPs in Trim2C/C mice, in contrast, had normal amplitudes and conduction velocities.To determine the anatomical correlates of these electrophysiological abnormalities, we examined semi-thin and thin sections of the ventral caudal nerves at the location where we stimulated the tail. As shown in Fig. 5B, the ventral caudal nerves of Trim2A/A mice contained few myelinated axons, even at 3 weeks of age. EM of the 8-week-old ventral caudal nerves from Trim2A/A mice confirmed the profound lack of myelinated axons and revealed abundant unmyelinated axons, a few myelinated axons at different stages of degeneration (Schwann cells containing myelin debris or lipid droplets), as well as enlarged, unmyelinated axons that contained excessive numbers of mitochondria (Fig. 6). The caudal nerves of Trim2C/C mice, in contrast, looked normal (Fig. 4).Because most neuropathies are more severe in longer nerves, we also examined the femoral nerve, which contains axons that are shorter than those in the caudal nerve, as well as the ventral and dorsal roots, whose axons are even more proximal to their cell bodies of origin. At 8 weeks, the femoral motor and sensory nerves of Trim2A/A mice appeared normal. At 26 weeks, there were a few degenerating myelinated axons in the femoral motor (Fig. 7) but not in the femoral sensory nerves (Supplemental Fig. 2). Except for an occasional degenerating myelinated axon, the ventral roots were normal (Fig. 8F). We did not find pathological changes in the sensory neurons in the DRG or in their axons in the dorsal roots (Fig. 8E&G). To compare the myelin thickness between WT and Trim2A/A mice, we measured the g-ratios of myelinated axons from 26-week-old mice (3 mice each); these were not significant different (Fig. 7E&F).These data, taken together, show that the longest myelinated axons in the peripheral nerves of Trim2A/A mice are more affected: the distal caudal nerves were affected even by 8 weeks of age, the femoral motor nerves were affected by 26 weeks, and the dorsal and ventral roots were relatively unaffected even at 26 weeks. Because inherited axonal neuropathies are typically the result of cell autonomous effects of mutations, we immunostained sections of the cervical spinal cord and DRG, where the primary motor and sensory neurons are respectively localized. In WT mice, both motor and sensory neurons strongly express TRIM2, whereas TRIM2-immunoreactivity was absent in Trim2A/A mice (Fig. 8A-D). We hypothesize that the lack of TRIM2 in these neurons makes their most distal axons vulnerable to degeneration.3.5.Loss of TRIM2 does not affect NFL levelsTRIM2 binds to NFL (Balastik et al., 2008; Sarute et al., 2019), and the NFL that accumulates in Purkinje cell axons in Trim2lacZ/lacZ mice was suggested to contribute to their degeneration (Balastik et al.,2008). We examined this issue by examining the levels of markers for different cell types – glial fibrillary acidic protein (GFAP) and S100β (astrocytes), IBA1 and P2RY12 (microglia), and NeuN and NFL (neurons) - in the brains of 8–12 week-old WT, Trim2A/A, and Trim2C/C mice. No differences were seen in the levels of NFL in Trim2A/A brains compared to either WT or Trim2C/C mice (Fig. 9A), or in the localization of NFL in neurons in the DRG and cerebella of Trim2A/A mice (Supplemental Fig. 3). IBA1 was the only marker that was consistently higher in the Trim2A/A mice. Thus, the pathological, focal accumulations of NFL in CNS axons in Trim2lacZ/lacZ and Trim2A/A mutant mice is not associated with a generalized accumulation of NFL. TRIM2 also co-localized with signal regulatory protein alpha (SIRPA) protein within neurons in both DRG and cerebellum from WT mice, and SIRPA localization did not change in the absence of TRIM2 in Trim2A/A mice (Supplemental Fig. 3).The lack of a phenotype in Trim2C/C mice was surprising, as the Trim2C allele deletes the C terminus of the RING domain, which is a key domain for TRIM2 ubiquitin ligase activity (Balastik et al., 2008). To determine whether the TRIM2 C protein (encoded by the Trim2C allele), which lacks part of the RING domain, results in an ubiquitin ligase-dead protein, we co-transfected cells to express either WT or TRIM2 C along with NFL and hemagglutinin (HA)-tagged ubiquitin, and immunoprecipitated HA-tagged proteins followed by western blotting for TRIM2 and NFL. As shown in Fig. 9B, WT TRIM2 resulted in extensive auto-ubiquitination, as we and others reported previously (Balastik et al., 2008; Sarute et al., 2019). In contrast, the TRIM2 C protein resulted in much lower levels of auto-ubiquitination, and low levels of ubiquitination of endogenous TRIM2 was also detected in the absence of transfected TRIM2 (upper right panel). Whereas the overexpression of WT TRIM2 resulted in a slight increase in ubiquitinated NFL, the over-expression of TRIM2 C did not increase ubiquitination of NFL beyond what was seen 293 T cells that were not transfected to express TRIM2 (compare the lanes 1–3 to lane 4 in the bottom right panel); there may be an endogenous ubiquitin ligase that modifies NFL in 293 T cells. Thus, WT TRIM2 may ubiquitinate NFL but the absence of TRIM2 does not affect the overall levels of NFL in brain. Although the TRIM2 C mutant protein is deficient in ubiquitin ligase activity, it produces in a protein that rescues the functional role(s) of TRIM2 in the CNS and PNS.

3.1.Generating Trim2 mutant miceWe previously described and partially characterized mice with targeted deletions in Trim2 using CRISPR-Cas9 and guide RNAs to exons 3 and 9, generating an allele, Trim2A, that is predicted to truncate TRIM2 after the RING domain (Supplemental Fig. 1); this allele does not produce a stable protein in transfected cells (Sarute et al., 2019). This targeting also produced a second strain of mice, Trim2C, with a deletion of the 3′ of exon 3, resulting in a stable protein that lacked 30 amino acids, including the C-terminal portion of the RING domain that is required for ubiquitin ligase activity (Sarute et al., 2019).

Generating Trim2 mutant mice

We previously described and partially characterized mice with targeted deletions in Trim2 using CRISPR-Cas9 and guide RNAs to exons 3 and 9, generating an allele, Trim2A, that is predicted to truncate TRIM2 after the RING domain (Supplemental Fig. 1); this allele does not produce a stable protein in transfected cells (Sarute et al., 2019). This targeting also produced a second strain of mice, Trim2C, with a deletion of the 3′ of exon 3, resulting in a stable protein that lacked 30 amino acids, including the C-terminal portion of the RING domain that is required for ubiquitin ligase activity (Sarute et al., 2019).

3.2.Behavioral observationsTrim2A/A mice were distinguishable from their heterozygous (Trim2A/+) and WT (Trim2+/+) littermates within a few weeks after birth, because they were ataxic (Supplementary Movie 1). We analyzed 6 to 10 mice per group for ataxia at 2, 4 and 6 months of age. Trim2A/A mice showed significantly higher scores on an ataxia test that includes ledge tests, gait, kyphosis, and hindlimb clasping (Guyenet et al., 2010), and their scores worsened with age (Fig. 1). Female Trim2A/A mice were fertile, but males were unable to sire offspring, likely due to ataxia, so that heterozygous males and females were used for breeding. In contrast, Trim2C/C mice, like WT mice, had no visible phenotype (Supplementary Movies 2 and 3).

Behavioral observations

Trim2A/A mice were distinguishable from their heterozygous (Trim2A/+) and WT (Trim2+/+) littermates within a few weeks after birth, because they were ataxic (Supplementary Movie 1). We analyzed 6 to 10 mice per group for ataxia at 2, 4 and 6 months of age. Trim2A/A mice showed significantly higher scores on an ataxia test that includes ledge tests, gait, kyphosis, and hindlimb clasping (Guyenet et al., 2010), and their scores worsened with age (Fig. 1). Female Trim2A/A mice were fertile, but males were unable to sire offspring, likely due to ataxia, so that heterozygous males and females were used for breeding. In contrast, Trim2C/C mice, like WT mice, had no visible phenotype (Supplementary Movies 2 and 3).

3.3.Pathological findings in the CNSBalastik et al. (2008) previously reported a gene trap line in which lacZ replaces Trim2. Trim2lacZ/lacZ mice develop ataxia and have axonal spheroids in multiple sites in the CNS, including the deep cerebellar nuclei, as well as a profound loss of Purkinje cells. In paraffin sections, we found reduced numbers of Purkinje cells in the Trim2A/A mice by immunostaining for TRIM2 or calbindin, a Purkinje cell maker, and quantified Purkinje cells (Fig. 2). To further investigate these findings, we examined semi-thin and thin sections of the cerebellum and cervical spinal cord in Trim2A/A mice. As shown in Fig. 2C-F, Purkinje cells were largely gone by 8 weeks of age. In addition, there were numerous axonal spheroids in the cerebellar nuclei and throughout the cervical spinal cord, especially in the gray matter (Fig. 2A&B). Each spheroid was surrounded by a myelin sheath, demonstrating that they were focal swellings within myelinated axons. EM showed that the spheroids contained numerous mitochondria and aggregates of neurofilaments (Fig. 3C&D), similar to what was described in the cerebellum of Trim2lacZ/lacZ mice (Balastik et al., 2008). In the cerebella of Trim2C/C mice, in contrast, there was no loss of Purkinje cells (Fig. 4) and axonal spheroids were not found (data not shown).

Pathological findings in the CNS

Balastik et al. (2008) previously reported a gene trap line in which lacZ replaces Trim2. Trim2lacZ/lacZ mice develop ataxia and have axonal spheroids in multiple sites in the CNS, including the deep cerebellar nuclei, as well as a profound loss of Purkinje cells. In paraffin sections, we found reduced numbers of Purkinje cells in the Trim2A/A mice by immunostaining for TRIM2 or calbindin, a Purkinje cell maker, and quantified Purkinje cells (Fig. 2). To further investigate these findings, we examined semi-thin and thin sections of the cerebellum and cervical spinal cord in Trim2A/A mice. As shown in Fig. 2C-F, Purkinje cells were largely gone by 8 weeks of age. In addition, there were numerous axonal spheroids in the cerebellar nuclei and throughout the cervical spinal cord, especially in the gray matter (Fig. 2A&B). Each spheroid was surrounded by a myelin sheath, demonstrating that they were focal swellings within myelinated axons. EM showed that the spheroids contained numerous mitochondria and aggregates of neurofilaments (Fig. 3C&D), similar to what was described in the cerebellum of Trim2lacZ/lacZ mice (Balastik et al., 2008). In the cerebella of Trim2C/C mice, in contrast, there was no loss of Purkinje cells (Fig. 4) and axonal spheroids were not found (data not shown).

3.4.Trim2A/A mice have an axonal neuropathyBecause humans with recessive TRIM2 mutations have peripheral neuropathy (Pehlivan et al., 2015; Ylikallio et al., 2013), we wished to determine whether Trim2A/A and Trim2C/C mice have a peripheral neuropathy. We measured the CAPs of the tail, a mixed nerve response from the bilateral dorsal and ventral caudal nerves, which are mainly comprised of myelinated and unmyelinated sensory axons (Maia et al., 2010; Miyoshi and Goto, 1973; Schmelzer and Low, 1987). Compared to WT/Trim2+/+ littermates, the CAP amplitudes for Trim2A/A mice were much smaller at 3, 8, and 26 weeks, with a moderately decreased conduction velocity (Fig. 5A). The CAPs in Trim2C/C mice, in contrast, had normal amplitudes and conduction velocities.To determine the anatomical correlates of these electrophysiological abnormalities, we examined semi-thin and thin sections of the ventral caudal nerves at the location where we stimulated the tail. As shown in Fig. 5B, the ventral caudal nerves of Trim2A/A mice contained few myelinated axons, even at 3 weeks of age. EM of the 8-week-old ventral caudal nerves from Trim2A/A mice confirmed the profound lack of myelinated axons and revealed abundant unmyelinated axons, a few myelinated axons at different stages of degeneration (Schwann cells containing myelin debris or lipid droplets), as well as enlarged, unmyelinated axons that contained excessive numbers of mitochondria (Fig. 6). The caudal nerves of Trim2C/C mice, in contrast, looked normal (Fig. 4).Because most neuropathies are more severe in longer nerves, we also examined the femoral nerve, which contains axons that are shorter than those in the caudal nerve, as well as the ventral and dorsal roots, whose axons are even more proximal to their cell bodies of origin. At 8 weeks, the femoral motor and sensory nerves of Trim2A/A mice appeared normal. At 26 weeks, there were a few degenerating myelinated axons in the femoral motor (Fig. 7) but not in the femoral sensory nerves (Supplemental Fig. 2). Except for an occasional degenerating myelinated axon, the ventral roots were normal (Fig. 8F). We did not find pathological changes in the sensory neurons in the DRG or in their axons in the dorsal roots (Fig. 8E&G). To compare the myelin thickness between WT and Trim2A/A mice, we measured the g-ratios of myelinated axons from 26-week-old mice (3 mice each); these were not significant different (Fig. 7E&F).These data, taken together, show that the longest myelinated axons in the peripheral nerves of Trim2A/A mice are more affected: the distal caudal nerves were affected even by 8 weeks of age, the femoral motor nerves were affected by 26 weeks, and the dorsal and ventral roots were relatively unaffected even at 26 weeks. Because inherited axonal neuropathies are typically the result of cell autonomous effects of mutations, we immunostained sections of the cervical spinal cord and DRG, where the primary motor and sensory neurons are respectively localized. In WT mice, both motor and sensory neurons strongly express TRIM2, whereas TRIM2-immunoreactivity was absent in Trim2A/A mice (Fig. 8A-D). We hypothesize that the lack of TRIM2 in these neurons makes their most distal axons vulnerable to degeneration.

Trim2A/A mice have an axonal neuropathy

Because humans with recessive TRIM2 mutations have peripheral neuropathy (Pehlivan et al., 2015; Ylikallio et al., 2013), we wished to determine whether Trim2A/A and Trim2C/C mice have a peripheral neuropathy. We measured the CAPs of the tail, a mixed nerve response from the bilateral dorsal and ventral caudal nerves, which are mainly comprised of myelinated and unmyelinated sensory axons (Maia et al., 2010; Miyoshi and Goto, 1973; Schmelzer and Low, 1987). Compared to WT/Trim2+/+ littermates, the CAP amplitudes for Trim2A/A mice were much smaller at 3, 8, and 26 weeks, with a moderately decreased conduction velocity (Fig. 5A). The CAPs in Trim2C/C mice, in contrast, had normal amplitudes and conduction velocities.

To determine the anatomical correlates of these electrophysiological abnormalities, we examined semi-thin and thin sections of the ventral caudal nerves at the location where we stimulated the tail. As shown in Fig. 5B, the ventral caudal nerves of Trim2A/A mice contained few myelinated axons, even at 3 weeks of age. EM of the 8-week-old ventral caudal nerves from Trim2A/A mice confirmed the profound lack of myelinated axons and revealed abundant unmyelinated axons, a few myelinated axons at different stages of degeneration (Schwann cells containing myelin debris or lipid droplets), as well as enlarged, unmyelinated axons that contained excessive numbers of mitochondria (Fig. 6). The caudal nerves of Trim2C/C mice, in contrast, looked normal (Fig. 4).

Because most neuropathies are more severe in longer nerves, we also examined the femoral nerve, which contains axons that are shorter than those in the caudal nerve, as well as the ventral and dorsal roots, whose axons are even more proximal to their cell bodies of origin. At 8 weeks, the femoral motor and sensory nerves of Trim2A/A mice appeared normal. At 26 weeks, there were a few degenerating myelinated axons in the femoral motor (Fig. 7) but not in the femoral sensory nerves (Supplemental Fig. 2). Except for an occasional degenerating myelinated axon, the ventral roots were normal (Fig. 8F). We did not find pathological changes in the sensory neurons in the DRG or in their axons in the dorsal roots (Fig. 8E&G). To compare the myelin thickness between WT and Trim2A/A mice, we measured the g-ratios of myelinated axons from 26-week-old mice (3 mice each); these were not significant different (Fig. 7E&F).

These data, taken together, show that the longest myelinated axons in the peripheral nerves of Trim2A/A mice are more affected: the distal caudal nerves were affected even by 8 weeks of age, the femoral motor nerves were affected by 26 weeks, and the dorsal and ventral roots were relatively unaffected even at 26 weeks. Because inherited axonal neuropathies are typically the result of cell autonomous effects of mutations, we immunostained sections of the cervical spinal cord and DRG, where the primary motor and sensory neurons are respectively localized. In WT mice, both motor and sensory neurons strongly express TRIM2, whereas TRIM2-immunoreactivity was absent in Trim2A/A mice (Fig. 8A-D). We hypothesize that the lack of TRIM2 in these neurons makes their most distal axons vulnerable to degeneration.

3.5.Loss of TRIM2 does not affect NFL levelsTRIM2 binds to NFL (Balastik et al., 2008; Sarute et al., 2019), and the NFL that accumulates in Purkinje cell axons in Trim2lacZ/lacZ mice was suggested to contribute to their degeneration (Balastik et al.,2008). We examined this issue by examining the levels of markers for different cell types – glial fibrillary acidic protein (GFAP) and S100β (astrocytes), IBA1 and P2RY12 (microglia), and NeuN and NFL (neurons) - in the brains of 8–12 week-old WT, Trim2A/A, and Trim2C/C mice. No differences were seen in the levels of NFL in Trim2A/A brains compared to either WT or Trim2C/C mice (Fig. 9A), or in the localization of NFL in neurons in the DRG and cerebella of Trim2A/A mice (Supplemental Fig. 3). IBA1 was the only marker that was consistently higher in the Trim2A/A mice. Thus, the pathological, focal accumulations of NFL in CNS axons in Trim2lacZ/lacZ and Trim2A/A mutant mice is not associated with a generalized accumulation of NFL. TRIM2 also co-localized with signal regulatory protein alpha (SIRPA) protein within neurons in both DRG and cerebellum from WT mice, and SIRPA localization did not change in the absence of TRIM2 in Trim2A/A mice (Supplemental Fig. 3).The lack of a phenotype in Trim2C/C mice was surprising, as the Trim2C allele deletes the C terminus of the RING domain, which is a key domain for TRIM2 ubiquitin ligase activity (Balastik et al., 2008). To determine whether the TRIM2 C protein (encoded by the Trim2C allele), which lacks part of the RING domain, results in an ubiquitin ligase-dead protein, we co-transfected cells to express either WT or TRIM2 C along with NFL and hemagglutinin (HA)-tagged ubiquitin, and immunoprecipitated HA-tagged proteins followed by western blotting for TRIM2 and NFL. As shown in Fig. 9B, WT TRIM2 resulted in extensive auto-ubiquitination, as we and others reported previously (Balastik et al., 2008; Sarute et al., 2019). In contrast, the TRIM2 C protein resulted in much lower levels of auto-ubiquitination, and low levels of ubiquitination of endogenous TRIM2 was also detected in the absence of transfected TRIM2 (upper right panel). Whereas the overexpression of WT TRIM2 resulted in a slight increase in ubiquitinated NFL, the over-expression of TRIM2 C did not increase ubiquitination of NFL beyond what was seen 293 T cells that were not transfected to express TRIM2 (compare the lanes 1–3 to lane 4 in the bottom right panel); there may be an endogenous ubiquitin ligase that modifies NFL in 293 T cells. Thus, WT TRIM2 may ubiquitinate NFL but the absence of TRIM2 does not affect the overall levels of NFL in brain. Although the TRIM2 C mutant protein is deficient in ubiquitin ligase activity, it produces in a protein that rescues the functional role(s) of TRIM2 in the CNS and PNS.

Loss of TRIM2 does not affect NFL levels

TRIM2 binds to NFL (Balastik et al., 2008; Sarute et al., 2019), and the NFL that accumulates in Purkinje cell axons in Trim2lacZ/lacZ mice was suggested to contribute to their degeneration (Balastik et al.,2008). We examined this issue by examining the levels of markers for different cell types – glial fibrillary acidic protein (GFAP) and S100β (astrocytes), IBA1 and P2RY12 (microglia), and NeuN and NFL (neurons) - in the brains of 8–12 week-old WT, Trim2A/A, and Trim2C/C mice. No differences were seen in the levels of NFL in Trim2A/A brains compared to either WT or Trim2C/C mice (Fig. 9A), or in the localization of NFL in neurons in the DRG and cerebella of Trim2A/A mice (Supplemental Fig. 3). IBA1 was the only marker that was consistently higher in the Trim2A/A mice. Thus, the pathological, focal accumulations of NFL in CNS axons in Trim2lacZ/lacZ and Trim2A/A mutant mice is not associated with a generalized accumulation of NFL. TRIM2 also co-localized with signal regulatory protein alpha (SIRPA) protein within neurons in both DRG and cerebellum from WT mice, and SIRPA localization did not change in the absence of TRIM2 in Trim2A/A mice (Supplemental Fig. 3).

The lack of a phenotype in Trim2C/C mice was surprising, as the Trim2C allele deletes the C terminus of the RING domain, which is a key domain for TRIM2 ubiquitin ligase activity (Balastik et al., 2008). To determine whether the TRIM2 C protein (encoded by the Trim2C allele), which lacks part of the RING domain, results in an ubiquitin ligase-dead protein, we co-transfected cells to express either WT or TRIM2 C along with NFL and hemagglutinin (HA)-tagged ubiquitin, and immunoprecipitated HA-tagged proteins followed by western blotting for TRIM2 and NFL. As shown in Fig. 9B, WT TRIM2 resulted in extensive auto-ubiquitination, as we and others reported previously (Balastik et al., 2008; Sarute et al., 2019). In contrast, the TRIM2 C protein resulted in much lower levels of auto-ubiquitination, and low levels of ubiquitination of endogenous TRIM2 was also detected in the absence of transfected TRIM2 (upper right panel). Whereas the overexpression of WT TRIM2 resulted in a slight increase in ubiquitinated NFL, the over-expression of TRIM2 C did not increase ubiquitination of NFL beyond what was seen 293 T cells that were not transfected to express TRIM2 (compare the lanes 1–3 to lane 4 in the bottom right panel); there may be an endogenous ubiquitin ligase that modifies NFL in 293 T cells. Thus, WT TRIM2 may ubiquitinate NFL but the absence of TRIM2 does not affect the overall levels of NFL in brain. Although the TRIM2 C mutant protein is deficient in ubiquitin ligase activity, it produces in a protein that rescues the functional role(s) of TRIM2 in the CNS and PNS.

4.DiscussionWe found that Trim2A/A mice develop an axonopathy in both the CNS and the PNS. In the CNS, this axonopathy is characterized by axonal spheroids within myelinated axons as previously described in the Purkinje cells of Trim2lacZ/lacZ mice (Balastik et al., 2008). We show that these axonal spheroids are not restricted to Purkinje cells, and that the myelinated axons in the peripheral nerves degenerate in a distally predominant pattern, even though axonal spheroids were not prominent. This neuropathy was not reported in Trim2lacZ/lacZ mice (Balastik et al., 2008), likely because that report predates the discovery that recessive mutations in TRIM2 cause an inherited axonal neuropathy in humans (Pehlivan et al., 2015; Ylikallio et al., 2013). The quantified electrophysiological and anatomical alterations we provide will enable potential therapies for CMT2R to be rigorously evaluated.Our data inform these clinical reports. The first report was an 18-year-old woman who harbored compound heterozygous TRIM2 mutation (c.680A > T/p.Glu227Val and c.1966delA/p.Lly567fs; (Ylikallio et al., 2013)). She had motor difficulties in infancy, and walked with a broad base, but overt cerebellar findings were not described. Her clinical electrophysiological findings at age 4 years were a motor greater than sensory neuropathy, and a sural nerve biopsy at age 5 years showed a diminished density of myelinated axons, but no actively degenerating axons. A homozygous c.2000A > C/p.Asp667Ala mutation was found in an infant with born to consanguineous parents (Pehlivan et al., 2015). The infant presented at age 6 months with respiratory insufficiency caused by bilateral vocal cord paralysis, and subsequently developed global weakness, and died before reaching 3 years of age. Clinical neurophysiology showed a markedly reduced amplitude of the median nerve compound muscle action potential. Thus, humans with recessive TRIM2 mutations have not been reported to have ataxia, as seen in Trim2lacZ/lacZ and Trim2A/A mice, but do manifest an axonal neuropathy. We speculate that there is a more widespread pathology of CNS axons in these affected patients, as we document in Trim2A/A mice and was previously found in Trim2lacZ/lacZ mice (Balastik et al., 2008), and that this contributes to the clinical phenotype of these patients. We suspect that these pathologies owe to cell autonomous effect of recessive TRIM2 mutation in neurons, as TRIM2 mRNA is expressed at much higher levels in the human CNS than in other tissues, including tibial nerve (https://gtexportal.org/home/transcriptPage), and we found no evidence of de/remyelination of PNS axons.It was previously reported that TRIM2 ubiquitinates NFL and the absence of TRIM2 (in Trim2lacZ/lacZ mice), was postulated to cause the focal build-up of NFL in axons as well as neuronal loss (Balastik et al., 2008). We did not find a global increase of NFL levels in the brains of Trim2A/A mice, and we did not confirm that TRIM2 (WT or TRIM C) ubiquitinates NFL. These results suggest that the lack of degradation of neurofilaments by mutant TRIM2 is not the cause of the Purkinje cells loss, CNS axonal spheroids, or the neuropathy in Trim2A/A and Trim2lacZ/lacZ mice. This conclusion contrasts with the effects of recessive mutations in the GAN gene, which result in the failure to degrade intermediate filaments, including neurofilaments (Mahammad et al., 2013). Individuals with recessive GAN mutations have a progressive neurodegenerative disorder that affects both PNS and CNS axons, with massive accumulations of neurofilaments (Johnson-Kerner et al., 2014).If the loss of ubiquitination activity is not the cause of the neurological phenotype, then another functional role of TRIM2 is disrupted in Trim2A/A and Trim2lacZ/lacZ mice, but not in Trim2C/C mice. It is not obvious what that functional role would be. In addition to NFL, TRIM2 interacts with several other cellular proteins, including Bcl-interacting mediator of cell death (BIM/BCL2l11), myosin5A, and SIRPA (Ohkawa et al., 2001; Rolland et al., 2014; Rual et al., 2005; Thompson et al., 2011). Myosin5A is involved in axonal transport, and interacts with actin and NFL (Bridgman, 1999). Increasing or decreasing the amount of NFL alters the transport of myosin5A in axons, and conversely, the loss of myosin5A alters the distribution and local transport of neurofilaments (Alami et al., 2009; Rao et al., 2002). SIRPA is a transmembrane glycoprotein that is expressed by macrophages and neurons (Matozaki et al., 2009). The binding of SIRPA to CD47 on the surface of target cells inhibits their engulfment by macrophages. Different neuronal population express SIRPA and CD47, but the role of this interaction is unknown. We recently showed a SIRPA-TRIM2 interaction inhibits infection by NWA and macrophage phagocytosis (Sarute et al., 2019). Current work is aimed at determining whether other members of the TRIM2 interactome, such as SIRPA, are involved in the axonopathy that occurs in TRIM2-deficient mice and humans.

We found that Trim2A/A mice develop an axonopathy in both the CNS and the PNS. In the CNS, this axonopathy is characterized by axonal spheroids within myelinated axons as previously described in the Purkinje cells of Trim2lacZ/lacZ mice (Balastik et al., 2008). We show that these axonal spheroids are not restricted to Purkinje cells, and that the myelinated axons in the peripheral nerves degenerate in a distally predominant pattern, even though axonal spheroids were not prominent. This neuropathy was not reported in Trim2lacZ/lacZ mice (Balastik et al., 2008), likely because that report predates the discovery that recessive mutations in TRIM2 cause an inherited axonal neuropathy in humans (Pehlivan et al., 2015; Ylikallio et al., 2013). The quantified electrophysiological and anatomical alterations we provide will enable potential therapies for CMT2R to be rigorously evaluated.

Our data inform these clinical reports. The first report was an 18-year-old woman who harbored compound heterozygous TRIM2 mutation (c.680A > T/p.Glu227Val and c.1966delA/p.Lly567fs; (Ylikallio et al., 2013)). She had motor difficulties in infancy, and walked with a broad base, but overt cerebellar findings were not described. Her clinical electrophysiological findings at age 4 years were a motor greater than sensory neuropathy, and a sural nerve biopsy at age 5 years showed a diminished density of myelinated axons, but no actively degenerating axons. A homozygous c.2000A > C/p.Asp667Ala mutation was found in an infant with born to consanguineous parents (Pehlivan et al., 2015). The infant presented at age 6 months with respiratory insufficiency caused by bilateral vocal cord paralysis, and subsequently developed global weakness, and died before reaching 3 years of age. Clinical neurophysiology showed a markedly reduced amplitude of the median nerve compound muscle action potential. Thus, humans with recessive TRIM2 mutations have not been reported to have ataxia, as seen in Trim2lacZ/lacZ and Trim2A/A mice, but do manifest an axonal neuropathy. We speculate that there is a more widespread pathology of CNS axons in these affected patients, as we document in Trim2A/A mice and was previously found in Trim2lacZ/lacZ mice (Balastik et al., 2008), and that this contributes to the clinical phenotype of these patients. We suspect that these pathologies owe to cell autonomous effect of recessive TRIM2 mutation in neurons, as TRIM2 mRNA is expressed at much higher levels in the human CNS than in other tissues, including tibial nerve (https://gtexportal.org/home/transcriptPage), and we found no evidence of de/remyelination of PNS axons.

It was previously reported that TRIM2 ubiquitinates NFL and the absence of TRIM2 (in Trim2lacZ/lacZ mice), was postulated to cause the focal build-up of NFL in axons as well as neuronal loss (Balastik et al., 2008). We did not find a global increase of NFL levels in the brains of Trim2A/A mice, and we did not confirm that TRIM2 (WT or TRIM C) ubiquitinates NFL. These results suggest that the lack of degradation of neurofilaments by mutant TRIM2 is not the cause of the Purkinje cells loss, CNS axonal spheroids, or the neuropathy in Trim2A/A and Trim2lacZ/lacZ mice. This conclusion contrasts with the effects of recessive mutations in the GAN gene, which result in the failure to degrade intermediate filaments, including neurofilaments (Mahammad et al., 2013). Individuals with recessive GAN mutations have a progressive neurodegenerative disorder that affects both PNS and CNS axons, with massive accumulations of neurofilaments (Johnson-Kerner et al., 2014).

If the loss of ubiquitination activity is not the cause of the neurological phenotype, then another functional role of TRIM2 is disrupted in Trim2A/A and Trim2lacZ/lacZ mice, but not in Trim2C/C mice. It is not obvious what that functional role would be. In addition to NFL, TRIM2 interacts with several other cellular proteins, including Bcl-interacting mediator of cell death (BIM/BCL2l11), myosin5A, and SIRPA (Ohkawa et al., 2001; Rolland et al., 2014; Rual et al., 2005; Thompson et al., 2011). Myosin5A is involved in axonal transport, and interacts with actin and NFL (Bridgman, 1999). Increasing or decreasing the amount of NFL alters the transport of myosin5A in axons, and conversely, the loss of myosin5A alters the distribution and local transport of neurofilaments (Alami et al., 2009; Rao et al., 2002). SIRPA is a transmembrane glycoprotein that is expressed by macrophages and neurons (Matozaki et al., 2009). The binding of SIRPA to CD47 on the surface of target cells inhibits their engulfment by macrophages. Different neuronal population express SIRPA and CD47, but the role of this interaction is unknown. We recently showed a SIRPA-TRIM2 interaction inhibits infection by NWA and macrophage phagocytosis (Sarute et al., 2019). Current work is aimed at determining whether other members of the TRIM2 interactome, such as SIRPA, are involved in the axonopathy that occurs in TRIM2-deficient mice and humans.

Supplementary MaterialSupplementary movie 1Supplementary movie 2Supplementary movie 3Supplementary figuresSupplementary data to this article can be found online at https://doi.org/10.1016/j.nbd.2020.104845.

Supplementary Material

Supplementary data to this article can be found online at https://doi.org/10.1016/j.nbd.2020.104845.
