---
pmid: '25046117'
title: Reciprocal conversion of Gtr1 and Gtr2 nucleotide-binding states by Npr2-Npr3
  inactivates TORC1 and induces autophagy.
authors:
- Kira S
- Tabata K
- Shirahama-Noda K
- Nozoe A
- Yoshimori T
- Noda T
journal: Autophagy
year: '2014'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC4206535
doi: 10.4161/auto.29397
---

# Reciprocal conversion of Gtr1 and Gtr2 nucleotide-binding states by Npr2-Npr3 inactivates TORC1 and induces autophagy.
**Authors:** Kira S, Tabata K, Shirahama-Noda K, Nozoe A, Yoshimori T, Noda T
**Journal:** Autophagy (2014)
**DOI:** [10.4161/auto.29397](https://doi.org/10.4161/auto.29397)
**PMC:** [PMC4206535](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4206535/)

## Abstract

1. Autophagy. 2014 Sep;10(9):1565-78. doi: 10.4161/auto.29397. Epub 2014 Jun 30.

Reciprocal conversion of Gtr1 and Gtr2 nucleotide-binding states by Npr2-Npr3 
inactivates TORC1 and induces autophagy.

Kira S(1), Tabata K(2), Shirahama-Noda K(3), Nozoe A(4), Yoshimori T(5), Noda 
T(1).

Author information:
(1)Center for Frontier Oral Science; Graduate School of Dentistry; Osaka 
University, Osaka, Japan; Graduate School of Frontier Bioscience; Osaka 
University; Osaka, Japan.
(2)Laboratory of Viral Infection; International Research Center for Infectious 
Diseases; Research Institute for Microbial Diseases; Osaka University; Osaka, 
Japan.
(3)Center for Frontier Oral Science; Graduate School of Dentistry; Osaka 
University, Osaka, Japan.
(4)Graduate School of Medicine, Osaka University; Osaka, Japan.
(5)Graduate School of Frontier Bioscience; Osaka University; Osaka, Japan; 
Graduate School of Medicine, Osaka University; Osaka, Japan.

Autophagy is an intracellular degradation process that delivers cytosolic 
material to lysosomes and vacuoles. To investigate the mechanisms that regulate 
autophagy, we performed a genome-wide screen using a yeast deletion-mutant 
collection, and found that Npr2 and Npr3 mutants were defective in autophagy. 
Their mammalian homologs, NPRL2 and NPRL3, were also involved in regulation of 
autophagy. Npr2-Npr3 function upstream of Gtr1-Gtr2, homologs of the mammalian 
RRAG GTPase complex, which is crucial for TORC1 regulation. Both npr2∆ mutants 
and a GTP-bound Gtr1 mutant suppressed autophagy and increased Tor1 vacuole 
localization. Furthermore, Gtr2 binds to the TORC1 subunit Kog1. A GDP-bound 
Gtr1 mutant induced autophagy even under nutrient-rich conditions, and this 
effect was dependent on the direct binding of Gtr2 to Kog1. These results 
revealed that 2 molecular mechanisms, Npr2-Npr3-dependent GTP hydrolysis of Gtr1 
and direct binding of Gtr2 to Kog1, are involved in TORC1 inactivation and 
autophagic induction.

DOI: 10.4161/auto.29397
PMCID: PMC4206535
PMID: 25046117 [Indexed for MEDLINE]

## Full Text

IntroductionAutophagy, an intracellular process that mediates the bulk degradation of cellular
components, is conserved from yeast to mammals.1 This process is induced under stressful conditions, such as nutrient
starvation. Autophagy is characterized by cytosolic membranous structures called
autophagosomes, and the delivery of cytosolic materials to lysosomes and vacuoles for
degradation. By supplying the products of degradation as resources for the cell, autophagy
helps cells adapt to stressful conditions.The TORC1 protein kinase complex, which in the yeast Saccharomyces
cerevisiae consists of the Tor1/2 protein kinase, Kog1, Lst8, and Tco89, plays a
central role in the regulation of autophagy, which is induced by TORC1 inactivation.2,3 TORC1 contributes to cell growth and homeostasis in response to
environmental cues. Various factors, including nutrients, stress, and intracellular energy
levels, affect the kinase activity of TORC1; TORC1 is activated under conditions favorable
to cellular growth, whereas stressful conditions, including nutritional limitation,
inactivate TORC1.4 In yeast autophagy,
inactivation of TORC1 leads to dephosphorylation of Atg13, a substrate of TORC1, and
induction of autophagy.5The heterodimeric G proteins Gtr1 and Gtr2 play a critical role in the regulation of TORC1
in yeast.6 In mammals, RRAGA and RRAGB are
orthologs of Gtr1, and RRAGC and RRAGD are orthologs of Gtr2.7,8
Together with Ego1 and Ego3, Gtr1 and Gtr2 are components of the EGO complex;9,10 Ego1 and Ego3 are the putative functional counterparts of the
mammalian Ragulator complex.10-12The nucleotide-binding state of Gtr1-Gtr2 is crucial for the regulation of TORC1. GTP-bound
Gtr1 activates TORC1, whereas GDP-bound Gtr1 inactivates TORC1 even under conditions that
promote wild-type cell growth.6 Gtr1 and
mammalian RRAGB are thought to switch from the GTP-bound form to the GDP-bound form in
response to amino-acid starvation.6,13 Thus,
switching of the nucleotide-binding state of Gtr1, and the resultant effects on TORC1
activity, are key events in cellular adaptation to extracellular conditions. However, the
mechanisms by which TORC1 is inactivated are poorly understood.We performed a genome-wide screen that identified Npr2 and Npr3 as key regulators of
autophagy in yeast; this function is also conserved in mammals. Genetic analyses revealed
that Npr2-Npr3 regulates autophagy via Gtr1-Gtr2. An NPR2 deletion mutant
mimicked the phenotypes produced by a GTP-bound Gtr1 mutant with respect to both autophagy
and Tor1 localization, suggesting that Npr2-Npr3 regulates GTP hydrolysis in Gtr1.
Furthermore, we found that Gtr2 binds to Kog1. GDP-bound Gtr1 mutant negatively regulates
TORC1, and the Gtr2-Kog1 interaction is necessary for this regulation. These 2 molecular
mechanisms are involved in inactivation of TORC1 and induction of autophagy.

Introduction

Autophagy, an intracellular process that mediates the bulk degradation of cellular
components, is conserved from yeast to mammals.1 This process is induced under stressful conditions, such as nutrient
starvation. Autophagy is characterized by cytosolic membranous structures called
autophagosomes, and the delivery of cytosolic materials to lysosomes and vacuoles for
degradation. By supplying the products of degradation as resources for the cell, autophagy
helps cells adapt to stressful conditions.

The TORC1 protein kinase complex, which in the yeast Saccharomyces
cerevisiae consists of the Tor1/2 protein kinase, Kog1, Lst8, and Tco89, plays a
central role in the regulation of autophagy, which is induced by TORC1 inactivation.2,3 TORC1 contributes to cell growth and homeostasis in response to
environmental cues. Various factors, including nutrients, stress, and intracellular energy
levels, affect the kinase activity of TORC1; TORC1 is activated under conditions favorable
to cellular growth, whereas stressful conditions, including nutritional limitation,
inactivate TORC1.4 In yeast autophagy,
inactivation of TORC1 leads to dephosphorylation of Atg13, a substrate of TORC1, and
induction of autophagy.5

The heterodimeric G proteins Gtr1 and Gtr2 play a critical role in the regulation of TORC1
in yeast.6 In mammals, RRAGA and RRAGB are
orthologs of Gtr1, and RRAGC and RRAGD are orthologs of Gtr2.7,8
Together with Ego1 and Ego3, Gtr1 and Gtr2 are components of the EGO complex;9,10 Ego1 and Ego3 are the putative functional counterparts of the
mammalian Ragulator complex.10-12

The nucleotide-binding state of Gtr1-Gtr2 is crucial for the regulation of TORC1. GTP-bound
Gtr1 activates TORC1, whereas GDP-bound Gtr1 inactivates TORC1 even under conditions that
promote wild-type cell growth.6 Gtr1 and
mammalian RRAGB are thought to switch from the GTP-bound form to the GDP-bound form in
response to amino-acid starvation.6,13 Thus,
switching of the nucleotide-binding state of Gtr1, and the resultant effects on TORC1
activity, are key events in cellular adaptation to extracellular conditions. However, the
mechanisms by which TORC1 is inactivated are poorly understood.

We performed a genome-wide screen that identified Npr2 and Npr3 as key regulators of
autophagy in yeast; this function is also conserved in mammals. Genetic analyses revealed
that Npr2-Npr3 regulates autophagy via Gtr1-Gtr2. An NPR2 deletion mutant
mimicked the phenotypes produced by a GTP-bound Gtr1 mutant with respect to both autophagy
and Tor1 localization, suggesting that Npr2-Npr3 regulates GTP hydrolysis in Gtr1.
Furthermore, we found that Gtr2 binds to Kog1. GDP-bound Gtr1 mutant negatively regulates
TORC1, and the Gtr2-Kog1 interaction is necessary for this regulation. These 2 molecular
mechanisms are involved in inactivation of TORC1 and induction of autophagy.

ResultsThe roles of Npr2 and Npr3 in TORC1 regulation are functionally conserved from yeast
to mammalsThe alkaline phosphatase (ALP) assay is widely used to quantitatively assess the amount
of cytosol delivered to vacuoles during autophagy in Saccharomyces
cerevisiae.14,15 To
examine the mechanisms regulating autophagy, we extended the ALP assay to a large-scale
format, using 96-well microtiter plates to perform a genome-wide screen in a yeast mutant
collection (see Materials and Methods). In this screen, approximately 4,600 mutant strains
from a nonessential gene-deletion mutant collection were starved of nitrogen sources for 4
h, and then subjected to the ALP assay. All core atg mutants, which have
been previously characterized as defective in autophagy,16 exhibited quite low levels of ALP activity, indicating that our
large-scale assay system was capable of measuring the autophagic capacity of each strain
in this collection (Fig. 1A and B). Deletions of
NPR2 and NPR3 also resulted in significant defects in
nitrogen-starvation-induced autophagy (Fig. 1B and
C). Recent studies have shown that Npr2 and Npr3, along with Iml1, are components
of the SEA complex.17,18 Consistent with this, the
iml1∆ deletion mutant (which was not included in the original mutant
collection) also exhibited defects in autophagy (Fig. 1C). Treatment with the TORC1-specific inhibitor rapamycin abolished the
defect caused by deletion of Npr2 or Npr3, and induced autophagy at a level similar to
that observed in wild-type cells (Fig. 1C). No additive defect in nitrogen-starvation-induced autophagy was
observed in npr2∆ npr3∆ double mutants (Fig. S1A). These results are
consistent with a previous report that Npr2 and Npr3 function as negative regulators
upstream of TORC1.19 Thus, cells lacking
Npr2, Npr3, and Iml1 exhibit defects in induction of autophagy through TORC1
inactivation.18Figure 1. Gtr1 contributes to the regulation of autophagy by Npr2-Npr3.
(A) ALP activities in the mutant strains bearing
pho8∆60 after 4 h of starvation. Activity measured
in the vacuolar proteinase A mutant strain pep4∆ (SKY001) was
subtracted as the background signal. Activities are shown as the percentage values for
each strain relative to that of the parental TNY509 strain. Deletion mutants lacking
the core ATG genes, NPR2, or NPR3
are denoted with arrows. (B) Detailed ALP activities for representative
atg mutants shown in (A). (C) ALP
activities in the wild-type (SKY084), pep4∆ (SKY100),
npr2∆ (SKY091), npr3∆ (SKY131), and
iml1∆ (SKY264) strains. Before the cells were subjected to the ALP
assay, the strains were incubated in YPD medium at log phase, under nitrogen
starvation conditions for 3 h, or in the presence of rapamycin for 3 h. Data represent
means ± standard deviation from 3 independent experiments. (D) Wild-type
(BY4741, SKY084) and pep4∆ (SKY100), gtr1∆ (SKY244),
gtr2∆ (SKY246), gtr1∆ npr2∆ (SKY245),
gtr2∆ npr2∆ (SKY247) yeast cells were grown in YPD, starved for 3
h, and subjected to ALP assays. Data represent means ± standard deviation from 3
independent experiments.NPRL2 and NPRL3 are the mammalian homologs of Npr2 and Npr3.19 To investigate the roles of the mammalian proteins, we
knocked down NPRL2 and NPRL3 expression in both HeLa and MCF7 cells. Specific small
interfering RNA (siRNA) reduced NPRL2 and NPRL3 mRNA
levels to less than 30% of the levels observed in control HeLa cells (Fig. S2A). siRNA treatment also
reduced NPRL2 and NPRL3 protein levels in MCF7 cells (Fig. S2B). In control cells under
nutrient-rich conditions, MTOR immunofluorescence signals were detected as multiple puncta
in the perinuclear region (Fig. 2A and B; Fig. S3A and S3B). These MTOR puncta
colocalized with a lysosomal protein, LAMP1 (Fig. 2C), whereas starvation conditions caused MTOR disperse throughout the
cytosol, as reported previously.13
However, knockdown of NPRL2 or NPRL3 caused MTOR to remain on the lysosomes even under
starvation conditions (Fig. 2A and B; Fig. S3A and S3B). Because lysosomal
localization is important for MTORC1 activation,12 this observation suggested that knocking down NPRL2 or NPRL3
allowed MTORC1 to remain active even under starvation conditions, indicating that MTORC1
activity is regulated by NPRL2 and NPRL3. To further test this idea, we estimated the
autophagic activity, as a read-out of MTORC1 activity, by counting the number of
autophagosomes labeled with the specific marker GFP-LC3.20 In control cells under starvation conditions, numerous
LC3-positive puncta were detected, indicating that autophagy was induced (Fig. 2D and E; Fig. S3C and S3D). On the other hand, markedly fewer LC3-positive
puncta were observed following knockdown of NPRL2 or NPRL3, even under starvation
conditions (Fig. 2D and E; Fig. S3C and S3D). These data
indicate that deficiencies in NPRL2 and NPRL3 influenced both induction of autophagy and
the starvation-dependent inactivation of MTORC1. Thus, Npr2 and Npr3 must function in some
step of TORC1 regulation, which is conserved from yeast to mammals.Figure 2. Effects of knockdown of NPRL2 and NPRL3 expression on MTOR
localization and autophagy. (A) Two and a half d after siRNA transfection
(clone #1), HeLa cells were incubated in nutrient-rich growth medium or EBSS
(starvation medium) for 1 h. Cells were immunostained with anti-MTOR antibodies and
subjected to fluorescence microscopy. Scale bar: 20 μm. (B) MTOR-positive
puncta (A) were counted in NPRL2- or NPRL3-knockdown HeLa cells using the
G-count software. Data represent means ± standard deviation of representative results
(n > 40 cells). (C) Effect of NPRL2 or NPRL3 knockdown on MTOR
localization. Knockdown MCF7 cells were incubated in growth medium (serum, amino acid:
+) or EBSS (serum, amino acid: -) for 60 min, and then stained with MTOR (green) and
LAMP1 (magenta) antibodies. Bars indicate 20 μm. Results shown are from siRNA clone #1
for NPRL2 knockdown and clone #2 for NPRL3 knockdown. (D) HeLa cells were
cotransfected with siRNA (clone #1) and a vector harboring EGFP-LC3. Two and a half d
after siRNA transfection, cells incubated in growth medium or starvation medium for 1
h were subjected to fluorescence microscopy. Scale bars: 20 μm. (E)
GFP-LC3-positive puncta in each cell from (D) were counted. Data
represent means ± standard deviation of representative results (n > 40 cells).The autophagic defect of npr2 mutant cells is rescued by deletion of
GTR1 or GTR2The heterodimeric G-protein complex Gtr1-Gtr2, the yeast ortholog of mammalian
RRAGA/B-RRAGC/D, is an upstream regulator of TORC1.7,8,21
Therefore, we assessed the potential relationships between Gtr1-Gtr2 and Npr2-Npr3 by
examining double-deletion mutants under nitrogen-starvation conditions. Intriguingly, the
autophagic defects observed in npr3∆ and npr2∆ cells
were abrogated by further deletion of GTR1 or GTR2,
which resulted in wild-type levels of autophagy (Fig. 1D; Fig. S1A).
Next, we monitored the phosphorylation status of Atg13, which is a direct substrate of
TORC1 kinase and essential for autophagy.5,22 In
wild-type cells, Atg13 was dephosphorylated in response to starvation conditions (Fig. S1B and S1C).22 By contrast, Atg13 remained
phosphorylated in npr2∆ or npr3∆ cells, even under
starvation conditions (Fig. S1B and
S1C). Meanwhile, deletion of GTR1 or
GTR2 from npr2∆ or npr3∆ cells caused
dephosphorylation of Atg13 under starvation conditions (Fig. S1B and S1C). These results
indicate that Npr2 and Npr3 function upstream of the Gtr1-Gtr2 complex in the regulation
of TORC1.GTP-bound Gtr1 suppresses autophagy under nitrogen starvationOur results suggested that Gtr1 is involved in the regulation of autophagy. Unexpectedly,
however, deletion of neither GTR1 nor GTR2 had any
effect on induction of autophagy (Fig. 1D and Fig. S1A, see ref. 9). Therefore, to ask whether the Gtr1-Gtr2 complex
played a role in the regulation of autophagy, we tested the effects of Gtr1 and Gtr2
mutants that mimic various nucleotide-bound states.23 As a result, autophagy induction in response to
nitrogen-starvation was severely deficient in gtr1∆ gtr2∆ cells
exogenously expressing both the GTP-bound form Gtr1 mutant and wild-type Gtr2, driven from
their native promoters (Fig. 3, column 8). These data
indicate that the GTP-bound Gtr1 mutant exerts a dominant effect on autophagy in the
gtr1∆ background. This effect was also observed in cells expressing
both the GTP-bound Gtr1 mutant and the GDP-bound form of Gtr2 (Fig. 3, column 10). On the other hand, when cells completely lacked
Gtr2, autophagy was induced normally in response to nitrogen-starvation (Fig. 3, column 12). Thus, Gtr2, at least in the GDP form,
appears to be critical for the dominant effects of the GTP-bound Gtr1 mutant. We also
found that vacuolar localization of Gtr16
was disrupted by deletion of Gtr2 (Fig.
S4A). Likewise, deletion of GTR1 disrupted vacuolar localization
of Gtr2 (Fig. S4A). Therefore,
Gtr2 seems to be important for functional localization of Gtr1, and vice versa. We also
noticed that the combination of the GTP-bound forms of Gtr1 and Gtr2 inhibited
starvation-induced autophagy (Fig.
S5A, column 18), but we interpret the role of Gtr2 in another way (see below and
Discussion)Figure 3. A GTP-bound Gtr1 mutant suppresses autophagy in a
Gtr2-dependent manner. gtr1∆ gtr2∆ double-deletion cells (SKY167)
harboring empty vector (-), plasmid encoding GTP-bound mutant Gtr1 (GTP), and/or
plasmid encoding wild-type (WT) or GDP-bound mutant Gtr2 (GDP) and tco89∆
gtr1∆ gtr2∆ triple-mutant cells (SKY277) expressing GTP-bound Gtr1 mutant
(GTP) and GDP-bound Gtr2 mutant (GDP) were grown in SD medium containing 0.5% casamino
acids, incubated under nitrogen-starvation conditions for 3 h, and then subjected to
ALP assays. Data represent means ± standard deviation from 3 independent experiments.
The ALP activities of wild-type (SKY084) and pep4∆ (SKY100) cells
harboring empty vector were also measured as control samples.To determine whether TORC1 played a role in these effects, we monitored autophagy in
cells lacking the nonessential TORC1 subunit Tco89.24 The tco89 mutant cells expressing GTP-bound
Gtr1 and GDP-bound Gtr2 mutants exhibited starvation-induced autophagy at levels
comparable to those in wild-type cells (Fig. 3,
column 14). These data indicate that GTP-bound Gtr1 causes defects in autophagy by
activating TORC1 even under nitrogen-starvation conditions.Tor1 vacuolar localization is increased in both GTP-bound Gtr1 and
npr2∆ mutantsNext, we observed the localization of N-terminally GFP-tagged Tor1, expressed from the
genome under the control of the native promoter. This GFP-Tor1 best replicated the native
Tor1 function among several constructs including previously reported ones (Fig. S4B). As previously reported,
GFP-Tor1 was detected on vacuolar membranes, and some puncta associated with vacuolar
membranes under nutrient-rich conditions (Fig. 4A).6,25 The identity of these perivacuolar puncta
remains obscure; they did not colocalize with markers of other compartments such as the
phagophore assembly site (Ape1-mCherry) or the endosome (Snf7-mCherry) (data not shown).
Deletion of GTR1 or GTR2 significantly reduced the
vacuolar localization of Tor1 (Fig. 4A and
B, columns 5 to 10), although perivacuolar localization of Tor1 persisted. The
expression level of native Tor1 was not affected in these mutants (Fig. S4C). Thus, Gtr1 and Gtr2
appear to maintain Tor1 localization on the vacuoles.Figure 4. Vacuolar localization of Tor1 is regulated by Gtr1, Gtr2,
Npr2, and nutrient availability. (A) Strains expressing N-terminally
GFP-tagged Tor1 under the control of the native promoter (WT, SKY222;
npr2∆, SKY226; gtr1∆, SKY278;
gtr2∆, SKY279; gtr1∆ gtr2∆, SKY299) were incubated
in SD medium containing 0.5% casamino acids and 100 µM CMAC for 30 min, and either
washed once with SD medium containing 0.5% casamino acids or starved for nitrogen
(SD-N) for 1 to 2 h, and then subjected to fluorescence microscopy. Scale bar: 5 μm.
(B) Percentage of cells in (A) exhibiting vacuolar
localization of Tor1. Cells with GFP-Tor1 signals surrounding vacuoles stained by CMAC
were counted. More than 100 cells were counted. Data represent means ± standard
deviation from 3 independent experiments. (C) Strains expressing
N-terminally GFP-tagged Tor1 (gtr1∆ gtr2∆, SKY299; gtr1∆
gtr2∆ npr2∆, SKY320) and expressing the indicated combinations of Gtr1 and
Gtr2 mutants were subjected to fluorescence microscopy under the same conditions as in
(A). Percentage of cells exhibiting vacuolar localization of Tor1 is
presented in (B).We noticed that the number of cells with vacuolar localization of Tor1 was reduced in
response to starvation conditions, although a significant fraction of cells (39%) still
exhibited vacuolar localization of Tor1 (Fig. 4A and
B, columns 1 and 2). The GTP-bound Gtr1 mutants also increased vacuolar
localization of GFP-Tor1 (Fig. 4C, columns 7 and 8).
For this increase to occur, Gtr2 was needed to be in the GDP-bound form rather than the
GTP-bound form (Fig. 4C, columns 5, 6, 9, and 10).
Thus, dynamics of Tor1 localization are regulated by Gtr1 and Gtr2.Furthermore, we found that deletion of NPR2 increased the cells with
vacuolar localization of Tor1 (Fig. 4A and B, columns
3 and 4; Fig. 4C, columns 3 and 4). Thus, the
npr2∆ mutant is similar to the GTP-bound Gtr1 mutant with respect to
Tor1 localization and defects in induction of autophagy. These results support the model
that the npr2∆ mutant is defective in hydrolysis of GTP bound to
Gtr1.The GDP-bound Gtr1 mutant induces autophagy even under nutrient-rich
conditionsNext, we investigated the phenotype of a Gtr1 mutant mimicking a permanently GDP-bound
state. Cells expressing the GDP-bound Gtr1 mutant induced autophagy even under
nutrient-rich conditions (Fig. 5A, columns 7 and 9),
but this phenotype was abrogated in cells lacking Gtr2 (Fig. 5A, column 13). Autophagic induction was still observed when wild-type Gtr2
was replaced with the GTP-bound mutant, but absent when Gtr2 was replaced by the GDP-bound
form (Fig. 5A, columns 9 and 11). Thus, GTP-bound
Gtr2 is required for the dominant effect of the GDP-bound Gtr1 mutant. Autophagic
induction by this dominant effect was not affected by the deletion of
NPR2 (Fig.
S5A, columns 27 and 29), supporting the idea that Npr2 functions upstream of
Gtr1-Gtr2.Figure 5. Gtr2 interacts with Kog1 and inactivates TORC1.
(A) Wild-type (SKY084), pep4∆ (SKY100), and
gtr1∆ gtr2∆ double-mutant cells harboring empty vector (-), and
gtr1∆ gtr2∆ cells harboring the GDP-bound mutant Gtr1 (top row:
GDP) along with wild-type, GTP-bound mutant, or GDP-bound mutant Gtr2 (bottom row: WT,
GTP, and GDP, respectively), were either grown in SD medium supplemented with 0.5%
casamino acids or starved of nitrogen for 3 h, and then subjected to ALP assays. Data
represent means ± standard deviation from 3 independent experiments. (B)
Cells of the strain for yeast 2-hybrid analysis (PJ69-4A) harboring pGBD-C1 vector
encoding wild-type, GTP-bound mutant, GDP-bound mutant, or E62K mutant Gtr2 (pSK150,
pSK151, pSK152, or pSK222, respectively) and pGAD-C1/Kog1 (pSK156) were serially
10-fold diluted and spotted onto SC medium plates either lacking histidine and leucine
or histidine, leucine, and uracil. Results after 2 d of culture at 30 °C are shown.
(C) Cells endogenously expressing TAP-tagged Kog1 and harboring
multicopy vector encoding wild-type, GTP-bound mutant, GDP-bound mutant, or E62K
mutant Gtr2 were subjected to TAP affinity isolation assays. The lysates and the
affinity isolates were subjected to western blot analysis with anti-protein A and
anti-GST antibodies. (D) Strains (wild-type, SKY084;
pep4∆, SKY100; and gtr1∆ gtr2∆, SKY167) harboring
empty vector (-), and gtr1∆ gtr2∆ (SKY167) harboring GDP-bound Gtr1
mutant (GDP) along wild-type (WT) or E62K mutant Gtr2E62K, were either
grown in SD medium supplemented with 0.5% casamino acids or starved of nitrogen for 3
h, and then subjected to ALP assays. Data represent means ± standard deviation from 3
independent experiments.The TORC1 subunit Kog1 is an effector of Gtr2Based on these results, we investigated the role of Gtr2 in the autophagic induction
dependent on GDP-bound Gtr1. In a 2-hybrid analysis, we found that Gtr2 binds to Kog1
(Fig. 5B), a subunit of TORC1 and a mammalian
ortholog of RAPTOR.26 Kog1 also strongly
bound to the GTP-bound Gtr2 mutant, but not to the GDP-bound mutant (Fig. 5B). To confirm this interaction biochemically, we performed
affinity isolation assays. Lysates of Kog1-TAP-expressing cells were precipitated by IgG
beads, and the affinity isolated fractions were subjected to western blotting. As shown in
Figure 5C, wild-type GST-Gtr2 was precipitated in a
Kog1-TAP-dependent manner (relative amount, 21.6); the amount of material pulled down in
the specific precipitation was significantly larger than the amount pulled down by
nonspecific binding, possibly to the resin (see control lane; relative amount 5.2), which
could not be entirely prevented. The GTP-bound form of GST-Gtr2 was also efficiently
precipitated, whereas the GDP-bound form of GST-Gtr2 was precipitated to a lesser extent
(Fig. 5C). These data indicate that Kog1 is a
direct effector of Gtr2.TORC1 is inactivated by binding to Gtr2A recent structural analysis of Gtr1 revealed the critical amino acids on its
effector-binding surface.27 Based on its
structural similarity to Gtr1, Gtr2 is predicted to bind to Kog1 via a similar surface.
Accordingly, we made 9 point-mutants on the possible surface area of Gtr2; in one of these
mutants, glutamine 62 of Gtr2, which corresponds to Gtr1 asparagine 61, was mutated to
lysine. This E62K mutation of Gtr2 abolished the interaction to Kog1, as demonstrated by
yeast 2-hybrid analysis (Fig. 5B). Furthermore,
coprecipitation with Kog1-TAP was severely defective in the Gtr2E62K mutant,
although the expression level of the E62K mutant was not affected (Fig. 5C). Using the E62K mutant, we asked whether the binding of Gtr2 to
Kog1 is required for autophagic induction by expression of the GDP-bound form Gtr1.
Indeed, the autophagic induction was abrogated in the Gtr2E62K mutant (Fig. 5D, columns 7 and 9). Collectively, these data
indicate that direct binding of Gtr2 to Kog1 inactivates TORC1 and eventually induces
autophagy, even under nutrient-rich conditions.

The roles of Npr2 and Npr3 in TORC1 regulation are functionally conserved from yeast
to mammalsThe alkaline phosphatase (ALP) assay is widely used to quantitatively assess the amount
of cytosol delivered to vacuoles during autophagy in Saccharomyces
cerevisiae.14,15 To
examine the mechanisms regulating autophagy, we extended the ALP assay to a large-scale
format, using 96-well microtiter plates to perform a genome-wide screen in a yeast mutant
collection (see Materials and Methods). In this screen, approximately 4,600 mutant strains
from a nonessential gene-deletion mutant collection were starved of nitrogen sources for 4
h, and then subjected to the ALP assay. All core atg mutants, which have
been previously characterized as defective in autophagy,16 exhibited quite low levels of ALP activity, indicating that our
large-scale assay system was capable of measuring the autophagic capacity of each strain
in this collection (Fig. 1A and B). Deletions of
NPR2 and NPR3 also resulted in significant defects in
nitrogen-starvation-induced autophagy (Fig. 1B and
C). Recent studies have shown that Npr2 and Npr3, along with Iml1, are components
of the SEA complex.17,18 Consistent with this, the
iml1∆ deletion mutant (which was not included in the original mutant
collection) also exhibited defects in autophagy (Fig. 1C). Treatment with the TORC1-specific inhibitor rapamycin abolished the
defect caused by deletion of Npr2 or Npr3, and induced autophagy at a level similar to
that observed in wild-type cells (Fig. 1C). No additive defect in nitrogen-starvation-induced autophagy was
observed in npr2∆ npr3∆ double mutants (Fig. S1A). These results are
consistent with a previous report that Npr2 and Npr3 function as negative regulators
upstream of TORC1.19 Thus, cells lacking
Npr2, Npr3, and Iml1 exhibit defects in induction of autophagy through TORC1
inactivation.18Figure 1. Gtr1 contributes to the regulation of autophagy by Npr2-Npr3.
(A) ALP activities in the mutant strains bearing
pho8∆60 after 4 h of starvation. Activity measured
in the vacuolar proteinase A mutant strain pep4∆ (SKY001) was
subtracted as the background signal. Activities are shown as the percentage values for
each strain relative to that of the parental TNY509 strain. Deletion mutants lacking
the core ATG genes, NPR2, or NPR3
are denoted with arrows. (B) Detailed ALP activities for representative
atg mutants shown in (A). (C) ALP
activities in the wild-type (SKY084), pep4∆ (SKY100),
npr2∆ (SKY091), npr3∆ (SKY131), and
iml1∆ (SKY264) strains. Before the cells were subjected to the ALP
assay, the strains were incubated in YPD medium at log phase, under nitrogen
starvation conditions for 3 h, or in the presence of rapamycin for 3 h. Data represent
means ± standard deviation from 3 independent experiments. (D) Wild-type
(BY4741, SKY084) and pep4∆ (SKY100), gtr1∆ (SKY244),
gtr2∆ (SKY246), gtr1∆ npr2∆ (SKY245),
gtr2∆ npr2∆ (SKY247) yeast cells were grown in YPD, starved for 3
h, and subjected to ALP assays. Data represent means ± standard deviation from 3
independent experiments.NPRL2 and NPRL3 are the mammalian homologs of Npr2 and Npr3.19 To investigate the roles of the mammalian proteins, we
knocked down NPRL2 and NPRL3 expression in both HeLa and MCF7 cells. Specific small
interfering RNA (siRNA) reduced NPRL2 and NPRL3 mRNA
levels to less than 30% of the levels observed in control HeLa cells (Fig. S2A). siRNA treatment also
reduced NPRL2 and NPRL3 protein levels in MCF7 cells (Fig. S2B). In control cells under
nutrient-rich conditions, MTOR immunofluorescence signals were detected as multiple puncta
in the perinuclear region (Fig. 2A and B; Fig. S3A and S3B). These MTOR puncta
colocalized with a lysosomal protein, LAMP1 (Fig. 2C), whereas starvation conditions caused MTOR disperse throughout the
cytosol, as reported previously.13
However, knockdown of NPRL2 or NPRL3 caused MTOR to remain on the lysosomes even under
starvation conditions (Fig. 2A and B; Fig. S3A and S3B). Because lysosomal
localization is important for MTORC1 activation,12 this observation suggested that knocking down NPRL2 or NPRL3
allowed MTORC1 to remain active even under starvation conditions, indicating that MTORC1
activity is regulated by NPRL2 and NPRL3. To further test this idea, we estimated the
autophagic activity, as a read-out of MTORC1 activity, by counting the number of
autophagosomes labeled with the specific marker GFP-LC3.20 In control cells under starvation conditions, numerous
LC3-positive puncta were detected, indicating that autophagy was induced (Fig. 2D and E; Fig. S3C and S3D). On the other hand, markedly fewer LC3-positive
puncta were observed following knockdown of NPRL2 or NPRL3, even under starvation
conditions (Fig. 2D and E; Fig. S3C and S3D). These data
indicate that deficiencies in NPRL2 and NPRL3 influenced both induction of autophagy and
the starvation-dependent inactivation of MTORC1. Thus, Npr2 and Npr3 must function in some
step of TORC1 regulation, which is conserved from yeast to mammals.Figure 2. Effects of knockdown of NPRL2 and NPRL3 expression on MTOR
localization and autophagy. (A) Two and a half d after siRNA transfection
(clone #1), HeLa cells were incubated in nutrient-rich growth medium or EBSS
(starvation medium) for 1 h. Cells were immunostained with anti-MTOR antibodies and
subjected to fluorescence microscopy. Scale bar: 20 μm. (B) MTOR-positive
puncta (A) were counted in NPRL2- or NPRL3-knockdown HeLa cells using the
G-count software. Data represent means ± standard deviation of representative results
(n > 40 cells). (C) Effect of NPRL2 or NPRL3 knockdown on MTOR
localization. Knockdown MCF7 cells were incubated in growth medium (serum, amino acid:
+) or EBSS (serum, amino acid: -) for 60 min, and then stained with MTOR (green) and
LAMP1 (magenta) antibodies. Bars indicate 20 μm. Results shown are from siRNA clone #1
for NPRL2 knockdown and clone #2 for NPRL3 knockdown. (D) HeLa cells were
cotransfected with siRNA (clone #1) and a vector harboring EGFP-LC3. Two and a half d
after siRNA transfection, cells incubated in growth medium or starvation medium for 1
h were subjected to fluorescence microscopy. Scale bars: 20 μm. (E)
GFP-LC3-positive puncta in each cell from (D) were counted. Data
represent means ± standard deviation of representative results (n > 40 cells).

The roles of Npr2 and Npr3 in TORC1 regulation are functionally conserved from yeast
to mammals

The alkaline phosphatase (ALP) assay is widely used to quantitatively assess the amount
of cytosol delivered to vacuoles during autophagy in Saccharomyces
cerevisiae.14,15 To
examine the mechanisms regulating autophagy, we extended the ALP assay to a large-scale
format, using 96-well microtiter plates to perform a genome-wide screen in a yeast mutant
collection (see Materials and Methods). In this screen, approximately 4,600 mutant strains
from a nonessential gene-deletion mutant collection were starved of nitrogen sources for 4
h, and then subjected to the ALP assay. All core atg mutants, which have
been previously characterized as defective in autophagy,16 exhibited quite low levels of ALP activity, indicating that our
large-scale assay system was capable of measuring the autophagic capacity of each strain
in this collection (Fig. 1A and B). Deletions of
NPR2 and NPR3 also resulted in significant defects in
nitrogen-starvation-induced autophagy (Fig. 1B and
C). Recent studies have shown that Npr2 and Npr3, along with Iml1, are components
of the SEA complex.17,18 Consistent with this, the
iml1∆ deletion mutant (which was not included in the original mutant
collection) also exhibited defects in autophagy (Fig. 1C). Treatment with the TORC1-specific inhibitor rapamycin abolished the
defect caused by deletion of Npr2 or Npr3, and induced autophagy at a level similar to
that observed in wild-type cells (Fig. 1C). No additive defect in nitrogen-starvation-induced autophagy was
observed in npr2∆ npr3∆ double mutants (Fig. S1A). These results are
consistent with a previous report that Npr2 and Npr3 function as negative regulators
upstream of TORC1.19 Thus, cells lacking
Npr2, Npr3, and Iml1 exhibit defects in induction of autophagy through TORC1
inactivation.18

Figure 1. Gtr1 contributes to the regulation of autophagy by Npr2-Npr3.
(A) ALP activities in the mutant strains bearing
pho8∆60 after 4 h of starvation. Activity measured
in the vacuolar proteinase A mutant strain pep4∆ (SKY001) was
subtracted as the background signal. Activities are shown as the percentage values for
each strain relative to that of the parental TNY509 strain. Deletion mutants lacking
the core ATG genes, NPR2, or NPR3
are denoted with arrows. (B) Detailed ALP activities for representative
atg mutants shown in (A). (C) ALP
activities in the wild-type (SKY084), pep4∆ (SKY100),
npr2∆ (SKY091), npr3∆ (SKY131), and
iml1∆ (SKY264) strains. Before the cells were subjected to the ALP
assay, the strains were incubated in YPD medium at log phase, under nitrogen
starvation conditions for 3 h, or in the presence of rapamycin for 3 h. Data represent
means ± standard deviation from 3 independent experiments. (D) Wild-type
(BY4741, SKY084) and pep4∆ (SKY100), gtr1∆ (SKY244),
gtr2∆ (SKY246), gtr1∆ npr2∆ (SKY245),
gtr2∆ npr2∆ (SKY247) yeast cells were grown in YPD, starved for 3
h, and subjected to ALP assays. Data represent means ± standard deviation from 3
independent experiments.

NPRL2 and NPRL3 are the mammalian homologs of Npr2 and Npr3.19 To investigate the roles of the mammalian proteins, we
knocked down NPRL2 and NPRL3 expression in both HeLa and MCF7 cells. Specific small
interfering RNA (siRNA) reduced NPRL2 and NPRL3 mRNA
levels to less than 30% of the levels observed in control HeLa cells (Fig. S2A). siRNA treatment also
reduced NPRL2 and NPRL3 protein levels in MCF7 cells (Fig. S2B). In control cells under
nutrient-rich conditions, MTOR immunofluorescence signals were detected as multiple puncta
in the perinuclear region (Fig. 2A and B; Fig. S3A and S3B). These MTOR puncta
colocalized with a lysosomal protein, LAMP1 (Fig. 2C), whereas starvation conditions caused MTOR disperse throughout the
cytosol, as reported previously.13
However, knockdown of NPRL2 or NPRL3 caused MTOR to remain on the lysosomes even under
starvation conditions (Fig. 2A and B; Fig. S3A and S3B). Because lysosomal
localization is important for MTORC1 activation,12 this observation suggested that knocking down NPRL2 or NPRL3
allowed MTORC1 to remain active even under starvation conditions, indicating that MTORC1
activity is regulated by NPRL2 and NPRL3. To further test this idea, we estimated the
autophagic activity, as a read-out of MTORC1 activity, by counting the number of
autophagosomes labeled with the specific marker GFP-LC3.20 In control cells under starvation conditions, numerous
LC3-positive puncta were detected, indicating that autophagy was induced (Fig. 2D and E; Fig. S3C and S3D). On the other hand, markedly fewer LC3-positive
puncta were observed following knockdown of NPRL2 or NPRL3, even under starvation
conditions (Fig. 2D and E; Fig. S3C and S3D). These data
indicate that deficiencies in NPRL2 and NPRL3 influenced both induction of autophagy and
the starvation-dependent inactivation of MTORC1. Thus, Npr2 and Npr3 must function in some
step of TORC1 regulation, which is conserved from yeast to mammals.

Figure 2. Effects of knockdown of NPRL2 and NPRL3 expression on MTOR
localization and autophagy. (A) Two and a half d after siRNA transfection
(clone #1), HeLa cells were incubated in nutrient-rich growth medium or EBSS
(starvation medium) for 1 h. Cells were immunostained with anti-MTOR antibodies and
subjected to fluorescence microscopy. Scale bar: 20 μm. (B) MTOR-positive
puncta (A) were counted in NPRL2- or NPRL3-knockdown HeLa cells using the
G-count software. Data represent means ± standard deviation of representative results
(n > 40 cells). (C) Effect of NPRL2 or NPRL3 knockdown on MTOR
localization. Knockdown MCF7 cells were incubated in growth medium (serum, amino acid:
+) or EBSS (serum, amino acid: -) for 60 min, and then stained with MTOR (green) and
LAMP1 (magenta) antibodies. Bars indicate 20 μm. Results shown are from siRNA clone #1
for NPRL2 knockdown and clone #2 for NPRL3 knockdown. (D) HeLa cells were
cotransfected with siRNA (clone #1) and a vector harboring EGFP-LC3. Two and a half d
after siRNA transfection, cells incubated in growth medium or starvation medium for 1
h were subjected to fluorescence microscopy. Scale bars: 20 μm. (E)
GFP-LC3-positive puncta in each cell from (D) were counted. Data
represent means ± standard deviation of representative results (n > 40 cells).

The autophagic defect of npr2 mutant cells is rescued by deletion of
GTR1 or GTR2The heterodimeric G-protein complex Gtr1-Gtr2, the yeast ortholog of mammalian
RRAGA/B-RRAGC/D, is an upstream regulator of TORC1.7,8,21
Therefore, we assessed the potential relationships between Gtr1-Gtr2 and Npr2-Npr3 by
examining double-deletion mutants under nitrogen-starvation conditions. Intriguingly, the
autophagic defects observed in npr3∆ and npr2∆ cells
were abrogated by further deletion of GTR1 or GTR2,
which resulted in wild-type levels of autophagy (Fig. 1D; Fig. S1A).
Next, we monitored the phosphorylation status of Atg13, which is a direct substrate of
TORC1 kinase and essential for autophagy.5,22 In
wild-type cells, Atg13 was dephosphorylated in response to starvation conditions (Fig. S1B and S1C).22 By contrast, Atg13 remained
phosphorylated in npr2∆ or npr3∆ cells, even under
starvation conditions (Fig. S1B and
S1C). Meanwhile, deletion of GTR1 or
GTR2 from npr2∆ or npr3∆ cells caused
dephosphorylation of Atg13 under starvation conditions (Fig. S1B and S1C). These results
indicate that Npr2 and Npr3 function upstream of the Gtr1-Gtr2 complex in the regulation
of TORC1.

The autophagic defect of npr2 mutant cells is rescued by deletion of
GTR1 or GTR2

The heterodimeric G-protein complex Gtr1-Gtr2, the yeast ortholog of mammalian
RRAGA/B-RRAGC/D, is an upstream regulator of TORC1.7,8,21
Therefore, we assessed the potential relationships between Gtr1-Gtr2 and Npr2-Npr3 by
examining double-deletion mutants under nitrogen-starvation conditions. Intriguingly, the
autophagic defects observed in npr3∆ and npr2∆ cells
were abrogated by further deletion of GTR1 or GTR2,
which resulted in wild-type levels of autophagy (Fig. 1D; Fig. S1A).
Next, we monitored the phosphorylation status of Atg13, which is a direct substrate of
TORC1 kinase and essential for autophagy.5,22 In
wild-type cells, Atg13 was dephosphorylated in response to starvation conditions (Fig. S1B and S1C).22 By contrast, Atg13 remained
phosphorylated in npr2∆ or npr3∆ cells, even under
starvation conditions (Fig. S1B and
S1C). Meanwhile, deletion of GTR1 or
GTR2 from npr2∆ or npr3∆ cells caused
dephosphorylation of Atg13 under starvation conditions (Fig. S1B and S1C). These results
indicate that Npr2 and Npr3 function upstream of the Gtr1-Gtr2 complex in the regulation
of TORC1.

GTP-bound Gtr1 suppresses autophagy under nitrogen starvationOur results suggested that Gtr1 is involved in the regulation of autophagy. Unexpectedly,
however, deletion of neither GTR1 nor GTR2 had any
effect on induction of autophagy (Fig. 1D and Fig. S1A, see ref. 9). Therefore, to ask whether the Gtr1-Gtr2 complex
played a role in the regulation of autophagy, we tested the effects of Gtr1 and Gtr2
mutants that mimic various nucleotide-bound states.23 As a result, autophagy induction in response to
nitrogen-starvation was severely deficient in gtr1∆ gtr2∆ cells
exogenously expressing both the GTP-bound form Gtr1 mutant and wild-type Gtr2, driven from
their native promoters (Fig. 3, column 8). These data
indicate that the GTP-bound Gtr1 mutant exerts a dominant effect on autophagy in the
gtr1∆ background. This effect was also observed in cells expressing
both the GTP-bound Gtr1 mutant and the GDP-bound form of Gtr2 (Fig. 3, column 10). On the other hand, when cells completely lacked
Gtr2, autophagy was induced normally in response to nitrogen-starvation (Fig. 3, column 12). Thus, Gtr2, at least in the GDP form,
appears to be critical for the dominant effects of the GTP-bound Gtr1 mutant. We also
found that vacuolar localization of Gtr16
was disrupted by deletion of Gtr2 (Fig.
S4A). Likewise, deletion of GTR1 disrupted vacuolar localization
of Gtr2 (Fig. S4A). Therefore,
Gtr2 seems to be important for functional localization of Gtr1, and vice versa. We also
noticed that the combination of the GTP-bound forms of Gtr1 and Gtr2 inhibited
starvation-induced autophagy (Fig.
S5A, column 18), but we interpret the role of Gtr2 in another way (see below and
Discussion)Figure 3. A GTP-bound Gtr1 mutant suppresses autophagy in a
Gtr2-dependent manner. gtr1∆ gtr2∆ double-deletion cells (SKY167)
harboring empty vector (-), plasmid encoding GTP-bound mutant Gtr1 (GTP), and/or
plasmid encoding wild-type (WT) or GDP-bound mutant Gtr2 (GDP) and tco89∆
gtr1∆ gtr2∆ triple-mutant cells (SKY277) expressing GTP-bound Gtr1 mutant
(GTP) and GDP-bound Gtr2 mutant (GDP) were grown in SD medium containing 0.5% casamino
acids, incubated under nitrogen-starvation conditions for 3 h, and then subjected to
ALP assays. Data represent means ± standard deviation from 3 independent experiments.
The ALP activities of wild-type (SKY084) and pep4∆ (SKY100) cells
harboring empty vector were also measured as control samples.To determine whether TORC1 played a role in these effects, we monitored autophagy in
cells lacking the nonessential TORC1 subunit Tco89.24 The tco89 mutant cells expressing GTP-bound
Gtr1 and GDP-bound Gtr2 mutants exhibited starvation-induced autophagy at levels
comparable to those in wild-type cells (Fig. 3,
column 14). These data indicate that GTP-bound Gtr1 causes defects in autophagy by
activating TORC1 even under nitrogen-starvation conditions.

GTP-bound Gtr1 suppresses autophagy under nitrogen starvation

Our results suggested that Gtr1 is involved in the regulation of autophagy. Unexpectedly,
however, deletion of neither GTR1 nor GTR2 had any
effect on induction of autophagy (Fig. 1D and Fig. S1A, see ref. 9). Therefore, to ask whether the Gtr1-Gtr2 complex
played a role in the regulation of autophagy, we tested the effects of Gtr1 and Gtr2
mutants that mimic various nucleotide-bound states.23 As a result, autophagy induction in response to
nitrogen-starvation was severely deficient in gtr1∆ gtr2∆ cells
exogenously expressing both the GTP-bound form Gtr1 mutant and wild-type Gtr2, driven from
their native promoters (Fig. 3, column 8). These data
indicate that the GTP-bound Gtr1 mutant exerts a dominant effect on autophagy in the
gtr1∆ background. This effect was also observed in cells expressing
both the GTP-bound Gtr1 mutant and the GDP-bound form of Gtr2 (Fig. 3, column 10). On the other hand, when cells completely lacked
Gtr2, autophagy was induced normally in response to nitrogen-starvation (Fig. 3, column 12). Thus, Gtr2, at least in the GDP form,
appears to be critical for the dominant effects of the GTP-bound Gtr1 mutant. We also
found that vacuolar localization of Gtr16
was disrupted by deletion of Gtr2 (Fig.
S4A). Likewise, deletion of GTR1 disrupted vacuolar localization
of Gtr2 (Fig. S4A). Therefore,
Gtr2 seems to be important for functional localization of Gtr1, and vice versa. We also
noticed that the combination of the GTP-bound forms of Gtr1 and Gtr2 inhibited
starvation-induced autophagy (Fig.
S5A, column 18), but we interpret the role of Gtr2 in another way (see below and
Discussion)

Figure 3. A GTP-bound Gtr1 mutant suppresses autophagy in a
Gtr2-dependent manner. gtr1∆ gtr2∆ double-deletion cells (SKY167)
harboring empty vector (-), plasmid encoding GTP-bound mutant Gtr1 (GTP), and/or
plasmid encoding wild-type (WT) or GDP-bound mutant Gtr2 (GDP) and tco89∆
gtr1∆ gtr2∆ triple-mutant cells (SKY277) expressing GTP-bound Gtr1 mutant
(GTP) and GDP-bound Gtr2 mutant (GDP) were grown in SD medium containing 0.5% casamino
acids, incubated under nitrogen-starvation conditions for 3 h, and then subjected to
ALP assays. Data represent means ± standard deviation from 3 independent experiments.
The ALP activities of wild-type (SKY084) and pep4∆ (SKY100) cells
harboring empty vector were also measured as control samples.

To determine whether TORC1 played a role in these effects, we monitored autophagy in
cells lacking the nonessential TORC1 subunit Tco89.24 The tco89 mutant cells expressing GTP-bound
Gtr1 and GDP-bound Gtr2 mutants exhibited starvation-induced autophagy at levels
comparable to those in wild-type cells (Fig. 3,
column 14). These data indicate that GTP-bound Gtr1 causes defects in autophagy by
activating TORC1 even under nitrogen-starvation conditions.

Tor1 vacuolar localization is increased in both GTP-bound Gtr1 and
npr2∆ mutantsNext, we observed the localization of N-terminally GFP-tagged Tor1, expressed from the
genome under the control of the native promoter. This GFP-Tor1 best replicated the native
Tor1 function among several constructs including previously reported ones (Fig. S4B). As previously reported,
GFP-Tor1 was detected on vacuolar membranes, and some puncta associated with vacuolar
membranes under nutrient-rich conditions (Fig. 4A).6,25 The identity of these perivacuolar puncta
remains obscure; they did not colocalize with markers of other compartments such as the
phagophore assembly site (Ape1-mCherry) or the endosome (Snf7-mCherry) (data not shown).
Deletion of GTR1 or GTR2 significantly reduced the
vacuolar localization of Tor1 (Fig. 4A and
B, columns 5 to 10), although perivacuolar localization of Tor1 persisted. The
expression level of native Tor1 was not affected in these mutants (Fig. S4C). Thus, Gtr1 and Gtr2
appear to maintain Tor1 localization on the vacuoles.Figure 4. Vacuolar localization of Tor1 is regulated by Gtr1, Gtr2,
Npr2, and nutrient availability. (A) Strains expressing N-terminally
GFP-tagged Tor1 under the control of the native promoter (WT, SKY222;
npr2∆, SKY226; gtr1∆, SKY278;
gtr2∆, SKY279; gtr1∆ gtr2∆, SKY299) were incubated
in SD medium containing 0.5% casamino acids and 100 µM CMAC for 30 min, and either
washed once with SD medium containing 0.5% casamino acids or starved for nitrogen
(SD-N) for 1 to 2 h, and then subjected to fluorescence microscopy. Scale bar: 5 μm.
(B) Percentage of cells in (A) exhibiting vacuolar
localization of Tor1. Cells with GFP-Tor1 signals surrounding vacuoles stained by CMAC
were counted. More than 100 cells were counted. Data represent means ± standard
deviation from 3 independent experiments. (C) Strains expressing
N-terminally GFP-tagged Tor1 (gtr1∆ gtr2∆, SKY299; gtr1∆
gtr2∆ npr2∆, SKY320) and expressing the indicated combinations of Gtr1 and
Gtr2 mutants were subjected to fluorescence microscopy under the same conditions as in
(A). Percentage of cells exhibiting vacuolar localization of Tor1 is
presented in (B).We noticed that the number of cells with vacuolar localization of Tor1 was reduced in
response to starvation conditions, although a significant fraction of cells (39%) still
exhibited vacuolar localization of Tor1 (Fig. 4A and
B, columns 1 and 2). The GTP-bound Gtr1 mutants also increased vacuolar
localization of GFP-Tor1 (Fig. 4C, columns 7 and 8).
For this increase to occur, Gtr2 was needed to be in the GDP-bound form rather than the
GTP-bound form (Fig. 4C, columns 5, 6, 9, and 10).
Thus, dynamics of Tor1 localization are regulated by Gtr1 and Gtr2.Furthermore, we found that deletion of NPR2 increased the cells with
vacuolar localization of Tor1 (Fig. 4A and B, columns
3 and 4; Fig. 4C, columns 3 and 4). Thus, the
npr2∆ mutant is similar to the GTP-bound Gtr1 mutant with respect to
Tor1 localization and defects in induction of autophagy. These results support the model
that the npr2∆ mutant is defective in hydrolysis of GTP bound to
Gtr1.

Tor1 vacuolar localization is increased in both GTP-bound Gtr1 and
npr2∆ mutants

Next, we observed the localization of N-terminally GFP-tagged Tor1, expressed from the
genome under the control of the native promoter. This GFP-Tor1 best replicated the native
Tor1 function among several constructs including previously reported ones (Fig. S4B). As previously reported,
GFP-Tor1 was detected on vacuolar membranes, and some puncta associated with vacuolar
membranes under nutrient-rich conditions (Fig. 4A).6,25 The identity of these perivacuolar puncta
remains obscure; they did not colocalize with markers of other compartments such as the
phagophore assembly site (Ape1-mCherry) or the endosome (Snf7-mCherry) (data not shown).
Deletion of GTR1 or GTR2 significantly reduced the
vacuolar localization of Tor1 (Fig. 4A and
B, columns 5 to 10), although perivacuolar localization of Tor1 persisted. The
expression level of native Tor1 was not affected in these mutants (Fig. S4C). Thus, Gtr1 and Gtr2
appear to maintain Tor1 localization on the vacuoles.

Figure 4. Vacuolar localization of Tor1 is regulated by Gtr1, Gtr2,
Npr2, and nutrient availability. (A) Strains expressing N-terminally
GFP-tagged Tor1 under the control of the native promoter (WT, SKY222;
npr2∆, SKY226; gtr1∆, SKY278;
gtr2∆, SKY279; gtr1∆ gtr2∆, SKY299) were incubated
in SD medium containing 0.5% casamino acids and 100 µM CMAC for 30 min, and either
washed once with SD medium containing 0.5% casamino acids or starved for nitrogen
(SD-N) for 1 to 2 h, and then subjected to fluorescence microscopy. Scale bar: 5 μm.
(B) Percentage of cells in (A) exhibiting vacuolar
localization of Tor1. Cells with GFP-Tor1 signals surrounding vacuoles stained by CMAC
were counted. More than 100 cells were counted. Data represent means ± standard
deviation from 3 independent experiments. (C) Strains expressing
N-terminally GFP-tagged Tor1 (gtr1∆ gtr2∆, SKY299; gtr1∆
gtr2∆ npr2∆, SKY320) and expressing the indicated combinations of Gtr1 and
Gtr2 mutants were subjected to fluorescence microscopy under the same conditions as in
(A). Percentage of cells exhibiting vacuolar localization of Tor1 is
presented in (B).

We noticed that the number of cells with vacuolar localization of Tor1 was reduced in
response to starvation conditions, although a significant fraction of cells (39%) still
exhibited vacuolar localization of Tor1 (Fig. 4A and
B, columns 1 and 2). The GTP-bound Gtr1 mutants also increased vacuolar
localization of GFP-Tor1 (Fig. 4C, columns 7 and 8).
For this increase to occur, Gtr2 was needed to be in the GDP-bound form rather than the
GTP-bound form (Fig. 4C, columns 5, 6, 9, and 10).
Thus, dynamics of Tor1 localization are regulated by Gtr1 and Gtr2.

Furthermore, we found that deletion of NPR2 increased the cells with
vacuolar localization of Tor1 (Fig. 4A and B, columns
3 and 4; Fig. 4C, columns 3 and 4). Thus, the
npr2∆ mutant is similar to the GTP-bound Gtr1 mutant with respect to
Tor1 localization and defects in induction of autophagy. These results support the model
that the npr2∆ mutant is defective in hydrolysis of GTP bound to
Gtr1.

The GDP-bound Gtr1 mutant induces autophagy even under nutrient-rich
conditionsNext, we investigated the phenotype of a Gtr1 mutant mimicking a permanently GDP-bound
state. Cells expressing the GDP-bound Gtr1 mutant induced autophagy even under
nutrient-rich conditions (Fig. 5A, columns 7 and 9),
but this phenotype was abrogated in cells lacking Gtr2 (Fig. 5A, column 13). Autophagic induction was still observed when wild-type Gtr2
was replaced with the GTP-bound mutant, but absent when Gtr2 was replaced by the GDP-bound
form (Fig. 5A, columns 9 and 11). Thus, GTP-bound
Gtr2 is required for the dominant effect of the GDP-bound Gtr1 mutant. Autophagic
induction by this dominant effect was not affected by the deletion of
NPR2 (Fig.
S5A, columns 27 and 29), supporting the idea that Npr2 functions upstream of
Gtr1-Gtr2.Figure 5. Gtr2 interacts with Kog1 and inactivates TORC1.
(A) Wild-type (SKY084), pep4∆ (SKY100), and
gtr1∆ gtr2∆ double-mutant cells harboring empty vector (-), and
gtr1∆ gtr2∆ cells harboring the GDP-bound mutant Gtr1 (top row:
GDP) along with wild-type, GTP-bound mutant, or GDP-bound mutant Gtr2 (bottom row: WT,
GTP, and GDP, respectively), were either grown in SD medium supplemented with 0.5%
casamino acids or starved of nitrogen for 3 h, and then subjected to ALP assays. Data
represent means ± standard deviation from 3 independent experiments. (B)
Cells of the strain for yeast 2-hybrid analysis (PJ69-4A) harboring pGBD-C1 vector
encoding wild-type, GTP-bound mutant, GDP-bound mutant, or E62K mutant Gtr2 (pSK150,
pSK151, pSK152, or pSK222, respectively) and pGAD-C1/Kog1 (pSK156) were serially
10-fold diluted and spotted onto SC medium plates either lacking histidine and leucine
or histidine, leucine, and uracil. Results after 2 d of culture at 30 °C are shown.
(C) Cells endogenously expressing TAP-tagged Kog1 and harboring
multicopy vector encoding wild-type, GTP-bound mutant, GDP-bound mutant, or E62K
mutant Gtr2 were subjected to TAP affinity isolation assays. The lysates and the
affinity isolates were subjected to western blot analysis with anti-protein A and
anti-GST antibodies. (D) Strains (wild-type, SKY084;
pep4∆, SKY100; and gtr1∆ gtr2∆, SKY167) harboring
empty vector (-), and gtr1∆ gtr2∆ (SKY167) harboring GDP-bound Gtr1
mutant (GDP) along wild-type (WT) or E62K mutant Gtr2E62K, were either
grown in SD medium supplemented with 0.5% casamino acids or starved of nitrogen for 3
h, and then subjected to ALP assays. Data represent means ± standard deviation from 3
independent experiments.

The GDP-bound Gtr1 mutant induces autophagy even under nutrient-rich
conditions

Next, we investigated the phenotype of a Gtr1 mutant mimicking a permanently GDP-bound
state. Cells expressing the GDP-bound Gtr1 mutant induced autophagy even under
nutrient-rich conditions (Fig. 5A, columns 7 and 9),
but this phenotype was abrogated in cells lacking Gtr2 (Fig. 5A, column 13). Autophagic induction was still observed when wild-type Gtr2
was replaced with the GTP-bound mutant, but absent when Gtr2 was replaced by the GDP-bound
form (Fig. 5A, columns 9 and 11). Thus, GTP-bound
Gtr2 is required for the dominant effect of the GDP-bound Gtr1 mutant. Autophagic
induction by this dominant effect was not affected by the deletion of
NPR2 (Fig.
S5A, columns 27 and 29), supporting the idea that Npr2 functions upstream of
Gtr1-Gtr2.

Figure 5. Gtr2 interacts with Kog1 and inactivates TORC1.
(A) Wild-type (SKY084), pep4∆ (SKY100), and
gtr1∆ gtr2∆ double-mutant cells harboring empty vector (-), and
gtr1∆ gtr2∆ cells harboring the GDP-bound mutant Gtr1 (top row:
GDP) along with wild-type, GTP-bound mutant, or GDP-bound mutant Gtr2 (bottom row: WT,
GTP, and GDP, respectively), were either grown in SD medium supplemented with 0.5%
casamino acids or starved of nitrogen for 3 h, and then subjected to ALP assays. Data
represent means ± standard deviation from 3 independent experiments. (B)
Cells of the strain for yeast 2-hybrid analysis (PJ69-4A) harboring pGBD-C1 vector
encoding wild-type, GTP-bound mutant, GDP-bound mutant, or E62K mutant Gtr2 (pSK150,
pSK151, pSK152, or pSK222, respectively) and pGAD-C1/Kog1 (pSK156) were serially
10-fold diluted and spotted onto SC medium plates either lacking histidine and leucine
or histidine, leucine, and uracil. Results after 2 d of culture at 30 °C are shown.
(C) Cells endogenously expressing TAP-tagged Kog1 and harboring
multicopy vector encoding wild-type, GTP-bound mutant, GDP-bound mutant, or E62K
mutant Gtr2 were subjected to TAP affinity isolation assays. The lysates and the
affinity isolates were subjected to western blot analysis with anti-protein A and
anti-GST antibodies. (D) Strains (wild-type, SKY084;
pep4∆, SKY100; and gtr1∆ gtr2∆, SKY167) harboring
empty vector (-), and gtr1∆ gtr2∆ (SKY167) harboring GDP-bound Gtr1
mutant (GDP) along wild-type (WT) or E62K mutant Gtr2E62K, were either
grown in SD medium supplemented with 0.5% casamino acids or starved of nitrogen for 3
h, and then subjected to ALP assays. Data represent means ± standard deviation from 3
independent experiments.

The TORC1 subunit Kog1 is an effector of Gtr2Based on these results, we investigated the role of Gtr2 in the autophagic induction
dependent on GDP-bound Gtr1. In a 2-hybrid analysis, we found that Gtr2 binds to Kog1
(Fig. 5B), a subunit of TORC1 and a mammalian
ortholog of RAPTOR.26 Kog1 also strongly
bound to the GTP-bound Gtr2 mutant, but not to the GDP-bound mutant (Fig. 5B). To confirm this interaction biochemically, we performed
affinity isolation assays. Lysates of Kog1-TAP-expressing cells were precipitated by IgG
beads, and the affinity isolated fractions were subjected to western blotting. As shown in
Figure 5C, wild-type GST-Gtr2 was precipitated in a
Kog1-TAP-dependent manner (relative amount, 21.6); the amount of material pulled down in
the specific precipitation was significantly larger than the amount pulled down by
nonspecific binding, possibly to the resin (see control lane; relative amount 5.2), which
could not be entirely prevented. The GTP-bound form of GST-Gtr2 was also efficiently
precipitated, whereas the GDP-bound form of GST-Gtr2 was precipitated to a lesser extent
(Fig. 5C). These data indicate that Kog1 is a
direct effector of Gtr2.

The TORC1 subunit Kog1 is an effector of Gtr2

Based on these results, we investigated the role of Gtr2 in the autophagic induction
dependent on GDP-bound Gtr1. In a 2-hybrid analysis, we found that Gtr2 binds to Kog1
(Fig. 5B), a subunit of TORC1 and a mammalian
ortholog of RAPTOR.26 Kog1 also strongly
bound to the GTP-bound Gtr2 mutant, but not to the GDP-bound mutant (Fig. 5B). To confirm this interaction biochemically, we performed
affinity isolation assays. Lysates of Kog1-TAP-expressing cells were precipitated by IgG
beads, and the affinity isolated fractions were subjected to western blotting. As shown in
Figure 5C, wild-type GST-Gtr2 was precipitated in a
Kog1-TAP-dependent manner (relative amount, 21.6); the amount of material pulled down in
the specific precipitation was significantly larger than the amount pulled down by
nonspecific binding, possibly to the resin (see control lane; relative amount 5.2), which
could not be entirely prevented. The GTP-bound form of GST-Gtr2 was also efficiently
precipitated, whereas the GDP-bound form of GST-Gtr2 was precipitated to a lesser extent
(Fig. 5C). These data indicate that Kog1 is a
direct effector of Gtr2.

TORC1 is inactivated by binding to Gtr2A recent structural analysis of Gtr1 revealed the critical amino acids on its
effector-binding surface.27 Based on its
structural similarity to Gtr1, Gtr2 is predicted to bind to Kog1 via a similar surface.
Accordingly, we made 9 point-mutants on the possible surface area of Gtr2; in one of these
mutants, glutamine 62 of Gtr2, which corresponds to Gtr1 asparagine 61, was mutated to
lysine. This E62K mutation of Gtr2 abolished the interaction to Kog1, as demonstrated by
yeast 2-hybrid analysis (Fig. 5B). Furthermore,
coprecipitation with Kog1-TAP was severely defective in the Gtr2E62K mutant,
although the expression level of the E62K mutant was not affected (Fig. 5C). Using the E62K mutant, we asked whether the binding of Gtr2 to
Kog1 is required for autophagic induction by expression of the GDP-bound form Gtr1.
Indeed, the autophagic induction was abrogated in the Gtr2E62K mutant (Fig. 5D, columns 7 and 9). Collectively, these data
indicate that direct binding of Gtr2 to Kog1 inactivates TORC1 and eventually induces
autophagy, even under nutrient-rich conditions.

TORC1 is inactivated by binding to Gtr2

A recent structural analysis of Gtr1 revealed the critical amino acids on its
effector-binding surface.27 Based on its
structural similarity to Gtr1, Gtr2 is predicted to bind to Kog1 via a similar surface.
Accordingly, we made 9 point-mutants on the possible surface area of Gtr2; in one of these
mutants, glutamine 62 of Gtr2, which corresponds to Gtr1 asparagine 61, was mutated to
lysine. This E62K mutation of Gtr2 abolished the interaction to Kog1, as demonstrated by
yeast 2-hybrid analysis (Fig. 5B). Furthermore,
coprecipitation with Kog1-TAP was severely defective in the Gtr2E62K mutant,
although the expression level of the E62K mutant was not affected (Fig. 5C). Using the E62K mutant, we asked whether the binding of Gtr2 to
Kog1 is required for autophagic induction by expression of the GDP-bound form Gtr1.
Indeed, the autophagic induction was abrogated in the Gtr2E62K mutant (Fig. 5D, columns 7 and 9). Collectively, these data
indicate that direct binding of Gtr2 to Kog1 inactivates TORC1 and eventually induces
autophagy, even under nutrient-rich conditions.

DiscussionIn this study, we elucidated the mechanism by which Gtr1-Gtr2 regulates autophagy via
TORC1. Npr2-Npr3 functions upstream of Gtr1-Gtr2, and the npr2∆ mutant
replicates the phenotypes of the GTP-bound Gtr1 mutant with respect to both autophagy and
Tor1 localization. Based on these findings, we reasoned that the function of Npr2-Npr3 is
closely linked to the hydrolysis of GTP in Gtr1. Indeed, it has recently been reported that
Iml1, a subunit of the SEA complex including Npr2 and Npr3,17 possesses GAP (GTPase activating protein) activity toward
Gtr1.28 In mammals, a protein complex
called GATOR1, which contains NPRL2 and NPRL3, exert GAP activity toward RRAGA/B.29 Therefore, our results are consistent with
the idea that the Npr2-Npr3 complex is required for GAP activity toward Gtr1. In addition,
our results demonstrate that Gtr2 directly binds the TORC1 subunit Kog1, resulting in
inactivation of TORC1. These 2 molecular mechanisms are involved in TORC1 inactivation and
autophagic induction.In mammals, RRAGB activates MTORC1 through direct binding to RAPTOR.13 In yeast, Gtr1 is believed to bind to the Kog1 in the same
manner, although direct binding has not been confirmed.6 Our results showed that GTP-bound Gtr2 is an inhibitor, whereas
GTP-bound Gtr1 is an activator, of TORC1 (Fig. 5). What
happens when both Gtr1 and Gtr2 are in the GTP-bound form? In that case, we found that
autophagy was not induced under nutrient-rich conditions and faintly induced in starvation
condition (Fig. S5A, columns 17
and 18). We reasoned that Gtr1-dependent activation and Gtr2-dependent inactivation occur
simultaneously, but the effect of Gtr1 is epistatic to that of Gtr2, at least in regard to
autophagy. The best evidence that Gtr2-dependent inactivation occurs in the cells harboring
both GTP-Gtr1 and GTP-Gtr2 is that it exhibited rapamycin-sensitive growth, indicating
weakened TORC1 function relative to wild-type and Gtr1-GTP Gtr2-GDP cells (Fig. S6). Further, Tor1 vacuolar
localization was decreased in comparison to the Gtr1-GTP Gtr2-GDP cells (Fig. 4C, columns 5 to 8). Thus, there may be some mechanism
that determines whether Gtr1 or Gtr2 will be dominant when both are in GTP-bound form.In mammals, MTOR is localized on the lysosome when amino acids are available, and this
localization depends on RRAGA/B, and RRAGC/D. We showed that Gtr1 and Gtr2 are also
important for Tor1 localization on vacuole in yeast, just as in mammals (Fig. 4A and B). Furthermore, the nucleotide-bound state of
Gtr1-Gtr2 was critical in determining the dynamics of Tor1 vacuolar localization: the
Gtr1-GTP form increased vacuolar localization, whereas the Gtr2-GTP form decreased it (Fig. 4C, columns 5 to 10; Fig. S4C, columns 5, 6, 11, and 12).
Cells in which Tor1 was localized to the vacuole became less abundant in response to
starvation (Fig. 4A and B). Via regulation of Gtr1,
Npr2 seems to play a critical role in Tor1 dynamics. This Npr2-dependent regulation must be
important, because even in mammals, NPRL2 and NPRL3 regulate MTOR localization (Fig. 2A–C; Fig. S3A and S3B). Where does this inactivation occur? Despite the
dynamic changes in Tor1 localization in response to starvation, a significant fraction of
Tor1 remained on the vacuole under nitrogen-starvation (Fig. 4A and B), as in the case of Gtr2 (our unpublished observation). Based on our
findings that Gtr2 binds to Kog1 (Fig. 5B and
C), it is possible that Gtr2 interacts with TORC1 on the vacuole and/or the
perivacuolar dot.A similar dominant effect of the GDP-bound mutant of Gtr1 is also observed in mammalian
RRAGB: overexpression of RRAGB-GDP inactivates MTORC1 irrespective of the availability of
amino acids. This RRAGB-GDP-dependent dominant effect has also been observed in cells
overexpressing GTP-bound RRAGD, but not GDP-bound RRAGD.30,31
Therefore it is possible that RRAGD negatively regulates MTORC1, as we have observed in
yeast. However, the target is unlikely to be a component of MTORC1: the GDP-bound form of
RRAGA/B in complex with the wild-type or GTP-bound form of RRAGC fails to bind RPTOR.32,33 Moreover, under starvation conditions, MTORC1 is not located on
lysosomes, but is instead dispersed throughout the cytosol, whereas wild-type and GTP-bound
mutant RRAGD were detected on lysosomes irrespective of the availability of amino
acids.12 SH3BP4, which binds to RRAG
proteins when MTOR is not localized on lysosomes,34 is one of the possible targets of RRAGD, and there may exist some
mechanism by which such a target could inactivate TORC1.Regardless of the findings described above, it is noteworthy that deletion of neither
GTR1 nor GTR2 markedly affected autophagy induced by
nitrogen-starvation conditions (Fig. 1D; Fig. S1A).9 Thus, autophagy can be induced under nitrogen-starvation by
an as-yet-unidentified signaling pathway independent of the Npr2-Npr3 and Gtr1-Gtr2 axis.
Expression of the GTP- or GDP-bound mutants of Gtr1-Gtr2 supersedes this unknown
Gtr1-Gtr2-independent pathway in the regulation of TORC1 and autophagy by locking TORC1 in
an activated or inactivated state irrespective of upstream signals. We reasoned that
deletion of NPR2 or NPR3 could provide these effects. One
candidate for such an Npr2-Npr3 and Gtr1-Gtr2-independent pathway is the cAMP-dependent
protein kinase pathway, which plays a role in autophagic regulation.2,35,36
Alternatively, Gcn4-dependent amino-acid metabosim may be involved, although that pathway is
also distinct from the effects of nitrogen starvation.37 In addition, our results do not necessarily support a model in
which Npr2-Npr3 complexes are regulated by nitrogen availability. Rather, our data are
consistent with the possibility that the Npr2-Npr3 and Gtr1-Gtr2-dependent pathway is
regulated by other factors.38 Indeed,
autophagy and TORC1 activity are regulated by variety of environmental factors, including a
lack of carbon sources or sulfates.38,39 A
recent study shows that the Npr2-Npr3-Iml1 complex is regulated by sulfur-containing amino
acids.40 Uncovering the relationship
between Npr2-Npr3 and various environmental stimuli will help to elucidate the functional
and regulatory mechanisms of TORC1.From these observations, Gtr1-Gtr2-dependent inactivation of TORC1 and induction of
autophagy occurs when the Gtr2 is able to adopt its GTP form. In that case, which molecule
is responsible for exchanging GDP for GTP in Gtr2? It is unlikely that Npr2-Npr3 is directly
involved in this exchange. Expression of Gtr1-GDP caused induction of autophagy when Gtr2 is
in the WT or GTP form, even under nutrient-rich conditions, suggesting that Gtr2 can adopt
its GTP form in a cell expressing Gtr1-GDP (Fig. 5A,
columns 7 and 9; Fig. S5A, columns
19 and 23). If Npr2 is involved in the exchange of GDP for GTP in Gtr2, Gtr2 is not able to
adopt its GTP form, and autophagy will not be induced in a npr2∆ cell
expressing Gtr1-GDP. However, in npr2∆ cells expressing Gtr1-GDP and
Gtr2-WT, autophagy was induced even under nutrient-rich conditions (Fig. S5A, column 29). The same results
were obtained in npr3∆ and npr2∆ npr3∆ cells (data not
shown). Some unknown mechanism seems to be involved in these phenomena, and this issue
should be addressed in future work.Hyperactivation of TORC1 has been observed in cancer cells.41,42
Intriguingly, NPRL2 has been reported to be a tumor suppressor.43,44 Furthermore, mice lacking NPRL3 die between 15 d after conception
and birth, and these animals exhibit developmental defects in the heart.45 Mice bearing only the permanently GTP-bound
form of RRAGA die during a neonatal period of starvation,46 underscoring the physiologic requirement for proper GTP and GDP
cycling by RRAGA. Our findings will help to provide a foundation for research into clinical
approaches that target TORC1 and autophagy.

In this study, we elucidated the mechanism by which Gtr1-Gtr2 regulates autophagy via
TORC1. Npr2-Npr3 functions upstream of Gtr1-Gtr2, and the npr2∆ mutant
replicates the phenotypes of the GTP-bound Gtr1 mutant with respect to both autophagy and
Tor1 localization. Based on these findings, we reasoned that the function of Npr2-Npr3 is
closely linked to the hydrolysis of GTP in Gtr1. Indeed, it has recently been reported that
Iml1, a subunit of the SEA complex including Npr2 and Npr3,17 possesses GAP (GTPase activating protein) activity toward
Gtr1.28 In mammals, a protein complex
called GATOR1, which contains NPRL2 and NPRL3, exert GAP activity toward RRAGA/B.29 Therefore, our results are consistent with
the idea that the Npr2-Npr3 complex is required for GAP activity toward Gtr1. In addition,
our results demonstrate that Gtr2 directly binds the TORC1 subunit Kog1, resulting in
inactivation of TORC1. These 2 molecular mechanisms are involved in TORC1 inactivation and
autophagic induction.

In mammals, RRAGB activates MTORC1 through direct binding to RAPTOR.13 In yeast, Gtr1 is believed to bind to the Kog1 in the same
manner, although direct binding has not been confirmed.6 Our results showed that GTP-bound Gtr2 is an inhibitor, whereas
GTP-bound Gtr1 is an activator, of TORC1 (Fig. 5). What
happens when both Gtr1 and Gtr2 are in the GTP-bound form? In that case, we found that
autophagy was not induced under nutrient-rich conditions and faintly induced in starvation
condition (Fig. S5A, columns 17
and 18). We reasoned that Gtr1-dependent activation and Gtr2-dependent inactivation occur
simultaneously, but the effect of Gtr1 is epistatic to that of Gtr2, at least in regard to
autophagy. The best evidence that Gtr2-dependent inactivation occurs in the cells harboring
both GTP-Gtr1 and GTP-Gtr2 is that it exhibited rapamycin-sensitive growth, indicating
weakened TORC1 function relative to wild-type and Gtr1-GTP Gtr2-GDP cells (Fig. S6). Further, Tor1 vacuolar
localization was decreased in comparison to the Gtr1-GTP Gtr2-GDP cells (Fig. 4C, columns 5 to 8). Thus, there may be some mechanism
that determines whether Gtr1 or Gtr2 will be dominant when both are in GTP-bound form.

In mammals, MTOR is localized on the lysosome when amino acids are available, and this
localization depends on RRAGA/B, and RRAGC/D. We showed that Gtr1 and Gtr2 are also
important for Tor1 localization on vacuole in yeast, just as in mammals (Fig. 4A and B). Furthermore, the nucleotide-bound state of
Gtr1-Gtr2 was critical in determining the dynamics of Tor1 vacuolar localization: the
Gtr1-GTP form increased vacuolar localization, whereas the Gtr2-GTP form decreased it (Fig. 4C, columns 5 to 10; Fig. S4C, columns 5, 6, 11, and 12).
Cells in which Tor1 was localized to the vacuole became less abundant in response to
starvation (Fig. 4A and B). Via regulation of Gtr1,
Npr2 seems to play a critical role in Tor1 dynamics. This Npr2-dependent regulation must be
important, because even in mammals, NPRL2 and NPRL3 regulate MTOR localization (Fig. 2A–C; Fig. S3A and S3B). Where does this inactivation occur? Despite the
dynamic changes in Tor1 localization in response to starvation, a significant fraction of
Tor1 remained on the vacuole under nitrogen-starvation (Fig. 4A and B), as in the case of Gtr2 (our unpublished observation). Based on our
findings that Gtr2 binds to Kog1 (Fig. 5B and
C), it is possible that Gtr2 interacts with TORC1 on the vacuole and/or the
perivacuolar dot.

A similar dominant effect of the GDP-bound mutant of Gtr1 is also observed in mammalian
RRAGB: overexpression of RRAGB-GDP inactivates MTORC1 irrespective of the availability of
amino acids. This RRAGB-GDP-dependent dominant effect has also been observed in cells
overexpressing GTP-bound RRAGD, but not GDP-bound RRAGD.30,31
Therefore it is possible that RRAGD negatively regulates MTORC1, as we have observed in
yeast. However, the target is unlikely to be a component of MTORC1: the GDP-bound form of
RRAGA/B in complex with the wild-type or GTP-bound form of RRAGC fails to bind RPTOR.32,33 Moreover, under starvation conditions, MTORC1 is not located on
lysosomes, but is instead dispersed throughout the cytosol, whereas wild-type and GTP-bound
mutant RRAGD were detected on lysosomes irrespective of the availability of amino
acids.12 SH3BP4, which binds to RRAG
proteins when MTOR is not localized on lysosomes,34 is one of the possible targets of RRAGD, and there may exist some
mechanism by which such a target could inactivate TORC1.

Regardless of the findings described above, it is noteworthy that deletion of neither
GTR1 nor GTR2 markedly affected autophagy induced by
nitrogen-starvation conditions (Fig. 1D; Fig. S1A).9 Thus, autophagy can be induced under nitrogen-starvation by
an as-yet-unidentified signaling pathway independent of the Npr2-Npr3 and Gtr1-Gtr2 axis.
Expression of the GTP- or GDP-bound mutants of Gtr1-Gtr2 supersedes this unknown
Gtr1-Gtr2-independent pathway in the regulation of TORC1 and autophagy by locking TORC1 in
an activated or inactivated state irrespective of upstream signals. We reasoned that
deletion of NPR2 or NPR3 could provide these effects. One
candidate for such an Npr2-Npr3 and Gtr1-Gtr2-independent pathway is the cAMP-dependent
protein kinase pathway, which plays a role in autophagic regulation.2,35,36
Alternatively, Gcn4-dependent amino-acid metabosim may be involved, although that pathway is
also distinct from the effects of nitrogen starvation.37 In addition, our results do not necessarily support a model in
which Npr2-Npr3 complexes are regulated by nitrogen availability. Rather, our data are
consistent with the possibility that the Npr2-Npr3 and Gtr1-Gtr2-dependent pathway is
regulated by other factors.38 Indeed,
autophagy and TORC1 activity are regulated by variety of environmental factors, including a
lack of carbon sources or sulfates.38,39 A
recent study shows that the Npr2-Npr3-Iml1 complex is regulated by sulfur-containing amino
acids.40 Uncovering the relationship
between Npr2-Npr3 and various environmental stimuli will help to elucidate the functional
and regulatory mechanisms of TORC1.

From these observations, Gtr1-Gtr2-dependent inactivation of TORC1 and induction of
autophagy occurs when the Gtr2 is able to adopt its GTP form. In that case, which molecule
is responsible for exchanging GDP for GTP in Gtr2? It is unlikely that Npr2-Npr3 is directly
involved in this exchange. Expression of Gtr1-GDP caused induction of autophagy when Gtr2 is
in the WT or GTP form, even under nutrient-rich conditions, suggesting that Gtr2 can adopt
its GTP form in a cell expressing Gtr1-GDP (Fig. 5A,
columns 7 and 9; Fig. S5A, columns
19 and 23). If Npr2 is involved in the exchange of GDP for GTP in Gtr2, Gtr2 is not able to
adopt its GTP form, and autophagy will not be induced in a npr2∆ cell
expressing Gtr1-GDP. However, in npr2∆ cells expressing Gtr1-GDP and
Gtr2-WT, autophagy was induced even under nutrient-rich conditions (Fig. S5A, column 29). The same results
were obtained in npr3∆ and npr2∆ npr3∆ cells (data not
shown). Some unknown mechanism seems to be involved in these phenomena, and this issue
should be addressed in future work.

Hyperactivation of TORC1 has been observed in cancer cells.41,42
Intriguingly, NPRL2 has been reported to be a tumor suppressor.43,44 Furthermore, mice lacking NPRL3 die between 15 d after conception
and birth, and these animals exhibit developmental defects in the heart.45 Mice bearing only the permanently GTP-bound
form of RRAGA die during a neonatal period of starvation,46 underscoring the physiologic requirement for proper GTP and GDP
cycling by RRAGA. Our findings will help to provide a foundation for research into clinical
approaches that target TORC1 and autophagy.

Materials and MethodsYeast growth mediaYeast cells were grown at 30 °C in YPD (1% yeast extract, 2% peptone, and 2% glucose) or
SD (0.67% yeast nitrogen base and 2% glucose) medium supplemented with amino acids and
0.5% casamino acid. For nitrogen starvation, cells were washed twice with SD-N (0.17%
yeast nitrogen base without amino acids and ammonium sulfate, and 2% glucose) and
resuspended in SD-N. Rapamycin (Sigma, R0395) in stock solution (1 mg/ml ethanol and
Triton X-100 at a ratio of 9:1 [v/v]) was added to YPD to achieve a final concentration of
0.2 μg/ml.Plasmid and yeast strainsThe PHO8 locus from the knockout collection was replaced with
pho8∆60 using a synthetic genetic-array method.47 Briefly, the TNY509 parental strain harboring
pho8∆60 was crossed with each mutant strain on YPD in a rectangular
plate (OmniTray, Nunc) using a 96-pin replicator (V&P Scientific, VP408). Haploid
cells containing the pho8∆60 allele and one of the mutant alleles were
selected using auxotrophic and drug-resistance markers and sequential replica
plating.15,48Yeast strains and plasmids used in this study, except for strains made using the
synthetic genetic-array method, are listed in Table
1 and Table 2. All gene disruptions,
tagged constructs, and other modifications were confirmed by PCR.53-56 N-terminal GFP-tagged Tor1 strains were generated by integrating
SpeI-digested pSK108, generated by cloning a fragment containing
PTOR1 (1000 bases)-GFP-TOR1 (426 N-terminal bases),
created in a 2-step PCR, into pRS305 using the SacI and
ApaI sites. An internally 3 × GFP-tagged Tor1 strain was generated
based on a previous report.25
GTR1 or GTR2 fragments containing approximately 1,000
bases of the promoter regions, the open reading frame, and 500 bases from the terminal
noncoding region were digested with SacI and XbaI or
BamHI and XhoI, respectively, and cloned into the
single-copy vector pRS316. GST-GTR2 fragments, in which the
GST coding region was integrated into the N terminus of
GTR2, were amplified by PCR from the yeast genome, digested with
NotI and EcoRI, and cloned into the multicopy vector
pRS426. GTP- or GDP-bound GTR1 and GTR2 mutants, and
other point mutants of GTR2, were generated using primer-based 2-step
PCRs, based on previous reports.23
ATG13 fragments, containing approximately 1000 bases of the promoter
regions and the open reading frame, were digested with SalI and
HindIII and cloned into pRS426.Table 1. Strains used in this studyStrainGenotypeReferenceTNY509MATα,
pho8Δ60::NatMX4Δcan1::
MFA1pr-HIS3-MF 1pr-LEU2
ura3Δleu2Δhis3Δmet15Δlys2Δ49BY4741MATa,
his3D1leu2D0met15D0ura3D050BY4742MATα,
his3D1leu2D0lys2D0ura3D050SKY084[BY4741]
pho8D60::NatMX4This studySKY100[SKY084]
pep4Δ::kanMX6This studySKY091[SKY084]
npr2Δ::kanMX6This studySKY131[SKY084]
npr3Δ::hphMX4This studySKY264[SKY084]
iml1Δ::kanMX6This studySKY244[SKY084]
gtr1Δ::zeoNT3This studySKY246[SKY084]
gtr2Δ::hphMX4This studySKY127[SKY084]
npr2Δ::kanMX6npr3Δ::hphMX4This studySKY167[SKY084]
gtr1Δ::kanMX6
gtr2Δ::hphMX4This studySKY245[SKY084]
gtr1Δ::zeoNT3
npr2Δ::kanMX6This studySKY247[SKY084]
gtr2Δ::hphMX4
npr2Δ::kanMX6This studySKY186[SKY084]
npr2Δ::zeoNT3 gtr1Δ::kanMX6
gtr2Δ::hphMX4This studySKY270[SKY084]
npr3Δ::hphMX4
gtr1Δ::zeoNT3This studySKY298[SKY084]
gtr2Δ::zeoNT3
npr3Δ::hphMX4This studySKY308[SKY084]
gtr2Δ::kanMX6 npr3Δ::hphMX4
gtr1Δ::zeoNT3This studySKY271[SKY084]
npr2Δ::kanMX6 npr3Δ::hphMX4
gtr1Δ::zeoNT3This studySKY291[SKY084]
npr2Δ::kanMX6 npr3Δ::hphMX4
gtr2Δ::zeoNT3This studySKY309[SKY084]
gtr2Δ::LEU2 gtr2Δ::kanMX6
npr3Δ::hphMX4
gtr1Δ::zeoNT3This studySKY277[SKY084]
gtr1Δ::kanMX6 gtr2Δ::hphMX4
tco89Δ::zeoThis studySKY222[BY4741]
LEU2::GFP-TOR1This studySKY226[SKY222]
npr2Δ::natMX4This studySKY278[SKY222]
gtr1Δ::hphMX6This studySKY279[SKY222]
gtr2Δ::hphMX4This studySKY299[SKY222]
gtr2Δ::zeoNT3
gtr1Δ::hphMX4This studySKY135[BY4741]
npr2Δ::natMX4This studyNKR002[BY4741]
ego1Δ::kanMX6EUROSCARFNKR003[BY4741]
tor1Δ::kanMX6EUROSCARFSKY173[BY4742]
Tor1-D330-3xGFPThis studySKY112[BY4741]
GTR1-GFP::hphMX4This studySKY249[BY4741]
GTR1-GFP::hphMX4,
gtr2Δ::natMX4This studySKY259[BY4741]
GTR2-GFP::hphMX4This studySKY260[BY4741] GTR2-GFP::hphMX4
gtr1Δ::zeoNT3This studySKY287[BY4741]
KOG1-TAP::kanMX6This studyPJ69-4AMATa
trpl-901 leu2-3,112 ura3-52 his3-200
ga14Δga180Δ
LYS2::GALl-HIS3GAL2-ADE2
met2::GAL7-lacZ52Table 2. Plasmids used in this studyPlasmidDescriptionReferencepRS316CEN, URA351pRS4262 micron,
URA351pGBD-C12 micron,
TRP152pGAD-C12 micron,
LEU252pSK122[pRS316] Gtr1
Gtr2This studypSK123[pRS316] Gtr1
Gtr2S23LThis studypSK124[pRS316] Gtr1
Gtr2Q66LThis studypSK125[pRS316] Gtr1S20L
Gtr2This studypSK126[pRS316] Gtr1S20L
Gtr2S23LThis studypSK127[pRS316] Gtr1S20L
Gtr2Q66LThis studypSK128[pRS316] Gtr1Q65L
Gtr2This studypSK129[pRS316] Gtr1Q65L
Gtr2S23LThis studypSK130[pRS316] Gtr1Q65L
Gtr2Q66LThis studypSK115[pRS316]
Gtr1S20LThis studypSK116[pRS316]
Gtr1Q65LThis studypSK256[pRS316] Gtr1S20L
Gtr2E62KThis studypSK150[pGBD-C1]
Gtr2This studypSK151[pGBD-C1]
Gtr2Q66LThis studypSK152[pGBD-C1]
Gtr2S23LThis studypSK222[pGBD-C1]
Gtr2E62KThis studypSK156[pGAD-C1]
Kog1This studypSK242[pRS426]
PGTR2-GST-Gtr2This studypSK243[pRS426]
PGTR2-GST-Gtr2Q66LThis studypSK244[pRS426]
PGTR2-GST-Gtr2S23LThis studypSK252[pRS426]
PGTR2-GST-Gtr2E62KThis studypSK048[pRS426]
PATG13-ATG13This studyALP assaysFor large-scale assays, YPD plates were inoculated with cells from each pool using a
96-pin replicator, and then incubated for 16 to 24 h at 30 °C. For conventional assays,
cells were cultured in liquid medium as described above. Subsequently, the cells were
collected by centrifugation, suspended in 200 μl of SD-N medium in 96-well plates, and
incubated for 4 h at 30 °C. The plates were centrifuged, and the supernatant fraction was
discarded. We then added 50 μl of ice-cold lysis buffer (10 mM Tris-Cl, pH 9.0, 10 mM
MgSO4, 10 μM ZnSO4) and approximately 10 μl of 0.6-mm zirconia
silica beads (Biomedical Science, ZS06-0001) to each well. The plates were sealed with
Parafilm and mixed vigorously (2,500/min) on a microplate mixer (Taitec, MBR-022) for 10
min at 4 °C. After a brief centrifugation, the Parafilm was removed, 150 μl of ice-cold
lysis buffer was added, and the plates were centrifuged for 15 min at 490 ×
g at 4 °C. Protein levels were quantified in 50-μl aliquots of the
supernatant fractions using a bicinchoninic acid kit (Nacalai, 06385-00), and the
enzymatic activity in the aliquots was measured as described previously with slight
modifications.15 One unit of ALP
activity was defined as one emission/μg protein/min.MicroscopyFor visualizing vacuolar lumen, 1 OD600 unit of cells were incubated for 30
min at 30 °C in SD medium containing 0.5% casamino acids and 100 mM CMAC fluorescent dye
(Molecular Probes, C2110). Yeast cells were observed Leica AF6500 fluorescent imaging
system (Leica Microsystems, Wetzlar, Germany) mounted on a DIM6000 B microscope (HCX PL
APO 63.6×/1.40–0.60 oil-immersion objective lens, xenon lamp, Leica Microsystems, Wetzlar,
Germany) under the control of LAS-AF software (Leica Microsystems). In Figure S4A, cells were attached on
35-mm glass bottom dishes (Matsunami, D110300) coated with concanavalin A (Sigma, C7275),
and examined using a Leica TCS SP8 confocal system (Leica Microsystems, Wetzlar, Germany)
mounted on a DMI 6000 CS microscope (HCX PL APO 100.6×/1.4 Oil STED objective lens, Leica
Microsystems, Wetzlar, Germany). Mammalian cells were cultured on coverslips and fixed by
incubation in 3% paraformaldehyde in phosphate-buffered saline (PBS; 137 mM NaCl, 2.7 mM
KCl, 10 mM Na2HPO4, 2 mM KH2PO4, pH 7.2) for 20 min. For
immunofluorescence microscopy, the cells were permeabilized with 50 μg/ml digitonin
(Sigma, D141) in PBS for 10 min at room temperature. The permeabilized cells were
incubated in blocking solution (0.1% gelatin in PBS), and then stained with anti-MTOR
(Cell Signaling Technology, 7C10; 2983) and LAMP1 (Santa Cruz Biotechnology, Inc., H4A3;
sc-20011). The coverslips were mounted in SlowFade Gold Reagent (Invitrogen, S36936), and
images were obtained using an Olympus FV1000 laser-scanning confocal microscope (Olympus,
Tokyo, Japan).Yeast 2-hybrid analysisYeast 2-hybrid analysis was performed as described in a previous report.52 The open reading frames of Kog1 and Gtr2
mutants were cloned into the pGAD and pGBD vectors respectively.52 Two OD600 units of PJ69-4A cells harboring
these vectors were dissolved in 200 µl of dH20, and then serially 10-fold
diluted in a 96-well plate. The cells were inoculated, using a 48-pin replicator (V&P
Scientific, VP407AH), onto SC medium lacking leucine and uracil or lacking leucine,
uracil, and histidine.TAP affinity isolation assayFrom a culture of logarithmically growing cells in SD medium containing 0.5% casamino
acids without uracil, 60 OD600 units of cells were collected. These cells were
resuspended in 10 ml of SD medium containing 0.5% casamino acids, 1 M sorbitol, and 300
units of Zymolyase 100T (Nacalai, 07665-55), and then incubated for 1 h at 30 °C to
convert cells into spheroplasts. The cells were washed twice with 10 ml of ice-cold 50 mM
Tris-Cl, pH 7.5, containing 1 M sorbitol, and then resuspended in 1,200 µl of lysis buffer
(50 mM Tris-Cl, pH 7.5, 150 mM NaCl, 20 mM MgCl2, 0.5% NP-40 [Wako, 160-21691])
supplemented with protease inhibitors (10 μg/ml pepstatin A [Peptide Instituite, 4397], 20
μg/ml leupeptin [Sigma, L2884], 20 μg/ml benzamidine [Wako, 028-09481], 40 μg/ml aprotinin
[Sigma, A1153], 1 mM phenylmethylsulfonyl fluoride [Wako, 164-12181]) in order to burst
the cells. Cell lysates were cleared by centrifugation at 20,000 g at 4
°C for 10 min. Fifteen microliters of 50% slurry of IgG–Sepharose 6 Fast Flow (GE
Healthcare, 17-0969) were washed once with 1 ml of lysis buffer, mixed with 1 ml of the
lysate, and incubated for 45 min at 4 °C. The beads were washed 10 times with 1 ml
ice-cold lysis buffer, and boiled with 2 × SDS-PAGE sample buffer to elute the proteins.
For immunoblot analysis, rabbit polyclonal anti-protein A (Sigma, P3775) and mouse
monoclonal anti-GST (Cell Signaling Technology, 26H1; 2624S) were used. Intensities of the
immunoreactive bands were measured using the ImageJ software.Cultured mammalian cellsHeLa and MCF7 cells were cultured in Dulbecco’s modified Eagle’s medium (DMEM; Sigma,
D6546) containing 10% fetal bovine serum supplemented with 4 mM L-glutamine in an
atmosphere containing 5% CO2 at 37 °C. Transient transfections were performed
using Lipofectamine 2000 or RNAiMAX Reagent (Invitrogen, 12566014, 13778150).siRNA-mediated knockdown in cultured mammalian cellsCells were transfected with siRNA twice. In each transfection, the final RNA
concentration was 10 nM. Cells were used in experiments 2.5 d after transfection. The
siRNA sequences in this study were as follows: NPRL2, AGCCAGAGCU
GCAGAACAAd TdT and GCAAUGCUCU CCUCUUCAAd TdT; NPRL3, GCGUAGUUCG
GCUUCACAUd TdT and CCACUGAACC AGAGGAUGA- dTdT; and control, UUCUCCGAAC GUGUCACGUd TdT.Immunoblot analysisYeast cells were harvested by centrifugation, resuspended in 0.3 M NaOH containing 1%
β-mercaptoethanol and incubated for 5 min. Then, 15% (v/v) ice-cold trichloroacetic acid
was added and the samples were incubated for 10 min on ice. Cells were washed once with
600 µl ice-cold acetone, and 2× SDS-PAGE sample buffer was added; the pellet fractions
were dissolved by sonication. Proteins were extracted by boiling at 100 °C for 5 min. The
samples were separated by SDS-PAGE and transferred to PVDF membranes. The membranes were
blocked with 1% skim milk in 0.1% Tween-20/TBS (25 mM Tris, 150 mM NaCl, 2 mM KCl, pH
7.4), and then incubated with the following antibodies: rabbit polyclonal anti-Atg13 (gift
from Dr Yoshinori Ohsumi, Tokyo Instituite of Technology), mouse monoclonal anti-Pgk1
(Life Technologies, 459250), anti-Tor1 antibody (Santa Cruz Biotechnology Inc, sc-11900).
Immunoreactive bands were detected using horseradish peroxidase-conjugated secondary
antibodies (Jackson ImmunoResearch Laboratories, 111-035-003) and luminol solution (ECL
plus; GE Healthcare, RPN2132).Real-time quantitative PCRTotal RNA was isolated from cells using the TRIzol reagent (Invitrogen, 10296010) and
converted to cDNA using the Transcriptor First Strand cDNA Synthesis Kit (Roche,
04379012001). The cDNA was amplified and quantified by real-time PCR on a PRISM 7900HT
(ABI) using Power SYBR Green PCR master mix (ABI, 4367659). The primer sequences used in
this study were as follows: NPRL2, TGATGCCCAG GCCAAGAC and GCCAGCCAGC
TTTTTAACAA TG; NPRL3, TGCCCTAGTG CGGGTGAT and GGGCTAGCTG CTGCAGGTT; and
ACTB (encoding β-actin), CCAGCTCACC ATGGATGATG and ATGCCGGAGC
CGTTGTC.

Materials and Methods

Yeast growth mediaYeast cells were grown at 30 °C in YPD (1% yeast extract, 2% peptone, and 2% glucose) or
SD (0.67% yeast nitrogen base and 2% glucose) medium supplemented with amino acids and
0.5% casamino acid. For nitrogen starvation, cells were washed twice with SD-N (0.17%
yeast nitrogen base without amino acids and ammonium sulfate, and 2% glucose) and
resuspended in SD-N. Rapamycin (Sigma, R0395) in stock solution (1 mg/ml ethanol and
Triton X-100 at a ratio of 9:1 [v/v]) was added to YPD to achieve a final concentration of
0.2 μg/ml.

Yeast growth media

Yeast cells were grown at 30 °C in YPD (1% yeast extract, 2% peptone, and 2% glucose) or
SD (0.67% yeast nitrogen base and 2% glucose) medium supplemented with amino acids and
0.5% casamino acid. For nitrogen starvation, cells were washed twice with SD-N (0.17%
yeast nitrogen base without amino acids and ammonium sulfate, and 2% glucose) and
resuspended in SD-N. Rapamycin (Sigma, R0395) in stock solution (1 mg/ml ethanol and
Triton X-100 at a ratio of 9:1 [v/v]) was added to YPD to achieve a final concentration of
0.2 μg/ml.

Plasmid and yeast strainsThe PHO8 locus from the knockout collection was replaced with
pho8∆60 using a synthetic genetic-array method.47 Briefly, the TNY509 parental strain harboring
pho8∆60 was crossed with each mutant strain on YPD in a rectangular
plate (OmniTray, Nunc) using a 96-pin replicator (V&P Scientific, VP408). Haploid
cells containing the pho8∆60 allele and one of the mutant alleles were
selected using auxotrophic and drug-resistance markers and sequential replica
plating.15,48Yeast strains and plasmids used in this study, except for strains made using the
synthetic genetic-array method, are listed in Table
1 and Table 2. All gene disruptions,
tagged constructs, and other modifications were confirmed by PCR.53-56 N-terminal GFP-tagged Tor1 strains were generated by integrating
SpeI-digested pSK108, generated by cloning a fragment containing
PTOR1 (1000 bases)-GFP-TOR1 (426 N-terminal bases),
created in a 2-step PCR, into pRS305 using the SacI and
ApaI sites. An internally 3 × GFP-tagged Tor1 strain was generated
based on a previous report.25
GTR1 or GTR2 fragments containing approximately 1,000
bases of the promoter regions, the open reading frame, and 500 bases from the terminal
noncoding region were digested with SacI and XbaI or
BamHI and XhoI, respectively, and cloned into the
single-copy vector pRS316. GST-GTR2 fragments, in which the
GST coding region was integrated into the N terminus of
GTR2, were amplified by PCR from the yeast genome, digested with
NotI and EcoRI, and cloned into the multicopy vector
pRS426. GTP- or GDP-bound GTR1 and GTR2 mutants, and
other point mutants of GTR2, were generated using primer-based 2-step
PCRs, based on previous reports.23
ATG13 fragments, containing approximately 1000 bases of the promoter
regions and the open reading frame, were digested with SalI and
HindIII and cloned into pRS426.Table 1. Strains used in this studyStrainGenotypeReferenceTNY509MATα,
pho8Δ60::NatMX4Δcan1::
MFA1pr-HIS3-MF 1pr-LEU2
ura3Δleu2Δhis3Δmet15Δlys2Δ49BY4741MATa,
his3D1leu2D0met15D0ura3D050BY4742MATα,
his3D1leu2D0lys2D0ura3D050SKY084[BY4741]
pho8D60::NatMX4This studySKY100[SKY084]
pep4Δ::kanMX6This studySKY091[SKY084]
npr2Δ::kanMX6This studySKY131[SKY084]
npr3Δ::hphMX4This studySKY264[SKY084]
iml1Δ::kanMX6This studySKY244[SKY084]
gtr1Δ::zeoNT3This studySKY246[SKY084]
gtr2Δ::hphMX4This studySKY127[SKY084]
npr2Δ::kanMX6npr3Δ::hphMX4This studySKY167[SKY084]
gtr1Δ::kanMX6
gtr2Δ::hphMX4This studySKY245[SKY084]
gtr1Δ::zeoNT3
npr2Δ::kanMX6This studySKY247[SKY084]
gtr2Δ::hphMX4
npr2Δ::kanMX6This studySKY186[SKY084]
npr2Δ::zeoNT3 gtr1Δ::kanMX6
gtr2Δ::hphMX4This studySKY270[SKY084]
npr3Δ::hphMX4
gtr1Δ::zeoNT3This studySKY298[SKY084]
gtr2Δ::zeoNT3
npr3Δ::hphMX4This studySKY308[SKY084]
gtr2Δ::kanMX6 npr3Δ::hphMX4
gtr1Δ::zeoNT3This studySKY271[SKY084]
npr2Δ::kanMX6 npr3Δ::hphMX4
gtr1Δ::zeoNT3This studySKY291[SKY084]
npr2Δ::kanMX6 npr3Δ::hphMX4
gtr2Δ::zeoNT3This studySKY309[SKY084]
gtr2Δ::LEU2 gtr2Δ::kanMX6
npr3Δ::hphMX4
gtr1Δ::zeoNT3This studySKY277[SKY084]
gtr1Δ::kanMX6 gtr2Δ::hphMX4
tco89Δ::zeoThis studySKY222[BY4741]
LEU2::GFP-TOR1This studySKY226[SKY222]
npr2Δ::natMX4This studySKY278[SKY222]
gtr1Δ::hphMX6This studySKY279[SKY222]
gtr2Δ::hphMX4This studySKY299[SKY222]
gtr2Δ::zeoNT3
gtr1Δ::hphMX4This studySKY135[BY4741]
npr2Δ::natMX4This studyNKR002[BY4741]
ego1Δ::kanMX6EUROSCARFNKR003[BY4741]
tor1Δ::kanMX6EUROSCARFSKY173[BY4742]
Tor1-D330-3xGFPThis studySKY112[BY4741]
GTR1-GFP::hphMX4This studySKY249[BY4741]
GTR1-GFP::hphMX4,
gtr2Δ::natMX4This studySKY259[BY4741]
GTR2-GFP::hphMX4This studySKY260[BY4741] GTR2-GFP::hphMX4
gtr1Δ::zeoNT3This studySKY287[BY4741]
KOG1-TAP::kanMX6This studyPJ69-4AMATa
trpl-901 leu2-3,112 ura3-52 his3-200
ga14Δga180Δ
LYS2::GALl-HIS3GAL2-ADE2
met2::GAL7-lacZ52Table 2. Plasmids used in this studyPlasmidDescriptionReferencepRS316CEN, URA351pRS4262 micron,
URA351pGBD-C12 micron,
TRP152pGAD-C12 micron,
LEU252pSK122[pRS316] Gtr1
Gtr2This studypSK123[pRS316] Gtr1
Gtr2S23LThis studypSK124[pRS316] Gtr1
Gtr2Q66LThis studypSK125[pRS316] Gtr1S20L
Gtr2This studypSK126[pRS316] Gtr1S20L
Gtr2S23LThis studypSK127[pRS316] Gtr1S20L
Gtr2Q66LThis studypSK128[pRS316] Gtr1Q65L
Gtr2This studypSK129[pRS316] Gtr1Q65L
Gtr2S23LThis studypSK130[pRS316] Gtr1Q65L
Gtr2Q66LThis studypSK115[pRS316]
Gtr1S20LThis studypSK116[pRS316]
Gtr1Q65LThis studypSK256[pRS316] Gtr1S20L
Gtr2E62KThis studypSK150[pGBD-C1]
Gtr2This studypSK151[pGBD-C1]
Gtr2Q66LThis studypSK152[pGBD-C1]
Gtr2S23LThis studypSK222[pGBD-C1]
Gtr2E62KThis studypSK156[pGAD-C1]
Kog1This studypSK242[pRS426]
PGTR2-GST-Gtr2This studypSK243[pRS426]
PGTR2-GST-Gtr2Q66LThis studypSK244[pRS426]
PGTR2-GST-Gtr2S23LThis studypSK252[pRS426]
PGTR2-GST-Gtr2E62KThis studypSK048[pRS426]
PATG13-ATG13This study

Plasmid and yeast strains

The PHO8 locus from the knockout collection was replaced with
pho8∆60 using a synthetic genetic-array method.47 Briefly, the TNY509 parental strain harboring
pho8∆60 was crossed with each mutant strain on YPD in a rectangular
plate (OmniTray, Nunc) using a 96-pin replicator (V&P Scientific, VP408). Haploid
cells containing the pho8∆60 allele and one of the mutant alleles were
selected using auxotrophic and drug-resistance markers and sequential replica
plating.15,48

Yeast strains and plasmids used in this study, except for strains made using the
synthetic genetic-array method, are listed in Table
1 and Table 2. All gene disruptions,
tagged constructs, and other modifications were confirmed by PCR.53-56 N-terminal GFP-tagged Tor1 strains were generated by integrating
SpeI-digested pSK108, generated by cloning a fragment containing
PTOR1 (1000 bases)-GFP-TOR1 (426 N-terminal bases),
created in a 2-step PCR, into pRS305 using the SacI and
ApaI sites. An internally 3 × GFP-tagged Tor1 strain was generated
based on a previous report.25
GTR1 or GTR2 fragments containing approximately 1,000
bases of the promoter regions, the open reading frame, and 500 bases from the terminal
noncoding region were digested with SacI and XbaI or
BamHI and XhoI, respectively, and cloned into the
single-copy vector pRS316. GST-GTR2 fragments, in which the
GST coding region was integrated into the N terminus of
GTR2, were amplified by PCR from the yeast genome, digested with
NotI and EcoRI, and cloned into the multicopy vector
pRS426. GTP- or GDP-bound GTR1 and GTR2 mutants, and
other point mutants of GTR2, were generated using primer-based 2-step
PCRs, based on previous reports.23
ATG13 fragments, containing approximately 1000 bases of the promoter
regions and the open reading frame, were digested with SalI and
HindIII and cloned into pRS426.

Table 1. Strains used in this study

Table 2. Plasmids used in this study

ALP assaysFor large-scale assays, YPD plates were inoculated with cells from each pool using a
96-pin replicator, and then incubated for 16 to 24 h at 30 °C. For conventional assays,
cells were cultured in liquid medium as described above. Subsequently, the cells were
collected by centrifugation, suspended in 200 μl of SD-N medium in 96-well plates, and
incubated for 4 h at 30 °C. The plates were centrifuged, and the supernatant fraction was
discarded. We then added 50 μl of ice-cold lysis buffer (10 mM Tris-Cl, pH 9.0, 10 mM
MgSO4, 10 μM ZnSO4) and approximately 10 μl of 0.6-mm zirconia
silica beads (Biomedical Science, ZS06-0001) to each well. The plates were sealed with
Parafilm and mixed vigorously (2,500/min) on a microplate mixer (Taitec, MBR-022) for 10
min at 4 °C. After a brief centrifugation, the Parafilm was removed, 150 μl of ice-cold
lysis buffer was added, and the plates were centrifuged for 15 min at 490 ×
g at 4 °C. Protein levels were quantified in 50-μl aliquots of the
supernatant fractions using a bicinchoninic acid kit (Nacalai, 06385-00), and the
enzymatic activity in the aliquots was measured as described previously with slight
modifications.15 One unit of ALP
activity was defined as one emission/μg protein/min.

For large-scale assays, YPD plates were inoculated with cells from each pool using a
96-pin replicator, and then incubated for 16 to 24 h at 30 °C. For conventional assays,
cells were cultured in liquid medium as described above. Subsequently, the cells were
collected by centrifugation, suspended in 200 μl of SD-N medium in 96-well plates, and
incubated for 4 h at 30 °C. The plates were centrifuged, and the supernatant fraction was
discarded. We then added 50 μl of ice-cold lysis buffer (10 mM Tris-Cl, pH 9.0, 10 mM
MgSO4, 10 μM ZnSO4) and approximately 10 μl of 0.6-mm zirconia
silica beads (Biomedical Science, ZS06-0001) to each well. The plates were sealed with
Parafilm and mixed vigorously (2,500/min) on a microplate mixer (Taitec, MBR-022) for 10
min at 4 °C. After a brief centrifugation, the Parafilm was removed, 150 μl of ice-cold
lysis buffer was added, and the plates were centrifuged for 15 min at 490 ×
g at 4 °C. Protein levels were quantified in 50-μl aliquots of the
supernatant fractions using a bicinchoninic acid kit (Nacalai, 06385-00), and the
enzymatic activity in the aliquots was measured as described previously with slight
modifications.15 One unit of ALP
activity was defined as one emission/μg protein/min.

MicroscopyFor visualizing vacuolar lumen, 1 OD600 unit of cells were incubated for 30
min at 30 °C in SD medium containing 0.5% casamino acids and 100 mM CMAC fluorescent dye
(Molecular Probes, C2110). Yeast cells were observed Leica AF6500 fluorescent imaging
system (Leica Microsystems, Wetzlar, Germany) mounted on a DIM6000 B microscope (HCX PL
APO 63.6×/1.40–0.60 oil-immersion objective lens, xenon lamp, Leica Microsystems, Wetzlar,
Germany) under the control of LAS-AF software (Leica Microsystems). In Figure S4A, cells were attached on
35-mm glass bottom dishes (Matsunami, D110300) coated with concanavalin A (Sigma, C7275),
and examined using a Leica TCS SP8 confocal system (Leica Microsystems, Wetzlar, Germany)
mounted on a DMI 6000 CS microscope (HCX PL APO 100.6×/1.4 Oil STED objective lens, Leica
Microsystems, Wetzlar, Germany). Mammalian cells were cultured on coverslips and fixed by
incubation in 3% paraformaldehyde in phosphate-buffered saline (PBS; 137 mM NaCl, 2.7 mM
KCl, 10 mM Na2HPO4, 2 mM KH2PO4, pH 7.2) for 20 min. For
immunofluorescence microscopy, the cells were permeabilized with 50 μg/ml digitonin
(Sigma, D141) in PBS for 10 min at room temperature. The permeabilized cells were
incubated in blocking solution (0.1% gelatin in PBS), and then stained with anti-MTOR
(Cell Signaling Technology, 7C10; 2983) and LAMP1 (Santa Cruz Biotechnology, Inc., H4A3;
sc-20011). The coverslips were mounted in SlowFade Gold Reagent (Invitrogen, S36936), and
images were obtained using an Olympus FV1000 laser-scanning confocal microscope (Olympus,
Tokyo, Japan).

For visualizing vacuolar lumen, 1 OD600 unit of cells were incubated for 30
min at 30 °C in SD medium containing 0.5% casamino acids and 100 mM CMAC fluorescent dye
(Molecular Probes, C2110). Yeast cells were observed Leica AF6500 fluorescent imaging
system (Leica Microsystems, Wetzlar, Germany) mounted on a DIM6000 B microscope (HCX PL
APO 63.6×/1.40–0.60 oil-immersion objective lens, xenon lamp, Leica Microsystems, Wetzlar,
Germany) under the control of LAS-AF software (Leica Microsystems). In Figure S4A, cells were attached on
35-mm glass bottom dishes (Matsunami, D110300) coated with concanavalin A (Sigma, C7275),
and examined using a Leica TCS SP8 confocal system (Leica Microsystems, Wetzlar, Germany)
mounted on a DMI 6000 CS microscope (HCX PL APO 100.6×/1.4 Oil STED objective lens, Leica
Microsystems, Wetzlar, Germany). Mammalian cells were cultured on coverslips and fixed by
incubation in 3% paraformaldehyde in phosphate-buffered saline (PBS; 137 mM NaCl, 2.7 mM
KCl, 10 mM Na2HPO4, 2 mM KH2PO4, pH 7.2) for 20 min. For
immunofluorescence microscopy, the cells were permeabilized with 50 μg/ml digitonin
(Sigma, D141) in PBS for 10 min at room temperature. The permeabilized cells were
incubated in blocking solution (0.1% gelatin in PBS), and then stained with anti-MTOR
(Cell Signaling Technology, 7C10; 2983) and LAMP1 (Santa Cruz Biotechnology, Inc., H4A3;
sc-20011). The coverslips were mounted in SlowFade Gold Reagent (Invitrogen, S36936), and
images were obtained using an Olympus FV1000 laser-scanning confocal microscope (Olympus,
Tokyo, Japan).

Yeast 2-hybrid analysisYeast 2-hybrid analysis was performed as described in a previous report.52 The open reading frames of Kog1 and Gtr2
mutants were cloned into the pGAD and pGBD vectors respectively.52 Two OD600 units of PJ69-4A cells harboring
these vectors were dissolved in 200 µl of dH20, and then serially 10-fold
diluted in a 96-well plate. The cells were inoculated, using a 48-pin replicator (V&P
Scientific, VP407AH), onto SC medium lacking leucine and uracil or lacking leucine,
uracil, and histidine.

Yeast 2-hybrid analysis

Yeast 2-hybrid analysis was performed as described in a previous report.52 The open reading frames of Kog1 and Gtr2
mutants were cloned into the pGAD and pGBD vectors respectively.52 Two OD600 units of PJ69-4A cells harboring
these vectors were dissolved in 200 µl of dH20, and then serially 10-fold
diluted in a 96-well plate. The cells were inoculated, using a 48-pin replicator (V&P
Scientific, VP407AH), onto SC medium lacking leucine and uracil or lacking leucine,
uracil, and histidine.

TAP affinity isolation assayFrom a culture of logarithmically growing cells in SD medium containing 0.5% casamino
acids without uracil, 60 OD600 units of cells were collected. These cells were
resuspended in 10 ml of SD medium containing 0.5% casamino acids, 1 M sorbitol, and 300
units of Zymolyase 100T (Nacalai, 07665-55), and then incubated for 1 h at 30 °C to
convert cells into spheroplasts. The cells were washed twice with 10 ml of ice-cold 50 mM
Tris-Cl, pH 7.5, containing 1 M sorbitol, and then resuspended in 1,200 µl of lysis buffer
(50 mM Tris-Cl, pH 7.5, 150 mM NaCl, 20 mM MgCl2, 0.5% NP-40 [Wako, 160-21691])
supplemented with protease inhibitors (10 μg/ml pepstatin A [Peptide Instituite, 4397], 20
μg/ml leupeptin [Sigma, L2884], 20 μg/ml benzamidine [Wako, 028-09481], 40 μg/ml aprotinin
[Sigma, A1153], 1 mM phenylmethylsulfonyl fluoride [Wako, 164-12181]) in order to burst
the cells. Cell lysates were cleared by centrifugation at 20,000 g at 4
°C for 10 min. Fifteen microliters of 50% slurry of IgG–Sepharose 6 Fast Flow (GE
Healthcare, 17-0969) were washed once with 1 ml of lysis buffer, mixed with 1 ml of the
lysate, and incubated for 45 min at 4 °C. The beads were washed 10 times with 1 ml
ice-cold lysis buffer, and boiled with 2 × SDS-PAGE sample buffer to elute the proteins.
For immunoblot analysis, rabbit polyclonal anti-protein A (Sigma, P3775) and mouse
monoclonal anti-GST (Cell Signaling Technology, 26H1; 2624S) were used. Intensities of the
immunoreactive bands were measured using the ImageJ software.

TAP affinity isolation assay

From a culture of logarithmically growing cells in SD medium containing 0.5% casamino
acids without uracil, 60 OD600 units of cells were collected. These cells were
resuspended in 10 ml of SD medium containing 0.5% casamino acids, 1 M sorbitol, and 300
units of Zymolyase 100T (Nacalai, 07665-55), and then incubated for 1 h at 30 °C to
convert cells into spheroplasts. The cells were washed twice with 10 ml of ice-cold 50 mM
Tris-Cl, pH 7.5, containing 1 M sorbitol, and then resuspended in 1,200 µl of lysis buffer
(50 mM Tris-Cl, pH 7.5, 150 mM NaCl, 20 mM MgCl2, 0.5% NP-40 [Wako, 160-21691])
supplemented with protease inhibitors (10 μg/ml pepstatin A [Peptide Instituite, 4397], 20
μg/ml leupeptin [Sigma, L2884], 20 μg/ml benzamidine [Wako, 028-09481], 40 μg/ml aprotinin
[Sigma, A1153], 1 mM phenylmethylsulfonyl fluoride [Wako, 164-12181]) in order to burst
the cells. Cell lysates were cleared by centrifugation at 20,000 g at 4
°C for 10 min. Fifteen microliters of 50% slurry of IgG–Sepharose 6 Fast Flow (GE
Healthcare, 17-0969) were washed once with 1 ml of lysis buffer, mixed with 1 ml of the
lysate, and incubated for 45 min at 4 °C. The beads were washed 10 times with 1 ml
ice-cold lysis buffer, and boiled with 2 × SDS-PAGE sample buffer to elute the proteins.
For immunoblot analysis, rabbit polyclonal anti-protein A (Sigma, P3775) and mouse
monoclonal anti-GST (Cell Signaling Technology, 26H1; 2624S) were used. Intensities of the
immunoreactive bands were measured using the ImageJ software.

Cultured mammalian cellsHeLa and MCF7 cells were cultured in Dulbecco’s modified Eagle’s medium (DMEM; Sigma,
D6546) containing 10% fetal bovine serum supplemented with 4 mM L-glutamine in an
atmosphere containing 5% CO2 at 37 °C. Transient transfections were performed
using Lipofectamine 2000 or RNAiMAX Reagent (Invitrogen, 12566014, 13778150).

Cultured mammalian cells

HeLa and MCF7 cells were cultured in Dulbecco’s modified Eagle’s medium (DMEM; Sigma,
D6546) containing 10% fetal bovine serum supplemented with 4 mM L-glutamine in an
atmosphere containing 5% CO2 at 37 °C. Transient transfections were performed
using Lipofectamine 2000 or RNAiMAX Reagent (Invitrogen, 12566014, 13778150).

siRNA-mediated knockdown in cultured mammalian cellsCells were transfected with siRNA twice. In each transfection, the final RNA
concentration was 10 nM. Cells were used in experiments 2.5 d after transfection. The
siRNA sequences in this study were as follows: NPRL2, AGCCAGAGCU
GCAGAACAAd TdT and GCAAUGCUCU CCUCUUCAAd TdT; NPRL3, GCGUAGUUCG
GCUUCACAUd TdT and CCACUGAACC AGAGGAUGA- dTdT; and control, UUCUCCGAAC GUGUCACGUd TdT.

siRNA-mediated knockdown in cultured mammalian cells

Cells were transfected with siRNA twice. In each transfection, the final RNA
concentration was 10 nM. Cells were used in experiments 2.5 d after transfection. The
siRNA sequences in this study were as follows: NPRL2, AGCCAGAGCU
GCAGAACAAd TdT and GCAAUGCUCU CCUCUUCAAd TdT; NPRL3, GCGUAGUUCG
GCUUCACAUd TdT and CCACUGAACC AGAGGAUGA- dTdT; and control, UUCUCCGAAC GUGUCACGUd TdT.

Immunoblot analysisYeast cells were harvested by centrifugation, resuspended in 0.3 M NaOH containing 1%
β-mercaptoethanol and incubated for 5 min. Then, 15% (v/v) ice-cold trichloroacetic acid
was added and the samples were incubated for 10 min on ice. Cells were washed once with
600 µl ice-cold acetone, and 2× SDS-PAGE sample buffer was added; the pellet fractions
were dissolved by sonication. Proteins were extracted by boiling at 100 °C for 5 min. The
samples were separated by SDS-PAGE and transferred to PVDF membranes. The membranes were
blocked with 1% skim milk in 0.1% Tween-20/TBS (25 mM Tris, 150 mM NaCl, 2 mM KCl, pH
7.4), and then incubated with the following antibodies: rabbit polyclonal anti-Atg13 (gift
from Dr Yoshinori Ohsumi, Tokyo Instituite of Technology), mouse monoclonal anti-Pgk1
(Life Technologies, 459250), anti-Tor1 antibody (Santa Cruz Biotechnology Inc, sc-11900).
Immunoreactive bands were detected using horseradish peroxidase-conjugated secondary
antibodies (Jackson ImmunoResearch Laboratories, 111-035-003) and luminol solution (ECL
plus; GE Healthcare, RPN2132).

Immunoblot analysis

Yeast cells were harvested by centrifugation, resuspended in 0.3 M NaOH containing 1%
β-mercaptoethanol and incubated for 5 min. Then, 15% (v/v) ice-cold trichloroacetic acid
was added and the samples were incubated for 10 min on ice. Cells were washed once with
600 µl ice-cold acetone, and 2× SDS-PAGE sample buffer was added; the pellet fractions
were dissolved by sonication. Proteins were extracted by boiling at 100 °C for 5 min. The
samples were separated by SDS-PAGE and transferred to PVDF membranes. The membranes were
blocked with 1% skim milk in 0.1% Tween-20/TBS (25 mM Tris, 150 mM NaCl, 2 mM KCl, pH
7.4), and then incubated with the following antibodies: rabbit polyclonal anti-Atg13 (gift
from Dr Yoshinori Ohsumi, Tokyo Instituite of Technology), mouse monoclonal anti-Pgk1
(Life Technologies, 459250), anti-Tor1 antibody (Santa Cruz Biotechnology Inc, sc-11900).
Immunoreactive bands were detected using horseradish peroxidase-conjugated secondary
antibodies (Jackson ImmunoResearch Laboratories, 111-035-003) and luminol solution (ECL
plus; GE Healthcare, RPN2132).

Real-time quantitative PCRTotal RNA was isolated from cells using the TRIzol reagent (Invitrogen, 10296010) and
converted to cDNA using the Transcriptor First Strand cDNA Synthesis Kit (Roche,
04379012001). The cDNA was amplified and quantified by real-time PCR on a PRISM 7900HT
(ABI) using Power SYBR Green PCR master mix (ABI, 4367659). The primer sequences used in
this study were as follows: NPRL2, TGATGCCCAG GCCAAGAC and GCCAGCCAGC
TTTTTAACAA TG; NPRL3, TGCCCTAGTG CGGGTGAT and GGGCTAGCTG CTGCAGGTT; and
ACTB (encoding β-actin), CCAGCTCACC ATGGATGATG and ATGCCGGAGC
CGTTGTC.

Real-time quantitative PCR

Total RNA was isolated from cells using the TRIzol reagent (Invitrogen, 10296010) and
converted to cDNA using the Transcriptor First Strand cDNA Synthesis Kit (Roche,
04379012001). The cDNA was amplified and quantified by real-time PCR on a PRISM 7900HT
(ABI) using Power SYBR Green PCR master mix (ABI, 4367659). The primer sequences used in
this study were as follows: NPRL2, TGATGCCCAG GCCAAGAC and GCCAGCCAGC
TTTTTAACAA TG; NPRL3, TGCCCTAGTG CGGGTGAT and GGGCTAGCTG CTGCAGGTT; and
ACTB (encoding β-actin), CCAGCTCACC ATGGATGATG and ATGCCGGAGC
CGTTGTC.

Supplementary MaterialAdditional material

Supplementary Material
