---
pmid: '19043414'
title: Molecular basis of Pirh2-mediated p53 ubiquitylation.
authors:
- Sheng Y
- Laister RC
- Lemak A
- Wu B
- Tai E
- Duan S
- Lukin J
- Sunnerhagen M
- Srisailam S
- Karra M
- Benchimol S
- Arrowsmith CH
journal: Nat Struct Mol Biol
year: '2008'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC4075976
doi: 10.1038/nsmb.1521
---

# Molecular basis of Pirh2-mediated p53 ubiquitylation.
**Authors:** Sheng Y, Laister RC, Lemak A, Wu B, Tai E, Duan S, Lukin J, Sunnerhagen M, Srisailam S, Karra M, Benchimol S, Arrowsmith CH
**Journal:** Nat Struct Mol Biol (2008)
**DOI:** [10.1038/nsmb.1521](https://doi.org/10.1038/nsmb.1521)
**PMC:** [PMC4075976](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4075976/)

## Abstract

1. Nat Struct Mol Biol. 2008 Dec;15(12):1334-42. doi: 10.1038/nsmb.1521. Epub
2008  Nov 30.

Molecular basis of Pirh2-mediated p53 ubiquitylation.

Sheng Y(1), Laister RC, Lemak A, Wu B, Tai E, Duan S, Lukin J, Sunnerhagen M, 
Srisailam S, Karra M, Benchimol S, Arrowsmith CH.

Author information:
(1)Ontario Cancer Institute and Department of Medical Biophysics, University of 
Toronto, Ontario M5G1L7, Canada.

Pirh2 (p53-induced RING-H2 domain protein; also known as Rchy1) is an E3 
ubiquitin ligase involved in a negative-feedback loop with p53. Using NMR 
spectroscopy, we show that Pirh2 is a unique cysteine-rich protein comprising 
three modular domains. The protein binds nine zinc ions using a variety of zinc 
coordination schemes, including a RING domain and a left-handed beta-spiral in 
which three zinc ions align three consecutive small beta-sheets in an 
interleaved fashion. We show that Pirh2-p53 interaction is dependent on the 
C-terminal zinc binding module of Pirh2, which binds to the tetramerization 
domain of p53. As a result, Pirh2 preferentially ubiquitylates the tetrameric 
form of p53 in vitro and in vivo, suggesting that Pirh2 regulates protein 
turnover of the transcriptionally active form of p53.

DOI: 10.1038/nsmb.1521
PMCID: PMC4075976
PMID: 19043414 [Indexed for MEDLINE]

## Full Text

Abstract

Pirh2 (p53-induced RING-H2 domain protein, also known as Rchy1), is an E3 ubiquitin ligase involved in a negative-feedback loop with p53. Using NMR spectroscopy we show that Pirh2 is a unique cysteine-rich protein comprising three modular domains. The protein binds nine zinc ions using a variety of zinc coordination schemes including a RING domain and a novel left-handed β-spiral in which three zinc ions align three consecutive small β-sheets in an interleaved fashion. We demonstrate that Pirh2-p53 interaction is dependent on the C-terminal Zn-binding module of Pirh2 which binds to the tetramerization domain (TET) of p53. As a result, Pirh2 ubiquitylates preferably the tetrameric form of p53 in vitro and in vivo, suggesting that Pirh2 regulates protein turnover of the transcriptionally active form of p53 in the cell.

RESULTS

Human Pirh2 is a 261-residue protein containing three domains, the NTD (residues 1–137), the central RING domain (residues 138–189) and the CTD (residues 190–261) ( Fig. 1a ). There are 29 cysteine residues throughout the protein with 19 clustered in the NTD, 6 in the RING and 4 in the CTD ( Fig. 1b ). Full length Pirh2 protein could only be purified and studied at low concentrations (< 50 uM) and therefore was not amenable to structural studies under a variety of conditions. However, we were able to prepare suitable NMR samples for the three individual domains of human Pirh2 and solve their solution structures using three-dimensional heteronuclear NMR spectroscopy. The zinc coordination and binding stoichiometry of Pirh2 were determined based on the ICP-AES analysis and NMR restraints. The ensembles of the twenty lowest-energy conformers for these three domains are shown in Supplementary Fig. 1 online. Figure 2 demonstrates the overall structure and the zinc ion coordination scheme within each domain.

DISCUSSION

The structures of Pirh2 domains reveal several novel features in protein architecture and zinc coordination. An unusually high content of cysteine (11%) and histidine (8%) enables the coordination of nine zinc ions within this medium sized protein. The N-terminal lobe of Pirh2 NTD is a member of the CHY zinc finger family (PF05495) by virtue of its N-terminal CxHY sequence motif and is found only in eukaryotes. The structure of the N-terminal lobe shows a unique zinc coordination which combines a cross-brace zinc binding motif with a C4 zinc finger. The cross-brace zinc binding motif uses the consensus sequence CxHx(10)CCx(5)Cx(2)CHx(5)H to chelate two zinc ions. Though the topology of this cross-brace zinc binding motif in the NTD appears the same as the RING domain, the permutation and spacing of the cysteine and histidine residues for the zinc coordination are very different from the RING domain. The typical RING finger (pfam zf-C3HC4, PF00097) uses the consensus sequence Cx2Cx(9–39)Cx(1–3)Hx(2–3)Cx2Cx(4–48)Cx2C to bind two zinc ions and inevitably contains a hydrophobic patch for the recruitment of ubiquitin E2 ligases 12 , 17 . Both of these features are lacking in the Pirh2 NTD, explaining the lack of E3 ligase activity in the N-terminal lobe, despite its structural similarity to the RING domain.

The C lobe of the Pirh2 NTD folds into a three-turn β spiral held together by three zinc ions. The structure is novel in two respects. First, this left-handed solenoid structure is made of three successive strand-turn-strand motifs. These motifs are packed together by a novel combination of zinc ion coordination and hydrophobic interactions between sidechains from the strands. Second, the three zinc ions are interlocked by the sequence motif: C x(2) C x(10) H C x(2) C x(2) C x(9) H Cx(2)Cx(2) C x(8)HxC, in which the 1 st histidine and 4 th cysteine residues of each HCCC repeat are paired with the N-terminal cysteine pair to coordinate a zinc ion whereas the 2 nd and 3 rd cysteine residues are paired with the C-terminal His and Cys to bind another zinc ion ( Fig. 2b ). This type of zinc coordination has not been previously reported and is found to be conserved in eukaryotes. Protein sequences alignment reveals that many eukaryotic proteins contain similar sequence motif (see Supplementary Fig. 6 online). All these proteins have the HCCC repeats and the spacings within the HCCC repeat are conserved, suggesting that these proteins may also hold the β solenoid zinc finger structure.

Our data indicate that the three domains of Pirh2 are independently folded and do not interact with one another, suggesting that the full length protein has a modular arrangement of domains. The linker between the Pirh2 NTD and the RING domain is relatively short (~ 10 residues), suggesting that the NTD and the RING may form a more rigid connection. In contrast, the 30 amino-acid linker between the RING and the CTD (~30 residues) gives Pirh2 great flexibility to bring together the ubiquitin-charged E2 enzyme with substrates.

P53 is known to be a substrate for Pirh2. Here we find that Pirh2 interacts with p53 primarily via CTD residues 249–256 which recognize the p53 TET domain with a Kd on the order of 600 uM. The NTD of Pirh2 binds more weakly to the DBD of p53 and while not required for activity may contribute to optimal activity. The region between the N-terminal CHY zinc-finger and the RING domain of Pirh2 (residues 120–137) was reported to be required for p53 recognition because a mutant Pirh2 lacking residues 120–137 was unable to bind p53 6 . In view of the present Pirh2 domain structures, this deletion disrupts both the 5 th Zn and 6 th Zn coordinating residues, and will lead to a drastically disturbed Pirh2 fold. Synthetic Pirh2 peptides derived from the same region (residues 120–130 and 126–137) bind very weakly (K d > 1 mM) under physiological ionic strength conditions, implicating a weak and transient interaction between this Pirh2 peptide motif alone and p53 16 . Notably, we did not observe a direct interaction between Pirh2 120–137 in the context of the full-length protein or folded individual domains, suggesting that prior reports of this interaction may have been due to improper folding of the NTD.

The binding site on Pirh2 for the E2 ligase, UBE2D2/UbcH5b, was mapped to the hydrophobic depression on the surface of the RING domain using NMR titration and mutatgenesis. Based on the NMR titration data, Pirh2 RING binds UBE2D2/UbcH5b with an affinity of ~ 200 μM, consistent with values observed for the other E2-E3 complexes 18 . These data confirmed the central role of the RING domain in recruiting the E2 and mediating p53 ubiquitylation, in a manner similar to known RING-E2 complexes 19 , 20 .

Using an in vitro ubiquitylation assay, we found that the interaction of the Pirh2 CTD and p53 TET is essential for Pirh2 mediated p53 ubiquitylation. The Pirh2 fragment spanning residues 138–261, containing the RING domain and intact CTD but no NTD, was sufficient to ubiquitylate p53 while the Pirh2 fragments with C-terminal truncations had greatly reduced or no ubiquitylation activity. Importantly, the fragment 1–195 which lacks the entire C-terminus is still able to mediate autoubiquitylation, but not p53 ubiquitylation, suggesting an intact Pirh2 C-terminus is necessary for p53 ubiquitylation. Thus, the interaction of p53 via its TET domain with the CTD of Pirh2 appears to be necessary and sufficient for p53 ubiquitylation in vitro . The additional interaction of p53 DBD with the Pirh2 NTD, although not required for ubiquitylation activity, appears to augment Pirh2-mediated p53 ubiquitylation. Further characterization of p53 mutants for their ability to be ubiquitylated by Pirh2 in vitro , supported the importance of the tetrameric state of p53 in this activity; Pirh2 ubiquitylation of dimeric and monomeric mutants of p53 was severely curtailed. Mutations in the DBD of p53 had no effect on activity, again indicating a lesser role for this domain. Finally, we tested the p53 TET domain mutants for stability in H1299 cells when co-transfected with Pirh2. Consistent with the in vitro data, the levels of p53 oligomerization mutants were much higher than that of the wild type protein, reflecting defects in protein degradation for the DM and MM p53 proteins in the presence of Pirh2.

Together these data suggest a working model for Pirh2 mediated p53 ubiquitylation in which the CTD-TET domain interaction provides the majority of the binding energy and the NTD may further interact transiently with one or more of the p53 DNA binding domains ( Fig. 7 ). Indeed, the combination of a primary and a secondary binding site together with the fact that WT p53 is tetrameric, may contribute to increased avidity and activity of the full length proteins as opposed to the individual domains studied here. This mode of interaction likely fixes p53 with respect to the ubiquitin-charged E2 which docks at the central RING domain of Pirh2. While the relative orientation of p53 domains is not likely to be rigid 21 , and that of the Pirh2 domains is not yet known, this model is conceivable in that it places the C-terminal Lysines of p53 in close proximity to the E2 active site. P53 ubiquitylation is thought to be the central cellular mechanism to downregulate p53 for proliferation in unstressed cells and to upregulate p53 in response to various stresses. As the main regulators of p53 stability, p53 E3 ligases have attracted enormous interest. They are often overexpressed in a variety of human malignancies and may represent a class of targets for the development of novel therapeutic compounds aimed at promoting p53-mediated growth inhibition and apoptosis in cancer cells. Indeed, Mdm2 has been an attractive therapeutic target and the subject of drug discovery activities in both academia and industry 5 . We note here that Pirh2 binds to domains distinct from the transactivation domain recognized by Mdm2, possibly offering an alternative (and/or additional) strategy for p53 regulation. The structures and interaction data presented here provide insight into the molecular mechanism of Pirh2-mediated p53 ubiquitylation and point to the disruption of the p53 TET - Pirh2 CTD interaction as potential strategy for development of therapeutics.
