---
pmid: '19372273'
title: Histone deacetylase Rpd3 antagonizes Sir2-dependent silent chromatin propagation.
authors:
- Zhou J
- Zhou BO
- Lenzmeier BA
- Zhou JQ
journal: Nucleic Acids Res
year: '2009'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC2699518
doi: 10.1093/nar/gkp233
---

# Histone deacetylase Rpd3 antagonizes Sir2-dependent silent chromatin propagation.
**Authors:** Zhou J, Zhou BO, Lenzmeier BA, Zhou JQ
**Journal:** Nucleic Acids Res (2009)
**DOI:** [10.1093/nar/gkp233](https://doi.org/10.1093/nar/gkp233)
**PMC:** [PMC2699518](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2699518/)

## Abstract

1. Nucleic Acids Res. 2009 Jun;37(11):3699-713. doi: 10.1093/nar/gkp233. Epub
2009  Apr 16.

Histone deacetylase Rpd3 antagonizes Sir2-dependent silent chromatin 
propagation.

Zhou J(1), Zhou BO, Lenzmeier BA, Zhou JQ.

Author information:
(1)The State Key Laboratory of Molecular Biology, Institute of Biochemistry and 
Cell Biology, Institutes for Biological Sciences, Chinese Academy of Sciences, 
The Graduate School, Shanghai 200031, China.

In the eukaryotic genome, transcriptionally silent chromatin tends to propagate 
along a chromosome and encroach upon adjacent active chromatin. The silencing 
machinery can be stopped by chromatin boundary elements. We performed a screen 
in Saccharomyces cerevisiae for proteins that may contribute to the 
establishment of a chromatin boundary. We found that disruption of histone 
deacetylase Rpd3p results in defective boundary activity, leading to a 
Sir-dependent local propagation of transcriptional repression. In rpd3 Delta 
cells, the amount of Sir2p that was normally found in the nucleolus decreased 
and the amount of Sir2p found at telomeres and at HM and its adjacent loci 
increased, leading to an extension of silent chromatin in those areas. In 
addition, Rpd3p interacted directly with chromatin at boundary regions to 
deacetylate histone H4 at lysine 5 and at lysine 12. Either the mutation of 
histone H4 at lysine 5 or a decrease in the histone acetyltransferase (HAT) 
activity of Esa1p abrogated the silencing phenotype associated with rpd3 
mutation, suggesting a novel role for the H4 amino terminus in Rpd3p-mediated 
heterochromatin boundary regulation. Together, these data provide insight into 
the molecular mechanisms for the anti-silencing functions of Rpd3p during the 
formation of heterochromatin boundaries.

DOI: 10.1093/nar/gkp233
PMCID: PMC2699518
PMID: 19372273 [Indexed for MEDLINE]

## Full Text

INTRODUCTIONThe eukaryotic genome is organized into chromosomal domains of distinct structure and function (1). The fraction of chromatin that condenses during mitosis and is found decondensed during the interphase of the cell cycle is termed euchromatin (2). In contrast, constitutively compacted chromatin often found at locations like centromeres and telomeres is called heterochromatin (3,4). In general, euchromatic domains bear transcriptionally active genes, whereas heterochromatic domains are largely inactive transcriptionally, leading to a silencing position effect on genes in the heterochromatic region (5,6). Heterochromatin forms a nuclease-resistant structure that can propagate along the chromosome and repress nearby genes in a stochastic manner (2,7). Boundary elements are often found between heterochromatic and euchromatic regions. The prevailing view of boundary elements, or insulators, is that they are specific DNA elements that actively recruit barrier proteins to inhibit the spread of silent chromatin into euchromatic regions, thereby insulating a euchromatic gene from the influence of silent chromatin that could spread into that transcriptionally active region (8–10). Some boundary elements can constitutively recruit epigenetic modification machineries, acting as a chain terminator to the spreading of a repressive chromatin (11–15). Other chromatin boundaries are defined by a gradient of chromatin modifications, such as differing degrees of histone hyperacetylation or hypoacetylation on opposing sides of the resulting boundary element (16–18). Positions of boundary elements can vary depending on the balance of chromatin modifications resulting from the sum of activities of different enzymatic proteins or complexes (19).The mating loci HMR and HML and the telomeres of Saccharomyces cerevisiae are well-characterized silenced chromatin domains that provide distinctive models for studying the formation of heterochromatin structure and the establishment of chromatin boundaries (12,13,20,21). Heterochromatin propagation depends on the roles of cis-acting elements, such as the telomeric DNA repeats and the sites flanking each HM locus that are known as silencers, as well as trans-acting proteins like the silent information regulator (Sir) complex of proteins and specific silencer-binding or telomere-binding proteins (6,11). The Sir complex, which contains Sir2p, Sir3p and Sir4p, is recruited by DNA-binding proteins that interact directly with the cis-acting elements. The Sir complex then propagates along an array of nucleosomes. Current evidence supports a sequential assembly model for Sir spreading where the histone deacetylase (HDAC) Sir2p removes acetyl groups from lysines on nearby nucleosomal histone tails and this promotes the direct binding of Sir3p and Sir4p to histone H3 and H4 N-terminal tails, with the Sir complex showing a preference for interacting with the hypoacetylated H4 tail (22,23). Studies on the chromatin boundary activity that restricts Sir-dependent spreading of heterochromatin indicate that numerous proteins associated with histone modification or chromatin remodeling, including Sas2p, Gcn5p, Bdf1p and H2A.Z, are involved in blocking gene silencing (24–26). Since the extent of histone acetylation is increased in transcriptionally active regions, acetylated histones are believed to facilitate an open and loose form of chromatin. A model where increased histone acetylation leads to the formation of euchromatin and prevents the spreading of silent chromatin is supported by several lines of experimental evidence. For example, histone acetyltransferase (HAT) Sas2p and the HDAC Sir2p compete to acetylate and deacetylate yeast histone H4K16, respectively, and the acetylation status of this lysine affects spreading of heterochromatin through a DNA sequence (18). Additionally, the HATs Esa1p and/or Gcn5p create a sizable region of hyperacetylated chromatin which serves as a barrier that can inhibit the propagation of silenced chromatin (27).In contrast to the established transcription repression roles associated with HDACs, the S. cerevisiae Rpd3p, which is a class I HDAC (28,29), appears to be required for transcriptional activation of specific genes (28–30). Deletion of RPD3 enhances the silencing of reporter genes inserted into ribosomal DNA (rDNA), the silent mating type locus and subtelomeric loci (31). Interestingly, when RPD3 and SIR2 (or SIR4) are simultaneously deleted, the expression of reporter genes were restored to wild-type levels (31). A genome-wide transcription profile of rpd3Δ cells also demonstrated that ∼40% of endogenous genes located within 20 kb of telomeres are down-regulated by the RPD3 deletion (32). These lines of evidence support a model where Rpd3p may antagonize the local spread of Sir-mediated silencing from heterochromatin to neighboring euchromatic regions, thus helping to define a heterochromatin boundary. How Rpd3p might function to establish and maintain this heterochromatin boundary remains elusive.In this study, we performed a screen for genes that affect chromatin boundary activity. Our genetic and biochemical evidence show that the absence of Rpd3p results in Sir-dependent repression of heterochromatin-adjacent regions. In an rpd3Δ mutant, we found that a portion of Sir2p was delocalized from nucleolus and became enriched at the regions of DNA adjacent to telomeres and the silent HM loci. Mutation of either histone H4 at K5 or the HAT gene ESA1 compromised the silencing phenotype associated with RPD3 disruption. The data presented in this manuscript provide insight into the molecular mechanism for the antagonizing–silencing functions of Rpd3p during the formation of heterochromatic boundaries.

INTRODUCTION

The eukaryotic genome is organized into chromosomal domains of distinct structure and function (1). The fraction of chromatin that condenses during mitosis and is found decondensed during the interphase of the cell cycle is termed euchromatin (2). In contrast, constitutively compacted chromatin often found at locations like centromeres and telomeres is called heterochromatin (3,4). In general, euchromatic domains bear transcriptionally active genes, whereas heterochromatic domains are largely inactive transcriptionally, leading to a silencing position effect on genes in the heterochromatic region (5,6). Heterochromatin forms a nuclease-resistant structure that can propagate along the chromosome and repress nearby genes in a stochastic manner (2,7). Boundary elements are often found between heterochromatic and euchromatic regions. The prevailing view of boundary elements, or insulators, is that they are specific DNA elements that actively recruit barrier proteins to inhibit the spread of silent chromatin into euchromatic regions, thereby insulating a euchromatic gene from the influence of silent chromatin that could spread into that transcriptionally active region (8–10). Some boundary elements can constitutively recruit epigenetic modification machineries, acting as a chain terminator to the spreading of a repressive chromatin (11–15). Other chromatin boundaries are defined by a gradient of chromatin modifications, such as differing degrees of histone hyperacetylation or hypoacetylation on opposing sides of the resulting boundary element (16–18). Positions of boundary elements can vary depending on the balance of chromatin modifications resulting from the sum of activities of different enzymatic proteins or complexes (19).

The mating loci HMR and HML and the telomeres of Saccharomyces cerevisiae are well-characterized silenced chromatin domains that provide distinctive models for studying the formation of heterochromatin structure and the establishment of chromatin boundaries (12,13,20,21). Heterochromatin propagation depends on the roles of cis-acting elements, such as the telomeric DNA repeats and the sites flanking each HM locus that are known as silencers, as well as trans-acting proteins like the silent information regulator (Sir) complex of proteins and specific silencer-binding or telomere-binding proteins (6,11). The Sir complex, which contains Sir2p, Sir3p and Sir4p, is recruited by DNA-binding proteins that interact directly with the cis-acting elements. The Sir complex then propagates along an array of nucleosomes. Current evidence supports a sequential assembly model for Sir spreading where the histone deacetylase (HDAC) Sir2p removes acetyl groups from lysines on nearby nucleosomal histone tails and this promotes the direct binding of Sir3p and Sir4p to histone H3 and H4 N-terminal tails, with the Sir complex showing a preference for interacting with the hypoacetylated H4 tail (22,23). Studies on the chromatin boundary activity that restricts Sir-dependent spreading of heterochromatin indicate that numerous proteins associated with histone modification or chromatin remodeling, including Sas2p, Gcn5p, Bdf1p and H2A.Z, are involved in blocking gene silencing (24–26). Since the extent of histone acetylation is increased in transcriptionally active regions, acetylated histones are believed to facilitate an open and loose form of chromatin. A model where increased histone acetylation leads to the formation of euchromatin and prevents the spreading of silent chromatin is supported by several lines of experimental evidence. For example, histone acetyltransferase (HAT) Sas2p and the HDAC Sir2p compete to acetylate and deacetylate yeast histone H4K16, respectively, and the acetylation status of this lysine affects spreading of heterochromatin through a DNA sequence (18). Additionally, the HATs Esa1p and/or Gcn5p create a sizable region of hyperacetylated chromatin which serves as a barrier that can inhibit the propagation of silenced chromatin (27).

In contrast to the established transcription repression roles associated with HDACs, the S. cerevisiae Rpd3p, which is a class I HDAC (28,29), appears to be required for transcriptional activation of specific genes (28–30). Deletion of RPD3 enhances the silencing of reporter genes inserted into ribosomal DNA (rDNA), the silent mating type locus and subtelomeric loci (31). Interestingly, when RPD3 and SIR2 (or SIR4) are simultaneously deleted, the expression of reporter genes were restored to wild-type levels (31). A genome-wide transcription profile of rpd3Δ cells also demonstrated that ∼40% of endogenous genes located within 20 kb of telomeres are down-regulated by the RPD3 deletion (32). These lines of evidence support a model where Rpd3p may antagonize the local spread of Sir-mediated silencing from heterochromatin to neighboring euchromatic regions, thus helping to define a heterochromatin boundary. How Rpd3p might function to establish and maintain this heterochromatin boundary remains elusive.

In this study, we performed a screen for genes that affect chromatin boundary activity. Our genetic and biochemical evidence show that the absence of Rpd3p results in Sir-dependent repression of heterochromatin-adjacent regions. In an rpd3Δ mutant, we found that a portion of Sir2p was delocalized from nucleolus and became enriched at the regions of DNA adjacent to telomeres and the silent HM loci. Mutation of either histone H4 at K5 or the HAT gene ESA1 compromised the silencing phenotype associated with RPD3 disruption. The data presented in this manuscript provide insight into the molecular mechanism for the antagonizing–silencing functions of Rpd3p during the formation of heterochromatic boundaries.

MATERIALS AND METHODSPlasmids and yeast strainsPlasmids used in this study are listed as following. Vectors pRS303, pRS305, pRS306, pRS315, pRS316 and pRS414 are described elsewhere (33). The rpd3Δ::LEU2 disruption construct, pRS305–RPD3CN, was generated by cloning the PCR-amplified HindIII–XhoI (nucleotides –550–0) and BamHI–HindIII (nucleotides 1302–2102) fragments of RPD3 into XhoI–BamHI site of pRS305. The XhoI–BamHI fragments of pRS305–RPD3CN were sub-cloned into BamHI and XhoI double digested pRS306 to give rise to pRS306–RPD3CN. pRS303–RPD3CN was constructed as pRS305–RPD3CN except the BamHI-EcoRI–XhoI sites were used. pRS303–SIR2CN was constructed by cloning the PCR-amplified EcoRI–XhoI fragment (nucleotides –306–0) and BamHI–EcoRI fragment (nucleotides 1578–1894) into XhoI–BamHI site of pRS303. pRR608 was generated by inserting the PCR-derived DNA fragment covering the desired RPD3 sequences and having BamHI sites attached into pRS315. Mutant versions of the rpd3-born plasmids pRR610 and pRR611 were obtained according to the protocol of PCR-based mutagenesis. Plasmid pET001 contained the full-length ESA1 was inserted into the BamHI–XbaI site of pRS315. PCR-based mutagenesis was used to generate the esa1 mutant version pET002. Plasmids pNS329 and pMS329, harboring HHF1–HHT1 on pRS414 and YRp14CEN4, respectively, were described previously (34). Plasmids pHR613, pHR616 and pHR620 were derived from pNS329, and site-directed mutagenesis was used to create the substituted sequences. All the derivative mutants were verified by DNA sequencing. Plasmid pRO363, containing a HMRΔI SacI–SalI fragment from pRO22, with a BamHI site engineered in the Mata2 gene cloned into pRS406, was described previously (12). pRO466 was constructed by PCR amplification of HMR tRNAThr(AGT) CR1 from chromosome III with BamHI sites in the primers and inserted into the BamHI sites of pRO363 (35).The yeast strains used in this study are listed in Table S1. The wild-type strains BY4741, BY4742 and deletion derivatives were described previously (36). Disruption of RPD3 was accomplished by transforming BY4742 wild-type strain with EcoRI linearized plasmid pRS305–RPD3CN and verified by PCR. The catalytic deficient mutants of RPD3 were constructed by introducing plasmids pRR608, pRR610, pRR611 into RPD3 disrupted strain JQB001, respectively. The Sir2p–13Myc, Rpd3p–13Myc and Htz1p–3HA expressing strains were obtained by introducing 13Myc or 3HA epitopes to the C terminus of SIR2, RPD3 and HTZ1 following standard PCR-based procedure (37). The ESA1 wild type and esa1 mutant strains JBQ061 and JBQ062 were constructed by transforming ESA1/esa1Δ diploid strain with pET001 and pET002, respectively, and the haploid strains were obtained from tetrad dissection selected with G418 and LEU2 marker. Strains carrying different histone mutations were constructed by transforming MX1–4C (kindly provided by Dr Morse lab), in which the wild-type HHF1–HHT1 was carried on a URA3-marked plasmid, with TRP1-marked plasmids harboring the appropriate mutant histone genes, and counter-selection being done on 5′-fluoroorotic acid (5′-FOA) plates. To construct the URA3–HMR-R and URA3–TELIX-R reporter strains, PCR products containing the full-length URA3 gene were transformed into specific strains. A site-targeted integration of URA3 gene was achieved by direct PCR-mediated homologous recombination. The resulting transformants were verified by PCR analysis. To construct the strains used to assay the influences of rpd3Δ on the boundary function of HMR-tRNA gene, the plasmid pRO363 (no boundary) or pRO466 (HMR-tRNAThr inserted) were transformed into BY4742 wild-type and rpd3Δ mutant strains, and URA+ transformants were isolated (12).Quantitative reverse transcription PCR (qRT–PCR)Total RNA was isolated from cells grown in a concentration of ∼1.0 × 107 cells/ml with RNeasy Mini Kit (Qiagen) and digested with RNase-free DNase (Qiagen). cDNA was synthesized using M–MLV Reverse Transcriptase System and oligo(dT) (Promega). One microliter of the RT reaction was used in the subsequent quantitative PCR reaction. cDNA was analyzed using an Applied Biosystems 7500 Fast system and Power SYBR Green PCR Master Mix (Applied Biosystems).URA3 silencing assaySilencing at telomeric and HMR boundary loci was scored as described previously (38). In brief, logarithmically growing cells whose genome contained a URA3 gene integrated at either the right end of chromosome IX (URA3–TELIX-R) or at regions adjacent to the right side of HMR loci (URA3–HMR-R) were serially diluted in 10-fold increments, were spotted onto the yeast complete plates with or without ∼0.1% 5′-FOA, and were incubated at 30°C. Growth was documented at 48 or 72 h as indicated.Mating assayMating assays to determine the influence of the RPD3 deletion on the boundary activity of an HMR-tRNA were performed as described previously (12). The HMR-tRNA boundary activity test strains (JQB071∼JQB074, Ura+ marked) harbored a modified HMR locus that was deleted for the HMR-I silencer and contained the downstream boundary sequence (with HMR-tRNA gene inserted or no insert) cloned into the HMRa2 gene. The test stains (MATα, Ura+) were grown to log phase and were incubated with Lys+ MATa strains for 4 h. Cells were then serial diluted 5-fold and spotted onto appropriately supplemented plates, and allowed to grow at 30°C for 48 h. Successful mating resulted in normal growth of cells on Ura-/Lys- plates. Insertion of HMR-tRNA gene caused non-mating phenotype of the cells, as the tRNA boundary blocked the spreading of silencing from HMR-E, allowing the a1 gene to be expressed in the MATα cells (12). The effect of deletion of RPD3 gene on HMR-tRNA boundary activity was analyzed by comparing the mating efficiency of wild type and rpd3Δ cells.Chromatin immunoprecipitation assayChromatin immunoprecipitation (ChIP) assays were performed as described previously (39) with some modification. Briefly, yeast cells were cross-linked with 1% formaldehyde and suspended in lysis buffer (50 mM HEPES, [pH 7.5], 140 mM NaCl, 1 mM EDTA, 1% Triton X-100, 0.1% Na-Deoxycholate, 1 mM PMSF, protease inhibitors cocktail). Cells were lysed using glass beads and were sonicated to shear the chromatin to fragment sizes of ∼200–500 base pairs. Cross-linked chromatin fragment were immunoprecipitated with antibodies that specifically recognized Myc or HA epitope tags, acetylated lysines (K5Ac, K8Ac and K12Ac) of the H4 histone tail (catalog number 07-327, 07-328, 07-595, Upstate), respectively. Protein G/A-Sepharose beads (GE) were then added into the samples and the immunoprecipitated complexes were washed with lysis buffer, lysis buffer containing 500 mM NaCl, wash buffer (50 mM HEPES, [pH 7.9], 300 mM NaCl, 1 mM EDTA, 1% Triton X-100, 0.5% NP-40, 0.1% Na-Deoxycholate), and TE (10 mM Tris HCl [pH 8.0], 1 mM EDTA). Next, the immunoprecipitated chromatin was eluted from beads with elution buffer (10 mM Tris–HCl [pH 8.0], 1 mM EDTA, 1% SDS). Formaldehyde cross-linking was reversed by incubating the eluates at 65°C overnight. Eluted DNA was treated with 100 µg/ml proteinase K and purified with QIAquick PCR purification Kit (Qiagen).Immunoprecipitated fractions and whole-cell extracts containing DNA were analyzed by PCR. Quantitative PCR was performed with Power SYBR Green PCR Master Mix (Applied Biosystems). Primers used in the PCR reactions were analyzed for the appropriate range of linearity and efficiency in order to accurately evaluate DNA occupancy by the protein (percent of IP/input). Relative enrichment values of Sir2p, histone H4 (acetylated K5, K8 and K12), Rpd3p and H2A.Z were normalized to the internal control ARO1, TEL0.5, TEL0.5 and PRP8, respectively, and these in turn were normalized to the corresponding input whole-cell extract.Immunofluorescence on yeast cellsCells were grown in YPD medium overnight to a density of ∼1 × 107 to 2 × 107 cells/ml and were fixed for 30 min by incubation with 3.7% formaldehyde. Next, cells were washed with 0.1 M potassium phosphate (pH 6.5) and P solution (1.2 M sorbitol, 1 M K2PO4), and re-suspended in P solution. Cells were subsequently treated with 0.1 mg/ml Zymolyase (20T, MP Biomedicals) for 10 min, washed with P solution, spotted on Poly-l-Lysine pre-treated slides. After rinsing in PBS-T buffer (PBS containing 0.1% Triton X-100 and 1% BSA), slides were incubated overnight at 4°C with anti-Myc, anti-Rap1 and anti-Nop1 antibody diluted in PBS containing 1% BSA. Slides were then washed with PBS-T and incubated with the appropriate secondary antibodies conjugated to Cy3 or fluorescein isothiocyanate (FITC). The DNA fluorescence signal was detected by DAPI (1 µg/ml in Phosphate Buffered Saline (PBS) solution) staining. Slides were mounted with PBS containing 1 mg/ml p-phenylenediamine, 2.5 µM NaOH and 90% glycerol.Confocal microscopy was performed on a Leica TCS SP2 microscope with a 63× lamda blue objective (oil). Image processing including similar filtration and threshold levels was standardized for all images.

MATERIALS AND METHODS

Plasmids and yeast strainsPlasmids used in this study are listed as following. Vectors pRS303, pRS305, pRS306, pRS315, pRS316 and pRS414 are described elsewhere (33). The rpd3Δ::LEU2 disruption construct, pRS305–RPD3CN, was generated by cloning the PCR-amplified HindIII–XhoI (nucleotides –550–0) and BamHI–HindIII (nucleotides 1302–2102) fragments of RPD3 into XhoI–BamHI site of pRS305. The XhoI–BamHI fragments of pRS305–RPD3CN were sub-cloned into BamHI and XhoI double digested pRS306 to give rise to pRS306–RPD3CN. pRS303–RPD3CN was constructed as pRS305–RPD3CN except the BamHI-EcoRI–XhoI sites were used. pRS303–SIR2CN was constructed by cloning the PCR-amplified EcoRI–XhoI fragment (nucleotides –306–0) and BamHI–EcoRI fragment (nucleotides 1578–1894) into XhoI–BamHI site of pRS303. pRR608 was generated by inserting the PCR-derived DNA fragment covering the desired RPD3 sequences and having BamHI sites attached into pRS315. Mutant versions of the rpd3-born plasmids pRR610 and pRR611 were obtained according to the protocol of PCR-based mutagenesis. Plasmid pET001 contained the full-length ESA1 was inserted into the BamHI–XbaI site of pRS315. PCR-based mutagenesis was used to generate the esa1 mutant version pET002. Plasmids pNS329 and pMS329, harboring HHF1–HHT1 on pRS414 and YRp14CEN4, respectively, were described previously (34). Plasmids pHR613, pHR616 and pHR620 were derived from pNS329, and site-directed mutagenesis was used to create the substituted sequences. All the derivative mutants were verified by DNA sequencing. Plasmid pRO363, containing a HMRΔI SacI–SalI fragment from pRO22, with a BamHI site engineered in the Mata2 gene cloned into pRS406, was described previously (12). pRO466 was constructed by PCR amplification of HMR tRNAThr(AGT) CR1 from chromosome III with BamHI sites in the primers and inserted into the BamHI sites of pRO363 (35).The yeast strains used in this study are listed in Table S1. The wild-type strains BY4741, BY4742 and deletion derivatives were described previously (36). Disruption of RPD3 was accomplished by transforming BY4742 wild-type strain with EcoRI linearized plasmid pRS305–RPD3CN and verified by PCR. The catalytic deficient mutants of RPD3 were constructed by introducing plasmids pRR608, pRR610, pRR611 into RPD3 disrupted strain JQB001, respectively. The Sir2p–13Myc, Rpd3p–13Myc and Htz1p–3HA expressing strains were obtained by introducing 13Myc or 3HA epitopes to the C terminus of SIR2, RPD3 and HTZ1 following standard PCR-based procedure (37). The ESA1 wild type and esa1 mutant strains JBQ061 and JBQ062 were constructed by transforming ESA1/esa1Δ diploid strain with pET001 and pET002, respectively, and the haploid strains were obtained from tetrad dissection selected with G418 and LEU2 marker. Strains carrying different histone mutations were constructed by transforming MX1–4C (kindly provided by Dr Morse lab), in which the wild-type HHF1–HHT1 was carried on a URA3-marked plasmid, with TRP1-marked plasmids harboring the appropriate mutant histone genes, and counter-selection being done on 5′-fluoroorotic acid (5′-FOA) plates. To construct the URA3–HMR-R and URA3–TELIX-R reporter strains, PCR products containing the full-length URA3 gene were transformed into specific strains. A site-targeted integration of URA3 gene was achieved by direct PCR-mediated homologous recombination. The resulting transformants were verified by PCR analysis. To construct the strains used to assay the influences of rpd3Δ on the boundary function of HMR-tRNA gene, the plasmid pRO363 (no boundary) or pRO466 (HMR-tRNAThr inserted) were transformed into BY4742 wild-type and rpd3Δ mutant strains, and URA+ transformants were isolated (12).

Plasmids and yeast strains

Plasmids used in this study are listed as following. Vectors pRS303, pRS305, pRS306, pRS315, pRS316 and pRS414 are described elsewhere (33). The rpd3Δ::LEU2 disruption construct, pRS305–RPD3CN, was generated by cloning the PCR-amplified HindIII–XhoI (nucleotides –550–0) and BamHI–HindIII (nucleotides 1302–2102) fragments of RPD3 into XhoI–BamHI site of pRS305. The XhoI–BamHI fragments of pRS305–RPD3CN were sub-cloned into BamHI and XhoI double digested pRS306 to give rise to pRS306–RPD3CN. pRS303–RPD3CN was constructed as pRS305–RPD3CN except the BamHI-EcoRI–XhoI sites were used. pRS303–SIR2CN was constructed by cloning the PCR-amplified EcoRI–XhoI fragment (nucleotides –306–0) and BamHI–EcoRI fragment (nucleotides 1578–1894) into XhoI–BamHI site of pRS303. pRR608 was generated by inserting the PCR-derived DNA fragment covering the desired RPD3 sequences and having BamHI sites attached into pRS315. Mutant versions of the rpd3-born plasmids pRR610 and pRR611 were obtained according to the protocol of PCR-based mutagenesis. Plasmid pET001 contained the full-length ESA1 was inserted into the BamHI–XbaI site of pRS315. PCR-based mutagenesis was used to generate the esa1 mutant version pET002. Plasmids pNS329 and pMS329, harboring HHF1–HHT1 on pRS414 and YRp14CEN4, respectively, were described previously (34). Plasmids pHR613, pHR616 and pHR620 were derived from pNS329, and site-directed mutagenesis was used to create the substituted sequences. All the derivative mutants were verified by DNA sequencing. Plasmid pRO363, containing a HMRΔI SacI–SalI fragment from pRO22, with a BamHI site engineered in the Mata2 gene cloned into pRS406, was described previously (12). pRO466 was constructed by PCR amplification of HMR tRNAThr(AGT) CR1 from chromosome III with BamHI sites in the primers and inserted into the BamHI sites of pRO363 (35).

The yeast strains used in this study are listed in Table S1. The wild-type strains BY4741, BY4742 and deletion derivatives were described previously (36). Disruption of RPD3 was accomplished by transforming BY4742 wild-type strain with EcoRI linearized plasmid pRS305–RPD3CN and verified by PCR. The catalytic deficient mutants of RPD3 were constructed by introducing plasmids pRR608, pRR610, pRR611 into RPD3 disrupted strain JQB001, respectively. The Sir2p–13Myc, Rpd3p–13Myc and Htz1p–3HA expressing strains were obtained by introducing 13Myc or 3HA epitopes to the C terminus of SIR2, RPD3 and HTZ1 following standard PCR-based procedure (37). The ESA1 wild type and esa1 mutant strains JBQ061 and JBQ062 were constructed by transforming ESA1/esa1Δ diploid strain with pET001 and pET002, respectively, and the haploid strains were obtained from tetrad dissection selected with G418 and LEU2 marker. Strains carrying different histone mutations were constructed by transforming MX1–4C (kindly provided by Dr Morse lab), in which the wild-type HHF1–HHT1 was carried on a URA3-marked plasmid, with TRP1-marked plasmids harboring the appropriate mutant histone genes, and counter-selection being done on 5′-fluoroorotic acid (5′-FOA) plates. To construct the URA3–HMR-R and URA3–TELIX-R reporter strains, PCR products containing the full-length URA3 gene were transformed into specific strains. A site-targeted integration of URA3 gene was achieved by direct PCR-mediated homologous recombination. The resulting transformants were verified by PCR analysis. To construct the strains used to assay the influences of rpd3Δ on the boundary function of HMR-tRNA gene, the plasmid pRO363 (no boundary) or pRO466 (HMR-tRNAThr inserted) were transformed into BY4742 wild-type and rpd3Δ mutant strains, and URA+ transformants were isolated (12).

Quantitative reverse transcription PCR (qRT–PCR)Total RNA was isolated from cells grown in a concentration of ∼1.0 × 107 cells/ml with RNeasy Mini Kit (Qiagen) and digested with RNase-free DNase (Qiagen). cDNA was synthesized using M–MLV Reverse Transcriptase System and oligo(dT) (Promega). One microliter of the RT reaction was used in the subsequent quantitative PCR reaction. cDNA was analyzed using an Applied Biosystems 7500 Fast system and Power SYBR Green PCR Master Mix (Applied Biosystems).

Quantitative reverse transcription PCR (qRT–PCR)

Total RNA was isolated from cells grown in a concentration of ∼1.0 × 107 cells/ml with RNeasy Mini Kit (Qiagen) and digested with RNase-free DNase (Qiagen). cDNA was synthesized using M–MLV Reverse Transcriptase System and oligo(dT) (Promega). One microliter of the RT reaction was used in the subsequent quantitative PCR reaction. cDNA was analyzed using an Applied Biosystems 7500 Fast system and Power SYBR Green PCR Master Mix (Applied Biosystems).

URA3 silencing assaySilencing at telomeric and HMR boundary loci was scored as described previously (38). In brief, logarithmically growing cells whose genome contained a URA3 gene integrated at either the right end of chromosome IX (URA3–TELIX-R) or at regions adjacent to the right side of HMR loci (URA3–HMR-R) were serially diluted in 10-fold increments, were spotted onto the yeast complete plates with or without ∼0.1% 5′-FOA, and were incubated at 30°C. Growth was documented at 48 or 72 h as indicated.

URA3 silencing assay

Silencing at telomeric and HMR boundary loci was scored as described previously (38). In brief, logarithmically growing cells whose genome contained a URA3 gene integrated at either the right end of chromosome IX (URA3–TELIX-R) or at regions adjacent to the right side of HMR loci (URA3–HMR-R) were serially diluted in 10-fold increments, were spotted onto the yeast complete plates with or without ∼0.1% 5′-FOA, and were incubated at 30°C. Growth was documented at 48 or 72 h as indicated.

Mating assayMating assays to determine the influence of the RPD3 deletion on the boundary activity of an HMR-tRNA were performed as described previously (12). The HMR-tRNA boundary activity test strains (JQB071∼JQB074, Ura+ marked) harbored a modified HMR locus that was deleted for the HMR-I silencer and contained the downstream boundary sequence (with HMR-tRNA gene inserted or no insert) cloned into the HMRa2 gene. The test stains (MATα, Ura+) were grown to log phase and were incubated with Lys+ MATa strains for 4 h. Cells were then serial diluted 5-fold and spotted onto appropriately supplemented plates, and allowed to grow at 30°C for 48 h. Successful mating resulted in normal growth of cells on Ura-/Lys- plates. Insertion of HMR-tRNA gene caused non-mating phenotype of the cells, as the tRNA boundary blocked the spreading of silencing from HMR-E, allowing the a1 gene to be expressed in the MATα cells (12). The effect of deletion of RPD3 gene on HMR-tRNA boundary activity was analyzed by comparing the mating efficiency of wild type and rpd3Δ cells.

Mating assay

Mating assays to determine the influence of the RPD3 deletion on the boundary activity of an HMR-tRNA were performed as described previously (12). The HMR-tRNA boundary activity test strains (JQB071∼JQB074, Ura+ marked) harbored a modified HMR locus that was deleted for the HMR-I silencer and contained the downstream boundary sequence (with HMR-tRNA gene inserted or no insert) cloned into the HMRa2 gene. The test stains (MATα, Ura+) were grown to log phase and were incubated with Lys+ MATa strains for 4 h. Cells were then serial diluted 5-fold and spotted onto appropriately supplemented plates, and allowed to grow at 30°C for 48 h. Successful mating resulted in normal growth of cells on Ura-/Lys- plates. Insertion of HMR-tRNA gene caused non-mating phenotype of the cells, as the tRNA boundary blocked the spreading of silencing from HMR-E, allowing the a1 gene to be expressed in the MATα cells (12). The effect of deletion of RPD3 gene on HMR-tRNA boundary activity was analyzed by comparing the mating efficiency of wild type and rpd3Δ cells.

Chromatin immunoprecipitation assayChromatin immunoprecipitation (ChIP) assays were performed as described previously (39) with some modification. Briefly, yeast cells were cross-linked with 1% formaldehyde and suspended in lysis buffer (50 mM HEPES, [pH 7.5], 140 mM NaCl, 1 mM EDTA, 1% Triton X-100, 0.1% Na-Deoxycholate, 1 mM PMSF, protease inhibitors cocktail). Cells were lysed using glass beads and were sonicated to shear the chromatin to fragment sizes of ∼200–500 base pairs. Cross-linked chromatin fragment were immunoprecipitated with antibodies that specifically recognized Myc or HA epitope tags, acetylated lysines (K5Ac, K8Ac and K12Ac) of the H4 histone tail (catalog number 07-327, 07-328, 07-595, Upstate), respectively. Protein G/A-Sepharose beads (GE) were then added into the samples and the immunoprecipitated complexes were washed with lysis buffer, lysis buffer containing 500 mM NaCl, wash buffer (50 mM HEPES, [pH 7.9], 300 mM NaCl, 1 mM EDTA, 1% Triton X-100, 0.5% NP-40, 0.1% Na-Deoxycholate), and TE (10 mM Tris HCl [pH 8.0], 1 mM EDTA). Next, the immunoprecipitated chromatin was eluted from beads with elution buffer (10 mM Tris–HCl [pH 8.0], 1 mM EDTA, 1% SDS). Formaldehyde cross-linking was reversed by incubating the eluates at 65°C overnight. Eluted DNA was treated with 100 µg/ml proteinase K and purified with QIAquick PCR purification Kit (Qiagen).Immunoprecipitated fractions and whole-cell extracts containing DNA were analyzed by PCR. Quantitative PCR was performed with Power SYBR Green PCR Master Mix (Applied Biosystems). Primers used in the PCR reactions were analyzed for the appropriate range of linearity and efficiency in order to accurately evaluate DNA occupancy by the protein (percent of IP/input). Relative enrichment values of Sir2p, histone H4 (acetylated K5, K8 and K12), Rpd3p and H2A.Z were normalized to the internal control ARO1, TEL0.5, TEL0.5 and PRP8, respectively, and these in turn were normalized to the corresponding input whole-cell extract.

Chromatin immunoprecipitation assay

Chromatin immunoprecipitation (ChIP) assays were performed as described previously (39) with some modification. Briefly, yeast cells were cross-linked with 1% formaldehyde and suspended in lysis buffer (50 mM HEPES, [pH 7.5], 140 mM NaCl, 1 mM EDTA, 1% Triton X-100, 0.1% Na-Deoxycholate, 1 mM PMSF, protease inhibitors cocktail). Cells were lysed using glass beads and were sonicated to shear the chromatin to fragment sizes of ∼200–500 base pairs. Cross-linked chromatin fragment were immunoprecipitated with antibodies that specifically recognized Myc or HA epitope tags, acetylated lysines (K5Ac, K8Ac and K12Ac) of the H4 histone tail (catalog number 07-327, 07-328, 07-595, Upstate), respectively. Protein G/A-Sepharose beads (GE) were then added into the samples and the immunoprecipitated complexes were washed with lysis buffer, lysis buffer containing 500 mM NaCl, wash buffer (50 mM HEPES, [pH 7.9], 300 mM NaCl, 1 mM EDTA, 1% Triton X-100, 0.5% NP-40, 0.1% Na-Deoxycholate), and TE (10 mM Tris HCl [pH 8.0], 1 mM EDTA). Next, the immunoprecipitated chromatin was eluted from beads with elution buffer (10 mM Tris–HCl [pH 8.0], 1 mM EDTA, 1% SDS). Formaldehyde cross-linking was reversed by incubating the eluates at 65°C overnight. Eluted DNA was treated with 100 µg/ml proteinase K and purified with QIAquick PCR purification Kit (Qiagen).

Immunoprecipitated fractions and whole-cell extracts containing DNA were analyzed by PCR. Quantitative PCR was performed with Power SYBR Green PCR Master Mix (Applied Biosystems). Primers used in the PCR reactions were analyzed for the appropriate range of linearity and efficiency in order to accurately evaluate DNA occupancy by the protein (percent of IP/input). Relative enrichment values of Sir2p, histone H4 (acetylated K5, K8 and K12), Rpd3p and H2A.Z were normalized to the internal control ARO1, TEL0.5, TEL0.5 and PRP8, respectively, and these in turn were normalized to the corresponding input whole-cell extract.

Immunofluorescence on yeast cellsCells were grown in YPD medium overnight to a density of ∼1 × 107 to 2 × 107 cells/ml and were fixed for 30 min by incubation with 3.7% formaldehyde. Next, cells were washed with 0.1 M potassium phosphate (pH 6.5) and P solution (1.2 M sorbitol, 1 M K2PO4), and re-suspended in P solution. Cells were subsequently treated with 0.1 mg/ml Zymolyase (20T, MP Biomedicals) for 10 min, washed with P solution, spotted on Poly-l-Lysine pre-treated slides. After rinsing in PBS-T buffer (PBS containing 0.1% Triton X-100 and 1% BSA), slides were incubated overnight at 4°C with anti-Myc, anti-Rap1 and anti-Nop1 antibody diluted in PBS containing 1% BSA. Slides were then washed with PBS-T and incubated with the appropriate secondary antibodies conjugated to Cy3 or fluorescein isothiocyanate (FITC). The DNA fluorescence signal was detected by DAPI (1 µg/ml in Phosphate Buffered Saline (PBS) solution) staining. Slides were mounted with PBS containing 1 mg/ml p-phenylenediamine, 2.5 µM NaOH and 90% glycerol.Confocal microscopy was performed on a Leica TCS SP2 microscope with a 63× lamda blue objective (oil). Image processing including similar filtration and threshold levels was standardized for all images.

Immunofluorescence on yeast cells

Cells were grown in YPD medium overnight to a density of ∼1 × 107 to 2 × 107 cells/ml and were fixed for 30 min by incubation with 3.7% formaldehyde. Next, cells were washed with 0.1 M potassium phosphate (pH 6.5) and P solution (1.2 M sorbitol, 1 M K2PO4), and re-suspended in P solution. Cells were subsequently treated with 0.1 mg/ml Zymolyase (20T, MP Biomedicals) for 10 min, washed with P solution, spotted on Poly-l-Lysine pre-treated slides. After rinsing in PBS-T buffer (PBS containing 0.1% Triton X-100 and 1% BSA), slides were incubated overnight at 4°C with anti-Myc, anti-Rap1 and anti-Nop1 antibody diluted in PBS containing 1% BSA. Slides were then washed with PBS-T and incubated with the appropriate secondary antibodies conjugated to Cy3 or fluorescein isothiocyanate (FITC). The DNA fluorescence signal was detected by DAPI (1 µg/ml in Phosphate Buffered Saline (PBS) solution) staining. Slides were mounted with PBS containing 1 mg/ml p-phenylenediamine, 2.5 µM NaOH and 90% glycerol.

Confocal microscopy was performed on a Leica TCS SP2 microscope with a 63× lamda blue objective (oil). Image processing including similar filtration and threshold levels was standardized for all images.

RESULTSScreen for genes antagonizing heterochromatic silencingTo screen for genes whose deletion might affect the silent chromatin at HMR and telomere loci, we concentrated our effort on 84 genes (Table S2) that have been shown to participate in modulating chromatin structure by such means as histone modification and chromatin remodeling. Quantitative RT–PCR (qRT–PCR) was used to determine relative mRNA levels of three genes located at the boundary regions. YPS6 and YIR042c are adjacent to the telomere of chromosome IX-R and GIT1 is proximal to the HMR right boundary (Figure 1A). These three genes were previously proposed to be located at boundary regions and are sensitive to the spreading of silent chromatin (40). ACT1, whose mRNA level is relatively stable, was used as an internal control. Down-regulation of YPS6, YIR042c or GIT1 transcription was observed when the genes listed in Table 1 were individually deleted. Among those, H2A.Z, Bdf1p, Sas2p, Gcn5p, Rad6p, Rpd3p, Itc1p, Rsc2p, Yta7 and Dpb4p have previously been reported to prevent silent chromatin from spreading to regulate gene transcription (18,20,21,31,40,41). The repression of marker gene expression in the rpd3Δ mutant was significant and comparable to the effects of inactivation of other known anti-silencing factors (Figure 1B). Additionally, other genes that carry novel anti-silencing function were identified, including histone acetyltransferase Hpa2p; peptidyl–prolyl cis–trans isomerase Fpr4p; Swc4p and Vps71p from the SWR1 complex; Isw1p and Isw2p from the ISW1 complex; Snf2p, Snf5p and Snf6p from the SWI/SNF complex; Npl6p and Rsc1p from the RSC chromatin remodeling complex; RNA polymerase II-associated proteins Paf1p and Cdc73p; and Taf14p and Ies3p from the INO80 complex. To our knowledge, these gene products have not previously been shown to carry anti-silencing function, and future investigation into their roles in this process might be warranted.
Table 1.Genes identified in this studyAnalysis subjectGenesChromatin assemblyHTZ1Chromatin componentBDF1Histone modificationSAS2, GCN5, RAD6, HPA2, RPD3Chromatin remodeling    Histone exchangeSWC4, VPS71    ISW1ISW1    TFIIDTAF14    ISW2/ITC1ISW2, ITC1    SWI/SNFSNF2, SNF5, SNF6    INO80IES3    RSCRSC1, RSC2, NPL6    OtherCDC73, PAF1, FPR4, DPB4, YTA7
Figure 1.Rpd3L complex is required for establishment of heterochromatic boundaries. (A) Schematic diagrams of the chromosomal locations of three boundary-proximal genes, YPS6 and YIR042C, which are adjacent to telomere, and GIT1, which is proximal to HMR. The HMR-tRNAThr gene and STAR (sub-telomeric anti-silencing region) sequence are also labeled. (B) qRT–PCR results of mRNA levels of boundary-proximal genes, YPS6, YIR042C and GIT1, in wild type and various mutant strains. (C) qRT–PCR results of mRNA levels of YPS6, YIR042C and GIT1 in the individual component deletion mutants of Rpd3L and Rpd3S. Fold transcription is relative to wild type. The log2 ratio less than zero indicates repression of transcription, whereas greater than zero indicates enhancement of transcription. Error bars represent standard error of the mean for three independent RNA purifications. (D) Mating assay to test the effects of mutation of RPD3 on boundary activity of HMR-tRNA. The ∼1.0 kb region flanking the right side of HMR with the HMR tRNAThr boundary gene was cloned into the a2 gene, and these constructs (pRO363 or pRO466) were integrated into chromosome III in a MATα strain in the presence or absence of Rpd3p (see ‘Materials and Methods’ section). The resulting strains and a MATa strain were mated, serially diluted and spotted onto a YC plate or a Ura–/Lys– plate, followed by incubation at 30°C. The photograph was taken after 48 h.Rpd3 complex counteracts heterochromatic silencingSir2p is a HDAC and its deletion has been shown to cause a loss of gene silencing at HMR, rDNA and telomeric loci (42). In contrast, inactivation of Rpd3p, a class I HDAC, dramatically reduced the transcription of telomere adjacent genes YPS6 and YIR042c, and the HMR proximal gene GIT1 (Figure 1A and B), indicating that RPD3 deletion mutant is defective in heterochromatin boundary activity. The opposing effects of the HDACs Rpd3p and Sir2p on silencing encouraged us to investigate further the role of Rpd3p in boundary element function.Rpd3p participates in two overlapping protein complexes named Rpd3L and Rpd3S (43). To determine if either or both of the Rpd3L and Rpd3S complexes play a general role in anti-silencing, a qRT–PCR assay examining RNA levels of the three potentially silenced genes described above (YPS6, YIR042C and GIT1) was performed in yeast strains deleted for individual components of the Rpd3L and Rpd3S complexes. As shown in Figure 1C, deletion of SIN3, which belongs to both of the complexes, mimicked the repression effect that the RPD3 deletion had on gene expression. Deletion of another overlapping core subunit, UME1, yielded a similar but more modest repression, whereas deletion of Rpd3S-specific subunit RCO1 or EAF3 had little effect on repression. Deletion of the Rpd3L-specific genes DEP1, PHO23, RXT2 or SDS3 enhanced repression significantly at all three locations. Deletion of RXT3, ASH1 and UME6 did not show a repression phenotype. These results are in agreement with previous studies by Keogh et al. (44) and lead us to conclude that the Rpd3L complex, but not the Rpd3S complex, is responsible for the anti-silencing phenotype. It has been suggested by others that Ume6p plays a recruitment role for the Rpd3L complex; however, deletion of UME6 weakened (instead of enhanced) silencing, inconsistent with a recruitment function previously proposed for Ume6p (45). Based on these data, we postulate that the recruitment of Rpd3L complex to the silent chromatin was mediated by a subunit or subunits other than Ume6p.
To validate that Rpd3p affects chromatin boundary activity, a silencing assay was performed with RPD3 deletion mutant strains harboring a URA3 reporter gene integrated at either the HMR right boundary locus or near subtelomeric regions of chromosome IX-R. The positions of the URA3 gene are illustrated in Figure 2A and B, and URA3 expression was monitored by cell growth on medium containing 5′-FOA, which is toxic to cells expressing URA3. Deletion of RPD3 promoted growth in cells with the URA3 gene inserted at ∼2 kb and ∼4 kb from the telomeric X element (∼700 bp to telomeric TG1-3 repeat sequence) of chromosome IX-R (Figure 2A, right). A similar result was obtained with the URA3 gene inserted at ∼1 kb, ∼2 kb and ∼4 kb from the right side of HMR silent cassette (Figure 2b, right). In contrast, at the very proximal subtelomeric locus ∼1 kb from chromosome IX-TEL-R, RPD3 deletion had little influence on relieving the silencing of URA3 and promoting cell growth (Figure 2A, right). These results support a model where Rpd3p is involved in heterochromatin boundary formation and has an anti-silencing function at loci adjacent to heterochromatin.
Figure 2.Enhanced silencing associated with RPD3 deletion is Sir2-dependent. (A) and (B) The wild-type, rpd3Δ, sir2Δ and rpd3Δ sir2Δ cells harboring a URA3 gene integrated at the subtelomeric regions of chromosome IX (TELIX) or HMR-adjacent loci were serially diluted and spotted onto YC medium with or without 5′-FOA, followed by incubation at 30°C. Photographs were taken after 48 or 72 h. The locations of the integrated URA3 gene on the chromosomes are illustrated on left. The numbers designated as 1–6 represent different positions of the URA3 marker. (C) qRT–PCR results of mRNA levels of YPS6, YIR042C and GIT1 genes in rpd3Δ, sir2Δ, sir3Δ, rpd3Δ sir2Δ, rpd3Δ sir3Δ and RPD3 cells. Fold increase in mRNA is relative to wild type. Error bars represent standard error of the mean for three trials.Previous studies revealed that the tRNAThr gene located at ∼1.5 kb downstream of the HMR locus (designated tRNAThr la [AGT] CR1 in the Saccharomyces Genome Database) acts as a cis-element for anti-silencing, and is required and sufficient for boundary activity (21). The expression of GIT1, which is about 4 kb downstream of HMRA1, was down-regulated upon RPD3 deletion, suggesting that inactivation of Rpd3p resulted in a decrease of the tRNA boundary activity. To further address whether mutation of RPD3 could eliminate the barrier function of HMR-tRNA to affect spreading of silent chromatin, we used an HMR-tRNA boundary activity assay where the HMR-I silencer was deleted and the 1.0 kb region downstream of HMR containing the HMR-tRNA gene was cloned into the a2 gene (Figure 1D) (12). The integrated tRNA gene was able to function as a boundary element to block the spread of silent chromatin from HMR-E into the a1 gene, thereby rendering MATα cells incompetent to mate with Mata cells to create a URA+ LYS+ diploid (Figure 1D, WT/tRNA boundary +) (12). When RPD3 was deleted, the mating efficiency of MATα cells was modestly increased (Figure 1D, rpd3Δ/tRNA boundary +), suggesting that Rpd3p facilitates the establishment of tRNA boundary activity at HMR loci.The enhanced repression associated with RPD3 deletion is Sir-dependentTo address whether the decreased expression of the URA3 reporter in the rpd3Δ mutant is Sir-mediated, the 5′-FOA growth phenotype for the rpd3Δsir2Δ mutant was examined. Though the RPD3 deletion increased silencing of URA3, the rpd3Δsir2Δ double mutant restored URA3 expression to the wild-type level (Figure 2A and B), suggesting that the repression of URA3 expression in the rpd3Δ mutant was Sir2p-dependent. A qRT–PCR assay was also performed with mutant strains where the SIR2 or SIR3 genes were individually deleted in the rpd3Δ background. Consistent with the URA3 silencing assay shown in Figure 2A and B, the transcriptional de-repression in both the rpd3Δsir2Δ and rpd3Δsir3Δ double mutants was also observed using the qRTPCR assay (Figure 2C), confirming that the repression associated with rpd3Δ is likely dependent on Sir proteins.Absence of Rpd3p alters the distribution of Sir2pSir2p is the core enzyme of Sir protein complex and an essential component of silent chromatin. The results described above suggested that Rpd3p is required for defining the boundaries that block the Sir-dependent propagation of silent chromatin. To address how Rpd3p might affect Sir2p's ability to regulate silencing propagation at telomeres and HM loci, the immunolocalization of 13Myc–Sir2p was examined in fixed wild-type and rpd3Δ yeast cells using anti-Myc antibody. In wild-type cells, a strong signal was detected within a restricted nuclear sub-domain (Figure 3A), resembling the staining of crescent-shaped nucleolus, along with a weaker punctuated pattern (46). The punctuated Sir2p staining, but not the nucleolar signal, has been previously shown to co-localize with the telomere-binding protein marker Rap1p (46), as indicated by the white arrows in the merging image of Figure 3A. In contrast, the nucleolar localization of Sir2p staining was strikingly weakened in rpd3Δ cells, as indicated by the red arrows in Figure 3B. Instead, the non-nucleolar staining of Sir2p was significantly enhanced and some of these enhanced regions had Sir2p that co-localized with Rap1p. Simultaneous immunostaining of Sir2p and Nop1p showed an intact nucleolus in the rpd3Δ mutant, and co-localization of Sir2p and Nop1p was dramatically weakened by the RPD3 deletion (Figure 3C and D), suggesting the nucleolus was intact but Sir2p had moved away from the nucleolus. In rsc1 or gcn5 deletion mutant cells, the Sir2p distribution was very similar to that in wild-type cells (Supplementary Figure S1A, B and C), suggesting the change in sub-nuclear localization for Sir2p was specifically dependent upon deletion of RPD3. In summary, we found cells deficient for Rpd3p displayed a great amount of Sir2p release from the nucleolus and redistribution to other sub-nuclear loci like telomeres and their adjacent euchromatic regions. This finding suggests a model where Rpd3p influences the propagation of silent chromatin by restricting Sir2p distribution within the nucleus.
Figure 3.Inactivation of Rpd3p causes redistribution of Sir2p. (A) and (B) Confocal images of the immunolocalization of Sir2p and Rap1p in wild type (A) and rpd3Δ (B) cells. A Sir2p–13Myc fusion protein was stained by mouse anti-Myc monoclonal antibody, detected by a Cy3-conjugated secondary antibody. Rap1p was stained by rabbit anti-Rap1 polyclonal antibody, and detected by a FITC-conjugated secondary antibody. Overlap of these two signals is yellow. DNA is stained by DAPI. The bar indicates 2.0 µm. (C) and (D) Immunolocalization of Sir2p–13Myc and Nop1p in wild-type (C) and rpd3Δ (D) cells. Sir2p–13Myc was stained by rabbit anti-Myc antibody, detected by a Cy3-conjugated secondary antibody. Nop1p was stained by mouse anti-Nop1 antibody, and detected by a FITC-conjugated secondary antibody. DNA is stained by DAPI. (E), (F) and (G) 13Myc–Sir2p binding to rDNA (E), HMR (F) and subtelomeric (G) loci was assayed by ChIP using anti-Myc antibody in wild type and rpd3Δ cells. Average relative Sir2p enrichments are shown for each primer set with its amplified region as indicated on the upper panel of each graph. The upper diagrams in (E), (F) and (G) show a representative portion of ribosomal DNA locus on chromosome XII, two prominent boundary regions in the HMR silent mating type cassette and subtelomeric loci found on chromosome IX-R, respectively (41). Two well-characterized boundary elements, the HMR-adjacent tRNAThr gene and STAR sequence of Chr-IX-R are also shown (8,12,14). Locations of primer sets used for ChIP analysis are designated by the distance to the start coden of HMRA1 gene, or to the start of telomeric X element (∼700 bp to telomere TG1-3 repeats sequence) of chromosome IX. The qPCR data were normalized to an internal control (ARO1) and the input DNA. The results are average of three independent ChIPs with error bars shown for the standard error of the mean for three independent experiments.
To analyse further the redistribution of Sir2p in rpd3Δ cells, we performed a chromatin immunoprecipation (ChIP) experiment to detect Sir2p at rDNA, HMR and telomeric loci. The schematic diagrams in the upper panel of Figure 3E, F and G showed the respective regions of rDNA, HMR and chromosome IX right arm telomere (Chr IX-TEL-R) that were tested. The DNA fragments labeled in these diagrams were amplified individually in the ChIP assays. The precise location of the DNA sequences examined by ChIP are presented as the indicated genes in and near the rDNA array (Figure 3E), the distance in kb from the start codon of the HMRA1 gene (Figure 3F) or the distance in kb from the X element on the right arm of chromosome IX (Figure 3G). A gene in euchromatic region on chromosome-IV, ARO1, was used for normalization (47). The ChIP result shown in Figure 3E revealed that Sir2p bound to rDNA was not lost entirely, but was moderately decreased in rpd3Δ cells. Correspondingly, the Sir2p binding at locations between 0.6–4.1 kb around the silent mating type cassette (including the location of HMR-tRNA gene) was enhanced in rpd3Δ cells (Figure 3F). In wild-type cells, the subtelomeric regions of chromosome IX-R showed a significant drop-off in Sir2p bindings at regions of 2.0 kb away from chromosome IX-R telomere X element in wild-type cells (Figure 3G). However, in rpd3Δ mutant cells, Sir2p binding was enhanced at telomere distal regions between 1.5 and 6.5 kb away from the telomeric X element. Interestingly, RPD3 deletion had little influence on the binding of Sir2p at the 1.0 kb site near the telomere. These results are consistent with the differing Sir2p immunostaining microscopy results obtained in wild-type and rpd3Δ cells (Figure 3B), and provide further support for a model where in rpd3Δ cells, a portion of Sir2p is delocalized from nucleolus and redistributed to regions adjacent to already silent chromatin (such as telomere and HMR loci), thereby establishing a new boundary location.Deacetylase activity is required for the anti-silencing effect of Rpd3pTo determine whether the HDAC activity of Rpd3p is required for counteracting heterochromatic silencing, we constructed the enzymatically defective forms of Rpd3p in which the conserved histidine residues at positions 151 or 188 were substituted with alanine, abolishing the catalytic activity while reserving the stability and integrity of the Rpd3p complex (48). qRT–PCR assays revealed that in the rpd3-H151A and -H188A mutants, the reporter genes YPS6, YIR042c and GIT1 remained repressed as seen in rpd3Δ cells (Figure 4A). Accordingly, a ChIP assay showed that, like in rpd3Δ mutant, the binding of Sir2p in the rpd3-H151A mutant was enhanced at the boundary of HMR locus (e.g. from 0.6 to 4.1 kb on chromosome III-HMR) or at subtelomeric regions (e.g. from 1.5 to 6.5 kb on chromosome IX-TEL-R). At the already silenced regions (noted by the 1.0 kb data point at chromosome III-HMR, and the 1.0 kb data point from chromosome IX-TEL-R), Sir2p binding displayed no significant changes. These data indicated that the HDAC activity of Rpd3p is required for the Rpd3p-dependent restriction of the Sir2p-mediated spread of silent chromatin.
Figure 4.Enzymatic activity of Rpd3p is required for its anti-silencing function. (A) qRT–PCR results of transcription level of boundary-proximal genes, YPS6, YIR042C and GIT1, in wild type and rpd3 mutant cells. Fold transcription relative to wild type are plotted on logarithmic scales. (B) and (C) ChIP assay to detect the Sir2p enrichments at HMR and subtelomeric regions of chromosome IX-R in wild-type and rpd3 mutant strains. Average relative Sir2p enrichments are shown for each primer set with its amplified region denoted as in Figure 3. The data were normalized to an internal control (ARO1) and the input DNA. Error bars represent the standard error of the mean for three independent experiments.RPD3 disruption results in hyperacetylation of histone H4, lysine 5 and lysine 12 at boundary regionsSince HDAC activity of Rpd3p is required for antagonizing Sir2p-mediated silencing, we sought to determine the role that histone modification might play in the observed Rpd3p-associated boundary formation. Histone acetylation is maintained through competing HAT and HDAC activities, and it is expected that a loss of HDAC activity would shift this equilibrium toward increased histone acetylation. Previous studies have shown that Rpd3p preferentially deacetylates histone H4 at K5, K8 and K12 (49,50). Using antibodies specific to acetylated yeast histone proteins, we performed western blots to determine the steady-state levels of histone acetylation in wild-type and rpd3Δ strains. We found that deletion of RPD3 led to an overall increase in the acetylation of H3 and H4 (Figure 5A). More specifically, deletion of RPD3 led to hyperacetylation of histone H4 at K5 and K12 when compared to wild-type cells; however, H4 acetylation at K8 and K16 were less affected by the rpd3Δ mutation (Figure 5A). These results suggest that Rpd3p has a preference for deacetylating H4K5 and H4K12. A ChIP analysis revealed that Rpd3p bound directly to the silent mating type cassette, as well as to the subtelomeric regions (Figure 5B and C and Supplementary Figure S2A). Inactivation of Rpd3 deacetylase resulted in an increase in the DNA bound by H4K5Ac and H4K12Ac in the boundary regions but did not show an increase in DNA bound by H4K8Ac in those same regions (Figure 5D and E and Supplementary Figure S2B). Together, these findings support a model where Rpd3p or the Rpd3 complex binds to boundary regions to deacetylate histone H4K5 and/or K12, and thus regulates boundary formation directly; however, we could not exclude the possibility that the enhanced acetylation of H4K5 at the subtelomere regions in rpd3Δ cells reflects the de-repression of genes in these chromosomal loci.
Figure 5.Rpd3p interacts with chromatin at boundary regions to deacetylate histone H4K5 and K12. (A) Protein level of histone Ac-H3, Ac-H4, Ac-H4K5, K8, K12 and K16 in wild-type and rpd3Δ cells was determined by immunoblotting using anti-Ac-H3, anti-Ac-H4, anti-Ac-H4K5, anti-Ac-H4K8, anti-Ac-H4K12 and anti-H4 antibodies, respectively. (B) and (C) Binding of Rpd3p to HMR (B) and subtelomeric regions of chromosome IX (C) was detected by ChIP assay. The qPCR data is normalized to a region approximately 500 bp from the end of chromosome VI-R (TEL 0.5), whereas Rpd3 binding is excluded (45). (D) and (E) Deletion of RPD3 resulted in enhanced acetylation on H4K5 and K12 at HMR-proximal (D) or subtelomeric chromatin (E). For ChIP assay, antibodies against acetylated lysines (Ac-K5, Ac-K8 and Ac-K12) of the H4 histone tail were used. The qPCR data were normalized to an internal control (TEL 0.5) and the input DNA. Average relative enrichments of Ac-H4K5, Ac-H4K8, Ac-H4K12, 13Myc-Rpd3 and no-tag control are shown for each primer set with its amplified region denoted as in Figure 3. The results are average of three independent ChIPs with error bars representing the standard error of the mean for three independent experiments.Mutation of histone H4K5 compromises RPD3 disruption of heterochromatin repressionTo address whether acetylation of both H4K5 and K12 is required for heterochromatin spreading caused by RPD3 deletion, we employed yeast strains expressing histone mutations where the amino-terminal lysine at site 5 and/or 12 was mutated to glutamine. The telomere silencing assay (Figure 6A) showed that the growth of H4 K5Q, H4 K12Q or H4 K5,12Q mutants were indistinguishable from that of the wild-type cells, suggesting these mutations did not affect cell viability. Mutation of H4 K12Q did not compromise the enhanced silencing we previously observed in the rpd3Δ mutant; however, the H4 K5Q or H4 K5,12Q mutations attenuated the silencing of the rpd3Δ mutant (Figure 6A). The qRT–PCR analysis showed that the expression of boundary-adjacent genes in the double mutant strain, rpd3Δ H4 K5Q, was comparable to that in the single mutant H4 K5Q or wild-type strain (Figure 6B). These results suggested that mutation of H4K5 compromises the repressive effect of rpd3Δ, and supports the idea that Rpd3p counteracts silencing at least in part by deacetylating histone H4K5.
Figure 6.Deacetylation of H4K5 by Rpd3p is required for antagonizing heterochromatic silencing. (A) The growth phenotype of mutation of histone H4 lysine residues combined with rpd3Δ on subtelomeric URA3 silencing was examined. 10-fold serial dilutions of each yeast cell were spotted on YC plates with 5’-FOA as indicated. (B) qRT–PCR results of transcription level of boundary proximal genes YPS6, YIR042C, and GIT1 genes in wild-type, H4 K5Q mutant, and H4 K5Q and rpd3Δ double mutant cells. Fold transcription is relative to wild-type and plotted on logarithmic scales. (C) and (D) 13Myc–Sir2p binding was assayed by ChIP using anti-Myc antibody in wild-type, H4K5 mutant and H4K5 and rpd3 double mutant cells, at HMR-proximal loci (C) and subtelomeric regions of chromosome IX-R (D). The qPCR data were normalized to an internal control (ARO1) and the input DNA. Average relative Sir2p enrichments are shown for each primer set with its amplified region denoted as in Figure 3. The results presented are an average of three independent ChIPs with error bars shown for standard error. (E) Acetylation of histone H4K5 was measured by immunoblotting in wild-type and eas1(L327S) mutant cells. Histone H4 was used as internal control. (F) The URA3 silencing assay was performed on ESA1, ESA1 rpd3Δ and eas1(L327S) rpd3Δ double mutant cells. Tenfold serial dilutions of each yeast cells were spotted on YC plate with 5′-FOA as indicated.Next, we carried out ChIP analyses to test the effects of the H4K5 mutation on Sir2p spreading when RPD3 is disrupted. At HMR adjacent regions (e.g. 1.6, 2.6, 3.3 and 3.5 kb; see top of Figure 3F) and subtelomeric loci of chromosome IX-R (e.g. 1.5, 2.0, 2.4, 3.2, 4.0 and 6.5 kb; see top of Figure 3G), the increased binding of Sir2p in rpd3Δ cells was abrogated when H4K5 was simultaneously mutated (Figure 6C and D); whereas at the silent regions (–1.0 and 0.6 kb of HMR locus, and 1.0 kb of chromosome IX-TEL-R), the Sir2p binding was less affected by H4K5 mutation and/or RPD3 disruption (Figure 6C and D). Additionally, immunolocalization studies of Sir2p showed that in H4 K5Q rpd3Δ cells (Supplementary Figure S1F, compared with the H4 wild-type cells in Supplementary Figure S1E), most of the Sir2p signal was congregated in the nucleolus instead of diffused within the nucleus as we observed in the rpd3Δ cells with wild-type histone proteins (Figure 3B and D). These observations indicate that deacetylation of H4K5 by Rpd3p is likely required for restricting the spread of Sir2p into previously euchromatic regions and is important for antagonizing heterochromatic silencing.To further validate the dependence of Rpd3p's anti-silencing role on deacetylation of H4K5, we employed the esa1(L327S) mutant of Esa1p (51,52) in rpd3Δ cells. Esa1p is a HAT that principally acetylates histone H4K5 in vitro (51,53). The esa1(L327S) mutant in the S288c strain background was temperature sensitive (51,52), but this mutant in the BY4742 strain background did not exhibit any growth defect at either 30°C or 37°C. At 30°C, the esa1(L327S) mutation caused a specific decrease in the amount of acetylated H4K5 that could not be attributed to an overall decrease in H4 levels because these remained constant (Figure 6E). Additionally, at 30°C, the esa1 (L327S) inhibited rpd3Δ cell growth on the 5′-FOA plate (Figure 6F). The Sir2p staining pattern in the esa1(L327S) rpd3Δ cells was the same as that in wild-type cells (Supplementary Figure S1D). These results indicated that the increased telomere silencing associated with RPD3 deletion is compromised by inactivation of a HAT, namely Esa1p, that acetylates H4K5.Disruption of Rpd3p enhances deposition of H2A.Z at boundary lociPrevious studies indicated that the histone variant H2A.Z is an intrinsic component of euchromatin and functions to antagonize the formation of Sir-dependent heterochromatin (41). We wondered whether there was crosstalk between Rpd3p and H2A.Z in the anti-silencing process. As shown in Figure 7, in the rpd3Δ mutant, the amount of H2A.Z found at both sides of the HMR-proximal boundary region was dramatically increased, suggesting that H2A.Z serves as an alternative mechanism to stop the spreading of silent chromatin in the absence of Rpd3p. These results imply that Rpd3p and H2A.Z may function independently to protect euchromatin from the influence of ectopic silencing.
Figure 7.Inactivation of Rpd3p enhances the deposition of H2A.Z at HMR boundary loci. ChIP assay was performed to determine if the 3HA tagged H2A.Z was enriched at boundary loci in wild-type and rpd3Δ cells. A PCR product corresponding to the middle of the open reading frame of PRP8, a gene for which is suggested to be excluded for H2A.Z binding, is used as the internal control. The chromosomal locations of primers were as indicated in the upper panel. The immunoprecipitation data were normalized to input DNA. The results are average of three independent experiments with error bars shown for standard error.

Screen for genes antagonizing heterochromatic silencingTo screen for genes whose deletion might affect the silent chromatin at HMR and telomere loci, we concentrated our effort on 84 genes (Table S2) that have been shown to participate in modulating chromatin structure by such means as histone modification and chromatin remodeling. Quantitative RT–PCR (qRT–PCR) was used to determine relative mRNA levels of three genes located at the boundary regions. YPS6 and YIR042c are adjacent to the telomere of chromosome IX-R and GIT1 is proximal to the HMR right boundary (Figure 1A). These three genes were previously proposed to be located at boundary regions and are sensitive to the spreading of silent chromatin (40). ACT1, whose mRNA level is relatively stable, was used as an internal control. Down-regulation of YPS6, YIR042c or GIT1 transcription was observed when the genes listed in Table 1 were individually deleted. Among those, H2A.Z, Bdf1p, Sas2p, Gcn5p, Rad6p, Rpd3p, Itc1p, Rsc2p, Yta7 and Dpb4p have previously been reported to prevent silent chromatin from spreading to regulate gene transcription (18,20,21,31,40,41). The repression of marker gene expression in the rpd3Δ mutant was significant and comparable to the effects of inactivation of other known anti-silencing factors (Figure 1B). Additionally, other genes that carry novel anti-silencing function were identified, including histone acetyltransferase Hpa2p; peptidyl–prolyl cis–trans isomerase Fpr4p; Swc4p and Vps71p from the SWR1 complex; Isw1p and Isw2p from the ISW1 complex; Snf2p, Snf5p and Snf6p from the SWI/SNF complex; Npl6p and Rsc1p from the RSC chromatin remodeling complex; RNA polymerase II-associated proteins Paf1p and Cdc73p; and Taf14p and Ies3p from the INO80 complex. To our knowledge, these gene products have not previously been shown to carry anti-silencing function, and future investigation into their roles in this process might be warranted.
Table 1.Genes identified in this studyAnalysis subjectGenesChromatin assemblyHTZ1Chromatin componentBDF1Histone modificationSAS2, GCN5, RAD6, HPA2, RPD3Chromatin remodeling    Histone exchangeSWC4, VPS71    ISW1ISW1    TFIIDTAF14    ISW2/ITC1ISW2, ITC1    SWI/SNFSNF2, SNF5, SNF6    INO80IES3    RSCRSC1, RSC2, NPL6    OtherCDC73, PAF1, FPR4, DPB4, YTA7
Figure 1.Rpd3L complex is required for establishment of heterochromatic boundaries. (A) Schematic diagrams of the chromosomal locations of three boundary-proximal genes, YPS6 and YIR042C, which are adjacent to telomere, and GIT1, which is proximal to HMR. The HMR-tRNAThr gene and STAR (sub-telomeric anti-silencing region) sequence are also labeled. (B) qRT–PCR results of mRNA levels of boundary-proximal genes, YPS6, YIR042C and GIT1, in wild type and various mutant strains. (C) qRT–PCR results of mRNA levels of YPS6, YIR042C and GIT1 in the individual component deletion mutants of Rpd3L and Rpd3S. Fold transcription is relative to wild type. The log2 ratio less than zero indicates repression of transcription, whereas greater than zero indicates enhancement of transcription. Error bars represent standard error of the mean for three independent RNA purifications. (D) Mating assay to test the effects of mutation of RPD3 on boundary activity of HMR-tRNA. The ∼1.0 kb region flanking the right side of HMR with the HMR tRNAThr boundary gene was cloned into the a2 gene, and these constructs (pRO363 or pRO466) were integrated into chromosome III in a MATα strain in the presence or absence of Rpd3p (see ‘Materials and Methods’ section). The resulting strains and a MATa strain were mated, serially diluted and spotted onto a YC plate or a Ura–/Lys– plate, followed by incubation at 30°C. The photograph was taken after 48 h.

Screen for genes antagonizing heterochromatic silencing

To screen for genes whose deletion might affect the silent chromatin at HMR and telomere loci, we concentrated our effort on 84 genes (Table S2) that have been shown to participate in modulating chromatin structure by such means as histone modification and chromatin remodeling. Quantitative RT–PCR (qRT–PCR) was used to determine relative mRNA levels of three genes located at the boundary regions. YPS6 and YIR042c are adjacent to the telomere of chromosome IX-R and GIT1 is proximal to the HMR right boundary (Figure 1A). These three genes were previously proposed to be located at boundary regions and are sensitive to the spreading of silent chromatin (40). ACT1, whose mRNA level is relatively stable, was used as an internal control. Down-regulation of YPS6, YIR042c or GIT1 transcription was observed when the genes listed in Table 1 were individually deleted. Among those, H2A.Z, Bdf1p, Sas2p, Gcn5p, Rad6p, Rpd3p, Itc1p, Rsc2p, Yta7 and Dpb4p have previously been reported to prevent silent chromatin from spreading to regulate gene transcription (18,20,21,31,40,41). The repression of marker gene expression in the rpd3Δ mutant was significant and comparable to the effects of inactivation of other known anti-silencing factors (Figure 1B). Additionally, other genes that carry novel anti-silencing function were identified, including histone acetyltransferase Hpa2p; peptidyl–prolyl cis–trans isomerase Fpr4p; Swc4p and Vps71p from the SWR1 complex; Isw1p and Isw2p from the ISW1 complex; Snf2p, Snf5p and Snf6p from the SWI/SNF complex; Npl6p and Rsc1p from the RSC chromatin remodeling complex; RNA polymerase II-associated proteins Paf1p and Cdc73p; and Taf14p and Ies3p from the INO80 complex. To our knowledge, these gene products have not previously been shown to carry anti-silencing function, and future investigation into their roles in this process might be warranted.
Table 1.Genes identified in this studyAnalysis subjectGenesChromatin assemblyHTZ1Chromatin componentBDF1Histone modificationSAS2, GCN5, RAD6, HPA2, RPD3Chromatin remodeling    Histone exchangeSWC4, VPS71    ISW1ISW1    TFIIDTAF14    ISW2/ITC1ISW2, ITC1    SWI/SNFSNF2, SNF5, SNF6    INO80IES3    RSCRSC1, RSC2, NPL6    OtherCDC73, PAF1, FPR4, DPB4, YTA7
Figure 1.Rpd3L complex is required for establishment of heterochromatic boundaries. (A) Schematic diagrams of the chromosomal locations of three boundary-proximal genes, YPS6 and YIR042C, which are adjacent to telomere, and GIT1, which is proximal to HMR. The HMR-tRNAThr gene and STAR (sub-telomeric anti-silencing region) sequence are also labeled. (B) qRT–PCR results of mRNA levels of boundary-proximal genes, YPS6, YIR042C and GIT1, in wild type and various mutant strains. (C) qRT–PCR results of mRNA levels of YPS6, YIR042C and GIT1 in the individual component deletion mutants of Rpd3L and Rpd3S. Fold transcription is relative to wild type. The log2 ratio less than zero indicates repression of transcription, whereas greater than zero indicates enhancement of transcription. Error bars represent standard error of the mean for three independent RNA purifications. (D) Mating assay to test the effects of mutation of RPD3 on boundary activity of HMR-tRNA. The ∼1.0 kb region flanking the right side of HMR with the HMR tRNAThr boundary gene was cloned into the a2 gene, and these constructs (pRO363 or pRO466) were integrated into chromosome III in a MATα strain in the presence or absence of Rpd3p (see ‘Materials and Methods’ section). The resulting strains and a MATa strain were mated, serially diluted and spotted onto a YC plate or a Ura–/Lys– plate, followed by incubation at 30°C. The photograph was taken after 48 h.

Genes identified in this study

Rpd3L complex is required for establishment of heterochromatic boundaries. (A) Schematic diagrams of the chromosomal locations of three boundary-proximal genes, YPS6 and YIR042C, which are adjacent to telomere, and GIT1, which is proximal to HMR. The HMR-tRNAThr gene and STAR (sub-telomeric anti-silencing region) sequence are also labeled. (B) qRT–PCR results of mRNA levels of boundary-proximal genes, YPS6, YIR042C and GIT1, in wild type and various mutant strains. (C) qRT–PCR results of mRNA levels of YPS6, YIR042C and GIT1 in the individual component deletion mutants of Rpd3L and Rpd3S. Fold transcription is relative to wild type. The log2 ratio less than zero indicates repression of transcription, whereas greater than zero indicates enhancement of transcription. Error bars represent standard error of the mean for three independent RNA purifications. (D) Mating assay to test the effects of mutation of RPD3 on boundary activity of HMR-tRNA. The ∼1.0 kb region flanking the right side of HMR with the HMR tRNAThr boundary gene was cloned into the a2 gene, and these constructs (pRO363 or pRO466) were integrated into chromosome III in a MATα strain in the presence or absence of Rpd3p (see ‘Materials and Methods’ section). The resulting strains and a MATa strain were mated, serially diluted and spotted onto a YC plate or a Ura–/Lys– plate, followed by incubation at 30°C. The photograph was taken after 48 h.

Rpd3 complex counteracts heterochromatic silencingSir2p is a HDAC and its deletion has been shown to cause a loss of gene silencing at HMR, rDNA and telomeric loci (42). In contrast, inactivation of Rpd3p, a class I HDAC, dramatically reduced the transcription of telomere adjacent genes YPS6 and YIR042c, and the HMR proximal gene GIT1 (Figure 1A and B), indicating that RPD3 deletion mutant is defective in heterochromatin boundary activity. The opposing effects of the HDACs Rpd3p and Sir2p on silencing encouraged us to investigate further the role of Rpd3p in boundary element function.Rpd3p participates in two overlapping protein complexes named Rpd3L and Rpd3S (43). To determine if either or both of the Rpd3L and Rpd3S complexes play a general role in anti-silencing, a qRT–PCR assay examining RNA levels of the three potentially silenced genes described above (YPS6, YIR042C and GIT1) was performed in yeast strains deleted for individual components of the Rpd3L and Rpd3S complexes. As shown in Figure 1C, deletion of SIN3, which belongs to both of the complexes, mimicked the repression effect that the RPD3 deletion had on gene expression. Deletion of another overlapping core subunit, UME1, yielded a similar but more modest repression, whereas deletion of Rpd3S-specific subunit RCO1 or EAF3 had little effect on repression. Deletion of the Rpd3L-specific genes DEP1, PHO23, RXT2 or SDS3 enhanced repression significantly at all three locations. Deletion of RXT3, ASH1 and UME6 did not show a repression phenotype. These results are in agreement with previous studies by Keogh et al. (44) and lead us to conclude that the Rpd3L complex, but not the Rpd3S complex, is responsible for the anti-silencing phenotype. It has been suggested by others that Ume6p plays a recruitment role for the Rpd3L complex; however, deletion of UME6 weakened (instead of enhanced) silencing, inconsistent with a recruitment function previously proposed for Ume6p (45). Based on these data, we postulate that the recruitment of Rpd3L complex to the silent chromatin was mediated by a subunit or subunits other than Ume6p.
To validate that Rpd3p affects chromatin boundary activity, a silencing assay was performed with RPD3 deletion mutant strains harboring a URA3 reporter gene integrated at either the HMR right boundary locus or near subtelomeric regions of chromosome IX-R. The positions of the URA3 gene are illustrated in Figure 2A and B, and URA3 expression was monitored by cell growth on medium containing 5′-FOA, which is toxic to cells expressing URA3. Deletion of RPD3 promoted growth in cells with the URA3 gene inserted at ∼2 kb and ∼4 kb from the telomeric X element (∼700 bp to telomeric TG1-3 repeat sequence) of chromosome IX-R (Figure 2A, right). A similar result was obtained with the URA3 gene inserted at ∼1 kb, ∼2 kb and ∼4 kb from the right side of HMR silent cassette (Figure 2b, right). In contrast, at the very proximal subtelomeric locus ∼1 kb from chromosome IX-TEL-R, RPD3 deletion had little influence on relieving the silencing of URA3 and promoting cell growth (Figure 2A, right). These results support a model where Rpd3p is involved in heterochromatin boundary formation and has an anti-silencing function at loci adjacent to heterochromatin.
Figure 2.Enhanced silencing associated with RPD3 deletion is Sir2-dependent. (A) and (B) The wild-type, rpd3Δ, sir2Δ and rpd3Δ sir2Δ cells harboring a URA3 gene integrated at the subtelomeric regions of chromosome IX (TELIX) or HMR-adjacent loci were serially diluted and spotted onto YC medium with or without 5′-FOA, followed by incubation at 30°C. Photographs were taken after 48 or 72 h. The locations of the integrated URA3 gene on the chromosomes are illustrated on left. The numbers designated as 1–6 represent different positions of the URA3 marker. (C) qRT–PCR results of mRNA levels of YPS6, YIR042C and GIT1 genes in rpd3Δ, sir2Δ, sir3Δ, rpd3Δ sir2Δ, rpd3Δ sir3Δ and RPD3 cells. Fold increase in mRNA is relative to wild type. Error bars represent standard error of the mean for three trials.Previous studies revealed that the tRNAThr gene located at ∼1.5 kb downstream of the HMR locus (designated tRNAThr la [AGT] CR1 in the Saccharomyces Genome Database) acts as a cis-element for anti-silencing, and is required and sufficient for boundary activity (21). The expression of GIT1, which is about 4 kb downstream of HMRA1, was down-regulated upon RPD3 deletion, suggesting that inactivation of Rpd3p resulted in a decrease of the tRNA boundary activity. To further address whether mutation of RPD3 could eliminate the barrier function of HMR-tRNA to affect spreading of silent chromatin, we used an HMR-tRNA boundary activity assay where the HMR-I silencer was deleted and the 1.0 kb region downstream of HMR containing the HMR-tRNA gene was cloned into the a2 gene (Figure 1D) (12). The integrated tRNA gene was able to function as a boundary element to block the spread of silent chromatin from HMR-E into the a1 gene, thereby rendering MATα cells incompetent to mate with Mata cells to create a URA+ LYS+ diploid (Figure 1D, WT/tRNA boundary +) (12). When RPD3 was deleted, the mating efficiency of MATα cells was modestly increased (Figure 1D, rpd3Δ/tRNA boundary +), suggesting that Rpd3p facilitates the establishment of tRNA boundary activity at HMR loci.

Rpd3 complex counteracts heterochromatic silencing

Sir2p is a HDAC and its deletion has been shown to cause a loss of gene silencing at HMR, rDNA and telomeric loci (42). In contrast, inactivation of Rpd3p, a class I HDAC, dramatically reduced the transcription of telomere adjacent genes YPS6 and YIR042c, and the HMR proximal gene GIT1 (Figure 1A and B), indicating that RPD3 deletion mutant is defective in heterochromatin boundary activity. The opposing effects of the HDACs Rpd3p and Sir2p on silencing encouraged us to investigate further the role of Rpd3p in boundary element function.

Rpd3p participates in two overlapping protein complexes named Rpd3L and Rpd3S (43). To determine if either or both of the Rpd3L and Rpd3S complexes play a general role in anti-silencing, a qRT–PCR assay examining RNA levels of the three potentially silenced genes described above (YPS6, YIR042C and GIT1) was performed in yeast strains deleted for individual components of the Rpd3L and Rpd3S complexes. As shown in Figure 1C, deletion of SIN3, which belongs to both of the complexes, mimicked the repression effect that the RPD3 deletion had on gene expression. Deletion of another overlapping core subunit, UME1, yielded a similar but more modest repression, whereas deletion of Rpd3S-specific subunit RCO1 or EAF3 had little effect on repression. Deletion of the Rpd3L-specific genes DEP1, PHO23, RXT2 or SDS3 enhanced repression significantly at all three locations. Deletion of RXT3, ASH1 and UME6 did not show a repression phenotype. These results are in agreement with previous studies by Keogh et al. (44) and lead us to conclude that the Rpd3L complex, but not the Rpd3S complex, is responsible for the anti-silencing phenotype. It has been suggested by others that Ume6p plays a recruitment role for the Rpd3L complex; however, deletion of UME6 weakened (instead of enhanced) silencing, inconsistent with a recruitment function previously proposed for Ume6p (45). Based on these data, we postulate that the recruitment of Rpd3L complex to the silent chromatin was mediated by a subunit or subunits other than Ume6p.

To validate that Rpd3p affects chromatin boundary activity, a silencing assay was performed with RPD3 deletion mutant strains harboring a URA3 reporter gene integrated at either the HMR right boundary locus or near subtelomeric regions of chromosome IX-R. The positions of the URA3 gene are illustrated in Figure 2A and B, and URA3 expression was monitored by cell growth on medium containing 5′-FOA, which is toxic to cells expressing URA3. Deletion of RPD3 promoted growth in cells with the URA3 gene inserted at ∼2 kb and ∼4 kb from the telomeric X element (∼700 bp to telomeric TG1-3 repeat sequence) of chromosome IX-R (Figure 2A, right). A similar result was obtained with the URA3 gene inserted at ∼1 kb, ∼2 kb and ∼4 kb from the right side of HMR silent cassette (Figure 2b, right). In contrast, at the very proximal subtelomeric locus ∼1 kb from chromosome IX-TEL-R, RPD3 deletion had little influence on relieving the silencing of URA3 and promoting cell growth (Figure 2A, right). These results support a model where Rpd3p is involved in heterochromatin boundary formation and has an anti-silencing function at loci adjacent to heterochromatin.
Figure 2.Enhanced silencing associated with RPD3 deletion is Sir2-dependent. (A) and (B) The wild-type, rpd3Δ, sir2Δ and rpd3Δ sir2Δ cells harboring a URA3 gene integrated at the subtelomeric regions of chromosome IX (TELIX) or HMR-adjacent loci were serially diluted and spotted onto YC medium with or without 5′-FOA, followed by incubation at 30°C. Photographs were taken after 48 or 72 h. The locations of the integrated URA3 gene on the chromosomes are illustrated on left. The numbers designated as 1–6 represent different positions of the URA3 marker. (C) qRT–PCR results of mRNA levels of YPS6, YIR042C and GIT1 genes in rpd3Δ, sir2Δ, sir3Δ, rpd3Δ sir2Δ, rpd3Δ sir3Δ and RPD3 cells. Fold increase in mRNA is relative to wild type. Error bars represent standard error of the mean for three trials.

Enhanced silencing associated with RPD3 deletion is Sir2-dependent. (A) and (B) The wild-type, rpd3Δ, sir2Δ and rpd3Δ sir2Δ cells harboring a URA3 gene integrated at the subtelomeric regions of chromosome IX (TELIX) or HMR-adjacent loci were serially diluted and spotted onto YC medium with or without 5′-FOA, followed by incubation at 30°C. Photographs were taken after 48 or 72 h. The locations of the integrated URA3 gene on the chromosomes are illustrated on left. The numbers designated as 1–6 represent different positions of the URA3 marker. (C) qRT–PCR results of mRNA levels of YPS6, YIR042C and GIT1 genes in rpd3Δ, sir2Δ, sir3Δ, rpd3Δ sir2Δ, rpd3Δ sir3Δ and RPD3 cells. Fold increase in mRNA is relative to wild type. Error bars represent standard error of the mean for three trials.

Previous studies revealed that the tRNAThr gene located at ∼1.5 kb downstream of the HMR locus (designated tRNAThr la [AGT] CR1 in the Saccharomyces Genome Database) acts as a cis-element for anti-silencing, and is required and sufficient for boundary activity (21). The expression of GIT1, which is about 4 kb downstream of HMRA1, was down-regulated upon RPD3 deletion, suggesting that inactivation of Rpd3p resulted in a decrease of the tRNA boundary activity. To further address whether mutation of RPD3 could eliminate the barrier function of HMR-tRNA to affect spreading of silent chromatin, we used an HMR-tRNA boundary activity assay where the HMR-I silencer was deleted and the 1.0 kb region downstream of HMR containing the HMR-tRNA gene was cloned into the a2 gene (Figure 1D) (12). The integrated tRNA gene was able to function as a boundary element to block the spread of silent chromatin from HMR-E into the a1 gene, thereby rendering MATα cells incompetent to mate with Mata cells to create a URA+ LYS+ diploid (Figure 1D, WT/tRNA boundary +) (12). When RPD3 was deleted, the mating efficiency of MATα cells was modestly increased (Figure 1D, rpd3Δ/tRNA boundary +), suggesting that Rpd3p facilitates the establishment of tRNA boundary activity at HMR loci.

The enhanced repression associated with RPD3 deletion is Sir-dependentTo address whether the decreased expression of the URA3 reporter in the rpd3Δ mutant is Sir-mediated, the 5′-FOA growth phenotype for the rpd3Δsir2Δ mutant was examined. Though the RPD3 deletion increased silencing of URA3, the rpd3Δsir2Δ double mutant restored URA3 expression to the wild-type level (Figure 2A and B), suggesting that the repression of URA3 expression in the rpd3Δ mutant was Sir2p-dependent. A qRT–PCR assay was also performed with mutant strains where the SIR2 or SIR3 genes were individually deleted in the rpd3Δ background. Consistent with the URA3 silencing assay shown in Figure 2A and B, the transcriptional de-repression in both the rpd3Δsir2Δ and rpd3Δsir3Δ double mutants was also observed using the qRTPCR assay (Figure 2C), confirming that the repression associated with rpd3Δ is likely dependent on Sir proteins.

The enhanced repression associated with RPD3 deletion is Sir-dependent

To address whether the decreased expression of the URA3 reporter in the rpd3Δ mutant is Sir-mediated, the 5′-FOA growth phenotype for the rpd3Δsir2Δ mutant was examined. Though the RPD3 deletion increased silencing of URA3, the rpd3Δsir2Δ double mutant restored URA3 expression to the wild-type level (Figure 2A and B), suggesting that the repression of URA3 expression in the rpd3Δ mutant was Sir2p-dependent. A qRT–PCR assay was also performed with mutant strains where the SIR2 or SIR3 genes were individually deleted in the rpd3Δ background. Consistent with the URA3 silencing assay shown in Figure 2A and B, the transcriptional de-repression in both the rpd3Δsir2Δ and rpd3Δsir3Δ double mutants was also observed using the qRTPCR assay (Figure 2C), confirming that the repression associated with rpd3Δ is likely dependent on Sir proteins.

Absence of Rpd3p alters the distribution of Sir2pSir2p is the core enzyme of Sir protein complex and an essential component of silent chromatin. The results described above suggested that Rpd3p is required for defining the boundaries that block the Sir-dependent propagation of silent chromatin. To address how Rpd3p might affect Sir2p's ability to regulate silencing propagation at telomeres and HM loci, the immunolocalization of 13Myc–Sir2p was examined in fixed wild-type and rpd3Δ yeast cells using anti-Myc antibody. In wild-type cells, a strong signal was detected within a restricted nuclear sub-domain (Figure 3A), resembling the staining of crescent-shaped nucleolus, along with a weaker punctuated pattern (46). The punctuated Sir2p staining, but not the nucleolar signal, has been previously shown to co-localize with the telomere-binding protein marker Rap1p (46), as indicated by the white arrows in the merging image of Figure 3A. In contrast, the nucleolar localization of Sir2p staining was strikingly weakened in rpd3Δ cells, as indicated by the red arrows in Figure 3B. Instead, the non-nucleolar staining of Sir2p was significantly enhanced and some of these enhanced regions had Sir2p that co-localized with Rap1p. Simultaneous immunostaining of Sir2p and Nop1p showed an intact nucleolus in the rpd3Δ mutant, and co-localization of Sir2p and Nop1p was dramatically weakened by the RPD3 deletion (Figure 3C and D), suggesting the nucleolus was intact but Sir2p had moved away from the nucleolus. In rsc1 or gcn5 deletion mutant cells, the Sir2p distribution was very similar to that in wild-type cells (Supplementary Figure S1A, B and C), suggesting the change in sub-nuclear localization for Sir2p was specifically dependent upon deletion of RPD3. In summary, we found cells deficient for Rpd3p displayed a great amount of Sir2p release from the nucleolus and redistribution to other sub-nuclear loci like telomeres and their adjacent euchromatic regions. This finding suggests a model where Rpd3p influences the propagation of silent chromatin by restricting Sir2p distribution within the nucleus.
Figure 3.Inactivation of Rpd3p causes redistribution of Sir2p. (A) and (B) Confocal images of the immunolocalization of Sir2p and Rap1p in wild type (A) and rpd3Δ (B) cells. A Sir2p–13Myc fusion protein was stained by mouse anti-Myc monoclonal antibody, detected by a Cy3-conjugated secondary antibody. Rap1p was stained by rabbit anti-Rap1 polyclonal antibody, and detected by a FITC-conjugated secondary antibody. Overlap of these two signals is yellow. DNA is stained by DAPI. The bar indicates 2.0 µm. (C) and (D) Immunolocalization of Sir2p–13Myc and Nop1p in wild-type (C) and rpd3Δ (D) cells. Sir2p–13Myc was stained by rabbit anti-Myc antibody, detected by a Cy3-conjugated secondary antibody. Nop1p was stained by mouse anti-Nop1 antibody, and detected by a FITC-conjugated secondary antibody. DNA is stained by DAPI. (E), (F) and (G) 13Myc–Sir2p binding to rDNA (E), HMR (F) and subtelomeric (G) loci was assayed by ChIP using anti-Myc antibody in wild type and rpd3Δ cells. Average relative Sir2p enrichments are shown for each primer set with its amplified region as indicated on the upper panel of each graph. The upper diagrams in (E), (F) and (G) show a representative portion of ribosomal DNA locus on chromosome XII, two prominent boundary regions in the HMR silent mating type cassette and subtelomeric loci found on chromosome IX-R, respectively (41). Two well-characterized boundary elements, the HMR-adjacent tRNAThr gene and STAR sequence of Chr-IX-R are also shown (8,12,14). Locations of primer sets used for ChIP analysis are designated by the distance to the start coden of HMRA1 gene, or to the start of telomeric X element (∼700 bp to telomere TG1-3 repeats sequence) of chromosome IX. The qPCR data were normalized to an internal control (ARO1) and the input DNA. The results are average of three independent ChIPs with error bars shown for the standard error of the mean for three independent experiments.
To analyse further the redistribution of Sir2p in rpd3Δ cells, we performed a chromatin immunoprecipation (ChIP) experiment to detect Sir2p at rDNA, HMR and telomeric loci. The schematic diagrams in the upper panel of Figure 3E, F and G showed the respective regions of rDNA, HMR and chromosome IX right arm telomere (Chr IX-TEL-R) that were tested. The DNA fragments labeled in these diagrams were amplified individually in the ChIP assays. The precise location of the DNA sequences examined by ChIP are presented as the indicated genes in and near the rDNA array (Figure 3E), the distance in kb from the start codon of the HMRA1 gene (Figure 3F) or the distance in kb from the X element on the right arm of chromosome IX (Figure 3G). A gene in euchromatic region on chromosome-IV, ARO1, was used for normalization (47). The ChIP result shown in Figure 3E revealed that Sir2p bound to rDNA was not lost entirely, but was moderately decreased in rpd3Δ cells. Correspondingly, the Sir2p binding at locations between 0.6–4.1 kb around the silent mating type cassette (including the location of HMR-tRNA gene) was enhanced in rpd3Δ cells (Figure 3F). In wild-type cells, the subtelomeric regions of chromosome IX-R showed a significant drop-off in Sir2p bindings at regions of 2.0 kb away from chromosome IX-R telomere X element in wild-type cells (Figure 3G). However, in rpd3Δ mutant cells, Sir2p binding was enhanced at telomere distal regions between 1.5 and 6.5 kb away from the telomeric X element. Interestingly, RPD3 deletion had little influence on the binding of Sir2p at the 1.0 kb site near the telomere. These results are consistent with the differing Sir2p immunostaining microscopy results obtained in wild-type and rpd3Δ cells (Figure 3B), and provide further support for a model where in rpd3Δ cells, a portion of Sir2p is delocalized from nucleolus and redistributed to regions adjacent to already silent chromatin (such as telomere and HMR loci), thereby establishing a new boundary location.

Absence of Rpd3p alters the distribution of Sir2p

Sir2p is the core enzyme of Sir protein complex and an essential component of silent chromatin. The results described above suggested that Rpd3p is required for defining the boundaries that block the Sir-dependent propagation of silent chromatin. To address how Rpd3p might affect Sir2p's ability to regulate silencing propagation at telomeres and HM loci, the immunolocalization of 13Myc–Sir2p was examined in fixed wild-type and rpd3Δ yeast cells using anti-Myc antibody. In wild-type cells, a strong signal was detected within a restricted nuclear sub-domain (Figure 3A), resembling the staining of crescent-shaped nucleolus, along with a weaker punctuated pattern (46). The punctuated Sir2p staining, but not the nucleolar signal, has been previously shown to co-localize with the telomere-binding protein marker Rap1p (46), as indicated by the white arrows in the merging image of Figure 3A. In contrast, the nucleolar localization of Sir2p staining was strikingly weakened in rpd3Δ cells, as indicated by the red arrows in Figure 3B. Instead, the non-nucleolar staining of Sir2p was significantly enhanced and some of these enhanced regions had Sir2p that co-localized with Rap1p. Simultaneous immunostaining of Sir2p and Nop1p showed an intact nucleolus in the rpd3Δ mutant, and co-localization of Sir2p and Nop1p was dramatically weakened by the RPD3 deletion (Figure 3C and D), suggesting the nucleolus was intact but Sir2p had moved away from the nucleolus. In rsc1 or gcn5 deletion mutant cells, the Sir2p distribution was very similar to that in wild-type cells (Supplementary Figure S1A, B and C), suggesting the change in sub-nuclear localization for Sir2p was specifically dependent upon deletion of RPD3. In summary, we found cells deficient for Rpd3p displayed a great amount of Sir2p release from the nucleolus and redistribution to other sub-nuclear loci like telomeres and their adjacent euchromatic regions. This finding suggests a model where Rpd3p influences the propagation of silent chromatin by restricting Sir2p distribution within the nucleus.
Figure 3.Inactivation of Rpd3p causes redistribution of Sir2p. (A) and (B) Confocal images of the immunolocalization of Sir2p and Rap1p in wild type (A) and rpd3Δ (B) cells. A Sir2p–13Myc fusion protein was stained by mouse anti-Myc monoclonal antibody, detected by a Cy3-conjugated secondary antibody. Rap1p was stained by rabbit anti-Rap1 polyclonal antibody, and detected by a FITC-conjugated secondary antibody. Overlap of these two signals is yellow. DNA is stained by DAPI. The bar indicates 2.0 µm. (C) and (D) Immunolocalization of Sir2p–13Myc and Nop1p in wild-type (C) and rpd3Δ (D) cells. Sir2p–13Myc was stained by rabbit anti-Myc antibody, detected by a Cy3-conjugated secondary antibody. Nop1p was stained by mouse anti-Nop1 antibody, and detected by a FITC-conjugated secondary antibody. DNA is stained by DAPI. (E), (F) and (G) 13Myc–Sir2p binding to rDNA (E), HMR (F) and subtelomeric (G) loci was assayed by ChIP using anti-Myc antibody in wild type and rpd3Δ cells. Average relative Sir2p enrichments are shown for each primer set with its amplified region as indicated on the upper panel of each graph. The upper diagrams in (E), (F) and (G) show a representative portion of ribosomal DNA locus on chromosome XII, two prominent boundary regions in the HMR silent mating type cassette and subtelomeric loci found on chromosome IX-R, respectively (41). Two well-characterized boundary elements, the HMR-adjacent tRNAThr gene and STAR sequence of Chr-IX-R are also shown (8,12,14). Locations of primer sets used for ChIP analysis are designated by the distance to the start coden of HMRA1 gene, or to the start of telomeric X element (∼700 bp to telomere TG1-3 repeats sequence) of chromosome IX. The qPCR data were normalized to an internal control (ARO1) and the input DNA. The results are average of three independent ChIPs with error bars shown for the standard error of the mean for three independent experiments.

Inactivation of Rpd3p causes redistribution of Sir2p. (A) and (B) Confocal images of the immunolocalization of Sir2p and Rap1p in wild type (A) and rpd3Δ (B) cells. A Sir2p–13Myc fusion protein was stained by mouse anti-Myc monoclonal antibody, detected by a Cy3-conjugated secondary antibody. Rap1p was stained by rabbit anti-Rap1 polyclonal antibody, and detected by a FITC-conjugated secondary antibody. Overlap of these two signals is yellow. DNA is stained by DAPI. The bar indicates 2.0 µm. (C) and (D) Immunolocalization of Sir2p–13Myc and Nop1p in wild-type (C) and rpd3Δ (D) cells. Sir2p–13Myc was stained by rabbit anti-Myc antibody, detected by a Cy3-conjugated secondary antibody. Nop1p was stained by mouse anti-Nop1 antibody, and detected by a FITC-conjugated secondary antibody. DNA is stained by DAPI. (E), (F) and (G) 13Myc–Sir2p binding to rDNA (E), HMR (F) and subtelomeric (G) loci was assayed by ChIP using anti-Myc antibody in wild type and rpd3Δ cells. Average relative Sir2p enrichments are shown for each primer set with its amplified region as indicated on the upper panel of each graph. The upper diagrams in (E), (F) and (G) show a representative portion of ribosomal DNA locus on chromosome XII, two prominent boundary regions in the HMR silent mating type cassette and subtelomeric loci found on chromosome IX-R, respectively (41). Two well-characterized boundary elements, the HMR-adjacent tRNAThr gene and STAR sequence of Chr-IX-R are also shown (8,12,14). Locations of primer sets used for ChIP analysis are designated by the distance to the start coden of HMRA1 gene, or to the start of telomeric X element (∼700 bp to telomere TG1-3 repeats sequence) of chromosome IX. The qPCR data were normalized to an internal control (ARO1) and the input DNA. The results are average of three independent ChIPs with error bars shown for the standard error of the mean for three independent experiments.

To analyse further the redistribution of Sir2p in rpd3Δ cells, we performed a chromatin immunoprecipation (ChIP) experiment to detect Sir2p at rDNA, HMR and telomeric loci. The schematic diagrams in the upper panel of Figure 3E, F and G showed the respective regions of rDNA, HMR and chromosome IX right arm telomere (Chr IX-TEL-R) that were tested. The DNA fragments labeled in these diagrams were amplified individually in the ChIP assays. The precise location of the DNA sequences examined by ChIP are presented as the indicated genes in and near the rDNA array (Figure 3E), the distance in kb from the start codon of the HMRA1 gene (Figure 3F) or the distance in kb from the X element on the right arm of chromosome IX (Figure 3G). A gene in euchromatic region on chromosome-IV, ARO1, was used for normalization (47). The ChIP result shown in Figure 3E revealed that Sir2p bound to rDNA was not lost entirely, but was moderately decreased in rpd3Δ cells. Correspondingly, the Sir2p binding at locations between 0.6–4.1 kb around the silent mating type cassette (including the location of HMR-tRNA gene) was enhanced in rpd3Δ cells (Figure 3F). In wild-type cells, the subtelomeric regions of chromosome IX-R showed a significant drop-off in Sir2p bindings at regions of 2.0 kb away from chromosome IX-R telomere X element in wild-type cells (Figure 3G). However, in rpd3Δ mutant cells, Sir2p binding was enhanced at telomere distal regions between 1.5 and 6.5 kb away from the telomeric X element. Interestingly, RPD3 deletion had little influence on the binding of Sir2p at the 1.0 kb site near the telomere. These results are consistent with the differing Sir2p immunostaining microscopy results obtained in wild-type and rpd3Δ cells (Figure 3B), and provide further support for a model where in rpd3Δ cells, a portion of Sir2p is delocalized from nucleolus and redistributed to regions adjacent to already silent chromatin (such as telomere and HMR loci), thereby establishing a new boundary location.

Deacetylase activity is required for the anti-silencing effect of Rpd3pTo determine whether the HDAC activity of Rpd3p is required for counteracting heterochromatic silencing, we constructed the enzymatically defective forms of Rpd3p in which the conserved histidine residues at positions 151 or 188 were substituted with alanine, abolishing the catalytic activity while reserving the stability and integrity of the Rpd3p complex (48). qRT–PCR assays revealed that in the rpd3-H151A and -H188A mutants, the reporter genes YPS6, YIR042c and GIT1 remained repressed as seen in rpd3Δ cells (Figure 4A). Accordingly, a ChIP assay showed that, like in rpd3Δ mutant, the binding of Sir2p in the rpd3-H151A mutant was enhanced at the boundary of HMR locus (e.g. from 0.6 to 4.1 kb on chromosome III-HMR) or at subtelomeric regions (e.g. from 1.5 to 6.5 kb on chromosome IX-TEL-R). At the already silenced regions (noted by the 1.0 kb data point at chromosome III-HMR, and the 1.0 kb data point from chromosome IX-TEL-R), Sir2p binding displayed no significant changes. These data indicated that the HDAC activity of Rpd3p is required for the Rpd3p-dependent restriction of the Sir2p-mediated spread of silent chromatin.
Figure 4.Enzymatic activity of Rpd3p is required for its anti-silencing function. (A) qRT–PCR results of transcription level of boundary-proximal genes, YPS6, YIR042C and GIT1, in wild type and rpd3 mutant cells. Fold transcription relative to wild type are plotted on logarithmic scales. (B) and (C) ChIP assay to detect the Sir2p enrichments at HMR and subtelomeric regions of chromosome IX-R in wild-type and rpd3 mutant strains. Average relative Sir2p enrichments are shown for each primer set with its amplified region denoted as in Figure 3. The data were normalized to an internal control (ARO1) and the input DNA. Error bars represent the standard error of the mean for three independent experiments.

Deacetylase activity is required for the anti-silencing effect of Rpd3p

To determine whether the HDAC activity of Rpd3p is required for counteracting heterochromatic silencing, we constructed the enzymatically defective forms of Rpd3p in which the conserved histidine residues at positions 151 or 188 were substituted with alanine, abolishing the catalytic activity while reserving the stability and integrity of the Rpd3p complex (48). qRT–PCR assays revealed that in the rpd3-H151A and -H188A mutants, the reporter genes YPS6, YIR042c and GIT1 remained repressed as seen in rpd3Δ cells (Figure 4A). Accordingly, a ChIP assay showed that, like in rpd3Δ mutant, the binding of Sir2p in the rpd3-H151A mutant was enhanced at the boundary of HMR locus (e.g. from 0.6 to 4.1 kb on chromosome III-HMR) or at subtelomeric regions (e.g. from 1.5 to 6.5 kb on chromosome IX-TEL-R). At the already silenced regions (noted by the 1.0 kb data point at chromosome III-HMR, and the 1.0 kb data point from chromosome IX-TEL-R), Sir2p binding displayed no significant changes. These data indicated that the HDAC activity of Rpd3p is required for the Rpd3p-dependent restriction of the Sir2p-mediated spread of silent chromatin.
Figure 4.Enzymatic activity of Rpd3p is required for its anti-silencing function. (A) qRT–PCR results of transcription level of boundary-proximal genes, YPS6, YIR042C and GIT1, in wild type and rpd3 mutant cells. Fold transcription relative to wild type are plotted on logarithmic scales. (B) and (C) ChIP assay to detect the Sir2p enrichments at HMR and subtelomeric regions of chromosome IX-R in wild-type and rpd3 mutant strains. Average relative Sir2p enrichments are shown for each primer set with its amplified region denoted as in Figure 3. The data were normalized to an internal control (ARO1) and the input DNA. Error bars represent the standard error of the mean for three independent experiments.

Enzymatic activity of Rpd3p is required for its anti-silencing function. (A) qRT–PCR results of transcription level of boundary-proximal genes, YPS6, YIR042C and GIT1, in wild type and rpd3 mutant cells. Fold transcription relative to wild type are plotted on logarithmic scales. (B) and (C) ChIP assay to detect the Sir2p enrichments at HMR and subtelomeric regions of chromosome IX-R in wild-type and rpd3 mutant strains. Average relative Sir2p enrichments are shown for each primer set with its amplified region denoted as in Figure 3. The data were normalized to an internal control (ARO1) and the input DNA. Error bars represent the standard error of the mean for three independent experiments.

RPD3 disruption results in hyperacetylation of histone H4, lysine 5 and lysine 12 at boundary regionsSince HDAC activity of Rpd3p is required for antagonizing Sir2p-mediated silencing, we sought to determine the role that histone modification might play in the observed Rpd3p-associated boundary formation. Histone acetylation is maintained through competing HAT and HDAC activities, and it is expected that a loss of HDAC activity would shift this equilibrium toward increased histone acetylation. Previous studies have shown that Rpd3p preferentially deacetylates histone H4 at K5, K8 and K12 (49,50). Using antibodies specific to acetylated yeast histone proteins, we performed western blots to determine the steady-state levels of histone acetylation in wild-type and rpd3Δ strains. We found that deletion of RPD3 led to an overall increase in the acetylation of H3 and H4 (Figure 5A). More specifically, deletion of RPD3 led to hyperacetylation of histone H4 at K5 and K12 when compared to wild-type cells; however, H4 acetylation at K8 and K16 were less affected by the rpd3Δ mutation (Figure 5A). These results suggest that Rpd3p has a preference for deacetylating H4K5 and H4K12. A ChIP analysis revealed that Rpd3p bound directly to the silent mating type cassette, as well as to the subtelomeric regions (Figure 5B and C and Supplementary Figure S2A). Inactivation of Rpd3 deacetylase resulted in an increase in the DNA bound by H4K5Ac and H4K12Ac in the boundary regions but did not show an increase in DNA bound by H4K8Ac in those same regions (Figure 5D and E and Supplementary Figure S2B). Together, these findings support a model where Rpd3p or the Rpd3 complex binds to boundary regions to deacetylate histone H4K5 and/or K12, and thus regulates boundary formation directly; however, we could not exclude the possibility that the enhanced acetylation of H4K5 at the subtelomere regions in rpd3Δ cells reflects the de-repression of genes in these chromosomal loci.
Figure 5.Rpd3p interacts with chromatin at boundary regions to deacetylate histone H4K5 and K12. (A) Protein level of histone Ac-H3, Ac-H4, Ac-H4K5, K8, K12 and K16 in wild-type and rpd3Δ cells was determined by immunoblotting using anti-Ac-H3, anti-Ac-H4, anti-Ac-H4K5, anti-Ac-H4K8, anti-Ac-H4K12 and anti-H4 antibodies, respectively. (B) and (C) Binding of Rpd3p to HMR (B) and subtelomeric regions of chromosome IX (C) was detected by ChIP assay. The qPCR data is normalized to a region approximately 500 bp from the end of chromosome VI-R (TEL 0.5), whereas Rpd3 binding is excluded (45). (D) and (E) Deletion of RPD3 resulted in enhanced acetylation on H4K5 and K12 at HMR-proximal (D) or subtelomeric chromatin (E). For ChIP assay, antibodies against acetylated lysines (Ac-K5, Ac-K8 and Ac-K12) of the H4 histone tail were used. The qPCR data were normalized to an internal control (TEL 0.5) and the input DNA. Average relative enrichments of Ac-H4K5, Ac-H4K8, Ac-H4K12, 13Myc-Rpd3 and no-tag control are shown for each primer set with its amplified region denoted as in Figure 3. The results are average of three independent ChIPs with error bars representing the standard error of the mean for three independent experiments.

RPD3 disruption results in hyperacetylation of histone H4, lysine 5 and lysine 12 at boundary regions

Since HDAC activity of Rpd3p is required for antagonizing Sir2p-mediated silencing, we sought to determine the role that histone modification might play in the observed Rpd3p-associated boundary formation. Histone acetylation is maintained through competing HAT and HDAC activities, and it is expected that a loss of HDAC activity would shift this equilibrium toward increased histone acetylation. Previous studies have shown that Rpd3p preferentially deacetylates histone H4 at K5, K8 and K12 (49,50). Using antibodies specific to acetylated yeast histone proteins, we performed western blots to determine the steady-state levels of histone acetylation in wild-type and rpd3Δ strains. We found that deletion of RPD3 led to an overall increase in the acetylation of H3 and H4 (Figure 5A). More specifically, deletion of RPD3 led to hyperacetylation of histone H4 at K5 and K12 when compared to wild-type cells; however, H4 acetylation at K8 and K16 were less affected by the rpd3Δ mutation (Figure 5A). These results suggest that Rpd3p has a preference for deacetylating H4K5 and H4K12. A ChIP analysis revealed that Rpd3p bound directly to the silent mating type cassette, as well as to the subtelomeric regions (Figure 5B and C and Supplementary Figure S2A). Inactivation of Rpd3 deacetylase resulted in an increase in the DNA bound by H4K5Ac and H4K12Ac in the boundary regions but did not show an increase in DNA bound by H4K8Ac in those same regions (Figure 5D and E and Supplementary Figure S2B). Together, these findings support a model where Rpd3p or the Rpd3 complex binds to boundary regions to deacetylate histone H4K5 and/or K12, and thus regulates boundary formation directly; however, we could not exclude the possibility that the enhanced acetylation of H4K5 at the subtelomere regions in rpd3Δ cells reflects the de-repression of genes in these chromosomal loci.
Figure 5.Rpd3p interacts with chromatin at boundary regions to deacetylate histone H4K5 and K12. (A) Protein level of histone Ac-H3, Ac-H4, Ac-H4K5, K8, K12 and K16 in wild-type and rpd3Δ cells was determined by immunoblotting using anti-Ac-H3, anti-Ac-H4, anti-Ac-H4K5, anti-Ac-H4K8, anti-Ac-H4K12 and anti-H4 antibodies, respectively. (B) and (C) Binding of Rpd3p to HMR (B) and subtelomeric regions of chromosome IX (C) was detected by ChIP assay. The qPCR data is normalized to a region approximately 500 bp from the end of chromosome VI-R (TEL 0.5), whereas Rpd3 binding is excluded (45). (D) and (E) Deletion of RPD3 resulted in enhanced acetylation on H4K5 and K12 at HMR-proximal (D) or subtelomeric chromatin (E). For ChIP assay, antibodies against acetylated lysines (Ac-K5, Ac-K8 and Ac-K12) of the H4 histone tail were used. The qPCR data were normalized to an internal control (TEL 0.5) and the input DNA. Average relative enrichments of Ac-H4K5, Ac-H4K8, Ac-H4K12, 13Myc-Rpd3 and no-tag control are shown for each primer set with its amplified region denoted as in Figure 3. The results are average of three independent ChIPs with error bars representing the standard error of the mean for three independent experiments.

Rpd3p interacts with chromatin at boundary regions to deacetylate histone H4K5 and K12. (A) Protein level of histone Ac-H3, Ac-H4, Ac-H4K5, K8, K12 and K16 in wild-type and rpd3Δ cells was determined by immunoblotting using anti-Ac-H3, anti-Ac-H4, anti-Ac-H4K5, anti-Ac-H4K8, anti-Ac-H4K12 and anti-H4 antibodies, respectively. (B) and (C) Binding of Rpd3p to HMR (B) and subtelomeric regions of chromosome IX (C) was detected by ChIP assay. The qPCR data is normalized to a region approximately 500 bp from the end of chromosome VI-R (TEL 0.5), whereas Rpd3 binding is excluded (45). (D) and (E) Deletion of RPD3 resulted in enhanced acetylation on H4K5 and K12 at HMR-proximal (D) or subtelomeric chromatin (E). For ChIP assay, antibodies against acetylated lysines (Ac-K5, Ac-K8 and Ac-K12) of the H4 histone tail were used. The qPCR data were normalized to an internal control (TEL 0.5) and the input DNA. Average relative enrichments of Ac-H4K5, Ac-H4K8, Ac-H4K12, 13Myc-Rpd3 and no-tag control are shown for each primer set with its amplified region denoted as in Figure 3. The results are average of three independent ChIPs with error bars representing the standard error of the mean for three independent experiments.

Mutation of histone H4K5 compromises RPD3 disruption of heterochromatin repressionTo address whether acetylation of both H4K5 and K12 is required for heterochromatin spreading caused by RPD3 deletion, we employed yeast strains expressing histone mutations where the amino-terminal lysine at site 5 and/or 12 was mutated to glutamine. The telomere silencing assay (Figure 6A) showed that the growth of H4 K5Q, H4 K12Q or H4 K5,12Q mutants were indistinguishable from that of the wild-type cells, suggesting these mutations did not affect cell viability. Mutation of H4 K12Q did not compromise the enhanced silencing we previously observed in the rpd3Δ mutant; however, the H4 K5Q or H4 K5,12Q mutations attenuated the silencing of the rpd3Δ mutant (Figure 6A). The qRT–PCR analysis showed that the expression of boundary-adjacent genes in the double mutant strain, rpd3Δ H4 K5Q, was comparable to that in the single mutant H4 K5Q or wild-type strain (Figure 6B). These results suggested that mutation of H4K5 compromises the repressive effect of rpd3Δ, and supports the idea that Rpd3p counteracts silencing at least in part by deacetylating histone H4K5.
Figure 6.Deacetylation of H4K5 by Rpd3p is required for antagonizing heterochromatic silencing. (A) The growth phenotype of mutation of histone H4 lysine residues combined with rpd3Δ on subtelomeric URA3 silencing was examined. 10-fold serial dilutions of each yeast cell were spotted on YC plates with 5’-FOA as indicated. (B) qRT–PCR results of transcription level of boundary proximal genes YPS6, YIR042C, and GIT1 genes in wild-type, H4 K5Q mutant, and H4 K5Q and rpd3Δ double mutant cells. Fold transcription is relative to wild-type and plotted on logarithmic scales. (C) and (D) 13Myc–Sir2p binding was assayed by ChIP using anti-Myc antibody in wild-type, H4K5 mutant and H4K5 and rpd3 double mutant cells, at HMR-proximal loci (C) and subtelomeric regions of chromosome IX-R (D). The qPCR data were normalized to an internal control (ARO1) and the input DNA. Average relative Sir2p enrichments are shown for each primer set with its amplified region denoted as in Figure 3. The results presented are an average of three independent ChIPs with error bars shown for standard error. (E) Acetylation of histone H4K5 was measured by immunoblotting in wild-type and eas1(L327S) mutant cells. Histone H4 was used as internal control. (F) The URA3 silencing assay was performed on ESA1, ESA1 rpd3Δ and eas1(L327S) rpd3Δ double mutant cells. Tenfold serial dilutions of each yeast cells were spotted on YC plate with 5′-FOA as indicated.Next, we carried out ChIP analyses to test the effects of the H4K5 mutation on Sir2p spreading when RPD3 is disrupted. At HMR adjacent regions (e.g. 1.6, 2.6, 3.3 and 3.5 kb; see top of Figure 3F) and subtelomeric loci of chromosome IX-R (e.g. 1.5, 2.0, 2.4, 3.2, 4.0 and 6.5 kb; see top of Figure 3G), the increased binding of Sir2p in rpd3Δ cells was abrogated when H4K5 was simultaneously mutated (Figure 6C and D); whereas at the silent regions (–1.0 and 0.6 kb of HMR locus, and 1.0 kb of chromosome IX-TEL-R), the Sir2p binding was less affected by H4K5 mutation and/or RPD3 disruption (Figure 6C and D). Additionally, immunolocalization studies of Sir2p showed that in H4 K5Q rpd3Δ cells (Supplementary Figure S1F, compared with the H4 wild-type cells in Supplementary Figure S1E), most of the Sir2p signal was congregated in the nucleolus instead of diffused within the nucleus as we observed in the rpd3Δ cells with wild-type histone proteins (Figure 3B and D). These observations indicate that deacetylation of H4K5 by Rpd3p is likely required for restricting the spread of Sir2p into previously euchromatic regions and is important for antagonizing heterochromatic silencing.To further validate the dependence of Rpd3p's anti-silencing role on deacetylation of H4K5, we employed the esa1(L327S) mutant of Esa1p (51,52) in rpd3Δ cells. Esa1p is a HAT that principally acetylates histone H4K5 in vitro (51,53). The esa1(L327S) mutant in the S288c strain background was temperature sensitive (51,52), but this mutant in the BY4742 strain background did not exhibit any growth defect at either 30°C or 37°C. At 30°C, the esa1(L327S) mutation caused a specific decrease in the amount of acetylated H4K5 that could not be attributed to an overall decrease in H4 levels because these remained constant (Figure 6E). Additionally, at 30°C, the esa1 (L327S) inhibited rpd3Δ cell growth on the 5′-FOA plate (Figure 6F). The Sir2p staining pattern in the esa1(L327S) rpd3Δ cells was the same as that in wild-type cells (Supplementary Figure S1D). These results indicated that the increased telomere silencing associated with RPD3 deletion is compromised by inactivation of a HAT, namely Esa1p, that acetylates H4K5.

Mutation of histone H4K5 compromises RPD3 disruption of heterochromatin repression

To address whether acetylation of both H4K5 and K12 is required for heterochromatin spreading caused by RPD3 deletion, we employed yeast strains expressing histone mutations where the amino-terminal lysine at site 5 and/or 12 was mutated to glutamine. The telomere silencing assay (Figure 6A) showed that the growth of H4 K5Q, H4 K12Q or H4 K5,12Q mutants were indistinguishable from that of the wild-type cells, suggesting these mutations did not affect cell viability. Mutation of H4 K12Q did not compromise the enhanced silencing we previously observed in the rpd3Δ mutant; however, the H4 K5Q or H4 K5,12Q mutations attenuated the silencing of the rpd3Δ mutant (Figure 6A). The qRT–PCR analysis showed that the expression of boundary-adjacent genes in the double mutant strain, rpd3Δ H4 K5Q, was comparable to that in the single mutant H4 K5Q or wild-type strain (Figure 6B). These results suggested that mutation of H4K5 compromises the repressive effect of rpd3Δ, and supports the idea that Rpd3p counteracts silencing at least in part by deacetylating histone H4K5.
Figure 6.Deacetylation of H4K5 by Rpd3p is required for antagonizing heterochromatic silencing. (A) The growth phenotype of mutation of histone H4 lysine residues combined with rpd3Δ on subtelomeric URA3 silencing was examined. 10-fold serial dilutions of each yeast cell were spotted on YC plates with 5’-FOA as indicated. (B) qRT–PCR results of transcription level of boundary proximal genes YPS6, YIR042C, and GIT1 genes in wild-type, H4 K5Q mutant, and H4 K5Q and rpd3Δ double mutant cells. Fold transcription is relative to wild-type and plotted on logarithmic scales. (C) and (D) 13Myc–Sir2p binding was assayed by ChIP using anti-Myc antibody in wild-type, H4K5 mutant and H4K5 and rpd3 double mutant cells, at HMR-proximal loci (C) and subtelomeric regions of chromosome IX-R (D). The qPCR data were normalized to an internal control (ARO1) and the input DNA. Average relative Sir2p enrichments are shown for each primer set with its amplified region denoted as in Figure 3. The results presented are an average of three independent ChIPs with error bars shown for standard error. (E) Acetylation of histone H4K5 was measured by immunoblotting in wild-type and eas1(L327S) mutant cells. Histone H4 was used as internal control. (F) The URA3 silencing assay was performed on ESA1, ESA1 rpd3Δ and eas1(L327S) rpd3Δ double mutant cells. Tenfold serial dilutions of each yeast cells were spotted on YC plate with 5′-FOA as indicated.

Deacetylation of H4K5 by Rpd3p is required for antagonizing heterochromatic silencing. (A) The growth phenotype of mutation of histone H4 lysine residues combined with rpd3Δ on subtelomeric URA3 silencing was examined. 10-fold serial dilutions of each yeast cell were spotted on YC plates with 5’-FOA as indicated. (B) qRT–PCR results of transcription level of boundary proximal genes YPS6, YIR042C, and GIT1 genes in wild-type, H4 K5Q mutant, and H4 K5Q and rpd3Δ double mutant cells. Fold transcription is relative to wild-type and plotted on logarithmic scales. (C) and (D) 13Myc–Sir2p binding was assayed by ChIP using anti-Myc antibody in wild-type, H4K5 mutant and H4K5 and rpd3 double mutant cells, at HMR-proximal loci (C) and subtelomeric regions of chromosome IX-R (D). The qPCR data were normalized to an internal control (ARO1) and the input DNA. Average relative Sir2p enrichments are shown for each primer set with its amplified region denoted as in Figure 3. The results presented are an average of three independent ChIPs with error bars shown for standard error. (E) Acetylation of histone H4K5 was measured by immunoblotting in wild-type and eas1(L327S) mutant cells. Histone H4 was used as internal control. (F) The URA3 silencing assay was performed on ESA1, ESA1 rpd3Δ and eas1(L327S) rpd3Δ double mutant cells. Tenfold serial dilutions of each yeast cells were spotted on YC plate with 5′-FOA as indicated.

Next, we carried out ChIP analyses to test the effects of the H4K5 mutation on Sir2p spreading when RPD3 is disrupted. At HMR adjacent regions (e.g. 1.6, 2.6, 3.3 and 3.5 kb; see top of Figure 3F) and subtelomeric loci of chromosome IX-R (e.g. 1.5, 2.0, 2.4, 3.2, 4.0 and 6.5 kb; see top of Figure 3G), the increased binding of Sir2p in rpd3Δ cells was abrogated when H4K5 was simultaneously mutated (Figure 6C and D); whereas at the silent regions (–1.0 and 0.6 kb of HMR locus, and 1.0 kb of chromosome IX-TEL-R), the Sir2p binding was less affected by H4K5 mutation and/or RPD3 disruption (Figure 6C and D). Additionally, immunolocalization studies of Sir2p showed that in H4 K5Q rpd3Δ cells (Supplementary Figure S1F, compared with the H4 wild-type cells in Supplementary Figure S1E), most of the Sir2p signal was congregated in the nucleolus instead of diffused within the nucleus as we observed in the rpd3Δ cells with wild-type histone proteins (Figure 3B and D). These observations indicate that deacetylation of H4K5 by Rpd3p is likely required for restricting the spread of Sir2p into previously euchromatic regions and is important for antagonizing heterochromatic silencing.

To further validate the dependence of Rpd3p's anti-silencing role on deacetylation of H4K5, we employed the esa1(L327S) mutant of Esa1p (51,52) in rpd3Δ cells. Esa1p is a HAT that principally acetylates histone H4K5 in vitro (51,53). The esa1(L327S) mutant in the S288c strain background was temperature sensitive (51,52), but this mutant in the BY4742 strain background did not exhibit any growth defect at either 30°C or 37°C. At 30°C, the esa1(L327S) mutation caused a specific decrease in the amount of acetylated H4K5 that could not be attributed to an overall decrease in H4 levels because these remained constant (Figure 6E). Additionally, at 30°C, the esa1 (L327S) inhibited rpd3Δ cell growth on the 5′-FOA plate (Figure 6F). The Sir2p staining pattern in the esa1(L327S) rpd3Δ cells was the same as that in wild-type cells (Supplementary Figure S1D). These results indicated that the increased telomere silencing associated with RPD3 deletion is compromised by inactivation of a HAT, namely Esa1p, that acetylates H4K5.

Disruption of Rpd3p enhances deposition of H2A.Z at boundary lociPrevious studies indicated that the histone variant H2A.Z is an intrinsic component of euchromatin and functions to antagonize the formation of Sir-dependent heterochromatin (41). We wondered whether there was crosstalk between Rpd3p and H2A.Z in the anti-silencing process. As shown in Figure 7, in the rpd3Δ mutant, the amount of H2A.Z found at both sides of the HMR-proximal boundary region was dramatically increased, suggesting that H2A.Z serves as an alternative mechanism to stop the spreading of silent chromatin in the absence of Rpd3p. These results imply that Rpd3p and H2A.Z may function independently to protect euchromatin from the influence of ectopic silencing.
Figure 7.Inactivation of Rpd3p enhances the deposition of H2A.Z at HMR boundary loci. ChIP assay was performed to determine if the 3HA tagged H2A.Z was enriched at boundary loci in wild-type and rpd3Δ cells. A PCR product corresponding to the middle of the open reading frame of PRP8, a gene for which is suggested to be excluded for H2A.Z binding, is used as the internal control. The chromosomal locations of primers were as indicated in the upper panel. The immunoprecipitation data were normalized to input DNA. The results are average of three independent experiments with error bars shown for standard error.

Disruption of Rpd3p enhances deposition of H2A.Z at boundary loci

Previous studies indicated that the histone variant H2A.Z is an intrinsic component of euchromatin and functions to antagonize the formation of Sir-dependent heterochromatin (41). We wondered whether there was crosstalk between Rpd3p and H2A.Z in the anti-silencing process. As shown in Figure 7, in the rpd3Δ mutant, the amount of H2A.Z found at both sides of the HMR-proximal boundary region was dramatically increased, suggesting that H2A.Z serves as an alternative mechanism to stop the spreading of silent chromatin in the absence of Rpd3p. These results imply that Rpd3p and H2A.Z may function independently to protect euchromatin from the influence of ectopic silencing.
Figure 7.Inactivation of Rpd3p enhances the deposition of H2A.Z at HMR boundary loci. ChIP assay was performed to determine if the 3HA tagged H2A.Z was enriched at boundary loci in wild-type and rpd3Δ cells. A PCR product corresponding to the middle of the open reading frame of PRP8, a gene for which is suggested to be excluded for H2A.Z binding, is used as the internal control. The chromosomal locations of primers were as indicated in the upper panel. The immunoprecipitation data were normalized to input DNA. The results are average of three independent experiments with error bars shown for standard error.

Inactivation of Rpd3p enhances the deposition of H2A.Z at HMR boundary loci. ChIP assay was performed to determine if the 3HA tagged H2A.Z was enriched at boundary loci in wild-type and rpd3Δ cells. A PCR product corresponding to the middle of the open reading frame of PRP8, a gene for which is suggested to be excluded for H2A.Z binding, is used as the internal control. The chromosomal locations of primers were as indicated in the upper panel. The immunoprecipitation data were normalized to input DNA. The results are average of three independent experiments with error bars shown for standard error.

DISCUSSIONThe spreading of Sir proteins along chromosomes is associated with the formation of silent chromatin (38). Sir2p is a central component of the repressive Sir complex, and its deacetylase activity is required for Sir spreading (54). Sir2p is mainly localized to two distinct sub-nuclear domains, the telomere and the nucleolus (46). The nucleolus has been proposed to serve as a reservoir for Sir2p storage, competing with subtelomeric regions and HM loci for a limiting supply of Sir2p (46,55,56). In the current study, we found that in rpd3Δ cells, a portion of Sir2p was delocalized from nucleolus and was consequentially enriched at the ectopic silencing regions found at telomeres and at HM and their adjacent loci (Figure 3). This observation is in agreement with a Sir2p–Rpd3 competition hypothesis and is consistent with previous reports by the Boeke and Hampsey laboratories (31,57). Paradoxically, the decrease of rDNA-associated Sir2p causes a deficiency of rDNA silencing (58) but in rpd3Δ cells the reduction of rDNA-associated Sir2p (Figure 3) does not weaken, but rather enhances rDNA silencing (31,57). Since not all the Sir2p is lost from the nucleolus and rDNA, it is possible that the remaining Sir2p in rpd3Δ cells at rDNA loci is sufficient and responsible for the maintenance of rDNA silencing. Alternatively, HDAC Rpd3p may act directly on histones in the rDNA to regulate rDNA silencing, and the consequential improvement of rDNA silencing in rpd3 deletion cells could be attributed to an increase of histone acetylation as well as some Sir2p remaining in the rDNA loci.The K5Q mutation in histone H4 compromised the enhanced telomere position effect in rpd3Δ cells (Figure 6A and B). Interestingly, the H4K12Q mutation could not attenuate rpd3Δ-associated repression, while double mutation of H4 K5,12Q decreased the rpd3Δ-associated repression (Figure 6A), suggesting that H4K12 is not as important as H4K5 for mediating the anti-silencing affects of Rpd3p. Considering that H4K5 is one of the principle targets of Rpd3p, it is conceivable that boundary formation in the subtelomeric and HMR regions requires H4K5 deacetylation; however, it is difficult to rationalize how an increase in H4 acetylation would be important for the establishment of heterochromatin. Previously completed in vivo formaldehyde cross-linking experiments have demonstrated that Sir2p can bind indirectly to chromatin far from telomeres (59), forming a weaker and more transient protein-protein interaction within euchromatic regions. Rpd3p was also associated with the subtelomeric regions (Figure 5C). Based on these data, we speculate that both Rpd3p and Sir2p compete for chromatin binding at subtelomeric boundary regions. When Rpd3p was present, the acetylation level of histone H4K5 was negatively regulated by Rpd3p (Figure 5A, D and E), and the Sir2-dependent propagation of silent chromatin was prohibited (Figure 6A–D). When Rpd3p was absent, an increase of H4K5 acetylation (Figure 5A, D and E) would have facilitated Sir2p binding (Figure 6C and D). In support of this model, we found that mutation of H4K5 decreased Sir2p binding in rpd3Δ cells (Figure 6C and D) and mutation of ESA1 [i.e. esa1(L327S)] in rpd3Δ cells restored telomere position effect (TPE) (Figure 6F). Consistently, deletion of Rpd3 caused redistribution of Sir2 from the nucleolus to the telomeres and HM loci (Figure 3). However, the mechanism as to how the increase of H4 acetylation caused by Rpd3 inactivation facilitates the establishment of silent chromatin remains mysterious, and requires further investigation.Previous studies revealed that Rpd3p possesses the activity of deacetylating histone H4K12 (49,60,61). Acetylation of histone H4K12 is required for Sir3p binding during the spreading of heterochromatin (62–64). It has been proposed that deletion of RPD3 may increase acetylation of H4K12 to facilitate Sir-mediated repression (32). Controversially, de Bruin et al. (65) reported that the lysine residues in the histone H4-terminal tail are all hypoacetylated at yeast telomeres, and H4K12 is not preferentially acetylated in the silent chromatin at both telomere and silent mating loci (49). An in vitro surface plasmon resonance (SPR) study also showed that acetylation of synthetic H4 peptides (residues 1–34) at K5, K8, K12 and K16 decreases Sir3p binding (22). Our ChIP analyses demonstrated that the three lysine residues exhibit comparatively lower acetylation levels at the silent mating type locus and at the proximal regions of telomeres when compared to an internal locus (Figure 5D and E), arguing that H4K12 at silencing loci is not preferentially acetylated, and hypoacetylation of histone H4 tail might be generally important for the formation and establishment of euchromatin.The fact that silent chromatin can encroach upon active chromatin poses a fundamentally important question as to how the repressive chromatin does not ultimately invade and occupy entirely all the active regions of the genome. Like deletion of any other anti-silencing factors (e.g. HTZ1, SAS2), or boundary elements [e.g. the tRNA gene located at the right side of HMR locus (66,67)], deletion of Rpd3p does not cause cell death, suggesting that the formation and spreading of silent chromatin is eventually stopped, probably by the reestablishment of the boundary. The reason for this is unclear. In telomere silencing, one possibility is that the amount of Sir proteins is limited and lack of enough Sir proteins would passively impair the further spreading of silent chromatin. A second possibility is that Rpd3p only acts on the discrete regions in the genome (Figure 5B and C), and inactivation of Rpd3p causes histone hyperacetylation at certain regions that are subsequently bound by Sir2p (Figure 3F and G). A third possibility is that a redundant anti-silencing pathway parallelly functions to prevent the spreading of silent chromatin regardless of Rpd3p. This final idea is supported by the observation that disruption of Rpd3p could promote the deposition of H2A.Z at boundary regions to protect euchromatin from ectopic silencing (Figure 7). Each of these three models for the prevention of the formation of global euchromatin is possible and we look forward to future studies into this topic.

The spreading of Sir proteins along chromosomes is associated with the formation of silent chromatin (38). Sir2p is a central component of the repressive Sir complex, and its deacetylase activity is required for Sir spreading (54). Sir2p is mainly localized to two distinct sub-nuclear domains, the telomere and the nucleolus (46). The nucleolus has been proposed to serve as a reservoir for Sir2p storage, competing with subtelomeric regions and HM loci for a limiting supply of Sir2p (46,55,56). In the current study, we found that in rpd3Δ cells, a portion of Sir2p was delocalized from nucleolus and was consequentially enriched at the ectopic silencing regions found at telomeres and at HM and their adjacent loci (Figure 3). This observation is in agreement with a Sir2p–Rpd3 competition hypothesis and is consistent with previous reports by the Boeke and Hampsey laboratories (31,57). Paradoxically, the decrease of rDNA-associated Sir2p causes a deficiency of rDNA silencing (58) but in rpd3Δ cells the reduction of rDNA-associated Sir2p (Figure 3) does not weaken, but rather enhances rDNA silencing (31,57). Since not all the Sir2p is lost from the nucleolus and rDNA, it is possible that the remaining Sir2p in rpd3Δ cells at rDNA loci is sufficient and responsible for the maintenance of rDNA silencing. Alternatively, HDAC Rpd3p may act directly on histones in the rDNA to regulate rDNA silencing, and the consequential improvement of rDNA silencing in rpd3 deletion cells could be attributed to an increase of histone acetylation as well as some Sir2p remaining in the rDNA loci.

The K5Q mutation in histone H4 compromised the enhanced telomere position effect in rpd3Δ cells (Figure 6A and B). Interestingly, the H4K12Q mutation could not attenuate rpd3Δ-associated repression, while double mutation of H4 K5,12Q decreased the rpd3Δ-associated repression (Figure 6A), suggesting that H4K12 is not as important as H4K5 for mediating the anti-silencing affects of Rpd3p. Considering that H4K5 is one of the principle targets of Rpd3p, it is conceivable that boundary formation in the subtelomeric and HMR regions requires H4K5 deacetylation; however, it is difficult to rationalize how an increase in H4 acetylation would be important for the establishment of heterochromatin. Previously completed in vivo formaldehyde cross-linking experiments have demonstrated that Sir2p can bind indirectly to chromatin far from telomeres (59), forming a weaker and more transient protein-protein interaction within euchromatic regions. Rpd3p was also associated with the subtelomeric regions (Figure 5C). Based on these data, we speculate that both Rpd3p and Sir2p compete for chromatin binding at subtelomeric boundary regions. When Rpd3p was present, the acetylation level of histone H4K5 was negatively regulated by Rpd3p (Figure 5A, D and E), and the Sir2-dependent propagation of silent chromatin was prohibited (Figure 6A–D). When Rpd3p was absent, an increase of H4K5 acetylation (Figure 5A, D and E) would have facilitated Sir2p binding (Figure 6C and D). In support of this model, we found that mutation of H4K5 decreased Sir2p binding in rpd3Δ cells (Figure 6C and D) and mutation of ESA1 [i.e. esa1(L327S)] in rpd3Δ cells restored telomere position effect (TPE) (Figure 6F). Consistently, deletion of Rpd3 caused redistribution of Sir2 from the nucleolus to the telomeres and HM loci (Figure 3). However, the mechanism as to how the increase of H4 acetylation caused by Rpd3 inactivation facilitates the establishment of silent chromatin remains mysterious, and requires further investigation.

Previous studies revealed that Rpd3p possesses the activity of deacetylating histone H4K12 (49,60,61). Acetylation of histone H4K12 is required for Sir3p binding during the spreading of heterochromatin (62–64). It has been proposed that deletion of RPD3 may increase acetylation of H4K12 to facilitate Sir-mediated repression (32). Controversially, de Bruin et al. (65) reported that the lysine residues in the histone H4-terminal tail are all hypoacetylated at yeast telomeres, and H4K12 is not preferentially acetylated in the silent chromatin at both telomere and silent mating loci (49). An in vitro surface plasmon resonance (SPR) study also showed that acetylation of synthetic H4 peptides (residues 1–34) at K5, K8, K12 and K16 decreases Sir3p binding (22). Our ChIP analyses demonstrated that the three lysine residues exhibit comparatively lower acetylation levels at the silent mating type locus and at the proximal regions of telomeres when compared to an internal locus (Figure 5D and E), arguing that H4K12 at silencing loci is not preferentially acetylated, and hypoacetylation of histone H4 tail might be generally important for the formation and establishment of euchromatin.

The fact that silent chromatin can encroach upon active chromatin poses a fundamentally important question as to how the repressive chromatin does not ultimately invade and occupy entirely all the active regions of the genome. Like deletion of any other anti-silencing factors (e.g. HTZ1, SAS2), or boundary elements [e.g. the tRNA gene located at the right side of HMR locus (66,67)], deletion of Rpd3p does not cause cell death, suggesting that the formation and spreading of silent chromatin is eventually stopped, probably by the reestablishment of the boundary. The reason for this is unclear. In telomere silencing, one possibility is that the amount of Sir proteins is limited and lack of enough Sir proteins would passively impair the further spreading of silent chromatin. A second possibility is that Rpd3p only acts on the discrete regions in the genome (Figure 5B and C), and inactivation of Rpd3p causes histone hyperacetylation at certain regions that are subsequently bound by Sir2p (Figure 3F and G). A third possibility is that a redundant anti-silencing pathway parallelly functions to prevent the spreading of silent chromatin regardless of Rpd3p. This final idea is supported by the observation that disruption of Rpd3p could promote the deposition of H2A.Z at boundary regions to protect euchromatin from ectopic silencing (Figure 7). Each of these three models for the prevention of the formation of global euchromatin is possible and we look forward to future studies into this topic.

SUPPLEMENTARY DATASupplementary Data are available at NAR Online.

SUPPLEMENTARY DATA

Supplementary Data are available at NAR Online.

Conflict of interest statement. None declared.

Supplementary Material[Supplementary Data]

Supplementary Material

[Supplementary Data]
