---
pmid: '20159469'
title: Crystal structure of the p53 core domain bound to a full consensus site as
  a self-assembled tetramer.
authors:
- Chen Y
- Dey R
- Chen L
journal: Structure
year: '2010'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC2824536
doi: 10.1016/j.str.2009.11.011
---

# Crystal structure of the p53 core domain bound to a full consensus site as a self-assembled tetramer.
**Authors:** Chen Y, Dey R, Chen L
**Journal:** Structure (2010)
**DOI:** [10.1016/j.str.2009.11.011](https://doi.org/10.1016/j.str.2009.11.011)
**PMC:** [PMC2824536](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2824536/)

## Abstract

1. Structure. 2010 Feb 10;18(2):246-56. doi: 10.1016/j.str.2009.11.011.

Crystal structure of the p53 core domain bound to a full consensus site as a 
self-assembled tetramer.

Chen Y(1), Dey R, Chen L.

Author information:
(1)Molecular and Computational Biology, Department of Biological Sciences, 
University of Southern California, Los Angeles, CA 90089, USA.

Recent studies suggest that p53 binds predominantly to consensus sites composed 
of two decameric half-sites with zero spacing in vivo. Here we report the 
crystal structure of the p53 core domain bound to a full consensus site as a 
tetramer at 2.13A resolution. Comparison with previously reported structures of 
p53 dimer:DNA complexes and a chemically trapped p53 tetramer:DNA complex 
reveals that DNA binding by the p53 core domain is a cooperative self-assembling 
process accompanied by structural changes of the p53 dimer and DNA. Each p53 
monomer interacts with its two neighboring subunits through two different 
protein-protein interfaces. The DNA is largely B-form and shows no discernible 
bend, but the central base-pairs between the two half-sites display a 
significant slide. The extensive protein-protein and protein-DNA interactions 
explain the high cooperativity and kinetic stability of p53 binding to 
contiguous decameric sites and the conservation of such binding-site 
configuration in vivo.

Copyright 2010 Elsevier Ltd. All rights reserved.

DOI: 10.1016/j.str.2009.11.011
PMCID: PMC2824536
PMID: 20159469 [Indexed for MEDLINE]

## Full Text

Introduction

P53 is essential for the prevention of cancer development; its function is inactivated by mutations and signal-dependent modifications in a majority of human cancers ( Horn and Vousden, 2007 ; Vogelstein et al., 2000 ). While p53 may suppress tumor formation and progression through multiple mechanisms ( Green and Kroemer, 2009 ; Suzuki et al., 2009 ), its primary function is to regulate transcription programs of cell growth control ( Vousden and Prives, 2009 ). In responses to oncogenic stresses, p53 binds specific DNA targets and regulates the expression of genes involved in cell cycle arrest, DNA repair or apoptosis. Like many eukaryotic transcription factors, p53 has a modular structure with multiple domains, including a core DNA-binding domain (residues ∼94-292) and a tetramerization domain (residues 325-356). Both domains are structurally folded and are connected by a flexible linker (residues 293-324). Conversely, the N- and C-terminal regions (residues 1-93 and 357-393, respectively) of p53 are largely unstructured. These regions harbor the transactivation domain (TAD) and a variety of motifs that can modulate the function of p53 through signal-specific posttranslational modifications and interactions with other proteins ( Joerger and Fersht, 2008 ).

More than 80% missense mutations of p53 found in human cancers are located in the core domain ( Olivier et al., 2002 ). Many of these mutations disrupt DNA binding directly or reduce the folding stability of the core domain. These observations suggest that DNA binding is important to p53 function. Earlier studies indicated that p53 binds two decameric sites RRRCWWGYYY (R=A, G; W=A, T; Y=C, T), separated by 0-13 base pairs ( el-Deiry et al., 1992 ; Funk et al., 1992 ). Further structural and biochemical analyses show that p53 binds each decameric half site as a dimer and the two dimers from the two half sites constitute the tetramer (or dimer of dimer) ( Ho et al., 2006 ; Kitayner et al., 2006 ; Klein et al., 2001 ; Malecka et al., 2009 ; Rippin et al., 2002 ). The flexible linker between the tetramerization domain and the DNA-binding domain can apparently accommodate the variable spacing between the two half sites. This model of DNA binding is consistent with the fact that p53 functions as a tetramer. It also suggests that the structure of the p53 tetramer on DNA varies depending on the DNA sequences it binds.

With new technologies capable of identifying binding sites of transcription factors at a genome-wide scale, recent studies reveal that p53 binds mostly to two decameric sites with no spacing in vivo ( Smeenk et al., 2008 ; Wei et al., 2006 ). In these studies, more than 80% p53-binding sequences identified by ChIP-on-chip or ChIP-PET (ChIP coupled with paired-end ditag sequencing) contain such a configuration. Considering the background of antibody-based pull-down and other factors (e.g. recruited to DNA by partner proteins), the percentage could be even higher so that p53 binds almost exclusively to elements made of contiguous decameric repeats. We will refer to such sites as the consensus p53 site throughout the text. It is important to point out that while the configuration of the p53 consensus site is conserved, its sequence could vary in different promoters for specific gene regulation ( Ma et al., 2007 ). The conservation of the binding-site configuration suggests that the structure of the p53 tetramer bound to DNA is well defined and that the stereo-specific structure may be important for function. The specific arrangement of binding sites in a composite element is often conserved for protein-protein interactions between neighboring transcription factors that bind DNA cooperatively ( Chen, 1999 ). These protein-protein interactions are usually mediated by DNA-proximal domains, including the DNA-binding domain ( Chen, 1999 ; Chen et al., 1998 ; Wu et al., 2006 ). Indeed, in vitro binding assays show that p53 or its core domain binds contiguous decameric sites cooperatively and any base insertion in between disrupts the cooperativity, however, the structural basis and functional significance of this cooperativity is not fully understood ( Balagurumoorthy et al., 1995 ; Kitayner et al., 2006 ; McLure and Lee, 1998 ; Weinberg et al., 2004 ).

The structure of the p53 core domain and its complexes with DNA has been extensively characterized ( Cho et al., 1994 ; Joerger and Fersht, 2008 ). These studies have revealed the details of protein folding, DNA recognition, and potential mechanisms of some cancer-associated mutations. Protein-protein interaction between p53 core domains in higher-order p53 complexes has also been a subject of experimental and modeling studies ( Cho et al., 1994 ) ( Ho et al., 2006 ) ( Kitayner et al., 2006 ; Klein et al., 2001 ; Ma et al., 2005 ; Rippin et al., 2002 ). These studies have identified a functionally important dimer interface between two p53 core domains bound to the decameric half site. In one study ( Kitayner et al., 2006 ), two such dimers, each bound to a separate DNA molecule, stack end-to-end along the DNA axis. As such, a p53 tetramer is formed in the crystal lattice. However, the two p53 dimers showed little interaction with each other because the two decameric half-sites are separated by two base pairs and the axes of the two DNA molecules are off-set. More recently, the crystal structure of a chemically trapped p53 tetramer bound to DNA has been solved ( Malecka et al., 2009 ). In this complex, the p53 core domain is covalently linked to each quarter site of a full p53-binding element that contains no spacing between the two decameric half sites. The structure showed that the two p53 dimers could bind the adjacent decameric half sites without significant DNA bending ( Ho et al., 2006 ; Nagaich et al., 1999 ; Pan and Nussinov, 2007 ). The structure also revealed an interaction interface between p53 dimers bound to the contiguous decameric half sites. However, as discussed below, the dimer-dimer interactions observed in the chemically trapped complex are likely affected by the covalent crosslinking and may differ significantly from those in the native complex.

In this study, we have solved the crystal structure of a p53 core domain tetramer assembled on a full consensus site. A striking feature of the complex is that the four p53 core domains form an enclosed structure to bind a large region of DNA. The stability of the complex is enhanced by the extensive and interconnected protein-protein and protein-DNA interactions of the core domain, which would function together with the tetramerization domain in the full protein to enhance DNA binding. The structure reveals a novel dimer-dimer interface in the p53 core tetramer and suggests a highly cooperative self-assembling mechanism for its formation. The dimer-dimer interface is absent in previous p53 complexes bound to short DNA and significantly different and smaller in the chemically crosslinked complex. These findings explain, at least partly, the zero spacing in the consensus p53-binding sites observed in vivo . The structural features observed in the naturally assembled p53 core domain tetramer complex underlie the high cooperativity and kinetic stability of DNA binding by p53 to contiguous decameric sites. These structural insights will help to understand and further study the function of p53 and its role in tumor suppression.

Discussion

Genome wide binding-site studies reveal that p53 binds mostly to contiguous decameric half sites in vivo ( Smeenk et al., 2008 ; Wei et al., 2006 ), suggesting that p53 complexes bound to physiological sites have a well-defined structure near the DNA-binding core. This is in contrast to the previous view that p53 binds variable DNA sequences that would implicate a range of different higher-order structures. The specific assembly of the p53 tetramer on DNA must be important for function, thereby imposing a strong selection pressure on the binding site configuration. In this study, we have solved the high-resolution crystal structure of the p53 core domains bound to a full consensus site as a self-assembled tetramer. The most important finding from the present structural study is the identification of a novel dimer-dimer interface that underlies the assembly of the p53 core tetramer on contiguous decameric half sites. Moreover, comparison of the present structure with the previous structures reveals that the binding of p53 to its consensus sites is a highly cooperative self-assembling process that involves structural changes of protein and DNA. However, in contrast to the widely suggested model of DNA bending, our study reveals that it is the structure of the p53 dimer bound to each decameric half sites that undergoes the most significant structural changes, although the center of the DNA does display a subtle but significant distortion from the B-form conformation. Taken together, detailed analyses of the present structure and its comparison with other p53 complexes provide insight into the assembly mechanism and potential functions of the p53 core tetramer.

The overall structure of the core tetramer:DNA complex is similar to that described in various modeling analyses ( Cho et al., 1994 ; Ho et al., 2006 ; McLure and Lee, 1998 ; Pan and Nussinov, 2007 ). The dimer interface within each decameric half site is also similar to that observed in a number of crystal structures and solution studies ( Ho et al., 2006 ; Kitayner et al., 2006 ; Klein et al., 2001 ; Rippin et al., 2002 ). However, the dimer-dimer interface, which would account for the stringent conservation of zero spacing between the decameric half sites, is completely different from previously described. Although the general location of the dimer-dimer interface and some of the interacting residues were correctly located by modeling analyses, the detailed interactions could not be predicted with high confidence because the interface interactions involve mostly flexible loops. In the complexes studied by Kitayner et al., the two p53 dimers bound to discontinuous DNA duplexes make few contacts with each other ( Kitayner et al., 2006 ). This is because the arrangement of the DNA molecules does not conform to the consensus configuration of the p53 binding site ( Smeenk et al., 2008 ; Wei et al., 2006 ). In the chemically trapped p53 tetramer bound to DNA ( Malecka et al., 2009 ), a much more extensive dimer-dimer interface is observed. This interface overlaps partially with that observed here in the naturally assembled complex, but the details of the protein-protein interactions differ significantly. Although we could not rule out the effect of different DNA sequences used in the two studies, the covalent modification is more likely the cause of the difference (see below). The dimer-dimer interface observed here has a larger contact area and better chemical and shape complementarity than that seen in the chemically trapped complex. Moreover, the N-terminal tail, which is predicted by modeling analyses to mediate dimer-dimer interactions ( Cho et al., 1994 ; Ho et al., 2006 ; McLure and Lee, 1998 ; Pan and Nussinov, 2008 ), is disordered in the chemically trapped complex. In the present structure, this region is indeed located at the dimer-dimer interface and engages in extensive protein-protein interactions.

Structural comparison between the tetramer complex and its subcomponents (monomer and dimer complexes) suggests that the binding of the four p53 core domains to DNA is a self-assembling process. The protein-protein interactions observed in the tetramer are not merely incidental contacts between proteins that happen to bind nearby DNA sites, but rather as the result of active self-assembling accompanied by structural changes in proteins and DNA. This is most apparent from the structural comparison between the tetramer and the dimer. Such a self-assembling mechanism presents a challenge to modeling studies, as most of these conformational changes would be difficult to predict. In fact, most modeling analyses suggested that DNA must bend for two rigid p53 dimers to bind contiguous decameric half sites ( Ho et al., 2006 ; Nagaich et al., 1999 ; Pan and Nussinov, 2007 , 2008 ). Our studies here suggest that the conformational changes of the dimer account for a major part of the mechanism of tetramer formation on the consensus site. However, experimental studies in solution indicated that DNA bound by p53 was indeed bent ( Nagaich et al., 1999 ). It is possible that the DNA bends at the junction between the p53 site and the flanking region. The simultaneous binding of the L1 loop and helix H2 in the major groove seems to create a strain in DNA, such that each p53 core domain has the potential to distort the DNA structure at the 5′ end of the quarter site. At the center of the DNA bound by p53, the two core domains apparently compensate for each other, leaving only localized structural deformation at the junction between the two half sites. At the two ends of the p53 site, however, a DNA bend may occur depending on the orientation and DNA contacts of the L1 loop, which is known to modulate the function of p53 in vitro and in vivo ( Sykes et al., 2006 ; Tang et al., 2006 ; Zupnick and Prives, 2006 ). Consistent with this hypothesis, the DNA ends in the present crystal are partially disordered, probably as a result of the “push” by the L1 loop of monomer A and C, which shows a tendency to “spring out” of the major groove based on the partially observed density of the backbone. The self-assembling model also reveals a potential limitation of the chemical trapping strategy used in the study of the p53 tetramer ( Malecka et al., 2009 ). This is because covalent crosslinking could interfere with the cooperation between protein-DNA interaction and protein-protein interaction. Although protein-DNA contacts and the dimer interface near the crosslinking site did not show substantial structural changes ( Malecka et al., 2009 ), a small angular bias imposed by the crosslinking would translate into a larger displacement over a longer distance. The dimer-dimer interface, which is far from the covalent linking site, could therefore be significantly altered.

The tetramer structure explains why the p53 core domain, which is a monomer with low intrinsic DNA affinity, can bind the consensus site cooperatively with high affinity ( Balagurumoorthy et al., 1995 ; McLure and Lee, 1998 ). Although DNA contacts by each subunit in the tetramer are similar to that of the isolated monomer, the DNA binding surfaces of the four monomers are connected by the protein-protein interactions, leading to a large combined DNA-binding surface and enhanced DNA affinity. The tetramer:DNA complex is also known to be kinetically stable ( McLure and Lee, 1998 ), with its dissociation half-life of 15 minutes being much longer than most protein:DNA complexes, including the p53 core dimer bound to a decameric half, which has a half-life of only 30 seconds. The enclosed structure of the p53 tetramer may be responsible for such unusual kinetic stability. While DNA binding and protein-protein interactions can enhance the affinity of the dimer for the decameric half site ( Klein et al., 2001 ), the dissociation of protein in such an open structure is still faster ( Kitayner et al., 2006 ; McLure and Lee, 1998 ). In the tetramer, on the other hand, each subunit is trapped by DNA binding and protein-protein interactions with two neighboring monomers. Therefore, the dimer-dimer interface, which is supported by the contiguous configuration of the p53 consensus site, is important for the thermodynamic, as well as kinetic, stability of DNA binding by p53. The functional importance of this interface is also supported by the conservation of dimer-dimer interactions, however, the dimer-dimer interface seems to be less frequently mutated in cancers compared to some of the hot spots ( Olivier et al., 2002 ). Two structural features of the dimer-dimer interface may partially account for this observation. First, the protein-protein interactions involve a large number of van der Waals contacts by the main chain atoms, which will be less sensitive to side chain changes from missense mutations. Second, the dimer-dimer interface contains multiple discontinuous foci of contacts. Similar protein-protein binding interfaces are also observed in a number of other higher-order transcription factor complexes ( Chen et al., 1998 ; Wu et al., 2006 ). Functional disruption of such interfaces often requires simultaneous mutations of multiple residues ( Chen et al., 1998 ; Wu et al., 2006 ).

P53 has a dedicated tetramerization domain near the C-terminus that plays a key role in cooperative DNA binding by p53. However, since the core DNA binding domain is linked to the tetramerization domain through a flexible linker ( Tidow et al., 2007 ), the tetramerization domain itself is unlikely the factor that constrains the configuration of the p53 consensus site, although it could facilitate the assembly of the p53 core domain tetramer by bringing four monomers in proximity. The fact that the p53 core domain is flexibly linked to the rest of the p53 region at both its N- and C-termini suggest that the self-assembling of the core tetramer is likely maintained in the full-length protein. Interestingly, a p53 variant (p53β) lacking the C-terminal tetramerization domain (residues 331-393) was shown to regulate the expression of specific p53 target genes ( Bourdon et al., 2005 ), suggesting that the presence or absence of the tetramerization domain can indeed affect the formation of the p53 core domain tetramer on selected target sites. Nonetheless, the function of p53β alone could not explain the evolution selection for the self-assembled p53 core domain tetramer and its required consensus site. Rather, tetramerization of the p53 core domain is an important function of the full-length native protein. In addition to enhancing DNA binding, the stereo-specific structure of the core domain tetramer can serve as a structural platform for the folding of other p53 domains and for the binding of proteins that interact with p53 and modulate its functions.
