---
pmid: '21522129'
title: An induced fit mechanism regulates p53 DNA binding kinetics to confer sequence
  specificity.
authors:
- Petty TJ
- Emamzadah S
- Costantino L
- Petkova I
- Stavridi ES
- Saven JG
- Vauthey E
- Halazonetis TD
journal: EMBO J
year: '2011'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC3117648
doi: 10.1038/emboj.2011.127
---

# An induced fit mechanism regulates p53 DNA binding kinetics to confer sequence specificity.
**Authors:** Petty TJ, Emamzadah S, Costantino L, Petkova I, Stavridi ES, Saven JG, Vauthey E, Halazonetis TD
**Journal:** EMBO J (2011)
**DOI:** [10.1038/emboj.2011.127](https://doi.org/10.1038/emboj.2011.127)
**PMC:** [PMC3117648](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3117648/)

## Abstract

1. EMBO J. 2011 Jun 1;30(11):2167-76. doi: 10.1038/emboj.2011.127. Epub 2011 Apr 
26.

An induced fit mechanism regulates p53 DNA binding kinetics to confer sequence 
specificity.

Petty TJ(1), Emamzadah S, Costantino L, Petkova I, Stavridi ES, Saven JG, 
Vauthey E, Halazonetis TD.

Author information:
(1)Department of Molecular Biology, University of Geneva, Geneva, Switzerland.

Comment in
    EMBO J. 2011 Jun 1;30(11):2099-100. doi: 10.1038/emboj.2011.133.

The p53 tumour suppressor gene, the most frequently mutated gene in human 
cancer, encodes a transcription factor that contains sequence-specific DNA 
binding and homo-tetramerization domains. Interestingly, the affinities of p53 
for specific and non-specific DNA sites differ by only one order of magnitude, 
making it hard to understand how this protein recognizes its specific DNA 
targets in vivo. We describe here the structure of a p53 polypeptide containing 
both the DNA binding and oligomerization domains in complex with DNA. The 
structure reveals that sequence-specific DNA binding proceeds via an induced fit 
mechanism that involves a conformational switch in loop L1 of the p53 DNA 
binding domain. Analysis of loop L1 mutants demonstrated that the conformational 
switch allows DNA binding off-rates to be regulated independently of affinities. 
These results may explain the universal prevalence of conformational switching 
in sequence-specific DNA binding proteins and suggest that proteins like p53 
rely more on differences in binding off-rates, than on differences in 
affinities, to recognize their specific DNA sites.

DOI: 10.1038/emboj.2011.127
PMCID: PMC3117648
PMID: 21522129 [Indexed for MEDLINE]

Conflict of interest statement: The authors declare that they have no conflict 
of interest.

## Full Text

Abstract

The p53 tumour suppressor gene, the most frequently mutated gene in human cancer, encodes a transcription factor that contains sequence-specific DNA binding and homo-tetramerization domains. Interestingly, the affinities of p53 for specific and non-specific DNA sites differ by only one order of magnitude, making it hard to understand how this protein recognizes its specific DNA targets in vivo . We describe here the structure of a p53 polypeptide containing both the DNA binding and oligomerization domains in complex with DNA. The structure reveals that sequence-specific DNA binding proceeds via an induced fit mechanism that involves a conformational switch in loop L1 of the p53 DNA binding domain. Analysis of loop L1 mutants demonstrated that the conformational switch allows DNA binding off-rates to be regulated independently of affinities. These results may explain the universal prevalence of conformational switching in sequence-specific DNA binding proteins and suggest that proteins like p53 rely more on differences in binding off-rates, than on differences in affinities, to recognize their specific DNA sites.

Introduction

The most frequently mutated gene in human cancer is p53 ( Hollstein et al, 1991 ; Kan et al, 2010 ), a gene encoding a sequence-specific DNA binding protein ( Kern et al, 1991 ). The p53 protein is activated in response to DNA damage and then enhances transcription of genes that induce cell-cycle arrest, apoptosis or senescence ( Kastan et al, 1991 ; Kuerbitz et al, 1992 ; Vogelstein et al, 2000 ). To explain the high frequency of p53 mutations in human cancer, it has been proposed that activation of oncogenes in precancerous lesions leads to DNA damage, which in turn curtails, in a p53-dependent manner, the macroscopic growth of the lesion ( Bartkova et al, 2005 ; Gorgoulis et al, 2005 ). According to this model, inactivation of p53 is an important step in the progression of precancerous lesions to cancer, because it allows cancer cells to proliferate despite the presence of oncogene-induced DNA damage ( Halazonetis et al, 2008 ).

Given the central role of the p53 protein in human cancer, it is not surprising that significant effort has been devoted towards elucidating its function and structure at the molecular level. These studies have revealed that the full-length p53 protein contains two independently folding domains: a sequence-specific DNA binding domain at the centre of the protein and a homo-tetramerization domain towards the C-terminus ( Vogelstein et al, 2000 ). In addition, p53 contains three unstructured regions: an N-terminal transactivation domain, a linker between the DNA binding and oligomerization domains and a C-terminal basic region ( Joerger and Fersht, 2008 ).

Several three-dimensional structures of the DNA binding domain of p53 have been determined both in the presence of specific DNA and in the absence of DNA ( Cho et al, 1994 ; Ho et al, 2006 ; Malecka et al, 2009 ; Chen et al, 2010 ; Kitayner et al, 2006 , 2010 ). All these structures encompass only the DNA binding domain. The most recent structures are derived from crystals containing four DNA binding domains in complex with specific DNA, thereby potentially recapitulating how full-length p53 tetramers recognize DNA ( Malecka et al, 2009 ; Chen et al, 2010 ; Kitayner et al, 2010 ). With the exception of one structure, in which the p53 DNA binding domains had been chemically crosslinked to DNA ( Malecka et al, 2009 ), the structures show that sequence-specific DNA binding is not accompanied by conformational changes within the p53 DNA binding domain ( Cho et al, 1994 ; Ho et al, 2006 ; Chen et al, 2010 ; Kitayner et al, 2006 , 2010 ). Yet, in the context of practically every other sequence-specific DNA binding protein characterized to date, the interaction with DNA is accompanied by conformational changes ( Frankel and Kim, 1991 ; Alber, 1993 ; Spolar and Record, 1994 ). The significance of these conformational changes is not well understood, but their universal prevalence suggests that they may have an important role and raises the question why p53 is an exception.

The DNA binding domain of p53 is monomeric in solution and has micromolar affinity for DNA ( Weinberg et al, 2005 ). Because of this, except for the crosslinked p53–DNA complex, all the other studied p53–DNA complexes assembled during crystallization, implying that crystal packing interactions have contributed to their formation and, hence, to their structure. In contrast, p53 polypeptides that encompass both the DNA binding and oligomerization domains have nanomolar affinity for sequence-specific DNA and form stable protein–DNA complexes in solution ( Weinberg et al, 2005 ). We envisioned, therefore, that the three-dimensional structures of such complexes would be much less likely to be affected by crystal packing interactions.

We describe here structures of a multidomain p53 oligomer in the presence and absence of DNA. The structures reveal a conformational switch in loop L1, when p53 binds to specific DNA. Analysis of loop L1 mutants further shows that the conformational switch alters the kinetic properties of p53 DNA binding, allowing binding off-rates to be regulated independently of affinities. Since conformational switching is a characteristic of practically all sequence-specific DNA binding proteins ( Frankel and Kim, 1991 ; Alber, 1993 ; Spolar and Record, 1994 ), our findings may be broadly relevant.

Discussion

Several structures of p53 with DNA have been previously determined ( Cho et al, 1994 ; Ho et al, 2006 ; Malecka et al, 2009 ; Chen et al, 2010 ; Kitayner et al, 2006 , 2010 ). An unusual feature of all these structures, with the exception of one structure of p53 crosslinked with DNA ( Malecka et al, 2009 ), is that neither the DNA binding domain of p53, nor the DNA undergo major conformational changes upon sequence-specific DNA binding. In contrast, conformational changes are a characteristic of practically every other sequence-specific DNA binding protein characterized to date ( Frankel and Kim, 1991 ; Alber, 1993 ; Spolar and Record, 1994 ), raising the question why p53 would be an exception. The p53–DNA structure reported here is the first structure of a p53 polypeptide that includes both the DNA binding domain and the oligomerization domain and the first structure to exhibit a conformational switch upon sequence-specific DNA binding.

We propose that the structure reported here is the physiologically relevant structure of a p53 tetramer–DNA complex. First, the presence of the oligomerization domain allows p53 to form stable complexes with DNA in solution ( Supplementary Figure S7 ). Therefore, p53–DNA complex formation preceded crystallization. In contrast, isolated p53 DNA binding domains bind DNA weakly ( Weinberg et al, 2005 ) and only form complexes with DNA during crystallization ( Cho et al, 1994 ; Chen et al, 2010 ; Kitayner et al, 2006 , 2010 ); therefore, the assembly of these complexes is guided both by p53–DNA contacts and by crystal packing contacts. In this regard, it is not surprising that the previously reported p53 tetramer–DNA structures all differ from one another ( Supplementary Figure S9 ). Second, the structure reported here is insensitive to variations in the linker length between the p53 DNA binding and oligomerization domains and insensitive to variations in the oligonucleotide used. While we had to truncate the linker between the DNA binding and oligomerization domains, a common practice in crystallography to reduce conformational flexibility that inhibits crystallization, two p53 polypeptides with different size linkers, p53CR2 and p53CR3, formed essentially identical structures ( Supplementary Figure S8 ). Thus, linker length variation did not impact on how the four DNA binding domains interacted with DNA. We also obtained crystals of p53CR2 and p53CR3 with oligonucleotides containing different specific DNA sites and/or different overhangs (5′ versus 3′ and different overhang sequences) and in every case the structures were identical to the one reported here (our own unpublished observations). Thus, so far, we have observed one unique p53 tetramer–DNA conformation. Third, we validated our structure by second-site revertant mutagenesis using full-length p53 proteins expressed in human cells ( Figure 3 ).

The main question that arises from our structure relates to the significance of the conformational switch targeting loop L1. Interestingly, it appears that the propensity of loop L1 to adopt a recessed conformation is conserved in evolution, since the recessed conformation, observed here in human p53, is practically identical to the recessed conformation observed in all four subunits of mouse p53 crosslinked to DNA ( Supplementary Figure S12 ; Malecka et al, 2009 ) and similar to the conformation of loop L1 of C. elegans p53 ( Huyen et al, 2004 ). Further, Lys120, at the tip of loop L1, is acetylated in response to DNA damage and this acetylation, which has the potential to affect the conformation of the loop, regulates p53 DNA binding specificity in vivo ( Sykes et al, 2006 ). As discussed below, our experiments suggest that the conformational switch of loop L1 is critical for the ability of p53 to distinguish specific from non-specific DNA.

It has been proposed that sequence-specific DNA binding proteins are able to recognize their specific sites in the genome because their affinities for specific and non-specific DNAs differ by 3–7 orders of magnitude ( Spolar and Record, 1994 ). However, in many studies in the past, DNA binding affinities were measured under non-equilibrium conditions by filter binding and electrophoretic mobility shift assays (e.g., Klig et al, 1987 ; Hellman and Fried, 2007 ). Because complexes of proteins with non-specific DNA, typically have half-lives in the order of milliseconds ( von Hippel and Berg, 1989 ; Elf et al, 2007 ), such complexes dissociate in non-equilibrium assays, leading to underestimation of their affinities. When p53 DNA binding was studied under equilibrium conditions and at physiological salt concentrations ( Weinberg et al, 2004 and this study), the difference in affinities for specific and non-specific DNAs was <10-fold, which makes it very difficult to explain how p53 recognizes its target sites in the genome. We propose that the conformational switch of loop L1 addresses this problem by allowing DNA binding off-rates to be regulated independently of affinities and that it is the large difference in binding off-rates that distinguishes specific from non-specific DNA sites.

To our knowledge, a model, whereby sequence-specific DNA binding proteins modulate DNA binding off-rates independently of affinities in order to discriminate specific targets from non-specific DNA, has not been previously proposed ( Kalodimos et al, 2004a ). Yet, this model may help explain the almost universal prevalence of conformational switches in protein–DNA interactions ( Frankel and Kim, 1991 ; Alber, 1993 ; Spolar and Record, 1994 ). One can envision that, at any given time, most molecules of a sequence-specific DNA binding protein would be bound to non-specific DNA, because of the relatively high affinity for such DNA. However, the very high off-rate for non-specific DNA means that the protein molecules would rapidly bounce from DNA site to DNA site, via sliding and intersegment transfer, as previously proposed ( von Hippel and Berg, 1989 ; McKinney et al, 2004 ). When a specific DNA site is identified, the protein and/or the DNA would switch conformation, resulting in a significant decrease in the on-rate and off-rate constants—akin to locking the protein at the specific DNA site—without a major increase in affinity. In support of this model, it is noteworthy that proteins other than p53 also have high affinity for non-specific DNA at physiological salt concentrations ( Oda et al, 1998 ) and that binding of sequence-specific proteins to non-specific DNA is not accompanied by conformational switching ( Winkler et al, 1993 ; Newman et al, 1995 ; Lewis et al, 1996 ; Viadiu and Aggarwal, 2000 ; Kalodimos et al, 2004b ).
