---
pmid: '25852190'
title: Integrative analysis of kinase networks in TRAIL-induced apoptosis provides
  a source of potential targets for combination therapy.
authors:
- So J
- Pasculescu A
- Dai AY
- Williton K
- James A
- Nguyen V
- Creixell P
- Schoof EM
- Sinclair J
- Barrios-Rodiles M
- Gu J
- Krizus A
- Williams R
- Olhovsky M
- Dennis JW
- Wrana JL
- Linding R
- Jorgensen C
- Pawson T
- Colwill K
journal: Sci Signal
year: '2015'
full_text_available: true
pmcid: PMC11666773
doi: 10.1126/scisignal.2005700
---

# Integrative analysis of kinase networks in TRAIL-induced apoptosis provides a source of potential targets for combination therapy.
**Authors:** So J, Pasculescu A, Dai AY, Williton K, James A, Nguyen V, Creixell P, Schoof EM, Sinclair J, Barrios-Rodiles M, Gu J, Krizus A, Williams R, Olhovsky M, Dennis JW, Wrana JL, Linding R, Jorgensen C, Pawson T, Colwill K
**Journal:** Sci Signal (2015)
**DOI:** [10.1126/scisignal.2005700](https://doi.org/10.1126/scisignal.2005700)
**PMC:** [PMC11666773](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11666773/)

## Abstract

1. Sci Signal. 2015 Apr 7;8(371):rs3. doi: 10.1126/scisignal.2005700.

Integrative analysis of kinase networks in TRAIL-induced apoptosis provides a 
source of potential targets for combination therapy.

So J(1), Pasculescu A(2), Dai AY(2), Williton K(2), James A(2), Nguyen V(2), 
Creixell P(3), Schoof EM(3), Sinclair J(4), Barrios-Rodiles M(2), Gu J(2), 
Krizus A(2), Williams R(2), Olhovsky M(2), Dennis JW(5), Wrana JL(5), Linding 
R(6), Jorgensen C(7), Pawson T(8), Colwill K(9).

Author information:
(1)Lunenfeld-Tanenbaum Research Institute, Mount Sinai Hospital, Toronto, 
Ontario M5G 1X5, Canada. Institute of Medical Science, University of Toronto, 
Toronto, Ontario M5S 1A8, Canada.
(2)Lunenfeld-Tanenbaum Research Institute, Mount Sinai Hospital, Toronto, 
Ontario M5G 1X5, Canada.
(3)Cellular Signal Integration Group (C-SIG), Technical University of Denmark 
(DTU), DK-2800 Lyngby, Denmark.
(4)Cell Communication Team, The Institute of Cancer Research, London SW3 6JB, 
UK.
(5)Lunenfeld-Tanenbaum Research Institute, Mount Sinai Hospital, Toronto, 
Ontario M5G 1X5, Canada. Department of Molecular Genetics, University of 
Toronto, Toronto, Ontario M5S 1A8, Canada.
(6)Cellular Signal Integration Group (C-SIG), Technical University of Denmark 
(DTU), DK-2800 Lyngby, Denmark. Biotech Research and Innovation Centre (BRIC), 
University of Copenhagen (UCPH), DK-2200 Copenhagen, Denmark. 
linding@lindinglab.org claus.jorgensen@cruk.manchester.ac.uk 
colwill@lunenfeld.ca.
(7)Cell Communication Team, The Institute of Cancer Research, London SW3 6JB, 
UK. linding@lindinglab.org claus.jorgensen@cruk.manchester.ac.uk 
colwill@lunenfeld.ca.
(8)Lunenfeld-Tanenbaum Research Institute, Mount Sinai Hospital, Toronto, 
Ontario M5G 1X5, Canada. Institute of Medical Science, University of Toronto, 
Toronto, Ontario M5S 1A8, Canada. Department of Molecular Genetics, University 
of Toronto, Toronto, Ontario M5S 1A8, Canada.
(9)Lunenfeld-Tanenbaum Research Institute, Mount Sinai Hospital, Toronto, 
Ontario M5G 1X5, Canada. linding@lindinglab.org 
claus.jorgensen@cruk.manchester.ac.uk colwill@lunenfeld.ca.

Tumor necrosis factor-related apoptosis-inducing ligand (TRAIL) is an endogenous 
secreted peptide and, in preclinical studies, preferentially induces apoptosis 
in tumor cells rather than in normal cells. The acquisition of resistance in 
cells exposed to TRAIL or its mimics limits their clinical efficacy. Because 
kinases are intimately involved in the regulation of apoptosis, we 
systematically characterized kinases involved in TRAIL signaling. Using RNA 
interference (RNAi) loss-of-function and cDNA overexpression screens, we 
identified 169 protein kinases that influenced the dynamics of TRAIL-induced 
apoptosis in the colon adenocarcinoma cell line DLD-1. We classified the kinases 
as sensitizers or resistors or modulators, depending on the effect that 
knockdown and overexpression had on TRAIL-induced apoptosis. Two of these 
kinases that were classified as resistors were PX domain-containing 
serine/threonine kinase (PXK) and AP2-associated kinase 1 (AAK1), which promote 
receptor endocytosis and may enable cells to resist TRAIL-induced apoptosis by 
enhancing endocytosis of the TRAIL receptors. We assembled protein interaction 
maps using mass spectrometry-based protein interaction analysis and quantitative 
phosphoproteomics. With these protein interaction maps, we modeled information 
flow through the networks and identified apoptosis-modifying kinases that are 
highly connected to regulated substrates downstream of TRAIL. The results of 
this analysis provide a resource of potential targets for the development of 
TRAIL combination therapies to selectively kill cancer cells.

Copyright © 2015, American Association for the Advancement of Science.

DOI: 10.1126/scisignal.2005700
PMID: 25852190 [Indexed for MEDLINE]

## Full Text

IntroductionSynthetic binding proteins with high affinity and selectivity can be engineered from natural immunoglobulin scaffolds (e.g., scFvs, Fabs, nanobodies) or non-immunoglobulin scaffolds (e.g., monobodies, DARPins, affibodies, anticalins)1–3. They are broadly used as research tools in structural-, cell- and molecular biology4. Some protein binder classes, such as DARPins, monobody and its analuegs, affibodies and anticalins, moved to clinical development stage for the treatment of cancer and other diseases. Their small size (~8–20 kDa), rapid generation by molecular display techniques, high-affinity target binding and facile recombinant production enables targeting of a broad range of diverse proteins and promises great therapeutic opportunities5.Among the most commonly used non-immunoglobulin binders are monobodies (Mb), which are generated from large combinatorial libraries using a fibronectin III domain (FN3; ~10 kDa) of human fibronectin as molecular scaffold6,7. Since their first report6, pharma and biotech industries developed monobody analogues, termed adnectins, tenascins and centyrins, which target extracellular proteins and receptors, such as VEGFR2, EGFR, PCSK9 and were evaluated in phase I-II clinical trials3,4,8,9. We have developed monobodies that target major (intracellular) oncogenes, including kinases (BCR::ABL1, SRC), phosphatases (SHP-2), transcription factors (STAT3), chromatin readers (WDR5) and small GTPases (H-/K-RAS)10–18. Monobodies to other targets, such as the SARS-CoV-2 spike protein and optineurin were recently developed by others19,20.Upon genetic expression of monobodies in tumor cell lines and primary cells, we and others observed selective inhibition of oncoprotein-dependent signaling11–13,15,16. We are developing technologies for intracellular monobody protein delivery and, in parallel, improved pharmacokinetics and biodistribution of monobodies2,21–23. For therapeutic translation, a high plasma and intracellular stability is an important prerequisite and of hallmark importance to enable in vivo application and efficacy of monobodies. Therefore, we set out to develop mirror-image d-monobodies.Mirror-image proteins that are formed from amino acids in d-configuration exhibit several advantages compared to naturally occurring l-proteins: Higher in vivo pharmacological stability, such as higher plasma half-life, is due to protease resistance, as d-peptide bonds cannot be cleaved by natural proteases. In addition, d-proteins display very low immunogenicity, as immune cells cannot proteolytically generate d-peptides, and even if fragments were generated, they would not be able to be displayed on the major histocompatibility complex (MHC)24–26.A common approach to develop d-binding proteins is the so-called mirror-image phage display, which circumvents the challenge of directly developing such binding proteins using a d-molecular display technology27. First, a relevant target protein is produced in the d-configuration through solid-phase peptide synthesis (SPPS), followed by stepwise native chemical ligation (NCL) of the peptides and refolding to obtain the folded mirror-image target protein28,29. This d-protein is then subjected to a selection process with the combinatorial l-protein binder libraries. As the selected l-binders provide a blueprint for d-protein binders to the natural l-target protein, the synthesized d-binder (by SPPS, NCL and refolding) will consequently, by symmetry, bind to the natural l-target (Fig. 1). This workflow was applied to develop d-peptide antagonists of different protein-protein interactions27,30–33.Fig. 1Workflow of Bcr-Abl SH2 synthesis as d-target for mirror-image monobody screening.The d-Bcr-Abl SH2 domain was prepared by native chemical ligation (NCL) and subsequent desulfurization from two fragments produced by solid-phase peptide synthesis (SPPS), as previously described34. After refolding and structure validation, the d-target was subjected to monobody selection through phage and yeast display yielding l-monobody binders. Following binder characterization, most promising monobodies were synthesized in d-configuration resulting in d-monobodies targeting the natural l-Bcr-Abl SH2 domain.While d-peptides can be readily made by SPPS, d-protein binder synthesis requires the same tedious process consisting of SPPS, NCL of peptides and subsequent refolding as production of the d-target protein for screening. Although pioneering work by the Kent and Sidhu groups reported mirror-image GB1 protein variants (56 amino acids) targeting VEGF-A, to our knowledge, their approach has not resulted in a therapeutic application or development of protein binders in d-configuration to other target proteins to date24,25. Other well-established engineered binding proteins, including antibody fragments and DARPins have not been reported in the d-form so far. Our previous work demonstrated the successful chemical synthesis of the BCR::ABL1 SH2 domain34 and a proof-of-concept study showed the synthesis of a d-monobody, but without target binding function, using a 3 peptide ligation strategy26. To explore the feasibility of d-monobody development and assess its full potential, we set out to develop a more facile strategy to synthesize d-monobodies and chose a therapeutically important intracellular target, the SH2 domain of BCR::ABL1. The Philadelphia chromosomal translocation produces a fusion of the breakpoint cluster region (BCR) and the Abelson tyrosine kinase (ABL1) genes, which results in the expression of the BCR::ABL1 protein. In chronic myeloid leukemia (CML) and a large fraction of B-cell acute lymphoblastic leukemias, the constitutive tyrosine kinase activity of BCR::ABL1 is the central driver of leukemogenesis35. While different BCR::ABL1 tyrosine kinase inhibitors (TKIs) have strongly improved the overall survival of most CML patients, TKI-resistance and -intolerance result in disease recurrence and prevent cure35,36. We previously developed a series of (l-)monobodies directed to the Src homology 2 (SH2) domain of BCR::ABL1, which resulted in inhibition of BCR::ABL1 activity, signaling and leukemogenesis in CML cells through different mechanisms-of-action10,11,13. SH2 domains are modular protein-protein interaction domains with 120 members in humans and bind tyrosine-phosphorylated peptide sequences with moderate selectivity and affinities in the low micromolar range37. Despite being important drug targets, selective inhibition of SH2 domains by high-affinity peptides, peptidomimetics or small molecules are notoriously difficult to develop, although some good progress was made recently38–41. While BCR::ABL1 is an intracellular protein and no established technologies for efficient cytosolic delivery of d-monobodies are available yet, we still decided to follow this ambitious path, which will enable the rapid assessment of delivered d-monobody proteins once fully validated delivery approaches, which we and others are developing, become available in the near future2. Furthermore, BCR::ABL1 is an established therapeutic target and our lab has made important contributions to the BCR::ABL1 field11,18,42–45. Therefore, many tools and assays for the characterization of BCR::ABL1-targeting monobodies are readily available in our lab and we are able to compare newly generated monobodies with previously published ones10,13.Here, we show the development of two d-monobodies targeting the BCR::ABL1 SH2 domain with high affinity and their structural characterization by solving the crystal structures of heterochiral monobody-SH2 complexes. For functional characterization, one of the d-monobodies is refolded to its native mirror-conformation after NCL of two d-peptide segments and subsequent desulfurization. For the other d-monobody clone, NCL is omitted and a functional split-d-monobody is created by refolding a 1:1 mixture of two peptides spanning the entire length of the monobody. Subsequent functional characterization shows high-affinity target binding, protease resistance, high plasma stability, inhibition of BCR::ABL1 kinase activity and binding to BCR::ABL1 in CML cell lysates and permeabilized cells. We also refer to a parallel study by Hayashi et al. that develops a d-monobody targeting the inflammation-related cytokine MCP-1 using a similar workflow46. Our two studies utilize different but complementary approaches and demonstrate the facile development of a functional mirror-image d-monobody targeting a therapeutically important protein. This convergence suggests the robustness of the platforms and provides strong evidence for the feasibility and potential impact of mirror-image monobody binders.

Introduction

Synthetic binding proteins with high affinity and selectivity can be engineered from natural immunoglobulin scaffolds (e.g., scFvs, Fabs, nanobodies) or non-immunoglobulin scaffolds (e.g., monobodies, DARPins, affibodies, anticalins)1–3. They are broadly used as research tools in structural-, cell- and molecular biology4. Some protein binder classes, such as DARPins, monobody and its analuegs, affibodies and anticalins, moved to clinical development stage for the treatment of cancer and other diseases. Their small size (~8–20 kDa), rapid generation by molecular display techniques, high-affinity target binding and facile recombinant production enables targeting of a broad range of diverse proteins and promises great therapeutic opportunities5.

Among the most commonly used non-immunoglobulin binders are monobodies (Mb), which are generated from large combinatorial libraries using a fibronectin III domain (FN3; ~10 kDa) of human fibronectin as molecular scaffold6,7. Since their first report6, pharma and biotech industries developed monobody analogues, termed adnectins, tenascins and centyrins, which target extracellular proteins and receptors, such as VEGFR2, EGFR, PCSK9 and were evaluated in phase I-II clinical trials3,4,8,9. We have developed monobodies that target major (intracellular) oncogenes, including kinases (BCR::ABL1, SRC), phosphatases (SHP-2), transcription factors (STAT3), chromatin readers (WDR5) and small GTPases (H-/K-RAS)10–18. Monobodies to other targets, such as the SARS-CoV-2 spike protein and optineurin were recently developed by others19,20.

Upon genetic expression of monobodies in tumor cell lines and primary cells, we and others observed selective inhibition of oncoprotein-dependent signaling11–13,15,16. We are developing technologies for intracellular monobody protein delivery and, in parallel, improved pharmacokinetics and biodistribution of monobodies2,21–23. For therapeutic translation, a high plasma and intracellular stability is an important prerequisite and of hallmark importance to enable in vivo application and efficacy of monobodies. Therefore, we set out to develop mirror-image d-monobodies.

Mirror-image proteins that are formed from amino acids in d-configuration exhibit several advantages compared to naturally occurring l-proteins: Higher in vivo pharmacological stability, such as higher plasma half-life, is due to protease resistance, as d-peptide bonds cannot be cleaved by natural proteases. In addition, d-proteins display very low immunogenicity, as immune cells cannot proteolytically generate d-peptides, and even if fragments were generated, they would not be able to be displayed on the major histocompatibility complex (MHC)24–26.

A common approach to develop d-binding proteins is the so-called mirror-image phage display, which circumvents the challenge of directly developing such binding proteins using a d-molecular display technology27. First, a relevant target protein is produced in the d-configuration through solid-phase peptide synthesis (SPPS), followed by stepwise native chemical ligation (NCL) of the peptides and refolding to obtain the folded mirror-image target protein28,29. This d-protein is then subjected to a selection process with the combinatorial l-protein binder libraries. As the selected l-binders provide a blueprint for d-protein binders to the natural l-target protein, the synthesized d-binder (by SPPS, NCL and refolding) will consequently, by symmetry, bind to the natural l-target (Fig. 1). This workflow was applied to develop d-peptide antagonists of different protein-protein interactions27,30–33.Fig. 1Workflow of Bcr-Abl SH2 synthesis as d-target for mirror-image monobody screening.The d-Bcr-Abl SH2 domain was prepared by native chemical ligation (NCL) and subsequent desulfurization from two fragments produced by solid-phase peptide synthesis (SPPS), as previously described34. After refolding and structure validation, the d-target was subjected to monobody selection through phage and yeast display yielding l-monobody binders. Following binder characterization, most promising monobodies were synthesized in d-configuration resulting in d-monobodies targeting the natural l-Bcr-Abl SH2 domain.

Workflow of Bcr-Abl SH2 synthesis as d-target for mirror-image monobody screening.

The d-Bcr-Abl SH2 domain was prepared by native chemical ligation (NCL) and subsequent desulfurization from two fragments produced by solid-phase peptide synthesis (SPPS), as previously described34. After refolding and structure validation, the d-target was subjected to monobody selection through phage and yeast display yielding l-monobody binders. Following binder characterization, most promising monobodies were synthesized in d-configuration resulting in d-monobodies targeting the natural l-Bcr-Abl SH2 domain.

While d-peptides can be readily made by SPPS, d-protein binder synthesis requires the same tedious process consisting of SPPS, NCL of peptides and subsequent refolding as production of the d-target protein for screening. Although pioneering work by the Kent and Sidhu groups reported mirror-image GB1 protein variants (56 amino acids) targeting VEGF-A, to our knowledge, their approach has not resulted in a therapeutic application or development of protein binders in d-configuration to other target proteins to date24,25. Other well-established engineered binding proteins, including antibody fragments and DARPins have not been reported in the d-form so far. Our previous work demonstrated the successful chemical synthesis of the BCR::ABL1 SH2 domain34 and a proof-of-concept study showed the synthesis of a d-monobody, but without target binding function, using a 3 peptide ligation strategy26. To explore the feasibility of d-monobody development and assess its full potential, we set out to develop a more facile strategy to synthesize d-monobodies and chose a therapeutically important intracellular target, the SH2 domain of BCR::ABL1. The Philadelphia chromosomal translocation produces a fusion of the breakpoint cluster region (BCR) and the Abelson tyrosine kinase (ABL1) genes, which results in the expression of the BCR::ABL1 protein. In chronic myeloid leukemia (CML) and a large fraction of B-cell acute lymphoblastic leukemias, the constitutive tyrosine kinase activity of BCR::ABL1 is the central driver of leukemogenesis35. While different BCR::ABL1 tyrosine kinase inhibitors (TKIs) have strongly improved the overall survival of most CML patients, TKI-resistance and -intolerance result in disease recurrence and prevent cure35,36. We previously developed a series of (l-)monobodies directed to the Src homology 2 (SH2) domain of BCR::ABL1, which resulted in inhibition of BCR::ABL1 activity, signaling and leukemogenesis in CML cells through different mechanisms-of-action10,11,13. SH2 domains are modular protein-protein interaction domains with 120 members in humans and bind tyrosine-phosphorylated peptide sequences with moderate selectivity and affinities in the low micromolar range37. Despite being important drug targets, selective inhibition of SH2 domains by high-affinity peptides, peptidomimetics or small molecules are notoriously difficult to develop, although some good progress was made recently38–41. While BCR::ABL1 is an intracellular protein and no established technologies for efficient cytosolic delivery of d-monobodies are available yet, we still decided to follow this ambitious path, which will enable the rapid assessment of delivered d-monobody proteins once fully validated delivery approaches, which we and others are developing, become available in the near future2. Furthermore, BCR::ABL1 is an established therapeutic target and our lab has made important contributions to the BCR::ABL1 field11,18,42–45. Therefore, many tools and assays for the characterization of BCR::ABL1-targeting monobodies are readily available in our lab and we are able to compare newly generated monobodies with previously published ones10,13.

Here, we show the development of two d-monobodies targeting the BCR::ABL1 SH2 domain with high affinity and their structural characterization by solving the crystal structures of heterochiral monobody-SH2 complexes. For functional characterization, one of the d-monobodies is refolded to its native mirror-conformation after NCL of two d-peptide segments and subsequent desulfurization. For the other d-monobody clone, NCL is omitted and a functional split-d-monobody is created by refolding a 1:1 mixture of two peptides spanning the entire length of the monobody. Subsequent functional characterization shows high-affinity target binding, protease resistance, high plasma stability, inhibition of BCR::ABL1 kinase activity and binding to BCR::ABL1 in CML cell lysates and permeabilized cells. We also refer to a parallel study by Hayashi et al. that develops a d-monobody targeting the inflammation-related cytokine MCP-1 using a similar workflow46. Our two studies utilize different but complementary approaches and demonstrate the facile development of a functional mirror-image d-monobody targeting a therapeutically important protein. This convergence suggests the robustness of the platforms and provides strong evidence for the feasibility and potential impact of mirror-image monobody binders.

ResultsGeneration of l-monobodies targeting the d-Abl SH2 domainWe first produced the d-version of the human BCR::ABL1 SH2 domain (Abl SH2) by native chemical ligation (NCL) of two d-peptide fragments obtained by solid-phase peptide synthesis (SPPS). The cysteine residue required for NCL was subsequently desulfurized to an alanine residue to yield the wildtype sequence (Fig. 1)34. As previously described, the protein was monomeric in size exclusion chromatography (SEC) after refolding, showed a similar secondary structure composition and thermal stability as the recombinant Abl SH2 domain, and bound a d-phospho tyrosine (pY) peptide with the same affinity as the l-counterpart to the corresponding l-pY peptide34.We selected monobodies from the combinatorial “side-and-loop” phage-display library and subsequent yeast display screening (Supplementary information, section 2)7. Monobody pools enriched with high-affinity binders were identified after four rounds of yeast display with decreasing target concentrations. Six monobody clones with unique sequences were isolated and further characterized (Fig. 2a). Three clones had a diversified CD loop, whereas the others had a CD loop corresponding to the wildtype FN3 sequence7. The FG loop, which was identical for four out of the six clones, differed in sequence, but not in length. All clones bound to the d-Abl SH2 domain with similar affinities between 20 and 71 nM, as measured in the yeast display format13, with clones 21 and 27 (hereafter termed DAM21 and DAM27) showing tightest apparent binding (Fig. 2b). We next expressed and purified the recombinant monobodies in E. coli with high yields (44–54 mg/L E. coli culture, Fig. 2c, d). Preparative SEC showed high purity and that the elution volumes of all monobodies corresponded to their calculated monomeric molecular weights, except for DAM28, which eluted at a higher molecular weight and was therefore not selected for further follow-up (Fig. 2e). These monobodies robustly blocked binding of a d-pY peptide to the d-Abl SH2 domain (Kd = ~5 µM, Fig. S1) as tested with a fluorescence polarization (FP) binding assay. All monobody clones produced a strong FP signal decrease, which was comparable to l-pY competition by the previously characterized l-Abl SH2-targeting monobody HA410. In contrast, monobody AS25 that targets the allosteric SH2-kinase domain interface of ABL1 did not show competition of the l-pY-SH2 interaction (Fig. 2f)13. These results suggest that the DAM monobodies bind to the pY binding site of the SH2 domain. To determine the thermodynamic binding parameters of the monobodies more precisely, we performed isothermal titration calorimetry (ITC) measurements using purified monobody and SH2 proteins. The determined binding affinities in the low to mid nanomolar range were generally consistent with the yeast display measurements (Figs. 2g–j, S2). All measurements suggested a d-Abl SH2:l-monobody binding stoichiometry of 1:1 and binding enthalpies in the range of −75 to −36 kJ/mol in line with the previously observed strongly enthalpically driven binding of l-monobodies to their l-targets. Given that all monobody clones were pY competitors we focused on DAM21 and DAM27 for further characterization, as they had the highest apparent binding affinities.Fig. 2l-monobody selection and characterization against d-Bcr-Abl SH2.a Amino acid sequences of d-Abl SH2 l-monobody binders (DAM) generated by phage and yeast display selection based on the combinatorial ‘side-and-loop’ library. In the library designs, “X” denotes a mixture of 30% Tyr, 15% Ser, 10% Gly, 5% Phe, 5% Trp, and 2.5% each of all the other amino acids except for Cys; “O” denotes a mixture of Asn, Asp, His, Ile, Leu, Phe, Tyr, and Val; “U” denotes a mixture of His, Leu, Phe, and Tyr; and “Z” denotes a mixture of Ala, Glu, Lys, and Thr. A hyphen indicates a deletion. b Binding titrations in the yeast surface display format to estimate binding affinities of l-monobodies to d-Abl SH2. The mean fluorescence intensities of yeast cells displaying a monobody are plotted as a function of the concentration of the target and fitted to a 1:1 binding model. c Sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) analysis (performed once), (d) retention volumes in size exclusion chromatography (SEC) purification and expression yields, and (e) SEC chromatograms of selected l-DAMs recombinantly expressed in E. coli. (f) Competitive fluorescence polarization (FP) assay of l-DAMs incubated with d-Abl SH2 and fluorescently labeled d-pYEEI peptide binders in comparison with previously published l-monobodies HA4 (pY peptide competitor) and AS25 (allosteric binder) against l-Abl SH2 incubated with the corresponding l-pYEEI peptide. Measured data from two independent experiments (depicted as dots) were averaged. g–j Isothermal titration calorimetry (ITC) measurements of recombinant l-monobodies (g) DAM21 and (h) DAM27 titrated to the synthetic d-Abl SH2 domain. Each panel shows the raw heat signal of an ITC experiment (top) and the integrated calorimetric data of the area of each peak (bottom). The continuous line represents the best fit of the data based on a 1:1 binding model computed from the MicroCal software. Binding parameters including Kd value, stoichiometry (N), enthalpy (∆H), free enthalpy (∆G) and −T∆S calculated from the fit of each experiment are shown in (i) and (j). A representative measurement of at least two ITC experiments for each monobody is shown. Source data of (a, c and f) are provided as a Source Data file.Structures of heterochiral Abl SH2-monobody complexesTo understand the structural basis for monobody recognition of the d-Abl SH2 domain, we set out to determine the crystal structures of monobody-SH2 complexes. We mixed the synthetic d-Abl SH2 domain with recombinantly expressed l-DAM21 or l-DAM27 monobody, respectively, and purified the heterochiral 1:1-complexes by SEC (Fig. 3a–e). Of note, to satisfy the large amounts of protein complex needed for crystallization, this batch of the d-Abl SH2 domain was not desulfurized to alanine after NCL and therefore contained a cysteine residue at position 198 (Table S3, Figs. S3–S5). The alanine to cysteine mutation at this position in recombinant l-Abl SH2 did not change binding affinity to l-monobody AS25 and l-pY peptide (Fig. S6). Crystals were obtained readily or after an additional additive screen. We determined the crystal structures of the l-DAM21-d-Abl SH2 and l-DAM27-d-Abl SH2 complexes at 2.7 Å and 2.9 Å resolution, respectively (Fig. 4a, b, Table 1, Fig. S7a, b). In both structures, we found the d-SH2 domain to be dimerized via a disulfide bridge of Cys198. In addition, we found two different interaction interfaces of the monobody with the SH2 domain, which were very similar in both complexes (Fig. S7c, d). The monobody FG loop, which represents the most extensively diversified region in the library, forms a large part of both interfaces, which have similar buried surface areas (431 Å2 and 440 Å2 in DAM21, 452 Å2 and 427 Å2 in DAM27, for interface 1 and 2, respectively). Interface 2 of both structures largely contains the pY peptide-binding interface (346 Å2 for the pYEEI peptide, Fig. 4c), whereas interface 1 comprises a region of the SH2 domain surface of unknown functional significance. Given the robust pY peptide competition of both monobodies (see Fig. 2f) and mutagenesis data described below, we considered interface 2 as the likely biologically relevant interface.Fig. 3Complex formation and structure elucidation of the heterochiral d-Abl SH2:l-monobody interaction.a Workflow for generation of the heterochiral d-Abl SH2:l-monobody complex. d-Abl SH2 was produced by solid-phase peptide synthesis (SPPS) and native chemical ligation (NCL) and the l-monobody by bacterial expression and subsequent purification. After mixing, the complex was purified by size exclusion chromatography (SEC). b, c Analytical SEC of complex formation between synthetic d-Abl SH2 and recombinantly expressed (b) l-DAM21 and (c) l-DAM27. d Preparative SEC of d-Abl SH2:l-monobody complexes generated for crystallization condition screening. All chromatograms include a calibration of the column with standard proteins on top. e Sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) analysis of the complex formation (performed once). Source data of (e) are provided as a Source Data file.Fig. 4Crystal structures of heterochiral monobody-Abl SH2 complexes.a, b Ribbon representation of the d-Abl SH2 domain structure (green) bound to the (a) DAM27 or (b) DAM21 monobodies in l-configuration. Arg171 of the SH2 FLVRES motif, which is critical for pY binding, as well as Glu85 in the FG loop of the monobodies is shown in stick representation. c Structure of an (l-) phosphotyrosine peptide (pYEEI, stick representation) bound to the (l-)SH2 domain of the Src family kinase Lck (PDB: 1LKK). The critical Arg residue of the FLVRES motif in the pY binding pocket is shown in stick representation. d, e Close-up view of the d-SH2:(l-)monobody interface (DAM27, panel d; DAM21, panel e) highlighting the heterochiral “rippled” anti-parallel β-sheet between strand Gly84 to Trp88/His88 in DAM21/27 of the monobody FG loop with Val190 to Arg194 of the βD-strand of the d-Abl SH2 domain. f, g Close-up view of the d-SH2:(l-)monobody interface (DAM27, panel f; DAM21, panel g). The ionic interactions discussed in the “Results” section are shown as sticks and are labeled. The position of randomized residues in the monobody side-and-loop library is shown as balls. h Isothermal titration calorimetry (ITC) measurements of recombinant l-DAM21 D83A/E85A mutant titrated to the synthetic d-Abl SH2 domain. The panel shows the raw heat signal of an ITC experiment (top) and the integrated calorimetric data of the area of each peak (bottom). i Structural superposition and sequence alignment of the FG loops of the DAM27 and DAM21 monobodies.Both monobodies use their long FG loops to present an extended segment that is positioned parallel to the central SH2 β-sheet and therefore perpendicular to the canonical backbone conformation of an SH2 domain-bound pY peptide (Fig. 4a–c). Likewise, several l-monobodies mimic pY peptide binding by also binding perpendicular across the central SH2 β-sheet, which thereby enables access to the pY and +3 pockets that are located on opposite sides of the SH2 β-sheet12,14. Unlike those, DAM21/27 form a heterochiral anti-parallel β-sheet between strand Gly84 to Trp88/His88 in DAM21/27 of their FG loop with the βD-strand (Val190 to Arg194) of the d-Abl SH2 domain (Fig. 4d, e). This interaction corresponds to a “rippled β-sheet”, which is a largely neglected structural motif predicted by Pauling and Corey in 1953 and meanwhile experimentally shown to be present in heterochiral peptide structures47. In contrast to the canonical “pleated” anti-parallel β-sheets of natural proteins, the side chains experience in rippled β-sheets much less steric crowding. Accordingly, in the DAM21/27 structures, Glu85, the key residue binding to the pY pocket, projects its sidechain in between the SH2 residues His192 and Arg194 in the observed heterochiral rippled β-sheet, whereas it would sterically clash with these residues in a pleated β-sheet, due to the different register of β-sheet hydrogen bonds (Fig. 4d, e).The rippled β-sheet position Glu85 in the FG loop is inserted into the SH2 pY pocket, where it forms an ionic interaction with SH2 residue Arg171, which is part of the conserved FLVRES motif and makes the most critical interaction with the phosphate group in pY peptide ligands (Fig. 4f, g)48. In contrast to DAM21/27, most other SH2-targeting l-monobodies that engaged the pY pocket rather used tyrosine residues in the FG loop and often a phosphate or sulfate from the crystallization buffer to mimic pY10,14. In DAM21, FG loop residue Asp83 forms an additional ionic interaction with Arg194, unique in the Abl1 SH2 domain and therefore a possible contributor to selective Abl1 binding (Fig. 4g). In DAM27, this residue is replaced with a serine. To probe the functional roles of Asp83 and Glu85, we produced a D83A/E85A mutant in DAM21. The mutant protein displayed an almost identical far-UV circular dichroism spectrum, and thus similar secondary structure content, as wildtype DAM21 (Fig. S8, Table S4). ITC measurements showed no detectable binding (Fig. 4h), thus validating the critical role of these residues for SH2 domain binding and interface 2 as the biologically relevant protein interaction site. This is noteworthy, as glutamate is not a suitable pY mimetic in SH2 ligand peptides.An additional ionic interaction was observed in the DAM21/27-SH2 interface: SH2 residue Arg189 interacts with Glu51 in the ßD strand of the monobody and is located at the other end of the heterochiral SH2-monobody interface (Fig. 4f, g). Interestingly, in the monobody side-and-loop library, this position in the ßD strand is a mixture of Ala, Glu, Lys, Thr, of which only Glu is suitable to interact with Arg189. Taken together, the DAM21/27 interfaces are much more prominent in ionic interactions when compared to other (homochiral) monobody-SH2 interfaces.Structural superposition of the FG loop conformations of DAM27 and DAM21 revealed a strikingly similar conformation despite their sequence differences in seven out of eleven amino acid positions (Fig. 4i). This can be explained by a pairwise compensatory exchange of bulky and less bulky hydrophobic amino acids. This is particularly notable at the first and second to last position. In DAM27, Tyr79 packs towards the β-hairpin and His88 is solvent exposed, whereas in DAM21, Trp88 is rotated inwards and would clash with large hydrophobic or aromatic residues at this position. Hence, Tyr79 is exchanged to Ser79 in DAM21 (Fig. 4i).In summary, monobody recognition is dominated by the FG loop of both monobodies that form a rippled β-sheet with the βD-strand of the SH2 domain, as well as ionic interactions between glutamate residues of the monobody and arginine residues of the SH2 domain.Synthesis of d-monobody DAM27Since the monobody scaffold lacks endogenous cysteines, we planned a two-segment synthesis for DAM27 by SPPS with Thr60-Ala61 as the ligation junction through NCL and subsequent desulfurization resulting in the N- and C-terminal monobody fragments DAM27(4-60) and DAM27(61-98), respectively, spanning the entire length of the FN3 domain (Fig. 5a). In the C-terminal peptide, the N-terminal alanine was mutated to cysteine as required for NCL. Synthesis was first optimized in l-configuration due to lower reagent costs and subsequently applied for d-monobody synthesis. Both DAM27 N- and C-peptides could be synthesized without major need for optimization and obtained with high purity after HPLC purification (Table S5, Figs. S9–S12). We used a similar strategy to prepare the two peptides for NCL, as previously optimized for the synthesis of the d-Abl SH2 domain including the usage of the 3-amino-4-(methylamino)benzoic acid (MeDbz) linker and N-terminal capping strategy34. However, the product after NCL was insoluble and hence could not be analyzed and purified.Fig. 5DAM21 and DAM27 synthesis, refolding and purification.a Strategy of DAM27 synthesis. The N- and C-terminal fragments are shown in orange and green, respectively, and synthesized via solid-phase peptide synthesis (SPPS). The N-terminal peptide corresponds to the biotinylated DAM27(4-60) fragment with C-terminal MeDbz linker, which is activated to MeNbz on resin, and the C-terminal peptide resembles DAM27(62-98) with an N-terminal cysteine (Cys61) residue. After cleavage from the resin, both peptides can undergo native chemical ligation (NCL) with subsequent desulfurization to yield full-length monobody DAM27(4-98). MeDbz: 3-amino-4-(methylamino)benzoic acid; MeNbz: N-acyl-N-methylacylurea. b Size exclusion chromatography (SEC) and (c) high-performance liquid chromatography-mass spectrometry (HPLC-MS) analysis of the final refolded l- and d-DAM27 proteins in comparison with the recombinantly expressed l-DAM27-XTEN reveals similar SEC retention volumes and expected masses of the synthetic proteins. d Strategy of DAM21 synthesis. The N- and C-terminal peptides in orange and green obtained by SPPS represent the biotinylated DAM21(4-45) and DAM21(46-98) fragments, respectively. After mixing of the peptides and refolding by dialysis, the full-length split-monobody DAM21(4-98) is obtained. e SEC of the final refolded l- and d-DAM21 proteins in comparison with the recombinantly expressed l-DAM21 reveals similar SEC retention volumes of the synthetic proteins. f, g HPLC-MS analysis of the (f) DAM21(4-45) and (g) DAM21(46-98) fragments.To improve the solubility of monobody fragments, we included three repeats of the highly soluble XTEN peptide (six amino acids) at the C-terminus of the C-terminal DAM27 peptide49. Gratifyingly, the product after NCL was soluble and obtained with ~20% yield after HPLC purification in both l- and d-configuration (Figs. S13, S14). Subsequent desulfurization using our optimized protocol34 was not successful for the DAM27 ligation product, but an alternative protocol using 2-mercaptoethane sulfonate (MESNa) instead of glutathione resulted in quantitative conversion of cysteine to alanine giving rise to the native DAM27 monobody sequence (Figs. S15, S16). Refolding of the DAM27 protein was achieved by dialysis into aqueous buffer directly using the desulfurization reaction without a need of HPLC purification due to complete conversion (Figs. S15, S16). The protein was purified by SEC to remove aggregates and to determine oligomerization state. DAM27 both in l- and d-configuration eluted in a predominantly monomeric peak in line with the recombinantly expressed DAM27 control (Fig. 5b, c) and overall yield was ~1%. In summary, a facile synthesis strategy for the DAM27 monobody from two peptide segments could be established, which supplied sufficient protein for further biophysical and functional characterization.Synthesis of the split-d-monobody DAM21After establishing the conventional synthesis of DAM27 from two pieces including NCL, we aimed at testing a more rapid and versatile approach to obtain d-monobodies without the need to optimize synthesis conditions. Inspired by early work that showed complementation of a functional monobody by split fragments in a yeast-surface-two-hybrid system50, we attempted to obtain a functional d-monobody by mixing N- and C-terminal fragments without NCL. As observed in the DAM21 crystal structure, the monobody CD loop is not involved in Abl SH2 binding. Therefore, we decided to split the sequence between Asn45 and Ser46 in the CD loop generating the fragments DAM21(4-45) and DAM21(46-98) (Fig. 5d). Luckily, these fragments were previously shown to be the most efficient in complementing a functional l-monobody50. Both DAM21 N- and C-peptides in l- and d-configuration could be synthesized without need for major optimization (Table S5, Figs. S17–S20). Although the C-terminal peptide was initially aggregating during HPLC purification, this peptide could be obtained employing an HPLC-free purification strategy, in which the desired peptide is linked to agarose beads and acetylated, truncated peptides are washed off, as they are not reacting with the beads (Figs. S19, S20). Both peptides were then mixed in a 1:1 molar ratio and refolded by dialysis. SEC purification showed a predominantly monomeric protein, but in contrast to DAM27 an appreciable peak close to the void volume was observed indicating formation of some higher-order oligomers or aggregates during refolding (Fig. 5e). In summary, we were able to generate a monomeric split version of DAM21 from two peptide fragments (Fig. 5f, g) for further biophysical and functional characterization.Folding and stability of synthetic DAM21 and DAM27We next analyzed purity, folding and stability of our synthetic l- and d-DAM21/27. Gel electrophoresis analysis revealed that the synthetic proteins are migrating at the expected molecular weight without detectable impurities (Fig. 6a, b). Of note, recombinant l-DAM27-XTEN migrated slightly higher than synthetic l- and d-DAM27-XTEN, because the recombinant construct carries four additional N-terminal amino acids from cloning and both synthetic constructs contain biotin. For small proteins, such as monobodies, these seemingly minor differences can result in a detectable difference in migration distance (Fig. 6b). Far-UV circular dichroism spectroscopy showed very similar spectra in terms of shape, signal strength and depth of minima for recombinant DAM27 and synthetic l-DAM27 both containing the XTEN extension (Fig. 6c, Table S6), in line with a pure β-sheet protein. The lower mean residue ellipticity (MRE) below 210 nm as compared to recombinant DAM27 lacking XTEN is in line with the XTEN peptide being unstructured. The synthetic d-DAM27 showed a mirrored spectrum at the x-axis (Fig. 6c). Secondary structure predictions with BeStSel51 revealed almost identical content of β-sheets and β-turns, and absence of α-helices, between recombinant and synthetic l-/d-DAM27 versions (Table S6). For the split-l-DAM21 monobody, a spectrum in line with a β-sheet protein was obtained although with some spectral differences to the recombinant (non-split) DAM21 below 215 nm (Fig. 6d, Table S7). As for split-d-DAM21 the spectrum is mirrored along the x-axis (Fig. 6d). Despite these spectral differences, secondary structure predictions still showed a similar secondary structure content especially between recombinant l-DAM21 and split-d-DAM21 (Table S7). Next, we measured thermal stability by nano differential scanning fluorimetry (nanoDSF). For DAM27 identical melting temperatures around 75 °C were observed for the synthetic l- and d-versions and its recombinant analogue (Fig. 6e, Fig. S21a, c). The presence of the XTEN peptide resulted in a mild stabilization by ~3 °C (Figs. 6e, S21a, c). Interestingly, unfolding was fully reversible for all proteins (Fig. S21b, d). As expected for a split protein, synthetic split-l- and d-DAM21 showed a lower melting temperature (Tm) around 45 °C as compared to the recombinant (non-split) version (~78 °C) (Figs. 6e,  S21e, g). Despite this, thermal unfolding was reversible as for DAM27 (Fig. S21f, h). In summary, both synthetic strategies resulted in folded monobodies.Fig. 6Folding and stability characterization of synthetic l- and d-monobodies.a, b Representative analysis of purity of synthetic (a) l- and d-DAM21 and (b) l- and d-DAM27 by sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) (performed at least twice). c, d Averaged far-UV circular dichroism (CD) spectra from three independent measurements of synthetic (c) l- and d-DAM27 and (d) l- and d-DAM21 in comparison with their recombinantly expressed analogues. Mean residue ellipticity (MRE) was calculated from three independent measurements. e Bar graph representation of thermal stability of recombinantly expressed and synthetic l- and d-monobodies assessed by nano differential scanning fluorimetry (nanoDSF). Melting temperatures (Tm) were measured at least three times (depicted as dots) and averaged. Error bars represent the standard deviation (SD). Source data of (a, b and e) are provided as a Source Data file.Protease and plasma stability of DAM27We next assessed protease stability of DAM27 by incubation with two different broad-spectrum proteases, pepsin and proteinase K, and analyzed the digests after 4 and 24 h of incubation. Both l-versions of the recombinant and synthetic DAM27 showed complete degradation after 24 h. Minor residual protein was detected after 4 h of pepsin digestion (Fig. 7a, b). In contrast, d-DAM27 was completely resistant to degradation by both proteases. Sufficient plasma stability is a crucial prerequisite for the envisioned use of d-monobodies as protein therapeutics. After incubation of l-DAM27 in mouse plasma for up to 96 h, we observed a strong reduction of l-DAM27 signal at 24 h and all later time points. In contrast, d-DAM27 showed no signal reduction for up to 96 h (Fig. 7c). These results show very high protease resistance and plasma stability of d-monobodies.Fig. 7Protease resistance and mouse plasma stability of synthetic l- and d-monobodies.a, b Protease resistance of recombinant l-DAM27-XTEN as well as synthetic l- and d-DAM27-XTEN after incubation with (a) pepsin and (b) proteinase K was analyzed by sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE). c Plasma stability of synthetic l- and d-DAM27-XTEN. Monobodies labeled with biotin at the N-terminus after incubation with mouse plasma were analyzed by Western blotting and detection of biotin using Streptavidin-IRDye680. A representative blot of three repeats is shown. Source data of (a–c) are provided as a Source Data file.d-DAM21 and d-DAM27 bind to native l-Bcr-Abl SH2After confirming folding and stability of the d-monobodies, target binding was measured by ITC. For DAM27 binding affinities between 100 and 128 nM were measured for the interaction of the d-monobody with the l-target and vice versa (Fig. 8a–c). Of note, in the absence of the XTEN peptide a slightly higher affinity was measured (Fig. 2h, j). For DAM21 both split-l- and d-versions showed an interaction of around 100 nM with d- and l-Bcr-Abl SH2, respectively (Fig. 8d, e). While this affinity is in the same range of the d-DAM27:l-Bcr-Abl SH2 interaction, it is around 10-fold lower than the recombinant (non-split) DAM21 with d-Abl SH2 (Fig. 2g, i). All measurements showed a 1:1 binding stoichiometry and negative binding enthalpies (Fig. 8a–e). As expected, no binding was detected for l-DAM21 and l-DAM27, respectively, to the l-Bcr-Abl SH2 domain (Fig. S22).Fig. 8Binding of synthetic l- and d-monobodies to d- and l-Bcr-Abl SH2.a–e Isothermal titration calorimetry (ITC) measurements of (a) synthetic d-DAM27-XTEN titrated to recombinantly expressed l-Abl SH2, (b) synthetic l-DAM27-XTEN and (c) recombinant l-DAM27-XTEN both titrated to synthetic d-Abl SH2, (d) synthetic split-l-DAM21 and (e) synthetic split-d-DAM21 titrated to synthetic d- and l-Abl SH2, respectively. Each panel shows the raw heat signal of an ITC experiment (top) and the integrated calorimetric data of the area of each peak (bottom). The continuous line represents the best fit of the data based on a 1:1 binding model computed from the MicroCal software. Binding parameters including Kd value, stoichiometry (N), enthalpy (∆H), free enthalpy (∆G) and −T∆S calculated from the fit of each experiment are shown below. A representative measurement of at least two ITC experiments for each monobody is shown. f, g Measurement of kinase activity of (f) Bcr-Abl kinase domain (KD) and (g) SH2-KD after incubation with synthetic split-l- and d-DAM21 as well as synthetic l- and d-DAM27-XTEN in comparison with binding control monobodies HA4 and AS25 and the non-binding control monobody HA4 Y87A using a radiometric kinase assay. All monobodies were used at a concentration of 5 µM. Here, radioactively labeled 32P was incorporated into a biotinylated substrate peptide by recombinantly expressed KD and SH2-KD and detected via scintillation counting. Six independent experiments were performed (depicted as dots) and averaged. Error bars represent the standard deviation (SD) and statistical analysis was done with a one-way ANOVA and Sidak’s test. The calculated p-values are depicted in (g) and were considered statistically significant below a value of 0.05. F values and degrees of freedom were 124.3 and 40. Source data of (f, g) are provided as a Source Data file.Inhibition of Bcr-Abl kinase activity by d-monobodiesAfter showing binding to the Bcr-Abl SH2 domain with high affinity, we next wanted to assess a possible inhibitory effect of monobodies on Bcr-Abl kinase activity in vitro. We recombinantly expressed and purified BCR::ABL1 fragments either containing the kinase domain (KD) alone or a larger fragment also containing the SH2 domain (SH2-KD)52. After addition of different monobody proteins, kinase activity was measured using a radiometric kinase assay13. Besides the l- and d-versions of DAM21 and DAM27, we included the previously characterized Abl SH2 binding monobodies HA4 and AS25 as well as the non-binding control monobody HA4 Y87A10,13. In vitro kinase activity of Abl1 KD (not including the Abl1 SH2 domain) remained unchanged upon incubation with all monobodies indicating no unspecific inhibitory or activating effect of the monobodies on Abl1 kinase activity (Fig. 8f). In contrast, d-DAM27 resulted in a major reduction of kinase activity of SH2-KD, comparable to inhibition by the validated allosteric Bcr-Abl inhibiting monobody AS25. In contrast, no inhibition was observed with l-DAM27. Also, no inhibition was observed with HA4 and HA4 Y87A, in line with previous observations (Fig. 8g)10. Although we noted a minor and unexplained increase in kinase activity by split-monobody l-DAM21, d-DAM21 showed significant inhibition as compared to its l-counterpart, although to a lesser extent than d-DAM27 (Fig. 8g). These data show that d-monobodies binding to the Abl1 SH2 domain are able to inhibit the kinase activity of Bcr-Abl, which is crucial for BCR::ABL1 signaling and CML maintenance.High selectivity of d-monobodies for the BCR::ABL1 SH2 domainNext, we studied selectivity of the d-monobodies for the BCR::ABL1 SH2 domain. Analysis of the DAM21/27 crystal structures (see Fig. 4a, b) indicated that the CD loop of the Abl SH2 domain is positioned close to the βC and βD strands of DAM27 and DAM21 (Fig. 9a). The Abl1 (and Abl2) SH2 domain has a particularly short CD loop, which seems to be a requirement for binding to DAM21 and DAM27 (Fig. 9a). In contrast, all SH2s of the Src- and Tec-family (9 and 5 members, respectively), the closest paralogues of the Abl family, have CD loops that are four to six amino acids longer. Molecular modeling indicates that these longer CD loops would clash with the βC/βD strands of DAM21 and DAM27 (Fig. 9b, c). Also, several other SH2 families including the Jak kinases, STAT transcription factors and SHP1/2 (C-SH2) tyrosine phosphatases have such long CD loops (Table S8). In addition, the ionic interaction of Glu51 and Arg189 (Fig. 4f, g), can only be formed with Abl kinase SH2 domains, as many other SH2 domains, including the Src kinase SH2s, do not have Arg (or Lys) in this position (Fig. 9c, Table S8). To assess this predicted selectivity experimentally, ITC measurements with the SH2 domains of Bruton’s tyrosine kinase (Btk) and lymphocyte-specific protein tyrosine kinase (Lck), as representatives of the closely related Tec- and Src-family showed that both d-DAM27 and d-DAM21 did not bind to these SH2 domains (Fig. 9d–g). Overall, these observations indicate a high selectivity of DAM21 and DAM27 for the Abl SH2 domain.Fig. 9Selectivity of d-DAM21/27 for the BCR::ABL1 SH2 domain.a
l-DAM27:d-Abl SH2 complex structure (PDB code: 9F00) highlighting the short Abl SH2 CD loop that enables binding to DAM27. The space where longer CD loops in other SH2 domains would be located is indicated by the dotted red circle. b Structural superposition of the Abl, Lck and Btk SH2 domains (PDB entries 3K2M, 1LKK and 6HTF, respectively). The additional four to six amino acid residues in the Lck and Btk SH2 CD loops are not compatible with binding to the DAM21 and DAM27 monobodies. c Multiple sequence alignment of the βC and βD strands of SH2 domains belonging to different SH2 domain-containing tyrosine kinase families. The βC and βD strands are colored in red and green, respectively, while the CD loop in between is black. Arg189 of ABL1, which is important for d-monobody binding via ionic interactions, is not conserved and colored in bold green to highlight the different amino acid sequences in this position. d, e Isothermal titration calorimetry (ITC) measurements of d-DAM27-XTEN titrated to the SH2 domains of (d) Btk and (e) Lck. f, g ITC measurements of split-d-DAM21 titrated to the SH2 domains of (f) Btk and (g) Lck. Each panel shows the raw heat signal of an ITC experiment (top) and the integrated calorimetric data of the area of each peak (bottom). h, i Volcano plots of identified proteins via mass spectrometry after pulldown from K562 cell lysates comparing synthetic (h) split-d-DAM21 with split-l-DAM21 and (i) d-DAM27-XTEN with l-DAM27-XTEN. Pulldowns were performed in three biological replicates for each monobody. Protein identification and statistical analysis of replicates were done with MaxQuant 2.5.1.0 and Autonomics (R package version 1.13.21)84 resulting in FDR-corrected p-values (Benjamini-Hochberg procedure) for each identified protein where a p-value below 0.05, which corresponds to -log10(p) above 1.3, was considered statistically significant (dotted line intersecting y-axis). A log2(ratio) above 1.00 of d- vs. l-monobody correlates to a ratio above 2.00 and a higher abundance of the protein when the pulldown was performed with the d-monobody compared to its l-counterpart (dotted lines intersecting x-axis). The color coding represents the protein groups BCR and ABL1 (light red), BCR::ABL1/ABL1 interactors (blue), BCR::ABL1/ABL1 interactors with SH2 domains (green) and other proteins containing SH2 domains (purple). Plotted values of all highlighted protein groups are listed in Tables S9, S10.Binding of d-monobodies to Bcr-Abl in cell lysates and permeabilized CML cellsIn order to determine if d-monobodies can bind to full-length BCR::ABL1 in the context of the complex proteome of human cells, we performed pulldown experiments with the biotinylated l- and d-DAM21/27 variants in three biological repeats from lysates of the BCR::ABL1-expressing cell line K562, one of the most commonly used cell lines in CML research, followed by quantitative proteomics analysis of bound proteins. This approach also enables to complement the findings on d-monobody selectivity described above and to determine its interactome. Comparison of d- vs. l-DAM21 showed higher abundance of BCR and ABL1 in the samples from the d-DAM21 pulldown when compared to its respective l-counterpart (Fig. 9h, Table S9). Comparing the pulldowns of d- vs. l-DAM27 showed higher abundance of BCR, but only a mild enrichment of ABL1, which was not significant (Fig. 9i, Table S10). Since BCR::ABL1 is a fusion oncoprotein, enrichment of either BCR or ABL1 is sufficient to show binding of d-DAM27 to BCR::ABL1. Interestingly, we also found several previously identified and validated BCR::ABL1 interactors, including kinases (CDK1, MTOR, PRKDC)53–55, proteins involved in transcription regulation, splicing and cell proliferation control (STAT1, FUS, GTF3C4, YTHDC1, EPS15)56–61, proteins mediating proteasomal degradation (CUL4B, TRIM25, PSMA4)62–64, transport proteins and chaperones (AP2A1, XPO1, HSPD1)61,65–67, as well as cytoskeletal components (EMD, SPTA1)68,69, significantly enriched with the two d-monobodies (Fig. 9h, i, Tables S9, S10). In line with previous observations with other Abl SH2-targeting monobodies10, it is likely that these proteins piggyback on BCR::ABL1 and are therefore not direct interactors of the monobodies. Importantly, no other SH2 domain-containing proteins, such as BTK, CSK, SH2B1 and TYK2, were significantly enriched with the d-monobody variants, despite the expression of ~70 SH2 domain-containing proteins in K562 cells, and thus confirming the ITC measurements with the Btk SH2 domain resulting in no binding (Fig. 9d, f). As expected for single-step pulldown mass spectrometry experiments, the dataset contains a lot of common background contaminants, which are listed in the Contaminant Repository for Affinity Purification (CRAPome)70 and were therefore disregarded. Together, the proteomics data show preferential binding of d-DAM21 and d-DAM27 to BCR::ABL1 complexes over their l-counterparts and that both d-monobodies are able to bind BCR::ABL1, but no other SH2-containing proteins in the complex proteome of a human cell.In addition, we used a FACS-based assay, in which we permeabilized K562 cells and monitored the binding of the d-monobodies by a shift in fluorescence signal. Without optimization of assay parameters, binding of d-DAM21 and d-DAM27 was readily observed to be much stronger than the binding of synthetic l-DAM21 and l-DAM27 and exceeded binding of the allosteric BCR::ABL1 binder/inhibitor AS25 (Fig. S23). No binding was observed for the negative control monobody HA4 Y87A (Fig. S23).These two assays show binding of the two d-monobodies to full-length BCR::ABL1 in cell lysates and permeabilized human BCR::ABL1-expressing cells.

Generation of l-monobodies targeting the d-Abl SH2 domainWe first produced the d-version of the human BCR::ABL1 SH2 domain (Abl SH2) by native chemical ligation (NCL) of two d-peptide fragments obtained by solid-phase peptide synthesis (SPPS). The cysteine residue required for NCL was subsequently desulfurized to an alanine residue to yield the wildtype sequence (Fig. 1)34. As previously described, the protein was monomeric in size exclusion chromatography (SEC) after refolding, showed a similar secondary structure composition and thermal stability as the recombinant Abl SH2 domain, and bound a d-phospho tyrosine (pY) peptide with the same affinity as the l-counterpart to the corresponding l-pY peptide34.We selected monobodies from the combinatorial “side-and-loop” phage-display library and subsequent yeast display screening (Supplementary information, section 2)7. Monobody pools enriched with high-affinity binders were identified after four rounds of yeast display with decreasing target concentrations. Six monobody clones with unique sequences were isolated and further characterized (Fig. 2a). Three clones had a diversified CD loop, whereas the others had a CD loop corresponding to the wildtype FN3 sequence7. The FG loop, which was identical for four out of the six clones, differed in sequence, but not in length. All clones bound to the d-Abl SH2 domain with similar affinities between 20 and 71 nM, as measured in the yeast display format13, with clones 21 and 27 (hereafter termed DAM21 and DAM27) showing tightest apparent binding (Fig. 2b). We next expressed and purified the recombinant monobodies in E. coli with high yields (44–54 mg/L E. coli culture, Fig. 2c, d). Preparative SEC showed high purity and that the elution volumes of all monobodies corresponded to their calculated monomeric molecular weights, except for DAM28, which eluted at a higher molecular weight and was therefore not selected for further follow-up (Fig. 2e). These monobodies robustly blocked binding of a d-pY peptide to the d-Abl SH2 domain (Kd = ~5 µM, Fig. S1) as tested with a fluorescence polarization (FP) binding assay. All monobody clones produced a strong FP signal decrease, which was comparable to l-pY competition by the previously characterized l-Abl SH2-targeting monobody HA410. In contrast, monobody AS25 that targets the allosteric SH2-kinase domain interface of ABL1 did not show competition of the l-pY-SH2 interaction (Fig. 2f)13. These results suggest that the DAM monobodies bind to the pY binding site of the SH2 domain. To determine the thermodynamic binding parameters of the monobodies more precisely, we performed isothermal titration calorimetry (ITC) measurements using purified monobody and SH2 proteins. The determined binding affinities in the low to mid nanomolar range were generally consistent with the yeast display measurements (Figs. 2g–j, S2). All measurements suggested a d-Abl SH2:l-monobody binding stoichiometry of 1:1 and binding enthalpies in the range of −75 to −36 kJ/mol in line with the previously observed strongly enthalpically driven binding of l-monobodies to their l-targets. Given that all monobody clones were pY competitors we focused on DAM21 and DAM27 for further characterization, as they had the highest apparent binding affinities.Fig. 2l-monobody selection and characterization against d-Bcr-Abl SH2.a Amino acid sequences of d-Abl SH2 l-monobody binders (DAM) generated by phage and yeast display selection based on the combinatorial ‘side-and-loop’ library. In the library designs, “X” denotes a mixture of 30% Tyr, 15% Ser, 10% Gly, 5% Phe, 5% Trp, and 2.5% each of all the other amino acids except for Cys; “O” denotes a mixture of Asn, Asp, His, Ile, Leu, Phe, Tyr, and Val; “U” denotes a mixture of His, Leu, Phe, and Tyr; and “Z” denotes a mixture of Ala, Glu, Lys, and Thr. A hyphen indicates a deletion. b Binding titrations in the yeast surface display format to estimate binding affinities of l-monobodies to d-Abl SH2. The mean fluorescence intensities of yeast cells displaying a monobody are plotted as a function of the concentration of the target and fitted to a 1:1 binding model. c Sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) analysis (performed once), (d) retention volumes in size exclusion chromatography (SEC) purification and expression yields, and (e) SEC chromatograms of selected l-DAMs recombinantly expressed in E. coli. (f) Competitive fluorescence polarization (FP) assay of l-DAMs incubated with d-Abl SH2 and fluorescently labeled d-pYEEI peptide binders in comparison with previously published l-monobodies HA4 (pY peptide competitor) and AS25 (allosteric binder) against l-Abl SH2 incubated with the corresponding l-pYEEI peptide. Measured data from two independent experiments (depicted as dots) were averaged. g–j Isothermal titration calorimetry (ITC) measurements of recombinant l-monobodies (g) DAM21 and (h) DAM27 titrated to the synthetic d-Abl SH2 domain. Each panel shows the raw heat signal of an ITC experiment (top) and the integrated calorimetric data of the area of each peak (bottom). The continuous line represents the best fit of the data based on a 1:1 binding model computed from the MicroCal software. Binding parameters including Kd value, stoichiometry (N), enthalpy (∆H), free enthalpy (∆G) and −T∆S calculated from the fit of each experiment are shown in (i) and (j). A representative measurement of at least two ITC experiments for each monobody is shown. Source data of (a, c and f) are provided as a Source Data file.

Generation of l-monobodies targeting the d-Abl SH2 domain

We first produced the d-version of the human BCR::ABL1 SH2 domain (Abl SH2) by native chemical ligation (NCL) of two d-peptide fragments obtained by solid-phase peptide synthesis (SPPS). The cysteine residue required for NCL was subsequently desulfurized to an alanine residue to yield the wildtype sequence (Fig. 1)34. As previously described, the protein was monomeric in size exclusion chromatography (SEC) after refolding, showed a similar secondary structure composition and thermal stability as the recombinant Abl SH2 domain, and bound a d-phospho tyrosine (pY) peptide with the same affinity as the l-counterpart to the corresponding l-pY peptide34.

We selected monobodies from the combinatorial “side-and-loop” phage-display library and subsequent yeast display screening (Supplementary information, section 2)7. Monobody pools enriched with high-affinity binders were identified after four rounds of yeast display with decreasing target concentrations. Six monobody clones with unique sequences were isolated and further characterized (Fig. 2a). Three clones had a diversified CD loop, whereas the others had a CD loop corresponding to the wildtype FN3 sequence7. The FG loop, which was identical for four out of the six clones, differed in sequence, but not in length. All clones bound to the d-Abl SH2 domain with similar affinities between 20 and 71 nM, as measured in the yeast display format13, with clones 21 and 27 (hereafter termed DAM21 and DAM27) showing tightest apparent binding (Fig. 2b). We next expressed and purified the recombinant monobodies in E. coli with high yields (44–54 mg/L E. coli culture, Fig. 2c, d). Preparative SEC showed high purity and that the elution volumes of all monobodies corresponded to their calculated monomeric molecular weights, except for DAM28, which eluted at a higher molecular weight and was therefore not selected for further follow-up (Fig. 2e). These monobodies robustly blocked binding of a d-pY peptide to the d-Abl SH2 domain (Kd = ~5 µM, Fig. S1) as tested with a fluorescence polarization (FP) binding assay. All monobody clones produced a strong FP signal decrease, which was comparable to l-pY competition by the previously characterized l-Abl SH2-targeting monobody HA410. In contrast, monobody AS25 that targets the allosteric SH2-kinase domain interface of ABL1 did not show competition of the l-pY-SH2 interaction (Fig. 2f)13. These results suggest that the DAM monobodies bind to the pY binding site of the SH2 domain. To determine the thermodynamic binding parameters of the monobodies more precisely, we performed isothermal titration calorimetry (ITC) measurements using purified monobody and SH2 proteins. The determined binding affinities in the low to mid nanomolar range were generally consistent with the yeast display measurements (Figs. 2g–j, S2). All measurements suggested a d-Abl SH2:l-monobody binding stoichiometry of 1:1 and binding enthalpies in the range of −75 to −36 kJ/mol in line with the previously observed strongly enthalpically driven binding of l-monobodies to their l-targets. Given that all monobody clones were pY competitors we focused on DAM21 and DAM27 for further characterization, as they had the highest apparent binding affinities.Fig. 2l-monobody selection and characterization against d-Bcr-Abl SH2.a Amino acid sequences of d-Abl SH2 l-monobody binders (DAM) generated by phage and yeast display selection based on the combinatorial ‘side-and-loop’ library. In the library designs, “X” denotes a mixture of 30% Tyr, 15% Ser, 10% Gly, 5% Phe, 5% Trp, and 2.5% each of all the other amino acids except for Cys; “O” denotes a mixture of Asn, Asp, His, Ile, Leu, Phe, Tyr, and Val; “U” denotes a mixture of His, Leu, Phe, and Tyr; and “Z” denotes a mixture of Ala, Glu, Lys, and Thr. A hyphen indicates a deletion. b Binding titrations in the yeast surface display format to estimate binding affinities of l-monobodies to d-Abl SH2. The mean fluorescence intensities of yeast cells displaying a monobody are plotted as a function of the concentration of the target and fitted to a 1:1 binding model. c Sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) analysis (performed once), (d) retention volumes in size exclusion chromatography (SEC) purification and expression yields, and (e) SEC chromatograms of selected l-DAMs recombinantly expressed in E. coli. (f) Competitive fluorescence polarization (FP) assay of l-DAMs incubated with d-Abl SH2 and fluorescently labeled d-pYEEI peptide binders in comparison with previously published l-monobodies HA4 (pY peptide competitor) and AS25 (allosteric binder) against l-Abl SH2 incubated with the corresponding l-pYEEI peptide. Measured data from two independent experiments (depicted as dots) were averaged. g–j Isothermal titration calorimetry (ITC) measurements of recombinant l-monobodies (g) DAM21 and (h) DAM27 titrated to the synthetic d-Abl SH2 domain. Each panel shows the raw heat signal of an ITC experiment (top) and the integrated calorimetric data of the area of each peak (bottom). The continuous line represents the best fit of the data based on a 1:1 binding model computed from the MicroCal software. Binding parameters including Kd value, stoichiometry (N), enthalpy (∆H), free enthalpy (∆G) and −T∆S calculated from the fit of each experiment are shown in (i) and (j). A representative measurement of at least two ITC experiments for each monobody is shown. Source data of (a, c and f) are provided as a Source Data file.

l-monobody selection and characterization against d-Bcr-Abl SH2.

a Amino acid sequences of d-Abl SH2 l-monobody binders (DAM) generated by phage and yeast display selection based on the combinatorial ‘side-and-loop’ library. In the library designs, “X” denotes a mixture of 30% Tyr, 15% Ser, 10% Gly, 5% Phe, 5% Trp, and 2.5% each of all the other amino acids except for Cys; “O” denotes a mixture of Asn, Asp, His, Ile, Leu, Phe, Tyr, and Val; “U” denotes a mixture of His, Leu, Phe, and Tyr; and “Z” denotes a mixture of Ala, Glu, Lys, and Thr. A hyphen indicates a deletion. b Binding titrations in the yeast surface display format to estimate binding affinities of l-monobodies to d-Abl SH2. The mean fluorescence intensities of yeast cells displaying a monobody are plotted as a function of the concentration of the target and fitted to a 1:1 binding model. c Sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) analysis (performed once), (d) retention volumes in size exclusion chromatography (SEC) purification and expression yields, and (e) SEC chromatograms of selected l-DAMs recombinantly expressed in E. coli. (f) Competitive fluorescence polarization (FP) assay of l-DAMs incubated with d-Abl SH2 and fluorescently labeled d-pYEEI peptide binders in comparison with previously published l-monobodies HA4 (pY peptide competitor) and AS25 (allosteric binder) against l-Abl SH2 incubated with the corresponding l-pYEEI peptide. Measured data from two independent experiments (depicted as dots) were averaged. g–j Isothermal titration calorimetry (ITC) measurements of recombinant l-monobodies (g) DAM21 and (h) DAM27 titrated to the synthetic d-Abl SH2 domain. Each panel shows the raw heat signal of an ITC experiment (top) and the integrated calorimetric data of the area of each peak (bottom). The continuous line represents the best fit of the data based on a 1:1 binding model computed from the MicroCal software. Binding parameters including Kd value, stoichiometry (N), enthalpy (∆H), free enthalpy (∆G) and −T∆S calculated from the fit of each experiment are shown in (i) and (j). A representative measurement of at least two ITC experiments for each monobody is shown. Source data of (a, c and f) are provided as a Source Data file.

Structures of heterochiral Abl SH2-monobody complexesTo understand the structural basis for monobody recognition of the d-Abl SH2 domain, we set out to determine the crystal structures of monobody-SH2 complexes. We mixed the synthetic d-Abl SH2 domain with recombinantly expressed l-DAM21 or l-DAM27 monobody, respectively, and purified the heterochiral 1:1-complexes by SEC (Fig. 3a–e). Of note, to satisfy the large amounts of protein complex needed for crystallization, this batch of the d-Abl SH2 domain was not desulfurized to alanine after NCL and therefore contained a cysteine residue at position 198 (Table S3, Figs. S3–S5). The alanine to cysteine mutation at this position in recombinant l-Abl SH2 did not change binding affinity to l-monobody AS25 and l-pY peptide (Fig. S6). Crystals were obtained readily or after an additional additive screen. We determined the crystal structures of the l-DAM21-d-Abl SH2 and l-DAM27-d-Abl SH2 complexes at 2.7 Å and 2.9 Å resolution, respectively (Fig. 4a, b, Table 1, Fig. S7a, b). In both structures, we found the d-SH2 domain to be dimerized via a disulfide bridge of Cys198. In addition, we found two different interaction interfaces of the monobody with the SH2 domain, which were very similar in both complexes (Fig. S7c, d). The monobody FG loop, which represents the most extensively diversified region in the library, forms a large part of both interfaces, which have similar buried surface areas (431 Å2 and 440 Å2 in DAM21, 452 Å2 and 427 Å2 in DAM27, for interface 1 and 2, respectively). Interface 2 of both structures largely contains the pY peptide-binding interface (346 Å2 for the pYEEI peptide, Fig. 4c), whereas interface 1 comprises a region of the SH2 domain surface of unknown functional significance. Given the robust pY peptide competition of both monobodies (see Fig. 2f) and mutagenesis data described below, we considered interface 2 as the likely biologically relevant interface.Fig. 3Complex formation and structure elucidation of the heterochiral d-Abl SH2:l-monobody interaction.a Workflow for generation of the heterochiral d-Abl SH2:l-monobody complex. d-Abl SH2 was produced by solid-phase peptide synthesis (SPPS) and native chemical ligation (NCL) and the l-monobody by bacterial expression and subsequent purification. After mixing, the complex was purified by size exclusion chromatography (SEC). b, c Analytical SEC of complex formation between synthetic d-Abl SH2 and recombinantly expressed (b) l-DAM21 and (c) l-DAM27. d Preparative SEC of d-Abl SH2:l-monobody complexes generated for crystallization condition screening. All chromatograms include a calibration of the column with standard proteins on top. e Sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) analysis of the complex formation (performed once). Source data of (e) are provided as a Source Data file.Fig. 4Crystal structures of heterochiral monobody-Abl SH2 complexes.a, b Ribbon representation of the d-Abl SH2 domain structure (green) bound to the (a) DAM27 or (b) DAM21 monobodies in l-configuration. Arg171 of the SH2 FLVRES motif, which is critical for pY binding, as well as Glu85 in the FG loop of the monobodies is shown in stick representation. c Structure of an (l-) phosphotyrosine peptide (pYEEI, stick representation) bound to the (l-)SH2 domain of the Src family kinase Lck (PDB: 1LKK). The critical Arg residue of the FLVRES motif in the pY binding pocket is shown in stick representation. d, e Close-up view of the d-SH2:(l-)monobody interface (DAM27, panel d; DAM21, panel e) highlighting the heterochiral “rippled” anti-parallel β-sheet between strand Gly84 to Trp88/His88 in DAM21/27 of the monobody FG loop with Val190 to Arg194 of the βD-strand of the d-Abl SH2 domain. f, g Close-up view of the d-SH2:(l-)monobody interface (DAM27, panel f; DAM21, panel g). The ionic interactions discussed in the “Results” section are shown as sticks and are labeled. The position of randomized residues in the monobody side-and-loop library is shown as balls. h Isothermal titration calorimetry (ITC) measurements of recombinant l-DAM21 D83A/E85A mutant titrated to the synthetic d-Abl SH2 domain. The panel shows the raw heat signal of an ITC experiment (top) and the integrated calorimetric data of the area of each peak (bottom). i Structural superposition and sequence alignment of the FG loops of the DAM27 and DAM21 monobodies.Both monobodies use their long FG loops to present an extended segment that is positioned parallel to the central SH2 β-sheet and therefore perpendicular to the canonical backbone conformation of an SH2 domain-bound pY peptide (Fig. 4a–c). Likewise, several l-monobodies mimic pY peptide binding by also binding perpendicular across the central SH2 β-sheet, which thereby enables access to the pY and +3 pockets that are located on opposite sides of the SH2 β-sheet12,14. Unlike those, DAM21/27 form a heterochiral anti-parallel β-sheet between strand Gly84 to Trp88/His88 in DAM21/27 of their FG loop with the βD-strand (Val190 to Arg194) of the d-Abl SH2 domain (Fig. 4d, e). This interaction corresponds to a “rippled β-sheet”, which is a largely neglected structural motif predicted by Pauling and Corey in 1953 and meanwhile experimentally shown to be present in heterochiral peptide structures47. In contrast to the canonical “pleated” anti-parallel β-sheets of natural proteins, the side chains experience in rippled β-sheets much less steric crowding. Accordingly, in the DAM21/27 structures, Glu85, the key residue binding to the pY pocket, projects its sidechain in between the SH2 residues His192 and Arg194 in the observed heterochiral rippled β-sheet, whereas it would sterically clash with these residues in a pleated β-sheet, due to the different register of β-sheet hydrogen bonds (Fig. 4d, e).The rippled β-sheet position Glu85 in the FG loop is inserted into the SH2 pY pocket, where it forms an ionic interaction with SH2 residue Arg171, which is part of the conserved FLVRES motif and makes the most critical interaction with the phosphate group in pY peptide ligands (Fig. 4f, g)48. In contrast to DAM21/27, most other SH2-targeting l-monobodies that engaged the pY pocket rather used tyrosine residues in the FG loop and often a phosphate or sulfate from the crystallization buffer to mimic pY10,14. In DAM21, FG loop residue Asp83 forms an additional ionic interaction with Arg194, unique in the Abl1 SH2 domain and therefore a possible contributor to selective Abl1 binding (Fig. 4g). In DAM27, this residue is replaced with a serine. To probe the functional roles of Asp83 and Glu85, we produced a D83A/E85A mutant in DAM21. The mutant protein displayed an almost identical far-UV circular dichroism spectrum, and thus similar secondary structure content, as wildtype DAM21 (Fig. S8, Table S4). ITC measurements showed no detectable binding (Fig. 4h), thus validating the critical role of these residues for SH2 domain binding and interface 2 as the biologically relevant protein interaction site. This is noteworthy, as glutamate is not a suitable pY mimetic in SH2 ligand peptides.An additional ionic interaction was observed in the DAM21/27-SH2 interface: SH2 residue Arg189 interacts with Glu51 in the ßD strand of the monobody and is located at the other end of the heterochiral SH2-monobody interface (Fig. 4f, g). Interestingly, in the monobody side-and-loop library, this position in the ßD strand is a mixture of Ala, Glu, Lys, Thr, of which only Glu is suitable to interact with Arg189. Taken together, the DAM21/27 interfaces are much more prominent in ionic interactions when compared to other (homochiral) monobody-SH2 interfaces.Structural superposition of the FG loop conformations of DAM27 and DAM21 revealed a strikingly similar conformation despite their sequence differences in seven out of eleven amino acid positions (Fig. 4i). This can be explained by a pairwise compensatory exchange of bulky and less bulky hydrophobic amino acids. This is particularly notable at the first and second to last position. In DAM27, Tyr79 packs towards the β-hairpin and His88 is solvent exposed, whereas in DAM21, Trp88 is rotated inwards and would clash with large hydrophobic or aromatic residues at this position. Hence, Tyr79 is exchanged to Ser79 in DAM21 (Fig. 4i).In summary, monobody recognition is dominated by the FG loop of both monobodies that form a rippled β-sheet with the βD-strand of the SH2 domain, as well as ionic interactions between glutamate residues of the monobody and arginine residues of the SH2 domain.

Structures of heterochiral Abl SH2-monobody complexes

To understand the structural basis for monobody recognition of the d-Abl SH2 domain, we set out to determine the crystal structures of monobody-SH2 complexes. We mixed the synthetic d-Abl SH2 domain with recombinantly expressed l-DAM21 or l-DAM27 monobody, respectively, and purified the heterochiral 1:1-complexes by SEC (Fig. 3a–e). Of note, to satisfy the large amounts of protein complex needed for crystallization, this batch of the d-Abl SH2 domain was not desulfurized to alanine after NCL and therefore contained a cysteine residue at position 198 (Table S3, Figs. S3–S5). The alanine to cysteine mutation at this position in recombinant l-Abl SH2 did not change binding affinity to l-monobody AS25 and l-pY peptide (Fig. S6). Crystals were obtained readily or after an additional additive screen. We determined the crystal structures of the l-DAM21-d-Abl SH2 and l-DAM27-d-Abl SH2 complexes at 2.7 Å and 2.9 Å resolution, respectively (Fig. 4a, b, Table 1, Fig. S7a, b). In both structures, we found the d-SH2 domain to be dimerized via a disulfide bridge of Cys198. In addition, we found two different interaction interfaces of the monobody with the SH2 domain, which were very similar in both complexes (Fig. S7c, d). The monobody FG loop, which represents the most extensively diversified region in the library, forms a large part of both interfaces, which have similar buried surface areas (431 Å2 and 440 Å2 in DAM21, 452 Å2 and 427 Å2 in DAM27, for interface 1 and 2, respectively). Interface 2 of both structures largely contains the pY peptide-binding interface (346 Å2 for the pYEEI peptide, Fig. 4c), whereas interface 1 comprises a region of the SH2 domain surface of unknown functional significance. Given the robust pY peptide competition of both monobodies (see Fig. 2f) and mutagenesis data described below, we considered interface 2 as the likely biologically relevant interface.Fig. 3Complex formation and structure elucidation of the heterochiral d-Abl SH2:l-monobody interaction.a Workflow for generation of the heterochiral d-Abl SH2:l-monobody complex. d-Abl SH2 was produced by solid-phase peptide synthesis (SPPS) and native chemical ligation (NCL) and the l-monobody by bacterial expression and subsequent purification. After mixing, the complex was purified by size exclusion chromatography (SEC). b, c Analytical SEC of complex formation between synthetic d-Abl SH2 and recombinantly expressed (b) l-DAM21 and (c) l-DAM27. d Preparative SEC of d-Abl SH2:l-monobody complexes generated for crystallization condition screening. All chromatograms include a calibration of the column with standard proteins on top. e Sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) analysis of the complex formation (performed once). Source data of (e) are provided as a Source Data file.Fig. 4Crystal structures of heterochiral monobody-Abl SH2 complexes.a, b Ribbon representation of the d-Abl SH2 domain structure (green) bound to the (a) DAM27 or (b) DAM21 monobodies in l-configuration. Arg171 of the SH2 FLVRES motif, which is critical for pY binding, as well as Glu85 in the FG loop of the monobodies is shown in stick representation. c Structure of an (l-) phosphotyrosine peptide (pYEEI, stick representation) bound to the (l-)SH2 domain of the Src family kinase Lck (PDB: 1LKK). The critical Arg residue of the FLVRES motif in the pY binding pocket is shown in stick representation. d, e Close-up view of the d-SH2:(l-)monobody interface (DAM27, panel d; DAM21, panel e) highlighting the heterochiral “rippled” anti-parallel β-sheet between strand Gly84 to Trp88/His88 in DAM21/27 of the monobody FG loop with Val190 to Arg194 of the βD-strand of the d-Abl SH2 domain. f, g Close-up view of the d-SH2:(l-)monobody interface (DAM27, panel f; DAM21, panel g). The ionic interactions discussed in the “Results” section are shown as sticks and are labeled. The position of randomized residues in the monobody side-and-loop library is shown as balls. h Isothermal titration calorimetry (ITC) measurements of recombinant l-DAM21 D83A/E85A mutant titrated to the synthetic d-Abl SH2 domain. The panel shows the raw heat signal of an ITC experiment (top) and the integrated calorimetric data of the area of each peak (bottom). i Structural superposition and sequence alignment of the FG loops of the DAM27 and DAM21 monobodies.

Complex formation and structure elucidation of the heterochiral d-Abl SH2:l-monobody interaction.

a Workflow for generation of the heterochiral d-Abl SH2:l-monobody complex. d-Abl SH2 was produced by solid-phase peptide synthesis (SPPS) and native chemical ligation (NCL) and the l-monobody by bacterial expression and subsequent purification. After mixing, the complex was purified by size exclusion chromatography (SEC). b, c Analytical SEC of complex formation between synthetic d-Abl SH2 and recombinantly expressed (b) l-DAM21 and (c) l-DAM27. d Preparative SEC of d-Abl SH2:l-monobody complexes generated for crystallization condition screening. All chromatograms include a calibration of the column with standard proteins on top. e Sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) analysis of the complex formation (performed once). Source data of (e) are provided as a Source Data file.

Crystal structures of heterochiral monobody-Abl SH2 complexes.

a, b Ribbon representation of the d-Abl SH2 domain structure (green) bound to the (a) DAM27 or (b) DAM21 monobodies in l-configuration. Arg171 of the SH2 FLVRES motif, which is critical for pY binding, as well as Glu85 in the FG loop of the monobodies is shown in stick representation. c Structure of an (l-) phosphotyrosine peptide (pYEEI, stick representation) bound to the (l-)SH2 domain of the Src family kinase Lck (PDB: 1LKK). The critical Arg residue of the FLVRES motif in the pY binding pocket is shown in stick representation. d, e Close-up view of the d-SH2:(l-)monobody interface (DAM27, panel d; DAM21, panel e) highlighting the heterochiral “rippled” anti-parallel β-sheet between strand Gly84 to Trp88/His88 in DAM21/27 of the monobody FG loop with Val190 to Arg194 of the βD-strand of the d-Abl SH2 domain. f, g Close-up view of the d-SH2:(l-)monobody interface (DAM27, panel f; DAM21, panel g). The ionic interactions discussed in the “Results” section are shown as sticks and are labeled. The position of randomized residues in the monobody side-and-loop library is shown as balls. h Isothermal titration calorimetry (ITC) measurements of recombinant l-DAM21 D83A/E85A mutant titrated to the synthetic d-Abl SH2 domain. The panel shows the raw heat signal of an ITC experiment (top) and the integrated calorimetric data of the area of each peak (bottom). i Structural superposition and sequence alignment of the FG loops of the DAM27 and DAM21 monobodies.

Both monobodies use their long FG loops to present an extended segment that is positioned parallel to the central SH2 β-sheet and therefore perpendicular to the canonical backbone conformation of an SH2 domain-bound pY peptide (Fig. 4a–c). Likewise, several l-monobodies mimic pY peptide binding by also binding perpendicular across the central SH2 β-sheet, which thereby enables access to the pY and +3 pockets that are located on opposite sides of the SH2 β-sheet12,14. Unlike those, DAM21/27 form a heterochiral anti-parallel β-sheet between strand Gly84 to Trp88/His88 in DAM21/27 of their FG loop with the βD-strand (Val190 to Arg194) of the d-Abl SH2 domain (Fig. 4d, e). This interaction corresponds to a “rippled β-sheet”, which is a largely neglected structural motif predicted by Pauling and Corey in 1953 and meanwhile experimentally shown to be present in heterochiral peptide structures47. In contrast to the canonical “pleated” anti-parallel β-sheets of natural proteins, the side chains experience in rippled β-sheets much less steric crowding. Accordingly, in the DAM21/27 structures, Glu85, the key residue binding to the pY pocket, projects its sidechain in between the SH2 residues His192 and Arg194 in the observed heterochiral rippled β-sheet, whereas it would sterically clash with these residues in a pleated β-sheet, due to the different register of β-sheet hydrogen bonds (Fig. 4d, e).

The rippled β-sheet position Glu85 in the FG loop is inserted into the SH2 pY pocket, where it forms an ionic interaction with SH2 residue Arg171, which is part of the conserved FLVRES motif and makes the most critical interaction with the phosphate group in pY peptide ligands (Fig. 4f, g)48. In contrast to DAM21/27, most other SH2-targeting l-monobodies that engaged the pY pocket rather used tyrosine residues in the FG loop and often a phosphate or sulfate from the crystallization buffer to mimic pY10,14. In DAM21, FG loop residue Asp83 forms an additional ionic interaction with Arg194, unique in the Abl1 SH2 domain and therefore a possible contributor to selective Abl1 binding (Fig. 4g). In DAM27, this residue is replaced with a serine. To probe the functional roles of Asp83 and Glu85, we produced a D83A/E85A mutant in DAM21. The mutant protein displayed an almost identical far-UV circular dichroism spectrum, and thus similar secondary structure content, as wildtype DAM21 (Fig. S8, Table S4). ITC measurements showed no detectable binding (Fig. 4h), thus validating the critical role of these residues for SH2 domain binding and interface 2 as the biologically relevant protein interaction site. This is noteworthy, as glutamate is not a suitable pY mimetic in SH2 ligand peptides.

An additional ionic interaction was observed in the DAM21/27-SH2 interface: SH2 residue Arg189 interacts with Glu51 in the ßD strand of the monobody and is located at the other end of the heterochiral SH2-monobody interface (Fig. 4f, g). Interestingly, in the monobody side-and-loop library, this position in the ßD strand is a mixture of Ala, Glu, Lys, Thr, of which only Glu is suitable to interact with Arg189. Taken together, the DAM21/27 interfaces are much more prominent in ionic interactions when compared to other (homochiral) monobody-SH2 interfaces.

Structural superposition of the FG loop conformations of DAM27 and DAM21 revealed a strikingly similar conformation despite their sequence differences in seven out of eleven amino acid positions (Fig. 4i). This can be explained by a pairwise compensatory exchange of bulky and less bulky hydrophobic amino acids. This is particularly notable at the first and second to last position. In DAM27, Tyr79 packs towards the β-hairpin and His88 is solvent exposed, whereas in DAM21, Trp88 is rotated inwards and would clash with large hydrophobic or aromatic residues at this position. Hence, Tyr79 is exchanged to Ser79 in DAM21 (Fig. 4i).

In summary, monobody recognition is dominated by the FG loop of both monobodies that form a rippled β-sheet with the βD-strand of the SH2 domain, as well as ionic interactions between glutamate residues of the monobody and arginine residues of the SH2 domain.

Synthesis of d-monobody DAM27Since the monobody scaffold lacks endogenous cysteines, we planned a two-segment synthesis for DAM27 by SPPS with Thr60-Ala61 as the ligation junction through NCL and subsequent desulfurization resulting in the N- and C-terminal monobody fragments DAM27(4-60) and DAM27(61-98), respectively, spanning the entire length of the FN3 domain (Fig. 5a). In the C-terminal peptide, the N-terminal alanine was mutated to cysteine as required for NCL. Synthesis was first optimized in l-configuration due to lower reagent costs and subsequently applied for d-monobody synthesis. Both DAM27 N- and C-peptides could be synthesized without major need for optimization and obtained with high purity after HPLC purification (Table S5, Figs. S9–S12). We used a similar strategy to prepare the two peptides for NCL, as previously optimized for the synthesis of the d-Abl SH2 domain including the usage of the 3-amino-4-(methylamino)benzoic acid (MeDbz) linker and N-terminal capping strategy34. However, the product after NCL was insoluble and hence could not be analyzed and purified.Fig. 5DAM21 and DAM27 synthesis, refolding and purification.a Strategy of DAM27 synthesis. The N- and C-terminal fragments are shown in orange and green, respectively, and synthesized via solid-phase peptide synthesis (SPPS). The N-terminal peptide corresponds to the biotinylated DAM27(4-60) fragment with C-terminal MeDbz linker, which is activated to MeNbz on resin, and the C-terminal peptide resembles DAM27(62-98) with an N-terminal cysteine (Cys61) residue. After cleavage from the resin, both peptides can undergo native chemical ligation (NCL) with subsequent desulfurization to yield full-length monobody DAM27(4-98). MeDbz: 3-amino-4-(methylamino)benzoic acid; MeNbz: N-acyl-N-methylacylurea. b Size exclusion chromatography (SEC) and (c) high-performance liquid chromatography-mass spectrometry (HPLC-MS) analysis of the final refolded l- and d-DAM27 proteins in comparison with the recombinantly expressed l-DAM27-XTEN reveals similar SEC retention volumes and expected masses of the synthetic proteins. d Strategy of DAM21 synthesis. The N- and C-terminal peptides in orange and green obtained by SPPS represent the biotinylated DAM21(4-45) and DAM21(46-98) fragments, respectively. After mixing of the peptides and refolding by dialysis, the full-length split-monobody DAM21(4-98) is obtained. e SEC of the final refolded l- and d-DAM21 proteins in comparison with the recombinantly expressed l-DAM21 reveals similar SEC retention volumes of the synthetic proteins. f, g HPLC-MS analysis of the (f) DAM21(4-45) and (g) DAM21(46-98) fragments.To improve the solubility of monobody fragments, we included three repeats of the highly soluble XTEN peptide (six amino acids) at the C-terminus of the C-terminal DAM27 peptide49. Gratifyingly, the product after NCL was soluble and obtained with ~20% yield after HPLC purification in both l- and d-configuration (Figs. S13, S14). Subsequent desulfurization using our optimized protocol34 was not successful for the DAM27 ligation product, but an alternative protocol using 2-mercaptoethane sulfonate (MESNa) instead of glutathione resulted in quantitative conversion of cysteine to alanine giving rise to the native DAM27 monobody sequence (Figs. S15, S16). Refolding of the DAM27 protein was achieved by dialysis into aqueous buffer directly using the desulfurization reaction without a need of HPLC purification due to complete conversion (Figs. S15, S16). The protein was purified by SEC to remove aggregates and to determine oligomerization state. DAM27 both in l- and d-configuration eluted in a predominantly monomeric peak in line with the recombinantly expressed DAM27 control (Fig. 5b, c) and overall yield was ~1%. In summary, a facile synthesis strategy for the DAM27 monobody from two peptide segments could be established, which supplied sufficient protein for further biophysical and functional characterization.

Synthesis of d-monobody DAM27

Since the monobody scaffold lacks endogenous cysteines, we planned a two-segment synthesis for DAM27 by SPPS with Thr60-Ala61 as the ligation junction through NCL and subsequent desulfurization resulting in the N- and C-terminal monobody fragments DAM27(4-60) and DAM27(61-98), respectively, spanning the entire length of the FN3 domain (Fig. 5a). In the C-terminal peptide, the N-terminal alanine was mutated to cysteine as required for NCL. Synthesis was first optimized in l-configuration due to lower reagent costs and subsequently applied for d-monobody synthesis. Both DAM27 N- and C-peptides could be synthesized without major need for optimization and obtained with high purity after HPLC purification (Table S5, Figs. S9–S12). We used a similar strategy to prepare the two peptides for NCL, as previously optimized for the synthesis of the d-Abl SH2 domain including the usage of the 3-amino-4-(methylamino)benzoic acid (MeDbz) linker and N-terminal capping strategy34. However, the product after NCL was insoluble and hence could not be analyzed and purified.Fig. 5DAM21 and DAM27 synthesis, refolding and purification.a Strategy of DAM27 synthesis. The N- and C-terminal fragments are shown in orange and green, respectively, and synthesized via solid-phase peptide synthesis (SPPS). The N-terminal peptide corresponds to the biotinylated DAM27(4-60) fragment with C-terminal MeDbz linker, which is activated to MeNbz on resin, and the C-terminal peptide resembles DAM27(62-98) with an N-terminal cysteine (Cys61) residue. After cleavage from the resin, both peptides can undergo native chemical ligation (NCL) with subsequent desulfurization to yield full-length monobody DAM27(4-98). MeDbz: 3-amino-4-(methylamino)benzoic acid; MeNbz: N-acyl-N-methylacylurea. b Size exclusion chromatography (SEC) and (c) high-performance liquid chromatography-mass spectrometry (HPLC-MS) analysis of the final refolded l- and d-DAM27 proteins in comparison with the recombinantly expressed l-DAM27-XTEN reveals similar SEC retention volumes and expected masses of the synthetic proteins. d Strategy of DAM21 synthesis. The N- and C-terminal peptides in orange and green obtained by SPPS represent the biotinylated DAM21(4-45) and DAM21(46-98) fragments, respectively. After mixing of the peptides and refolding by dialysis, the full-length split-monobody DAM21(4-98) is obtained. e SEC of the final refolded l- and d-DAM21 proteins in comparison with the recombinantly expressed l-DAM21 reveals similar SEC retention volumes of the synthetic proteins. f, g HPLC-MS analysis of the (f) DAM21(4-45) and (g) DAM21(46-98) fragments.

DAM21 and DAM27 synthesis, refolding and purification.

a Strategy of DAM27 synthesis. The N- and C-terminal fragments are shown in orange and green, respectively, and synthesized via solid-phase peptide synthesis (SPPS). The N-terminal peptide corresponds to the biotinylated DAM27(4-60) fragment with C-terminal MeDbz linker, which is activated to MeNbz on resin, and the C-terminal peptide resembles DAM27(62-98) with an N-terminal cysteine (Cys61) residue. After cleavage from the resin, both peptides can undergo native chemical ligation (NCL) with subsequent desulfurization to yield full-length monobody DAM27(4-98). MeDbz: 3-amino-4-(methylamino)benzoic acid; MeNbz: N-acyl-N-methylacylurea. b Size exclusion chromatography (SEC) and (c) high-performance liquid chromatography-mass spectrometry (HPLC-MS) analysis of the final refolded l- and d-DAM27 proteins in comparison with the recombinantly expressed l-DAM27-XTEN reveals similar SEC retention volumes and expected masses of the synthetic proteins. d Strategy of DAM21 synthesis. The N- and C-terminal peptides in orange and green obtained by SPPS represent the biotinylated DAM21(4-45) and DAM21(46-98) fragments, respectively. After mixing of the peptides and refolding by dialysis, the full-length split-monobody DAM21(4-98) is obtained. e SEC of the final refolded l- and d-DAM21 proteins in comparison with the recombinantly expressed l-DAM21 reveals similar SEC retention volumes of the synthetic proteins. f, g HPLC-MS analysis of the (f) DAM21(4-45) and (g) DAM21(46-98) fragments.

To improve the solubility of monobody fragments, we included three repeats of the highly soluble XTEN peptide (six amino acids) at the C-terminus of the C-terminal DAM27 peptide49. Gratifyingly, the product after NCL was soluble and obtained with ~20% yield after HPLC purification in both l- and d-configuration (Figs. S13, S14). Subsequent desulfurization using our optimized protocol34 was not successful for the DAM27 ligation product, but an alternative protocol using 2-mercaptoethane sulfonate (MESNa) instead of glutathione resulted in quantitative conversion of cysteine to alanine giving rise to the native DAM27 monobody sequence (Figs. S15, S16). Refolding of the DAM27 protein was achieved by dialysis into aqueous buffer directly using the desulfurization reaction without a need of HPLC purification due to complete conversion (Figs. S15, S16). The protein was purified by SEC to remove aggregates and to determine oligomerization state. DAM27 both in l- and d-configuration eluted in a predominantly monomeric peak in line with the recombinantly expressed DAM27 control (Fig. 5b, c) and overall yield was ~1%. In summary, a facile synthesis strategy for the DAM27 monobody from two peptide segments could be established, which supplied sufficient protein for further biophysical and functional characterization.

Synthesis of the split-d-monobody DAM21After establishing the conventional synthesis of DAM27 from two pieces including NCL, we aimed at testing a more rapid and versatile approach to obtain d-monobodies without the need to optimize synthesis conditions. Inspired by early work that showed complementation of a functional monobody by split fragments in a yeast-surface-two-hybrid system50, we attempted to obtain a functional d-monobody by mixing N- and C-terminal fragments without NCL. As observed in the DAM21 crystal structure, the monobody CD loop is not involved in Abl SH2 binding. Therefore, we decided to split the sequence between Asn45 and Ser46 in the CD loop generating the fragments DAM21(4-45) and DAM21(46-98) (Fig. 5d). Luckily, these fragments were previously shown to be the most efficient in complementing a functional l-monobody50. Both DAM21 N- and C-peptides in l- and d-configuration could be synthesized without need for major optimization (Table S5, Figs. S17–S20). Although the C-terminal peptide was initially aggregating during HPLC purification, this peptide could be obtained employing an HPLC-free purification strategy, in which the desired peptide is linked to agarose beads and acetylated, truncated peptides are washed off, as they are not reacting with the beads (Figs. S19, S20). Both peptides were then mixed in a 1:1 molar ratio and refolded by dialysis. SEC purification showed a predominantly monomeric protein, but in contrast to DAM27 an appreciable peak close to the void volume was observed indicating formation of some higher-order oligomers or aggregates during refolding (Fig. 5e). In summary, we were able to generate a monomeric split version of DAM21 from two peptide fragments (Fig. 5f, g) for further biophysical and functional characterization.

Synthesis of the split-d-monobody DAM21

After establishing the conventional synthesis of DAM27 from two pieces including NCL, we aimed at testing a more rapid and versatile approach to obtain d-monobodies without the need to optimize synthesis conditions. Inspired by early work that showed complementation of a functional monobody by split fragments in a yeast-surface-two-hybrid system50, we attempted to obtain a functional d-monobody by mixing N- and C-terminal fragments without NCL. As observed in the DAM21 crystal structure, the monobody CD loop is not involved in Abl SH2 binding. Therefore, we decided to split the sequence between Asn45 and Ser46 in the CD loop generating the fragments DAM21(4-45) and DAM21(46-98) (Fig. 5d). Luckily, these fragments were previously shown to be the most efficient in complementing a functional l-monobody50. Both DAM21 N- and C-peptides in l- and d-configuration could be synthesized without need for major optimization (Table S5, Figs. S17–S20). Although the C-terminal peptide was initially aggregating during HPLC purification, this peptide could be obtained employing an HPLC-free purification strategy, in which the desired peptide is linked to agarose beads and acetylated, truncated peptides are washed off, as they are not reacting with the beads (Figs. S19, S20). Both peptides were then mixed in a 1:1 molar ratio and refolded by dialysis. SEC purification showed a predominantly monomeric protein, but in contrast to DAM27 an appreciable peak close to the void volume was observed indicating formation of some higher-order oligomers or aggregates during refolding (Fig. 5e). In summary, we were able to generate a monomeric split version of DAM21 from two peptide fragments (Fig. 5f, g) for further biophysical and functional characterization.

Folding and stability of synthetic DAM21 and DAM27We next analyzed purity, folding and stability of our synthetic l- and d-DAM21/27. Gel electrophoresis analysis revealed that the synthetic proteins are migrating at the expected molecular weight without detectable impurities (Fig. 6a, b). Of note, recombinant l-DAM27-XTEN migrated slightly higher than synthetic l- and d-DAM27-XTEN, because the recombinant construct carries four additional N-terminal amino acids from cloning and both synthetic constructs contain biotin. For small proteins, such as monobodies, these seemingly minor differences can result in a detectable difference in migration distance (Fig. 6b). Far-UV circular dichroism spectroscopy showed very similar spectra in terms of shape, signal strength and depth of minima for recombinant DAM27 and synthetic l-DAM27 both containing the XTEN extension (Fig. 6c, Table S6), in line with a pure β-sheet protein. The lower mean residue ellipticity (MRE) below 210 nm as compared to recombinant DAM27 lacking XTEN is in line with the XTEN peptide being unstructured. The synthetic d-DAM27 showed a mirrored spectrum at the x-axis (Fig. 6c). Secondary structure predictions with BeStSel51 revealed almost identical content of β-sheets and β-turns, and absence of α-helices, between recombinant and synthetic l-/d-DAM27 versions (Table S6). For the split-l-DAM21 monobody, a spectrum in line with a β-sheet protein was obtained although with some spectral differences to the recombinant (non-split) DAM21 below 215 nm (Fig. 6d, Table S7). As for split-d-DAM21 the spectrum is mirrored along the x-axis (Fig. 6d). Despite these spectral differences, secondary structure predictions still showed a similar secondary structure content especially between recombinant l-DAM21 and split-d-DAM21 (Table S7). Next, we measured thermal stability by nano differential scanning fluorimetry (nanoDSF). For DAM27 identical melting temperatures around 75 °C were observed for the synthetic l- and d-versions and its recombinant analogue (Fig. 6e, Fig. S21a, c). The presence of the XTEN peptide resulted in a mild stabilization by ~3 °C (Figs. 6e, S21a, c). Interestingly, unfolding was fully reversible for all proteins (Fig. S21b, d). As expected for a split protein, synthetic split-l- and d-DAM21 showed a lower melting temperature (Tm) around 45 °C as compared to the recombinant (non-split) version (~78 °C) (Figs. 6e,  S21e, g). Despite this, thermal unfolding was reversible as for DAM27 (Fig. S21f, h). In summary, both synthetic strategies resulted in folded monobodies.Fig. 6Folding and stability characterization of synthetic l- and d-monobodies.a, b Representative analysis of purity of synthetic (a) l- and d-DAM21 and (b) l- and d-DAM27 by sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) (performed at least twice). c, d Averaged far-UV circular dichroism (CD) spectra from three independent measurements of synthetic (c) l- and d-DAM27 and (d) l- and d-DAM21 in comparison with their recombinantly expressed analogues. Mean residue ellipticity (MRE) was calculated from three independent measurements. e Bar graph representation of thermal stability of recombinantly expressed and synthetic l- and d-monobodies assessed by nano differential scanning fluorimetry (nanoDSF). Melting temperatures (Tm) were measured at least three times (depicted as dots) and averaged. Error bars represent the standard deviation (SD). Source data of (a, b and e) are provided as a Source Data file.

Folding and stability of synthetic DAM21 and DAM27

We next analyzed purity, folding and stability of our synthetic l- and d-DAM21/27. Gel electrophoresis analysis revealed that the synthetic proteins are migrating at the expected molecular weight without detectable impurities (Fig. 6a, b). Of note, recombinant l-DAM27-XTEN migrated slightly higher than synthetic l- and d-DAM27-XTEN, because the recombinant construct carries four additional N-terminal amino acids from cloning and both synthetic constructs contain biotin. For small proteins, such as monobodies, these seemingly minor differences can result in a detectable difference in migration distance (Fig. 6b). Far-UV circular dichroism spectroscopy showed very similar spectra in terms of shape, signal strength and depth of minima for recombinant DAM27 and synthetic l-DAM27 both containing the XTEN extension (Fig. 6c, Table S6), in line with a pure β-sheet protein. The lower mean residue ellipticity (MRE) below 210 nm as compared to recombinant DAM27 lacking XTEN is in line with the XTEN peptide being unstructured. The synthetic d-DAM27 showed a mirrored spectrum at the x-axis (Fig. 6c). Secondary structure predictions with BeStSel51 revealed almost identical content of β-sheets and β-turns, and absence of α-helices, between recombinant and synthetic l-/d-DAM27 versions (Table S6). For the split-l-DAM21 monobody, a spectrum in line with a β-sheet protein was obtained although with some spectral differences to the recombinant (non-split) DAM21 below 215 nm (Fig. 6d, Table S7). As for split-d-DAM21 the spectrum is mirrored along the x-axis (Fig. 6d). Despite these spectral differences, secondary structure predictions still showed a similar secondary structure content especially between recombinant l-DAM21 and split-d-DAM21 (Table S7). Next, we measured thermal stability by nano differential scanning fluorimetry (nanoDSF). For DAM27 identical melting temperatures around 75 °C were observed for the synthetic l- and d-versions and its recombinant analogue (Fig. 6e, Fig. S21a, c). The presence of the XTEN peptide resulted in a mild stabilization by ~3 °C (Figs. 6e, S21a, c). Interestingly, unfolding was fully reversible for all proteins (Fig. S21b, d). As expected for a split protein, synthetic split-l- and d-DAM21 showed a lower melting temperature (Tm) around 45 °C as compared to the recombinant (non-split) version (~78 °C) (Figs. 6e,  S21e, g). Despite this, thermal unfolding was reversible as for DAM27 (Fig. S21f, h). In summary, both synthetic strategies resulted in folded monobodies.Fig. 6Folding and stability characterization of synthetic l- and d-monobodies.a, b Representative analysis of purity of synthetic (a) l- and d-DAM21 and (b) l- and d-DAM27 by sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) (performed at least twice). c, d Averaged far-UV circular dichroism (CD) spectra from three independent measurements of synthetic (c) l- and d-DAM27 and (d) l- and d-DAM21 in comparison with their recombinantly expressed analogues. Mean residue ellipticity (MRE) was calculated from three independent measurements. e Bar graph representation of thermal stability of recombinantly expressed and synthetic l- and d-monobodies assessed by nano differential scanning fluorimetry (nanoDSF). Melting temperatures (Tm) were measured at least three times (depicted as dots) and averaged. Error bars represent the standard deviation (SD). Source data of (a, b and e) are provided as a Source Data file.

Folding and stability characterization of synthetic l- and d-monobodies.

a, b Representative analysis of purity of synthetic (a) l- and d-DAM21 and (b) l- and d-DAM27 by sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) (performed at least twice). c, d Averaged far-UV circular dichroism (CD) spectra from three independent measurements of synthetic (c) l- and d-DAM27 and (d) l- and d-DAM21 in comparison with their recombinantly expressed analogues. Mean residue ellipticity (MRE) was calculated from three independent measurements. e Bar graph representation of thermal stability of recombinantly expressed and synthetic l- and d-monobodies assessed by nano differential scanning fluorimetry (nanoDSF). Melting temperatures (Tm) were measured at least three times (depicted as dots) and averaged. Error bars represent the standard deviation (SD). Source data of (a, b and e) are provided as a Source Data file.

Protease and plasma stability of DAM27We next assessed protease stability of DAM27 by incubation with two different broad-spectrum proteases, pepsin and proteinase K, and analyzed the digests after 4 and 24 h of incubation. Both l-versions of the recombinant and synthetic DAM27 showed complete degradation after 24 h. Minor residual protein was detected after 4 h of pepsin digestion (Fig. 7a, b). In contrast, d-DAM27 was completely resistant to degradation by both proteases. Sufficient plasma stability is a crucial prerequisite for the envisioned use of d-monobodies as protein therapeutics. After incubation of l-DAM27 in mouse plasma for up to 96 h, we observed a strong reduction of l-DAM27 signal at 24 h and all later time points. In contrast, d-DAM27 showed no signal reduction for up to 96 h (Fig. 7c). These results show very high protease resistance and plasma stability of d-monobodies.Fig. 7Protease resistance and mouse plasma stability of synthetic l- and d-monobodies.a, b Protease resistance of recombinant l-DAM27-XTEN as well as synthetic l- and d-DAM27-XTEN after incubation with (a) pepsin and (b) proteinase K was analyzed by sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE). c Plasma stability of synthetic l- and d-DAM27-XTEN. Monobodies labeled with biotin at the N-terminus after incubation with mouse plasma were analyzed by Western blotting and detection of biotin using Streptavidin-IRDye680. A representative blot of three repeats is shown. Source data of (a–c) are provided as a Source Data file.

Protease and plasma stability of DAM27

We next assessed protease stability of DAM27 by incubation with two different broad-spectrum proteases, pepsin and proteinase K, and analyzed the digests after 4 and 24 h of incubation. Both l-versions of the recombinant and synthetic DAM27 showed complete degradation after 24 h. Minor residual protein was detected after 4 h of pepsin digestion (Fig. 7a, b). In contrast, d-DAM27 was completely resistant to degradation by both proteases. Sufficient plasma stability is a crucial prerequisite for the envisioned use of d-monobodies as protein therapeutics. After incubation of l-DAM27 in mouse plasma for up to 96 h, we observed a strong reduction of l-DAM27 signal at 24 h and all later time points. In contrast, d-DAM27 showed no signal reduction for up to 96 h (Fig. 7c). These results show very high protease resistance and plasma stability of d-monobodies.Fig. 7Protease resistance and mouse plasma stability of synthetic l- and d-monobodies.a, b Protease resistance of recombinant l-DAM27-XTEN as well as synthetic l- and d-DAM27-XTEN after incubation with (a) pepsin and (b) proteinase K was analyzed by sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE). c Plasma stability of synthetic l- and d-DAM27-XTEN. Monobodies labeled with biotin at the N-terminus after incubation with mouse plasma were analyzed by Western blotting and detection of biotin using Streptavidin-IRDye680. A representative blot of three repeats is shown. Source data of (a–c) are provided as a Source Data file.

Protease resistance and mouse plasma stability of synthetic l- and d-monobodies.

a, b Protease resistance of recombinant l-DAM27-XTEN as well as synthetic l- and d-DAM27-XTEN after incubation with (a) pepsin and (b) proteinase K was analyzed by sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE). c Plasma stability of synthetic l- and d-DAM27-XTEN. Monobodies labeled with biotin at the N-terminus after incubation with mouse plasma were analyzed by Western blotting and detection of biotin using Streptavidin-IRDye680. A representative blot of three repeats is shown. Source data of (a–c) are provided as a Source Data file.

d-DAM21 and d-DAM27 bind to native l-Bcr-Abl SH2After confirming folding and stability of the d-monobodies, target binding was measured by ITC. For DAM27 binding affinities between 100 and 128 nM were measured for the interaction of the d-monobody with the l-target and vice versa (Fig. 8a–c). Of note, in the absence of the XTEN peptide a slightly higher affinity was measured (Fig. 2h, j). For DAM21 both split-l- and d-versions showed an interaction of around 100 nM with d- and l-Bcr-Abl SH2, respectively (Fig. 8d, e). While this affinity is in the same range of the d-DAM27:l-Bcr-Abl SH2 interaction, it is around 10-fold lower than the recombinant (non-split) DAM21 with d-Abl SH2 (Fig. 2g, i). All measurements showed a 1:1 binding stoichiometry and negative binding enthalpies (Fig. 8a–e). As expected, no binding was detected for l-DAM21 and l-DAM27, respectively, to the l-Bcr-Abl SH2 domain (Fig. S22).Fig. 8Binding of synthetic l- and d-monobodies to d- and l-Bcr-Abl SH2.a–e Isothermal titration calorimetry (ITC) measurements of (a) synthetic d-DAM27-XTEN titrated to recombinantly expressed l-Abl SH2, (b) synthetic l-DAM27-XTEN and (c) recombinant l-DAM27-XTEN both titrated to synthetic d-Abl SH2, (d) synthetic split-l-DAM21 and (e) synthetic split-d-DAM21 titrated to synthetic d- and l-Abl SH2, respectively. Each panel shows the raw heat signal of an ITC experiment (top) and the integrated calorimetric data of the area of each peak (bottom). The continuous line represents the best fit of the data based on a 1:1 binding model computed from the MicroCal software. Binding parameters including Kd value, stoichiometry (N), enthalpy (∆H), free enthalpy (∆G) and −T∆S calculated from the fit of each experiment are shown below. A representative measurement of at least two ITC experiments for each monobody is shown. f, g Measurement of kinase activity of (f) Bcr-Abl kinase domain (KD) and (g) SH2-KD after incubation with synthetic split-l- and d-DAM21 as well as synthetic l- and d-DAM27-XTEN in comparison with binding control monobodies HA4 and AS25 and the non-binding control monobody HA4 Y87A using a radiometric kinase assay. All monobodies were used at a concentration of 5 µM. Here, radioactively labeled 32P was incorporated into a biotinylated substrate peptide by recombinantly expressed KD and SH2-KD and detected via scintillation counting. Six independent experiments were performed (depicted as dots) and averaged. Error bars represent the standard deviation (SD) and statistical analysis was done with a one-way ANOVA and Sidak’s test. The calculated p-values are depicted in (g) and were considered statistically significant below a value of 0.05. F values and degrees of freedom were 124.3 and 40. Source data of (f, g) are provided as a Source Data file.

d-DAM21 and d-DAM27 bind to native l-Bcr-Abl SH2

After confirming folding and stability of the d-monobodies, target binding was measured by ITC. For DAM27 binding affinities between 100 and 128 nM were measured for the interaction of the d-monobody with the l-target and vice versa (Fig. 8a–c). Of note, in the absence of the XTEN peptide a slightly higher affinity was measured (Fig. 2h, j). For DAM21 both split-l- and d-versions showed an interaction of around 100 nM with d- and l-Bcr-Abl SH2, respectively (Fig. 8d, e). While this affinity is in the same range of the d-DAM27:l-Bcr-Abl SH2 interaction, it is around 10-fold lower than the recombinant (non-split) DAM21 with d-Abl SH2 (Fig. 2g, i). All measurements showed a 1:1 binding stoichiometry and negative binding enthalpies (Fig. 8a–e). As expected, no binding was detected for l-DAM21 and l-DAM27, respectively, to the l-Bcr-Abl SH2 domain (Fig. S22).Fig. 8Binding of synthetic l- and d-monobodies to d- and l-Bcr-Abl SH2.a–e Isothermal titration calorimetry (ITC) measurements of (a) synthetic d-DAM27-XTEN titrated to recombinantly expressed l-Abl SH2, (b) synthetic l-DAM27-XTEN and (c) recombinant l-DAM27-XTEN both titrated to synthetic d-Abl SH2, (d) synthetic split-l-DAM21 and (e) synthetic split-d-DAM21 titrated to synthetic d- and l-Abl SH2, respectively. Each panel shows the raw heat signal of an ITC experiment (top) and the integrated calorimetric data of the area of each peak (bottom). The continuous line represents the best fit of the data based on a 1:1 binding model computed from the MicroCal software. Binding parameters including Kd value, stoichiometry (N), enthalpy (∆H), free enthalpy (∆G) and −T∆S calculated from the fit of each experiment are shown below. A representative measurement of at least two ITC experiments for each monobody is shown. f, g Measurement of kinase activity of (f) Bcr-Abl kinase domain (KD) and (g) SH2-KD after incubation with synthetic split-l- and d-DAM21 as well as synthetic l- and d-DAM27-XTEN in comparison with binding control monobodies HA4 and AS25 and the non-binding control monobody HA4 Y87A using a radiometric kinase assay. All monobodies were used at a concentration of 5 µM. Here, radioactively labeled 32P was incorporated into a biotinylated substrate peptide by recombinantly expressed KD and SH2-KD and detected via scintillation counting. Six independent experiments were performed (depicted as dots) and averaged. Error bars represent the standard deviation (SD) and statistical analysis was done with a one-way ANOVA and Sidak’s test. The calculated p-values are depicted in (g) and were considered statistically significant below a value of 0.05. F values and degrees of freedom were 124.3 and 40. Source data of (f, g) are provided as a Source Data file.

Binding of synthetic l- and d-monobodies to d- and l-Bcr-Abl SH2.

a–e Isothermal titration calorimetry (ITC) measurements of (a) synthetic d-DAM27-XTEN titrated to recombinantly expressed l-Abl SH2, (b) synthetic l-DAM27-XTEN and (c) recombinant l-DAM27-XTEN both titrated to synthetic d-Abl SH2, (d) synthetic split-l-DAM21 and (e) synthetic split-d-DAM21 titrated to synthetic d- and l-Abl SH2, respectively. Each panel shows the raw heat signal of an ITC experiment (top) and the integrated calorimetric data of the area of each peak (bottom). The continuous line represents the best fit of the data based on a 1:1 binding model computed from the MicroCal software. Binding parameters including Kd value, stoichiometry (N), enthalpy (∆H), free enthalpy (∆G) and −T∆S calculated from the fit of each experiment are shown below. A representative measurement of at least two ITC experiments for each monobody is shown. f, g Measurement of kinase activity of (f) Bcr-Abl kinase domain (KD) and (g) SH2-KD after incubation with synthetic split-l- and d-DAM21 as well as synthetic l- and d-DAM27-XTEN in comparison with binding control monobodies HA4 and AS25 and the non-binding control monobody HA4 Y87A using a radiometric kinase assay. All monobodies were used at a concentration of 5 µM. Here, radioactively labeled 32P was incorporated into a biotinylated substrate peptide by recombinantly expressed KD and SH2-KD and detected via scintillation counting. Six independent experiments were performed (depicted as dots) and averaged. Error bars represent the standard deviation (SD) and statistical analysis was done with a one-way ANOVA and Sidak’s test. The calculated p-values are depicted in (g) and were considered statistically significant below a value of 0.05. F values and degrees of freedom were 124.3 and 40. Source data of (f, g) are provided as a Source Data file.

Inhibition of Bcr-Abl kinase activity by d-monobodiesAfter showing binding to the Bcr-Abl SH2 domain with high affinity, we next wanted to assess a possible inhibitory effect of monobodies on Bcr-Abl kinase activity in vitro. We recombinantly expressed and purified BCR::ABL1 fragments either containing the kinase domain (KD) alone or a larger fragment also containing the SH2 domain (SH2-KD)52. After addition of different monobody proteins, kinase activity was measured using a radiometric kinase assay13. Besides the l- and d-versions of DAM21 and DAM27, we included the previously characterized Abl SH2 binding monobodies HA4 and AS25 as well as the non-binding control monobody HA4 Y87A10,13. In vitro kinase activity of Abl1 KD (not including the Abl1 SH2 domain) remained unchanged upon incubation with all monobodies indicating no unspecific inhibitory or activating effect of the monobodies on Abl1 kinase activity (Fig. 8f). In contrast, d-DAM27 resulted in a major reduction of kinase activity of SH2-KD, comparable to inhibition by the validated allosteric Bcr-Abl inhibiting monobody AS25. In contrast, no inhibition was observed with l-DAM27. Also, no inhibition was observed with HA4 and HA4 Y87A, in line with previous observations (Fig. 8g)10. Although we noted a minor and unexplained increase in kinase activity by split-monobody l-DAM21, d-DAM21 showed significant inhibition as compared to its l-counterpart, although to a lesser extent than d-DAM27 (Fig. 8g). These data show that d-monobodies binding to the Abl1 SH2 domain are able to inhibit the kinase activity of Bcr-Abl, which is crucial for BCR::ABL1 signaling and CML maintenance.

Inhibition of Bcr-Abl kinase activity by d-monobodies

After showing binding to the Bcr-Abl SH2 domain with high affinity, we next wanted to assess a possible inhibitory effect of monobodies on Bcr-Abl kinase activity in vitro. We recombinantly expressed and purified BCR::ABL1 fragments either containing the kinase domain (KD) alone or a larger fragment also containing the SH2 domain (SH2-KD)52. After addition of different monobody proteins, kinase activity was measured using a radiometric kinase assay13. Besides the l- and d-versions of DAM21 and DAM27, we included the previously characterized Abl SH2 binding monobodies HA4 and AS25 as well as the non-binding control monobody HA4 Y87A10,13. In vitro kinase activity of Abl1 KD (not including the Abl1 SH2 domain) remained unchanged upon incubation with all monobodies indicating no unspecific inhibitory or activating effect of the monobodies on Abl1 kinase activity (Fig. 8f). In contrast, d-DAM27 resulted in a major reduction of kinase activity of SH2-KD, comparable to inhibition by the validated allosteric Bcr-Abl inhibiting monobody AS25. In contrast, no inhibition was observed with l-DAM27. Also, no inhibition was observed with HA4 and HA4 Y87A, in line with previous observations (Fig. 8g)10. Although we noted a minor and unexplained increase in kinase activity by split-monobody l-DAM21, d-DAM21 showed significant inhibition as compared to its l-counterpart, although to a lesser extent than d-DAM27 (Fig. 8g). These data show that d-monobodies binding to the Abl1 SH2 domain are able to inhibit the kinase activity of Bcr-Abl, which is crucial for BCR::ABL1 signaling and CML maintenance.

High selectivity of d-monobodies for the BCR::ABL1 SH2 domainNext, we studied selectivity of the d-monobodies for the BCR::ABL1 SH2 domain. Analysis of the DAM21/27 crystal structures (see Fig. 4a, b) indicated that the CD loop of the Abl SH2 domain is positioned close to the βC and βD strands of DAM27 and DAM21 (Fig. 9a). The Abl1 (and Abl2) SH2 domain has a particularly short CD loop, which seems to be a requirement for binding to DAM21 and DAM27 (Fig. 9a). In contrast, all SH2s of the Src- and Tec-family (9 and 5 members, respectively), the closest paralogues of the Abl family, have CD loops that are four to six amino acids longer. Molecular modeling indicates that these longer CD loops would clash with the βC/βD strands of DAM21 and DAM27 (Fig. 9b, c). Also, several other SH2 families including the Jak kinases, STAT transcription factors and SHP1/2 (C-SH2) tyrosine phosphatases have such long CD loops (Table S8). In addition, the ionic interaction of Glu51 and Arg189 (Fig. 4f, g), can only be formed with Abl kinase SH2 domains, as many other SH2 domains, including the Src kinase SH2s, do not have Arg (or Lys) in this position (Fig. 9c, Table S8). To assess this predicted selectivity experimentally, ITC measurements with the SH2 domains of Bruton’s tyrosine kinase (Btk) and lymphocyte-specific protein tyrosine kinase (Lck), as representatives of the closely related Tec- and Src-family showed that both d-DAM27 and d-DAM21 did not bind to these SH2 domains (Fig. 9d–g). Overall, these observations indicate a high selectivity of DAM21 and DAM27 for the Abl SH2 domain.Fig. 9Selectivity of d-DAM21/27 for the BCR::ABL1 SH2 domain.a
l-DAM27:d-Abl SH2 complex structure (PDB code: 9F00) highlighting the short Abl SH2 CD loop that enables binding to DAM27. The space where longer CD loops in other SH2 domains would be located is indicated by the dotted red circle. b Structural superposition of the Abl, Lck and Btk SH2 domains (PDB entries 3K2M, 1LKK and 6HTF, respectively). The additional four to six amino acid residues in the Lck and Btk SH2 CD loops are not compatible with binding to the DAM21 and DAM27 monobodies. c Multiple sequence alignment of the βC and βD strands of SH2 domains belonging to different SH2 domain-containing tyrosine kinase families. The βC and βD strands are colored in red and green, respectively, while the CD loop in between is black. Arg189 of ABL1, which is important for d-monobody binding via ionic interactions, is not conserved and colored in bold green to highlight the different amino acid sequences in this position. d, e Isothermal titration calorimetry (ITC) measurements of d-DAM27-XTEN titrated to the SH2 domains of (d) Btk and (e) Lck. f, g ITC measurements of split-d-DAM21 titrated to the SH2 domains of (f) Btk and (g) Lck. Each panel shows the raw heat signal of an ITC experiment (top) and the integrated calorimetric data of the area of each peak (bottom). h, i Volcano plots of identified proteins via mass spectrometry after pulldown from K562 cell lysates comparing synthetic (h) split-d-DAM21 with split-l-DAM21 and (i) d-DAM27-XTEN with l-DAM27-XTEN. Pulldowns were performed in three biological replicates for each monobody. Protein identification and statistical analysis of replicates were done with MaxQuant 2.5.1.0 and Autonomics (R package version 1.13.21)84 resulting in FDR-corrected p-values (Benjamini-Hochberg procedure) for each identified protein where a p-value below 0.05, which corresponds to -log10(p) above 1.3, was considered statistically significant (dotted line intersecting y-axis). A log2(ratio) above 1.00 of d- vs. l-monobody correlates to a ratio above 2.00 and a higher abundance of the protein when the pulldown was performed with the d-monobody compared to its l-counterpart (dotted lines intersecting x-axis). The color coding represents the protein groups BCR and ABL1 (light red), BCR::ABL1/ABL1 interactors (blue), BCR::ABL1/ABL1 interactors with SH2 domains (green) and other proteins containing SH2 domains (purple). Plotted values of all highlighted protein groups are listed in Tables S9, S10.

High selectivity of d-monobodies for the BCR::ABL1 SH2 domain

Next, we studied selectivity of the d-monobodies for the BCR::ABL1 SH2 domain. Analysis of the DAM21/27 crystal structures (see Fig. 4a, b) indicated that the CD loop of the Abl SH2 domain is positioned close to the βC and βD strands of DAM27 and DAM21 (Fig. 9a). The Abl1 (and Abl2) SH2 domain has a particularly short CD loop, which seems to be a requirement for binding to DAM21 and DAM27 (Fig. 9a). In contrast, all SH2s of the Src- and Tec-family (9 and 5 members, respectively), the closest paralogues of the Abl family, have CD loops that are four to six amino acids longer. Molecular modeling indicates that these longer CD loops would clash with the βC/βD strands of DAM21 and DAM27 (Fig. 9b, c). Also, several other SH2 families including the Jak kinases, STAT transcription factors and SHP1/2 (C-SH2) tyrosine phosphatases have such long CD loops (Table S8). In addition, the ionic interaction of Glu51 and Arg189 (Fig. 4f, g), can only be formed with Abl kinase SH2 domains, as many other SH2 domains, including the Src kinase SH2s, do not have Arg (or Lys) in this position (Fig. 9c, Table S8). To assess this predicted selectivity experimentally, ITC measurements with the SH2 domains of Bruton’s tyrosine kinase (Btk) and lymphocyte-specific protein tyrosine kinase (Lck), as representatives of the closely related Tec- and Src-family showed that both d-DAM27 and d-DAM21 did not bind to these SH2 domains (Fig. 9d–g). Overall, these observations indicate a high selectivity of DAM21 and DAM27 for the Abl SH2 domain.Fig. 9Selectivity of d-DAM21/27 for the BCR::ABL1 SH2 domain.a
l-DAM27:d-Abl SH2 complex structure (PDB code: 9F00) highlighting the short Abl SH2 CD loop that enables binding to DAM27. The space where longer CD loops in other SH2 domains would be located is indicated by the dotted red circle. b Structural superposition of the Abl, Lck and Btk SH2 domains (PDB entries 3K2M, 1LKK and 6HTF, respectively). The additional four to six amino acid residues in the Lck and Btk SH2 CD loops are not compatible with binding to the DAM21 and DAM27 monobodies. c Multiple sequence alignment of the βC and βD strands of SH2 domains belonging to different SH2 domain-containing tyrosine kinase families. The βC and βD strands are colored in red and green, respectively, while the CD loop in between is black. Arg189 of ABL1, which is important for d-monobody binding via ionic interactions, is not conserved and colored in bold green to highlight the different amino acid sequences in this position. d, e Isothermal titration calorimetry (ITC) measurements of d-DAM27-XTEN titrated to the SH2 domains of (d) Btk and (e) Lck. f, g ITC measurements of split-d-DAM21 titrated to the SH2 domains of (f) Btk and (g) Lck. Each panel shows the raw heat signal of an ITC experiment (top) and the integrated calorimetric data of the area of each peak (bottom). h, i Volcano plots of identified proteins via mass spectrometry after pulldown from K562 cell lysates comparing synthetic (h) split-d-DAM21 with split-l-DAM21 and (i) d-DAM27-XTEN with l-DAM27-XTEN. Pulldowns were performed in three biological replicates for each monobody. Protein identification and statistical analysis of replicates were done with MaxQuant 2.5.1.0 and Autonomics (R package version 1.13.21)84 resulting in FDR-corrected p-values (Benjamini-Hochberg procedure) for each identified protein where a p-value below 0.05, which corresponds to -log10(p) above 1.3, was considered statistically significant (dotted line intersecting y-axis). A log2(ratio) above 1.00 of d- vs. l-monobody correlates to a ratio above 2.00 and a higher abundance of the protein when the pulldown was performed with the d-monobody compared to its l-counterpart (dotted lines intersecting x-axis). The color coding represents the protein groups BCR and ABL1 (light red), BCR::ABL1/ABL1 interactors (blue), BCR::ABL1/ABL1 interactors with SH2 domains (green) and other proteins containing SH2 domains (purple). Plotted values of all highlighted protein groups are listed in Tables S9, S10.

Selectivity of d-DAM21/27 for the BCR::ABL1 SH2 domain.

a
l-DAM27:d-Abl SH2 complex structure (PDB code: 9F00) highlighting the short Abl SH2 CD loop that enables binding to DAM27. The space where longer CD loops in other SH2 domains would be located is indicated by the dotted red circle. b Structural superposition of the Abl, Lck and Btk SH2 domains (PDB entries 3K2M, 1LKK and 6HTF, respectively). The additional four to six amino acid residues in the Lck and Btk SH2 CD loops are not compatible with binding to the DAM21 and DAM27 monobodies. c Multiple sequence alignment of the βC and βD strands of SH2 domains belonging to different SH2 domain-containing tyrosine kinase families. The βC and βD strands are colored in red and green, respectively, while the CD loop in between is black. Arg189 of ABL1, which is important for d-monobody binding via ionic interactions, is not conserved and colored in bold green to highlight the different amino acid sequences in this position. d, e Isothermal titration calorimetry (ITC) measurements of d-DAM27-XTEN titrated to the SH2 domains of (d) Btk and (e) Lck. f, g ITC measurements of split-d-DAM21 titrated to the SH2 domains of (f) Btk and (g) Lck. Each panel shows the raw heat signal of an ITC experiment (top) and the integrated calorimetric data of the area of each peak (bottom). h, i Volcano plots of identified proteins via mass spectrometry after pulldown from K562 cell lysates comparing synthetic (h) split-d-DAM21 with split-l-DAM21 and (i) d-DAM27-XTEN with l-DAM27-XTEN. Pulldowns were performed in three biological replicates for each monobody. Protein identification and statistical analysis of replicates were done with MaxQuant 2.5.1.0 and Autonomics (R package version 1.13.21)84 resulting in FDR-corrected p-values (Benjamini-Hochberg procedure) for each identified protein where a p-value below 0.05, which corresponds to -log10(p) above 1.3, was considered statistically significant (dotted line intersecting y-axis). A log2(ratio) above 1.00 of d- vs. l-monobody correlates to a ratio above 2.00 and a higher abundance of the protein when the pulldown was performed with the d-monobody compared to its l-counterpart (dotted lines intersecting x-axis). The color coding represents the protein groups BCR and ABL1 (light red), BCR::ABL1/ABL1 interactors (blue), BCR::ABL1/ABL1 interactors with SH2 domains (green) and other proteins containing SH2 domains (purple). Plotted values of all highlighted protein groups are listed in Tables S9, S10.

Binding of d-monobodies to Bcr-Abl in cell lysates and permeabilized CML cellsIn order to determine if d-monobodies can bind to full-length BCR::ABL1 in the context of the complex proteome of human cells, we performed pulldown experiments with the biotinylated l- and d-DAM21/27 variants in three biological repeats from lysates of the BCR::ABL1-expressing cell line K562, one of the most commonly used cell lines in CML research, followed by quantitative proteomics analysis of bound proteins. This approach also enables to complement the findings on d-monobody selectivity described above and to determine its interactome. Comparison of d- vs. l-DAM21 showed higher abundance of BCR and ABL1 in the samples from the d-DAM21 pulldown when compared to its respective l-counterpart (Fig. 9h, Table S9). Comparing the pulldowns of d- vs. l-DAM27 showed higher abundance of BCR, but only a mild enrichment of ABL1, which was not significant (Fig. 9i, Table S10). Since BCR::ABL1 is a fusion oncoprotein, enrichment of either BCR or ABL1 is sufficient to show binding of d-DAM27 to BCR::ABL1. Interestingly, we also found several previously identified and validated BCR::ABL1 interactors, including kinases (CDK1, MTOR, PRKDC)53–55, proteins involved in transcription regulation, splicing and cell proliferation control (STAT1, FUS, GTF3C4, YTHDC1, EPS15)56–61, proteins mediating proteasomal degradation (CUL4B, TRIM25, PSMA4)62–64, transport proteins and chaperones (AP2A1, XPO1, HSPD1)61,65–67, as well as cytoskeletal components (EMD, SPTA1)68,69, significantly enriched with the two d-monobodies (Fig. 9h, i, Tables S9, S10). In line with previous observations with other Abl SH2-targeting monobodies10, it is likely that these proteins piggyback on BCR::ABL1 and are therefore not direct interactors of the monobodies. Importantly, no other SH2 domain-containing proteins, such as BTK, CSK, SH2B1 and TYK2, were significantly enriched with the d-monobody variants, despite the expression of ~70 SH2 domain-containing proteins in K562 cells, and thus confirming the ITC measurements with the Btk SH2 domain resulting in no binding (Fig. 9d, f). As expected for single-step pulldown mass spectrometry experiments, the dataset contains a lot of common background contaminants, which are listed in the Contaminant Repository for Affinity Purification (CRAPome)70 and were therefore disregarded. Together, the proteomics data show preferential binding of d-DAM21 and d-DAM27 to BCR::ABL1 complexes over their l-counterparts and that both d-monobodies are able to bind BCR::ABL1, but no other SH2-containing proteins in the complex proteome of a human cell.In addition, we used a FACS-based assay, in which we permeabilized K562 cells and monitored the binding of the d-monobodies by a shift in fluorescence signal. Without optimization of assay parameters, binding of d-DAM21 and d-DAM27 was readily observed to be much stronger than the binding of synthetic l-DAM21 and l-DAM27 and exceeded binding of the allosteric BCR::ABL1 binder/inhibitor AS25 (Fig. S23). No binding was observed for the negative control monobody HA4 Y87A (Fig. S23).These two assays show binding of the two d-monobodies to full-length BCR::ABL1 in cell lysates and permeabilized human BCR::ABL1-expressing cells.

Binding of d-monobodies to Bcr-Abl in cell lysates and permeabilized CML cells

In order to determine if d-monobodies can bind to full-length BCR::ABL1 in the context of the complex proteome of human cells, we performed pulldown experiments with the biotinylated l- and d-DAM21/27 variants in three biological repeats from lysates of the BCR::ABL1-expressing cell line K562, one of the most commonly used cell lines in CML research, followed by quantitative proteomics analysis of bound proteins. This approach also enables to complement the findings on d-monobody selectivity described above and to determine its interactome. Comparison of d- vs. l-DAM21 showed higher abundance of BCR and ABL1 in the samples from the d-DAM21 pulldown when compared to its respective l-counterpart (Fig. 9h, Table S9). Comparing the pulldowns of d- vs. l-DAM27 showed higher abundance of BCR, but only a mild enrichment of ABL1, which was not significant (Fig. 9i, Table S10). Since BCR::ABL1 is a fusion oncoprotein, enrichment of either BCR or ABL1 is sufficient to show binding of d-DAM27 to BCR::ABL1. Interestingly, we also found several previously identified and validated BCR::ABL1 interactors, including kinases (CDK1, MTOR, PRKDC)53–55, proteins involved in transcription regulation, splicing and cell proliferation control (STAT1, FUS, GTF3C4, YTHDC1, EPS15)56–61, proteins mediating proteasomal degradation (CUL4B, TRIM25, PSMA4)62–64, transport proteins and chaperones (AP2A1, XPO1, HSPD1)61,65–67, as well as cytoskeletal components (EMD, SPTA1)68,69, significantly enriched with the two d-monobodies (Fig. 9h, i, Tables S9, S10). In line with previous observations with other Abl SH2-targeting monobodies10, it is likely that these proteins piggyback on BCR::ABL1 and are therefore not direct interactors of the monobodies. Importantly, no other SH2 domain-containing proteins, such as BTK, CSK, SH2B1 and TYK2, were significantly enriched with the d-monobody variants, despite the expression of ~70 SH2 domain-containing proteins in K562 cells, and thus confirming the ITC measurements with the Btk SH2 domain resulting in no binding (Fig. 9d, f). As expected for single-step pulldown mass spectrometry experiments, the dataset contains a lot of common background contaminants, which are listed in the Contaminant Repository for Affinity Purification (CRAPome)70 and were therefore disregarded. Together, the proteomics data show preferential binding of d-DAM21 and d-DAM27 to BCR::ABL1 complexes over their l-counterparts and that both d-monobodies are able to bind BCR::ABL1, but no other SH2-containing proteins in the complex proteome of a human cell.

In addition, we used a FACS-based assay, in which we permeabilized K562 cells and monitored the binding of the d-monobodies by a shift in fluorescence signal. Without optimization of assay parameters, binding of d-DAM21 and d-DAM27 was readily observed to be much stronger than the binding of synthetic l-DAM21 and l-DAM27 and exceeded binding of the allosteric BCR::ABL1 binder/inhibitor AS25 (Fig. S23). No binding was observed for the negative control monobody HA4 Y87A (Fig. S23).

These two assays show binding of the two d-monobodies to full-length BCR::ABL1 in cell lysates and permeabilized human BCR::ABL1-expressing cells.

DiscussionWe and a parallel study by Hayashi et al.46 show that d-monobodies for two therapeutically validated targets with different fold, sequence and function can be generated. Some main technical differences of the two studies are the use of monobody libraries with different positions and degrees of residue randomization and different selection techniques: Phage and yeast display in this study and mRNA display in the Hayashi et al. paper. Importantly, also different strategies were used to chemically synthesize d-monobodies: A 2-piece NCL strategy and assembly of a split-d-monobody was used in our study, whereas Hayashi et al. employed a 3-piece NCL strategy. Despite these strong differences, both studies converge to a common endpoint, the development of a functional high-affinity d-monobody. This convergence argues for the robustness of the workflows and promise that these can be applied to develop mirror-image monobodies for a broad range of additional targets.Monobodies hit a sweet spot for the development of d-protein therapeutics, as they can be developed to bind diverse paratope topologies, which enables a broad spectrum of possible targets8. Furthermore, the small size of monobodies promises a more straightforward chemical synthesis as compared to larger synthetic binders, such as DARPins, repebodies or anticalins or binder classes that have disulfide bridges.Our study also illustrates an alternative way of how an SH2 domain can be targeted by heterochiral protein-protein interactions that are distinct from previously developed monobodies. Including the co-crystal structures presented here, we have solved 8 structures of SH2 domain-monobody complexes in which the monobody inhibits pY ligand binding10,12,14. Collectively, five different binding/inhibition modes can be distinguished: (1) The FG loops of monobodies HA4 and Mb(Yes_1) targeting the Abl1- and Yes-SH2, respectively, carry a tyrosine residue, which, together with an inorganic sulfate or phosphate ion, respectively, closely mimics a pY peptide ligand10,14; (2) Monobodies Mb(Lck_1) and Mb(Lck_3) bind the Lck SH2 domain and block the +3 specificity pocket with their respective CD loop14; (3) and (4) Monobodies NSa1 and CS1 show a pY-independent mode of interaction with the N- and C-terminal SH2 domain of SHP2, respectively. They still mimic the bound peptide and bind the same epitope, but bind in two opposite orientations10,12,14; (5) The DAM21/27 monobodies described in this study engage the pY pocket using a glutamate residue in the FG loop, several salt bridges and a heterochiral rippled β-sheet that runs perpendicular to the peptide-binding groove and hence drastically distinct to all other binding modes. Hence, one may envision that d-monobodies could increase the number of accessible binding modes for this and other l-targets. Overall, the five different SH2 binding modes and mechanisms of pY competition that were observed so far may be one reason for the high selectivity of SH2-targeting monobodies.The recent rise of flow chemistry has enabled the production of peptide chains of up to 164 amino acids in one step and will facilitate d-protein synthesis71. On the other hand, our 2-piece d-monobody synthesis strategy has the advantage that the C-terminal peptide contains the FG loop and the randomized library residues of the βC/βD strands, both major contributors to monobody-target binding, whereas the N-terminal peptide only contains that CD loop, which often is not involved in target binding. Hence, we envisage reusing the N-peptides and only swapping the C-peptide for the synthesis of other continuous and split-d-monobodies without the need for further protocol modifications. We also demonstrated that split-d-monobodies can be made by 1:1 mixing of N- and C-terminal monobody peptides. Advantages of this strategy include a more facile and quicker development, as no NCL needs to be optimized and circumventing a possible insoluble ligation product, saving a purification step and making desulfurization unnecessary. Also, there is no need to change the native sequence, as the split site, in contrast to NCL does not require a particular sequence. On the other hand, the CD loop cannot be used as the preferential split, if it is involved in target binding, which might be hard to evaluate in the absence of structural information or mutagenesis experiments. In addition, while high-affinity binding is retained, the split monobody had a reduced binding affinity and also some impact on folding and stability was observed.The next steps that go beyond the scope of this paper are to determine pharmacokinetics and biodistribution of d-monobodies in vivo, to incorporate suitable intracellular delivery approaches and to evaluate a possible superior intracellular stability of d-monobodies. Our lab is currently finishing work on cellular delivery using recombinant “supercharged” (l-)monobodies and hence will be the focus of future research.While there are enormous efforts and progress to facilitate development strategies for d-protein binders, a clear bottleneck is the necessity to synthesize both target protein and binder in d-configuration, as combinatorial libraries for phage, yeast or mRNA display selection are only available in l-configuration. Therefore, efforts to replicate biological systems in d-configuration are important developments to eventually construct a self-replicating and -expressing mirror-image biological system. Advances include the synthesis of mirror-image DNA polymerases Dpo472 and Pfu73, DNA ligase74, T7 RNA polymerase75 and 5S ribonucleoprotein complexes as components of a functional mirror-image ribosome76. Additionally, d-proteins can be sequenced by mirror-image trypsin digestion77.

We and a parallel study by Hayashi et al.46 show that d-monobodies for two therapeutically validated targets with different fold, sequence and function can be generated. Some main technical differences of the two studies are the use of monobody libraries with different positions and degrees of residue randomization and different selection techniques: Phage and yeast display in this study and mRNA display in the Hayashi et al. paper. Importantly, also different strategies were used to chemically synthesize d-monobodies: A 2-piece NCL strategy and assembly of a split-d-monobody was used in our study, whereas Hayashi et al. employed a 3-piece NCL strategy. Despite these strong differences, both studies converge to a common endpoint, the development of a functional high-affinity d-monobody. This convergence argues for the robustness of the workflows and promise that these can be applied to develop mirror-image monobodies for a broad range of additional targets.

Monobodies hit a sweet spot for the development of d-protein therapeutics, as they can be developed to bind diverse paratope topologies, which enables a broad spectrum of possible targets8. Furthermore, the small size of monobodies promises a more straightforward chemical synthesis as compared to larger synthetic binders, such as DARPins, repebodies or anticalins or binder classes that have disulfide bridges.

Our study also illustrates an alternative way of how an SH2 domain can be targeted by heterochiral protein-protein interactions that are distinct from previously developed monobodies. Including the co-crystal structures presented here, we have solved 8 structures of SH2 domain-monobody complexes in which the monobody inhibits pY ligand binding10,12,14. Collectively, five different binding/inhibition modes can be distinguished: (1) The FG loops of monobodies HA4 and Mb(Yes_1) targeting the Abl1- and Yes-SH2, respectively, carry a tyrosine residue, which, together with an inorganic sulfate or phosphate ion, respectively, closely mimics a pY peptide ligand10,14; (2) Monobodies Mb(Lck_1) and Mb(Lck_3) bind the Lck SH2 domain and block the +3 specificity pocket with their respective CD loop14; (3) and (4) Monobodies NSa1 and CS1 show a pY-independent mode of interaction with the N- and C-terminal SH2 domain of SHP2, respectively. They still mimic the bound peptide and bind the same epitope, but bind in two opposite orientations10,12,14; (5) The DAM21/27 monobodies described in this study engage the pY pocket using a glutamate residue in the FG loop, several salt bridges and a heterochiral rippled β-sheet that runs perpendicular to the peptide-binding groove and hence drastically distinct to all other binding modes. Hence, one may envision that d-monobodies could increase the number of accessible binding modes for this and other l-targets. Overall, the five different SH2 binding modes and mechanisms of pY competition that were observed so far may be one reason for the high selectivity of SH2-targeting monobodies.

The recent rise of flow chemistry has enabled the production of peptide chains of up to 164 amino acids in one step and will facilitate d-protein synthesis71. On the other hand, our 2-piece d-monobody synthesis strategy has the advantage that the C-terminal peptide contains the FG loop and the randomized library residues of the βC/βD strands, both major contributors to monobody-target binding, whereas the N-terminal peptide only contains that CD loop, which often is not involved in target binding. Hence, we envisage reusing the N-peptides and only swapping the C-peptide for the synthesis of other continuous and split-d-monobodies without the need for further protocol modifications. We also demonstrated that split-d-monobodies can be made by 1:1 mixing of N- and C-terminal monobody peptides. Advantages of this strategy include a more facile and quicker development, as no NCL needs to be optimized and circumventing a possible insoluble ligation product, saving a purification step and making desulfurization unnecessary. Also, there is no need to change the native sequence, as the split site, in contrast to NCL does not require a particular sequence. On the other hand, the CD loop cannot be used as the preferential split, if it is involved in target binding, which might be hard to evaluate in the absence of structural information or mutagenesis experiments. In addition, while high-affinity binding is retained, the split monobody had a reduced binding affinity and also some impact on folding and stability was observed.

The next steps that go beyond the scope of this paper are to determine pharmacokinetics and biodistribution of d-monobodies in vivo, to incorporate suitable intracellular delivery approaches and to evaluate a possible superior intracellular stability of d-monobodies. Our lab is currently finishing work on cellular delivery using recombinant “supercharged” (l-)monobodies and hence will be the focus of future research.

While there are enormous efforts and progress to facilitate development strategies for d-protein binders, a clear bottleneck is the necessity to synthesize both target protein and binder in d-configuration, as combinatorial libraries for phage, yeast or mRNA display selection are only available in l-configuration. Therefore, efforts to replicate biological systems in d-configuration are important developments to eventually construct a self-replicating and -expressing mirror-image biological system. Advances include the synthesis of mirror-image DNA polymerases Dpo472 and Pfu73, DNA ligase74, T7 RNA polymerase75 and 5S ribonucleoprotein complexes as components of a functional mirror-image ribosome76. Additionally, d-proteins can be sequenced by mirror-image trypsin digestion77.

MethodsAntibodies, cell lines and reagentsThe following antibodies, secondary antibodies and fluorophore-coupled streptavidin were used for monobody selection and yeast display binding assays: mouse anti-V5 (Thermo Fisher Scientific, MA5-15253) and FITC-coupled anti-mouse IgG (Sigma-Aldrich, F0257-5ML), streptavidin-DyLight650 (Thermo Fisher Scientific, 84547). AlexaFluor488-coupled Streptavidin (S32354) used for detection of monobodies binding to Bcr-Abl in cells was purchased from Thermo Fisher Scientific. For Western blots of plasma stability assays IRDye 680-Streptavidin (926-68079) was purchased from LiCOR and used at 1:10,000 dilution in 5% Milk in Phosphate-Buffered Saline (PBS-T). Non-sterile mouse plasma with sodium heparin (ABIN925342) was purchased from antibodies-online/Rockland Immunochemicals. Streptavidin MagneSphere Paramagnetic Particles (Z5481, Promega) were used during monobody selection. K562 cells were purchased from DSMZ (ACC-10, Deutsche Sammlung von Mikroorganismen und Zellkulturen, Braunschweig, Germany). Further reagents used are listed in the Supplementary Information.Monobody selectionMonobodies were selected according to methods previously described using biotinylated target proteins7,78. Briefly, four rounds of phage display were followed by the amplification and transformation of EBY100 yeast cells with the DNA sequences corresponding to binding monobodies. The yeast cells were next sorted using FACS based on a strict gating strategy comprising double positive cells for monobody display and binding to biotinylated target. Isolated clones were next sequenced and cloned into the pHFT2 vector, digested with BamHI and XhoI, by a standard Gibson assembly protocol (NEB) for further characterization. Here, the monobody fragments were amplified from the isolated yeast plasmids via PCR using 5’-gtgaaaacctgtacttccagggatccatggctgcttcttctg-3’ and 5’- cagtggtggtggtggtggtgctcgagctaggtacggtagttaatc-3’ as primers. A detailed description of the selection procedure can be found in the Supplementary Information.Binding assay in yeast display formatIncreasing concentrations of synthetic biotinylated protein target d-Abl SH2 resuspended in Tris-buffered saline (TBS) with 0.1% BSA were incubated with 5 × 105 yeast cells per sample displaying monobodies and a mouse anti-V5 antibody (5 µL of 1:300 dilution) in a total volume of 20 µL for 30 min at room temperature. Three washes with TBS with 0.1% BSA followed together with incubation for 30 min with a streptavidin-DyLight650 and a FITC-coupled anti-mouse IgG (both diluted 1:1000 in the same tube and 20 µL used per sample) at room temperature in the dark. Samples were washed twice with Tris-buffered saline (TBS) with 0.1% BSA and analyzed on a Guava easyCyte flow cytometer (Luminex). Data were fitted on a 1:1 binding model using the Prism software (GraphPad) to determine Kd values.Recombinant protein expressionGenerally, E. coli BL21 Star (DE3) cells were transformed with plasmids containing the desired proteins. An overnight pre-culture was added to medium (1:20, v/v) and cells were grown at 37 °C, 200 rpm. For expressions in lysogeny broth (LB; until OD600 0.5–0.8) and terrific broth (TB; until OD600 1.2) medium, the culture was next induced with 0.5 mM IPTG final concentration and cells were further grown overnight at 18 °C, 200 rpm. Expressions in auto induction (AI) medium were placed at 18 °C without IPTG addition. The next day, cells were harvested at 6000×g for 10 min at 4 °C.The Abl SH2, Btk SH2 and Lck SH2 domains were produced in LB medium with an N-terminal tag including a His6 and Glutathione S-transferase (GST), and tobacco etch virus (TEV) protease recognition motifs using the pETM30 vector. The Abl SH2 A198C mutation was introduced using the QuikChange II Site-Directed Mutagenesis Kit (Agilent) with 5’-cattacaggatcaacacttgctctgatggcaagctc-3’ and 5’-gagcttgccatcagagcaagtgttgatcctgtaatg-3’ as primers. Protein purification and tag cleavage was performed as described previously by nickel-affinity chromatography (His-Trap FF crude, Cytiva) and SEC (HiLoad 16/600 Superdex 75 pg, Cytiva) on an Äkta Avant or Äkta go system (Cytiva)34.The Abl KD and SH2-KD proteins in pET21d vectors containing a C-terminal His6 tag were each co-expressed with YopH phosphatase in TB medium and purification was performed as described previously using nickel-affinity chromatography (His-Trap FF crude, Cytiva) and anion exchange chromatography (Mono Q 5/50 GL, Cytiva) on an Äkta Avant system (Cytiva)13,52.All recombinantly expressed monobodies were produced with an N-terminal tag containing His10, FLAG and a TEV recognition motif. Purification was carried out on Ni-NTA Agarose beads (Qiagen) with 1 mL bead volume by gravity flow and subsequent SEC purification (HiLoad 16/600 Superdex 75 pg, Cytiva) into 20 mM HEPES, 150 mM NaCl, 0.5 mM TCEP-HCl on an Äkta Avant or Äkta go system (Cytiva). The D83A/E85A mutation of DAM21 was introduced using the QuikChange II Site-Directed Mutagenesis Kit (Agilent) with 5’-ctgactactacgcgggtgcgatctcttggtac-3’ and 5’-gtaccaagagatcgcacccgcgtagtagtcag-3’ as primers. The XTEN peptide extension of DAM27 was introduced through standard Gibson assembly cloning (NEB) using a DAM27 fragment containing the XTEN peptide ordered from Twist Bioscience.Biotinylation of recombinantly expressed proteinsRecombinantly expressed monobodies were biotinylated using two approaches: Either the monobody contained an Avi tag (-GLNDIFEAQKIEWHE-) and was biotinylated with BirA in a reaction mixture containing 952 µL 100 µM monobody in PBS, pH 7.4, 5 µL 1 M MgCl2, 20 µL 100 mM ATP, 20 µL 50 µM BirA, 3 µL 50 mM d-biotin. The reaction proceeded first for 3 h at 30 °C, then fresh BirA and d-biotin were added and incubated for 3 h at 30 °C. The reaction mixture was purified via SEC on a Superdex 75 Increase 10/300 GL column. Another biotinylation approach included coupling of biotin-maleimide to a cysteine residue of the monobody. Here, the reaction mixture contained 1 mL 70 µM monobody in PBS, pH 7.4 and 7 µL 100 mM biotin-maleimide (B1267-25MG, Sigma-Aldrich) in DMSO. After 2 h at rt, excess biotin-maleimide was removed and the storage buffer was exchanged using 5 mL Pierce Dye & Biotin Removal Spin Columns (Thermo Fisher Scientific) according to the manufacturer’s instructions.Competitive fluorescence polarization (FP) assayA pY-peptide known to bind SH2 domains of Src family kinases with the sequence 5-carboxyfluoresceine-EPQpYEEIPIYLK-CONH2 ordered from Peptide Synthetics (UK) was used in d-configuration. A 100 µL solution consisting of the peptide (50 nM) mixed with synthetic d-Abl SH2 (10 µM) followed by an addition of recombinantly expressed l-monobodies DAM21, DAM26, DAM27, DAM28, DAM30.1 and DAM30.2 (20 µM) in 20 mM HEPES, 150 mM NaCl, 0.5 mM TCEP-HCl and 0.02% Triton-X, pH 7.4. For comparison, binding reactions were also set up for recombinantly expressed l-Abl SH2 (10 µM) complexed with the l-pY peptide (50 nM) and addition of recombinantly expressed l-monobodies AS25, HA4 and HA4 Y87A (20 µM) in the same buffer. Measurements were performed in FP compatible dark bottom 96-well plates (675076, Greiner Bio-One) on a M5 plate reader from Molecular Devices at room temperature. FP data was acquired using 492 nm as excitation and 525 nm as emission wavelength with a filter at 515 nm. Raw data were subtracted from measurements of peptide complexed to Abl SH2 without monobodies in solution.Isothermal titration calorimetry (ITC)Recombinant and synthetic proteins were used in ITC buffer (20 mM HEPES, pH 7.4, 150 mM NaCl, 0.5 mM TCEP-HCl). Protein concentration was measured on a NanoDrop 2000c (Thermo Scientific). ITC measurements were acquired on a MicroCal PEAQ-ITC instrument (Malvern Panalytical) and consisted of the titration of the monobody solution from the syringe in 19 steps with 0.4 μL for the first injection followed by 2.0 μL for the remaining steps with a spacing of 150 s between injections at 25 °C. The reference power, feedback and stir speed were set to 3.00 µcal/s, high and 750 rpm, respectively. Protein concentrations were used as indicated in each figure. Thermodynamic parameters were determined with the MicroCal PEAQ-ITC analysis software.X-ray crystallographyComplex formation was performed by mixing TEV-cleaved DAM21 or DAM27 (1.5 eq.) with refolded, synthetic d-Abl SH2 A198C (1 eq.) and subsequent purification of each complex by SEC on a HiLoad 16/600 Superdex 75 pg column (Cytiva) column using the Äkta Avant system (Cytiva) into 20 mM HEPES, 150 mM NaCl, 0.5 mM TCEP-HCl. The complexes were concentrated in Amicon Ultra-15 centrifuge filter tubes (MWCO: 3.5 kDa) to 7.3 mg/mL (with DAM21) and 6.9 mg/mL (with DAM27) before crystallization trials were performed in sitting drop plates by mixing 100 nL of protein with 100 nL of buffer conditions. The best diffracting crystals were obtained in 0.16 M ammonium sulfate, 0.08 M sodium acetate pH 4.6, 20% (w/v) PEG 4000, 20% (v/v) glycerol (DAM21) and 0.8 M tri-sodium citrate pH 6.5, 20% (v/v) ethylene glycol (DAM27) and frozen in 15% (v/v) glycerol in liquid nitrogen.X-ray data were collected at 100 K using the beamlines X06SA and ID23-2 of the Swiss Light Source (SLS), Villigen, Switzerland, and the European Synchrotron Radiation Facility (ESRF), Grenoble, France, respectively. After data processing by XDS79, the structures were determined by molecular replacement using PHASER80, the inverted structure of the Arg SH2 domain (PDB code: 4EIH) and AlphaFold models of the monobodies as initial search models. Manual model rebuilding was done with Coot81 prior to subsequent refinement with phenix.refine82. Overall data and refinement statistics are summarized in Table 1.Table 1Data collection and refinement statisticsDAM21•d-SH2 complex (9F01)DAM27•d-SH2 complex (9F00)SourceSLS, X06SAESRF, ID23-2Wavelength (Å)1.000030.8731Resolution range48.15–2.73 (2.81–2.73)48.96–2.91 (3.01–2.91)Space groupP 21P 65 2 2Unit cell (Å,°)76.64 62.29 116.6690 98.06 90111.33 111.33 205.7590 90 120Total reflections93,888 (4103)677,041 (67967)Unique reflections25,843 (1293)17,328 (1672)Multiplicity3.6 (3.2)39.1 (39.9)Completeness, spherical (%)a87.9 (51.1)99.7 (98.1)Completeness, ellipsoidal (%)a95.7 (88.7)n.a.Mean I/sigma(I)6.9 (2.3)15.7 (0.8)Wilson B-factor (Å2)36.7684.31R-merge0.146 (0.810)0.270 (3.634)R-meas0.171 (0.968)0.273 (3.680)R-pim0.087 (0.520)0.043 (0.577)CC1/20.987 (0.530)0.999 (0.526)Refl. used in refinement25,777 (3412)17,285 (1672)Refl. used for R-free644 (39)859 (82)R-work0.2221 (0.2370)0.2094 (0.3647)R-free0.2704 (0.2981)0.2565 (0.3794)Non-hydrogen atoms61023012 Macromolecules59722999 Ligands40 Solvent12613Protein residues768387RMSD bonds (Å)0.0070.010RMSD angles (°)1.171.34Ramachandran favored (%)96.9894.59Ramachandran allowed (%)2.752.70Ramachandran outliers (%)0.272.70Rotamer outliers (%)2.816.41Clashscore10.3414.13Average B-factor (Å2)39.6587.73 Macromolecules (Å2)41.6989.20 Ligands (Å2)38.20– Solvent (Å2)27.1168.01TLS groups84Statistics for the highest-resolution shell are shown in parentheses.aDAM21•d-SH2 dataset was corrected by STARANISO85 for anisotropic diffraction. Cut-offs used direction 0.477 a* + 0.511 b* + 0.715 c* for best diffraction (2.73 Å), −0.725 a* + 0.689 c* for worst diffraction (3.44 Å).Solid-phase peptide synthesis (SPPS), native chemical ligation (NCL) and desulfurizationPeptides were synthesized by standard Fmoc-strategy on a ResPep SLi (Intavis) parallel synthesizer using Fmoc- and sidechain-protected amino acids in either l- or d-configuration obtained from Carbolution. Native chemical ligations were generally performed in 6 M guanidine-HCl, 200 mM Na2HPO4, 100 mM MPAA and 20 mM TCEP-HCl, pH 7.0 where the N-terminal peptide with MeNbz-linker was used at 2 mM (1 eq.) and the C-terminal peptide at 1.2 eq for at least 24 h. Desulfurizations of synthetic monobodies were carried out in 6 M guanidine-HCl, 200 mM Na2HPO4, 375 mM TCEP-HCl, 150 mM MESNa, 115 eq. VA-044, pH 7.5 with the peptide at 325 µM for 16 h. Detailed synthesis protocols can be found in the Supplementary Information.Refolding of synthetic monobodiesThe polypeptides were dissolved in solubilization buffer (6 M guanidine-HCl, 500 mM arginine-HCl, 20 mM HEPES, pH 8.5, 150 mM NaCl, 0.5 mM TCEP-HCl) at a concentration of 0.5 mg/mL and transferred into a Slide-A-Lyzer G2 dialysis cassette with a cut-off of 3.5 kDa (Thermo Scientific). The solution was dialyzed against a 200-fold volume of refolding buffer 1 (500 mM arginine-HCl, 20 mM HEPES, pH 7.4, 150 mM NaCl, 0.5 mM TCEP-HCl) for 2 h at 4 °C. Then, the buffer was replaced twice for refolding buffer 2 (20 mM HEPES, pH 7.4, 150 mM NaCl, 0.5 mM TCEP-HCl) and the dialysis was carried out for another 2 and 16 h. Afterwards, the solution containing the refolded protein was purified via SEC in refolding buffer 2 on the Äkta Avant or Go system (Cytiva) with a Superdex 75 Increase 10/300 GL or HiLoad 16/600 Superdex 75 pg column (Cytiva) to check for and remove aggregates at a flow rate of 0.5 or 1 mL/min. The fractions containing the synthetic refolded and monomeric proteins were collected, concentrated, flash-frozen in liquid nitrogen and stored at −80 °C.Circular dichroism (CD) spectroscopyThe recombinant and synthetic proteins originally in HEPES buffer (20 mM HEPES, pH 7.4, 150 mM NaCl, 0.5 mM TCEP-HCl) were dialyzed three times against a 200-fold volume of phosphate-buffered saline (PBS) at pH 7.4, twice for 2 h and then for 16 h at 4 °C. Dialysis was carried out in Slide-A-Lyzer G2 dialysis cassettes (cut-off 3.5 kDa, Thermo Scientific). CD spectra were recorded in a quartz cuvette (path length: 0.1 cm, Hellma Analytics) containing 20 µg of sample in 300 µL PBS buffer (pH 7.4) on a JASCO J-815 circular dichroism spectrometer at 20 °C and a data interval of 0.1 nm. The mean residue ellipticity (MRE) was calculated according to T. E. Creighton83, secondary structure predictions were calculated with BeStSel51 and the data was plotted using the software GraphPad Prism 8.Nano differential scanning fluorimetry (nanoDSF)The thermal denaturation curves were determined by measurements of intrinsic tryptophan fluorescence. This analysis was performed using label-free, native differential scanning fluorimetry on a Prometheus NT.48 instrument (NanoTemper). Approximately 10 µL of the protein samples at a concentration of 0.07 mg/mL in PBS (pH 7.4) were loaded in Prometheus NT.48 capillaries. The tryptophan residues of the proteins were excited at 280 nm, and the fluorescence intensity was recorded at 330 and 350 nm. Excitation power was set to 50%, and the temperature of the measurement compartment increased from 20 to 95 °C at a rate of 1 °C/min. Melting temperatures (Tm) were determined by the Prometheus software through calculation of the fluorescence ratio at 330 and 350 nm and of the first derivative. The data was plotted using the software GraphPad Prism 8.Protease digest assayProteins were used at 0.5 mg/mL and incubated with 0.025 mg/mL of proteases Pepsin (dissolved in Milli Q water; 1071920001, Sigma-Aldrich) or Proteinase K (dissolved in 50 mM Tris-HCl, pH 8.0, 1 mM CaCl2; P2308-25MG, Sigma-Aldrich) (protein-to-protease ratio of 20:1) in 40 µL per reaction. Digests with Pepsin were performed in 20 mM HEPES, 150 mM NaCl, 0.5 mM TCEP-HCl at a pH of 2.0 and 37 °C. Digests with Proteinase K were performed in 50 mM Tris-HCl, pH 8.0, 1 mM CaCl2 at 37 °C. After 0, 4 and 24 h, 10 µL samples were taken for SDS-PAGE analysis.Plasma stability assayMonobody stocks were prepared at a concentration of 50 µM in PBS, pH 7.4. 60 µL of stocks were subsequently mixed at a volume ratio of 1:1 with either mouse plasma (plasma) or PBS (control) and incubated at 37 °C. At indicated time points, 20 µL samples were diluted 1:5 in PBS. 20 µL of this dilution were mixed with 10 µL of 4X Laemmli SDS buffer (400 mM DTT, 8% SDS, 200 mM Tris-HCl, pH 6.8, 40% Glycerol, 0.02% bromophenol blue) and denatured at 95 °C for 5 min. 25 µL of these samples were separated using SDS-PAGE and blotted onto a 0.2 µm Amersham Protran nitrocellulose membrane (10600004, Cytiva). Monobodies were detected using IRDye680-coupled Streptavidin (1:10,000 dilution, 926-68079, LiCOR).Radiometric kinase assayRecombinantly expressed proteins Abl KD or SH2-KD, each at 50 ng, were preincubated with the recombinant or refolded, synthetic monobodies at 5 µM for 10 min at room temperature in a volume of 10 µL. Kinase activity was determined by the addition of 25 µM ATP, 3 µCi of [γ-32P]-ATP (SRP-301, Hartmann Analytic), and 100 µM of an optimal Abl substrate sequence carrying an N-terminal Biotin (Biotin-GGEAIYAAPFKK-amide) in kinase assay buffer (20 mM Tris-HCl, pH 7.5, 10 mM MgCl2,1 mM DTT) for 15 min at room temperature in a final assay volume of 20 µL. 8 µL of each reaction terminated with 10 µL of 7.5 M Guanidine-HCl were spotted onto a SAM2 Biotin Capture membrane square (V2861, Promega) and further treated according to the instructions of the manufacturer. 32P radiation of membrane squares was measured on the scintillation counter Hidex 300 SL by counting Cerenkov radiation for 1 min per sample. Data was analyzed using GraphPad Prism 8.Flow cytometry analysis of monobodies binding to Bcr-Abl in K562 cellsK562 suspension cells were cultivated in RPMI medium supplemented with 10% FBS and Penicillin/Streptomycin and split every 2–3 days. For flow cytometry analysis, cells were taken up in PBS, pH 7.4, at a density of 5 × 105 cells/mL and fixed in 3.2% Paraformaldehyde (PFA; v/v) by addition of 200 µL PFA solution (E15710, Science Services) for every 1 mL of cell suspension and incubated for 10 min at room temperature (rt). The suspension was centrifuged (5 min, 500×g), the pellet was slowly resuspended in ice-cold methanol at a density of 2.5 × 105 cells/mL while vortexing, placed for 20 min on ice and then stored at −20 °C overnight. Then, cells were washed three times with the same amount of PBS, resuspended in FACS buffer (4% FBS in PBS, pH 7.4) at a density of 1 × 106 cells/mL and placed for 2 h on ice. Afterwards, the cell suspension was split and 5 × 105 cells were used per sample, which were incubated with 20 µL FcBlock (diluted 1:20 (v/v) in FACS buffer; BUF070, Bio-Rad) for 10 min at rt. Then, monobody solutions were added at a concentration of 2 µM in 50 µL total volume and incubated for 45 min at rt. 1 mL FACS buffer was added to each sample, the samples were centrifuged (5 min, 500×g) and the supernatant was removed. 5 µL of AlexaFluor488-coupled Streptavidin (diluted 1:1000; S32354, Thermo Fisher Scientific) in 45 µL FACS buffer was added to each sample, incubated for 45 min at rt in the dark, then mixed with 2 mL FACS buffer, centrifuged (5 min, 500×g), taken up in 200 µL PBS and placed on ice until measurement. Fluorescence intensity of samples was analyzed on a Guava easyCyte flow cytometer (Luminex). Data was analyzed using GraphPad Prism 8.K562 cell lysis, BCR::ABL1 pulldown and mass spectrometryK562 cells were harvested by centrifugation (500×g, 5 min) at a density of ~1.0 × 106 cells/mL. The cell pellet was washed with PBS, frozen in liquid nitrogen and stored at −80 °C until lysis. The cells were lysed for 5 min on ice in IP buffer (50 mM Tris-HCl, pH 7.5, 150 mM NaCl, 1% NP-40, 5 mM EDTA, 5 mM EGTA) containing additionally 50 mM NaF, 1 mM vanadate, 1 mM PMSF, 10 µg/mL TPCK and 1x protease inhibitor cocktail (Roche) and centrifuged for 10 min at 20,000×g and 4 °C. The total protein amount of the supernatant was determined by standard Bradford assay. The pulldown was performed in three biological replicates for each monobody on Streptavidin MagneSphere Paramagnetic Particles (Z5481, Promega) where 200 µL of beads were used per sample. The beads were washed twice with IP buffer, then 1.5 µg of each monobody were added to the beads and incubated for 2 h at 4 °C. After washing the beads once with IP buffer, the K562 cell lysate containing 3 mg of total protein was added to each sample and incubated for 16 h at 4 °C. The beads were washed three times with IP buffer without NP-40 (50 mM Tris-HCl, pH 7.5, 150 mM NaCl, 5 mM EDTA, 5 mM EGTA). Trypsin (0.1 µg in 50 µL 50 mM ammonium-bicarbonate buffer, pH 8.0) was added to the beads and samples were incubated at 37 °C for 45 min. Subsequently, the supernatant was transferred into fresh tubes and digested overnight at 37 °C to completeness. For the reduction of disulfide bridges 5 mM DTT was added. Samples were then incubated for 15 min at 95 °C. Subsequently, the resulting sulfhydryl groups were chemically modified by adding iodoacetamide to a final concentration of 25 mM and incubating samples for 45 min at RT in the dark. Excess iodoacetamide was quenched by the addition of 50 mM DTT and incubation for one more hour at RT. Reduced and alkylated peptides were then desalted and concentrated using Chromabond C18WP spin columns (730522, Macherey-Nagel) according to manufacturer protocols. Finally, peptides were dissolved in 20 µL of water with 5% acetonitrile and 0.1% formic acid. The mass spectrometric analysis of the samples was performed using a timsTOF Pro mass spectrometer (Bruker Daltonic). A nanoElute HPLC system (Bruker Daltonics), equipped with an Aurora C18 RP column (25 cm × 75 µm ID) filled with 1.7 µm beads (IonOpticks, Australia) was connected online to the mass spectrometer. A portion of 2 µL of the peptide solution was injected directly on the separation column. Sample loading was performed at a constant pressure of 800 bar. Separation of the tryptic peptides was achieved at 60 °C column temperature with the following gradient of water/0.1% formic acid (solvent A) and acetonitrile/0.1% formic acid (solvent B) at a flow rate of 400 nL/min: Linear increase from 2% B to 17% B within 18 min, followed by a linear gradient to 25% B within 9 min and linear increase to 37% solvent B in additional 3 min. Finally, B was increased to 95% within 10 min and held at 95% for additional 10 min. The built-in “DDA PASEF-standard_1.1sec_cycletime” method developed by Bruker Daltonics was used for mass spectrometric measurement. Data analysis was performed using MaxQuant 2.5.1.0 (MPI of Biochemistry, Germany) and statistical analysis of replicates was done with Autonomics (R package version 1.13.21)84.Reporting summaryFurther information on research design is available in the Nature Portfolio Reporting Summary linked to this article.

Antibodies, cell lines and reagentsThe following antibodies, secondary antibodies and fluorophore-coupled streptavidin were used for monobody selection and yeast display binding assays: mouse anti-V5 (Thermo Fisher Scientific, MA5-15253) and FITC-coupled anti-mouse IgG (Sigma-Aldrich, F0257-5ML), streptavidin-DyLight650 (Thermo Fisher Scientific, 84547). AlexaFluor488-coupled Streptavidin (S32354) used for detection of monobodies binding to Bcr-Abl in cells was purchased from Thermo Fisher Scientific. For Western blots of plasma stability assays IRDye 680-Streptavidin (926-68079) was purchased from LiCOR and used at 1:10,000 dilution in 5% Milk in Phosphate-Buffered Saline (PBS-T). Non-sterile mouse plasma with sodium heparin (ABIN925342) was purchased from antibodies-online/Rockland Immunochemicals. Streptavidin MagneSphere Paramagnetic Particles (Z5481, Promega) were used during monobody selection. K562 cells were purchased from DSMZ (ACC-10, Deutsche Sammlung von Mikroorganismen und Zellkulturen, Braunschweig, Germany). Further reagents used are listed in the Supplementary Information.

Antibodies, cell lines and reagents

The following antibodies, secondary antibodies and fluorophore-coupled streptavidin were used for monobody selection and yeast display binding assays: mouse anti-V5 (Thermo Fisher Scientific, MA5-15253) and FITC-coupled anti-mouse IgG (Sigma-Aldrich, F0257-5ML), streptavidin-DyLight650 (Thermo Fisher Scientific, 84547). AlexaFluor488-coupled Streptavidin (S32354) used for detection of monobodies binding to Bcr-Abl in cells was purchased from Thermo Fisher Scientific. For Western blots of plasma stability assays IRDye 680-Streptavidin (926-68079) was purchased from LiCOR and used at 1:10,000 dilution in 5% Milk in Phosphate-Buffered Saline (PBS-T). Non-sterile mouse plasma with sodium heparin (ABIN925342) was purchased from antibodies-online/Rockland Immunochemicals. Streptavidin MagneSphere Paramagnetic Particles (Z5481, Promega) were used during monobody selection. K562 cells were purchased from DSMZ (ACC-10, Deutsche Sammlung von Mikroorganismen und Zellkulturen, Braunschweig, Germany). Further reagents used are listed in the Supplementary Information.

Monobody selectionMonobodies were selected according to methods previously described using biotinylated target proteins7,78. Briefly, four rounds of phage display were followed by the amplification and transformation of EBY100 yeast cells with the DNA sequences corresponding to binding monobodies. The yeast cells were next sorted using FACS based on a strict gating strategy comprising double positive cells for monobody display and binding to biotinylated target. Isolated clones were next sequenced and cloned into the pHFT2 vector, digested with BamHI and XhoI, by a standard Gibson assembly protocol (NEB) for further characterization. Here, the monobody fragments were amplified from the isolated yeast plasmids via PCR using 5’-gtgaaaacctgtacttccagggatccatggctgcttcttctg-3’ and 5’- cagtggtggtggtggtggtgctcgagctaggtacggtagttaatc-3’ as primers. A detailed description of the selection procedure can be found in the Supplementary Information.

Monobody selection

Monobodies were selected according to methods previously described using biotinylated target proteins7,78. Briefly, four rounds of phage display were followed by the amplification and transformation of EBY100 yeast cells with the DNA sequences corresponding to binding monobodies. The yeast cells were next sorted using FACS based on a strict gating strategy comprising double positive cells for monobody display and binding to biotinylated target. Isolated clones were next sequenced and cloned into the pHFT2 vector, digested with BamHI and XhoI, by a standard Gibson assembly protocol (NEB) for further characterization. Here, the monobody fragments were amplified from the isolated yeast plasmids via PCR using 5’-gtgaaaacctgtacttccagggatccatggctgcttcttctg-3’ and 5’- cagtggtggtggtggtggtgctcgagctaggtacggtagttaatc-3’ as primers. A detailed description of the selection procedure can be found in the Supplementary Information.

Binding assay in yeast display formatIncreasing concentrations of synthetic biotinylated protein target d-Abl SH2 resuspended in Tris-buffered saline (TBS) with 0.1% BSA were incubated with 5 × 105 yeast cells per sample displaying monobodies and a mouse anti-V5 antibody (5 µL of 1:300 dilution) in a total volume of 20 µL for 30 min at room temperature. Three washes with TBS with 0.1% BSA followed together with incubation for 30 min with a streptavidin-DyLight650 and a FITC-coupled anti-mouse IgG (both diluted 1:1000 in the same tube and 20 µL used per sample) at room temperature in the dark. Samples were washed twice with Tris-buffered saline (TBS) with 0.1% BSA and analyzed on a Guava easyCyte flow cytometer (Luminex). Data were fitted on a 1:1 binding model using the Prism software (GraphPad) to determine Kd values.

Binding assay in yeast display format

Increasing concentrations of synthetic biotinylated protein target d-Abl SH2 resuspended in Tris-buffered saline (TBS) with 0.1% BSA were incubated with 5 × 105 yeast cells per sample displaying monobodies and a mouse anti-V5 antibody (5 µL of 1:300 dilution) in a total volume of 20 µL for 30 min at room temperature. Three washes with TBS with 0.1% BSA followed together with incubation for 30 min with a streptavidin-DyLight650 and a FITC-coupled anti-mouse IgG (both diluted 1:1000 in the same tube and 20 µL used per sample) at room temperature in the dark. Samples were washed twice with Tris-buffered saline (TBS) with 0.1% BSA and analyzed on a Guava easyCyte flow cytometer (Luminex). Data were fitted on a 1:1 binding model using the Prism software (GraphPad) to determine Kd values.

Recombinant protein expressionGenerally, E. coli BL21 Star (DE3) cells were transformed with plasmids containing the desired proteins. An overnight pre-culture was added to medium (1:20, v/v) and cells were grown at 37 °C, 200 rpm. For expressions in lysogeny broth (LB; until OD600 0.5–0.8) and terrific broth (TB; until OD600 1.2) medium, the culture was next induced with 0.5 mM IPTG final concentration and cells were further grown overnight at 18 °C, 200 rpm. Expressions in auto induction (AI) medium were placed at 18 °C without IPTG addition. The next day, cells were harvested at 6000×g for 10 min at 4 °C.The Abl SH2, Btk SH2 and Lck SH2 domains were produced in LB medium with an N-terminal tag including a His6 and Glutathione S-transferase (GST), and tobacco etch virus (TEV) protease recognition motifs using the pETM30 vector. The Abl SH2 A198C mutation was introduced using the QuikChange II Site-Directed Mutagenesis Kit (Agilent) with 5’-cattacaggatcaacacttgctctgatggcaagctc-3’ and 5’-gagcttgccatcagagcaagtgttgatcctgtaatg-3’ as primers. Protein purification and tag cleavage was performed as described previously by nickel-affinity chromatography (His-Trap FF crude, Cytiva) and SEC (HiLoad 16/600 Superdex 75 pg, Cytiva) on an Äkta Avant or Äkta go system (Cytiva)34.The Abl KD and SH2-KD proteins in pET21d vectors containing a C-terminal His6 tag were each co-expressed with YopH phosphatase in TB medium and purification was performed as described previously using nickel-affinity chromatography (His-Trap FF crude, Cytiva) and anion exchange chromatography (Mono Q 5/50 GL, Cytiva) on an Äkta Avant system (Cytiva)13,52.All recombinantly expressed monobodies were produced with an N-terminal tag containing His10, FLAG and a TEV recognition motif. Purification was carried out on Ni-NTA Agarose beads (Qiagen) with 1 mL bead volume by gravity flow and subsequent SEC purification (HiLoad 16/600 Superdex 75 pg, Cytiva) into 20 mM HEPES, 150 mM NaCl, 0.5 mM TCEP-HCl on an Äkta Avant or Äkta go system (Cytiva). The D83A/E85A mutation of DAM21 was introduced using the QuikChange II Site-Directed Mutagenesis Kit (Agilent) with 5’-ctgactactacgcgggtgcgatctcttggtac-3’ and 5’-gtaccaagagatcgcacccgcgtagtagtcag-3’ as primers. The XTEN peptide extension of DAM27 was introduced through standard Gibson assembly cloning (NEB) using a DAM27 fragment containing the XTEN peptide ordered from Twist Bioscience.

Recombinant protein expression

Generally, E. coli BL21 Star (DE3) cells were transformed with plasmids containing the desired proteins. An overnight pre-culture was added to medium (1:20, v/v) and cells were grown at 37 °C, 200 rpm. For expressions in lysogeny broth (LB; until OD600 0.5–0.8) and terrific broth (TB; until OD600 1.2) medium, the culture was next induced with 0.5 mM IPTG final concentration and cells were further grown overnight at 18 °C, 200 rpm. Expressions in auto induction (AI) medium were placed at 18 °C without IPTG addition. The next day, cells were harvested at 6000×g for 10 min at 4 °C.

The Abl SH2, Btk SH2 and Lck SH2 domains were produced in LB medium with an N-terminal tag including a His6 and Glutathione S-transferase (GST), and tobacco etch virus (TEV) protease recognition motifs using the pETM30 vector. The Abl SH2 A198C mutation was introduced using the QuikChange II Site-Directed Mutagenesis Kit (Agilent) with 5’-cattacaggatcaacacttgctctgatggcaagctc-3’ and 5’-gagcttgccatcagagcaagtgttgatcctgtaatg-3’ as primers. Protein purification and tag cleavage was performed as described previously by nickel-affinity chromatography (His-Trap FF crude, Cytiva) and SEC (HiLoad 16/600 Superdex 75 pg, Cytiva) on an Äkta Avant or Äkta go system (Cytiva)34.

The Abl KD and SH2-KD proteins in pET21d vectors containing a C-terminal His6 tag were each co-expressed with YopH phosphatase in TB medium and purification was performed as described previously using nickel-affinity chromatography (His-Trap FF crude, Cytiva) and anion exchange chromatography (Mono Q 5/50 GL, Cytiva) on an Äkta Avant system (Cytiva)13,52.

All recombinantly expressed monobodies were produced with an N-terminal tag containing His10, FLAG and a TEV recognition motif. Purification was carried out on Ni-NTA Agarose beads (Qiagen) with 1 mL bead volume by gravity flow and subsequent SEC purification (HiLoad 16/600 Superdex 75 pg, Cytiva) into 20 mM HEPES, 150 mM NaCl, 0.5 mM TCEP-HCl on an Äkta Avant or Äkta go system (Cytiva). The D83A/E85A mutation of DAM21 was introduced using the QuikChange II Site-Directed Mutagenesis Kit (Agilent) with 5’-ctgactactacgcgggtgcgatctcttggtac-3’ and 5’-gtaccaagagatcgcacccgcgtagtagtcag-3’ as primers. The XTEN peptide extension of DAM27 was introduced through standard Gibson assembly cloning (NEB) using a DAM27 fragment containing the XTEN peptide ordered from Twist Bioscience.

Biotinylation of recombinantly expressed proteinsRecombinantly expressed monobodies were biotinylated using two approaches: Either the monobody contained an Avi tag (-GLNDIFEAQKIEWHE-) and was biotinylated with BirA in a reaction mixture containing 952 µL 100 µM monobody in PBS, pH 7.4, 5 µL 1 M MgCl2, 20 µL 100 mM ATP, 20 µL 50 µM BirA, 3 µL 50 mM d-biotin. The reaction proceeded first for 3 h at 30 °C, then fresh BirA and d-biotin were added and incubated for 3 h at 30 °C. The reaction mixture was purified via SEC on a Superdex 75 Increase 10/300 GL column. Another biotinylation approach included coupling of biotin-maleimide to a cysteine residue of the monobody. Here, the reaction mixture contained 1 mL 70 µM monobody in PBS, pH 7.4 and 7 µL 100 mM biotin-maleimide (B1267-25MG, Sigma-Aldrich) in DMSO. After 2 h at rt, excess biotin-maleimide was removed and the storage buffer was exchanged using 5 mL Pierce Dye & Biotin Removal Spin Columns (Thermo Fisher Scientific) according to the manufacturer’s instructions.

Biotinylation of recombinantly expressed proteins

Recombinantly expressed monobodies were biotinylated using two approaches: Either the monobody contained an Avi tag (-GLNDIFEAQKIEWHE-) and was biotinylated with BirA in a reaction mixture containing 952 µL 100 µM monobody in PBS, pH 7.4, 5 µL 1 M MgCl2, 20 µL 100 mM ATP, 20 µL 50 µM BirA, 3 µL 50 mM d-biotin. The reaction proceeded first for 3 h at 30 °C, then fresh BirA and d-biotin were added and incubated for 3 h at 30 °C. The reaction mixture was purified via SEC on a Superdex 75 Increase 10/300 GL column. Another biotinylation approach included coupling of biotin-maleimide to a cysteine residue of the monobody. Here, the reaction mixture contained 1 mL 70 µM monobody in PBS, pH 7.4 and 7 µL 100 mM biotin-maleimide (B1267-25MG, Sigma-Aldrich) in DMSO. After 2 h at rt, excess biotin-maleimide was removed and the storage buffer was exchanged using 5 mL Pierce Dye & Biotin Removal Spin Columns (Thermo Fisher Scientific) according to the manufacturer’s instructions.

Competitive fluorescence polarization (FP) assayA pY-peptide known to bind SH2 domains of Src family kinases with the sequence 5-carboxyfluoresceine-EPQpYEEIPIYLK-CONH2 ordered from Peptide Synthetics (UK) was used in d-configuration. A 100 µL solution consisting of the peptide (50 nM) mixed with synthetic d-Abl SH2 (10 µM) followed by an addition of recombinantly expressed l-monobodies DAM21, DAM26, DAM27, DAM28, DAM30.1 and DAM30.2 (20 µM) in 20 mM HEPES, 150 mM NaCl, 0.5 mM TCEP-HCl and 0.02% Triton-X, pH 7.4. For comparison, binding reactions were also set up for recombinantly expressed l-Abl SH2 (10 µM) complexed with the l-pY peptide (50 nM) and addition of recombinantly expressed l-monobodies AS25, HA4 and HA4 Y87A (20 µM) in the same buffer. Measurements were performed in FP compatible dark bottom 96-well plates (675076, Greiner Bio-One) on a M5 plate reader from Molecular Devices at room temperature. FP data was acquired using 492 nm as excitation and 525 nm as emission wavelength with a filter at 515 nm. Raw data were subtracted from measurements of peptide complexed to Abl SH2 without monobodies in solution.

Competitive fluorescence polarization (FP) assay

A pY-peptide known to bind SH2 domains of Src family kinases with the sequence 5-carboxyfluoresceine-EPQpYEEIPIYLK-CONH2 ordered from Peptide Synthetics (UK) was used in d-configuration. A 100 µL solution consisting of the peptide (50 nM) mixed with synthetic d-Abl SH2 (10 µM) followed by an addition of recombinantly expressed l-monobodies DAM21, DAM26, DAM27, DAM28, DAM30.1 and DAM30.2 (20 µM) in 20 mM HEPES, 150 mM NaCl, 0.5 mM TCEP-HCl and 0.02% Triton-X, pH 7.4. For comparison, binding reactions were also set up for recombinantly expressed l-Abl SH2 (10 µM) complexed with the l-pY peptide (50 nM) and addition of recombinantly expressed l-monobodies AS25, HA4 and HA4 Y87A (20 µM) in the same buffer. Measurements were performed in FP compatible dark bottom 96-well plates (675076, Greiner Bio-One) on a M5 plate reader from Molecular Devices at room temperature. FP data was acquired using 492 nm as excitation and 525 nm as emission wavelength with a filter at 515 nm. Raw data were subtracted from measurements of peptide complexed to Abl SH2 without monobodies in solution.

Isothermal titration calorimetry (ITC)Recombinant and synthetic proteins were used in ITC buffer (20 mM HEPES, pH 7.4, 150 mM NaCl, 0.5 mM TCEP-HCl). Protein concentration was measured on a NanoDrop 2000c (Thermo Scientific). ITC measurements were acquired on a MicroCal PEAQ-ITC instrument (Malvern Panalytical) and consisted of the titration of the monobody solution from the syringe in 19 steps with 0.4 μL for the first injection followed by 2.0 μL for the remaining steps with a spacing of 150 s between injections at 25 °C. The reference power, feedback and stir speed were set to 3.00 µcal/s, high and 750 rpm, respectively. Protein concentrations were used as indicated in each figure. Thermodynamic parameters were determined with the MicroCal PEAQ-ITC analysis software.

Isothermal titration calorimetry (ITC)

Recombinant and synthetic proteins were used in ITC buffer (20 mM HEPES, pH 7.4, 150 mM NaCl, 0.5 mM TCEP-HCl). Protein concentration was measured on a NanoDrop 2000c (Thermo Scientific). ITC measurements were acquired on a MicroCal PEAQ-ITC instrument (Malvern Panalytical) and consisted of the titration of the monobody solution from the syringe in 19 steps with 0.4 μL for the first injection followed by 2.0 μL for the remaining steps with a spacing of 150 s between injections at 25 °C. The reference power, feedback and stir speed were set to 3.00 µcal/s, high and 750 rpm, respectively. Protein concentrations were used as indicated in each figure. Thermodynamic parameters were determined with the MicroCal PEAQ-ITC analysis software.

X-ray crystallographyComplex formation was performed by mixing TEV-cleaved DAM21 or DAM27 (1.5 eq.) with refolded, synthetic d-Abl SH2 A198C (1 eq.) and subsequent purification of each complex by SEC on a HiLoad 16/600 Superdex 75 pg column (Cytiva) column using the Äkta Avant system (Cytiva) into 20 mM HEPES, 150 mM NaCl, 0.5 mM TCEP-HCl. The complexes were concentrated in Amicon Ultra-15 centrifuge filter tubes (MWCO: 3.5 kDa) to 7.3 mg/mL (with DAM21) and 6.9 mg/mL (with DAM27) before crystallization trials were performed in sitting drop plates by mixing 100 nL of protein with 100 nL of buffer conditions. The best diffracting crystals were obtained in 0.16 M ammonium sulfate, 0.08 M sodium acetate pH 4.6, 20% (w/v) PEG 4000, 20% (v/v) glycerol (DAM21) and 0.8 M tri-sodium citrate pH 6.5, 20% (v/v) ethylene glycol (DAM27) and frozen in 15% (v/v) glycerol in liquid nitrogen.X-ray data were collected at 100 K using the beamlines X06SA and ID23-2 of the Swiss Light Source (SLS), Villigen, Switzerland, and the European Synchrotron Radiation Facility (ESRF), Grenoble, France, respectively. After data processing by XDS79, the structures were determined by molecular replacement using PHASER80, the inverted structure of the Arg SH2 domain (PDB code: 4EIH) and AlphaFold models of the monobodies as initial search models. Manual model rebuilding was done with Coot81 prior to subsequent refinement with phenix.refine82. Overall data and refinement statistics are summarized in Table 1.Table 1Data collection and refinement statisticsDAM21•d-SH2 complex (9F01)DAM27•d-SH2 complex (9F00)SourceSLS, X06SAESRF, ID23-2Wavelength (Å)1.000030.8731Resolution range48.15–2.73 (2.81–2.73)48.96–2.91 (3.01–2.91)Space groupP 21P 65 2 2Unit cell (Å,°)76.64 62.29 116.6690 98.06 90111.33 111.33 205.7590 90 120Total reflections93,888 (4103)677,041 (67967)Unique reflections25,843 (1293)17,328 (1672)Multiplicity3.6 (3.2)39.1 (39.9)Completeness, spherical (%)a87.9 (51.1)99.7 (98.1)Completeness, ellipsoidal (%)a95.7 (88.7)n.a.Mean I/sigma(I)6.9 (2.3)15.7 (0.8)Wilson B-factor (Å2)36.7684.31R-merge0.146 (0.810)0.270 (3.634)R-meas0.171 (0.968)0.273 (3.680)R-pim0.087 (0.520)0.043 (0.577)CC1/20.987 (0.530)0.999 (0.526)Refl. used in refinement25,777 (3412)17,285 (1672)Refl. used for R-free644 (39)859 (82)R-work0.2221 (0.2370)0.2094 (0.3647)R-free0.2704 (0.2981)0.2565 (0.3794)Non-hydrogen atoms61023012 Macromolecules59722999 Ligands40 Solvent12613Protein residues768387RMSD bonds (Å)0.0070.010RMSD angles (°)1.171.34Ramachandran favored (%)96.9894.59Ramachandran allowed (%)2.752.70Ramachandran outliers (%)0.272.70Rotamer outliers (%)2.816.41Clashscore10.3414.13Average B-factor (Å2)39.6587.73 Macromolecules (Å2)41.6989.20 Ligands (Å2)38.20– Solvent (Å2)27.1168.01TLS groups84Statistics for the highest-resolution shell are shown in parentheses.aDAM21•d-SH2 dataset was corrected by STARANISO85 for anisotropic diffraction. Cut-offs used direction 0.477 a* + 0.511 b* + 0.715 c* for best diffraction (2.73 Å), −0.725 a* + 0.689 c* for worst diffraction (3.44 Å).

X-ray crystallography

Complex formation was performed by mixing TEV-cleaved DAM21 or DAM27 (1.5 eq.) with refolded, synthetic d-Abl SH2 A198C (1 eq.) and subsequent purification of each complex by SEC on a HiLoad 16/600 Superdex 75 pg column (Cytiva) column using the Äkta Avant system (Cytiva) into 20 mM HEPES, 150 mM NaCl, 0.5 mM TCEP-HCl. The complexes were concentrated in Amicon Ultra-15 centrifuge filter tubes (MWCO: 3.5 kDa) to 7.3 mg/mL (with DAM21) and 6.9 mg/mL (with DAM27) before crystallization trials were performed in sitting drop plates by mixing 100 nL of protein with 100 nL of buffer conditions. The best diffracting crystals were obtained in 0.16 M ammonium sulfate, 0.08 M sodium acetate pH 4.6, 20% (w/v) PEG 4000, 20% (v/v) glycerol (DAM21) and 0.8 M tri-sodium citrate pH 6.5, 20% (v/v) ethylene glycol (DAM27) and frozen in 15% (v/v) glycerol in liquid nitrogen.

X-ray data were collected at 100 K using the beamlines X06SA and ID23-2 of the Swiss Light Source (SLS), Villigen, Switzerland, and the European Synchrotron Radiation Facility (ESRF), Grenoble, France, respectively. After data processing by XDS79, the structures were determined by molecular replacement using PHASER80, the inverted structure of the Arg SH2 domain (PDB code: 4EIH) and AlphaFold models of the monobodies as initial search models. Manual model rebuilding was done with Coot81 prior to subsequent refinement with phenix.refine82. Overall data and refinement statistics are summarized in Table 1.Table 1Data collection and refinement statisticsDAM21•d-SH2 complex (9F01)DAM27•d-SH2 complex (9F00)SourceSLS, X06SAESRF, ID23-2Wavelength (Å)1.000030.8731Resolution range48.15–2.73 (2.81–2.73)48.96–2.91 (3.01–2.91)Space groupP 21P 65 2 2Unit cell (Å,°)76.64 62.29 116.6690 98.06 90111.33 111.33 205.7590 90 120Total reflections93,888 (4103)677,041 (67967)Unique reflections25,843 (1293)17,328 (1672)Multiplicity3.6 (3.2)39.1 (39.9)Completeness, spherical (%)a87.9 (51.1)99.7 (98.1)Completeness, ellipsoidal (%)a95.7 (88.7)n.a.Mean I/sigma(I)6.9 (2.3)15.7 (0.8)Wilson B-factor (Å2)36.7684.31R-merge0.146 (0.810)0.270 (3.634)R-meas0.171 (0.968)0.273 (3.680)R-pim0.087 (0.520)0.043 (0.577)CC1/20.987 (0.530)0.999 (0.526)Refl. used in refinement25,777 (3412)17,285 (1672)Refl. used for R-free644 (39)859 (82)R-work0.2221 (0.2370)0.2094 (0.3647)R-free0.2704 (0.2981)0.2565 (0.3794)Non-hydrogen atoms61023012 Macromolecules59722999 Ligands40 Solvent12613Protein residues768387RMSD bonds (Å)0.0070.010RMSD angles (°)1.171.34Ramachandran favored (%)96.9894.59Ramachandran allowed (%)2.752.70Ramachandran outliers (%)0.272.70Rotamer outliers (%)2.816.41Clashscore10.3414.13Average B-factor (Å2)39.6587.73 Macromolecules (Å2)41.6989.20 Ligands (Å2)38.20– Solvent (Å2)27.1168.01TLS groups84Statistics for the highest-resolution shell are shown in parentheses.aDAM21•d-SH2 dataset was corrected by STARANISO85 for anisotropic diffraction. Cut-offs used direction 0.477 a* + 0.511 b* + 0.715 c* for best diffraction (2.73 Å), −0.725 a* + 0.689 c* for worst diffraction (3.44 Å).

Data collection and refinement statistics

76.64 62.29 116.66

90 98.06 90

111.33 111.33 205.75

Statistics for the highest-resolution shell are shown in parentheses.

aDAM21•d-SH2 dataset was corrected by STARANISO85 for anisotropic diffraction. Cut-offs used direction 0.477 a* + 0.511 b* + 0.715 c* for best diffraction (2.73 Å), −0.725 a* + 0.689 c* for worst diffraction (3.44 Å).

Solid-phase peptide synthesis (SPPS), native chemical ligation (NCL) and desulfurizationPeptides were synthesized by standard Fmoc-strategy on a ResPep SLi (Intavis) parallel synthesizer using Fmoc- and sidechain-protected amino acids in either l- or d-configuration obtained from Carbolution. Native chemical ligations were generally performed in 6 M guanidine-HCl, 200 mM Na2HPO4, 100 mM MPAA and 20 mM TCEP-HCl, pH 7.0 where the N-terminal peptide with MeNbz-linker was used at 2 mM (1 eq.) and the C-terminal peptide at 1.2 eq for at least 24 h. Desulfurizations of synthetic monobodies were carried out in 6 M guanidine-HCl, 200 mM Na2HPO4, 375 mM TCEP-HCl, 150 mM MESNa, 115 eq. VA-044, pH 7.5 with the peptide at 325 µM for 16 h. Detailed synthesis protocols can be found in the Supplementary Information.

Solid-phase peptide synthesis (SPPS), native chemical ligation (NCL) and desulfurization

Peptides were synthesized by standard Fmoc-strategy on a ResPep SLi (Intavis) parallel synthesizer using Fmoc- and sidechain-protected amino acids in either l- or d-configuration obtained from Carbolution. Native chemical ligations were generally performed in 6 M guanidine-HCl, 200 mM Na2HPO4, 100 mM MPAA and 20 mM TCEP-HCl, pH 7.0 where the N-terminal peptide with MeNbz-linker was used at 2 mM (1 eq.) and the C-terminal peptide at 1.2 eq for at least 24 h. Desulfurizations of synthetic monobodies were carried out in 6 M guanidine-HCl, 200 mM Na2HPO4, 375 mM TCEP-HCl, 150 mM MESNa, 115 eq. VA-044, pH 7.5 with the peptide at 325 µM for 16 h. Detailed synthesis protocols can be found in the Supplementary Information.

Refolding of synthetic monobodiesThe polypeptides were dissolved in solubilization buffer (6 M guanidine-HCl, 500 mM arginine-HCl, 20 mM HEPES, pH 8.5, 150 mM NaCl, 0.5 mM TCEP-HCl) at a concentration of 0.5 mg/mL and transferred into a Slide-A-Lyzer G2 dialysis cassette with a cut-off of 3.5 kDa (Thermo Scientific). The solution was dialyzed against a 200-fold volume of refolding buffer 1 (500 mM arginine-HCl, 20 mM HEPES, pH 7.4, 150 mM NaCl, 0.5 mM TCEP-HCl) for 2 h at 4 °C. Then, the buffer was replaced twice for refolding buffer 2 (20 mM HEPES, pH 7.4, 150 mM NaCl, 0.5 mM TCEP-HCl) and the dialysis was carried out for another 2 and 16 h. Afterwards, the solution containing the refolded protein was purified via SEC in refolding buffer 2 on the Äkta Avant or Go system (Cytiva) with a Superdex 75 Increase 10/300 GL or HiLoad 16/600 Superdex 75 pg column (Cytiva) to check for and remove aggregates at a flow rate of 0.5 or 1 mL/min. The fractions containing the synthetic refolded and monomeric proteins were collected, concentrated, flash-frozen in liquid nitrogen and stored at −80 °C.

Refolding of synthetic monobodies

The polypeptides were dissolved in solubilization buffer (6 M guanidine-HCl, 500 mM arginine-HCl, 20 mM HEPES, pH 8.5, 150 mM NaCl, 0.5 mM TCEP-HCl) at a concentration of 0.5 mg/mL and transferred into a Slide-A-Lyzer G2 dialysis cassette with a cut-off of 3.5 kDa (Thermo Scientific). The solution was dialyzed against a 200-fold volume of refolding buffer 1 (500 mM arginine-HCl, 20 mM HEPES, pH 7.4, 150 mM NaCl, 0.5 mM TCEP-HCl) for 2 h at 4 °C. Then, the buffer was replaced twice for refolding buffer 2 (20 mM HEPES, pH 7.4, 150 mM NaCl, 0.5 mM TCEP-HCl) and the dialysis was carried out for another 2 and 16 h. Afterwards, the solution containing the refolded protein was purified via SEC in refolding buffer 2 on the Äkta Avant or Go system (Cytiva) with a Superdex 75 Increase 10/300 GL or HiLoad 16/600 Superdex 75 pg column (Cytiva) to check for and remove aggregates at a flow rate of 0.5 or 1 mL/min. The fractions containing the synthetic refolded and monomeric proteins were collected, concentrated, flash-frozen in liquid nitrogen and stored at −80 °C.

Circular dichroism (CD) spectroscopyThe recombinant and synthetic proteins originally in HEPES buffer (20 mM HEPES, pH 7.4, 150 mM NaCl, 0.5 mM TCEP-HCl) were dialyzed three times against a 200-fold volume of phosphate-buffered saline (PBS) at pH 7.4, twice for 2 h and then for 16 h at 4 °C. Dialysis was carried out in Slide-A-Lyzer G2 dialysis cassettes (cut-off 3.5 kDa, Thermo Scientific). CD spectra were recorded in a quartz cuvette (path length: 0.1 cm, Hellma Analytics) containing 20 µg of sample in 300 µL PBS buffer (pH 7.4) on a JASCO J-815 circular dichroism spectrometer at 20 °C and a data interval of 0.1 nm. The mean residue ellipticity (MRE) was calculated according to T. E. Creighton83, secondary structure predictions were calculated with BeStSel51 and the data was plotted using the software GraphPad Prism 8.

Circular dichroism (CD) spectroscopy

The recombinant and synthetic proteins originally in HEPES buffer (20 mM HEPES, pH 7.4, 150 mM NaCl, 0.5 mM TCEP-HCl) were dialyzed three times against a 200-fold volume of phosphate-buffered saline (PBS) at pH 7.4, twice for 2 h and then for 16 h at 4 °C. Dialysis was carried out in Slide-A-Lyzer G2 dialysis cassettes (cut-off 3.5 kDa, Thermo Scientific). CD spectra were recorded in a quartz cuvette (path length: 0.1 cm, Hellma Analytics) containing 20 µg of sample in 300 µL PBS buffer (pH 7.4) on a JASCO J-815 circular dichroism spectrometer at 20 °C and a data interval of 0.1 nm. The mean residue ellipticity (MRE) was calculated according to T. E. Creighton83, secondary structure predictions were calculated with BeStSel51 and the data was plotted using the software GraphPad Prism 8.

Nano differential scanning fluorimetry (nanoDSF)The thermal denaturation curves were determined by measurements of intrinsic tryptophan fluorescence. This analysis was performed using label-free, native differential scanning fluorimetry on a Prometheus NT.48 instrument (NanoTemper). Approximately 10 µL of the protein samples at a concentration of 0.07 mg/mL in PBS (pH 7.4) were loaded in Prometheus NT.48 capillaries. The tryptophan residues of the proteins were excited at 280 nm, and the fluorescence intensity was recorded at 330 and 350 nm. Excitation power was set to 50%, and the temperature of the measurement compartment increased from 20 to 95 °C at a rate of 1 °C/min. Melting temperatures (Tm) were determined by the Prometheus software through calculation of the fluorescence ratio at 330 and 350 nm and of the first derivative. The data was plotted using the software GraphPad Prism 8.

Nano differential scanning fluorimetry (nanoDSF)

The thermal denaturation curves were determined by measurements of intrinsic tryptophan fluorescence. This analysis was performed using label-free, native differential scanning fluorimetry on a Prometheus NT.48 instrument (NanoTemper). Approximately 10 µL of the protein samples at a concentration of 0.07 mg/mL in PBS (pH 7.4) were loaded in Prometheus NT.48 capillaries. The tryptophan residues of the proteins were excited at 280 nm, and the fluorescence intensity was recorded at 330 and 350 nm. Excitation power was set to 50%, and the temperature of the measurement compartment increased from 20 to 95 °C at a rate of 1 °C/min. Melting temperatures (Tm) were determined by the Prometheus software through calculation of the fluorescence ratio at 330 and 350 nm and of the first derivative. The data was plotted using the software GraphPad Prism 8.

Protease digest assayProteins were used at 0.5 mg/mL and incubated with 0.025 mg/mL of proteases Pepsin (dissolved in Milli Q water; 1071920001, Sigma-Aldrich) or Proteinase K (dissolved in 50 mM Tris-HCl, pH 8.0, 1 mM CaCl2; P2308-25MG, Sigma-Aldrich) (protein-to-protease ratio of 20:1) in 40 µL per reaction. Digests with Pepsin were performed in 20 mM HEPES, 150 mM NaCl, 0.5 mM TCEP-HCl at a pH of 2.0 and 37 °C. Digests with Proteinase K were performed in 50 mM Tris-HCl, pH 8.0, 1 mM CaCl2 at 37 °C. After 0, 4 and 24 h, 10 µL samples were taken for SDS-PAGE analysis.

Protease digest assay

Proteins were used at 0.5 mg/mL and incubated with 0.025 mg/mL of proteases Pepsin (dissolved in Milli Q water; 1071920001, Sigma-Aldrich) or Proteinase K (dissolved in 50 mM Tris-HCl, pH 8.0, 1 mM CaCl2; P2308-25MG, Sigma-Aldrich) (protein-to-protease ratio of 20:1) in 40 µL per reaction. Digests with Pepsin were performed in 20 mM HEPES, 150 mM NaCl, 0.5 mM TCEP-HCl at a pH of 2.0 and 37 °C. Digests with Proteinase K were performed in 50 mM Tris-HCl, pH 8.0, 1 mM CaCl2 at 37 °C. After 0, 4 and 24 h, 10 µL samples were taken for SDS-PAGE analysis.

Plasma stability assayMonobody stocks were prepared at a concentration of 50 µM in PBS, pH 7.4. 60 µL of stocks were subsequently mixed at a volume ratio of 1:1 with either mouse plasma (plasma) or PBS (control) and incubated at 37 °C. At indicated time points, 20 µL samples were diluted 1:5 in PBS. 20 µL of this dilution were mixed with 10 µL of 4X Laemmli SDS buffer (400 mM DTT, 8% SDS, 200 mM Tris-HCl, pH 6.8, 40% Glycerol, 0.02% bromophenol blue) and denatured at 95 °C for 5 min. 25 µL of these samples were separated using SDS-PAGE and blotted onto a 0.2 µm Amersham Protran nitrocellulose membrane (10600004, Cytiva). Monobodies were detected using IRDye680-coupled Streptavidin (1:10,000 dilution, 926-68079, LiCOR).

Plasma stability assay

Monobody stocks were prepared at a concentration of 50 µM in PBS, pH 7.4. 60 µL of stocks were subsequently mixed at a volume ratio of 1:1 with either mouse plasma (plasma) or PBS (control) and incubated at 37 °C. At indicated time points, 20 µL samples were diluted 1:5 in PBS. 20 µL of this dilution were mixed with 10 µL of 4X Laemmli SDS buffer (400 mM DTT, 8% SDS, 200 mM Tris-HCl, pH 6.8, 40% Glycerol, 0.02% bromophenol blue) and denatured at 95 °C for 5 min. 25 µL of these samples were separated using SDS-PAGE and blotted onto a 0.2 µm Amersham Protran nitrocellulose membrane (10600004, Cytiva). Monobodies were detected using IRDye680-coupled Streptavidin (1:10,000 dilution, 926-68079, LiCOR).

Radiometric kinase assayRecombinantly expressed proteins Abl KD or SH2-KD, each at 50 ng, were preincubated with the recombinant or refolded, synthetic monobodies at 5 µM for 10 min at room temperature in a volume of 10 µL. Kinase activity was determined by the addition of 25 µM ATP, 3 µCi of [γ-32P]-ATP (SRP-301, Hartmann Analytic), and 100 µM of an optimal Abl substrate sequence carrying an N-terminal Biotin (Biotin-GGEAIYAAPFKK-amide) in kinase assay buffer (20 mM Tris-HCl, pH 7.5, 10 mM MgCl2,1 mM DTT) for 15 min at room temperature in a final assay volume of 20 µL. 8 µL of each reaction terminated with 10 µL of 7.5 M Guanidine-HCl were spotted onto a SAM2 Biotin Capture membrane square (V2861, Promega) and further treated according to the instructions of the manufacturer. 32P radiation of membrane squares was measured on the scintillation counter Hidex 300 SL by counting Cerenkov radiation for 1 min per sample. Data was analyzed using GraphPad Prism 8.

Radiometric kinase assay

Recombinantly expressed proteins Abl KD or SH2-KD, each at 50 ng, were preincubated with the recombinant or refolded, synthetic monobodies at 5 µM for 10 min at room temperature in a volume of 10 µL. Kinase activity was determined by the addition of 25 µM ATP, 3 µCi of [γ-32P]-ATP (SRP-301, Hartmann Analytic), and 100 µM of an optimal Abl substrate sequence carrying an N-terminal Biotin (Biotin-GGEAIYAAPFKK-amide) in kinase assay buffer (20 mM Tris-HCl, pH 7.5, 10 mM MgCl2,1 mM DTT) for 15 min at room temperature in a final assay volume of 20 µL. 8 µL of each reaction terminated with 10 µL of 7.5 M Guanidine-HCl were spotted onto a SAM2 Biotin Capture membrane square (V2861, Promega) and further treated according to the instructions of the manufacturer. 32P radiation of membrane squares was measured on the scintillation counter Hidex 300 SL by counting Cerenkov radiation for 1 min per sample. Data was analyzed using GraphPad Prism 8.

Flow cytometry analysis of monobodies binding to Bcr-Abl in K562 cellsK562 suspension cells were cultivated in RPMI medium supplemented with 10% FBS and Penicillin/Streptomycin and split every 2–3 days. For flow cytometry analysis, cells were taken up in PBS, pH 7.4, at a density of 5 × 105 cells/mL and fixed in 3.2% Paraformaldehyde (PFA; v/v) by addition of 200 µL PFA solution (E15710, Science Services) for every 1 mL of cell suspension and incubated for 10 min at room temperature (rt). The suspension was centrifuged (5 min, 500×g), the pellet was slowly resuspended in ice-cold methanol at a density of 2.5 × 105 cells/mL while vortexing, placed for 20 min on ice and then stored at −20 °C overnight. Then, cells were washed three times with the same amount of PBS, resuspended in FACS buffer (4% FBS in PBS, pH 7.4) at a density of 1 × 106 cells/mL and placed for 2 h on ice. Afterwards, the cell suspension was split and 5 × 105 cells were used per sample, which were incubated with 20 µL FcBlock (diluted 1:20 (v/v) in FACS buffer; BUF070, Bio-Rad) for 10 min at rt. Then, monobody solutions were added at a concentration of 2 µM in 50 µL total volume and incubated for 45 min at rt. 1 mL FACS buffer was added to each sample, the samples were centrifuged (5 min, 500×g) and the supernatant was removed. 5 µL of AlexaFluor488-coupled Streptavidin (diluted 1:1000; S32354, Thermo Fisher Scientific) in 45 µL FACS buffer was added to each sample, incubated for 45 min at rt in the dark, then mixed with 2 mL FACS buffer, centrifuged (5 min, 500×g), taken up in 200 µL PBS and placed on ice until measurement. Fluorescence intensity of samples was analyzed on a Guava easyCyte flow cytometer (Luminex). Data was analyzed using GraphPad Prism 8.

Flow cytometry analysis of monobodies binding to Bcr-Abl in K562 cells

K562 suspension cells were cultivated in RPMI medium supplemented with 10% FBS and Penicillin/Streptomycin and split every 2–3 days. For flow cytometry analysis, cells were taken up in PBS, pH 7.4, at a density of 5 × 105 cells/mL and fixed in 3.2% Paraformaldehyde (PFA; v/v) by addition of 200 µL PFA solution (E15710, Science Services) for every 1 mL of cell suspension and incubated for 10 min at room temperature (rt). The suspension was centrifuged (5 min, 500×g), the pellet was slowly resuspended in ice-cold methanol at a density of 2.5 × 105 cells/mL while vortexing, placed for 20 min on ice and then stored at −20 °C overnight. Then, cells were washed three times with the same amount of PBS, resuspended in FACS buffer (4% FBS in PBS, pH 7.4) at a density of 1 × 106 cells/mL and placed for 2 h on ice. Afterwards, the cell suspension was split and 5 × 105 cells were used per sample, which were incubated with 20 µL FcBlock (diluted 1:20 (v/v) in FACS buffer; BUF070, Bio-Rad) for 10 min at rt. Then, monobody solutions were added at a concentration of 2 µM in 50 µL total volume and incubated for 45 min at rt. 1 mL FACS buffer was added to each sample, the samples were centrifuged (5 min, 500×g) and the supernatant was removed. 5 µL of AlexaFluor488-coupled Streptavidin (diluted 1:1000; S32354, Thermo Fisher Scientific) in 45 µL FACS buffer was added to each sample, incubated for 45 min at rt in the dark, then mixed with 2 mL FACS buffer, centrifuged (5 min, 500×g), taken up in 200 µL PBS and placed on ice until measurement. Fluorescence intensity of samples was analyzed on a Guava easyCyte flow cytometer (Luminex). Data was analyzed using GraphPad Prism 8.

K562 cell lysis, BCR::ABL1 pulldown and mass spectrometryK562 cells were harvested by centrifugation (500×g, 5 min) at a density of ~1.0 × 106 cells/mL. The cell pellet was washed with PBS, frozen in liquid nitrogen and stored at −80 °C until lysis. The cells were lysed for 5 min on ice in IP buffer (50 mM Tris-HCl, pH 7.5, 150 mM NaCl, 1% NP-40, 5 mM EDTA, 5 mM EGTA) containing additionally 50 mM NaF, 1 mM vanadate, 1 mM PMSF, 10 µg/mL TPCK and 1x protease inhibitor cocktail (Roche) and centrifuged for 10 min at 20,000×g and 4 °C. The total protein amount of the supernatant was determined by standard Bradford assay. The pulldown was performed in three biological replicates for each monobody on Streptavidin MagneSphere Paramagnetic Particles (Z5481, Promega) where 200 µL of beads were used per sample. The beads were washed twice with IP buffer, then 1.5 µg of each monobody were added to the beads and incubated for 2 h at 4 °C. After washing the beads once with IP buffer, the K562 cell lysate containing 3 mg of total protein was added to each sample and incubated for 16 h at 4 °C. The beads were washed three times with IP buffer without NP-40 (50 mM Tris-HCl, pH 7.5, 150 mM NaCl, 5 mM EDTA, 5 mM EGTA). Trypsin (0.1 µg in 50 µL 50 mM ammonium-bicarbonate buffer, pH 8.0) was added to the beads and samples were incubated at 37 °C for 45 min. Subsequently, the supernatant was transferred into fresh tubes and digested overnight at 37 °C to completeness. For the reduction of disulfide bridges 5 mM DTT was added. Samples were then incubated for 15 min at 95 °C. Subsequently, the resulting sulfhydryl groups were chemically modified by adding iodoacetamide to a final concentration of 25 mM and incubating samples for 45 min at RT in the dark. Excess iodoacetamide was quenched by the addition of 50 mM DTT and incubation for one more hour at RT. Reduced and alkylated peptides were then desalted and concentrated using Chromabond C18WP spin columns (730522, Macherey-Nagel) according to manufacturer protocols. Finally, peptides were dissolved in 20 µL of water with 5% acetonitrile and 0.1% formic acid. The mass spectrometric analysis of the samples was performed using a timsTOF Pro mass spectrometer (Bruker Daltonic). A nanoElute HPLC system (Bruker Daltonics), equipped with an Aurora C18 RP column (25 cm × 75 µm ID) filled with 1.7 µm beads (IonOpticks, Australia) was connected online to the mass spectrometer. A portion of 2 µL of the peptide solution was injected directly on the separation column. Sample loading was performed at a constant pressure of 800 bar. Separation of the tryptic peptides was achieved at 60 °C column temperature with the following gradient of water/0.1% formic acid (solvent A) and acetonitrile/0.1% formic acid (solvent B) at a flow rate of 400 nL/min: Linear increase from 2% B to 17% B within 18 min, followed by a linear gradient to 25% B within 9 min and linear increase to 37% solvent B in additional 3 min. Finally, B was increased to 95% within 10 min and held at 95% for additional 10 min. The built-in “DDA PASEF-standard_1.1sec_cycletime” method developed by Bruker Daltonics was used for mass spectrometric measurement. Data analysis was performed using MaxQuant 2.5.1.0 (MPI of Biochemistry, Germany) and statistical analysis of replicates was done with Autonomics (R package version 1.13.21)84.

K562 cell lysis, BCR::ABL1 pulldown and mass spectrometry

K562 cells were harvested by centrifugation (500×g, 5 min) at a density of ~1.0 × 106 cells/mL. The cell pellet was washed with PBS, frozen in liquid nitrogen and stored at −80 °C until lysis. The cells were lysed for 5 min on ice in IP buffer (50 mM Tris-HCl, pH 7.5, 150 mM NaCl, 1% NP-40, 5 mM EDTA, 5 mM EGTA) containing additionally 50 mM NaF, 1 mM vanadate, 1 mM PMSF, 10 µg/mL TPCK and 1x protease inhibitor cocktail (Roche) and centrifuged for 10 min at 20,000×g and 4 °C. The total protein amount of the supernatant was determined by standard Bradford assay. The pulldown was performed in three biological replicates for each monobody on Streptavidin MagneSphere Paramagnetic Particles (Z5481, Promega) where 200 µL of beads were used per sample. The beads were washed twice with IP buffer, then 1.5 µg of each monobody were added to the beads and incubated for 2 h at 4 °C. After washing the beads once with IP buffer, the K562 cell lysate containing 3 mg of total protein was added to each sample and incubated for 16 h at 4 °C. The beads were washed three times with IP buffer without NP-40 (50 mM Tris-HCl, pH 7.5, 150 mM NaCl, 5 mM EDTA, 5 mM EGTA). Trypsin (0.1 µg in 50 µL 50 mM ammonium-bicarbonate buffer, pH 8.0) was added to the beads and samples were incubated at 37 °C for 45 min. Subsequently, the supernatant was transferred into fresh tubes and digested overnight at 37 °C to completeness. For the reduction of disulfide bridges 5 mM DTT was added. Samples were then incubated for 15 min at 95 °C. Subsequently, the resulting sulfhydryl groups were chemically modified by adding iodoacetamide to a final concentration of 25 mM and incubating samples for 45 min at RT in the dark. Excess iodoacetamide was quenched by the addition of 50 mM DTT and incubation for one more hour at RT. Reduced and alkylated peptides were then desalted and concentrated using Chromabond C18WP spin columns (730522, Macherey-Nagel) according to manufacturer protocols. Finally, peptides were dissolved in 20 µL of water with 5% acetonitrile and 0.1% formic acid. The mass spectrometric analysis of the samples was performed using a timsTOF Pro mass spectrometer (Bruker Daltonic). A nanoElute HPLC system (Bruker Daltonics), equipped with an Aurora C18 RP column (25 cm × 75 µm ID) filled with 1.7 µm beads (IonOpticks, Australia) was connected online to the mass spectrometer. A portion of 2 µL of the peptide solution was injected directly on the separation column. Sample loading was performed at a constant pressure of 800 bar. Separation of the tryptic peptides was achieved at 60 °C column temperature with the following gradient of water/0.1% formic acid (solvent A) and acetonitrile/0.1% formic acid (solvent B) at a flow rate of 400 nL/min: Linear increase from 2% B to 17% B within 18 min, followed by a linear gradient to 25% B within 9 min and linear increase to 37% solvent B in additional 3 min. Finally, B was increased to 95% within 10 min and held at 95% for additional 10 min. The built-in “DDA PASEF-standard_1.1sec_cycletime” method developed by Bruker Daltonics was used for mass spectrometric measurement. Data analysis was performed using MaxQuant 2.5.1.0 (MPI of Biochemistry, Germany) and statistical analysis of replicates was done with Autonomics (R package version 1.13.21)84.

Reporting summaryFurther information on research design is available in the Nature Portfolio Reporting Summary linked to this article.

Reporting summary

Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article.

Supplementary information
Supplementary Information
Reporting Summary
Transparent Peer Review file

Supplementary information

Supplementary Information
Reporting Summary
Transparent Peer Review file

Supplementary Information

Reporting Summary

Transparent Peer Review file

Source data
Source Data

Source data

Source Data

Source Data
