---
pmid: '27532772'
title: Mitochondrial Bol1 and Bol3 function as assembly factors for specific iron-sulfur
  proteins.
authors:
- Uzarska MA
- Nasta V
- Weiler BD
- Spantgar F
- Ciofi-Baffoni S
- Saviello MR
- Gonnelli L
- Mühlenhoff U
- Banci L
- Lill R
journal: Elife
year: '2016'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC5014550
doi: 10.7554/eLife.16673
---

# Mitochondrial Bol1 and Bol3 function as assembly factors for specific iron-sulfur proteins.
**Authors:** Uzarska MA, Nasta V, Weiler BD, Spantgar F, Ciofi-Baffoni S, Saviello MR, Gonnelli L, Mühlenhoff U, Banci L, Lill R
**Journal:** Elife (2016)
**DOI:** [10.7554/eLife.16673](https://doi.org/10.7554/eLife.16673)
**PMC:** [PMC5014550](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5014550/)

## Abstract

1. Elife. 2016 Aug 17;5:e16673. doi: 10.7554/eLife.16673.

Mitochondrial Bol1 and Bol3 function as assembly factors for specific 
iron-sulfur proteins.

Uzarska MA(1), Nasta V(2), Weiler BD(1), Spantgar F(1), Ciofi-Baffoni S(2)(3), 
Saviello MR(2)(3), Gonnelli L(2)(3), Mühlenhoff U(1), Banci L(2)(3), Lill 
R(1)(4).

Author information:
(1)Institut für Zytobiologie und Zytopathologie, Philipps-Universität, Marburg, 
Germany.
(2)Magnetic Resonance Center CERM, University of Florence, Florence, Italy.
(3)Department of Chemistry, University of Florence, Florence, Italy.
(4)LOEWE Zentrum für Synthetische Mikrobiologie SynMikro, Marburg, Germany.

Assembly of mitochondrial iron-sulfur (Fe/S) proteins is a key process of cells, 
and defects cause many rare diseases. In the first phase of this pathway, ten 
Fe/S cluster (ISC) assembly components synthesize and insert [2Fe-2S] clusters. 
The second phase is dedicated to the assembly of [4Fe-4S] proteins, yet this 
part is poorly understood. Here, we characterize the BOLA family proteins Bol1 
and Bol3 as specific mitochondrial ISC assembly factors that facilitate [4Fe-4S] 
cluster insertion into a subset of mitochondrial proteins such as lipoate 
synthase and succinate dehydrogenase. Bol1-Bol3 perform largely overlapping 
functions, yet cannot replace the ISC protein Nfu1 that also participates in 
this phase of Fe/S protein biogenesis. Bol1 and Bol3 form dimeric complexes with 
both monothiol glutaredoxin Grx5 and Nfu1. Complex formation differentially 
influences the stability of the Grx5-Bol-shared Fe/S clusters. Our findings 
provide the biochemical basis for explaining the pathological phenotypes of 
patients with mutations in BOLA3.

DOI: 10.7554/eLife.16673
PMCID: PMC5014550
PMID: 27532772 [Indexed for MEDLINE]

Conflict of interest statement: The authors declare that no competing interests 
exist.

## Full Text

IntroductionMitochondria are essential organelles and are involved in numerous biological tasks including fatty acid degradation, citric acid cycle, respiration, ATP production, and the biogenesis of various protein cofactors such as heme, lipoic acid and iron-sulfur (Fe/S) clusters. Genetic defects in these pathways are associated with a spectrum of diverse mitochondriopathies with neurological, neurodegenerative, hematological, and metabolic phenotypes. Knowledge of the molecular deficits of the affected mitochondrial components is key for the development of a comprehensive understanding of mitochondrial diseases. A mutation in human BOLA3 encoding a mitochondrial protein is associated with diverse defects summarized as multiple mitochondrial dysfunction syndrome 2 (MMDS2; Baker et al., 2014; Cameron et al., 2011; Haack et al., 2013). In particular, BOLA3 deficiency results in decreased functions of respiratory complexes I and II as well as lipoic acid-dependent enzymes such as pyruvate dehydrogenase (PDH), 2-ketoglutarate dehydrogenase (KGDH), and glycine cleavage system (GCS) (Mayr et al., 2014). BOLA3 was suggested to play a role in mitochondrial Fe/S protein biogenesis because of similar clinical and biochemical phenotypes in patients with mutations in the known ISC factors NFU1 (causing MMDS1; Cameron et al., 2011; Invernizzi et al., 2014; Navarro-Sastre et al., 2011) and IBA57 (causing MMDS3; Ajit Bolar et al., 2013; Lossos et al., 2015; for review see Beilschmidt and Puccio, 2014; Stehling et al., 2014). Another link for BOLA3 to mitochondrial Fe/S protein biogenesis is provided by the fact that related bacterial and plant Bol proteins interact with monothiol glutaredoxins, factors that play a critical role in Fe/S protein biogenesis [Boutigny et al., 2013; Roret et al., 2014; Yeung et al., 2011]). However, dedicated biochemical investigations of the molecular role of BOLA3 in mitochondria have not been reported hitherto.Human BOLA3 belongs to a large protein family of bacterial origin (Aldea et al., 1988; Willems et al., 2013). Eukaryotes such as yeast and humans possess three BOLA family members that can be discriminated by conserved sequence elements (Figure 1—figure supplement 1). To date, the precise function of the BOLA proteins and the functional relationship between the three eukaryotic BOLA family members is unclear. Yeast Bol2 (formerly termed Fra2; [Kumanovics et al., 2008]) is cytosolic, and is involved in cellular iron regulation by forming heterodimeric, [2Fe-2S] cluster-containing complexes with the cytosolic monothiol glutaredoxins Grx3-Grx4 (Li et al., 2009; Mühlenhoff et al., 2010). The human relative BOLA2 also forms hetero-complexes with human GRX3 (Banci et al., 2015; Li et al., 2012), yet the exact physiological role of the Bol2 proteins is still poorly defined. Additionally to BOLA3, human mitochondria contain the homologous BOLA1 (Willems et al., 2013). Ablation of BOLA1 in cultured human cells increases mitochondrial protein thiol oxidation and elicits alterations in mitochondrial morphology. BOLA1 (but not BOLA3) co-purified with tagged mitochondrial monothiol glutaredoxin GLRX5 that is crucial for Fe/S protein biogenesis (Willems et al., 2013; Ye et al., 2010). Both BOLA1 and BOLA3 have structural counterparts in many eukaryotes including yeast Bol1 and Bol3, respectively (Figure 1—figure supplement 1). Apart from the BOLA3 patient cell analysis, no solid evidence for a direct role of the BOLA family proteins in cellular Fe/S protein metabolism has been described. We therefore sought to better define the physiological role of the mitochondrial Bol1-Bol3 (mBols) and cytosolic Bol2 proteins in this essential biosynthetic pathway by using the yeast S. cerevisiae as a model organism (Beilschmidt and Puccio, 2014; Lill, 2009; Lill et al., 2012; Netz et al., 2014; Rouault, 2012).Mitochondrial Fe/S protein biogenesis in yeast involves 17 known ISC components which were inherited from bacteria and are conserved in eukaryotes (Johnson et al., 2005; Lill, 2009). The pathway can be divided into two major phases. First, components of the ‘core ISC machinery’ including the cysteine desulfurase Nfs1 and the scaffold protein Isu1 synthesize a [2Fe-2S] cluster, and transfer it transiently to the monothiol glutaredoxin Grx5 for subsequent assembly of mitochondrial [2Fe-2S] proteins. Second, dedicated ISC factors use the Grx5-bound Fe/S cluster to assemble a [4Fe-4S] cluster and to facilitate its insertion into mitochondrial apoproteins such as aconitase, respiratory complexes, and the radical SAM Fe/S protein lipoic acid synthase (LIAS). Even though the molecular mechanisms of this late phase of Fe/S protein biogenesis are poorly resolved, recent studies have shown that the Isa1, Isa2 and Iba57 proteins cooperate to generate a [4Fe-4S] cluster (Brancaccio et al., 2014; Gelling et al., 2008; Mühlenhoff et al., 2011; Sheftel et al., 2012). Its insertion into specific target apoproteins is subsequently facilitated by Nfu1 and the P-loop ATPase Ind1, both transiently binding [4Fe-4S] clusters (Bych et al., 2008; Sheftel et al., 2009; Tong et al., 2003). The latter ISC protein is specific for maturation of respiratory complex I, but nothing is known about the molecular mechanisms underlying [4Fe-4S] cluster insertion by eukaryotic Nfu1 and Ind1.While most proteins of the core ISC machinery are essential for cell viability, impairment of the ISC factors involved in the second phase of Fe/S protein biogenesis only compromises mitochondrial function of yeast cells. This striking difference for phases I and II is explained by the central, additional role of the core ISC machinery in both cytosolic Fe/S protein biogenesis and cellular iron regulation. Functional defects of core ISC factors lead to impaired assembly of essential Fe/S proteins such as nuclear DNA polymerases and helicases (Gari et al., 2012; Netz et al., 2012; Stehling et al., 2012) and an induction of the yeast iron regulon involving the above mentioned Bol2 (Lill et al., 2012; Outten and Albetel, 2013; Paul and Lill, 2015). Here, we employed a combination of cell biological, biochemical and ultrastructural methods to characterize the potential role of the Bol proteins in cellular Fe/S protein biogenesis and to define their position in the complex pathway. We also investigate the involvement of the mBols in cellular iron regulation in comparison to Bol2.

Introduction

Mitochondria are essential organelles and are involved in numerous biological tasks including fatty acid degradation, citric acid cycle, respiration, ATP production, and the biogenesis of various protein cofactors such as heme, lipoic acid and iron-sulfur (Fe/S) clusters. Genetic defects in these pathways are associated with a spectrum of diverse mitochondriopathies with neurological, neurodegenerative, hematological, and metabolic phenotypes. Knowledge of the molecular deficits of the affected mitochondrial components is key for the development of a comprehensive understanding of mitochondrial diseases. A mutation in human BOLA3 encoding a mitochondrial protein is associated with diverse defects summarized as multiple mitochondrial dysfunction syndrome 2 (MMDS2; Baker et al., 2014; Cameron et al., 2011; Haack et al., 2013). In particular, BOLA3 deficiency results in decreased functions of respiratory complexes I and II as well as lipoic acid-dependent enzymes such as pyruvate dehydrogenase (PDH), 2-ketoglutarate dehydrogenase (KGDH), and glycine cleavage system (GCS) (Mayr et al., 2014). BOLA3 was suggested to play a role in mitochondrial Fe/S protein biogenesis because of similar clinical and biochemical phenotypes in patients with mutations in the known ISC factors NFU1 (causing MMDS1; Cameron et al., 2011; Invernizzi et al., 2014; Navarro-Sastre et al., 2011) and IBA57 (causing MMDS3; Ajit Bolar et al., 2013; Lossos et al., 2015; for review see Beilschmidt and Puccio, 2014; Stehling et al., 2014). Another link for BOLA3 to mitochondrial Fe/S protein biogenesis is provided by the fact that related bacterial and plant Bol proteins interact with monothiol glutaredoxins, factors that play a critical role in Fe/S protein biogenesis [Boutigny et al., 2013; Roret et al., 2014; Yeung et al., 2011]). However, dedicated biochemical investigations of the molecular role of BOLA3 in mitochondria have not been reported hitherto.

Human BOLA3 belongs to a large protein family of bacterial origin (Aldea et al., 1988; Willems et al., 2013). Eukaryotes such as yeast and humans possess three BOLA family members that can be discriminated by conserved sequence elements (Figure 1—figure supplement 1). To date, the precise function of the BOLA proteins and the functional relationship between the three eukaryotic BOLA family members is unclear. Yeast Bol2 (formerly termed Fra2; [Kumanovics et al., 2008]) is cytosolic, and is involved in cellular iron regulation by forming heterodimeric, [2Fe-2S] cluster-containing complexes with the cytosolic monothiol glutaredoxins Grx3-Grx4 (Li et al., 2009; Mühlenhoff et al., 2010). The human relative BOLA2 also forms hetero-complexes with human GRX3 (Banci et al., 2015; Li et al., 2012), yet the exact physiological role of the Bol2 proteins is still poorly defined. Additionally to BOLA3, human mitochondria contain the homologous BOLA1 (Willems et al., 2013). Ablation of BOLA1 in cultured human cells increases mitochondrial protein thiol oxidation and elicits alterations in mitochondrial morphology. BOLA1 (but not BOLA3) co-purified with tagged mitochondrial monothiol glutaredoxin GLRX5 that is crucial for Fe/S protein biogenesis (Willems et al., 2013; Ye et al., 2010). Both BOLA1 and BOLA3 have structural counterparts in many eukaryotes including yeast Bol1 and Bol3, respectively (Figure 1—figure supplement 1). Apart from the BOLA3 patient cell analysis, no solid evidence for a direct role of the BOLA family proteins in cellular Fe/S protein metabolism has been described. We therefore sought to better define the physiological role of the mitochondrial Bol1-Bol3 (mBols) and cytosolic Bol2 proteins in this essential biosynthetic pathway by using the yeast S. cerevisiae as a model organism (Beilschmidt and Puccio, 2014; Lill, 2009; Lill et al., 2012; Netz et al., 2014; Rouault, 2012).

Mitochondrial Fe/S protein biogenesis in yeast involves 17 known ISC components which were inherited from bacteria and are conserved in eukaryotes (Johnson et al., 2005; Lill, 2009). The pathway can be divided into two major phases. First, components of the ‘core ISC machinery’ including the cysteine desulfurase Nfs1 and the scaffold protein Isu1 synthesize a [2Fe-2S] cluster, and transfer it transiently to the monothiol glutaredoxin Grx5 for subsequent assembly of mitochondrial [2Fe-2S] proteins. Second, dedicated ISC factors use the Grx5-bound Fe/S cluster to assemble a [4Fe-4S] cluster and to facilitate its insertion into mitochondrial apoproteins such as aconitase, respiratory complexes, and the radical SAM Fe/S protein lipoic acid synthase (LIAS). Even though the molecular mechanisms of this late phase of Fe/S protein biogenesis are poorly resolved, recent studies have shown that the Isa1, Isa2 and Iba57 proteins cooperate to generate a [4Fe-4S] cluster (Brancaccio et al., 2014; Gelling et al., 2008; Mühlenhoff et al., 2011; Sheftel et al., 2012). Its insertion into specific target apoproteins is subsequently facilitated by Nfu1 and the P-loop ATPase Ind1, both transiently binding [4Fe-4S] clusters (Bych et al., 2008; Sheftel et al., 2009; Tong et al., 2003). The latter ISC protein is specific for maturation of respiratory complex I, but nothing is known about the molecular mechanisms underlying [4Fe-4S] cluster insertion by eukaryotic Nfu1 and Ind1.

While most proteins of the core ISC machinery are essential for cell viability, impairment of the ISC factors involved in the second phase of Fe/S protein biogenesis only compromises mitochondrial function of yeast cells. This striking difference for phases I and II is explained by the central, additional role of the core ISC machinery in both cytosolic Fe/S protein biogenesis and cellular iron regulation. Functional defects of core ISC factors lead to impaired assembly of essential Fe/S proteins such as nuclear DNA polymerases and helicases (Gari et al., 2012; Netz et al., 2012; Stehling et al., 2012) and an induction of the yeast iron regulon involving the above mentioned Bol2 (Lill et al., 2012; Outten and Albetel, 2013; Paul and Lill, 2015). Here, we employed a combination of cell biological, biochemical and ultrastructural methods to characterize the potential role of the Bol proteins in cellular Fe/S protein biogenesis and to define their position in the complex pathway. We also investigate the involvement of the mBols in cellular iron regulation in comparison to Bol2.

ResultsDeficiency of Bol1-Bol3 causes defects in a subset of mitochondrial [4Fe-4S] enzymesWe first analyzed the sub-cellular localization of S. cerevisiae Bol1 (YAL044W-A) and Bol3 (formerly termed Aim1; (Hess et al., 2009) by cell fractionation. Both proteins were exclusively present in the mitochondrial fraction and absent in the cytosol (Figure 1—figure supplement 2A). Upon sub-fractionation of mitochondria by hypotonic swelling or detergent lysis, the Bol proteins behaved similar to the matrix proteins Tim44 and Mge1 (Figure 1—figure supplement 2B). Thus, Bol1 and Bol3 are constituents of the mitochondrial matrix. To examine the physiological function of mitochondrial Bol1-Bol3 (mBols), we created single and double deletion cells, as well as combined deletions with BOL2 (Supplementary file 1A; strain background BY4742). All deletion strains grew at wild-type rates on minimal media containing the non-fermentable carbon sources glycerol or acetate with the exception of bol13Δ and bol123Δ strains (Figure 1—figure supplement 3). Apparently, simultaneous deletion of BOL1 and BOL3 created a weak respiratory defect which is slightly weaker than that seen for NFU1 deletion (Schilke et al., 1999).We then analyzed key mitochondrial enzyme activities to detect potential defects in Fe/S protein biogenesis, and we compared these results to data for a NFU1 deletion strain which shows mild, yet specific Fe/S enzyme defects (Navarro-Sastre et al., 2011; Schilke et al., 1999). This analysis included the mitochondrial Fe/S proteins aconitase, succinate dehydrogenase (SDH or complex II), and yeast LIAS (Lip5). The latter can be assayed indirectly either by the enzyme activities or the lipoylation extent of pyruvate dehydrogenase (PDH) and 2-ketoglutarate dehydrogenase (KGDH) (Gelling et al., 2008; Schonauer et al., 2009). Further, we measured the activity of cytochrome c oxidase (COX). Even though this enzyme does not contain Fe/S clusters, it is frequently affected by mitochondrial Fe/S protein biogenesis defects, for unknown reasons (see, e.g., [Mühlenhoff et al., 2011; Sheftel et al., 2012]). All enzyme activities were normalized to malate dehydrogenase (MDH) that showed no significant changes in all these strains relative to wild type (not shown).Single deletions of BOL1 or BOL3 did not affect any of these enzyme activities relative to wild-type cells with the exception of a slight, but significant decrease of SDH in bol3Δ cells (Figure 1A–E, bars 1–3). In contrast, double BOL1-BOL3 deletions resulted in substantial, up to threefold decreases of the Fe/S cluster-dependent enzyme activities, an effect comparable to that seen for nfu1Δ cells (bars 4 and 6) (Navarro-Sastre et al., 2011). A notable exception was aconitase which retained normal activity in bol13Δ cells (Figure 1A). Thus, double but not single yeast BOL gene deletions created a similar phenotype as that observed in BOLA3 patient cells (Cameron et al., 2011). Additional ablation of BOL2 (strain bol123Δ, bar 5) did not substantially exacerbate these effects suggesting that the mitochondrial and cytosolic Bol proteins act independently. The decrease in SDH was likely due to a direct defect in Fe/S cluster assembly, because SDH protein levels remained unchanged in immunoblots (Figure 1F). In keeping with the decreased PDH and KGDH activities, we also found a lipoylation defect of the E2 subunits of these enzymes by immunostaining against lipoic acid (Figure 1F) (Gelling et al., 2008; Schonauer et al., 2009). All the observed effects were rather mild in comparison to defects reported for deletion mutants of other late-acting ISC factors such as Isa1, Isa2, and Iba57 (Gelling et al., 2008; Mühlenhoff et al., 2011). This notion was also evident from the lack of any effects on COX activity in all tested BOL deletion strains (Figure 1E). A severe COX diminution is observed in ISA1, ISA2, or IBA57 mutants, yet the exact reason is unknown (Gelling et al., 2008; Mühlenhoff et al., 2011). In order to verify that the loss of the enzyme activities was specific, BOL1 and BOL3 genes were reintroduced into the bol123Δ deletion strain by expression from plasmids. Both Bol1 and Bol3 were able to efficiently rescue the deficiencies in SDH activities (Figure 1—figure supplement 4).10.7554/eLife.16673.003Figure 1.Deficiency of both mitochondrial Bol proteins causes defects in a subset of mitochondrial [4Fe-4S] enzymes.Wild-type (WT; strain BY4742), and the indicated BOL and NFU1 deletion yeast strains were grown in minimal medium containing 2% galactose supplemented with 50 µM ferric ammonium citrate and used for the preparation of mitochondria. Mitochondrial extracts were assayed for specific activities of (A) aconitase (ACO), (B) succinate dehydrogenase (SDH), (C) pyruvate dehydrogenase (PDH), (D) 2-ketoglutarate dehydrogenase (KGDH), and (E) cytochrome c oxidase (COX). Values were normalized to those of malate dehydrogenase (MDH). Error bars indicate the SEM (n≥4). (F) Mitochondrial extracts were subjected to TCA precipitation and the levels of the indicated proteins and of lipoic acid attached to E2 subunits of PDH and KGDH were determined by immunostaining. Staining for porin served as a loading control.DOI:
http://dx.doi.org/10.7554/eLife.16673.00310.7554/eLife.16673.004Figure 1—figure supplement 1.Cartoon of bacterial BolA and eukaryotic Bol1, Bol2 and Bol3 proteins.Conserved amino acid sequence elements that characterize the various members of the BOLA protein family are shown. A conserved His (H*) in Bol2 serves as a Fe/S cluster ligand in a hetero-dimeric complex with Grx3 (Li et al., 2011). MTS; mitochondrial targeting sequence.DOI:
http://dx.doi.org/10.7554/eLife.16673.00410.7554/eLife.16673.005Figure 1—figure supplement 2.Bol1 and Bol3 are proteins of the mitochondrial matrix.(A) Wild-type yeast cells (W303) expressing Myc-tagged Bol1 (vector p424-MET25; Supplementary file 1B) or HA-tagged Bol3 (from vector p425-TDH3) were grown in SD medium, and mitochondria (Mito) and post-mitochondrial supernatant (PMS) fractions were isolated. Immunostaining was performed using antisera raised against the mitochondrial matrix protein Mge1, cytosolic Grx4 (also recognizing Grx3), and monoclonal anti-Myc (A14, Santa Cruz) or anti-HA. (B) Wild-type yeast cells (W303) expressing Bol1-Myc or Bol3-HA were grown in SD medium. Isolated mitochondria were either left intact (sorbitol), hypotonically swollen by tenfold dilution in HEPES buffer, or lysed with 0.2% Triton X-100 detergent (Diekert et al., 2001). Samples were incubated in the presence or absence of 100 µg/ml proteinase K for 20 min on ice, and proteinase K was inactivated by PMSF. After TCA precipitation, samples were analyzed by immunostaining for the Myc-tag of Bol1, the HA-tag of Bol3, the intermembrane space proteins Erv1 and cytochrome b2, and the matrix proteins Tim44 and Mge1.DOI:
http://dx.doi.org/10.7554/eLife.16673.00510.7554/eLife.16673.006Figure 1—figure supplement 3.The growth behavior of the various BOL gene deletion strains in comparison to NFU1 deletion cells.The indicated yeast strains were grown overnight on minimal medium with 2% glucose as carbon source. Cells were diluted to OD600 = 0.5 in water and spotted on rich medium (YP) agar plates with 2% galactose, 3% glycerol or 2% acetate as indicated. Plates were incubated at 30°C for 3 days. WT, wild-type.DOI:
http://dx.doi.org/10.7554/eLife.16673.00610.7554/eLife.16673.007Figure 1—figure supplement 4.Specific rescue of succinate dehydrogenase activities in BOL gene deletion cells by both mitochondrial Bol proteins.Wild-type (WT; strain BY4742) and the indicated BOL deletion strains were transformed with vector p416-MET25 lacking (empty) or overexpressing BOL1 or BOL3 as indicated. Cells were grown in minimal medium with 2% glucose without uracil and used for preparation of mitochondria. Mitochondrial extracts were assayed for the specific activity of succinate dehydrogenase (SDH) and normalized to malate dehydrogenase (MDH) activities. Error bars indicate the SEM (n≥4).DOI:
http://dx.doi.org/10.7554/eLife.16673.00710.7554/eLife.16673.008Figure 1—figure supplement 5.Fe/S enzyme activity defects in cells lacking Bol1-Bol3 after growth in lactate medium.Wild-type (WT; strain BY4742), the indicated BOL and NFU1 deletion yeast strains were grown in lactate medium and used for the preparation of mitochondria. Mitochondrial extracts were assayed for specific activities of (A) aconitase (ACO), (B) succinate dehydrogenase (SDH), (C) pyruvate dehydrogenase (PDH), and (D) 2-ketoglutarate dehydrogenase (KGDH). Values were normalized to those of malate dehydrogenase (MDH). Error bars indicate the SEM (n≥4).DOI:
http://dx.doi.org/10.7554/eLife.16673.008Previous studies on yeast ISC protein depletions, e.g., of the Isa proteins, had shown that the Fe/S protein defects were much more pronounced under strict respiratory conditions, e.g., upon growth in lactate medium (Kispal et al., 1999; Mühlenhoff et al., 2011) (see also Figure 1—figure supplement 3). Both aconitase and SDH activities of the various BOL deletion strains behaved similarly after growth in lactate- compared to galactose-containing media (Figure 1—figure supplement 5). In contrast, PDH and KGDH activities were much more severely (ten- vs. threefold) diminished in bol13Δ, bol123Δ, and nfu1Δ cells grown in lactate medium, similar to previous reports on Isa protein depletion (Kaut et al., 2000; Mühlenhoff et al., 2011). Together, these data show that the mBols were crucial for efficient maturation of LIAS, yet they play only an auxiliary role in SDH maturation. The levels of Bol1 and Nfu1 did not change or rather slightly decreased in the various deletion cells making a compensatory effect unlikely (data not shown; Bol3 levels could not be tested due to the lack of antibody detection). Obviously, Bol1 and Bol3 perform largely complementary roles, because only simultaneous deletion of both BOL1-BOL3 genes created a significant LIAS defect.Bol1 and Bol3 are crucial for de novo Fe/S cluster maturation of specific mitochondrial [4Fe-4S] but not [2Fe-2S] proteinsTo directly characterize the mBol involvement in mitochondrial Fe/S cluster assembly, we employed a well-established 55Fe radiolabeling and immunoprecipitation assay. This also allowed the testing of additional Fe/S proteins including such with [2Fe-2S] clusters. The various BOL deletion strains were radiolabeled with 55Fe, the Fe/S proteins were immunoprecipitated, and the precipitated radioactivity was quantified by scintillation counting (Pierik et al., 2009). Consistent with the enzyme activity data (Figure 1), we found no significant effect on 55Fe incorporation into aconitase upon single, double or triple BOL gene deletion (Figure 2A). For SDH (assayed by precipitation of the myc-tagged Fe/S cluster subunit Sdh2 co-expressed with Sdh1) a 40% decrease in 55Fe binding was observed upon deletion of BOL3, also in combination with BOL1 and BOL2 deletions (Figure 2B). Compared to other ISC gene deletions (see, e.g., Gelling et al., 2008; Gerber et al., 2004; Mühlenhoff et al., 2011; Rodriguez-Manzaneque et al., 2002; Voisine et al., 2001) this effect was rather weak. Similar, yet slightly stronger decreases in 55Fe binding (60%) were detected for mitochondria-targeted HiPIP, a bacterial [4Fe-4S] cluster-containing ferredoxin (Figure 2C). For all the tested Fe/S proteins, single deletion of BOL1 had no effects on 55Fe binding. The observed defects were specific, because reintroduction of Bol1 or Bol3 into the bol123Δ strain via overproducing vectors completely restored wild-type 55Fe binding activity, whereas cytosolic Bol2 or Grx5, as a control, were ineffective (Figure 2D). Taken together, these data support a specific function of the mBols in the maturation of a subset of mitochondrial Fe/S proteins, whereby Bol1 and Bol3 perform largely overlapping, non-essential functions. Interestingly, the 55Fe radiolabeling assay revealed a slightly more important role for Bol3 as compared to Bol1. This tendency, yet less well pronounced, was also observed for the enzyme activities (see Figure 1). The ability of the mBols to mutually substitute each other explains why Fe/S protein defects are best visible when both BOL1 and BOL3 are deleted.10.7554/eLife.16673.009Figure 2.Bol1-Bol3 are required for de novo Fe/S cluster incorporation into specific mitochondrial [4Fe-4S] but not [2Fe-2S] proteins.(A–F) Wild-type (WT, strain BY4742) and the indicated BOL deletion strains were transformed with vectors overproducing (B) Sdh2-Myc and Sdh1, (C and D) HiPIP-Myc, (E) human FDX2-HA, or (F) Schizosaccharomyces pombe (Sp) Grx5-Myc. In part D, bol123Δ cells were additionally transformed with p414-MET25 lacking or containing BOL1, BOL2, or BOL3 genes or p424-TDH3-GRX5. Cells were grown overnight in iron-poor SD medium and radiolabeled with 10 µCi 55Fe for 2 hr. The overproduced proteins were immunoprecipitated from cell extracts with specific antibodies. The amount of co-precipitated 55Fe was quantified by scintillation counting. Error bars indicate the SEM (n≥4).DOI:
http://dx.doi.org/10.7554/eLife.16673.009All Fe/S proteins tested so far contain [4Fe-4S] clusters. To get an insight into the Fe/S cluster-type specificity of mBol function, we examined the 55Fe incorporation into two mitochondrial [2Fe-2S] proteins, namely ferredoxin FDX2-HA, the human ortholog of yeast Yah1, and Grx5 from Schizosaccharomyces pombe (SpGrx5-Myc) (Gelling et al., 2008; Uzarska et al., 2013). The deficiency in mBols did not elicit any significant decrease in 55Fe binding to these [2Fe-2S] proteins (Figure 2E,F). This behavior is strikingly different from that of members of the core ISC machinery, and shows that the mBols were not required for maturation of [2Fe-2S] proteins. We conclude that Bol1-Bol3 are dedicated ISC assembly factors specifically involved in the maturation of a subset of mitochondrial [4Fe-4S] but not of [2Fe-2S] proteins. Further, the mBols act after Grx5, the final core ISC component (Uzarska et al., 2013), because they were not needed for 55Fe incorporation into Grx5.Neither mitochondrial Bol1-Bol3 nor cytosolic Bol2 are required for cytosolic Fe/S protein biogenesisWe next examined if the mitochondrial Bol proteins perform an additional role in cytosolic Fe/S protein biogenesis, if they functionally cooperate with cytosolic Bol2, or if Bol2 alone plays a crucial role in this process. Previous studies on yeast Bol2 and human BOLA2 have suggested an involvement of these cytosolic proteins in intracellular iron metabolism (Kumanovics et al., 2008; Li et al., 2012). First, we measured the consequences of various BOL gene deletions on the activation of the Aft1-Aft2-dependent iron regulon, a hallmark of all cells with defects in the core ISC machinery (Lill et al., 2012; Outten and Albetel, 2013). As expected (Kumanovics et al., 2008), bol2Δ cells displayed a substantial activation of the Aft1-dependent FIT3 and FET3 genes in a GFP-based promoter assay (Figure 3A–B). In contrast, yeast strains lacking BOL1 and/or BOL3 behaved like wild-type cells indicating that the mBols are not involved in cellular iron regulation. This fits nicely to our conclusion above that Bol1-Bol3 do not belong to the core ISC system. The same situation was observed for cells lacking Nfu1 (Figure 3B), another known late-acting specific ISC assembly factor (Navarro-Sastre et al., 2011). Surprisingly, when the BOL2 deletion was combined with genetic ablation of BOL1 and/or BOL3, the iron regulon was activated slightly stronger than in bol2Δ cells (Figure 3A–B). This amplifying effect was specific for a Bol protein deficiency because it was reversed to the FET3 induction level of bol2Δ cells by reintroducing BOL1 and/or BOL3 into bol123Δ cells, and to wild-type FET3 levels by BOL2 expression (Figure 3B).10.7554/eLife.16673.010Figure 3.Mitochondrial Bol1-Bol3 and cytosolic Bol2 are not involved in cytosolic Fe/S protein biogenesis.Wild-type (WT, strain BY4742) and the indicated BOL deletion strains harboring vector (A) pFIT3-Luc2 or (B) pFET3-GFP were cultivated in iron-replete medium to mid-log phase and the activities of the FIT3 and FET3 promoters, respectively, were determined. In (B) the bol123Δ cells were also transformed with centromeric plasmids producing the indicated Bol proteins. Abbreviation: Bol13; Bol1 plus Bol3. (C) Leu1 (relative to MDH) activities were determined in the indicated deletion stains cultivated in iron-replete medium. (D) The indicated BOL deletion stains were radiolabeled with 55Fe, and the Leu1-bound radioactivity was determined by immunoprecipitation with α-Leu1 antibodies followed by scintillation counting. (E) 55Fe incorporation into Rli1-HA was determined accordingly using WT and bol123Δ cells transformed with centromeric plasmids producing the three Bol proteins as indicated or with the empty vector (-). (F) Representative immunoblots determining the protein levels of Leu1 (D) and Rli1-HA (E) in the indicated strains. Porin served as a loading control. Error bars indicate the SEM (n≥4).DOI:
http://dx.doi.org/10.7554/eLife.16673.010Second, we investigated the effects of the three BOL gene deletions on cytosolic Fe/S proteins by assaying the enzyme activity of isopropylmalate isomerase (Leu1). No changes were observed when BOL1-BOL3 genes were deleted either alone or in combination (Figure 3C). In contrast, bol2Δ cells showed a twofold decrease in Leu1 activity, which further declined upon additional deletion of BOL1-BOL3. Collectively, these results show that the mBols are not involved in cytosolic Fe/S protein biogenesis, yet there is a cross-talk of the mBol deficiency with the BOL2 deletion. To better understand the reason of this combinatory effect, we took advantage of the 55Fe radiolabeling assay described above. Deletion of either BOL1-BOL3 or BOL2 hardly affected 55Fe/S cluster binding to Leu1 indicating that neither mitochondrial Bol1-Bol3 nor cytosolic Bol2 perform a decisive role in Leu1 Fe/S cluster incorporation (Figure 3D,F). However, combining the BOL2 deletion with that of BOL1 and/or BOL3 resulted in an up to twofold decrease in 55Fe/S cluster association with Leu1. A similar twofold decrease (compared to wild-type) was seen for 55Fe incorporation into another cytosolic Fe/S protein, the ABC protein Rli1 using the triple BOL deletion strain bol123Δ (Figure 3E–F). We note, however, that these decreases in 55Fe incorporation were rather weak compared to defects in core ISC or CIA components (see, e.g., Netz et al., 2007, 2010; Paul et al., 2015). The Rli1 maturation defect was completely restored when any of the BOL genes was reintroduced into the bol123Δ mutant. This supports the notion that neither the mBols nor Bol2 are directly involved in cytosolic Fe/S protein maturation. Obviously, their combined absence causes alterations in both mitochondrial metabolism and cellular iron homeostasis, thereby creating conditions that negatively affect cytosolic Fe/S cluster insertion. The physiological conditions prevailing in bol123Δ cells might also explain the slight decrease in the enzyme activity of aconitase which, like Leu1, is a highly susceptible protein to, e.g., oxidative stress conditions or changes in iron availability (cf. Figures 1A and 3C). Together, our data indicate that the mBols execute a mitochondria-specific function in [4Fe-4S] protein assembly, whereas Bol2 plays a specific role in cellular iron regulation (Kumanovics et al., 2008).Bol1-Bol3 and Nfu1 perform distinct tasks in mitochondrial [4Fe-4S] protein maturationOur current and previous results suggested functions of the mBols and of Nfu1 as specialized ISC assembly factors (Cameron et al., 2011; Navarro-Sastre et al., 2011). Are these functions independent or overlapping? We first analyzed whether simultaneous inactivation of Bol1-Bol3 and Nfu1 aggravates the Fe/S defects. As observed previously (Navarro-Sastre et al., 2011; Schilke et al., 1999), nfu1Δ cells showed a diminution in the activities of both aconitase and SDH relative to wild type (Figure 4A–B). A substantial further decrease to rather low aconitase and SDH activities was found for additional deletion of BOL3, but not of BOL1. The BOL3 effect was not further amplified in nfu1Δbol13Δ cells. These findings show that Bol3 plays a distinct, Nfu1-independent role that cannot be executed by Bol1. Apparently, even though Bol1 and Bol3 play overlapping roles (see above), their functions are not entirely identical.10.7554/eLife.16673.011Figure 4.Bol1-Bol3 and Nfu1 cannot functionally replace each other in Fe/S protein maturation.(A–B) Wild-type (WT, strain BY4742) and the indicated deletion strains were grown in minimal medium containing 2% glucose and used for the preparation of mitochondria. Mitochondrial extracts were assayed for the indicated specific enzyme activities as described in Figure 1. (C–F) The indicated deletion strains were transformed with vector p416-MET25 lacking (empty) or containing BOL3 or NFU1 as indicated. Cells were grown in minimal medium containing 2% galactose and used for the preparation of mitochondria. Mitochondrial extracts were assayed for the indicated enzyme activities as described in Figure 1. Error bars indicate the SEM (n≥4).DOI:
http://dx.doi.org/10.7554/eLife.16673.01110.7554/eLife.16673.012Figure 4—figure supplement 1.Growth complementation test of nfu1Δ and bol13Δ cells.(A) nfu1Δ and (B) bol13Δ cells were transformed with vector p414-MET25 containing no gene (empty), BOL1, BOL3 or BOL1-BOL3 (BOL13) or p416-MET25-NFU1 as indicated. Cells were grown overnight on minimal medium without uracil containing 2% glucose as a carbon source. Cells were diluted to OD600 of 0.1 in 500 µl of liquid minimal medium without uracil containing 2% acetate as sole carbon source, and put into a 48 well plate. Plates were incubated at 30°C for 3 days with shaking, and the OD600 was measured by a TECAN plate reader every 30 min.DOI:
http://dx.doi.org/10.7554/eLife.16673.012We next tested whether the Bol1-Bol3 and Nfu1 proteins can mutually substitute each other’s functions. To this end, we first took advantage of the growth defects of the bol13Δ and nfu1Δ strains in acetate-containing minimal medium (see above). Upon complementation of bol13Δ cells with BOL1 and/or BOL3 or of nfu1Δ cells with NFU1, wild-type growth rates were restored (Figure 4—figure supplement 1). In contrast, no significant growth defect complementation was observed for nfu1Δ cells upon BOL1 and/or BOL3 expression (part A). A weak complementation was seen for bol13Δ cells by NFU1 expression (part B). Notably, this slight rescue effect was not observed for growth on solid acetate medium (not shown). Further, we used these cells for measuring the enzyme activities of aconitase, SDH, PDH and KGDH. In agreement with the growth behavior, Bol3 was not able to rescue any of the enzyme defects of nfu1Δ cells, while Nfu1 did so (Figure 4C–F, left). Likewise, Nfu1 failed to restore the enzyme activities of bol13Δ cells, while Bol3 was complementing (Figure 4C–F, right). These findings show that Nfu1 and Bol3 cannot mutually substitute each other’s biochemical function, even upon overproduction. Despite the fact that Bol3 and Nfu1 both act late in mitochondrial Fe/S protein biogenesis, they appear to fulfil different tasks that cannot be taken over by the other protein. Hence, these data suggest that Bol1-Bol3 and Nfu1 perform individual, non-overlapping functions during Fe/S cluster assembly.Structural insights into the human mitochondrial BOLA proteins and the interaction with GLRX5To extend our cell biological insights into Bol protein function, we employed NMR structural studies to examine their physical interaction with Grx5. We used the human proteins for these in vitro studies because they were more stable than the yeast counterparts. First, the solution structures of human BOLA1 and BOLA3 were determined (Figure 5). A detailed description of the two structures and of their backbone dynamic properties is provided in Figure 5—figure supplement 1. According to the DALI server (Holm and Sander, 1993), both BOLA1 and BOLA3 show the highest structural similarity to BolA-like proteins from Arabidopsis thaliana, Babesia bovis, and Mus musculus (Abendroth et al., 2011; Kasai et al., 2004; Roret et al., 2014). In particular, the BOLA1 structure matches better with those homologues that have a longer loop between β1 and β2, while the BOLA3 structure is more similar to homologues having a shorter β1-β2 loop (Figure 5C and Figure 5—figure supplement 2). Apparently, the structural features of the loop between β1 and β2 are key for differentiating BOLA1 from BOLA3. Remarkably, this loop is closer to the ‘invariant’ His residue (His102 in BOLA1 and His96 in BOLA3; Figure 5), which has been shown to coordinate the [2Fe-2S] cluster in the yeast Bol2-Grx3 complex (Li et al., 2011). This region contains further possible Fe/S cluster ligands, namely His58, His67 in BOLA1 and Cys59 in BOLA3. Another conserved His residue (His86 in BOLA1 and His81 in BOLA3) is distant from this region and thus excluded as a cluster ligand (Figure 5). Together, structural data indicate that BOLA1 and BOLA3 have a similar fold, but show local structural differences in the regions containing the potential Fe/S cluster ligands.10.7554/eLife.16673.013Figure 5.NMR solution structures of human BOLA1 and BOLA3.The structures of (A) BOLA1 and (B) BOLA3 were solved by solution NMR. Residues His58, His67 and His102 are conserved within the eukaryotic BOLA1 proteins, and residues Cys59 and His96 are conserved within the BOLA3 proteins. (C) Backbone superimposition of BOLA1 (red) and BOLA3 (blue) structures depicting conserved His and Cys residues in both proteins. Yellow sticks represent potential Fe/S cluster ligands, and green sticks indicate other conserved His residues.DOI:
http://dx.doi.org/10.7554/eLife.16673.01310.7554/eLife.16673.014Figure 5—figure supplement 1.Structural and dynamic properties of BOLA1 and BOLA3 by solution NMR.(A) 2D 1H-15N HSQC spectra of 15N-labeled BOLA1 and BOLA3. (B) 15N R1, R2 relaxation rates and 15N(1H) NOE values per residue of BOLA1 and BOLA3 obtained at 500 MHz and 298 K. The proteins were in buffer N with 10% (v/v) D2O). The 1H-15N HSQC spectra of the apo forms of BOLA3 and BOLA1 showed well-dispersed resonances indicative of essentially folded proteins. The overall topology of the solution structure of human BOLA3 is α1β1β2η1α2η2β3 (h: 310-helix), in which β1 and β2 are antiparallel, and β3 is parallel to β2 (Figure 5). All α-helices are on the same side of the β-sheet and the 310-helix located between α2 and β3 contains the His96 residue that is invariant in all prokaryotic and eukaryotic BolA homologues (Li and Outten, 2012). A cysteine residue (Cys59), typically conserved in Bol3 homologues, is located close to the invariant His96 residue in an extended loop between β1 and β2. The two residues are not in the proper orientation to coordinate a [2Fe-2S] cluster (Sγ of Cys59 is at an average distance of ~8 Å from Nε2 of His96). The overall topology of the solution structure of BOLA1 is similar to that of BOLA3, with the exception of the presence, in BOLA1, of short helices in the loop regions and at the C-terminus (α1η1β1β2η2α2α3β3α4 topology) (Figure 5). In addition, at both termini BOLA1 has ca. 15 residues that are unstructured and whose backbone NH resonances are clustered in the central region of the 1H-15N HSQC spectrum (part B). The three His residues typically conserved in Bol1 homologues are located in two spatially close regions (Figure 5). Although they are close to each other, it is not possible to predict which ones may be involved in [2Fe-2S] cluster coordination. Information on backbone motions and on the protein oligomerization state of BOLA1 and BOLA3 were obtained through 15N R1, R2, heteronuclear 15N(1H)-NOEs NMR experiments. The 15N backbone relaxation properties showed that the N and C termini of BOLA1 are highly flexible, as about 15 to 20 residues on each terminus have negative 15N NOE values, while the folded domain is essentially rigid. BOLA3 is a rigid molecule, with the exception of the first detected residue at the N terminus, the last two residues at the C terminus and residues around invariant His96. All of them experience fast backbone motions in the ns-ps time scale, as indicated by their low or negative 15N NOE values.DOI:
http://dx.doi.org/10.7554/eLife.16673.01410.7554/eLife.16673.015Figure 5—figure supplement 2.Multi-sequence alignment of the mitochondrial Bol proteins.Multalin (Corpet, 1988) was used to generate a multi-sequence alignment of Bol1- and Bol3-like proteins from fungi and various higher eukaryotes. The conserved His and Cys residues are indicated. The numbers refer to the residues of the human mBols. Two His (58 and 67) in BOLA1 are candidates for the structural counterpart of Cys59 of BOLA3.DOI:
http://dx.doi.org/10.7554/eLife.16673.015In order to characterize the interaction between BOLA1 or BOLA3 and GLRX5 at the atomic level, we first studied the apo situation. 15N labeled BOLA proteins were titrated with unlabeled apo-GLRX5, and, vice versa,
15N labeled apo-GLRX5 with unlabeled BOLA proteins in the presence of 5 mM GSH. Meaningful chemical shift changes were observed for both titrations in 2D 1H-15N HSQC NMR spectra acquired upon addition of the unlabeled partner (Figure 6—figure supplements 1A and 2A). The free and bound proteins are in fast and intermediate exchange regimes relative to NMR time scale; the observed chemical shift changes of the protein amide resonances saturated at a 1:1 protein ratio. 15N R1 and R2 NMR relaxation data performed on the final 15N labeled BOLA1-unlabelled GLRX5 or 15N labeled BOLA3-unlabelled GLRX5 mixtures showed rotational correlation times consistent with the formation of a 1:1 heterodimeric complex (Figure 6—figure supplement 3). NMR titrations performed by adding unlabeled apo-GLRX5 to a 15N-labeled BOLA1/15N-labeled BOLA3 1:1 mixture showed the formation of both hetero-complexes at comparable levels, indicating similar affinities. This data fits nicely to thermophoresis results showing that both human BOLA proteins interacted with apo-GLRX5 with Kd values of 3 µM (Melber et al., 2016; accompanying manuscript).The chemical shift changes observed upon formation of apo-GLRX5-BOLA1 and apo-GLRX5-BOLA3 complexes were mapped on the solution structures of BOLA1, BOLA3 and apo-GLRX5. The interaction surface on GLRX5 in the two complexes comprises the GSH binding site and its surroundings (Figure 6A,B and Figure 6—figure supplement 2A). On the BOLAs, the interactions involve helix α2, part of the β-sheet and the invariant His residue (see above). The other possible Fe/S cluster ligands (His58, His67 in BOLA1 and Cys59 in BOLA3) showed no significant chemical shifts. In conclusion, NMR data indicate that apo-GLRX5-BOLA1 and apo-GLRX5-BOLA3 complexes specifically interact involving the region surrounding the invariant His in BOLAs and the GSH binding site in GLRX5.10.7554/eLife.16673.016Figure 6.Structural basis of the interaction between human apo- and holo-GLRX5 with the BOLA proteins.(A and B) Backbone chemical shift differences, obtained from a comparison of the 1H-15N HSQC spectra of apo-GLRX5 or BOLA proteins with that of (A) BOLA1-apo-GLRX5 or (B) BOLA3-apo-GLRX5 (1:1 mixture in buffer N), were mapped on the solution structures of the proteins. (C and D) Backbone chemical shift differences, obtained from a comparison of the 1H-15N HSQC spectrum of apo-GLXR5 or BOLA proteins with that of (C) BOLA1-apo-GLRX5 or (D) BOLA3-apo-GLRX5 (1:1 mixture in buffer N) chemically reconstituted with a [2Fe-2S] cluster, were mapped on the solution structures of the proteins. Green regions show residues with significant chemical shift changes (that is both in terms of chemical shift and broadening beyond detection effects) observed upon formation of the apo- or holo-complexes. Red areas depict those residues additionally affected upon holo-complex formation. Critical residues and GLRX5-bound GSH are depicted as sticks.DOI:
http://dx.doi.org/10.7554/eLife.16673.01610.7554/eLife.16673.017Figure 6—figure supplement 1.Chemical shift changes upon complex formation between GLRX5 and BOLA proteins monitoring backbone chemical shift changes of BOLA1 and BOLA3.Backbone weighted average chemical shift differences Δavg(HN) obtained by comparing 1H-15N HSQC map of (A) 15N-labeled BOLA1 or BOLA3 with an equimolar mixture of 15N-labeled BOLA1 or BOLA3 with unlabeled apo-GLRX5, and of (B) 15N-labeled BOLA1 or BOLA3 with an equimolar mixture of 15N-labeled BOLA1 or BOLA3 chemically reconstituted with unlabeled GLRX5 to generate a [2Fe-2S] cluster (buffer N). Bars with Δavg(HN) = 1 indicate residues whose backbone NH signals broaden beyond detection. The indicated thresholds (obtained by averaging Δavg(HN) values plus 1σ) were used to define significant chemical shift differences. Green bars show significant chemical shift changes (that is both chemical shift differences and broadening beyond detection effects, respectively) in the apo interactions, and red bars those additionally occurring upon holo complex formation. The position of conserved residues is highlighted.DOI:
http://dx.doi.org/10.7554/eLife.16673.01710.7554/eLife.16673.018Figure 6—figure supplement 2.Chemical shift changes upon complex formation between GLRX5 and BOLA proteins, monitoring backbone chemical shift changes of GLRX5.Backbone weighted average chemical shift differences Δavg(HN) obtained by comparing 1H-15N HSQC map of (A) 15N-labeled apo-GLRX5 with that of a 1:1 mixture of 15N-labeled apo-GLRX5 and unlabeled BOLA1 or BOLA3, and of (B) 15N-labeled apo-GLRX5 with that of a 1:1 mixture of 15N-labeled apo-GLRX5 with unlabeled BOLA1 or BOLA3 after chemical reconstitution of a [2Fe-2S] cluster in buffer N. The data are presented as in Figure 6—figure supplement 1. The position of the conserved Cys67 and of the residues of the GSH binding site are highlighted.DOI:
http://dx.doi.org/10.7554/eLife.16673.01810.7554/eLife.16673.019Figure 6—figure supplement 3.Experimental and predicted rotational correlation times (τC).For the isolated proteins the τC values were obtained by the HYDRONMR program. For the apo-complexes the τC values were calculated from the sum of the τC values of the two isolated proteins.DOI:
http://dx.doi.org/10.7554/eLife.16673.01910.7554/eLife.16673.020Figure 6—figure supplement 4.Comparison of the apo-AtGrxS14-AtBolA2 complex with apo-GLRX5/BOLA3.Significant chemical shift changes (red and green for A. thaliana and human proteins, respectively) are mapped on the structures of (A) apo-AtGrxS14 and apo-GLRX5 upon the interaction with AtBolA2 and human BOLA3, respectively. Vice versa, part (B) shows the significant chemical shift changes of AtBolA2 and human BOLA3 upon interaction with AtGrxS14 and GLRX5, respectively. Conserved residues and the GLRX5-bound GSH are highlighted as sticks.DOI:
http://dx.doi.org/10.7554/eLife.16673.020Recently, a heterodimeric complex formed between A. thaliana (At) apo-GrxS14 and AtBolA2 (showing best similarity to BOLA3 and possessing a conserved Cys in addition to the invariant C-terminal His) was characterized by NMR (Roret et al., 2014). Surprisingly, the human and plant complexes share only some regions of chemical shift changes, and exhibit substantial differences (Figure 6—figure supplement 4). For instance, the invariant His66 region of AtBolA2 does not appear to be an interaction interface, at variance with what we observed for both human BOLA complexes (Figure 6A,B). In addition, the GrxS14-interacting regions include the two C-terminal α4 and α5 helices, whereas helix α5 of GLRX5 of both human complexes did not show chemical shifts, and a different segment of helix α4 was involved. Moreover, helix α2 of GLRX5 showed signs of interaction only in the human BOLA-GLRX5 complexes. We conclude that there is a high degree of organismic variability in the precise interaction mode between Grx and Bol proteins.We next investigated the interaction of holo-GLRX5 with the BOLA proteins by chemically reconstituting a [2Fe-2S] cluster on equimolar GLRX5 and BOLAs. NMR 1H-15N HSQC spectra were acquired for the co-reconstituted complexes of 15N labeled BOLA3 or BOLA1 with GLRX5. The 1H-15N HSQC spectra of these co-reconstituted complexes were similar to spectra obtained by mixing BOLA1 or BOLA3 with [2Fe-2S]-containing holo-GLRX5 (not shown). Upon overlaying the 1H-15N HSQC spectra of the holo-complexes with those acquired for the corresponding apo-complexes, meaningful spectral variations (either as chemical shift changes and/or line-broadenings beyond signal detection) were detected on both proteins (Figure 6—figure supplements 1B and 2B). The 1H-15N HSQC maps of the co-reconstituted complexes having 15N labeled GLRX5 with unlabeled BOLA1 or BOLA3 were not superimposable to the 1H-15N HSQC map of the holo-GLRX5 homodimer indicating formation of [2Fe-2S] hetero-complexes 15N R1 and R2 NMR relaxation data of BOLA1 or BOLA3, in the [2Fe-2S] chemically reconstituted complexes with GLRX5, indicated no significant changes in protein size with respect to the apo-complexes. Overall, the NMR data showed the presence of 1:1 heterodimeric holo-complexes between GLRX5 and BOLA1 or BOLA3.Chemical shift perturbation and line broadening analyses were performed by comparing the 1H-15N HSQC spectrum of chemically reconstituted 15N-labeled BOLA3-GLRX5 with that of BOLA3. The residues of the loop containing the conserved Cys59 of BOLA3 showed strong chemical shift changes only in the holo hetero-complex relative to the apoform (Figure 6B,D and Figure 6—figure supplement 1, right parts). A similar comparison for the 15N-labeled BOLA1-GLRX5 complex showed that the region around His67 of BOLA1 (structurally close to Cys59 of BOLA3) exhibited similar chemical shift changes, even though fewer residues were affected in BOLA1 (Figure 6A,C, and Figure 6—figure supplement 1, left parts). The chemical shift perturbation analysis for 15N-labeled GLRX5 revealed that the residues of GLRX5 involved in the apo interaction are also affected in the holo-complex and that the conserved Cys67 of GLRX5 was additionally altered in the holo complexes only (Figure 6—figure supplement 2). In addition, two other adjacent GLRX5 residues (Lys59 for BOLA3 and Gly60 for BOLA1 interaction) were altered that are part of the GSH binding site. The regions of GLRX5 affected by the interaction with BOLA1 and BOLA3 are essentially the same as those in the [2Fe-2S]2+ GLRX5 homodimer (Banci et al., 2014). This indicates that, for GLRX5, the same interaction regions are involved in both homo- and hetero-dimer formation, and the same groups (the conserved Cys67 and GSH) can act as iron ligands of the [2Fe-2S]2+ cluster.Preferential interaction of BOLA1 with holo-GLRX5 and of BOLA3 with holo-NFU1We used circular dichroism (CD) spectroscopy to follow the characteristics of Fe/S cluster coordination upon ligand exchange from the holo-Grx5 homo-dimer to the Grx5-Bol hetero-dimer (Banci et al., 2015; Li et al., 2009; Li et al., 2012). GLRX5 and stoichiometric amounts of BOLA1 or BOLA3 were chemically co-reconstituted with a [2Fe-2S] cluster and analyzed by CD (Figure 7). Similar results were obtained when GLRX5 was reconstituted first and then the BOLA proteins were added (Figure 7—figure supplement 1). The CD spectrum of GLRX5-BOLA1 characteristically differed from that of holo-GLRX5 in a positive versus negative ellipticity around 400 nm with a 6 nm blue shift and 40% decrease of the ~460 nm peak with a 6 nm red shift (Figure 7A). These spectral changes saturated at an equimolar ratio of holo-GLRX5 and BOLA1 suggesting a high affinity equimolar hetero-complex formation with a shared [2Fe-2S] cluster (Figure 7—figure supplement 1B). The cluster coordinated by the GLRX5-BOLA1 hetero-dimer was stable against reduction by dithionite creating a CD spectrum in which the peak at 460 nm was maintained but that at 400 nm was lost. In contrast, the cluster of holo-GLRX5 was destroyed upon dithionite treatment with no residual CD peaks (Figure 7A and Figure 7—figure supplement 1A). These results demonstrate that BOLA1 stabilized the shared [2Fe-2S] cluster in the hetero-dimer with GLRX5.10.7554/eLife.16673.021Figure 7.Human BOLA1 but not BOLA3 stabilizes the [2Fe-2S] cluster of holo-GLRX5 upon heterodimer formation.(A–B) Chemical reconstitution of Fe/S clusters in buffer R was performed with GLRX5 in the absence or presence of stoichiometric amounts of (A) BOLA1 or (B) BOLA3, and CD spectra were monitored under anaerobic conditions. Additional spectra were recorded after addition of 2 mM dithionite (DT). (C–D) Chemically reconstituted GLRX5 was titrated with the indicated concentrations of BOLA3 in (C) buffer N or (D) buffer P, and CD spectra (corrected for dilution) were recorded under anaerobic conditions. Abbreviations for varied buffer conditions: 7, 8: pH; N, 150 mM NaCl; G, 5 mM GSH.DOI:
http://dx.doi.org/10.7554/eLife.16673.02110.7554/eLife.16673.022Figure 7—figure supplement 1.Stoichiometric hetero-complex formation of human holo-GLRX5 with BOLA1 results in characteristic CD spectral changes indicating shared binding of the [2Fe-2S] cluster.(A,C) Chemically reconstituted GLRX5 was mixed in buffer R (lacking additional GSH) with a stoichiometric amount of (A) BOLA1 or (C) BOLA3, and CD spectra were recorded under anaerobic conditions. Additional spectra were recorded after addition of 2 mM dithionite (DT). (B, D) Chemically reconstituted GLRX5 was titrated with the indicated concentrations of (B) BOLA1 or (D) BOLA3, and CD spectra (corrected for dilution) were recorded under anaerobic conditions.DOI:
http://dx.doi.org/10.7554/eLife.16673.02210.7554/eLife.16673.023Figure 7—figure supplement 2.Gel filtration verifies the complex formation between human holo-GLRX5 and BOLA1.(A) The gel filtration elution profile of co-reconstituted equimolar holo-GLRX5-BOLA3, holo-GLRX5 (offset by OD 0.015), or co-reconstituted equimolar GLRX5-BOLA1 (offset by OD 0.045) was recorded at 420 nm in buffer R. Elution positions of molecular marker proteins are indicated in kDa. (B) As a control, the elution behavior of the holo-GLRX5+BOLA3 mixture or BOLA3 alone were recorded at 280 nm in parallel to part A. Fractions of the holo-GLRX5+BOLA3 mixture were analyzed for GLRX5 and BOLA3 by Coomassie and immunostaining (bottom part).DOI:
http://dx.doi.org/10.7554/eLife.16673.02310.7554/eLife.16673.024Figure 7—figure supplement 3.Preferential binding of human BOLA3 to the holo-form of NFU1.(A) NFU1 was used in either apo- or holo-form and mixed at increasing concentrations with 200 nM fluorescently labeled BOLA1 or BOLA3. Microscale thermophoresis (MST) analyses were performed under anaerobic conditions, and dissociation constants (Kd) were determined. Control experiments were performed with human [2Fe-2S] ferredoxin FDX2, cytochrome c, and RNase A. Error bars indicate the SD (n = 3). (B–E) Examples for original MST data.DOI:
http://dx.doi.org/10.7554/eLife.16673.024A strikingly different Fe/S cluster behavior was observed upon BOLA3-GLRX5 complex formation. Hardly any CD spectral changes were observed upon co-reconstituting or mixing GLRX5 and BOLA3 (Figure 7B and Figure 7—figure supplement 1C,D). Further, the Fe/S cluster was not protected against destruction by dithionite treatment indicating rather different coordination situations for the [2Fe-2S] clusters in the two GLRX5-BOLA heterodimers. These results are consistent with the different NMR chemical shift changes in the putative Fe/S cluster binding regions of the two BOLAs in the hetero-complexes (Figure 6C,D). They do not explain, however, why NMR still detected holo-specific changes in the putative cluster-binding region of the BOLA3 hetero-complex (Figure 6D). Since the NMR data were recorded in GSH-containing buffer at lower pH, we repeated the CD analysis under NMR buffer conditions. We found a general decrease of the CD peaks upon GLRX5-BOLA3 hetero-complex formation (Figure 7C). These changes mainly depended on the presence of GSH, and not salt or pH (Figure 7D). In all cases, the CD signal decrease did not reach saturation even after addition of a more than tenfold molar excess of BOLA3 over holo-GLRX5. This result indicated a rather low affinity for this interference of BOLA3 with the [2Fe-2S] cluster. Overall, the CD data indicate a rather weak and GSH-dependent influence of BOLA3 on the GLRX5 [2Fe-2S] cluster which is not stabilized by hetero-complex formation.To independently analyze the GLRX5-BOLA complex formation, we used size exclusion chromatography. Holo-GLRX5 (recorded at 420 nm) eluted at 44 kDa indicative of homodimer with bound Fe/S cluster (Figure 7—figure supplement 2A). A significantly smaller molecular mass was observed for the heterodimeric GLRX5-BOLA1 (32 kDa), while the mixture of holo-GLRX5 and BOLA3 did not alter the elution behavior of GLRX5 at 420 nm. The elution profile at 280 nm showed no significant protein peak around 30 kDa that might represent a GLRX5-BOLA3 heterodimer (Figure 7—figure supplement 2B). Further, BOLA3 from the GLRX5-BOLA3 mixture eluted exclusively at the position of BOLA3 alone. These results support the formation of a stable heterodimer between GLRX5 and BOLA1, yet the interaction of GLRX5 with BOLA3 appears to be kinetically unstable, especially in the absence of GSH.The lack of stable interaction between GLRX5 and BOLA3 raised the question of whether the latter could interact with NFU1. Since Fe/S cluster binding to NFU1 was CD-silent, we performed in vitro binding studies by microscale thermophoresis (MST), an equilibrium method that can be performed under anaerobic conditions (Webert et al., 2014). For these interaction experiments we used both apo-NFU1 and chemically reconstituted holo-NFU1 carrying a [4Fe-4S] cluster (Tong et al., 2003). MST indeed showed a specific interaction between the two BOLA proteins and both apo- and holo-NFU1 (Figure 7—figure supplement 3). The dissociation constants (Kd) of the interactions were in the range of 3 µM, with the notable exception of a fourfold higher affinity detected for BOLA3 and holo-NFU1 (Kd = 0.8 µM). As a control, no significant affinity was observed for human [2Fe-2S] ferredoxin FDX2, cytochrome c or RNase A as control proteins. Collectively, these results data suggest that holo-GLRX5 preferentially cooperates with BOLA1, and holo-NFU1 with BOLA3.

Deficiency of Bol1-Bol3 causes defects in a subset of mitochondrial [4Fe-4S] enzymesWe first analyzed the sub-cellular localization of S. cerevisiae Bol1 (YAL044W-A) and Bol3 (formerly termed Aim1; (Hess et al., 2009) by cell fractionation. Both proteins were exclusively present in the mitochondrial fraction and absent in the cytosol (Figure 1—figure supplement 2A). Upon sub-fractionation of mitochondria by hypotonic swelling or detergent lysis, the Bol proteins behaved similar to the matrix proteins Tim44 and Mge1 (Figure 1—figure supplement 2B). Thus, Bol1 and Bol3 are constituents of the mitochondrial matrix. To examine the physiological function of mitochondrial Bol1-Bol3 (mBols), we created single and double deletion cells, as well as combined deletions with BOL2 (Supplementary file 1A; strain background BY4742). All deletion strains grew at wild-type rates on minimal media containing the non-fermentable carbon sources glycerol or acetate with the exception of bol13Δ and bol123Δ strains (Figure 1—figure supplement 3). Apparently, simultaneous deletion of BOL1 and BOL3 created a weak respiratory defect which is slightly weaker than that seen for NFU1 deletion (Schilke et al., 1999).We then analyzed key mitochondrial enzyme activities to detect potential defects in Fe/S protein biogenesis, and we compared these results to data for a NFU1 deletion strain which shows mild, yet specific Fe/S enzyme defects (Navarro-Sastre et al., 2011; Schilke et al., 1999). This analysis included the mitochondrial Fe/S proteins aconitase, succinate dehydrogenase (SDH or complex II), and yeast LIAS (Lip5). The latter can be assayed indirectly either by the enzyme activities or the lipoylation extent of pyruvate dehydrogenase (PDH) and 2-ketoglutarate dehydrogenase (KGDH) (Gelling et al., 2008; Schonauer et al., 2009). Further, we measured the activity of cytochrome c oxidase (COX). Even though this enzyme does not contain Fe/S clusters, it is frequently affected by mitochondrial Fe/S protein biogenesis defects, for unknown reasons (see, e.g., [Mühlenhoff et al., 2011; Sheftel et al., 2012]). All enzyme activities were normalized to malate dehydrogenase (MDH) that showed no significant changes in all these strains relative to wild type (not shown).Single deletions of BOL1 or BOL3 did not affect any of these enzyme activities relative to wild-type cells with the exception of a slight, but significant decrease of SDH in bol3Δ cells (Figure 1A–E, bars 1–3). In contrast, double BOL1-BOL3 deletions resulted in substantial, up to threefold decreases of the Fe/S cluster-dependent enzyme activities, an effect comparable to that seen for nfu1Δ cells (bars 4 and 6) (Navarro-Sastre et al., 2011). A notable exception was aconitase which retained normal activity in bol13Δ cells (Figure 1A). Thus, double but not single yeast BOL gene deletions created a similar phenotype as that observed in BOLA3 patient cells (Cameron et al., 2011). Additional ablation of BOL2 (strain bol123Δ, bar 5) did not substantially exacerbate these effects suggesting that the mitochondrial and cytosolic Bol proteins act independently. The decrease in SDH was likely due to a direct defect in Fe/S cluster assembly, because SDH protein levels remained unchanged in immunoblots (Figure 1F). In keeping with the decreased PDH and KGDH activities, we also found a lipoylation defect of the E2 subunits of these enzymes by immunostaining against lipoic acid (Figure 1F) (Gelling et al., 2008; Schonauer et al., 2009). All the observed effects were rather mild in comparison to defects reported for deletion mutants of other late-acting ISC factors such as Isa1, Isa2, and Iba57 (Gelling et al., 2008; Mühlenhoff et al., 2011). This notion was also evident from the lack of any effects on COX activity in all tested BOL deletion strains (Figure 1E). A severe COX diminution is observed in ISA1, ISA2, or IBA57 mutants, yet the exact reason is unknown (Gelling et al., 2008; Mühlenhoff et al., 2011). In order to verify that the loss of the enzyme activities was specific, BOL1 and BOL3 genes were reintroduced into the bol123Δ deletion strain by expression from plasmids. Both Bol1 and Bol3 were able to efficiently rescue the deficiencies in SDH activities (Figure 1—figure supplement 4).10.7554/eLife.16673.003Figure 1.Deficiency of both mitochondrial Bol proteins causes defects in a subset of mitochondrial [4Fe-4S] enzymes.Wild-type (WT; strain BY4742), and the indicated BOL and NFU1 deletion yeast strains were grown in minimal medium containing 2% galactose supplemented with 50 µM ferric ammonium citrate and used for the preparation of mitochondria. Mitochondrial extracts were assayed for specific activities of (A) aconitase (ACO), (B) succinate dehydrogenase (SDH), (C) pyruvate dehydrogenase (PDH), (D) 2-ketoglutarate dehydrogenase (KGDH), and (E) cytochrome c oxidase (COX). Values were normalized to those of malate dehydrogenase (MDH). Error bars indicate the SEM (n≥4). (F) Mitochondrial extracts were subjected to TCA precipitation and the levels of the indicated proteins and of lipoic acid attached to E2 subunits of PDH and KGDH were determined by immunostaining. Staining for porin served as a loading control.DOI:
http://dx.doi.org/10.7554/eLife.16673.00310.7554/eLife.16673.004Figure 1—figure supplement 1.Cartoon of bacterial BolA and eukaryotic Bol1, Bol2 and Bol3 proteins.Conserved amino acid sequence elements that characterize the various members of the BOLA protein family are shown. A conserved His (H*) in Bol2 serves as a Fe/S cluster ligand in a hetero-dimeric complex with Grx3 (Li et al., 2011). MTS; mitochondrial targeting sequence.DOI:
http://dx.doi.org/10.7554/eLife.16673.00410.7554/eLife.16673.005Figure 1—figure supplement 2.Bol1 and Bol3 are proteins of the mitochondrial matrix.(A) Wild-type yeast cells (W303) expressing Myc-tagged Bol1 (vector p424-MET25; Supplementary file 1B) or HA-tagged Bol3 (from vector p425-TDH3) were grown in SD medium, and mitochondria (Mito) and post-mitochondrial supernatant (PMS) fractions were isolated. Immunostaining was performed using antisera raised against the mitochondrial matrix protein Mge1, cytosolic Grx4 (also recognizing Grx3), and monoclonal anti-Myc (A14, Santa Cruz) or anti-HA. (B) Wild-type yeast cells (W303) expressing Bol1-Myc or Bol3-HA were grown in SD medium. Isolated mitochondria were either left intact (sorbitol), hypotonically swollen by tenfold dilution in HEPES buffer, or lysed with 0.2% Triton X-100 detergent (Diekert et al., 2001). Samples were incubated in the presence or absence of 100 µg/ml proteinase K for 20 min on ice, and proteinase K was inactivated by PMSF. After TCA precipitation, samples were analyzed by immunostaining for the Myc-tag of Bol1, the HA-tag of Bol3, the intermembrane space proteins Erv1 and cytochrome b2, and the matrix proteins Tim44 and Mge1.DOI:
http://dx.doi.org/10.7554/eLife.16673.00510.7554/eLife.16673.006Figure 1—figure supplement 3.The growth behavior of the various BOL gene deletion strains in comparison to NFU1 deletion cells.The indicated yeast strains were grown overnight on minimal medium with 2% glucose as carbon source. Cells were diluted to OD600 = 0.5 in water and spotted on rich medium (YP) agar plates with 2% galactose, 3% glycerol or 2% acetate as indicated. Plates were incubated at 30°C for 3 days. WT, wild-type.DOI:
http://dx.doi.org/10.7554/eLife.16673.00610.7554/eLife.16673.007Figure 1—figure supplement 4.Specific rescue of succinate dehydrogenase activities in BOL gene deletion cells by both mitochondrial Bol proteins.Wild-type (WT; strain BY4742) and the indicated BOL deletion strains were transformed with vector p416-MET25 lacking (empty) or overexpressing BOL1 or BOL3 as indicated. Cells were grown in minimal medium with 2% glucose without uracil and used for preparation of mitochondria. Mitochondrial extracts were assayed for the specific activity of succinate dehydrogenase (SDH) and normalized to malate dehydrogenase (MDH) activities. Error bars indicate the SEM (n≥4).DOI:
http://dx.doi.org/10.7554/eLife.16673.00710.7554/eLife.16673.008Figure 1—figure supplement 5.Fe/S enzyme activity defects in cells lacking Bol1-Bol3 after growth in lactate medium.Wild-type (WT; strain BY4742), the indicated BOL and NFU1 deletion yeast strains were grown in lactate medium and used for the preparation of mitochondria. Mitochondrial extracts were assayed for specific activities of (A) aconitase (ACO), (B) succinate dehydrogenase (SDH), (C) pyruvate dehydrogenase (PDH), and (D) 2-ketoglutarate dehydrogenase (KGDH). Values were normalized to those of malate dehydrogenase (MDH). Error bars indicate the SEM (n≥4).DOI:
http://dx.doi.org/10.7554/eLife.16673.008Previous studies on yeast ISC protein depletions, e.g., of the Isa proteins, had shown that the Fe/S protein defects were much more pronounced under strict respiratory conditions, e.g., upon growth in lactate medium (Kispal et al., 1999; Mühlenhoff et al., 2011) (see also Figure 1—figure supplement 3). Both aconitase and SDH activities of the various BOL deletion strains behaved similarly after growth in lactate- compared to galactose-containing media (Figure 1—figure supplement 5). In contrast, PDH and KGDH activities were much more severely (ten- vs. threefold) diminished in bol13Δ, bol123Δ, and nfu1Δ cells grown in lactate medium, similar to previous reports on Isa protein depletion (Kaut et al., 2000; Mühlenhoff et al., 2011). Together, these data show that the mBols were crucial for efficient maturation of LIAS, yet they play only an auxiliary role in SDH maturation. The levels of Bol1 and Nfu1 did not change or rather slightly decreased in the various deletion cells making a compensatory effect unlikely (data not shown; Bol3 levels could not be tested due to the lack of antibody detection). Obviously, Bol1 and Bol3 perform largely complementary roles, because only simultaneous deletion of both BOL1-BOL3 genes created a significant LIAS defect.

Deficiency of Bol1-Bol3 causes defects in a subset of mitochondrial [4Fe-4S] enzymes

We first analyzed the sub-cellular localization of S. cerevisiae Bol1 (YAL044W-A) and Bol3 (formerly termed Aim1; (Hess et al., 2009) by cell fractionation. Both proteins were exclusively present in the mitochondrial fraction and absent in the cytosol (Figure 1—figure supplement 2A). Upon sub-fractionation of mitochondria by hypotonic swelling or detergent lysis, the Bol proteins behaved similar to the matrix proteins Tim44 and Mge1 (Figure 1—figure supplement 2B). Thus, Bol1 and Bol3 are constituents of the mitochondrial matrix. To examine the physiological function of mitochondrial Bol1-Bol3 (mBols), we created single and double deletion cells, as well as combined deletions with BOL2 (Supplementary file 1A; strain background BY4742). All deletion strains grew at wild-type rates on minimal media containing the non-fermentable carbon sources glycerol or acetate with the exception of bol13Δ and bol123Δ strains (Figure 1—figure supplement 3). Apparently, simultaneous deletion of BOL1 and BOL3 created a weak respiratory defect which is slightly weaker than that seen for NFU1 deletion (Schilke et al., 1999).

We then analyzed key mitochondrial enzyme activities to detect potential defects in Fe/S protein biogenesis, and we compared these results to data for a NFU1 deletion strain which shows mild, yet specific Fe/S enzyme defects (Navarro-Sastre et al., 2011; Schilke et al., 1999). This analysis included the mitochondrial Fe/S proteins aconitase, succinate dehydrogenase (SDH or complex II), and yeast LIAS (Lip5). The latter can be assayed indirectly either by the enzyme activities or the lipoylation extent of pyruvate dehydrogenase (PDH) and 2-ketoglutarate dehydrogenase (KGDH) (Gelling et al., 2008; Schonauer et al., 2009). Further, we measured the activity of cytochrome c oxidase (COX). Even though this enzyme does not contain Fe/S clusters, it is frequently affected by mitochondrial Fe/S protein biogenesis defects, for unknown reasons (see, e.g., [Mühlenhoff et al., 2011; Sheftel et al., 2012]). All enzyme activities were normalized to malate dehydrogenase (MDH) that showed no significant changes in all these strains relative to wild type (not shown).

Single deletions of BOL1 or BOL3 did not affect any of these enzyme activities relative to wild-type cells with the exception of a slight, but significant decrease of SDH in bol3Δ cells (Figure 1A–E, bars 1–3). In contrast, double BOL1-BOL3 deletions resulted in substantial, up to threefold decreases of the Fe/S cluster-dependent enzyme activities, an effect comparable to that seen for nfu1Δ cells (bars 4 and 6) (Navarro-Sastre et al., 2011). A notable exception was aconitase which retained normal activity in bol13Δ cells (Figure 1A). Thus, double but not single yeast BOL gene deletions created a similar phenotype as that observed in BOLA3 patient cells (Cameron et al., 2011). Additional ablation of BOL2 (strain bol123Δ, bar 5) did not substantially exacerbate these effects suggesting that the mitochondrial and cytosolic Bol proteins act independently. The decrease in SDH was likely due to a direct defect in Fe/S cluster assembly, because SDH protein levels remained unchanged in immunoblots (Figure 1F). In keeping with the decreased PDH and KGDH activities, we also found a lipoylation defect of the E2 subunits of these enzymes by immunostaining against lipoic acid (Figure 1F) (Gelling et al., 2008; Schonauer et al., 2009). All the observed effects were rather mild in comparison to defects reported for deletion mutants of other late-acting ISC factors such as Isa1, Isa2, and Iba57 (Gelling et al., 2008; Mühlenhoff et al., 2011). This notion was also evident from the lack of any effects on COX activity in all tested BOL deletion strains (Figure 1E). A severe COX diminution is observed in ISA1, ISA2, or IBA57 mutants, yet the exact reason is unknown (Gelling et al., 2008; Mühlenhoff et al., 2011). In order to verify that the loss of the enzyme activities was specific, BOL1 and BOL3 genes were reintroduced into the bol123Δ deletion strain by expression from plasmids. Both Bol1 and Bol3 were able to efficiently rescue the deficiencies in SDH activities (Figure 1—figure supplement 4).10.7554/eLife.16673.003Figure 1.Deficiency of both mitochondrial Bol proteins causes defects in a subset of mitochondrial [4Fe-4S] enzymes.Wild-type (WT; strain BY4742), and the indicated BOL and NFU1 deletion yeast strains were grown in minimal medium containing 2% galactose supplemented with 50 µM ferric ammonium citrate and used for the preparation of mitochondria. Mitochondrial extracts were assayed for specific activities of (A) aconitase (ACO), (B) succinate dehydrogenase (SDH), (C) pyruvate dehydrogenase (PDH), (D) 2-ketoglutarate dehydrogenase (KGDH), and (E) cytochrome c oxidase (COX). Values were normalized to those of malate dehydrogenase (MDH). Error bars indicate the SEM (n≥4). (F) Mitochondrial extracts were subjected to TCA precipitation and the levels of the indicated proteins and of lipoic acid attached to E2 subunits of PDH and KGDH were determined by immunostaining. Staining for porin served as a loading control.DOI:
http://dx.doi.org/10.7554/eLife.16673.00310.7554/eLife.16673.004Figure 1—figure supplement 1.Cartoon of bacterial BolA and eukaryotic Bol1, Bol2 and Bol3 proteins.Conserved amino acid sequence elements that characterize the various members of the BOLA protein family are shown. A conserved His (H*) in Bol2 serves as a Fe/S cluster ligand in a hetero-dimeric complex with Grx3 (Li et al., 2011). MTS; mitochondrial targeting sequence.DOI:
http://dx.doi.org/10.7554/eLife.16673.00410.7554/eLife.16673.005Figure 1—figure supplement 2.Bol1 and Bol3 are proteins of the mitochondrial matrix.(A) Wild-type yeast cells (W303) expressing Myc-tagged Bol1 (vector p424-MET25; Supplementary file 1B) or HA-tagged Bol3 (from vector p425-TDH3) were grown in SD medium, and mitochondria (Mito) and post-mitochondrial supernatant (PMS) fractions were isolated. Immunostaining was performed using antisera raised against the mitochondrial matrix protein Mge1, cytosolic Grx4 (also recognizing Grx3), and monoclonal anti-Myc (A14, Santa Cruz) or anti-HA. (B) Wild-type yeast cells (W303) expressing Bol1-Myc or Bol3-HA were grown in SD medium. Isolated mitochondria were either left intact (sorbitol), hypotonically swollen by tenfold dilution in HEPES buffer, or lysed with 0.2% Triton X-100 detergent (Diekert et al., 2001). Samples were incubated in the presence or absence of 100 µg/ml proteinase K for 20 min on ice, and proteinase K was inactivated by PMSF. After TCA precipitation, samples were analyzed by immunostaining for the Myc-tag of Bol1, the HA-tag of Bol3, the intermembrane space proteins Erv1 and cytochrome b2, and the matrix proteins Tim44 and Mge1.DOI:
http://dx.doi.org/10.7554/eLife.16673.00510.7554/eLife.16673.006Figure 1—figure supplement 3.The growth behavior of the various BOL gene deletion strains in comparison to NFU1 deletion cells.The indicated yeast strains were grown overnight on minimal medium with 2% glucose as carbon source. Cells were diluted to OD600 = 0.5 in water and spotted on rich medium (YP) agar plates with 2% galactose, 3% glycerol or 2% acetate as indicated. Plates were incubated at 30°C for 3 days. WT, wild-type.DOI:
http://dx.doi.org/10.7554/eLife.16673.00610.7554/eLife.16673.007Figure 1—figure supplement 4.Specific rescue of succinate dehydrogenase activities in BOL gene deletion cells by both mitochondrial Bol proteins.Wild-type (WT; strain BY4742) and the indicated BOL deletion strains were transformed with vector p416-MET25 lacking (empty) or overexpressing BOL1 or BOL3 as indicated. Cells were grown in minimal medium with 2% glucose without uracil and used for preparation of mitochondria. Mitochondrial extracts were assayed for the specific activity of succinate dehydrogenase (SDH) and normalized to malate dehydrogenase (MDH) activities. Error bars indicate the SEM (n≥4).DOI:
http://dx.doi.org/10.7554/eLife.16673.00710.7554/eLife.16673.008Figure 1—figure supplement 5.Fe/S enzyme activity defects in cells lacking Bol1-Bol3 after growth in lactate medium.Wild-type (WT; strain BY4742), the indicated BOL and NFU1 deletion yeast strains were grown in lactate medium and used for the preparation of mitochondria. Mitochondrial extracts were assayed for specific activities of (A) aconitase (ACO), (B) succinate dehydrogenase (SDH), (C) pyruvate dehydrogenase (PDH), and (D) 2-ketoglutarate dehydrogenase (KGDH). Values were normalized to those of malate dehydrogenase (MDH). Error bars indicate the SEM (n≥4).DOI:
http://dx.doi.org/10.7554/eLife.16673.008

Deficiency of both mitochondrial Bol proteins causes defects in a subset of mitochondrial [4Fe-4S] enzymes.

Wild-type (WT; strain BY4742), and the indicated BOL and NFU1 deletion yeast strains were grown in minimal medium containing 2% galactose supplemented with 50 µM ferric ammonium citrate and used for the preparation of mitochondria. Mitochondrial extracts were assayed for specific activities of (A) aconitase (ACO), (B) succinate dehydrogenase (SDH), (C) pyruvate dehydrogenase (PDH), (D) 2-ketoglutarate dehydrogenase (KGDH), and (E) cytochrome c oxidase (COX). Values were normalized to those of malate dehydrogenase (MDH). Error bars indicate the SEM (n≥4). (F) Mitochondrial extracts were subjected to TCA precipitation and the levels of the indicated proteins and of lipoic acid attached to E2 subunits of PDH and KGDH were determined by immunostaining. Staining for porin served as a loading control.

DOI:
http://dx.doi.org/10.7554/eLife.16673.003

10.7554/eLife.16673.004Figure 1—figure supplement 1.Cartoon of bacterial BolA and eukaryotic Bol1, Bol2 and Bol3 proteins.Conserved amino acid sequence elements that characterize the various members of the BOLA protein family are shown. A conserved His (H*) in Bol2 serves as a Fe/S cluster ligand in a hetero-dimeric complex with Grx3 (Li et al., 2011). MTS; mitochondrial targeting sequence.DOI:
http://dx.doi.org/10.7554/eLife.16673.004

Cartoon of bacterial BolA and eukaryotic Bol1, Bol2 and Bol3 proteins.

Conserved amino acid sequence elements that characterize the various members of the BOLA protein family are shown. A conserved His (H*) in Bol2 serves as a Fe/S cluster ligand in a hetero-dimeric complex with Grx3 (Li et al., 2011). MTS; mitochondrial targeting sequence.

DOI:
http://dx.doi.org/10.7554/eLife.16673.004

10.7554/eLife.16673.005Figure 1—figure supplement 2.Bol1 and Bol3 are proteins of the mitochondrial matrix.(A) Wild-type yeast cells (W303) expressing Myc-tagged Bol1 (vector p424-MET25; Supplementary file 1B) or HA-tagged Bol3 (from vector p425-TDH3) were grown in SD medium, and mitochondria (Mito) and post-mitochondrial supernatant (PMS) fractions were isolated. Immunostaining was performed using antisera raised against the mitochondrial matrix protein Mge1, cytosolic Grx4 (also recognizing Grx3), and monoclonal anti-Myc (A14, Santa Cruz) or anti-HA. (B) Wild-type yeast cells (W303) expressing Bol1-Myc or Bol3-HA were grown in SD medium. Isolated mitochondria were either left intact (sorbitol), hypotonically swollen by tenfold dilution in HEPES buffer, or lysed with 0.2% Triton X-100 detergent (Diekert et al., 2001). Samples were incubated in the presence or absence of 100 µg/ml proteinase K for 20 min on ice, and proteinase K was inactivated by PMSF. After TCA precipitation, samples were analyzed by immunostaining for the Myc-tag of Bol1, the HA-tag of Bol3, the intermembrane space proteins Erv1 and cytochrome b2, and the matrix proteins Tim44 and Mge1.DOI:
http://dx.doi.org/10.7554/eLife.16673.005

Bol1 and Bol3 are proteins of the mitochondrial matrix.

(A) Wild-type yeast cells (W303) expressing Myc-tagged Bol1 (vector p424-MET25; Supplementary file 1B) or HA-tagged Bol3 (from vector p425-TDH3) were grown in SD medium, and mitochondria (Mito) and post-mitochondrial supernatant (PMS) fractions were isolated. Immunostaining was performed using antisera raised against the mitochondrial matrix protein Mge1, cytosolic Grx4 (also recognizing Grx3), and monoclonal anti-Myc (A14, Santa Cruz) or anti-HA. (B) Wild-type yeast cells (W303) expressing Bol1-Myc or Bol3-HA were grown in SD medium. Isolated mitochondria were either left intact (sorbitol), hypotonically swollen by tenfold dilution in HEPES buffer, or lysed with 0.2% Triton X-100 detergent (Diekert et al., 2001). Samples were incubated in the presence or absence of 100 µg/ml proteinase K for 20 min on ice, and proteinase K was inactivated by PMSF. After TCA precipitation, samples were analyzed by immunostaining for the Myc-tag of Bol1, the HA-tag of Bol3, the intermembrane space proteins Erv1 and cytochrome b2, and the matrix proteins Tim44 and Mge1.

DOI:
http://dx.doi.org/10.7554/eLife.16673.005

10.7554/eLife.16673.006Figure 1—figure supplement 3.The growth behavior of the various BOL gene deletion strains in comparison to NFU1 deletion cells.The indicated yeast strains were grown overnight on minimal medium with 2% glucose as carbon source. Cells were diluted to OD600 = 0.5 in water and spotted on rich medium (YP) agar plates with 2% galactose, 3% glycerol or 2% acetate as indicated. Plates were incubated at 30°C for 3 days. WT, wild-type.DOI:
http://dx.doi.org/10.7554/eLife.16673.006

The growth behavior of the various BOL gene deletion strains in comparison to NFU1 deletion cells.

The indicated yeast strains were grown overnight on minimal medium with 2% glucose as carbon source. Cells were diluted to OD600 = 0.5 in water and spotted on rich medium (YP) agar plates with 2% galactose, 3% glycerol or 2% acetate as indicated. Plates were incubated at 30°C for 3 days. WT, wild-type.

DOI:
http://dx.doi.org/10.7554/eLife.16673.006

10.7554/eLife.16673.007Figure 1—figure supplement 4.Specific rescue of succinate dehydrogenase activities in BOL gene deletion cells by both mitochondrial Bol proteins.Wild-type (WT; strain BY4742) and the indicated BOL deletion strains were transformed with vector p416-MET25 lacking (empty) or overexpressing BOL1 or BOL3 as indicated. Cells were grown in minimal medium with 2% glucose without uracil and used for preparation of mitochondria. Mitochondrial extracts were assayed for the specific activity of succinate dehydrogenase (SDH) and normalized to malate dehydrogenase (MDH) activities. Error bars indicate the SEM (n≥4).DOI:
http://dx.doi.org/10.7554/eLife.16673.007

Specific rescue of succinate dehydrogenase activities in BOL gene deletion cells by both mitochondrial Bol proteins.

Wild-type (WT; strain BY4742) and the indicated BOL deletion strains were transformed with vector p416-MET25 lacking (empty) or overexpressing BOL1 or BOL3 as indicated. Cells were grown in minimal medium with 2% glucose without uracil and used for preparation of mitochondria. Mitochondrial extracts were assayed for the specific activity of succinate dehydrogenase (SDH) and normalized to malate dehydrogenase (MDH) activities. Error bars indicate the SEM (n≥4).

DOI:
http://dx.doi.org/10.7554/eLife.16673.007

10.7554/eLife.16673.008Figure 1—figure supplement 5.Fe/S enzyme activity defects in cells lacking Bol1-Bol3 after growth in lactate medium.Wild-type (WT; strain BY4742), the indicated BOL and NFU1 deletion yeast strains were grown in lactate medium and used for the preparation of mitochondria. Mitochondrial extracts were assayed for specific activities of (A) aconitase (ACO), (B) succinate dehydrogenase (SDH), (C) pyruvate dehydrogenase (PDH), and (D) 2-ketoglutarate dehydrogenase (KGDH). Values were normalized to those of malate dehydrogenase (MDH). Error bars indicate the SEM (n≥4).DOI:
http://dx.doi.org/10.7554/eLife.16673.008

Fe/S enzyme activity defects in cells lacking Bol1-Bol3 after growth in lactate medium.

Wild-type (WT; strain BY4742), the indicated BOL and NFU1 deletion yeast strains were grown in lactate medium and used for the preparation of mitochondria. Mitochondrial extracts were assayed for specific activities of (A) aconitase (ACO), (B) succinate dehydrogenase (SDH), (C) pyruvate dehydrogenase (PDH), and (D) 2-ketoglutarate dehydrogenase (KGDH). Values were normalized to those of malate dehydrogenase (MDH). Error bars indicate the SEM (n≥4).

DOI:
http://dx.doi.org/10.7554/eLife.16673.008

Previous studies on yeast ISC protein depletions, e.g., of the Isa proteins, had shown that the Fe/S protein defects were much more pronounced under strict respiratory conditions, e.g., upon growth in lactate medium (Kispal et al., 1999; Mühlenhoff et al., 2011) (see also Figure 1—figure supplement 3). Both aconitase and SDH activities of the various BOL deletion strains behaved similarly after growth in lactate- compared to galactose-containing media (Figure 1—figure supplement 5). In contrast, PDH and KGDH activities were much more severely (ten- vs. threefold) diminished in bol13Δ, bol123Δ, and nfu1Δ cells grown in lactate medium, similar to previous reports on Isa protein depletion (Kaut et al., 2000; Mühlenhoff et al., 2011). Together, these data show that the mBols were crucial for efficient maturation of LIAS, yet they play only an auxiliary role in SDH maturation. The levels of Bol1 and Nfu1 did not change or rather slightly decreased in the various deletion cells making a compensatory effect unlikely (data not shown; Bol3 levels could not be tested due to the lack of antibody detection). Obviously, Bol1 and Bol3 perform largely complementary roles, because only simultaneous deletion of both BOL1-BOL3 genes created a significant LIAS defect.

Bol1 and Bol3 are crucial for de novo Fe/S cluster maturation of specific mitochondrial [4Fe-4S] but not [2Fe-2S] proteinsTo directly characterize the mBol involvement in mitochondrial Fe/S cluster assembly, we employed a well-established 55Fe radiolabeling and immunoprecipitation assay. This also allowed the testing of additional Fe/S proteins including such with [2Fe-2S] clusters. The various BOL deletion strains were radiolabeled with 55Fe, the Fe/S proteins were immunoprecipitated, and the precipitated radioactivity was quantified by scintillation counting (Pierik et al., 2009). Consistent with the enzyme activity data (Figure 1), we found no significant effect on 55Fe incorporation into aconitase upon single, double or triple BOL gene deletion (Figure 2A). For SDH (assayed by precipitation of the myc-tagged Fe/S cluster subunit Sdh2 co-expressed with Sdh1) a 40% decrease in 55Fe binding was observed upon deletion of BOL3, also in combination with BOL1 and BOL2 deletions (Figure 2B). Compared to other ISC gene deletions (see, e.g., Gelling et al., 2008; Gerber et al., 2004; Mühlenhoff et al., 2011; Rodriguez-Manzaneque et al., 2002; Voisine et al., 2001) this effect was rather weak. Similar, yet slightly stronger decreases in 55Fe binding (60%) were detected for mitochondria-targeted HiPIP, a bacterial [4Fe-4S] cluster-containing ferredoxin (Figure 2C). For all the tested Fe/S proteins, single deletion of BOL1 had no effects on 55Fe binding. The observed defects were specific, because reintroduction of Bol1 or Bol3 into the bol123Δ strain via overproducing vectors completely restored wild-type 55Fe binding activity, whereas cytosolic Bol2 or Grx5, as a control, were ineffective (Figure 2D). Taken together, these data support a specific function of the mBols in the maturation of a subset of mitochondrial Fe/S proteins, whereby Bol1 and Bol3 perform largely overlapping, non-essential functions. Interestingly, the 55Fe radiolabeling assay revealed a slightly more important role for Bol3 as compared to Bol1. This tendency, yet less well pronounced, was also observed for the enzyme activities (see Figure 1). The ability of the mBols to mutually substitute each other explains why Fe/S protein defects are best visible when both BOL1 and BOL3 are deleted.10.7554/eLife.16673.009Figure 2.Bol1-Bol3 are required for de novo Fe/S cluster incorporation into specific mitochondrial [4Fe-4S] but not [2Fe-2S] proteins.(A–F) Wild-type (WT, strain BY4742) and the indicated BOL deletion strains were transformed with vectors overproducing (B) Sdh2-Myc and Sdh1, (C and D) HiPIP-Myc, (E) human FDX2-HA, or (F) Schizosaccharomyces pombe (Sp) Grx5-Myc. In part D, bol123Δ cells were additionally transformed with p414-MET25 lacking or containing BOL1, BOL2, or BOL3 genes or p424-TDH3-GRX5. Cells were grown overnight in iron-poor SD medium and radiolabeled with 10 µCi 55Fe for 2 hr. The overproduced proteins were immunoprecipitated from cell extracts with specific antibodies. The amount of co-precipitated 55Fe was quantified by scintillation counting. Error bars indicate the SEM (n≥4).DOI:
http://dx.doi.org/10.7554/eLife.16673.009All Fe/S proteins tested so far contain [4Fe-4S] clusters. To get an insight into the Fe/S cluster-type specificity of mBol function, we examined the 55Fe incorporation into two mitochondrial [2Fe-2S] proteins, namely ferredoxin FDX2-HA, the human ortholog of yeast Yah1, and Grx5 from Schizosaccharomyces pombe (SpGrx5-Myc) (Gelling et al., 2008; Uzarska et al., 2013). The deficiency in mBols did not elicit any significant decrease in 55Fe binding to these [2Fe-2S] proteins (Figure 2E,F). This behavior is strikingly different from that of members of the core ISC machinery, and shows that the mBols were not required for maturation of [2Fe-2S] proteins. We conclude that Bol1-Bol3 are dedicated ISC assembly factors specifically involved in the maturation of a subset of mitochondrial [4Fe-4S] but not of [2Fe-2S] proteins. Further, the mBols act after Grx5, the final core ISC component (Uzarska et al., 2013), because they were not needed for 55Fe incorporation into Grx5.

Bol1 and Bol3 are crucial for de novo Fe/S cluster maturation of specific mitochondrial [4Fe-4S] but not [2Fe-2S] proteins

To directly characterize the mBol involvement in mitochondrial Fe/S cluster assembly, we employed a well-established 55Fe radiolabeling and immunoprecipitation assay. This also allowed the testing of additional Fe/S proteins including such with [2Fe-2S] clusters. The various BOL deletion strains were radiolabeled with 55Fe, the Fe/S proteins were immunoprecipitated, and the precipitated radioactivity was quantified by scintillation counting (Pierik et al., 2009). Consistent with the enzyme activity data (Figure 1), we found no significant effect on 55Fe incorporation into aconitase upon single, double or triple BOL gene deletion (Figure 2A). For SDH (assayed by precipitation of the myc-tagged Fe/S cluster subunit Sdh2 co-expressed with Sdh1) a 40% decrease in 55Fe binding was observed upon deletion of BOL3, also in combination with BOL1 and BOL2 deletions (Figure 2B). Compared to other ISC gene deletions (see, e.g., Gelling et al., 2008; Gerber et al., 2004; Mühlenhoff et al., 2011; Rodriguez-Manzaneque et al., 2002; Voisine et al., 2001) this effect was rather weak. Similar, yet slightly stronger decreases in 55Fe binding (60%) were detected for mitochondria-targeted HiPIP, a bacterial [4Fe-4S] cluster-containing ferredoxin (Figure 2C). For all the tested Fe/S proteins, single deletion of BOL1 had no effects on 55Fe binding. The observed defects were specific, because reintroduction of Bol1 or Bol3 into the bol123Δ strain via overproducing vectors completely restored wild-type 55Fe binding activity, whereas cytosolic Bol2 or Grx5, as a control, were ineffective (Figure 2D). Taken together, these data support a specific function of the mBols in the maturation of a subset of mitochondrial Fe/S proteins, whereby Bol1 and Bol3 perform largely overlapping, non-essential functions. Interestingly, the 55Fe radiolabeling assay revealed a slightly more important role for Bol3 as compared to Bol1. This tendency, yet less well pronounced, was also observed for the enzyme activities (see Figure 1). The ability of the mBols to mutually substitute each other explains why Fe/S protein defects are best visible when both BOL1 and BOL3 are deleted.10.7554/eLife.16673.009Figure 2.Bol1-Bol3 are required for de novo Fe/S cluster incorporation into specific mitochondrial [4Fe-4S] but not [2Fe-2S] proteins.(A–F) Wild-type (WT, strain BY4742) and the indicated BOL deletion strains were transformed with vectors overproducing (B) Sdh2-Myc and Sdh1, (C and D) HiPIP-Myc, (E) human FDX2-HA, or (F) Schizosaccharomyces pombe (Sp) Grx5-Myc. In part D, bol123Δ cells were additionally transformed with p414-MET25 lacking or containing BOL1, BOL2, or BOL3 genes or p424-TDH3-GRX5. Cells were grown overnight in iron-poor SD medium and radiolabeled with 10 µCi 55Fe for 2 hr. The overproduced proteins were immunoprecipitated from cell extracts with specific antibodies. The amount of co-precipitated 55Fe was quantified by scintillation counting. Error bars indicate the SEM (n≥4).DOI:
http://dx.doi.org/10.7554/eLife.16673.009

Bol1-Bol3 are required for de novo Fe/S cluster incorporation into specific mitochondrial [4Fe-4S] but not [2Fe-2S] proteins.

(A–F) Wild-type (WT, strain BY4742) and the indicated BOL deletion strains were transformed with vectors overproducing (B) Sdh2-Myc and Sdh1, (C and D) HiPIP-Myc, (E) human FDX2-HA, or (F) Schizosaccharomyces pombe (Sp) Grx5-Myc. In part D, bol123Δ cells were additionally transformed with p414-MET25 lacking or containing BOL1, BOL2, or BOL3 genes or p424-TDH3-GRX5. Cells were grown overnight in iron-poor SD medium and radiolabeled with 10 µCi 55Fe for 2 hr. The overproduced proteins were immunoprecipitated from cell extracts with specific antibodies. The amount of co-precipitated 55Fe was quantified by scintillation counting. Error bars indicate the SEM (n≥4).

DOI:
http://dx.doi.org/10.7554/eLife.16673.009

All Fe/S proteins tested so far contain [4Fe-4S] clusters. To get an insight into the Fe/S cluster-type specificity of mBol function, we examined the 55Fe incorporation into two mitochondrial [2Fe-2S] proteins, namely ferredoxin FDX2-HA, the human ortholog of yeast Yah1, and Grx5 from Schizosaccharomyces pombe (SpGrx5-Myc) (Gelling et al., 2008; Uzarska et al., 2013). The deficiency in mBols did not elicit any significant decrease in 55Fe binding to these [2Fe-2S] proteins (Figure 2E,F). This behavior is strikingly different from that of members of the core ISC machinery, and shows that the mBols were not required for maturation of [2Fe-2S] proteins. We conclude that Bol1-Bol3 are dedicated ISC assembly factors specifically involved in the maturation of a subset of mitochondrial [4Fe-4S] but not of [2Fe-2S] proteins. Further, the mBols act after Grx5, the final core ISC component (Uzarska et al., 2013), because they were not needed for 55Fe incorporation into Grx5.

Neither mitochondrial Bol1-Bol3 nor cytosolic Bol2 are required for cytosolic Fe/S protein biogenesisWe next examined if the mitochondrial Bol proteins perform an additional role in cytosolic Fe/S protein biogenesis, if they functionally cooperate with cytosolic Bol2, or if Bol2 alone plays a crucial role in this process. Previous studies on yeast Bol2 and human BOLA2 have suggested an involvement of these cytosolic proteins in intracellular iron metabolism (Kumanovics et al., 2008; Li et al., 2012). First, we measured the consequences of various BOL gene deletions on the activation of the Aft1-Aft2-dependent iron regulon, a hallmark of all cells with defects in the core ISC machinery (Lill et al., 2012; Outten and Albetel, 2013). As expected (Kumanovics et al., 2008), bol2Δ cells displayed a substantial activation of the Aft1-dependent FIT3 and FET3 genes in a GFP-based promoter assay (Figure 3A–B). In contrast, yeast strains lacking BOL1 and/or BOL3 behaved like wild-type cells indicating that the mBols are not involved in cellular iron regulation. This fits nicely to our conclusion above that Bol1-Bol3 do not belong to the core ISC system. The same situation was observed for cells lacking Nfu1 (Figure 3B), another known late-acting specific ISC assembly factor (Navarro-Sastre et al., 2011). Surprisingly, when the BOL2 deletion was combined with genetic ablation of BOL1 and/or BOL3, the iron regulon was activated slightly stronger than in bol2Δ cells (Figure 3A–B). This amplifying effect was specific for a Bol protein deficiency because it was reversed to the FET3 induction level of bol2Δ cells by reintroducing BOL1 and/or BOL3 into bol123Δ cells, and to wild-type FET3 levels by BOL2 expression (Figure 3B).10.7554/eLife.16673.010Figure 3.Mitochondrial Bol1-Bol3 and cytosolic Bol2 are not involved in cytosolic Fe/S protein biogenesis.Wild-type (WT, strain BY4742) and the indicated BOL deletion strains harboring vector (A) pFIT3-Luc2 or (B) pFET3-GFP were cultivated in iron-replete medium to mid-log phase and the activities of the FIT3 and FET3 promoters, respectively, were determined. In (B) the bol123Δ cells were also transformed with centromeric plasmids producing the indicated Bol proteins. Abbreviation: Bol13; Bol1 plus Bol3. (C) Leu1 (relative to MDH) activities were determined in the indicated deletion stains cultivated in iron-replete medium. (D) The indicated BOL deletion stains were radiolabeled with 55Fe, and the Leu1-bound radioactivity was determined by immunoprecipitation with α-Leu1 antibodies followed by scintillation counting. (E) 55Fe incorporation into Rli1-HA was determined accordingly using WT and bol123Δ cells transformed with centromeric plasmids producing the three Bol proteins as indicated or with the empty vector (-). (F) Representative immunoblots determining the protein levels of Leu1 (D) and Rli1-HA (E) in the indicated strains. Porin served as a loading control. Error bars indicate the SEM (n≥4).DOI:
http://dx.doi.org/10.7554/eLife.16673.010Second, we investigated the effects of the three BOL gene deletions on cytosolic Fe/S proteins by assaying the enzyme activity of isopropylmalate isomerase (Leu1). No changes were observed when BOL1-BOL3 genes were deleted either alone or in combination (Figure 3C). In contrast, bol2Δ cells showed a twofold decrease in Leu1 activity, which further declined upon additional deletion of BOL1-BOL3. Collectively, these results show that the mBols are not involved in cytosolic Fe/S protein biogenesis, yet there is a cross-talk of the mBol deficiency with the BOL2 deletion. To better understand the reason of this combinatory effect, we took advantage of the 55Fe radiolabeling assay described above. Deletion of either BOL1-BOL3 or BOL2 hardly affected 55Fe/S cluster binding to Leu1 indicating that neither mitochondrial Bol1-Bol3 nor cytosolic Bol2 perform a decisive role in Leu1 Fe/S cluster incorporation (Figure 3D,F). However, combining the BOL2 deletion with that of BOL1 and/or BOL3 resulted in an up to twofold decrease in 55Fe/S cluster association with Leu1. A similar twofold decrease (compared to wild-type) was seen for 55Fe incorporation into another cytosolic Fe/S protein, the ABC protein Rli1 using the triple BOL deletion strain bol123Δ (Figure 3E–F). We note, however, that these decreases in 55Fe incorporation were rather weak compared to defects in core ISC or CIA components (see, e.g., Netz et al., 2007, 2010; Paul et al., 2015). The Rli1 maturation defect was completely restored when any of the BOL genes was reintroduced into the bol123Δ mutant. This supports the notion that neither the mBols nor Bol2 are directly involved in cytosolic Fe/S protein maturation. Obviously, their combined absence causes alterations in both mitochondrial metabolism and cellular iron homeostasis, thereby creating conditions that negatively affect cytosolic Fe/S cluster insertion. The physiological conditions prevailing in bol123Δ cells might also explain the slight decrease in the enzyme activity of aconitase which, like Leu1, is a highly susceptible protein to, e.g., oxidative stress conditions or changes in iron availability (cf. Figures 1A and 3C). Together, our data indicate that the mBols execute a mitochondria-specific function in [4Fe-4S] protein assembly, whereas Bol2 plays a specific role in cellular iron regulation (Kumanovics et al., 2008).

Neither mitochondrial Bol1-Bol3 nor cytosolic Bol2 are required for cytosolic Fe/S protein biogenesis

We next examined if the mitochondrial Bol proteins perform an additional role in cytosolic Fe/S protein biogenesis, if they functionally cooperate with cytosolic Bol2, or if Bol2 alone plays a crucial role in this process. Previous studies on yeast Bol2 and human BOLA2 have suggested an involvement of these cytosolic proteins in intracellular iron metabolism (Kumanovics et al., 2008; Li et al., 2012). First, we measured the consequences of various BOL gene deletions on the activation of the Aft1-Aft2-dependent iron regulon, a hallmark of all cells with defects in the core ISC machinery (Lill et al., 2012; Outten and Albetel, 2013). As expected (Kumanovics et al., 2008), bol2Δ cells displayed a substantial activation of the Aft1-dependent FIT3 and FET3 genes in a GFP-based promoter assay (Figure 3A–B). In contrast, yeast strains lacking BOL1 and/or BOL3 behaved like wild-type cells indicating that the mBols are not involved in cellular iron regulation. This fits nicely to our conclusion above that Bol1-Bol3 do not belong to the core ISC system. The same situation was observed for cells lacking Nfu1 (Figure 3B), another known late-acting specific ISC assembly factor (Navarro-Sastre et al., 2011). Surprisingly, when the BOL2 deletion was combined with genetic ablation of BOL1 and/or BOL3, the iron regulon was activated slightly stronger than in bol2Δ cells (Figure 3A–B). This amplifying effect was specific for a Bol protein deficiency because it was reversed to the FET3 induction level of bol2Δ cells by reintroducing BOL1 and/or BOL3 into bol123Δ cells, and to wild-type FET3 levels by BOL2 expression (Figure 3B).10.7554/eLife.16673.010Figure 3.Mitochondrial Bol1-Bol3 and cytosolic Bol2 are not involved in cytosolic Fe/S protein biogenesis.Wild-type (WT, strain BY4742) and the indicated BOL deletion strains harboring vector (A) pFIT3-Luc2 or (B) pFET3-GFP were cultivated in iron-replete medium to mid-log phase and the activities of the FIT3 and FET3 promoters, respectively, were determined. In (B) the bol123Δ cells were also transformed with centromeric plasmids producing the indicated Bol proteins. Abbreviation: Bol13; Bol1 plus Bol3. (C) Leu1 (relative to MDH) activities were determined in the indicated deletion stains cultivated in iron-replete medium. (D) The indicated BOL deletion stains were radiolabeled with 55Fe, and the Leu1-bound radioactivity was determined by immunoprecipitation with α-Leu1 antibodies followed by scintillation counting. (E) 55Fe incorporation into Rli1-HA was determined accordingly using WT and bol123Δ cells transformed with centromeric plasmids producing the three Bol proteins as indicated or with the empty vector (-). (F) Representative immunoblots determining the protein levels of Leu1 (D) and Rli1-HA (E) in the indicated strains. Porin served as a loading control. Error bars indicate the SEM (n≥4).DOI:
http://dx.doi.org/10.7554/eLife.16673.010

Mitochondrial Bol1-Bol3 and cytosolic Bol2 are not involved in cytosolic Fe/S protein biogenesis.

Wild-type (WT, strain BY4742) and the indicated BOL deletion strains harboring vector (A) pFIT3-Luc2 or (B) pFET3-GFP were cultivated in iron-replete medium to mid-log phase and the activities of the FIT3 and FET3 promoters, respectively, were determined. In (B) the bol123Δ cells were also transformed with centromeric plasmids producing the indicated Bol proteins. Abbreviation: Bol13; Bol1 plus Bol3. (C) Leu1 (relative to MDH) activities were determined in the indicated deletion stains cultivated in iron-replete medium. (D) The indicated BOL deletion stains were radiolabeled with 55Fe, and the Leu1-bound radioactivity was determined by immunoprecipitation with α-Leu1 antibodies followed by scintillation counting. (E) 55Fe incorporation into Rli1-HA was determined accordingly using WT and bol123Δ cells transformed with centromeric plasmids producing the three Bol proteins as indicated or with the empty vector (-). (F) Representative immunoblots determining the protein levels of Leu1 (D) and Rli1-HA (E) in the indicated strains. Porin served as a loading control. Error bars indicate the SEM (n≥4).

DOI:
http://dx.doi.org/10.7554/eLife.16673.010

Second, we investigated the effects of the three BOL gene deletions on cytosolic Fe/S proteins by assaying the enzyme activity of isopropylmalate isomerase (Leu1). No changes were observed when BOL1-BOL3 genes were deleted either alone or in combination (Figure 3C). In contrast, bol2Δ cells showed a twofold decrease in Leu1 activity, which further declined upon additional deletion of BOL1-BOL3. Collectively, these results show that the mBols are not involved in cytosolic Fe/S protein biogenesis, yet there is a cross-talk of the mBol deficiency with the BOL2 deletion. To better understand the reason of this combinatory effect, we took advantage of the 55Fe radiolabeling assay described above. Deletion of either BOL1-BOL3 or BOL2 hardly affected 55Fe/S cluster binding to Leu1 indicating that neither mitochondrial Bol1-Bol3 nor cytosolic Bol2 perform a decisive role in Leu1 Fe/S cluster incorporation (Figure 3D,F). However, combining the BOL2 deletion with that of BOL1 and/or BOL3 resulted in an up to twofold decrease in 55Fe/S cluster association with Leu1. A similar twofold decrease (compared to wild-type) was seen for 55Fe incorporation into another cytosolic Fe/S protein, the ABC protein Rli1 using the triple BOL deletion strain bol123Δ (Figure 3E–F). We note, however, that these decreases in 55Fe incorporation were rather weak compared to defects in core ISC or CIA components (see, e.g., Netz et al., 2007, 2010; Paul et al., 2015). The Rli1 maturation defect was completely restored when any of the BOL genes was reintroduced into the bol123Δ mutant. This supports the notion that neither the mBols nor Bol2 are directly involved in cytosolic Fe/S protein maturation. Obviously, their combined absence causes alterations in both mitochondrial metabolism and cellular iron homeostasis, thereby creating conditions that negatively affect cytosolic Fe/S cluster insertion. The physiological conditions prevailing in bol123Δ cells might also explain the slight decrease in the enzyme activity of aconitase which, like Leu1, is a highly susceptible protein to, e.g., oxidative stress conditions or changes in iron availability (cf. Figures 1A and 3C). Together, our data indicate that the mBols execute a mitochondria-specific function in [4Fe-4S] protein assembly, whereas Bol2 plays a specific role in cellular iron regulation (Kumanovics et al., 2008).

Bol1-Bol3 and Nfu1 perform distinct tasks in mitochondrial [4Fe-4S] protein maturationOur current and previous results suggested functions of the mBols and of Nfu1 as specialized ISC assembly factors (Cameron et al., 2011; Navarro-Sastre et al., 2011). Are these functions independent or overlapping? We first analyzed whether simultaneous inactivation of Bol1-Bol3 and Nfu1 aggravates the Fe/S defects. As observed previously (Navarro-Sastre et al., 2011; Schilke et al., 1999), nfu1Δ cells showed a diminution in the activities of both aconitase and SDH relative to wild type (Figure 4A–B). A substantial further decrease to rather low aconitase and SDH activities was found for additional deletion of BOL3, but not of BOL1. The BOL3 effect was not further amplified in nfu1Δbol13Δ cells. These findings show that Bol3 plays a distinct, Nfu1-independent role that cannot be executed by Bol1. Apparently, even though Bol1 and Bol3 play overlapping roles (see above), their functions are not entirely identical.10.7554/eLife.16673.011Figure 4.Bol1-Bol3 and Nfu1 cannot functionally replace each other in Fe/S protein maturation.(A–B) Wild-type (WT, strain BY4742) and the indicated deletion strains were grown in minimal medium containing 2% glucose and used for the preparation of mitochondria. Mitochondrial extracts were assayed for the indicated specific enzyme activities as described in Figure 1. (C–F) The indicated deletion strains were transformed with vector p416-MET25 lacking (empty) or containing BOL3 or NFU1 as indicated. Cells were grown in minimal medium containing 2% galactose and used for the preparation of mitochondria. Mitochondrial extracts were assayed for the indicated enzyme activities as described in Figure 1. Error bars indicate the SEM (n≥4).DOI:
http://dx.doi.org/10.7554/eLife.16673.01110.7554/eLife.16673.012Figure 4—figure supplement 1.Growth complementation test of nfu1Δ and bol13Δ cells.(A) nfu1Δ and (B) bol13Δ cells were transformed with vector p414-MET25 containing no gene (empty), BOL1, BOL3 or BOL1-BOL3 (BOL13) or p416-MET25-NFU1 as indicated. Cells were grown overnight on minimal medium without uracil containing 2% glucose as a carbon source. Cells were diluted to OD600 of 0.1 in 500 µl of liquid minimal medium without uracil containing 2% acetate as sole carbon source, and put into a 48 well plate. Plates were incubated at 30°C for 3 days with shaking, and the OD600 was measured by a TECAN plate reader every 30 min.DOI:
http://dx.doi.org/10.7554/eLife.16673.012We next tested whether the Bol1-Bol3 and Nfu1 proteins can mutually substitute each other’s functions. To this end, we first took advantage of the growth defects of the bol13Δ and nfu1Δ strains in acetate-containing minimal medium (see above). Upon complementation of bol13Δ cells with BOL1 and/or BOL3 or of nfu1Δ cells with NFU1, wild-type growth rates were restored (Figure 4—figure supplement 1). In contrast, no significant growth defect complementation was observed for nfu1Δ cells upon BOL1 and/or BOL3 expression (part A). A weak complementation was seen for bol13Δ cells by NFU1 expression (part B). Notably, this slight rescue effect was not observed for growth on solid acetate medium (not shown). Further, we used these cells for measuring the enzyme activities of aconitase, SDH, PDH and KGDH. In agreement with the growth behavior, Bol3 was not able to rescue any of the enzyme defects of nfu1Δ cells, while Nfu1 did so (Figure 4C–F, left). Likewise, Nfu1 failed to restore the enzyme activities of bol13Δ cells, while Bol3 was complementing (Figure 4C–F, right). These findings show that Nfu1 and Bol3 cannot mutually substitute each other’s biochemical function, even upon overproduction. Despite the fact that Bol3 and Nfu1 both act late in mitochondrial Fe/S protein biogenesis, they appear to fulfil different tasks that cannot be taken over by the other protein. Hence, these data suggest that Bol1-Bol3 and Nfu1 perform individual, non-overlapping functions during Fe/S cluster assembly.

Bol1-Bol3 and Nfu1 perform distinct tasks in mitochondrial [4Fe-4S] protein maturation

Our current and previous results suggested functions of the mBols and of Nfu1 as specialized ISC assembly factors (Cameron et al., 2011; Navarro-Sastre et al., 2011). Are these functions independent or overlapping? We first analyzed whether simultaneous inactivation of Bol1-Bol3 and Nfu1 aggravates the Fe/S defects. As observed previously (Navarro-Sastre et al., 2011; Schilke et al., 1999), nfu1Δ cells showed a diminution in the activities of both aconitase and SDH relative to wild type (Figure 4A–B). A substantial further decrease to rather low aconitase and SDH activities was found for additional deletion of BOL3, but not of BOL1. The BOL3 effect was not further amplified in nfu1Δbol13Δ cells. These findings show that Bol3 plays a distinct, Nfu1-independent role that cannot be executed by Bol1. Apparently, even though Bol1 and Bol3 play overlapping roles (see above), their functions are not entirely identical.10.7554/eLife.16673.011Figure 4.Bol1-Bol3 and Nfu1 cannot functionally replace each other in Fe/S protein maturation.(A–B) Wild-type (WT, strain BY4742) and the indicated deletion strains were grown in minimal medium containing 2% glucose and used for the preparation of mitochondria. Mitochondrial extracts were assayed for the indicated specific enzyme activities as described in Figure 1. (C–F) The indicated deletion strains were transformed with vector p416-MET25 lacking (empty) or containing BOL3 or NFU1 as indicated. Cells were grown in minimal medium containing 2% galactose and used for the preparation of mitochondria. Mitochondrial extracts were assayed for the indicated enzyme activities as described in Figure 1. Error bars indicate the SEM (n≥4).DOI:
http://dx.doi.org/10.7554/eLife.16673.01110.7554/eLife.16673.012Figure 4—figure supplement 1.Growth complementation test of nfu1Δ and bol13Δ cells.(A) nfu1Δ and (B) bol13Δ cells were transformed with vector p414-MET25 containing no gene (empty), BOL1, BOL3 or BOL1-BOL3 (BOL13) or p416-MET25-NFU1 as indicated. Cells were grown overnight on minimal medium without uracil containing 2% glucose as a carbon source. Cells were diluted to OD600 of 0.1 in 500 µl of liquid minimal medium without uracil containing 2% acetate as sole carbon source, and put into a 48 well plate. Plates were incubated at 30°C for 3 days with shaking, and the OD600 was measured by a TECAN plate reader every 30 min.DOI:
http://dx.doi.org/10.7554/eLife.16673.012

Bol1-Bol3 and Nfu1 cannot functionally replace each other in Fe/S protein maturation.

(A–B) Wild-type (WT, strain BY4742) and the indicated deletion strains were grown in minimal medium containing 2% glucose and used for the preparation of mitochondria. Mitochondrial extracts were assayed for the indicated specific enzyme activities as described in Figure 1. (C–F) The indicated deletion strains were transformed with vector p416-MET25 lacking (empty) or containing BOL3 or NFU1 as indicated. Cells were grown in minimal medium containing 2% galactose and used for the preparation of mitochondria. Mitochondrial extracts were assayed for the indicated enzyme activities as described in Figure 1. Error bars indicate the SEM (n≥4).

DOI:
http://dx.doi.org/10.7554/eLife.16673.011

10.7554/eLife.16673.012Figure 4—figure supplement 1.Growth complementation test of nfu1Δ and bol13Δ cells.(A) nfu1Δ and (B) bol13Δ cells were transformed with vector p414-MET25 containing no gene (empty), BOL1, BOL3 or BOL1-BOL3 (BOL13) or p416-MET25-NFU1 as indicated. Cells were grown overnight on minimal medium without uracil containing 2% glucose as a carbon source. Cells were diluted to OD600 of 0.1 in 500 µl of liquid minimal medium without uracil containing 2% acetate as sole carbon source, and put into a 48 well plate. Plates were incubated at 30°C for 3 days with shaking, and the OD600 was measured by a TECAN plate reader every 30 min.DOI:
http://dx.doi.org/10.7554/eLife.16673.012

Growth complementation test of nfu1Δ and bol13Δ cells.

(A) nfu1Δ and (B) bol13Δ cells were transformed with vector p414-MET25 containing no gene (empty), BOL1, BOL3 or BOL1-BOL3 (BOL13) or p416-MET25-NFU1 as indicated. Cells were grown overnight on minimal medium without uracil containing 2% glucose as a carbon source. Cells were diluted to OD600 of 0.1 in 500 µl of liquid minimal medium without uracil containing 2% acetate as sole carbon source, and put into a 48 well plate. Plates were incubated at 30°C for 3 days with shaking, and the OD600 was measured by a TECAN plate reader every 30 min.

DOI:
http://dx.doi.org/10.7554/eLife.16673.012

We next tested whether the Bol1-Bol3 and Nfu1 proteins can mutually substitute each other’s functions. To this end, we first took advantage of the growth defects of the bol13Δ and nfu1Δ strains in acetate-containing minimal medium (see above). Upon complementation of bol13Δ cells with BOL1 and/or BOL3 or of nfu1Δ cells with NFU1, wild-type growth rates were restored (Figure 4—figure supplement 1). In contrast, no significant growth defect complementation was observed for nfu1Δ cells upon BOL1 and/or BOL3 expression (part A). A weak complementation was seen for bol13Δ cells by NFU1 expression (part B). Notably, this slight rescue effect was not observed for growth on solid acetate medium (not shown). Further, we used these cells for measuring the enzyme activities of aconitase, SDH, PDH and KGDH. In agreement with the growth behavior, Bol3 was not able to rescue any of the enzyme defects of nfu1Δ cells, while Nfu1 did so (Figure 4C–F, left). Likewise, Nfu1 failed to restore the enzyme activities of bol13Δ cells, while Bol3 was complementing (Figure 4C–F, right). These findings show that Nfu1 and Bol3 cannot mutually substitute each other’s biochemical function, even upon overproduction. Despite the fact that Bol3 and Nfu1 both act late in mitochondrial Fe/S protein biogenesis, they appear to fulfil different tasks that cannot be taken over by the other protein. Hence, these data suggest that Bol1-Bol3 and Nfu1 perform individual, non-overlapping functions during Fe/S cluster assembly.

Structural insights into the human mitochondrial BOLA proteins and the interaction with GLRX5To extend our cell biological insights into Bol protein function, we employed NMR structural studies to examine their physical interaction with Grx5. We used the human proteins for these in vitro studies because they were more stable than the yeast counterparts. First, the solution structures of human BOLA1 and BOLA3 were determined (Figure 5). A detailed description of the two structures and of their backbone dynamic properties is provided in Figure 5—figure supplement 1. According to the DALI server (Holm and Sander, 1993), both BOLA1 and BOLA3 show the highest structural similarity to BolA-like proteins from Arabidopsis thaliana, Babesia bovis, and Mus musculus (Abendroth et al., 2011; Kasai et al., 2004; Roret et al., 2014). In particular, the BOLA1 structure matches better with those homologues that have a longer loop between β1 and β2, while the BOLA3 structure is more similar to homologues having a shorter β1-β2 loop (Figure 5C and Figure 5—figure supplement 2). Apparently, the structural features of the loop between β1 and β2 are key for differentiating BOLA1 from BOLA3. Remarkably, this loop is closer to the ‘invariant’ His residue (His102 in BOLA1 and His96 in BOLA3; Figure 5), which has been shown to coordinate the [2Fe-2S] cluster in the yeast Bol2-Grx3 complex (Li et al., 2011). This region contains further possible Fe/S cluster ligands, namely His58, His67 in BOLA1 and Cys59 in BOLA3. Another conserved His residue (His86 in BOLA1 and His81 in BOLA3) is distant from this region and thus excluded as a cluster ligand (Figure 5). Together, structural data indicate that BOLA1 and BOLA3 have a similar fold, but show local structural differences in the regions containing the potential Fe/S cluster ligands.10.7554/eLife.16673.013Figure 5.NMR solution structures of human BOLA1 and BOLA3.The structures of (A) BOLA1 and (B) BOLA3 were solved by solution NMR. Residues His58, His67 and His102 are conserved within the eukaryotic BOLA1 proteins, and residues Cys59 and His96 are conserved within the BOLA3 proteins. (C) Backbone superimposition of BOLA1 (red) and BOLA3 (blue) structures depicting conserved His and Cys residues in both proteins. Yellow sticks represent potential Fe/S cluster ligands, and green sticks indicate other conserved His residues.DOI:
http://dx.doi.org/10.7554/eLife.16673.01310.7554/eLife.16673.014Figure 5—figure supplement 1.Structural and dynamic properties of BOLA1 and BOLA3 by solution NMR.(A) 2D 1H-15N HSQC spectra of 15N-labeled BOLA1 and BOLA3. (B) 15N R1, R2 relaxation rates and 15N(1H) NOE values per residue of BOLA1 and BOLA3 obtained at 500 MHz and 298 K. The proteins were in buffer N with 10% (v/v) D2O). The 1H-15N HSQC spectra of the apo forms of BOLA3 and BOLA1 showed well-dispersed resonances indicative of essentially folded proteins. The overall topology of the solution structure of human BOLA3 is α1β1β2η1α2η2β3 (h: 310-helix), in which β1 and β2 are antiparallel, and β3 is parallel to β2 (Figure 5). All α-helices are on the same side of the β-sheet and the 310-helix located between α2 and β3 contains the His96 residue that is invariant in all prokaryotic and eukaryotic BolA homologues (Li and Outten, 2012). A cysteine residue (Cys59), typically conserved in Bol3 homologues, is located close to the invariant His96 residue in an extended loop between β1 and β2. The two residues are not in the proper orientation to coordinate a [2Fe-2S] cluster (Sγ of Cys59 is at an average distance of ~8 Å from Nε2 of His96). The overall topology of the solution structure of BOLA1 is similar to that of BOLA3, with the exception of the presence, in BOLA1, of short helices in the loop regions and at the C-terminus (α1η1β1β2η2α2α3β3α4 topology) (Figure 5). In addition, at both termini BOLA1 has ca. 15 residues that are unstructured and whose backbone NH resonances are clustered in the central region of the 1H-15N HSQC spectrum (part B). The three His residues typically conserved in Bol1 homologues are located in two spatially close regions (Figure 5). Although they are close to each other, it is not possible to predict which ones may be involved in [2Fe-2S] cluster coordination. Information on backbone motions and on the protein oligomerization state of BOLA1 and BOLA3 were obtained through 15N R1, R2, heteronuclear 15N(1H)-NOEs NMR experiments. The 15N backbone relaxation properties showed that the N and C termini of BOLA1 are highly flexible, as about 15 to 20 residues on each terminus have negative 15N NOE values, while the folded domain is essentially rigid. BOLA3 is a rigid molecule, with the exception of the first detected residue at the N terminus, the last two residues at the C terminus and residues around invariant His96. All of them experience fast backbone motions in the ns-ps time scale, as indicated by their low or negative 15N NOE values.DOI:
http://dx.doi.org/10.7554/eLife.16673.01410.7554/eLife.16673.015Figure 5—figure supplement 2.Multi-sequence alignment of the mitochondrial Bol proteins.Multalin (Corpet, 1988) was used to generate a multi-sequence alignment of Bol1- and Bol3-like proteins from fungi and various higher eukaryotes. The conserved His and Cys residues are indicated. The numbers refer to the residues of the human mBols. Two His (58 and 67) in BOLA1 are candidates for the structural counterpart of Cys59 of BOLA3.DOI:
http://dx.doi.org/10.7554/eLife.16673.015In order to characterize the interaction between BOLA1 or BOLA3 and GLRX5 at the atomic level, we first studied the apo situation. 15N labeled BOLA proteins were titrated with unlabeled apo-GLRX5, and, vice versa,
15N labeled apo-GLRX5 with unlabeled BOLA proteins in the presence of 5 mM GSH. Meaningful chemical shift changes were observed for both titrations in 2D 1H-15N HSQC NMR spectra acquired upon addition of the unlabeled partner (Figure 6—figure supplements 1A and 2A). The free and bound proteins are in fast and intermediate exchange regimes relative to NMR time scale; the observed chemical shift changes of the protein amide resonances saturated at a 1:1 protein ratio. 15N R1 and R2 NMR relaxation data performed on the final 15N labeled BOLA1-unlabelled GLRX5 or 15N labeled BOLA3-unlabelled GLRX5 mixtures showed rotational correlation times consistent with the formation of a 1:1 heterodimeric complex (Figure 6—figure supplement 3). NMR titrations performed by adding unlabeled apo-GLRX5 to a 15N-labeled BOLA1/15N-labeled BOLA3 1:1 mixture showed the formation of both hetero-complexes at comparable levels, indicating similar affinities. This data fits nicely to thermophoresis results showing that both human BOLA proteins interacted with apo-GLRX5 with Kd values of 3 µM (Melber et al., 2016; accompanying manuscript).The chemical shift changes observed upon formation of apo-GLRX5-BOLA1 and apo-GLRX5-BOLA3 complexes were mapped on the solution structures of BOLA1, BOLA3 and apo-GLRX5. The interaction surface on GLRX5 in the two complexes comprises the GSH binding site and its surroundings (Figure 6A,B and Figure 6—figure supplement 2A). On the BOLAs, the interactions involve helix α2, part of the β-sheet and the invariant His residue (see above). The other possible Fe/S cluster ligands (His58, His67 in BOLA1 and Cys59 in BOLA3) showed no significant chemical shifts. In conclusion, NMR data indicate that apo-GLRX5-BOLA1 and apo-GLRX5-BOLA3 complexes specifically interact involving the region surrounding the invariant His in BOLAs and the GSH binding site in GLRX5.10.7554/eLife.16673.016Figure 6.Structural basis of the interaction between human apo- and holo-GLRX5 with the BOLA proteins.(A and B) Backbone chemical shift differences, obtained from a comparison of the 1H-15N HSQC spectra of apo-GLRX5 or BOLA proteins with that of (A) BOLA1-apo-GLRX5 or (B) BOLA3-apo-GLRX5 (1:1 mixture in buffer N), were mapped on the solution structures of the proteins. (C and D) Backbone chemical shift differences, obtained from a comparison of the 1H-15N HSQC spectrum of apo-GLXR5 or BOLA proteins with that of (C) BOLA1-apo-GLRX5 or (D) BOLA3-apo-GLRX5 (1:1 mixture in buffer N) chemically reconstituted with a [2Fe-2S] cluster, were mapped on the solution structures of the proteins. Green regions show residues with significant chemical shift changes (that is both in terms of chemical shift and broadening beyond detection effects) observed upon formation of the apo- or holo-complexes. Red areas depict those residues additionally affected upon holo-complex formation. Critical residues and GLRX5-bound GSH are depicted as sticks.DOI:
http://dx.doi.org/10.7554/eLife.16673.01610.7554/eLife.16673.017Figure 6—figure supplement 1.Chemical shift changes upon complex formation between GLRX5 and BOLA proteins monitoring backbone chemical shift changes of BOLA1 and BOLA3.Backbone weighted average chemical shift differences Δavg(HN) obtained by comparing 1H-15N HSQC map of (A) 15N-labeled BOLA1 or BOLA3 with an equimolar mixture of 15N-labeled BOLA1 or BOLA3 with unlabeled apo-GLRX5, and of (B) 15N-labeled BOLA1 or BOLA3 with an equimolar mixture of 15N-labeled BOLA1 or BOLA3 chemically reconstituted with unlabeled GLRX5 to generate a [2Fe-2S] cluster (buffer N). Bars with Δavg(HN) = 1 indicate residues whose backbone NH signals broaden beyond detection. The indicated thresholds (obtained by averaging Δavg(HN) values plus 1σ) were used to define significant chemical shift differences. Green bars show significant chemical shift changes (that is both chemical shift differences and broadening beyond detection effects, respectively) in the apo interactions, and red bars those additionally occurring upon holo complex formation. The position of conserved residues is highlighted.DOI:
http://dx.doi.org/10.7554/eLife.16673.01710.7554/eLife.16673.018Figure 6—figure supplement 2.Chemical shift changes upon complex formation between GLRX5 and BOLA proteins, monitoring backbone chemical shift changes of GLRX5.Backbone weighted average chemical shift differences Δavg(HN) obtained by comparing 1H-15N HSQC map of (A) 15N-labeled apo-GLRX5 with that of a 1:1 mixture of 15N-labeled apo-GLRX5 and unlabeled BOLA1 or BOLA3, and of (B) 15N-labeled apo-GLRX5 with that of a 1:1 mixture of 15N-labeled apo-GLRX5 with unlabeled BOLA1 or BOLA3 after chemical reconstitution of a [2Fe-2S] cluster in buffer N. The data are presented as in Figure 6—figure supplement 1. The position of the conserved Cys67 and of the residues of the GSH binding site are highlighted.DOI:
http://dx.doi.org/10.7554/eLife.16673.01810.7554/eLife.16673.019Figure 6—figure supplement 3.Experimental and predicted rotational correlation times (τC).For the isolated proteins the τC values were obtained by the HYDRONMR program. For the apo-complexes the τC values were calculated from the sum of the τC values of the two isolated proteins.DOI:
http://dx.doi.org/10.7554/eLife.16673.01910.7554/eLife.16673.020Figure 6—figure supplement 4.Comparison of the apo-AtGrxS14-AtBolA2 complex with apo-GLRX5/BOLA3.Significant chemical shift changes (red and green for A. thaliana and human proteins, respectively) are mapped on the structures of (A) apo-AtGrxS14 and apo-GLRX5 upon the interaction with AtBolA2 and human BOLA3, respectively. Vice versa, part (B) shows the significant chemical shift changes of AtBolA2 and human BOLA3 upon interaction with AtGrxS14 and GLRX5, respectively. Conserved residues and the GLRX5-bound GSH are highlighted as sticks.DOI:
http://dx.doi.org/10.7554/eLife.16673.020Recently, a heterodimeric complex formed between A. thaliana (At) apo-GrxS14 and AtBolA2 (showing best similarity to BOLA3 and possessing a conserved Cys in addition to the invariant C-terminal His) was characterized by NMR (Roret et al., 2014). Surprisingly, the human and plant complexes share only some regions of chemical shift changes, and exhibit substantial differences (Figure 6—figure supplement 4). For instance, the invariant His66 region of AtBolA2 does not appear to be an interaction interface, at variance with what we observed for both human BOLA complexes (Figure 6A,B). In addition, the GrxS14-interacting regions include the two C-terminal α4 and α5 helices, whereas helix α5 of GLRX5 of both human complexes did not show chemical shifts, and a different segment of helix α4 was involved. Moreover, helix α2 of GLRX5 showed signs of interaction only in the human BOLA-GLRX5 complexes. We conclude that there is a high degree of organismic variability in the precise interaction mode between Grx and Bol proteins.We next investigated the interaction of holo-GLRX5 with the BOLA proteins by chemically reconstituting a [2Fe-2S] cluster on equimolar GLRX5 and BOLAs. NMR 1H-15N HSQC spectra were acquired for the co-reconstituted complexes of 15N labeled BOLA3 or BOLA1 with GLRX5. The 1H-15N HSQC spectra of these co-reconstituted complexes were similar to spectra obtained by mixing BOLA1 or BOLA3 with [2Fe-2S]-containing holo-GLRX5 (not shown). Upon overlaying the 1H-15N HSQC spectra of the holo-complexes with those acquired for the corresponding apo-complexes, meaningful spectral variations (either as chemical shift changes and/or line-broadenings beyond signal detection) were detected on both proteins (Figure 6—figure supplements 1B and 2B). The 1H-15N HSQC maps of the co-reconstituted complexes having 15N labeled GLRX5 with unlabeled BOLA1 or BOLA3 were not superimposable to the 1H-15N HSQC map of the holo-GLRX5 homodimer indicating formation of [2Fe-2S] hetero-complexes 15N R1 and R2 NMR relaxation data of BOLA1 or BOLA3, in the [2Fe-2S] chemically reconstituted complexes with GLRX5, indicated no significant changes in protein size with respect to the apo-complexes. Overall, the NMR data showed the presence of 1:1 heterodimeric holo-complexes between GLRX5 and BOLA1 or BOLA3.Chemical shift perturbation and line broadening analyses were performed by comparing the 1H-15N HSQC spectrum of chemically reconstituted 15N-labeled BOLA3-GLRX5 with that of BOLA3. The residues of the loop containing the conserved Cys59 of BOLA3 showed strong chemical shift changes only in the holo hetero-complex relative to the apoform (Figure 6B,D and Figure 6—figure supplement 1, right parts). A similar comparison for the 15N-labeled BOLA1-GLRX5 complex showed that the region around His67 of BOLA1 (structurally close to Cys59 of BOLA3) exhibited similar chemical shift changes, even though fewer residues were affected in BOLA1 (Figure 6A,C, and Figure 6—figure supplement 1, left parts). The chemical shift perturbation analysis for 15N-labeled GLRX5 revealed that the residues of GLRX5 involved in the apo interaction are also affected in the holo-complex and that the conserved Cys67 of GLRX5 was additionally altered in the holo complexes only (Figure 6—figure supplement 2). In addition, two other adjacent GLRX5 residues (Lys59 for BOLA3 and Gly60 for BOLA1 interaction) were altered that are part of the GSH binding site. The regions of GLRX5 affected by the interaction with BOLA1 and BOLA3 are essentially the same as those in the [2Fe-2S]2+ GLRX5 homodimer (Banci et al., 2014). This indicates that, for GLRX5, the same interaction regions are involved in both homo- and hetero-dimer formation, and the same groups (the conserved Cys67 and GSH) can act as iron ligands of the [2Fe-2S]2+ cluster.

Structural insights into the human mitochondrial BOLA proteins and the interaction with GLRX5

To extend our cell biological insights into Bol protein function, we employed NMR structural studies to examine their physical interaction with Grx5. We used the human proteins for these in vitro studies because they were more stable than the yeast counterparts. First, the solution structures of human BOLA1 and BOLA3 were determined (Figure 5). A detailed description of the two structures and of their backbone dynamic properties is provided in Figure 5—figure supplement 1. According to the DALI server (Holm and Sander, 1993), both BOLA1 and BOLA3 show the highest structural similarity to BolA-like proteins from Arabidopsis thaliana, Babesia bovis, and Mus musculus (Abendroth et al., 2011; Kasai et al., 2004; Roret et al., 2014). In particular, the BOLA1 structure matches better with those homologues that have a longer loop between β1 and β2, while the BOLA3 structure is more similar to homologues having a shorter β1-β2 loop (Figure 5C and Figure 5—figure supplement 2). Apparently, the structural features of the loop between β1 and β2 are key for differentiating BOLA1 from BOLA3. Remarkably, this loop is closer to the ‘invariant’ His residue (His102 in BOLA1 and His96 in BOLA3; Figure 5), which has been shown to coordinate the [2Fe-2S] cluster in the yeast Bol2-Grx3 complex (Li et al., 2011). This region contains further possible Fe/S cluster ligands, namely His58, His67 in BOLA1 and Cys59 in BOLA3. Another conserved His residue (His86 in BOLA1 and His81 in BOLA3) is distant from this region and thus excluded as a cluster ligand (Figure 5). Together, structural data indicate that BOLA1 and BOLA3 have a similar fold, but show local structural differences in the regions containing the potential Fe/S cluster ligands.10.7554/eLife.16673.013Figure 5.NMR solution structures of human BOLA1 and BOLA3.The structures of (A) BOLA1 and (B) BOLA3 were solved by solution NMR. Residues His58, His67 and His102 are conserved within the eukaryotic BOLA1 proteins, and residues Cys59 and His96 are conserved within the BOLA3 proteins. (C) Backbone superimposition of BOLA1 (red) and BOLA3 (blue) structures depicting conserved His and Cys residues in both proteins. Yellow sticks represent potential Fe/S cluster ligands, and green sticks indicate other conserved His residues.DOI:
http://dx.doi.org/10.7554/eLife.16673.01310.7554/eLife.16673.014Figure 5—figure supplement 1.Structural and dynamic properties of BOLA1 and BOLA3 by solution NMR.(A) 2D 1H-15N HSQC spectra of 15N-labeled BOLA1 and BOLA3. (B) 15N R1, R2 relaxation rates and 15N(1H) NOE values per residue of BOLA1 and BOLA3 obtained at 500 MHz and 298 K. The proteins were in buffer N with 10% (v/v) D2O). The 1H-15N HSQC spectra of the apo forms of BOLA3 and BOLA1 showed well-dispersed resonances indicative of essentially folded proteins. The overall topology of the solution structure of human BOLA3 is α1β1β2η1α2η2β3 (h: 310-helix), in which β1 and β2 are antiparallel, and β3 is parallel to β2 (Figure 5). All α-helices are on the same side of the β-sheet and the 310-helix located between α2 and β3 contains the His96 residue that is invariant in all prokaryotic and eukaryotic BolA homologues (Li and Outten, 2012). A cysteine residue (Cys59), typically conserved in Bol3 homologues, is located close to the invariant His96 residue in an extended loop between β1 and β2. The two residues are not in the proper orientation to coordinate a [2Fe-2S] cluster (Sγ of Cys59 is at an average distance of ~8 Å from Nε2 of His96). The overall topology of the solution structure of BOLA1 is similar to that of BOLA3, with the exception of the presence, in BOLA1, of short helices in the loop regions and at the C-terminus (α1η1β1β2η2α2α3β3α4 topology) (Figure 5). In addition, at both termini BOLA1 has ca. 15 residues that are unstructured and whose backbone NH resonances are clustered in the central region of the 1H-15N HSQC spectrum (part B). The three His residues typically conserved in Bol1 homologues are located in two spatially close regions (Figure 5). Although they are close to each other, it is not possible to predict which ones may be involved in [2Fe-2S] cluster coordination. Information on backbone motions and on the protein oligomerization state of BOLA1 and BOLA3 were obtained through 15N R1, R2, heteronuclear 15N(1H)-NOEs NMR experiments. The 15N backbone relaxation properties showed that the N and C termini of BOLA1 are highly flexible, as about 15 to 20 residues on each terminus have negative 15N NOE values, while the folded domain is essentially rigid. BOLA3 is a rigid molecule, with the exception of the first detected residue at the N terminus, the last two residues at the C terminus and residues around invariant His96. All of them experience fast backbone motions in the ns-ps time scale, as indicated by their low or negative 15N NOE values.DOI:
http://dx.doi.org/10.7554/eLife.16673.01410.7554/eLife.16673.015Figure 5—figure supplement 2.Multi-sequence alignment of the mitochondrial Bol proteins.Multalin (Corpet, 1988) was used to generate a multi-sequence alignment of Bol1- and Bol3-like proteins from fungi and various higher eukaryotes. The conserved His and Cys residues are indicated. The numbers refer to the residues of the human mBols. Two His (58 and 67) in BOLA1 are candidates for the structural counterpart of Cys59 of BOLA3.DOI:
http://dx.doi.org/10.7554/eLife.16673.015

NMR solution structures of human BOLA1 and BOLA3.

The structures of (A) BOLA1 and (B) BOLA3 were solved by solution NMR. Residues His58, His67 and His102 are conserved within the eukaryotic BOLA1 proteins, and residues Cys59 and His96 are conserved within the BOLA3 proteins. (C) Backbone superimposition of BOLA1 (red) and BOLA3 (blue) structures depicting conserved His and Cys residues in both proteins. Yellow sticks represent potential Fe/S cluster ligands, and green sticks indicate other conserved His residues.

DOI:
http://dx.doi.org/10.7554/eLife.16673.013

10.7554/eLife.16673.014Figure 5—figure supplement 1.Structural and dynamic properties of BOLA1 and BOLA3 by solution NMR.(A) 2D 1H-15N HSQC spectra of 15N-labeled BOLA1 and BOLA3. (B) 15N R1, R2 relaxation rates and 15N(1H) NOE values per residue of BOLA1 and BOLA3 obtained at 500 MHz and 298 K. The proteins were in buffer N with 10% (v/v) D2O). The 1H-15N HSQC spectra of the apo forms of BOLA3 and BOLA1 showed well-dispersed resonances indicative of essentially folded proteins. The overall topology of the solution structure of human BOLA3 is α1β1β2η1α2η2β3 (h: 310-helix), in which β1 and β2 are antiparallel, and β3 is parallel to β2 (Figure 5). All α-helices are on the same side of the β-sheet and the 310-helix located between α2 and β3 contains the His96 residue that is invariant in all prokaryotic and eukaryotic BolA homologues (Li and Outten, 2012). A cysteine residue (Cys59), typically conserved in Bol3 homologues, is located close to the invariant His96 residue in an extended loop between β1 and β2. The two residues are not in the proper orientation to coordinate a [2Fe-2S] cluster (Sγ of Cys59 is at an average distance of ~8 Å from Nε2 of His96). The overall topology of the solution structure of BOLA1 is similar to that of BOLA3, with the exception of the presence, in BOLA1, of short helices in the loop regions and at the C-terminus (α1η1β1β2η2α2α3β3α4 topology) (Figure 5). In addition, at both termini BOLA1 has ca. 15 residues that are unstructured and whose backbone NH resonances are clustered in the central region of the 1H-15N HSQC spectrum (part B). The three His residues typically conserved in Bol1 homologues are located in two spatially close regions (Figure 5). Although they are close to each other, it is not possible to predict which ones may be involved in [2Fe-2S] cluster coordination. Information on backbone motions and on the protein oligomerization state of BOLA1 and BOLA3 were obtained through 15N R1, R2, heteronuclear 15N(1H)-NOEs NMR experiments. The 15N backbone relaxation properties showed that the N and C termini of BOLA1 are highly flexible, as about 15 to 20 residues on each terminus have negative 15N NOE values, while the folded domain is essentially rigid. BOLA3 is a rigid molecule, with the exception of the first detected residue at the N terminus, the last two residues at the C terminus and residues around invariant His96. All of them experience fast backbone motions in the ns-ps time scale, as indicated by their low or negative 15N NOE values.DOI:
http://dx.doi.org/10.7554/eLife.16673.014

Structural and dynamic properties of BOLA1 and BOLA3 by solution NMR.

(A) 2D 1H-15N HSQC spectra of 15N-labeled BOLA1 and BOLA3. (B) 15N R1, R2 relaxation rates and 15N(1H) NOE values per residue of BOLA1 and BOLA3 obtained at 500 MHz and 298 K. The proteins were in buffer N with 10% (v/v) D2O). The 1H-15N HSQC spectra of the apo forms of BOLA3 and BOLA1 showed well-dispersed resonances indicative of essentially folded proteins. The overall topology of the solution structure of human BOLA3 is α1β1β2η1α2η2β3 (h: 310-helix), in which β1 and β2 are antiparallel, and β3 is parallel to β2 (Figure 5). All α-helices are on the same side of the β-sheet and the 310-helix located between α2 and β3 contains the His96 residue that is invariant in all prokaryotic and eukaryotic BolA homologues (Li and Outten, 2012). A cysteine residue (Cys59), typically conserved in Bol3 homologues, is located close to the invariant His96 residue in an extended loop between β1 and β2. The two residues are not in the proper orientation to coordinate a [2Fe-2S] cluster (Sγ of Cys59 is at an average distance of ~8 Å from Nε2 of His96). The overall topology of the solution structure of BOLA1 is similar to that of BOLA3, with the exception of the presence, in BOLA1, of short helices in the loop regions and at the C-terminus (α1η1β1β2η2α2α3β3α4 topology) (Figure 5). In addition, at both termini BOLA1 has ca. 15 residues that are unstructured and whose backbone NH resonances are clustered in the central region of the 1H-15N HSQC spectrum (part B). The three His residues typically conserved in Bol1 homologues are located in two spatially close regions (Figure 5). Although they are close to each other, it is not possible to predict which ones may be involved in [2Fe-2S] cluster coordination. Information on backbone motions and on the protein oligomerization state of BOLA1 and BOLA3 were obtained through 15N R1, R2, heteronuclear 15N(1H)-NOEs NMR experiments. The 15N backbone relaxation properties showed that the N and C termini of BOLA1 are highly flexible, as about 15 to 20 residues on each terminus have negative 15N NOE values, while the folded domain is essentially rigid. BOLA3 is a rigid molecule, with the exception of the first detected residue at the N terminus, the last two residues at the C terminus and residues around invariant His96. All of them experience fast backbone motions in the ns-ps time scale, as indicated by their low or negative 15N NOE values.

DOI:
http://dx.doi.org/10.7554/eLife.16673.014

10.7554/eLife.16673.015Figure 5—figure supplement 2.Multi-sequence alignment of the mitochondrial Bol proteins.Multalin (Corpet, 1988) was used to generate a multi-sequence alignment of Bol1- and Bol3-like proteins from fungi and various higher eukaryotes. The conserved His and Cys residues are indicated. The numbers refer to the residues of the human mBols. Two His (58 and 67) in BOLA1 are candidates for the structural counterpart of Cys59 of BOLA3.DOI:
http://dx.doi.org/10.7554/eLife.16673.015

Multi-sequence alignment of the mitochondrial Bol proteins.

Multalin (Corpet, 1988) was used to generate a multi-sequence alignment of Bol1- and Bol3-like proteins from fungi and various higher eukaryotes. The conserved His and Cys residues are indicated. The numbers refer to the residues of the human mBols. Two His (58 and 67) in BOLA1 are candidates for the structural counterpart of Cys59 of BOLA3.

DOI:
http://dx.doi.org/10.7554/eLife.16673.015

In order to characterize the interaction between BOLA1 or BOLA3 and GLRX5 at the atomic level, we first studied the apo situation. 15N labeled BOLA proteins were titrated with unlabeled apo-GLRX5, and, vice versa,
15N labeled apo-GLRX5 with unlabeled BOLA proteins in the presence of 5 mM GSH. Meaningful chemical shift changes were observed for both titrations in 2D 1H-15N HSQC NMR spectra acquired upon addition of the unlabeled partner (Figure 6—figure supplements 1A and 2A). The free and bound proteins are in fast and intermediate exchange regimes relative to NMR time scale; the observed chemical shift changes of the protein amide resonances saturated at a 1:1 protein ratio. 15N R1 and R2 NMR relaxation data performed on the final 15N labeled BOLA1-unlabelled GLRX5 or 15N labeled BOLA3-unlabelled GLRX5 mixtures showed rotational correlation times consistent with the formation of a 1:1 heterodimeric complex (Figure 6—figure supplement 3). NMR titrations performed by adding unlabeled apo-GLRX5 to a 15N-labeled BOLA1/15N-labeled BOLA3 1:1 mixture showed the formation of both hetero-complexes at comparable levels, indicating similar affinities. This data fits nicely to thermophoresis results showing that both human BOLA proteins interacted with apo-GLRX5 with Kd values of 3 µM (Melber et al., 2016; accompanying manuscript).

The chemical shift changes observed upon formation of apo-GLRX5-BOLA1 and apo-GLRX5-BOLA3 complexes were mapped on the solution structures of BOLA1, BOLA3 and apo-GLRX5. The interaction surface on GLRX5 in the two complexes comprises the GSH binding site and its surroundings (Figure 6A,B and Figure 6—figure supplement 2A). On the BOLAs, the interactions involve helix α2, part of the β-sheet and the invariant His residue (see above). The other possible Fe/S cluster ligands (His58, His67 in BOLA1 and Cys59 in BOLA3) showed no significant chemical shifts. In conclusion, NMR data indicate that apo-GLRX5-BOLA1 and apo-GLRX5-BOLA3 complexes specifically interact involving the region surrounding the invariant His in BOLAs and the GSH binding site in GLRX5.10.7554/eLife.16673.016Figure 6.Structural basis of the interaction between human apo- and holo-GLRX5 with the BOLA proteins.(A and B) Backbone chemical shift differences, obtained from a comparison of the 1H-15N HSQC spectra of apo-GLRX5 or BOLA proteins with that of (A) BOLA1-apo-GLRX5 or (B) BOLA3-apo-GLRX5 (1:1 mixture in buffer N), were mapped on the solution structures of the proteins. (C and D) Backbone chemical shift differences, obtained from a comparison of the 1H-15N HSQC spectrum of apo-GLXR5 or BOLA proteins with that of (C) BOLA1-apo-GLRX5 or (D) BOLA3-apo-GLRX5 (1:1 mixture in buffer N) chemically reconstituted with a [2Fe-2S] cluster, were mapped on the solution structures of the proteins. Green regions show residues with significant chemical shift changes (that is both in terms of chemical shift and broadening beyond detection effects) observed upon formation of the apo- or holo-complexes. Red areas depict those residues additionally affected upon holo-complex formation. Critical residues and GLRX5-bound GSH are depicted as sticks.DOI:
http://dx.doi.org/10.7554/eLife.16673.01610.7554/eLife.16673.017Figure 6—figure supplement 1.Chemical shift changes upon complex formation between GLRX5 and BOLA proteins monitoring backbone chemical shift changes of BOLA1 and BOLA3.Backbone weighted average chemical shift differences Δavg(HN) obtained by comparing 1H-15N HSQC map of (A) 15N-labeled BOLA1 or BOLA3 with an equimolar mixture of 15N-labeled BOLA1 or BOLA3 with unlabeled apo-GLRX5, and of (B) 15N-labeled BOLA1 or BOLA3 with an equimolar mixture of 15N-labeled BOLA1 or BOLA3 chemically reconstituted with unlabeled GLRX5 to generate a [2Fe-2S] cluster (buffer N). Bars with Δavg(HN) = 1 indicate residues whose backbone NH signals broaden beyond detection. The indicated thresholds (obtained by averaging Δavg(HN) values plus 1σ) were used to define significant chemical shift differences. Green bars show significant chemical shift changes (that is both chemical shift differences and broadening beyond detection effects, respectively) in the apo interactions, and red bars those additionally occurring upon holo complex formation. The position of conserved residues is highlighted.DOI:
http://dx.doi.org/10.7554/eLife.16673.01710.7554/eLife.16673.018Figure 6—figure supplement 2.Chemical shift changes upon complex formation between GLRX5 and BOLA proteins, monitoring backbone chemical shift changes of GLRX5.Backbone weighted average chemical shift differences Δavg(HN) obtained by comparing 1H-15N HSQC map of (A) 15N-labeled apo-GLRX5 with that of a 1:1 mixture of 15N-labeled apo-GLRX5 and unlabeled BOLA1 or BOLA3, and of (B) 15N-labeled apo-GLRX5 with that of a 1:1 mixture of 15N-labeled apo-GLRX5 with unlabeled BOLA1 or BOLA3 after chemical reconstitution of a [2Fe-2S] cluster in buffer N. The data are presented as in Figure 6—figure supplement 1. The position of the conserved Cys67 and of the residues of the GSH binding site are highlighted.DOI:
http://dx.doi.org/10.7554/eLife.16673.01810.7554/eLife.16673.019Figure 6—figure supplement 3.Experimental and predicted rotational correlation times (τC).For the isolated proteins the τC values were obtained by the HYDRONMR program. For the apo-complexes the τC values were calculated from the sum of the τC values of the two isolated proteins.DOI:
http://dx.doi.org/10.7554/eLife.16673.01910.7554/eLife.16673.020Figure 6—figure supplement 4.Comparison of the apo-AtGrxS14-AtBolA2 complex with apo-GLRX5/BOLA3.Significant chemical shift changes (red and green for A. thaliana and human proteins, respectively) are mapped on the structures of (A) apo-AtGrxS14 and apo-GLRX5 upon the interaction with AtBolA2 and human BOLA3, respectively. Vice versa, part (B) shows the significant chemical shift changes of AtBolA2 and human BOLA3 upon interaction with AtGrxS14 and GLRX5, respectively. Conserved residues and the GLRX5-bound GSH are highlighted as sticks.DOI:
http://dx.doi.org/10.7554/eLife.16673.020

Structural basis of the interaction between human apo- and holo-GLRX5 with the BOLA proteins.

(A and B) Backbone chemical shift differences, obtained from a comparison of the 1H-15N HSQC spectra of apo-GLRX5 or BOLA proteins with that of (A) BOLA1-apo-GLRX5 or (B) BOLA3-apo-GLRX5 (1:1 mixture in buffer N), were mapped on the solution structures of the proteins. (C and D) Backbone chemical shift differences, obtained from a comparison of the 1H-15N HSQC spectrum of apo-GLXR5 or BOLA proteins with that of (C) BOLA1-apo-GLRX5 or (D) BOLA3-apo-GLRX5 (1:1 mixture in buffer N) chemically reconstituted with a [2Fe-2S] cluster, were mapped on the solution structures of the proteins. Green regions show residues with significant chemical shift changes (that is both in terms of chemical shift and broadening beyond detection effects) observed upon formation of the apo- or holo-complexes. Red areas depict those residues additionally affected upon holo-complex formation. Critical residues and GLRX5-bound GSH are depicted as sticks.

DOI:
http://dx.doi.org/10.7554/eLife.16673.016

10.7554/eLife.16673.017Figure 6—figure supplement 1.Chemical shift changes upon complex formation between GLRX5 and BOLA proteins monitoring backbone chemical shift changes of BOLA1 and BOLA3.Backbone weighted average chemical shift differences Δavg(HN) obtained by comparing 1H-15N HSQC map of (A) 15N-labeled BOLA1 or BOLA3 with an equimolar mixture of 15N-labeled BOLA1 or BOLA3 with unlabeled apo-GLRX5, and of (B) 15N-labeled BOLA1 or BOLA3 with an equimolar mixture of 15N-labeled BOLA1 or BOLA3 chemically reconstituted with unlabeled GLRX5 to generate a [2Fe-2S] cluster (buffer N). Bars with Δavg(HN) = 1 indicate residues whose backbone NH signals broaden beyond detection. The indicated thresholds (obtained by averaging Δavg(HN) values plus 1σ) were used to define significant chemical shift differences. Green bars show significant chemical shift changes (that is both chemical shift differences and broadening beyond detection effects, respectively) in the apo interactions, and red bars those additionally occurring upon holo complex formation. The position of conserved residues is highlighted.DOI:
http://dx.doi.org/10.7554/eLife.16673.017

Chemical shift changes upon complex formation between GLRX5 and BOLA proteins monitoring backbone chemical shift changes of BOLA1 and BOLA3.

Backbone weighted average chemical shift differences Δavg(HN) obtained by comparing 1H-15N HSQC map of (A) 15N-labeled BOLA1 or BOLA3 with an equimolar mixture of 15N-labeled BOLA1 or BOLA3 with unlabeled apo-GLRX5, and of (B) 15N-labeled BOLA1 or BOLA3 with an equimolar mixture of 15N-labeled BOLA1 or BOLA3 chemically reconstituted with unlabeled GLRX5 to generate a [2Fe-2S] cluster (buffer N). Bars with Δavg(HN) = 1 indicate residues whose backbone NH signals broaden beyond detection. The indicated thresholds (obtained by averaging Δavg(HN) values plus 1σ) were used to define significant chemical shift differences. Green bars show significant chemical shift changes (that is both chemical shift differences and broadening beyond detection effects, respectively) in the apo interactions, and red bars those additionally occurring upon holo complex formation. The position of conserved residues is highlighted.

DOI:
http://dx.doi.org/10.7554/eLife.16673.017

10.7554/eLife.16673.018Figure 6—figure supplement 2.Chemical shift changes upon complex formation between GLRX5 and BOLA proteins, monitoring backbone chemical shift changes of GLRX5.Backbone weighted average chemical shift differences Δavg(HN) obtained by comparing 1H-15N HSQC map of (A) 15N-labeled apo-GLRX5 with that of a 1:1 mixture of 15N-labeled apo-GLRX5 and unlabeled BOLA1 or BOLA3, and of (B) 15N-labeled apo-GLRX5 with that of a 1:1 mixture of 15N-labeled apo-GLRX5 with unlabeled BOLA1 or BOLA3 after chemical reconstitution of a [2Fe-2S] cluster in buffer N. The data are presented as in Figure 6—figure supplement 1. The position of the conserved Cys67 and of the residues of the GSH binding site are highlighted.DOI:
http://dx.doi.org/10.7554/eLife.16673.018

Chemical shift changes upon complex formation between GLRX5 and BOLA proteins, monitoring backbone chemical shift changes of GLRX5.

Backbone weighted average chemical shift differences Δavg(HN) obtained by comparing 1H-15N HSQC map of (A) 15N-labeled apo-GLRX5 with that of a 1:1 mixture of 15N-labeled apo-GLRX5 and unlabeled BOLA1 or BOLA3, and of (B) 15N-labeled apo-GLRX5 with that of a 1:1 mixture of 15N-labeled apo-GLRX5 with unlabeled BOLA1 or BOLA3 after chemical reconstitution of a [2Fe-2S] cluster in buffer N. The data are presented as in Figure 6—figure supplement 1. The position of the conserved Cys67 and of the residues of the GSH binding site are highlighted.

DOI:
http://dx.doi.org/10.7554/eLife.16673.018

10.7554/eLife.16673.019Figure 6—figure supplement 3.Experimental and predicted rotational correlation times (τC).For the isolated proteins the τC values were obtained by the HYDRONMR program. For the apo-complexes the τC values were calculated from the sum of the τC values of the two isolated proteins.DOI:
http://dx.doi.org/10.7554/eLife.16673.019

Experimental and predicted rotational correlation times (τC).

For the isolated proteins the τC values were obtained by the HYDRONMR program. For the apo-complexes the τC values were calculated from the sum of the τC values of the two isolated proteins.

DOI:
http://dx.doi.org/10.7554/eLife.16673.019

10.7554/eLife.16673.020Figure 6—figure supplement 4.Comparison of the apo-AtGrxS14-AtBolA2 complex with apo-GLRX5/BOLA3.Significant chemical shift changes (red and green for A. thaliana and human proteins, respectively) are mapped on the structures of (A) apo-AtGrxS14 and apo-GLRX5 upon the interaction with AtBolA2 and human BOLA3, respectively. Vice versa, part (B) shows the significant chemical shift changes of AtBolA2 and human BOLA3 upon interaction with AtGrxS14 and GLRX5, respectively. Conserved residues and the GLRX5-bound GSH are highlighted as sticks.DOI:
http://dx.doi.org/10.7554/eLife.16673.020

Comparison of the apo-AtGrxS14-AtBolA2 complex with apo-GLRX5/BOLA3.

Significant chemical shift changes (red and green for A. thaliana and human proteins, respectively) are mapped on the structures of (A) apo-AtGrxS14 and apo-GLRX5 upon the interaction with AtBolA2 and human BOLA3, respectively. Vice versa, part (B) shows the significant chemical shift changes of AtBolA2 and human BOLA3 upon interaction with AtGrxS14 and GLRX5, respectively. Conserved residues and the GLRX5-bound GSH are highlighted as sticks.

DOI:
http://dx.doi.org/10.7554/eLife.16673.020

Recently, a heterodimeric complex formed between A. thaliana (At) apo-GrxS14 and AtBolA2 (showing best similarity to BOLA3 and possessing a conserved Cys in addition to the invariant C-terminal His) was characterized by NMR (Roret et al., 2014). Surprisingly, the human and plant complexes share only some regions of chemical shift changes, and exhibit substantial differences (Figure 6—figure supplement 4). For instance, the invariant His66 region of AtBolA2 does not appear to be an interaction interface, at variance with what we observed for both human BOLA complexes (Figure 6A,B). In addition, the GrxS14-interacting regions include the two C-terminal α4 and α5 helices, whereas helix α5 of GLRX5 of both human complexes did not show chemical shifts, and a different segment of helix α4 was involved. Moreover, helix α2 of GLRX5 showed signs of interaction only in the human BOLA-GLRX5 complexes. We conclude that there is a high degree of organismic variability in the precise interaction mode between Grx and Bol proteins.

We next investigated the interaction of holo-GLRX5 with the BOLA proteins by chemically reconstituting a [2Fe-2S] cluster on equimolar GLRX5 and BOLAs. NMR 1H-15N HSQC spectra were acquired for the co-reconstituted complexes of 15N labeled BOLA3 or BOLA1 with GLRX5. The 1H-15N HSQC spectra of these co-reconstituted complexes were similar to spectra obtained by mixing BOLA1 or BOLA3 with [2Fe-2S]-containing holo-GLRX5 (not shown). Upon overlaying the 1H-15N HSQC spectra of the holo-complexes with those acquired for the corresponding apo-complexes, meaningful spectral variations (either as chemical shift changes and/or line-broadenings beyond signal detection) were detected on both proteins (Figure 6—figure supplements 1B and 2B). The 1H-15N HSQC maps of the co-reconstituted complexes having 15N labeled GLRX5 with unlabeled BOLA1 or BOLA3 were not superimposable to the 1H-15N HSQC map of the holo-GLRX5 homodimer indicating formation of [2Fe-2S] hetero-complexes 15N R1 and R2 NMR relaxation data of BOLA1 or BOLA3, in the [2Fe-2S] chemically reconstituted complexes with GLRX5, indicated no significant changes in protein size with respect to the apo-complexes. Overall, the NMR data showed the presence of 1:1 heterodimeric holo-complexes between GLRX5 and BOLA1 or BOLA3.

Chemical shift perturbation and line broadening analyses were performed by comparing the 1H-15N HSQC spectrum of chemically reconstituted 15N-labeled BOLA3-GLRX5 with that of BOLA3. The residues of the loop containing the conserved Cys59 of BOLA3 showed strong chemical shift changes only in the holo hetero-complex relative to the apoform (Figure 6B,D and Figure 6—figure supplement 1, right parts). A similar comparison for the 15N-labeled BOLA1-GLRX5 complex showed that the region around His67 of BOLA1 (structurally close to Cys59 of BOLA3) exhibited similar chemical shift changes, even though fewer residues were affected in BOLA1 (Figure 6A,C, and Figure 6—figure supplement 1, left parts). The chemical shift perturbation analysis for 15N-labeled GLRX5 revealed that the residues of GLRX5 involved in the apo interaction are also affected in the holo-complex and that the conserved Cys67 of GLRX5 was additionally altered in the holo complexes only (Figure 6—figure supplement 2). In addition, two other adjacent GLRX5 residues (Lys59 for BOLA3 and Gly60 for BOLA1 interaction) were altered that are part of the GSH binding site. The regions of GLRX5 affected by the interaction with BOLA1 and BOLA3 are essentially the same as those in the [2Fe-2S]2+ GLRX5 homodimer (Banci et al., 2014). This indicates that, for GLRX5, the same interaction regions are involved in both homo- and hetero-dimer formation, and the same groups (the conserved Cys67 and GSH) can act as iron ligands of the [2Fe-2S]2+ cluster.

Preferential interaction of BOLA1 with holo-GLRX5 and of BOLA3 with holo-NFU1We used circular dichroism (CD) spectroscopy to follow the characteristics of Fe/S cluster coordination upon ligand exchange from the holo-Grx5 homo-dimer to the Grx5-Bol hetero-dimer (Banci et al., 2015; Li et al., 2009; Li et al., 2012). GLRX5 and stoichiometric amounts of BOLA1 or BOLA3 were chemically co-reconstituted with a [2Fe-2S] cluster and analyzed by CD (Figure 7). Similar results were obtained when GLRX5 was reconstituted first and then the BOLA proteins were added (Figure 7—figure supplement 1). The CD spectrum of GLRX5-BOLA1 characteristically differed from that of holo-GLRX5 in a positive versus negative ellipticity around 400 nm with a 6 nm blue shift and 40% decrease of the ~460 nm peak with a 6 nm red shift (Figure 7A). These spectral changes saturated at an equimolar ratio of holo-GLRX5 and BOLA1 suggesting a high affinity equimolar hetero-complex formation with a shared [2Fe-2S] cluster (Figure 7—figure supplement 1B). The cluster coordinated by the GLRX5-BOLA1 hetero-dimer was stable against reduction by dithionite creating a CD spectrum in which the peak at 460 nm was maintained but that at 400 nm was lost. In contrast, the cluster of holo-GLRX5 was destroyed upon dithionite treatment with no residual CD peaks (Figure 7A and Figure 7—figure supplement 1A). These results demonstrate that BOLA1 stabilized the shared [2Fe-2S] cluster in the hetero-dimer with GLRX5.10.7554/eLife.16673.021Figure 7.Human BOLA1 but not BOLA3 stabilizes the [2Fe-2S] cluster of holo-GLRX5 upon heterodimer formation.(A–B) Chemical reconstitution of Fe/S clusters in buffer R was performed with GLRX5 in the absence or presence of stoichiometric amounts of (A) BOLA1 or (B) BOLA3, and CD spectra were monitored under anaerobic conditions. Additional spectra were recorded after addition of 2 mM dithionite (DT). (C–D) Chemically reconstituted GLRX5 was titrated with the indicated concentrations of BOLA3 in (C) buffer N or (D) buffer P, and CD spectra (corrected for dilution) were recorded under anaerobic conditions. Abbreviations for varied buffer conditions: 7, 8: pH; N, 150 mM NaCl; G, 5 mM GSH.DOI:
http://dx.doi.org/10.7554/eLife.16673.02110.7554/eLife.16673.022Figure 7—figure supplement 1.Stoichiometric hetero-complex formation of human holo-GLRX5 with BOLA1 results in characteristic CD spectral changes indicating shared binding of the [2Fe-2S] cluster.(A,C) Chemically reconstituted GLRX5 was mixed in buffer R (lacking additional GSH) with a stoichiometric amount of (A) BOLA1 or (C) BOLA3, and CD spectra were recorded under anaerobic conditions. Additional spectra were recorded after addition of 2 mM dithionite (DT). (B, D) Chemically reconstituted GLRX5 was titrated with the indicated concentrations of (B) BOLA1 or (D) BOLA3, and CD spectra (corrected for dilution) were recorded under anaerobic conditions.DOI:
http://dx.doi.org/10.7554/eLife.16673.02210.7554/eLife.16673.023Figure 7—figure supplement 2.Gel filtration verifies the complex formation between human holo-GLRX5 and BOLA1.(A) The gel filtration elution profile of co-reconstituted equimolar holo-GLRX5-BOLA3, holo-GLRX5 (offset by OD 0.015), or co-reconstituted equimolar GLRX5-BOLA1 (offset by OD 0.045) was recorded at 420 nm in buffer R. Elution positions of molecular marker proteins are indicated in kDa. (B) As a control, the elution behavior of the holo-GLRX5+BOLA3 mixture or BOLA3 alone were recorded at 280 nm in parallel to part A. Fractions of the holo-GLRX5+BOLA3 mixture were analyzed for GLRX5 and BOLA3 by Coomassie and immunostaining (bottom part).DOI:
http://dx.doi.org/10.7554/eLife.16673.02310.7554/eLife.16673.024Figure 7—figure supplement 3.Preferential binding of human BOLA3 to the holo-form of NFU1.(A) NFU1 was used in either apo- or holo-form and mixed at increasing concentrations with 200 nM fluorescently labeled BOLA1 or BOLA3. Microscale thermophoresis (MST) analyses were performed under anaerobic conditions, and dissociation constants (Kd) were determined. Control experiments were performed with human [2Fe-2S] ferredoxin FDX2, cytochrome c, and RNase A. Error bars indicate the SD (n = 3). (B–E) Examples for original MST data.DOI:
http://dx.doi.org/10.7554/eLife.16673.024A strikingly different Fe/S cluster behavior was observed upon BOLA3-GLRX5 complex formation. Hardly any CD spectral changes were observed upon co-reconstituting or mixing GLRX5 and BOLA3 (Figure 7B and Figure 7—figure supplement 1C,D). Further, the Fe/S cluster was not protected against destruction by dithionite treatment indicating rather different coordination situations for the [2Fe-2S] clusters in the two GLRX5-BOLA heterodimers. These results are consistent with the different NMR chemical shift changes in the putative Fe/S cluster binding regions of the two BOLAs in the hetero-complexes (Figure 6C,D). They do not explain, however, why NMR still detected holo-specific changes in the putative cluster-binding region of the BOLA3 hetero-complex (Figure 6D). Since the NMR data were recorded in GSH-containing buffer at lower pH, we repeated the CD analysis under NMR buffer conditions. We found a general decrease of the CD peaks upon GLRX5-BOLA3 hetero-complex formation (Figure 7C). These changes mainly depended on the presence of GSH, and not salt or pH (Figure 7D). In all cases, the CD signal decrease did not reach saturation even after addition of a more than tenfold molar excess of BOLA3 over holo-GLRX5. This result indicated a rather low affinity for this interference of BOLA3 with the [2Fe-2S] cluster. Overall, the CD data indicate a rather weak and GSH-dependent influence of BOLA3 on the GLRX5 [2Fe-2S] cluster which is not stabilized by hetero-complex formation.To independently analyze the GLRX5-BOLA complex formation, we used size exclusion chromatography. Holo-GLRX5 (recorded at 420 nm) eluted at 44 kDa indicative of homodimer with bound Fe/S cluster (Figure 7—figure supplement 2A). A significantly smaller molecular mass was observed for the heterodimeric GLRX5-BOLA1 (32 kDa), while the mixture of holo-GLRX5 and BOLA3 did not alter the elution behavior of GLRX5 at 420 nm. The elution profile at 280 nm showed no significant protein peak around 30 kDa that might represent a GLRX5-BOLA3 heterodimer (Figure 7—figure supplement 2B). Further, BOLA3 from the GLRX5-BOLA3 mixture eluted exclusively at the position of BOLA3 alone. These results support the formation of a stable heterodimer between GLRX5 and BOLA1, yet the interaction of GLRX5 with BOLA3 appears to be kinetically unstable, especially in the absence of GSH.The lack of stable interaction between GLRX5 and BOLA3 raised the question of whether the latter could interact with NFU1. Since Fe/S cluster binding to NFU1 was CD-silent, we performed in vitro binding studies by microscale thermophoresis (MST), an equilibrium method that can be performed under anaerobic conditions (Webert et al., 2014). For these interaction experiments we used both apo-NFU1 and chemically reconstituted holo-NFU1 carrying a [4Fe-4S] cluster (Tong et al., 2003). MST indeed showed a specific interaction between the two BOLA proteins and both apo- and holo-NFU1 (Figure 7—figure supplement 3). The dissociation constants (Kd) of the interactions were in the range of 3 µM, with the notable exception of a fourfold higher affinity detected for BOLA3 and holo-NFU1 (Kd = 0.8 µM). As a control, no significant affinity was observed for human [2Fe-2S] ferredoxin FDX2, cytochrome c or RNase A as control proteins. Collectively, these results data suggest that holo-GLRX5 preferentially cooperates with BOLA1, and holo-NFU1 with BOLA3.

Preferential interaction of BOLA1 with holo-GLRX5 and of BOLA3 with holo-NFU1

We used circular dichroism (CD) spectroscopy to follow the characteristics of Fe/S cluster coordination upon ligand exchange from the holo-Grx5 homo-dimer to the Grx5-Bol hetero-dimer (Banci et al., 2015; Li et al., 2009; Li et al., 2012). GLRX5 and stoichiometric amounts of BOLA1 or BOLA3 were chemically co-reconstituted with a [2Fe-2S] cluster and analyzed by CD (Figure 7). Similar results were obtained when GLRX5 was reconstituted first and then the BOLA proteins were added (Figure 7—figure supplement 1). The CD spectrum of GLRX5-BOLA1 characteristically differed from that of holo-GLRX5 in a positive versus negative ellipticity around 400 nm with a 6 nm blue shift and 40% decrease of the ~460 nm peak with a 6 nm red shift (Figure 7A). These spectral changes saturated at an equimolar ratio of holo-GLRX5 and BOLA1 suggesting a high affinity equimolar hetero-complex formation with a shared [2Fe-2S] cluster (Figure 7—figure supplement 1B). The cluster coordinated by the GLRX5-BOLA1 hetero-dimer was stable against reduction by dithionite creating a CD spectrum in which the peak at 460 nm was maintained but that at 400 nm was lost. In contrast, the cluster of holo-GLRX5 was destroyed upon dithionite treatment with no residual CD peaks (Figure 7A and Figure 7—figure supplement 1A). These results demonstrate that BOLA1 stabilized the shared [2Fe-2S] cluster in the hetero-dimer with GLRX5.10.7554/eLife.16673.021Figure 7.Human BOLA1 but not BOLA3 stabilizes the [2Fe-2S] cluster of holo-GLRX5 upon heterodimer formation.(A–B) Chemical reconstitution of Fe/S clusters in buffer R was performed with GLRX5 in the absence or presence of stoichiometric amounts of (A) BOLA1 or (B) BOLA3, and CD spectra were monitored under anaerobic conditions. Additional spectra were recorded after addition of 2 mM dithionite (DT). (C–D) Chemically reconstituted GLRX5 was titrated with the indicated concentrations of BOLA3 in (C) buffer N or (D) buffer P, and CD spectra (corrected for dilution) were recorded under anaerobic conditions. Abbreviations for varied buffer conditions: 7, 8: pH; N, 150 mM NaCl; G, 5 mM GSH.DOI:
http://dx.doi.org/10.7554/eLife.16673.02110.7554/eLife.16673.022Figure 7—figure supplement 1.Stoichiometric hetero-complex formation of human holo-GLRX5 with BOLA1 results in characteristic CD spectral changes indicating shared binding of the [2Fe-2S] cluster.(A,C) Chemically reconstituted GLRX5 was mixed in buffer R (lacking additional GSH) with a stoichiometric amount of (A) BOLA1 or (C) BOLA3, and CD spectra were recorded under anaerobic conditions. Additional spectra were recorded after addition of 2 mM dithionite (DT). (B, D) Chemically reconstituted GLRX5 was titrated with the indicated concentrations of (B) BOLA1 or (D) BOLA3, and CD spectra (corrected for dilution) were recorded under anaerobic conditions.DOI:
http://dx.doi.org/10.7554/eLife.16673.02210.7554/eLife.16673.023Figure 7—figure supplement 2.Gel filtration verifies the complex formation between human holo-GLRX5 and BOLA1.(A) The gel filtration elution profile of co-reconstituted equimolar holo-GLRX5-BOLA3, holo-GLRX5 (offset by OD 0.015), or co-reconstituted equimolar GLRX5-BOLA1 (offset by OD 0.045) was recorded at 420 nm in buffer R. Elution positions of molecular marker proteins are indicated in kDa. (B) As a control, the elution behavior of the holo-GLRX5+BOLA3 mixture or BOLA3 alone were recorded at 280 nm in parallel to part A. Fractions of the holo-GLRX5+BOLA3 mixture were analyzed for GLRX5 and BOLA3 by Coomassie and immunostaining (bottom part).DOI:
http://dx.doi.org/10.7554/eLife.16673.02310.7554/eLife.16673.024Figure 7—figure supplement 3.Preferential binding of human BOLA3 to the holo-form of NFU1.(A) NFU1 was used in either apo- or holo-form and mixed at increasing concentrations with 200 nM fluorescently labeled BOLA1 or BOLA3. Microscale thermophoresis (MST) analyses were performed under anaerobic conditions, and dissociation constants (Kd) were determined. Control experiments were performed with human [2Fe-2S] ferredoxin FDX2, cytochrome c, and RNase A. Error bars indicate the SD (n = 3). (B–E) Examples for original MST data.DOI:
http://dx.doi.org/10.7554/eLife.16673.024

Human BOLA1 but not BOLA3 stabilizes the [2Fe-2S] cluster of holo-GLRX5 upon heterodimer formation.

(A–B) Chemical reconstitution of Fe/S clusters in buffer R was performed with GLRX5 in the absence or presence of stoichiometric amounts of (A) BOLA1 or (B) BOLA3, and CD spectra were monitored under anaerobic conditions. Additional spectra were recorded after addition of 2 mM dithionite (DT). (C–D) Chemically reconstituted GLRX5 was titrated with the indicated concentrations of BOLA3 in (C) buffer N or (D) buffer P, and CD spectra (corrected for dilution) were recorded under anaerobic conditions. Abbreviations for varied buffer conditions: 7, 8: pH; N, 150 mM NaCl; G, 5 mM GSH.

DOI:
http://dx.doi.org/10.7554/eLife.16673.021

10.7554/eLife.16673.022Figure 7—figure supplement 1.Stoichiometric hetero-complex formation of human holo-GLRX5 with BOLA1 results in characteristic CD spectral changes indicating shared binding of the [2Fe-2S] cluster.(A,C) Chemically reconstituted GLRX5 was mixed in buffer R (lacking additional GSH) with a stoichiometric amount of (A) BOLA1 or (C) BOLA3, and CD spectra were recorded under anaerobic conditions. Additional spectra were recorded after addition of 2 mM dithionite (DT). (B, D) Chemically reconstituted GLRX5 was titrated with the indicated concentrations of (B) BOLA1 or (D) BOLA3, and CD spectra (corrected for dilution) were recorded under anaerobic conditions.DOI:
http://dx.doi.org/10.7554/eLife.16673.022

Stoichiometric hetero-complex formation of human holo-GLRX5 with BOLA1 results in characteristic CD spectral changes indicating shared binding of the [2Fe-2S] cluster.

(A,C) Chemically reconstituted GLRX5 was mixed in buffer R (lacking additional GSH) with a stoichiometric amount of (A) BOLA1 or (C) BOLA3, and CD spectra were recorded under anaerobic conditions. Additional spectra were recorded after addition of 2 mM dithionite (DT). (B, D) Chemically reconstituted GLRX5 was titrated with the indicated concentrations of (B) BOLA1 or (D) BOLA3, and CD spectra (corrected for dilution) were recorded under anaerobic conditions.

DOI:
http://dx.doi.org/10.7554/eLife.16673.022

10.7554/eLife.16673.023Figure 7—figure supplement 2.Gel filtration verifies the complex formation between human holo-GLRX5 and BOLA1.(A) The gel filtration elution profile of co-reconstituted equimolar holo-GLRX5-BOLA3, holo-GLRX5 (offset by OD 0.015), or co-reconstituted equimolar GLRX5-BOLA1 (offset by OD 0.045) was recorded at 420 nm in buffer R. Elution positions of molecular marker proteins are indicated in kDa. (B) As a control, the elution behavior of the holo-GLRX5+BOLA3 mixture or BOLA3 alone were recorded at 280 nm in parallel to part A. Fractions of the holo-GLRX5+BOLA3 mixture were analyzed for GLRX5 and BOLA3 by Coomassie and immunostaining (bottom part).DOI:
http://dx.doi.org/10.7554/eLife.16673.023

Gel filtration verifies the complex formation between human holo-GLRX5 and BOLA1.

(A) The gel filtration elution profile of co-reconstituted equimolar holo-GLRX5-BOLA3, holo-GLRX5 (offset by OD 0.015), or co-reconstituted equimolar GLRX5-BOLA1 (offset by OD 0.045) was recorded at 420 nm in buffer R. Elution positions of molecular marker proteins are indicated in kDa. (B) As a control, the elution behavior of the holo-GLRX5+BOLA3 mixture or BOLA3 alone were recorded at 280 nm in parallel to part A. Fractions of the holo-GLRX5+BOLA3 mixture were analyzed for GLRX5 and BOLA3 by Coomassie and immunostaining (bottom part).

DOI:
http://dx.doi.org/10.7554/eLife.16673.023

10.7554/eLife.16673.024Figure 7—figure supplement 3.Preferential binding of human BOLA3 to the holo-form of NFU1.(A) NFU1 was used in either apo- or holo-form and mixed at increasing concentrations with 200 nM fluorescently labeled BOLA1 or BOLA3. Microscale thermophoresis (MST) analyses were performed under anaerobic conditions, and dissociation constants (Kd) were determined. Control experiments were performed with human [2Fe-2S] ferredoxin FDX2, cytochrome c, and RNase A. Error bars indicate the SD (n = 3). (B–E) Examples for original MST data.DOI:
http://dx.doi.org/10.7554/eLife.16673.024

Preferential binding of human BOLA3 to the holo-form of NFU1.

(A) NFU1 was used in either apo- or holo-form and mixed at increasing concentrations with 200 nM fluorescently labeled BOLA1 or BOLA3. Microscale thermophoresis (MST) analyses were performed under anaerobic conditions, and dissociation constants (Kd) were determined. Control experiments were performed with human [2Fe-2S] ferredoxin FDX2, cytochrome c, and RNase A. Error bars indicate the SD (n = 3). (B–E) Examples for original MST data.

DOI:
http://dx.doi.org/10.7554/eLife.16673.024

A strikingly different Fe/S cluster behavior was observed upon BOLA3-GLRX5 complex formation. Hardly any CD spectral changes were observed upon co-reconstituting or mixing GLRX5 and BOLA3 (Figure 7B and Figure 7—figure supplement 1C,D). Further, the Fe/S cluster was not protected against destruction by dithionite treatment indicating rather different coordination situations for the [2Fe-2S] clusters in the two GLRX5-BOLA heterodimers. These results are consistent with the different NMR chemical shift changes in the putative Fe/S cluster binding regions of the two BOLAs in the hetero-complexes (Figure 6C,D). They do not explain, however, why NMR still detected holo-specific changes in the putative cluster-binding region of the BOLA3 hetero-complex (Figure 6D). Since the NMR data were recorded in GSH-containing buffer at lower pH, we repeated the CD analysis under NMR buffer conditions. We found a general decrease of the CD peaks upon GLRX5-BOLA3 hetero-complex formation (Figure 7C). These changes mainly depended on the presence of GSH, and not salt or pH (Figure 7D). In all cases, the CD signal decrease did not reach saturation even after addition of a more than tenfold molar excess of BOLA3 over holo-GLRX5. This result indicated a rather low affinity for this interference of BOLA3 with the [2Fe-2S] cluster. Overall, the CD data indicate a rather weak and GSH-dependent influence of BOLA3 on the GLRX5 [2Fe-2S] cluster which is not stabilized by hetero-complex formation.

To independently analyze the GLRX5-BOLA complex formation, we used size exclusion chromatography. Holo-GLRX5 (recorded at 420 nm) eluted at 44 kDa indicative of homodimer with bound Fe/S cluster (Figure 7—figure supplement 2A). A significantly smaller molecular mass was observed for the heterodimeric GLRX5-BOLA1 (32 kDa), while the mixture of holo-GLRX5 and BOLA3 did not alter the elution behavior of GLRX5 at 420 nm. The elution profile at 280 nm showed no significant protein peak around 30 kDa that might represent a GLRX5-BOLA3 heterodimer (Figure 7—figure supplement 2B). Further, BOLA3 from the GLRX5-BOLA3 mixture eluted exclusively at the position of BOLA3 alone. These results support the formation of a stable heterodimer between GLRX5 and BOLA1, yet the interaction of GLRX5 with BOLA3 appears to be kinetically unstable, especially in the absence of GSH.

The lack of stable interaction between GLRX5 and BOLA3 raised the question of whether the latter could interact with NFU1. Since Fe/S cluster binding to NFU1 was CD-silent, we performed in vitro binding studies by microscale thermophoresis (MST), an equilibrium method that can be performed under anaerobic conditions (Webert et al., 2014). For these interaction experiments we used both apo-NFU1 and chemically reconstituted holo-NFU1 carrying a [4Fe-4S] cluster (Tong et al., 2003). MST indeed showed a specific interaction between the two BOLA proteins and both apo- and holo-NFU1 (Figure 7—figure supplement 3). The dissociation constants (Kd) of the interactions were in the range of 3 µM, with the notable exception of a fourfold higher affinity detected for BOLA3 and holo-NFU1 (Kd = 0.8 µM). As a control, no significant affinity was observed for human [2Fe-2S] ferredoxin FDX2, cytochrome c or RNase A as control proteins. Collectively, these results data suggest that holo-GLRX5 preferentially cooperates with BOLA1, and holo-NFU1 with BOLA3.

DiscussionIn this work, we have defined the molecular function of the mitochondrial Bol1 and Bol3 proteins (mBols) as specific ISC assembly factors required for the insertion of [4Fe-4S] clusters into a small subset of mitochondrial target apoproteins. The two mBols act late in the ISC pathway, and execute a largely overlapping function, because only simultaneous deletion of both BOL genes was associated with appreciable effects on mitochondrial [4Fe-4S] target proteins (Figure 8). Lipoic acid synthase (LIAS) with its two [4Fe-4S] clusters is the primary client of mBol function because of strong effects on lipoic acid-dependent proteins (such as PDH and KGDH) in bol13Δ cells. These severe effects were observed under all experimental conditions tested, unlike the comparatively weak effects on SDH (complex II) which typically is one of the more sensitive mitochondrial Fe/S proteins upon ISC factor defects. Therefore, we cannot exclude an indirect effect of the BOL gene deletion on SDH maturation. The [4Fe-4S] aconitase, on the other hand, was not or hardly affected in bol13Δ cells distinguishing the BOL deletion phenotype from that of all other known ISC genes, with the exception of the complex I-specific IND1 (Bych et al., 2008; Sheftel et al., 2009). We propose that the rather inconspicuous effects on aconitase are indirect consequences of the metabolic and respiratory alterations arising from lipoic acid-dependent enzyme defects which mainly affect the citric acid cycle (PDH and KGDH) and the amino acid metabolism (BCKDH, GCS; Figure 8). The functions of the mBols are largely overlapping, yet not entirely identical. This was evident from small effects on LIAS and SDH in bol3Δ cells, versus no detectable alterations upon BOL1 deletion. Moreover, double deletion of BOL3 and NFU1, encoding another late-acting ISC factor (Navarro-Sastre et al., 2011), (Melber et al., 2016; accompanying manuscript), exacerbated the Fe/S protein defects compared to single deletions, while double deletion of BOL1 and NFU1 behaved similarly to nfu1Δ cells (Figure 4A,B). Two models may explain the largely complementary functions of the two mBols. i) The mBols assist the same biochemical reaction. In this case, the subtle differences between bol1Δ and bol3Δ strains may be due to minor target apoprotein specificities of the two mBols or the different protein interactions with Grx5 and Nfu1. ii) The mBols act consecutively in two independent maturation steps that both can be bypassed to some extent. Even though we biochemically favor the first possibility, current knowledge does not allow a more precise mechanistic definition of mBol function. Taken together, the major evident physiological function of the yeast mBol proteins is in lipoate synthase maturation. A similar defect is seen in human cells obtained from BOLA3 patients (Baker et al., 2014; Cameron et al., 2011; Haack et al., 2013). This phenotypical resemblance suggests that the yeast mBols and at least human BOLA3 are functionally similar. Even though yeast appears to be a suitable model system for the physiological and mechanistic investigation of mBols, the two human BOLAs now need to be studied in cell culture to gain better insights into their relative function.10.7554/eLife.16673.025Figure 8.Working model for the role of mitochondrial BOLA proteins as specific ISC assembly factors in the late phase of mitochondrial Fe/S protein biogenesis.Members of the core ISC assembly machinery including the sulfur donor NFS1, the scaffold protein ISCU, and dedicated chaperones (Chap.) mediate the assembly of a transiently bound, glutathione (GS)-coordinated [2Fe-2S] cluster on the monothiol glutaredoxin GLRX5 which is essential for [2Fe-2S] protein maturation, cytosolic Fe/S protein assembly (CIA), and cellular iron regulation in yeast. With the help of ISCA1, ISCA2, and IBA57, the GLRX5-bound [2Fe-2S] cluster is converted into a [4Fe-4S] type. BOLA1 and BOLA3 together with NFU1 and IND1 function in a specific assembly of mitochondrial [4Fe-4S] proteins as indicated. The central target of the BOLA proteins is the Fe/S protein lipoic acid (LA) synthase (LIAS). Its product is used as a cofactor of five mitochondrial enzymes including pyruvate dehydrogenase (PDH) and 2-ketoglutarate dehydrogenase (KGDH). Based on the hetero-complex formation with the BOLA proteins an additional function of GLRX5 in this late phase of mitochondrial Fe/S protein assembly is likely. Mutations in human NFU1 and BOLA3 cause multiple mitochondrial dysfunction syndromes (MMDS; for review see [Beilschmidt and Puccio, 2014; Stehling et al., 2014]).DOI:
http://dx.doi.org/10.7554/eLife.16673.025Several findings allowed us to define the site of action of the mBols in the multi-step ISC assembly pathway (Figure 8). First, the mBols were not required for maturation of mitochondrial [2Fe-2S] proteins or cytosolic Fe/S proteins. Second, their deletion did not affect the yeast cellular iron regulon. These criteria distinguish the mBol phenotype from that of the components of the core ISC machinery including Grx5, that is the final core ISC factor (Uzarska et al., 2013), and suggest that mBols act subsequently to Grx5. This view is further corroborated by our finding that, third, 55Fe incorporation into Grx5 occurred independently of mBols. Finally, the function of the mBols must be after the involvement of Isa-Iba57 proteins in the [4Fe-4S] cluster formation, because the mBols exhibit high [4Fe-4S] target specificity. In Isa-Iba57-defective cells, all mitochondrial [4Fe-4S] proteins and the non-Fe/S protein cytochrome oxidase are severely affected, unlike in bol13Δ cells (Mühlenhoff et al., 2011; Sheftel et al., 2012). Overall, the site of action of the mBols is similar to that of the late-acting ISC assembly factor Nfu1 (Navarro-Sastre et al., 2011) (Figure 8). However, we found that the mBol and Nfu1 proteins cannot complement each other’s function, even after overexpression, suggesting that they both fulfil a highly specific task in the dedicated insertion of [4Fe-4S] clusters into target apoproteins.The observation that the mBols function subsequently to Grx5 is somewhat unexpected in light of our findings that the mBols form binary complexes with this core ISC component. Apparently, complex formation with the mBols is not critical for Grx5’s major function as a core ISC factor (Figure 8), as the GRX5 deficiency is associated with rather severe defects in virtually all mitochondrial and cytosolic Fe/S proteins plus a misregulation of cellular iron homeostasis (Rodriguez-Manzaneque et al., 2002; Uzarska et al., 2013; Ye et al., 2010), radically distinguishing this phenotype from that of the rather inconspicuous BOL1-BOL3 deletion. It is well possible that the Grx5-mBol complexes play a role at the later stage of Fe/S cluster insertion into apoproteins. This seems likely based on interaction between BOLA1 and GLRX5 observed in human cells (Willems et al., 2013), and similar interactions reported in the accompanying manuscript (Melber et al., 2016). At present, it is impossible to verify such a function by in vivo studies, because the potential Grx5 involvement at this later stage of the ISC pathway is hidden by the severe GRX5 deletion phenotype.In our present work, we have characterized the Grx5-mBol interactions by a number of in vitro techniques including gel filtration, CD spectroscopy and NMR structural analyses. Human GLRX5 interacted with both mBols, stoichiometrically forming a 1:1 complex both in the apo- and chemically reconstituted holo-forms, involving similar regions on the three proteins as revealed by NMR studies. However, for the holo-forms, further residues were affected, and were clustered around the region containing the invariant histidine (His102 in BOLA1 and His96 in BOLA3; Figure 7C,D), which in cytosolic Bol2 is a ligand for Fe/S cluster binding with Grx3-Grx4 (Li et al., 2011). These Bol regions include other potential cluster ligands, which are two His residues in BOLA1, located at the end of β2 and in the loop between β1 and β2, and one Cys in BOLA3, located in the loop between β1 and β2. CD spectroscopy revealed striking differences in Fe/S cluster binding to the two holo-GLRX5-BOLA complexes. Compared to holo-GLRX5, the holo-GLRX5-BOLA1 complex showed characteristic new spectral features including a new peak at ~400 nm. GLRX5 and BOLA1 coordinate the [2Fe-2S] cluster with high affinity. The spectral features of the complex were rather insensitive to changes in ionic conditions or to the presence of additional GSH. Most characteristically, the [2Fe-2S] cluster of GLRX5 was stabilized by BOLA1 binding against destruction by the reductant dithionite indicating a stable holo-GLRX5-BOLA1 complex.All these criteria were different for the holo-GLRX5-BOLA3 complex. First, the CD spectral features of the holo-GLRX5-BOLA3 complex differed from those of holo-GLRX5 by a general signal decrease, and were observed mainly in the presence of GSH suggesting that without added GSH the GLRX5 cluster is not shared with BOLA3. Second, the spectral changes induced by BOLA3 addition to holo-GLRX5 were not saturable indicating a rather weak influence of BOLA3 on the GLRX5 Fe/S cluster. Third, the CD signals became rather weak at high BOLA3 or salt concentrations. Finally, dithionite readily destroyed the [2Fe-2S] cluster of GLRX5 in the presence and absence of BOLA3. In support of this data, gel filtration did not identify a stable complex between holo-GLRX5 and BOLA3 indicating that their interaction is kinetically labile, and a hetero-dimer may dissociate during the non-equilibrium method. All these findings document radically different Fe/S cluster binding properties of the two GLXR5-BOLA complexes with BOLA1 stabilizing and BOLA3 destabilizing the [2Fe-2S] cluster. Interestingly, we found a preferential interaction of BOLA3 with the holoform of NFU1 by thermophoresis. These data suggest that BOLA3 may stabilize the [4Fe-4S] cluster bound on NFU1. A specific genetic interaction of NFU1 and BOL3 supports our biochemical findings. Double deletion of these genes exacerbated the defects of SDH and LIAS, whereas simultaneous deletions of NFU1 and BOL1 did not (Figure 4A and B). This suggests that Bol3 and Nfu1 cooperate in SDH and LIAS maturation. Future biochemical studies have to address the physiological meaning of the stabilizing or destabilizing function of the mBols for the different Fe/S clusters on Grx5 and Nfu1.The NMR structures of the mBols share a similar fold, alike to those of other eukaryotic homologous (Abendroth et al., 2011; Kasai et al., 2004; Roret et al., 2014). Further, similar interacting regions for the apo- and holo-states of the GLRX5-BOLA heterodimers were found, and a similar location of the invariant His residue, a known Fe/S cluster ligand in Bol2 (Li et al., 2011). However, we note that its surrounding is structurally different in the two BOLAs, that is His102 of BOLA1 is the first residue following a long β-strand, while His96 of BOLA3 is in a 310 helix (Figure 6C). The other potential Fe/S cluster ligands are located in structurally different positions of the two mBols. In particular, the location of the two conserved His58, His67 residues in BOLA1 and of Cys59 in BOLA3 are different (Figure 6C). These features can possibly explain the observed differences in the binding and stability of [2Fe-2S] clusters to the holo heterodimers with GLRX5. The distinct functional roles of the mBols could be viewed as consequence of the different coordination spheres of the Fe/S cluster.The role of the mBols in a late step of the mitochondrial ISC pathway fits well with our observation that their functional deficiency (bol13Δ cells) does not cause any significant defects on cytosolic Fe/S proteins. Interestingly, a combination of these gene deletions with BOL2 elicited significant effects, even though these were rather weak compared to known core ISC or CIA deficiencies (see, e.g., Gerber et al., 2004; Hausmann et al., 2005; Netz et al., 2010). The defects, especially in the triple deletion mutant bol123Δ, are therefore best explained by combined detrimental effects of mitochondrial respiratory and metabolic changes arising from the BOL1-BOL3 deletion (see above) and the disturbed cellular iron homeostasis resulting from BOL2 ablation. Single BOL2 deletion is not associated with major phenotypes apart from iron homeostasis (Kumanovics et al., 2008) (Figure 1—figure supplement 3). This is clearly different from the usual lethality of CIA gene deletions (Netz et al., 2014; Paul and Lill, 2015), and fits well with the rather weak or even inconspicuous effects of a BOL2 deletion on the maturation of the cytosolic Fe/S proteins Leu1 and Rli1. Any slight effects are most likely due to the auxiliary role of Bol2 in iron metabolism (Kumanovics et al., 2008). In this function, Bol2 cooperates with Grx3-Grx4 (human PICOT) which, in its role in cellular iron mobilization, is essential for cytosolic Fe/S protein maturation (Banci et al., 2015; Haunhorst et al., 2013; Mühlenhoff et al., 2010).Our comprehensive physiological investigations of mBol function in yeast reveal that bol13Δ cells behave similarly to human patients deficient in BOLA3 (Baker et al., 2014; Cameron et al., 2011; Haack et al., 2013). Patients with the resulting multiple mitochondrial dysfunction syndrome 2 (MMDS 2) show major defects in lipoic acid-dependent enzymes, similar to the most severe effect observed in bol13Δ yeast. Moreover, the patient cells are deficient in respiratory complexes I and II, the latter also being affected in yeast. We conclude that the mBol protein function is conserved from yeast to man with a major role in [4Fe-4S] cluster insertion into LIAS and respiratory complexes I and II, whereas Fe/S proteins such as aconitase are matured independently of mBol function. The conclusion of similar mBol functions in yeast and man may be surprising based on the rather inconspicuous yeast deletion phenotypes versus the lethal effect in BOLA3-deficient individuals. This difference may simply reflect the higher complexity of a multi-cellular organism and its dependence on respiration and metabolic homeostasis, whereas yeast can tolerate substantial deviations from optimal conditions. Notably, RNAi-mediated depletion of BOLA1 was associated with rather mild effects (Willems et al., 2013) predicting that the combination with a BOLA3 depletion may create a more pathological phenotype mimicking that of human patients and of bol13Δ yeast cells. Our elucidation of the target apoprotein-specific ISC assembly function of mBols will be instrumental for future mechanistic studies on how the mBols facilitate the insertion of [4Fe-4S] clusters into LIAS and possibly other Fe/S proteins. These studies will also have to reveal why other Fe/S proteins such as aconitase are matured efficiently in the absence of these specialized ISC assembly factors.

In this work, we have defined the molecular function of the mitochondrial Bol1 and Bol3 proteins (mBols) as specific ISC assembly factors required for the insertion of [4Fe-4S] clusters into a small subset of mitochondrial target apoproteins. The two mBols act late in the ISC pathway, and execute a largely overlapping function, because only simultaneous deletion of both BOL genes was associated with appreciable effects on mitochondrial [4Fe-4S] target proteins (Figure 8). Lipoic acid synthase (LIAS) with its two [4Fe-4S] clusters is the primary client of mBol function because of strong effects on lipoic acid-dependent proteins (such as PDH and KGDH) in bol13Δ cells. These severe effects were observed under all experimental conditions tested, unlike the comparatively weak effects on SDH (complex II) which typically is one of the more sensitive mitochondrial Fe/S proteins upon ISC factor defects. Therefore, we cannot exclude an indirect effect of the BOL gene deletion on SDH maturation. The [4Fe-4S] aconitase, on the other hand, was not or hardly affected in bol13Δ cells distinguishing the BOL deletion phenotype from that of all other known ISC genes, with the exception of the complex I-specific IND1 (Bych et al., 2008; Sheftel et al., 2009). We propose that the rather inconspicuous effects on aconitase are indirect consequences of the metabolic and respiratory alterations arising from lipoic acid-dependent enzyme defects which mainly affect the citric acid cycle (PDH and KGDH) and the amino acid metabolism (BCKDH, GCS; Figure 8). The functions of the mBols are largely overlapping, yet not entirely identical. This was evident from small effects on LIAS and SDH in bol3Δ cells, versus no detectable alterations upon BOL1 deletion. Moreover, double deletion of BOL3 and NFU1, encoding another late-acting ISC factor (Navarro-Sastre et al., 2011), (Melber et al., 2016; accompanying manuscript), exacerbated the Fe/S protein defects compared to single deletions, while double deletion of BOL1 and NFU1 behaved similarly to nfu1Δ cells (Figure 4A,B). Two models may explain the largely complementary functions of the two mBols. i) The mBols assist the same biochemical reaction. In this case, the subtle differences between bol1Δ and bol3Δ strains may be due to minor target apoprotein specificities of the two mBols or the different protein interactions with Grx5 and Nfu1. ii) The mBols act consecutively in two independent maturation steps that both can be bypassed to some extent. Even though we biochemically favor the first possibility, current knowledge does not allow a more precise mechanistic definition of mBol function. Taken together, the major evident physiological function of the yeast mBol proteins is in lipoate synthase maturation. A similar defect is seen in human cells obtained from BOLA3 patients (Baker et al., 2014; Cameron et al., 2011; Haack et al., 2013). This phenotypical resemblance suggests that the yeast mBols and at least human BOLA3 are functionally similar. Even though yeast appears to be a suitable model system for the physiological and mechanistic investigation of mBols, the two human BOLAs now need to be studied in cell culture to gain better insights into their relative function.10.7554/eLife.16673.025Figure 8.Working model for the role of mitochondrial BOLA proteins as specific ISC assembly factors in the late phase of mitochondrial Fe/S protein biogenesis.Members of the core ISC assembly machinery including the sulfur donor NFS1, the scaffold protein ISCU, and dedicated chaperones (Chap.) mediate the assembly of a transiently bound, glutathione (GS)-coordinated [2Fe-2S] cluster on the monothiol glutaredoxin GLRX5 which is essential for [2Fe-2S] protein maturation, cytosolic Fe/S protein assembly (CIA), and cellular iron regulation in yeast. With the help of ISCA1, ISCA2, and IBA57, the GLRX5-bound [2Fe-2S] cluster is converted into a [4Fe-4S] type. BOLA1 and BOLA3 together with NFU1 and IND1 function in a specific assembly of mitochondrial [4Fe-4S] proteins as indicated. The central target of the BOLA proteins is the Fe/S protein lipoic acid (LA) synthase (LIAS). Its product is used as a cofactor of five mitochondrial enzymes including pyruvate dehydrogenase (PDH) and 2-ketoglutarate dehydrogenase (KGDH). Based on the hetero-complex formation with the BOLA proteins an additional function of GLRX5 in this late phase of mitochondrial Fe/S protein assembly is likely. Mutations in human NFU1 and BOLA3 cause multiple mitochondrial dysfunction syndromes (MMDS; for review see [Beilschmidt and Puccio, 2014; Stehling et al., 2014]).DOI:
http://dx.doi.org/10.7554/eLife.16673.025

Working model for the role of mitochondrial BOLA proteins as specific ISC assembly factors in the late phase of mitochondrial Fe/S protein biogenesis.

Members of the core ISC assembly machinery including the sulfur donor NFS1, the scaffold protein ISCU, and dedicated chaperones (Chap.) mediate the assembly of a transiently bound, glutathione (GS)-coordinated [2Fe-2S] cluster on the monothiol glutaredoxin GLRX5 which is essential for [2Fe-2S] protein maturation, cytosolic Fe/S protein assembly (CIA), and cellular iron regulation in yeast. With the help of ISCA1, ISCA2, and IBA57, the GLRX5-bound [2Fe-2S] cluster is converted into a [4Fe-4S] type. BOLA1 and BOLA3 together with NFU1 and IND1 function in a specific assembly of mitochondrial [4Fe-4S] proteins as indicated. The central target of the BOLA proteins is the Fe/S protein lipoic acid (LA) synthase (LIAS). Its product is used as a cofactor of five mitochondrial enzymes including pyruvate dehydrogenase (PDH) and 2-ketoglutarate dehydrogenase (KGDH). Based on the hetero-complex formation with the BOLA proteins an additional function of GLRX5 in this late phase of mitochondrial Fe/S protein assembly is likely. Mutations in human NFU1 and BOLA3 cause multiple mitochondrial dysfunction syndromes (MMDS; for review see [Beilschmidt and Puccio, 2014; Stehling et al., 2014]).

DOI:
http://dx.doi.org/10.7554/eLife.16673.025

Several findings allowed us to define the site of action of the mBols in the multi-step ISC assembly pathway (Figure 8). First, the mBols were not required for maturation of mitochondrial [2Fe-2S] proteins or cytosolic Fe/S proteins. Second, their deletion did not affect the yeast cellular iron regulon. These criteria distinguish the mBol phenotype from that of the components of the core ISC machinery including Grx5, that is the final core ISC factor (Uzarska et al., 2013), and suggest that mBols act subsequently to Grx5. This view is further corroborated by our finding that, third, 55Fe incorporation into Grx5 occurred independently of mBols. Finally, the function of the mBols must be after the involvement of Isa-Iba57 proteins in the [4Fe-4S] cluster formation, because the mBols exhibit high [4Fe-4S] target specificity. In Isa-Iba57-defective cells, all mitochondrial [4Fe-4S] proteins and the non-Fe/S protein cytochrome oxidase are severely affected, unlike in bol13Δ cells (Mühlenhoff et al., 2011; Sheftel et al., 2012). Overall, the site of action of the mBols is similar to that of the late-acting ISC assembly factor Nfu1 (Navarro-Sastre et al., 2011) (Figure 8). However, we found that the mBol and Nfu1 proteins cannot complement each other’s function, even after overexpression, suggesting that they both fulfil a highly specific task in the dedicated insertion of [4Fe-4S] clusters into target apoproteins.

The observation that the mBols function subsequently to Grx5 is somewhat unexpected in light of our findings that the mBols form binary complexes with this core ISC component. Apparently, complex formation with the mBols is not critical for Grx5’s major function as a core ISC factor (Figure 8), as the GRX5 deficiency is associated with rather severe defects in virtually all mitochondrial and cytosolic Fe/S proteins plus a misregulation of cellular iron homeostasis (Rodriguez-Manzaneque et al., 2002; Uzarska et al., 2013; Ye et al., 2010), radically distinguishing this phenotype from that of the rather inconspicuous BOL1-BOL3 deletion. It is well possible that the Grx5-mBol complexes play a role at the later stage of Fe/S cluster insertion into apoproteins. This seems likely based on interaction between BOLA1 and GLRX5 observed in human cells (Willems et al., 2013), and similar interactions reported in the accompanying manuscript (Melber et al., 2016). At present, it is impossible to verify such a function by in vivo studies, because the potential Grx5 involvement at this later stage of the ISC pathway is hidden by the severe GRX5 deletion phenotype.

In our present work, we have characterized the Grx5-mBol interactions by a number of in vitro techniques including gel filtration, CD spectroscopy and NMR structural analyses. Human GLRX5 interacted with both mBols, stoichiometrically forming a 1:1 complex both in the apo- and chemically reconstituted holo-forms, involving similar regions on the three proteins as revealed by NMR studies. However, for the holo-forms, further residues were affected, and were clustered around the region containing the invariant histidine (His102 in BOLA1 and His96 in BOLA3; Figure 7C,D), which in cytosolic Bol2 is a ligand for Fe/S cluster binding with Grx3-Grx4 (Li et al., 2011). These Bol regions include other potential cluster ligands, which are two His residues in BOLA1, located at the end of β2 and in the loop between β1 and β2, and one Cys in BOLA3, located in the loop between β1 and β2. CD spectroscopy revealed striking differences in Fe/S cluster binding to the two holo-GLRX5-BOLA complexes. Compared to holo-GLRX5, the holo-GLRX5-BOLA1 complex showed characteristic new spectral features including a new peak at ~400 nm. GLRX5 and BOLA1 coordinate the [2Fe-2S] cluster with high affinity. The spectral features of the complex were rather insensitive to changes in ionic conditions or to the presence of additional GSH. Most characteristically, the [2Fe-2S] cluster of GLRX5 was stabilized by BOLA1 binding against destruction by the reductant dithionite indicating a stable holo-GLRX5-BOLA1 complex.

All these criteria were different for the holo-GLRX5-BOLA3 complex. First, the CD spectral features of the holo-GLRX5-BOLA3 complex differed from those of holo-GLRX5 by a general signal decrease, and were observed mainly in the presence of GSH suggesting that without added GSH the GLRX5 cluster is not shared with BOLA3. Second, the spectral changes induced by BOLA3 addition to holo-GLRX5 were not saturable indicating a rather weak influence of BOLA3 on the GLRX5 Fe/S cluster. Third, the CD signals became rather weak at high BOLA3 or salt concentrations. Finally, dithionite readily destroyed the [2Fe-2S] cluster of GLRX5 in the presence and absence of BOLA3. In support of this data, gel filtration did not identify a stable complex between holo-GLRX5 and BOLA3 indicating that their interaction is kinetically labile, and a hetero-dimer may dissociate during the non-equilibrium method. All these findings document radically different Fe/S cluster binding properties of the two GLXR5-BOLA complexes with BOLA1 stabilizing and BOLA3 destabilizing the [2Fe-2S] cluster. Interestingly, we found a preferential interaction of BOLA3 with the holoform of NFU1 by thermophoresis. These data suggest that BOLA3 may stabilize the [4Fe-4S] cluster bound on NFU1. A specific genetic interaction of NFU1 and BOL3 supports our biochemical findings. Double deletion of these genes exacerbated the defects of SDH and LIAS, whereas simultaneous deletions of NFU1 and BOL1 did not (Figure 4A and B). This suggests that Bol3 and Nfu1 cooperate in SDH and LIAS maturation. Future biochemical studies have to address the physiological meaning of the stabilizing or destabilizing function of the mBols for the different Fe/S clusters on Grx5 and Nfu1.

The NMR structures of the mBols share a similar fold, alike to those of other eukaryotic homologous (Abendroth et al., 2011; Kasai et al., 2004; Roret et al., 2014). Further, similar interacting regions for the apo- and holo-states of the GLRX5-BOLA heterodimers were found, and a similar location of the invariant His residue, a known Fe/S cluster ligand in Bol2 (Li et al., 2011). However, we note that its surrounding is structurally different in the two BOLAs, that is His102 of BOLA1 is the first residue following a long β-strand, while His96 of BOLA3 is in a 310 helix (Figure 6C). The other potential Fe/S cluster ligands are located in structurally different positions of the two mBols. In particular, the location of the two conserved His58, His67 residues in BOLA1 and of Cys59 in BOLA3 are different (Figure 6C). These features can possibly explain the observed differences in the binding and stability of [2Fe-2S] clusters to the holo heterodimers with GLRX5. The distinct functional roles of the mBols could be viewed as consequence of the different coordination spheres of the Fe/S cluster.

The role of the mBols in a late step of the mitochondrial ISC pathway fits well with our observation that their functional deficiency (bol13Δ cells) does not cause any significant defects on cytosolic Fe/S proteins. Interestingly, a combination of these gene deletions with BOL2 elicited significant effects, even though these were rather weak compared to known core ISC or CIA deficiencies (see, e.g., Gerber et al., 2004; Hausmann et al., 2005; Netz et al., 2010). The defects, especially in the triple deletion mutant bol123Δ, are therefore best explained by combined detrimental effects of mitochondrial respiratory and metabolic changes arising from the BOL1-BOL3 deletion (see above) and the disturbed cellular iron homeostasis resulting from BOL2 ablation. Single BOL2 deletion is not associated with major phenotypes apart from iron homeostasis (Kumanovics et al., 2008) (Figure 1—figure supplement 3). This is clearly different from the usual lethality of CIA gene deletions (Netz et al., 2014; Paul and Lill, 2015), and fits well with the rather weak or even inconspicuous effects of a BOL2 deletion on the maturation of the cytosolic Fe/S proteins Leu1 and Rli1. Any slight effects are most likely due to the auxiliary role of Bol2 in iron metabolism (Kumanovics et al., 2008). In this function, Bol2 cooperates with Grx3-Grx4 (human PICOT) which, in its role in cellular iron mobilization, is essential for cytosolic Fe/S protein maturation (Banci et al., 2015; Haunhorst et al., 2013; Mühlenhoff et al., 2010).

Our comprehensive physiological investigations of mBol function in yeast reveal that bol13Δ cells behave similarly to human patients deficient in BOLA3 (Baker et al., 2014; Cameron et al., 2011; Haack et al., 2013). Patients with the resulting multiple mitochondrial dysfunction syndrome 2 (MMDS 2) show major defects in lipoic acid-dependent enzymes, similar to the most severe effect observed in bol13Δ yeast. Moreover, the patient cells are deficient in respiratory complexes I and II, the latter also being affected in yeast. We conclude that the mBol protein function is conserved from yeast to man with a major role in [4Fe-4S] cluster insertion into LIAS and respiratory complexes I and II, whereas Fe/S proteins such as aconitase are matured independently of mBol function. The conclusion of similar mBol functions in yeast and man may be surprising based on the rather inconspicuous yeast deletion phenotypes versus the lethal effect in BOLA3-deficient individuals. This difference may simply reflect the higher complexity of a multi-cellular organism and its dependence on respiration and metabolic homeostasis, whereas yeast can tolerate substantial deviations from optimal conditions. Notably, RNAi-mediated depletion of BOLA1 was associated with rather mild effects (Willems et al., 2013) predicting that the combination with a BOLA3 depletion may create a more pathological phenotype mimicking that of human patients and of bol13Δ yeast cells. Our elucidation of the target apoprotein-specific ISC assembly function of mBols will be instrumental for future mechanistic studies on how the mBols facilitate the insertion of [4Fe-4S] clusters into LIAS and possibly other Fe/S proteins. These studies will also have to reveal why other Fe/S proteins such as aconitase are matured efficiently in the absence of these specialized ISC assembly factors.

Materials and methodsYeast strains and growth conditionsS. cerevisiae strains used in this study are listed in Supplementary file 1A. Cells were cultivated in rich medium (YP) or synthetic minimal medium (SC) supplemented with amino acids as required and 2% (w/v) glucose, galactose or acetate, or 3% (w/v) glycerol (Sherman, 2002). Iron-depleted minimal media were prepared using yeast nitrogen base without FeCl3 (ForMedium). Plasmids used in this study are listed in Supplementary file 1B. Plasmid constructs were verified by DNA sequencing and/or functional complementation of a corresponding yeast mutant.Purification of recombinant proteinsGenes of human BOLA1 (residues 30–137), BOLA3 (25–107), GLRX5 (32–157) and NFU1 (57–254) (each lacking its mitochondrial presequence) were inserted into the multiple cloning site I of pET-Duet1 vector (Novagen-Merck, Darmstadt, Germany) with a N-terminal His-tag. N-terminally tagged proteins were purified from E. coli BL21 (DE3) using a HisTrap HP column (GE Healthcare). The eluted proteins were treated with 5 mM DTT and isolated by gel filtration on a Superdex 200 16/60 gel filtration column (GE Healthcare) in reconstitution buffer R (50 mM Tris-HCl, pH 8.0, 150 mM NaCl, 5% glycerol). For NMR the BOLA proteins (BOLA1, 21–137; BOLA3, 27–107) were obtained from BL21(DE3)GOLD Escherichia coli (Novagen) transformed with pETG20A vector (BOLA1) and pET15 vector (BOLA3). Cells were grown at 37°C in LB or minimal media with (15NH4)2SO4 and/or 13C-glucose containing 100 μg/mL ampicillin. Both proteins contained a N-terminal His-tag, which was cleaved by tobacco etch virus protease in 50 mM Tris-HCl pH 8, 0.5 mM EDTA, 5 mM GSH, and 1 mM DTT overnight at room temperature. GLRX5 protein in its apo- and holo-forms was produced as previously reported (Banci et al., 2014).Chemical reconstitution of Fe/S proteinsAll solutions used for chemical reconstitution were prepared freshly in a COY anaerobic chamber by dissolving in degassed water. Protein samples were reduced in buffer R containing 5 mM DTT for 2–3 hr on ice in buffer R containing a two fold excess of GSH. Reconstitution was initiated at room temperature by the addition of a 2–5-fold excess of ferric ammonium citrate. After five minutes a 2–5-fold excess of lithium sulfide was slowly added. After 2 hr reconstituted proteins were desalted on a PD-10 column (GE Healthcare) equilibrated with buffer R. Incorporation of the Fe/S clusters into apoproteins was verified by UV-Vis (V-550, Jasco Inc.) and CD spectroscopy (J-815, Jasco Inc.) and the determination of bound iron and acid-labile sulfur (Pierik et al., 1992).Circular dichroism spectroscopyCD spectroscopy (J-815, Jasco Inc.) of human GLRX5 reconstituted either alone (>85% holo based on iron and acid-labile sulfur determination) or in the presence of stoichiometric amounts of either BOLA1 or BOLA3 was performed anaerobically (protein concentrations 200–300 µM) in buffer R, buffer N (50 mM phosphate buffer pH 7, 5 mM GSH, and 5 mM DTT), or buffer P (50 mM phosphate buffer pH 8, 150 mM NaCl, 5 mM GSH). Alternatively, reconstituted holo-GLRX5 was titrated with increasing concentrations of BOLA1 or BOLA3. CD spectra were recorded anaerobically at 21°C in 1 mm cuvettes.Affinity measurements using microscale thermophoresis (MST)The bait proteins were fluorescently labeled using the Monolith NT Protein Labeling Kit RED (NanoTemper Technologies) with NT-647 dye as recommended by the manufacturer. Labeled bait proteins (200 nM) were titrated with the indicated unlabeled proteins (from 200 µM to 6.1 nM) in buffer T (50 mM KPi, pH 7.4, 150 mM NaCl, 5% glycerol, 0.05 mg/ml BSA, 0.05% Tween20). Binding assays were performed using Monolith NT.115 (NanoTemper Technologies) at 21°C (LED power between 40% and 60%, IR laser power 75%) in standard capillaries under anaerobic conditions at 680 nm. The results were processed by NanoTemper Analysis 1.2.009 and GraphPadPrism5 software to estimate Kd values.NMR experimentsAll NMR experiments required for resonance assignment and structure calculations of BOLA1 (PDB ID 5LCI; BMRB 34013) and BOLA3 (PDB ID 2NCL; BMRB 26031) were recorded on Bruker AVANCE 500, 700 and 900 MHz spectrometers on 0.5–1 mM 13C,15N-labeled or 15N-labeled BOLA1 and BOLA3 samples in 50 mM phosphate buffer, pH 7.0, 5 mM DTT, containing 10% (v/v) D2O. All NMR spectra were collected at 298 K, processed using the standard BRUKER software (Topspin) and analyzed through the CARA program. The 1 H, 13C and 15N resonance assignment of BOLAs were performed following a standard triple-resonance and TOCSY-based approach. Secondary structure analysis was performed by TALOS+. Structure calculations of BOLA3 and BOLA1 were performed with the software package UNIO (ATNOS/CANDID/CYANA). The 20 conformers with the lowest residual target function values were subjected to restrained energy minimization in explicit water with the program AMBER 12.0. (D.A. Case et al. University of California, San Francisco). The quality of the structures was evaluated using the Protein Structure Validation Software suite (PSVS) and the iCING program. 15N heteronuclear relaxation experiments on 15N-labeled samples of BOLA1 and BOLA3 were recorded on Bruker AVANCE 500 MHz spectrometer at 298 K to measure 15N backbone longitudinal (R1) and transverse (R2) relaxation rates, as well as the heteronuclear 15N[1H] NOEs. The rotational correlation time values were estimated from the R2/R1 ratio using the program QUADRATIC_DIFFUSION. The relaxation data for those NHs having an exchange contribution to the R2 value or exhibiting large-amplitude fast internal motions were excluded from the analysis. Theoretical estimates of the rotational correlation time under the chosen experimental conditions of magnetic field and temperature were obtained using HYDRONMR program following a standard procedure (Figure 6—figure supplement 3).The interaction between apo-GLRX5 and BOLA proteins was investigated by 1H-15N HSQC NMR spectra, titrating 15N-labeled apo-GLRX5 with unlabeled BOLA1 or BOLA3, and 15N labeled BOLA1 or BOLA3 with unlabeled apo-GLRX5 in degassed buffer N containing 10% (v/v) D2O) at 298K. Spectral changes were monitored upon the addition of increasing amounts of the unlabeled partner. Protein interaction between [2Fe-2S] holo-GLRX5 and BOLA proteins was investigated comparing 1H-15N HSQC NMR spectra of 15N-labeled BOLA1 (or BOLA3) with that of a 1:1 15N-labeled BOLA1 (or BOLA3)-unlabeled apo GLRX5 mixture chemically reconstituted with [2Fe-2S], or of 15N-labeled apo-GLRX5 with that of a 1:1 15N-labeled apo-GLRX5-unlabeled BOLA1 (or BOLA3) mixture chemically reconstituted with [2Fe-2S], in degassed buffer N containing 10% (v/v) D2O) at 298K. NMR data were analyzed with CARA program.Miscellaneous methodsThe following published methods were used: manipulation of DNA and PCR (Sambrook and Russell, 2001); transformation of yeast cells (Gietz and Woods, 2002); isolation of yeast mitochondria and post-mitochondrial supernatant (Diekert et al., 2001); immunostaining (Harlow and Lane, 1998); determination of enzyme activities and of promoter strength using FET3-GFP or FIT3-GFP reporter plasmids (Molik et al., 2007); in vivo labeling of yeast cells with 55FeCl (ICN) and measurement of 55Fe-incorporation into Fe/S proteins by immunoprecipitation and scintillation counting (Molik et al., 2007; Pierik et al., 2009). Antibodies were raised in rabbits against recombinant purified proteins. Antibodies against c-Myc or HA were obtained from Santa-Cruz, protein A sepharose from GE Healthcare.

Materials and methods

Yeast strains and growth conditionsS. cerevisiae strains used in this study are listed in Supplementary file 1A. Cells were cultivated in rich medium (YP) or synthetic minimal medium (SC) supplemented with amino acids as required and 2% (w/v) glucose, galactose or acetate, or 3% (w/v) glycerol (Sherman, 2002). Iron-depleted minimal media were prepared using yeast nitrogen base without FeCl3 (ForMedium). Plasmids used in this study are listed in Supplementary file 1B. Plasmid constructs were verified by DNA sequencing and/or functional complementation of a corresponding yeast mutant.

Yeast strains and growth conditions

S. cerevisiae strains used in this study are listed in Supplementary file 1A. Cells were cultivated in rich medium (YP) or synthetic minimal medium (SC) supplemented with amino acids as required and 2% (w/v) glucose, galactose or acetate, or 3% (w/v) glycerol (Sherman, 2002). Iron-depleted minimal media were prepared using yeast nitrogen base without FeCl3 (ForMedium). Plasmids used in this study are listed in Supplementary file 1B. Plasmid constructs were verified by DNA sequencing and/or functional complementation of a corresponding yeast mutant.

Purification of recombinant proteinsGenes of human BOLA1 (residues 30–137), BOLA3 (25–107), GLRX5 (32–157) and NFU1 (57–254) (each lacking its mitochondrial presequence) were inserted into the multiple cloning site I of pET-Duet1 vector (Novagen-Merck, Darmstadt, Germany) with a N-terminal His-tag. N-terminally tagged proteins were purified from E. coli BL21 (DE3) using a HisTrap HP column (GE Healthcare). The eluted proteins were treated with 5 mM DTT and isolated by gel filtration on a Superdex 200 16/60 gel filtration column (GE Healthcare) in reconstitution buffer R (50 mM Tris-HCl, pH 8.0, 150 mM NaCl, 5% glycerol). For NMR the BOLA proteins (BOLA1, 21–137; BOLA3, 27–107) were obtained from BL21(DE3)GOLD Escherichia coli (Novagen) transformed with pETG20A vector (BOLA1) and pET15 vector (BOLA3). Cells were grown at 37°C in LB or minimal media with (15NH4)2SO4 and/or 13C-glucose containing 100 μg/mL ampicillin. Both proteins contained a N-terminal His-tag, which was cleaved by tobacco etch virus protease in 50 mM Tris-HCl pH 8, 0.5 mM EDTA, 5 mM GSH, and 1 mM DTT overnight at room temperature. GLRX5 protein in its apo- and holo-forms was produced as previously reported (Banci et al., 2014).

Purification of recombinant proteins

Genes of human BOLA1 (residues 30–137), BOLA3 (25–107), GLRX5 (32–157) and NFU1 (57–254) (each lacking its mitochondrial presequence) were inserted into the multiple cloning site I of pET-Duet1 vector (Novagen-Merck, Darmstadt, Germany) with a N-terminal His-tag. N-terminally tagged proteins were purified from E. coli BL21 (DE3) using a HisTrap HP column (GE Healthcare). The eluted proteins were treated with 5 mM DTT and isolated by gel filtration on a Superdex 200 16/60 gel filtration column (GE Healthcare) in reconstitution buffer R (50 mM Tris-HCl, pH 8.0, 150 mM NaCl, 5% glycerol). For NMR the BOLA proteins (BOLA1, 21–137; BOLA3, 27–107) were obtained from BL21(DE3)GOLD Escherichia coli (Novagen) transformed with pETG20A vector (BOLA1) and pET15 vector (BOLA3). Cells were grown at 37°C in LB or minimal media with (15NH4)2SO4 and/or 13C-glucose containing 100 μg/mL ampicillin. Both proteins contained a N-terminal His-tag, which was cleaved by tobacco etch virus protease in 50 mM Tris-HCl pH 8, 0.5 mM EDTA, 5 mM GSH, and 1 mM DTT overnight at room temperature. GLRX5 protein in its apo- and holo-forms was produced as previously reported (Banci et al., 2014).

Chemical reconstitution of Fe/S proteinsAll solutions used for chemical reconstitution were prepared freshly in a COY anaerobic chamber by dissolving in degassed water. Protein samples were reduced in buffer R containing 5 mM DTT for 2–3 hr on ice in buffer R containing a two fold excess of GSH. Reconstitution was initiated at room temperature by the addition of a 2–5-fold excess of ferric ammonium citrate. After five minutes a 2–5-fold excess of lithium sulfide was slowly added. After 2 hr reconstituted proteins were desalted on a PD-10 column (GE Healthcare) equilibrated with buffer R. Incorporation of the Fe/S clusters into apoproteins was verified by UV-Vis (V-550, Jasco Inc.) and CD spectroscopy (J-815, Jasco Inc.) and the determination of bound iron and acid-labile sulfur (Pierik et al., 1992).

Chemical reconstitution of Fe/S proteins

All solutions used for chemical reconstitution were prepared freshly in a COY anaerobic chamber by dissolving in degassed water. Protein samples were reduced in buffer R containing 5 mM DTT for 2–3 hr on ice in buffer R containing a two fold excess of GSH. Reconstitution was initiated at room temperature by the addition of a 2–5-fold excess of ferric ammonium citrate. After five minutes a 2–5-fold excess of lithium sulfide was slowly added. After 2 hr reconstituted proteins were desalted on a PD-10 column (GE Healthcare) equilibrated with buffer R. Incorporation of the Fe/S clusters into apoproteins was verified by UV-Vis (V-550, Jasco Inc.) and CD spectroscopy (J-815, Jasco Inc.) and the determination of bound iron and acid-labile sulfur (Pierik et al., 1992).

Circular dichroism spectroscopyCD spectroscopy (J-815, Jasco Inc.) of human GLRX5 reconstituted either alone (>85% holo based on iron and acid-labile sulfur determination) or in the presence of stoichiometric amounts of either BOLA1 or BOLA3 was performed anaerobically (protein concentrations 200–300 µM) in buffer R, buffer N (50 mM phosphate buffer pH 7, 5 mM GSH, and 5 mM DTT), or buffer P (50 mM phosphate buffer pH 8, 150 mM NaCl, 5 mM GSH). Alternatively, reconstituted holo-GLRX5 was titrated with increasing concentrations of BOLA1 or BOLA3. CD spectra were recorded anaerobically at 21°C in 1 mm cuvettes.

Circular dichroism spectroscopy

CD spectroscopy (J-815, Jasco Inc.) of human GLRX5 reconstituted either alone (>85% holo based on iron and acid-labile sulfur determination) or in the presence of stoichiometric amounts of either BOLA1 or BOLA3 was performed anaerobically (protein concentrations 200–300 µM) in buffer R, buffer N (50 mM phosphate buffer pH 7, 5 mM GSH, and 5 mM DTT), or buffer P (50 mM phosphate buffer pH 8, 150 mM NaCl, 5 mM GSH). Alternatively, reconstituted holo-GLRX5 was titrated with increasing concentrations of BOLA1 or BOLA3. CD spectra were recorded anaerobically at 21°C in 1 mm cuvettes.

Affinity measurements using microscale thermophoresis (MST)The bait proteins were fluorescently labeled using the Monolith NT Protein Labeling Kit RED (NanoTemper Technologies) with NT-647 dye as recommended by the manufacturer. Labeled bait proteins (200 nM) were titrated with the indicated unlabeled proteins (from 200 µM to 6.1 nM) in buffer T (50 mM KPi, pH 7.4, 150 mM NaCl, 5% glycerol, 0.05 mg/ml BSA, 0.05% Tween20). Binding assays were performed using Monolith NT.115 (NanoTemper Technologies) at 21°C (LED power between 40% and 60%, IR laser power 75%) in standard capillaries under anaerobic conditions at 680 nm. The results were processed by NanoTemper Analysis 1.2.009 and GraphPadPrism5 software to estimate Kd values.

Affinity measurements using microscale thermophoresis (MST)

The bait proteins were fluorescently labeled using the Monolith NT Protein Labeling Kit RED (NanoTemper Technologies) with NT-647 dye as recommended by the manufacturer. Labeled bait proteins (200 nM) were titrated with the indicated unlabeled proteins (from 200 µM to 6.1 nM) in buffer T (50 mM KPi, pH 7.4, 150 mM NaCl, 5% glycerol, 0.05 mg/ml BSA, 0.05% Tween20). Binding assays were performed using Monolith NT.115 (NanoTemper Technologies) at 21°C (LED power between 40% and 60%, IR laser power 75%) in standard capillaries under anaerobic conditions at 680 nm. The results were processed by NanoTemper Analysis 1.2.009 and GraphPadPrism5 software to estimate Kd values.

NMR experimentsAll NMR experiments required for resonance assignment and structure calculations of BOLA1 (PDB ID 5LCI; BMRB 34013) and BOLA3 (PDB ID 2NCL; BMRB 26031) were recorded on Bruker AVANCE 500, 700 and 900 MHz spectrometers on 0.5–1 mM 13C,15N-labeled or 15N-labeled BOLA1 and BOLA3 samples in 50 mM phosphate buffer, pH 7.0, 5 mM DTT, containing 10% (v/v) D2O. All NMR spectra were collected at 298 K, processed using the standard BRUKER software (Topspin) and analyzed through the CARA program. The 1 H, 13C and 15N resonance assignment of BOLAs were performed following a standard triple-resonance and TOCSY-based approach. Secondary structure analysis was performed by TALOS+. Structure calculations of BOLA3 and BOLA1 were performed with the software package UNIO (ATNOS/CANDID/CYANA). The 20 conformers with the lowest residual target function values were subjected to restrained energy minimization in explicit water with the program AMBER 12.0. (D.A. Case et al. University of California, San Francisco). The quality of the structures was evaluated using the Protein Structure Validation Software suite (PSVS) and the iCING program. 15N heteronuclear relaxation experiments on 15N-labeled samples of BOLA1 and BOLA3 were recorded on Bruker AVANCE 500 MHz spectrometer at 298 K to measure 15N backbone longitudinal (R1) and transverse (R2) relaxation rates, as well as the heteronuclear 15N[1H] NOEs. The rotational correlation time values were estimated from the R2/R1 ratio using the program QUADRATIC_DIFFUSION. The relaxation data for those NHs having an exchange contribution to the R2 value or exhibiting large-amplitude fast internal motions were excluded from the analysis. Theoretical estimates of the rotational correlation time under the chosen experimental conditions of magnetic field and temperature were obtained using HYDRONMR program following a standard procedure (Figure 6—figure supplement 3).The interaction between apo-GLRX5 and BOLA proteins was investigated by 1H-15N HSQC NMR spectra, titrating 15N-labeled apo-GLRX5 with unlabeled BOLA1 or BOLA3, and 15N labeled BOLA1 or BOLA3 with unlabeled apo-GLRX5 in degassed buffer N containing 10% (v/v) D2O) at 298K. Spectral changes were monitored upon the addition of increasing amounts of the unlabeled partner. Protein interaction between [2Fe-2S] holo-GLRX5 and BOLA proteins was investigated comparing 1H-15N HSQC NMR spectra of 15N-labeled BOLA1 (or BOLA3) with that of a 1:1 15N-labeled BOLA1 (or BOLA3)-unlabeled apo GLRX5 mixture chemically reconstituted with [2Fe-2S], or of 15N-labeled apo-GLRX5 with that of a 1:1 15N-labeled apo-GLRX5-unlabeled BOLA1 (or BOLA3) mixture chemically reconstituted with [2Fe-2S], in degassed buffer N containing 10% (v/v) D2O) at 298K. NMR data were analyzed with CARA program.

NMR experiments

All NMR experiments required for resonance assignment and structure calculations of BOLA1 (PDB ID 5LCI; BMRB 34013) and BOLA3 (PDB ID 2NCL; BMRB 26031) were recorded on Bruker AVANCE 500, 700 and 900 MHz spectrometers on 0.5–1 mM 13C,15N-labeled or 15N-labeled BOLA1 and BOLA3 samples in 50 mM phosphate buffer, pH 7.0, 5 mM DTT, containing 10% (v/v) D2O. All NMR spectra were collected at 298 K, processed using the standard BRUKER software (Topspin) and analyzed through the CARA program. The 1 H, 13C and 15N resonance assignment of BOLAs were performed following a standard triple-resonance and TOCSY-based approach. Secondary structure analysis was performed by TALOS+. Structure calculations of BOLA3 and BOLA1 were performed with the software package UNIO (ATNOS/CANDID/CYANA). The 20 conformers with the lowest residual target function values were subjected to restrained energy minimization in explicit water with the program AMBER 12.0. (D.A. Case et al. University of California, San Francisco). The quality of the structures was evaluated using the Protein Structure Validation Software suite (PSVS) and the iCING program. 15N heteronuclear relaxation experiments on 15N-labeled samples of BOLA1 and BOLA3 were recorded on Bruker AVANCE 500 MHz spectrometer at 298 K to measure 15N backbone longitudinal (R1) and transverse (R2) relaxation rates, as well as the heteronuclear 15N[1H] NOEs. The rotational correlation time values were estimated from the R2/R1 ratio using the program QUADRATIC_DIFFUSION. The relaxation data for those NHs having an exchange contribution to the R2 value or exhibiting large-amplitude fast internal motions were excluded from the analysis. Theoretical estimates of the rotational correlation time under the chosen experimental conditions of magnetic field and temperature were obtained using HYDRONMR program following a standard procedure (Figure 6—figure supplement 3).

The interaction between apo-GLRX5 and BOLA proteins was investigated by 1H-15N HSQC NMR spectra, titrating 15N-labeled apo-GLRX5 with unlabeled BOLA1 or BOLA3, and 15N labeled BOLA1 or BOLA3 with unlabeled apo-GLRX5 in degassed buffer N containing 10% (v/v) D2O) at 298K. Spectral changes were monitored upon the addition of increasing amounts of the unlabeled partner. Protein interaction between [2Fe-2S] holo-GLRX5 and BOLA proteins was investigated comparing 1H-15N HSQC NMR spectra of 15N-labeled BOLA1 (or BOLA3) with that of a 1:1 15N-labeled BOLA1 (or BOLA3)-unlabeled apo GLRX5 mixture chemically reconstituted with [2Fe-2S], or of 15N-labeled apo-GLRX5 with that of a 1:1 15N-labeled apo-GLRX5-unlabeled BOLA1 (or BOLA3) mixture chemically reconstituted with [2Fe-2S], in degassed buffer N containing 10% (v/v) D2O) at 298K. NMR data were analyzed with CARA program.

Miscellaneous methodsThe following published methods were used: manipulation of DNA and PCR (Sambrook and Russell, 2001); transformation of yeast cells (Gietz and Woods, 2002); isolation of yeast mitochondria and post-mitochondrial supernatant (Diekert et al., 2001); immunostaining (Harlow and Lane, 1998); determination of enzyme activities and of promoter strength using FET3-GFP or FIT3-GFP reporter plasmids (Molik et al., 2007); in vivo labeling of yeast cells with 55FeCl (ICN) and measurement of 55Fe-incorporation into Fe/S proteins by immunoprecipitation and scintillation counting (Molik et al., 2007; Pierik et al., 2009). Antibodies were raised in rabbits against recombinant purified proteins. Antibodies against c-Myc or HA were obtained from Santa-Cruz, protein A sepharose from GE Healthcare.

Miscellaneous methods

The following published methods were used: manipulation of DNA and PCR (Sambrook and Russell, 2001); transformation of yeast cells (Gietz and Woods, 2002); isolation of yeast mitochondria and post-mitochondrial supernatant (Diekert et al., 2001); immunostaining (Harlow and Lane, 1998); determination of enzyme activities and of promoter strength using FET3-GFP or FIT3-GFP reporter plasmids (Molik et al., 2007); in vivo labeling of yeast cells with 55FeCl (ICN) and measurement of 55Fe-incorporation into Fe/S proteins by immunoprecipitation and scintillation counting (Molik et al., 2007; Pierik et al., 2009). Antibodies were raised in rabbits against recombinant purified proteins. Antibodies against c-Myc or HA were obtained from Santa-Cruz, protein A sepharose from GE Healthcare.
