---
pmid: '20890303'
title: Int6 regulates both proteasomal degradation and translation initiation and
  is critical for proper formation of acini by human mammary epithelium.
authors:
- Suo J
- Snider SJ
- Mills GB
- Creighton CJ
- Chen AC
- Schiff R
- Lloyd RE
- Chang EC
journal: Oncogene
year: '2011'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC3017639
doi: 10.1038/onc.2010.445
---

# Int6 regulates both proteasomal degradation and translation initiation and is critical for proper formation of acini by human mammary epithelium.
**Authors:** Suo J, Snider SJ, Mills GB, Creighton CJ, Chen AC, Schiff R, Lloyd RE, Chang EC
**Journal:** Oncogene (2011)
**DOI:** [10.1038/onc.2010.445](https://doi.org/10.1038/onc.2010.445)
**PMC:** [PMC3017639](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3017639/)

## Abstract

1. Oncogene. 2011 Feb 10;30(6):724-36. doi: 10.1038/onc.2010.445. Epub 2010 Oct
4.

Int6 regulates both proteasomal degradation and translation initiation and is 
critical for proper formation of acini by human mammary epithelium.

Suo J(1), Snider SJ, Mills GB, Creighton CJ, Chen AC, Schiff R, Lloyd RE, Chang 
EC.

Author information:
(1)Department of Molecular and Cellular Biology, Baylor College of Medicine, 
Houston, TX 77030, USA.

INT6/EIF3E has been implicated in breast tumorigenesis, but its functional 
activities remain poorly defined. We found that, repressing INT6 expression 
induced transformed properties in normal human mammary epithelium (MCF10A); in 
contrast, Int6 silencing induced apoptosis in HeLa cells. As in fission yeast, 
Int6 in human cells was required for assembly of active proteasomes. A 
reverse-phase protein array screen identified SRC3/AIB1 as one oncoprotein the 
level and stability of which increased when Int6 was silenced in MCF10A cells. 
Our data further show that Int6 binds SRC3 and its ubiquitin ligase Fbw7, thus 
perhaps mediating the interaction between SRC3-Fbw7 and proteasomes. Consistent 
with this, Int6 silencing did not increase SRC3 levels in HeLa cells, which have 
low Fbw7 levels. It is surprising that, however, polyubiquitylated proteins do 
not accumulate or may even decrease in Int6-silenced cells that contain 
defective proteasomes. Considering that decreased ubiquitin might explain this 
observation and that Int6 might control ubiquitin levels in its role as a 
subunit of eIF3 (eukaryote translation initiation factor 3), we found that 
silencing Int6 reduced monoubiquitin protein levels, which correlated with a 
shift of ubiquitin mRNAs from larger polysomes to non-translating ribosomes. In 
contrast, levels of many housekeeping proteins did not change. This apparent 
reduction in the translation of ubiquitin genes correlated with a modest 
reduction in protein synthesis rate and formation of large polysomes. To further 
determine whether Int6 can selectively control translation, we analyzed 
translation of different 5'-untranslated region reporters and found that indeed, 
loss of Int6 had differential effects on these reporters. Together the data 
suggest that Int6 depletion blocks ubiquitin-dependent proteolysis by decreasing 
both ubiquitin levels and the assembly of functional proteasome machinery, 
leading to accumulation of oncoproteins, such as SRC3 that can transform mammary 
epithelium. Our data also raise the possibility that Int6 can further fine-tune 
protein levels by selectively controlling translation of specific mRNAs.

DOI: 10.1038/onc.2010.445
PMCID: PMC3017639
PMID: 20890303 [Indexed for MEDLINE]

Conflict of interest statement: Conflicts of interest The authors declared no 
financial conflict of interest.

## Full Text

IntroductionThe mammalian INT6 gene was discovered in a genetic screen using the mouse mammary tumor virus (MMTV) to isolate INT genes that were critical for mediating breast tumorigenesis (Marchetti et al., 1995). MMTV insertions into the INT6 gene cause expression of C-terminally truncated Int6 proteins (Int6ΔC). Expression of Int6ΔC in mouse mammary glands induces hyperplasia and tumor formation (Mack et al., 2007). Furthermore, Int6ΔC can transform human cells in tissue culture, and these transformed cells produce tumors when injected into immunodeficient mice (Mack et al., 2007; Mayeur and Hershey, 2002; Rasmussen et al., 2001). In human breast tumors Int6 levels appear to be deregulated. For example, several studies have reported that Int6 levels are down-regulated in both the tumors as well as the in the stroma surrounding the tumor (Finak et al., 2008; Marchetti et al., 2001; Umar et al., 2009; van’t Veer et al., 2002). In one recent study, Int6 levels have also been shown to be higher in poorly differentiated higher-grade breast tumors (Grzmil et al., 2010). Despite the connection of Int6 with breast cancer, the molecular functions of Int6 in normal human mammary epithelium and the role of Int6 in the development of human breast tumors remain largely unclear.Int6 protein contains a PCI domain located at the C-terminus. This domain is frequently found in components of the 26S proteasome, COP9-signalosome, and translation initiation factors (Aravind and Ponting, 1998; Hofmann and Bucher, 1998). The 26S proteasome functions to degrade proteins that are marked by polyubiquitylation. The COP9-signalsome also influences degradation by the proteasome by regulating the SCF class of ubiquitin ligases (also known as E3). Translation initiation in eukaryotes is catalyzed by interactions of eukaryotic translation initiation factor 3 (eIF3) with other initiation factors and the 40S ribosome subunit to assemble complete ribosomes on mRNAs, creating polysomes. The presence of PCI domain proteins in these three protein assemblies supports the idea that their PCI domains can interact with one another via a common mechanism. In keeping with this, Int6 binds both proteasomes and eIF3 subunits, and in part because of the latter, Int6 is also known as eIF3e, one of the thirteen subunits of the eIF3 complex (Asano et al., 1997). The MMTV-disrupted Int6 proteins (Int6ΔC) lack the PCI domain, suggesting that loss of the PCI domain disrupts Int6 functions.From the fission yeast Schizosaccharomyces pombe, we isolated an Int6 ortholog called Yin6 (yeast int6), in a study of the Ras-Cdc42 signaling pathway (Yen and Chang, 2000). The null mutant lacking yin6 (yin6Δ) is viable but displays phenotypes that are similar to those of proteasome mutants. Guided by a yeast two-hybrid screen, we determined that among all the PCI proteasome subunits, Yin6 appears to selectively bind Rpn5, and this finding led to the discovery that proteasomes are improperly assembled in yin6Δ cells (Yen and Chang, 2000; Yen et al., 2003). Since expression of full-length human Int6 (but not Int6ΔC) rescues phenotypes of yin6Δ cells (Yen and Chang, 2000; Yen et al., 2003), we propose that regulation of proteasomes is an evolutionarily conserved function of Int6. This idea has not been rigorously tested in human cells.Like Int6 in human cells, S. pombe Yin6 also copurifies with eIF3 components (Bandyopadhyay et al., 2000; Crane et al., 2000; Sha et al., 2009). Despite these findings, a clear role of Int6 in translation initiation remains elusive. For example, budding yeast does not have an Int6 ortholog. In S. pombe, when yin6Δ cells are grown in rich medium, there is only a slight decrease in polysomes (Bandyopadhyay et al., 2000); however, this defect is somewhat more significant when these cells are grown in minimal medium (Yen and Chang, 2000). Intriguingly, when S. pombe eIF3 interactomes were analyzed by mass spectroscopy (MS), at least two eIF3 complexes were found, but only one of them seems to contain Yin6 (Sha et al., 2009; Zhou et al., 2005). Moreover, while these two eIF3 complexes contain an overlapping set of mRNAs, the Yin6-containing eIF3 is enriched with approximately 200 mRNAs (Zhou et al., 2005). These data support the possibility that Yin6 may play a regulatory role in translation by controlling translation of a specific set of mRNAs. To define the role of Int6 in translation in human cells, Masutani et al (2007) expressed and purified human eIF3 subunits from Sf9 cells and then sought the minimum number of eIF3 subunits required using toeprint assays (Masutani et al., 2007). They determined that Int6/eIF3e is one of the five subunits needed for function in this in vitro system. Whether and how Int6 regulates translation initiation in intact human cells remains largely unknown.In agreement with the observation that Int6 is frequently down-regulated in breast cancer, we report here that knocking down INT6 expression transforms normal MCF10A human mammary epithelial cells. Furthermore, as in S. pombe cells, Int6 binds and regulates proteasome assembly in human cells. Moreover, we demonstrate that degradation of the SRC3/AIB1 oncoprotein (Xu et al., 2009) is regulated by Int6. Although Int6 regulates proteasomes, polyubiquitylated protein levels were decreased in Int6-depleted cells. We present evidence suggesting that loss of Int6 may selectively inhibit translation of specific mRNAs, such as ubiquitin mRNAs. These data suggest that Int6 is critical for maintaining proper levels of regulatory proteins by its dual regulation of proteasome-dependent degradation and translation initiation. Thus deregulation of critical proteins may be one mechanism by which normal morphogenesis of mammary epithelial cells can be disrupted by altering Int6 function.

Introduction

The mammalian INT6 gene was discovered in a genetic screen using the mouse mammary tumor virus (MMTV) to isolate INT genes that were critical for mediating breast tumorigenesis (Marchetti et al., 1995). MMTV insertions into the INT6 gene cause expression of C-terminally truncated Int6 proteins (Int6ΔC). Expression of Int6ΔC in mouse mammary glands induces hyperplasia and tumor formation (Mack et al., 2007). Furthermore, Int6ΔC can transform human cells in tissue culture, and these transformed cells produce tumors when injected into immunodeficient mice (Mack et al., 2007; Mayeur and Hershey, 2002; Rasmussen et al., 2001). In human breast tumors Int6 levels appear to be deregulated. For example, several studies have reported that Int6 levels are down-regulated in both the tumors as well as the in the stroma surrounding the tumor (Finak et al., 2008; Marchetti et al., 2001; Umar et al., 2009; van’t Veer et al., 2002). In one recent study, Int6 levels have also been shown to be higher in poorly differentiated higher-grade breast tumors (Grzmil et al., 2010). Despite the connection of Int6 with breast cancer, the molecular functions of Int6 in normal human mammary epithelium and the role of Int6 in the development of human breast tumors remain largely unclear.

Int6 protein contains a PCI domain located at the C-terminus. This domain is frequently found in components of the 26S proteasome, COP9-signalosome, and translation initiation factors (Aravind and Ponting, 1998; Hofmann and Bucher, 1998). The 26S proteasome functions to degrade proteins that are marked by polyubiquitylation. The COP9-signalsome also influences degradation by the proteasome by regulating the SCF class of ubiquitin ligases (also known as E3). Translation initiation in eukaryotes is catalyzed by interactions of eukaryotic translation initiation factor 3 (eIF3) with other initiation factors and the 40S ribosome subunit to assemble complete ribosomes on mRNAs, creating polysomes. The presence of PCI domain proteins in these three protein assemblies supports the idea that their PCI domains can interact with one another via a common mechanism. In keeping with this, Int6 binds both proteasomes and eIF3 subunits, and in part because of the latter, Int6 is also known as eIF3e, one of the thirteen subunits of the eIF3 complex (Asano et al., 1997). The MMTV-disrupted Int6 proteins (Int6ΔC) lack the PCI domain, suggesting that loss of the PCI domain disrupts Int6 functions.

From the fission yeast Schizosaccharomyces pombe, we isolated an Int6 ortholog called Yin6 (yeast int6), in a study of the Ras-Cdc42 signaling pathway (Yen and Chang, 2000). The null mutant lacking yin6 (yin6Δ) is viable but displays phenotypes that are similar to those of proteasome mutants. Guided by a yeast two-hybrid screen, we determined that among all the PCI proteasome subunits, Yin6 appears to selectively bind Rpn5, and this finding led to the discovery that proteasomes are improperly assembled in yin6Δ cells (Yen and Chang, 2000; Yen et al., 2003). Since expression of full-length human Int6 (but not Int6ΔC) rescues phenotypes of yin6Δ cells (Yen and Chang, 2000; Yen et al., 2003), we propose that regulation of proteasomes is an evolutionarily conserved function of Int6. This idea has not been rigorously tested in human cells.

Like Int6 in human cells, S. pombe Yin6 also copurifies with eIF3 components (Bandyopadhyay et al., 2000; Crane et al., 2000; Sha et al., 2009). Despite these findings, a clear role of Int6 in translation initiation remains elusive. For example, budding yeast does not have an Int6 ortholog. In S. pombe, when yin6Δ cells are grown in rich medium, there is only a slight decrease in polysomes (Bandyopadhyay et al., 2000); however, this defect is somewhat more significant when these cells are grown in minimal medium (Yen and Chang, 2000). Intriguingly, when S. pombe eIF3 interactomes were analyzed by mass spectroscopy (MS), at least two eIF3 complexes were found, but only one of them seems to contain Yin6 (Sha et al., 2009; Zhou et al., 2005). Moreover, while these two eIF3 complexes contain an overlapping set of mRNAs, the Yin6-containing eIF3 is enriched with approximately 200 mRNAs (Zhou et al., 2005). These data support the possibility that Yin6 may play a regulatory role in translation by controlling translation of a specific set of mRNAs. To define the role of Int6 in translation in human cells, Masutani et al (2007) expressed and purified human eIF3 subunits from Sf9 cells and then sought the minimum number of eIF3 subunits required using toeprint assays (Masutani et al., 2007). They determined that Int6/eIF3e is one of the five subunits needed for function in this in vitro system. Whether and how Int6 regulates translation initiation in intact human cells remains largely unknown.

In agreement with the observation that Int6 is frequently down-regulated in breast cancer, we report here that knocking down INT6 expression transforms normal MCF10A human mammary epithelial cells. Furthermore, as in S. pombe cells, Int6 binds and regulates proteasome assembly in human cells. Moreover, we demonstrate that degradation of the SRC3/AIB1 oncoprotein (Xu et al., 2009) is regulated by Int6. Although Int6 regulates proteasomes, polyubiquitylated protein levels were decreased in Int6-depleted cells. We present evidence suggesting that loss of Int6 may selectively inhibit translation of specific mRNAs, such as ubiquitin mRNAs. These data suggest that Int6 is critical for maintaining proper levels of regulatory proteins by its dual regulation of proteasome-dependent degradation and translation initiation. Thus deregulation of critical proteins may be one mechanism by which normal morphogenesis of mammary epithelial cells can be disrupted by altering Int6 function.

Materials and methodsCells and culture conditionsHeLa, MCF7, MCF10A, and MCF10AT cells were obtained from the American Type Culture Collection. HeLa and MCF7 cells were cultured in DMEM (GIBCO) supplemented with 10% FBS, 100 IU/ml penicillin, 0.1 mg/ml streptomycin, and 2 mM glutamine (GIBCO). The media and methods of culturing MCF10A and MCF10AT cells, including the 3-D culture in Matrigel (BD Biosciences), were as described (Debnath et al., 2003). Cell proliferation was measured by staining trypsinized cells with trypan blue to mark dead cells, and the viable cells were then counted microscopically. Synchronization of cells in the G1-S phase was performed using 2.5 mM hydroxyurea for 17 hours. Cells were examined by BrdU and DAPI staining to confirm that approximately 60% of the cells were in S phase while the remaining cells were in G1 with relaxed chromosomes. Mitotic cells (with condensed chromosomes) were barely detectable after such treatment. Synchronization of cells in mitosis for the half-life experiment was performed using thymidine and nocodazole block. Briefly, cells were first treated with 2mM thymidine for 18 hour, and then were released from the block. Three hours later, the cells were blocked again with 100 ng/ml nocodazole for 16 hours. By microscopy, we estimate that approximately 50% cells were effectively synchronized.Gene silencingThe siRNAs #1 and #2 used to silence INT6 expressions represent siRNA I6.3 and I6.1 previously described (Morris and Jalinot, 2005). siRNA #3 designed to target the INT6 sequence GCAGCC ATGGAAGACCTTACA was synthesized by Invitrogen. Rpn5 siRNA was purchased from Dharmacon (Cat# M-011368-00). Knock-down of INT6 can persist for up to 120 hrs after transfection (not presented). The negative (scrambled) siRNA control and lamin A/C siRNA were purchased from Qiagen and Dharmacon. For routine siRNA transfection, subconfluent proliferating cells in 24-well plates were transfected with 1 μl each siRNA (from a 20 μM stock) using Lipofectamine 2000 (Invitrogen). To build an shRNA vector to knock down INT6 expression, a DNA sequence was designed based on the sequence of siRNA #1 (underlined) plus the cloning site for the lentiviral vector pLL3.7 (Rubinson et al., 2003): 5′-T GTC TTT CCG CTT CTT GAA T TTCAAGAGA A TTC AAG AAG CGG AAA GAC TTTTTTC-3′. The resulting vector is named pSH-INT6 with lentivirus production as described (Rubinson et al., 2003).Plasmid construction and transfectionTo tag human Rpn5 with GFP, the human RPN5 cDNA was modified by PCR to include EcoRI sites at both ends before being cloned into pEGFP-C1 (Clontech) to create pEGFP-RPN5 with orientation confirmed by sequencing. For routine plasmid transfection, subconfluent cells in a 24-well plate were transfected with 200 ng of plasmid using Lipofectamine 2000 (Invitrogen).Anchorage-independent growth in soft agarCells were suspended in growth medium containing 0.35% agar, and plated at a density of 20,000 cells/well in a 6-well plate containing 0.7% agar in growth medium. The cells were then incubated at 37 °C in 5% CO2 for 30 days. Colonies were stained with 0.005% crystal violet and counted microscopically.Antibody preparationOur Int6 antibody was produced (Proteintech Group) using rabbits injected with a peptide containing the last 20 amino acids in the human Int6 protein, as previously described (Diella et al., 1997). Human Rpn5 antibody was made similarly against the first 20 amino acids of the Rpn5 protein. Pre-immune sera were collected to provide control sera.ImmunoprecipitationCells grown on 100-mm dishes were washed twice with phosphate-buffered saline (PBS) before being lysed in lysis buffer (50 mM Tris, pH 8.0, 150 mM NaCl, 1% NP-40). The whole cell lysate was centrifuged at 14,000 rpm for 20 min at 4°C, the supernatant collected and then incubated overnight at 4°C with 10 μl of antibody or pre-immune serum together with Protein A-agarose beads on an orbital shaker. The beads were washed 4 times with lysis buffer before boiling 10 min in 2× SDS sample buffer prior to Western blotting.Immunoblotting and protein quantificationCells were washed with PBS and lysed at 0°C for 30 min in RIPA buffer (50 μl/well) in a 24-well plate. Protein content was measured by the BCA™ protein assay (Thermo Scientific). Proteins were analyzed by SDS-PAGE gels after boiling for 5 min in SDS-PAGE reducing sample buffer. Proteins were transferred to nitrocellulose membranes (Bio-Rad) and blocked for 30 min in 5% non-fat milk. Int6 and Rpn5 polyclonal antibodies were used at a dilution of 1:500. Antibodies against proteasome 20S core subunits (MCP231; 1:1,000), GFP (1:1,000), Rpt4 (1:1,000), hMoe1/eIF3d (1:1,000), lamin A/C (1:1,000), Fbw7 (1:1,000), and SRC3 (1:1,000) were from Biomol, Fitzgerald, Calbiochem, Protein Tech Group, Santa Cruz, Invitrogen, and BD Biosciences, respectively. To detect both mono- and poly-ubiquitin, an antibody from Biomol was used (FK2, 1:1,000). Antibodies against phospho-histone H3 (1:1,000), total histone H3 (1:1,000), cleaved Caspase 3 (1:1,000), and actin (1:10,000) were from Cell Signaling. The anti-tubulin antibody (1:1,000) was a gift from Keith Gull (Woods et al., 1989). The secondary antibodies were the fluorescently conjugated antibodies IRDye™ 680 and IRDye™ 800 from LI-COR Biosciences (1:20,000). Proteins on the membrane were visualized and quantified by the Odyssey infrared imaging system (LI-COR Biosciences).Immunostaining and microscopyCells were seeded on poly-L-lysine coated coverslips (BD Biosciences) and later fixed in 3.7% paraformaldehyde, followed by permeabilization in 0.5% Triton X-100. Anti-phospho-histone H3 antibody (Cell Signaling) and anti-tubulin antibody (Cell Signaling) were used at a dilution of 1:200, and the incubation was 2-hr at room temperature. The secondary antibodies were Alexa Fluro 488 or Alexa Fluro 594-conjugated antibodies (Invitrogen, 1:250, 1 hr at room temperature). For TUNEL staining, permeabilized cells were incubated with the TUNEL mixture (Roche) for 60 minutes at 37 °C. All samples were finally mounted in Vectashield with DAPI (Vector). Images were captured using an Olympus IX70 microscope via a 60×/1.4 oil objective. Deconvolution (Constrained Iterative) was performed using Slidebook (Intelligent Imaging Innovations) on a stack of 12 images collected at 0.5 μm intervals. Microscopy analysis of acini was performed as described (Debnath et al., 2003). Briefly, primary antibodies for Ki-67 (Dako) and cleaved caspase 3 (Cell Signaling) were used at dilutions of 1:200, and the samples were then incubated with fluorescently-conjugated secondary antibodies as described above. Nuclear staining was performed by 5 μM TOPRO-3 (Molecular Probes) in PBS for 15 minutes, followed by mounting with Vectashield (Vector). Confocal analyses were performed with a Nikon Eclipse E1000 confocal microscopy system.Semiquantitative RT-PCRTotal RNA was extracted using the RNeasy Mini Kit (Qiagen), treated with DNase I, and subjected to reverse transcription (SuperScript™ First-Strand Synthesis System, Invitrogen). PCR conditions were: 1 min at 94 °C, 1 min at 58 °C, and 1 min at 72°C. The cycle number was adjusted to allow detection to fall within a linear range of product amplification. All PCR reactions ended with a 7-min incubation at 72 °C. The forward and reverse primers for the ubiquitin and actin genes were: ACTGGTAAGACCATCACCCTCGAGG and GAGACGGAGCACCAGG-TGCAGGGT, and GTGGGGCGCCCCAGGCACCA and CTCCTTAATGTCACGCA-CGATTTC. For firefly luciferase, the forward and reverse primers were TCTGATTACACCCGAGGGGG and CGGTAAGACCTTTCGGTACTTCGTC. PCR products were analyzed by electrophoresis through 1.5% agarose gels containing 0.1 mg/ml of ethidium bromide, and finally photographed and analyzed using the Molecular Imager Gel Documentation System (Bio-Rad).Proteasome pull-down and activity assayCells were washed with PBS and lysed on ice for 30 min in 50 mM HEPES (pH7.5), 5 mM EDTA, 150 mM NaCl, 1% Triton X-100, 2 mM ATP, and protease inhibitors (Roche). The same amounts of total protein were mixed with beads from the Proteasome Isolation Kit (Calbiochem). After pull-down, the same amount of beads were analyzed by Western blots or by a proteasome activity assay. For the latter, the fluorogenic substrate Suc-LLVY-AMC (Biomol) was added to the pulled-down proteasomes and the mixture was then incubated for 30 min at 37 °C. Fluorescence (Ex. 360 nm, Em. 450 nm) was measured with an ISS PC1 fluorometer. For native PAGE, cell lysates were loaded on 3–8% Tris-Acetate gel (Bio-Rad), and electrophoresis was carried out first at 50V overnight, and then at 100V for 2 more hours. The resulting gels were incubated in LLVY-buffer (50mM Tris-HCl pH 8, 10mM MgCl2, 1mM ATP, 1mM DTT, 50μM Suc-LLVY-AMC) for 4 hrs at 37°C before being visualized by the Molecular Imager Gel Documentation System (Bio-Rad).Statistical analysesAll error bars are standard error of the mean, except the one from Figure 4b, which are standard deviation. All p-values were calculated for the un-paired Student’s t tests.RPPA screenAfter siRNA transfection, asynchronized cells were either analyzed directly or separated into interphase and mitotic cells by light trypsinization (0.005% in PBS) at room temperature for 5 minutes with gentle shaking and rocking to allow mitotic cells to detach. The presence of mitotic cells was determined by light microscopy based on cell and nuclear morphology. This process was repeated until at least 80% of mitotic cells were collected. Trypsin at 0.05% was added to collect the rest of the cells, which were considered to be mostly in interphase. Collected cells were gently rinsed with PBS, resuspended in lysis buffer (50 mM Hepes, pH 7.4, 150 mM NaCl, 1 mM EGTA, 10 mM sodium pyrophosphate, 100 mM NaF, 10% glycerol, 1.5 mM MgCl, 1% Triton X-100, 1 mM Na3VO4, and protease inhibitor from Roche), incubated on ice for 20 minutes, and finally centrifuged at 14,000 rpm for 10 minutes at 4 °C. Supernatants were then collected and protein concentrations were determined by the BCA. The efficiency of collecting mitotic cells was further confirmed by Western blots using anti-phospho-histone H3 antibody. Cell lysates with equal amount of proteins were mixed with 4× SDS sample buffer and boiled. Sample printing and probing was done as described (Tibes et al., 2006). The median protein concentration of each set of slides was used as the control level. The cut off for this screen was a 1.5-fold increase of the apparent protein levels over the control. While the screen identified several hits, SRC3 is the only one that validated by Western Blot.Translation analysesCells were pulse-labeled with 30 μCi/ml [35S]-Met/Cys ([35S]-Tran, MP Biomedicals) for 60 min as previously described (Byrd et al., 2005) before preparation of cell lysates. Labeled proteins in lysates were analyzed by SDS-PAGE autoradiography or precipitation with 10% TCA on glass fiber filters and scintillation counting. For polysome analysis, cells were treated with cycloheximide (150 μg/ml in normal growth media) for 10 min immediately before collection for lysis. Cell were lysed in 50 mM Tris-HCl (pH8.0), 100 mM NaCl, 5 mM MgCl2, 0.5% NP-40, 20 mM DTT, protease inhibitor (Roche), 0.1 mg/ml cycloheximide, and 0.5 mg/ml heparin-Na by Dounce homogenization, and post nuclear lysates were collected by sedimentation at 5000 × g. The cell lysates were then subjected to 10–50% continuous sucrose gradient centrifugation at 43,000 rpm in a SW55 rotor at 4 °C for 60 min and the eluted fractions were continuously monitored by UV spectroscopy with an ISCO gradient fractionator as described (Rivera and Lloyd, 2008). For analysis of distribution of ubiquitin mRNAs in polysomes, pooled polysome sucrose gradient fractions were combined with 50 μg carrier yeast tRNA then RNA precipitated with three volumes ethanol overnight. Pelleted RNA was extracted with Trizol reagent (Invitrogen), precipitated in ethanol, and washed in 75% ethanol before RT-PCR analysis. For luciferase reporter assays, HeLa Tet-OFF cells (4 × 104) in 12-well plates in duplicate were transfected with either pTRE2-luciferase, pTRE2-hpBCL2-IRES-FL, or pTRE2-CVB3-IRES-FL (400 ng) together with Int6 siRNA#1 or control siRNA (2 μl of a 20 μM stock) in Lipofectamine 2000. At 24 or 48 hrs, reporter mRNA expression was induced for 6 hrs with Tet (−) medium and then cells were harvested for luciferase activity assay (Promega). Duplicate samples were subjected to semi-quantitative RT-PCR to assess levels of Luc reporter RNA to normalize translation results.

Materials and methods

Cells and culture conditionsHeLa, MCF7, MCF10A, and MCF10AT cells were obtained from the American Type Culture Collection. HeLa and MCF7 cells were cultured in DMEM (GIBCO) supplemented with 10% FBS, 100 IU/ml penicillin, 0.1 mg/ml streptomycin, and 2 mM glutamine (GIBCO). The media and methods of culturing MCF10A and MCF10AT cells, including the 3-D culture in Matrigel (BD Biosciences), were as described (Debnath et al., 2003). Cell proliferation was measured by staining trypsinized cells with trypan blue to mark dead cells, and the viable cells were then counted microscopically. Synchronization of cells in the G1-S phase was performed using 2.5 mM hydroxyurea for 17 hours. Cells were examined by BrdU and DAPI staining to confirm that approximately 60% of the cells were in S phase while the remaining cells were in G1 with relaxed chromosomes. Mitotic cells (with condensed chromosomes) were barely detectable after such treatment. Synchronization of cells in mitosis for the half-life experiment was performed using thymidine and nocodazole block. Briefly, cells were first treated with 2mM thymidine for 18 hour, and then were released from the block. Three hours later, the cells were blocked again with 100 ng/ml nocodazole for 16 hours. By microscopy, we estimate that approximately 50% cells were effectively synchronized.

Cells and culture conditions

HeLa, MCF7, MCF10A, and MCF10AT cells were obtained from the American Type Culture Collection. HeLa and MCF7 cells were cultured in DMEM (GIBCO) supplemented with 10% FBS, 100 IU/ml penicillin, 0.1 mg/ml streptomycin, and 2 mM glutamine (GIBCO). The media and methods of culturing MCF10A and MCF10AT cells, including the 3-D culture in Matrigel (BD Biosciences), were as described (Debnath et al., 2003). Cell proliferation was measured by staining trypsinized cells with trypan blue to mark dead cells, and the viable cells were then counted microscopically. Synchronization of cells in the G1-S phase was performed using 2.5 mM hydroxyurea for 17 hours. Cells were examined by BrdU and DAPI staining to confirm that approximately 60% of the cells were in S phase while the remaining cells were in G1 with relaxed chromosomes. Mitotic cells (with condensed chromosomes) were barely detectable after such treatment. Synchronization of cells in mitosis for the half-life experiment was performed using thymidine and nocodazole block. Briefly, cells were first treated with 2mM thymidine for 18 hour, and then were released from the block. Three hours later, the cells were blocked again with 100 ng/ml nocodazole for 16 hours. By microscopy, we estimate that approximately 50% cells were effectively synchronized.

Gene silencingThe siRNAs #1 and #2 used to silence INT6 expressions represent siRNA I6.3 and I6.1 previously described (Morris and Jalinot, 2005). siRNA #3 designed to target the INT6 sequence GCAGCC ATGGAAGACCTTACA was synthesized by Invitrogen. Rpn5 siRNA was purchased from Dharmacon (Cat# M-011368-00). Knock-down of INT6 can persist for up to 120 hrs after transfection (not presented). The negative (scrambled) siRNA control and lamin A/C siRNA were purchased from Qiagen and Dharmacon. For routine siRNA transfection, subconfluent proliferating cells in 24-well plates were transfected with 1 μl each siRNA (from a 20 μM stock) using Lipofectamine 2000 (Invitrogen). To build an shRNA vector to knock down INT6 expression, a DNA sequence was designed based on the sequence of siRNA #1 (underlined) plus the cloning site for the lentiviral vector pLL3.7 (Rubinson et al., 2003): 5′-T GTC TTT CCG CTT CTT GAA T TTCAAGAGA A TTC AAG AAG CGG AAA GAC TTTTTTC-3′. The resulting vector is named pSH-INT6 with lentivirus production as described (Rubinson et al., 2003).

Gene silencing

The siRNAs #1 and #2 used to silence INT6 expressions represent siRNA I6.3 and I6.1 previously described (Morris and Jalinot, 2005). siRNA #3 designed to target the INT6 sequence GCAGCC ATGGAAGACCTTACA was synthesized by Invitrogen. Rpn5 siRNA was purchased from Dharmacon (Cat# M-011368-00). Knock-down of INT6 can persist for up to 120 hrs after transfection (not presented). The negative (scrambled) siRNA control and lamin A/C siRNA were purchased from Qiagen and Dharmacon. For routine siRNA transfection, subconfluent proliferating cells in 24-well plates were transfected with 1 μl each siRNA (from a 20 μM stock) using Lipofectamine 2000 (Invitrogen). To build an shRNA vector to knock down INT6 expression, a DNA sequence was designed based on the sequence of siRNA #1 (underlined) plus the cloning site for the lentiviral vector pLL3.7 (Rubinson et al., 2003): 5′-T GTC TTT CCG CTT CTT GAA T TTCAAGAGA A TTC AAG AAG CGG AAA GAC TTTTTTC-3′. The resulting vector is named pSH-INT6 with lentivirus production as described (Rubinson et al., 2003).

Plasmid construction and transfectionTo tag human Rpn5 with GFP, the human RPN5 cDNA was modified by PCR to include EcoRI sites at both ends before being cloned into pEGFP-C1 (Clontech) to create pEGFP-RPN5 with orientation confirmed by sequencing. For routine plasmid transfection, subconfluent cells in a 24-well plate were transfected with 200 ng of plasmid using Lipofectamine 2000 (Invitrogen).

Plasmid construction and transfection

To tag human Rpn5 with GFP, the human RPN5 cDNA was modified by PCR to include EcoRI sites at both ends before being cloned into pEGFP-C1 (Clontech) to create pEGFP-RPN5 with orientation confirmed by sequencing. For routine plasmid transfection, subconfluent cells in a 24-well plate were transfected with 200 ng of plasmid using Lipofectamine 2000 (Invitrogen).

Anchorage-independent growth in soft agarCells were suspended in growth medium containing 0.35% agar, and plated at a density of 20,000 cells/well in a 6-well plate containing 0.7% agar in growth medium. The cells were then incubated at 37 °C in 5% CO2 for 30 days. Colonies were stained with 0.005% crystal violet and counted microscopically.

Anchorage-independent growth in soft agar

Cells were suspended in growth medium containing 0.35% agar, and plated at a density of 20,000 cells/well in a 6-well plate containing 0.7% agar in growth medium. The cells were then incubated at 37 °C in 5% CO2 for 30 days. Colonies were stained with 0.005% crystal violet and counted microscopically.

Antibody preparationOur Int6 antibody was produced (Proteintech Group) using rabbits injected with a peptide containing the last 20 amino acids in the human Int6 protein, as previously described (Diella et al., 1997). Human Rpn5 antibody was made similarly against the first 20 amino acids of the Rpn5 protein. Pre-immune sera were collected to provide control sera.

Antibody preparation

Our Int6 antibody was produced (Proteintech Group) using rabbits injected with a peptide containing the last 20 amino acids in the human Int6 protein, as previously described (Diella et al., 1997). Human Rpn5 antibody was made similarly against the first 20 amino acids of the Rpn5 protein. Pre-immune sera were collected to provide control sera.

ImmunoprecipitationCells grown on 100-mm dishes were washed twice with phosphate-buffered saline (PBS) before being lysed in lysis buffer (50 mM Tris, pH 8.0, 150 mM NaCl, 1% NP-40). The whole cell lysate was centrifuged at 14,000 rpm for 20 min at 4°C, the supernatant collected and then incubated overnight at 4°C with 10 μl of antibody or pre-immune serum together with Protein A-agarose beads on an orbital shaker. The beads were washed 4 times with lysis buffer before boiling 10 min in 2× SDS sample buffer prior to Western blotting.

Immunoprecipitation

Cells grown on 100-mm dishes were washed twice with phosphate-buffered saline (PBS) before being lysed in lysis buffer (50 mM Tris, pH 8.0, 150 mM NaCl, 1% NP-40). The whole cell lysate was centrifuged at 14,000 rpm for 20 min at 4°C, the supernatant collected and then incubated overnight at 4°C with 10 μl of antibody or pre-immune serum together with Protein A-agarose beads on an orbital shaker. The beads were washed 4 times with lysis buffer before boiling 10 min in 2× SDS sample buffer prior to Western blotting.

Immunoblotting and protein quantificationCells were washed with PBS and lysed at 0°C for 30 min in RIPA buffer (50 μl/well) in a 24-well plate. Protein content was measured by the BCA™ protein assay (Thermo Scientific). Proteins were analyzed by SDS-PAGE gels after boiling for 5 min in SDS-PAGE reducing sample buffer. Proteins were transferred to nitrocellulose membranes (Bio-Rad) and blocked for 30 min in 5% non-fat milk. Int6 and Rpn5 polyclonal antibodies were used at a dilution of 1:500. Antibodies against proteasome 20S core subunits (MCP231; 1:1,000), GFP (1:1,000), Rpt4 (1:1,000), hMoe1/eIF3d (1:1,000), lamin A/C (1:1,000), Fbw7 (1:1,000), and SRC3 (1:1,000) were from Biomol, Fitzgerald, Calbiochem, Protein Tech Group, Santa Cruz, Invitrogen, and BD Biosciences, respectively. To detect both mono- and poly-ubiquitin, an antibody from Biomol was used (FK2, 1:1,000). Antibodies against phospho-histone H3 (1:1,000), total histone H3 (1:1,000), cleaved Caspase 3 (1:1,000), and actin (1:10,000) were from Cell Signaling. The anti-tubulin antibody (1:1,000) was a gift from Keith Gull (Woods et al., 1989). The secondary antibodies were the fluorescently conjugated antibodies IRDye™ 680 and IRDye™ 800 from LI-COR Biosciences (1:20,000). Proteins on the membrane were visualized and quantified by the Odyssey infrared imaging system (LI-COR Biosciences).

Immunoblotting and protein quantification

Cells were washed with PBS and lysed at 0°C for 30 min in RIPA buffer (50 μl/well) in a 24-well plate. Protein content was measured by the BCA™ protein assay (Thermo Scientific). Proteins were analyzed by SDS-PAGE gels after boiling for 5 min in SDS-PAGE reducing sample buffer. Proteins were transferred to nitrocellulose membranes (Bio-Rad) and blocked for 30 min in 5% non-fat milk. Int6 and Rpn5 polyclonal antibodies were used at a dilution of 1:500. Antibodies against proteasome 20S core subunits (MCP231; 1:1,000), GFP (1:1,000), Rpt4 (1:1,000), hMoe1/eIF3d (1:1,000), lamin A/C (1:1,000), Fbw7 (1:1,000), and SRC3 (1:1,000) were from Biomol, Fitzgerald, Calbiochem, Protein Tech Group, Santa Cruz, Invitrogen, and BD Biosciences, respectively. To detect both mono- and poly-ubiquitin, an antibody from Biomol was used (FK2, 1:1,000). Antibodies against phospho-histone H3 (1:1,000), total histone H3 (1:1,000), cleaved Caspase 3 (1:1,000), and actin (1:10,000) were from Cell Signaling. The anti-tubulin antibody (1:1,000) was a gift from Keith Gull (Woods et al., 1989). The secondary antibodies were the fluorescently conjugated antibodies IRDye™ 680 and IRDye™ 800 from LI-COR Biosciences (1:20,000). Proteins on the membrane were visualized and quantified by the Odyssey infrared imaging system (LI-COR Biosciences).

Immunostaining and microscopyCells were seeded on poly-L-lysine coated coverslips (BD Biosciences) and later fixed in 3.7% paraformaldehyde, followed by permeabilization in 0.5% Triton X-100. Anti-phospho-histone H3 antibody (Cell Signaling) and anti-tubulin antibody (Cell Signaling) were used at a dilution of 1:200, and the incubation was 2-hr at room temperature. The secondary antibodies were Alexa Fluro 488 or Alexa Fluro 594-conjugated antibodies (Invitrogen, 1:250, 1 hr at room temperature). For TUNEL staining, permeabilized cells were incubated with the TUNEL mixture (Roche) for 60 minutes at 37 °C. All samples were finally mounted in Vectashield with DAPI (Vector). Images were captured using an Olympus IX70 microscope via a 60×/1.4 oil objective. Deconvolution (Constrained Iterative) was performed using Slidebook (Intelligent Imaging Innovations) on a stack of 12 images collected at 0.5 μm intervals. Microscopy analysis of acini was performed as described (Debnath et al., 2003). Briefly, primary antibodies for Ki-67 (Dako) and cleaved caspase 3 (Cell Signaling) were used at dilutions of 1:200, and the samples were then incubated with fluorescently-conjugated secondary antibodies as described above. Nuclear staining was performed by 5 μM TOPRO-3 (Molecular Probes) in PBS for 15 minutes, followed by mounting with Vectashield (Vector). Confocal analyses were performed with a Nikon Eclipse E1000 confocal microscopy system.

Immunostaining and microscopy

Cells were seeded on poly-L-lysine coated coverslips (BD Biosciences) and later fixed in 3.7% paraformaldehyde, followed by permeabilization in 0.5% Triton X-100. Anti-phospho-histone H3 antibody (Cell Signaling) and anti-tubulin antibody (Cell Signaling) were used at a dilution of 1:200, and the incubation was 2-hr at room temperature. The secondary antibodies were Alexa Fluro 488 or Alexa Fluro 594-conjugated antibodies (Invitrogen, 1:250, 1 hr at room temperature). For TUNEL staining, permeabilized cells were incubated with the TUNEL mixture (Roche) for 60 minutes at 37 °C. All samples were finally mounted in Vectashield with DAPI (Vector). Images were captured using an Olympus IX70 microscope via a 60×/1.4 oil objective. Deconvolution (Constrained Iterative) was performed using Slidebook (Intelligent Imaging Innovations) on a stack of 12 images collected at 0.5 μm intervals. Microscopy analysis of acini was performed as described (Debnath et al., 2003). Briefly, primary antibodies for Ki-67 (Dako) and cleaved caspase 3 (Cell Signaling) were used at dilutions of 1:200, and the samples were then incubated with fluorescently-conjugated secondary antibodies as described above. Nuclear staining was performed by 5 μM TOPRO-3 (Molecular Probes) in PBS for 15 minutes, followed by mounting with Vectashield (Vector). Confocal analyses were performed with a Nikon Eclipse E1000 confocal microscopy system.

Semiquantitative RT-PCRTotal RNA was extracted using the RNeasy Mini Kit (Qiagen), treated with DNase I, and subjected to reverse transcription (SuperScript™ First-Strand Synthesis System, Invitrogen). PCR conditions were: 1 min at 94 °C, 1 min at 58 °C, and 1 min at 72°C. The cycle number was adjusted to allow detection to fall within a linear range of product amplification. All PCR reactions ended with a 7-min incubation at 72 °C. The forward and reverse primers for the ubiquitin and actin genes were: ACTGGTAAGACCATCACCCTCGAGG and GAGACGGAGCACCAGG-TGCAGGGT, and GTGGGGCGCCCCAGGCACCA and CTCCTTAATGTCACGCA-CGATTTC. For firefly luciferase, the forward and reverse primers were TCTGATTACACCCGAGGGGG and CGGTAAGACCTTTCGGTACTTCGTC. PCR products were analyzed by electrophoresis through 1.5% agarose gels containing 0.1 mg/ml of ethidium bromide, and finally photographed and analyzed using the Molecular Imager Gel Documentation System (Bio-Rad).

Semiquantitative RT-PCR

Total RNA was extracted using the RNeasy Mini Kit (Qiagen), treated with DNase I, and subjected to reverse transcription (SuperScript™ First-Strand Synthesis System, Invitrogen). PCR conditions were: 1 min at 94 °C, 1 min at 58 °C, and 1 min at 72°C. The cycle number was adjusted to allow detection to fall within a linear range of product amplification. All PCR reactions ended with a 7-min incubation at 72 °C. The forward and reverse primers for the ubiquitin and actin genes were: ACTGGTAAGACCATCACCCTCGAGG and GAGACGGAGCACCAGG-TGCAGGGT, and GTGGGGCGCCCCAGGCACCA and CTCCTTAATGTCACGCA-CGATTTC. For firefly luciferase, the forward and reverse primers were TCTGATTACACCCGAGGGGG and CGGTAAGACCTTTCGGTACTTCGTC. PCR products were analyzed by electrophoresis through 1.5% agarose gels containing 0.1 mg/ml of ethidium bromide, and finally photographed and analyzed using the Molecular Imager Gel Documentation System (Bio-Rad).

Proteasome pull-down and activity assayCells were washed with PBS and lysed on ice for 30 min in 50 mM HEPES (pH7.5), 5 mM EDTA, 150 mM NaCl, 1% Triton X-100, 2 mM ATP, and protease inhibitors (Roche). The same amounts of total protein were mixed with beads from the Proteasome Isolation Kit (Calbiochem). After pull-down, the same amount of beads were analyzed by Western blots or by a proteasome activity assay. For the latter, the fluorogenic substrate Suc-LLVY-AMC (Biomol) was added to the pulled-down proteasomes and the mixture was then incubated for 30 min at 37 °C. Fluorescence (Ex. 360 nm, Em. 450 nm) was measured with an ISS PC1 fluorometer. For native PAGE, cell lysates were loaded on 3–8% Tris-Acetate gel (Bio-Rad), and electrophoresis was carried out first at 50V overnight, and then at 100V for 2 more hours. The resulting gels were incubated in LLVY-buffer (50mM Tris-HCl pH 8, 10mM MgCl2, 1mM ATP, 1mM DTT, 50μM Suc-LLVY-AMC) for 4 hrs at 37°C before being visualized by the Molecular Imager Gel Documentation System (Bio-Rad).

Proteasome pull-down and activity assay

Cells were washed with PBS and lysed on ice for 30 min in 50 mM HEPES (pH7.5), 5 mM EDTA, 150 mM NaCl, 1% Triton X-100, 2 mM ATP, and protease inhibitors (Roche). The same amounts of total protein were mixed with beads from the Proteasome Isolation Kit (Calbiochem). After pull-down, the same amount of beads were analyzed by Western blots or by a proteasome activity assay. For the latter, the fluorogenic substrate Suc-LLVY-AMC (Biomol) was added to the pulled-down proteasomes and the mixture was then incubated for 30 min at 37 °C. Fluorescence (Ex. 360 nm, Em. 450 nm) was measured with an ISS PC1 fluorometer. For native PAGE, cell lysates were loaded on 3–8% Tris-Acetate gel (Bio-Rad), and electrophoresis was carried out first at 50V overnight, and then at 100V for 2 more hours. The resulting gels were incubated in LLVY-buffer (50mM Tris-HCl pH 8, 10mM MgCl2, 1mM ATP, 1mM DTT, 50μM Suc-LLVY-AMC) for 4 hrs at 37°C before being visualized by the Molecular Imager Gel Documentation System (Bio-Rad).

Statistical analysesAll error bars are standard error of the mean, except the one from Figure 4b, which are standard deviation. All p-values were calculated for the un-paired Student’s t tests.

Statistical analyses

All error bars are standard error of the mean, except the one from Figure 4b, which are standard deviation. All p-values were calculated for the un-paired Student’s t tests.

RPPA screenAfter siRNA transfection, asynchronized cells were either analyzed directly or separated into interphase and mitotic cells by light trypsinization (0.005% in PBS) at room temperature for 5 minutes with gentle shaking and rocking to allow mitotic cells to detach. The presence of mitotic cells was determined by light microscopy based on cell and nuclear morphology. This process was repeated until at least 80% of mitotic cells were collected. Trypsin at 0.05% was added to collect the rest of the cells, which were considered to be mostly in interphase. Collected cells were gently rinsed with PBS, resuspended in lysis buffer (50 mM Hepes, pH 7.4, 150 mM NaCl, 1 mM EGTA, 10 mM sodium pyrophosphate, 100 mM NaF, 10% glycerol, 1.5 mM MgCl, 1% Triton X-100, 1 mM Na3VO4, and protease inhibitor from Roche), incubated on ice for 20 minutes, and finally centrifuged at 14,000 rpm for 10 minutes at 4 °C. Supernatants were then collected and protein concentrations were determined by the BCA. The efficiency of collecting mitotic cells was further confirmed by Western blots using anti-phospho-histone H3 antibody. Cell lysates with equal amount of proteins were mixed with 4× SDS sample buffer and boiled. Sample printing and probing was done as described (Tibes et al., 2006). The median protein concentration of each set of slides was used as the control level. The cut off for this screen was a 1.5-fold increase of the apparent protein levels over the control. While the screen identified several hits, SRC3 is the only one that validated by Western Blot.

RPPA screen

After siRNA transfection, asynchronized cells were either analyzed directly or separated into interphase and mitotic cells by light trypsinization (0.005% in PBS) at room temperature for 5 minutes with gentle shaking and rocking to allow mitotic cells to detach. The presence of mitotic cells was determined by light microscopy based on cell and nuclear morphology. This process was repeated until at least 80% of mitotic cells were collected. Trypsin at 0.05% was added to collect the rest of the cells, which were considered to be mostly in interphase. Collected cells were gently rinsed with PBS, resuspended in lysis buffer (50 mM Hepes, pH 7.4, 150 mM NaCl, 1 mM EGTA, 10 mM sodium pyrophosphate, 100 mM NaF, 10% glycerol, 1.5 mM MgCl, 1% Triton X-100, 1 mM Na3VO4, and protease inhibitor from Roche), incubated on ice for 20 minutes, and finally centrifuged at 14,000 rpm for 10 minutes at 4 °C. Supernatants were then collected and protein concentrations were determined by the BCA. The efficiency of collecting mitotic cells was further confirmed by Western blots using anti-phospho-histone H3 antibody. Cell lysates with equal amount of proteins were mixed with 4× SDS sample buffer and boiled. Sample printing and probing was done as described (Tibes et al., 2006). The median protein concentration of each set of slides was used as the control level. The cut off for this screen was a 1.5-fold increase of the apparent protein levels over the control. While the screen identified several hits, SRC3 is the only one that validated by Western Blot.

Translation analysesCells were pulse-labeled with 30 μCi/ml [35S]-Met/Cys ([35S]-Tran, MP Biomedicals) for 60 min as previously described (Byrd et al., 2005) before preparation of cell lysates. Labeled proteins in lysates were analyzed by SDS-PAGE autoradiography or precipitation with 10% TCA on glass fiber filters and scintillation counting. For polysome analysis, cells were treated with cycloheximide (150 μg/ml in normal growth media) for 10 min immediately before collection for lysis. Cell were lysed in 50 mM Tris-HCl (pH8.0), 100 mM NaCl, 5 mM MgCl2, 0.5% NP-40, 20 mM DTT, protease inhibitor (Roche), 0.1 mg/ml cycloheximide, and 0.5 mg/ml heparin-Na by Dounce homogenization, and post nuclear lysates were collected by sedimentation at 5000 × g. The cell lysates were then subjected to 10–50% continuous sucrose gradient centrifugation at 43,000 rpm in a SW55 rotor at 4 °C for 60 min and the eluted fractions were continuously monitored by UV spectroscopy with an ISCO gradient fractionator as described (Rivera and Lloyd, 2008). For analysis of distribution of ubiquitin mRNAs in polysomes, pooled polysome sucrose gradient fractions were combined with 50 μg carrier yeast tRNA then RNA precipitated with three volumes ethanol overnight. Pelleted RNA was extracted with Trizol reagent (Invitrogen), precipitated in ethanol, and washed in 75% ethanol before RT-PCR analysis. For luciferase reporter assays, HeLa Tet-OFF cells (4 × 104) in 12-well plates in duplicate were transfected with either pTRE2-luciferase, pTRE2-hpBCL2-IRES-FL, or pTRE2-CVB3-IRES-FL (400 ng) together with Int6 siRNA#1 or control siRNA (2 μl of a 20 μM stock) in Lipofectamine 2000. At 24 or 48 hrs, reporter mRNA expression was induced for 6 hrs with Tet (−) medium and then cells were harvested for luciferase activity assay (Promega). Duplicate samples were subjected to semi-quantitative RT-PCR to assess levels of Luc reporter RNA to normalize translation results.

Translation analyses

Cells were pulse-labeled with 30 μCi/ml [35S]-Met/Cys ([35S]-Tran, MP Biomedicals) for 60 min as previously described (Byrd et al., 2005) before preparation of cell lysates. Labeled proteins in lysates were analyzed by SDS-PAGE autoradiography or precipitation with 10% TCA on glass fiber filters and scintillation counting. For polysome analysis, cells were treated with cycloheximide (150 μg/ml in normal growth media) for 10 min immediately before collection for lysis. Cell were lysed in 50 mM Tris-HCl (pH8.0), 100 mM NaCl, 5 mM MgCl2, 0.5% NP-40, 20 mM DTT, protease inhibitor (Roche), 0.1 mg/ml cycloheximide, and 0.5 mg/ml heparin-Na by Dounce homogenization, and post nuclear lysates were collected by sedimentation at 5000 × g. The cell lysates were then subjected to 10–50% continuous sucrose gradient centrifugation at 43,000 rpm in a SW55 rotor at 4 °C for 60 min and the eluted fractions were continuously monitored by UV spectroscopy with an ISCO gradient fractionator as described (Rivera and Lloyd, 2008). For analysis of distribution of ubiquitin mRNAs in polysomes, pooled polysome sucrose gradient fractions were combined with 50 μg carrier yeast tRNA then RNA precipitated with three volumes ethanol overnight. Pelleted RNA was extracted with Trizol reagent (Invitrogen), precipitated in ethanol, and washed in 75% ethanol before RT-PCR analysis. For luciferase reporter assays, HeLa Tet-OFF cells (4 × 104) in 12-well plates in duplicate were transfected with either pTRE2-luciferase, pTRE2-hpBCL2-IRES-FL, or pTRE2-CVB3-IRES-FL (400 ng) together with Int6 siRNA#1 or control siRNA (2 μl of a 20 μM stock) in Lipofectamine 2000. At 24 or 48 hrs, reporter mRNA expression was induced for 6 hrs with Tet (−) medium and then cells were harvested for luciferase activity assay (Promega). Duplicate samples were subjected to semi-quantitative RT-PCR to assess levels of Luc reporter RNA to normalize translation results.

ResultsLoss of Int6 enhances transformation and disrupts morphogenesis of human mammary epithelial cellsA detailed analyses of molecular activities of Int6 using HeLa cells showed that when Int6 expression is knocked down, entry into and progression through mitosis are both delayed, leading to a decrease of cells in G1 with a concurrent rise of cells in M-phase (Morris and Jalinot, 2005). We confirmed these results by FACS and immunostaining (Figure S1). Furthermore, upon careful examination of our FACS data, we found that sub-G1 cells increased after Int6 expression was repressed. Subsequent TUNEL assays detected a 6-fold increase of positive cells, with nearly 20% of the cells undergoing apoptosis (Figure 1a). The doubling time of Int6 repressed HeLa cells was substantially increased compatible with increased apoptosis (Figure 1b).The observation that knocking down Int6 induces apoptosis and an increase of doubling time seems at odds with the concept that Int6 deficiency promotes breast tumorigenesis. We thus questioned whether the response of human mammary epithelial cells to Int6 silencing might be different from HeLa cells, which were derived from cervical carcinoma. We knocked down INT6 expression in MCF10A normal human mammary epithelial cells, using the same siRNA sequences used in the HeLa cell study, and found that the same two siRNAs (#1 and #3) efficiently repress INT6 expression (50–70% reduction by #1 and 30–40% reduction by #3). siRNA #1 was thus chosen for further studies. Strikingly, in contrast to HeLa cells, when Int6 expression was knocked down in MCF10A cells, there was no obvious change in doubling time (Figure 1b). Furthermore, TUNEL assays showed no change, or even a slight decrease, in the fraction of apoptotic cells (Figure 1a). Similar to HeLa cells, when Int6-repressed MCF10A cells were fixed and immunostained, there was a reduction of cells in early mitosis and an increase of cells in late mitosis (Figure 1c). However, these changes were much smaller than those observed in HeLa cells. These data collectively suggest that while Int6 down-regulation in both HeLa and MCF10A cells dysregulates mitosis, MCF10A knock-down cells appear resistant to apoptosis with no detectable change in doubling time.To define the role of Int6 as a tumor suppressor for breast cancer, we first performed soft agar colony formation assays using MCF10A cells, and found that knocking down INT6 expression did not detectably induce colony formation by these cells. However in MCF10AT cells (Dawson et al., 1996), which are H-RasV12 expressing MCF10A cells that display a weak transformation phenotype, INT6 down-regulation markedly enhanced both the frequency and numbers of colonies formed (Figure 2a). We constructed an INT6 cDNA, INT6R, that is refractory to the siRNA. We then showed that expression of INT6R rescued the transformed phenotype of MCF10AT cells (Figure 2a) and the growth defect of HeLa cells (data not shown) when Int6 expression was knocked-down, suggesting that the described siRNA represses Int6 expression in a highly selective manner. These data indicate that while inactivating Int6 alone is too weak to detectably transform MCF10A cells in 2-D culture, it can collaborate with oncogenic Ras to more efficiently transform mammary epithelial cells.3-D culture systems have been established that allow normal mammary epithelial cells to form hollow, growth-arrested structures that recapitulate many features of the architecture of mammary epithelium. This system makes it feasible to follow many phenotypic changes associated with early phases of tumorigenesis. Normal MCF10A mammary epithelial cells first proliferate, and then polarize and form spheres within 8 days (Figure 2b). Between Days 8–12, the interior cells undergo apoptosis to create a hollow chamber. These hollow spheres are called acini and resemble mammary alveoli and terminal end buds (Figure 2b and c). When INT6 expression was reduced, at Day-8, these acini were substantially larger than those of the control cells, and at Day-12, these acini continued to increase in size and became deformed (Figure 2b and c). These structures were further examined by confocal microscopy to visualize DNA, Ki-67, and cleaved caspase 3, in order to analyze the number of cells, proliferation status, and apoptosis, respectively. Our data showed that in acini formed by control cells, proliferative cells were restricted to the periphery while the cells in the interior underwent apoptosis. In contrast, when INT6 was repressed, cells in the interior of the abnormal acini remained highly proliferative and showed little sign of apoptosis. We repeated these experiments with siRNA#3 with similar results; however, since siRNA#3 silenced Int6 expression less efficiently, it took these cells about a week longer to develop similar acinar abnormalities (data not shown). Furthermore, we constructed a lentivirus shRNA vector based on the sequence of siRNA#1. The vector backbone contains GFP as a marker to allow for identifying acini that are derived from cells carrying the Int6 shRNA. Our data showed that all GFP-positive acini expressing the INT6 shRNA were abnormal (Figure S2). We conclude from these data that Int6 is likely to be required for proper development of human mammary structures.Int6 forms a complex with Rpn5 and proteasomesOur S. pombe data suggest that binding and regulating the proteasome is an evolutionarily conserved function of Int6. We thus investigated whether Int6 could do so in human cells and how such regulation might shed light on Int6’s roles in transformation. To first measure the association between Int6 and proteasomes in human cells, we prepared a polyclonal antibody against the proteasome regulatory subunit Rpn5, to which the S. pombe Int6 ortholog selectively binds (Yen et al., 2003). This antibody readily detected a band of 55 kDa, the predicted molecular weight of Rpn5, in HeLa, MCF7, and MCF10A human mammary epithelial cell lines as well as the GFP-tagged Rpn5 (Figure 3a and b). To further confirm that the antibody detects Rpn5, in MCF7 cells, siRNA targeting Rpn5 decreased protein levels by 50%, leading to a 2-fold increase in levels of polyubiquitylated proteins (Figure 3a), which is comparable to treating the cell with a proteasome inhibitor MG132 at 0.5 μM (data not shown). When Rpn5 was immunoprecipitated by this antibody, multiple proteasome subunits, including those in the catalytic core, as well as Int6, were co-precipitated (Figure 3c). Consistent with our previous findings that S. pombe Int6 binds directly to eIF3d/Moe1 (Yen and Chang, 2000) and that proteasome (Sha et al., 2009) and eIF3 subunits co-purify, we found that eIF3d/Moe1 co-precipitated with Rpn5. To confirm association with the proteasome complex, we transiently expressed GFP-Rpn5 and then immunoprecipitated Int6 and detected GFP-Rpn5, together with at least one other proteasome subunit, Rpt4, as previously reported (Figure 3d; (Hoareau Alves et al., 2002)). These data suggest that Int6 forms a stable complex with Rpn5 and the proteasome.Int6 is required for proteasome assembly and optimal functionSince Int6 associates with the proteasome, we asked whether Int6 also regulates proteasome functions. After INT6 expression was reduced, we analyzed proteasome integrity by a proteasome pull-down assay, which uses the UBL ubiquitin-like domain from the HHR23-B protein (UBLHRB) as bait. The activity of the purified proteasomes was then measured using an artificial substrate, Suc-LLVY-AMC, from which the fluorescent AMC can be cleaved and released by the proteasome. When the purified intact proteasomes were examined by Western blotting, the levels of several proteasome subunits were reduced by almost 50% in INT6-silenced HeLa cells (Figure 4a), and these proteasomes displayed a concurrent 50% decrease in activity in an in vitro assay using a fluorescent peptide substrate (Suc-LLVY-AMC, Figure 4b). In contrast, the total levels of proteasome subunit proteins were the same in the normal vs. the INT6-repressed cells (see “Lysate” in Figure 4a). These data suggest that loss of Int6 does not substantially decrease proteasome subunit levels; rather it decreases the assembly of active proteasomes. Similar results were obtained with MCF10A cells (data not shown). To directly analyze assembled proteasomes, we ran cell lysates in native gels in the presence of the same fluorescent peptide substrate to allow visualization of 20S proteasomes assembled with one or both 19S regulatory particles, which can be readily separated by this method. The data showed that both forms of proteasomes are substantially reduced in Int6 repressed MCF10A cells (Figure 4d). To rule out the possibility that abnormalities in proteasome assembly are a secondary effect caused by mitotic abnormalities in the knock down cells, we examined MCF10A cells synchronized in G1/S-phase by hydroxyurea and obtained similar results (Figure 4c).Stabilization of SRC3 in Int6-depleted cellsWe hypothesized that proteasome deficiencies caused by Int6 reduction could disrupt protein homeostasis leading to accumulation of oncoproteins. To test this, we performed a screen using reverse phase protein array (RPPA), in which a panel of antibodies optimized to detect 105 proteins that are relevant to cancer formation was assessed for effects of Int6 reduction (see Materials and Methods). We prepared cell lysates from asynchronized MCF10A cells that were transfected by either control siRNA or a siRNA against Int6. We also separated mitotic cells from interphase cells to better detect proteins whose accumulation might be influenced by the cell cycle. HeLa cells were similarly examined as a control. Several proteins were found to accumulate in Int6 silenced cells, among them is SRC3 (steroid receptor co-activator 3 aka AIB1,
amplified in breast cancer 1), which was chosen for this study because it is frequently overexpressed and/or amplified in breast cancer (Xu et al., 2009) and because its levels have been firmly established to be controlled by proteasomes (Lonard and O’Malley, 2008; Wu et al., 2007). Intriguingly, SRC3 accumulation appears to be influenced by the cell cycle, as the effect of Int6 knock down on SRC3 levels was readily detected in mitotic MCF10A cells, but not in interphase MCF10A cells (Figure 5a). This result was independently validated by Western blotting showing a 2–4 fold increase of SRC3 levels in MCF10A cells (Figure 5b), as well as MCF7 cells (data not shown). In HeLa cells, where Int6 is not transforming, however, Int6 silencing did not alter SRC3 levels as assessed by RPPA (data not shown) or by Western blot (Figure 5b).Proteasomes play a key role in the regulation of SRC3 levels. Consistent with the hypothesis that Int6 controls SRC3 stability in a proteasome-dependent manner, we measured the half-life of SRC3 in mitotically arrested MCF10A cells and found that SRC3 protein is substantially stabilized when Int6 was repressed (Figure 5c). No detectable change in half-life was detected in non-synchronized cells (data not shown). To investigate whether Int6 mediates SRC3 degradation by physically binding to SRC3 and its E3 ubiquitin ligase, Fbw7 (Wu et al., 2007), we performed coimmunoprecipitation experiments. Our data showed that when endogenous Int6 was immunoprecipitated, endogenous SRC3 and Fbw7 were co-immunoprecipitated in both MCF10A and MCF7 cells (Figure 5d and data not shown). To determine whether difference in Fbw7 levels can explain why SCR3 is selectively stabilized in Int6-silenced MCF10A and MCF7 cells but not HeLa cells, we performed Western blot to examine Fbw7 in these cells. Our data show that while Fbw7 is abundant in MCF10A and MCF7 cells, its levels are very low in HeLa cells (Figure 5e). These observations support the possibility that transformation of mammary epithelial cells resulting from Int6 down-regulation can be caused, at least in part, by impairment in proteasome activity, leading to accumulation of oncoproteins such as SRC3.Regulation of translation by Int6During the course of analyzing Int6’s role in proteasome regulation, we came upon a puzzling observation: Poly-ubiquitylated proteins would be expected to accumulate when proteasomes are inactivated, but surprisingly, the levels of these proteins were actually 30% lower in Int6-silenced HeLa and MCF10A cells (Figure 6a and data not shown). Efficient translation is critically required to maintain ubiquitin levels, a limiting factor for poly-ubiquitylation (Hanna et al., 2003). Since Int6 is known as an eIF3 component, we tested whether it played a role in translation of ubiquitin mRNAs. There are five known ubiquitin genes in the human genome, and two of them (UBB and UBC) contain tandem repeats of the same coding sequence (3 and 9 respectively). As assessed by RT-PCR, levels of various ubiquitin mRNAs were not detectably different between INT6 knocked-down cells and control cells (Figure 6b). To determine steady state ubiquitin levels, we used an antibody that can also detect monomeric ubiquitin and found its levels were decreased by nearly half (Figure 6a and c; b and c). We measured half-life of mono-ubiquitin and found no difference between those in normal and Int6 knocked-down MCF10A cells (data not shown). Finally we examined the distribution of ubiquitin mRNAs between non-translating 40–80S ribosomes and polysomes of different sizes. Our data showed that in Int6-silenced cells there was a shift of ubiquitin mRNAs away from polysomes to the non-translating ribosomes (Figure 6d). Collectively, these data suggest that translation of ubiquitin mRNAs is inefficient in Int6-repressed cells, causing a reduction of ubiquitin protein levels.In contrast to ubiquitin, levels of “house-keeping” proteins, e.g., actin, tubulin, and proteasome subunits, as shown earlier, were not decreased in Int6-repressed cells. Therefore, it is possible that Int6 can selectively control translation and/or turnover of some mRNAs, which may include those of human ubiquitin genes. In keeping with this concept, we detected a slight decrease of [35S]-Met incorporation into protein fractions and a modest shift from larger to smaller polysomes in both Int6-repressed HeLa and MCF10A cells (Figure 6e and f). These results are consistent with defects in translation initiation. We also examined cells synchronized in G1-S phase and obtained similar results (Figure S3), ruling out the possibility that such abnormalities are due to abnormal mitosis.Translation of the majority of mRNAs is highly dependent on their 5′cap. However, the mRNAs of a subset of genes including many oncogenes contain IRES sequences, which allow these transcripts to be translated under conditions (e.g., stress, hypoxia, etc) that inhibit translation of cap-dependent mRNAs. To more directly investigate whether Int6 differentially mediated one translation mode vs. the other, as a proof of principle we co-transfected our previously established Tet-OFF cell line with luciferase translation reporter mRNAs from Tet-inducible plasmids together with the INT6 siRNA. Our data show that the luciferase activity from the cap-dependent reporter with a short unstructured 5′ UTR is substantially reduced in INT6-repressed cells (Figure 6g). Of the two IRES reporters, INT6 repression inhibited translation activity from the Bcl2 IRES, but increased the activity of coxsackievirus B3 (CVB3) IRES. These data support the hypothesis that Int6 can differentially regulate translation initiation.

Loss of Int6 enhances transformation and disrupts morphogenesis of human mammary epithelial cellsA detailed analyses of molecular activities of Int6 using HeLa cells showed that when Int6 expression is knocked down, entry into and progression through mitosis are both delayed, leading to a decrease of cells in G1 with a concurrent rise of cells in M-phase (Morris and Jalinot, 2005). We confirmed these results by FACS and immunostaining (Figure S1). Furthermore, upon careful examination of our FACS data, we found that sub-G1 cells increased after Int6 expression was repressed. Subsequent TUNEL assays detected a 6-fold increase of positive cells, with nearly 20% of the cells undergoing apoptosis (Figure 1a). The doubling time of Int6 repressed HeLa cells was substantially increased compatible with increased apoptosis (Figure 1b).The observation that knocking down Int6 induces apoptosis and an increase of doubling time seems at odds with the concept that Int6 deficiency promotes breast tumorigenesis. We thus questioned whether the response of human mammary epithelial cells to Int6 silencing might be different from HeLa cells, which were derived from cervical carcinoma. We knocked down INT6 expression in MCF10A normal human mammary epithelial cells, using the same siRNA sequences used in the HeLa cell study, and found that the same two siRNAs (#1 and #3) efficiently repress INT6 expression (50–70% reduction by #1 and 30–40% reduction by #3). siRNA #1 was thus chosen for further studies. Strikingly, in contrast to HeLa cells, when Int6 expression was knocked down in MCF10A cells, there was no obvious change in doubling time (Figure 1b). Furthermore, TUNEL assays showed no change, or even a slight decrease, in the fraction of apoptotic cells (Figure 1a). Similar to HeLa cells, when Int6-repressed MCF10A cells were fixed and immunostained, there was a reduction of cells in early mitosis and an increase of cells in late mitosis (Figure 1c). However, these changes were much smaller than those observed in HeLa cells. These data collectively suggest that while Int6 down-regulation in both HeLa and MCF10A cells dysregulates mitosis, MCF10A knock-down cells appear resistant to apoptosis with no detectable change in doubling time.To define the role of Int6 as a tumor suppressor for breast cancer, we first performed soft agar colony formation assays using MCF10A cells, and found that knocking down INT6 expression did not detectably induce colony formation by these cells. However in MCF10AT cells (Dawson et al., 1996), which are H-RasV12 expressing MCF10A cells that display a weak transformation phenotype, INT6 down-regulation markedly enhanced both the frequency and numbers of colonies formed (Figure 2a). We constructed an INT6 cDNA, INT6R, that is refractory to the siRNA. We then showed that expression of INT6R rescued the transformed phenotype of MCF10AT cells (Figure 2a) and the growth defect of HeLa cells (data not shown) when Int6 expression was knocked-down, suggesting that the described siRNA represses Int6 expression in a highly selective manner. These data indicate that while inactivating Int6 alone is too weak to detectably transform MCF10A cells in 2-D culture, it can collaborate with oncogenic Ras to more efficiently transform mammary epithelial cells.3-D culture systems have been established that allow normal mammary epithelial cells to form hollow, growth-arrested structures that recapitulate many features of the architecture of mammary epithelium. This system makes it feasible to follow many phenotypic changes associated with early phases of tumorigenesis. Normal MCF10A mammary epithelial cells first proliferate, and then polarize and form spheres within 8 days (Figure 2b). Between Days 8–12, the interior cells undergo apoptosis to create a hollow chamber. These hollow spheres are called acini and resemble mammary alveoli and terminal end buds (Figure 2b and c). When INT6 expression was reduced, at Day-8, these acini were substantially larger than those of the control cells, and at Day-12, these acini continued to increase in size and became deformed (Figure 2b and c). These structures were further examined by confocal microscopy to visualize DNA, Ki-67, and cleaved caspase 3, in order to analyze the number of cells, proliferation status, and apoptosis, respectively. Our data showed that in acini formed by control cells, proliferative cells were restricted to the periphery while the cells in the interior underwent apoptosis. In contrast, when INT6 was repressed, cells in the interior of the abnormal acini remained highly proliferative and showed little sign of apoptosis. We repeated these experiments with siRNA#3 with similar results; however, since siRNA#3 silenced Int6 expression less efficiently, it took these cells about a week longer to develop similar acinar abnormalities (data not shown). Furthermore, we constructed a lentivirus shRNA vector based on the sequence of siRNA#1. The vector backbone contains GFP as a marker to allow for identifying acini that are derived from cells carrying the Int6 shRNA. Our data showed that all GFP-positive acini expressing the INT6 shRNA were abnormal (Figure S2). We conclude from these data that Int6 is likely to be required for proper development of human mammary structures.

Loss of Int6 enhances transformation and disrupts morphogenesis of human mammary epithelial cells

A detailed analyses of molecular activities of Int6 using HeLa cells showed that when Int6 expression is knocked down, entry into and progression through mitosis are both delayed, leading to a decrease of cells in G1 with a concurrent rise of cells in M-phase (Morris and Jalinot, 2005). We confirmed these results by FACS and immunostaining (Figure S1). Furthermore, upon careful examination of our FACS data, we found that sub-G1 cells increased after Int6 expression was repressed. Subsequent TUNEL assays detected a 6-fold increase of positive cells, with nearly 20% of the cells undergoing apoptosis (Figure 1a). The doubling time of Int6 repressed HeLa cells was substantially increased compatible with increased apoptosis (Figure 1b).

The observation that knocking down Int6 induces apoptosis and an increase of doubling time seems at odds with the concept that Int6 deficiency promotes breast tumorigenesis. We thus questioned whether the response of human mammary epithelial cells to Int6 silencing might be different from HeLa cells, which were derived from cervical carcinoma. We knocked down INT6 expression in MCF10A normal human mammary epithelial cells, using the same siRNA sequences used in the HeLa cell study, and found that the same two siRNAs (#1 and #3) efficiently repress INT6 expression (50–70% reduction by #1 and 30–40% reduction by #3). siRNA #1 was thus chosen for further studies. Strikingly, in contrast to HeLa cells, when Int6 expression was knocked down in MCF10A cells, there was no obvious change in doubling time (Figure 1b). Furthermore, TUNEL assays showed no change, or even a slight decrease, in the fraction of apoptotic cells (Figure 1a). Similar to HeLa cells, when Int6-repressed MCF10A cells were fixed and immunostained, there was a reduction of cells in early mitosis and an increase of cells in late mitosis (Figure 1c). However, these changes were much smaller than those observed in HeLa cells. These data collectively suggest that while Int6 down-regulation in both HeLa and MCF10A cells dysregulates mitosis, MCF10A knock-down cells appear resistant to apoptosis with no detectable change in doubling time.

To define the role of Int6 as a tumor suppressor for breast cancer, we first performed soft agar colony formation assays using MCF10A cells, and found that knocking down INT6 expression did not detectably induce colony formation by these cells. However in MCF10AT cells (Dawson et al., 1996), which are H-RasV12 expressing MCF10A cells that display a weak transformation phenotype, INT6 down-regulation markedly enhanced both the frequency and numbers of colonies formed (Figure 2a). We constructed an INT6 cDNA, INT6R, that is refractory to the siRNA. We then showed that expression of INT6R rescued the transformed phenotype of MCF10AT cells (Figure 2a) and the growth defect of HeLa cells (data not shown) when Int6 expression was knocked-down, suggesting that the described siRNA represses Int6 expression in a highly selective manner. These data indicate that while inactivating Int6 alone is too weak to detectably transform MCF10A cells in 2-D culture, it can collaborate with oncogenic Ras to more efficiently transform mammary epithelial cells.

3-D culture systems have been established that allow normal mammary epithelial cells to form hollow, growth-arrested structures that recapitulate many features of the architecture of mammary epithelium. This system makes it feasible to follow many phenotypic changes associated with early phases of tumorigenesis. Normal MCF10A mammary epithelial cells first proliferate, and then polarize and form spheres within 8 days (Figure 2b). Between Days 8–12, the interior cells undergo apoptosis to create a hollow chamber. These hollow spheres are called acini and resemble mammary alveoli and terminal end buds (Figure 2b and c). When INT6 expression was reduced, at Day-8, these acini were substantially larger than those of the control cells, and at Day-12, these acini continued to increase in size and became deformed (Figure 2b and c). These structures were further examined by confocal microscopy to visualize DNA, Ki-67, and cleaved caspase 3, in order to analyze the number of cells, proliferation status, and apoptosis, respectively. Our data showed that in acini formed by control cells, proliferative cells were restricted to the periphery while the cells in the interior underwent apoptosis. In contrast, when INT6 was repressed, cells in the interior of the abnormal acini remained highly proliferative and showed little sign of apoptosis. We repeated these experiments with siRNA#3 with similar results; however, since siRNA#3 silenced Int6 expression less efficiently, it took these cells about a week longer to develop similar acinar abnormalities (data not shown). Furthermore, we constructed a lentivirus shRNA vector based on the sequence of siRNA#1. The vector backbone contains GFP as a marker to allow for identifying acini that are derived from cells carrying the Int6 shRNA. Our data showed that all GFP-positive acini expressing the INT6 shRNA were abnormal (Figure S2). We conclude from these data that Int6 is likely to be required for proper development of human mammary structures.

Int6 forms a complex with Rpn5 and proteasomesOur S. pombe data suggest that binding and regulating the proteasome is an evolutionarily conserved function of Int6. We thus investigated whether Int6 could do so in human cells and how such regulation might shed light on Int6’s roles in transformation. To first measure the association between Int6 and proteasomes in human cells, we prepared a polyclonal antibody against the proteasome regulatory subunit Rpn5, to which the S. pombe Int6 ortholog selectively binds (Yen et al., 2003). This antibody readily detected a band of 55 kDa, the predicted molecular weight of Rpn5, in HeLa, MCF7, and MCF10A human mammary epithelial cell lines as well as the GFP-tagged Rpn5 (Figure 3a and b). To further confirm that the antibody detects Rpn5, in MCF7 cells, siRNA targeting Rpn5 decreased protein levels by 50%, leading to a 2-fold increase in levels of polyubiquitylated proteins (Figure 3a), which is comparable to treating the cell with a proteasome inhibitor MG132 at 0.5 μM (data not shown). When Rpn5 was immunoprecipitated by this antibody, multiple proteasome subunits, including those in the catalytic core, as well as Int6, were co-precipitated (Figure 3c). Consistent with our previous findings that S. pombe Int6 binds directly to eIF3d/Moe1 (Yen and Chang, 2000) and that proteasome (Sha et al., 2009) and eIF3 subunits co-purify, we found that eIF3d/Moe1 co-precipitated with Rpn5. To confirm association with the proteasome complex, we transiently expressed GFP-Rpn5 and then immunoprecipitated Int6 and detected GFP-Rpn5, together with at least one other proteasome subunit, Rpt4, as previously reported (Figure 3d; (Hoareau Alves et al., 2002)). These data suggest that Int6 forms a stable complex with Rpn5 and the proteasome.

Int6 forms a complex with Rpn5 and proteasomes

Our S. pombe data suggest that binding and regulating the proteasome is an evolutionarily conserved function of Int6. We thus investigated whether Int6 could do so in human cells and how such regulation might shed light on Int6’s roles in transformation. To first measure the association between Int6 and proteasomes in human cells, we prepared a polyclonal antibody against the proteasome regulatory subunit Rpn5, to which the S. pombe Int6 ortholog selectively binds (Yen et al., 2003). This antibody readily detected a band of 55 kDa, the predicted molecular weight of Rpn5, in HeLa, MCF7, and MCF10A human mammary epithelial cell lines as well as the GFP-tagged Rpn5 (Figure 3a and b). To further confirm that the antibody detects Rpn5, in MCF7 cells, siRNA targeting Rpn5 decreased protein levels by 50%, leading to a 2-fold increase in levels of polyubiquitylated proteins (Figure 3a), which is comparable to treating the cell with a proteasome inhibitor MG132 at 0.5 μM (data not shown). When Rpn5 was immunoprecipitated by this antibody, multiple proteasome subunits, including those in the catalytic core, as well as Int6, were co-precipitated (Figure 3c). Consistent with our previous findings that S. pombe Int6 binds directly to eIF3d/Moe1 (Yen and Chang, 2000) and that proteasome (Sha et al., 2009) and eIF3 subunits co-purify, we found that eIF3d/Moe1 co-precipitated with Rpn5. To confirm association with the proteasome complex, we transiently expressed GFP-Rpn5 and then immunoprecipitated Int6 and detected GFP-Rpn5, together with at least one other proteasome subunit, Rpt4, as previously reported (Figure 3d; (Hoareau Alves et al., 2002)). These data suggest that Int6 forms a stable complex with Rpn5 and the proteasome.

Int6 is required for proteasome assembly and optimal functionSince Int6 associates with the proteasome, we asked whether Int6 also regulates proteasome functions. After INT6 expression was reduced, we analyzed proteasome integrity by a proteasome pull-down assay, which uses the UBL ubiquitin-like domain from the HHR23-B protein (UBLHRB) as bait. The activity of the purified proteasomes was then measured using an artificial substrate, Suc-LLVY-AMC, from which the fluorescent AMC can be cleaved and released by the proteasome. When the purified intact proteasomes were examined by Western blotting, the levels of several proteasome subunits were reduced by almost 50% in INT6-silenced HeLa cells (Figure 4a), and these proteasomes displayed a concurrent 50% decrease in activity in an in vitro assay using a fluorescent peptide substrate (Suc-LLVY-AMC, Figure 4b). In contrast, the total levels of proteasome subunit proteins were the same in the normal vs. the INT6-repressed cells (see “Lysate” in Figure 4a). These data suggest that loss of Int6 does not substantially decrease proteasome subunit levels; rather it decreases the assembly of active proteasomes. Similar results were obtained with MCF10A cells (data not shown). To directly analyze assembled proteasomes, we ran cell lysates in native gels in the presence of the same fluorescent peptide substrate to allow visualization of 20S proteasomes assembled with one or both 19S regulatory particles, which can be readily separated by this method. The data showed that both forms of proteasomes are substantially reduced in Int6 repressed MCF10A cells (Figure 4d). To rule out the possibility that abnormalities in proteasome assembly are a secondary effect caused by mitotic abnormalities in the knock down cells, we examined MCF10A cells synchronized in G1/S-phase by hydroxyurea and obtained similar results (Figure 4c).

Int6 is required for proteasome assembly and optimal function

Since Int6 associates with the proteasome, we asked whether Int6 also regulates proteasome functions. After INT6 expression was reduced, we analyzed proteasome integrity by a proteasome pull-down assay, which uses the UBL ubiquitin-like domain from the HHR23-B protein (UBLHRB) as bait. The activity of the purified proteasomes was then measured using an artificial substrate, Suc-LLVY-AMC, from which the fluorescent AMC can be cleaved and released by the proteasome. When the purified intact proteasomes were examined by Western blotting, the levels of several proteasome subunits were reduced by almost 50% in INT6-silenced HeLa cells (Figure 4a), and these proteasomes displayed a concurrent 50% decrease in activity in an in vitro assay using a fluorescent peptide substrate (Suc-LLVY-AMC, Figure 4b). In contrast, the total levels of proteasome subunit proteins were the same in the normal vs. the INT6-repressed cells (see “Lysate” in Figure 4a). These data suggest that loss of Int6 does not substantially decrease proteasome subunit levels; rather it decreases the assembly of active proteasomes. Similar results were obtained with MCF10A cells (data not shown). To directly analyze assembled proteasomes, we ran cell lysates in native gels in the presence of the same fluorescent peptide substrate to allow visualization of 20S proteasomes assembled with one or both 19S regulatory particles, which can be readily separated by this method. The data showed that both forms of proteasomes are substantially reduced in Int6 repressed MCF10A cells (Figure 4d). To rule out the possibility that abnormalities in proteasome assembly are a secondary effect caused by mitotic abnormalities in the knock down cells, we examined MCF10A cells synchronized in G1/S-phase by hydroxyurea and obtained similar results (Figure 4c).

Stabilization of SRC3 in Int6-depleted cellsWe hypothesized that proteasome deficiencies caused by Int6 reduction could disrupt protein homeostasis leading to accumulation of oncoproteins. To test this, we performed a screen using reverse phase protein array (RPPA), in which a panel of antibodies optimized to detect 105 proteins that are relevant to cancer formation was assessed for effects of Int6 reduction (see Materials and Methods). We prepared cell lysates from asynchronized MCF10A cells that were transfected by either control siRNA or a siRNA against Int6. We also separated mitotic cells from interphase cells to better detect proteins whose accumulation might be influenced by the cell cycle. HeLa cells were similarly examined as a control. Several proteins were found to accumulate in Int6 silenced cells, among them is SRC3 (steroid receptor co-activator 3 aka AIB1,
amplified in breast cancer 1), which was chosen for this study because it is frequently overexpressed and/or amplified in breast cancer (Xu et al., 2009) and because its levels have been firmly established to be controlled by proteasomes (Lonard and O’Malley, 2008; Wu et al., 2007). Intriguingly, SRC3 accumulation appears to be influenced by the cell cycle, as the effect of Int6 knock down on SRC3 levels was readily detected in mitotic MCF10A cells, but not in interphase MCF10A cells (Figure 5a). This result was independently validated by Western blotting showing a 2–4 fold increase of SRC3 levels in MCF10A cells (Figure 5b), as well as MCF7 cells (data not shown). In HeLa cells, where Int6 is not transforming, however, Int6 silencing did not alter SRC3 levels as assessed by RPPA (data not shown) or by Western blot (Figure 5b).Proteasomes play a key role in the regulation of SRC3 levels. Consistent with the hypothesis that Int6 controls SRC3 stability in a proteasome-dependent manner, we measured the half-life of SRC3 in mitotically arrested MCF10A cells and found that SRC3 protein is substantially stabilized when Int6 was repressed (Figure 5c). No detectable change in half-life was detected in non-synchronized cells (data not shown). To investigate whether Int6 mediates SRC3 degradation by physically binding to SRC3 and its E3 ubiquitin ligase, Fbw7 (Wu et al., 2007), we performed coimmunoprecipitation experiments. Our data showed that when endogenous Int6 was immunoprecipitated, endogenous SRC3 and Fbw7 were co-immunoprecipitated in both MCF10A and MCF7 cells (Figure 5d and data not shown). To determine whether difference in Fbw7 levels can explain why SCR3 is selectively stabilized in Int6-silenced MCF10A and MCF7 cells but not HeLa cells, we performed Western blot to examine Fbw7 in these cells. Our data show that while Fbw7 is abundant in MCF10A and MCF7 cells, its levels are very low in HeLa cells (Figure 5e). These observations support the possibility that transformation of mammary epithelial cells resulting from Int6 down-regulation can be caused, at least in part, by impairment in proteasome activity, leading to accumulation of oncoproteins such as SRC3.

Stabilization of SRC3 in Int6-depleted cells

We hypothesized that proteasome deficiencies caused by Int6 reduction could disrupt protein homeostasis leading to accumulation of oncoproteins. To test this, we performed a screen using reverse phase protein array (RPPA), in which a panel of antibodies optimized to detect 105 proteins that are relevant to cancer formation was assessed for effects of Int6 reduction (see Materials and Methods). We prepared cell lysates from asynchronized MCF10A cells that were transfected by either control siRNA or a siRNA against Int6. We also separated mitotic cells from interphase cells to better detect proteins whose accumulation might be influenced by the cell cycle. HeLa cells were similarly examined as a control. Several proteins were found to accumulate in Int6 silenced cells, among them is SRC3 (steroid receptor co-activator 3 aka AIB1,
amplified in breast cancer 1), which was chosen for this study because it is frequently overexpressed and/or amplified in breast cancer (Xu et al., 2009) and because its levels have been firmly established to be controlled by proteasomes (Lonard and O’Malley, 2008; Wu et al., 2007). Intriguingly, SRC3 accumulation appears to be influenced by the cell cycle, as the effect of Int6 knock down on SRC3 levels was readily detected in mitotic MCF10A cells, but not in interphase MCF10A cells (Figure 5a). This result was independently validated by Western blotting showing a 2–4 fold increase of SRC3 levels in MCF10A cells (Figure 5b), as well as MCF7 cells (data not shown). In HeLa cells, where Int6 is not transforming, however, Int6 silencing did not alter SRC3 levels as assessed by RPPA (data not shown) or by Western blot (Figure 5b).

Proteasomes play a key role in the regulation of SRC3 levels. Consistent with the hypothesis that Int6 controls SRC3 stability in a proteasome-dependent manner, we measured the half-life of SRC3 in mitotically arrested MCF10A cells and found that SRC3 protein is substantially stabilized when Int6 was repressed (Figure 5c). No detectable change in half-life was detected in non-synchronized cells (data not shown). To investigate whether Int6 mediates SRC3 degradation by physically binding to SRC3 and its E3 ubiquitin ligase, Fbw7 (Wu et al., 2007), we performed coimmunoprecipitation experiments. Our data showed that when endogenous Int6 was immunoprecipitated, endogenous SRC3 and Fbw7 were co-immunoprecipitated in both MCF10A and MCF7 cells (Figure 5d and data not shown). To determine whether difference in Fbw7 levels can explain why SCR3 is selectively stabilized in Int6-silenced MCF10A and MCF7 cells but not HeLa cells, we performed Western blot to examine Fbw7 in these cells. Our data show that while Fbw7 is abundant in MCF10A and MCF7 cells, its levels are very low in HeLa cells (Figure 5e). These observations support the possibility that transformation of mammary epithelial cells resulting from Int6 down-regulation can be caused, at least in part, by impairment in proteasome activity, leading to accumulation of oncoproteins such as SRC3.

Regulation of translation by Int6During the course of analyzing Int6’s role in proteasome regulation, we came upon a puzzling observation: Poly-ubiquitylated proteins would be expected to accumulate when proteasomes are inactivated, but surprisingly, the levels of these proteins were actually 30% lower in Int6-silenced HeLa and MCF10A cells (Figure 6a and data not shown). Efficient translation is critically required to maintain ubiquitin levels, a limiting factor for poly-ubiquitylation (Hanna et al., 2003). Since Int6 is known as an eIF3 component, we tested whether it played a role in translation of ubiquitin mRNAs. There are five known ubiquitin genes in the human genome, and two of them (UBB and UBC) contain tandem repeats of the same coding sequence (3 and 9 respectively). As assessed by RT-PCR, levels of various ubiquitin mRNAs were not detectably different between INT6 knocked-down cells and control cells (Figure 6b). To determine steady state ubiquitin levels, we used an antibody that can also detect monomeric ubiquitin and found its levels were decreased by nearly half (Figure 6a and c; b and c). We measured half-life of mono-ubiquitin and found no difference between those in normal and Int6 knocked-down MCF10A cells (data not shown). Finally we examined the distribution of ubiquitin mRNAs between non-translating 40–80S ribosomes and polysomes of different sizes. Our data showed that in Int6-silenced cells there was a shift of ubiquitin mRNAs away from polysomes to the non-translating ribosomes (Figure 6d). Collectively, these data suggest that translation of ubiquitin mRNAs is inefficient in Int6-repressed cells, causing a reduction of ubiquitin protein levels.In contrast to ubiquitin, levels of “house-keeping” proteins, e.g., actin, tubulin, and proteasome subunits, as shown earlier, were not decreased in Int6-repressed cells. Therefore, it is possible that Int6 can selectively control translation and/or turnover of some mRNAs, which may include those of human ubiquitin genes. In keeping with this concept, we detected a slight decrease of [35S]-Met incorporation into protein fractions and a modest shift from larger to smaller polysomes in both Int6-repressed HeLa and MCF10A cells (Figure 6e and f). These results are consistent with defects in translation initiation. We also examined cells synchronized in G1-S phase and obtained similar results (Figure S3), ruling out the possibility that such abnormalities are due to abnormal mitosis.Translation of the majority of mRNAs is highly dependent on their 5′cap. However, the mRNAs of a subset of genes including many oncogenes contain IRES sequences, which allow these transcripts to be translated under conditions (e.g., stress, hypoxia, etc) that inhibit translation of cap-dependent mRNAs. To more directly investigate whether Int6 differentially mediated one translation mode vs. the other, as a proof of principle we co-transfected our previously established Tet-OFF cell line with luciferase translation reporter mRNAs from Tet-inducible plasmids together with the INT6 siRNA. Our data show that the luciferase activity from the cap-dependent reporter with a short unstructured 5′ UTR is substantially reduced in INT6-repressed cells (Figure 6g). Of the two IRES reporters, INT6 repression inhibited translation activity from the Bcl2 IRES, but increased the activity of coxsackievirus B3 (CVB3) IRES. These data support the hypothesis that Int6 can differentially regulate translation initiation.

Regulation of translation by Int6

During the course of analyzing Int6’s role in proteasome regulation, we came upon a puzzling observation: Poly-ubiquitylated proteins would be expected to accumulate when proteasomes are inactivated, but surprisingly, the levels of these proteins were actually 30% lower in Int6-silenced HeLa and MCF10A cells (Figure 6a and data not shown). Efficient translation is critically required to maintain ubiquitin levels, a limiting factor for poly-ubiquitylation (Hanna et al., 2003). Since Int6 is known as an eIF3 component, we tested whether it played a role in translation of ubiquitin mRNAs. There are five known ubiquitin genes in the human genome, and two of them (UBB and UBC) contain tandem repeats of the same coding sequence (3 and 9 respectively). As assessed by RT-PCR, levels of various ubiquitin mRNAs were not detectably different between INT6 knocked-down cells and control cells (Figure 6b). To determine steady state ubiquitin levels, we used an antibody that can also detect monomeric ubiquitin and found its levels were decreased by nearly half (Figure 6a and c; b and c). We measured half-life of mono-ubiquitin and found no difference between those in normal and Int6 knocked-down MCF10A cells (data not shown). Finally we examined the distribution of ubiquitin mRNAs between non-translating 40–80S ribosomes and polysomes of different sizes. Our data showed that in Int6-silenced cells there was a shift of ubiquitin mRNAs away from polysomes to the non-translating ribosomes (Figure 6d). Collectively, these data suggest that translation of ubiquitin mRNAs is inefficient in Int6-repressed cells, causing a reduction of ubiquitin protein levels.

In contrast to ubiquitin, levels of “house-keeping” proteins, e.g., actin, tubulin, and proteasome subunits, as shown earlier, were not decreased in Int6-repressed cells. Therefore, it is possible that Int6 can selectively control translation and/or turnover of some mRNAs, which may include those of human ubiquitin genes. In keeping with this concept, we detected a slight decrease of [35S]-Met incorporation into protein fractions and a modest shift from larger to smaller polysomes in both Int6-repressed HeLa and MCF10A cells (Figure 6e and f). These results are consistent with defects in translation initiation. We also examined cells synchronized in G1-S phase and obtained similar results (Figure S3), ruling out the possibility that such abnormalities are due to abnormal mitosis.

Translation of the majority of mRNAs is highly dependent on their 5′cap. However, the mRNAs of a subset of genes including many oncogenes contain IRES sequences, which allow these transcripts to be translated under conditions (e.g., stress, hypoxia, etc) that inhibit translation of cap-dependent mRNAs. To more directly investigate whether Int6 differentially mediated one translation mode vs. the other, as a proof of principle we co-transfected our previously established Tet-OFF cell line with luciferase translation reporter mRNAs from Tet-inducible plasmids together with the INT6 siRNA. Our data show that the luciferase activity from the cap-dependent reporter with a short unstructured 5′ UTR is substantially reduced in INT6-repressed cells (Figure 6g). Of the two IRES reporters, INT6 repression inhibited translation activity from the Bcl2 IRES, but increased the activity of coxsackievirus B3 (CVB3) IRES. These data support the hypothesis that Int6 can differentially regulate translation initiation.

DiscussionIn support of the model that Int6 is a potential breast tumor suppressor, we show that reducing Int6 expression is sufficient to confer characteristics associated with transformation in MCF10A cells, and in 3D culture these cells form abnormally large acini, which contain proliferative cells in the interior that would otherwise undergo apoptosis in the presence of Int6. These abnormalities can be partly explained by Int6 binding and regulating proteasomes in human cells as has been demonstrated with the Int6 ortholog in S. pombe. Down-regulation of Int6 weakens proteasome activity, leading to stabilization of oncoproteins such as SRC3 in human mammary epithelial cells.In breast cancers, SRC3 levels are frequently up-regulated and positively correlate with larger tumor size, higher tumor grade, and poor disease-free survival (Xu et al., 2009). The role of SRC3 as an oncogene has been firmly established in mouse models. When human SRC3 is overexpressed by the MMTV promoter in mice, hypertrophy, hyperplasia, and abnormal post-weaning involution can be readily detected in the mammary glands, and these mice efficiently develop breast tumors. In contrast, when mouse SRC3 is deleted, tumor formation induced by several oncogenes and chemical carcinogens is greatly reduced. SRC3 can promote cell growth in response to estrogen by acting as a transcription coactivator for estrogen receptors. Furthermore, when SRC3 is overexpressed, post-weaning involution appears to be blocked by a lack of apoptosis. Conversely, when SRC3 is knocked down by shRNA in cells derived from the MMTV-SRC3 transgenic mice, apoptosis readily occurs. These results suggest that SRC3 also has potent anti-apoptotic activities. Thus it is likely that the increase in SRC3 levels contributes to the increased proliferation and reduction of apoptosis in Int6 silenced MCF10A cells.SRC3 may also contribute to therapy responsiveness as there is a correlation between high levels of SRC3 and tamoxifen resistance (Osborne et al., 2003). In search of differential INT6 expression among different breast cancer subtypes at Oncomine, we found that INT6 down-regulation is selectively associated with ER+ breast tumors, but not with ER− tumors, with p values in the order of 10−7 (“Liu Breast,” “Sotiriou Breast 3,” “Richardson Breast 2,” and “Minn Breast 2,”). In another study (“Farmer Breast”), INT6 down-regulation associates with luminal-like breast tumors, which are frequently ER+ (p = 8.3 × 10−6, n = 27), but not with basal-like tumors (p=1, n = 16), which are usually ER−. These microarray studies suggest that a reduction of Int6 levels may be advantageous for the development of ER+ tumors. In agreement with this, reduction of Int6 protein levels has been shown to correlate with tamoxifen resistance (Umar et al., 2009). The two studies raise the possibility that the effect of Int6 on SRC3 levels may be most clinically relevant in the context of response to tamoxifen. Indeed, such a relationship at the level of gene expression is present in a previously reported xenograft model for tamoxifen resistance (Massarweh et al., 2006) (Supplemental Figure S4).Since Int6 knock-down reduces levels of properly assembled proteasomes, it is expected that many proteasome substrates would be stabilized in Int6-silenced cells. In agreement with this, we did detect a slight increase of cyclin-B levels in Int6-repressed HeLa (Figure S5) and MCF10A cells (data not shown), and the RPPA screen also identified more than one hit. Despite this, loss of Int6 may selectively stabilize some proteins, such as SRC3, in a cell-type specific fashion. Our data suggest that this selectivity is at least partially dependent on the E3 for SRC3, Fbw7. This is supported by the fact that Fbw7 levels are very low in HeLa cells as compared to MCF10A and MCF7 cells; thus, there may not be enough Fbw7 for Int6 to interact in HeLa cells to efficiently target SRC3 for destruction. In addition, SRC3 is stabilized readily in mitotic Int6-silenced cells, but such stabilization is not detectable in non-synchronized cells. It is possible that SRC3 has a mitotic function in mammary epithelial cells and its levels are tightly and specifically regulated in an Int6-dependent manner. How Int6 regulates SRC3 stability in mitosis is not resolved; for example, we examined Fbw7 levels but found no difference between non-synchronized and mitotic MCF10A cells (data not shown). However, we speculate that SRC3 undergoes complex covalent modifications to influence its stability and activities in transcription (Li et al., 2008). We note that when cells are arrested in mitosis by nocodazol, we and others observe that SRC3 frequently appears as a doublet in Western blot (Figure 5c and Sean McGuire’s unpublished data), suggesting that specific covalent modifications may occur during mitosis to influence its stability.Our analyses of ubiquitin protein and mRNA levels raise several outstanding issues. First, besides causing proteasome disassembly, Int6 down-regulation can also inhibit proteolysis by decreasing ubiquitin protein levels. As such, in addition to directly raising levels of oncoproteins such as SRC3, low ubiquitin levels may also inhibit activities of tumor suppressors, such as BRCA1, a ubiquitin ligase. Second, Int6 is not essential for translation in some lower eukaryotes, suggesting that Int6 play a regulatory role in translation. Our data may illuminate one such role. Levels of several housekeeping proteins remain unchanged in Int6-silenced cells; this observation agrees with the modest decrease in global protein synthesis and polysomes. Thus it is possible that Int6 controls translation of a selective set of genes. As a proof of principle, our experiments using the 5′-UTR reporters support this concept. Our findings agree with studies in S. pombe that there are at least two eIF3 complexes, only one of which contains Int6 and associates with a unique set of mRNAs (Sha et al., 2009; Zhou et al., 2005). A recent study found a shift from cap-dependent to cap-independent translation that parallels the development of breast cancer (Braunstein et al., 2007). It will be important to determine whether down-regulation of Int6 can play a role in this transition. Since Int6 can apparently influence protein degradation as well as protein synthesis, it may be uniquely suited to fine-tune levels of regulatory proteins to prevent tumorigenesis. In a recent study analyzing S. pombe Int6 interactomes by LC-MS/MS (Sha et al., 2009), we found that Int6 is part of a supercomplex “translasome” that also contains proteasomes. The formation of translasomes can conceivably facilitate the coordination between the synthesis and degradation of proteins.In a recent study, Girzmil et al (2010) examined a triple negative (negative for ER, PR, and Her2) breast cancer cell line, MDA-MB-231, and came to the same conclusion as ours that Int6 is not needed for global translation but controls synthesis of a subset of genes (Grzmil et al., 2010). However, they did not find ubiquitin genes as Int6 targeted genes, suggesting that in different breast cell types, Int6 can regulate different sets of genes. Intriguingly, they examined Int6 protein levels in a set of 80 human tumors and reported that higher Int6 levels associated with poorly differentiated higher grade tumors. The breast cancer subtype information on these tumors is unclear; however, poorly differentiated breast tumors are more likely to be ER−. Consistent with this concept that high Int6 levels may associate with ER− tumors, they showed that Int6 repression in MDA-MB-231 blocks cell migration. These data together with ours suggest that Int6 levels are tightly regulated in different types of breast tumors — while Int6 down-regulation can transform normal mammary cells and induce endocrine therapy resistance in ER+ tumors, Int6 up-regulation may favor the development of ER tumors.In mice, breast cancer can be readily induced by Int6ΔC. We have shown that full-length Int6 can rescue the growth defect of S. pombe Int6-null cells, but Int6ΔC cannot (Yen and Chang, 2000; Yen et al., 2003). Therefore, Int6ΔC most likely acts in a dominant-negative fashion in mammalian cells. We and others have examined many human breast cancer cell lines but did not detect the presence of Int6ΔC ((Grzmil et al., 2010) and our unpublished results). Gunawardane et al. have previously reported a screen using a cDNA library from a human breast cancer cell line to identify cDNAs that when overexpressed can induce cytokine-independent migration of MCF10A cells (Gunawardane et al., 2005). Intriguingly, an N-terminally truncated form of Int6 (amino acid residues 354 – 446), which we called Int6ΔN, was also isolated from this screen (Sizhen Gao and Joan Brugge, unpublished results). In contrast to Int6ΔC, which lacks the PCI domain, Int6ΔN retains most of the PCI domain. During the course of studying S. pombe Int6 (Yen et al., 2003), we found that both the N- and C-termini of S. pombe Int6 are required for efficient binding to the proteasome subunit Rpn5. Therefore, we surmise that when Int6 is truncated either N- or C-terminally, the resulting proteins can function as dominant-negatives to disrupt normal Int6 function. It will be of interest to examine whether expression of truncated Int6 proteins can be found in human tumors, in order to determine whether Int6 truncations may similarly induce breast tumorigenesis in humans.

In support of the model that Int6 is a potential breast tumor suppressor, we show that reducing Int6 expression is sufficient to confer characteristics associated with transformation in MCF10A cells, and in 3D culture these cells form abnormally large acini, which contain proliferative cells in the interior that would otherwise undergo apoptosis in the presence of Int6. These abnormalities can be partly explained by Int6 binding and regulating proteasomes in human cells as has been demonstrated with the Int6 ortholog in S. pombe. Down-regulation of Int6 weakens proteasome activity, leading to stabilization of oncoproteins such as SRC3 in human mammary epithelial cells.

In breast cancers, SRC3 levels are frequently up-regulated and positively correlate with larger tumor size, higher tumor grade, and poor disease-free survival (Xu et al., 2009). The role of SRC3 as an oncogene has been firmly established in mouse models. When human SRC3 is overexpressed by the MMTV promoter in mice, hypertrophy, hyperplasia, and abnormal post-weaning involution can be readily detected in the mammary glands, and these mice efficiently develop breast tumors. In contrast, when mouse SRC3 is deleted, tumor formation induced by several oncogenes and chemical carcinogens is greatly reduced. SRC3 can promote cell growth in response to estrogen by acting as a transcription coactivator for estrogen receptors. Furthermore, when SRC3 is overexpressed, post-weaning involution appears to be blocked by a lack of apoptosis. Conversely, when SRC3 is knocked down by shRNA in cells derived from the MMTV-SRC3 transgenic mice, apoptosis readily occurs. These results suggest that SRC3 also has potent anti-apoptotic activities. Thus it is likely that the increase in SRC3 levels contributes to the increased proliferation and reduction of apoptosis in Int6 silenced MCF10A cells.

SRC3 may also contribute to therapy responsiveness as there is a correlation between high levels of SRC3 and tamoxifen resistance (Osborne et al., 2003). In search of differential INT6 expression among different breast cancer subtypes at Oncomine, we found that INT6 down-regulation is selectively associated with ER+ breast tumors, but not with ER− tumors, with p values in the order of 10−7 (“Liu Breast,” “Sotiriou Breast 3,” “Richardson Breast 2,” and “Minn Breast 2,”). In another study (“Farmer Breast”), INT6 down-regulation associates with luminal-like breast tumors, which are frequently ER+ (p = 8.3 × 10−6, n = 27), but not with basal-like tumors (p=1, n = 16), which are usually ER−. These microarray studies suggest that a reduction of Int6 levels may be advantageous for the development of ER+ tumors. In agreement with this, reduction of Int6 protein levels has been shown to correlate with tamoxifen resistance (Umar et al., 2009). The two studies raise the possibility that the effect of Int6 on SRC3 levels may be most clinically relevant in the context of response to tamoxifen. Indeed, such a relationship at the level of gene expression is present in a previously reported xenograft model for tamoxifen resistance (Massarweh et al., 2006) (Supplemental Figure S4).

Since Int6 knock-down reduces levels of properly assembled proteasomes, it is expected that many proteasome substrates would be stabilized in Int6-silenced cells. In agreement with this, we did detect a slight increase of cyclin-B levels in Int6-repressed HeLa (Figure S5) and MCF10A cells (data not shown), and the RPPA screen also identified more than one hit. Despite this, loss of Int6 may selectively stabilize some proteins, such as SRC3, in a cell-type specific fashion. Our data suggest that this selectivity is at least partially dependent on the E3 for SRC3, Fbw7. This is supported by the fact that Fbw7 levels are very low in HeLa cells as compared to MCF10A and MCF7 cells; thus, there may not be enough Fbw7 for Int6 to interact in HeLa cells to efficiently target SRC3 for destruction. In addition, SRC3 is stabilized readily in mitotic Int6-silenced cells, but such stabilization is not detectable in non-synchronized cells. It is possible that SRC3 has a mitotic function in mammary epithelial cells and its levels are tightly and specifically regulated in an Int6-dependent manner. How Int6 regulates SRC3 stability in mitosis is not resolved; for example, we examined Fbw7 levels but found no difference between non-synchronized and mitotic MCF10A cells (data not shown). However, we speculate that SRC3 undergoes complex covalent modifications to influence its stability and activities in transcription (Li et al., 2008). We note that when cells are arrested in mitosis by nocodazol, we and others observe that SRC3 frequently appears as a doublet in Western blot (Figure 5c and Sean McGuire’s unpublished data), suggesting that specific covalent modifications may occur during mitosis to influence its stability.

Our analyses of ubiquitin protein and mRNA levels raise several outstanding issues. First, besides causing proteasome disassembly, Int6 down-regulation can also inhibit proteolysis by decreasing ubiquitin protein levels. As such, in addition to directly raising levels of oncoproteins such as SRC3, low ubiquitin levels may also inhibit activities of tumor suppressors, such as BRCA1, a ubiquitin ligase. Second, Int6 is not essential for translation in some lower eukaryotes, suggesting that Int6 play a regulatory role in translation. Our data may illuminate one such role. Levels of several housekeeping proteins remain unchanged in Int6-silenced cells; this observation agrees with the modest decrease in global protein synthesis and polysomes. Thus it is possible that Int6 controls translation of a selective set of genes. As a proof of principle, our experiments using the 5′-UTR reporters support this concept. Our findings agree with studies in S. pombe that there are at least two eIF3 complexes, only one of which contains Int6 and associates with a unique set of mRNAs (Sha et al., 2009; Zhou et al., 2005). A recent study found a shift from cap-dependent to cap-independent translation that parallels the development of breast cancer (Braunstein et al., 2007). It will be important to determine whether down-regulation of Int6 can play a role in this transition. Since Int6 can apparently influence protein degradation as well as protein synthesis, it may be uniquely suited to fine-tune levels of regulatory proteins to prevent tumorigenesis. In a recent study analyzing S. pombe Int6 interactomes by LC-MS/MS (Sha et al., 2009), we found that Int6 is part of a supercomplex “translasome” that also contains proteasomes. The formation of translasomes can conceivably facilitate the coordination between the synthesis and degradation of proteins.

In a recent study, Girzmil et al (2010) examined a triple negative (negative for ER, PR, and Her2) breast cancer cell line, MDA-MB-231, and came to the same conclusion as ours that Int6 is not needed for global translation but controls synthesis of a subset of genes (Grzmil et al., 2010). However, they did not find ubiquitin genes as Int6 targeted genes, suggesting that in different breast cell types, Int6 can regulate different sets of genes. Intriguingly, they examined Int6 protein levels in a set of 80 human tumors and reported that higher Int6 levels associated with poorly differentiated higher grade tumors. The breast cancer subtype information on these tumors is unclear; however, poorly differentiated breast tumors are more likely to be ER−. Consistent with this concept that high Int6 levels may associate with ER− tumors, they showed that Int6 repression in MDA-MB-231 blocks cell migration. These data together with ours suggest that Int6 levels are tightly regulated in different types of breast tumors — while Int6 down-regulation can transform normal mammary cells and induce endocrine therapy resistance in ER+ tumors, Int6 up-regulation may favor the development of ER tumors.

In mice, breast cancer can be readily induced by Int6ΔC. We have shown that full-length Int6 can rescue the growth defect of S. pombe Int6-null cells, but Int6ΔC cannot (Yen and Chang, 2000; Yen et al., 2003). Therefore, Int6ΔC most likely acts in a dominant-negative fashion in mammalian cells. We and others have examined many human breast cancer cell lines but did not detect the presence of Int6ΔC ((Grzmil et al., 2010) and our unpublished results). Gunawardane et al. have previously reported a screen using a cDNA library from a human breast cancer cell line to identify cDNAs that when overexpressed can induce cytokine-independent migration of MCF10A cells (Gunawardane et al., 2005). Intriguingly, an N-terminally truncated form of Int6 (amino acid residues 354 – 446), which we called Int6ΔN, was also isolated from this screen (Sizhen Gao and Joan Brugge, unpublished results). In contrast to Int6ΔC, which lacks the PCI domain, Int6ΔN retains most of the PCI domain. During the course of studying S. pombe Int6 (Yen et al., 2003), we found that both the N- and C-termini of S. pombe Int6 are required for efficient binding to the proteasome subunit Rpn5. Therefore, we surmise that when Int6 is truncated either N- or C-terminally, the resulting proteins can function as dominant-negatives to disrupt normal Int6 function. It will be of interest to examine whether expression of truncated Int6 proteins can be found in human tumors, in order to determine whether Int6 truncations may similarly induce breast tumorigenesis in humans.

Supplementary Material1

Supplementary Material
