---
pmid: '20118233'
title: G9a and Glp methylate lysine 373 in the tumor suppressor p53.
authors:
- Huang J
- Dorsey J
- Chuikov S
- Zhang X
- Jenuwein T
- Reinberg D
- Berger SL
journal: J Biol Chem
year: '2010'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC2843213
doi: 10.1074/jbc.M109.062588
---

# G9a and Glp methylate lysine 373 in the tumor suppressor p53.
**Authors:** Huang J, Dorsey J, Chuikov S, Zhang X, Jenuwein T, Reinberg D, Berger SL
**Journal:** J Biol Chem (2010)
**DOI:** [10.1074/jbc.M109.062588](https://doi.org/10.1074/jbc.M109.062588)
**PMC:** [PMC2843213](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2843213/)

## Abstract

1. J Biol Chem. 2010 Mar 26;285(13):9636-9641. doi: 10.1074/jbc.M109.062588. Epub
 2010 Jan 29.

G9a and Glp methylate lysine 373 in the tumor suppressor p53.

Huang J(1), Dorsey J(2), Chuikov S(3), Zhang X(4), Jenuwein T(5), Reinberg D(6), 
Berger SL(7).

Author information:
(1)Laboratory of Cancer Biology and Genetics, NCI, National Institutes of 
Health, Bethesda, Maryland 20892; Wistar Institute, Philadelphia, Pennsylvania 
19104. Electronic address: huangj3@mail.nih.gov.
(2)Wistar Institute, Philadelphia, Pennsylvania 19104.
(3)New York University School of Medicine-Smilow Research Center, New York, New 
York 10016.
(4)Laboratory of Cancer Biology and Genetics, NCI, National Institutes of 
Health, Bethesda, Maryland 20892.
(5)Research Institute of Molecular Pathology, Vienna A-1030, Austria; Max Planck 
Institute of Immunobiology, Freiburg im Breisgau 0761 5108-0, Germany.
(6)New York University School of Medicine-Smilow Research Center, New York, New 
York 10016; University of Medicine and Dentistry of New Jersey, Piscataway, New 
Jersey 08854.
(7)Wistar Institute, Philadelphia, Pennsylvania 19104; Departments of Cell and 
Developmental Biology, Biology, and Genetics, University of Pennsylvania, 
Philadelphia, Pennsylvania 19104.

Erratum in
    J Biol Chem. 2010 Jun 4;285(23):18122. Pérez-Burgos, Laura [added].

The tumor suppressor p53 is regulated by numerous post-translational 
modifications. Lysine methylation has recently emerged as a key 
post-translational modification that alters the activity of p53. Here, we 
describe a novel lysine methylation site in p53 that is carried out by two 
homologous histone methyltransferases, G9a and Glp. G9a and Glp specifically 
methylate p53 at Lys(373), resulting mainly in dimethylation. During DNA damage, 
the overall level of p53 modified at Lys(373)me2 does not increase, despite the 
dramatic increase in total p53, indicating that Lys(373)me2 correlates with 
inactive p53. Further, reduction of G9a and/or Glp levels leads to a larger 
population of apoptotic cells. Examination of the Oncomine data base shows that 
G9a and Glp are overexpressed in various cancers compared with corresponding 
normal tissues, suggesting that they are putative oncogenes. These data reveal a 
new methylation site within p53 mediated by the methylases G9a and Glp and 
indicate that G9a is a potential inhibitory target for cancer treatment.

DOI: 10.1074/jbc.M109.062588
PMCID: PMC2843213
PMID: 20118233 [Indexed for MEDLINE]

## Full Text

Abstract

The tumor suppressor p53 is regulated by numerous post-translational modifications. Lysine methylation has recently emerged as a key post-translational modification that alters the activity of p53. Here, we describe a novel lysine methylation site in p53 that is carried out by two homologous histone methyltransferases, G9a and Glp. G9a and Glp specifically methylate p53 at Lys 373 , resulting mainly in dimethylation. During DNA damage, the overall level of p53 modified at Lys 373 me2 does not increase, despite the dramatic increase in total p53, indicating that Lys 373 me2 correlates with inactive p53. Further, reduction of G9a and/or Glp levels leads to a larger population of apoptotic cells. Examination of the Oncomine data base shows that G9a and Glp are overexpressed in various cancers compared with corresponding normal tissues, suggesting that they are putative oncogenes. These data reveal a new methylation site within p53 mediated by the methylases G9a and Glp and indicate that G9a is a potential inhibitory target for cancer treatment.

Introduction

The tumor suppressor p53 is a sequence-specific transcription factor. The transcription activity is critical for its tumor suppressive function; indeed, more than half of human tumors bear a mutation in the p53 gene, largely within the DNA binding domain ( 1 ). Many mechanisms regulate transcriptional activation by p53, including nuclear localization, DNA binding, post-translational modifications (PTMs), 2 and co-factor recruitment. These regulatory pathways are interdependent; for example, p53 acetylation promotes both its ability to bind to DNA and its recruitment of cofactors ( 2 ). Among all of these regulatory steps during the activation of p53, PTMs have attracted extensive attention because of their complex roles in tumorigenesis and possible clinical applications.

Lysine methylation is a relatively stable protein PTM relative to acetylation; and similar to acetylation, lysine methylation regulates diverse substrates ( 3 ). Regulatory potential is greatly increased because multiple methyl groups can be added to a single lysine residue, leading to mono-, di-, or trimethylation of a substrate protein. Indeed, within histone substrates, these various levels of methylation localize to different chromosomal regions and correlate with distinct biological outcomes. For example, histone H3 lysine 9 trimethylation (H3Lys 9 me3) generally localizes in the heterochromatic regions, whereas mono- and dimethylation (H3Lys 9 me1 and H3Lys 9 me2) localize within euchromatin ( 4 ).

Recently, lysine methylation has been shown to regulate the activity of p53 ( 3 ). Multiple lysines on p53 are methylated (Lys 370 , Lys 372 , and Lys 382 ) by the methyl transferases Smyd2, Set7/9, and PR-Set7 (also known as Set8), respectively, which were previously characterized as histone methylases ( 5 – 7 ). These methylases specifically monomethylate their corresponding sites in p53. However, Lys 370 and Lys 382 dimethylation (Lys 370 me2 and Lys 382 me2) have both been detected on p53, but the cognate enzymes to these methylation sites remain unknown ( 8 , 9 ).

Interestingly, similar to histones, the variable levels of p53 methylation result in different biological outcomes. For example, Lys 370 me1 is associated with repression of p53 function, whereas Lys 370 me2 is associated with p53 activation. In addition, Lys 382 me1 antagonizes Lys 382 acetylation (Lys 382 ac), whereas Lys 382 me2 is linked to p53 response to DNA damage ( 7 , 9 ). Further, as has also been observed for histone modifications, the PTMs of p53 cross-talk between one another, which greatly increases the potential complexity of p53 regulation by modification. For example, Lys 372 me1 suppresses Lys 370 me1, and Lys 382 me1 blocks Lys 382 ac ( 5 , 7 ).

G9a and Glp are homologous histone methyl transferases that methylate histone H3 at Lys 9 and Lys 27 . Gene knockouts of G9a or Glp in the mouse result in embryonic lethality ( 10 , 11 ), suggesting that G9a and Glp are critical during mammalian development. However, little is known about the pathophysiological roles of G9a and Glp in oncogenesis. Here, we investigate p53 methylation mediated by G9a and Glp. Our results indicate that there is a potential role for G9a and Glp in human cancers and link p53 methylation site to tumorigenesis.

DISCUSSION

The tumor suppressor p53 is heavily decorated with a variety of PTMs, which are centrally involved in regulation of p53 activity ( 1 ). Most of these modification sites reside within the N-terminal transactivation and C-terminal regulatory domains. However, these two regions contain many fewer inactivating mutations occurring in human cancer compared with the central DNA binding domain ( 1 ); this relationship leads to provocative questions about the role and overall importance of these modifications in p53 biology ( 13 ). In particular, regarding lysine modifications, mouse genetic models expressing p53 with seven (p53 7KR : lysines 367, 369, 370, 378, 379, 383, and 384 in mouse) or six (p53 6KR : lysines 367, 369, 370, 378, 379, and 383 in mouse) lysine-to-arginine mutations at its C terminus display only minor effects in either tumorigenesis or the DNA damage response ( 13 – 15 ). Lys 373 (Lys 370 in mouse), the site described in this study, is one of these lysine sites. One simple explanation for the modest defects and low incidence of mutations found at these modification sites is that redundancies exist between the sites. For example, p300/CBP acetylates p53 at numerous lysines (120, 164, 370, 372, 373, 381, 382, and 386 in human). p53 with substitutions at individual lysine residues behaves similarly to wild-type p53. However, substitution of all eight substrate lysines to arginine (8KR) leads to a strong defect in p53 activity ( 16 ). This result also potentially explains the observations that p53 7KR and p53 6KR mice are normal ( 14 , 15 ).

However, in contrast to acetylation, all identified p53 methylases are site-specific, i.e. each of Smyd2, Set7/9, PR-Set7 (Set8), G9a, and Glp methylates a single site on p53. Therefore, the lack of mutation(s) in these lysine sites cannot simply be explained by redundancies among these sites. Thus, a key question is whether these sites regulate p53 activity in growth and cancer pathways. One explanation is that the methylases engage in cross-talk, to cooperate or antagonize each other's activity. Indeed, positive-acting Lys 372 me1 by Set7/9 inhibits negative-acting Lys 370 me1 mediated by Smyd2 ( 5 ). This coordination between sites and enzymes could lead to altered p53 activity beyond the effect of altering a single modification site. Another explanation is that the same lysine residue could be positively (acetylation) or negatively (methylation) modified. Therefore, mutation from lysine to arginine not only blocks positive modification, acetylation, but also prevents negative modifications such as methylation. The ultimate outcome of lysine-to-arginine mutation is the combinatorial effect of the loss of these positive and negative modifications.

Compared with acetylation, which seems to act only positively to up-regulate p53 activity, a subset of the methylation sites behaves negatively, to maintain p53 in an inactive state. These negatively acting sites include Lys 370 me1, Lys 382 me1, and, in this current study based on increased apoptosis in the reduced enzyme conditions, Lys 373 me2. Hence, these methylations are all potentially oncogenic because high levels of the enzymes, leading to constitutive methylation, would lead to underactive p53.

Relevant to this issue is the fact, mentioned above, that the p53 gene is mutated in only half of human tumors. It remains unclear how tumor formation escapes p53-mediated surveillance mechanisms in the other half of cancers apparently lacking p53 gene mutations. Our results and observations from other studies suggest that inappropriate levels of PTMs, due to overexpression of the cognate enzymes, may contribute to the inactivation of p53. One example is LSD1, a histone and p53 demethylase, which is overexpressed in neuroblastoma ( 17 ) and is associated with the relapse of prostate cancer ( 18 ). Our previous findings show that LSD1 removes a positive-acting p53 methylation at Lys 372 me2 ( 9 ), thus, high levels of LSD1 could be oncogenic because of constitutive repression of p53.

In a similar vein, our current observations indicate that p53Lys 373 me2 is negative acting and that the cognate methylases G9a and Glp are up-regulated in many types of cancers. These findings, both for LSD1 removing an activating methylation and for G9a/Glp adding a repressing methylation, strongly suggest that enzymes that negatively regulate p53 activity can act as oncogenes when expressed at aberrantly high levels. These results suggest that new therapies are possible via inhibition of these enzymes, which in turn would restore the activity of p53. This concept is particularly intriguing in tumor/cancers that bear a wild-type p53 gene. Future efforts will be needed to address the precise roles of p53 lysine modifications in tumor pathways.
