---
pmid: '26582391'
title: TORC1 and TORC2 work together to regulate ribosomal protein S6 phosphorylation
  in Saccharomyces cerevisiae.
authors:
- Yerlikaya S
- Meusburger M
- Kumari R
- Huber A
- Anrather D
- Costanzo M
- Boone C
- Ammerer G
- Baranov PV
- Loewith R
journal: Mol Biol Cell
year: '2016'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC4713140
doi: 10.1091/mbc.E15-08-0594
---

# TORC1 and TORC2 work together to regulate ribosomal protein S6 phosphorylation in Saccharomyces cerevisiae.
**Authors:** Yerlikaya S, Meusburger M, Kumari R, Huber A, Anrather D, Costanzo M, Boone C, Ammerer G, Baranov PV, Loewith R
**Journal:** Mol Biol Cell (2016)
**DOI:** [10.1091/mbc.E15-08-0594](https://doi.org/10.1091/mbc.E15-08-0594)
**PMC:** [PMC4713140](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4713140/)

## Abstract

1. Mol Biol Cell. 2016 Jan 15;27(2):397-409. doi: 10.1091/mbc.E15-08-0594. Epub 
2015 Nov 18.

TORC1 and TORC2 work together to regulate ribosomal protein S6 phosphorylation 
in Saccharomyces cerevisiae.

Yerlikaya S(1), Meusburger M(1), Kumari R(2), Huber A(1), Anrather D(3), 
Costanzo M(4), Boone C(4), Ammerer G(3), Baranov PV(2), Loewith R(5).

Author information:
(1)Department of Molecular Biology and Institute of Genetics and Genomics of 
Geneva, University of Geneva, CH-1211 Geneva, Switzerland.
(2)School of Biochemistry and Cell Biology, University College Cork, Cork, 
Ireland.
(3)Max F. Perutz Laboratories, Department of Biochemistry, University of Vienna, 
A1030 Vienna, Austria.
(4)Banting and Best Department of Medical Research, Donnelly Center for Cellular 
and Biomolecular Research, University of Toronto, Toronto, ON M5S 3E1, Canada.
(5)Department of Molecular Biology and Institute of Genetics and Genomics of 
Geneva, University of Geneva, CH-1211 Geneva, Switzerland Swiss National Centre 
for Competence in Research Programme Chemical Biology, 1211 Geneva, Switzerland 
robbie.loewith@unige.ch.

Nutrient-sensitive phosphorylation of the S6 protein of the 40S subunit of the 
eukaryote ribosome is highly conserved. However, despite four decades of 
research, the functional consequences of this modification remain unknown. 
Revisiting this enigma in Saccharomyces cerevisiae, we found that the regulation 
of Rps6 phosphorylation on Ser-232 and Ser-233 is mediated by both TOR complex 1 
(TORC1) and TORC2. TORC1 regulates phosphorylation of both sites via the poorly 
characterized AGC-family kinase Ypk3 and the PP1 phosphatase Glc7, whereas TORC2 
regulates phosphorylation of only the N-terminal phosphosite via Ypk1. Cells 
expressing a nonphosphorylatable variant of Rps6 display a reduced growth rate 
and a 40S biogenesis defect, but these phenotypes are not observed in cells in 
which Rps6 kinase activity is compromised. Furthermore, using polysome profiling 
and ribosome profiling, we failed to uncover a role of Rps6 phosphorylation in 
either global translation or translation of individual mRNAs. Taking the results 
together, this work depicts the signaling cascades orchestrating Rps6 
phosphorylation in budding yeast, challenges the notion that Rps6 
phosphorylation plays a role in translation, and demonstrates that observations 
made with Rps6 knock-ins must be interpreted cautiously.

© 2016 Yerlikaya, Meusburger, et al. This article is distributed by The American 
Society for Cell Biology under license from the author(s). Two months after 
publication it is available to the public under an 
Attribution–Noncommercial–Share Alike 3.0 Unported Creative Commons License 
(http://creativecommons.org/licenses/by-nc-sa/3.0).

DOI: 10.1091/mbc.E15-08-0594
PMCID: PMC4713140
PMID: 26582391 [Indexed for MEDLINE]

## Full Text

INTRODUCTIONThe target of rapamycin (TOR) signaling network plays an important role in the adaptation of cells to their changing environment by linking extracellular and cell-intrinsic cues to the major cellular processes that regulate growth. For example, TOR controls cell growth by regulating gene transcription, translation, ribosome biogenesis, autophagy, lipid synthesis, and actin polarization (Loewith and Hall, 2011). In yeast, there are two TOR genes, TOR1 and TOR2, whereas in higher eukaryotes there is only one. TOR kinases are found in two functionally and structurally distinct multiprotein complexes named TOR complex 1 (TORC1) and TOR complex 2 (TORC2). Rapamycin, an antifungal and immunosuppressant macrolide, inhibits TORC1 but not TORC2. When applied in low millimolar concentrations, caffeine also specifically inhibits TORC1 (Kuranda et al., 2006; Reinke et al., 2006; Wanke et al., 2008; Loewith and Hall, 2011). Pharmacological inhibition of TORC2 in yeast was achieved only recently with the characterization of an imidazoquinoline called NVP-BHS345 (BHS), an ATP-competitive inhibitor of TOR that inhibits both complexes (Shimada et al., 2013; Rispal et al., 2015).In yeast, as well as in higher eukaryotes, arguably the best-characterized substrates of TOR are members of the AGC family of kinases. These include S6 kinases (S6Ks) and Sch9, which are phosphorylated by mammalian TORC1 (mTORC1) and yeast TORC1, respectively; and Akt/protein kinase B and Sgk1, and Ypk1 and Ypk2, which are phosphorylated by mTORC2 and TORC2, respectively (Pearce et al., 2010; Loewith and Hall, 2011). Phosphorylation of AGC kinases by TOR complexes occurs in distinct, carboxyl-terminal sequences known as the turn motif and hydrophobic motif and increases their activity (Pearce et al., 2010).Ribosomal protein S6 (Rps6) was one of the first ribosomal proteins found to undergo phosphorylation (Kabat, 1970; Gressner and Wool, 1974). Rps6 phosphorylation is conserved from yeast to mammalian cells. In higher eukaryotes, there are five phosphorylation sites on its carboxyl terminus, whereas only two of these sites (Ser-232 and Ser-233), corresponding to Ser-235 and Ser-236 in higher eukaryotes, are conserved in yeast (Supplemental Figure S1A). Rps6 phosphorylation is sensitive to nutrients, hormones, growth factors, and a variety of stress conditions (Meyuhas, 2008). In higher eukaryotes, S6 kinases 1 and 2 (S6K1 and S6K2) are the major kinases phosphorylating Rps6 on several sites (S235/S236/S240/S244) in response to mTORC1 activity (Pende et al., 2004). Rps6 is additionally phosphorylated by RSKs (90-kDa ribosomal protein S6 kinases) on S235/S236 downstream of mitogen-activated protein kinase signaling (Roux et al., 2007). Members of casein kinase I kinase family were found to phosphorylate Rps6 on Ser-247 in response to mitogenic stimuli (Hutchinson et al., 2011). Intriguingly, both TORC1 and TORC2 regulate Rps6 phosphorylation in fission yeast, via Psk1 and Gad8, respectively (Du et al., 2012; Nakashima et al., 2012). Rps6 dephosphorylation is less well studied but, at least in mammals, appears to require the type 1 protein phosphatase (PP1) (Belandia et al., 1994; Hutchinson et al., 2011).The major regulators of Rps6 phosphorylation in budding yeast remain unknown. Considering its role in translation and its ability to phosphorylate Rps6 in vitro, Sch9 was proposed to be the functional orthologue of S6 kinase in higher eukaryotes (Urban et al., 2007). However, phylogenetic analyses of the TOR signaling network has suggested that Ypk3, rather than Sch9, is more likely to be the S6 kinase orthologue in S. cerevisiae (van Dam et al., 2011). Consistently, YPK3 is orthologous to Schizosaccharomyces pombe PSK1. In addition, a recent study proposed that Ypk3 activity, but not Sch9 activity, is required for Rps6 phosphorylation in vivo in budding yeast (Gonzalez et al., 2015).Despite years of research, the physiological role of Rps6 phosphorylation remains mysterious. Knock-in mice expressing only the alanine-substituted nonphosphorylatable version of Rps6 exhibited severe phenotypes, including a reduced size, glucose intolerance, and muscle weakness (Ruvinsky et al., 2005). Studies in yeast, however, suggested that Rps6 phosphorylation had no observable effect on the growth of cells under different conditions (Kruse et al., 1985; Johnson and Warner, 1987). Rps6 phosphorylation was once believed to specifically couple mTORC1 signals to the translation of so-called 5′-TOP mRNAs (Fumagalli, 2000), which predominantly encode proteins required for protein synthesis (Levy et al., 1991; Iadevaia et al., 2008). However, another mTORC1 effector, 4E-BP1, is now believed to serve in this capacity (Hsieh et al., 2012; Thoreen et al., 2012). More recently, Rps6 phosphorylation has again been linked to the translation machinery via regulation of the transcription of genes encoding ribosome biogenesis factors (Chauvin et al., 2014).Here we revisited Rps6 phosphorylation in the model eukaryote S. cerevisiae. We found that Rps6 phosphorylation is differentially regulated downstream of TORC1 via Ypk3 and TORC2 via Ypk1 and Ypk2. TORC1 additionally appears to regulate Rps6 dephosphorylation via Glc7/Shp1. Cells expressing nonphosphorylatable variants of Rps6 presented phenotypes associated with 40S ribosome biogenesis defects, but these phenotypes were not recapitulated under conditions in which wild-type (wt) Rps6 protein is hypophosphorylated. Cells expressing nonphosphorylatable variants of Rps6 were partially resistant to the loss of Glc7 activity, suggesting that Rps6 hyperphosphorylation is toxic. Curiously, ribosome-footprinting analyses revealed that neither Rps6 phosphorylation nor Ypk1–3 activities are overtly involved in the regulation of gene expression at the level of transcription or translation and thus challenge the present models on the function of Rps6 phosphorylation derived from studies in higher eukaryotes.

INTRODUCTION

The target of rapamycin (TOR) signaling network plays an important role in the adaptation of cells to their changing environment by linking extracellular and cell-intrinsic cues to the major cellular processes that regulate growth. For example, TOR controls cell growth by regulating gene transcription, translation, ribosome biogenesis, autophagy, lipid synthesis, and actin polarization (Loewith and Hall, 2011). In yeast, there are two TOR genes, TOR1 and TOR2, whereas in higher eukaryotes there is only one. TOR kinases are found in two functionally and structurally distinct multiprotein complexes named TOR complex 1 (TORC1) and TOR complex 2 (TORC2). Rapamycin, an antifungal and immunosuppressant macrolide, inhibits TORC1 but not TORC2. When applied in low millimolar concentrations, caffeine also specifically inhibits TORC1 (Kuranda et al., 2006; Reinke et al., 2006; Wanke et al., 2008; Loewith and Hall, 2011). Pharmacological inhibition of TORC2 in yeast was achieved only recently with the characterization of an imidazoquinoline called NVP-BHS345 (BHS), an ATP-competitive inhibitor of TOR that inhibits both complexes (Shimada et al., 2013; Rispal et al., 2015).

In yeast, as well as in higher eukaryotes, arguably the best-characterized substrates of TOR are members of the AGC family of kinases. These include S6 kinases (S6Ks) and Sch9, which are phosphorylated by mammalian TORC1 (mTORC1) and yeast TORC1, respectively; and Akt/protein kinase B and Sgk1, and Ypk1 and Ypk2, which are phosphorylated by mTORC2 and TORC2, respectively (Pearce et al., 2010; Loewith and Hall, 2011). Phosphorylation of AGC kinases by TOR complexes occurs in distinct, carboxyl-terminal sequences known as the turn motif and hydrophobic motif and increases their activity (Pearce et al., 2010).

Ribosomal protein S6 (Rps6) was one of the first ribosomal proteins found to undergo phosphorylation (Kabat, 1970; Gressner and Wool, 1974). Rps6 phosphorylation is conserved from yeast to mammalian cells. In higher eukaryotes, there are five phosphorylation sites on its carboxyl terminus, whereas only two of these sites (Ser-232 and Ser-233), corresponding to Ser-235 and Ser-236 in higher eukaryotes, are conserved in yeast (Supplemental Figure S1A). Rps6 phosphorylation is sensitive to nutrients, hormones, growth factors, and a variety of stress conditions (Meyuhas, 2008). In higher eukaryotes, S6 kinases 1 and 2 (S6K1 and S6K2) are the major kinases phosphorylating Rps6 on several sites (S235/S236/S240/S244) in response to mTORC1 activity (Pende et al., 2004). Rps6 is additionally phosphorylated by RSKs (90-kDa ribosomal protein S6 kinases) on S235/S236 downstream of mitogen-activated protein kinase signaling (Roux et al., 2007). Members of casein kinase I kinase family were found to phosphorylate Rps6 on Ser-247 in response to mitogenic stimuli (Hutchinson et al., 2011). Intriguingly, both TORC1 and TORC2 regulate Rps6 phosphorylation in fission yeast, via Psk1 and Gad8, respectively (Du et al., 2012; Nakashima et al., 2012). Rps6 dephosphorylation is less well studied but, at least in mammals, appears to require the type 1 protein phosphatase (PP1) (Belandia et al., 1994; Hutchinson et al., 2011).

The major regulators of Rps6 phosphorylation in budding yeast remain unknown. Considering its role in translation and its ability to phosphorylate Rps6 in vitro, Sch9 was proposed to be the functional orthologue of S6 kinase in higher eukaryotes (Urban et al., 2007). However, phylogenetic analyses of the TOR signaling network has suggested that Ypk3, rather than Sch9, is more likely to be the S6 kinase orthologue in S. cerevisiae (van Dam et al., 2011). Consistently, YPK3 is orthologous to Schizosaccharomyces pombe PSK1. In addition, a recent study proposed that Ypk3 activity, but not Sch9 activity, is required for Rps6 phosphorylation in vivo in budding yeast (Gonzalez et al., 2015).

Despite years of research, the physiological role of Rps6 phosphorylation remains mysterious. Knock-in mice expressing only the alanine-substituted nonphosphorylatable version of Rps6 exhibited severe phenotypes, including a reduced size, glucose intolerance, and muscle weakness (Ruvinsky et al., 2005). Studies in yeast, however, suggested that Rps6 phosphorylation had no observable effect on the growth of cells under different conditions (Kruse et al., 1985; Johnson and Warner, 1987). Rps6 phosphorylation was once believed to specifically couple mTORC1 signals to the translation of so-called 5′-TOP mRNAs (Fumagalli, 2000), which predominantly encode proteins required for protein synthesis (Levy et al., 1991; Iadevaia et al., 2008). However, another mTORC1 effector, 4E-BP1, is now believed to serve in this capacity (Hsieh et al., 2012; Thoreen et al., 2012). More recently, Rps6 phosphorylation has again been linked to the translation machinery via regulation of the transcription of genes encoding ribosome biogenesis factors (Chauvin et al., 2014).

Here we revisited Rps6 phosphorylation in the model eukaryote S. cerevisiae. We found that Rps6 phosphorylation is differentially regulated downstream of TORC1 via Ypk3 and TORC2 via Ypk1 and Ypk2. TORC1 additionally appears to regulate Rps6 dephosphorylation via Glc7/Shp1. Cells expressing nonphosphorylatable variants of Rps6 presented phenotypes associated with 40S ribosome biogenesis defects, but these phenotypes were not recapitulated under conditions in which wild-type (wt) Rps6 protein is hypophosphorylated. Cells expressing nonphosphorylatable variants of Rps6 were partially resistant to the loss of Glc7 activity, suggesting that Rps6 hyperphosphorylation is toxic. Curiously, ribosome-footprinting analyses revealed that neither Rps6 phosphorylation nor Ypk1–3 activities are overtly involved in the regulation of gene expression at the level of transcription or translation and thus challenge the present models on the function of Rps6 phosphorylation derived from studies in higher eukaryotes.

RESULTSRps6 is differentially phosphorylated on Ser-232 and Ser-233 in a TORC1- and TORC2-dependent mannerRps6 is encoded by two paralogues, RPS6A and RPS6B, in S. cerevisiae, which encode for identical proteins, Rps6A and Rps6B. Both proteins are phosphorylated on Ser-232 and -233, and knock-ins targeting these phosphorylation sites were made in both RPS6A and RPS6B genes.To functionally characterize Rps6 phosphorylation, we initially generated antisera that recognize Rps6 proteins regardless of their phosphorylation status (Figure 1A). We also screened a number of commercially available antisera and found one (which we refer to as Rps6-PP) that cross-reacts with Rps6 when doubly phosphorylated on Ser-232 and -233 (Figure 1A). Immunoreactivity with this antiserum is lost if either or both of these serines are replaced with an alanine (Rps6AA, Rps6SA, or Rps6AS; Figure 1A). We additionally investigated immunoreactivity of Rps6 variants with antiserum recognizing phospho-RXXS*/T* motifs. Rps6SS and Rps6SA but not the Rps6AS variant are recognized by this antiserum (Figure 1A). This result implies that either the antiserum cannot recognize Rps6 phosphorylated on Ser-233 alone or that Ser-233 can only be phosphorylated after prior Ser-232 phosphorylation. We believe that the first hypothesis is correct, as we observe a slower SDS–PAGE migration of the Rps6AS variant compared with the Rps6AA variant (Supplemental Figure S1, B and C).FIGURE 1:Rps6 phosphorylation is differentially regulated on Ser-232 and -233. (A) Western blot of denaturing total protein extracts prepared from yeast cells with the indicated genetic modifications. Membranes were probed with the following antibodies: rabbit anti–phospho-S6 ribosomal protein (S235/S236; Rps6-PP) or rabbit anti–phospho-Akt substrate (RXXS*/T*), together with guinea pig anti-Rps6. Binding of phosphospecific antibodies was revealed with IRDye 680CW–conjugated anti-rabbit immunoglobulin G (IgG; red signal). Binding of anti-Rps6 antibody was revealed with IRDye 800CW–conjugated anti-guinea pig IgG (green signal). (B, C) Exponentially growing wild-type cells were treated with 200 nM rapamycin or 100 μM BHS. After treatment, the cells were collected at the indicated time points. and protein extracts were processed as described. (D) Rps6 phosphorylation status over time after a shift from glucose-containing medium (SDGlc) to galactose-containing medium (SDGal). Glucose was added 60 min after the carbon downshift, and Rps6 rephosphorylation was followed over the subsequent 20 min. (E) Rps6 phosphorylation status over time after TORC2 inhibition (in cells expressing an Avo1-AID fusion protein treated with 100 μM indole-3-acetic acid [IAA]) and/or TORC1 inhibition with rapamycin. Sch9 and Ypk1 phosphorylations were additionally assessed as readouts of TORC1 and TORC2 activities, respectively.Using these antisera, we examined how Rps6 phosphorylation responds to TORC1 and TORC2 inhibition. Inhibition of TORC1 with either rapamycin (Figure 1B) or caffeine (Supplemental Figure S1D) triggered rapid dephosphorylation of Ser-233 but not Ser-232. In contrast, inhibition of both TORC1 and TORC2 with BHS triggered rapid dephosphorylation of both serines (Figure 1C). TORC1 (Urban et al., 2007) and TORC2 (Y. Bouza, M. Piccolis, and R. Loewith, unpublished data) are sensitive to carbon quality, and consistent with the BHS result, downshift in carbon quality also triggered rapid dephosphorylation of both serines (Figure 1D). Nitrogen depletion also triggered dephosphorylation of both serines, with slower but otherwise parallel kinetics (Supplemental Figure S1E). Collectively these observations suggested that TORC1 regulates phosphorylation of Ser-233, whereas TORC1 and/or TORC2 regulate phosphorylation of Ser-232.To distinguish between the two possibilities, we constructed a strain in which TORC2 activity could be inhibited upon addition of the plant hormone auxin, which triggered proteolysis of the essential TORC2 subunit Avo1 fused to an auxin-activated degron (Nishimura et al., 2009; Supplemental Figure S1F). Before this experiment, we first confirmed that TORC1 and TORC2 activities in wild-type cells were not affected by auxin treatment per se (Supplemental Figure S1G). Subsequently, we found that inhibition of TORC2 resulted in Ser-232 dephosphorylation only when TORC1 was additionally inhibited (Figure 1E). Thus we conclude that TORC1 signals promote phosphorylation of both Ser-232 and Ser-233, whereas TORC2 signals promote phosphorylation of Ser-232 (Figure 4D).TORC1 regulates Rps6 phosphorylation on Ser-233 via Ypk3We screened a panel of kinase deletion mutants (Bodenmiller et al., 2010) to identify the TORC1-effector kinase that phosphorylates Rps6. This screen demonstrated that Rps6 is hypophosphorylated on Ser-233 in ypk3Δ cells (Figure 2A and Supplemental Figure S2A). Using an analogue-sensitive, Ypk3-expressing strain, we also found that Ypk3 activity is required for phosphorylation of Rps6 on Ser-233 upon glucose repletion (Figure 2B). In contrast, inhibition of analogue-sensitive protein kinase A (tpk1as tpk2as tpk3as cells) did not trigger dephosphorylation of Rps6 (Supplemental Figure S2B), arguing against a role for this related AGC-family kinase in Rps6 phosphorylation.FIGURE 2:TORC1 regulates Rps6 phosphorylation on Ser-233 via Ypk3. (A) Rps6 phosphorylation on Ser-233 is impaired in cells lacking Ypk3 activity. Rps6 phosphorylation in ypk3 cells expressing an empty vector (–), YPK3-FLAG (WT), or kinase-dead ypk3-FLAG (kd) was assessed by Western blot. Hog1 protein levels were assessed as a loading control. (B) A ypk3as strain was grown in SD medium with 2% glucose, filtered, and resuspended in SD with 2% galactose. After 1 h of incubation at 30°C, the culture was split into two: one supplemented with 2% glucose and DMSO, and the other with 2% glucose and 500 nM 1NM-PP1 to inhibit Ypk3as activity. Rps6 phosphorylation status and Ypk3-FLAG and Hog1 protein levels were assessed at various time points before, during, and after carbon downshift as indicated. (C) The migration pattern of Ypk3-FLAG collected from exponentially growing cells expressing before and after treatment with 200 nM rapamycin was assessed by Western blot. (D) Ypk3-TAP interacts with Tor1. Protein extracts were prepared from vehicle- or rapamycin-treated (30 min) cells expressing or not Ypk3-TAP and/or HA3-Tor1. Ypk3 was immunoprecipitated using IgG Dynabeads, and associated proteins were detected by Western blot. (E) TORC1 phosphorylates Ypk3 in vitro. GST-Ypk3as purified from yeast was used as a substrate for TORC1 purified from yeast by immunoprecipitating Tco89-TAP. The combination of wortmannin (which inhibits TORC1) and 1NMPP1 (which inhibits GST-Ypk3as) was used to show the specificity of the kinase reaction.Because Ypk3 is an AGC-family kinase, we predicted that it could be a direct substrate of TORC1. Consistent with this hypothesis, we found that Ypk3 is hypophosphorylated upon TORC1 inhibition with rapamycin (Figure 2C), carbon downshift (Supplemental Figure S2C), or nitrogen starvation (Supplemental Figure S2D) and that Ypk3 coprecipitates TORC1 in a rapamycin-sensitive manner (Figure 2D). Ypk3 phosphorylation was not obviously affected upon TORC2 inhibition (Supplemental Figure S2E) but was decreased upon direct inhibition of analogue-sensitive Ypk3 with 1NM-PP1 (Figure 2B), suggesting that it autophosphorylates, which was confirmed by in vitro kinase assays (Figure 2E). Addition of TORC1 in the absence but not the presence of wortmannin further increased Ypk3 phosphorylation in vitro. Together, these observations strongly suggest that Ypk3 is a direct target of TORC1.TOR targets a highly conserved hydrophobic motif at the C-terminal end of AGC kinases. Based on the homology among Ypk1, Ypk2, and Ypk3, we predicted Ser-513 to be the residue phosphorylated within the hydrophobic motif of Ypk3 (Supplemental Figure S2F). Conversion of Ser-513 to Ala altered the SDS–PAGE mobility of Ypk3 but did not affect its catalytic activity (Supplemental Figure S2G). Substitution of this residue with glutamic acid and aspartic acid did not suppress the rapamycin-induced hypophosphorylation of Rps6 (unpublished data), suggesting that either these substitutions do not adequately mimic phospho–Ser-513 or that TORC1 regulates Ypk3 and/or Rps6 phosphorylation through additional mechanisms. To map other rapamycin-sensitive phosphorylation sites on Ypk3, we decided to immunoprecipitate Flag-tagged Ypk3 from untreated and rapamycin-treated cells and analyze the phosphorylation patterns by mass spectrometry. This analysis revealed three rapamycin-sensitive sites in the N-terminus (S86, S92, S94) and three in the C-terminus of the protein (S505, S517, S519; Supplemental Figure S2F). Alanine substitution of the N-terminal residues S86, S92, and S94 did not visibly alter the SDS–PAGE migration pattern or the activity of Ypk3 (Supplemental Figure S2G), and thus we decided to focus on the C-terminal phosphorylation sites. Intriguingly, alanine substitution of S505, S517, and S519 residues in combination with that of S513 (Ypk34A) greatly diminished the phosphorylation level of Rps6 nearly to the level observed in ypk3 cells (Supplemental Figure S2H). We substituted these four serines with aspartic acid to see whether the phosphomimetic Ypk3 could suppress Rps6 dephosphorylation induced by rapamycin. However, this was not the case (Supplemental Figure S2H), leading us to believe that Ypk3 is not the only factor through which TORC1 regulates Rps6 phosphorylation (see Discussion).TOR complexes 1 and 2 regulate Rps6 phosphorylation on Ser-232 via Ypk1/2/3Unlike Ser-233, we could still detect phosphorylation of Ser-232 in the absence of YPK3 (Figure 3A). Ypk1 and Ypk2 are bona fide substrates of TORC2 (Kamada et al., 2005; Niles et al., 2012) and are also closely related to Ypk3, suggesting that all three kinases could redundantly mediate Rps6 phosphorylation on Ser-232. Loss of Ypk1 and/or Ypk2 function did not affect the phosphorylation levels of Rps6 (Figure 3A and Supplemental Figure S3A). However, inhibition of Ypk1 and Ypk2 in combination with rapamycin treatment induced hypophosphorylation of Ser-232 (Supplemental Figure S3B). Next we combined the deletion of YPK3 with the deletion of YPK2 and an analogue-sensitive allele of YPK1. In the absence of the activities of all three kinases, both Ser-232 and Ser-233 were hypophosphorylated (Figure 3B). Rephosphorylation of Rps6 on both Ser-232 and Ser-233 upon refeeding of glucose also appeared to require the activity of all three Ypk kinases (Figure 3C), although we note that the contribution of Ypk2 to Rps6 phosphorylation appears to be minimal.FIGURE 3:TOR complexes 1 and 2 regulate Rps6 phosphorylation on Ser-232 via Ypk1–3. (A) Individual deletion of neither YPK1 nor YPK2 altered Rps6 phosphorylation. (B) Inhibition of Ypk1as with 1NM-PP1 in ypk2Δ ypk3Δ and ypk3Δ backgrounds triggered loss of Ser-232 phosphorylation. (C) Inhibition of Ypk1as and Ypk3as with 1NM-PP1 prevented Rps6 rephosphorylation upon glucose refeeding. (D) Ypk3 phosphorylates Rps6 in vitro. Rps6 in yeast ribosomes was phosphorylated by wild-type Ypk3-FLAG purified from exponentially growing cells. Rapamycin treatment before purification reduced Ypk3-FLAG activity. Kinase-dead (kd) or nonphosphorylatable Ypk3 variant (4A) also failed to phosphorylate Rps6 in vitro.Wild-type Ypk3 but not inactive variants phosphorylated Rps6 in vitro, strongly suggesting that Rps6 is a direct target of Ypk3 (Figure 3D). Purified ribosomes were used as the source of Rps6 in this assay, as recombinant Rps6 was extremely unstable in our hands. Curiously, we failed to observe phosphorylation of Rps6 by Ypk1 in a similar assay (Supplemental Figure S3C). This negative result prevents us from concluding that Ypk1 directly phosphorylates Rps6. We did, however, observe synthetic lethality in ypk1 and ypk3 double mutants (Supplemental Figure S3D). This observation provides genetic support to the idea that Ypk1 and Ypk3 share common targets.Glc7/Shp1 mediates TORC1/2-dependent phosphorylation of Rps6Protein phosphatase 1 was recently proposed to mediate dephosphorylation of Ser-247 in Rps6 in mammalian cells (Hutchinson et al., 2011). Thus we investigated the role of PP1 in Rps6 dephosphorylation in yeast, using a loss-of-function mutant of Glc7, the catalytic subunit of yeast PP1 (Baker et al., 1997). Expression of the glc7-127 allele, which contains two point mutations, K110A and K112A, displays defects in glycogen accumulation and elevated levels of phosphorylation of histone H3 (Baker et al., 1997), blocked the dephosphorylation of Rps6 in response to rapamycin and BHS treatments, and led to higher basal phosphorylation levels under normal conditions (Figure 4A). In yeast, calyculin A is a known inhibitor of PP1 and to a lesser extent PP2A (Hoon et al., 2008). Intriguingly, we found that strains expressing RPS6AA exhibited slight resistance to calyculin A treatment compared with the wild-type strains (Figure 4B). Calyculin A sensitivity of these cells could be partially rescued by the overexpression of GLC7, supporting the notion that Glc7 is the target of this drug (Supplemental Figure S4A). This observation linked Glc7 activity and Rps6 phosphorylation and suggested that constitutive hyperphosphorylation of Rps6 is toxic, although other, indirect mechanisms could explain the observed growth rescue.FIGURE 4:Glc7/Shp1 mediates TORC1/2-dependent dephosphorylation of Rps6. (A) Exponentially growing cells of the indicated genotypes were treated with 200 nM rapamycin or 100 μM BHS for the indicated times before analysis of Rps6 phosphorylation. The glc7-127 allele suppressed both the rapamycin- and the BHS-induced dephosphorylation of Rps6. (B) Strains of the indicated genotypes were diluted into medium containing drug vehicle (DMSO) or 5 μM calyculin A, and growth (optical density at 600 nM) was measured every 15 min for the following 16 h. (C) Like the glc7-127 allele, deletion of SHP1 also suppressed both the rapamycin- and the BHS-induced dephosphorylation of Rps6. (D) Cartoon depicting the TORC1- and TORC2-regulated signaling pathways governing Rps6 phosphorylation.Glc7 activity is regulated by numerous regulatory subunits. To identify the regulatory subunit involved in Rps6 dephosphorylation, we screened deletion mutants of genes encoding all known nonessential Glc7-interacting proteins for effects on Rps6 phosphorylation. This approach identified Shp1 as a protein required for the dephosphorylation of Rps6 upon rapamycin treatment (Supplemental Figure S4B). Deletion of SHP1 suppressed the effect of rapamycin and BHS on Rps6 phosphorylation to an extent comparable to that of the glc7-127 allele (Figure 4C). These results functionally link Shp1 and Glc7 in the regulation of Rps6 dephosphorylation downstream of TORC1 and TORC2. The kinase and phosphatase pathways governing Rps6 phosphorylation described here are summarized in Figure 4D.Physiological characterization of Rps6 phosphomutantsTo identify the physiological significance of Rps6 phosphorylation, we first compared the growth rates of cells expressing either wild-type Rps6 protein or the nonphosphorylatable variant Rps6AA. In our hands, expression of the RPS6AA alleles resulted in an almost 30% decrease in the proliferation rate (Figure 5A and Supplemental Figure S5A). The RPS6AA cells also had lower protein content than the wild-type cells as assessed by fluorescein isothiocyanate (FITC) staining and fluorescence-activated cell sorting (FACS) analysis (Figure 5B and Supplemental Figure S5B).FIGURE 5:Physiological characterization of Rps6 phosphorylation. (A) Growth (OD600) of wt and RPS6AA cells in rich medium at 30°C was recorded every 15min for 16 h. (B) The protein content of exponentially growing wt and RPS6AA cells was measured by FITC staining and FACS analysis. (C) Polysome profiles of wt and RPS6AA cells. Cells were grown to exponential growth phase in SD medium containing 2% glucose. After filtering, cells were resuspended in SD medium lacking a carbon source and incubated at 30°C for 1 h. Subsequently, cells were refed with 2% glucose. Aliquots of cells were treated with cycloheximide before and after 60 min of starvation and after 20 min of refeeding. Polysome profiles from these three time points are shown. (D) Ribosomes extracted from wt and RPS6AA cells were incubated in the presence of 800 mM KCl. Free 40S and 60S subunits were analyzed by sucrose gradient centrifugation. (E) The relative amount of Rps6 normalized to Rpl3 in protein extracts prepared from exponentially growing wt and RPS6AA cells was assessed by Western blot. (F) ypk1as ypk2 ypk3as cells starved of a carbon source for 60 min were subsequently refed with glucose in the presence of DMSO or 1NM-PP1. Twenty minutes later, cells were treated with cycloheximide, and polysome profiles were generated.To explore the effect of Rps6 phosphorylation on global translation, we monitored the abundance of free 40S and 60S subunits, 80S monosomes, and actively translating ribosomes (polysomes) in wild-type and RPS6AA cells using sucrose gradient centrifugation. The ratios of polysomes to 80S monosomes were the same in these two strains (Figure 5C). However, the mutant cells exhibited an elevated 60S peak, indicating an imbalance between the free subunits. Next we analyzed the polysome profiles of wild-type and RPS6AA cells upon glucose starvation and refeeding with glucose. Although the RPS6AA cells displayed an increase in the 60S peak in all of the conditions we tested, we did not detect other overt changes in the profiles compared with the wild-type cells under these conditions (Figure 5C and Supplemental Figure S5C).To further investigate whether the elevated 60S peak is due to a defect in the production of the small subunit, we dissociated the monosomes and polysomes into free subunits in the presence of high salt. We observed that the level of 40S subunits was indeed lower in the mutant strain than in the wild type strain (Figure 5D). Quantitative Western blots confirmed that the ratio of Rps6 to a large-subunit protein, Rpl3, is reduced in RPS6AA cells (Figure 5E and Supplemental Figure S5D). Furthermore, by Northern blot, we observed a delay in rRNA maturation in RPS6AA cells (Supplemental Figure 5E). In summary, RPS6AA cells display a 40S biogenesis defect but no obvious defect in global translation. These phenotypes are similar to what is observed in the absence of one of the two copies of the RPS6 genes (Pachler et al., 2004), and thus it is unclear whether the 40S biogenesis defect results from Rps6 hypophosphorylation or instead is due to reduced levels of Rps6AA protein (see Discussion).To probe for a role of Ypk kinases in translation, we compared polysome profiles generated from wt cells to those generated from cells lacking Ypk1–3 activities. Because chronic inhibition of these kinases was not an option (the cells die), we chose to use an experimental setup in which we acutely activate translation in the presence or absence of Ypk1–3 activity. Specifically, ypk1as, ypk2Δ, ypk3 as cells were starved of glucose for 1 h and then refed with glucose with or without the analogue (1NM-PP1). Analysis of the abundance of the free subunits (40S and 60S), the monosomes (80S), and the polysomes collected after refeeding with or without 1NM-PP1 revealed that acute inhibition of Ypks had no significant effect on the profiles under any of the conditions tested despite the expectant hypophosphorylation of Rps6 (Figure 5F and Supplemental Figure S5, F and G). These results are in accordance with a recently published study, which showed that the polysome profiles from wild-type and S6K1;S6K2−/− liver cells exhibited no significant differences under starvation or refeeding conditions (Chauvin et al., 2014). Unlike RPS6AA cells, ypk3 cells displayed no defects in proliferation, protein accumulation, or 40S biogenesis (Supplemental Figure 5E; unpublished data).Our polysome profiling results thus far demonstrate that Rps6 phosphorylation plays no role in global translation but do not query for a potential role in the translation of individual messages. To probe for such a role, we performed ribosome profiling on ypk1as ypk2 ypk3as RPS6wt and ypk1as ypk2 ypk3as RPS6AA cells with or without 1NM-PP1 (protocol in Supplemental Figure S6A). Specifically, we compared the profiles of these cells after 20 and 40 min of refeeding with glucose. Under these conditions, we observed expected patterns of Rps6, Ypk1, and Ypk3 phosphorylation (Supplemental Figure S6B). The majority of ribosome footprints were 28–32 nucleotides (nt) long, and metagene analysis showed the expected triplet periodicity within coding regions (Supplemental Figure S6, C and D). For both RNA sequencing (RNA-seq) and ribo-seq reads, the ribosome profiling data exhibited good correlations for coverage between the replicas (Supplemental Figure S6E). We observed no strain-dependent differences in global mRNA levels (RNA-seq) or global translational efficiencies (TEs; Figure 6A), demonstrating that, under these conditions, Rps6 phosphorylation does not strongly affect gene regulation at the level of either transcription or translation. To identify potential differential gene expression, we used Z-score transformation as described earlier (Andreev et al., 2015). Only five (SED1, YCR007C, RPS21A, UIP5, Q0050) of 6692 transcripts displayed TE levels with absolute Z-scores ≥3 relative to the wild type in both Rps6AA and Ypk-inhibited cells (Figure 6B and Supplemental Figure S7). We note that five is close to the expected number of transcripts to be selected by chance with this Z-score cutoff. Indeed, the profiles of SED1 and UIP5 when assessed manually did not display obvious differences between conditions (Figure 7A). We also assessed in detail the transcript profiles of a number of known housekeeping genes (e.g., ACT1), as well as ribosomal protein genes (e.g., RPS6B) and Ribi genes (e.g., TEF4) whose expressions are known to be strongly dependent on TORC1 activity (Lippman and Broach, 2009; Huber et al., 2011). None of the selected transcripts exhibited a striking change in profile across the different conditions (Figure 7A). Neither Rps6AA mutation nor Ypk activity affected translational efficiency of transcripts possessing introns (e.g., ACT1, RPS6B, TEF4;
Figure 7A) and/or transcripts whose translation is regulated by upstream open reading frames (ORFs; e.g., GCN4). These observations do not support a role of Rps6 phosphorylation in the pioneer round of translation and/or in reinitiation.FIGURE 6:Differential gene expression analysis. (A) Scatter plots showing Z-scores (y-axis) of fold changes relative to wild-type control of ribosome occupancy (Riboseq), transcript levels (RNAseq), and translation efficiency (TE). Transcripts are ordered based on their log2 minimal expression signal (x-axis). Transcripts with |Z| ≥ 3 are shown in red. (B) Venn diagrams showing the number of transcripts whose changes in ribosome occupancy, transcript levels, or TE levels exceed a |Z| ≥ 3 threshold vs. wild-type controls in the indicated conditions.FIGURE 7:Alternative translational processes occur independently of Rps6 phosphorylation status. (A) Ribo-seq and RNA-seq densities obtained from the six different conditions/time points mapped to genomic loci of individual genes. Ribo-seq densities are indicated with different colors, depending on the condition (indicated at the bottom). Combined RNA-seq densities of all six conditions are shown in gray. Locations of coding regions are indicated with black bars under each profile, introns with black lines, and upstream open reading frames with red bars. Genomic coordinates are given for the sense strand relative to the start codons, with the first nucleotide of the start codons as the zero coordinate. (B) The number of footprints mapped to 5′ leaders and 3′ UTRs normalized to the number of footprints mapped to coding sequences (CDS) are shown for each Ribo-seq replicate. (C) Analysis of ribosome footprint densities on mRNAs for the genes that require ribosomal frameshifting for their expression. Footprint densities are shown separately for all conditions and two replicas. The ORF plot for each gene is shown below the density plots, ORF1 is highlighted in blue, and a nonoverlapping part of ORF2 is highlighted in yellow. The total number of reads aligning to each of the two areas is indicated on the right side of each plot, along with footprint density within ORF2 relative to ORF1. Histograms above represent periodicity within both ORFs. It can be seen that the phase of periodicity shifts according to the expected +1 directionality of ribosomal frameshifting.Next we compared ribosome occupancy of 5′ leaders and 3′ untranslated regions (UTRs) across the different conditions. The number of footprints mapping to 3′ UTRs remained constantly low regardless of the phosphorylation status of Rps6 (Figure 7B). At 20 min, the number of footprints mapping to 5′ leaders was lower in RPS6AA cells in both replicas; however, the difference was not higher than replicate variation. We hypothesized that the changes in 5′ leader densities may occur in specific mRNAs and analyzed the changes in the relative density between 5′ leaders and coding regions. The observed changes between different conditions did not exceed those observed for the replicas obtained for the same conditions (Supplemental Figure S7B). A similar, less robust trend was observed in Ypk-inhibited cells.Finally, we explored a possible role for Rps6 phosphorylation in frameshifting. During ribosomal frameshifting, a proportion of ribosomes shift their reading frame to produce a protein product that is encoded by two ORFs. The ribosome density profiles for such mRNAs are normally characterized by a drop of density downstream of ORF1 and a change in the phase of the triplet periodicity between nonoverlapping regions of ORF1 and ORF2, which is consistent with the direction of frameshifting (Michel et al., 2012). The drop of density should correlate with frameshifting efficiency and thus could be used to assess differences in frameshifting efficiencies between samples.To obtain information on S. cerevisiae genes that require programmed ribosomal frameshifting for their expression (ABP140, EST3, OAZ1, and TY1 transposable element), we used the Recode-2 database (Bekaert et al., 2010). The EST3 mRNA had insufficient coverage for the analysis. Profiles of ribosome density for ABP140, OAZ1, and two arbitrary chosen copies of TY1 element are shown in Figure 7C. Because the ambiguous alignments were used for frameshifting analysis (see Materials and Methods), profiles for TY1 elements represent mRNA translation of several TY1 copies that share strong sequence similarity.We observed a change of triplet periodicity consistent with the known directionality of frameshifting in all cases; however, the observed variation in the drop of ribosome density was insignificant among various samples in comparison to the variation observed between replicas (Figure 7C).In summary, consistent with previous work (Chauvin et al., 2014), we found that Rps6 phosphorylation plays no role in global protein synthesis upon refeeding as assessed by polysome profiling. To probe for potential affects on the translation of individual transcripts, we exploited a recent technological advance in the translation field—ribosome profiling. Specifically, using ribosome profiling, we assessed changes in RNA levels and translation efficiencies of individual mRNAs at the scale of the whole transcriptome (Supplemental Figure S7) under conditions in which Rps6 was or was not phosphorylated. We used a relaxed threshold of statistical significance to call for differential expression (|Z|-score ≥ 3), for which 0.25% of analyzed genes are expected to be classified as differentially expressed by chance under the assumption that changes in coverage due to technical factors follow a normal distribution. Even under these relaxed thresholds, we failed to identify differentially expressed genes that would be shared between Rps6 mutants and Ypk-inhibited cells at all time points (Figure 6B).

Rps6 is differentially phosphorylated on Ser-232 and Ser-233 in a TORC1- and TORC2-dependent mannerRps6 is encoded by two paralogues, RPS6A and RPS6B, in S. cerevisiae, which encode for identical proteins, Rps6A and Rps6B. Both proteins are phosphorylated on Ser-232 and -233, and knock-ins targeting these phosphorylation sites were made in both RPS6A and RPS6B genes.To functionally characterize Rps6 phosphorylation, we initially generated antisera that recognize Rps6 proteins regardless of their phosphorylation status (Figure 1A). We also screened a number of commercially available antisera and found one (which we refer to as Rps6-PP) that cross-reacts with Rps6 when doubly phosphorylated on Ser-232 and -233 (Figure 1A). Immunoreactivity with this antiserum is lost if either or both of these serines are replaced with an alanine (Rps6AA, Rps6SA, or Rps6AS; Figure 1A). We additionally investigated immunoreactivity of Rps6 variants with antiserum recognizing phospho-RXXS*/T* motifs. Rps6SS and Rps6SA but not the Rps6AS variant are recognized by this antiserum (Figure 1A). This result implies that either the antiserum cannot recognize Rps6 phosphorylated on Ser-233 alone or that Ser-233 can only be phosphorylated after prior Ser-232 phosphorylation. We believe that the first hypothesis is correct, as we observe a slower SDS–PAGE migration of the Rps6AS variant compared with the Rps6AA variant (Supplemental Figure S1, B and C).FIGURE 1:Rps6 phosphorylation is differentially regulated on Ser-232 and -233. (A) Western blot of denaturing total protein extracts prepared from yeast cells with the indicated genetic modifications. Membranes were probed with the following antibodies: rabbit anti–phospho-S6 ribosomal protein (S235/S236; Rps6-PP) or rabbit anti–phospho-Akt substrate (RXXS*/T*), together with guinea pig anti-Rps6. Binding of phosphospecific antibodies was revealed with IRDye 680CW–conjugated anti-rabbit immunoglobulin G (IgG; red signal). Binding of anti-Rps6 antibody was revealed with IRDye 800CW–conjugated anti-guinea pig IgG (green signal). (B, C) Exponentially growing wild-type cells were treated with 200 nM rapamycin or 100 μM BHS. After treatment, the cells were collected at the indicated time points. and protein extracts were processed as described. (D) Rps6 phosphorylation status over time after a shift from glucose-containing medium (SDGlc) to galactose-containing medium (SDGal). Glucose was added 60 min after the carbon downshift, and Rps6 rephosphorylation was followed over the subsequent 20 min. (E) Rps6 phosphorylation status over time after TORC2 inhibition (in cells expressing an Avo1-AID fusion protein treated with 100 μM indole-3-acetic acid [IAA]) and/or TORC1 inhibition with rapamycin. Sch9 and Ypk1 phosphorylations were additionally assessed as readouts of TORC1 and TORC2 activities, respectively.Using these antisera, we examined how Rps6 phosphorylation responds to TORC1 and TORC2 inhibition. Inhibition of TORC1 with either rapamycin (Figure 1B) or caffeine (Supplemental Figure S1D) triggered rapid dephosphorylation of Ser-233 but not Ser-232. In contrast, inhibition of both TORC1 and TORC2 with BHS triggered rapid dephosphorylation of both serines (Figure 1C). TORC1 (Urban et al., 2007) and TORC2 (Y. Bouza, M. Piccolis, and R. Loewith, unpublished data) are sensitive to carbon quality, and consistent with the BHS result, downshift in carbon quality also triggered rapid dephosphorylation of both serines (Figure 1D). Nitrogen depletion also triggered dephosphorylation of both serines, with slower but otherwise parallel kinetics (Supplemental Figure S1E). Collectively these observations suggested that TORC1 regulates phosphorylation of Ser-233, whereas TORC1 and/or TORC2 regulate phosphorylation of Ser-232.To distinguish between the two possibilities, we constructed a strain in which TORC2 activity could be inhibited upon addition of the plant hormone auxin, which triggered proteolysis of the essential TORC2 subunit Avo1 fused to an auxin-activated degron (Nishimura et al., 2009; Supplemental Figure S1F). Before this experiment, we first confirmed that TORC1 and TORC2 activities in wild-type cells were not affected by auxin treatment per se (Supplemental Figure S1G). Subsequently, we found that inhibition of TORC2 resulted in Ser-232 dephosphorylation only when TORC1 was additionally inhibited (Figure 1E). Thus we conclude that TORC1 signals promote phosphorylation of both Ser-232 and Ser-233, whereas TORC2 signals promote phosphorylation of Ser-232 (Figure 4D).

Rps6 is differentially phosphorylated on Ser-232 and Ser-233 in a TORC1- and TORC2-dependent manner

Rps6 is encoded by two paralogues, RPS6A and RPS6B, in S. cerevisiae, which encode for identical proteins, Rps6A and Rps6B. Both proteins are phosphorylated on Ser-232 and -233, and knock-ins targeting these phosphorylation sites were made in both RPS6A and RPS6B genes.

To functionally characterize Rps6 phosphorylation, we initially generated antisera that recognize Rps6 proteins regardless of their phosphorylation status (Figure 1A). We also screened a number of commercially available antisera and found one (which we refer to as Rps6-PP) that cross-reacts with Rps6 when doubly phosphorylated on Ser-232 and -233 (Figure 1A). Immunoreactivity with this antiserum is lost if either or both of these serines are replaced with an alanine (Rps6AA, Rps6SA, or Rps6AS; Figure 1A). We additionally investigated immunoreactivity of Rps6 variants with antiserum recognizing phospho-RXXS*/T* motifs. Rps6SS and Rps6SA but not the Rps6AS variant are recognized by this antiserum (Figure 1A). This result implies that either the antiserum cannot recognize Rps6 phosphorylated on Ser-233 alone or that Ser-233 can only be phosphorylated after prior Ser-232 phosphorylation. We believe that the first hypothesis is correct, as we observe a slower SDS–PAGE migration of the Rps6AS variant compared with the Rps6AA variant (Supplemental Figure S1, B and C).

Rps6 phosphorylation is differentially regulated on Ser-232 and -233. (A) Western blot of denaturing total protein extracts prepared from yeast cells with the indicated genetic modifications. Membranes were probed with the following antibodies: rabbit anti–phospho-S6 ribosomal protein (S235/S236; Rps6-PP) or rabbit anti–phospho-Akt substrate (RXXS*/T*), together with guinea pig anti-Rps6. Binding of phosphospecific antibodies was revealed with IRDye 680CW–conjugated anti-rabbit immunoglobulin G (IgG; red signal). Binding of anti-Rps6 antibody was revealed with IRDye 800CW–conjugated anti-guinea pig IgG (green signal). (B, C) Exponentially growing wild-type cells were treated with 200 nM rapamycin or 100 μM BHS. After treatment, the cells were collected at the indicated time points. and protein extracts were processed as described. (D) Rps6 phosphorylation status over time after a shift from glucose-containing medium (SDGlc) to galactose-containing medium (SDGal). Glucose was added 60 min after the carbon downshift, and Rps6 rephosphorylation was followed over the subsequent 20 min. (E) Rps6 phosphorylation status over time after TORC2 inhibition (in cells expressing an Avo1-AID fusion protein treated with 100 μM indole-3-acetic acid [IAA]) and/or TORC1 inhibition with rapamycin. Sch9 and Ypk1 phosphorylations were additionally assessed as readouts of TORC1 and TORC2 activities, respectively.

Using these antisera, we examined how Rps6 phosphorylation responds to TORC1 and TORC2 inhibition. Inhibition of TORC1 with either rapamycin (Figure 1B) or caffeine (Supplemental Figure S1D) triggered rapid dephosphorylation of Ser-233 but not Ser-232. In contrast, inhibition of both TORC1 and TORC2 with BHS triggered rapid dephosphorylation of both serines (Figure 1C). TORC1 (Urban et al., 2007) and TORC2 (Y. Bouza, M. Piccolis, and R. Loewith, unpublished data) are sensitive to carbon quality, and consistent with the BHS result, downshift in carbon quality also triggered rapid dephosphorylation of both serines (Figure 1D). Nitrogen depletion also triggered dephosphorylation of both serines, with slower but otherwise parallel kinetics (Supplemental Figure S1E). Collectively these observations suggested that TORC1 regulates phosphorylation of Ser-233, whereas TORC1 and/or TORC2 regulate phosphorylation of Ser-232.

To distinguish between the two possibilities, we constructed a strain in which TORC2 activity could be inhibited upon addition of the plant hormone auxin, which triggered proteolysis of the essential TORC2 subunit Avo1 fused to an auxin-activated degron (Nishimura et al., 2009; Supplemental Figure S1F). Before this experiment, we first confirmed that TORC1 and TORC2 activities in wild-type cells were not affected by auxin treatment per se (Supplemental Figure S1G). Subsequently, we found that inhibition of TORC2 resulted in Ser-232 dephosphorylation only when TORC1 was additionally inhibited (Figure 1E). Thus we conclude that TORC1 signals promote phosphorylation of both Ser-232 and Ser-233, whereas TORC2 signals promote phosphorylation of Ser-232 (Figure 4D).

TORC1 regulates Rps6 phosphorylation on Ser-233 via Ypk3We screened a panel of kinase deletion mutants (Bodenmiller et al., 2010) to identify the TORC1-effector kinase that phosphorylates Rps6. This screen demonstrated that Rps6 is hypophosphorylated on Ser-233 in ypk3Δ cells (Figure 2A and Supplemental Figure S2A). Using an analogue-sensitive, Ypk3-expressing strain, we also found that Ypk3 activity is required for phosphorylation of Rps6 on Ser-233 upon glucose repletion (Figure 2B). In contrast, inhibition of analogue-sensitive protein kinase A (tpk1as tpk2as tpk3as cells) did not trigger dephosphorylation of Rps6 (Supplemental Figure S2B), arguing against a role for this related AGC-family kinase in Rps6 phosphorylation.FIGURE 2:TORC1 regulates Rps6 phosphorylation on Ser-233 via Ypk3. (A) Rps6 phosphorylation on Ser-233 is impaired in cells lacking Ypk3 activity. Rps6 phosphorylation in ypk3 cells expressing an empty vector (–), YPK3-FLAG (WT), or kinase-dead ypk3-FLAG (kd) was assessed by Western blot. Hog1 protein levels were assessed as a loading control. (B) A ypk3as strain was grown in SD medium with 2% glucose, filtered, and resuspended in SD with 2% galactose. After 1 h of incubation at 30°C, the culture was split into two: one supplemented with 2% glucose and DMSO, and the other with 2% glucose and 500 nM 1NM-PP1 to inhibit Ypk3as activity. Rps6 phosphorylation status and Ypk3-FLAG and Hog1 protein levels were assessed at various time points before, during, and after carbon downshift as indicated. (C) The migration pattern of Ypk3-FLAG collected from exponentially growing cells expressing before and after treatment with 200 nM rapamycin was assessed by Western blot. (D) Ypk3-TAP interacts with Tor1. Protein extracts were prepared from vehicle- or rapamycin-treated (30 min) cells expressing or not Ypk3-TAP and/or HA3-Tor1. Ypk3 was immunoprecipitated using IgG Dynabeads, and associated proteins were detected by Western blot. (E) TORC1 phosphorylates Ypk3 in vitro. GST-Ypk3as purified from yeast was used as a substrate for TORC1 purified from yeast by immunoprecipitating Tco89-TAP. The combination of wortmannin (which inhibits TORC1) and 1NMPP1 (which inhibits GST-Ypk3as) was used to show the specificity of the kinase reaction.Because Ypk3 is an AGC-family kinase, we predicted that it could be a direct substrate of TORC1. Consistent with this hypothesis, we found that Ypk3 is hypophosphorylated upon TORC1 inhibition with rapamycin (Figure 2C), carbon downshift (Supplemental Figure S2C), or nitrogen starvation (Supplemental Figure S2D) and that Ypk3 coprecipitates TORC1 in a rapamycin-sensitive manner (Figure 2D). Ypk3 phosphorylation was not obviously affected upon TORC2 inhibition (Supplemental Figure S2E) but was decreased upon direct inhibition of analogue-sensitive Ypk3 with 1NM-PP1 (Figure 2B), suggesting that it autophosphorylates, which was confirmed by in vitro kinase assays (Figure 2E). Addition of TORC1 in the absence but not the presence of wortmannin further increased Ypk3 phosphorylation in vitro. Together, these observations strongly suggest that Ypk3 is a direct target of TORC1.TOR targets a highly conserved hydrophobic motif at the C-terminal end of AGC kinases. Based on the homology among Ypk1, Ypk2, and Ypk3, we predicted Ser-513 to be the residue phosphorylated within the hydrophobic motif of Ypk3 (Supplemental Figure S2F). Conversion of Ser-513 to Ala altered the SDS–PAGE mobility of Ypk3 but did not affect its catalytic activity (Supplemental Figure S2G). Substitution of this residue with glutamic acid and aspartic acid did not suppress the rapamycin-induced hypophosphorylation of Rps6 (unpublished data), suggesting that either these substitutions do not adequately mimic phospho–Ser-513 or that TORC1 regulates Ypk3 and/or Rps6 phosphorylation through additional mechanisms. To map other rapamycin-sensitive phosphorylation sites on Ypk3, we decided to immunoprecipitate Flag-tagged Ypk3 from untreated and rapamycin-treated cells and analyze the phosphorylation patterns by mass spectrometry. This analysis revealed three rapamycin-sensitive sites in the N-terminus (S86, S92, S94) and three in the C-terminus of the protein (S505, S517, S519; Supplemental Figure S2F). Alanine substitution of the N-terminal residues S86, S92, and S94 did not visibly alter the SDS–PAGE migration pattern or the activity of Ypk3 (Supplemental Figure S2G), and thus we decided to focus on the C-terminal phosphorylation sites. Intriguingly, alanine substitution of S505, S517, and S519 residues in combination with that of S513 (Ypk34A) greatly diminished the phosphorylation level of Rps6 nearly to the level observed in ypk3 cells (Supplemental Figure S2H). We substituted these four serines with aspartic acid to see whether the phosphomimetic Ypk3 could suppress Rps6 dephosphorylation induced by rapamycin. However, this was not the case (Supplemental Figure S2H), leading us to believe that Ypk3 is not the only factor through which TORC1 regulates Rps6 phosphorylation (see Discussion).

TORC1 regulates Rps6 phosphorylation on Ser-233 via Ypk3

We screened a panel of kinase deletion mutants (Bodenmiller et al., 2010) to identify the TORC1-effector kinase that phosphorylates Rps6. This screen demonstrated that Rps6 is hypophosphorylated on Ser-233 in ypk3Δ cells (Figure 2A and Supplemental Figure S2A). Using an analogue-sensitive, Ypk3-expressing strain, we also found that Ypk3 activity is required for phosphorylation of Rps6 on Ser-233 upon glucose repletion (Figure 2B). In contrast, inhibition of analogue-sensitive protein kinase A (tpk1as tpk2as tpk3as cells) did not trigger dephosphorylation of Rps6 (Supplemental Figure S2B), arguing against a role for this related AGC-family kinase in Rps6 phosphorylation.

TORC1 regulates Rps6 phosphorylation on Ser-233 via Ypk3. (A) Rps6 phosphorylation on Ser-233 is impaired in cells lacking Ypk3 activity. Rps6 phosphorylation in ypk3 cells expressing an empty vector (–), YPK3-FLAG (WT), or kinase-dead ypk3-FLAG (kd) was assessed by Western blot. Hog1 protein levels were assessed as a loading control. (B) A ypk3as strain was grown in SD medium with 2% glucose, filtered, and resuspended in SD with 2% galactose. After 1 h of incubation at 30°C, the culture was split into two: one supplemented with 2% glucose and DMSO, and the other with 2% glucose and 500 nM 1NM-PP1 to inhibit Ypk3as activity. Rps6 phosphorylation status and Ypk3-FLAG and Hog1 protein levels were assessed at various time points before, during, and after carbon downshift as indicated. (C) The migration pattern of Ypk3-FLAG collected from exponentially growing cells expressing before and after treatment with 200 nM rapamycin was assessed by Western blot. (D) Ypk3-TAP interacts with Tor1. Protein extracts were prepared from vehicle- or rapamycin-treated (30 min) cells expressing or not Ypk3-TAP and/or HA3-Tor1. Ypk3 was immunoprecipitated using IgG Dynabeads, and associated proteins were detected by Western blot. (E) TORC1 phosphorylates Ypk3 in vitro. GST-Ypk3as purified from yeast was used as a substrate for TORC1 purified from yeast by immunoprecipitating Tco89-TAP. The combination of wortmannin (which inhibits TORC1) and 1NMPP1 (which inhibits GST-Ypk3as) was used to show the specificity of the kinase reaction.

Because Ypk3 is an AGC-family kinase, we predicted that it could be a direct substrate of TORC1. Consistent with this hypothesis, we found that Ypk3 is hypophosphorylated upon TORC1 inhibition with rapamycin (Figure 2C), carbon downshift (Supplemental Figure S2C), or nitrogen starvation (Supplemental Figure S2D) and that Ypk3 coprecipitates TORC1 in a rapamycin-sensitive manner (Figure 2D). Ypk3 phosphorylation was not obviously affected upon TORC2 inhibition (Supplemental Figure S2E) but was decreased upon direct inhibition of analogue-sensitive Ypk3 with 1NM-PP1 (Figure 2B), suggesting that it autophosphorylates, which was confirmed by in vitro kinase assays (Figure 2E). Addition of TORC1 in the absence but not the presence of wortmannin further increased Ypk3 phosphorylation in vitro. Together, these observations strongly suggest that Ypk3 is a direct target of TORC1.

TOR targets a highly conserved hydrophobic motif at the C-terminal end of AGC kinases. Based on the homology among Ypk1, Ypk2, and Ypk3, we predicted Ser-513 to be the residue phosphorylated within the hydrophobic motif of Ypk3 (Supplemental Figure S2F). Conversion of Ser-513 to Ala altered the SDS–PAGE mobility of Ypk3 but did not affect its catalytic activity (Supplemental Figure S2G). Substitution of this residue with glutamic acid and aspartic acid did not suppress the rapamycin-induced hypophosphorylation of Rps6 (unpublished data), suggesting that either these substitutions do not adequately mimic phospho–Ser-513 or that TORC1 regulates Ypk3 and/or Rps6 phosphorylation through additional mechanisms. To map other rapamycin-sensitive phosphorylation sites on Ypk3, we decided to immunoprecipitate Flag-tagged Ypk3 from untreated and rapamycin-treated cells and analyze the phosphorylation patterns by mass spectrometry. This analysis revealed three rapamycin-sensitive sites in the N-terminus (S86, S92, S94) and three in the C-terminus of the protein (S505, S517, S519; Supplemental Figure S2F). Alanine substitution of the N-terminal residues S86, S92, and S94 did not visibly alter the SDS–PAGE migration pattern or the activity of Ypk3 (Supplemental Figure S2G), and thus we decided to focus on the C-terminal phosphorylation sites. Intriguingly, alanine substitution of S505, S517, and S519 residues in combination with that of S513 (Ypk34A) greatly diminished the phosphorylation level of Rps6 nearly to the level observed in ypk3 cells (Supplemental Figure S2H). We substituted these four serines with aspartic acid to see whether the phosphomimetic Ypk3 could suppress Rps6 dephosphorylation induced by rapamycin. However, this was not the case (Supplemental Figure S2H), leading us to believe that Ypk3 is not the only factor through which TORC1 regulates Rps6 phosphorylation (see Discussion).

TOR complexes 1 and 2 regulate Rps6 phosphorylation on Ser-232 via Ypk1/2/3Unlike Ser-233, we could still detect phosphorylation of Ser-232 in the absence of YPK3 (Figure 3A). Ypk1 and Ypk2 are bona fide substrates of TORC2 (Kamada et al., 2005; Niles et al., 2012) and are also closely related to Ypk3, suggesting that all three kinases could redundantly mediate Rps6 phosphorylation on Ser-232. Loss of Ypk1 and/or Ypk2 function did not affect the phosphorylation levels of Rps6 (Figure 3A and Supplemental Figure S3A). However, inhibition of Ypk1 and Ypk2 in combination with rapamycin treatment induced hypophosphorylation of Ser-232 (Supplemental Figure S3B). Next we combined the deletion of YPK3 with the deletion of YPK2 and an analogue-sensitive allele of YPK1. In the absence of the activities of all three kinases, both Ser-232 and Ser-233 were hypophosphorylated (Figure 3B). Rephosphorylation of Rps6 on both Ser-232 and Ser-233 upon refeeding of glucose also appeared to require the activity of all three Ypk kinases (Figure 3C), although we note that the contribution of Ypk2 to Rps6 phosphorylation appears to be minimal.FIGURE 3:TOR complexes 1 and 2 regulate Rps6 phosphorylation on Ser-232 via Ypk1–3. (A) Individual deletion of neither YPK1 nor YPK2 altered Rps6 phosphorylation. (B) Inhibition of Ypk1as with 1NM-PP1 in ypk2Δ ypk3Δ and ypk3Δ backgrounds triggered loss of Ser-232 phosphorylation. (C) Inhibition of Ypk1as and Ypk3as with 1NM-PP1 prevented Rps6 rephosphorylation upon glucose refeeding. (D) Ypk3 phosphorylates Rps6 in vitro. Rps6 in yeast ribosomes was phosphorylated by wild-type Ypk3-FLAG purified from exponentially growing cells. Rapamycin treatment before purification reduced Ypk3-FLAG activity. Kinase-dead (kd) or nonphosphorylatable Ypk3 variant (4A) also failed to phosphorylate Rps6 in vitro.Wild-type Ypk3 but not inactive variants phosphorylated Rps6 in vitro, strongly suggesting that Rps6 is a direct target of Ypk3 (Figure 3D). Purified ribosomes were used as the source of Rps6 in this assay, as recombinant Rps6 was extremely unstable in our hands. Curiously, we failed to observe phosphorylation of Rps6 by Ypk1 in a similar assay (Supplemental Figure S3C). This negative result prevents us from concluding that Ypk1 directly phosphorylates Rps6. We did, however, observe synthetic lethality in ypk1 and ypk3 double mutants (Supplemental Figure S3D). This observation provides genetic support to the idea that Ypk1 and Ypk3 share common targets.

TOR complexes 1 and 2 regulate Rps6 phosphorylation on Ser-232 via Ypk1/2/3

Unlike Ser-233, we could still detect phosphorylation of Ser-232 in the absence of YPK3 (Figure 3A). Ypk1 and Ypk2 are bona fide substrates of TORC2 (Kamada et al., 2005; Niles et al., 2012) and are also closely related to Ypk3, suggesting that all three kinases could redundantly mediate Rps6 phosphorylation on Ser-232. Loss of Ypk1 and/or Ypk2 function did not affect the phosphorylation levels of Rps6 (Figure 3A and Supplemental Figure S3A). However, inhibition of Ypk1 and Ypk2 in combination with rapamycin treatment induced hypophosphorylation of Ser-232 (Supplemental Figure S3B). Next we combined the deletion of YPK3 with the deletion of YPK2 and an analogue-sensitive allele of YPK1. In the absence of the activities of all three kinases, both Ser-232 and Ser-233 were hypophosphorylated (Figure 3B). Rephosphorylation of Rps6 on both Ser-232 and Ser-233 upon refeeding of glucose also appeared to require the activity of all three Ypk kinases (Figure 3C), although we note that the contribution of Ypk2 to Rps6 phosphorylation appears to be minimal.

TOR complexes 1 and 2 regulate Rps6 phosphorylation on Ser-232 via Ypk1–3. (A) Individual deletion of neither YPK1 nor YPK2 altered Rps6 phosphorylation. (B) Inhibition of Ypk1as with 1NM-PP1 in ypk2Δ ypk3Δ and ypk3Δ backgrounds triggered loss of Ser-232 phosphorylation. (C) Inhibition of Ypk1as and Ypk3as with 1NM-PP1 prevented Rps6 rephosphorylation upon glucose refeeding. (D) Ypk3 phosphorylates Rps6 in vitro. Rps6 in yeast ribosomes was phosphorylated by wild-type Ypk3-FLAG purified from exponentially growing cells. Rapamycin treatment before purification reduced Ypk3-FLAG activity. Kinase-dead (kd) or nonphosphorylatable Ypk3 variant (4A) also failed to phosphorylate Rps6 in vitro.

Wild-type Ypk3 but not inactive variants phosphorylated Rps6 in vitro, strongly suggesting that Rps6 is a direct target of Ypk3 (Figure 3D). Purified ribosomes were used as the source of Rps6 in this assay, as recombinant Rps6 was extremely unstable in our hands. Curiously, we failed to observe phosphorylation of Rps6 by Ypk1 in a similar assay (Supplemental Figure S3C). This negative result prevents us from concluding that Ypk1 directly phosphorylates Rps6. We did, however, observe synthetic lethality in ypk1 and ypk3 double mutants (Supplemental Figure S3D). This observation provides genetic support to the idea that Ypk1 and Ypk3 share common targets.

Glc7/Shp1 mediates TORC1/2-dependent phosphorylation of Rps6Protein phosphatase 1 was recently proposed to mediate dephosphorylation of Ser-247 in Rps6 in mammalian cells (Hutchinson et al., 2011). Thus we investigated the role of PP1 in Rps6 dephosphorylation in yeast, using a loss-of-function mutant of Glc7, the catalytic subunit of yeast PP1 (Baker et al., 1997). Expression of the glc7-127 allele, which contains two point mutations, K110A and K112A, displays defects in glycogen accumulation and elevated levels of phosphorylation of histone H3 (Baker et al., 1997), blocked the dephosphorylation of Rps6 in response to rapamycin and BHS treatments, and led to higher basal phosphorylation levels under normal conditions (Figure 4A). In yeast, calyculin A is a known inhibitor of PP1 and to a lesser extent PP2A (Hoon et al., 2008). Intriguingly, we found that strains expressing RPS6AA exhibited slight resistance to calyculin A treatment compared with the wild-type strains (Figure 4B). Calyculin A sensitivity of these cells could be partially rescued by the overexpression of GLC7, supporting the notion that Glc7 is the target of this drug (Supplemental Figure S4A). This observation linked Glc7 activity and Rps6 phosphorylation and suggested that constitutive hyperphosphorylation of Rps6 is toxic, although other, indirect mechanisms could explain the observed growth rescue.FIGURE 4:Glc7/Shp1 mediates TORC1/2-dependent dephosphorylation of Rps6. (A) Exponentially growing cells of the indicated genotypes were treated with 200 nM rapamycin or 100 μM BHS for the indicated times before analysis of Rps6 phosphorylation. The glc7-127 allele suppressed both the rapamycin- and the BHS-induced dephosphorylation of Rps6. (B) Strains of the indicated genotypes were diluted into medium containing drug vehicle (DMSO) or 5 μM calyculin A, and growth (optical density at 600 nM) was measured every 15 min for the following 16 h. (C) Like the glc7-127 allele, deletion of SHP1 also suppressed both the rapamycin- and the BHS-induced dephosphorylation of Rps6. (D) Cartoon depicting the TORC1- and TORC2-regulated signaling pathways governing Rps6 phosphorylation.Glc7 activity is regulated by numerous regulatory subunits. To identify the regulatory subunit involved in Rps6 dephosphorylation, we screened deletion mutants of genes encoding all known nonessential Glc7-interacting proteins for effects on Rps6 phosphorylation. This approach identified Shp1 as a protein required for the dephosphorylation of Rps6 upon rapamycin treatment (Supplemental Figure S4B). Deletion of SHP1 suppressed the effect of rapamycin and BHS on Rps6 phosphorylation to an extent comparable to that of the glc7-127 allele (Figure 4C). These results functionally link Shp1 and Glc7 in the regulation of Rps6 dephosphorylation downstream of TORC1 and TORC2. The kinase and phosphatase pathways governing Rps6 phosphorylation described here are summarized in Figure 4D.

Glc7/Shp1 mediates TORC1/2-dependent phosphorylation of Rps6

Protein phosphatase 1 was recently proposed to mediate dephosphorylation of Ser-247 in Rps6 in mammalian cells (Hutchinson et al., 2011). Thus we investigated the role of PP1 in Rps6 dephosphorylation in yeast, using a loss-of-function mutant of Glc7, the catalytic subunit of yeast PP1 (Baker et al., 1997). Expression of the glc7-127 allele, which contains two point mutations, K110A and K112A, displays defects in glycogen accumulation and elevated levels of phosphorylation of histone H3 (Baker et al., 1997), blocked the dephosphorylation of Rps6 in response to rapamycin and BHS treatments, and led to higher basal phosphorylation levels under normal conditions (Figure 4A). In yeast, calyculin A is a known inhibitor of PP1 and to a lesser extent PP2A (Hoon et al., 2008). Intriguingly, we found that strains expressing RPS6AA exhibited slight resistance to calyculin A treatment compared with the wild-type strains (Figure 4B). Calyculin A sensitivity of these cells could be partially rescued by the overexpression of GLC7, supporting the notion that Glc7 is the target of this drug (Supplemental Figure S4A). This observation linked Glc7 activity and Rps6 phosphorylation and suggested that constitutive hyperphosphorylation of Rps6 is toxic, although other, indirect mechanisms could explain the observed growth rescue.

Glc7/Shp1 mediates TORC1/2-dependent dephosphorylation of Rps6. (A) Exponentially growing cells of the indicated genotypes were treated with 200 nM rapamycin or 100 μM BHS for the indicated times before analysis of Rps6 phosphorylation. The glc7-127 allele suppressed both the rapamycin- and the BHS-induced dephosphorylation of Rps6. (B) Strains of the indicated genotypes were diluted into medium containing drug vehicle (DMSO) or 5 μM calyculin A, and growth (optical density at 600 nM) was measured every 15 min for the following 16 h. (C) Like the glc7-127 allele, deletion of SHP1 also suppressed both the rapamycin- and the BHS-induced dephosphorylation of Rps6. (D) Cartoon depicting the TORC1- and TORC2-regulated signaling pathways governing Rps6 phosphorylation.

Glc7 activity is regulated by numerous regulatory subunits. To identify the regulatory subunit involved in Rps6 dephosphorylation, we screened deletion mutants of genes encoding all known nonessential Glc7-interacting proteins for effects on Rps6 phosphorylation. This approach identified Shp1 as a protein required for the dephosphorylation of Rps6 upon rapamycin treatment (Supplemental Figure S4B). Deletion of SHP1 suppressed the effect of rapamycin and BHS on Rps6 phosphorylation to an extent comparable to that of the glc7-127 allele (Figure 4C). These results functionally link Shp1 and Glc7 in the regulation of Rps6 dephosphorylation downstream of TORC1 and TORC2. The kinase and phosphatase pathways governing Rps6 phosphorylation described here are summarized in Figure 4D.

Physiological characterization of Rps6 phosphomutantsTo identify the physiological significance of Rps6 phosphorylation, we first compared the growth rates of cells expressing either wild-type Rps6 protein or the nonphosphorylatable variant Rps6AA. In our hands, expression of the RPS6AA alleles resulted in an almost 30% decrease in the proliferation rate (Figure 5A and Supplemental Figure S5A). The RPS6AA cells also had lower protein content than the wild-type cells as assessed by fluorescein isothiocyanate (FITC) staining and fluorescence-activated cell sorting (FACS) analysis (Figure 5B and Supplemental Figure S5B).FIGURE 5:Physiological characterization of Rps6 phosphorylation. (A) Growth (OD600) of wt and RPS6AA cells in rich medium at 30°C was recorded every 15min for 16 h. (B) The protein content of exponentially growing wt and RPS6AA cells was measured by FITC staining and FACS analysis. (C) Polysome profiles of wt and RPS6AA cells. Cells were grown to exponential growth phase in SD medium containing 2% glucose. After filtering, cells were resuspended in SD medium lacking a carbon source and incubated at 30°C for 1 h. Subsequently, cells were refed with 2% glucose. Aliquots of cells were treated with cycloheximide before and after 60 min of starvation and after 20 min of refeeding. Polysome profiles from these three time points are shown. (D) Ribosomes extracted from wt and RPS6AA cells were incubated in the presence of 800 mM KCl. Free 40S and 60S subunits were analyzed by sucrose gradient centrifugation. (E) The relative amount of Rps6 normalized to Rpl3 in protein extracts prepared from exponentially growing wt and RPS6AA cells was assessed by Western blot. (F) ypk1as ypk2 ypk3as cells starved of a carbon source for 60 min were subsequently refed with glucose in the presence of DMSO or 1NM-PP1. Twenty minutes later, cells were treated with cycloheximide, and polysome profiles were generated.To explore the effect of Rps6 phosphorylation on global translation, we monitored the abundance of free 40S and 60S subunits, 80S monosomes, and actively translating ribosomes (polysomes) in wild-type and RPS6AA cells using sucrose gradient centrifugation. The ratios of polysomes to 80S monosomes were the same in these two strains (Figure 5C). However, the mutant cells exhibited an elevated 60S peak, indicating an imbalance between the free subunits. Next we analyzed the polysome profiles of wild-type and RPS6AA cells upon glucose starvation and refeeding with glucose. Although the RPS6AA cells displayed an increase in the 60S peak in all of the conditions we tested, we did not detect other overt changes in the profiles compared with the wild-type cells under these conditions (Figure 5C and Supplemental Figure S5C).To further investigate whether the elevated 60S peak is due to a defect in the production of the small subunit, we dissociated the monosomes and polysomes into free subunits in the presence of high salt. We observed that the level of 40S subunits was indeed lower in the mutant strain than in the wild type strain (Figure 5D). Quantitative Western blots confirmed that the ratio of Rps6 to a large-subunit protein, Rpl3, is reduced in RPS6AA cells (Figure 5E and Supplemental Figure S5D). Furthermore, by Northern blot, we observed a delay in rRNA maturation in RPS6AA cells (Supplemental Figure 5E). In summary, RPS6AA cells display a 40S biogenesis defect but no obvious defect in global translation. These phenotypes are similar to what is observed in the absence of one of the two copies of the RPS6 genes (Pachler et al., 2004), and thus it is unclear whether the 40S biogenesis defect results from Rps6 hypophosphorylation or instead is due to reduced levels of Rps6AA protein (see Discussion).To probe for a role of Ypk kinases in translation, we compared polysome profiles generated from wt cells to those generated from cells lacking Ypk1–3 activities. Because chronic inhibition of these kinases was not an option (the cells die), we chose to use an experimental setup in which we acutely activate translation in the presence or absence of Ypk1–3 activity. Specifically, ypk1as, ypk2Δ, ypk3 as cells were starved of glucose for 1 h and then refed with glucose with or without the analogue (1NM-PP1). Analysis of the abundance of the free subunits (40S and 60S), the monosomes (80S), and the polysomes collected after refeeding with or without 1NM-PP1 revealed that acute inhibition of Ypks had no significant effect on the profiles under any of the conditions tested despite the expectant hypophosphorylation of Rps6 (Figure 5F and Supplemental Figure S5, F and G). These results are in accordance with a recently published study, which showed that the polysome profiles from wild-type and S6K1;S6K2−/− liver cells exhibited no significant differences under starvation or refeeding conditions (Chauvin et al., 2014). Unlike RPS6AA cells, ypk3 cells displayed no defects in proliferation, protein accumulation, or 40S biogenesis (Supplemental Figure 5E; unpublished data).Our polysome profiling results thus far demonstrate that Rps6 phosphorylation plays no role in global translation but do not query for a potential role in the translation of individual messages. To probe for such a role, we performed ribosome profiling on ypk1as ypk2 ypk3as RPS6wt and ypk1as ypk2 ypk3as RPS6AA cells with or without 1NM-PP1 (protocol in Supplemental Figure S6A). Specifically, we compared the profiles of these cells after 20 and 40 min of refeeding with glucose. Under these conditions, we observed expected patterns of Rps6, Ypk1, and Ypk3 phosphorylation (Supplemental Figure S6B). The majority of ribosome footprints were 28–32 nucleotides (nt) long, and metagene analysis showed the expected triplet periodicity within coding regions (Supplemental Figure S6, C and D). For both RNA sequencing (RNA-seq) and ribo-seq reads, the ribosome profiling data exhibited good correlations for coverage between the replicas (Supplemental Figure S6E). We observed no strain-dependent differences in global mRNA levels (RNA-seq) or global translational efficiencies (TEs; Figure 6A), demonstrating that, under these conditions, Rps6 phosphorylation does not strongly affect gene regulation at the level of either transcription or translation. To identify potential differential gene expression, we used Z-score transformation as described earlier (Andreev et al., 2015). Only five (SED1, YCR007C, RPS21A, UIP5, Q0050) of 6692 transcripts displayed TE levels with absolute Z-scores ≥3 relative to the wild type in both Rps6AA and Ypk-inhibited cells (Figure 6B and Supplemental Figure S7). We note that five is close to the expected number of transcripts to be selected by chance with this Z-score cutoff. Indeed, the profiles of SED1 and UIP5 when assessed manually did not display obvious differences between conditions (Figure 7A). We also assessed in detail the transcript profiles of a number of known housekeeping genes (e.g., ACT1), as well as ribosomal protein genes (e.g., RPS6B) and Ribi genes (e.g., TEF4) whose expressions are known to be strongly dependent on TORC1 activity (Lippman and Broach, 2009; Huber et al., 2011). None of the selected transcripts exhibited a striking change in profile across the different conditions (Figure 7A). Neither Rps6AA mutation nor Ypk activity affected translational efficiency of transcripts possessing introns (e.g., ACT1, RPS6B, TEF4;
Figure 7A) and/or transcripts whose translation is regulated by upstream open reading frames (ORFs; e.g., GCN4). These observations do not support a role of Rps6 phosphorylation in the pioneer round of translation and/or in reinitiation.FIGURE 6:Differential gene expression analysis. (A) Scatter plots showing Z-scores (y-axis) of fold changes relative to wild-type control of ribosome occupancy (Riboseq), transcript levels (RNAseq), and translation efficiency (TE). Transcripts are ordered based on their log2 minimal expression signal (x-axis). Transcripts with |Z| ≥ 3 are shown in red. (B) Venn diagrams showing the number of transcripts whose changes in ribosome occupancy, transcript levels, or TE levels exceed a |Z| ≥ 3 threshold vs. wild-type controls in the indicated conditions.FIGURE 7:Alternative translational processes occur independently of Rps6 phosphorylation status. (A) Ribo-seq and RNA-seq densities obtained from the six different conditions/time points mapped to genomic loci of individual genes. Ribo-seq densities are indicated with different colors, depending on the condition (indicated at the bottom). Combined RNA-seq densities of all six conditions are shown in gray. Locations of coding regions are indicated with black bars under each profile, introns with black lines, and upstream open reading frames with red bars. Genomic coordinates are given for the sense strand relative to the start codons, with the first nucleotide of the start codons as the zero coordinate. (B) The number of footprints mapped to 5′ leaders and 3′ UTRs normalized to the number of footprints mapped to coding sequences (CDS) are shown for each Ribo-seq replicate. (C) Analysis of ribosome footprint densities on mRNAs for the genes that require ribosomal frameshifting for their expression. Footprint densities are shown separately for all conditions and two replicas. The ORF plot for each gene is shown below the density plots, ORF1 is highlighted in blue, and a nonoverlapping part of ORF2 is highlighted in yellow. The total number of reads aligning to each of the two areas is indicated on the right side of each plot, along with footprint density within ORF2 relative to ORF1. Histograms above represent periodicity within both ORFs. It can be seen that the phase of periodicity shifts according to the expected +1 directionality of ribosomal frameshifting.Next we compared ribosome occupancy of 5′ leaders and 3′ untranslated regions (UTRs) across the different conditions. The number of footprints mapping to 3′ UTRs remained constantly low regardless of the phosphorylation status of Rps6 (Figure 7B). At 20 min, the number of footprints mapping to 5′ leaders was lower in RPS6AA cells in both replicas; however, the difference was not higher than replicate variation. We hypothesized that the changes in 5′ leader densities may occur in specific mRNAs and analyzed the changes in the relative density between 5′ leaders and coding regions. The observed changes between different conditions did not exceed those observed for the replicas obtained for the same conditions (Supplemental Figure S7B). A similar, less robust trend was observed in Ypk-inhibited cells.Finally, we explored a possible role for Rps6 phosphorylation in frameshifting. During ribosomal frameshifting, a proportion of ribosomes shift their reading frame to produce a protein product that is encoded by two ORFs. The ribosome density profiles for such mRNAs are normally characterized by a drop of density downstream of ORF1 and a change in the phase of the triplet periodicity between nonoverlapping regions of ORF1 and ORF2, which is consistent with the direction of frameshifting (Michel et al., 2012). The drop of density should correlate with frameshifting efficiency and thus could be used to assess differences in frameshifting efficiencies between samples.To obtain information on S. cerevisiae genes that require programmed ribosomal frameshifting for their expression (ABP140, EST3, OAZ1, and TY1 transposable element), we used the Recode-2 database (Bekaert et al., 2010). The EST3 mRNA had insufficient coverage for the analysis. Profiles of ribosome density for ABP140, OAZ1, and two arbitrary chosen copies of TY1 element are shown in Figure 7C. Because the ambiguous alignments were used for frameshifting analysis (see Materials and Methods), profiles for TY1 elements represent mRNA translation of several TY1 copies that share strong sequence similarity.We observed a change of triplet periodicity consistent with the known directionality of frameshifting in all cases; however, the observed variation in the drop of ribosome density was insignificant among various samples in comparison to the variation observed between replicas (Figure 7C).In summary, consistent with previous work (Chauvin et al., 2014), we found that Rps6 phosphorylation plays no role in global protein synthesis upon refeeding as assessed by polysome profiling. To probe for potential affects on the translation of individual transcripts, we exploited a recent technological advance in the translation field—ribosome profiling. Specifically, using ribosome profiling, we assessed changes in RNA levels and translation efficiencies of individual mRNAs at the scale of the whole transcriptome (Supplemental Figure S7) under conditions in which Rps6 was or was not phosphorylated. We used a relaxed threshold of statistical significance to call for differential expression (|Z|-score ≥ 3), for which 0.25% of analyzed genes are expected to be classified as differentially expressed by chance under the assumption that changes in coverage due to technical factors follow a normal distribution. Even under these relaxed thresholds, we failed to identify differentially expressed genes that would be shared between Rps6 mutants and Ypk-inhibited cells at all time points (Figure 6B).

Physiological characterization of Rps6 phosphomutants

To identify the physiological significance of Rps6 phosphorylation, we first compared the growth rates of cells expressing either wild-type Rps6 protein or the nonphosphorylatable variant Rps6AA. In our hands, expression of the RPS6AA alleles resulted in an almost 30% decrease in the proliferation rate (Figure 5A and Supplemental Figure S5A). The RPS6AA cells also had lower protein content than the wild-type cells as assessed by fluorescein isothiocyanate (FITC) staining and fluorescence-activated cell sorting (FACS) analysis (Figure 5B and Supplemental Figure S5B).

Physiological characterization of Rps6 phosphorylation. (A) Growth (OD600) of wt and RPS6AA cells in rich medium at 30°C was recorded every 15min for 16 h. (B) The protein content of exponentially growing wt and RPS6AA cells was measured by FITC staining and FACS analysis. (C) Polysome profiles of wt and RPS6AA cells. Cells were grown to exponential growth phase in SD medium containing 2% glucose. After filtering, cells were resuspended in SD medium lacking a carbon source and incubated at 30°C for 1 h. Subsequently, cells were refed with 2% glucose. Aliquots of cells were treated with cycloheximide before and after 60 min of starvation and after 20 min of refeeding. Polysome profiles from these three time points are shown. (D) Ribosomes extracted from wt and RPS6AA cells were incubated in the presence of 800 mM KCl. Free 40S and 60S subunits were analyzed by sucrose gradient centrifugation. (E) The relative amount of Rps6 normalized to Rpl3 in protein extracts prepared from exponentially growing wt and RPS6AA cells was assessed by Western blot. (F) ypk1as ypk2 ypk3as cells starved of a carbon source for 60 min were subsequently refed with glucose in the presence of DMSO or 1NM-PP1. Twenty minutes later, cells were treated with cycloheximide, and polysome profiles were generated.

To explore the effect of Rps6 phosphorylation on global translation, we monitored the abundance of free 40S and 60S subunits, 80S monosomes, and actively translating ribosomes (polysomes) in wild-type and RPS6AA cells using sucrose gradient centrifugation. The ratios of polysomes to 80S monosomes were the same in these two strains (Figure 5C). However, the mutant cells exhibited an elevated 60S peak, indicating an imbalance between the free subunits. Next we analyzed the polysome profiles of wild-type and RPS6AA cells upon glucose starvation and refeeding with glucose. Although the RPS6AA cells displayed an increase in the 60S peak in all of the conditions we tested, we did not detect other overt changes in the profiles compared with the wild-type cells under these conditions (Figure 5C and Supplemental Figure S5C).

To further investigate whether the elevated 60S peak is due to a defect in the production of the small subunit, we dissociated the monosomes and polysomes into free subunits in the presence of high salt. We observed that the level of 40S subunits was indeed lower in the mutant strain than in the wild type strain (Figure 5D). Quantitative Western blots confirmed that the ratio of Rps6 to a large-subunit protein, Rpl3, is reduced in RPS6AA cells (Figure 5E and Supplemental Figure S5D). Furthermore, by Northern blot, we observed a delay in rRNA maturation in RPS6AA cells (Supplemental Figure 5E). In summary, RPS6AA cells display a 40S biogenesis defect but no obvious defect in global translation. These phenotypes are similar to what is observed in the absence of one of the two copies of the RPS6 genes (Pachler et al., 2004), and thus it is unclear whether the 40S biogenesis defect results from Rps6 hypophosphorylation or instead is due to reduced levels of Rps6AA protein (see Discussion).

To probe for a role of Ypk kinases in translation, we compared polysome profiles generated from wt cells to those generated from cells lacking Ypk1–3 activities. Because chronic inhibition of these kinases was not an option (the cells die), we chose to use an experimental setup in which we acutely activate translation in the presence or absence of Ypk1–3 activity. Specifically, ypk1as, ypk2Δ, ypk3 as cells were starved of glucose for 1 h and then refed with glucose with or without the analogue (1NM-PP1). Analysis of the abundance of the free subunits (40S and 60S), the monosomes (80S), and the polysomes collected after refeeding with or without 1NM-PP1 revealed that acute inhibition of Ypks had no significant effect on the profiles under any of the conditions tested despite the expectant hypophosphorylation of Rps6 (Figure 5F and Supplemental Figure S5, F and G). These results are in accordance with a recently published study, which showed that the polysome profiles from wild-type and S6K1;S6K2−/− liver cells exhibited no significant differences under starvation or refeeding conditions (Chauvin et al., 2014). Unlike RPS6AA cells, ypk3 cells displayed no defects in proliferation, protein accumulation, or 40S biogenesis (Supplemental Figure 5E; unpublished data).

Our polysome profiling results thus far demonstrate that Rps6 phosphorylation plays no role in global translation but do not query for a potential role in the translation of individual messages. To probe for such a role, we performed ribosome profiling on ypk1as ypk2 ypk3as RPS6wt and ypk1as ypk2 ypk3as RPS6AA cells with or without 1NM-PP1 (protocol in Supplemental Figure S6A). Specifically, we compared the profiles of these cells after 20 and 40 min of refeeding with glucose. Under these conditions, we observed expected patterns of Rps6, Ypk1, and Ypk3 phosphorylation (Supplemental Figure S6B). The majority of ribosome footprints were 28–32 nucleotides (nt) long, and metagene analysis showed the expected triplet periodicity within coding regions (Supplemental Figure S6, C and D). For both RNA sequencing (RNA-seq) and ribo-seq reads, the ribosome profiling data exhibited good correlations for coverage between the replicas (Supplemental Figure S6E). We observed no strain-dependent differences in global mRNA levels (RNA-seq) or global translational efficiencies (TEs; Figure 6A), demonstrating that, under these conditions, Rps6 phosphorylation does not strongly affect gene regulation at the level of either transcription or translation. To identify potential differential gene expression, we used Z-score transformation as described earlier (Andreev et al., 2015). Only five (SED1, YCR007C, RPS21A, UIP5, Q0050) of 6692 transcripts displayed TE levels with absolute Z-scores ≥3 relative to the wild type in both Rps6AA and Ypk-inhibited cells (Figure 6B and Supplemental Figure S7). We note that five is close to the expected number of transcripts to be selected by chance with this Z-score cutoff. Indeed, the profiles of SED1 and UIP5 when assessed manually did not display obvious differences between conditions (Figure 7A). We also assessed in detail the transcript profiles of a number of known housekeeping genes (e.g., ACT1), as well as ribosomal protein genes (e.g., RPS6B) and Ribi genes (e.g., TEF4) whose expressions are known to be strongly dependent on TORC1 activity (Lippman and Broach, 2009; Huber et al., 2011). None of the selected transcripts exhibited a striking change in profile across the different conditions (Figure 7A). Neither Rps6AA mutation nor Ypk activity affected translational efficiency of transcripts possessing introns (e.g., ACT1, RPS6B, TEF4;
Figure 7A) and/or transcripts whose translation is regulated by upstream open reading frames (ORFs; e.g., GCN4). These observations do not support a role of Rps6 phosphorylation in the pioneer round of translation and/or in reinitiation.

Differential gene expression analysis. (A) Scatter plots showing Z-scores (y-axis) of fold changes relative to wild-type control of ribosome occupancy (Riboseq), transcript levels (RNAseq), and translation efficiency (TE). Transcripts are ordered based on their log2 minimal expression signal (x-axis). Transcripts with |Z| ≥ 3 are shown in red. (B) Venn diagrams showing the number of transcripts whose changes in ribosome occupancy, transcript levels, or TE levels exceed a |Z| ≥ 3 threshold vs. wild-type controls in the indicated conditions.

Alternative translational processes occur independently of Rps6 phosphorylation status. (A) Ribo-seq and RNA-seq densities obtained from the six different conditions/time points mapped to genomic loci of individual genes. Ribo-seq densities are indicated with different colors, depending on the condition (indicated at the bottom). Combined RNA-seq densities of all six conditions are shown in gray. Locations of coding regions are indicated with black bars under each profile, introns with black lines, and upstream open reading frames with red bars. Genomic coordinates are given for the sense strand relative to the start codons, with the first nucleotide of the start codons as the zero coordinate. (B) The number of footprints mapped to 5′ leaders and 3′ UTRs normalized to the number of footprints mapped to coding sequences (CDS) are shown for each Ribo-seq replicate. (C) Analysis of ribosome footprint densities on mRNAs for the genes that require ribosomal frameshifting for their expression. Footprint densities are shown separately for all conditions and two replicas. The ORF plot for each gene is shown below the density plots, ORF1 is highlighted in blue, and a nonoverlapping part of ORF2 is highlighted in yellow. The total number of reads aligning to each of the two areas is indicated on the right side of each plot, along with footprint density within ORF2 relative to ORF1. Histograms above represent periodicity within both ORFs. It can be seen that the phase of periodicity shifts according to the expected +1 directionality of ribosomal frameshifting.

Next we compared ribosome occupancy of 5′ leaders and 3′ untranslated regions (UTRs) across the different conditions. The number of footprints mapping to 3′ UTRs remained constantly low regardless of the phosphorylation status of Rps6 (Figure 7B). At 20 min, the number of footprints mapping to 5′ leaders was lower in RPS6AA cells in both replicas; however, the difference was not higher than replicate variation. We hypothesized that the changes in 5′ leader densities may occur in specific mRNAs and analyzed the changes in the relative density between 5′ leaders and coding regions. The observed changes between different conditions did not exceed those observed for the replicas obtained for the same conditions (Supplemental Figure S7B). A similar, less robust trend was observed in Ypk-inhibited cells.

Finally, we explored a possible role for Rps6 phosphorylation in frameshifting. During ribosomal frameshifting, a proportion of ribosomes shift their reading frame to produce a protein product that is encoded by two ORFs. The ribosome density profiles for such mRNAs are normally characterized by a drop of density downstream of ORF1 and a change in the phase of the triplet periodicity between nonoverlapping regions of ORF1 and ORF2, which is consistent with the direction of frameshifting (Michel et al., 2012). The drop of density should correlate with frameshifting efficiency and thus could be used to assess differences in frameshifting efficiencies between samples.

To obtain information on S. cerevisiae genes that require programmed ribosomal frameshifting for their expression (ABP140, EST3, OAZ1, and TY1 transposable element), we used the Recode-2 database (Bekaert et al., 2010). The EST3 mRNA had insufficient coverage for the analysis. Profiles of ribosome density for ABP140, OAZ1, and two arbitrary chosen copies of TY1 element are shown in Figure 7C. Because the ambiguous alignments were used for frameshifting analysis (see Materials and Methods), profiles for TY1 elements represent mRNA translation of several TY1 copies that share strong sequence similarity.

We observed a change of triplet periodicity consistent with the known directionality of frameshifting in all cases; however, the observed variation in the drop of ribosome density was insignificant among various samples in comparison to the variation observed between replicas (Figure 7C).

In summary, consistent with previous work (Chauvin et al., 2014), we found that Rps6 phosphorylation plays no role in global protein synthesis upon refeeding as assessed by polysome profiling. To probe for potential affects on the translation of individual transcripts, we exploited a recent technological advance in the translation field—ribosome profiling. Specifically, using ribosome profiling, we assessed changes in RNA levels and translation efficiencies of individual mRNAs at the scale of the whole transcriptome (Supplemental Figure S7) under conditions in which Rps6 was or was not phosphorylated. We used a relaxed threshold of statistical significance to call for differential expression (|Z|-score ≥ 3), for which 0.25% of analyzed genes are expected to be classified as differentially expressed by chance under the assumption that changes in coverage due to technical factors follow a normal distribution. Even under these relaxed thresholds, we failed to identify differentially expressed genes that would be shared between Rps6 mutants and Ypk-inhibited cells at all time points (Figure 6B).

DISCUSSIONThe pathways regulating Rps6 phosphorylation in budding yeast have remained ambiguous and, in higher eukaryotes, surprisingly complex. Furthermore, the physiological role of Rps6 phosphorylation has remained mysterious for the past four decades. Rps6 is a constitutive component of the 40S small subunit of the ribosome, and, since its phosphorylation is promoted downstream of anabolic stimuli, this modification has long been assumed to promote protein synthesis. However, experimental evidence supporting such a role has not been forthcoming. In this study, we defined the signaling cascades that regulate Rps6 phosphorylation in budding yeast. We demonstrated that nonphosphorylatable Rps6 knock-ins do not accurately mimic the function of hypophosphorylated protein. Finally, using ribosome profiling, we found little evidence to support a role for Rps6 phosphorylation in the regulation of translation per se. We are thus compelled to conclude that whatever role Rps6 phosphorylation plays, this role is not directly related to mRNA translation.Both TORC1 and TORC2 influence Rps6 phosphorylationIn the present work, we demonstrate for the first time that the two phosphorylation sites on the C-terminus of Rps6 are differentially regulated in yeast and that this regulation is dependent on both TORC1 and TORC2. Specifically, we show that TORC1, via Ypk3, regulates Ser-232 and Ser-233 phosphorylation, whereas TORC2, via Ypk1 and, to a lesser extent, Ypk2, regulates Ser-232 phosphorylation. Rps6 phosphorylation thus appears to be similarly regulated in budding and fission yeast (Du et al., 2012; Nakashima et al., 2012). This conservation across the distantly related yeasts S. cerevisiae and S. pombe predicts that TORC2 will similarly be important for S6 phosphorylation in higher eukaryotes. Curiously, although we observed direct phosphorylation of Rps6 by Ypk3 in vitro, we failed to observe Rps6 phosphorylation by Ypk1 in the same assay. This was surprising, given the dependence of Rps6 phosphorylation on the activity of these kinases in vivo and the strong homology among all three Ypks and mammalian S6K1/2. Perhaps phosphorylation of Rps6 by Ypk1/2 occurs in a cellular context that is not recapitulated in the in vitro assay.Although Ypk1 and 2 are well-established effectors of TORC2, direct phosphorylation of Ypk3 by TORC1 is novel and extends previous work (Gonzalez et al., 2015). Specifically, we found that Ypk3 physically interacts with TORC1 in vivo in a rapamycin-sensitive manner. We mapped four rapamycin-sensitive phosphorylation sites on the C-terminal end of Ypk3 and demonstrated that they are indispensable for its activity as an S6 kinase. However, expression of the phosphomimetic mutant of Ypk3 failed to suppress rapamycin-induced Rps6 dephosphorylation, leading us to hypothesize that TORC1 might regulate not only Rps6 phosphorylation, but also its dephosphorylation.In this vein, we found that Glc7/PP1 is the phosphatase responsible for dephosphorylation of both Ser-232 and Ser-233. We further demonstrated that the Glc7 regulator Shp1 is also necessary for Rps6 dephosphorylation. Shp1, in complex with the AAA-ATPase Cdc48, promotes the assembly of Glc7–Sds22–Ypi1 complexes (Cheng and Chen, 2015). However, the mechanism by which TOR complexes interfere with Glc7 and/or Shp1 activities remains to be elucidated. Intriguingly, preliminary data show that Shp1 is phosphorylated in a rapamycin-sensitive manner (unpublished data).Previously we proposed that Sch9, another AGC kinase and TORC1 effector, could serve as the yeast S6K (Urban et al., 2007). Consistently, we found that expression of a rapamycin-insensitive variant of Sch9 delays rapamycin-induced Rps6 dephosphorylation (unpublished data). However, unlike YPK3 deletion, SCH9 deletion did not compromise Rps6 phosphorylation (unpublished data). Thus we no longer believe that Sch9 is truly an S6 kinase. At present, we favor the idea that Sch9 signals block Rps6 dephosphorylation. This could be direct, via Shp1 phosphorylation, or, indirect, via its influence on translation initiation (Urban et al., 2007).What function does Rps6 phosphorylation serve?Previous attempts to identify the physiological importance of Rps6 phosphorylation in yeast led to the observation that Rps6 phosphorylation had no obvious effect on growth under a variety of stress conditions or during sporulation (Kruse et al., 1985; Johnson and Warner, 1987). Unexpectedly, in our yeast background (TB50a), we found that the RPS6AA mutations result in a lower proliferation rate and protein content than with the wild-type cells, which is likely due to reduced levels of Rps6AA protein. Furthermore these cells present a 40S ribosome biogenesis defect evident at both the protein level and rRNA processing level. These phenotypes are not apparent under conditions in which wild-type Rps6 protein is hypophosphorylated. From these results, we conclude that one must be extremely cautious when assigning phenotypes obtained from cells expressing nonphosphorylatable variants of Rps6 to lack of Rps6 phosphorylation.To complement our biochemical efforts, we also used synthetic genetic array (SGA) technology (Tong et al., 2001) to probe for synthetic interactions in triple mutants containing RPS6AAA, RPS6BAA, and each nonessential gene deletion. The two validated hits that we obtained from this assay were dom34 and hbs1, each of which displayed a synthetic sick phenotype in combination with RPS6AAA, RPS6BAA (unpublished data). Dom34 and Hbs1 function together in no-go mRNA decay (Doma and Parker, 2006; Harigaya and Parker, 2010), but dom34 and hbs1 display synthetic sick interactions with most deletion mutants of 40S ribosomal protein paralogues (Costanzo et al., 2010). This observation again suggests that it is the reduced levels of Rps6AA protein and not the lack of Rps6 phosphorylation that underlies this synthetic interaction. There are now ample data to suggest that many ribosomal proteins perform non–ribosome-related tasks (Warner and McIntosh, 2009). If Rps6 phosphorylation were involved in such a function, one might hope that this could have been discovered through this SGA approach.We also made the interesting observation that RPS6AA mutation increases resistance to the PP1 inhibitor calyculin A. The simplest interpretation of this result is that hyperphosphorylation of Rps6 is toxic. Consistently, RPS6DD cells were found to be inviable, which, unfortunately, makes their further characterization an intriguing but challenging prospect.Recent structural studies showed that Rps6 is located far from the decoding center, accessible from the solvent side of the 40S subunit (Ben-Shem et al., 2011; Khatter et al., 2015). According to these structures, the C-terminal end of Rpl24 relocates and interacts with Rps6 during translocation. The functional significance of this interaction remains to be elucidated. Unfortunately, the C-terminus of Rps6 is missing from these structures, making it challenging to define a function for the phosphorylation of this domain from these structural studies.Previous studies addressing the physiological function of Rps6 phosphorylation in higher eukaryotes suggested that Rps6 phosphorylation slows translation elongation (Ruvinsky et al., 2005; Chauvin et al., 2014). These are elegant studies, but because they use deletion of the S6 kinases and knock-in alleles of Rps6, they cannot account for the likely cellular adaptation that occurs in response to these chronic deficiencies, as the authors acknowledge. Here we also queried whether Rps6 phosphorylation plays a role in translation in yeast. To this end, we generated both polysome profiles and ribosome profiles from wt, RPS6AA, and Ypk-inhibited cells after refeeding with glucose to stimulate translation. Of importance, our ability to acutely inactivate the kinases responsible for Rps6 phosphorylation avoids potential confounding effects arising from cellular adaptation. After extensive analyses, an overt role for Rps6 phosphorylation could be ascribed to neither global translation nor translational regulation of specific subsets of mRNAs. Specifically, we sought changes in global transcription that would have been apparent in the polysome profiling, whereas transcript-specific effects would have been observable in the ribosome profiling. We found no changes in the density of ribosome-protected fragments across conditions, implying that Rps6 phosphorylation does not affect the rate of protein synthesis of individual transcripts. In addition, alternative translational processes, including pausing, frameshifting of specific messages, reinitiation on GCN4 transcripts, and global stop-codon readthrough, appeared to occur independently of the phosphorylation status of Rps6. Finally, and in contrast to previous studies, our experimental setup was chosen to potentially expose a role of Rps6 phosphorylation in the pioneering round of translation of individual transcripts. Specifically, we assumed that refeeding of cells after 1 h of glucose starvation would trigger inaugural translation of newly transcribed messages, which would be visualized at one of the two ribosome profiling time points. Thus, under the experimental conditions used, we find no evidence to support a role for Rps6 phosphorylation in any aspect of translation. One exception to this general conclusion is the reduction of ribosomes on 5’ leaders in RPS6AA cells and, to a lesser extent, in cells lacking Ypk1–3 activities. Ribosomes accumulate on 5’ leaders in starved cells (Ingolia et al., 2009), but the physiological role underlying this phenomenon is unknown. The observed reduction requires further study to assess its statistical significance under the tested conditions.Recent work implicated S6 kinases and Rps6 phosphorylation in the ribosome biogenesis transcriptional program in mouse liver (Chauvin et al., 2014). Although such a function could potentially be conserved in yeast, after analysis of our mRNA-seq data used for the ribosome profiling, we failed to detect Rps6 phosphorylation–dependent transcriptional regulation of any mRNA subgroups. Thus, like translation elongation, the influence of Rps6 phosphorylation on transcription appears to be restricted to higher eukaryotes.

The pathways regulating Rps6 phosphorylation in budding yeast have remained ambiguous and, in higher eukaryotes, surprisingly complex. Furthermore, the physiological role of Rps6 phosphorylation has remained mysterious for the past four decades. Rps6 is a constitutive component of the 40S small subunit of the ribosome, and, since its phosphorylation is promoted downstream of anabolic stimuli, this modification has long been assumed to promote protein synthesis. However, experimental evidence supporting such a role has not been forthcoming. In this study, we defined the signaling cascades that regulate Rps6 phosphorylation in budding yeast. We demonstrated that nonphosphorylatable Rps6 knock-ins do not accurately mimic the function of hypophosphorylated protein. Finally, using ribosome profiling, we found little evidence to support a role for Rps6 phosphorylation in the regulation of translation per se. We are thus compelled to conclude that whatever role Rps6 phosphorylation plays, this role is not directly related to mRNA translation.

Both TORC1 and TORC2 influence Rps6 phosphorylationIn the present work, we demonstrate for the first time that the two phosphorylation sites on the C-terminus of Rps6 are differentially regulated in yeast and that this regulation is dependent on both TORC1 and TORC2. Specifically, we show that TORC1, via Ypk3, regulates Ser-232 and Ser-233 phosphorylation, whereas TORC2, via Ypk1 and, to a lesser extent, Ypk2, regulates Ser-232 phosphorylation. Rps6 phosphorylation thus appears to be similarly regulated in budding and fission yeast (Du et al., 2012; Nakashima et al., 2012). This conservation across the distantly related yeasts S. cerevisiae and S. pombe predicts that TORC2 will similarly be important for S6 phosphorylation in higher eukaryotes. Curiously, although we observed direct phosphorylation of Rps6 by Ypk3 in vitro, we failed to observe Rps6 phosphorylation by Ypk1 in the same assay. This was surprising, given the dependence of Rps6 phosphorylation on the activity of these kinases in vivo and the strong homology among all three Ypks and mammalian S6K1/2. Perhaps phosphorylation of Rps6 by Ypk1/2 occurs in a cellular context that is not recapitulated in the in vitro assay.Although Ypk1 and 2 are well-established effectors of TORC2, direct phosphorylation of Ypk3 by TORC1 is novel and extends previous work (Gonzalez et al., 2015). Specifically, we found that Ypk3 physically interacts with TORC1 in vivo in a rapamycin-sensitive manner. We mapped four rapamycin-sensitive phosphorylation sites on the C-terminal end of Ypk3 and demonstrated that they are indispensable for its activity as an S6 kinase. However, expression of the phosphomimetic mutant of Ypk3 failed to suppress rapamycin-induced Rps6 dephosphorylation, leading us to hypothesize that TORC1 might regulate not only Rps6 phosphorylation, but also its dephosphorylation.In this vein, we found that Glc7/PP1 is the phosphatase responsible for dephosphorylation of both Ser-232 and Ser-233. We further demonstrated that the Glc7 regulator Shp1 is also necessary for Rps6 dephosphorylation. Shp1, in complex with the AAA-ATPase Cdc48, promotes the assembly of Glc7–Sds22–Ypi1 complexes (Cheng and Chen, 2015). However, the mechanism by which TOR complexes interfere with Glc7 and/or Shp1 activities remains to be elucidated. Intriguingly, preliminary data show that Shp1 is phosphorylated in a rapamycin-sensitive manner (unpublished data).Previously we proposed that Sch9, another AGC kinase and TORC1 effector, could serve as the yeast S6K (Urban et al., 2007). Consistently, we found that expression of a rapamycin-insensitive variant of Sch9 delays rapamycin-induced Rps6 dephosphorylation (unpublished data). However, unlike YPK3 deletion, SCH9 deletion did not compromise Rps6 phosphorylation (unpublished data). Thus we no longer believe that Sch9 is truly an S6 kinase. At present, we favor the idea that Sch9 signals block Rps6 dephosphorylation. This could be direct, via Shp1 phosphorylation, or, indirect, via its influence on translation initiation (Urban et al., 2007).

Both TORC1 and TORC2 influence Rps6 phosphorylation

In the present work, we demonstrate for the first time that the two phosphorylation sites on the C-terminus of Rps6 are differentially regulated in yeast and that this regulation is dependent on both TORC1 and TORC2. Specifically, we show that TORC1, via Ypk3, regulates Ser-232 and Ser-233 phosphorylation, whereas TORC2, via Ypk1 and, to a lesser extent, Ypk2, regulates Ser-232 phosphorylation. Rps6 phosphorylation thus appears to be similarly regulated in budding and fission yeast (Du et al., 2012; Nakashima et al., 2012). This conservation across the distantly related yeasts S. cerevisiae and S. pombe predicts that TORC2 will similarly be important for S6 phosphorylation in higher eukaryotes. Curiously, although we observed direct phosphorylation of Rps6 by Ypk3 in vitro, we failed to observe Rps6 phosphorylation by Ypk1 in the same assay. This was surprising, given the dependence of Rps6 phosphorylation on the activity of these kinases in vivo and the strong homology among all three Ypks and mammalian S6K1/2. Perhaps phosphorylation of Rps6 by Ypk1/2 occurs in a cellular context that is not recapitulated in the in vitro assay.

Although Ypk1 and 2 are well-established effectors of TORC2, direct phosphorylation of Ypk3 by TORC1 is novel and extends previous work (Gonzalez et al., 2015). Specifically, we found that Ypk3 physically interacts with TORC1 in vivo in a rapamycin-sensitive manner. We mapped four rapamycin-sensitive phosphorylation sites on the C-terminal end of Ypk3 and demonstrated that they are indispensable for its activity as an S6 kinase. However, expression of the phosphomimetic mutant of Ypk3 failed to suppress rapamycin-induced Rps6 dephosphorylation, leading us to hypothesize that TORC1 might regulate not only Rps6 phosphorylation, but also its dephosphorylation.

In this vein, we found that Glc7/PP1 is the phosphatase responsible for dephosphorylation of both Ser-232 and Ser-233. We further demonstrated that the Glc7 regulator Shp1 is also necessary for Rps6 dephosphorylation. Shp1, in complex with the AAA-ATPase Cdc48, promotes the assembly of Glc7–Sds22–Ypi1 complexes (Cheng and Chen, 2015). However, the mechanism by which TOR complexes interfere with Glc7 and/or Shp1 activities remains to be elucidated. Intriguingly, preliminary data show that Shp1 is phosphorylated in a rapamycin-sensitive manner (unpublished data).

Previously we proposed that Sch9, another AGC kinase and TORC1 effector, could serve as the yeast S6K (Urban et al., 2007). Consistently, we found that expression of a rapamycin-insensitive variant of Sch9 delays rapamycin-induced Rps6 dephosphorylation (unpublished data). However, unlike YPK3 deletion, SCH9 deletion did not compromise Rps6 phosphorylation (unpublished data). Thus we no longer believe that Sch9 is truly an S6 kinase. At present, we favor the idea that Sch9 signals block Rps6 dephosphorylation. This could be direct, via Shp1 phosphorylation, or, indirect, via its influence on translation initiation (Urban et al., 2007).

What function does Rps6 phosphorylation serve?Previous attempts to identify the physiological importance of Rps6 phosphorylation in yeast led to the observation that Rps6 phosphorylation had no obvious effect on growth under a variety of stress conditions or during sporulation (Kruse et al., 1985; Johnson and Warner, 1987). Unexpectedly, in our yeast background (TB50a), we found that the RPS6AA mutations result in a lower proliferation rate and protein content than with the wild-type cells, which is likely due to reduced levels of Rps6AA protein. Furthermore these cells present a 40S ribosome biogenesis defect evident at both the protein level and rRNA processing level. These phenotypes are not apparent under conditions in which wild-type Rps6 protein is hypophosphorylated. From these results, we conclude that one must be extremely cautious when assigning phenotypes obtained from cells expressing nonphosphorylatable variants of Rps6 to lack of Rps6 phosphorylation.To complement our biochemical efforts, we also used synthetic genetic array (SGA) technology (Tong et al., 2001) to probe for synthetic interactions in triple mutants containing RPS6AAA, RPS6BAA, and each nonessential gene deletion. The two validated hits that we obtained from this assay were dom34 and hbs1, each of which displayed a synthetic sick phenotype in combination with RPS6AAA, RPS6BAA (unpublished data). Dom34 and Hbs1 function together in no-go mRNA decay (Doma and Parker, 2006; Harigaya and Parker, 2010), but dom34 and hbs1 display synthetic sick interactions with most deletion mutants of 40S ribosomal protein paralogues (Costanzo et al., 2010). This observation again suggests that it is the reduced levels of Rps6AA protein and not the lack of Rps6 phosphorylation that underlies this synthetic interaction. There are now ample data to suggest that many ribosomal proteins perform non–ribosome-related tasks (Warner and McIntosh, 2009). If Rps6 phosphorylation were involved in such a function, one might hope that this could have been discovered through this SGA approach.We also made the interesting observation that RPS6AA mutation increases resistance to the PP1 inhibitor calyculin A. The simplest interpretation of this result is that hyperphosphorylation of Rps6 is toxic. Consistently, RPS6DD cells were found to be inviable, which, unfortunately, makes their further characterization an intriguing but challenging prospect.Recent structural studies showed that Rps6 is located far from the decoding center, accessible from the solvent side of the 40S subunit (Ben-Shem et al., 2011; Khatter et al., 2015). According to these structures, the C-terminal end of Rpl24 relocates and interacts with Rps6 during translocation. The functional significance of this interaction remains to be elucidated. Unfortunately, the C-terminus of Rps6 is missing from these structures, making it challenging to define a function for the phosphorylation of this domain from these structural studies.Previous studies addressing the physiological function of Rps6 phosphorylation in higher eukaryotes suggested that Rps6 phosphorylation slows translation elongation (Ruvinsky et al., 2005; Chauvin et al., 2014). These are elegant studies, but because they use deletion of the S6 kinases and knock-in alleles of Rps6, they cannot account for the likely cellular adaptation that occurs in response to these chronic deficiencies, as the authors acknowledge. Here we also queried whether Rps6 phosphorylation plays a role in translation in yeast. To this end, we generated both polysome profiles and ribosome profiles from wt, RPS6AA, and Ypk-inhibited cells after refeeding with glucose to stimulate translation. Of importance, our ability to acutely inactivate the kinases responsible for Rps6 phosphorylation avoids potential confounding effects arising from cellular adaptation. After extensive analyses, an overt role for Rps6 phosphorylation could be ascribed to neither global translation nor translational regulation of specific subsets of mRNAs. Specifically, we sought changes in global transcription that would have been apparent in the polysome profiling, whereas transcript-specific effects would have been observable in the ribosome profiling. We found no changes in the density of ribosome-protected fragments across conditions, implying that Rps6 phosphorylation does not affect the rate of protein synthesis of individual transcripts. In addition, alternative translational processes, including pausing, frameshifting of specific messages, reinitiation on GCN4 transcripts, and global stop-codon readthrough, appeared to occur independently of the phosphorylation status of Rps6. Finally, and in contrast to previous studies, our experimental setup was chosen to potentially expose a role of Rps6 phosphorylation in the pioneering round of translation of individual transcripts. Specifically, we assumed that refeeding of cells after 1 h of glucose starvation would trigger inaugural translation of newly transcribed messages, which would be visualized at one of the two ribosome profiling time points. Thus, under the experimental conditions used, we find no evidence to support a role for Rps6 phosphorylation in any aspect of translation. One exception to this general conclusion is the reduction of ribosomes on 5’ leaders in RPS6AA cells and, to a lesser extent, in cells lacking Ypk1–3 activities. Ribosomes accumulate on 5’ leaders in starved cells (Ingolia et al., 2009), but the physiological role underlying this phenomenon is unknown. The observed reduction requires further study to assess its statistical significance under the tested conditions.Recent work implicated S6 kinases and Rps6 phosphorylation in the ribosome biogenesis transcriptional program in mouse liver (Chauvin et al., 2014). Although such a function could potentially be conserved in yeast, after analysis of our mRNA-seq data used for the ribosome profiling, we failed to detect Rps6 phosphorylation–dependent transcriptional regulation of any mRNA subgroups. Thus, like translation elongation, the influence of Rps6 phosphorylation on transcription appears to be restricted to higher eukaryotes.

What function does Rps6 phosphorylation serve?

Previous attempts to identify the physiological importance of Rps6 phosphorylation in yeast led to the observation that Rps6 phosphorylation had no obvious effect on growth under a variety of stress conditions or during sporulation (Kruse et al., 1985; Johnson and Warner, 1987). Unexpectedly, in our yeast background (TB50a), we found that the RPS6AA mutations result in a lower proliferation rate and protein content than with the wild-type cells, which is likely due to reduced levels of Rps6AA protein. Furthermore these cells present a 40S ribosome biogenesis defect evident at both the protein level and rRNA processing level. These phenotypes are not apparent under conditions in which wild-type Rps6 protein is hypophosphorylated. From these results, we conclude that one must be extremely cautious when assigning phenotypes obtained from cells expressing nonphosphorylatable variants of Rps6 to lack of Rps6 phosphorylation.

To complement our biochemical efforts, we also used synthetic genetic array (SGA) technology (Tong et al., 2001) to probe for synthetic interactions in triple mutants containing RPS6AAA, RPS6BAA, and each nonessential gene deletion. The two validated hits that we obtained from this assay were dom34 and hbs1, each of which displayed a synthetic sick phenotype in combination with RPS6AAA, RPS6BAA (unpublished data). Dom34 and Hbs1 function together in no-go mRNA decay (Doma and Parker, 2006; Harigaya and Parker, 2010), but dom34 and hbs1 display synthetic sick interactions with most deletion mutants of 40S ribosomal protein paralogues (Costanzo et al., 2010). This observation again suggests that it is the reduced levels of Rps6AA protein and not the lack of Rps6 phosphorylation that underlies this synthetic interaction. There are now ample data to suggest that many ribosomal proteins perform non–ribosome-related tasks (Warner and McIntosh, 2009). If Rps6 phosphorylation were involved in such a function, one might hope that this could have been discovered through this SGA approach.

We also made the interesting observation that RPS6AA mutation increases resistance to the PP1 inhibitor calyculin A. The simplest interpretation of this result is that hyperphosphorylation of Rps6 is toxic. Consistently, RPS6DD cells were found to be inviable, which, unfortunately, makes their further characterization an intriguing but challenging prospect.

Recent structural studies showed that Rps6 is located far from the decoding center, accessible from the solvent side of the 40S subunit (Ben-Shem et al., 2011; Khatter et al., 2015). According to these structures, the C-terminal end of Rpl24 relocates and interacts with Rps6 during translocation. The functional significance of this interaction remains to be elucidated. Unfortunately, the C-terminus of Rps6 is missing from these structures, making it challenging to define a function for the phosphorylation of this domain from these structural studies.

Previous studies addressing the physiological function of Rps6 phosphorylation in higher eukaryotes suggested that Rps6 phosphorylation slows translation elongation (Ruvinsky et al., 2005; Chauvin et al., 2014). These are elegant studies, but because they use deletion of the S6 kinases and knock-in alleles of Rps6, they cannot account for the likely cellular adaptation that occurs in response to these chronic deficiencies, as the authors acknowledge. Here we also queried whether Rps6 phosphorylation plays a role in translation in yeast. To this end, we generated both polysome profiles and ribosome profiles from wt, RPS6AA, and Ypk-inhibited cells after refeeding with glucose to stimulate translation. Of importance, our ability to acutely inactivate the kinases responsible for Rps6 phosphorylation avoids potential confounding effects arising from cellular adaptation. After extensive analyses, an overt role for Rps6 phosphorylation could be ascribed to neither global translation nor translational regulation of specific subsets of mRNAs. Specifically, we sought changes in global transcription that would have been apparent in the polysome profiling, whereas transcript-specific effects would have been observable in the ribosome profiling. We found no changes in the density of ribosome-protected fragments across conditions, implying that Rps6 phosphorylation does not affect the rate of protein synthesis of individual transcripts. In addition, alternative translational processes, including pausing, frameshifting of specific messages, reinitiation on GCN4 transcripts, and global stop-codon readthrough, appeared to occur independently of the phosphorylation status of Rps6. Finally, and in contrast to previous studies, our experimental setup was chosen to potentially expose a role of Rps6 phosphorylation in the pioneering round of translation of individual transcripts. Specifically, we assumed that refeeding of cells after 1 h of glucose starvation would trigger inaugural translation of newly transcribed messages, which would be visualized at one of the two ribosome profiling time points. Thus, under the experimental conditions used, we find no evidence to support a role for Rps6 phosphorylation in any aspect of translation. One exception to this general conclusion is the reduction of ribosomes on 5’ leaders in RPS6AA cells and, to a lesser extent, in cells lacking Ypk1–3 activities. Ribosomes accumulate on 5’ leaders in starved cells (Ingolia et al., 2009), but the physiological role underlying this phenomenon is unknown. The observed reduction requires further study to assess its statistical significance under the tested conditions.

Recent work implicated S6 kinases and Rps6 phosphorylation in the ribosome biogenesis transcriptional program in mouse liver (Chauvin et al., 2014). Although such a function could potentially be conserved in yeast, after analysis of our mRNA-seq data used for the ribosome profiling, we failed to detect Rps6 phosphorylation–dependent transcriptional regulation of any mRNA subgroups. Thus, like translation elongation, the influence of Rps6 phosphorylation on transcription appears to be restricted to higher eukaryotes.

MATERIALS AND METHODSYeast cultures and inhibitor concentrationsS. cerevisiae strains and plasmids used in the study are listed in Supplemental Tables S1 and S2. Standard protocols were followed to obtain the strains. Cells were grown in YPD (1% yeast extract, 2% peptone, 2% glucose) at 30°C if not indicated otherwise.Rapamycin was used at 200 ng/ml, wortmannin at 10 μg/ml, and cycloheximide at 25 μg/ml.Cells for carbon starvation experiments were grown in complete synthetic medium with glutamine (Gln) as the nitrogen source at 30°C, filtered, and resuspended in the same medium lacking a carbon source or with galactose instead of glucose (Glc). At the indicated times after the medium change, either 2% Glc or an equal amount of water was added into the cultures. After further incubation for the indicated times, aliquots of cells were collected for protein extraction.Cells for nitrogen starvation experiments were grown in yeast nitrogen base (YNB) (-N) + 2% Glc, 0.2% Gln, filtered, and resuspended in YNB (-N) + 2% Glc. At 20 min later, either 0.2% Gln or an equal amount of water was added to the cell cultures.For spot assays, cells grown overnight were diluted to OD600 nM = 0.1 and grown to exponential phase. Serial dilutions at 1:10 were spotted after normalization of cell numbers. The plates were incubated for 2–3 d at 30°C.For growth assays, exponentially growing cells were diluted to OD600 nM = 0.005 in the indicated media. A 200-μl amount of cells along with 2 μl of drug solution or drug vehicle alone per well was dispensed in 96-well plates, and the OD600 nM of the wells was recorded every 15 min for 16–24 h at 30°C. The average and SD values were obtained from at least three independent experiments.AntibodiesFor total Rps6 antibody, guinea pig polyclonal antiserum was raised against the peptides RVFFDKRIGQEVDGE and QRALKVRNAQAQREA, and affinity purified (Eurogentec, Seraing, Belgium). Supplemental Figure S1b shows the specificity of the antibodies. Other antibodies used in this study were rabbit anti–phospho-S6 ribosomal protein (S235/S236) at 1:2000 (Rps6-PP; 4858S; Cell Signaling Technology, Danvers, MA), rabbit anti–phospho-Akt substrate (RXXS*/T*) at 1:5000 (9614S; Cell Signaling), rabbit anti-Hog1 at 1:1000 (y-215; Santa Cruz Biotechnology, Dallas, TX), mouse anti-Rpl3 at 1:5000 (Developmental Studies Hybridoma Bank, Iowa City, IA), mouse anti-Flag at 1:1000 (F1804; Sigma-Aldrich, St. Louis, MO), rabbit anti–Ypk1 pT662 at 1:500, rabbit anti–Sch9 pT737 at 1:500, goat anti-Ypk1 at 1:1000 (yl-15; Santa Cruz Biotechnology), and the corresponding IRDye infrared dye–labeled secondary antibodies at 1:10,000 (Li-Cor Biosciences, Lincoln, NE). The signals were detected by Odyssey Imaging Systems.Denaturing protein extractionProtein extraction was performed based on the trichloroacetic acid–urea extraction method as previously described (Urban et al., 2007). PPi: 10 mM NaF, 10 mM p-nitrophenylphosphate, 10 mM Na2P2O4, and 10 mM β-glycerophosphate. PI: 1× Roche protease inhibitor cocktail and 1 mM phenylmethylsulfonyl fluoride.Extraction of ribosomesExponentially growing cells were treated with 0.1 mg/ml cycloheximide (CHX). After 10 min of incubation, cells were harvested, washed, and lysed with ribosome lysis buffer (10 mM Tris-HCl, pH 7.4, 100 mM NaCl, 30 mM MgCl2, 100 μg/ml CHX, 200 μg/ml heparin). The extract was centrifuged for 10 min at 13,000 rpm at 4°C. The OD260 of 1:100 dilutions of the supernatant was measured. After the addition of glycerol with the final concentration of 10%, the samples were stored at −70°C.Preparation of recombinant proteins and kinase assaysPurification of TORC1 from RL170-2c cells was performed as described previously for TORC2 (Gaubitz et al., 2015). Glutathione S-transferase (GST)–Ypk3 was expressed under a galactose-inducible promoter in yeast from a 2-μ plasmid. After a 4-h galactose induction, yeast extract was prepared, and the fusion protein was purified using Glutathione Sepharose 4B from GE Healthcare Life Sciences (Little Chalfont, United Kingdom) according to the manufacturer’s instructions. The eluted protein was dialyzed, aliquotted, and frozen at −70°C. TORC1 kinase reactions were performed in kinase buffer (2.5 mM 3-[(3-cholamidopropyl)dimethylammonio]-1-propanesulfonate [CHAPS], 25 mM 4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid [HEPES], 150 mM KCl, 5 mM MgCl2, and 1 mM dithiothreitol [DTT]) in a final volume of 30 μl. Reactions were started with the addition of 300 μM ATP, 125 mM MgCl2, and 10 μCi of [γ-32P]ATP and then shaken for 20 min at 30°C. After the addition of 6 μl of 6× SDS–PAGE sample buffer, samples were heated to 65°C for 10 min. Samples run in SDS–PAGE gels were stained with SYPRO Ruby Protein Gel Stain (Thermo Fisher Scientific, Waltham, MA) and analyzed using a Bio-Rad Molecular Imager.For Ypk3 and Ypk1 kinase assays, ribosomes were used as substrates. The ribosomal extract prepared as described was loaded on top of a 10% sucrose cushion prepared in ribosome lysis buffer and ultracentrifuged for 3.5 h at 35,000 rpm. The pellet was resuspended in the corresponding kinase buffer.A yeast lysate with Ypk3-FLAG fusion protein was prepared, and the fusion protein was affinity purified using anti-Flag antibody–coupled magnetic beads (Dynabeads M-280 Sheep Anti-Mouse IgG; Thermo Fisher Scientific). The fusion protein was eluted in 1× phosphate-buffered saline (PBS) with 100 μg/ml of 3xFLAG Peptide (F4799; Sigma-Aldrich). Ypk3 kinase reactions were performed in kinase buffer (1× PBS, 20% glycerol, 0.5% Tween-20, 4 mM MgCl2, 10 mM DTT, and 600 μM ATP) in a final volume of 30 μl for 20 min at 30°C.GST-Ypk1 was purified according to the cited protocol for GST-Ypk3 and eluted in the following buffer: 2.5 mM CHAPS, 25 mM HEPES, 150 mM KCl, 5 mM MgCl2, and 1 mM DTT. Ypk1 kinase reactions were performed in kinase buffer (2.5 mM CHAPS, 25 mM HEPES, 150 mM KCl, 5 mM MgCl2, 1 mM DTT, and 600μM ATP) in a final volume of 30 μl for 20 min at 30°C.Sucrose density gradient centrifugation–polysome profilingRibosomal extracts were prepared as described. Volumes of supernatant containing 10 OD260 units were loaded on top of a 7–50% sucrose gradient prepared in gradient buffer (50 mM Tris-AcO, pH 7.0, 50 mM NH4Cl, 12 mM MgCl2, 1 mM DTT). Gradients were ultracentrifuged for 3.5 h at 35,000 rpm, and the ultraviolet profile was recorded using an IG gradient fractionator.Ribosome profilingThe ypk1as ypk2 ypk3as RPS6wt and ypk1as ypk2 ypk3as RPS6AA cells were grown in minimal synthetic medium with 2% Glc and 0.2% Gln at 30°C. Once they reached exponential growth, the cells were filtered and resuspended in minimal synthetic medium with 0.2% Gln without any carbon source. After a 1-h incubation at 30°C, cells were treated with 500 nM 1N-PP1 or dimethyl sulfoxide (DMSO). After a 5-min incubation with the drug/vehicle, the cells were refed with 2% glucose. After 20 or 40 min, cells were collected by filtration, resuspended in polysome buffer, and frozen rapidly in liquid nitrogen.Ribosome footprint and total RNA libraries were prepared using the ARTseq Ribosome Profiling Kit (Yeast, RPYSC12116; Epicentre, Madison, WI) according to the manufacturer’s protocol with slight changes. For the preparation of ribosome-protected RNA fragments, 20 μg of nuclease-digested RNA samples were run on a 12% urea-polyacrylamide gel before rRNA depletion. The gel slices corresponding to fragment sizes of 28–30 nt were excised, and samples purified from these slices were further used for rRNA depletion and library preparation. The libraries were sequenced on an Illumina HiSeq2500 machine on the Genomics Platform (Geneva, Switzerland).Initial processing and alignment of sequence readsAdapter sequence (5′-AGATCGGAAGAGCACACGTCT-3′) was trimmed from raw reads using Cutadapt (Martin, 2011). To reduce contamination from noncoding RNAs (such as rRNAs, tRNAs, small nuclear RNAs, and small nucleolar RNAs), trimmed reads were aligned to relevant sequences downloaded from the Saccharomyces Genome Database (Cherry et al., 2012; (downloads .yeastgenome.org/sequence/S288C_reference/rna/rna_coding .fasta.gz), using the Bowtie short read alignment program (Langmead et al., 2009), and no more than three mismatches were allowed for these alignments (-v 3). Reads aligning to these sequences were removed from the data set for further analysis. Read-length distributions were analyzed to determine the most prevalent read lengths among the samples. Reads of <20 or >80 nt were discarded from further analyses.The S. cerevisiae genome was obtained from the National Center for Biotechnology Information (GCA_000146055.2, dated December 2013; Engel et al., 2014), to which reads were aligned using Bowtie (Langmead et al., 2009). Parameters used for Bowtie alignments were -m 1 -v 3, that is, no more than three mismatches were allowed, and nonunique mappings were discarded. To evaluate the number of reads mapped on each gene, gene annotations were procured from the Saccharomyces Genome Database (Cherry et al., 2012). Reproducibility of the data from all the replicates was evaluated by calculating Pearson’s r. Specifically, the number of raw reads mapped on each gene for ribosomal profiling (two replicates) and total RNA-seq (three replicates) were used for calculating a Pearson’s r.Metagene analysisGenes with at least 100 mapped alignments were used for the production of metagene profiles at the location of translation initiation and termination. To analyze the read density around the initiation and termination sites, 30 nt upstream and downstream of the start and stop codons were used, respectively.Differential gene expression analysisNormalization of mapped reads was performed by dividing the total number of reads mapped in a sample by the total number of the reads mapped for the sample containing the lowest number of mapped reads. This was performed separately for ribo-seq and RNA-seq samples. Ribosomal profiling reads were assigned to gene coordinates based on the inferred location of the A-site. The A-site codon of the elongating ribosome was assigned at 15 nt downstream of the 5′ end of reads of length 28–32.Average read counts for each gene for ribosomal profiling data (two replicates) and RNA-seq data (three replicates) were used for differential gene expression calculation. The read counts were compared in a pairwise manner between mutants and wild type and between different time points for the same treatments. The Z-score transformation of change fold log ratios was carried out as described previously (Andreev et al., 2015). In brief, genes were grouped into bins of 300 based on the lowest expression signal (ribo-seq or RNA-seq or both across compared conditions). Variation in the distribution of change fold ratios was used to calculate the Z-score. Profiles for differentially regulated genes were constructed by calculating the average of normalized reads mapped on each position of the gene based on the inferred location of the A-site.Assigning mapped reads to 5′ leaders, 3′ UTRs, and coding sequence regionsGenomic alignments were used to assign reads to 5′ leaders, 3′ UTRs, and coding sequence (CDS) regions. The 5′ leaders and 3′ UTR annotations were obtained from Nagalakshmi et al. (2008). Reads were assigned to 2755 genes that have both 5′ leaders and 3′ UTRs annotated, and genes with overlapping features were discarded for this analysis. End positions of 5′ leaders were modified to 17 nt upstream of the first base of the CDS, and start positions of 3′ UTRs were modified to 13 nt upstream of the last base of the CDS. Upstream ORFs were analyzed for differential expression using the same procedure as for CDS regions.Frameshift analysisRibo-seq reads aligning to no more than 100 locations were used for generating ribosome density profiles for S. cerevisiae genes (ABP140, EST3, OAZ1) and the TY1 transposable element, which are known to use +1 frameshifting in their expression (Bekaert et al., 2010). For analysis of triplet periodicity, only 29-nt-long reads were used. The fraction of reads aligning to each subcodon position of the A-site (using 15-nt offset from its 5′ end) from all conditions was calculated to assess triplet periodicity.

MATERIALS AND METHODS

Yeast cultures and inhibitor concentrationsS. cerevisiae strains and plasmids used in the study are listed in Supplemental Tables S1 and S2. Standard protocols were followed to obtain the strains. Cells were grown in YPD (1% yeast extract, 2% peptone, 2% glucose) at 30°C if not indicated otherwise.Rapamycin was used at 200 ng/ml, wortmannin at 10 μg/ml, and cycloheximide at 25 μg/ml.Cells for carbon starvation experiments were grown in complete synthetic medium with glutamine (Gln) as the nitrogen source at 30°C, filtered, and resuspended in the same medium lacking a carbon source or with galactose instead of glucose (Glc). At the indicated times after the medium change, either 2% Glc or an equal amount of water was added into the cultures. After further incubation for the indicated times, aliquots of cells were collected for protein extraction.Cells for nitrogen starvation experiments were grown in yeast nitrogen base (YNB) (-N) + 2% Glc, 0.2% Gln, filtered, and resuspended in YNB (-N) + 2% Glc. At 20 min later, either 0.2% Gln or an equal amount of water was added to the cell cultures.For spot assays, cells grown overnight were diluted to OD600 nM = 0.1 and grown to exponential phase. Serial dilutions at 1:10 were spotted after normalization of cell numbers. The plates were incubated for 2–3 d at 30°C.For growth assays, exponentially growing cells were diluted to OD600 nM = 0.005 in the indicated media. A 200-μl amount of cells along with 2 μl of drug solution or drug vehicle alone per well was dispensed in 96-well plates, and the OD600 nM of the wells was recorded every 15 min for 16–24 h at 30°C. The average and SD values were obtained from at least three independent experiments.

Yeast cultures and inhibitor concentrations

S. cerevisiae strains and plasmids used in the study are listed in Supplemental Tables S1 and S2. Standard protocols were followed to obtain the strains. Cells were grown in YPD (1% yeast extract, 2% peptone, 2% glucose) at 30°C if not indicated otherwise.

Rapamycin was used at 200 ng/ml, wortmannin at 10 μg/ml, and cycloheximide at 25 μg/ml.

Cells for carbon starvation experiments were grown in complete synthetic medium with glutamine (Gln) as the nitrogen source at 30°C, filtered, and resuspended in the same medium lacking a carbon source or with galactose instead of glucose (Glc). At the indicated times after the medium change, either 2% Glc or an equal amount of water was added into the cultures. After further incubation for the indicated times, aliquots of cells were collected for protein extraction.

Cells for nitrogen starvation experiments were grown in yeast nitrogen base (YNB) (-N) + 2% Glc, 0.2% Gln, filtered, and resuspended in YNB (-N) + 2% Glc. At 20 min later, either 0.2% Gln or an equal amount of water was added to the cell cultures.

For spot assays, cells grown overnight were diluted to OD600 nM = 0.1 and grown to exponential phase. Serial dilutions at 1:10 were spotted after normalization of cell numbers. The plates were incubated for 2–3 d at 30°C.

For growth assays, exponentially growing cells were diluted to OD600 nM = 0.005 in the indicated media. A 200-μl amount of cells along with 2 μl of drug solution or drug vehicle alone per well was dispensed in 96-well plates, and the OD600 nM of the wells was recorded every 15 min for 16–24 h at 30°C. The average and SD values were obtained from at least three independent experiments.

AntibodiesFor total Rps6 antibody, guinea pig polyclonal antiserum was raised against the peptides RVFFDKRIGQEVDGE and QRALKVRNAQAQREA, and affinity purified (Eurogentec, Seraing, Belgium). Supplemental Figure S1b shows the specificity of the antibodies. Other antibodies used in this study were rabbit anti–phospho-S6 ribosomal protein (S235/S236) at 1:2000 (Rps6-PP; 4858S; Cell Signaling Technology, Danvers, MA), rabbit anti–phospho-Akt substrate (RXXS*/T*) at 1:5000 (9614S; Cell Signaling), rabbit anti-Hog1 at 1:1000 (y-215; Santa Cruz Biotechnology, Dallas, TX), mouse anti-Rpl3 at 1:5000 (Developmental Studies Hybridoma Bank, Iowa City, IA), mouse anti-Flag at 1:1000 (F1804; Sigma-Aldrich, St. Louis, MO), rabbit anti–Ypk1 pT662 at 1:500, rabbit anti–Sch9 pT737 at 1:500, goat anti-Ypk1 at 1:1000 (yl-15; Santa Cruz Biotechnology), and the corresponding IRDye infrared dye–labeled secondary antibodies at 1:10,000 (Li-Cor Biosciences, Lincoln, NE). The signals were detected by Odyssey Imaging Systems.

For total Rps6 antibody, guinea pig polyclonal antiserum was raised against the peptides RVFFDKRIGQEVDGE and QRALKVRNAQAQREA, and affinity purified (Eurogentec, Seraing, Belgium). Supplemental Figure S1b shows the specificity of the antibodies. Other antibodies used in this study were rabbit anti–phospho-S6 ribosomal protein (S235/S236) at 1:2000 (Rps6-PP; 4858S; Cell Signaling Technology, Danvers, MA), rabbit anti–phospho-Akt substrate (RXXS*/T*) at 1:5000 (9614S; Cell Signaling), rabbit anti-Hog1 at 1:1000 (y-215; Santa Cruz Biotechnology, Dallas, TX), mouse anti-Rpl3 at 1:5000 (Developmental Studies Hybridoma Bank, Iowa City, IA), mouse anti-Flag at 1:1000 (F1804; Sigma-Aldrich, St. Louis, MO), rabbit anti–Ypk1 pT662 at 1:500, rabbit anti–Sch9 pT737 at 1:500, goat anti-Ypk1 at 1:1000 (yl-15; Santa Cruz Biotechnology), and the corresponding IRDye infrared dye–labeled secondary antibodies at 1:10,000 (Li-Cor Biosciences, Lincoln, NE). The signals were detected by Odyssey Imaging Systems.

Denaturing protein extractionProtein extraction was performed based on the trichloroacetic acid–urea extraction method as previously described (Urban et al., 2007). PPi: 10 mM NaF, 10 mM p-nitrophenylphosphate, 10 mM Na2P2O4, and 10 mM β-glycerophosphate. PI: 1× Roche protease inhibitor cocktail and 1 mM phenylmethylsulfonyl fluoride.

Denaturing protein extraction

Protein extraction was performed based on the trichloroacetic acid–urea extraction method as previously described (Urban et al., 2007). PPi: 10 mM NaF, 10 mM p-nitrophenylphosphate, 10 mM Na2P2O4, and 10 mM β-glycerophosphate. PI: 1× Roche protease inhibitor cocktail and 1 mM phenylmethylsulfonyl fluoride.

Extraction of ribosomesExponentially growing cells were treated with 0.1 mg/ml cycloheximide (CHX). After 10 min of incubation, cells were harvested, washed, and lysed with ribosome lysis buffer (10 mM Tris-HCl, pH 7.4, 100 mM NaCl, 30 mM MgCl2, 100 μg/ml CHX, 200 μg/ml heparin). The extract was centrifuged for 10 min at 13,000 rpm at 4°C. The OD260 of 1:100 dilutions of the supernatant was measured. After the addition of glycerol with the final concentration of 10%, the samples were stored at −70°C.

Extraction of ribosomes

Exponentially growing cells were treated with 0.1 mg/ml cycloheximide (CHX). After 10 min of incubation, cells were harvested, washed, and lysed with ribosome lysis buffer (10 mM Tris-HCl, pH 7.4, 100 mM NaCl, 30 mM MgCl2, 100 μg/ml CHX, 200 μg/ml heparin). The extract was centrifuged for 10 min at 13,000 rpm at 4°C. The OD260 of 1:100 dilutions of the supernatant was measured. After the addition of glycerol with the final concentration of 10%, the samples were stored at −70°C.

Preparation of recombinant proteins and kinase assaysPurification of TORC1 from RL170-2c cells was performed as described previously for TORC2 (Gaubitz et al., 2015). Glutathione S-transferase (GST)–Ypk3 was expressed under a galactose-inducible promoter in yeast from a 2-μ plasmid. After a 4-h galactose induction, yeast extract was prepared, and the fusion protein was purified using Glutathione Sepharose 4B from GE Healthcare Life Sciences (Little Chalfont, United Kingdom) according to the manufacturer’s instructions. The eluted protein was dialyzed, aliquotted, and frozen at −70°C. TORC1 kinase reactions were performed in kinase buffer (2.5 mM 3-[(3-cholamidopropyl)dimethylammonio]-1-propanesulfonate [CHAPS], 25 mM 4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid [HEPES], 150 mM KCl, 5 mM MgCl2, and 1 mM dithiothreitol [DTT]) in a final volume of 30 μl. Reactions were started with the addition of 300 μM ATP, 125 mM MgCl2, and 10 μCi of [γ-32P]ATP and then shaken for 20 min at 30°C. After the addition of 6 μl of 6× SDS–PAGE sample buffer, samples were heated to 65°C for 10 min. Samples run in SDS–PAGE gels were stained with SYPRO Ruby Protein Gel Stain (Thermo Fisher Scientific, Waltham, MA) and analyzed using a Bio-Rad Molecular Imager.For Ypk3 and Ypk1 kinase assays, ribosomes were used as substrates. The ribosomal extract prepared as described was loaded on top of a 10% sucrose cushion prepared in ribosome lysis buffer and ultracentrifuged for 3.5 h at 35,000 rpm. The pellet was resuspended in the corresponding kinase buffer.A yeast lysate with Ypk3-FLAG fusion protein was prepared, and the fusion protein was affinity purified using anti-Flag antibody–coupled magnetic beads (Dynabeads M-280 Sheep Anti-Mouse IgG; Thermo Fisher Scientific). The fusion protein was eluted in 1× phosphate-buffered saline (PBS) with 100 μg/ml of 3xFLAG Peptide (F4799; Sigma-Aldrich). Ypk3 kinase reactions were performed in kinase buffer (1× PBS, 20% glycerol, 0.5% Tween-20, 4 mM MgCl2, 10 mM DTT, and 600 μM ATP) in a final volume of 30 μl for 20 min at 30°C.GST-Ypk1 was purified according to the cited protocol for GST-Ypk3 and eluted in the following buffer: 2.5 mM CHAPS, 25 mM HEPES, 150 mM KCl, 5 mM MgCl2, and 1 mM DTT. Ypk1 kinase reactions were performed in kinase buffer (2.5 mM CHAPS, 25 mM HEPES, 150 mM KCl, 5 mM MgCl2, 1 mM DTT, and 600μM ATP) in a final volume of 30 μl for 20 min at 30°C.

Preparation of recombinant proteins and kinase assays

Purification of TORC1 from RL170-2c cells was performed as described previously for TORC2 (Gaubitz et al., 2015). Glutathione S-transferase (GST)–Ypk3 was expressed under a galactose-inducible promoter in yeast from a 2-μ plasmid. After a 4-h galactose induction, yeast extract was prepared, and the fusion protein was purified using Glutathione Sepharose 4B from GE Healthcare Life Sciences (Little Chalfont, United Kingdom) according to the manufacturer’s instructions. The eluted protein was dialyzed, aliquotted, and frozen at −70°C. TORC1 kinase reactions were performed in kinase buffer (2.5 mM 3-[(3-cholamidopropyl)dimethylammonio]-1-propanesulfonate [CHAPS], 25 mM 4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid [HEPES], 150 mM KCl, 5 mM MgCl2, and 1 mM dithiothreitol [DTT]) in a final volume of 30 μl. Reactions were started with the addition of 300 μM ATP, 125 mM MgCl2, and 10 μCi of [γ-32P]ATP and then shaken for 20 min at 30°C. After the addition of 6 μl of 6× SDS–PAGE sample buffer, samples were heated to 65°C for 10 min. Samples run in SDS–PAGE gels were stained with SYPRO Ruby Protein Gel Stain (Thermo Fisher Scientific, Waltham, MA) and analyzed using a Bio-Rad Molecular Imager.

For Ypk3 and Ypk1 kinase assays, ribosomes were used as substrates. The ribosomal extract prepared as described was loaded on top of a 10% sucrose cushion prepared in ribosome lysis buffer and ultracentrifuged for 3.5 h at 35,000 rpm. The pellet was resuspended in the corresponding kinase buffer.

A yeast lysate with Ypk3-FLAG fusion protein was prepared, and the fusion protein was affinity purified using anti-Flag antibody–coupled magnetic beads (Dynabeads M-280 Sheep Anti-Mouse IgG; Thermo Fisher Scientific). The fusion protein was eluted in 1× phosphate-buffered saline (PBS) with 100 μg/ml of 3xFLAG Peptide (F4799; Sigma-Aldrich). Ypk3 kinase reactions were performed in kinase buffer (1× PBS, 20% glycerol, 0.5% Tween-20, 4 mM MgCl2, 10 mM DTT, and 600 μM ATP) in a final volume of 30 μl for 20 min at 30°C.

GST-Ypk1 was purified according to the cited protocol for GST-Ypk3 and eluted in the following buffer: 2.5 mM CHAPS, 25 mM HEPES, 150 mM KCl, 5 mM MgCl2, and 1 mM DTT. Ypk1 kinase reactions were performed in kinase buffer (2.5 mM CHAPS, 25 mM HEPES, 150 mM KCl, 5 mM MgCl2, 1 mM DTT, and 600μM ATP) in a final volume of 30 μl for 20 min at 30°C.

Sucrose density gradient centrifugation–polysome profilingRibosomal extracts were prepared as described. Volumes of supernatant containing 10 OD260 units were loaded on top of a 7–50% sucrose gradient prepared in gradient buffer (50 mM Tris-AcO, pH 7.0, 50 mM NH4Cl, 12 mM MgCl2, 1 mM DTT). Gradients were ultracentrifuged for 3.5 h at 35,000 rpm, and the ultraviolet profile was recorded using an IG gradient fractionator.

Sucrose density gradient centrifugation–polysome profiling

Ribosomal extracts were prepared as described. Volumes of supernatant containing 10 OD260 units were loaded on top of a 7–50% sucrose gradient prepared in gradient buffer (50 mM Tris-AcO, pH 7.0, 50 mM NH4Cl, 12 mM MgCl2, 1 mM DTT). Gradients were ultracentrifuged for 3.5 h at 35,000 rpm, and the ultraviolet profile was recorded using an IG gradient fractionator.

Ribosome profilingThe ypk1as ypk2 ypk3as RPS6wt and ypk1as ypk2 ypk3as RPS6AA cells were grown in minimal synthetic medium with 2% Glc and 0.2% Gln at 30°C. Once they reached exponential growth, the cells were filtered and resuspended in minimal synthetic medium with 0.2% Gln without any carbon source. After a 1-h incubation at 30°C, cells were treated with 500 nM 1N-PP1 or dimethyl sulfoxide (DMSO). After a 5-min incubation with the drug/vehicle, the cells were refed with 2% glucose. After 20 or 40 min, cells were collected by filtration, resuspended in polysome buffer, and frozen rapidly in liquid nitrogen.Ribosome footprint and total RNA libraries were prepared using the ARTseq Ribosome Profiling Kit (Yeast, RPYSC12116; Epicentre, Madison, WI) according to the manufacturer’s protocol with slight changes. For the preparation of ribosome-protected RNA fragments, 20 μg of nuclease-digested RNA samples were run on a 12% urea-polyacrylamide gel before rRNA depletion. The gel slices corresponding to fragment sizes of 28–30 nt were excised, and samples purified from these slices were further used for rRNA depletion and library preparation. The libraries were sequenced on an Illumina HiSeq2500 machine on the Genomics Platform (Geneva, Switzerland).

Ribosome profiling

The ypk1as ypk2 ypk3as RPS6wt and ypk1as ypk2 ypk3as RPS6AA cells were grown in minimal synthetic medium with 2% Glc and 0.2% Gln at 30°C. Once they reached exponential growth, the cells were filtered and resuspended in minimal synthetic medium with 0.2% Gln without any carbon source. After a 1-h incubation at 30°C, cells were treated with 500 nM 1N-PP1 or dimethyl sulfoxide (DMSO). After a 5-min incubation with the drug/vehicle, the cells were refed with 2% glucose. After 20 or 40 min, cells were collected by filtration, resuspended in polysome buffer, and frozen rapidly in liquid nitrogen.

Ribosome footprint and total RNA libraries were prepared using the ARTseq Ribosome Profiling Kit (Yeast, RPYSC12116; Epicentre, Madison, WI) according to the manufacturer’s protocol with slight changes. For the preparation of ribosome-protected RNA fragments, 20 μg of nuclease-digested RNA samples were run on a 12% urea-polyacrylamide gel before rRNA depletion. The gel slices corresponding to fragment sizes of 28–30 nt were excised, and samples purified from these slices were further used for rRNA depletion and library preparation. The libraries were sequenced on an Illumina HiSeq2500 machine on the Genomics Platform (Geneva, Switzerland).

Initial processing and alignment of sequence readsAdapter sequence (5′-AGATCGGAAGAGCACACGTCT-3′) was trimmed from raw reads using Cutadapt (Martin, 2011). To reduce contamination from noncoding RNAs (such as rRNAs, tRNAs, small nuclear RNAs, and small nucleolar RNAs), trimmed reads were aligned to relevant sequences downloaded from the Saccharomyces Genome Database (Cherry et al., 2012; (downloads .yeastgenome.org/sequence/S288C_reference/rna/rna_coding .fasta.gz), using the Bowtie short read alignment program (Langmead et al., 2009), and no more than three mismatches were allowed for these alignments (-v 3). Reads aligning to these sequences were removed from the data set for further analysis. Read-length distributions were analyzed to determine the most prevalent read lengths among the samples. Reads of <20 or >80 nt were discarded from further analyses.The S. cerevisiae genome was obtained from the National Center for Biotechnology Information (GCA_000146055.2, dated December 2013; Engel et al., 2014), to which reads were aligned using Bowtie (Langmead et al., 2009). Parameters used for Bowtie alignments were -m 1 -v 3, that is, no more than three mismatches were allowed, and nonunique mappings were discarded. To evaluate the number of reads mapped on each gene, gene annotations were procured from the Saccharomyces Genome Database (Cherry et al., 2012). Reproducibility of the data from all the replicates was evaluated by calculating Pearson’s r. Specifically, the number of raw reads mapped on each gene for ribosomal profiling (two replicates) and total RNA-seq (three replicates) were used for calculating a Pearson’s r.

Initial processing and alignment of sequence reads

Adapter sequence (5′-AGATCGGAAGAGCACACGTCT-3′) was trimmed from raw reads using Cutadapt (Martin, 2011). To reduce contamination from noncoding RNAs (such as rRNAs, tRNAs, small nuclear RNAs, and small nucleolar RNAs), trimmed reads were aligned to relevant sequences downloaded from the Saccharomyces Genome Database (Cherry et al., 2012; (downloads .yeastgenome.org/sequence/S288C_reference/rna/rna_coding .fasta.gz), using the Bowtie short read alignment program (Langmead et al., 2009), and no more than three mismatches were allowed for these alignments (-v 3). Reads aligning to these sequences were removed from the data set for further analysis. Read-length distributions were analyzed to determine the most prevalent read lengths among the samples. Reads of <20 or >80 nt were discarded from further analyses.

The S. cerevisiae genome was obtained from the National Center for Biotechnology Information (GCA_000146055.2, dated December 2013; Engel et al., 2014), to which reads were aligned using Bowtie (Langmead et al., 2009). Parameters used for Bowtie alignments were -m 1 -v 3, that is, no more than three mismatches were allowed, and nonunique mappings were discarded. To evaluate the number of reads mapped on each gene, gene annotations were procured from the Saccharomyces Genome Database (Cherry et al., 2012). Reproducibility of the data from all the replicates was evaluated by calculating Pearson’s r. Specifically, the number of raw reads mapped on each gene for ribosomal profiling (two replicates) and total RNA-seq (three replicates) were used for calculating a Pearson’s r.

Metagene analysisGenes with at least 100 mapped alignments were used for the production of metagene profiles at the location of translation initiation and termination. To analyze the read density around the initiation and termination sites, 30 nt upstream and downstream of the start and stop codons were used, respectively.

Metagene analysis

Genes with at least 100 mapped alignments were used for the production of metagene profiles at the location of translation initiation and termination. To analyze the read density around the initiation and termination sites, 30 nt upstream and downstream of the start and stop codons were used, respectively.

Differential gene expression analysisNormalization of mapped reads was performed by dividing the total number of reads mapped in a sample by the total number of the reads mapped for the sample containing the lowest number of mapped reads. This was performed separately for ribo-seq and RNA-seq samples. Ribosomal profiling reads were assigned to gene coordinates based on the inferred location of the A-site. The A-site codon of the elongating ribosome was assigned at 15 nt downstream of the 5′ end of reads of length 28–32.Average read counts for each gene for ribosomal profiling data (two replicates) and RNA-seq data (three replicates) were used for differential gene expression calculation. The read counts were compared in a pairwise manner between mutants and wild type and between different time points for the same treatments. The Z-score transformation of change fold log ratios was carried out as described previously (Andreev et al., 2015). In brief, genes were grouped into bins of 300 based on the lowest expression signal (ribo-seq or RNA-seq or both across compared conditions). Variation in the distribution of change fold ratios was used to calculate the Z-score. Profiles for differentially regulated genes were constructed by calculating the average of normalized reads mapped on each position of the gene based on the inferred location of the A-site.

Differential gene expression analysis

Normalization of mapped reads was performed by dividing the total number of reads mapped in a sample by the total number of the reads mapped for the sample containing the lowest number of mapped reads. This was performed separately for ribo-seq and RNA-seq samples. Ribosomal profiling reads were assigned to gene coordinates based on the inferred location of the A-site. The A-site codon of the elongating ribosome was assigned at 15 nt downstream of the 5′ end of reads of length 28–32.

Average read counts for each gene for ribosomal profiling data (two replicates) and RNA-seq data (three replicates) were used for differential gene expression calculation. The read counts were compared in a pairwise manner between mutants and wild type and between different time points for the same treatments. The Z-score transformation of change fold log ratios was carried out as described previously (Andreev et al., 2015). In brief, genes were grouped into bins of 300 based on the lowest expression signal (ribo-seq or RNA-seq or both across compared conditions). Variation in the distribution of change fold ratios was used to calculate the Z-score. Profiles for differentially regulated genes were constructed by calculating the average of normalized reads mapped on each position of the gene based on the inferred location of the A-site.

Assigning mapped reads to 5′ leaders, 3′ UTRs, and coding sequence regionsGenomic alignments were used to assign reads to 5′ leaders, 3′ UTRs, and coding sequence (CDS) regions. The 5′ leaders and 3′ UTR annotations were obtained from Nagalakshmi et al. (2008). Reads were assigned to 2755 genes that have both 5′ leaders and 3′ UTRs annotated, and genes with overlapping features were discarded for this analysis. End positions of 5′ leaders were modified to 17 nt upstream of the first base of the CDS, and start positions of 3′ UTRs were modified to 13 nt upstream of the last base of the CDS. Upstream ORFs were analyzed for differential expression using the same procedure as for CDS regions.

Assigning mapped reads to 5′ leaders, 3′ UTRs, and coding sequence regions

Genomic alignments were used to assign reads to 5′ leaders, 3′ UTRs, and coding sequence (CDS) regions. The 5′ leaders and 3′ UTR annotations were obtained from Nagalakshmi et al. (2008). Reads were assigned to 2755 genes that have both 5′ leaders and 3′ UTRs annotated, and genes with overlapping features were discarded for this analysis. End positions of 5′ leaders were modified to 17 nt upstream of the first base of the CDS, and start positions of 3′ UTRs were modified to 13 nt upstream of the last base of the CDS. Upstream ORFs were analyzed for differential expression using the same procedure as for CDS regions.

Frameshift analysisRibo-seq reads aligning to no more than 100 locations were used for generating ribosome density profiles for S. cerevisiae genes (ABP140, EST3, OAZ1) and the TY1 transposable element, which are known to use +1 frameshifting in their expression (Bekaert et al., 2010). For analysis of triplet periodicity, only 29-nt-long reads were used. The fraction of reads aligning to each subcodon position of the A-site (using 15-nt offset from its 5′ end) from all conditions was calculated to assess triplet periodicity.

Frameshift analysis

Ribo-seq reads aligning to no more than 100 locations were used for generating ribosome density profiles for S. cerevisiae genes (ABP140, EST3, OAZ1) and the TY1 transposable element, which are known to use +1 frameshifting in their expression (Bekaert et al., 2010). For analysis of triplet periodicity, only 29-nt-long reads were used. The fraction of reads aligning to each subcodon position of the A-site (using 15-nt offset from its 5′ end) from all conditions was calculated to assess triplet periodicity.

Supplementary MaterialSupplemental Materials

Supplementary Material

Supplemental Materials
