---
pmid: '21984211'
title: Structure and nucleosome interaction of the yeast NuA4 and Piccolo-NuA4 histone
  acetyltransferase complexes.
authors:
- Chittuluru JR
- Chaban Y
- Monnet-Saksouk J
- Carrozza MJ
- Sapountzi V
- Selleck W
- Huang J
- Utley RT
- Cramet M
- Allard S
- Cai G
- Workman JL
- Fried MG
- Tan S
- Côté J
- Asturias FJ
journal: Nat Struct Mol Biol
year: '2011'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC3210417
doi: 10.1038/nsmb.2128
---

# Structure and nucleosome interaction of the yeast NuA4 and Piccolo-NuA4 histone acetyltransferase complexes.
**Authors:** Chittuluru JR, Chaban Y, Monnet-Saksouk J, Carrozza MJ, Sapountzi V, Selleck W, Huang J, Utley RT, Cramet M, Allard S, Cai G, Workman JL, Fried MG, Tan S, Côté J, Asturias FJ
**Journal:** Nat Struct Mol Biol (2011)
**DOI:** [10.1038/nsmb.2128](https://doi.org/10.1038/nsmb.2128)
**PMC:** [PMC3210417](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3210417/)

## Abstract

1. Nat Struct Mol Biol. 2011 Oct 9;18(11):1196-203. doi: 10.1038/nsmb.2128.

Structure and nucleosome interaction of the yeast NuA4 and Piccolo-NuA4 histone 
acetyltransferase complexes.

Chittuluru JR(1), Chaban Y, Monnet-Saksouk J, Carrozza MJ, Sapountzi V, Selleck 
W, Huang J, Utley RT, Cramet M, Allard S, Cai G, Workman JL, Fried MG, Tan S, 
Côté J, Asturias FJ.

Author information:
(1)Department of Cell Biology, The Scripps Research Institute, La Jolla, 
California, USA.

We have used EM and biochemistry to characterize the structure of NuA4, an 
essential yeast histone acetyltransferase (HAT) complex conserved throughout 
eukaryotes, and we have determined the interaction of NuA4 with the nucleosome 
core particle (NCP). The ATM-related Tra1 subunit, which is shared with the SAGA 
coactivator complex, forms a large domain joined to a second region that 
accommodates the catalytic subcomplex Piccolo and other NuA4 subunits. EM 
analysis of a NuA4-NCP complex shows the NCP bound at the periphery of NuA4. EM 
characterization of Piccolo and Piccolo-NCP provided further information about 
subunit organization and confirmed that histone acetylation requires minimal 
contact with the NCP. A small conserved region at the N terminus of Piccolo 
subunit enhancer of Polycomb-like 1 (Epl1) is essential for NCP interaction, 
whereas the subunit yeast homolog of mammalian Ing1 2 (Yng2) apparently 
positions Piccolo for efficient acetylation of histone H4 or histone H2A tails. 
Taken together, these results provide an understanding of the NuA4 subunit 
organization and the NuA4-NCP interactions.

DOI: 10.1038/nsmb.2128
PMCID: PMC3210417
PMID: 21984211 [Indexed for MEDLINE]

## Full Text

Acetylation is perhaps the best characterized of the histone tail covalent modifications that play a critical role in the regulation of chromatin structure. Large histone acetyltransferase (HAT) complexes target specific histone tail lysines, and the role of different HATs is a matter of intense investigation. NuA4, a large HAT comprising 13 unique polypeptide subunits with a combined molecular weight of 1.0 MDa, is the only HAT whose catalytic subunit, Esa1, is required for cell viability in Saccharomyces cerevisiae1,2. Three NuA4 subunits (Esa1, Yng2, and Epl1) are responsible for NuA4's HAT activity, and they comprise the Piccolo-NuA4 (Piccolo) catalytic subcomplex3,4. NuA4 and Piccolo perform targeted and non-targeted histone H4 and H2A acetylation, respectively. NuA4 is well conserved throughout eukaryotes, and its human homologue, TIP60, plays a critical role in DNA damage response and stem cell regulation (reviewed in5-8). Biochemical analysis of NuA4 subunit interactions has resulted in a model for organization of the complex (Fig. 1a, adapted from9,10) that presents two modules arranged around the scaffold protein Eaf1. Piccolo also exists as a free complex that is responsible for maintaining basal levels of histone H4 and H2A acetylation, an essential cellular function4. The remaining NuA4 subunits have been shown to constitute a recruitment module that targets NuA4 participation in a number of DNA transactions, including DNA repair, transcription initiation, and transcription elongation10-15.

Recent studies have shown that NuA4 often works in tandem with other complexes9,11,15-17. For example, NuA4 and the SAGA coactivator complex (whose catalytic subunit Gcn5 preferentially acetylates histones H3 and H2B)18 can be co-recruited to promoter regions through their single common subunit, the 433 kDa Tra1 polypeptide11,19. Alternatively, through separate mechanisms, both complexes can be recruited to the phosphorylated C-terminal domain of elongating RNA polymerase II15,20,21. Consistent with a common understanding that chromatin modification precedes chromatin remodeling, NuA4-dependent histone acetylation allows binding of the RSC and SWI/SNF chromatin remodelers and stimulates RSC activity16,22,23. RSC and SWI/SNF activities are also stimulated by the HAT function of SAGA. A similar relationship exists between NuA4 and the ATP-dependent SWR1 complex responsible for the incorporation of the H2A.Z histone variant in chromatin17.

In contrast with the wealth of knowledge available about NuA4's functions and functional interactions, little is known about its structure. Previous studies have elucidated the atomic structures of NuA4 subunit domains, such as the HAT domain of Esa124, the Yeats domain of Yaf925, and the Eaf3 chromodomain26,27. However, no structural information is available for either the full NuA4 or Piccolo complexes. Despite the paramount biological importance of chromatin remodelers and modifiers, little structural information is available about their interactions with their nucleosomal substrates. Several studies focusing on individual NuA4 component subunits have attempted to address the means by which NuA4 binds to the nucleosome, but the general mechanism for this interaction is still poorly understood. In this study, we applied single-particle EM and biochemistry to determine the structures of NuA4 and Piccolo and to investigate their mode of interaction with the nucleosome core particle.

ResultsStructural study of the NuA4 complex by single-particle EMWe purified NuA4 from yeast extracts as reported4, through a tandem affinity purification (TAP) tag28 engineered into the C-terminus of its Epl1 subunit (Fig. 1b). Initial EM characterization was performed on NuA4 particles adsorbed onto a continuous amorphous carbon layer and preserved in stain. An average obtained after alignment of stained particle images shows two large NuA4 domains joined by thin connections (Fig. 1c). By applying the random conical tilt (RCT) method29 to tilt-pair images of stained particles, we were able to calculate an initial 3D structure of NuA4 (Fig. 1d). This RCT volume was used as an initial reference for iterative refinement of a three-dimensional structure of NuA4 calculated from ∼35,000 images of NuA4 particles preserved in amorphous ice30 (Fig. 1e).The cryo-EM map of NuA4 shows the structure of the complex, free of specimen preservation artifacts (Fig. 1f). The angular distribution of images in the cryo-EM dataset indicates that an adequate number of NuA4 orientations were recorded and analyzed (Supplementary Fig. 1a), and we estimate the resolution of the NuA4 cryo-EM structure to be ∼20 Å by the 0.5 Fourier Shell Correlation criterion31 (Supplementary Fig. 1b). Class averages obtained by reference-free alignment within groups of cryo-images segregated by orientation show good agreement with projections of the cryo-EM NuA4 volume (Supplementary Fig. 2).NuA4 and SAGA structures and localization of Tra1Subunit Tra1 accounts for nearly half (∼43%) of the total mass of NuA4. Biochemical studies10 have detected only limited contacts between Tra1 and other NuA4 subunits (Fig. 1a), which suggested that Tra1 might constitute a sizable portion of one of the large NuA4 domains. Direct localization of Tra1 by deletion mapping is complicated because the subunit is essential in all of its length32. However, the location of Tra1 within the yeast SAGA HAT complex has been determined by EM analysis,33 and we decided to compare the NuA4 and SAGA structures to elucidate the distribution of Tra1 density within NuA4. We purified SAGA by TAP tagging and recorded EM images of SAGA particles preserved in stain. A 2D map of SAGA (Fig. 2a, left) shows that the top portion of the SAGA structure is formed by two well ordered domains that show a striking resemblance in shape and density distribution to the entirety of the NuA4 structure (Fig. 2a, right).There is no evident sequence homology between NuA4 and SAGA subunits other than Tra1, making a correspondence between the NuA4 and SAGA structures that extends beyond Tra1 an interesting and unexpected finding. Fortunately, besides determining the location of the Tra1 N-terminus by immunolabeling, the previous EM study of SAGA also established the overall boundaries for Tra1-related density by analyzing the structure of a SAGA subunit deletion mutant in which Tra1 was absent33. Tra1 occupies the upper left domain of the SAGA projection map (circled in Fig. 2a), which is nearly identical to the left half of the NuA4 projection map. Therefore, we conclude that most of this half of NuA4, is formed by Tra1. Consequently, the other half of the NuA4 map must contain the majority of the remaining NuA4 subunits, including the catalytic Piccolo subcomplex.Localization of the Piccolo subcomplexWe employed several approaches to localize the Piccolo subcomplex. First, we purified NuA4 from a mutant yeast strain lacking the C-terminus (residues 486–833) of the Epl1 subunit. This Epl1 truncation separates Piccolo subunits from the remainder of NuA44, and this smaller (ΔPiccolo) NuA4 can be purified by Eaf1–TAP10 (Supplementary Fig. 3). Comparing 2D EM maps of ΔPiccolo and wild-type NuA4 shows that whereas the Tra1 domain retains its structural integrity, the other half of the NuA4 structure appears considerably reduced in the absence of Piccolo (Fig. 2b). Therefore, Tra1 and Piccolo are located in opposite domains of the NuA4 structure and, in agreement with results from biochemical analysis of subunit interactions in NuA4 (Fig. 1a), do not contact each other directly10.Localization of the Epl1 Piccolo subunit provided further information about the arrangement of Piccolo within NuA4. A calmodulin binding peptide (CBP) left on the C-terminus of the Piccolo subunit Epl1 after TAP tagging was targeted with an anti-CBP polyclonal antibody (Open Biosystems). A class average calculated from images of antibody-decorated NuA4 particles shows diffuse density near the middle of the lower side of the NuA4 structure (Fig. 2c), indicating that the Epl1 C-terminus is likely mobile and located in that general area. A second labeling experiment was carried out with a gold-conjugated calmodulin probe (Au–CAM)34,35 also directed toward the CBP tag on Epl1. Roughly 18% of NuA4 particles show a small, well-defined area of high density with a radius of ∼2 nm, matching the expected size of the gold cluster after incubation with Au–CAM and calcium (Fig. 2d). To establish the statistical relevance of the signal attributed to the gold probe, we calculated a difference map by subtracting a normalized 2D average of NuA4 imaged with gold in the absence of calcium (calcium is essential for the calmodulin–CBP binding interaction) from an average obtained from gold-labeled NuA4 particles. The brightest peak in this difference map corresponds in position to the gold density apparent in the average obtained from images of gold-labeled NuA4 and has intensity above 3 standard deviations (Fig. 2d and Supplementary Fig. 4). The results from gold cluster and antibody labeling of the Epl1 C-terminus agree, and the gold cluster labeling more precisely localizes the C-terminus of the Epl1 Piccolo subunit to the non-Tra1 (right) domain of the NuA4 structure, near the inter-domain boundary.Interaction of NuA4 with the nucleosome core particleOur next goal was to investigate the interaction between NuA4 and the NCP. NuA4 particles preserved in stain were imaged following incubation with a 9-fold molar excess of recombinant NCPs formed from full-length Xenopus recombinant histones and mouse mammary tumor virus long terminal repeat NucB DNA36,37. The NuA4 concentration in the EM samples (∼20 nM) was close to the estimated Kd for NuA4–NCP interaction and, accordingly, only ∼5% of NuA4 particles from this experiment were bound to NCPs. However, a class average calculated from images of NuA4–NCP complexes clearly shows NCP density adjacent to the position previously identified by labeling of Epl1 with anti-CBP antibodies and Au–CAM (Fig. 2e). The NCP density appears partially disordered (the diameter of the NCP density is somewhat smaller than the X-ray structure of the NCP) indicating a flexible interaction with NuA4. The approximate location of this position can be mapped to the cryo-EM 3D structure of NuA4 (Fig. 2f, black arrowhead).Characterization of the Piccolo–NCP interactionThe results from our NuA4 labeling experiment with the NCP indicated that, at least under the conditions of our EM experiment, NuA4 and the NCP interact with a 1:1 stoichiometry. To determine if this stoichiometry also applied to the Piccolo–NCP complex in solution, we undertook sedimentation equilibrium (analytical) ultracentrifugation (AUC) studies of Piccolo and the Piccolo–NCP complex. For each of the samples analyzed (Piccolo, NCP, and Piccolo–NCP complex), the AUC curves are very well described by a single species. These AUC results establish that Piccolo in solution contains single copies of Esa1, Epl1, and Yng2. Consistent with the EM observations about NuA4 interaction with the NCP, the AUC results show that Piccolo also binds the NCP with a 1:1 stoichiometry (Fig. 3a, Table 1).Analysis of EM images of Piccolo–NCP particles preserved in stain resulted in a class average in which densities corresponding to the NCP and to Piccolo could be readily identified (Fig. 3b). Features in the Piccolo–NCP projection map appeared blurred, suggesting some conformational variability in the Piccolo–NCP particles. Focusing alignment on the NCP or Piccolo portions of Piccolo–NCP images resulted in better-defined maps for interacting Piccolo and NCP. When Piccolo–NCP image alignment is focused on Piccolo, the projection structure of Piccolo bound to the NCP shows a small particle with well-defined features lying adjacent to blurred NCP density (Fig. 3c). To verify this result, we imaged Piccolo in the absence of the NCP. The resulting Piccolo projection map (Fig. 3d) shows strong resemblance to the Piccolo average obtained from the Piccolo–NCP images. We also imaged an additional Piccolo variant (Piccolo v03) that differed from Piccolo v07 and v55 by inclusion of two additional portions of Epl1 (residues 1–49 and 381–485) and the Yng2 PHD domain (residues 221–282) (see Supplementary Figs. 5 through 7 for additional details about the composition and structural analysis of Piccolo deletion variants). The Piccolo v03 projection structure (Fig. 3e) suggests that additional regions of Epl1 and Yng2 contribute density (highlighted by the yellow arrowhead) that is proximal and possibly connected to the portion of the Piccolo structure involved in direct contact with the NCP.When Piccolo–NCP image alignment is focused on NCP density, the resulting Piccolo–NCP projection map shows a well-defined nucleosome in contact with blurred Piccolo density. Interestingly, in this map the NCP appears somewhat flat along the side contacted by Piccolo (Supplementary Fig. 8a, flat portion flanked by yellow arrowheads). A 2D map of the NCP alone calculated from images recorded from the same EM samples used to generate the Piccolo–NCP 2D map is remarkably similar to the NCP portion of the Piccolo–NCP 2D map and clearly shows the same flat feature along the top face (Supplementary Fig. 8b). Comparison with a projection of the nucleosome X-ray structure in approximately the same orientation (Supplementary Fig. 8c) underscores that the flat side of the nucleosome is defined by the entry and exit points of nucleosomal DNA across from the dyad. Because the NCP portion of the Piccolo–NCP 2D map appears circular, except around the area where the NCP is contacted by Piccolo, this 2D analysis establishes the approximate rotational orientation of the NCP in the Piccolo–NCP 2D map, indicating that Piccolo contacts the NCP at a position roughly across (± ∼20°) from the dyad.The recombinant v07 Piccolo construct used in the EM and AUC studies was shown to be fully active on chromatin substrates3,4 and comprised full-length, 6 × His-tagged Esa1 (residues 2–445) but truncated forms of Epl1 and Yng2. The Epl1 truncation comprised residues 51–380 (conserved EPcA domain) and lacked 453 C-terminal residues; the Yng2 truncation mutant comprised residues 2–218, which did not include the Yng2 PHD domain (as with Piccolo v55 in Supplementary Fig. 5). Therefore, a stable 1:1 stoichiometric Piccolo–NCP complex is formed even in the absence of both the Epl1 C-terminus and the Yng2 PHD domain.A 3D map of the Piccolo–NCP complex determined using the RCT method (Fig. 3f) also helped us understand the relative spatial arrangement of Piccolo and the NCP. The NCP X-ray structure38, or an RCT reconstruction of the NCP determined from images of free nucleosomes present in the Piccolo–NCP EM samples (not shown), could be directly docked into the RCT 3D structure of the Piccolo–NCP complex. As was the case with the 2D analysis, the NCP orientation indicated by the 3D docking suggests that Piccolo interacts with a region of the NCP approximately opposite the dyad (Fig. 3g and Supplementary Fig. 8d). This orientation of the NCP in the Piccolo–NCP complex would place an extended Piccolo density near the histone-fold domain of histone H4 (H4HFD), which has been previously identified as the binding site for the Esa1 Piccolo subunit39.To reconcile the information about the Piccolo–NCP interaction with our results about the location of Piccolo within the NuA4 structure and the NuA4–NCP interaction, we sought to dock the Piccolo–NCP RCT map into the larger NuA4 3D structure. Because the resolution of the Piccolo–NCP RCT map is limited as a result of the flexible interaction between Piccolo and the NCP, which in turns limits the accuracy of orientation parameters used to calculate the RCT Piccolo–NCP map, we calculated an RCT reconstruction of Piccolo alone. We docked this structure into the cryo-EM NuA4 map (Supplementary Fig. 9a) and found that Piccolo could be docked in a unique position and orientation within the right half of the NuA4 structure. Docking of the lower resolution Piccolo–NCP map was then guided by the position of Piccolo (Supplementary Fig. 9b). The position of the Piccolo–NCP map determined through this stepwise procedure is consistent with the observed interaction between NuA4 and the NCP (Fig. 2e and Supplementary Fig. 9c). Finally, since our results suggest that Piccolo might have extended contacts with histones and DNA, we pursued a series of biochemical experiments designed to identify elements necessary for binding of NuA4 and Piccolo to the NCP.Nucleosome binding and chromatin acetylation by NuA4We pursued several experiments to investigate in more detail how NuA4 and Piccolo bind the NCP and which subunits make the greatest contributions to binding.First, we determined that the chromodomain subunit Eaf3 does not play a primary role in NuA4 binding to chromatin in vivo. When NuA4 is affinity purified from cell extracts in low-salt conditions, a sizeable amount of all four core histones is detected (Fig. 4a). However, when this is repeated in a mutant strain (eaf1) that disconnects Eaf3 from Epl1 (Fig. 1a;10), histones no longer co-purify with the Eaf3-containing module (Fig. 4a, lane 2).Second, we investigated Epl1 involvement in NCP binding by performing mobility shift assays with constructs comprising different portions of Epl1 and NCPs. We have previously shown that a recombinant Esa1–Epl1 heterodimer has much stronger acetylation activity on free histones and chromatin when compared to Esa1 alone, whereas inclusion of Yng2 in Piccolo drives acetylation only toward chromatin4. Mobility shift assay results now indicate that the Epl1 N-terminal region, encompassing the conserved EPcA domain4,40, has a very strong affinity for the NCP and that in its presence, Epl1 forms a stable complex with mononucleosomes (Fig. 4b).Third, we investigated the effect of Piccolo subunit interactions on the HAT activity of Esa1. We started by obtaining evidence for simultaneous association of Esa1 and Yng2 to the Epl1 N-terminal region by mixing the three recombinant proteins together followed by GST pull down analysis. We could detect efficient binding of both Esa1 and Yng2 (Fig. 4c), since the Epl1 N-terminal region independently binds Esa1 and Yng2, physically linking them together in Piccolo NuA43,4. We then analyzed how interaction with Epl1 and Yng2 modulates the HAT activity of Esa1 toward free histones and a chromatin substrate (Fig. 4d, keeping the amount of Esa1 protein constant). As previously shown, recombinant Esa1 (rEsa1) can only acetylate free histones1. The addition of Yng2 does not affect Esa1 activity, as expected, since no direct interaction between the two proteins was detected4. Addition of Epl1 to Esa1 (rEsa1 + rEpl1) had a striking positive effect on its activity toward free histones but did not enable activity on chromatin. This indicates that Epl1 association with Esa1 increases its specific enzymatic activity, but that the strong affinity of Epl1 for the NCP is non-productive in positioning Esa1 to acetylate H4 or H2A tails. Finally, a mixture of all three proteins (rEsa1 + rEpl1 + rYng2) increased Esa1 specific activity toward free histones to about the same level as Epl1 alone, but acetylation of chromatin was also detected in the presence of Yng2. These data suggest that while Epl1–Esa1 dimers bind efficiently to nucleosomes, Yng2 is required to position Piccolo in a specific orientation that places H4 or H2A histone tails in a productive location near the Esa1 catalytic site.Fourth, we used NCP mobility shift assays to investigate interaction with individual Piccolo subunits (Fig. 4e). The results establish that while Esa1 shows no interaction and tight binding is again observed for the Epl1 N-terminus, Yng2 does seem to directly bind the nucleosome to some detectable level. Furthermore, this interaction does not seem to involve the Yng2 PHD domain (Fig. 4e, lanes 9–11).Fifth, we used additional NCP mobility shift assays to determine the critical component for interaction between the NCP and the Piccolo complex. The assays were performed on two Piccolo constructs identical in their Yng2 (Yng2ΔPHD 1–220) and Esa1 (Esa1 1–445) polypeptides, but differing in their Epl1 component (Fig. 5a). NCP binding was detected for a construct including Epl1 residues 51–380 (the minimal Epl1 domain required for cell viability4), while no NCP binding interaction was observed for a construct including only Epl1 residues 72–380 (Fig. 5b). These results show that Epl1 residues 51–71 are essential for the Piccolo–NCP binding interaction. In contrast, the Yng2 PHD domain is dispensable for binding of mononucleosomes by a recombinant Piccolo complex (Fig. 5c). The observation from this mobility shift assay that the Yng2 PHD domain has no striking effect on Piccolo's ability to bind nucleosomes is in agreement with the negligible role played by this domain in NuA4 and Piccolo HAT activity toward chromatin3,4,41. These observations, along with results from previous and parallel studies3,4, provide an overview of Epl1 binding interactions (Fig. 5d).To summarize, in remarkable agreement with our EM observations, these biochemical results indicate that the N-terminal portion of the Epl1 Piccolo subunit plays a primary role in enabling binding of NuA4 to chromatin, with additional Piccolo subunits being important for modulating the Epl1-based interaction and Esa1 HAT activity.

Structural study of the NuA4 complex by single-particle EMWe purified NuA4 from yeast extracts as reported4, through a tandem affinity purification (TAP) tag28 engineered into the C-terminus of its Epl1 subunit (Fig. 1b). Initial EM characterization was performed on NuA4 particles adsorbed onto a continuous amorphous carbon layer and preserved in stain. An average obtained after alignment of stained particle images shows two large NuA4 domains joined by thin connections (Fig. 1c). By applying the random conical tilt (RCT) method29 to tilt-pair images of stained particles, we were able to calculate an initial 3D structure of NuA4 (Fig. 1d). This RCT volume was used as an initial reference for iterative refinement of a three-dimensional structure of NuA4 calculated from ∼35,000 images of NuA4 particles preserved in amorphous ice30 (Fig. 1e).The cryo-EM map of NuA4 shows the structure of the complex, free of specimen preservation artifacts (Fig. 1f). The angular distribution of images in the cryo-EM dataset indicates that an adequate number of NuA4 orientations were recorded and analyzed (Supplementary Fig. 1a), and we estimate the resolution of the NuA4 cryo-EM structure to be ∼20 Å by the 0.5 Fourier Shell Correlation criterion31 (Supplementary Fig. 1b). Class averages obtained by reference-free alignment within groups of cryo-images segregated by orientation show good agreement with projections of the cryo-EM NuA4 volume (Supplementary Fig. 2).

Structural study of the NuA4 complex by single-particle EM

We purified NuA4 from yeast extracts as reported4, through a tandem affinity purification (TAP) tag28 engineered into the C-terminus of its Epl1 subunit (Fig. 1b). Initial EM characterization was performed on NuA4 particles adsorbed onto a continuous amorphous carbon layer and preserved in stain. An average obtained after alignment of stained particle images shows two large NuA4 domains joined by thin connections (Fig. 1c). By applying the random conical tilt (RCT) method29 to tilt-pair images of stained particles, we were able to calculate an initial 3D structure of NuA4 (Fig. 1d). This RCT volume was used as an initial reference for iterative refinement of a three-dimensional structure of NuA4 calculated from ∼35,000 images of NuA4 particles preserved in amorphous ice30 (Fig. 1e).

The cryo-EM map of NuA4 shows the structure of the complex, free of specimen preservation artifacts (Fig. 1f). The angular distribution of images in the cryo-EM dataset indicates that an adequate number of NuA4 orientations were recorded and analyzed (Supplementary Fig. 1a), and we estimate the resolution of the NuA4 cryo-EM structure to be ∼20 Å by the 0.5 Fourier Shell Correlation criterion31 (Supplementary Fig. 1b). Class averages obtained by reference-free alignment within groups of cryo-images segregated by orientation show good agreement with projections of the cryo-EM NuA4 volume (Supplementary Fig. 2).

NuA4 and SAGA structures and localization of Tra1Subunit Tra1 accounts for nearly half (∼43%) of the total mass of NuA4. Biochemical studies10 have detected only limited contacts between Tra1 and other NuA4 subunits (Fig. 1a), which suggested that Tra1 might constitute a sizable portion of one of the large NuA4 domains. Direct localization of Tra1 by deletion mapping is complicated because the subunit is essential in all of its length32. However, the location of Tra1 within the yeast SAGA HAT complex has been determined by EM analysis,33 and we decided to compare the NuA4 and SAGA structures to elucidate the distribution of Tra1 density within NuA4. We purified SAGA by TAP tagging and recorded EM images of SAGA particles preserved in stain. A 2D map of SAGA (Fig. 2a, left) shows that the top portion of the SAGA structure is formed by two well ordered domains that show a striking resemblance in shape and density distribution to the entirety of the NuA4 structure (Fig. 2a, right).There is no evident sequence homology between NuA4 and SAGA subunits other than Tra1, making a correspondence between the NuA4 and SAGA structures that extends beyond Tra1 an interesting and unexpected finding. Fortunately, besides determining the location of the Tra1 N-terminus by immunolabeling, the previous EM study of SAGA also established the overall boundaries for Tra1-related density by analyzing the structure of a SAGA subunit deletion mutant in which Tra1 was absent33. Tra1 occupies the upper left domain of the SAGA projection map (circled in Fig. 2a), which is nearly identical to the left half of the NuA4 projection map. Therefore, we conclude that most of this half of NuA4, is formed by Tra1. Consequently, the other half of the NuA4 map must contain the majority of the remaining NuA4 subunits, including the catalytic Piccolo subcomplex.

NuA4 and SAGA structures and localization of Tra1

Subunit Tra1 accounts for nearly half (∼43%) of the total mass of NuA4. Biochemical studies10 have detected only limited contacts between Tra1 and other NuA4 subunits (Fig. 1a), which suggested that Tra1 might constitute a sizable portion of one of the large NuA4 domains. Direct localization of Tra1 by deletion mapping is complicated because the subunit is essential in all of its length32. However, the location of Tra1 within the yeast SAGA HAT complex has been determined by EM analysis,33 and we decided to compare the NuA4 and SAGA structures to elucidate the distribution of Tra1 density within NuA4. We purified SAGA by TAP tagging and recorded EM images of SAGA particles preserved in stain. A 2D map of SAGA (Fig. 2a, left) shows that the top portion of the SAGA structure is formed by two well ordered domains that show a striking resemblance in shape and density distribution to the entirety of the NuA4 structure (Fig. 2a, right).

There is no evident sequence homology between NuA4 and SAGA subunits other than Tra1, making a correspondence between the NuA4 and SAGA structures that extends beyond Tra1 an interesting and unexpected finding. Fortunately, besides determining the location of the Tra1 N-terminus by immunolabeling, the previous EM study of SAGA also established the overall boundaries for Tra1-related density by analyzing the structure of a SAGA subunit deletion mutant in which Tra1 was absent33. Tra1 occupies the upper left domain of the SAGA projection map (circled in Fig. 2a), which is nearly identical to the left half of the NuA4 projection map. Therefore, we conclude that most of this half of NuA4, is formed by Tra1. Consequently, the other half of the NuA4 map must contain the majority of the remaining NuA4 subunits, including the catalytic Piccolo subcomplex.

Localization of the Piccolo subcomplexWe employed several approaches to localize the Piccolo subcomplex. First, we purified NuA4 from a mutant yeast strain lacking the C-terminus (residues 486–833) of the Epl1 subunit. This Epl1 truncation separates Piccolo subunits from the remainder of NuA44, and this smaller (ΔPiccolo) NuA4 can be purified by Eaf1–TAP10 (Supplementary Fig. 3). Comparing 2D EM maps of ΔPiccolo and wild-type NuA4 shows that whereas the Tra1 domain retains its structural integrity, the other half of the NuA4 structure appears considerably reduced in the absence of Piccolo (Fig. 2b). Therefore, Tra1 and Piccolo are located in opposite domains of the NuA4 structure and, in agreement with results from biochemical analysis of subunit interactions in NuA4 (Fig. 1a), do not contact each other directly10.Localization of the Epl1 Piccolo subunit provided further information about the arrangement of Piccolo within NuA4. A calmodulin binding peptide (CBP) left on the C-terminus of the Piccolo subunit Epl1 after TAP tagging was targeted with an anti-CBP polyclonal antibody (Open Biosystems). A class average calculated from images of antibody-decorated NuA4 particles shows diffuse density near the middle of the lower side of the NuA4 structure (Fig. 2c), indicating that the Epl1 C-terminus is likely mobile and located in that general area. A second labeling experiment was carried out with a gold-conjugated calmodulin probe (Au–CAM)34,35 also directed toward the CBP tag on Epl1. Roughly 18% of NuA4 particles show a small, well-defined area of high density with a radius of ∼2 nm, matching the expected size of the gold cluster after incubation with Au–CAM and calcium (Fig. 2d). To establish the statistical relevance of the signal attributed to the gold probe, we calculated a difference map by subtracting a normalized 2D average of NuA4 imaged with gold in the absence of calcium (calcium is essential for the calmodulin–CBP binding interaction) from an average obtained from gold-labeled NuA4 particles. The brightest peak in this difference map corresponds in position to the gold density apparent in the average obtained from images of gold-labeled NuA4 and has intensity above 3 standard deviations (Fig. 2d and Supplementary Fig. 4). The results from gold cluster and antibody labeling of the Epl1 C-terminus agree, and the gold cluster labeling more precisely localizes the C-terminus of the Epl1 Piccolo subunit to the non-Tra1 (right) domain of the NuA4 structure, near the inter-domain boundary.

Localization of the Piccolo subcomplex

We employed several approaches to localize the Piccolo subcomplex. First, we purified NuA4 from a mutant yeast strain lacking the C-terminus (residues 486–833) of the Epl1 subunit. This Epl1 truncation separates Piccolo subunits from the remainder of NuA44, and this smaller (ΔPiccolo) NuA4 can be purified by Eaf1–TAP10 (Supplementary Fig. 3). Comparing 2D EM maps of ΔPiccolo and wild-type NuA4 shows that whereas the Tra1 domain retains its structural integrity, the other half of the NuA4 structure appears considerably reduced in the absence of Piccolo (Fig. 2b). Therefore, Tra1 and Piccolo are located in opposite domains of the NuA4 structure and, in agreement with results from biochemical analysis of subunit interactions in NuA4 (Fig. 1a), do not contact each other directly10.

Localization of the Epl1 Piccolo subunit provided further information about the arrangement of Piccolo within NuA4. A calmodulin binding peptide (CBP) left on the C-terminus of the Piccolo subunit Epl1 after TAP tagging was targeted with an anti-CBP polyclonal antibody (Open Biosystems). A class average calculated from images of antibody-decorated NuA4 particles shows diffuse density near the middle of the lower side of the NuA4 structure (Fig. 2c), indicating that the Epl1 C-terminus is likely mobile and located in that general area. A second labeling experiment was carried out with a gold-conjugated calmodulin probe (Au–CAM)34,35 also directed toward the CBP tag on Epl1. Roughly 18% of NuA4 particles show a small, well-defined area of high density with a radius of ∼2 nm, matching the expected size of the gold cluster after incubation with Au–CAM and calcium (Fig. 2d). To establish the statistical relevance of the signal attributed to the gold probe, we calculated a difference map by subtracting a normalized 2D average of NuA4 imaged with gold in the absence of calcium (calcium is essential for the calmodulin–CBP binding interaction) from an average obtained from gold-labeled NuA4 particles. The brightest peak in this difference map corresponds in position to the gold density apparent in the average obtained from images of gold-labeled NuA4 and has intensity above 3 standard deviations (Fig. 2d and Supplementary Fig. 4). The results from gold cluster and antibody labeling of the Epl1 C-terminus agree, and the gold cluster labeling more precisely localizes the C-terminus of the Epl1 Piccolo subunit to the non-Tra1 (right) domain of the NuA4 structure, near the inter-domain boundary.

Interaction of NuA4 with the nucleosome core particleOur next goal was to investigate the interaction between NuA4 and the NCP. NuA4 particles preserved in stain were imaged following incubation with a 9-fold molar excess of recombinant NCPs formed from full-length Xenopus recombinant histones and mouse mammary tumor virus long terminal repeat NucB DNA36,37. The NuA4 concentration in the EM samples (∼20 nM) was close to the estimated Kd for NuA4–NCP interaction and, accordingly, only ∼5% of NuA4 particles from this experiment were bound to NCPs. However, a class average calculated from images of NuA4–NCP complexes clearly shows NCP density adjacent to the position previously identified by labeling of Epl1 with anti-CBP antibodies and Au–CAM (Fig. 2e). The NCP density appears partially disordered (the diameter of the NCP density is somewhat smaller than the X-ray structure of the NCP) indicating a flexible interaction with NuA4. The approximate location of this position can be mapped to the cryo-EM 3D structure of NuA4 (Fig. 2f, black arrowhead).

Interaction of NuA4 with the nucleosome core particle

Our next goal was to investigate the interaction between NuA4 and the NCP. NuA4 particles preserved in stain were imaged following incubation with a 9-fold molar excess of recombinant NCPs formed from full-length Xenopus recombinant histones and mouse mammary tumor virus long terminal repeat NucB DNA36,37. The NuA4 concentration in the EM samples (∼20 nM) was close to the estimated Kd for NuA4–NCP interaction and, accordingly, only ∼5% of NuA4 particles from this experiment were bound to NCPs. However, a class average calculated from images of NuA4–NCP complexes clearly shows NCP density adjacent to the position previously identified by labeling of Epl1 with anti-CBP antibodies and Au–CAM (Fig. 2e). The NCP density appears partially disordered (the diameter of the NCP density is somewhat smaller than the X-ray structure of the NCP) indicating a flexible interaction with NuA4. The approximate location of this position can be mapped to the cryo-EM 3D structure of NuA4 (Fig. 2f, black arrowhead).

Characterization of the Piccolo–NCP interactionThe results from our NuA4 labeling experiment with the NCP indicated that, at least under the conditions of our EM experiment, NuA4 and the NCP interact with a 1:1 stoichiometry. To determine if this stoichiometry also applied to the Piccolo–NCP complex in solution, we undertook sedimentation equilibrium (analytical) ultracentrifugation (AUC) studies of Piccolo and the Piccolo–NCP complex. For each of the samples analyzed (Piccolo, NCP, and Piccolo–NCP complex), the AUC curves are very well described by a single species. These AUC results establish that Piccolo in solution contains single copies of Esa1, Epl1, and Yng2. Consistent with the EM observations about NuA4 interaction with the NCP, the AUC results show that Piccolo also binds the NCP with a 1:1 stoichiometry (Fig. 3a, Table 1).Analysis of EM images of Piccolo–NCP particles preserved in stain resulted in a class average in which densities corresponding to the NCP and to Piccolo could be readily identified (Fig. 3b). Features in the Piccolo–NCP projection map appeared blurred, suggesting some conformational variability in the Piccolo–NCP particles. Focusing alignment on the NCP or Piccolo portions of Piccolo–NCP images resulted in better-defined maps for interacting Piccolo and NCP. When Piccolo–NCP image alignment is focused on Piccolo, the projection structure of Piccolo bound to the NCP shows a small particle with well-defined features lying adjacent to blurred NCP density (Fig. 3c). To verify this result, we imaged Piccolo in the absence of the NCP. The resulting Piccolo projection map (Fig. 3d) shows strong resemblance to the Piccolo average obtained from the Piccolo–NCP images. We also imaged an additional Piccolo variant (Piccolo v03) that differed from Piccolo v07 and v55 by inclusion of two additional portions of Epl1 (residues 1–49 and 381–485) and the Yng2 PHD domain (residues 221–282) (see Supplementary Figs. 5 through 7 for additional details about the composition and structural analysis of Piccolo deletion variants). The Piccolo v03 projection structure (Fig. 3e) suggests that additional regions of Epl1 and Yng2 contribute density (highlighted by the yellow arrowhead) that is proximal and possibly connected to the portion of the Piccolo structure involved in direct contact with the NCP.When Piccolo–NCP image alignment is focused on NCP density, the resulting Piccolo–NCP projection map shows a well-defined nucleosome in contact with blurred Piccolo density. Interestingly, in this map the NCP appears somewhat flat along the side contacted by Piccolo (Supplementary Fig. 8a, flat portion flanked by yellow arrowheads). A 2D map of the NCP alone calculated from images recorded from the same EM samples used to generate the Piccolo–NCP 2D map is remarkably similar to the NCP portion of the Piccolo–NCP 2D map and clearly shows the same flat feature along the top face (Supplementary Fig. 8b). Comparison with a projection of the nucleosome X-ray structure in approximately the same orientation (Supplementary Fig. 8c) underscores that the flat side of the nucleosome is defined by the entry and exit points of nucleosomal DNA across from the dyad. Because the NCP portion of the Piccolo–NCP 2D map appears circular, except around the area where the NCP is contacted by Piccolo, this 2D analysis establishes the approximate rotational orientation of the NCP in the Piccolo–NCP 2D map, indicating that Piccolo contacts the NCP at a position roughly across (± ∼20°) from the dyad.The recombinant v07 Piccolo construct used in the EM and AUC studies was shown to be fully active on chromatin substrates3,4 and comprised full-length, 6 × His-tagged Esa1 (residues 2–445) but truncated forms of Epl1 and Yng2. The Epl1 truncation comprised residues 51–380 (conserved EPcA domain) and lacked 453 C-terminal residues; the Yng2 truncation mutant comprised residues 2–218, which did not include the Yng2 PHD domain (as with Piccolo v55 in Supplementary Fig. 5). Therefore, a stable 1:1 stoichiometric Piccolo–NCP complex is formed even in the absence of both the Epl1 C-terminus and the Yng2 PHD domain.A 3D map of the Piccolo–NCP complex determined using the RCT method (Fig. 3f) also helped us understand the relative spatial arrangement of Piccolo and the NCP. The NCP X-ray structure38, or an RCT reconstruction of the NCP determined from images of free nucleosomes present in the Piccolo–NCP EM samples (not shown), could be directly docked into the RCT 3D structure of the Piccolo–NCP complex. As was the case with the 2D analysis, the NCP orientation indicated by the 3D docking suggests that Piccolo interacts with a region of the NCP approximately opposite the dyad (Fig. 3g and Supplementary Fig. 8d). This orientation of the NCP in the Piccolo–NCP complex would place an extended Piccolo density near the histone-fold domain of histone H4 (H4HFD), which has been previously identified as the binding site for the Esa1 Piccolo subunit39.To reconcile the information about the Piccolo–NCP interaction with our results about the location of Piccolo within the NuA4 structure and the NuA4–NCP interaction, we sought to dock the Piccolo–NCP RCT map into the larger NuA4 3D structure. Because the resolution of the Piccolo–NCP RCT map is limited as a result of the flexible interaction between Piccolo and the NCP, which in turns limits the accuracy of orientation parameters used to calculate the RCT Piccolo–NCP map, we calculated an RCT reconstruction of Piccolo alone. We docked this structure into the cryo-EM NuA4 map (Supplementary Fig. 9a) and found that Piccolo could be docked in a unique position and orientation within the right half of the NuA4 structure. Docking of the lower resolution Piccolo–NCP map was then guided by the position of Piccolo (Supplementary Fig. 9b). The position of the Piccolo–NCP map determined through this stepwise procedure is consistent with the observed interaction between NuA4 and the NCP (Fig. 2e and Supplementary Fig. 9c). Finally, since our results suggest that Piccolo might have extended contacts with histones and DNA, we pursued a series of biochemical experiments designed to identify elements necessary for binding of NuA4 and Piccolo to the NCP.

Characterization of the Piccolo–NCP interaction

The results from our NuA4 labeling experiment with the NCP indicated that, at least under the conditions of our EM experiment, NuA4 and the NCP interact with a 1:1 stoichiometry. To determine if this stoichiometry also applied to the Piccolo–NCP complex in solution, we undertook sedimentation equilibrium (analytical) ultracentrifugation (AUC) studies of Piccolo and the Piccolo–NCP complex. For each of the samples analyzed (Piccolo, NCP, and Piccolo–NCP complex), the AUC curves are very well described by a single species. These AUC results establish that Piccolo in solution contains single copies of Esa1, Epl1, and Yng2. Consistent with the EM observations about NuA4 interaction with the NCP, the AUC results show that Piccolo also binds the NCP with a 1:1 stoichiometry (Fig. 3a, Table 1).

Analysis of EM images of Piccolo–NCP particles preserved in stain resulted in a class average in which densities corresponding to the NCP and to Piccolo could be readily identified (Fig. 3b). Features in the Piccolo–NCP projection map appeared blurred, suggesting some conformational variability in the Piccolo–NCP particles. Focusing alignment on the NCP or Piccolo portions of Piccolo–NCP images resulted in better-defined maps for interacting Piccolo and NCP. When Piccolo–NCP image alignment is focused on Piccolo, the projection structure of Piccolo bound to the NCP shows a small particle with well-defined features lying adjacent to blurred NCP density (Fig. 3c). To verify this result, we imaged Piccolo in the absence of the NCP. The resulting Piccolo projection map (Fig. 3d) shows strong resemblance to the Piccolo average obtained from the Piccolo–NCP images. We also imaged an additional Piccolo variant (Piccolo v03) that differed from Piccolo v07 and v55 by inclusion of two additional portions of Epl1 (residues 1–49 and 381–485) and the Yng2 PHD domain (residues 221–282) (see Supplementary Figs. 5 through 7 for additional details about the composition and structural analysis of Piccolo deletion variants). The Piccolo v03 projection structure (Fig. 3e) suggests that additional regions of Epl1 and Yng2 contribute density (highlighted by the yellow arrowhead) that is proximal and possibly connected to the portion of the Piccolo structure involved in direct contact with the NCP.

When Piccolo–NCP image alignment is focused on NCP density, the resulting Piccolo–NCP projection map shows a well-defined nucleosome in contact with blurred Piccolo density. Interestingly, in this map the NCP appears somewhat flat along the side contacted by Piccolo (Supplementary Fig. 8a, flat portion flanked by yellow arrowheads). A 2D map of the NCP alone calculated from images recorded from the same EM samples used to generate the Piccolo–NCP 2D map is remarkably similar to the NCP portion of the Piccolo–NCP 2D map and clearly shows the same flat feature along the top face (Supplementary Fig. 8b). Comparison with a projection of the nucleosome X-ray structure in approximately the same orientation (Supplementary Fig. 8c) underscores that the flat side of the nucleosome is defined by the entry and exit points of nucleosomal DNA across from the dyad. Because the NCP portion of the Piccolo–NCP 2D map appears circular, except around the area where the NCP is contacted by Piccolo, this 2D analysis establishes the approximate rotational orientation of the NCP in the Piccolo–NCP 2D map, indicating that Piccolo contacts the NCP at a position roughly across (± ∼20°) from the dyad.

The recombinant v07 Piccolo construct used in the EM and AUC studies was shown to be fully active on chromatin substrates3,4 and comprised full-length, 6 × His-tagged Esa1 (residues 2–445) but truncated forms of Epl1 and Yng2. The Epl1 truncation comprised residues 51–380 (conserved EPcA domain) and lacked 453 C-terminal residues; the Yng2 truncation mutant comprised residues 2–218, which did not include the Yng2 PHD domain (as with Piccolo v55 in Supplementary Fig. 5). Therefore, a stable 1:1 stoichiometric Piccolo–NCP complex is formed even in the absence of both the Epl1 C-terminus and the Yng2 PHD domain.

A 3D map of the Piccolo–NCP complex determined using the RCT method (Fig. 3f) also helped us understand the relative spatial arrangement of Piccolo and the NCP. The NCP X-ray structure38, or an RCT reconstruction of the NCP determined from images of free nucleosomes present in the Piccolo–NCP EM samples (not shown), could be directly docked into the RCT 3D structure of the Piccolo–NCP complex. As was the case with the 2D analysis, the NCP orientation indicated by the 3D docking suggests that Piccolo interacts with a region of the NCP approximately opposite the dyad (Fig. 3g and Supplementary Fig. 8d). This orientation of the NCP in the Piccolo–NCP complex would place an extended Piccolo density near the histone-fold domain of histone H4 (H4HFD), which has been previously identified as the binding site for the Esa1 Piccolo subunit39.

To reconcile the information about the Piccolo–NCP interaction with our results about the location of Piccolo within the NuA4 structure and the NuA4–NCP interaction, we sought to dock the Piccolo–NCP RCT map into the larger NuA4 3D structure. Because the resolution of the Piccolo–NCP RCT map is limited as a result of the flexible interaction between Piccolo and the NCP, which in turns limits the accuracy of orientation parameters used to calculate the RCT Piccolo–NCP map, we calculated an RCT reconstruction of Piccolo alone. We docked this structure into the cryo-EM NuA4 map (Supplementary Fig. 9a) and found that Piccolo could be docked in a unique position and orientation within the right half of the NuA4 structure. Docking of the lower resolution Piccolo–NCP map was then guided by the position of Piccolo (Supplementary Fig. 9b). The position of the Piccolo–NCP map determined through this stepwise procedure is consistent with the observed interaction between NuA4 and the NCP (Fig. 2e and Supplementary Fig. 9c). Finally, since our results suggest that Piccolo might have extended contacts with histones and DNA, we pursued a series of biochemical experiments designed to identify elements necessary for binding of NuA4 and Piccolo to the NCP.

Nucleosome binding and chromatin acetylation by NuA4We pursued several experiments to investigate in more detail how NuA4 and Piccolo bind the NCP and which subunits make the greatest contributions to binding.First, we determined that the chromodomain subunit Eaf3 does not play a primary role in NuA4 binding to chromatin in vivo. When NuA4 is affinity purified from cell extracts in low-salt conditions, a sizeable amount of all four core histones is detected (Fig. 4a). However, when this is repeated in a mutant strain (eaf1) that disconnects Eaf3 from Epl1 (Fig. 1a;10), histones no longer co-purify with the Eaf3-containing module (Fig. 4a, lane 2).Second, we investigated Epl1 involvement in NCP binding by performing mobility shift assays with constructs comprising different portions of Epl1 and NCPs. We have previously shown that a recombinant Esa1–Epl1 heterodimer has much stronger acetylation activity on free histones and chromatin when compared to Esa1 alone, whereas inclusion of Yng2 in Piccolo drives acetylation only toward chromatin4. Mobility shift assay results now indicate that the Epl1 N-terminal region, encompassing the conserved EPcA domain4,40, has a very strong affinity for the NCP and that in its presence, Epl1 forms a stable complex with mononucleosomes (Fig. 4b).Third, we investigated the effect of Piccolo subunit interactions on the HAT activity of Esa1. We started by obtaining evidence for simultaneous association of Esa1 and Yng2 to the Epl1 N-terminal region by mixing the three recombinant proteins together followed by GST pull down analysis. We could detect efficient binding of both Esa1 and Yng2 (Fig. 4c), since the Epl1 N-terminal region independently binds Esa1 and Yng2, physically linking them together in Piccolo NuA43,4. We then analyzed how interaction with Epl1 and Yng2 modulates the HAT activity of Esa1 toward free histones and a chromatin substrate (Fig. 4d, keeping the amount of Esa1 protein constant). As previously shown, recombinant Esa1 (rEsa1) can only acetylate free histones1. The addition of Yng2 does not affect Esa1 activity, as expected, since no direct interaction between the two proteins was detected4. Addition of Epl1 to Esa1 (rEsa1 + rEpl1) had a striking positive effect on its activity toward free histones but did not enable activity on chromatin. This indicates that Epl1 association with Esa1 increases its specific enzymatic activity, but that the strong affinity of Epl1 for the NCP is non-productive in positioning Esa1 to acetylate H4 or H2A tails. Finally, a mixture of all three proteins (rEsa1 + rEpl1 + rYng2) increased Esa1 specific activity toward free histones to about the same level as Epl1 alone, but acetylation of chromatin was also detected in the presence of Yng2. These data suggest that while Epl1–Esa1 dimers bind efficiently to nucleosomes, Yng2 is required to position Piccolo in a specific orientation that places H4 or H2A histone tails in a productive location near the Esa1 catalytic site.Fourth, we used NCP mobility shift assays to investigate interaction with individual Piccolo subunits (Fig. 4e). The results establish that while Esa1 shows no interaction and tight binding is again observed for the Epl1 N-terminus, Yng2 does seem to directly bind the nucleosome to some detectable level. Furthermore, this interaction does not seem to involve the Yng2 PHD domain (Fig. 4e, lanes 9–11).Fifth, we used additional NCP mobility shift assays to determine the critical component for interaction between the NCP and the Piccolo complex. The assays were performed on two Piccolo constructs identical in their Yng2 (Yng2ΔPHD 1–220) and Esa1 (Esa1 1–445) polypeptides, but differing in their Epl1 component (Fig. 5a). NCP binding was detected for a construct including Epl1 residues 51–380 (the minimal Epl1 domain required for cell viability4), while no NCP binding interaction was observed for a construct including only Epl1 residues 72–380 (Fig. 5b). These results show that Epl1 residues 51–71 are essential for the Piccolo–NCP binding interaction. In contrast, the Yng2 PHD domain is dispensable for binding of mononucleosomes by a recombinant Piccolo complex (Fig. 5c). The observation from this mobility shift assay that the Yng2 PHD domain has no striking effect on Piccolo's ability to bind nucleosomes is in agreement with the negligible role played by this domain in NuA4 and Piccolo HAT activity toward chromatin3,4,41. These observations, along with results from previous and parallel studies3,4, provide an overview of Epl1 binding interactions (Fig. 5d).To summarize, in remarkable agreement with our EM observations, these biochemical results indicate that the N-terminal portion of the Epl1 Piccolo subunit plays a primary role in enabling binding of NuA4 to chromatin, with additional Piccolo subunits being important for modulating the Epl1-based interaction and Esa1 HAT activity.

Nucleosome binding and chromatin acetylation by NuA4

We pursued several experiments to investigate in more detail how NuA4 and Piccolo bind the NCP and which subunits make the greatest contributions to binding.

First, we determined that the chromodomain subunit Eaf3 does not play a primary role in NuA4 binding to chromatin in vivo. When NuA4 is affinity purified from cell extracts in low-salt conditions, a sizeable amount of all four core histones is detected (Fig. 4a). However, when this is repeated in a mutant strain (eaf1) that disconnects Eaf3 from Epl1 (Fig. 1a;10), histones no longer co-purify with the Eaf3-containing module (Fig. 4a, lane 2).

Second, we investigated Epl1 involvement in NCP binding by performing mobility shift assays with constructs comprising different portions of Epl1 and NCPs. We have previously shown that a recombinant Esa1–Epl1 heterodimer has much stronger acetylation activity on free histones and chromatin when compared to Esa1 alone, whereas inclusion of Yng2 in Piccolo drives acetylation only toward chromatin4. Mobility shift assay results now indicate that the Epl1 N-terminal region, encompassing the conserved EPcA domain4,40, has a very strong affinity for the NCP and that in its presence, Epl1 forms a stable complex with mononucleosomes (Fig. 4b).

Third, we investigated the effect of Piccolo subunit interactions on the HAT activity of Esa1. We started by obtaining evidence for simultaneous association of Esa1 and Yng2 to the Epl1 N-terminal region by mixing the three recombinant proteins together followed by GST pull down analysis. We could detect efficient binding of both Esa1 and Yng2 (Fig. 4c), since the Epl1 N-terminal region independently binds Esa1 and Yng2, physically linking them together in Piccolo NuA43,4. We then analyzed how interaction with Epl1 and Yng2 modulates the HAT activity of Esa1 toward free histones and a chromatin substrate (Fig. 4d, keeping the amount of Esa1 protein constant). As previously shown, recombinant Esa1 (rEsa1) can only acetylate free histones1. The addition of Yng2 does not affect Esa1 activity, as expected, since no direct interaction between the two proteins was detected4. Addition of Epl1 to Esa1 (rEsa1 + rEpl1) had a striking positive effect on its activity toward free histones but did not enable activity on chromatin. This indicates that Epl1 association with Esa1 increases its specific enzymatic activity, but that the strong affinity of Epl1 for the NCP is non-productive in positioning Esa1 to acetylate H4 or H2A tails. Finally, a mixture of all three proteins (rEsa1 + rEpl1 + rYng2) increased Esa1 specific activity toward free histones to about the same level as Epl1 alone, but acetylation of chromatin was also detected in the presence of Yng2. These data suggest that while Epl1–Esa1 dimers bind efficiently to nucleosomes, Yng2 is required to position Piccolo in a specific orientation that places H4 or H2A histone tails in a productive location near the Esa1 catalytic site.

Fourth, we used NCP mobility shift assays to investigate interaction with individual Piccolo subunits (Fig. 4e). The results establish that while Esa1 shows no interaction and tight binding is again observed for the Epl1 N-terminus, Yng2 does seem to directly bind the nucleosome to some detectable level. Furthermore, this interaction does not seem to involve the Yng2 PHD domain (Fig. 4e, lanes 9–11).

Fifth, we used additional NCP mobility shift assays to determine the critical component for interaction between the NCP and the Piccolo complex. The assays were performed on two Piccolo constructs identical in their Yng2 (Yng2ΔPHD 1–220) and Esa1 (Esa1 1–445) polypeptides, but differing in their Epl1 component (Fig. 5a). NCP binding was detected for a construct including Epl1 residues 51–380 (the minimal Epl1 domain required for cell viability4), while no NCP binding interaction was observed for a construct including only Epl1 residues 72–380 (Fig. 5b). These results show that Epl1 residues 51–71 are essential for the Piccolo–NCP binding interaction. In contrast, the Yng2 PHD domain is dispensable for binding of mononucleosomes by a recombinant Piccolo complex (Fig. 5c). The observation from this mobility shift assay that the Yng2 PHD domain has no striking effect on Piccolo's ability to bind nucleosomes is in agreement with the negligible role played by this domain in NuA4 and Piccolo HAT activity toward chromatin3,4,41. These observations, along with results from previous and parallel studies3,4, provide an overview of Epl1 binding interactions (Fig. 5d).

To summarize, in remarkable agreement with our EM observations, these biochemical results indicate that the N-terminal portion of the Epl1 Piccolo subunit plays a primary role in enabling binding of NuA4 to chromatin, with additional Piccolo subunits being important for modulating the Epl1-based interaction and Esa1 HAT activity.

DiscussionThe structure of the NuA4 HAT complex comprises two large globular domains joined by multiple connections (Fig 1). One of these domains is mostly formed by Tra1, the largest NuA4 subunit (433 kDa MW, accounting for ∼43% of the mass and ∼48% of the volume of the 1 MDa complex) (Fig. 2). Presumably, most of the remaining subunits, including the Piccolo catalytic subcomplex, are localized in the second NuA4 domain. This is confirmed by the results from antibody and gold-cluster labeling of the Epl1 Piccolo subunit (Fig. 2), and it is further supported by a diminished size and large-scale changes in the structure of the non-Tra1 domain in images of the ΔPiccolo NuA4 mutant, which comprises all NuA4 subunits except those found in the Piccolo subcomplex. Incubation of NuA4 with NCPs results in NCP density adjacent to the non-Tra1 NuA4 domain, again indicating that Piccolo is located in that portion of NuA4 (Fig. 2). The structure of the non-Tra1 NuA4 domain indicates that Piccolo and two other NuA4 subunit complexes identified by biochemical analyses, Eaf5–Eaf7–Eaf3 and Eaf2–Yaf9–Act1–Arp4 (Fig. 1), are in close physical proximity10. It has been reported that deletion of the Eaf1 scaffold protein10 results in a partial NuA4 complex that lacks only Tra1 and Eaf115, suggesting that Eaf1 must form most the interface between the two globular NuA4 domains. Despite biochemical evidence that Eaf1 provides most of the contacts that keep together the Piccolo, Eaf3–Eaf5–Eaf7, and Eaf2–Yaf9–Act1–Arp4 subcomplexes10,14, it seems plausible from their relatively compact arrangement in the NuA4 structure that additional contacts among these three subcomplexes could account for their observed association in cell extracts in the absence of Eaf115, which is not detected in affinity-purified complexes10,14.Our results also provide structural insight into the interaction of a HAT complex with the NCP. The absence of any large cavities in the NuA4 structure and the direct observation of NCP binding to NuA4 (Fig. 2) indicate that interaction of NuA4 with the nucleosome occurs at the periphery of NuA4. This is in contrast with what we observed for the RSC chromatin-remodeling complex, which binds the nucleosome in an internal cavity where extensive interactions take place42. NuA4 binds the nucleosome in a non-enveloping fashion through limited, localized contacts. The nimble interaction between NuA4 and the NCP might explain how NuA4 could operate in a condensed chromatin environment (e.g. the 30 nm fiber). Relaxation of heterochromatin packing following histone tail acetylation by NuA4 may alleviate steric inhibition and provide a platform for bromodomain interaction, allowing complexes like RSC to bind the nucleosome and carry out physical remodeling of chromatin. Future biochemical studies of RSC activity on condensed and relaxed chromatin templates, and in the absence and presence of histone acetylation, could provide evidence to test this hypothesis.We used a number of binding assays to further investigate the interaction between NuA4 (or Piccolo) and the NCP. Mobility shift assays showed that, on its own, Esa1 made no or only negligible stable contacts with the NCP, whereas Yng2 and Epl1 made observable binding contacts (Fig. 4b, e). The interaction between Epl1 and the NCP was the strongest, and deletion analysis showed that the conserved N-terminus of Epl1 was mainly responsible for interaction with the NCP. Further deletion analysis showed that Epl1 residues 51–71 are required for the binding interaction between Piccolo and the NCP (Fig. 5b). We also obtained evidence that NuA4 association with chromatin in vivo is primarily dependent on the presence of the Piccolo subcomplex (Fig. 4a), further supporting the role of Epl1 as the primary chromatin-binding subunit of NuA4. In our model, the chromatin-specific role of Yng2 in stimulating acetylation would be explained by correctly orienting the binding of Epl1–Esa1 to the nucleosome, leading to productive co-localization of the Esa1 catalytic site and H4 or H2A histone tails.These binding assay results are consistent with our structural observations. EM analysis of a recombinant Piccolo–NCP complex allowed us to determine its 3D structure, which we partially interpreted by docking of an atomic resolution model of the NCP. Piccolo has been reported to bind the H4HFD through the Esa1 HAT subunit39. However, in order to account for the increased catalytic efficiency of Piccolo over Esa1 alone (an increase of ∼2–3 orders of magnitude) the authors of that study propose that the Piccolo subunits Epl1 and Yng2 might also contact the nucleosome. Our EM structure of the Piccolo–NCP complex shows a large Piccolo–NCP interface comprising multiple contacts, primarily on one face of the histone octamer, that includes the H4HFD but also extends to possibly include other histones and DNA. This observation is consistent with the hypothesis that Yng2 and/or Epl1 make additional contacts with the nucleosome. Additionally, we note that the Piccolo–NCP interaction we report occurs in the absence of the Yng2 PHD domain, providing structural evidence supporting the hypothesis that the binding interaction between Piccolo and the NCP does not require this domain.Further insight into Piccolo subunit organization derived from EM analysis of a number of Piccolo variants is also consistent with the description of Piccolo–NCP contacts we propose. Differences between the structures in projection of different Piccolo deletion variants (Supplementary Figs. 5 through 7) suggest that at least some portions of Esa1 are localized in the central region of the Piccolo structure, distal to the area of NCP interaction. On the other hand, portions of Epl1 (and perhaps also the Yng2 PHD domain) are in close proximity to the nucleosome-binding region of Piccolo. These structural observations are based on comparison of 2D structures calculated from images of particles preserved in stain (a low resolution technique), and comparison between the projection structures of different Piccolo variants highlights regions where density disappears upon subunit deletion, providing only indirect evidence for localization of specific subunits. Nonetheless, taken together with our other structural and biochemical observations, these results are consistent with a model for Piccolo–NCP interaction where Epl1 plays a primary role in binding the NCP, Yng2 stabilizes this interaction and positions the NCP for optimal catalysis, and Esa1 plays a lesser (though catalytically relevant) role in binding and is primarily involved in catalysis.Finally, one of the most intriguing observations from this study is an uncanny resemblance between the structures of NuA4 and SAGA that extends well beyond what could be explained by the presence of the common Tra1 subunit. The SAGA subunits (Taf5, Taf6, Taf9, Taf10, and Taf12) that presumably comprise part of the SAGA domain that corresponds structurally to the non-Tra1 portion of NuA433 share no sequence homology with NuA4 subunits. A search for subunit interaction partners shared by NuA4 and SAGA components failed to identify any. Still, NuA4 and SAGA acetylate nucleosomes and often act in concert in the same environment inside the nucleus19. Furthermore, it has been recently shown that NuA4 and SAGA are both recruited to the phosphorylated C-terminal domain of elongating RNA polymerase II15. Structural similarities between the two complexes that extend beyond Tra1 may represent convergent evolution of structure highlighting common aspects of two different solutions to a shared set of general molecular functions. Alternatively, it is possible that NuA4 and SAGA may, at some point during their shared roles in DNA transactions, bind to a common interface (perhaps a particular chromatin architecture or another large protein complex) that dictates their structural similarity.

The structure of the NuA4 HAT complex comprises two large globular domains joined by multiple connections (Fig 1). One of these domains is mostly formed by Tra1, the largest NuA4 subunit (433 kDa MW, accounting for ∼43% of the mass and ∼48% of the volume of the 1 MDa complex) (Fig. 2). Presumably, most of the remaining subunits, including the Piccolo catalytic subcomplex, are localized in the second NuA4 domain. This is confirmed by the results from antibody and gold-cluster labeling of the Epl1 Piccolo subunit (Fig. 2), and it is further supported by a diminished size and large-scale changes in the structure of the non-Tra1 domain in images of the ΔPiccolo NuA4 mutant, which comprises all NuA4 subunits except those found in the Piccolo subcomplex. Incubation of NuA4 with NCPs results in NCP density adjacent to the non-Tra1 NuA4 domain, again indicating that Piccolo is located in that portion of NuA4 (Fig. 2). The structure of the non-Tra1 NuA4 domain indicates that Piccolo and two other NuA4 subunit complexes identified by biochemical analyses, Eaf5–Eaf7–Eaf3 and Eaf2–Yaf9–Act1–Arp4 (Fig. 1), are in close physical proximity10. It has been reported that deletion of the Eaf1 scaffold protein10 results in a partial NuA4 complex that lacks only Tra1 and Eaf115, suggesting that Eaf1 must form most the interface between the two globular NuA4 domains. Despite biochemical evidence that Eaf1 provides most of the contacts that keep together the Piccolo, Eaf3–Eaf5–Eaf7, and Eaf2–Yaf9–Act1–Arp4 subcomplexes10,14, it seems plausible from their relatively compact arrangement in the NuA4 structure that additional contacts among these three subcomplexes could account for their observed association in cell extracts in the absence of Eaf115, which is not detected in affinity-purified complexes10,14.

Our results also provide structural insight into the interaction of a HAT complex with the NCP. The absence of any large cavities in the NuA4 structure and the direct observation of NCP binding to NuA4 (Fig. 2) indicate that interaction of NuA4 with the nucleosome occurs at the periphery of NuA4. This is in contrast with what we observed for the RSC chromatin-remodeling complex, which binds the nucleosome in an internal cavity where extensive interactions take place42. NuA4 binds the nucleosome in a non-enveloping fashion through limited, localized contacts. The nimble interaction between NuA4 and the NCP might explain how NuA4 could operate in a condensed chromatin environment (e.g. the 30 nm fiber). Relaxation of heterochromatin packing following histone tail acetylation by NuA4 may alleviate steric inhibition and provide a platform for bromodomain interaction, allowing complexes like RSC to bind the nucleosome and carry out physical remodeling of chromatin. Future biochemical studies of RSC activity on condensed and relaxed chromatin templates, and in the absence and presence of histone acetylation, could provide evidence to test this hypothesis.

We used a number of binding assays to further investigate the interaction between NuA4 (or Piccolo) and the NCP. Mobility shift assays showed that, on its own, Esa1 made no or only negligible stable contacts with the NCP, whereas Yng2 and Epl1 made observable binding contacts (Fig. 4b, e). The interaction between Epl1 and the NCP was the strongest, and deletion analysis showed that the conserved N-terminus of Epl1 was mainly responsible for interaction with the NCP. Further deletion analysis showed that Epl1 residues 51–71 are required for the binding interaction between Piccolo and the NCP (Fig. 5b). We also obtained evidence that NuA4 association with chromatin in vivo is primarily dependent on the presence of the Piccolo subcomplex (Fig. 4a), further supporting the role of Epl1 as the primary chromatin-binding subunit of NuA4. In our model, the chromatin-specific role of Yng2 in stimulating acetylation would be explained by correctly orienting the binding of Epl1–Esa1 to the nucleosome, leading to productive co-localization of the Esa1 catalytic site and H4 or H2A histone tails.

These binding assay results are consistent with our structural observations. EM analysis of a recombinant Piccolo–NCP complex allowed us to determine its 3D structure, which we partially interpreted by docking of an atomic resolution model of the NCP. Piccolo has been reported to bind the H4HFD through the Esa1 HAT subunit39. However, in order to account for the increased catalytic efficiency of Piccolo over Esa1 alone (an increase of ∼2–3 orders of magnitude) the authors of that study propose that the Piccolo subunits Epl1 and Yng2 might also contact the nucleosome. Our EM structure of the Piccolo–NCP complex shows a large Piccolo–NCP interface comprising multiple contacts, primarily on one face of the histone octamer, that includes the H4HFD but also extends to possibly include other histones and DNA. This observation is consistent with the hypothesis that Yng2 and/or Epl1 make additional contacts with the nucleosome. Additionally, we note that the Piccolo–NCP interaction we report occurs in the absence of the Yng2 PHD domain, providing structural evidence supporting the hypothesis that the binding interaction between Piccolo and the NCP does not require this domain.

Further insight into Piccolo subunit organization derived from EM analysis of a number of Piccolo variants is also consistent with the description of Piccolo–NCP contacts we propose. Differences between the structures in projection of different Piccolo deletion variants (Supplementary Figs. 5 through 7) suggest that at least some portions of Esa1 are localized in the central region of the Piccolo structure, distal to the area of NCP interaction. On the other hand, portions of Epl1 (and perhaps also the Yng2 PHD domain) are in close proximity to the nucleosome-binding region of Piccolo. These structural observations are based on comparison of 2D structures calculated from images of particles preserved in stain (a low resolution technique), and comparison between the projection structures of different Piccolo variants highlights regions where density disappears upon subunit deletion, providing only indirect evidence for localization of specific subunits. Nonetheless, taken together with our other structural and biochemical observations, these results are consistent with a model for Piccolo–NCP interaction where Epl1 plays a primary role in binding the NCP, Yng2 stabilizes this interaction and positions the NCP for optimal catalysis, and Esa1 plays a lesser (though catalytically relevant) role in binding and is primarily involved in catalysis.

Finally, one of the most intriguing observations from this study is an uncanny resemblance between the structures of NuA4 and SAGA that extends well beyond what could be explained by the presence of the common Tra1 subunit. The SAGA subunits (Taf5, Taf6, Taf9, Taf10, and Taf12) that presumably comprise part of the SAGA domain that corresponds structurally to the non-Tra1 portion of NuA433 share no sequence homology with NuA4 subunits. A search for subunit interaction partners shared by NuA4 and SAGA components failed to identify any. Still, NuA4 and SAGA acetylate nucleosomes and often act in concert in the same environment inside the nucleus19. Furthermore, it has been recently shown that NuA4 and SAGA are both recruited to the phosphorylated C-terminal domain of elongating RNA polymerase II15. Structural similarities between the two complexes that extend beyond Tra1 may represent convergent evolution of structure highlighting common aspects of two different solutions to a shared set of general molecular functions. Alternatively, it is possible that NuA4 and SAGA may, at some point during their shared roles in DNA transactions, bind to a common interface (perhaps a particular chromatin architecture or another large protein complex) that dictates their structural similarity.

MethodsPurification and electron microscopy of NuA4 and SAGANuA4 and SAGA were purified by the TAP method28 essentially as described43. TAP tags were inserted at the C-termini of the Epl1 and Spt7 subunits of NuA4 and SAGA, respectively (for NuA4 purified from an Epl1(1–485) truncation background (ΔPiccolo), the Eaf1 subunit was TAP tagged). Purified complexes were analyzed by SDS–PAGE, and western blot (data not shown, see examples in17). In some cases, samples were further purified by gel filtration on a Superose 6 column (GE Healthcare).EM samples of NuA4 and SAGA preserved in stain were prepared as described42. Samples were imaged under low-dose conditions on a field emission gun (FEG) Tecnai F-20 microscope (FEI) (acceleration voltage 120 kV). Images were recorded on a Gatan 4096 × 4096 CCD camera at 50,000 × magnification and ∼500 nm underfocus. Two-fold pixel binning resulted in a final pixel size of 4.34 Å. Images of individual NuA4 and SAGA particles (∼5,000 for each complex) were selected by hand. Image processing was done using the Spider and SPARX software packages44,45. Single-particle images were aligned without a reference and subjected to hierarchical ascendant clustering46 to produce class averages.Calculation of 3D NuA4 structuresAn initial NuA4 3D map was calculated by the random conical tilt (RCT) method29 as described42. Cryo-EM samples were also prepared as described42 and imaged under low-dose conditions using a FEG CM200 (Philips/FEI) microscope (acceleration voltage 120 kV). Images were recorded on Kodak SO-163 film at a magnification of 66,000 × and with 0.9–4.0 μm underfocus values. Micrographs were digitized on a Zeiss/SCAIA flat bed densitometer (ZI/Zeiss) using a step size of 7 μm. Digitized images were 4-fold pixel averaged to 4.11 Å per pixel. NuA4 particle images (∼35,000) were selected by hand, divided into groups according to defocus, and a cryo-EM map of NuA4 was calculated by projection matching refinement as described42 using the NuA4 RCT reconstruction as initial reference. Three-dimensional structure interpretation and image rendering were done using UCSF Chimera47.Epl1 labeling and NuA4 incubation with the NCPNuA4 at 20 μg ml−1 was incubated for 30 min at 25 °C with a five-fold molar excess of polyclonal anti-CBP antibody (Open Biosystems) targeting the calmodulin binding peptide (CBP) tag on the Epl1 subunit. EM samples were then prepared and imaged as described above. Individual particle images (∼23,000) were selected by hand, and ∼500 were categorized by visual inspection as images showing the NuA4–anti-TAP interaction. Gold-cluster labeling of Epl1 was done by incubating NuA4 at 20 μg ml−1 with a 20-fold molar excess of thiol-protected calmodulin-bioconjugated gold nanoparticles34,35 for 2 h at 4 °C. Incubation with the gold probe was carried out in the presence and in the absence of calcium, and about 4,000 images from each condition were recorded and analyzed. To form the NuA4–NCP complex, NuA4 at 50 μg ml−1 was incubated with a 9-fold molar excess of NCPs reconstituted from full-length recombinant histones from Xenopus and mouse mammary tumor virus long terminal repeat NucB DNA36,37. EM samples were prepared and imaged as before.EM characterization of the Piccolo and Piccolo–NCP complexesRecombinant Piccolo NuA4 complex was expressed in bacteria using a polycistronic vector and purified as described3,4. The Piccolo–NCP complex was assembled as described3. To promote complex stability, Piccolo–NCP (∼215 μg ml−1) was crosslinked for 10 min with 1% (v/v) formaldehyde and quenched with 320 mM glycine. The sample was diluted to a final Piccolo–NCP concentration of 20 μg ml−1 before preparing and imaging EM samples as described for stained NuA4 specimens. Images were recorded on film and digitized using a Coolscan 9000 scanner (Nikkon). Individual particle images were selected by hand or by automated particle selection with the TiltPicker and DoGPicker software tools48.Analytical ultracentrifugation experimentsSamples equilibrated in 20 mM Tris–Cl pH 8.0, 50 mM KCl, 5 mM 2-mercaptoethanol buffer were brought to sedimentation equilibrium in a Beckman XL-A analytical ultracentrifuge fitted with an AN-60Ti rotor, operating at 4.0 ± 0.1 °C. Five data sets were collected at 260 nm and five at 280 nm for each sample, at three rotor speeds (2,600, 5,900 and 14,200 × g) and three nominal concentrations (0.37, 0.19 and 0.09 mg ml−1 for Piccolo; 0.15, 0.07 and 0.04 μg ml−1 for the DNA-containing complexes). Absorbance versus radial position data were fit by one- and two-species models using partial specific volumes calculated from the amino acid and nucleic acid compositions of the complexes49. Global analysis of multiple data sets was performed with the program PRISM (Graphpad, Inc.).Piccolo NCP binding and acetyltransferase activityBacterial expression vectors for different His–Esa1, GST–Epl1 (1–485), GST–Epl1 (486–832), His–Yng2 (1–282) and His–Yng2 (1–218) have been used before4, and cloning details are available upon request. Piccolo NuA4 complexes used in the experiments shown in Figs. 3 and 5 were produced by coexpression in E. coli and purified as described previously3. Single GST- and His-recombinant proteins were expressed and purified on glutathione Sepharose or nickel–agarose beads following standard procedures. His-tagged recombinant proteins were precleared on GST–Sepharose for 1 h at 4 °C prior to pulldown. 300 ng of rEsa1 and 200 ng of rYng2 were incubated (3 h at 4 °C) with 300 ng of appropriate GST fusion (or 1 ng of GST control). Equivalent amounts of input, bound, and free fractions were analyzed by western blot using an anti-HIS antibody (BabCo). For HAT assays, individual soluble recombinant proteins were incubated together at a similar ratio for 15 min on ice and 20 min at 30 °C before the assay, which was done with short oligonucleosomes purified from HeLa cells as previously described1. Different mixtures of recombinant proteins were prepared with a constant amount of rEsa1. All values were standardized to the activity of rEsa1 alone on free histones. Data shown in Fig. 4d are representative of two different experiments done in duplicate. For nucleosome binding experiments, a labeled 5S RNA gene was reconstituted in mononucleosomes as described50. rEsa1, rEpl1, rYng2 or rPiccolo trimeric complex were incubated with 12.5 ng of total nucleosomes for 30 min at 30 °C. Binding reactions were resolved on a 4% (w/v) PAGE gel in 0.5 × TBE. For measurement of histone copurification with NuA4 from whole cell extracts, wild-type and EAF1-deleted Eaf5–TAP strains10 were fractionated over IgG Sepharose followed by TEV elution, keeping salt concentration at 150 mM. Co-purifying Piccolo subunits and histones were detected by western blot and silver staining after separation by gel electrophoresis.

Purification and electron microscopy of NuA4 and SAGANuA4 and SAGA were purified by the TAP method28 essentially as described43. TAP tags were inserted at the C-termini of the Epl1 and Spt7 subunits of NuA4 and SAGA, respectively (for NuA4 purified from an Epl1(1–485) truncation background (ΔPiccolo), the Eaf1 subunit was TAP tagged). Purified complexes were analyzed by SDS–PAGE, and western blot (data not shown, see examples in17). In some cases, samples were further purified by gel filtration on a Superose 6 column (GE Healthcare).EM samples of NuA4 and SAGA preserved in stain were prepared as described42. Samples were imaged under low-dose conditions on a field emission gun (FEG) Tecnai F-20 microscope (FEI) (acceleration voltage 120 kV). Images were recorded on a Gatan 4096 × 4096 CCD camera at 50,000 × magnification and ∼500 nm underfocus. Two-fold pixel binning resulted in a final pixel size of 4.34 Å. Images of individual NuA4 and SAGA particles (∼5,000 for each complex) were selected by hand. Image processing was done using the Spider and SPARX software packages44,45. Single-particle images were aligned without a reference and subjected to hierarchical ascendant clustering46 to produce class averages.

Purification and electron microscopy of NuA4 and SAGA

NuA4 and SAGA were purified by the TAP method28 essentially as described43. TAP tags were inserted at the C-termini of the Epl1 and Spt7 subunits of NuA4 and SAGA, respectively (for NuA4 purified from an Epl1(1–485) truncation background (ΔPiccolo), the Eaf1 subunit was TAP tagged). Purified complexes were analyzed by SDS–PAGE, and western blot (data not shown, see examples in17). In some cases, samples were further purified by gel filtration on a Superose 6 column (GE Healthcare).

EM samples of NuA4 and SAGA preserved in stain were prepared as described42. Samples were imaged under low-dose conditions on a field emission gun (FEG) Tecnai F-20 microscope (FEI) (acceleration voltage 120 kV). Images were recorded on a Gatan 4096 × 4096 CCD camera at 50,000 × magnification and ∼500 nm underfocus. Two-fold pixel binning resulted in a final pixel size of 4.34 Å. Images of individual NuA4 and SAGA particles (∼5,000 for each complex) were selected by hand. Image processing was done using the Spider and SPARX software packages44,45. Single-particle images were aligned without a reference and subjected to hierarchical ascendant clustering46 to produce class averages.

Calculation of 3D NuA4 structuresAn initial NuA4 3D map was calculated by the random conical tilt (RCT) method29 as described42. Cryo-EM samples were also prepared as described42 and imaged under low-dose conditions using a FEG CM200 (Philips/FEI) microscope (acceleration voltage 120 kV). Images were recorded on Kodak SO-163 film at a magnification of 66,000 × and with 0.9–4.0 μm underfocus values. Micrographs were digitized on a Zeiss/SCAIA flat bed densitometer (ZI/Zeiss) using a step size of 7 μm. Digitized images were 4-fold pixel averaged to 4.11 Å per pixel. NuA4 particle images (∼35,000) were selected by hand, divided into groups according to defocus, and a cryo-EM map of NuA4 was calculated by projection matching refinement as described42 using the NuA4 RCT reconstruction as initial reference. Three-dimensional structure interpretation and image rendering were done using UCSF Chimera47.

Calculation of 3D NuA4 structures

An initial NuA4 3D map was calculated by the random conical tilt (RCT) method29 as described42. Cryo-EM samples were also prepared as described42 and imaged under low-dose conditions using a FEG CM200 (Philips/FEI) microscope (acceleration voltage 120 kV). Images were recorded on Kodak SO-163 film at a magnification of 66,000 × and with 0.9–4.0 μm underfocus values. Micrographs were digitized on a Zeiss/SCAIA flat bed densitometer (ZI/Zeiss) using a step size of 7 μm. Digitized images were 4-fold pixel averaged to 4.11 Å per pixel. NuA4 particle images (∼35,000) were selected by hand, divided into groups according to defocus, and a cryo-EM map of NuA4 was calculated by projection matching refinement as described42 using the NuA4 RCT reconstruction as initial reference. Three-dimensional structure interpretation and image rendering were done using UCSF Chimera47.

Epl1 labeling and NuA4 incubation with the NCPNuA4 at 20 μg ml−1 was incubated for 30 min at 25 °C with a five-fold molar excess of polyclonal anti-CBP antibody (Open Biosystems) targeting the calmodulin binding peptide (CBP) tag on the Epl1 subunit. EM samples were then prepared and imaged as described above. Individual particle images (∼23,000) were selected by hand, and ∼500 were categorized by visual inspection as images showing the NuA4–anti-TAP interaction. Gold-cluster labeling of Epl1 was done by incubating NuA4 at 20 μg ml−1 with a 20-fold molar excess of thiol-protected calmodulin-bioconjugated gold nanoparticles34,35 for 2 h at 4 °C. Incubation with the gold probe was carried out in the presence and in the absence of calcium, and about 4,000 images from each condition were recorded and analyzed. To form the NuA4–NCP complex, NuA4 at 50 μg ml−1 was incubated with a 9-fold molar excess of NCPs reconstituted from full-length recombinant histones from Xenopus and mouse mammary tumor virus long terminal repeat NucB DNA36,37. EM samples were prepared and imaged as before.

Epl1 labeling and NuA4 incubation with the NCP

NuA4 at 20 μg ml−1 was incubated for 30 min at 25 °C with a five-fold molar excess of polyclonal anti-CBP antibody (Open Biosystems) targeting the calmodulin binding peptide (CBP) tag on the Epl1 subunit. EM samples were then prepared and imaged as described above. Individual particle images (∼23,000) were selected by hand, and ∼500 were categorized by visual inspection as images showing the NuA4–anti-TAP interaction. Gold-cluster labeling of Epl1 was done by incubating NuA4 at 20 μg ml−1 with a 20-fold molar excess of thiol-protected calmodulin-bioconjugated gold nanoparticles34,35 for 2 h at 4 °C. Incubation with the gold probe was carried out in the presence and in the absence of calcium, and about 4,000 images from each condition were recorded and analyzed. To form the NuA4–NCP complex, NuA4 at 50 μg ml−1 was incubated with a 9-fold molar excess of NCPs reconstituted from full-length recombinant histones from Xenopus and mouse mammary tumor virus long terminal repeat NucB DNA36,37. EM samples were prepared and imaged as before.

EM characterization of the Piccolo and Piccolo–NCP complexesRecombinant Piccolo NuA4 complex was expressed in bacteria using a polycistronic vector and purified as described3,4. The Piccolo–NCP complex was assembled as described3. To promote complex stability, Piccolo–NCP (∼215 μg ml−1) was crosslinked for 10 min with 1% (v/v) formaldehyde and quenched with 320 mM glycine. The sample was diluted to a final Piccolo–NCP concentration of 20 μg ml−1 before preparing and imaging EM samples as described for stained NuA4 specimens. Images were recorded on film and digitized using a Coolscan 9000 scanner (Nikkon). Individual particle images were selected by hand or by automated particle selection with the TiltPicker and DoGPicker software tools48.

EM characterization of the Piccolo and Piccolo–NCP complexes

Recombinant Piccolo NuA4 complex was expressed in bacteria using a polycistronic vector and purified as described3,4. The Piccolo–NCP complex was assembled as described3. To promote complex stability, Piccolo–NCP (∼215 μg ml−1) was crosslinked for 10 min with 1% (v/v) formaldehyde and quenched with 320 mM glycine. The sample was diluted to a final Piccolo–NCP concentration of 20 μg ml−1 before preparing and imaging EM samples as described for stained NuA4 specimens. Images were recorded on film and digitized using a Coolscan 9000 scanner (Nikkon). Individual particle images were selected by hand or by automated particle selection with the TiltPicker and DoGPicker software tools48.

Analytical ultracentrifugation experimentsSamples equilibrated in 20 mM Tris–Cl pH 8.0, 50 mM KCl, 5 mM 2-mercaptoethanol buffer were brought to sedimentation equilibrium in a Beckman XL-A analytical ultracentrifuge fitted with an AN-60Ti rotor, operating at 4.0 ± 0.1 °C. Five data sets were collected at 260 nm and five at 280 nm for each sample, at three rotor speeds (2,600, 5,900 and 14,200 × g) and three nominal concentrations (0.37, 0.19 and 0.09 mg ml−1 for Piccolo; 0.15, 0.07 and 0.04 μg ml−1 for the DNA-containing complexes). Absorbance versus radial position data were fit by one- and two-species models using partial specific volumes calculated from the amino acid and nucleic acid compositions of the complexes49. Global analysis of multiple data sets was performed with the program PRISM (Graphpad, Inc.).

Analytical ultracentrifugation experiments

Samples equilibrated in 20 mM Tris–Cl pH 8.0, 50 mM KCl, 5 mM 2-mercaptoethanol buffer were brought to sedimentation equilibrium in a Beckman XL-A analytical ultracentrifuge fitted with an AN-60Ti rotor, operating at 4.0 ± 0.1 °C. Five data sets were collected at 260 nm and five at 280 nm for each sample, at three rotor speeds (2,600, 5,900 and 14,200 × g) and three nominal concentrations (0.37, 0.19 and 0.09 mg ml−1 for Piccolo; 0.15, 0.07 and 0.04 μg ml−1 for the DNA-containing complexes). Absorbance versus radial position data were fit by one- and two-species models using partial specific volumes calculated from the amino acid and nucleic acid compositions of the complexes49. Global analysis of multiple data sets was performed with the program PRISM (Graphpad, Inc.).

Piccolo NCP binding and acetyltransferase activityBacterial expression vectors for different His–Esa1, GST–Epl1 (1–485), GST–Epl1 (486–832), His–Yng2 (1–282) and His–Yng2 (1–218) have been used before4, and cloning details are available upon request. Piccolo NuA4 complexes used in the experiments shown in Figs. 3 and 5 were produced by coexpression in E. coli and purified as described previously3. Single GST- and His-recombinant proteins were expressed and purified on glutathione Sepharose or nickel–agarose beads following standard procedures. His-tagged recombinant proteins were precleared on GST–Sepharose for 1 h at 4 °C prior to pulldown. 300 ng of rEsa1 and 200 ng of rYng2 were incubated (3 h at 4 °C) with 300 ng of appropriate GST fusion (or 1 ng of GST control). Equivalent amounts of input, bound, and free fractions were analyzed by western blot using an anti-HIS antibody (BabCo). For HAT assays, individual soluble recombinant proteins were incubated together at a similar ratio for 15 min on ice and 20 min at 30 °C before the assay, which was done with short oligonucleosomes purified from HeLa cells as previously described1. Different mixtures of recombinant proteins were prepared with a constant amount of rEsa1. All values were standardized to the activity of rEsa1 alone on free histones. Data shown in Fig. 4d are representative of two different experiments done in duplicate. For nucleosome binding experiments, a labeled 5S RNA gene was reconstituted in mononucleosomes as described50. rEsa1, rEpl1, rYng2 or rPiccolo trimeric complex were incubated with 12.5 ng of total nucleosomes for 30 min at 30 °C. Binding reactions were resolved on a 4% (w/v) PAGE gel in 0.5 × TBE. For measurement of histone copurification with NuA4 from whole cell extracts, wild-type and EAF1-deleted Eaf5–TAP strains10 were fractionated over IgG Sepharose followed by TEV elution, keeping salt concentration at 150 mM. Co-purifying Piccolo subunits and histones were detected by western blot and silver staining after separation by gel electrophoresis.

Piccolo NCP binding and acetyltransferase activity

Bacterial expression vectors for different His–Esa1, GST–Epl1 (1–485), GST–Epl1 (486–832), His–Yng2 (1–282) and His–Yng2 (1–218) have been used before4, and cloning details are available upon request. Piccolo NuA4 complexes used in the experiments shown in Figs. 3 and 5 were produced by coexpression in E. coli and purified as described previously3. Single GST- and His-recombinant proteins were expressed and purified on glutathione Sepharose or nickel–agarose beads following standard procedures. His-tagged recombinant proteins were precleared on GST–Sepharose for 1 h at 4 °C prior to pulldown. 300 ng of rEsa1 and 200 ng of rYng2 were incubated (3 h at 4 °C) with 300 ng of appropriate GST fusion (or 1 ng of GST control). Equivalent amounts of input, bound, and free fractions were analyzed by western blot using an anti-HIS antibody (BabCo). For HAT assays, individual soluble recombinant proteins were incubated together at a similar ratio for 15 min on ice and 20 min at 30 °C before the assay, which was done with short oligonucleosomes purified from HeLa cells as previously described1. Different mixtures of recombinant proteins were prepared with a constant amount of rEsa1. All values were standardized to the activity of rEsa1 alone on free histones. Data shown in Fig. 4d are representative of two different experiments done in duplicate. For nucleosome binding experiments, a labeled 5S RNA gene was reconstituted in mononucleosomes as described50. rEsa1, rEpl1, rYng2 or rPiccolo trimeric complex were incubated with 12.5 ng of total nucleosomes for 30 min at 30 °C. Binding reactions were resolved on a 4% (w/v) PAGE gel in 0.5 × TBE. For measurement of histone copurification with NuA4 from whole cell extracts, wild-type and EAF1-deleted Eaf5–TAP strains10 were fractionated over IgG Sepharose followed by TEV elution, keeping salt concentration at 150 mM. Co-purifying Piccolo subunits and histones were detected by western blot and silver staining after separation by gel electrophoresis.

Supplementary Material1

Supplementary Material
