---
pmid: '14739338'
title: Genome-wide analysis of gene expression reveals function of the bZIP transcription
  factor HY5 in the UV-B response of Arabidopsis.
authors:
- Ulm R
- Baumann A
- Oravecz A
- Máté Z
- Adám E
- Oakeley EJ
- Schäfer E
- Nagy F
journal: Proc Natl Acad Sci U S A
year: '2004'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC337064
doi: 10.1073/pnas.0308044100
---

# Genome-wide analysis of gene expression reveals function of the bZIP transcription factor HY5 in the UV-B response of Arabidopsis.
**Authors:** Ulm R, Baumann A, Oravecz A, Máté Z, Adám E, Oakeley EJ, Schäfer E, Nagy F
**Journal:** Proc Natl Acad Sci U S A (2004)
**DOI:** [10.1073/pnas.0308044100](https://doi.org/10.1073/pnas.0308044100)
**PMC:** [PMC337064](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC337064/)

## Abstract

1. Proc Natl Acad Sci U S A. 2004 Feb 3;101(5):1397-402. doi: 
10.1073/pnas.0308044100. Epub 2004 Jan 22.

Genome-wide analysis of gene expression reveals function of the bZIP 
transcription factor HY5 in the UV-B response of Arabidopsis.

Ulm R(1), Baumann A, Oravecz A, Máté Z, Adám E, Oakeley EJ, Schäfer E, Nagy F.

Author information:
(1)Institute of Biology II/Botany, University of Freiburg, Schänzlestrasse 1, 
D-79104 Freiburg, Germany. roman.ulm@biologie.uni-freiburg.de

The light environment is a key factor that governs a multitude of developmental 
processes during the entire life cycle of plants. An important and increasing 
part of the incident sunlight encompasses a segment of the UV-B region (280-320 
nm) that is not entirely absorbed by the ozone layer in the stratosphere of the 
earth. This portion of the solar radiation, which inevitably reaches the sessile 
plants, can act both as an environmental stress factor and an informational 
signal. To identify Arabidopsis genes involved in the UV response, we monitored 
the gene expression profile of UV-B-irradiated seedlings by using high-density 
oligonucleotide microarrays comprising almost the full Arabidopsis genome 
(>24,000 genes). A robust set of early low-level UV-B-responsive genes, 100 
activated and 7 repressed, was identified. In all cases analyzed, UV-B induction 
was found to be independent of known photoreceptors. This group of genes is 
suggested to represent the molecular readout of the signaling cascade triggered 
by the elusive UV-B photoreceptor(s). Moreover, our analysis identified 
interactions between cellular responses to different UV-B ranges that led us to 
postulate the presence of partially distinct but interacting UV-B perception and 
signaling mechanisms. Finally, we demonstrate that the bZIP transcription factor 
HY5 is required for UV-B-mediated regulation of a subset of genes.

DOI: 10.1073/pnas.0308044100
PMCID: PMC337064
PMID: 14739338 [Indexed for MEDLINE]

## Full Text

Abstract

The light environment is a key factor that governs a multitude of developmental processes during the entire life cycle of plants. An important and increasing part of the incident sunlight encompasses a segment of the UV-B region (280–320 nm) that is not entirely absorbed by the ozone layer in the stratosphere of the earth. This portion of the solar radiation, which inevitably reaches the sessile plants, can act both as an environmental stress factor and an informational signal. To identify Arabidopsis genes involved in the UV response, we monitored the gene expression profile of UV-B-irradiated seedlings by using high-density oligonucleotide microarrays comprising almost the full Arabidopsis genome (>24,000 genes). A robust set of early low-level UV-B-responsive genes, 100 activated and 7 repressed, was identified. In all cases analyzed, UV-B induction was found to be independent of known photoreceptors. This group of genes is suggested to represent the molecular readout of the signaling cascade triggered by the elusive UV-B photoreceptor(s). Moreover, our analysis identified interactions between cellular responses to different UV-B ranges that led us to postulate the presence of partially distinct but interacting UV-B perception and signaling mechanisms. Finally, we demonstrate that the bZIP transcription factor HY5 is required for UV-B-mediated regulation of a subset of genes.

Materials and Methods

Plant Material and Growth Conditions. In all experiments, except as noted, the wild-type A. thaliana ecotype was Wassilewskija. The cry1-304cry2-1 ( 22 ) and phot1-5phot2-1 ( 23 ) mutants are in the Columbia ecotype, whereas the hy5-1 ( 24 ), phyA-201phyB-5 ( 25 ), and uvr2-1 ( 26 ) mutants are in the Landsberg erecta ecotype.

Arabidopsis seeds were surface-sterilized with sodiumhypochlorite and plated on Murashige and Skoog medium (Sigma) containing 1% sucrose and 0.8% agar. Seeds were stratified for at least 2 days at 4°C and germinated aseptically at 25°C in a standard growth chamber (MLR-350, Sanyo, Gunma, Japan) with a 12-h/12-h light/dark cycle.

UV-B Irradiation. Seedlings of Arabidopsis were irradiated at midday in a UV-B light field consisting of six Philips TL 40W/12 UV fluorescent tubes (λ max = 310 nm, half-bandwidth = 40 nm, fluence rate = 7 W/m 2 ) filtered through 3-mm transmission cutoff filters of the WG series with half-maximal transmission at the indicated wavelength (WG295, WG305, and WG327; Schott, Mainz, Germany), or unfiltered through a 3-mm quartz plate ( Fig. 1 ). After 15-min irradiation, the seedlings were immediately transferred back into the standard growth chamber, where in parallel the nonirradiated controls were kept. Spectral energy distributions of UV-B sources were measured with an OL 754 UV-visible spectroradiometer (Optronix Laboratories, Orlando, FL). UV-B irradiance and radiant exposure were weighted with the generalized plant action spectrum, normalized at 300 nm (according to ref. 27 ), giving the biologically effective (BE) quantity, UV BE [Wm –2 ], for WG327: 0.0004, WG305: 0.12; WG295: 0.42, quartz: 1.18. In comparison, sunlight on a sunny day in July in Freiburg was measured as UV BE = 0.05 Wm –2 . After irradiation of the seedlings for the indicated times, plates were immediately returned to the standard growth chamber until the tissue was harvested and snap-frozen in liquid nitrogen.

Molecular Methods. Arabidopsis RNA was isolated with the Plant RNeasy Kit (Qiagen, Chatsworth, CA), according to the manufacturer's instructions. Gene-specific probes (detailed information for each probe can be obtained from the authors) were amplified by PCR from Arabidopsis cDNA, cloned into the pCR2.1-TOPO vector (Invitrogen) and verified by sequencing.

For RNA gel blot analysis, RNA samples of 10 μg were electrophoretically separated in 1% formaldehyde-agarose gels and transferred to Hybond-N+ membranes (Amersham Biosciences). Probes were 32 P-dCTP-labeled with the Random Primers DNA Labeling System (Invitrogen), and hybridization was performed in 50% formamide/0.5% SDS/5× SSC/50 mM NaHPO 4 , pH6.5/5× Denhardt's solution/0.1 mg/ml salmon sperm DNA. Membranes were washed sequentially with 2×SSC/0.2% SDS, 1× SSC/0.2% SDS, and 0.5× SSC/0.2% SDS and analyzed by autoradiography.

Microarray Analysis. Ten micrograms of total RNA (isolated from ≈50 7-day-old Arabidopsis seedlings) was reverse transcribed by using the SuperScript Choice system for cDNA synthesis (Life Technologies, Grand Island, NY) according to the protocol recommended by Affymetrix (Santa Clara, CA; GeneChip Expression Analysis). The oligonucleotide used for priming was 5′-ggccagtgaattgtaatacgactcactatagggaggcgg-(t) 24 -3′ (Genset Oligos, Paris), as recommended by Affymetrix. Double-stranded cDNA was purified by phenol/chloroform extraction, and the aqueous phase was removed by centrifugation through Phase-lock Gel (Eppendorf). In vitro transcription was performed on 1 μg of cDNA by using the Enzo BioArray High Yield RNA transcript labeling kit (Enzo Diagnostics) following the manufacturer's protocol. The cRNA was purified by using RNAeasy clean-up columns (Qiagen). To improve recovery from the columns, the elution water was spun into the matrix at 27 × g and then left to stand for 1 min before the standard 8,000 × g centrifugation recommended by Qiagen. The cRNA was fragmented by heating in 1× fragmentation buffer (40 mM Trisacetate, pH 8.1/100 mM KOAc/30 mM MgOAc) as recommended by Affymetrix. Ten micrograms of fragmented cRNA was hybridized to an Arabidopsis ATH1 GeneChip (Affymetrix) by using their standard procedure (45°C, 16 h). Washing and staining were performed in a Fluidics Station 400 (Affymetrix) by using the protocol EukGE-WS2v4 and scanned in an Affymetrix GeneChip scanner. Chip analysis was performed by using the Affymetrix microarray suite Version 5 (target intensity 500 was used for chip scaling) and genespring 5.0 (Silicon Genetics, Redwood City, CA). Changes in gene expression were assessed by looking for concordant changes between replicates by using a signed Wilcoxon rank test (as recommended by Affymetrix). The “change” P value threshold was <0.003 for increase and >0.997 for decrease. After concordance analysis, these values become <9 × 10 –6 and >0.999991, respectively. Any gene whose detection P value was >0.05 in all experimental conditions was discarded from the analysis as unreliable data.

Luciferase Imaging. Fragments of ≈1.5 kb upstream of the start ATG of select UV-B-responsive genes were obtained by PCR reactions on genomic DNA of Arabidopsis (Wassilewskija ecotype) and fused to the luciferase ( Luc + ) reporter gene (Promega) in a pPCV812-derived binary vector ( 28 ) (detailed information on the promoter fragments can be obtained from the authors). The identity and integrity of the promoter fragments were confirmed by sequencing. Arabidopsis plants were transformed by the floral dip method ( 29 ). Seven-day-old seedlings of T2 segregating populations of the promoter:: Luc + lines were sprayed with 5 mM luciferin solution (Biosynth, Basel), and luciferase luminescence was measured by a liquid nitrogen-cooled charge-coupled device camera (Astrocam, Paris).

Results and Discussion

Genomic UV-B-Response in Arabidopsis Detected by Whole-Genome Microarray Analysis. Oligonucleotide microarrays containing >24,000 genes (Affymetrix ATH1 GeneChip) were used to quantitatively assess changes in gene expression in response to UV-B radiation in Arabidopsis . Seven-day-old white-light-grown seedlings were exposed for 15 min to polychromatic radiation with decreasing short-wave cutoff in the UV range, transferred back to the standard growth chamber, and samples were taken 1 and 6 h after the start of irradiation. Three different filter glasses with transmission cutoffs at 327 (i.e., transmitted wavelength >327 nm), 305 (>305 nm), and 295 nm (>295 nm), and a quartz glass (unfiltered) were used to produce four different UV spectra ( Fig. 1 ), of which the 327-nm cutoff represents the minus UV-B control. It should be noted that with this experimental setup, all parts of the spectrum except the UV-B region are held constant. The irradiation conditions used here span the range from very low (WG305) to low (unfiltered through quartz glass) UV-B levels, according to the recently proposed categories ( 1 ).

The comparison of the 327-nm cutoff control to the UV-B spectra under the 305-nm cutoff filter at 1 h postirradiation was deduced from four independent biological samples and array hybridizations, producing a robust set of 100 and 7 genes that are induced and repressed >2-fold by very low-level UV-B, respectively (Figs. 2 and 3 ), most of which had not previously been associated with UV responses in Arabidopsis . The numbers are 145 activated and 29 repressed genes, when no fold-threshold is applied (see Tables 1 and 2, which are published as supporting information on the PNAS web site). All other comparisons are from two independent biological repetitions. Use of a 295-nm cutoff or unfiltered UV-B (with quartz glass) led to the identification of genes that are inducible by shorter wavelength ranges of UV-B ( Fig. 3 ). Applying a 2-fold threshold, irradiation with UV-B >295 nm results in the activation of 601 genes, whereas 117 genes are repressed after 1 h ( Fig. 3 , Tables 3 and 4, which are published as supporting information on the PNAS web site). Under quartz, 661 and 365 genes are up- and down-regulated, respectively ( Fig. 3 , Tables 5 and 6, which are published as supporting information on the PNAS web site). It is of note that at 6 h postirradiation, exposure to UV-B >305 nm resulted in only two genes with >2-fold change in expression (Tables 7 and 8, which are published as supporting information on the PNAS web site), in stark contrast to the number of affected genes detected after treatment with UV-B >295 nm (165 up- and 114 down-regulated genes; Tables 9 and 10, which are published as supporting information on the PNAS web site) and particularly with unfiltered UV light (under quartz) (1,716 and 1,535; Tables 11 and 12, which are published as supporting information on the PNAS web site) ( Fig. 3 ). This clearly indicates that there is no sustained cellular effect after irradiation under the 305-nm (and to a lesser extent under the 295-nm) cutoff filter. This treatment is therefore considered marginal, capable of eliciting changes of gene expression with negligible damage. Moreover, these results also suggest that this set of genes is regulated by specific UV-B perception and signaling mechanism. Consistent with the transient nature of the gene expression changes, however, the short and low-level UV-B exposure does not cause any visible phenotype in Arabidopsis seedlings. It is also of note that the irradiation is carried out on white-light-grown plants, under photoreactivating conditions ( 30 ).

Several of the previously described low-level UV-B-induced or repressed genes ( 1 , 13 ) were also identified in our analysis : MEB5.2 (At3g17800, Fig. 2 , Tables 1 and 11), PyroA (At5g01410, Table 1), and the negative UV-B regulator MYB4 (At4g38620, Table 6) ( 31 ). Interestingly, another MYB transcription factor, MYB34 / ATR1 (At5g60890), implicated in tryptophan pathway gene regulation ( 32 ), is identified as a robust UV-B-responsive transcript with a fast turnover that is down-regulated under all UV ranges already after 1 h ( Fig. 2 and Tables 2, 4, and 6). However, the UV-B level used in our experiments is too low, even in the case of unfiltered UV-B (i.e., quartz), to activate the “intermediate and high-level UV-B pathway markers” (according to a recent model in ref. 1 ), including PR-1 (At2g19990), PR-5 (At1g75030), or PDF1.2 (At5g44420). We focus this report on the set of early-responsive genes that are induced by the very low level UV-B under the 305-nm cutoff ( Fig. 2 ), including their responses to UV-B extended to shorter wavelength ranges.

A Subset of Early Low-Level UV-B-Inducible Genes Is Negatively Regulated by Shorter Wavelength UV Irradiation. To identify genes that are coordinately regulated by UV-B and that might be downstream of shared signaling pathways, we carried out a cluster analysis on the 145 UV-B-induced genes (no fold-threshold applied). Surprisingly, as already indicated by Venn diagram analysis ( Fig. 3 ), a number of genes are antagonized by the shorter wavelength ranges included in the quartz treatment compared to the 305-nm cutoff ( Fig. 4 A ). This is particularly interesting because the spectra are identical except for the extension to shorter wavelength ranges ( Fig. 1 ). Obviously, this is true only for a specific subset of genes; many other genes are regulated as expected (Figs. 3 and 4 B ). Thus, the data strongly indicate the presence and interaction of at least two UV-B perception and signaling pathways. One pathway is triggered by the longer wavelengths of UV-B radiation, whereas a second pathway is activated by shorter wavelengths of the UV-B spectrum, with the latter negatively interfering with the former ( Fig. 4 C ). The P1 perception system may illustrate a specific UV-B photoreceptor, whereas the P2 system may represent indirect effects of UV-B exposure through general cellular stress pathway or a distinct UV-B photoreceptor ( Fig. 4 C ). Interestingly, neither the induction under 305-nm cutoff of the genes analyzed nor the antagonistic effect under quartz was found to be enhanced in the DNA repair mutant uvr2 , devoid of the photolyase specific for the repair of cyclobutane-pyrimidine dimers, the major UV-B-generated DNA damage ( Fig. 5 ) ( 26 ). This argues against the involvement of particular DNA damage in signaling to gene expression changes, in the induction as well as the antagonistic shorter wavelength pathway. However, verification of the postulated UV-B pathways will require the identification of the main components of this regulatory interaction.

A Number of Transcription Factors Are UV-B-Responsive. Of the core 107 genes that are regulated by low-level UV-B (Figs. 2 and 3 ), 64% are currently annotated as encoding proteins of known or putative functions. The remaining fraction comprises predicted proteins of unknown function that, however, may now be connected with UV responses in Arabidopsis . The functionally annotated genes indicate the importance of diverse cellular processes in response to UV-B ( Fig. 2 ). In particular, a number of these UV-responsive genes encode transcription regulators (>30% of genes with known or predicted functions), including genes encoding transcription factors implicated in response to abiotic stress (DREB2A, ABF3, ZAT10, and ZAT12), during development (CIA2, COL1, and MYB13), in light responses (HY5 and HYH), and unknown functions (MYB44, MYB111, bHLH34, bHLH149, bHLH150, and two NAM-related proteins). The bZIP protein HY5 and its homolog HYH have crucial roles in light-regulated deetiolation ( 33 , 34 ). Characterization of the interactions of the UV response with other environmental signal-mediated pathways, in particular those triggered by other light qualities, will yield information into integration processes. However, the UV responsiveness of numerous transcription factors indicates the activation of a network of transcription factors downstream of the putative UV-B photoreceptor, similar to the phytochrome A mode of action ( 5 ). To our knowledge, none of the transcription-related factors identified, except for MYB4 ( 31 ), has previously been linked to UV-B responses in plants; however, they now clearly represent major candidates for the functional assessment of their involvement in the conceivable UV-B transcriptional network.

The Low-Level UV-B Inducible Genes Are Independent of Known Photoreceptors, Whereas a Subset Depends on HY5. The UV-B-mediated transcriptional activation of the well established photomorphogenic transcription factor HY5 suggests that it plays a role during UV response. Indeed, we found that loss of HY5 impairs the UV-B-responsive expression of several genes, including At3g24750, At4g14690 ( ELIP1 ), At4g15480, At4g21200, At5g05410 ( DREB2A ), and At5g52250 ( Fig. 5 and data not shown). Thus, our data demonstrate the HY5 requirement for appropriate response to UV-B and the use of shared components in response to visible light and UV-B.

To investigate the possibility that known photoreceptors of Arabidopsis are involved in the UV-B perception leading to changes in gene expression, we analyzed compound mutants of phytochromes A and B ( phyAphyB ), cryptochromes 1 and 2 ( cry1cry2 ), and phototropins 1 and 2 ( phot1phot2 ). The results on UV-B-induced expression of select genes clearly indicate independence of the corresponding photoreceptors ( Fig. 5 and data not shown). In addition, it should be noted that the Arabidopsis Wassilewskija ecotype used for our expression analysis is phyD-deficient ( 35 ). Thus, the analyzed genes are UV-B-induced independently of photoreceptors that perceive far-red/red or blue/UV-A light, strongly suggesting that they are activated through a specific UV-B photoreceptor, with HY5 as a down-stream signaling component.

It is known that HY5 itself is transcriptionally activated in a phytochrome-dependent manner in etiolated seedlings exposed to light ( 3 , 5 ). In contrast, UV-B-mediated transcriptional activation of HY5 is independent of phyA and -B ( Fig. 5 ), suggesting an alternative input pathway to its transcriptional regulation. Moreover, the HY5 homolog HYH that also functions during light responses and interacts with HY5 ( 33 ) is up-regulated in response to UV-B ( Fig. 2 ), independent of phyA, phyB, and HY5 (data not shown). This finding indicates potential overlapping functions of the two bZIP transcription factors during UV-B responses and may be responsible for the retained partial gene activation in the hy5-1 null mutant ( Fig. 5 ) ( 34 ).

Transcriptional Regulation of Select Genes Operates at the Promoter Level. Expression analysis using microarray and RNA gel blot analysis detects alterations in the steady-state levels of transcripts but does not differentiate between altered transcription rate and stability. This, however, can be done with the luciferase reporter gene under the control of select promoters ( 36 ). We generated transgenic lines for a number of UV-B-responsive promoters and analyzed luciferase activity after UV-B exposure. Indeed, we were able to demonstrate that the UV response operates at the level of transcription for the genes analyzed (At1g32870, At2g36750, At3g21890, At4g14690, At4g15480, At5g05410, and At5g59820; data not shown), including HY5 ( Fig. 6 ).

Conclusion

The data presented here describe an extensive assessment of the Arabidopsis UV transcriptome at the genome-wide level and link the key photomorphogenic transcriptional activator HY5 to responses to the UV-B region of the light spectrum. Together, these developments set the stage for further investigation of molecular mechanisms enabling plants to cope with increasing levels of UV-B, ultimately leading to a more complete understanding of plants' responses to the complex light environment.

In sharp contrast to the success of forward genetic approaches for mutants with altered responses to the visible light spectrum that led to the identification of the molecular nature of photoreceptors, their downstream signaling components, and effector proteins ( 3 ), our molecular understanding of these processes in the response to UV-B is rather limited, a fact made most apparent by the lack of a molecularly identified UV-B photoreceptor ( 1 ). This might be due to the paucity of well-defined visible phenotypes and confounding damaging aspects, which might have rendered conventional genetic screens problematic. Here we established a number of promoter:: Luc + transgenic Arabidopsis lines that will enable luciferase reporter-based genetic screens for mutants affected in UV-B light-regulated gene transcription, to approach the missing UV-B photoreceptor(s) and the related signaling components.
