---
pmid: '38848361'
title: Mechanism of human PINK1 activation at the TOM complex in a reconstituted system.
authors:
- Raimi OG
- Ojha H
- Ehses K
- Dederer V
- Lange SM
- Rivera CP
- Deegan TD
- Chen Y
- Wightman M
- Toth R
- Labib KPM
- Mathea S
- Ranson N
- Fernández-Busnadiego R
- Muqit MMK
journal: Sci Adv
year: '2024'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC11160474
doi: 10.1126/sciadv.adn7191
---

# Mechanism of human PINK1 activation at the TOM complex in a reconstituted system.
**Authors:** Raimi OG, Ojha H, Ehses K, Dederer V, Lange SM, Rivera CP, Deegan TD, Chen Y, Wightman M, Toth R, Labib KPM, Mathea S, Ranson N, Fernández-Busnadiego R, Muqit MMK
**Journal:** Sci Adv (2024)
**DOI:** [10.1126/sciadv.adn7191](https://doi.org/10.1126/sciadv.adn7191)
**PMC:** [PMC11160474](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11160474/)

## Abstract

1. Sci Adv. 2024 Jun 7;10(23):eadn7191. doi: 10.1126/sciadv.adn7191. Epub 2024
Jun  7.

Mechanism of human PINK1 activation at the TOM complex in a reconstituted 
system.

Raimi OG(1), Ojha H(1), Ehses K(2)(3), Dederer V(4)(5)(6), Lange SM(1), Rivera 
CP(1), Deegan TD(1)(7), Chen Y(1), Wightman M(1), Toth R(1), Labib KPM(1), 
Mathea S(4)(5)(6), Ranson N(8), Fernández-Busnadiego R(2)(3)(4)(9), Muqit 
MMK(1)(4).

Author information:
(1)MRC Protein Phosphorylation and Ubiquitylation Unit, School of Life Sciences, 
University of Dundee, Dundee DD1 5EH, UK.
(2)Institute of Neuropathology, University Medical Center Göttingen, 37099 
Göttingen, Germany.
(3)Cluster of Excellence "Multiscale Bioimaging: from Molecular Machines to 
Networks of Excitable Cells" (MBExC), University of Göttingen, Göttingen, 
Germany.
(4)Aligning Science Across Parkinson's (ASAP) Collaborative Research Network, 
Chevy Chase, MD 20815, USA.
(5)Institute of Pharmaceutical Chemistry, Goethe-Universität, 60438 Frankfurt, 
Germany.
(6)Structural Genomics Consortium (SGC), Buchmann Institute for Life Sciences, 
Goethe-Universität, 60438 Frankfurt, Germany.
(7)MRC Human Genetics Unit, Institute of Genetics and Cancer, University of 
Edinburgh, Western General Hospital, Edinburgh EH4 2XU, UK.
(8)Astbury Centre for Structural Molecular Biology, School of Molecular and 
Cellular Biology, Faculty of Biological Sciences, University of Leeds, Leeds LS2 
9JT, UK.
(9)Faculty of Physics, University of Göttingen, Friedrich-Hund-Platz 1, 37077 
Göttingen, Germany.

Loss-of-function mutations in PTEN-induced kinase 1 (PINK1) are a frequent cause 
of early-onset Parkinson's disease (PD). Stabilization of PINK1 at the 
translocase of outer membrane (TOM) complex of damaged mitochondria is critical 
for its activation. The mechanism of how PINK1 is activated in the TOM complex 
is unclear. Here, we report that co-expression of human PINK1 and all seven TOM 
subunits in Saccharomyces cerevisiae is sufficient for PINK1 activation. We use 
this reconstitution system to systematically assess the role of each TOM subunit 
toward PINK1 activation. We unambiguously demonstrate that the TOM20 and TOM70 
receptor subunits are required for optimal PINK1 activation and map their sites 
of interaction with PINK1 using AlphaFold structural modeling and mutagenesis. 
We also demonstrate an essential role of the pore-containing subunit TOM40 and 
its structurally associated subunits TOM7 and TOM22 for PINK1 activation. These 
findings will aid in the development of small-molecule activators of PINK1 as a 
therapeutic strategy for PD.

DOI: 10.1126/sciadv.adn7191
PMCID: PMC11160474
PMID: 38848361 [Indexed for MEDLINE]

## Full Text

INTRODUCTIONAutosomal recessive mutations in PTEN-induced kinase 1 (PINK1) and the RING-IBR-RING ubiquitin E3 ligase Parkin are causal for early-onset Parkinson’s disease (PD) (1, 2). Cell-based studies have demonstrated that these proteins function together in a common mitochondrial quality control pathway (3–5). Active PINK1 phosphorylates both Parkin and ubiquitin at an equivalent serine-65 (Ser65) residue (6–9), resulting in activation of Parkin via a feed-forward mechanism triggering ubiquitin-dependent elimination of damaged mitochondria by autophagy (mitophagy) (3–5). Active PINK1 also indirectly induces the phosphorylation of a subset of Rab guanosine triphosphatases including Rab8A at a highly conserved serine residue (Ser111) that lies within the RabSF3 motif (10, 11). PINK1 encodes a 581–amino acid Ser/Thr protein kinase containing an N-terminal canonical mitochondrial targeting sequence (MTS) (residues 1 to 34), catalytic kinase domain containing three unique loop insertions (residues 156 to 513), and N-terminal and C-terminal extensions (NTE, residues 111 to 132; CTE, residues 514 to 581) that flank the kinase domain (12–14). Most PD-associated mutations are located within the kinase domain highlighting the protective role of PINK1 kinase activity against the development of PD (15, 16).Under basal conditions, newly translated PINK1 protein is rapidly imported into mitochondria through the translocase of outer membrane (TOM) complex whereupon it undergoes consecutive N-terminal cleavage by matrix mitochondrial processing peptidase (MPP) proteases and the inner mitochondrial membrane presenilin associated rhomboid-like protein (PARL) protease followed by retro-translocation into the cytosol and degradation by the 26S proteasome (17–20). Upon mitochondrial membrane depolarization that can be induced by mitochondrial uncouplers, for example, antimycin A/oligomycin, PINK1 import is blocked, leading to the accumulation/stabilization of full-length PINK1 at the outer mitochondrial membrane (OMM) (21–24) and catalytic activation (8, 25). Mitochondrial depolarization promotes PINK1 stabilization at the TOM complex that can be visualized by blue native polyacrylamide gel electrophoresis (BN-PAGE) as a ~700-kDa complex distinct from the native TOM complex that migrates as a ~500-kDa band (26, 27).The TOM complex is composed of seven subunits—TOM5, TOM6, TOM7, TOM20, TOM22, TOM40, and TOM70—and is highly conserved through evolution (28–30). It has been best characterized in yeast for its role in the recognition of mitochondrial precursors synthesized in the cytosol and their directed import through the pore-containing subunit TOM40 that spans the OMM (31, 32). Cryo–electron microscopy (cryo-EM) studies of yeast and the human TOM complex have revealed that they form dimeric structures requiring TOM22 binding to TOM40 to form the core complex (33–35). TOM70 and TOM20 are both loosely associated with the complex and have not been visualized in high-resolution cryo-EM structures to date (33–35) although a low resolution 6.8-Å cryo-EM structure of the N. crassa TOM complex has been reported with TOM20 (36). Both TOM70 and TOM20 contain large hydrophilic domains that extend into the cytosol and act as receptors with overlapping preference for mitochondrial precursor proteins (31, 32). The role of the smaller TOM subunits 5, 6, and 7 have been shown in yeast to be important for assembly of the core complex, but their role in mammalian cells has been less well understood (30–32, 37).Despite considerable research in the field, the mechanism of mammalian PINK1 activation at the TOM complex remains incompletely understood (3). In a genetic screen, it was discovered that the TOM7 subunit played an essential role for PINK1 stabilization and activation at the OMM (38, 39). We and others recently found an intramolecular interaction between the PINK1 NTE and CTE regions (12–14), and that this is required for stabilization of human PINK1 to the TOM complex (13, 14) and its subsequent activation by autophosphorylation at Ser228 (13). Several pathogenic PD mutations are located within the NTE:CTE interface that prevent human PINK1 recruitment to the TOM complex, highlighting the importance of this activation mechanism to disease (13, 14).Because of the essential role of the TOM complex in mammalian cells, a systematic analysis of the contribution of TOM subunits for PINK1 activation has not been possible to date, and, furthermore, it has not been established whether the stabilization of PINK1 to the TOM complex is necessary or sufficient for its activation. To dissect the molecular mechanisms of PINK1 stabilization at the TOM complex, we have used a reconstitution system in which genes encoding human PINK1 and the seven subunits of the human TOM complex have been introduced into the budding yeast, Saccharomyces cerevisiae, which do not express PINK1. Notably, we observe that co-expression of wild-type (WT) human PINK1 and the seven TOM subunits is sufficient to reconstitute PINK1 activation, and we have exploited this to assess the role of individual TOM subunits toward PINK1 activation. Combining the yeast reconstitution system with AlphaFold modeling and mutagenesis studies in mammalian cells, we propose the mechanism of how PINK1 is stabilized and activated at the TOM complex via interaction with the TOM20 and TOM70 subunits. This systematic analysis provides fundamental insights into the regulation of human PINK1 that will aid in the development of small-molecule activators of PINK1.

INTRODUCTION

Autosomal recessive mutations in PTEN-induced kinase 1 (PINK1) and the RING-IBR-RING ubiquitin E3 ligase Parkin are causal for early-onset Parkinson’s disease (PD) (1, 2). Cell-based studies have demonstrated that these proteins function together in a common mitochondrial quality control pathway (3–5). Active PINK1 phosphorylates both Parkin and ubiquitin at an equivalent serine-65 (Ser65) residue (6–9), resulting in activation of Parkin via a feed-forward mechanism triggering ubiquitin-dependent elimination of damaged mitochondria by autophagy (mitophagy) (3–5). Active PINK1 also indirectly induces the phosphorylation of a subset of Rab guanosine triphosphatases including Rab8A at a highly conserved serine residue (Ser111) that lies within the RabSF3 motif (10, 11). PINK1 encodes a 581–amino acid Ser/Thr protein kinase containing an N-terminal canonical mitochondrial targeting sequence (MTS) (residues 1 to 34), catalytic kinase domain containing three unique loop insertions (residues 156 to 513), and N-terminal and C-terminal extensions (NTE, residues 111 to 132; CTE, residues 514 to 581) that flank the kinase domain (12–14). Most PD-associated mutations are located within the kinase domain highlighting the protective role of PINK1 kinase activity against the development of PD (15, 16).

Under basal conditions, newly translated PINK1 protein is rapidly imported into mitochondria through the translocase of outer membrane (TOM) complex whereupon it undergoes consecutive N-terminal cleavage by matrix mitochondrial processing peptidase (MPP) proteases and the inner mitochondrial membrane presenilin associated rhomboid-like protein (PARL) protease followed by retro-translocation into the cytosol and degradation by the 26S proteasome (17–20). Upon mitochondrial membrane depolarization that can be induced by mitochondrial uncouplers, for example, antimycin A/oligomycin, PINK1 import is blocked, leading to the accumulation/stabilization of full-length PINK1 at the outer mitochondrial membrane (OMM) (21–24) and catalytic activation (8, 25). Mitochondrial depolarization promotes PINK1 stabilization at the TOM complex that can be visualized by blue native polyacrylamide gel electrophoresis (BN-PAGE) as a ~700-kDa complex distinct from the native TOM complex that migrates as a ~500-kDa band (26, 27).

The TOM complex is composed of seven subunits—TOM5, TOM6, TOM7, TOM20, TOM22, TOM40, and TOM70—and is highly conserved through evolution (28–30). It has been best characterized in yeast for its role in the recognition of mitochondrial precursors synthesized in the cytosol and their directed import through the pore-containing subunit TOM40 that spans the OMM (31, 32). Cryo–electron microscopy (cryo-EM) studies of yeast and the human TOM complex have revealed that they form dimeric structures requiring TOM22 binding to TOM40 to form the core complex (33–35). TOM70 and TOM20 are both loosely associated with the complex and have not been visualized in high-resolution cryo-EM structures to date (33–35) although a low resolution 6.8-Å cryo-EM structure of the N. crassa TOM complex has been reported with TOM20 (36). Both TOM70 and TOM20 contain large hydrophilic domains that extend into the cytosol and act as receptors with overlapping preference for mitochondrial precursor proteins (31, 32). The role of the smaller TOM subunits 5, 6, and 7 have been shown in yeast to be important for assembly of the core complex, but their role in mammalian cells has been less well understood (30–32, 37).

Despite considerable research in the field, the mechanism of mammalian PINK1 activation at the TOM complex remains incompletely understood (3). In a genetic screen, it was discovered that the TOM7 subunit played an essential role for PINK1 stabilization and activation at the OMM (38, 39). We and others recently found an intramolecular interaction between the PINK1 NTE and CTE regions (12–14), and that this is required for stabilization of human PINK1 to the TOM complex (13, 14) and its subsequent activation by autophosphorylation at Ser228 (13). Several pathogenic PD mutations are located within the NTE:CTE interface that prevent human PINK1 recruitment to the TOM complex, highlighting the importance of this activation mechanism to disease (13, 14).

Because of the essential role of the TOM complex in mammalian cells, a systematic analysis of the contribution of TOM subunits for PINK1 activation has not been possible to date, and, furthermore, it has not been established whether the stabilization of PINK1 to the TOM complex is necessary or sufficient for its activation. To dissect the molecular mechanisms of PINK1 stabilization at the TOM complex, we have used a reconstitution system in which genes encoding human PINK1 and the seven subunits of the human TOM complex have been introduced into the budding yeast, Saccharomyces cerevisiae, which do not express PINK1. Notably, we observe that co-expression of wild-type (WT) human PINK1 and the seven TOM subunits is sufficient to reconstitute PINK1 activation, and we have exploited this to assess the role of individual TOM subunits toward PINK1 activation. Combining the yeast reconstitution system with AlphaFold modeling and mutagenesis studies in mammalian cells, we propose the mechanism of how PINK1 is stabilized and activated at the TOM complex via interaction with the TOM20 and TOM70 subunits. This systematic analysis provides fundamental insights into the regulation of human PINK1 that will aid in the development of small-molecule activators of PINK1.

RESULTSReconstitution of human PINK1 activation at the TOM complex in yeastWe initially generated stable transformant strains of the budding yeast, S. cerevisiae, expressing human WT or kinase-inactive (KI; D384A) full-length PINK1-3FLAG with or without all seven subunits of the human TOM complex (TOM70, residues 1 to 608; TOM40, residues 1 to 361; TOM22, residues 1 to 142; TOM20, residues 1 to 144; TOM7, residues 1 to 55; TOM6, residues 1 to 64; and TOM5, residues 1 to 51) (Fig. 1A and fig. S1, A and B). Following induction of protein expression with 2% galactose, PINK1 pathway activity was determined by immunoblotting of whole-cell yeast extracts for endogenous ubiquitin phosphorylation or human PINK1 Ser228 (trans)autophosphorylation. In the yeast strain expressing human PINK1 alone, we did not observe substantial activation of PINK1 (Fig. 1B and fig. S2) consistent with previous studies showing that recombinant human PINK1 expressed in Escherichia coli or insect cells displays little catalytic activity (40) and that transiently transfected PINK1 alone in human cell lines exhibits low levels of activity under basal conditions (41). Notably, we observed that induced co-expression of WT human PINK1 together with all TOM complex subunits led to robust increase in ubiquitin phosphorylation (pS65 Ub) and PINK1 Ser228 autophosphorylation (pS228 PINK1) associated with a increase in the levels of PINK1 protein compared to expression of PINK1 alone (Fig. 1B and fig. S2, A to C). This was not observed when KI PINK1 was co-expressed with all TOM subunits (Fig. 1B and fig. S2, A to C). We further confirmed this in four independent yeast strains expressing WT PINK1 with or without TOM complex components (fig. S3). These data suggest that PINK1 stabilization at the TOM complex is sufficient for PINK1 activation. To validate the relevance of PINK1 activation in this reconstitution system, we next generated stable transformant yeast strains of PD-associated pathogenic mutations of PINK1 that have been previously characterized in mammalian cell–based assays following conditions of mitochondrial damage–induced membrane depolarization (13) including an NTE region mutant Q126P, adenosine 5′-triphosphate binding mutant E240K, insertion 3 substrate binding mutant G309D, and a CTE region mutant 534_535InsQ (fig. S4A). Consistent with previous studies in mammalian cells (13), we observed complete loss of phosphorylated ubiquitin in all mutants (fig. S4A). Further, we observed loss of Ser228 autophosphorylation except for the G309D mutant consistent with the impact of this mutant on substrate binding (fig. S4A). For all strains expressing WT or mutant PINK1-3FLAG, we observed two bands of PINK1 corresponding to full-length and N-terminal cleaved protein, suggesting that recombinant PINK1 in this system is being targeted to mitochondria (Fig. 1B; figs. S3 and S4A). To confirm this, we performed live cell imaging studies of PINK1-GFP, and this demonstrated co-localization of PINK1 with the mitochondrial marker red CMXRos (Fig. 1C). Immunoblotting analysis of mitochondrial fractions of yeast strains expressing PINK1-3FLAG and the TOM complex also revealed that PINK1 and all human TOM subunits tested were localized in the mitochondrial fraction (fig. S4B).Fig. 1.Co-expression of human PINK1 and TOM complex in yeast is sufficient for PINK1 activation.(A) Schematic of experimental workflow of PINK1 reconstitution in yeast. Image was created using BioRender.com. Yeast cells were transformed with all four plasmids carrying two plasmids each of the eight components for reconstitution. Cells were selected on a synthetic complete dropout plate, and positive clones were used for protein expression. After expression, cells were harvested and lysed, and the cell lysate was analyzed for protein expression. (B) Co-expression of human PINK1 and TOM complex subunits induces PINK1 activation. Stable yeast transformants were selected expressing wild-type (WT) or kinase-inactive (KI; D384A) full-length human PINK1-3FLAG and TOM5, TOM6, TOM7, TOM20, TOM22, TOM40, and TOM70 subunits (TOM complex) or WT human PINK1 alone. Expression was induced by supplementing the growth medium with 2% galactose. 20 μg of whole-cell lysates was run on 4 to 12% bis-tris gel and transferred onto nitrocellulose membrane followed by immunoblotting with anti-Ub pS65, anti-PINK1 pS228, anti-total PINK1, and other indicated antibodies. Data for three independent experiments are shown in fig. S14. (C) Localization of expressed human PINK1 to yeast mitochondria. Stable yeast transformants expressing WT full-length human PINK1-GFP and TOM5, TOM6, TOM7, TOM20, TOM22, TOM40, and TOM70 subunits (TOM complex) were generated, and mitochondria were stained by addition of 500 nM of MitoTracker CMXRos Red. Following incubation, cells were briefly spun down, washed twice with phosphate-buffered saline, and applied to concavalin A–coated coverslips that were placed on glass slides for immediate image acquisition using a Leica DMi8 microscope. Further processing was carried out in the Leica LAS X software platform that includes histogram adjustment and denoising with THUNDER. Images correspond to bright-field microscopy, and mitochondria were stained by MitoTracker (red) and PINK1-GFP (green).Role of TOM complex subunits on PINK1 activationWe next undertook systematic genetic analysis of the role of the TOM complex on PINK1 activation by generating stable transformant strains missing each of the seven subunits of the TOM complex co-expressed with WT human full-length PINK1-3FLAG (Fig. 2). Immunoblotting analysis of PINK1 activation for all strains revealed that the removal of TOM5 or TOM6 subunits did not significantly impair PINK1 activation as assessed by ubiquitin phosphorylation and PINK1 Ser228 autophosphorylation (Fig. 2, A and B, and figs. S5 and S6). Consistent with the previous reports of Youle (38, 39), we observed that removal of TOM7 largely abolished PINK1 activation and, furthermore, had a similar impact to removal of TOM40 or TOM22, and these findings were confirmed in four independent yeast strains for each genotype (Fig. 2, A and B, and figs. S5 and S6). The similarity of the defect of TOM7 to TOM22 is consistent with their critical roles in the assembly and maintenance of the core TOM40 pore-containing complex (33–35). We also observed a reduction in PINK1 activation following removal of the accessory receptor subunits TOM20 or TOM70, and the activation was further reduced in strains lacking both TOM20 and TOM70 (Fig. 2, A and B, and figs. S5 and S6). Overall, our findings indicate TOM subunits can be stratified into three groups based on their effect on PINK1 activation in this reconstitution system: group 1 (no effect), TOM5 and TOM6; group 2 (essential for core complex assembly), TOM7, TOM22, and TOM40; and group 3 (regulatory receptor binding role), TOM20 and TOM70.Fig. 2.Intact TOM complex required for optimal PINK1 activation.(A) Genetic analysis of role of TOM subunits on PINK1 activation. Stable yeast transformants were selected expressing WT or KI (D384A) full-length human PINK1-3FLAG and TOM5, TOM6, TOM7, TOM20, TOM22, TOM40, and TOM70 subunits (TOM complex or TOM complex minus indicated subunits). Expression was induced by supplementing the growth medium with 2% galactose. Whole-cell lysates (20 μg) were run on 4 to 12% bis-tris gel and transferred onto nitrocellulose membrane followed by immunoblotting with anti-Ub pS65, anti-PINK1 pS228, anti-total PINK1, and other indicated antibodies. Data representative of two independent experiments. (B) Summary quantification of Ub pS65 levels normalized to WT PINK1 + TOM complex. Data represents means ± SEM of five independent yeast clones.Structural model of PINK1-TOM complex predicts PINK1 interaction with TOM20TOM20 and TOM70 are not visible on high-resolution cryo-EM structures of the yeast and human TOM complex, indicating that these subunits are likely to be highly dynamic within the complex (33–35). To investigate how PINK1 stabilization at the TOM complex is mediated by TOM20 and TOM70, we used a locally installed ColabFold notebook (42) to run AlphaFold (43) structure predictions of the PINK1-TOM complex, imputing full-length human PINK1 and varying combinations of TOM subunit sequences in an iterative manner. Assisted Model Building with Energy Refinement (AMBER) structure relaxation (44) was used to ensure appropriate orientation of the side chains and to avoid steric clashes. We were unable to generate a high-confidence model of PINK1 and all subunits of the TOM complex together; however, five high-confidence models of a single molecule of PINK1, in complex with a TOM dimer containing TOM40, TOM22, TOM20, and TOM7 subunits, were generated based on interchain predicted alignment error (inter-PAE) (Fig. 3, A to C; and fig. S7, A to C; and movie S1). All models correctly predicted the structural interfaces of the core of the TOM complex formed by TOM40, TOM22, and TOM7 in line with existing cryo-EM structures (Fig. 3, A and B, and fig. S7, A and B).Fig. 3.Structural modeling of PINK1-TOM complex predicts direct interaction between PINK1 and TOM20.(A) Overall structure of AlphaFold prediction of PINK1-TOM complex (TOM7, green; TOM22, yellow/red; and TOM40, orange/gray) indicates direct PINK1 (pink) interaction with TOM20 (cyan). (B) Predicted aligned error (PAE) plot highlights predicted interaction between PINK1 and TOM20, marked by red boxes. N-terminal segment of PINK1 transverses one TOM40 pore [moderate confidence (blue boxes)], while other TOM components form high confident model as indicated. (C) A close-up view illustrates binding interface between PINK1 and TOM20, wherein the N-terminal extension (NTE) and C-terminal extension (CTE) regions of PINK1 interact with C-terminal region of TOM20 (α1 to α3 helices). Key interactions between conserved amino acids of PINK1 and TOM20 are indicated. (D and E) Mutational analysis in yeast cells confirms critical role of PINK1 NTE/CTE interaction with TOM20 for PINK1 activation. (D) Hydrophobic leucines on PINK1 at the interface were mutated to alanine (L532A, L539A, and L540A and L532A/L539A/L540A), cells expressing these mutants were grown on YP medium supplemented with 2% raffinose, and protein expression was induced by the addition of galactose. Whole-cell lysate (20 μg) was subjected to immunoblot analysis, phospho-ubiquitin was blotted as a readout of PINK1 activation, and PINK1 was detected by anti-total PINK1 antibody. (E) Residues on TOM20 making interactions with PINK1 were mutated to alanine (Q67A and E78A), cells expressing these mutants were grown on YP medium supplemented with 2% raffinose, and protein expression was induced by the addition of galactose. Whole-cell lysate (20 μg) was subjected to immunoblot analysis, phospho-ubiquitin was blotted as a readout of PINK1 activation, and PINK1 was detected by anti-total PINK1 antibody. The effect of these mutations on PINK1 activity was assayed and compared with the WT, KI PINK1, and the minus TOM20 cells.All five models predicted direct interaction between the NTE:CTE interface of PINK1 and residues within the C-terminal half of TOM20 (Fig. 3, A to C, and fig. S7, A and B). TOM20 is composed of an N-terminal transmembrane domain (TMD) anchoring it to the OMM and the C-terminal region (residues 50 to 145) that spans five α helices (α1 to α5) exposed to the cytosol where it binds mitochondrial precursor proteins via their MTS (fig. S8A). Several key conserved residues within the C terminus of TOM20, namely, Gln67, Leu71, Ile74, Glu78, Glu79, Phe70, and Val109, have been found to play a critical role in the recognition and binding of mitochondrial precursor proteins (fig. S8, A and B) (45, 46). Inspection of the AlphaFold PINK1-TOM complex model revealed a major hydrophobic interface between the αK helix of the CTE region of PINK1 and the hydrophobic patch formed by TOM20 α1 and α3 helices comprising multiple highly conserved residues including Leu532, Leu539, and Leu540 of PINK1 CTE and Phe70, Leu71, Ile74, Leu81, Leu110, and Leu114 of TOM20 (Fig. 3C and fig. S8, A to E). Furthermore, the NTE of PINK1 formed polar interactions with TOM20 including between the conserved Lys135 residue of PINK1 NTE and Gln78 located at the periphery of the α1 helix of TOM20 (Fig. 3C and fig. S8, A to E).To investigate the functional impact of mutations of these residues, we generated yeast strains in which we expressed PINK1 CTE mutants—L532A, L539A, and L540A—and a combinatorial L532A/L539A/L540A triple mutant (CTE 3A) together with all TOM complex subunits (Fig. 3D). Immunoblotting analysis of total PINK1 did not reveal any differences in PINK1 processing, suggesting that these mutations do not affect mitochondrial import (Fig. 3D). Single CTE mutants led to mild-to-moderate effects on PINK1 activation, but, notably, PINK1 activation was completely abolished in the CTE 3A mutant, and this was also associated with a reduction in PINK1 stabilization (Fig. 3D). In addition, the AlphaFold model predicted electrostatic and hydrogen bond interactions between the NTE region of PINK1 and TOM20 including a predicted salt bridge between the conserved Lys135 of PINK1 and Glu78 of TOM20 and predicted hydrogen bonding between Gln67 of TOM20 and PINK1 (Fig. 3C and fig. S8, A and B). We initially generated a yeast strain in which we expressed the PINK1 NTE mutants and K135E and K135M mutants in complex with all intact TOM complex subunits, and immunoblot analysis revealed reduction in ubiquitin phosphorylation consistent with reduced activation (fig. S9). We next generated strains of WT PINK1 co-expressed with all TOM complex subunits but in which we expressed TOM20 mutants Q67E, E78Q, or E78R (Fig. 3E). Immunoblot analysis revealed that these mutations in TOM20 led to reduced levels of ubiquitin phosphorylation to a similar degree as the minus TOM20 strain (Fig. 3E). Overall, these studies suggest that the PINK1 NTE:CTE interface promotes binding to TOM20.Structural modeling predicts PINK1 interaction with TOM70We further investigated how PINK1 activation is regulated by TOM70. TOM70 consists of multiple repeating units known as tetratricopeptide repeat (TPR) domains (fig. S10A) (47). The N-terminal TPRs form a loosely structured region called the N-terminal domain (NTD) pocket, which primarily interacts with heat shock proteins (fig. S10A) (48, 49). The C-terminal TPRs form the C-terminal domain (CTD) pocket that specifically binds to mitochondrial preproteins for import into the mitochondria (fig. S10A) (50). We used the ColabFold notebook (42) to run AlphaFold (43) structure predictions of the PINK1 with TOM70, imputing full-length human PINK1 and TOM70 sequences. AMBER structure relaxation was used to ensure appropriate orientation of the side chains to avoid steric clashes. Five models of a PINK1-TOM70 complex were generated from higher to lower confidence based on inter-PAE values (Fig. 4, A and B, and fig. S11, A and B). Three of the five models predicted a highly consistent binding interface between an N-terminal region of PINK1 (residues 71 to 106) and a region located within the CTD of TOM70 (Fig. 4C and fig. S11, A and B) with high PAE scores for interacting regions (fig. S11, A and B). Previous studies have suggested that PINK1 contains an internal MTS sequence in this region that acts redundantly with the canonical N-terminal MTS (39, 51). The Hermann lab previously elaborated an algorithm to predict internal MTS-like signals for TOM70 in the mature region (52). We imputed PINK1 into this algorithm (https://csb-imlp.bio.rptu.de), and this predicted several MTS-like regions of which the strongest peak mapped to the TOM70-binding region predicted by AlphaFold (fig. S11C). This predicted interface represents an extensive binding surface with multiple contacts involving hydrogen bonding, hydrophobic interactions, and electrostatic interactions. Electrostatic surface mapping visualization of TOM70 demonstrated that the CTD pocket was largely negatively charged and the corresponding N-terminal interacting surface region of PINK1 was positively charged (fig. S11D). Furthermore, a series of conserved positively charged residues in PINK1 (Arg83, Arg88, and Arg98) are predicted to form salt bridges with a series of conserved negatively charged residues within TOM70 (Asp488, Asp545, Glu549, and Asp229, respectively) (Fig. 4C). To test the predicted mode of binding model, we generated yeast strains expressing PINK1 mutations in the TOM70 interface (R83A/R88A/R98A triple mutant and R83E/R88E/R98E triple mutant). We observed that the combined triple mutation largely abolished ubiquitin phosphorylation (Fig. 4D). We next generated strains expressing TOM70 mutations within the PINK1-TOM70 binding interface, namely, D488A, D545A, and E549A, and all of these substantively lowered ubiquitin phosphorylation to a similar degree as the minus TOM70 strain (Fig. 4E and fig. S10, B to D). Overall, our data indicate that TOM70 is also required for optimal stabilization and activation of PINK1 at the TOM complex via an internal MTS-like region that we define as the TOM70 interacting region (TIR).Fig. 4.PINK1 binds TOM70 via N-terminal region preceding NTE.(A) AlphaFold model of PINK1 in complex with TOM70. PINK1 is colored in pink and TOM70 in purple. (B) PAE plot highlights the interaction between PINK1 and TOM70, indicated with red boxes. (C) Close-up view shows residues making direct interaction between the two proteins. (D and E) Mutational analysis in yeast cells confirms role of PINK1-TOM70 interaction for PINK1 activation. Cells carrying mutations and the corresponding controls were grown on YP medium supplemented with 2% raffinose, and protein expression was induced by the addition of galactose. Cells were harvested and processed, and phospho-ubiquitin was blotted as a readout of PINK1 activation with anti-total PINK1 antibody and indicated antibodies using Li-COR Odyssey CLx imaging system. (D) N-terminal residues on PINK1 at the interface were mutated (R83A, R88A, and R98A and R83A/R88A/R98A or R83E/R88E/R98E), cells expressing these mutants were grown on YP medium supplemented with 2% raffinose, and protein expression was induced by the addition of galactose. Whole-cell lysate (20 μg) was subjected to immunoblot analysis, phospho-ubiquitin was blotted as a readout of PINK1 activation, and PINK1 was detected by anti-total PINK1 antibody. The membranes were also blotted using the indicated antibodies. (E) Residues on TOM70 making interactions with PINK1 were mutated to alanine (D488A, D545A, and E549A), cells expressing these mutants were grown on YP medium supplemented with 2% raffinose, and protein expression was induced by the addition of galactose. Whole-cell lysate (20 μg) was subjected to immunoblot analysis, phospho-ubiquitin was blotted as a readout of PINK1 activation, and PINK1 was detected by anti-total PINK1 antibody. The membranes were also blotted using the indicated antibodies using Li-COR. The effect of these mutations on PINK1 activity was assayed and compared with the WT, KI PINK1, and the minus TOM70 cells.Validation of predicted PINK1-TOM interactions in mammalian cellsTo validate the PINK1-TOM20 predicted interaction in a mammalian cell system following mitochondrial depolarization, we generated stable cell lines in which we reintroduced full-length WT PINK1-3FLAG (WT); KI mutant PINK1 (KI); CTE PINK1 mutants, namely, L532A, L539A, and L540A; and a combined CTE 3A mutant into Flp-In T-REx HeLa PINK1-knockout cells [generated by exon 2-targeted CRISPR-Cas9 Research Resource IDentifier (RRID) CVCL_D5JI (13)]. To determine the effect of the selected PINK1 mutants on activation, cells were treated with dimethyl sulfoxide (DMSO) or 10 μM antimycin A/1 μM oligomycin (AO) for 3 hours to induce mitochondrial depolarization (Fig. 5A). Immunoblot analysis of whole-cell extracts revealed that CTE single-point mutants led to a slight reduction of phosphorylated ubiquitin upon mitochondrial depolarization of which L540A had a more noticeable effect; however, consistent with the analysis in yeast, phosphorylated ubiquitin was completely abolished in the CTE 3A mutant (Fig. 5B and fig. S12A). We next investigated the PINK1-TOM70 interaction and generated stable cell lines in which we reintroduced PINK1 TIR mutants, namely, R83A, R88A, and R98A, and combined R83A/R88A/R98A or R83E/R88E/R98E triple mutants into Flp-In T-REx HeLa PINK1-knockout cells (Fig. 5C and fig. S12B). Immunoblot analysis of whole-cell extracts revealed that TIR single-point mutants led to minimal reduction of phosphorylated ubiquitin upon mitochondrial depolarization; however, phosphorylated ubiquitin was most reduced in the TIR 3A or 3E mutant (Fig. 5C and fig. S12B). These data indicate that the PINK1:TOM interfaces identified in our yeast system are also functionally relevant in mammalian cells.Fig. 5.Optimal PINK1 activation requires interaction with TOM20 and TOM70 in mammalian cells following mitochondrial depolarization.(A) Schematic of workflow of analysis of PINK1 TOM-binding mutants in PINK1 knockout (KO) Flp-In Trex HeLa cells. The schematic image was made using BioRender. (B) PINK1 CTE TOM20-defective binding mutants lead to reduced PINK1 activation. Stably expressing PINK1-3FLAG WT, KI (D384A), and CTE mutant [L532A, L539A, and L540A and triple mutant (L532A/L539A/L540A)] cell lines were generated in PINK1-knockout Flp-In TRex HeLa cells. PINK1-3FLAG expression was induced by 24 hours of treatment with 0.2 μM doxycycline (Dox), and mitochondrial depolarization was induced by 3 hours of treatment with 10 μM antimycin A/1 μM oligomycin (AO) where indicated. Whole-cell lysates were subjected to immunoblotting with anti-PINK1, anti-Ub pS65 [Cell Signaling Technology (CST)], anti-OPA1 [BD Biosciences (BD)], and anti–glyceraldehyde-3-phosphate dehydrogenase (GAPDH) primary antibodies. Data representative of three independent experiments. (C) PINK1 TOM70-defective binding mutants lead to reduced PINK1 activation. Stably expressing PINK1-3FLAG WT, KI (D384A), and N-terminal mutant [R83A, R88A, and L98A and triple (R83A/R88A/L98A or R83E/R88E/R98E)] cell lines were generated in PINK1 knockout Flp-In TRex HeLa cells. PINK1-3FLAG expression was induced by 24 hours of treatment with 0.2 μM doxycycline, and mitochondrial depolarization was induced by 3 hours of treatment with AO where indicated. Whole-cell lysates were subjected to immunoblotting with anti-PINK1, anti-Ub pS65 (CST), anti-OPA1 (BD), and anti-GAPDH primary antibodies. Data representative of three independent experiments.We further investigated the role of TOM20 and TOM70 in stabilizing PINK1 at the TOM complex of mammalian cells following mitochondrial depolarization. We performed BN-PAGE analysis in HeLa cells stably expressing WT PINK1-3FLAG and observed two complexes (fig. S13) consistent with previous studies (13, 14, 26, 27). Immunoblotting analysis of these complexes indicated that TOM20 and TOM40 were present in the native ~500-kDa TOM complex and the ~700-kDa PINK1-TOM complex, whereas TOM70 was not detectable in either of the complexes (fig. S13, A to D). PINK1 resided in two HMW complexes, namely, the ~700-kDa PINK1-TOM complex and a complex of intermediate size between the ~500-kDa and ~700-kDa complexes as previously reported (fig. S13A) (13, 14, 26, 27). We next performed BN-PAGE analysis on stable cell lines in which we reintroduced full-length WT PINK1-3FLAG (WT), KI mutant PINK1 (KI), a combined CTE L532A/L539A/L540A mutant (TOM20-binding deficient), and a combined TIR R83E/R88E/R98E mutant (TOM70-binding deficient) into Flp-In T-REx HeLa PINK1-knockout cells (Fig. 6A). Notably, immunoblotting analysis with anti-TOM40 antibodies demonstrated that both the CTE L532A/L539A/L540A and TIR R83E/R88E/R98E triple mutants prevented the formation of the ~700-kDa PINK1-TOM complex following mitochondrial depolarization, and this was also confirmed by immunoblotting with anti-PINK1 antibody (Fig. 6B). Overall, these studies demonstrate that PINK1 recruitment, stabilization, and activation at the TOM complex requires interaction with TOM20 and TOM70 in mammalian cells following mitochondrial depolarization and supports our findings from the yeast reconstitution system.Fig. 6.PINK1 stabilization at the TOM complex is dependent on TOM20 and TOM70 interaction in mammalian cells upon mitochondrial depolarization.(A) Schematic of workflow of BN-PAGE analysis of PINK1 TOM-binding mutants in HeLa cells. (B) Mitochondrial enriched fractions were isolated from PINK1 knockout Flp-In TRex HeLa cells stably expressing PINK1-3FLAG WT, KI (D384A), and TOM20-defective binding mutant (L532A/L539A/L540A), and TOM70-defective binding mutant (R83E/R88E/R98E) mutants were treated with antimycin/oligomycin for 3 hours and then subjected to BN-PAGE and immunoblotted (IB) for anti-TOM40 or anti-PINK1 antibodies. Samples were also subjected to SDS-PAGE and immunoblotting with anti-TOM40 or anti-PINK1 antibodies, and total protein was visualized by Ponceau S staining.

Reconstitution of human PINK1 activation at the TOM complex in yeastWe initially generated stable transformant strains of the budding yeast, S. cerevisiae, expressing human WT or kinase-inactive (KI; D384A) full-length PINK1-3FLAG with or without all seven subunits of the human TOM complex (TOM70, residues 1 to 608; TOM40, residues 1 to 361; TOM22, residues 1 to 142; TOM20, residues 1 to 144; TOM7, residues 1 to 55; TOM6, residues 1 to 64; and TOM5, residues 1 to 51) (Fig. 1A and fig. S1, A and B). Following induction of protein expression with 2% galactose, PINK1 pathway activity was determined by immunoblotting of whole-cell yeast extracts for endogenous ubiquitin phosphorylation or human PINK1 Ser228 (trans)autophosphorylation. In the yeast strain expressing human PINK1 alone, we did not observe substantial activation of PINK1 (Fig. 1B and fig. S2) consistent with previous studies showing that recombinant human PINK1 expressed in Escherichia coli or insect cells displays little catalytic activity (40) and that transiently transfected PINK1 alone in human cell lines exhibits low levels of activity under basal conditions (41). Notably, we observed that induced co-expression of WT human PINK1 together with all TOM complex subunits led to robust increase in ubiquitin phosphorylation (pS65 Ub) and PINK1 Ser228 autophosphorylation (pS228 PINK1) associated with a increase in the levels of PINK1 protein compared to expression of PINK1 alone (Fig. 1B and fig. S2, A to C). This was not observed when KI PINK1 was co-expressed with all TOM subunits (Fig. 1B and fig. S2, A to C). We further confirmed this in four independent yeast strains expressing WT PINK1 with or without TOM complex components (fig. S3). These data suggest that PINK1 stabilization at the TOM complex is sufficient for PINK1 activation. To validate the relevance of PINK1 activation in this reconstitution system, we next generated stable transformant yeast strains of PD-associated pathogenic mutations of PINK1 that have been previously characterized in mammalian cell–based assays following conditions of mitochondrial damage–induced membrane depolarization (13) including an NTE region mutant Q126P, adenosine 5′-triphosphate binding mutant E240K, insertion 3 substrate binding mutant G309D, and a CTE region mutant 534_535InsQ (fig. S4A). Consistent with previous studies in mammalian cells (13), we observed complete loss of phosphorylated ubiquitin in all mutants (fig. S4A). Further, we observed loss of Ser228 autophosphorylation except for the G309D mutant consistent with the impact of this mutant on substrate binding (fig. S4A). For all strains expressing WT or mutant PINK1-3FLAG, we observed two bands of PINK1 corresponding to full-length and N-terminal cleaved protein, suggesting that recombinant PINK1 in this system is being targeted to mitochondria (Fig. 1B; figs. S3 and S4A). To confirm this, we performed live cell imaging studies of PINK1-GFP, and this demonstrated co-localization of PINK1 with the mitochondrial marker red CMXRos (Fig. 1C). Immunoblotting analysis of mitochondrial fractions of yeast strains expressing PINK1-3FLAG and the TOM complex also revealed that PINK1 and all human TOM subunits tested were localized in the mitochondrial fraction (fig. S4B).Fig. 1.Co-expression of human PINK1 and TOM complex in yeast is sufficient for PINK1 activation.(A) Schematic of experimental workflow of PINK1 reconstitution in yeast. Image was created using BioRender.com. Yeast cells were transformed with all four plasmids carrying two plasmids each of the eight components for reconstitution. Cells were selected on a synthetic complete dropout plate, and positive clones were used for protein expression. After expression, cells were harvested and lysed, and the cell lysate was analyzed for protein expression. (B) Co-expression of human PINK1 and TOM complex subunits induces PINK1 activation. Stable yeast transformants were selected expressing wild-type (WT) or kinase-inactive (KI; D384A) full-length human PINK1-3FLAG and TOM5, TOM6, TOM7, TOM20, TOM22, TOM40, and TOM70 subunits (TOM complex) or WT human PINK1 alone. Expression was induced by supplementing the growth medium with 2% galactose. 20 μg of whole-cell lysates was run on 4 to 12% bis-tris gel and transferred onto nitrocellulose membrane followed by immunoblotting with anti-Ub pS65, anti-PINK1 pS228, anti-total PINK1, and other indicated antibodies. Data for three independent experiments are shown in fig. S14. (C) Localization of expressed human PINK1 to yeast mitochondria. Stable yeast transformants expressing WT full-length human PINK1-GFP and TOM5, TOM6, TOM7, TOM20, TOM22, TOM40, and TOM70 subunits (TOM complex) were generated, and mitochondria were stained by addition of 500 nM of MitoTracker CMXRos Red. Following incubation, cells were briefly spun down, washed twice with phosphate-buffered saline, and applied to concavalin A–coated coverslips that were placed on glass slides for immediate image acquisition using a Leica DMi8 microscope. Further processing was carried out in the Leica LAS X software platform that includes histogram adjustment and denoising with THUNDER. Images correspond to bright-field microscopy, and mitochondria were stained by MitoTracker (red) and PINK1-GFP (green).

Reconstitution of human PINK1 activation at the TOM complex in yeast

We initially generated stable transformant strains of the budding yeast, S. cerevisiae, expressing human WT or kinase-inactive (KI; D384A) full-length PINK1-3FLAG with or without all seven subunits of the human TOM complex (TOM70, residues 1 to 608; TOM40, residues 1 to 361; TOM22, residues 1 to 142; TOM20, residues 1 to 144; TOM7, residues 1 to 55; TOM6, residues 1 to 64; and TOM5, residues 1 to 51) (Fig. 1A and fig. S1, A and B). Following induction of protein expression with 2% galactose, PINK1 pathway activity was determined by immunoblotting of whole-cell yeast extracts for endogenous ubiquitin phosphorylation or human PINK1 Ser228 (trans)autophosphorylation. In the yeast strain expressing human PINK1 alone, we did not observe substantial activation of PINK1 (Fig. 1B and fig. S2) consistent with previous studies showing that recombinant human PINK1 expressed in Escherichia coli or insect cells displays little catalytic activity (40) and that transiently transfected PINK1 alone in human cell lines exhibits low levels of activity under basal conditions (41). Notably, we observed that induced co-expression of WT human PINK1 together with all TOM complex subunits led to robust increase in ubiquitin phosphorylation (pS65 Ub) and PINK1 Ser228 autophosphorylation (pS228 PINK1) associated with a increase in the levels of PINK1 protein compared to expression of PINK1 alone (Fig. 1B and fig. S2, A to C). This was not observed when KI PINK1 was co-expressed with all TOM subunits (Fig. 1B and fig. S2, A to C). We further confirmed this in four independent yeast strains expressing WT PINK1 with or without TOM complex components (fig. S3). These data suggest that PINK1 stabilization at the TOM complex is sufficient for PINK1 activation. To validate the relevance of PINK1 activation in this reconstitution system, we next generated stable transformant yeast strains of PD-associated pathogenic mutations of PINK1 that have been previously characterized in mammalian cell–based assays following conditions of mitochondrial damage–induced membrane depolarization (13) including an NTE region mutant Q126P, adenosine 5′-triphosphate binding mutant E240K, insertion 3 substrate binding mutant G309D, and a CTE region mutant 534_535InsQ (fig. S4A). Consistent with previous studies in mammalian cells (13), we observed complete loss of phosphorylated ubiquitin in all mutants (fig. S4A). Further, we observed loss of Ser228 autophosphorylation except for the G309D mutant consistent with the impact of this mutant on substrate binding (fig. S4A). For all strains expressing WT or mutant PINK1-3FLAG, we observed two bands of PINK1 corresponding to full-length and N-terminal cleaved protein, suggesting that recombinant PINK1 in this system is being targeted to mitochondria (Fig. 1B; figs. S3 and S4A). To confirm this, we performed live cell imaging studies of PINK1-GFP, and this demonstrated co-localization of PINK1 with the mitochondrial marker red CMXRos (Fig. 1C). Immunoblotting analysis of mitochondrial fractions of yeast strains expressing PINK1-3FLAG and the TOM complex also revealed that PINK1 and all human TOM subunits tested were localized in the mitochondrial fraction (fig. S4B).

Co-expression of human PINK1 and TOM complex in yeast is sufficient for PINK1 activation.

(A) Schematic of experimental workflow of PINK1 reconstitution in yeast. Image was created using BioRender.com. Yeast cells were transformed with all four plasmids carrying two plasmids each of the eight components for reconstitution. Cells were selected on a synthetic complete dropout plate, and positive clones were used for protein expression. After expression, cells were harvested and lysed, and the cell lysate was analyzed for protein expression. (B) Co-expression of human PINK1 and TOM complex subunits induces PINK1 activation. Stable yeast transformants were selected expressing wild-type (WT) or kinase-inactive (KI; D384A) full-length human PINK1-3FLAG and TOM5, TOM6, TOM7, TOM20, TOM22, TOM40, and TOM70 subunits (TOM complex) or WT human PINK1 alone. Expression was induced by supplementing the growth medium with 2% galactose. 20 μg of whole-cell lysates was run on 4 to 12% bis-tris gel and transferred onto nitrocellulose membrane followed by immunoblotting with anti-Ub pS65, anti-PINK1 pS228, anti-total PINK1, and other indicated antibodies. Data for three independent experiments are shown in fig. S14. (C) Localization of expressed human PINK1 to yeast mitochondria. Stable yeast transformants expressing WT full-length human PINK1-GFP and TOM5, TOM6, TOM7, TOM20, TOM22, TOM40, and TOM70 subunits (TOM complex) were generated, and mitochondria were stained by addition of 500 nM of MitoTracker CMXRos Red. Following incubation, cells were briefly spun down, washed twice with phosphate-buffered saline, and applied to concavalin A–coated coverslips that were placed on glass slides for immediate image acquisition using a Leica DMi8 microscope. Further processing was carried out in the Leica LAS X software platform that includes histogram adjustment and denoising with THUNDER. Images correspond to bright-field microscopy, and mitochondria were stained by MitoTracker (red) and PINK1-GFP (green).

Role of TOM complex subunits on PINK1 activationWe next undertook systematic genetic analysis of the role of the TOM complex on PINK1 activation by generating stable transformant strains missing each of the seven subunits of the TOM complex co-expressed with WT human full-length PINK1-3FLAG (Fig. 2). Immunoblotting analysis of PINK1 activation for all strains revealed that the removal of TOM5 or TOM6 subunits did not significantly impair PINK1 activation as assessed by ubiquitin phosphorylation and PINK1 Ser228 autophosphorylation (Fig. 2, A and B, and figs. S5 and S6). Consistent with the previous reports of Youle (38, 39), we observed that removal of TOM7 largely abolished PINK1 activation and, furthermore, had a similar impact to removal of TOM40 or TOM22, and these findings were confirmed in four independent yeast strains for each genotype (Fig. 2, A and B, and figs. S5 and S6). The similarity of the defect of TOM7 to TOM22 is consistent with their critical roles in the assembly and maintenance of the core TOM40 pore-containing complex (33–35). We also observed a reduction in PINK1 activation following removal of the accessory receptor subunits TOM20 or TOM70, and the activation was further reduced in strains lacking both TOM20 and TOM70 (Fig. 2, A and B, and figs. S5 and S6). Overall, our findings indicate TOM subunits can be stratified into three groups based on their effect on PINK1 activation in this reconstitution system: group 1 (no effect), TOM5 and TOM6; group 2 (essential for core complex assembly), TOM7, TOM22, and TOM40; and group 3 (regulatory receptor binding role), TOM20 and TOM70.Fig. 2.Intact TOM complex required for optimal PINK1 activation.(A) Genetic analysis of role of TOM subunits on PINK1 activation. Stable yeast transformants were selected expressing WT or KI (D384A) full-length human PINK1-3FLAG and TOM5, TOM6, TOM7, TOM20, TOM22, TOM40, and TOM70 subunits (TOM complex or TOM complex minus indicated subunits). Expression was induced by supplementing the growth medium with 2% galactose. Whole-cell lysates (20 μg) were run on 4 to 12% bis-tris gel and transferred onto nitrocellulose membrane followed by immunoblotting with anti-Ub pS65, anti-PINK1 pS228, anti-total PINK1, and other indicated antibodies. Data representative of two independent experiments. (B) Summary quantification of Ub pS65 levels normalized to WT PINK1 + TOM complex. Data represents means ± SEM of five independent yeast clones.

Role of TOM complex subunits on PINK1 activation

We next undertook systematic genetic analysis of the role of the TOM complex on PINK1 activation by generating stable transformant strains missing each of the seven subunits of the TOM complex co-expressed with WT human full-length PINK1-3FLAG (Fig. 2). Immunoblotting analysis of PINK1 activation for all strains revealed that the removal of TOM5 or TOM6 subunits did not significantly impair PINK1 activation as assessed by ubiquitin phosphorylation and PINK1 Ser228 autophosphorylation (Fig. 2, A and B, and figs. S5 and S6). Consistent with the previous reports of Youle (38, 39), we observed that removal of TOM7 largely abolished PINK1 activation and, furthermore, had a similar impact to removal of TOM40 or TOM22, and these findings were confirmed in four independent yeast strains for each genotype (Fig. 2, A and B, and figs. S5 and S6). The similarity of the defect of TOM7 to TOM22 is consistent with their critical roles in the assembly and maintenance of the core TOM40 pore-containing complex (33–35). We also observed a reduction in PINK1 activation following removal of the accessory receptor subunits TOM20 or TOM70, and the activation was further reduced in strains lacking both TOM20 and TOM70 (Fig. 2, A and B, and figs. S5 and S6). Overall, our findings indicate TOM subunits can be stratified into three groups based on their effect on PINK1 activation in this reconstitution system: group 1 (no effect), TOM5 and TOM6; group 2 (essential for core complex assembly), TOM7, TOM22, and TOM40; and group 3 (regulatory receptor binding role), TOM20 and TOM70.

Intact TOM complex required for optimal PINK1 activation.

(A) Genetic analysis of role of TOM subunits on PINK1 activation. Stable yeast transformants were selected expressing WT or KI (D384A) full-length human PINK1-3FLAG and TOM5, TOM6, TOM7, TOM20, TOM22, TOM40, and TOM70 subunits (TOM complex or TOM complex minus indicated subunits). Expression was induced by supplementing the growth medium with 2% galactose. Whole-cell lysates (20 μg) were run on 4 to 12% bis-tris gel and transferred onto nitrocellulose membrane followed by immunoblotting with anti-Ub pS65, anti-PINK1 pS228, anti-total PINK1, and other indicated antibodies. Data representative of two independent experiments. (B) Summary quantification of Ub pS65 levels normalized to WT PINK1 + TOM complex. Data represents means ± SEM of five independent yeast clones.

Structural model of PINK1-TOM complex predicts PINK1 interaction with TOM20TOM20 and TOM70 are not visible on high-resolution cryo-EM structures of the yeast and human TOM complex, indicating that these subunits are likely to be highly dynamic within the complex (33–35). To investigate how PINK1 stabilization at the TOM complex is mediated by TOM20 and TOM70, we used a locally installed ColabFold notebook (42) to run AlphaFold (43) structure predictions of the PINK1-TOM complex, imputing full-length human PINK1 and varying combinations of TOM subunit sequences in an iterative manner. Assisted Model Building with Energy Refinement (AMBER) structure relaxation (44) was used to ensure appropriate orientation of the side chains and to avoid steric clashes. We were unable to generate a high-confidence model of PINK1 and all subunits of the TOM complex together; however, five high-confidence models of a single molecule of PINK1, in complex with a TOM dimer containing TOM40, TOM22, TOM20, and TOM7 subunits, were generated based on interchain predicted alignment error (inter-PAE) (Fig. 3, A to C; and fig. S7, A to C; and movie S1). All models correctly predicted the structural interfaces of the core of the TOM complex formed by TOM40, TOM22, and TOM7 in line with existing cryo-EM structures (Fig. 3, A and B, and fig. S7, A and B).Fig. 3.Structural modeling of PINK1-TOM complex predicts direct interaction between PINK1 and TOM20.(A) Overall structure of AlphaFold prediction of PINK1-TOM complex (TOM7, green; TOM22, yellow/red; and TOM40, orange/gray) indicates direct PINK1 (pink) interaction with TOM20 (cyan). (B) Predicted aligned error (PAE) plot highlights predicted interaction between PINK1 and TOM20, marked by red boxes. N-terminal segment of PINK1 transverses one TOM40 pore [moderate confidence (blue boxes)], while other TOM components form high confident model as indicated. (C) A close-up view illustrates binding interface between PINK1 and TOM20, wherein the N-terminal extension (NTE) and C-terminal extension (CTE) regions of PINK1 interact with C-terminal region of TOM20 (α1 to α3 helices). Key interactions between conserved amino acids of PINK1 and TOM20 are indicated. (D and E) Mutational analysis in yeast cells confirms critical role of PINK1 NTE/CTE interaction with TOM20 for PINK1 activation. (D) Hydrophobic leucines on PINK1 at the interface were mutated to alanine (L532A, L539A, and L540A and L532A/L539A/L540A), cells expressing these mutants were grown on YP medium supplemented with 2% raffinose, and protein expression was induced by the addition of galactose. Whole-cell lysate (20 μg) was subjected to immunoblot analysis, phospho-ubiquitin was blotted as a readout of PINK1 activation, and PINK1 was detected by anti-total PINK1 antibody. (E) Residues on TOM20 making interactions with PINK1 were mutated to alanine (Q67A and E78A), cells expressing these mutants were grown on YP medium supplemented with 2% raffinose, and protein expression was induced by the addition of galactose. Whole-cell lysate (20 μg) was subjected to immunoblot analysis, phospho-ubiquitin was blotted as a readout of PINK1 activation, and PINK1 was detected by anti-total PINK1 antibody. The effect of these mutations on PINK1 activity was assayed and compared with the WT, KI PINK1, and the minus TOM20 cells.All five models predicted direct interaction between the NTE:CTE interface of PINK1 and residues within the C-terminal half of TOM20 (Fig. 3, A to C, and fig. S7, A and B). TOM20 is composed of an N-terminal transmembrane domain (TMD) anchoring it to the OMM and the C-terminal region (residues 50 to 145) that spans five α helices (α1 to α5) exposed to the cytosol where it binds mitochondrial precursor proteins via their MTS (fig. S8A). Several key conserved residues within the C terminus of TOM20, namely, Gln67, Leu71, Ile74, Glu78, Glu79, Phe70, and Val109, have been found to play a critical role in the recognition and binding of mitochondrial precursor proteins (fig. S8, A and B) (45, 46). Inspection of the AlphaFold PINK1-TOM complex model revealed a major hydrophobic interface between the αK helix of the CTE region of PINK1 and the hydrophobic patch formed by TOM20 α1 and α3 helices comprising multiple highly conserved residues including Leu532, Leu539, and Leu540 of PINK1 CTE and Phe70, Leu71, Ile74, Leu81, Leu110, and Leu114 of TOM20 (Fig. 3C and fig. S8, A to E). Furthermore, the NTE of PINK1 formed polar interactions with TOM20 including between the conserved Lys135 residue of PINK1 NTE and Gln78 located at the periphery of the α1 helix of TOM20 (Fig. 3C and fig. S8, A to E).To investigate the functional impact of mutations of these residues, we generated yeast strains in which we expressed PINK1 CTE mutants—L532A, L539A, and L540A—and a combinatorial L532A/L539A/L540A triple mutant (CTE 3A) together with all TOM complex subunits (Fig. 3D). Immunoblotting analysis of total PINK1 did not reveal any differences in PINK1 processing, suggesting that these mutations do not affect mitochondrial import (Fig. 3D). Single CTE mutants led to mild-to-moderate effects on PINK1 activation, but, notably, PINK1 activation was completely abolished in the CTE 3A mutant, and this was also associated with a reduction in PINK1 stabilization (Fig. 3D). In addition, the AlphaFold model predicted electrostatic and hydrogen bond interactions between the NTE region of PINK1 and TOM20 including a predicted salt bridge between the conserved Lys135 of PINK1 and Glu78 of TOM20 and predicted hydrogen bonding between Gln67 of TOM20 and PINK1 (Fig. 3C and fig. S8, A and B). We initially generated a yeast strain in which we expressed the PINK1 NTE mutants and K135E and K135M mutants in complex with all intact TOM complex subunits, and immunoblot analysis revealed reduction in ubiquitin phosphorylation consistent with reduced activation (fig. S9). We next generated strains of WT PINK1 co-expressed with all TOM complex subunits but in which we expressed TOM20 mutants Q67E, E78Q, or E78R (Fig. 3E). Immunoblot analysis revealed that these mutations in TOM20 led to reduced levels of ubiquitin phosphorylation to a similar degree as the minus TOM20 strain (Fig. 3E). Overall, these studies suggest that the PINK1 NTE:CTE interface promotes binding to TOM20.

Structural model of PINK1-TOM complex predicts PINK1 interaction with TOM20

TOM20 and TOM70 are not visible on high-resolution cryo-EM structures of the yeast and human TOM complex, indicating that these subunits are likely to be highly dynamic within the complex (33–35). To investigate how PINK1 stabilization at the TOM complex is mediated by TOM20 and TOM70, we used a locally installed ColabFold notebook (42) to run AlphaFold (43) structure predictions of the PINK1-TOM complex, imputing full-length human PINK1 and varying combinations of TOM subunit sequences in an iterative manner. Assisted Model Building with Energy Refinement (AMBER) structure relaxation (44) was used to ensure appropriate orientation of the side chains and to avoid steric clashes. We were unable to generate a high-confidence model of PINK1 and all subunits of the TOM complex together; however, five high-confidence models of a single molecule of PINK1, in complex with a TOM dimer containing TOM40, TOM22, TOM20, and TOM7 subunits, were generated based on interchain predicted alignment error (inter-PAE) (Fig. 3, A to C; and fig. S7, A to C; and movie S1). All models correctly predicted the structural interfaces of the core of the TOM complex formed by TOM40, TOM22, and TOM7 in line with existing cryo-EM structures (Fig. 3, A and B, and fig. S7, A and B).

Structural modeling of PINK1-TOM complex predicts direct interaction between PINK1 and TOM20.

(A) Overall structure of AlphaFold prediction of PINK1-TOM complex (TOM7, green; TOM22, yellow/red; and TOM40, orange/gray) indicates direct PINK1 (pink) interaction with TOM20 (cyan). (B) Predicted aligned error (PAE) plot highlights predicted interaction between PINK1 and TOM20, marked by red boxes. N-terminal segment of PINK1 transverses one TOM40 pore [moderate confidence (blue boxes)], while other TOM components form high confident model as indicated. (C) A close-up view illustrates binding interface between PINK1 and TOM20, wherein the N-terminal extension (NTE) and C-terminal extension (CTE) regions of PINK1 interact with C-terminal region of TOM20 (α1 to α3 helices). Key interactions between conserved amino acids of PINK1 and TOM20 are indicated. (D and E) Mutational analysis in yeast cells confirms critical role of PINK1 NTE/CTE interaction with TOM20 for PINK1 activation. (D) Hydrophobic leucines on PINK1 at the interface were mutated to alanine (L532A, L539A, and L540A and L532A/L539A/L540A), cells expressing these mutants were grown on YP medium supplemented with 2% raffinose, and protein expression was induced by the addition of galactose. Whole-cell lysate (20 μg) was subjected to immunoblot analysis, phospho-ubiquitin was blotted as a readout of PINK1 activation, and PINK1 was detected by anti-total PINK1 antibody. (E) Residues on TOM20 making interactions with PINK1 were mutated to alanine (Q67A and E78A), cells expressing these mutants were grown on YP medium supplemented with 2% raffinose, and protein expression was induced by the addition of galactose. Whole-cell lysate (20 μg) was subjected to immunoblot analysis, phospho-ubiquitin was blotted as a readout of PINK1 activation, and PINK1 was detected by anti-total PINK1 antibody. The effect of these mutations on PINK1 activity was assayed and compared with the WT, KI PINK1, and the minus TOM20 cells.

All five models predicted direct interaction between the NTE:CTE interface of PINK1 and residues within the C-terminal half of TOM20 (Fig. 3, A to C, and fig. S7, A and B). TOM20 is composed of an N-terminal transmembrane domain (TMD) anchoring it to the OMM and the C-terminal region (residues 50 to 145) that spans five α helices (α1 to α5) exposed to the cytosol where it binds mitochondrial precursor proteins via their MTS (fig. S8A). Several key conserved residues within the C terminus of TOM20, namely, Gln67, Leu71, Ile74, Glu78, Glu79, Phe70, and Val109, have been found to play a critical role in the recognition and binding of mitochondrial precursor proteins (fig. S8, A and B) (45, 46). Inspection of the AlphaFold PINK1-TOM complex model revealed a major hydrophobic interface between the αK helix of the CTE region of PINK1 and the hydrophobic patch formed by TOM20 α1 and α3 helices comprising multiple highly conserved residues including Leu532, Leu539, and Leu540 of PINK1 CTE and Phe70, Leu71, Ile74, Leu81, Leu110, and Leu114 of TOM20 (Fig. 3C and fig. S8, A to E). Furthermore, the NTE of PINK1 formed polar interactions with TOM20 including between the conserved Lys135 residue of PINK1 NTE and Gln78 located at the periphery of the α1 helix of TOM20 (Fig. 3C and fig. S8, A to E).

To investigate the functional impact of mutations of these residues, we generated yeast strains in which we expressed PINK1 CTE mutants—L532A, L539A, and L540A—and a combinatorial L532A/L539A/L540A triple mutant (CTE 3A) together with all TOM complex subunits (Fig. 3D). Immunoblotting analysis of total PINK1 did not reveal any differences in PINK1 processing, suggesting that these mutations do not affect mitochondrial import (Fig. 3D). Single CTE mutants led to mild-to-moderate effects on PINK1 activation, but, notably, PINK1 activation was completely abolished in the CTE 3A mutant, and this was also associated with a reduction in PINK1 stabilization (Fig. 3D). In addition, the AlphaFold model predicted electrostatic and hydrogen bond interactions between the NTE region of PINK1 and TOM20 including a predicted salt bridge between the conserved Lys135 of PINK1 and Glu78 of TOM20 and predicted hydrogen bonding between Gln67 of TOM20 and PINK1 (Fig. 3C and fig. S8, A and B). We initially generated a yeast strain in which we expressed the PINK1 NTE mutants and K135E and K135M mutants in complex with all intact TOM complex subunits, and immunoblot analysis revealed reduction in ubiquitin phosphorylation consistent with reduced activation (fig. S9). We next generated strains of WT PINK1 co-expressed with all TOM complex subunits but in which we expressed TOM20 mutants Q67E, E78Q, or E78R (Fig. 3E). Immunoblot analysis revealed that these mutations in TOM20 led to reduced levels of ubiquitin phosphorylation to a similar degree as the minus TOM20 strain (Fig. 3E). Overall, these studies suggest that the PINK1 NTE:CTE interface promotes binding to TOM20.

Structural modeling predicts PINK1 interaction with TOM70We further investigated how PINK1 activation is regulated by TOM70. TOM70 consists of multiple repeating units known as tetratricopeptide repeat (TPR) domains (fig. S10A) (47). The N-terminal TPRs form a loosely structured region called the N-terminal domain (NTD) pocket, which primarily interacts with heat shock proteins (fig. S10A) (48, 49). The C-terminal TPRs form the C-terminal domain (CTD) pocket that specifically binds to mitochondrial preproteins for import into the mitochondria (fig. S10A) (50). We used the ColabFold notebook (42) to run AlphaFold (43) structure predictions of the PINK1 with TOM70, imputing full-length human PINK1 and TOM70 sequences. AMBER structure relaxation was used to ensure appropriate orientation of the side chains to avoid steric clashes. Five models of a PINK1-TOM70 complex were generated from higher to lower confidence based on inter-PAE values (Fig. 4, A and B, and fig. S11, A and B). Three of the five models predicted a highly consistent binding interface between an N-terminal region of PINK1 (residues 71 to 106) and a region located within the CTD of TOM70 (Fig. 4C and fig. S11, A and B) with high PAE scores for interacting regions (fig. S11, A and B). Previous studies have suggested that PINK1 contains an internal MTS sequence in this region that acts redundantly with the canonical N-terminal MTS (39, 51). The Hermann lab previously elaborated an algorithm to predict internal MTS-like signals for TOM70 in the mature region (52). We imputed PINK1 into this algorithm (https://csb-imlp.bio.rptu.de), and this predicted several MTS-like regions of which the strongest peak mapped to the TOM70-binding region predicted by AlphaFold (fig. S11C). This predicted interface represents an extensive binding surface with multiple contacts involving hydrogen bonding, hydrophobic interactions, and electrostatic interactions. Electrostatic surface mapping visualization of TOM70 demonstrated that the CTD pocket was largely negatively charged and the corresponding N-terminal interacting surface region of PINK1 was positively charged (fig. S11D). Furthermore, a series of conserved positively charged residues in PINK1 (Arg83, Arg88, and Arg98) are predicted to form salt bridges with a series of conserved negatively charged residues within TOM70 (Asp488, Asp545, Glu549, and Asp229, respectively) (Fig. 4C). To test the predicted mode of binding model, we generated yeast strains expressing PINK1 mutations in the TOM70 interface (R83A/R88A/R98A triple mutant and R83E/R88E/R98E triple mutant). We observed that the combined triple mutation largely abolished ubiquitin phosphorylation (Fig. 4D). We next generated strains expressing TOM70 mutations within the PINK1-TOM70 binding interface, namely, D488A, D545A, and E549A, and all of these substantively lowered ubiquitin phosphorylation to a similar degree as the minus TOM70 strain (Fig. 4E and fig. S10, B to D). Overall, our data indicate that TOM70 is also required for optimal stabilization and activation of PINK1 at the TOM complex via an internal MTS-like region that we define as the TOM70 interacting region (TIR).Fig. 4.PINK1 binds TOM70 via N-terminal region preceding NTE.(A) AlphaFold model of PINK1 in complex with TOM70. PINK1 is colored in pink and TOM70 in purple. (B) PAE plot highlights the interaction between PINK1 and TOM70, indicated with red boxes. (C) Close-up view shows residues making direct interaction between the two proteins. (D and E) Mutational analysis in yeast cells confirms role of PINK1-TOM70 interaction for PINK1 activation. Cells carrying mutations and the corresponding controls were grown on YP medium supplemented with 2% raffinose, and protein expression was induced by the addition of galactose. Cells were harvested and processed, and phospho-ubiquitin was blotted as a readout of PINK1 activation with anti-total PINK1 antibody and indicated antibodies using Li-COR Odyssey CLx imaging system. (D) N-terminal residues on PINK1 at the interface were mutated (R83A, R88A, and R98A and R83A/R88A/R98A or R83E/R88E/R98E), cells expressing these mutants were grown on YP medium supplemented with 2% raffinose, and protein expression was induced by the addition of galactose. Whole-cell lysate (20 μg) was subjected to immunoblot analysis, phospho-ubiquitin was blotted as a readout of PINK1 activation, and PINK1 was detected by anti-total PINK1 antibody. The membranes were also blotted using the indicated antibodies. (E) Residues on TOM70 making interactions with PINK1 were mutated to alanine (D488A, D545A, and E549A), cells expressing these mutants were grown on YP medium supplemented with 2% raffinose, and protein expression was induced by the addition of galactose. Whole-cell lysate (20 μg) was subjected to immunoblot analysis, phospho-ubiquitin was blotted as a readout of PINK1 activation, and PINK1 was detected by anti-total PINK1 antibody. The membranes were also blotted using the indicated antibodies using Li-COR. The effect of these mutations on PINK1 activity was assayed and compared with the WT, KI PINK1, and the minus TOM70 cells.

Structural modeling predicts PINK1 interaction with TOM70

We further investigated how PINK1 activation is regulated by TOM70. TOM70 consists of multiple repeating units known as tetratricopeptide repeat (TPR) domains (fig. S10A) (47). The N-terminal TPRs form a loosely structured region called the N-terminal domain (NTD) pocket, which primarily interacts with heat shock proteins (fig. S10A) (48, 49). The C-terminal TPRs form the C-terminal domain (CTD) pocket that specifically binds to mitochondrial preproteins for import into the mitochondria (fig. S10A) (50). We used the ColabFold notebook (42) to run AlphaFold (43) structure predictions of the PINK1 with TOM70, imputing full-length human PINK1 and TOM70 sequences. AMBER structure relaxation was used to ensure appropriate orientation of the side chains to avoid steric clashes. Five models of a PINK1-TOM70 complex were generated from higher to lower confidence based on inter-PAE values (Fig. 4, A and B, and fig. S11, A and B). Three of the five models predicted a highly consistent binding interface between an N-terminal region of PINK1 (residues 71 to 106) and a region located within the CTD of TOM70 (Fig. 4C and fig. S11, A and B) with high PAE scores for interacting regions (fig. S11, A and B). Previous studies have suggested that PINK1 contains an internal MTS sequence in this region that acts redundantly with the canonical N-terminal MTS (39, 51). The Hermann lab previously elaborated an algorithm to predict internal MTS-like signals for TOM70 in the mature region (52). We imputed PINK1 into this algorithm (https://csb-imlp.bio.rptu.de), and this predicted several MTS-like regions of which the strongest peak mapped to the TOM70-binding region predicted by AlphaFold (fig. S11C). This predicted interface represents an extensive binding surface with multiple contacts involving hydrogen bonding, hydrophobic interactions, and electrostatic interactions. Electrostatic surface mapping visualization of TOM70 demonstrated that the CTD pocket was largely negatively charged and the corresponding N-terminal interacting surface region of PINK1 was positively charged (fig. S11D). Furthermore, a series of conserved positively charged residues in PINK1 (Arg83, Arg88, and Arg98) are predicted to form salt bridges with a series of conserved negatively charged residues within TOM70 (Asp488, Asp545, Glu549, and Asp229, respectively) (Fig. 4C). To test the predicted mode of binding model, we generated yeast strains expressing PINK1 mutations in the TOM70 interface (R83A/R88A/R98A triple mutant and R83E/R88E/R98E triple mutant). We observed that the combined triple mutation largely abolished ubiquitin phosphorylation (Fig. 4D). We next generated strains expressing TOM70 mutations within the PINK1-TOM70 binding interface, namely, D488A, D545A, and E549A, and all of these substantively lowered ubiquitin phosphorylation to a similar degree as the minus TOM70 strain (Fig. 4E and fig. S10, B to D). Overall, our data indicate that TOM70 is also required for optimal stabilization and activation of PINK1 at the TOM complex via an internal MTS-like region that we define as the TOM70 interacting region (TIR).

PINK1 binds TOM70 via N-terminal region preceding NTE.

(A) AlphaFold model of PINK1 in complex with TOM70. PINK1 is colored in pink and TOM70 in purple. (B) PAE plot highlights the interaction between PINK1 and TOM70, indicated with red boxes. (C) Close-up view shows residues making direct interaction between the two proteins. (D and E) Mutational analysis in yeast cells confirms role of PINK1-TOM70 interaction for PINK1 activation. Cells carrying mutations and the corresponding controls were grown on YP medium supplemented with 2% raffinose, and protein expression was induced by the addition of galactose. Cells were harvested and processed, and phospho-ubiquitin was blotted as a readout of PINK1 activation with anti-total PINK1 antibody and indicated antibodies using Li-COR Odyssey CLx imaging system. (D) N-terminal residues on PINK1 at the interface were mutated (R83A, R88A, and R98A and R83A/R88A/R98A or R83E/R88E/R98E), cells expressing these mutants were grown on YP medium supplemented with 2% raffinose, and protein expression was induced by the addition of galactose. Whole-cell lysate (20 μg) was subjected to immunoblot analysis, phospho-ubiquitin was blotted as a readout of PINK1 activation, and PINK1 was detected by anti-total PINK1 antibody. The membranes were also blotted using the indicated antibodies. (E) Residues on TOM70 making interactions with PINK1 were mutated to alanine (D488A, D545A, and E549A), cells expressing these mutants were grown on YP medium supplemented with 2% raffinose, and protein expression was induced by the addition of galactose. Whole-cell lysate (20 μg) was subjected to immunoblot analysis, phospho-ubiquitin was blotted as a readout of PINK1 activation, and PINK1 was detected by anti-total PINK1 antibody. The membranes were also blotted using the indicated antibodies using Li-COR. The effect of these mutations on PINK1 activity was assayed and compared with the WT, KI PINK1, and the minus TOM70 cells.

Validation of predicted PINK1-TOM interactions in mammalian cellsTo validate the PINK1-TOM20 predicted interaction in a mammalian cell system following mitochondrial depolarization, we generated stable cell lines in which we reintroduced full-length WT PINK1-3FLAG (WT); KI mutant PINK1 (KI); CTE PINK1 mutants, namely, L532A, L539A, and L540A; and a combined CTE 3A mutant into Flp-In T-REx HeLa PINK1-knockout cells [generated by exon 2-targeted CRISPR-Cas9 Research Resource IDentifier (RRID) CVCL_D5JI (13)]. To determine the effect of the selected PINK1 mutants on activation, cells were treated with dimethyl sulfoxide (DMSO) or 10 μM antimycin A/1 μM oligomycin (AO) for 3 hours to induce mitochondrial depolarization (Fig. 5A). Immunoblot analysis of whole-cell extracts revealed that CTE single-point mutants led to a slight reduction of phosphorylated ubiquitin upon mitochondrial depolarization of which L540A had a more noticeable effect; however, consistent with the analysis in yeast, phosphorylated ubiquitin was completely abolished in the CTE 3A mutant (Fig. 5B and fig. S12A). We next investigated the PINK1-TOM70 interaction and generated stable cell lines in which we reintroduced PINK1 TIR mutants, namely, R83A, R88A, and R98A, and combined R83A/R88A/R98A or R83E/R88E/R98E triple mutants into Flp-In T-REx HeLa PINK1-knockout cells (Fig. 5C and fig. S12B). Immunoblot analysis of whole-cell extracts revealed that TIR single-point mutants led to minimal reduction of phosphorylated ubiquitin upon mitochondrial depolarization; however, phosphorylated ubiquitin was most reduced in the TIR 3A or 3E mutant (Fig. 5C and fig. S12B). These data indicate that the PINK1:TOM interfaces identified in our yeast system are also functionally relevant in mammalian cells.Fig. 5.Optimal PINK1 activation requires interaction with TOM20 and TOM70 in mammalian cells following mitochondrial depolarization.(A) Schematic of workflow of analysis of PINK1 TOM-binding mutants in PINK1 knockout (KO) Flp-In Trex HeLa cells. The schematic image was made using BioRender. (B) PINK1 CTE TOM20-defective binding mutants lead to reduced PINK1 activation. Stably expressing PINK1-3FLAG WT, KI (D384A), and CTE mutant [L532A, L539A, and L540A and triple mutant (L532A/L539A/L540A)] cell lines were generated in PINK1-knockout Flp-In TRex HeLa cells. PINK1-3FLAG expression was induced by 24 hours of treatment with 0.2 μM doxycycline (Dox), and mitochondrial depolarization was induced by 3 hours of treatment with 10 μM antimycin A/1 μM oligomycin (AO) where indicated. Whole-cell lysates were subjected to immunoblotting with anti-PINK1, anti-Ub pS65 [Cell Signaling Technology (CST)], anti-OPA1 [BD Biosciences (BD)], and anti–glyceraldehyde-3-phosphate dehydrogenase (GAPDH) primary antibodies. Data representative of three independent experiments. (C) PINK1 TOM70-defective binding mutants lead to reduced PINK1 activation. Stably expressing PINK1-3FLAG WT, KI (D384A), and N-terminal mutant [R83A, R88A, and L98A and triple (R83A/R88A/L98A or R83E/R88E/R98E)] cell lines were generated in PINK1 knockout Flp-In TRex HeLa cells. PINK1-3FLAG expression was induced by 24 hours of treatment with 0.2 μM doxycycline, and mitochondrial depolarization was induced by 3 hours of treatment with AO where indicated. Whole-cell lysates were subjected to immunoblotting with anti-PINK1, anti-Ub pS65 (CST), anti-OPA1 (BD), and anti-GAPDH primary antibodies. Data representative of three independent experiments.We further investigated the role of TOM20 and TOM70 in stabilizing PINK1 at the TOM complex of mammalian cells following mitochondrial depolarization. We performed BN-PAGE analysis in HeLa cells stably expressing WT PINK1-3FLAG and observed two complexes (fig. S13) consistent with previous studies (13, 14, 26, 27). Immunoblotting analysis of these complexes indicated that TOM20 and TOM40 were present in the native ~500-kDa TOM complex and the ~700-kDa PINK1-TOM complex, whereas TOM70 was not detectable in either of the complexes (fig. S13, A to D). PINK1 resided in two HMW complexes, namely, the ~700-kDa PINK1-TOM complex and a complex of intermediate size between the ~500-kDa and ~700-kDa complexes as previously reported (fig. S13A) (13, 14, 26, 27). We next performed BN-PAGE analysis on stable cell lines in which we reintroduced full-length WT PINK1-3FLAG (WT), KI mutant PINK1 (KI), a combined CTE L532A/L539A/L540A mutant (TOM20-binding deficient), and a combined TIR R83E/R88E/R98E mutant (TOM70-binding deficient) into Flp-In T-REx HeLa PINK1-knockout cells (Fig. 6A). Notably, immunoblotting analysis with anti-TOM40 antibodies demonstrated that both the CTE L532A/L539A/L540A and TIR R83E/R88E/R98E triple mutants prevented the formation of the ~700-kDa PINK1-TOM complex following mitochondrial depolarization, and this was also confirmed by immunoblotting with anti-PINK1 antibody (Fig. 6B). Overall, these studies demonstrate that PINK1 recruitment, stabilization, and activation at the TOM complex requires interaction with TOM20 and TOM70 in mammalian cells following mitochondrial depolarization and supports our findings from the yeast reconstitution system.Fig. 6.PINK1 stabilization at the TOM complex is dependent on TOM20 and TOM70 interaction in mammalian cells upon mitochondrial depolarization.(A) Schematic of workflow of BN-PAGE analysis of PINK1 TOM-binding mutants in HeLa cells. (B) Mitochondrial enriched fractions were isolated from PINK1 knockout Flp-In TRex HeLa cells stably expressing PINK1-3FLAG WT, KI (D384A), and TOM20-defective binding mutant (L532A/L539A/L540A), and TOM70-defective binding mutant (R83E/R88E/R98E) mutants were treated with antimycin/oligomycin for 3 hours and then subjected to BN-PAGE and immunoblotted (IB) for anti-TOM40 or anti-PINK1 antibodies. Samples were also subjected to SDS-PAGE and immunoblotting with anti-TOM40 or anti-PINK1 antibodies, and total protein was visualized by Ponceau S staining.

Validation of predicted PINK1-TOM interactions in mammalian cells

To validate the PINK1-TOM20 predicted interaction in a mammalian cell system following mitochondrial depolarization, we generated stable cell lines in which we reintroduced full-length WT PINK1-3FLAG (WT); KI mutant PINK1 (KI); CTE PINK1 mutants, namely, L532A, L539A, and L540A; and a combined CTE 3A mutant into Flp-In T-REx HeLa PINK1-knockout cells [generated by exon 2-targeted CRISPR-Cas9 Research Resource IDentifier (RRID) CVCL_D5JI (13)]. To determine the effect of the selected PINK1 mutants on activation, cells were treated with dimethyl sulfoxide (DMSO) or 10 μM antimycin A/1 μM oligomycin (AO) for 3 hours to induce mitochondrial depolarization (Fig. 5A). Immunoblot analysis of whole-cell extracts revealed that CTE single-point mutants led to a slight reduction of phosphorylated ubiquitin upon mitochondrial depolarization of which L540A had a more noticeable effect; however, consistent with the analysis in yeast, phosphorylated ubiquitin was completely abolished in the CTE 3A mutant (Fig. 5B and fig. S12A). We next investigated the PINK1-TOM70 interaction and generated stable cell lines in which we reintroduced PINK1 TIR mutants, namely, R83A, R88A, and R98A, and combined R83A/R88A/R98A or R83E/R88E/R98E triple mutants into Flp-In T-REx HeLa PINK1-knockout cells (Fig. 5C and fig. S12B). Immunoblot analysis of whole-cell extracts revealed that TIR single-point mutants led to minimal reduction of phosphorylated ubiquitin upon mitochondrial depolarization; however, phosphorylated ubiquitin was most reduced in the TIR 3A or 3E mutant (Fig. 5C and fig. S12B). These data indicate that the PINK1:TOM interfaces identified in our yeast system are also functionally relevant in mammalian cells.

Optimal PINK1 activation requires interaction with TOM20 and TOM70 in mammalian cells following mitochondrial depolarization.

(A) Schematic of workflow of analysis of PINK1 TOM-binding mutants in PINK1 knockout (KO) Flp-In Trex HeLa cells. The schematic image was made using BioRender. (B) PINK1 CTE TOM20-defective binding mutants lead to reduced PINK1 activation. Stably expressing PINK1-3FLAG WT, KI (D384A), and CTE mutant [L532A, L539A, and L540A and triple mutant (L532A/L539A/L540A)] cell lines were generated in PINK1-knockout Flp-In TRex HeLa cells. PINK1-3FLAG expression was induced by 24 hours of treatment with 0.2 μM doxycycline (Dox), and mitochondrial depolarization was induced by 3 hours of treatment with 10 μM antimycin A/1 μM oligomycin (AO) where indicated. Whole-cell lysates were subjected to immunoblotting with anti-PINK1, anti-Ub pS65 [Cell Signaling Technology (CST)], anti-OPA1 [BD Biosciences (BD)], and anti–glyceraldehyde-3-phosphate dehydrogenase (GAPDH) primary antibodies. Data representative of three independent experiments. (C) PINK1 TOM70-defective binding mutants lead to reduced PINK1 activation. Stably expressing PINK1-3FLAG WT, KI (D384A), and N-terminal mutant [R83A, R88A, and L98A and triple (R83A/R88A/L98A or R83E/R88E/R98E)] cell lines were generated in PINK1 knockout Flp-In TRex HeLa cells. PINK1-3FLAG expression was induced by 24 hours of treatment with 0.2 μM doxycycline, and mitochondrial depolarization was induced by 3 hours of treatment with AO where indicated. Whole-cell lysates were subjected to immunoblotting with anti-PINK1, anti-Ub pS65 (CST), anti-OPA1 (BD), and anti-GAPDH primary antibodies. Data representative of three independent experiments.

We further investigated the role of TOM20 and TOM70 in stabilizing PINK1 at the TOM complex of mammalian cells following mitochondrial depolarization. We performed BN-PAGE analysis in HeLa cells stably expressing WT PINK1-3FLAG and observed two complexes (fig. S13) consistent with previous studies (13, 14, 26, 27). Immunoblotting analysis of these complexes indicated that TOM20 and TOM40 were present in the native ~500-kDa TOM complex and the ~700-kDa PINK1-TOM complex, whereas TOM70 was not detectable in either of the complexes (fig. S13, A to D). PINK1 resided in two HMW complexes, namely, the ~700-kDa PINK1-TOM complex and a complex of intermediate size between the ~500-kDa and ~700-kDa complexes as previously reported (fig. S13A) (13, 14, 26, 27). We next performed BN-PAGE analysis on stable cell lines in which we reintroduced full-length WT PINK1-3FLAG (WT), KI mutant PINK1 (KI), a combined CTE L532A/L539A/L540A mutant (TOM20-binding deficient), and a combined TIR R83E/R88E/R98E mutant (TOM70-binding deficient) into Flp-In T-REx HeLa PINK1-knockout cells (Fig. 6A). Notably, immunoblotting analysis with anti-TOM40 antibodies demonstrated that both the CTE L532A/L539A/L540A and TIR R83E/R88E/R98E triple mutants prevented the formation of the ~700-kDa PINK1-TOM complex following mitochondrial depolarization, and this was also confirmed by immunoblotting with anti-PINK1 antibody (Fig. 6B). Overall, these studies demonstrate that PINK1 recruitment, stabilization, and activation at the TOM complex requires interaction with TOM20 and TOM70 in mammalian cells following mitochondrial depolarization and supports our findings from the yeast reconstitution system.

PINK1 stabilization at the TOM complex is dependent on TOM20 and TOM70 interaction in mammalian cells upon mitochondrial depolarization.

(A) Schematic of workflow of BN-PAGE analysis of PINK1 TOM-binding mutants in HeLa cells. (B) Mitochondrial enriched fractions were isolated from PINK1 knockout Flp-In TRex HeLa cells stably expressing PINK1-3FLAG WT, KI (D384A), and TOM20-defective binding mutant (L532A/L539A/L540A), and TOM70-defective binding mutant (R83E/R88E/R98E) mutants were treated with antimycin/oligomycin for 3 hours and then subjected to BN-PAGE and immunoblotted (IB) for anti-TOM40 or anti-PINK1 antibodies. Samples were also subjected to SDS-PAGE and immunoblotting with anti-TOM40 or anti-PINK1 antibodies, and total protein was visualized by Ponceau S staining.

DISCUSSIONOur yeast reconstitution system enables genetic dissection of the contribution of human TOM complex subunits toward human PINK1 activation. These findings highlight the essential requirement of the TOM40 pore-containing core complex as well as a role for the TOM20 and TOM70 receptor subunits for optimal PINK1 activation (Figs. 2 to 7).Fig. 7.Schematic models of role of TOM70 and TOM20 interaction with PINK1 mediating stabilization and activation at the TOM complex at sites of damaged mitochondria.(A) Interaction of the PINK1 TIR region with the CTD pocket of TOM70 occurs concurrently with interaction of the PINK1 NTE:CTE interface with the C-terminal α1 to α3 helices of TOM20 and is required for PINK1 stabilization at the TOM complex. Image was created using BioRender.com. (B) Sequential binding of PINK1 TIR region to CTD pocket of TOM70 followed by PINK1 NTE:CTE interface with the C-terminal α1 to α3 helices of TOM20 is required for PINK1 stabilization at the TOM complex. Image was created using BioRender.com.TOM20 typically binds mitochondrial proteins via N-terminal MTS consisting of 15 to 40 amino acids (45, 46, 53–57) that form an amphipathic helix (58). Previous analysis of the PINK1 MTS sequence (amino acids 1 to 34) showed that it is sufficient for mitochondrial targeting to the matrix in polarized mitochondria of mammalian cells (59–61). Following mitochondrial depolarization, the stabilization of PINK1 at the TOM complex is distinct compared to other MTS-containing mitochondrial proteins that are generally released into the cytosol and not stabilized at the TOM complex. A role for TOM20 in PINK1 stabilization upon mitochondrial depolarization was first reported using cross-linking/immunoprecipitation studies of PINK1 under denaturing conditions and showed that only TOM20 could be co-isolated with PINK1 under these conditions, suggesting a direct interaction (26). However, the mechanism remained elusive because full-length PINK1 lacking the MTS could still be imported to mitochondria and stabilized and activated on depolarized mitochondria (39, 51). We and others recently defined the NTE:CTE interface as being critical for PINK1 recruitment to the TOM complex (12–14), and our new findings support a model whereby the NTE:CTE interface of active PINK1 binds to the C-terminal cytosolic domain of TOM20 (Figs. 2, 3, and 7). While this manuscript was under review, another study also reported the interaction of PINK1 with TOM20 (62). The cytosolic domain of TOM20 has previously been shown to have a diverse selectivity for client mitochondrial proteins mediated by a hydrophobic MTS sequence motif that is extremely broad (φχχφφ; φ is hydrophobic acid and χ is any other amino acid) (45). Structural studies have mapped the TOM20 peptide binding site to a hydrophobic pocket within its cytosolic domain [e.g., (45)]. Similar to MTS presequences, the binding of PINK1 CTE to TOM20 is majorly directed by hydrophobic leucine residues although the sequence of the PINK1 hydrophobic patch (amino acid 532–LQQSAATLL–amino acid 540) displays notable differences from the general MTS sequence motif (fig. S8E). A further notable difference is that polar interactions between TOM20 and PINK1 are directed by residues located within the distinct NTE α helix, whereas these typically occur at the periphery of MTS presequences (Fig. 3C) (45). Overall, the interaction between PINK1 and TOM20 that we have identified further expands the diversity of TOM20 client interactions (Figs. 3 and 7).In future work, it will be interesting to understand how TOM20 switches from binding the PINK1 MTS under polarized mitochondria conditions to the NTE:CTE interface in depolarized mitochondria. This is likely to be due to the flexibility of the cytosolic domain of TOM20 that renders it highly dynamic and accounts for it not being visible on previous high-resolution cryo-EM structures of the TOM complex (33–35). Recently, low-resolution cryo-EM structures of the TOM complex in N. crassa have revealed multiple sub-complexes containing TOM20, TOM22, and TOM40 in different stoichiometries including one TOM20 in a dimeric TOM40 complex (36). This suggests that TOM20 can adopt myriad conformations and interactions within the overall complex (36). AlphaFold modeling predicted an asymmetric PINK1-TOM complex with one molecule of PINK1 bound to one molecule of TOM20 within a dimeric TOM40 complex (Fig. 3); however, in view of the flexibility of TOM20, we cannot rule out additional subcomplexes of PINK1 and TOM20 with the rest of the TOM components with varying stoichiometries (Fig. 3). Furthermore, the N. crassa structure revealed a strong interaction of the acidic patch of the N terminus of TOM22 with a positively charged patch in cytosolic TOM20 helix, suggesting that TOM20 requires to dock with TOM22 for optimal receptor function (36), which was also not observed in the AlphaFold model.A previous study of PINK1 had mapped an outer mitochondrial localization signal (OMS) (residues 70 to 94) at its N terminus located between the MTS and TMD (51). This region is dispensable for mitochondrial import but is required for PINK1 stabilization and activation at the TOM complex on depolarized mitochondria in a TOM40-dependent manner (39, 51). TOM70 has been reported to recognize internal signals in hydrophobic regions and consistent with this was shown to be required for PINK1 import (63); however, a role for TOM70 in PINK1 activation in depolarized mitochondria was lacking because BN-PAGE studies reported that TOM70 was not detectable in the ~700-kDa PINK1-TOM complex (26, 27). It was recently reported that TOM70 acts as the major receptor for PINK1 import and stabilization in depolarized mitochondria based on small interfering RNA knockdown studies of TOM subunits in mammalian cells (49). Our genetic dissection of TOM subunits in the reconstitution assay provides a clearcut role of TOM70 for PINK1 activation (Figs. 2 and 4) and we have mapped a TIR that contains overlapping residues with the previously elaborated OMS region (Fig. 4). AlphaFold modeling predicted binding of the N-terminal region of PINK1 to the C-terminal cytosolic domain of TOM70 (Fig. 4). Our results contrast with a previous study that used peptide mapping studies to define the TOM70 binding site on PINK1 and suggested that TOM70 can interact with the N-terminal MTS, OMS, or TMD regions with equivalent affinity (49). To date, there is no experimental structure of mammalian TOM70, but crystal structures of yeast TOM70 revealed that it contains 26 α helices that are involved in the formation of 11 TPR motifs (64). The NTD functions to bind chaperones including Hsp90 in mammals (48, 64, 65). Binding sites for selected mitochondrial preproteins have been mapped to the C terminus (α helices 8 to 26) although, to date, a general receptor site/motif has yet to be defined (50). The crystal structure of yeast TOM70 has revealed a highly conserved groove located in the center of the CTD, which has been attributed to be the major binding site for mitochondrial preproteins (TPR 4 to 11) (64). The distal side of the binding groove is made of hydrophobic and polar residues, whereas the proximal side contains three highly conserved negatively charged residues Glu473, Glu542, and Glu577 (64). AlphaFold modeling suggests that PINK1 forms specific interactions with the proximal residues and has interactions with the distal side that may aid in interaction by acting as docking site as it has been suggested for other preproteins (50). Previous studies have mapped numerous Parkin-mediated ubiquitylation sites in the TOM70 C terminus (K473, K504, K539, K573, and K607) (66–68); however, these are located on the outside of the C terminus and not likely to affect the binding surface (fig. S10). Similarly, regulatory phosphorylation sites on the C terminus of TOM70 have been reported (69), and it will be interesting to assess the role of these modifications, as well as chaperone binding to the N terminus, on the interaction between PINK1 and TOM70 under depolarized mitochondrial conditions.In contrast to a previous study (49), our studies unambiguously show that both TOM70 and TOM20 are important for PINK1 stabilization and activation at the TOM complex (Fig. 7). AlphaFold was unable to predict a high-confidence model with both TOM70 and TOM20 in the PINK1-TOM complex, suggesting that PINK1 interactions with these subunits are likely to be dynamic, and this is compounded by dynamic interactions of the cytosolic domains of TOM20 and TOM70 with other subunits of the TOM complex if bound to PINK1 simultaneously. We propose a model that PINK1 is engaged by both TOM20 and TOM70 for recruitment and stabilization at the TOM complex (Fig. 7A); however, we cannot exclude that the association of PINK1 with TOM70 and TOM20 subunits occurs in a sequential manner and because TOM70 is not detectable in the PINK1-TOM complex on BN-PAGE (fig. S13D), suggesting that it may bind to PINK1 earlier to direct it to the TOM20 subunit and core complex (Fig. 7B).Our reconstitution analysis confirms the essential role of the TOM7 subunit for PINK1 activation as first revealed by the Youle lab (38, 39). AlphaFold modeling did not predict a direct interaction between TOM7 and PINK1 and instead was in line with previous cryo-EM analysis showing that the three small TOM subunits—5, 6, and 7—are peripherally bound to TOM40 via distinct interactions (33–35). We did not observe much effect of loss of TOM5 or TOM6 on activation, whereas loss of TOM7 largely abolished PINK1 activation akin to loss of TOM22 that is critical for stabilizing the TOM40 dimeric pore structure (Fig. 2). While the role of the small TOMs in the mammalian TOM complex is still to be fully elucidated (30), our findings are in keeping with previous analysis that have shown a critical role for TOM7 (but not TOM5 and TOM6) in maintaining stability of the mammalian TOM40 core complex (37).An unexpected and notable finding from our studies was that co-expression of human PINK1 with TOM subunits was sufficient for its activation (Fig. 1B and fig. S2). Over the past decade, PINK1 activation has mainly been studied in the context of mitophagic signaling following mitochondrial depolarization, induced by mitochondrial uncouplers [e.g., AO or carbonyl cyanide m-chlorophenyl hydrazone (CCCP)]; mitochondrial matrix misfolding stress, triggered by matrix Hsp90 inhibitors; or overexpression of the deletion mutant of OTC (3–5). Because all lead to PINK1 activation, our reconstitution results suggest that, in the context of mitochondrial damage, the key event for PINK1 activation is its stabilization at the TOM complex. Previous studies have shown weak activation of PINK1 when it is transiently overexpressed in mammalian cells without any mitochondrial damage (41), and we speculate that the low activity is due to the sub-stoichiometric levels of the TOMs compared to overexpressed PINK1 under these conditions. Thus, it would be interesting to assess whether this activity is increased if PINK1 is co-transfected with exogenous human TOM subunits. Similarly, we detected weak activation of full-length human PINK1 when expressed alone in yeast or in yeast strains in which human TOM20 or TOM70 were mutated to prevent PINK1 interaction (Figs. 3E and 4E), and this may be due to interaction with the yeast TOM20 and/or TOM70 because the PINK1 interaction sites are conserved (figs. S8B and S10B). Previous studies have reported that recombinant expression of catalytic domain-containing fragments of human PINK1 in E. coli or insect cells displays no significant catalytic activity associated with low expression yields and unstable protein (40). In contrast, recombinantly expressed catalytic domain-containing fragments of insect orthologs of PINK1 (e.g., Pediculus humanus corporis) are active with high expression yields of monodispersible protein (40). In our hands, we have been unable to identify any N-terminal or C-terminal truncated His-tagged human PINK1 constructs, out of >30 tested, that, when expressed in insect cells, lead to high yields of soluble human PINK1 as detected by colloidal Coomassie gels (fig. S14, A to D). Furthermore, replacement of the Ins1 loop of human PINK1 with the orthologous region of Pediculus PINK1 or the pseudokinase PEAK1 (fig. S14, A and B) (70) did not lead to enhanced expression of soluble protein (fig. S14D). Collectively, our findings would suggest that recombinant co-expression of TOM subunits with human PINK1 in insect cells is needed to generate high yields of stable PINK1 and reconstitute PINK1 activation in vitro.In future work, experimental structural data are required to better understand PINK1 regulation within the TOM complex, which, although challenging due to transient association of TOM20 and TOM70 as well as PINK1, may provide additional molecular insights into PINK1 activation including dimerization, which was recently shown for insect PINK1 and how this mediates Ser228 transphosphorylation (12, 14). In addition to Ser228, we recently identified Ser167 autophosphorylation as being important for activation of human PINK1, which is not conserved in insect PINK1 (71). Furthermore, several critical residues on PINK1 NTE/CTE and TIR that bind TOM20 (fig. S8, D and E) and TOM70 (fig. S10D), respectively, are also not conserved in insect PINK1 orthologs, suggesting that there will be important differences in active human PINK1 from previously solved insect PINK1 structures. Overall, our current analysis provides insights into human PINK1 activation that will be of utility in the development of small-molecule activators as a therapeutic strategy against PD.

Our yeast reconstitution system enables genetic dissection of the contribution of human TOM complex subunits toward human PINK1 activation. These findings highlight the essential requirement of the TOM40 pore-containing core complex as well as a role for the TOM20 and TOM70 receptor subunits for optimal PINK1 activation (Figs. 2 to 7).

Schematic models of role of TOM70 and TOM20 interaction with PINK1 mediating stabilization and activation at the TOM complex at sites of damaged mitochondria.

(A) Interaction of the PINK1 TIR region with the CTD pocket of TOM70 occurs concurrently with interaction of the PINK1 NTE:CTE interface with the C-terminal α1 to α3 helices of TOM20 and is required for PINK1 stabilization at the TOM complex. Image was created using BioRender.com. (B) Sequential binding of PINK1 TIR region to CTD pocket of TOM70 followed by PINK1 NTE:CTE interface with the C-terminal α1 to α3 helices of TOM20 is required for PINK1 stabilization at the TOM complex. Image was created using BioRender.com.

TOM20 typically binds mitochondrial proteins via N-terminal MTS consisting of 15 to 40 amino acids (45, 46, 53–57) that form an amphipathic helix (58). Previous analysis of the PINK1 MTS sequence (amino acids 1 to 34) showed that it is sufficient for mitochondrial targeting to the matrix in polarized mitochondria of mammalian cells (59–61). Following mitochondrial depolarization, the stabilization of PINK1 at the TOM complex is distinct compared to other MTS-containing mitochondrial proteins that are generally released into the cytosol and not stabilized at the TOM complex. A role for TOM20 in PINK1 stabilization upon mitochondrial depolarization was first reported using cross-linking/immunoprecipitation studies of PINK1 under denaturing conditions and showed that only TOM20 could be co-isolated with PINK1 under these conditions, suggesting a direct interaction (26). However, the mechanism remained elusive because full-length PINK1 lacking the MTS could still be imported to mitochondria and stabilized and activated on depolarized mitochondria (39, 51). We and others recently defined the NTE:CTE interface as being critical for PINK1 recruitment to the TOM complex (12–14), and our new findings support a model whereby the NTE:CTE interface of active PINK1 binds to the C-terminal cytosolic domain of TOM20 (Figs. 2, 3, and 7). While this manuscript was under review, another study also reported the interaction of PINK1 with TOM20 (62). The cytosolic domain of TOM20 has previously been shown to have a diverse selectivity for client mitochondrial proteins mediated by a hydrophobic MTS sequence motif that is extremely broad (φχχφφ; φ is hydrophobic acid and χ is any other amino acid) (45). Structural studies have mapped the TOM20 peptide binding site to a hydrophobic pocket within its cytosolic domain [e.g., (45)]. Similar to MTS presequences, the binding of PINK1 CTE to TOM20 is majorly directed by hydrophobic leucine residues although the sequence of the PINK1 hydrophobic patch (amino acid 532–LQQSAATLL–amino acid 540) displays notable differences from the general MTS sequence motif (fig. S8E). A further notable difference is that polar interactions between TOM20 and PINK1 are directed by residues located within the distinct NTE α helix, whereas these typically occur at the periphery of MTS presequences (Fig. 3C) (45). Overall, the interaction between PINK1 and TOM20 that we have identified further expands the diversity of TOM20 client interactions (Figs. 3 and 7).

In future work, it will be interesting to understand how TOM20 switches from binding the PINK1 MTS under polarized mitochondria conditions to the NTE:CTE interface in depolarized mitochondria. This is likely to be due to the flexibility of the cytosolic domain of TOM20 that renders it highly dynamic and accounts for it not being visible on previous high-resolution cryo-EM structures of the TOM complex (33–35). Recently, low-resolution cryo-EM structures of the TOM complex in N. crassa have revealed multiple sub-complexes containing TOM20, TOM22, and TOM40 in different stoichiometries including one TOM20 in a dimeric TOM40 complex (36). This suggests that TOM20 can adopt myriad conformations and interactions within the overall complex (36). AlphaFold modeling predicted an asymmetric PINK1-TOM complex with one molecule of PINK1 bound to one molecule of TOM20 within a dimeric TOM40 complex (Fig. 3); however, in view of the flexibility of TOM20, we cannot rule out additional subcomplexes of PINK1 and TOM20 with the rest of the TOM components with varying stoichiometries (Fig. 3). Furthermore, the N. crassa structure revealed a strong interaction of the acidic patch of the N terminus of TOM22 with a positively charged patch in cytosolic TOM20 helix, suggesting that TOM20 requires to dock with TOM22 for optimal receptor function (36), which was also not observed in the AlphaFold model.

A previous study of PINK1 had mapped an outer mitochondrial localization signal (OMS) (residues 70 to 94) at its N terminus located between the MTS and TMD (51). This region is dispensable for mitochondrial import but is required for PINK1 stabilization and activation at the TOM complex on depolarized mitochondria in a TOM40-dependent manner (39, 51). TOM70 has been reported to recognize internal signals in hydrophobic regions and consistent with this was shown to be required for PINK1 import (63); however, a role for TOM70 in PINK1 activation in depolarized mitochondria was lacking because BN-PAGE studies reported that TOM70 was not detectable in the ~700-kDa PINK1-TOM complex (26, 27). It was recently reported that TOM70 acts as the major receptor for PINK1 import and stabilization in depolarized mitochondria based on small interfering RNA knockdown studies of TOM subunits in mammalian cells (49). Our genetic dissection of TOM subunits in the reconstitution assay provides a clearcut role of TOM70 for PINK1 activation (Figs. 2 and 4) and we have mapped a TIR that contains overlapping residues with the previously elaborated OMS region (Fig. 4). AlphaFold modeling predicted binding of the N-terminal region of PINK1 to the C-terminal cytosolic domain of TOM70 (Fig. 4). Our results contrast with a previous study that used peptide mapping studies to define the TOM70 binding site on PINK1 and suggested that TOM70 can interact with the N-terminal MTS, OMS, or TMD regions with equivalent affinity (49). To date, there is no experimental structure of mammalian TOM70, but crystal structures of yeast TOM70 revealed that it contains 26 α helices that are involved in the formation of 11 TPR motifs (64). The NTD functions to bind chaperones including Hsp90 in mammals (48, 64, 65). Binding sites for selected mitochondrial preproteins have been mapped to the C terminus (α helices 8 to 26) although, to date, a general receptor site/motif has yet to be defined (50). The crystal structure of yeast TOM70 has revealed a highly conserved groove located in the center of the CTD, which has been attributed to be the major binding site for mitochondrial preproteins (TPR 4 to 11) (64). The distal side of the binding groove is made of hydrophobic and polar residues, whereas the proximal side contains three highly conserved negatively charged residues Glu473, Glu542, and Glu577 (64). AlphaFold modeling suggests that PINK1 forms specific interactions with the proximal residues and has interactions with the distal side that may aid in interaction by acting as docking site as it has been suggested for other preproteins (50). Previous studies have mapped numerous Parkin-mediated ubiquitylation sites in the TOM70 C terminus (K473, K504, K539, K573, and K607) (66–68); however, these are located on the outside of the C terminus and not likely to affect the binding surface (fig. S10). Similarly, regulatory phosphorylation sites on the C terminus of TOM70 have been reported (69), and it will be interesting to assess the role of these modifications, as well as chaperone binding to the N terminus, on the interaction between PINK1 and TOM70 under depolarized mitochondrial conditions.

In contrast to a previous study (49), our studies unambiguously show that both TOM70 and TOM20 are important for PINK1 stabilization and activation at the TOM complex (Fig. 7). AlphaFold was unable to predict a high-confidence model with both TOM70 and TOM20 in the PINK1-TOM complex, suggesting that PINK1 interactions with these subunits are likely to be dynamic, and this is compounded by dynamic interactions of the cytosolic domains of TOM20 and TOM70 with other subunits of the TOM complex if bound to PINK1 simultaneously. We propose a model that PINK1 is engaged by both TOM20 and TOM70 for recruitment and stabilization at the TOM complex (Fig. 7A); however, we cannot exclude that the association of PINK1 with TOM70 and TOM20 subunits occurs in a sequential manner and because TOM70 is not detectable in the PINK1-TOM complex on BN-PAGE (fig. S13D), suggesting that it may bind to PINK1 earlier to direct it to the TOM20 subunit and core complex (Fig. 7B).

Our reconstitution analysis confirms the essential role of the TOM7 subunit for PINK1 activation as first revealed by the Youle lab (38, 39). AlphaFold modeling did not predict a direct interaction between TOM7 and PINK1 and instead was in line with previous cryo-EM analysis showing that the three small TOM subunits—5, 6, and 7—are peripherally bound to TOM40 via distinct interactions (33–35). We did not observe much effect of loss of TOM5 or TOM6 on activation, whereas loss of TOM7 largely abolished PINK1 activation akin to loss of TOM22 that is critical for stabilizing the TOM40 dimeric pore structure (Fig. 2). While the role of the small TOMs in the mammalian TOM complex is still to be fully elucidated (30), our findings are in keeping with previous analysis that have shown a critical role for TOM7 (but not TOM5 and TOM6) in maintaining stability of the mammalian TOM40 core complex (37).

An unexpected and notable finding from our studies was that co-expression of human PINK1 with TOM subunits was sufficient for its activation (Fig. 1B and fig. S2). Over the past decade, PINK1 activation has mainly been studied in the context of mitophagic signaling following mitochondrial depolarization, induced by mitochondrial uncouplers [e.g., AO or carbonyl cyanide m-chlorophenyl hydrazone (CCCP)]; mitochondrial matrix misfolding stress, triggered by matrix Hsp90 inhibitors; or overexpression of the deletion mutant of OTC (3–5). Because all lead to PINK1 activation, our reconstitution results suggest that, in the context of mitochondrial damage, the key event for PINK1 activation is its stabilization at the TOM complex. Previous studies have shown weak activation of PINK1 when it is transiently overexpressed in mammalian cells without any mitochondrial damage (41), and we speculate that the low activity is due to the sub-stoichiometric levels of the TOMs compared to overexpressed PINK1 under these conditions. Thus, it would be interesting to assess whether this activity is increased if PINK1 is co-transfected with exogenous human TOM subunits. Similarly, we detected weak activation of full-length human PINK1 when expressed alone in yeast or in yeast strains in which human TOM20 or TOM70 were mutated to prevent PINK1 interaction (Figs. 3E and 4E), and this may be due to interaction with the yeast TOM20 and/or TOM70 because the PINK1 interaction sites are conserved (figs. S8B and S10B). Previous studies have reported that recombinant expression of catalytic domain-containing fragments of human PINK1 in E. coli or insect cells displays no significant catalytic activity associated with low expression yields and unstable protein (40). In contrast, recombinantly expressed catalytic domain-containing fragments of insect orthologs of PINK1 (e.g., Pediculus humanus corporis) are active with high expression yields of monodispersible protein (40). In our hands, we have been unable to identify any N-terminal or C-terminal truncated His-tagged human PINK1 constructs, out of >30 tested, that, when expressed in insect cells, lead to high yields of soluble human PINK1 as detected by colloidal Coomassie gels (fig. S14, A to D). Furthermore, replacement of the Ins1 loop of human PINK1 with the orthologous region of Pediculus PINK1 or the pseudokinase PEAK1 (fig. S14, A and B) (70) did not lead to enhanced expression of soluble protein (fig. S14D). Collectively, our findings would suggest that recombinant co-expression of TOM subunits with human PINK1 in insect cells is needed to generate high yields of stable PINK1 and reconstitute PINK1 activation in vitro.

In future work, experimental structural data are required to better understand PINK1 regulation within the TOM complex, which, although challenging due to transient association of TOM20 and TOM70 as well as PINK1, may provide additional molecular insights into PINK1 activation including dimerization, which was recently shown for insect PINK1 and how this mediates Ser228 transphosphorylation (12, 14). In addition to Ser228, we recently identified Ser167 autophosphorylation as being important for activation of human PINK1, which is not conserved in insect PINK1 (71). Furthermore, several critical residues on PINK1 NTE/CTE and TIR that bind TOM20 (fig. S8, D and E) and TOM70 (fig. S10D), respectively, are also not conserved in insect PINK1 orthologs, suggesting that there will be important differences in active human PINK1 from previously solved insect PINK1 structures. Overall, our current analysis provides insights into human PINK1 activation that will be of utility in the development of small-molecule activators as a therapeutic strategy against PD.

MATERIALS AND METHODSMolecular biology and cloningFor mammalian expression, PINK1-3FLAG constructs were cloned into pcDNA5 vectors for recombination into Flp-In TRex HeLa PINK1 knockout cell lines (RRID:CVCL_D5JI) (13). For the yeast strains, codon-optimized plasmids to express human PINK1-3FLAG, TOM70, TOM40, TOM22, TOM20, TOM7, TOM6, and TOM5 were generated, using previously described methods (72). Human genes were paired into four genetically modified yeast vectors (pORs), based on the pRS vector series that enables efficient shuttling of vectors in specific yeast strains for facile manipulation (fig. S1) (73). yOR1 strain harbors human PINK1-3FLAG + TOM40; yOR2 strain harbors PINK1-3FLAG + TOM40 + TOM22 + TOM7; yOR3 strain harbors PINK1-3FLAG + TOM40 + TOM22 + TOM7 + TOM70 + TOM20; and yOR4 strain harbors PINK1-3FLAG + TOM40 + TOM22 + TOM7 + TOM70 + TOM20 + TOM5 + TOM6 (table S1). The paired genes were cloned on either side of the GAL1_10 promoter that allows the bidirectional induction of both genes upon addition of galactose to the growth medium. In addition, we made a yeast strain (yOR5) expressing just PINK1 alone, and this was done by transforming yeast cells with plasmid (pOR1) containing just PINK1-3FLAG next to the Gal 1_10 promoter without TOM40. A full list of yeast strains and what plasmids are expressed is listed in table S1. Site-directed mutagenesis was carried out using the Quik-Change method with Thermococcus kodakaraensis (KOD) polymerase (Novagen). All yeast and mammalian constructs (table S2) were verified by The Sequencing Services (School of Life Sciences, University of Dundee) and are now available to request via the Medical Research Council Protein Phosphorylation and Ubiquitylation Unit (MRC PPU) Reagents and Services website (https://mrcppureagents.dundee.ac.uk/). For cloning of hPINK1 constructs for insect cell expression, the coding sequences for the PINK1 constructs were polymerase chain reaction (PCR)–amplified using clone OHu25380D (GenScript) as a template and cloned into the vector pFB-6HZB [Structural Genomics Consortium (SGC); Addgene ID: 218680] as previously described (74). Expression from this vector yields proteins with a TEV protease-cleavable N-terminal His6-Z tag.Antibodies for biochemical analysisThe following antibodies were used in this study: α–glyceraldehyde-3-phosphate dehydrogenase (Santa Cruz Biotechnology, cat. no. sc-32233, RRID:AB_627679), α-OPA1 (BD Biosciences, cat. no. 612606, RRID:AB_399888), α-PINK1(Novus, cat. no. BC 100-494, RRID:AB_526188), α-PINK1 (DCP RRID:AB_3095414), α-ubiquitin pSer65 (Cell Signaling Technology, cat. no. 62802, RRID:AB_2799632), α-TOM20 (Santa Cruz Biotechnology, cat. no. sc-17764, RRID:AB_628381), α-TOM22 (Abcam, cat. no. ab179826, RRID:AB_3095411), α-TOM40 (Abcam, cat. no. ab185543, RRID:AB_3095412), α-TOM70 (Aviva Systems Biology, cat. no. OAAN01138, RRID:AB_3095413), and CDC28 (Santa Cruz Biotechnology, cat. no. sc-6709, RRID:AB_671808). We sought to generate in-house antibodies against TOM5, TOM6, and TOM7 due to lack of robust commercial antibodies. Successful generation of TOM6 (fig. S15, A and B) and TOM7 (fig. S15, C and D) antibodies was confirmed by immunoblotting of whole-cell lysates from human embryonic kidney 293 cells and whole-cell lysates and mitochondrial fractions from yeast cells. TOM5 antibody generation was not successful (fig. S15E). TOM6 (University of Dundee, cat. no. DA289, RRID:AB_3095484) and TOM7 (University of Dundee, cat. no. DA290, RRID:AB_3095485) antibodies are available via the MRC PPU Reagents and Services website (https://mrcppureagents.dundee.ac.uk/). The polyclonal α-PINK1 pSer228 antibody was generated by the Michael J. Fox Foundation’s research tools program in partnership with Abcam (Development of a monoclonal antibody is underway. Please contact tools@michaeljfox.org for questions). All fluorophore-conjugated mouse, rabbit, and sheep secondary antibodies for immunoblotting and immunofluorescence were obtained from Sigma-Aldrich.Expression of human PINK1 and TOM complex in yeast cellsCodon-optimized plasmids to express the eight subunits of human PINK1-TOM complex from the bidirectional GAL1-10 promoters in budding yeast were generated, using previously described methods (72). The S. cerevisiae strain YCE1164 (MATa ade2-1 ura3-1 his3-11, 15 trp1-1 leu2-3,112 can1- 100 bar1Δ::hphNT pep4Δ::ADE2) was transformed with linearized plasmids using standard procedures to generate protein expression strains. For the protein expression strains, the codon usage of the synthetic gene constructs was optimized for high-level expression in S. cerevisiae, as described previously (72). A detailed protocol describing the expression of PINK1 and the TOM subunits has been reported (dx.doi.org/10.17504/protocols.io.n2bvj37kwlk5/v1).Yeast protein induction and expressionOvernight culture (300 μl) from positive clones were inoculated into 10 ml of fresh Yeast Peptone (YP) medium supplemented with 2% raffinose and then grown at 30°C with 180 rpm shaking, to an optical density of 600 nm (OD600) of 1.7. Protein expression was subsequently induced by adding galactose to a final concentration of 2% and continuing growth for a further 10 hours. Cells were harvested by centrifugation at 3000g for 10 min, flash-frozen in liquid nitrogen, and stored until needed. A detailed protocol describing the expression of PINK1 and the TOM subunits has been reported (dx.doi.org/10.17504/protocols.io.n2bvj37kwlk5/v1).Yeast protein extraction and quantificationTo extract proteins, frozen cells were allowed to thaw on ice, resuspended in 200 μl of 20% trichloroacetic acid (TCA), and lysed by vortexing for 35 s in the presence of glass beads. After the beads settle, 100 μl of the supernatant was collected into a fresh microfuge tube. An additional 200 μl of 5% TCA was added to the beads and vortexed for a further 35 s, and 150 μl of the supernatant was also collected after allowing the beads settle and added to the previous supernatant collection. The supernatant was centrifugated at maximum speed at 4°C in a microfuge to collect the precipitated protein, which was then resuspended in 200 μl of 25 mM tris, 300 mM NaCl, 10% glycerol, and 0.5 mM tris(2-carboxyethyl)phosphine (TCEP) (pH 7.5) containing protease inhibitor cocktail. Protein quantification was done using bicinchoninic acid (BCA) method and bovine serum albumin (BSA) as standard. A detailed protocol describing protein extraction and quantification from yeast has been reported (dx.doi.org/10.17504/protocols.io.n2bvj37kwlk5/v1).ImmunoblottingThis was carried out as described by Tonelli and Alessi 2022 (dx.doi.org/10.17504/protocols.io.bsgrnbv6). Briefly, 20 μg of protein was subjected to SDS-PAGE (4 to 12% bis-tris gel) and transferred onto nitrocellulose membranes. Membranes were blocked for 1 hour in tris-buffered saline with 0.1% Tween (TBST) containing 5% (w/v) milk. Membranes were then probed with the indicated antibodies in TBST containing 5% (w/v) BSA overnight on a roller at 4°C. Detection was performed using appropriate secondary antibodies and scanned using Li-COR Odyssey CLx imaging system.Isolation of yeast mitochondriaYeast mitochondria were purified following the protocol outlined by Gregg et al. (75). In brief, a 2-liter culture of yeast cells carrying WT PINK1 alone or WT or KI PINK1 with all TOMs was cultured, and expression was induced as described. Cells were harvested by centrifugation at 3000g for 10 min at room temperature. The resulting pellet was washed twice with 1.2 M sorbitol and resuspended in dithiothreitol (DTT) buffer [100 mM tris-H2SO4 (pH 9.4) and 10 mM DTT] at a ratio of 2 ml of buffer per gram of cells. The suspension was gently rotated at 70 rpm at 30°C for 30 min. After a subsequent centrifugation and resuspension in Zymolyase buffer [20 mM K3PO4 (pH 7.4) and 1.2 M sorbitol], the cells were treated with Zymolyase powder (1 mg/g of wet cells) and rotated gently at 70 rpm for 60 min at 30°C. The resulting spheroplasts were centrifuged for 5 min at 3000g at 4°C. Maintaining a cool temperature, the spheroplast was then resuspended in ice-cold homogenization buffer [10 mM tris-HCl (pH 7.4), 0.6 M sorbitol, 1 mM EDTA, and 0.2% BSA] and transferred into a precooled ice glass homogenizer. Using a tight pestle, homogenization was performed with 15 strokes. Following homogenization, differential centrifugation steps at 2000g and 3000g were performed to discard cell debris. The isolated mitochondria were then centrifuged at 15,000g, and, to enhance purity, the mitochondria were resuspended in scanning electron microscopy (SEM) buffer [10 mM Mops/KOH (pH 7.2), 250 mM sucrose, and 1 mM EDTA buffer] and subjected to another centrifugation step. The final mitochondria were resuspended in SEM buffer, and their protein concentration was adjusted to 10 mg/ml using a Bradford protein assay.AlphaFold modelingTo gain insight into how PINK1 might interact with the TOM complex or how PINK1 might interact with TOM70, AlphaFold (RRID:SCR_023662) prediction tool was deployed (76). AMBER (RRID:SCR_016151) structure relaxation was used to ensure appropriate orientation of the side chains to avoid steric clashes. The resulting output models were ranked by confidence level and analyzed by visualization using PyMol (RRID:SCR_000305). All key PINK1-TOM interactions are listed in table S3. The coordinates of the resultant AlphaFold models have been deposited in the public repository Zenodo (doi: http://dx.doi.org/10.5281/zenodo.10792792).Blue native PAGEThe samples for BN-PAGE analysis were prepared using a Native PAGE Sample Prep Kit (Invitrogen). For BN-PAGE, mitochondria-enriched fractions were gently pipetted up and down 10 times in 1× Native PAGE buffer with 1% digitonin followed by an incubation for 30 min at 4°C. The samples were centrifuged at 20,000g for 30 min at 4°C. Samples were quantified by BCA assay and supplemented with 0.002% G-250 (Invitrogen). BN-PAGE was performed by Native PAGE Running Buffers (Invitrogen). The gels were transferred on to polyvinylidene difluoride (PVDF) membranes for immunoblot analysis. PVDF membranes were washed in 100% methanol and subjected to immunoblotting. A detailed protocol describing steps in BN-PAGE has been reported (https://dx.doi.org/10.17504/protocols.io.5qpvo3mezv4o/v1).Mammalian cell culture and transfectionHeLa WT and PINK1 knockout cells were routinely cultured in standard Dulbecco’s modified Eagle’s medium (DMEM) supplemented with 10% fetal bovine serum (FBS), 2 mM l-glutamine, penicillin (100 U ml−1) and streptomycin (100 mg ml−1; 1× Pen/Strep), and 1× nonessential amino acids (Life Technologies). HeLa Flp-In TREx cells were cultured using DMEM supplemented with 10% FBS, 2 mM l-glutamine, 1× Pen/Strep, and blasticidin (15 μg/ml). Culture medium was further supplemented with zeocin (100 μg/ml) pre-recombination with PINK1-3FLAG constructs. Transfections were performed using the polyethylenimine method. To ensure uniform expression of recombinant proteins, stable cell lines were generated in a doxycycline-inducible manner. HeLa Flp-In TREx CRISPR-mediated PINK1 knockout cells were generated. The PINK1 null host cells containing integrated flippase recognition target (FRT) recombination site sequences and Tet repressor were co-transfected with 4.5/9 μg of pOG44 plasmid (which constitutively expresses the Flp recombinase) and 0.5/1 μg of pcDNA5-FRT/TO vector containing a hygromycin resistance gene for selection of the gene of interest with FLAG tag under the control of a doxycycline-regulated promoter. Cells were selected for hygromycin and blasticidin resistance 3 days after transfection by adding fresh medium supplemented with blasticidin (15 μg/ml) and hygromycin (100 μg/ml). Protein expression was induced by the addition of doxycycline (0.1 μg/ml) for 24 hours. Mitochondrial depolarization was induced by treatment with AO (Sigma-Aldrich; prepared in DMSO) for 3 hours. Cells were harvested and resuspended in mitochondrial fractionation buffer [20 mM Hepes (pH 7.5), 250 mM sucrose, 3 mM EDTA, 5 mM sodium β-glycerophosphate, 50 mM sodium fluoride, 5 mM sodium pyrophosphate, 1 mM sodium orthovanadate, and 200 mM chloroacetamide]. Cell suspensions were physically disrupted by 25 passes through a 25-gauge needle, and debris were removed by centrifugation at 800g. The resulting supernatant was subject to centrifugation at 16,600g to harvest a mitochondria-enriched pellet. Samples for SDS-PAGE were generated from mitochondrial lysates resulting from resuspension in mitochondrial fractionation buffer with 1% Triton X-100. A detailed protocol describing the mammalian cell culture and transfection for stable cell line generation has been reported (dx.doi.org/10.17504/protocols.io.kxygxyk9dl8j/v1).Insect cell expression
Cloning of hPINK1 constructs for insect cell expression and test purifications
The coding sequences for the PINK1 constructs were PCR-amplified using clone OHu25380D (GenScript) as a template and cloned into the vector pFB-6HZB (SGC) as previously described (74). Expression from this vector yields in proteins with a Tobacco Etch Virus (TEV) protease-cleavable N-terminal His6-Z tag. Baculoviruses were then generated according to protocols from the Bac-to-Bac expression system (Invitrogen). For protein expression, exponentially growing TriEx cells (3 ml of suspension, 2 × 106 cells/ml, Novagen) in serum-free Insect-XPRESS medium (Lonza) were infected with recombinant virus (multiplicity of infection > 2) and cultured for 66 hours at 27°C under gentle agitation. Cells were harvested by centrifugation (20 min, 1000g, 4°C), resuspended in lysis buffer [50 mM Hepes (pH 7.4), 500 mM NaCl, 0.5 mM TCEP, 5% glycerol, and either with or without 0.05% digitonin], and lysed via sonication (24-tip horn, 35% amplitude, 5-s pulse/10-s pause, 3-min total pulse time). The lysate was cleared by another round of centrifugation (30 min, 13,000 rpm, 4°C) and loaded onto 25 μl of pre-equilibrated Ni-NTA beads (no. 17526802, Cytiva) in gravity flow columns. After washing the beads with lysis buffer, His6-Z-PINK1 was eluted in lysis buffer containing 300 mM imidazole. Samples of the total lysate and elution were analyzed side-by-side in CriterionTM T Precast gels (Bio-Rad). Gels were stained with Coomassie or further processed for Western blotting and immunodetection using anti-PINK1 (no. BC100-494, Novus Biologicals) or anti-hexahistidine antibody (no. SAB2702220, Sigma-Aldrich). A detailed description of all tested constructs is listed in table S4. A detailed protocol describing the expression of PINK1 in insect cells has been reported (dx.doi.org/10.17504/protocols.io.n2bvj3mxxlk5/v1).Live cell imagingYeast strains expressing PINK1-GFP with all the TOM subunits were inoculated into YP liquid medium with 2% raffinose as carbon source and incubated overnight at 30°C. On the following day, cells were diluted and grown to exponential phase (OD600 of 0.3 to 0.6), protein expression was initiated by the addition of 2% galactose for a total of 3 hours and 45 min before completion of the incubation period, and mitochondria were stained by addition of 500 nM of MitoTracker CMXRos Red. At the end of the incubation time, cells were briefly spun down, washed 2× with phosphate-buffered saline and applied to concavalin A–coated coverslips that were then placed on glass slides for immediate image acquisition using a Leica DMi8. Further processing was carried out in the Leica LAS X software (RRID:SCR_013673) platform that includes histogram adjustment and denoising with THUNDER (Leica) (RRID:SCR_023794). A detailed protocol of the yeast cell imaging has been reported (https://dx.doi.org/10.17504/protocols.io.dm6gp3p71vzp/v1).Data analysisRaw values from LICOR analyzed immunoblots for individual replicate experiments are shown in relevant supplementary figures. Data in Fig. 2B are normalized to the WT for five independent clones.

MATERIALS AND METHODS

Molecular biology and cloningFor mammalian expression, PINK1-3FLAG constructs were cloned into pcDNA5 vectors for recombination into Flp-In TRex HeLa PINK1 knockout cell lines (RRID:CVCL_D5JI) (13). For the yeast strains, codon-optimized plasmids to express human PINK1-3FLAG, TOM70, TOM40, TOM22, TOM20, TOM7, TOM6, and TOM5 were generated, using previously described methods (72). Human genes were paired into four genetically modified yeast vectors (pORs), based on the pRS vector series that enables efficient shuttling of vectors in specific yeast strains for facile manipulation (fig. S1) (73). yOR1 strain harbors human PINK1-3FLAG + TOM40; yOR2 strain harbors PINK1-3FLAG + TOM40 + TOM22 + TOM7; yOR3 strain harbors PINK1-3FLAG + TOM40 + TOM22 + TOM7 + TOM70 + TOM20; and yOR4 strain harbors PINK1-3FLAG + TOM40 + TOM22 + TOM7 + TOM70 + TOM20 + TOM5 + TOM6 (table S1). The paired genes were cloned on either side of the GAL1_10 promoter that allows the bidirectional induction of both genes upon addition of galactose to the growth medium. In addition, we made a yeast strain (yOR5) expressing just PINK1 alone, and this was done by transforming yeast cells with plasmid (pOR1) containing just PINK1-3FLAG next to the Gal 1_10 promoter without TOM40. A full list of yeast strains and what plasmids are expressed is listed in table S1. Site-directed mutagenesis was carried out using the Quik-Change method with Thermococcus kodakaraensis (KOD) polymerase (Novagen). All yeast and mammalian constructs (table S2) were verified by The Sequencing Services (School of Life Sciences, University of Dundee) and are now available to request via the Medical Research Council Protein Phosphorylation and Ubiquitylation Unit (MRC PPU) Reagents and Services website (https://mrcppureagents.dundee.ac.uk/). For cloning of hPINK1 constructs for insect cell expression, the coding sequences for the PINK1 constructs were polymerase chain reaction (PCR)–amplified using clone OHu25380D (GenScript) as a template and cloned into the vector pFB-6HZB [Structural Genomics Consortium (SGC); Addgene ID: 218680] as previously described (74). Expression from this vector yields proteins with a TEV protease-cleavable N-terminal His6-Z tag.

Molecular biology and cloning

For mammalian expression, PINK1-3FLAG constructs were cloned into pcDNA5 vectors for recombination into Flp-In TRex HeLa PINK1 knockout cell lines (RRID:CVCL_D5JI) (13). For the yeast strains, codon-optimized plasmids to express human PINK1-3FLAG, TOM70, TOM40, TOM22, TOM20, TOM7, TOM6, and TOM5 were generated, using previously described methods (72). Human genes were paired into four genetically modified yeast vectors (pORs), based on the pRS vector series that enables efficient shuttling of vectors in specific yeast strains for facile manipulation (fig. S1) (73). yOR1 strain harbors human PINK1-3FLAG + TOM40; yOR2 strain harbors PINK1-3FLAG + TOM40 + TOM22 + TOM7; yOR3 strain harbors PINK1-3FLAG + TOM40 + TOM22 + TOM7 + TOM70 + TOM20; and yOR4 strain harbors PINK1-3FLAG + TOM40 + TOM22 + TOM7 + TOM70 + TOM20 + TOM5 + TOM6 (table S1). The paired genes were cloned on either side of the GAL1_10 promoter that allows the bidirectional induction of both genes upon addition of galactose to the growth medium. In addition, we made a yeast strain (yOR5) expressing just PINK1 alone, and this was done by transforming yeast cells with plasmid (pOR1) containing just PINK1-3FLAG next to the Gal 1_10 promoter without TOM40. A full list of yeast strains and what plasmids are expressed is listed in table S1. Site-directed mutagenesis was carried out using the Quik-Change method with Thermococcus kodakaraensis (KOD) polymerase (Novagen). All yeast and mammalian constructs (table S2) were verified by The Sequencing Services (School of Life Sciences, University of Dundee) and are now available to request via the Medical Research Council Protein Phosphorylation and Ubiquitylation Unit (MRC PPU) Reagents and Services website (https://mrcppureagents.dundee.ac.uk/). For cloning of hPINK1 constructs for insect cell expression, the coding sequences for the PINK1 constructs were polymerase chain reaction (PCR)–amplified using clone OHu25380D (GenScript) as a template and cloned into the vector pFB-6HZB [Structural Genomics Consortium (SGC); Addgene ID: 218680] as previously described (74). Expression from this vector yields proteins with a TEV protease-cleavable N-terminal His6-Z tag.

Antibodies for biochemical analysisThe following antibodies were used in this study: α–glyceraldehyde-3-phosphate dehydrogenase (Santa Cruz Biotechnology, cat. no. sc-32233, RRID:AB_627679), α-OPA1 (BD Biosciences, cat. no. 612606, RRID:AB_399888), α-PINK1(Novus, cat. no. BC 100-494, RRID:AB_526188), α-PINK1 (DCP RRID:AB_3095414), α-ubiquitin pSer65 (Cell Signaling Technology, cat. no. 62802, RRID:AB_2799632), α-TOM20 (Santa Cruz Biotechnology, cat. no. sc-17764, RRID:AB_628381), α-TOM22 (Abcam, cat. no. ab179826, RRID:AB_3095411), α-TOM40 (Abcam, cat. no. ab185543, RRID:AB_3095412), α-TOM70 (Aviva Systems Biology, cat. no. OAAN01138, RRID:AB_3095413), and CDC28 (Santa Cruz Biotechnology, cat. no. sc-6709, RRID:AB_671808). We sought to generate in-house antibodies against TOM5, TOM6, and TOM7 due to lack of robust commercial antibodies. Successful generation of TOM6 (fig. S15, A and B) and TOM7 (fig. S15, C and D) antibodies was confirmed by immunoblotting of whole-cell lysates from human embryonic kidney 293 cells and whole-cell lysates and mitochondrial fractions from yeast cells. TOM5 antibody generation was not successful (fig. S15E). TOM6 (University of Dundee, cat. no. DA289, RRID:AB_3095484) and TOM7 (University of Dundee, cat. no. DA290, RRID:AB_3095485) antibodies are available via the MRC PPU Reagents and Services website (https://mrcppureagents.dundee.ac.uk/). The polyclonal α-PINK1 pSer228 antibody was generated by the Michael J. Fox Foundation’s research tools program in partnership with Abcam (Development of a monoclonal antibody is underway. Please contact tools@michaeljfox.org for questions). All fluorophore-conjugated mouse, rabbit, and sheep secondary antibodies for immunoblotting and immunofluorescence were obtained from Sigma-Aldrich.

Antibodies for biochemical analysis

The following antibodies were used in this study: α–glyceraldehyde-3-phosphate dehydrogenase (Santa Cruz Biotechnology, cat. no. sc-32233, RRID:AB_627679), α-OPA1 (BD Biosciences, cat. no. 612606, RRID:AB_399888), α-PINK1(Novus, cat. no. BC 100-494, RRID:AB_526188), α-PINK1 (DCP RRID:AB_3095414), α-ubiquitin pSer65 (Cell Signaling Technology, cat. no. 62802, RRID:AB_2799632), α-TOM20 (Santa Cruz Biotechnology, cat. no. sc-17764, RRID:AB_628381), α-TOM22 (Abcam, cat. no. ab179826, RRID:AB_3095411), α-TOM40 (Abcam, cat. no. ab185543, RRID:AB_3095412), α-TOM70 (Aviva Systems Biology, cat. no. OAAN01138, RRID:AB_3095413), and CDC28 (Santa Cruz Biotechnology, cat. no. sc-6709, RRID:AB_671808). We sought to generate in-house antibodies against TOM5, TOM6, and TOM7 due to lack of robust commercial antibodies. Successful generation of TOM6 (fig. S15, A and B) and TOM7 (fig. S15, C and D) antibodies was confirmed by immunoblotting of whole-cell lysates from human embryonic kidney 293 cells and whole-cell lysates and mitochondrial fractions from yeast cells. TOM5 antibody generation was not successful (fig. S15E). TOM6 (University of Dundee, cat. no. DA289, RRID:AB_3095484) and TOM7 (University of Dundee, cat. no. DA290, RRID:AB_3095485) antibodies are available via the MRC PPU Reagents and Services website (https://mrcppureagents.dundee.ac.uk/). The polyclonal α-PINK1 pSer228 antibody was generated by the Michael J. Fox Foundation’s research tools program in partnership with Abcam (Development of a monoclonal antibody is underway. Please contact tools@michaeljfox.org for questions). All fluorophore-conjugated mouse, rabbit, and sheep secondary antibodies for immunoblotting and immunofluorescence were obtained from Sigma-Aldrich.

Expression of human PINK1 and TOM complex in yeast cellsCodon-optimized plasmids to express the eight subunits of human PINK1-TOM complex from the bidirectional GAL1-10 promoters in budding yeast were generated, using previously described methods (72). The S. cerevisiae strain YCE1164 (MATa ade2-1 ura3-1 his3-11, 15 trp1-1 leu2-3,112 can1- 100 bar1Δ::hphNT pep4Δ::ADE2) was transformed with linearized plasmids using standard procedures to generate protein expression strains. For the protein expression strains, the codon usage of the synthetic gene constructs was optimized for high-level expression in S. cerevisiae, as described previously (72). A detailed protocol describing the expression of PINK1 and the TOM subunits has been reported (dx.doi.org/10.17504/protocols.io.n2bvj37kwlk5/v1).

Expression of human PINK1 and TOM complex in yeast cells

Codon-optimized plasmids to express the eight subunits of human PINK1-TOM complex from the bidirectional GAL1-10 promoters in budding yeast were generated, using previously described methods (72). The S. cerevisiae strain YCE1164 (MATa ade2-1 ura3-1 his3-11, 15 trp1-1 leu2-3,112 can1- 100 bar1Δ::hphNT pep4Δ::ADE2) was transformed with linearized plasmids using standard procedures to generate protein expression strains. For the protein expression strains, the codon usage of the synthetic gene constructs was optimized for high-level expression in S. cerevisiae, as described previously (72). A detailed protocol describing the expression of PINK1 and the TOM subunits has been reported (dx.doi.org/10.17504/protocols.io.n2bvj37kwlk5/v1).

Yeast protein induction and expressionOvernight culture (300 μl) from positive clones were inoculated into 10 ml of fresh Yeast Peptone (YP) medium supplemented with 2% raffinose and then grown at 30°C with 180 rpm shaking, to an optical density of 600 nm (OD600) of 1.7. Protein expression was subsequently induced by adding galactose to a final concentration of 2% and continuing growth for a further 10 hours. Cells were harvested by centrifugation at 3000g for 10 min, flash-frozen in liquid nitrogen, and stored until needed. A detailed protocol describing the expression of PINK1 and the TOM subunits has been reported (dx.doi.org/10.17504/protocols.io.n2bvj37kwlk5/v1).

Yeast protein induction and expression

Overnight culture (300 μl) from positive clones were inoculated into 10 ml of fresh Yeast Peptone (YP) medium supplemented with 2% raffinose and then grown at 30°C with 180 rpm shaking, to an optical density of 600 nm (OD600) of 1.7. Protein expression was subsequently induced by adding galactose to a final concentration of 2% and continuing growth for a further 10 hours. Cells were harvested by centrifugation at 3000g for 10 min, flash-frozen in liquid nitrogen, and stored until needed. A detailed protocol describing the expression of PINK1 and the TOM subunits has been reported (dx.doi.org/10.17504/protocols.io.n2bvj37kwlk5/v1).

Yeast protein extraction and quantificationTo extract proteins, frozen cells were allowed to thaw on ice, resuspended in 200 μl of 20% trichloroacetic acid (TCA), and lysed by vortexing for 35 s in the presence of glass beads. After the beads settle, 100 μl of the supernatant was collected into a fresh microfuge tube. An additional 200 μl of 5% TCA was added to the beads and vortexed for a further 35 s, and 150 μl of the supernatant was also collected after allowing the beads settle and added to the previous supernatant collection. The supernatant was centrifugated at maximum speed at 4°C in a microfuge to collect the precipitated protein, which was then resuspended in 200 μl of 25 mM tris, 300 mM NaCl, 10% glycerol, and 0.5 mM tris(2-carboxyethyl)phosphine (TCEP) (pH 7.5) containing protease inhibitor cocktail. Protein quantification was done using bicinchoninic acid (BCA) method and bovine serum albumin (BSA) as standard. A detailed protocol describing protein extraction and quantification from yeast has been reported (dx.doi.org/10.17504/protocols.io.n2bvj37kwlk5/v1).

Yeast protein extraction and quantification

To extract proteins, frozen cells were allowed to thaw on ice, resuspended in 200 μl of 20% trichloroacetic acid (TCA), and lysed by vortexing for 35 s in the presence of glass beads. After the beads settle, 100 μl of the supernatant was collected into a fresh microfuge tube. An additional 200 μl of 5% TCA was added to the beads and vortexed for a further 35 s, and 150 μl of the supernatant was also collected after allowing the beads settle and added to the previous supernatant collection. The supernatant was centrifugated at maximum speed at 4°C in a microfuge to collect the precipitated protein, which was then resuspended in 200 μl of 25 mM tris, 300 mM NaCl, 10% glycerol, and 0.5 mM tris(2-carboxyethyl)phosphine (TCEP) (pH 7.5) containing protease inhibitor cocktail. Protein quantification was done using bicinchoninic acid (BCA) method and bovine serum albumin (BSA) as standard. A detailed protocol describing protein extraction and quantification from yeast has been reported (dx.doi.org/10.17504/protocols.io.n2bvj37kwlk5/v1).

ImmunoblottingThis was carried out as described by Tonelli and Alessi 2022 (dx.doi.org/10.17504/protocols.io.bsgrnbv6). Briefly, 20 μg of protein was subjected to SDS-PAGE (4 to 12% bis-tris gel) and transferred onto nitrocellulose membranes. Membranes were blocked for 1 hour in tris-buffered saline with 0.1% Tween (TBST) containing 5% (w/v) milk. Membranes were then probed with the indicated antibodies in TBST containing 5% (w/v) BSA overnight on a roller at 4°C. Detection was performed using appropriate secondary antibodies and scanned using Li-COR Odyssey CLx imaging system.

Immunoblotting

This was carried out as described by Tonelli and Alessi 2022 (dx.doi.org/10.17504/protocols.io.bsgrnbv6). Briefly, 20 μg of protein was subjected to SDS-PAGE (4 to 12% bis-tris gel) and transferred onto nitrocellulose membranes. Membranes were blocked for 1 hour in tris-buffered saline with 0.1% Tween (TBST) containing 5% (w/v) milk. Membranes were then probed with the indicated antibodies in TBST containing 5% (w/v) BSA overnight on a roller at 4°C. Detection was performed using appropriate secondary antibodies and scanned using Li-COR Odyssey CLx imaging system.

Isolation of yeast mitochondriaYeast mitochondria were purified following the protocol outlined by Gregg et al. (75). In brief, a 2-liter culture of yeast cells carrying WT PINK1 alone or WT or KI PINK1 with all TOMs was cultured, and expression was induced as described. Cells were harvested by centrifugation at 3000g for 10 min at room temperature. The resulting pellet was washed twice with 1.2 M sorbitol and resuspended in dithiothreitol (DTT) buffer [100 mM tris-H2SO4 (pH 9.4) and 10 mM DTT] at a ratio of 2 ml of buffer per gram of cells. The suspension was gently rotated at 70 rpm at 30°C for 30 min. After a subsequent centrifugation and resuspension in Zymolyase buffer [20 mM K3PO4 (pH 7.4) and 1.2 M sorbitol], the cells were treated with Zymolyase powder (1 mg/g of wet cells) and rotated gently at 70 rpm for 60 min at 30°C. The resulting spheroplasts were centrifuged for 5 min at 3000g at 4°C. Maintaining a cool temperature, the spheroplast was then resuspended in ice-cold homogenization buffer [10 mM tris-HCl (pH 7.4), 0.6 M sorbitol, 1 mM EDTA, and 0.2% BSA] and transferred into a precooled ice glass homogenizer. Using a tight pestle, homogenization was performed with 15 strokes. Following homogenization, differential centrifugation steps at 2000g and 3000g were performed to discard cell debris. The isolated mitochondria were then centrifuged at 15,000g, and, to enhance purity, the mitochondria were resuspended in scanning electron microscopy (SEM) buffer [10 mM Mops/KOH (pH 7.2), 250 mM sucrose, and 1 mM EDTA buffer] and subjected to another centrifugation step. The final mitochondria were resuspended in SEM buffer, and their protein concentration was adjusted to 10 mg/ml using a Bradford protein assay.

Isolation of yeast mitochondria

Yeast mitochondria were purified following the protocol outlined by Gregg et al. (75). In brief, a 2-liter culture of yeast cells carrying WT PINK1 alone or WT or KI PINK1 with all TOMs was cultured, and expression was induced as described. Cells were harvested by centrifugation at 3000g for 10 min at room temperature. The resulting pellet was washed twice with 1.2 M sorbitol and resuspended in dithiothreitol (DTT) buffer [100 mM tris-H2SO4 (pH 9.4) and 10 mM DTT] at a ratio of 2 ml of buffer per gram of cells. The suspension was gently rotated at 70 rpm at 30°C for 30 min. After a subsequent centrifugation and resuspension in Zymolyase buffer [20 mM K3PO4 (pH 7.4) and 1.2 M sorbitol], the cells were treated with Zymolyase powder (1 mg/g of wet cells) and rotated gently at 70 rpm for 60 min at 30°C. The resulting spheroplasts were centrifuged for 5 min at 3000g at 4°C. Maintaining a cool temperature, the spheroplast was then resuspended in ice-cold homogenization buffer [10 mM tris-HCl (pH 7.4), 0.6 M sorbitol, 1 mM EDTA, and 0.2% BSA] and transferred into a precooled ice glass homogenizer. Using a tight pestle, homogenization was performed with 15 strokes. Following homogenization, differential centrifugation steps at 2000g and 3000g were performed to discard cell debris. The isolated mitochondria were then centrifuged at 15,000g, and, to enhance purity, the mitochondria were resuspended in scanning electron microscopy (SEM) buffer [10 mM Mops/KOH (pH 7.2), 250 mM sucrose, and 1 mM EDTA buffer] and subjected to another centrifugation step. The final mitochondria were resuspended in SEM buffer, and their protein concentration was adjusted to 10 mg/ml using a Bradford protein assay.

AlphaFold modelingTo gain insight into how PINK1 might interact with the TOM complex or how PINK1 might interact with TOM70, AlphaFold (RRID:SCR_023662) prediction tool was deployed (76). AMBER (RRID:SCR_016151) structure relaxation was used to ensure appropriate orientation of the side chains to avoid steric clashes. The resulting output models were ranked by confidence level and analyzed by visualization using PyMol (RRID:SCR_000305). All key PINK1-TOM interactions are listed in table S3. The coordinates of the resultant AlphaFold models have been deposited in the public repository Zenodo (doi: http://dx.doi.org/10.5281/zenodo.10792792).

AlphaFold modeling

To gain insight into how PINK1 might interact with the TOM complex or how PINK1 might interact with TOM70, AlphaFold (RRID:SCR_023662) prediction tool was deployed (76). AMBER (RRID:SCR_016151) structure relaxation was used to ensure appropriate orientation of the side chains to avoid steric clashes. The resulting output models were ranked by confidence level and analyzed by visualization using PyMol (RRID:SCR_000305). All key PINK1-TOM interactions are listed in table S3. The coordinates of the resultant AlphaFold models have been deposited in the public repository Zenodo (doi: http://dx.doi.org/10.5281/zenodo.10792792).

Blue native PAGEThe samples for BN-PAGE analysis were prepared using a Native PAGE Sample Prep Kit (Invitrogen). For BN-PAGE, mitochondria-enriched fractions were gently pipetted up and down 10 times in 1× Native PAGE buffer with 1% digitonin followed by an incubation for 30 min at 4°C. The samples were centrifuged at 20,000g for 30 min at 4°C. Samples were quantified by BCA assay and supplemented with 0.002% G-250 (Invitrogen). BN-PAGE was performed by Native PAGE Running Buffers (Invitrogen). The gels were transferred on to polyvinylidene difluoride (PVDF) membranes for immunoblot analysis. PVDF membranes were washed in 100% methanol and subjected to immunoblotting. A detailed protocol describing steps in BN-PAGE has been reported (https://dx.doi.org/10.17504/protocols.io.5qpvo3mezv4o/v1).

Blue native PAGE

The samples for BN-PAGE analysis were prepared using a Native PAGE Sample Prep Kit (Invitrogen). For BN-PAGE, mitochondria-enriched fractions were gently pipetted up and down 10 times in 1× Native PAGE buffer with 1% digitonin followed by an incubation for 30 min at 4°C. The samples were centrifuged at 20,000g for 30 min at 4°C. Samples were quantified by BCA assay and supplemented with 0.002% G-250 (Invitrogen). BN-PAGE was performed by Native PAGE Running Buffers (Invitrogen). The gels were transferred on to polyvinylidene difluoride (PVDF) membranes for immunoblot analysis. PVDF membranes were washed in 100% methanol and subjected to immunoblotting. A detailed protocol describing steps in BN-PAGE has been reported (https://dx.doi.org/10.17504/protocols.io.5qpvo3mezv4o/v1).

Mammalian cell culture and transfectionHeLa WT and PINK1 knockout cells were routinely cultured in standard Dulbecco’s modified Eagle’s medium (DMEM) supplemented with 10% fetal bovine serum (FBS), 2 mM l-glutamine, penicillin (100 U ml−1) and streptomycin (100 mg ml−1; 1× Pen/Strep), and 1× nonessential amino acids (Life Technologies). HeLa Flp-In TREx cells were cultured using DMEM supplemented with 10% FBS, 2 mM l-glutamine, 1× Pen/Strep, and blasticidin (15 μg/ml). Culture medium was further supplemented with zeocin (100 μg/ml) pre-recombination with PINK1-3FLAG constructs. Transfections were performed using the polyethylenimine method. To ensure uniform expression of recombinant proteins, stable cell lines were generated in a doxycycline-inducible manner. HeLa Flp-In TREx CRISPR-mediated PINK1 knockout cells were generated. The PINK1 null host cells containing integrated flippase recognition target (FRT) recombination site sequences and Tet repressor were co-transfected with 4.5/9 μg of pOG44 plasmid (which constitutively expresses the Flp recombinase) and 0.5/1 μg of pcDNA5-FRT/TO vector containing a hygromycin resistance gene for selection of the gene of interest with FLAG tag under the control of a doxycycline-regulated promoter. Cells were selected for hygromycin and blasticidin resistance 3 days after transfection by adding fresh medium supplemented with blasticidin (15 μg/ml) and hygromycin (100 μg/ml). Protein expression was induced by the addition of doxycycline (0.1 μg/ml) for 24 hours. Mitochondrial depolarization was induced by treatment with AO (Sigma-Aldrich; prepared in DMSO) for 3 hours. Cells were harvested and resuspended in mitochondrial fractionation buffer [20 mM Hepes (pH 7.5), 250 mM sucrose, 3 mM EDTA, 5 mM sodium β-glycerophosphate, 50 mM sodium fluoride, 5 mM sodium pyrophosphate, 1 mM sodium orthovanadate, and 200 mM chloroacetamide]. Cell suspensions were physically disrupted by 25 passes through a 25-gauge needle, and debris were removed by centrifugation at 800g. The resulting supernatant was subject to centrifugation at 16,600g to harvest a mitochondria-enriched pellet. Samples for SDS-PAGE were generated from mitochondrial lysates resulting from resuspension in mitochondrial fractionation buffer with 1% Triton X-100. A detailed protocol describing the mammalian cell culture and transfection for stable cell line generation has been reported (dx.doi.org/10.17504/protocols.io.kxygxyk9dl8j/v1).

Mammalian cell culture and transfection

HeLa WT and PINK1 knockout cells were routinely cultured in standard Dulbecco’s modified Eagle’s medium (DMEM) supplemented with 10% fetal bovine serum (FBS), 2 mM l-glutamine, penicillin (100 U ml−1) and streptomycin (100 mg ml−1; 1× Pen/Strep), and 1× nonessential amino acids (Life Technologies). HeLa Flp-In TREx cells were cultured using DMEM supplemented with 10% FBS, 2 mM l-glutamine, 1× Pen/Strep, and blasticidin (15 μg/ml). Culture medium was further supplemented with zeocin (100 μg/ml) pre-recombination with PINK1-3FLAG constructs. Transfections were performed using the polyethylenimine method. To ensure uniform expression of recombinant proteins, stable cell lines were generated in a doxycycline-inducible manner. HeLa Flp-In TREx CRISPR-mediated PINK1 knockout cells were generated. The PINK1 null host cells containing integrated flippase recognition target (FRT) recombination site sequences and Tet repressor were co-transfected with 4.5/9 μg of pOG44 plasmid (which constitutively expresses the Flp recombinase) and 0.5/1 μg of pcDNA5-FRT/TO vector containing a hygromycin resistance gene for selection of the gene of interest with FLAG tag under the control of a doxycycline-regulated promoter. Cells were selected for hygromycin and blasticidin resistance 3 days after transfection by adding fresh medium supplemented with blasticidin (15 μg/ml) and hygromycin (100 μg/ml). Protein expression was induced by the addition of doxycycline (0.1 μg/ml) for 24 hours. Mitochondrial depolarization was induced by treatment with AO (Sigma-Aldrich; prepared in DMSO) for 3 hours. Cells were harvested and resuspended in mitochondrial fractionation buffer [20 mM Hepes (pH 7.5), 250 mM sucrose, 3 mM EDTA, 5 mM sodium β-glycerophosphate, 50 mM sodium fluoride, 5 mM sodium pyrophosphate, 1 mM sodium orthovanadate, and 200 mM chloroacetamide]. Cell suspensions were physically disrupted by 25 passes through a 25-gauge needle, and debris were removed by centrifugation at 800g. The resulting supernatant was subject to centrifugation at 16,600g to harvest a mitochondria-enriched pellet. Samples for SDS-PAGE were generated from mitochondrial lysates resulting from resuspension in mitochondrial fractionation buffer with 1% Triton X-100. A detailed protocol describing the mammalian cell culture and transfection for stable cell line generation has been reported (dx.doi.org/10.17504/protocols.io.kxygxyk9dl8j/v1).

Insect cell expression
Cloning of hPINK1 constructs for insect cell expression and test purifications
The coding sequences for the PINK1 constructs were PCR-amplified using clone OHu25380D (GenScript) as a template and cloned into the vector pFB-6HZB (SGC) as previously described (74). Expression from this vector yields in proteins with a Tobacco Etch Virus (TEV) protease-cleavable N-terminal His6-Z tag. Baculoviruses were then generated according to protocols from the Bac-to-Bac expression system (Invitrogen). For protein expression, exponentially growing TriEx cells (3 ml of suspension, 2 × 106 cells/ml, Novagen) in serum-free Insect-XPRESS medium (Lonza) were infected with recombinant virus (multiplicity of infection > 2) and cultured for 66 hours at 27°C under gentle agitation. Cells were harvested by centrifugation (20 min, 1000g, 4°C), resuspended in lysis buffer [50 mM Hepes (pH 7.4), 500 mM NaCl, 0.5 mM TCEP, 5% glycerol, and either with or without 0.05% digitonin], and lysed via sonication (24-tip horn, 35% amplitude, 5-s pulse/10-s pause, 3-min total pulse time). The lysate was cleared by another round of centrifugation (30 min, 13,000 rpm, 4°C) and loaded onto 25 μl of pre-equilibrated Ni-NTA beads (no. 17526802, Cytiva) in gravity flow columns. After washing the beads with lysis buffer, His6-Z-PINK1 was eluted in lysis buffer containing 300 mM imidazole. Samples of the total lysate and elution were analyzed side-by-side in CriterionTM T Precast gels (Bio-Rad). Gels were stained with Coomassie or further processed for Western blotting and immunodetection using anti-PINK1 (no. BC100-494, Novus Biologicals) or anti-hexahistidine antibody (no. SAB2702220, Sigma-Aldrich). A detailed description of all tested constructs is listed in table S4. A detailed protocol describing the expression of PINK1 in insect cells has been reported (dx.doi.org/10.17504/protocols.io.n2bvj3mxxlk5/v1).

Insect cell expression

Cloning of hPINK1 constructs for insect cell expression and test purifications
The coding sequences for the PINK1 constructs were PCR-amplified using clone OHu25380D (GenScript) as a template and cloned into the vector pFB-6HZB (SGC) as previously described (74). Expression from this vector yields in proteins with a Tobacco Etch Virus (TEV) protease-cleavable N-terminal His6-Z tag. Baculoviruses were then generated according to protocols from the Bac-to-Bac expression system (Invitrogen). For protein expression, exponentially growing TriEx cells (3 ml of suspension, 2 × 106 cells/ml, Novagen) in serum-free Insect-XPRESS medium (Lonza) were infected with recombinant virus (multiplicity of infection > 2) and cultured for 66 hours at 27°C under gentle agitation. Cells were harvested by centrifugation (20 min, 1000g, 4°C), resuspended in lysis buffer [50 mM Hepes (pH 7.4), 500 mM NaCl, 0.5 mM TCEP, 5% glycerol, and either with or without 0.05% digitonin], and lysed via sonication (24-tip horn, 35% amplitude, 5-s pulse/10-s pause, 3-min total pulse time). The lysate was cleared by another round of centrifugation (30 min, 13,000 rpm, 4°C) and loaded onto 25 μl of pre-equilibrated Ni-NTA beads (no. 17526802, Cytiva) in gravity flow columns. After washing the beads with lysis buffer, His6-Z-PINK1 was eluted in lysis buffer containing 300 mM imidazole. Samples of the total lysate and elution were analyzed side-by-side in CriterionTM T Precast gels (Bio-Rad). Gels were stained with Coomassie or further processed for Western blotting and immunodetection using anti-PINK1 (no. BC100-494, Novus Biologicals) or anti-hexahistidine antibody (no. SAB2702220, Sigma-Aldrich). A detailed description of all tested constructs is listed in table S4. A detailed protocol describing the expression of PINK1 in insect cells has been reported (dx.doi.org/10.17504/protocols.io.n2bvj3mxxlk5/v1).

Cloning of hPINK1 constructs for insect cell expression and test purifications

The coding sequences for the PINK1 constructs were PCR-amplified using clone OHu25380D (GenScript) as a template and cloned into the vector pFB-6HZB (SGC) as previously described (74). Expression from this vector yields in proteins with a Tobacco Etch Virus (TEV) protease-cleavable N-terminal His6-Z tag. Baculoviruses were then generated according to protocols from the Bac-to-Bac expression system (Invitrogen). For protein expression, exponentially growing TriEx cells (3 ml of suspension, 2 × 106 cells/ml, Novagen) in serum-free Insect-XPRESS medium (Lonza) were infected with recombinant virus (multiplicity of infection > 2) and cultured for 66 hours at 27°C under gentle agitation. Cells were harvested by centrifugation (20 min, 1000g, 4°C), resuspended in lysis buffer [50 mM Hepes (pH 7.4), 500 mM NaCl, 0.5 mM TCEP, 5% glycerol, and either with or without 0.05% digitonin], and lysed via sonication (24-tip horn, 35% amplitude, 5-s pulse/10-s pause, 3-min total pulse time). The lysate was cleared by another round of centrifugation (30 min, 13,000 rpm, 4°C) and loaded onto 25 μl of pre-equilibrated Ni-NTA beads (no. 17526802, Cytiva) in gravity flow columns. After washing the beads with lysis buffer, His6-Z-PINK1 was eluted in lysis buffer containing 300 mM imidazole. Samples of the total lysate and elution were analyzed side-by-side in CriterionTM T Precast gels (Bio-Rad). Gels were stained with Coomassie or further processed for Western blotting and immunodetection using anti-PINK1 (no. BC100-494, Novus Biologicals) or anti-hexahistidine antibody (no. SAB2702220, Sigma-Aldrich). A detailed description of all tested constructs is listed in table S4. A detailed protocol describing the expression of PINK1 in insect cells has been reported (dx.doi.org/10.17504/protocols.io.n2bvj3mxxlk5/v1).

Live cell imagingYeast strains expressing PINK1-GFP with all the TOM subunits were inoculated into YP liquid medium with 2% raffinose as carbon source and incubated overnight at 30°C. On the following day, cells were diluted and grown to exponential phase (OD600 of 0.3 to 0.6), protein expression was initiated by the addition of 2% galactose for a total of 3 hours and 45 min before completion of the incubation period, and mitochondria were stained by addition of 500 nM of MitoTracker CMXRos Red. At the end of the incubation time, cells were briefly spun down, washed 2× with phosphate-buffered saline and applied to concavalin A–coated coverslips that were then placed on glass slides for immediate image acquisition using a Leica DMi8. Further processing was carried out in the Leica LAS X software (RRID:SCR_013673) platform that includes histogram adjustment and denoising with THUNDER (Leica) (RRID:SCR_023794). A detailed protocol of the yeast cell imaging has been reported (https://dx.doi.org/10.17504/protocols.io.dm6gp3p71vzp/v1).

Live cell imaging

Yeast strains expressing PINK1-GFP with all the TOM subunits were inoculated into YP liquid medium with 2% raffinose as carbon source and incubated overnight at 30°C. On the following day, cells were diluted and grown to exponential phase (OD600 of 0.3 to 0.6), protein expression was initiated by the addition of 2% galactose for a total of 3 hours and 45 min before completion of the incubation period, and mitochondria were stained by addition of 500 nM of MitoTracker CMXRos Red. At the end of the incubation time, cells were briefly spun down, washed 2× with phosphate-buffered saline and applied to concavalin A–coated coverslips that were then placed on glass slides for immediate image acquisition using a Leica DMi8. Further processing was carried out in the Leica LAS X software (RRID:SCR_013673) platform that includes histogram adjustment and denoising with THUNDER (Leica) (RRID:SCR_023794). A detailed protocol of the yeast cell imaging has been reported (https://dx.doi.org/10.17504/protocols.io.dm6gp3p71vzp/v1).

Data analysisRaw values from LICOR analyzed immunoblots for individual replicate experiments are shown in relevant supplementary figures. Data in Fig. 2B are normalized to the WT for five independent clones.

Data analysis

Raw values from LICOR analyzed immunoblots for individual replicate experiments are shown in relevant supplementary figures. Data in Fig. 2B are normalized to the WT for five independent clones.
