---
pmid: '36950384'
title: Protein interaction studies in human induced neurons indicate convergent biology
  underlying autism spectrum disorders.
authors:
- Pintacuda G
- Hsu YH
- Tsafou K
- Li KW
- Martín JM
- Riseman J
- Biagini JC
- Ching JKT
- Mena D
- Gonzalez-Lozano MA
- Egri SB
- Jaffe J
- Smit AB
- Fornelos N
- Eggan KC
- Lage K
journal: Cell Genom
year: '2023'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC10025425
doi: 10.1016/j.xgen.2022.100250
---

# Protein interaction studies in human induced neurons indicate convergent biology underlying autism spectrum disorders.
**Authors:** Pintacuda G, Hsu YH, Tsafou K, Li KW, Martín JM, Riseman J, Biagini JC, Ching JKT, Mena D, Gonzalez-Lozano MA, Egri SB, Jaffe J, Smit AB, Fornelos N, Eggan KC, Lage K
**Journal:** Cell Genom (2023)
**DOI:** [10.1016/j.xgen.2022.100250](https://doi.org/10.1016/j.xgen.2022.100250)
**PMC:** [PMC10025425](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10025425/)

## Abstract

1. Cell Genom. 2023 Jan 24;3(3):100250. doi: 10.1016/j.xgen.2022.100250. 
eCollection 2023 Mar 8.

Protein interaction studies in human induced neurons indicate convergent biology 
underlying autism spectrum disorders.

Pintacuda G(1)(2)(3), Hsu YH(1)(2)(4), Tsafou K(1)(4), Li KW(5), Martín 
JM(1)(3), Riseman J(1)(3), Biagini JC(1)(2), Ching JKT(1)(2), Mena D(1)(2), 
Gonzalez-Lozano MA(5), Egri SB(6), Jaffe J(6), Smit AB(5), Fornelos N(1)(2)(4), 
Eggan KC(1)(3), Lage K(1)(2)(4)(7).

Author information:
(1)Stanley Center for Psychiatric Research, Broad Institute of MIT and Harvard, 
Cambridge, MA 02142, USA.
(2)Novo Nordisk Foundation Center for Genomic Mechanisms of Disease, Broad 
Institute of MIT and Harvard, Cambridge, MA 02142, USA.
(3)Harvard Stem Cell Institute and Department of Stem Cell and Regenerative 
Biology, Harvard University, Cambridge, MA 02138, USA.
(4)Department of Surgery, Massachusetts General Hospital, Boston, MA 02114, USA.
(5)Department of Molecular and Cellular Neurobiology, CNCR, VU University 
Amsterdam, 1081 HV Amsterdam, The Netherlands.
(6)Proteomics Platform, Broad Institute of MIT and Harvard, Cambridge, MA 02142, 
USA.
(7)Institute of Biological Psychiatry, Mental Health Centre Sct. Hans, Mental 
Health Services Copenhagen, 4000 Roskilde, Denmark.

Autism spectrum disorders (ASDs) have been linked to genes with enriched 
expression in the brain, but it is unclear how these genes converge into 
cell-type-specific networks. We built a protein-protein interaction network for 
13 ASD-associated genes in human excitatory neurons derived from induced 
pluripotent stem cells (iPSCs). The network contains newly reported interactions 
and is enriched for genetic and transcriptional perturbations observed in 
individuals with ASDs. We leveraged the network data to show that the ASD-linked 
brain-specific isoform of ANK2 is important for its interactions with synaptic 
proteins and to characterize a PTEN-AKAP8L interaction that influences neuronal 
growth. The IGF2BP1-3 complex emerged as a convergent point in the network that 
may regulate a transcriptional circuit of ASD-associated genes. Our findings 
showcase cell-type-specific interactomes as a framework to complement genetic 
and transcriptomic data and illustrate how both individual and convergent 
interactions can lead to biological insights into ASDs.

© 2023 The Authors.

DOI: 10.1016/j.xgen.2022.100250
PMCID: PMC10025425
PMID: 36950384

Conflict of interest statement: K.C.E. is a co-founder of Q-State Biosciences, 
Quralis, and Enclear and is currently employed at BioMarin Pharmaceutical.

## Full Text

IntroductionAutism spectrum disorders (ASDs) comprise a group of heritable neurodevelopmental conditions associated with challenges in social interaction, cognition, and behavior. Recent progress indicates that the diverse phenotypes in ASDs reflect polygenic contributions from both common and rare genetic variants.1,2,3 While common variants account for most of ASD heritability,4 rare variants tend to have larger effect sizes, affect protein-coding genes, and are therefore more easily amenable to functional investigation.5,6 However, even though some rare variants in ASD-associated genes affect specific physiological processes such as synaptic morphogenesis,7,8 the majority do not individually pinpoint the mechanistic basis of ASD biology. More broadly, it remains to be determined how ASD-associated genes integrate into high-order pathways and networks during neurodevelopment.9The integration of genetic and biochemical data shows that genes implicated in complex disorders often converge into protein-protein interaction (PPI) networks.10,11,12,13 In the context of ASDs, earlier studies of de novo mutations and structural variations also mapped ASD-associated variants onto interconnected PPI networks.14,15,16 However, these studies mostly relied on PPI data derived from public databases and thus may lack cell-type specificity. Indeed, experiments from highly proliferative cell types (e.g., cancer cell lines) with no immediate relevance to the brain are vastly over-represented in these databases.12,17,18 In contrast, the integration of genetic and transcriptomic data shows that ASD-associated genes have enriched expression in specific neuronal and non-neuronal cell populations in the brain, with excitatory neurons showing one of the strongest signals.5,6,19,20,21 Transcriptomic profiling in postmortem brain tissues from ASD individuals have also identified cortical excitatory neurons as a key dysregulated cell type in ASDs.22,23,24 Therefore, induced pluripotent stem cell (iPSC)-derived neurons (iNs) that resemble cortical excitatory neurons provide an in vitro neuronal cell model of key relevance to study ASDs.25,26Here, we used iPSC-derived excitatory iNs, interaction proteomics, and computational approaches to build a brain cell-type-specific PPI network for genes associated with ASDs. Specifically, we combined data from 26 immunoprecipitation (IP) experiments of 13 ASD-associated proteins in iNs to generate a PPI network in which >90% of the interactions are unreported in the literature. The PPI data are reproducible in iNs (with a >91% replication rate in western blots) and significantly overlap with interactions identified in complex brain tissues. The network is enriched for transcriptionally perturbed genes in layer 2/3 cortical neurons of individuals with ASDs, agreeing with evidence that implicates this cell type in the development of ASDs. To exemplify how perturbing specific PPIs in the network provides mechanistic insights into ASD-relevant cellular phenotypes, we ablated a brain-specific exon of ANK2 to modulate its interactions with synaptic proteins and disrupted a newly found PTEN-AKAP8L interaction affecting neuronal growth. Moreover, we observed that convergent signals in the network can also point to ASD-related pathways. This was illustrated by the mRNA-binding complex IGF2BP1-3, which connects to multiple ASD-associated proteins in the network and binds to the transcripts of many ASD-associated genes. Finally, the network is enriched for rare variant associations to ASDs, indicating that it can complement large-scale genetic studies to prioritize genes for further study.

Introduction

Autism spectrum disorders (ASDs) comprise a group of heritable neurodevelopmental conditions associated with challenges in social interaction, cognition, and behavior. Recent progress indicates that the diverse phenotypes in ASDs reflect polygenic contributions from both common and rare genetic variants.1,2,3 While common variants account for most of ASD heritability,4 rare variants tend to have larger effect sizes, affect protein-coding genes, and are therefore more easily amenable to functional investigation.5,6 However, even though some rare variants in ASD-associated genes affect specific physiological processes such as synaptic morphogenesis,7,8 the majority do not individually pinpoint the mechanistic basis of ASD biology. More broadly, it remains to be determined how ASD-associated genes integrate into high-order pathways and networks during neurodevelopment.9

The integration of genetic and biochemical data shows that genes implicated in complex disorders often converge into protein-protein interaction (PPI) networks.10,11,12,13 In the context of ASDs, earlier studies of de novo mutations and structural variations also mapped ASD-associated variants onto interconnected PPI networks.14,15,16 However, these studies mostly relied on PPI data derived from public databases and thus may lack cell-type specificity. Indeed, experiments from highly proliferative cell types (e.g., cancer cell lines) with no immediate relevance to the brain are vastly over-represented in these databases.12,17,18 In contrast, the integration of genetic and transcriptomic data shows that ASD-associated genes have enriched expression in specific neuronal and non-neuronal cell populations in the brain, with excitatory neurons showing one of the strongest signals.5,6,19,20,21 Transcriptomic profiling in postmortem brain tissues from ASD individuals have also identified cortical excitatory neurons as a key dysregulated cell type in ASDs.22,23,24 Therefore, induced pluripotent stem cell (iPSC)-derived neurons (iNs) that resemble cortical excitatory neurons provide an in vitro neuronal cell model of key relevance to study ASDs.25,26

Here, we used iPSC-derived excitatory iNs, interaction proteomics, and computational approaches to build a brain cell-type-specific PPI network for genes associated with ASDs. Specifically, we combined data from 26 immunoprecipitation (IP) experiments of 13 ASD-associated proteins in iNs to generate a PPI network in which >90% of the interactions are unreported in the literature. The PPI data are reproducible in iNs (with a >91% replication rate in western blots) and significantly overlap with interactions identified in complex brain tissues. The network is enriched for transcriptionally perturbed genes in layer 2/3 cortical neurons of individuals with ASDs, agreeing with evidence that implicates this cell type in the development of ASDs. To exemplify how perturbing specific PPIs in the network provides mechanistic insights into ASD-relevant cellular phenotypes, we ablated a brain-specific exon of ANK2 to modulate its interactions with synaptic proteins and disrupted a newly found PTEN-AKAP8L interaction affecting neuronal growth. Moreover, we observed that convergent signals in the network can also point to ASD-related pathways. This was illustrated by the mRNA-binding complex IGF2BP1-3, which connects to multiple ASD-associated proteins in the network and binds to the transcripts of many ASD-associated genes. Finally, the network is enriched for rare variant associations to ASDs, indicating that it can complement large-scale genetic studies to prioritize genes for further study.

ResultsASD-associated genes and proteins are expressed in induced neuronsAn exome sequencing study5 that combined de novo and case-control rare variation data identified 24 high-confidence ASD-associated genes at exome-wide significance (Q ≤ 1.35 × 10−4). We used these 24 genes as the starting point of our workflow to generate an ASD PPI network in iNs (Figure 1A), and hereafter refer to them and their encoded proteins as index genes and index proteins, respectively.Figure 1Study workflow and expression of ASD index genes and proteins in human iNs(A) Workflow to generate and analyze a PPI network for high-confidence ASD-associated genes5 in human iNs.(B) Key steps in the glutamatergic-patterning protocol25 utilized to generate neural progenitor cells (NPCs) and excitatory iNs from human iPSCs.(C) Heatmap of ASD index gene expression over the course of neuronal maturation derived from RNA-seq data. CPM, mean normalized counts per million of two replicates; maximum log2(CPM) was capped at 9 for visualization.(D) Flowchart of index protein filtering.(E) Expression of the 13 index proteins in differentiating iPSCs, mouse cortex, and HEK293 cells detected by immunoblotting. The antibody utilized against ANK2 recognizes a human-specific epitope, and SYNGAP1 is not expressed in HEK293 cells. B-actin (B-ACT) was used as a loading control. Molecular weights (kDa) are marked on the left of each blot.In terms of the cell model for our experiments, we adopted and optimized a protocol25 that combines programming of the neurogenic factor Neurogenin 2 (NGN2) with developmental patterning to differentiate iPSCs into iNs. We used an iPSC line (iPS3, STAR Methods) where NGN2 is integrated in the genome under a tetON promoter for rapid and controlled doxycycline-induced expression, resulting in highly homogeneous iN populations25 (Figures 1B and S1A). We measured the expression of ASD-associated index genes through RNA sequencing (RNA-seq) of cells during weeks 0, 3, and 6 of neuronal differentiation (Figure 1C). The expression patterns of these genes varied across time, but most were expressed after 3 weeks of differentiation, including genes with previously reported neuronal identity.27 Importantly, gene expression observed in iNs as early as week 3 (Figure 1C) is consistent with presence of the same transcripts in excitatory neuron populations of the human cortex24 (Figures S1B and S1C), supporting this cell model as a proxy for cell types in the brain.Next, given that RNA-seq data can only capture the overall number of transcripts at steady state without accounting for the translation rate and turnover of the encoded protein products,28 we assessed expression of the index proteins using immunoblotting. We were unable to obtain antibodies to detect seven of the index proteins (Figure 1D; Table S1). Four proteins were successfully detected by immunoblotting (Figure S1D) but lacked IP-competent antibodies. The remaining 13 proteins (Figure 1D) were detected by immunoblotting in the form of at least one isoform at week 3 of differentiation (Figure 1E) and were amenable to IP. Many of these proteins showed distinct isoform patterns across species, neuronal differentiation time points, or cell types (Figure 1E). These observations highlight the importance of using human iNs to capture cell-type-specific biochemical interactions of relevance to ASDs.29,30Interaction proteomics of ASD index proteins in induced neuronsFor the 13 index proteins with IP-competent antibodies, we executed IPs followed by mass spectrometry (IP-MS) in iNs at week 4 of differentiation (Figures 2A, 2B, and S2; Table S2). Each experiment was conducted in duplicates with ∼15 million cells per replicate. Protein abundances in the IP samples were quantified with labeled or label-free liquid chromatography followed by tandem mass spectrometry (LC-MS/MS). In total, we conducted 37 IP-MS experiments, generating two to five experimental datasets for each index protein.Figure 2Generation of a combined PPI network for 13 ASD index proteins in iNs(A) Example of an IP-MS experiment. Left: immunoblot of a SHANK3 IP with its main isoforms marked by a box and asterisk; L, ladder; IN, input; FT, flow-through; IP, immunoprecipitation; IgG, IgG control; H + L IgG, heavy and light IgG chains. Molecular weights are in kilodaltons. Right: log2 fold change (FC) correlation between IP-MS replicates.(B) Volcano plot of the SHANK3 IP-MS experiment, showing SHANK3 in red, its significant interactors (log2 FC > 0 and FDR ≤ 0.1) in green, and non-interactors in blue; known InWeb interactors identified as interactors or non-interactors in the experiment are highlighted in yellow or white, respectively. Statistics were derived from two replicates using a two-tailed one-sample moderated t test.(C) PPI network derived from 26 IP-MS experiments. Nodes represent index proteins (red) and their interactors (purple); color intensity and size of the interactor nodes scale with interactor frequency (i.e., number of linked index proteins). Edges indicate observed interactions, with known InWeb interactions highlighted in blue.(D) Distribution of interactor frequency in the network.(E) Distribution of InWeb versus newly reported interactions in the network.(F) Replication rates of interactions tested in forward or reverse IPs followed by western blotting (IP-WB).(G) Overlap between SCN2A, SHANK3, and SYNGAP1 interactors derived from iN versus brain IPs. Overlap enrichment p value was calculated using a one-tailed hypergeometric test that only considered proteins detected in both sample types.(H) Pairwise co-expression Z scores between index genes and their interactors (Int), non-interactors (NonInt), known InWeb interactors (InWeb), and all protein-coding genes (All) derived from a spatial transcriptomic dataset in human dorsolateral prefrontal cortex.31 Boxes and whiskers in violin plots indicate the interquartile range (IQR) and 1.5x IQR, respectively. Double asterisks indicate p < 0.05/6 (adjusting for six pairwise comparisons) as calculated by two-tailed Wilcoxon rank-sum tests. Number of gene pairs included for each gene type is indicated toward the bottom.For each IP-MS experiment, we performed quality control (QC) and data analysis using Genoppi.18 Specifically, we calculated the log2 fold change (FC) and corresponding statistical significance of each protein identified in the index protein IPs compared with control IPs (Table S2). Eleven datasets failed to pass QC thresholds (i.e., log2 FC correlation between replicates was ≤ 0.6 or the index protein was not enriched at false discovery rate [FDR] ≤ 0.1), leaving us with 26 high-quality datasets. We defined proteins with log2 FC > 0 and FDR ≤ 0.1 as significant interactors of the index protein; other detected proteins were defined as non-interactors (Table S3). We also intersected each dataset with known interactors of the index protein, as determined from the InWeb database,17,18 which aggregates human PPI data from >40,000 publications (Table S3). This allowed us to assess whether the identified interactions have been reported.Our analysis workflow is illustrated with an IP of SHANK3. Enrichment of SHANK3 in the IP was confirmed by immunoblotting (Figure 2A, left) and quantified by IP-MS (log2 FC = 1.2 and FDR = 0.012; Figure 2B). The IP-MS replicate log2 FC correlation was 0.79 (Figure 2A, right). Only two out of 104 significant interactors identified here were reported (Figure 2B). Overall, the enrichment of SHANK3 in both immunoblot and MS results, in combination with the correlation between replicates, support the reproducibility and robustness of our data. The low number of previously identified SHANK3 interactors likely reflects our experimental design (Data S1) and supports the use of neuron-specific proteomics to discover new gene relationships.Since the IP-MS datasets were generated across different MS facilities, we compared the data between facilities and between IPs of the same index protein to evaluate their agreement (Data S2; Figure S3; Table S4). While we observed differences in data structures and detected proteins across the facilities, the identified PPIs have similar quality and degree of biological relevance regardless of facility origin, suggesting these data are complementary. Moreover, we did not observe systematic differences between interactors identified in only one IP versus in multiple IPs of the same index protein (Data S2), or between interactors detected across different FC ranges (Data S3). Therefore, we included all identified interactors in downstream analyses.The PPI network consists of convergent, novel, and reproducible interactionsWe merged the 26 high-quality IP-MS datasets to generate a combined PPI network (Figure 2C; Table S3). This network comprises 1,021 interactors of the 13 index proteins, with >20% of them being linked to multiple index proteins (Figures 2D and S4A). Although the interactors of several index proteins significantly overlap with reported InWeb interactors (Table S3), >90% of all interactions in the network have not yet been reported at the time of this study (Figure 2E). This was not unexpected given the limited number of neuronal PPI datasets in the literature (Data S1) and suggests an opportunity for biological discovery linked to the characterization of these interactions.To support the robustness of MS-detected interactions, we validated the technical reproducibility of both unique (i.e., interacting with only one index protein) and shared (i.e., interacting with multiple index proteins) interactors in independent IPs by immunoblotting (Figures S4B–S4D; Table S5). Overall, we validated 65 out of 71 tested interactions (91.5% replication rate; Figure 2F, left), according to criteria detailed in STAR Methods. In reverse IPs, performed with a panel of interactors as baits, we detected the original index proteins in 19 out of 23 IPs with bait enrichment (82.6% replication rate; Figure 2F, right, Figures S5 and S6; Table S5). These results are in line with the expected replication rate of ∼90% (from the FDR ≤ 0.1 threshold used to define interactors), and with previously reported replication rates in similar experimental designs.18The PPI network recapitulates interactions occurring in the human brainTo compare our data with PPIs in the human brain, we performed IP-MS experiments in postmortem brain homogenates for a subset of index proteins, focusing on proteins with known neuronal function due to limited materials (Table S2). On average, brain IP-MS datasets had lower log2 FC correlations between replicates compared with the iN data (average correlation = 0.63 and 0.72 for brain and iN IPs, respectively). The lower reproducibility likely reflects noise and artifacts. After QC, we considered SCN2A, SHANK3, and SYNGAP1 brain IPs to be high-quality and compared them against the analogous iN IPs. Considering that brain samples contain a mix of cells, whereas iNs are highly homogeneous, we observed agreement in terms of log2 FC correlation for proteins detected in both sample types (median correlation = 0.517 between brain versus iN IPs, compared with median = 0.628 among iN IPs; Figure S7A). Furthermore, 42% of the brain interactors were also identified as interactors in iNs (p = 2.6 × 10−13; Figures 2G, S7B, and S7C). Throughout brain development32 (www.brainspan.org), both brain- and iN-derived interactors have elevated gene expression in the cortex compared with random genes; interactors identified in both sample types have expression profiles comparable with the ASD-associated genes identified through exome sequencing (Figures S8A and S8B).We also characterized the co-expression patterns of interacting proteins in the iN-derived PPI network across four independent datasets derived from human or mouse brains with spatial transcriptomic,31,33 single-cell RNA-seq24 (scRNA-seq), or bulk RNA-seq (www.brainspan.org) approaches (Figures 2H and S8C). In general, index proteins have higher co-expression with their interactors compared with non-interactors detected in iNs during IP-MS, InWeb interactors from mostly non-neuronal contexts, and all protein-coding genes. Therefore, both the IP-MS data from brain homogenates and the brain co-expression analysis indicate that we captured gene relationships found in complex tissues in the human brain.The PPI network implicates layer 2/3 cortical excitatory neurons in ASDsWe assessed the tissue specificity of the PPI network by calculating its overlap enrichment with GTEx tissue-specific genes34 compared with the rest of the genome. The network showed significant enrichment (p < 0.05/53, adjusting for 53 tissues) in the central nervous system (CNS) and several other tissue categories (Figure 3A and Table S6). In contrast, an analogous InWeb-based network showed less pronounced enrichment for the CNS (Figure S8D). When repeated with brain region-specific genes, the network was most enriched (p < 0.05/13, adjusting for 13 brain regions) in the frontal cortex (Figure 3B; Table S6). In addition, we performed SynGO35 gene set analysis, which confirmed that the network is enriched for genes involved in biological processes in the synapse (Figure S8E; Table S6).Figure 3Tissue and cell type enrichment of the ASD PPI network(A–D) Overlap enrichment between the network and GTEx tissue-specific genes34 ([A]; 2,408 genes per tissue), GTEx brain region-specific genes34 ([B]; 2,408 genes per tissue), genes expressed in >50% of cells in each cell type in the postmortem cortex24 ([C]; gene counts in parentheses), and DEGs in each cell type in the postmortem cortex of ASD individuals versus controls24 ([D]; gene counts in parentheses). The p values were derived from one-tailed hypergeometric tests; for (A) and (B), a global background (i.e., all genes in the GTEx data) was used as the population in the test; for (C) and (D), a conditional neuronal background (i.e., iN-expressed genes detected by IP-MS) was used. Left and right vertical dashed lines indicate p < 0.05 and p < 0.05/number of tissues or cell types, respectively; in (B)–(D), results passing these thresholds are labeled with the number of genes in the overlap. AST-FB and AST-PP, fibrous and protoplasmic astrocytes; OPC, oligodendrocyte precursor cells; IN-PV, IN-SST, IN-SV2C, and IN-VIP, parvalbumin, somatostatin, SV2C, and VIP interneurons; L2/3 and L4, layer 2/3 and layer 4 excitatory neurons; L5/6 and L5/6-CC, layer 5/6 corticofugal projection and cortico-cortical projection neurons; Neu-mat, maturing neurons; Neu-NRGN-I and Neu-NRGN-II, NRGN-expression neurons.To link the network to specific cell types implicated in ASDs, we integrated our data with a scRNA-seq study,24 which measured gene expression in the postmortem cortex of individuals with ASDs versus controls and identified differentially expressed genes (DEGs) between the groups in 17 annotated cell types. First, to test whether the network is enriched for commonly expressed genes in each cell type (i.e., expressed in >50% of cells), we performed a global analysis to compare the network genes with other genes in the genome, as well as a conditional analysis to compare the network genes with other iN-expressed genes (i.e., non-interactors detected in IP-MS; Table S3). In both analyses, the network showed significant enrichment (p < 0.05/17, adjusting for 17 cell types) in several neuronal cell types, including both excitatory neurons and inhibitory interneurons (Figures 3C, and S9A; Table S6). Next, we tested the enrichment of the network in terms of its overlap with cell-type-specific DEGs in individuals with ASDs compared with controls. In the global analysis, the network showed enrichment in several cell types, including excitatory neurons, inhibitory interneurons, astrocytes, and microglia (Figure S9B; Table S6). In the more conservative conditional analysis, the layer 2/3 excitatory neurons stood out as the only significant cell type (p < 0.05/17; Figure 3D; Table S6). This enrichment signal is mostly driven by DEGs that were upregulated in individuals with ASDs (Figures S9C and S9D; Table S6).Taken together, these results indicate that the PPI network captures biochemical perturbations and pathways in ASD-relevant tissues and cell types, and specifically implicate layer 2/3 excitatory neurons as a key cell type in ASDs.Dissecting cell-type-specific PPIs of giant ANK2 and PTEN in maturing neuronsTo illustrate that many of the PPIs in our network reflect neuron-specific functions of the ASD index proteins, we further dissected the interactions observed for ANK2 and PTEN. Giant ankyrin-2 is a neuron-specific alternatively spliced variant of ANK2.29,36 This isoform is differentiated from canonical ANK2 due to inclusion of exon 37 (giant exon), which encodes 9,240 amino acids and contains the majority of ANK2 mutations identified in individuals with autism (Figure 4A). While we technically validated many of the identified ANK2 interactors (Figures 4B and S10A; Table S5), they are likely a mix of interactors of all ANK2 isoforms. Thus, to focus specifically on the ASD-associated giant ANK2 isoform, we built an in vitro model of selective loss of giant ANK2 using CRISPR-Cas9 gene editing (Figure 4C).Figure 4Giant ANK2 is responsible for neuron-specific PPIs(A) Depiction of the ANK2 gene with ASD-associated mutations leading to splicing defects or early decay. Giant ANK2 includes exon 37, which is alternatively spliced to produce short ANK2. The epitope recognized by the ANK2 antibody used in this study is marked with an asterisk.(B) Volcano plot of an IP-MS experiment for ANK2 in iNs, showing ANK2 in red, significant interactors (log2 FC > 0 and FDR ≤ 0.1) in green, and non-interactors in blue; known InWeb interactors identified as interactors or non-interactors are highlighted in yellow or white, respectively; interactors replicated by immunoblotting are highlighted in orange. Statistics were derived from two replicates using a two-tailed one-sample moderated t test.(C) Experimental workflow to identify giant ANK2-specific PPIs.(D) Top: CRISPR-Cas9 editing strategy to generate a cell line exclusively expressing the short ANK2 isoform, obtained by deletion of the giant ANK2 exon splicing acceptor. Bottom: validation of giant ANK2 KO cell line. The deletion was confirmed by PCR on genomic DNA using primers flanking the deletion site (left), and by immunoblotting (right) showing the absence of the giant ANK2 isoform in homozygous KO iNs (−/−) compared with WT (+/+) and heterozygous KO (−/−) cells. Giant and canonical ANK2 isoforms are indicated with an asterisk.(E) C3 and CACNA2D1 western blots on ANK2 IPs in WT and giant ANK2 KO iNs. FC was calculated as a ratio of the intensity of the bands detected in the IP versus IN lanes. L, ladder; IN, input; IP, immunoprecipitation. Molecular weights are in kilodaltons. Fifty micrograms of total lysate were loaded in the IN lanes, and 10% of the total immunoprecipitates were loaded in the IP lanes.We edited an iPSC line (iPS3, STAR Methods) to exclude retention of exon 37, which differentiates giant ANK2 from canonical (short) ANK2 isoforms (Figure 4D, top). Fully characterized giant ANK2 knockout (KO) iPSCs (Figure 4D, bottom) were viable, but showed morphological defects, defective ANK2 distribution, and high cell death upon neuronal differentiation. We could not collect sufficient giant ANK2 KO iNs for IP-MS due to high cell death. Nonetheless, we performed immunoblot analysis in KO iNs to test for presence of ANK2 interactors identified by IP-MS in corresponding wild-type (WT) neurons. KO iNs were still expressing proteins essential for neuronal physiology and synaptic function, including the calcium channel subunit CACNA2D137 and the complement protein C3,38 but these proteins were no longer associated with ANK2 (Figure 4E), suggesting that their localization might rely on giant ANK2-dependent cytoskeletal architecture.We also performed IP-MS experiments of ANK2 in isogenic WT and giant ANK2 KO neural progenitor cells (NPCs; Figure S10B; Table S2). ANK2 interactors identified exclusively in the WT cells (which expressed both canonical and giant ANK2 isoforms) were most enriched for “glutamatergic synapses” in Gene Ontology (GO) analysis.39,40 In contrast, interactors specific to the KO cells (which lacked giant ANK2) were enriched for “microtubules” (Table S7). This suggests that the giant ANK2 isoform may have a role in localizing the protein to form cell projections of maturing NPCs and in guiding neuronal morphology. Accordingly, ARPC5L, which is essential for both structural and functional maturation of the growth cone in maturating neurons,41 is among the WT-specific ANK2 interactors (Figure S10B).We next investigated a robust interaction identified between PTEN and the A-kinase anchor protein AKAP8L, which was consistently replicated by immunoblotting and reverse IP (Figure 5A, 5B, and S10C). AKAP8L is implicated in the mammalian target of rapamycin (mTOR) pathway,42 but has not been linked to PTEN in previous biochemical studies (which were mostly conducted in cancer cell models). Thus, our findings in iNs suggest a neuron-specific mTOR circuitry of PTEN that acts through AKAP8L. To test this hypothesis, we generated AKAP8L KO iPSC lines through CRISPR-Cas9 gene editing (Figure 5C). We obtained isogenic lines with both heterozygous and homozygous mutations, with the homozygous KO line resulting in complete loss of AKAP8L (Figure 5D). Interestingly, the AKAP8L homozygous KO line showed an increase in growth rate at the NPC stage compared with isogenic WT and heterozygous lines (p = 0.0287 and 0.0162, respectively; Figure 5E).Figure 5The neuronal PTEN-AKAP8L interaction regulates cell proliferation(A) Volcano plot of an IP-MS experiment for PTEN in iNs, showing PTEN in red, significant interactors (log2 FC > 0 and FDR ≤ 0.1) in green, and non-interactors in blue; AKAP8L is marked in orange. Statistics were derived from two replicates using a two-tailed one-sample moderated t test.(B) AKAP8L immunoblot on the PTEN IP. AKAP8L is marked with an asterisk, IgG heavy chains are marked to the right; L, ladder; IN, input; FT, flow-through; IP, immunoprecipitation; IgG, IgG control. Molecular weights are in kilodaltons.(C) Schematic of the CRISPR-Cas9 editing strategy to generate AKAP8L KO iPSC lines.(D) Immunoblot validation of AKAP8L heterozygous (+/−) and homozygous (−/−) KO lines generated using the strategy in (C). B-actin is used as a loading control. Molecular weights (kDa) are marked on the side of each blot.(E) WT (+/+), AKAP8L heterozygous KO (+/−), and AKAP8L homozygous KO (−/−) neural progenitor cell growth over 4 days upon seeding cells at identical confluences normalized to their seeding density. Error bars indicate standard deviations of the mean of three biological replicates. Asterisks indicate p < 0.05 as calculated using a two-tailed t test.(F) Immunoblot analysis of PTEN, phosphorylated S6 (P-S6), S6, and AKT in WT and AKAP8L homozygous KO iNs. Molecular weights (kDa) are marked on the side of each blot and B-actin is used as loading control. FC was calculated as a ratio of the intensity of the bands detected in the AKAP8L KO (−/−) versus WT (+/+) lanes.Although AKAP8L KO did not result in changes of PTEN levels in iNs as measured by immunoblotting, we observed a small but detectable increase in the levels of phosphorylated ribosomal protein S6 (Figure 5F). This is a marker of hyperactivation of the mTOR pathway and suggests an AKAP8L-mediated defect in the PTEN signaling cascade in ASDs. Furthermore, we detected similar levels of the serine/threonine-protein kinase AKT in WT versus KO cells, suggesting that AKAP8L acts in the mTORC1 but not the mTORC2 pathway.43 These results support a neuronal PTEN-mTOR circuitry that includes AKAP8L and suggest that the PTEN-AKAP8L interaction affects downstream mTORC1 signaling and cell proliferation without affecting PTEN protein levels directly (Figures S10C and S10D).Our investigation of giant ANK2 and PTEN interactions demonstrate the potential of using cell-type-specific PPIs to gain mechanistic insights into ASD-relevant molecular cascades.Network convergence implicates the IGF2BP1-3 complex in an ASD-relevant regulatory circuitrySince the PPI network contains interactors linked to multiple index proteins (Figures 2C and 2D; Table S3), we hypothesized that convergent signals in the network could point to biological themes of relevance to ASDs. By ranking the interactors in descending order based on the number of index proteins they interact with (Table S3), three insulin-like growth factor 2 mRNA-binding proteins, IGF2BP1, IGF2BP3, and IGF2BP2, emerged among the top interactors. Together, these proteins form an N6-methyladenosine (m6A) reader complex broadly involved in post-transcriptional regulation and affect mRNA stability and translation.44 We therefore tested whether this complex is involved in a regulatory circuit that connects transcription of ASD-associated genes to the protein network we identified.We found that RNA targets of IGF2BP1-3 in HEK cells44 are significantly enriched for ASD-associated genes identified through exome sequencing5 (p = 3.9 × 10−10; Figure 6A; Table S8), including nine of the index genes in this study. Genetic enrichment analysis using ASD genome-wide association studies (GWAS) data45 and MAGMA46 also found the IGF2BP1-3 targets to be enriched for common variants associated with ASDs (p = 1.4 × 10−3; Figure 6B; Table S8). Next, we performed a conditional enrichment analysis to test whether genes in both our combined network and index protein-specific sub-networks are enriched for IGF2BP1-3 targets compared with the non-interactors (Figure 6C; Table S8). At a Bonferroni-corrected significance threshold (p < 0.05/56, adjusting for 14 networks and four IGF2BP target lists), the combined and DYRK1A networks both showed significant enrichment (p = 1.7 × 10−17 and 7.6 × 10−9, respectively).Figure 6The IGF2BP1-3 complex interacts with ASD-associated transcripts and proteins(A) Overlap enrichment between RNA targets of IGF2BP1-344 (target counts in parentheses) and ASD-associated genes5 (FDR ≤ 0.1 in exome sequencing data) calculated using one-tailed hypergeometric tests. IGF2BP1-3 indicates the combined target list. Vertical dashed line indicates p < 0.05/4 (adjusting for four target lists). The number of genes in the overlap is shown to the right of each bar.(B) Common variant enrichment of IGF2BP1-3 targets derived using ASD GWAS data45 and MAGMA.46 Vertical dashed line indicates p < 0.05/4. The enrichment coefficient is shown to the right of each bar.(C) Overlap enrichment between IGF2BP1-3 targets and the ASD PPI network or index protein-specific sub-networks (compared with non-interactors) calculated using one-tailed hypergeometric tests. Gene counts in overlaps with p < 0.05 (∗) or p < 0.05/4 (∗∗) are labeled in the heatmap. Networks that contain any of the IGF2BPs are highlighted in blue on the x axis.(D) Volcano plot of an IP-MS experiment for DYRK1A in iNs, showing DYRK1A in red, significant interactors (log2 FC > 0 and FDR ≤ 0.1) in green, and non-interactors in blue; IGF2BP1-3, which were technically replicated by western blot, are marked in orange. Statistics were derived from two replicates using a two-tailed one-sample moderated t test.(E) Immunoblot on an DYRK1A IP with and without addition of 25 U of benzonase (indicated at the top). An asterisk marks the expected band of each protein (named on the right). L, ladder; IN, input; IP, immunoprecipitation; IgG, IgG control. Molecular weights are in kilodaltons.(F) Immunoblot showing protein levels (named on the right) upon DYRK1A siRNA-mediated knockdown for 48 h in iPSCs. B-actin is used as loading control. FC was calculated as a ratio of the intensity of the bands detected in the DYRK1A knockdown (KD) versus WT lanes across three biological replicates.DYRK1A is a ubiquitously expressed protein kinase encoded by a gene localized in the Down syndrome critical region of chromosome 21 and is associated with learning disabilities in individuals with Down syndrome and ASDs.47,48 It has also been implicated in post-transcriptional regulation of cellular processes involved in brain development and function.49 Given the enriched overlap observed between the IGF2BP1-3 targets and the DYRK1A network, we hypothesized that DYRK1A might be part of an ASD-relevant feedback circuitry acting upstream of the IGF2BP1-3 complex. We thus verified that the interactions between DYRK1A and IGF2BP1-3 identified by IP-MS (Figure 6D) are not mediated by nucleic acids (Figure 6E), indicating a direct regulatory relationship. We also observed that partial knockdown (30%) with siRNA of DYRK1A in iPSCs was sufficient to trigger increased expression of IGF2BP1-3 (Figure 6F).Overall, our results suggest that the IGF2BP1-3 complex regulates an ASD-relevant transcriptional circuitry, and that the identified interactions between DYRK1A and the complex might be key in modulating the expression of numerous ASD-associated genes.The network is enriched for rare variants associated with ASDs and developmental disordersTo evaluate whether the combined PPI network and index protein-specific sub-networks are enriched for genes associated with ASDs, we performed global and conditional enrichment analyses using rare variant association statistics derived from exome sequencing.5 In the global analysis, we tested whether the network genes are more significantly associated with ASDs compared with other protein-coding genes in the genome; in the conditional analysis, we compared the network genes against the non-interactors detected in iNs. Five networks reached significance (p < 0.05/14, adjusting for 14 networks) in the global analysis (Figure S11A), and only the combined network was significant in the conditional analysis (p = 1.1 × 10−5; Figure 7A; Table S9). Notably, the conditional enrichment was robust even after we removed known InWeb interactors from the combined network (p = 5.5 × 10−5; Figure S11B), indicating that this signal is not driven by the background proteome in iNs or by previously reported interactions. Rather, the neuron-specific interactions identified in this study are enriched for rare variants associated with ASDs.Figure 7Genetic enrichment in the ASD PPI network(A) Rare variant5,50,51 or pLI score52 enrichment of the network or index protein-specific sub-networks compared with non-interactors, derived using one-tailed Kolmogorov-Smirnov tests. The number of genes in each network is shown in parentheses on the y axis. Left and right vertical dashed lines indicate p < 0.05 and p < 0.05/14 (adjusting for 14 networks), respectively; results passing these thresholds are labeled with the corresponding KS test statistics.(B) Social Manhattan plot of index genes (red) and interactors that are suggestive ASD-associated genes (blue or cyan for FDR ≤ 0.1 or 0.25, respectively) in exome sequencing data.5 Sizes of the interactor nodes scale with the number of linked index genes. Connecting lines indicate observed interactions in the ASD PPI network; interactions replicated by IP-WB are highlighted in orange.We also performed analogous enrichment analyses using exome sequencing data for developmental disorders50 (DDs) and schizophrenia51 (SCZ) and gnomAD52 pLI scores, which represent the probability of a gene being loss-of-function intolerant (Figures 7A, S11A, and S11B; Table S9). In the conditional analysis, we observed that the combined network was significantly enriched for DD genes (p = 6.2 × 10−7) and genes with high pLI scores (p = 2.6 × 10−31); five index protein-specific sub-networks were also enriched for high pLI scores (p = 1.1 × 10−12 to 3.4 × 10−4). In contrast, there was no significant SCZ enrichment for any of the networks. These results indicate that the index protein interactors contribute to rare variants associated with both ASDs and DDs more than one would expect. The interactors are also more likely to be intolerant to loss-of-function mutations, suggesting their importance for cell fate specification or survival.To assess the common variant enrichment of the PPI networks, we performed global and conditional analyses using ASD GWAS data45 and MAGMA.46 Only the SYNGAP1 network came close to, but did not reach, significance in both the global and conditional analyses (conditional p = 3.71 × 10−3, which is just shy of the Bonferroni-corrected threshold of p < 0.05/14; Figure S11C; Table S10). We also repeated the analyses using GWAS data of attention deficit hyperactivity disorder53 (ADHD), bipolar disorder54 (BIP), major depressive disorder55 (MDD), and SCZ,56 as well as height57 as an additional control trait. None of the tested networks reached significance (Figure S11C; Table S10).Network analyses can prioritize additional ASD-associated genes from genetic studiesSince the PPI network is enriched for rare variants associated with ASDs, we used it to prioritize ASD-associated genes that did not reach stringent statistical significance in the exome sequencing study.5 Specifically, we generated a “social Manhattan plot” to highlight PPIs between the index genes and other ASD-associated genes with FDR ≤ 0.25 in the exome sequencing data (Figure 7B). Consistent with the rare variant enrichment results, the overlap between our network and these ASD-associated genes is significant (p = 4.9 × 10−4), with DYRK1A being linked to the most genes out of all the index genes. In total, we prioritized 32 suggestive ASD-associated genes that overlap with our network. Interestingly, these genes have higher average expression throughout cortical development compared with the other ASD-associated genes identified at the same significance threshold and have comparable expression with the exome-wide significant genes (Figure S11D). We successfully replicated 14 of the prioritized interactions using forward and/or reverse IPs followed by western blotting (Table S5), including multiple interactions involving SMARCC2, ELAVL3, DPYSL2, and DYNC1H1. While follow-up investigation on the prioritized genes is necessary, our analysis showcases how the ASD PPI network can be used to corroborate findings from genetic studies of ASDs.

ASD-associated genes and proteins are expressed in induced neuronsAn exome sequencing study5 that combined de novo and case-control rare variation data identified 24 high-confidence ASD-associated genes at exome-wide significance (Q ≤ 1.35 × 10−4). We used these 24 genes as the starting point of our workflow to generate an ASD PPI network in iNs (Figure 1A), and hereafter refer to them and their encoded proteins as index genes and index proteins, respectively.Figure 1Study workflow and expression of ASD index genes and proteins in human iNs(A) Workflow to generate and analyze a PPI network for high-confidence ASD-associated genes5 in human iNs.(B) Key steps in the glutamatergic-patterning protocol25 utilized to generate neural progenitor cells (NPCs) and excitatory iNs from human iPSCs.(C) Heatmap of ASD index gene expression over the course of neuronal maturation derived from RNA-seq data. CPM, mean normalized counts per million of two replicates; maximum log2(CPM) was capped at 9 for visualization.(D) Flowchart of index protein filtering.(E) Expression of the 13 index proteins in differentiating iPSCs, mouse cortex, and HEK293 cells detected by immunoblotting. The antibody utilized against ANK2 recognizes a human-specific epitope, and SYNGAP1 is not expressed in HEK293 cells. B-actin (B-ACT) was used as a loading control. Molecular weights (kDa) are marked on the left of each blot.In terms of the cell model for our experiments, we adopted and optimized a protocol25 that combines programming of the neurogenic factor Neurogenin 2 (NGN2) with developmental patterning to differentiate iPSCs into iNs. We used an iPSC line (iPS3, STAR Methods) where NGN2 is integrated in the genome under a tetON promoter for rapid and controlled doxycycline-induced expression, resulting in highly homogeneous iN populations25 (Figures 1B and S1A). We measured the expression of ASD-associated index genes through RNA sequencing (RNA-seq) of cells during weeks 0, 3, and 6 of neuronal differentiation (Figure 1C). The expression patterns of these genes varied across time, but most were expressed after 3 weeks of differentiation, including genes with previously reported neuronal identity.27 Importantly, gene expression observed in iNs as early as week 3 (Figure 1C) is consistent with presence of the same transcripts in excitatory neuron populations of the human cortex24 (Figures S1B and S1C), supporting this cell model as a proxy for cell types in the brain.Next, given that RNA-seq data can only capture the overall number of transcripts at steady state without accounting for the translation rate and turnover of the encoded protein products,28 we assessed expression of the index proteins using immunoblotting. We were unable to obtain antibodies to detect seven of the index proteins (Figure 1D; Table S1). Four proteins were successfully detected by immunoblotting (Figure S1D) but lacked IP-competent antibodies. The remaining 13 proteins (Figure 1D) were detected by immunoblotting in the form of at least one isoform at week 3 of differentiation (Figure 1E) and were amenable to IP. Many of these proteins showed distinct isoform patterns across species, neuronal differentiation time points, or cell types (Figure 1E). These observations highlight the importance of using human iNs to capture cell-type-specific biochemical interactions of relevance to ASDs.29,30

ASD-associated genes and proteins are expressed in induced neurons

An exome sequencing study5 that combined de novo and case-control rare variation data identified 24 high-confidence ASD-associated genes at exome-wide significance (Q ≤ 1.35 × 10−4). We used these 24 genes as the starting point of our workflow to generate an ASD PPI network in iNs (Figure 1A), and hereafter refer to them and their encoded proteins as index genes and index proteins, respectively.Figure 1Study workflow and expression of ASD index genes and proteins in human iNs(A) Workflow to generate and analyze a PPI network for high-confidence ASD-associated genes5 in human iNs.(B) Key steps in the glutamatergic-patterning protocol25 utilized to generate neural progenitor cells (NPCs) and excitatory iNs from human iPSCs.(C) Heatmap of ASD index gene expression over the course of neuronal maturation derived from RNA-seq data. CPM, mean normalized counts per million of two replicates; maximum log2(CPM) was capped at 9 for visualization.(D) Flowchart of index protein filtering.(E) Expression of the 13 index proteins in differentiating iPSCs, mouse cortex, and HEK293 cells detected by immunoblotting. The antibody utilized against ANK2 recognizes a human-specific epitope, and SYNGAP1 is not expressed in HEK293 cells. B-actin (B-ACT) was used as a loading control. Molecular weights (kDa) are marked on the left of each blot.

Study workflow and expression of ASD index genes and proteins in human iNs

(A) Workflow to generate and analyze a PPI network for high-confidence ASD-associated genes5 in human iNs.

(B) Key steps in the glutamatergic-patterning protocol25 utilized to generate neural progenitor cells (NPCs) and excitatory iNs from human iPSCs.

(C) Heatmap of ASD index gene expression over the course of neuronal maturation derived from RNA-seq data. CPM, mean normalized counts per million of two replicates; maximum log2(CPM) was capped at 9 for visualization.

(D) Flowchart of index protein filtering.

(E) Expression of the 13 index proteins in differentiating iPSCs, mouse cortex, and HEK293 cells detected by immunoblotting. The antibody utilized against ANK2 recognizes a human-specific epitope, and SYNGAP1 is not expressed in HEK293 cells. B-actin (B-ACT) was used as a loading control. Molecular weights (kDa) are marked on the left of each blot.

In terms of the cell model for our experiments, we adopted and optimized a protocol25 that combines programming of the neurogenic factor Neurogenin 2 (NGN2) with developmental patterning to differentiate iPSCs into iNs. We used an iPSC line (iPS3, STAR Methods) where NGN2 is integrated in the genome under a tetON promoter for rapid and controlled doxycycline-induced expression, resulting in highly homogeneous iN populations25 (Figures 1B and S1A). We measured the expression of ASD-associated index genes through RNA sequencing (RNA-seq) of cells during weeks 0, 3, and 6 of neuronal differentiation (Figure 1C). The expression patterns of these genes varied across time, but most were expressed after 3 weeks of differentiation, including genes with previously reported neuronal identity.27 Importantly, gene expression observed in iNs as early as week 3 (Figure 1C) is consistent with presence of the same transcripts in excitatory neuron populations of the human cortex24 (Figures S1B and S1C), supporting this cell model as a proxy for cell types in the brain.

Next, given that RNA-seq data can only capture the overall number of transcripts at steady state without accounting for the translation rate and turnover of the encoded protein products,28 we assessed expression of the index proteins using immunoblotting. We were unable to obtain antibodies to detect seven of the index proteins (Figure 1D; Table S1). Four proteins were successfully detected by immunoblotting (Figure S1D) but lacked IP-competent antibodies. The remaining 13 proteins (Figure 1D) were detected by immunoblotting in the form of at least one isoform at week 3 of differentiation (Figure 1E) and were amenable to IP. Many of these proteins showed distinct isoform patterns across species, neuronal differentiation time points, or cell types (Figure 1E). These observations highlight the importance of using human iNs to capture cell-type-specific biochemical interactions of relevance to ASDs.29,30

Interaction proteomics of ASD index proteins in induced neuronsFor the 13 index proteins with IP-competent antibodies, we executed IPs followed by mass spectrometry (IP-MS) in iNs at week 4 of differentiation (Figures 2A, 2B, and S2; Table S2). Each experiment was conducted in duplicates with ∼15 million cells per replicate. Protein abundances in the IP samples were quantified with labeled or label-free liquid chromatography followed by tandem mass spectrometry (LC-MS/MS). In total, we conducted 37 IP-MS experiments, generating two to five experimental datasets for each index protein.Figure 2Generation of a combined PPI network for 13 ASD index proteins in iNs(A) Example of an IP-MS experiment. Left: immunoblot of a SHANK3 IP with its main isoforms marked by a box and asterisk; L, ladder; IN, input; FT, flow-through; IP, immunoprecipitation; IgG, IgG control; H + L IgG, heavy and light IgG chains. Molecular weights are in kilodaltons. Right: log2 fold change (FC) correlation between IP-MS replicates.(B) Volcano plot of the SHANK3 IP-MS experiment, showing SHANK3 in red, its significant interactors (log2 FC > 0 and FDR ≤ 0.1) in green, and non-interactors in blue; known InWeb interactors identified as interactors or non-interactors in the experiment are highlighted in yellow or white, respectively. Statistics were derived from two replicates using a two-tailed one-sample moderated t test.(C) PPI network derived from 26 IP-MS experiments. Nodes represent index proteins (red) and their interactors (purple); color intensity and size of the interactor nodes scale with interactor frequency (i.e., number of linked index proteins). Edges indicate observed interactions, with known InWeb interactions highlighted in blue.(D) Distribution of interactor frequency in the network.(E) Distribution of InWeb versus newly reported interactions in the network.(F) Replication rates of interactions tested in forward or reverse IPs followed by western blotting (IP-WB).(G) Overlap between SCN2A, SHANK3, and SYNGAP1 interactors derived from iN versus brain IPs. Overlap enrichment p value was calculated using a one-tailed hypergeometric test that only considered proteins detected in both sample types.(H) Pairwise co-expression Z scores between index genes and their interactors (Int), non-interactors (NonInt), known InWeb interactors (InWeb), and all protein-coding genes (All) derived from a spatial transcriptomic dataset in human dorsolateral prefrontal cortex.31 Boxes and whiskers in violin plots indicate the interquartile range (IQR) and 1.5x IQR, respectively. Double asterisks indicate p < 0.05/6 (adjusting for six pairwise comparisons) as calculated by two-tailed Wilcoxon rank-sum tests. Number of gene pairs included for each gene type is indicated toward the bottom.For each IP-MS experiment, we performed quality control (QC) and data analysis using Genoppi.18 Specifically, we calculated the log2 fold change (FC) and corresponding statistical significance of each protein identified in the index protein IPs compared with control IPs (Table S2). Eleven datasets failed to pass QC thresholds (i.e., log2 FC correlation between replicates was ≤ 0.6 or the index protein was not enriched at false discovery rate [FDR] ≤ 0.1), leaving us with 26 high-quality datasets. We defined proteins with log2 FC > 0 and FDR ≤ 0.1 as significant interactors of the index protein; other detected proteins were defined as non-interactors (Table S3). We also intersected each dataset with known interactors of the index protein, as determined from the InWeb database,17,18 which aggregates human PPI data from >40,000 publications (Table S3). This allowed us to assess whether the identified interactions have been reported.Our analysis workflow is illustrated with an IP of SHANK3. Enrichment of SHANK3 in the IP was confirmed by immunoblotting (Figure 2A, left) and quantified by IP-MS (log2 FC = 1.2 and FDR = 0.012; Figure 2B). The IP-MS replicate log2 FC correlation was 0.79 (Figure 2A, right). Only two out of 104 significant interactors identified here were reported (Figure 2B). Overall, the enrichment of SHANK3 in both immunoblot and MS results, in combination with the correlation between replicates, support the reproducibility and robustness of our data. The low number of previously identified SHANK3 interactors likely reflects our experimental design (Data S1) and supports the use of neuron-specific proteomics to discover new gene relationships.Since the IP-MS datasets were generated across different MS facilities, we compared the data between facilities and between IPs of the same index protein to evaluate their agreement (Data S2; Figure S3; Table S4). While we observed differences in data structures and detected proteins across the facilities, the identified PPIs have similar quality and degree of biological relevance regardless of facility origin, suggesting these data are complementary. Moreover, we did not observe systematic differences between interactors identified in only one IP versus in multiple IPs of the same index protein (Data S2), or between interactors detected across different FC ranges (Data S3). Therefore, we included all identified interactors in downstream analyses.

Interaction proteomics of ASD index proteins in induced neurons

For the 13 index proteins with IP-competent antibodies, we executed IPs followed by mass spectrometry (IP-MS) in iNs at week 4 of differentiation (Figures 2A, 2B, and S2; Table S2). Each experiment was conducted in duplicates with ∼15 million cells per replicate. Protein abundances in the IP samples were quantified with labeled or label-free liquid chromatography followed by tandem mass spectrometry (LC-MS/MS). In total, we conducted 37 IP-MS experiments, generating two to five experimental datasets for each index protein.Figure 2Generation of a combined PPI network for 13 ASD index proteins in iNs(A) Example of an IP-MS experiment. Left: immunoblot of a SHANK3 IP with its main isoforms marked by a box and asterisk; L, ladder; IN, input; FT, flow-through; IP, immunoprecipitation; IgG, IgG control; H + L IgG, heavy and light IgG chains. Molecular weights are in kilodaltons. Right: log2 fold change (FC) correlation between IP-MS replicates.(B) Volcano plot of the SHANK3 IP-MS experiment, showing SHANK3 in red, its significant interactors (log2 FC > 0 and FDR ≤ 0.1) in green, and non-interactors in blue; known InWeb interactors identified as interactors or non-interactors in the experiment are highlighted in yellow or white, respectively. Statistics were derived from two replicates using a two-tailed one-sample moderated t test.(C) PPI network derived from 26 IP-MS experiments. Nodes represent index proteins (red) and their interactors (purple); color intensity and size of the interactor nodes scale with interactor frequency (i.e., number of linked index proteins). Edges indicate observed interactions, with known InWeb interactions highlighted in blue.(D) Distribution of interactor frequency in the network.(E) Distribution of InWeb versus newly reported interactions in the network.(F) Replication rates of interactions tested in forward or reverse IPs followed by western blotting (IP-WB).(G) Overlap between SCN2A, SHANK3, and SYNGAP1 interactors derived from iN versus brain IPs. Overlap enrichment p value was calculated using a one-tailed hypergeometric test that only considered proteins detected in both sample types.(H) Pairwise co-expression Z scores between index genes and their interactors (Int), non-interactors (NonInt), known InWeb interactors (InWeb), and all protein-coding genes (All) derived from a spatial transcriptomic dataset in human dorsolateral prefrontal cortex.31 Boxes and whiskers in violin plots indicate the interquartile range (IQR) and 1.5x IQR, respectively. Double asterisks indicate p < 0.05/6 (adjusting for six pairwise comparisons) as calculated by two-tailed Wilcoxon rank-sum tests. Number of gene pairs included for each gene type is indicated toward the bottom.

Generation of a combined PPI network for 13 ASD index proteins in iNs

(A) Example of an IP-MS experiment. Left: immunoblot of a SHANK3 IP with its main isoforms marked by a box and asterisk; L, ladder; IN, input; FT, flow-through; IP, immunoprecipitation; IgG, IgG control; H + L IgG, heavy and light IgG chains. Molecular weights are in kilodaltons. Right: log2 fold change (FC) correlation between IP-MS replicates.

(B) Volcano plot of the SHANK3 IP-MS experiment, showing SHANK3 in red, its significant interactors (log2 FC > 0 and FDR ≤ 0.1) in green, and non-interactors in blue; known InWeb interactors identified as interactors or non-interactors in the experiment are highlighted in yellow or white, respectively. Statistics were derived from two replicates using a two-tailed one-sample moderated t test.

(C) PPI network derived from 26 IP-MS experiments. Nodes represent index proteins (red) and their interactors (purple); color intensity and size of the interactor nodes scale with interactor frequency (i.e., number of linked index proteins). Edges indicate observed interactions, with known InWeb interactions highlighted in blue.

(D) Distribution of interactor frequency in the network.

(E) Distribution of InWeb versus newly reported interactions in the network.

(F) Replication rates of interactions tested in forward or reverse IPs followed by western blotting (IP-WB).

(G) Overlap between SCN2A, SHANK3, and SYNGAP1 interactors derived from iN versus brain IPs. Overlap enrichment p value was calculated using a one-tailed hypergeometric test that only considered proteins detected in both sample types.

(H) Pairwise co-expression Z scores between index genes and their interactors (Int), non-interactors (NonInt), known InWeb interactors (InWeb), and all protein-coding genes (All) derived from a spatial transcriptomic dataset in human dorsolateral prefrontal cortex.31 Boxes and whiskers in violin plots indicate the interquartile range (IQR) and 1.5x IQR, respectively. Double asterisks indicate p < 0.05/6 (adjusting for six pairwise comparisons) as calculated by two-tailed Wilcoxon rank-sum tests. Number of gene pairs included for each gene type is indicated toward the bottom.

For each IP-MS experiment, we performed quality control (QC) and data analysis using Genoppi.18 Specifically, we calculated the log2 fold change (FC) and corresponding statistical significance of each protein identified in the index protein IPs compared with control IPs (Table S2). Eleven datasets failed to pass QC thresholds (i.e., log2 FC correlation between replicates was ≤ 0.6 or the index protein was not enriched at false discovery rate [FDR] ≤ 0.1), leaving us with 26 high-quality datasets. We defined proteins with log2 FC > 0 and FDR ≤ 0.1 as significant interactors of the index protein; other detected proteins were defined as non-interactors (Table S3). We also intersected each dataset with known interactors of the index protein, as determined from the InWeb database,17,18 which aggregates human PPI data from >40,000 publications (Table S3). This allowed us to assess whether the identified interactions have been reported.

Our analysis workflow is illustrated with an IP of SHANK3. Enrichment of SHANK3 in the IP was confirmed by immunoblotting (Figure 2A, left) and quantified by IP-MS (log2 FC = 1.2 and FDR = 0.012; Figure 2B). The IP-MS replicate log2 FC correlation was 0.79 (Figure 2A, right). Only two out of 104 significant interactors identified here were reported (Figure 2B). Overall, the enrichment of SHANK3 in both immunoblot and MS results, in combination with the correlation between replicates, support the reproducibility and robustness of our data. The low number of previously identified SHANK3 interactors likely reflects our experimental design (Data S1) and supports the use of neuron-specific proteomics to discover new gene relationships.

Since the IP-MS datasets were generated across different MS facilities, we compared the data between facilities and between IPs of the same index protein to evaluate their agreement (Data S2; Figure S3; Table S4). While we observed differences in data structures and detected proteins across the facilities, the identified PPIs have similar quality and degree of biological relevance regardless of facility origin, suggesting these data are complementary. Moreover, we did not observe systematic differences between interactors identified in only one IP versus in multiple IPs of the same index protein (Data S2), or between interactors detected across different FC ranges (Data S3). Therefore, we included all identified interactors in downstream analyses.

The PPI network consists of convergent, novel, and reproducible interactionsWe merged the 26 high-quality IP-MS datasets to generate a combined PPI network (Figure 2C; Table S3). This network comprises 1,021 interactors of the 13 index proteins, with >20% of them being linked to multiple index proteins (Figures 2D and S4A). Although the interactors of several index proteins significantly overlap with reported InWeb interactors (Table S3), >90% of all interactions in the network have not yet been reported at the time of this study (Figure 2E). This was not unexpected given the limited number of neuronal PPI datasets in the literature (Data S1) and suggests an opportunity for biological discovery linked to the characterization of these interactions.To support the robustness of MS-detected interactions, we validated the technical reproducibility of both unique (i.e., interacting with only one index protein) and shared (i.e., interacting with multiple index proteins) interactors in independent IPs by immunoblotting (Figures S4B–S4D; Table S5). Overall, we validated 65 out of 71 tested interactions (91.5% replication rate; Figure 2F, left), according to criteria detailed in STAR Methods. In reverse IPs, performed with a panel of interactors as baits, we detected the original index proteins in 19 out of 23 IPs with bait enrichment (82.6% replication rate; Figure 2F, right, Figures S5 and S6; Table S5). These results are in line with the expected replication rate of ∼90% (from the FDR ≤ 0.1 threshold used to define interactors), and with previously reported replication rates in similar experimental designs.18

The PPI network consists of convergent, novel, and reproducible interactions

We merged the 26 high-quality IP-MS datasets to generate a combined PPI network (Figure 2C; Table S3). This network comprises 1,021 interactors of the 13 index proteins, with >20% of them being linked to multiple index proteins (Figures 2D and S4A). Although the interactors of several index proteins significantly overlap with reported InWeb interactors (Table S3), >90% of all interactions in the network have not yet been reported at the time of this study (Figure 2E). This was not unexpected given the limited number of neuronal PPI datasets in the literature (Data S1) and suggests an opportunity for biological discovery linked to the characterization of these interactions.

To support the robustness of MS-detected interactions, we validated the technical reproducibility of both unique (i.e., interacting with only one index protein) and shared (i.e., interacting with multiple index proteins) interactors in independent IPs by immunoblotting (Figures S4B–S4D; Table S5). Overall, we validated 65 out of 71 tested interactions (91.5% replication rate; Figure 2F, left), according to criteria detailed in STAR Methods. In reverse IPs, performed with a panel of interactors as baits, we detected the original index proteins in 19 out of 23 IPs with bait enrichment (82.6% replication rate; Figure 2F, right, Figures S5 and S6; Table S5). These results are in line with the expected replication rate of ∼90% (from the FDR ≤ 0.1 threshold used to define interactors), and with previously reported replication rates in similar experimental designs.18

The PPI network recapitulates interactions occurring in the human brainTo compare our data with PPIs in the human brain, we performed IP-MS experiments in postmortem brain homogenates for a subset of index proteins, focusing on proteins with known neuronal function due to limited materials (Table S2). On average, brain IP-MS datasets had lower log2 FC correlations between replicates compared with the iN data (average correlation = 0.63 and 0.72 for brain and iN IPs, respectively). The lower reproducibility likely reflects noise and artifacts. After QC, we considered SCN2A, SHANK3, and SYNGAP1 brain IPs to be high-quality and compared them against the analogous iN IPs. Considering that brain samples contain a mix of cells, whereas iNs are highly homogeneous, we observed agreement in terms of log2 FC correlation for proteins detected in both sample types (median correlation = 0.517 between brain versus iN IPs, compared with median = 0.628 among iN IPs; Figure S7A). Furthermore, 42% of the brain interactors were also identified as interactors in iNs (p = 2.6 × 10−13; Figures 2G, S7B, and S7C). Throughout brain development32 (www.brainspan.org), both brain- and iN-derived interactors have elevated gene expression in the cortex compared with random genes; interactors identified in both sample types have expression profiles comparable with the ASD-associated genes identified through exome sequencing (Figures S8A and S8B).We also characterized the co-expression patterns of interacting proteins in the iN-derived PPI network across four independent datasets derived from human or mouse brains with spatial transcriptomic,31,33 single-cell RNA-seq24 (scRNA-seq), or bulk RNA-seq (www.brainspan.org) approaches (Figures 2H and S8C). In general, index proteins have higher co-expression with their interactors compared with non-interactors detected in iNs during IP-MS, InWeb interactors from mostly non-neuronal contexts, and all protein-coding genes. Therefore, both the IP-MS data from brain homogenates and the brain co-expression analysis indicate that we captured gene relationships found in complex tissues in the human brain.

The PPI network recapitulates interactions occurring in the human brain

To compare our data with PPIs in the human brain, we performed IP-MS experiments in postmortem brain homogenates for a subset of index proteins, focusing on proteins with known neuronal function due to limited materials (Table S2). On average, brain IP-MS datasets had lower log2 FC correlations between replicates compared with the iN data (average correlation = 0.63 and 0.72 for brain and iN IPs, respectively). The lower reproducibility likely reflects noise and artifacts. After QC, we considered SCN2A, SHANK3, and SYNGAP1 brain IPs to be high-quality and compared them against the analogous iN IPs. Considering that brain samples contain a mix of cells, whereas iNs are highly homogeneous, we observed agreement in terms of log2 FC correlation for proteins detected in both sample types (median correlation = 0.517 between brain versus iN IPs, compared with median = 0.628 among iN IPs; Figure S7A). Furthermore, 42% of the brain interactors were also identified as interactors in iNs (p = 2.6 × 10−13; Figures 2G, S7B, and S7C). Throughout brain development32 (www.brainspan.org), both brain- and iN-derived interactors have elevated gene expression in the cortex compared with random genes; interactors identified in both sample types have expression profiles comparable with the ASD-associated genes identified through exome sequencing (Figures S8A and S8B).

We also characterized the co-expression patterns of interacting proteins in the iN-derived PPI network across four independent datasets derived from human or mouse brains with spatial transcriptomic,31,33 single-cell RNA-seq24 (scRNA-seq), or bulk RNA-seq (www.brainspan.org) approaches (Figures 2H and S8C). In general, index proteins have higher co-expression with their interactors compared with non-interactors detected in iNs during IP-MS, InWeb interactors from mostly non-neuronal contexts, and all protein-coding genes. Therefore, both the IP-MS data from brain homogenates and the brain co-expression analysis indicate that we captured gene relationships found in complex tissues in the human brain.

The PPI network implicates layer 2/3 cortical excitatory neurons in ASDsWe assessed the tissue specificity of the PPI network by calculating its overlap enrichment with GTEx tissue-specific genes34 compared with the rest of the genome. The network showed significant enrichment (p < 0.05/53, adjusting for 53 tissues) in the central nervous system (CNS) and several other tissue categories (Figure 3A and Table S6). In contrast, an analogous InWeb-based network showed less pronounced enrichment for the CNS (Figure S8D). When repeated with brain region-specific genes, the network was most enriched (p < 0.05/13, adjusting for 13 brain regions) in the frontal cortex (Figure 3B; Table S6). In addition, we performed SynGO35 gene set analysis, which confirmed that the network is enriched for genes involved in biological processes in the synapse (Figure S8E; Table S6).Figure 3Tissue and cell type enrichment of the ASD PPI network(A–D) Overlap enrichment between the network and GTEx tissue-specific genes34 ([A]; 2,408 genes per tissue), GTEx brain region-specific genes34 ([B]; 2,408 genes per tissue), genes expressed in >50% of cells in each cell type in the postmortem cortex24 ([C]; gene counts in parentheses), and DEGs in each cell type in the postmortem cortex of ASD individuals versus controls24 ([D]; gene counts in parentheses). The p values were derived from one-tailed hypergeometric tests; for (A) and (B), a global background (i.e., all genes in the GTEx data) was used as the population in the test; for (C) and (D), a conditional neuronal background (i.e., iN-expressed genes detected by IP-MS) was used. Left and right vertical dashed lines indicate p < 0.05 and p < 0.05/number of tissues or cell types, respectively; in (B)–(D), results passing these thresholds are labeled with the number of genes in the overlap. AST-FB and AST-PP, fibrous and protoplasmic astrocytes; OPC, oligodendrocyte precursor cells; IN-PV, IN-SST, IN-SV2C, and IN-VIP, parvalbumin, somatostatin, SV2C, and VIP interneurons; L2/3 and L4, layer 2/3 and layer 4 excitatory neurons; L5/6 and L5/6-CC, layer 5/6 corticofugal projection and cortico-cortical projection neurons; Neu-mat, maturing neurons; Neu-NRGN-I and Neu-NRGN-II, NRGN-expression neurons.To link the network to specific cell types implicated in ASDs, we integrated our data with a scRNA-seq study,24 which measured gene expression in the postmortem cortex of individuals with ASDs versus controls and identified differentially expressed genes (DEGs) between the groups in 17 annotated cell types. First, to test whether the network is enriched for commonly expressed genes in each cell type (i.e., expressed in >50% of cells), we performed a global analysis to compare the network genes with other genes in the genome, as well as a conditional analysis to compare the network genes with other iN-expressed genes (i.e., non-interactors detected in IP-MS; Table S3). In both analyses, the network showed significant enrichment (p < 0.05/17, adjusting for 17 cell types) in several neuronal cell types, including both excitatory neurons and inhibitory interneurons (Figures 3C, and S9A; Table S6). Next, we tested the enrichment of the network in terms of its overlap with cell-type-specific DEGs in individuals with ASDs compared with controls. In the global analysis, the network showed enrichment in several cell types, including excitatory neurons, inhibitory interneurons, astrocytes, and microglia (Figure S9B; Table S6). In the more conservative conditional analysis, the layer 2/3 excitatory neurons stood out as the only significant cell type (p < 0.05/17; Figure 3D; Table S6). This enrichment signal is mostly driven by DEGs that were upregulated in individuals with ASDs (Figures S9C and S9D; Table S6).Taken together, these results indicate that the PPI network captures biochemical perturbations and pathways in ASD-relevant tissues and cell types, and specifically implicate layer 2/3 excitatory neurons as a key cell type in ASDs.

The PPI network implicates layer 2/3 cortical excitatory neurons in ASDs

We assessed the tissue specificity of the PPI network by calculating its overlap enrichment with GTEx tissue-specific genes34 compared with the rest of the genome. The network showed significant enrichment (p < 0.05/53, adjusting for 53 tissues) in the central nervous system (CNS) and several other tissue categories (Figure 3A and Table S6). In contrast, an analogous InWeb-based network showed less pronounced enrichment for the CNS (Figure S8D). When repeated with brain region-specific genes, the network was most enriched (p < 0.05/13, adjusting for 13 brain regions) in the frontal cortex (Figure 3B; Table S6). In addition, we performed SynGO35 gene set analysis, which confirmed that the network is enriched for genes involved in biological processes in the synapse (Figure S8E; Table S6).Figure 3Tissue and cell type enrichment of the ASD PPI network(A–D) Overlap enrichment between the network and GTEx tissue-specific genes34 ([A]; 2,408 genes per tissue), GTEx brain region-specific genes34 ([B]; 2,408 genes per tissue), genes expressed in >50% of cells in each cell type in the postmortem cortex24 ([C]; gene counts in parentheses), and DEGs in each cell type in the postmortem cortex of ASD individuals versus controls24 ([D]; gene counts in parentheses). The p values were derived from one-tailed hypergeometric tests; for (A) and (B), a global background (i.e., all genes in the GTEx data) was used as the population in the test; for (C) and (D), a conditional neuronal background (i.e., iN-expressed genes detected by IP-MS) was used. Left and right vertical dashed lines indicate p < 0.05 and p < 0.05/number of tissues or cell types, respectively; in (B)–(D), results passing these thresholds are labeled with the number of genes in the overlap. AST-FB and AST-PP, fibrous and protoplasmic astrocytes; OPC, oligodendrocyte precursor cells; IN-PV, IN-SST, IN-SV2C, and IN-VIP, parvalbumin, somatostatin, SV2C, and VIP interneurons; L2/3 and L4, layer 2/3 and layer 4 excitatory neurons; L5/6 and L5/6-CC, layer 5/6 corticofugal projection and cortico-cortical projection neurons; Neu-mat, maturing neurons; Neu-NRGN-I and Neu-NRGN-II, NRGN-expression neurons.

Tissue and cell type enrichment of the ASD PPI network

(A–D) Overlap enrichment between the network and GTEx tissue-specific genes34 ([A]; 2,408 genes per tissue), GTEx brain region-specific genes34 ([B]; 2,408 genes per tissue), genes expressed in >50% of cells in each cell type in the postmortem cortex24 ([C]; gene counts in parentheses), and DEGs in each cell type in the postmortem cortex of ASD individuals versus controls24 ([D]; gene counts in parentheses). The p values were derived from one-tailed hypergeometric tests; for (A) and (B), a global background (i.e., all genes in the GTEx data) was used as the population in the test; for (C) and (D), a conditional neuronal background (i.e., iN-expressed genes detected by IP-MS) was used. Left and right vertical dashed lines indicate p < 0.05 and p < 0.05/number of tissues or cell types, respectively; in (B)–(D), results passing these thresholds are labeled with the number of genes in the overlap. AST-FB and AST-PP, fibrous and protoplasmic astrocytes; OPC, oligodendrocyte precursor cells; IN-PV, IN-SST, IN-SV2C, and IN-VIP, parvalbumin, somatostatin, SV2C, and VIP interneurons; L2/3 and L4, layer 2/3 and layer 4 excitatory neurons; L5/6 and L5/6-CC, layer 5/6 corticofugal projection and cortico-cortical projection neurons; Neu-mat, maturing neurons; Neu-NRGN-I and Neu-NRGN-II, NRGN-expression neurons.

To link the network to specific cell types implicated in ASDs, we integrated our data with a scRNA-seq study,24 which measured gene expression in the postmortem cortex of individuals with ASDs versus controls and identified differentially expressed genes (DEGs) between the groups in 17 annotated cell types. First, to test whether the network is enriched for commonly expressed genes in each cell type (i.e., expressed in >50% of cells), we performed a global analysis to compare the network genes with other genes in the genome, as well as a conditional analysis to compare the network genes with other iN-expressed genes (i.e., non-interactors detected in IP-MS; Table S3). In both analyses, the network showed significant enrichment (p < 0.05/17, adjusting for 17 cell types) in several neuronal cell types, including both excitatory neurons and inhibitory interneurons (Figures 3C, and S9A; Table S6). Next, we tested the enrichment of the network in terms of its overlap with cell-type-specific DEGs in individuals with ASDs compared with controls. In the global analysis, the network showed enrichment in several cell types, including excitatory neurons, inhibitory interneurons, astrocytes, and microglia (Figure S9B; Table S6). In the more conservative conditional analysis, the layer 2/3 excitatory neurons stood out as the only significant cell type (p < 0.05/17; Figure 3D; Table S6). This enrichment signal is mostly driven by DEGs that were upregulated in individuals with ASDs (Figures S9C and S9D; Table S6).

Taken together, these results indicate that the PPI network captures biochemical perturbations and pathways in ASD-relevant tissues and cell types, and specifically implicate layer 2/3 excitatory neurons as a key cell type in ASDs.

Dissecting cell-type-specific PPIs of giant ANK2 and PTEN in maturing neuronsTo illustrate that many of the PPIs in our network reflect neuron-specific functions of the ASD index proteins, we further dissected the interactions observed for ANK2 and PTEN. Giant ankyrin-2 is a neuron-specific alternatively spliced variant of ANK2.29,36 This isoform is differentiated from canonical ANK2 due to inclusion of exon 37 (giant exon), which encodes 9,240 amino acids and contains the majority of ANK2 mutations identified in individuals with autism (Figure 4A). While we technically validated many of the identified ANK2 interactors (Figures 4B and S10A; Table S5), they are likely a mix of interactors of all ANK2 isoforms. Thus, to focus specifically on the ASD-associated giant ANK2 isoform, we built an in vitro model of selective loss of giant ANK2 using CRISPR-Cas9 gene editing (Figure 4C).Figure 4Giant ANK2 is responsible for neuron-specific PPIs(A) Depiction of the ANK2 gene with ASD-associated mutations leading to splicing defects or early decay. Giant ANK2 includes exon 37, which is alternatively spliced to produce short ANK2. The epitope recognized by the ANK2 antibody used in this study is marked with an asterisk.(B) Volcano plot of an IP-MS experiment for ANK2 in iNs, showing ANK2 in red, significant interactors (log2 FC > 0 and FDR ≤ 0.1) in green, and non-interactors in blue; known InWeb interactors identified as interactors or non-interactors are highlighted in yellow or white, respectively; interactors replicated by immunoblotting are highlighted in orange. Statistics were derived from two replicates using a two-tailed one-sample moderated t test.(C) Experimental workflow to identify giant ANK2-specific PPIs.(D) Top: CRISPR-Cas9 editing strategy to generate a cell line exclusively expressing the short ANK2 isoform, obtained by deletion of the giant ANK2 exon splicing acceptor. Bottom: validation of giant ANK2 KO cell line. The deletion was confirmed by PCR on genomic DNA using primers flanking the deletion site (left), and by immunoblotting (right) showing the absence of the giant ANK2 isoform in homozygous KO iNs (−/−) compared with WT (+/+) and heterozygous KO (−/−) cells. Giant and canonical ANK2 isoforms are indicated with an asterisk.(E) C3 and CACNA2D1 western blots on ANK2 IPs in WT and giant ANK2 KO iNs. FC was calculated as a ratio of the intensity of the bands detected in the IP versus IN lanes. L, ladder; IN, input; IP, immunoprecipitation. Molecular weights are in kilodaltons. Fifty micrograms of total lysate were loaded in the IN lanes, and 10% of the total immunoprecipitates were loaded in the IP lanes.We edited an iPSC line (iPS3, STAR Methods) to exclude retention of exon 37, which differentiates giant ANK2 from canonical (short) ANK2 isoforms (Figure 4D, top). Fully characterized giant ANK2 knockout (KO) iPSCs (Figure 4D, bottom) were viable, but showed morphological defects, defective ANK2 distribution, and high cell death upon neuronal differentiation. We could not collect sufficient giant ANK2 KO iNs for IP-MS due to high cell death. Nonetheless, we performed immunoblot analysis in KO iNs to test for presence of ANK2 interactors identified by IP-MS in corresponding wild-type (WT) neurons. KO iNs were still expressing proteins essential for neuronal physiology and synaptic function, including the calcium channel subunit CACNA2D137 and the complement protein C3,38 but these proteins were no longer associated with ANK2 (Figure 4E), suggesting that their localization might rely on giant ANK2-dependent cytoskeletal architecture.We also performed IP-MS experiments of ANK2 in isogenic WT and giant ANK2 KO neural progenitor cells (NPCs; Figure S10B; Table S2). ANK2 interactors identified exclusively in the WT cells (which expressed both canonical and giant ANK2 isoforms) were most enriched for “glutamatergic synapses” in Gene Ontology (GO) analysis.39,40 In contrast, interactors specific to the KO cells (which lacked giant ANK2) were enriched for “microtubules” (Table S7). This suggests that the giant ANK2 isoform may have a role in localizing the protein to form cell projections of maturing NPCs and in guiding neuronal morphology. Accordingly, ARPC5L, which is essential for both structural and functional maturation of the growth cone in maturating neurons,41 is among the WT-specific ANK2 interactors (Figure S10B).We next investigated a robust interaction identified between PTEN and the A-kinase anchor protein AKAP8L, which was consistently replicated by immunoblotting and reverse IP (Figure 5A, 5B, and S10C). AKAP8L is implicated in the mammalian target of rapamycin (mTOR) pathway,42 but has not been linked to PTEN in previous biochemical studies (which were mostly conducted in cancer cell models). Thus, our findings in iNs suggest a neuron-specific mTOR circuitry of PTEN that acts through AKAP8L. To test this hypothesis, we generated AKAP8L KO iPSC lines through CRISPR-Cas9 gene editing (Figure 5C). We obtained isogenic lines with both heterozygous and homozygous mutations, with the homozygous KO line resulting in complete loss of AKAP8L (Figure 5D). Interestingly, the AKAP8L homozygous KO line showed an increase in growth rate at the NPC stage compared with isogenic WT and heterozygous lines (p = 0.0287 and 0.0162, respectively; Figure 5E).Figure 5The neuronal PTEN-AKAP8L interaction regulates cell proliferation(A) Volcano plot of an IP-MS experiment for PTEN in iNs, showing PTEN in red, significant interactors (log2 FC > 0 and FDR ≤ 0.1) in green, and non-interactors in blue; AKAP8L is marked in orange. Statistics were derived from two replicates using a two-tailed one-sample moderated t test.(B) AKAP8L immunoblot on the PTEN IP. AKAP8L is marked with an asterisk, IgG heavy chains are marked to the right; L, ladder; IN, input; FT, flow-through; IP, immunoprecipitation; IgG, IgG control. Molecular weights are in kilodaltons.(C) Schematic of the CRISPR-Cas9 editing strategy to generate AKAP8L KO iPSC lines.(D) Immunoblot validation of AKAP8L heterozygous (+/−) and homozygous (−/−) KO lines generated using the strategy in (C). B-actin is used as a loading control. Molecular weights (kDa) are marked on the side of each blot.(E) WT (+/+), AKAP8L heterozygous KO (+/−), and AKAP8L homozygous KO (−/−) neural progenitor cell growth over 4 days upon seeding cells at identical confluences normalized to their seeding density. Error bars indicate standard deviations of the mean of three biological replicates. Asterisks indicate p < 0.05 as calculated using a two-tailed t test.(F) Immunoblot analysis of PTEN, phosphorylated S6 (P-S6), S6, and AKT in WT and AKAP8L homozygous KO iNs. Molecular weights (kDa) are marked on the side of each blot and B-actin is used as loading control. FC was calculated as a ratio of the intensity of the bands detected in the AKAP8L KO (−/−) versus WT (+/+) lanes.Although AKAP8L KO did not result in changes of PTEN levels in iNs as measured by immunoblotting, we observed a small but detectable increase in the levels of phosphorylated ribosomal protein S6 (Figure 5F). This is a marker of hyperactivation of the mTOR pathway and suggests an AKAP8L-mediated defect in the PTEN signaling cascade in ASDs. Furthermore, we detected similar levels of the serine/threonine-protein kinase AKT in WT versus KO cells, suggesting that AKAP8L acts in the mTORC1 but not the mTORC2 pathway.43 These results support a neuronal PTEN-mTOR circuitry that includes AKAP8L and suggest that the PTEN-AKAP8L interaction affects downstream mTORC1 signaling and cell proliferation without affecting PTEN protein levels directly (Figures S10C and S10D).Our investigation of giant ANK2 and PTEN interactions demonstrate the potential of using cell-type-specific PPIs to gain mechanistic insights into ASD-relevant molecular cascades.

Dissecting cell-type-specific PPIs of giant ANK2 and PTEN in maturing neurons

To illustrate that many of the PPIs in our network reflect neuron-specific functions of the ASD index proteins, we further dissected the interactions observed for ANK2 and PTEN. Giant ankyrin-2 is a neuron-specific alternatively spliced variant of ANK2.29,36 This isoform is differentiated from canonical ANK2 due to inclusion of exon 37 (giant exon), which encodes 9,240 amino acids and contains the majority of ANK2 mutations identified in individuals with autism (Figure 4A). While we technically validated many of the identified ANK2 interactors (Figures 4B and S10A; Table S5), they are likely a mix of interactors of all ANK2 isoforms. Thus, to focus specifically on the ASD-associated giant ANK2 isoform, we built an in vitro model of selective loss of giant ANK2 using CRISPR-Cas9 gene editing (Figure 4C).Figure 4Giant ANK2 is responsible for neuron-specific PPIs(A) Depiction of the ANK2 gene with ASD-associated mutations leading to splicing defects or early decay. Giant ANK2 includes exon 37, which is alternatively spliced to produce short ANK2. The epitope recognized by the ANK2 antibody used in this study is marked with an asterisk.(B) Volcano plot of an IP-MS experiment for ANK2 in iNs, showing ANK2 in red, significant interactors (log2 FC > 0 and FDR ≤ 0.1) in green, and non-interactors in blue; known InWeb interactors identified as interactors or non-interactors are highlighted in yellow or white, respectively; interactors replicated by immunoblotting are highlighted in orange. Statistics were derived from two replicates using a two-tailed one-sample moderated t test.(C) Experimental workflow to identify giant ANK2-specific PPIs.(D) Top: CRISPR-Cas9 editing strategy to generate a cell line exclusively expressing the short ANK2 isoform, obtained by deletion of the giant ANK2 exon splicing acceptor. Bottom: validation of giant ANK2 KO cell line. The deletion was confirmed by PCR on genomic DNA using primers flanking the deletion site (left), and by immunoblotting (right) showing the absence of the giant ANK2 isoform in homozygous KO iNs (−/−) compared with WT (+/+) and heterozygous KO (−/−) cells. Giant and canonical ANK2 isoforms are indicated with an asterisk.(E) C3 and CACNA2D1 western blots on ANK2 IPs in WT and giant ANK2 KO iNs. FC was calculated as a ratio of the intensity of the bands detected in the IP versus IN lanes. L, ladder; IN, input; IP, immunoprecipitation. Molecular weights are in kilodaltons. Fifty micrograms of total lysate were loaded in the IN lanes, and 10% of the total immunoprecipitates were loaded in the IP lanes.

Giant ANK2 is responsible for neuron-specific PPIs

(A) Depiction of the ANK2 gene with ASD-associated mutations leading to splicing defects or early decay. Giant ANK2 includes exon 37, which is alternatively spliced to produce short ANK2. The epitope recognized by the ANK2 antibody used in this study is marked with an asterisk.

(B) Volcano plot of an IP-MS experiment for ANK2 in iNs, showing ANK2 in red, significant interactors (log2 FC > 0 and FDR ≤ 0.1) in green, and non-interactors in blue; known InWeb interactors identified as interactors or non-interactors are highlighted in yellow or white, respectively; interactors replicated by immunoblotting are highlighted in orange. Statistics were derived from two replicates using a two-tailed one-sample moderated t test.

(C) Experimental workflow to identify giant ANK2-specific PPIs.

(D) Top: CRISPR-Cas9 editing strategy to generate a cell line exclusively expressing the short ANK2 isoform, obtained by deletion of the giant ANK2 exon splicing acceptor. Bottom: validation of giant ANK2 KO cell line. The deletion was confirmed by PCR on genomic DNA using primers flanking the deletion site (left), and by immunoblotting (right) showing the absence of the giant ANK2 isoform in homozygous KO iNs (−/−) compared with WT (+/+) and heterozygous KO (−/−) cells. Giant and canonical ANK2 isoforms are indicated with an asterisk.

(E) C3 and CACNA2D1 western blots on ANK2 IPs in WT and giant ANK2 KO iNs. FC was calculated as a ratio of the intensity of the bands detected in the IP versus IN lanes. L, ladder; IN, input; IP, immunoprecipitation. Molecular weights are in kilodaltons. Fifty micrograms of total lysate were loaded in the IN lanes, and 10% of the total immunoprecipitates were loaded in the IP lanes.

We edited an iPSC line (iPS3, STAR Methods) to exclude retention of exon 37, which differentiates giant ANK2 from canonical (short) ANK2 isoforms (Figure 4D, top). Fully characterized giant ANK2 knockout (KO) iPSCs (Figure 4D, bottom) were viable, but showed morphological defects, defective ANK2 distribution, and high cell death upon neuronal differentiation. We could not collect sufficient giant ANK2 KO iNs for IP-MS due to high cell death. Nonetheless, we performed immunoblot analysis in KO iNs to test for presence of ANK2 interactors identified by IP-MS in corresponding wild-type (WT) neurons. KO iNs were still expressing proteins essential for neuronal physiology and synaptic function, including the calcium channel subunit CACNA2D137 and the complement protein C3,38 but these proteins were no longer associated with ANK2 (Figure 4E), suggesting that their localization might rely on giant ANK2-dependent cytoskeletal architecture.

We also performed IP-MS experiments of ANK2 in isogenic WT and giant ANK2 KO neural progenitor cells (NPCs; Figure S10B; Table S2). ANK2 interactors identified exclusively in the WT cells (which expressed both canonical and giant ANK2 isoforms) were most enriched for “glutamatergic synapses” in Gene Ontology (GO) analysis.39,40 In contrast, interactors specific to the KO cells (which lacked giant ANK2) were enriched for “microtubules” (Table S7). This suggests that the giant ANK2 isoform may have a role in localizing the protein to form cell projections of maturing NPCs and in guiding neuronal morphology. Accordingly, ARPC5L, which is essential for both structural and functional maturation of the growth cone in maturating neurons,41 is among the WT-specific ANK2 interactors (Figure S10B).

We next investigated a robust interaction identified between PTEN and the A-kinase anchor protein AKAP8L, which was consistently replicated by immunoblotting and reverse IP (Figure 5A, 5B, and S10C). AKAP8L is implicated in the mammalian target of rapamycin (mTOR) pathway,42 but has not been linked to PTEN in previous biochemical studies (which were mostly conducted in cancer cell models). Thus, our findings in iNs suggest a neuron-specific mTOR circuitry of PTEN that acts through AKAP8L. To test this hypothesis, we generated AKAP8L KO iPSC lines through CRISPR-Cas9 gene editing (Figure 5C). We obtained isogenic lines with both heterozygous and homozygous mutations, with the homozygous KO line resulting in complete loss of AKAP8L (Figure 5D). Interestingly, the AKAP8L homozygous KO line showed an increase in growth rate at the NPC stage compared with isogenic WT and heterozygous lines (p = 0.0287 and 0.0162, respectively; Figure 5E).Figure 5The neuronal PTEN-AKAP8L interaction regulates cell proliferation(A) Volcano plot of an IP-MS experiment for PTEN in iNs, showing PTEN in red, significant interactors (log2 FC > 0 and FDR ≤ 0.1) in green, and non-interactors in blue; AKAP8L is marked in orange. Statistics were derived from two replicates using a two-tailed one-sample moderated t test.(B) AKAP8L immunoblot on the PTEN IP. AKAP8L is marked with an asterisk, IgG heavy chains are marked to the right; L, ladder; IN, input; FT, flow-through; IP, immunoprecipitation; IgG, IgG control. Molecular weights are in kilodaltons.(C) Schematic of the CRISPR-Cas9 editing strategy to generate AKAP8L KO iPSC lines.(D) Immunoblot validation of AKAP8L heterozygous (+/−) and homozygous (−/−) KO lines generated using the strategy in (C). B-actin is used as a loading control. Molecular weights (kDa) are marked on the side of each blot.(E) WT (+/+), AKAP8L heterozygous KO (+/−), and AKAP8L homozygous KO (−/−) neural progenitor cell growth over 4 days upon seeding cells at identical confluences normalized to their seeding density. Error bars indicate standard deviations of the mean of three biological replicates. Asterisks indicate p < 0.05 as calculated using a two-tailed t test.(F) Immunoblot analysis of PTEN, phosphorylated S6 (P-S6), S6, and AKT in WT and AKAP8L homozygous KO iNs. Molecular weights (kDa) are marked on the side of each blot and B-actin is used as loading control. FC was calculated as a ratio of the intensity of the bands detected in the AKAP8L KO (−/−) versus WT (+/+) lanes.

The neuronal PTEN-AKAP8L interaction regulates cell proliferation

(A) Volcano plot of an IP-MS experiment for PTEN in iNs, showing PTEN in red, significant interactors (log2 FC > 0 and FDR ≤ 0.1) in green, and non-interactors in blue; AKAP8L is marked in orange. Statistics were derived from two replicates using a two-tailed one-sample moderated t test.

(B) AKAP8L immunoblot on the PTEN IP. AKAP8L is marked with an asterisk, IgG heavy chains are marked to the right; L, ladder; IN, input; FT, flow-through; IP, immunoprecipitation; IgG, IgG control. Molecular weights are in kilodaltons.

(C) Schematic of the CRISPR-Cas9 editing strategy to generate AKAP8L KO iPSC lines.

(D) Immunoblot validation of AKAP8L heterozygous (+/−) and homozygous (−/−) KO lines generated using the strategy in (C). B-actin is used as a loading control. Molecular weights (kDa) are marked on the side of each blot.

(E) WT (+/+), AKAP8L heterozygous KO (+/−), and AKAP8L homozygous KO (−/−) neural progenitor cell growth over 4 days upon seeding cells at identical confluences normalized to their seeding density. Error bars indicate standard deviations of the mean of three biological replicates. Asterisks indicate p < 0.05 as calculated using a two-tailed t test.

(F) Immunoblot analysis of PTEN, phosphorylated S6 (P-S6), S6, and AKT in WT and AKAP8L homozygous KO iNs. Molecular weights (kDa) are marked on the side of each blot and B-actin is used as loading control. FC was calculated as a ratio of the intensity of the bands detected in the AKAP8L KO (−/−) versus WT (+/+) lanes.

Although AKAP8L KO did not result in changes of PTEN levels in iNs as measured by immunoblotting, we observed a small but detectable increase in the levels of phosphorylated ribosomal protein S6 (Figure 5F). This is a marker of hyperactivation of the mTOR pathway and suggests an AKAP8L-mediated defect in the PTEN signaling cascade in ASDs. Furthermore, we detected similar levels of the serine/threonine-protein kinase AKT in WT versus KO cells, suggesting that AKAP8L acts in the mTORC1 but not the mTORC2 pathway.43 These results support a neuronal PTEN-mTOR circuitry that includes AKAP8L and suggest that the PTEN-AKAP8L interaction affects downstream mTORC1 signaling and cell proliferation without affecting PTEN protein levels directly (Figures S10C and S10D).

Our investigation of giant ANK2 and PTEN interactions demonstrate the potential of using cell-type-specific PPIs to gain mechanistic insights into ASD-relevant molecular cascades.

Network convergence implicates the IGF2BP1-3 complex in an ASD-relevant regulatory circuitrySince the PPI network contains interactors linked to multiple index proteins (Figures 2C and 2D; Table S3), we hypothesized that convergent signals in the network could point to biological themes of relevance to ASDs. By ranking the interactors in descending order based on the number of index proteins they interact with (Table S3), three insulin-like growth factor 2 mRNA-binding proteins, IGF2BP1, IGF2BP3, and IGF2BP2, emerged among the top interactors. Together, these proteins form an N6-methyladenosine (m6A) reader complex broadly involved in post-transcriptional regulation and affect mRNA stability and translation.44 We therefore tested whether this complex is involved in a regulatory circuit that connects transcription of ASD-associated genes to the protein network we identified.We found that RNA targets of IGF2BP1-3 in HEK cells44 are significantly enriched for ASD-associated genes identified through exome sequencing5 (p = 3.9 × 10−10; Figure 6A; Table S8), including nine of the index genes in this study. Genetic enrichment analysis using ASD genome-wide association studies (GWAS) data45 and MAGMA46 also found the IGF2BP1-3 targets to be enriched for common variants associated with ASDs (p = 1.4 × 10−3; Figure 6B; Table S8). Next, we performed a conditional enrichment analysis to test whether genes in both our combined network and index protein-specific sub-networks are enriched for IGF2BP1-3 targets compared with the non-interactors (Figure 6C; Table S8). At a Bonferroni-corrected significance threshold (p < 0.05/56, adjusting for 14 networks and four IGF2BP target lists), the combined and DYRK1A networks both showed significant enrichment (p = 1.7 × 10−17 and 7.6 × 10−9, respectively).Figure 6The IGF2BP1-3 complex interacts with ASD-associated transcripts and proteins(A) Overlap enrichment between RNA targets of IGF2BP1-344 (target counts in parentheses) and ASD-associated genes5 (FDR ≤ 0.1 in exome sequencing data) calculated using one-tailed hypergeometric tests. IGF2BP1-3 indicates the combined target list. Vertical dashed line indicates p < 0.05/4 (adjusting for four target lists). The number of genes in the overlap is shown to the right of each bar.(B) Common variant enrichment of IGF2BP1-3 targets derived using ASD GWAS data45 and MAGMA.46 Vertical dashed line indicates p < 0.05/4. The enrichment coefficient is shown to the right of each bar.(C) Overlap enrichment between IGF2BP1-3 targets and the ASD PPI network or index protein-specific sub-networks (compared with non-interactors) calculated using one-tailed hypergeometric tests. Gene counts in overlaps with p < 0.05 (∗) or p < 0.05/4 (∗∗) are labeled in the heatmap. Networks that contain any of the IGF2BPs are highlighted in blue on the x axis.(D) Volcano plot of an IP-MS experiment for DYRK1A in iNs, showing DYRK1A in red, significant interactors (log2 FC > 0 and FDR ≤ 0.1) in green, and non-interactors in blue; IGF2BP1-3, which were technically replicated by western blot, are marked in orange. Statistics were derived from two replicates using a two-tailed one-sample moderated t test.(E) Immunoblot on an DYRK1A IP with and without addition of 25 U of benzonase (indicated at the top). An asterisk marks the expected band of each protein (named on the right). L, ladder; IN, input; IP, immunoprecipitation; IgG, IgG control. Molecular weights are in kilodaltons.(F) Immunoblot showing protein levels (named on the right) upon DYRK1A siRNA-mediated knockdown for 48 h in iPSCs. B-actin is used as loading control. FC was calculated as a ratio of the intensity of the bands detected in the DYRK1A knockdown (KD) versus WT lanes across three biological replicates.DYRK1A is a ubiquitously expressed protein kinase encoded by a gene localized in the Down syndrome critical region of chromosome 21 and is associated with learning disabilities in individuals with Down syndrome and ASDs.47,48 It has also been implicated in post-transcriptional regulation of cellular processes involved in brain development and function.49 Given the enriched overlap observed between the IGF2BP1-3 targets and the DYRK1A network, we hypothesized that DYRK1A might be part of an ASD-relevant feedback circuitry acting upstream of the IGF2BP1-3 complex. We thus verified that the interactions between DYRK1A and IGF2BP1-3 identified by IP-MS (Figure 6D) are not mediated by nucleic acids (Figure 6E), indicating a direct regulatory relationship. We also observed that partial knockdown (30%) with siRNA of DYRK1A in iPSCs was sufficient to trigger increased expression of IGF2BP1-3 (Figure 6F).Overall, our results suggest that the IGF2BP1-3 complex regulates an ASD-relevant transcriptional circuitry, and that the identified interactions between DYRK1A and the complex might be key in modulating the expression of numerous ASD-associated genes.

Network convergence implicates the IGF2BP1-3 complex in an ASD-relevant regulatory circuitry

Since the PPI network contains interactors linked to multiple index proteins (Figures 2C and 2D; Table S3), we hypothesized that convergent signals in the network could point to biological themes of relevance to ASDs. By ranking the interactors in descending order based on the number of index proteins they interact with (Table S3), three insulin-like growth factor 2 mRNA-binding proteins, IGF2BP1, IGF2BP3, and IGF2BP2, emerged among the top interactors. Together, these proteins form an N6-methyladenosine (m6A) reader complex broadly involved in post-transcriptional regulation and affect mRNA stability and translation.44 We therefore tested whether this complex is involved in a regulatory circuit that connects transcription of ASD-associated genes to the protein network we identified.

We found that RNA targets of IGF2BP1-3 in HEK cells44 are significantly enriched for ASD-associated genes identified through exome sequencing5 (p = 3.9 × 10−10; Figure 6A; Table S8), including nine of the index genes in this study. Genetic enrichment analysis using ASD genome-wide association studies (GWAS) data45 and MAGMA46 also found the IGF2BP1-3 targets to be enriched for common variants associated with ASDs (p = 1.4 × 10−3; Figure 6B; Table S8). Next, we performed a conditional enrichment analysis to test whether genes in both our combined network and index protein-specific sub-networks are enriched for IGF2BP1-3 targets compared with the non-interactors (Figure 6C; Table S8). At a Bonferroni-corrected significance threshold (p < 0.05/56, adjusting for 14 networks and four IGF2BP target lists), the combined and DYRK1A networks both showed significant enrichment (p = 1.7 × 10−17 and 7.6 × 10−9, respectively).Figure 6The IGF2BP1-3 complex interacts with ASD-associated transcripts and proteins(A) Overlap enrichment between RNA targets of IGF2BP1-344 (target counts in parentheses) and ASD-associated genes5 (FDR ≤ 0.1 in exome sequencing data) calculated using one-tailed hypergeometric tests. IGF2BP1-3 indicates the combined target list. Vertical dashed line indicates p < 0.05/4 (adjusting for four target lists). The number of genes in the overlap is shown to the right of each bar.(B) Common variant enrichment of IGF2BP1-3 targets derived using ASD GWAS data45 and MAGMA.46 Vertical dashed line indicates p < 0.05/4. The enrichment coefficient is shown to the right of each bar.(C) Overlap enrichment between IGF2BP1-3 targets and the ASD PPI network or index protein-specific sub-networks (compared with non-interactors) calculated using one-tailed hypergeometric tests. Gene counts in overlaps with p < 0.05 (∗) or p < 0.05/4 (∗∗) are labeled in the heatmap. Networks that contain any of the IGF2BPs are highlighted in blue on the x axis.(D) Volcano plot of an IP-MS experiment for DYRK1A in iNs, showing DYRK1A in red, significant interactors (log2 FC > 0 and FDR ≤ 0.1) in green, and non-interactors in blue; IGF2BP1-3, which were technically replicated by western blot, are marked in orange. Statistics were derived from two replicates using a two-tailed one-sample moderated t test.(E) Immunoblot on an DYRK1A IP with and without addition of 25 U of benzonase (indicated at the top). An asterisk marks the expected band of each protein (named on the right). L, ladder; IN, input; IP, immunoprecipitation; IgG, IgG control. Molecular weights are in kilodaltons.(F) Immunoblot showing protein levels (named on the right) upon DYRK1A siRNA-mediated knockdown for 48 h in iPSCs. B-actin is used as loading control. FC was calculated as a ratio of the intensity of the bands detected in the DYRK1A knockdown (KD) versus WT lanes across three biological replicates.

The IGF2BP1-3 complex interacts with ASD-associated transcripts and proteins

(A) Overlap enrichment between RNA targets of IGF2BP1-344 (target counts in parentheses) and ASD-associated genes5 (FDR ≤ 0.1 in exome sequencing data) calculated using one-tailed hypergeometric tests. IGF2BP1-3 indicates the combined target list. Vertical dashed line indicates p < 0.05/4 (adjusting for four target lists). The number of genes in the overlap is shown to the right of each bar.

(B) Common variant enrichment of IGF2BP1-3 targets derived using ASD GWAS data45 and MAGMA.46 Vertical dashed line indicates p < 0.05/4. The enrichment coefficient is shown to the right of each bar.

(C) Overlap enrichment between IGF2BP1-3 targets and the ASD PPI network or index protein-specific sub-networks (compared with non-interactors) calculated using one-tailed hypergeometric tests. Gene counts in overlaps with p < 0.05 (∗) or p < 0.05/4 (∗∗) are labeled in the heatmap. Networks that contain any of the IGF2BPs are highlighted in blue on the x axis.

(D) Volcano plot of an IP-MS experiment for DYRK1A in iNs, showing DYRK1A in red, significant interactors (log2 FC > 0 and FDR ≤ 0.1) in green, and non-interactors in blue; IGF2BP1-3, which were technically replicated by western blot, are marked in orange. Statistics were derived from two replicates using a two-tailed one-sample moderated t test.

(E) Immunoblot on an DYRK1A IP with and without addition of 25 U of benzonase (indicated at the top). An asterisk marks the expected band of each protein (named on the right). L, ladder; IN, input; IP, immunoprecipitation; IgG, IgG control. Molecular weights are in kilodaltons.

(F) Immunoblot showing protein levels (named on the right) upon DYRK1A siRNA-mediated knockdown for 48 h in iPSCs. B-actin is used as loading control. FC was calculated as a ratio of the intensity of the bands detected in the DYRK1A knockdown (KD) versus WT lanes across three biological replicates.

DYRK1A is a ubiquitously expressed protein kinase encoded by a gene localized in the Down syndrome critical region of chromosome 21 and is associated with learning disabilities in individuals with Down syndrome and ASDs.47,48 It has also been implicated in post-transcriptional regulation of cellular processes involved in brain development and function.49 Given the enriched overlap observed between the IGF2BP1-3 targets and the DYRK1A network, we hypothesized that DYRK1A might be part of an ASD-relevant feedback circuitry acting upstream of the IGF2BP1-3 complex. We thus verified that the interactions between DYRK1A and IGF2BP1-3 identified by IP-MS (Figure 6D) are not mediated by nucleic acids (Figure 6E), indicating a direct regulatory relationship. We also observed that partial knockdown (30%) with siRNA of DYRK1A in iPSCs was sufficient to trigger increased expression of IGF2BP1-3 (Figure 6F).

Overall, our results suggest that the IGF2BP1-3 complex regulates an ASD-relevant transcriptional circuitry, and that the identified interactions between DYRK1A and the complex might be key in modulating the expression of numerous ASD-associated genes.

The network is enriched for rare variants associated with ASDs and developmental disordersTo evaluate whether the combined PPI network and index protein-specific sub-networks are enriched for genes associated with ASDs, we performed global and conditional enrichment analyses using rare variant association statistics derived from exome sequencing.5 In the global analysis, we tested whether the network genes are more significantly associated with ASDs compared with other protein-coding genes in the genome; in the conditional analysis, we compared the network genes against the non-interactors detected in iNs. Five networks reached significance (p < 0.05/14, adjusting for 14 networks) in the global analysis (Figure S11A), and only the combined network was significant in the conditional analysis (p = 1.1 × 10−5; Figure 7A; Table S9). Notably, the conditional enrichment was robust even after we removed known InWeb interactors from the combined network (p = 5.5 × 10−5; Figure S11B), indicating that this signal is not driven by the background proteome in iNs or by previously reported interactions. Rather, the neuron-specific interactions identified in this study are enriched for rare variants associated with ASDs.Figure 7Genetic enrichment in the ASD PPI network(A) Rare variant5,50,51 or pLI score52 enrichment of the network or index protein-specific sub-networks compared with non-interactors, derived using one-tailed Kolmogorov-Smirnov tests. The number of genes in each network is shown in parentheses on the y axis. Left and right vertical dashed lines indicate p < 0.05 and p < 0.05/14 (adjusting for 14 networks), respectively; results passing these thresholds are labeled with the corresponding KS test statistics.(B) Social Manhattan plot of index genes (red) and interactors that are suggestive ASD-associated genes (blue or cyan for FDR ≤ 0.1 or 0.25, respectively) in exome sequencing data.5 Sizes of the interactor nodes scale with the number of linked index genes. Connecting lines indicate observed interactions in the ASD PPI network; interactions replicated by IP-WB are highlighted in orange.We also performed analogous enrichment analyses using exome sequencing data for developmental disorders50 (DDs) and schizophrenia51 (SCZ) and gnomAD52 pLI scores, which represent the probability of a gene being loss-of-function intolerant (Figures 7A, S11A, and S11B; Table S9). In the conditional analysis, we observed that the combined network was significantly enriched for DD genes (p = 6.2 × 10−7) and genes with high pLI scores (p = 2.6 × 10−31); five index protein-specific sub-networks were also enriched for high pLI scores (p = 1.1 × 10−12 to 3.4 × 10−4). In contrast, there was no significant SCZ enrichment for any of the networks. These results indicate that the index protein interactors contribute to rare variants associated with both ASDs and DDs more than one would expect. The interactors are also more likely to be intolerant to loss-of-function mutations, suggesting their importance for cell fate specification or survival.To assess the common variant enrichment of the PPI networks, we performed global and conditional analyses using ASD GWAS data45 and MAGMA.46 Only the SYNGAP1 network came close to, but did not reach, significance in both the global and conditional analyses (conditional p = 3.71 × 10−3, which is just shy of the Bonferroni-corrected threshold of p < 0.05/14; Figure S11C; Table S10). We also repeated the analyses using GWAS data of attention deficit hyperactivity disorder53 (ADHD), bipolar disorder54 (BIP), major depressive disorder55 (MDD), and SCZ,56 as well as height57 as an additional control trait. None of the tested networks reached significance (Figure S11C; Table S10).

The network is enriched for rare variants associated with ASDs and developmental disorders

To evaluate whether the combined PPI network and index protein-specific sub-networks are enriched for genes associated with ASDs, we performed global and conditional enrichment analyses using rare variant association statistics derived from exome sequencing.5 In the global analysis, we tested whether the network genes are more significantly associated with ASDs compared with other protein-coding genes in the genome; in the conditional analysis, we compared the network genes against the non-interactors detected in iNs. Five networks reached significance (p < 0.05/14, adjusting for 14 networks) in the global analysis (Figure S11A), and only the combined network was significant in the conditional analysis (p = 1.1 × 10−5; Figure 7A; Table S9). Notably, the conditional enrichment was robust even after we removed known InWeb interactors from the combined network (p = 5.5 × 10−5; Figure S11B), indicating that this signal is not driven by the background proteome in iNs or by previously reported interactions. Rather, the neuron-specific interactions identified in this study are enriched for rare variants associated with ASDs.Figure 7Genetic enrichment in the ASD PPI network(A) Rare variant5,50,51 or pLI score52 enrichment of the network or index protein-specific sub-networks compared with non-interactors, derived using one-tailed Kolmogorov-Smirnov tests. The number of genes in each network is shown in parentheses on the y axis. Left and right vertical dashed lines indicate p < 0.05 and p < 0.05/14 (adjusting for 14 networks), respectively; results passing these thresholds are labeled with the corresponding KS test statistics.(B) Social Manhattan plot of index genes (red) and interactors that are suggestive ASD-associated genes (blue or cyan for FDR ≤ 0.1 or 0.25, respectively) in exome sequencing data.5 Sizes of the interactor nodes scale with the number of linked index genes. Connecting lines indicate observed interactions in the ASD PPI network; interactions replicated by IP-WB are highlighted in orange.

Genetic enrichment in the ASD PPI network

(A) Rare variant5,50,51 or pLI score52 enrichment of the network or index protein-specific sub-networks compared with non-interactors, derived using one-tailed Kolmogorov-Smirnov tests. The number of genes in each network is shown in parentheses on the y axis. Left and right vertical dashed lines indicate p < 0.05 and p < 0.05/14 (adjusting for 14 networks), respectively; results passing these thresholds are labeled with the corresponding KS test statistics.

(B) Social Manhattan plot of index genes (red) and interactors that are suggestive ASD-associated genes (blue or cyan for FDR ≤ 0.1 or 0.25, respectively) in exome sequencing data.5 Sizes of the interactor nodes scale with the number of linked index genes. Connecting lines indicate observed interactions in the ASD PPI network; interactions replicated by IP-WB are highlighted in orange.

We also performed analogous enrichment analyses using exome sequencing data for developmental disorders50 (DDs) and schizophrenia51 (SCZ) and gnomAD52 pLI scores, which represent the probability of a gene being loss-of-function intolerant (Figures 7A, S11A, and S11B; Table S9). In the conditional analysis, we observed that the combined network was significantly enriched for DD genes (p = 6.2 × 10−7) and genes with high pLI scores (p = 2.6 × 10−31); five index protein-specific sub-networks were also enriched for high pLI scores (p = 1.1 × 10−12 to 3.4 × 10−4). In contrast, there was no significant SCZ enrichment for any of the networks. These results indicate that the index protein interactors contribute to rare variants associated with both ASDs and DDs more than one would expect. The interactors are also more likely to be intolerant to loss-of-function mutations, suggesting their importance for cell fate specification or survival.

To assess the common variant enrichment of the PPI networks, we performed global and conditional analyses using ASD GWAS data45 and MAGMA.46 Only the SYNGAP1 network came close to, but did not reach, significance in both the global and conditional analyses (conditional p = 3.71 × 10−3, which is just shy of the Bonferroni-corrected threshold of p < 0.05/14; Figure S11C; Table S10). We also repeated the analyses using GWAS data of attention deficit hyperactivity disorder53 (ADHD), bipolar disorder54 (BIP), major depressive disorder55 (MDD), and SCZ,56 as well as height57 as an additional control trait. None of the tested networks reached significance (Figure S11C; Table S10).

Network analyses can prioritize additional ASD-associated genes from genetic studiesSince the PPI network is enriched for rare variants associated with ASDs, we used it to prioritize ASD-associated genes that did not reach stringent statistical significance in the exome sequencing study.5 Specifically, we generated a “social Manhattan plot” to highlight PPIs between the index genes and other ASD-associated genes with FDR ≤ 0.25 in the exome sequencing data (Figure 7B). Consistent with the rare variant enrichment results, the overlap between our network and these ASD-associated genes is significant (p = 4.9 × 10−4), with DYRK1A being linked to the most genes out of all the index genes. In total, we prioritized 32 suggestive ASD-associated genes that overlap with our network. Interestingly, these genes have higher average expression throughout cortical development compared with the other ASD-associated genes identified at the same significance threshold and have comparable expression with the exome-wide significant genes (Figure S11D). We successfully replicated 14 of the prioritized interactions using forward and/or reverse IPs followed by western blotting (Table S5), including multiple interactions involving SMARCC2, ELAVL3, DPYSL2, and DYNC1H1. While follow-up investigation on the prioritized genes is necessary, our analysis showcases how the ASD PPI network can be used to corroborate findings from genetic studies of ASDs.

Network analyses can prioritize additional ASD-associated genes from genetic studies

Since the PPI network is enriched for rare variants associated with ASDs, we used it to prioritize ASD-associated genes that did not reach stringent statistical significance in the exome sequencing study.5 Specifically, we generated a “social Manhattan plot” to highlight PPIs between the index genes and other ASD-associated genes with FDR ≤ 0.25 in the exome sequencing data (Figure 7B). Consistent with the rare variant enrichment results, the overlap between our network and these ASD-associated genes is significant (p = 4.9 × 10−4), with DYRK1A being linked to the most genes out of all the index genes. In total, we prioritized 32 suggestive ASD-associated genes that overlap with our network. Interestingly, these genes have higher average expression throughout cortical development compared with the other ASD-associated genes identified at the same significance threshold and have comparable expression with the exome-wide significant genes (Figure S11D). We successfully replicated 14 of the prioritized interactions using forward and/or reverse IPs followed by western blotting (Table S5), including multiple interactions involving SMARCC2, ELAVL3, DPYSL2, and DYNC1H1. While follow-up investigation on the prioritized genes is necessary, our analysis showcases how the ASD PPI network can be used to corroborate findings from genetic studies of ASDs.

DiscussionASDs are a heterogeneous set of neurodevelopmental conditions with contributions from both rare and common genetic variants, and their genetic and biological complexity is reflected in the clinical diversity of individuals with autism.1 In the past decade, rare variants with large effect sizes in protein-coding genes have been shown to shift individual predisposition to autism during prenatal development and early childhood.5,6 There is evidence that mutations in several of these genes result in similar impairments in social and communication skills.58 In other cases, specific mutations are associated with altered brain size and synaptopathies.59,60 However, how such ASD-associated mutations act singularly or in ensembles in the human brain remains largely unknown.9In this study, we combined human genetics, iPSC technology, interaction proteomics, and computational approaches to shed light on how a subset of ASD-associated genes implicated by rare variations may map onto disease-relevant and brain cell-type-specific pathways. We built a PPI network for 13 ASD index proteins in iPSC-derived excitatory neurons, which represent a key cell type in ASDs. We demonstrated the reproducibility and biological relevance of the identified PPIs using immunoblotting, IP-MS in brain homogenates, brain co-expression, and tissue and cell type enrichment analyses. Thus, the fact that over 90% of these interactions have yet to be reported in the literature indicates the potential for biological discovery using neuron-specific interaction proteomics, as opposed to relying on existing PPI data from highly proliferative or immortalized cell types (e.g., cancer cell lines).17,18To showcase how our cell-type-specific approach can reconcile genetic signals of ASDs with molecular mechanisms that lead to neuronal phenotypes, we further dissected the PPIs for ANK2. The ANK2 gene encodes multiple isoforms due to alternative splicing, including a large 440-kDa isoform, giant ANK2, that is only expressed in the nervous system.29 Known ASD-associated ANK2 mutations all affect giant ANK2 either exclusively (P1843S, R2608 frameshift, and E3429V) or in combination with other isoforms (R895∗, R990∗, Q3683fs).61 By performing ANK2 IPs in WT versus giant ANK2 KO NPCs and iNs, we showed that the ASD-associated giant ANK2 isoform and its interactions are important for neuronal maturation and proper localization of synaptic components to cell projections. We also investigated the neuron-specific interaction of PTEN with the pseudo-kinase AKAP8L. Although this interaction has not been reported in the literature, a previous study of a family with autism and macrocephaly identified AKAP8L as a likely candidate gene affected by the phenotype-associated genomic alteration at 19p13.12.62 This suggests a phenotypic convergence with germline mutations of PTEN that have also been described in individuals with autism and macrocephaly.63 The PTEN gene is frequently mutated in cancers due to its antagonizing role in PI3K/AKT/mTOR signaling,64 but there is no consensus on whether PTEN-mediated cell proliferation defects are tied to neuronal phenotypes. We found that an iPSC-derived KO model of AKAP8L led to increased cellular proliferation at early stages of neuronal differentiation. This, together with the recently reported interaction between AKAP8L and mTORC1,42 indicates that the PTEN-AKAP8L interaction may influence neurodevelopmental phenotypes through the mTOR signaling cascades.Besides investigating the PPIs of individual ASD index proteins, we explored shared interactions between them to identify convergent pathways. Intriguingly, we found IGF2BP1-3 to be among the most recurrent interactors in the PPI network. IGF2BP1-3 were recently characterized as part of an m6A reader complex that regulates mRNA stability and translation rates.44 In parallel, other studies have shown that mRNA transport, turnover, and translation of synaptic proteins are key to maintaining neuronal health, synaptic plasticity, learning, and memory,65 and m6A deposition has emerged as a critical regulatory mechanism for these functions in neurons and neural progenitors.66,67,68,69 Although the IGF2BP1-3 complex has a well-documented role in modulating neural stem cell growth and division in Drosophila,70 there is no significant genetic evidence linking them to ASDs. However, based on our findings, we hypothesize that the complex may be broadly involved in the regulation of ASD-associated genes, and that its lack of genetic signals in ASDs may be due to intolerance to loss-of-function mutations in humans or power limitations in existing genetic datasets (Table S11). Indeed, we found that RNA targets of IGF2BP1-3 derived from HEK cells are enriched for both rare and common variants of ASDs and significantly overlap with genes in our PPI network. Out of all the index proteins, the DYRK1A interactome showed the strongest enrichment; partial knockdown of DYRK1A also led to increased expression of IGF2BP1-3. While the IGF2BP1-3 targets still need to be validated in a neuronal context, these findings suggest that the IGF2BP1-3 complex and DYRK1A might be part of a feedback mechanism reinforcing the expression of ASD-relevant transcripts. This vignette demonstrates the value of studying convergent signals in the PPI network.Convergence between the PPI network and other data types can also lead to biological insights. For instance, by integrating our network with scRNA-seq data derived from postmortem cortex of ASD individuals, we found the network to be enriched for transcriptionally perturbed genes in layer 2/3 cortical excitatory neurons, supporting previous findings that implicate this cell type in ASDs.24 By integrating the network with genetic data, we showed that it is enriched for rare variants of ASDs and thus can be used to prioritize ASD-associated genes with suggestive genetic significance, based on their PPIs with our ASD index genes. Although we did not observe an analogous enrichment for common variants of ASDs in the network, this could be due to power limitations of the ASD GWAS. In a parallel study,71 we generated an iN-derived PPI network for SCZ-associated genes prioritized from well-powered GWAS data, showed that the network is enriched for common variant risk of SCZ, and used it to complement fine-mapping and expression quantitative trait loci (eQTL) co-localization approaches to prioritize additional genes from GWAS loci. Performing similar analyses on the ASD PPI network when larger ASD GWAS become available may reveal whether rare and common variants of ASDs converge onto shared pathways represented in the network, as we have already observed for the transcriptional circuit regulated by IGF2BP1-3.Limitations of the studyWe acknowledge several limitations that are either intrinsic to the study design or linked to deliberate experimental choices. First, we chose to perform IPs using specific antibodies against the index proteins, as opposed to using a tagging and overexpression system, to avoid the necessity of selecting one or a few isoforms for IP and the caveat of non-endogenous expression. Consequently, we were constrained by the availability of IP-competent antibodies during index protein selection. An appealing alternative to overcome this bottleneck will be to utilize the CRISPR-Cas9 technology to perform endogenous protein tagging.72,73 Second, we compared protein abundances in the index protein IPs against matched immunoglobulin G (IgG) or negative control IPs to control for non-specific binding and define significant interactors. An alternative strategy will be to perform IPs in isogenic iPSC lines with the index proteins knocked out. This could provide additional control against false-positive interactions, although the viability and proper differentiation of the edited lines will be a main constraint. Third, all IPs were performed in a salt/detergent environment optimized to identify both direct, stable interactions (e.g., between members of core protein complexes), and indirect, more transient interactions. While both types of interactions can be functionally relevant to ASDs, we cannot distinguish between them in our data. Furthermore, MS analyses of the IPs had known technical biases, including incomplete coverage and underrepresentation of lowly expressed or highly hydrophobic proteins (e.g., membrane proteins).74,75 The reader should keep these caveats in mind when interpreting the nature of our PPI data.To evaluate the robustness of the PPI network despite these caveats, we performed independent IPs and reverse IPs followed by immunoblotting and estimated a ∼90% replication rate (or ∼10% false-positive rate) in the network. These results support the technical reproducibility of our data, but they relied on a biochemical method that may share some of the biases of the IP-MS approach. Although we further demonstrated the biological relevance of the network using computational analyses, we recognize that technologies such as multichannel microscopy,76 super-resolution microscopy,77 and cryoelectron microscopy (cryo-EM)78 may constitute avenues to orthogonally validate our PPI data in the future.Finally, the PPI network was anchored by 13 index proteins and derived from excitatory iNs; thus, it does not represent a complete interactome of all ASD-associated genes across different brain cell types of relevance to ASDs (e.g., inhibitory interneurons). Similarly, while our findings indicate that convergent signals in the network could point to ASD-related pathways, further work is required to determine the extent of convergence between all ASD-associated genes. It will be critical to apply similar approaches to additional ASD-associated genes and cell types as larger genetic datasets and improved cell modeling protocols become available, to build toward a comprehensive cell-type-specific PPI atlas for ASDs.ConclusionsSo far, very few PPI studies have taken advantage of neuronal cell models, and even fewer used human genetics as both an entry point for index gene selection and a downstream source of data integration. Here, we built a PPI network of 13 index proteins encoded by high-confidence ASD-associated genes in human iPSC-derived excitatory neurons. This network contains many newly reported interactions, is enriched for genetic and transcriptional perturbations observed in ASD individuals, and can be leveraged to uncover neuron-specific biology related to ASDs. Our data not only empower hypothesis generation to study the mechanisms of individual ASD-associated genes but also suggest convergent pathways that may inform therapeutic intervention or clinical stratification strategies. More generally, the approach described in this study and the parallel SCZ study71 can be expanded to generate larger PPI networks across brain cell types and neurodevelopmental stages to uncover novel pathways underlying psychiatric disorders.

ASDs are a heterogeneous set of neurodevelopmental conditions with contributions from both rare and common genetic variants, and their genetic and biological complexity is reflected in the clinical diversity of individuals with autism.1 In the past decade, rare variants with large effect sizes in protein-coding genes have been shown to shift individual predisposition to autism during prenatal development and early childhood.5,6 There is evidence that mutations in several of these genes result in similar impairments in social and communication skills.58 In other cases, specific mutations are associated with altered brain size and synaptopathies.59,60 However, how such ASD-associated mutations act singularly or in ensembles in the human brain remains largely unknown.9

In this study, we combined human genetics, iPSC technology, interaction proteomics, and computational approaches to shed light on how a subset of ASD-associated genes implicated by rare variations may map onto disease-relevant and brain cell-type-specific pathways. We built a PPI network for 13 ASD index proteins in iPSC-derived excitatory neurons, which represent a key cell type in ASDs. We demonstrated the reproducibility and biological relevance of the identified PPIs using immunoblotting, IP-MS in brain homogenates, brain co-expression, and tissue and cell type enrichment analyses. Thus, the fact that over 90% of these interactions have yet to be reported in the literature indicates the potential for biological discovery using neuron-specific interaction proteomics, as opposed to relying on existing PPI data from highly proliferative or immortalized cell types (e.g., cancer cell lines).17,18

To showcase how our cell-type-specific approach can reconcile genetic signals of ASDs with molecular mechanisms that lead to neuronal phenotypes, we further dissected the PPIs for ANK2. The ANK2 gene encodes multiple isoforms due to alternative splicing, including a large 440-kDa isoform, giant ANK2, that is only expressed in the nervous system.29 Known ASD-associated ANK2 mutations all affect giant ANK2 either exclusively (P1843S, R2608 frameshift, and E3429V) or in combination with other isoforms (R895∗, R990∗, Q3683fs).61 By performing ANK2 IPs in WT versus giant ANK2 KO NPCs and iNs, we showed that the ASD-associated giant ANK2 isoform and its interactions are important for neuronal maturation and proper localization of synaptic components to cell projections. We also investigated the neuron-specific interaction of PTEN with the pseudo-kinase AKAP8L. Although this interaction has not been reported in the literature, a previous study of a family with autism and macrocephaly identified AKAP8L as a likely candidate gene affected by the phenotype-associated genomic alteration at 19p13.12.62 This suggests a phenotypic convergence with germline mutations of PTEN that have also been described in individuals with autism and macrocephaly.63 The PTEN gene is frequently mutated in cancers due to its antagonizing role in PI3K/AKT/mTOR signaling,64 but there is no consensus on whether PTEN-mediated cell proliferation defects are tied to neuronal phenotypes. We found that an iPSC-derived KO model of AKAP8L led to increased cellular proliferation at early stages of neuronal differentiation. This, together with the recently reported interaction between AKAP8L and mTORC1,42 indicates that the PTEN-AKAP8L interaction may influence neurodevelopmental phenotypes through the mTOR signaling cascades.

Besides investigating the PPIs of individual ASD index proteins, we explored shared interactions between them to identify convergent pathways. Intriguingly, we found IGF2BP1-3 to be among the most recurrent interactors in the PPI network. IGF2BP1-3 were recently characterized as part of an m6A reader complex that regulates mRNA stability and translation rates.44 In parallel, other studies have shown that mRNA transport, turnover, and translation of synaptic proteins are key to maintaining neuronal health, synaptic plasticity, learning, and memory,65 and m6A deposition has emerged as a critical regulatory mechanism for these functions in neurons and neural progenitors.66,67,68,69 Although the IGF2BP1-3 complex has a well-documented role in modulating neural stem cell growth and division in Drosophila,70 there is no significant genetic evidence linking them to ASDs. However, based on our findings, we hypothesize that the complex may be broadly involved in the regulation of ASD-associated genes, and that its lack of genetic signals in ASDs may be due to intolerance to loss-of-function mutations in humans or power limitations in existing genetic datasets (Table S11). Indeed, we found that RNA targets of IGF2BP1-3 derived from HEK cells are enriched for both rare and common variants of ASDs and significantly overlap with genes in our PPI network. Out of all the index proteins, the DYRK1A interactome showed the strongest enrichment; partial knockdown of DYRK1A also led to increased expression of IGF2BP1-3. While the IGF2BP1-3 targets still need to be validated in a neuronal context, these findings suggest that the IGF2BP1-3 complex and DYRK1A might be part of a feedback mechanism reinforcing the expression of ASD-relevant transcripts. This vignette demonstrates the value of studying convergent signals in the PPI network.

Convergence between the PPI network and other data types can also lead to biological insights. For instance, by integrating our network with scRNA-seq data derived from postmortem cortex of ASD individuals, we found the network to be enriched for transcriptionally perturbed genes in layer 2/3 cortical excitatory neurons, supporting previous findings that implicate this cell type in ASDs.24 By integrating the network with genetic data, we showed that it is enriched for rare variants of ASDs and thus can be used to prioritize ASD-associated genes with suggestive genetic significance, based on their PPIs with our ASD index genes. Although we did not observe an analogous enrichment for common variants of ASDs in the network, this could be due to power limitations of the ASD GWAS. In a parallel study,71 we generated an iN-derived PPI network for SCZ-associated genes prioritized from well-powered GWAS data, showed that the network is enriched for common variant risk of SCZ, and used it to complement fine-mapping and expression quantitative trait loci (eQTL) co-localization approaches to prioritize additional genes from GWAS loci. Performing similar analyses on the ASD PPI network when larger ASD GWAS become available may reveal whether rare and common variants of ASDs converge onto shared pathways represented in the network, as we have already observed for the transcriptional circuit regulated by IGF2BP1-3.

Limitations of the studyWe acknowledge several limitations that are either intrinsic to the study design or linked to deliberate experimental choices. First, we chose to perform IPs using specific antibodies against the index proteins, as opposed to using a tagging and overexpression system, to avoid the necessity of selecting one or a few isoforms for IP and the caveat of non-endogenous expression. Consequently, we were constrained by the availability of IP-competent antibodies during index protein selection. An appealing alternative to overcome this bottleneck will be to utilize the CRISPR-Cas9 technology to perform endogenous protein tagging.72,73 Second, we compared protein abundances in the index protein IPs against matched immunoglobulin G (IgG) or negative control IPs to control for non-specific binding and define significant interactors. An alternative strategy will be to perform IPs in isogenic iPSC lines with the index proteins knocked out. This could provide additional control against false-positive interactions, although the viability and proper differentiation of the edited lines will be a main constraint. Third, all IPs were performed in a salt/detergent environment optimized to identify both direct, stable interactions (e.g., between members of core protein complexes), and indirect, more transient interactions. While both types of interactions can be functionally relevant to ASDs, we cannot distinguish between them in our data. Furthermore, MS analyses of the IPs had known technical biases, including incomplete coverage and underrepresentation of lowly expressed or highly hydrophobic proteins (e.g., membrane proteins).74,75 The reader should keep these caveats in mind when interpreting the nature of our PPI data.To evaluate the robustness of the PPI network despite these caveats, we performed independent IPs and reverse IPs followed by immunoblotting and estimated a ∼90% replication rate (or ∼10% false-positive rate) in the network. These results support the technical reproducibility of our data, but they relied on a biochemical method that may share some of the biases of the IP-MS approach. Although we further demonstrated the biological relevance of the network using computational analyses, we recognize that technologies such as multichannel microscopy,76 super-resolution microscopy,77 and cryoelectron microscopy (cryo-EM)78 may constitute avenues to orthogonally validate our PPI data in the future.Finally, the PPI network was anchored by 13 index proteins and derived from excitatory iNs; thus, it does not represent a complete interactome of all ASD-associated genes across different brain cell types of relevance to ASDs (e.g., inhibitory interneurons). Similarly, while our findings indicate that convergent signals in the network could point to ASD-related pathways, further work is required to determine the extent of convergence between all ASD-associated genes. It will be critical to apply similar approaches to additional ASD-associated genes and cell types as larger genetic datasets and improved cell modeling protocols become available, to build toward a comprehensive cell-type-specific PPI atlas for ASDs.

Limitations of the study

We acknowledge several limitations that are either intrinsic to the study design or linked to deliberate experimental choices. First, we chose to perform IPs using specific antibodies against the index proteins, as opposed to using a tagging and overexpression system, to avoid the necessity of selecting one or a few isoforms for IP and the caveat of non-endogenous expression. Consequently, we were constrained by the availability of IP-competent antibodies during index protein selection. An appealing alternative to overcome this bottleneck will be to utilize the CRISPR-Cas9 technology to perform endogenous protein tagging.72,73 Second, we compared protein abundances in the index protein IPs against matched immunoglobulin G (IgG) or negative control IPs to control for non-specific binding and define significant interactors. An alternative strategy will be to perform IPs in isogenic iPSC lines with the index proteins knocked out. This could provide additional control against false-positive interactions, although the viability and proper differentiation of the edited lines will be a main constraint. Third, all IPs were performed in a salt/detergent environment optimized to identify both direct, stable interactions (e.g., between members of core protein complexes), and indirect, more transient interactions. While both types of interactions can be functionally relevant to ASDs, we cannot distinguish between them in our data. Furthermore, MS analyses of the IPs had known technical biases, including incomplete coverage and underrepresentation of lowly expressed or highly hydrophobic proteins (e.g., membrane proteins).74,75 The reader should keep these caveats in mind when interpreting the nature of our PPI data.

To evaluate the robustness of the PPI network despite these caveats, we performed independent IPs and reverse IPs followed by immunoblotting and estimated a ∼90% replication rate (or ∼10% false-positive rate) in the network. These results support the technical reproducibility of our data, but they relied on a biochemical method that may share some of the biases of the IP-MS approach. Although we further demonstrated the biological relevance of the network using computational analyses, we recognize that technologies such as multichannel microscopy,76 super-resolution microscopy,77 and cryoelectron microscopy (cryo-EM)78 may constitute avenues to orthogonally validate our PPI data in the future.

Finally, the PPI network was anchored by 13 index proteins and derived from excitatory iNs; thus, it does not represent a complete interactome of all ASD-associated genes across different brain cell types of relevance to ASDs (e.g., inhibitory interneurons). Similarly, while our findings indicate that convergent signals in the network could point to ASD-related pathways, further work is required to determine the extent of convergence between all ASD-associated genes. It will be critical to apply similar approaches to additional ASD-associated genes and cell types as larger genetic datasets and improved cell modeling protocols become available, to build toward a comprehensive cell-type-specific PPI atlas for ASDs.

ConclusionsSo far, very few PPI studies have taken advantage of neuronal cell models, and even fewer used human genetics as both an entry point for index gene selection and a downstream source of data integration. Here, we built a PPI network of 13 index proteins encoded by high-confidence ASD-associated genes in human iPSC-derived excitatory neurons. This network contains many newly reported interactions, is enriched for genetic and transcriptional perturbations observed in ASD individuals, and can be leveraged to uncover neuron-specific biology related to ASDs. Our data not only empower hypothesis generation to study the mechanisms of individual ASD-associated genes but also suggest convergent pathways that may inform therapeutic intervention or clinical stratification strategies. More generally, the approach described in this study and the parallel SCZ study71 can be expanded to generate larger PPI networks across brain cell types and neurodevelopmental stages to uncover novel pathways underlying psychiatric disorders.

Conclusions

So far, very few PPI studies have taken advantage of neuronal cell models, and even fewer used human genetics as both an entry point for index gene selection and a downstream source of data integration. Here, we built a PPI network of 13 index proteins encoded by high-confidence ASD-associated genes in human iPSC-derived excitatory neurons. This network contains many newly reported interactions, is enriched for genetic and transcriptional perturbations observed in ASD individuals, and can be leveraged to uncover neuron-specific biology related to ASDs. Our data not only empower hypothesis generation to study the mechanisms of individual ASD-associated genes but also suggest convergent pathways that may inform therapeutic intervention or clinical stratification strategies. More generally, the approach described in this study and the parallel SCZ study71 can be expanded to generate larger PPI networks across brain cell types and neurodevelopmental stages to uncover novel pathways underlying psychiatric disorders.

STAR★MethodsKey resources table
REAGENT or RESOURCESOURCEIDENTIFIERAntibodiesACTB (B-actin)AbcamCat# ab8227; RRID: AB_2305186ADD1AbcamCat# ab51130; RRID: AB_867519ADD3Proteintech GroupCat# 17585-1-AP;RRID: AB_2223579ADNPFortis Life SciencesCat# A300-104A; RRID: AB_242522AKAP8LFortis Life SciencesCat# A301-366A; RRID: AB_937701ANK2Santa Cruz BiotechnologyCat# sc-365757; RRID: AB_10844971ARID1ACell Signaling TechnologyCat# 12354S; RRID: AB_2637010ARID1BFortis Life SciencesCat# A301-046A; RRID: AB_872976ATXN2L (ATAXN2L)Proteintech GroupCat# 24822-1-AP; RRID: AB_2879743BLOC1S2 (CEAP)AbcamCat# ab174929BORCS5Proteintech GroupCat# 17169-1-AP; RRID: AB_2137150C3AbcamCat# ab97462; RRID: AB_10679468CACNA2D1 (CACNA2D)AbcamCat# ab2864; RRID: AB_303365CALM2 (CALM1/2/3)AbcamCat# ab45689; RRID: AB_725815CBX3AbcamCat# ab213167CDH11InvitrogenCat# 32-1700; RRID: AB_86943CHD4AbcamCat# ab70469; RRID: AB_2229454CHD8AbcamCat# ab84527; RRID: AB_1860139CNTNAP1AbcamCat# ab34151; RRID: AB_869934CTNNA2Sigma AldrichCat# C2081; RRID: AB_476830CTNNB1Cell Signaling TechnologyCat# 8480; RRID: AB_11127855DCAF7AbcamCat# ab138490DLSTCell Signaling TechnologyCat# 11954; RRID: AB_2732907DPYSL2 (CRMP2)AbcamCat# ab129082; RRID: AB_11154701DYNC1H1Fortis Life SciencesCat# A304-720A; RRID: AB_2620915DYRK1AAbcamCat# ab69811; RRID: AB_1209558EIF3GFortis Life SciencesCat# A301-757A; RRID: AB_1210991ELAVL2Proteintech GroupCat# 14008-1-AP; RRID: AB_2096356ELAVL3AbclonalCat# A6091; RRID: AB_2766743ERC1 (ERC)Cell Signaling GroupCat# 2884S; RRID: AB_2262380FAM53CAbcamCat# ab105679; RRID: AB_10862174FMR1 (FMRP)AbcamCat# ab17722; RRID: AB_2278530GAP43AbcamCat# ab75810; RRID: AB_1310252GIGYF1AbcamCat# ab177509HNRNPA2B1 (HNRPA2B1)AbcamCat# ab6102; RRID: AB_305293IGF2BP1Proteintech GroupCat# 22803-1-AP; RRID: AB_2879173IGF2BP2AbcamCat# ab124930; RRID: AB_11131218IGF2BP3Fortis Life SciencesCat# A303-426A; RRID: AB_10951696IGSF3AbcamCat# ab97704; RRID: AB_10679978ISL1AbcamCat# ab109517; RRID: AB_10866454L1CAMAbcamCat# ab208155; RRID: AB_2687508LDB1Novus BiologicalsCat# NBP1-77832; RRID: AB_11038158LRRC8DProteintech GroupCat# 11537-1-AP; RRID: AB_2137767MAP1A (MAP1)Fortis Life SciencesCat# A301-445A; RRID: AB_961038MAP1BAbcamCat# ab224115MED13LFortis Life SciencesCat# A302-420A; RRID: AB_1907303MED14AbcamCat# ab72141; RRID: AB_1209425MED4AbcamCat# ab129170; RRID: AB_11145268mTORCell Signaling TechnologyCat# 2972; RRID: AB_330978PHB2Cell Signaling TechnologyCat# 14085; RRID: AB_2798387PTENCell Signaling TechnologyCat# 9188; RRID: AB_2253290RAB11AAbcamCat# ab128913; RRID: AB_11140633S100A8AbcamCat# ab92331; RRID: AB_2050283SATB1AbcamCat# ab70004; RRID: AB_1270545SCN3AAbcamCat# ab65164; RRID: AB_1141789SCN9AAbcamCat# ab65167; RID: AB_2301407SHANK3Fortis Life SciencesCat# A304-178A; RRID: AB_2621427SMARCB1Cell Signaling TechnologyCat# 91735S; RRID: AB_2800172SMARCC1Cell Signaling TechnologyCat# 11956; RRID: AB_2797776SMARCC2Cell Signaling TechnologyCat# 12760S; RRID: AB_2798017SPTBN1AbcamCat# ab72239; RRID: AB_1270902SYNGAPCell Signaling TechnologyCat# 5539; RRID: AB_10694401SCN2AMilliporeSigmaCat# MABN1586TARDBPAbcamCat# ab41881; RRID: AB_2200643TLK2Fortis Life SciencesCat# A301-257ATRIP12FortisCat# A301-814A; RRID: AB_1264344ZMYND8Cell Signaling TechnologyCat# 97845S; RRID: AB_2800291Rabbit IgG, monoclonal [EPR25A] - Isotype ControlAbcamCat# ab172730; RRID: AB_2687931Rabbit IgG, polyclonal - Isotype ControlAbcamCat# ab37415; RRID: AB_2631996Mouse IgG1, kappa monoclonal [MOPC-21] - isotype controlAbcamCat# ab18443; RRID: AB_2736846Mouse IgG2a, Kappa Monoclonal [MOPC-173] - Isotype ControlAbcamCat# ab18413; RRID: AB_2631983Chemicals, peptides, and recombinant proteinsAccutaseGibcoA11105Agilent RNA 6000 Pico kitAgilent Technologies5067-1513Ambion DNA-free DNase Treatment kitLife TechnologiesAM1906B-27 supplementThermo Scientific17504044BDNFR&D Systems248-BD/CF2100 Bioanalyzer InstrumentAgilent TechnologiesG2939BABlotting-Grade BlockerBio-Rad Laboratories1706404CNTFR&D Systems257-NTCryoStor CS10Stem Cell Technologies07930DMEM/F - 12Life Technologies11320033Doxycycline hyclateSigma-Aldrich24390-14-5Fetal Bovine SerumR&D SystemsS11195GDNFR&D Systems212-GDGeltrexLife TechnologiesA1413301GeneticinGibco10131027D-(+)-GlucoseSigma-Aldrich50-99-7GlutamaxLife Technologies35050079Gridded Plate 150 x 25 mmVWR International25383-103Halt™ Protease and Phosphatase InhibitorThermo Scientific78442Laminin, MouseVWR International47743-734LDN-193189Stem Cell Technologies1435934-00-1MEM NEAALife Technologies11140076Methanol ≥99.8%Fisher Scientific67-56-1N-2 SupplementLife Technologies17502048NeurobasalLife Technologies21103049NormocinInvitrogenant-nr-2NuPAGE™ 12%, Bis-Tris GelInvitrogenNP0341PK2NuPAGE Transfer Buffer (20X)Life TechnologiesNP00061PBSLife Technologies10010049PVDF/Filter Paper SandwichLife TechnologiesLC2002ROCK Inhibitor (Y-27632)Stemgent04-0012SB 431542Tocris1614StemFlex™ MediumLife TechnologiesA3349401Microcentrifuge TubesVWR International0011-702Sterile SleevesVWR International414004-510TrizolLife Technologies15596026TruSeq Stranded Total RNA Library Prep KitIllumina20020596TrypLELife Technologies12604039TrypsinPromegaV5111XAV939Stemgent04-00046Critical commercial assaysPierce™ Protein A/G Magnetic BeadsThermo Scientific88803SuperSignal™ West Femto Maximum Sensitivity SubstrateThermo Scientific34094Pierce™ IP Lysis BufferThermo Scientific87788EnGen Spy Cas9 NLSNew England BiolabsM0646TDeposited dataRNA-seq dataThis studyGEO: GSE178896IP-MS data from Whitehead, CNCR, HarvardThis studyMassIVE: MSV000087676IP-MS data from Broad Proteomics PlatformThis studyMassIVE: MSV000089445Western blot images of IP-WBThis studyhttps://doi.org/10.17632/ppgn28nxkp.1Experimental models: Cell linesiPSC line from human fibroblasts with integrated NGN2Novartis Institutes for BioMedical ResearchiPS hDFn 83/22 iNgn2#9 (iPS3 in Nehme et al.25)Software and algorithmsTrim Galore (v0.6.7)Krueger et al.79https://github.com/FelixKrueger/TrimGaloreHISAT2 (v2.2.1)Kim et al.80http://daehwankimlab.github.io/hisat2/HTSeq (v0.13.5)Anders et al.81https://htseq.readthedocs.io/en/master/EdgeR (v3.34.0)Robinson et al.82https://bioconductor.org/packages/release/bioc/html/edgeR.htmlSeurat (v3.2.0)Stuart et al.83https://github.com/satijalab/seurat/Spectrum Mill (v7.0)Broad Proteomics Platformhttps://proteomics.broadinstitute.orgPEAKS Studio (v8.5)Bioinformatics SolutionsN/AMaxQuant (v1.6.3.4)Cox and Mann84https://www.maxquant.orgProteome Discoverer (v2.4)Thermo ScientificN/AGenoppi (v1.0)Pintacuda et al.18https://github.com/lagelab/Genoppipropr (v4.2.6)Quinn et al.85https://github.com/tpq/proprMAGMA (v1.09)de Leeuw et al.46https://ctg.cncr.nl/software/magmaCustom codeThis studyhttps://github.com/lagelab/ASD_PPI, https://doi.org/10.5281/zenodo.7366631OtherSilencer select siRNA, DYRK1AThermo Scientific4390824Hs.Cas9.AKAP8L.1.AA (sgRNA, AKAP8L)IDT209191778CD.Cas9.VYBY9746.AA (sgRNA, ANK2)IDT210358998CD.Cas9.YTSN3051.AA (sgRNA, ANK2)IDT210358999
Resource availabilityLead contactFurther information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Kasper Lage (lage.kasper@mgh.harvard.edu).Materials availabilityThis study did not generate new unique reagents.Experimental model and subject detailsHuman brain samplesPostmortem brain tissue was obtained from the Netherlands Brain Bank (NBB) at the Netherlands Institute for Neuroscience (NIN), Amsterdam. Brain tissue was collected from two adult donors with written informed consent for brain autopsy and the use of the material and clinical information for research purposes has been obtained by the NBB. PMD was less than 12 hours.Cell linesGlutamatergic patterned induced neurons (iNs) were differentiated from a male induced pluripotent stem cell (iPSC) line (iPS hDFn 83/22 iNgn2#9 [iPS3] in Nehme et al.25) by conditional expression of the neuralizing transcription factor NGN2, as described previously.18,25 A plate was coated with GelTrex (Life Technologies, A1413301) adhesion matrix (1:100 in DMEM/F:12, Life Technologies, 11320033) and cells were seeded at a density of 40,000 cells cm−2 in Stemflex media (Life Technologies, A3349401) containing Geneticin as selective antibiotic (Life Technologies, 10131027) (1:400). Once the cells achieved 60% confluency, the monolayer was detached using Accutase (Life Technologies, A11105) and transferred onto new plates for cell expansion. Human embryonic kidney 293 (HEK293) cells were grown as monolayers in DMEM media (Life Technologies, 11320033) supplemented with 10% FBS (R&D Systems, S11195). Once the cells achieved 60% confluency, the monolayer was detached using Trypsin (Promega, V5111) and transferred onto new plates for cell expansion. Both iNs and HEK293 cells were incubated at 37°C and 5% CO2.Method detailsDifferentiation of iNsiPSCs stably expressing TetO-Ngn2-Neo and reverse tetracycline-controlled transactivator (rtTA) were plated at a density of 40,000 cells cm−2 with rock inhibitor Y27632 (Stemgent, 04-0012). Day 1 cells were differentiated in N2 media (Life Technologies) supplemented with 10 μM SB431542 (Tocris, 1614), 2 μM XAV939 (Stemgent, 04-00046) and 100 nM LDN-193189 (Stemgent, 04-0074) along with doxycycline hyclate (2 μg mL−1). Day 2 media was N2+SB/XAV/LDN/doxycycline hyclate and differentiation media were as previously described. On day 3, cell differentiation was continued in neurobasal media (Life Technologies) supplemented with B27 (50X, Thermo Scientific), brain-derived neurotrophic factor (BDNF), ciliary neurotrophic factor (CTNF), glial cell-derived neurotrophic factor (GDNF) (R&D Systems 248-BD/CF, 257-NT/CF, and 212-GD/CF at 10 ng mL−1) and doxycyclinehyclate (2 μg mL−1).Protein extraction and immunoblottingTotal protein extract was obtained by harvesting cells and either processing them immediately or snap-freezing them on dry ice for storage at −80°C. In both cases, cell pellets were washed with PBS and resuspended in 10× packed cell volume (PCV) IP lysis buffer (Thermo Scientific), with freshly added Halt protease and phosphatase inhibitors (Thermo Scientific). After a 20 min incubation time at 4°C, cells were collected by centrifugation (16,200 g, 20 min, 4°C) and resuspended in 3× PCV lysis buffer. The concentration of the samples was quantified using the Thermo BCA protein assay and when not used immediately, samples were stored at −80°C. Samples for immunoblotting were diluted in 6× SMASH buffer (50 mM Tris HCl pH 6.8, 10% glycerol, 2% SDS, 0.02% bromophenol blue, 1% b-mercaptoethanol), boiled for 10 min at 95°C, separated on a NuPAGE 4-12% Bis-Tris Protein Gel (Invitrogen), and transferred onto a PVDF membrane (Life Technologies) by wet transfer (100 V for 2 hours). Membranes were blocked by incubation for one hour at room temperature in 10 mL TBS and 0.1% Tween (TBST) with 5% w/v BioRad Blotting-grade Blocker. Blots were incubated overnight at 4°C with the primary antibody, washed 3 times for 10 min with TBST and incubated for 45 min with secondary antibody conjugated to horseradish peroxidase. After washing 3 times for 10 min with TBST, bands were visualized using SuperSignal™ West Femto Maximum Sensitivity Substrate (Thermo Scientific). “Forward” IP western blots were performed using membranes loaded with immunoprecipitate of the index protein, which was incubated overnight at 4°C with the primary antibody of the interactor protein; “reverse” IP western blots were performed using membranes loaded with immunoprecipitate of the interactor protein, which was incubated overnight at 4°C with the primary antibody of the index protein. To evaluate the reproducibility of IP-MS interactions by western blot analysis, we used the following criteria: if a band at the expected molecular weight was detected in the IP lane, the interaction was considered to be “replicated” only if the signal exceeded the negative control (IgG lane). If the band of the expected molecular weight was either absent in the IP lane, or yielding a weaker signal than the control lane (IgG) the interaction was considered “not replicated”. A graphical representation of these criteria is shown in Table S5. All antibodies used in this study (including those that were not IP-competent) are listed in Tables S1, S2, and S5.Mouse cortex preparationMouse cortices were isolated from p0 pups of C57BL/6 background, cut into small pieces, and flash frozen. Lysis was performed by adding Pierce IP lysis buffer (Pierce) to frozen pieces and mechanically dissociating them by vigorous pipetting. All subsequent steps were the same as those for iNs.Human brain preparationFor human postmortem brain samples, 0.6 g of tissue was homogenized in 34 mL extraction buffer (0.5% n-Dodecyl β-D-maltoside, 150 mM NaCl, 50 mM Hepes, pH 7.4) with a glass homogenizer (PotterS from B. Braun) set at 900 rpm for 12 strokes. The homogenate was centrifuged twice at 16,000g for 10 min. All subsequent steps were the same as those for iNs.ImmunoprecipitationsFor each individual experiment, 1-2 mg of protein extract from the same cell differentiation batch was incubated at 4°C overnight in the presence of 1-2 μg of the relevant antibody. On the next day, 50 μL of Protein G beads (Pierce) were added to each sample and incubated at 4°C for 4 hours. Flow-through was collected and beads were washed once with 1 mL lysis buffer (Pierce) supplemented with Halt protease and phosphatase inhibitors (Thermo Scientific), and twice with PBS. Beads were resuspended in 60 μL of PBS and 10% of the volume was employed for immunoblotting, after being boiled in 6xSMASH buffer (50 mM Tris HCl pH 6.8, 10% Glycerol, 2% SDS, 0.02% bromophenol blue, 1% b-mercaptoethanol) for 10 min at 95°C. The remaining volume was stored at −80°C and subsequently used for mass spectrometry analysis.Mass spectrometryTable S2 provides an overview of where liquid chromatography-tandem mass spectrometry (LC-MS/MS) for each IP-MS dataset was performed; MS protocols for each facility/site are described below.Broad Proteomics Platform (Broad): Proteins in IP samples were digested on beads using 90 μL of digestion buffer (2 M urea/50 mM Tris buffer with 1 mM DTT and 5 μg/mL Trypsin) for 1 h, shaking at 1000 rpm. The suspension was then transferred to a new tube, and the beads were washed twice with 60 μL of wash buffer (2 M urea/50 mM Tris buffer). The wash buffer was added to the suspension with digestion. The digestion and wash process was repeated a second time pooling the suspensions with the suspensions from the first round. The pooled solution was reduced using 4 mM DTT for 30 min at 25°C shaking at 1000 rpm. The proteins were then alkylated using 10 mM iodoacetamide and incubating for 45 min at 25°C shaking at 1000 rpm and protected from light. Proteins were then digested with 0.5 μg of trypsin overnight at 25°C shaking at 700 rpm. The next day proteins were quenched using 40 μL of 10% formic acid and desalted using an Oasis Cartridge. Samples were vacuum dried and labeled with iTRAQ4 kits (Sciex). Each iTRAQ 4-plex consisted of 2 replicate experimental IPs using an antibody against an index protein (labeled with iTRAQ reagent labels 114 and 115) and 2 replicate control IPs using a control IgG antibody (labeled with 116 and 117). Reconstituted peptides were separated on an online nanoflow EASY-nLC 1000 UHPLC system (Thermo Scientific) and analyzed on a benchtop Orbitrap Q Exactive Plus (DYRK1A, PTEN) or Q Exactive HF (SCN2A, SHANK3) mass spectrometer (Thermo Scientific). The peptide samples were injected onto a capillary column (Picofrit with 10 μm tip opening/75 μm diameter, New Objective, PF360-75-10-N-5) packed in-house with 20 cm C18 silica material (1.9 μm ReproSil-Pur C18-AQ medium, Dr. Maisch GmbH, r119.aq). The UHPLC setup was connected with a custom-fit micro-adapting tee (360 μm, IDEX Health & Science, UH-753), and capillary columns were heated to 50°C in column heater sleeves (Phoenix-ST) to reduce back-pressure during UHPLC separation. Injected peptides were separated at a flow rate of 200 nL/min with a linear 196 min (SCN2A, SHANK3, PTEN) or 234 min (DYRK1A) gradient from 94% solvent A (3% acetonitrile, 0.1% formic acid) to 30% (DYRK1A) or 40% (SCN2A, SHANK3, PTEN) solvent B (90% acetonitrile, 0.1% formic acid), followed by a linear 8 or 9 min gradient to 60% (DYRK1A) or 65% (SCN2A, SHANK3, PTEN) solvent B and a 1 or 3 min ramp to 90% B. The Q Exactive instrument was operated in the data-dependent mode acquiring HCD MS/MS scans (R = 17.5K –QE + or R = 45K-QE-HF) after each MS1 scan (R = 70K-QE + or R = 60K-QE-HF) on the 12(QE+) or 16(QE-HF) most abundant ions using an MS1 ion target of 3 × 106 ions and an MS2 target of 5 × 104 ions. The maximum ion time utilized for the MS/MS scans was 120 ms; the HCD-normalized collision energy was set to 27; the dynamic exclusion time was set to 20s, and the peptide match preferred and isotope exclusion functions were enabled.Whitehead Proteomics Core Facility (Whitehead): IP samples on beads were resuspended in 100 μL of 100 mM triethylammonium bicarbonate (TEAB), reduced using 2 μL of 50 mM tris(2-carboxyethyl)phosphine for 60 min at 60°C, and alkylated using 1 μL of 2% S-methyl methanethiosulfonate in isopropanol for 10 min at room temperature. Proteins were then digested with 250 ng of trypsin overnight at 37°C with gentle shaking. iTRAQ4 reagents (Sciex) were resuspended in 50 μL isopropanol and added to each sample followed by vortex and spin; the samples in each 4-plex consisted of 2 replicate experimental IPs using an antibody against an index protein (labeled with iTRAQ reagent labels 114 and 115) and 2 replicate control IPs using a control IgG antibody (labeled with 116 and 117). The samples were combined and incubated at room temperature for 2 h, and then washed, extracted, and concentrated by solid phase extraction using Waters Sep-Pak Plus C18 cartridges. Organic solvent was removed and the sample volumes were reduced to 80 μL via speed vacuum. The labeled peptides were subjected to basic (high pH) reversed phase high-performance liquid chromatography (HPLC) with fraction collection using Shimadzu LC-20AD pumps and an FRC-10A fraction collector. Samples were loaded on a 10 cm × 2.1 mm column packed with 2.6 micron Aeris PEPTIDE XB-C18 media (Phenomenex). The initial gradient condition was isocratic 1% buffer A (20 mM ammonium formate in water, pH = 10) at 150 μL min-1, with increasing buffer B (acetonitrile) concentrations to 16.7% B at 20.5 min, 30% B at 31 min, and 45% B at 36 min. The column was washed with high percent B and re-equilibrated between analytical runs for a total cycle time of ∼55 min. Sixteen 450 μL fractions (fx) were collected, combined into eight samples (fx1+2, fx3+9, fx4+10, fx5+11, fx6+12, fx7+13, fx8+14, fx15 + 16), then reduced to 20 μL via speed vacuum. The combined samples were subjected to reversed phase HPLC using Thermo EASY-nLC 1200 pumps and autosampler, followed by mass spectrometry using a Thermo Q Exactive HF-X Hybrid Quadrupole-Orbitrap mass spectrometer and a nanoflow configuration. Samples were loaded on a 6 cm x 100 micron column packed with 10 micron ODS-A C18 material (YMC), washed with 4 μL total volume to trap and wash peptides, then eluted onto the analytical column packed with 1.7 micron Aeris C18 material (Phenomenex) in a fritted 14 cm x 75 micron fused silica tubing pulled to a 5 micron tip. The initial gradient condition was 1% buffer A (1% formic acid in water) at 300 nL min-1, with increasing buffer B (1% formic acid in acetonitrile) concentrations to 6% B at 1 min, 21% B at 42.5 min, 36% B at 63.15 min, and 50% B at 73 min. The column was washed with high percent B and re-equilibrated between analytical runs for a total cycle time of ∼97 min. The mass spectrometer was operated in a data-dependent acquisition mode where the 20 most abundant peptides detected in the Orbitrap using full scan mode with a resolution of 60,000 were subjected to daughter ion fragmentation using a resolution of 15,000. A running list of parent ions was tabulated to an exclusion list to increase the number of peptides analyzed throughout the chromatographic run.Center for Neurogenomics and Cognitive Research (CNCR): IP samples were treated and digested by two protocols. For in-gel digestion, proteins were briefly separated by SDS-PAGE electrophoresis. Each sample was split in three fractions covering protein masses of >70kDa, 70-40 kDa and <40 kDa, and proteins were digested by trypsin overnight at 37°C. Filter-aided sample preparation (FASP) protocol was used for human brain postmortem samples. Sample was solubilized in 2% SDS, 100 mM Tris (pH = 8.8) and transferred to the Microcon-30 filter tube (Millipore). After serial washes with 8M urea and 50 mM NH4HCO3, proteins were digested by trypsin overnight at 37°C and collected as a single fraction. The resulting peptides were analyzed by the TripleTOF 5600 + mass spectrometer coupled to an Ultimate 3000 LC system (Dionex, Thermo Scientific). The mass spectrometer was run in micro-mode. Peptides were fractionated on a 200 mm Alltima C18 column (300 μm i.d., 3 μm particle size) at a flow rate of 5 μL min-1. For in-gel digestion protocol, the tryptic peptides from each gel slice were analyzed with a 1 h LC gradient. For FASP protocol, a 2 h LC gradient was used. The eluted peptides were electro-sprayed with a micro-spray needle voltage of 5500 V. The MS survey scan range was 350–1250 m/z acquired for 250 ms. The top 20 precursor ions were selected for 90 ms per MS/MS acquisition, with a threshold of 90 counts. Dynamic exclusion was 10 s. Rolling CID function was activated, with an energy spread of 5 eV.Harvard Center for Mass Spectrometry (Harvard): IP samples were stored in PBS buffer on beads. PBS was removed and samples were dissolved in 50 μL of 50 mM TEAB, followed by trypsin (Promega) digestion for 3 h at 38°C. Digested samples were dried to 20 μL and 10 μL of each sample was injected in the mass spectrometer. LC-MS/MS was performed on a Lumos Tribrid Orbitrap Mass Spectrometer (Thermo Scientific) equipped with Ultimate 3000 (Thermo Scientific) nano-high-performance liquid chromatography. Peptides were separated onto a 150-μm inner diameter microcapillary trapping column, packed with ∼2 cm of C18 Reprosil resin (5 μm, 100 Å, Dr. Maisch GmbH, Germany), followed by separation on a 50-cm analytical column (PharmaFluidics, Ghent, Belgium). Separation was achieved by applying a gradient from 5 to 27% acetonitrile in 0.1% formic acid for >90 min at 200 nL min-1. Electrospray ionization was enabled by applying a voltage of 2 kV using a home-made electrode junction at the end of the microcapillary column and sprayed from metal tips (PepSep, Denmark). MS survey scan was performed in the Orbitrap, in a range of 400–1800 m/z at a resolution of 60,000, followed by the selection of the 20 most intense ions (TOP20) for CID-MS2 fragmentation in the ion trap using a precursor isolation width window of 2 m/z, automatic gain control setting of 10,000, and a maximum ion accumulation of 100 ms. Singly charged ion species were not subjected to collision-induced dissociation fragmentation. Normalized collision energy was set to 35 V and an activation time of 10 ms. Ions within a 10 ppm m/z window around ions selected for MS2 were excluded from further selection for fragmentation for 60 s.RNA extraction and quantificationCell culture medium was rapidly removed from cells cultured on 150mm Petri dishes and 5 mL of Trizol reagent (Life Technologies) was added. After total RNA extraction, samples were treated with DNase using the Ambion DNA-free DNase Treatment kit (Life Technologies) according to the manufacturer’s instructions, and resuspended in water. Samples were subsequently centrifuged at 13,000 rpm for 20 min at 4°C. Pellets were washed in 75% ethanol and resuspended in water. 1 μL RNA was diluted 1:100 in water and quality-checked using the Agilent RNA 6000 Pico kit (Agilent Technologies) according to the manufacturer’s instructions, and ran on a 2100 Bioanalyzer Instrument (Agilent Technologies).RNA-seq library preparation and sequencingLibraries for RNA-seq were constructed using TruSeq Stranded Total RNA Library Prep Kit (Illumina), with the incorporation of dUTP in the second strand synthesis, and sequenced at the UMass Medical School. Sequencing was performed in paired-end reads (2 × 100 bp) using a HiSeq 4000 system (Illumina).Generation of CRISPR/Cas9 edited cell linesAll transgenic iPSC cells in this study are stable expressing lines. To generate stable iPSC cell lines, cells were electroporated using the NEON transfection system and reagents (Life Technologies), according to the manufacturer’s instructions. RNP complexes were generated using EnGen Spy Cas9 NLS (NEB, M0646T), in combination with a single sgRNA for AKAP8L (Hs.Cas9.AKAP8L.1.AA, IDT), and a mix of 2 sgRNAs for ANK2 (CD.Cas9.VYBY9746.AA and CD.Cas9.YTSN3051.AA, IDT). After 24 h, cells were transferred to 90 mm gelatinized Petri dishes. Cells were grown for 10-12 days, with medium being changed every day. Individual iPSC colonies were picked and expanded for further screening.siRNA knock-downSilencer Select siRNA against DYRK1A were purchased from Thermo Scientific (4,390,824). Transfection into cells was performed on coverslips with RNAiMAX Transfection Reagent (Invitrogen 13,778,100), according to the manufacturer’s instructions. KD were assessed 48 h after transfection.Genomic DNA extractionCells from a confluent 150 mm gelatinized Petri dish were harvested and resuspended in 1-3 mL of lysis buffer (10 mM NaCl, 10 mM Tris-HCl pH 7.5, 10 mM EDTA-NaOH pH 8.0, 0.5% Sodium lauroyl sarcosinate) with proteinase K (Thermo Scientific EO0491) added to a final concentration of 200 μg mL-1. Samples were incubated overnight at 55°C. ½5 volume of 5 M NaCl and 2.5 volume of ice-cold 100% ethanol were added. After mixing, a visible white cloud of DNA was extracted using a bent pipette tip and transferred to a clean tube containing 1 mL 70% ethanol. DNA was pelleted (16,100 g, 5 min, 4°C) and air-dried. Subsequently the pellet was resuspended in 300-400 μL 10 mM Tris pH 8.5 and the concentration measured by Nanodrop.Quantification and statistical analysisRNA-seq data analysisStarting with RNA-seq data in FASTQ format, we ran Trim Galore79 (v0.6.7) to trim adapters and low-quality reads using “--stringency 7” and other default settings for Illumina paired-end reads. Sequence alignment was performed using HISAT280 (v2.2.1) with the “--rna-strandness FR” setting; index files for the GRCh38 reference genome with transcripts were downloaded from the HISAT2 website (https://daehwankimlab.github.io/hisat2/download/). The htseq-count script in Htseq81 (v0.13.5) was used to generate gene counts from aligned reads with “--stranded yes --minaqual 1” and other default settings; GRCh38 GTF gene annotation file was obtained from Ensembl86 (release 84). The EdgeR package82 (v3.34.0) was used to remove low-count genes (min.count = 5 in filterByExpr function) before calculating normalized gene counts (method = “TMM” in calcNormFactors function).scRNA-seq t-SNE plotsSingle-cell RNA sequencing (scRNA-seq) data from Velmeshev et al.24 were downloaded from the UCSC Cell Browser (https://cells.ucsc.edu/?ds=autism#). The gene expression matrix (exprMatrix.tsv.gz), cell meta annotations (meta.tsv), dimensionality reduction coordinates (tSNE.coords.tsv.gz), and the Seurat R package83 (v3.2.0) were used to generate t-SNE plots showing either gene expression or cell type annotations across cell clusters.Protein quantification from raw IP-MS dataSoftware and methods used by each MS facility/site to generate protein-level quantification reports from raw IP-MS data are described below; Table S2 links each IP-MS dataset we analyzed to each facility.Broad: Mass spectra were analyzed using Spectrum Mill (v7.0; https://proteomics.broadinstitute.org). For peptide identification, MS/MS spectra were searched against a sequence database for the UniProt human reference proteome, including isoforms, with a set of common laboratory contaminant proteins appended (2017: 65,068 entries). Search parameters included: ESI-QEXACTIVE-HCD scoring parameters, trypsin enzyme specificity with a maximum of two missed cleavages, 40% minimum matched peak intensity, ±20 ppm precursor mass tolerance, ±20 ppm product mass tolerance. Carbamidomethylation of cysteines and iTRAQ4 full labeling of lysines and peptide n-termini were set as fixed modifications. Allowed variable modifications were oxidation of methionine (M), acetyl (ProtN-term), and deamidated (N), with a precursor MH + shift range of −18 to 64 Da. Identities interpreted for individual spectra were automatically designated as valid by optimizing score and delta rank1-rank2 score thresholds separately for each precursor charge state in each LC-MS/MS while allowing a maximum target-decoy-based false discovery rate (FDR) of 1.0% at the spectrum level. Identified peptides were organized into protein groups and subgroups (isoforms and family members) with Spectrum Mill’s subgroup specific option enabled, so that peptides shared between subgroups are ignored when using reporter ion intensities to perform protein-level quantitation.Whitehead: Mass spectra were analyzed using PEAKS Studio (v8.5; Bioinformatics Solutions). Peptide and protein identification was performed by searching against the UniProt human reference proteome, including isoforms (Swiss-Prot/TrEMBL, release 2019_01) together with a set of common contaminants. An FDR threshold of 1% was used for identification of peptides and proteins. Relative ratios of the iTRAQ 4-plex reporter ions were used for protein-level quantitation.CNCR: Mass spectra were analyzed using MaxQuant84 (v1.6.1.0 or v1.6.3.4; https://www.maxquant.org). Peptide and protein identification was performed by searching against the UniProt human database, and label-free quantitation was performed to generate protein-level quantification reports. MaxQuant versions and settings used for each dataset can be found in the deposited parameters file linked to each dataset (Table S12).Harvard: Mass spectra were analyzed using Proteome Discoverer (v2.4; Thermo Scientific). Assignment of MS/MS spectra was performed using the Sequest HT algorithm by searching the data against the UniProt human reference proteome, including isoforms (Swiss-Prot, release 2019_01) as well as other known contaminants such as human keratins and common laboratory contaminants. Sequest HT searches were performed using a 10 ppm precursor ion tolerance and requiring each peptide’s N/C termini to adhere with trypsin protease specificity, while allowing up to two missed cleavages. CID-MS2 spectra were searched with 0.5 Da ion tolerance for fragmentation. Methionine oxidation (+15.99492 Da) was set as variable modification. An MS2 spectra assignment FDR of 1% was applied to both proteins and peptides using the Percolator target-decoy database search. Label-free quantitation was performed to generate protein-level quantification reports.IP-MS data processing and analysisData processing: Starting with protein-level quantification reports generated by the MS analytical software described above, we generally performed the following data processing procedures for each IP-MS dataset: (1) log2 transformation and median normalization of protein intensity values in each bait (i.e., index protein) or IgG control IP sample; (2) removing contaminants, proteins supported by < 1 unique peptide, and proteins detected in < 2 bait samples; (3) imputing missing protein intensity values in each sample by randomly sampling from a normal distribution with mean of μ - 1.8σ and SD of 0.3σ, where μ and σ are the mean and SD of the observed intensity values in the sample87; (4) calculating log2 FC for each replicate pair of bait vs. IgG control samples. However, due to differences in MS protocols, we modified these steps for specific datasets: for data generated at Broad and Whitehead, few missing values were present due to the labeled quantification approach, therefore imputation was not performed and only proteins with no missingness across all bait and control samples were included for analysis; for data generated at CNCR, instead of comparing bait samples against IgG controls to calculate log2 FC, the median of several negative IPs (i.e., bait samples in which the bait protein itself was not detected; Table S12) from the same MS run were used as controls.Genoppi analysis: We analyzed each processed IP-MS dataset using the Genoppi R package18 (v1.0), including: (1) calculating Pearson’s correlation of log2 FC values between replicates; (2) calculating average log2 FC, and corresponding P-value and false discovery rate (FDR), for each protein across replicates using a two-tailed one-sample moderated t-test88; (3) identifying statistically significant (log2 FC > 0 and FDR ≤ 0.1) interactors of the bait; (4) assessing overlap between the identified interactors and known interactors from the InWeb database,17 as curated in the Genoppi R package. We QC’ed each dataset using two criteria: the replicate log2 FC correlation must be > 0.6 and the bait protein itself must be significant (log2 FC > 0 and FDR ≤ 0.1); datasets that failed to meet these criteria were excluded from further analysis. All IP-MS analysis results are provided in Table S2.Defining non-interactors: For each QC’ed IP-MS dataset derived from iNs, besides identifying significant interactors as described above, we also defined a matching set of “non-interactors” (Table S3) to serve as proxy for the background iN proteome in conditional enrichment analyses. We applied different definitions for datasets generated by the labeled vs. label-free quantification approaches: for labeled datasets generated at Broad and Whitehead, we considered all non-significant proteins (i.e., log2 FC ≤ 0 or FDR > 0.1) in Genoppi analysis to be non-interactors; for label-free datasets generated at CNCR, we included both non-significant proteins in Genoppi analysis and other proteins in the MS quantification report that were detected in < 2 bait samples (which were filtered out prior to Genoppi analysis).Generating PPI networks: We consolidated all QC’ed iN-derived IP-MS datasets into a combined PPI network; we also combined the datasets for the same index proteins into index protein-specific networks. The interactors in each network are proteins that show up as significant interactors in ≥ 1 source IP-MS datasets contributing to the network; the matching “non-interactors” are proteins that show up as non-interactors in ≥ 1 source dataset but never as interactors in any source dataset. The full lists of interactors and non-interactors associated with each PPI network are provided in Table S3; we note that in this table, we removed all index proteins from both the interactor and non-interactor lists for the combined PPI network to avoid biasing downstream enrichment analyses.Co-expression analysisPairwise co-expression between each index gene and all other protein-coding genes were estimated using data from four independent studies, including (1) Stickels et al.33: spatial transcriptomics in mouse neocortex (Puck_190921_19.digital_expression.txt.gz retrieved from: https://singlecell.broadinstitute.org/single_cell/study/SCP815/highly-sensitive-spatial-transcriptomics-at-near-cellular-resolution-with-slide-seqv2#study-download); (2) Maynard et al.31: spatial transcriptomics in human dorsolateral prefrontal cortex (count matrix retrieved from spatialLIBD R package: https://github.com/LieberInstitute/HumanPilot); (3) Velmeshev et al.24: single-cell RNA-seq in human cortex (rawMatrix.zip retrieved from: https://cells.ucsc.edu/?ds=autism); and (4) BrainSpan: bulk RNA-seq across human brain regions and developmental stages (“RNA-Seq Gencode v10 summarized to genes” dataset retrieved from: https://www.brainspan.org/static/download.html). We used two different methods for estimating co-expression to account for different statistical properties in these datasets. For the spatial transcriptomic datasets with very sparse gene expression matrices, we reasoned that the binary presence/absence of genes across physical locations would be the most informative, and therefore performed one-tailed Fisher’s exact tests to calculate the significance of co-occurrence for each gene pair across locations. For the other datasets, we calculated a proportionality metric, ⍴, for each gene pair using the propr R package85 (v4.2.6); this metric is analogous to conventional correlation measures but has been shown to be better at capturing functional associations between genes in RNA-seq data.89 After calculating either the Fisher’s exact P-values or the proportionality ⍴ values for all gene pairs involving each index gene, we then performed rank-based inverse normal transformation to convert the values into co-expression Z-scores (where a positive score indicates that a gene has higher than average co-expression with the index gene compared to the rest of the genome). We performed two-tailed Wilcoxon rank-sum tests to assess if the co-expression Z-scores between index genes and their interactors are significantly different from the scores between the index genes and other gene groups including non-interactors detected in IP-MS, known interactors from InWeb, and all protein-coding genes.BrainSpan analysisGene expression in four regions of the frontal cortex (dorsolateral prefrontal cortex, medial prefrontal cortex, ventrolateral prefrontal cortex, orbital frontal cortex) across 10 developmental stages were obtained from the BrainSpan exon microarray dataset32 (www.brainspan.org). At each developmental stage, we calculated the median and SE of the expression values for various gene sets, limiting to genes with available BrainSpan data: (1) ASD-associated genes reaching FWER ≤ 0.05, FDR ≤ 0.1, and FDR ≤ 0.25 in Satterstrom et al.5; (2) index and interactor genes in our combined network; (3) SCN2A, SHANK3, and SYNGAP1 interactors found only in iNs, only in brain homogenates, and shared by both sample types; (4) overlap between the FDR ≤ 0.25 genes from Satterstrom et al. and interactor genes in the combined network; (5) random genes sampled from the BrainSpan dataset for comparison against the other gene sets.Tissue enrichment analyses using GTEx dataGene tissue specificity scores in all (GTEx.tstat.tsv) and brain region (GTEx_brain.tstat.tsv) GTEx tissues, as described in Finucane et al.,34 were downloaded from: https://data.broadinstitute.org/alkesgroup/LDSCORE/LDSC_SEG_ldscores/tstats/. Following the definition in Finucane et al., we defined tissue-specific genes as the 10% of genes with the highest scores in each tissue. One-tailed hypergeometric tests were performed to assess the overlap enrichment between interactors in the combined PPI network vs. the tissue-specific genes in all tissues or the brain region tissues. In each test, the “population” was defined as all genes found in the GTEx data; within this population, “success” was the interactors in the network, “sample” was the tissue-specific genes, and “success in sample” was the overlap.Cell type enrichment analyses using scRNA-seq dataWe downloaded scRNA-seq data reported in Velmeshev et al.24 from the UCSC Cell Browser (https://cells.ucsc.edu/?ds=autism). To define “commonly expressed genes” in each of the 17 annotated cell types, we used the gene expression matrix (exprMatrix.tsv.gz) and the cell meta annotations (meta.tsv) to identify genes that have non-zero expression (i.e., log2(UMI count) > 0) in >50% of cells in a cell type. Separately, we obtained differentially expressed genes (DEGs) between ASD individuals vs. controls in each cell type from Data S4 of Velmeshev et al. One-tailed hypergeometric tests were performed to assess the overlap enrichment between interactors in the combined PPI network vs. either the commonly expressed genes or the DEGs in each cell type. In each test, the “population” was defined as either all genes found in the scRNA-seq dataset (for the global enrichment tests) or the subset of genes that are interactors or non-interactors in the network (for the conditional enrichment tests). Within this population, “success” was the interactors in the network, “sample” was the commonly expressed genes or the DEGs in a cell type, and “success in sample” was the overlap.SynGO analysisWe performed SynGO gene set enrichment analysis for genes in the combined PPI network by inputting their gene symbols into the SynGO browser35 (https://syngoportal.org; dataset release: 20180731). The SynGO “Biological Processes” annotations and the “brain expressed” background set were used in one-tailed Fisher’s exact tests to identify enriched gene sets.GO analysis for ANK2 WT- and KO-specific interactorsWe identified ANK2 “WT-specific” and “KO-specific” interactors by analyzing ANK2 IP-MS experiments performed in wild-type (WT) and giant ANK2 knockout (KO) neural progenitor cells. Specifically, the WT-specific interactors are proteins that are significant (log2 FC > 0 and FDR ≤ 0.1) in the ANK2 IP in WT cells (compared to IgG controls) but not in the ANK2 IP in KO cells; and vice versa for the KO-specific interactors. We performed GO term enrichment analysis using the GO cellular component (CC) annotations provided in the Genoppi R package18 (v1.0), which were originally downloaded from the Gene Ontology Consortium39,40 (2020-03-23 release; http://current.geneontology.org/products/pages/downloads.html). One-tailed hypergeometric tests were performed to assess the overrepresentation of CC terms among the WT- or KO-specific interactors. In each test, the “population” contained genes detected in ≥ 2 ANK2 IP replicates in WT cells or ≥ 2 ANK2 IP replicates in KO cells. Within this population, “success” was the genes annotated with a CC term, “sample” was the WT- or KO-specific interactors, and “success in sample” was the overlap.Enrichment analyses for IGF2BP1-3 targetsRNA targets of IGF2BP1, IGF2BP2, and IGF2BP3 were obtained from Table S1 of Huang et al.44; these targets were identified by both RIP and PAR-CLIP methods in HEK293T cells. First, we performed one-tailed hypergeometric tests to assess the overlap enrichment between target genes of each IGF2BP or their combined target list vs. the ASD-associated genes reaching FDR ≤0.1 in Satterstrom et al.5. In each test, the “population” was defined as all autosomal genes in Table S2 of Satterstrom et al.; within this population, “success” was the target genes, “sample” was the ASD-associated genes, and “success in sample” was the overlap. Second, to test whether the IGF2BP1-3 targets are enriched for common variants of ASDs, we performed a global MAGMA analysis as described in Common variant enrichment analysis below, except the “gene sets” tested here were the IGF2BP1-3 targets instead of the PPI network genes. Finally, we performed another set of one-tailed hypergeometric tests to assess the overlap enrichment between the IGF2BP1-3 targets vs. interactors in the combined or index protein-specific PPI networks. In each test, the “population” contained genes that are interactors or non-interactors in a network; within this population, “success” was the target genes, “sample” was the interactors in the network, and “success in sample” was the overlap.Rare variant enrichment analysisGene-based rare variant association scores were extracted from exome sequencing studies of ASD (“qval_dnccPTV” column in Table S2 of Satterstrom et al.5), DD (“denovoWEST_p_full” column in Table S2 of Kaplanis et al.50), and SCZ (“P meta” column in Table S5 of Singh et al.51). In the global enrichment analysis, for each PPI network and each trait, a one-tailed Kolmogorov-Smirnov test was performed to compare the score distribution of the interactor genes against the rest of the genome (with available score data), testing the alternative hypothesis that the cumulative distribution function of the interactors lies above that of other genes (i.e., the interactors have smaller/more significant scores). In the conditional enrichment analysis, an analogous one-tailed Kolmogorov-Smirnov test was performed to compare the scores of the interactors vs. the non-interactors associated with each PPI network.pLI score enrichment analysisGene pLI scores were obtained from the gnomAD52 (v2.1.1) “pLoF Metrics by Gene TSV” dataset. For each PPI network, a one-tailed Kolmorgov-Smirnov test was performed to compare the pLI scores of the interactor genes against other genes in the pLI dataset (for the global enrichment analysis) or against the non-interactors (for the conditional enrichment analysis), testing the alternative hypothesis that the cumulative distribution function of the interactors lies below that of other genes (i.e., the interactors have higher pLI scores).Common variant enrichment analysisGWAS summary statistics for ASD,45 ADHD,53 BIP,54 MDD,55 and SCZ56 were downloaded from the Psychiatric Genomics Consortium (www.med.unc.edu/pgc/download-results/). GWAS summary statistics for height57 were downloaded from the GIANT consortium (https://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium_data_files). All these GWAS were conducted using individuals of European (EUR) ancestry. The following analyses were performed using MAGMA46 (v1.09) to calculate the common variant enrichment of each PPI network for each GWAS phenotype. First, variants in the 1000 Genomes90 (phase 3) EUR panel were mapped to protein-coding genes in the Ensembl86 GRCh37 database with a flanking window of ±50 kb; variants in the major histocompatibility complex region (chr6:28.5M–33.4M) were excluded. Next, for each GWAS dataset, gene-based P-values were calculated using the SNP-wise Mean model and linkage disequilibrium information estimated from the 1000 Genomes EUR panel. Finally, for each GWAS dataset and each PPI network, the gene-set analysis model in MAGMA was used to compare the interactor genes in the network against the rest of the protein-coding genome (for the global enrichment tests) or the non-interactor genes (for the conditional enrichment tests), computing a one-tailed P-value to support the alternative hypothesis that the interactors are more strongly associated with the GWAS phenotype.Social Manhattan plotASD-associated genes with FDR ≤ 0.25 were obtained from Table S2 of Satterstrom et al.,5 along with their GRCh38 genomic positions and Q-values. The combined PPI network was intersected with the ASD gene list to identify the subset of genes that are either an index gene or an interactor in the network. The genomic positions and -log10 Q-values of these genes were plotted in a Manhattan plot, with visual links between the genes to indicate PPIs in the network. For visualization, the minimum Q-value was set to 1e-16 in the plot.

STAR★Methods

Key resources table
REAGENT or RESOURCESOURCEIDENTIFIERAntibodiesACTB (B-actin)AbcamCat# ab8227; RRID: AB_2305186ADD1AbcamCat# ab51130; RRID: AB_867519ADD3Proteintech GroupCat# 17585-1-AP;RRID: AB_2223579ADNPFortis Life SciencesCat# A300-104A; RRID: AB_242522AKAP8LFortis Life SciencesCat# A301-366A; RRID: AB_937701ANK2Santa Cruz BiotechnologyCat# sc-365757; RRID: AB_10844971ARID1ACell Signaling TechnologyCat# 12354S; RRID: AB_2637010ARID1BFortis Life SciencesCat# A301-046A; RRID: AB_872976ATXN2L (ATAXN2L)Proteintech GroupCat# 24822-1-AP; RRID: AB_2879743BLOC1S2 (CEAP)AbcamCat# ab174929BORCS5Proteintech GroupCat# 17169-1-AP; RRID: AB_2137150C3AbcamCat# ab97462; RRID: AB_10679468CACNA2D1 (CACNA2D)AbcamCat# ab2864; RRID: AB_303365CALM2 (CALM1/2/3)AbcamCat# ab45689; RRID: AB_725815CBX3AbcamCat# ab213167CDH11InvitrogenCat# 32-1700; RRID: AB_86943CHD4AbcamCat# ab70469; RRID: AB_2229454CHD8AbcamCat# ab84527; RRID: AB_1860139CNTNAP1AbcamCat# ab34151; RRID: AB_869934CTNNA2Sigma AldrichCat# C2081; RRID: AB_476830CTNNB1Cell Signaling TechnologyCat# 8480; RRID: AB_11127855DCAF7AbcamCat# ab138490DLSTCell Signaling TechnologyCat# 11954; RRID: AB_2732907DPYSL2 (CRMP2)AbcamCat# ab129082; RRID: AB_11154701DYNC1H1Fortis Life SciencesCat# A304-720A; RRID: AB_2620915DYRK1AAbcamCat# ab69811; RRID: AB_1209558EIF3GFortis Life SciencesCat# A301-757A; RRID: AB_1210991ELAVL2Proteintech GroupCat# 14008-1-AP; RRID: AB_2096356ELAVL3AbclonalCat# A6091; RRID: AB_2766743ERC1 (ERC)Cell Signaling GroupCat# 2884S; RRID: AB_2262380FAM53CAbcamCat# ab105679; RRID: AB_10862174FMR1 (FMRP)AbcamCat# ab17722; RRID: AB_2278530GAP43AbcamCat# ab75810; RRID: AB_1310252GIGYF1AbcamCat# ab177509HNRNPA2B1 (HNRPA2B1)AbcamCat# ab6102; RRID: AB_305293IGF2BP1Proteintech GroupCat# 22803-1-AP; RRID: AB_2879173IGF2BP2AbcamCat# ab124930; RRID: AB_11131218IGF2BP3Fortis Life SciencesCat# A303-426A; RRID: AB_10951696IGSF3AbcamCat# ab97704; RRID: AB_10679978ISL1AbcamCat# ab109517; RRID: AB_10866454L1CAMAbcamCat# ab208155; RRID: AB_2687508LDB1Novus BiologicalsCat# NBP1-77832; RRID: AB_11038158LRRC8DProteintech GroupCat# 11537-1-AP; RRID: AB_2137767MAP1A (MAP1)Fortis Life SciencesCat# A301-445A; RRID: AB_961038MAP1BAbcamCat# ab224115MED13LFortis Life SciencesCat# A302-420A; RRID: AB_1907303MED14AbcamCat# ab72141; RRID: AB_1209425MED4AbcamCat# ab129170; RRID: AB_11145268mTORCell Signaling TechnologyCat# 2972; RRID: AB_330978PHB2Cell Signaling TechnologyCat# 14085; RRID: AB_2798387PTENCell Signaling TechnologyCat# 9188; RRID: AB_2253290RAB11AAbcamCat# ab128913; RRID: AB_11140633S100A8AbcamCat# ab92331; RRID: AB_2050283SATB1AbcamCat# ab70004; RRID: AB_1270545SCN3AAbcamCat# ab65164; RRID: AB_1141789SCN9AAbcamCat# ab65167; RID: AB_2301407SHANK3Fortis Life SciencesCat# A304-178A; RRID: AB_2621427SMARCB1Cell Signaling TechnologyCat# 91735S; RRID: AB_2800172SMARCC1Cell Signaling TechnologyCat# 11956; RRID: AB_2797776SMARCC2Cell Signaling TechnologyCat# 12760S; RRID: AB_2798017SPTBN1AbcamCat# ab72239; RRID: AB_1270902SYNGAPCell Signaling TechnologyCat# 5539; RRID: AB_10694401SCN2AMilliporeSigmaCat# MABN1586TARDBPAbcamCat# ab41881; RRID: AB_2200643TLK2Fortis Life SciencesCat# A301-257ATRIP12FortisCat# A301-814A; RRID: AB_1264344ZMYND8Cell Signaling TechnologyCat# 97845S; RRID: AB_2800291Rabbit IgG, monoclonal [EPR25A] - Isotype ControlAbcamCat# ab172730; RRID: AB_2687931Rabbit IgG, polyclonal - Isotype ControlAbcamCat# ab37415; RRID: AB_2631996Mouse IgG1, kappa monoclonal [MOPC-21] - isotype controlAbcamCat# ab18443; RRID: AB_2736846Mouse IgG2a, Kappa Monoclonal [MOPC-173] - Isotype ControlAbcamCat# ab18413; RRID: AB_2631983Chemicals, peptides, and recombinant proteinsAccutaseGibcoA11105Agilent RNA 6000 Pico kitAgilent Technologies5067-1513Ambion DNA-free DNase Treatment kitLife TechnologiesAM1906B-27 supplementThermo Scientific17504044BDNFR&D Systems248-BD/CF2100 Bioanalyzer InstrumentAgilent TechnologiesG2939BABlotting-Grade BlockerBio-Rad Laboratories1706404CNTFR&D Systems257-NTCryoStor CS10Stem Cell Technologies07930DMEM/F - 12Life Technologies11320033Doxycycline hyclateSigma-Aldrich24390-14-5Fetal Bovine SerumR&D SystemsS11195GDNFR&D Systems212-GDGeltrexLife TechnologiesA1413301GeneticinGibco10131027D-(+)-GlucoseSigma-Aldrich50-99-7GlutamaxLife Technologies35050079Gridded Plate 150 x 25 mmVWR International25383-103Halt™ Protease and Phosphatase InhibitorThermo Scientific78442Laminin, MouseVWR International47743-734LDN-193189Stem Cell Technologies1435934-00-1MEM NEAALife Technologies11140076Methanol ≥99.8%Fisher Scientific67-56-1N-2 SupplementLife Technologies17502048NeurobasalLife Technologies21103049NormocinInvitrogenant-nr-2NuPAGE™ 12%, Bis-Tris GelInvitrogenNP0341PK2NuPAGE Transfer Buffer (20X)Life TechnologiesNP00061PBSLife Technologies10010049PVDF/Filter Paper SandwichLife TechnologiesLC2002ROCK Inhibitor (Y-27632)Stemgent04-0012SB 431542Tocris1614StemFlex™ MediumLife TechnologiesA3349401Microcentrifuge TubesVWR International0011-702Sterile SleevesVWR International414004-510TrizolLife Technologies15596026TruSeq Stranded Total RNA Library Prep KitIllumina20020596TrypLELife Technologies12604039TrypsinPromegaV5111XAV939Stemgent04-00046Critical commercial assaysPierce™ Protein A/G Magnetic BeadsThermo Scientific88803SuperSignal™ West Femto Maximum Sensitivity SubstrateThermo Scientific34094Pierce™ IP Lysis BufferThermo Scientific87788EnGen Spy Cas9 NLSNew England BiolabsM0646TDeposited dataRNA-seq dataThis studyGEO: GSE178896IP-MS data from Whitehead, CNCR, HarvardThis studyMassIVE: MSV000087676IP-MS data from Broad Proteomics PlatformThis studyMassIVE: MSV000089445Western blot images of IP-WBThis studyhttps://doi.org/10.17632/ppgn28nxkp.1Experimental models: Cell linesiPSC line from human fibroblasts with integrated NGN2Novartis Institutes for BioMedical ResearchiPS hDFn 83/22 iNgn2#9 (iPS3 in Nehme et al.25)Software and algorithmsTrim Galore (v0.6.7)Krueger et al.79https://github.com/FelixKrueger/TrimGaloreHISAT2 (v2.2.1)Kim et al.80http://daehwankimlab.github.io/hisat2/HTSeq (v0.13.5)Anders et al.81https://htseq.readthedocs.io/en/master/EdgeR (v3.34.0)Robinson et al.82https://bioconductor.org/packages/release/bioc/html/edgeR.htmlSeurat (v3.2.0)Stuart et al.83https://github.com/satijalab/seurat/Spectrum Mill (v7.0)Broad Proteomics Platformhttps://proteomics.broadinstitute.orgPEAKS Studio (v8.5)Bioinformatics SolutionsN/AMaxQuant (v1.6.3.4)Cox and Mann84https://www.maxquant.orgProteome Discoverer (v2.4)Thermo ScientificN/AGenoppi (v1.0)Pintacuda et al.18https://github.com/lagelab/Genoppipropr (v4.2.6)Quinn et al.85https://github.com/tpq/proprMAGMA (v1.09)de Leeuw et al.46https://ctg.cncr.nl/software/magmaCustom codeThis studyhttps://github.com/lagelab/ASD_PPI, https://doi.org/10.5281/zenodo.7366631OtherSilencer select siRNA, DYRK1AThermo Scientific4390824Hs.Cas9.AKAP8L.1.AA (sgRNA, AKAP8L)IDT209191778CD.Cas9.VYBY9746.AA (sgRNA, ANK2)IDT210358998CD.Cas9.YTSN3051.AA (sgRNA, ANK2)IDT210358999

Key resources table

REAGENT or RESOURCESOURCEIDENTIFIERAntibodiesACTB (B-actin)AbcamCat# ab8227; RRID: AB_2305186ADD1AbcamCat# ab51130; RRID: AB_867519ADD3Proteintech GroupCat# 17585-1-AP;RRID: AB_2223579ADNPFortis Life SciencesCat# A300-104A; RRID: AB_242522AKAP8LFortis Life SciencesCat# A301-366A; RRID: AB_937701ANK2Santa Cruz BiotechnologyCat# sc-365757; RRID: AB_10844971ARID1ACell Signaling TechnologyCat# 12354S; RRID: AB_2637010ARID1BFortis Life SciencesCat# A301-046A; RRID: AB_872976ATXN2L (ATAXN2L)Proteintech GroupCat# 24822-1-AP; RRID: AB_2879743BLOC1S2 (CEAP)AbcamCat# ab174929BORCS5Proteintech GroupCat# 17169-1-AP; RRID: AB_2137150C3AbcamCat# ab97462; RRID: AB_10679468CACNA2D1 (CACNA2D)AbcamCat# ab2864; RRID: AB_303365CALM2 (CALM1/2/3)AbcamCat# ab45689; RRID: AB_725815CBX3AbcamCat# ab213167CDH11InvitrogenCat# 32-1700; RRID: AB_86943CHD4AbcamCat# ab70469; RRID: AB_2229454CHD8AbcamCat# ab84527; RRID: AB_1860139CNTNAP1AbcamCat# ab34151; RRID: AB_869934CTNNA2Sigma AldrichCat# C2081; RRID: AB_476830CTNNB1Cell Signaling TechnologyCat# 8480; RRID: AB_11127855DCAF7AbcamCat# ab138490DLSTCell Signaling TechnologyCat# 11954; RRID: AB_2732907DPYSL2 (CRMP2)AbcamCat# ab129082; RRID: AB_11154701DYNC1H1Fortis Life SciencesCat# A304-720A; RRID: AB_2620915DYRK1AAbcamCat# ab69811; RRID: AB_1209558EIF3GFortis Life SciencesCat# A301-757A; RRID: AB_1210991ELAVL2Proteintech GroupCat# 14008-1-AP; RRID: AB_2096356ELAVL3AbclonalCat# A6091; RRID: AB_2766743ERC1 (ERC)Cell Signaling GroupCat# 2884S; RRID: AB_2262380FAM53CAbcamCat# ab105679; RRID: AB_10862174FMR1 (FMRP)AbcamCat# ab17722; RRID: AB_2278530GAP43AbcamCat# ab75810; RRID: AB_1310252GIGYF1AbcamCat# ab177509HNRNPA2B1 (HNRPA2B1)AbcamCat# ab6102; RRID: AB_305293IGF2BP1Proteintech GroupCat# 22803-1-AP; RRID: AB_2879173IGF2BP2AbcamCat# ab124930; RRID: AB_11131218IGF2BP3Fortis Life SciencesCat# A303-426A; RRID: AB_10951696IGSF3AbcamCat# ab97704; RRID: AB_10679978ISL1AbcamCat# ab109517; RRID: AB_10866454L1CAMAbcamCat# ab208155; RRID: AB_2687508LDB1Novus BiologicalsCat# NBP1-77832; RRID: AB_11038158LRRC8DProteintech GroupCat# 11537-1-AP; RRID: AB_2137767MAP1A (MAP1)Fortis Life SciencesCat# A301-445A; RRID: AB_961038MAP1BAbcamCat# ab224115MED13LFortis Life SciencesCat# A302-420A; RRID: AB_1907303MED14AbcamCat# ab72141; RRID: AB_1209425MED4AbcamCat# ab129170; RRID: AB_11145268mTORCell Signaling TechnologyCat# 2972; RRID: AB_330978PHB2Cell Signaling TechnologyCat# 14085; RRID: AB_2798387PTENCell Signaling TechnologyCat# 9188; RRID: AB_2253290RAB11AAbcamCat# ab128913; RRID: AB_11140633S100A8AbcamCat# ab92331; RRID: AB_2050283SATB1AbcamCat# ab70004; RRID: AB_1270545SCN3AAbcamCat# ab65164; RRID: AB_1141789SCN9AAbcamCat# ab65167; RID: AB_2301407SHANK3Fortis Life SciencesCat# A304-178A; RRID: AB_2621427SMARCB1Cell Signaling TechnologyCat# 91735S; RRID: AB_2800172SMARCC1Cell Signaling TechnologyCat# 11956; RRID: AB_2797776SMARCC2Cell Signaling TechnologyCat# 12760S; RRID: AB_2798017SPTBN1AbcamCat# ab72239; RRID: AB_1270902SYNGAPCell Signaling TechnologyCat# 5539; RRID: AB_10694401SCN2AMilliporeSigmaCat# MABN1586TARDBPAbcamCat# ab41881; RRID: AB_2200643TLK2Fortis Life SciencesCat# A301-257ATRIP12FortisCat# A301-814A; RRID: AB_1264344ZMYND8Cell Signaling TechnologyCat# 97845S; RRID: AB_2800291Rabbit IgG, monoclonal [EPR25A] - Isotype ControlAbcamCat# ab172730; RRID: AB_2687931Rabbit IgG, polyclonal - Isotype ControlAbcamCat# ab37415; RRID: AB_2631996Mouse IgG1, kappa monoclonal [MOPC-21] - isotype controlAbcamCat# ab18443; RRID: AB_2736846Mouse IgG2a, Kappa Monoclonal [MOPC-173] - Isotype ControlAbcamCat# ab18413; RRID: AB_2631983Chemicals, peptides, and recombinant proteinsAccutaseGibcoA11105Agilent RNA 6000 Pico kitAgilent Technologies5067-1513Ambion DNA-free DNase Treatment kitLife TechnologiesAM1906B-27 supplementThermo Scientific17504044BDNFR&D Systems248-BD/CF2100 Bioanalyzer InstrumentAgilent TechnologiesG2939BABlotting-Grade BlockerBio-Rad Laboratories1706404CNTFR&D Systems257-NTCryoStor CS10Stem Cell Technologies07930DMEM/F - 12Life Technologies11320033Doxycycline hyclateSigma-Aldrich24390-14-5Fetal Bovine SerumR&D SystemsS11195GDNFR&D Systems212-GDGeltrexLife TechnologiesA1413301GeneticinGibco10131027D-(+)-GlucoseSigma-Aldrich50-99-7GlutamaxLife Technologies35050079Gridded Plate 150 x 25 mmVWR International25383-103Halt™ Protease and Phosphatase InhibitorThermo Scientific78442Laminin, MouseVWR International47743-734LDN-193189Stem Cell Technologies1435934-00-1MEM NEAALife Technologies11140076Methanol ≥99.8%Fisher Scientific67-56-1N-2 SupplementLife Technologies17502048NeurobasalLife Technologies21103049NormocinInvitrogenant-nr-2NuPAGE™ 12%, Bis-Tris GelInvitrogenNP0341PK2NuPAGE Transfer Buffer (20X)Life TechnologiesNP00061PBSLife Technologies10010049PVDF/Filter Paper SandwichLife TechnologiesLC2002ROCK Inhibitor (Y-27632)Stemgent04-0012SB 431542Tocris1614StemFlex™ MediumLife TechnologiesA3349401Microcentrifuge TubesVWR International0011-702Sterile SleevesVWR International414004-510TrizolLife Technologies15596026TruSeq Stranded Total RNA Library Prep KitIllumina20020596TrypLELife Technologies12604039TrypsinPromegaV5111XAV939Stemgent04-00046Critical commercial assaysPierce™ Protein A/G Magnetic BeadsThermo Scientific88803SuperSignal™ West Femto Maximum Sensitivity SubstrateThermo Scientific34094Pierce™ IP Lysis BufferThermo Scientific87788EnGen Spy Cas9 NLSNew England BiolabsM0646TDeposited dataRNA-seq dataThis studyGEO: GSE178896IP-MS data from Whitehead, CNCR, HarvardThis studyMassIVE: MSV000087676IP-MS data from Broad Proteomics PlatformThis studyMassIVE: MSV000089445Western blot images of IP-WBThis studyhttps://doi.org/10.17632/ppgn28nxkp.1Experimental models: Cell linesiPSC line from human fibroblasts with integrated NGN2Novartis Institutes for BioMedical ResearchiPS hDFn 83/22 iNgn2#9 (iPS3 in Nehme et al.25)Software and algorithmsTrim Galore (v0.6.7)Krueger et al.79https://github.com/FelixKrueger/TrimGaloreHISAT2 (v2.2.1)Kim et al.80http://daehwankimlab.github.io/hisat2/HTSeq (v0.13.5)Anders et al.81https://htseq.readthedocs.io/en/master/EdgeR (v3.34.0)Robinson et al.82https://bioconductor.org/packages/release/bioc/html/edgeR.htmlSeurat (v3.2.0)Stuart et al.83https://github.com/satijalab/seurat/Spectrum Mill (v7.0)Broad Proteomics Platformhttps://proteomics.broadinstitute.orgPEAKS Studio (v8.5)Bioinformatics SolutionsN/AMaxQuant (v1.6.3.4)Cox and Mann84https://www.maxquant.orgProteome Discoverer (v2.4)Thermo ScientificN/AGenoppi (v1.0)Pintacuda et al.18https://github.com/lagelab/Genoppipropr (v4.2.6)Quinn et al.85https://github.com/tpq/proprMAGMA (v1.09)de Leeuw et al.46https://ctg.cncr.nl/software/magmaCustom codeThis studyhttps://github.com/lagelab/ASD_PPI, https://doi.org/10.5281/zenodo.7366631OtherSilencer select siRNA, DYRK1AThermo Scientific4390824Hs.Cas9.AKAP8L.1.AA (sgRNA, AKAP8L)IDT209191778CD.Cas9.VYBY9746.AA (sgRNA, ANK2)IDT210358998CD.Cas9.YTSN3051.AA (sgRNA, ANK2)IDT210358999

Resource availabilityLead contactFurther information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Kasper Lage (lage.kasper@mgh.harvard.edu).Materials availabilityThis study did not generate new unique reagents.

Resource availability

Lead contactFurther information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Kasper Lage (lage.kasper@mgh.harvard.edu).

Lead contact

Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Kasper Lage (lage.kasper@mgh.harvard.edu).

Materials availabilityThis study did not generate new unique reagents.

Materials availability

This study did not generate new unique reagents.

Experimental model and subject detailsHuman brain samplesPostmortem brain tissue was obtained from the Netherlands Brain Bank (NBB) at the Netherlands Institute for Neuroscience (NIN), Amsterdam. Brain tissue was collected from two adult donors with written informed consent for brain autopsy and the use of the material and clinical information for research purposes has been obtained by the NBB. PMD was less than 12 hours.Cell linesGlutamatergic patterned induced neurons (iNs) were differentiated from a male induced pluripotent stem cell (iPSC) line (iPS hDFn 83/22 iNgn2#9 [iPS3] in Nehme et al.25) by conditional expression of the neuralizing transcription factor NGN2, as described previously.18,25 A plate was coated with GelTrex (Life Technologies, A1413301) adhesion matrix (1:100 in DMEM/F:12, Life Technologies, 11320033) and cells were seeded at a density of 40,000 cells cm−2 in Stemflex media (Life Technologies, A3349401) containing Geneticin as selective antibiotic (Life Technologies, 10131027) (1:400). Once the cells achieved 60% confluency, the monolayer was detached using Accutase (Life Technologies, A11105) and transferred onto new plates for cell expansion. Human embryonic kidney 293 (HEK293) cells were grown as monolayers in DMEM media (Life Technologies, 11320033) supplemented with 10% FBS (R&D Systems, S11195). Once the cells achieved 60% confluency, the monolayer was detached using Trypsin (Promega, V5111) and transferred onto new plates for cell expansion. Both iNs and HEK293 cells were incubated at 37°C and 5% CO2.

Experimental model and subject details

Human brain samplesPostmortem brain tissue was obtained from the Netherlands Brain Bank (NBB) at the Netherlands Institute for Neuroscience (NIN), Amsterdam. Brain tissue was collected from two adult donors with written informed consent for brain autopsy and the use of the material and clinical information for research purposes has been obtained by the NBB. PMD was less than 12 hours.

Human brain samples

Postmortem brain tissue was obtained from the Netherlands Brain Bank (NBB) at the Netherlands Institute for Neuroscience (NIN), Amsterdam. Brain tissue was collected from two adult donors with written informed consent for brain autopsy and the use of the material and clinical information for research purposes has been obtained by the NBB. PMD was less than 12 hours.

Cell linesGlutamatergic patterned induced neurons (iNs) were differentiated from a male induced pluripotent stem cell (iPSC) line (iPS hDFn 83/22 iNgn2#9 [iPS3] in Nehme et al.25) by conditional expression of the neuralizing transcription factor NGN2, as described previously.18,25 A plate was coated with GelTrex (Life Technologies, A1413301) adhesion matrix (1:100 in DMEM/F:12, Life Technologies, 11320033) and cells were seeded at a density of 40,000 cells cm−2 in Stemflex media (Life Technologies, A3349401) containing Geneticin as selective antibiotic (Life Technologies, 10131027) (1:400). Once the cells achieved 60% confluency, the monolayer was detached using Accutase (Life Technologies, A11105) and transferred onto new plates for cell expansion. Human embryonic kidney 293 (HEK293) cells were grown as monolayers in DMEM media (Life Technologies, 11320033) supplemented with 10% FBS (R&D Systems, S11195). Once the cells achieved 60% confluency, the monolayer was detached using Trypsin (Promega, V5111) and transferred onto new plates for cell expansion. Both iNs and HEK293 cells were incubated at 37°C and 5% CO2.

Glutamatergic patterned induced neurons (iNs) were differentiated from a male induced pluripotent stem cell (iPSC) line (iPS hDFn 83/22 iNgn2#9 [iPS3] in Nehme et al.25) by conditional expression of the neuralizing transcription factor NGN2, as described previously.18,25 A plate was coated with GelTrex (Life Technologies, A1413301) adhesion matrix (1:100 in DMEM/F:12, Life Technologies, 11320033) and cells were seeded at a density of 40,000 cells cm−2 in Stemflex media (Life Technologies, A3349401) containing Geneticin as selective antibiotic (Life Technologies, 10131027) (1:400). Once the cells achieved 60% confluency, the monolayer was detached using Accutase (Life Technologies, A11105) and transferred onto new plates for cell expansion. Human embryonic kidney 293 (HEK293) cells were grown as monolayers in DMEM media (Life Technologies, 11320033) supplemented with 10% FBS (R&D Systems, S11195). Once the cells achieved 60% confluency, the monolayer was detached using Trypsin (Promega, V5111) and transferred onto new plates for cell expansion. Both iNs and HEK293 cells were incubated at 37°C and 5% CO2.

Method detailsDifferentiation of iNsiPSCs stably expressing TetO-Ngn2-Neo and reverse tetracycline-controlled transactivator (rtTA) were plated at a density of 40,000 cells cm−2 with rock inhibitor Y27632 (Stemgent, 04-0012). Day 1 cells were differentiated in N2 media (Life Technologies) supplemented with 10 μM SB431542 (Tocris, 1614), 2 μM XAV939 (Stemgent, 04-00046) and 100 nM LDN-193189 (Stemgent, 04-0074) along with doxycycline hyclate (2 μg mL−1). Day 2 media was N2+SB/XAV/LDN/doxycycline hyclate and differentiation media were as previously described. On day 3, cell differentiation was continued in neurobasal media (Life Technologies) supplemented with B27 (50X, Thermo Scientific), brain-derived neurotrophic factor (BDNF), ciliary neurotrophic factor (CTNF), glial cell-derived neurotrophic factor (GDNF) (R&D Systems 248-BD/CF, 257-NT/CF, and 212-GD/CF at 10 ng mL−1) and doxycyclinehyclate (2 μg mL−1).Protein extraction and immunoblottingTotal protein extract was obtained by harvesting cells and either processing them immediately or snap-freezing them on dry ice for storage at −80°C. In both cases, cell pellets were washed with PBS and resuspended in 10× packed cell volume (PCV) IP lysis buffer (Thermo Scientific), with freshly added Halt protease and phosphatase inhibitors (Thermo Scientific). After a 20 min incubation time at 4°C, cells were collected by centrifugation (16,200 g, 20 min, 4°C) and resuspended in 3× PCV lysis buffer. The concentration of the samples was quantified using the Thermo BCA protein assay and when not used immediately, samples were stored at −80°C. Samples for immunoblotting were diluted in 6× SMASH buffer (50 mM Tris HCl pH 6.8, 10% glycerol, 2% SDS, 0.02% bromophenol blue, 1% b-mercaptoethanol), boiled for 10 min at 95°C, separated on a NuPAGE 4-12% Bis-Tris Protein Gel (Invitrogen), and transferred onto a PVDF membrane (Life Technologies) by wet transfer (100 V for 2 hours). Membranes were blocked by incubation for one hour at room temperature in 10 mL TBS and 0.1% Tween (TBST) with 5% w/v BioRad Blotting-grade Blocker. Blots were incubated overnight at 4°C with the primary antibody, washed 3 times for 10 min with TBST and incubated for 45 min with secondary antibody conjugated to horseradish peroxidase. After washing 3 times for 10 min with TBST, bands were visualized using SuperSignal™ West Femto Maximum Sensitivity Substrate (Thermo Scientific). “Forward” IP western blots were performed using membranes loaded with immunoprecipitate of the index protein, which was incubated overnight at 4°C with the primary antibody of the interactor protein; “reverse” IP western blots were performed using membranes loaded with immunoprecipitate of the interactor protein, which was incubated overnight at 4°C with the primary antibody of the index protein. To evaluate the reproducibility of IP-MS interactions by western blot analysis, we used the following criteria: if a band at the expected molecular weight was detected in the IP lane, the interaction was considered to be “replicated” only if the signal exceeded the negative control (IgG lane). If the band of the expected molecular weight was either absent in the IP lane, or yielding a weaker signal than the control lane (IgG) the interaction was considered “not replicated”. A graphical representation of these criteria is shown in Table S5. All antibodies used in this study (including those that were not IP-competent) are listed in Tables S1, S2, and S5.Mouse cortex preparationMouse cortices were isolated from p0 pups of C57BL/6 background, cut into small pieces, and flash frozen. Lysis was performed by adding Pierce IP lysis buffer (Pierce) to frozen pieces and mechanically dissociating them by vigorous pipetting. All subsequent steps were the same as those for iNs.Human brain preparationFor human postmortem brain samples, 0.6 g of tissue was homogenized in 34 mL extraction buffer (0.5% n-Dodecyl β-D-maltoside, 150 mM NaCl, 50 mM Hepes, pH 7.4) with a glass homogenizer (PotterS from B. Braun) set at 900 rpm for 12 strokes. The homogenate was centrifuged twice at 16,000g for 10 min. All subsequent steps were the same as those for iNs.ImmunoprecipitationsFor each individual experiment, 1-2 mg of protein extract from the same cell differentiation batch was incubated at 4°C overnight in the presence of 1-2 μg of the relevant antibody. On the next day, 50 μL of Protein G beads (Pierce) were added to each sample and incubated at 4°C for 4 hours. Flow-through was collected and beads were washed once with 1 mL lysis buffer (Pierce) supplemented with Halt protease and phosphatase inhibitors (Thermo Scientific), and twice with PBS. Beads were resuspended in 60 μL of PBS and 10% of the volume was employed for immunoblotting, after being boiled in 6xSMASH buffer (50 mM Tris HCl pH 6.8, 10% Glycerol, 2% SDS, 0.02% bromophenol blue, 1% b-mercaptoethanol) for 10 min at 95°C. The remaining volume was stored at −80°C and subsequently used for mass spectrometry analysis.Mass spectrometryTable S2 provides an overview of where liquid chromatography-tandem mass spectrometry (LC-MS/MS) for each IP-MS dataset was performed; MS protocols for each facility/site are described below.Broad Proteomics Platform (Broad): Proteins in IP samples were digested on beads using 90 μL of digestion buffer (2 M urea/50 mM Tris buffer with 1 mM DTT and 5 μg/mL Trypsin) for 1 h, shaking at 1000 rpm. The suspension was then transferred to a new tube, and the beads were washed twice with 60 μL of wash buffer (2 M urea/50 mM Tris buffer). The wash buffer was added to the suspension with digestion. The digestion and wash process was repeated a second time pooling the suspensions with the suspensions from the first round. The pooled solution was reduced using 4 mM DTT for 30 min at 25°C shaking at 1000 rpm. The proteins were then alkylated using 10 mM iodoacetamide and incubating for 45 min at 25°C shaking at 1000 rpm and protected from light. Proteins were then digested with 0.5 μg of trypsin overnight at 25°C shaking at 700 rpm. The next day proteins were quenched using 40 μL of 10% formic acid and desalted using an Oasis Cartridge. Samples were vacuum dried and labeled with iTRAQ4 kits (Sciex). Each iTRAQ 4-plex consisted of 2 replicate experimental IPs using an antibody against an index protein (labeled with iTRAQ reagent labels 114 and 115) and 2 replicate control IPs using a control IgG antibody (labeled with 116 and 117). Reconstituted peptides were separated on an online nanoflow EASY-nLC 1000 UHPLC system (Thermo Scientific) and analyzed on a benchtop Orbitrap Q Exactive Plus (DYRK1A, PTEN) or Q Exactive HF (SCN2A, SHANK3) mass spectrometer (Thermo Scientific). The peptide samples were injected onto a capillary column (Picofrit with 10 μm tip opening/75 μm diameter, New Objective, PF360-75-10-N-5) packed in-house with 20 cm C18 silica material (1.9 μm ReproSil-Pur C18-AQ medium, Dr. Maisch GmbH, r119.aq). The UHPLC setup was connected with a custom-fit micro-adapting tee (360 μm, IDEX Health & Science, UH-753), and capillary columns were heated to 50°C in column heater sleeves (Phoenix-ST) to reduce back-pressure during UHPLC separation. Injected peptides were separated at a flow rate of 200 nL/min with a linear 196 min (SCN2A, SHANK3, PTEN) or 234 min (DYRK1A) gradient from 94% solvent A (3% acetonitrile, 0.1% formic acid) to 30% (DYRK1A) or 40% (SCN2A, SHANK3, PTEN) solvent B (90% acetonitrile, 0.1% formic acid), followed by a linear 8 or 9 min gradient to 60% (DYRK1A) or 65% (SCN2A, SHANK3, PTEN) solvent B and a 1 or 3 min ramp to 90% B. The Q Exactive instrument was operated in the data-dependent mode acquiring HCD MS/MS scans (R = 17.5K –QE + or R = 45K-QE-HF) after each MS1 scan (R = 70K-QE + or R = 60K-QE-HF) on the 12(QE+) or 16(QE-HF) most abundant ions using an MS1 ion target of 3 × 106 ions and an MS2 target of 5 × 104 ions. The maximum ion time utilized for the MS/MS scans was 120 ms; the HCD-normalized collision energy was set to 27; the dynamic exclusion time was set to 20s, and the peptide match preferred and isotope exclusion functions were enabled.Whitehead Proteomics Core Facility (Whitehead): IP samples on beads were resuspended in 100 μL of 100 mM triethylammonium bicarbonate (TEAB), reduced using 2 μL of 50 mM tris(2-carboxyethyl)phosphine for 60 min at 60°C, and alkylated using 1 μL of 2% S-methyl methanethiosulfonate in isopropanol for 10 min at room temperature. Proteins were then digested with 250 ng of trypsin overnight at 37°C with gentle shaking. iTRAQ4 reagents (Sciex) were resuspended in 50 μL isopropanol and added to each sample followed by vortex and spin; the samples in each 4-plex consisted of 2 replicate experimental IPs using an antibody against an index protein (labeled with iTRAQ reagent labels 114 and 115) and 2 replicate control IPs using a control IgG antibody (labeled with 116 and 117). The samples were combined and incubated at room temperature for 2 h, and then washed, extracted, and concentrated by solid phase extraction using Waters Sep-Pak Plus C18 cartridges. Organic solvent was removed and the sample volumes were reduced to 80 μL via speed vacuum. The labeled peptides were subjected to basic (high pH) reversed phase high-performance liquid chromatography (HPLC) with fraction collection using Shimadzu LC-20AD pumps and an FRC-10A fraction collector. Samples were loaded on a 10 cm × 2.1 mm column packed with 2.6 micron Aeris PEPTIDE XB-C18 media (Phenomenex). The initial gradient condition was isocratic 1% buffer A (20 mM ammonium formate in water, pH = 10) at 150 μL min-1, with increasing buffer B (acetonitrile) concentrations to 16.7% B at 20.5 min, 30% B at 31 min, and 45% B at 36 min. The column was washed with high percent B and re-equilibrated between analytical runs for a total cycle time of ∼55 min. Sixteen 450 μL fractions (fx) were collected, combined into eight samples (fx1+2, fx3+9, fx4+10, fx5+11, fx6+12, fx7+13, fx8+14, fx15 + 16), then reduced to 20 μL via speed vacuum. The combined samples were subjected to reversed phase HPLC using Thermo EASY-nLC 1200 pumps and autosampler, followed by mass spectrometry using a Thermo Q Exactive HF-X Hybrid Quadrupole-Orbitrap mass spectrometer and a nanoflow configuration. Samples were loaded on a 6 cm x 100 micron column packed with 10 micron ODS-A C18 material (YMC), washed with 4 μL total volume to trap and wash peptides, then eluted onto the analytical column packed with 1.7 micron Aeris C18 material (Phenomenex) in a fritted 14 cm x 75 micron fused silica tubing pulled to a 5 micron tip. The initial gradient condition was 1% buffer A (1% formic acid in water) at 300 nL min-1, with increasing buffer B (1% formic acid in acetonitrile) concentrations to 6% B at 1 min, 21% B at 42.5 min, 36% B at 63.15 min, and 50% B at 73 min. The column was washed with high percent B and re-equilibrated between analytical runs for a total cycle time of ∼97 min. The mass spectrometer was operated in a data-dependent acquisition mode where the 20 most abundant peptides detected in the Orbitrap using full scan mode with a resolution of 60,000 were subjected to daughter ion fragmentation using a resolution of 15,000. A running list of parent ions was tabulated to an exclusion list to increase the number of peptides analyzed throughout the chromatographic run.Center for Neurogenomics and Cognitive Research (CNCR): IP samples were treated and digested by two protocols. For in-gel digestion, proteins were briefly separated by SDS-PAGE electrophoresis. Each sample was split in three fractions covering protein masses of >70kDa, 70-40 kDa and <40 kDa, and proteins were digested by trypsin overnight at 37°C. Filter-aided sample preparation (FASP) protocol was used for human brain postmortem samples. Sample was solubilized in 2% SDS, 100 mM Tris (pH = 8.8) and transferred to the Microcon-30 filter tube (Millipore). After serial washes with 8M urea and 50 mM NH4HCO3, proteins were digested by trypsin overnight at 37°C and collected as a single fraction. The resulting peptides were analyzed by the TripleTOF 5600 + mass spectrometer coupled to an Ultimate 3000 LC system (Dionex, Thermo Scientific). The mass spectrometer was run in micro-mode. Peptides were fractionated on a 200 mm Alltima C18 column (300 μm i.d., 3 μm particle size) at a flow rate of 5 μL min-1. For in-gel digestion protocol, the tryptic peptides from each gel slice were analyzed with a 1 h LC gradient. For FASP protocol, a 2 h LC gradient was used. The eluted peptides were electro-sprayed with a micro-spray needle voltage of 5500 V. The MS survey scan range was 350–1250 m/z acquired for 250 ms. The top 20 precursor ions were selected for 90 ms per MS/MS acquisition, with a threshold of 90 counts. Dynamic exclusion was 10 s. Rolling CID function was activated, with an energy spread of 5 eV.Harvard Center for Mass Spectrometry (Harvard): IP samples were stored in PBS buffer on beads. PBS was removed and samples were dissolved in 50 μL of 50 mM TEAB, followed by trypsin (Promega) digestion for 3 h at 38°C. Digested samples were dried to 20 μL and 10 μL of each sample was injected in the mass spectrometer. LC-MS/MS was performed on a Lumos Tribrid Orbitrap Mass Spectrometer (Thermo Scientific) equipped with Ultimate 3000 (Thermo Scientific) nano-high-performance liquid chromatography. Peptides were separated onto a 150-μm inner diameter microcapillary trapping column, packed with ∼2 cm of C18 Reprosil resin (5 μm, 100 Å, Dr. Maisch GmbH, Germany), followed by separation on a 50-cm analytical column (PharmaFluidics, Ghent, Belgium). Separation was achieved by applying a gradient from 5 to 27% acetonitrile in 0.1% formic acid for >90 min at 200 nL min-1. Electrospray ionization was enabled by applying a voltage of 2 kV using a home-made electrode junction at the end of the microcapillary column and sprayed from metal tips (PepSep, Denmark). MS survey scan was performed in the Orbitrap, in a range of 400–1800 m/z at a resolution of 60,000, followed by the selection of the 20 most intense ions (TOP20) for CID-MS2 fragmentation in the ion trap using a precursor isolation width window of 2 m/z, automatic gain control setting of 10,000, and a maximum ion accumulation of 100 ms. Singly charged ion species were not subjected to collision-induced dissociation fragmentation. Normalized collision energy was set to 35 V and an activation time of 10 ms. Ions within a 10 ppm m/z window around ions selected for MS2 were excluded from further selection for fragmentation for 60 s.RNA extraction and quantificationCell culture medium was rapidly removed from cells cultured on 150mm Petri dishes and 5 mL of Trizol reagent (Life Technologies) was added. After total RNA extraction, samples were treated with DNase using the Ambion DNA-free DNase Treatment kit (Life Technologies) according to the manufacturer’s instructions, and resuspended in water. Samples were subsequently centrifuged at 13,000 rpm for 20 min at 4°C. Pellets were washed in 75% ethanol and resuspended in water. 1 μL RNA was diluted 1:100 in water and quality-checked using the Agilent RNA 6000 Pico kit (Agilent Technologies) according to the manufacturer’s instructions, and ran on a 2100 Bioanalyzer Instrument (Agilent Technologies).RNA-seq library preparation and sequencingLibraries for RNA-seq were constructed using TruSeq Stranded Total RNA Library Prep Kit (Illumina), with the incorporation of dUTP in the second strand synthesis, and sequenced at the UMass Medical School. Sequencing was performed in paired-end reads (2 × 100 bp) using a HiSeq 4000 system (Illumina).Generation of CRISPR/Cas9 edited cell linesAll transgenic iPSC cells in this study are stable expressing lines. To generate stable iPSC cell lines, cells were electroporated using the NEON transfection system and reagents (Life Technologies), according to the manufacturer’s instructions. RNP complexes were generated using EnGen Spy Cas9 NLS (NEB, M0646T), in combination with a single sgRNA for AKAP8L (Hs.Cas9.AKAP8L.1.AA, IDT), and a mix of 2 sgRNAs for ANK2 (CD.Cas9.VYBY9746.AA and CD.Cas9.YTSN3051.AA, IDT). After 24 h, cells were transferred to 90 mm gelatinized Petri dishes. Cells were grown for 10-12 days, with medium being changed every day. Individual iPSC colonies were picked and expanded for further screening.siRNA knock-downSilencer Select siRNA against DYRK1A were purchased from Thermo Scientific (4,390,824). Transfection into cells was performed on coverslips with RNAiMAX Transfection Reagent (Invitrogen 13,778,100), according to the manufacturer’s instructions. KD were assessed 48 h after transfection.Genomic DNA extractionCells from a confluent 150 mm gelatinized Petri dish were harvested and resuspended in 1-3 mL of lysis buffer (10 mM NaCl, 10 mM Tris-HCl pH 7.5, 10 mM EDTA-NaOH pH 8.0, 0.5% Sodium lauroyl sarcosinate) with proteinase K (Thermo Scientific EO0491) added to a final concentration of 200 μg mL-1. Samples were incubated overnight at 55°C. ½5 volume of 5 M NaCl and 2.5 volume of ice-cold 100% ethanol were added. After mixing, a visible white cloud of DNA was extracted using a bent pipette tip and transferred to a clean tube containing 1 mL 70% ethanol. DNA was pelleted (16,100 g, 5 min, 4°C) and air-dried. Subsequently the pellet was resuspended in 300-400 μL 10 mM Tris pH 8.5 and the concentration measured by Nanodrop.

Method details

Differentiation of iNsiPSCs stably expressing TetO-Ngn2-Neo and reverse tetracycline-controlled transactivator (rtTA) were plated at a density of 40,000 cells cm−2 with rock inhibitor Y27632 (Stemgent, 04-0012). Day 1 cells were differentiated in N2 media (Life Technologies) supplemented with 10 μM SB431542 (Tocris, 1614), 2 μM XAV939 (Stemgent, 04-00046) and 100 nM LDN-193189 (Stemgent, 04-0074) along with doxycycline hyclate (2 μg mL−1). Day 2 media was N2+SB/XAV/LDN/doxycycline hyclate and differentiation media were as previously described. On day 3, cell differentiation was continued in neurobasal media (Life Technologies) supplemented with B27 (50X, Thermo Scientific), brain-derived neurotrophic factor (BDNF), ciliary neurotrophic factor (CTNF), glial cell-derived neurotrophic factor (GDNF) (R&D Systems 248-BD/CF, 257-NT/CF, and 212-GD/CF at 10 ng mL−1) and doxycyclinehyclate (2 μg mL−1).

Differentiation of iNs

iPSCs stably expressing TetO-Ngn2-Neo and reverse tetracycline-controlled transactivator (rtTA) were plated at a density of 40,000 cells cm−2 with rock inhibitor Y27632 (Stemgent, 04-0012). Day 1 cells were differentiated in N2 media (Life Technologies) supplemented with 10 μM SB431542 (Tocris, 1614), 2 μM XAV939 (Stemgent, 04-00046) and 100 nM LDN-193189 (Stemgent, 04-0074) along with doxycycline hyclate (2 μg mL−1). Day 2 media was N2+SB/XAV/LDN/doxycycline hyclate and differentiation media were as previously described. On day 3, cell differentiation was continued in neurobasal media (Life Technologies) supplemented with B27 (50X, Thermo Scientific), brain-derived neurotrophic factor (BDNF), ciliary neurotrophic factor (CTNF), glial cell-derived neurotrophic factor (GDNF) (R&D Systems 248-BD/CF, 257-NT/CF, and 212-GD/CF at 10 ng mL−1) and doxycyclinehyclate (2 μg mL−1).

Protein extraction and immunoblottingTotal protein extract was obtained by harvesting cells and either processing them immediately or snap-freezing them on dry ice for storage at −80°C. In both cases, cell pellets were washed with PBS and resuspended in 10× packed cell volume (PCV) IP lysis buffer (Thermo Scientific), with freshly added Halt protease and phosphatase inhibitors (Thermo Scientific). After a 20 min incubation time at 4°C, cells were collected by centrifugation (16,200 g, 20 min, 4°C) and resuspended in 3× PCV lysis buffer. The concentration of the samples was quantified using the Thermo BCA protein assay and when not used immediately, samples were stored at −80°C. Samples for immunoblotting were diluted in 6× SMASH buffer (50 mM Tris HCl pH 6.8, 10% glycerol, 2% SDS, 0.02% bromophenol blue, 1% b-mercaptoethanol), boiled for 10 min at 95°C, separated on a NuPAGE 4-12% Bis-Tris Protein Gel (Invitrogen), and transferred onto a PVDF membrane (Life Technologies) by wet transfer (100 V for 2 hours). Membranes were blocked by incubation for one hour at room temperature in 10 mL TBS and 0.1% Tween (TBST) with 5% w/v BioRad Blotting-grade Blocker. Blots were incubated overnight at 4°C with the primary antibody, washed 3 times for 10 min with TBST and incubated for 45 min with secondary antibody conjugated to horseradish peroxidase. After washing 3 times for 10 min with TBST, bands were visualized using SuperSignal™ West Femto Maximum Sensitivity Substrate (Thermo Scientific). “Forward” IP western blots were performed using membranes loaded with immunoprecipitate of the index protein, which was incubated overnight at 4°C with the primary antibody of the interactor protein; “reverse” IP western blots were performed using membranes loaded with immunoprecipitate of the interactor protein, which was incubated overnight at 4°C with the primary antibody of the index protein. To evaluate the reproducibility of IP-MS interactions by western blot analysis, we used the following criteria: if a band at the expected molecular weight was detected in the IP lane, the interaction was considered to be “replicated” only if the signal exceeded the negative control (IgG lane). If the band of the expected molecular weight was either absent in the IP lane, or yielding a weaker signal than the control lane (IgG) the interaction was considered “not replicated”. A graphical representation of these criteria is shown in Table S5. All antibodies used in this study (including those that were not IP-competent) are listed in Tables S1, S2, and S5.

Protein extraction and immunoblotting

Total protein extract was obtained by harvesting cells and either processing them immediately or snap-freezing them on dry ice for storage at −80°C. In both cases, cell pellets were washed with PBS and resuspended in 10× packed cell volume (PCV) IP lysis buffer (Thermo Scientific), with freshly added Halt protease and phosphatase inhibitors (Thermo Scientific). After a 20 min incubation time at 4°C, cells were collected by centrifugation (16,200 g, 20 min, 4°C) and resuspended in 3× PCV lysis buffer. The concentration of the samples was quantified using the Thermo BCA protein assay and when not used immediately, samples were stored at −80°C. Samples for immunoblotting were diluted in 6× SMASH buffer (50 mM Tris HCl pH 6.8, 10% glycerol, 2% SDS, 0.02% bromophenol blue, 1% b-mercaptoethanol), boiled for 10 min at 95°C, separated on a NuPAGE 4-12% Bis-Tris Protein Gel (Invitrogen), and transferred onto a PVDF membrane (Life Technologies) by wet transfer (100 V for 2 hours). Membranes were blocked by incubation for one hour at room temperature in 10 mL TBS and 0.1% Tween (TBST) with 5% w/v BioRad Blotting-grade Blocker. Blots were incubated overnight at 4°C with the primary antibody, washed 3 times for 10 min with TBST and incubated for 45 min with secondary antibody conjugated to horseradish peroxidase. After washing 3 times for 10 min with TBST, bands were visualized using SuperSignal™ West Femto Maximum Sensitivity Substrate (Thermo Scientific). “Forward” IP western blots were performed using membranes loaded with immunoprecipitate of the index protein, which was incubated overnight at 4°C with the primary antibody of the interactor protein; “reverse” IP western blots were performed using membranes loaded with immunoprecipitate of the interactor protein, which was incubated overnight at 4°C with the primary antibody of the index protein. To evaluate the reproducibility of IP-MS interactions by western blot analysis, we used the following criteria: if a band at the expected molecular weight was detected in the IP lane, the interaction was considered to be “replicated” only if the signal exceeded the negative control (IgG lane). If the band of the expected molecular weight was either absent in the IP lane, or yielding a weaker signal than the control lane (IgG) the interaction was considered “not replicated”. A graphical representation of these criteria is shown in Table S5. All antibodies used in this study (including those that were not IP-competent) are listed in Tables S1, S2, and S5.

Mouse cortex preparationMouse cortices were isolated from p0 pups of C57BL/6 background, cut into small pieces, and flash frozen. Lysis was performed by adding Pierce IP lysis buffer (Pierce) to frozen pieces and mechanically dissociating them by vigorous pipetting. All subsequent steps were the same as those for iNs.

Mouse cortex preparation

Mouse cortices were isolated from p0 pups of C57BL/6 background, cut into small pieces, and flash frozen. Lysis was performed by adding Pierce IP lysis buffer (Pierce) to frozen pieces and mechanically dissociating them by vigorous pipetting. All subsequent steps were the same as those for iNs.

Human brain preparationFor human postmortem brain samples, 0.6 g of tissue was homogenized in 34 mL extraction buffer (0.5% n-Dodecyl β-D-maltoside, 150 mM NaCl, 50 mM Hepes, pH 7.4) with a glass homogenizer (PotterS from B. Braun) set at 900 rpm for 12 strokes. The homogenate was centrifuged twice at 16,000g for 10 min. All subsequent steps were the same as those for iNs.

Human brain preparation

For human postmortem brain samples, 0.6 g of tissue was homogenized in 34 mL extraction buffer (0.5% n-Dodecyl β-D-maltoside, 150 mM NaCl, 50 mM Hepes, pH 7.4) with a glass homogenizer (PotterS from B. Braun) set at 900 rpm for 12 strokes. The homogenate was centrifuged twice at 16,000g for 10 min. All subsequent steps were the same as those for iNs.

ImmunoprecipitationsFor each individual experiment, 1-2 mg of protein extract from the same cell differentiation batch was incubated at 4°C overnight in the presence of 1-2 μg of the relevant antibody. On the next day, 50 μL of Protein G beads (Pierce) were added to each sample and incubated at 4°C for 4 hours. Flow-through was collected and beads were washed once with 1 mL lysis buffer (Pierce) supplemented with Halt protease and phosphatase inhibitors (Thermo Scientific), and twice with PBS. Beads were resuspended in 60 μL of PBS and 10% of the volume was employed for immunoblotting, after being boiled in 6xSMASH buffer (50 mM Tris HCl pH 6.8, 10% Glycerol, 2% SDS, 0.02% bromophenol blue, 1% b-mercaptoethanol) for 10 min at 95°C. The remaining volume was stored at −80°C and subsequently used for mass spectrometry analysis.

Immunoprecipitations

For each individual experiment, 1-2 mg of protein extract from the same cell differentiation batch was incubated at 4°C overnight in the presence of 1-2 μg of the relevant antibody. On the next day, 50 μL of Protein G beads (Pierce) were added to each sample and incubated at 4°C for 4 hours. Flow-through was collected and beads were washed once with 1 mL lysis buffer (Pierce) supplemented with Halt protease and phosphatase inhibitors (Thermo Scientific), and twice with PBS. Beads were resuspended in 60 μL of PBS and 10% of the volume was employed for immunoblotting, after being boiled in 6xSMASH buffer (50 mM Tris HCl pH 6.8, 10% Glycerol, 2% SDS, 0.02% bromophenol blue, 1% b-mercaptoethanol) for 10 min at 95°C. The remaining volume was stored at −80°C and subsequently used for mass spectrometry analysis.

Mass spectrometryTable S2 provides an overview of where liquid chromatography-tandem mass spectrometry (LC-MS/MS) for each IP-MS dataset was performed; MS protocols for each facility/site are described below.Broad Proteomics Platform (Broad): Proteins in IP samples were digested on beads using 90 μL of digestion buffer (2 M urea/50 mM Tris buffer with 1 mM DTT and 5 μg/mL Trypsin) for 1 h, shaking at 1000 rpm. The suspension was then transferred to a new tube, and the beads were washed twice with 60 μL of wash buffer (2 M urea/50 mM Tris buffer). The wash buffer was added to the suspension with digestion. The digestion and wash process was repeated a second time pooling the suspensions with the suspensions from the first round. The pooled solution was reduced using 4 mM DTT for 30 min at 25°C shaking at 1000 rpm. The proteins were then alkylated using 10 mM iodoacetamide and incubating for 45 min at 25°C shaking at 1000 rpm and protected from light. Proteins were then digested with 0.5 μg of trypsin overnight at 25°C shaking at 700 rpm. The next day proteins were quenched using 40 μL of 10% formic acid and desalted using an Oasis Cartridge. Samples were vacuum dried and labeled with iTRAQ4 kits (Sciex). Each iTRAQ 4-plex consisted of 2 replicate experimental IPs using an antibody against an index protein (labeled with iTRAQ reagent labels 114 and 115) and 2 replicate control IPs using a control IgG antibody (labeled with 116 and 117). Reconstituted peptides were separated on an online nanoflow EASY-nLC 1000 UHPLC system (Thermo Scientific) and analyzed on a benchtop Orbitrap Q Exactive Plus (DYRK1A, PTEN) or Q Exactive HF (SCN2A, SHANK3) mass spectrometer (Thermo Scientific). The peptide samples were injected onto a capillary column (Picofrit with 10 μm tip opening/75 μm diameter, New Objective, PF360-75-10-N-5) packed in-house with 20 cm C18 silica material (1.9 μm ReproSil-Pur C18-AQ medium, Dr. Maisch GmbH, r119.aq). The UHPLC setup was connected with a custom-fit micro-adapting tee (360 μm, IDEX Health & Science, UH-753), and capillary columns were heated to 50°C in column heater sleeves (Phoenix-ST) to reduce back-pressure during UHPLC separation. Injected peptides were separated at a flow rate of 200 nL/min with a linear 196 min (SCN2A, SHANK3, PTEN) or 234 min (DYRK1A) gradient from 94% solvent A (3% acetonitrile, 0.1% formic acid) to 30% (DYRK1A) or 40% (SCN2A, SHANK3, PTEN) solvent B (90% acetonitrile, 0.1% formic acid), followed by a linear 8 or 9 min gradient to 60% (DYRK1A) or 65% (SCN2A, SHANK3, PTEN) solvent B and a 1 or 3 min ramp to 90% B. The Q Exactive instrument was operated in the data-dependent mode acquiring HCD MS/MS scans (R = 17.5K –QE + or R = 45K-QE-HF) after each MS1 scan (R = 70K-QE + or R = 60K-QE-HF) on the 12(QE+) or 16(QE-HF) most abundant ions using an MS1 ion target of 3 × 106 ions and an MS2 target of 5 × 104 ions. The maximum ion time utilized for the MS/MS scans was 120 ms; the HCD-normalized collision energy was set to 27; the dynamic exclusion time was set to 20s, and the peptide match preferred and isotope exclusion functions were enabled.Whitehead Proteomics Core Facility (Whitehead): IP samples on beads were resuspended in 100 μL of 100 mM triethylammonium bicarbonate (TEAB), reduced using 2 μL of 50 mM tris(2-carboxyethyl)phosphine for 60 min at 60°C, and alkylated using 1 μL of 2% S-methyl methanethiosulfonate in isopropanol for 10 min at room temperature. Proteins were then digested with 250 ng of trypsin overnight at 37°C with gentle shaking. iTRAQ4 reagents (Sciex) were resuspended in 50 μL isopropanol and added to each sample followed by vortex and spin; the samples in each 4-plex consisted of 2 replicate experimental IPs using an antibody against an index protein (labeled with iTRAQ reagent labels 114 and 115) and 2 replicate control IPs using a control IgG antibody (labeled with 116 and 117). The samples were combined and incubated at room temperature for 2 h, and then washed, extracted, and concentrated by solid phase extraction using Waters Sep-Pak Plus C18 cartridges. Organic solvent was removed and the sample volumes were reduced to 80 μL via speed vacuum. The labeled peptides were subjected to basic (high pH) reversed phase high-performance liquid chromatography (HPLC) with fraction collection using Shimadzu LC-20AD pumps and an FRC-10A fraction collector. Samples were loaded on a 10 cm × 2.1 mm column packed with 2.6 micron Aeris PEPTIDE XB-C18 media (Phenomenex). The initial gradient condition was isocratic 1% buffer A (20 mM ammonium formate in water, pH = 10) at 150 μL min-1, with increasing buffer B (acetonitrile) concentrations to 16.7% B at 20.5 min, 30% B at 31 min, and 45% B at 36 min. The column was washed with high percent B and re-equilibrated between analytical runs for a total cycle time of ∼55 min. Sixteen 450 μL fractions (fx) were collected, combined into eight samples (fx1+2, fx3+9, fx4+10, fx5+11, fx6+12, fx7+13, fx8+14, fx15 + 16), then reduced to 20 μL via speed vacuum. The combined samples were subjected to reversed phase HPLC using Thermo EASY-nLC 1200 pumps and autosampler, followed by mass spectrometry using a Thermo Q Exactive HF-X Hybrid Quadrupole-Orbitrap mass spectrometer and a nanoflow configuration. Samples were loaded on a 6 cm x 100 micron column packed with 10 micron ODS-A C18 material (YMC), washed with 4 μL total volume to trap and wash peptides, then eluted onto the analytical column packed with 1.7 micron Aeris C18 material (Phenomenex) in a fritted 14 cm x 75 micron fused silica tubing pulled to a 5 micron tip. The initial gradient condition was 1% buffer A (1% formic acid in water) at 300 nL min-1, with increasing buffer B (1% formic acid in acetonitrile) concentrations to 6% B at 1 min, 21% B at 42.5 min, 36% B at 63.15 min, and 50% B at 73 min. The column was washed with high percent B and re-equilibrated between analytical runs for a total cycle time of ∼97 min. The mass spectrometer was operated in a data-dependent acquisition mode where the 20 most abundant peptides detected in the Orbitrap using full scan mode with a resolution of 60,000 were subjected to daughter ion fragmentation using a resolution of 15,000. A running list of parent ions was tabulated to an exclusion list to increase the number of peptides analyzed throughout the chromatographic run.Center for Neurogenomics and Cognitive Research (CNCR): IP samples were treated and digested by two protocols. For in-gel digestion, proteins were briefly separated by SDS-PAGE electrophoresis. Each sample was split in three fractions covering protein masses of >70kDa, 70-40 kDa and <40 kDa, and proteins were digested by trypsin overnight at 37°C. Filter-aided sample preparation (FASP) protocol was used for human brain postmortem samples. Sample was solubilized in 2% SDS, 100 mM Tris (pH = 8.8) and transferred to the Microcon-30 filter tube (Millipore). After serial washes with 8M urea and 50 mM NH4HCO3, proteins were digested by trypsin overnight at 37°C and collected as a single fraction. The resulting peptides were analyzed by the TripleTOF 5600 + mass spectrometer coupled to an Ultimate 3000 LC system (Dionex, Thermo Scientific). The mass spectrometer was run in micro-mode. Peptides were fractionated on a 200 mm Alltima C18 column (300 μm i.d., 3 μm particle size) at a flow rate of 5 μL min-1. For in-gel digestion protocol, the tryptic peptides from each gel slice were analyzed with a 1 h LC gradient. For FASP protocol, a 2 h LC gradient was used. The eluted peptides were electro-sprayed with a micro-spray needle voltage of 5500 V. The MS survey scan range was 350–1250 m/z acquired for 250 ms. The top 20 precursor ions were selected for 90 ms per MS/MS acquisition, with a threshold of 90 counts. Dynamic exclusion was 10 s. Rolling CID function was activated, with an energy spread of 5 eV.Harvard Center for Mass Spectrometry (Harvard): IP samples were stored in PBS buffer on beads. PBS was removed and samples were dissolved in 50 μL of 50 mM TEAB, followed by trypsin (Promega) digestion for 3 h at 38°C. Digested samples were dried to 20 μL and 10 μL of each sample was injected in the mass spectrometer. LC-MS/MS was performed on a Lumos Tribrid Orbitrap Mass Spectrometer (Thermo Scientific) equipped with Ultimate 3000 (Thermo Scientific) nano-high-performance liquid chromatography. Peptides were separated onto a 150-μm inner diameter microcapillary trapping column, packed with ∼2 cm of C18 Reprosil resin (5 μm, 100 Å, Dr. Maisch GmbH, Germany), followed by separation on a 50-cm analytical column (PharmaFluidics, Ghent, Belgium). Separation was achieved by applying a gradient from 5 to 27% acetonitrile in 0.1% formic acid for >90 min at 200 nL min-1. Electrospray ionization was enabled by applying a voltage of 2 kV using a home-made electrode junction at the end of the microcapillary column and sprayed from metal tips (PepSep, Denmark). MS survey scan was performed in the Orbitrap, in a range of 400–1800 m/z at a resolution of 60,000, followed by the selection of the 20 most intense ions (TOP20) for CID-MS2 fragmentation in the ion trap using a precursor isolation width window of 2 m/z, automatic gain control setting of 10,000, and a maximum ion accumulation of 100 ms. Singly charged ion species were not subjected to collision-induced dissociation fragmentation. Normalized collision energy was set to 35 V and an activation time of 10 ms. Ions within a 10 ppm m/z window around ions selected for MS2 were excluded from further selection for fragmentation for 60 s.

Mass spectrometry

Table S2 provides an overview of where liquid chromatography-tandem mass spectrometry (LC-MS/MS) for each IP-MS dataset was performed; MS protocols for each facility/site are described below.

Broad Proteomics Platform (Broad): Proteins in IP samples were digested on beads using 90 μL of digestion buffer (2 M urea/50 mM Tris buffer with 1 mM DTT and 5 μg/mL Trypsin) for 1 h, shaking at 1000 rpm. The suspension was then transferred to a new tube, and the beads were washed twice with 60 μL of wash buffer (2 M urea/50 mM Tris buffer). The wash buffer was added to the suspension with digestion. The digestion and wash process was repeated a second time pooling the suspensions with the suspensions from the first round. The pooled solution was reduced using 4 mM DTT for 30 min at 25°C shaking at 1000 rpm. The proteins were then alkylated using 10 mM iodoacetamide and incubating for 45 min at 25°C shaking at 1000 rpm and protected from light. Proteins were then digested with 0.5 μg of trypsin overnight at 25°C shaking at 700 rpm. The next day proteins were quenched using 40 μL of 10% formic acid and desalted using an Oasis Cartridge. Samples were vacuum dried and labeled with iTRAQ4 kits (Sciex). Each iTRAQ 4-plex consisted of 2 replicate experimental IPs using an antibody against an index protein (labeled with iTRAQ reagent labels 114 and 115) and 2 replicate control IPs using a control IgG antibody (labeled with 116 and 117). Reconstituted peptides were separated on an online nanoflow EASY-nLC 1000 UHPLC system (Thermo Scientific) and analyzed on a benchtop Orbitrap Q Exactive Plus (DYRK1A, PTEN) or Q Exactive HF (SCN2A, SHANK3) mass spectrometer (Thermo Scientific). The peptide samples were injected onto a capillary column (Picofrit with 10 μm tip opening/75 μm diameter, New Objective, PF360-75-10-N-5) packed in-house with 20 cm C18 silica material (1.9 μm ReproSil-Pur C18-AQ medium, Dr. Maisch GmbH, r119.aq). The UHPLC setup was connected with a custom-fit micro-adapting tee (360 μm, IDEX Health & Science, UH-753), and capillary columns were heated to 50°C in column heater sleeves (Phoenix-ST) to reduce back-pressure during UHPLC separation. Injected peptides were separated at a flow rate of 200 nL/min with a linear 196 min (SCN2A, SHANK3, PTEN) or 234 min (DYRK1A) gradient from 94% solvent A (3% acetonitrile, 0.1% formic acid) to 30% (DYRK1A) or 40% (SCN2A, SHANK3, PTEN) solvent B (90% acetonitrile, 0.1% formic acid), followed by a linear 8 or 9 min gradient to 60% (DYRK1A) or 65% (SCN2A, SHANK3, PTEN) solvent B and a 1 or 3 min ramp to 90% B. The Q Exactive instrument was operated in the data-dependent mode acquiring HCD MS/MS scans (R = 17.5K –QE + or R = 45K-QE-HF) after each MS1 scan (R = 70K-QE + or R = 60K-QE-HF) on the 12(QE+) or 16(QE-HF) most abundant ions using an MS1 ion target of 3 × 106 ions and an MS2 target of 5 × 104 ions. The maximum ion time utilized for the MS/MS scans was 120 ms; the HCD-normalized collision energy was set to 27; the dynamic exclusion time was set to 20s, and the peptide match preferred and isotope exclusion functions were enabled.

Whitehead Proteomics Core Facility (Whitehead): IP samples on beads were resuspended in 100 μL of 100 mM triethylammonium bicarbonate (TEAB), reduced using 2 μL of 50 mM tris(2-carboxyethyl)phosphine for 60 min at 60°C, and alkylated using 1 μL of 2% S-methyl methanethiosulfonate in isopropanol for 10 min at room temperature. Proteins were then digested with 250 ng of trypsin overnight at 37°C with gentle shaking. iTRAQ4 reagents (Sciex) were resuspended in 50 μL isopropanol and added to each sample followed by vortex and spin; the samples in each 4-plex consisted of 2 replicate experimental IPs using an antibody against an index protein (labeled with iTRAQ reagent labels 114 and 115) and 2 replicate control IPs using a control IgG antibody (labeled with 116 and 117). The samples were combined and incubated at room temperature for 2 h, and then washed, extracted, and concentrated by solid phase extraction using Waters Sep-Pak Plus C18 cartridges. Organic solvent was removed and the sample volumes were reduced to 80 μL via speed vacuum. The labeled peptides were subjected to basic (high pH) reversed phase high-performance liquid chromatography (HPLC) with fraction collection using Shimadzu LC-20AD pumps and an FRC-10A fraction collector. Samples were loaded on a 10 cm × 2.1 mm column packed with 2.6 micron Aeris PEPTIDE XB-C18 media (Phenomenex). The initial gradient condition was isocratic 1% buffer A (20 mM ammonium formate in water, pH = 10) at 150 μL min-1, with increasing buffer B (acetonitrile) concentrations to 16.7% B at 20.5 min, 30% B at 31 min, and 45% B at 36 min. The column was washed with high percent B and re-equilibrated between analytical runs for a total cycle time of ∼55 min. Sixteen 450 μL fractions (fx) were collected, combined into eight samples (fx1+2, fx3+9, fx4+10, fx5+11, fx6+12, fx7+13, fx8+14, fx15 + 16), then reduced to 20 μL via speed vacuum. The combined samples were subjected to reversed phase HPLC using Thermo EASY-nLC 1200 pumps and autosampler, followed by mass spectrometry using a Thermo Q Exactive HF-X Hybrid Quadrupole-Orbitrap mass spectrometer and a nanoflow configuration. Samples were loaded on a 6 cm x 100 micron column packed with 10 micron ODS-A C18 material (YMC), washed with 4 μL total volume to trap and wash peptides, then eluted onto the analytical column packed with 1.7 micron Aeris C18 material (Phenomenex) in a fritted 14 cm x 75 micron fused silica tubing pulled to a 5 micron tip. The initial gradient condition was 1% buffer A (1% formic acid in water) at 300 nL min-1, with increasing buffer B (1% formic acid in acetonitrile) concentrations to 6% B at 1 min, 21% B at 42.5 min, 36% B at 63.15 min, and 50% B at 73 min. The column was washed with high percent B and re-equilibrated between analytical runs for a total cycle time of ∼97 min. The mass spectrometer was operated in a data-dependent acquisition mode where the 20 most abundant peptides detected in the Orbitrap using full scan mode with a resolution of 60,000 were subjected to daughter ion fragmentation using a resolution of 15,000. A running list of parent ions was tabulated to an exclusion list to increase the number of peptides analyzed throughout the chromatographic run.

Center for Neurogenomics and Cognitive Research (CNCR): IP samples were treated and digested by two protocols. For in-gel digestion, proteins were briefly separated by SDS-PAGE electrophoresis. Each sample was split in three fractions covering protein masses of >70kDa, 70-40 kDa and <40 kDa, and proteins were digested by trypsin overnight at 37°C. Filter-aided sample preparation (FASP) protocol was used for human brain postmortem samples. Sample was solubilized in 2% SDS, 100 mM Tris (pH = 8.8) and transferred to the Microcon-30 filter tube (Millipore). After serial washes with 8M urea and 50 mM NH4HCO3, proteins were digested by trypsin overnight at 37°C and collected as a single fraction. The resulting peptides were analyzed by the TripleTOF 5600 + mass spectrometer coupled to an Ultimate 3000 LC system (Dionex, Thermo Scientific). The mass spectrometer was run in micro-mode. Peptides were fractionated on a 200 mm Alltima C18 column (300 μm i.d., 3 μm particle size) at a flow rate of 5 μL min-1. For in-gel digestion protocol, the tryptic peptides from each gel slice were analyzed with a 1 h LC gradient. For FASP protocol, a 2 h LC gradient was used. The eluted peptides were electro-sprayed with a micro-spray needle voltage of 5500 V. The MS survey scan range was 350–1250 m/z acquired for 250 ms. The top 20 precursor ions were selected for 90 ms per MS/MS acquisition, with a threshold of 90 counts. Dynamic exclusion was 10 s. Rolling CID function was activated, with an energy spread of 5 eV.

Harvard Center for Mass Spectrometry (Harvard): IP samples were stored in PBS buffer on beads. PBS was removed and samples were dissolved in 50 μL of 50 mM TEAB, followed by trypsin (Promega) digestion for 3 h at 38°C. Digested samples were dried to 20 μL and 10 μL of each sample was injected in the mass spectrometer. LC-MS/MS was performed on a Lumos Tribrid Orbitrap Mass Spectrometer (Thermo Scientific) equipped with Ultimate 3000 (Thermo Scientific) nano-high-performance liquid chromatography. Peptides were separated onto a 150-μm inner diameter microcapillary trapping column, packed with ∼2 cm of C18 Reprosil resin (5 μm, 100 Å, Dr. Maisch GmbH, Germany), followed by separation on a 50-cm analytical column (PharmaFluidics, Ghent, Belgium). Separation was achieved by applying a gradient from 5 to 27% acetonitrile in 0.1% formic acid for >90 min at 200 nL min-1. Electrospray ionization was enabled by applying a voltage of 2 kV using a home-made electrode junction at the end of the microcapillary column and sprayed from metal tips (PepSep, Denmark). MS survey scan was performed in the Orbitrap, in a range of 400–1800 m/z at a resolution of 60,000, followed by the selection of the 20 most intense ions (TOP20) for CID-MS2 fragmentation in the ion trap using a precursor isolation width window of 2 m/z, automatic gain control setting of 10,000, and a maximum ion accumulation of 100 ms. Singly charged ion species were not subjected to collision-induced dissociation fragmentation. Normalized collision energy was set to 35 V and an activation time of 10 ms. Ions within a 10 ppm m/z window around ions selected for MS2 were excluded from further selection for fragmentation for 60 s.

RNA extraction and quantificationCell culture medium was rapidly removed from cells cultured on 150mm Petri dishes and 5 mL of Trizol reagent (Life Technologies) was added. After total RNA extraction, samples were treated with DNase using the Ambion DNA-free DNase Treatment kit (Life Technologies) according to the manufacturer’s instructions, and resuspended in water. Samples were subsequently centrifuged at 13,000 rpm for 20 min at 4°C. Pellets were washed in 75% ethanol and resuspended in water. 1 μL RNA was diluted 1:100 in water and quality-checked using the Agilent RNA 6000 Pico kit (Agilent Technologies) according to the manufacturer’s instructions, and ran on a 2100 Bioanalyzer Instrument (Agilent Technologies).

RNA extraction and quantification

Cell culture medium was rapidly removed from cells cultured on 150mm Petri dishes and 5 mL of Trizol reagent (Life Technologies) was added. After total RNA extraction, samples were treated with DNase using the Ambion DNA-free DNase Treatment kit (Life Technologies) according to the manufacturer’s instructions, and resuspended in water. Samples were subsequently centrifuged at 13,000 rpm for 20 min at 4°C. Pellets were washed in 75% ethanol and resuspended in water. 1 μL RNA was diluted 1:100 in water and quality-checked using the Agilent RNA 6000 Pico kit (Agilent Technologies) according to the manufacturer’s instructions, and ran on a 2100 Bioanalyzer Instrument (Agilent Technologies).

RNA-seq library preparation and sequencingLibraries for RNA-seq were constructed using TruSeq Stranded Total RNA Library Prep Kit (Illumina), with the incorporation of dUTP in the second strand synthesis, and sequenced at the UMass Medical School. Sequencing was performed in paired-end reads (2 × 100 bp) using a HiSeq 4000 system (Illumina).

RNA-seq library preparation and sequencing

Libraries for RNA-seq were constructed using TruSeq Stranded Total RNA Library Prep Kit (Illumina), with the incorporation of dUTP in the second strand synthesis, and sequenced at the UMass Medical School. Sequencing was performed in paired-end reads (2 × 100 bp) using a HiSeq 4000 system (Illumina).

Generation of CRISPR/Cas9 edited cell linesAll transgenic iPSC cells in this study are stable expressing lines. To generate stable iPSC cell lines, cells were electroporated using the NEON transfection system and reagents (Life Technologies), according to the manufacturer’s instructions. RNP complexes were generated using EnGen Spy Cas9 NLS (NEB, M0646T), in combination with a single sgRNA for AKAP8L (Hs.Cas9.AKAP8L.1.AA, IDT), and a mix of 2 sgRNAs for ANK2 (CD.Cas9.VYBY9746.AA and CD.Cas9.YTSN3051.AA, IDT). After 24 h, cells were transferred to 90 mm gelatinized Petri dishes. Cells were grown for 10-12 days, with medium being changed every day. Individual iPSC colonies were picked and expanded for further screening.

Generation of CRISPR/Cas9 edited cell lines

All transgenic iPSC cells in this study are stable expressing lines. To generate stable iPSC cell lines, cells were electroporated using the NEON transfection system and reagents (Life Technologies), according to the manufacturer’s instructions. RNP complexes were generated using EnGen Spy Cas9 NLS (NEB, M0646T), in combination with a single sgRNA for AKAP8L (Hs.Cas9.AKAP8L.1.AA, IDT), and a mix of 2 sgRNAs for ANK2 (CD.Cas9.VYBY9746.AA and CD.Cas9.YTSN3051.AA, IDT). After 24 h, cells were transferred to 90 mm gelatinized Petri dishes. Cells were grown for 10-12 days, with medium being changed every day. Individual iPSC colonies were picked and expanded for further screening.

siRNA knock-downSilencer Select siRNA against DYRK1A were purchased from Thermo Scientific (4,390,824). Transfection into cells was performed on coverslips with RNAiMAX Transfection Reagent (Invitrogen 13,778,100), according to the manufacturer’s instructions. KD were assessed 48 h after transfection.

siRNA knock-down

Silencer Select siRNA against DYRK1A were purchased from Thermo Scientific (4,390,824). Transfection into cells was performed on coverslips with RNAiMAX Transfection Reagent (Invitrogen 13,778,100), according to the manufacturer’s instructions. KD were assessed 48 h after transfection.

Genomic DNA extractionCells from a confluent 150 mm gelatinized Petri dish were harvested and resuspended in 1-3 mL of lysis buffer (10 mM NaCl, 10 mM Tris-HCl pH 7.5, 10 mM EDTA-NaOH pH 8.0, 0.5% Sodium lauroyl sarcosinate) with proteinase K (Thermo Scientific EO0491) added to a final concentration of 200 μg mL-1. Samples were incubated overnight at 55°C. ½5 volume of 5 M NaCl and 2.5 volume of ice-cold 100% ethanol were added. After mixing, a visible white cloud of DNA was extracted using a bent pipette tip and transferred to a clean tube containing 1 mL 70% ethanol. DNA was pelleted (16,100 g, 5 min, 4°C) and air-dried. Subsequently the pellet was resuspended in 300-400 μL 10 mM Tris pH 8.5 and the concentration measured by Nanodrop.

Genomic DNA extraction

Cells from a confluent 150 mm gelatinized Petri dish were harvested and resuspended in 1-3 mL of lysis buffer (10 mM NaCl, 10 mM Tris-HCl pH 7.5, 10 mM EDTA-NaOH pH 8.0, 0.5% Sodium lauroyl sarcosinate) with proteinase K (Thermo Scientific EO0491) added to a final concentration of 200 μg mL-1. Samples were incubated overnight at 55°C. ½5 volume of 5 M NaCl and 2.5 volume of ice-cold 100% ethanol were added. After mixing, a visible white cloud of DNA was extracted using a bent pipette tip and transferred to a clean tube containing 1 mL 70% ethanol. DNA was pelleted (16,100 g, 5 min, 4°C) and air-dried. Subsequently the pellet was resuspended in 300-400 μL 10 mM Tris pH 8.5 and the concentration measured by Nanodrop.

Quantification and statistical analysisRNA-seq data analysisStarting with RNA-seq data in FASTQ format, we ran Trim Galore79 (v0.6.7) to trim adapters and low-quality reads using “--stringency 7” and other default settings for Illumina paired-end reads. Sequence alignment was performed using HISAT280 (v2.2.1) with the “--rna-strandness FR” setting; index files for the GRCh38 reference genome with transcripts were downloaded from the HISAT2 website (https://daehwankimlab.github.io/hisat2/download/). The htseq-count script in Htseq81 (v0.13.5) was used to generate gene counts from aligned reads with “--stranded yes --minaqual 1” and other default settings; GRCh38 GTF gene annotation file was obtained from Ensembl86 (release 84). The EdgeR package82 (v3.34.0) was used to remove low-count genes (min.count = 5 in filterByExpr function) before calculating normalized gene counts (method = “TMM” in calcNormFactors function).scRNA-seq t-SNE plotsSingle-cell RNA sequencing (scRNA-seq) data from Velmeshev et al.24 were downloaded from the UCSC Cell Browser (https://cells.ucsc.edu/?ds=autism#). The gene expression matrix (exprMatrix.tsv.gz), cell meta annotations (meta.tsv), dimensionality reduction coordinates (tSNE.coords.tsv.gz), and the Seurat R package83 (v3.2.0) were used to generate t-SNE plots showing either gene expression or cell type annotations across cell clusters.Protein quantification from raw IP-MS dataSoftware and methods used by each MS facility/site to generate protein-level quantification reports from raw IP-MS data are described below; Table S2 links each IP-MS dataset we analyzed to each facility.Broad: Mass spectra were analyzed using Spectrum Mill (v7.0; https://proteomics.broadinstitute.org). For peptide identification, MS/MS spectra were searched against a sequence database for the UniProt human reference proteome, including isoforms, with a set of common laboratory contaminant proteins appended (2017: 65,068 entries). Search parameters included: ESI-QEXACTIVE-HCD scoring parameters, trypsin enzyme specificity with a maximum of two missed cleavages, 40% minimum matched peak intensity, ±20 ppm precursor mass tolerance, ±20 ppm product mass tolerance. Carbamidomethylation of cysteines and iTRAQ4 full labeling of lysines and peptide n-termini were set as fixed modifications. Allowed variable modifications were oxidation of methionine (M), acetyl (ProtN-term), and deamidated (N), with a precursor MH + shift range of −18 to 64 Da. Identities interpreted for individual spectra were automatically designated as valid by optimizing score and delta rank1-rank2 score thresholds separately for each precursor charge state in each LC-MS/MS while allowing a maximum target-decoy-based false discovery rate (FDR) of 1.0% at the spectrum level. Identified peptides were organized into protein groups and subgroups (isoforms and family members) with Spectrum Mill’s subgroup specific option enabled, so that peptides shared between subgroups are ignored when using reporter ion intensities to perform protein-level quantitation.Whitehead: Mass spectra were analyzed using PEAKS Studio (v8.5; Bioinformatics Solutions). Peptide and protein identification was performed by searching against the UniProt human reference proteome, including isoforms (Swiss-Prot/TrEMBL, release 2019_01) together with a set of common contaminants. An FDR threshold of 1% was used for identification of peptides and proteins. Relative ratios of the iTRAQ 4-plex reporter ions were used for protein-level quantitation.CNCR: Mass spectra were analyzed using MaxQuant84 (v1.6.1.0 or v1.6.3.4; https://www.maxquant.org). Peptide and protein identification was performed by searching against the UniProt human database, and label-free quantitation was performed to generate protein-level quantification reports. MaxQuant versions and settings used for each dataset can be found in the deposited parameters file linked to each dataset (Table S12).Harvard: Mass spectra were analyzed using Proteome Discoverer (v2.4; Thermo Scientific). Assignment of MS/MS spectra was performed using the Sequest HT algorithm by searching the data against the UniProt human reference proteome, including isoforms (Swiss-Prot, release 2019_01) as well as other known contaminants such as human keratins and common laboratory contaminants. Sequest HT searches were performed using a 10 ppm precursor ion tolerance and requiring each peptide’s N/C termini to adhere with trypsin protease specificity, while allowing up to two missed cleavages. CID-MS2 spectra were searched with 0.5 Da ion tolerance for fragmentation. Methionine oxidation (+15.99492 Da) was set as variable modification. An MS2 spectra assignment FDR of 1% was applied to both proteins and peptides using the Percolator target-decoy database search. Label-free quantitation was performed to generate protein-level quantification reports.IP-MS data processing and analysisData processing: Starting with protein-level quantification reports generated by the MS analytical software described above, we generally performed the following data processing procedures for each IP-MS dataset: (1) log2 transformation and median normalization of protein intensity values in each bait (i.e., index protein) or IgG control IP sample; (2) removing contaminants, proteins supported by < 1 unique peptide, and proteins detected in < 2 bait samples; (3) imputing missing protein intensity values in each sample by randomly sampling from a normal distribution with mean of μ - 1.8σ and SD of 0.3σ, where μ and σ are the mean and SD of the observed intensity values in the sample87; (4) calculating log2 FC for each replicate pair of bait vs. IgG control samples. However, due to differences in MS protocols, we modified these steps for specific datasets: for data generated at Broad and Whitehead, few missing values were present due to the labeled quantification approach, therefore imputation was not performed and only proteins with no missingness across all bait and control samples were included for analysis; for data generated at CNCR, instead of comparing bait samples against IgG controls to calculate log2 FC, the median of several negative IPs (i.e., bait samples in which the bait protein itself was not detected; Table S12) from the same MS run were used as controls.Genoppi analysis: We analyzed each processed IP-MS dataset using the Genoppi R package18 (v1.0), including: (1) calculating Pearson’s correlation of log2 FC values between replicates; (2) calculating average log2 FC, and corresponding P-value and false discovery rate (FDR), for each protein across replicates using a two-tailed one-sample moderated t-test88; (3) identifying statistically significant (log2 FC > 0 and FDR ≤ 0.1) interactors of the bait; (4) assessing overlap between the identified interactors and known interactors from the InWeb database,17 as curated in the Genoppi R package. We QC’ed each dataset using two criteria: the replicate log2 FC correlation must be > 0.6 and the bait protein itself must be significant (log2 FC > 0 and FDR ≤ 0.1); datasets that failed to meet these criteria were excluded from further analysis. All IP-MS analysis results are provided in Table S2.Defining non-interactors: For each QC’ed IP-MS dataset derived from iNs, besides identifying significant interactors as described above, we also defined a matching set of “non-interactors” (Table S3) to serve as proxy for the background iN proteome in conditional enrichment analyses. We applied different definitions for datasets generated by the labeled vs. label-free quantification approaches: for labeled datasets generated at Broad and Whitehead, we considered all non-significant proteins (i.e., log2 FC ≤ 0 or FDR > 0.1) in Genoppi analysis to be non-interactors; for label-free datasets generated at CNCR, we included both non-significant proteins in Genoppi analysis and other proteins in the MS quantification report that were detected in < 2 bait samples (which were filtered out prior to Genoppi analysis).Generating PPI networks: We consolidated all QC’ed iN-derived IP-MS datasets into a combined PPI network; we also combined the datasets for the same index proteins into index protein-specific networks. The interactors in each network are proteins that show up as significant interactors in ≥ 1 source IP-MS datasets contributing to the network; the matching “non-interactors” are proteins that show up as non-interactors in ≥ 1 source dataset but never as interactors in any source dataset. The full lists of interactors and non-interactors associated with each PPI network are provided in Table S3; we note that in this table, we removed all index proteins from both the interactor and non-interactor lists for the combined PPI network to avoid biasing downstream enrichment analyses.Co-expression analysisPairwise co-expression between each index gene and all other protein-coding genes were estimated using data from four independent studies, including (1) Stickels et al.33: spatial transcriptomics in mouse neocortex (Puck_190921_19.digital_expression.txt.gz retrieved from: https://singlecell.broadinstitute.org/single_cell/study/SCP815/highly-sensitive-spatial-transcriptomics-at-near-cellular-resolution-with-slide-seqv2#study-download); (2) Maynard et al.31: spatial transcriptomics in human dorsolateral prefrontal cortex (count matrix retrieved from spatialLIBD R package: https://github.com/LieberInstitute/HumanPilot); (3) Velmeshev et al.24: single-cell RNA-seq in human cortex (rawMatrix.zip retrieved from: https://cells.ucsc.edu/?ds=autism); and (4) BrainSpan: bulk RNA-seq across human brain regions and developmental stages (“RNA-Seq Gencode v10 summarized to genes” dataset retrieved from: https://www.brainspan.org/static/download.html). We used two different methods for estimating co-expression to account for different statistical properties in these datasets. For the spatial transcriptomic datasets with very sparse gene expression matrices, we reasoned that the binary presence/absence of genes across physical locations would be the most informative, and therefore performed one-tailed Fisher’s exact tests to calculate the significance of co-occurrence for each gene pair across locations. For the other datasets, we calculated a proportionality metric, ⍴, for each gene pair using the propr R package85 (v4.2.6); this metric is analogous to conventional correlation measures but has been shown to be better at capturing functional associations between genes in RNA-seq data.89 After calculating either the Fisher’s exact P-values or the proportionality ⍴ values for all gene pairs involving each index gene, we then performed rank-based inverse normal transformation to convert the values into co-expression Z-scores (where a positive score indicates that a gene has higher than average co-expression with the index gene compared to the rest of the genome). We performed two-tailed Wilcoxon rank-sum tests to assess if the co-expression Z-scores between index genes and their interactors are significantly different from the scores between the index genes and other gene groups including non-interactors detected in IP-MS, known interactors from InWeb, and all protein-coding genes.BrainSpan analysisGene expression in four regions of the frontal cortex (dorsolateral prefrontal cortex, medial prefrontal cortex, ventrolateral prefrontal cortex, orbital frontal cortex) across 10 developmental stages were obtained from the BrainSpan exon microarray dataset32 (www.brainspan.org). At each developmental stage, we calculated the median and SE of the expression values for various gene sets, limiting to genes with available BrainSpan data: (1) ASD-associated genes reaching FWER ≤ 0.05, FDR ≤ 0.1, and FDR ≤ 0.25 in Satterstrom et al.5; (2) index and interactor genes in our combined network; (3) SCN2A, SHANK3, and SYNGAP1 interactors found only in iNs, only in brain homogenates, and shared by both sample types; (4) overlap between the FDR ≤ 0.25 genes from Satterstrom et al. and interactor genes in the combined network; (5) random genes sampled from the BrainSpan dataset for comparison against the other gene sets.Tissue enrichment analyses using GTEx dataGene tissue specificity scores in all (GTEx.tstat.tsv) and brain region (GTEx_brain.tstat.tsv) GTEx tissues, as described in Finucane et al.,34 were downloaded from: https://data.broadinstitute.org/alkesgroup/LDSCORE/LDSC_SEG_ldscores/tstats/. Following the definition in Finucane et al., we defined tissue-specific genes as the 10% of genes with the highest scores in each tissue. One-tailed hypergeometric tests were performed to assess the overlap enrichment between interactors in the combined PPI network vs. the tissue-specific genes in all tissues or the brain region tissues. In each test, the “population” was defined as all genes found in the GTEx data; within this population, “success” was the interactors in the network, “sample” was the tissue-specific genes, and “success in sample” was the overlap.Cell type enrichment analyses using scRNA-seq dataWe downloaded scRNA-seq data reported in Velmeshev et al.24 from the UCSC Cell Browser (https://cells.ucsc.edu/?ds=autism). To define “commonly expressed genes” in each of the 17 annotated cell types, we used the gene expression matrix (exprMatrix.tsv.gz) and the cell meta annotations (meta.tsv) to identify genes that have non-zero expression (i.e., log2(UMI count) > 0) in >50% of cells in a cell type. Separately, we obtained differentially expressed genes (DEGs) between ASD individuals vs. controls in each cell type from Data S4 of Velmeshev et al. One-tailed hypergeometric tests were performed to assess the overlap enrichment between interactors in the combined PPI network vs. either the commonly expressed genes or the DEGs in each cell type. In each test, the “population” was defined as either all genes found in the scRNA-seq dataset (for the global enrichment tests) or the subset of genes that are interactors or non-interactors in the network (for the conditional enrichment tests). Within this population, “success” was the interactors in the network, “sample” was the commonly expressed genes or the DEGs in a cell type, and “success in sample” was the overlap.SynGO analysisWe performed SynGO gene set enrichment analysis for genes in the combined PPI network by inputting their gene symbols into the SynGO browser35 (https://syngoportal.org; dataset release: 20180731). The SynGO “Biological Processes” annotations and the “brain expressed” background set were used in one-tailed Fisher’s exact tests to identify enriched gene sets.GO analysis for ANK2 WT- and KO-specific interactorsWe identified ANK2 “WT-specific” and “KO-specific” interactors by analyzing ANK2 IP-MS experiments performed in wild-type (WT) and giant ANK2 knockout (KO) neural progenitor cells. Specifically, the WT-specific interactors are proteins that are significant (log2 FC > 0 and FDR ≤ 0.1) in the ANK2 IP in WT cells (compared to IgG controls) but not in the ANK2 IP in KO cells; and vice versa for the KO-specific interactors. We performed GO term enrichment analysis using the GO cellular component (CC) annotations provided in the Genoppi R package18 (v1.0), which were originally downloaded from the Gene Ontology Consortium39,40 (2020-03-23 release; http://current.geneontology.org/products/pages/downloads.html). One-tailed hypergeometric tests were performed to assess the overrepresentation of CC terms among the WT- or KO-specific interactors. In each test, the “population” contained genes detected in ≥ 2 ANK2 IP replicates in WT cells or ≥ 2 ANK2 IP replicates in KO cells. Within this population, “success” was the genes annotated with a CC term, “sample” was the WT- or KO-specific interactors, and “success in sample” was the overlap.Enrichment analyses for IGF2BP1-3 targetsRNA targets of IGF2BP1, IGF2BP2, and IGF2BP3 were obtained from Table S1 of Huang et al.44; these targets were identified by both RIP and PAR-CLIP methods in HEK293T cells. First, we performed one-tailed hypergeometric tests to assess the overlap enrichment between target genes of each IGF2BP or their combined target list vs. the ASD-associated genes reaching FDR ≤0.1 in Satterstrom et al.5. In each test, the “population” was defined as all autosomal genes in Table S2 of Satterstrom et al.; within this population, “success” was the target genes, “sample” was the ASD-associated genes, and “success in sample” was the overlap. Second, to test whether the IGF2BP1-3 targets are enriched for common variants of ASDs, we performed a global MAGMA analysis as described in Common variant enrichment analysis below, except the “gene sets” tested here were the IGF2BP1-3 targets instead of the PPI network genes. Finally, we performed another set of one-tailed hypergeometric tests to assess the overlap enrichment between the IGF2BP1-3 targets vs. interactors in the combined or index protein-specific PPI networks. In each test, the “population” contained genes that are interactors or non-interactors in a network; within this population, “success” was the target genes, “sample” was the interactors in the network, and “success in sample” was the overlap.Rare variant enrichment analysisGene-based rare variant association scores were extracted from exome sequencing studies of ASD (“qval_dnccPTV” column in Table S2 of Satterstrom et al.5), DD (“denovoWEST_p_full” column in Table S2 of Kaplanis et al.50), and SCZ (“P meta” column in Table S5 of Singh et al.51). In the global enrichment analysis, for each PPI network and each trait, a one-tailed Kolmogorov-Smirnov test was performed to compare the score distribution of the interactor genes against the rest of the genome (with available score data), testing the alternative hypothesis that the cumulative distribution function of the interactors lies above that of other genes (i.e., the interactors have smaller/more significant scores). In the conditional enrichment analysis, an analogous one-tailed Kolmogorov-Smirnov test was performed to compare the scores of the interactors vs. the non-interactors associated with each PPI network.pLI score enrichment analysisGene pLI scores were obtained from the gnomAD52 (v2.1.1) “pLoF Metrics by Gene TSV” dataset. For each PPI network, a one-tailed Kolmorgov-Smirnov test was performed to compare the pLI scores of the interactor genes against other genes in the pLI dataset (for the global enrichment analysis) or against the non-interactors (for the conditional enrichment analysis), testing the alternative hypothesis that the cumulative distribution function of the interactors lies below that of other genes (i.e., the interactors have higher pLI scores).Common variant enrichment analysisGWAS summary statistics for ASD,45 ADHD,53 BIP,54 MDD,55 and SCZ56 were downloaded from the Psychiatric Genomics Consortium (www.med.unc.edu/pgc/download-results/). GWAS summary statistics for height57 were downloaded from the GIANT consortium (https://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium_data_files). All these GWAS were conducted using individuals of European (EUR) ancestry. The following analyses were performed using MAGMA46 (v1.09) to calculate the common variant enrichment of each PPI network for each GWAS phenotype. First, variants in the 1000 Genomes90 (phase 3) EUR panel were mapped to protein-coding genes in the Ensembl86 GRCh37 database with a flanking window of ±50 kb; variants in the major histocompatibility complex region (chr6:28.5M–33.4M) were excluded. Next, for each GWAS dataset, gene-based P-values were calculated using the SNP-wise Mean model and linkage disequilibrium information estimated from the 1000 Genomes EUR panel. Finally, for each GWAS dataset and each PPI network, the gene-set analysis model in MAGMA was used to compare the interactor genes in the network against the rest of the protein-coding genome (for the global enrichment tests) or the non-interactor genes (for the conditional enrichment tests), computing a one-tailed P-value to support the alternative hypothesis that the interactors are more strongly associated with the GWAS phenotype.Social Manhattan plotASD-associated genes with FDR ≤ 0.25 were obtained from Table S2 of Satterstrom et al.,5 along with their GRCh38 genomic positions and Q-values. The combined PPI network was intersected with the ASD gene list to identify the subset of genes that are either an index gene or an interactor in the network. The genomic positions and -log10 Q-values of these genes were plotted in a Manhattan plot, with visual links between the genes to indicate PPIs in the network. For visualization, the minimum Q-value was set to 1e-16 in the plot.

Quantification and statistical analysis

RNA-seq data analysisStarting with RNA-seq data in FASTQ format, we ran Trim Galore79 (v0.6.7) to trim adapters and low-quality reads using “--stringency 7” and other default settings for Illumina paired-end reads. Sequence alignment was performed using HISAT280 (v2.2.1) with the “--rna-strandness FR” setting; index files for the GRCh38 reference genome with transcripts were downloaded from the HISAT2 website (https://daehwankimlab.github.io/hisat2/download/). The htseq-count script in Htseq81 (v0.13.5) was used to generate gene counts from aligned reads with “--stranded yes --minaqual 1” and other default settings; GRCh38 GTF gene annotation file was obtained from Ensembl86 (release 84). The EdgeR package82 (v3.34.0) was used to remove low-count genes (min.count = 5 in filterByExpr function) before calculating normalized gene counts (method = “TMM” in calcNormFactors function).

RNA-seq data analysis

Starting with RNA-seq data in FASTQ format, we ran Trim Galore79 (v0.6.7) to trim adapters and low-quality reads using “--stringency 7” and other default settings for Illumina paired-end reads. Sequence alignment was performed using HISAT280 (v2.2.1) with the “--rna-strandness FR” setting; index files for the GRCh38 reference genome with transcripts were downloaded from the HISAT2 website (https://daehwankimlab.github.io/hisat2/download/). The htseq-count script in Htseq81 (v0.13.5) was used to generate gene counts from aligned reads with “--stranded yes --minaqual 1” and other default settings; GRCh38 GTF gene annotation file was obtained from Ensembl86 (release 84). The EdgeR package82 (v3.34.0) was used to remove low-count genes (min.count = 5 in filterByExpr function) before calculating normalized gene counts (method = “TMM” in calcNormFactors function).

scRNA-seq t-SNE plotsSingle-cell RNA sequencing (scRNA-seq) data from Velmeshev et al.24 were downloaded from the UCSC Cell Browser (https://cells.ucsc.edu/?ds=autism#). The gene expression matrix (exprMatrix.tsv.gz), cell meta annotations (meta.tsv), dimensionality reduction coordinates (tSNE.coords.tsv.gz), and the Seurat R package83 (v3.2.0) were used to generate t-SNE plots showing either gene expression or cell type annotations across cell clusters.

scRNA-seq t-SNE plots

Single-cell RNA sequencing (scRNA-seq) data from Velmeshev et al.24 were downloaded from the UCSC Cell Browser (https://cells.ucsc.edu/?ds=autism#). The gene expression matrix (exprMatrix.tsv.gz), cell meta annotations (meta.tsv), dimensionality reduction coordinates (tSNE.coords.tsv.gz), and the Seurat R package83 (v3.2.0) were used to generate t-SNE plots showing either gene expression or cell type annotations across cell clusters.

Protein quantification from raw IP-MS dataSoftware and methods used by each MS facility/site to generate protein-level quantification reports from raw IP-MS data are described below; Table S2 links each IP-MS dataset we analyzed to each facility.Broad: Mass spectra were analyzed using Spectrum Mill (v7.0; https://proteomics.broadinstitute.org). For peptide identification, MS/MS spectra were searched against a sequence database for the UniProt human reference proteome, including isoforms, with a set of common laboratory contaminant proteins appended (2017: 65,068 entries). Search parameters included: ESI-QEXACTIVE-HCD scoring parameters, trypsin enzyme specificity with a maximum of two missed cleavages, 40% minimum matched peak intensity, ±20 ppm precursor mass tolerance, ±20 ppm product mass tolerance. Carbamidomethylation of cysteines and iTRAQ4 full labeling of lysines and peptide n-termini were set as fixed modifications. Allowed variable modifications were oxidation of methionine (M), acetyl (ProtN-term), and deamidated (N), with a precursor MH + shift range of −18 to 64 Da. Identities interpreted for individual spectra were automatically designated as valid by optimizing score and delta rank1-rank2 score thresholds separately for each precursor charge state in each LC-MS/MS while allowing a maximum target-decoy-based false discovery rate (FDR) of 1.0% at the spectrum level. Identified peptides were organized into protein groups and subgroups (isoforms and family members) with Spectrum Mill’s subgroup specific option enabled, so that peptides shared between subgroups are ignored when using reporter ion intensities to perform protein-level quantitation.Whitehead: Mass spectra were analyzed using PEAKS Studio (v8.5; Bioinformatics Solutions). Peptide and protein identification was performed by searching against the UniProt human reference proteome, including isoforms (Swiss-Prot/TrEMBL, release 2019_01) together with a set of common contaminants. An FDR threshold of 1% was used for identification of peptides and proteins. Relative ratios of the iTRAQ 4-plex reporter ions were used for protein-level quantitation.CNCR: Mass spectra were analyzed using MaxQuant84 (v1.6.1.0 or v1.6.3.4; https://www.maxquant.org). Peptide and protein identification was performed by searching against the UniProt human database, and label-free quantitation was performed to generate protein-level quantification reports. MaxQuant versions and settings used for each dataset can be found in the deposited parameters file linked to each dataset (Table S12).Harvard: Mass spectra were analyzed using Proteome Discoverer (v2.4; Thermo Scientific). Assignment of MS/MS spectra was performed using the Sequest HT algorithm by searching the data against the UniProt human reference proteome, including isoforms (Swiss-Prot, release 2019_01) as well as other known contaminants such as human keratins and common laboratory contaminants. Sequest HT searches were performed using a 10 ppm precursor ion tolerance and requiring each peptide’s N/C termini to adhere with trypsin protease specificity, while allowing up to two missed cleavages. CID-MS2 spectra were searched with 0.5 Da ion tolerance for fragmentation. Methionine oxidation (+15.99492 Da) was set as variable modification. An MS2 spectra assignment FDR of 1% was applied to both proteins and peptides using the Percolator target-decoy database search. Label-free quantitation was performed to generate protein-level quantification reports.

Protein quantification from raw IP-MS data

Software and methods used by each MS facility/site to generate protein-level quantification reports from raw IP-MS data are described below; Table S2 links each IP-MS dataset we analyzed to each facility.

Broad: Mass spectra were analyzed using Spectrum Mill (v7.0; https://proteomics.broadinstitute.org). For peptide identification, MS/MS spectra were searched against a sequence database for the UniProt human reference proteome, including isoforms, with a set of common laboratory contaminant proteins appended (2017: 65,068 entries). Search parameters included: ESI-QEXACTIVE-HCD scoring parameters, trypsin enzyme specificity with a maximum of two missed cleavages, 40% minimum matched peak intensity, ±20 ppm precursor mass tolerance, ±20 ppm product mass tolerance. Carbamidomethylation of cysteines and iTRAQ4 full labeling of lysines and peptide n-termini were set as fixed modifications. Allowed variable modifications were oxidation of methionine (M), acetyl (ProtN-term), and deamidated (N), with a precursor MH + shift range of −18 to 64 Da. Identities interpreted for individual spectra were automatically designated as valid by optimizing score and delta rank1-rank2 score thresholds separately for each precursor charge state in each LC-MS/MS while allowing a maximum target-decoy-based false discovery rate (FDR) of 1.0% at the spectrum level. Identified peptides were organized into protein groups and subgroups (isoforms and family members) with Spectrum Mill’s subgroup specific option enabled, so that peptides shared between subgroups are ignored when using reporter ion intensities to perform protein-level quantitation.

Whitehead: Mass spectra were analyzed using PEAKS Studio (v8.5; Bioinformatics Solutions). Peptide and protein identification was performed by searching against the UniProt human reference proteome, including isoforms (Swiss-Prot/TrEMBL, release 2019_01) together with a set of common contaminants. An FDR threshold of 1% was used for identification of peptides and proteins. Relative ratios of the iTRAQ 4-plex reporter ions were used for protein-level quantitation.

CNCR: Mass spectra were analyzed using MaxQuant84 (v1.6.1.0 or v1.6.3.4; https://www.maxquant.org). Peptide and protein identification was performed by searching against the UniProt human database, and label-free quantitation was performed to generate protein-level quantification reports. MaxQuant versions and settings used for each dataset can be found in the deposited parameters file linked to each dataset (Table S12).

Harvard: Mass spectra were analyzed using Proteome Discoverer (v2.4; Thermo Scientific). Assignment of MS/MS spectra was performed using the Sequest HT algorithm by searching the data against the UniProt human reference proteome, including isoforms (Swiss-Prot, release 2019_01) as well as other known contaminants such as human keratins and common laboratory contaminants. Sequest HT searches were performed using a 10 ppm precursor ion tolerance and requiring each peptide’s N/C termini to adhere with trypsin protease specificity, while allowing up to two missed cleavages. CID-MS2 spectra were searched with 0.5 Da ion tolerance for fragmentation. Methionine oxidation (+15.99492 Da) was set as variable modification. An MS2 spectra assignment FDR of 1% was applied to both proteins and peptides using the Percolator target-decoy database search. Label-free quantitation was performed to generate protein-level quantification reports.

IP-MS data processing and analysisData processing: Starting with protein-level quantification reports generated by the MS analytical software described above, we generally performed the following data processing procedures for each IP-MS dataset: (1) log2 transformation and median normalization of protein intensity values in each bait (i.e., index protein) or IgG control IP sample; (2) removing contaminants, proteins supported by < 1 unique peptide, and proteins detected in < 2 bait samples; (3) imputing missing protein intensity values in each sample by randomly sampling from a normal distribution with mean of μ - 1.8σ and SD of 0.3σ, where μ and σ are the mean and SD of the observed intensity values in the sample87; (4) calculating log2 FC for each replicate pair of bait vs. IgG control samples. However, due to differences in MS protocols, we modified these steps for specific datasets: for data generated at Broad and Whitehead, few missing values were present due to the labeled quantification approach, therefore imputation was not performed and only proteins with no missingness across all bait and control samples were included for analysis; for data generated at CNCR, instead of comparing bait samples against IgG controls to calculate log2 FC, the median of several negative IPs (i.e., bait samples in which the bait protein itself was not detected; Table S12) from the same MS run were used as controls.Genoppi analysis: We analyzed each processed IP-MS dataset using the Genoppi R package18 (v1.0), including: (1) calculating Pearson’s correlation of log2 FC values between replicates; (2) calculating average log2 FC, and corresponding P-value and false discovery rate (FDR), for each protein across replicates using a two-tailed one-sample moderated t-test88; (3) identifying statistically significant (log2 FC > 0 and FDR ≤ 0.1) interactors of the bait; (4) assessing overlap between the identified interactors and known interactors from the InWeb database,17 as curated in the Genoppi R package. We QC’ed each dataset using two criteria: the replicate log2 FC correlation must be > 0.6 and the bait protein itself must be significant (log2 FC > 0 and FDR ≤ 0.1); datasets that failed to meet these criteria were excluded from further analysis. All IP-MS analysis results are provided in Table S2.Defining non-interactors: For each QC’ed IP-MS dataset derived from iNs, besides identifying significant interactors as described above, we also defined a matching set of “non-interactors” (Table S3) to serve as proxy for the background iN proteome in conditional enrichment analyses. We applied different definitions for datasets generated by the labeled vs. label-free quantification approaches: for labeled datasets generated at Broad and Whitehead, we considered all non-significant proteins (i.e., log2 FC ≤ 0 or FDR > 0.1) in Genoppi analysis to be non-interactors; for label-free datasets generated at CNCR, we included both non-significant proteins in Genoppi analysis and other proteins in the MS quantification report that were detected in < 2 bait samples (which were filtered out prior to Genoppi analysis).Generating PPI networks: We consolidated all QC’ed iN-derived IP-MS datasets into a combined PPI network; we also combined the datasets for the same index proteins into index protein-specific networks. The interactors in each network are proteins that show up as significant interactors in ≥ 1 source IP-MS datasets contributing to the network; the matching “non-interactors” are proteins that show up as non-interactors in ≥ 1 source dataset but never as interactors in any source dataset. The full lists of interactors and non-interactors associated with each PPI network are provided in Table S3; we note that in this table, we removed all index proteins from both the interactor and non-interactor lists for the combined PPI network to avoid biasing downstream enrichment analyses.

IP-MS data processing and analysis

Data processing: Starting with protein-level quantification reports generated by the MS analytical software described above, we generally performed the following data processing procedures for each IP-MS dataset: (1) log2 transformation and median normalization of protein intensity values in each bait (i.e., index protein) or IgG control IP sample; (2) removing contaminants, proteins supported by < 1 unique peptide, and proteins detected in < 2 bait samples; (3) imputing missing protein intensity values in each sample by randomly sampling from a normal distribution with mean of μ - 1.8σ and SD of 0.3σ, where μ and σ are the mean and SD of the observed intensity values in the sample87; (4) calculating log2 FC for each replicate pair of bait vs. IgG control samples. However, due to differences in MS protocols, we modified these steps for specific datasets: for data generated at Broad and Whitehead, few missing values were present due to the labeled quantification approach, therefore imputation was not performed and only proteins with no missingness across all bait and control samples were included for analysis; for data generated at CNCR, instead of comparing bait samples against IgG controls to calculate log2 FC, the median of several negative IPs (i.e., bait samples in which the bait protein itself was not detected; Table S12) from the same MS run were used as controls.

Genoppi analysis: We analyzed each processed IP-MS dataset using the Genoppi R package18 (v1.0), including: (1) calculating Pearson’s correlation of log2 FC values between replicates; (2) calculating average log2 FC, and corresponding P-value and false discovery rate (FDR), for each protein across replicates using a two-tailed one-sample moderated t-test88; (3) identifying statistically significant (log2 FC > 0 and FDR ≤ 0.1) interactors of the bait; (4) assessing overlap between the identified interactors and known interactors from the InWeb database,17 as curated in the Genoppi R package. We QC’ed each dataset using two criteria: the replicate log2 FC correlation must be > 0.6 and the bait protein itself must be significant (log2 FC > 0 and FDR ≤ 0.1); datasets that failed to meet these criteria were excluded from further analysis. All IP-MS analysis results are provided in Table S2.

Defining non-interactors: For each QC’ed IP-MS dataset derived from iNs, besides identifying significant interactors as described above, we also defined a matching set of “non-interactors” (Table S3) to serve as proxy for the background iN proteome in conditional enrichment analyses. We applied different definitions for datasets generated by the labeled vs. label-free quantification approaches: for labeled datasets generated at Broad and Whitehead, we considered all non-significant proteins (i.e., log2 FC ≤ 0 or FDR > 0.1) in Genoppi analysis to be non-interactors; for label-free datasets generated at CNCR, we included both non-significant proteins in Genoppi analysis and other proteins in the MS quantification report that were detected in < 2 bait samples (which were filtered out prior to Genoppi analysis).

Generating PPI networks: We consolidated all QC’ed iN-derived IP-MS datasets into a combined PPI network; we also combined the datasets for the same index proteins into index protein-specific networks. The interactors in each network are proteins that show up as significant interactors in ≥ 1 source IP-MS datasets contributing to the network; the matching “non-interactors” are proteins that show up as non-interactors in ≥ 1 source dataset but never as interactors in any source dataset. The full lists of interactors and non-interactors associated with each PPI network are provided in Table S3; we note that in this table, we removed all index proteins from both the interactor and non-interactor lists for the combined PPI network to avoid biasing downstream enrichment analyses.

Co-expression analysisPairwise co-expression between each index gene and all other protein-coding genes were estimated using data from four independent studies, including (1) Stickels et al.33: spatial transcriptomics in mouse neocortex (Puck_190921_19.digital_expression.txt.gz retrieved from: https://singlecell.broadinstitute.org/single_cell/study/SCP815/highly-sensitive-spatial-transcriptomics-at-near-cellular-resolution-with-slide-seqv2#study-download); (2) Maynard et al.31: spatial transcriptomics in human dorsolateral prefrontal cortex (count matrix retrieved from spatialLIBD R package: https://github.com/LieberInstitute/HumanPilot); (3) Velmeshev et al.24: single-cell RNA-seq in human cortex (rawMatrix.zip retrieved from: https://cells.ucsc.edu/?ds=autism); and (4) BrainSpan: bulk RNA-seq across human brain regions and developmental stages (“RNA-Seq Gencode v10 summarized to genes” dataset retrieved from: https://www.brainspan.org/static/download.html). We used two different methods for estimating co-expression to account for different statistical properties in these datasets. For the spatial transcriptomic datasets with very sparse gene expression matrices, we reasoned that the binary presence/absence of genes across physical locations would be the most informative, and therefore performed one-tailed Fisher’s exact tests to calculate the significance of co-occurrence for each gene pair across locations. For the other datasets, we calculated a proportionality metric, ⍴, for each gene pair using the propr R package85 (v4.2.6); this metric is analogous to conventional correlation measures but has been shown to be better at capturing functional associations between genes in RNA-seq data.89 After calculating either the Fisher’s exact P-values or the proportionality ⍴ values for all gene pairs involving each index gene, we then performed rank-based inverse normal transformation to convert the values into co-expression Z-scores (where a positive score indicates that a gene has higher than average co-expression with the index gene compared to the rest of the genome). We performed two-tailed Wilcoxon rank-sum tests to assess if the co-expression Z-scores between index genes and their interactors are significantly different from the scores between the index genes and other gene groups including non-interactors detected in IP-MS, known interactors from InWeb, and all protein-coding genes.

Co-expression analysis

Pairwise co-expression between each index gene and all other protein-coding genes were estimated using data from four independent studies, including (1) Stickels et al.33: spatial transcriptomics in mouse neocortex (Puck_190921_19.digital_expression.txt.gz retrieved from: https://singlecell.broadinstitute.org/single_cell/study/SCP815/highly-sensitive-spatial-transcriptomics-at-near-cellular-resolution-with-slide-seqv2#study-download); (2) Maynard et al.31: spatial transcriptomics in human dorsolateral prefrontal cortex (count matrix retrieved from spatialLIBD R package: https://github.com/LieberInstitute/HumanPilot); (3) Velmeshev et al.24: single-cell RNA-seq in human cortex (rawMatrix.zip retrieved from: https://cells.ucsc.edu/?ds=autism); and (4) BrainSpan: bulk RNA-seq across human brain regions and developmental stages (“RNA-Seq Gencode v10 summarized to genes” dataset retrieved from: https://www.brainspan.org/static/download.html). We used two different methods for estimating co-expression to account for different statistical properties in these datasets. For the spatial transcriptomic datasets with very sparse gene expression matrices, we reasoned that the binary presence/absence of genes across physical locations would be the most informative, and therefore performed one-tailed Fisher’s exact tests to calculate the significance of co-occurrence for each gene pair across locations. For the other datasets, we calculated a proportionality metric, ⍴, for each gene pair using the propr R package85 (v4.2.6); this metric is analogous to conventional correlation measures but has been shown to be better at capturing functional associations between genes in RNA-seq data.89 After calculating either the Fisher’s exact P-values or the proportionality ⍴ values for all gene pairs involving each index gene, we then performed rank-based inverse normal transformation to convert the values into co-expression Z-scores (where a positive score indicates that a gene has higher than average co-expression with the index gene compared to the rest of the genome). We performed two-tailed Wilcoxon rank-sum tests to assess if the co-expression Z-scores between index genes and their interactors are significantly different from the scores between the index genes and other gene groups including non-interactors detected in IP-MS, known interactors from InWeb, and all protein-coding genes.

BrainSpan analysisGene expression in four regions of the frontal cortex (dorsolateral prefrontal cortex, medial prefrontal cortex, ventrolateral prefrontal cortex, orbital frontal cortex) across 10 developmental stages were obtained from the BrainSpan exon microarray dataset32 (www.brainspan.org). At each developmental stage, we calculated the median and SE of the expression values for various gene sets, limiting to genes with available BrainSpan data: (1) ASD-associated genes reaching FWER ≤ 0.05, FDR ≤ 0.1, and FDR ≤ 0.25 in Satterstrom et al.5; (2) index and interactor genes in our combined network; (3) SCN2A, SHANK3, and SYNGAP1 interactors found only in iNs, only in brain homogenates, and shared by both sample types; (4) overlap between the FDR ≤ 0.25 genes from Satterstrom et al. and interactor genes in the combined network; (5) random genes sampled from the BrainSpan dataset for comparison against the other gene sets.

BrainSpan analysis

Gene expression in four regions of the frontal cortex (dorsolateral prefrontal cortex, medial prefrontal cortex, ventrolateral prefrontal cortex, orbital frontal cortex) across 10 developmental stages were obtained from the BrainSpan exon microarray dataset32 (www.brainspan.org). At each developmental stage, we calculated the median and SE of the expression values for various gene sets, limiting to genes with available BrainSpan data: (1) ASD-associated genes reaching FWER ≤ 0.05, FDR ≤ 0.1, and FDR ≤ 0.25 in Satterstrom et al.5; (2) index and interactor genes in our combined network; (3) SCN2A, SHANK3, and SYNGAP1 interactors found only in iNs, only in brain homogenates, and shared by both sample types; (4) overlap between the FDR ≤ 0.25 genes from Satterstrom et al. and interactor genes in the combined network; (5) random genes sampled from the BrainSpan dataset for comparison against the other gene sets.

Tissue enrichment analyses using GTEx dataGene tissue specificity scores in all (GTEx.tstat.tsv) and brain region (GTEx_brain.tstat.tsv) GTEx tissues, as described in Finucane et al.,34 were downloaded from: https://data.broadinstitute.org/alkesgroup/LDSCORE/LDSC_SEG_ldscores/tstats/. Following the definition in Finucane et al., we defined tissue-specific genes as the 10% of genes with the highest scores in each tissue. One-tailed hypergeometric tests were performed to assess the overlap enrichment between interactors in the combined PPI network vs. the tissue-specific genes in all tissues or the brain region tissues. In each test, the “population” was defined as all genes found in the GTEx data; within this population, “success” was the interactors in the network, “sample” was the tissue-specific genes, and “success in sample” was the overlap.

Tissue enrichment analyses using GTEx data

Gene tissue specificity scores in all (GTEx.tstat.tsv) and brain region (GTEx_brain.tstat.tsv) GTEx tissues, as described in Finucane et al.,34 were downloaded from: https://data.broadinstitute.org/alkesgroup/LDSCORE/LDSC_SEG_ldscores/tstats/. Following the definition in Finucane et al., we defined tissue-specific genes as the 10% of genes with the highest scores in each tissue. One-tailed hypergeometric tests were performed to assess the overlap enrichment between interactors in the combined PPI network vs. the tissue-specific genes in all tissues or the brain region tissues. In each test, the “population” was defined as all genes found in the GTEx data; within this population, “success” was the interactors in the network, “sample” was the tissue-specific genes, and “success in sample” was the overlap.

Cell type enrichment analyses using scRNA-seq dataWe downloaded scRNA-seq data reported in Velmeshev et al.24 from the UCSC Cell Browser (https://cells.ucsc.edu/?ds=autism). To define “commonly expressed genes” in each of the 17 annotated cell types, we used the gene expression matrix (exprMatrix.tsv.gz) and the cell meta annotations (meta.tsv) to identify genes that have non-zero expression (i.e., log2(UMI count) > 0) in >50% of cells in a cell type. Separately, we obtained differentially expressed genes (DEGs) between ASD individuals vs. controls in each cell type from Data S4 of Velmeshev et al. One-tailed hypergeometric tests were performed to assess the overlap enrichment between interactors in the combined PPI network vs. either the commonly expressed genes or the DEGs in each cell type. In each test, the “population” was defined as either all genes found in the scRNA-seq dataset (for the global enrichment tests) or the subset of genes that are interactors or non-interactors in the network (for the conditional enrichment tests). Within this population, “success” was the interactors in the network, “sample” was the commonly expressed genes or the DEGs in a cell type, and “success in sample” was the overlap.

Cell type enrichment analyses using scRNA-seq data

We downloaded scRNA-seq data reported in Velmeshev et al.24 from the UCSC Cell Browser (https://cells.ucsc.edu/?ds=autism). To define “commonly expressed genes” in each of the 17 annotated cell types, we used the gene expression matrix (exprMatrix.tsv.gz) and the cell meta annotations (meta.tsv) to identify genes that have non-zero expression (i.e., log2(UMI count) > 0) in >50% of cells in a cell type. Separately, we obtained differentially expressed genes (DEGs) between ASD individuals vs. controls in each cell type from Data S4 of Velmeshev et al. One-tailed hypergeometric tests were performed to assess the overlap enrichment between interactors in the combined PPI network vs. either the commonly expressed genes or the DEGs in each cell type. In each test, the “population” was defined as either all genes found in the scRNA-seq dataset (for the global enrichment tests) or the subset of genes that are interactors or non-interactors in the network (for the conditional enrichment tests). Within this population, “success” was the interactors in the network, “sample” was the commonly expressed genes or the DEGs in a cell type, and “success in sample” was the overlap.

SynGO analysisWe performed SynGO gene set enrichment analysis for genes in the combined PPI network by inputting their gene symbols into the SynGO browser35 (https://syngoportal.org; dataset release: 20180731). The SynGO “Biological Processes” annotations and the “brain expressed” background set were used in one-tailed Fisher’s exact tests to identify enriched gene sets.

SynGO analysis

We performed SynGO gene set enrichment analysis for genes in the combined PPI network by inputting their gene symbols into the SynGO browser35 (https://syngoportal.org; dataset release: 20180731). The SynGO “Biological Processes” annotations and the “brain expressed” background set were used in one-tailed Fisher’s exact tests to identify enriched gene sets.

GO analysis for ANK2 WT- and KO-specific interactorsWe identified ANK2 “WT-specific” and “KO-specific” interactors by analyzing ANK2 IP-MS experiments performed in wild-type (WT) and giant ANK2 knockout (KO) neural progenitor cells. Specifically, the WT-specific interactors are proteins that are significant (log2 FC > 0 and FDR ≤ 0.1) in the ANK2 IP in WT cells (compared to IgG controls) but not in the ANK2 IP in KO cells; and vice versa for the KO-specific interactors. We performed GO term enrichment analysis using the GO cellular component (CC) annotations provided in the Genoppi R package18 (v1.0), which were originally downloaded from the Gene Ontology Consortium39,40 (2020-03-23 release; http://current.geneontology.org/products/pages/downloads.html). One-tailed hypergeometric tests were performed to assess the overrepresentation of CC terms among the WT- or KO-specific interactors. In each test, the “population” contained genes detected in ≥ 2 ANK2 IP replicates in WT cells or ≥ 2 ANK2 IP replicates in KO cells. Within this population, “success” was the genes annotated with a CC term, “sample” was the WT- or KO-specific interactors, and “success in sample” was the overlap.

GO analysis for ANK2 WT- and KO-specific interactors

We identified ANK2 “WT-specific” and “KO-specific” interactors by analyzing ANK2 IP-MS experiments performed in wild-type (WT) and giant ANK2 knockout (KO) neural progenitor cells. Specifically, the WT-specific interactors are proteins that are significant (log2 FC > 0 and FDR ≤ 0.1) in the ANK2 IP in WT cells (compared to IgG controls) but not in the ANK2 IP in KO cells; and vice versa for the KO-specific interactors. We performed GO term enrichment analysis using the GO cellular component (CC) annotations provided in the Genoppi R package18 (v1.0), which were originally downloaded from the Gene Ontology Consortium39,40 (2020-03-23 release; http://current.geneontology.org/products/pages/downloads.html). One-tailed hypergeometric tests were performed to assess the overrepresentation of CC terms among the WT- or KO-specific interactors. In each test, the “population” contained genes detected in ≥ 2 ANK2 IP replicates in WT cells or ≥ 2 ANK2 IP replicates in KO cells. Within this population, “success” was the genes annotated with a CC term, “sample” was the WT- or KO-specific interactors, and “success in sample” was the overlap.

Enrichment analyses for IGF2BP1-3 targetsRNA targets of IGF2BP1, IGF2BP2, and IGF2BP3 were obtained from Table S1 of Huang et al.44; these targets were identified by both RIP and PAR-CLIP methods in HEK293T cells. First, we performed one-tailed hypergeometric tests to assess the overlap enrichment between target genes of each IGF2BP or their combined target list vs. the ASD-associated genes reaching FDR ≤0.1 in Satterstrom et al.5. In each test, the “population” was defined as all autosomal genes in Table S2 of Satterstrom et al.; within this population, “success” was the target genes, “sample” was the ASD-associated genes, and “success in sample” was the overlap. Second, to test whether the IGF2BP1-3 targets are enriched for common variants of ASDs, we performed a global MAGMA analysis as described in Common variant enrichment analysis below, except the “gene sets” tested here were the IGF2BP1-3 targets instead of the PPI network genes. Finally, we performed another set of one-tailed hypergeometric tests to assess the overlap enrichment between the IGF2BP1-3 targets vs. interactors in the combined or index protein-specific PPI networks. In each test, the “population” contained genes that are interactors or non-interactors in a network; within this population, “success” was the target genes, “sample” was the interactors in the network, and “success in sample” was the overlap.

Enrichment analyses for IGF2BP1-3 targets

RNA targets of IGF2BP1, IGF2BP2, and IGF2BP3 were obtained from Table S1 of Huang et al.44; these targets were identified by both RIP and PAR-CLIP methods in HEK293T cells. First, we performed one-tailed hypergeometric tests to assess the overlap enrichment between target genes of each IGF2BP or their combined target list vs. the ASD-associated genes reaching FDR ≤0.1 in Satterstrom et al.5. In each test, the “population” was defined as all autosomal genes in Table S2 of Satterstrom et al.; within this population, “success” was the target genes, “sample” was the ASD-associated genes, and “success in sample” was the overlap. Second, to test whether the IGF2BP1-3 targets are enriched for common variants of ASDs, we performed a global MAGMA analysis as described in Common variant enrichment analysis below, except the “gene sets” tested here were the IGF2BP1-3 targets instead of the PPI network genes. Finally, we performed another set of one-tailed hypergeometric tests to assess the overlap enrichment between the IGF2BP1-3 targets vs. interactors in the combined or index protein-specific PPI networks. In each test, the “population” contained genes that are interactors or non-interactors in a network; within this population, “success” was the target genes, “sample” was the interactors in the network, and “success in sample” was the overlap.

Rare variant enrichment analysisGene-based rare variant association scores were extracted from exome sequencing studies of ASD (“qval_dnccPTV” column in Table S2 of Satterstrom et al.5), DD (“denovoWEST_p_full” column in Table S2 of Kaplanis et al.50), and SCZ (“P meta” column in Table S5 of Singh et al.51). In the global enrichment analysis, for each PPI network and each trait, a one-tailed Kolmogorov-Smirnov test was performed to compare the score distribution of the interactor genes against the rest of the genome (with available score data), testing the alternative hypothesis that the cumulative distribution function of the interactors lies above that of other genes (i.e., the interactors have smaller/more significant scores). In the conditional enrichment analysis, an analogous one-tailed Kolmogorov-Smirnov test was performed to compare the scores of the interactors vs. the non-interactors associated with each PPI network.

Rare variant enrichment analysis

Gene-based rare variant association scores were extracted from exome sequencing studies of ASD (“qval_dnccPTV” column in Table S2 of Satterstrom et al.5), DD (“denovoWEST_p_full” column in Table S2 of Kaplanis et al.50), and SCZ (“P meta” column in Table S5 of Singh et al.51). In the global enrichment analysis, for each PPI network and each trait, a one-tailed Kolmogorov-Smirnov test was performed to compare the score distribution of the interactor genes against the rest of the genome (with available score data), testing the alternative hypothesis that the cumulative distribution function of the interactors lies above that of other genes (i.e., the interactors have smaller/more significant scores). In the conditional enrichment analysis, an analogous one-tailed Kolmogorov-Smirnov test was performed to compare the scores of the interactors vs. the non-interactors associated with each PPI network.

pLI score enrichment analysisGene pLI scores were obtained from the gnomAD52 (v2.1.1) “pLoF Metrics by Gene TSV” dataset. For each PPI network, a one-tailed Kolmorgov-Smirnov test was performed to compare the pLI scores of the interactor genes against other genes in the pLI dataset (for the global enrichment analysis) or against the non-interactors (for the conditional enrichment analysis), testing the alternative hypothesis that the cumulative distribution function of the interactors lies below that of other genes (i.e., the interactors have higher pLI scores).

pLI score enrichment analysis

Gene pLI scores were obtained from the gnomAD52 (v2.1.1) “pLoF Metrics by Gene TSV” dataset. For each PPI network, a one-tailed Kolmorgov-Smirnov test was performed to compare the pLI scores of the interactor genes against other genes in the pLI dataset (for the global enrichment analysis) or against the non-interactors (for the conditional enrichment analysis), testing the alternative hypothesis that the cumulative distribution function of the interactors lies below that of other genes (i.e., the interactors have higher pLI scores).

Common variant enrichment analysisGWAS summary statistics for ASD,45 ADHD,53 BIP,54 MDD,55 and SCZ56 were downloaded from the Psychiatric Genomics Consortium (www.med.unc.edu/pgc/download-results/). GWAS summary statistics for height57 were downloaded from the GIANT consortium (https://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium_data_files). All these GWAS were conducted using individuals of European (EUR) ancestry. The following analyses were performed using MAGMA46 (v1.09) to calculate the common variant enrichment of each PPI network for each GWAS phenotype. First, variants in the 1000 Genomes90 (phase 3) EUR panel were mapped to protein-coding genes in the Ensembl86 GRCh37 database with a flanking window of ±50 kb; variants in the major histocompatibility complex region (chr6:28.5M–33.4M) were excluded. Next, for each GWAS dataset, gene-based P-values were calculated using the SNP-wise Mean model and linkage disequilibrium information estimated from the 1000 Genomes EUR panel. Finally, for each GWAS dataset and each PPI network, the gene-set analysis model in MAGMA was used to compare the interactor genes in the network against the rest of the protein-coding genome (for the global enrichment tests) or the non-interactor genes (for the conditional enrichment tests), computing a one-tailed P-value to support the alternative hypothesis that the interactors are more strongly associated with the GWAS phenotype.

Common variant enrichment analysis

GWAS summary statistics for ASD,45 ADHD,53 BIP,54 MDD,55 and SCZ56 were downloaded from the Psychiatric Genomics Consortium (www.med.unc.edu/pgc/download-results/). GWAS summary statistics for height57 were downloaded from the GIANT consortium (https://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium_data_files). All these GWAS were conducted using individuals of European (EUR) ancestry. The following analyses were performed using MAGMA46 (v1.09) to calculate the common variant enrichment of each PPI network for each GWAS phenotype. First, variants in the 1000 Genomes90 (phase 3) EUR panel were mapped to protein-coding genes in the Ensembl86 GRCh37 database with a flanking window of ±50 kb; variants in the major histocompatibility complex region (chr6:28.5M–33.4M) were excluded. Next, for each GWAS dataset, gene-based P-values were calculated using the SNP-wise Mean model and linkage disequilibrium information estimated from the 1000 Genomes EUR panel. Finally, for each GWAS dataset and each PPI network, the gene-set analysis model in MAGMA was used to compare the interactor genes in the network against the rest of the protein-coding genome (for the global enrichment tests) or the non-interactor genes (for the conditional enrichment tests), computing a one-tailed P-value to support the alternative hypothesis that the interactors are more strongly associated with the GWAS phenotype.

Social Manhattan plotASD-associated genes with FDR ≤ 0.25 were obtained from Table S2 of Satterstrom et al.,5 along with their GRCh38 genomic positions and Q-values. The combined PPI network was intersected with the ASD gene list to identify the subset of genes that are either an index gene or an interactor in the network. The genomic positions and -log10 Q-values of these genes were plotted in a Manhattan plot, with visual links between the genes to indicate PPIs in the network. For visualization, the minimum Q-value was set to 1e-16 in the plot.

Social Manhattan plot

ASD-associated genes with FDR ≤ 0.25 were obtained from Table S2 of Satterstrom et al.,5 along with their GRCh38 genomic positions and Q-values. The combined PPI network was intersected with the ASD gene list to identify the subset of genes that are either an index gene or an interactor in the network. The genomic positions and -log10 Q-values of these genes were plotted in a Manhattan plot, with visual links between the genes to indicate PPIs in the network. For visualization, the minimum Q-value was set to 1e-16 in the plot.
