---
pmid: '22653443'
title: Mutant p53 interactome identifies nardilysin as a p53R273H-specific binding
  partner that promotes invasion.
authors:
- Coffill CR
- Muller PA
- Oh HK
- Neo SP
- Hogue KA
- Cheok CF
- Vousden KH
- Lane DP
- Blackstock WP
- Gunaratne J
journal: EMBO Rep
year: '2012'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC3388785
doi: 10.1038/embor.2012.74
---

# Mutant p53 interactome identifies nardilysin as a p53R273H-specific binding partner that promotes invasion.
**Authors:** Coffill CR, Muller PA, Oh HK, Neo SP, Hogue KA, Cheok CF, Vousden KH, Lane DP, Blackstock WP, Gunaratne J
**Journal:** EMBO Rep (2012)
**DOI:** [10.1038/embor.2012.74](https://doi.org/10.1038/embor.2012.74)
**PMC:** [PMC3388785](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3388785/)

## Abstract

1. EMBO Rep. 2012 Jun 29;13(7):638-44. doi: 10.1038/embor.2012.74.

Mutant p53 interactome identifies nardilysin as a p53R273H-specific binding 
partner that promotes invasion.

Coffill CR(1), Muller PA, Oh HK, Neo SP, Hogue KA, Cheok CF, Vousden KH, Lane 
DP, Blackstock WP, Gunaratne J.

Author information:
(1)Quantitative Proteomics Group, Institute of Molecular and Cell Biology, 61 
Biopolis Drive, Proteos, Singapore 138673, Singapore.

The invasiveness of tumour cells depends on changes in cell shape, polarity and 
migration. Mutant p53 induces enhanced tumour metastasis in mice, and human 
cells overexpressing p53R273H have aberrant polarity and increased invasiveness, 
demonstrating the 'gain of function' of mutant p53 in carcinogenesis. We 
hypothesize that p53R273H interacts with mutant p53-specific binding partners 
that control polarity, migration or invasion. Here we analyze the p53R273H 
interactome using stable isotope labelling by amino acids in cell culture and 
quantitative mass spectrometry, and identify at least 15 new potential mutant 
p53-specific binding partners. The interaction of p53R273H with one of 
them--nardilysin (NRD1)--promotes an invasive response to heparin 
binding-epidermal growth factor-like growth factor that is p53R273H-dependant 
but does not require Rab coupling protein or p63. Advanced proteomics has thus 
allowed the detection of a new mechanism of p53-driven invasion.

DOI: 10.1038/embor.2012.74
PMCID: PMC3388785
PMID: 22653443 [Indexed for MEDLINE]

Conflict of interest statement: The authors declare that they have no conflict 
of interest.

## Full Text

Abstract

The invasiveness of tumour cells depends on changes in cell shape, polarity and migration. Mutant p53 induces enhanced tumour metastasis in mice, and human cells overexpressing p53R273H have aberrant polarity and increased invasiveness, demonstrating the ‘gain of function’ of mutant p53 in carcinogenesis. We hypothesize that p53R273H interacts with mutant p53-specific binding partners that control polarity, migration or invasion. Here we analyze the p53R273H interactome using stable isotope labelling by amino acids in cell culture and quantitative mass spectrometry, and identify at least 15 new potential mutant p53-specific binding partners. The interaction of p53R273H with one of them—nardilysin (NRD1)—promotes an invasive response to heparin binding–epidermal growth factor-like growth factor that is p53R273H-dependant but does not require Rab coupling protein or p63. Advanced proteomics has thus allowed the detection of a new mechanism of p53-driven invasion.

Introduction

p53 is a tumour suppressor protein whose gene (TP53) is frequently (>50%) altered in human cancers through missense mutations, resulting in high expression levels of full-length mutant protein [ 1 ]. Of the missense mutations reported in the IARC TP53 database, the largest number of single base changes (>7.5% each) occur at amino acid positions 248 and 273 ((Database R15), [ 2 ]). Some of these mutations may be neomorphic or gain functions not normally exhibited by wild-type (wt)-p53 (reviewed in [ 3 , 4 ]), including invasion and metastasis as shown by studies of knock-in mice that express p53R270H (the equivalent of human p53R273H) [ 5–7 ]. These mice developed a different spectrum of tumours compared with p53 −/− mice including invasive and metastatic carcinomas [ 6 ]. Following ectopic expression of p53R273H and other p53 mutations in p53-null human cell lines, enhanced tumourigenicity in nude mice, colony formation in agar [ 8 ], increased proliferation [ 9 ] and augmented metastatic potentials [ 10 , 11 ] have been demonstrated.

Mechanisms underlying the oncogenic nature of mutant p53 include interaction or regulation of other transcription factors including p63, p73, NF-Y and VDR [ 12–14 , 15 ] or additional binding proteins such as TOPBP1, ANKRD11 and PIN1 [ 16–18 ]. Nearly all of the large number of interacting proteins reported in IARC p53 database [ 2 ] bind to wt-p53 and are therefore not good candidates for the neomorphic mutant hypothesis. To search for new mutant p53R273H-specific binding proteins, we undertook a comparative immunoprecipitation (IP) analysis using stable isotope labelling by amino acids in cell culture (SILAC) and mass spectrometry. SILAC has proved to be a powerful approach to distinguish adventitious from specific interactors, while retaining many weaker or transient binding partners [ 19–25 ]. Through this method we discovered several new mutant p53-specific binding proteins. Furthermore, we demonstrated the promotion of an invasive response to heparin binding–epidermal growth factor-like growth factor (HB–EGF) in mutant p53-expressing tumour cells that is dependent upon p53R273H and one of these mutant p53-specific interactors, Nardilysin (NRD1).

CONCLUSION

In conclusion, through the use of SILAC and comparative IPs we identified several new p53 binding proteins, including ones that specifically interacted with mutant p53 and not wt-p53. We confirmed a p53R273H-specific interaction with NRD1 and demonstrated that this cooperation was important in the promotion of cellular invasion driven by p53R273H towards HB–EGF. In addition to NRD1, we report at least 15 new potential mutant p53-specific binding partners that will be valuable in elucidating further the neomorphic functions of mutant p53 and strongly support the concept that distinct mutations in p53 may have selective new functions. Hence, inhibiting the expression or activity of mutant p53 proteins and modulating mutant p53-specific binding partners becomes an exciting and defined therapeutic approach to reduce tumour invasion and metastasis.

METHODS

Cell culture. The cells were cultured in either normal or SILAC media as described in supplementary information online. In SILAC experiments, mutant- and wt-p53-expressing cells were differentially labelled using SILAC to incorporate isotopic forms of lysine and arginine present in the DMEM media. For double-labelling experiments, the vector-only control cells were grown in media containing normal (or ‘light’ (L)) isotopes of lysine and arginine, while heavy (H) media was used for cells expressing p53. For the triple-label experiments, the vector was grown in light medium, the p53R273H-expressing cells in heavy and the wt-p53 cells in media containing an intermediate isotope (or ‘medium’(M)). To ensure that the cells were treated in a similar manner during the triple labelling, all were transiently transfected after stable isotope incorporation.

Immunoaffinity purification of protein complexes. Cells were lysed in modified RIPA buffer and the protein complexes were immunoprecipitated using either anti-FLAG or anti-p53 antibody as detailed in supplementary information online. Protein complexes were eluted either using 3 × FLAG peptides or SDS buffer. Either the affinity-purified protein complexes were subjected to liquid chromatography–mass spectrometry analysis as described below or they were used for western blot analysis.

Mass spectrometry and data analysis. Eluted protein complexes were separated by 1D SDS–PAGE, and digested with trypsin using published procedures [ 34 ]. Samples were analysed on an Orbitrap or Orbitrap XL (Thermo Fisher) coupled to a Proxeon Easy-nLC under the conditions described in supplementary information online. Mass spectrometry data were analysed by MaxQuant version 1.0.13.13 [ 26 ] using default parameter settings. Maximum false discovery rates were set to 0.01 for both protein and peptide.

Invasion assays. Matrigel assays were performed as described previously [ 11 , 35 ] and in the supplementary information online. Briefly, cells were seeded on the base of transwell chambers and invasion towards a gradient of 10% FBS combined with 10 ng/ml HB–EGF or 25 ng/ml EGF (unless stated otherwise), and was measured by confocal microscopy in serial sections. Quantification of Matrigel images was performed in ImageJ. siRNA transfections for NRD1, p53, p63 or RCP were carried out using standard and according to manufacturers’ procedures as described in supplementary information online. The silencing effect was assayed by western blots or reverse transcription–PCR or both

Mass spectrometry data sets were deposited in PeptideAtlas under PASS00051 data set identifier ( https://db.systemsbiology.net/sbeams/cgi/PeptideAtlas/PASS_View?identifier=PASS00051&browseArea=&path=/p53%20mutant-wt%20data%20sets ).
