---
pmid: '10515938'
title: Identification and characterization of the single-stranded DNA-binding protein
  of bacteriophage P1.
authors:
- Lehnherr H
- Bendtsen JD
- Preuss F
- Ilyina TV
journal: J Bacteriol
year: '1999'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC103783
doi: 10.1128/JB.181.20.6463-6468.1999
---

# Identification and characterization of the single-stranded DNA-binding protein of bacteriophage P1.
**Authors:** Lehnherr H, Bendtsen JD, Preuss F, Ilyina TV
**Journal:** J Bacteriol (1999)
**DOI:** [10.1128/JB.181.20.6463-6468.1999](https://doi.org/10.1128/JB.181.20.6463-6468.1999)
**PMC:** [PMC103783](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC103783/)

## Abstract

1. J Bacteriol. 1999 Oct;181(20):6463-8. doi: 10.1128/JB.181.20.6463-6468.1999.

Identification and characterization of the single-stranded DNA-binding protein 
of bacteriophage P1.

Lehnherr H(1), Bendtsen JD, Preuss F, Ilyina TV.

Author information:
(1)Institute of Molecular Biology, University of Southern Denmark, Main Campus 
Odense University, DK-5230 Odense M, Denmark. lehnherr@biobase.dk

The genome of bacteriophage P1 harbors a gene coding for a 162-amino-acid 
protein which shows 66% amino acid sequence identity to the Escherichia coli 
single-stranded DNA-binding protein (SSB). The expression of the P1 gene is 
tightly regulated by P1 immunity proteins. It is completely repressed during 
lysogenic growth and only weakly expressed during lytic growth, as assayed by an 
ssb-P1/lacZ fusion construct. When cloned on an intermediate-copy-number 
plasmid, the P1 gene is able to suppress the temperature-sensitive defect of an 
E. coli ssb mutant, indicating that the two proteins are functionally 
interchangeable. Many bacteriophages and conjugative plasmids do not rely on the 
SSB protein provided by their host organism but code for their own SSB proteins. 
However, the close relationship between SSB-P1 and the SSB protein of the P1 
host, E. coli, raises questions about the functional significance of the phage 
protein.

DOI: 10.1128/JB.181.20.6463-6468.1999
PMCID: PMC103783
PMID: 10515938 [Indexed for MEDLINE]

## Full Text

Abstract

The genome of bacteriophage P1 harbors a gene coding for a 162-amino-acid protein which shows 66% amino acid sequence identity to the Escherichia coli single-stranded DNA-binding protein (SSB). The expression of the P1 gene is tightly regulated by P1 immunity proteins. It is completely repressed during lysogenic growth and only weakly expressed during lytic growth, as assayed by an ssb -P1/ lacZ fusion construct. When cloned on an intermediate-copy-number plasmid, the P1 gene is able to suppress the temperature-sensitive defect of an E. coli ssb mutant, indicating that the two proteins are functionally interchangeable. Many bacteriophages and conjugative plasmids do not rely on the SSB protein provided by their host organism but code for their own SSB proteins. However, the close relationship between SSB-P1 and the SSB protein of the P1 host, E. coli , raises questions about the functional significance of the phage protein.

DISCUSSION

Several studies reporting mutational analyses of bacteriophage P1 failed to identify ssb -P1 ( 48 , 49 , 54 , 55 ). Our result showing that the SSB proteins of P1 and E. coli are functionally interchangeable might account for this failure, as any mutations in ssb -P1 might well go unnoticed when assayed in E. coli . Also, several attempts to clone ssb -P1 failed ( 47 ), perhaps due to the close proximity of ssb -P1 and lysA , the gene coding for the P1 lysozyme ( 46 ). It was reported that even weak expression of lysA is very deleterious to host cells ( 46 ). While lysA in its natural context is expressed from a P1-specific late promoter sequence ( 33 ) and thus is not expressed in the absence of the phage-specific activator protein gp10 ( 32 ), indirect low-level expression from a promoter in the cloning vector might be sufficient to kill cells containing a plasmid carrying the lysA gene. In the presence of P1 repressor proteins, the inadvertent expression of the lysozyme might be prevented by the C1-Lxc repressor complex binding to Op21 (Fig. 2 ). However, under such conditions the ssb -P1 gene will not be expressed and thus might again go unnoticed. Only after we managed to separate ssb -P1 from lysA was it possible to analyze the function of the former gene.

The P1 ssb gene is located in close proximity to the resident IS 1 element, and thus it can be speculated that P1 obtained the gene during a transposition event. However, the very strict and phage-specific regulation of ssb -P1 argues against a recent acquisition of the gene by the phage. The expression of SSB-P1 exclusively during lytic growth indicates a function of the protein related to vegetative DNA replication. Some bacteriophages, like T4, T7, and Ï†29, specify a complete set of replication proteins and are therefore independent of the host replication machinery ( 31 ). Unlike these phages, bacteriophage P1 does not specify a complete set of replication proteins, as its vegetative replication depends on DNA polymerase III (DnaE) and primase (DnaG) activities of the host ( 18 ). Nevertheless, P1 does specify several replication-associated proteins, like the lytic replication initiator protein RepL ( 7 ), a DnaB-like helicase ( 9 ), a Dam methyltransferase ( 13 ), and a homologue of the theta subunit of DNA polymerase III ( 34 ), in addition to SSB-P1. These proteins, with the exception of RepL, are homologous to the respective E. coli proteins and thus appear redundant. Indeed, it has been shown that the ssb genes of several conjugative plasmids are dispensable ( 11 , 16 , 23 ). However, the strong conservation of key residues important for SSB function indicates that the maintenance of ssb has to have some selective advantage for the phage or the plasmids. It is conceivable that subtle differences between the proteins of the episome and the host might allow the former to exert specific control over key regulatory steps during vegetative replication or conjugation. Alternatively, it cannot be excluded that P1 or the analyzed conjugative plasmids might encounter host bacteria which differ considerably from E. coli , and in such a host the SSB proteins expressed by the episomal genetic elements might well turn out to be essential.

That highly homologous ssb genes are encoded by both gram-negative and gram-positive bacteria, as well as by some of their plasmids and bacteriophages, raises some evolutionary questions about the possible origin of the gene and the mechanisms by which it is disseminated. A careful phylogenetic analysis might provide some answers to such questions.
