---
pmid: '27706847'
title: Structurally diverse dehydroshikimate dehydratase variants participate in microbial
  quinate catabolism.
authors:
- Peek J
- Roman J
- Moran GR
- Christendat D
journal: Mol Microbiol
year: '2017'
full_text_available: false
doi: 10.1111/mmi.13542
---

# Structurally diverse dehydroshikimate dehydratase variants participate in microbial quinate catabolism.
**Authors:** Peek J, Roman J, Moran GR, Christendat D
**Journal:** Mol Microbiol (2017)
**DOI:** [10.1111/mmi.13542](https://doi.org/10.1111/mmi.13542)

## Abstract

1. Mol Microbiol. 2017 Jan;103(1):39-54. doi: 10.1111/mmi.13542. Epub 2016 Nov 3.

Structurally diverse dehydroshikimate dehydratase variants participate in 
microbial quinate catabolism.

Peek J(1), Roman J(2), Moran GR(2), Christendat D(1)(3).

Author information:
(1)Department of Cell and Systems Biology, University of Toronto, 25 Willcocks 
Street, Toronto, Ontario, Canada, M5S 3B2.
(2)Department of Chemistry and Biochemistry, University of Wisconsin-Milwaukee, 
3210 North Cramer Street, Milwaukee, WI, 53211-3209, USA.
(3)Centre for the Analysis of Genome Evolution and Function, University of 
Toronto, Toronto, Ontario, Canada, M5S 3B2.

Quinate and shikimate can be degraded by a number of microbes. Dehydroshikimate 
dehydratases (DSDs) play a central role in this process, catalyzing the 
conversion of 3-dehydroshikimate to protocatechuate, a common intermediate of 
aromatic degradation pathways. DSDs have applications in metabolic engineering 
for the production of valuable protocatechuate-derived molecules. Although a 
number of Gram-negative bacteria are known to catabolize quinate and shikimate, 
only limited information exists on the quinate/shikimate catabolic enzymes found 
in these organisms. Here, we have functionally and structurally characterized a 
putative DSD designated QuiC1, which is present in some pseudomonads. The QuiC1 
protein is not related by sequence with previously identified DSDs from the 
Gram-negative genus, Acinetobacter, but instead shows limited sequence identity 
in its N-terminal half with fungal DSDs. Analysis of a Pseudomonas aeruginosa 
quiC1 gene knock-out demonstrates that it is important for growth on either 
quinate or shikimate. The structure of a QuiC1 enzyme from P. putida reveals 
that the protein is a fusion of two distinct modules: an N-terminal sugar 
phosphate isomerase-like domain associated with DSD activity and a novel 
C-terminal hydroxyphenylpyruvate dioxygenase-like domain. The results of this 
study highlight the considerable diversity of enzymes that participate in 
quinate/shikimate catabolism in different microbes.

Â© 2016 John Wiley & Sons Ltd.

DOI: 10.1111/mmi.13542
PMID: 27706847 [Indexed for MEDLINE]
