---
pmid: '22446626'
title: DBIRD complex integrates alternative mRNA splicing with RNA polymerase II transcript
  elongation.
authors:
- Close P
- East P
- Dirac-Svejstrup AB
- Hartmann H
- Heron M
- Maslen S
- Chariot A
- Söding J
- Skehel M
- Svejstrup JQ
journal: Nature
year: '2012'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC3378035
doi: 10.1038/nature10925
---

# DBIRD complex integrates alternative mRNA splicing with RNA polymerase II transcript elongation.
**Authors:** Close P, East P, Dirac-Svejstrup AB, Hartmann H, Heron M, Maslen S, Chariot A, Söding J, Skehel M, Svejstrup JQ
**Journal:** Nature (2012)
**DOI:** [10.1038/nature10925](https://doi.org/10.1038/nature10925)
**PMC:** [PMC3378035](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3378035/)

## Abstract

1. Nature. 2012 Mar 25;484(7394):386-9. doi: 10.1038/nature10925.

DBIRD complex integrates alternative mRNA splicing with RNA polymerase II 
transcript elongation.

Close P(1), East P, Dirac-Svejstrup AB, Hartmann H, Heron M, Maslen S, Chariot 
A, Söding J, Skehel M, Svejstrup JQ.

Author information:
(1)Mechanisms of Transcription Laboratory, Cancer Research UK London Research 
Institute, Clare Hall Laboratories, South Mimms EN6 3LD, UK.

Alternative messenger RNA splicing is the main reason that vast mammalian 
proteomic complexity can be achieved with a limited number of genes. Splicing is 
physically and functionally coupled to transcription, and is greatly affected by 
the rate of transcript elongation. As the nascent pre-mRNA emerges from 
transcribing RNA polymerase II (RNAPII), it is assembled into a messenger 
ribonucleoprotein (mRNP) particle; this is the functional form of the nascent 
pre-mRNA and determines the fate of the mature transcript. However, factors that 
connect the transcribing polymerase with the mRNP particle and help to integrate 
transcript elongation with mRNA splicing remain unclear. Here we characterize 
the human interactome of chromatin-associated mRNP particles. This led us to 
identify deleted in breast cancer 1 (DBC1) and ZNF326 (which we call ZNF-protein 
interacting with nuclear mRNPs and DBC1 (ZIRD)) as subunits of a novel protein 
complex--named DBIRD--that binds directly to RNAPII. DBIRD regulates alternative 
splicing of a large set of exons embedded in (A + T)-rich DNA, and is present at 
the affected exons. RNA-interference-mediated DBIRD depletion results in 
region-specific decreases in transcript elongation, particularly across areas 
encompassing affected exons. Together, these data indicate that the DBIRD 
complex acts at the interface between mRNP particles and RNAPII, integrating 
transcript elongation with the regulation of alternative splicing.

DOI: 10.1038/nature10925
PMCID: PMC3378035
PMID: 22446626 [Indexed for MEDLINE]

## Full Text

The composition of mRNP particles has been the subject of a number of studies, using a variety of approaches (see, for example, ref5 and references therein). There are likely to be different types of mRNP particles with distinct compositions and interaction partners. We sought to specifically purify native mRNP particles and interacting proteins from the chromatin in which they are generated and active in co-transcriptional processes. As a starting point, we generated HEK293 cells expressing near-normal levels of Flag-tagged hnRNP A1 (A1), an abundant hnRNP protein in human cells6. A1 shuttles between the nucleus and the cytoplasm7, but at steady state it is mainly nuclear and concentrated in chromatin (Fig. 1a), from where it can be released by RNAse A treatment (Fig. 1b, compare lanes 2 and 4). We used DNAse I digestion and mild sonication to release mRNP particles from chromatin for purification. RNAse inhibitors were present during the whole process, outlined in Fig. 1c. mRNP particles isolated by this approach are predominantly of nuclear (chromatin) origin (Suppl. Fig. S1). Native mRNP particles and their interacting partners were purified from chromatin isolated from ~108 nuclei (Fig. 1d). Only two major bands (namely the added, proteinaceous RNAse inhibitors) were detected upon purification from control cells (left panel), while numerous proteins were detected in A1-flag elutions (right panel). These represent a heterogeneous mixture of core mRNP particle subunits and proteins interacting with such particles. Individual protein bands were excised and identified by mass spectrometric analysis (subset indicated in Fig. 1d; see also list in Suppl. Fig S2a). Most of the known ‘core’ mRNP proteins, such as the hnRNP proteins were present in the purified fraction, confirming the biological relevance of this approach. Many other pre-mRNA processing proteins were also identified, including splicing factors, ATP-dependent RNA helicases, and a substantial number of mRNA 3′-end processing and termination factors. Co-immunoprecipitation (co-IP) experiments confirmed the RNA-dependent interaction of some of these proteins with A1-Flag (Suppl. Fig. S2b).

We next focused on two proteins that had not previously been connected to mRNP particles or mRNA processing. One of these, DBC1, is otherwise best known for its association with and regulation of the sirtuin-like deacetylase SIRT18,9. We also investigated the uncharacterized zinc finger-containing protein ZNF326. Stable cell lines expressing near-normal levels of Flag-tagged versions of these proteins were established, and co-IP experiments confirmed that both DBC1 and ZNF326 interact with mRNP particles in an RNA-dependent manner (Suppl. Fig. S3a-f). Furthermore, we discovered that ZNF326 and DBC1 associate directly, in an RNA-independent manner (Fig. 2a and e). We therefore renamed ZN326 as ZIRD (ZNF-protein Interacting with nuclear RNPs and DBC1).

We previously identified DBC1 as an RNAPII-interacting protein in another proteomic screen10, making it a particularly interesting candidate. Co-IP experiments confirmed that RNAPII associates with DBC1-Flag, in an RNA-independent manner (Fig. 2b). ZIRD was detected in RNAPII (RPB3-Flag) purifications as well, and this interaction was also RNA-independent (Fig. 2c). In further support of a ZIRD-RNAPII interaction, ZIRD-Flag also brought down RNAPII (Fig. 2d). In contrast, we failed to detect an interaction between A1 and RNAPII under the same conditions (Fig. 2c, middle panel, and data not shown), although co-IP experiments after formaldehyde cross-linking indicated that, as expected, the proteins are in close proximity in vivo (Suppl. Fig. S4). Together, these results indicate that DBC1 and ZIRD are not part of the core mRNP particle, but that they might work at the interface between the mRNP particle and RNAPII.

Others reported that DBC1 interacts with SIRT18,9. While we confirmed that DBC1 co-precipitates SIRT1, endogenous ZIRD and ZIRD-Flag did not (Fig. 2e, and data not shown). SIRT1 is also absent from A1-containing mRNP particles (Suppl. Fig. S3g). This indicates that ZIRD and DBC1 form a complex that lacks SIRT1. To further characterize ZIRD-DBC1 interaction, ZIRD-Flag was purified. Size exclusion chromatography of highly purified material showed that ZIRD-Flag and DBC1 are part of a salt-stable ~800 kDa complex (Fig. 2f, upper two panels), which also co-purified on MonoQ (data not shown). As expected, SIRT1 is not part of this protein complex (Fig. 2g, and data not shown). We named the complex DBIRD (DBC1/ZIRD complex).

DBC1 and ZIRD interact with RNAPII in crude extracts (Fig. 2b-d). To investigate if this interaction is direct, DBIRD complex was characterized by gel-filtration after mixing with an excess of RNAPII. In the absence of RNAPII, the DBIRD complex peaked in fractions 13-15 (Fig. 2f, upper two panels). However, when mixed with RNAPII (Fig. 2f, lower three panels), DBIRD complex elution shifted to earlier eluting fractions, peaking in fraction 10 with a sub-fraction of RNAPII, while polymerase alone peaked in fractions 17-19 (~500 kDa), as expected. DBIRD complex thus appears to form a bridging complex, which interacts with both mRNP particles and RNAPII. Interestingly, DBIRD also interacted with mRNP particles lacking hnRNP A1 (Fig. 2h), pointing to a general bridging role.

To examine the role of the DBIRD complex in transcription-associated processes in vivo, we analysed the transcriptome of cells that had been depleted for DBC1 or ZIRD by RNAi (Suppl. Fig. 5a). Total mRNA was hybridised to GeneChip HUMAN EXON 1.0 ST arrays, on which the abundance of individual exons can be analyzed independently. In the absence of ZIRD, a >1.5 fold increase in exon inclusion was observed in more than 2800 situations, whereas exon exclusion was observed in only 390 cases (Suppl. Table T1a). The absence of DBC1 led to increased inclusion of an exon in 796 cases (Suppl. Table T1b), and, strikingly, the majority of these events were also on the list of ZIRD-dependent exon inclusions (567 of 796 = 71%; p-value for shared exons = 6.705e-261; Suppl. Table T1c), strongly supporting the close functional relationship between the two factors, and providing a high degree of confidence in the genome-wide alternative splicing data-sets. The effect was at the level of alternative splicing, as depletion of ZIRD or DBC1 only affected the expression of a very small number of genes (Suppl. Fig. S6).

A full list of inclusion events observed in both DBC1 and ZIRD-depleted cells is in Suppl. Table T1c. Sample results were confirmed by quantitative RT-PCR (Suppl. Fig. S7). To investigate whether DBIRD was present at affected exons, we performed RNA immunoprecipitation (RIP) experiments11. DBC1 and ZIRD bound the relevant exon in mRNAs from seven tested genes, while other regions (or control tRNA) were either not detected, or detected to a much lower extent (Fig. 3a and b; Suppl. Fig. S8). Interestingly, some exons of the β-actin gene (whose splicing was unaffected by DBIRD depletion) had significant levels of DBIRD complex (Suppl. Fig. S8), indicating that the interaction of DBIRD with mRNA is not invariably correlated with DBIRD-dependent splicing changes.

To investigate the mechanism underlying exon inclusion, we first searched for sequence motifs in the DNA encompassing the included exons, but failed to uncover other motifs than those known to typify splice junctions. We then looked for nucleotide patterns that might be overrepresented in the sequences surrounding the included exons by counting how often each of the 1024 possible 5-mer oligonucleotides occurred. Intriguingly, A/T-rich 5-mers were significantly enriched around included exons (Fig. 3c). The frequencies of the four nucleotides in the regions around the splice sites were also analyzed. A and T were strongly overrepresented around the splice sites of DBIRD-affected exons, as well as across the exons themselves. (Fig. 3d). The observed difference in A/T content is sufficient to explain the over-representation of A/T-rich 5-mers (Suppl. Fig. S9).

The A/T-rich DNA surrounding the affected exons might influence fundamental aspects of transcription. Indeed, A- and T-tracts are difficult for RNAPII to transcribe, constituting very efficient elongation pause sites in vitro12,13. To investigate the effect of DBIRD on transcript elongation, we performed RNAPII chromatin-immunoprecipitation (ChIP) analysis after DBIRD knockdown. As control, we also knocked down SIRT1 (Suppl. Fig. 5b). Remarkably, although overall transcription of RAD50 and SLC36A4 is not affected (see Suppl. Fig. S7), depletion of DBC1 or ZIRD (but not SIRT1) dramatically affected RNAPII transcription, distinctively in regions encompassing affected exons (Fig. 4; Suppl. Fig. S11). Quantification of newly produced mRNA by Bromo-UTP incorporation/immunoprecipitation supported the idea that elongation rates were decreased in these regions (Suppl. Fig. S10). DBIRD depletion also affected RNAPII density at other genes whose splicing was exon-specifically affected, while little or no change in RNAPII density was observed at the unaffected β-actin control gene, even at exons that had an elevated DBIRD level (Suppl. Fig. S11-12; Compare to Suppl. Fig. S8).

Our data support the idea that the DBIRD complex represents a new type of factor, which functions at the interface between ‘core’ mRNP particles and RNAPII, affecting local transcript elongation rates and alternative splicing at a subset of A/T-rich exon-intron junctions (Graphic model in Suppl. Fig. S13). Interestingly, several studies have shown that the rate of RNAPII elongation affects the efficiency of splicing, with slow elongation favouring exon inclusion1,3. One possible explanation for our data is thus that DBIRD complex acts as an elongation factor, which facilitates transcript elongation across A/T-rich regions, and thereby affects alternative splicing of exons in these regions. It has also been suggested that exons in the nascent pre-mRNA become tethered to the elongating transcription complex14,15. Given that DBIRD binds both mRNPs and RNAPII, it might affect such tethering as well, and thereby splicing.

Interestingly, DBC1 has been implicated in tumourigenesis as a potential tumour suppressor, regulating apoptosis and cell survival16. Whether DBC1’s role in the DBIRD complex and alternative splicing impacts on tumourigenesis is an interesting possibility, especially in light of the recent finding that genes encoding components of the splicing machinery are often mutated in myelodysplastic syndromes and related disorders17. ZIRD has not previously been characterized in human cells, but its mouse homologue, ZAN75, is highly expressed in neuronal tissues18, suggesting that regulation of DBIRD complex might contribute to tissue-specific splicing. Other proteins with homology to ZIRD and DBC1 exist in the human genome, raising the intriguing possibility that other DBIRD-like complexes are specific for other sets of genes or exons, or are involved in other transcription-related nuclear events.

METHODS SUMMARYORFs encoding A1, DBC1 and ZIRD were cloned into pIRESpuro (Clontech) with a C-terminal Flag tag. HEK293 cells were grown in DMEM containing 10% FBS in 5% CO2 at 37°C. For proteomic analysis, nuclei were isolated from A1-Flag cells. These were sonicated, DNAse I treated, and the sample cleared by centrifugation and the supernatant subjected to M2 agarose (Sigma) chromatography. Proteins were eluted with 3xFLAG peptide, and mass spectrometry performed as described10. DBIRD was purified by M2 agarose chromatography from nuclease-treated nuclear extract from cells expressing ZIRD-Flag. DBIRD was analyzed by MonoQ, or size exclusion chromatography with or without an excess of RNAPII. Stealth siRNAs were double transfected in HEK293 cells using lipofectamine 2000 (Invitrogen). For microarray analysis, RNA was hybridized on Human Exon 1.0 ST arrays (Affymetrix) using standard techniques (bioinformatics analysis described in Full Methods). For assessment of exon abundance and transcript expression, quantitative RT-PCR was performed using primers against affected and unaffected exons. Primer details are available on request. RNA immunoprecipitation and ChIP assays were performed as described11,19.

METHODS SUMMARY

ORFs encoding A1, DBC1 and ZIRD were cloned into pIRESpuro (Clontech) with a C-terminal Flag tag. HEK293 cells were grown in DMEM containing 10% FBS in 5% CO2 at 37°C. For proteomic analysis, nuclei were isolated from A1-Flag cells. These were sonicated, DNAse I treated, and the sample cleared by centrifugation and the supernatant subjected to M2 agarose (Sigma) chromatography. Proteins were eluted with 3xFLAG peptide, and mass spectrometry performed as described10. DBIRD was purified by M2 agarose chromatography from nuclease-treated nuclear extract from cells expressing ZIRD-Flag. DBIRD was analyzed by MonoQ, or size exclusion chromatography with or without an excess of RNAPII. Stealth siRNAs were double transfected in HEK293 cells using lipofectamine 2000 (Invitrogen). For microarray analysis, RNA was hybridized on Human Exon 1.0 ST arrays (Affymetrix) using standard techniques (bioinformatics analysis described in Full Methods). For assessment of exon abundance and transcript expression, quantitative RT-PCR was performed using primers against affected and unaffected exons. Primer details are available on request. RNA immunoprecipitation and ChIP assays were performed as described11,19.

Supplementary Material12

Supplementary Material
