---
pmid: '18809673'
title: BRD7, a novel PBAF-specific SWI/SNF subunit, is required for target gene activation
  and repression in embryonic stem cells.
authors:
- Kaeser MD
- Aslanian A
- Dong MQ
- Yates JR 3rd
- Emerson BM
journal: J Biol Chem
year: '2008'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC2583284
doi: 10.1074/jbc.M806061200
---

# BRD7, a novel PBAF-specific SWI/SNF subunit, is required for target gene activation and repression in embryonic stem cells.
**Authors:** Kaeser MD, Aslanian A, Dong MQ, Yates JR 3rd, Emerson BM
**Journal:** J Biol Chem (2008)
**DOI:** [10.1074/jbc.M806061200](https://doi.org/10.1074/jbc.M806061200)
**PMC:** [PMC2583284](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2583284/)

## Abstract

1. J Biol Chem. 2008 Nov 21;283(47):32254-63. doi: 10.1074/jbc.M806061200. Epub 
2008 Sep 22.

BRD7, a novel PBAF-specific SWI/SNF subunit, is required for target gene 
activation and repression in embryonic stem cells.

Kaeser MD(1), Aslanian A, Dong MQ, Yates JR 3rd, Emerson BM.

Author information:
(1)Regulatory Biology, The Salk Institute for Biological Studies, La Jolla, 
California 92037, USA.

The composition of chromatin-remodeling complexes dictates how these enzymes 
control transcriptional programs and cellular identity. In the present study we 
investigated the composition of SWI/SNF complexes in embryonic stem cells 
(ESCs). In contrast to differentiated cells, ESCs have a biased incorporation of 
certain paralogous SWI/SNF subunits with low levels of BRM, BAF170, and ARID1B. 
Upon differentiation, the expression of these subunits increases, resulting in a 
higher diversity of compositionally distinct SWI/SNF enzymes. We also identified 
BRD7 as a novel component of the Polybromo-associated BRG1-associated factor 
(PBAF) complex in both ESCs and differentiated cells. Using short hairpin 
RNA-mediated depletion of BRG1, we showed that SWI/SNF can function as both a 
repressor and an activator in pluripotent cells, regulating expression of 
developmental modifiers and signaling components such as Nodal, ADAMTS1, BMI-1, 
CRABP1, and thyroid releasing hormone. Knockdown studies of PBAF-specific BRD7 
and of a signature subunit within the BAF complex, ARID1A, showed that these two 
subcomplexes affect SWI/SNF target genes differentially, in some cases even 
antagonistically. This may be due to their different biochemical properties. 
Finally we examined the role of SWI/SNF in regulating its target genes during 
differentiation. We found that SWI/SNF affects recruitment of components of the 
preinitiation complex in a promoter-specific manner to modulate transcription 
positively or negatively. Taken together, our results provide insight into the 
function of compositionally diverse SWI/SNF enzymes that underlie their inherent 
gene-specific mode of action.

DOI: 10.1074/jbc.M806061200
PMCID: PMC2583284
PMID: 18809673 [Indexed for MEDLINE]

## Full Text

Abstract

The composition of chromatin-remodeling complexes dictates how these
 enzymes control transcriptional programs and cellular identity. In the present
 study we investigated the composition of SWI/SNF complexes in embryonic stem
 cells (ESCs). In contrast to differentiated cells, ESCs have a biased
 incorporation of certain paralogous SWI/SNF subunits with low levels of BRM,
 BAF170, and ARID1B. Upon differentiation, the expression of these subunits
 increases, resulting in a higher diversity of compositionally distinct SWI/SNF
 enzymes. We also identified BRD7 as a novel component of the
 Polybromo-associated BRG1-associated factor (PBAF) complex in both ESCs and
 differentiated cells. Using short hairpin RNA-mediated depletion of BRG1, we
 showed that SWI/SNF can function as both a repressor and an activator in
 pluripotent cells, regulating expression of developmental modifiers and
 signaling components such as Nodal, ADAMTS1, BMI-1, CRABP1, and thyroid
 releasing hormone. Knockdown studies of PBAF-specific BRD7 and of a signature
 subunit within the BAF complex, ARID1A, showed that these two subcomplexes
 affect SWI/SNF target genes differentially, in some cases even
 antagonistically. This may be due to their different biochemical properties.
 Finally we examined the role of SWI/SNF in regulating its target genes during
 differentiation. We found that SWI/SNF affects recruitment of components of
 the preinitiation complex in a promoter-specific manner to modulate
 transcription positively or negatively. Taken together, our results provide
 insight into the function of compositionally diverse SWI/SNF enzymes that
 underlie their inherent gene-specific mode of action.

RESULTS

Composition of SWI/SNF in Pluripotent ES Cells —Initially we
 investigated which forms of the SWI/SNF multisubunit complex exist in ESCs. We
 used a strategy in which epitope-tagged subunits are virally integrated into
 the ESC genome. Affinity purification is then accomplished in a simple
 one-step procedure, resulting in native multisubunit protein preparations of
 high yield and purity ( 33 ). We
 expressed several well characterized subunits to increase our ability to
 purify most of the compositionally distinct forms of SWI/SNF that may be
 present in ESCs. BAF47, BAF57, BAF155, BAF170, and BRG1 are all core subunits
 that associate with both BAF and PBAF complexes
 ( Fig. 1 A ). To purify
 SWI/SNF from substantial amounts of homogeneously undifferentiated ESCs, we
 created the cell lines R201 and R218. Both were derived from the R1 ESC line
 by infection with lentiviruses expressing Nanog-HA and Nanog-MYC,
 respectively. Both cell lines exhibit morphological features similar to
 pluripotent ES cells and differentiate upon leukemia-inhibitory factor
 withdrawal or RA treatment (supplemental Fig. 1). We also compared gene
 expression in parental R1 and R218 cells in response to RA and found them to
 be very similar albeit with a slower Oct4 mRNA decrease in R218
 cells. Additionally the R218 line was capable of differentiating into
 spontaneously beating cardiomyocytes with efficiency similar to that reported
 previously ( 34 ) (data not
 shown).

SWI/SNF in Embryonic Stem Cells Is Composed of a Limited Subset of
 Components —To prepare purified SWI/SNF complexes, R201 cells were
 transduced with lentiviruses expressing a C-terminal FLAG-tagged cDNA of Baf47, Baf57, Baf155, Baf170 , or Brg1 . Except for BAF170,
 ectopic expression of SWI/SNF subunits did not lead to an overall increase in
 protein levels ( Fig.
 1 B ). To examine SWI/SNF composition in an unbiased
 manner, we purified sufficient material to analyze by silver staining and
 MudPIT. As shown in Fig.
 1 C , anti-FLAG eluates from ESCs expressing individual
 FLAG-tagged BAFs 47, 57, 155, or 170 or BRG1 contained a similar set of
 proteins, which resembles the subunit pattern observed in the initial SWI/SNF
 purifications ( 35 ). Using
 Western blotting, mass spectroscopy on the individual bands, and differences
 in migration upon expression of the FLAG-tagged subunits (supplemental Fig.
 2), we co-localized the known SWI/SNF subunits ARID1A, ARID2, BRG1, BAF170,
 BAF155, BAF60, BAF57, and BAF53 with the indicated bands.

The Majority of SWI/SNF Complexes Contain the Core Subunits BAFs 47,
 57, and 155 and BRG1 —We then examined by Western blotting whether
 any subunits were preferentially assembled into particular complexes by unique
 associations with other BAF proteins. We observed that the bait for the
 purification was in general slightly overrepresented
 ( Fig. 1 D ). However,
 with the exception of the BAF170 sample, the preparations showed a remarkably
 similar ratio between the other examined core components. This is consistent
 with the notion that the majority of purified complexes contain all four
 subunits: BAFs 47, 57, and 155 and BRG1. BAF170 was below detection in all
 samples except where it was ectopically expressed, suggesting that it is rare
 in ESCs and that the lentiviral expression increased its abundance above
 normal, endogenous levels. As expected, its ectopic expression and
 incorporation in SWI/SNF complexes resulted in the replacement of one or both
 molecules of its paralogue BAF155, explaining why BAF155 levels are lower in
 complexes purified through BAF170. Because of the overexpression of BAF170, we
 did not analyze this sample any further.

Several Paralogous Subunits Are Overrepresented by Specific
 Forms —A comparison of the samples by MudPIT revealed unique
 peptides of 14 previously documented SWI/SNF components in the four
 subunit-specific ESC and HeLa purifications
 ( Table 1 ). The control sample
 only contained a total of five peptides from two different subunits (see
 “Experimental Procedures”). Because certain groups of paralogous
 subunits (BRG1/BRM, BAF155/BAF170, and ARID1A/ARID1B) share substantial
 sequence homology (>50% in each group), we only included peptides in our
 analysis that could be unambiguously attributed to one specific polypeptide.
 Because BRM and BRG1 are mutually exclusive subunits
 ( 21 ), the single unique BRM
 peptide in our BRG1 purification reflects the expected false positive
 identification rate of 2% or lower. We expected roughly similar amounts of
 unambiguous peptides when comparing the paralogous subunits within their
 groups as observed in our analysis of SWI/SNF from HeLa cells. However, we
 found that in ESCs specific paralogous subunits were preferentially
 incorporated. In the case of BRG1/BRM, our results suggest that the
 underrepresented protein BRM might exist in negligible quantities in ESCs in
 agreement with its low expression during early development
 ( 36 ). This is also the case
 for the postmitotic neuron-specific BAF53B of which we could not find any
 unambiguous peptides ( 37 ).
 Also in contrast to HeLa cells, BAF170 and ARID1B had considerably fewer
 unambiguous peptides in ESCs than their counterparts BAF155 and ARID1A,
 implying that these subunits are less abundant. ARID1B-containing BAF enzymes
 were shown to interact with transcriptional activators as opposed to complexes
 with ARID1A that associate with transcriptional repressors
 ( 25 ), suggesting that in ESCs
 SWI/SNF might be compositionally better suited for a repressive role.

Differentiation Increases Incorporation of Previously Under-represented
 SWI/SNF Components —Because we observed a surprisingly biased usage
 of paralogous SWI/SNF subunits in pluripotent cells, we examined the changes
 in SWI/SNF composition during ESC differentiation. We used RA for 2 and 6 days
 to differentiate pluripotent stem cells into restricted descendants. SWI/SNF
 complexes were purified from differentiating cells using a tagged version of
 BAF47 as described previously
 ( 33 ) and compared with a
 preparation from undifferentiated cells
 ( Fig. 2 A ). To identify
 subunit-specific changes, we performed Western blot analyses
 ( Fig. 2 B ). Each
 SWI/SNF preparation was standardized to give approximately the same levels of
 the core components BAF47 and BAF57. Throughout the course of differentiation,
 we observed considerable induction of some subunits, such as BAF53, BAF170,
 and BRM. Conversely ARID2 and ARID1A decreased by day 6 of differentiation.
 BAF155 exhibited a decrease in the immunoreactive 155-kDa band but an increase
 in the signal at 120 kDa. Mass spectroscopy of this silver-stainable band at
 120 kDa identified 18 individual peptides of BAF155, supporting the notion of
 a splice variant, specific cleavage, or degradation product of BAF155, which
 may represent the previously observed BAF110
 ( 21 ).

RA-mediated Differentiation Increases ARID1B at the Expense of
 ARID1A —Several distinct mechanisms could be responsible for the
 changes in SWI/SNF composition, such as differences in subunit transcription,
 protein stability, or variations in subunit incorporation rates. It was shown
 previously that differentiation leads to changes in cellular protein levels
 that are very similar to the variations we observed in SWI/SNF composition
 ( 17 ). We therefore tested
 whether differences in subunit transcription might explain the changes in
 SWI/SNF composition by measuring their individual RNA expression levels
 ( Fig. 2 C ). We observed
 little change, if any, in expression of Baf155 and Brg1 ,
 whereas considerable increases in Brm and Baf170 transcripts
 were apparent. This corroborated our measurements of protein levels and the
 indications obtained from MudPIT analysis of these samples
 ( Table 2 ). We also detected a
 reduction in Arid1a message, confirming the decrease observed in
 Western blotting by 6 days of differentiation. Interestingly Arid1b exhibits an opposing phenotype revealed by the increase in both transcript and
 unique peptides in mass spectroscopy. Unfortunately we could not confirm this
 by Western blotting because we lacked antibodies that recognized ARID1B under
 these conditions. The compositional changes we observed by different
 approaches (and suggested by MudPIT for ARID1B) are summarized in Fig. 2 D .

B-cell Leukemia Protein 7 (BCL7) Family Members Are Novel SWI/SNF
 Subunits —Our MudPIT analyses of complexes from ESCs and HeLa cells
 consistently identified unique peptides from a protein family and another
 single protein that were not previously recognized as SWI/SNF subunits
 ( Fig. 3 A ). These
 peptides are found in all individual SWI/SNF preparations independent of the
 subunit used for purification. The family represented by the BCL7 A/B/C
 proteins is of unknown function except for its involvement in chromosomal
 translocations in Burkitt lymphoma
 ( 38 ). We also found peptides
 from BRD7, which was shown to interact with IRF-2
 ( 39 ) and acetylated histones
 ( 40 ). One additional group,
 represented by the proteins BAF45A/B/D, has been well characterized by
 Crabtree and co-workers ( 41 ).
 To confirm the incorporation of these proteins into SWI/SNF complexes in ESCs,
 we affinity-purified lentiviral expressed 2FLAG-tagged versions of these
 proteins and compared the interacting proteins with SWI/SNF from a 2FLAG-BAF57
 purification ( Fig.
 3 B ). Both BAF45D and BCL7C preparations showed a
 remarkably similar interacting protein pattern compared with BAF57 as judged
 by Coomassie and silver staining. The only obvious differences between
 individual purifications correspond to shifted bands in the molecular weight
 range where the differentially tagged proteins are expected. We conclude that
 both BAF45D and BCL7C co-purify with SWI/SNF in a stoichiometry similar to
 that of BAF57, establishing these proteins as bona fide subunits of
 SWI/SNF.

BRD7 Is a Novel PBAF Subunit and Defines Complexes with Distinct
 Biochemical Properties —Compared with BAF57, BRD7 exhibits a
 similar, albeit slightly different protein interaction pattern
 ( Fig. 3 B ). The absence
 of a 250-kDa band and the comparatively overrepresented 200-kDa band suggested
 that the BRD7-containing complexes lacked ARID1A/B proteins but had close to
 stoichiometric amounts of ARID2. This suggests that BRD7 is a potential
 PBAF-specific subunit. We tested this hypothesis by Western blot analysis
 ( Fig. 3 C ). Both BRG1
 and ARID2 co-precipitated with tagged versions of BAF57, BRD7, and BAF45D,
 underscoring the incorporation of all three baits into SWI/SNF complexes. In
 contrast, ARID1A can only be detected in association with BAF57 and BAF45D,
 confirming the selective incorporation of BRD7 into PBAF complexes.

Different SWI/SNF subunits contribute specific functionalities to the
 enzymatic complex. We tested whether BRD7-containing PBAF complexes possessed
 the same biochemical properties as the mixture of SWI/SNF complexes containing
 both BAF and PBAF. To this end, we assayed both 2FLAG-BRD7- and
 2FLAG-BAF57-purified complexes in an ATP hydrolysis assay. Both preparations
 contained similar amounts of the catalytic subunit BRG1, the only ATPase
 present in ESCs ( Fig.
 3 D , inset ). As expected, the total mixture of
 SWI/SNF exhibited a basal ATPase activity that was stimulated by the addition
 of DNA ( Fig. 3 D ,
 light and dark gray circles , respectively). Surprisingly
 BRD7-purified PBAF complexes had a substantially higher basal activity that
 was comparable to the DNA-stimulated activity of total SWI/SNF. This activity,
 however, was not further stimulated by the addition of DNA. This indicates
 that compositionally distinct subcomplexes have different enzymatic
 properties, which may impart specialized functions to BAF and PBAF and
 facilitate their ability to differentially regulate individual target
 genes.

SWI/SNF Functions as Either an Activator or a Repressor —To
 define the transcriptional functions of SWI/SNF in pluripotent ESCs and
 identify its target genes, we performed a BRG1 depletion experiment coupled to
 a transcriptome survey. R1 ESCs were infected in duplicate with shRNA
 targeting BRG1 or a non-expressed control, and RNA was harvested. Microarray
 analysis revealed changes of >2-fold in a total of 104 individual genes of
 which 88 transcripts were induced by the RNAi and expression of 16 genes was
 reduced. This suggests that SWI/SNF can act both as an activator and a
 repressor in pluripotent cells (supplemental Table 1). Interestingly when
 analyzing the gene ontology annotation of all target genes, we found that in
 the SWI/SNF-repressed group, angiogenic modulators were overrepresented
 ( p value below 10 –4 as annotated below).

Antagonism between BAF and PBAF Subcomplexes —Next we asked
 whether regulation of these BRG1-responsive genes relied on BAF, PBAF, or both
 subcomplexes. To this end, we infected R218 cells with shRNA targeting either
 the BAF signature subunit ARID1A, the PBAF-specific subunit BRD7, or the only
 ATPase present in ESCs, BRG1. All shRNAs reduced the protein level of their
 target significantly ( Fig.
 4 A ) with a decrease in target RNA to 60 (BRG1), 80
 (ARID1A), and 80% (BRD7) (data not shown).

To identify complex-specific regulation, we tested a subset of 12
 identified target genes after infection with the shRNA-lentiviruses
 ( Fig. 4 B ). The
 expression of four of these genes was decreased by RNAi targeting BRG1,
 suggesting that SWI/SNF activates their transcription: Mm.34431 (a
 conserved gene with unknown function), Crabp1 (cellular retinoic
 acid-binding protein 1), Trh (thyroid releasing hormone), and Nupr1 (nuclear protein 1). By contrast, eight genes were activated by
 interference with BRG1, suggesting that SWI/SNF represses their expression.
 These genes included Eng (endoglin, angiogenic modulator), Ia-1 (insulinoma associated-1), Nodal (angiogenic
 modulator), Rnf128 (ring finger protein 128), Bmi-1 (polycomb component), Adamts1 (extracellular protease, angiogenic
 modulator), Thbs1 (thrombospondin 1, angiogenic modulator), and Dub1 (deubiquitinating enzyme 1). Interference with ARID1A affected
 all 12 SWI/SNF target genes significantly, indicating that BAF complexes are
 required for SWI/SNF-imposed regulation. RNAi targeting BRD7 revealed a very
 different situation. Six genes, including Crabp1, Nupr1, Ia-1, Rnf128,
 Bmi-1 , and Thbs1 , were deregulated in the absence of the
 PBAF-specific subunit BRD7, whereas the expression of four genes, Mm.3443,
 Eng, Nodal , and Adamts1 , did not change upon loss of BRD7.
 Interestingly transcription of two genes, Trh and Dub1 , was
 affected by the absence of BRD7 but in an opposite manner compared with that
 obtained upon loss of ARID1A.

Next we measured association of the PBAF complex with the genomic loci
 containing differently regulated groups of target genes
 ( Fig. 4 C ). Our ChIP
 experiments in ESCs localized BRG1 binding near the transcriptional start site
 of all four tested target genes, Adamts1, Nodal, Trh , and Crabp1 , as indicated by the more than 15-fold increase in DNA
 precipitated by the BRG1-specific antibody J1 compared with a control antibody
 (IgG). Comparing HA precipitates from ESCs infected with a control virus or
 with HA-tagged BAF57 or BRD7, respectively, we observed an increase of greater
 than 70-fold on all target genes, emphasizing the presence of both subunits.
 Taken together, these results suggest that BAF and PBAF subcomplexes can
 antagonize the expression of specific target genes possibly because of their
 intrinsically different biochemical properties.

SWI/SNF Target Gene Regulation throughout Differentiation —To
 examine whether differentiation affects SWI/SNF-mediated regulation of target
 genes, we analyzed the chromatin status of two target genes throughout
 RA-induced differentiation ( Fig.
 5 ). We chose one SWI/SNF-activated gene, Trh , and one
 SWI/SNF-repressed gene, Adamts1 , because of the extent and
 consistency of changes in their transcript levels throughout differentiation
 and BRG1 interference. TRH is a secreted hormone that participates in the
 cascade resulting in the release of thyroid hormones. Trh expression
 is reported to be modulated by several signaling pathways, including
 cAMP-response element-binding protein and signal transducers and activators of
 transcription ( 42 ), epidermal
 growth factor ( 43 ), and
 thyroid hormone. ADAMTS1 is an extracellular protease involved in heart
 development very recently shown to be repressed by BRG1
 ( 44 ) and speculated to be
 regulated by E2Fs, Sma and MAD-related proteins (SMADs), or the T-cell factor
 pathway. Retinoic acid differentiation of ESCs resulted in considerable
 changes in the expression profiles of both genes. Trh was strongly
 repressed, whereas expression of Adamts1 was substantially increased
 ( Fig. 5 A ). To
 correlate any alterations in chromatin structure and transcription factor
 occupancy at these promoters with differentiation-induced changes in
 transcription, we performed ChIP analyses for acetylated histone H3;
 components of the preinitiation complex (PIC), TFIIB and TFIID; and POLII
 ( Fig. 5 B ). On the Trh gene, we observed a general decrease of histone acetylation and a
 loss of both TFIIB and TFIID binding. Because this was accompanied by a
 significant reduction of total POLII, we conclude that retinoic acid
 differentiation led to transcriptional shutdown of the Trh gene in
 part through loss of stable PIC formation. The opposite was true for Adamts1 where differentiation led to a considerable increase in
 binding of PIC components and POLII accompanied by a slight increase in H3
 acetylation. In contrast to the events at the proximal promoter, we saw little
 if any occupancy of these components at a control site 5 kb upstream of the
 transcriptional start sites of either gene.

SWI/SNF Can Regulate Target Genes by Generating Accessibility to the
 PIC —Next we analyzed how differentiation affects binding of SWI/SNF
 to the Trh and Adamts1 promoters. To address this, we
 performed ChIP for the SWI/SNF subunit BAF57 in an ESC line expressing
 3HA-BAF57 ( Fig. 5 C ).
 Spatially resolved analysis revealed that SWI/SNF has two occupancy peaks in
 the Trh promoter at –2 kb and near the transcription start
 site. Differentiation led to a decrease in SWI/SNF occupancy in both sites,
 closely resembling the decrease of PIC components TFIIB and TFIID. To
 establish a causal relationship between SWI/SNF binding and PIC occupancy, we
 performed similar ChIP experiments in a population of cells where knockdown of
 BRG1 led to a 3-fold decrease in Trh expression
 ( Fig. 5 D ). In these
 cells, decreased occupancy of PIC components TFIIB, TFIID, and POLII was
 observed at the proximal promoter despite any significant changes in H3
 acetylation. We conclude that SWI/SNF regulates Trh expression by
 modulating PIC accessibility to the promoter in ESCs. Together with diminished
 H3 acetylation, loss of SWI/SNF occupancy is a possible mechanism by which the Trh gene is silenced upon differentiation. By contrast, this
 mechanism does not apply for Adamts1 . Induction of Adamts1 expression by either differentiation or BRG1 interference was correlated with
 increased occupancy of PIC components and POLII
 ( Fig. 5, B and E ), and SWI/SNF remained bound during differentiation
 ( Fig. 5 C ). Taken
 together, these results underscore the ability of SWI/SNF to regulate its
 target genes in a positive or negative manner using very distinct
 mechanisms.

DISCUSSION

In this study, we investigated different compositional aspects of the
 SWI/SNF complex in embryonic stem cells. Our results show that, in contrast to
 differentiated cells, ESCs have comparably low levels of BAF170 and BRM in
 agreement with other recent reports
 ( 17 ). During mouse
 development, Brm expression is concomitant with the onset of
 vasculogenesis ( 36 ). The
 mutually exclusive BRM and BRG1 compete for incorporation into the same
 remodeling complex, and both ATPases can recruit SWI/SNF to their specific
 target genes through selective interactions with transcription factors
 ( 45 ). Interestingly BRG1
 negatively regulates a subset of angiogenic modulators in ESCs. Therefore, the
 onset of Brm expression is likely to impact BRG1 target genes by
 reducing the overall abundance of BRG1-containing SWI/SNF complexes. This may
 provide a potential Brm expression-dependent mechanism for the onset
 of vasculogenesis and heart development. However, neither Brm nor Baf170 expression alone in ESCs is sufficient to derepress a subset
 of target genes including several angiogenic modulators that are repressed by
 SWI/SNF in ESCs (supplemental Fig. 3).

Selective incorporation of two mutually exclusive signature subunits of the
 BAF complex, ARID1A and ARID1B, was shown to confer activating versus repressive functions of SWI/SNF on cell cycle regulatory genes
 ( 25 ). Our BRG1 interference
 experiments indicate that in ESCs SWI/SNF has a predominantly repressive role,
 which correlates with higher incorporation of the repressive ARID1A subunit.
 Upon differentiation, the increase in ARID1B and decrease in ARID1A suggests
 that compositionally repressive SWI/SNF is converted into a more
 transcriptionally activating enzyme. Other studies have not reported an
 increase in ARID1B ( 17 ) upon
 differentiation. However, Wang and co-workers
 ( 17 ) monitored total ARID1B
 protein levels, whereas our data were derived from detection of ARID1B
 incorporated into SWI/SNF complexes by MudPIT mass spectrometry and
 transcriptional measurements of ARID1B message. Overall we found that the
 diversity of core and specificity subunits increases upon differentiation,
 presumably reflecting the need to regulate more intricate transcriptional
 programs.

Among our main observations is the identification of BRD7 as a new
 PBAF-specific subunit. We found BRD7 present in purifications from pluripotent
 ESCs, differentiated ESCs, and HeLa cells, arguing for its incorporation in a
 variety of cell types. By performing MudPIT mass spectrometry on
 BRD7-containing complexes, we found five unambiguous peptides of the Polybromo
 protein (data not shown), suggesting that the PBAF signature module consists
 of BRD7, ARID2, and Polybromo. We were unable to confirm the presence of
 Polybromo in BRD7-containing complexes because of the lack of antibodies that
 recognize the Polybromo protein. BRD7 contains a bromodomain that interacts
 with acetylated histone H3
 ( 40 ), potentially providing
 additional tethering of the PBAF complex to transcriptionally active
 chromatin.

We also found that BRD7-containing PBAF complexes have very distinct
 biochemical properties from those of BAF as reflected by differences in the
 ability of DNA to stimulate their enzymatic activities. We estimate that PBAF
 complexes represent only a minor fraction of total SWI/SNF in ESCs, which
 contain predominately BAF complexes, because ARID2 was considerably less
 abundant in SWI/SNF complexes (purified by tagged BAF57) than ARID1A. It is
 possible that the basal activity of total SWI/SNF in our ATPase assays was
 contributed by the high ATPase activity of PBAF. The stimulation observed
 after addition of DNA would therefore be entirely due to the biochemical
 properties of BAF. Given the multitude of bromodomains in PBAF complexes, it
 will be interesting to determine whether PBAF exhibits a greater ATPase
 activity stimulation by specifically modified histones than BAF.

It is well documented that PBAF and BAF complexes have non-redundant
 functions, for example, in transactivation of interferon-inducible genes
 ( 46 ). In addition, there is
 considerable overlap of common target genes as might be expected for genes in
 which common subunits are responsible for SWI/SNF recruitment. Interestingly
 in ESCs, BAF appears to be required for the majority of BRG1-dependent
 transcriptional regulation. By contrast, BRD7/PBAF function was dispensable
 for a sizeable subset of the genes we examined, although this complex could be
 readily detected at some of these promoters such as ADAMTS1. A possible
 explanation is that PBAF may be recruited through a subunit other than BRD7.
 Overall our RNAi results are consistent with the severity of the knock-out
 phenotype of respective signature subunits. ARID1A knock-out mice
 ( 14 ) arrest very early in
 development and display deficiencies in early germ layer formation. On the
 other hand, Polybromo knock-outs
 ( 47 ) die relatively late
 (embryonic day 12.5–15) probably because of heart defects.

A majority of SWI/SNF target genes were derepressed by RNAi-mediated
 knockdown of BRG1, indicating that functional SWI/SNFs are involved in
 repressing their expression. Repression by SWI/SNF is well documented and
 shown to be required for the repressive function of the RB (Retinoblastoma)
 protein complex ( 48 , 49 ) and transcriptional
 repression of neuronal genes by the repressor element 1-silencing
 transcription factor (REST) protein
 ( 50 , 51 ). SWI/SNF has also been
 implicated in silencing of methylated genes
 ( 52 ). These studies highlight
 the ability of SWI/SNF to play critical roles as either a transcriptional
 coactivator or corepressor. In ESCs, this correlates with the biased
 incorporation of the repressive ARID1A subunit
 ( 25 ). However, our
 interference experiments of ARID1A show that ARID1A is not exclusively
 repressive as it was required for SWI/SNF-mediated activation of a subset of
 target genes.

Different models have been proposed to explain the function of
 chromatin-remodeling complexes in the sequence of events that govern
 transcription. For example, some studies show that promoter activation
 requires SWI/SNF recruitment before association of the histone
 acetyltransferase S pt- A da- G cn5-acetyltransferase (SAGA)
 ( 53 ) and the PIC
 ( 54 ). In other models,
 transient recruitment of remodeling complexes to poised promoters allows
 release of elongating RNA polymerase II
 ( 55 ). Repression by SWI/SNF
 (and the Rsc complex in yeast) is mediated by establishing an inhibiting
 nucleosomal distribution ( 56 ),
 recruitment of histone deacetylases
 ( 57 ), or indirectly
 stimulating expression of inhibitory non-coding RNAs
 ( 58 ). Taken together, these
 studies reveal that chromatin remodeling can function at many discrete steps
 to impact transcriptional regulation. This mechanistic diversity is likely to
 be dictated by the particular chromatin environment of individual target genes
 and the specific remodeler. Because histone acetylation can serve as a
 recruitment platform for chromatin-remodeling complexes, we speculate that H3
 acetylation is a contributing factor in selective SWI/SNF recruitment and
 function on the Trh promoter. Such a model could explain the loss of
 SWI/SNF occupancy throughout differentiation-mediated H3 acetylation decrease
 and also why BRG1 knockdown leads to lowered transcription without affecting
 H3 acetylation. From this perspective, it will be very interesting to analyze
 the basis of selective BAF and PBAF complex recruitment and how they regulate
 PIC access to the Trh core promoter in ESCs. From a broader
 viewpoint, several components of SWI/SNF have been shown to be necessary for
 long term survival of the inner cell mass
 ( 11 – 14 ).
 Moreover decreased cell numbers are observed after BRG1 depletion suggesting
 that BRG1 is required for ESC growth
 ( 4 ) perhaps through functions
 in addition to transcriptional regulation. In this regard, SWI/SNF has been
 linked to genome organization, recombination, cell cycle progression, and the
 DNA damage response ( 8 , 18 , 59 ). Future investigations
 into the potential roles for SWI/SNF in these and other processes should
 generate a more comprehensive understanding of its contribution to ES cell
 pluripotency.
