---
pmid: '30224647'
title: De novo mutations in MSL3 cause an X-linked syndrome marked by impaired histone
  H4 lysine 16 acetylation.
authors:
- Basilicata MF
- Bruel AL
- Semplicio G
- Valsecchi CIK
- Aktaş T
- Duffourd Y
- Rumpf T
- Morton J
- Bache I
- Szymanski WG
- Gilissen C
- Vanakker O
- Õunap K
- Mittler G
- van der Burgt I
- El Chehadeh S
- Cho MT
- Pfundt R
- Tan TY
- Kirchhoff M
- Menten B
- Vergult S
- Lindstrom K
- Reis A
- Johnson DS
- Fryer A
- McKay V
- DDD Study
- Fisher RB
- Thauvin-Robinet C
- Francis D
- Roscioli T
- Pajusalu S
- Radtke K
- Ganesh J
- Brunner HG
- Wilson M
- Faivre L
- Kalscheuer VM
- Thevenon J
- Akhtar A
journal: Nat Genet
year: '2018'
full_text_available: false
pmcid: PMC7398719
doi: 10.1038/s41588-018-0220-y
---

# De novo mutations in MSL3 cause an X-linked syndrome marked by impaired histone H4 lysine 16 acetylation.
**Authors:** Basilicata MF, Bruel AL, Semplicio G, Valsecchi CIK, Aktaş T, Duffourd Y, Rumpf T, Morton J, Bache I, Szymanski WG, Gilissen C, Vanakker O, Õunap K, Mittler G, van der Burgt I, El Chehadeh S, Cho MT, Pfundt R, Tan TY, Kirchhoff M, Menten B, Vergult S, Lindstrom K, Reis A, Johnson DS, Fryer A, McKay V, DDD Study, Fisher RB, Thauvin-Robinet C, Francis D, Roscioli T, Pajusalu S, Radtke K, Ganesh J, Brunner HG, Wilson M, Faivre L, Kalscheuer VM, Thevenon J, Akhtar A
**Journal:** Nat Genet (2018)
**DOI:** [10.1038/s41588-018-0220-y](https://doi.org/10.1038/s41588-018-0220-y)
**PMC:** [PMC7398719](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7398719/)

## Abstract

1. Nat Genet. 2018 Oct;50(10):1442-1451. doi: 10.1038/s41588-018-0220-y. Epub
2018  Sep 17.

De novo mutations in MSL3 cause an X-linked syndrome marked by impaired histone 
H4 lysine 16 acetylation.

Basilicata MF(1), Bruel AL(2), Semplicio G(1), Valsecchi CIK(1), Aktaş T(1), 
Duffourd Y(2), Rumpf T(1), Morton J(3), Bache I(4)(5), Szymanski WG(1), Gilissen 
C(6), Vanakker O(7), Õunap K(8), Mittler G(1), van der Burgt I(6), El Chehadeh 
S(2)(9), Cho MT(10), Pfundt R(6), Tan TY(11), Kirchhoff M(4), Menten B(7), 
Vergult S(7), Lindstrom K(12), Reis A(13), Johnson DS(14), Fryer A(15), McKay 
V(15); DDD Study; Fisher RB(16), Thauvin-Robinet C(2), Francis D(17), Roscioli 
T(18)(19)(20), Pajusalu S(8), Radtke K(21), Ganesh J(22), Brunner HG(6)(23), 
Wilson M(24), Faivre L(2), Kalscheuer VM(25), Thevenon J(26)(27), Akhtar A(28).

Author information:
(1)Max Planck Institute of Immunobiology and Epigenetics, Freiburg im Breisgau, 
Germany.
(2)Inserm UMR 1231 GAD, Genetics of Developmental disorders and Centre de 
Référence Maladies Rares Anomalies du Développement et syndromes malformatifs 
FHU TRANSLAD, Université de Bourgogne-Franche Comté, Dijon, France.
(3)West Midlands Regional Clinical Genetics Service and Birmingham Health 
Partners, Birmingham Women's Hospital NHS Foundation Trust, Birmingham, UK.
(4)Department of Clinical Genetics, Copenhagen University Hospital, 
Rigshospitalet, Copenhagen, Denmark.
(5)Wilhelm Johannsen Centre for Functional Genome Research, Department of 
Cellular and Molecular Medicine, University of Copenhagen, Copenhagen, Denmark.
(6)Department of Human Genetics, Radboud University Medical Center, Donders 
Institute for Brain, Cognition and Behaviour, Nijmegen, The Netherlands.
(7)Center for Medical Genetics, Ghent University Hospital, Ghent, Belgium.
(8)Department of Clinical Genetics, United Laboratories, Tartu University 
Hospital and Institute of Clinical Medicine, University of Tartu, Tartu, 
Estonia.
(9)Service de Génétique Médicale, Hôpital de Hautepierre, Strasbourg, France.
(10)GeneDx, Gaithersburg, MD, USA.
(11)Victorian Clinical Genetics Services, Murdoch Children's Research Institute, 
Royal Children's Hospital, University of Melbourne Department of Paediatrics, 
Parkville, VIC, Australia.
(12)Division of Genetics and Metabolism, Phoenix Children's Hospital, Phoenix, 
AZ, USA.
(13)Institute of Human Genetics, Friedrich-Alexander-Universität 
Erlangen-Nürnberg, Erlangen, Germany.
(14)Sheffield Clinical Genetics Service, Sheffield Children's NHS Foundation 
Trust, Sheffield, UK.
(15)Department of Clinical Genetics, Liverpool Women's NHS Foundation Trust, 
Liverpool, UK.
(16)Northern Genetics Service, Teesside Genetics Unit, The James Cook University 
Hospital, Middlesbrough, UK.
(17)Cytogenetic Laboratory, Victorian Clinical Genetics Services, Murdoch 
Children's Research Institute, Royal Children's Hospital, Melbourne, Victoria, 
Australia.
(18)Neuroscience Research Australia, Sydney, New South Wales, Australia.
(19)Prince of Wales Clinical School, University of New South Wales, Sydney, New 
South Wales, Australia.
(20)Department of Medical Genetics, Sydney Children's Hospital, Sydney, New 
South Wales, Australia.
(21)Department of Clinical Genomics, Ambry Genetics, Aliso Viejo, CA, USA.
(22)Division of Genetics, Cooper University Hospital and Cooper Medical School 
at Rowan University, Camden, NJ, USA.
(23)Department of Clinical Genetics and School for Oncology and Developmental 
Biology, Maastricht University Medical Center, Maastricht, The Netherlands.
(24)Department of Clinical Genetics, Children's Hospital at Westmead, 
Disciplines of Genetic Medicine and Child and Adolescent Health, University of 
Sydney, Sydney, New South Wales, Australia.
(25)Research Group Development and Disease, Max Planck Institute for Molecular 
Genetics, Berlin, Germany.
(26)Inserm UMR 1231 GAD, Genetics of Developmental disorders and Centre de 
Référence Maladies Rares Anomalies du Développement et syndromes malformatifs 
FHU TRANSLAD, Université de Bourgogne-Franche Comté, Dijon, France. 
jthevenon@chu-grenoble.fr.
(27)CNRS UMR 5309, INSERM, U1209, Institute of Advanced Biosciences, Université 
Grenoble-Alpes CHU Grenoble, Grenoble, France. jthevenon@chu-grenoble.fr.
(28)Max Planck Institute of Immunobiology and Epigenetics, Freiburg im Breisgau, 
Germany. akhtar@ie-freiburg.mpg.de.

The etiological spectrum of ultra-rare developmental disorders remains to be 
fully defined. Chromatin regulatory mechanisms maintain cellular identity and 
function, where misregulation may lead to developmental defects. Here, we report 
pathogenic variations in MSL3, which encodes a member of the 
chromatin-associated male-specific lethal (MSL) complex responsible for bulk 
histone H4 lysine 16 acetylation (H4K16ac) in flies and mammals. These variants 
cause an X-linked syndrome affecting both sexes. Clinical features of the 
syndrome include global developmental delay, progressive gait disturbance, and 
recognizable facial dysmorphism. MSL3 mutations affect MSL complex assembly and 
activity, accompanied by a pronounced loss of H4K16ac levels in vivo. 
Patient-derived cells display global transcriptome alterations of pathways 
involved in morphogenesis and cell migration. Finally, we use histone 
deacetylase inhibitors to rebalance acetylation levels, alleviating some of the 
molecular and cellular phenotypes of patient cells. Taken together, we 
characterize a syndrome that allowed us to decipher the developmental importance 
of MSL3 in humans.

DOI: 10.1038/s41588-018-0220-y
PMCID: PMC7398719
PMID: 30224647 [Indexed for MEDLINE]

Conflict of interest statement: Competing Financial Interests The authors 
declare no competing financial interests
