---
pmid: '27499296'
title: Mitochondrial Protein Interaction Mapping Identifies Regulators of Respiratory
  Chain Function.
authors:
- Floyd BJ
- Wilkerson EM
- Veling MT
- Minogue CE
- Xia C
- Beebe ET
- Wrobel RL
- Cho H
- Kremer LS
- Alston CL
- Gromek KA
- Dolan BK
- Ulbrich A
- Stefely JA
- Bohl SL
- Werner KM
- Jochem A
- Westphall MS
- Rensvold JW
- Taylor RW
- Prokisch H
- Kim JP
- Coon JJ
- Pagliarini DJ
journal: Mol Cell
year: '2016'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC4992456
doi: 10.1016/j.molcel.2016.06.033
---

# Mitochondrial Protein Interaction Mapping Identifies Regulators of Respiratory Chain Function.
**Authors:** Floyd BJ, Wilkerson EM, Veling MT, Minogue CE, Xia C, Beebe ET, Wrobel RL, Cho H, Kremer LS, Alston CL, Gromek KA, Dolan BK, Ulbrich A, Stefely JA, Bohl SL, Werner KM, Jochem A, Westphall MS, Rensvold JW, Taylor RW, Prokisch H, Kim JP, Coon JJ, Pagliarini DJ
**Journal:** Mol Cell (2016)
**DOI:** [10.1016/j.molcel.2016.06.033](https://doi.org/10.1016/j.molcel.2016.06.033)
**PMC:** [PMC4992456](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4992456/)

## Abstract

1. Mol Cell. 2016 Aug 18;63(4):621-632. doi: 10.1016/j.molcel.2016.06.033. Epub 
2016 Aug 4.

Mitochondrial Protein Interaction Mapping Identifies Regulators of Respiratory 
Chain Function.

Floyd BJ(1), Wilkerson EM(2), Veling MT(1), Minogue CE(2), Xia C(3), Beebe 
ET(4), Wrobel RL(4), Cho H(1), Kremer LS(5), Alston CL(6), Gromek KA(4), Dolan 
BK(4), Ulbrich A(2), Stefely JA(1), Bohl SL(1), Werner KM(4), Jochem A(7), 
Westphall MS(8), Rensvold JW(7), Taylor RW(6), Prokisch H(5), Kim JP(3), Coon 
JJ(9), Pagliarini DJ(10).

Author information:
(1)Morgridge Institute for Research, Madison, WI 53715, USA; Department of 
Biochemistry, University of Wisconsin-Madison, Madison, WI 53706, USA.
(2)Department of Chemistry, University of Wisconsin-Madison, Madison, WI 53706, 
USA.
(3)Department of Biochemistry, Medical College of Wisconsin, Milwaukee, WI 
53226, USA.
(4)Department of Biochemistry, University of Wisconsin-Madison, Madison, WI 
53706, USA.
(5)Institute of Human Genetics, Technische Universität München, 81675 München, 
Germany; Institute of Human Genetics, Helmholtz Zentrum München, 85764 
Neuherberg, Germany.
(6)Wellcome Trust Centre for Mitochondrial Research, Institute of Neuroscience, 
The Medical School, Newcastle University, Newcastle upon Tyne NE2 4HH, UK.
(7)Morgridge Institute for Research, Madison, WI 53715, USA.
(8)Genome Center of Wisconsin, University of Wisconsin-Madison, Madison, WI 
53706, USA.
(9)Department of Chemistry, University of Wisconsin-Madison, Madison, WI 53706, 
USA; Department of Biomolecular Chemistry, University of Wisconsin-Madison, 
Madison, WI 53706, USA; Genome Center of Wisconsin, University of 
Wisconsin-Madison, Madison, WI 53706, USA.
(10)Morgridge Institute for Research, Madison, WI 53715, USA; Department of 
Biochemistry, University of Wisconsin-Madison, Madison, WI 53706, USA. 
Electronic address: dpagliarini@morgridge.org.

Mitochondria are essential for numerous cellular processes, yet hundreds of 
their proteins lack robust functional annotation. To reveal functions for these 
proteins (termed MXPs), we assessed condition-specific protein-protein 
interactions for 50 select MXPs using affinity enrichment mass spectrometry. Our 
data connect MXPs to diverse mitochondrial processes, including multiple aspects 
of respiratory chain function. Building upon these observations, we validated 
C17orf89 as a complex I (CI) assembly factor. Disruption of C17orf89 markedly 
reduced CI activity, and its depletion is found in an unresolved case of CI 
deficiency. We likewise discovered that LYRM5 interacts with and deflavinates 
the electron-transferring flavoprotein that shuttles electrons to coenzyme Q 
(CoQ). Finally, we identified a dynamic human CoQ biosynthetic complex involving 
multiple MXPs whose topology we map using purified components. Collectively, our 
data lend mechanistic insight into respiratory chain-related activities and 
prioritize hundreds of additional interactions for further exploration of 
mitochondrial protein function.

Copyright © 2016 Elsevier Inc. All rights reserved.

DOI: 10.1016/j.molcel.2016.06.033
PMCID: PMC4992456
PMID: 27499296 [Indexed for MEDLINE]

## Full Text

INTRODUCTION

Mitochondria are centers of metabolism for nearly all eukaryotic cells. Once considered to be mere sites of ATP generation, it is now appreciated that mitochondria participate in a wide range of essential functions related to cellular metabolism, signaling, and programmed cell death ( Pagliarini and Rutter, 2013 ). Consistently, large-scale proteomics- and computation-based efforts during the past decade have revealed that the mitochondrial proteome is much more extensive than once thought, and dysfunction of these organelles is now associated with hundreds of inborn errors of metabolism and common diseases ( Koopman et al., 2012 ; Nunnari and Suomalainen, 2012 ).

Despite our advanced cataloging of the mammalian mitochondrial proteome, functional characterization of these proteins is far from complete ( Calvo et al., 2016 ; Pagliarini et al., 2008 ). This gap in knowledge has limited our understanding of basic mitochondrial biology and has obscured the nature and cause of many mitochondrial diseases. For instance, many patients with biochemically-established mitochondrial disease lack mutations in known mitochondrial disease genes, implying the existence of unidentified proteins whose proper function is necessary for the affected process ( Calvo et al., 2010 ; Haack et al., 2011 ). Alternatively, other diseases arise from mutations in mitochondrial proteins with no known function, making it difficult to interrogate the molecular mechanism of the disease.

Protein-protein interactions (PPIs) can provide powerful insight into protein function. Recent advancements in affinity enrichment mass spectrometry (AE-MS) ( Hein et al., 2015 ; Hosp et al., 2015 ; Huttlin et al., 2015 ; Keilhauer et al., 2015 ) have improved the ability to accurately detect PPIs, enabling researchers to overcome the systematic bias against poorly characterized proteins inherent in many large-scale “interactome” analyses ( Sahni et al., 2015 ). With AE-MS, a protein of interest (i.e., the bait) is enriched from a sample; it, along with any co-enriching proteins, is then analyzed by mass spectrometry (MS). In this process, the majority of captured proteins are typically not meaningful interactors of the bait. To sort the so-called wheat from the chaff, the AE-MS approach combines the analyses of multiple baits with a robust quantitative MS platform and a scoring algorithm to differentiate between informative interactions and nonspecific background co-enrichment.

In this study, we began by curating a list of proteins from the mitochondrial proteome that lack significant functional annotation, which we call m itochondrial uncharacterized ( x ) p roteins (MXPs). Using stringent criteria, we conservatively estimate there to be 228 MXPs in humans, including more than 25 that have been associated with human diseases. We then designed a robust AE-MS strategy to define an extensive cell type- and condition-specific interactome of 50 select MXPs, enabling us to propose new functions and disease associations for these uncharacterized proteins.

Informed by our large-scale AE-MS analyses, we conducted extensive in vitro biochemistry and cell biology experiments that validated roles for MXPs in various aspects of the mitochondrial respiratory chain. We establish C17orf89 as a complex I (CI) assembly factor, whose silencing markedly impairs CI activity, and whose depletion is found in an unresolved case of isolated CI deficiency. We also establish LYRM5 as a “deflavinase” that directly regulates the electron transferring flavoprotein (ETF), and identify a dynamic human coenzyme Q biosynthetic complex that includes multiple MXPs. We propose functions for a variety of other MXPs and make our data freely available to the community for accelerate further annotation of mitochondrial proteins.
