---
pmid: '21969609'
title: Proteogenomic analysis of Mycobacterium tuberculosis by high resolution mass
  spectrometry.
authors:
- Kelkar DS
- Kumar D
- Kumar P
- Balakrishnan L
- Muthusamy B
- Yadav AK
- Shrivastava P
- Marimuthu A
- Anand S
- Sundaram H
- Kingsbury R
- Harsha HC
- Nair B
- Prasad TS
- Chauhan DS
- Katoch K
- Katoch VM
- Kumar P
- Chaerkady R
- Ramachandran S
- Dash D
- Pandey A
journal: Mol Cell Proteomics
year: '2011'
full_text_available: false
full_text_extraction_method: html_abstract_only
pmcid: PMC3275902
doi: 10.1074/mcp.M111.011445
---

# Proteogenomic analysis of Mycobacterium tuberculosis by high resolution mass spectrometry.
**Authors:** Kelkar DS, Kumar D, Kumar P, Balakrishnan L, Muthusamy B, Yadav AK, Shrivastava P, Marimuthu A, Anand S, Sundaram H, Kingsbury R, Harsha HC, Nair B, Prasad TS, Chauhan DS, Katoch K, Katoch VM, Kumar P, Chaerkady R, Ramachandran S, Dash D, Pandey A
**Journal:** Mol Cell Proteomics (2011)
**DOI:** [10.1074/mcp.M111.011445](https://doi.org/10.1074/mcp.M111.011445)
**PMC:** [PMC3275902](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3275902/)

## Abstract

1. Mol Cell Proteomics. 2011 Dec;10(12):M111.011627. doi:
10.1074/mcp.M111.011445.  Epub 2011 Oct 3.

Proteogenomic analysis of Mycobacterium tuberculosis by high resolution mass 
spectrometry.

Kelkar DS(1), Kumar D, Kumar P, Balakrishnan L, Muthusamy B, Yadav AK, 
Shrivastava P, Marimuthu A, Anand S, Sundaram H, Kingsbury R, Harsha HC, Nair B, 
Prasad TS, Chauhan DS, Katoch K, Katoch VM, Kumar P, Chaerkady R, Ramachandran 
S, Dash D, Pandey A.

Author information:
(1)Institute of Bioinformatics, International Technology Park, Bangalore, India.

The genome sequencing of H37Rv strain of Mycobacterium tuberculosis was 
completed in 1998 followed by the whole genome sequencing of a clinical isolate, 
CDC1551 in 2002. Since then, the genomic sequences of a number of other strains 
have become available making it one of the better studied pathogenic bacterial 
species at the genomic level. However, annotation of its genome remains 
challenging because of high GC content and dissimilarity to other model 
prokaryotes. To this end, we carried out an in-depth proteogenomic analysis of 
the M. tuberculosis H37Rv strain using Fourier transform mass spectrometry with 
high resolution at both MS and tandem MS levels. In all, we identified 3176 
proteins from Mycobacterium tuberculosis representing ~80% of its total 
predicted gene count. In addition to protein database search, we carried out a 
genome database search, which led to identification of ~250 novel peptides. 
Based on these novel genome search-specific peptides, we discovered 41 novel 
protein coding genes in the H37Rv genome. Using peptide evidence and alternative 
gene prediction tools, we also corrected 79 gene models. Finally, mass 
spectrometric data from N terminus-derived peptides confirmed 727 existing 
annotations for translational start sites while correcting those for 33 
proteins. We report creation of a high confidence set of protein coding regions 
in Mycobacterium tuberculosis genome obtained by high resolution tandem 
mass-spectrometry at both precursor and fragment detection steps for the first 
time. This proteogenomic approach should be generally applicable to other 
organisms whose genomes have already been sequenced for obtaining a more 
accurate catalogue of protein-coding genes.

DOI: 10.1074/mcp.M111.011445
PMCID: PMC3275902
PMID: 21969609 [Indexed for MEDLINE]

## Full Text

Abstract

The genome sequencing of H37Rv strain of Mycobacterium tuberculosis was completed in 1998 followed by the whole genome sequencing of a clinical isolate, CDC1551 in 2002. Since then, the genomic sequences of a number of other strains have become available making it one of the better studied pathogenic bacterial species at the genomic level. However, annotation of its genome remains challenging because of high GC content and dissimilarity to other model prokaryotes. To this end, we carried out an in-depth proteogenomic analysis of the M. tuberculosis H37Rv strain using Fourier transform mass spectrometry with high resolution at both MS and tandem MS levels. In all, we identified 3176 proteins from Mycobacterium tuberculosis representing ∼80% of its total predicted gene count. In addition to protein database search, we carried out a genome database search, which led to identification of ∼250 novel peptides. Based on these novel genome search-specific peptides, we discovered 41 novel protein coding genes in the H37Rv genome. Using peptide evidence and alternative gene prediction tools, we also corrected 79 gene models. Finally, mass spectrometric data from N terminus-derived peptides confirmed 727 existing annotations for translational start sites while correcting those for 33 proteins. We report creation of a high confidence set of protein coding regions in Mycobacterium tuberculosis genome obtained by high resolution tandem mass-spectrometry at both precursor and fragment detection steps for the first time. This proteogenomic approach should be generally applicable to other organisms whose genomes have already been sequenced for obtaining a more accurate catalogue of protein-coding genes.

DISCUSSION

M. tuberculosis H37Rv genome sequence has been available for more than 10 years and has been characterized previously by a number of groups. Thus, it was surprising to identify many novel regions with protein coding potential. Even more surprising was the fact that most of these novel ORFs were already annotated in other strains of M. tuberculosis but were missing from the primary genome sequence of M. tuberculosis . We have demonstrated the power of the proteogenomic approach to annotate a genome and refine the annotation of a well studied genome. Though relatively new, this approach of using mass spectrometry-based proteomic data to identify protein coding regions in the genome can prove to be an essential complementary method along with computational methods for annotating newly sequenced genomes in the future.
