---
pmid: '19796622'
title: Uncovering early response of gene regulatory networks in ESCs by systematic
  induction of transcription factors.
authors:
- Nishiyama A
- Xin L
- Sharov AA
- Thomas M
- Mowrer G
- Meyers E
- Piao Y
- Mehta S
- Yee S
- Nakatake Y
- Stagg C
- Sharova L
- Correa-Cerro LS
- Bassey U
- Hoang H
- Kim E
- Tapnio R
- Qian Y
- Dudekula D
- Zalzman M
- Li M
- Falco G
- Yang HT
- Lee SL
- Monti M
- Stanghellini I
- Islam MN
- Nagaraja R
- Goldberg I
- Wang W
- Longo DL
- Schlessinger D
- Ko MS
journal: Cell Stem Cell
year: '2009'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC2770715
doi: 10.1016/j.stem.2009.07.012
---

# Uncovering early response of gene regulatory networks in ESCs by systematic induction of transcription factors.
**Authors:** Nishiyama A, Xin L, Sharov AA, Thomas M, Mowrer G, Meyers E, Piao Y, Mehta S, Yee S, Nakatake Y, Stagg C, Sharova L, Correa-Cerro LS, Bassey U, Hoang H, Kim E, Tapnio R, Qian Y, Dudekula D, Zalzman M, Li M, Falco G, Yang HT, Lee SL, Monti M, Stanghellini I, Islam MN, Nagaraja R, Goldberg I, Wang W, Longo DL, Schlessinger D, Ko MS
**Journal:** Cell Stem Cell (2009)
**DOI:** [10.1016/j.stem.2009.07.012](https://doi.org/10.1016/j.stem.2009.07.012)
**PMC:** [PMC2770715](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2770715/)

## Abstract

1. Cell Stem Cell. 2009 Oct 2;5(4):420-33. doi: 10.1016/j.stem.2009.07.012.

Uncovering early response of gene regulatory networks in ESCs by systematic 
induction of transcription factors.

Nishiyama A(1), Xin L, Sharov AA, Thomas M, Mowrer G, Meyers E, Piao Y, Mehta S, 
Yee S, Nakatake Y, Stagg C, Sharova L, Correa-Cerro LS, Bassey U, Hoang H, Kim 
E, Tapnio R, Qian Y, Dudekula D, Zalzman M, Li M, Falco G, Yang HT, Lee SL, 
Monti M, Stanghellini I, Islam MN, Nagaraja R, Goldberg I, Wang W, Longo DL, 
Schlessinger D, Ko MS.

Author information:
(1)National Institute on Aging, National Institutes of Health, Baltimore, MD 
21224, USA.

To examine transcription factor (TF) network(s), we created mouse ESC lines, in 
each of which 1 of 50 TFs tagged with a FLAG moiety is inserted into a 
ubiquitously controllable tetracycline-repressible locus. Of the 50 TFs, Cdx2 
provoked the most extensive transcriptome perturbation in ESCs, followed by 
Esx1, Sox9, Tcf3, Klf4, and Gata3. ChIP-Seq revealed that CDX2 binds to 
promoters of upregulated target genes. By contrast, genes downregulated by CDX2 
did not show CDX2 binding but were enriched with binding sites for POU5F1, SOX2, 
and NANOG. Genes with binding sites for these core TFs were also downregulated 
by the induction of at least 15 other TFs, suggesting a common initial step for 
ESC differentiation mediated by interference with the binding of core TFs to 
their target genes. These ESC lines provide a fundamental resource to study 
biological networks in ESCs and mice.

DOI: 10.1016/j.stem.2009.07.012
PMCID: PMC2770715
PMID: 19796622 [Indexed for MEDLINE]

## Full Text

INTRODUCTION

The prevailing paradigm of modern biology states that gene regulatory networks determine the identity of cells, and their alteration by environmental factors dictates changes of cell identity, i.e., cell differentiation ( Davidson, 2006 ). Analysis of the structure and dynamics of gene regulatory networks is key to the understanding of biological systems but poses a great challenge due to the vast and manifold complexity of regulatory mechanisms.

One possible approach is to carry out a systematic gene perturbation study in order to “reverse engineer” these regulatory networks. Ideally, all the transcription factors would be manipulated one at a time and in different combinations in a variety of cell types. Readout of the impact of such manipulation would be monitored in a variety of ways, including the profiling of all RNA transcripts and proteins. This type of approach is complementary to conventional studies in which systematic gene manipulations have been successfully carried out, but usually with a focus on the phenotype, e.g., cell morphology, growth property, and differentiation markers ( Chambers et al., 2003 ; Ivanova et al., 2006 ; Pritsker et al., 2006 ). To this end, mouse ES cells ( Evans and Kaufman, 1981 ; Martin, 1981 ) are most suitable, as they can be differentiated into a variety of cell types in distinct cell culture conditions in vitro ( Murry and Keller, 2008 ) and can also be developed into animal models to further study the effects of gene perturbation in vivo ( Solter, 2006 ). As a complementary approach to the comprehensive “loss-of-function” study of all mouse genes ( Collins et al., 2007 ; Skarnes et al., 2004 ), we aim to generate ES cell lines in which a TF can be induced for “gain-of-function” in a controlled manner, enabling observations of the network perturbation caused by each TF in a uniform condition across all the ES cell lines.

Global gene regulatory networks have been intensively studied in mouse ES cells by expression profiling ( Walker et al., 2007 ), protein complex analysis ( Wang et al., 2006 ), and genome-wide chromatin immunoprecipitation ( Loh et al., 2006 ). The critical roles of three transcription factors - Pou5f1 ( Oct4 or Oct3/4 ) ( Nichols et al., 1998 ), Sox2 ( Yuan et al., 1995 ), and Nanog ( Chambers et al., 2003 ; Mitsui et al., 2003 ) – discovered earlier (reviewed in ( Niwa, 2007 ; Silva and Smith, 2008 )) have recently been rationalized by the discovery of core transcriptional regulatory networks between these genes ( Boyer et al., 2005 ; Loh et al., 2006 ). Furthermore, similar analyses of other key TFs in mouse ES cells have successfully extended the core transcriptional network ( Chen et al., 2008 ; Kim et al., 2008 ). Obviously, the analysis of many more TFs, including genes that are not expressed in ES cells, is required to explore global TF network(s) that may be outside of the core transcriptional network of Pou5f1 , Sox2 , and Nanog . To this end, an appropriate mouse ES cell bank could facilitate a variety of high-throughput, genome-wide analysis methodologies.

Here we describe a strategy for and the establishment of TF-inducible ES cell lines, and we show how these ES cell lines can be used in several ways. As a proof of principle for the strategy, we characterize how an exemplary differentiation-inducing TF, Cdx2 , regulates the global transcriptome and shifts the balance of gene regulatory network toward ES cell differentiation.

DISCUSSION

Initial analyses of ~3% (50) of all TF genes have provided a glimpse of the structure and dynamics of global gene regulatory networks as well as proof-of-principle that this experimental system provides potentially universal tools and resources for “gain-of-function” analyses of transcription factors (TFs) in vitro and in vivo ( Figure 7B ). The ES cell lines reported here will be freely available to the research community, which could facilitate rapid accumulation and comparison of a variety of information on these standardized ES cells.
