---
pmid: '16762840'
title: Swi6/HP1 recruits a JmjC domain protein to facilitate transcription of heterochromatic
  repeats.
authors:
- Zofall M
- Grewal SI
journal: Mol Cell
year: '2006'
full_text_available: true
pmcid: PMC12124358
doi: 10.1016/j.molcel.2006.05.010
---

# Swi6/HP1 recruits a JmjC domain protein to facilitate transcription of heterochromatic repeats.
**Authors:** Zofall M, Grewal SI
**Journal:** Mol Cell (2006)
**DOI:** [10.1016/j.molcel.2006.05.010](https://doi.org/10.1016/j.molcel.2006.05.010)
**PMC:** [PMC12124358](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC12124358/)

## Abstract

1. Mol Cell. 2006 Jun 9;22(5):681-92. doi: 10.1016/j.molcel.2006.05.010.

Swi6/HP1 recruits a JmjC domain protein to facilitate transcription of 
heterochromatic repeats.

Zofall M(1), Grewal SI.

Author information:
(1)Laboratory of Molecular Cell Biology, National Cancer Institute, National 
Institutes of Health, Bethesda, Maryland 20892, USA.

Comment in
    Mol Cell. 2006 Jun 23;22(6):709-710. doi: 10.1016/j.molcel.2006.06.004.

Heterochromatin formation is generally thought to result in transcriptional 
repression of target loci. However, RNAi-mediated heterochromatin assembly 
requires RNA polymerase II (Pol II) transcription. The mechanism facilitating 
Pol II accessibility to heterochromatin is unknown. We show that the fission 
yeast Epe1, a JmjC domain-containing protein and a negative regulator of 
heterochromatin, is distributed across all major heterochromatic domains and at 
certain meiotic genes. Remarkably, Epe1 is recruited to heterochromatic loci by 
the heterochromatin protein Swi6/HP1. Moreover, Epe1 acts in a 
heterochromatin-specific context to promote Pol II accessibility by 
counteracting repressive chromatin. This requires Epe1's JmjC domain, although 
the mechanism utilized might be distinct from other JmjC proteins that possess 
known demethylase activities. We also find that Epe1 is preferentially recruited 
to inverted repeats flanking centromeres to restrain the spread of 
pericentromeric heterochromatin. Our analyses suggest that Swi6/HP1 recruits 
opposing chromatin-modifying activities, the balancing of which is crucial for 
heterochromatin maintenance.

DOI: 10.1016/j.molcel.2006.05.010
PMID: 16762840 [Indexed for MEDLINE]

## Full Text

INTRODUCTIONHistone H3K9 methylation (H3K9me) is a central and conserved epigenetic hallmark of transcriptional silencing associated with heterochromatin formation (1–5). H3K9me deposition is a highly efficient process in vivo, as it induces a positive feedback loop involving chromodomain (CD)–mediated read-write process (6–8) and cross-talk with RNA interference (RNAi)–dependent cotranscriptional gene silencing (9), leading to rapid H3K9me spreading. Multiple regulatory pathways are involved in controlling H3K9me generation, whose dysregulation is closely related to cancer, neurodegenerative, and viral diseases in humans (5, 10–12). Related pathways include H3K9 demethylation (13–17), H3K9 acetylation (H3K9ac) occlusion and deacetylation (18–24), and Argonaute (Ago1)–small RNA–dependent (7, 25–32) or –independent (24, 33) methyltransferase localization. These pathways precisely control H3K9me deposition and prevent aberrant heterochromatin deposition and deleterious gene silencing.In addition to the above regulatory pathways, direct tuning of the histone methyltransferase (HMTase) activity that catalyzes H3K9me is indispensable for controlling H3K9me deposition. In Schizosaccharomyces pombe (S. pombe), which serves as a paradigmatic model organism for understanding the establishment and inheritance of heterochromatin, the sole H3K9 HMTase belonging to SUV39 superfamily, Clr4, undergoes automethylation at K455 and K472. This automethylation releases Clr4 from its intrinsic autoinhibitory state, which is necessary for appropriate heterochromatin establishment and epigenetic stability (34). Besides being activated by automethylation, the HMTase activity of Clr4 is strongly promoted by H3K14 ubiquitination (H3K14Ub) (35, 36), which is catalyzed by the Clr4-containing Clr4-Rik1-Cul4 methyltransferase complex (CLRC) (37–39). The stimulatory effect of H3K14Ub on Clr4 is considerably more pronounced than that of the Clr4 automethylation (34, 40). Intriguingly, bioinformatics analysis suggested that Clr4 lacks a ubiquitin-binding domain, and experimental evidence indicates that it does not sense free ubiquitin (40). The mechanism underlying the substantial stimulatory effect of H3K14Ub on Clr4 activity remains unknown.Here, we determined the crystal structures of Clr4 bound to ubiquitinated and unmodified H3 peptides at resolutions of 2.60 and 2.39 Å, respectively. Combined with the results of biochemical experiments, our findings reveal the molecular mechanism underlying the synergistic effects of affinity and turnover enhancement that results in significant stimulation by H3K14Ub. Specifically, compared with the faint increase in substrate recruitment induced by Clr4 automethylation, H3K14Ub substantially enhances the affinity of Clr4 for substrate. The ubiquitin constrained by the isopeptide bond linkage at the H3K14 position forms multivalent interactions with the Clr4 catalytic domain, stabilizing substrate binding and rapidly promoting H3K9me2/3 deposition. Moreover, our findings indicate that the previously reported automethylated state of Clr4 requires additional allosteric changes to transition into the “catalyzing state,” which is characterized by conformational changes in the compact αC-SET [Su(var), enhancer of zeste, trithorax]-insertion (SI) region, complete release of the autoregulatory loop (ARL), and retraction of the β9/10 loop. Furthermore, H3K14Ub facilitates substrate turnover by promoting the transition of Clr4 to a hyperactive catalyzing state, primarily through nascent interactions and conformational changes within the structural elements surrounding the β9/10 loop. Our work elucidates the continuous conformational changes occurring during the disinhibition of Clr4 via automethylation, substrate recruitment, and stimulation by H3K14Ub, thereby advancing our understanding of the regulation of H3K9me.

INTRODUCTION

Histone H3K9 methylation (H3K9me) is a central and conserved epigenetic hallmark of transcriptional silencing associated with heterochromatin formation (1–5). H3K9me deposition is a highly efficient process in vivo, as it induces a positive feedback loop involving chromodomain (CD)–mediated read-write process (6–8) and cross-talk with RNA interference (RNAi)–dependent cotranscriptional gene silencing (9), leading to rapid H3K9me spreading. Multiple regulatory pathways are involved in controlling H3K9me generation, whose dysregulation is closely related to cancer, neurodegenerative, and viral diseases in humans (5, 10–12). Related pathways include H3K9 demethylation (13–17), H3K9 acetylation (H3K9ac) occlusion and deacetylation (18–24), and Argonaute (Ago1)–small RNA–dependent (7, 25–32) or –independent (24, 33) methyltransferase localization. These pathways precisely control H3K9me deposition and prevent aberrant heterochromatin deposition and deleterious gene silencing.

In addition to the above regulatory pathways, direct tuning of the histone methyltransferase (HMTase) activity that catalyzes H3K9me is indispensable for controlling H3K9me deposition. In Schizosaccharomyces pombe (S. pombe), which serves as a paradigmatic model organism for understanding the establishment and inheritance of heterochromatin, the sole H3K9 HMTase belonging to SUV39 superfamily, Clr4, undergoes automethylation at K455 and K472. This automethylation releases Clr4 from its intrinsic autoinhibitory state, which is necessary for appropriate heterochromatin establishment and epigenetic stability (34). Besides being activated by automethylation, the HMTase activity of Clr4 is strongly promoted by H3K14 ubiquitination (H3K14Ub) (35, 36), which is catalyzed by the Clr4-containing Clr4-Rik1-Cul4 methyltransferase complex (CLRC) (37–39). The stimulatory effect of H3K14Ub on Clr4 is considerably more pronounced than that of the Clr4 automethylation (34, 40). Intriguingly, bioinformatics analysis suggested that Clr4 lacks a ubiquitin-binding domain, and experimental evidence indicates that it does not sense free ubiquitin (40). The mechanism underlying the substantial stimulatory effect of H3K14Ub on Clr4 activity remains unknown.

Here, we determined the crystal structures of Clr4 bound to ubiquitinated and unmodified H3 peptides at resolutions of 2.60 and 2.39 Å, respectively. Combined with the results of biochemical experiments, our findings reveal the molecular mechanism underlying the synergistic effects of affinity and turnover enhancement that results in significant stimulation by H3K14Ub. Specifically, compared with the faint increase in substrate recruitment induced by Clr4 automethylation, H3K14Ub substantially enhances the affinity of Clr4 for substrate. The ubiquitin constrained by the isopeptide bond linkage at the H3K14 position forms multivalent interactions with the Clr4 catalytic domain, stabilizing substrate binding and rapidly promoting H3K9me2/3 deposition. Moreover, our findings indicate that the previously reported automethylated state of Clr4 requires additional allosteric changes to transition into the “catalyzing state,” which is characterized by conformational changes in the compact αC-SET [Su(var), enhancer of zeste, trithorax]-insertion (SI) region, complete release of the autoregulatory loop (ARL), and retraction of the β9/10 loop. Furthermore, H3K14Ub facilitates substrate turnover by promoting the transition of Clr4 to a hyperactive catalyzing state, primarily through nascent interactions and conformational changes within the structural elements surrounding the β9/10 loop. Our work elucidates the continuous conformational changes occurring during the disinhibition of Clr4 via automethylation, substrate recruitment, and stimulation by H3K14Ub, thereby advancing our understanding of the regulation of H3K9me.

RESULTSFunctional interplay of Clr4 activation by H3K14Ub and automethylationOur work began with biochemical reconstitution of the H3K14Ub-mediated stimulation of H3K9me in vitro using full-length Clr4 (residues 1 to 490) or the Clr4 lysine methyltransferase (KMT) domain (residues 192 to 490) with truncation of the N-terminal CD (residues 1 to 70) and the disordered hinge region (residues 71 to 191) (Fig. 1B). H3K14 ubiquitinated and unmodified histone H3 N-terminal tail (residues 1 to 20) (hereafter referred to as Ub-H3t and H3t, respectively) were chemically synthesized as substrates (figs. S1A and S2, A and B). HMTase activity assay of full-length Clr4 and the KMT domain on H3 tail substrates indicated that H3K14Ub significantly increased the production rate of S-adenosylhomocysteine (SAH) (fig. S1, B and F). The rate of Clr4 automethylation is significantly slower than that of substrate methylation, rendering its contribution to SAH production negligible in the context of our assays (fig. S2F). The results validate the significant stimulatory effect of H3K14Ub on Clr4 methyltransferase activity (35, 40).Fig. 1.Biochemical reconstruction and global structures of complex KMT-Ub-H3 and KMT-H3.(A) Michaelis-Menten curves of the full-length Clr4 (without automethylation) methylation reaction with ubiquitinated (left) or unmodified (right) peptide as the substrate, and automethylation-mimicking Clr4 as the enzyme (n = 4, error bars indicate SD). (B) Construction of the complex KMT-Ub-H3 and KMT-H3 and their resulting crystal structures: KMT-H3 on the right and KMT-Ub-H3 on the left. Domains and subdomains in Clr4 (top), H3 (in magentas), and ubiquitin (in yellow) are distinguished by different colors, and the colors in this picture correspond to the colors of the surface in the following structure figures. (C and E) Global structure of KMT-H3 complex and KMT-Ub-H3 complex, the surface of domain KMT is colored through different subdomains, histone H3 is shown in magentas, and the Ub is shown in yellow. (D and F) The surface electrostatic diagram of KMT in KMT-H3 and KMT-Ub-H3 structures, the H3 and Ub are still displayed in cartoon, and the SAM is represented by sticks.Next, we explored the functional interplay between the activation of Clr4 methyltransferase activity by K455/K472 automethylation and H3K14Ub. Previous work (34) constructed Clr4 hypoactive mutant-K455R/K472R, which is unable to be automethylated and released from the ARL-mediated autoinhibition, and hyperactive mutants-K455A/K472A eliminating the interaction between ARL and the catalytic site or K455W/K472W being too bulky to fit into the catalytic site, hence destabilize the autoinhibited conformation. The average methyl transfer rate per micromolar Clr4 calculated directly from the end point quantity of SAH showed that automethylation of Clr4 (fig. S3, A and B) and automethylation-mimicking hyperactive mutations activate the catalysis of both unmodified and ubiquitinated substrates (fig. S1B). In turn, the average methyl transfer rate of automethylated Clr4 can be further elevated by H3K14Ub for approximately 50 folds (fig. S1B). In addition, immunoblotting of methylated Lys showed that the presence of neither H3t nor Ub-H3t had any discernible effect on the Clr4 automethylation level, implying independence between the two processes (fig. S1C). Collectively, the influences of the Clr4 automethylation and H3K14Ub are cumulative, collaborating to regulate Clr4 activity.The above measurements revealed that the activating effects of Clr4 automethylation and hyperactive mutations on methylation catalysis were significantly inferior to those of H3K14Ub. We conducted Michaelis-Menten kinetics experiments to analyze stimulatory effects quantitatively (Fig. 1A). The Clr4 automethylation–mimicking mutation resulted in only a 2.9-fold increase in substrate affinity [Michaelis constant (Km) = 26.5 ± 1.2 μM versus wild-type (WT) Km = 75.8 ± 6.5 μM] and an 8.5-fold enhancement of methylation turnover (kcat = 0.820 ± 0.017 min−1 versus WT kcat = 0.0969 ± 0.0052 min−1). In contrast, the H3K14Ub significantly enhanced both the substrate affinity by 135.6-fold (Km = 0.559 ± 0.052 μM versus unmodified Km = 75.8 ± 6.5 μM) and turnover by 188.8-fold (kcat = 18.3 ± 0.5 min−1 versus unmodified kcat = 0.0969 ± 0.0052 min−1) in H3K9me, resulting in a 1061.5-fold increase in the specific constant kcat/Km (32.8 min−1 μM−1) compared with that of the Clr4 automethylation–mimicking mutation (0.0309 min−1 μM−1). These results indicated that the stimulatory effects of the reaction by H3K14Ub are more pronounced than that of Clr4 automethylation in terms of substrate affinity and intrinsic Clr4 activity.Crystal structures of Clr4 bound to H3K14Ub or unmodified H3 peptideTo elucidate the structural mechanism of the significant stimulatory effect of H3K14Ub on Clr4, we cocrystallized the catalytic KMT domain [composed of the N-terminal subdomain (NT), pre-SET, SET catalytic core, and post-SET subdomains] with the H3K14 ubiquitinated or unmodified H3 N-terminal tail peptide in the presence of S-adenosylmethionine (SAM). However, our initial attempts to crystallize the complex using a chemically synthesized homogeneous isopeptide-linked ubiquitinated H3K14 tail as a substrate were unsuccessful. After extensive optimization, including testing various covalent linkage strategies for ubiquitin and truncation of the H3 peptide, we succeeded in obtaining a crystal structure when we used a substrate mimic with an H3 N-terminal tail peptide (residues 3 to 19) and ubiquitin attached to KC14 via a disulfide bond (Fig. 1B). This disulfide bond–mimicking strategy has been validated in previous ubiquitin-linked systems (41, 42). Specifically, K14 of H3 (3–19) was mutated to Cys to attach UbG76C, and K9 was replaced with norleucine (Nle), which leads to SAM-dependent high affinity to Clr4 catalytic pocket and was widely used in MTase structural studies (figs. S1D and S2, C and D) (43–45). Compared to the native K14-ubiquitinated substrate, the disulfide bond–linked substrate exhibited comparable affinity in the isothermal titration calorimetry (ITC) test dissociation constant (KD) = 133 ± 13 μM for the disulfide-bond substrate versus KD = 115 ± 18 μM for the isopeptide-bond substrate) (fig. S1E) and comparable stimulation efficiency (figs. S1F and S2E). Last, a 2.60-Å complex structure of the Clr4 KMT domain bound to H3 (3–19)K9Nle-K14C-UbG76C (fig. S2G), and a 2.39-Å complex structure of the Clr4 KMT domain bound to H3 (3–19)K9Nle-K14C without Ub modification (Fig. 1B) were resolved (hereafter abbreviated as the KMT-Ub-H3 and KMT-H3 structures, respectively) (figs. S4, C to H and S5, A to G and table S1).In these two complex structures, the H3-derived substrates and the SAM cofactor exhibit a 1:1:1 stoichiometry in their binding to the surface of the ovoid-shaped KMT domain. (Fig. 1, C and E). The KMT-Ub-H3 complex assembled into space group P212121 and KMT-H3 assembled into P22121. Each asymmetric unit of the two crystals contains two (P212121) copies and one (P22121) copy of the complex, respectively. For the crystal structure of the KMT-Ub-H3 complex, two copies of the complex adopt highly similar conformations and interfaces (fig. S5A). In our structures, the KMT domain adopts a global conformation similar to that of the apo form [Protein Data Bank (PDB): 6BOX] (34) with a root mean square deviation (RMSD) of ~0.37 Å across 221 pairs of Cα (fig. S4A). The H3 N-terminal tail peptide inserts into a histone-binding groove and straddles the SET subdomain of KMT, resulting in an interaction surface area of more than 2000 Å2. The section of the histone-binding groove near the catalytic active site is negatively charged since the acidic residue–rich SI region forms the outer “wall” of the groove and interacts with residues R8 to G13 of the H3 N-terminal tail (Fig. 1, D and F). The SAM cofactor is located on the larger side of the ovoid structure, embedded in the SAM pocket of the SET domain, with the post-SET subdomain covering above as a lid. Moreover, local allosteric changes are observed in certain structural elements such as the αC helices (residues 366 to 372), the SI region (residues 371 to 394), and the ARL (residues 453 to 472) (fig. S4, A and B).Notably, in the KMT-Ub-H3 structure, the Ub moiety is clearly visible and docks with the KMT domain on the opposite side of the SAM-binding pocket, forming extensive interfaces, which are 1542 and 1559 Å2 in two NCS units, respectively. The C-terminal tail of Ub extends to the histone-binding groove along a neutral-hydrophobic canyon, sandwiched between the acidic pre-SET and SI region (Fig. 1F).Conserved Clr4 residues in the histone-binding groove contribute to activity and selectivityThe N-terminal tail of histone H3 extensively interacts with the SET subdomain within the KMT domain. Most of these contacts involve residues A7 to KC14 of H3, which are situated within a narrow groove formed by the parallel acidic SI region, the C terminus of Clr4, and the initial segment (451YAGA454) of ARL (Fig. 2, A and B, and fig. S6, A and D). Within this groove, L382, D384, and D386 in the SI region form hydrogen bonds with the backbone of the H3 tail. The guanidine group of H3 residue R8 is further stabilized by hydrogen bonds with the side chains of D371 and D384 of Clr4. In addition, the carbonyl oxygen of A367 and the hydroxy group of T395 interact with the guanidine group of R8, forming a quadrangular clamp structure that secures this residue (Fig. 2A and fig. S6, B and E). These interactions contribute to the substrate H3K9 specificity of Clr4, as the enzyme selectively recognizes an RK core motif within the histone tail. H3 KNle9 is positioned within the catalytic pocket of Clr4, where its backbone along with the backbone of S10 forms main-chain hydrogen bonds with the SI region. The carboxy group of D386 further forms polar contacts with the hydroxy groups of H3 residues S10 and T11 (fig. S6, C and F).Fig. 2.Conserved interactions between Clr4 KMT and H3 N-terminal peptide.(A) Residues that form quadrangular clamp construct surrounding R8 and interact with the KNle9 backbone. H3 is in magentas, and residues of Clr4 in the KMT-Ub-H3 are represented by cyan sticks, while in KMT-H3 are green. (B) Clr4 residues that contact with the H3 10STGG13 in the KMT-Ub-H3 structure and KMT-H3 structure. (C) Conformation of SAM and KNle9 binding with the Clr4 catalytic pocket/site in KMT-Ub-H3 structure. (D) Evolutionary conservation of residues in Clr4 between homologous proteins, conserved residues appear redder on the surface of KMT. The conserved SI region and post-SET C terminus are marked by the yellow circle. The picture on the right shows the evolutionary tree and sequence alignment between SUV39 family proteins, the conserved SI region and catalytic pocket are marked by orange and red triangles. (E) Comparison of H3 conformation among structures of Suv39 proteins binding with H3 N-terminal tail. H3 in our structure is in magenta, while the H3 binding with DIM5 (PDB: 1PEG) is in purple, binding with G9a (PDB: 5JIY) is in pale green, and binding with GLP1 (PDB: 2RFI) is in wheat. (F and G) Chemiluminescence HMTase activity assay measuring SAH production during the WT Clr4 or Clr4 accommodating mutations in histone-binding groove or catalytic site catalyzing substrate (F) Ub-H3t and (G) H3t methylation (n = 3, error bars indicate SD; ***, 0.0001 < P < 0.001; ****, P < 0.0001), SDS-PAGE gels of the reactions are provided in Supplementary figures.The side chain of histone H3 K9Nle inserts into a hydrophobic tunnel and attacks the S-methyl group of SAM (Fig. 2C and fig. S6G), which is consistent with the active site conformations of other H3K9-specific HMTase. Four aromatic residues (Y357, F383, F449, and Y451) within the aromatic catalytic site surrounding the side chain of K9Nle are highly conserved among various Clr4 homologs in eumycetes and higher living organisms, such as Arabidopsis thaliana, Drosophila melanogaster, Mus musculus, and Homo sapiens (Fig. 2, C and D, and figs. S6, G and H). Besides the classic catalytic site, the acidic SI region is also conserved among H3K9-specific HMTases (Fig. 2D). Compared with the substrate-binding structure of Clr4 homologs like DIM5 in Neurospora crassa (46), G9a (47), and GLP1 (48) in H sapiens, the conformation of H3 residues 7 to 13 in our structure is similar, and the SI region adopts a concerted recognition behavior when it is tightly interdigitated with residues 7 to 13 of the H3-tail peptide (Fig. 2E). This observation suggests that the H3K9-specific HMTases preserve H3K9 recognition patterns and selectivity across various species throughout evolution.To validate the interfaces between Clr4 and H3, we introduced single or double alanine mutations at key interface residues of Clr4 and tested the catalytic behavior of these mutants using unmodified or ubiquitinated H3 N-terminal tail (1 to 20) (H3t and Ub-H3t). We found that alanine mutations of either SI region residues (D371, D384, and D386), whose side chains are involved in hydrogen bond formation, or residues at the catalytic site (Y357, F383, F449, and Y451) weakened the methylation activity, regardless of whether the substrate was unmodified H3 or ubiquitinated H3 (Fig. 2, F and G, and fig. S7, A and B). Therefore, our structural and biochemical results indicate that the highly conserved interactions between the Clr4 histone–binding groove and H3 (residues 7 to 14) are critical for selective substrate binding and methyltransferase activity.Clr4-Ub interfaces are crucial for the stimulationWe next focused on the structure of the KMT-Ub-H3 complex, in which the Ub moiety resembles a balloon tied to a string (Fig. 3A and fig. S5A). The interfaces between Ub and Clr4 can be categorized into three distinct regions (Fig. 3A): First, the canonical Ub interacting spots, including residues L8, I44, V70, and the aliphatic portion of the K6 side chain, create a hydrophobic pocket that accommodates F256 on Clr4 loop 2. Clr4 residue F256 is further stabilized by π-π interaction with Ub residue H68 (Fig. 3B and fig. S8A). Second, R42 and Q49 of Ub are inserted into an acidic pocket of the pre-SET domain. R42 forms salt bridge interactions with Clr4 D281 and forms a hydrogen bond with S258, whereas Q49 contacts with Clr4 D280 (Fig. 3C and fig. S8, B and D). The third interface is at the C-terminal tail of Ub (residues 70 to 76), whose main chain along with the side chains of V70 and L73, lies in the neutral-hydrophobic canyon, which starts from F256 of Clr4 and extends to the histone-binding groove. Meanwhile, the side chain of R72 is fastened to the acidic pre-SET base through two salt bridge interactions with D280 and D281 (Fig. 3D and fig. S8, C and E).Fig. 3.Multivalent binding between ubiquitin and Clr4 KMT.(A) Multivalent binding behavior between ubiquitin and Clr4 KMT; ubiquitin is shown in yellow, Clr4 is shown in cyan, while the key acidic regions are highlighted in red. (B to D) Close-up views of three interfaces (I, II, and III) between Ub and KMT. (E) HMTase activity assay of WT Clr4 or Clr4 accommodating mutations in ubiquitin interfaces working on substrate Ub-H3t (top) and H3t (bottom). (F) HMTase activity assay of WT Clr4 working on Ub-mutated Ub-H3t. (G) KD value measured by ITC experiment performed with WT Clr4 KMT and Ub interface–mutated Ub-H3t. (H) Silencing assays of mat2P::ade6+ on low-adenine medium (Low ade) to assess ade6+ silencing in the indicated genotypes involving Clr4 and Clr4 mutants. ns, no significance; ***, 0.0001 < P < 0.001; ****, P < 0.0001.The importance of the Clr4-Ub interface was assessed by evaluating the impact of Clr4 mutations. The Clr4 F256A mutation, which diminishes hydrophobic interactions, reduced the HMTase activity of Ub-H3t by 60% (Fig. 3E, top, and fig. S7C). Mutations in the pre-SET acidic pocket (D280A/D281A) strongly impaired HMTase activity by 82%. In contrast, these Clr4 mutants (F256A and D280A/D281A) did not affect H3K9me activity on unmodified H3t (Fig. 3E, bottom, and fig. S7C). We also conducted thermoshift experiments and revealed that the melting temperature (ΔTm) of these mutants significantly decreased when binding to the ubiquitinated H3 tail, indicating the importance of these interfaces for ubiquitin binding and thermodynamic stabilization of Clr4 (fig. S8G). Furthermore, Clr4 methylation activity was noticeably decreased on the Ub-H3t substrate bearing mutations in the Ub R42 anchor (R42A), I44/L8 patch (L8R/H68A, L69A/V70A), and C-terminal tail (L71A/L73A, R72A/R74A). Mutation of R42 led to the most significant decrease in the ability to be methylated (92%), while that of the other four mutants was reduced by 22~89% (Fig. 3F and figs. S7D and S9, A to E). Further measurement of the binding affinity between the above Ub-H3t variants and Clr4 by ITC revealed KD values ranging from 0.239 to 1.46 μM, which are much lower than that of the WT substrate (0.115 ± 0.018 μM) (Fig. 3G and fig. S8F). Collectively, these findings indicate that the interfaces between Clr4 and H3K14 ubiquitin are crucial for substrate recognition, enzyme stabilization, and methylation stimulation.Validation of the key roles of the Clr4-H3 and Clr4-Ub interfaces in vivoTo validate the pivotal roles of the Clr4-H3 and Clr4-Ub interfaces identified in our crystal structure in the context of Clr4 function in vivo, we constructed S. pombe cells that express endogenous FLAG-tagged Clr4 mutant proteins with single or double substitutions at the Clr4-H3 (Y357A and D384A) and Clr4-Ub interfaces (F256A and D280A/D281A) (fig. S10A). Clr4-mediated heterochromatin deposition and gene silencing were examined using an ade6+ reporter transgene inserted within the heterochromatic mat locus. The propagation of surrounding heterochromatin leads to the silencing of ade6+, resulting in red colonies on low-adenine plates. Our experiments showed that disruption of the catalytic pocket (Y357A) and histone-binding groove (D384A) completely abolished the heterochromatin deposition, yielding colonies as white as those observed with the previously reported Clr4 dead mutant (Y451N) (Fig. 3H) (40). This outcome underscores the essential roles of both the histone-binding groove and Clr4 methylation activity in Clr4-mediated heterochromatin deposition.In addition, mutations at the Clr4-Ub interfaces (F256A and D280A/D281A) resulted in a significant decrease in silencing, as evidenced by light pink to white colonies (Fig. 3H). Consistent with the observed phenotype, H3K9me3 levels in nuclear chromatin were substantially decreased in the Clr4 knockout/dead cells as well as in strains with histone binding–eliminated Clr4 mutants (D384A and Y357A), whereas mutations (D280A/D281A and F256A) in the Ub-binding interface also resulted in a reduction in H3K9me levels (fig. S10, B and C). Besides the heterochromatin deposition at mat2P::ade6+ reporter, other constitutive heterochromatic domains were also investigated. Reverse transcription quantitative polymerase chain reaction (qPCR) was performed to measure the transcript levels of the pericentromeric dg/dh repeats and subtelomeric marker gene tlh1 in S. pombe strains mentioned above. Transcript levels of dh/dg and tlh1 were activated by three orders of magnitude relative to WT in knockout strains, Clr4 catalytic pocket mutant Y357A, Y451N strains, and histone-binding groove mutant D384A strains. Meanwhile, the Ub-binding interface mutated (F256A and D280A/D281A) strains also significantly impair the suppression of transcription, as the transcript level of dh/dg and tlh1 increased by two orders of magnitude (fig. S10D). These findings indicate that the Clr4-Ub interfaces are important for the regulation of heterochromatin deposition and transcription suppression, while the disruption of Ub interactions still allows minimal heterochromatin formation.Essentiality of the H3K14-specific junction for ubiquitin-mediated stimulationNotably, the interaction between Clr4 KMT and free ubiquitin was undetectable in the ITC experiment (fig. S8H), which is consistent with previous findings that Clr4 does not sense free ubiquitin to facilitate H3K9me (Fig. 4A and fig. S11A) (40). These findings suggest that the stimulatory effect of ubiquitin is contingent on its linkage to H3K14. We therefore hypothesized that the binding of ubiquitin to KMT is constrained by the H3K14 junction, which in turn enhances binding affinity by stabilizing the enzyme-substrate complex. In alignment with this hypothesis, the presence of free ubiquitin did not influence the change in the ΔTm of the complex in the thermal shift assay (Fig. 4B and figs. S12A and S11, C and D). Furthermore, extending the C-terminal tail of ubiquitin with an AEEA {[2- (2-aminoethoxy) ethoxy] acetic acid} linker (fig. S12C) led to an approximately 25% reduction in methyltransferase activity with the Ub-AEEA-H3t substrate (Fig. 4A) and a 1.12°C decrease in ΔTm compared with that of the native Ub-H3t substrate (Fig. 4C and figs. S11B and S12B). This result suggested that a less constrained ubiquitin motif resulted in reduced thermodynamic stabilization and consequently weaker activation of Clr4.Fig. 4.Biochemical experiment about H3-Ub junction.(A) HMTase activity assay of WT Clr4 working on the substrates with or without Ub and AEEA-linked Ub (n = 5, error bars indicate SD). (B) Thermo-shift assay working on Clr4 KMT binding with Ub-H3t (KNle9), H3t (KNle9), H3t (KNle9) with free Ub. The ΔTm was determined by the first derivative of the barycentric mean (BCM) of fluorescence intensity with respect to temperature (dBCM/dT) (n = 3, error bars indicate SD). (C) Thermo-shift assay performed with Clr4 KMT and Ub-H3t, Ub-AEEA-H3t, respectively (n = 3, error bars indicate SD). (D) Scheme of “caliper construct” functioning in the recognition of K14-ubiquitinated H3 N-terminal tail. The catalytic site is highlighted by a red circle, and the intersection point of the histone-binding groove and neutral-hydrophobic canyon is labeled by an orange dot. The caliper shows the approximate distance between the catalytic site and the intersection point. aa, amino acid. (E) HMTase activity assay of WT Clr4 working on the substrates that ubiquitin linked to residue 13 or residue 16 of H3 N-terminal tail (n = 5, error bars indicate SD). (F) HMTase activity assay of WT Clr4 catalyzing NCP substrate with H3K14/K18/K23Ub or without modification (n = 3, error bars indicate SD). ****, P < 0.0001.In our KMT-Ub-H3 complex structure, the distance from the catalytic site to the point where the histone-binding groove intersects with the neutral-hydrophobic canyon is approximately 16 Å. This distance is slightly shorter than that of an extended hexapeptide (~19 Å, corresponding to residues 9KSTGGK14), as the sequential Cα−Cα distance is consistently restricted to ~3.8 Å (fig. S12E). Notably, the K14 junction is located at this intersection, allowing the histone-binding groove to effectively accommodate the extended configuration of H3 residues K9 to K14 while enabling the C-terminal tail of Ub to reside precisely within the neutral-hydrophobic canyon (Fig. 4D). This recognition mechanism implies that the stimulatory effect is highly sensitive to the position of the Ub site. Attempts to fine-tune the Ub site, either upstream or downstream of H3K14, led to impaired methylation activity with the Ub-modified H3 N-terminal tail (Fig. 4E and fig. S13A), despite only shifting the junction by two amino acids forward or backward (H3tGC12-Ub, H3tPC16-Ub; fig. S13, A and D). This finding highlights the low tolerance for alterations in the substrate Ub site.Furthermore, we investigated biologically relevant ubiquitinated H3 substrates, including K18 and K23-ubiquitinated nucleosomes involved in DNMT1 recruitment and activation in the context of hemimethylated DNA inheritance (NCPH3K14Ub, NCPH3K18Ub, and NCPH3K23Ub; fig. S13, B and E to G) (49–51). HMTase assay revealed that the methylation rates of unmodified nucleosome core particle (NCP), NCPH3K18Ub, NCPH3K23Ub were much lower than that of NCPH3K14Ub (Fig. 4F). H3K18ub is also activating albeit to a lesser degree than H3K14ub. This phenomenon closely parallels the previously reported H3K9me (G9a, SUV39H1/2)-DNA methylation (DNMT1)–H3K18 Ub (UHRF1) cross-talk network in H. sapiens (52–55). Especially for Clr4 homologs SUV39H1/2: in DNMT-inhibited cells, UHRF1 catalyzes H3K18 Ub at DNA hypomethylation region, and then the H3K18Ub can recruit SUV39H1/H2 and promote the H3K9me (54). The electrophoretic mobility shift assay results indicated that almost all NCPH3K14Ub was bound to Clr4 at a Clr4 concentration of 0.5 μM, whereas a portion of NCPH3K18Ub, along with unmodified NCP, persisted as free NCP until the concentration of Clr4 was increased to 1 to 2 μM (fig. S12D). These results imply that Clr4 is selectively recruited to H3K14-ubiquitinated nucleosomes.Allosteric effects during substrate turnover and its stimulation by H3K14UbNext, we compared the structures of Clr4 in the KMT-H3, KMT-Ub-H3, and apo form to analyze the conformational changes mediated by histone H3 and Ub, respectively, which underlie the pronounced difference in Clr4 substrate turnover (kcat) observed during catalysis of unmodified and H3K14Ub-modified substrates. We first superimposed our KMT-H3 complex structure with the previously reported Clr4 KMT apo conformation structure (PDB: 6BOX) to identify key conformational changes required for substrate engagement. Structural rearrangements of Clr4 were observed in four main regions: the ARL (residues 453 to 472), the helix αC-SI region (residues 365 to 395), the Clr4 C terminus (residues 486 to 490), and the β9/10 loop (residues 423 to 431) (fig. S14, A to B). Once H3 binds, the ARL, which previously occluded the active site (34), is released to expose the active site and accommodate the binding of the H3 N-terminal tail (Fig. 5B). The Clr4 C terminus undergoes an ~75° rotation with G486 as the vertex, positioning itself toward the substrate (Fig. 5B). Helix αC (residues 364 to 472) is rotated approximately 9° toward the substrate on the axis of T363, generating a hydrogen bond between Clr4 D371 and H3 R8 (Fig. 5A). The β-hairpin loop within the SI region (residues 381 to 394) shifts toward the histone-binding groove by 1.62-Å backbone RMSD to interact with H3 (Fig. 5C). Meanwhile, the β9/10 loop, located at the center of the neutral-hydrophobic canyon and in close proximity to the β-hairpin loop (residues 385 to 394) of SI region, undergoes retraction in the catalyzing state, which facilitates the formation of a nascent salt bridge interaction between Clr4 residues R428 and D390 on the β-hairpin loop in SI region (Fig. 5D). These observations suggest that specific conformational changes within structural elements of Clr4 are required for the positioning and alignment of H3 for methylation.Fig. 5.Allostery effects during substrate turnover and stimulation by ubiquitin.(A) Conformation changes of helix αC between apo structure and our KMT-H3 structure. Two structures are superposed by KMT alignment. Structural elements in the apo structure are colored in blue, and those in the KMT-H3 structure are colored in pink. The directions of allostery are highlighted by magenta arrows. Clr4 D371 and H3 R8 forming nascent interactions are shown as sticks in map density. (B) Conformation changes of ARL and Clr4 C terminus between apo structure and our KMT-H3 structure. (C and D) Conformation changes of β9/10 loop and SI region between apo structure and our KMT-H3 structure. Clr4 D390 and R428 forming nascent interactions are shown as sticks in map density in (D). (E) Flipping of Clr4 H425 once the H3K14 is ubiquitinated. Key residues forming nascent contacts are represented by sticks in map density. KMT-H3 structure and our KMT-Ub-H3 structure are superposed by KMT alignment. The KMT-H3 structure is colored in pink, and that in the KMT-Ub-H3 structure is colored in cyan. (F and H) Michaelis-Menten curves of the full-length Clr4 accommodate (F) H425A and (H) F427A mutations catalyzing ubiquitinated peptide methylation (n = 4, error bars indicate SD). (G) Clash between Ub L8 and Clr4 F427 when superposing apo form Clr4 into the KMT-Ub-H3 structure. Ubiquitin is colored in yellow, and L8 is represented by yellow sticks and surface.To further understand how H3K14Ub drives the establishment of a hyperactive state of Clr4, we compared the KMT-H3 complex structure with the KMT-Ub-H3 complex structure to investigate the conformational change induced by H3K14Ub, which revealed structural changes in Clr4 at the β9/10 loop (residues 423 to 431), the beginning of the pre-SET subdomain (residues 254 to 268), and the onset of the SI region (residues 372 to 378), as well as structural changes at H3 residues 15 to 19 (fig. S14, C and D). Specifically, H425 in the β9/10 loop, which is well resolved in both the KMT-H3 and KMT-Ub-H3 structures, undergoes a 180° flip in the KMT-Ub-H3 structure. This flip allows it to engage in a π-π interaction with the aryl group of pre-SET residue Y305 and form a polar interaction with the backbone carbonyl oxygen of E306 (Fig. 5E). This interaction may stabilize the retracted conformation of the β9/10 loop in the ubiquitin-induced hyperactive state. Further experiments involving the Clr4 H425A mutation demonstrated a significant reduction in kcat for the Ub-H3t substrate, with an approximately fivefold decrease (kcat = 3.85 ± 0.171 min−1 versus kcat = 18.3 ± 0.5 min−1 for WT). This mutation did not diminish the substrate recruitment (Km = 0.254 ± 0.048 μM versus Km = 0.559 ± 0.052 μM for WT) or affect methylation activity with unmodified H3t (Fig. 5F and fig. S14E). These results suggest that the allostery triggered by H3K14Ub and the Clr4 β9/10 loop, along with the induced interactions between this loop and the pre-SET region, play essential roles in enhancing enzymatic turnover.To identify the requisite conformational alterations that Clr4 undergoes upon binding to the H3K14Ub substrate, we compared the inactive Clr4 KMT apo structure (PDB: 6BOX) (34) with the hyperactive KMT-Ub-H3 structure. This comparison revealed a steric clash between the Ub L8 loop and apo-form Clr4 residue F427 (Fig. 5G). In both the H3-KMT and H3-Ub-KMT structures, F427 is in a retracted position, forming a nascent hydrophobic pocket that accommodates the Ub L8 loop. This arrangement, in turn, restricts the β9/10 loop to the retracted conformation, which may be essential for the hyperactive state of the H3-Ub-KMT structure (Fig. 5G). Further analysis revealed that the Clr4 F427A mutation, which reduces the spatial size of the side chain, results in an approximately one-third reduction in kcat (kcat = 6.66 ± 0.33 min−1 versus kcat = 18.3 ± 0.5 min−1 of WT), while it does not significantly affect Km (Km = 0.528 ± 0.092 μM versus Km = 0.559 ± 0.052 μM of WT) or basal methylation activity with unmodified H3t (Fig. 5H and fig. S14E). These findings support the hypothesis that the newly formed interaction between Clr4 F427 and Ub is critical for driving H3K14Ub-mediated activation of enzymatic turnover.

Functional interplay of Clr4 activation by H3K14Ub and automethylationOur work began with biochemical reconstitution of the H3K14Ub-mediated stimulation of H3K9me in vitro using full-length Clr4 (residues 1 to 490) or the Clr4 lysine methyltransferase (KMT) domain (residues 192 to 490) with truncation of the N-terminal CD (residues 1 to 70) and the disordered hinge region (residues 71 to 191) (Fig. 1B). H3K14 ubiquitinated and unmodified histone H3 N-terminal tail (residues 1 to 20) (hereafter referred to as Ub-H3t and H3t, respectively) were chemically synthesized as substrates (figs. S1A and S2, A and B). HMTase activity assay of full-length Clr4 and the KMT domain on H3 tail substrates indicated that H3K14Ub significantly increased the production rate of S-adenosylhomocysteine (SAH) (fig. S1, B and F). The rate of Clr4 automethylation is significantly slower than that of substrate methylation, rendering its contribution to SAH production negligible in the context of our assays (fig. S2F). The results validate the significant stimulatory effect of H3K14Ub on Clr4 methyltransferase activity (35, 40).Fig. 1.Biochemical reconstruction and global structures of complex KMT-Ub-H3 and KMT-H3.(A) Michaelis-Menten curves of the full-length Clr4 (without automethylation) methylation reaction with ubiquitinated (left) or unmodified (right) peptide as the substrate, and automethylation-mimicking Clr4 as the enzyme (n = 4, error bars indicate SD). (B) Construction of the complex KMT-Ub-H3 and KMT-H3 and their resulting crystal structures: KMT-H3 on the right and KMT-Ub-H3 on the left. Domains and subdomains in Clr4 (top), H3 (in magentas), and ubiquitin (in yellow) are distinguished by different colors, and the colors in this picture correspond to the colors of the surface in the following structure figures. (C and E) Global structure of KMT-H3 complex and KMT-Ub-H3 complex, the surface of domain KMT is colored through different subdomains, histone H3 is shown in magentas, and the Ub is shown in yellow. (D and F) The surface electrostatic diagram of KMT in KMT-H3 and KMT-Ub-H3 structures, the H3 and Ub are still displayed in cartoon, and the SAM is represented by sticks.Next, we explored the functional interplay between the activation of Clr4 methyltransferase activity by K455/K472 automethylation and H3K14Ub. Previous work (34) constructed Clr4 hypoactive mutant-K455R/K472R, which is unable to be automethylated and released from the ARL-mediated autoinhibition, and hyperactive mutants-K455A/K472A eliminating the interaction between ARL and the catalytic site or K455W/K472W being too bulky to fit into the catalytic site, hence destabilize the autoinhibited conformation. The average methyl transfer rate per micromolar Clr4 calculated directly from the end point quantity of SAH showed that automethylation of Clr4 (fig. S3, A and B) and automethylation-mimicking hyperactive mutations activate the catalysis of both unmodified and ubiquitinated substrates (fig. S1B). In turn, the average methyl transfer rate of automethylated Clr4 can be further elevated by H3K14Ub for approximately 50 folds (fig. S1B). In addition, immunoblotting of methylated Lys showed that the presence of neither H3t nor Ub-H3t had any discernible effect on the Clr4 automethylation level, implying independence between the two processes (fig. S1C). Collectively, the influences of the Clr4 automethylation and H3K14Ub are cumulative, collaborating to regulate Clr4 activity.The above measurements revealed that the activating effects of Clr4 automethylation and hyperactive mutations on methylation catalysis were significantly inferior to those of H3K14Ub. We conducted Michaelis-Menten kinetics experiments to analyze stimulatory effects quantitatively (Fig. 1A). The Clr4 automethylation–mimicking mutation resulted in only a 2.9-fold increase in substrate affinity [Michaelis constant (Km) = 26.5 ± 1.2 μM versus wild-type (WT) Km = 75.8 ± 6.5 μM] and an 8.5-fold enhancement of methylation turnover (kcat = 0.820 ± 0.017 min−1 versus WT kcat = 0.0969 ± 0.0052 min−1). In contrast, the H3K14Ub significantly enhanced both the substrate affinity by 135.6-fold (Km = 0.559 ± 0.052 μM versus unmodified Km = 75.8 ± 6.5 μM) and turnover by 188.8-fold (kcat = 18.3 ± 0.5 min−1 versus unmodified kcat = 0.0969 ± 0.0052 min−1) in H3K9me, resulting in a 1061.5-fold increase in the specific constant kcat/Km (32.8 min−1 μM−1) compared with that of the Clr4 automethylation–mimicking mutation (0.0309 min−1 μM−1). These results indicated that the stimulatory effects of the reaction by H3K14Ub are more pronounced than that of Clr4 automethylation in terms of substrate affinity and intrinsic Clr4 activity.

Functional interplay of Clr4 activation by H3K14Ub and automethylation

Our work began with biochemical reconstitution of the H3K14Ub-mediated stimulation of H3K9me in vitro using full-length Clr4 (residues 1 to 490) or the Clr4 lysine methyltransferase (KMT) domain (residues 192 to 490) with truncation of the N-terminal CD (residues 1 to 70) and the disordered hinge region (residues 71 to 191) (Fig. 1B). H3K14 ubiquitinated and unmodified histone H3 N-terminal tail (residues 1 to 20) (hereafter referred to as Ub-H3t and H3t, respectively) were chemically synthesized as substrates (figs. S1A and S2, A and B). HMTase activity assay of full-length Clr4 and the KMT domain on H3 tail substrates indicated that H3K14Ub significantly increased the production rate of S-adenosylhomocysteine (SAH) (fig. S1, B and F). The rate of Clr4 automethylation is significantly slower than that of substrate methylation, rendering its contribution to SAH production negligible in the context of our assays (fig. S2F). The results validate the significant stimulatory effect of H3K14Ub on Clr4 methyltransferase activity (35, 40).

Biochemical reconstruction and global structures of complex KMT-Ub-H3 and KMT-H3.

(A) Michaelis-Menten curves of the full-length Clr4 (without automethylation) methylation reaction with ubiquitinated (left) or unmodified (right) peptide as the substrate, and automethylation-mimicking Clr4 as the enzyme (n = 4, error bars indicate SD). (B) Construction of the complex KMT-Ub-H3 and KMT-H3 and their resulting crystal structures: KMT-H3 on the right and KMT-Ub-H3 on the left. Domains and subdomains in Clr4 (top), H3 (in magentas), and ubiquitin (in yellow) are distinguished by different colors, and the colors in this picture correspond to the colors of the surface in the following structure figures. (C and E) Global structure of KMT-H3 complex and KMT-Ub-H3 complex, the surface of domain KMT is colored through different subdomains, histone H3 is shown in magentas, and the Ub is shown in yellow. (D and F) The surface electrostatic diagram of KMT in KMT-H3 and KMT-Ub-H3 structures, the H3 and Ub are still displayed in cartoon, and the SAM is represented by sticks.

Next, we explored the functional interplay between the activation of Clr4 methyltransferase activity by K455/K472 automethylation and H3K14Ub. Previous work (34) constructed Clr4 hypoactive mutant-K455R/K472R, which is unable to be automethylated and released from the ARL-mediated autoinhibition, and hyperactive mutants-K455A/K472A eliminating the interaction between ARL and the catalytic site or K455W/K472W being too bulky to fit into the catalytic site, hence destabilize the autoinhibited conformation. The average methyl transfer rate per micromolar Clr4 calculated directly from the end point quantity of SAH showed that automethylation of Clr4 (fig. S3, A and B) and automethylation-mimicking hyperactive mutations activate the catalysis of both unmodified and ubiquitinated substrates (fig. S1B). In turn, the average methyl transfer rate of automethylated Clr4 can be further elevated by H3K14Ub for approximately 50 folds (fig. S1B). In addition, immunoblotting of methylated Lys showed that the presence of neither H3t nor Ub-H3t had any discernible effect on the Clr4 automethylation level, implying independence between the two processes (fig. S1C). Collectively, the influences of the Clr4 automethylation and H3K14Ub are cumulative, collaborating to regulate Clr4 activity.

The above measurements revealed that the activating effects of Clr4 automethylation and hyperactive mutations on methylation catalysis were significantly inferior to those of H3K14Ub. We conducted Michaelis-Menten kinetics experiments to analyze stimulatory effects quantitatively (Fig. 1A). The Clr4 automethylation–mimicking mutation resulted in only a 2.9-fold increase in substrate affinity [Michaelis constant (Km) = 26.5 ± 1.2 μM versus wild-type (WT) Km = 75.8 ± 6.5 μM] and an 8.5-fold enhancement of methylation turnover (kcat = 0.820 ± 0.017 min−1 versus WT kcat = 0.0969 ± 0.0052 min−1). In contrast, the H3K14Ub significantly enhanced both the substrate affinity by 135.6-fold (Km = 0.559 ± 0.052 μM versus unmodified Km = 75.8 ± 6.5 μM) and turnover by 188.8-fold (kcat = 18.3 ± 0.5 min−1 versus unmodified kcat = 0.0969 ± 0.0052 min−1) in H3K9me, resulting in a 1061.5-fold increase in the specific constant kcat/Km (32.8 min−1 μM−1) compared with that of the Clr4 automethylation–mimicking mutation (0.0309 min−1 μM−1). These results indicated that the stimulatory effects of the reaction by H3K14Ub are more pronounced than that of Clr4 automethylation in terms of substrate affinity and intrinsic Clr4 activity.

Crystal structures of Clr4 bound to H3K14Ub or unmodified H3 peptideTo elucidate the structural mechanism of the significant stimulatory effect of H3K14Ub on Clr4, we cocrystallized the catalytic KMT domain [composed of the N-terminal subdomain (NT), pre-SET, SET catalytic core, and post-SET subdomains] with the H3K14 ubiquitinated or unmodified H3 N-terminal tail peptide in the presence of S-adenosylmethionine (SAM). However, our initial attempts to crystallize the complex using a chemically synthesized homogeneous isopeptide-linked ubiquitinated H3K14 tail as a substrate were unsuccessful. After extensive optimization, including testing various covalent linkage strategies for ubiquitin and truncation of the H3 peptide, we succeeded in obtaining a crystal structure when we used a substrate mimic with an H3 N-terminal tail peptide (residues 3 to 19) and ubiquitin attached to KC14 via a disulfide bond (Fig. 1B). This disulfide bond–mimicking strategy has been validated in previous ubiquitin-linked systems (41, 42). Specifically, K14 of H3 (3–19) was mutated to Cys to attach UbG76C, and K9 was replaced with norleucine (Nle), which leads to SAM-dependent high affinity to Clr4 catalytic pocket and was widely used in MTase structural studies (figs. S1D and S2, C and D) (43–45). Compared to the native K14-ubiquitinated substrate, the disulfide bond–linked substrate exhibited comparable affinity in the isothermal titration calorimetry (ITC) test dissociation constant (KD) = 133 ± 13 μM for the disulfide-bond substrate versus KD = 115 ± 18 μM for the isopeptide-bond substrate) (fig. S1E) and comparable stimulation efficiency (figs. S1F and S2E). Last, a 2.60-Å complex structure of the Clr4 KMT domain bound to H3 (3–19)K9Nle-K14C-UbG76C (fig. S2G), and a 2.39-Å complex structure of the Clr4 KMT domain bound to H3 (3–19)K9Nle-K14C without Ub modification (Fig. 1B) were resolved (hereafter abbreviated as the KMT-Ub-H3 and KMT-H3 structures, respectively) (figs. S4, C to H and S5, A to G and table S1).In these two complex structures, the H3-derived substrates and the SAM cofactor exhibit a 1:1:1 stoichiometry in their binding to the surface of the ovoid-shaped KMT domain. (Fig. 1, C and E). The KMT-Ub-H3 complex assembled into space group P212121 and KMT-H3 assembled into P22121. Each asymmetric unit of the two crystals contains two (P212121) copies and one (P22121) copy of the complex, respectively. For the crystal structure of the KMT-Ub-H3 complex, two copies of the complex adopt highly similar conformations and interfaces (fig. S5A). In our structures, the KMT domain adopts a global conformation similar to that of the apo form [Protein Data Bank (PDB): 6BOX] (34) with a root mean square deviation (RMSD) of ~0.37 Å across 221 pairs of Cα (fig. S4A). The H3 N-terminal tail peptide inserts into a histone-binding groove and straddles the SET subdomain of KMT, resulting in an interaction surface area of more than 2000 Å2. The section of the histone-binding groove near the catalytic active site is negatively charged since the acidic residue–rich SI region forms the outer “wall” of the groove and interacts with residues R8 to G13 of the H3 N-terminal tail (Fig. 1, D and F). The SAM cofactor is located on the larger side of the ovoid structure, embedded in the SAM pocket of the SET domain, with the post-SET subdomain covering above as a lid. Moreover, local allosteric changes are observed in certain structural elements such as the αC helices (residues 366 to 372), the SI region (residues 371 to 394), and the ARL (residues 453 to 472) (fig. S4, A and B).Notably, in the KMT-Ub-H3 structure, the Ub moiety is clearly visible and docks with the KMT domain on the opposite side of the SAM-binding pocket, forming extensive interfaces, which are 1542 and 1559 Å2 in two NCS units, respectively. The C-terminal tail of Ub extends to the histone-binding groove along a neutral-hydrophobic canyon, sandwiched between the acidic pre-SET and SI region (Fig. 1F).

Crystal structures of Clr4 bound to H3K14Ub or unmodified H3 peptide

To elucidate the structural mechanism of the significant stimulatory effect of H3K14Ub on Clr4, we cocrystallized the catalytic KMT domain [composed of the N-terminal subdomain (NT), pre-SET, SET catalytic core, and post-SET subdomains] with the H3K14 ubiquitinated or unmodified H3 N-terminal tail peptide in the presence of S-adenosylmethionine (SAM). However, our initial attempts to crystallize the complex using a chemically synthesized homogeneous isopeptide-linked ubiquitinated H3K14 tail as a substrate were unsuccessful. After extensive optimization, including testing various covalent linkage strategies for ubiquitin and truncation of the H3 peptide, we succeeded in obtaining a crystal structure when we used a substrate mimic with an H3 N-terminal tail peptide (residues 3 to 19) and ubiquitin attached to KC14 via a disulfide bond (Fig. 1B). This disulfide bond–mimicking strategy has been validated in previous ubiquitin-linked systems (41, 42). Specifically, K14 of H3 (3–19) was mutated to Cys to attach UbG76C, and K9 was replaced with norleucine (Nle), which leads to SAM-dependent high affinity to Clr4 catalytic pocket and was widely used in MTase structural studies (figs. S1D and S2, C and D) (43–45). Compared to the native K14-ubiquitinated substrate, the disulfide bond–linked substrate exhibited comparable affinity in the isothermal titration calorimetry (ITC) test dissociation constant (KD) = 133 ± 13 μM for the disulfide-bond substrate versus KD = 115 ± 18 μM for the isopeptide-bond substrate) (fig. S1E) and comparable stimulation efficiency (figs. S1F and S2E). Last, a 2.60-Å complex structure of the Clr4 KMT domain bound to H3 (3–19)K9Nle-K14C-UbG76C (fig. S2G), and a 2.39-Å complex structure of the Clr4 KMT domain bound to H3 (3–19)K9Nle-K14C without Ub modification (Fig. 1B) were resolved (hereafter abbreviated as the KMT-Ub-H3 and KMT-H3 structures, respectively) (figs. S4, C to H and S5, A to G and table S1).

In these two complex structures, the H3-derived substrates and the SAM cofactor exhibit a 1:1:1 stoichiometry in their binding to the surface of the ovoid-shaped KMT domain. (Fig. 1, C and E). The KMT-Ub-H3 complex assembled into space group P212121 and KMT-H3 assembled into P22121. Each asymmetric unit of the two crystals contains two (P212121) copies and one (P22121) copy of the complex, respectively. For the crystal structure of the KMT-Ub-H3 complex, two copies of the complex adopt highly similar conformations and interfaces (fig. S5A). In our structures, the KMT domain adopts a global conformation similar to that of the apo form [Protein Data Bank (PDB): 6BOX] (34) with a root mean square deviation (RMSD) of ~0.37 Å across 221 pairs of Cα (fig. S4A). The H3 N-terminal tail peptide inserts into a histone-binding groove and straddles the SET subdomain of KMT, resulting in an interaction surface area of more than 2000 Å2. The section of the histone-binding groove near the catalytic active site is negatively charged since the acidic residue–rich SI region forms the outer “wall” of the groove and interacts with residues R8 to G13 of the H3 N-terminal tail (Fig. 1, D and F). The SAM cofactor is located on the larger side of the ovoid structure, embedded in the SAM pocket of the SET domain, with the post-SET subdomain covering above as a lid. Moreover, local allosteric changes are observed in certain structural elements such as the αC helices (residues 366 to 372), the SI region (residues 371 to 394), and the ARL (residues 453 to 472) (fig. S4, A and B).

Notably, in the KMT-Ub-H3 structure, the Ub moiety is clearly visible and docks with the KMT domain on the opposite side of the SAM-binding pocket, forming extensive interfaces, which are 1542 and 1559 Å2 in two NCS units, respectively. The C-terminal tail of Ub extends to the histone-binding groove along a neutral-hydrophobic canyon, sandwiched between the acidic pre-SET and SI region (Fig. 1F).

Conserved Clr4 residues in the histone-binding groove contribute to activity and selectivityThe N-terminal tail of histone H3 extensively interacts with the SET subdomain within the KMT domain. Most of these contacts involve residues A7 to KC14 of H3, which are situated within a narrow groove formed by the parallel acidic SI region, the C terminus of Clr4, and the initial segment (451YAGA454) of ARL (Fig. 2, A and B, and fig. S6, A and D). Within this groove, L382, D384, and D386 in the SI region form hydrogen bonds with the backbone of the H3 tail. The guanidine group of H3 residue R8 is further stabilized by hydrogen bonds with the side chains of D371 and D384 of Clr4. In addition, the carbonyl oxygen of A367 and the hydroxy group of T395 interact with the guanidine group of R8, forming a quadrangular clamp structure that secures this residue (Fig. 2A and fig. S6, B and E). These interactions contribute to the substrate H3K9 specificity of Clr4, as the enzyme selectively recognizes an RK core motif within the histone tail. H3 KNle9 is positioned within the catalytic pocket of Clr4, where its backbone along with the backbone of S10 forms main-chain hydrogen bonds with the SI region. The carboxy group of D386 further forms polar contacts with the hydroxy groups of H3 residues S10 and T11 (fig. S6, C and F).Fig. 2.Conserved interactions between Clr4 KMT and H3 N-terminal peptide.(A) Residues that form quadrangular clamp construct surrounding R8 and interact with the KNle9 backbone. H3 is in magentas, and residues of Clr4 in the KMT-Ub-H3 are represented by cyan sticks, while in KMT-H3 are green. (B) Clr4 residues that contact with the H3 10STGG13 in the KMT-Ub-H3 structure and KMT-H3 structure. (C) Conformation of SAM and KNle9 binding with the Clr4 catalytic pocket/site in KMT-Ub-H3 structure. (D) Evolutionary conservation of residues in Clr4 between homologous proteins, conserved residues appear redder on the surface of KMT. The conserved SI region and post-SET C terminus are marked by the yellow circle. The picture on the right shows the evolutionary tree and sequence alignment between SUV39 family proteins, the conserved SI region and catalytic pocket are marked by orange and red triangles. (E) Comparison of H3 conformation among structures of Suv39 proteins binding with H3 N-terminal tail. H3 in our structure is in magenta, while the H3 binding with DIM5 (PDB: 1PEG) is in purple, binding with G9a (PDB: 5JIY) is in pale green, and binding with GLP1 (PDB: 2RFI) is in wheat. (F and G) Chemiluminescence HMTase activity assay measuring SAH production during the WT Clr4 or Clr4 accommodating mutations in histone-binding groove or catalytic site catalyzing substrate (F) Ub-H3t and (G) H3t methylation (n = 3, error bars indicate SD; ***, 0.0001 < P < 0.001; ****, P < 0.0001), SDS-PAGE gels of the reactions are provided in Supplementary figures.The side chain of histone H3 K9Nle inserts into a hydrophobic tunnel and attacks the S-methyl group of SAM (Fig. 2C and fig. S6G), which is consistent with the active site conformations of other H3K9-specific HMTase. Four aromatic residues (Y357, F383, F449, and Y451) within the aromatic catalytic site surrounding the side chain of K9Nle are highly conserved among various Clr4 homologs in eumycetes and higher living organisms, such as Arabidopsis thaliana, Drosophila melanogaster, Mus musculus, and Homo sapiens (Fig. 2, C and D, and figs. S6, G and H). Besides the classic catalytic site, the acidic SI region is also conserved among H3K9-specific HMTases (Fig. 2D). Compared with the substrate-binding structure of Clr4 homologs like DIM5 in Neurospora crassa (46), G9a (47), and GLP1 (48) in H sapiens, the conformation of H3 residues 7 to 13 in our structure is similar, and the SI region adopts a concerted recognition behavior when it is tightly interdigitated with residues 7 to 13 of the H3-tail peptide (Fig. 2E). This observation suggests that the H3K9-specific HMTases preserve H3K9 recognition patterns and selectivity across various species throughout evolution.To validate the interfaces between Clr4 and H3, we introduced single or double alanine mutations at key interface residues of Clr4 and tested the catalytic behavior of these mutants using unmodified or ubiquitinated H3 N-terminal tail (1 to 20) (H3t and Ub-H3t). We found that alanine mutations of either SI region residues (D371, D384, and D386), whose side chains are involved in hydrogen bond formation, or residues at the catalytic site (Y357, F383, F449, and Y451) weakened the methylation activity, regardless of whether the substrate was unmodified H3 or ubiquitinated H3 (Fig. 2, F and G, and fig. S7, A and B). Therefore, our structural and biochemical results indicate that the highly conserved interactions between the Clr4 histone–binding groove and H3 (residues 7 to 14) are critical for selective substrate binding and methyltransferase activity.

Conserved Clr4 residues in the histone-binding groove contribute to activity and selectivity

The N-terminal tail of histone H3 extensively interacts with the SET subdomain within the KMT domain. Most of these contacts involve residues A7 to KC14 of H3, which are situated within a narrow groove formed by the parallel acidic SI region, the C terminus of Clr4, and the initial segment (451YAGA454) of ARL (Fig. 2, A and B, and fig. S6, A and D). Within this groove, L382, D384, and D386 in the SI region form hydrogen bonds with the backbone of the H3 tail. The guanidine group of H3 residue R8 is further stabilized by hydrogen bonds with the side chains of D371 and D384 of Clr4. In addition, the carbonyl oxygen of A367 and the hydroxy group of T395 interact with the guanidine group of R8, forming a quadrangular clamp structure that secures this residue (Fig. 2A and fig. S6, B and E). These interactions contribute to the substrate H3K9 specificity of Clr4, as the enzyme selectively recognizes an RK core motif within the histone tail. H3 KNle9 is positioned within the catalytic pocket of Clr4, where its backbone along with the backbone of S10 forms main-chain hydrogen bonds with the SI region. The carboxy group of D386 further forms polar contacts with the hydroxy groups of H3 residues S10 and T11 (fig. S6, C and F).

Conserved interactions between Clr4 KMT and H3 N-terminal peptide.

(A) Residues that form quadrangular clamp construct surrounding R8 and interact with the KNle9 backbone. H3 is in magentas, and residues of Clr4 in the KMT-Ub-H3 are represented by cyan sticks, while in KMT-H3 are green. (B) Clr4 residues that contact with the H3 10STGG13 in the KMT-Ub-H3 structure and KMT-H3 structure. (C) Conformation of SAM and KNle9 binding with the Clr4 catalytic pocket/site in KMT-Ub-H3 structure. (D) Evolutionary conservation of residues in Clr4 between homologous proteins, conserved residues appear redder on the surface of KMT. The conserved SI region and post-SET C terminus are marked by the yellow circle. The picture on the right shows the evolutionary tree and sequence alignment between SUV39 family proteins, the conserved SI region and catalytic pocket are marked by orange and red triangles. (E) Comparison of H3 conformation among structures of Suv39 proteins binding with H3 N-terminal tail. H3 in our structure is in magenta, while the H3 binding with DIM5 (PDB: 1PEG) is in purple, binding with G9a (PDB: 5JIY) is in pale green, and binding with GLP1 (PDB: 2RFI) is in wheat. (F and G) Chemiluminescence HMTase activity assay measuring SAH production during the WT Clr4 or Clr4 accommodating mutations in histone-binding groove or catalytic site catalyzing substrate (F) Ub-H3t and (G) H3t methylation (n = 3, error bars indicate SD; ***, 0.0001 < P < 0.001; ****, P < 0.0001), SDS-PAGE gels of the reactions are provided in Supplementary figures.

The side chain of histone H3 K9Nle inserts into a hydrophobic tunnel and attacks the S-methyl group of SAM (Fig. 2C and fig. S6G), which is consistent with the active site conformations of other H3K9-specific HMTase. Four aromatic residues (Y357, F383, F449, and Y451) within the aromatic catalytic site surrounding the side chain of K9Nle are highly conserved among various Clr4 homologs in eumycetes and higher living organisms, such as Arabidopsis thaliana, Drosophila melanogaster, Mus musculus, and Homo sapiens (Fig. 2, C and D, and figs. S6, G and H). Besides the classic catalytic site, the acidic SI region is also conserved among H3K9-specific HMTases (Fig. 2D). Compared with the substrate-binding structure of Clr4 homologs like DIM5 in Neurospora crassa (46), G9a (47), and GLP1 (48) in H sapiens, the conformation of H3 residues 7 to 13 in our structure is similar, and the SI region adopts a concerted recognition behavior when it is tightly interdigitated with residues 7 to 13 of the H3-tail peptide (Fig. 2E). This observation suggests that the H3K9-specific HMTases preserve H3K9 recognition patterns and selectivity across various species throughout evolution.

To validate the interfaces between Clr4 and H3, we introduced single or double alanine mutations at key interface residues of Clr4 and tested the catalytic behavior of these mutants using unmodified or ubiquitinated H3 N-terminal tail (1 to 20) (H3t and Ub-H3t). We found that alanine mutations of either SI region residues (D371, D384, and D386), whose side chains are involved in hydrogen bond formation, or residues at the catalytic site (Y357, F383, F449, and Y451) weakened the methylation activity, regardless of whether the substrate was unmodified H3 or ubiquitinated H3 (Fig. 2, F and G, and fig. S7, A and B). Therefore, our structural and biochemical results indicate that the highly conserved interactions between the Clr4 histone–binding groove and H3 (residues 7 to 14) are critical for selective substrate binding and methyltransferase activity.

Clr4-Ub interfaces are crucial for the stimulationWe next focused on the structure of the KMT-Ub-H3 complex, in which the Ub moiety resembles a balloon tied to a string (Fig. 3A and fig. S5A). The interfaces between Ub and Clr4 can be categorized into three distinct regions (Fig. 3A): First, the canonical Ub interacting spots, including residues L8, I44, V70, and the aliphatic portion of the K6 side chain, create a hydrophobic pocket that accommodates F256 on Clr4 loop 2. Clr4 residue F256 is further stabilized by π-π interaction with Ub residue H68 (Fig. 3B and fig. S8A). Second, R42 and Q49 of Ub are inserted into an acidic pocket of the pre-SET domain. R42 forms salt bridge interactions with Clr4 D281 and forms a hydrogen bond with S258, whereas Q49 contacts with Clr4 D280 (Fig. 3C and fig. S8, B and D). The third interface is at the C-terminal tail of Ub (residues 70 to 76), whose main chain along with the side chains of V70 and L73, lies in the neutral-hydrophobic canyon, which starts from F256 of Clr4 and extends to the histone-binding groove. Meanwhile, the side chain of R72 is fastened to the acidic pre-SET base through two salt bridge interactions with D280 and D281 (Fig. 3D and fig. S8, C and E).Fig. 3.Multivalent binding between ubiquitin and Clr4 KMT.(A) Multivalent binding behavior between ubiquitin and Clr4 KMT; ubiquitin is shown in yellow, Clr4 is shown in cyan, while the key acidic regions are highlighted in red. (B to D) Close-up views of three interfaces (I, II, and III) between Ub and KMT. (E) HMTase activity assay of WT Clr4 or Clr4 accommodating mutations in ubiquitin interfaces working on substrate Ub-H3t (top) and H3t (bottom). (F) HMTase activity assay of WT Clr4 working on Ub-mutated Ub-H3t. (G) KD value measured by ITC experiment performed with WT Clr4 KMT and Ub interface–mutated Ub-H3t. (H) Silencing assays of mat2P::ade6+ on low-adenine medium (Low ade) to assess ade6+ silencing in the indicated genotypes involving Clr4 and Clr4 mutants. ns, no significance; ***, 0.0001 < P < 0.001; ****, P < 0.0001.The importance of the Clr4-Ub interface was assessed by evaluating the impact of Clr4 mutations. The Clr4 F256A mutation, which diminishes hydrophobic interactions, reduced the HMTase activity of Ub-H3t by 60% (Fig. 3E, top, and fig. S7C). Mutations in the pre-SET acidic pocket (D280A/D281A) strongly impaired HMTase activity by 82%. In contrast, these Clr4 mutants (F256A and D280A/D281A) did not affect H3K9me activity on unmodified H3t (Fig. 3E, bottom, and fig. S7C). We also conducted thermoshift experiments and revealed that the melting temperature (ΔTm) of these mutants significantly decreased when binding to the ubiquitinated H3 tail, indicating the importance of these interfaces for ubiquitin binding and thermodynamic stabilization of Clr4 (fig. S8G). Furthermore, Clr4 methylation activity was noticeably decreased on the Ub-H3t substrate bearing mutations in the Ub R42 anchor (R42A), I44/L8 patch (L8R/H68A, L69A/V70A), and C-terminal tail (L71A/L73A, R72A/R74A). Mutation of R42 led to the most significant decrease in the ability to be methylated (92%), while that of the other four mutants was reduced by 22~89% (Fig. 3F and figs. S7D and S9, A to E). Further measurement of the binding affinity between the above Ub-H3t variants and Clr4 by ITC revealed KD values ranging from 0.239 to 1.46 μM, which are much lower than that of the WT substrate (0.115 ± 0.018 μM) (Fig. 3G and fig. S8F). Collectively, these findings indicate that the interfaces between Clr4 and H3K14 ubiquitin are crucial for substrate recognition, enzyme stabilization, and methylation stimulation.

Clr4-Ub interfaces are crucial for the stimulation

We next focused on the structure of the KMT-Ub-H3 complex, in which the Ub moiety resembles a balloon tied to a string (Fig. 3A and fig. S5A). The interfaces between Ub and Clr4 can be categorized into three distinct regions (Fig. 3A): First, the canonical Ub interacting spots, including residues L8, I44, V70, and the aliphatic portion of the K6 side chain, create a hydrophobic pocket that accommodates F256 on Clr4 loop 2. Clr4 residue F256 is further stabilized by π-π interaction with Ub residue H68 (Fig. 3B and fig. S8A). Second, R42 and Q49 of Ub are inserted into an acidic pocket of the pre-SET domain. R42 forms salt bridge interactions with Clr4 D281 and forms a hydrogen bond with S258, whereas Q49 contacts with Clr4 D280 (Fig. 3C and fig. S8, B and D). The third interface is at the C-terminal tail of Ub (residues 70 to 76), whose main chain along with the side chains of V70 and L73, lies in the neutral-hydrophobic canyon, which starts from F256 of Clr4 and extends to the histone-binding groove. Meanwhile, the side chain of R72 is fastened to the acidic pre-SET base through two salt bridge interactions with D280 and D281 (Fig. 3D and fig. S8, C and E).

Multivalent binding between ubiquitin and Clr4 KMT.

(A) Multivalent binding behavior between ubiquitin and Clr4 KMT; ubiquitin is shown in yellow, Clr4 is shown in cyan, while the key acidic regions are highlighted in red. (B to D) Close-up views of three interfaces (I, II, and III) between Ub and KMT. (E) HMTase activity assay of WT Clr4 or Clr4 accommodating mutations in ubiquitin interfaces working on substrate Ub-H3t (top) and H3t (bottom). (F) HMTase activity assay of WT Clr4 working on Ub-mutated Ub-H3t. (G) KD value measured by ITC experiment performed with WT Clr4 KMT and Ub interface–mutated Ub-H3t. (H) Silencing assays of mat2P::ade6+ on low-adenine medium (Low ade) to assess ade6+ silencing in the indicated genotypes involving Clr4 and Clr4 mutants. ns, no significance; ***, 0.0001 < P < 0.001; ****, P < 0.0001.

The importance of the Clr4-Ub interface was assessed by evaluating the impact of Clr4 mutations. The Clr4 F256A mutation, which diminishes hydrophobic interactions, reduced the HMTase activity of Ub-H3t by 60% (Fig. 3E, top, and fig. S7C). Mutations in the pre-SET acidic pocket (D280A/D281A) strongly impaired HMTase activity by 82%. In contrast, these Clr4 mutants (F256A and D280A/D281A) did not affect H3K9me activity on unmodified H3t (Fig. 3E, bottom, and fig. S7C). We also conducted thermoshift experiments and revealed that the melting temperature (ΔTm) of these mutants significantly decreased when binding to the ubiquitinated H3 tail, indicating the importance of these interfaces for ubiquitin binding and thermodynamic stabilization of Clr4 (fig. S8G). Furthermore, Clr4 methylation activity was noticeably decreased on the Ub-H3t substrate bearing mutations in the Ub R42 anchor (R42A), I44/L8 patch (L8R/H68A, L69A/V70A), and C-terminal tail (L71A/L73A, R72A/R74A). Mutation of R42 led to the most significant decrease in the ability to be methylated (92%), while that of the other four mutants was reduced by 22~89% (Fig. 3F and figs. S7D and S9, A to E). Further measurement of the binding affinity between the above Ub-H3t variants and Clr4 by ITC revealed KD values ranging from 0.239 to 1.46 μM, which are much lower than that of the WT substrate (0.115 ± 0.018 μM) (Fig. 3G and fig. S8F). Collectively, these findings indicate that the interfaces between Clr4 and H3K14 ubiquitin are crucial for substrate recognition, enzyme stabilization, and methylation stimulation.

Validation of the key roles of the Clr4-H3 and Clr4-Ub interfaces in vivoTo validate the pivotal roles of the Clr4-H3 and Clr4-Ub interfaces identified in our crystal structure in the context of Clr4 function in vivo, we constructed S. pombe cells that express endogenous FLAG-tagged Clr4 mutant proteins with single or double substitutions at the Clr4-H3 (Y357A and D384A) and Clr4-Ub interfaces (F256A and D280A/D281A) (fig. S10A). Clr4-mediated heterochromatin deposition and gene silencing were examined using an ade6+ reporter transgene inserted within the heterochromatic mat locus. The propagation of surrounding heterochromatin leads to the silencing of ade6+, resulting in red colonies on low-adenine plates. Our experiments showed that disruption of the catalytic pocket (Y357A) and histone-binding groove (D384A) completely abolished the heterochromatin deposition, yielding colonies as white as those observed with the previously reported Clr4 dead mutant (Y451N) (Fig. 3H) (40). This outcome underscores the essential roles of both the histone-binding groove and Clr4 methylation activity in Clr4-mediated heterochromatin deposition.In addition, mutations at the Clr4-Ub interfaces (F256A and D280A/D281A) resulted in a significant decrease in silencing, as evidenced by light pink to white colonies (Fig. 3H). Consistent with the observed phenotype, H3K9me3 levels in nuclear chromatin were substantially decreased in the Clr4 knockout/dead cells as well as in strains with histone binding–eliminated Clr4 mutants (D384A and Y357A), whereas mutations (D280A/D281A and F256A) in the Ub-binding interface also resulted in a reduction in H3K9me levels (fig. S10, B and C). Besides the heterochromatin deposition at mat2P::ade6+ reporter, other constitutive heterochromatic domains were also investigated. Reverse transcription quantitative polymerase chain reaction (qPCR) was performed to measure the transcript levels of the pericentromeric dg/dh repeats and subtelomeric marker gene tlh1 in S. pombe strains mentioned above. Transcript levels of dh/dg and tlh1 were activated by three orders of magnitude relative to WT in knockout strains, Clr4 catalytic pocket mutant Y357A, Y451N strains, and histone-binding groove mutant D384A strains. Meanwhile, the Ub-binding interface mutated (F256A and D280A/D281A) strains also significantly impair the suppression of transcription, as the transcript level of dh/dg and tlh1 increased by two orders of magnitude (fig. S10D). These findings indicate that the Clr4-Ub interfaces are important for the regulation of heterochromatin deposition and transcription suppression, while the disruption of Ub interactions still allows minimal heterochromatin formation.

Validation of the key roles of the Clr4-H3 and Clr4-Ub interfaces in vivo

To validate the pivotal roles of the Clr4-H3 and Clr4-Ub interfaces identified in our crystal structure in the context of Clr4 function in vivo, we constructed S. pombe cells that express endogenous FLAG-tagged Clr4 mutant proteins with single or double substitutions at the Clr4-H3 (Y357A and D384A) and Clr4-Ub interfaces (F256A and D280A/D281A) (fig. S10A). Clr4-mediated heterochromatin deposition and gene silencing were examined using an ade6+ reporter transgene inserted within the heterochromatic mat locus. The propagation of surrounding heterochromatin leads to the silencing of ade6+, resulting in red colonies on low-adenine plates. Our experiments showed that disruption of the catalytic pocket (Y357A) and histone-binding groove (D384A) completely abolished the heterochromatin deposition, yielding colonies as white as those observed with the previously reported Clr4 dead mutant (Y451N) (Fig. 3H) (40). This outcome underscores the essential roles of both the histone-binding groove and Clr4 methylation activity in Clr4-mediated heterochromatin deposition.

In addition, mutations at the Clr4-Ub interfaces (F256A and D280A/D281A) resulted in a significant decrease in silencing, as evidenced by light pink to white colonies (Fig. 3H). Consistent with the observed phenotype, H3K9me3 levels in nuclear chromatin were substantially decreased in the Clr4 knockout/dead cells as well as in strains with histone binding–eliminated Clr4 mutants (D384A and Y357A), whereas mutations (D280A/D281A and F256A) in the Ub-binding interface also resulted in a reduction in H3K9me levels (fig. S10, B and C). Besides the heterochromatin deposition at mat2P::ade6+ reporter, other constitutive heterochromatic domains were also investigated. Reverse transcription quantitative polymerase chain reaction (qPCR) was performed to measure the transcript levels of the pericentromeric dg/dh repeats and subtelomeric marker gene tlh1 in S. pombe strains mentioned above. Transcript levels of dh/dg and tlh1 were activated by three orders of magnitude relative to WT in knockout strains, Clr4 catalytic pocket mutant Y357A, Y451N strains, and histone-binding groove mutant D384A strains. Meanwhile, the Ub-binding interface mutated (F256A and D280A/D281A) strains also significantly impair the suppression of transcription, as the transcript level of dh/dg and tlh1 increased by two orders of magnitude (fig. S10D). These findings indicate that the Clr4-Ub interfaces are important for the regulation of heterochromatin deposition and transcription suppression, while the disruption of Ub interactions still allows minimal heterochromatin formation.

Essentiality of the H3K14-specific junction for ubiquitin-mediated stimulationNotably, the interaction between Clr4 KMT and free ubiquitin was undetectable in the ITC experiment (fig. S8H), which is consistent with previous findings that Clr4 does not sense free ubiquitin to facilitate H3K9me (Fig. 4A and fig. S11A) (40). These findings suggest that the stimulatory effect of ubiquitin is contingent on its linkage to H3K14. We therefore hypothesized that the binding of ubiquitin to KMT is constrained by the H3K14 junction, which in turn enhances binding affinity by stabilizing the enzyme-substrate complex. In alignment with this hypothesis, the presence of free ubiquitin did not influence the change in the ΔTm of the complex in the thermal shift assay (Fig. 4B and figs. S12A and S11, C and D). Furthermore, extending the C-terminal tail of ubiquitin with an AEEA {[2- (2-aminoethoxy) ethoxy] acetic acid} linker (fig. S12C) led to an approximately 25% reduction in methyltransferase activity with the Ub-AEEA-H3t substrate (Fig. 4A) and a 1.12°C decrease in ΔTm compared with that of the native Ub-H3t substrate (Fig. 4C and figs. S11B and S12B). This result suggested that a less constrained ubiquitin motif resulted in reduced thermodynamic stabilization and consequently weaker activation of Clr4.Fig. 4.Biochemical experiment about H3-Ub junction.(A) HMTase activity assay of WT Clr4 working on the substrates with or without Ub and AEEA-linked Ub (n = 5, error bars indicate SD). (B) Thermo-shift assay working on Clr4 KMT binding with Ub-H3t (KNle9), H3t (KNle9), H3t (KNle9) with free Ub. The ΔTm was determined by the first derivative of the barycentric mean (BCM) of fluorescence intensity with respect to temperature (dBCM/dT) (n = 3, error bars indicate SD). (C) Thermo-shift assay performed with Clr4 KMT and Ub-H3t, Ub-AEEA-H3t, respectively (n = 3, error bars indicate SD). (D) Scheme of “caliper construct” functioning in the recognition of K14-ubiquitinated H3 N-terminal tail. The catalytic site is highlighted by a red circle, and the intersection point of the histone-binding groove and neutral-hydrophobic canyon is labeled by an orange dot. The caliper shows the approximate distance between the catalytic site and the intersection point. aa, amino acid. (E) HMTase activity assay of WT Clr4 working on the substrates that ubiquitin linked to residue 13 or residue 16 of H3 N-terminal tail (n = 5, error bars indicate SD). (F) HMTase activity assay of WT Clr4 catalyzing NCP substrate with H3K14/K18/K23Ub or without modification (n = 3, error bars indicate SD). ****, P < 0.0001.In our KMT-Ub-H3 complex structure, the distance from the catalytic site to the point where the histone-binding groove intersects with the neutral-hydrophobic canyon is approximately 16 Å. This distance is slightly shorter than that of an extended hexapeptide (~19 Å, corresponding to residues 9KSTGGK14), as the sequential Cα−Cα distance is consistently restricted to ~3.8 Å (fig. S12E). Notably, the K14 junction is located at this intersection, allowing the histone-binding groove to effectively accommodate the extended configuration of H3 residues K9 to K14 while enabling the C-terminal tail of Ub to reside precisely within the neutral-hydrophobic canyon (Fig. 4D). This recognition mechanism implies that the stimulatory effect is highly sensitive to the position of the Ub site. Attempts to fine-tune the Ub site, either upstream or downstream of H3K14, led to impaired methylation activity with the Ub-modified H3 N-terminal tail (Fig. 4E and fig. S13A), despite only shifting the junction by two amino acids forward or backward (H3tGC12-Ub, H3tPC16-Ub; fig. S13, A and D). This finding highlights the low tolerance for alterations in the substrate Ub site.Furthermore, we investigated biologically relevant ubiquitinated H3 substrates, including K18 and K23-ubiquitinated nucleosomes involved in DNMT1 recruitment and activation in the context of hemimethylated DNA inheritance (NCPH3K14Ub, NCPH3K18Ub, and NCPH3K23Ub; fig. S13, B and E to G) (49–51). HMTase assay revealed that the methylation rates of unmodified nucleosome core particle (NCP), NCPH3K18Ub, NCPH3K23Ub were much lower than that of NCPH3K14Ub (Fig. 4F). H3K18ub is also activating albeit to a lesser degree than H3K14ub. This phenomenon closely parallels the previously reported H3K9me (G9a, SUV39H1/2)-DNA methylation (DNMT1)–H3K18 Ub (UHRF1) cross-talk network in H. sapiens (52–55). Especially for Clr4 homologs SUV39H1/2: in DNMT-inhibited cells, UHRF1 catalyzes H3K18 Ub at DNA hypomethylation region, and then the H3K18Ub can recruit SUV39H1/H2 and promote the H3K9me (54). The electrophoretic mobility shift assay results indicated that almost all NCPH3K14Ub was bound to Clr4 at a Clr4 concentration of 0.5 μM, whereas a portion of NCPH3K18Ub, along with unmodified NCP, persisted as free NCP until the concentration of Clr4 was increased to 1 to 2 μM (fig. S12D). These results imply that Clr4 is selectively recruited to H3K14-ubiquitinated nucleosomes.

Essentiality of the H3K14-specific junction for ubiquitin-mediated stimulation

Notably, the interaction between Clr4 KMT and free ubiquitin was undetectable in the ITC experiment (fig. S8H), which is consistent with previous findings that Clr4 does not sense free ubiquitin to facilitate H3K9me (Fig. 4A and fig. S11A) (40). These findings suggest that the stimulatory effect of ubiquitin is contingent on its linkage to H3K14. We therefore hypothesized that the binding of ubiquitin to KMT is constrained by the H3K14 junction, which in turn enhances binding affinity by stabilizing the enzyme-substrate complex. In alignment with this hypothesis, the presence of free ubiquitin did not influence the change in the ΔTm of the complex in the thermal shift assay (Fig. 4B and figs. S12A and S11, C and D). Furthermore, extending the C-terminal tail of ubiquitin with an AEEA {[2- (2-aminoethoxy) ethoxy] acetic acid} linker (fig. S12C) led to an approximately 25% reduction in methyltransferase activity with the Ub-AEEA-H3t substrate (Fig. 4A) and a 1.12°C decrease in ΔTm compared with that of the native Ub-H3t substrate (Fig. 4C and figs. S11B and S12B). This result suggested that a less constrained ubiquitin motif resulted in reduced thermodynamic stabilization and consequently weaker activation of Clr4.

Biochemical experiment about H3-Ub junction.

(A) HMTase activity assay of WT Clr4 working on the substrates with or without Ub and AEEA-linked Ub (n = 5, error bars indicate SD). (B) Thermo-shift assay working on Clr4 KMT binding with Ub-H3t (KNle9), H3t (KNle9), H3t (KNle9) with free Ub. The ΔTm was determined by the first derivative of the barycentric mean (BCM) of fluorescence intensity with respect to temperature (dBCM/dT) (n = 3, error bars indicate SD). (C) Thermo-shift assay performed with Clr4 KMT and Ub-H3t, Ub-AEEA-H3t, respectively (n = 3, error bars indicate SD). (D) Scheme of “caliper construct” functioning in the recognition of K14-ubiquitinated H3 N-terminal tail. The catalytic site is highlighted by a red circle, and the intersection point of the histone-binding groove and neutral-hydrophobic canyon is labeled by an orange dot. The caliper shows the approximate distance between the catalytic site and the intersection point. aa, amino acid. (E) HMTase activity assay of WT Clr4 working on the substrates that ubiquitin linked to residue 13 or residue 16 of H3 N-terminal tail (n = 5, error bars indicate SD). (F) HMTase activity assay of WT Clr4 catalyzing NCP substrate with H3K14/K18/K23Ub or without modification (n = 3, error bars indicate SD). ****, P < 0.0001.

In our KMT-Ub-H3 complex structure, the distance from the catalytic site to the point where the histone-binding groove intersects with the neutral-hydrophobic canyon is approximately 16 Å. This distance is slightly shorter than that of an extended hexapeptide (~19 Å, corresponding to residues 9KSTGGK14), as the sequential Cα−Cα distance is consistently restricted to ~3.8 Å (fig. S12E). Notably, the K14 junction is located at this intersection, allowing the histone-binding groove to effectively accommodate the extended configuration of H3 residues K9 to K14 while enabling the C-terminal tail of Ub to reside precisely within the neutral-hydrophobic canyon (Fig. 4D). This recognition mechanism implies that the stimulatory effect is highly sensitive to the position of the Ub site. Attempts to fine-tune the Ub site, either upstream or downstream of H3K14, led to impaired methylation activity with the Ub-modified H3 N-terminal tail (Fig. 4E and fig. S13A), despite only shifting the junction by two amino acids forward or backward (H3tGC12-Ub, H3tPC16-Ub; fig. S13, A and D). This finding highlights the low tolerance for alterations in the substrate Ub site.

Furthermore, we investigated biologically relevant ubiquitinated H3 substrates, including K18 and K23-ubiquitinated nucleosomes involved in DNMT1 recruitment and activation in the context of hemimethylated DNA inheritance (NCPH3K14Ub, NCPH3K18Ub, and NCPH3K23Ub; fig. S13, B and E to G) (49–51). HMTase assay revealed that the methylation rates of unmodified nucleosome core particle (NCP), NCPH3K18Ub, NCPH3K23Ub were much lower than that of NCPH3K14Ub (Fig. 4F). H3K18ub is also activating albeit to a lesser degree than H3K14ub. This phenomenon closely parallels the previously reported H3K9me (G9a, SUV39H1/2)-DNA methylation (DNMT1)–H3K18 Ub (UHRF1) cross-talk network in H. sapiens (52–55). Especially for Clr4 homologs SUV39H1/2: in DNMT-inhibited cells, UHRF1 catalyzes H3K18 Ub at DNA hypomethylation region, and then the H3K18Ub can recruit SUV39H1/H2 and promote the H3K9me (54). The electrophoretic mobility shift assay results indicated that almost all NCPH3K14Ub was bound to Clr4 at a Clr4 concentration of 0.5 μM, whereas a portion of NCPH3K18Ub, along with unmodified NCP, persisted as free NCP until the concentration of Clr4 was increased to 1 to 2 μM (fig. S12D). These results imply that Clr4 is selectively recruited to H3K14-ubiquitinated nucleosomes.

Allosteric effects during substrate turnover and its stimulation by H3K14UbNext, we compared the structures of Clr4 in the KMT-H3, KMT-Ub-H3, and apo form to analyze the conformational changes mediated by histone H3 and Ub, respectively, which underlie the pronounced difference in Clr4 substrate turnover (kcat) observed during catalysis of unmodified and H3K14Ub-modified substrates. We first superimposed our KMT-H3 complex structure with the previously reported Clr4 KMT apo conformation structure (PDB: 6BOX) to identify key conformational changes required for substrate engagement. Structural rearrangements of Clr4 were observed in four main regions: the ARL (residues 453 to 472), the helix αC-SI region (residues 365 to 395), the Clr4 C terminus (residues 486 to 490), and the β9/10 loop (residues 423 to 431) (fig. S14, A to B). Once H3 binds, the ARL, which previously occluded the active site (34), is released to expose the active site and accommodate the binding of the H3 N-terminal tail (Fig. 5B). The Clr4 C terminus undergoes an ~75° rotation with G486 as the vertex, positioning itself toward the substrate (Fig. 5B). Helix αC (residues 364 to 472) is rotated approximately 9° toward the substrate on the axis of T363, generating a hydrogen bond between Clr4 D371 and H3 R8 (Fig. 5A). The β-hairpin loop within the SI region (residues 381 to 394) shifts toward the histone-binding groove by 1.62-Å backbone RMSD to interact with H3 (Fig. 5C). Meanwhile, the β9/10 loop, located at the center of the neutral-hydrophobic canyon and in close proximity to the β-hairpin loop (residues 385 to 394) of SI region, undergoes retraction in the catalyzing state, which facilitates the formation of a nascent salt bridge interaction between Clr4 residues R428 and D390 on the β-hairpin loop in SI region (Fig. 5D). These observations suggest that specific conformational changes within structural elements of Clr4 are required for the positioning and alignment of H3 for methylation.Fig. 5.Allostery effects during substrate turnover and stimulation by ubiquitin.(A) Conformation changes of helix αC between apo structure and our KMT-H3 structure. Two structures are superposed by KMT alignment. Structural elements in the apo structure are colored in blue, and those in the KMT-H3 structure are colored in pink. The directions of allostery are highlighted by magenta arrows. Clr4 D371 and H3 R8 forming nascent interactions are shown as sticks in map density. (B) Conformation changes of ARL and Clr4 C terminus between apo structure and our KMT-H3 structure. (C and D) Conformation changes of β9/10 loop and SI region between apo structure and our KMT-H3 structure. Clr4 D390 and R428 forming nascent interactions are shown as sticks in map density in (D). (E) Flipping of Clr4 H425 once the H3K14 is ubiquitinated. Key residues forming nascent contacts are represented by sticks in map density. KMT-H3 structure and our KMT-Ub-H3 structure are superposed by KMT alignment. The KMT-H3 structure is colored in pink, and that in the KMT-Ub-H3 structure is colored in cyan. (F and H) Michaelis-Menten curves of the full-length Clr4 accommodate (F) H425A and (H) F427A mutations catalyzing ubiquitinated peptide methylation (n = 4, error bars indicate SD). (G) Clash between Ub L8 and Clr4 F427 when superposing apo form Clr4 into the KMT-Ub-H3 structure. Ubiquitin is colored in yellow, and L8 is represented by yellow sticks and surface.To further understand how H3K14Ub drives the establishment of a hyperactive state of Clr4, we compared the KMT-H3 complex structure with the KMT-Ub-H3 complex structure to investigate the conformational change induced by H3K14Ub, which revealed structural changes in Clr4 at the β9/10 loop (residues 423 to 431), the beginning of the pre-SET subdomain (residues 254 to 268), and the onset of the SI region (residues 372 to 378), as well as structural changes at H3 residues 15 to 19 (fig. S14, C and D). Specifically, H425 in the β9/10 loop, which is well resolved in both the KMT-H3 and KMT-Ub-H3 structures, undergoes a 180° flip in the KMT-Ub-H3 structure. This flip allows it to engage in a π-π interaction with the aryl group of pre-SET residue Y305 and form a polar interaction with the backbone carbonyl oxygen of E306 (Fig. 5E). This interaction may stabilize the retracted conformation of the β9/10 loop in the ubiquitin-induced hyperactive state. Further experiments involving the Clr4 H425A mutation demonstrated a significant reduction in kcat for the Ub-H3t substrate, with an approximately fivefold decrease (kcat = 3.85 ± 0.171 min−1 versus kcat = 18.3 ± 0.5 min−1 for WT). This mutation did not diminish the substrate recruitment (Km = 0.254 ± 0.048 μM versus Km = 0.559 ± 0.052 μM for WT) or affect methylation activity with unmodified H3t (Fig. 5F and fig. S14E). These results suggest that the allostery triggered by H3K14Ub and the Clr4 β9/10 loop, along with the induced interactions between this loop and the pre-SET region, play essential roles in enhancing enzymatic turnover.To identify the requisite conformational alterations that Clr4 undergoes upon binding to the H3K14Ub substrate, we compared the inactive Clr4 KMT apo structure (PDB: 6BOX) (34) with the hyperactive KMT-Ub-H3 structure. This comparison revealed a steric clash between the Ub L8 loop and apo-form Clr4 residue F427 (Fig. 5G). In both the H3-KMT and H3-Ub-KMT structures, F427 is in a retracted position, forming a nascent hydrophobic pocket that accommodates the Ub L8 loop. This arrangement, in turn, restricts the β9/10 loop to the retracted conformation, which may be essential for the hyperactive state of the H3-Ub-KMT structure (Fig. 5G). Further analysis revealed that the Clr4 F427A mutation, which reduces the spatial size of the side chain, results in an approximately one-third reduction in kcat (kcat = 6.66 ± 0.33 min−1 versus kcat = 18.3 ± 0.5 min−1 of WT), while it does not significantly affect Km (Km = 0.528 ± 0.092 μM versus Km = 0.559 ± 0.052 μM of WT) or basal methylation activity with unmodified H3t (Fig. 5H and fig. S14E). These findings support the hypothesis that the newly formed interaction between Clr4 F427 and Ub is critical for driving H3K14Ub-mediated activation of enzymatic turnover.

Allosteric effects during substrate turnover and its stimulation by H3K14Ub

Next, we compared the structures of Clr4 in the KMT-H3, KMT-Ub-H3, and apo form to analyze the conformational changes mediated by histone H3 and Ub, respectively, which underlie the pronounced difference in Clr4 substrate turnover (kcat) observed during catalysis of unmodified and H3K14Ub-modified substrates. We first superimposed our KMT-H3 complex structure with the previously reported Clr4 KMT apo conformation structure (PDB: 6BOX) to identify key conformational changes required for substrate engagement. Structural rearrangements of Clr4 were observed in four main regions: the ARL (residues 453 to 472), the helix αC-SI region (residues 365 to 395), the Clr4 C terminus (residues 486 to 490), and the β9/10 loop (residues 423 to 431) (fig. S14, A to B). Once H3 binds, the ARL, which previously occluded the active site (34), is released to expose the active site and accommodate the binding of the H3 N-terminal tail (Fig. 5B). The Clr4 C terminus undergoes an ~75° rotation with G486 as the vertex, positioning itself toward the substrate (Fig. 5B). Helix αC (residues 364 to 472) is rotated approximately 9° toward the substrate on the axis of T363, generating a hydrogen bond between Clr4 D371 and H3 R8 (Fig. 5A). The β-hairpin loop within the SI region (residues 381 to 394) shifts toward the histone-binding groove by 1.62-Å backbone RMSD to interact with H3 (Fig. 5C). Meanwhile, the β9/10 loop, located at the center of the neutral-hydrophobic canyon and in close proximity to the β-hairpin loop (residues 385 to 394) of SI region, undergoes retraction in the catalyzing state, which facilitates the formation of a nascent salt bridge interaction between Clr4 residues R428 and D390 on the β-hairpin loop in SI region (Fig. 5D). These observations suggest that specific conformational changes within structural elements of Clr4 are required for the positioning and alignment of H3 for methylation.

Allostery effects during substrate turnover and stimulation by ubiquitin.

(A) Conformation changes of helix αC between apo structure and our KMT-H3 structure. Two structures are superposed by KMT alignment. Structural elements in the apo structure are colored in blue, and those in the KMT-H3 structure are colored in pink. The directions of allostery are highlighted by magenta arrows. Clr4 D371 and H3 R8 forming nascent interactions are shown as sticks in map density. (B) Conformation changes of ARL and Clr4 C terminus between apo structure and our KMT-H3 structure. (C and D) Conformation changes of β9/10 loop and SI region between apo structure and our KMT-H3 structure. Clr4 D390 and R428 forming nascent interactions are shown as sticks in map density in (D). (E) Flipping of Clr4 H425 once the H3K14 is ubiquitinated. Key residues forming nascent contacts are represented by sticks in map density. KMT-H3 structure and our KMT-Ub-H3 structure are superposed by KMT alignment. The KMT-H3 structure is colored in pink, and that in the KMT-Ub-H3 structure is colored in cyan. (F and H) Michaelis-Menten curves of the full-length Clr4 accommodate (F) H425A and (H) F427A mutations catalyzing ubiquitinated peptide methylation (n = 4, error bars indicate SD). (G) Clash between Ub L8 and Clr4 F427 when superposing apo form Clr4 into the KMT-Ub-H3 structure. Ubiquitin is colored in yellow, and L8 is represented by yellow sticks and surface.

To further understand how H3K14Ub drives the establishment of a hyperactive state of Clr4, we compared the KMT-H3 complex structure with the KMT-Ub-H3 complex structure to investigate the conformational change induced by H3K14Ub, which revealed structural changes in Clr4 at the β9/10 loop (residues 423 to 431), the beginning of the pre-SET subdomain (residues 254 to 268), and the onset of the SI region (residues 372 to 378), as well as structural changes at H3 residues 15 to 19 (fig. S14, C and D). Specifically, H425 in the β9/10 loop, which is well resolved in both the KMT-H3 and KMT-Ub-H3 structures, undergoes a 180° flip in the KMT-Ub-H3 structure. This flip allows it to engage in a π-π interaction with the aryl group of pre-SET residue Y305 and form a polar interaction with the backbone carbonyl oxygen of E306 (Fig. 5E). This interaction may stabilize the retracted conformation of the β9/10 loop in the ubiquitin-induced hyperactive state. Further experiments involving the Clr4 H425A mutation demonstrated a significant reduction in kcat for the Ub-H3t substrate, with an approximately fivefold decrease (kcat = 3.85 ± 0.171 min−1 versus kcat = 18.3 ± 0.5 min−1 for WT). This mutation did not diminish the substrate recruitment (Km = 0.254 ± 0.048 μM versus Km = 0.559 ± 0.052 μM for WT) or affect methylation activity with unmodified H3t (Fig. 5F and fig. S14E). These results suggest that the allostery triggered by H3K14Ub and the Clr4 β9/10 loop, along with the induced interactions between this loop and the pre-SET region, play essential roles in enhancing enzymatic turnover.

To identify the requisite conformational alterations that Clr4 undergoes upon binding to the H3K14Ub substrate, we compared the inactive Clr4 KMT apo structure (PDB: 6BOX) (34) with the hyperactive KMT-Ub-H3 structure. This comparison revealed a steric clash between the Ub L8 loop and apo-form Clr4 residue F427 (Fig. 5G). In both the H3-KMT and H3-Ub-KMT structures, F427 is in a retracted position, forming a nascent hydrophobic pocket that accommodates the Ub L8 loop. This arrangement, in turn, restricts the β9/10 loop to the retracted conformation, which may be essential for the hyperactive state of the H3-Ub-KMT structure (Fig. 5G). Further analysis revealed that the Clr4 F427A mutation, which reduces the spatial size of the side chain, results in an approximately one-third reduction in kcat (kcat = 6.66 ± 0.33 min−1 versus kcat = 18.3 ± 0.5 min−1 of WT), while it does not significantly affect Km (Km = 0.528 ± 0.092 μM versus Km = 0.559 ± 0.052 μM of WT) or basal methylation activity with unmodified H3t (Fig. 5H and fig. S14E). These findings support the hypothesis that the newly formed interaction between Clr4 F427 and Ub is critical for driving H3K14Ub-mediated activation of enzymatic turnover.

DISCUSSIONOur study elucidates the molecular mechanism by which H3K14Ub significantly enhances the catalytic activity of Clr4 in the context of H3K9me, a process central to heterochromatin formation (1, 4, 5). Our structural and biochemical analyses revealed a synergistic mechanism involving both increased substrate affinity and allosteric activation (Fig. 6B). First, H3K14Ub dramatically increases Clr4’s affinity for histone substrate through multivalent hydrophobic patches and salt bridge interactions between the Clr4 pre-SET subdomain and ubiquitin. The increase in enzyme-substrate binding is highly specific to H3K14Ub, highlighting the critical role of this precise modification site. Second, H3K14Ub triggers a conformational shift in Clr4, transitioning it to a hyperactive catalyzing state. This state is characterized by a compacted αC helix-SI region, complete release of the ARL, and stable retraction of the β9/10 loop, which contribute to increased substrate turnover. In contrast, Clr4 K455/K472 automethylation, while contributing to Clr4 activity, demonstrates a less pronounced effect in comparison to H3K14Ub. Collectively, H3K14Ub acts as a potent dual-function regulator of Clr4, significantly boosting its activity through enhanced substrate binding and allosteric modulation, thereby playing a crucial role in the efficient establishment of heterochromatin.Fig. 6.Multilayered regulation of Clr4 KMT catalyzation.(A) Overview of conformation changes in the regulatory and catalyzation of Clr4. The element colors in cartoon schematic diagrams of structures correspond to the font fill colors of their names. (B) Scheme of the synergetic mechanism of Clr4 stimulation involving both increased substrate affinity and allosteric activation by H3K14Ub.By aligning our structures of catalyzing state and ubiquitin-induced “hyperactive state” of Clr4 with the previously resolved (34) structures of Clr4 before (“autoinhibited state,” PDB: 6BOX) and after (“activated state,” PDB: 6BP4) automethylation, we propose the perspective of different levels of conformational regulation mediated by these two pathways (Fig. 6A). Compared with the apo autoinhibited state, Clr4 in the activated state undergoes the relief of the ARL from the histone-binding groove and stabilization of the helix αC-SI region mediated by the ARL nascent C-terminal helix (residues 468 to 473) (fig. S15A). However, the helix αC-SI region and β9/10 loop remain in unaltered original conformations in the activated state, which is different from the catalyzing state (fig. S15, A and B). The ARL in the activated state structure is supposed to be further opened when Clr4 is transferred to the catalyzing state. Because the ARL N terminus (residues 451 to 455) in the activated state structure remains a steric clash with substrate H3 (fig. S15C) and the ARL C-terminal helix R469, I470 would impede the conformational shift of D371, K372, and the formation of D371-R8 hydrogen bonds (fig. S15C). Briefly, automethylation partially alleviates the clashes but cannot completely remove them. However, H3K14Ub induces a hyperactive state, involving the retracted β9/10 loop by the H425-pre-SET Y305/E306 and F427-Ub L8 interactions, as well as the shifted SI region contacting with β9/10 loop. Compared with automethylation, H3K14Ub adopts a distinct and likely more adequate mechanism to allosterically promote substrate turnover, contributing to the pronounced stimulation of H3K9me.The identification of Clr4 ubiquitin–binding interfaces provides valuable insights into the regulation of Ub during gene silencing and heterochromatin formation. We found that mutation of F256 and D280/D281 of Clr4 significantly reduces the stimulatory effect of H3K14Ub on Clr4 activity without affecting its intrinsic H3K9me activity (Fig. 3E). The combined mutations of these amino acids present an ideal strategy for uncoupling the cross-talk between H3K14Ub and H3K9me, enabling the elucidation of the functional implications of this cross-talk for the downstream pathways associated with H3K9me. For example, to investigate the function of H3K4 demethylation affected by H3K14 acetylation (H3K14ac), a recent study resolved the structure of the demethylase complex LSD1-CoREST recognizing H3K14ac, engineered a protein mutant that can uncouple this cross-talk, and clarified the role of this cross-talk in cellular adhesion and myeloid leukocyte activation (56). Previous chromatin immunoprecipitation–qPCR study (40) introducing multiple mutations to the potential ubiquitin interface on Clr4 in S. pombe led to a substantial reduction of H3K9me2 and H3K9me3 levels at the dh repeats, as well as a marked increase in the localization of RNA polymerase II. These results further substantiate our conclusion that the mutations on Clr4-Ub interfaces identified here are promising functional mutants for future biological studies.Although the crystals of KMT-H3 and KMT-Ub-H3 accommodate different crystal packing, the αC helix (residues 366 to 372), the SI region (residues 371 to 394), the visible part of the ARL (residues 453 to 472), the Clr4 C terminus (residues 486 to 490), and the β9/10 loop (residues 423 to 431), the primary focus of our work, do not interact directly with the symmetric molecules in both two crystal structures (fig. S14, F and G). Consequently, it can be deduced that these elements are minimally affected by crystal stacking. Furthermore, the conformational changes of the αC helix, the SI region, the ARL, and the Clr4 C terminus can be corroborated by previous structures, and the allostery of the β9/10 loop has been verified by Michaelis-Menten kinetics experiments of Clr4 H425A and F427A mutants. Collectively, crystal packing has little effect on our model interpretation.Clr4-mediated H3K9me creates binding sites for CD-containing proteins such as Swi6 (16, 57, 58), facilitating the formation of a repressive chromatin structure. Once a threshold level of H3K9me is achieved, Clr4 may rapidly diffuse across chromatin through a “read-write” mechanism mediated by the CD to establish heterochromatin and gene silencing. Our biochemical and structural analysis suggested that H3K14Ub may play a critical initiator role in the early establishment of H3K9me in fission yeast. Specifically, (i) the CLRC ubiquitin ligase complex is recruited by Ago1/RNAi-related complexes such as RNA-induced transcriptional silencing complex, facilitating histone H3K14Ub at specific loci while assisting in the recruitment of Clr4 (28–32). (ii) H3K14Ub can directly stimulate Clr4 activity irrespective of whether Clr4 is in an intrinsic autoinhibited state or not. (iii) H3K14Ub enables Clr4 to efficiently override the rate-limiting step (8) from dimethylation to trimethylation according to our enzymological kinetic measurement (fig. S15D), which produces the suitable substrate for the Clr4 CD, promoting the rapid accumulation of H3K9me3 and enhancing subsequent H3K9me diffusion. (iv) Ub is a relatively complicated and consumable process involving adenosine triphosphate, E1 activating enzymes, E2 conjugating enzymes, E3 ligases, and ubiquitin. It seems difficult for ubiquitin to function as an extensive factor in the rapid H3K9me diffusion process. Conversely, it is likely to be more suitable as the “gating” for H3K9 deposition. The integration of these regulatory pathways illustrates a sophisticated mechanism ensuring precise control over heterochromatin formation and maintenance, which is essential for appropriate gene regulation and chromosomal integrity.In recent years, multiple structural studies have demonstrated how histone Ub regulates the activity of HMTases, significantly improving our understanding of the mechanisms underlying Ub-methylation trans cross-talk (59–61). For instance, the monoubiquitination of histone H2B (H2BK120Ub) (62–66) and H2B lysine 34 (H2BK34Ub) (44, 67) stimulate the methylation activity of Dot1L on histone H3 lysine 79 (H3K79) (68, 69) via enzyme positioning and nucleosome distortion–induced orientation restriction, respectively. H2BK120ub can promote histone H3 lysine 4 methylation (H3K4me) through allosteric COMPASS (70, 71)/MLL (72–77). In addition, histone H2A lysine 119 monoubiquitination (H2AK119Ub) enhances the recruitment of PRC2 to increase the methylation level of histone H3 lysine 27 (H3K27) (78–80). These studies revealed various mechanisms by which histone Ub on the histone globular domain of nucleosomes establishes interchain cross-talk (i.e., in trans cross-talk) with methylation on nucleosomes. Stimulation of H3K9me by H3K14Ub represents a form of histone intrachain cis cross-talk that occurs on the histone tail and is independent of the histone globular domain. Our work provides structural insights into the ubiquitin-mediated cross-talk within the “histone tail network”, characterized by a synergistic mechanism that enhances both affinity and turnover of histone substrate.It remains to be investigated whether the structural mechanism of H3K14Ub-stimulated Clr4 is conserved in metazoans. While Clr4 is the sole H3K9 methyltransferase found in yeast, humans have various types of Suv39 family methyltransferases responsible for H3K9me. Notably, evidence of mammalian Suv39 family methyltransferase regulation by Ub has been reported, such as the essential role of K867 monoubiquitination in the SETDB1 SI-region for its enzymatic activity and function (81). The Ub site is close to the histone-binding groove, but this modification occurs on methyltransferases rather than the histone. Thus, further investigation is needed to elucidate the mechanism of this cross-talk. In addition, SUV39H2 (also known as KMT1B) was found to be enzymatically activated by H3K14Ub (40, 54). The structure of SUV39H2 is very similar to that of Clr4, and especially, the hydrophobic and electrostatic interfaces with ubiquitin are both preserved: Residue F189 of SUV39H2 may form the same critical hydrophobic interactions with ubiquitin as Clr4 F256; meanwhile in SUV39H2, the homology sites of Clr4 D280 and D281 are replaced by 204AE205, and Glu (E) likewise potentially form a salt bridge with Ub R42 (fig. S15, E and F). In contrast, none of the corresponding Phe (F) or acidic amino acids are conserved in the G9a and SUVH4 proteins, which is consistent with the previously reported phenomenon that they do not sense ubiquitin (fig. S15F). Other acylation modifications have been reported for H3K14 (82–84). Among them, H3K14ac is a marker of transcriptional activation (85), functionally similar to H3K9ac, which opposes the cross-talk of H3K14Ub-H3K9me3 associated with the heterochromatin formation (86, 87). Exploring how these acylation modifications affect Ub, heterochromatin formation, and maintenance of epigenetic traits may provide deeper insights into the intricate network of regulatory cross-talk mediated by Suv39 family methyltransferases.Chemical protein synthesis provides a powerful tool for investigating the functions of complex histone modifications (88). By chemically synthesizing histone substrates with precise modifications, we can introduce modifications that are challenging to achieve using traditional biological methods (e.g., recombinant expression) (89). In this work, this approach has allowed us to generate various samples for biochemical tests such as ubiquitinated peptides with native isopeptide bonds, structural analysis samples that mimic catalytic states (e.g., by replacing lysine with Nle), and hypothesis verification samples such as those constructed with AEEA to artificially extend peptide bonds (44, 90–93). These synthetically derived samples enable controlled investigations of the effects of individual post-translational modifications on enzymatic activity and chromatin structure, as well as the study of regulatory cross-talk among multiple modifications. The advancement of these synthetic strategies has significantly improved our understanding of histone modifications and their broader implications for epigenetic regulation.

Our study elucidates the molecular mechanism by which H3K14Ub significantly enhances the catalytic activity of Clr4 in the context of H3K9me, a process central to heterochromatin formation (1, 4, 5). Our structural and biochemical analyses revealed a synergistic mechanism involving both increased substrate affinity and allosteric activation (Fig. 6B). First, H3K14Ub dramatically increases Clr4’s affinity for histone substrate through multivalent hydrophobic patches and salt bridge interactions between the Clr4 pre-SET subdomain and ubiquitin. The increase in enzyme-substrate binding is highly specific to H3K14Ub, highlighting the critical role of this precise modification site. Second, H3K14Ub triggers a conformational shift in Clr4, transitioning it to a hyperactive catalyzing state. This state is characterized by a compacted αC helix-SI region, complete release of the ARL, and stable retraction of the β9/10 loop, which contribute to increased substrate turnover. In contrast, Clr4 K455/K472 automethylation, while contributing to Clr4 activity, demonstrates a less pronounced effect in comparison to H3K14Ub. Collectively, H3K14Ub acts as a potent dual-function regulator of Clr4, significantly boosting its activity through enhanced substrate binding and allosteric modulation, thereby playing a crucial role in the efficient establishment of heterochromatin.

Multilayered regulation of Clr4 KMT catalyzation.

(A) Overview of conformation changes in the regulatory and catalyzation of Clr4. The element colors in cartoon schematic diagrams of structures correspond to the font fill colors of their names. (B) Scheme of the synergetic mechanism of Clr4 stimulation involving both increased substrate affinity and allosteric activation by H3K14Ub.

By aligning our structures of catalyzing state and ubiquitin-induced “hyperactive state” of Clr4 with the previously resolved (34) structures of Clr4 before (“autoinhibited state,” PDB: 6BOX) and after (“activated state,” PDB: 6BP4) automethylation, we propose the perspective of different levels of conformational regulation mediated by these two pathways (Fig. 6A). Compared with the apo autoinhibited state, Clr4 in the activated state undergoes the relief of the ARL from the histone-binding groove and stabilization of the helix αC-SI region mediated by the ARL nascent C-terminal helix (residues 468 to 473) (fig. S15A). However, the helix αC-SI region and β9/10 loop remain in unaltered original conformations in the activated state, which is different from the catalyzing state (fig. S15, A and B). The ARL in the activated state structure is supposed to be further opened when Clr4 is transferred to the catalyzing state. Because the ARL N terminus (residues 451 to 455) in the activated state structure remains a steric clash with substrate H3 (fig. S15C) and the ARL C-terminal helix R469, I470 would impede the conformational shift of D371, K372, and the formation of D371-R8 hydrogen bonds (fig. S15C). Briefly, automethylation partially alleviates the clashes but cannot completely remove them. However, H3K14Ub induces a hyperactive state, involving the retracted β9/10 loop by the H425-pre-SET Y305/E306 and F427-Ub L8 interactions, as well as the shifted SI region contacting with β9/10 loop. Compared with automethylation, H3K14Ub adopts a distinct and likely more adequate mechanism to allosterically promote substrate turnover, contributing to the pronounced stimulation of H3K9me.

The identification of Clr4 ubiquitin–binding interfaces provides valuable insights into the regulation of Ub during gene silencing and heterochromatin formation. We found that mutation of F256 and D280/D281 of Clr4 significantly reduces the stimulatory effect of H3K14Ub on Clr4 activity without affecting its intrinsic H3K9me activity (Fig. 3E). The combined mutations of these amino acids present an ideal strategy for uncoupling the cross-talk between H3K14Ub and H3K9me, enabling the elucidation of the functional implications of this cross-talk for the downstream pathways associated with H3K9me. For example, to investigate the function of H3K4 demethylation affected by H3K14 acetylation (H3K14ac), a recent study resolved the structure of the demethylase complex LSD1-CoREST recognizing H3K14ac, engineered a protein mutant that can uncouple this cross-talk, and clarified the role of this cross-talk in cellular adhesion and myeloid leukocyte activation (56). Previous chromatin immunoprecipitation–qPCR study (40) introducing multiple mutations to the potential ubiquitin interface on Clr4 in S. pombe led to a substantial reduction of H3K9me2 and H3K9me3 levels at the dh repeats, as well as a marked increase in the localization of RNA polymerase II. These results further substantiate our conclusion that the mutations on Clr4-Ub interfaces identified here are promising functional mutants for future biological studies.

Although the crystals of KMT-H3 and KMT-Ub-H3 accommodate different crystal packing, the αC helix (residues 366 to 372), the SI region (residues 371 to 394), the visible part of the ARL (residues 453 to 472), the Clr4 C terminus (residues 486 to 490), and the β9/10 loop (residues 423 to 431), the primary focus of our work, do not interact directly with the symmetric molecules in both two crystal structures (fig. S14, F and G). Consequently, it can be deduced that these elements are minimally affected by crystal stacking. Furthermore, the conformational changes of the αC helix, the SI region, the ARL, and the Clr4 C terminus can be corroborated by previous structures, and the allostery of the β9/10 loop has been verified by Michaelis-Menten kinetics experiments of Clr4 H425A and F427A mutants. Collectively, crystal packing has little effect on our model interpretation.

Clr4-mediated H3K9me creates binding sites for CD-containing proteins such as Swi6 (16, 57, 58), facilitating the formation of a repressive chromatin structure. Once a threshold level of H3K9me is achieved, Clr4 may rapidly diffuse across chromatin through a “read-write” mechanism mediated by the CD to establish heterochromatin and gene silencing. Our biochemical and structural analysis suggested that H3K14Ub may play a critical initiator role in the early establishment of H3K9me in fission yeast. Specifically, (i) the CLRC ubiquitin ligase complex is recruited by Ago1/RNAi-related complexes such as RNA-induced transcriptional silencing complex, facilitating histone H3K14Ub at specific loci while assisting in the recruitment of Clr4 (28–32). (ii) H3K14Ub can directly stimulate Clr4 activity irrespective of whether Clr4 is in an intrinsic autoinhibited state or not. (iii) H3K14Ub enables Clr4 to efficiently override the rate-limiting step (8) from dimethylation to trimethylation according to our enzymological kinetic measurement (fig. S15D), which produces the suitable substrate for the Clr4 CD, promoting the rapid accumulation of H3K9me3 and enhancing subsequent H3K9me diffusion. (iv) Ub is a relatively complicated and consumable process involving adenosine triphosphate, E1 activating enzymes, E2 conjugating enzymes, E3 ligases, and ubiquitin. It seems difficult for ubiquitin to function as an extensive factor in the rapid H3K9me diffusion process. Conversely, it is likely to be more suitable as the “gating” for H3K9 deposition. The integration of these regulatory pathways illustrates a sophisticated mechanism ensuring precise control over heterochromatin formation and maintenance, which is essential for appropriate gene regulation and chromosomal integrity.

In recent years, multiple structural studies have demonstrated how histone Ub regulates the activity of HMTases, significantly improving our understanding of the mechanisms underlying Ub-methylation trans cross-talk (59–61). For instance, the monoubiquitination of histone H2B (H2BK120Ub) (62–66) and H2B lysine 34 (H2BK34Ub) (44, 67) stimulate the methylation activity of Dot1L on histone H3 lysine 79 (H3K79) (68, 69) via enzyme positioning and nucleosome distortion–induced orientation restriction, respectively. H2BK120ub can promote histone H3 lysine 4 methylation (H3K4me) through allosteric COMPASS (70, 71)/MLL (72–77). In addition, histone H2A lysine 119 monoubiquitination (H2AK119Ub) enhances the recruitment of PRC2 to increase the methylation level of histone H3 lysine 27 (H3K27) (78–80). These studies revealed various mechanisms by which histone Ub on the histone globular domain of nucleosomes establishes interchain cross-talk (i.e., in trans cross-talk) with methylation on nucleosomes. Stimulation of H3K9me by H3K14Ub represents a form of histone intrachain cis cross-talk that occurs on the histone tail and is independent of the histone globular domain. Our work provides structural insights into the ubiquitin-mediated cross-talk within the “histone tail network”, characterized by a synergistic mechanism that enhances both affinity and turnover of histone substrate.

It remains to be investigated whether the structural mechanism of H3K14Ub-stimulated Clr4 is conserved in metazoans. While Clr4 is the sole H3K9 methyltransferase found in yeast, humans have various types of Suv39 family methyltransferases responsible for H3K9me. Notably, evidence of mammalian Suv39 family methyltransferase regulation by Ub has been reported, such as the essential role of K867 monoubiquitination in the SETDB1 SI-region for its enzymatic activity and function (81). The Ub site is close to the histone-binding groove, but this modification occurs on methyltransferases rather than the histone. Thus, further investigation is needed to elucidate the mechanism of this cross-talk. In addition, SUV39H2 (also known as KMT1B) was found to be enzymatically activated by H3K14Ub (40, 54). The structure of SUV39H2 is very similar to that of Clr4, and especially, the hydrophobic and electrostatic interfaces with ubiquitin are both preserved: Residue F189 of SUV39H2 may form the same critical hydrophobic interactions with ubiquitin as Clr4 F256; meanwhile in SUV39H2, the homology sites of Clr4 D280 and D281 are replaced by 204AE205, and Glu (E) likewise potentially form a salt bridge with Ub R42 (fig. S15, E and F). In contrast, none of the corresponding Phe (F) or acidic amino acids are conserved in the G9a and SUVH4 proteins, which is consistent with the previously reported phenomenon that they do not sense ubiquitin (fig. S15F). Other acylation modifications have been reported for H3K14 (82–84). Among them, H3K14ac is a marker of transcriptional activation (85), functionally similar to H3K9ac, which opposes the cross-talk of H3K14Ub-H3K9me3 associated with the heterochromatin formation (86, 87). Exploring how these acylation modifications affect Ub, heterochromatin formation, and maintenance of epigenetic traits may provide deeper insights into the intricate network of regulatory cross-talk mediated by Suv39 family methyltransferases.

Chemical protein synthesis provides a powerful tool for investigating the functions of complex histone modifications (88). By chemically synthesizing histone substrates with precise modifications, we can introduce modifications that are challenging to achieve using traditional biological methods (e.g., recombinant expression) (89). In this work, this approach has allowed us to generate various samples for biochemical tests such as ubiquitinated peptides with native isopeptide bonds, structural analysis samples that mimic catalytic states (e.g., by replacing lysine with Nle), and hypothesis verification samples such as those constructed with AEEA to artificially extend peptide bonds (44, 90–93). These synthetically derived samples enable controlled investigations of the effects of individual post-translational modifications on enzymatic activity and chromatin structure, as well as the study of regulatory cross-talk among multiple modifications. The advancement of these synthetic strategies has significantly improved our understanding of histone modifications and their broader implications for epigenetic regulation.

MATERIALS AND METHODSCloning and plasmid constructionThe cDNA of S. pombe Clr4 (Suv39h) was synthesized by GenScript Biotech (Nanjing, China) and subcloned into pGEX6p-1 vector downstream the glutathione S-transferase (GST) tag followed by an HRV3C cleavage site. Clr4 truncations including Clr4-KMT for crystallization and biochemistry studies, Clr4-ΔCD, and all the Clr4 mutants (F256A, D280A/D281A, Y357A, D371A, F383A, D384A, D386A, H425A, F427A, F449A/Y451A, K455R/K472R, K455A/K472A, and K455W/K472W) were generated from the Clr4-pGEX6p-1 plasmid by standard site-directed PCR mutagenesis and homologous recombination, tagged by GST-HRV3C at N terminus as well. The DNA sequences encoding the WT Ub and UbG76C were cloned into the pET22b vector without tag. All the Ub mutants used in this work (R42A, L8R/H68A, L69A/V70A, L71A/L73A, and R72A/R74A) were generated from UbG76C pET22b plasmid. Genes of histone H2A, H2B, H3 (all Cys to Ser), and H4, were cloned into pET22b, respectively, without tag. H3 mutants H3K14C, H3K18C, and H3K23C were expressed in plasmid derived from H3-pET22b (all Cys to Ser). Meanwhile, the 147–base pair (bp) Widom 601 DNA we used for nucleosome assembly along with histones was 64× repeat inserted into a vector. The sequence of 147-bp Widom 601 DNA is as follows: CTGGAGAATCCCGGTGCCGAGGCCGCTCAATTGGTCGTAGACAGCTCTAGCACCGCTTAAACGCACGTACGCGCTGTCCCCCGCGTTTTAACCGCCAAGGGGATTACTCCCTAGTCTCCAGGCACGTGTCAGATATATACATCCTGT. At last, the genes of 3xFlag-tagged Clr4 or Clr4 mutants for in vivo assay were cloned into the pFA6a vector with hph+ resistance from D. Moazed by homologous recombination, resulting in plasmids pFA6a-hph-clr4up-3xFlag-clr4variants-clr4down for S. pombe transfection.Protein expression and purificationPlasmids containing WT full-length Clr4 (or its variants F256A, D280A/D281A, Y357A, D371A, F383A, D386A, F449A/Y451A, F427A, K455A/K472A, K455R/K472R, and K455W/K472W) were transformed into BL21 (DE3) Escherichia coli cells. The E. coli cells were grown in LB medium containing ampicillin (50 μg/ml) until an optical density at 600 nm (OD600) of 0.6. The cells were then induced by 0.4 mM isopropyl β-d-thiogalactopyranoside (IPTG) and 0.1 mM ZnCl2 at 16°C for 16 hours. The cells were collected by centrifugation at 4000 rpm for 30 min and then lysed by sonication in the ice-cold lysis buffer [25 mM Hepes (pH 7.3), 200 mM NaCl, 10 μM ZnCl2, 1 mM dithiothreitol (DTT), and 1 mM phenylmethyl sulfonyl fluoride (PMSF)]. After centrifugation at 14,000 rpm for 30 min, the supernatant was incubated with glutathione-Sepharose 4B beads (GE Healthcare) for 2 hours at 4°C. The proteins were eluted with the elution buffer [25 mM Hepes (pH 7.3), 200 mM NaCl, 10 μM ZnCl2, 1 mM DTT, and 30 mM glutathione (reduced form, GSH)]. The protein elution was incubated with PreScission Protease and dialyzed against 25 mM Hepes (pH 7.3) and 100 mM NaCl overnight at 4°C to remove the GST tag. The proteins were subsequently loaded into a Source 15S cation exchange column (GE Healthcare) equilibrated with the ion-exchange (IEX) buffer A [25 mM Hepes (pH 7.3) 50 mM NaCl) and then eluted by a gradient of the IEX buffer A and IEX buffer B [50 mM Hepes (pH 7.3) and 1 M NaCl]. The eluted Clr4 proteins were further purified by a Superdex 75 10/300 GL size-exclusion column (GE Healthcare) preequilibrated in size-exclusion chromatography (SEC) buffer [25 mM Hepes (pH 7.5) 100 mM NaCl]. The KMT domain of Clr4 [Clr4 (192–490)] and its variants were purified as described above, except that a Source 15Q anion exchange column (GE Healthcare) was used to purify the proteins after the PreScission Protease digestion. The proteins were characterized by SDS–polyacrylamide gel electrophoresis (SDS-PAGE) and visualized using a ChemiDoc MP Imaging System (Bio-Rad).Ubiquitin and its variants (R42A, L8R/H68A, L69/V70A, L71/L73A, and R72/R74A) were purified as previously described (44), and the Gly76 of ubiquitin (or its variants) was mutated to Cys for the synthesis of disulfide bond–linked Ub substrates. In brief, plasmids for expression were transformed into BL21 (DE3) competent cells. Cells were induced by 1 mM IPTG at 37°C overnight and were collected by centrifugation at 4000 rpm for 30 min. The cells were lysed by sonication in Ub lysis buffer [25 mM Hepes (pH 7.5) and 150 mM NaCl]. Cell lysates were supplemented with 1% perchloric acid and then clarified by centrifugation. The supernatant was dialyzed against 0.1% trifluoroacetic acid (TFA) overnight. The dialyzed mixture was further purified by a Source 15Q anion exchange column (GE Healthcare). The proteins were characterized by SDS-PAGE and visualized using a ChemiDoc MP Imaging System (Bio-Rad).Histones including H2A, H2B, H3 (C96S/C110S), H4, and their mutants (H3K14C, H3K18C, and H3K23C) were purified as previously described (44). In brief, histones expressed inclusion body in BL21 (DE3) E. coli cells and were dissolved in the unfolding buffer [6 M guanidinium chloride (Gn·HCl), 20 mM tris (pH 7.5), and 1 mM DTT]. After centrifugation at 14,000 rpm for 30 min, the clarified supernatant was dialyzed against 0.1% TFA overnight. The dialyzed mixture was purified by reversed-phase high-performance liquid chromatography (RP-HPLC) and characterized by liquid chromatography–electrospray ionization mass spectrometry (LC-ESI-MS).Peptides synthesisHistone H3 N-terminal peptides {H3(1–20) (H3t), H3(1–20)G12C, H3(1–20)K14C, H3(1–20)P16C, H3(3–19)K9Nle-K14C, H3(1–20)K9Nle [H3t (KNle9)]} and Ub(1–45)NHNH2 were synthesized using standard 9-fluorenyl methoxycarbonyl (Fmoc) solid-phase peptide synthesis (SPPS) protocols under standard microwave conditions (CEM Liberty Blue). Rink amino resin was used for the synthesis of C-terminal substrate, while hydrazine resin was used for the synthesis of hydrazine terminal substrate. The coupling cycle was programmed as previously reported (94). In brief, 0.5 mg of resin was used for coupling. Dimethylformamide (DMF) containing 10% piperidine and 0.1 M oxyma was used for deprotection. Fmoc-protected amino acids (0.2 M), 0.14 M N,N′-diisopropylcarbodiimide, and 0.7 M oxyma in DMF were used for amino acid coupling (10 min at 50°C for His and Cys, 90°C for other residues). After the coupling cycle, the resin was washed with dichloromethane (DCM) and treated with the cleavage mixture (1 ml of thioanisole, 1 ml of tri-isopropylsilane, 0.5 ml of 1,2-ethanedithiol, 1 ml of ddH2O, and 16.5 ml of TFA) for 2 hours at 37°C. The crude peptides were obtained by precipitating the cleavage mixture with cold diethyl ether and then purified by RP-HPLC and characterized by LC-ESI-MS.Synthesis of isopeptide bond–linked Ub substratesFor the synthesis of isopeptide bond–linked Ub substrates H3(1–20)K14Ub (Ub-H3t), H3(1–20)K9NleK14Ub, H3(1–20)K14AEEAUb (Ub-AEEA-H3t), and H3 terminal tail peptides [H3(1–20)K14Alloc and H3(1–20)K9NleK14Alloc] were first synthesized by SPPS protocol, except that a building block of Fmoc-Lys (Alloc)–OH was used for coupling at position Lys14. After the coupling cycle, the peptide-resin was incubated with the butoxycarbonyl (BOC) protection mixture (di-tertbutyl decarbonate: N,N-Diisopropylethylamine: DMF = 1:1:5) for 10 min to add the BOC protecting group at the N-terminal of the peptide. After protection, the resin was washed by DCM, and the Alloc protecting group was removed by incubating with Pd[P (C6H5)3]4 (60 mg) and Ph3SiH (600 μl) in 2 ml of DCM at 37°C overnight. The peptide resin was washed with sodium diethyldithiocarbamate (200 mg) in 40 ml of DMF to remove Pd. The peptide-resin was then used for SPPS and the ε-amino group on Lys14 can be further coupled with Ub (A46C-76). The products H3 (1-20)K14Ub (A46C-76), and H3(1–20)K9NleK14Ub (A46C-76) were purified from the resin by using the cleavage mixture as described above (95). H3(1–20)K14AEEAUb (A46C-76) was synthesized as described above, except that Fmoc-AEEA-OH was used as a building block in the subsequent coupling of the ε-amino group on Lys14 of H3(1–20).Ala46 in Ub was mutated to Cys to enable hydrazide-based native chemical ligation (96) for the next fragment Ub(1–45)NHNH2. In brief, Ub(1–45)NHNH2 (1 μM, 1.0 eq.) was dissolved in ligation buffer [6 M Gn·HCl and 100 mM NaH2PO4 (pH 2.3)] precooled to −15°C. Then, NaNO2 (10 μM, 10 eq.) was added, and the reaction was stirred for 30 min at −15°C to fully convert the hydrazide to acyl azide. The mixture was further treated with 4-mercaptophenylacetic acid (50 μM, 50 eq.), and the pH was adjusted to 5.0 to convert the acyl azide to thioester. Next, the N-terminal Cys peptides (1.2 μM, 1.2 eq.) were added into the mixture. Then, the pH was adjusted to 6.4 and stirred at 37°C overnight. After the reaction, 50 mM tris (2-carboxyethyl)phosphine (TCEP) was added to the mixture to reduce disulfide bonds and the pH was adjusted to 7.0. The products [H3(1-20)K14Ub (A46C), H3(1-20)K9NleK14Ub (A46C), and H3(1-20)K14AEEAUb (A46C)] were then purified by RP-HPLC.Then, the thiol group of Cys46 in Ub was removed by desulfurization reaction. The peptide was dissolved in a desulfurization buffer [6 M Gn·HCl and 100 mM Na2HPO4 (pH 7.4)]. Subsequently, GSH (230 mg μM−1 peptide), 500 mM TCEP, and 20 mM VA-044 were dissolved in desulfurization buffer and added to the peptide solution. The pH was adjusted to 7.0, and the reaction solution was stirred at 37°C overnight. The products were purified by RP-HPLC. All the proteins (or peptides) involved in this section were characterized by LC-ESI-MS and lyophilized into powder for the next use.To refold the Ub in H3 side chains, 6 mg of freeze-dried products were dissolved in 6 M Gn·HCl and 100 mM Hepes (pH 7.4). Next, 250 μl of 100 mM Hepes (pH 7.5) was added to reduce the concentration of Gn·HCl to 3 M, and then the solution was dialyzed into a dialysis buffer [1 M Gn·HCl and 25 mM Hepes (pH 7.5)]. After 6 hours, the dialysis buffer was replaced with no Gn·HCl buffer [25 mM Hepes (pH 7.5) and 150 Mm NaCl] for the next 6 hours of dialysis. Last, the product was purified by a Superdex 75 10/300 GL size-exclusion column (GE Healthcare) preequilibrated in SEC buffer [25 mM Hepes (pH 7.5) and 100 mM NaCl].Synthesis of disulfide bond linked Ub substratesH3 peptides containing Cys mutation [H3(1–20)G12C, H3(1–20)K14C, H3(1–20)P16C, H3(3–19)K9NleK14C, H3K14C, H3K18C, and H3K23C] were concentrated to 10 mg/ml in disulfide ligation buffer [6 M Gn·HCl and 100 mM Hepes (pH 7.5)]. Then, 5,5′-dithiobis- (2-nitrobenzoic acid) [Sigma-Aldrich; 3.0 eq., from 0.1 M stock solution prepared in 25 mM Hepes (pH 7.5) and 100 mM NaCl] was added. The reaction was placed at room temperature for 1 hour and quenched by adding HCl to adjust the pH to 1.0. All the H3-S-TNB products were purified by RP-HPLC and lyophilized into powder for the next use.The product H3-S-TNB (10 mg/ml, 1.0 eq.) and UbG76C (10 mg/ml, 1.0 eq., or its variants) were separately dissolved in the disulfide ligation buffer. Next, the UbG76C solution was titrated to the H3-S-TNB solution and the pH was adjusted to 7.0. The reaction was bathed at 30°C water for 15 min and quenched by adding HCl to adjust the pH to 1.0. All the H3-UbG76C products [H3(1–20)G12C-UbG76C (also noted as H3tGC12-Ub), H3(1–20)K14C-UbG76C, H3(1–20)P16C-UbG76C (also noted as H3tPC16-Ub), H3(3–19)K9Nle-K14C-UbG76C, H3(1–20)K14C-UbG76C (R42A), H3(1–20)K14C-UbG76C (L8R/H68A), H3(1–20)K14C-UbG76C (L69A/V70A), H3(1–20)K14C-UbG76C (L71A/L73A), H3(1–20)K14C-UbG76C (R72A/R74A), H3K14C-UbG76C, H3K18C-UbG76C, and H3K23C-UbG76C] were purified by RP-HPLC and lyophilized into powder for the next use.The refolding of the Ub modified on H3t was performed as described above. The full-length H3-UbG76C (H3K14C-UbG76C, H3K18C-UbG76C, and H3K23C-UbG76C) will be refolded in the process of octamer reconstitution and assembled into NCPH3K14Ub, NCPH3K18Ub, and NCPH3K23Ub.Crystallization and structure determinationFor the crystal structure of KMT-Ub-H3, the Clr4 KMT, H3(3–19)K9NleK14C-UbG76C, and SAM were incubated in the ratio of 1:1.05:5 at a final KMT concentration of 24.5 mg/ml for 6 hours, 4°C. Crystallization was under 291 K with sitting-drop vapor diffusion. The complex was first crystallized in the reservoir buffer of 20% polyethylene glycol, molecular weight 1000 (PEG-1000), 100 mM imidazole, and 200 mM CaAc2 (pH 7.0) (1:1). The initial crystals were broken down by ultrasonication in the reservoir buffer and then used as seeds in the crystallization system consisting of 0.5 μl of protein complex solution and 0.5 μl of 18 to 20% PEG-1000, 100 mM imidazole, and 200 mM CaAc2 (pH 7.0). The KMT-H3 complex sample was obtained by incubating the KMT, H3(3–19)K9NleK14C, and SAM at 1:10:10 with KMT (20 mg/ml). Satisfying crystals were also grown under 291 K with sitting-drop vapor diffusion, in the reservoir solution containing 0.1 M magnesium chloride hexahydrate, 0.1 M MES (pH 6.0), and 8% w/v PEG-6000. All harvested crystals were cryoprotected in the solution with 10% glycerol in the respective reservoir solution and preserved in liquid nitrogen.Both the diffraction of the crystals of KMT-Ub-H3 and KMT-H3 complex was collected under the wavelength of 0.979176 Å and temperature of 100 K at the SSRF BEAMLINE BL02U1 of Shanghai Synchrotron Radiation Facility. Raw diffraction data KMT-Ub-H3 and KMT-H3 complex were processed and scaled by Aimless 0.7.4 (97) and HKL2000 (98), respectively. We used the previously reported apo WT Clr4 KMT structure (PDB 6BP4) and Ub crystal structure (PDB 1UBQ) for molecular replacement phasing in PHENIX 1.19.2_4158 (99). The model building was visualized and operated by COOT (100), and collection and refinement statistics of our structures are presented in table S1.Enzyme/substrate mutation methyltransferase assayThe methyltransferase activity of WT Clr4 or its variants (F256A, D280A/D281A, Y357A, D371A, F383A, D384A, D386A, H425A, F427A, F449A/Y451A, K455A/K472A, K455R/K472R, and K455W/K472W) was detected by the MTase-Glo Methyltransferase Assay (Promega Corporation). In brief, 40 μl 2× enzyme solution [100/400 nM WT Clr4 or its variants (400 nM Clr4 was used when the substrate was unmodified H3t while 100 nM Clr4 for Ub-H3t), 80 μM SAM, 10 μM ZnCl2, diluted by SEC buffer] and 40 μl 2× substrate solution (80 μM H3t or 4 μM Ub-H3t, diluted by SEC buffer) were incubated at 30°C and then were mixed as the start of the reaction. The mixed solution was taken 8 μl at 0 min, 10 min, 20 min, or 1 hour and subsequently added into 384-well solid white assay plate prefilled with 2 μl 0.5% TFA at each time point to stop the reaction. Each well of the assay plate was added 2 μl 6× MTase-Glo reagent and incubated at room temperature for 30 min. Then, each well of the assay plate was added 12 μl 2× MTase-Glo Detection Solution and incubated at room temperature for 30 min. After the incubation, the luminescence was measured by using a plate-reading luminometer (BioTek). The influence of substrate variants in the Clr4 methyltransferase activity was detected in a similar protocol to that for Clr4 variants, except that only the WT Clr4 was used in 2× enzyme solution and the substrate variants [Ub-H3t mutants, H3tGC12-Ub, H3tPC16-Ub, Ub-AEEA-H3t, NCP (2 μM), NCPH3K14Ub (2 μM), NCPH3K18Ub (2 μM), and NCPH3K23Ub (2 μM)] were used in 2× substrate solution.The production of SAH was calculated by a standard curve using serial dilutions of SAH. The protocol was described as follows. SAH (2 μM) was serially diluted to half the concentration each time in the range of 2 to 0 μM (2, 1, 0.5, 0.25, 0.125, 0.063, 0.031, and 0 μM, 8 μl each sample). Consistent with the above processing, 2 μl of 0.5% TFA, 2 μl of 6× MTase-Glo reagent, and 12 μl of 2× MTase-Glo Detection Solution were added and incubated for the corresponding time. The luminescence was measured by the plate-reading luminometer. The standard curve was generated by plotting luminescence (y axis) against SAH concentration (x axis) so that the linear equation was established.Automethylated Clr4 was performed as in the previous report with minor modifications (34). In brief, 2.4 μM Clr4 or its variants were dialyzed in the HMT buffer [50 mM tris-HCl (pH 8), 20 mM KCl, 10 mM MgCl2, 1 mM DTT, 0.02% Triton, 5% glycerol, and 1 mM PMSF] supplemented with 250 μM SAM overnight at room temperature with mild agitation and then were dialyzed in the SEC buffer for 6 hours. After incubation, the methyltransferase activity of Clr4 was analyzed as described above.Determination of KmKm of WT Clr4 (or its variants Clr4-H425A, F427A, and K455A/K472A) with its substrates (Ub-H3t or H3t) was also determined by using MTase-Glo Methyltransferase Assay. Ub-H3t was serially diluted in SEC buffer with the concentration in a range of 16 to 0 μM (16, 8, 4, 2, 1, 0.5, 0.25, 0.125, and 0 μM, 55 μl of each sample) to prepare a series of substrate solution. Enzyme solution (2×, 20 nM Clr4 or its variants, 160 μM SAM, and 2 μM ZnCl2, diluted by SEC buffer) and 2× substrate solution were incubated at 30°C. Then, each substrate solution was added with 55 μl 2× enzyme solution to start the reaction. The reaction solution was taken for 8 μl at 6 min and then subsequently added into a 384-well solid white assay plate prefilled with 2 μl of 0.5% TFA to stop the reaction. Consistent with the methyltransferase assay, 2 μl of 6× MTase-Glo Reagent, and 12 μl of 2× MTase-Glo Detection Solution were then added and incubated for the corresponding time. When the substrate was H3t, the experiment process was the same as that of Ub-H3t, except that the enzyme concentration of 2× enzyme solution was changed to 200 nM and the concentration of 2× substrate was changed to a range of 128 to 0 μM and the sampling time was changed to 60 min.The luminescence was measured by the plate-reading luminometer. The production of SAH was also calculated by an SAH standard curve as described in the methyltransferase assay. Data fitting and analyses were performed using GraphPad Prism 9.0.ITC analysisThe ITC data were collected using a MicroCal PEAQ-ITC (Malven Pananlytical). Clr4 KMT and ligands [Ub, Ub-H3t, H3(1–20)K14C-UbG76C, H3(1–20)K14C-UbG76C (R42A), H3(1–20)K14C-UbG76C (L71A/L73A), and H3(1–20)K14C-UbG76C (L8R/H68A)] were buffer-exchanged into dialyzed overnight against ITC buffer [25 mM Na2HPO4 (pH 7.5) and 100 mM NaCl] before the experiments. For the experiments, 300 μl of 20 μM substrates solution in the sample cell was titrated with 200 μM Clr4 KMT solution through 19 injections (2.0 μl each) at 25°C and 750 rpm stirring speed. Data fitting and analyses were performed using PEAQ-ITC instrument client software.Thermo-shift assayThe thermos-shift of Clr4-KMT banding with substrates was evaluated using an UNcle all-in-one biologics stability screening platform (Unchained Labs). Clr4-KMT (60 μM, 1 eq.), SAM (600 μM, 10 eq.), and a corresponding substrate (60 μM, 1 eq.) were mixed in SEC buffer and loaded on the UNcle. The temperature started at 20°C and ended at 95°C with an increasing rate of 0.3 to 1°C/min. The fluorescence spectra of each sample were analyzed by UNcle Analysis software. The ΔTm was determined by the first derivative of the barycentric mean (BCM) of fluorescence intensity with respect to temperature (dBCM/dT).Reconstitution of octamers and nucleosomesNucleosomes containing unmodified H3, H3K14C-UbG76C, H3K18C-UbG76C, and H3K23C-UbG76C were reconstituted as previous report (101). In brief, the four core histones H2A, H2B, H3 (C96S/C110S), and H4 were mixed at a stoichiometry of 1.1:1.1:1:1 and dialyzed against the refolding buffer [10 mM Hepes (pH 7.5), 2 M NaCl, and 1 mM EDTA] for 18 hours with buffer changed every 6 hours. The desired octamers were purified by using a Superdex 200 10/300 GL size-exclusion column (GE Healthcare) preequilibrated in refolding buffer. Next, the nucleosomes were reconstituted by mixing the octamers and 147-bp Widom 601 DNA at a stoichiometry of 1:1.1 in the refolding buffer, and the NaCl concentration was gradient reduced to below 200 mM by adding HE buffer [10 mM Hepes (pH 7.5) and 1 mM EDTA] with the use of a peristaltic pump at 4°C overnight. To further reduce the NaCl concentration, the mixture was dialyzed against the HE buffer at 4°C overnight. Assembled nucleosomes were purified by using anion exchange on a DEAE-5PW column (TSKgel) preequilibrated in HE buffer. Desired nucleosomes were eluted by a gradient of HE buffer and 1 M NaCl HE buffer. Octamers and nucleosomes were characterized by SDS-PAGE (GeneScript).Electrophoretic mobility shift assayWT full-length Clr4 was prepared as a twofold dilution series in SEC buffer (25 mM HEPES, pH 7.5, 100 mM NaCl) in a range of 8 μM to 0.625 μM. Next, 5 μL Clr4 with indicated concentrations was mixed with 5 μL 200 nM NCP (or NCPH3K14Ub, NCPH3K18Ub) diluted by the HE buffer. The mixed samples were incubated for 10 min at 4°C. Five microliter of Native gel loading [0.15% (v/v) bromophenol blue, 50% (v/v) glycerol, and 0.25% (v/v) bovine serum albumin] was then added to each sample and mixed well. Subsequently, 5 μl of each sample was resolved by 4.5% native-PAGE gels running in a precooled 0.5 × TBE buffer [40 mM tris-HCl (pH 8.3), 45 mM boric acid, and 1 mM EDTA] at 150 V for 45 min in ice water. Gels were stained with 0.01% (v/v) SYBR-Gold dye (Thermo Fisher Scientific) and visualized using a ChemiDoc MP Imaging System (Bio-Rad).Strains construction and silencing assaysAll yeast mutants used in this article (except the Clr4 knockout strain) were transformed from gift strains (SPY7283) obtained from Moazed laboratory, and the transformation method was the same as reported before (34). Among them, the Clr4 knockout strain (SPY7305) was also obtained from the Moazed laboratory. The mutant construction was confirmed by PCR amplification and sequencing to determine its correctness. Western blot was performed on cell lysate of different strains using ANTI-FLAG M2-Peroxidase [horseradish peroxidase (HRP)] antibodies (A8592-1MG, Sigma-Aldrich, 1/1000 dilution) to detect the expression level of Flag-Clr4 (WT and mutants) in different strains.Silencing experiments were performed similarly to the previously reported article (34). Cells were harvested at a final concentration of 1 × 107 cells/ml, washed once with sterile water, resuspended in 150 μl of sterile water, and then serially diluted 10-fold. 3 μl of each dilution was dropped onto the appropriate growth medium (YE). The plates were incubated in a 32°C incubator for 2 days and then left at 4°C for 36 hours to enhance red pigmentation before imaging to assess the silencing of the ade6+ reporter gene.Each mutant strain was cultured in 5 ml of YES (OD600 = 1), and cells were harvested. Yeast cells were lysed using strong acid (0.8 M HCl), followed by centrifugation to isolate nuclear histones. The histones were precipitated with trichloroacetic acid, and excess acid was removed by acetone washing. The histone pellet was dissolved in loading buffer and subjected to SDS-PAGE, followed by Western blot analysis using anti-H3K9me3 antibodies (ab8898, Abcam, 1/1000 dilution).Total RNA extraction and reverse transcriptionYeast cultures were cultivated until they reached the exponential phase (OD600 = 1.5), after which 50 OD600 units of cells were harvested. Cell lysis was carried out using the acid phenol-chloro method. The extracted RNA was then treated with Turbo DNase (Thermo Fisher Scientific) and subjected to a second round of purification with the RNeasy Mini Kit (Qiagen). A total of 500 ng of high-purity RNA was reverse transcribed using the Superscript III kit (Invitrogen) with primers listed in table S2. The resulting cDNA was quantified by qPCR on an Applied Biosystems qPCR instrument, with fbp1+ as the internal control. Statistical analysis was conducted using data from three biological replicates.Western blot for methyltransferase activityNCP (WT NCP, NCPH3K14Ub; 200 nM), SAM (100 μM, 500 eq.), ZnCl2 (10 μM, 50 eq.) were diluted by SEC buffer and incubated at 30°C. Reactions were commenced by the addition of Clr4 (200 nM, 1 eq.). One microliter was removed from each reaction at 10, 20, 60, 120, 200, and 300 min and then boiled with LDS sample buffer (Thermo Fisher Scientific) at 95°C for 5 min. Boiled samples were resolved on a 4 to 12% gradient SDS-PAGE gel at 160 V for 25 min. Gels were transferred onto polyvinylidene difluoride membranes by eBlotTM L1 (GenScript Biotech). Membranes were blocked for 40 min with QuickBlock blocking buffer (Beyotime) and incubated with corresponding primary antibodies at 4°C overnight. After incubation, membranes were washed with TBST buffer [50 mM tris-HCl (pH 7.6), 150 mM NaCl, and 0.1% (v/v) Tween 20] for 5 min for five times, and then membranes were incubated with HRP-conjugated secondary antibody for 45 min at room temperature followed by 5-min TBST buffer wash for five times. Last, membranes were treated with an enhanced chemiluminesence substrate (Clarity MaxTM Western, Bio-Rad) and visualized using a ChemiDoc MP Imaging System (Bio-Rad). Antibodies were diluted by antibody dilution buffer (Primary & Secondary Antibody Diluent for WB, YEASEN). Antibody information is listed as follows: rabbit polyclonal anti-H3K9me1 antibody (39888, Active Motif, 1/2000 dilution), mouse monoclonal anti-H3K9me2 antibody (ab1220, Abcam, 1/4000 dilution), rabbit polyclonal anti-H3K9me3 antibody (ab8898, Abcam, 1/5000 dilution), rabbit polyclonal anti-H2A antibody (ab18255, Abcam, 1/1000 dilution), HRP* goat anti-mouse immunoglobulin G (IgG) (H + L) (RS0001, Immunoway, 1/10000 dilution), and HRP* goat anti-rabbit IgG (H + L) (RS0002, Immunoway, 1/10000 dilution).

MATERIALS AND METHODS

Cloning and plasmid constructionThe cDNA of S. pombe Clr4 (Suv39h) was synthesized by GenScript Biotech (Nanjing, China) and subcloned into pGEX6p-1 vector downstream the glutathione S-transferase (GST) tag followed by an HRV3C cleavage site. Clr4 truncations including Clr4-KMT for crystallization and biochemistry studies, Clr4-ΔCD, and all the Clr4 mutants (F256A, D280A/D281A, Y357A, D371A, F383A, D384A, D386A, H425A, F427A, F449A/Y451A, K455R/K472R, K455A/K472A, and K455W/K472W) were generated from the Clr4-pGEX6p-1 plasmid by standard site-directed PCR mutagenesis and homologous recombination, tagged by GST-HRV3C at N terminus as well. The DNA sequences encoding the WT Ub and UbG76C were cloned into the pET22b vector without tag. All the Ub mutants used in this work (R42A, L8R/H68A, L69A/V70A, L71A/L73A, and R72A/R74A) were generated from UbG76C pET22b plasmid. Genes of histone H2A, H2B, H3 (all Cys to Ser), and H4, were cloned into pET22b, respectively, without tag. H3 mutants H3K14C, H3K18C, and H3K23C were expressed in plasmid derived from H3-pET22b (all Cys to Ser). Meanwhile, the 147–base pair (bp) Widom 601 DNA we used for nucleosome assembly along with histones was 64× repeat inserted into a vector. The sequence of 147-bp Widom 601 DNA is as follows: CTGGAGAATCCCGGTGCCGAGGCCGCTCAATTGGTCGTAGACAGCTCTAGCACCGCTTAAACGCACGTACGCGCTGTCCCCCGCGTTTTAACCGCCAAGGGGATTACTCCCTAGTCTCCAGGCACGTGTCAGATATATACATCCTGT. At last, the genes of 3xFlag-tagged Clr4 or Clr4 mutants for in vivo assay were cloned into the pFA6a vector with hph+ resistance from D. Moazed by homologous recombination, resulting in plasmids pFA6a-hph-clr4up-3xFlag-clr4variants-clr4down for S. pombe transfection.

Cloning and plasmid construction

The cDNA of S. pombe Clr4 (Suv39h) was synthesized by GenScript Biotech (Nanjing, China) and subcloned into pGEX6p-1 vector downstream the glutathione S-transferase (GST) tag followed by an HRV3C cleavage site. Clr4 truncations including Clr4-KMT for crystallization and biochemistry studies, Clr4-ΔCD, and all the Clr4 mutants (F256A, D280A/D281A, Y357A, D371A, F383A, D384A, D386A, H425A, F427A, F449A/Y451A, K455R/K472R, K455A/K472A, and K455W/K472W) were generated from the Clr4-pGEX6p-1 plasmid by standard site-directed PCR mutagenesis and homologous recombination, tagged by GST-HRV3C at N terminus as well. The DNA sequences encoding the WT Ub and UbG76C were cloned into the pET22b vector without tag. All the Ub mutants used in this work (R42A, L8R/H68A, L69A/V70A, L71A/L73A, and R72A/R74A) were generated from UbG76C pET22b plasmid. Genes of histone H2A, H2B, H3 (all Cys to Ser), and H4, were cloned into pET22b, respectively, without tag. H3 mutants H3K14C, H3K18C, and H3K23C were expressed in plasmid derived from H3-pET22b (all Cys to Ser). Meanwhile, the 147–base pair (bp) Widom 601 DNA we used for nucleosome assembly along with histones was 64× repeat inserted into a vector. The sequence of 147-bp Widom 601 DNA is as follows: CTGGAGAATCCCGGTGCCGAGGCCGCTCAATTGGTCGTAGACAGCTCTAGCACCGCTTAAACGCACGTACGCGCTGTCCCCCGCGTTTTAACCGCCAAGGGGATTACTCCCTAGTCTCCAGGCACGTGTCAGATATATACATCCTGT. At last, the genes of 3xFlag-tagged Clr4 or Clr4 mutants for in vivo assay were cloned into the pFA6a vector with hph+ resistance from D. Moazed by homologous recombination, resulting in plasmids pFA6a-hph-clr4up-3xFlag-clr4variants-clr4down for S. pombe transfection.

Protein expression and purificationPlasmids containing WT full-length Clr4 (or its variants F256A, D280A/D281A, Y357A, D371A, F383A, D386A, F449A/Y451A, F427A, K455A/K472A, K455R/K472R, and K455W/K472W) were transformed into BL21 (DE3) Escherichia coli cells. The E. coli cells were grown in LB medium containing ampicillin (50 μg/ml) until an optical density at 600 nm (OD600) of 0.6. The cells were then induced by 0.4 mM isopropyl β-d-thiogalactopyranoside (IPTG) and 0.1 mM ZnCl2 at 16°C for 16 hours. The cells were collected by centrifugation at 4000 rpm for 30 min and then lysed by sonication in the ice-cold lysis buffer [25 mM Hepes (pH 7.3), 200 mM NaCl, 10 μM ZnCl2, 1 mM dithiothreitol (DTT), and 1 mM phenylmethyl sulfonyl fluoride (PMSF)]. After centrifugation at 14,000 rpm for 30 min, the supernatant was incubated with glutathione-Sepharose 4B beads (GE Healthcare) for 2 hours at 4°C. The proteins were eluted with the elution buffer [25 mM Hepes (pH 7.3), 200 mM NaCl, 10 μM ZnCl2, 1 mM DTT, and 30 mM glutathione (reduced form, GSH)]. The protein elution was incubated with PreScission Protease and dialyzed against 25 mM Hepes (pH 7.3) and 100 mM NaCl overnight at 4°C to remove the GST tag. The proteins were subsequently loaded into a Source 15S cation exchange column (GE Healthcare) equilibrated with the ion-exchange (IEX) buffer A [25 mM Hepes (pH 7.3) 50 mM NaCl) and then eluted by a gradient of the IEX buffer A and IEX buffer B [50 mM Hepes (pH 7.3) and 1 M NaCl]. The eluted Clr4 proteins were further purified by a Superdex 75 10/300 GL size-exclusion column (GE Healthcare) preequilibrated in size-exclusion chromatography (SEC) buffer [25 mM Hepes (pH 7.5) 100 mM NaCl]. The KMT domain of Clr4 [Clr4 (192–490)] and its variants were purified as described above, except that a Source 15Q anion exchange column (GE Healthcare) was used to purify the proteins after the PreScission Protease digestion. The proteins were characterized by SDS–polyacrylamide gel electrophoresis (SDS-PAGE) and visualized using a ChemiDoc MP Imaging System (Bio-Rad).Ubiquitin and its variants (R42A, L8R/H68A, L69/V70A, L71/L73A, and R72/R74A) were purified as previously described (44), and the Gly76 of ubiquitin (or its variants) was mutated to Cys for the synthesis of disulfide bond–linked Ub substrates. In brief, plasmids for expression were transformed into BL21 (DE3) competent cells. Cells were induced by 1 mM IPTG at 37°C overnight and were collected by centrifugation at 4000 rpm for 30 min. The cells were lysed by sonication in Ub lysis buffer [25 mM Hepes (pH 7.5) and 150 mM NaCl]. Cell lysates were supplemented with 1% perchloric acid and then clarified by centrifugation. The supernatant was dialyzed against 0.1% trifluoroacetic acid (TFA) overnight. The dialyzed mixture was further purified by a Source 15Q anion exchange column (GE Healthcare). The proteins were characterized by SDS-PAGE and visualized using a ChemiDoc MP Imaging System (Bio-Rad).Histones including H2A, H2B, H3 (C96S/C110S), H4, and their mutants (H3K14C, H3K18C, and H3K23C) were purified as previously described (44). In brief, histones expressed inclusion body in BL21 (DE3) E. coli cells and were dissolved in the unfolding buffer [6 M guanidinium chloride (Gn·HCl), 20 mM tris (pH 7.5), and 1 mM DTT]. After centrifugation at 14,000 rpm for 30 min, the clarified supernatant was dialyzed against 0.1% TFA overnight. The dialyzed mixture was purified by reversed-phase high-performance liquid chromatography (RP-HPLC) and characterized by liquid chromatography–electrospray ionization mass spectrometry (LC-ESI-MS).

Protein expression and purification

Plasmids containing WT full-length Clr4 (or its variants F256A, D280A/D281A, Y357A, D371A, F383A, D386A, F449A/Y451A, F427A, K455A/K472A, K455R/K472R, and K455W/K472W) were transformed into BL21 (DE3) Escherichia coli cells. The E. coli cells were grown in LB medium containing ampicillin (50 μg/ml) until an optical density at 600 nm (OD600) of 0.6. The cells were then induced by 0.4 mM isopropyl β-d-thiogalactopyranoside (IPTG) and 0.1 mM ZnCl2 at 16°C for 16 hours. The cells were collected by centrifugation at 4000 rpm for 30 min and then lysed by sonication in the ice-cold lysis buffer [25 mM Hepes (pH 7.3), 200 mM NaCl, 10 μM ZnCl2, 1 mM dithiothreitol (DTT), and 1 mM phenylmethyl sulfonyl fluoride (PMSF)]. After centrifugation at 14,000 rpm for 30 min, the supernatant was incubated with glutathione-Sepharose 4B beads (GE Healthcare) for 2 hours at 4°C. The proteins were eluted with the elution buffer [25 mM Hepes (pH 7.3), 200 mM NaCl, 10 μM ZnCl2, 1 mM DTT, and 30 mM glutathione (reduced form, GSH)]. The protein elution was incubated with PreScission Protease and dialyzed against 25 mM Hepes (pH 7.3) and 100 mM NaCl overnight at 4°C to remove the GST tag. The proteins were subsequently loaded into a Source 15S cation exchange column (GE Healthcare) equilibrated with the ion-exchange (IEX) buffer A [25 mM Hepes (pH 7.3) 50 mM NaCl) and then eluted by a gradient of the IEX buffer A and IEX buffer B [50 mM Hepes (pH 7.3) and 1 M NaCl]. The eluted Clr4 proteins were further purified by a Superdex 75 10/300 GL size-exclusion column (GE Healthcare) preequilibrated in size-exclusion chromatography (SEC) buffer [25 mM Hepes (pH 7.5) 100 mM NaCl]. The KMT domain of Clr4 [Clr4 (192–490)] and its variants were purified as described above, except that a Source 15Q anion exchange column (GE Healthcare) was used to purify the proteins after the PreScission Protease digestion. The proteins were characterized by SDS–polyacrylamide gel electrophoresis (SDS-PAGE) and visualized using a ChemiDoc MP Imaging System (Bio-Rad).

Ubiquitin and its variants (R42A, L8R/H68A, L69/V70A, L71/L73A, and R72/R74A) were purified as previously described (44), and the Gly76 of ubiquitin (or its variants) was mutated to Cys for the synthesis of disulfide bond–linked Ub substrates. In brief, plasmids for expression were transformed into BL21 (DE3) competent cells. Cells were induced by 1 mM IPTG at 37°C overnight and were collected by centrifugation at 4000 rpm for 30 min. The cells were lysed by sonication in Ub lysis buffer [25 mM Hepes (pH 7.5) and 150 mM NaCl]. Cell lysates were supplemented with 1% perchloric acid and then clarified by centrifugation. The supernatant was dialyzed against 0.1% trifluoroacetic acid (TFA) overnight. The dialyzed mixture was further purified by a Source 15Q anion exchange column (GE Healthcare). The proteins were characterized by SDS-PAGE and visualized using a ChemiDoc MP Imaging System (Bio-Rad).

Histones including H2A, H2B, H3 (C96S/C110S), H4, and their mutants (H3K14C, H3K18C, and H3K23C) were purified as previously described (44). In brief, histones expressed inclusion body in BL21 (DE3) E. coli cells and were dissolved in the unfolding buffer [6 M guanidinium chloride (Gn·HCl), 20 mM tris (pH 7.5), and 1 mM DTT]. After centrifugation at 14,000 rpm for 30 min, the clarified supernatant was dialyzed against 0.1% TFA overnight. The dialyzed mixture was purified by reversed-phase high-performance liquid chromatography (RP-HPLC) and characterized by liquid chromatography–electrospray ionization mass spectrometry (LC-ESI-MS).

Peptides synthesisHistone H3 N-terminal peptides {H3(1–20) (H3t), H3(1–20)G12C, H3(1–20)K14C, H3(1–20)P16C, H3(3–19)K9Nle-K14C, H3(1–20)K9Nle [H3t (KNle9)]} and Ub(1–45)NHNH2 were synthesized using standard 9-fluorenyl methoxycarbonyl (Fmoc) solid-phase peptide synthesis (SPPS) protocols under standard microwave conditions (CEM Liberty Blue). Rink amino resin was used for the synthesis of C-terminal substrate, while hydrazine resin was used for the synthesis of hydrazine terminal substrate. The coupling cycle was programmed as previously reported (94). In brief, 0.5 mg of resin was used for coupling. Dimethylformamide (DMF) containing 10% piperidine and 0.1 M oxyma was used for deprotection. Fmoc-protected amino acids (0.2 M), 0.14 M N,N′-diisopropylcarbodiimide, and 0.7 M oxyma in DMF were used for amino acid coupling (10 min at 50°C for His and Cys, 90°C for other residues). After the coupling cycle, the resin was washed with dichloromethane (DCM) and treated with the cleavage mixture (1 ml of thioanisole, 1 ml of tri-isopropylsilane, 0.5 ml of 1,2-ethanedithiol, 1 ml of ddH2O, and 16.5 ml of TFA) for 2 hours at 37°C. The crude peptides were obtained by precipitating the cleavage mixture with cold diethyl ether and then purified by RP-HPLC and characterized by LC-ESI-MS.

Peptides synthesis

Histone H3 N-terminal peptides {H3(1–20) (H3t), H3(1–20)G12C, H3(1–20)K14C, H3(1–20)P16C, H3(3–19)K9Nle-K14C, H3(1–20)K9Nle [H3t (KNle9)]} and Ub(1–45)NHNH2 were synthesized using standard 9-fluorenyl methoxycarbonyl (Fmoc) solid-phase peptide synthesis (SPPS) protocols under standard microwave conditions (CEM Liberty Blue). Rink amino resin was used for the synthesis of C-terminal substrate, while hydrazine resin was used for the synthesis of hydrazine terminal substrate. The coupling cycle was programmed as previously reported (94). In brief, 0.5 mg of resin was used for coupling. Dimethylformamide (DMF) containing 10% piperidine and 0.1 M oxyma was used for deprotection. Fmoc-protected amino acids (0.2 M), 0.14 M N,N′-diisopropylcarbodiimide, and 0.7 M oxyma in DMF were used for amino acid coupling (10 min at 50°C for His and Cys, 90°C for other residues). After the coupling cycle, the resin was washed with dichloromethane (DCM) and treated with the cleavage mixture (1 ml of thioanisole, 1 ml of tri-isopropylsilane, 0.5 ml of 1,2-ethanedithiol, 1 ml of ddH2O, and 16.5 ml of TFA) for 2 hours at 37°C. The crude peptides were obtained by precipitating the cleavage mixture with cold diethyl ether and then purified by RP-HPLC and characterized by LC-ESI-MS.

Synthesis of isopeptide bond–linked Ub substratesFor the synthesis of isopeptide bond–linked Ub substrates H3(1–20)K14Ub (Ub-H3t), H3(1–20)K9NleK14Ub, H3(1–20)K14AEEAUb (Ub-AEEA-H3t), and H3 terminal tail peptides [H3(1–20)K14Alloc and H3(1–20)K9NleK14Alloc] were first synthesized by SPPS protocol, except that a building block of Fmoc-Lys (Alloc)–OH was used for coupling at position Lys14. After the coupling cycle, the peptide-resin was incubated with the butoxycarbonyl (BOC) protection mixture (di-tertbutyl decarbonate: N,N-Diisopropylethylamine: DMF = 1:1:5) for 10 min to add the BOC protecting group at the N-terminal of the peptide. After protection, the resin was washed by DCM, and the Alloc protecting group was removed by incubating with Pd[P (C6H5)3]4 (60 mg) and Ph3SiH (600 μl) in 2 ml of DCM at 37°C overnight. The peptide resin was washed with sodium diethyldithiocarbamate (200 mg) in 40 ml of DMF to remove Pd. The peptide-resin was then used for SPPS and the ε-amino group on Lys14 can be further coupled with Ub (A46C-76). The products H3 (1-20)K14Ub (A46C-76), and H3(1–20)K9NleK14Ub (A46C-76) were purified from the resin by using the cleavage mixture as described above (95). H3(1–20)K14AEEAUb (A46C-76) was synthesized as described above, except that Fmoc-AEEA-OH was used as a building block in the subsequent coupling of the ε-amino group on Lys14 of H3(1–20).Ala46 in Ub was mutated to Cys to enable hydrazide-based native chemical ligation (96) for the next fragment Ub(1–45)NHNH2. In brief, Ub(1–45)NHNH2 (1 μM, 1.0 eq.) was dissolved in ligation buffer [6 M Gn·HCl and 100 mM NaH2PO4 (pH 2.3)] precooled to −15°C. Then, NaNO2 (10 μM, 10 eq.) was added, and the reaction was stirred for 30 min at −15°C to fully convert the hydrazide to acyl azide. The mixture was further treated with 4-mercaptophenylacetic acid (50 μM, 50 eq.), and the pH was adjusted to 5.0 to convert the acyl azide to thioester. Next, the N-terminal Cys peptides (1.2 μM, 1.2 eq.) were added into the mixture. Then, the pH was adjusted to 6.4 and stirred at 37°C overnight. After the reaction, 50 mM tris (2-carboxyethyl)phosphine (TCEP) was added to the mixture to reduce disulfide bonds and the pH was adjusted to 7.0. The products [H3(1-20)K14Ub (A46C), H3(1-20)K9NleK14Ub (A46C), and H3(1-20)K14AEEAUb (A46C)] were then purified by RP-HPLC.Then, the thiol group of Cys46 in Ub was removed by desulfurization reaction. The peptide was dissolved in a desulfurization buffer [6 M Gn·HCl and 100 mM Na2HPO4 (pH 7.4)]. Subsequently, GSH (230 mg μM−1 peptide), 500 mM TCEP, and 20 mM VA-044 were dissolved in desulfurization buffer and added to the peptide solution. The pH was adjusted to 7.0, and the reaction solution was stirred at 37°C overnight. The products were purified by RP-HPLC. All the proteins (or peptides) involved in this section were characterized by LC-ESI-MS and lyophilized into powder for the next use.To refold the Ub in H3 side chains, 6 mg of freeze-dried products were dissolved in 6 M Gn·HCl and 100 mM Hepes (pH 7.4). Next, 250 μl of 100 mM Hepes (pH 7.5) was added to reduce the concentration of Gn·HCl to 3 M, and then the solution was dialyzed into a dialysis buffer [1 M Gn·HCl and 25 mM Hepes (pH 7.5)]. After 6 hours, the dialysis buffer was replaced with no Gn·HCl buffer [25 mM Hepes (pH 7.5) and 150 Mm NaCl] for the next 6 hours of dialysis. Last, the product was purified by a Superdex 75 10/300 GL size-exclusion column (GE Healthcare) preequilibrated in SEC buffer [25 mM Hepes (pH 7.5) and 100 mM NaCl].

Synthesis of isopeptide bond–linked Ub substrates

For the synthesis of isopeptide bond–linked Ub substrates H3(1–20)K14Ub (Ub-H3t), H3(1–20)K9NleK14Ub, H3(1–20)K14AEEAUb (Ub-AEEA-H3t), and H3 terminal tail peptides [H3(1–20)K14Alloc and H3(1–20)K9NleK14Alloc] were first synthesized by SPPS protocol, except that a building block of Fmoc-Lys (Alloc)–OH was used for coupling at position Lys14. After the coupling cycle, the peptide-resin was incubated with the butoxycarbonyl (BOC) protection mixture (di-tertbutyl decarbonate: N,N-Diisopropylethylamine: DMF = 1:1:5) for 10 min to add the BOC protecting group at the N-terminal of the peptide. After protection, the resin was washed by DCM, and the Alloc protecting group was removed by incubating with Pd[P (C6H5)3]4 (60 mg) and Ph3SiH (600 μl) in 2 ml of DCM at 37°C overnight. The peptide resin was washed with sodium diethyldithiocarbamate (200 mg) in 40 ml of DMF to remove Pd. The peptide-resin was then used for SPPS and the ε-amino group on Lys14 can be further coupled with Ub (A46C-76). The products H3 (1-20)K14Ub (A46C-76), and H3(1–20)K9NleK14Ub (A46C-76) were purified from the resin by using the cleavage mixture as described above (95). H3(1–20)K14AEEAUb (A46C-76) was synthesized as described above, except that Fmoc-AEEA-OH was used as a building block in the subsequent coupling of the ε-amino group on Lys14 of H3(1–20).

Ala46 in Ub was mutated to Cys to enable hydrazide-based native chemical ligation (96) for the next fragment Ub(1–45)NHNH2. In brief, Ub(1–45)NHNH2 (1 μM, 1.0 eq.) was dissolved in ligation buffer [6 M Gn·HCl and 100 mM NaH2PO4 (pH 2.3)] precooled to −15°C. Then, NaNO2 (10 μM, 10 eq.) was added, and the reaction was stirred for 30 min at −15°C to fully convert the hydrazide to acyl azide. The mixture was further treated with 4-mercaptophenylacetic acid (50 μM, 50 eq.), and the pH was adjusted to 5.0 to convert the acyl azide to thioester. Next, the N-terminal Cys peptides (1.2 μM, 1.2 eq.) were added into the mixture. Then, the pH was adjusted to 6.4 and stirred at 37°C overnight. After the reaction, 50 mM tris (2-carboxyethyl)phosphine (TCEP) was added to the mixture to reduce disulfide bonds and the pH was adjusted to 7.0. The products [H3(1-20)K14Ub (A46C), H3(1-20)K9NleK14Ub (A46C), and H3(1-20)K14AEEAUb (A46C)] were then purified by RP-HPLC.

Then, the thiol group of Cys46 in Ub was removed by desulfurization reaction. The peptide was dissolved in a desulfurization buffer [6 M Gn·HCl and 100 mM Na2HPO4 (pH 7.4)]. Subsequently, GSH (230 mg μM−1 peptide), 500 mM TCEP, and 20 mM VA-044 were dissolved in desulfurization buffer and added to the peptide solution. The pH was adjusted to 7.0, and the reaction solution was stirred at 37°C overnight. The products were purified by RP-HPLC. All the proteins (or peptides) involved in this section were characterized by LC-ESI-MS and lyophilized into powder for the next use.

To refold the Ub in H3 side chains, 6 mg of freeze-dried products were dissolved in 6 M Gn·HCl and 100 mM Hepes (pH 7.4). Next, 250 μl of 100 mM Hepes (pH 7.5) was added to reduce the concentration of Gn·HCl to 3 M, and then the solution was dialyzed into a dialysis buffer [1 M Gn·HCl and 25 mM Hepes (pH 7.5)]. After 6 hours, the dialysis buffer was replaced with no Gn·HCl buffer [25 mM Hepes (pH 7.5) and 150 Mm NaCl] for the next 6 hours of dialysis. Last, the product was purified by a Superdex 75 10/300 GL size-exclusion column (GE Healthcare) preequilibrated in SEC buffer [25 mM Hepes (pH 7.5) and 100 mM NaCl].

Synthesis of disulfide bond linked Ub substratesH3 peptides containing Cys mutation [H3(1–20)G12C, H3(1–20)K14C, H3(1–20)P16C, H3(3–19)K9NleK14C, H3K14C, H3K18C, and H3K23C] were concentrated to 10 mg/ml in disulfide ligation buffer [6 M Gn·HCl and 100 mM Hepes (pH 7.5)]. Then, 5,5′-dithiobis- (2-nitrobenzoic acid) [Sigma-Aldrich; 3.0 eq., from 0.1 M stock solution prepared in 25 mM Hepes (pH 7.5) and 100 mM NaCl] was added. The reaction was placed at room temperature for 1 hour and quenched by adding HCl to adjust the pH to 1.0. All the H3-S-TNB products were purified by RP-HPLC and lyophilized into powder for the next use.The product H3-S-TNB (10 mg/ml, 1.0 eq.) and UbG76C (10 mg/ml, 1.0 eq., or its variants) were separately dissolved in the disulfide ligation buffer. Next, the UbG76C solution was titrated to the H3-S-TNB solution and the pH was adjusted to 7.0. The reaction was bathed at 30°C water for 15 min and quenched by adding HCl to adjust the pH to 1.0. All the H3-UbG76C products [H3(1–20)G12C-UbG76C (also noted as H3tGC12-Ub), H3(1–20)K14C-UbG76C, H3(1–20)P16C-UbG76C (also noted as H3tPC16-Ub), H3(3–19)K9Nle-K14C-UbG76C, H3(1–20)K14C-UbG76C (R42A), H3(1–20)K14C-UbG76C (L8R/H68A), H3(1–20)K14C-UbG76C (L69A/V70A), H3(1–20)K14C-UbG76C (L71A/L73A), H3(1–20)K14C-UbG76C (R72A/R74A), H3K14C-UbG76C, H3K18C-UbG76C, and H3K23C-UbG76C] were purified by RP-HPLC and lyophilized into powder for the next use.The refolding of the Ub modified on H3t was performed as described above. The full-length H3-UbG76C (H3K14C-UbG76C, H3K18C-UbG76C, and H3K23C-UbG76C) will be refolded in the process of octamer reconstitution and assembled into NCPH3K14Ub, NCPH3K18Ub, and NCPH3K23Ub.

Synthesis of disulfide bond linked Ub substrates

H3 peptides containing Cys mutation [H3(1–20)G12C, H3(1–20)K14C, H3(1–20)P16C, H3(3–19)K9NleK14C, H3K14C, H3K18C, and H3K23C] were concentrated to 10 mg/ml in disulfide ligation buffer [6 M Gn·HCl and 100 mM Hepes (pH 7.5)]. Then, 5,5′-dithiobis- (2-nitrobenzoic acid) [Sigma-Aldrich; 3.0 eq., from 0.1 M stock solution prepared in 25 mM Hepes (pH 7.5) and 100 mM NaCl] was added. The reaction was placed at room temperature for 1 hour and quenched by adding HCl to adjust the pH to 1.0. All the H3-S-TNB products were purified by RP-HPLC and lyophilized into powder for the next use.

The product H3-S-TNB (10 mg/ml, 1.0 eq.) and UbG76C (10 mg/ml, 1.0 eq., or its variants) were separately dissolved in the disulfide ligation buffer. Next, the UbG76C solution was titrated to the H3-S-TNB solution and the pH was adjusted to 7.0. The reaction was bathed at 30°C water for 15 min and quenched by adding HCl to adjust the pH to 1.0. All the H3-UbG76C products [H3(1–20)G12C-UbG76C (also noted as H3tGC12-Ub), H3(1–20)K14C-UbG76C, H3(1–20)P16C-UbG76C (also noted as H3tPC16-Ub), H3(3–19)K9Nle-K14C-UbG76C, H3(1–20)K14C-UbG76C (R42A), H3(1–20)K14C-UbG76C (L8R/H68A), H3(1–20)K14C-UbG76C (L69A/V70A), H3(1–20)K14C-UbG76C (L71A/L73A), H3(1–20)K14C-UbG76C (R72A/R74A), H3K14C-UbG76C, H3K18C-UbG76C, and H3K23C-UbG76C] were purified by RP-HPLC and lyophilized into powder for the next use.

The refolding of the Ub modified on H3t was performed as described above. The full-length H3-UbG76C (H3K14C-UbG76C, H3K18C-UbG76C, and H3K23C-UbG76C) will be refolded in the process of octamer reconstitution and assembled into NCPH3K14Ub, NCPH3K18Ub, and NCPH3K23Ub.

Crystallization and structure determinationFor the crystal structure of KMT-Ub-H3, the Clr4 KMT, H3(3–19)K9NleK14C-UbG76C, and SAM were incubated in the ratio of 1:1.05:5 at a final KMT concentration of 24.5 mg/ml for 6 hours, 4°C. Crystallization was under 291 K with sitting-drop vapor diffusion. The complex was first crystallized in the reservoir buffer of 20% polyethylene glycol, molecular weight 1000 (PEG-1000), 100 mM imidazole, and 200 mM CaAc2 (pH 7.0) (1:1). The initial crystals were broken down by ultrasonication in the reservoir buffer and then used as seeds in the crystallization system consisting of 0.5 μl of protein complex solution and 0.5 μl of 18 to 20% PEG-1000, 100 mM imidazole, and 200 mM CaAc2 (pH 7.0). The KMT-H3 complex sample was obtained by incubating the KMT, H3(3–19)K9NleK14C, and SAM at 1:10:10 with KMT (20 mg/ml). Satisfying crystals were also grown under 291 K with sitting-drop vapor diffusion, in the reservoir solution containing 0.1 M magnesium chloride hexahydrate, 0.1 M MES (pH 6.0), and 8% w/v PEG-6000. All harvested crystals were cryoprotected in the solution with 10% glycerol in the respective reservoir solution and preserved in liquid nitrogen.Both the diffraction of the crystals of KMT-Ub-H3 and KMT-H3 complex was collected under the wavelength of 0.979176 Å and temperature of 100 K at the SSRF BEAMLINE BL02U1 of Shanghai Synchrotron Radiation Facility. Raw diffraction data KMT-Ub-H3 and KMT-H3 complex were processed and scaled by Aimless 0.7.4 (97) and HKL2000 (98), respectively. We used the previously reported apo WT Clr4 KMT structure (PDB 6BP4) and Ub crystal structure (PDB 1UBQ) for molecular replacement phasing in PHENIX 1.19.2_4158 (99). The model building was visualized and operated by COOT (100), and collection and refinement statistics of our structures are presented in table S1.

Crystallization and structure determination

For the crystal structure of KMT-Ub-H3, the Clr4 KMT, H3(3–19)K9NleK14C-UbG76C, and SAM were incubated in the ratio of 1:1.05:5 at a final KMT concentration of 24.5 mg/ml for 6 hours, 4°C. Crystallization was under 291 K with sitting-drop vapor diffusion. The complex was first crystallized in the reservoir buffer of 20% polyethylene glycol, molecular weight 1000 (PEG-1000), 100 mM imidazole, and 200 mM CaAc2 (pH 7.0) (1:1). The initial crystals were broken down by ultrasonication in the reservoir buffer and then used as seeds in the crystallization system consisting of 0.5 μl of protein complex solution and 0.5 μl of 18 to 20% PEG-1000, 100 mM imidazole, and 200 mM CaAc2 (pH 7.0). The KMT-H3 complex sample was obtained by incubating the KMT, H3(3–19)K9NleK14C, and SAM at 1:10:10 with KMT (20 mg/ml). Satisfying crystals were also grown under 291 K with sitting-drop vapor diffusion, in the reservoir solution containing 0.1 M magnesium chloride hexahydrate, 0.1 M MES (pH 6.0), and 8% w/v PEG-6000. All harvested crystals were cryoprotected in the solution with 10% glycerol in the respective reservoir solution and preserved in liquid nitrogen.

Both the diffraction of the crystals of KMT-Ub-H3 and KMT-H3 complex was collected under the wavelength of 0.979176 Å and temperature of 100 K at the SSRF BEAMLINE BL02U1 of Shanghai Synchrotron Radiation Facility. Raw diffraction data KMT-Ub-H3 and KMT-H3 complex were processed and scaled by Aimless 0.7.4 (97) and HKL2000 (98), respectively. We used the previously reported apo WT Clr4 KMT structure (PDB 6BP4) and Ub crystal structure (PDB 1UBQ) for molecular replacement phasing in PHENIX 1.19.2_4158 (99). The model building was visualized and operated by COOT (100), and collection and refinement statistics of our structures are presented in table S1.

Enzyme/substrate mutation methyltransferase assayThe methyltransferase activity of WT Clr4 or its variants (F256A, D280A/D281A, Y357A, D371A, F383A, D384A, D386A, H425A, F427A, F449A/Y451A, K455A/K472A, K455R/K472R, and K455W/K472W) was detected by the MTase-Glo Methyltransferase Assay (Promega Corporation). In brief, 40 μl 2× enzyme solution [100/400 nM WT Clr4 or its variants (400 nM Clr4 was used when the substrate was unmodified H3t while 100 nM Clr4 for Ub-H3t), 80 μM SAM, 10 μM ZnCl2, diluted by SEC buffer] and 40 μl 2× substrate solution (80 μM H3t or 4 μM Ub-H3t, diluted by SEC buffer) were incubated at 30°C and then were mixed as the start of the reaction. The mixed solution was taken 8 μl at 0 min, 10 min, 20 min, or 1 hour and subsequently added into 384-well solid white assay plate prefilled with 2 μl 0.5% TFA at each time point to stop the reaction. Each well of the assay plate was added 2 μl 6× MTase-Glo reagent and incubated at room temperature for 30 min. Then, each well of the assay plate was added 12 μl 2× MTase-Glo Detection Solution and incubated at room temperature for 30 min. After the incubation, the luminescence was measured by using a plate-reading luminometer (BioTek). The influence of substrate variants in the Clr4 methyltransferase activity was detected in a similar protocol to that for Clr4 variants, except that only the WT Clr4 was used in 2× enzyme solution and the substrate variants [Ub-H3t mutants, H3tGC12-Ub, H3tPC16-Ub, Ub-AEEA-H3t, NCP (2 μM), NCPH3K14Ub (2 μM), NCPH3K18Ub (2 μM), and NCPH3K23Ub (2 μM)] were used in 2× substrate solution.The production of SAH was calculated by a standard curve using serial dilutions of SAH. The protocol was described as follows. SAH (2 μM) was serially diluted to half the concentration each time in the range of 2 to 0 μM (2, 1, 0.5, 0.25, 0.125, 0.063, 0.031, and 0 μM, 8 μl each sample). Consistent with the above processing, 2 μl of 0.5% TFA, 2 μl of 6× MTase-Glo reagent, and 12 μl of 2× MTase-Glo Detection Solution were added and incubated for the corresponding time. The luminescence was measured by the plate-reading luminometer. The standard curve was generated by plotting luminescence (y axis) against SAH concentration (x axis) so that the linear equation was established.Automethylated Clr4 was performed as in the previous report with minor modifications (34). In brief, 2.4 μM Clr4 or its variants were dialyzed in the HMT buffer [50 mM tris-HCl (pH 8), 20 mM KCl, 10 mM MgCl2, 1 mM DTT, 0.02% Triton, 5% glycerol, and 1 mM PMSF] supplemented with 250 μM SAM overnight at room temperature with mild agitation and then were dialyzed in the SEC buffer for 6 hours. After incubation, the methyltransferase activity of Clr4 was analyzed as described above.

Enzyme/substrate mutation methyltransferase assay

The methyltransferase activity of WT Clr4 or its variants (F256A, D280A/D281A, Y357A, D371A, F383A, D384A, D386A, H425A, F427A, F449A/Y451A, K455A/K472A, K455R/K472R, and K455W/K472W) was detected by the MTase-Glo Methyltransferase Assay (Promega Corporation). In brief, 40 μl 2× enzyme solution [100/400 nM WT Clr4 or its variants (400 nM Clr4 was used when the substrate was unmodified H3t while 100 nM Clr4 for Ub-H3t), 80 μM SAM, 10 μM ZnCl2, diluted by SEC buffer] and 40 μl 2× substrate solution (80 μM H3t or 4 μM Ub-H3t, diluted by SEC buffer) were incubated at 30°C and then were mixed as the start of the reaction. The mixed solution was taken 8 μl at 0 min, 10 min, 20 min, or 1 hour and subsequently added into 384-well solid white assay plate prefilled with 2 μl 0.5% TFA at each time point to stop the reaction. Each well of the assay plate was added 2 μl 6× MTase-Glo reagent and incubated at room temperature for 30 min. Then, each well of the assay plate was added 12 μl 2× MTase-Glo Detection Solution and incubated at room temperature for 30 min. After the incubation, the luminescence was measured by using a plate-reading luminometer (BioTek). The influence of substrate variants in the Clr4 methyltransferase activity was detected in a similar protocol to that for Clr4 variants, except that only the WT Clr4 was used in 2× enzyme solution and the substrate variants [Ub-H3t mutants, H3tGC12-Ub, H3tPC16-Ub, Ub-AEEA-H3t, NCP (2 μM), NCPH3K14Ub (2 μM), NCPH3K18Ub (2 μM), and NCPH3K23Ub (2 μM)] were used in 2× substrate solution.

The production of SAH was calculated by a standard curve using serial dilutions of SAH. The protocol was described as follows. SAH (2 μM) was serially diluted to half the concentration each time in the range of 2 to 0 μM (2, 1, 0.5, 0.25, 0.125, 0.063, 0.031, and 0 μM, 8 μl each sample). Consistent with the above processing, 2 μl of 0.5% TFA, 2 μl of 6× MTase-Glo reagent, and 12 μl of 2× MTase-Glo Detection Solution were added and incubated for the corresponding time. The luminescence was measured by the plate-reading luminometer. The standard curve was generated by plotting luminescence (y axis) against SAH concentration (x axis) so that the linear equation was established.

Automethylated Clr4 was performed as in the previous report with minor modifications (34). In brief, 2.4 μM Clr4 or its variants were dialyzed in the HMT buffer [50 mM tris-HCl (pH 8), 20 mM KCl, 10 mM MgCl2, 1 mM DTT, 0.02% Triton, 5% glycerol, and 1 mM PMSF] supplemented with 250 μM SAM overnight at room temperature with mild agitation and then were dialyzed in the SEC buffer for 6 hours. After incubation, the methyltransferase activity of Clr4 was analyzed as described above.

Determination of KmKm of WT Clr4 (or its variants Clr4-H425A, F427A, and K455A/K472A) with its substrates (Ub-H3t or H3t) was also determined by using MTase-Glo Methyltransferase Assay. Ub-H3t was serially diluted in SEC buffer with the concentration in a range of 16 to 0 μM (16, 8, 4, 2, 1, 0.5, 0.25, 0.125, and 0 μM, 55 μl of each sample) to prepare a series of substrate solution. Enzyme solution (2×, 20 nM Clr4 or its variants, 160 μM SAM, and 2 μM ZnCl2, diluted by SEC buffer) and 2× substrate solution were incubated at 30°C. Then, each substrate solution was added with 55 μl 2× enzyme solution to start the reaction. The reaction solution was taken for 8 μl at 6 min and then subsequently added into a 384-well solid white assay plate prefilled with 2 μl of 0.5% TFA to stop the reaction. Consistent with the methyltransferase assay, 2 μl of 6× MTase-Glo Reagent, and 12 μl of 2× MTase-Glo Detection Solution were then added and incubated for the corresponding time. When the substrate was H3t, the experiment process was the same as that of Ub-H3t, except that the enzyme concentration of 2× enzyme solution was changed to 200 nM and the concentration of 2× substrate was changed to a range of 128 to 0 μM and the sampling time was changed to 60 min.The luminescence was measured by the plate-reading luminometer. The production of SAH was also calculated by an SAH standard curve as described in the methyltransferase assay. Data fitting and analyses were performed using GraphPad Prism 9.0.

Determination of Km

Km of WT Clr4 (or its variants Clr4-H425A, F427A, and K455A/K472A) with its substrates (Ub-H3t or H3t) was also determined by using MTase-Glo Methyltransferase Assay. Ub-H3t was serially diluted in SEC buffer with the concentration in a range of 16 to 0 μM (16, 8, 4, 2, 1, 0.5, 0.25, 0.125, and 0 μM, 55 μl of each sample) to prepare a series of substrate solution. Enzyme solution (2×, 20 nM Clr4 or its variants, 160 μM SAM, and 2 μM ZnCl2, diluted by SEC buffer) and 2× substrate solution were incubated at 30°C. Then, each substrate solution was added with 55 μl 2× enzyme solution to start the reaction. The reaction solution was taken for 8 μl at 6 min and then subsequently added into a 384-well solid white assay plate prefilled with 2 μl of 0.5% TFA to stop the reaction. Consistent with the methyltransferase assay, 2 μl of 6× MTase-Glo Reagent, and 12 μl of 2× MTase-Glo Detection Solution were then added and incubated for the corresponding time. When the substrate was H3t, the experiment process was the same as that of Ub-H3t, except that the enzyme concentration of 2× enzyme solution was changed to 200 nM and the concentration of 2× substrate was changed to a range of 128 to 0 μM and the sampling time was changed to 60 min.

The luminescence was measured by the plate-reading luminometer. The production of SAH was also calculated by an SAH standard curve as described in the methyltransferase assay. Data fitting and analyses were performed using GraphPad Prism 9.0.

ITC analysisThe ITC data were collected using a MicroCal PEAQ-ITC (Malven Pananlytical). Clr4 KMT and ligands [Ub, Ub-H3t, H3(1–20)K14C-UbG76C, H3(1–20)K14C-UbG76C (R42A), H3(1–20)K14C-UbG76C (L71A/L73A), and H3(1–20)K14C-UbG76C (L8R/H68A)] were buffer-exchanged into dialyzed overnight against ITC buffer [25 mM Na2HPO4 (pH 7.5) and 100 mM NaCl] before the experiments. For the experiments, 300 μl of 20 μM substrates solution in the sample cell was titrated with 200 μM Clr4 KMT solution through 19 injections (2.0 μl each) at 25°C and 750 rpm stirring speed. Data fitting and analyses were performed using PEAQ-ITC instrument client software.

ITC analysis

The ITC data were collected using a MicroCal PEAQ-ITC (Malven Pananlytical). Clr4 KMT and ligands [Ub, Ub-H3t, H3(1–20)K14C-UbG76C, H3(1–20)K14C-UbG76C (R42A), H3(1–20)K14C-UbG76C (L71A/L73A), and H3(1–20)K14C-UbG76C (L8R/H68A)] were buffer-exchanged into dialyzed overnight against ITC buffer [25 mM Na2HPO4 (pH 7.5) and 100 mM NaCl] before the experiments. For the experiments, 300 μl of 20 μM substrates solution in the sample cell was titrated with 200 μM Clr4 KMT solution through 19 injections (2.0 μl each) at 25°C and 750 rpm stirring speed. Data fitting and analyses were performed using PEAQ-ITC instrument client software.

Thermo-shift assayThe thermos-shift of Clr4-KMT banding with substrates was evaluated using an UNcle all-in-one biologics stability screening platform (Unchained Labs). Clr4-KMT (60 μM, 1 eq.), SAM (600 μM, 10 eq.), and a corresponding substrate (60 μM, 1 eq.) were mixed in SEC buffer and loaded on the UNcle. The temperature started at 20°C and ended at 95°C with an increasing rate of 0.3 to 1°C/min. The fluorescence spectra of each sample were analyzed by UNcle Analysis software. The ΔTm was determined by the first derivative of the barycentric mean (BCM) of fluorescence intensity with respect to temperature (dBCM/dT).

Thermo-shift assay

The thermos-shift of Clr4-KMT banding with substrates was evaluated using an UNcle all-in-one biologics stability screening platform (Unchained Labs). Clr4-KMT (60 μM, 1 eq.), SAM (600 μM, 10 eq.), and a corresponding substrate (60 μM, 1 eq.) were mixed in SEC buffer and loaded on the UNcle. The temperature started at 20°C and ended at 95°C with an increasing rate of 0.3 to 1°C/min. The fluorescence spectra of each sample were analyzed by UNcle Analysis software. The ΔTm was determined by the first derivative of the barycentric mean (BCM) of fluorescence intensity with respect to temperature (dBCM/dT).

Reconstitution of octamers and nucleosomesNucleosomes containing unmodified H3, H3K14C-UbG76C, H3K18C-UbG76C, and H3K23C-UbG76C were reconstituted as previous report (101). In brief, the four core histones H2A, H2B, H3 (C96S/C110S), and H4 were mixed at a stoichiometry of 1.1:1.1:1:1 and dialyzed against the refolding buffer [10 mM Hepes (pH 7.5), 2 M NaCl, and 1 mM EDTA] for 18 hours with buffer changed every 6 hours. The desired octamers were purified by using a Superdex 200 10/300 GL size-exclusion column (GE Healthcare) preequilibrated in refolding buffer. Next, the nucleosomes were reconstituted by mixing the octamers and 147-bp Widom 601 DNA at a stoichiometry of 1:1.1 in the refolding buffer, and the NaCl concentration was gradient reduced to below 200 mM by adding HE buffer [10 mM Hepes (pH 7.5) and 1 mM EDTA] with the use of a peristaltic pump at 4°C overnight. To further reduce the NaCl concentration, the mixture was dialyzed against the HE buffer at 4°C overnight. Assembled nucleosomes were purified by using anion exchange on a DEAE-5PW column (TSKgel) preequilibrated in HE buffer. Desired nucleosomes were eluted by a gradient of HE buffer and 1 M NaCl HE buffer. Octamers and nucleosomes were characterized by SDS-PAGE (GeneScript).

Reconstitution of octamers and nucleosomes

Nucleosomes containing unmodified H3, H3K14C-UbG76C, H3K18C-UbG76C, and H3K23C-UbG76C were reconstituted as previous report (101). In brief, the four core histones H2A, H2B, H3 (C96S/C110S), and H4 were mixed at a stoichiometry of 1.1:1.1:1:1 and dialyzed against the refolding buffer [10 mM Hepes (pH 7.5), 2 M NaCl, and 1 mM EDTA] for 18 hours with buffer changed every 6 hours. The desired octamers were purified by using a Superdex 200 10/300 GL size-exclusion column (GE Healthcare) preequilibrated in refolding buffer. Next, the nucleosomes were reconstituted by mixing the octamers and 147-bp Widom 601 DNA at a stoichiometry of 1:1.1 in the refolding buffer, and the NaCl concentration was gradient reduced to below 200 mM by adding HE buffer [10 mM Hepes (pH 7.5) and 1 mM EDTA] with the use of a peristaltic pump at 4°C overnight. To further reduce the NaCl concentration, the mixture was dialyzed against the HE buffer at 4°C overnight. Assembled nucleosomes were purified by using anion exchange on a DEAE-5PW column (TSKgel) preequilibrated in HE buffer. Desired nucleosomes were eluted by a gradient of HE buffer and 1 M NaCl HE buffer. Octamers and nucleosomes were characterized by SDS-PAGE (GeneScript).

Electrophoretic mobility shift assayWT full-length Clr4 was prepared as a twofold dilution series in SEC buffer (25 mM HEPES, pH 7.5, 100 mM NaCl) in a range of 8 μM to 0.625 μM. Next, 5 μL Clr4 with indicated concentrations was mixed with 5 μL 200 nM NCP (or NCPH3K14Ub, NCPH3K18Ub) diluted by the HE buffer. The mixed samples were incubated for 10 min at 4°C. Five microliter of Native gel loading [0.15% (v/v) bromophenol blue, 50% (v/v) glycerol, and 0.25% (v/v) bovine serum albumin] was then added to each sample and mixed well. Subsequently, 5 μl of each sample was resolved by 4.5% native-PAGE gels running in a precooled 0.5 × TBE buffer [40 mM tris-HCl (pH 8.3), 45 mM boric acid, and 1 mM EDTA] at 150 V for 45 min in ice water. Gels were stained with 0.01% (v/v) SYBR-Gold dye (Thermo Fisher Scientific) and visualized using a ChemiDoc MP Imaging System (Bio-Rad).

Electrophoretic mobility shift assay

WT full-length Clr4 was prepared as a twofold dilution series in SEC buffer (25 mM HEPES, pH 7.5, 100 mM NaCl) in a range of 8 μM to 0.625 μM. Next, 5 μL Clr4 with indicated concentrations was mixed with 5 μL 200 nM NCP (or NCPH3K14Ub, NCPH3K18Ub) diluted by the HE buffer. The mixed samples were incubated for 10 min at 4°C. Five microliter of Native gel loading [0.15% (v/v) bromophenol blue, 50% (v/v) glycerol, and 0.25% (v/v) bovine serum albumin] was then added to each sample and mixed well. Subsequently, 5 μl of each sample was resolved by 4.5% native-PAGE gels running in a precooled 0.5 × TBE buffer [40 mM tris-HCl (pH 8.3), 45 mM boric acid, and 1 mM EDTA] at 150 V for 45 min in ice water. Gels were stained with 0.01% (v/v) SYBR-Gold dye (Thermo Fisher Scientific) and visualized using a ChemiDoc MP Imaging System (Bio-Rad).

Strains construction and silencing assaysAll yeast mutants used in this article (except the Clr4 knockout strain) were transformed from gift strains (SPY7283) obtained from Moazed laboratory, and the transformation method was the same as reported before (34). Among them, the Clr4 knockout strain (SPY7305) was also obtained from the Moazed laboratory. The mutant construction was confirmed by PCR amplification and sequencing to determine its correctness. Western blot was performed on cell lysate of different strains using ANTI-FLAG M2-Peroxidase [horseradish peroxidase (HRP)] antibodies (A8592-1MG, Sigma-Aldrich, 1/1000 dilution) to detect the expression level of Flag-Clr4 (WT and mutants) in different strains.Silencing experiments were performed similarly to the previously reported article (34). Cells were harvested at a final concentration of 1 × 107 cells/ml, washed once with sterile water, resuspended in 150 μl of sterile water, and then serially diluted 10-fold. 3 μl of each dilution was dropped onto the appropriate growth medium (YE). The plates were incubated in a 32°C incubator for 2 days and then left at 4°C for 36 hours to enhance red pigmentation before imaging to assess the silencing of the ade6+ reporter gene.Each mutant strain was cultured in 5 ml of YES (OD600 = 1), and cells were harvested. Yeast cells were lysed using strong acid (0.8 M HCl), followed by centrifugation to isolate nuclear histones. The histones were precipitated with trichloroacetic acid, and excess acid was removed by acetone washing. The histone pellet was dissolved in loading buffer and subjected to SDS-PAGE, followed by Western blot analysis using anti-H3K9me3 antibodies (ab8898, Abcam, 1/1000 dilution).

Strains construction and silencing assays

All yeast mutants used in this article (except the Clr4 knockout strain) were transformed from gift strains (SPY7283) obtained from Moazed laboratory, and the transformation method was the same as reported before (34). Among them, the Clr4 knockout strain (SPY7305) was also obtained from the Moazed laboratory. The mutant construction was confirmed by PCR amplification and sequencing to determine its correctness. Western blot was performed on cell lysate of different strains using ANTI-FLAG M2-Peroxidase [horseradish peroxidase (HRP)] antibodies (A8592-1MG, Sigma-Aldrich, 1/1000 dilution) to detect the expression level of Flag-Clr4 (WT and mutants) in different strains.

Silencing experiments were performed similarly to the previously reported article (34). Cells were harvested at a final concentration of 1 × 107 cells/ml, washed once with sterile water, resuspended in 150 μl of sterile water, and then serially diluted 10-fold. 3 μl of each dilution was dropped onto the appropriate growth medium (YE). The plates were incubated in a 32°C incubator for 2 days and then left at 4°C for 36 hours to enhance red pigmentation before imaging to assess the silencing of the ade6+ reporter gene.

Each mutant strain was cultured in 5 ml of YES (OD600 = 1), and cells were harvested. Yeast cells were lysed using strong acid (0.8 M HCl), followed by centrifugation to isolate nuclear histones. The histones were precipitated with trichloroacetic acid, and excess acid was removed by acetone washing. The histone pellet was dissolved in loading buffer and subjected to SDS-PAGE, followed by Western blot analysis using anti-H3K9me3 antibodies (ab8898, Abcam, 1/1000 dilution).

Total RNA extraction and reverse transcriptionYeast cultures were cultivated until they reached the exponential phase (OD600 = 1.5), after which 50 OD600 units of cells were harvested. Cell lysis was carried out using the acid phenol-chloro method. The extracted RNA was then treated with Turbo DNase (Thermo Fisher Scientific) and subjected to a second round of purification with the RNeasy Mini Kit (Qiagen). A total of 500 ng of high-purity RNA was reverse transcribed using the Superscript III kit (Invitrogen) with primers listed in table S2. The resulting cDNA was quantified by qPCR on an Applied Biosystems qPCR instrument, with fbp1+ as the internal control. Statistical analysis was conducted using data from three biological replicates.

Total RNA extraction and reverse transcription

Yeast cultures were cultivated until they reached the exponential phase (OD600 = 1.5), after which 50 OD600 units of cells were harvested. Cell lysis was carried out using the acid phenol-chloro method. The extracted RNA was then treated with Turbo DNase (Thermo Fisher Scientific) and subjected to a second round of purification with the RNeasy Mini Kit (Qiagen). A total of 500 ng of high-purity RNA was reverse transcribed using the Superscript III kit (Invitrogen) with primers listed in table S2. The resulting cDNA was quantified by qPCR on an Applied Biosystems qPCR instrument, with fbp1+ as the internal control. Statistical analysis was conducted using data from three biological replicates.

Western blot for methyltransferase activityNCP (WT NCP, NCPH3K14Ub; 200 nM), SAM (100 μM, 500 eq.), ZnCl2 (10 μM, 50 eq.) were diluted by SEC buffer and incubated at 30°C. Reactions were commenced by the addition of Clr4 (200 nM, 1 eq.). One microliter was removed from each reaction at 10, 20, 60, 120, 200, and 300 min and then boiled with LDS sample buffer (Thermo Fisher Scientific) at 95°C for 5 min. Boiled samples were resolved on a 4 to 12% gradient SDS-PAGE gel at 160 V for 25 min. Gels were transferred onto polyvinylidene difluoride membranes by eBlotTM L1 (GenScript Biotech). Membranes were blocked for 40 min with QuickBlock blocking buffer (Beyotime) and incubated with corresponding primary antibodies at 4°C overnight. After incubation, membranes were washed with TBST buffer [50 mM tris-HCl (pH 7.6), 150 mM NaCl, and 0.1% (v/v) Tween 20] for 5 min for five times, and then membranes were incubated with HRP-conjugated secondary antibody for 45 min at room temperature followed by 5-min TBST buffer wash for five times. Last, membranes were treated with an enhanced chemiluminesence substrate (Clarity MaxTM Western, Bio-Rad) and visualized using a ChemiDoc MP Imaging System (Bio-Rad). Antibodies were diluted by antibody dilution buffer (Primary & Secondary Antibody Diluent for WB, YEASEN). Antibody information is listed as follows: rabbit polyclonal anti-H3K9me1 antibody (39888, Active Motif, 1/2000 dilution), mouse monoclonal anti-H3K9me2 antibody (ab1220, Abcam, 1/4000 dilution), rabbit polyclonal anti-H3K9me3 antibody (ab8898, Abcam, 1/5000 dilution), rabbit polyclonal anti-H2A antibody (ab18255, Abcam, 1/1000 dilution), HRP* goat anti-mouse immunoglobulin G (IgG) (H + L) (RS0001, Immunoway, 1/10000 dilution), and HRP* goat anti-rabbit IgG (H + L) (RS0002, Immunoway, 1/10000 dilution).

Western blot for methyltransferase activity

NCP (WT NCP, NCPH3K14Ub; 200 nM), SAM (100 μM, 500 eq.), ZnCl2 (10 μM, 50 eq.) were diluted by SEC buffer and incubated at 30°C. Reactions were commenced by the addition of Clr4 (200 nM, 1 eq.). One microliter was removed from each reaction at 10, 20, 60, 120, 200, and 300 min and then boiled with LDS sample buffer (Thermo Fisher Scientific) at 95°C for 5 min. Boiled samples were resolved on a 4 to 12% gradient SDS-PAGE gel at 160 V for 25 min. Gels were transferred onto polyvinylidene difluoride membranes by eBlotTM L1 (GenScript Biotech). Membranes were blocked for 40 min with QuickBlock blocking buffer (Beyotime) and incubated with corresponding primary antibodies at 4°C overnight. After incubation, membranes were washed with TBST buffer [50 mM tris-HCl (pH 7.6), 150 mM NaCl, and 0.1% (v/v) Tween 20] for 5 min for five times, and then membranes were incubated with HRP-conjugated secondary antibody for 45 min at room temperature followed by 5-min TBST buffer wash for five times. Last, membranes were treated with an enhanced chemiluminesence substrate (Clarity MaxTM Western, Bio-Rad) and visualized using a ChemiDoc MP Imaging System (Bio-Rad). Antibodies were diluted by antibody dilution buffer (Primary & Secondary Antibody Diluent for WB, YEASEN). Antibody information is listed as follows: rabbit polyclonal anti-H3K9me1 antibody (39888, Active Motif, 1/2000 dilution), mouse monoclonal anti-H3K9me2 antibody (ab1220, Abcam, 1/4000 dilution), rabbit polyclonal anti-H3K9me3 antibody (ab8898, Abcam, 1/5000 dilution), rabbit polyclonal anti-H2A antibody (ab18255, Abcam, 1/1000 dilution), HRP* goat anti-mouse immunoglobulin G (IgG) (H + L) (RS0001, Immunoway, 1/10000 dilution), and HRP* goat anti-rabbit IgG (H + L) (RS0002, Immunoway, 1/10000 dilution).
