---
pmid: '34591642'
title: A protein network map of head and neck cancer reveals PIK3CA mutant drug sensitivity.
authors:
- Swaney DL
- Ramms DJ
- Wang Z
- Park J
- Goto Y
- Soucheray M
- Bhola N
- Kim K
- Zheng F
- Zeng Y
- McGregor M
- Herrington KA
- O'Keefe R
- Jin N
- VanLandingham NK
- Foussard H
- Von Dollen J
- Bouhaddou M
- Jimenez-Morales D
- Obernier K
- Kreisberg JF
- Kim M
- Johnson DE
- Jura N
- Grandis JR
- Gutkind JS
- Ideker T
- Krogan NJ
journal: Science
year: '2021'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC9005332
doi: 10.1126/science.abf2911
---

# A protein network map of head and neck cancer reveals PIK3CA mutant drug sensitivity.
**Authors:** Swaney DL, Ramms DJ, Wang Z, Park J, Goto Y, Soucheray M, Bhola N, Kim K, Zheng F, Zeng Y, McGregor M, Herrington KA, O'Keefe R, Jin N, VanLandingham NK, Foussard H, Von Dollen J, Bouhaddou M, Jimenez-Morales D, Obernier K, Kreisberg JF, Kim M, Johnson DE, Jura N, Grandis JR, Gutkind JS, Ideker T, Krogan NJ
**Journal:** Science (2021)
**DOI:** [10.1126/science.abf2911](https://doi.org/10.1126/science.abf2911)
**PMC:** [PMC9005332](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9005332/)

## Abstract

1. Science. 2021 Oct;374(6563):eabf2911. doi: 10.1126/science.abf2911. Epub 2021 
Oct 1.

A protein network map of head and neck cancer reveals PIK3CA mutant drug 
sensitivity.

Swaney DL(1)(2)(3)(4), Ramms DJ(4)(5)(6), Wang Z(4)(6), Park J(4)(7), Goto 
Y(4)(6), Soucheray M(1)(2)(3)(4), Bhola N(4)(8), Kim K(1)(2)(3)(4), Zheng 
F(4)(7), Zeng Y(4)(8), McGregor M(1)(2)(3)(4), Herrington KA(9), O'Keefe 
R(4)(8), Jin N(4)(8), VanLandingham NK(4)(8), Foussard H(1)(2)(3)(4), Von Dollen 
J(1)(2)(3)(4), Bouhaddou M(1)(2)(3)(4), Jimenez-Morales D(1)(2)(3)(4), Obernier 
K(1)(2)(3)(4), Kreisberg JF(4)(7), Kim M(1)(2)(3)(4), Johnson DE(8), Jura 
N(3)(4)(10), Grandis JR(4)(8), Gutkind JS(4)(5)(6), Ideker T(4)(7)(11)(12), 
Krogan NJ(1)(2)(3)(4).

Author information:
(1)Quantitative Biosciences Institute (QBI), University of California San 
Francisco, San Francisco, CA, USA.
(2)J. David Gladstone Institutes, San Francisco, CA, USA.
(3)Department of Cellular and Molecular Pharmacology, University of California 
San Francisco, San Francisco, CA, USA.
(4)The Cancer Cell Map Initiative, San Francisco and La Jolla, CA.
(5)Department of Pharmacology, University of California San Diego, La Jolla, CA.
(6)Moores Cancer Center, University of California San Diego, La Jolla, CA.
(7)Division of Genetics, Department of Medicine, University of California San 
Diego, La Jolla, CA.
(8)Helen Diller Family Comprehensive Cancer Center, University of California San 
Francisco, San Francisco, CA, USA.
(9)Department of Biochemistry and Biophysics Center for Advanced Light 
Microscopy at UCSF, University of California San Francisco, San Francisco, CA, 
USA.
(10)Cardiovascular Research Institute, University of California San Francisco, 
San Francisco, CA, USA.
(11)Department of Bioengineering, University of California San Diego, La Jolla, 
CA, USA.
(12)Department of Computer Science, University of California San Diego, La 
Jolla, CA, USA.

Comment in
    Science. 2021 Oct;374(6563):38-39. doi: 10.1126/science.abl9080.

We outline a framework for elucidating tumor genetic complexity through 
multidimensional protein-protein interaction maps and apply it to enhancing our 
understanding of head and neck squamous cell carcinoma. This network uncovers 
771 interactions from cancer and noncancerous cell states, including WT and 
mutant protein isoforms. Prioritization of cancer-enriched interactions reveals 
a previously unidentified association of the fibroblast growth factor receptor 
tyrosine kinase 3 with Daple, a guanine-nucleotide exchange factor, resulting in 
activation of Gαi- and p21-activated protein kinase 1/2 to promote cancer cell 
migration. Additionally, we observe mutation-enriched interactions between the 
human epidermal growth factor receptor 3 (HER3) receptor tyrosine kinase and 
PIK3CA (the alpha catalytic subunit of phosphatidylinositol 3-kinase) that can 
inform the response to HER3 inhibition in vivo. We anticipate that the 
application of this framework will be valuable for translating genetic 
alterations into a molecular and clinical understanding of the underlying 
biology of many disease areas.

DOI: 10.1126/science.abf2911
PMCID: PMC9005332
PMID: 34591642 [Indexed for MEDLINE]

## Full Text

Abstract

We outline a framework for elucidating tumor genetic complexity through multidimensional protein-protein interaction maps and apply it to enhance the understanding of head-and-neck squamous cell carcinoma. This network uncovers 771 interactions from cancer and non-cancerous cell states including wild-type and mutant protein isoforms. Prioritization of cancer enriched interactions reveals a previously unidentified association of the FGF receptor tyrosine kinase 3 with Daple, a guanine-nucleotide exchange factor, resulting in activation of Gαi and PAK1/2 to promote cancer cell migration. Additionally, we observe mutation enriched interactions between the HER3 receptor tyrosine kinase and PIK3CA that can inform the response to HER3 inhibition in vivo . We anticipate the application of this framework will be valuable to translate genetic alterations into a molecular and clinical understanding of the underlying biology of many disease areas.

Discussion

In order to truly understand the complexities of the cell, it has been hypothesized that one would need a list of the protein machines, or complexes, that carry out all of its functions ( 69 ). Obtaining such a map would be instrumental in not only understanding how a healthy cell functions but how mutations impact on these machines, and the pathways in which they function ( 70 ), in disease states like cancer. Using this premise as a motivation, in this study, we examined the physical landscape of protein-protein interactions targeting genes genetically linked to HNSCC, revealing hundreds of novel PPIs. We observe that these interactions are highly specific to the cell line of study and that mutations in key cancer genes impact on PPI interactions in ways that can provide important mechanistic insight and inform response to targeted treatments. In support of previous observations ( 25 ) and our accompanying manuscript (Kim et al., submitted), these results suggest the exciting premise that there remains a vast network of PPIs left to discover beyond the thousands annotated from HEK293T and HeLa cells ( 22 - 24 ). We anticipate that developments in high-throughput protein complex determination, such as co-elution ( 71 ), proximity-labeling ( 72 , 73 ), and cross-linking MS ( 74 ), will enable the rapid advancement of systematic PPI mapping in a diverse array of cancer cell contexts.

An important goal of cancer therapy is to identify drug targets that are applicable across many patients and that achieve high specificity for cancer cells among a heterogeneous tumor cell population. In the context of PPIs, this goal requires moving beyond simply cataloging protein-protein interactions towards robust comparative analysis of PPIs across cellular contexts. For this purpose, we have created and demonstrated the value of a differential interaction score (DIS) to statistically compare PPIs across contexts, which will aid in not only understanding the underlying biology behind HNSCC, but other cancers and disease in general, as evidenced by our application of this approach for breast cancer (Kim et al., accompanying manuscript). While the DIS revealed a subset of interactions to be cancer enriched in the three cell lines used in our study, future analysis in additional cell lines would be beneficial to further support these observations.

One interaction uncovered by our DIS approach is a connection between the FGFR3 receptor tyrosine kinase and the guanine-nucleotide exchange factor, Daple, which was seen exclusively in the cancer cell lines. Our findings build upon previous work by demonstrating that FGF stimulation can activate Gαi in a Daple- and FGFR3-dependent manner, which results in activation of PAK1/2 kinases and cell motility. PAK1 expression is highly correlated with aggressive tumor behavior and poor patient prognosis in HNSCC ( 61 , 75 ). Our work becomes increasingly important as FGFR inhibitors progress towards the clinic. Phase II clinical trials with rogaratinib, an FGFR inhibitor, are underway for HNSCC patients with FGFR1/2/3 mRNA overexpression ( NCT03088059 ), after phase I trials demonstrated a 67% objective response rate for solid tumors with FGFR mRNA overexpression ( 76 ). Additionally, a complete response was observed in a metastatic HNSCC tumor with multiple FGFR amplifications, including FGFR3, when treated with a pan-FGFR inhibitor ( 77 ). Further work may determine if the FGFR3:Daple interaction results in frequent coupling of FGFR and PAK1/2 activity in HNSCC patients and if other cancer types exploit this signaling mechanism. More direct studies are necessary to determine the extent to which FGFR and PAK1/2 activity contribute to clinical outcomes, as PAK1/2 activity could serve as an additional biomarker of patients benefiting from FGFR targeted therapy.

Our results pertaining to PIK3CA also highlight that oncogenic mechanisms of individual mutations in cancer genes can be reflected in their differences in the corresponding PPIs, and that these differences can be exploited for therapeutic benefit. We postulate that the mechanism for the selectivity we uncovered using PIK3CA mutants lies in the dependence on HER3 signaling that the helical domain mutations maintain. These features of PI3K mutants seemingly contradict previous studies showing that addition of the phosphorylated YxxM motif-containing peptides increases in vitro catalytic activity of the H1047R mutant but not the helical domain mutants ( 78 ). However, we hypothesize that phosphorylated RTK tails are necessary not for activation of the helical domain PI3K mutants, but for their recruitment to the plasma membrane where they need to interact with RasGTP for full activation ( 67 ). This strong dependence renders cells with such mutations sensitive to HER3 inhibition. Most importantly, we also identify a number of other PI3K mutants that share HER3 binding features with the helical domain mutants, and we predict that their oncogenic potential will also be HER3-dependent ( Fig. 4G ). Our data also indicate that upstream PI3K activators with a high density of tyrosine phosphorylation sites represented by the YxxM consensus motifs, such as HER3 and IRS1/2, will be particularly efficient in synergizing with the PI3K helical domain mutants.

Clinical inhibition of HER3 in HNSCC patients is currently being pursued in phase II clinical trials with the monoclonal antibody CDX3379 ( NCT03254927 ) ( 79 ). This drug locks the HER3 extracellular domain in an inactive configuration ( 80 ) and prevents not only dimerization with co-activating RTKs (e.g., HER2) but also activation of HER3 by neuregulins (e.g. NRG1). These properties make HER3 a particularly promising target, as NRG1 is expressed at higher levels in HNSCC than in any other tumor type ( 81 ). The results presented here further suggest that HER3 inhibitors present an opportunity to potently target specific PIK3CA mutant tumors, a utility that had not been evaluated previously. This is important, as PIK3CA is one of the most commonly mutated oncogenes in HNSCC ( 7 ), yet direct targeting of PIK3CA in the clinic has been limited by toxicity ( 82 ), likely due to its pleiotropic roles in cancer and maintenance of normal cell states. In light of our findings, patient pre-selection, such as exclusion of PIK3CA H1047R mutation carriers and inclusion of those harboring helical domain mutants, may be a valuable consideration as future phases of clinical trials proceed.

In summary, this study, along with the accompanying manuscripts (Kim et al., accompanying manuscript and Zheng et al., accompanying manuscript), outlines a framework for elucidating genetic complexity through multidimensional maps of cancer cell biology and demonstrates that such maps can reveal mechanisms of cancer pathogenesis, instructs the selection of therapeutic targets, and informs which point mutations in the tumor are most likely to respond to treatment ( 26 ). As such, we anticipate the generation and incorporation of cancer-specific physical and functional networks may represent a critical component to interpret and predict cancer biology and its clinical outcomes. Finally, the framework described here applies not only to cancer but to many other genetically defined disease areas as well.
