---
pmid: '23904267'
title: Nuclear-localized Asunder regulates cytoplasmic dynein localization via its
  role in the integrator complex.
authors:
- Jodoin JN
- Sitaram P
- Albrecht TR
- May SB
- Shboul M
- Lee E
- Reversade B
- Wagner EJ
- Lee LA
journal: Mol Biol Cell
year: '2013'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC3771956
doi: 10.1091/mbc.E13-05-0254
---

# Nuclear-localized Asunder regulates cytoplasmic dynein localization via its role in the integrator complex.
**Authors:** Jodoin JN, Sitaram P, Albrecht TR, May SB, Shboul M, Lee E, Reversade B, Wagner EJ, Lee LA
**Journal:** Mol Biol Cell (2013)
**DOI:** [10.1091/mbc.E13-05-0254](https://doi.org/10.1091/mbc.E13-05-0254)
**PMC:** [PMC3771956](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3771956/)

## Abstract

1. Mol Biol Cell. 2013 Sep;24(18):2954-65. doi: 10.1091/mbc.E13-05-0254. Epub
2013  Jul 31.

Nuclear-localized Asunder regulates cytoplasmic dynein localization via its role 
in the integrator complex.

Jodoin JN(1), Sitaram P, Albrecht TR, May SB, Shboul M, Lee E, Reversade B, 
Wagner EJ, Lee LA.

Author information:
(1)Department of Cell and Developmental Biology, Vanderbilt University Medical 
Center, Nashville, TN 37232-8240 Department of Biochemistry and Molecular 
Biology, University of Texas Medical School at Houston, Houston, TX 77030 
Institute of Medical Biology, A*STAR, Singapore 138648 Department of Pediatrics, 
National University of Singapore, Singapore 119228.

We previously reported that Asunder (ASUN) is essential for recruitment of 
dynein motors to the nuclear envelope (NE) and nucleus-centrosome coupling at 
the onset of cell division in cultured human cells and Drosophila spermatocytes, 
although the mechanisms underlying this regulation remain unknown. We also 
identified ASUN as a functional component of Integrator (INT), a multisubunit 
complex required for 3'-end processing of small nuclear RNAs. We now provide 
evidence that ASUN acts in the nucleus in concert with other INT components to 
mediate recruitment of dynein to the NE. Knockdown of other individual INT 
subunits in HeLa cells recapitulates the loss of perinuclear dynein in 
ASUN-small interfering RNA cells. Forced localization of ASUN to the cytoplasm 
via mutation of its nuclear localization sequence blocks its capacity to restore 
perinuclear dynein in both cultured human cells lacking ASUN and Drosophila asun 
spermatocytes. In addition, the levels of several INT subunits are reduced at 
G2/M when dynein is recruited to the NE, suggesting that INT does not directly 
mediate this step. Taken together, our data support a model in which a nuclear 
INT complex promotes recruitment of cytoplasmic dynein to the NE, possibly via a 
mechanism involving RNA processing.

DOI: 10.1091/mbc.E13-05-0254
PMCID: PMC3771956
PMID: 23904267 [Indexed for MEDLINE]

## Full Text

INTRODUCTIONDynein, a minus end–directed molecular motor, is a large multimeric complex that can be divided into distinct regions (Holzbaur and Vallee, 1994; Kardon and Vale, 2009). Protruding from the head region are two microtubule-binding domains that allow the motor to walk processively along the microtubule toward its minus end. This movement is driven by the force-generating ATPase activity of the catalytic domains found within the head region of the motor. The stem region, consisting of multiple light, light intermediate, and intermediate chains, is the most variable and is widely considered to serve as the binding site for dynein adaptors.Within the cell, dynein exists in association with its activating complex, dynactin (Schroer, 2004). The dynein–dynactin complex performs diverse functions within the cell, ranging from cargo transport, centrosome assembly, and organelle positioning to roles in chromosome alignment and spindle positioning during mitosis (Holzbaur and Vallee, 1994; Kardon and Vale, 2009). Dynein–dynactin complexes are subject to multiple layers of regulation, including binding of accessory proteins, phosphorylation, subunit composition, and subcellular localization (Kardon and Vale, 2009). Localized pools of dynein were identified and shown to be required for critical processes in the cell, although the mechanisms underlying the control of dynein localization are poorly understood (Kardon and Vale, 2009).Across phyla, a stably anchored subpopulation of dynein exists on the nuclear envelope (NE) of cells (Gonczy et al., 1999; Robinson et al., 1999; Salina et al., 2002; Payne et al., 2003; Anderson et al., 2009). This pool of dynein is required for both stable attachment of centrosomes to the NE before nuclear envelope breakdown (NEBD) and migration of centrosomes to opposite sides of the nucleus, thereby ensuring proper positioning of the bipolar spindle (Vaisberg et al., 1993; Gonczy et al., 1999; Robinson et al., 1999; Malone et al., 2003; Anderson et al., 2009; Splinter et al., 2010; Bolhy et al., 2011; Jodoin et al., 2012; Sitaram et al., 2012). In addition, this pool of dynein appears to facilitate NEBD by forcibly tearing the NE, although the precise mechanism remains unknown (Beaudouin et al., 2002; Salina et al., 2002).Two proteins, Bicaudal D2 (BICD2) and centromere protein F (CENP-F), have been shown to directly anchor dynein to the nuclear surface at the G2/M transition in HeLa cells (Splinter et al., 2010; Bolhy et al., 2011). BICD2, a dynein adaptor protein, directly binds dynein and nucleoporin RanBP2, thereby anchoring the motors to the NE (Splinter et al., 2010). CENP-F directly interacts with dynein adaptor proteins NudE/EL and nucleoporin Nup133 to effectively anchor dynein to the NE (Bolhy et al., 2011). In both Drosophila spermatocytes and cultured human cells, we previously identified ASUN as an additional regulator of dynein recruitment to the NE at G2/M of meiosis and mitosis, respectively, although physical interaction between ASUN and dynein has not been demonstrated (Anderson et al., 2009; Jodoin et al., 2012).Spermatocytes within the testes of Drosophila asun males arrest at prophase of meiosis I with a severely reduced pool of perinuclear dynein and centrosomes that are not attached to the nuclear surface (hence the name “asunder”; Anderson et al., 2009). Spermatocytes that progress beyond this arrest exhibit defects in spindle assembly, chromosome segregation, and cytokinesis during the meiotic divisions. Using cultured human cells, we found that small interfering RNA (siRNA)–mediated down-regulation of the human homologue of Asunder (hASUN) similarly resulted in reduction of perinuclear dynein during prophase of mitosis (Jodoin et al., 2012). Additional defects observed after loss of hASUN included nucleus–centrosome uncoupling, abnormal mitotic spindles, and impaired progression through mitosis.In either Drosophila or cultured human cells, a direct mechanism for promotion of perinuclear dynein by ASUN has not been elucidated, although localization changes in ASUN coincide with the accumulation of dynein on the NE. Drosophila ASUN (dASUN) is largely restricted within the nucleus of early G2 spermatocytes and first appears in the cytoplasm during late G2, roughly coincident with the initiation of dynein recruitment to the nuclear surface (Anderson et al., 2009). Similarly, in prophase HeLa cells, when a perinuclear pool of dynein forms transiently at G2/M, hASUN is diffusely present in the cytoplasm (Jodoin et al., 2012). On the basis of these temporal associations of the localizations of ASUN and perinuclear dynein, we previously proposed that the cytoplasmic pool of ASUN likely mediates recruitment of dynein motors to the NE (Anderson et al., 2009; Jodoin et al., 2012).Integrator complex (INT) is an evolutionarily conserved complex consisting of 14 subunits, although its biology is poorly understood (reviewed in Chen and Wagner, 2010). INT was originally identified due to its association with the C-terminal tail of RNA polymerase II and was subsequently shown to be required for 3′-end processing of small nuclear RNAs (snRNAs; Baillat et al., 2005). These processed snRNAs play roles in gene expression via intron removal and further processing of pre-mRNAs (Matera et al., 2007). To discover novel components of INT that are required for its snRNA-processing function, a cell-based assay was developed in which generation of a green fluorescent protein (GFP) signal due to incomplete processing of a reporter U7 snRNA served as a readout of INT activity (Chen et al., 2012). With this approach, dASUN was identified as a functional component of INT: down-regulation of dASUN led to increase misprocessing of U7 and spliceosomal snRNA, indicating its requirement for activity of the complex (Chen et al., 2012). Furthermore, dASUN was shown to biochemically interact with INT in a stoichiometric manner, an association that it is conserved in humans (Malovannaya et al., 2010; Chen et al., 2012). Collectively these data provide compelling evidence that ASUN is a core Integrator subunit.Given the divergent nature of the known activities of ASUN—critical regulator of cytoplasmic dynein localization and essential component of a nuclear snRNA-processing complex—we sought to determine whether these roles are independent of each other or derived from a common function. We find that depletion of individual INT components from HeLa cells results in loss of perinuclear dynein, recapitulating the phenotype observed in hASUN-siRNA cells (Jodoin et al., 2012). In addition, we find that forced localization of either hASUN or dASUN to the cytoplasm inhibits its capacity to recruit dynein to the NE in the absence of endogenous ASUN. We present a model in which ASUN acts within the nucleus in concert with other subunits of the Integrator complex, likely via processing of a critical RNA target(s), to promote recruitment of cytoplasmic dynein motors to the NE at G2/M.

INTRODUCTION

Dynein, a minus end–directed molecular motor, is a large multimeric complex that can be divided into distinct regions (Holzbaur and Vallee, 1994; Kardon and Vale, 2009). Protruding from the head region are two microtubule-binding domains that allow the motor to walk processively along the microtubule toward its minus end. This movement is driven by the force-generating ATPase activity of the catalytic domains found within the head region of the motor. The stem region, consisting of multiple light, light intermediate, and intermediate chains, is the most variable and is widely considered to serve as the binding site for dynein adaptors.

Within the cell, dynein exists in association with its activating complex, dynactin (Schroer, 2004). The dynein–dynactin complex performs diverse functions within the cell, ranging from cargo transport, centrosome assembly, and organelle positioning to roles in chromosome alignment and spindle positioning during mitosis (Holzbaur and Vallee, 1994; Kardon and Vale, 2009). Dynein–dynactin complexes are subject to multiple layers of regulation, including binding of accessory proteins, phosphorylation, subunit composition, and subcellular localization (Kardon and Vale, 2009). Localized pools of dynein were identified and shown to be required for critical processes in the cell, although the mechanisms underlying the control of dynein localization are poorly understood (Kardon and Vale, 2009).

Across phyla, a stably anchored subpopulation of dynein exists on the nuclear envelope (NE) of cells (Gonczy et al., 1999; Robinson et al., 1999; Salina et al., 2002; Payne et al., 2003; Anderson et al., 2009). This pool of dynein is required for both stable attachment of centrosomes to the NE before nuclear envelope breakdown (NEBD) and migration of centrosomes to opposite sides of the nucleus, thereby ensuring proper positioning of the bipolar spindle (Vaisberg et al., 1993; Gonczy et al., 1999; Robinson et al., 1999; Malone et al., 2003; Anderson et al., 2009; Splinter et al., 2010; Bolhy et al., 2011; Jodoin et al., 2012; Sitaram et al., 2012). In addition, this pool of dynein appears to facilitate NEBD by forcibly tearing the NE, although the precise mechanism remains unknown (Beaudouin et al., 2002; Salina et al., 2002).

Two proteins, Bicaudal D2 (BICD2) and centromere protein F (CENP-F), have been shown to directly anchor dynein to the nuclear surface at the G2/M transition in HeLa cells (Splinter et al., 2010; Bolhy et al., 2011). BICD2, a dynein adaptor protein, directly binds dynein and nucleoporin RanBP2, thereby anchoring the motors to the NE (Splinter et al., 2010). CENP-F directly interacts with dynein adaptor proteins NudE/EL and nucleoporin Nup133 to effectively anchor dynein to the NE (Bolhy et al., 2011). In both Drosophila spermatocytes and cultured human cells, we previously identified ASUN as an additional regulator of dynein recruitment to the NE at G2/M of meiosis and mitosis, respectively, although physical interaction between ASUN and dynein has not been demonstrated (Anderson et al., 2009; Jodoin et al., 2012).

Spermatocytes within the testes of Drosophila asun males arrest at prophase of meiosis I with a severely reduced pool of perinuclear dynein and centrosomes that are not attached to the nuclear surface (hence the name “asunder”; Anderson et al., 2009). Spermatocytes that progress beyond this arrest exhibit defects in spindle assembly, chromosome segregation, and cytokinesis during the meiotic divisions. Using cultured human cells, we found that small interfering RNA (siRNA)–mediated down-regulation of the human homologue of Asunder (hASUN) similarly resulted in reduction of perinuclear dynein during prophase of mitosis (Jodoin et al., 2012). Additional defects observed after loss of hASUN included nucleus–centrosome uncoupling, abnormal mitotic spindles, and impaired progression through mitosis.

In either Drosophila or cultured human cells, a direct mechanism for promotion of perinuclear dynein by ASUN has not been elucidated, although localization changes in ASUN coincide with the accumulation of dynein on the NE. Drosophila ASUN (dASUN) is largely restricted within the nucleus of early G2 spermatocytes and first appears in the cytoplasm during late G2, roughly coincident with the initiation of dynein recruitment to the nuclear surface (Anderson et al., 2009). Similarly, in prophase HeLa cells, when a perinuclear pool of dynein forms transiently at G2/M, hASUN is diffusely present in the cytoplasm (Jodoin et al., 2012). On the basis of these temporal associations of the localizations of ASUN and perinuclear dynein, we previously proposed that the cytoplasmic pool of ASUN likely mediates recruitment of dynein motors to the NE (Anderson et al., 2009; Jodoin et al., 2012).

Integrator complex (INT) is an evolutionarily conserved complex consisting of 14 subunits, although its biology is poorly understood (reviewed in Chen and Wagner, 2010). INT was originally identified due to its association with the C-terminal tail of RNA polymerase II and was subsequently shown to be required for 3′-end processing of small nuclear RNAs (snRNAs; Baillat et al., 2005). These processed snRNAs play roles in gene expression via intron removal and further processing of pre-mRNAs (Matera et al., 2007). To discover novel components of INT that are required for its snRNA-processing function, a cell-based assay was developed in which generation of a green fluorescent protein (GFP) signal due to incomplete processing of a reporter U7 snRNA served as a readout of INT activity (Chen et al., 2012). With this approach, dASUN was identified as a functional component of INT: down-regulation of dASUN led to increase misprocessing of U7 and spliceosomal snRNA, indicating its requirement for activity of the complex (Chen et al., 2012). Furthermore, dASUN was shown to biochemically interact with INT in a stoichiometric manner, an association that it is conserved in humans (Malovannaya et al., 2010; Chen et al., 2012). Collectively these data provide compelling evidence that ASUN is a core Integrator subunit.

Given the divergent nature of the known activities of ASUN—critical regulator of cytoplasmic dynein localization and essential component of a nuclear snRNA-processing complex—we sought to determine whether these roles are independent of each other or derived from a common function. We find that depletion of individual INT components from HeLa cells results in loss of perinuclear dynein, recapitulating the phenotype observed in hASUN-siRNA cells (Jodoin et al., 2012). In addition, we find that forced localization of either hASUN or dASUN to the cytoplasm inhibits its capacity to recruit dynein to the NE in the absence of endogenous ASUN. We present a model in which ASUN acts within the nucleus in concert with other subunits of the Integrator complex, likely via processing of a critical RNA target(s), to promote recruitment of cytoplasmic dynein motors to the NE at G2/M.

RESULTSMultiple INT subunits are required for dynein recruitment to the NETwo ASUN-dependent cellular functions have been reported: dynein recruitment to the NE at G2/M, and proper processing of snRNA by INT (Chen et al., 2012; Jodoin et al., 2012). We hypothesized that other components of the INT complex, such as hASUN, may be required to promote dynein recruitment to the NE. To test this hypothesis, we performed siRNA-mediated knockdown of individual INT subunits in HeLa cells and assessed dynein localization. Before fixation and immunostaining for dynein IC (DIC), siRNA-treated cells were incubated briefly with 5 μM nocodazole to stimulate recruitment of dynein–dynactin complexes to the NE. This treatment has been documented to recapitulate, in non-G1 cells, the enrichment of functional dynein–dynactin complexes on the NE that normally occurs at G2/M, making this an ideal assay for identifying factors involved in dynein localization (Beswick et al., 2006; Hebbar et al., 2008; Splinter et al., 2010; Bolhy et al., 2011; Jodoin et al., 2012).Consistent with our previous report, we found that 78% of nontargeting (NT) control siRNA cells had a striking enrichment of dynein on the NE after brief nocodazole treatment (Figure 1, A and O); in contrast, the percentage of cells with perinuclear dynein was reduced to 20% after hASUN depletion (Figure 1, C and O; Jodoin et al., 2012). In most cases, we found that treatment of cells with siRNA targeting individual INT subunits (IntS, each followed by a unique identifying number) resulted in a similar reduction of cells with perinuclear dynein (Figure 1, D–I, K, M, and N). Of importance, we did not observe any overt effect on cellular health or growth after treatment with INT-targeting siRNAs, arguing against any potential reduction in cellular fitness as the cause of reduced perinuclear dynein. To further quantify the dynein phenotype, we compared the DIC immunofluorescence signals on the NE to that of the cytoplasm and also determined the average peak DIC intensity on the NE for each knockdown as previously described (Supplemental Figure S1; Jodoin et al., 2012). Depletion of IntS1–6, 9, 11, or 12 resulted in a marked decrease in both the ratio of NE to cytoplasmic dynein and the peak intensity of DIC on the NE, comparable to that observed for hASUN. IntS7 and IntS10 were the two exceptions: depletion of either of these INT subunits had no effect on perinuclear dynein accumulation compared with control cells (Figure 1, J, L, and O, and Supplemental Figure S1). We confirmed that all targeted proteins were efficiently depleted by immunoblotting of cell lysates after siRNA treatment or by a second, nonoverlapping, siRNA (Supplemental Figure S2). Taken together, these data show that the majority of the individual INT subunits are required for dynein recruitment to the NE.FIGURE 1:INT subunits are individually required for dynein recruitment to the NE. HeLa cells were transfected with siRNA as indicated. (A–N) After siRNA treatment, cells were incubated in nocodazole, fixed, and stained for DIC and DNA. Loss of perinuclear dynein was observed upon individual knockdown of the majority of INT subunits. Scale bar, 10 μm. (O) Quantification of perinuclear dynein in cells after knockdown of individual INT subunits. *p < 0.0001 (compared with NT control). (P) hASUN immunoblot analysis of cell lysates after knockdown of individual INT subunits. Tubulin was used as loading control.We considered the possibility that loss of dynein accumulation on the NE upon INT depletion could be secondary to cell cycle arrest. We performed fluorescence-activated cell sorting (FACS) analysis of DNA-stained HeLa cells after knockdown of individual INT subunits (Supplemental Figure S3). We observed no differences between the cell cycle profile of hASUN- or other INT subunit-siRNA cells and that of control NT-siRNA cells (Supplemental Figure S3A). We previously reported that hASUN depletion from HeLa cells results in a slightly increased mitotic index (Jodoin et al., 2012); we show here that depletion of other INT subunits has a similar effect (Supplemental Figure S3B). We also found that the percentage of prophase cells, the stage at which dynein normally accumulates on the NE, is slightly increased upon knockdown of ASUN or other INT subunits from HeLa cells (Supplemental Figure S3C). These results indicate that loss of dynein recruitment to the NE in cells depleted of INT is not due to any substantial cell cycle perturbation.Table 1 summarizes our observations of the requirements for INT subunits in dynein localization and the previously reported requirements for INT subunits in snRNA processing (Ezzeddine et al., 2011; Chen et al., 2012). The two data sets compare favorably in that, for both processes, most INT subunits are required, whereas IntS10 is expendable. IntS7, however, was the sole exception, in that it is required for snRNA processing, yet we found no effect of its down-regulation on dynein recruitment to the NE (Chen et al., 2012). Overall these data are consistent with a model in which hASUN regulates dynein localization in an INT complex–dependent manner.TABLE 1:Comparison of INT subunit requirements in snRNA processing versus dynein localization.SubunitsnRNA processingaDynein localizationbhASUN++IntS1++IntS2++IntS3++IntS4++IntS5++IntS6++IntS7+–IntS8+N.D.IntS9++IntS10––IntS11++IntS12+++, Required; –, not required; N.D., not determined.aAnalysis of requirements for INT subunits in U7 snRNA processing was previously reported (Chen et al. 2012).bAnalysis of requirements for INT subunits in dynein recruitment to the NE is presented here (Figure 1).To show that loss of dynein localization is specific to disruption of an INT-mediated RNA processing event and not secondary to a general disruption of RNA processing, we depleted cells of cleavage polyadenylation specificity factor 30 (CPSF30) and assessed perinuclear dynein. CPSF30 is involved in the recruitment of machinery that mediates 3′-mRNA cleavage and poly(A) tail synthesis (Barabino et al., 1997). We found that siRNA-mediated down-regulation of CPSF30 had no effect on perinuclear dynein accumulation, suggesting a specific role for INT in this process (Figure 1, B and O, and Supplemental Figure S1).hASUN levels are normal after depletion of INTGiven the established role of INT in snRNA processing, we considered the possibility that hASUN could be a downstream target (i.e., formation/splicing of mature hASUN transcripts might require a functional INT complex). In this case, lack of perinuclear dynein in cells with INT down-regulation would be secondary to a reduction in hASUN levels. To test this idea, we used previously generated anti-hASUN antibodies to probe immunoblots of lysates of HeLa cells after depletion of individual INT subunits (Jodoin et al., 2012). We found that hASUN protein levels remained largely unchanged in all lysates tested, with the exception of a slight reduction after IntS11 depletion (Figure 1P). We next assessed the reciprocal possibility that hASUN might be required for stability of the INT complex. By immunoblotting, we observed decreased levels of IntS3, 4, 7, 10, and 11 in lysates of hASUN-siRNA HeLa cells relative to control cells (Supplemental Figure S2). This observation is consistent with a recent report showing that levels of Drosophila IntS1 and 12 are interdependent, which may be due to their direct association within the complex (Chen et al., 2013). Taken together, these data indicate that failure of dynein localization in INT-depleted cells is not due to decreased hASUN production and hASUN is required for the stability of several other subunits of the INT complex.INT subunits exhibit a range of subcellular localizationsWe previously showed that Drosophila ASUN exhibits a dynamic localization pattern: in the Drosophila testes, mCherry-tagged dASUN (CHY-dASUN) expressed via a transgene shifts from exclusively nuclear in early G2 to diffusely present throughout the spermatocyte by late G2 (Anderson et al., 2009). Similar localizations (ranging from nuclear to cytoplasmic to diffuse throughout the cell) were observed when GFP-tagged dASUN was expressed in HeLa cells (Anderson et al., 2009). Similarly, we find in the present study that Myc-tagged hASUN expressed in HeLa cells localizes to the nucleus and/or cytoplasm, albeit with a predominantly nuclear pattern in most cells (Figure 2, A1–A3; quantified in Table 2 and later in Figure 4D).FIGURE 2:INT subunits localize to the nucleus, cytoplasm, or both. HeLa cells were transfected with the indicated expression constructs encoding tagged versions of INT subunits. (A–L) After fixation, cells were stained with phalloidin and DAPI; cells expressing Myc-hASUN were also immunostained for Myc. (A) Myc-hASUN was either exclusively nuclear (A1), predominantly cytoplasmic (A2), or distributed throughout the cell (A3). (B) GFP (control) was diffusely present throughout the cell. (C–L) GFP-tagged versions of other INT subunits localized to the nucleus (E, G, I, J, L), cytoplasm (C, H), or both (D, F). Scale bar, 10 μm. (M) Immunoblot analysis of lysates of transfected cells using antibodies against Myc or GFP tags revealed fusion proteins of the predicted sizes. Tubulin was used as loading control.FIGURE 3:Levels of a subset of INT subunits are reduced at G2/M. HeLa cells were transfected with the indicated GFP- or Myc-fusion constructs and either left untreated (asynchronous population; –) or treated with a CDK1 inhibitor (G2/M-arrested population; +) for 16 h. Immunoblot analysis of lysates of transfected cells using antibodies against GFP or Myc tags revealed decreased levels of several INT subunits at G2/M (A), whereas no changes were observed for other subunits (B). hASUN antibodies were used to assess endogenous hASUN levels. (C) No changes were observed in the levels of GFP (control), GFP-BICD2, or endogenous CENP-F at G2/M. Tubulin was used as loading control.FIGURE 4:Identification of a functional NLS in human ASUN. (A) Schematic of Myc-tagged hASUN with predicted NLS (yellow box) in the C-terminal region of the protein. Letter A (red) indicates each charged residue of the endogenous NLS mutated to alanine in Myc-hASUNmutNLS and Myc-NLS-hASUNmutNLS. An exogenous NLS (blue box) was added between the Myc tag and either hASUN or hASUNmutNLS to generate Myc-NLS-hASUN and Myc-NLS-hASUNmutNLS, respectively. (B) Myc immunoblot analysis of lysates of transfected HeLa cells revealed Myc-hASUN fusion proteins of the predicted sizes. Tubulin was used as loading control. (C) Representative images showing predominant localizations of Myc-hASUN fusions in transfected HeLa cells. Scale bars, 10 μm. (D) Quantification of localization patterns of Myc-hASUN fusion proteins in transfected HeLa cells.TABLE 2:Quantification of localizations of tagged INT subunits in transfected HeLa cells.SubunitNuclearCytoplasmicDiffuseMyc-hASUN87 ± 2.24 ± 1.99 ± 2.4GFP-IntS2—100 ± 0—GFP-IntS3——100 ± 0GFP-IntS497 ± 2.1—3 ± 2.1GFP-IntS5——100 ± 0GFP-IntS693 ± 1.4—7 ± 1GFP-IntS7—93 ± 17 ± 1GFP-IntS985 ± 2.5—15 ± 2.5IntS10-GFP81 ± 6.5—19 ± 6GFP-IntS11—4 ± 0.596 ± 0GFP-IntS12100 ± 0——HeLa cells were transfected with the indicated expression constructs encoding GFP- or Myc-tagged versions of human INT subunits, fixed, and stained with phalloidin and DAPI. Cells expressing Myc-hASUN were also immunostained for Myc. Immunofluorescence microscopy was performed to assess localization patterns of INT fusion proteins (representative images presented in Figure 2, with quantification shown here). Localizations were scored as the percentage of cells with nuclear, cytoplasmic, or diffuse localization.In both fly testes and HeLa cells, we previously observed a temporal correlation in which a pool of cytoplasmic ASUN is present at the onset of dynein accumulation on the NE at G2/M (Anderson et al., 2009; Jodoin et al., 2012). In fly spermatocytes, dASUN undergoes a shift from nuclear to first appearing in the cytoplasm during late G2, roughly coinciding with dynein recruitment to the NE (Anderson et al., 2009). In HeLa cells, hASUN localizes to the cytoplasm at prophase, a cell cycle stage at which dynein is enriched on the NE (Jodoin et al., 2012). Owing to this correlation, we previously hypothesized that a cytoplasmic pool of ASUN is required for recruitment of dynein motors to the NE at G2/M (Jodoin et al., 2012).We considered the possibility that the INT complex might function within the cytoplasm to promote dynein recruitment to the NE in a manner independent of its role in snRNA processing within the nucleus. A broad survey of subcellular localization patterns of INT subunits has not been previously reported. To address whether the INT complex exists in the cytoplasm, we expressed GFP fusions of the majority of individual INT subunits in HeLa cells to visualize their localizations. We confirmed by immunoblotting for GFP that fusion proteins of the predicted sizes were stably expressed in transfected cells (Figure 2M).We divided the INT subunits into three categories based on their localization: predominantly nuclear (hASUN; IntS4, 6, 9, 10, and 12), predominantly cytoplasmic (IntS2 and 7), or evenly distributed between the nucleus and cytoplasm (IntS3, 5, and 11; Figure 2, C–L; quantified in Table 2). Given these complex patterns, we could not exclude the possibility that the INT complex (or perhaps INT subcomplexes) could have potential roles in the cytoplasm in addition to its established role in snRNA processing in the nucleus. It is interesting to note, however, that knockdown of any of several INT subunits that localized predominantly to the nucleus (hASUN; IntS4, 6, 9, or 12; excluding the nonessential IntS10) resulted in loss of perinuclear dynein in HeLa cells; these observations suggest that a nuclear pool of INT might be required to promote recruitment of cytoplasmic dynein to the NE (Figures 1 and 2 and Table 2).Reduced levels of several INT subunits at G2/MWe further considered the possibility that a cytoplasmic pool of the INT complex might be required for dynein localization. In this case, the simplest model would be that INT acts in the cytoplasm at G2/M to mediate dynein recruitment to the NE. As a test of this model, we asked whether INT subunits are present at G2/M. HeLa cells expressing tagged INT subunits were either left untreated (asynchronous population) or treated with an inhibitor of cyclin-dependent kinase 1 (CDK1; to obtain a G2/M-arrested population), followed by immunoblotting of cell lysates to assess fusion protein levels (Vassilev, 2006). Treated cells acquired a rounded morphology consistent with a G2/M arrest, confirming efficacy of CDK1 inhibition. We found that the levels of 60% (6 of 10) of the INT subunits tested were markedly decreased in G2/M-arrested cells compared with asynchronously dividing cells (Figure 3A). In contrast, levels of 4 INT subunits, including hASUN (both endogenous and tagged), were unchanged at G2/M (Figure 3B). Other known regulators of dynein recruitment, BICD2 (tagged) and CENP-F (endogenous), as well as a GFP control, showed no change in levels at G2/M (Figure 3C). Of note, four subunits required for dynein localization (IntS2, IntS5, IntS6, and IntS9) were present at relatively low levels at G2/M (Figures 1 and 3A). These data led us to conclude that INT is unlikely to directly mediate dynein recruitment to the NE at this stage. Instead, we propose that INT functions earlier in the cell cycle, possibly at the level of RNA processing during interphase, to subsequently affect dynein localization at G2/M.Mammalian ASUN homologues contain a functional nuclear localization sequenceTo determine whether dynein recruitment to the NE is promoted by cytoplasmic ASUN, as we originally hypothesized, or by nuclear ASUN, as suggested by data presented here, we needed a method to direct the localization of ASUN within cells to either of these two compartments (Jodoin et al., 2012). To identify critical regions of ASUN required for its nuclear localization, we began by performing a structure–function analysis of the mouse ASUN homologue (mASUN). Full-length Myc-tagged mASUN expressed in transfected HeLa cells was concentrated in the nucleus, with some cells showing diffuse or cytoplasmic localization (Supplemental Figure S4, A and G). We divided full-length mASUN into three overlapping fragments (F1–F3) and generated constructs for expression of each fragment fused to GFP at its N-terminal end (Supplemental Figure S4A). On expression in transfected HeLa cells, we found that the three GFP-tagged mASUN fragments had distinct localization patterns. GFP-mASUN-F1 (N-terminal fragment) was present throughout the cell, with slight enrichment in the nucleus (Supplemental Figure S4, C and G), whereas GFP-mASUN-F2 (middle fragment) was predominantly cytoplasmic, with slight perinuclear enrichment (Supplemental Figure S4, D and G). GFP-mASUN-F3 (C-terminal fragment) appeared to be exclusively nuclear, suggesting that it likely contains critical sequences that mediate nuclear localization of full-length mASUN (Supplemental Figure S4, E and G). For all mASUN constructs used here, fusion proteins of the predicted sizes were observed by immunoblotting transfected HeLa cell lysates (Supplemental Figure S4B).We used nuclear localization sequence (NLS) prediction software to identify a putative NLS in the C-terminal region of mASUN (Supplemental Figure S4A). We generated Myc-mASUNmutNLS by performing site-directed mutagenesis to alter several charged residues to alanines within this region (Supplemental Figure S4A). When expressed in transfected HeLa cells, Myc-mASUN was predominantly nuclear, whereas Myc-mASUNmutNLS was predominantly cytoplasmic, confirming functionality of the candidate NLS (Supplemental Figure S4, F and G).We also used NLS prediction software to identify a putative NLS in the C-terminal region of the human ASUN homologue that was 100% identical to the verified NLS of mASUN (Figure 4A and Supplemental Figure S4A). We performed site-directed mutagenesis to change several charged residues in this sequence to alanines, thereby generating Myc-hASUNmutNLS (Figure 4A). When expressed in HeLa cells, Myc-hASUN was predominantly nuclear, with some cytoplasmic or diffuse localization (Figure 2, A1–A3; quantified in Table 2 and Figure 4D). In contrast, Myc-hASUNmutNLS was predominantly cytoplasmic, confirming that we had identified a functional NLS (Figure 4, C and D). For experiments presented later in Figure 6, we also generated constructs for addition of a strong exogenous NLS (PKKKRKV; derived from SV40 large T antigen) to both hASUN and hASUNmutNLS (C-terminal to the Myc tag) to produce Myc-NLS-hASUN and Myc-NLS-hASUNmutNLS, respectively (Figure 4A; Kalderon et al., 1984). As expected, both fusion proteins were predominantly nuclear when expressed in HeLa cells (Figure 4, C and D). We performed immunoblotting of lysates of transfected cells to confirm that all Myc-hASUN constructs used here produced fusion proteins of the predicted sizes (Figure 4B).FIGURE 5:Functional NLS is conserved in Drosophila ASUN homologue. (A) Schematic of CHY-tagged dASUN with predicted NLS (yellow box) in the C-terminal region of the protein. Letter A indicates each charged residue of the endogenous NLS mutated to alanine in CHY-dASUNmutNLS. An exogenous NLS (blue box) was added to the N-terminal end of CHY-tagged dASUN to generate NLS-CHY-dASUN. (B) CHY immunoblot analysis of testes lysates from asun males with or without germline expression of CHY-dASUN fusions revealed proteins of the predicted sizes. An intervening lane just left of the last lane was removed. Wild-type (WT) males and asun males lacking a transgene were used as negative controls. Tubulin was used as loading control. (C) Representative images showing localizations of CHY-dASUN fusions in transgenic G2 spermatocytes. Scale bar, 10 μm. (D) Quantification of localization patterns of CHY-dASUN fusion proteins in transgenic G2 spermatocytes.FIGURE 6:Nuclear pool of hASUN is required for dynein recruitment to the NE in HeLa cells. HeLa cells were transfected with NT or hASUN siRNA plus Myc (vector control) or Myc-hASUN expression constructs as indicated. (A) After nocodazole treatment, cells were fixed and stained for DIC and DNA. Representative images of cells. Perinuclear dynein was restored only by expression of hASUN fusion proteins with nuclear localization. Scale bar, 10 μm. (B) Quantification of cells with perinuclear dynein after the indicated siRNA and DNA transfections. *p < 0.0001 (pairwise comparisons indicated). N.S., nonsignificant statistical differences. (C) hASUN immunoblot of lysates of transfected HeLa cells confirmed depletion of hASUN in hASUN-siRNA cells, and Myc immunoblot confirmed expression of Myc-hASUN fusion proteins of the predicted sizes. Tubulin was used as loading control.Drosophila ASUN contains a functional NLSWe also identified a putative NLS in the C-terminal region of dASUN by using NLS prediction software (Supplemental Figure S5A). We previously showed that GFP-tagged dASUN expressed in transfected HeLa cells localizes in the nucleus, cytoplasm, and between these two compartments, and we obtained similar results here using Myc-tagged dASUN (Supplemental Figure S5, A and B; Anderson et al., 2009). By site-directed mutagenesis, we changed several charged residues to alanines within the candidate NLS of Myc-dASUN to generate Myc-dASUNmutNLS (Supplemental Figure S5A). Introduction of these mutations resulted in a significant shift of the fusion protein from the nucleus to the cytoplasm of transfected HeLa cells, verifying functionality of the putative NLS in this system (Supplemental Figure S5B). For experiments presented later in this study, we also generated constructs encoding Myc-NLS-dASUN and Myc-NLS-dASUNmutNLS (each containing a strong exogenous NLS placed C-terminal to the Myc tag) and confirmed that both of these fusion proteins were predominantly nuclear when expressed in HeLa cells (Supplemental Figure S5, A and B).We next tested whether the NLS that we identified in dASUN using transfected HeLa cells was also functional in vivo. We previously established transgenic Drosophila lines with testes-specific expression of CHY-tagged dASUN and introduced a copy of this transgene into the asun background (Figure 5A; Anderson et al., 2009). In G2 spermatocytes, CHY-dASUN was predominantly nuclear, with some diffuse localization (Figure 5, C and D). We used the same approach to express CHY-tagged dASUNmutNLS (carrying mutations in the predicted NLS as described earlier) in asun testes (Figure 5A). CHY-dASUNmutNLS was tightly localized to the cytoplasm of G2 spermatocytes, a pattern not observed for wild-type CHY-dASUN; thus, the NLS sequence we identified in dASUN using cultured human cells was also functional in vivo (Figure 5, C and D). For experiments presented later in Figure 7, to enrich for dASUN in the nucleus, we used the same approach to express CHY-dASUN with a strong exogenous NLS (NLS-CHY-dASUN) in asun testes (Figure 5A). The NLS-CHY-dASUN fusion was even more tightly localized to the nucleus of G2 spermatocytes than CHY-dASUN (Figure 5, C and D). For all CHY-dASUN constructs described here, fusion proteins of the predicted sizes were observed by immunoblotting of transgenic testes lysates (Figure 5B).FIGURE 7:Nuclear pool of dASUN is required for dynein recruitment to the NE in Drosophila spermatocytes. (A, B) Testes dissected from wild-type (WT) or asun males with or without germline expression of CHY-tagged dASUN fusion proteins were stained for dynein heavy chain and DNA. (A) Representative images of G2 spermatocytes. Perinuclear dynein was restored to asun G2 spermatocytes only by expression of dASUN fusion proteins with nuclear localization. Scale bar, 10 μm. (B) Quantification of G2 spermatocytes with perinuclear dynein. (C) Fertility assay shows the average number of progeny per fertile male. *p < 0.0001 (compared with wild-type control).A nuclear pool of ASUN is required in HeLa cells for dynein localizationWe used our hASUN expression constructs to address whether hASUN functions in the cytoplasm or nucleus of HeLa cells to promote dynein recruitment to the NE. We tested the capacity of cytoplasmic Myc-hASUNmutNLS to restore perinuclear dynein to cells depleted of endogenous hASUN. Cells treated with NT or hASUN siRNA and transfected with Myc, Myc-hASUN, or Myc-hASUNmutNLS constructs (refractory to hASUN siRNA) were analyzed using our dynein localization assay (Figure 6, A and B). As expected, we found that Myc-hASUN rescued the loss of perinuclear dynein caused by endogenous hASUN depletion (Figure 6A; quantified in Figure 6B). In contrast, dynein remained diffuse in the cytoplasm upon expression of Myc-hASUNmutNLS in cells lacking endogenous hASUN, suggesting that hASUN may function in the nucleus to regulate dynein localization (Figure 6, A and B).To further confirm that a nuclear pool of hASUN is required for dynein localization and rule out the trivial possibility that mutation of the NLS of hASUN might interfere with its activity in other ways (e.g., improper protein folding), we asked whether forced localization of Myc-hASUNmutNLS to the nucleus via addition of an exogenous NLS would render it competent to restore perinuclear dynein in cells depleted of endogenous hASUN. Indeed, we found that dynein localized normally upon expression of Myc-NLS-hASUN or Myc-NLS-hASUNmutNLS in hASUN-siRNA cells (Figure 6, A and B). These findings suggest that the mutations we introduced into the endogenous NLS of hASUN were not deleterious per se to the protein, but rather that nuclear residence is required for hASUN to exert its effect on dynein localization. Immunoblotting of cell lysates for endogenous hASUN and the Myc tag confirmed efficient depletion of hASUN by siRNA treatment and expression of all Myc-tagged hASUN fusions used here (Figure 6C).We next used our dASUN constructs to test whether dASUN acts within the cytoplasm or nucleus of transfected HeLa cells to promote dynein localization. We previously reported that loss of perinuclear dynein in hASUN-depleted HeLa cells was rescued by expression of Cherry-dASUN, and similar results were obtained here by expressing Myc-dASUN in hASUN-siRNA cells (Supplemental Figure S5C; Anderson et al., 2009). Consistent with our hASUN results, we found that expression of cytoplasmic-localized Myc-dASUNmutNLS in hASUN-siRNA cells failed to restore perinuclear dynein, whereas nuclear-localized Myc-NLS-dASUN or Myc-NLS-dASUNmutNLS were both competent to promote dynein localization (Supplemental Figure S5C). Immunoblotting of cell lysates for endogenous hASUN and the Myc tag confirmed efficient depletion of hASUN by siRNA treatment and expression of all Myc-tagged dASUN fusions used here (Supplemental Figure S5D). Taken together, these data provide further support for a model in which a nuclear pool of ASUN is required for recruitment of cytoplasmic dynein to the NE in cultured human cells and suggest that this mechanism might be conserved in Drosophila.A nuclear pool of dASUN is required in Drosophila spermatocytes for dynein localizationWe previously identified ASUN as a critical regulator of dynein localization during Drosophila spermatogenesis (Anderson et al., 2009). We used this system to determine whether the requirement for nuclear ASUN in regulating cytoplasmic dynein localization that we observed in HeLa cells is conserved in vivo. Using transgenic Drosophila lines that we established (Figure 5A), we assessed the capacity of cytoplasmic CHY-dASUNmutNLS or nuclear NLS-CHY-dASUN expressed in asun testes to promote dynein localization. As previously reported, we found that CHY-dASUN rescued loss of perinuclear dynein in asun spermatocytes (Figure 7, A and B; Anderson et al., 2009). asun spermatocytes expressing CHY-dASUNmutNLS, however, lacked perinuclear dynein; instead, dynein remained diffuse in the cytoplasm, similar to the phenotype of asun spermatocytes lacking a transgenic rescue construct (Figure 7, A and B). Expression of NLS-CHY-dASUN in asun spermatocytes restored perinuclear dynein to nearly wild-type levels, suggesting that a nuclear pool of dASUN is responsible for regulating the localization of cytoplasmic dynein motors in this system.In addition to the failure of dynein localization, asun mutants display a range of defects: spermatocytes arrested at prophase I with unattached centrosomes, impaired sperm bundling, and male sterility (Anderson et al., 2009). This constellation of asun phenotypes is likely secondary to loss of perinuclear dynein in G2 spermatocytes, an event that occurred upstream of these defects. To determine whether restoration of dynein on the NE of asun spermatocytes by NLS-CHY-dASUN expression was sufficient to rescue the most downstream defect, male sterility, we scored the number of live progeny per fertile male. We found that NLS-CHY-dASUN could restore fertility to asun males to the same degree as the CHY-dASUN control, whereas only partial rescue was achieved with CHY-dASUNmutNLS (Figure 7C).Taken together, these data suggest that a nuclear pool of ASUN plays a conserved role, from Drosophila to humans, in the recruitment of cytoplasmic dynein motors to the NE at G2/M. Combined with evidence presented here that additional INT subunits likewise play an essential role in this recruitment step, we hypothesize that the Integrator complex mediates the processing of a critical RNA target(s) required for the production of a cytoplasmic protein that is directly involved in the regulation of dynein localization.

Multiple INT subunits are required for dynein recruitment to the NETwo ASUN-dependent cellular functions have been reported: dynein recruitment to the NE at G2/M, and proper processing of snRNA by INT (Chen et al., 2012; Jodoin et al., 2012). We hypothesized that other components of the INT complex, such as hASUN, may be required to promote dynein recruitment to the NE. To test this hypothesis, we performed siRNA-mediated knockdown of individual INT subunits in HeLa cells and assessed dynein localization. Before fixation and immunostaining for dynein IC (DIC), siRNA-treated cells were incubated briefly with 5 μM nocodazole to stimulate recruitment of dynein–dynactin complexes to the NE. This treatment has been documented to recapitulate, in non-G1 cells, the enrichment of functional dynein–dynactin complexes on the NE that normally occurs at G2/M, making this an ideal assay for identifying factors involved in dynein localization (Beswick et al., 2006; Hebbar et al., 2008; Splinter et al., 2010; Bolhy et al., 2011; Jodoin et al., 2012).Consistent with our previous report, we found that 78% of nontargeting (NT) control siRNA cells had a striking enrichment of dynein on the NE after brief nocodazole treatment (Figure 1, A and O); in contrast, the percentage of cells with perinuclear dynein was reduced to 20% after hASUN depletion (Figure 1, C and O; Jodoin et al., 2012). In most cases, we found that treatment of cells with siRNA targeting individual INT subunits (IntS, each followed by a unique identifying number) resulted in a similar reduction of cells with perinuclear dynein (Figure 1, D–I, K, M, and N). Of importance, we did not observe any overt effect on cellular health or growth after treatment with INT-targeting siRNAs, arguing against any potential reduction in cellular fitness as the cause of reduced perinuclear dynein. To further quantify the dynein phenotype, we compared the DIC immunofluorescence signals on the NE to that of the cytoplasm and also determined the average peak DIC intensity on the NE for each knockdown as previously described (Supplemental Figure S1; Jodoin et al., 2012). Depletion of IntS1–6, 9, 11, or 12 resulted in a marked decrease in both the ratio of NE to cytoplasmic dynein and the peak intensity of DIC on the NE, comparable to that observed for hASUN. IntS7 and IntS10 were the two exceptions: depletion of either of these INT subunits had no effect on perinuclear dynein accumulation compared with control cells (Figure 1, J, L, and O, and Supplemental Figure S1). We confirmed that all targeted proteins were efficiently depleted by immunoblotting of cell lysates after siRNA treatment or by a second, nonoverlapping, siRNA (Supplemental Figure S2). Taken together, these data show that the majority of the individual INT subunits are required for dynein recruitment to the NE.FIGURE 1:INT subunits are individually required for dynein recruitment to the NE. HeLa cells were transfected with siRNA as indicated. (A–N) After siRNA treatment, cells were incubated in nocodazole, fixed, and stained for DIC and DNA. Loss of perinuclear dynein was observed upon individual knockdown of the majority of INT subunits. Scale bar, 10 μm. (O) Quantification of perinuclear dynein in cells after knockdown of individual INT subunits. *p < 0.0001 (compared with NT control). (P) hASUN immunoblot analysis of cell lysates after knockdown of individual INT subunits. Tubulin was used as loading control.We considered the possibility that loss of dynein accumulation on the NE upon INT depletion could be secondary to cell cycle arrest. We performed fluorescence-activated cell sorting (FACS) analysis of DNA-stained HeLa cells after knockdown of individual INT subunits (Supplemental Figure S3). We observed no differences between the cell cycle profile of hASUN- or other INT subunit-siRNA cells and that of control NT-siRNA cells (Supplemental Figure S3A). We previously reported that hASUN depletion from HeLa cells results in a slightly increased mitotic index (Jodoin et al., 2012); we show here that depletion of other INT subunits has a similar effect (Supplemental Figure S3B). We also found that the percentage of prophase cells, the stage at which dynein normally accumulates on the NE, is slightly increased upon knockdown of ASUN or other INT subunits from HeLa cells (Supplemental Figure S3C). These results indicate that loss of dynein recruitment to the NE in cells depleted of INT is not due to any substantial cell cycle perturbation.Table 1 summarizes our observations of the requirements for INT subunits in dynein localization and the previously reported requirements for INT subunits in snRNA processing (Ezzeddine et al., 2011; Chen et al., 2012). The two data sets compare favorably in that, for both processes, most INT subunits are required, whereas IntS10 is expendable. IntS7, however, was the sole exception, in that it is required for snRNA processing, yet we found no effect of its down-regulation on dynein recruitment to the NE (Chen et al., 2012). Overall these data are consistent with a model in which hASUN regulates dynein localization in an INT complex–dependent manner.TABLE 1:Comparison of INT subunit requirements in snRNA processing versus dynein localization.SubunitsnRNA processingaDynein localizationbhASUN++IntS1++IntS2++IntS3++IntS4++IntS5++IntS6++IntS7+–IntS8+N.D.IntS9++IntS10––IntS11++IntS12+++, Required; –, not required; N.D., not determined.aAnalysis of requirements for INT subunits in U7 snRNA processing was previously reported (Chen et al. 2012).bAnalysis of requirements for INT subunits in dynein recruitment to the NE is presented here (Figure 1).To show that loss of dynein localization is specific to disruption of an INT-mediated RNA processing event and not secondary to a general disruption of RNA processing, we depleted cells of cleavage polyadenylation specificity factor 30 (CPSF30) and assessed perinuclear dynein. CPSF30 is involved in the recruitment of machinery that mediates 3′-mRNA cleavage and poly(A) tail synthesis (Barabino et al., 1997). We found that siRNA-mediated down-regulation of CPSF30 had no effect on perinuclear dynein accumulation, suggesting a specific role for INT in this process (Figure 1, B and O, and Supplemental Figure S1).

Multiple INT subunits are required for dynein recruitment to the NE

Two ASUN-dependent cellular functions have been reported: dynein recruitment to the NE at G2/M, and proper processing of snRNA by INT (Chen et al., 2012; Jodoin et al., 2012). We hypothesized that other components of the INT complex, such as hASUN, may be required to promote dynein recruitment to the NE. To test this hypothesis, we performed siRNA-mediated knockdown of individual INT subunits in HeLa cells and assessed dynein localization. Before fixation and immunostaining for dynein IC (DIC), siRNA-treated cells were incubated briefly with 5 μM nocodazole to stimulate recruitment of dynein–dynactin complexes to the NE. This treatment has been documented to recapitulate, in non-G1 cells, the enrichment of functional dynein–dynactin complexes on the NE that normally occurs at G2/M, making this an ideal assay for identifying factors involved in dynein localization (Beswick et al., 2006; Hebbar et al., 2008; Splinter et al., 2010; Bolhy et al., 2011; Jodoin et al., 2012).

Consistent with our previous report, we found that 78% of nontargeting (NT) control siRNA cells had a striking enrichment of dynein on the NE after brief nocodazole treatment (Figure 1, A and O); in contrast, the percentage of cells with perinuclear dynein was reduced to 20% after hASUN depletion (Figure 1, C and O; Jodoin et al., 2012). In most cases, we found that treatment of cells with siRNA targeting individual INT subunits (IntS, each followed by a unique identifying number) resulted in a similar reduction of cells with perinuclear dynein (Figure 1, D–I, K, M, and N). Of importance, we did not observe any overt effect on cellular health or growth after treatment with INT-targeting siRNAs, arguing against any potential reduction in cellular fitness as the cause of reduced perinuclear dynein. To further quantify the dynein phenotype, we compared the DIC immunofluorescence signals on the NE to that of the cytoplasm and also determined the average peak DIC intensity on the NE for each knockdown as previously described (Supplemental Figure S1; Jodoin et al., 2012). Depletion of IntS1–6, 9, 11, or 12 resulted in a marked decrease in both the ratio of NE to cytoplasmic dynein and the peak intensity of DIC on the NE, comparable to that observed for hASUN. IntS7 and IntS10 were the two exceptions: depletion of either of these INT subunits had no effect on perinuclear dynein accumulation compared with control cells (Figure 1, J, L, and O, and Supplemental Figure S1). We confirmed that all targeted proteins were efficiently depleted by immunoblotting of cell lysates after siRNA treatment or by a second, nonoverlapping, siRNA (Supplemental Figure S2). Taken together, these data show that the majority of the individual INT subunits are required for dynein recruitment to the NE.

INT subunits are individually required for dynein recruitment to the NE. HeLa cells were transfected with siRNA as indicated. (A–N) After siRNA treatment, cells were incubated in nocodazole, fixed, and stained for DIC and DNA. Loss of perinuclear dynein was observed upon individual knockdown of the majority of INT subunits. Scale bar, 10 μm. (O) Quantification of perinuclear dynein in cells after knockdown of individual INT subunits. *p < 0.0001 (compared with NT control). (P) hASUN immunoblot analysis of cell lysates after knockdown of individual INT subunits. Tubulin was used as loading control.

We considered the possibility that loss of dynein accumulation on the NE upon INT depletion could be secondary to cell cycle arrest. We performed fluorescence-activated cell sorting (FACS) analysis of DNA-stained HeLa cells after knockdown of individual INT subunits (Supplemental Figure S3). We observed no differences between the cell cycle profile of hASUN- or other INT subunit-siRNA cells and that of control NT-siRNA cells (Supplemental Figure S3A). We previously reported that hASUN depletion from HeLa cells results in a slightly increased mitotic index (Jodoin et al., 2012); we show here that depletion of other INT subunits has a similar effect (Supplemental Figure S3B). We also found that the percentage of prophase cells, the stage at which dynein normally accumulates on the NE, is slightly increased upon knockdown of ASUN or other INT subunits from HeLa cells (Supplemental Figure S3C). These results indicate that loss of dynein recruitment to the NE in cells depleted of INT is not due to any substantial cell cycle perturbation.

Table 1 summarizes our observations of the requirements for INT subunits in dynein localization and the previously reported requirements for INT subunits in snRNA processing (Ezzeddine et al., 2011; Chen et al., 2012). The two data sets compare favorably in that, for both processes, most INT subunits are required, whereas IntS10 is expendable. IntS7, however, was the sole exception, in that it is required for snRNA processing, yet we found no effect of its down-regulation on dynein recruitment to the NE (Chen et al., 2012). Overall these data are consistent with a model in which hASUN regulates dynein localization in an INT complex–dependent manner.

Comparison of INT subunit requirements in snRNA processing versus dynein localization.

+, Required; –, not required; N.D., not determined.

aAnalysis of requirements for INT subunits in U7 snRNA processing was previously reported (Chen et al. 2012).

bAnalysis of requirements for INT subunits in dynein recruitment to the NE is presented here (Figure 1).

To show that loss of dynein localization is specific to disruption of an INT-mediated RNA processing event and not secondary to a general disruption of RNA processing, we depleted cells of cleavage polyadenylation specificity factor 30 (CPSF30) and assessed perinuclear dynein. CPSF30 is involved in the recruitment of machinery that mediates 3′-mRNA cleavage and poly(A) tail synthesis (Barabino et al., 1997). We found that siRNA-mediated down-regulation of CPSF30 had no effect on perinuclear dynein accumulation, suggesting a specific role for INT in this process (Figure 1, B and O, and Supplemental Figure S1).

hASUN levels are normal after depletion of INTGiven the established role of INT in snRNA processing, we considered the possibility that hASUN could be a downstream target (i.e., formation/splicing of mature hASUN transcripts might require a functional INT complex). In this case, lack of perinuclear dynein in cells with INT down-regulation would be secondary to a reduction in hASUN levels. To test this idea, we used previously generated anti-hASUN antibodies to probe immunoblots of lysates of HeLa cells after depletion of individual INT subunits (Jodoin et al., 2012). We found that hASUN protein levels remained largely unchanged in all lysates tested, with the exception of a slight reduction after IntS11 depletion (Figure 1P). We next assessed the reciprocal possibility that hASUN might be required for stability of the INT complex. By immunoblotting, we observed decreased levels of IntS3, 4, 7, 10, and 11 in lysates of hASUN-siRNA HeLa cells relative to control cells (Supplemental Figure S2). This observation is consistent with a recent report showing that levels of Drosophila IntS1 and 12 are interdependent, which may be due to their direct association within the complex (Chen et al., 2013). Taken together, these data indicate that failure of dynein localization in INT-depleted cells is not due to decreased hASUN production and hASUN is required for the stability of several other subunits of the INT complex.

hASUN levels are normal after depletion of INT

Given the established role of INT in snRNA processing, we considered the possibility that hASUN could be a downstream target (i.e., formation/splicing of mature hASUN transcripts might require a functional INT complex). In this case, lack of perinuclear dynein in cells with INT down-regulation would be secondary to a reduction in hASUN levels. To test this idea, we used previously generated anti-hASUN antibodies to probe immunoblots of lysates of HeLa cells after depletion of individual INT subunits (Jodoin et al., 2012). We found that hASUN protein levels remained largely unchanged in all lysates tested, with the exception of a slight reduction after IntS11 depletion (Figure 1P). We next assessed the reciprocal possibility that hASUN might be required for stability of the INT complex. By immunoblotting, we observed decreased levels of IntS3, 4, 7, 10, and 11 in lysates of hASUN-siRNA HeLa cells relative to control cells (Supplemental Figure S2). This observation is consistent with a recent report showing that levels of Drosophila IntS1 and 12 are interdependent, which may be due to their direct association within the complex (Chen et al., 2013). Taken together, these data indicate that failure of dynein localization in INT-depleted cells is not due to decreased hASUN production and hASUN is required for the stability of several other subunits of the INT complex.

INT subunits exhibit a range of subcellular localizationsWe previously showed that Drosophila ASUN exhibits a dynamic localization pattern: in the Drosophila testes, mCherry-tagged dASUN (CHY-dASUN) expressed via a transgene shifts from exclusively nuclear in early G2 to diffusely present throughout the spermatocyte by late G2 (Anderson et al., 2009). Similar localizations (ranging from nuclear to cytoplasmic to diffuse throughout the cell) were observed when GFP-tagged dASUN was expressed in HeLa cells (Anderson et al., 2009). Similarly, we find in the present study that Myc-tagged hASUN expressed in HeLa cells localizes to the nucleus and/or cytoplasm, albeit with a predominantly nuclear pattern in most cells (Figure 2, A1–A3; quantified in Table 2 and later in Figure 4D).FIGURE 2:INT subunits localize to the nucleus, cytoplasm, or both. HeLa cells were transfected with the indicated expression constructs encoding tagged versions of INT subunits. (A–L) After fixation, cells were stained with phalloidin and DAPI; cells expressing Myc-hASUN were also immunostained for Myc. (A) Myc-hASUN was either exclusively nuclear (A1), predominantly cytoplasmic (A2), or distributed throughout the cell (A3). (B) GFP (control) was diffusely present throughout the cell. (C–L) GFP-tagged versions of other INT subunits localized to the nucleus (E, G, I, J, L), cytoplasm (C, H), or both (D, F). Scale bar, 10 μm. (M) Immunoblot analysis of lysates of transfected cells using antibodies against Myc or GFP tags revealed fusion proteins of the predicted sizes. Tubulin was used as loading control.FIGURE 3:Levels of a subset of INT subunits are reduced at G2/M. HeLa cells were transfected with the indicated GFP- or Myc-fusion constructs and either left untreated (asynchronous population; –) or treated with a CDK1 inhibitor (G2/M-arrested population; +) for 16 h. Immunoblot analysis of lysates of transfected cells using antibodies against GFP or Myc tags revealed decreased levels of several INT subunits at G2/M (A), whereas no changes were observed for other subunits (B). hASUN antibodies were used to assess endogenous hASUN levels. (C) No changes were observed in the levels of GFP (control), GFP-BICD2, or endogenous CENP-F at G2/M. Tubulin was used as loading control.FIGURE 4:Identification of a functional NLS in human ASUN. (A) Schematic of Myc-tagged hASUN with predicted NLS (yellow box) in the C-terminal region of the protein. Letter A (red) indicates each charged residue of the endogenous NLS mutated to alanine in Myc-hASUNmutNLS and Myc-NLS-hASUNmutNLS. An exogenous NLS (blue box) was added between the Myc tag and either hASUN or hASUNmutNLS to generate Myc-NLS-hASUN and Myc-NLS-hASUNmutNLS, respectively. (B) Myc immunoblot analysis of lysates of transfected HeLa cells revealed Myc-hASUN fusion proteins of the predicted sizes. Tubulin was used as loading control. (C) Representative images showing predominant localizations of Myc-hASUN fusions in transfected HeLa cells. Scale bars, 10 μm. (D) Quantification of localization patterns of Myc-hASUN fusion proteins in transfected HeLa cells.TABLE 2:Quantification of localizations of tagged INT subunits in transfected HeLa cells.SubunitNuclearCytoplasmicDiffuseMyc-hASUN87 ± 2.24 ± 1.99 ± 2.4GFP-IntS2—100 ± 0—GFP-IntS3——100 ± 0GFP-IntS497 ± 2.1—3 ± 2.1GFP-IntS5——100 ± 0GFP-IntS693 ± 1.4—7 ± 1GFP-IntS7—93 ± 17 ± 1GFP-IntS985 ± 2.5—15 ± 2.5IntS10-GFP81 ± 6.5—19 ± 6GFP-IntS11—4 ± 0.596 ± 0GFP-IntS12100 ± 0——HeLa cells were transfected with the indicated expression constructs encoding GFP- or Myc-tagged versions of human INT subunits, fixed, and stained with phalloidin and DAPI. Cells expressing Myc-hASUN were also immunostained for Myc. Immunofluorescence microscopy was performed to assess localization patterns of INT fusion proteins (representative images presented in Figure 2, with quantification shown here). Localizations were scored as the percentage of cells with nuclear, cytoplasmic, or diffuse localization.In both fly testes and HeLa cells, we previously observed a temporal correlation in which a pool of cytoplasmic ASUN is present at the onset of dynein accumulation on the NE at G2/M (Anderson et al., 2009; Jodoin et al., 2012). In fly spermatocytes, dASUN undergoes a shift from nuclear to first appearing in the cytoplasm during late G2, roughly coinciding with dynein recruitment to the NE (Anderson et al., 2009). In HeLa cells, hASUN localizes to the cytoplasm at prophase, a cell cycle stage at which dynein is enriched on the NE (Jodoin et al., 2012). Owing to this correlation, we previously hypothesized that a cytoplasmic pool of ASUN is required for recruitment of dynein motors to the NE at G2/M (Jodoin et al., 2012).We considered the possibility that the INT complex might function within the cytoplasm to promote dynein recruitment to the NE in a manner independent of its role in snRNA processing within the nucleus. A broad survey of subcellular localization patterns of INT subunits has not been previously reported. To address whether the INT complex exists in the cytoplasm, we expressed GFP fusions of the majority of individual INT subunits in HeLa cells to visualize their localizations. We confirmed by immunoblotting for GFP that fusion proteins of the predicted sizes were stably expressed in transfected cells (Figure 2M).We divided the INT subunits into three categories based on their localization: predominantly nuclear (hASUN; IntS4, 6, 9, 10, and 12), predominantly cytoplasmic (IntS2 and 7), or evenly distributed between the nucleus and cytoplasm (IntS3, 5, and 11; Figure 2, C–L; quantified in Table 2). Given these complex patterns, we could not exclude the possibility that the INT complex (or perhaps INT subcomplexes) could have potential roles in the cytoplasm in addition to its established role in snRNA processing in the nucleus. It is interesting to note, however, that knockdown of any of several INT subunits that localized predominantly to the nucleus (hASUN; IntS4, 6, 9, or 12; excluding the nonessential IntS10) resulted in loss of perinuclear dynein in HeLa cells; these observations suggest that a nuclear pool of INT might be required to promote recruitment of cytoplasmic dynein to the NE (Figures 1 and 2 and Table 2).

INT subunits exhibit a range of subcellular localizations

We previously showed that Drosophila ASUN exhibits a dynamic localization pattern: in the Drosophila testes, mCherry-tagged dASUN (CHY-dASUN) expressed via a transgene shifts from exclusively nuclear in early G2 to diffusely present throughout the spermatocyte by late G2 (Anderson et al., 2009). Similar localizations (ranging from nuclear to cytoplasmic to diffuse throughout the cell) were observed when GFP-tagged dASUN was expressed in HeLa cells (Anderson et al., 2009). Similarly, we find in the present study that Myc-tagged hASUN expressed in HeLa cells localizes to the nucleus and/or cytoplasm, albeit with a predominantly nuclear pattern in most cells (Figure 2, A1–A3; quantified in Table 2 and later in Figure 4D).

INT subunits localize to the nucleus, cytoplasm, or both. HeLa cells were transfected with the indicated expression constructs encoding tagged versions of INT subunits. (A–L) After fixation, cells were stained with phalloidin and DAPI; cells expressing Myc-hASUN were also immunostained for Myc. (A) Myc-hASUN was either exclusively nuclear (A1), predominantly cytoplasmic (A2), or distributed throughout the cell (A3). (B) GFP (control) was diffusely present throughout the cell. (C–L) GFP-tagged versions of other INT subunits localized to the nucleus (E, G, I, J, L), cytoplasm (C, H), or both (D, F). Scale bar, 10 μm. (M) Immunoblot analysis of lysates of transfected cells using antibodies against Myc or GFP tags revealed fusion proteins of the predicted sizes. Tubulin was used as loading control.

Levels of a subset of INT subunits are reduced at G2/M. HeLa cells were transfected with the indicated GFP- or Myc-fusion constructs and either left untreated (asynchronous population; –) or treated with a CDK1 inhibitor (G2/M-arrested population; +) for 16 h. Immunoblot analysis of lysates of transfected cells using antibodies against GFP or Myc tags revealed decreased levels of several INT subunits at G2/M (A), whereas no changes were observed for other subunits (B). hASUN antibodies were used to assess endogenous hASUN levels. (C) No changes were observed in the levels of GFP (control), GFP-BICD2, or endogenous CENP-F at G2/M. Tubulin was used as loading control.

Identification of a functional NLS in human ASUN. (A) Schematic of Myc-tagged hASUN with predicted NLS (yellow box) in the C-terminal region of the protein. Letter A (red) indicates each charged residue of the endogenous NLS mutated to alanine in Myc-hASUNmutNLS and Myc-NLS-hASUNmutNLS. An exogenous NLS (blue box) was added between the Myc tag and either hASUN or hASUNmutNLS to generate Myc-NLS-hASUN and Myc-NLS-hASUNmutNLS, respectively. (B) Myc immunoblot analysis of lysates of transfected HeLa cells revealed Myc-hASUN fusion proteins of the predicted sizes. Tubulin was used as loading control. (C) Representative images showing predominant localizations of Myc-hASUN fusions in transfected HeLa cells. Scale bars, 10 μm. (D) Quantification of localization patterns of Myc-hASUN fusion proteins in transfected HeLa cells.

Quantification of localizations of tagged INT subunits in transfected HeLa cells.

HeLa cells were transfected with the indicated expression constructs encoding GFP- or Myc-tagged versions of human INT subunits, fixed, and stained with phalloidin and DAPI. Cells expressing Myc-hASUN were also immunostained for Myc. Immunofluorescence microscopy was performed to assess localization patterns of INT fusion proteins (representative images presented in Figure 2, with quantification shown here). Localizations were scored as the percentage of cells with nuclear, cytoplasmic, or diffuse localization.

In both fly testes and HeLa cells, we previously observed a temporal correlation in which a pool of cytoplasmic ASUN is present at the onset of dynein accumulation on the NE at G2/M (Anderson et al., 2009; Jodoin et al., 2012). In fly spermatocytes, dASUN undergoes a shift from nuclear to first appearing in the cytoplasm during late G2, roughly coinciding with dynein recruitment to the NE (Anderson et al., 2009). In HeLa cells, hASUN localizes to the cytoplasm at prophase, a cell cycle stage at which dynein is enriched on the NE (Jodoin et al., 2012). Owing to this correlation, we previously hypothesized that a cytoplasmic pool of ASUN is required for recruitment of dynein motors to the NE at G2/M (Jodoin et al., 2012).

We considered the possibility that the INT complex might function within the cytoplasm to promote dynein recruitment to the NE in a manner independent of its role in snRNA processing within the nucleus. A broad survey of subcellular localization patterns of INT subunits has not been previously reported. To address whether the INT complex exists in the cytoplasm, we expressed GFP fusions of the majority of individual INT subunits in HeLa cells to visualize their localizations. We confirmed by immunoblotting for GFP that fusion proteins of the predicted sizes were stably expressed in transfected cells (Figure 2M).

We divided the INT subunits into three categories based on their localization: predominantly nuclear (hASUN; IntS4, 6, 9, 10, and 12), predominantly cytoplasmic (IntS2 and 7), or evenly distributed between the nucleus and cytoplasm (IntS3, 5, and 11; Figure 2, C–L; quantified in Table 2). Given these complex patterns, we could not exclude the possibility that the INT complex (or perhaps INT subcomplexes) could have potential roles in the cytoplasm in addition to its established role in snRNA processing in the nucleus. It is interesting to note, however, that knockdown of any of several INT subunits that localized predominantly to the nucleus (hASUN; IntS4, 6, 9, or 12; excluding the nonessential IntS10) resulted in loss of perinuclear dynein in HeLa cells; these observations suggest that a nuclear pool of INT might be required to promote recruitment of cytoplasmic dynein to the NE (Figures 1 and 2 and Table 2).

Reduced levels of several INT subunits at G2/MWe further considered the possibility that a cytoplasmic pool of the INT complex might be required for dynein localization. In this case, the simplest model would be that INT acts in the cytoplasm at G2/M to mediate dynein recruitment to the NE. As a test of this model, we asked whether INT subunits are present at G2/M. HeLa cells expressing tagged INT subunits were either left untreated (asynchronous population) or treated with an inhibitor of cyclin-dependent kinase 1 (CDK1; to obtain a G2/M-arrested population), followed by immunoblotting of cell lysates to assess fusion protein levels (Vassilev, 2006). Treated cells acquired a rounded morphology consistent with a G2/M arrest, confirming efficacy of CDK1 inhibition. We found that the levels of 60% (6 of 10) of the INT subunits tested were markedly decreased in G2/M-arrested cells compared with asynchronously dividing cells (Figure 3A). In contrast, levels of 4 INT subunits, including hASUN (both endogenous and tagged), were unchanged at G2/M (Figure 3B). Other known regulators of dynein recruitment, BICD2 (tagged) and CENP-F (endogenous), as well as a GFP control, showed no change in levels at G2/M (Figure 3C). Of note, four subunits required for dynein localization (IntS2, IntS5, IntS6, and IntS9) were present at relatively low levels at G2/M (Figures 1 and 3A). These data led us to conclude that INT is unlikely to directly mediate dynein recruitment to the NE at this stage. Instead, we propose that INT functions earlier in the cell cycle, possibly at the level of RNA processing during interphase, to subsequently affect dynein localization at G2/M.

Reduced levels of several INT subunits at G2/M

We further considered the possibility that a cytoplasmic pool of the INT complex might be required for dynein localization. In this case, the simplest model would be that INT acts in the cytoplasm at G2/M to mediate dynein recruitment to the NE. As a test of this model, we asked whether INT subunits are present at G2/M. HeLa cells expressing tagged INT subunits were either left untreated (asynchronous population) or treated with an inhibitor of cyclin-dependent kinase 1 (CDK1; to obtain a G2/M-arrested population), followed by immunoblotting of cell lysates to assess fusion protein levels (Vassilev, 2006). Treated cells acquired a rounded morphology consistent with a G2/M arrest, confirming efficacy of CDK1 inhibition. We found that the levels of 60% (6 of 10) of the INT subunits tested were markedly decreased in G2/M-arrested cells compared with asynchronously dividing cells (Figure 3A). In contrast, levels of 4 INT subunits, including hASUN (both endogenous and tagged), were unchanged at G2/M (Figure 3B). Other known regulators of dynein recruitment, BICD2 (tagged) and CENP-F (endogenous), as well as a GFP control, showed no change in levels at G2/M (Figure 3C). Of note, four subunits required for dynein localization (IntS2, IntS5, IntS6, and IntS9) were present at relatively low levels at G2/M (Figures 1 and 3A). These data led us to conclude that INT is unlikely to directly mediate dynein recruitment to the NE at this stage. Instead, we propose that INT functions earlier in the cell cycle, possibly at the level of RNA processing during interphase, to subsequently affect dynein localization at G2/M.

Mammalian ASUN homologues contain a functional nuclear localization sequenceTo determine whether dynein recruitment to the NE is promoted by cytoplasmic ASUN, as we originally hypothesized, or by nuclear ASUN, as suggested by data presented here, we needed a method to direct the localization of ASUN within cells to either of these two compartments (Jodoin et al., 2012). To identify critical regions of ASUN required for its nuclear localization, we began by performing a structure–function analysis of the mouse ASUN homologue (mASUN). Full-length Myc-tagged mASUN expressed in transfected HeLa cells was concentrated in the nucleus, with some cells showing diffuse or cytoplasmic localization (Supplemental Figure S4, A and G). We divided full-length mASUN into three overlapping fragments (F1–F3) and generated constructs for expression of each fragment fused to GFP at its N-terminal end (Supplemental Figure S4A). On expression in transfected HeLa cells, we found that the three GFP-tagged mASUN fragments had distinct localization patterns. GFP-mASUN-F1 (N-terminal fragment) was present throughout the cell, with slight enrichment in the nucleus (Supplemental Figure S4, C and G), whereas GFP-mASUN-F2 (middle fragment) was predominantly cytoplasmic, with slight perinuclear enrichment (Supplemental Figure S4, D and G). GFP-mASUN-F3 (C-terminal fragment) appeared to be exclusively nuclear, suggesting that it likely contains critical sequences that mediate nuclear localization of full-length mASUN (Supplemental Figure S4, E and G). For all mASUN constructs used here, fusion proteins of the predicted sizes were observed by immunoblotting transfected HeLa cell lysates (Supplemental Figure S4B).We used nuclear localization sequence (NLS) prediction software to identify a putative NLS in the C-terminal region of mASUN (Supplemental Figure S4A). We generated Myc-mASUNmutNLS by performing site-directed mutagenesis to alter several charged residues to alanines within this region (Supplemental Figure S4A). When expressed in transfected HeLa cells, Myc-mASUN was predominantly nuclear, whereas Myc-mASUNmutNLS was predominantly cytoplasmic, confirming functionality of the candidate NLS (Supplemental Figure S4, F and G).We also used NLS prediction software to identify a putative NLS in the C-terminal region of the human ASUN homologue that was 100% identical to the verified NLS of mASUN (Figure 4A and Supplemental Figure S4A). We performed site-directed mutagenesis to change several charged residues in this sequence to alanines, thereby generating Myc-hASUNmutNLS (Figure 4A). When expressed in HeLa cells, Myc-hASUN was predominantly nuclear, with some cytoplasmic or diffuse localization (Figure 2, A1–A3; quantified in Table 2 and Figure 4D). In contrast, Myc-hASUNmutNLS was predominantly cytoplasmic, confirming that we had identified a functional NLS (Figure 4, C and D). For experiments presented later in Figure 6, we also generated constructs for addition of a strong exogenous NLS (PKKKRKV; derived from SV40 large T antigen) to both hASUN and hASUNmutNLS (C-terminal to the Myc tag) to produce Myc-NLS-hASUN and Myc-NLS-hASUNmutNLS, respectively (Figure 4A; Kalderon et al., 1984). As expected, both fusion proteins were predominantly nuclear when expressed in HeLa cells (Figure 4, C and D). We performed immunoblotting of lysates of transfected cells to confirm that all Myc-hASUN constructs used here produced fusion proteins of the predicted sizes (Figure 4B).FIGURE 5:Functional NLS is conserved in Drosophila ASUN homologue. (A) Schematic of CHY-tagged dASUN with predicted NLS (yellow box) in the C-terminal region of the protein. Letter A indicates each charged residue of the endogenous NLS mutated to alanine in CHY-dASUNmutNLS. An exogenous NLS (blue box) was added to the N-terminal end of CHY-tagged dASUN to generate NLS-CHY-dASUN. (B) CHY immunoblot analysis of testes lysates from asun males with or without germline expression of CHY-dASUN fusions revealed proteins of the predicted sizes. An intervening lane just left of the last lane was removed. Wild-type (WT) males and asun males lacking a transgene were used as negative controls. Tubulin was used as loading control. (C) Representative images showing localizations of CHY-dASUN fusions in transgenic G2 spermatocytes. Scale bar, 10 μm. (D) Quantification of localization patterns of CHY-dASUN fusion proteins in transgenic G2 spermatocytes.FIGURE 6:Nuclear pool of hASUN is required for dynein recruitment to the NE in HeLa cells. HeLa cells were transfected with NT or hASUN siRNA plus Myc (vector control) or Myc-hASUN expression constructs as indicated. (A) After nocodazole treatment, cells were fixed and stained for DIC and DNA. Representative images of cells. Perinuclear dynein was restored only by expression of hASUN fusion proteins with nuclear localization. Scale bar, 10 μm. (B) Quantification of cells with perinuclear dynein after the indicated siRNA and DNA transfections. *p < 0.0001 (pairwise comparisons indicated). N.S., nonsignificant statistical differences. (C) hASUN immunoblot of lysates of transfected HeLa cells confirmed depletion of hASUN in hASUN-siRNA cells, and Myc immunoblot confirmed expression of Myc-hASUN fusion proteins of the predicted sizes. Tubulin was used as loading control.

Mammalian ASUN homologues contain a functional nuclear localization sequence

To determine whether dynein recruitment to the NE is promoted by cytoplasmic ASUN, as we originally hypothesized, or by nuclear ASUN, as suggested by data presented here, we needed a method to direct the localization of ASUN within cells to either of these two compartments (Jodoin et al., 2012). To identify critical regions of ASUN required for its nuclear localization, we began by performing a structure–function analysis of the mouse ASUN homologue (mASUN). Full-length Myc-tagged mASUN expressed in transfected HeLa cells was concentrated in the nucleus, with some cells showing diffuse or cytoplasmic localization (Supplemental Figure S4, A and G). We divided full-length mASUN into three overlapping fragments (F1–F3) and generated constructs for expression of each fragment fused to GFP at its N-terminal end (Supplemental Figure S4A). On expression in transfected HeLa cells, we found that the three GFP-tagged mASUN fragments had distinct localization patterns. GFP-mASUN-F1 (N-terminal fragment) was present throughout the cell, with slight enrichment in the nucleus (Supplemental Figure S4, C and G), whereas GFP-mASUN-F2 (middle fragment) was predominantly cytoplasmic, with slight perinuclear enrichment (Supplemental Figure S4, D and G). GFP-mASUN-F3 (C-terminal fragment) appeared to be exclusively nuclear, suggesting that it likely contains critical sequences that mediate nuclear localization of full-length mASUN (Supplemental Figure S4, E and G). For all mASUN constructs used here, fusion proteins of the predicted sizes were observed by immunoblotting transfected HeLa cell lysates (Supplemental Figure S4B).

We used nuclear localization sequence (NLS) prediction software to identify a putative NLS in the C-terminal region of mASUN (Supplemental Figure S4A). We generated Myc-mASUNmutNLS by performing site-directed mutagenesis to alter several charged residues to alanines within this region (Supplemental Figure S4A). When expressed in transfected HeLa cells, Myc-mASUN was predominantly nuclear, whereas Myc-mASUNmutNLS was predominantly cytoplasmic, confirming functionality of the candidate NLS (Supplemental Figure S4, F and G).

We also used NLS prediction software to identify a putative NLS in the C-terminal region of the human ASUN homologue that was 100% identical to the verified NLS of mASUN (Figure 4A and Supplemental Figure S4A). We performed site-directed mutagenesis to change several charged residues in this sequence to alanines, thereby generating Myc-hASUNmutNLS (Figure 4A). When expressed in HeLa cells, Myc-hASUN was predominantly nuclear, with some cytoplasmic or diffuse localization (Figure 2, A1–A3; quantified in Table 2 and Figure 4D). In contrast, Myc-hASUNmutNLS was predominantly cytoplasmic, confirming that we had identified a functional NLS (Figure 4, C and D). For experiments presented later in Figure 6, we also generated constructs for addition of a strong exogenous NLS (PKKKRKV; derived from SV40 large T antigen) to both hASUN and hASUNmutNLS (C-terminal to the Myc tag) to produce Myc-NLS-hASUN and Myc-NLS-hASUNmutNLS, respectively (Figure 4A; Kalderon et al., 1984). As expected, both fusion proteins were predominantly nuclear when expressed in HeLa cells (Figure 4, C and D). We performed immunoblotting of lysates of transfected cells to confirm that all Myc-hASUN constructs used here produced fusion proteins of the predicted sizes (Figure 4B).

Functional NLS is conserved in Drosophila ASUN homologue. (A) Schematic of CHY-tagged dASUN with predicted NLS (yellow box) in the C-terminal region of the protein. Letter A indicates each charged residue of the endogenous NLS mutated to alanine in CHY-dASUNmutNLS. An exogenous NLS (blue box) was added to the N-terminal end of CHY-tagged dASUN to generate NLS-CHY-dASUN. (B) CHY immunoblot analysis of testes lysates from asun males with or without germline expression of CHY-dASUN fusions revealed proteins of the predicted sizes. An intervening lane just left of the last lane was removed. Wild-type (WT) males and asun males lacking a transgene were used as negative controls. Tubulin was used as loading control. (C) Representative images showing localizations of CHY-dASUN fusions in transgenic G2 spermatocytes. Scale bar, 10 μm. (D) Quantification of localization patterns of CHY-dASUN fusion proteins in transgenic G2 spermatocytes.

Nuclear pool of hASUN is required for dynein recruitment to the NE in HeLa cells. HeLa cells were transfected with NT or hASUN siRNA plus Myc (vector control) or Myc-hASUN expression constructs as indicated. (A) After nocodazole treatment, cells were fixed and stained for DIC and DNA. Representative images of cells. Perinuclear dynein was restored only by expression of hASUN fusion proteins with nuclear localization. Scale bar, 10 μm. (B) Quantification of cells with perinuclear dynein after the indicated siRNA and DNA transfections. *p < 0.0001 (pairwise comparisons indicated). N.S., nonsignificant statistical differences. (C) hASUN immunoblot of lysates of transfected HeLa cells confirmed depletion of hASUN in hASUN-siRNA cells, and Myc immunoblot confirmed expression of Myc-hASUN fusion proteins of the predicted sizes. Tubulin was used as loading control.

Drosophila ASUN contains a functional NLSWe also identified a putative NLS in the C-terminal region of dASUN by using NLS prediction software (Supplemental Figure S5A). We previously showed that GFP-tagged dASUN expressed in transfected HeLa cells localizes in the nucleus, cytoplasm, and between these two compartments, and we obtained similar results here using Myc-tagged dASUN (Supplemental Figure S5, A and B; Anderson et al., 2009). By site-directed mutagenesis, we changed several charged residues to alanines within the candidate NLS of Myc-dASUN to generate Myc-dASUNmutNLS (Supplemental Figure S5A). Introduction of these mutations resulted in a significant shift of the fusion protein from the nucleus to the cytoplasm of transfected HeLa cells, verifying functionality of the putative NLS in this system (Supplemental Figure S5B). For experiments presented later in this study, we also generated constructs encoding Myc-NLS-dASUN and Myc-NLS-dASUNmutNLS (each containing a strong exogenous NLS placed C-terminal to the Myc tag) and confirmed that both of these fusion proteins were predominantly nuclear when expressed in HeLa cells (Supplemental Figure S5, A and B).We next tested whether the NLS that we identified in dASUN using transfected HeLa cells was also functional in vivo. We previously established transgenic Drosophila lines with testes-specific expression of CHY-tagged dASUN and introduced a copy of this transgene into the asun background (Figure 5A; Anderson et al., 2009). In G2 spermatocytes, CHY-dASUN was predominantly nuclear, with some diffuse localization (Figure 5, C and D). We used the same approach to express CHY-tagged dASUNmutNLS (carrying mutations in the predicted NLS as described earlier) in asun testes (Figure 5A). CHY-dASUNmutNLS was tightly localized to the cytoplasm of G2 spermatocytes, a pattern not observed for wild-type CHY-dASUN; thus, the NLS sequence we identified in dASUN using cultured human cells was also functional in vivo (Figure 5, C and D). For experiments presented later in Figure 7, to enrich for dASUN in the nucleus, we used the same approach to express CHY-dASUN with a strong exogenous NLS (NLS-CHY-dASUN) in asun testes (Figure 5A). The NLS-CHY-dASUN fusion was even more tightly localized to the nucleus of G2 spermatocytes than CHY-dASUN (Figure 5, C and D). For all CHY-dASUN constructs described here, fusion proteins of the predicted sizes were observed by immunoblotting of transgenic testes lysates (Figure 5B).FIGURE 7:Nuclear pool of dASUN is required for dynein recruitment to the NE in Drosophila spermatocytes. (A, B) Testes dissected from wild-type (WT) or asun males with or without germline expression of CHY-tagged dASUN fusion proteins were stained for dynein heavy chain and DNA. (A) Representative images of G2 spermatocytes. Perinuclear dynein was restored to asun G2 spermatocytes only by expression of dASUN fusion proteins with nuclear localization. Scale bar, 10 μm. (B) Quantification of G2 spermatocytes with perinuclear dynein. (C) Fertility assay shows the average number of progeny per fertile male. *p < 0.0001 (compared with wild-type control).

Drosophila ASUN contains a functional NLS

We also identified a putative NLS in the C-terminal region of dASUN by using NLS prediction software (Supplemental Figure S5A). We previously showed that GFP-tagged dASUN expressed in transfected HeLa cells localizes in the nucleus, cytoplasm, and between these two compartments, and we obtained similar results here using Myc-tagged dASUN (Supplemental Figure S5, A and B; Anderson et al., 2009). By site-directed mutagenesis, we changed several charged residues to alanines within the candidate NLS of Myc-dASUN to generate Myc-dASUNmutNLS (Supplemental Figure S5A). Introduction of these mutations resulted in a significant shift of the fusion protein from the nucleus to the cytoplasm of transfected HeLa cells, verifying functionality of the putative NLS in this system (Supplemental Figure S5B). For experiments presented later in this study, we also generated constructs encoding Myc-NLS-dASUN and Myc-NLS-dASUNmutNLS (each containing a strong exogenous NLS placed C-terminal to the Myc tag) and confirmed that both of these fusion proteins were predominantly nuclear when expressed in HeLa cells (Supplemental Figure S5, A and B).

We next tested whether the NLS that we identified in dASUN using transfected HeLa cells was also functional in vivo. We previously established transgenic Drosophila lines with testes-specific expression of CHY-tagged dASUN and introduced a copy of this transgene into the asun background (Figure 5A; Anderson et al., 2009). In G2 spermatocytes, CHY-dASUN was predominantly nuclear, with some diffuse localization (Figure 5, C and D). We used the same approach to express CHY-tagged dASUNmutNLS (carrying mutations in the predicted NLS as described earlier) in asun testes (Figure 5A). CHY-dASUNmutNLS was tightly localized to the cytoplasm of G2 spermatocytes, a pattern not observed for wild-type CHY-dASUN; thus, the NLS sequence we identified in dASUN using cultured human cells was also functional in vivo (Figure 5, C and D). For experiments presented later in Figure 7, to enrich for dASUN in the nucleus, we used the same approach to express CHY-dASUN with a strong exogenous NLS (NLS-CHY-dASUN) in asun testes (Figure 5A). The NLS-CHY-dASUN fusion was even more tightly localized to the nucleus of G2 spermatocytes than CHY-dASUN (Figure 5, C and D). For all CHY-dASUN constructs described here, fusion proteins of the predicted sizes were observed by immunoblotting of transgenic testes lysates (Figure 5B).

Nuclear pool of dASUN is required for dynein recruitment to the NE in Drosophila spermatocytes. (A, B) Testes dissected from wild-type (WT) or asun males with or without germline expression of CHY-tagged dASUN fusion proteins were stained for dynein heavy chain and DNA. (A) Representative images of G2 spermatocytes. Perinuclear dynein was restored to asun G2 spermatocytes only by expression of dASUN fusion proteins with nuclear localization. Scale bar, 10 μm. (B) Quantification of G2 spermatocytes with perinuclear dynein. (C) Fertility assay shows the average number of progeny per fertile male. *p < 0.0001 (compared with wild-type control).

A nuclear pool of ASUN is required in HeLa cells for dynein localizationWe used our hASUN expression constructs to address whether hASUN functions in the cytoplasm or nucleus of HeLa cells to promote dynein recruitment to the NE. We tested the capacity of cytoplasmic Myc-hASUNmutNLS to restore perinuclear dynein to cells depleted of endogenous hASUN. Cells treated with NT or hASUN siRNA and transfected with Myc, Myc-hASUN, or Myc-hASUNmutNLS constructs (refractory to hASUN siRNA) were analyzed using our dynein localization assay (Figure 6, A and B). As expected, we found that Myc-hASUN rescued the loss of perinuclear dynein caused by endogenous hASUN depletion (Figure 6A; quantified in Figure 6B). In contrast, dynein remained diffuse in the cytoplasm upon expression of Myc-hASUNmutNLS in cells lacking endogenous hASUN, suggesting that hASUN may function in the nucleus to regulate dynein localization (Figure 6, A and B).To further confirm that a nuclear pool of hASUN is required for dynein localization and rule out the trivial possibility that mutation of the NLS of hASUN might interfere with its activity in other ways (e.g., improper protein folding), we asked whether forced localization of Myc-hASUNmutNLS to the nucleus via addition of an exogenous NLS would render it competent to restore perinuclear dynein in cells depleted of endogenous hASUN. Indeed, we found that dynein localized normally upon expression of Myc-NLS-hASUN or Myc-NLS-hASUNmutNLS in hASUN-siRNA cells (Figure 6, A and B). These findings suggest that the mutations we introduced into the endogenous NLS of hASUN were not deleterious per se to the protein, but rather that nuclear residence is required for hASUN to exert its effect on dynein localization. Immunoblotting of cell lysates for endogenous hASUN and the Myc tag confirmed efficient depletion of hASUN by siRNA treatment and expression of all Myc-tagged hASUN fusions used here (Figure 6C).We next used our dASUN constructs to test whether dASUN acts within the cytoplasm or nucleus of transfected HeLa cells to promote dynein localization. We previously reported that loss of perinuclear dynein in hASUN-depleted HeLa cells was rescued by expression of Cherry-dASUN, and similar results were obtained here by expressing Myc-dASUN in hASUN-siRNA cells (Supplemental Figure S5C; Anderson et al., 2009). Consistent with our hASUN results, we found that expression of cytoplasmic-localized Myc-dASUNmutNLS in hASUN-siRNA cells failed to restore perinuclear dynein, whereas nuclear-localized Myc-NLS-dASUN or Myc-NLS-dASUNmutNLS were both competent to promote dynein localization (Supplemental Figure S5C). Immunoblotting of cell lysates for endogenous hASUN and the Myc tag confirmed efficient depletion of hASUN by siRNA treatment and expression of all Myc-tagged dASUN fusions used here (Supplemental Figure S5D). Taken together, these data provide further support for a model in which a nuclear pool of ASUN is required for recruitment of cytoplasmic dynein to the NE in cultured human cells and suggest that this mechanism might be conserved in Drosophila.

A nuclear pool of ASUN is required in HeLa cells for dynein localization

We used our hASUN expression constructs to address whether hASUN functions in the cytoplasm or nucleus of HeLa cells to promote dynein recruitment to the NE. We tested the capacity of cytoplasmic Myc-hASUNmutNLS to restore perinuclear dynein to cells depleted of endogenous hASUN. Cells treated with NT or hASUN siRNA and transfected with Myc, Myc-hASUN, or Myc-hASUNmutNLS constructs (refractory to hASUN siRNA) were analyzed using our dynein localization assay (Figure 6, A and B). As expected, we found that Myc-hASUN rescued the loss of perinuclear dynein caused by endogenous hASUN depletion (Figure 6A; quantified in Figure 6B). In contrast, dynein remained diffuse in the cytoplasm upon expression of Myc-hASUNmutNLS in cells lacking endogenous hASUN, suggesting that hASUN may function in the nucleus to regulate dynein localization (Figure 6, A and B).

To further confirm that a nuclear pool of hASUN is required for dynein localization and rule out the trivial possibility that mutation of the NLS of hASUN might interfere with its activity in other ways (e.g., improper protein folding), we asked whether forced localization of Myc-hASUNmutNLS to the nucleus via addition of an exogenous NLS would render it competent to restore perinuclear dynein in cells depleted of endogenous hASUN. Indeed, we found that dynein localized normally upon expression of Myc-NLS-hASUN or Myc-NLS-hASUNmutNLS in hASUN-siRNA cells (Figure 6, A and B). These findings suggest that the mutations we introduced into the endogenous NLS of hASUN were not deleterious per se to the protein, but rather that nuclear residence is required for hASUN to exert its effect on dynein localization. Immunoblotting of cell lysates for endogenous hASUN and the Myc tag confirmed efficient depletion of hASUN by siRNA treatment and expression of all Myc-tagged hASUN fusions used here (Figure 6C).

We next used our dASUN constructs to test whether dASUN acts within the cytoplasm or nucleus of transfected HeLa cells to promote dynein localization. We previously reported that loss of perinuclear dynein in hASUN-depleted HeLa cells was rescued by expression of Cherry-dASUN, and similar results were obtained here by expressing Myc-dASUN in hASUN-siRNA cells (Supplemental Figure S5C; Anderson et al., 2009). Consistent with our hASUN results, we found that expression of cytoplasmic-localized Myc-dASUNmutNLS in hASUN-siRNA cells failed to restore perinuclear dynein, whereas nuclear-localized Myc-NLS-dASUN or Myc-NLS-dASUNmutNLS were both competent to promote dynein localization (Supplemental Figure S5C). Immunoblotting of cell lysates for endogenous hASUN and the Myc tag confirmed efficient depletion of hASUN by siRNA treatment and expression of all Myc-tagged dASUN fusions used here (Supplemental Figure S5D). Taken together, these data provide further support for a model in which a nuclear pool of ASUN is required for recruitment of cytoplasmic dynein to the NE in cultured human cells and suggest that this mechanism might be conserved in Drosophila.

A nuclear pool of dASUN is required in Drosophila spermatocytes for dynein localizationWe previously identified ASUN as a critical regulator of dynein localization during Drosophila spermatogenesis (Anderson et al., 2009). We used this system to determine whether the requirement for nuclear ASUN in regulating cytoplasmic dynein localization that we observed in HeLa cells is conserved in vivo. Using transgenic Drosophila lines that we established (Figure 5A), we assessed the capacity of cytoplasmic CHY-dASUNmutNLS or nuclear NLS-CHY-dASUN expressed in asun testes to promote dynein localization. As previously reported, we found that CHY-dASUN rescued loss of perinuclear dynein in asun spermatocytes (Figure 7, A and B; Anderson et al., 2009). asun spermatocytes expressing CHY-dASUNmutNLS, however, lacked perinuclear dynein; instead, dynein remained diffuse in the cytoplasm, similar to the phenotype of asun spermatocytes lacking a transgenic rescue construct (Figure 7, A and B). Expression of NLS-CHY-dASUN in asun spermatocytes restored perinuclear dynein to nearly wild-type levels, suggesting that a nuclear pool of dASUN is responsible for regulating the localization of cytoplasmic dynein motors in this system.In addition to the failure of dynein localization, asun mutants display a range of defects: spermatocytes arrested at prophase I with unattached centrosomes, impaired sperm bundling, and male sterility (Anderson et al., 2009). This constellation of asun phenotypes is likely secondary to loss of perinuclear dynein in G2 spermatocytes, an event that occurred upstream of these defects. To determine whether restoration of dynein on the NE of asun spermatocytes by NLS-CHY-dASUN expression was sufficient to rescue the most downstream defect, male sterility, we scored the number of live progeny per fertile male. We found that NLS-CHY-dASUN could restore fertility to asun males to the same degree as the CHY-dASUN control, whereas only partial rescue was achieved with CHY-dASUNmutNLS (Figure 7C).Taken together, these data suggest that a nuclear pool of ASUN plays a conserved role, from Drosophila to humans, in the recruitment of cytoplasmic dynein motors to the NE at G2/M. Combined with evidence presented here that additional INT subunits likewise play an essential role in this recruitment step, we hypothesize that the Integrator complex mediates the processing of a critical RNA target(s) required for the production of a cytoplasmic protein that is directly involved in the regulation of dynein localization.

A nuclear pool of dASUN is required in Drosophila spermatocytes for dynein localization

We previously identified ASUN as a critical regulator of dynein localization during Drosophila spermatogenesis (Anderson et al., 2009). We used this system to determine whether the requirement for nuclear ASUN in regulating cytoplasmic dynein localization that we observed in HeLa cells is conserved in vivo. Using transgenic Drosophila lines that we established (Figure 5A), we assessed the capacity of cytoplasmic CHY-dASUNmutNLS or nuclear NLS-CHY-dASUN expressed in asun testes to promote dynein localization. As previously reported, we found that CHY-dASUN rescued loss of perinuclear dynein in asun spermatocytes (Figure 7, A and B; Anderson et al., 2009). asun spermatocytes expressing CHY-dASUNmutNLS, however, lacked perinuclear dynein; instead, dynein remained diffuse in the cytoplasm, similar to the phenotype of asun spermatocytes lacking a transgenic rescue construct (Figure 7, A and B). Expression of NLS-CHY-dASUN in asun spermatocytes restored perinuclear dynein to nearly wild-type levels, suggesting that a nuclear pool of dASUN is responsible for regulating the localization of cytoplasmic dynein motors in this system.

In addition to the failure of dynein localization, asun mutants display a range of defects: spermatocytes arrested at prophase I with unattached centrosomes, impaired sperm bundling, and male sterility (Anderson et al., 2009). This constellation of asun phenotypes is likely secondary to loss of perinuclear dynein in G2 spermatocytes, an event that occurred upstream of these defects. To determine whether restoration of dynein on the NE of asun spermatocytes by NLS-CHY-dASUN expression was sufficient to rescue the most downstream defect, male sterility, we scored the number of live progeny per fertile male. We found that NLS-CHY-dASUN could restore fertility to asun males to the same degree as the CHY-dASUN control, whereas only partial rescue was achieved with CHY-dASUNmutNLS (Figure 7C).

Taken together, these data suggest that a nuclear pool of ASUN plays a conserved role, from Drosophila to humans, in the recruitment of cytoplasmic dynein motors to the NE at G2/M. Combined with evidence presented here that additional INT subunits likewise play an essential role in this recruitment step, we hypothesize that the Integrator complex mediates the processing of a critical RNA target(s) required for the production of a cytoplasmic protein that is directly involved in the regulation of dynein localization.

DISCUSSIONA newly identified role for Integrator in promoting dynein recruitment to the NEWe previously described an essential role for ASUN in regulating dynein localization in Drosophila spermatogenesis and cultured human cells (Anderson et al., 2009; Jodoin et al., 2012). Recent studies identified a second role for ASUN as an Integrator subunit required for snRNA-processing activity of the complex (Baillat et al., 2005; Malovannaya et al., 2010; Chen et al., 2012). Owing to the seemingly divergent nature of these two roles, we initially speculated that ASUN might “moonlight” by performing cellular functions within the cytoplasm independently of INT. To disprove this model, we here asked whether additional INT subunits, such as ASUN, are required for dynein localization. Our finding that depletion of individual INT subunits recapitulates the loss of perinuclear dynein in hASUN-siRNA cells argues that ASUN functions within the Integrator complex to mediate this process as opposed to working independently. Given the established role of INT in snRNA processing, we considered that loss of perinuclear dynein in INT-depleted cells could be an indirect consequence of a failure to produce mature ASUN transcripts; our data, however, do not support this idea. Conversely, we find that ASUN is required for stability of several INT subunits, further underscoring its importance for functioning of the complex.A model for regulation of dynein localization by integratorWe previously hypothesized that ASUN functions in the cytoplasm to promote redistribution of dynein from diffuse in the cytoplasm to NE anchored at G2/M (Anderson et al., 2009; Jodoin et al., 2012). We considered that a pool of Integrator might exist in the cytoplasm that could mediate dynein recruitment to the NE via a mechanism independent of its snRNA-processing role in the nucleus. Our finding that INT subunits exhibit a range of localization patterns in HeLa cells (predominantly nuclear to predominantly cytoplasmic) neither strongly supports nor refutes this model.Our identification of a functional NLS in Drosophila and mammalian ASUN homologues allowed us to manipulate the subcellular compartmentalization of ASUN and assess effects on dynein localization. We find that forced cytoplasmic localization of ASUN (via endogenous NLS mutation) hinders its capacity to promote dynein recruitment to the NE, whereas redirection of this mutant protein into the nucleus (via exogenous NLS addition) restores its function. These data indicate that Integrator likely acts from within the nucleus to control dynein localization in the cytoplasm. Furthermore, our observation that levels of several INT subunits are paradoxically decreased at G2/M, the stage at which dynein normally accumulates on the NE, suggests that Integrator may exert its effect on dynein by acting earlier in the cell cycle. We cannot exclude the possible existence of a subcomplex composed of a subset of INT subunits that are stable at G2/M; however, based on our finding that four subunits (IntS2, 5, 6, and 9) present at reduced levels at G2/M are nonetheless required for dynein localization, it is unlikely that any such subcomplex alone would be sufficient to mediate dynein recruitment to the NE.We propose a model in which nuclear-localized Integrator complex, including ASUN, mediates 3′-end processing of snRNA, which in turn is required for normal processing of mRNA encoding a key regulator(s) of cytoplasmic dynein localization. When Integrator activity is compromised (e.g., by knockdown of an essential subunit), production of critical transcript(s) during interphase is impaired, leading to reduction of perinuclear dynein at G2/M. To further elucidate mechanisms underlying the temporal and spatial control of dynein, it will be important to identify critical target(s) of INT involved in this process.What are the critical targets of Integrator that mediate dynein localization?On the basis of data presented here, we hypothesize that transcripts encoding a key regulator(s) of dynein localization would be misprocessed after down-regulation of INT. In a preliminary effort to identify such a target(s), we performed a high-throughput RNA-seq screen using both control siRNA cells and IntS11-siRNA HeLa cells (T.R.A. and E.J.W., unpublished observations). We compared mRNA isolated from both populations, with an emphasis on identifying transcripts that were aberrantly spliced in the absence of functional INT. Although we observed several thousand alterations in splicing throughout the genome, we found no evidence for misprocessing of transcripts encoding 1) dynein–dynactin subunits or adaptor proteins or 2) components of the BICD2-RanBP2 or NudE/EL-CENP-F-Nup133 dynein-binding cassettes. We speculate that novel regulators of dynein localization could be the critical mRNA targets of INT involved in this process. The identification of Drosophila ASUN (also known as Mat89Bb) as a positive regulator of siRNA, endo-siRNA, and microRNA pathways in three high-throughput screens, however, suggests that the Integrator complex may impinge on other classes of small RNAs and highlights the need for further studies of its activities (Zhou et al., 2008).When comparing the list of individual INT subunits required for U7 snRNA processing and those required for dynein localization, only one discrepancy emerges (Table 1). IntS7 was previously shown to be essential for processing of U7 and spliceosomal snRNA using a cell-based reporter and through measurement of endogenous transcripts, but we find here that it is dispensable for dynein recruitment to the NE (Ezzeddine et al., 2011; Chen et al., 2012). Given that we observed significant RNA interference-mediated depletion (Supplemental Figure S2), one possible explanation for this differential requirement is that the assays used to measure RNA processing might be more sensitive to perturbations of INT function than the dynein localization assay, perhaps because the latter is a more downstream event.Additional cellular and developmental roles of integratorThe Integrator complex mediates 3′-end processing of snRNAs that play essential roles in global gene expression. Given that Integrator is likely required for accurate production of a plethora of proteins, it is not surprising that loss of its activity is associated with a wide range of cellular and developmental phenotypes. It was recently reported that siRNA-mediated down-regulation of IntS4 leads to defects in formation of nuclear structures known as Cajal bodies (Takata et al., 2012). Another group found that IntS6 and IntS11 are required for proper differentiation of adipocytes in a cultured cell system; although the underlying mechanism is unknown, the authors hypothesized that U1 and U2 snRNAs are involved (Otani et al., 2013). In large-scale screens performed in zebrafish and C. elegans, IntS7 homologues were shown to be required for normal craniofacial development (Golling et al., 2002; Kamath et al., 2003). Mutation of Drosophila IntS7 (deflated) results in abdominal phenotypes due to cell cycle and signaling defects (Rutkowski and Warren, 2009). Integrator is also essential for hematopoiesis in zebrafish: down-regulation of IntS5, IntS9, or IntS11 causes aberrant splicing of smad1 and smad5 transcripts, generating a dominant-negative form of Smad that disrupts erythrocyte differentiation (Tao et al., 2009).What remains to be elucidated is how perturbations in snRNA 3′-end formation generate phenotypes so specific and yet so diverse. Although the favored hypothesis is that global reduction in snRNA biosynthesis negatively affects splicing of a subset of transcripts, the criteria for defining this set are unclear. It is likely that these sensitive transcripts will be enriched for either suboptimal splice sites, alternative splice sites, or minor spliceosome-dependent introns. Regardless of the root cause, our discovery of a new role for the Integrator complex in regulating dynein localization adds to the growing list of INT-dependent processes.

A newly identified role for Integrator in promoting dynein recruitment to the NEWe previously described an essential role for ASUN in regulating dynein localization in Drosophila spermatogenesis and cultured human cells (Anderson et al., 2009; Jodoin et al., 2012). Recent studies identified a second role for ASUN as an Integrator subunit required for snRNA-processing activity of the complex (Baillat et al., 2005; Malovannaya et al., 2010; Chen et al., 2012). Owing to the seemingly divergent nature of these two roles, we initially speculated that ASUN might “moonlight” by performing cellular functions within the cytoplasm independently of INT. To disprove this model, we here asked whether additional INT subunits, such as ASUN, are required for dynein localization. Our finding that depletion of individual INT subunits recapitulates the loss of perinuclear dynein in hASUN-siRNA cells argues that ASUN functions within the Integrator complex to mediate this process as opposed to working independently. Given the established role of INT in snRNA processing, we considered that loss of perinuclear dynein in INT-depleted cells could be an indirect consequence of a failure to produce mature ASUN transcripts; our data, however, do not support this idea. Conversely, we find that ASUN is required for stability of several INT subunits, further underscoring its importance for functioning of the complex.

A newly identified role for Integrator in promoting dynein recruitment to the NE

We previously described an essential role for ASUN in regulating dynein localization in Drosophila spermatogenesis and cultured human cells (Anderson et al., 2009; Jodoin et al., 2012). Recent studies identified a second role for ASUN as an Integrator subunit required for snRNA-processing activity of the complex (Baillat et al., 2005; Malovannaya et al., 2010; Chen et al., 2012). Owing to the seemingly divergent nature of these two roles, we initially speculated that ASUN might “moonlight” by performing cellular functions within the cytoplasm independently of INT. To disprove this model, we here asked whether additional INT subunits, such as ASUN, are required for dynein localization. Our finding that depletion of individual INT subunits recapitulates the loss of perinuclear dynein in hASUN-siRNA cells argues that ASUN functions within the Integrator complex to mediate this process as opposed to working independently. Given the established role of INT in snRNA processing, we considered that loss of perinuclear dynein in INT-depleted cells could be an indirect consequence of a failure to produce mature ASUN transcripts; our data, however, do not support this idea. Conversely, we find that ASUN is required for stability of several INT subunits, further underscoring its importance for functioning of the complex.

A model for regulation of dynein localization by integratorWe previously hypothesized that ASUN functions in the cytoplasm to promote redistribution of dynein from diffuse in the cytoplasm to NE anchored at G2/M (Anderson et al., 2009; Jodoin et al., 2012). We considered that a pool of Integrator might exist in the cytoplasm that could mediate dynein recruitment to the NE via a mechanism independent of its snRNA-processing role in the nucleus. Our finding that INT subunits exhibit a range of localization patterns in HeLa cells (predominantly nuclear to predominantly cytoplasmic) neither strongly supports nor refutes this model.Our identification of a functional NLS in Drosophila and mammalian ASUN homologues allowed us to manipulate the subcellular compartmentalization of ASUN and assess effects on dynein localization. We find that forced cytoplasmic localization of ASUN (via endogenous NLS mutation) hinders its capacity to promote dynein recruitment to the NE, whereas redirection of this mutant protein into the nucleus (via exogenous NLS addition) restores its function. These data indicate that Integrator likely acts from within the nucleus to control dynein localization in the cytoplasm. Furthermore, our observation that levels of several INT subunits are paradoxically decreased at G2/M, the stage at which dynein normally accumulates on the NE, suggests that Integrator may exert its effect on dynein by acting earlier in the cell cycle. We cannot exclude the possible existence of a subcomplex composed of a subset of INT subunits that are stable at G2/M; however, based on our finding that four subunits (IntS2, 5, 6, and 9) present at reduced levels at G2/M are nonetheless required for dynein localization, it is unlikely that any such subcomplex alone would be sufficient to mediate dynein recruitment to the NE.We propose a model in which nuclear-localized Integrator complex, including ASUN, mediates 3′-end processing of snRNA, which in turn is required for normal processing of mRNA encoding a key regulator(s) of cytoplasmic dynein localization. When Integrator activity is compromised (e.g., by knockdown of an essential subunit), production of critical transcript(s) during interphase is impaired, leading to reduction of perinuclear dynein at G2/M. To further elucidate mechanisms underlying the temporal and spatial control of dynein, it will be important to identify critical target(s) of INT involved in this process.

A model for regulation of dynein localization by integrator

We previously hypothesized that ASUN functions in the cytoplasm to promote redistribution of dynein from diffuse in the cytoplasm to NE anchored at G2/M (Anderson et al., 2009; Jodoin et al., 2012). We considered that a pool of Integrator might exist in the cytoplasm that could mediate dynein recruitment to the NE via a mechanism independent of its snRNA-processing role in the nucleus. Our finding that INT subunits exhibit a range of localization patterns in HeLa cells (predominantly nuclear to predominantly cytoplasmic) neither strongly supports nor refutes this model.

Our identification of a functional NLS in Drosophila and mammalian ASUN homologues allowed us to manipulate the subcellular compartmentalization of ASUN and assess effects on dynein localization. We find that forced cytoplasmic localization of ASUN (via endogenous NLS mutation) hinders its capacity to promote dynein recruitment to the NE, whereas redirection of this mutant protein into the nucleus (via exogenous NLS addition) restores its function. These data indicate that Integrator likely acts from within the nucleus to control dynein localization in the cytoplasm. Furthermore, our observation that levels of several INT subunits are paradoxically decreased at G2/M, the stage at which dynein normally accumulates on the NE, suggests that Integrator may exert its effect on dynein by acting earlier in the cell cycle. We cannot exclude the possible existence of a subcomplex composed of a subset of INT subunits that are stable at G2/M; however, based on our finding that four subunits (IntS2, 5, 6, and 9) present at reduced levels at G2/M are nonetheless required for dynein localization, it is unlikely that any such subcomplex alone would be sufficient to mediate dynein recruitment to the NE.

We propose a model in which nuclear-localized Integrator complex, including ASUN, mediates 3′-end processing of snRNA, which in turn is required for normal processing of mRNA encoding a key regulator(s) of cytoplasmic dynein localization. When Integrator activity is compromised (e.g., by knockdown of an essential subunit), production of critical transcript(s) during interphase is impaired, leading to reduction of perinuclear dynein at G2/M. To further elucidate mechanisms underlying the temporal and spatial control of dynein, it will be important to identify critical target(s) of INT involved in this process.

What are the critical targets of Integrator that mediate dynein localization?On the basis of data presented here, we hypothesize that transcripts encoding a key regulator(s) of dynein localization would be misprocessed after down-regulation of INT. In a preliminary effort to identify such a target(s), we performed a high-throughput RNA-seq screen using both control siRNA cells and IntS11-siRNA HeLa cells (T.R.A. and E.J.W., unpublished observations). We compared mRNA isolated from both populations, with an emphasis on identifying transcripts that were aberrantly spliced in the absence of functional INT. Although we observed several thousand alterations in splicing throughout the genome, we found no evidence for misprocessing of transcripts encoding 1) dynein–dynactin subunits or adaptor proteins or 2) components of the BICD2-RanBP2 or NudE/EL-CENP-F-Nup133 dynein-binding cassettes. We speculate that novel regulators of dynein localization could be the critical mRNA targets of INT involved in this process. The identification of Drosophila ASUN (also known as Mat89Bb) as a positive regulator of siRNA, endo-siRNA, and microRNA pathways in three high-throughput screens, however, suggests that the Integrator complex may impinge on other classes of small RNAs and highlights the need for further studies of its activities (Zhou et al., 2008).When comparing the list of individual INT subunits required for U7 snRNA processing and those required for dynein localization, only one discrepancy emerges (Table 1). IntS7 was previously shown to be essential for processing of U7 and spliceosomal snRNA using a cell-based reporter and through measurement of endogenous transcripts, but we find here that it is dispensable for dynein recruitment to the NE (Ezzeddine et al., 2011; Chen et al., 2012). Given that we observed significant RNA interference-mediated depletion (Supplemental Figure S2), one possible explanation for this differential requirement is that the assays used to measure RNA processing might be more sensitive to perturbations of INT function than the dynein localization assay, perhaps because the latter is a more downstream event.

What are the critical targets of Integrator that mediate dynein localization?

On the basis of data presented here, we hypothesize that transcripts encoding a key regulator(s) of dynein localization would be misprocessed after down-regulation of INT. In a preliminary effort to identify such a target(s), we performed a high-throughput RNA-seq screen using both control siRNA cells and IntS11-siRNA HeLa cells (T.R.A. and E.J.W., unpublished observations). We compared mRNA isolated from both populations, with an emphasis on identifying transcripts that were aberrantly spliced in the absence of functional INT. Although we observed several thousand alterations in splicing throughout the genome, we found no evidence for misprocessing of transcripts encoding 1) dynein–dynactin subunits or adaptor proteins or 2) components of the BICD2-RanBP2 or NudE/EL-CENP-F-Nup133 dynein-binding cassettes. We speculate that novel regulators of dynein localization could be the critical mRNA targets of INT involved in this process. The identification of Drosophila ASUN (also known as Mat89Bb) as a positive regulator of siRNA, endo-siRNA, and microRNA pathways in three high-throughput screens, however, suggests that the Integrator complex may impinge on other classes of small RNAs and highlights the need for further studies of its activities (Zhou et al., 2008).

When comparing the list of individual INT subunits required for U7 snRNA processing and those required for dynein localization, only one discrepancy emerges (Table 1). IntS7 was previously shown to be essential for processing of U7 and spliceosomal snRNA using a cell-based reporter and through measurement of endogenous transcripts, but we find here that it is dispensable for dynein recruitment to the NE (Ezzeddine et al., 2011; Chen et al., 2012). Given that we observed significant RNA interference-mediated depletion (Supplemental Figure S2), one possible explanation for this differential requirement is that the assays used to measure RNA processing might be more sensitive to perturbations of INT function than the dynein localization assay, perhaps because the latter is a more downstream event.

Additional cellular and developmental roles of integratorThe Integrator complex mediates 3′-end processing of snRNAs that play essential roles in global gene expression. Given that Integrator is likely required for accurate production of a plethora of proteins, it is not surprising that loss of its activity is associated with a wide range of cellular and developmental phenotypes. It was recently reported that siRNA-mediated down-regulation of IntS4 leads to defects in formation of nuclear structures known as Cajal bodies (Takata et al., 2012). Another group found that IntS6 and IntS11 are required for proper differentiation of adipocytes in a cultured cell system; although the underlying mechanism is unknown, the authors hypothesized that U1 and U2 snRNAs are involved (Otani et al., 2013). In large-scale screens performed in zebrafish and C. elegans, IntS7 homologues were shown to be required for normal craniofacial development (Golling et al., 2002; Kamath et al., 2003). Mutation of Drosophila IntS7 (deflated) results in abdominal phenotypes due to cell cycle and signaling defects (Rutkowski and Warren, 2009). Integrator is also essential for hematopoiesis in zebrafish: down-regulation of IntS5, IntS9, or IntS11 causes aberrant splicing of smad1 and smad5 transcripts, generating a dominant-negative form of Smad that disrupts erythrocyte differentiation (Tao et al., 2009).What remains to be elucidated is how perturbations in snRNA 3′-end formation generate phenotypes so specific and yet so diverse. Although the favored hypothesis is that global reduction in snRNA biosynthesis negatively affects splicing of a subset of transcripts, the criteria for defining this set are unclear. It is likely that these sensitive transcripts will be enriched for either suboptimal splice sites, alternative splice sites, or minor spliceosome-dependent introns. Regardless of the root cause, our discovery of a new role for the Integrator complex in regulating dynein localization adds to the growing list of INT-dependent processes.

Additional cellular and developmental roles of integrator

The Integrator complex mediates 3′-end processing of snRNAs that play essential roles in global gene expression. Given that Integrator is likely required for accurate production of a plethora of proteins, it is not surprising that loss of its activity is associated with a wide range of cellular and developmental phenotypes. It was recently reported that siRNA-mediated down-regulation of IntS4 leads to defects in formation of nuclear structures known as Cajal bodies (Takata et al., 2012). Another group found that IntS6 and IntS11 are required for proper differentiation of adipocytes in a cultured cell system; although the underlying mechanism is unknown, the authors hypothesized that U1 and U2 snRNAs are involved (Otani et al., 2013). In large-scale screens performed in zebrafish and C. elegans, IntS7 homologues were shown to be required for normal craniofacial development (Golling et al., 2002; Kamath et al., 2003). Mutation of Drosophila IntS7 (deflated) results in abdominal phenotypes due to cell cycle and signaling defects (Rutkowski and Warren, 2009). Integrator is also essential for hematopoiesis in zebrafish: down-regulation of IntS5, IntS9, or IntS11 causes aberrant splicing of smad1 and smad5 transcripts, generating a dominant-negative form of Smad that disrupts erythrocyte differentiation (Tao et al., 2009).

What remains to be elucidated is how perturbations in snRNA 3′-end formation generate phenotypes so specific and yet so diverse. Although the favored hypothesis is that global reduction in snRNA biosynthesis negatively affects splicing of a subset of transcripts, the criteria for defining this set are unclear. It is likely that these sensitive transcripts will be enriched for either suboptimal splice sites, alternative splice sites, or minor spliceosome-dependent introns. Regardless of the root cause, our discovery of a new role for the Integrator complex in regulating dynein localization adds to the growing list of INT-dependent processes.

MATERIALS AND METHODSDrosophila spermatocyte experimentsFlies were maintained at 25°C using standard techniques (Greenspan, 2004). y w flies obtained from Bloomington Drosophila Stock Center (Indiana University, Bloomington, IN) were used as the “wild-type” stock. The asunf02815 allele (Exelixis Collection, Harvard Medical School, Boston, MA) and a transgenic line with male germline–specific expression of CHY-tagged wild-type dASUN were previously described (Anderson et al., 2009). Transgenic lines for male germline–specific expression of CHY-tagged dASUN fusion proteins (constructs described later) were generated by P-element–mediated transformation via embryo injection (Rubin and Spradling, 1982). For each transgene, a single insertion mapping to the X chromosome was crossed into the asunf02815 background using standard genetic crosses.To test male fertility, individual adult males (2 d old) were placed in vials with five wild-type females (2 d old) and allowed to mate for 5 d. The average number of live adult progeny produced per fertile male was scored (≥10 males tested per genotype). Statistical analysis was performed using an unpaired Student's t test.Protein extracts were prepared by homogenizing dissected testes from newly eclosed males in SDS sample buffer. The equivalent of eight testes pairs was loaded per lane. After SDS–PAGE, immunoblotting was performed as described later.Live testes cells were prepared for examination by fluorescence microscopy as described previously (Kemphues et al., 1980). Briefly, testes were dissected from newly eclosed adult males, placed in a drop of phosphate-buffered saline (PBS) on a microscopic slide, and gently squashed under a glass coverslip after making a small incision near the stem cell hub. Formaldehyde fixation was performed as described previously (Gunsalus et al., 1995). Briefly, slides of squashed testes were snap-frozen, immersed in 4% formaldehyde (in PBS with 0.1% Triton X-100) for 7 min at −20°C after coverslip removal, and washed three times in PBS. Mouse anti–dynein heavy chain primary antibody (P1H4, 1:120; gift from T. Hays, University of Minnesota, Minneapolis, MN) and Cy3-conjugated secondary antibody (Invitrogen, Carlsbad, CA) were used (McGrail and Hays, 1997). Fixed samples were mounted in PBS with 4′,6-diamidino-2-phenylindole (DAPI) to visualize DNA. Wide-field fluorescence images were obtained using an Eclipse 80i microscope (Nikon, Melville, NY) with Plan-Fluor 40× objective. In experiments to determine the percentage of late G2 spermatocytes with perinuclear dynein, at least 200 cells were scored per genotype. Statistical analysis was performed using Fisher's exact test.Cell culture and treatmentsHeLa cells were maintained at 37°C and 5% CO2 in DMEM containing 10% fetal bovine serum, 1% l-glutamine, 100 μg/ml streptomycin, and 100 U/ml penicillin (Life Technologies, Carlsbad, CA). Plasmid DNA was transfected into cells using FuGENE HD (Promega, Madison, WI). siGENOME NT siRNA#5 (Dharmacon, Lafayette, CO) was used as negative control. siRNA used to silence hASUN (3′-UTR region; 5′-CAG CAA GAU GGU AUA GUU A-3′) was obtained from Dharmacon. siRNAs used to silence IntS1 (5′-CAU UUC UCC GUC GAU UAA A-3′), IntS2 (#1: 5′-CUC GUU UAG CUG UCA AUG U-3′ and #2: 5′-CCU UAA UCA GCG UUU CAG U-3′), IntS3 (5′-GAA GUA CUG AGU UCA GAU A-3′), IntS4 (5′-CAG CAU UGU UCU CAG AUC A-3′), IntS5 (5′-CAA GUU UGU CCA GUC ACG A-3′), IntS6 (#1: 5′-CAC UAA UGA UUC GAU AAU A-3′ and #2: 5′-CCA UGA AGA GGU CAA UAC U-3′), IntS7 (5′-GGC UAA AUA GUU UGA AGG A-3′), IntS9 (5′-GAA AUG CUU UCU UGG ACA A-3′), IntS10 (5′-GGA UAC UUG GCU UUG GUU A-3′), IntS11 (Albrecht and Wagner, 2012), IntS12 (5′-GUC AAG ACA UCC ACA GUU A-3′), and CPSF30 (5′-GUG CCU AUA UCU GUG AUU U-3′) were obtained from Sigma-Aldrich (St. Louis, MO).Cells were transfected with siRNA duplexes using DharmaFECT 1 transfection reagent (Dharmacon) and analyzed 3 d posttransfection. FuGENE HD transfection reagent was used for cotransfection of cells with siRNA and DNA constructs. Where indicated, cells were incubated in 5 μg/ml (16.6 μM) nocodazole (Sigma-Aldrich) for 3 h before fixation to enhance perinuclear localization of dynein. For G2/M arrest, cells were incubated for 16 h in 10 μM RO-3306 (Cdk1 inhibitor; Enzo Life Sciences, Plymouth, PA).Cell fixation, immunostaining, and microscopyCells were fixed in methanol (5 min at −20°C followed by washing with Tris-buffered saline [TBS] plus 0.01% Triton X-100) and blocked in TBS plus 0.01% Triton X-100 and 0.02% bovine serum albumin before immunostaining. Primary antibodies were used as follows: DIC (clone 74.1, 1:500; Abcam, Cambridge, MA) and c-Myc (9E10, 1:1000). Alexa Fluor 546–conjugated phalloidin (1:1000; Invitrogen) was used to stain F-actin. Appropriate secondary antibodies conjugated to Alexa Fluor 488 or Cy3 were used (1:1000; Invitrogen). Cells were mounted in ProLong Gold Antifade Reagent with DAPI (Invitrogen). Wide-field fluorescence images were obtained using an Eclipse 80i microscope (Nikon) with Plano-Apo 100× objective.Line scan analyses to quantify perinuclear dynein accumulation in HeLa cells were performed using ImageJ (National Institutes of Health, Bethesda, MD). Ten representative cells were measured per condition; for each cell, 12 line scans distributed equally around the nuclear circumference were obtained. To quantify the ratios of perinuclear to diffusely cytoplasmic DIC in stained HeLa cells, the average intensity of the DIC signal within a small rectangular region was sampled near the nuclear surface and in the surrounding cytoplasm using ImageJ. The ratio of the intensities was determined. At least 20 cells were scored per condition.Statistical analyses of data from cultured cell experiments reported here were performed using Student's unpaired t test. Error bars indicate SEM for all bar graphs. All experiments were performed a minimum of three times with at least 200 cells scored per condition.FACS analysissiRNA-treated HeLa cells (∼107) were fixed with 70% ethanol at 4°C for 24 h and incubated in PBS containing propidium iodide (20 μg/ml) and RNase A (0.2 mg/ml; Sigma-Aldrich) at 4°C for 24 h. FACS analysis was performed to determine propidium iodide intensity levels (a measure of DNA content). Gating was used to exclude cell debris and doublets from the DNA analysis. FACS experiments were performed in the Flow Cytometry Shared Resource of Vanderbilt University Medical Center.DNA constructscDNA clones encoding mASUN and dASUN were previously described (Jodoin et al., 2012). The full-length hASUN open reading frame (ASUN; National Center for Biotechnology Information Reference Sequence, NM_018164 [gene], NP_060634.2 [protein]) was amplified from a human primary skin fibroblast cDNA library. The following forward and reverse primers were used to incorporate EcoRI and StuI restriction sites into the 5′ and 3′ ends, respectively, of the hASUN coding region, followed by subcloning of the digested, purified fragment into expression vector pCS2: 5′-CCG GAA TTC CCA GGC ACG AAA GTT AAA AC-3′ (ASUN-EcoRI-5′) and 5′-AAA AGG CCT TTC TTC AAG TCA CTC TTC ACT GC-3′ (ASUN-StuI-3′).Constructs for expression in HeLa cells of the following N-terminally tagged proteins were generated by subcloning into vector pCS2-Myc: Myc-dASUN, Myc-mASUN (previously described in Jodoin et al. 2012), and Myc-hASUN. To generate pCS2-Myc-NLS vector, the following nucleotide sequence was engineered into the pCS2-Myc vector to add a strong exogenous NLS (PKKKRKV; derived from SV40 large T antigen) C-terminal to the Myc tag: CCC AAG AAG AAG CGC AAG GTC (Kalderon et al., 1984). hASUN and dASUN were subcloned into pCS2-Myc-NLS for production of Myc-NLS-hASUN and Myc-NLS-dASUN, respectively, in transfected cells. We used a PCR-based approach to generate mASUN fragments for subcloning into pCS2-GFP expression vector (N-terminal tag; illustrated in Supplemental Figure S4).Vector tv3 (gift from J. Brill, The Hospital for Sick Children, Toronto, Canada) containing the testes-specific β2-tubulin promoter was used to make constructs for Drosophila transgenesis (Wong et al., 2005). Sequence encoding a strong exogenous NLS (described earlier) was engineered into a previously described tv3-CHY vector to generate tv3-NLS-CHY. dASUN was subcloned into modified tv3 vector to produce NLS-CHY-dASUN in transgenic fly testes.We used NLStradamus software to identify putative NLS motifs in hASUN, mASUN, and dASUN (Nguyen Ba et al., 2009). We used the QuikChange II XL Site-Directed Mutagenesis Kit (Agilent Technologies, Santa Clara, CA) to mutate charged residues to alanines within these motifs of hASUN (Figure 4A), mASUN (Supplemental Figure S4A), and dASUN (Figure 5A and Supplemental Figure S5).GFP-Integrator subunits were subcloned into pcDNA4/myc-His A (Invitrogen) using purchased cDNA templates (Open Biosystems/Thermo Scientific, Huntsville, AL). The GFP-BICD2 expression construct was a gift from A. Akhmanova (Erasmus Medical Center, Rotterdam, Netherlands; Hoogenraad et al., 2001).ImmunoblottingHeLa cell lysates were prepared using nondenaturing lysis buffer (50 mM Tris-Cl, pH 7.4, 300 mM NaCl, 5 mM EDTA, 1% Triton X-100). After SDS–PAGE, proteins were transferred to nitrocellulose for immunoblotting using standard techniques. Immunoblotting was performed using the following primary antibodies: c-Myc (9E10, 1:1000), β-tubulin (clone E7, 1:1000; Developmental Studies Hybridoma Bank, University of Iowa, Iowa City, IA), mCherry (1:500; Clontech, Mountain View, CA), CENP-F (clone 14C10 1D8, 1:500; Abcam), C-hASUN (1:300; Jodoin et al., 2012), GFP B-2 (1:1000; Santa Cruz Biotechnology, Dallas, TX), and IntS1, IntS4, IntS7, IntS9, IntS10, IntS11, IntS12, and CPSF30 (1:1000; Bethyl Labs, Montgomery, TX). Horseradish peroxidase–conjugated secondary antibodies (1:5000) and chemiluminescence were used to detect primary antibodies.

MATERIALS AND METHODS

Drosophila spermatocyte experimentsFlies were maintained at 25°C using standard techniques (Greenspan, 2004). y w flies obtained from Bloomington Drosophila Stock Center (Indiana University, Bloomington, IN) were used as the “wild-type” stock. The asunf02815 allele (Exelixis Collection, Harvard Medical School, Boston, MA) and a transgenic line with male germline–specific expression of CHY-tagged wild-type dASUN were previously described (Anderson et al., 2009). Transgenic lines for male germline–specific expression of CHY-tagged dASUN fusion proteins (constructs described later) were generated by P-element–mediated transformation via embryo injection (Rubin and Spradling, 1982). For each transgene, a single insertion mapping to the X chromosome was crossed into the asunf02815 background using standard genetic crosses.To test male fertility, individual adult males (2 d old) were placed in vials with five wild-type females (2 d old) and allowed to mate for 5 d. The average number of live adult progeny produced per fertile male was scored (≥10 males tested per genotype). Statistical analysis was performed using an unpaired Student's t test.Protein extracts were prepared by homogenizing dissected testes from newly eclosed males in SDS sample buffer. The equivalent of eight testes pairs was loaded per lane. After SDS–PAGE, immunoblotting was performed as described later.Live testes cells were prepared for examination by fluorescence microscopy as described previously (Kemphues et al., 1980). Briefly, testes were dissected from newly eclosed adult males, placed in a drop of phosphate-buffered saline (PBS) on a microscopic slide, and gently squashed under a glass coverslip after making a small incision near the stem cell hub. Formaldehyde fixation was performed as described previously (Gunsalus et al., 1995). Briefly, slides of squashed testes were snap-frozen, immersed in 4% formaldehyde (in PBS with 0.1% Triton X-100) for 7 min at −20°C after coverslip removal, and washed three times in PBS. Mouse anti–dynein heavy chain primary antibody (P1H4, 1:120; gift from T. Hays, University of Minnesota, Minneapolis, MN) and Cy3-conjugated secondary antibody (Invitrogen, Carlsbad, CA) were used (McGrail and Hays, 1997). Fixed samples were mounted in PBS with 4′,6-diamidino-2-phenylindole (DAPI) to visualize DNA. Wide-field fluorescence images were obtained using an Eclipse 80i microscope (Nikon, Melville, NY) with Plan-Fluor 40× objective. In experiments to determine the percentage of late G2 spermatocytes with perinuclear dynein, at least 200 cells were scored per genotype. Statistical analysis was performed using Fisher's exact test.

Drosophila spermatocyte experiments

Flies were maintained at 25°C using standard techniques (Greenspan, 2004). y w flies obtained from Bloomington Drosophila Stock Center (Indiana University, Bloomington, IN) were used as the “wild-type” stock. The asunf02815 allele (Exelixis Collection, Harvard Medical School, Boston, MA) and a transgenic line with male germline–specific expression of CHY-tagged wild-type dASUN were previously described (Anderson et al., 2009). Transgenic lines for male germline–specific expression of CHY-tagged dASUN fusion proteins (constructs described later) were generated by P-element–mediated transformation via embryo injection (Rubin and Spradling, 1982). For each transgene, a single insertion mapping to the X chromosome was crossed into the asunf02815 background using standard genetic crosses.

To test male fertility, individual adult males (2 d old) were placed in vials with five wild-type females (2 d old) and allowed to mate for 5 d. The average number of live adult progeny produced per fertile male was scored (≥10 males tested per genotype). Statistical analysis was performed using an unpaired Student's t test.

Protein extracts were prepared by homogenizing dissected testes from newly eclosed males in SDS sample buffer. The equivalent of eight testes pairs was loaded per lane. After SDS–PAGE, immunoblotting was performed as described later.

Live testes cells were prepared for examination by fluorescence microscopy as described previously (Kemphues et al., 1980). Briefly, testes were dissected from newly eclosed adult males, placed in a drop of phosphate-buffered saline (PBS) on a microscopic slide, and gently squashed under a glass coverslip after making a small incision near the stem cell hub. Formaldehyde fixation was performed as described previously (Gunsalus et al., 1995). Briefly, slides of squashed testes were snap-frozen, immersed in 4% formaldehyde (in PBS with 0.1% Triton X-100) for 7 min at −20°C after coverslip removal, and washed three times in PBS. Mouse anti–dynein heavy chain primary antibody (P1H4, 1:120; gift from T. Hays, University of Minnesota, Minneapolis, MN) and Cy3-conjugated secondary antibody (Invitrogen, Carlsbad, CA) were used (McGrail and Hays, 1997). Fixed samples were mounted in PBS with 4′,6-diamidino-2-phenylindole (DAPI) to visualize DNA. Wide-field fluorescence images were obtained using an Eclipse 80i microscope (Nikon, Melville, NY) with Plan-Fluor 40× objective. In experiments to determine the percentage of late G2 spermatocytes with perinuclear dynein, at least 200 cells were scored per genotype. Statistical analysis was performed using Fisher's exact test.

Cell culture and treatmentsHeLa cells were maintained at 37°C and 5% CO2 in DMEM containing 10% fetal bovine serum, 1% l-glutamine, 100 μg/ml streptomycin, and 100 U/ml penicillin (Life Technologies, Carlsbad, CA). Plasmid DNA was transfected into cells using FuGENE HD (Promega, Madison, WI). siGENOME NT siRNA#5 (Dharmacon, Lafayette, CO) was used as negative control. siRNA used to silence hASUN (3′-UTR region; 5′-CAG CAA GAU GGU AUA GUU A-3′) was obtained from Dharmacon. siRNAs used to silence IntS1 (5′-CAU UUC UCC GUC GAU UAA A-3′), IntS2 (#1: 5′-CUC GUU UAG CUG UCA AUG U-3′ and #2: 5′-CCU UAA UCA GCG UUU CAG U-3′), IntS3 (5′-GAA GUA CUG AGU UCA GAU A-3′), IntS4 (5′-CAG CAU UGU UCU CAG AUC A-3′), IntS5 (5′-CAA GUU UGU CCA GUC ACG A-3′), IntS6 (#1: 5′-CAC UAA UGA UUC GAU AAU A-3′ and #2: 5′-CCA UGA AGA GGU CAA UAC U-3′), IntS7 (5′-GGC UAA AUA GUU UGA AGG A-3′), IntS9 (5′-GAA AUG CUU UCU UGG ACA A-3′), IntS10 (5′-GGA UAC UUG GCU UUG GUU A-3′), IntS11 (Albrecht and Wagner, 2012), IntS12 (5′-GUC AAG ACA UCC ACA GUU A-3′), and CPSF30 (5′-GUG CCU AUA UCU GUG AUU U-3′) were obtained from Sigma-Aldrich (St. Louis, MO).Cells were transfected with siRNA duplexes using DharmaFECT 1 transfection reagent (Dharmacon) and analyzed 3 d posttransfection. FuGENE HD transfection reagent was used for cotransfection of cells with siRNA and DNA constructs. Where indicated, cells were incubated in 5 μg/ml (16.6 μM) nocodazole (Sigma-Aldrich) for 3 h before fixation to enhance perinuclear localization of dynein. For G2/M arrest, cells were incubated for 16 h in 10 μM RO-3306 (Cdk1 inhibitor; Enzo Life Sciences, Plymouth, PA).

Cell culture and treatments

HeLa cells were maintained at 37°C and 5% CO2 in DMEM containing 10% fetal bovine serum, 1% l-glutamine, 100 μg/ml streptomycin, and 100 U/ml penicillin (Life Technologies, Carlsbad, CA). Plasmid DNA was transfected into cells using FuGENE HD (Promega, Madison, WI). siGENOME NT siRNA#5 (Dharmacon, Lafayette, CO) was used as negative control. siRNA used to silence hASUN (3′-UTR region; 5′-CAG CAA GAU GGU AUA GUU A-3′) was obtained from Dharmacon. siRNAs used to silence IntS1 (5′-CAU UUC UCC GUC GAU UAA A-3′), IntS2 (#1: 5′-CUC GUU UAG CUG UCA AUG U-3′ and #2: 5′-CCU UAA UCA GCG UUU CAG U-3′), IntS3 (5′-GAA GUA CUG AGU UCA GAU A-3′), IntS4 (5′-CAG CAU UGU UCU CAG AUC A-3′), IntS5 (5′-CAA GUU UGU CCA GUC ACG A-3′), IntS6 (#1: 5′-CAC UAA UGA UUC GAU AAU A-3′ and #2: 5′-CCA UGA AGA GGU CAA UAC U-3′), IntS7 (5′-GGC UAA AUA GUU UGA AGG A-3′), IntS9 (5′-GAA AUG CUU UCU UGG ACA A-3′), IntS10 (5′-GGA UAC UUG GCU UUG GUU A-3′), IntS11 (Albrecht and Wagner, 2012), IntS12 (5′-GUC AAG ACA UCC ACA GUU A-3′), and CPSF30 (5′-GUG CCU AUA UCU GUG AUU U-3′) were obtained from Sigma-Aldrich (St. Louis, MO).

Cells were transfected with siRNA duplexes using DharmaFECT 1 transfection reagent (Dharmacon) and analyzed 3 d posttransfection. FuGENE HD transfection reagent was used for cotransfection of cells with siRNA and DNA constructs. Where indicated, cells were incubated in 5 μg/ml (16.6 μM) nocodazole (Sigma-Aldrich) for 3 h before fixation to enhance perinuclear localization of dynein. For G2/M arrest, cells were incubated for 16 h in 10 μM RO-3306 (Cdk1 inhibitor; Enzo Life Sciences, Plymouth, PA).

Cell fixation, immunostaining, and microscopyCells were fixed in methanol (5 min at −20°C followed by washing with Tris-buffered saline [TBS] plus 0.01% Triton X-100) and blocked in TBS plus 0.01% Triton X-100 and 0.02% bovine serum albumin before immunostaining. Primary antibodies were used as follows: DIC (clone 74.1, 1:500; Abcam, Cambridge, MA) and c-Myc (9E10, 1:1000). Alexa Fluor 546–conjugated phalloidin (1:1000; Invitrogen) was used to stain F-actin. Appropriate secondary antibodies conjugated to Alexa Fluor 488 or Cy3 were used (1:1000; Invitrogen). Cells were mounted in ProLong Gold Antifade Reagent with DAPI (Invitrogen). Wide-field fluorescence images were obtained using an Eclipse 80i microscope (Nikon) with Plano-Apo 100× objective.Line scan analyses to quantify perinuclear dynein accumulation in HeLa cells were performed using ImageJ (National Institutes of Health, Bethesda, MD). Ten representative cells were measured per condition; for each cell, 12 line scans distributed equally around the nuclear circumference were obtained. To quantify the ratios of perinuclear to diffusely cytoplasmic DIC in stained HeLa cells, the average intensity of the DIC signal within a small rectangular region was sampled near the nuclear surface and in the surrounding cytoplasm using ImageJ. The ratio of the intensities was determined. At least 20 cells were scored per condition.Statistical analyses of data from cultured cell experiments reported here were performed using Student's unpaired t test. Error bars indicate SEM for all bar graphs. All experiments were performed a minimum of three times with at least 200 cells scored per condition.

Cell fixation, immunostaining, and microscopy

Cells were fixed in methanol (5 min at −20°C followed by washing with Tris-buffered saline [TBS] plus 0.01% Triton X-100) and blocked in TBS plus 0.01% Triton X-100 and 0.02% bovine serum albumin before immunostaining. Primary antibodies were used as follows: DIC (clone 74.1, 1:500; Abcam, Cambridge, MA) and c-Myc (9E10, 1:1000). Alexa Fluor 546–conjugated phalloidin (1:1000; Invitrogen) was used to stain F-actin. Appropriate secondary antibodies conjugated to Alexa Fluor 488 or Cy3 were used (1:1000; Invitrogen). Cells were mounted in ProLong Gold Antifade Reagent with DAPI (Invitrogen). Wide-field fluorescence images were obtained using an Eclipse 80i microscope (Nikon) with Plano-Apo 100× objective.

Line scan analyses to quantify perinuclear dynein accumulation in HeLa cells were performed using ImageJ (National Institutes of Health, Bethesda, MD). Ten representative cells were measured per condition; for each cell, 12 line scans distributed equally around the nuclear circumference were obtained. To quantify the ratios of perinuclear to diffusely cytoplasmic DIC in stained HeLa cells, the average intensity of the DIC signal within a small rectangular region was sampled near the nuclear surface and in the surrounding cytoplasm using ImageJ. The ratio of the intensities was determined. At least 20 cells were scored per condition.

Statistical analyses of data from cultured cell experiments reported here were performed using Student's unpaired t test. Error bars indicate SEM for all bar graphs. All experiments were performed a minimum of three times with at least 200 cells scored per condition.

FACS analysissiRNA-treated HeLa cells (∼107) were fixed with 70% ethanol at 4°C for 24 h and incubated in PBS containing propidium iodide (20 μg/ml) and RNase A (0.2 mg/ml; Sigma-Aldrich) at 4°C for 24 h. FACS analysis was performed to determine propidium iodide intensity levels (a measure of DNA content). Gating was used to exclude cell debris and doublets from the DNA analysis. FACS experiments were performed in the Flow Cytometry Shared Resource of Vanderbilt University Medical Center.

FACS analysis

siRNA-treated HeLa cells (∼107) were fixed with 70% ethanol at 4°C for 24 h and incubated in PBS containing propidium iodide (20 μg/ml) and RNase A (0.2 mg/ml; Sigma-Aldrich) at 4°C for 24 h. FACS analysis was performed to determine propidium iodide intensity levels (a measure of DNA content). Gating was used to exclude cell debris and doublets from the DNA analysis. FACS experiments were performed in the Flow Cytometry Shared Resource of Vanderbilt University Medical Center.

DNA constructscDNA clones encoding mASUN and dASUN were previously described (Jodoin et al., 2012). The full-length hASUN open reading frame (ASUN; National Center for Biotechnology Information Reference Sequence, NM_018164 [gene], NP_060634.2 [protein]) was amplified from a human primary skin fibroblast cDNA library. The following forward and reverse primers were used to incorporate EcoRI and StuI restriction sites into the 5′ and 3′ ends, respectively, of the hASUN coding region, followed by subcloning of the digested, purified fragment into expression vector pCS2: 5′-CCG GAA TTC CCA GGC ACG AAA GTT AAA AC-3′ (ASUN-EcoRI-5′) and 5′-AAA AGG CCT TTC TTC AAG TCA CTC TTC ACT GC-3′ (ASUN-StuI-3′).Constructs for expression in HeLa cells of the following N-terminally tagged proteins were generated by subcloning into vector pCS2-Myc: Myc-dASUN, Myc-mASUN (previously described in Jodoin et al. 2012), and Myc-hASUN. To generate pCS2-Myc-NLS vector, the following nucleotide sequence was engineered into the pCS2-Myc vector to add a strong exogenous NLS (PKKKRKV; derived from SV40 large T antigen) C-terminal to the Myc tag: CCC AAG AAG AAG CGC AAG GTC (Kalderon et al., 1984). hASUN and dASUN were subcloned into pCS2-Myc-NLS for production of Myc-NLS-hASUN and Myc-NLS-dASUN, respectively, in transfected cells. We used a PCR-based approach to generate mASUN fragments for subcloning into pCS2-GFP expression vector (N-terminal tag; illustrated in Supplemental Figure S4).Vector tv3 (gift from J. Brill, The Hospital for Sick Children, Toronto, Canada) containing the testes-specific β2-tubulin promoter was used to make constructs for Drosophila transgenesis (Wong et al., 2005). Sequence encoding a strong exogenous NLS (described earlier) was engineered into a previously described tv3-CHY vector to generate tv3-NLS-CHY. dASUN was subcloned into modified tv3 vector to produce NLS-CHY-dASUN in transgenic fly testes.We used NLStradamus software to identify putative NLS motifs in hASUN, mASUN, and dASUN (Nguyen Ba et al., 2009). We used the QuikChange II XL Site-Directed Mutagenesis Kit (Agilent Technologies, Santa Clara, CA) to mutate charged residues to alanines within these motifs of hASUN (Figure 4A), mASUN (Supplemental Figure S4A), and dASUN (Figure 5A and Supplemental Figure S5).GFP-Integrator subunits were subcloned into pcDNA4/myc-His A (Invitrogen) using purchased cDNA templates (Open Biosystems/Thermo Scientific, Huntsville, AL). The GFP-BICD2 expression construct was a gift from A. Akhmanova (Erasmus Medical Center, Rotterdam, Netherlands; Hoogenraad et al., 2001).

DNA constructs

cDNA clones encoding mASUN and dASUN were previously described (Jodoin et al., 2012). The full-length hASUN open reading frame (ASUN; National Center for Biotechnology Information Reference Sequence, NM_018164 [gene], NP_060634.2 [protein]) was amplified from a human primary skin fibroblast cDNA library. The following forward and reverse primers were used to incorporate EcoRI and StuI restriction sites into the 5′ and 3′ ends, respectively, of the hASUN coding region, followed by subcloning of the digested, purified fragment into expression vector pCS2: 5′-CCG GAA TTC CCA GGC ACG AAA GTT AAA AC-3′ (ASUN-EcoRI-5′) and 5′-AAA AGG CCT TTC TTC AAG TCA CTC TTC ACT GC-3′ (ASUN-StuI-3′).

Constructs for expression in HeLa cells of the following N-terminally tagged proteins were generated by subcloning into vector pCS2-Myc: Myc-dASUN, Myc-mASUN (previously described in Jodoin et al. 2012), and Myc-hASUN. To generate pCS2-Myc-NLS vector, the following nucleotide sequence was engineered into the pCS2-Myc vector to add a strong exogenous NLS (PKKKRKV; derived from SV40 large T antigen) C-terminal to the Myc tag: CCC AAG AAG AAG CGC AAG GTC (Kalderon et al., 1984). hASUN and dASUN were subcloned into pCS2-Myc-NLS for production of Myc-NLS-hASUN and Myc-NLS-dASUN, respectively, in transfected cells. We used a PCR-based approach to generate mASUN fragments for subcloning into pCS2-GFP expression vector (N-terminal tag; illustrated in Supplemental Figure S4).

Vector tv3 (gift from J. Brill, The Hospital for Sick Children, Toronto, Canada) containing the testes-specific β2-tubulin promoter was used to make constructs for Drosophila transgenesis (Wong et al., 2005). Sequence encoding a strong exogenous NLS (described earlier) was engineered into a previously described tv3-CHY vector to generate tv3-NLS-CHY. dASUN was subcloned into modified tv3 vector to produce NLS-CHY-dASUN in transgenic fly testes.

We used NLStradamus software to identify putative NLS motifs in hASUN, mASUN, and dASUN (Nguyen Ba et al., 2009). We used the QuikChange II XL Site-Directed Mutagenesis Kit (Agilent Technologies, Santa Clara, CA) to mutate charged residues to alanines within these motifs of hASUN (Figure 4A), mASUN (Supplemental Figure S4A), and dASUN (Figure 5A and Supplemental Figure S5).

GFP-Integrator subunits were subcloned into pcDNA4/myc-His A (Invitrogen) using purchased cDNA templates (Open Biosystems/Thermo Scientific, Huntsville, AL). The GFP-BICD2 expression construct was a gift from A. Akhmanova (Erasmus Medical Center, Rotterdam, Netherlands; Hoogenraad et al., 2001).

ImmunoblottingHeLa cell lysates were prepared using nondenaturing lysis buffer (50 mM Tris-Cl, pH 7.4, 300 mM NaCl, 5 mM EDTA, 1% Triton X-100). After SDS–PAGE, proteins were transferred to nitrocellulose for immunoblotting using standard techniques. Immunoblotting was performed using the following primary antibodies: c-Myc (9E10, 1:1000), β-tubulin (clone E7, 1:1000; Developmental Studies Hybridoma Bank, University of Iowa, Iowa City, IA), mCherry (1:500; Clontech, Mountain View, CA), CENP-F (clone 14C10 1D8, 1:500; Abcam), C-hASUN (1:300; Jodoin et al., 2012), GFP B-2 (1:1000; Santa Cruz Biotechnology, Dallas, TX), and IntS1, IntS4, IntS7, IntS9, IntS10, IntS11, IntS12, and CPSF30 (1:1000; Bethyl Labs, Montgomery, TX). Horseradish peroxidase–conjugated secondary antibodies (1:5000) and chemiluminescence were used to detect primary antibodies.

Immunoblotting

HeLa cell lysates were prepared using nondenaturing lysis buffer (50 mM Tris-Cl, pH 7.4, 300 mM NaCl, 5 mM EDTA, 1% Triton X-100). After SDS–PAGE, proteins were transferred to nitrocellulose for immunoblotting using standard techniques. Immunoblotting was performed using the following primary antibodies: c-Myc (9E10, 1:1000), β-tubulin (clone E7, 1:1000; Developmental Studies Hybridoma Bank, University of Iowa, Iowa City, IA), mCherry (1:500; Clontech, Mountain View, CA), CENP-F (clone 14C10 1D8, 1:500; Abcam), C-hASUN (1:300; Jodoin et al., 2012), GFP B-2 (1:1000; Santa Cruz Biotechnology, Dallas, TX), and IntS1, IntS4, IntS7, IntS9, IntS10, IntS11, IntS12, and CPSF30 (1:1000; Bethyl Labs, Montgomery, TX). Horseradish peroxidase–conjugated secondary antibodies (1:5000) and chemiluminescence were used to detect primary antibodies.

Supplementary MaterialSupplemental Materials

Supplementary Material

Supplemental Materials
