---
pmid: '19498464'
title: The HP1alpha-CAF1-SetDB1-containing complex provides H3K9me1 for Suv39-mediated
  K9me3 in pericentric heterochromatin.
authors:
- Loyola A
- Tagami H
- Bonaldi T
- Roche D
- Quivy JP
- Imhof A
- Nakatani Y
- Dent SY
- Almouzni G
journal: EMBO Rep
year: '2009'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC2727428
doi: 10.1038/embor.2009.90
---

# The HP1alpha-CAF1-SetDB1-containing complex provides H3K9me1 for Suv39-mediated K9me3 in pericentric heterochromatin.
**Authors:** Loyola A, Tagami H, Bonaldi T, Roche D, Quivy JP, Imhof A, Nakatani Y, Dent SY, Almouzni G
**Journal:** EMBO Rep (2009)
**DOI:** [10.1038/embor.2009.90](https://doi.org/10.1038/embor.2009.90)
**PMC:** [PMC2727428](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2727428/)

## Abstract

1. EMBO Rep. 2009 Jul;10(7):769-75. doi: 10.1038/embor.2009.90. Epub 2009 Jun 5.

The HP1alpha-CAF1-SetDB1-containing complex provides H3K9me1 for Suv39-mediated 
K9me3 in pericentric heterochromatin.

Loyola A(1), Tagami H, Bonaldi T, Roche D, Quivy JP, Imhof A, Nakatani Y, Dent 
SY, Almouzni G.

Author information:
(1)Fundación Ciencia para la Vida and Universidad San Sebastián, Santiago, 
Chile.

Trimethylation of lysine 9 in histone H3 (H3K9me3) enrichment is a 
characteristic of pericentric heterochromatin. The hypothesis of a stepwise 
mechanism to establish and maintain this mark during DNA replication suggests 
that newly synthesized histone H3 goes through an intermediate methylation state 
to become a substrate for the histone methyltransferase Suppressor of 
variegation 39 (Suv39H1/H2). How this intermediate methylation state is achieved 
and how it is targeted to the correct place at the right time is not yet known. 
Here, we show that the histone H3K9 methyltransferase SetDB1 associates with the 
specific heterochromatin protein 1alpha (HP1alpha)-chromatin assembly factor 1 
(CAF1) chaperone complex. This complex monomethylates K9 on non-nucleosomal 
histone H3. Therefore, the heterochromatic HP1alpha-CAF1-SetDB1 complex probably 
provides H3K9me1 for subsequent trimethylation by Suv39H1/H2 in pericentric 
regions. The connection of CAF1 with DNA replication, HP1alpha with 
heterochromatin formation and SetDB1 for H3K9me1 suggests a highly coordinated 
mechanism to ensure the propagation of H3K9me3 in pericentric heterochromatin 
during DNA replication.

DOI: 10.1038/embor.2009.90
PMCID: PMC2727428
PMID: 19498464 [Indexed for MEDLINE]

Conflict of interest statement: The authors declare that they have no conflict 
of interest.

## Full Text

Abstract

Trimethylation of lysine 9 in histone H3 (H3K9me3) enrichment is a characteristic of pericentric heterochromatin. The hypothesis of a stepwise mechanism to establish and maintain this mark during DNA replication suggests that newly synthesized histone H3 goes through an intermediate methylation state to become a substrate for the histone methyltransferase Suppressor of variegation 39 (Suv39H1/H2). How this intermediate methylation state is achieved and how it is targeted to the correct place at the right time is not yet known. Here, we show that the histone H3K9 methyltransferase SetDB1 associates with the specific heterochromatin protein 1α (HP1α)–chromatin assembly factor 1 (CAF1) chaperone complex. This complex monomethylates K9 on non-nucleosomal histone H3. Therefore, the heterochromatic HP1α–CAF1–SetDB1 complex probably provides H3K9me1 for subsequent trimethylation by Suv39H1/H2 in pericentric regions. The connection of CAF1 with DNA replication, HP1α with heterochromatin formation and SetDB1 for H3K9me1 suggests a highly coordinated mechanism to ensure the propagation of H3K9me3 in pericentric heterochromatin during DNA replication.

Introduction

Histone proteins are heavily modified post-translationally, and patterns of these modifications mark specific functional nuclear domains. For example, trimethylation of lysine 9 in histone H3 (H3K9me3) is highly enriched in pericentric heterochromatin, the constitutive heterochromatin domain that flanks centromeres. In addition to H3K9me3, this domain is enriched in H4K20me3, hypoacetylated histones, DNA methylation, heterochromatin protein 1α (HP1α) and HP1β, and an RNA component ( Maison & Almouzni, 2004 ; Grewal & Jia, 2007 ). A current model for the propagation of H3K9 methylation and HP1 binding in this region through cell division proposes a self-sustaining loop in which HP1 binds to H3K9me3 through its chromodomain, which, in turn, recruits more of the H3K9–histone methyltransferase (HMTase) Suv39H1/H2. However, in this scheme, some level of H3K9me3 is necessary to nucleate heterochromatin formation. In addition, it is unclear whether Suv39H1/H2 is responsible for only H3K9me3 or whether it affects other intermediate K9 methylation states.

Interestingly, in Suv39H1/H2 double null cells, accumulation of both H3K9me3 and HP1α in pericentric heterochromatin is lost with a concomitant enrichment of H3K9me1 ( Peters et al , 2001 ; Loyola et al , 2006 ). These results indicate that Suv39H1/H2 is not involved in the monomethylation of H3K9 in these regions and instead suggest that H3K9me1 might be used as a substrate for trimethylation. In support of this view, transfection of the Suv39H1/H2 double null cells with recombinant wild-type Suv39H1/H2 is sufficient to restore the transition from H3K9me1 towards H3K9me3 ( Loyola et al , 2006 ). Moreover, in vitro methylation assays show that H3K9me1 is an optimal substrate for Suv39H1/H2-mediated trimethylation of H3K9 ( Loyola et al , 2006 ). Therefore, the question that arises is how H3K9me1 is directed to pericentric heterochromatin.

The above self-sustaining model for the inheritance of heterochromatin marks would work at any time during the cell cycle, assuming that some level of H3K9me3 is available for the nucleation of HP1 recruitment. However, a particular situation arises during DNA replication in which nucleosome disruption occurs ahead of the replication fork and newly synthesized plus recycled histones are deposited behind the fork onto newly synthesized DNA ( Groth et al , 2007a ). Parental H3K9me3 might provide nucleation events to maintain HP1 and Suv39H1/H2 in the pericentric domain; however, a replication-coupled mechanism of H3K9 methylation might be needed to promote the deposition of this histone mark onto newly synthesized histones at the time of incorporation into chromatin.

One attractive model is that the chromatin assembly factor CAF1—which functions in a manner coupled with DNA synthesis and is present at all replication foci—might promote the loading of H3.1K9me1 and HP1α in heterochromatin, thereby facilitating the subsequent trimethylation of H3.1K9me1 by Suv39H1/H2 and HP1α binding. Indeed, CAF1 might act as a platform to integrate and coordinate a series of activities necessary for the inheritance of both H3K9 and DNA methylation. CAF1 interacts with the methyl-CpG-binding domain protein MBD1 and the H3K9–HMTase SetDB1 during S phase ( Sarraf & Stancheva, 2004 ). However, how SetDB1 functions in conjunction with Suv39H1/H2 to promote H3K9me3 at pericentric heterochromatin during DNA replication is not known. Furthermore, no studies have yet addressed how these two lysine methyltransferases (KMTs) contribute to H3K9 mono-, di- and trimethylated states to ultimately establish the characteristic H3K9me3 pattern found in pericentric heterochromatin.

Here, we report that the HP1α–CAF1 complex associates with SetDB1, and that this complex monomethylates non-nucleosomal H3K9. Furthermore, we show that SetDB1 localizes to pericentric heterochromatin, correlating with the replication of these sites. On the basis of these results, we propose a model for how the CAF1 chromatin assembly pathway coupled with DNA replication affects the establishment of H3K9 methylation patterns.

Methods

Complex purification. We grew Flag–HA-tagged HP1α- and H3.1-expressing HeLa S3 cell lines as described previously ( Quivy et al , 2004 ; Tagami et al , 2004 ). We purified H3.1, HP1α–CAF1 and p150–CAF1 complexes from nuclear extracts by immunoprecipitation using anti-Flag antibody conjugated agarose beads.

Histone methyltransferase assay. We performed the assays as described by Eskeland et al (2004 ), using either 4 μg of recombinant histones or nucleosomes, or 1 μg of histone H3 peptides containing the first 19 amino acids either unmodified or trimethylated at K9.

MALDI-mass spectrometry. We acquired and analysed matrix-assisted laser desorption/ionization (MALDI) spectra as described previously ( Bonaldi et al , 2004 ; Loyola et al , 2006 ).

Immunofluorescence. We performed immunofluorescence after extraction with Triton-X100 to remove soluble proteins on mouse 3T3 cells as described previously ( Taddei et al , 2001 ; Maison et al , 2002 ).

SetDB1 siRNA. The assay is shown in Fig 2C . In brief, we used two distinct SetDB1 siRNAs, 50 nM of SetDB1 siRNA duplex (Silencer ® Pre-designed siRNA, Ambion, Austin, TX, USA; #1: ID: 138242, sequence 5′-3′: GGGCAGUGACUAAUUGUGAtt, #2: ID 3897, sequence 5′-3′: GGGUGUUUUCAUUAACACAtt), to ensure that our observations were not due to off-target effects. As a negative control, we used 50 nM of negative control siRNA (Silencer ® Negative Control #1 siRNA, Ambion), a sequence designed without any significant sequence similarity with mouse, rat or human transcripts sequences. Two rounds of transfections were necessary to effectively knock down SetDB1 present in both the soluble fraction and in the HP1α–CAF1 complex.

Supplementary information is available at EMBO reports online ( http://www.emboreports.org ).
