---
pmid: '27462815'
title: eIF3d is an mRNA cap-binding protein that is required for specialized translation
  initiation.
authors:
- Lee AS
- Kranzusch PJ
- Doudna JA
- Cate JH
journal: Nature
year: '2016'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC5003174
doi: 10.1038/nature18954
---

# eIF3d is an mRNA cap-binding protein that is required for specialized translation initiation.
**Authors:** Lee AS, Kranzusch PJ, Doudna JA, Cate JH
**Journal:** Nature (2016)
**DOI:** [10.1038/nature18954](https://doi.org/10.1038/nature18954)
**PMC:** [PMC5003174](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5003174/)

## Abstract

1. Nature. 2016 Aug 4;536(7614):96-9. doi: 10.1038/nature18954. Epub 2016 Jul 27.

eIF3d is an mRNA cap-binding protein that is required for specialized 
translation initiation.

Lee AS, Kranzusch PJ, Doudna JA, Cate JH.

Eukaryotic mRNAs contain a 5′ cap structure that is crucial for recruitment of 
the translation machinery and initiation of protein synthesis. mRNA recognition 
is thought to require direct interactions between eukaryotic initiation factor 
4E (eIF4E) and the mRNA cap. However, translation of numerous capped mRNAs 
remains robust during cellular stress, early development, and cell cycle 
progression despite inactivation of eIF4E. Here we describe a cap-dependent 
pathway of translation initiation in human cells that relies on a previously 
unknown cap-binding activity of eIF3d, a subunit of the 800-kilodalton eIF3 
complex. A 1.4 Å crystal structure of the eIF3d cap-binding domain reveals 
unexpected homology to endonucleases involved in RNA turnover, and allows 
modelling of cap recognition by eIF3d. eIF3d makes specific contacts with the 
cap, as exemplified by cap analogue competition, and these interactions are 
essential for assembly of translation initiation complexes on eIF3-specialized 
mRNAs such as the cell proliferation regulator c-Jun (also known as JUN). The 
c-Jun mRNA further encodes an inhibitory RNA element that blocks eIF4E 
recruitment, thus enforcing alternative cap recognition by eIF3d. Our results 
reveal a mechanism of cap-dependent translation that is independent of eIF4E, 
and illustrate how modular RNA elements work together to direct specialized 
forms of translation initiation.

DOI: 10.1038/nature18954
PMCID: PMC5003174
PMID: 27462815 [Indexed for MEDLINE]

Conflict of interest statement: The authors declare no competing financial 
interests.

## Full Text

The rate-limiting step of translation initiation is the recognition of the 5' cap structure by eIF4E3,4. eIF4E activity is highly regulated by extracellular stimuli, predominantly through steric hindrance by the eIF4E-binding proteins (4E-BPs)5,6. The translational efficiencies of mRNAs range in sensitivity to 4E-BP inhibition7-9, and these differences have traditionally been addressed by categorizing translation into cap-dependent versus cap-independent pathways10. However, the mechanisms underlying mRNA sensitivity to active eIF4E levels remain enigmatic as all cellular mRNAs maintain the same 5' cap structure11.

Recently we discovered a new translation pathway driven by RNA interactions with eIF3 that is utilized by a subset of cell proliferation mRNAs, with the prototype member being the mRNA encoding the early response transcription factor c-Jun2. eIF3-specialized translation is cap-dependent and requires recruitment of eIF3 to an internal stem loop structure in the 5' untranslated region (UTR). However, the translational efficiency of a subset of these mRNAs is unaffected by eIF4E inactivation7-9, suggesting that cap recognition may proceed by a non-canonical mechanism (Supplementary Table 1).

To understand how cap recognition occurs during eIF3-specialized translation, we examined whether c-Jun mRNA utilizes the canonical eIF4F cap-binding complex during initiation. We programmed in vitro translation extracts from human 293T cells with capped and polyadenylated c-Jun mRNA, and isolated the 48S complex to assess the presence of the eIF4F factors (eIF4G1, eIF4A1, eIF4E) (Fig. 1a,b). Unexpectedly, although c-Jun mRNA translation initiation complexes contain eIF3 and the small ribosomal subunit, they are depleted of all eIF4F components. In contrast, eIF4F is readily detectable in 48S initiation complexes formed on a canonical eIF4E-dependent mRNA, ACTB (Fig. 1b)12. In agreement with the absence of eIF4F, c-Jun levels are unaffected by cell treatment with the mTOR inhibitor INK1287, which inactivates eIF4E, and eIF4A inhibitors13 (Extended Data Fig. 1). These results indicate that c-Jun mRNA translation occurs independently of eIF4F and that the process of eIF3-specialized translation is fundamentally distinct at the initial stage of 5' cap recognition.

eIF3-specialized translation requires recognition of an internal RNA stem loop for efficient translation2. Therefore, we asked if eIF3 might also be involved in 5' cap recognition. In agreement with the previously demonstrated RNA-binding capability of eIF3, the four eIF3 RNA-binding subunits, eIF3a, b, d, and g, provide RNase protection to internally 32P-labeled c-Jun 5' UTR RNA upon UV254-induced crosslinking (Fig. 1c)2. In contrast, when the 32P label is placed in the 5' cap of c-Jun mRNA, RNase protection is observed with a single subunit of eIF3, corresponding to eIF3d (Fig. 1c, Extended Data Fig. 2a). We confirmed subunit identity by limited proteolysis and mass spectrometry, and defined a C-terminal region of eIF3d that is responsible for protection of the 5' mRNA terminus (Extended Data Fig. 2). The mapped C-terminal region of eIF3d is broadly conserved throughout plant, fungal and animal phylogeny (Fig. 2a, Extended Data Fig. 3), suggesting the apparent 5' end recognition activity of eIF3d is an evolutionarily preserved function of the eIF3 complex.

To understand how eIF3d recognizes the 5' RNA terminus, we determined a 1.4 Å crystal structure of the conserved C-terminal domain of eIF3d from Nasonia vitripennis (65% identical, 84% similar to human eIF3d) using sulfur anomalous dispersion for phase determination (Extended Data Table 1, Extended Data Fig. 3). The structure of eIF3d reveals a complex fold that forms a cup-shaped architecture with a positively charged central tunnel that is negatively charged at its base (Fig. 2b). Remarkably, in spite of no significant sequence homology, the structural topology of eIF3d is nearly identical to the DXO proteins, a recently described family of 5' cap-endonucleases involved in RNA quality control14-16 (Fig. 2c, Extended Data Fig. 4). In contrast to DXO, eIF3d contains a unique insertion of ~15 highly conserved amino acids between strand β5 and helix α6. The eIF3d-specific insertion folds down along the front face of the domain, making loosely packed charged interactions that close off the RNA binding tunnel (Extended Data Fig. 5). We term this insertion an “RNA gate,” as the sequence clashes with the path of single-stranded RNA (ssRNA) bound to DXO15 and must undergo a conformational change for eIF3d to become competent for RNA recognition (Fig. 2d). We determined the structure of eIF3d in two additional crystal forms, and confirmed the RNA gate exhibits a closed conformation regardless of crystal packing (Extended Data Fig. 6). As eIF3d does not bind all capped RNAs17,18 we postulate that the RNA gate regulates cap recognition to prevent promiscuous mRNA binding prior to assembly of eIF3d into the full eIF3 complex. We tested this model using c-Jun mRNA and verified eIF3d cap-recognition only occurs in the context of a full eIF3 complex and requires prior eIF3-sequence specific RNA interactions with the eIF3-recruitment stem loop (Extended Data Fig. 7). Allosteric communication between eIF3 subunits during initial RNA recruitment likely facilitates eIF3d RNA gate opening to allow 5' end recognition. The structure of eIF3d therefore reveals a new cap-binding protein and explains the ability of the eIF3 complex to protect the 5' end of mRNA (Fig. 1c).

To validate the structural finding that eIF3d is a cap-binding protein, we examined the ability of eIF3 to bind the c-Jun mRNA 5' cap in the presence of competitor ligands. eIF3d-cap recognition is sensitive to m7GDP competition but resistant to GDP, indicating that, analogous to eIF4E-cap cap binding4, eIF3d specifically interacts with the 5' cap and requires a mature methylated cap structure for recognition (Fig. 3a). Using the DXO–RNA structure as a template15, we modeled a capped ssRNA along the basic binding groove shared between eIF3d and DXO and identified two conserved helices (α5 and α11) likely to be involved in cap recognition (Fig. 3b). We purified recombinant eIF3 containing helix α5 or α11-mutated eIF3d and demonstrated that both mutants have drastically reduced ability to crosslink to the c-Jun mRNA cap (Fig. 3c). eIF3d-mutated complexes retain wild type-levels of RNA-binding, indicating that these residues specifically coordinate 5' mRNA cap recognition (Extended Data Fig. 8). We next introduced HA epitope-tagged wild type or mutant eIF3d into 293T cells, and measured assembly of 48S initiation complexes on c-Jun mRNA by quantitative RT-PCR19,20. Mutations to the predicted eIF3d cap-binding surface dramatically reduce c-Jun mRNA incorporation into translation complexes, while the control ACTB mRNA is unaffected (Fig. 3d; Extended Data Fig. 8). These results demonstrate that cap binding by eIF3d is required for efficient initiation complex formation during eIF3-specialized translation.

eIF3d recognition of the 5' cap structure provides an alternative cap-dependent translation mechanism from canonical eIF4F cap recognition. Perplexingly, when the RNA stem loop element that recruits eIF3 to the c-Jun mRNA is deleted, translation is inhibited even though the mRNA contains a 5' cap2. We hypothesized that an RNA element within the c-Jun mRNA blocks recruitment of the eIF4F complex. In support, the 5' cap of c-Jun mRNA crosslinks less efficiently to purified eIF4E than that of the ACTB mRNA (Extended Data Fig. 9). To identify the eIF4F inhibitory element, we constructed luciferase reporters to test deletions in the c-Jun 5' UTR (Fig. 4a). Deletion of the 5' 153 nucleotides, but not the initial 67 nucleotides, was sufficient to allow c-Jun mRNA translation to occur independently of the eIF3-recruitment stem loop, suggesting that canonical cap dependent translation is no longer blocked (Fig. 4b). We confirmed by western blot analysis of the 48S initiation complex formed on c-Jun mRNA lacking the 5' 153 nucleotides that the eIF4F components are now present (Fig. 4c).

Together, we put forth a model of a previously undiscovered cap-dependent translation initiation pathway controlled by eIF3d recognition of the 5' mRNA cap (Fig. 4d). We postulate that encoding more than one mechanism of cap-dependent translation allows cells to specifically control protein synthesis in cellular environments where eIF4E is inactivated. In support, c-Jun mRNA translation is resistant to treatment of cells with chemicals that activate the 4E-BPs7,8,21 (Extended Data Fig. 1). As modulation of eIF4E cap-binding activity allows cells to incorporate extracellular stimuli into altered translation outputs22, it will be important to discover whether eIF3d activity is analogously regulated. Furthermore, our data indicates that the c-Jun mRNA encodes an additional cis-acting RNA element that blocks eIF4F to ensure translation can only occur through an eIF3-specialized pathway. RNA elements that block eIF4F recruitment may be a common theme to direct mRNAs into specific translation pathways to ensure controlled protein expression. For example, a subset of homeobox mRNAs contain an RNA element that blocks cap-dependent translation to ensure usage of an internal ribosome entry site and to allow for correct homeobox expression during embryonic development23. A number of eIF3-specialized mRNAs encode proteins involved in the control of cell proliferation, suggesting their translation may also require enhanced regulation2,24.

While considerable advances have been made in the structural understanding of eIF3 bound to the ribosome, direct localization of eIF3d in a 48S complex remains unclear25. Thus, understanding how eIF3d functions and assembles within the full translation initiation complex will have significant mechanistic implications in how cap recognition links to mRNA ribosomal recruitment. Our discovery of eIF3d as a cap-binding protein now reveals a new translation pathway independent of eIF4E, and adds another layer of cap-dependent translation.

MethodsCells and transfectionsHuman 293T cells were maintained in DMEM (Invitrogen) supplemented with 10% FBS (Seradigm). The cells were obtained from the University of California, Berkeley, Cell Culture Facility, which authenticates cells by STR profiling and tests for mycoplasma contamination. Plasmid transfections were performed using Lipofectamine 2000 (Invitrogen), following the manufacturer’s protocol, and polysome or immunoprecipitation analyses were performed at 48 h after transfection. For INK128 (Cayman Chemical) cell treatment, 293T cells were incubated with the indicated concentration of INK128 for ~14–16 h before cell lysis.PlasmidsTo generate the eIF3d expression plasmids, eIF3d was amplified from human cDNA and inserted into pcDNA5/FRT. A 39-nucleotide linker followed by the HA epitope tag (YPYDVPDYA) was subsequently inserted prior to the eIF3d stop codon. The wild type c-Jun 5′ UTR luciferase reporter plasmid was previously described2. To generate the c-Jun in vitro transcription template, the 5′ UTR, ORF, and 3′ UTR were separately amplified from human cDNA and stitched together downstream of a T7 promoter by Gibson cloning into pcDNA4. The ACTB in vitro transcription template was constructed by addition of a T7 promoter during amplification of the full mRNA from human cDNA and inserted into pcDNA4.Western blotWestern blot analyses were performed using the following antibodies: anti-eIF3d (Bethyl A301-758A), anti-eIF4A1 (Cell Signaling 2490), anti-eIF4G1 (Cell Signaling 2858), anti-rpS19 (Bethyl A304-002A), anti-eIF4E (Bethyl A301-154A), anti-rpLP0 (Bethyl A302-882A), anti-HA epitope tag (Pierce 26183), anti-c-Jun (Cell Signaling 9165), anti-Hsp90 (BD Biosciences 610418), and anti-4EBP1 (Cell Signaling 9644).In vitro RNA transcription and labelingUnlabeled RNAs were in vitro transcribed, polyadenylated, and capped as previously described2. For internal radiolabeling of RNAs, in vitro transcription was performed in the presence of 0.1 µM [α-32P]-ATP, then the RNA was subsequently capped with vaccinia virus enzymes (NEB). For radiolabeling of the 5′ cap, in vitro transcribed RNAs were capped with vaccinia virus enzymes and [α-32P]-GTP. RNAs were purified by phenol-chloroform extraction and ethanol precipitation.In vitro translationIn vitro translation extracts were made from human 293T cells as previously described21. Lysates were nuclease-treated with 18 gel units μl-1 micrococcal nuclease (NEB) in the presence of 0.7 mM CaCl2 for 10 min at 25 °C, and the digestion was stopped by addition of 2.24 mM EGTA. Each translation reaction contained 50% in vitro translation lysate and buffer to make the final reaction with 0.84 mM ATP, 0.21 mM GTP, 21 mM creatine phosphate (Roche), 45 U ml-1 creatine phosphokinase (Roche), 10 mM HEPES-KOH pH 7.6, 2 mM DTT, 8 mM amino acids (Promega), 255 mM spermidine, 1 U ml-1 murine RNase inhibitor (NEB), and mRNA-specific concentrations of Mg(OAc)2 and KOAc. The optimal magnesium and potassium levels to add were determined to be 1.5 mM Mg(OAc)2 and 150 mM KOAc for c-Jun mRNA, and 1 mM and 150 mM KOAc for ACTB mRNA. For luciferase assays, translation reactions were incubated for 1 h at 30 °C, then luciferase activity was assayed.48S initiation complex purificationFor 48S initiation complex purification from in vitro translation reactions, 180 μl reactions were incubated for 20 min at 30 °C and centrifuged for 6 min at 12,000g at 4 °C. Lysates were purified by size exclusion chromatography through a 1 ml column packed with Sephacryl S-400 gel filtration resin (GE Healthcare) and the elutant was centrifuged through a 10-25% w/v sucrose gradient by centrifugation for 3.5 h at 38,000 rpm at 4 °C in a Beckman SW41 Ti rotor27. Fractions were collected from the top of the gradient using a peristaltic pump with a Brandel tube piercer. From the appropriate fractions, RNA was purified by phenol-chloroform extraction and ethanol precipitation and protein was precipitated with trichloroacetic acid.For affinity purification of HA epitope-tagged eIF3d-associated 48S initiation complexes from cells, three 10 cm plates of transfected 293T cells were treated with 100 μg ml-1 cycloheximide for 5 min. Cells were washed with ice cold PBS (137 mM NaCl, 2.7 mM KCl, 100 mM Na2HPO4, 2 mM KH2PO4) with 100 μg ml-1 cycloheximide and collected in lysis buffer (20 mM HEPES-KOH pH 7.4, 150 mM KOAc, 2.5 mM Mg(OAc)2, 1 mM DTT, 100 μg ml-1 cycloheximide, 1% v/v Triton X-100). Lysates were centrifuged for 6 min at 12000g at 4 °C and purified by S-400 size exclusion chromatography. 80 μl of anti-HA antibody-conjugated agarose beads (Sigma) were added to the elutants, tumbled for 1.5 h at 4 °C, and beads were washed three times with lysis buffer without Triton X-100. Bound complexes were eluted twice with 100 μg ml-1 HA peptide and elutants were centrifuged and analyzed the same as for the in vitro purification reactions.Quantitative real-time PCRcDNA was reverse-transcribed from RNA using random hexamers and Superscript III (Invitrogen), following the manufacturer’s protocol. Real-time PCR was performed using DyNAmo HS Sybr Green (ThermoFisher), with a 20 μl reaction volume containing 2 μl cDNA and 0.5 μM of each primer. The following oligonucleotides were used: 18S rRNA-Forward, 5′ – GGCCCTGTAATTGGAATGAGTC–3′; 18S rRNA-Reverse, 5′ –CCAAGATCCAACTACGAGCTT–3′; ACTB-Forward, 5′ – CTCTTCCAGCCTTCCTTCCT–3′; ACTB-Reverse, 5′ -AGCACTGTGTTGGCGTACAG-3′; c-Jun-Forward, 5′–TGACTGCAAAGATGGAAACG–3′; c-Jun-Reverse; 5′–CAGGGTCATGCTCTGTTTCA–3′.eIF3-RNA crosslinking and gel shiftRecombinant eIF3 was expressed and purified from E. coli as previously described26. For each crosslinking reaction, 1 μl water, 1 μl 125 nM labeled RNA, 1 μl 10 μg ml-1 heparan sulfate (Sigma), 1 μl 5× binding buffer (125 mM Tris-HCl pH 7.5, 25mM Mg(OAc)2, 350 mM KCl, 0.5 mM CaCl2, 0.5 mg ml-1 BSA, 10 mM TCEP), and 1 μl 1.5 μM purified eIF3 were added, in the listed order, and incubated for 30 min at 25 °C. For competition experiments, the water was substituted with 1 μl of 1 mM m7GDP/Mg2+ or GDP/Mg2+. UV254-induced crosslinking was performed using a short-wave UV lamp placed ~4 cm above the samples on ice for 10 min. After treatment with RNase for 10 min at 37 °C, proteins were separated by 12% SDS-PAGE, the gel was dried, and imaged using a phosphorimager. For digestion of internal labeled RNA, 2.5 U benzonase (Novagen) and 250 U RNase T1 (ThermoFisher) were used; for digestion of cap labeled RNA, 4 U RNase R (Epicentre) and 1 U RiboShredder (Epicentre) were used. For eIF3d subunit identification, after RNase treatment, samples were denatured and immunoprecipitation was performed as previously described2. For limited proteolysis, after RNase treatment, the reactions were treated with 2 or 20 μg ml-1 sequencing grade trypsin (Promega) for 30 min at 25 °C, prior to gel electrophoresis. Mass spectrometry samples were prepared as previously described2. Gel shift assays were performed as previously described, using 50 nM labeled c-Jun stem loop RNA and 300 nM purified eIF32.Recombinant eIF3d protein purificationCandidate eIF3d cap-binding domain fragments were amplified by PCR and cloned into a modified pET vector to express an N-terminal 6× His (KSSHHHHHHGSS)-MBP-TEV fusion protein as previously described28. Extensive expression trials were conducted to determine optimal amino- and carboxy-terminal domain boundaries and identified a minimal stable human cap-binding domain S161–F527. Recombinant protein was expressed in BL21-RIL DE3 E. coli cells co-transformed with a pRARE2 tRNA plasmid (Agilent). E. coli was grown in 2×YT media at 37 °C to an OD600 of ~0.5, cooled at 4 °C for 15 min, induced with addition of 0.5 mM IPTG and then incubated with shaking for ~20 h at 16 °C. Pelleted cells were washed with PBS and then lysed by sonication in lysis buffer (20 mM HEPES-KOH pH 7.5, 400 mM NaCl, 10% glycerol, 30 mM imidazole, 1 mM TCEP) in the presence of EDTA-free Complete Protease Inhibitor (Roche). Following centrifugation for 30 min at 23,000g and 4 °C, clarified lysate was incubated with Ni-NTA agarose resin (QIAGEN) for 1 h at 4 °C with gentle rocking. Resin was washed with lysis buffer supplemented to 1 M NaCl and eluted by gravity-flow chromatography at 4 °C with lysis buffer supplemented to 300 mM imidazole. The eluted fraction was diluted to ~50 mM imidazole and 5% glycerol prior, concentrated to ~50 mg ml-1 and incubated with Tobacco Etch Virus protease for ~12 h at 4 °C to remove the MBP tag. Recombinant eIF3d was isolated from free MBP by diluting with gel-filtration buffer (20 mM HEPES-KOH pH 7.5, 250 mM KCl, 1 mM TCEP) and passing over a 5 ml Ni-NTA column (QIAGEN) connected in line with a 5 ml MBP-Trap column (GE Life Sciences) prior to additional purification by size-exclusion chromatography on a Superdex 75 16/60 column. Final purified eIF3d was concentrated to ~20–50 mg ml-1, used immediately for crystallography, or flash frozen in liquid nitrogen for storage at −80 °C.Crystallization and structure determinationInitial crystals of human eIF3d were grown at 18 °C by hanging drop vapor diffusion, but diffracted poorly. Analogous eIF3d cap-binding domain sequences were cloned from a panel of highly homologous animal sequences, with the equivalent domain from the parasitic wasp Nasonia vitripennis producing the best crystals. Optimized N. vitripennis eIF3d crystals were grown in 2 μl hanging drops set at a 1:1 ratio over 300 μl of reservoir liquid: 200 mM (NH4)2SO4, 100 mM Bis-Tris 6.5, 23–27% PEG-3350 (Crystal Form 1), 1.6–1.8 M Ammonium Citrate pH 7.0 (Crystal Form 2), or 200 mM NaCl, 100 mM Tris 8.5, 25% PEG-3350 (Crystal Form 3). eIF3d crystals (Crystal Forms 1 and 2) were cryoprotected by covering the drop with a layer of saturated paratone-N or NVH oil (Hampton) and crystals were transferred into the oil emersion and cleaned using a Kozak cat whisker as previously described29, or cryoprotected by transferring to a reservoir solution supplemented with 20% ethylene glycol (Crystal Form 3). Crystals were harvested with a nylon loop and then flash-frozen in liquid nitrogen. X-ray diffraction data were collected under cryogenic conditions at the Lawrence Berkeley National Laboratory Advanced Light Source (Beamline 8.3.1).Data were processed with XDS and AIMLESS30 using the SSRL autoxds script (A. Gonzalez, Stanford SSRL). eIF3d crystals belonged to the orthorhombic space group P 21 21 21, and contained either two copies per asymmetric unit (Crystal Form 1) or one copy (Crystal Form 2) or the space group P 21 and contained two copies per asymmetric unit (Crystal Form 3). Experimental phase information was collected from a native crystal using sulfur single-wavelength anomalous dispersion. Data was collected at a minimal accessible wavelength (~7235 eV) and iterative data sets were completed and merged from independent portions of an exceptionally large eIF3d crystal. After ~90× multiplicity, anomalous signal was detected to ~2.4 Å, and a clear phase solution was obtained at ~120× multiplicity. 35 sites were identified with HySS in PHENIX31 corresponding to 32 sulfur atoms in eIF3d and 3 potassium/chloride positions. Phases were extended to the native eIF3d data set processed to ~1.40 Å using SOLVE/RESOLVE32, and model building was completed in Coot33 prior to refinement with PHENIX. X-ray data for refinement were extended according to an I/σ resolution cutoff of ~1.5, CC* correlation and Rpim parameters, and visual inspection of the resulting map34. A completed eIF3d cap-binding domain from Crystal Form 1 was used as a search model to determine phases for Crystal Form 2 and 3 using molecular replacement. Final structures were refined to stereochemistry statistics for Ramachandran Plot (Favored/Allowed), Rotamer outliers, and MolProbity score as follows: Crystal Form 1, 96.8% / 3.2%, 0.2% and 1.40; Crystal Form 2, 97.3% / 2.7%, 0% and 1.29; Crystal Form 3 97.4% / 2.6%. 0.9% and 1.26.Recombinant eIF4E protein purification and RNA crosslinkingFull length human eIF4E was cloned and expressed using the same protocol as for eIF3d. eIF4E–RNA crosslinking was performed as described for eIF3–RNA crosslinking, but using 25 nM RNA with normalized cpm and the indicated concentration of eIF4E. RNase treatment was performed using 4 U RNase R and 250 U RNase T1.

Cells and transfectionsHuman 293T cells were maintained in DMEM (Invitrogen) supplemented with 10% FBS (Seradigm). The cells were obtained from the University of California, Berkeley, Cell Culture Facility, which authenticates cells by STR profiling and tests for mycoplasma contamination. Plasmid transfections were performed using Lipofectamine 2000 (Invitrogen), following the manufacturer’s protocol, and polysome or immunoprecipitation analyses were performed at 48 h after transfection. For INK128 (Cayman Chemical) cell treatment, 293T cells were incubated with the indicated concentration of INK128 for ~14–16 h before cell lysis.

Cells and transfections

Human 293T cells were maintained in DMEM (Invitrogen) supplemented with 10% FBS (Seradigm). The cells were obtained from the University of California, Berkeley, Cell Culture Facility, which authenticates cells by STR profiling and tests for mycoplasma contamination. Plasmid transfections were performed using Lipofectamine 2000 (Invitrogen), following the manufacturer’s protocol, and polysome or immunoprecipitation analyses were performed at 48 h after transfection. For INK128 (Cayman Chemical) cell treatment, 293T cells were incubated with the indicated concentration of INK128 for ~14–16 h before cell lysis.

PlasmidsTo generate the eIF3d expression plasmids, eIF3d was amplified from human cDNA and inserted into pcDNA5/FRT. A 39-nucleotide linker followed by the HA epitope tag (YPYDVPDYA) was subsequently inserted prior to the eIF3d stop codon. The wild type c-Jun 5′ UTR luciferase reporter plasmid was previously described2. To generate the c-Jun in vitro transcription template, the 5′ UTR, ORF, and 3′ UTR were separately amplified from human cDNA and stitched together downstream of a T7 promoter by Gibson cloning into pcDNA4. The ACTB in vitro transcription template was constructed by addition of a T7 promoter during amplification of the full mRNA from human cDNA and inserted into pcDNA4.

To generate the eIF3d expression plasmids, eIF3d was amplified from human cDNA and inserted into pcDNA5/FRT. A 39-nucleotide linker followed by the HA epitope tag (YPYDVPDYA) was subsequently inserted prior to the eIF3d stop codon. The wild type c-Jun 5′ UTR luciferase reporter plasmid was previously described2. To generate the c-Jun in vitro transcription template, the 5′ UTR, ORF, and 3′ UTR were separately amplified from human cDNA and stitched together downstream of a T7 promoter by Gibson cloning into pcDNA4. The ACTB in vitro transcription template was constructed by addition of a T7 promoter during amplification of the full mRNA from human cDNA and inserted into pcDNA4.

Western blotWestern blot analyses were performed using the following antibodies: anti-eIF3d (Bethyl A301-758A), anti-eIF4A1 (Cell Signaling 2490), anti-eIF4G1 (Cell Signaling 2858), anti-rpS19 (Bethyl A304-002A), anti-eIF4E (Bethyl A301-154A), anti-rpLP0 (Bethyl A302-882A), anti-HA epitope tag (Pierce 26183), anti-c-Jun (Cell Signaling 9165), anti-Hsp90 (BD Biosciences 610418), and anti-4EBP1 (Cell Signaling 9644).

Western blot

Western blot analyses were performed using the following antibodies: anti-eIF3d (Bethyl A301-758A), anti-eIF4A1 (Cell Signaling 2490), anti-eIF4G1 (Cell Signaling 2858), anti-rpS19 (Bethyl A304-002A), anti-eIF4E (Bethyl A301-154A), anti-rpLP0 (Bethyl A302-882A), anti-HA epitope tag (Pierce 26183), anti-c-Jun (Cell Signaling 9165), anti-Hsp90 (BD Biosciences 610418), and anti-4EBP1 (Cell Signaling 9644).

In vitro RNA transcription and labelingUnlabeled RNAs were in vitro transcribed, polyadenylated, and capped as previously described2. For internal radiolabeling of RNAs, in vitro transcription was performed in the presence of 0.1 µM [α-32P]-ATP, then the RNA was subsequently capped with vaccinia virus enzymes (NEB). For radiolabeling of the 5′ cap, in vitro transcribed RNAs were capped with vaccinia virus enzymes and [α-32P]-GTP. RNAs were purified by phenol-chloroform extraction and ethanol precipitation.

In vitro RNA transcription and labeling

Unlabeled RNAs were in vitro transcribed, polyadenylated, and capped as previously described2. For internal radiolabeling of RNAs, in vitro transcription was performed in the presence of 0.1 µM [α-32P]-ATP, then the RNA was subsequently capped with vaccinia virus enzymes (NEB). For radiolabeling of the 5′ cap, in vitro transcribed RNAs were capped with vaccinia virus enzymes and [α-32P]-GTP. RNAs were purified by phenol-chloroform extraction and ethanol precipitation.

In vitro translationIn vitro translation extracts were made from human 293T cells as previously described21. Lysates were nuclease-treated with 18 gel units μl-1 micrococcal nuclease (NEB) in the presence of 0.7 mM CaCl2 for 10 min at 25 °C, and the digestion was stopped by addition of 2.24 mM EGTA. Each translation reaction contained 50% in vitro translation lysate and buffer to make the final reaction with 0.84 mM ATP, 0.21 mM GTP, 21 mM creatine phosphate (Roche), 45 U ml-1 creatine phosphokinase (Roche), 10 mM HEPES-KOH pH 7.6, 2 mM DTT, 8 mM amino acids (Promega), 255 mM spermidine, 1 U ml-1 murine RNase inhibitor (NEB), and mRNA-specific concentrations of Mg(OAc)2 and KOAc. The optimal magnesium and potassium levels to add were determined to be 1.5 mM Mg(OAc)2 and 150 mM KOAc for c-Jun mRNA, and 1 mM and 150 mM KOAc for ACTB mRNA. For luciferase assays, translation reactions were incubated for 1 h at 30 °C, then luciferase activity was assayed.

In vitro translation

In vitro translation extracts were made from human 293T cells as previously described21. Lysates were nuclease-treated with 18 gel units μl-1 micrococcal nuclease (NEB) in the presence of 0.7 mM CaCl2 for 10 min at 25 °C, and the digestion was stopped by addition of 2.24 mM EGTA. Each translation reaction contained 50% in vitro translation lysate and buffer to make the final reaction with 0.84 mM ATP, 0.21 mM GTP, 21 mM creatine phosphate (Roche), 45 U ml-1 creatine phosphokinase (Roche), 10 mM HEPES-KOH pH 7.6, 2 mM DTT, 8 mM amino acids (Promega), 255 mM spermidine, 1 U ml-1 murine RNase inhibitor (NEB), and mRNA-specific concentrations of Mg(OAc)2 and KOAc. The optimal magnesium and potassium levels to add were determined to be 1.5 mM Mg(OAc)2 and 150 mM KOAc for c-Jun mRNA, and 1 mM and 150 mM KOAc for ACTB mRNA. For luciferase assays, translation reactions were incubated for 1 h at 30 °C, then luciferase activity was assayed.

48S initiation complex purificationFor 48S initiation complex purification from in vitro translation reactions, 180 μl reactions were incubated for 20 min at 30 °C and centrifuged for 6 min at 12,000g at 4 °C. Lysates were purified by size exclusion chromatography through a 1 ml column packed with Sephacryl S-400 gel filtration resin (GE Healthcare) and the elutant was centrifuged through a 10-25% w/v sucrose gradient by centrifugation for 3.5 h at 38,000 rpm at 4 °C in a Beckman SW41 Ti rotor27. Fractions were collected from the top of the gradient using a peristaltic pump with a Brandel tube piercer. From the appropriate fractions, RNA was purified by phenol-chloroform extraction and ethanol precipitation and protein was precipitated with trichloroacetic acid.For affinity purification of HA epitope-tagged eIF3d-associated 48S initiation complexes from cells, three 10 cm plates of transfected 293T cells were treated with 100 μg ml-1 cycloheximide for 5 min. Cells were washed with ice cold PBS (137 mM NaCl, 2.7 mM KCl, 100 mM Na2HPO4, 2 mM KH2PO4) with 100 μg ml-1 cycloheximide and collected in lysis buffer (20 mM HEPES-KOH pH 7.4, 150 mM KOAc, 2.5 mM Mg(OAc)2, 1 mM DTT, 100 μg ml-1 cycloheximide, 1% v/v Triton X-100). Lysates were centrifuged for 6 min at 12000g at 4 °C and purified by S-400 size exclusion chromatography. 80 μl of anti-HA antibody-conjugated agarose beads (Sigma) were added to the elutants, tumbled for 1.5 h at 4 °C, and beads were washed three times with lysis buffer without Triton X-100. Bound complexes were eluted twice with 100 μg ml-1 HA peptide and elutants were centrifuged and analyzed the same as for the in vitro purification reactions.

48S initiation complex purification

For 48S initiation complex purification from in vitro translation reactions, 180 μl reactions were incubated for 20 min at 30 °C and centrifuged for 6 min at 12,000g at 4 °C. Lysates were purified by size exclusion chromatography through a 1 ml column packed with Sephacryl S-400 gel filtration resin (GE Healthcare) and the elutant was centrifuged through a 10-25% w/v sucrose gradient by centrifugation for 3.5 h at 38,000 rpm at 4 °C in a Beckman SW41 Ti rotor27. Fractions were collected from the top of the gradient using a peristaltic pump with a Brandel tube piercer. From the appropriate fractions, RNA was purified by phenol-chloroform extraction and ethanol precipitation and protein was precipitated with trichloroacetic acid.

For affinity purification of HA epitope-tagged eIF3d-associated 48S initiation complexes from cells, three 10 cm plates of transfected 293T cells were treated with 100 μg ml-1 cycloheximide for 5 min. Cells were washed with ice cold PBS (137 mM NaCl, 2.7 mM KCl, 100 mM Na2HPO4, 2 mM KH2PO4) with 100 μg ml-1 cycloheximide and collected in lysis buffer (20 mM HEPES-KOH pH 7.4, 150 mM KOAc, 2.5 mM Mg(OAc)2, 1 mM DTT, 100 μg ml-1 cycloheximide, 1% v/v Triton X-100). Lysates were centrifuged for 6 min at 12000g at 4 °C and purified by S-400 size exclusion chromatography. 80 μl of anti-HA antibody-conjugated agarose beads (Sigma) were added to the elutants, tumbled for 1.5 h at 4 °C, and beads were washed three times with lysis buffer without Triton X-100. Bound complexes were eluted twice with 100 μg ml-1 HA peptide and elutants were centrifuged and analyzed the same as for the in vitro purification reactions.

Quantitative real-time PCRcDNA was reverse-transcribed from RNA using random hexamers and Superscript III (Invitrogen), following the manufacturer’s protocol. Real-time PCR was performed using DyNAmo HS Sybr Green (ThermoFisher), with a 20 μl reaction volume containing 2 μl cDNA and 0.5 μM of each primer. The following oligonucleotides were used: 18S rRNA-Forward, 5′ – GGCCCTGTAATTGGAATGAGTC–3′; 18S rRNA-Reverse, 5′ –CCAAGATCCAACTACGAGCTT–3′; ACTB-Forward, 5′ – CTCTTCCAGCCTTCCTTCCT–3′; ACTB-Reverse, 5′ -AGCACTGTGTTGGCGTACAG-3′; c-Jun-Forward, 5′–TGACTGCAAAGATGGAAACG–3′; c-Jun-Reverse; 5′–CAGGGTCATGCTCTGTTTCA–3′.

Quantitative real-time PCR

cDNA was reverse-transcribed from RNA using random hexamers and Superscript III (Invitrogen), following the manufacturer’s protocol. Real-time PCR was performed using DyNAmo HS Sybr Green (ThermoFisher), with a 20 μl reaction volume containing 2 μl cDNA and 0.5 μM of each primer. The following oligonucleotides were used: 18S rRNA-Forward, 5′ – GGCCCTGTAATTGGAATGAGTC–3′; 18S rRNA-Reverse, 5′ –CCAAGATCCAACTACGAGCTT–3′; ACTB-Forward, 5′ – CTCTTCCAGCCTTCCTTCCT–3′; ACTB-Reverse, 5′ -AGCACTGTGTTGGCGTACAG-3′; c-Jun-Forward, 5′–TGACTGCAAAGATGGAAACG–3′; c-Jun-Reverse; 5′–CAGGGTCATGCTCTGTTTCA–3′.

eIF3-RNA crosslinking and gel shiftRecombinant eIF3 was expressed and purified from E. coli as previously described26. For each crosslinking reaction, 1 μl water, 1 μl 125 nM labeled RNA, 1 μl 10 μg ml-1 heparan sulfate (Sigma), 1 μl 5× binding buffer (125 mM Tris-HCl pH 7.5, 25mM Mg(OAc)2, 350 mM KCl, 0.5 mM CaCl2, 0.5 mg ml-1 BSA, 10 mM TCEP), and 1 μl 1.5 μM purified eIF3 were added, in the listed order, and incubated for 30 min at 25 °C. For competition experiments, the water was substituted with 1 μl of 1 mM m7GDP/Mg2+ or GDP/Mg2+. UV254-induced crosslinking was performed using a short-wave UV lamp placed ~4 cm above the samples on ice for 10 min. After treatment with RNase for 10 min at 37 °C, proteins were separated by 12% SDS-PAGE, the gel was dried, and imaged using a phosphorimager. For digestion of internal labeled RNA, 2.5 U benzonase (Novagen) and 250 U RNase T1 (ThermoFisher) were used; for digestion of cap labeled RNA, 4 U RNase R (Epicentre) and 1 U RiboShredder (Epicentre) were used. For eIF3d subunit identification, after RNase treatment, samples were denatured and immunoprecipitation was performed as previously described2. For limited proteolysis, after RNase treatment, the reactions were treated with 2 or 20 μg ml-1 sequencing grade trypsin (Promega) for 30 min at 25 °C, prior to gel electrophoresis. Mass spectrometry samples were prepared as previously described2. Gel shift assays were performed as previously described, using 50 nM labeled c-Jun stem loop RNA and 300 nM purified eIF32.

eIF3-RNA crosslinking and gel shift

Recombinant eIF3 was expressed and purified from E. coli as previously described26. For each crosslinking reaction, 1 μl water, 1 μl 125 nM labeled RNA, 1 μl 10 μg ml-1 heparan sulfate (Sigma), 1 μl 5× binding buffer (125 mM Tris-HCl pH 7.5, 25mM Mg(OAc)2, 350 mM KCl, 0.5 mM CaCl2, 0.5 mg ml-1 BSA, 10 mM TCEP), and 1 μl 1.5 μM purified eIF3 were added, in the listed order, and incubated for 30 min at 25 °C. For competition experiments, the water was substituted with 1 μl of 1 mM m7GDP/Mg2+ or GDP/Mg2+. UV254-induced crosslinking was performed using a short-wave UV lamp placed ~4 cm above the samples on ice for 10 min. After treatment with RNase for 10 min at 37 °C, proteins were separated by 12% SDS-PAGE, the gel was dried, and imaged using a phosphorimager. For digestion of internal labeled RNA, 2.5 U benzonase (Novagen) and 250 U RNase T1 (ThermoFisher) were used; for digestion of cap labeled RNA, 4 U RNase R (Epicentre) and 1 U RiboShredder (Epicentre) were used. For eIF3d subunit identification, after RNase treatment, samples were denatured and immunoprecipitation was performed as previously described2. For limited proteolysis, after RNase treatment, the reactions were treated with 2 or 20 μg ml-1 sequencing grade trypsin (Promega) for 30 min at 25 °C, prior to gel electrophoresis. Mass spectrometry samples were prepared as previously described2. Gel shift assays were performed as previously described, using 50 nM labeled c-Jun stem loop RNA and 300 nM purified eIF32.

Recombinant eIF3d protein purificationCandidate eIF3d cap-binding domain fragments were amplified by PCR and cloned into a modified pET vector to express an N-terminal 6× His (KSSHHHHHHGSS)-MBP-TEV fusion protein as previously described28. Extensive expression trials were conducted to determine optimal amino- and carboxy-terminal domain boundaries and identified a minimal stable human cap-binding domain S161–F527. Recombinant protein was expressed in BL21-RIL DE3 E. coli cells co-transformed with a pRARE2 tRNA plasmid (Agilent). E. coli was grown in 2×YT media at 37 °C to an OD600 of ~0.5, cooled at 4 °C for 15 min, induced with addition of 0.5 mM IPTG and then incubated with shaking for ~20 h at 16 °C. Pelleted cells were washed with PBS and then lysed by sonication in lysis buffer (20 mM HEPES-KOH pH 7.5, 400 mM NaCl, 10% glycerol, 30 mM imidazole, 1 mM TCEP) in the presence of EDTA-free Complete Protease Inhibitor (Roche). Following centrifugation for 30 min at 23,000g and 4 °C, clarified lysate was incubated with Ni-NTA agarose resin (QIAGEN) for 1 h at 4 °C with gentle rocking. Resin was washed with lysis buffer supplemented to 1 M NaCl and eluted by gravity-flow chromatography at 4 °C with lysis buffer supplemented to 300 mM imidazole. The eluted fraction was diluted to ~50 mM imidazole and 5% glycerol prior, concentrated to ~50 mg ml-1 and incubated with Tobacco Etch Virus protease for ~12 h at 4 °C to remove the MBP tag. Recombinant eIF3d was isolated from free MBP by diluting with gel-filtration buffer (20 mM HEPES-KOH pH 7.5, 250 mM KCl, 1 mM TCEP) and passing over a 5 ml Ni-NTA column (QIAGEN) connected in line with a 5 ml MBP-Trap column (GE Life Sciences) prior to additional purification by size-exclusion chromatography on a Superdex 75 16/60 column. Final purified eIF3d was concentrated to ~20–50 mg ml-1, used immediately for crystallography, or flash frozen in liquid nitrogen for storage at −80 °C.

Recombinant eIF3d protein purification

Candidate eIF3d cap-binding domain fragments were amplified by PCR and cloned into a modified pET vector to express an N-terminal 6× His (KSSHHHHHHGSS)-MBP-TEV fusion protein as previously described28. Extensive expression trials were conducted to determine optimal amino- and carboxy-terminal domain boundaries and identified a minimal stable human cap-binding domain S161–F527. Recombinant protein was expressed in BL21-RIL DE3 E. coli cells co-transformed with a pRARE2 tRNA plasmid (Agilent). E. coli was grown in 2×YT media at 37 °C to an OD600 of ~0.5, cooled at 4 °C for 15 min, induced with addition of 0.5 mM IPTG and then incubated with shaking for ~20 h at 16 °C. Pelleted cells were washed with PBS and then lysed by sonication in lysis buffer (20 mM HEPES-KOH pH 7.5, 400 mM NaCl, 10% glycerol, 30 mM imidazole, 1 mM TCEP) in the presence of EDTA-free Complete Protease Inhibitor (Roche). Following centrifugation for 30 min at 23,000g and 4 °C, clarified lysate was incubated with Ni-NTA agarose resin (QIAGEN) for 1 h at 4 °C with gentle rocking. Resin was washed with lysis buffer supplemented to 1 M NaCl and eluted by gravity-flow chromatography at 4 °C with lysis buffer supplemented to 300 mM imidazole. The eluted fraction was diluted to ~50 mM imidazole and 5% glycerol prior, concentrated to ~50 mg ml-1 and incubated with Tobacco Etch Virus protease for ~12 h at 4 °C to remove the MBP tag. Recombinant eIF3d was isolated from free MBP by diluting with gel-filtration buffer (20 mM HEPES-KOH pH 7.5, 250 mM KCl, 1 mM TCEP) and passing over a 5 ml Ni-NTA column (QIAGEN) connected in line with a 5 ml MBP-Trap column (GE Life Sciences) prior to additional purification by size-exclusion chromatography on a Superdex 75 16/60 column. Final purified eIF3d was concentrated to ~20–50 mg ml-1, used immediately for crystallography, or flash frozen in liquid nitrogen for storage at −80 °C.

Crystallization and structure determinationInitial crystals of human eIF3d were grown at 18 °C by hanging drop vapor diffusion, but diffracted poorly. Analogous eIF3d cap-binding domain sequences were cloned from a panel of highly homologous animal sequences, with the equivalent domain from the parasitic wasp Nasonia vitripennis producing the best crystals. Optimized N. vitripennis eIF3d crystals were grown in 2 μl hanging drops set at a 1:1 ratio over 300 μl of reservoir liquid: 200 mM (NH4)2SO4, 100 mM Bis-Tris 6.5, 23–27% PEG-3350 (Crystal Form 1), 1.6–1.8 M Ammonium Citrate pH 7.0 (Crystal Form 2), or 200 mM NaCl, 100 mM Tris 8.5, 25% PEG-3350 (Crystal Form 3). eIF3d crystals (Crystal Forms 1 and 2) were cryoprotected by covering the drop with a layer of saturated paratone-N or NVH oil (Hampton) and crystals were transferred into the oil emersion and cleaned using a Kozak cat whisker as previously described29, or cryoprotected by transferring to a reservoir solution supplemented with 20% ethylene glycol (Crystal Form 3). Crystals were harvested with a nylon loop and then flash-frozen in liquid nitrogen. X-ray diffraction data were collected under cryogenic conditions at the Lawrence Berkeley National Laboratory Advanced Light Source (Beamline 8.3.1).Data were processed with XDS and AIMLESS30 using the SSRL autoxds script (A. Gonzalez, Stanford SSRL). eIF3d crystals belonged to the orthorhombic space group P 21 21 21, and contained either two copies per asymmetric unit (Crystal Form 1) or one copy (Crystal Form 2) or the space group P 21 and contained two copies per asymmetric unit (Crystal Form 3). Experimental phase information was collected from a native crystal using sulfur single-wavelength anomalous dispersion. Data was collected at a minimal accessible wavelength (~7235 eV) and iterative data sets were completed and merged from independent portions of an exceptionally large eIF3d crystal. After ~90× multiplicity, anomalous signal was detected to ~2.4 Å, and a clear phase solution was obtained at ~120× multiplicity. 35 sites were identified with HySS in PHENIX31 corresponding to 32 sulfur atoms in eIF3d and 3 potassium/chloride positions. Phases were extended to the native eIF3d data set processed to ~1.40 Å using SOLVE/RESOLVE32, and model building was completed in Coot33 prior to refinement with PHENIX. X-ray data for refinement were extended according to an I/σ resolution cutoff of ~1.5, CC* correlation and Rpim parameters, and visual inspection of the resulting map34. A completed eIF3d cap-binding domain from Crystal Form 1 was used as a search model to determine phases for Crystal Form 2 and 3 using molecular replacement. Final structures were refined to stereochemistry statistics for Ramachandran Plot (Favored/Allowed), Rotamer outliers, and MolProbity score as follows: Crystal Form 1, 96.8% / 3.2%, 0.2% and 1.40; Crystal Form 2, 97.3% / 2.7%, 0% and 1.29; Crystal Form 3 97.4% / 2.6%. 0.9% and 1.26.

Crystallization and structure determination

Initial crystals of human eIF3d were grown at 18 °C by hanging drop vapor diffusion, but diffracted poorly. Analogous eIF3d cap-binding domain sequences were cloned from a panel of highly homologous animal sequences, with the equivalent domain from the parasitic wasp Nasonia vitripennis producing the best crystals. Optimized N. vitripennis eIF3d crystals were grown in 2 μl hanging drops set at a 1:1 ratio over 300 μl of reservoir liquid: 200 mM (NH4)2SO4, 100 mM Bis-Tris 6.5, 23–27% PEG-3350 (Crystal Form 1), 1.6–1.8 M Ammonium Citrate pH 7.0 (Crystal Form 2), or 200 mM NaCl, 100 mM Tris 8.5, 25% PEG-3350 (Crystal Form 3). eIF3d crystals (Crystal Forms 1 and 2) were cryoprotected by covering the drop with a layer of saturated paratone-N or NVH oil (Hampton) and crystals were transferred into the oil emersion and cleaned using a Kozak cat whisker as previously described29, or cryoprotected by transferring to a reservoir solution supplemented with 20% ethylene glycol (Crystal Form 3). Crystals were harvested with a nylon loop and then flash-frozen in liquid nitrogen. X-ray diffraction data were collected under cryogenic conditions at the Lawrence Berkeley National Laboratory Advanced Light Source (Beamline 8.3.1).

Data were processed with XDS and AIMLESS30 using the SSRL autoxds script (A. Gonzalez, Stanford SSRL). eIF3d crystals belonged to the orthorhombic space group P 21 21 21, and contained either two copies per asymmetric unit (Crystal Form 1) or one copy (Crystal Form 2) or the space group P 21 and contained two copies per asymmetric unit (Crystal Form 3). Experimental phase information was collected from a native crystal using sulfur single-wavelength anomalous dispersion. Data was collected at a minimal accessible wavelength (~7235 eV) and iterative data sets were completed and merged from independent portions of an exceptionally large eIF3d crystal. After ~90× multiplicity, anomalous signal was detected to ~2.4 Å, and a clear phase solution was obtained at ~120× multiplicity. 35 sites were identified with HySS in PHENIX31 corresponding to 32 sulfur atoms in eIF3d and 3 potassium/chloride positions. Phases were extended to the native eIF3d data set processed to ~1.40 Å using SOLVE/RESOLVE32, and model building was completed in Coot33 prior to refinement with PHENIX. X-ray data for refinement were extended according to an I/σ resolution cutoff of ~1.5, CC* correlation and Rpim parameters, and visual inspection of the resulting map34. A completed eIF3d cap-binding domain from Crystal Form 1 was used as a search model to determine phases for Crystal Form 2 and 3 using molecular replacement. Final structures were refined to stereochemistry statistics for Ramachandran Plot (Favored/Allowed), Rotamer outliers, and MolProbity score as follows: Crystal Form 1, 96.8% / 3.2%, 0.2% and 1.40; Crystal Form 2, 97.3% / 2.7%, 0% and 1.29; Crystal Form 3 97.4% / 2.6%. 0.9% and 1.26.

Recombinant eIF4E protein purification and RNA crosslinkingFull length human eIF4E was cloned and expressed using the same protocol as for eIF3d. eIF4E–RNA crosslinking was performed as described for eIF3–RNA crosslinking, but using 25 nM RNA with normalized cpm and the indicated concentration of eIF4E. RNase treatment was performed using 4 U RNase R and 250 U RNase T1.

Recombinant eIF4E protein purification and RNA crosslinking

Full length human eIF4E was cloned and expressed using the same protocol as for eIF3d. eIF4E–RNA crosslinking was performed as described for eIF3–RNA crosslinking, but using 25 nM RNA with normalized cpm and the indicated concentration of eIF4E. RNase treatment was performed using 4 U RNase R and 250 U RNase T1.

Extended DataExtended Data Figure 1c-Jun expression is unaffected by 4E-BP1 activationRepresentative western blot of 293T cells after 24 h treatment with INK128. The results are representative of three independent experiments. For gel source data, see Supplementary Figure 1.Extended Data Figure 2Mapping of a C-terminal region of eIF3d that protects the c-Jun 5′ cap structurea, Validation of eIF3d subunit identification. eIF3d-cap crosslinking was validated by immunoprecipitation of eIF3d after crosslinking and denaturing the eIF3 complex by boiling in SDS. The result is representative of biological replicates. b, Limited proteolysis of eIF3 crosslinked to 32P-cap-labeled c-Jun 5′ UTR RNA. Full length and proteolysis fragments of eIF3d are indicated with black and maroon arrows, respectively, on the phosphorimage and coomassie-stained SDS gels. c, Mass spectrometry identification of trypsinized peptides from limited proteolysis of cap-crosslinked eIF3d. Identified peptides are highlighted in blue. The results in b-c are representative of three independent experiments.Extended Data Figure 3Purification of eIF3d cap-binding domaina, Alignment is colored by phylogenetic conservation of amino acid physiochemical property similarity and a cartoon schematic of the eIF3d secondary structure is depicted below the sequences. Coloring begins at 30% conservation (lightest blue). b, Coomassie blue stained SDS gel of recombinant Nasonia vitripennis eIF3d S162–F527 protein expressed in E. coli.Extended Data Figure 4Structure-based alignment of eIF3d cap-binding domain and DXO cap-endonuclease domain sequencesStructure-based alignment of eIF3d and DXO sequences according to superposition of eIF3d and DXO (PDB 4J7L) structures15. Alignment is colored by phylogenetic conservation as in Extended Data Fig. 2 and cartoon schematics of the secondary structures are depicted below the sequences. eIF3d is colored in blue and magenta as in Fig. 2, and DXO is colored in green and magenta.Extended Data Figure 5Structural details of eIF3d “RNA gate” stabilizing interactionsa–c. Structural overview of eIF3d with cut-away sections highlighting charged-interactions stabilizing the closed “RNA gate” conformation. No significant van der Waals interactions stabilize the closed gate conformation, supporting likely repositioning of the RNA gate prior to 5′ mRNA cap recognition. Charged interactions occur in three areas: a, at the beginning of the gate insertion sequence (Gate Beginning); b, at the tip of the unstructured loop (Gate Tip); and c, at an “Arginine Anchor” point stabilizing the return of the loop insertion sequence to the α-helix shared with DXO-family endonucleases. Residues are numbered according to the human eIF3d sequence, and all positions are conserved between human and N. vitripennis except S292N. eIF3d RNA gate residues are displayed with blue side-chains and the residues making stabilizing contacts are colored as green. 2Fo–Fc map regions are shown at 1.5σ.Extended Data Figure 6Packing interactions observed in alternative eIF3d crystal formsCartoon representation of crystallographic packing in eIF3d crystal form 1, 2 and 3 (a, b, c). Crystal forms 1 and 3 have two copies of eIF3d in the asymmetric unit colored in blue/magenta and green/magenta respectively; crystal form 2 has only one copy of eIF3d per asymmetric unit. Symmetry related molecules are depicted in grey. Cut-away zoom illustrates position of the eIF3d RNA gate (red) relative to nearest symmetry related molecule. In crystal form 1 the RNA gate is packed against a neighboring symmetry molecule, but in crystal forms 2 and 3 the RNA gate is positioned towards a major solvent channel. Relative conformation of the RNA gate remains unchanged in either eIF3d crystal form.Extended Data Figure 7eIF3d cap-binding activity requires the eIF3-recruitment stem loop RNAPhosphorimage of SDS gel resolving RNase-protected 32P-cap-labeled c-Jun stem loop RNA crosslinked to eIF3 subunits. The result is representative of three independent experiments.Extended Data Figure 8Incorporation of HA epitope-tagged eIF3d into translation initiation complexesa, Coomassie blue stained SDS gel of recombinant eIF3 containing wild type (WT) or helix α5 or α11 mutated eIF3d. b, Representative native agarose gel electrophoresis of recombinant wild type and mutant eIF3 complexes bound to the c-Jun stem loop. c, Polysome profiles of untransfected 293T cells, plotted as relative absorbance at 254 nm versus elution fractions. d, Western blot analysis of eIF3d and the small (rpS19) and large (rpLP0) ribosomal subunits. The results in b-d are representative of three independent experiments. For gel source data, see Supplementary Figure 1.Extended Data Figure 9eIF4E recognizes the 5′ end of the c-Jun mRNA less efficiently than ACTB mRNAa, Coomassie blue stained SDS gel of recombinant human eIF4E expressed in E. coli.
b, Phosphorimage of SDS gel resolving RNase-protected 32P-cap-labeled ACTB or c-Jun 5′ UTR RNA crosslinked to eIF4E. The result is representative of three independent experiments. For gel source data, see Supplementary Figure 1.Extended Data Table 1Summary of data collection, phasing and refinement statisticsSingle crystals were used to collect data for each structure.eIF3d Crystal Form 1 (5K4B)eIF3d Crystal Form 2 (5K4C)eIF3d Crystal Form 3 (5K4D)eIF3d (S-SAD)Data collectionSpace groupP 212121P 212121P 21P 212121Cell dimensions a, b, c (Å)61.90, 62.93, 192.2449.01, 61.84, 138.3149.97, 144.32, 55.3061.66, 62.72, 193.28 α, β, γ (°)90.0, 90.0, 90.090.0, 90.0, 90.090.0, 109.12, 90.090.0, 90.0, 90.0Wavelength1.115861.115861.115871.71371Resolution (Å)a48.06–1.40 (1.42–1.40)46.20–1.70 (1.73–1.70)49.13–2.00 (2.05–2.00)38.66–1.92 (1.96–1.92)Rpim2.6 (45.3)5.0 (45.0)8.6 (52.3)0.9 (8.1)I/σ (I)12.9 (1.5)9.1 (1.6)6.5 (1.4)65.2 (6.7)CC1/299.9 (57.9)99.6 (63.9)98.8 (55.6)100 (94.7)Completeness (%)100 (99.2)99.4 (89.4)99.8 (98.3)98.5 (86.1)Redundancy15.8 (10.1)4.1 (3.6)3.0 (2.7)129.0 (46.6)RefinementResolution (Å)48.06–1.4046.20–1.7049.13–2.00No reflections Total2,349,428193,627147,613 Unique148,45547,12349,757 Free (%)255Rwork / Rfree17.5 / 19.316.2 / 19.617.7 / 20.8No. atoms Protein583729535686 Ligand/ion3 (CI)18 (glycerol)- Water910385741B factors Protein21.724.226.6 Ligand/ion31.729.1- Water33.136.734.3r.m.s deviations Bond lengths (Å)0.0070.0120.004 Bond angles (°)1.1671.3590.638aValues in parentheses are for highest-resolution shell.

Extended Data

c-Jun expression is unaffected by 4E-BP1 activation

Representative western blot of 293T cells after 24 h treatment with INK128. The results are representative of three independent experiments. For gel source data, see Supplementary Figure 1.

Mapping of a C-terminal region of eIF3d that protects the c-Jun 5′ cap structure

a, Validation of eIF3d subunit identification. eIF3d-cap crosslinking was validated by immunoprecipitation of eIF3d after crosslinking and denaturing the eIF3 complex by boiling in SDS. The result is representative of biological replicates. b, Limited proteolysis of eIF3 crosslinked to 32P-cap-labeled c-Jun 5′ UTR RNA. Full length and proteolysis fragments of eIF3d are indicated with black and maroon arrows, respectively, on the phosphorimage and coomassie-stained SDS gels. c, Mass spectrometry identification of trypsinized peptides from limited proteolysis of cap-crosslinked eIF3d. Identified peptides are highlighted in blue. The results in b-c are representative of three independent experiments.

Purification of eIF3d cap-binding domain

a, Alignment is colored by phylogenetic conservation of amino acid physiochemical property similarity and a cartoon schematic of the eIF3d secondary structure is depicted below the sequences. Coloring begins at 30% conservation (lightest blue). b, Coomassie blue stained SDS gel of recombinant Nasonia vitripennis eIF3d S162–F527 protein expressed in E. coli.

Structure-based alignment of eIF3d cap-binding domain and DXO cap-endonuclease domain sequences

Structure-based alignment of eIF3d and DXO sequences according to superposition of eIF3d and DXO (PDB 4J7L) structures15. Alignment is colored by phylogenetic conservation as in Extended Data Fig. 2 and cartoon schematics of the secondary structures are depicted below the sequences. eIF3d is colored in blue and magenta as in Fig. 2, and DXO is colored in green and magenta.

Structural details of eIF3d “RNA gate” stabilizing interactions

a–c. Structural overview of eIF3d with cut-away sections highlighting charged-interactions stabilizing the closed “RNA gate” conformation. No significant van der Waals interactions stabilize the closed gate conformation, supporting likely repositioning of the RNA gate prior to 5′ mRNA cap recognition. Charged interactions occur in three areas: a, at the beginning of the gate insertion sequence (Gate Beginning); b, at the tip of the unstructured loop (Gate Tip); and c, at an “Arginine Anchor” point stabilizing the return of the loop insertion sequence to the α-helix shared with DXO-family endonucleases. Residues are numbered according to the human eIF3d sequence, and all positions are conserved between human and N. vitripennis except S292N. eIF3d RNA gate residues are displayed with blue side-chains and the residues making stabilizing contacts are colored as green. 2Fo–Fc map regions are shown at 1.5σ.

Packing interactions observed in alternative eIF3d crystal forms

Cartoon representation of crystallographic packing in eIF3d crystal form 1, 2 and 3 (a, b, c). Crystal forms 1 and 3 have two copies of eIF3d in the asymmetric unit colored in blue/magenta and green/magenta respectively; crystal form 2 has only one copy of eIF3d per asymmetric unit. Symmetry related molecules are depicted in grey. Cut-away zoom illustrates position of the eIF3d RNA gate (red) relative to nearest symmetry related molecule. In crystal form 1 the RNA gate is packed against a neighboring symmetry molecule, but in crystal forms 2 and 3 the RNA gate is positioned towards a major solvent channel. Relative conformation of the RNA gate remains unchanged in either eIF3d crystal form.

eIF3d cap-binding activity requires the eIF3-recruitment stem loop RNA

Phosphorimage of SDS gel resolving RNase-protected 32P-cap-labeled c-Jun stem loop RNA crosslinked to eIF3 subunits. The result is representative of three independent experiments.

Incorporation of HA epitope-tagged eIF3d into translation initiation complexes

a, Coomassie blue stained SDS gel of recombinant eIF3 containing wild type (WT) or helix α5 or α11 mutated eIF3d. b, Representative native agarose gel electrophoresis of recombinant wild type and mutant eIF3 complexes bound to the c-Jun stem loop. c, Polysome profiles of untransfected 293T cells, plotted as relative absorbance at 254 nm versus elution fractions. d, Western blot analysis of eIF3d and the small (rpS19) and large (rpLP0) ribosomal subunits. The results in b-d are representative of three independent experiments. For gel source data, see Supplementary Figure 1.

eIF4E recognizes the 5′ end of the c-Jun mRNA less efficiently than ACTB mRNA

a, Coomassie blue stained SDS gel of recombinant human eIF4E expressed in E. coli.
b, Phosphorimage of SDS gel resolving RNase-protected 32P-cap-labeled ACTB or c-Jun 5′ UTR RNA crosslinked to eIF4E. The result is representative of three independent experiments. For gel source data, see Supplementary Figure 1.

Summary of data collection, phasing and refinement statistics

Single crystals were used to collect data for each structure.

Values in parentheses are for highest-resolution shell.

Supplementary Materialsupp_fig1supp_table1

Supplementary Material
