---
pmid: '20596027'
title: SCF(Cyclin F) controls centrosome homeostasis and mitotic fidelity through
  CP110 degradation.
authors:
- D'Angiolella V
- Donato V
- Vijayakumar S
- Saraf A
- Florens L
- Washburn MP
- Dynlacht B
- Pagano M
journal: Nature
year: '2010'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC2946399
doi: 10.1038/nature09140
---

# SCF(Cyclin F) controls centrosome homeostasis and mitotic fidelity through CP110 degradation.
**Authors:** D'Angiolella V, Donato V, Vijayakumar S, Saraf A, Florens L, Washburn MP, Dynlacht B, Pagano M
**Journal:** Nature (2010)
**DOI:** [10.1038/nature09140](https://doi.org/10.1038/nature09140)
**PMC:** [PMC2946399](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2946399/)

## Abstract

1. Nature. 2010 Jul 1;466(7302):138-42. doi: 10.1038/nature09140.

SCF(Cyclin F) controls centrosome homeostasis and mitotic fidelity through CP110 
degradation.

D'Angiolella V(1), Donato V, Vijayakumar S, Saraf A, Florens L, Washburn MP, 
Dynlacht B, Pagano M.

Author information:
(1)Department of Pathology, NYU Cancer Institute, New York University School of 
Medicine, 522 First Avenue, SRB 1107, New York, New York 10016, USA.

Generally, F-box proteins are the substrate recognition subunits of SCF 
(Skp1-Cul1-F-box protein) ubiquitin ligase complexes, which mediate the timely 
proteolysis of important eukaryotic regulatory proteins. Mammalian genomes 
encode roughly 70 F-box proteins, but only a handful have established functions. 
The F-box protein family obtained its name from Cyclin F (also called Fbxo1), in 
which the F-box motif (the approximately 40-amino-acid domain required for 
binding to Skp1) was first described. Cyclin F, which is encoded by an essential 
gene, also contains a cyclin box domain, but in contrast to most cyclins, it 
does not bind or activate any cyclin-dependent kinases (CDKs). However, like 
other cyclins, Cyclin F oscillates during the cell cycle, with protein levels 
peaking in G2. Despite its essential nature and status as the founding member of 
the F-box protein family, Cyclin F remains an orphan protein, whose functions 
are unknown. Starting from an unbiased screen, we identified CP110, a protein 
that is essential for centrosome duplication, as an interactor and substrate of 
Cyclin F. Using a mode of substrate binding distinct from other F-box 
protein-substrate pairs, CP110 and Cyclin F physically associate on the 
centrioles during the G2 phase of the cell cycle, and CP110 is ubiquitylated by 
the SCF(Cyclin F) ubiquitin ligase complex, leading to its degradation. 
siRNA-mediated depletion of Cyclin F in G2 induces centrosomal and mitotic 
abnormalities, such as multipolar spindles and asymmetric, bipolar spindles with 
lagging chromosomes. These phenotypes were reverted by co-silencing CP110 and 
were recapitulated by expressing a stable mutant of CP110 that cannot bind 
Cyclin F. Finally, expression of a stable CP110 mutant in cultured cells also 
promotes the formation of micronuclei, a hallmark of chromosome instability. We 
propose that SCF(Cyclin F)-mediated degradation of CP110 is required for the 
fidelity of mitosis and genome integrity.

DOI: 10.1038/nature09140
PMCID: PMC2946399
PMID: 20596027 [Indexed for MEDLINE]

## Full Text

To identify substrates of the SCFCyclin F ubiquitin ligase, FLAG-HA-tagged Cyclin F was transiently expressed in either HeLa or HEK-293T cells and immunopurified for analysis by Multidimensional Protein Identification Technology (MudPIT)8. In both cases, MudPIT revealed the presence of peptides corresponding to Skp1 and Cul1, whereas, in agreement with previous reports6,9, no peptides corresponding to CDKs were identified (see also Supplementary Fig. 1a). Instead, MudPIT revealed the presence of peptides derived from CP110 (Supplementary Table 1). Combining both analyses, 21 total spectra, corresponding to 12 unique CP110 peptides, were identified. In two additional experiments, we immunopurified a Cyclin F mutant lacking the cyclin box [Cyclin F(1–270)], and although Skp1 and Cul1 still co-immunoprecipitated with Cyclin F(1–270), CP110 was not present (Supplementary Table 1).

CP110 localizes to the distal ends of the centrioles, and its depletion interferes with centrosome re-duplication generated either by arresting cells in S phase for prolonged time periods or by overexpressing Plk4 (refs. 10,11), indicating that CP110 has a pivotal role in new centriole formation. Similarly, the fly ortholog of CP110 is necessary for both centriole duplication and centrosome maturation12. Finally, CP110 has additional roles in the regulation of centriole length and cilium formation13,14.

To investigate whether the binding between CP110 and Cyclin F is specific, we expressed in HEK-293T fourteen F-box proteins that were then immunoprecipitated to evaluate their interaction with CP110. We found that the only F-box protein able to co-immunoprecipitate endogenous CP110 was Cyclin F (Supplementary Fig. 1b). Using synchronized HeLa cells, the interaction between endogenous Cyclin F and endogenous CP110 was observed exclusively in G2 and M, as monitored by immunoblotting for cell cycle markers and flow cytometry (Fig. 1a and data not shown). Subsequently, we mapped the Cyclin F binding motif of CP110. A series of binding experiments, using multiple CP110 deletion mutants, narrowed the binding motif to a region of human CP110 located between amino acids 565–620 (Supplementary Fig. 2). This region contains one putative RxL motif, an established cyclin binding domain. A mutant in this motif [CP110(RxL/AxA)] failed to co-immunoprecipitate endogenous Cyclin F (Supplementary Fig. 2a, last lane), indicating that the RxL motif, located at residues 588–590, mediates binding to Cyclin F. Cyclins recruit RxL-containing proteins through a hydrophobic patch present in the cyclin box domain15. We observed that Cyclin F requires its cyclin box to bind endogenous CP110 (Supplementary Fig. 3a–b and Supplementary Table 1). Moreover, Cyclin F displays a conserved hydrophobic patch and a Cyclin F mutated in this domain [Cyclin F(M/A;L/A)] lost the ability to bind CP110 (Supplementary Fig. 3c–d).

Because of the centrosomal localization of CP110, we investigated the subcellular localization of Cyclin F. Synchronized U-2OS cells were co-stained with antibodies to Cyclin F and γ-tubulin (a centrosome marker). As expected, both S and G2 phase cells displayed two γ-tubulin dots, which were distinct (but adjacent) in S phase and separated by at least 2 µM in G2. We found that, similar to CP110 (ref.10,11), endogenous Cyclin F partially colocalized with γ-tubulin (Fig. 1b); however, in contrast to CP110, which is exclusively centrosomal (ref.10,11, Fig. 1c, and Supplementary Fig. 4), Cyclin F also localized to the nucleus. Moreover, Cyclin F staining on the centrosomes was rare in S phase cells and increased in G2 cells, whereas the intensity of CP110 staining was predominant in S and decreased in G2 (Fig. 1b and Supplementary Fig. 4), correlating with the total CP110 levels detected by immunoblotting. We also observed that Cherry-Cyclin F colocalized with GFP-CP110 and was present on both the mother and daughter centrioles (Fig. 1b–c). Finally, we found that the centrosomal localization of Cyclin F does not require its binding to CP110 and is present at the N-terminus since: (i) Cyclin F(1–270) and Cyclin F(M/A;L/A) (which both do not interact with CP110) still localized to the centrioles and (ii) Cyclin F(271–786) (which contains the CP110-binding domain) completely lost its centrosomal localization and binding to CP110 (Supplementary Fig. 3a and data not shown).

As part of our investigation of CP110 binding to Cyclin F, we noticed that, compared to wild type Cyclin F, the Cyclin F(LP/AA) mutant (in which the first two amino acids of the F-box domain were mutated to alanine) bound less Skp1 and Cul1 (as expected) but more CP110 (Supplementary Fig. 3b). This result suggested that CP110 is targeted for proteolysis by Cyclin F, because Cyclin F(LP/AA) is unable to form an active ubiquitin ligase: it can sequester CP110 in a more stable complex that is easier to detect. In agreement with this interpretation, expression of wild type Cyclin F resulted in a dramatic reduction of endogenous CP110 levels in four different cell lines. Moreover, expression of either Cyclin F(LP/AA) or Cyclin F(M/A;L/A) had no effect on CP110 levels (Fig. 2a and Supplementary Fig. 5).

To further test whether Cyclin F might regulate the degradation of CP110, we used three different siRNA oligos to reduce the expression of Cyclin F in synchronized HeLa cells. We also silenced Cyclin F expression in synchronized U-2OS and RPE1-hTERT cells using the most effective of the three oligos. In all cases, depletion of Cyclin F inhibited the G2-specific degradation of CP110 (Fig. 2b and Supplementary Fig. 6). Finally, immunopurified wild type Cyclin F, but not Cyclin F(LP/AA), promoted the in vitro ubiquitylation of CP110 (Fig. 2c).

Together, the results in Figs. 1–2 and Supplementary Figs. 1–6 demonstrate that Cyclin F mediates the degradation of CP110 in G2 by forming an active SCF ubiquitin ligase complex.

To study the biological significance of Cyclin F-mediated degradation of CP110, we investigated whether overexpression of Cyclin F would affect centrosome reduplication in cells in which DNA synthesis is inhibited by hydroxyurea16. Therefore, we forced the expression of Cyclin F in U-2OS cells that were subsequently treated for 48 hours with hydroxyurea, inducing a block in S phase [when endogenous protein levels are low (Supplementary Fig. 7) and Cyclin F is rarely localized to the centrosome (Fig. 1b)]. Overexpression of Cyclin F induced its temporal mislocalization to the centrosome (not shown). Cells were analyzed for centrosome number by dual-color, indirect immunofluorescence using an antibody to γ-tubulin and an antibody to Centrin 2 (a centriole marker). Control S-phase-arrested U-2OS cells underwent centrosome overduplication as determined by the presence of more than two γ-tubulin foci and more than four Centrin 2 foci. In contrast, cells expressing high levels of Cyclin F did not display centrosome overduplication. Importantly, co-expression of a CP110 mutant unable to bind Cyclin F reverted the Cyclin F-mediated inhibition of centrosome overduplication (Supplementary Fig. 7), suggesting that Cyclin F acts as an inhibitor of centrosome reduplication via its ability to restrain the expression of CP110.

In a complementary approach, synchronized and siRNA-treated U-2OS cells were analyzed for centrosome defects. At both 9 and 12 hours after release from a G1/S block (when most cells were in G2 and early M, respectively), treatment with siRNAs targeting Cyclin F produced a significant increase in the percentage of cells showing more than two γ-tubulin foci and more than four foci for both Centrin 2 and CP110 (Fig. 3a and Supplementary Fig. 8). Significantly, expression of a wild type, but siRNA-insensitive, Cyclin F rescued the induction of excess CP110 dots. However, expression of a siRNA-resistant Cyclin F(M/A;L/A) failed to rescue the phenotype (Supplementary Fig. 8). Finally, in agreement with the siRNA results, we observed that Cyclin F−/− mouse embryonic fibroblasts (MEFs) displayed CP110 accumulation and increased number of γ-tubulin and Centrin 2 foci compared to Cyclin FFlox/− MEFs (Supplementary Fig. 9).

During mitosis, the assembly of a bipolar spindle is mediated by the centrosomes (from which microtubules nucleate) and ensures the proper segregation of the genetic material. Cells react to centrosome overduplication by centrosome clustering, to prevent the formation of multipolar spindles17–19. However, before centrosome clustering, cells with extra centrosomes pass through transient, multipolar intermediates that promote merotelic and syntelic attachments, with consequent lagging chromosomes20. Therefore, we analyzed mitotic figures using an antibody to α-tubulin (to stain mitotic spindles), an antibody to Centrin 2, and DAPI. Mitotic cells in which Cyclin F levels were reduced showed an increase in both multipolar spindles and asymmetric, bipolar spindles with lagging chromosomes (Fig. 3b and Supplementary Fig. 10). Significantly, when CP110 was silenced together with Cyclin F, the number of Centrin 2/γ-tubulin foci and mitotic aberrations reverted to levels approximating controls (Fig. 3a–b and Supplementary Figs. 10–11), strongly indicating that the G2 accumulation of CP110 in Cyclin F-depleted cells is responsible for the observed phenotypes.

To further investigate the effects produced by the failure to degrade CP110 in G2, we analyzed synchronized U-2OS cells expressing either FLAG-tagged wild-type CP110 or FLAG-tagged CP110(RxL/AxA). Wild-type CP110 was degraded in G2 and M, while CP110(RxL/AxA), in agreement with its inability to bind Cyclin F (Supplementary Fig. 2), was stable (Fig. 4a). Importantly, expression of the stable CP110 mutant recapitulated the centrosome phenotypes observed upon Cyclin F silencing, namely an increased number of foci positive for Centrin 2, CP110, and γ–tubulin (Fig. 4b). Moreover, 100% of the mitotic cells that stained positive for CP110(RxL/AxA) showed abnormal spindles and lagging chromosomes (n= 29/29), whereas only 25% of mitotic cells that did not express CP110(RxL/AxA) showed mitotic aberrations (n=7/28) (Fig. 4c). Finally, expression of CP110(RxL/AxA) promoted the formation of micronuclei, a hallmark of chromosomal instability (Fig. 4d).

Chromosome instability can result from centrosome overduplication18–20. Here, we demonstrate that a defect in the SCFCyclin F-mediated degradation of CP110 results in centrosome and chromosome aberrations. Thus, we propose that Cyclin F plays a role in limiting centrosome duplication to once per cell cycle to maintain chromosome stability.

It is known that the Cyclin E is necessary for centrosome duplication (via an unknown mechanism)21,22. The work presented here indicates that Cyclin F limits centrosome duplication by targeting CP110 for proteolysis. Thus, Cyclin E and Cyclin F have opposing roles, explaining why the former is expressed in S (when centrosome duplication occurs) and the latter in G2 (when centrosome duplication is restricted).

In summary, our study shows that Cyclin F forms an active SCF ubiquitin ligase complex (via its F-box motif), and it binds CP110 using a mode of substrate recognition identical to that used by the canonical cyclins involved in protein phosphorylation (i.e. via the hydrophobic patch present in the cyclin box domain and the RxL motif present in the substrate), thus unifying the functions of the two Cyclin F homology domains. Importantly, we show both temporal and spatial regulation of the SCFCyclin F-mediated proteolysis of CP110 and demonstrate a physiological role for this event in controlling genome integrity.

Methods SummaryBiochemical MethodsExtract preparation, immunoprecipitation, and immunoblotting have previously been described23,24PlasmidsCP110 cDNA was from OpenBioSystems (clone MHS1010-7295919). Cyclin F cDNA was amplified by PCR using a cDNA library from HEK-293T cells. CP110 and Cyclin F mutants were generated either using the QuikChange Site-directed Mutagenesis kit (Stratagene) or by standard PCR methods. pCBF GFP-CP110 WT and CP110 truncation mutants were previously described10,13.Immunofluorescence microscopyFor indirect immunofluorescence staining, cells were grown on glass coverslips and then fixed with methanol for 10 minutes. The cells were permeabilized with PBS/1% Triton X-100 for 10 minutes and blocked for one hour in PBS/0.1% Triton X-100 containing 3% BSA prior to incubation with primary antibodies. Alexa Fluor 568 conjugated goat anti-rabbit and Alexa Fluor 488-conjugated goat anti-mouse IgGs (Invitrogen) were used as secondary antibodies. DAPI was used to counterstain DNA. Slides were mounted with Prolong-Gold (Invitrogen). Image acquisition was performed using a Zeiss Axiovert 200 M microscope (63× objective lens, N.A. 1.4, 1.6× Optovar), equipped with a cooled Retiga 2000R CCD (QImaging). Mitotic images were deconvoluted using Metamorph (Molecular Devices). The images represent the maximum projection of 12 deconvoluted planes. Confocal microscopy (used in Fig. 2 and Supplementary Fig. 4) was performed using a Zeiss LSM 510, equipped with Zeiss LSM 510 software.

Methods Summary

Biochemical MethodsExtract preparation, immunoprecipitation, and immunoblotting have previously been described23,24

Biochemical Methods

Extract preparation, immunoprecipitation, and immunoblotting have previously been described23,24

PlasmidsCP110 cDNA was from OpenBioSystems (clone MHS1010-7295919). Cyclin F cDNA was amplified by PCR using a cDNA library from HEK-293T cells. CP110 and Cyclin F mutants were generated either using the QuikChange Site-directed Mutagenesis kit (Stratagene) or by standard PCR methods. pCBF GFP-CP110 WT and CP110 truncation mutants were previously described10,13.

CP110 cDNA was from OpenBioSystems (clone MHS1010-7295919). Cyclin F cDNA was amplified by PCR using a cDNA library from HEK-293T cells. CP110 and Cyclin F mutants were generated either using the QuikChange Site-directed Mutagenesis kit (Stratagene) or by standard PCR methods. pCBF GFP-CP110 WT and CP110 truncation mutants were previously described10,13.

Immunofluorescence microscopyFor indirect immunofluorescence staining, cells were grown on glass coverslips and then fixed with methanol for 10 minutes. The cells were permeabilized with PBS/1% Triton X-100 for 10 minutes and blocked for one hour in PBS/0.1% Triton X-100 containing 3% BSA prior to incubation with primary antibodies. Alexa Fluor 568 conjugated goat anti-rabbit and Alexa Fluor 488-conjugated goat anti-mouse IgGs (Invitrogen) were used as secondary antibodies. DAPI was used to counterstain DNA. Slides were mounted with Prolong-Gold (Invitrogen). Image acquisition was performed using a Zeiss Axiovert 200 M microscope (63× objective lens, N.A. 1.4, 1.6× Optovar), equipped with a cooled Retiga 2000R CCD (QImaging). Mitotic images were deconvoluted using Metamorph (Molecular Devices). The images represent the maximum projection of 12 deconvoluted planes. Confocal microscopy (used in Fig. 2 and Supplementary Fig. 4) was performed using a Zeiss LSM 510, equipped with Zeiss LSM 510 software.

Immunofluorescence microscopy

For indirect immunofluorescence staining, cells were grown on glass coverslips and then fixed with methanol for 10 minutes. The cells were permeabilized with PBS/1% Triton X-100 for 10 minutes and blocked for one hour in PBS/0.1% Triton X-100 containing 3% BSA prior to incubation with primary antibodies. Alexa Fluor 568 conjugated goat anti-rabbit and Alexa Fluor 488-conjugated goat anti-mouse IgGs (Invitrogen) were used as secondary antibodies. DAPI was used to counterstain DNA. Slides were mounted with Prolong-Gold (Invitrogen). Image acquisition was performed using a Zeiss Axiovert 200 M microscope (63× objective lens, N.A. 1.4, 1.6× Optovar), equipped with a cooled Retiga 2000R CCD (QImaging). Mitotic images were deconvoluted using Metamorph (Molecular Devices). The images represent the maximum projection of 12 deconvoluted planes. Confocal microscopy (used in Fig. 2 and Supplementary Fig. 4) was performed using a Zeiss LSM 510, equipped with Zeiss LSM 510 software.

Supplementary Material12

Supplementary Material
