---
pmid: '21734647'
title: The auxin signalling network translates dynamic input into robust patterning
  at the shoot apex.
authors:
- Vernoux T
- Brunoud G
- Farcot E
- Morin V
- Van den Daele H
- Legrand J
- Oliva M
- Das P
- Larrieu A
- Wells D
- Guédon Y
- Armitage L
- Picard F
- Guyomarc'h S
- Cellier C
- Parry G
- Koumproglou R
- Doonan JH
- Estelle M
- Godin C
- Kepinski S
- Bennett M
- De Veylder L
- Traas J
journal: Mol Syst Biol
year: '2011'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC3167386
doi: 10.1038/msb.2011.39
---

# The auxin signalling network translates dynamic input into robust patterning at the shoot apex.
**Authors:** Vernoux T, Brunoud G, Farcot E, Morin V, Van den Daele H, Legrand J, Oliva M, Das P, Larrieu A, Wells D, Guédon Y, Armitage L, Picard F, Guyomarc'h S, Cellier C, Parry G, Koumproglou R, Doonan JH, Estelle M, Godin C, Kepinski S, Bennett M, De Veylder L, Traas J
**Journal:** Mol Syst Biol (2011)
**DOI:** [10.1038/msb.2011.39](https://doi.org/10.1038/msb.2011.39)
**PMC:** [PMC3167386](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3167386/)

## Abstract

1. Mol Syst Biol. 2011 Jul 5;7:508. doi: 10.1038/msb.2011.39.

The auxin signalling network translates dynamic input into robust patterning at 
the shoot apex.

Vernoux T(1), Brunoud G, Farcot E, Morin V, Van den Daele H, Legrand J, Oliva M, 
Das P, Larrieu A, Wells D, Guédon Y, Armitage L, Picard F, Guyomarc'h S, Cellier 
C, Parry G, Koumproglou R, Doonan JH, Estelle M, Godin C, Kepinski S, Bennett M, 
De Veylder L, Traas J.

Author information:
(1)Laboratoire de Reproduction et Développement des Plantes, CNRS, INRA, ENS 
Lyon, UCBL, Université de Lyon, Lyon, France. teva.vernoux@ens-lyon.fr

Comment in
    Mol Syst Biol. 2011 Jul 05;7:507. doi: 10.1038/msb.2011.45.

The plant hormone auxin is thought to provide positional information for 
patterning during development. It is still unclear, however, precisely how auxin 
is distributed across tissues and how the hormone is sensed in space and time. 
The control of gene expression in response to auxin involves a complex network 
of over 50 potentially interacting transcriptional activators and repressors, 
the auxin response factors (ARFs) and Aux/IAAs. Here, we perform a large-scale 
analysis of the Aux/IAA-ARF pathway in the shoot apex of Arabidopsis, where 
dynamic auxin-based patterning controls organogenesis. A comprehensive 
expression map and full interactome uncovered an unexpectedly simple 
distribution and structure of this pathway in the shoot apex. A mathematical 
model of the Aux/IAA-ARF network predicted a strong buffering capacity along 
with spatial differences in auxin sensitivity. We then tested and confirmed 
these predictions using a novel auxin signalling sensor that reports input into 
the signalling pathway, in conjunction with the published DR5 transcriptional 
output reporter. Our results provide evidence that the auxin signalling network 
is essential to create robust patterns at the shoot apex.

DOI: 10.1038/msb.2011.39
PMCID: PMC3167386
PMID: 21734647 [Indexed for MEDLINE]

Conflict of interest statement: The authors declare that they have no conflict 
of interest.

## Full Text

IntroductionAuxin is a key morphogenetic signal involved in the control of cell identity throughout plant development. A striking example of auxin action is in the regulation of organogenesis at the shoot apical meristem (SAM). The SAM, a population of stem cells generating the aerial parts of the plant (Traas and Doonan, 2001), continuously produces new organs at precise positions at its periphery (Vernoux et al, 2010). The dynamics and robustness of organ positioning and patterning is thought to depend on local accumulations of auxin, generated by the PIN-FORMED1 (PIN1) efflux carrier controlling the direction of auxin polar fluxes (Galweiler et al, 1998; Vernoux et al, 2000; Reinhardt et al, 2003; Heisler et al, 2005; Barbier de Reuille et al, 2006; Jonsson et al, 2006; Smith et al, 2006; Bayer et al, 2009) together with the AUX1/LAX influx carriers (Reinhardt et al, 2003; Bainbridge et al, 2008). While the role of polar auxin transport has received extensive attention in the recent years, both auxin distribution and the contribution of its signal transduction pathway to patterning in the SAM are still largely uncharacterized.A complex ensemble of 29 Aux/IAAs and 23 auxin response factors (ARFs) is central to the regulation of gene transcription in response to auxin (for review, see Leyser, 2006; Guilfoyle and Hagen, 2007; Chapman and Estelle, 2009). Protein–protein interactions govern the properties of this transduction pathway (Del Bianco and Kepinski, 2011). Limited interaction studies suggest that, in the absence of auxin, the Aux/IAA repressors form heterodimers with the ARF transcription factors (for review, see Guilfoyle and Hagen, 2007)) and recruit co-repressors of the TOPLESS (TPL) family, preventing the ARFs from regulating target genes (Szemenyei et al, 2008). In the presence of auxin the Aux/IAA proteins are targeted to the proteasome, by an SCF E3 ubiquitin ligase complex (Chapman and Estelle, 2009; Leyser, 2006). In this process, auxin promotes the interaction between Aux/IAA proteins and the TIR1 F-box of the SCF complex (or its AFB homologues) that acts as an auxin co-receptor (Kepinski and Leyser, 2005; Dharmasiri et al, 2005a, 2005b; Tan et al, 2007). The auxin-induced degradation of Aux/IAAs would then release ARFs to regulate transcription of their target genes. This includes activation of most of the Aux/IAA genes themselves, thus establishing a negative feedback loop (Guilfoyle and Hagen, 2007). Although this general scenario provides a framework for understanding gene regulation by auxin, the underlying protein–protein network remains to be fully characterized. In addition, while auxin predominantly activates transcription (Paponov et al, 2008 and references therein), sequence analysis and transient assays suggest that most of the 23 ARFs act as transcriptional repressors, while only 5 (ARF5, 6, 7, 8 and 19) of them are activators (Guilfoyle and Hagen, 2007), further highlighting the need for an integrated view of this pathway.To understand how the Aux/IAA-ARF pathway contributes to sensing auxin in space and time, we have conducted a large-scale analysis of the Aux/IAA-ARF network in the inflorescence SAM. Our strategy and findings are summarized in the flowchart presented in Figure 1. A comprehensive expression map and full interactome uncovered a relatively simple distribution and structure of this pathway in the shoot apex. Using ordinary differential equation (ODE) modelling, we then predicted spatial differences in auxin sensitivity and a strong buffering capacity of the Aux/IAA-ARF network in the SAM. Last, the development of a novel auxin signalling sensor allowed us to dynamically visualize the input into the signalling pathway (a parameter that depends on both auxin concentration and perception) and to confirm the predictions of our model. Taken together, our data provide evidence that robust patterning at the SAM depends not only on auxin distribution but also on the local properties of the Aux/IAA-ARF signalling network.

Introduction

Auxin is a key morphogenetic signal involved in the control of cell identity throughout plant development. A striking example of auxin action is in the regulation of organogenesis at the shoot apical meristem (SAM). The SAM, a population of stem cells generating the aerial parts of the plant (Traas and Doonan, 2001), continuously produces new organs at precise positions at its periphery (Vernoux et al, 2010). The dynamics and robustness of organ positioning and patterning is thought to depend on local accumulations of auxin, generated by the PIN-FORMED1 (PIN1) efflux carrier controlling the direction of auxin polar fluxes (Galweiler et al, 1998; Vernoux et al, 2000; Reinhardt et al, 2003; Heisler et al, 2005; Barbier de Reuille et al, 2006; Jonsson et al, 2006; Smith et al, 2006; Bayer et al, 2009) together with the AUX1/LAX influx carriers (Reinhardt et al, 2003; Bainbridge et al, 2008). While the role of polar auxin transport has received extensive attention in the recent years, both auxin distribution and the contribution of its signal transduction pathway to patterning in the SAM are still largely uncharacterized.

A complex ensemble of 29 Aux/IAAs and 23 auxin response factors (ARFs) is central to the regulation of gene transcription in response to auxin (for review, see Leyser, 2006; Guilfoyle and Hagen, 2007; Chapman and Estelle, 2009). Protein–protein interactions govern the properties of this transduction pathway (Del Bianco and Kepinski, 2011). Limited interaction studies suggest that, in the absence of auxin, the Aux/IAA repressors form heterodimers with the ARF transcription factors (for review, see Guilfoyle and Hagen, 2007)) and recruit co-repressors of the TOPLESS (TPL) family, preventing the ARFs from regulating target genes (Szemenyei et al, 2008). In the presence of auxin the Aux/IAA proteins are targeted to the proteasome, by an SCF E3 ubiquitin ligase complex (Chapman and Estelle, 2009; Leyser, 2006). In this process, auxin promotes the interaction between Aux/IAA proteins and the TIR1 F-box of the SCF complex (or its AFB homologues) that acts as an auxin co-receptor (Kepinski and Leyser, 2005; Dharmasiri et al, 2005a, 2005b; Tan et al, 2007). The auxin-induced degradation of Aux/IAAs would then release ARFs to regulate transcription of their target genes. This includes activation of most of the Aux/IAA genes themselves, thus establishing a negative feedback loop (Guilfoyle and Hagen, 2007). Although this general scenario provides a framework for understanding gene regulation by auxin, the underlying protein–protein network remains to be fully characterized. In addition, while auxin predominantly activates transcription (Paponov et al, 2008 and references therein), sequence analysis and transient assays suggest that most of the 23 ARFs act as transcriptional repressors, while only 5 (ARF5, 6, 7, 8 and 19) of them are activators (Guilfoyle and Hagen, 2007), further highlighting the need for an integrated view of this pathway.

To understand how the Aux/IAA-ARF pathway contributes to sensing auxin in space and time, we have conducted a large-scale analysis of the Aux/IAA-ARF network in the inflorescence SAM. Our strategy and findings are summarized in the flowchart presented in Figure 1. A comprehensive expression map and full interactome uncovered a relatively simple distribution and structure of this pathway in the shoot apex. Using ordinary differential equation (ODE) modelling, we then predicted spatial differences in auxin sensitivity and a strong buffering capacity of the Aux/IAA-ARF network in the SAM. Last, the development of a novel auxin signalling sensor allowed us to dynamically visualize the input into the signalling pathway (a parameter that depends on both auxin concentration and perception) and to confirm the predictions of our model. Taken together, our data provide evidence that robust patterning at the SAM depends not only on auxin distribution but also on the local properties of the Aux/IAA-ARF signalling network.

ResultsAuxin signalling is spatially regulated in the SAMTo fully understand how auxin signalling is regulated in the SAM, we analysed the expression patterns of the F-box co-receptors as well as of the Aux/IAA and ARF genes. First, we used translational fusions to the GUS reporter gene to study the expression of the AFB1-3 and TIR1 auxin co-receptors in the inflorescence (Figure 2A–C; Supplementary Figure S1A–D for serial sections) (Parry et al, 2009). We found that AFB1 is the most abundant auxin co-receptor (based on GUS activity), homogeneously expressed in the meristem, while AFB3 is expressed in tissues below the meristem. AFB2 is undetectable in the meristem proper. TIR1 is weakly expressed throughout the meristem and shows a reduced expression in the L2 and L3 layers at the centre. Auxin receptor activity of the AFB4 and AFB5 F-box proteins was also recently demonstrated (Greenham et al, 2011). We thus used in situ hybridization to study the expression patterns of the corresponding genes in the SAM. While AFB4 could not be detected at significant levels, we did detect a low expression of AFB5 throughout the meristem that was slightly higher in the L2 and L3 layers of organ primordia (Figure 2D; Supplementary Figure S1E and F). Hence, our results suggest that TIR1, AFB1 and AFB5 control auxin-induced degradation of Aux/IAAs in the inflorescence meristem.We next used systematic RNA in situ hybridization to obtain the patterns of all ARFs and Aux/IAAs (with the exception of ARF15 and 21 for which we could not obtain a cDNA). We found 13 ARFs and 12 Aux/IAAs expressed in the meristem (Figure 2E–ZC; Supplementary Figures S2 and S3 for serial sections). Expression was confirmed using quantitative RT–PCR on total RNA from dissected meristems (Figure 2ZD), except for ARF4 and IAA12 that could not be amplified, as well as IAA2 and 11 that were amplified, but not detected by in situ hybridization. The expression of 2 ARFs (ARF11 and 19: Figure 2O and Q) and 4 Aux/IAAs (IAA20, 26, 29 and 30: Figure 2Y, Z, ZB and ZC) appeared to be restricted to the presumptive vasculature in the organ primordia or just a few cells in the epidermis (see also Supplementary Figures S2 and S3), while ARF9 and 10 showed a weak homogenous expression (Figure 2M and N; Supplementary Figure S2). Most strikingly, all the other ARF and Aux/IAA genes (9 and 8 genes, respectively) were expressed at higher levels at the periphery and at lower levels in the centre of the meristem (Figure 2E–ZC; Supplementary Figures S2 and S3) (Sessions et al, 1997; Hardtke and Berleth, 1998; Pekker et al, 2005; Wu et al, 2006). Among these, ARF1, 2, 5, 7, 8, 18 and IAA12, 13, 18, 27 were expressed homogeneously throughout the periphery of the meristem (although with different intensities), while ARF3, 4, 6 and IAA8, 9 showed an even stronger expression in organ primordia. ARF4 and IAA18 were also expressed higher in the inner core of the meristem, below the stem cells at the centre of the meristem.These in situ hybridization results show that the expression of the Aux/IAA and ARF genes defines five different domains in the meristem (Figure 2ZE). However, the general tendency observed for most of the 25 ARFs and Aux/IAAs detected in the SAM is a differential expression with low levels at the centre of the meristem (where the stem cells are located) and high levels at the periphery of the meristem (where organ initiation takes place). For some of the genes, an even higher expression is observed in the organ primordia. This differential expression of auxin signalling regulators is also paralleled by a lower expression of TIR1 in the inner core of the meristem and the higher expression of AFB5 in the internal part of primordia. Furthermore, our data show that ARF activators and repressors are largely co-expressed, suggesting a role for this co-expression in the regulation of transcription in response to auxin.The Aux/IAA-ARF interaction network has a simple structureTo understand the functional significance of the distribution of ARFs and Aux/IAAs in the SAM, we next investigated the global structure of the Aux/IAA-ARF network using a high-throughput yeast two-hybrid approach. This analysis revealed 433 positive interactions among 1225 tested, indicating a highly connected network (Figure 3A; Supplementary Table S1). A total of 78% (51 in 65) of the interactions tested in the literature were confirmed in our data, indicating a very good coverage (Supplementary Table S2 and references therein). We also confirmed 28 interactions out of 31 (90%) tested in planta using bimolecular fluorescence complementation, further supporting the biological significance of our data (Supplementary Figure S4).To explore the organization of this network, we applied a graph clustering method (Picard et al, 2009) that groups proteins based on their connectivity profile (i.e. proteins with similar interactors). Three well separated clusters, characterized by contrasting probabilities for within- and between-cluster connectivity, were uncovered (Figure 2B; Supplementary Figure S5): proteins in cluster I were strongly and similarly connected to each other and to the proteins of cluster II; proteins of the cluster II were strongly connected to those of cluster I, but sparsely to themselves; finally, proteins from cluster III showed a low connectivity to the rest of the network and to each other. Similar results were obtained when we restricted the analysis to the subset of ARFs and Aux/IAAs present in the SAM, indicating a similar interactome topology (Supplementary Figure S6). Further examination showed that cluster I contained mainly Aux/IAAs, cluster II mainly ARF activators and cluster III ARF repressors (Figure 2C and D; Supplementary Figures S5 and S6). For the SAM-specific network (Figure 2D), the only exceptions were the ARF9 repressor and IAA9, which showed a connectivity profile closer to the ARF activators and were assigned to cluster II.Our results thus indicate that the topology of the whole network, as well as of the SAM-specific network, relies on three principal features (Figure 2E): (i) Aux/IAA proteins interact with themselves, (ii) Aux/IAA proteins interact with ARF activators and (iii) ARF repressors have no or very limited interactions with other proteins in the network.The structure of the Aux/IAA-ARF network provides a plausible model for auxin signallingHaving obtained the structure of the Aux/IAA-ARF network, we next investigated the impact of this structure on transcriptional regulation in response to auxin. The results of our interaction analyses suggest a model for the Aux/IAA-ARF signalling pathway in the SAM, where transcriptional activation by ARF activators would be negatively regulated by two independent systems, one involving the ARF repressors and the other, the Aux/IAAs. The presence of auxin would remove the inhibitory action of Aux/IAAs, but leave the ARF repressors to compete with ARF activators for promoter-binding sites. To explore the regulatory properties of this signalling network involving multiple elementary reactions and a feedback loop, we developed a mathematical model using ODEs.In addition to data on the Aux/IAA-ARF interactome, the model was based on a set of four other general considerations. First, that ARF activators and ARF repressors may regulate the same target genes, since they can bind the same AuxRE element found in the promoters of auxin-induced genes (Ulmasov et al, 1999; Pufky et al, 2003; Goda et al, 2004; Nemhauser et al, 2004). Second, that the promoters of auxin-induced genes have one or two AuxREs, though a few may contain three or more (Goda et al, 2004). Third, that all the Aux/IAA genes expressed in the SAM except IAA16 and 27 are induced by auxin (Paponov et al, 2008 and references therein). Fourth, that among the 23 ARF genes, only ARF4 and ARF19 expression can be induced by auxin (Paponov et al, 2008), thus indicating that expression of the majority of ARFs is independent of auxin.These experimental observations lead to the reaction scheme shown in Figure 4A (a more detailed version of this scheme is given in Figure 1 of Supplementary Note S1), where
the interactions between proteins occur according to Figure 3E;target genes promoter contains two AuxREs;target genes are regulated by both ARF activators and ARF repressors;the target genes include the Aux/IAA genes, but no ARF genes.In this scheme, we also considered three additional assumptions:A cooperative effect occurs when two ARF activators are bound on target gene promoters as suggested by Ulmasov et al (1999).Binding of auxin to TIR1/AFB co-receptors and Aux/IAAs is a faster process than ubiquitination, implying that the effect of auxin on the system can be directly described as an increase of Aux/IAA degradation rate (see Supplementary Note S1 for further details).The auxin-mediated degradation affects the stability of both Aux/IAA monomers and dimers. In the latter case, the partner of the Aux/IAA (ARF activator or Aux/IAA) is released.Concentrations of populations of Aux/IAAs, ARF activators, ARF repressors and of mRNA of auxin-induced genes were then described mathematically using a system of five ODEs (Figure 4A; see Materials and methods for full description of the model). For the analysis of gene transcription in response to auxin, the most important aspect of this mathematical description is that it allows analysing the transcriptional output R as a function of the auxin level x. It is important to stress that, upon assumption (vi), the auxin level x directly modulates the Aux/IAA degradation rate as a function of both auxin concentration and perception of auxin (see Materials and methods). The parameter x can thus be viewed more generally as the input into the signalling pathway and changes in this parameter can reflect either changes in auxin concentrations or changes in the TIR/AFB co-receptors levels.We performed a mathematical and numerical study of the model (detailed in Supplementary Note S1). The main conclusions of this analysis are as follows:
We proved that in a range of plausible parameters, the system always reaches a unique steady state. Simulations performed by varying the different parameters further indicate that this steady state is stable. The effect of modifying the auxin concentration x was to shift the system to a new value of the steady state, in other words to change the level of all variables and notably the transcriptional output R.The model reproduced gene activation in response to an increase in the auxin concentration x for all tested parameters. More generally, all variables of the system displayed stereotypical response curves upon variations in auxin concentration showing the robustness of our model.Both the robustness and the ability to reproduce a biological observation support the plausibility of this model and prompted us to use it for investigating the role of auxin signalling in SAM function.The auxin signalling model allows predicting differential sensitivity to auxin and buffering capacities for the Aux/IAA-ARF pathway in the meristemOur expression study suggested a simplified view of the meristem, where the same population of Aux/IAAs, ARF activators and ARF repressors exhibit a low expression at the centre and a high expression in the peripheral zone (Figure 2). In order to study the significance of such a spatial distribution for auxin-regulated patterning, we used the auxin signalling model described above.As could be intuitively expected, the ARF repressors limit the intensity of target gene induction by ARF activators in our model (Figure 4B; see also Figures 4 and 5 in Supplementary Note S1), which implies that the activation level of transcription in response to auxin (or more generally to an increase in the auxin signalling input) depends on both the absolute levels and the balance between ARF activators and repressors. This can be visualized by a 3D plot representing the gene induction levels in response to auxin as a function of both the absolute levels of ARF proteins (both ARF activators and repressors) and the ratio between ARF activators and repressors (Figure 4C). For a given ARF activator-to-repressor ratio, the gene induction capacity increases with the absolute levels of ARF proteins, although with a slight decrease at the highest concentrations of proteins when this ratio is elevated. Based on these simulations, we propose that, at the periphery of the meristem and notably in the organ primordia, the higher expression of ARF activators might allow for a high capacity of induction of gene transcription in response to auxin, despite the high expression of ARF repressors. On the other hand, the low expression of ARF activators at the centre of the SAM would create a low sensitivity to auxin. However, the fact that the expression of ARF repressors is also reduced at the centre of the SAM might allow these cells to not be completely insensitive to auxin. In addition, the distribution of the TIR1/AFB co-receptors could also contribute to creating a difference in sensitivity between the centre and the periphery of the SAM by reducing it in the L2 and L3 layers at the centre (lower TIR1 expression) and increasing it in the internal tissues of the organ primordia (higher AFB5 expression). In conclusion, we predict that the differential expression of the ARF activators and repressors in the meristem (Figure 2) generates differences in auxin sensitivities between the centre (low sensitivity) and the periphery (high sensitivity), and thus leads to a differential expression of the Aux/IAAs (prediction 1).Next, we sought to use our model to investigate the effects of altered auxin levels on the auxin signalling, since it is known that auxin levels can be modified in response to various environmental factors (Gray et al, 1998; Stepanova et al, 2008; Tao et al, 2008). We analysed the transcriptional response to changing auxin concentrations by simulating sinusoidal variations. These variations in auxin trigger oscillations in the transcriptional level of target genes for all the parameter tested (see Supplementary Note S1). However, we observed that increasing the level of ARF repressors for a fixed level of ARF activators leads to an attenuation of the amplitude of the fluctuations. Decreasing the level of ARF activators for a fixed level of ARF repressors gave a similar result (see Supplementary Note S1). This suggests that the ARF activator-to-repressor ratio can affect the stability of gene expression in response to auxin, or more generally to fluctuations in the auxin signalling input. This result can be related directly to our simplified representation of the SAM by simulating a situation equivalent to the centre of the meristem, where both ARF repressors and activators are present at similarly low levels. This simulation showed a low, but stable induction of target genes when the auxin input varies (‘CZ’ in Figure 4D, middle panel). Likewise, a situation equivalent to the periphery of the meristem with high levels of both ARF activators and ARF repressors (‘PZ’ in Figure 4D, lower panel) also resulted in a relatively stable output. In both cases, stability was perturbed when the balance between positive and negative ARFs was altered (Figure 4D). Increasing ARF activator levels when ARF repressor levels are low (‘CZ + ARF+’ in Figure 4D, middle panel), or decreasing ARF repressor levels when ARF activator levels are high (‘PZ − ARF−‘ in Figure 4D, lower panel) caused strong fluctuations in target gene transcription levels. Thus, we predict that the signalling pathway buffers its response to the auxin input via the balance between ARF activators and repressors, in turn generated by their differential spatial distributions (prediction 2).A novel signalling sensor, DII-VENUS, reports on the auxin signalling input in the meristemTo test the predictions from our model experimentally, we needed to assess both the input (auxin level and/or perception) and the output (target gene induction) of the signalling cascade. For measuring the transcriptional output, the widely used DR5 reporter, which consists of several ARF-binding sites driving the expression of a reporter gene, is perfectly adapted (Figure 5A) (Ulmasov et al, 1997; Sabatini et al, 1999; Benkova et al, 2003; Heisler et al, 2005). For assaying pathway input, given the absence of tools to visualize auxin in situ, we designed a novel auxin signalling sensor that relates more directly to auxin concentrations. This sensor comprises a constitutively expressed fusion of the auxin-binding domain (termed domain II or DII) (Dreher et al, 2006; Tan et al, 2007) of several Aux/IAA proteins (IAA8, 9 and 28: see Materials and methods; Vernoux et al, in preparation) to a fast-maturating variant of YFP, VENUS (Nagai et al, 2002). By design, these fusion proteins should monitor local degradation of Aux/IAAs, and thus the input in the signalling pathway depending on auxin levels and/or on local differences in auxin perception (Figure 5A). Confocal imaging revealed that the strongest fluorescence signal was obtained using a nuclear-targeted VENUS sequence fused to the IAA28 domain II (Vernoux et al, in preparation), which we henceforth refer to DII-VENUS.DII-VENUS fluorescence could be detected in both shoot and root apical tissues, where it showed clear differential distributions (Figure 5B; Supplementary Figure S7A; Vernoux et al, in preparation). Using root tissues, we showed that DII-VENUS abundance is dependent on auxin and the TIR1/AFB1–3 receptors (Vernoux et al, in preparation). We also observed that the disruption of ubiquitin-dependent breakdown of Aux/IAA proteins using a proteasome inhibitor blocks the auxin-induced degradation of DII-VENUS in the SAM (Supplementary Figure S8). In addition, introducing a mutation in domain II of DII-VENUS (mDII-VENUS), which disrupts the interaction between Aux/IAAs and the auxin co-receptors (Tan et al, 2007) largely abolished the differential distribution of fluorescence in the SAM (Figure 5C; Supplementary Figure S7B). We conclude that DII-VENUS abundance is regulated by auxin via TIR1/AFB activities, consistent with the model for Aux/IAA degradation (Chapman and Estelle, 2009).We next tested auxin-dependent binding of the domain II of IAA28 to several TIR/AFB co-receptors, to estimate their relative contributions to DII-VENUS degradation. Pull-down experiments revealed that TIR1, AFB1 and AFB5 exhibited auxin-enhanced binding to domain II of IAA28 (Figure 5E). Similarly to what had been observed for other Aux/IAAs (Parry et al, 2009), the strongest binding was with TIR1, while the binding and the magnitude in the auxin-induced increase in binding with AFB1 appeared lower. Binding to IAA28 domain II was also low for AFB5. Hence, in the SAM, the homogenous expression of AFB1 (Figure 2B) might not be sufficient to ensure a homogenous degradation capacity of DII-VENUS throughout the structure. The lower expression of TIR1 in the L2 and L3 layers at the centre of the meristem (Figure 2A) might significantly diminish DII-VENUS degradation capacity in this part of the meristem, while AFB5 might lead to a more limited increase of this capacity in the internal layers of primordia (Figure 2D). Taken together, our results suggest that DII-VENUS reports auxin signalling input in the SAM. DII-VENUS thus likely provides information on auxin distribution, but the differential of TIR1 and AFB5 needs to be taken into account.Auxin signalling sensors distribution and dynamics confirm the model predictionsThe degradation patterns displayed in the SAM from the DII-VENUS line indicated a high auxin signalling input in flower primordia and a low input in the cells immediately surrounding the primordia, in agreement with the organ-specific expression pattern of DR5::VENUS (Figure 5B and D) (Heisler et al, 2005). A low DII-VENUS signal was also observed at the centre of both inflorescence and vegetative SAM (Figure 5B; Supplementary Figure S7A). DII-VENUS is thus efficiently degraded at the centre of the meristem, despite the lower expression of TIR1 in most cells of this domain. This demonstrates a high auxin signalling input at the centre of the meristem, in contrast to the complete exclusion of DR5::VENUS expression from these cells (Figure 5D), indicating that the signalling pathway limits gene activation in response to auxin at the meristem centre. These results confirm the prediction that ARF distribution in the meristem creates a differential sensitivity to auxin between the centre and the periphery (prediction 1), thus contributing to patterning of the meristem and higher expression of Aux/IAAs at the periphery of the SAM.To test the buffering capacities of the signalling pathway (prediction 2), we next took advantage of the dynamic properties of DII-VENUS by carrying out live imaging experiments to monitor DII-VENUS levels in real time (Figure 5F) (Fernandez et al, 2010). Our initial intention was to perform auxin treatments to test the prediction. However, this turned out to be unnecessary as we observed notable fluctuations in DII-VENUS signal intensities that occurred naturally and without any treatments in the SAM. These variations were most evident in the epidermal (L1) layer in a zone connecting the centre to the sites of initia I1 and I2. This demonstrates the presence of important temporal variations in the overall auxin signalling input, especially in the centre and during the earlier steps of organ initiation. In contrast to DII-VENUS, the auxin signalling output, visualized with DR5::VENUS, did not show such global variations: the fluorescence remained stable in the organs after initiation and no signal was ever detected outside the organs (Figure 5G; see notably I2 and I3). Since the VENUS protein does not appear to be unduly stable (Supplementary Figure S9), we conclude that the stable expression of DR5::VENUS over time reflects a stable output of the signalling pathway. Taken together, our results thus indicate that the Aux/IAA-ARF network in the SAM buffers fluctuations in the auxin signalling input and stabilizes the activation of genes in response to auxin, thus providing biological evidence in support of the second prediction made by our model.

Auxin signalling is spatially regulated in the SAMTo fully understand how auxin signalling is regulated in the SAM, we analysed the expression patterns of the F-box co-receptors as well as of the Aux/IAA and ARF genes. First, we used translational fusions to the GUS reporter gene to study the expression of the AFB1-3 and TIR1 auxin co-receptors in the inflorescence (Figure 2A–C; Supplementary Figure S1A–D for serial sections) (Parry et al, 2009). We found that AFB1 is the most abundant auxin co-receptor (based on GUS activity), homogeneously expressed in the meristem, while AFB3 is expressed in tissues below the meristem. AFB2 is undetectable in the meristem proper. TIR1 is weakly expressed throughout the meristem and shows a reduced expression in the L2 and L3 layers at the centre. Auxin receptor activity of the AFB4 and AFB5 F-box proteins was also recently demonstrated (Greenham et al, 2011). We thus used in situ hybridization to study the expression patterns of the corresponding genes in the SAM. While AFB4 could not be detected at significant levels, we did detect a low expression of AFB5 throughout the meristem that was slightly higher in the L2 and L3 layers of organ primordia (Figure 2D; Supplementary Figure S1E and F). Hence, our results suggest that TIR1, AFB1 and AFB5 control auxin-induced degradation of Aux/IAAs in the inflorescence meristem.We next used systematic RNA in situ hybridization to obtain the patterns of all ARFs and Aux/IAAs (with the exception of ARF15 and 21 for which we could not obtain a cDNA). We found 13 ARFs and 12 Aux/IAAs expressed in the meristem (Figure 2E–ZC; Supplementary Figures S2 and S3 for serial sections). Expression was confirmed using quantitative RT–PCR on total RNA from dissected meristems (Figure 2ZD), except for ARF4 and IAA12 that could not be amplified, as well as IAA2 and 11 that were amplified, but not detected by in situ hybridization. The expression of 2 ARFs (ARF11 and 19: Figure 2O and Q) and 4 Aux/IAAs (IAA20, 26, 29 and 30: Figure 2Y, Z, ZB and ZC) appeared to be restricted to the presumptive vasculature in the organ primordia or just a few cells in the epidermis (see also Supplementary Figures S2 and S3), while ARF9 and 10 showed a weak homogenous expression (Figure 2M and N; Supplementary Figure S2). Most strikingly, all the other ARF and Aux/IAA genes (9 and 8 genes, respectively) were expressed at higher levels at the periphery and at lower levels in the centre of the meristem (Figure 2E–ZC; Supplementary Figures S2 and S3) (Sessions et al, 1997; Hardtke and Berleth, 1998; Pekker et al, 2005; Wu et al, 2006). Among these, ARF1, 2, 5, 7, 8, 18 and IAA12, 13, 18, 27 were expressed homogeneously throughout the periphery of the meristem (although with different intensities), while ARF3, 4, 6 and IAA8, 9 showed an even stronger expression in organ primordia. ARF4 and IAA18 were also expressed higher in the inner core of the meristem, below the stem cells at the centre of the meristem.These in situ hybridization results show that the expression of the Aux/IAA and ARF genes defines five different domains in the meristem (Figure 2ZE). However, the general tendency observed for most of the 25 ARFs and Aux/IAAs detected in the SAM is a differential expression with low levels at the centre of the meristem (where the stem cells are located) and high levels at the periphery of the meristem (where organ initiation takes place). For some of the genes, an even higher expression is observed in the organ primordia. This differential expression of auxin signalling regulators is also paralleled by a lower expression of TIR1 in the inner core of the meristem and the higher expression of AFB5 in the internal part of primordia. Furthermore, our data show that ARF activators and repressors are largely co-expressed, suggesting a role for this co-expression in the regulation of transcription in response to auxin.

Auxin signalling is spatially regulated in the SAM

To fully understand how auxin signalling is regulated in the SAM, we analysed the expression patterns of the F-box co-receptors as well as of the Aux/IAA and ARF genes. First, we used translational fusions to the GUS reporter gene to study the expression of the AFB1-3 and TIR1 auxin co-receptors in the inflorescence (Figure 2A–C; Supplementary Figure S1A–D for serial sections) (Parry et al, 2009). We found that AFB1 is the most abundant auxin co-receptor (based on GUS activity), homogeneously expressed in the meristem, while AFB3 is expressed in tissues below the meristem. AFB2 is undetectable in the meristem proper. TIR1 is weakly expressed throughout the meristem and shows a reduced expression in the L2 and L3 layers at the centre. Auxin receptor activity of the AFB4 and AFB5 F-box proteins was also recently demonstrated (Greenham et al, 2011). We thus used in situ hybridization to study the expression patterns of the corresponding genes in the SAM. While AFB4 could not be detected at significant levels, we did detect a low expression of AFB5 throughout the meristem that was slightly higher in the L2 and L3 layers of organ primordia (Figure 2D; Supplementary Figure S1E and F). Hence, our results suggest that TIR1, AFB1 and AFB5 control auxin-induced degradation of Aux/IAAs in the inflorescence meristem.

We next used systematic RNA in situ hybridization to obtain the patterns of all ARFs and Aux/IAAs (with the exception of ARF15 and 21 for which we could not obtain a cDNA). We found 13 ARFs and 12 Aux/IAAs expressed in the meristem (Figure 2E–ZC; Supplementary Figures S2 and S3 for serial sections). Expression was confirmed using quantitative RT–PCR on total RNA from dissected meristems (Figure 2ZD), except for ARF4 and IAA12 that could not be amplified, as well as IAA2 and 11 that were amplified, but not detected by in situ hybridization. The expression of 2 ARFs (ARF11 and 19: Figure 2O and Q) and 4 Aux/IAAs (IAA20, 26, 29 and 30: Figure 2Y, Z, ZB and ZC) appeared to be restricted to the presumptive vasculature in the organ primordia or just a few cells in the epidermis (see also Supplementary Figures S2 and S3), while ARF9 and 10 showed a weak homogenous expression (Figure 2M and N; Supplementary Figure S2). Most strikingly, all the other ARF and Aux/IAA genes (9 and 8 genes, respectively) were expressed at higher levels at the periphery and at lower levels in the centre of the meristem (Figure 2E–ZC; Supplementary Figures S2 and S3) (Sessions et al, 1997; Hardtke and Berleth, 1998; Pekker et al, 2005; Wu et al, 2006). Among these, ARF1, 2, 5, 7, 8, 18 and IAA12, 13, 18, 27 were expressed homogeneously throughout the periphery of the meristem (although with different intensities), while ARF3, 4, 6 and IAA8, 9 showed an even stronger expression in organ primordia. ARF4 and IAA18 were also expressed higher in the inner core of the meristem, below the stem cells at the centre of the meristem.

These in situ hybridization results show that the expression of the Aux/IAA and ARF genes defines five different domains in the meristem (Figure 2ZE). However, the general tendency observed for most of the 25 ARFs and Aux/IAAs detected in the SAM is a differential expression with low levels at the centre of the meristem (where the stem cells are located) and high levels at the periphery of the meristem (where organ initiation takes place). For some of the genes, an even higher expression is observed in the organ primordia. This differential expression of auxin signalling regulators is also paralleled by a lower expression of TIR1 in the inner core of the meristem and the higher expression of AFB5 in the internal part of primordia. Furthermore, our data show that ARF activators and repressors are largely co-expressed, suggesting a role for this co-expression in the regulation of transcription in response to auxin.

The Aux/IAA-ARF interaction network has a simple structureTo understand the functional significance of the distribution of ARFs and Aux/IAAs in the SAM, we next investigated the global structure of the Aux/IAA-ARF network using a high-throughput yeast two-hybrid approach. This analysis revealed 433 positive interactions among 1225 tested, indicating a highly connected network (Figure 3A; Supplementary Table S1). A total of 78% (51 in 65) of the interactions tested in the literature were confirmed in our data, indicating a very good coverage (Supplementary Table S2 and references therein). We also confirmed 28 interactions out of 31 (90%) tested in planta using bimolecular fluorescence complementation, further supporting the biological significance of our data (Supplementary Figure S4).To explore the organization of this network, we applied a graph clustering method (Picard et al, 2009) that groups proteins based on their connectivity profile (i.e. proteins with similar interactors). Three well separated clusters, characterized by contrasting probabilities for within- and between-cluster connectivity, were uncovered (Figure 2B; Supplementary Figure S5): proteins in cluster I were strongly and similarly connected to each other and to the proteins of cluster II; proteins of the cluster II were strongly connected to those of cluster I, but sparsely to themselves; finally, proteins from cluster III showed a low connectivity to the rest of the network and to each other. Similar results were obtained when we restricted the analysis to the subset of ARFs and Aux/IAAs present in the SAM, indicating a similar interactome topology (Supplementary Figure S6). Further examination showed that cluster I contained mainly Aux/IAAs, cluster II mainly ARF activators and cluster III ARF repressors (Figure 2C and D; Supplementary Figures S5 and S6). For the SAM-specific network (Figure 2D), the only exceptions were the ARF9 repressor and IAA9, which showed a connectivity profile closer to the ARF activators and were assigned to cluster II.Our results thus indicate that the topology of the whole network, as well as of the SAM-specific network, relies on three principal features (Figure 2E): (i) Aux/IAA proteins interact with themselves, (ii) Aux/IAA proteins interact with ARF activators and (iii) ARF repressors have no or very limited interactions with other proteins in the network.

The Aux/IAA-ARF interaction network has a simple structure

To understand the functional significance of the distribution of ARFs and Aux/IAAs in the SAM, we next investigated the global structure of the Aux/IAA-ARF network using a high-throughput yeast two-hybrid approach. This analysis revealed 433 positive interactions among 1225 tested, indicating a highly connected network (Figure 3A; Supplementary Table S1). A total of 78% (51 in 65) of the interactions tested in the literature were confirmed in our data, indicating a very good coverage (Supplementary Table S2 and references therein). We also confirmed 28 interactions out of 31 (90%) tested in planta using bimolecular fluorescence complementation, further supporting the biological significance of our data (Supplementary Figure S4).

To explore the organization of this network, we applied a graph clustering method (Picard et al, 2009) that groups proteins based on their connectivity profile (i.e. proteins with similar interactors). Three well separated clusters, characterized by contrasting probabilities for within- and between-cluster connectivity, were uncovered (Figure 2B; Supplementary Figure S5): proteins in cluster I were strongly and similarly connected to each other and to the proteins of cluster II; proteins of the cluster II were strongly connected to those of cluster I, but sparsely to themselves; finally, proteins from cluster III showed a low connectivity to the rest of the network and to each other. Similar results were obtained when we restricted the analysis to the subset of ARFs and Aux/IAAs present in the SAM, indicating a similar interactome topology (Supplementary Figure S6). Further examination showed that cluster I contained mainly Aux/IAAs, cluster II mainly ARF activators and cluster III ARF repressors (Figure 2C and D; Supplementary Figures S5 and S6). For the SAM-specific network (Figure 2D), the only exceptions were the ARF9 repressor and IAA9, which showed a connectivity profile closer to the ARF activators and were assigned to cluster II.

Our results thus indicate that the topology of the whole network, as well as of the SAM-specific network, relies on three principal features (Figure 2E): (i) Aux/IAA proteins interact with themselves, (ii) Aux/IAA proteins interact with ARF activators and (iii) ARF repressors have no or very limited interactions with other proteins in the network.

The structure of the Aux/IAA-ARF network provides a plausible model for auxin signallingHaving obtained the structure of the Aux/IAA-ARF network, we next investigated the impact of this structure on transcriptional regulation in response to auxin. The results of our interaction analyses suggest a model for the Aux/IAA-ARF signalling pathway in the SAM, where transcriptional activation by ARF activators would be negatively regulated by two independent systems, one involving the ARF repressors and the other, the Aux/IAAs. The presence of auxin would remove the inhibitory action of Aux/IAAs, but leave the ARF repressors to compete with ARF activators for promoter-binding sites. To explore the regulatory properties of this signalling network involving multiple elementary reactions and a feedback loop, we developed a mathematical model using ODEs.In addition to data on the Aux/IAA-ARF interactome, the model was based on a set of four other general considerations. First, that ARF activators and ARF repressors may regulate the same target genes, since they can bind the same AuxRE element found in the promoters of auxin-induced genes (Ulmasov et al, 1999; Pufky et al, 2003; Goda et al, 2004; Nemhauser et al, 2004). Second, that the promoters of auxin-induced genes have one or two AuxREs, though a few may contain three or more (Goda et al, 2004). Third, that all the Aux/IAA genes expressed in the SAM except IAA16 and 27 are induced by auxin (Paponov et al, 2008 and references therein). Fourth, that among the 23 ARF genes, only ARF4 and ARF19 expression can be induced by auxin (Paponov et al, 2008), thus indicating that expression of the majority of ARFs is independent of auxin.These experimental observations lead to the reaction scheme shown in Figure 4A (a more detailed version of this scheme is given in Figure 1 of Supplementary Note S1), where
the interactions between proteins occur according to Figure 3E;target genes promoter contains two AuxREs;target genes are regulated by both ARF activators and ARF repressors;the target genes include the Aux/IAA genes, but no ARF genes.In this scheme, we also considered three additional assumptions:A cooperative effect occurs when two ARF activators are bound on target gene promoters as suggested by Ulmasov et al (1999).Binding of auxin to TIR1/AFB co-receptors and Aux/IAAs is a faster process than ubiquitination, implying that the effect of auxin on the system can be directly described as an increase of Aux/IAA degradation rate (see Supplementary Note S1 for further details).The auxin-mediated degradation affects the stability of both Aux/IAA monomers and dimers. In the latter case, the partner of the Aux/IAA (ARF activator or Aux/IAA) is released.Concentrations of populations of Aux/IAAs, ARF activators, ARF repressors and of mRNA of auxin-induced genes were then described mathematically using a system of five ODEs (Figure 4A; see Materials and methods for full description of the model). For the analysis of gene transcription in response to auxin, the most important aspect of this mathematical description is that it allows analysing the transcriptional output R as a function of the auxin level x. It is important to stress that, upon assumption (vi), the auxin level x directly modulates the Aux/IAA degradation rate as a function of both auxin concentration and perception of auxin (see Materials and methods). The parameter x can thus be viewed more generally as the input into the signalling pathway and changes in this parameter can reflect either changes in auxin concentrations or changes in the TIR/AFB co-receptors levels.We performed a mathematical and numerical study of the model (detailed in Supplementary Note S1). The main conclusions of this analysis are as follows:
We proved that in a range of plausible parameters, the system always reaches a unique steady state. Simulations performed by varying the different parameters further indicate that this steady state is stable. The effect of modifying the auxin concentration x was to shift the system to a new value of the steady state, in other words to change the level of all variables and notably the transcriptional output R.The model reproduced gene activation in response to an increase in the auxin concentration x for all tested parameters. More generally, all variables of the system displayed stereotypical response curves upon variations in auxin concentration showing the robustness of our model.Both the robustness and the ability to reproduce a biological observation support the plausibility of this model and prompted us to use it for investigating the role of auxin signalling in SAM function.

The structure of the Aux/IAA-ARF network provides a plausible model for auxin signalling

Having obtained the structure of the Aux/IAA-ARF network, we next investigated the impact of this structure on transcriptional regulation in response to auxin. The results of our interaction analyses suggest a model for the Aux/IAA-ARF signalling pathway in the SAM, where transcriptional activation by ARF activators would be negatively regulated by two independent systems, one involving the ARF repressors and the other, the Aux/IAAs. The presence of auxin would remove the inhibitory action of Aux/IAAs, but leave the ARF repressors to compete with ARF activators for promoter-binding sites. To explore the regulatory properties of this signalling network involving multiple elementary reactions and a feedback loop, we developed a mathematical model using ODEs.

In addition to data on the Aux/IAA-ARF interactome, the model was based on a set of four other general considerations. First, that ARF activators and ARF repressors may regulate the same target genes, since they can bind the same AuxRE element found in the promoters of auxin-induced genes (Ulmasov et al, 1999; Pufky et al, 2003; Goda et al, 2004; Nemhauser et al, 2004). Second, that the promoters of auxin-induced genes have one or two AuxREs, though a few may contain three or more (Goda et al, 2004). Third, that all the Aux/IAA genes expressed in the SAM except IAA16 and 27 are induced by auxin (Paponov et al, 2008 and references therein). Fourth, that among the 23 ARF genes, only ARF4 and ARF19 expression can be induced by auxin (Paponov et al, 2008), thus indicating that expression of the majority of ARFs is independent of auxin.

These experimental observations lead to the reaction scheme shown in Figure 4A (a more detailed version of this scheme is given in Figure 1 of Supplementary Note S1), where
the interactions between proteins occur according to Figure 3E;target genes promoter contains two AuxREs;target genes are regulated by both ARF activators and ARF repressors;the target genes include the Aux/IAA genes, but no ARF genes.In this scheme, we also considered three additional assumptions:A cooperative effect occurs when two ARF activators are bound on target gene promoters as suggested by Ulmasov et al (1999).Binding of auxin to TIR1/AFB co-receptors and Aux/IAAs is a faster process than ubiquitination, implying that the effect of auxin on the system can be directly described as an increase of Aux/IAA degradation rate (see Supplementary Note S1 for further details).The auxin-mediated degradation affects the stability of both Aux/IAA monomers and dimers. In the latter case, the partner of the Aux/IAA (ARF activator or Aux/IAA) is released.

the interactions between proteins occur according to Figure 3E;

target genes promoter contains two AuxREs;

target genes are regulated by both ARF activators and ARF repressors;

the target genes include the Aux/IAA genes, but no ARF genes.In this scheme, we also considered three additional assumptions:

A cooperative effect occurs when two ARF activators are bound on target gene promoters as suggested by Ulmasov et al (1999).

Binding of auxin to TIR1/AFB co-receptors and Aux/IAAs is a faster process than ubiquitination, implying that the effect of auxin on the system can be directly described as an increase of Aux/IAA degradation rate (see Supplementary Note S1 for further details).

The auxin-mediated degradation affects the stability of both Aux/IAA monomers and dimers. In the latter case, the partner of the Aux/IAA (ARF activator or Aux/IAA) is released.

Concentrations of populations of Aux/IAAs, ARF activators, ARF repressors and of mRNA of auxin-induced genes were then described mathematically using a system of five ODEs (Figure 4A; see Materials and methods for full description of the model). For the analysis of gene transcription in response to auxin, the most important aspect of this mathematical description is that it allows analysing the transcriptional output R as a function of the auxin level x. It is important to stress that, upon assumption (vi), the auxin level x directly modulates the Aux/IAA degradation rate as a function of both auxin concentration and perception of auxin (see Materials and methods). The parameter x can thus be viewed more generally as the input into the signalling pathway and changes in this parameter can reflect either changes in auxin concentrations or changes in the TIR/AFB co-receptors levels.

We performed a mathematical and numerical study of the model (detailed in Supplementary Note S1). The main conclusions of this analysis are as follows:
We proved that in a range of plausible parameters, the system always reaches a unique steady state. Simulations performed by varying the different parameters further indicate that this steady state is stable. The effect of modifying the auxin concentration x was to shift the system to a new value of the steady state, in other words to change the level of all variables and notably the transcriptional output R.The model reproduced gene activation in response to an increase in the auxin concentration x for all tested parameters. More generally, all variables of the system displayed stereotypical response curves upon variations in auxin concentration showing the robustness of our model.

We proved that in a range of plausible parameters, the system always reaches a unique steady state. Simulations performed by varying the different parameters further indicate that this steady state is stable. The effect of modifying the auxin concentration x was to shift the system to a new value of the steady state, in other words to change the level of all variables and notably the transcriptional output R.

The model reproduced gene activation in response to an increase in the auxin concentration x for all tested parameters. More generally, all variables of the system displayed stereotypical response curves upon variations in auxin concentration showing the robustness of our model.

Both the robustness and the ability to reproduce a biological observation support the plausibility of this model and prompted us to use it for investigating the role of auxin signalling in SAM function.

The auxin signalling model allows predicting differential sensitivity to auxin and buffering capacities for the Aux/IAA-ARF pathway in the meristemOur expression study suggested a simplified view of the meristem, where the same population of Aux/IAAs, ARF activators and ARF repressors exhibit a low expression at the centre and a high expression in the peripheral zone (Figure 2). In order to study the significance of such a spatial distribution for auxin-regulated patterning, we used the auxin signalling model described above.As could be intuitively expected, the ARF repressors limit the intensity of target gene induction by ARF activators in our model (Figure 4B; see also Figures 4 and 5 in Supplementary Note S1), which implies that the activation level of transcription in response to auxin (or more generally to an increase in the auxin signalling input) depends on both the absolute levels and the balance between ARF activators and repressors. This can be visualized by a 3D plot representing the gene induction levels in response to auxin as a function of both the absolute levels of ARF proteins (both ARF activators and repressors) and the ratio between ARF activators and repressors (Figure 4C). For a given ARF activator-to-repressor ratio, the gene induction capacity increases with the absolute levels of ARF proteins, although with a slight decrease at the highest concentrations of proteins when this ratio is elevated. Based on these simulations, we propose that, at the periphery of the meristem and notably in the organ primordia, the higher expression of ARF activators might allow for a high capacity of induction of gene transcription in response to auxin, despite the high expression of ARF repressors. On the other hand, the low expression of ARF activators at the centre of the SAM would create a low sensitivity to auxin. However, the fact that the expression of ARF repressors is also reduced at the centre of the SAM might allow these cells to not be completely insensitive to auxin. In addition, the distribution of the TIR1/AFB co-receptors could also contribute to creating a difference in sensitivity between the centre and the periphery of the SAM by reducing it in the L2 and L3 layers at the centre (lower TIR1 expression) and increasing it in the internal tissues of the organ primordia (higher AFB5 expression). In conclusion, we predict that the differential expression of the ARF activators and repressors in the meristem (Figure 2) generates differences in auxin sensitivities between the centre (low sensitivity) and the periphery (high sensitivity), and thus leads to a differential expression of the Aux/IAAs (prediction 1).Next, we sought to use our model to investigate the effects of altered auxin levels on the auxin signalling, since it is known that auxin levels can be modified in response to various environmental factors (Gray et al, 1998; Stepanova et al, 2008; Tao et al, 2008). We analysed the transcriptional response to changing auxin concentrations by simulating sinusoidal variations. These variations in auxin trigger oscillations in the transcriptional level of target genes for all the parameter tested (see Supplementary Note S1). However, we observed that increasing the level of ARF repressors for a fixed level of ARF activators leads to an attenuation of the amplitude of the fluctuations. Decreasing the level of ARF activators for a fixed level of ARF repressors gave a similar result (see Supplementary Note S1). This suggests that the ARF activator-to-repressor ratio can affect the stability of gene expression in response to auxin, or more generally to fluctuations in the auxin signalling input. This result can be related directly to our simplified representation of the SAM by simulating a situation equivalent to the centre of the meristem, where both ARF repressors and activators are present at similarly low levels. This simulation showed a low, but stable induction of target genes when the auxin input varies (‘CZ’ in Figure 4D, middle panel). Likewise, a situation equivalent to the periphery of the meristem with high levels of both ARF activators and ARF repressors (‘PZ’ in Figure 4D, lower panel) also resulted in a relatively stable output. In both cases, stability was perturbed when the balance between positive and negative ARFs was altered (Figure 4D). Increasing ARF activator levels when ARF repressor levels are low (‘CZ + ARF+’ in Figure 4D, middle panel), or decreasing ARF repressor levels when ARF activator levels are high (‘PZ − ARF−‘ in Figure 4D, lower panel) caused strong fluctuations in target gene transcription levels. Thus, we predict that the signalling pathway buffers its response to the auxin input via the balance between ARF activators and repressors, in turn generated by their differential spatial distributions (prediction 2).

The auxin signalling model allows predicting differential sensitivity to auxin and buffering capacities for the Aux/IAA-ARF pathway in the meristem

Our expression study suggested a simplified view of the meristem, where the same population of Aux/IAAs, ARF activators and ARF repressors exhibit a low expression at the centre and a high expression in the peripheral zone (Figure 2). In order to study the significance of such a spatial distribution for auxin-regulated patterning, we used the auxin signalling model described above.

As could be intuitively expected, the ARF repressors limit the intensity of target gene induction by ARF activators in our model (Figure 4B; see also Figures 4 and 5 in Supplementary Note S1), which implies that the activation level of transcription in response to auxin (or more generally to an increase in the auxin signalling input) depends on both the absolute levels and the balance between ARF activators and repressors. This can be visualized by a 3D plot representing the gene induction levels in response to auxin as a function of both the absolute levels of ARF proteins (both ARF activators and repressors) and the ratio between ARF activators and repressors (Figure 4C). For a given ARF activator-to-repressor ratio, the gene induction capacity increases with the absolute levels of ARF proteins, although with a slight decrease at the highest concentrations of proteins when this ratio is elevated. Based on these simulations, we propose that, at the periphery of the meristem and notably in the organ primordia, the higher expression of ARF activators might allow for a high capacity of induction of gene transcription in response to auxin, despite the high expression of ARF repressors. On the other hand, the low expression of ARF activators at the centre of the SAM would create a low sensitivity to auxin. However, the fact that the expression of ARF repressors is also reduced at the centre of the SAM might allow these cells to not be completely insensitive to auxin. In addition, the distribution of the TIR1/AFB co-receptors could also contribute to creating a difference in sensitivity between the centre and the periphery of the SAM by reducing it in the L2 and L3 layers at the centre (lower TIR1 expression) and increasing it in the internal tissues of the organ primordia (higher AFB5 expression). In conclusion, we predict that the differential expression of the ARF activators and repressors in the meristem (Figure 2) generates differences in auxin sensitivities between the centre (low sensitivity) and the periphery (high sensitivity), and thus leads to a differential expression of the Aux/IAAs (prediction 1).

Next, we sought to use our model to investigate the effects of altered auxin levels on the auxin signalling, since it is known that auxin levels can be modified in response to various environmental factors (Gray et al, 1998; Stepanova et al, 2008; Tao et al, 2008). We analysed the transcriptional response to changing auxin concentrations by simulating sinusoidal variations. These variations in auxin trigger oscillations in the transcriptional level of target genes for all the parameter tested (see Supplementary Note S1). However, we observed that increasing the level of ARF repressors for a fixed level of ARF activators leads to an attenuation of the amplitude of the fluctuations. Decreasing the level of ARF activators for a fixed level of ARF repressors gave a similar result (see Supplementary Note S1). This suggests that the ARF activator-to-repressor ratio can affect the stability of gene expression in response to auxin, or more generally to fluctuations in the auxin signalling input. This result can be related directly to our simplified representation of the SAM by simulating a situation equivalent to the centre of the meristem, where both ARF repressors and activators are present at similarly low levels. This simulation showed a low, but stable induction of target genes when the auxin input varies (‘CZ’ in Figure 4D, middle panel). Likewise, a situation equivalent to the periphery of the meristem with high levels of both ARF activators and ARF repressors (‘PZ’ in Figure 4D, lower panel) also resulted in a relatively stable output. In both cases, stability was perturbed when the balance between positive and negative ARFs was altered (Figure 4D). Increasing ARF activator levels when ARF repressor levels are low (‘CZ + ARF+’ in Figure 4D, middle panel), or decreasing ARF repressor levels when ARF activator levels are high (‘PZ − ARF−‘ in Figure 4D, lower panel) caused strong fluctuations in target gene transcription levels. Thus, we predict that the signalling pathway buffers its response to the auxin input via the balance between ARF activators and repressors, in turn generated by their differential spatial distributions (prediction 2).

A novel signalling sensor, DII-VENUS, reports on the auxin signalling input in the meristemTo test the predictions from our model experimentally, we needed to assess both the input (auxin level and/or perception) and the output (target gene induction) of the signalling cascade. For measuring the transcriptional output, the widely used DR5 reporter, which consists of several ARF-binding sites driving the expression of a reporter gene, is perfectly adapted (Figure 5A) (Ulmasov et al, 1997; Sabatini et al, 1999; Benkova et al, 2003; Heisler et al, 2005). For assaying pathway input, given the absence of tools to visualize auxin in situ, we designed a novel auxin signalling sensor that relates more directly to auxin concentrations. This sensor comprises a constitutively expressed fusion of the auxin-binding domain (termed domain II or DII) (Dreher et al, 2006; Tan et al, 2007) of several Aux/IAA proteins (IAA8, 9 and 28: see Materials and methods; Vernoux et al, in preparation) to a fast-maturating variant of YFP, VENUS (Nagai et al, 2002). By design, these fusion proteins should monitor local degradation of Aux/IAAs, and thus the input in the signalling pathway depending on auxin levels and/or on local differences in auxin perception (Figure 5A). Confocal imaging revealed that the strongest fluorescence signal was obtained using a nuclear-targeted VENUS sequence fused to the IAA28 domain II (Vernoux et al, in preparation), which we henceforth refer to DII-VENUS.DII-VENUS fluorescence could be detected in both shoot and root apical tissues, where it showed clear differential distributions (Figure 5B; Supplementary Figure S7A; Vernoux et al, in preparation). Using root tissues, we showed that DII-VENUS abundance is dependent on auxin and the TIR1/AFB1–3 receptors (Vernoux et al, in preparation). We also observed that the disruption of ubiquitin-dependent breakdown of Aux/IAA proteins using a proteasome inhibitor blocks the auxin-induced degradation of DII-VENUS in the SAM (Supplementary Figure S8). In addition, introducing a mutation in domain II of DII-VENUS (mDII-VENUS), which disrupts the interaction between Aux/IAAs and the auxin co-receptors (Tan et al, 2007) largely abolished the differential distribution of fluorescence in the SAM (Figure 5C; Supplementary Figure S7B). We conclude that DII-VENUS abundance is regulated by auxin via TIR1/AFB activities, consistent with the model for Aux/IAA degradation (Chapman and Estelle, 2009).We next tested auxin-dependent binding of the domain II of IAA28 to several TIR/AFB co-receptors, to estimate their relative contributions to DII-VENUS degradation. Pull-down experiments revealed that TIR1, AFB1 and AFB5 exhibited auxin-enhanced binding to domain II of IAA28 (Figure 5E). Similarly to what had been observed for other Aux/IAAs (Parry et al, 2009), the strongest binding was with TIR1, while the binding and the magnitude in the auxin-induced increase in binding with AFB1 appeared lower. Binding to IAA28 domain II was also low for AFB5. Hence, in the SAM, the homogenous expression of AFB1 (Figure 2B) might not be sufficient to ensure a homogenous degradation capacity of DII-VENUS throughout the structure. The lower expression of TIR1 in the L2 and L3 layers at the centre of the meristem (Figure 2A) might significantly diminish DII-VENUS degradation capacity in this part of the meristem, while AFB5 might lead to a more limited increase of this capacity in the internal layers of primordia (Figure 2D). Taken together, our results suggest that DII-VENUS reports auxin signalling input in the SAM. DII-VENUS thus likely provides information on auxin distribution, but the differential of TIR1 and AFB5 needs to be taken into account.

A novel signalling sensor, DII-VENUS, reports on the auxin signalling input in the meristem

To test the predictions from our model experimentally, we needed to assess both the input (auxin level and/or perception) and the output (target gene induction) of the signalling cascade. For measuring the transcriptional output, the widely used DR5 reporter, which consists of several ARF-binding sites driving the expression of a reporter gene, is perfectly adapted (Figure 5A) (Ulmasov et al, 1997; Sabatini et al, 1999; Benkova et al, 2003; Heisler et al, 2005). For assaying pathway input, given the absence of tools to visualize auxin in situ, we designed a novel auxin signalling sensor that relates more directly to auxin concentrations. This sensor comprises a constitutively expressed fusion of the auxin-binding domain (termed domain II or DII) (Dreher et al, 2006; Tan et al, 2007) of several Aux/IAA proteins (IAA8, 9 and 28: see Materials and methods; Vernoux et al, in preparation) to a fast-maturating variant of YFP, VENUS (Nagai et al, 2002). By design, these fusion proteins should monitor local degradation of Aux/IAAs, and thus the input in the signalling pathway depending on auxin levels and/or on local differences in auxin perception (Figure 5A). Confocal imaging revealed that the strongest fluorescence signal was obtained using a nuclear-targeted VENUS sequence fused to the IAA28 domain II (Vernoux et al, in preparation), which we henceforth refer to DII-VENUS.

DII-VENUS fluorescence could be detected in both shoot and root apical tissues, where it showed clear differential distributions (Figure 5B; Supplementary Figure S7A; Vernoux et al, in preparation). Using root tissues, we showed that DII-VENUS abundance is dependent on auxin and the TIR1/AFB1–3 receptors (Vernoux et al, in preparation). We also observed that the disruption of ubiquitin-dependent breakdown of Aux/IAA proteins using a proteasome inhibitor blocks the auxin-induced degradation of DII-VENUS in the SAM (Supplementary Figure S8). In addition, introducing a mutation in domain II of DII-VENUS (mDII-VENUS), which disrupts the interaction between Aux/IAAs and the auxin co-receptors (Tan et al, 2007) largely abolished the differential distribution of fluorescence in the SAM (Figure 5C; Supplementary Figure S7B). We conclude that DII-VENUS abundance is regulated by auxin via TIR1/AFB activities, consistent with the model for Aux/IAA degradation (Chapman and Estelle, 2009).

We next tested auxin-dependent binding of the domain II of IAA28 to several TIR/AFB co-receptors, to estimate their relative contributions to DII-VENUS degradation. Pull-down experiments revealed that TIR1, AFB1 and AFB5 exhibited auxin-enhanced binding to domain II of IAA28 (Figure 5E). Similarly to what had been observed for other Aux/IAAs (Parry et al, 2009), the strongest binding was with TIR1, while the binding and the magnitude in the auxin-induced increase in binding with AFB1 appeared lower. Binding to IAA28 domain II was also low for AFB5. Hence, in the SAM, the homogenous expression of AFB1 (Figure 2B) might not be sufficient to ensure a homogenous degradation capacity of DII-VENUS throughout the structure. The lower expression of TIR1 in the L2 and L3 layers at the centre of the meristem (Figure 2A) might significantly diminish DII-VENUS degradation capacity in this part of the meristem, while AFB5 might lead to a more limited increase of this capacity in the internal layers of primordia (Figure 2D). Taken together, our results suggest that DII-VENUS reports auxin signalling input in the SAM. DII-VENUS thus likely provides information on auxin distribution, but the differential of TIR1 and AFB5 needs to be taken into account.

Auxin signalling sensors distribution and dynamics confirm the model predictionsThe degradation patterns displayed in the SAM from the DII-VENUS line indicated a high auxin signalling input in flower primordia and a low input in the cells immediately surrounding the primordia, in agreement with the organ-specific expression pattern of DR5::VENUS (Figure 5B and D) (Heisler et al, 2005). A low DII-VENUS signal was also observed at the centre of both inflorescence and vegetative SAM (Figure 5B; Supplementary Figure S7A). DII-VENUS is thus efficiently degraded at the centre of the meristem, despite the lower expression of TIR1 in most cells of this domain. This demonstrates a high auxin signalling input at the centre of the meristem, in contrast to the complete exclusion of DR5::VENUS expression from these cells (Figure 5D), indicating that the signalling pathway limits gene activation in response to auxin at the meristem centre. These results confirm the prediction that ARF distribution in the meristem creates a differential sensitivity to auxin between the centre and the periphery (prediction 1), thus contributing to patterning of the meristem and higher expression of Aux/IAAs at the periphery of the SAM.To test the buffering capacities of the signalling pathway (prediction 2), we next took advantage of the dynamic properties of DII-VENUS by carrying out live imaging experiments to monitor DII-VENUS levels in real time (Figure 5F) (Fernandez et al, 2010). Our initial intention was to perform auxin treatments to test the prediction. However, this turned out to be unnecessary as we observed notable fluctuations in DII-VENUS signal intensities that occurred naturally and without any treatments in the SAM. These variations were most evident in the epidermal (L1) layer in a zone connecting the centre to the sites of initia I1 and I2. This demonstrates the presence of important temporal variations in the overall auxin signalling input, especially in the centre and during the earlier steps of organ initiation. In contrast to DII-VENUS, the auxin signalling output, visualized with DR5::VENUS, did not show such global variations: the fluorescence remained stable in the organs after initiation and no signal was ever detected outside the organs (Figure 5G; see notably I2 and I3). Since the VENUS protein does not appear to be unduly stable (Supplementary Figure S9), we conclude that the stable expression of DR5::VENUS over time reflects a stable output of the signalling pathway. Taken together, our results thus indicate that the Aux/IAA-ARF network in the SAM buffers fluctuations in the auxin signalling input and stabilizes the activation of genes in response to auxin, thus providing biological evidence in support of the second prediction made by our model.

Auxin signalling sensors distribution and dynamics confirm the model predictions

The degradation patterns displayed in the SAM from the DII-VENUS line indicated a high auxin signalling input in flower primordia and a low input in the cells immediately surrounding the primordia, in agreement with the organ-specific expression pattern of DR5::VENUS (Figure 5B and D) (Heisler et al, 2005). A low DII-VENUS signal was also observed at the centre of both inflorescence and vegetative SAM (Figure 5B; Supplementary Figure S7A). DII-VENUS is thus efficiently degraded at the centre of the meristem, despite the lower expression of TIR1 in most cells of this domain. This demonstrates a high auxin signalling input at the centre of the meristem, in contrast to the complete exclusion of DR5::VENUS expression from these cells (Figure 5D), indicating that the signalling pathway limits gene activation in response to auxin at the meristem centre. These results confirm the prediction that ARF distribution in the meristem creates a differential sensitivity to auxin between the centre and the periphery (prediction 1), thus contributing to patterning of the meristem and higher expression of Aux/IAAs at the periphery of the SAM.

To test the buffering capacities of the signalling pathway (prediction 2), we next took advantage of the dynamic properties of DII-VENUS by carrying out live imaging experiments to monitor DII-VENUS levels in real time (Figure 5F) (Fernandez et al, 2010). Our initial intention was to perform auxin treatments to test the prediction. However, this turned out to be unnecessary as we observed notable fluctuations in DII-VENUS signal intensities that occurred naturally and without any treatments in the SAM. These variations were most evident in the epidermal (L1) layer in a zone connecting the centre to the sites of initia I1 and I2. This demonstrates the presence of important temporal variations in the overall auxin signalling input, especially in the centre and during the earlier steps of organ initiation. In contrast to DII-VENUS, the auxin signalling output, visualized with DR5::VENUS, did not show such global variations: the fluorescence remained stable in the organs after initiation and no signal was ever detected outside the organs (Figure 5G; see notably I2 and I3). Since the VENUS protein does not appear to be unduly stable (Supplementary Figure S9), we conclude that the stable expression of DR5::VENUS over time reflects a stable output of the signalling pathway. Taken together, our results thus indicate that the Aux/IAA-ARF network in the SAM buffers fluctuations in the auxin signalling input and stabilizes the activation of genes in response to auxin, thus providing biological evidence in support of the second prediction made by our model.

DiscussionIt has been proposed that the plant could modulate auxin signalling during development by combining subsets of regulators in specific domains of tissues (Weijers et al, 2005; Muto et al, 2007; De Rybel et al, 2010; De Smet et al, 2010). Our results, showing that ARF and Aux/IAAs are generally expressed at a lower level at the meristem centre and at a higher level at the periphery (Figure 2), lead to an alternative scenario for the SAM, where the network is at least partially regulated by alterations in global expression levels. By combining expression data with the interactome, mathematical modelling and two auxin-related markers (Figure 1), we provide evidence that this simple mode of regulation helps to establish a differential sensitivity between the centre and the periphery of the SAM that would be instrumental in translating dynamic auxin distributions into robust patterns.A notable exception to this general regulatory principle at the SAM seems to be the provascular tissue, where specific expression of most notably IAA20 and 30 could provide different signalling properties to the cells (Figure 2). It has been demonstrated that IAA20 and 30 lack a conserved domain II and are likely insensitive to auxin (Dreher et al, 2006; Sato and Yamamoto, 2008). They could thus locally diminish auxin sensitivity during the patterning of the provascular tissue. Further analyses of the corresponding mutants will likely help understand how these two non-canonical Aux/IAAs contribute to the development of the vasculature in the shoot apex.In contrast to most Aux/IAAs, the expression of 10 of the 12 ARF active in the SAM is independent of auxin (Paponov et al, 2008). This indicates that the distribution of ARFs is most likely the primary factor controlling auxin sensitivity in the SAM and could be regulated by auxin-independent patterning factors. Recent evidence indicating that modifying the activities of two stem cell regulators, CLAVATA3 and WUSCHEL, can affect the expression level of the auxin-inducible DR5 reporter supports this hypothesis (Yadav et al, 2010), although the precise link between these patterning genes and auxin signal transduction remains to be established.We have also identified a differential regulation of the TIR1/AFB co-receptors in the SAM (Figure 2). Genetic evidence indicates that the TIR1/AFB genes act redundantly in the regulation of auxin responses, although with different contributions (Dharmasiri et al, 2005b; Parry et al, 2009). Previous work has shown that the Aux/IAA proteins IAA3 and IAA7 exhibit a stronger interaction with TIR1 than with AFB1 (Parry et al, 2009) Our analysis of the interactions between TIR1/AFBs and the domain II of IAA28 (Figure 5) are consistent with these observations (Dharmasiri et al, 2005b; Parry et al, 2009) and supports the conclusion that at least one important component of the differences in TIR1/AFB co-receptor function relates to a broad variation in the affinity of their interactions with Aux/IAAs. Taken together, this suggests that each of the F-box co-receptors present in the SAM might be able to interact with the Aux/IAAs with a different affinity: high affinity for TIR1 and lower affinity for AFB1 and AFB5. Although further analysis will be needed to confirm these ideas, spatial changes in one of the TIR1/AFBs would, therefore, affect globally the capacity to degrade Aux/IAAs. The distribution of TIR1/AFB co-receptors in the SAM would thus be expected to participate in the control of auxin sensitivity in parallel with the ARFs by lowering the Aux/IAA degradation capacity at the centre (lower expression of TIR1), while increasing it in the organs (higher expression of AFB5).In our theoretical analysis of auxin signalling in the SAM (Figure 4), we use a model that appears to be the simplest possible interpretation of our current understanding of auxin signal transduction. It is largely based on the rather simple topology of the network revealed by the Aux/IAA-ARF interactome (Figure 3). In this network, the vast majority of the Aux/IAAs interact with all the ARF activators, suggesting that most Aux/IAAs have the potential to act as repressors of the transcriptional activity of ARF activators. In addition, the limited connectivity of ARF repressors suggests that the role of these transcription factors is essentially auxin independent and that they might simply compete with the ARF activators for binding to the promoter of auxin-inducible genes. Whereas this general scenario most likely applies to the SAM, it is important to point out that more specific interactions might affect the dynamics of the ARF-Aux/IAA signalling pathway elsewhere in the plant. For instance, ARF9 is the only repressor that interacts with a high number of Aux/IAAs, thus suggesting a different mode of action for this ARF compared with the other repressors. Its weak homogenous expression in the SAM suggests that it likely does not have a dominant role in the SAM. However, it might be dominant in other tissues and identification of the local network together with simulations using a modified version of our model would help unravelling a putative function in these tissues. A limited number of interactions between the other ARF repressors and some of the Aux/IAAs have also been detected, even in the SAM-specific network. It is thus possible that such interactions participate in regulating the activity of the auxin signalling network, but further analysis will be necessary to explore this possibility and understand its functional significance.The development of the DII-VENUS sensor was instrumental in confirming the predictions made by our model (Figure 5). Although the level of DII-VENUS depends on both absolute auxin concentrations and the levels of the TIR/AFB co-receptors, given the TIR1/AFB expression patterns, DII-VENUS levels can be used to estimate relative auxin levels in the SAM. Both TIR1 and AFB1 appear to be homogenous throughout the periphery of the SAM, while AFB5 is expressed specifically in the internal tissue of the primordia (Figure 2). Thus, given the low affinity of AFB5 for IAA28, the degradation of the DII-VENUS sensor in the organ primordia and its accumulation in the organ frontiers likely indicate a high concentration of auxin in the organ primordia and a low concentration in the surrounding cells. In the centre of the SAM, the lower TIR1 expression in the internal tissues should lead to less degradation of the DII-VENUS sensor and thus to an under-estimation of the auxin parameter. The low level of DII-VENUS in these cells is thus most likely due to a high auxin concentration at the centre of the SAM. Whereas high concentrations of auxin at young organ primordia have been predicted, for instance based on the high activity of the DR5 promoter (Benkova et al, 2003; Reinhardt et al, 2003; Heisler et al, 2005; Barbier de Reuille et al, 2006; Smith et al, 2006), the presence of auxin at the meristem summit was still a matter of debate. Low DR5 activity in the central zone seemed to indicate low auxin levels (Smith et al, 2006), but this was contradicted by the patterns of PIN transporters at the SAM, suggesting that auxin was preferentially transported to the meristem centre (Barbier de Reuille et al, 2006). Here, we provide a simple explanation for this apparent discrepancy, showing that significant amounts of auxin do accumulate at the meristem centre, but without resulting in a high transcriptional output due to the low sensitivity there.In conclusion, our work supports a key role for local auxin signalling in the regulation of patterning at the SAM. It also provides a plausible auxin distribution map in the structure that will need to be taken into account when further exploring the role of auxin in the SAM. In this context, it will be also important to further analyse the mechanisms controlling DII-VENUS level fluctuations in the SAM that might be linked to changes in auxin concentration and to explore a potential physiological role for these fluctuations.

It has been proposed that the plant could modulate auxin signalling during development by combining subsets of regulators in specific domains of tissues (Weijers et al, 2005; Muto et al, 2007; De Rybel et al, 2010; De Smet et al, 2010). Our results, showing that ARF and Aux/IAAs are generally expressed at a lower level at the meristem centre and at a higher level at the periphery (Figure 2), lead to an alternative scenario for the SAM, where the network is at least partially regulated by alterations in global expression levels. By combining expression data with the interactome, mathematical modelling and two auxin-related markers (Figure 1), we provide evidence that this simple mode of regulation helps to establish a differential sensitivity between the centre and the periphery of the SAM that would be instrumental in translating dynamic auxin distributions into robust patterns.

A notable exception to this general regulatory principle at the SAM seems to be the provascular tissue, where specific expression of most notably IAA20 and 30 could provide different signalling properties to the cells (Figure 2). It has been demonstrated that IAA20 and 30 lack a conserved domain II and are likely insensitive to auxin (Dreher et al, 2006; Sato and Yamamoto, 2008). They could thus locally diminish auxin sensitivity during the patterning of the provascular tissue. Further analyses of the corresponding mutants will likely help understand how these two non-canonical Aux/IAAs contribute to the development of the vasculature in the shoot apex.

In contrast to most Aux/IAAs, the expression of 10 of the 12 ARF active in the SAM is independent of auxin (Paponov et al, 2008). This indicates that the distribution of ARFs is most likely the primary factor controlling auxin sensitivity in the SAM and could be regulated by auxin-independent patterning factors. Recent evidence indicating that modifying the activities of two stem cell regulators, CLAVATA3 and WUSCHEL, can affect the expression level of the auxin-inducible DR5 reporter supports this hypothesis (Yadav et al, 2010), although the precise link between these patterning genes and auxin signal transduction remains to be established.

We have also identified a differential regulation of the TIR1/AFB co-receptors in the SAM (Figure 2). Genetic evidence indicates that the TIR1/AFB genes act redundantly in the regulation of auxin responses, although with different contributions (Dharmasiri et al, 2005b; Parry et al, 2009). Previous work has shown that the Aux/IAA proteins IAA3 and IAA7 exhibit a stronger interaction with TIR1 than with AFB1 (Parry et al, 2009) Our analysis of the interactions between TIR1/AFBs and the domain II of IAA28 (Figure 5) are consistent with these observations (Dharmasiri et al, 2005b; Parry et al, 2009) and supports the conclusion that at least one important component of the differences in TIR1/AFB co-receptor function relates to a broad variation in the affinity of their interactions with Aux/IAAs. Taken together, this suggests that each of the F-box co-receptors present in the SAM might be able to interact with the Aux/IAAs with a different affinity: high affinity for TIR1 and lower affinity for AFB1 and AFB5. Although further analysis will be needed to confirm these ideas, spatial changes in one of the TIR1/AFBs would, therefore, affect globally the capacity to degrade Aux/IAAs. The distribution of TIR1/AFB co-receptors in the SAM would thus be expected to participate in the control of auxin sensitivity in parallel with the ARFs by lowering the Aux/IAA degradation capacity at the centre (lower expression of TIR1), while increasing it in the organs (higher expression of AFB5).

In our theoretical analysis of auxin signalling in the SAM (Figure 4), we use a model that appears to be the simplest possible interpretation of our current understanding of auxin signal transduction. It is largely based on the rather simple topology of the network revealed by the Aux/IAA-ARF interactome (Figure 3). In this network, the vast majority of the Aux/IAAs interact with all the ARF activators, suggesting that most Aux/IAAs have the potential to act as repressors of the transcriptional activity of ARF activators. In addition, the limited connectivity of ARF repressors suggests that the role of these transcription factors is essentially auxin independent and that they might simply compete with the ARF activators for binding to the promoter of auxin-inducible genes. Whereas this general scenario most likely applies to the SAM, it is important to point out that more specific interactions might affect the dynamics of the ARF-Aux/IAA signalling pathway elsewhere in the plant. For instance, ARF9 is the only repressor that interacts with a high number of Aux/IAAs, thus suggesting a different mode of action for this ARF compared with the other repressors. Its weak homogenous expression in the SAM suggests that it likely does not have a dominant role in the SAM. However, it might be dominant in other tissues and identification of the local network together with simulations using a modified version of our model would help unravelling a putative function in these tissues. A limited number of interactions between the other ARF repressors and some of the Aux/IAAs have also been detected, even in the SAM-specific network. It is thus possible that such interactions participate in regulating the activity of the auxin signalling network, but further analysis will be necessary to explore this possibility and understand its functional significance.

The development of the DII-VENUS sensor was instrumental in confirming the predictions made by our model (Figure 5). Although the level of DII-VENUS depends on both absolute auxin concentrations and the levels of the TIR/AFB co-receptors, given the TIR1/AFB expression patterns, DII-VENUS levels can be used to estimate relative auxin levels in the SAM. Both TIR1 and AFB1 appear to be homogenous throughout the periphery of the SAM, while AFB5 is expressed specifically in the internal tissue of the primordia (Figure 2). Thus, given the low affinity of AFB5 for IAA28, the degradation of the DII-VENUS sensor in the organ primordia and its accumulation in the organ frontiers likely indicate a high concentration of auxin in the organ primordia and a low concentration in the surrounding cells. In the centre of the SAM, the lower TIR1 expression in the internal tissues should lead to less degradation of the DII-VENUS sensor and thus to an under-estimation of the auxin parameter. The low level of DII-VENUS in these cells is thus most likely due to a high auxin concentration at the centre of the SAM. Whereas high concentrations of auxin at young organ primordia have been predicted, for instance based on the high activity of the DR5 promoter (Benkova et al, 2003; Reinhardt et al, 2003; Heisler et al, 2005; Barbier de Reuille et al, 2006; Smith et al, 2006), the presence of auxin at the meristem summit was still a matter of debate. Low DR5 activity in the central zone seemed to indicate low auxin levels (Smith et al, 2006), but this was contradicted by the patterns of PIN transporters at the SAM, suggesting that auxin was preferentially transported to the meristem centre (Barbier de Reuille et al, 2006). Here, we provide a simple explanation for this apparent discrepancy, showing that significant amounts of auxin do accumulate at the meristem centre, but without resulting in a high transcriptional output due to the low sensitivity there.

In conclusion, our work supports a key role for local auxin signalling in the regulation of patterning at the SAM. It also provides a plausible auxin distribution map in the structure that will need to be taken into account when further exploring the role of auxin in the SAM. In this context, it will be also important to further analyse the mechanisms controlling DII-VENUS level fluctuations in the SAM that might be linked to changes in auxin concentration and to explore a potential physiological role for these fluctuations.

Materials and methodsPlant material, growth conditions and plant treatmentsAll transgenic plants (see below) were generated in the Columbia (Col-0) ecotype of Arabidopsis thaliana. Plants expressing translational fusion of TIR1, AFB1, AFB2 and AFB3 to GUS have been described (Dharmasiri et al, 2005b; Parry et al, 2009). For the live imaging, plants were grown on soil at 20°C in short-day conditions (8 h light/16 h darkness) for 4 weeks before being transferred in long-day conditions (16 h light/8 h darkness). The chemical treatments were done by immerging dissected inflorescence apices into liquid MS supplemented with the chemicals. Indole-acetic acid (IAA; Sigma) was dissolved in ethanol and used at the indicated final concentration. MG132 was dissolved in DMSO and used at the final concentration of 50 μM for 2h30. For MG132/IAA co-treatments plants were pre-treated with MG132 for 1h30 before adding IAA and waiting an extra hour.Analysis of gene expressionRNA in situ hybridization was performed as described (Vernoux et al, 2000) with at least three independent experiments for each probe tested. Analysis of GUS expression in the inflorescence meristem was done according to Vernoux et al (2000).The RT–qPCR was performed on a step one plus cycler (Applied Biosystems) using the SYBR green reagent kit (Roche). mRNA were extracted from around 100 dissected inflorescence meristems (with all the flowers older than the P1 stage removed) from soil-grown plants. The primers used are from Czechowski et al (2004) except for ARF8, IAA29 and IAA32 (Supplementary Table S3). Expression of several ARF genes could not be assessed due to non-reliable amplification (ARF1, 9, 15, 20, 21, 22 and 23). Expression of the TCTP gene (Supplementary Table S3) was used as a standard and calculations were as described (Pfaffl, 2001). Expressions were analysed on two independent mRNA extractions.Confocal microscopy and live imagingLive imaging of living SAMs was performed as described previously (Fernandez et al, 2010). Observations were done either on an LSM-510 laser-scanning confocal microscope (Zeiss, Jena, Germany) or a SP5 spectral detection confocal Microscope (Leica, Germany).Generation of plasmids and transgenic plantsThe DR5::VENUS plasmid has been described in Heisler et al (2005). The DII-VENUS and mDII-VENUS binary vectors were generated using Gateway technology and following the Multisite Gateway three-fragment vector construction kit instructions (Invitrogen). Full details of the cloning are given elsewhere (Vernoux et al, in preparation). Briefly, we used the degron region of IAA8, 9 and 28 starting from the conserved lysine to the end of domain II (IAA8: 107–178, IAA9: 120–195, IAA28: 28–61) (Dreher et al, 2006). To generate mDII-VENUS, we introduced the previously described P53L mutation (Rogg et al, 2001) in the wild-type IAA28 degron sequence. These sequences were then fused in frame to VENUS tagged with the N7 nuclear localization signal (Heisler et al, 2005) and put under the control of the strong constitutive 35S promoter. The different plasmids were then introduced in plants by floral dipping (Clough and Bent, 1998).For the plasmids used for generating the pull-down data (see below), a plant expression vector containing a 3xFLAG tag was created by annealing the 3xFLAG-F and the 3xFLAG-R oligonucleotides (Supplementary Table S3) and cloned into a pFP101 binary vector (digested with HindIII and BamHI to remove the 2x35SPro). A Gateway® Cassette B (Invitrogen) was cloned in the Klenow filled-in HindIII site flanking the 3xFLAG fragment to form the pFPGW-3xFLAG vector.The coding sequences for TIR1, AFB1 and AFB5 (obtained from ABRC) were PCR amplified to add Gateway attB sites (Supplementary Table S3) and recombined into pDONR P5-P2 Gateway entry vector (Invitrogen). The TIR1/AFBs were then recombined in a multi-gene Gateway reaction with pDONR P1P5r (Invitrogen) containing the cauliflower mosaic virus 35S promoter (CaMV35S) into the pFPGW-3xFLAG vector, in frame with the 3xFLAG tag, to create the plasmids pFPGW-35S-TIR1/AFB-3xFLAG. The TIR1/AFB-3xFLAG sequence (without the promoter) was PCR amplified from the pFPGW-35S–TIR1/AFB-3xFLAG plasmid by using oligonucleotides AsiSIAFB1, AsiSITIR1 and FLAGPmeI (Supplementary Table S3) and cloned as an AsiSI-PmeI fragment into the pF3A WG (BYDV) Flexi® vector (Promega).A 6xHIS tag was fused at the N-terminus of the ASK1 coding sequence by PCR using oligonucleotide ASK1PmeI successively with the nested primers HISASK1 and AsiSIHIS, and cloned as an AsiSI-PmeI fragment into the pF3A WG vector (generating the pF3A WG ASK1 vector).The 35S:AFB5:3xFLAG (AFB5-FLAG) transgenic line used for pull-down assays from plant extract (see below) was created by first transforming pFPGW-35S-AFB5-3xFLAG into Agrobacterium tumefaciens strain GV3101 (Koncz and Schell, 1986) by electroporation. Transgenic plants were generated using the floral dip method (Clough and Bent, 1998) and transformants were selected by seed coat fluorescence. In the T2 generation, lines showing a 3:1 ratio of seed coat fluorescent:non-fluorescent plants were selected for further study. Homozygous lines were selected from the T3 generation.In vitro transcription/translation of TIR1/AFB-tagged proteins, immunoprecipitations and pull-down assaysFor pull-down assays with TIR1-FLAG and AFB1-FLAG expressed in wheat germ extract, the pFPGW-35S-TIR1/AFB-3xFLAG and pF3A WG ASK1 plasmids were used with the TnT SP6 high-yield wheat germ protein expression system (Promega) to synthesize TIR1/AFB–3xFLAG co-expressed with HIS-ASK1 by in vitro transcription/translation in accordance with the manufacturer's instructions.Pull-down assays with wheat germ expressed TIR1/AFB–3xFLAG were performed by combining 22 μl of IVVT reaction extract with 6.5 μg of biotinylated IAA28 domain II peptide (biotinyl-NH-EVAPVVGWPPVRSSRRN-COOH, synthesized by Thermo Scientific), 70 μl 50% streptavidin-agarose suspension and 440 μl of extraction buffer (EB; 0.15 M NaCl, 0.5% nonidet P40, 0.1 M Tris–HCl pH 7.5, containing 1 mM phenylmethylsulphonyl fluoride, 1 μM dithiothreitol, 10 μM MG132 and 1 mg/ml BSA (Sigma) with auxin treatments as indicated. The assays were incubated for 1 h at 4°C with mixing then washed three times for 5 min in EB containing the auxin treatment.For pull-down assays with plant-expressed AFB5-FLAG, extracts of 10-day-old AFB5-FLAG seedling were made as described previously (Kepinski and Leyser, 2005) and used in pull-down assays by combining 2.5 mg of crude extract with 6.5 μg of biotinylated IAA28 domain II peptide, 70 μl 50% streptavidin-agarose suspension. The assays were incubated for 1 h at 4°C with mixing, then washed three times for 5 min in EB containing the auxin treatment.The final processing of all pull-down assays including electrophoresis and western transfer were performed as described previously (Kepinski and Leyser, 2005). The immunodetection of TIR1/AFB-FLAG was performed with a 1:10 000 dilution of anti-FLAG 2-peroxidase (HRP) antibody (Sigma) followed by chemiluminescent detection with ECL plus reagents (Amersham).Protein interaction analysesFor generating the different constructs for the interaction analyses, full-length cDNA for all the Aux/IAAs and ARFs (except ARF15 and ARF21) were either obtained from ABRC for the majority of them or RIKEN (for ARF4) or cloned by RT–PCR from cDNA libraries obtained from various tissues (for ARF6, ARF14, IAA8, IAA9 and IAA28). Gateway technology was then used to generate the different constructs following the multisite gateway three-fragment vector construction kit instructions (Invitrogen).For the Y2H analysis, full-length cDNAs were used for all the Aux/IAA and ARF3, 13 and 17 (ATG to Stop). Partial cDNAs encoding domain III and IV were used for the other ARFs and corresponded to the following region of the proteins—ARF1: 538-Stop; ARF2: 727-Stop; ARF4: 660-Stop; ARF5: 788-Stop; ARF6: 790-Stop; ARF7: 1030-Stop; ARF8: 699-Stop; ARF9: 519-Stop; ARF10: 575-Stop; ARF11: 500-Stop; ARF12: 500-Stop; ARF14: 497-Stop; ARF16: 570-Stop; ARF18: 482-Stop; ARF19: 948-Stop; ARF20: 484-Stop; ARF22: 498-Stop. The cDNAs or parts of cDNA were cloned directionally in pENTR/D-Topo (Invitrogen). They were then systematically transferred by recombination to a gateway-compatible pACT2-based vector (downstream of Gal4-AD) and a pGBKT7-based vector (downstream of Gal4-BD). The Y2H screen was performed using the mating method in a microtiter plate format as described (Boruc et al, 2010). Briefly, each interaction was tested in the two directions (each protein was used alternatively as a bait or as a prey) and using two independent reporters (LacZ and HIS3). LacZ activity was detected visually in the presence of 5-bromo-4-chloro-3-indolyl-β-D-galactopyranoside (X-gal; Blue staining). HIS3 activity was detected by the restoration of growth in the absence of histidine (see below for details on the analysis of the Y2H data). We also tested expression in yeast for all the constructs using standard western blot analysis with an anti-HA antibody (clone 3F10, Roche) for the fusions to Gal4-AD and a Gal4-BD monoclonal antibody (Clontech) for the fusions to Gal4-BD. We confirmed expression for all the Gal4-BD fusions, but a few Gal4-AD fusion were very weak or could not be detected, indicating that the number of interactors might be underestimated for ARF9, ARF17, IAA11, IAA20 and IAA30.For BiFC analysis, we tested various interactions (see Supplementary Figure S4) between ARF5, ARF6, ARF9, ARF18 and/or the following stabilized Aux/IAAs: IAA8 P172S, IAA12 P74S, IAA17 P88L, IAA18 P101S and IAA28 P53L. These mutations have been described for IAA12, 17 and 28 (Rouse et al, 1998; Rogg et al, 2001; Hamann et al, 2002) and similar mutations were introduced in IAA8 and 18. The full-length cDNAs were fused in frame to YFP molecule halves and interactions were tested by transient expression in Nicotiana benthamina leaves as described (Desprez et al, 2007). Empty vectors were used as a negative control. Each interaction was tested independently two to four times and similar leaves areas were scanned for the different tests. The protein interactions from this publication have been submitted to the IMEx (http://imex.sf.net) consortium through IntAct (pmid 19850723) and assigned the identifier IM-15409.Cluster analysis of Aux/IAA-ARF interaction networkConstruction of the interaction network on the basis of the yeast two-hybrid dataThe output of the X-gal test took the form of a mark chosen from among the following ordered marks: –; +; ++; +++. The output of the HIS3 test took the form of an optical density (OD) ratio reflecting the efficiency of recovery (ratio of the OD in the absence of histidine to the OD in the presence of histidine). The Spearman's rank correlation coefficient between the outputs of the X-gal and HIS3 tests was approximately 0.7, suggesting a partial but good agreement between these two tests. We thus chose to build a decision rule that exploits the redundancy between the X-gal and HIS3 tests and the two possible configurations for a given pair of proteins. This decision rule involved two thresholds, one for the X-gal and another for the HIS3 test. In order to determine these thresholds, we analysed the empirical distributions of the OD ratio (HIS3 test output) for each possible X-gal test output using Gaussian mixtures estimated with the mclust R package (http://www.stat.washington.edu/fraley/mclust). In particular, the empirical OD ratio distribution for the ‘+’ X-gal test mark was well fitted by a two-component Gaussian mixture, where the component of lowest mean was interpreted as false positives. This interpretation was supported by the fact that the Gaussian component of lowest mean almost disappeared in the empirical OD ratio distribution for the ‘++’ and ‘+++’ X-gal test marks (i.e. for higher stringency of the X-gal test). On the contrary, it was over-represented for the ‘−’ mark. Based on this statistical analysis, the thresholds were fixed between ‘−’ and ‘+’ for the X-gal test and at 0.45 for the HIS3 test. We finally defined the following decision rule in order to minimize false positives: there is interaction between proteins if at least an X-gal test and an HIS3 test are positive.Cluster analysisStructuring connectivity patterns were uncovered using a probabilistic clustering method implemented in the MixNet software (Daudin et al, 2008; Picard et al, 2009). The key feature of the MixNet model is to give a probabilistic summary of the connectivity structure by uncovering clusters of proteins that share the same connectivity profiles. Briefly, instead of directly describing the clustered structure of vertices, the MixNet model describes the topology of the network using connectivity probabilities πqℓ that is the probability for a vertex from class q to be connected with a vertex from class ℓ. The protein interaction network was modelled as a random graph with (Xij;i, j=1,…N) representing its adjacency matrix, such that Xij=1 if vertices i and j are connected and 0 otherwise. The idea of the MixNet model is to consider that vertices can be spread into Q connectivity clusters that are hidden, with Q being unknown as well. The parameters of this model are the proportions of each cluster α=(αq;q=1,…Q), and the connectivity probability matrix Π=(πqℓ;q, ℓ=1,…,Q). To this extent, Π is a summary of the connectivity of the protein interaction network at the cluster level. For a given number of clusters Q, the outputs of the MixNet algorithm are the estimated model parameters αˆ and Πˆ and the probabilities of assignment of vertices to clusters (τiq;i=1, K, N;q=1, K, Q). The posterior distribution (τiq;q=1,…Q) represents the probabilistic assignment of vertex i to the clusters.Analysis of the adequacy of the clusteringWe assessed the adequacy of the clustering obtained by the MixNet algorithm by evaluating the separability of the clusters and the dispersion of the proteins within the clusters. Since, in our case, the assignment of proteins to clusters is almost deterministic (i.e. τiq≈1 for a unique cluster q and τiℓ≈0 for ℓ≠q), this assignment can be viewed as a partition. The model parameters (πqℓ), which parameterized the edges of the graph, cannot be used directly to define dispersion measures of the vertices assigned to a given cluster. We thus used the edges incident to the vertices to derive dissimilarity measures for the vertices using the adjacency information. The Sokal–Michener distance between vertices i and j defined as Dij=ΣkI(xik≠xjk)/N, where I( ) denotes the indicator function, is the proportion of mismatches or disagreement between the ith and jth rows of the adjacency matrix. This distance naturally expresses the difference in connectivity profiles between vertices.The distance between vertex i and cluster q is then given by

If the vertices are deterministically assigned to a given cluster, this distance simplifies to

where nq is the number of elements of cluster q.The distance between cluster q and cluster ℓ is given by

If the vertices are deterministically assigned to a given cluster, this distance simplifies to

Graphical representation of proteins preserving pairwise distances between themWe applied a multidimensional scaling (MDS) method (Hastie et al, 2005) to visualize the proteins. Briefly, the MDS approach allowed us to find a low-dimensional projection of the data such as to preserve, as closely as possible, the pairwise distances between data points {D(i, j);i,j=1,…,N}.Modelling of the Aux/IAA-ARF signalling pathwayModel descriptionExcept for gene transcription, we assumed that all reactions involved in this network are well described using mass action law kinetics. Variables are denoted using upper case characters, and parameters with lower case. Let I and A denote the concentrations of the IAA and ARF proteins, respectively, and DII and DIA the concentration of the dimers formed by these proteins. R denotes the total concentration of mRNA that is transcribed from a pool of auxin responsive genes. This set of genes includes Aux/IAAs, so that I has a production rate proportional to R. Auxin is represented by the variable x appearing in decay rates involving Aux/IAA proteins. The system can be written as follows:

where h is the transcription rate of the target genes, which depends on the levels of the different transcription factors. This function was determined based on thermodynamic assumptions, as described for example in (Bintu et al, 2005):

These equations include the following parameters:
πI is the rate of translation of mRNA (R) into Aux/IAA (I).πA is the production rate of ARF activators, supposed constant.The parameters kIX′ and kIX for X∈{I, A}, respectively, denote dissociation and association rates of the dimer forming reactions.κA− represents a background level of transcriptional repression, due to ARF repressors.f and fA represent the strength of transcriptional enhancement due to a single ARF activator and two ARF activators being bound to the promoter, respectively.Kd (Bd, respectively) is the dissociation constant of the ARF:Aux/IAA dimerization reaction (ARF to promoter-binding reaction, respectively). Hence, Kd=kIA′/kIA.The coefficients ωA, ωI and ωDA represent cooperativity effects induced by the binding of two ARF activators (ωA) on the promoter or by ARF:Aux/IAA dimer formation (ωI and ωDA). For the latter, the occurrence of two terms, proportional to IA and DIA is due to the fact that ARF and Aux/IAA may dimerize either prior to DNA binding, or on the promoter after ARF binds to DNA.The parameters of the form δX denote degradation rates. In case of Aux/IAA proteins, the latter is auxin dependent. Then, δI(x) takes the form of saturating functions of the auxin level x:, which was obtained by applying a quasi-steady state assumption to the equations governing the kinetics of auxin-mediated degradation (see Supplementary Note S1). In this function, the parameters δI, γI and K, respectively, represent the basal decay rate of Aux/IAAs, the maximum fold increase in decay rate that can be induced by auxin and the affinity of auxin to TIR1/AFB co-receptors.As mentioned above, we have also assumed that even bound in a dimer, Aux/IAA proteins are accessible to auxin and degraded, freeing the other protein involved in the dimer. Hence the terms δII(x) and δIA(x) in the equations above, which take the same form as δI(x). An additional decay rate of the form δII* has been included as well, to account for the natural decay of the dimers.Numerical simulationsAll simulations were performed using tools from the scientific library Scipy (http://www.scipy.org) of the programming language Python (http://www.python.org). More specifically, after preliminary tests of different solvers, we have chosen to use the odeint routine, which relies on lsoda, an adaptive step-size solver for stiff and non-stiff systems from the FORTRAN library odepack. All figures were produced using the Matplotlib (2D) and Mayavi2 (3D) libraries. A python script containing the main routines can be provided upon request. The model is also available with this paper as an SBML file and has been deposited in the Biomodels database (model accession number: MODEL1105290000).

Materials and methods

Plant material, growth conditions and plant treatmentsAll transgenic plants (see below) were generated in the Columbia (Col-0) ecotype of Arabidopsis thaliana. Plants expressing translational fusion of TIR1, AFB1, AFB2 and AFB3 to GUS have been described (Dharmasiri et al, 2005b; Parry et al, 2009). For the live imaging, plants were grown on soil at 20°C in short-day conditions (8 h light/16 h darkness) for 4 weeks before being transferred in long-day conditions (16 h light/8 h darkness). The chemical treatments were done by immerging dissected inflorescence apices into liquid MS supplemented with the chemicals. Indole-acetic acid (IAA; Sigma) was dissolved in ethanol and used at the indicated final concentration. MG132 was dissolved in DMSO and used at the final concentration of 50 μM for 2h30. For MG132/IAA co-treatments plants were pre-treated with MG132 for 1h30 before adding IAA and waiting an extra hour.

Plant material, growth conditions and plant treatments

All transgenic plants (see below) were generated in the Columbia (Col-0) ecotype of Arabidopsis thaliana. Plants expressing translational fusion of TIR1, AFB1, AFB2 and AFB3 to GUS have been described (Dharmasiri et al, 2005b; Parry et al, 2009). For the live imaging, plants were grown on soil at 20°C in short-day conditions (8 h light/16 h darkness) for 4 weeks before being transferred in long-day conditions (16 h light/8 h darkness). The chemical treatments were done by immerging dissected inflorescence apices into liquid MS supplemented with the chemicals. Indole-acetic acid (IAA; Sigma) was dissolved in ethanol and used at the indicated final concentration. MG132 was dissolved in DMSO and used at the final concentration of 50 μM for 2h30. For MG132/IAA co-treatments plants were pre-treated with MG132 for 1h30 before adding IAA and waiting an extra hour.

Analysis of gene expressionRNA in situ hybridization was performed as described (Vernoux et al, 2000) with at least three independent experiments for each probe tested. Analysis of GUS expression in the inflorescence meristem was done according to Vernoux et al (2000).The RT–qPCR was performed on a step one plus cycler (Applied Biosystems) using the SYBR green reagent kit (Roche). mRNA were extracted from around 100 dissected inflorescence meristems (with all the flowers older than the P1 stage removed) from soil-grown plants. The primers used are from Czechowski et al (2004) except for ARF8, IAA29 and IAA32 (Supplementary Table S3). Expression of several ARF genes could not be assessed due to non-reliable amplification (ARF1, 9, 15, 20, 21, 22 and 23). Expression of the TCTP gene (Supplementary Table S3) was used as a standard and calculations were as described (Pfaffl, 2001). Expressions were analysed on two independent mRNA extractions.

Analysis of gene expression

RNA in situ hybridization was performed as described (Vernoux et al, 2000) with at least three independent experiments for each probe tested. Analysis of GUS expression in the inflorescence meristem was done according to Vernoux et al (2000).

The RT–qPCR was performed on a step one plus cycler (Applied Biosystems) using the SYBR green reagent kit (Roche). mRNA were extracted from around 100 dissected inflorescence meristems (with all the flowers older than the P1 stage removed) from soil-grown plants. The primers used are from Czechowski et al (2004) except for ARF8, IAA29 and IAA32 (Supplementary Table S3). Expression of several ARF genes could not be assessed due to non-reliable amplification (ARF1, 9, 15, 20, 21, 22 and 23). Expression of the TCTP gene (Supplementary Table S3) was used as a standard and calculations were as described (Pfaffl, 2001). Expressions were analysed on two independent mRNA extractions.

Confocal microscopy and live imagingLive imaging of living SAMs was performed as described previously (Fernandez et al, 2010). Observations were done either on an LSM-510 laser-scanning confocal microscope (Zeiss, Jena, Germany) or a SP5 spectral detection confocal Microscope (Leica, Germany).

Confocal microscopy and live imaging

Live imaging of living SAMs was performed as described previously (Fernandez et al, 2010). Observations were done either on an LSM-510 laser-scanning confocal microscope (Zeiss, Jena, Germany) or a SP5 spectral detection confocal Microscope (Leica, Germany).

Generation of plasmids and transgenic plantsThe DR5::VENUS plasmid has been described in Heisler et al (2005). The DII-VENUS and mDII-VENUS binary vectors were generated using Gateway technology and following the Multisite Gateway three-fragment vector construction kit instructions (Invitrogen). Full details of the cloning are given elsewhere (Vernoux et al, in preparation). Briefly, we used the degron region of IAA8, 9 and 28 starting from the conserved lysine to the end of domain II (IAA8: 107–178, IAA9: 120–195, IAA28: 28–61) (Dreher et al, 2006). To generate mDII-VENUS, we introduced the previously described P53L mutation (Rogg et al, 2001) in the wild-type IAA28 degron sequence. These sequences were then fused in frame to VENUS tagged with the N7 nuclear localization signal (Heisler et al, 2005) and put under the control of the strong constitutive 35S promoter. The different plasmids were then introduced in plants by floral dipping (Clough and Bent, 1998).For the plasmids used for generating the pull-down data (see below), a plant expression vector containing a 3xFLAG tag was created by annealing the 3xFLAG-F and the 3xFLAG-R oligonucleotides (Supplementary Table S3) and cloned into a pFP101 binary vector (digested with HindIII and BamHI to remove the 2x35SPro). A Gateway® Cassette B (Invitrogen) was cloned in the Klenow filled-in HindIII site flanking the 3xFLAG fragment to form the pFPGW-3xFLAG vector.The coding sequences for TIR1, AFB1 and AFB5 (obtained from ABRC) were PCR amplified to add Gateway attB sites (Supplementary Table S3) and recombined into pDONR P5-P2 Gateway entry vector (Invitrogen). The TIR1/AFBs were then recombined in a multi-gene Gateway reaction with pDONR P1P5r (Invitrogen) containing the cauliflower mosaic virus 35S promoter (CaMV35S) into the pFPGW-3xFLAG vector, in frame with the 3xFLAG tag, to create the plasmids pFPGW-35S-TIR1/AFB-3xFLAG. The TIR1/AFB-3xFLAG sequence (without the promoter) was PCR amplified from the pFPGW-35S–TIR1/AFB-3xFLAG plasmid by using oligonucleotides AsiSIAFB1, AsiSITIR1 and FLAGPmeI (Supplementary Table S3) and cloned as an AsiSI-PmeI fragment into the pF3A WG (BYDV) Flexi® vector (Promega).A 6xHIS tag was fused at the N-terminus of the ASK1 coding sequence by PCR using oligonucleotide ASK1PmeI successively with the nested primers HISASK1 and AsiSIHIS, and cloned as an AsiSI-PmeI fragment into the pF3A WG vector (generating the pF3A WG ASK1 vector).The 35S:AFB5:3xFLAG (AFB5-FLAG) transgenic line used for pull-down assays from plant extract (see below) was created by first transforming pFPGW-35S-AFB5-3xFLAG into Agrobacterium tumefaciens strain GV3101 (Koncz and Schell, 1986) by electroporation. Transgenic plants were generated using the floral dip method (Clough and Bent, 1998) and transformants were selected by seed coat fluorescence. In the T2 generation, lines showing a 3:1 ratio of seed coat fluorescent:non-fluorescent plants were selected for further study. Homozygous lines were selected from the T3 generation.

Generation of plasmids and transgenic plants

The DR5::VENUS plasmid has been described in Heisler et al (2005). The DII-VENUS and mDII-VENUS binary vectors were generated using Gateway technology and following the Multisite Gateway three-fragment vector construction kit instructions (Invitrogen). Full details of the cloning are given elsewhere (Vernoux et al, in preparation). Briefly, we used the degron region of IAA8, 9 and 28 starting from the conserved lysine to the end of domain II (IAA8: 107–178, IAA9: 120–195, IAA28: 28–61) (Dreher et al, 2006). To generate mDII-VENUS, we introduced the previously described P53L mutation (Rogg et al, 2001) in the wild-type IAA28 degron sequence. These sequences were then fused in frame to VENUS tagged with the N7 nuclear localization signal (Heisler et al, 2005) and put under the control of the strong constitutive 35S promoter. The different plasmids were then introduced in plants by floral dipping (Clough and Bent, 1998).

For the plasmids used for generating the pull-down data (see below), a plant expression vector containing a 3xFLAG tag was created by annealing the 3xFLAG-F and the 3xFLAG-R oligonucleotides (Supplementary Table S3) and cloned into a pFP101 binary vector (digested with HindIII and BamHI to remove the 2x35SPro). A Gateway® Cassette B (Invitrogen) was cloned in the Klenow filled-in HindIII site flanking the 3xFLAG fragment to form the pFPGW-3xFLAG vector.

The coding sequences for TIR1, AFB1 and AFB5 (obtained from ABRC) were PCR amplified to add Gateway attB sites (Supplementary Table S3) and recombined into pDONR P5-P2 Gateway entry vector (Invitrogen). The TIR1/AFBs were then recombined in a multi-gene Gateway reaction with pDONR P1P5r (Invitrogen) containing the cauliflower mosaic virus 35S promoter (CaMV35S) into the pFPGW-3xFLAG vector, in frame with the 3xFLAG tag, to create the plasmids pFPGW-35S-TIR1/AFB-3xFLAG. The TIR1/AFB-3xFLAG sequence (without the promoter) was PCR amplified from the pFPGW-35S–TIR1/AFB-3xFLAG plasmid by using oligonucleotides AsiSIAFB1, AsiSITIR1 and FLAGPmeI (Supplementary Table S3) and cloned as an AsiSI-PmeI fragment into the pF3A WG (BYDV) Flexi® vector (Promega).

A 6xHIS tag was fused at the N-terminus of the ASK1 coding sequence by PCR using oligonucleotide ASK1PmeI successively with the nested primers HISASK1 and AsiSIHIS, and cloned as an AsiSI-PmeI fragment into the pF3A WG vector (generating the pF3A WG ASK1 vector).

The 35S:AFB5:3xFLAG (AFB5-FLAG) transgenic line used for pull-down assays from plant extract (see below) was created by first transforming pFPGW-35S-AFB5-3xFLAG into Agrobacterium tumefaciens strain GV3101 (Koncz and Schell, 1986) by electroporation. Transgenic plants were generated using the floral dip method (Clough and Bent, 1998) and transformants were selected by seed coat fluorescence. In the T2 generation, lines showing a 3:1 ratio of seed coat fluorescent:non-fluorescent plants were selected for further study. Homozygous lines were selected from the T3 generation.

In vitro transcription/translation of TIR1/AFB-tagged proteins, immunoprecipitations and pull-down assaysFor pull-down assays with TIR1-FLAG and AFB1-FLAG expressed in wheat germ extract, the pFPGW-35S-TIR1/AFB-3xFLAG and pF3A WG ASK1 plasmids were used with the TnT SP6 high-yield wheat germ protein expression system (Promega) to synthesize TIR1/AFB–3xFLAG co-expressed with HIS-ASK1 by in vitro transcription/translation in accordance with the manufacturer's instructions.Pull-down assays with wheat germ expressed TIR1/AFB–3xFLAG were performed by combining 22 μl of IVVT reaction extract with 6.5 μg of biotinylated IAA28 domain II peptide (biotinyl-NH-EVAPVVGWPPVRSSRRN-COOH, synthesized by Thermo Scientific), 70 μl 50% streptavidin-agarose suspension and 440 μl of extraction buffer (EB; 0.15 M NaCl, 0.5% nonidet P40, 0.1 M Tris–HCl pH 7.5, containing 1 mM phenylmethylsulphonyl fluoride, 1 μM dithiothreitol, 10 μM MG132 and 1 mg/ml BSA (Sigma) with auxin treatments as indicated. The assays were incubated for 1 h at 4°C with mixing then washed three times for 5 min in EB containing the auxin treatment.For pull-down assays with plant-expressed AFB5-FLAG, extracts of 10-day-old AFB5-FLAG seedling were made as described previously (Kepinski and Leyser, 2005) and used in pull-down assays by combining 2.5 mg of crude extract with 6.5 μg of biotinylated IAA28 domain II peptide, 70 μl 50% streptavidin-agarose suspension. The assays were incubated for 1 h at 4°C with mixing, then washed three times for 5 min in EB containing the auxin treatment.The final processing of all pull-down assays including electrophoresis and western transfer were performed as described previously (Kepinski and Leyser, 2005). The immunodetection of TIR1/AFB-FLAG was performed with a 1:10 000 dilution of anti-FLAG 2-peroxidase (HRP) antibody (Sigma) followed by chemiluminescent detection with ECL plus reagents (Amersham).

In vitro transcription/translation of TIR1/AFB-tagged proteins, immunoprecipitations and pull-down assays

For pull-down assays with TIR1-FLAG and AFB1-FLAG expressed in wheat germ extract, the pFPGW-35S-TIR1/AFB-3xFLAG and pF3A WG ASK1 plasmids were used with the TnT SP6 high-yield wheat germ protein expression system (Promega) to synthesize TIR1/AFB–3xFLAG co-expressed with HIS-ASK1 by in vitro transcription/translation in accordance with the manufacturer's instructions.

Pull-down assays with wheat germ expressed TIR1/AFB–3xFLAG were performed by combining 22 μl of IVVT reaction extract with 6.5 μg of biotinylated IAA28 domain II peptide (biotinyl-NH-EVAPVVGWPPVRSSRRN-COOH, synthesized by Thermo Scientific), 70 μl 50% streptavidin-agarose suspension and 440 μl of extraction buffer (EB; 0.15 M NaCl, 0.5% nonidet P40, 0.1 M Tris–HCl pH 7.5, containing 1 mM phenylmethylsulphonyl fluoride, 1 μM dithiothreitol, 10 μM MG132 and 1 mg/ml BSA (Sigma) with auxin treatments as indicated. The assays were incubated for 1 h at 4°C with mixing then washed three times for 5 min in EB containing the auxin treatment.

For pull-down assays with plant-expressed AFB5-FLAG, extracts of 10-day-old AFB5-FLAG seedling were made as described previously (Kepinski and Leyser, 2005) and used in pull-down assays by combining 2.5 mg of crude extract with 6.5 μg of biotinylated IAA28 domain II peptide, 70 μl 50% streptavidin-agarose suspension. The assays were incubated for 1 h at 4°C with mixing, then washed three times for 5 min in EB containing the auxin treatment.

The final processing of all pull-down assays including electrophoresis and western transfer were performed as described previously (Kepinski and Leyser, 2005). The immunodetection of TIR1/AFB-FLAG was performed with a 1:10 000 dilution of anti-FLAG 2-peroxidase (HRP) antibody (Sigma) followed by chemiluminescent detection with ECL plus reagents (Amersham).

Protein interaction analysesFor generating the different constructs for the interaction analyses, full-length cDNA for all the Aux/IAAs and ARFs (except ARF15 and ARF21) were either obtained from ABRC for the majority of them or RIKEN (for ARF4) or cloned by RT–PCR from cDNA libraries obtained from various tissues (for ARF6, ARF14, IAA8, IAA9 and IAA28). Gateway technology was then used to generate the different constructs following the multisite gateway three-fragment vector construction kit instructions (Invitrogen).For the Y2H analysis, full-length cDNAs were used for all the Aux/IAA and ARF3, 13 and 17 (ATG to Stop). Partial cDNAs encoding domain III and IV were used for the other ARFs and corresponded to the following region of the proteins—ARF1: 538-Stop; ARF2: 727-Stop; ARF4: 660-Stop; ARF5: 788-Stop; ARF6: 790-Stop; ARF7: 1030-Stop; ARF8: 699-Stop; ARF9: 519-Stop; ARF10: 575-Stop; ARF11: 500-Stop; ARF12: 500-Stop; ARF14: 497-Stop; ARF16: 570-Stop; ARF18: 482-Stop; ARF19: 948-Stop; ARF20: 484-Stop; ARF22: 498-Stop. The cDNAs or parts of cDNA were cloned directionally in pENTR/D-Topo (Invitrogen). They were then systematically transferred by recombination to a gateway-compatible pACT2-based vector (downstream of Gal4-AD) and a pGBKT7-based vector (downstream of Gal4-BD). The Y2H screen was performed using the mating method in a microtiter plate format as described (Boruc et al, 2010). Briefly, each interaction was tested in the two directions (each protein was used alternatively as a bait or as a prey) and using two independent reporters (LacZ and HIS3). LacZ activity was detected visually in the presence of 5-bromo-4-chloro-3-indolyl-β-D-galactopyranoside (X-gal; Blue staining). HIS3 activity was detected by the restoration of growth in the absence of histidine (see below for details on the analysis of the Y2H data). We also tested expression in yeast for all the constructs using standard western blot analysis with an anti-HA antibody (clone 3F10, Roche) for the fusions to Gal4-AD and a Gal4-BD monoclonal antibody (Clontech) for the fusions to Gal4-BD. We confirmed expression for all the Gal4-BD fusions, but a few Gal4-AD fusion were very weak or could not be detected, indicating that the number of interactors might be underestimated for ARF9, ARF17, IAA11, IAA20 and IAA30.For BiFC analysis, we tested various interactions (see Supplementary Figure S4) between ARF5, ARF6, ARF9, ARF18 and/or the following stabilized Aux/IAAs: IAA8 P172S, IAA12 P74S, IAA17 P88L, IAA18 P101S and IAA28 P53L. These mutations have been described for IAA12, 17 and 28 (Rouse et al, 1998; Rogg et al, 2001; Hamann et al, 2002) and similar mutations were introduced in IAA8 and 18. The full-length cDNAs were fused in frame to YFP molecule halves and interactions were tested by transient expression in Nicotiana benthamina leaves as described (Desprez et al, 2007). Empty vectors were used as a negative control. Each interaction was tested independently two to four times and similar leaves areas were scanned for the different tests. The protein interactions from this publication have been submitted to the IMEx (http://imex.sf.net) consortium through IntAct (pmid 19850723) and assigned the identifier IM-15409.

Protein interaction analyses

For generating the different constructs for the interaction analyses, full-length cDNA for all the Aux/IAAs and ARFs (except ARF15 and ARF21) were either obtained from ABRC for the majority of them or RIKEN (for ARF4) or cloned by RT–PCR from cDNA libraries obtained from various tissues (for ARF6, ARF14, IAA8, IAA9 and IAA28). Gateway technology was then used to generate the different constructs following the multisite gateway three-fragment vector construction kit instructions (Invitrogen).

For the Y2H analysis, full-length cDNAs were used for all the Aux/IAA and ARF3, 13 and 17 (ATG to Stop). Partial cDNAs encoding domain III and IV were used for the other ARFs and corresponded to the following region of the proteins—ARF1: 538-Stop; ARF2: 727-Stop; ARF4: 660-Stop; ARF5: 788-Stop; ARF6: 790-Stop; ARF7: 1030-Stop; ARF8: 699-Stop; ARF9: 519-Stop; ARF10: 575-Stop; ARF11: 500-Stop; ARF12: 500-Stop; ARF14: 497-Stop; ARF16: 570-Stop; ARF18: 482-Stop; ARF19: 948-Stop; ARF20: 484-Stop; ARF22: 498-Stop. The cDNAs or parts of cDNA were cloned directionally in pENTR/D-Topo (Invitrogen). They were then systematically transferred by recombination to a gateway-compatible pACT2-based vector (downstream of Gal4-AD) and a pGBKT7-based vector (downstream of Gal4-BD). The Y2H screen was performed using the mating method in a microtiter plate format as described (Boruc et al, 2010). Briefly, each interaction was tested in the two directions (each protein was used alternatively as a bait or as a prey) and using two independent reporters (LacZ and HIS3). LacZ activity was detected visually in the presence of 5-bromo-4-chloro-3-indolyl-β-D-galactopyranoside (X-gal; Blue staining). HIS3 activity was detected by the restoration of growth in the absence of histidine (see below for details on the analysis of the Y2H data). We also tested expression in yeast for all the constructs using standard western blot analysis with an anti-HA antibody (clone 3F10, Roche) for the fusions to Gal4-AD and a Gal4-BD monoclonal antibody (Clontech) for the fusions to Gal4-BD. We confirmed expression for all the Gal4-BD fusions, but a few Gal4-AD fusion were very weak or could not be detected, indicating that the number of interactors might be underestimated for ARF9, ARF17, IAA11, IAA20 and IAA30.

For BiFC analysis, we tested various interactions (see Supplementary Figure S4) between ARF5, ARF6, ARF9, ARF18 and/or the following stabilized Aux/IAAs: IAA8 P172S, IAA12 P74S, IAA17 P88L, IAA18 P101S and IAA28 P53L. These mutations have been described for IAA12, 17 and 28 (Rouse et al, 1998; Rogg et al, 2001; Hamann et al, 2002) and similar mutations were introduced in IAA8 and 18. The full-length cDNAs were fused in frame to YFP molecule halves and interactions were tested by transient expression in Nicotiana benthamina leaves as described (Desprez et al, 2007). Empty vectors were used as a negative control. Each interaction was tested independently two to four times and similar leaves areas were scanned for the different tests. The protein interactions from this publication have been submitted to the IMEx (http://imex.sf.net) consortium through IntAct (pmid 19850723) and assigned the identifier IM-15409.

Cluster analysis of Aux/IAA-ARF interaction networkConstruction of the interaction network on the basis of the yeast two-hybrid dataThe output of the X-gal test took the form of a mark chosen from among the following ordered marks: –; +; ++; +++. The output of the HIS3 test took the form of an optical density (OD) ratio reflecting the efficiency of recovery (ratio of the OD in the absence of histidine to the OD in the presence of histidine). The Spearman's rank correlation coefficient between the outputs of the X-gal and HIS3 tests was approximately 0.7, suggesting a partial but good agreement between these two tests. We thus chose to build a decision rule that exploits the redundancy between the X-gal and HIS3 tests and the two possible configurations for a given pair of proteins. This decision rule involved two thresholds, one for the X-gal and another for the HIS3 test. In order to determine these thresholds, we analysed the empirical distributions of the OD ratio (HIS3 test output) for each possible X-gal test output using Gaussian mixtures estimated with the mclust R package (http://www.stat.washington.edu/fraley/mclust). In particular, the empirical OD ratio distribution for the ‘+’ X-gal test mark was well fitted by a two-component Gaussian mixture, where the component of lowest mean was interpreted as false positives. This interpretation was supported by the fact that the Gaussian component of lowest mean almost disappeared in the empirical OD ratio distribution for the ‘++’ and ‘+++’ X-gal test marks (i.e. for higher stringency of the X-gal test). On the contrary, it was over-represented for the ‘−’ mark. Based on this statistical analysis, the thresholds were fixed between ‘−’ and ‘+’ for the X-gal test and at 0.45 for the HIS3 test. We finally defined the following decision rule in order to minimize false positives: there is interaction between proteins if at least an X-gal test and an HIS3 test are positive.Cluster analysisStructuring connectivity patterns were uncovered using a probabilistic clustering method implemented in the MixNet software (Daudin et al, 2008; Picard et al, 2009). The key feature of the MixNet model is to give a probabilistic summary of the connectivity structure by uncovering clusters of proteins that share the same connectivity profiles. Briefly, instead of directly describing the clustered structure of vertices, the MixNet model describes the topology of the network using connectivity probabilities πqℓ that is the probability for a vertex from class q to be connected with a vertex from class ℓ. The protein interaction network was modelled as a random graph with (Xij;i, j=1,…N) representing its adjacency matrix, such that Xij=1 if vertices i and j are connected and 0 otherwise. The idea of the MixNet model is to consider that vertices can be spread into Q connectivity clusters that are hidden, with Q being unknown as well. The parameters of this model are the proportions of each cluster α=(αq;q=1,…Q), and the connectivity probability matrix Π=(πqℓ;q, ℓ=1,…,Q). To this extent, Π is a summary of the connectivity of the protein interaction network at the cluster level. For a given number of clusters Q, the outputs of the MixNet algorithm are the estimated model parameters αˆ and Πˆ and the probabilities of assignment of vertices to clusters (τiq;i=1, K, N;q=1, K, Q). The posterior distribution (τiq;q=1,…Q) represents the probabilistic assignment of vertex i to the clusters.Analysis of the adequacy of the clusteringWe assessed the adequacy of the clustering obtained by the MixNet algorithm by evaluating the separability of the clusters and the dispersion of the proteins within the clusters. Since, in our case, the assignment of proteins to clusters is almost deterministic (i.e. τiq≈1 for a unique cluster q and τiℓ≈0 for ℓ≠q), this assignment can be viewed as a partition. The model parameters (πqℓ), which parameterized the edges of the graph, cannot be used directly to define dispersion measures of the vertices assigned to a given cluster. We thus used the edges incident to the vertices to derive dissimilarity measures for the vertices using the adjacency information. The Sokal–Michener distance between vertices i and j defined as Dij=ΣkI(xik≠xjk)/N, where I( ) denotes the indicator function, is the proportion of mismatches or disagreement between the ith and jth rows of the adjacency matrix. This distance naturally expresses the difference in connectivity profiles between vertices.The distance between vertex i and cluster q is then given by

If the vertices are deterministically assigned to a given cluster, this distance simplifies to

where nq is the number of elements of cluster q.The distance between cluster q and cluster ℓ is given by

If the vertices are deterministically assigned to a given cluster, this distance simplifies to

Graphical representation of proteins preserving pairwise distances between themWe applied a multidimensional scaling (MDS) method (Hastie et al, 2005) to visualize the proteins. Briefly, the MDS approach allowed us to find a low-dimensional projection of the data such as to preserve, as closely as possible, the pairwise distances between data points {D(i, j);i,j=1,…,N}.

Cluster analysis of Aux/IAA-ARF interaction network

Construction of the interaction network on the basis of the yeast two-hybrid dataThe output of the X-gal test took the form of a mark chosen from among the following ordered marks: –; +; ++; +++. The output of the HIS3 test took the form of an optical density (OD) ratio reflecting the efficiency of recovery (ratio of the OD in the absence of histidine to the OD in the presence of histidine). The Spearman's rank correlation coefficient between the outputs of the X-gal and HIS3 tests was approximately 0.7, suggesting a partial but good agreement between these two tests. We thus chose to build a decision rule that exploits the redundancy between the X-gal and HIS3 tests and the two possible configurations for a given pair of proteins. This decision rule involved two thresholds, one for the X-gal and another for the HIS3 test. In order to determine these thresholds, we analysed the empirical distributions of the OD ratio (HIS3 test output) for each possible X-gal test output using Gaussian mixtures estimated with the mclust R package (http://www.stat.washington.edu/fraley/mclust). In particular, the empirical OD ratio distribution for the ‘+’ X-gal test mark was well fitted by a two-component Gaussian mixture, where the component of lowest mean was interpreted as false positives. This interpretation was supported by the fact that the Gaussian component of lowest mean almost disappeared in the empirical OD ratio distribution for the ‘++’ and ‘+++’ X-gal test marks (i.e. for higher stringency of the X-gal test). On the contrary, it was over-represented for the ‘−’ mark. Based on this statistical analysis, the thresholds were fixed between ‘−’ and ‘+’ for the X-gal test and at 0.45 for the HIS3 test. We finally defined the following decision rule in order to minimize false positives: there is interaction between proteins if at least an X-gal test and an HIS3 test are positive.

Construction of the interaction network on the basis of the yeast two-hybrid data

The output of the X-gal test took the form of a mark chosen from among the following ordered marks: –; +; ++; +++. The output of the HIS3 test took the form of an optical density (OD) ratio reflecting the efficiency of recovery (ratio of the OD in the absence of histidine to the OD in the presence of histidine). The Spearman's rank correlation coefficient between the outputs of the X-gal and HIS3 tests was approximately 0.7, suggesting a partial but good agreement between these two tests. We thus chose to build a decision rule that exploits the redundancy between the X-gal and HIS3 tests and the two possible configurations for a given pair of proteins. This decision rule involved two thresholds, one for the X-gal and another for the HIS3 test. In order to determine these thresholds, we analysed the empirical distributions of the OD ratio (HIS3 test output) for each possible X-gal test output using Gaussian mixtures estimated with the mclust R package (http://www.stat.washington.edu/fraley/mclust). In particular, the empirical OD ratio distribution for the ‘+’ X-gal test mark was well fitted by a two-component Gaussian mixture, where the component of lowest mean was interpreted as false positives. This interpretation was supported by the fact that the Gaussian component of lowest mean almost disappeared in the empirical OD ratio distribution for the ‘++’ and ‘+++’ X-gal test marks (i.e. for higher stringency of the X-gal test). On the contrary, it was over-represented for the ‘−’ mark. Based on this statistical analysis, the thresholds were fixed between ‘−’ and ‘+’ for the X-gal test and at 0.45 for the HIS3 test. We finally defined the following decision rule in order to minimize false positives: there is interaction between proteins if at least an X-gal test and an HIS3 test are positive.

Cluster analysisStructuring connectivity patterns were uncovered using a probabilistic clustering method implemented in the MixNet software (Daudin et al, 2008; Picard et al, 2009). The key feature of the MixNet model is to give a probabilistic summary of the connectivity structure by uncovering clusters of proteins that share the same connectivity profiles. Briefly, instead of directly describing the clustered structure of vertices, the MixNet model describes the topology of the network using connectivity probabilities πqℓ that is the probability for a vertex from class q to be connected with a vertex from class ℓ. The protein interaction network was modelled as a random graph with (Xij;i, j=1,…N) representing its adjacency matrix, such that Xij=1 if vertices i and j are connected and 0 otherwise. The idea of the MixNet model is to consider that vertices can be spread into Q connectivity clusters that are hidden, with Q being unknown as well. The parameters of this model are the proportions of each cluster α=(αq;q=1,…Q), and the connectivity probability matrix Π=(πqℓ;q, ℓ=1,…,Q). To this extent, Π is a summary of the connectivity of the protein interaction network at the cluster level. For a given number of clusters Q, the outputs of the MixNet algorithm are the estimated model parameters αˆ and Πˆ and the probabilities of assignment of vertices to clusters (τiq;i=1, K, N;q=1, K, Q). The posterior distribution (τiq;q=1,…Q) represents the probabilistic assignment of vertex i to the clusters.

Cluster analysis

Structuring connectivity patterns were uncovered using a probabilistic clustering method implemented in the MixNet software (Daudin et al, 2008; Picard et al, 2009). The key feature of the MixNet model is to give a probabilistic summary of the connectivity structure by uncovering clusters of proteins that share the same connectivity profiles. Briefly, instead of directly describing the clustered structure of vertices, the MixNet model describes the topology of the network using connectivity probabilities πqℓ that is the probability for a vertex from class q to be connected with a vertex from class ℓ. The protein interaction network was modelled as a random graph with (Xij;i, j=1,…N) representing its adjacency matrix, such that Xij=1 if vertices i and j are connected and 0 otherwise. The idea of the MixNet model is to consider that vertices can be spread into Q connectivity clusters that are hidden, with Q being unknown as well. The parameters of this model are the proportions of each cluster α=(αq;q=1,…Q), and the connectivity probability matrix Π=(πqℓ;q, ℓ=1,…,Q). To this extent, Π is a summary of the connectivity of the protein interaction network at the cluster level. For a given number of clusters Q, the outputs of the MixNet algorithm are the estimated model parameters αˆ and Πˆ and the probabilities of assignment of vertices to clusters (τiq;i=1, K, N;q=1, K, Q). The posterior distribution (τiq;q=1,…Q) represents the probabilistic assignment of vertex i to the clusters.

Analysis of the adequacy of the clusteringWe assessed the adequacy of the clustering obtained by the MixNet algorithm by evaluating the separability of the clusters and the dispersion of the proteins within the clusters. Since, in our case, the assignment of proteins to clusters is almost deterministic (i.e. τiq≈1 for a unique cluster q and τiℓ≈0 for ℓ≠q), this assignment can be viewed as a partition. The model parameters (πqℓ), which parameterized the edges of the graph, cannot be used directly to define dispersion measures of the vertices assigned to a given cluster. We thus used the edges incident to the vertices to derive dissimilarity measures for the vertices using the adjacency information. The Sokal–Michener distance between vertices i and j defined as Dij=ΣkI(xik≠xjk)/N, where I( ) denotes the indicator function, is the proportion of mismatches or disagreement between the ith and jth rows of the adjacency matrix. This distance naturally expresses the difference in connectivity profiles between vertices.The distance between vertex i and cluster q is then given by

If the vertices are deterministically assigned to a given cluster, this distance simplifies to

where nq is the number of elements of cluster q.The distance between cluster q and cluster ℓ is given by

If the vertices are deterministically assigned to a given cluster, this distance simplifies to

Analysis of the adequacy of the clustering

We assessed the adequacy of the clustering obtained by the MixNet algorithm by evaluating the separability of the clusters and the dispersion of the proteins within the clusters. Since, in our case, the assignment of proteins to clusters is almost deterministic (i.e. τiq≈1 for a unique cluster q and τiℓ≈0 for ℓ≠q), this assignment can be viewed as a partition. The model parameters (πqℓ), which parameterized the edges of the graph, cannot be used directly to define dispersion measures of the vertices assigned to a given cluster. We thus used the edges incident to the vertices to derive dissimilarity measures for the vertices using the adjacency information. The Sokal–Michener distance between vertices i and j defined as Dij=ΣkI(xik≠xjk)/N, where I( ) denotes the indicator function, is the proportion of mismatches or disagreement between the ith and jth rows of the adjacency matrix. This distance naturally expresses the difference in connectivity profiles between vertices.

The distance between vertex i and cluster q is then given by

If the vertices are deterministically assigned to a given cluster, this distance simplifies to

where nq is the number of elements of cluster q.

The distance between cluster q and cluster ℓ is given by

If the vertices are deterministically assigned to a given cluster, this distance simplifies to

Graphical representation of proteins preserving pairwise distances between themWe applied a multidimensional scaling (MDS) method (Hastie et al, 2005) to visualize the proteins. Briefly, the MDS approach allowed us to find a low-dimensional projection of the data such as to preserve, as closely as possible, the pairwise distances between data points {D(i, j);i,j=1,…,N}.

Graphical representation of proteins preserving pairwise distances between them

We applied a multidimensional scaling (MDS) method (Hastie et al, 2005) to visualize the proteins. Briefly, the MDS approach allowed us to find a low-dimensional projection of the data such as to preserve, as closely as possible, the pairwise distances between data points {D(i, j);i,j=1,…,N}.

Modelling of the Aux/IAA-ARF signalling pathwayModel descriptionExcept for gene transcription, we assumed that all reactions involved in this network are well described using mass action law kinetics. Variables are denoted using upper case characters, and parameters with lower case. Let I and A denote the concentrations of the IAA and ARF proteins, respectively, and DII and DIA the concentration of the dimers formed by these proteins. R denotes the total concentration of mRNA that is transcribed from a pool of auxin responsive genes. This set of genes includes Aux/IAAs, so that I has a production rate proportional to R. Auxin is represented by the variable x appearing in decay rates involving Aux/IAA proteins. The system can be written as follows:

where h is the transcription rate of the target genes, which depends on the levels of the different transcription factors. This function was determined based on thermodynamic assumptions, as described for example in (Bintu et al, 2005):

These equations include the following parameters:
πI is the rate of translation of mRNA (R) into Aux/IAA (I).πA is the production rate of ARF activators, supposed constant.The parameters kIX′ and kIX for X∈{I, A}, respectively, denote dissociation and association rates of the dimer forming reactions.κA− represents a background level of transcriptional repression, due to ARF repressors.f and fA represent the strength of transcriptional enhancement due to a single ARF activator and two ARF activators being bound to the promoter, respectively.Kd (Bd, respectively) is the dissociation constant of the ARF:Aux/IAA dimerization reaction (ARF to promoter-binding reaction, respectively). Hence, Kd=kIA′/kIA.The coefficients ωA, ωI and ωDA represent cooperativity effects induced by the binding of two ARF activators (ωA) on the promoter or by ARF:Aux/IAA dimer formation (ωI and ωDA). For the latter, the occurrence of two terms, proportional to IA and DIA is due to the fact that ARF and Aux/IAA may dimerize either prior to DNA binding, or on the promoter after ARF binds to DNA.The parameters of the form δX denote degradation rates. In case of Aux/IAA proteins, the latter is auxin dependent. Then, δI(x) takes the form of saturating functions of the auxin level x:, which was obtained by applying a quasi-steady state assumption to the equations governing the kinetics of auxin-mediated degradation (see Supplementary Note S1). In this function, the parameters δI, γI and K, respectively, represent the basal decay rate of Aux/IAAs, the maximum fold increase in decay rate that can be induced by auxin and the affinity of auxin to TIR1/AFB co-receptors.As mentioned above, we have also assumed that even bound in a dimer, Aux/IAA proteins are accessible to auxin and degraded, freeing the other protein involved in the dimer. Hence the terms δII(x) and δIA(x) in the equations above, which take the same form as δI(x). An additional decay rate of the form δII* has been included as well, to account for the natural decay of the dimers.Numerical simulationsAll simulations were performed using tools from the scientific library Scipy (http://www.scipy.org) of the programming language Python (http://www.python.org). More specifically, after preliminary tests of different solvers, we have chosen to use the odeint routine, which relies on lsoda, an adaptive step-size solver for stiff and non-stiff systems from the FORTRAN library odepack. All figures were produced using the Matplotlib (2D) and Mayavi2 (3D) libraries. A python script containing the main routines can be provided upon request. The model is also available with this paper as an SBML file and has been deposited in the Biomodels database (model accession number: MODEL1105290000).

Modelling of the Aux/IAA-ARF signalling pathway

Model descriptionExcept for gene transcription, we assumed that all reactions involved in this network are well described using mass action law kinetics. Variables are denoted using upper case characters, and parameters with lower case. Let I and A denote the concentrations of the IAA and ARF proteins, respectively, and DII and DIA the concentration of the dimers formed by these proteins. R denotes the total concentration of mRNA that is transcribed from a pool of auxin responsive genes. This set of genes includes Aux/IAAs, so that I has a production rate proportional to R. Auxin is represented by the variable x appearing in decay rates involving Aux/IAA proteins. The system can be written as follows:

where h is the transcription rate of the target genes, which depends on the levels of the different transcription factors. This function was determined based on thermodynamic assumptions, as described for example in (Bintu et al, 2005):

These equations include the following parameters:
πI is the rate of translation of mRNA (R) into Aux/IAA (I).πA is the production rate of ARF activators, supposed constant.The parameters kIX′ and kIX for X∈{I, A}, respectively, denote dissociation and association rates of the dimer forming reactions.κA− represents a background level of transcriptional repression, due to ARF repressors.f and fA represent the strength of transcriptional enhancement due to a single ARF activator and two ARF activators being bound to the promoter, respectively.Kd (Bd, respectively) is the dissociation constant of the ARF:Aux/IAA dimerization reaction (ARF to promoter-binding reaction, respectively). Hence, Kd=kIA′/kIA.The coefficients ωA, ωI and ωDA represent cooperativity effects induced by the binding of two ARF activators (ωA) on the promoter or by ARF:Aux/IAA dimer formation (ωI and ωDA). For the latter, the occurrence of two terms, proportional to IA and DIA is due to the fact that ARF and Aux/IAA may dimerize either prior to DNA binding, or on the promoter after ARF binds to DNA.The parameters of the form δX denote degradation rates. In case of Aux/IAA proteins, the latter is auxin dependent. Then, δI(x) takes the form of saturating functions of the auxin level x:, which was obtained by applying a quasi-steady state assumption to the equations governing the kinetics of auxin-mediated degradation (see Supplementary Note S1). In this function, the parameters δI, γI and K, respectively, represent the basal decay rate of Aux/IAAs, the maximum fold increase in decay rate that can be induced by auxin and the affinity of auxin to TIR1/AFB co-receptors.As mentioned above, we have also assumed that even bound in a dimer, Aux/IAA proteins are accessible to auxin and degraded, freeing the other protein involved in the dimer. Hence the terms δII(x) and δIA(x) in the equations above, which take the same form as δI(x). An additional decay rate of the form δII* has been included as well, to account for the natural decay of the dimers.

Model description

Except for gene transcription, we assumed that all reactions involved in this network are well described using mass action law kinetics. Variables are denoted using upper case characters, and parameters with lower case. Let I and A denote the concentrations of the IAA and ARF proteins, respectively, and DII and DIA the concentration of the dimers formed by these proteins. R denotes the total concentration of mRNA that is transcribed from a pool of auxin responsive genes. This set of genes includes Aux/IAAs, so that I has a production rate proportional to R. Auxin is represented by the variable x appearing in decay rates involving Aux/IAA proteins. The system can be written as follows:

where h is the transcription rate of the target genes, which depends on the levels of the different transcription factors. This function was determined based on thermodynamic assumptions, as described for example in (Bintu et al, 2005):

These equations include the following parameters:
πI is the rate of translation of mRNA (R) into Aux/IAA (I).πA is the production rate of ARF activators, supposed constant.The parameters kIX′ and kIX for X∈{I, A}, respectively, denote dissociation and association rates of the dimer forming reactions.κA− represents a background level of transcriptional repression, due to ARF repressors.f and fA represent the strength of transcriptional enhancement due to a single ARF activator and two ARF activators being bound to the promoter, respectively.Kd (Bd, respectively) is the dissociation constant of the ARF:Aux/IAA dimerization reaction (ARF to promoter-binding reaction, respectively). Hence, Kd=kIA′/kIA.The coefficients ωA, ωI and ωDA represent cooperativity effects induced by the binding of two ARF activators (ωA) on the promoter or by ARF:Aux/IAA dimer formation (ωI and ωDA). For the latter, the occurrence of two terms, proportional to IA and DIA is due to the fact that ARF and Aux/IAA may dimerize either prior to DNA binding, or on the promoter after ARF binds to DNA.The parameters of the form δX denote degradation rates. In case of Aux/IAA proteins, the latter is auxin dependent. Then, δI(x) takes the form of saturating functions of the auxin level x:, which was obtained by applying a quasi-steady state assumption to the equations governing the kinetics of auxin-mediated degradation (see Supplementary Note S1). In this function, the parameters δI, γI and K, respectively, represent the basal decay rate of Aux/IAAs, the maximum fold increase in decay rate that can be induced by auxin and the affinity of auxin to TIR1/AFB co-receptors.As mentioned above, we have also assumed that even bound in a dimer, Aux/IAA proteins are accessible to auxin and degraded, freeing the other protein involved in the dimer. Hence the terms δII(x) and δIA(x) in the equations above, which take the same form as δI(x). An additional decay rate of the form δII* has been included as well, to account for the natural decay of the dimers.

πI is the rate of translation of mRNA (R) into Aux/IAA (I).

πA is the production rate of ARF activators, supposed constant.

The parameters kIX′ and kIX for X∈{I, A}, respectively, denote dissociation and association rates of the dimer forming reactions.

κA− represents a background level of transcriptional repression, due to ARF repressors.

f and fA represent the strength of transcriptional enhancement due to a single ARF activator and two ARF activators being bound to the promoter, respectively.

Kd (Bd, respectively) is the dissociation constant of the ARF:Aux/IAA dimerization reaction (ARF to promoter-binding reaction, respectively). Hence, Kd=kIA′/kIA.

The coefficients ωA, ωI and ωDA represent cooperativity effects induced by the binding of two ARF activators (ωA) on the promoter or by ARF:Aux/IAA dimer formation (ωI and ωDA). For the latter, the occurrence of two terms, proportional to IA and DIA is due to the fact that ARF and Aux/IAA may dimerize either prior to DNA binding, or on the promoter after ARF binds to DNA.

The parameters of the form δX denote degradation rates. In case of Aux/IAA proteins, the latter is auxin dependent. Then, δI(x) takes the form of saturating functions of the auxin level x:, which was obtained by applying a quasi-steady state assumption to the equations governing the kinetics of auxin-mediated degradation (see Supplementary Note S1). In this function, the parameters δI, γI and K, respectively, represent the basal decay rate of Aux/IAAs, the maximum fold increase in decay rate that can be induced by auxin and the affinity of auxin to TIR1/AFB co-receptors.

As mentioned above, we have also assumed that even bound in a dimer, Aux/IAA proteins are accessible to auxin and degraded, freeing the other protein involved in the dimer. Hence the terms δII(x) and δIA(x) in the equations above, which take the same form as δI(x). An additional decay rate of the form δII* has been included as well, to account for the natural decay of the dimers.

Numerical simulationsAll simulations were performed using tools from the scientific library Scipy (http://www.scipy.org) of the programming language Python (http://www.python.org). More specifically, after preliminary tests of different solvers, we have chosen to use the odeint routine, which relies on lsoda, an adaptive step-size solver for stiff and non-stiff systems from the FORTRAN library odepack. All figures were produced using the Matplotlib (2D) and Mayavi2 (3D) libraries. A python script containing the main routines can be provided upon request. The model is also available with this paper as an SBML file and has been deposited in the Biomodels database (model accession number: MODEL1105290000).

Numerical simulations

All simulations were performed using tools from the scientific library Scipy (http://www.scipy.org) of the programming language Python (http://www.python.org). More specifically, after preliminary tests of different solvers, we have chosen to use the odeint routine, which relies on lsoda, an adaptive step-size solver for stiff and non-stiff systems from the FORTRAN library odepack. All figures were produced using the Matplotlib (2D) and Mayavi2 (3D) libraries. A python script containing the main routines can be provided upon request. The model is also available with this paper as an SBML file and has been deposited in the Biomodels database (model accession number: MODEL1105290000).

Supplementary MaterialSupplementary InformationSupplementary Figures S1–9, Supplementary Tables S2–3Note S1Detailed mathematical and simulation study of the model presented in the manuscriptAuxin Signalling ODE modelSBML-fomat of the model presented in the manuscriptSupplementary Table 1Review Process File

Supplementary Material

Supplementary Information

Supplementary Figures S1–9, Supplementary Tables S2–3

Detailed mathematical and simulation study of the model presented in the manuscript

Auxin Signalling ODE model

SBML-fomat of the model presented in the manuscript

Supplementary Table 1

Review Process File
