---
pmid: '27680706'
title: Frizzled proteins are colonic epithelial receptors for C. difficile toxin B.
authors:
- Tao L
- Zhang J
- Meraner P
- Tovaglieri A
- Wu X
- Gerhard R
- Zhang X
- Stallcup WB
- Miao J
- He X
- Hurdle JG
- Breault DT
- Brass AL
- Dong M
journal: Nature
year: '2016'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC5519134
doi: 10.1038/nature19799
---

# Frizzled proteins are colonic epithelial receptors for C. difficile toxin B.
**Authors:** Tao L, Zhang J, Meraner P, Tovaglieri A, Wu X, Gerhard R, Zhang X, Stallcup WB, Miao J, He X, Hurdle JG, Breault DT, Brass AL, Dong M
**Journal:** Nature (2016)
**DOI:** [10.1038/nature19799](https://doi.org/10.1038/nature19799)
**PMC:** [PMC5519134](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5519134/)

## Abstract

1. Nature. 2016 Oct 20;538(7625):350-355. doi: 10.1038/nature19799. Epub 2016 Sep
 28.

Frizzled proteins are colonic epithelial receptors for C. difficile toxin B.

Tao L(1)(2), Zhang J(1)(2), Meraner P(3), Tovaglieri A(4), Wu X(5), Gerhard 
R(6), Zhang X(7)(8), Stallcup WB(9), Miao J(4)(10), He X(7)(8), Hurdle JG(5), 
Breault DT(4)(10)(11), Brass AL(3)(12), Dong M(1)(2).

Author information:
(1)Department of Urology, Boston Children's Hospital, Harvard Medical School, 
Boston, Massachusetts 02115, USA.
(2)Department of Microbiology and Immunobiology and Department of Surgery, 
Harvard Medical School, Boston, Massachusetts 02115, USA.
(3)Department of Microbiology and Physiological Systems (MaPS), University of 
Massachusetts Medical School, Worcester, Massachusetts 01655, USA.
(4)Division of Endocrinology, Boston Children's Hospital, Boston, Massachusetts 
02115, USA.
(5)Center for Infectious and Inflammatory Diseases, Texas A &M Health Science 
Center, Houston, Texas 77030, USA.
(6)Institute of Toxicology, Hannover Medical School, 30625 Hannover, Germany.
(7)The F. M. Kirby Neurobiology Center, Boston Children's Hospital, Harvard 
Medical School, Boston, Massachusetts 02115, USA.
(8)Department of Neurology, Harvard Medical School, Boston, Massachusetts 02115, 
USA.
(9)Tumor Microenvironment and Cancer Immunology Program, Sanford-Burnham Prebys 
Medical Discovery Institute, Cancer Center, La Jolla, California 92037, USA.
(10)Department of Pediatrics, Harvard Medical School, Boston, Massachusetts 
02115, USA.
(11)Harvard Stem Cell Institute, Cambridge, Massachusetts 02138, USA.
(12)Gastroenterology Division, Department of Medicine, University of 
Massachusetts Medical School, Worcester, Massachusetts 01655, USA.

Comment in
    Nat Rev Microbiol. 2016 Oct 12;14(11):666-667. doi: 
10.1038/nrmicro.2016.152.

Clostridium difficile toxin B (TcdB) is a critical virulence factor that causes 
diseases associated with C. difficile infection. Here we carried out 
CRISPR-Cas9-mediated genome-wide screens and identified the members of the Wnt 
receptor frizzled family (FZDs) as TcdB receptors. TcdB binds to the conserved 
Wnt-binding site known as the cysteine-rich domain (CRD), with the highest 
affinity towards FZD1, 2 and 7. TcdB competes with Wnt for binding to FZDs, and 
its binding blocks Wnt signalling. FZD1/2/7 triple-knockout cells are highly 
resistant to TcdB, and recombinant FZD2-CRD prevented TcdB binding to the 
colonic epithelium. Colonic organoids cultured from FZD7-knockout mice, combined 
with knockdown of FZD1 and 2, showed increased resistance to TcdB. The colonic 
epithelium in FZD7-knockout mice was less susceptible to TcdB-induced tissue 
damage in vivo. These findings establish FZDs as physiologically relevant 
receptors for TcdB in the colonic epithelium.

DOI: 10.1038/nature19799
PMCID: PMC5519134
PMID: 27680706 [Indexed for MEDLINE]

Conflict of interest statement: Conflict of Interests The authors declare no 
conflict of interests.

## Full Text

IntroductionInfection of the colon by the opportunistic Gram-positive bacterium Clostridium difficile leads to a range of manifestations from diarrhea to life-threatening pseudomembranous colitis and toxic megacolon1–5. It is the most common cause of antibiotic-associated diarrhea and a leading cause of gastroenteritis-associated death in developed countries, accounting for nearly a half-million cases and 29,000 deaths annually in the United States6. Two homologous exotoxins, toxin A (TcdA) and toxin B (TcdB), are the causal agents for diseases associated with Clostridium difficile infection (CDI)4,7–9. These toxins enter cells via receptor-mediated endocytosis and inactivate small GTPases by glucosylating a key residue, resulting in cell-rounding and eventual death of cells4,7,10. Of the two toxins, TcdB alone is capable of causing the full spectrum of diseases, as TcdA−B+ strains have been clinically isolated and engineered TcdA−B+ strains induced death in animal models11–14.How TcdB targets the colonic epithelium remains unknown. TcdB can enter a variety of cell lines, suggesting that its receptor(s) are widely expressed in transformed cells. It has also been reported that TcdB is enriched in the heart after injection into zebrafish embryos15. Chondroitin sulfate proteoglycan 4 (CSPG4, also known as neuron-glial antigen 2 (NG2)) has been identified as a TcdB receptor in a shRNA-mediated knock-down (KD) screen16, and was shown to be a functional receptor for TcdB in HeLa cells and in HT-29 cells, a human colorectal cell line. However, CSPG4 is not expressed in the colonic epithelium17. Poliovirus receptor-like 3 (PVRL3) was recently identified from a gene-trap insertional mutagenesis screen in Caco-2 cells, a human colorectal cell line, as a factor involved in necrotic cell death (cytotoxicity) induced by TcdB18, but whether it functions as a TcdB receptor remains to be established.Here we carried out unbiased genome-wide screens using the CRISPR (clustered regularly interspaced short palindromic repeats) / Cas9 approach19,20 and identified the members of the Frizzled (FZDs) family as TcdB receptors. Utilizing colonic organoid models and FZD7 KO mice, we established FZDs as physiologically relevant receptors for TcdB in the colonic epithelium.

Introduction

Infection of the colon by the opportunistic Gram-positive bacterium Clostridium difficile leads to a range of manifestations from diarrhea to life-threatening pseudomembranous colitis and toxic megacolon1–5. It is the most common cause of antibiotic-associated diarrhea and a leading cause of gastroenteritis-associated death in developed countries, accounting for nearly a half-million cases and 29,000 deaths annually in the United States6. Two homologous exotoxins, toxin A (TcdA) and toxin B (TcdB), are the causal agents for diseases associated with Clostridium difficile infection (CDI)4,7–9. These toxins enter cells via receptor-mediated endocytosis and inactivate small GTPases by glucosylating a key residue, resulting in cell-rounding and eventual death of cells4,7,10. Of the two toxins, TcdB alone is capable of causing the full spectrum of diseases, as TcdA−B+ strains have been clinically isolated and engineered TcdA−B+ strains induced death in animal models11–14.

How TcdB targets the colonic epithelium remains unknown. TcdB can enter a variety of cell lines, suggesting that its receptor(s) are widely expressed in transformed cells. It has also been reported that TcdB is enriched in the heart after injection into zebrafish embryos15. Chondroitin sulfate proteoglycan 4 (CSPG4, also known as neuron-glial antigen 2 (NG2)) has been identified as a TcdB receptor in a shRNA-mediated knock-down (KD) screen16, and was shown to be a functional receptor for TcdB in HeLa cells and in HT-29 cells, a human colorectal cell line. However, CSPG4 is not expressed in the colonic epithelium17. Poliovirus receptor-like 3 (PVRL3) was recently identified from a gene-trap insertional mutagenesis screen in Caco-2 cells, a human colorectal cell line, as a factor involved in necrotic cell death (cytotoxicity) induced by TcdB18, but whether it functions as a TcdB receptor remains to be established.

Here we carried out unbiased genome-wide screens using the CRISPR (clustered regularly interspaced short palindromic repeats) / Cas9 approach19,20 and identified the members of the Frizzled (FZDs) family as TcdB receptors. Utilizing colonic organoid models and FZD7 KO mice, we established FZDs as physiologically relevant receptors for TcdB in the colonic epithelium.

ResultsCRISPR/Cas9 screen for TcdB receptorsThe C-terminal domains of TcdA and TcdB contain a region known as combined repetitive oligopeptides (CROPs) (Extended Data Fig. 1a), which can bind carbohydrates and may mediate toxin binding to cells21. Recent studies suggested the presence of an additional receptor-binding region beyond the CROPs22–25. Consistently, we found that a truncated toxin (TcdB1-1830) lacking the CROPs induced cell-rounding in various cell lines at picomolar concentrations (Extended Data Figs. 1b–d)26. In order to identify both the receptor(s) recognized by the CROPs and the receptor(s) recognized by other regions, we carried out two separate screens, with either full-length TcdB or TcdB1-1830 (Fig. 1a).HeLa cells that stably express RNA-guided endonuclease Cas9 were transduced with lentiviral libraries that express short guide RNAs (sgRNA) targeting 19,052 genes, with six sgRNAs per gene19. After four rounds of selection with increasing concentrations of toxins, the sgRNA sequences from the surviving cells were identified via next-generation sequencing (NGS). We ranked candidate genes based on the number of unique sgRNAs versus NGS reads (Fig. 1b, c, Extended Data Fig. 2, Source Data).UDP-glucose pyrophosphorylase (UGP2) stood out in both screens (Fig. 1b, c). UGP2 is a cytosolic enzyme producing UDP-glucose, which is used by TcdA and TcdB to glucosylate small GTPases. Mutations in UGP2 have been shown to render cells resistant to TcdA and TcdB27,28. Besides UGP2, the top hit from the full-length TcdB screen is CSPG4 (Fig. 1b), confirming a previous report that identified CSPG4 as a TcdB receptor16. The highest-ranking plasma membrane protein from the TcdB1-1830 screen is Frizzled 2 (FZD2) (Fig. 1c). FZD2 is a member of the Frizzled family of receptors for Wnt signaling, which is a key signaling pathway regulating proliferation and self-renewal of colonic epithelial cells29,30. Besides FZD2, an unusual group of high-ranking hits are subunits of the endoplasmic reticulum (ER) membrane protein complex (EMC)31,32.To validate the screening results, we generated UGP2−/−, CSPG4−/−, FZD2−/−, and EMC4−/− HeLa cell lines using the CRISPR/Cas9 approach (Supplementary Table 1). Two additional KO cells were also generated and examined: SGMS1−/− (sphingomyelin synthase 1) and IL1RAPL2−/− (Interleukin-1 receptor accessory protein-like 2) (Fig. 1c). These cells were challenged with either TcdB or TcdB1-1830 using the cytopathic cell-rounding assay1 (Extended Data Fig. 3a, b). UGP2−/− cells were highly resistant (~3000-fold) to both TcdB and TcdB1-1830 compared to wild-type (WT) HeLa cells (Fig. 2a, Supplementary Table 2). CSPG4−/− cells showed increased resistance to TcdB (~240-fold), but not to TcdB1-1830. FZD2−/− and EMC4−/− cells both showed increased resistance (~15 and ~11-fold, respectively) to TcdB1-1830, but not to TcdB. SGMS1−/− and IL1RAPL2−/− cells did not show significant changes in the sensitivity to toxins under our assay conditions. Increased resistance of UGP2−/−, CSPG4−/−, FZD2−/−, and EMC4−/− cells was further confirmed by immunoblot analysis for glucosylation of Rac1, a small GTPase (Extended Data Fig. 3c).CSPG4 is a CROPs-dependent TcdB receptorWe next focused on CSPG4 and FZD2 as potential TcdB receptors. Binding of TcdB to CSPG4−/− cells was reduced compared to WT HeLa cells (Fig. 2b). Ectopic expression of NG2 (rat CSPG4) restored binding and entry of TcdB (Fig. 2b, d). TcdB binds directly to purified NG2 extracellular (EC) fragments, independent of the chondroitin sulfate glycan in NG2 (Extended Data Fig. 4a, b). The above results are consistent with a previous report16. However, contrary to the previous suggestion that CSPG4 does not bind to the CROPs16, we conclude that the CROPs is essential for TcdB binding to CSPG4 because: (1) TcdB1-1830 does not bind to either purified NG2-EC or NG2 expressed on cell surfaces (Extended Data Fig. 4b, c); (2) CSPG4−/− cells showed similar levels of sensitivity to TcdB1-1830 as WT cells (Fig. 2a); and (3) the CROPs is capable of competing with TcdB for binding to CSPG4 on cell surfaces (Extended Data Fig. 4d, e). We note that the previous study utilized TcdB1851-2366 as the CROPs16. Recent structural studies confirmed that the CROPs region starts around residue 1834 instead of 185133. Here we used full-length CROPs (residues 1831–2366). It is possible that the region 1831–1850 is required for TcdB binding to CSPG4.FZDs are CROPs-independent receptorsTransfecting CSPG4−/− cells with FZD2 also increased binding of TcdB (Fig. 2c) and restored entry of TcdB into CSPG4−/− cells (Fig. 2d), suggesting that FZD2 is an alternative receptor. In contrast to CSPG4, ectopically expressed FZD2 increased binding of TcdB1-1830 and TcdB1501-2366 on cell surfaces, but not the CROPs (Extended Data Figs. 4c, f), suggesting that it is a CROPs-independent receptor. FZD2 is a 7-pass transmembrane protein and contains a sole distinct extracellular domain known as the cysteine-rich domain (CRD) (Fig. 2e)29. Consistently, recombinant Fc-tagged FZD2-CRD binds directly to glutathione S-transferase (GST) tagged TcdB1501-2366, but not to the CROPs in pull-down assays (Fig. 2e).It is possible that CSPG4 is expressed at a much higher level than FZD2 in HeLa cells, which may explain why TcdB binding to CSPG4−/− cells is barely detectable using immunostaining and immunoblot assays. Notably, TcdB can enter CSPG4−/− cells at picomolar concentrations, as detected by the sensitive cytopathic cell-rounding assay (Fig. 2f). Such entry is blocked by recombinant FZD2-CRD in a dose-dependent manner, as evidenced by a lack of cell-rounding and Rac1 glucosylation (Fig. 2f, g), suggesting that endogenous FZD2 mediates TcdB binding and entry in CSPG4−/− cells.The FZD family includes ten members (FZD1–10) in humans29. The ectopic expression of FZD1, 2, and 7 each increased binding of TcdB to CSPG4−/− cells (Fig. 2h, Extended Data Fig. 5a), likely because the CRDs of FZD1, 2, and 7 share ~98% sequence similarity (Extended Data Fig. 5b)29. Consistently, FZD7-CRD, but not FZD8-CRD, when expressed on cell surfaces via a fused GPI anchor, mediated strong binding of TcdB to cells (Extended Data Fig. 5c).HeLa cells express multiple FZDs34. We next generated FZD1 and FZD7 single-KO HeLa cells, as well as FZD1/2/7 triple-KO cells. FZD1/2/7−/− cells exhibited normal growth rates, likely because HeLa cells still express other FZDs. FZD1−/− and FZD7−/− cells showed reductions in sensitivity to TcdB1-1830 similar to those of FZD2−/− cells (Fig. 2i). FZD1/2/7−/− cells were highly resistant to TcdB1-1830 (~300-fold), confirming that FZD1, 2, and 7 all contribute to TcdB1-1830 entry into HeLa cells. Transfection of FZD1, 2, or 7 restored TcdB1-1830 entry into FZD1/2/7−/− cells (Fig. 2j). FZD1/2/7−/− cells also become ~10-fold more resistant to full-length TcdB than WT cells (Fig. 2i), indicating that endogenous FZD1/2/7 are responsible for a portion of TcdB entry into WT HeLa cells. FZD1/2/7−/− cells showed the same level of sensitivity to TcdA as WT cells (Extended Data Fig. 5d), confirming that the resistance of FZD1/2/7−/− cells is specific to TcdB.We further quantified the binding kinetics between CRDs of FZD1, 2, and 7 and TcdB using the bio-layer interferometry (BLI) assay. The results revealed a single binding site with low nanomolar affinities (KD = 32 nM for FZD1, 19 nM for FZD2, and 21 nM for FZD7) (Fig. 2k, Extended Data Fig. 5e, Supplementary Table 3). Furthermore, FZD2-CRD showed the same binding affinity to TcdB1-1830 (KD = 17 nM) as to full-length TcdB (Extended Data Fig. 5f). FZD5-CRD also binds to TcdB when measured by the sensitive BLI assay, but with a much weaker affinity than FZD1/2/7 (KD = 670 nM, Fig. 2k, Extended Data Fig. 5e). It is possible that additional FZD family members may function as low-affinity receptors for TcdB.The finding that EMC4−/− cells showed a similar level of toxin resistance as FZD2−/− cells is also consistent with FZDs being TcdB receptors (Fig. 2a). Although its function remains to be established, the EMC appears to be critical for the folding/stability of multi-transmembrane proteins35,36. Consistently, expression of transfected FZD1, 2, or 7 was reduced in EMC4−/− cells compared to WT cells (Extended Data Fig. 5g, h).CSPG4 versus FZDs in cell linesWe next addressed whether TcdB is capable of simultaneous binding to both CSPG4 and FZDs. As shown in Extended Data Fig. 6a, FZD2-CRD binds to TcdB pre-bound by immobilized NG2-EC on the microtiter plate, confirming that CSPG4 and FZDs do not compete with each other for binding to TcdB.We then examined the receptors responsible for TcdB entry in HT-29 and Caco-2 cells, which are known to express multiple FZDs37. FZD2-CRD protected both HT-29 and Caco-2 from TcdB1-1830 (Fig. 3a, b), suggesting that FZDs are functional receptors in these two cell lines. Interestingly, CSPG4 is expressed at high levels in HeLa, much lower levels in HT-29, and is undetectable in Caco-2 cells (Fig. 3c). Consistently, NG2-EC alone was sufficient to reduce TcdB entry into HeLa cells, whereas a combination of NG2-EC and FZD2-CRD was required to reduce TcdB entry into HT-29 cells, and FZD2-CRD alone protected Caco-2 cells (Fig. 3d, Extended Data Fig. 6b). These data suggest that CSPG4 and FZDs represent non-competing TcdB receptors, each capable of mediating binding and entry of TcdB. Their particular contribution in a given cell type may depend on their expression levels.We also tested the potential role of PVRL3. Ectopically expressed PVRL3 did not increase either binding or entry of TcdB into CSPG4−/− HeLa cells (Extended Data Fig. 7a, b). The recombinant ecto-domain of PVRL3 failed to protect Caco-2 cells from TcdB in the cytopathic cell-rounding assay, whereas FZD2-CRD protected cells (Extended Data Fig. 7c). Thus, PVRL3 is not likely a relevant receptor for the cytopathic effect of TcdB in HeLa and Caco-2 cells.FZDs are TcdB receptors in colonic organoidsTo determine the receptors that mediate TcdB entry into the colonic epithelium, we first utilized colonic organoids, an in vitro “mini-gut” model that recapitulates many important features of normal colonic epithelium38. Exposure to TcdB caused dose-dependent atrophy and death of organoids, which was quantified using a viability assay (Fig. 4a). We found that TcdB1-1830 and TcdB were equally potent, suggesting that CSPG4 is not a relevant receptor in colonic organoids (Extended Data Fig. 8a). It has been reported that CSPG4 is not expressed in the colonic epithelium17, which was confirmed by immunoblot analysis of colonic organoids and isolated mouse colonic epithelium (Extended Data Fig. 8b).We next utilized colonic organoids cultured from FZD7 KO mice, combined with adenovirus-mediated KD of FZD1 and FZD2 (Extended Data Fig. 8c, d). FZD7 is critical for maintaining intestinal organoids, but FZD7−/− organoids can be cultured in the presence of CHIR99021, a small molecule inhibitor of glycogen synthase kinase-3 (GSK3), which activates Wnt/β-catenin signaling downstream of FZDs39. FZD7−/− organoids showed 3-fold more resistance to TcdB than WT organoids (Fig. 4b). Further KD of FZD1/2 in FZD7−/− organoids yielded 9-fold greater resistance to TcdB than WT organoids (Fig. 4b), demonstrating that FZDs are relevant TcdB receptors in colonic organoids.As both TcdB and Wnt bind to the FZD-CRD, we examined whether TcdB binding competes with Wnt and inhibits Wnt signaling. We utilized a non-toxic TcdB fragment (residues 1114–1835), which contains the FZD binding region but not the enzymatic domain of TcdB. This fragment blocked Wnt3a-mediated signaling in 293T cells in a dose-dependent manner, demonstrated by the TOPFLASH/TK-Renilla dual luciferase reporter assay (Fig. 4c, Extended Data Fig. 9a), as well as by phosphorylation levels of LRP6 (a FZD co-receptor) and Dvl2 (a downstream Wnt signaling component) (Extended Data Fig. 9b)29. TcdB1114-1835 did not glucosylate small GTPases in colonic organoids (Extended Data Fig. 9c), yet it inhibited organoid growth and induced death (Fig. 4d, Extended Data Fig. 9d, e). The death of colonic organoids was rescued by CHIR99021, demonstrating that the effect of TcdB1114-1835 is due to blockage of Wnt signaling. These data raised the intriguing possibility that binding of TcdB to FZDs may directly contribute to disruption of the colon epithelium by inhibiting Wnt signaling.FZDs are TcdB receptors in colonic epitheliumFinally, we examined the colonic epithelium in vivo. Immunohistochemistry (IHC) analysis showed that FZD2 and FZD7 are expressed in mouse and human colonic epithelium (Fig. 5a, b, Extended Data Fig. 10a–f). In contrast, CSPG4 is predominantly expressed in the multi-nucleated sub-epithelial cells termed ISEMFs (intestinal sub-epithelial myofibroblasts) and is not detectable in the colonic epithelium (Fig. 5c, Extended Data Figs. 10c), which is consistent with a previous report17.As TcdB is released into the lumen of the colon during CDI, we developed a model in which we injected TcdB directly into the lumen of ligated colon segments in mice (Fig. 5d), which resulted in binding and entry of TcdB into the colonic epithelium (Fig. 5e). Co-injection of FZD2-CRD largely abolished binding of TcdB (Fig. 5e), suggesting that FZDs are the dominant receptors in the colonic epithelium.To further verify the role of FZDs in vivo, we turned to FZD KO mouse models. FZD2/7 double-KO mice are embryonic lethal, and FZD2−/− mice also displayed developmental defects40. FZD7−/− mice appear to develop normally and exhibit no overt intestinal defects under basal conditions39,40. Thus, we chose FZD7−/− mice to assess whether a loss of a major colonic FZD member may reduce TcdB toxicity in vivo. To focus the analysis on the colonic epithelium and avoid the potential effects of TcdB entry into CSPG4-expressing ISEMFs, we utilized TcdB1-1830 and injected the toxin into the lumen of ligated colon segments of live mice. Following an 8-hour incubation period, fluid accumulation was observed in WT mice, but was significantly reduced in FZD7−/− mice (Fig. 5f). Histological scoring revealed extensive disruption of the epithelium, inflammatory cell filtration, and edema in WT mice, but much less in FZD7−/− mice (Fig. 5g, Extended Data Fig. 10g). To further assess epithelial integrity, we performed immunofluorescent analysis on colonic sections for the cell-cell junction markers Claudin-3 and ZO-1. Both markers were extensively disrupted in WT mice after exposure to TcdB1-1830 but remained largely intact in FZD7−/− mice (Fig. 5h, Extended Data Fig. 10h). Together, these data demonstrate that FZDs are physiologically relevant receptors for TcdB in the colonic epithelium in vivo.

CRISPR/Cas9 screen for TcdB receptorsThe C-terminal domains of TcdA and TcdB contain a region known as combined repetitive oligopeptides (CROPs) (Extended Data Fig. 1a), which can bind carbohydrates and may mediate toxin binding to cells21. Recent studies suggested the presence of an additional receptor-binding region beyond the CROPs22–25. Consistently, we found that a truncated toxin (TcdB1-1830) lacking the CROPs induced cell-rounding in various cell lines at picomolar concentrations (Extended Data Figs. 1b–d)26. In order to identify both the receptor(s) recognized by the CROPs and the receptor(s) recognized by other regions, we carried out two separate screens, with either full-length TcdB or TcdB1-1830 (Fig. 1a).HeLa cells that stably express RNA-guided endonuclease Cas9 were transduced with lentiviral libraries that express short guide RNAs (sgRNA) targeting 19,052 genes, with six sgRNAs per gene19. After four rounds of selection with increasing concentrations of toxins, the sgRNA sequences from the surviving cells were identified via next-generation sequencing (NGS). We ranked candidate genes based on the number of unique sgRNAs versus NGS reads (Fig. 1b, c, Extended Data Fig. 2, Source Data).UDP-glucose pyrophosphorylase (UGP2) stood out in both screens (Fig. 1b, c). UGP2 is a cytosolic enzyme producing UDP-glucose, which is used by TcdA and TcdB to glucosylate small GTPases. Mutations in UGP2 have been shown to render cells resistant to TcdA and TcdB27,28. Besides UGP2, the top hit from the full-length TcdB screen is CSPG4 (Fig. 1b), confirming a previous report that identified CSPG4 as a TcdB receptor16. The highest-ranking plasma membrane protein from the TcdB1-1830 screen is Frizzled 2 (FZD2) (Fig. 1c). FZD2 is a member of the Frizzled family of receptors for Wnt signaling, which is a key signaling pathway regulating proliferation and self-renewal of colonic epithelial cells29,30. Besides FZD2, an unusual group of high-ranking hits are subunits of the endoplasmic reticulum (ER) membrane protein complex (EMC)31,32.To validate the screening results, we generated UGP2−/−, CSPG4−/−, FZD2−/−, and EMC4−/− HeLa cell lines using the CRISPR/Cas9 approach (Supplementary Table 1). Two additional KO cells were also generated and examined: SGMS1−/− (sphingomyelin synthase 1) and IL1RAPL2−/− (Interleukin-1 receptor accessory protein-like 2) (Fig. 1c). These cells were challenged with either TcdB or TcdB1-1830 using the cytopathic cell-rounding assay1 (Extended Data Fig. 3a, b). UGP2−/− cells were highly resistant (~3000-fold) to both TcdB and TcdB1-1830 compared to wild-type (WT) HeLa cells (Fig. 2a, Supplementary Table 2). CSPG4−/− cells showed increased resistance to TcdB (~240-fold), but not to TcdB1-1830. FZD2−/− and EMC4−/− cells both showed increased resistance (~15 and ~11-fold, respectively) to TcdB1-1830, but not to TcdB. SGMS1−/− and IL1RAPL2−/− cells did not show significant changes in the sensitivity to toxins under our assay conditions. Increased resistance of UGP2−/−, CSPG4−/−, FZD2−/−, and EMC4−/− cells was further confirmed by immunoblot analysis for glucosylation of Rac1, a small GTPase (Extended Data Fig. 3c).

CRISPR/Cas9 screen for TcdB receptors

The C-terminal domains of TcdA and TcdB contain a region known as combined repetitive oligopeptides (CROPs) (Extended Data Fig. 1a), which can bind carbohydrates and may mediate toxin binding to cells21. Recent studies suggested the presence of an additional receptor-binding region beyond the CROPs22–25. Consistently, we found that a truncated toxin (TcdB1-1830) lacking the CROPs induced cell-rounding in various cell lines at picomolar concentrations (Extended Data Figs. 1b–d)26. In order to identify both the receptor(s) recognized by the CROPs and the receptor(s) recognized by other regions, we carried out two separate screens, with either full-length TcdB or TcdB1-1830 (Fig. 1a).

HeLa cells that stably express RNA-guided endonuclease Cas9 were transduced with lentiviral libraries that express short guide RNAs (sgRNA) targeting 19,052 genes, with six sgRNAs per gene19. After four rounds of selection with increasing concentrations of toxins, the sgRNA sequences from the surviving cells were identified via next-generation sequencing (NGS). We ranked candidate genes based on the number of unique sgRNAs versus NGS reads (Fig. 1b, c, Extended Data Fig. 2, Source Data).

UDP-glucose pyrophosphorylase (UGP2) stood out in both screens (Fig. 1b, c). UGP2 is a cytosolic enzyme producing UDP-glucose, which is used by TcdA and TcdB to glucosylate small GTPases. Mutations in UGP2 have been shown to render cells resistant to TcdA and TcdB27,28. Besides UGP2, the top hit from the full-length TcdB screen is CSPG4 (Fig. 1b), confirming a previous report that identified CSPG4 as a TcdB receptor16. The highest-ranking plasma membrane protein from the TcdB1-1830 screen is Frizzled 2 (FZD2) (Fig. 1c). FZD2 is a member of the Frizzled family of receptors for Wnt signaling, which is a key signaling pathway regulating proliferation and self-renewal of colonic epithelial cells29,30. Besides FZD2, an unusual group of high-ranking hits are subunits of the endoplasmic reticulum (ER) membrane protein complex (EMC)31,32.

To validate the screening results, we generated UGP2−/−, CSPG4−/−, FZD2−/−, and EMC4−/− HeLa cell lines using the CRISPR/Cas9 approach (Supplementary Table 1). Two additional KO cells were also generated and examined: SGMS1−/− (sphingomyelin synthase 1) and IL1RAPL2−/− (Interleukin-1 receptor accessory protein-like 2) (Fig. 1c). These cells were challenged with either TcdB or TcdB1-1830 using the cytopathic cell-rounding assay1 (Extended Data Fig. 3a, b). UGP2−/− cells were highly resistant (~3000-fold) to both TcdB and TcdB1-1830 compared to wild-type (WT) HeLa cells (Fig. 2a, Supplementary Table 2). CSPG4−/− cells showed increased resistance to TcdB (~240-fold), but not to TcdB1-1830. FZD2−/− and EMC4−/− cells both showed increased resistance (~15 and ~11-fold, respectively) to TcdB1-1830, but not to TcdB. SGMS1−/− and IL1RAPL2−/− cells did not show significant changes in the sensitivity to toxins under our assay conditions. Increased resistance of UGP2−/−, CSPG4−/−, FZD2−/−, and EMC4−/− cells was further confirmed by immunoblot analysis for glucosylation of Rac1, a small GTPase (Extended Data Fig. 3c).

CSPG4 is a CROPs-dependent TcdB receptorWe next focused on CSPG4 and FZD2 as potential TcdB receptors. Binding of TcdB to CSPG4−/− cells was reduced compared to WT HeLa cells (Fig. 2b). Ectopic expression of NG2 (rat CSPG4) restored binding and entry of TcdB (Fig. 2b, d). TcdB binds directly to purified NG2 extracellular (EC) fragments, independent of the chondroitin sulfate glycan in NG2 (Extended Data Fig. 4a, b). The above results are consistent with a previous report16. However, contrary to the previous suggestion that CSPG4 does not bind to the CROPs16, we conclude that the CROPs is essential for TcdB binding to CSPG4 because: (1) TcdB1-1830 does not bind to either purified NG2-EC or NG2 expressed on cell surfaces (Extended Data Fig. 4b, c); (2) CSPG4−/− cells showed similar levels of sensitivity to TcdB1-1830 as WT cells (Fig. 2a); and (3) the CROPs is capable of competing with TcdB for binding to CSPG4 on cell surfaces (Extended Data Fig. 4d, e). We note that the previous study utilized TcdB1851-2366 as the CROPs16. Recent structural studies confirmed that the CROPs region starts around residue 1834 instead of 185133. Here we used full-length CROPs (residues 1831–2366). It is possible that the region 1831–1850 is required for TcdB binding to CSPG4.

CSPG4 is a CROPs-dependent TcdB receptor

We next focused on CSPG4 and FZD2 as potential TcdB receptors. Binding of TcdB to CSPG4−/− cells was reduced compared to WT HeLa cells (Fig. 2b). Ectopic expression of NG2 (rat CSPG4) restored binding and entry of TcdB (Fig. 2b, d). TcdB binds directly to purified NG2 extracellular (EC) fragments, independent of the chondroitin sulfate glycan in NG2 (Extended Data Fig. 4a, b). The above results are consistent with a previous report16. However, contrary to the previous suggestion that CSPG4 does not bind to the CROPs16, we conclude that the CROPs is essential for TcdB binding to CSPG4 because: (1) TcdB1-1830 does not bind to either purified NG2-EC or NG2 expressed on cell surfaces (Extended Data Fig. 4b, c); (2) CSPG4−/− cells showed similar levels of sensitivity to TcdB1-1830 as WT cells (Fig. 2a); and (3) the CROPs is capable of competing with TcdB for binding to CSPG4 on cell surfaces (Extended Data Fig. 4d, e). We note that the previous study utilized TcdB1851-2366 as the CROPs16. Recent structural studies confirmed that the CROPs region starts around residue 1834 instead of 185133. Here we used full-length CROPs (residues 1831–2366). It is possible that the region 1831–1850 is required for TcdB binding to CSPG4.

FZDs are CROPs-independent receptorsTransfecting CSPG4−/− cells with FZD2 also increased binding of TcdB (Fig. 2c) and restored entry of TcdB into CSPG4−/− cells (Fig. 2d), suggesting that FZD2 is an alternative receptor. In contrast to CSPG4, ectopically expressed FZD2 increased binding of TcdB1-1830 and TcdB1501-2366 on cell surfaces, but not the CROPs (Extended Data Figs. 4c, f), suggesting that it is a CROPs-independent receptor. FZD2 is a 7-pass transmembrane protein and contains a sole distinct extracellular domain known as the cysteine-rich domain (CRD) (Fig. 2e)29. Consistently, recombinant Fc-tagged FZD2-CRD binds directly to glutathione S-transferase (GST) tagged TcdB1501-2366, but not to the CROPs in pull-down assays (Fig. 2e).It is possible that CSPG4 is expressed at a much higher level than FZD2 in HeLa cells, which may explain why TcdB binding to CSPG4−/− cells is barely detectable using immunostaining and immunoblot assays. Notably, TcdB can enter CSPG4−/− cells at picomolar concentrations, as detected by the sensitive cytopathic cell-rounding assay (Fig. 2f). Such entry is blocked by recombinant FZD2-CRD in a dose-dependent manner, as evidenced by a lack of cell-rounding and Rac1 glucosylation (Fig. 2f, g), suggesting that endogenous FZD2 mediates TcdB binding and entry in CSPG4−/− cells.The FZD family includes ten members (FZD1–10) in humans29. The ectopic expression of FZD1, 2, and 7 each increased binding of TcdB to CSPG4−/− cells (Fig. 2h, Extended Data Fig. 5a), likely because the CRDs of FZD1, 2, and 7 share ~98% sequence similarity (Extended Data Fig. 5b)29. Consistently, FZD7-CRD, but not FZD8-CRD, when expressed on cell surfaces via a fused GPI anchor, mediated strong binding of TcdB to cells (Extended Data Fig. 5c).HeLa cells express multiple FZDs34. We next generated FZD1 and FZD7 single-KO HeLa cells, as well as FZD1/2/7 triple-KO cells. FZD1/2/7−/− cells exhibited normal growth rates, likely because HeLa cells still express other FZDs. FZD1−/− and FZD7−/− cells showed reductions in sensitivity to TcdB1-1830 similar to those of FZD2−/− cells (Fig. 2i). FZD1/2/7−/− cells were highly resistant to TcdB1-1830 (~300-fold), confirming that FZD1, 2, and 7 all contribute to TcdB1-1830 entry into HeLa cells. Transfection of FZD1, 2, or 7 restored TcdB1-1830 entry into FZD1/2/7−/− cells (Fig. 2j). FZD1/2/7−/− cells also become ~10-fold more resistant to full-length TcdB than WT cells (Fig. 2i), indicating that endogenous FZD1/2/7 are responsible for a portion of TcdB entry into WT HeLa cells. FZD1/2/7−/− cells showed the same level of sensitivity to TcdA as WT cells (Extended Data Fig. 5d), confirming that the resistance of FZD1/2/7−/− cells is specific to TcdB.We further quantified the binding kinetics between CRDs of FZD1, 2, and 7 and TcdB using the bio-layer interferometry (BLI) assay. The results revealed a single binding site with low nanomolar affinities (KD = 32 nM for FZD1, 19 nM for FZD2, and 21 nM for FZD7) (Fig. 2k, Extended Data Fig. 5e, Supplementary Table 3). Furthermore, FZD2-CRD showed the same binding affinity to TcdB1-1830 (KD = 17 nM) as to full-length TcdB (Extended Data Fig. 5f). FZD5-CRD also binds to TcdB when measured by the sensitive BLI assay, but with a much weaker affinity than FZD1/2/7 (KD = 670 nM, Fig. 2k, Extended Data Fig. 5e). It is possible that additional FZD family members may function as low-affinity receptors for TcdB.The finding that EMC4−/− cells showed a similar level of toxin resistance as FZD2−/− cells is also consistent with FZDs being TcdB receptors (Fig. 2a). Although its function remains to be established, the EMC appears to be critical for the folding/stability of multi-transmembrane proteins35,36. Consistently, expression of transfected FZD1, 2, or 7 was reduced in EMC4−/− cells compared to WT cells (Extended Data Fig. 5g, h).

FZDs are CROPs-independent receptors

Transfecting CSPG4−/− cells with FZD2 also increased binding of TcdB (Fig. 2c) and restored entry of TcdB into CSPG4−/− cells (Fig. 2d), suggesting that FZD2 is an alternative receptor. In contrast to CSPG4, ectopically expressed FZD2 increased binding of TcdB1-1830 and TcdB1501-2366 on cell surfaces, but not the CROPs (Extended Data Figs. 4c, f), suggesting that it is a CROPs-independent receptor. FZD2 is a 7-pass transmembrane protein and contains a sole distinct extracellular domain known as the cysteine-rich domain (CRD) (Fig. 2e)29. Consistently, recombinant Fc-tagged FZD2-CRD binds directly to glutathione S-transferase (GST) tagged TcdB1501-2366, but not to the CROPs in pull-down assays (Fig. 2e).

It is possible that CSPG4 is expressed at a much higher level than FZD2 in HeLa cells, which may explain why TcdB binding to CSPG4−/− cells is barely detectable using immunostaining and immunoblot assays. Notably, TcdB can enter CSPG4−/− cells at picomolar concentrations, as detected by the sensitive cytopathic cell-rounding assay (Fig. 2f). Such entry is blocked by recombinant FZD2-CRD in a dose-dependent manner, as evidenced by a lack of cell-rounding and Rac1 glucosylation (Fig. 2f, g), suggesting that endogenous FZD2 mediates TcdB binding and entry in CSPG4−/− cells.

The FZD family includes ten members (FZD1–10) in humans29. The ectopic expression of FZD1, 2, and 7 each increased binding of TcdB to CSPG4−/− cells (Fig. 2h, Extended Data Fig. 5a), likely because the CRDs of FZD1, 2, and 7 share ~98% sequence similarity (Extended Data Fig. 5b)29. Consistently, FZD7-CRD, but not FZD8-CRD, when expressed on cell surfaces via a fused GPI anchor, mediated strong binding of TcdB to cells (Extended Data Fig. 5c).

HeLa cells express multiple FZDs34. We next generated FZD1 and FZD7 single-KO HeLa cells, as well as FZD1/2/7 triple-KO cells. FZD1/2/7−/− cells exhibited normal growth rates, likely because HeLa cells still express other FZDs. FZD1−/− and FZD7−/− cells showed reductions in sensitivity to TcdB1-1830 similar to those of FZD2−/− cells (Fig. 2i). FZD1/2/7−/− cells were highly resistant to TcdB1-1830 (~300-fold), confirming that FZD1, 2, and 7 all contribute to TcdB1-1830 entry into HeLa cells. Transfection of FZD1, 2, or 7 restored TcdB1-1830 entry into FZD1/2/7−/− cells (Fig. 2j). FZD1/2/7−/− cells also become ~10-fold more resistant to full-length TcdB than WT cells (Fig. 2i), indicating that endogenous FZD1/2/7 are responsible for a portion of TcdB entry into WT HeLa cells. FZD1/2/7−/− cells showed the same level of sensitivity to TcdA as WT cells (Extended Data Fig. 5d), confirming that the resistance of FZD1/2/7−/− cells is specific to TcdB.

We further quantified the binding kinetics between CRDs of FZD1, 2, and 7 and TcdB using the bio-layer interferometry (BLI) assay. The results revealed a single binding site with low nanomolar affinities (KD = 32 nM for FZD1, 19 nM for FZD2, and 21 nM for FZD7) (Fig. 2k, Extended Data Fig. 5e, Supplementary Table 3). Furthermore, FZD2-CRD showed the same binding affinity to TcdB1-1830 (KD = 17 nM) as to full-length TcdB (Extended Data Fig. 5f). FZD5-CRD also binds to TcdB when measured by the sensitive BLI assay, but with a much weaker affinity than FZD1/2/7 (KD = 670 nM, Fig. 2k, Extended Data Fig. 5e). It is possible that additional FZD family members may function as low-affinity receptors for TcdB.

The finding that EMC4−/− cells showed a similar level of toxin resistance as FZD2−/− cells is also consistent with FZDs being TcdB receptors (Fig. 2a). Although its function remains to be established, the EMC appears to be critical for the folding/stability of multi-transmembrane proteins35,36. Consistently, expression of transfected FZD1, 2, or 7 was reduced in EMC4−/− cells compared to WT cells (Extended Data Fig. 5g, h).

CSPG4 versus FZDs in cell linesWe next addressed whether TcdB is capable of simultaneous binding to both CSPG4 and FZDs. As shown in Extended Data Fig. 6a, FZD2-CRD binds to TcdB pre-bound by immobilized NG2-EC on the microtiter plate, confirming that CSPG4 and FZDs do not compete with each other for binding to TcdB.We then examined the receptors responsible for TcdB entry in HT-29 and Caco-2 cells, which are known to express multiple FZDs37. FZD2-CRD protected both HT-29 and Caco-2 from TcdB1-1830 (Fig. 3a, b), suggesting that FZDs are functional receptors in these two cell lines. Interestingly, CSPG4 is expressed at high levels in HeLa, much lower levels in HT-29, and is undetectable in Caco-2 cells (Fig. 3c). Consistently, NG2-EC alone was sufficient to reduce TcdB entry into HeLa cells, whereas a combination of NG2-EC and FZD2-CRD was required to reduce TcdB entry into HT-29 cells, and FZD2-CRD alone protected Caco-2 cells (Fig. 3d, Extended Data Fig. 6b). These data suggest that CSPG4 and FZDs represent non-competing TcdB receptors, each capable of mediating binding and entry of TcdB. Their particular contribution in a given cell type may depend on their expression levels.We also tested the potential role of PVRL3. Ectopically expressed PVRL3 did not increase either binding or entry of TcdB into CSPG4−/− HeLa cells (Extended Data Fig. 7a, b). The recombinant ecto-domain of PVRL3 failed to protect Caco-2 cells from TcdB in the cytopathic cell-rounding assay, whereas FZD2-CRD protected cells (Extended Data Fig. 7c). Thus, PVRL3 is not likely a relevant receptor for the cytopathic effect of TcdB in HeLa and Caco-2 cells.

CSPG4 versus FZDs in cell lines

We next addressed whether TcdB is capable of simultaneous binding to both CSPG4 and FZDs. As shown in Extended Data Fig. 6a, FZD2-CRD binds to TcdB pre-bound by immobilized NG2-EC on the microtiter plate, confirming that CSPG4 and FZDs do not compete with each other for binding to TcdB.

We then examined the receptors responsible for TcdB entry in HT-29 and Caco-2 cells, which are known to express multiple FZDs37. FZD2-CRD protected both HT-29 and Caco-2 from TcdB1-1830 (Fig. 3a, b), suggesting that FZDs are functional receptors in these two cell lines. Interestingly, CSPG4 is expressed at high levels in HeLa, much lower levels in HT-29, and is undetectable in Caco-2 cells (Fig. 3c). Consistently, NG2-EC alone was sufficient to reduce TcdB entry into HeLa cells, whereas a combination of NG2-EC and FZD2-CRD was required to reduce TcdB entry into HT-29 cells, and FZD2-CRD alone protected Caco-2 cells (Fig. 3d, Extended Data Fig. 6b). These data suggest that CSPG4 and FZDs represent non-competing TcdB receptors, each capable of mediating binding and entry of TcdB. Their particular contribution in a given cell type may depend on their expression levels.

We also tested the potential role of PVRL3. Ectopically expressed PVRL3 did not increase either binding or entry of TcdB into CSPG4−/− HeLa cells (Extended Data Fig. 7a, b). The recombinant ecto-domain of PVRL3 failed to protect Caco-2 cells from TcdB in the cytopathic cell-rounding assay, whereas FZD2-CRD protected cells (Extended Data Fig. 7c). Thus, PVRL3 is not likely a relevant receptor for the cytopathic effect of TcdB in HeLa and Caco-2 cells.

FZDs are TcdB receptors in colonic organoidsTo determine the receptors that mediate TcdB entry into the colonic epithelium, we first utilized colonic organoids, an in vitro “mini-gut” model that recapitulates many important features of normal colonic epithelium38. Exposure to TcdB caused dose-dependent atrophy and death of organoids, which was quantified using a viability assay (Fig. 4a). We found that TcdB1-1830 and TcdB were equally potent, suggesting that CSPG4 is not a relevant receptor in colonic organoids (Extended Data Fig. 8a). It has been reported that CSPG4 is not expressed in the colonic epithelium17, which was confirmed by immunoblot analysis of colonic organoids and isolated mouse colonic epithelium (Extended Data Fig. 8b).We next utilized colonic organoids cultured from FZD7 KO mice, combined with adenovirus-mediated KD of FZD1 and FZD2 (Extended Data Fig. 8c, d). FZD7 is critical for maintaining intestinal organoids, but FZD7−/− organoids can be cultured in the presence of CHIR99021, a small molecule inhibitor of glycogen synthase kinase-3 (GSK3), which activates Wnt/β-catenin signaling downstream of FZDs39. FZD7−/− organoids showed 3-fold more resistance to TcdB than WT organoids (Fig. 4b). Further KD of FZD1/2 in FZD7−/− organoids yielded 9-fold greater resistance to TcdB than WT organoids (Fig. 4b), demonstrating that FZDs are relevant TcdB receptors in colonic organoids.As both TcdB and Wnt bind to the FZD-CRD, we examined whether TcdB binding competes with Wnt and inhibits Wnt signaling. We utilized a non-toxic TcdB fragment (residues 1114–1835), which contains the FZD binding region but not the enzymatic domain of TcdB. This fragment blocked Wnt3a-mediated signaling in 293T cells in a dose-dependent manner, demonstrated by the TOPFLASH/TK-Renilla dual luciferase reporter assay (Fig. 4c, Extended Data Fig. 9a), as well as by phosphorylation levels of LRP6 (a FZD co-receptor) and Dvl2 (a downstream Wnt signaling component) (Extended Data Fig. 9b)29. TcdB1114-1835 did not glucosylate small GTPases in colonic organoids (Extended Data Fig. 9c), yet it inhibited organoid growth and induced death (Fig. 4d, Extended Data Fig. 9d, e). The death of colonic organoids was rescued by CHIR99021, demonstrating that the effect of TcdB1114-1835 is due to blockage of Wnt signaling. These data raised the intriguing possibility that binding of TcdB to FZDs may directly contribute to disruption of the colon epithelium by inhibiting Wnt signaling.

FZDs are TcdB receptors in colonic organoids

To determine the receptors that mediate TcdB entry into the colonic epithelium, we first utilized colonic organoids, an in vitro “mini-gut” model that recapitulates many important features of normal colonic epithelium38. Exposure to TcdB caused dose-dependent atrophy and death of organoids, which was quantified using a viability assay (Fig. 4a). We found that TcdB1-1830 and TcdB were equally potent, suggesting that CSPG4 is not a relevant receptor in colonic organoids (Extended Data Fig. 8a). It has been reported that CSPG4 is not expressed in the colonic epithelium17, which was confirmed by immunoblot analysis of colonic organoids and isolated mouse colonic epithelium (Extended Data Fig. 8b).

We next utilized colonic organoids cultured from FZD7 KO mice, combined with adenovirus-mediated KD of FZD1 and FZD2 (Extended Data Fig. 8c, d). FZD7 is critical for maintaining intestinal organoids, but FZD7−/− organoids can be cultured in the presence of CHIR99021, a small molecule inhibitor of glycogen synthase kinase-3 (GSK3), which activates Wnt/β-catenin signaling downstream of FZDs39. FZD7−/− organoids showed 3-fold more resistance to TcdB than WT organoids (Fig. 4b). Further KD of FZD1/2 in FZD7−/− organoids yielded 9-fold greater resistance to TcdB than WT organoids (Fig. 4b), demonstrating that FZDs are relevant TcdB receptors in colonic organoids.

As both TcdB and Wnt bind to the FZD-CRD, we examined whether TcdB binding competes with Wnt and inhibits Wnt signaling. We utilized a non-toxic TcdB fragment (residues 1114–1835), which contains the FZD binding region but not the enzymatic domain of TcdB. This fragment blocked Wnt3a-mediated signaling in 293T cells in a dose-dependent manner, demonstrated by the TOPFLASH/TK-Renilla dual luciferase reporter assay (Fig. 4c, Extended Data Fig. 9a), as well as by phosphorylation levels of LRP6 (a FZD co-receptor) and Dvl2 (a downstream Wnt signaling component) (Extended Data Fig. 9b)29. TcdB1114-1835 did not glucosylate small GTPases in colonic organoids (Extended Data Fig. 9c), yet it inhibited organoid growth and induced death (Fig. 4d, Extended Data Fig. 9d, e). The death of colonic organoids was rescued by CHIR99021, demonstrating that the effect of TcdB1114-1835 is due to blockage of Wnt signaling. These data raised the intriguing possibility that binding of TcdB to FZDs may directly contribute to disruption of the colon epithelium by inhibiting Wnt signaling.

FZDs are TcdB receptors in colonic epitheliumFinally, we examined the colonic epithelium in vivo. Immunohistochemistry (IHC) analysis showed that FZD2 and FZD7 are expressed in mouse and human colonic epithelium (Fig. 5a, b, Extended Data Fig. 10a–f). In contrast, CSPG4 is predominantly expressed in the multi-nucleated sub-epithelial cells termed ISEMFs (intestinal sub-epithelial myofibroblasts) and is not detectable in the colonic epithelium (Fig. 5c, Extended Data Figs. 10c), which is consistent with a previous report17.As TcdB is released into the lumen of the colon during CDI, we developed a model in which we injected TcdB directly into the lumen of ligated colon segments in mice (Fig. 5d), which resulted in binding and entry of TcdB into the colonic epithelium (Fig. 5e). Co-injection of FZD2-CRD largely abolished binding of TcdB (Fig. 5e), suggesting that FZDs are the dominant receptors in the colonic epithelium.To further verify the role of FZDs in vivo, we turned to FZD KO mouse models. FZD2/7 double-KO mice are embryonic lethal, and FZD2−/− mice also displayed developmental defects40. FZD7−/− mice appear to develop normally and exhibit no overt intestinal defects under basal conditions39,40. Thus, we chose FZD7−/− mice to assess whether a loss of a major colonic FZD member may reduce TcdB toxicity in vivo. To focus the analysis on the colonic epithelium and avoid the potential effects of TcdB entry into CSPG4-expressing ISEMFs, we utilized TcdB1-1830 and injected the toxin into the lumen of ligated colon segments of live mice. Following an 8-hour incubation period, fluid accumulation was observed in WT mice, but was significantly reduced in FZD7−/− mice (Fig. 5f). Histological scoring revealed extensive disruption of the epithelium, inflammatory cell filtration, and edema in WT mice, but much less in FZD7−/− mice (Fig. 5g, Extended Data Fig. 10g). To further assess epithelial integrity, we performed immunofluorescent analysis on colonic sections for the cell-cell junction markers Claudin-3 and ZO-1. Both markers were extensively disrupted in WT mice after exposure to TcdB1-1830 but remained largely intact in FZD7−/− mice (Fig. 5h, Extended Data Fig. 10h). Together, these data demonstrate that FZDs are physiologically relevant receptors for TcdB in the colonic epithelium in vivo.

FZDs are TcdB receptors in colonic epithelium

Finally, we examined the colonic epithelium in vivo. Immunohistochemistry (IHC) analysis showed that FZD2 and FZD7 are expressed in mouse and human colonic epithelium (Fig. 5a, b, Extended Data Fig. 10a–f). In contrast, CSPG4 is predominantly expressed in the multi-nucleated sub-epithelial cells termed ISEMFs (intestinal sub-epithelial myofibroblasts) and is not detectable in the colonic epithelium (Fig. 5c, Extended Data Figs. 10c), which is consistent with a previous report17.

As TcdB is released into the lumen of the colon during CDI, we developed a model in which we injected TcdB directly into the lumen of ligated colon segments in mice (Fig. 5d), which resulted in binding and entry of TcdB into the colonic epithelium (Fig. 5e). Co-injection of FZD2-CRD largely abolished binding of TcdB (Fig. 5e), suggesting that FZDs are the dominant receptors in the colonic epithelium.

To further verify the role of FZDs in vivo, we turned to FZD KO mouse models. FZD2/7 double-KO mice are embryonic lethal, and FZD2−/− mice also displayed developmental defects40. FZD7−/− mice appear to develop normally and exhibit no overt intestinal defects under basal conditions39,40. Thus, we chose FZD7−/− mice to assess whether a loss of a major colonic FZD member may reduce TcdB toxicity in vivo. To focus the analysis on the colonic epithelium and avoid the potential effects of TcdB entry into CSPG4-expressing ISEMFs, we utilized TcdB1-1830 and injected the toxin into the lumen of ligated colon segments of live mice. Following an 8-hour incubation period, fluid accumulation was observed in WT mice, but was significantly reduced in FZD7−/− mice (Fig. 5f). Histological scoring revealed extensive disruption of the epithelium, inflammatory cell filtration, and edema in WT mice, but much less in FZD7−/− mice (Fig. 5g, Extended Data Fig. 10g). To further assess epithelial integrity, we performed immunofluorescent analysis on colonic sections for the cell-cell junction markers Claudin-3 and ZO-1. Both markers were extensively disrupted in WT mice after exposure to TcdB1-1830 but remained largely intact in FZD7−/− mice (Fig. 5h, Extended Data Fig. 10h). Together, these data demonstrate that FZDs are physiologically relevant receptors for TcdB in the colonic epithelium in vivo.

DiscussionOur findings support a previously proposed two-receptor model for TcdB20, but with a notable amendment: FZDs and CSPG4 may act as receptors in different cell types. CSPG4 is expressed in the ISEMFs, which are involved in diverse processes from wound healing to inflammation41. Although the role of ISEMFs in CDI remains to be established, it is conceivable that targeting these cells by TcdB could contribute to the disease progress following FZD-mediated disruption of the colonic epithelium.Our unbiased genome-wide screens revealed multiple host factors involved in all major steps of toxin actions, from receptors (FZDs and CSPG4) to acidification in endosomes (Vacuolar-type H+-ATPase)42,43, to enzymatic activity in the cytosol (UGP2) (Extended Data Fig. 10i). Many other top-ranking hits remain to be validated, such as FBXO11 and enzymes involved in phospholipid metabolism/signaling including phosphatidylinositol 5-phosphate 4-kinase (PIP4K2B), phosphatidylinositol 4-kinase (PI4KB), and phospholipase C (PLCG1) (Extended Data Fig. 10i).Our screen identified many key players in Wnt signaling pathways, including APC, GSK-3β, Wnt5a, and LRP6 (Extended Data Fig. 10i). It has been suggested that TcdA attenuates Wnt signaling in cells, although the effects appear to be indirect, largely due to deactivation of Rho GTPase by TcdA44. Wnt signaling is particularly important for maintaining colonic stem cells30,45, which continuously give rise to new colonic epithelial cells. The health of these stem cells is critical for self-renewal and repair of the colonic epithelium, which has an extraordinarily fast turnover rate44. Our findings suggest that colonic stem cells are a major target of TcdB. The potential role of Wnt signaling inhibition in CDI pathogenesis, and the therapeutic potential of modulating Wnt signaling downstream of FZDs warrant further study. Finally, dysregulation of Wnt signaling pathways is associated with many cancers, particularly colorectal cancers30,46. The receptor-binding domain of TcdB, or its homologs, may serve as valuable tools and potential therapeutics for targeting Wnt signaling pathways.

Our findings support a previously proposed two-receptor model for TcdB20, but with a notable amendment: FZDs and CSPG4 may act as receptors in different cell types. CSPG4 is expressed in the ISEMFs, which are involved in diverse processes from wound healing to inflammation41. Although the role of ISEMFs in CDI remains to be established, it is conceivable that targeting these cells by TcdB could contribute to the disease progress following FZD-mediated disruption of the colonic epithelium.

Our unbiased genome-wide screens revealed multiple host factors involved in all major steps of toxin actions, from receptors (FZDs and CSPG4) to acidification in endosomes (Vacuolar-type H+-ATPase)42,43, to enzymatic activity in the cytosol (UGP2) (Extended Data Fig. 10i). Many other top-ranking hits remain to be validated, such as FBXO11 and enzymes involved in phospholipid metabolism/signaling including phosphatidylinositol 5-phosphate 4-kinase (PIP4K2B), phosphatidylinositol 4-kinase (PI4KB), and phospholipase C (PLCG1) (Extended Data Fig. 10i).

Our screen identified many key players in Wnt signaling pathways, including APC, GSK-3β, Wnt5a, and LRP6 (Extended Data Fig. 10i). It has been suggested that TcdA attenuates Wnt signaling in cells, although the effects appear to be indirect, largely due to deactivation of Rho GTPase by TcdA44. Wnt signaling is particularly important for maintaining colonic stem cells30,45, which continuously give rise to new colonic epithelial cells. The health of these stem cells is critical for self-renewal and repair of the colonic epithelium, which has an extraordinarily fast turnover rate44. Our findings suggest that colonic stem cells are a major target of TcdB. The potential role of Wnt signaling inhibition in CDI pathogenesis, and the therapeutic potential of modulating Wnt signaling downstream of FZDs warrant further study. Finally, dysregulation of Wnt signaling pathways is associated with many cancers, particularly colorectal cancers30,46. The receptor-binding domain of TcdB, or its homologs, may serve as valuable tools and potential therapeutics for targeting Wnt signaling pathways.

MethodsCell lines, antibodies, and constructsHeLa (H1, #CRL-1958), CHO (K1, #CCL-61), HT-29 (#HTB-38), Caco-2 (#HTB-37), and 293T (#CRL-3216) cells were originally obtained from ATCC. They tested negative for mycoplasma contamination, but have not been authenticated. The following mouse monoclonal antibodies were purchased from the indicated vendors: Rac1 (23A8, Abcam), non-glucosylated Rac1 (Clone 102, BD Biosciences), 1D4 tag (MA1-722, ThermoFisher Scientific), HA tag (16B12, Covance), β-actin (AC-15, Sigma), ZO-1 (339100, Life technology). Rabbit monoclonal IgG against human CSPG4 (ab139406) and rabbit polyclonal antibodies against FZD1 (ab150553), FZD2 (ab150477), FZD7 (ab51049), PVRL3 (ab63931), and Claudin-3 (ab15102) were all purchased from Abcam. Rabbit monoclonal antibodies against Dvl2 (30D2) and LRP6 (C5C7), and a rabbit polyclonal antibody against phosphorylated LRP6 (Ser1490) were all purchased from Cell Signaling. Chicken polyclonal IgY (#754A) against TcdB was purchased from List Biological Labs. Antibody validation is available on the manufacturers’ websites. A rabbit polyclonal antibody against rodent NG2 and a construct expressing full-length rat NG2 (in pcDNA vector) were both generated in W. Stallcup’s lab. 1D4-tagged full-length FZD1–10 constructs (in pRK5 vector) were originally generated in J. Nathans’ lab (Baltimore, MD) and were obtained from Addgene. FZD7 and FZD8 CRD-myc-GPI constructs were generously provided by J. Nathans and have been described previously47. Constructs expressing full-length human IL1RAPL2 and full-length PVRL3 were purchased from Vigene Biosciences. A construct expressing full-length mouse Syt II was described previously 48.TcdB and other recombinant proteinsRecombinant TcdB (from C. difficile strain VPI 10463) and TcdA were expressed in Bacillus megaterium as previously described49 and purified as His6-tagged proteins. TcdB1-1830 was cloned into pHis1522 vector (MoBiTec) and expressed in B. megaterium. TcdB1831-2366, TcdB1501-2366, and TcdB1114-1835 were cloned into pGEX-6P-1 or pET28a vectors and purified as GST-tagged or His6-tagged proteins in E. coli. NG2-EC (P1 and P2) was expressed in HEK293 cells, purified from medium with DEAE-Sepharose columns, and eluted with a gradient buffer (NaCl from 0.2 to 0.8 M, 50 mM Tris-Cl, pH 8.6) as previously described50. Recombinant human proteins were purchased from ACRO Biosystems (IgG1 Fc and FZD2-CRD-Fc), R&D Systems (FZD1-CRD-Fc, FZD5-CRD-Fc, and FZD7-CRD-Fc), and Sino Biologics (PVRL3-EC).Generating stable HeLa-Cas9 cells and lentivirus sgRNA librariesThe human codon-optimized sequence of S. pyogenes Cas9 was subcloned from plasmid lentiCas9-Blast (Addgene #52962) into the pQCXIH retroviral vector (Clontech), which was used to generate retroviruses to transduce HeLa cells. Mixed stable cells were selected in the presence of hygromycin B (200 μg/ml, Life Technologies). Lentivirus sgRNA libraries were generated following published protocols using the human GeCKO v2 sgRNA library (Addgene #1000000049)19. The GeCKO v2 library is composed of two half-libraries (library A and library B). Each half-library contains three unique sgRNA per gene and was independently screened with toxins. Cells were transduced with lentivirus-packaged sgRNA library at a MOI of 0.2.Screening CRISPR libraries with TcdB and TcdB1-1830For each CRISPR half-library of cells, 4×107 cells were plated onto two 15-cm culture dishes to ensure sufficient coverage of sgRNAs, with each sgRNA on average being represented about 650 times (i.e., there are on average 650 cells transduced with the same sgRNA). This over-representation rate was calculated from titration plates that were set up in parallel with the library. These cells were exposed to either TcdB or TcdB1-1830 for 48 h. Cells were then washed three times with phosphate-buffered saline (PBS) to remove loosely attached round-shaped cells. The remaining cells were re-seeded and cultured with normal medium without toxins until ~70% confluence. Cells were then subjected to the next round of screening with increased concentrations of toxins. Four rounds of screenings were carried out with TcdB (0.05, 0.1, 0.2, and 0.5 pM) and TcdB1-1830 (5, 10, 20, and 50 pM). The remaining cells were harvested and their genomic DNA extracted using the Blood and Cell Culture DNA mini kit (Qiagen). DNA fragments containing the sgRNA sequences were amplified by PCR using primers lentiGP-1_F (AATGGACTATCATATGCTTACCGTAACTTGAAAGTATTTCG) and lentiGP-3_R (ATGAATACTGCCATTTGTCTCAAGATCTAGTTACGC). Next-generation sequencing (Illumina MiSeq) was performed by a commercial vendor (Genewiz).Generating HeLa KO cell linesThe following sgRNA sequences were cloned into LentiGuide-Puro vectors (Addgene) to target the indicated genes: CCGGAGACACGGAGCAGTGG (CSPG4), GCGCTGCTGGGACATCGCCT (EMC4), ACCTTATACCACACAACATC (IL1RAPL2), TGCGAGCACTTCCCGCGCCA (FZD2), AGCGCATGACCACTACACTG (SGMS1), ACAGGCAGAAAACGGCTCCT (UGP2), GTGTAATGACAAGTTCGCCG (FZD1), and GAGAACGGTAAAGAGCGTCG (FZD7). HeLa-Cas9 cells were transduced with lentiviruses that express these sgRNAs. Mixed populations of stable cells were selected with puromycin (2.5 μg/ml) and hygromycin B (200 μg/ml). FZD1/2/7−/− cells were created by sequentially transducing FZD1 and FZD7 sgRNA lentiviruses into FZD2−/− cells and further selected in the presence of 100 pM TcdB1-1830. The mutagenesis rate in these mixed stable cells was determined by NGS (Supplementary Table 1).Cytopathic assayThe cytopathic effect (cell-rounding) of TcdA and TcdB was analyzed using standard cell-rounding assay as previously described1. Briefly, cells were exposed to a gradient of TcdB and TcdB1-1830 for 24 h. Phase-contrast images of cells were taken (Olympus IX51, 10–20X objectives). The numbers of round-shaped and normal shaped cells were counted manually. The percentage of round-shaped cells was plotted and fitted using the Origin software.Blocking TcdB entry into cells with NG2-EC and FZD2-CRD-FcRecombinant proteins used for cell protection assays were pre-filtered (0.22 μm, Millipore). Toxins were pre-incubated with FZD2-CRD-Fc and/or NG2-EC (P1) for 30 min on ice with a toxin/protein ratio of 1:400 (except when specifically noted in the figure legend). The mixtures were added into cell culture medium and cells were analyzed by the cytopathic assay.Transfection, TcdB binding to cells, and immunoblot analysisTransient transfection of HeLa cells was carried out using PolyJetTM (SignaGen). Binding of TcdB to cells was analyzed by exposing cells to TcdB or truncated TcdB fragments for 10 min at room temperature. Cells were washed three times with PBS and then either fixed for immunostaining analysis or harvested with RIPA buffer (50 mM Tris, 1% NP40, 150 mM NaCl, 0.5% sodium deoxycholate, 0.1% SDS, plus a protease inhibitor cocktail (Sigma-Aldrich)). Cell lysates were centrifuged and supernatants were subjected to SDS-PAGE and immunoblot analysis using the enhanced chemiluminescence method (Pierce). The full blot images are shown in Supplementary Figure.Pull-down assaysPull-down assays were carried out using glutathione Sepharose 4B as previously described48. Briefly, 5 μg of glutathione S-transferase (GST) tagged TcdB1831-2366 and TcdB1501-2366 were immobilized on glutathione beads and incubated with FZD2-CRD-Fc (10 nM) for 1 h at 4 °C. Beads were then washed, pelleted, boiled in SDS sample buffer, and subjected to immunoblot analysis.Biolayer interferometry (BLI) assayThe binding affinities between TcdB and FZD-CRDs were measured by BLI assay using the Blitz system (ForteBio). Briefly, the CRDs-Fc of FZD1, 2, 5, 7 or human IgG1 Fc (20 μg/ml) were immobilized onto capture biosensors (Dip and Read Anti-hIgG-Fc, ForteBio) and balanced with PBS. The biosensors were then exposed to TcdB or TcdB1-1830, followed by washing with PBS. Binding affinities (KD) were calculated using the Blitz system software (ForteBio).Wnt signaling assayThe TOPFLASH/TK-Renilla dual luciferase reporter assay was utilized to detect Wnt signaling activities as previously described51. Briefly, 293T cells in 24-well plates were co-transfected with TOPFLASH (50 ng/well), TK-Renilla (internal control, 10 ng/well), and pcDNA3 (200 ng/well). After 24 h, cells were exposed to Wnt3a (50 ng/ml) and TcdB1114-1835 (1:8, 1:40, and 1:200 to Wnt3a) in culture medium for 6 h. Cell lysates were harvested and subjected to either firefly/renilla dual luciferase assay or immunoblot analysis for detecting phosphorylated Dvl2 and LRP6. Wnt signaling activates expression of TOPFLASH luciferase reporter (firefly luciferase). Co-transfected renilla luciferase serves as an internal control.Microtiter plate-based binding assayBinding assays were performed on 96-well plates (EIA/RIA plate, Corning Costar) as described previously50. Briefly, microtiter plates were coated with 10 μg/ml NG2 proteins in coating buffer (0.1 M NaHCO3, pH 8.3) at 4 °C overnight, and then blocked with 1% bovine serum albumin in PBS for 1 h. Plates were then incubated with the indicated proteins for 1 h in PBS. Wells were washed three times with PBS plus 0.05% tween-20 at room temperature. One-step Turbo TMB (ThermoFisher Scientific) was used as the substrate, and absorbance at 450 nm was measured with a microplate reader.Organoid Culture, KD, and TcdB challenge assayCrypt isolation from WT or FZD7−/− mouse colon was carried out as previously described, and organoids were expanded as spheroid cultures using conditioned medium52. Except for WT organoids used for Wnt signaling inhibition assay, CHIR99021 (3 μM) was also added to the medium39. Five days after passaging, organoids were re-suspended with Cell Recovery Solution (ThermoFisher Scientific) and mechanically fragmented. Fragments were transduced with adenoviruses expressing shRNA for FZD1, FZD2, or a control shRNA sequence using medium supplemented with Nicotinamide (10 mM, Sigma), Polybrene (8 μg/ml, Sigma), and Y-27632 (10 μM, Sigma), washed, and plated in growth factor-reduced Matrigel (Corning)53. Three days following viral transduction, organoids were challenged with TcdB by adding the toxin into the medium. Viability of organoids was quantified after 72 h.Wnt signaling inhibition in WT colon organoidsTcdB1114-1835 was added into the culture medium of WT colon organoids. For rescue experiments, 5 μM CHIR99021 was also added to the medium. The medium was changed every 48 h with the constant presence of TcdB1114-1835 and/or CHIR99021. Viability of cells was analyzed after six days.Generating FZD1 and FZD2 KD adenovirusAll shRNAs were purchased from Sigma (MISSION shRNA library). The KD efficiency was validated as described in Extended Data Fig. 8c, d. ShRNA sequences showing the highest efficiency were selected to generate adenoviruses. Adenoviruses expressing a control shRNA (5’-CTGGACTTCCAGAAGAACA-3’), shRNAs against mouse FZD1 (shRNA#2: 5’-TGGTGTGCAACGACAAGTTTG-3’), or FZD2 (shRNA#5: 5’-CGCTTCTCAGAGGACGGTTAT-3’) were constructed using the Block-it U6 adenoviral RNAi system (Life Technologies), followed by viral packaging and multiple rounds of amplification in 293A cells (Life Technologies).Viability assay for colonic organoidsThe viability of colonic organoids was assessed using the MTT assay as previously described54. Briefly, the MTT solution was added to the organoid culture (500 μg/ml). After incubation at 37 °C for 2 h, the medium was discarded. For each well (containing 20 μl of Matrigel, in a 48-well plate), 60 μl of 2% SDS solution was added to solubilize the Matrigel (1 h, 37 °C), followed by the addition of 300 μl of DMSO to solubilize reduced MTT (2 h, 37 °C). The absorbance at 562 nm was measured on a microplate reader. Twenty μl of Matrigel without organoids was used as blank control. Normal organoids without exposure to toxins were considered as 100% viable.Immunohistochemistry (IHC), immunofluorescence (IF), and histology analysisColons from adult mice (C57BL/6 strain (purchased from The Jackson Laboratory, #000664), 10–12 weeks old, both male and female mice were used and randomly distributed into experimental groups) were dissected out and subjected to cryosectioning into sections 8–10 μm thick. Colonic sections were fixed in cold acetone for 5 min and then washed three times with PBS. The colonic sections were then blocked with 5% goat serum in PBS for 30 min at room temperature and incubated with primary antibodies overnight (anti-TcdB: 1:600; anti-FZDs: 1:250; rabbit anti-NG2: 1:250), followed with biotinylated goat anti-chicken or rabbit IgG secondary antibodies (1:200, Vector Lab) for 1 h at room temperature. The sections were then incubated with HRP-conjugated streptavidin (1:500, DAKO) for 30 min. Immuno-reactivity was visualized as red color with 3-amino-9-thyl carbazole (DAKO). Cell nuclei were labeled blue with Gill’s Hematoxylin (1:3.5, Sigma). Frozen human colon tissue slides were purchased from BioChain Institute Inc. and subjected to IHC analysis. IF analysis of Claudin-3 and ZO-1 was carried out using mouse colon tissues fixed in 10% formalin and embedded in paraffin (anti-Claudin-3: 1:100; anti-ZO-1: 1:100). Confocal images were captured with the Ultraview Vox Spinning Disk Confocal System. Histology analysis was carried out with hematoxylin and eosin (H&E) staining of paraffin-embedded sections. Stained sections were coded and scored by observers blinded to experimental groups, based on disruption of the colonic epithelium, inflammatory cell filtration, and edema, on a scale of 0 to 3 (mild to severe).Competition assays in colonic tissues with recombinant proteinsAll procedures were conducted in accordance with the guidelines approved by the Boston Children’s Hospital IACUC (#3028). TcdB (40 nM) was pre-incubated with either human IgG1-Fc or FZD2-Fc (2.4 μM) for 30 min on ice. To generate the ex vivo colon segments, mice (C57BL/6, 6–8 weeks, both male and female mice were used, repeated three times, each time four mice per group, the experiments were not randomized or blinded) were euthanized and the colon exposed via laparotomy. A segment in the ascending colon (~ 2 cm long) was sealed by tying both ends with silk ligatures. The toxin samples (40 μl) were injected through a LV catheter into the sealed colon segment. The injection site was then sealed with a hemostat. The colon was covered with PBS-soaked gauze for 2 h, then excised and its lumen flushed with PBS three times, and subjected to IHC analysis.Colon loop ligation assayAll procedures were conducted in accordance with the guidelines approved by the Boston Children’s Hospital IACUC (#3028). WT or FZD7−/− mice (The Jackson Laboratory, #012825, strain B6;129-Fzd7tm1.1Nat/J, 6–8 weeks old, sample size indicated in Fig. 5f, g, both male and female mice were used, the experiments were not randomized or blinded) were anesthetized following overnight fasting. A midline laparotomy was performed to locate the ascending colon and seal a ~2 cm loop with silk ligatures. Two μg of TcdB1-1830 in 80 μl of normal saline or 80 μl of normal saline were injected through a LV catheter into the sealed colon segment, followed by closing the wounds with stitches. Mice were allowed to recover. After 8 h, mice were euthanized and the ligated colon segments were excised, weighed, and measured. The colon segments were fixed, paraffin-embedded, sectioned, and subjected to either H&E staining for histological score analysis or immunofluorescent staining for Claudin-3 and ZO-1.

Cell lines, antibodies, and constructsHeLa (H1, #CRL-1958), CHO (K1, #CCL-61), HT-29 (#HTB-38), Caco-2 (#HTB-37), and 293T (#CRL-3216) cells were originally obtained from ATCC. They tested negative for mycoplasma contamination, but have not been authenticated. The following mouse monoclonal antibodies were purchased from the indicated vendors: Rac1 (23A8, Abcam), non-glucosylated Rac1 (Clone 102, BD Biosciences), 1D4 tag (MA1-722, ThermoFisher Scientific), HA tag (16B12, Covance), β-actin (AC-15, Sigma), ZO-1 (339100, Life technology). Rabbit monoclonal IgG against human CSPG4 (ab139406) and rabbit polyclonal antibodies against FZD1 (ab150553), FZD2 (ab150477), FZD7 (ab51049), PVRL3 (ab63931), and Claudin-3 (ab15102) were all purchased from Abcam. Rabbit monoclonal antibodies against Dvl2 (30D2) and LRP6 (C5C7), and a rabbit polyclonal antibody against phosphorylated LRP6 (Ser1490) were all purchased from Cell Signaling. Chicken polyclonal IgY (#754A) against TcdB was purchased from List Biological Labs. Antibody validation is available on the manufacturers’ websites. A rabbit polyclonal antibody against rodent NG2 and a construct expressing full-length rat NG2 (in pcDNA vector) were both generated in W. Stallcup’s lab. 1D4-tagged full-length FZD1–10 constructs (in pRK5 vector) were originally generated in J. Nathans’ lab (Baltimore, MD) and were obtained from Addgene. FZD7 and FZD8 CRD-myc-GPI constructs were generously provided by J. Nathans and have been described previously47. Constructs expressing full-length human IL1RAPL2 and full-length PVRL3 were purchased from Vigene Biosciences. A construct expressing full-length mouse Syt II was described previously 48.

Cell lines, antibodies, and constructs

HeLa (H1, #CRL-1958), CHO (K1, #CCL-61), HT-29 (#HTB-38), Caco-2 (#HTB-37), and 293T (#CRL-3216) cells were originally obtained from ATCC. They tested negative for mycoplasma contamination, but have not been authenticated. The following mouse monoclonal antibodies were purchased from the indicated vendors: Rac1 (23A8, Abcam), non-glucosylated Rac1 (Clone 102, BD Biosciences), 1D4 tag (MA1-722, ThermoFisher Scientific), HA tag (16B12, Covance), β-actin (AC-15, Sigma), ZO-1 (339100, Life technology). Rabbit monoclonal IgG against human CSPG4 (ab139406) and rabbit polyclonal antibodies against FZD1 (ab150553), FZD2 (ab150477), FZD7 (ab51049), PVRL3 (ab63931), and Claudin-3 (ab15102) were all purchased from Abcam. Rabbit monoclonal antibodies against Dvl2 (30D2) and LRP6 (C5C7), and a rabbit polyclonal antibody against phosphorylated LRP6 (Ser1490) were all purchased from Cell Signaling. Chicken polyclonal IgY (#754A) against TcdB was purchased from List Biological Labs. Antibody validation is available on the manufacturers’ websites. A rabbit polyclonal antibody against rodent NG2 and a construct expressing full-length rat NG2 (in pcDNA vector) were both generated in W. Stallcup’s lab. 1D4-tagged full-length FZD1–10 constructs (in pRK5 vector) were originally generated in J. Nathans’ lab (Baltimore, MD) and were obtained from Addgene. FZD7 and FZD8 CRD-myc-GPI constructs were generously provided by J. Nathans and have been described previously47. Constructs expressing full-length human IL1RAPL2 and full-length PVRL3 were purchased from Vigene Biosciences. A construct expressing full-length mouse Syt II was described previously 48.

TcdB and other recombinant proteinsRecombinant TcdB (from C. difficile strain VPI 10463) and TcdA were expressed in Bacillus megaterium as previously described49 and purified as His6-tagged proteins. TcdB1-1830 was cloned into pHis1522 vector (MoBiTec) and expressed in B. megaterium. TcdB1831-2366, TcdB1501-2366, and TcdB1114-1835 were cloned into pGEX-6P-1 or pET28a vectors and purified as GST-tagged or His6-tagged proteins in E. coli. NG2-EC (P1 and P2) was expressed in HEK293 cells, purified from medium with DEAE-Sepharose columns, and eluted with a gradient buffer (NaCl from 0.2 to 0.8 M, 50 mM Tris-Cl, pH 8.6) as previously described50. Recombinant human proteins were purchased from ACRO Biosystems (IgG1 Fc and FZD2-CRD-Fc), R&D Systems (FZD1-CRD-Fc, FZD5-CRD-Fc, and FZD7-CRD-Fc), and Sino Biologics (PVRL3-EC).

TcdB and other recombinant proteins

Recombinant TcdB (from C. difficile strain VPI 10463) and TcdA were expressed in Bacillus megaterium as previously described49 and purified as His6-tagged proteins. TcdB1-1830 was cloned into pHis1522 vector (MoBiTec) and expressed in B. megaterium. TcdB1831-2366, TcdB1501-2366, and TcdB1114-1835 were cloned into pGEX-6P-1 or pET28a vectors and purified as GST-tagged or His6-tagged proteins in E. coli. NG2-EC (P1 and P2) was expressed in HEK293 cells, purified from medium with DEAE-Sepharose columns, and eluted with a gradient buffer (NaCl from 0.2 to 0.8 M, 50 mM Tris-Cl, pH 8.6) as previously described50. Recombinant human proteins were purchased from ACRO Biosystems (IgG1 Fc and FZD2-CRD-Fc), R&D Systems (FZD1-CRD-Fc, FZD5-CRD-Fc, and FZD7-CRD-Fc), and Sino Biologics (PVRL3-EC).

Generating stable HeLa-Cas9 cells and lentivirus sgRNA librariesThe human codon-optimized sequence of S. pyogenes Cas9 was subcloned from plasmid lentiCas9-Blast (Addgene #52962) into the pQCXIH retroviral vector (Clontech), which was used to generate retroviruses to transduce HeLa cells. Mixed stable cells were selected in the presence of hygromycin B (200 μg/ml, Life Technologies). Lentivirus sgRNA libraries were generated following published protocols using the human GeCKO v2 sgRNA library (Addgene #1000000049)19. The GeCKO v2 library is composed of two half-libraries (library A and library B). Each half-library contains three unique sgRNA per gene and was independently screened with toxins. Cells were transduced with lentivirus-packaged sgRNA library at a MOI of 0.2.

Generating stable HeLa-Cas9 cells and lentivirus sgRNA libraries

The human codon-optimized sequence of S. pyogenes Cas9 was subcloned from plasmid lentiCas9-Blast (Addgene #52962) into the pQCXIH retroviral vector (Clontech), which was used to generate retroviruses to transduce HeLa cells. Mixed stable cells were selected in the presence of hygromycin B (200 μg/ml, Life Technologies). Lentivirus sgRNA libraries were generated following published protocols using the human GeCKO v2 sgRNA library (Addgene #1000000049)19. The GeCKO v2 library is composed of two half-libraries (library A and library B). Each half-library contains three unique sgRNA per gene and was independently screened with toxins. Cells were transduced with lentivirus-packaged sgRNA library at a MOI of 0.2.

Screening CRISPR libraries with TcdB and TcdB1-1830For each CRISPR half-library of cells, 4×107 cells were plated onto two 15-cm culture dishes to ensure sufficient coverage of sgRNAs, with each sgRNA on average being represented about 650 times (i.e., there are on average 650 cells transduced with the same sgRNA). This over-representation rate was calculated from titration plates that were set up in parallel with the library. These cells were exposed to either TcdB or TcdB1-1830 for 48 h. Cells were then washed three times with phosphate-buffered saline (PBS) to remove loosely attached round-shaped cells. The remaining cells were re-seeded and cultured with normal medium without toxins until ~70% confluence. Cells were then subjected to the next round of screening with increased concentrations of toxins. Four rounds of screenings were carried out with TcdB (0.05, 0.1, 0.2, and 0.5 pM) and TcdB1-1830 (5, 10, 20, and 50 pM). The remaining cells were harvested and their genomic DNA extracted using the Blood and Cell Culture DNA mini kit (Qiagen). DNA fragments containing the sgRNA sequences were amplified by PCR using primers lentiGP-1_F (AATGGACTATCATATGCTTACCGTAACTTGAAAGTATTTCG) and lentiGP-3_R (ATGAATACTGCCATTTGTCTCAAGATCTAGTTACGC). Next-generation sequencing (Illumina MiSeq) was performed by a commercial vendor (Genewiz).

Screening CRISPR libraries with TcdB and TcdB1-1830

For each CRISPR half-library of cells, 4×107 cells were plated onto two 15-cm culture dishes to ensure sufficient coverage of sgRNAs, with each sgRNA on average being represented about 650 times (i.e., there are on average 650 cells transduced with the same sgRNA). This over-representation rate was calculated from titration plates that were set up in parallel with the library. These cells were exposed to either TcdB or TcdB1-1830 for 48 h. Cells were then washed three times with phosphate-buffered saline (PBS) to remove loosely attached round-shaped cells. The remaining cells were re-seeded and cultured with normal medium without toxins until ~70% confluence. Cells were then subjected to the next round of screening with increased concentrations of toxins. Four rounds of screenings were carried out with TcdB (0.05, 0.1, 0.2, and 0.5 pM) and TcdB1-1830 (5, 10, 20, and 50 pM). The remaining cells were harvested and their genomic DNA extracted using the Blood and Cell Culture DNA mini kit (Qiagen). DNA fragments containing the sgRNA sequences were amplified by PCR using primers lentiGP-1_F (AATGGACTATCATATGCTTACCGTAACTTGAAAGTATTTCG) and lentiGP-3_R (ATGAATACTGCCATTTGTCTCAAGATCTAGTTACGC). Next-generation sequencing (Illumina MiSeq) was performed by a commercial vendor (Genewiz).

Generating HeLa KO cell linesThe following sgRNA sequences were cloned into LentiGuide-Puro vectors (Addgene) to target the indicated genes: CCGGAGACACGGAGCAGTGG (CSPG4), GCGCTGCTGGGACATCGCCT (EMC4), ACCTTATACCACACAACATC (IL1RAPL2), TGCGAGCACTTCCCGCGCCA (FZD2), AGCGCATGACCACTACACTG (SGMS1), ACAGGCAGAAAACGGCTCCT (UGP2), GTGTAATGACAAGTTCGCCG (FZD1), and GAGAACGGTAAAGAGCGTCG (FZD7). HeLa-Cas9 cells were transduced with lentiviruses that express these sgRNAs. Mixed populations of stable cells were selected with puromycin (2.5 μg/ml) and hygromycin B (200 μg/ml). FZD1/2/7−/− cells were created by sequentially transducing FZD1 and FZD7 sgRNA lentiviruses into FZD2−/− cells and further selected in the presence of 100 pM TcdB1-1830. The mutagenesis rate in these mixed stable cells was determined by NGS (Supplementary Table 1).

Generating HeLa KO cell lines

The following sgRNA sequences were cloned into LentiGuide-Puro vectors (Addgene) to target the indicated genes: CCGGAGACACGGAGCAGTGG (CSPG4), GCGCTGCTGGGACATCGCCT (EMC4), ACCTTATACCACACAACATC (IL1RAPL2), TGCGAGCACTTCCCGCGCCA (FZD2), AGCGCATGACCACTACACTG (SGMS1), ACAGGCAGAAAACGGCTCCT (UGP2), GTGTAATGACAAGTTCGCCG (FZD1), and GAGAACGGTAAAGAGCGTCG (FZD7). HeLa-Cas9 cells were transduced with lentiviruses that express these sgRNAs. Mixed populations of stable cells were selected with puromycin (2.5 μg/ml) and hygromycin B (200 μg/ml). FZD1/2/7−/− cells were created by sequentially transducing FZD1 and FZD7 sgRNA lentiviruses into FZD2−/− cells and further selected in the presence of 100 pM TcdB1-1830. The mutagenesis rate in these mixed stable cells was determined by NGS (Supplementary Table 1).

Cytopathic assayThe cytopathic effect (cell-rounding) of TcdA and TcdB was analyzed using standard cell-rounding assay as previously described1. Briefly, cells were exposed to a gradient of TcdB and TcdB1-1830 for 24 h. Phase-contrast images of cells were taken (Olympus IX51, 10–20X objectives). The numbers of round-shaped and normal shaped cells were counted manually. The percentage of round-shaped cells was plotted and fitted using the Origin software.

Cytopathic assay

The cytopathic effect (cell-rounding) of TcdA and TcdB was analyzed using standard cell-rounding assay as previously described1. Briefly, cells were exposed to a gradient of TcdB and TcdB1-1830 for 24 h. Phase-contrast images of cells were taken (Olympus IX51, 10–20X objectives). The numbers of round-shaped and normal shaped cells were counted manually. The percentage of round-shaped cells was plotted and fitted using the Origin software.

Blocking TcdB entry into cells with NG2-EC and FZD2-CRD-FcRecombinant proteins used for cell protection assays were pre-filtered (0.22 μm, Millipore). Toxins were pre-incubated with FZD2-CRD-Fc and/or NG2-EC (P1) for 30 min on ice with a toxin/protein ratio of 1:400 (except when specifically noted in the figure legend). The mixtures were added into cell culture medium and cells were analyzed by the cytopathic assay.

Blocking TcdB entry into cells with NG2-EC and FZD2-CRD-Fc

Recombinant proteins used for cell protection assays were pre-filtered (0.22 μm, Millipore). Toxins were pre-incubated with FZD2-CRD-Fc and/or NG2-EC (P1) for 30 min on ice with a toxin/protein ratio of 1:400 (except when specifically noted in the figure legend). The mixtures were added into cell culture medium and cells were analyzed by the cytopathic assay.

Transfection, TcdB binding to cells, and immunoblot analysisTransient transfection of HeLa cells was carried out using PolyJetTM (SignaGen). Binding of TcdB to cells was analyzed by exposing cells to TcdB or truncated TcdB fragments for 10 min at room temperature. Cells were washed three times with PBS and then either fixed for immunostaining analysis or harvested with RIPA buffer (50 mM Tris, 1% NP40, 150 mM NaCl, 0.5% sodium deoxycholate, 0.1% SDS, plus a protease inhibitor cocktail (Sigma-Aldrich)). Cell lysates were centrifuged and supernatants were subjected to SDS-PAGE and immunoblot analysis using the enhanced chemiluminescence method (Pierce). The full blot images are shown in Supplementary Figure.

Transfection, TcdB binding to cells, and immunoblot analysis

Transient transfection of HeLa cells was carried out using PolyJetTM (SignaGen). Binding of TcdB to cells was analyzed by exposing cells to TcdB or truncated TcdB fragments for 10 min at room temperature. Cells were washed three times with PBS and then either fixed for immunostaining analysis or harvested with RIPA buffer (50 mM Tris, 1% NP40, 150 mM NaCl, 0.5% sodium deoxycholate, 0.1% SDS, plus a protease inhibitor cocktail (Sigma-Aldrich)). Cell lysates were centrifuged and supernatants were subjected to SDS-PAGE and immunoblot analysis using the enhanced chemiluminescence method (Pierce). The full blot images are shown in Supplementary Figure.

Pull-down assaysPull-down assays were carried out using glutathione Sepharose 4B as previously described48. Briefly, 5 μg of glutathione S-transferase (GST) tagged TcdB1831-2366 and TcdB1501-2366 were immobilized on glutathione beads and incubated with FZD2-CRD-Fc (10 nM) for 1 h at 4 °C. Beads were then washed, pelleted, boiled in SDS sample buffer, and subjected to immunoblot analysis.

Pull-down assays

Pull-down assays were carried out using glutathione Sepharose 4B as previously described48. Briefly, 5 μg of glutathione S-transferase (GST) tagged TcdB1831-2366 and TcdB1501-2366 were immobilized on glutathione beads and incubated with FZD2-CRD-Fc (10 nM) for 1 h at 4 °C. Beads were then washed, pelleted, boiled in SDS sample buffer, and subjected to immunoblot analysis.

Biolayer interferometry (BLI) assayThe binding affinities between TcdB and FZD-CRDs were measured by BLI assay using the Blitz system (ForteBio). Briefly, the CRDs-Fc of FZD1, 2, 5, 7 or human IgG1 Fc (20 μg/ml) were immobilized onto capture biosensors (Dip and Read Anti-hIgG-Fc, ForteBio) and balanced with PBS. The biosensors were then exposed to TcdB or TcdB1-1830, followed by washing with PBS. Binding affinities (KD) were calculated using the Blitz system software (ForteBio).

Biolayer interferometry (BLI) assay

The binding affinities between TcdB and FZD-CRDs were measured by BLI assay using the Blitz system (ForteBio). Briefly, the CRDs-Fc of FZD1, 2, 5, 7 or human IgG1 Fc (20 μg/ml) were immobilized onto capture biosensors (Dip and Read Anti-hIgG-Fc, ForteBio) and balanced with PBS. The biosensors were then exposed to TcdB or TcdB1-1830, followed by washing with PBS. Binding affinities (KD) were calculated using the Blitz system software (ForteBio).

Wnt signaling assayThe TOPFLASH/TK-Renilla dual luciferase reporter assay was utilized to detect Wnt signaling activities as previously described51. Briefly, 293T cells in 24-well plates were co-transfected with TOPFLASH (50 ng/well), TK-Renilla (internal control, 10 ng/well), and pcDNA3 (200 ng/well). After 24 h, cells were exposed to Wnt3a (50 ng/ml) and TcdB1114-1835 (1:8, 1:40, and 1:200 to Wnt3a) in culture medium for 6 h. Cell lysates were harvested and subjected to either firefly/renilla dual luciferase assay or immunoblot analysis for detecting phosphorylated Dvl2 and LRP6. Wnt signaling activates expression of TOPFLASH luciferase reporter (firefly luciferase). Co-transfected renilla luciferase serves as an internal control.

Wnt signaling assay

The TOPFLASH/TK-Renilla dual luciferase reporter assay was utilized to detect Wnt signaling activities as previously described51. Briefly, 293T cells in 24-well plates were co-transfected with TOPFLASH (50 ng/well), TK-Renilla (internal control, 10 ng/well), and pcDNA3 (200 ng/well). After 24 h, cells were exposed to Wnt3a (50 ng/ml) and TcdB1114-1835 (1:8, 1:40, and 1:200 to Wnt3a) in culture medium for 6 h. Cell lysates were harvested and subjected to either firefly/renilla dual luciferase assay or immunoblot analysis for detecting phosphorylated Dvl2 and LRP6. Wnt signaling activates expression of TOPFLASH luciferase reporter (firefly luciferase). Co-transfected renilla luciferase serves as an internal control.

Microtiter plate-based binding assayBinding assays were performed on 96-well plates (EIA/RIA plate, Corning Costar) as described previously50. Briefly, microtiter plates were coated with 10 μg/ml NG2 proteins in coating buffer (0.1 M NaHCO3, pH 8.3) at 4 °C overnight, and then blocked with 1% bovine serum albumin in PBS for 1 h. Plates were then incubated with the indicated proteins for 1 h in PBS. Wells were washed three times with PBS plus 0.05% tween-20 at room temperature. One-step Turbo TMB (ThermoFisher Scientific) was used as the substrate, and absorbance at 450 nm was measured with a microplate reader.

Microtiter plate-based binding assay

Binding assays were performed on 96-well plates (EIA/RIA plate, Corning Costar) as described previously50. Briefly, microtiter plates were coated with 10 μg/ml NG2 proteins in coating buffer (0.1 M NaHCO3, pH 8.3) at 4 °C overnight, and then blocked with 1% bovine serum albumin in PBS for 1 h. Plates were then incubated with the indicated proteins for 1 h in PBS. Wells were washed three times with PBS plus 0.05% tween-20 at room temperature. One-step Turbo TMB (ThermoFisher Scientific) was used as the substrate, and absorbance at 450 nm was measured with a microplate reader.

Organoid Culture, KD, and TcdB challenge assayCrypt isolation from WT or FZD7−/− mouse colon was carried out as previously described, and organoids were expanded as spheroid cultures using conditioned medium52. Except for WT organoids used for Wnt signaling inhibition assay, CHIR99021 (3 μM) was also added to the medium39. Five days after passaging, organoids were re-suspended with Cell Recovery Solution (ThermoFisher Scientific) and mechanically fragmented. Fragments were transduced with adenoviruses expressing shRNA for FZD1, FZD2, or a control shRNA sequence using medium supplemented with Nicotinamide (10 mM, Sigma), Polybrene (8 μg/ml, Sigma), and Y-27632 (10 μM, Sigma), washed, and plated in growth factor-reduced Matrigel (Corning)53. Three days following viral transduction, organoids were challenged with TcdB by adding the toxin into the medium. Viability of organoids was quantified after 72 h.

Organoid Culture, KD, and TcdB challenge assay

Crypt isolation from WT or FZD7−/− mouse colon was carried out as previously described, and organoids were expanded as spheroid cultures using conditioned medium52. Except for WT organoids used for Wnt signaling inhibition assay, CHIR99021 (3 μM) was also added to the medium39. Five days after passaging, organoids were re-suspended with Cell Recovery Solution (ThermoFisher Scientific) and mechanically fragmented. Fragments were transduced with adenoviruses expressing shRNA for FZD1, FZD2, or a control shRNA sequence using medium supplemented with Nicotinamide (10 mM, Sigma), Polybrene (8 μg/ml, Sigma), and Y-27632 (10 μM, Sigma), washed, and plated in growth factor-reduced Matrigel (Corning)53. Three days following viral transduction, organoids were challenged with TcdB by adding the toxin into the medium. Viability of organoids was quantified after 72 h.

Wnt signaling inhibition in WT colon organoidsTcdB1114-1835 was added into the culture medium of WT colon organoids. For rescue experiments, 5 μM CHIR99021 was also added to the medium. The medium was changed every 48 h with the constant presence of TcdB1114-1835 and/or CHIR99021. Viability of cells was analyzed after six days.

Wnt signaling inhibition in WT colon organoids

TcdB1114-1835 was added into the culture medium of WT colon organoids. For rescue experiments, 5 μM CHIR99021 was also added to the medium. The medium was changed every 48 h with the constant presence of TcdB1114-1835 and/or CHIR99021. Viability of cells was analyzed after six days.

Generating FZD1 and FZD2 KD adenovirusAll shRNAs were purchased from Sigma (MISSION shRNA library). The KD efficiency was validated as described in Extended Data Fig. 8c, d. ShRNA sequences showing the highest efficiency were selected to generate adenoviruses. Adenoviruses expressing a control shRNA (5’-CTGGACTTCCAGAAGAACA-3’), shRNAs against mouse FZD1 (shRNA#2: 5’-TGGTGTGCAACGACAAGTTTG-3’), or FZD2 (shRNA#5: 5’-CGCTTCTCAGAGGACGGTTAT-3’) were constructed using the Block-it U6 adenoviral RNAi system (Life Technologies), followed by viral packaging and multiple rounds of amplification in 293A cells (Life Technologies).

Generating FZD1 and FZD2 KD adenovirus

All shRNAs were purchased from Sigma (MISSION shRNA library). The KD efficiency was validated as described in Extended Data Fig. 8c, d. ShRNA sequences showing the highest efficiency were selected to generate adenoviruses. Adenoviruses expressing a control shRNA (5’-CTGGACTTCCAGAAGAACA-3’), shRNAs against mouse FZD1 (shRNA#2: 5’-TGGTGTGCAACGACAAGTTTG-3’), or FZD2 (shRNA#5: 5’-CGCTTCTCAGAGGACGGTTAT-3’) were constructed using the Block-it U6 adenoviral RNAi system (Life Technologies), followed by viral packaging and multiple rounds of amplification in 293A cells (Life Technologies).

Viability assay for colonic organoidsThe viability of colonic organoids was assessed using the MTT assay as previously described54. Briefly, the MTT solution was added to the organoid culture (500 μg/ml). After incubation at 37 °C for 2 h, the medium was discarded. For each well (containing 20 μl of Matrigel, in a 48-well plate), 60 μl of 2% SDS solution was added to solubilize the Matrigel (1 h, 37 °C), followed by the addition of 300 μl of DMSO to solubilize reduced MTT (2 h, 37 °C). The absorbance at 562 nm was measured on a microplate reader. Twenty μl of Matrigel without organoids was used as blank control. Normal organoids without exposure to toxins were considered as 100% viable.

Viability assay for colonic organoids

The viability of colonic organoids was assessed using the MTT assay as previously described54. Briefly, the MTT solution was added to the organoid culture (500 μg/ml). After incubation at 37 °C for 2 h, the medium was discarded. For each well (containing 20 μl of Matrigel, in a 48-well plate), 60 μl of 2% SDS solution was added to solubilize the Matrigel (1 h, 37 °C), followed by the addition of 300 μl of DMSO to solubilize reduced MTT (2 h, 37 °C). The absorbance at 562 nm was measured on a microplate reader. Twenty μl of Matrigel without organoids was used as blank control. Normal organoids without exposure to toxins were considered as 100% viable.

Immunohistochemistry (IHC), immunofluorescence (IF), and histology analysisColons from adult mice (C57BL/6 strain (purchased from The Jackson Laboratory, #000664), 10–12 weeks old, both male and female mice were used and randomly distributed into experimental groups) were dissected out and subjected to cryosectioning into sections 8–10 μm thick. Colonic sections were fixed in cold acetone for 5 min and then washed three times with PBS. The colonic sections were then blocked with 5% goat serum in PBS for 30 min at room temperature and incubated with primary antibodies overnight (anti-TcdB: 1:600; anti-FZDs: 1:250; rabbit anti-NG2: 1:250), followed with biotinylated goat anti-chicken or rabbit IgG secondary antibodies (1:200, Vector Lab) for 1 h at room temperature. The sections were then incubated with HRP-conjugated streptavidin (1:500, DAKO) for 30 min. Immuno-reactivity was visualized as red color with 3-amino-9-thyl carbazole (DAKO). Cell nuclei were labeled blue with Gill’s Hematoxylin (1:3.5, Sigma). Frozen human colon tissue slides were purchased from BioChain Institute Inc. and subjected to IHC analysis. IF analysis of Claudin-3 and ZO-1 was carried out using mouse colon tissues fixed in 10% formalin and embedded in paraffin (anti-Claudin-3: 1:100; anti-ZO-1: 1:100). Confocal images were captured with the Ultraview Vox Spinning Disk Confocal System. Histology analysis was carried out with hematoxylin and eosin (H&E) staining of paraffin-embedded sections. Stained sections were coded and scored by observers blinded to experimental groups, based on disruption of the colonic epithelium, inflammatory cell filtration, and edema, on a scale of 0 to 3 (mild to severe).

Immunohistochemistry (IHC), immunofluorescence (IF), and histology analysis

Colons from adult mice (C57BL/6 strain (purchased from The Jackson Laboratory, #000664), 10–12 weeks old, both male and female mice were used and randomly distributed into experimental groups) were dissected out and subjected to cryosectioning into sections 8–10 μm thick. Colonic sections were fixed in cold acetone for 5 min and then washed three times with PBS. The colonic sections were then blocked with 5% goat serum in PBS for 30 min at room temperature and incubated with primary antibodies overnight (anti-TcdB: 1:600; anti-FZDs: 1:250; rabbit anti-NG2: 1:250), followed with biotinylated goat anti-chicken or rabbit IgG secondary antibodies (1:200, Vector Lab) for 1 h at room temperature. The sections were then incubated with HRP-conjugated streptavidin (1:500, DAKO) for 30 min. Immuno-reactivity was visualized as red color with 3-amino-9-thyl carbazole (DAKO). Cell nuclei were labeled blue with Gill’s Hematoxylin (1:3.5, Sigma). Frozen human colon tissue slides were purchased from BioChain Institute Inc. and subjected to IHC analysis. IF analysis of Claudin-3 and ZO-1 was carried out using mouse colon tissues fixed in 10% formalin and embedded in paraffin (anti-Claudin-3: 1:100; anti-ZO-1: 1:100). Confocal images were captured with the Ultraview Vox Spinning Disk Confocal System. Histology analysis was carried out with hematoxylin and eosin (H&E) staining of paraffin-embedded sections. Stained sections were coded and scored by observers blinded to experimental groups, based on disruption of the colonic epithelium, inflammatory cell filtration, and edema, on a scale of 0 to 3 (mild to severe).

Competition assays in colonic tissues with recombinant proteinsAll procedures were conducted in accordance with the guidelines approved by the Boston Children’s Hospital IACUC (#3028). TcdB (40 nM) was pre-incubated with either human IgG1-Fc or FZD2-Fc (2.4 μM) for 30 min on ice. To generate the ex vivo colon segments, mice (C57BL/6, 6–8 weeks, both male and female mice were used, repeated three times, each time four mice per group, the experiments were not randomized or blinded) were euthanized and the colon exposed via laparotomy. A segment in the ascending colon (~ 2 cm long) was sealed by tying both ends with silk ligatures. The toxin samples (40 μl) were injected through a LV catheter into the sealed colon segment. The injection site was then sealed with a hemostat. The colon was covered with PBS-soaked gauze for 2 h, then excised and its lumen flushed with PBS three times, and subjected to IHC analysis.

Competition assays in colonic tissues with recombinant proteins

All procedures were conducted in accordance with the guidelines approved by the Boston Children’s Hospital IACUC (#3028). TcdB (40 nM) was pre-incubated with either human IgG1-Fc or FZD2-Fc (2.4 μM) for 30 min on ice. To generate the ex vivo colon segments, mice (C57BL/6, 6–8 weeks, both male and female mice were used, repeated three times, each time four mice per group, the experiments were not randomized or blinded) were euthanized and the colon exposed via laparotomy. A segment in the ascending colon (~ 2 cm long) was sealed by tying both ends with silk ligatures. The toxin samples (40 μl) were injected through a LV catheter into the sealed colon segment. The injection site was then sealed with a hemostat. The colon was covered with PBS-soaked gauze for 2 h, then excised and its lumen flushed with PBS three times, and subjected to IHC analysis.

Colon loop ligation assayAll procedures were conducted in accordance with the guidelines approved by the Boston Children’s Hospital IACUC (#3028). WT or FZD7−/− mice (The Jackson Laboratory, #012825, strain B6;129-Fzd7tm1.1Nat/J, 6–8 weeks old, sample size indicated in Fig. 5f, g, both male and female mice were used, the experiments were not randomized or blinded) were anesthetized following overnight fasting. A midline laparotomy was performed to locate the ascending colon and seal a ~2 cm loop with silk ligatures. Two μg of TcdB1-1830 in 80 μl of normal saline or 80 μl of normal saline were injected through a LV catheter into the sealed colon segment, followed by closing the wounds with stitches. Mice were allowed to recover. After 8 h, mice were euthanized and the ligated colon segments were excised, weighed, and measured. The colon segments were fixed, paraffin-embedded, sectioned, and subjected to either H&E staining for histological score analysis or immunofluorescent staining for Claudin-3 and ZO-1.

Colon loop ligation assay

All procedures were conducted in accordance with the guidelines approved by the Boston Children’s Hospital IACUC (#3028). WT or FZD7−/− mice (The Jackson Laboratory, #012825, strain B6;129-Fzd7tm1.1Nat/J, 6–8 weeks old, sample size indicated in Fig. 5f, g, both male and female mice were used, the experiments were not randomized or blinded) were anesthetized following overnight fasting. A midline laparotomy was performed to locate the ascending colon and seal a ~2 cm loop with silk ligatures. Two μg of TcdB1-1830 in 80 μl of normal saline or 80 μl of normal saline were injected through a LV catheter into the sealed colon segment, followed by closing the wounds with stitches. Mice were allowed to recover. After 8 h, mice were euthanized and the ligated colon segments were excised, weighed, and measured. The colon segments were fixed, paraffin-embedded, sectioned, and subjected to either H&E staining for histological score analysis or immunofluorescent staining for Claudin-3 and ZO-1.

Extended DataExtended Data Figure 1Recombinant TcdB and TcdB1-1830a, Schematic drawings of TcdB and a truncated TcdB lacking the CROPs region (TcdB1-1830). GTD: glucosyltransferase domain; CPD: cysteine protease domain; TD: translocation domain; RBD: receptor-binding domain, including a putative receptor-binding region and the CROPs region.b, Coomassie blue staining (left panel) and immunoblot (right panel, chicken poly-clonal TcdB antibody) showing TcdB and TcdB1-1830 recombinantly expressed in Bacillus megaterium. We note that TcdB1-1830 contains a contaminating protein visible on Coomassie blue-stained gel. Mass spectrometry analysis confirmed that this band is not a fragment of TcdB. The top matching protein is the bacterial chaperone protein ClpB.c, Cytopathic toxicity of recombinant TcdB and TcdB1-1830 on HeLa cells was neutralized by anti-TcdB polyclonal antibody (pAb), confirming that the toxicity is from TcdB and TcdB1-1830 (error bars indicate mean ± s.d., two independent experiments).d, HeLa, CHO, HT-29, and Caco-2 cells were exposed to TcdB or TcdB1-1830 as indicated for 24 h. TcdB1-1830 induced cell rounding at picomolar concentrations. Scale bars = 25 μm (HT-29) or 50 μm (HeLa, CHO, and Caco-2). Representative images are from one of three independent experiments.Extended Data Figure 2Top-ranking sgRNAsa, Sequences of sgRNA were amplified by PCR after screening and subjected to NGS. The GeCKO v2 sgRNA library is composed of two half-libraries (library A and library B). Each half library contains 3 unique sgRNA per gene. These two half-libraries were prepared and subjected to screens independently.b–e, Lists of top-ranking sgRNAs. See Source Data for lists of all identified sgRNAs.Extended Data Figure 3Assessing the sensitivity of HeLa KO cells to TcdB and TcdB1-1830a–b, HeLa-Cas9 cells with the indicated genes mutated via CRISPR/Cas9, as well as WT Hela-Cas9 cells, were exposed to TcdB (a) or TcdB1-1830 (b) for 24 h. The percentages of rounded cells were quantified and plotted (error bars indicate mean ± s.d., three independent experiments).c, HeLa KO cells were exposed to TcdB or TcdB1-1830 for 3 h. Cell lysates were subjected to immunoblot analysis for Rac1 and non-glucosylated Rac1. UGP2−/− cells retained high levels of non-glucosylated Rac1 after exposure to TcdB or TcdB1-1830. CSPG4−/− cells retained high levels of non-glucosylated Rac1 after exposure to TcdB. FZD2−/− and EMC4−/− cells showed slightly higher levels of non-glucosylated Rac1 compared to WT cells after exposure to TcdB1-1830. Representative blots are one from two independent experiments.Extended Data Figure 4The CROPs is essential for TcdB binding to CSPG4, but not required for TcdB binding to FZDsa, Schematic drawings of NG2 (rat CSPG4). Two fractions of recombinant extracellular domain (EC) fragments were used: one that does not contain chondroitin sulfate (CS) chains (EC P1), and the other that contains CS (EC P2). TMD-cyto: transmembrane and cytoplasmic domain.b, TcdB, but not TcdB1-1830, binds directly to both EC P1 and EC P2 of NG2 in a microtiter plate-based binding assay (error bars indicate mean ± s.d., two independent experiments).c,
CSPG4−/− cells transfected with the indicated constructs were exposed to TcdB (10 nM), TcdB1-1830 (10 nM), or the receptor-binding domain of botulinum neurotoxin B (BoNT/B, 100 nM) for 10 min. Cell lysates were collected and subjected to immunoblot analysis. IL1RAPL2 and synaptotagmin II (Syt II, a receptor for BoNT/B) served as controls. Transfection of NG2 increased binding of TcdB, but not TcdB1-1830, whereas transfection of FZD2 increased binding of both TcdB and TcdB1-1830. One of three independent experiments is shown.d, The CROPs domain binds to CSPG4 on cell surfaces in a dose-dependent manner. High concentrations of recombinant CROPs reduced CSPG4-dependent binding of TcdB to cell surfaces, indicating that the CROPs can compete with TcdB for binding to CSPG4 on cell surfaces. One of three independent experiments is shown.e, The CROPs domain reduced cytopathic toxicity of TcdB (5 pM) on WT HeLa cells (error bars indicate mean ± s.d., two independent experiments).f,
CSPG4−/− cells were transfected with FZD2 and then exposed to TcdB or indicated TcdB fragments. FZD2 mediated binding of TcdB, TcdB1-1830, and TcdB1501-2366, but not the CROPs (TcdB1831-2366). One of three independent experiments is shown.Extended Data Figure 5Characterizing TcdB binding to FZDsa,
CSPG4−/− cells were transfected with 1D4-tagged FZD1, 2, 5, 7, and 9. Cells were exposed to TcdB (10 nM, 10 min), washed, fixed, permeabilized, and subjected to immunostaining analysis. Scale bar = 20 μm. One of three independent experiments is shown.b, The CRD domains of human FZD1 (residues 102–235), FZD2 (residues 25–158), and FZD7 (residues 35–168) were aligned using the Vector NTI software.c, FZD7-CRD, but not FZD8-CRD, when expressed on the surface of CSPG4−/− cells via a GPI anchor, mediated binding of TcdB (10 nM, 10 min) to cells. One of three independent experiments is shown.d, WT HeLa cells, FZD1/2/7−/− cells, and CSPG4−/− cells were exposed to TcdA and subjected to cytopathic cell-rounding assay. No reduction in sensitivity to TcdA was observed for FZD1/2/7−/− cells or CSPG4−/− cells, suggesting that TcdA does not use FZD1/2/7 or CSPG4 as its receptors (error bars indicate mean ± s.d., two independent experiments).e–f, Representative binding/dissociation curves for TcdB binding to Fc-tagged CRDs of FZD1, 2, 5, and 7 (e), and for TcdB1-1830 binding to FZD2-CRD-Fc (f). Binding parameters are listed in Supplementary Table 3. Representative curves are from one of three independent experiments.g, WT and EMC4−/− cells were transfected with 1D4-tagged FZD1, 2, or 7. Cell lysates were subjected to immunoblot analysis. Expression of FZD1, 2, and 7 are reduced in EMC4−/− cells compared to WT cells (n = 6, *p < 0.005, one-way ANOVA). Representative blots are from one of three independent experiments.h, Expression levels of CSPG4 in EMC4−/− cells is similar to those in WT cells, suggesting that EMC is not required for single-pass transmembrane proteins. One of three independent experiments is shown.Extended Data Figure 6TcdB can bind to both FZD and CSPG4 simultaneouslya, NG2-EC was immobilized on microtiter plates, followed by binding of TcdB, washing away unbound TcdB, and addition of FZD-CRD. FZD2-CRD binds robustly to TcdB that is pre-bound by NG2-EC on the microtiter plate. FZD2-CRD did not bind to NG2-EC without TcdB, and FZD5-CRD showed no detectable binding to NG2-TcdB in this assay (error bars indicate mean ± s.d., two independent experiments).b, Experiments are described in Fig. 3d on HeLa (5 pM TcdB), HT-29 (50 pM TcdB), and Caco-2 cells (150 pM TcdB). Scale bars = 50 μm (HeLa and Caco-2) or 25 μm (HT-29). Representative images are from one of four independent experiments.Extended Data Figure 7PVRL3 failed to mediate binding and entry of TcdB in HeLa and Caco-2 cellsa,
CSPG4−/− HeLa cells transfected with the indicated constructs were exposed to TcdB in medium for 10 min. Cell lysates were collected and subjected to immunoblot analysis. Expression of PVRL3 was confirmed using an anti-PVRL3 antibody. Transfection of FZD2, but not PVRL-3, increased binding of TcdB to CSPG4−/− cells. One of three independent experiments is shown.b, Cells were challenged with TcdB (300 pM). Ectopic expression of PVRL3 failed to restore the sensitivity of CSPG4−/− HeLa cells towards TcdB, while expression of FZD2 restored entry of TcdB and resulted in rounding of transfected cells. Co-transfected GFP marked transfected cells. Scale bar = 50 μm. One of three independent experiments is shown.c, Recombinant extracellular domain of PVRL3 (PVRL3-EC) did not reduce TcdB entry into Caco-2 cells, analyzed by cytopathic cell-rounding assay. In contrast, FZD2-CRD prevented entry of TcdB into Caco-2 cells. Scale bar = 50 μm. One of three independent experiments is shown.Extended Data Figure 8Colonic organoids showed similar levels of sensitivity to TcdB and TcdB1-1830 and validation of FZD1 and FZD2 KD efficiencya, Colonic organoids were cultured from WT mice. They were exposed to a gradient of TcdB or TcdB1-1830. Viability of organoids was quantified using the MTT assay. TcdB and TcdB1-1830 showed similar IC50, suggesting that WT organoids are equally susceptible to TcdB and TcdB1-1830 (n = 8, error bars indicate mean ± s.d., two independent experiments). n.s.: not significant.b, Immunoblot analysis of CSPG4 expression in mouse brain, colonic organoids, mouse whole colon tissue, and isolated mouse colonic epithelium (200 μg cell/tissue lysates). The colonic epithelium was isolated from colon tissues by EDTA treatment (10 mM, 2 h at 4 °C). One of three independent experiments is shown.c–d, ShRNA sequences targeting FZD1 and FZD2 were validated by measuring KD efficiency of transfected 1D4-tagged FZD1 and FZD2 in 293T cells. ShRNAs marked with asterisks (shRNA2 for FZD1 and shRNA5 for FZD2) were used to generate adenoviruses. Actin served as the loading control. One of two independent experiments is shown.Extended Data Figure 9TcdB1114-1835 inhibits Wnt signaling and induces death of colonic organoidsa, TcdB1114-1835 blocked Wnt3a-mediated signaling in 293T cells in a dose-dependent manner. Increasing concentrations of Wnt restored Wnt reporter activity blocked by TcdB1114-1835. Wnt signaling activity was analyzed using the TOPFLASH/TK-Renilla dual luciferase reporter assay (error bars indicate mean ± s.d., two independent experiments). We note that 1.25 nM Wnt3a equals 50 ng/ml concentration used in Fig. 4c.b, 293T cells in 24-well plates were exposed to Wnt3a (50 ng/ml) and TcdB1114-1835 in culture medium for 6 h. Cell lysates were harvested and subjected to immunoblot analysis for detecting phosphorylated Dvl2 and LRP6. Wnt signaling activation results in phosphorylation of Dvl2 and LRP6. Phosphorylated Dvl2 is marked with an asterisk. One of three independent experiments is shown.c, Mouse colonic organoids were exposed to TcdB or TcdB1114-1835 for 12 h. Cell lysates were subjected to immunoblot analysis. No glucosylation of Rac1 was observed in organoids treated with TcdB1114-1835. One of two independent experiments is shown.d, Colonic organoids were exposed to TcdB1114-1835 for 72 h, with or without CHIR99021 (5 μM). Normal organoids (green arrow), growth inhibited organoids (red arrow), and disrupted/dead organoids (*) are marked. Scale bar represents 200 μm. One of three independent experiments is shown.e, Time-course images of colonic organoids exposed to CHIR99021 (5 μM), TcdB1114-1835 (25 nM), or a combination of TcdB1114-1835 plus CHIR99021, at 0, 2, 4, and 6 days. Normal organoids (green arrow), growth inhibited organoids (red arrow), and disrupted/dead organoids (*) are marked. Scale bar represents 500 μm. One of four independent experiments is shown.Extended Data Figure 10FZDs are receptors for TcdB in the colonic epitheliuma–c, Human colon cryosections were obtained from a commercial vendor and subjected to IHC analysis for detecting FZD7 (a), FZD2 (b), and CSPG4 (c). Ep: epithelial cells; Mf: sub-epithelial myofibroblasts. Scale bar = 50 μm. Representative images are from one of three independent experiments.d, Expression of FZD1 is not detectable in mouse or human colonic tissues. One of three independent experiments is shown.e, FZD7 antibody labeled WT colonic sections, but showed no signals on colonic tissues from FZD7−/− mice in IHC analysis, confirming the specificity of this antibody. One of three independent experiments is shown.f, Immunostaining of FZD2 (green) is reduced in FZD2-KD colonic organoids compared to control organoids, confirming the specificity of FZD2 antibody. Cell nuclei were labeled by DAPI (blue). Scale bar = 30 μm. One of three independent experiments is shown.g, Experiments are described in Fig. 5g. Representative images from one of three independent experiments are shown. Scale bar = 100 μm.h, Experiments were carried out as described in Fig. 5h. Low-magnification images of immunofluorescent staining of the cell-cell junction markers Claudin-3 (green) and ZO-1 (red) were stitched together to show an overview of the colon tissue. The middle panel (WT/TcdB) showed disruption of the normal staining pattern for Claudin-3 and ZO-1, indicating a loss of epithelial integrity, compared with both control and FZD7−/−/TcdB1-1830. Scale bar represents 200 μm. Representative images are from one of three independent experiments.i, A schematic overview of cellular factors identified in the CRISPR/Cas9 screen. Validated and plausible cellular factors identified in our unbiased genome-wide screens were grouped based on their presence in the same protein complexes and/or signaling pathways. The color of the gene names reflects the number of unique sgRNA identified. The arrows link these genes to either confirmed or plausible roles in four major steps of TcdB action: (1) receptor-mediated endocytosis; (2) low pH in the endosomes triggers conformational changes of the TD, which translocates the GTD across endosomal membranes; (3) GTD is later released via auto-proteolysis by the CPD, which is activated by the cytosolic co-factor inositol hexakisphosphate (InsP6); (4) released GTD glucosylates small GTPases such as Rho, Rac, and CDC42.

Extended Data

Recombinant TcdB and TcdB1-1830

a, Schematic drawings of TcdB and a truncated TcdB lacking the CROPs region (TcdB1-1830). GTD: glucosyltransferase domain; CPD: cysteine protease domain; TD: translocation domain; RBD: receptor-binding domain, including a putative receptor-binding region and the CROPs region.

b, Coomassie blue staining (left panel) and immunoblot (right panel, chicken poly-clonal TcdB antibody) showing TcdB and TcdB1-1830 recombinantly expressed in Bacillus megaterium. We note that TcdB1-1830 contains a contaminating protein visible on Coomassie blue-stained gel. Mass spectrometry analysis confirmed that this band is not a fragment of TcdB. The top matching protein is the bacterial chaperone protein ClpB.

c, Cytopathic toxicity of recombinant TcdB and TcdB1-1830 on HeLa cells was neutralized by anti-TcdB polyclonal antibody (pAb), confirming that the toxicity is from TcdB and TcdB1-1830 (error bars indicate mean ± s.d., two independent experiments).

d, HeLa, CHO, HT-29, and Caco-2 cells were exposed to TcdB or TcdB1-1830 as indicated for 24 h. TcdB1-1830 induced cell rounding at picomolar concentrations. Scale bars = 25 μm (HT-29) or 50 μm (HeLa, CHO, and Caco-2). Representative images are from one of three independent experiments.

Top-ranking sgRNAs

a, Sequences of sgRNA were amplified by PCR after screening and subjected to NGS. The GeCKO v2 sgRNA library is composed of two half-libraries (library A and library B). Each half library contains 3 unique sgRNA per gene. These two half-libraries were prepared and subjected to screens independently.

b–e, Lists of top-ranking sgRNAs. See Source Data for lists of all identified sgRNAs.

Assessing the sensitivity of HeLa KO cells to TcdB and TcdB1-1830

a–b, HeLa-Cas9 cells with the indicated genes mutated via CRISPR/Cas9, as well as WT Hela-Cas9 cells, were exposed to TcdB (a) or TcdB1-1830 (b) for 24 h. The percentages of rounded cells were quantified and plotted (error bars indicate mean ± s.d., three independent experiments).

c, HeLa KO cells were exposed to TcdB or TcdB1-1830 for 3 h. Cell lysates were subjected to immunoblot analysis for Rac1 and non-glucosylated Rac1. UGP2−/− cells retained high levels of non-glucosylated Rac1 after exposure to TcdB or TcdB1-1830. CSPG4−/− cells retained high levels of non-glucosylated Rac1 after exposure to TcdB. FZD2−/− and EMC4−/− cells showed slightly higher levels of non-glucosylated Rac1 compared to WT cells after exposure to TcdB1-1830. Representative blots are one from two independent experiments.

The CROPs is essential for TcdB binding to CSPG4, but not required for TcdB binding to FZDs

a, Schematic drawings of NG2 (rat CSPG4). Two fractions of recombinant extracellular domain (EC) fragments were used: one that does not contain chondroitin sulfate (CS) chains (EC P1), and the other that contains CS (EC P2). TMD-cyto: transmembrane and cytoplasmic domain.

b, TcdB, but not TcdB1-1830, binds directly to both EC P1 and EC P2 of NG2 in a microtiter plate-based binding assay (error bars indicate mean ± s.d., two independent experiments).

c,
CSPG4−/− cells transfected with the indicated constructs were exposed to TcdB (10 nM), TcdB1-1830 (10 nM), or the receptor-binding domain of botulinum neurotoxin B (BoNT/B, 100 nM) for 10 min. Cell lysates were collected and subjected to immunoblot analysis. IL1RAPL2 and synaptotagmin II (Syt II, a receptor for BoNT/B) served as controls. Transfection of NG2 increased binding of TcdB, but not TcdB1-1830, whereas transfection of FZD2 increased binding of both TcdB and TcdB1-1830. One of three independent experiments is shown.

d, The CROPs domain binds to CSPG4 on cell surfaces in a dose-dependent manner. High concentrations of recombinant CROPs reduced CSPG4-dependent binding of TcdB to cell surfaces, indicating that the CROPs can compete with TcdB for binding to CSPG4 on cell surfaces. One of three independent experiments is shown.

e, The CROPs domain reduced cytopathic toxicity of TcdB (5 pM) on WT HeLa cells (error bars indicate mean ± s.d., two independent experiments).

f,
CSPG4−/− cells were transfected with FZD2 and then exposed to TcdB or indicated TcdB fragments. FZD2 mediated binding of TcdB, TcdB1-1830, and TcdB1501-2366, but not the CROPs (TcdB1831-2366). One of three independent experiments is shown.

Characterizing TcdB binding to FZDs

a,
CSPG4−/− cells were transfected with 1D4-tagged FZD1, 2, 5, 7, and 9. Cells were exposed to TcdB (10 nM, 10 min), washed, fixed, permeabilized, and subjected to immunostaining analysis. Scale bar = 20 μm. One of three independent experiments is shown.

b, The CRD domains of human FZD1 (residues 102–235), FZD2 (residues 25–158), and FZD7 (residues 35–168) were aligned using the Vector NTI software.

c, FZD7-CRD, but not FZD8-CRD, when expressed on the surface of CSPG4−/− cells via a GPI anchor, mediated binding of TcdB (10 nM, 10 min) to cells. One of three independent experiments is shown.

d, WT HeLa cells, FZD1/2/7−/− cells, and CSPG4−/− cells were exposed to TcdA and subjected to cytopathic cell-rounding assay. No reduction in sensitivity to TcdA was observed for FZD1/2/7−/− cells or CSPG4−/− cells, suggesting that TcdA does not use FZD1/2/7 or CSPG4 as its receptors (error bars indicate mean ± s.d., two independent experiments).

e–f, Representative binding/dissociation curves for TcdB binding to Fc-tagged CRDs of FZD1, 2, 5, and 7 (e), and for TcdB1-1830 binding to FZD2-CRD-Fc (f). Binding parameters are listed in Supplementary Table 3. Representative curves are from one of three independent experiments.

g, WT and EMC4−/− cells were transfected with 1D4-tagged FZD1, 2, or 7. Cell lysates were subjected to immunoblot analysis. Expression of FZD1, 2, and 7 are reduced in EMC4−/− cells compared to WT cells (n = 6, *p < 0.005, one-way ANOVA). Representative blots are from one of three independent experiments.

h, Expression levels of CSPG4 in EMC4−/− cells is similar to those in WT cells, suggesting that EMC is not required for single-pass transmembrane proteins. One of three independent experiments is shown.

TcdB can bind to both FZD and CSPG4 simultaneously

a, NG2-EC was immobilized on microtiter plates, followed by binding of TcdB, washing away unbound TcdB, and addition of FZD-CRD. FZD2-CRD binds robustly to TcdB that is pre-bound by NG2-EC on the microtiter plate. FZD2-CRD did not bind to NG2-EC without TcdB, and FZD5-CRD showed no detectable binding to NG2-TcdB in this assay (error bars indicate mean ± s.d., two independent experiments).

b, Experiments are described in Fig. 3d on HeLa (5 pM TcdB), HT-29 (50 pM TcdB), and Caco-2 cells (150 pM TcdB). Scale bars = 50 μm (HeLa and Caco-2) or 25 μm (HT-29). Representative images are from one of four independent experiments.

PVRL3 failed to mediate binding and entry of TcdB in HeLa and Caco-2 cells

a,
CSPG4−/− HeLa cells transfected with the indicated constructs were exposed to TcdB in medium for 10 min. Cell lysates were collected and subjected to immunoblot analysis. Expression of PVRL3 was confirmed using an anti-PVRL3 antibody. Transfection of FZD2, but not PVRL-3, increased binding of TcdB to CSPG4−/− cells. One of three independent experiments is shown.

b, Cells were challenged with TcdB (300 pM). Ectopic expression of PVRL3 failed to restore the sensitivity of CSPG4−/− HeLa cells towards TcdB, while expression of FZD2 restored entry of TcdB and resulted in rounding of transfected cells. Co-transfected GFP marked transfected cells. Scale bar = 50 μm. One of three independent experiments is shown.

c, Recombinant extracellular domain of PVRL3 (PVRL3-EC) did not reduce TcdB entry into Caco-2 cells, analyzed by cytopathic cell-rounding assay. In contrast, FZD2-CRD prevented entry of TcdB into Caco-2 cells. Scale bar = 50 μm. One of three independent experiments is shown.

Colonic organoids showed similar levels of sensitivity to TcdB and TcdB1-1830 and validation of FZD1 and FZD2 KD efficiency

a, Colonic organoids were cultured from WT mice. They were exposed to a gradient of TcdB or TcdB1-1830. Viability of organoids was quantified using the MTT assay. TcdB and TcdB1-1830 showed similar IC50, suggesting that WT organoids are equally susceptible to TcdB and TcdB1-1830 (n = 8, error bars indicate mean ± s.d., two independent experiments). n.s.: not significant.

b, Immunoblot analysis of CSPG4 expression in mouse brain, colonic organoids, mouse whole colon tissue, and isolated mouse colonic epithelium (200 μg cell/tissue lysates). The colonic epithelium was isolated from colon tissues by EDTA treatment (10 mM, 2 h at 4 °C). One of three independent experiments is shown.

c–d, ShRNA sequences targeting FZD1 and FZD2 were validated by measuring KD efficiency of transfected 1D4-tagged FZD1 and FZD2 in 293T cells. ShRNAs marked with asterisks (shRNA2 for FZD1 and shRNA5 for FZD2) were used to generate adenoviruses. Actin served as the loading control. One of two independent experiments is shown.

TcdB1114-1835 inhibits Wnt signaling and induces death of colonic organoids

a, TcdB1114-1835 blocked Wnt3a-mediated signaling in 293T cells in a dose-dependent manner. Increasing concentrations of Wnt restored Wnt reporter activity blocked by TcdB1114-1835. Wnt signaling activity was analyzed using the TOPFLASH/TK-Renilla dual luciferase reporter assay (error bars indicate mean ± s.d., two independent experiments). We note that 1.25 nM Wnt3a equals 50 ng/ml concentration used in Fig. 4c.

b, 293T cells in 24-well plates were exposed to Wnt3a (50 ng/ml) and TcdB1114-1835 in culture medium for 6 h. Cell lysates were harvested and subjected to immunoblot analysis for detecting phosphorylated Dvl2 and LRP6. Wnt signaling activation results in phosphorylation of Dvl2 and LRP6. Phosphorylated Dvl2 is marked with an asterisk. One of three independent experiments is shown.

c, Mouse colonic organoids were exposed to TcdB or TcdB1114-1835 for 12 h. Cell lysates were subjected to immunoblot analysis. No glucosylation of Rac1 was observed in organoids treated with TcdB1114-1835. One of two independent experiments is shown.

d, Colonic organoids were exposed to TcdB1114-1835 for 72 h, with or without CHIR99021 (5 μM). Normal organoids (green arrow), growth inhibited organoids (red arrow), and disrupted/dead organoids (*) are marked. Scale bar represents 200 μm. One of three independent experiments is shown.

e, Time-course images of colonic organoids exposed to CHIR99021 (5 μM), TcdB1114-1835 (25 nM), or a combination of TcdB1114-1835 plus CHIR99021, at 0, 2, 4, and 6 days. Normal organoids (green arrow), growth inhibited organoids (red arrow), and disrupted/dead organoids (*) are marked. Scale bar represents 500 μm. One of four independent experiments is shown.

FZDs are receptors for TcdB in the colonic epithelium

a–c, Human colon cryosections were obtained from a commercial vendor and subjected to IHC analysis for detecting FZD7 (a), FZD2 (b), and CSPG4 (c). Ep: epithelial cells; Mf: sub-epithelial myofibroblasts. Scale bar = 50 μm. Representative images are from one of three independent experiments.

d, Expression of FZD1 is not detectable in mouse or human colonic tissues. One of three independent experiments is shown.

e, FZD7 antibody labeled WT colonic sections, but showed no signals on colonic tissues from FZD7−/− mice in IHC analysis, confirming the specificity of this antibody. One of three independent experiments is shown.

f, Immunostaining of FZD2 (green) is reduced in FZD2-KD colonic organoids compared to control organoids, confirming the specificity of FZD2 antibody. Cell nuclei were labeled by DAPI (blue). Scale bar = 30 μm. One of three independent experiments is shown.

g, Experiments are described in Fig. 5g. Representative images from one of three independent experiments are shown. Scale bar = 100 μm.

h, Experiments were carried out as described in Fig. 5h. Low-magnification images of immunofluorescent staining of the cell-cell junction markers Claudin-3 (green) and ZO-1 (red) were stitched together to show an overview of the colon tissue. The middle panel (WT/TcdB) showed disruption of the normal staining pattern for Claudin-3 and ZO-1, indicating a loss of epithelial integrity, compared with both control and FZD7−/−/TcdB1-1830. Scale bar represents 200 μm. Representative images are from one of three independent experiments.

i, A schematic overview of cellular factors identified in the CRISPR/Cas9 screen. Validated and plausible cellular factors identified in our unbiased genome-wide screens were grouped based on their presence in the same protein complexes and/or signaling pathways. The color of the gene names reflects the number of unique sgRNA identified. The arrows link these genes to either confirmed or plausible roles in four major steps of TcdB action: (1) receptor-mediated endocytosis; (2) low pH in the endosomes triggers conformational changes of the TD, which translocates the GTD across endosomal membranes; (3) GTD is later released via auto-proteolysis by the CPD, which is activated by the cytosolic co-factor inositol hexakisphosphate (InsP6); (4) released GTD glucosylates small GTPases such as Rho, Rac, and CDC42.

Supplementary Materialsupp_figuresupp_info

Supplementary Material
