---
pmid: '24008843'
title: Structure homology and interaction redundancy for discovering virus-host protein
  interactions.
authors:
- de Chassey B
- Meyniel-Schicklin L
- Aublin-Gex A
- Navratil V
- Chantier T
- André P
- Lotteau V
journal: EMBO Rep
year: '2013'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC3807215
doi: 10.1038/embor.2013.130
---

# Structure homology and interaction redundancy for discovering virus-host protein interactions.
**Authors:** de Chassey B, Meyniel-Schicklin L, Aublin-Gex A, Navratil V, Chantier T, André P, Lotteau V
**Journal:** EMBO Rep (2013)
**DOI:** [10.1038/embor.2013.130](https://doi.org/10.1038/embor.2013.130)
**PMC:** [PMC3807215](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3807215/)

## Abstract

1. EMBO Rep. 2013 Oct;14(10):938-44. doi: 10.1038/embor.2013.130. Epub 2013 Sep
6.

Structure homology and interaction redundancy for discovering virus-host protein 
interactions.

de Chassey B(1), Meyniel-Schicklin L, Aublin-Gex A, Navratil V, Chantier T, 
André P, Lotteau V.

Author information:
(1)Hospices Civils de Lyon, Hôpital de la Croix-Rousse, Laboratoire de 
virologie, 103 grande rue de la Croix-Rousse, Lyon, F-69004, France.

Virus-host interactomes are instrumental to understand global perturbations of 
cellular functions induced by infection and discover new therapies. The 
construction of such interactomes is, however, technically challenging and time 
consuming. Here we describe an original method for the prediction of 
high-confidence interactions between viral and human proteins through a 
combination of structure and high-quality interactome data. Validation was 
performed for the NS1 protein of the influenza virus, which led to the 
identification of new host factors that control viral replication.

DOI: 10.1038/embor.2013.130
PMCID: PMC3807215
PMID: 24008843 [Indexed for MEDLINE]

Conflict of interest statement: The authors declare that they have no conflict 
of interest.

## Full Text

Abstract

Virus–host interactomes are instrumental to understand global perturbations of cellular functions induced by infection and discover new therapies. The construction of such interactomes is, however, technically challenging and time consuming. Here we describe an original method for the prediction of high-confidence interactions between viral and human proteins through a combination of structure and high-quality interactome data. Validation was performed for the NS1 protein of the influenza virus, which led to the identification of new host factors that control viral replication.

Introduction

Viruses are obligate parasites that rely on cellular functions to replicate. Identifying the cellular functions that a virus must use, counteract or interfere with to assure its replication is therefore a major issue for both basic knowledge and drug discovery. Biological processes being largely dependent on protein–protein interactions, cellular functions involved in viral replication can be identified by mapping virus–host protein–protein interactions and by constructing virus–host interactomes. Until 2007, viral protein–host protein interactions have essentially been identified in low-throughput experiments. Several databases are integrating these data spread in the literature [ 1–2 ]. The construction of physical viral ORFeomes and the development of high-throughput technologies, such as yeast two-hybrid (Y2H) and tandem affinity purification coupled to mass spectrometry, led to the publication of the first virus–host interactomes [ 3–4 ]. Although incorporation of data from different methods or variation of the same method [ 5 ] has improved the quality of the data sets, diversification of methods is still clearly needed to generate high-quality comprehensive virus–host interactomes. In addition, regarding the size of host genome and the huge diversity of viruses, millions of binary interactions remain to be tested. Therefore, accurate and rapid methods for the identification of cellular interactors controlling viral replication is a major issue and a crucial step towards the selection of original therapeutic targets and drug development. This could benefit from predictive methods preselecting subsets of putative interactions. Computational methods are essentially dedicated to predict intraspecies interactions [ 6 ] but applications to pathogen–host protein interactions are now emerging [ 7 ]. These methods are generally evaluated by the overlap between predicted and published interaction data sets. Here, we present and experimentally validate an original method for the prediction of virus–host interactions combining protein structure homology and interaction redundancy.

Principle of the method

Several methods have proposed to use structural information to predict protein–protein interactions [ 8–10 ]. The method described here relies on the assumption that when two proteins are structurally homologous, they are more likely to have interactors in common ( Fig 1A ). Using the Structural Classification of Proteins from SCOP database [ 11 ] and the high-quality data sets of protein–protein interactions from the VirHostNet database [ 2 ], we showed that this assumption is a general feature of human and viral proteins ( supplementary information online). Briefly, for a viral protein having a solved structure, structurally homologous human and viral proteins are first selected. Interactors of these homologous proteins are then identified. These proteins are considered as putative interactors and ranked according to a score that favors proteins independently identified from multiple structural homologues.

Methods

Retrieval and annotation of viral and human structures . All sequences from viral and human protein structures stored at the PDB were retrieved in a Fasta format and blasted to be annotated with a GenBank identification for viral proteins and Ensembl protein identification for human proteins as it is the case in the VirHostNet database. A total of 4,933 PDB sequences (viral GenBank identification) and 22,052 PDB sequences (human Ensembl protein identification) have been retrieved and assigned to 440 distinct viral proteins and 3,163 distinct human proteins. For the full-length influenza A virus NS1 protein, the chain A of the 3F5T PDB structure has been used.

High-quality protein–protein interaction data from VirHostNet . Protein–protein interaction data were retrieved from the VirHostNet database [ 2 ]. A high-quality data set of human protein–protein interactions were defined by selecting interactions identified by two different methods or in two independent papers. Indeed, the human interactome is an integration of data from several databases, and might be polluted by low-confidence interactions [ 25 ]. A high-quality data set of virus–human protein–protein interactions were constructed with exclusively manually curated interactions. To experimentally validate the predicted interactions, an additional level of stringency was added by focusing on interactions described by classical laboratory technologies. Altogether, the predictions are on the basis of 16,187 human protein–protein interactions involving 6,378 distinct human proteins and 2,539 virus–host protein–protein interactions involving 434 viral proteins and 1,395 human proteins ( supplementary Tables S6 and S7 online).

Score definition . All putative interactors were individually scored according to several criteria. First, an interactor is more confident when predicted by several independent NS1 homologues. Interactors independently predicted by both viral and human homologues are favored. The prediction from viral homologues only is also better considered than the prediction from human homologues only. Finally, as a correlation was observed between the score and the degree of a putative interactor (that is, the number of partners in the protein interaction data set), the score was weighted by the degree. Altogether, the score formula is as follows: score (putative interactor=[( n H 2 / K H )+( n V 2 / k V )] × a , with: (i) n H and n V number of distinct human interactors and viral interactors, respectively. (ii) k h and k V degree of the protein in the human protein–protein interaction data set and in the virus–human protein–protein interaction data set, respectively. (iii) a =3 for a prediction from both human and virus–human data sets, a =2 from only virus–human data set, 1 else. (iv) H: human, V: virus.

Cells and virus . A549 human lung epithelial cell line and Madin-Darby canine kidney (MDCK) cell line were grown in DMEM media supplemented with 100 U ml −1 penicillin/streptomycin and 10% fetal calf serum at 37 °C and 5% CO 2 The epidemic A/H1N1/New Caledonia/2006 strain was propagated in MDCK cells in DMEM supplemented with 1 μg ml −1 modified trypsin TPCK in absence of fetal calf serum. Virus stocks were titrated by standard plaque assay on MDCK cells using an agar overlay medium.

GST pull down . Cellular proteins (corresponding to the 32 cDNAs available in orfeotech ORFeome v3.1 [ 26 ], description of the hORFeome v3.1 in supplementary information online) were amplified from the MGC cDNA plasmid collection using KOD polymerase (Toyobo), cloned by recombinational cloning into pDONR207 (Life technologies) and transferred to pDEST-myc or pCIneo3xFlag (kind gift of Y. Jacob). H5N1 NS1 was transferred into pDEST27 (GST fusion in N-term). A total of 4.10 5 HEK293T cells were then co-transfected (6 μl JetPei, Polyplus) with 1.5 μg of each pair of plasmid. Two days after transfection, cells were harvested and lysed (0.5% NP-40, 20 mM Tris–HCl (pH 8.0), 150 mM NaCl, 1 mM EDTA and Roche complete protease inhibitor cocktail). Soluble protein complexes were purified by incubating 300 μg of cleared cell lysate with 40 μl glutathione sepharose 4B beads (GE Healthcare). A total of 50 μg of cleared cell lysate was separated on SDS–PAGE and transferred to nitrocellulose membrane. GST-tagged viral proteins and Myc or 3XFlag-tagged cellular proteins were detected using standard immunoblotting techniques with anti-GST (Covance), anti-Myc or anti-Flag (Sigma) monoclonal antibodies.

Yeast two-hybrid . Pairwise Y2H interactions were analysed by yeast mating, using Y187 and AH109 yeast strains (Clontech). Prey and bait vectors were transformed into Y187 and AH109. Bait and prey strains were mated in an all-against-all array and plated on a selective medium lacking histidine and supplemented with increasing concentrations of 3-amino-triazole to test the interaction-dependent transactivation of HIS3 reporter geneSome of the preys are membrane proteins but reported in the literature to give positive results in Y2H (for example, BCL2L11, BAD) or proteins whose signal peptide is likely to be masked ( supplementary information online). Therefore, we did not exclude them from the Y2H.

siRNA screening . Five picomoles of each siRNA (stealth select RNAi, Invitrogen) was arrayed in 96 plates in 10 μl of OptiMEM (GibCo). Ten microliters per well of a mix lipofectamine RNAiMAX (Invitrogen)-OptiMEM (0.2 μl of lipofectamine RNAiMAX in 10 μl of OptiMEM) was added. After 20 min room temperature-incubation, siRNA–lipofectamine mixes were added to 30 × 10 3 A549 suspension cells. Cells were incubated for 48 h at 37 °C and 5% CO 2 before influenza virus infection at multiplicity of infection 0.5. Forty-eight hours after infection, supernatants were titrated.

Virus infection . siRNA-transfected cells were washed twice with Dulbecco’s phosphate-buffered saline 1 × and infected with H1N1 influenza strain at multiplicity of infection 0.5 in Dulbecco’s modified Eagle medium supplemented with 0.2 μg ml −1 trypsin TPCK (infection medium). After 60 min at 37 °C, the inoculum was discarded and cells were washed again and incubated for 48 h in infection medium at 37 °C and 5% CO 2 .

Titre measure by neuraminidase activity . Influenza virus neuraminidase is able to cleave the methyl-umbelliferyl- N -acetylneuraminic acid (4-MUNANA, Sigma) modifying its emission wavelength in a dose-dependent manner. In 96-black plate, 25 μl infection supernatants were diluted in 25 μl Dulbecco’s phosphate-buffered saline 1 × containing calcium and magnesium and 50 μl of 20 μM 4-MUNANA. After 1 h incubation at 37 °C, 100 μl of glycine 0.1 M 25% ethanol pH 10.7 was added. Measures were done with TECAN infinite M1000 instrument at 365 nm excitation and 450 nm emission wavelengths.

Plaque-forming unit assay . To quantify infectious virus particles in infected cell culture supernatants, 300,000 MDCK cells were seeded in 6-well plates. Three days later, cells were washed twice with DMEM supplemented with 100 U ml −1 penicillin/streptomycin. Dilutions of infected cell culture supernatants were dispensed on the cells. After 1 h 30 min incubation at 37 °C and 5% CO 2 cells were washed again and covered with overlay medium containing MEM 1 ×, 1% agarose (Lonza) and modified trypsin TPCK 1 μg ml −1 . Plates were incubated upside down at 37 °C and 5% CO 2 up to 72 h. Cells were then fixed with formol 10% and coloured with crystal violet 0.3% in methanol 20% and lysis plaques were counted.

Functional annotation . DAVID functional annotation chart tool was used to perform enrichment analysis, in which an annotation term was considered as significant with a Benjamini–Hochberg corrected P -value smaller than 0.05 [ 27 ].

Supplementary information is available at EMBO reports online ( http://www.emboreports.org ).
