---
pmid: '16217013'
title: Heterogeneous nuclear ribonucleoprotein C1/C2, MeCP1, and SWI/SNF form a chromatin
  remodeling complex at the beta-globin locus control region.
authors:
- Mahajan MC
- Narlikar GJ
- Boyapaty G
- Kingston RE
- Weissman SM
journal: Proc Natl Acad Sci U S A
year: '2005'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC1257739
doi: 10.1073/pnas.0507596102
---

# Heterogeneous nuclear ribonucleoprotein C1/C2, MeCP1, and SWI/SNF form a chromatin remodeling complex at the beta-globin locus control region.
**Authors:** Mahajan MC, Narlikar GJ, Boyapaty G, Kingston RE, Weissman SM
**Journal:** Proc Natl Acad Sci U S A (2005)
**DOI:** [10.1073/pnas.0507596102](https://doi.org/10.1073/pnas.0507596102)
**PMC:** [PMC1257739](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1257739/)

## Abstract

1. Proc Natl Acad Sci U S A. 2005 Oct 18;102(42):15012-7. doi: 
10.1073/pnas.0507596102. Epub 2005 Oct 10.

Heterogeneous nuclear ribonucleoprotein C1/C2, MeCP1, and SWI/SNF form a 
chromatin remodeling complex at the beta-globin locus control region.

Mahajan MC(1), Narlikar GJ, Boyapaty G, Kingston RE, Weissman SM.

Author information:
(1)Department of Genetics, The Anlyan Center, Yale University School of 
Medicine, New Haven, CT 06511, USA.

Locus control regions (LCRs) are regulatory DNA sequences that are situated many 
kilobases away from their cognate promoters. LCRs protect transgenes from 
position effect variegation and heterochromatinization and also promote 
copy-number dependence of the levels of transgene expression. In this work, we 
describe the biochemical purification of a previously undescribed LCR-associated 
remodeling complex (LARC) that consists of heterogeneous nuclear 
ribonucleoprotein C1/C2, nucleosome remodeling SWI/SNF, and nucleosome 
remodeling and deacetylating (NuRD)/MeCP1 as a single homogeneous complex. LARC 
binds to the hypersensitive 2 (HS2)-Maf recognition element (MARE) DNA in a 
sequence-specific manner and remodels nucleosomes. Heterogeneous nuclear 
ribonucleoprotein C1/C2, previously known as a general RNA binding protein, 
provides a sequence-specific DNA recognition element for LARC, and the LARC 
DNA-recognition sequence is essential for the enhancement of transcription by 
HS2. Independently of the initiation of transcription, LARC becomes associated 
with beta-like globin promoters.

DOI: 10.1073/pnas.0507596102
PMCID: PMC1257739
PMID: 16217013 [Indexed for MEDLINE]

## Full Text

Abstract

Locus control regions (LCRs) are regulatory DNA sequences that are situated many kilobases away from their cognate promoters. LCRs protect transgenes from position effect variegation and heterochromatinization and also promote copy-number dependence of the levels of transgene expression. In this work, we describe the biochemical purification of a previously undescribed LCR-associated remodeling complex (LARC) that consists of heterogeneous nuclear ribonucleoprotein C1/C2, nucleosome remodeling SWI/SNF, and nucleosome remodeling and deacetylating (NuRD)/MeCP1 as a single homogeneous complex. LARC binds to the hypersensitive 2 (HS2)-Maf recognition element (MARE) DNA in a sequence-specific manner and remodels nucleosomes. Heterogeneous nuclear ribonucleoprotein C1/C2, previously known as a general RNA binding protein, provides a sequence-specific DNA recognition element for LARC, and the LARC DNA-recognition sequence is essential for the enhancement of transcription by HS2. Independently of the initiation of transcription, LARC becomes associated with β-like globin promoters.

Materials and Methods

Cell Culture and Preparation of Nuclear Extracts. Growth of the K562 cells and preparation of the nuclear extract is described in ref. 24 . For large-scale cultures, 2 liters of the K562 cells grown in 175-cm flasks were transferred to the 10-liter Cytostir Cell Culture jar and grown until the cell density reached 0.75 × 10 6 per ml.

Electrophoretic Mobility Shift Assay (EMSA). The EMSA and transient transfection assays were conducted as described in ref. 24 . Unless otherwise indicated, the EMSA binding mixture contained 4 ng of 32 P-labeled double-stranded oligonucleotide and 5 μg of crude k562 nuclear extract or 2 μg of K562 nuclear extract partially purified on a heparin-agarose column.

Antibodies (Abs), Immunoprecipitation, and Western Blotting. The Abs against Brg1 (catalog no. SC-17796 and SC-12520x), BAF155 (SC-10756x), actin (SC-10731), HDAC1 (SC-6298), INI1 (SC-13055x), MBD2 (SC-9397), MBD3 (SC-9402), Mi2β (SC-12541), and heterogeneous nuclear ribonucleoprotein (hnRNP) C1/C2 (SC-15286) were purchased from Santa Cruz Biotechnology. The Ab against MTA2 (PC656-1) was purchased from Oncogene Research (Cambridge, MA). The RbAP48 Ab (catalog no. 05-524) was purchased from Upstate Biotechnology (Lake Placid, NY). Immunoprecipitation (IP) and immunodepletion procedures were carried out at 4°C. The LCR-associated remodeling complex (LARC) eluted from the heparin-agarose column was dialyzed against 10 mM Hepes buffer (pH 7.9) containing 10% glycerol, 50 mM KCl, 1 mM MgCl 2 , 1 mM DTT, 1 mM PMSF, and Complete protease inhibitor tablet (at one tablet per 100-ml buffer concentration; Roche Diagnostics), concentrated to ≈1 mg/ml, and used for IP and immunodepletions. For each IP, 500 μl of the K562 nuclear protein containing ≈500 μg of the protein and 3 μg of each of the anti-Brg1, Mi2β, MTA2, and hnRNP C1/C2 Ab was used. Three micrograms of the goat and rabbit IgG were added to the K562 nuclear proteins in a control experiment. Western blotting was performed as described in ref. 24 . Five hundred micrograms of K562 nuclear extract, eluted with 0.6 M NaCl from heparin-agarose column, was immunodepleted with 70 μg each of Abs against Brg1, Mi2β, and hnRNP C1/C2. A mixture of 35 μg each of normal rabbit and goat IgG was used as control. The Abs and K562 extracts were incubated overnight followed by 2-h incubation with 100 μl of Protein-G beads. The beads were separated by centrifugation, and the supernatants were analyzed by Western blotting using Abs against several LARC components.

Purification of the LARC as a MARE Binding Activity. Throughout the purification procedure, the LARC was assayed by EMSA as a MARE binding activity. The entire purification procedure was carried out at 4°C, and, unless otherwise indicated, all of the buffers contained 1 mM PMSF, 1 mM bestatin, 1 mM diisopropyl fluorophosphate, 1 μg/ml l -1-tosylamido-2-phenylethyl chloromethyl ketone, and 7-amino-1-chloro-3-tosylamido-2-heptanone, and Complete protease inhibitor pill (Roche) per 50 ml of buffer. Typically, the starting K562 nuclear extract isolated from 8–10 × 10 9 K562 cells containing 85–120 mg of protein in a nuclear extract storage buffer (20 mM Hepes, pH 7.9/100 mM KCl/1 mM MgCl 2 /20% glycerol/1 mM DTT) was loaded onto a 30-ml heparin-agarose column preequilibrated with the same buffer. The column was washed with 300 ml of the storage buffer followed by 300 ml of 0.2 M NaCl and 300 ml of 0.42 M NaCl in 10 mM Hepes buffer (pH 7.9) containing 10% glycerol, 1 mM MgCl 2 , and 1 mM DTT. The MARE-binding activity was eluted from the column by 0.6 M NaCl in the same buffer. The active fractions were detected by EMSA, pooled, and dialyzed against the DNA-affinity column buffer consisting of 10 mM Hepes buffer (pH 7.9) containing 10% glycerol, 50 mM KCl, 1 mM MgCl 2 , and 1 mM DTT. After dialysis, the protein was centrifuged at 15,000 rpm in a Sorvall SS-34 rotor for 15 min and the clear supernatant was collected. Two hundred micrograms of the poly(dIdC) was added to the dialyzed sample and loaded on a MARE–DNA–Sepharose 4B-CL affinity column preequilibrated with the same buffer. The column was washed with the 50 ml of the DNA-affinity column buffer containing 200 μg of the poly(dIdC) followed by 50 ml of the 0.1 M NaCl in DNA-affinity buffer. The MARE-binding activity from the column was eluted by 0.2 M NaCl in the DNA-affinity column buffer, pooled, dialyzed against the DNA-affinity column buffer, and concentrated to 100 μl in a spin concentrator (Orbital Biosciences, Topsfield, OH). At this stage the MARE-binding activity is purified to electrophoretic homogeneity and is used for further analytical and functional studies.

Additional Methods. Procedures for ChIP, preparation of nucleosomes and remodeling assays, UV-crosslinking of the DNA–protein complex, MALDI, and liquid chromatography-tandem MS (LC-MS/MS) analysis are described in Supporting Text , which is published as supporting information on the PNAS web site.

Results

The MARE sequence of the HS2 contains the core enhancer activity of the HS2 ( 11 , 12 ). The first AP1 sequence of the MARE with an extended GC dinucleotide at its 5′ end is the binding site for p45 NF-E2 ( Fig. 1 A ). Mutation of this GC dinucleotide disturbs the binding of the p45-p18 NF-E2 protein dimer ( 25 ) and drastically reduces the enhancer activity of the HS2 ( 11 ). In an in vitro gel shift assay, MARE sequence interacts with the NF-E2 protein complex as well as several unidentified proteins ( 13 , 26 ). When we fractionated the K562 nuclear extracts on a heparin-agarose column, the p45 NF-E2 eluted with the 0.42 M NaCl ( Fig. 1 B ). Further elution of the proteins with 0.6 M NaCl did not contain the p45 NF-E2, but these proteins were able to bind to the MARE sequence ( Fig. 1 C ). We have previously shown these bands to be sequence specific ( 24 ). Mutation in the NF-E2 binding site did not disturb these EMSA bands. However, mutations in the second AP1 sequence at the 3′ end of the NF-E2 sequence (Mut1 and Mut2) neutralize these EMSA bands, suggesting that they are sequence specific and distinct from the NF-E2 complex ( Fig. 1 C ). To explore whether these unaccounted major EMSA bands contribute to the LCR function, we undertook the biochemical purification of this MARE-binding activity.

The MARE-Binding Activity Is a Heteromeric Complex of SWI / SNF, Nucleosome Remodeling and Deacetylating (NuRD) / MeCP1, and RNA Binding Proteins. We purified the non-NF-E2-containing EMSA activity obtained with K562 nuclear extracts eluted from a heparin-agarose column with 0.6 M NaCl by specific DNA-affinity chromatography ( Fig. 2 ). The nuclear extract from 10 10 K562 cells was fractionated on a heparin-agarose column followed by a DNA-affinity column in which the double-stranded MARE oligonucleotide with a GATC overhang was attached to CNBr-activated Sepharose-CL-4B ( Fig. 2 A ). In a typical purification procedure, we started with 95 mg of crude nuclear extract and obtained 35 μg of the purified protein for a total yield of 0.037% of the starting nuclear extract. Resolution of the purified protein complex on 10% SDS/PAGE gave number protein bands of near similar intensity ( Fig. 2 C ). These bands were identified by MALDI-TOF, by sequencing the tryptic peptides by LC-MS/MS, and Western blotting. Several of the SDS/PAGE protein bands contained more than one protein. By these methods, we identified 18 proteins from the most highly purified MARE-binding complex. These proteins included all of the components of the chromatin remodeling SWI/SNF ( 27 ) complex and most of the NuRD/MeCP1 subunits ( 28 ) listed in Fig. 2 C . We could not detect the MBD2 and MBD3 components of the NuRD/MeCP1 complex by Coomassie blue staining, MALDI, or LC-MS/MS analysis but were able to detect them by Western analysis ( Fig. 2 D ). It is possible that these proteins associate weakly with this complex and fall off during purification or that their association with the other components may be regulated in vivo . Interestingly, we identified the 40- and 37-kDa subunits of this complex as RNA-binding protein hnRNP C1/C2. The LC-MS/MS analysis of the tryptic digest of the 45-kDa band revealed the presence of actin and requiem, a zinc finger protein that is associated with apoptosis ( Fig. 2 C ) ( 29 ). However, because there was no available Ab against requiem, we could not confirm its presence in this complex.

To investigate the possibility of coelution of separate SWI/SNF and NurD/MeCP1 chromatin remodeling complexes during the purification of the MARE-binding activity, we passed the proteins eluted from the heparin-agarose column over a Superose-6 sizing column. We obtained a protein peak that is >2 mDa in size and coincides with the EMSA peak at the same position ( Fig. 3 A ). Western blotting of the Superpose-6 column fractions with Abs against Brg1, MTA2, Mi2β, RbAP48, INI1, and hnRNP C1/C2 ( Fig. 3 B ) showed that the peak position of material containing these proteins coincided with the >2 mDa protein complex and peak EMSA fractions of the Superose-6 column, suggesting that all these proteins might exist as a single complex and together constitute the MARE-binding activity. To further confirm the existence of these proteins as a single complex, we conducted coimmunoprecipitation ( Fig. 3 Ci ) and immunodepletion ( Fig. 3 Cii ) experiments with the partially purified complex from a heparin-agarose column. When sufficient specific Ab against the Mi2β, Brg1, and hnRNP C1/C2 was used to completely deplete these proteins from the K562 nuclear extracts passed through the heparin-agarose column, all of the tested components of SWI/SNF and NuRD/MeCP1 complexes and hnRNP C1/C2 proteins were removed from the supernatant, confirming their association in a single complex ( Fig. 3 Cii ). We call this protein complex LARC. Gel supershift experiments demonstrated the direct interaction between the LARC and the MARE sequences ( Fig. 3 D ). Ab against NuRD/MeCP1 component Mi2β supershifted the LARC bands, and the Ab against SWI/SNF component BAF155 neutralized them, indicating the presence of these proteins in EMSA bands.

The hnRNP C1 / C2 Component of LARC Directly Contacts the Target Nucleotide Sequence. To further investigate the proteins involved in determining the sequence specificity of LARC, we performed Southwestern analysis (see Fig. 4 A ). The proteins in a purified LARC preparation were fractionated on a 4–15% SDS/PAGE gel and blotted onto an Immobilon-P membrane. Hybridization of this membrane with 32P-labeled MARE oligonucleotide yielded a radioactive 37-kDa and a 30-kDa band not seen with a mutated oligonucleotide. Western analysis of the same South-western blot and of a mirror blot run alongside clearly showed that the radioactive bands coincided with the position of hnRNP C1/C2 proteins (see Fig. 4 A ). The relative intensities of the Western bands suggest that the lower 30-kDa band could be a minor degraded product of the 37-kDa hnRNP C1/C2 protein.

To confirm the direct contact between hnRNP C1/C2 and the MARE oligonucleotide, the LARC complex was crosslinked with 5-bromodeoxy uridine containing MARE DNA and analyzed the result by Western blotting using anti-hnRNP C1/C2 Ab ( Fig. 4 B ). A part of the hnRNP C1/C2 protein supershifted to 250-kDa molecular mass range, whereas such a supershift of the MTA2 protein was not seen, suggesting the specificity of the hnRNP C1/C2-MARE DNA crosslinking. As a control, when LARC was UV-crosslinked in the absence of DNA, there was no change in the mobility of the 37-kDa hnRNP C1/C2 band. Moreover, addition of the anti-hnRNP C1/C2 Ab in an EMSA with 32 P MARE oligo and K562 nuclear extract resulted in the supershifting of the LARC–EMSA bands ( Fig. 4 C ). These data suggest that hnRNP C1/C2 is at least partly responsible for the binding of the LARC complex to its target DNA sequence.

LARC Has Chromatin Remodeling Activity. We used a restriction enzyme accessibility assay in which the accessibility of nucleosomal DNA was monitored by measuring the rate at which a restriction enzyme such as PstI cuts its cognate sequence. Nucleosomes were assembled on a 375-bp fragment of the core HS2 and its mutant variant (Mut2) that contained a PstI site at the 5′ end of the MARE sequences ( Fig. 5 A ). MNase treatment of this 375-bp nucleosomal HS2 suggested the formation of dinucleosome (data not shown). LARC increased the rate of PstI cutting of wild-type (WT) nucleosomal HS2 DNA by 10-fold in an ATP-dependent manner ( Fig. 5 A ), indicating that LARC contains a bona fide chromatin remodeling activity. Under subsaturating LARC concentrations (0.3 nM), the WT template is remodeled ≈1.5 fold faster than the mutant (Mut2) template ( Fig. 5 A ). At saturating LARC (1.2 nM) this difference disappears, probably due to differences in binding. The MARE sequence is ≈35 bp. However, the template has 375 bp, and so there are several alternative nonspecific sequences that LARC can bind to within the same template and reduce the specific targeting effect of the MARE sequence. This in vitro nonspecific targeting effect might explain the reduced differentiation between the WT and mutant sequence.

We also compared LARC activity with the activity of human SWI/SNF complex purified from HeLa cells. We used a template containing five tandem repeats of the 3′ portion of the MARE sequence followed by a HhaI restriction enzyme site right next to it ( Fig. 5 B ). Both the LARC and HeLa SWI/SNF exposed all three restriction sites. When compared with the PstI site, both the HeLa-SWI/SNF and LARC expose the BglII site with equal efficiency. However, in comparison with the HeLa SWI/SNF, LARC exposes the HhaI site with 4-fold less efficiency. These data suggest that LARC preferentially binds to its specific sites and protects the neighboring HhaI site from cleavage.

LARC Is Present in Vivo on HS2 and β -Globin-Like Promoters. We used chromatin IP experiments to analyze the in vivo recruitment of the LARC on the HS2 as well as other regions of the β-globin locus ( Fig. 6 A ). We immunoprecipitated formaldehyde fixed fragments of K562 chromatin with Abs against Brg1, Mi2β, MTA2, and hnRNP C1/C2 and amplified the regions from the LCR, β-like globin promoters and intergenic regions of the β-globin locus. These ChIP data suggest that LARC is also recruited in vivo to the γ-globin and β-globin promoters. This observation prompted us to look for the LARC-binding elements on the β- and γ-globin promoters. An EMSA with 10 double-stranded oligonucleotides spanning a 286-bp region of the A γ -globin promoter showed small amounts of binding of the LARC ( Fig. 6 B ). In comparison to the competition with 100-fold molar excess of the nonradioactive MARE sequence, none of the A γ -globin promoter oligonucleotides could compete well with the MARE–LARC complex ( Fig. 6 C ). A sequence called the promoter conserved sequence (PCS) is situated between –115 and –140 bp from the β-globin promoter transcription start site and is partly homologous with the MARE sequence of the HS2. EMSA with PCS oligonucleotides incubated with crude nuclear extracts did not show bands at the position of LARC complexes but instead a series of bands that contained the DNA-dependent ATPase HLTF ( 24 ). However, EMSA with the PCS oligonucleotide and partially purified LARC from a heparin-agarose column showed a significant amount of binding ( Fig. 6 D ). In a competitive EMSA, we observed almost complete competition of the MARE–LARC and PCS–LARC complex with an excess of MARE oligonucleotide but only partial competition of the MARE–LARC complex with an excess of PCS oligonucleotide ( Fig. 6 D ). These results suggest that LARC binds sequence specifically to HS2 with higher affinity than to the promoter sequences.

Discussion

Transcriptional control of the β-globin locus has been extensively studied, and several of the sequence specific transcription factors that bind to this locus are well characterized. Reports on the recruitment of chromatin remodeling complexes on the β-globin locus are restricted to the β-globin promoter and a pyrimidine-rich DNA sequence upstream of the δ-globin promoter ( 30 – 32 ). GATA-1 and MafK have been described as associated with chromatin remodeling complexes ( 33 , 34 , 44 ). However, their specific binding sites on the β-globin locus have not been determined. The role of a boundary element in controlling the spread of the heterochromatin from an upstream end of the β-globin LCR is well studied ( 3 , 4 , 35 , 36 ), but so far the isolation of any chromatin opening activity of the LCR has proved elusive. Here, we describe the structural and functional properties of a biochemically purified β-globin LARC.

Several lines of evidence indicate that LARC exists as a homogenous single complex of chromatin remodeling SWI/SNF and NurD/MeCP1 complexes and RNA-binding proteins hnRNP C1 and hnRNP C2 (Figs. 2 and 3 ). The association of SWI/SNF and MeCP1/NuRD components in the LARC is interesting because the former is generally involved in the promotion of transcription, and the latter functions as a repressor ( 27 , 28 ). NurD binds to unmethylated histone N-terminal “tails” as well as to histone tails methylated at Lys-9, a modification associated with transcriptionally inactive chromatin ( 37 ). One possibility is that LARC is recruited to the transcriptionally inactive heterochromatic β-globin LCR by its NuRD/MeCP1 component, after which the SWI/SNF component may facilitate the recruitment of other transcription factors.

LARC has intrinsic sequence-specific DNA-binding activity (Figs. 1 C and 6 ). Mutations in the LARC-binding sequences (Mut2 in Fig. 1 C ) inhibit the enhancer activity of the HS2 as seen by transient transfection assays (data not shown). The binding sites for the LARC and the second AP1/NF-E2 site on the MARE sequence overlap ( Fig. 1 ). Further studies are needed to separate these binding activities to elucidate the role of NF-E2 and LARC in the HS2 enhancer activity and modulation in the chromatin structure of the β-globin locus. Data obtained in the present study with Southwestern hybridization, UV-crosslinking experiments with purified LARC, and gel supershift assays with the anti-hnRNP C1/C2 Ab show that hnRNP C1/C2 makes sequence-specific contacts with the DNA ( Fig. 4 ). However, we could not observe appreciable DNA-binding activity either by EMSA or by Southwestern hybridization using recombinant hnRNP C1 and hnRNP C2 proteins purified from bacterial extracts.

We detected LARC components on the active A γ - and G γ -globin promoters and the inactive β-promoter in K562 cells ( Fig. 6 A ). The erythroid transcription factor GATA1 is present in K562 cells and binds to the transcriptionally active γ-globin promoters. However, we and others could not detect binding of GATA1 to the β-globin promoter ( 38 , 39 ) in K562 cells, consistent with the lack of β-globin gene expression. Also, K562 cells do not express erythroid Krüppel-like factor (EKLF), a factor necessary for β-globin transcription ( 40 ). The presence of the LARC on the β-globin promoter implies that this recruitment may occur independently of GATA1 or EKLF binding and before the initiation of the β-globin transcription.

Although the ChIP experiments demonstrates the in vivo recruitment of LARC to HS2 as well as β-like promoters, the EMSA experiments with the 32 P-labeled MARE oligonucleotide or oligonucleotides from the γ-promoter region and β-promoter PCS sequences suggest that in comparison with the β-like promoters, the LARC binds to the HS2 with higher affinity and specificity ( Fig. 6 ). Thus, HS2 is the primary site for the binding of the LARC with a subsequent transfer to its cognate β-like promoters. The presence of hnRNP C1/C2 may suggest that RNA, such as the nascent intergenic RNA initiating in or near HS2 ( 41 – 43 ), could be relevant for the binding and/or intralocus transfer of LARC.
