---
pmid: '20061580'
title: AT_CHLORO, a comprehensive chloroplast proteome database with subplastidial
  localization and curated information on envelope proteins.
authors:
- Ferro M
- Brugière S
- Salvi D
- Seigneurin-Berny D
- Court M
- Moyet L
- Ramus C
- Miras S
- Mellal M
- Le Gall S
- Kieffer-Jaquinod S
- Bruley C
- Garin J
- Joyard J
- Masselon C
- Rolland N
journal: Mol Cell Proteomics
year: '2010'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC2877971
doi: 10.1074/mcp.M900325-MCP200
---

# AT_CHLORO, a comprehensive chloroplast proteome database with subplastidial localization and curated information on envelope proteins.
**Authors:** Ferro M, Brugière S, Salvi D, Seigneurin-Berny D, Court M, Moyet L, Ramus C, Miras S, Mellal M, Le Gall S, Kieffer-Jaquinod S, Bruley C, Garin J, Joyard J, Masselon C, Rolland N
**Journal:** Mol Cell Proteomics (2010)
**DOI:** [10.1074/mcp.M900325-MCP200](https://doi.org/10.1074/mcp.M900325-MCP200)
**PMC:** [PMC2877971](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2877971/)

## Abstract

1. Mol Cell Proteomics. 2010 Jun;9(6):1063-84. doi: 10.1074/mcp.M900325-MCP200. 
Epub 2010 Jan 10.

AT_CHLORO, a comprehensive chloroplast proteome database with subplastidial 
localization and curated information on envelope proteins.

Ferro M(1), Brugière S, Salvi D, Seigneurin-Berny D, Court M, Moyet L, Ramus C, 
Miras S, Mellal M, Le Gall S, Kieffer-Jaquinod S, Bruley C, Garin J, Joyard J, 
Masselon C, Rolland N.

Author information:
(1)INSERM, Laboratoire d'Etude de Dynamique des Protéomes, U880, France.

Recent advances in the proteomics field have allowed a series of high throughput 
experiments to be conducted on chloroplast samples, and the data are available 
in several public databases. However, the accurate localization of many 
chloroplast proteins often remains hypothetical. This is especially true for 
envelope proteins. We went a step further into the knowledge of the chloroplast 
proteome by focusing, in the same set of experiments, on the localization of 
proteins in the stroma, the thylakoids, and envelope membranes. LC-MS/MS-based 
analyses first allowed building the AT_CHLORO database 
(http://www.grenoble.prabi.fr/protehome/grenoble-plant-proteomics/), a 
comprehensive repertoire of the 1323 proteins, identified by 10,654 unique 
peptide sequences, present in highly purified chloroplasts and their 
subfractions prepared from Arabidopsis thaliana leaves. This database also 
provides extensive proteomics information (peptide sequences and molecular 
weight, chromatographic retention times, MS/MS spectra, and spectral count) for 
a unique chloroplast protein accurate mass and time tag database gathering 
identified peptides with their respective and precise analytical coordinates, 
molecular weight, and retention time. We assessed the partitioning of each 
protein in the three chloroplast compartments by using a semiquantitative 
proteomics approach (spectral count). These data together with an in-depth 
investigation of the literature were compiled to provide accurate subplastidial 
localization of previously known and newly identified proteins. A unique 
knowledge base containing extensive information on the proteins identified in 
envelope fractions was thus obtained, allowing new insights into this membrane 
system to be revealed. Altogether, the data we obtained provide unexpected 
information about plastidial or subplastidial localization of some proteins that 
were not suspected to be associated to this membrane system. The spectral 
counting-based strategy was further validated as the compartmentation of well 
known pathways (for instance, photosynthesis and amino acid, fatty acid, or 
glycerolipid biosynthesis) within chloroplasts could be dissected. It also 
allowed revisiting the compartmentation of the chloroplast metabolism and 
functions.

DOI: 10.1074/mcp.M900325-MCP200
PMCID: PMC2877971
PMID: 20061580 [Indexed for MEDLINE]

## Full Text

Abstract

Recent advances in the proteomics field have allowed a series of high throughput experiments to be conducted on chloroplast samples, and the data are available in several public databases. However, the accurate localization of many chloroplast proteins often remains hypothetical. This is especially true for envelope proteins. We went a step further into the knowledge of the chloroplast proteome by focusing, in the same set of experiments, on the localization of proteins in the stroma, the thylakoids, and envelope membranes. LC-MS/MS-based analyses first allowed building the AT_CHLORO database ( http://www.grenoble.prabi.fr/protehome/grenoble-plant-proteomics/ ), a comprehensive repertoire of the 1323 proteins, identified by 10,654 unique peptide sequences, present in highly purified chloroplasts and their subfractions prepared from Arabidopsis thaliana leaves. This database also provides extensive proteomics information (peptide sequences and molecular weight, chromatographic retention times, MS/MS spectra, and spectral count) for a unique chloroplast protein accurate mass and time tag database gathering identified peptides with their respective and precise analytical coordinates, molecular weight, and retention time. We assessed the partitioning of each protein in the three chloroplast compartments by using a semiquantitative proteomics approach (spectral count). These data together with an in-depth investigation of the literature were compiled to provide accurate subplastidial localization of previously known and newly identified proteins. A unique knowledge base containing extensive information on the proteins identified in envelope fractions was thus obtained, allowing new insights into this membrane system to be revealed. Altogether, the data we obtained provide unexpected information about plastidial or subplastidial localization of some proteins that were not suspected to be associated to this membrane system. The spectral counting-based strategy was further validated as the compartmentation of well known pathways (for instance, photosynthesis and amino acid, fatty acid, or glycerolipid biosynthesis) within chloroplasts could be dissected. It also allowed revisiting the compartmentation of the chloroplast metabolism and functions.

Conclusion

This investigation concerns chloroplasts, the organelles of the plant cell that harbor the chemical process by which green plants synthesize organic compounds from carbon dioxide and water in the presence of sunlight. Analyzing the chloroplast proteome not only helps understanding how plants efficiently harvest light energy but will also lead to a better understanding of the multitude of functions of the chloroplast as the chloroplast is essential for carbon fixation, lipid metabolism, and starch and amino acid biosynthesis. This constitutes an important economic issue not only from the plant biotechnology standpoint but also for the production of highly energetic compounds via cultivation of microalgae, such as Chlamydomonas reinhardtii . The present work can also provide a strong basis to carry out differential proteomics in other physiologically and economically important plastids such as amyloplasts and chromoplasts.

The present work is the first study designed to address the accurate proteomics-based localization of chloroplast proteins with respect to the three major chloroplast compartments: stroma, thylakoids, and envelope. We identified 1323 proteins, thus matching previous results obtained in a comparable high throughput experiment ( 57 ). As a whole, the data currently available indicate that almost 1,700 chloroplast proteins are amenable to proteomics investigations with current MS technologies and conventional fractionation techniques. This only represents one-third to one-half of the total proteins that are estimated to be localized within the chloroplast ( 1 ). As mentioned above, several key proteins are still clearly missing from the chloroplast protein repertoire. To get a more exhaustive repertoire of the Arabidopsis chloroplast, forthcoming generations of mass spectrometers will probably be more sensitive and will help to increase this repertoire. Nevertheless, getting a better overview of the chloroplast proteome will mainly rely on more targeted strategies. First, sample preparation and fractionation of plastid subfractions (outer and inner envelope, stroma, thylakoids, lumen, and plastoglobules) could be improved to gain access to minor chloroplast proteins. Those types of fractions will be worth revisiting with current MS instrumentations and bioinformatics tools. Also, other types of plastids (amyloplasts, chromoplasts, etc.) could be analyzed to identify proteins that are more abundant in those types of plastids ( 98 ). Unfortunately, not all plants species can be used for purifying specialized plastids. Second, MS-targeted approaches can be used to check whether proteins that are suspected to be chloroplast-localized ( e.g. by prediction tools) are actually localized in the chloroplast. To do so, selected reaction monitoring could be used to detect proteins of interest. With the use of internal standards (peptides or proteins), an assessment of the amount of the targeted proteins, within the a given fraction, could also be given ( 99 ). Another interesting extension of our work would be through the use of a “top-down” analytical strategy ( 100 , 101 ) to study full-length envelope proteins from their native environment to access information about protein processing and post-translational modifications.

The present report also represents a unique resource for plastid envelope proteins with respect to both functional and localization issues. Highly curated information on proteins identified in the envelope fraction revealed new insight and ascertained some hypotheses about the different metabolisms associated to the envelope. These data provide unexpected localization information for many chloroplast proteins that were often either not known to reside within the envelope membranes or incorrectly associated to the wrong subplastidial compartment (see for instance supplemental Data 10 ). Forthcoming review articles, targeted to specific chloroplast metabolisms, are required to delve deeper into the wealth of information contained in our experimental data and to compare these results with previously known or suspected subplastidial localization of the corresponding proteins (see for instance Refs. 102 and 103 ). The information derived from the AT-CHLORO database will thus be of valuable interest for the plant community. This is particularly true for the envelope compartment as the envelope membranes are the only membrane structure common to all types of plastids and present at all stages of plastid differentiation ( 19 ).

Finally, the present work also provides the first AMT database dedicated to plant and usable for label-free quantification experiments. Thus, it now becomes possible to identify and quantify thousands of chloroplast peptides via high throughput nano-LC-MS experiments ( 17 , 18 ). Such experiments are currently being carried out to screen the impact of different environmental conditions or of knock-out mutations at the chloroplast level. Moreover, such an AT_CHLORO database is usable by any laboratory having a high resolution mass spectrometer (FT-ICR or Orbitrap) provided that standardization of the retention times is achieved ( 104 ) and that tools dedicated to the AMT strategy are used.
