---
pmid: '25753420'
title: Ascl1 Coordinately Regulates Gene Expression and the Chromatin Landscape during
  Neurogenesis.
authors:
- Raposo AASF
- Vasconcelos FF
- Drechsel D
- Marie C
- Johnston C
- Dolle D
- Bithell A
- Gillotin S
- van den Berg DLC
- Ettwiller L
- Flicek P
- Crawford GE
- Parras CM
- Berninger B
- Buckley NJ
- Guillemot F
- Castro DS
journal: Cell Rep
year: '2015'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC5383937
doi: 10.1016/j.celrep.2015.02.025
---

# Ascl1 Coordinately Regulates Gene Expression and the Chromatin Landscape during Neurogenesis.
**Authors:** Raposo AASF, Vasconcelos FF, Drechsel D, Marie C, Johnston C, Dolle D, Bithell A, Gillotin S, van den Berg DLC, Ettwiller L, Flicek P, Crawford GE, Parras CM, Berninger B, Buckley NJ, Guillemot F, Castro DS
**Journal:** Cell Rep (2015)
**DOI:** [10.1016/j.celrep.2015.02.025](https://doi.org/10.1016/j.celrep.2015.02.025)
**PMC:** [PMC5383937](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5383937/)

## Abstract

1. Cell Rep. 2015 Mar 10;10(9):1544-1556. doi: 10.1016/j.celrep.2015.02.025. Epub
 2015 Mar 5.

Ascl1 Coordinately Regulates Gene Expression and the Chromatin Landscape during 
Neurogenesis.

Raposo AASF(1), Vasconcelos FF(1), Drechsel D(2), Marie C(3), Johnston C(4), 
Dolle D(5), Bithell A(6), Gillotin S(2), van den Berg DLC(2), Ettwiller L(7), 
Flicek P(5), Crawford GE(8), Parras CM(3), Berninger B(9), Buckley NJ(10), 
Guillemot F(2), Castro DS(11).

Author information:
(1)Instituto Gulbenkian de Ciência, 2780-156 Oeiras, Portugal.
(2)MRC National Institute for Medical Research, London NW7 1AA, UK.
(3)Inserm U1127, CNRS UMR 7225, Sorbonne Universités, UPMC University Paris 06, 
UMR-S 1127, Institut du Cerveau et de la Moelle Épinière, ICM, 75013 Paris, 
France.
(4)Centre for the Cellular Basis of Behavior, Institute of Psychiatry, King's 
College London, London SE5 9NU, UK.
(5)European Molecular Biology Laboratory, European Bioinformatics Institute, 
Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SD, UK; Wellcome Trust 
Sanger Institute, Welcome Trust Genome Campus, Cambridge CB10 1SA, UK.
(6)University of Reading, School of Pharmacy, Hopkins Life Sciences Building, 
Reading RG6 6AP, UK.
(7)Centre for Organismal Studies (COS), Ruprecht-Karls-University, 69120 
Heidelberg, Germany.
(8)Institute of Genome Sciences & Policy, Duke University, Durham, NC 27708, 
USA.
(9)Institute of Physiological Chemistry, University Medical Center Johannes 
Gutenberg University Mainz, 55128 Mainz, Germany; Department of Physiological 
Genomics, Institute of Physiology, Ludwig-Maximilians University Munich, 80336 
Munich, Germany.
(10)Department of Psychiatry, University of Oxford, Oxford OX3 7JX, UK.
(11)Instituto Gulbenkian de Ciência, 2780-156 Oeiras, Portugal. Electronic 
address: dscastro@igc.gulbenkian.pt.

The proneural transcription factor Ascl1 coordinates gene expression in both 
proliferating and differentiating progenitors along the neuronal lineage. Here, 
we used a cellular model of neurogenesis to investigate how Ascl1 interacts with 
the chromatin landscape to regulate gene expression when promoting neuronal 
differentiation. We find that Ascl1 binding occurs mostly at distal enhancers 
and is associated with activation of gene transcription. Surprisingly, the 
accessibility of Ascl1 to its binding sites in neural stem/progenitor cells 
remains largely unchanged throughout their differentiation, as Ascl1 targets 
regions of both readily accessible and closed chromatin in proliferating cells. 
Moreover, binding of Ascl1 often precedes an increase in chromatin accessibility 
and the appearance of new regions of open chromatin, associated with de novo 
gene expression during differentiation. Our results reveal a function of Ascl1 
in promoting chromatin accessibility during neurogenesis, linking the chromatin 
landscape at Ascl1 target regions with the temporal progression of its 
transcriptional program.

Copyright © 2015 The Authors. Published by Elsevier Inc. All rights reserved.

DOI: 10.1016/j.celrep.2015.02.025
PMCID: PMC5383937
PMID: 25753420

## Full Text

IntroductionThe generation of neurons in the developing central nervous system requires a number of precisely orchestrated steps, whereby proliferating neural progenitors become committed to the neuronal fate, exit cell cycle, and undergo a long and complex program of migration and differentiation (Kriegstein and Alvarez-Buylla, 2009). Proneural transcription factors (TFs) of the bHLH family, such as Ascl1/Mash1, are the main regulators of neurogenesis in the mammalian brain, and gain and loss-of-function analyses have shown that they are both required and sufficient to promote neurogenesis (Bertrand et al., 2002, Wilkinson et al., 2013). Accordingly, while genetic ablation of proneural genes in mice results in neural developmental defects associated with reduced neurogenesis, overexpression of proneural factors in neural progenitors induces a full neuronal differentiation program (Berninger et al., 2007b, Casarosa et al., 1999, Geoffroy et al., 2009). In addition to its pivotal role in development, Ascl1 has been extensively used in protocols to reprogram somatic cells, including fibroblasts, astrocytes, and pericytes, into induced neurons (Berninger et al., 2007a, Karow et al., 2012, Vierbuchen et al., 2010), renewing interest in understanding the neurogenic activity of this proneural factor.Previously, we characterized the transcriptional program of Ascl1 in the ventral telencephalic region of the embryonic mouse brain by combining gene expression profiling with chromatin immunoprecipitation (ChIP), followed by hybridization to promoter oligonucleotide arrays (ChIP-chip). This work resulted in the identification of a set of Ascl1 target genes with various biological roles at distinct stages of the differentiation program, raising intriguing questions concerning the molecular basis for such temporal pattern (Castro et al., 2011, Vasconcelos and Castro, 2014). In addition, it led to the identification of a novel function for Ascl1 in maintaining cell proliferation, mediated by the direct activation of genes that promote cell cycle progression. This resulted in a model whereby this proneural factor sequentially promotes the proliferation and differentiation of progenitor cells along the neuronal lineage, reconciling the classical view of this proneural protein as a differentiation factor with the fact that it is mostly expressed in cycling progenitors. Moreover, a recent study has shown that these two opposing activities are associated with distinct modes of Ascl1 expression, with oscillating or sustained Ascl1 promoting proliferation or differentiation, respectively (Imayoshi et al., 2013).In spite of the significant progress made on the characterization of its transcriptional targets, little is still known about how Ascl1 regulates gene expression. In particular, the relationship between Ascl1 binding, regulation of the chromatin landscape, and gene transcription is poorly understood. It was recently shown that during neuronal reprogramming, Ascl1 can access its cognate sites in nucleosomal-DNA when ectopically expressed in fibroblasts, defining it as a pioneer TF (Wapinski et al., 2013). However, it remains to be seen whether Ascl1 works as a pioneer factor in a neurogenic context and whether binding of Ascl1 results in alterations to the chromatin landscape at its target regions, as it has been shown for some, but not all, other pioneer TFs (Zaret and Carroll, 2011).Mammalian neurogenesis is not a synchronized process at a cell population level and studies to investigate the mechanistic basis of Ascl1 function at a genome-wide scale are difficult to perform in the developing embryo or in the adult brain. An alternative is the use of adherent cultures of neural stem (NS) cell lines derived from embryonic stem cells or embryonic neural precursors (Conti et al., 2005, Pollard et al., 2006). These cultures provide us with reliable models to study neurogenesis in culture, without the confounding effects of cellular heterogeneity, characteristic of other cellular models such as neurospheres. In proliferating culture conditions, endogenous Ascl1 regulates a progenitor program that functions to maintain cell proliferation (Castro et al., 2011), whereas overexpression of Ascl1 leads to efficient cell cycle exit and neuronal differentiation.Here we investigate how Ascl1 activity is restricted by and impacts the chromatin landscape, when driving neuronal differentiation. We combined expression profiling with genome-wide mapping of Ascl1 binding sites (ChIP-seq) (Park, 2009), and DNase I hypersensitivity sites (DNase-seq) (Song and Crawford, 2010), in a cellular model of neurogenesis driven by overexpressed Ascl1. We identify a large number of genes directly regulated by Ascl1 and characterize widespread changes in chromatin accessibility during differentiation. Ascl1 binding correlates with activation of gene transcription, targeting not only regions of accessible but also of closed chromatin. In addition, binding of Ascl1 to DNA precedes a local increase in chromatin accessibility at the regulatory regions of its target genes, providing the first direct link between Ascl1 regulation of gene expression and local changes in chromatin landscape.

Introduction

The generation of neurons in the developing central nervous system requires a number of precisely orchestrated steps, whereby proliferating neural progenitors become committed to the neuronal fate, exit cell cycle, and undergo a long and complex program of migration and differentiation (Kriegstein and Alvarez-Buylla, 2009). Proneural transcription factors (TFs) of the bHLH family, such as Ascl1/Mash1, are the main regulators of neurogenesis in the mammalian brain, and gain and loss-of-function analyses have shown that they are both required and sufficient to promote neurogenesis (Bertrand et al., 2002, Wilkinson et al., 2013). Accordingly, while genetic ablation of proneural genes in mice results in neural developmental defects associated with reduced neurogenesis, overexpression of proneural factors in neural progenitors induces a full neuronal differentiation program (Berninger et al., 2007b, Casarosa et al., 1999, Geoffroy et al., 2009). In addition to its pivotal role in development, Ascl1 has been extensively used in protocols to reprogram somatic cells, including fibroblasts, astrocytes, and pericytes, into induced neurons (Berninger et al., 2007a, Karow et al., 2012, Vierbuchen et al., 2010), renewing interest in understanding the neurogenic activity of this proneural factor.

Previously, we characterized the transcriptional program of Ascl1 in the ventral telencephalic region of the embryonic mouse brain by combining gene expression profiling with chromatin immunoprecipitation (ChIP), followed by hybridization to promoter oligonucleotide arrays (ChIP-chip). This work resulted in the identification of a set of Ascl1 target genes with various biological roles at distinct stages of the differentiation program, raising intriguing questions concerning the molecular basis for such temporal pattern (Castro et al., 2011, Vasconcelos and Castro, 2014). In addition, it led to the identification of a novel function for Ascl1 in maintaining cell proliferation, mediated by the direct activation of genes that promote cell cycle progression. This resulted in a model whereby this proneural factor sequentially promotes the proliferation and differentiation of progenitor cells along the neuronal lineage, reconciling the classical view of this proneural protein as a differentiation factor with the fact that it is mostly expressed in cycling progenitors. Moreover, a recent study has shown that these two opposing activities are associated with distinct modes of Ascl1 expression, with oscillating or sustained Ascl1 promoting proliferation or differentiation, respectively (Imayoshi et al., 2013).

In spite of the significant progress made on the characterization of its transcriptional targets, little is still known about how Ascl1 regulates gene expression. In particular, the relationship between Ascl1 binding, regulation of the chromatin landscape, and gene transcription is poorly understood. It was recently shown that during neuronal reprogramming, Ascl1 can access its cognate sites in nucleosomal-DNA when ectopically expressed in fibroblasts, defining it as a pioneer TF (Wapinski et al., 2013). However, it remains to be seen whether Ascl1 works as a pioneer factor in a neurogenic context and whether binding of Ascl1 results in alterations to the chromatin landscape at its target regions, as it has been shown for some, but not all, other pioneer TFs (Zaret and Carroll, 2011).

Mammalian neurogenesis is not a synchronized process at a cell population level and studies to investigate the mechanistic basis of Ascl1 function at a genome-wide scale are difficult to perform in the developing embryo or in the adult brain. An alternative is the use of adherent cultures of neural stem (NS) cell lines derived from embryonic stem cells or embryonic neural precursors (Conti et al., 2005, Pollard et al., 2006). These cultures provide us with reliable models to study neurogenesis in culture, without the confounding effects of cellular heterogeneity, characteristic of other cellular models such as neurospheres. In proliferating culture conditions, endogenous Ascl1 regulates a progenitor program that functions to maintain cell proliferation (Castro et al., 2011), whereas overexpression of Ascl1 leads to efficient cell cycle exit and neuronal differentiation.

Here we investigate how Ascl1 activity is restricted by and impacts the chromatin landscape, when driving neuronal differentiation. We combined expression profiling with genome-wide mapping of Ascl1 binding sites (ChIP-seq) (Park, 2009), and DNase I hypersensitivity sites (DNase-seq) (Song and Crawford, 2010), in a cellular model of neurogenesis driven by overexpressed Ascl1. We identify a large number of genes directly regulated by Ascl1 and characterize widespread changes in chromatin accessibility during differentiation. Ascl1 binding correlates with activation of gene transcription, targeting not only regions of accessible but also of closed chromatin. In addition, binding of Ascl1 to DNA precedes a local increase in chromatin accessibility at the regulatory regions of its target genes, providing the first direct link between Ascl1 regulation of gene expression and local changes in chromatin landscape.

ResultsA Cellular Model of Ascl1-Driven NeurogenesisOverexpression of Ascl1 promotes cell cycle exit and neuronal differentiation of neural progenitor cells (Castro et al., 2006). To study this process in controlled conditions, we established a cellular model of Ascl1-driven neurogenesis by expressing an inducible version of Ascl1 in NS cells in culture (NS5 cell line) (Pollard et al., 2006). Fusion of full-length Ascl1 to the modified ligand binding domain of the estrogen receptor (ERT2) renders Ascl1 activity dependent on the presence of 4-hydroxytamoxifen (herein referred to as tamoxifen) (Bergstrom et al., 2002, Burk and Klempnauer, 1991, Littlewood et al., 1995). In a transcriptional assay in transfected NS cells, Ascl1-ERT2 induces the transcriptional activation of an enhancer of the Ascl1 target gene Dll1 (Castro et al., 2006) in an inducible manner to levels that are similar to its WT counterpart (Figure S1A). In order to test for the ability of the inducible Ascl1 protein to promote neuronal differentiation, we transduced NS cells in culture with a retrovirus vector co-expressing Ascl1-ERT2 and green fluorescent protein (GFP). In the vast majority of transduced cells, drastic morphological changes characterized by the extension of long processes and associated with the expression of the neuronal marker Tuj1 were observed only upon the addition of tamoxifen, confirming the ability of Ascl1-ERT2 to activate the neurogenic differentiation program in an inducible manner (Figure S1B). Although activation is associated with the nuclear translocation of a small fraction of Ascl1-ERT2, most of the protein is already nuclear prior to addition of tamoxifen (as previously reported with other cases of TFs fused to ERT2) (Burk and Klempnauer, 1991, Roemer and Friedmann, 1993), and additional mechanisms must therefore contribute to the inducibility of Ascl1-ERT2 (Figure 1A).To obtain large numbers of synchronized differentiating neural progenitors, we used antibiotic selection following the retroviral delivery of the Ascl1-ERT2 transgene. Quantification of immunofluorescence levels upon immunohistochemistry estimated the total Ascl1 protein level in transduced cells to be 8- to 9-fold higher when compared with endogenous Ascl1 expressed in embryonic ventral telencephalic progenitors (Figure 1B). NS cells expressing Ascl1-ERT2 plated at low density differentiate with great efficiency upon addition of tamoxifen (Figure 1C). Four days after induction, immunocytochemical analysis shows expression of the rate limiting enzyme for GABA synthesis glutamic acid decarboxylase GAD65/67 (15.1% ± 0.3%) (Figure 1D). Whole-cell patch-clamp recordings of retrovirus-transduced cells 14 days after onset of tamoxifen treatment show that these cells consistently exhibit electrophysiological properties of neurons (Figure 1E). Moreover, their action potential discharges pattern in response to step current injections is highly reminiscent of low-threshold burst spiking interneurons and very similar to those generated from medial ganglionic eminence progenitors of the ventral telencephalon (Figure 1D) (Martínez-Cerdeño et al., 2010). To characterize the transcriptome changes in our model, we performed expression profiling at various time points −4, 12, 24, and 50 hr after the onset of differentiation. A large number of genes (1,508) are differentially regulated at least at one time point (fold change >1.5, p < 10−3; see Figure 1F for a breakdown of numbers), as expected upon induction of a cellular differentiation program (Table S1). Functional annotation of differentially expressed genes by Gene Ontology shows considerable enrichment in terms associated with distinct phases of neurogenesis, describing both early events (e.g., “Notch signaling,” “cell division”) and later steps of the differentiation program (e.g., “cell migration,” “axon guidance”) (Figure 1G). Overall, these results show Ascl1 drives an extended program of neuronal differentiation in our model.Genome-wide Location Analysis of Ascl1 in Differentiating NS CellsTo understand how the observed changes in gene expression relate to Ascl1 function, we mapped Ascl1 binding sites genome-wide by ChIP followed by massive parallel sequencing (ChIP-seq) (Wapinski et al., 2013), using an antibody against the HA tag of Ascl1-ERT2. This was performed 18 hr after the onset of differentiation, at a time point when NS cells have already undergone large transcriptional changes. Using input chromatin as control, we defined an extended list comprising 21,582 Ascl1 binding sites, with FDR <0.5% and p < 10−10 (Table S2).In order to validate binding events (BEs) mediated by the Ascl1-ERT2 protein, we used ChIP-PCR against WT Ascl1, with chromatin extracted from embryonic E14.5 ventral telencephalic progenitors, or WT NS5 cells (Figure S2A). We reasoned that most BEs in differentiating NS cells should be identified in at least one of the chromatin samples. Indeed, this is the case for all ChIP-seq peaks associated with p < 10−18 (with the majority of BEs validated in both chromatins), indicating a good match between Ascl1-ERT2 and WT Ascl1. Most of the 11,782 Ascl1 BEs defined by this high-confidence list (p < 10−18) are located within intergenic regions (63%) and at long distances from the nearest identified transcription start site (TSS). Less than one third of Ascl1 binding occurs inside genes (30%) or their promoter regions (7%) (Figures 2A and 2B), suggesting that Ascl1 binds predominantly to distal enhancer regions. We next used a Hidden-Markov model to characterize the chromatin states at Ascl1-bound regions, using genome-wide profiles of histone modifications generated from proliferating and differentiating NS cells (before or 24 hr after addition of tamoxifen, respectively) (Ernst and Kellis, 2012). Ascl1 BEs fall mostly within regions of chromatin highly co-enriched for H3K4me1 and H3K27ac, characteristic of active enhancers (same result for p < 10−10; data not shown). Moreover, the same association with this chromatin state is found in proliferating NS cells, indicating that during differentiation Ascl1 binds predominantly to regions that are already marked as active enhancers in proliferating NS cells (Figures 2C, 2D, and S2B).To determine the DNA sequence mediating Ascl1 binding, we searched for motifs enriched within the 20 nucleotides region centered under the Ascl1 peak summits. A strong preference is found for the hexamer sequence CAGCTG, corresponding to the E-box type sequence previously associated with Ascl1 binding (Castro et al., 2011) and in agreement with its role in mediating direct DNA binding (Figure 2E).Ascl1 Functions Globally as a Transcriptional ActivatorAlthough it is generally believed that proneural factors function by activating gene transcription, some reports have challenged this view (Alvarez-Rodríguez and Pons, 2009, Rheinbay et al., 2013). In order to investigate this issue, we started by performing an in depth cross-comparison between the location analysis and expression profiling results, annotating each Ascl1 BE to its nearest TSS. Direct comparison with the set of deregulated genes previously described shows that only a small fraction of BEs are considered to be regulatory using nearest gene annotation (Figure 3A). Nonetheless, we find a positive correlation between peak height (which is proportional to significance of binding, or p value) and its potential to upregulate gene expression (top peaks ∼20%). In contrast, the fraction of regulatory peaks that can be associated to repressed genes is invariant with changes in peak size, suggesting no direct relation between Ascl1 binding significance and downregulation of gene expression. Therefore, mere binding of Ascl1 to chromatin does not predict a regulatory event; however, the size of an Ascl1 peak may be correlated with its potential to activate gene transcription.To have a global view of the association between the data sets of genes bound by Ascl1 and regulated during differentiation, we determined the frequency at which deregulated genes (grouped in bins of increasing thresholds of fold change of expression) are associated with Ascl1 BEs (grouped in bins of increasing p value cutoff) and therefore considered to be directly regulated by Ascl1 (Figure 3B, left). The statistical significance of the overlaps was assessed by a similar comparison against 100 randomly generated control ChIP-seq data sets of identical size (Figure 3B, right). The resulting heat maps indicate a strong association between Ascl1-bound genes and those upregulated during differentiation and a much less pronounced association with downregulated genes (Figure 3B, top left versus bottom left). Moreover, the fraction of downregulated genes that are considered Ascl1 targets is very similar to that expected by chance (Figure 3B, left versus right). Altogether, the above analysis indicates that Ascl1 functions globally as a transcriptional activator. In addition, this function is independent of the progress of differentiation during the first 50 hr, as shown with similar analyses for the different time points considered in the transcription profiling (Figures 3C and S3). Importantly, restricting the analysis to the Ascl1 BEs located within ±5 kb of a TSS, producing a more stringent gene annotation, results in a similar conclusion (Figure S4).To integrate the transcriptional profiling data sets collected at distinct time points, we applied a fuzzy c-means clustering algorithm, resulting in the identification of seven distinct temporal clusters (clusters 1–4 comprised of upregulated genes, clusters 5–7 of downregulated genes) (Figure S5; Table S3). Clusters of genes that are upregulated during differentiation are all strongly associated with Ascl1 BEs (p ≤ 1.3 × 10−11, 8.0 × 10−126, 1.1 × 10−22, and 7.9 × 10−10 for clusters 1, 2, 3, and 4, respectively), whereas clusters of downregulated genes show no such association (Figure 3D), further indicating that Ascl1 binding correlates with transcriptional activation.Characterization of Ascl1 Transcriptional ProgramThe relative size of each cluster (Figure S5B) and their enrichment for BEs indicate that most Ascl1 target genes fall within a cluster characterized by an early activation profile (cluster 2), while a smaller but significant number is upregulated at mid or late time points (clusters 3 and 4, respectively). We then compiled a high-confidence list of 272 Ascl1 direct targets comprising genes from clusters 2, 3, and 4 that are associated with at least one Ascl1 BE with high regulatory potential (p < 10−25) (Table S4). A search for terms describing “Biological Process,” “Molecular Function,” and “Pathways” (Panther Classification System) revealed the segregation in each temporal cluster of Ascl1 target genes encoding proteins of distinct classes and functions, suggesting the control of various subprograms at distinct developmental stages (Figure S5C). Genes with the earliest onset of activation (belonging to cluster 2) are associated with signal transduction and cell communication, while targets with a mid-onset of activation (cluster 3) encode mostly proteins with a function related to transcription or Notch signaling. Genes with latest onset (cluster 4) are enriched for generic terms associated with neural development and relate to late events in the differentiation program, from cell migration to axonal growth and guidance (Figure S5C). In summary and in agreement with our previous study performed on a smaller scale, the identification of Ascl1 target genes in our neurogenesis model demonstrates the direct control of distinct components of the neurogenic differentiation program by Ascl1.Ascl1 Access to Its Target Sites Does Not Depend on the Differentiation Stage of NS CellsIt was previously shown that Ascl1 target genes display different onsets of expression at distinct stages of the neuronal lineage (Borromeo et al., 2014, Castro et al., 2011). Such temporal patterning could be associated with distinct accessibilities of Ascl1 to its binding sites (e.g., due to differences in the chromatin landscape). To investigate this, we compared the binding profile of overexpressed Ascl1 at two distinct stages by extending the previous ChIP-seq analysis to a time point at the onset of differentiation (30 min upon addition of tamoxifen) (Table S9). Surprisingly, visual inspection of the genomic distribution of Ascl1 peaks finds a remarkable similarity between Ascl1 BEs at t = 30 min and t = 18 hr (Figure 4A). Comparison of the two Ascl1 binding profiles in a bin-by-bin approach within the confidence limits defined by the previous ChIP-PCR validation (Figure 4B, dashed line) shows an overlap of 86% between the two stages (Figure 4B). Furthermore, the presence of normalized sequencing signal across both samples in genomic regions centered at putative sample-specific BEs suggests that the overlap of occupancies may have been underestimated by peak calling conditions (Figure 4C). Overall, our results indicate that the accessibility of Ascl1 to the full complement of its binding sites remains largely identical throughout differentiation.Characterization of Chromatin Landscape Changes during DifferentiationWe next asked what impact Ascl1 may have on the chromatin landscape when it promotes neuronal differentiation of NS cells. We started by characterizing the chromatin landscape of proliferating NS cells and of NS cells undergoing differentiation by Ascl1, using a DNase I hypersensitivity assay coupled to massive parallel sequencing (DNase-seq). This technique identifies regions of decreased nucleosome occupancy (herein referred as “open chromatin”) on a genome-wide scale, which correspond mostly to active regulatory elements such as promoters, enhancers, insulators, and silencers (Boyle et al., 2008, Natarajan et al., 2012, Thurman et al., 2012). The density of mapped reads for each genome position was computed to generate a comprehensive list of DNase I hypersensitivity sites (DHSs). Using a constant threshold of p < 10−5, we identified ∼87,000 and ∼94,000 DHSs in proliferating and differentiating NS cells, respectively (Tables S5 and S6), numbers with a magnitude consistent with those obtained in other cell types (Song et al., 2011). Although the majority of these sites is shared by both experimental conditions, each cell state exhibits a specific set of ∼20,000 DHSs (Figure 5A).Since Ascl1 functions as a transcriptional activator, we focused on the DHSs induced during differentiation (Table S7), as these are more likely to provide new insights into Ascl1 function. To validate these DHSs, we used formaldehyde-assisted isolation of regulatory elements (FAIRE), an alternative method to identify regions of open chromatin. Although distinct preferences have been described for each technique in the identification of distal or proximal regulatory regions, the two techniques have been shown to yield largely overlapping results on a genome-wide basis (Song et al., 2011). Indeed, 9 of 13 tested regions presented more than 2-fold enrichment by FAIRE-PCR across samples collected before and 24 hr after addition of tamoxifen, as opposed to control regions (Figure 5B).Induced DHSs Are Associated with Genes Expressed De Novo during DifferentiationNext, we investigated how regions of chromatin newly opened during differentiation may be associated with the observed changes in gene expression. We find a statistically significant enrichment of differentiation-induced DHSs in the vicinity of upregulated genes (the sum of clusters 1–4; p < 1.3 × 10−29), in sharp contrast with downregulated genes (clusters 5–7) (Figure 5C). Moreover, a large fraction of all upregulated genes (413 of 760) is associated with at least one differentiation-induced DHS, suggesting the importance of these putative regulatory regions in activating gene expression during neurogenesis. Notably, upregulated genes associated with differentiation-induced DHSs are either not expressed in proliferating NS cells or expressed at a low level when compared with upregulated genes near constant DHSs (Figure 5D). This is well exemplified by the induction of NeuroD4, a pro-differentiation TF only expressed in post-mitotic precursors (Ohsawa et al., 2005), and which is associated with newly open DHSs (Figure 6B). Overall, our results indicate that activation of differentiation genes is strongly associated with the appearance of new DHSs.Ascl1 Promotes Chromatin AccessibilityIn order to investigate which DNA-binding TFs may be associated with the appearance of newly open chromatin regions, we took advantage of the high resolution potential of Digital Genomic Footprinting (DGF) applied to DNase-seq data to determine the position of sites occupied by TFs within differentiation-induced DHSs (Figure 5E) (Piper et al., 2013). DGF identified almost 17,000 genomic sites (Table S8) with significant occupancy levels (p < 10−10), and the corresponding sequences were then subject to a search for frequency of motif occurrence. The most abundant motif found corresponds to the consensus sequence of CTCF and is likely to reflect its frequent binding to insulator regions identified by DNase-seq (Figure 5E) (Thurman et al., 2012). Three additional motifs reveal binding by TFs of the bHLH or NFI families, with the most frequent one corresponding to the E-box consensus sequence, which mediates Ascl1 binding. This observation establishes Ascl1 as a prime candidate to promote the opening of chromatin structure and induce new DHSs during differentiation.To understand how Ascl1 binding may relate to the observed changes in chromatin compaction, we compared the genomic distribution of Ascl1 BEs with that of DHSs before and after neuronal differentiation (Figure 5F). Ascl1 ChIP-seq and DNase-seq data sets show a high degree of overlap (e.g., 80.3% of Ascl1 BEs with p < 10−18 at t = 30 min fall within DHSs in proliferating cells), indicating that a large fraction of BEs, but not all, occur in regions of open chromatin in proliferating NS cells (Figure 5F). Strikingly, this overlap increases significantly under differentiation conditions (to 91.05% after differentiation, in the aforementioned case; Figure 5F). Altogether, a comparison between the Ascl1 ChIP-seq and DNase-seq suggests that (1) the Ascl1 binding profile remains constant at the two stages analyzed, (2) a large fraction of Ascl1 BEs occurs in regions of open chromatin in proliferating cells, with a subset falling within closed chromatin, and (3) binding of Ascl1 to closed chromatin precedes the appearance of new DHSs during differentiation.Visualization of aligned Ascl1 ChIP-seq and DNase-seq data suggests that many differentiation-induced DHSs overlap with Ascl1 BEs (Figure 6A), as exemplified by those in the vicinity of genes activated during differentiation such as NeuroD4, Ap3b2, Mcf2l, and Nrxn3 (Figure 6B). Validation of changes in chromatin compaction at the selected Ascl1 binding sites was performed by FAIRE-PCR using chromatin extracted from NS5 cells expressing full-length Ascl1 under the regulation of a doxycycline inducible promoter, before and after Ascl1 induction (Figure 6C). Although not all DHSs that arise de novo during differentiation are associated with an Ascl1 BE, as not all Ascl1 BEs overlap with changes detected by DNase-seq, the vast majority of Ascl1 BEs fall within DHSs in differentiating cells (91.05% or 91.27% with p < 10−18, at t = 30 min or t = 18 hr, respectively; Figure 5F).To investigate the correlation between Ascl1 binding and changes in chromatin compaction on a large scale, we quantified the DNase-seq signal from both time points centered on Ascl1 peak summits. Confirming previous observations, the overall DNase-seq profile at Ascl1 peak summits is consistent with many BEs occurring in regions of already open chromatin at t = 30 min (Figure 6D). Nevertheless, a significant increase is observed when comparing profiles before and after differentiation (2.33 increase of median read count) at Ascl1 peak summits genome-wide (Figure 6D). This increase is amplified when focusing our analysis on Ascl1 peaks that fall within differentiation-induced DHSs (4.37 increase of median read count; Figure 6D). Ascl1 BEs co-localizing with differentiation-induced DHSs are strongly associated with gene activation and not with gene repression (p < 6.8 × 10−32; Figure 6E), targeting a total number of 98 genes during differentiation. Overall, the integration of Ascl1 location analysis, DNase-seq results, and expression profiling demonstrate Ascl1 promotes genome-wide changes in chromatin compaction at its target sites during neuronal differentiation and the appearance of new DHSs associated with activation of gene expression.Our results are in agreement with the observation that Ascl1 can bind to nucleosomal DNA when overexpressed in fibroblasts (Wapinski et al., 2013). To verify that Ascl1 can bind to closed chromatin in a physiological context, we focused on the Ascl1-bound regulatory region identified immediately downstream the NeuroD4 promoter (BS2 in Figure 6A), a target gene of Ascl1 in our cellular system that is not expressed in ventral telencephalon. We reasoned that binding to closed chromatin to NeuroD4 may still occur in this embryonic brain region at the neurogenic period, even in the absence of gene expression. Indeed, we detected strong binding of Ascl1, as compared with a negative control region within the NeuroD4 locus by ChIP-PCR in chromatin extracted from mouse ventral telencephalon (Figure 6F). However, in contrast to our previous findings with differentiating NS cells in culture, a lack of enrichment for non-nucleosomal DNA assessed by FAIRE-PCR shows this Ascl1 target site is found within closed chromatin in ventral telencephalon (Figure 6G, compare with Ascl1 site in Dll1). Thus, our results with the NeuroD4 regulatory region demonstrate that endogenous Ascl1 can bind to closed chromatin in an in vivo context.Finally, we wished to analyze the accessibility of chromatin at Ascl1 target sites in the absence of Ascl1 expression. For this, we used chromatin extracted from ventral telencephalon of Ascl1-null embryos and probed Ascl1-bound regions associated with previously validated Ascl1 target genes (Dll1, Fbxw7, and Stk33) in this embryonic structure (Castro et al., 2006). Ascl1 null embryos show a very significant reduction of FAIRE-PCR signal relative to WT chromatin at all tested Ascl1 binding sites, as compared with pairwise control regions, which is consistent with the results above and in support of a role for Ascl1 in regulating chromatin compaction in vivo (Figure S6).

A Cellular Model of Ascl1-Driven NeurogenesisOverexpression of Ascl1 promotes cell cycle exit and neuronal differentiation of neural progenitor cells (Castro et al., 2006). To study this process in controlled conditions, we established a cellular model of Ascl1-driven neurogenesis by expressing an inducible version of Ascl1 in NS cells in culture (NS5 cell line) (Pollard et al., 2006). Fusion of full-length Ascl1 to the modified ligand binding domain of the estrogen receptor (ERT2) renders Ascl1 activity dependent on the presence of 4-hydroxytamoxifen (herein referred to as tamoxifen) (Bergstrom et al., 2002, Burk and Klempnauer, 1991, Littlewood et al., 1995). In a transcriptional assay in transfected NS cells, Ascl1-ERT2 induces the transcriptional activation of an enhancer of the Ascl1 target gene Dll1 (Castro et al., 2006) in an inducible manner to levels that are similar to its WT counterpart (Figure S1A). In order to test for the ability of the inducible Ascl1 protein to promote neuronal differentiation, we transduced NS cells in culture with a retrovirus vector co-expressing Ascl1-ERT2 and green fluorescent protein (GFP). In the vast majority of transduced cells, drastic morphological changes characterized by the extension of long processes and associated with the expression of the neuronal marker Tuj1 were observed only upon the addition of tamoxifen, confirming the ability of Ascl1-ERT2 to activate the neurogenic differentiation program in an inducible manner (Figure S1B). Although activation is associated with the nuclear translocation of a small fraction of Ascl1-ERT2, most of the protein is already nuclear prior to addition of tamoxifen (as previously reported with other cases of TFs fused to ERT2) (Burk and Klempnauer, 1991, Roemer and Friedmann, 1993), and additional mechanisms must therefore contribute to the inducibility of Ascl1-ERT2 (Figure 1A).To obtain large numbers of synchronized differentiating neural progenitors, we used antibiotic selection following the retroviral delivery of the Ascl1-ERT2 transgene. Quantification of immunofluorescence levels upon immunohistochemistry estimated the total Ascl1 protein level in transduced cells to be 8- to 9-fold higher when compared with endogenous Ascl1 expressed in embryonic ventral telencephalic progenitors (Figure 1B). NS cells expressing Ascl1-ERT2 plated at low density differentiate with great efficiency upon addition of tamoxifen (Figure 1C). Four days after induction, immunocytochemical analysis shows expression of the rate limiting enzyme for GABA synthesis glutamic acid decarboxylase GAD65/67 (15.1% ± 0.3%) (Figure 1D). Whole-cell patch-clamp recordings of retrovirus-transduced cells 14 days after onset of tamoxifen treatment show that these cells consistently exhibit electrophysiological properties of neurons (Figure 1E). Moreover, their action potential discharges pattern in response to step current injections is highly reminiscent of low-threshold burst spiking interneurons and very similar to those generated from medial ganglionic eminence progenitors of the ventral telencephalon (Figure 1D) (Martínez-Cerdeño et al., 2010). To characterize the transcriptome changes in our model, we performed expression profiling at various time points −4, 12, 24, and 50 hr after the onset of differentiation. A large number of genes (1,508) are differentially regulated at least at one time point (fold change >1.5, p < 10−3; see Figure 1F for a breakdown of numbers), as expected upon induction of a cellular differentiation program (Table S1). Functional annotation of differentially expressed genes by Gene Ontology shows considerable enrichment in terms associated with distinct phases of neurogenesis, describing both early events (e.g., “Notch signaling,” “cell division”) and later steps of the differentiation program (e.g., “cell migration,” “axon guidance”) (Figure 1G). Overall, these results show Ascl1 drives an extended program of neuronal differentiation in our model.

A Cellular Model of Ascl1-Driven Neurogenesis

Overexpression of Ascl1 promotes cell cycle exit and neuronal differentiation of neural progenitor cells (Castro et al., 2006). To study this process in controlled conditions, we established a cellular model of Ascl1-driven neurogenesis by expressing an inducible version of Ascl1 in NS cells in culture (NS5 cell line) (Pollard et al., 2006). Fusion of full-length Ascl1 to the modified ligand binding domain of the estrogen receptor (ERT2) renders Ascl1 activity dependent on the presence of 4-hydroxytamoxifen (herein referred to as tamoxifen) (Bergstrom et al., 2002, Burk and Klempnauer, 1991, Littlewood et al., 1995). In a transcriptional assay in transfected NS cells, Ascl1-ERT2 induces the transcriptional activation of an enhancer of the Ascl1 target gene Dll1 (Castro et al., 2006) in an inducible manner to levels that are similar to its WT counterpart (Figure S1A). In order to test for the ability of the inducible Ascl1 protein to promote neuronal differentiation, we transduced NS cells in culture with a retrovirus vector co-expressing Ascl1-ERT2 and green fluorescent protein (GFP). In the vast majority of transduced cells, drastic morphological changes characterized by the extension of long processes and associated with the expression of the neuronal marker Tuj1 were observed only upon the addition of tamoxifen, confirming the ability of Ascl1-ERT2 to activate the neurogenic differentiation program in an inducible manner (Figure S1B). Although activation is associated with the nuclear translocation of a small fraction of Ascl1-ERT2, most of the protein is already nuclear prior to addition of tamoxifen (as previously reported with other cases of TFs fused to ERT2) (Burk and Klempnauer, 1991, Roemer and Friedmann, 1993), and additional mechanisms must therefore contribute to the inducibility of Ascl1-ERT2 (Figure 1A).

To obtain large numbers of synchronized differentiating neural progenitors, we used antibiotic selection following the retroviral delivery of the Ascl1-ERT2 transgene. Quantification of immunofluorescence levels upon immunohistochemistry estimated the total Ascl1 protein level in transduced cells to be 8- to 9-fold higher when compared with endogenous Ascl1 expressed in embryonic ventral telencephalic progenitors (Figure 1B). NS cells expressing Ascl1-ERT2 plated at low density differentiate with great efficiency upon addition of tamoxifen (Figure 1C). Four days after induction, immunocytochemical analysis shows expression of the rate limiting enzyme for GABA synthesis glutamic acid decarboxylase GAD65/67 (15.1% ± 0.3%) (Figure 1D). Whole-cell patch-clamp recordings of retrovirus-transduced cells 14 days after onset of tamoxifen treatment show that these cells consistently exhibit electrophysiological properties of neurons (Figure 1E). Moreover, their action potential discharges pattern in response to step current injections is highly reminiscent of low-threshold burst spiking interneurons and very similar to those generated from medial ganglionic eminence progenitors of the ventral telencephalon (Figure 1D) (Martínez-Cerdeño et al., 2010). To characterize the transcriptome changes in our model, we performed expression profiling at various time points −4, 12, 24, and 50 hr after the onset of differentiation. A large number of genes (1,508) are differentially regulated at least at one time point (fold change >1.5, p < 10−3; see Figure 1F for a breakdown of numbers), as expected upon induction of a cellular differentiation program (Table S1). Functional annotation of differentially expressed genes by Gene Ontology shows considerable enrichment in terms associated with distinct phases of neurogenesis, describing both early events (e.g., “Notch signaling,” “cell division”) and later steps of the differentiation program (e.g., “cell migration,” “axon guidance”) (Figure 1G). Overall, these results show Ascl1 drives an extended program of neuronal differentiation in our model.

Genome-wide Location Analysis of Ascl1 in Differentiating NS CellsTo understand how the observed changes in gene expression relate to Ascl1 function, we mapped Ascl1 binding sites genome-wide by ChIP followed by massive parallel sequencing (ChIP-seq) (Wapinski et al., 2013), using an antibody against the HA tag of Ascl1-ERT2. This was performed 18 hr after the onset of differentiation, at a time point when NS cells have already undergone large transcriptional changes. Using input chromatin as control, we defined an extended list comprising 21,582 Ascl1 binding sites, with FDR <0.5% and p < 10−10 (Table S2).In order to validate binding events (BEs) mediated by the Ascl1-ERT2 protein, we used ChIP-PCR against WT Ascl1, with chromatin extracted from embryonic E14.5 ventral telencephalic progenitors, or WT NS5 cells (Figure S2A). We reasoned that most BEs in differentiating NS cells should be identified in at least one of the chromatin samples. Indeed, this is the case for all ChIP-seq peaks associated with p < 10−18 (with the majority of BEs validated in both chromatins), indicating a good match between Ascl1-ERT2 and WT Ascl1. Most of the 11,782 Ascl1 BEs defined by this high-confidence list (p < 10−18) are located within intergenic regions (63%) and at long distances from the nearest identified transcription start site (TSS). Less than one third of Ascl1 binding occurs inside genes (30%) or their promoter regions (7%) (Figures 2A and 2B), suggesting that Ascl1 binds predominantly to distal enhancer regions. We next used a Hidden-Markov model to characterize the chromatin states at Ascl1-bound regions, using genome-wide profiles of histone modifications generated from proliferating and differentiating NS cells (before or 24 hr after addition of tamoxifen, respectively) (Ernst and Kellis, 2012). Ascl1 BEs fall mostly within regions of chromatin highly co-enriched for H3K4me1 and H3K27ac, characteristic of active enhancers (same result for p < 10−10; data not shown). Moreover, the same association with this chromatin state is found in proliferating NS cells, indicating that during differentiation Ascl1 binds predominantly to regions that are already marked as active enhancers in proliferating NS cells (Figures 2C, 2D, and S2B).To determine the DNA sequence mediating Ascl1 binding, we searched for motifs enriched within the 20 nucleotides region centered under the Ascl1 peak summits. A strong preference is found for the hexamer sequence CAGCTG, corresponding to the E-box type sequence previously associated with Ascl1 binding (Castro et al., 2011) and in agreement with its role in mediating direct DNA binding (Figure 2E).

Genome-wide Location Analysis of Ascl1 in Differentiating NS Cells

To understand how the observed changes in gene expression relate to Ascl1 function, we mapped Ascl1 binding sites genome-wide by ChIP followed by massive parallel sequencing (ChIP-seq) (Wapinski et al., 2013), using an antibody against the HA tag of Ascl1-ERT2. This was performed 18 hr after the onset of differentiation, at a time point when NS cells have already undergone large transcriptional changes. Using input chromatin as control, we defined an extended list comprising 21,582 Ascl1 binding sites, with FDR <0.5% and p < 10−10 (Table S2).

In order to validate binding events (BEs) mediated by the Ascl1-ERT2 protein, we used ChIP-PCR against WT Ascl1, with chromatin extracted from embryonic E14.5 ventral telencephalic progenitors, or WT NS5 cells (Figure S2A). We reasoned that most BEs in differentiating NS cells should be identified in at least one of the chromatin samples. Indeed, this is the case for all ChIP-seq peaks associated with p < 10−18 (with the majority of BEs validated in both chromatins), indicating a good match between Ascl1-ERT2 and WT Ascl1. Most of the 11,782 Ascl1 BEs defined by this high-confidence list (p < 10−18) are located within intergenic regions (63%) and at long distances from the nearest identified transcription start site (TSS). Less than one third of Ascl1 binding occurs inside genes (30%) or their promoter regions (7%) (Figures 2A and 2B), suggesting that Ascl1 binds predominantly to distal enhancer regions. We next used a Hidden-Markov model to characterize the chromatin states at Ascl1-bound regions, using genome-wide profiles of histone modifications generated from proliferating and differentiating NS cells (before or 24 hr after addition of tamoxifen, respectively) (Ernst and Kellis, 2012). Ascl1 BEs fall mostly within regions of chromatin highly co-enriched for H3K4me1 and H3K27ac, characteristic of active enhancers (same result for p < 10−10; data not shown). Moreover, the same association with this chromatin state is found in proliferating NS cells, indicating that during differentiation Ascl1 binds predominantly to regions that are already marked as active enhancers in proliferating NS cells (Figures 2C, 2D, and S2B).

To determine the DNA sequence mediating Ascl1 binding, we searched for motifs enriched within the 20 nucleotides region centered under the Ascl1 peak summits. A strong preference is found for the hexamer sequence CAGCTG, corresponding to the E-box type sequence previously associated with Ascl1 binding (Castro et al., 2011) and in agreement with its role in mediating direct DNA binding (Figure 2E).

Ascl1 Functions Globally as a Transcriptional ActivatorAlthough it is generally believed that proneural factors function by activating gene transcription, some reports have challenged this view (Alvarez-Rodríguez and Pons, 2009, Rheinbay et al., 2013). In order to investigate this issue, we started by performing an in depth cross-comparison between the location analysis and expression profiling results, annotating each Ascl1 BE to its nearest TSS. Direct comparison with the set of deregulated genes previously described shows that only a small fraction of BEs are considered to be regulatory using nearest gene annotation (Figure 3A). Nonetheless, we find a positive correlation between peak height (which is proportional to significance of binding, or p value) and its potential to upregulate gene expression (top peaks ∼20%). In contrast, the fraction of regulatory peaks that can be associated to repressed genes is invariant with changes in peak size, suggesting no direct relation between Ascl1 binding significance and downregulation of gene expression. Therefore, mere binding of Ascl1 to chromatin does not predict a regulatory event; however, the size of an Ascl1 peak may be correlated with its potential to activate gene transcription.To have a global view of the association between the data sets of genes bound by Ascl1 and regulated during differentiation, we determined the frequency at which deregulated genes (grouped in bins of increasing thresholds of fold change of expression) are associated with Ascl1 BEs (grouped in bins of increasing p value cutoff) and therefore considered to be directly regulated by Ascl1 (Figure 3B, left). The statistical significance of the overlaps was assessed by a similar comparison against 100 randomly generated control ChIP-seq data sets of identical size (Figure 3B, right). The resulting heat maps indicate a strong association between Ascl1-bound genes and those upregulated during differentiation and a much less pronounced association with downregulated genes (Figure 3B, top left versus bottom left). Moreover, the fraction of downregulated genes that are considered Ascl1 targets is very similar to that expected by chance (Figure 3B, left versus right). Altogether, the above analysis indicates that Ascl1 functions globally as a transcriptional activator. In addition, this function is independent of the progress of differentiation during the first 50 hr, as shown with similar analyses for the different time points considered in the transcription profiling (Figures 3C and S3). Importantly, restricting the analysis to the Ascl1 BEs located within ±5 kb of a TSS, producing a more stringent gene annotation, results in a similar conclusion (Figure S4).To integrate the transcriptional profiling data sets collected at distinct time points, we applied a fuzzy c-means clustering algorithm, resulting in the identification of seven distinct temporal clusters (clusters 1–4 comprised of upregulated genes, clusters 5–7 of downregulated genes) (Figure S5; Table S3). Clusters of genes that are upregulated during differentiation are all strongly associated with Ascl1 BEs (p ≤ 1.3 × 10−11, 8.0 × 10−126, 1.1 × 10−22, and 7.9 × 10−10 for clusters 1, 2, 3, and 4, respectively), whereas clusters of downregulated genes show no such association (Figure 3D), further indicating that Ascl1 binding correlates with transcriptional activation.

Ascl1 Functions Globally as a Transcriptional Activator

Although it is generally believed that proneural factors function by activating gene transcription, some reports have challenged this view (Alvarez-Rodríguez and Pons, 2009, Rheinbay et al., 2013). In order to investigate this issue, we started by performing an in depth cross-comparison between the location analysis and expression profiling results, annotating each Ascl1 BE to its nearest TSS. Direct comparison with the set of deregulated genes previously described shows that only a small fraction of BEs are considered to be regulatory using nearest gene annotation (Figure 3A). Nonetheless, we find a positive correlation between peak height (which is proportional to significance of binding, or p value) and its potential to upregulate gene expression (top peaks ∼20%). In contrast, the fraction of regulatory peaks that can be associated to repressed genes is invariant with changes in peak size, suggesting no direct relation between Ascl1 binding significance and downregulation of gene expression. Therefore, mere binding of Ascl1 to chromatin does not predict a regulatory event; however, the size of an Ascl1 peak may be correlated with its potential to activate gene transcription.

To have a global view of the association between the data sets of genes bound by Ascl1 and regulated during differentiation, we determined the frequency at which deregulated genes (grouped in bins of increasing thresholds of fold change of expression) are associated with Ascl1 BEs (grouped in bins of increasing p value cutoff) and therefore considered to be directly regulated by Ascl1 (Figure 3B, left). The statistical significance of the overlaps was assessed by a similar comparison against 100 randomly generated control ChIP-seq data sets of identical size (Figure 3B, right). The resulting heat maps indicate a strong association between Ascl1-bound genes and those upregulated during differentiation and a much less pronounced association with downregulated genes (Figure 3B, top left versus bottom left). Moreover, the fraction of downregulated genes that are considered Ascl1 targets is very similar to that expected by chance (Figure 3B, left versus right). Altogether, the above analysis indicates that Ascl1 functions globally as a transcriptional activator. In addition, this function is independent of the progress of differentiation during the first 50 hr, as shown with similar analyses for the different time points considered in the transcription profiling (Figures 3C and S3). Importantly, restricting the analysis to the Ascl1 BEs located within ±5 kb of a TSS, producing a more stringent gene annotation, results in a similar conclusion (Figure S4).

To integrate the transcriptional profiling data sets collected at distinct time points, we applied a fuzzy c-means clustering algorithm, resulting in the identification of seven distinct temporal clusters (clusters 1–4 comprised of upregulated genes, clusters 5–7 of downregulated genes) (Figure S5; Table S3). Clusters of genes that are upregulated during differentiation are all strongly associated with Ascl1 BEs (p ≤ 1.3 × 10−11, 8.0 × 10−126, 1.1 × 10−22, and 7.9 × 10−10 for clusters 1, 2, 3, and 4, respectively), whereas clusters of downregulated genes show no such association (Figure 3D), further indicating that Ascl1 binding correlates with transcriptional activation.

Characterization of Ascl1 Transcriptional ProgramThe relative size of each cluster (Figure S5B) and their enrichment for BEs indicate that most Ascl1 target genes fall within a cluster characterized by an early activation profile (cluster 2), while a smaller but significant number is upregulated at mid or late time points (clusters 3 and 4, respectively). We then compiled a high-confidence list of 272 Ascl1 direct targets comprising genes from clusters 2, 3, and 4 that are associated with at least one Ascl1 BE with high regulatory potential (p < 10−25) (Table S4). A search for terms describing “Biological Process,” “Molecular Function,” and “Pathways” (Panther Classification System) revealed the segregation in each temporal cluster of Ascl1 target genes encoding proteins of distinct classes and functions, suggesting the control of various subprograms at distinct developmental stages (Figure S5C). Genes with the earliest onset of activation (belonging to cluster 2) are associated with signal transduction and cell communication, while targets with a mid-onset of activation (cluster 3) encode mostly proteins with a function related to transcription or Notch signaling. Genes with latest onset (cluster 4) are enriched for generic terms associated with neural development and relate to late events in the differentiation program, from cell migration to axonal growth and guidance (Figure S5C). In summary and in agreement with our previous study performed on a smaller scale, the identification of Ascl1 target genes in our neurogenesis model demonstrates the direct control of distinct components of the neurogenic differentiation program by Ascl1.

Characterization of Ascl1 Transcriptional Program

The relative size of each cluster (Figure S5B) and their enrichment for BEs indicate that most Ascl1 target genes fall within a cluster characterized by an early activation profile (cluster 2), while a smaller but significant number is upregulated at mid or late time points (clusters 3 and 4, respectively). We then compiled a high-confidence list of 272 Ascl1 direct targets comprising genes from clusters 2, 3, and 4 that are associated with at least one Ascl1 BE with high regulatory potential (p < 10−25) (Table S4). A search for terms describing “Biological Process,” “Molecular Function,” and “Pathways” (Panther Classification System) revealed the segregation in each temporal cluster of Ascl1 target genes encoding proteins of distinct classes and functions, suggesting the control of various subprograms at distinct developmental stages (Figure S5C). Genes with the earliest onset of activation (belonging to cluster 2) are associated with signal transduction and cell communication, while targets with a mid-onset of activation (cluster 3) encode mostly proteins with a function related to transcription or Notch signaling. Genes with latest onset (cluster 4) are enriched for generic terms associated with neural development and relate to late events in the differentiation program, from cell migration to axonal growth and guidance (Figure S5C). In summary and in agreement with our previous study performed on a smaller scale, the identification of Ascl1 target genes in our neurogenesis model demonstrates the direct control of distinct components of the neurogenic differentiation program by Ascl1.

Ascl1 Access to Its Target Sites Does Not Depend on the Differentiation Stage of NS CellsIt was previously shown that Ascl1 target genes display different onsets of expression at distinct stages of the neuronal lineage (Borromeo et al., 2014, Castro et al., 2011). Such temporal patterning could be associated with distinct accessibilities of Ascl1 to its binding sites (e.g., due to differences in the chromatin landscape). To investigate this, we compared the binding profile of overexpressed Ascl1 at two distinct stages by extending the previous ChIP-seq analysis to a time point at the onset of differentiation (30 min upon addition of tamoxifen) (Table S9). Surprisingly, visual inspection of the genomic distribution of Ascl1 peaks finds a remarkable similarity between Ascl1 BEs at t = 30 min and t = 18 hr (Figure 4A). Comparison of the two Ascl1 binding profiles in a bin-by-bin approach within the confidence limits defined by the previous ChIP-PCR validation (Figure 4B, dashed line) shows an overlap of 86% between the two stages (Figure 4B). Furthermore, the presence of normalized sequencing signal across both samples in genomic regions centered at putative sample-specific BEs suggests that the overlap of occupancies may have been underestimated by peak calling conditions (Figure 4C). Overall, our results indicate that the accessibility of Ascl1 to the full complement of its binding sites remains largely identical throughout differentiation.

Ascl1 Access to Its Target Sites Does Not Depend on the Differentiation Stage of NS Cells

It was previously shown that Ascl1 target genes display different onsets of expression at distinct stages of the neuronal lineage (Borromeo et al., 2014, Castro et al., 2011). Such temporal patterning could be associated with distinct accessibilities of Ascl1 to its binding sites (e.g., due to differences in the chromatin landscape). To investigate this, we compared the binding profile of overexpressed Ascl1 at two distinct stages by extending the previous ChIP-seq analysis to a time point at the onset of differentiation (30 min upon addition of tamoxifen) (Table S9). Surprisingly, visual inspection of the genomic distribution of Ascl1 peaks finds a remarkable similarity between Ascl1 BEs at t = 30 min and t = 18 hr (Figure 4A). Comparison of the two Ascl1 binding profiles in a bin-by-bin approach within the confidence limits defined by the previous ChIP-PCR validation (Figure 4B, dashed line) shows an overlap of 86% between the two stages (Figure 4B). Furthermore, the presence of normalized sequencing signal across both samples in genomic regions centered at putative sample-specific BEs suggests that the overlap of occupancies may have been underestimated by peak calling conditions (Figure 4C). Overall, our results indicate that the accessibility of Ascl1 to the full complement of its binding sites remains largely identical throughout differentiation.

Characterization of Chromatin Landscape Changes during DifferentiationWe next asked what impact Ascl1 may have on the chromatin landscape when it promotes neuronal differentiation of NS cells. We started by characterizing the chromatin landscape of proliferating NS cells and of NS cells undergoing differentiation by Ascl1, using a DNase I hypersensitivity assay coupled to massive parallel sequencing (DNase-seq). This technique identifies regions of decreased nucleosome occupancy (herein referred as “open chromatin”) on a genome-wide scale, which correspond mostly to active regulatory elements such as promoters, enhancers, insulators, and silencers (Boyle et al., 2008, Natarajan et al., 2012, Thurman et al., 2012). The density of mapped reads for each genome position was computed to generate a comprehensive list of DNase I hypersensitivity sites (DHSs). Using a constant threshold of p < 10−5, we identified ∼87,000 and ∼94,000 DHSs in proliferating and differentiating NS cells, respectively (Tables S5 and S6), numbers with a magnitude consistent with those obtained in other cell types (Song et al., 2011). Although the majority of these sites is shared by both experimental conditions, each cell state exhibits a specific set of ∼20,000 DHSs (Figure 5A).Since Ascl1 functions as a transcriptional activator, we focused on the DHSs induced during differentiation (Table S7), as these are more likely to provide new insights into Ascl1 function. To validate these DHSs, we used formaldehyde-assisted isolation of regulatory elements (FAIRE), an alternative method to identify regions of open chromatin. Although distinct preferences have been described for each technique in the identification of distal or proximal regulatory regions, the two techniques have been shown to yield largely overlapping results on a genome-wide basis (Song et al., 2011). Indeed, 9 of 13 tested regions presented more than 2-fold enrichment by FAIRE-PCR across samples collected before and 24 hr after addition of tamoxifen, as opposed to control regions (Figure 5B).

Characterization of Chromatin Landscape Changes during Differentiation

We next asked what impact Ascl1 may have on the chromatin landscape when it promotes neuronal differentiation of NS cells. We started by characterizing the chromatin landscape of proliferating NS cells and of NS cells undergoing differentiation by Ascl1, using a DNase I hypersensitivity assay coupled to massive parallel sequencing (DNase-seq). This technique identifies regions of decreased nucleosome occupancy (herein referred as “open chromatin”) on a genome-wide scale, which correspond mostly to active regulatory elements such as promoters, enhancers, insulators, and silencers (Boyle et al., 2008, Natarajan et al., 2012, Thurman et al., 2012). The density of mapped reads for each genome position was computed to generate a comprehensive list of DNase I hypersensitivity sites (DHSs). Using a constant threshold of p < 10−5, we identified ∼87,000 and ∼94,000 DHSs in proliferating and differentiating NS cells, respectively (Tables S5 and S6), numbers with a magnitude consistent with those obtained in other cell types (Song et al., 2011). Although the majority of these sites is shared by both experimental conditions, each cell state exhibits a specific set of ∼20,000 DHSs (Figure 5A).

Since Ascl1 functions as a transcriptional activator, we focused on the DHSs induced during differentiation (Table S7), as these are more likely to provide new insights into Ascl1 function. To validate these DHSs, we used formaldehyde-assisted isolation of regulatory elements (FAIRE), an alternative method to identify regions of open chromatin. Although distinct preferences have been described for each technique in the identification of distal or proximal regulatory regions, the two techniques have been shown to yield largely overlapping results on a genome-wide basis (Song et al., 2011). Indeed, 9 of 13 tested regions presented more than 2-fold enrichment by FAIRE-PCR across samples collected before and 24 hr after addition of tamoxifen, as opposed to control regions (Figure 5B).

Induced DHSs Are Associated with Genes Expressed De Novo during DifferentiationNext, we investigated how regions of chromatin newly opened during differentiation may be associated with the observed changes in gene expression. We find a statistically significant enrichment of differentiation-induced DHSs in the vicinity of upregulated genes (the sum of clusters 1–4; p < 1.3 × 10−29), in sharp contrast with downregulated genes (clusters 5–7) (Figure 5C). Moreover, a large fraction of all upregulated genes (413 of 760) is associated with at least one differentiation-induced DHS, suggesting the importance of these putative regulatory regions in activating gene expression during neurogenesis. Notably, upregulated genes associated with differentiation-induced DHSs are either not expressed in proliferating NS cells or expressed at a low level when compared with upregulated genes near constant DHSs (Figure 5D). This is well exemplified by the induction of NeuroD4, a pro-differentiation TF only expressed in post-mitotic precursors (Ohsawa et al., 2005), and which is associated with newly open DHSs (Figure 6B). Overall, our results indicate that activation of differentiation genes is strongly associated with the appearance of new DHSs.

Induced DHSs Are Associated with Genes Expressed De Novo during Differentiation

Next, we investigated how regions of chromatin newly opened during differentiation may be associated with the observed changes in gene expression. We find a statistically significant enrichment of differentiation-induced DHSs in the vicinity of upregulated genes (the sum of clusters 1–4; p < 1.3 × 10−29), in sharp contrast with downregulated genes (clusters 5–7) (Figure 5C). Moreover, a large fraction of all upregulated genes (413 of 760) is associated with at least one differentiation-induced DHS, suggesting the importance of these putative regulatory regions in activating gene expression during neurogenesis. Notably, upregulated genes associated with differentiation-induced DHSs are either not expressed in proliferating NS cells or expressed at a low level when compared with upregulated genes near constant DHSs (Figure 5D). This is well exemplified by the induction of NeuroD4, a pro-differentiation TF only expressed in post-mitotic precursors (Ohsawa et al., 2005), and which is associated with newly open DHSs (Figure 6B). Overall, our results indicate that activation of differentiation genes is strongly associated with the appearance of new DHSs.

Ascl1 Promotes Chromatin AccessibilityIn order to investigate which DNA-binding TFs may be associated with the appearance of newly open chromatin regions, we took advantage of the high resolution potential of Digital Genomic Footprinting (DGF) applied to DNase-seq data to determine the position of sites occupied by TFs within differentiation-induced DHSs (Figure 5E) (Piper et al., 2013). DGF identified almost 17,000 genomic sites (Table S8) with significant occupancy levels (p < 10−10), and the corresponding sequences were then subject to a search for frequency of motif occurrence. The most abundant motif found corresponds to the consensus sequence of CTCF and is likely to reflect its frequent binding to insulator regions identified by DNase-seq (Figure 5E) (Thurman et al., 2012). Three additional motifs reveal binding by TFs of the bHLH or NFI families, with the most frequent one corresponding to the E-box consensus sequence, which mediates Ascl1 binding. This observation establishes Ascl1 as a prime candidate to promote the opening of chromatin structure and induce new DHSs during differentiation.To understand how Ascl1 binding may relate to the observed changes in chromatin compaction, we compared the genomic distribution of Ascl1 BEs with that of DHSs before and after neuronal differentiation (Figure 5F). Ascl1 ChIP-seq and DNase-seq data sets show a high degree of overlap (e.g., 80.3% of Ascl1 BEs with p < 10−18 at t = 30 min fall within DHSs in proliferating cells), indicating that a large fraction of BEs, but not all, occur in regions of open chromatin in proliferating NS cells (Figure 5F). Strikingly, this overlap increases significantly under differentiation conditions (to 91.05% after differentiation, in the aforementioned case; Figure 5F). Altogether, a comparison between the Ascl1 ChIP-seq and DNase-seq suggests that (1) the Ascl1 binding profile remains constant at the two stages analyzed, (2) a large fraction of Ascl1 BEs occurs in regions of open chromatin in proliferating cells, with a subset falling within closed chromatin, and (3) binding of Ascl1 to closed chromatin precedes the appearance of new DHSs during differentiation.Visualization of aligned Ascl1 ChIP-seq and DNase-seq data suggests that many differentiation-induced DHSs overlap with Ascl1 BEs (Figure 6A), as exemplified by those in the vicinity of genes activated during differentiation such as NeuroD4, Ap3b2, Mcf2l, and Nrxn3 (Figure 6B). Validation of changes in chromatin compaction at the selected Ascl1 binding sites was performed by FAIRE-PCR using chromatin extracted from NS5 cells expressing full-length Ascl1 under the regulation of a doxycycline inducible promoter, before and after Ascl1 induction (Figure 6C). Although not all DHSs that arise de novo during differentiation are associated with an Ascl1 BE, as not all Ascl1 BEs overlap with changes detected by DNase-seq, the vast majority of Ascl1 BEs fall within DHSs in differentiating cells (91.05% or 91.27% with p < 10−18, at t = 30 min or t = 18 hr, respectively; Figure 5F).To investigate the correlation between Ascl1 binding and changes in chromatin compaction on a large scale, we quantified the DNase-seq signal from both time points centered on Ascl1 peak summits. Confirming previous observations, the overall DNase-seq profile at Ascl1 peak summits is consistent with many BEs occurring in regions of already open chromatin at t = 30 min (Figure 6D). Nevertheless, a significant increase is observed when comparing profiles before and after differentiation (2.33 increase of median read count) at Ascl1 peak summits genome-wide (Figure 6D). This increase is amplified when focusing our analysis on Ascl1 peaks that fall within differentiation-induced DHSs (4.37 increase of median read count; Figure 6D). Ascl1 BEs co-localizing with differentiation-induced DHSs are strongly associated with gene activation and not with gene repression (p < 6.8 × 10−32; Figure 6E), targeting a total number of 98 genes during differentiation. Overall, the integration of Ascl1 location analysis, DNase-seq results, and expression profiling demonstrate Ascl1 promotes genome-wide changes in chromatin compaction at its target sites during neuronal differentiation and the appearance of new DHSs associated with activation of gene expression.Our results are in agreement with the observation that Ascl1 can bind to nucleosomal DNA when overexpressed in fibroblasts (Wapinski et al., 2013). To verify that Ascl1 can bind to closed chromatin in a physiological context, we focused on the Ascl1-bound regulatory region identified immediately downstream the NeuroD4 promoter (BS2 in Figure 6A), a target gene of Ascl1 in our cellular system that is not expressed in ventral telencephalon. We reasoned that binding to closed chromatin to NeuroD4 may still occur in this embryonic brain region at the neurogenic period, even in the absence of gene expression. Indeed, we detected strong binding of Ascl1, as compared with a negative control region within the NeuroD4 locus by ChIP-PCR in chromatin extracted from mouse ventral telencephalon (Figure 6F). However, in contrast to our previous findings with differentiating NS cells in culture, a lack of enrichment for non-nucleosomal DNA assessed by FAIRE-PCR shows this Ascl1 target site is found within closed chromatin in ventral telencephalon (Figure 6G, compare with Ascl1 site in Dll1). Thus, our results with the NeuroD4 regulatory region demonstrate that endogenous Ascl1 can bind to closed chromatin in an in vivo context.Finally, we wished to analyze the accessibility of chromatin at Ascl1 target sites in the absence of Ascl1 expression. For this, we used chromatin extracted from ventral telencephalon of Ascl1-null embryos and probed Ascl1-bound regions associated with previously validated Ascl1 target genes (Dll1, Fbxw7, and Stk33) in this embryonic structure (Castro et al., 2006). Ascl1 null embryos show a very significant reduction of FAIRE-PCR signal relative to WT chromatin at all tested Ascl1 binding sites, as compared with pairwise control regions, which is consistent with the results above and in support of a role for Ascl1 in regulating chromatin compaction in vivo (Figure S6).

Ascl1 Promotes Chromatin Accessibility

In order to investigate which DNA-binding TFs may be associated with the appearance of newly open chromatin regions, we took advantage of the high resolution potential of Digital Genomic Footprinting (DGF) applied to DNase-seq data to determine the position of sites occupied by TFs within differentiation-induced DHSs (Figure 5E) (Piper et al., 2013). DGF identified almost 17,000 genomic sites (Table S8) with significant occupancy levels (p < 10−10), and the corresponding sequences were then subject to a search for frequency of motif occurrence. The most abundant motif found corresponds to the consensus sequence of CTCF and is likely to reflect its frequent binding to insulator regions identified by DNase-seq (Figure 5E) (Thurman et al., 2012). Three additional motifs reveal binding by TFs of the bHLH or NFI families, with the most frequent one corresponding to the E-box consensus sequence, which mediates Ascl1 binding. This observation establishes Ascl1 as a prime candidate to promote the opening of chromatin structure and induce new DHSs during differentiation.

To understand how Ascl1 binding may relate to the observed changes in chromatin compaction, we compared the genomic distribution of Ascl1 BEs with that of DHSs before and after neuronal differentiation (Figure 5F). Ascl1 ChIP-seq and DNase-seq data sets show a high degree of overlap (e.g., 80.3% of Ascl1 BEs with p < 10−18 at t = 30 min fall within DHSs in proliferating cells), indicating that a large fraction of BEs, but not all, occur in regions of open chromatin in proliferating NS cells (Figure 5F). Strikingly, this overlap increases significantly under differentiation conditions (to 91.05% after differentiation, in the aforementioned case; Figure 5F). Altogether, a comparison between the Ascl1 ChIP-seq and DNase-seq suggests that (1) the Ascl1 binding profile remains constant at the two stages analyzed, (2) a large fraction of Ascl1 BEs occurs in regions of open chromatin in proliferating cells, with a subset falling within closed chromatin, and (3) binding of Ascl1 to closed chromatin precedes the appearance of new DHSs during differentiation.

Visualization of aligned Ascl1 ChIP-seq and DNase-seq data suggests that many differentiation-induced DHSs overlap with Ascl1 BEs (Figure 6A), as exemplified by those in the vicinity of genes activated during differentiation such as NeuroD4, Ap3b2, Mcf2l, and Nrxn3 (Figure 6B). Validation of changes in chromatin compaction at the selected Ascl1 binding sites was performed by FAIRE-PCR using chromatin extracted from NS5 cells expressing full-length Ascl1 under the regulation of a doxycycline inducible promoter, before and after Ascl1 induction (Figure 6C). Although not all DHSs that arise de novo during differentiation are associated with an Ascl1 BE, as not all Ascl1 BEs overlap with changes detected by DNase-seq, the vast majority of Ascl1 BEs fall within DHSs in differentiating cells (91.05% or 91.27% with p < 10−18, at t = 30 min or t = 18 hr, respectively; Figure 5F).

To investigate the correlation between Ascl1 binding and changes in chromatin compaction on a large scale, we quantified the DNase-seq signal from both time points centered on Ascl1 peak summits. Confirming previous observations, the overall DNase-seq profile at Ascl1 peak summits is consistent with many BEs occurring in regions of already open chromatin at t = 30 min (Figure 6D). Nevertheless, a significant increase is observed when comparing profiles before and after differentiation (2.33 increase of median read count) at Ascl1 peak summits genome-wide (Figure 6D). This increase is amplified when focusing our analysis on Ascl1 peaks that fall within differentiation-induced DHSs (4.37 increase of median read count; Figure 6D). Ascl1 BEs co-localizing with differentiation-induced DHSs are strongly associated with gene activation and not with gene repression (p < 6.8 × 10−32; Figure 6E), targeting a total number of 98 genes during differentiation. Overall, the integration of Ascl1 location analysis, DNase-seq results, and expression profiling demonstrate Ascl1 promotes genome-wide changes in chromatin compaction at its target sites during neuronal differentiation and the appearance of new DHSs associated with activation of gene expression.

Our results are in agreement with the observation that Ascl1 can bind to nucleosomal DNA when overexpressed in fibroblasts (Wapinski et al., 2013). To verify that Ascl1 can bind to closed chromatin in a physiological context, we focused on the Ascl1-bound regulatory region identified immediately downstream the NeuroD4 promoter (BS2 in Figure 6A), a target gene of Ascl1 in our cellular system that is not expressed in ventral telencephalon. We reasoned that binding to closed chromatin to NeuroD4 may still occur in this embryonic brain region at the neurogenic period, even in the absence of gene expression. Indeed, we detected strong binding of Ascl1, as compared with a negative control region within the NeuroD4 locus by ChIP-PCR in chromatin extracted from mouse ventral telencephalon (Figure 6F). However, in contrast to our previous findings with differentiating NS cells in culture, a lack of enrichment for non-nucleosomal DNA assessed by FAIRE-PCR shows this Ascl1 target site is found within closed chromatin in ventral telencephalon (Figure 6G, compare with Ascl1 site in Dll1). Thus, our results with the NeuroD4 regulatory region demonstrate that endogenous Ascl1 can bind to closed chromatin in an in vivo context.

Finally, we wished to analyze the accessibility of chromatin at Ascl1 target sites in the absence of Ascl1 expression. For this, we used chromatin extracted from ventral telencephalon of Ascl1-null embryos and probed Ascl1-bound regions associated with previously validated Ascl1 target genes (Dll1, Fbxw7, and Stk33) in this embryonic structure (Castro et al., 2006). Ascl1 null embryos show a very significant reduction of FAIRE-PCR signal relative to WT chromatin at all tested Ascl1 binding sites, as compared with pairwise control regions, which is consistent with the results above and in support of a role for Ascl1 in regulating chromatin compaction in vivo (Figure S6).

DiscussionIn spite of its pivotal role during neurogenesis, little is known on how Ascl1 functions to regulate gene expression. Here we investigated the reciprocal interactions between Ascl1 and the chromatin landscape when promoting the neuronal differentiation of NS cells in culture. We found that Ascl1 can bind to and promote the opening of closed chromatin regions in its native context. Below we discuss our findings and their implications to the mechanisms that govern the temporal progression of the Ascl1 transcriptional program.Our work resulted in a fine-grained characterization of a differentiation program driven by Ascl1. In spite of displaying a wide range of biological roles, Ascl1 target genes are remarkably enriched in two categories. The first consists of genes encoding transcriptional regulators, an observation consistent with the master regulator function of Ascl1 in this developmental process. Importantly, the activation of some TFs may define feedback loops that modulate Ascl1 activity. This is the case of Id1, an inhibitor of bHLH activity that functions by sequestering E-proteins (Ruzinova and Benezra, 2003), and Cbfa2t2/MTGR1, a zinc-finger protein that has been shown to function as a co-repressor of proneural proteins, including Ascl1 (Aaker et al., 2010). Other examples, suggesting positive feedback mechanisms, are TFs previously linked to a relief of Notch inhibition by counteracting the activity of Hes1 (Hes6), repressing Notch1 (Prox1), or by as yet unknown mechanisms (MyT1) (Bellefroid et al., 1996, Gratton et al., 2003, Kaltezioti et al., 2010). These regulatory events define a less well-understood role of Ascl1 in modulating Notch signaling cell autonomously, of putative importance to the progression of differentiation. The second most prominent group of Ascl1 target genes encodes cytoskeleton-related proteins. The abundance of this group of genes is likely to result from its recognized role in many cellular functions, from signal transduction (where it may serve as a scaffold for components of signaling pathways) to cell shape change and locomotion, activities that are essential to the neurogenic differentiation program (Forgacs et al., 2004).It was recently shown that Ascl1 promotes sequentially the proliferation and differentiation along the neuronal lineage, with the concomitant activation of distinct target genes (Castro et al., 2011). Our study focused on the transcriptional program activated by Ascl1 when promoting neuronal differentiation. Comparing the identity and molecular determinants for target gene specificity associated with both cellular functions will likely require the characterization of Ascl1 binding profile in an oscillatory versus sustained mode of expression. Nevertheless, the observation that overexpressed Ascl1 can readily access the full complement of its differentiation sites at the onset of differentiation suggests these may already be accessible to this TF in proliferating cells, an important observation in the context of the dynamics of Ascl1 function along the neuronal lineage.It was previously shown that overexpressed Ascl1 can bind to closed chromatin in fibroblasts (as assessed by FAIRE-PCR), defining it as a pioneer TF (Wapinski et al., 2013), results that the present study validates for the first time in a neural context. Moreover, our analysis of binding of Ascl1 to the NeuroD4 locus in ventral telencephalic progenitors that do not express this gene allowed for the dissociation between binding and opening of chromatin, revealing binding of endogenous Ascl1 to closed chromatin in vivo. Altogether, our study supports the idea that Ascl1 displays its pioneer activity when regulating gene expression in its native context.Our analysis of chromatin states suggests that most Ascl1 binding in our differentiation model occurs within enhancer regions that are already active in proliferating NS cells as shown by their enrichment for both H3K4me1 and H3K27ac. This may be due to the fact that many genes upregulated by Ascl1 during differentiation are already expressed in proliferating NS cells. Alternatively, it may reflect the trivalent mark (H3K4me1, H3K27ac, and H3K9m3) identified at Ascl1 binding sites in various cell types permissive to Ascl1-mediated reprogramming (Wapinski et al., 2013). Most importantly, Ascl1 promotes DNase I accessibility to regions of closed chromatin at a smaller but significant number of its target sites associated with the activation of a differentiation-specific component of its transcriptional program. Our results cast light into a previously unknown function of Ascl1 and establish the first link between the chromatin landscape at Ascl1-bound regulatory regions and the temporal pattern of the Ascl1 program along the neuronal lineage. Moreover, they suggest that the hierarchical model recently proposed for the reprograming paradigm (Wapinski et al., 2013) may also be representative of the neurogenesis process and that opening of chromatin regions at Ascl1 target sites may be important to allow subsequent binding of other TFs in differentiating cells. Finally, it is tempting to speculate that the broad effect of Ascl1 in promoting chromatin accessibility described in this study will be important for the reprogramming capacity of this TF upon its ectopic expression in various cell types.

In spite of its pivotal role during neurogenesis, little is known on how Ascl1 functions to regulate gene expression. Here we investigated the reciprocal interactions between Ascl1 and the chromatin landscape when promoting the neuronal differentiation of NS cells in culture. We found that Ascl1 can bind to and promote the opening of closed chromatin regions in its native context. Below we discuss our findings and their implications to the mechanisms that govern the temporal progression of the Ascl1 transcriptional program.

Our work resulted in a fine-grained characterization of a differentiation program driven by Ascl1. In spite of displaying a wide range of biological roles, Ascl1 target genes are remarkably enriched in two categories. The first consists of genes encoding transcriptional regulators, an observation consistent with the master regulator function of Ascl1 in this developmental process. Importantly, the activation of some TFs may define feedback loops that modulate Ascl1 activity. This is the case of Id1, an inhibitor of bHLH activity that functions by sequestering E-proteins (Ruzinova and Benezra, 2003), and Cbfa2t2/MTGR1, a zinc-finger protein that has been shown to function as a co-repressor of proneural proteins, including Ascl1 (Aaker et al., 2010). Other examples, suggesting positive feedback mechanisms, are TFs previously linked to a relief of Notch inhibition by counteracting the activity of Hes1 (Hes6), repressing Notch1 (Prox1), or by as yet unknown mechanisms (MyT1) (Bellefroid et al., 1996, Gratton et al., 2003, Kaltezioti et al., 2010). These regulatory events define a less well-understood role of Ascl1 in modulating Notch signaling cell autonomously, of putative importance to the progression of differentiation. The second most prominent group of Ascl1 target genes encodes cytoskeleton-related proteins. The abundance of this group of genes is likely to result from its recognized role in many cellular functions, from signal transduction (where it may serve as a scaffold for components of signaling pathways) to cell shape change and locomotion, activities that are essential to the neurogenic differentiation program (Forgacs et al., 2004).

It was recently shown that Ascl1 promotes sequentially the proliferation and differentiation along the neuronal lineage, with the concomitant activation of distinct target genes (Castro et al., 2011). Our study focused on the transcriptional program activated by Ascl1 when promoting neuronal differentiation. Comparing the identity and molecular determinants for target gene specificity associated with both cellular functions will likely require the characterization of Ascl1 binding profile in an oscillatory versus sustained mode of expression. Nevertheless, the observation that overexpressed Ascl1 can readily access the full complement of its differentiation sites at the onset of differentiation suggests these may already be accessible to this TF in proliferating cells, an important observation in the context of the dynamics of Ascl1 function along the neuronal lineage.

It was previously shown that overexpressed Ascl1 can bind to closed chromatin in fibroblasts (as assessed by FAIRE-PCR), defining it as a pioneer TF (Wapinski et al., 2013), results that the present study validates for the first time in a neural context. Moreover, our analysis of binding of Ascl1 to the NeuroD4 locus in ventral telencephalic progenitors that do not express this gene allowed for the dissociation between binding and opening of chromatin, revealing binding of endogenous Ascl1 to closed chromatin in vivo. Altogether, our study supports the idea that Ascl1 displays its pioneer activity when regulating gene expression in its native context.

Our analysis of chromatin states suggests that most Ascl1 binding in our differentiation model occurs within enhancer regions that are already active in proliferating NS cells as shown by their enrichment for both H3K4me1 and H3K27ac. This may be due to the fact that many genes upregulated by Ascl1 during differentiation are already expressed in proliferating NS cells. Alternatively, it may reflect the trivalent mark (H3K4me1, H3K27ac, and H3K9m3) identified at Ascl1 binding sites in various cell types permissive to Ascl1-mediated reprogramming (Wapinski et al., 2013). Most importantly, Ascl1 promotes DNase I accessibility to regions of closed chromatin at a smaller but significant number of its target sites associated with the activation of a differentiation-specific component of its transcriptional program. Our results cast light into a previously unknown function of Ascl1 and establish the first link between the chromatin landscape at Ascl1-bound regulatory regions and the temporal pattern of the Ascl1 program along the neuronal lineage. Moreover, they suggest that the hierarchical model recently proposed for the reprograming paradigm (Wapinski et al., 2013) may also be representative of the neurogenesis process and that opening of chromatin regions at Ascl1 target sites may be important to allow subsequent binding of other TFs in differentiating cells. Finally, it is tempting to speculate that the broad effect of Ascl1 in promoting chromatin accessibility described in this study will be important for the reprogramming capacity of this TF upon its ectopic expression in various cell types.

Experimental ProceduresPlasmids and Virus ProductionAscl1-ERT2 encodes full-length mouse Ascl1 in frame with the modified ligand binding domain of the human estrogen receptor (ERT2) and an N-terminal Flag/HA tag sequence. For virus production, Ascl1-ERT2 was subcloned into pMX-IRES-GFP or pBABE-IRES-GFP-puro. Replication-incompetent retroviruses were produced from HEK293T cells transiently transfected with retroviral, viral envelope-, and VSVG-pseudotyping plasmids. Retroviral particles were concentrated from supernatant by ultracentrifugation at 90,000 g for 90 min. Viral titers were typically 105–106 infectious particles/ml.Culture and Infection of NS5 CellsNS5 cells were cultured as previously described (Conti et al., 2005) using laminin as media supplement (20 μg/ml). NS5 Ascl1-ERT2 cells were generated upon infection with pBABE-Ascl1-ERT2-IRES-GFP-puro, followed by selection in complete medium containing 1-μg/ml puromycin (Calbiochem). For differentiation, NS5 Ascl1-ERT2 cells were plated at 28,000 per cm2 of density and induced with 50-nM 4-Hydroxytamoxifen, while reducing EGF concentration to 5 ng/ml.Quantification of Ascl1 Protein LevelsAll experiments were carried out upon approval and following the guidelines of the ethics committee of Instituto Gulbenkian de Ciência.Quantification of Ascl1 (endogenous + Ascl1-ERT2) done by parallel immunostaining with anti-Ascl1 antibody in acutely cultured primary ventral telencephalon progenitors from E14.5 mouse embryos and NS5 Ascl1-ERT2 cells (three replicates each). A total of 939 and 751 cells for embryonic and NS5 progenitors, respectively, was imaged with identical exposure and images thresholded and normalized—mean fluorescence per cell x (mean fluorescence of all cells in picture/mean fluorescence of all cells per sample)—to define positive cells. Fluorescence intensity was measured with ImageJ.Expression Analysis of NS5 Ascl1-ERT2 CellsExtraction of mRNA and cDNA synthesis performed as previously described (Castro et al., 2011). Quantitative real-time PCR (qRT-PCR) was performed according to the manufacturer using SYBR Green super-mix (Quanta BioSciences) on an ABI7500 machine (Applied Biosystems). For microarray analysis, quality of total RNA from biological triplicates was assessed using Agilent Bioanalyser, and samples were processed according to the Illumina Whole-genome Gene Expression Direct Hybridization Assay Guide, using Illumina TotalPrep-96 RNA Amplification kit (Life Technologies). Quality control was performed on labeled cRNA, and 750 ng of labeled cRNA was hybridized to Mouse Ref-8v2 arrays and scanned by BeedStudio v.3.1.3 (quantile normalization with background correction). Normalization among arrays and significant analysis were performed using GeneSpring X (Agilent). Further details of data processing, including gene clustering, are described in Supplemental Information.ChIP-seqFor Ascl1 ChIP-seq, NS5 Ascl1-ERT2 cells were fixed sequentially with 2 mM di(N-succimidyl) glutarate and 1% formaldehyde in PBS and then lysed, sonicated, and immunoprecipitated with anti-HA antibody, as previously described (Castro et al., 2011). DNA libraries were prepared from 10 ng of immunoprecipitated DNA according to the standard Illumina ChIP-seq protocol and sequenced with Illumina GAIIx. Raw reads were mapped to the mouse genome (NCBI37/mm9) with Bowtie 0.12.7 (Langmead et al., 2009). Uniquely mapped reads data (∼11.5 million for both Ascl1 ChIP and input chromatin samples) were then subsampled before peak calling with MACS 1.4.1 (Zhang et al., 2008). For histone marks ChIP-seq, Ascl1-ERT2 cells were fixed in 1% formaldehyde in PBS, prepared as described above and immunoprecipitated with anti-H3K27ac and anti-H3K4me1 antibodies. Libraries were sequenced with Illumina HiSeq 2000. Raw reads were mapped as above (25 million uniquely mapped reads for all samples) and data processed with MACS 2.1.0. For chromatin state characterization, we used a Hidden Markov modeling of chromatin enrichment software—ChromHMM (Ernst and Kellis, 2012). Further details of data processing are described in Supplemental Information.FAIRE-PCRChromatin preparation was done with a single fixation with 1% formaldehyde, from NS5 Ascl1-ERT2 cells, or when indicated with NS5 cells infected with a doxycycline inducible lentivirus expressing WT Ascl1 (Wapinski et al., 2013). Three rounds of phenol/chloroform extraction were followed by isopropanol precipitation of the DNA. Quantification of genomic regions was done using a standard curve generated with decross-linked input chromatin by qRT-PCR as above.DNase-seqDNase-seq samples from differentiating NS5 Ascl1-ERT2 cells or proliferating NS5 cells were prepared as previously described (Song et al., 2011). Libraries were generated as previously described (Boyle et al., 2008, Song and Crawford, 2010) with a slight modification to the linkers to increase ligation efficiency (Song et al., 2011) and then sequenced with Illumina GAIIx. Mapping was done as for ChIP-seq, resulting in 165.9 and 168.7 millions of unique reads for proliferating and differentiating conditions, respectively. Genomic location of DHSs was identified with MACS version 1.4.1, following a protocol described in Supplemental Information.For oligonucleotides used in this study, see also Table S10.

Experimental Procedures

Plasmids and Virus ProductionAscl1-ERT2 encodes full-length mouse Ascl1 in frame with the modified ligand binding domain of the human estrogen receptor (ERT2) and an N-terminal Flag/HA tag sequence. For virus production, Ascl1-ERT2 was subcloned into pMX-IRES-GFP or pBABE-IRES-GFP-puro. Replication-incompetent retroviruses were produced from HEK293T cells transiently transfected with retroviral, viral envelope-, and VSVG-pseudotyping plasmids. Retroviral particles were concentrated from supernatant by ultracentrifugation at 90,000 g for 90 min. Viral titers were typically 105–106 infectious particles/ml.

Plasmids and Virus Production

Ascl1-ERT2 encodes full-length mouse Ascl1 in frame with the modified ligand binding domain of the human estrogen receptor (ERT2) and an N-terminal Flag/HA tag sequence. For virus production, Ascl1-ERT2 was subcloned into pMX-IRES-GFP or pBABE-IRES-GFP-puro. Replication-incompetent retroviruses were produced from HEK293T cells transiently transfected with retroviral, viral envelope-, and VSVG-pseudotyping plasmids. Retroviral particles were concentrated from supernatant by ultracentrifugation at 90,000 g for 90 min. Viral titers were typically 105–106 infectious particles/ml.

Culture and Infection of NS5 CellsNS5 cells were cultured as previously described (Conti et al., 2005) using laminin as media supplement (20 μg/ml). NS5 Ascl1-ERT2 cells were generated upon infection with pBABE-Ascl1-ERT2-IRES-GFP-puro, followed by selection in complete medium containing 1-μg/ml puromycin (Calbiochem). For differentiation, NS5 Ascl1-ERT2 cells were plated at 28,000 per cm2 of density and induced with 50-nM 4-Hydroxytamoxifen, while reducing EGF concentration to 5 ng/ml.

Culture and Infection of NS5 Cells

NS5 cells were cultured as previously described (Conti et al., 2005) using laminin as media supplement (20 μg/ml). NS5 Ascl1-ERT2 cells were generated upon infection with pBABE-Ascl1-ERT2-IRES-GFP-puro, followed by selection in complete medium containing 1-μg/ml puromycin (Calbiochem). For differentiation, NS5 Ascl1-ERT2 cells were plated at 28,000 per cm2 of density and induced with 50-nM 4-Hydroxytamoxifen, while reducing EGF concentration to 5 ng/ml.

Quantification of Ascl1 Protein LevelsAll experiments were carried out upon approval and following the guidelines of the ethics committee of Instituto Gulbenkian de Ciência.Quantification of Ascl1 (endogenous + Ascl1-ERT2) done by parallel immunostaining with anti-Ascl1 antibody in acutely cultured primary ventral telencephalon progenitors from E14.5 mouse embryos and NS5 Ascl1-ERT2 cells (three replicates each). A total of 939 and 751 cells for embryonic and NS5 progenitors, respectively, was imaged with identical exposure and images thresholded and normalized—mean fluorescence per cell x (mean fluorescence of all cells in picture/mean fluorescence of all cells per sample)—to define positive cells. Fluorescence intensity was measured with ImageJ.

Quantification of Ascl1 Protein Levels

All experiments were carried out upon approval and following the guidelines of the ethics committee of Instituto Gulbenkian de Ciência.

Quantification of Ascl1 (endogenous + Ascl1-ERT2) done by parallel immunostaining with anti-Ascl1 antibody in acutely cultured primary ventral telencephalon progenitors from E14.5 mouse embryos and NS5 Ascl1-ERT2 cells (three replicates each). A total of 939 and 751 cells for embryonic and NS5 progenitors, respectively, was imaged with identical exposure and images thresholded and normalized—mean fluorescence per cell x (mean fluorescence of all cells in picture/mean fluorescence of all cells per sample)—to define positive cells. Fluorescence intensity was measured with ImageJ.

Expression Analysis of NS5 Ascl1-ERT2 CellsExtraction of mRNA and cDNA synthesis performed as previously described (Castro et al., 2011). Quantitative real-time PCR (qRT-PCR) was performed according to the manufacturer using SYBR Green super-mix (Quanta BioSciences) on an ABI7500 machine (Applied Biosystems). For microarray analysis, quality of total RNA from biological triplicates was assessed using Agilent Bioanalyser, and samples were processed according to the Illumina Whole-genome Gene Expression Direct Hybridization Assay Guide, using Illumina TotalPrep-96 RNA Amplification kit (Life Technologies). Quality control was performed on labeled cRNA, and 750 ng of labeled cRNA was hybridized to Mouse Ref-8v2 arrays and scanned by BeedStudio v.3.1.3 (quantile normalization with background correction). Normalization among arrays and significant analysis were performed using GeneSpring X (Agilent). Further details of data processing, including gene clustering, are described in Supplemental Information.

Expression Analysis of NS5 Ascl1-ERT2 Cells

Extraction of mRNA and cDNA synthesis performed as previously described (Castro et al., 2011). Quantitative real-time PCR (qRT-PCR) was performed according to the manufacturer using SYBR Green super-mix (Quanta BioSciences) on an ABI7500 machine (Applied Biosystems). For microarray analysis, quality of total RNA from biological triplicates was assessed using Agilent Bioanalyser, and samples were processed according to the Illumina Whole-genome Gene Expression Direct Hybridization Assay Guide, using Illumina TotalPrep-96 RNA Amplification kit (Life Technologies). Quality control was performed on labeled cRNA, and 750 ng of labeled cRNA was hybridized to Mouse Ref-8v2 arrays and scanned by BeedStudio v.3.1.3 (quantile normalization with background correction). Normalization among arrays and significant analysis were performed using GeneSpring X (Agilent). Further details of data processing, including gene clustering, are described in Supplemental Information.

ChIP-seqFor Ascl1 ChIP-seq, NS5 Ascl1-ERT2 cells were fixed sequentially with 2 mM di(N-succimidyl) glutarate and 1% formaldehyde in PBS and then lysed, sonicated, and immunoprecipitated with anti-HA antibody, as previously described (Castro et al., 2011). DNA libraries were prepared from 10 ng of immunoprecipitated DNA according to the standard Illumina ChIP-seq protocol and sequenced with Illumina GAIIx. Raw reads were mapped to the mouse genome (NCBI37/mm9) with Bowtie 0.12.7 (Langmead et al., 2009). Uniquely mapped reads data (∼11.5 million for both Ascl1 ChIP and input chromatin samples) were then subsampled before peak calling with MACS 1.4.1 (Zhang et al., 2008). For histone marks ChIP-seq, Ascl1-ERT2 cells were fixed in 1% formaldehyde in PBS, prepared as described above and immunoprecipitated with anti-H3K27ac and anti-H3K4me1 antibodies. Libraries were sequenced with Illumina HiSeq 2000. Raw reads were mapped as above (25 million uniquely mapped reads for all samples) and data processed with MACS 2.1.0. For chromatin state characterization, we used a Hidden Markov modeling of chromatin enrichment software—ChromHMM (Ernst and Kellis, 2012). Further details of data processing are described in Supplemental Information.

For Ascl1 ChIP-seq, NS5 Ascl1-ERT2 cells were fixed sequentially with 2 mM di(N-succimidyl) glutarate and 1% formaldehyde in PBS and then lysed, sonicated, and immunoprecipitated with anti-HA antibody, as previously described (Castro et al., 2011). DNA libraries were prepared from 10 ng of immunoprecipitated DNA according to the standard Illumina ChIP-seq protocol and sequenced with Illumina GAIIx. Raw reads were mapped to the mouse genome (NCBI37/mm9) with Bowtie 0.12.7 (Langmead et al., 2009). Uniquely mapped reads data (∼11.5 million for both Ascl1 ChIP and input chromatin samples) were then subsampled before peak calling with MACS 1.4.1 (Zhang et al., 2008). For histone marks ChIP-seq, Ascl1-ERT2 cells were fixed in 1% formaldehyde in PBS, prepared as described above and immunoprecipitated with anti-H3K27ac and anti-H3K4me1 antibodies. Libraries were sequenced with Illumina HiSeq 2000. Raw reads were mapped as above (25 million uniquely mapped reads for all samples) and data processed with MACS 2.1.0. For chromatin state characterization, we used a Hidden Markov modeling of chromatin enrichment software—ChromHMM (Ernst and Kellis, 2012). Further details of data processing are described in Supplemental Information.

FAIRE-PCRChromatin preparation was done with a single fixation with 1% formaldehyde, from NS5 Ascl1-ERT2 cells, or when indicated with NS5 cells infected with a doxycycline inducible lentivirus expressing WT Ascl1 (Wapinski et al., 2013). Three rounds of phenol/chloroform extraction were followed by isopropanol precipitation of the DNA. Quantification of genomic regions was done using a standard curve generated with decross-linked input chromatin by qRT-PCR as above.

Chromatin preparation was done with a single fixation with 1% formaldehyde, from NS5 Ascl1-ERT2 cells, or when indicated with NS5 cells infected with a doxycycline inducible lentivirus expressing WT Ascl1 (Wapinski et al., 2013). Three rounds of phenol/chloroform extraction were followed by isopropanol precipitation of the DNA. Quantification of genomic regions was done using a standard curve generated with decross-linked input chromatin by qRT-PCR as above.

DNase-seqDNase-seq samples from differentiating NS5 Ascl1-ERT2 cells or proliferating NS5 cells were prepared as previously described (Song et al., 2011). Libraries were generated as previously described (Boyle et al., 2008, Song and Crawford, 2010) with a slight modification to the linkers to increase ligation efficiency (Song et al., 2011) and then sequenced with Illumina GAIIx. Mapping was done as for ChIP-seq, resulting in 165.9 and 168.7 millions of unique reads for proliferating and differentiating conditions, respectively. Genomic location of DHSs was identified with MACS version 1.4.1, following a protocol described in Supplemental Information.For oligonucleotides used in this study, see also Table S10.

DNase-seq samples from differentiating NS5 Ascl1-ERT2 cells or proliferating NS5 cells were prepared as previously described (Song et al., 2011). Libraries were generated as previously described (Boyle et al., 2008, Song and Crawford, 2010) with a slight modification to the linkers to increase ligation efficiency (Song et al., 2011) and then sequenced with Illumina GAIIx. Mapping was done as for ChIP-seq, resulting in 165.9 and 168.7 millions of unique reads for proliferating and differentiating conditions, respectively. Genomic location of DHSs was identified with MACS version 1.4.1, following a protocol described in Supplemental Information.

For oligonucleotides used in this study, see also Table S10.

A.A.S.F.R. did the bioinformatics analysis. F.F.V. and D.D. designed research and performed experimental work, except electrophysiology tests performed by B.B. D.S.C. and A.A.S.F.R. wrote the manuscript, which was read and approved by all others. All other authors contributed with reagents and analytical tools.
