---
pmid: '29374058'
title: Glioma tumor suppressor candidate region gene 1 (GLTSCR1) and its paralog GLTSCR1-like
  form SWI/SNF chromatin remodeling subcomplexes.
authors:
- Alpsoy A
- Dykhuizen EC
journal: J Biol Chem
year: '2018'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC5858003
doi: 10.1074/jbc.RA117.001065
---

# Glioma tumor suppressor candidate region gene 1 (GLTSCR1) and its paralog GLTSCR1-like form SWI/SNF chromatin remodeling subcomplexes.
**Authors:** Alpsoy A, Dykhuizen EC
**Journal:** J Biol Chem (2018)
**DOI:** [10.1074/jbc.RA117.001065](https://doi.org/10.1074/jbc.RA117.001065)
**PMC:** [PMC5858003](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5858003/)

## Abstract

1. J Biol Chem. 2018 Mar 16;293(11):3892-3903. doi: 10.1074/jbc.RA117.001065.
Epub  2018 Jan 26.

Glioma tumor suppressor candidate region gene 1 (GLTSCR1) and its paralog 
GLTSCR1-like form SWI/SNF chromatin remodeling subcomplexes.

Alpsoy A(1), Dykhuizen EC(2).

Author information:
(1)From the Department of Medicinal Chemistry and Molecular Pharmacology, Purdue 
University, West Lafayette, Indiana 47907.
(2)From the Department of Medicinal Chemistry and Molecular Pharmacology, Purdue 
University, West Lafayette, Indiana 47907 edykhui@purdue.edu.

The mammalian SWI/SNF chromatin remodeling complex is a heterogeneous collection 
of related protein complexes required for gene regulation and genome integrity. 
It contains a central ATPase (BRM or BRG1) and various combinations of 10-14 
accessory subunits (BAFs for BRM/BRG1 Associated Factors). Two distinct 
complexes differing in size, BAF and the slightly larger polybromo-BAF (PBAF), 
share many of the same core subunits but are differentiated primarily by having 
either AT-rich interaction domain 1A/B (ARID1A/B in BAF) or ARID2 (in PBAF). 
Using density gradient centrifugation and immunoprecipitation, we have 
identified and characterized a third and smaller SWI/SNF subcomplex. We termed 
this complex GBAF because it incorporates two mutually exclusive paralogs, 
GLTSCR1 (glioma tumor suppressor candidate region gene 1) or GLTSCR1L 
(GLTSCR1-like), instead of an ARID protein. In addition to GLTSCR1 or GLTSCR1L, 
the GBAF complex contains BRD9 (bromodomain-containing 9) and the BAF subunits 
BAF155, BAF60, SS18, BAF53a, and BRG1/BRM. We observed that GBAF does not 
contain the core BAF subunits BAF45, BAF47, or BAF57. Even without these 
subunits, GBAF displayed in vitro ATPase activity and bulk chromatin affinity 
comparable to those of BAF. GBAF associated with BRD4, but, unlike BRD4, the 
GBAF component GLTSCR1 was not required for the viability of the LNCaP prostate 
cancer cell line. In contrast, GLTSCR1 or GLTSCR1L knockouts in the metastatic 
prostate cancer cell line PC3 resulted in a loss in proliferation and 
colony-forming ability. Taken together, our results provide evidence for a 
compositionally novel SWI/SNF subcomplex with cell type-specific functions.

© 2018 by The American Society for Biochemistry and Molecular Biology, Inc.

DOI: 10.1074/jbc.RA117.001065
PMCID: PMC5858003
PMID: 29374058 [Indexed for MEDLINE]

Conflict of interest statement: The authors declare that they have no conflicts 
of interest with the contents of this article

## Full Text

Abstract

The mammalian SWI/SNF chromatin remodeling complex is a heterogeneous collection of related protein complexes required for gene regulation and genome integrity. It contains a central ATPase (BRM or BRG1) and various combinations of 10–14 accessory subunits (BAFs for B RM/BRG1 A ssociated F actor s ). Two distinct complexes differing in size, BAF and the slightly larger polybromo-BAF (PBAF), share many of the same core subunits but are differentiated primarily by having either AT-rich interaction domain 1A/B (ARID1A/B in BAF) or ARID2 (in PBAF). Using density gradient centrifugation and immunoprecipitation, we have identified and characterized a third and smaller SWI/SNF subcomplex. We termed this complex GBAF because it incorporates two mutually exclusive paralogs, GLTSCR1 (glioma tumor suppressor candidate region gene 1) or GLTSCR1L (GLTSCR1-like), instead of an ARID protein. In addition to GLTSCR1 or GLTSCR1L, the GBAF complex contains BRD9 (bromodomain-containing 9) and the BAF subunits BAF155, BAF60, SS18, BAF53a, and BRG1/BRM. We observed that GBAF does not contain the core BAF subunits BAF45, BAF47, or BAF57. Even without these subunits, GBAF displayed in vitro ATPase activity and bulk chromatin affinity comparable to those of BAF. GBAF associated with BRD4, but, unlike BRD4, the GBAF component GLTSCR1 was not required for the viability of the LNCaP prostate cancer cell line. In contrast, GLTSCR1 or GLTSCR1L knockouts in the metastatic prostate cancer cell line PC3 resulted in a loss in proliferation and colony-forming ability. Taken together, our results provide evidence for a compositionally novel SWI/SNF subcomplex with cell type–specific functions.

Introduction

The mammalian SWI/SNF (or BAF) complex is an ATP-dependent chromatin remodeler composed of 10–14 subunits ( 1 ). The mammalian SWI/SNF chromatin remodeling complex is implicated in a variety of processes including mitosis, DNA replication, DNA damage repair, genomic looping, and gene splicing, in addition to its well-established roles in the transcriptional regulation of genes involved in cellular differentiation, cellular maintenance, and adaptation to stimuli ( 2 ). Mutations in specific SWI/SNF complex members are common in cancer ( 3 , 4 ) and neural disorders ( 5 ), and the altered expression of specific subunits is associated with tumorigenesis ( 6 ), viral infection ( 7 ), viral latency ( 8 ), alcohol addiction ( 9 , 10 ), heart disease ( 11 ), and immune function ( 12 ). The ability of this complex to direct such numerous and diverse functions is facilitated through the increase in SWI/SNF subunit number and diversity during vertebrate evolution ( 13 ), which led to an exponential increase in the potential combinations of subunits ( 14 , 15 ). All SWI/SNF complexes contain the ATPase subunit BRG1 or BRM, along with the structural subunits BAF155/BAF170, which are required for full ATPase and nucleosome remodeling activity in vitro ( 16 ). In addition, SWI/SNF complexes contain BAF60 (A, B, or C), BAF47, BAF57, BAF53 (A or B), and actin. The larger and less abundant polybromo-BAF (PBAF) 2 complex uniquely contains ARID2, PBRM1, BAF45D, and BRD7, whereas the more abundant BAF complex contains ARID1 (A or B), BAF45 (B, C, or D), SS18, BCL7 (A, B, or C), and BCL11 (A or B) ( Fig. 1 A ). The altered expression of SWI/SNF paralogs during cellular differentiation results in subunit switching, which is an important determinant of cell identity and cell-type transcriptional programs ( 17 ). Additionally, paralogs are often expressed simultaneously, leading to distinct subcomplexes within the same cell with both unique and redundant functions ( 18 ). For example, ARID1A is high in embryonic stem cells, whereas ARID1B is up-regulated upon differentiation ( 19 ). The different BAF complexes containing these two paralogs share many of their genomic targets; however, they also bind unique genomic targets, and deletions are non-synonymous for gene regulation ( 20 ). ARID1A is the most commonly mutated SWI/SNF subunit in cancer, because of transcriptional functions that are non-redundant with ARID1B ( 21 , 22 ); however, cancers with deletions in ARID1A are dependent on ARID1B for viability ( 23 ) because of redundant, essential functions at enhancers ( 22 ). Additionally, homologous complexes can display transcriptionally antagonistic roles, as has been observed for ARID1A and ARID2-containing complexes at specific gene targets ( 8 , 20 , 24 ). Targeting specific SWI/SNF complexes has been proposed both for alleviating subunit-specific pathogenic function and to target essential redundant functions in cancers with mutations in the genes for specific subunits ( 25 , 26 ). Both of these strategies are dependent on a better understanding of the different biochemical and transcriptional functions of homologous SWI/SNF complexes. We report here for the first time a novel, ubiquitously expressed SWI/SNF subcomplex defined by mutually exclusive paralogs GLTSCR1 (or BICRA for BRD4-interacting chromatin remodeling complex associated) and GLTSCR1L (or BICRAL for BRD4-interacting chromatin remodeling complex associated like), which also contains BRD9 and a subset of shared canonical SWI/SNF subunits.

Results

Proteomic analysis of BRG1 immunoprecipitations from two renal clear cell carcinoma cell lines identified multiple unique peptides from the uncharacterized protein GLTSCR1 ( Fig. 1 B ). GLTSCR1 has been identified in previous proteomic analyses of the SWI/SNF chromatin remodeling complex ( 18 , 27 – 29 ) but has never been validated or characterized as a BAF complex subunit. After screening multiple commercially available antibodies against GLTSCR1, we identified an antibody that stained a band in the predicted region of 180 kDa using immunoblot analysis. Further, this band disappeared after CRISPR-mediated Gltscr1 knockout in mouse embryonic stem cell lines ( Fig. 1 C ). Using this validated antibody, we confirmed the mass spectrometry data using immunoblot analysis, detecting robust enrichment of GLTSCR1 in BRG1 immunoprecipitations ( Fig. 1 D ). To define whether GLTSCR1 is a true subunit of BAF and not an associating factor, we performed urea denaturation followed by BRG1 immunoprecipitation and found that GLTSCR1 stably associates with BRG1 at urea concentrations up to 2.5 m , consistent with known BAF subunits ARID1A and BAF60A ( Fig. 1 D ).

To determine which SWI/SNF subcomplex contains GLTSCR1, we performed glycerol gradient analysis to separate the two closely related SWI/SNF complexes, BAF and PBAF, based on density. Surprisingly, GLTSCR1 staining was detected in earlier gradient fractions 11–13, which did not overlap with ARID1A (a subunit exclusive to the BAF complex) in fractions 14–16 or PBRM1 (a subunit exclusive to the PBAF complex) in fractions 17–19 or with BAF47 (a subunit shared by both BAF and PBAF) ( Fig. 2 A , left panel ). To ensure that this was not an aberrant partial complex caused by specific cancerous alterations or cell culture artifacts, we performed similar analysis in a second cell line (PC3) and observed the same pattern for GLTSCR1 ( Fig. 2 A , right panel ). To define whether any additional SWI/SNF subunits in addition to BRG1 associate with this subcomplex, we performed a series of immunoprecipitations to various known subunits of the BAF or PBAF complex ( Fig. 2 B ). From this panel, only antibodies against BRG1 and BRM were able to precipitate GLTSCR1, and as expected from the glycerol gradient analysis, GLTSCR1 did not associate with BAF-specific subunits ARID1A/ARID1B or PBAF-specific subunit PBRM1. Surprisingly, however, we did not observe GLTSCR1 association with BAF45D or BAF57, subunits thought to be canonical subunits, although we did observe association with core subunits BAF155 and BAF53a ( Fig. 2 C ). Using GLTSCR1 immunoprecipitations in HEK293T cells, we further identified that BAF60A, SS18, and BRD9 are GBAF subunits, whereas BAF170 and BAF47 are not ( Fig. 2 D ), which was confirmed in THP1 cells ( Fig. S1 A ). Further, glycerol gradient analysis and co-immunoprecipitation experiments identified BRD9 as a subunit of GBAF but not BAF or PBAF, SS18 as a subunit shared by BAF and GBAF, and BAF170 and BAF47 as subunits exclusive to BAF and PBAF ( Fig. 2 , E and F ). An illustration of the proposed composition of these complexes based on the immunoprecipitation experiments is depicted in Fig. 2 G . To validate GBAF as a potential chromatin remodeling complex, we next performed ATPase assays on immunoprecipitations of GLTSCR1 and BRG1 from HEK293T cells. We used GLTSCR1 immunoprecipitations containing comparable amounts of BRG1 (∼90%) ( Fig. 2 H and Fig. S1 B ) and found that GBAF complexes display robust DNA-stimulated ATPase activity ( Fig. 2 H ). In fact, the ATPase activity of GLTSCR1 immunoprecipitations was higher than BRG1 immunoprecipitations, although this is complicated by possible contributions from BRM, which is lowly expressed in HEK293T cells ( 30 ). We next used sequential salt extraction assays and determined that even in the absence of association with DNA-binding subunits BAF57 and ARID1/2, GLTSCR1 elutes from bulk chromatin with similar salt concentrations as BAF-specific subunit ARID1A, whereas PBAF-specific subunit PBRM1 requires slightly higher salt concentrations to elute off bulk chromatin ( 31 ) ( Fig. 2 I ).

Because we established GLTSCR1 as the unique subunit of GBAF, we set to define whether GLTSCR1 is required for GBAF formation. Using GLTSCR1 knockout ES cells ( Fig. 1 C ), we performed glycerol gradients with and without GLTSCR1 ( Fig. 3 A ). We observed a decrease in BRG1 and BAF60A staining in GBAF fractions 11–13 but not complete loss of staining. We hypothesized that this was due to the presence of the predicted GLTSCR1 paralog, GLTSCR1L (now referred to as BICRAL for clarity), which has also been detected in BAF subunit IP mass spectrometry studies as KIAA0240 ( 28 , 29 ). GLTSCR1 and BICRAL share 32% sequence homology (21% identity), and both contain a well-conserved “GLTSCR1” domain, which is also conserved between GLTSCR1 orthologs predicted in all multicellular organisms ( Fig. 3 B ). We screened commercially available antibodies for BICRAL and identified one with weak staining at the predicted size of 140 kDa, along with many nonspecific bands. To confirm that the band is the correct protein, we developed a cell line with doxycycline-inducible FLAG-tagged BICRAL. Overexpression of BICRAL-FLAG in HEK293T cells resulted in a robust FLAG band at 140 kDa and an increase in staining with the endogenous antibody at the same molecular weight ( Fig. 3 C ). To confirm that BICRAL is mutually exclusive with GLTSCR1 in the GBAF complex, we performed co-immunoprecipitations of GLTSCR1 and FLAG in our BICRAL-FLAG overexpression system and using endogenous proteins in HEK293T cells and found that GLTSCR1 and BICRAL do not associate with each other ( Fig. 3 , D and E ). In addition, both GLTSCR1 and BICRAL-FLAG enrich BRG1, BAF53A, and BRD9 but not ARID1A, indicating incorporation into comparable SWI/SNF subcomplexes. Intriguingly, we also found that overexpression of BICRAL decreases GLTSCR1 expression, possibly indicating its ability to compete with and replace GLTSCR1 in GBAF complexes.

To further investigate the role of BICRAL in GBAF formation, we performed glycerol gradient analysis of BICRAL-FLAG overexpression in HEK293T cells. We found that BICRAL overexpression results in BICRAL incorporation into GBAF, as indicated by its expression in fractions 11–13, similar to the profile of GLTSCR1 staining ( Fig. 4 A ). In addition, we confirmed that it is able to replace GLTSCR1 in GBAF, as indicated by an overall decrease in GLTSCR1 staining. Interestingly, we also saw an increase in BRG1 staining in fractions 11–13 upon BICRAL overexpression, indicating that BICRAL is able to alter overall SWI/SNF complex stoichiometry. To investigate this further, we performed BRG1 immunoprecipitations in the BICRAL overexpression cells. We found no changes in BRG1 expression or immunoprecipitation efficiency and confirmed the decrease in GLTSCR1 association with BRG1. In addition, we observed an increase in both the expression and the BRG1 association of BRD9 and a decrease in the BRG1 association with BAF subunits ARID1A, BAF47, and BAF57 ( Fig. 4 B ). To test whether this effect is due to a transcriptional or post-translational outcome of BICRAL expression, we performed RT-qPCR in BICRAL-FLAG overexpression line and found no alterations in endogenous transcript levels for any of the subunits in question ( Fig. 4 C ). This suggests that increased BRD9 protein levels and decreased GLTSCR1 levels are due to post-translational events, most likely degradation of free monomer. To confirm that GBAF is dependent on GLTSCR1 or BICRAL for formation, we also generated a double knockout HEK293T cell line. We observe that BRG1-associated BRD9 is undetectable in GLTSCR1 knockout and double knockout cells, indicating loss of GBAF formation ( Fig. 4 D ). It is worth noting that differences in knockout efficiencies and possibly the relative levels of GLTSCR1 and BICRAL made it difficult to distinguish additive or GLTSCR1-dominant effects of the paralogs on GBAF formation. Similar to decreased GLTSCR1 levels in BICRAL-overexpression lines, we consistently observed an increase in BICRAL levels upon GLTSCR1 knockout ( Fig. 4 D and Fig. S3 B ) via a similar increase of protein stability through complex incorporation. This provides further evidence for a compensatory role of BICRAL for GBAF formation in the absence of GLTSCR1. These results indicate that GLTSCR1/BICRAL are mutually exclusive subunits of GBAF that can, in part, define SWI/SNF complex stoichiometry.

GLTSCR1 has also been identified in a proteomics study of BRD4-associating factors ( 32 ), as reflected by the recent change in HUGO gene name from GLTSCR1 to BICRA for B RD4- I nteracting C hromatin R emodeling C omplex A ssociated protein. The BRD4 extraterminal domain was found to associate with several proteins, including NSD3 (and NSD2), ATAD5, GLTSCR1, and CHD4 (and CHD7), in an extraterminal domain-specific manner ( 33 ). Using BRD4 immunoprecipitations we confirmed that BRD4 associates with GLTSCR1, BAF155, BRD9, and BAF60A but not BAF-specific subunit BAF47 ( Fig. 5 A ). Because BRD4 protein association and co-regulator activity are known to be regulated by phosphorylation ( 34 ), we treated BRD4 immunoprecipitation with alkaline phosphatase (or used phosphatase inhibitors in lysis, IP, and washes) but did not find that the association between GLTSCR1 and BRD4 is dependent on phosphorylation. We confirmed previously published findings that the androgen-sensitive prostate cancer cell line LNCaP is 10-fold more sensitive to BRD4 inhibition than androgen-insensitive prostate cancer cell line PC3 ( Fig. S2 A ) ( 35 ); however, LNCaPs are not dependent on GLTSCR1 for viability ( Fig. 5 B and Fig. S2 B ). Instead, GLTSCR1 knockout produces a small but significant increase in sensitivity to BRD4 inhibitor ( Fig. 5 C and Fig. S2 C ). To test the effect of GLTSCR1 on expression of well-characterized BRD4 target MYCC , we measured MYCC mRNA levels in the GLTSCR1 knockout in LNCaP cells and found an increase in MYC levels, which was reversed upon low dose (50 n m ) treatment with JQ1 ( Fig. 5 D ). This provides evidence that GLTSCR1 only slightly modulates BRD4 function in LNCaP cells, potentially by sequestering it from transcriptional activators such as NSD3 that are required for the activation of MYCC transcription by BRD4 ( 36 ).

We next performed immunoblot analysis to evaluate the expression levels of BICRAL and GLTSCR1 in a series of cell lines. We found that the majority of cell lines have similar expression of these subunits ( Fig. 6 A ). Gltscr1 knockout ( Fig. 1 C ) in mouse ESCs or epithelial cell line NMuMG ( Fig. S3 A ) did not affect cell viability ( Fig. 6 B ). In addition, we observed no support for GLTSCR1 as a glioma tumor suppressor because GLTSCR1 knockout in human astrocyte cell line SVG p12 and glioblastoma cell line T98G both resulted in no change in viability ( Fig. 6 C and Fig. S3 B ). Lastly, we did not find evidence that BICRAL and GLTSCR1 have redundant, necessary functions because knockout of both GLTSCR1 and BICRAL in HEK293T cells did not produce any viability defect ( Figs. 4 D and 6 D ). We did, however, detect a dramatic decrease in both proliferation and colony formation upon GLTSCR1 knockout in prostate cancer cell line PC3 ( Fig. 6 , E and F ). We further knocked out BICRAL in this cell line and found similar defects in cell growth, indicating an overall dependence on GBAF function in this cell line ( Fig. 6 , E and F ). Although PC3 cells are dependent on GLTSCR1 and BICRAL, they have low expression of BRD9 (data not shown), and are not responsive to the BRD9 inhibitor BI-7273 ( Fig. 6 G ), ( 37 ), indicating that GLTSCR1 function is not always dependent on, or synonymous with, BRD9 function.

Discussion

SWI/SNF complexes play diverse roles in normal function and disease; however, most of our understanding of SWI/SNF function is from studying the ATPase subunit BRG1, which is found in multiple different SWI/SNF subcomplexes. The fact that many of the disease-related mutations are in subcomplex-specific subunits has placed increased importance in defining the composition and function of individual SWI/SNF subcomplexes. Our discovery of the ubiquitous new subcomplex GBAF, which is defined by novel subunit paralogs GLTSCR1 and BICRAL, provides another potential mechanism by which BRG1 exerts its functions. We have identified GBAF as a ubiquitously expressed SWI/SNF subcomplex with only a subset of the canonical SWI/SNF subunits but full in vitro ATPase activity. Gene and protein expression data indicate that these paralogs are expressed ubiquitously ( 30 ); however, knockout in many cell lines provides no immediate viability phenotype. Although this complex does not appear to be generally essential for basic cellular viability, mouse knockout data report an embryonic lethal phenotype for Bicral knockout animals ( 38 ). Whether these developmental roles will be shared with Gltscr1 remains to be seen.

In contrast to the high mutation rates for subunits of the BAF and PBAF complex, subunits of GBAF (with the exception of BRG1) are not highly mutated in cancer ( 39 ). Nevertheless, our data in metastatic prostate cancer cell line PC3 suggest a possible dependence of select cancers on GLTSCR1 and/or BICRAL. Intriguingly, prostate cancers have BRG1 up-regulation, but not SNF5, and display dependences on BRG1 ( 40 ). In addition to prostate cancer, many other cancers display increased dependence on BRG1, although the associated SWI/SNF subcomplex involved in this dependence is unexplored. Inhibitors to SWI/SNF complexes have been proposed as therapies; however, inhibitors of BRG1 ATPase activity will likely have severe toxicity because of the role of BRG1 in general viability in many cell types. Therefore, the development of inhibitors to GBAF subunits may be a more promising approach. Malignant rhabdoid tumors with mutations in SNF5 are dependent on BRG1 ( 41 ), and recent reports of malignant rhabdoid tumor sensitivity to BRD9 inhibitors ( 42 ) might be due to dependence of these cancers on GBAF function, although it is possible that BRD9 can have functions outside of GBAF. Similarly, AML is dependent on SWI/SNF subunits consistent with GBAF ( 43 ), including BRD9, ( 37 , 44 ) providing a potential therapeutic target in these cancers.

Our results also indicate a potential unexplored role for GBAF in BRD4-dependent function. Although several studies have noted the association between BRD4 and BRG1 ( 45 ), it has not been clear how they might be functionally related in cancer. For example, AML is dependent on both BRD4 ( 46 , 47 ) and BRG1 ( 43 , 47 ); however, their roles in AML transcriptional regulation are very different, making it difficult to determine the functional relevance of this association. We find that the association between BRD4 and BRG1 is specific to GLTSCR1, which will provide a framework for deciphering the functional relevance of this association in both normal and cancer settings. Further defining the importance of the association between GLTSCR1 and BRD4, as well as defining the general contribution of GBAF in chromatin targeting, nucleosome remodeling, and transcriptional regulation, will be critical for defining its contribution to human development and disease.
