---
pmid: '30111894'
title: PLD3 and PLD4 are single-stranded acid exonucleases that regulate endosomal
  nucleic-acid sensing.
authors:
- Gavin AL
- Huang D
- Huber C
- Mårtensson A
- Tardif V
- Skog PD
- Blane TR
- Thinnes TC
- Osborn K
- Chong HS
- Kargaran F
- Kimm P
- Zeitjian A
- Sielski RL
- Briggs M
- Schulz SR
- Zarpellon A
- Cravatt B
- Pang ES
- Teijaro J
- de la Torre JC
- O'Keeffe M
- Hochrein H
- Damme M
- Teyton L
- Lawson BR
- Nemazee D
journal: Nat Immunol
year: '2018'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC6105523
doi: 10.1038/s41590-018-0179-y
---

# PLD3 and PLD4 are single-stranded acid exonucleases that regulate endosomal nucleic-acid sensing.
**Authors:** Gavin AL, Huang D, Huber C, Mårtensson A, Tardif V, Skog PD, Blane TR, Thinnes TC, Osborn K, Chong HS, Kargaran F, Kimm P, Zeitjian A, Sielski RL, Briggs M, Schulz SR, Zarpellon A, Cravatt B, Pang ES, Teijaro J, de la Torre JC, O'Keeffe M, Hochrein H, Damme M, Teyton L, Lawson BR, Nemazee D
**Journal:** Nat Immunol (2018)
**DOI:** [10.1038/s41590-018-0179-y](https://doi.org/10.1038/s41590-018-0179-y)
**PMC:** [PMC6105523](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6105523/)

## Abstract

1. Nat Immunol. 2018 Sep;19(9):942-953. doi: 10.1038/s41590-018-0179-y. Epub 2018
 Aug 13.

PLD3 and PLD4 are single-stranded acid exonucleases that regulate endosomal 
nucleic-acid sensing.

Gavin AL(1), Huang D(1), Huber C(1)(2), Mårtensson A(1)(3), Tardif V(1)(4), Skog 
PD(1), Blane TR(1), Thinnes TC(1), Osborn K(5), Chong HS(5), Kargaran F(5), Kimm 
P(1), Zeitjian A(5), Sielski RL(5), Briggs M(5), Schulz SR(6), Zarpellon A(1), 
Cravatt B(7), Pang ES(8), Teijaro J(1), de la Torre JC(1), O'Keeffe M(8), 
Hochrein H(9), Damme M(10), Teyton L(1), Lawson BR(1), Nemazee D(11).

Author information:
(1)The Department of Immunology and Microbiology, The Scripps Research 
Institute, La Jolla, CA, USA.
(2), Bottmingen, Switzerland.
(3)Sophiris Bio, La Jolla, CA, USA.
(4)Department of Medicine, Division of Infectious Diseases and HIV Medicine, 
Drexel University, Philadelphia, PA, USA.
(5)The University of California, San Diego, La Jolla, CA, USA.
(6)Division of Molecular Immunology, University of Erlangen-Nürnberg, Erlangen, 
Germany.
(7)The Department of Chemical Biology, The Scripps Research Institute, La Jolla, 
CA, USA.
(8)Biomedicine Discovery Institute, Monash University, Clayton, Victoria, 
Australia.
(9)Bavarian Nordic, Martinsried, Germany.
(10)Biochemisches Institut, Christian-Albrechts-Universität, Kiel, Germany.
(11)The Department of Immunology and Microbiology, The Scripps Research 
Institute, La Jolla, CA, USA. nemazee@scripps.edu.

The sensing of microbial genetic material by leukocytes often elicits beneficial 
pro-inflammatory cytokines, but dysregulated responses can cause severe 
pathogenesis. Genome-wide association studies have linked the gene encoding 
phospholipase D3 (PLD3) to Alzheimer's disease and have linked PLD4 to 
rheumatoid arthritis and systemic sclerosis. PLD3 and PLD4 are endolysosomal 
proteins whose functions are obscure. Here, PLD4-deficient mice were found to 
have an inflammatory disease, marked by elevated levels of interferon-γ (IFN-γ) 
and splenomegaly. These phenotypes were traced to altered responsiveness of 
PLD4-deficient dendritic cells to ligands of the single-stranded DNA sensor 
TLR9. Macrophages from PLD3-deficient mice also had exaggerated TLR9 responses. 
Although PLD4 and PLD3 were presumed to be phospholipases, we found that they 
are 5' exonucleases, probably identical to spleen phosphodiesterase, that break 
down TLR9 ligands. Mice deficient in both PLD3 and PLD4 developed lethal liver 
inflammation in early life, which indicates that both enzymes are needed to 
regulate inflammatory cytokine responses via the degradation of nucleic acids.

DOI: 10.1038/s41590-018-0179-y
PMCID: PMC6105523
PMID: 30111894 [Indexed for MEDLINE]

Conflict of interest statement: Declaration of interests The authors have no 
competing interests

## Full Text

IntroductionLeukocytes carry conserved sensors for endocytosed DNA and RNA that trigger the production of proinflammatory cytokines. The ensuing inflammation can be beneficial for host defense to microorganisms, viruses, and tumors, but can also be pathogenic. Host recognition of nucleic acids triggers production of the type I interferons (IFN), IFN-α and IFN-β, and inflammatory cytokines such as interleukin-6 (IL-6) and IL-121, 2. Viral, bacterial or host DNA and RNA can be recognized depending upon sequence motifs, concentration, intracellular localization, and rate of degradation. One class of sensors resides in endosomes and includes TLRs 3, 7, 8, 9, and 13, which recognize, respectively, double-stranded (ds) RNA, ssRNA, ssRNA, ssDNA carrying unmethylated CpG motifs, and bacterial 23S RNA. A second sensor class resides in the cytoplasm. The RIG-I-like receptors (RLRs) recognize uncapped dsRNA, whereas the cyclic GMP-AMP synthase (cGAS) and Absent in Melanoma 2 (AIM2) are cytoplasmic DNA sensors. These sensors are believed to preferentially recognize foreign genetic material because host DNA and RNA often have distinct sequence abundances and modifications, are localized away from the sensors, or are degraded in a more efficient or organized manner.Evidence is mounting that regulation of nucleic acid abundance by nucleases prevents autoinflammatory and autoimmune diseases driven by nucleic acid sensors3. Deficiency in Dnase1 or Dnase1L3 leads to systemic lupus erythematosus in both humans and mice4,5,6. In mice, Dnase2a deficiency leads to a lethal IFN-mediated disease, while patients deficient for DNASE2a exhibit an autoinflammatory anemia, kidney and joint disease7,8. Patients with TREX1 deficiency develop Aicardi-Goutières syndrome, whilst Trex1-deficient mice develop a lethal cardiomyopathy inflammatory disease9,10,11. The prevention of abnormal accumulation of host nucleic acid in differing tissues and cellular compartments is thus performed by an array of enzymes with non-redundant functions.Phospholipase D family proteins share a catalytic site sequence signature (HxxKxxxxD) that is found in PLD1 and PLD2 cytosolic phospholipases, but most family members have distinct functions and intracellular localization12, 13, 14. PLD3 and PLD4 are glycosylated type II transmembrane proteins localized in endolysosomes15, 16, 17, residing in subcellular compartments distinct from most other PLD family members. Studies have linked PLD4 polymorphisms to systemic sclerosis and rheumatoid arthritis18, 19, 20. Several works link PLD3 to Alzheimer’s disease21,22 with some controversy23, 24. The functions of PLD3 and PLD4 are unknown. Despite their names, PLD3 and PLD4 have not been demonstrated to have phospholipase activity25, 26. PLD4 is highly expressed in dendritic cells (DCs) and other myeloid cells, and in lower amounts in B cells. PLD3 is structurally similar to PLD4, but more broadly expressed. We find that PLD3 and PLD4 are 5′ exonucleases with properties strikingly similar to the enzymatic activity described many years ago as spleen acid exonuclease or phosphodiesterase27, 28. Mutants in Pld3 and Pld4 genes have heightened sensitivity to, and diminished turnover of, ligands of TLR9.

Introduction

Leukocytes carry conserved sensors for endocytosed DNA and RNA that trigger the production of proinflammatory cytokines. The ensuing inflammation can be beneficial for host defense to microorganisms, viruses, and tumors, but can also be pathogenic. Host recognition of nucleic acids triggers production of the type I interferons (IFN), IFN-α and IFN-β, and inflammatory cytokines such as interleukin-6 (IL-6) and IL-121, 2. Viral, bacterial or host DNA and RNA can be recognized depending upon sequence motifs, concentration, intracellular localization, and rate of degradation. One class of sensors resides in endosomes and includes TLRs 3, 7, 8, 9, and 13, which recognize, respectively, double-stranded (ds) RNA, ssRNA, ssRNA, ssDNA carrying unmethylated CpG motifs, and bacterial 23S RNA. A second sensor class resides in the cytoplasm. The RIG-I-like receptors (RLRs) recognize uncapped dsRNA, whereas the cyclic GMP-AMP synthase (cGAS) and Absent in Melanoma 2 (AIM2) are cytoplasmic DNA sensors. These sensors are believed to preferentially recognize foreign genetic material because host DNA and RNA often have distinct sequence abundances and modifications, are localized away from the sensors, or are degraded in a more efficient or organized manner.

Evidence is mounting that regulation of nucleic acid abundance by nucleases prevents autoinflammatory and autoimmune diseases driven by nucleic acid sensors3. Deficiency in Dnase1 or Dnase1L3 leads to systemic lupus erythematosus in both humans and mice4,5,6. In mice, Dnase2a deficiency leads to a lethal IFN-mediated disease, while patients deficient for DNASE2a exhibit an autoinflammatory anemia, kidney and joint disease7,8. Patients with TREX1 deficiency develop Aicardi-Goutières syndrome, whilst Trex1-deficient mice develop a lethal cardiomyopathy inflammatory disease9,10,11. The prevention of abnormal accumulation of host nucleic acid in differing tissues and cellular compartments is thus performed by an array of enzymes with non-redundant functions.

Phospholipase D family proteins share a catalytic site sequence signature (HxxKxxxxD) that is found in PLD1 and PLD2 cytosolic phospholipases, but most family members have distinct functions and intracellular localization12, 13, 14. PLD3 and PLD4 are glycosylated type II transmembrane proteins localized in endolysosomes15, 16, 17, residing in subcellular compartments distinct from most other PLD family members. Studies have linked PLD4 polymorphisms to systemic sclerosis and rheumatoid arthritis18, 19, 20. Several works link PLD3 to Alzheimer’s disease21,22 with some controversy23, 24. The functions of PLD3 and PLD4 are unknown. Despite their names, PLD3 and PLD4 have not been demonstrated to have phospholipase activity25, 26. PLD4 is highly expressed in dendritic cells (DCs) and other myeloid cells, and in lower amounts in B cells. PLD3 is structurally similar to PLD4, but more broadly expressed. We find that PLD3 and PLD4 are 5′ exonucleases with properties strikingly similar to the enzymatic activity described many years ago as spleen acid exonuclease or phosphodiesterase27, 28. Mutants in Pld3 and Pld4 genes have heightened sensitivity to, and diminished turnover of, ligands of TLR9.

ResultsPLD4-deficient mice display a TLR9-driven inflammatory syndromeTo assess effects of PLD4 deficiency, a conditional knockout allele (Pld4fl) was engineered, from which we generated homozygous knockout (Pld4−/−) and conditional strains (Pld4fl/fl) (Supplementary Fig. 1a–c); Pld4−/− mice lacked detectable PLD4 protein (Supplementary Fig. 1d) and exhibited a phenotype of chronic immune activation, with splenomegaly (Fig. 1a) and elevated major histocompatibility class II (MHCII) expression on resident peritoneal macrophages (Fig. 1b,c), whereas CD86 was not elevated (Supplementary Fig. 1g). The MHCII upregulation was driven by IFN-γ produced by innate immune cells, as it occurred in Pld4−/−Rag1−/− mice (Fig. 1b,c), which lack B and T lymphocytes, but not in IFN-γ-deficient Ifng−/−Pld4−/− mice (Fig. 1c). Expression of MHCII by macrophages was not elevated in Pld4fl/flLysM-Cre mice, in which PLD4 deficiency was limited to macrophages, but was evident in Pld4fl/flCD11c-Cre mice, suggesting that PLD4-deficient CD11c+ DCs indirectly promoted MHCII upregulation (Fig. 1c). Plasma of Pld4−/− mice had significantly elevated concentrations of IFN-γ and CXCL10 (Fig. 1d,e). Pld4−/− mice had altered leukocyte subsets, including fewer natural killer (NK) cells, peritoneal B1 lymphocytes, and platelets, and more marginal zone (MZ) B cells and blood monocytes (Fig. 1f–j, Supplementary Fig. 1h). However, splenic DC numbers were similar to those of PLD4-sufficient mice (Supplementary Fig. 1e,f). Consistent with elevated IFN-γ concentrations, altered B1 and MZ B cell numbers seen in Pld4−/− mice were partly reversed in Ifng−/−Pld4−/− mice (Fig. 1f,g and Supplementary Fig. 1i). Moreover, Pld4−/− but not Ifng−/−Pld4−/− mice, were resistant to the induction of experimental autoimmune encephalomyelitis (EAE) (Supplementary Fig. 2), in which IFN-γ is protective29. RNA sequencing comparison of Pld4−/−Rag1−/− and Rag1−/− splenocytes revealed an IFN signature in Pld4−/− animals, as >40 of the 109 genes whose expression was significantly elevated were known to be inducible by IFN-γ or type I IFNs (Fig. 1k, Supplementary Table 1). Expression of IL12b, Cxcl9 and Il10 were also significantly elevated. Of the 33 down-regulated genes, some were known to be suppressed by IFN, but most were NK-specific genes, consistent with their lower numbers in the spleens of Pld4−/− mice and Pld4−/−Rag1−/− mice.Certain features of the phenotype of Pld4−/− mice resembled the human disease Macrophage Activation Syndrome, which can be mimicked in mice by repeated challenge with ligands for TLR930. To assess the role of TLR9 in the PLD4-deficient phenotype, we bred Tlr9−/−Pld4−/− mice. Notably, analysis of these mice indicated that all of the abnormalities of PLD4-deficient mice tested were dependent upon TLR9, including splenomegaly, MHCII upregulation, and alterations in numbers of monocytes, platelets, NK, and B1 cells (Fig. 1c,f–j). In contrast, combined deficiency in both PLD4 and IFN-γ reversed only a subset of phenotypes associated with PLD4 deficiency (Fig. 1c,f,g), suggesting that IFN-γ production by Pld4−/− mice was secondary to TLR9 triggering. Histological analysis of Pld4−/− livers revealed a TLR9-dependent increase in the frequency and size of CD68+ macrophages, consistent with mild liver inflammation (Fig. 2). Overall, these results suggested that the major effects of PLD4 deficiency stemmed from enhanced or dysregulated TLR9 recognition of ssDNA in endolysosomes.That Pld4−/− mice had excessive TLR9-driven IFN-γ production and Il12b mRNA suggested that activation of CD8+ DCs might be dysregulated in these mice. Secretion of IL-12 by CD8+ DCs is known to stimulate IFN-γ production, particularly by NK cells31. PLD4-deficient sorted splenic DCs were found to produce more IL-12p70 and IL-6 in response to two TLR9 ligands, VACV70 and 2216PS (an A-type CpG oligodeoxynucleotide (ODN)), compared to PLD4-sufficient controls, however responses to the B-type CpG ODN PS, were similar (Fig. 1l,m). 1668PS is fully modified by phosphorothioate (PS) linkages, 2216PS is partially modified, and VACV70 is unmodified dsDNA containing CpG motifs (Table 1). In addition, VACV70, but not 1668PS and 2216PS, elicited IFN-λ responses in CD8+ DCs through the STING cytoplasmic DNA sensing pathway, but did not differ between wild-type and PLD4-deficient cells (Fig. 1n, Supplementary Fig. 3a,b). IFN-γ signals are known to promote IL-12 responses32, however sorted CD8+ DCs isolated from Pld4−/−Ifng−/− mice also showed elevated IL-12 responses to TLR9 stimuli (Supplementary Fig. 3c). CD8−CD11bhi splenic DCs also had enhanced IL-6 and IL-12 responses, especially to VACV70 (Supplementary Fig. 3d). These results supported the notion that Pld4−/− CD8+ DCs had enhanced cytokine responses to at least some TLR9 ligands.PLD4 and PLD3 are single-stranded 5′ exonucleasesThese results suggested a role for PLD4 in the processing of TLR9 ligands. To test for nuclease activity, recombinant soluble mouse PLD4 was expressed by substituting a secretion signal peptide for the N-terminal transmembrane and cytoplasmic tail. Using DNA substrates shown in Fig. 3a, the activity of purified recombinant mouse PLD4 was then assessed (Fig. 3b). PLD4 degraded a 55 nt ssDNA (lane 2), but not a dsDNA version of the same sequence (lane 6). When paired with shorter ssDNAs complementary with the 5′ or 3′ ends, PLD4 degraded only substrate with an unpaired 5′ end (lane 4), indicating 5′-to-3′, but not 3′-to-5′, ssDNA exonuclease activity, and demonstrating a lack of endonuclease activity (lane 5). PLD4 protein carrying H-to-A mutations in both HKD motifs (PLD4-AA) lacked detectable enzymatic activity (lanes 7–11). PLD4 only cleaved substrates that lacked phosphorylated 5′ ends and was unaffected by 3′ phosphorylation (Fig. 3c, lanes 4,6). As predicted for an endosomal nuclease, the reaction was most efficient at pH<6, but was still detectable at pH 7 (Fig. 3d). Parallel experiments with mouse PLD3 revealed a similar profile of substrate specificity and an acidic pH optimum (Fig. 3e–g). We conclude that PLD3 and PLD4 are 5′ exonucleases that are able to cleave ssDNA.PLD3 and PLD4 appear to be classic spleen phosphodiesterasesThe enzymatic properties of PLD4 and PLD3 that we observed were reminiscent of the activity previously described as “spleen acid exonuclease” or “spleen phosphodiesterase”28, 33 (Enzyme Commission registry number 3.1.16.1). We therefore directly compared PLD3 and PLD4 with a commercial preparation of bovine spleen phosphodiesterase as a positive control using a diagnostic assay that measured their abilities to digest GpA or ApG dinucleotides (Fig. 3h). Digestion was readily tracked spectroscopically by the release and loss of adenosine, which is deaminated in a coupled assay34. Mouse and human recombinant soluble PLD3 and PLD4, but not PLD4-AA, specifically cleaved the phosphodiester bond between the phosphate and the second nucleotide, matching the activity described of bovine spleen exonuclease. Snake venom phosphodiesterase served as a control for possible cleavage at the upstream phosphodiester bond, leaving 5′ phosphorylated mononucleotides. The results indicated that digestion by PLD3 and PLD4 generated 3′ mononucleotides and nucleosides from short oligonucleotide substrates. The commercial Phosphodiesterase II (Methods) was further purified by size exclusion to reduce complexity of the sample and the enzymatically active fractions were subjected to trypsin-Liquid Chromatography-Mass Spectrometry (LC-MS) analysis. Six peptides unique to bovine PLD3 were identified (Supplementary Fig. 4a,b). Other proteins, however, were also identified (Supplementary Fig. 4c), consistent with low specific activity of the commercial preparation in the dinucleotide cleavage assay (Fig. 3h). We conclude that PLD3 and PLD4 have activities indistinguishable from classical spleen acid exonuclease.PLD3 and PLD4 can degrade PS-linked CpG agonists of TLR9 in vitroPhosphorothioate (PS) linkages are often used in synthetic ODNs to hinder their digestion by nucleases. CpG-B-type ODNs 2006PS and 1668PS are fully PS-modified. They stimulate TLR9 preferentially through the NF-κB pathway to produce cytokines such as IL-6 and IL-1235. CpG-A-type ODN 2216PS is a potent inducer of IFN-α and has an internal palindromic sequence composed of phosphodiester-linked nucleotides flanked by strings of PS-modified G nucleotides known to form unusual secondary structures35. Human and mouse PLD3 and PLD4 were able to digest B-type CpG ODNs 2006PS and 1668PS despite their PS linkages (Fig 3i), albeit between 20–50 times less efficiently than versions with phosphodiester bonds (Supplementary Fig. 5a,c), but failed to digest 2216 or 2216PS (Fig. 2i, right; Supplementary Fig. 5b,d). We conclude that PLD3 and PLD4 digest ssODNs, including those with phosphorothioate linkages, but are blocked by features of A-type CpG ODNs, likely their secondary structure.PLD3 and PLD4 limit cellular responses to exogenous TLR9 ligandsTo assess PLD3 function and potential redundancy with PLD4, we used CRISPR/Cas9 mutagenesis of zygotes to generate Pld3 strains lacking PLD3 protein (Supplementary Fig. 1j,k). Thioglycolate-elicited Pld3−/− macrophages expressed more IL-6 in response to 2216PS, relative to wild-type or Pld4−/− thioglycolate-elicited macrophages (Fig. 4a). The enhanced response was observed using macrophages from multiple founder lines, but not from wild-type or Pld3+/− littermates. In contrast, Pld3−/− macrophage responses to the TLR4 agonist lipopolysaccharide (LPS) were comparable to wild-type and Pld4−/− macrophages (Fig. 4b). RNA expression analysis revealed that wild-type thioglycolate-elicited macrophages highly expressed Pld3 and poorly expressed Pld4, whereas wild-type DCs showed the opposite pattern of expression (Fig. 4c,d). Pld3−/− mice lacked other phenotypes observed in Pld4−/− mice and had similar spleen size, splenic NK proportions, and resident peritoneal macrophage MHCII surface expression as their heterozygous littermate controls (Supplementary Fig. 1l–n) and FLT3L-DC responses were unchanged (Supplementary Fig. 6f). Functional redundancy and distinct expression profiles are thus consistent with the altered responses of Pld3−/− thioglycolate-elicited macrophages and of Pld4−/− CD8+ DCs.Recombinant PLD3 was unable to degrade the TLR9 ligand 2216PS in vitro (Fig. 3i, Supplementary Fig. 5d), so it was surprising that thioglycolate-elicited Pld3−/− macrophages had an enhanced IL-6 response to 2216PS (Fig. 4a,b). TLR9 responses to 2216PS require cleavage by DNase II in the palindromic phosphodiester-linked central region36. As DNAse II cleaves dsDNA or palindromic ssDNA, creating fragments with unphosphorylated 5′ ends and 3′ phosphates, we hypothesized that DNAse II-cleaved 2216PS fragments should be naturally degraded by PLD3 or PLD4. We therefore tested a series of synthesized fragments of 2216 or 2216PS (Fig. 5a) for their ability to stimulate FLT3L-cultured DCs and thioglycolate-elicited macrophages deficient in PLD3 or PLD4 (Fig. 5b–e). Pld4−/− DCs and Pld3−/− thioglycolate-elicited macrophages showed enhanced responses to these shorter ligands compared to PLD3/4-sufficient controls (Fig. 5b–e). When packaged in the transfection reagent Lyovec, the same ODN ligands gave similar strong responses in both wild-type and PLD4-deficient DCs (Supplementary Fig. 6a,b), suggesting that ligand stability rather than TLR9 signaling was affected by PLD4 deficiency and could be bypassed using this reagent. Interestingly, PLD3-deficient, thioglycolate-elicited macrophages still had enhanced responses to Lyovec complexes of 2216 subfragments, indicating some cell type-specific differences (Supplementary Fig. 6c). In both DCs and macrophages, responses to ODNs 1668PS and 2216PS were dependent on TLR9 (Fig. 5b–d). The enhanced bioactivity of particular CpG fragments in PLD3- or PLD4-deficient cells correlated with their sensitivity to PLD3 and PLD4 in vitro (Fig. 5f).To enable the analysis of the function of PLD3 and PLD4 in human cells, PLD3-deficient clones of the human cell line HEK-BluehTLR9 were generated by CRISPR/Cas9 treatment. Independent PLD3-deficient clones had greatly enhanced responses to 2216PS and its synthetic fragments, as well as a phosphodiester form of B-type CpG 2006 (Fig. 5g, Supplementary Fig. 6d–e). Reconstitution with PLD4 inhibited the response to baseline, whereas reconstitution with PLD4-AA did not (Fig. 5g), suggesting that nuclease activity was required to limit the response.Enhanced stability of TLR9 ligands in cells lacking PLD3 and PLD4To assess ODN degradation in vivo, we stimulated wild-type or mutant thioglycolate-elicited macrophages with 2216 carrying a 3′-biotin tag; after 3.5 or 8 hours, we recovered ODNs from cell lysates using streptavidin beads and visualized them by 32P end-labelling (Fig. 5h). The analysis revealed that the 2216-biotin fragments that demonstrated enhanced stimulatory activity in Pld3−/− macrophages and Pld4−/− DCs were less degraded in PLD3-deficient lysates than in those from control macrophages (Fig. 5h, asterisks), which had shorter species composed mainly of the poly-G 3′ end (<10 nt). The degradation pattern in Unc93b3d/3d macrophages, which lack signaling by TLR 3, 7 and 937, was identical to wild-type macrophages, indicating that a TLR9 signal is not required for enzymatic degradation by PLD3. Similar results were obtained in PLD3−/− HEK293 cells fed 2216-biotin, where DNA fragments of 10–15 nt were more abundant, and were specifically depleted in cells reconstituted with PLD4 or PLD3 but not PLD4-AA or PLD3-AA mutants (Fig. 5i). We conclude that PLD3 and PLD4 have redundant exonuclease function that can limit TLR9 responses to CpG ODNs, particularly those lacking phosphorothioate linkages, by reducing the concentration of ssDNA able to stimulate TLR9 and that these activities are similar in human and mouse.PLD3 and PLD4 prevent a lethal hepatic autoinflammatory diseaseThe overlapping enzymatic activities of PLD3 and PLD4 and their co-expression in macrophage subsets prompted us to interbreed these strains to generate animals deficient for both enzymes. No animals deficient for both genes survived to weaning age (Supplementary Fig. 7a). Pld3−/−Pld4−/− mice were identified of either sex, but these animals were undersized compared to littermate controls and died between 12 and 21 days of age with severe liver inflammation (Fig 6a–d, Supplementary Fig. 7b,c). The livers from Pld3−/−Pld4−/− animals showed macroscopic white areas of necrosis (Supplementary Fig. 7c), with regions of vesicular steatosis, hemophagocytosis, and multinucleated hepatocytes (Fig. 6a–c, respectively). Importantly, only a single allele of either Pld3 or Pld4 was required to allow survival and to prevent the substantial liver pathology (Fig. 6d and Supplementary Table 2). Although CD68+ myeloid cells in the livers from 16 day old Pld3+/−Pld4−/− animals were more prevalent than in Pld3+/−Pld4+/− or Pld3−/−Pld4+/− littermates, livers from Pld3−/−Pld4−/− animals demonstrated extensive infiltration of CD68+ myeloid cells (Fig. 6d). Sera from Pld3−/−Pld4−/− animals revealed elevated concentrations of ferritin, IL-6, TNF, IFN-γ, CXCL10, MCP3, MCP1, and IL-10 compared to controls of the same age, whereas IFN-α and IL-1α were not significantly elevated (Fig 6e,f). We infer that neonatal mice require PLD3 and PLD4 to prevent an inflammatory disease consistent with hemophagocytic lymphohistiocytosis (HLH).Analysis of TLR9 ligand responses of macrophages grown from Pld3−/−Pld4−/− bone marrow revealed elevated IL-6 secretion to 2216 subfragments, VACV70, and 1668 (DNA form), but responses to 1668PS, R848 and LPS were similar to wild-type or single deficient controls (Fig. 6g). As both Pld3 and Pld4 genes are expressed in bone marrow derived macrophages (Fig. 4c), the enhanced responses to particular TLR9 ligands occurred when both genes were deficient, but not when either gene was present (Fig. 6g). Pld3−/−Pld4−/− bone marrow failed to generate dendritic cells in FLT3L culture, suggesting that either the on-going inflammation was affecting DC precursors in the bone marrow or that both proteins are required for in vitro FLT3L-driven DC development.PLD3 and PLD4 in hematopoietic cells prevent autoinflammationTo determine whether the immune system itself was the driver of lethal inflammation in Pld3−/−Pld4−/− mice, bone marrow transplantation into lethally irradiated 6 week old wild-type CD45.1+ recipients was performed using 19 day littermate Pld3+/−Pld4−/−, Pld3−/−Pld4+/− or Pld3−/−Pld4−/− bone marrow as donors. Eight weeks post transfer, blood analysis revealed low platelet counts, anemia, elevated reticulocytes and leukopenia in the recipients receiving Pld3−/−Pld4−/− bone marrow, but not in recipients transplanted with littermate bone marrow containing a single wild-type allele of either Pld3 or Pld4 (Fig. 7a). Despite overt splenomegaly, the number of splenic leukocytes recovered after erythrocyte lysis was lower in Pld3−/−Pld4−/− bone marrow recipients compared to controls (Fig. 7b). Flow cytometric analysis of CD45.2+ splenocytes revealed a marked reduction in B cell proportions and numbers when both Pld3 and Pld4 were absent (Fig. 7c,d). The total numbers of TCRβ+ CD4+ and CD8+ T cells were also reduced in the spleens of Pld3−/−Pld4−/− bone marrow recipients compared to controls (Fig. 7e), however the majority of CD8+ T cells exhibited an activated phenotype (CD44hiCD62Llo) and very few naïve CD8+ T cells (CD44loCD62L+) were present (Fig. 7f,g). Consistent with disease observed in Pld3−/−Pld4−/− mice, recipients of Pld3−/−Pld4−/− bone marrow had elevated expression of MHCII (with CD86 expression similar to controls) on peritoneal macrophages, elevated blood monocytes, and elevated concentrations of serum cytokines CXCL10, IFN-γ, MCP3, IL-6, but IFN-α was not detected (Supplementary Fig. 7d–k). We conclude that both PLD3 and PLD4 limit autoinflammation, and that a single wild-type allele of either can prevent the development of the inflammatory disease transferred by hematopoietic stem cells.

PLD4-deficient mice display a TLR9-driven inflammatory syndromeTo assess effects of PLD4 deficiency, a conditional knockout allele (Pld4fl) was engineered, from which we generated homozygous knockout (Pld4−/−) and conditional strains (Pld4fl/fl) (Supplementary Fig. 1a–c); Pld4−/− mice lacked detectable PLD4 protein (Supplementary Fig. 1d) and exhibited a phenotype of chronic immune activation, with splenomegaly (Fig. 1a) and elevated major histocompatibility class II (MHCII) expression on resident peritoneal macrophages (Fig. 1b,c), whereas CD86 was not elevated (Supplementary Fig. 1g). The MHCII upregulation was driven by IFN-γ produced by innate immune cells, as it occurred in Pld4−/−Rag1−/− mice (Fig. 1b,c), which lack B and T lymphocytes, but not in IFN-γ-deficient Ifng−/−Pld4−/− mice (Fig. 1c). Expression of MHCII by macrophages was not elevated in Pld4fl/flLysM-Cre mice, in which PLD4 deficiency was limited to macrophages, but was evident in Pld4fl/flCD11c-Cre mice, suggesting that PLD4-deficient CD11c+ DCs indirectly promoted MHCII upregulation (Fig. 1c). Plasma of Pld4−/− mice had significantly elevated concentrations of IFN-γ and CXCL10 (Fig. 1d,e). Pld4−/− mice had altered leukocyte subsets, including fewer natural killer (NK) cells, peritoneal B1 lymphocytes, and platelets, and more marginal zone (MZ) B cells and blood monocytes (Fig. 1f–j, Supplementary Fig. 1h). However, splenic DC numbers were similar to those of PLD4-sufficient mice (Supplementary Fig. 1e,f). Consistent with elevated IFN-γ concentrations, altered B1 and MZ B cell numbers seen in Pld4−/− mice were partly reversed in Ifng−/−Pld4−/− mice (Fig. 1f,g and Supplementary Fig. 1i). Moreover, Pld4−/− but not Ifng−/−Pld4−/− mice, were resistant to the induction of experimental autoimmune encephalomyelitis (EAE) (Supplementary Fig. 2), in which IFN-γ is protective29. RNA sequencing comparison of Pld4−/−Rag1−/− and Rag1−/− splenocytes revealed an IFN signature in Pld4−/− animals, as >40 of the 109 genes whose expression was significantly elevated were known to be inducible by IFN-γ or type I IFNs (Fig. 1k, Supplementary Table 1). Expression of IL12b, Cxcl9 and Il10 were also significantly elevated. Of the 33 down-regulated genes, some were known to be suppressed by IFN, but most were NK-specific genes, consistent with their lower numbers in the spleens of Pld4−/− mice and Pld4−/−Rag1−/− mice.Certain features of the phenotype of Pld4−/− mice resembled the human disease Macrophage Activation Syndrome, which can be mimicked in mice by repeated challenge with ligands for TLR930. To assess the role of TLR9 in the PLD4-deficient phenotype, we bred Tlr9−/−Pld4−/− mice. Notably, analysis of these mice indicated that all of the abnormalities of PLD4-deficient mice tested were dependent upon TLR9, including splenomegaly, MHCII upregulation, and alterations in numbers of monocytes, platelets, NK, and B1 cells (Fig. 1c,f–j). In contrast, combined deficiency in both PLD4 and IFN-γ reversed only a subset of phenotypes associated with PLD4 deficiency (Fig. 1c,f,g), suggesting that IFN-γ production by Pld4−/− mice was secondary to TLR9 triggering. Histological analysis of Pld4−/− livers revealed a TLR9-dependent increase in the frequency and size of CD68+ macrophages, consistent with mild liver inflammation (Fig. 2). Overall, these results suggested that the major effects of PLD4 deficiency stemmed from enhanced or dysregulated TLR9 recognition of ssDNA in endolysosomes.That Pld4−/− mice had excessive TLR9-driven IFN-γ production and Il12b mRNA suggested that activation of CD8+ DCs might be dysregulated in these mice. Secretion of IL-12 by CD8+ DCs is known to stimulate IFN-γ production, particularly by NK cells31. PLD4-deficient sorted splenic DCs were found to produce more IL-12p70 and IL-6 in response to two TLR9 ligands, VACV70 and 2216PS (an A-type CpG oligodeoxynucleotide (ODN)), compared to PLD4-sufficient controls, however responses to the B-type CpG ODN PS, were similar (Fig. 1l,m). 1668PS is fully modified by phosphorothioate (PS) linkages, 2216PS is partially modified, and VACV70 is unmodified dsDNA containing CpG motifs (Table 1). In addition, VACV70, but not 1668PS and 2216PS, elicited IFN-λ responses in CD8+ DCs through the STING cytoplasmic DNA sensing pathway, but did not differ between wild-type and PLD4-deficient cells (Fig. 1n, Supplementary Fig. 3a,b). IFN-γ signals are known to promote IL-12 responses32, however sorted CD8+ DCs isolated from Pld4−/−Ifng−/− mice also showed elevated IL-12 responses to TLR9 stimuli (Supplementary Fig. 3c). CD8−CD11bhi splenic DCs also had enhanced IL-6 and IL-12 responses, especially to VACV70 (Supplementary Fig. 3d). These results supported the notion that Pld4−/− CD8+ DCs had enhanced cytokine responses to at least some TLR9 ligands.

PLD4-deficient mice display a TLR9-driven inflammatory syndrome

To assess effects of PLD4 deficiency, a conditional knockout allele (Pld4fl) was engineered, from which we generated homozygous knockout (Pld4−/−) and conditional strains (Pld4fl/fl) (Supplementary Fig. 1a–c); Pld4−/− mice lacked detectable PLD4 protein (Supplementary Fig. 1d) and exhibited a phenotype of chronic immune activation, with splenomegaly (Fig. 1a) and elevated major histocompatibility class II (MHCII) expression on resident peritoneal macrophages (Fig. 1b,c), whereas CD86 was not elevated (Supplementary Fig. 1g). The MHCII upregulation was driven by IFN-γ produced by innate immune cells, as it occurred in Pld4−/−Rag1−/− mice (Fig. 1b,c), which lack B and T lymphocytes, but not in IFN-γ-deficient Ifng−/−Pld4−/− mice (Fig. 1c). Expression of MHCII by macrophages was not elevated in Pld4fl/flLysM-Cre mice, in which PLD4 deficiency was limited to macrophages, but was evident in Pld4fl/flCD11c-Cre mice, suggesting that PLD4-deficient CD11c+ DCs indirectly promoted MHCII upregulation (Fig. 1c). Plasma of Pld4−/− mice had significantly elevated concentrations of IFN-γ and CXCL10 (Fig. 1d,e). Pld4−/− mice had altered leukocyte subsets, including fewer natural killer (NK) cells, peritoneal B1 lymphocytes, and platelets, and more marginal zone (MZ) B cells and blood monocytes (Fig. 1f–j, Supplementary Fig. 1h). However, splenic DC numbers were similar to those of PLD4-sufficient mice (Supplementary Fig. 1e,f). Consistent with elevated IFN-γ concentrations, altered B1 and MZ B cell numbers seen in Pld4−/− mice were partly reversed in Ifng−/−Pld4−/− mice (Fig. 1f,g and Supplementary Fig. 1i). Moreover, Pld4−/− but not Ifng−/−Pld4−/− mice, were resistant to the induction of experimental autoimmune encephalomyelitis (EAE) (Supplementary Fig. 2), in which IFN-γ is protective29. RNA sequencing comparison of Pld4−/−Rag1−/− and Rag1−/− splenocytes revealed an IFN signature in Pld4−/− animals, as >40 of the 109 genes whose expression was significantly elevated were known to be inducible by IFN-γ or type I IFNs (Fig. 1k, Supplementary Table 1). Expression of IL12b, Cxcl9 and Il10 were also significantly elevated. Of the 33 down-regulated genes, some were known to be suppressed by IFN, but most were NK-specific genes, consistent with their lower numbers in the spleens of Pld4−/− mice and Pld4−/−Rag1−/− mice.

Certain features of the phenotype of Pld4−/− mice resembled the human disease Macrophage Activation Syndrome, which can be mimicked in mice by repeated challenge with ligands for TLR930. To assess the role of TLR9 in the PLD4-deficient phenotype, we bred Tlr9−/−Pld4−/− mice. Notably, analysis of these mice indicated that all of the abnormalities of PLD4-deficient mice tested were dependent upon TLR9, including splenomegaly, MHCII upregulation, and alterations in numbers of monocytes, platelets, NK, and B1 cells (Fig. 1c,f–j). In contrast, combined deficiency in both PLD4 and IFN-γ reversed only a subset of phenotypes associated with PLD4 deficiency (Fig. 1c,f,g), suggesting that IFN-γ production by Pld4−/− mice was secondary to TLR9 triggering. Histological analysis of Pld4−/− livers revealed a TLR9-dependent increase in the frequency and size of CD68+ macrophages, consistent with mild liver inflammation (Fig. 2). Overall, these results suggested that the major effects of PLD4 deficiency stemmed from enhanced or dysregulated TLR9 recognition of ssDNA in endolysosomes.

That Pld4−/− mice had excessive TLR9-driven IFN-γ production and Il12b mRNA suggested that activation of CD8+ DCs might be dysregulated in these mice. Secretion of IL-12 by CD8+ DCs is known to stimulate IFN-γ production, particularly by NK cells31. PLD4-deficient sorted splenic DCs were found to produce more IL-12p70 and IL-6 in response to two TLR9 ligands, VACV70 and 2216PS (an A-type CpG oligodeoxynucleotide (ODN)), compared to PLD4-sufficient controls, however responses to the B-type CpG ODN PS, were similar (Fig. 1l,m). 1668PS is fully modified by phosphorothioate (PS) linkages, 2216PS is partially modified, and VACV70 is unmodified dsDNA containing CpG motifs (Table 1). In addition, VACV70, but not 1668PS and 2216PS, elicited IFN-λ responses in CD8+ DCs through the STING cytoplasmic DNA sensing pathway, but did not differ between wild-type and PLD4-deficient cells (Fig. 1n, Supplementary Fig. 3a,b). IFN-γ signals are known to promote IL-12 responses32, however sorted CD8+ DCs isolated from Pld4−/−Ifng−/− mice also showed elevated IL-12 responses to TLR9 stimuli (Supplementary Fig. 3c). CD8−CD11bhi splenic DCs also had enhanced IL-6 and IL-12 responses, especially to VACV70 (Supplementary Fig. 3d). These results supported the notion that Pld4−/− CD8+ DCs had enhanced cytokine responses to at least some TLR9 ligands.

PLD4 and PLD3 are single-stranded 5′ exonucleasesThese results suggested a role for PLD4 in the processing of TLR9 ligands. To test for nuclease activity, recombinant soluble mouse PLD4 was expressed by substituting a secretion signal peptide for the N-terminal transmembrane and cytoplasmic tail. Using DNA substrates shown in Fig. 3a, the activity of purified recombinant mouse PLD4 was then assessed (Fig. 3b). PLD4 degraded a 55 nt ssDNA (lane 2), but not a dsDNA version of the same sequence (lane 6). When paired with shorter ssDNAs complementary with the 5′ or 3′ ends, PLD4 degraded only substrate with an unpaired 5′ end (lane 4), indicating 5′-to-3′, but not 3′-to-5′, ssDNA exonuclease activity, and demonstrating a lack of endonuclease activity (lane 5). PLD4 protein carrying H-to-A mutations in both HKD motifs (PLD4-AA) lacked detectable enzymatic activity (lanes 7–11). PLD4 only cleaved substrates that lacked phosphorylated 5′ ends and was unaffected by 3′ phosphorylation (Fig. 3c, lanes 4,6). As predicted for an endosomal nuclease, the reaction was most efficient at pH<6, but was still detectable at pH 7 (Fig. 3d). Parallel experiments with mouse PLD3 revealed a similar profile of substrate specificity and an acidic pH optimum (Fig. 3e–g). We conclude that PLD3 and PLD4 are 5′ exonucleases that are able to cleave ssDNA.

PLD4 and PLD3 are single-stranded 5′ exonucleases

These results suggested a role for PLD4 in the processing of TLR9 ligands. To test for nuclease activity, recombinant soluble mouse PLD4 was expressed by substituting a secretion signal peptide for the N-terminal transmembrane and cytoplasmic tail. Using DNA substrates shown in Fig. 3a, the activity of purified recombinant mouse PLD4 was then assessed (Fig. 3b). PLD4 degraded a 55 nt ssDNA (lane 2), but not a dsDNA version of the same sequence (lane 6). When paired with shorter ssDNAs complementary with the 5′ or 3′ ends, PLD4 degraded only substrate with an unpaired 5′ end (lane 4), indicating 5′-to-3′, but not 3′-to-5′, ssDNA exonuclease activity, and demonstrating a lack of endonuclease activity (lane 5). PLD4 protein carrying H-to-A mutations in both HKD motifs (PLD4-AA) lacked detectable enzymatic activity (lanes 7–11). PLD4 only cleaved substrates that lacked phosphorylated 5′ ends and was unaffected by 3′ phosphorylation (Fig. 3c, lanes 4,6). As predicted for an endosomal nuclease, the reaction was most efficient at pH<6, but was still detectable at pH 7 (Fig. 3d). Parallel experiments with mouse PLD3 revealed a similar profile of substrate specificity and an acidic pH optimum (Fig. 3e–g). We conclude that PLD3 and PLD4 are 5′ exonucleases that are able to cleave ssDNA.

PLD3 and PLD4 appear to be classic spleen phosphodiesterasesThe enzymatic properties of PLD4 and PLD3 that we observed were reminiscent of the activity previously described as “spleen acid exonuclease” or “spleen phosphodiesterase”28, 33 (Enzyme Commission registry number 3.1.16.1). We therefore directly compared PLD3 and PLD4 with a commercial preparation of bovine spleen phosphodiesterase as a positive control using a diagnostic assay that measured their abilities to digest GpA or ApG dinucleotides (Fig. 3h). Digestion was readily tracked spectroscopically by the release and loss of adenosine, which is deaminated in a coupled assay34. Mouse and human recombinant soluble PLD3 and PLD4, but not PLD4-AA, specifically cleaved the phosphodiester bond between the phosphate and the second nucleotide, matching the activity described of bovine spleen exonuclease. Snake venom phosphodiesterase served as a control for possible cleavage at the upstream phosphodiester bond, leaving 5′ phosphorylated mononucleotides. The results indicated that digestion by PLD3 and PLD4 generated 3′ mononucleotides and nucleosides from short oligonucleotide substrates. The commercial Phosphodiesterase II (Methods) was further purified by size exclusion to reduce complexity of the sample and the enzymatically active fractions were subjected to trypsin-Liquid Chromatography-Mass Spectrometry (LC-MS) analysis. Six peptides unique to bovine PLD3 were identified (Supplementary Fig. 4a,b). Other proteins, however, were also identified (Supplementary Fig. 4c), consistent with low specific activity of the commercial preparation in the dinucleotide cleavage assay (Fig. 3h). We conclude that PLD3 and PLD4 have activities indistinguishable from classical spleen acid exonuclease.

PLD3 and PLD4 appear to be classic spleen phosphodiesterases

The enzymatic properties of PLD4 and PLD3 that we observed were reminiscent of the activity previously described as “spleen acid exonuclease” or “spleen phosphodiesterase”28, 33 (Enzyme Commission registry number 3.1.16.1). We therefore directly compared PLD3 and PLD4 with a commercial preparation of bovine spleen phosphodiesterase as a positive control using a diagnostic assay that measured their abilities to digest GpA or ApG dinucleotides (Fig. 3h). Digestion was readily tracked spectroscopically by the release and loss of adenosine, which is deaminated in a coupled assay34. Mouse and human recombinant soluble PLD3 and PLD4, but not PLD4-AA, specifically cleaved the phosphodiester bond between the phosphate and the second nucleotide, matching the activity described of bovine spleen exonuclease. Snake venom phosphodiesterase served as a control for possible cleavage at the upstream phosphodiester bond, leaving 5′ phosphorylated mononucleotides. The results indicated that digestion by PLD3 and PLD4 generated 3′ mononucleotides and nucleosides from short oligonucleotide substrates. The commercial Phosphodiesterase II (Methods) was further purified by size exclusion to reduce complexity of the sample and the enzymatically active fractions were subjected to trypsin-Liquid Chromatography-Mass Spectrometry (LC-MS) analysis. Six peptides unique to bovine PLD3 were identified (Supplementary Fig. 4a,b). Other proteins, however, were also identified (Supplementary Fig. 4c), consistent with low specific activity of the commercial preparation in the dinucleotide cleavage assay (Fig. 3h). We conclude that PLD3 and PLD4 have activities indistinguishable from classical spleen acid exonuclease.

PLD3 and PLD4 can degrade PS-linked CpG agonists of TLR9 in vitroPhosphorothioate (PS) linkages are often used in synthetic ODNs to hinder their digestion by nucleases. CpG-B-type ODNs 2006PS and 1668PS are fully PS-modified. They stimulate TLR9 preferentially through the NF-κB pathway to produce cytokines such as IL-6 and IL-1235. CpG-A-type ODN 2216PS is a potent inducer of IFN-α and has an internal palindromic sequence composed of phosphodiester-linked nucleotides flanked by strings of PS-modified G nucleotides known to form unusual secondary structures35. Human and mouse PLD3 and PLD4 were able to digest B-type CpG ODNs 2006PS and 1668PS despite their PS linkages (Fig 3i), albeit between 20–50 times less efficiently than versions with phosphodiester bonds (Supplementary Fig. 5a,c), but failed to digest 2216 or 2216PS (Fig. 2i, right; Supplementary Fig. 5b,d). We conclude that PLD3 and PLD4 digest ssODNs, including those with phosphorothioate linkages, but are blocked by features of A-type CpG ODNs, likely their secondary structure.

PLD3 and PLD4 can degrade PS-linked CpG agonists of TLR9 in vitro

Phosphorothioate (PS) linkages are often used in synthetic ODNs to hinder their digestion by nucleases. CpG-B-type ODNs 2006PS and 1668PS are fully PS-modified. They stimulate TLR9 preferentially through the NF-κB pathway to produce cytokines such as IL-6 and IL-1235. CpG-A-type ODN 2216PS is a potent inducer of IFN-α and has an internal palindromic sequence composed of phosphodiester-linked nucleotides flanked by strings of PS-modified G nucleotides known to form unusual secondary structures35. Human and mouse PLD3 and PLD4 were able to digest B-type CpG ODNs 2006PS and 1668PS despite their PS linkages (Fig 3i), albeit between 20–50 times less efficiently than versions with phosphodiester bonds (Supplementary Fig. 5a,c), but failed to digest 2216 or 2216PS (Fig. 2i, right; Supplementary Fig. 5b,d). We conclude that PLD3 and PLD4 digest ssODNs, including those with phosphorothioate linkages, but are blocked by features of A-type CpG ODNs, likely their secondary structure.

PLD3 and PLD4 limit cellular responses to exogenous TLR9 ligandsTo assess PLD3 function and potential redundancy with PLD4, we used CRISPR/Cas9 mutagenesis of zygotes to generate Pld3 strains lacking PLD3 protein (Supplementary Fig. 1j,k). Thioglycolate-elicited Pld3−/− macrophages expressed more IL-6 in response to 2216PS, relative to wild-type or Pld4−/− thioglycolate-elicited macrophages (Fig. 4a). The enhanced response was observed using macrophages from multiple founder lines, but not from wild-type or Pld3+/− littermates. In contrast, Pld3−/− macrophage responses to the TLR4 agonist lipopolysaccharide (LPS) were comparable to wild-type and Pld4−/− macrophages (Fig. 4b). RNA expression analysis revealed that wild-type thioglycolate-elicited macrophages highly expressed Pld3 and poorly expressed Pld4, whereas wild-type DCs showed the opposite pattern of expression (Fig. 4c,d). Pld3−/− mice lacked other phenotypes observed in Pld4−/− mice and had similar spleen size, splenic NK proportions, and resident peritoneal macrophage MHCII surface expression as their heterozygous littermate controls (Supplementary Fig. 1l–n) and FLT3L-DC responses were unchanged (Supplementary Fig. 6f). Functional redundancy and distinct expression profiles are thus consistent with the altered responses of Pld3−/− thioglycolate-elicited macrophages and of Pld4−/− CD8+ DCs.Recombinant PLD3 was unable to degrade the TLR9 ligand 2216PS in vitro (Fig. 3i, Supplementary Fig. 5d), so it was surprising that thioglycolate-elicited Pld3−/− macrophages had an enhanced IL-6 response to 2216PS (Fig. 4a,b). TLR9 responses to 2216PS require cleavage by DNase II in the palindromic phosphodiester-linked central region36. As DNAse II cleaves dsDNA or palindromic ssDNA, creating fragments with unphosphorylated 5′ ends and 3′ phosphates, we hypothesized that DNAse II-cleaved 2216PS fragments should be naturally degraded by PLD3 or PLD4. We therefore tested a series of synthesized fragments of 2216 or 2216PS (Fig. 5a) for their ability to stimulate FLT3L-cultured DCs and thioglycolate-elicited macrophages deficient in PLD3 or PLD4 (Fig. 5b–e). Pld4−/− DCs and Pld3−/− thioglycolate-elicited macrophages showed enhanced responses to these shorter ligands compared to PLD3/4-sufficient controls (Fig. 5b–e). When packaged in the transfection reagent Lyovec, the same ODN ligands gave similar strong responses in both wild-type and PLD4-deficient DCs (Supplementary Fig. 6a,b), suggesting that ligand stability rather than TLR9 signaling was affected by PLD4 deficiency and could be bypassed using this reagent. Interestingly, PLD3-deficient, thioglycolate-elicited macrophages still had enhanced responses to Lyovec complexes of 2216 subfragments, indicating some cell type-specific differences (Supplementary Fig. 6c). In both DCs and macrophages, responses to ODNs 1668PS and 2216PS were dependent on TLR9 (Fig. 5b–d). The enhanced bioactivity of particular CpG fragments in PLD3- or PLD4-deficient cells correlated with their sensitivity to PLD3 and PLD4 in vitro (Fig. 5f).To enable the analysis of the function of PLD3 and PLD4 in human cells, PLD3-deficient clones of the human cell line HEK-BluehTLR9 were generated by CRISPR/Cas9 treatment. Independent PLD3-deficient clones had greatly enhanced responses to 2216PS and its synthetic fragments, as well as a phosphodiester form of B-type CpG 2006 (Fig. 5g, Supplementary Fig. 6d–e). Reconstitution with PLD4 inhibited the response to baseline, whereas reconstitution with PLD4-AA did not (Fig. 5g), suggesting that nuclease activity was required to limit the response.

PLD3 and PLD4 limit cellular responses to exogenous TLR9 ligands

To assess PLD3 function and potential redundancy with PLD4, we used CRISPR/Cas9 mutagenesis of zygotes to generate Pld3 strains lacking PLD3 protein (Supplementary Fig. 1j,k). Thioglycolate-elicited Pld3−/− macrophages expressed more IL-6 in response to 2216PS, relative to wild-type or Pld4−/− thioglycolate-elicited macrophages (Fig. 4a). The enhanced response was observed using macrophages from multiple founder lines, but not from wild-type or Pld3+/− littermates. In contrast, Pld3−/− macrophage responses to the TLR4 agonist lipopolysaccharide (LPS) were comparable to wild-type and Pld4−/− macrophages (Fig. 4b). RNA expression analysis revealed that wild-type thioglycolate-elicited macrophages highly expressed Pld3 and poorly expressed Pld4, whereas wild-type DCs showed the opposite pattern of expression (Fig. 4c,d). Pld3−/− mice lacked other phenotypes observed in Pld4−/− mice and had similar spleen size, splenic NK proportions, and resident peritoneal macrophage MHCII surface expression as their heterozygous littermate controls (Supplementary Fig. 1l–n) and FLT3L-DC responses were unchanged (Supplementary Fig. 6f). Functional redundancy and distinct expression profiles are thus consistent with the altered responses of Pld3−/− thioglycolate-elicited macrophages and of Pld4−/− CD8+ DCs.

Recombinant PLD3 was unable to degrade the TLR9 ligand 2216PS in vitro (Fig. 3i, Supplementary Fig. 5d), so it was surprising that thioglycolate-elicited Pld3−/− macrophages had an enhanced IL-6 response to 2216PS (Fig. 4a,b). TLR9 responses to 2216PS require cleavage by DNase II in the palindromic phosphodiester-linked central region36. As DNAse II cleaves dsDNA or palindromic ssDNA, creating fragments with unphosphorylated 5′ ends and 3′ phosphates, we hypothesized that DNAse II-cleaved 2216PS fragments should be naturally degraded by PLD3 or PLD4. We therefore tested a series of synthesized fragments of 2216 or 2216PS (Fig. 5a) for their ability to stimulate FLT3L-cultured DCs and thioglycolate-elicited macrophages deficient in PLD3 or PLD4 (Fig. 5b–e). Pld4−/− DCs and Pld3−/− thioglycolate-elicited macrophages showed enhanced responses to these shorter ligands compared to PLD3/4-sufficient controls (Fig. 5b–e). When packaged in the transfection reagent Lyovec, the same ODN ligands gave similar strong responses in both wild-type and PLD4-deficient DCs (Supplementary Fig. 6a,b), suggesting that ligand stability rather than TLR9 signaling was affected by PLD4 deficiency and could be bypassed using this reagent. Interestingly, PLD3-deficient, thioglycolate-elicited macrophages still had enhanced responses to Lyovec complexes of 2216 subfragments, indicating some cell type-specific differences (Supplementary Fig. 6c). In both DCs and macrophages, responses to ODNs 1668PS and 2216PS were dependent on TLR9 (Fig. 5b–d). The enhanced bioactivity of particular CpG fragments in PLD3- or PLD4-deficient cells correlated with their sensitivity to PLD3 and PLD4 in vitro (Fig. 5f).

To enable the analysis of the function of PLD3 and PLD4 in human cells, PLD3-deficient clones of the human cell line HEK-BluehTLR9 were generated by CRISPR/Cas9 treatment. Independent PLD3-deficient clones had greatly enhanced responses to 2216PS and its synthetic fragments, as well as a phosphodiester form of B-type CpG 2006 (Fig. 5g, Supplementary Fig. 6d–e). Reconstitution with PLD4 inhibited the response to baseline, whereas reconstitution with PLD4-AA did not (Fig. 5g), suggesting that nuclease activity was required to limit the response.

Enhanced stability of TLR9 ligands in cells lacking PLD3 and PLD4To assess ODN degradation in vivo, we stimulated wild-type or mutant thioglycolate-elicited macrophages with 2216 carrying a 3′-biotin tag; after 3.5 or 8 hours, we recovered ODNs from cell lysates using streptavidin beads and visualized them by 32P end-labelling (Fig. 5h). The analysis revealed that the 2216-biotin fragments that demonstrated enhanced stimulatory activity in Pld3−/− macrophages and Pld4−/− DCs were less degraded in PLD3-deficient lysates than in those from control macrophages (Fig. 5h, asterisks), which had shorter species composed mainly of the poly-G 3′ end (<10 nt). The degradation pattern in Unc93b3d/3d macrophages, which lack signaling by TLR 3, 7 and 937, was identical to wild-type macrophages, indicating that a TLR9 signal is not required for enzymatic degradation by PLD3. Similar results were obtained in PLD3−/− HEK293 cells fed 2216-biotin, where DNA fragments of 10–15 nt were more abundant, and were specifically depleted in cells reconstituted with PLD4 or PLD3 but not PLD4-AA or PLD3-AA mutants (Fig. 5i). We conclude that PLD3 and PLD4 have redundant exonuclease function that can limit TLR9 responses to CpG ODNs, particularly those lacking phosphorothioate linkages, by reducing the concentration of ssDNA able to stimulate TLR9 and that these activities are similar in human and mouse.

Enhanced stability of TLR9 ligands in cells lacking PLD3 and PLD4

To assess ODN degradation in vivo, we stimulated wild-type or mutant thioglycolate-elicited macrophages with 2216 carrying a 3′-biotin tag; after 3.5 or 8 hours, we recovered ODNs from cell lysates using streptavidin beads and visualized them by 32P end-labelling (Fig. 5h). The analysis revealed that the 2216-biotin fragments that demonstrated enhanced stimulatory activity in Pld3−/− macrophages and Pld4−/− DCs were less degraded in PLD3-deficient lysates than in those from control macrophages (Fig. 5h, asterisks), which had shorter species composed mainly of the poly-G 3′ end (<10 nt). The degradation pattern in Unc93b3d/3d macrophages, which lack signaling by TLR 3, 7 and 937, was identical to wild-type macrophages, indicating that a TLR9 signal is not required for enzymatic degradation by PLD3. Similar results were obtained in PLD3−/− HEK293 cells fed 2216-biotin, where DNA fragments of 10–15 nt were more abundant, and were specifically depleted in cells reconstituted with PLD4 or PLD3 but not PLD4-AA or PLD3-AA mutants (Fig. 5i). We conclude that PLD3 and PLD4 have redundant exonuclease function that can limit TLR9 responses to CpG ODNs, particularly those lacking phosphorothioate linkages, by reducing the concentration of ssDNA able to stimulate TLR9 and that these activities are similar in human and mouse.

PLD3 and PLD4 prevent a lethal hepatic autoinflammatory diseaseThe overlapping enzymatic activities of PLD3 and PLD4 and their co-expression in macrophage subsets prompted us to interbreed these strains to generate animals deficient for both enzymes. No animals deficient for both genes survived to weaning age (Supplementary Fig. 7a). Pld3−/−Pld4−/− mice were identified of either sex, but these animals were undersized compared to littermate controls and died between 12 and 21 days of age with severe liver inflammation (Fig 6a–d, Supplementary Fig. 7b,c). The livers from Pld3−/−Pld4−/− animals showed macroscopic white areas of necrosis (Supplementary Fig. 7c), with regions of vesicular steatosis, hemophagocytosis, and multinucleated hepatocytes (Fig. 6a–c, respectively). Importantly, only a single allele of either Pld3 or Pld4 was required to allow survival and to prevent the substantial liver pathology (Fig. 6d and Supplementary Table 2). Although CD68+ myeloid cells in the livers from 16 day old Pld3+/−Pld4−/− animals were more prevalent than in Pld3+/−Pld4+/− or Pld3−/−Pld4+/− littermates, livers from Pld3−/−Pld4−/− animals demonstrated extensive infiltration of CD68+ myeloid cells (Fig. 6d). Sera from Pld3−/−Pld4−/− animals revealed elevated concentrations of ferritin, IL-6, TNF, IFN-γ, CXCL10, MCP3, MCP1, and IL-10 compared to controls of the same age, whereas IFN-α and IL-1α were not significantly elevated (Fig 6e,f). We infer that neonatal mice require PLD3 and PLD4 to prevent an inflammatory disease consistent with hemophagocytic lymphohistiocytosis (HLH).Analysis of TLR9 ligand responses of macrophages grown from Pld3−/−Pld4−/− bone marrow revealed elevated IL-6 secretion to 2216 subfragments, VACV70, and 1668 (DNA form), but responses to 1668PS, R848 and LPS were similar to wild-type or single deficient controls (Fig. 6g). As both Pld3 and Pld4 genes are expressed in bone marrow derived macrophages (Fig. 4c), the enhanced responses to particular TLR9 ligands occurred when both genes were deficient, but not when either gene was present (Fig. 6g). Pld3−/−Pld4−/− bone marrow failed to generate dendritic cells in FLT3L culture, suggesting that either the on-going inflammation was affecting DC precursors in the bone marrow or that both proteins are required for in vitro FLT3L-driven DC development.

PLD3 and PLD4 prevent a lethal hepatic autoinflammatory disease

The overlapping enzymatic activities of PLD3 and PLD4 and their co-expression in macrophage subsets prompted us to interbreed these strains to generate animals deficient for both enzymes. No animals deficient for both genes survived to weaning age (Supplementary Fig. 7a). Pld3−/−Pld4−/− mice were identified of either sex, but these animals were undersized compared to littermate controls and died between 12 and 21 days of age with severe liver inflammation (Fig 6a–d, Supplementary Fig. 7b,c). The livers from Pld3−/−Pld4−/− animals showed macroscopic white areas of necrosis (Supplementary Fig. 7c), with regions of vesicular steatosis, hemophagocytosis, and multinucleated hepatocytes (Fig. 6a–c, respectively). Importantly, only a single allele of either Pld3 or Pld4 was required to allow survival and to prevent the substantial liver pathology (Fig. 6d and Supplementary Table 2). Although CD68+ myeloid cells in the livers from 16 day old Pld3+/−Pld4−/− animals were more prevalent than in Pld3+/−Pld4+/− or Pld3−/−Pld4+/− littermates, livers from Pld3−/−Pld4−/− animals demonstrated extensive infiltration of CD68+ myeloid cells (Fig. 6d). Sera from Pld3−/−Pld4−/− animals revealed elevated concentrations of ferritin, IL-6, TNF, IFN-γ, CXCL10, MCP3, MCP1, and IL-10 compared to controls of the same age, whereas IFN-α and IL-1α were not significantly elevated (Fig 6e,f). We infer that neonatal mice require PLD3 and PLD4 to prevent an inflammatory disease consistent with hemophagocytic lymphohistiocytosis (HLH).

Analysis of TLR9 ligand responses of macrophages grown from Pld3−/−Pld4−/− bone marrow revealed elevated IL-6 secretion to 2216 subfragments, VACV70, and 1668 (DNA form), but responses to 1668PS, R848 and LPS were similar to wild-type or single deficient controls (Fig. 6g). As both Pld3 and Pld4 genes are expressed in bone marrow derived macrophages (Fig. 4c), the enhanced responses to particular TLR9 ligands occurred when both genes were deficient, but not when either gene was present (Fig. 6g). Pld3−/−Pld4−/− bone marrow failed to generate dendritic cells in FLT3L culture, suggesting that either the on-going inflammation was affecting DC precursors in the bone marrow or that both proteins are required for in vitro FLT3L-driven DC development.

PLD3 and PLD4 in hematopoietic cells prevent autoinflammationTo determine whether the immune system itself was the driver of lethal inflammation in Pld3−/−Pld4−/− mice, bone marrow transplantation into lethally irradiated 6 week old wild-type CD45.1+ recipients was performed using 19 day littermate Pld3+/−Pld4−/−, Pld3−/−Pld4+/− or Pld3−/−Pld4−/− bone marrow as donors. Eight weeks post transfer, blood analysis revealed low platelet counts, anemia, elevated reticulocytes and leukopenia in the recipients receiving Pld3−/−Pld4−/− bone marrow, but not in recipients transplanted with littermate bone marrow containing a single wild-type allele of either Pld3 or Pld4 (Fig. 7a). Despite overt splenomegaly, the number of splenic leukocytes recovered after erythrocyte lysis was lower in Pld3−/−Pld4−/− bone marrow recipients compared to controls (Fig. 7b). Flow cytometric analysis of CD45.2+ splenocytes revealed a marked reduction in B cell proportions and numbers when both Pld3 and Pld4 were absent (Fig. 7c,d). The total numbers of TCRβ+ CD4+ and CD8+ T cells were also reduced in the spleens of Pld3−/−Pld4−/− bone marrow recipients compared to controls (Fig. 7e), however the majority of CD8+ T cells exhibited an activated phenotype (CD44hiCD62Llo) and very few naïve CD8+ T cells (CD44loCD62L+) were present (Fig. 7f,g). Consistent with disease observed in Pld3−/−Pld4−/− mice, recipients of Pld3−/−Pld4−/− bone marrow had elevated expression of MHCII (with CD86 expression similar to controls) on peritoneal macrophages, elevated blood monocytes, and elevated concentrations of serum cytokines CXCL10, IFN-γ, MCP3, IL-6, but IFN-α was not detected (Supplementary Fig. 7d–k). We conclude that both PLD3 and PLD4 limit autoinflammation, and that a single wild-type allele of either can prevent the development of the inflammatory disease transferred by hematopoietic stem cells.

PLD3 and PLD4 in hematopoietic cells prevent autoinflammation

To determine whether the immune system itself was the driver of lethal inflammation in Pld3−/−Pld4−/− mice, bone marrow transplantation into lethally irradiated 6 week old wild-type CD45.1+ recipients was performed using 19 day littermate Pld3+/−Pld4−/−, Pld3−/−Pld4+/− or Pld3−/−Pld4−/− bone marrow as donors. Eight weeks post transfer, blood analysis revealed low platelet counts, anemia, elevated reticulocytes and leukopenia in the recipients receiving Pld3−/−Pld4−/− bone marrow, but not in recipients transplanted with littermate bone marrow containing a single wild-type allele of either Pld3 or Pld4 (Fig. 7a). Despite overt splenomegaly, the number of splenic leukocytes recovered after erythrocyte lysis was lower in Pld3−/−Pld4−/− bone marrow recipients compared to controls (Fig. 7b). Flow cytometric analysis of CD45.2+ splenocytes revealed a marked reduction in B cell proportions and numbers when both Pld3 and Pld4 were absent (Fig. 7c,d). The total numbers of TCRβ+ CD4+ and CD8+ T cells were also reduced in the spleens of Pld3−/−Pld4−/− bone marrow recipients compared to controls (Fig. 7e), however the majority of CD8+ T cells exhibited an activated phenotype (CD44hiCD62Llo) and very few naïve CD8+ T cells (CD44loCD62L+) were present (Fig. 7f,g). Consistent with disease observed in Pld3−/−Pld4−/− mice, recipients of Pld3−/−Pld4−/− bone marrow had elevated expression of MHCII (with CD86 expression similar to controls) on peritoneal macrophages, elevated blood monocytes, and elevated concentrations of serum cytokines CXCL10, IFN-γ, MCP3, IL-6, but IFN-α was not detected (Supplementary Fig. 7d–k). We conclude that both PLD3 and PLD4 limit autoinflammation, and that a single wild-type allele of either can prevent the development of the inflammatory disease transferred by hematopoietic stem cells.

DiscussionWe present evidence that the endolysosomal proteins PLD3 and PLD4 trim the 5′ end of ss oligodeoxynucleotides, thus degrading them and limiting their ability to stimulate TLR9. Consistent with this conclusion, PLD3 and PLD4 are most active at acidic pH (5–5.5) and are localized to endolysosomes15, 16, 17. Importantly, neither enzyme appears to have phospholipase activity25, 38.Human GWAS studies have implicated polymorphisms of PLD3 and PLD4 in inflammatory diseases. In animals, spontaneous Pld4 null mutations have been linked to skin disease and runting. A mouse mutant identified on the BALB/c background called “Thin hair with small size” (Thss) appears to have abnormalities distinct from PLD4-deficiency on the C57BL/6 background that we have investigated here (http://www.informatics.jax.org/downloads/Reference_texts/J171492.pdf). A spontaneous PLD4-null mutation occurred in a herd of cattle, where homozygotes presented as runts with skin lesions, respiratory and digestive tract inflammation and mild anemia39. It will be of interest to determine if these phenotypes are also connected to TLR9 activation.The similarities in enzymatic activities of PLD3 and PLD4 and their partly overlapping expression patterns suggest that they have redundant functions in key cell types. RNA profiling data imply that PLD4 has a narrow tissue distribution, with high levels in DCs and myeloid cells, whereas PLD3 has a broad distribution and appears to digest artificial substrates with higher efficiency than does PLD4. PLD3 enters the lysosome by an unusual ESCRT-regulated sorting and ubiquitination pathway17 and other PLD3-deficient mice reportedly showed no phenotype except for effects on the morphology of the lysosomal system16. The phenotype of our Pld3−/− mice also seems benign compared to that of Pld4−/− mice. This observation might be explained by the high levels of PLD4 normally present in cells expressing high levels of TLR9. Although Pld4−/− mice have a relatively mild form of Macrophage Activation Syndrome, residual PLD3 in myeloid cells likely provides protection from more severe disease as a single allele of either Pld3 or Pld4 protect against lethal inflammation early in life.Nucleic acid sensors are endowed with a restricted fine specificity and affinity for ligands. Responses are limited by the quantity of ligands available and their intracellular localization. PLD3 and PLD4 provide ssDNAse activity in endolysosomes; it is likely no coincidence that they reside in the same compartment as active TLR9. Importantly, the DNA substrates of PLD3 and PLD4 require a non-phosphorylated 5′ end and the absence of secondary structure. Nucleic acids with such features are relatively unusual in nature, but are generated efficiently in endolysosomes by DNAse II, which cleaves dsDNA down to ssDNA fragments of ~10 nt carrying 3′ phosphate and 5′ hydroxyl groups36. DNAse II is also required to process DNA hairpins, such as A-type CpG ligands, for TLR9 recognition. Recent work indicates that TLR9 has two binding sites, one binding ssDNA with a CpG motif and the second binding short ssDNA carrying a 5′ hydroxyl40. Thus, the short stimulatory ligands that TLR9 is predicted to encounter should also be good substrates for PLD3 and PLD4. Endosomal build-up of such short ssDNA in the absence of PLD3 or PLD4 is unlikely to trigger STING/cGAS-mediated IFN responses as lengths of ssDNA greater than 21 nt are required and may be prevented by the presence of Trex1 exonuclease in the cytoplasm41, 42.Although mild, the phenotype of Pld4−/− mice is reminiscent of human macrophage activation syndrome, which is associated with elevated IFN-γ, IL-12, CXCL9, CXCL10, low NK cell number, an increase of liver CD68+ cells, and can be mimicked in mice by repeated injection of a TLR9 ligand3, 30, 43, 44. Importantly, all of the phenotypes of Pld4−/− mice were absent in mice also lacking TLR9, consistent with the notion that defective degradation of ligands for TLR9 is central to the phenotype. This is further supported by evidence that Pld3−/− macrophages and HEK293 cells have exaggerated responses to particular ssDNA TLR9 ligands. Several features of the disease transferred by Pld3−/−Pld4−/− bone marrow into wild-type recipients mimic dysregulated TLR9 signaling45. Animals receiving stem cells expressing a mutated TLR9 that does not require proteolytic activation or traffic normally to endolysosomes, succumb to a DC-driven fatal inflammation characterised by profound anemia and reduced splenic B cells45. The lethality of Pld3−/−Pld4−/− animals in early life reflects the important role these enzymes play in limiting autoinflammation and although we predict that TLR9 signaling is required for the disease, this needs to be addressed by further breeding.Presenting in infancy, livers from children with primary HLH show several features similar to those from Pld3−/−Pld4−/− animals, including giant cell hepatitis, steatosis, necrosis, and the name sake hemophagocytosis46. High serum ferritin and cytokine levels and extensive CD68+ cell hepatic infiltration in Pld3−/−Pld4−/− mice provide further similarities to primary HLH, although this disease is usually associated with mutations resulting in defective cytotoxic granule content or exocytosis from NK and T cells47.PS-linkages are used in synthetic ODNs to hinder their digestion by nucleases. Fully phosphorothioated CpG-B type ODNs (1668PS) could drive TLR9-mediated IL-6 and IL-12 secretion from macrophages and dendritic cells independently of PLD3 or PLD4. However, PS linkages are by no means strictly artificial, and are found widely in nature in the DNA of bacteria48. That PLD3 and PLD4 digest ODNs carrying PS linkages or strings of Gs more slowly than host DNA might contribute to preferential innate recognition of bacteria or viruses.We find that PLD3 and PLD4 are 5′ exonucleases with properties strikingly similar to the enzymatic activity described many years ago as spleen acid exonuclease27, 28. The bovine enzyme has a pH optimum of 5.5, and is 5′ single strand-specific; it also was characterized to be a non-processive nuclease, inhibited by 5′ phosphate, and generating 3′ nucleotide monophosphate products27, 49. Degradation of ssDNA substrates by PLD3 and PLD4 was similarly non-processive, blocked by 5′ phosphate, generated 3′ nucleotide monophosphates, and was able to digest to completion, generating mononucleotides and nucleosides as does the spleen exonuclease34. Although the gene encoding spleen exonuclease has never been cloned, the enzymatic activity has been described in human, rat, cow and pig, whose predicted PLD3 proteins are highly conserved (93–96% identical between species). The similarities in enzymatic features and the high amino acid identity between bovine, mouse and human PLD3, suggest that PLD3 may be identical to the spleen phosphodiesterase activity previously described, though PLD4 cannot be excluded, as it is also conserved among these species (>70% identical) and has a very similar enzymatic activity.

We present evidence that the endolysosomal proteins PLD3 and PLD4 trim the 5′ end of ss oligodeoxynucleotides, thus degrading them and limiting their ability to stimulate TLR9. Consistent with this conclusion, PLD3 and PLD4 are most active at acidic pH (5–5.5) and are localized to endolysosomes15, 16, 17. Importantly, neither enzyme appears to have phospholipase activity25, 38.

Human GWAS studies have implicated polymorphisms of PLD3 and PLD4 in inflammatory diseases. In animals, spontaneous Pld4 null mutations have been linked to skin disease and runting. A mouse mutant identified on the BALB/c background called “Thin hair with small size” (Thss) appears to have abnormalities distinct from PLD4-deficiency on the C57BL/6 background that we have investigated here (http://www.informatics.jax.org/downloads/Reference_texts/J171492.pdf). A spontaneous PLD4-null mutation occurred in a herd of cattle, where homozygotes presented as runts with skin lesions, respiratory and digestive tract inflammation and mild anemia39. It will be of interest to determine if these phenotypes are also connected to TLR9 activation.

The similarities in enzymatic activities of PLD3 and PLD4 and their partly overlapping expression patterns suggest that they have redundant functions in key cell types. RNA profiling data imply that PLD4 has a narrow tissue distribution, with high levels in DCs and myeloid cells, whereas PLD3 has a broad distribution and appears to digest artificial substrates with higher efficiency than does PLD4. PLD3 enters the lysosome by an unusual ESCRT-regulated sorting and ubiquitination pathway17 and other PLD3-deficient mice reportedly showed no phenotype except for effects on the morphology of the lysosomal system16. The phenotype of our Pld3−/− mice also seems benign compared to that of Pld4−/− mice. This observation might be explained by the high levels of PLD4 normally present in cells expressing high levels of TLR9. Although Pld4−/− mice have a relatively mild form of Macrophage Activation Syndrome, residual PLD3 in myeloid cells likely provides protection from more severe disease as a single allele of either Pld3 or Pld4 protect against lethal inflammation early in life.

Nucleic acid sensors are endowed with a restricted fine specificity and affinity for ligands. Responses are limited by the quantity of ligands available and their intracellular localization. PLD3 and PLD4 provide ssDNAse activity in endolysosomes; it is likely no coincidence that they reside in the same compartment as active TLR9. Importantly, the DNA substrates of PLD3 and PLD4 require a non-phosphorylated 5′ end and the absence of secondary structure. Nucleic acids with such features are relatively unusual in nature, but are generated efficiently in endolysosomes by DNAse II, which cleaves dsDNA down to ssDNA fragments of ~10 nt carrying 3′ phosphate and 5′ hydroxyl groups36. DNAse II is also required to process DNA hairpins, such as A-type CpG ligands, for TLR9 recognition. Recent work indicates that TLR9 has two binding sites, one binding ssDNA with a CpG motif and the second binding short ssDNA carrying a 5′ hydroxyl40. Thus, the short stimulatory ligands that TLR9 is predicted to encounter should also be good substrates for PLD3 and PLD4. Endosomal build-up of such short ssDNA in the absence of PLD3 or PLD4 is unlikely to trigger STING/cGAS-mediated IFN responses as lengths of ssDNA greater than 21 nt are required and may be prevented by the presence of Trex1 exonuclease in the cytoplasm41, 42.

Although mild, the phenotype of Pld4−/− mice is reminiscent of human macrophage activation syndrome, which is associated with elevated IFN-γ, IL-12, CXCL9, CXCL10, low NK cell number, an increase of liver CD68+ cells, and can be mimicked in mice by repeated injection of a TLR9 ligand3, 30, 43, 44. Importantly, all of the phenotypes of Pld4−/− mice were absent in mice also lacking TLR9, consistent with the notion that defective degradation of ligands for TLR9 is central to the phenotype. This is further supported by evidence that Pld3−/− macrophages and HEK293 cells have exaggerated responses to particular ssDNA TLR9 ligands. Several features of the disease transferred by Pld3−/−Pld4−/− bone marrow into wild-type recipients mimic dysregulated TLR9 signaling45. Animals receiving stem cells expressing a mutated TLR9 that does not require proteolytic activation or traffic normally to endolysosomes, succumb to a DC-driven fatal inflammation characterised by profound anemia and reduced splenic B cells45. The lethality of Pld3−/−Pld4−/− animals in early life reflects the important role these enzymes play in limiting autoinflammation and although we predict that TLR9 signaling is required for the disease, this needs to be addressed by further breeding.

Presenting in infancy, livers from children with primary HLH show several features similar to those from Pld3−/−Pld4−/− animals, including giant cell hepatitis, steatosis, necrosis, and the name sake hemophagocytosis46. High serum ferritin and cytokine levels and extensive CD68+ cell hepatic infiltration in Pld3−/−Pld4−/− mice provide further similarities to primary HLH, although this disease is usually associated with mutations resulting in defective cytotoxic granule content or exocytosis from NK and T cells47.

PS-linkages are used in synthetic ODNs to hinder their digestion by nucleases. Fully phosphorothioated CpG-B type ODNs (1668PS) could drive TLR9-mediated IL-6 and IL-12 secretion from macrophages and dendritic cells independently of PLD3 or PLD4. However, PS linkages are by no means strictly artificial, and are found widely in nature in the DNA of bacteria48. That PLD3 and PLD4 digest ODNs carrying PS linkages or strings of Gs more slowly than host DNA might contribute to preferential innate recognition of bacteria or viruses.

We find that PLD3 and PLD4 are 5′ exonucleases with properties strikingly similar to the enzymatic activity described many years ago as spleen acid exonuclease27, 28. The bovine enzyme has a pH optimum of 5.5, and is 5′ single strand-specific; it also was characterized to be a non-processive nuclease, inhibited by 5′ phosphate, and generating 3′ nucleotide monophosphate products27, 49. Degradation of ssDNA substrates by PLD3 and PLD4 was similarly non-processive, blocked by 5′ phosphate, generated 3′ nucleotide monophosphates, and was able to digest to completion, generating mononucleotides and nucleosides as does the spleen exonuclease34. Although the gene encoding spleen exonuclease has never been cloned, the enzymatic activity has been described in human, rat, cow and pig, whose predicted PLD3 proteins are highly conserved (93–96% identical between species). The similarities in enzymatic features and the high amino acid identity between bovine, mouse and human PLD3, suggest that PLD3 may be identical to the spleen phosphodiesterase activity previously described, though PLD4 cannot be excluded, as it is also conserved among these species (>70% identical) and has a very similar enzymatic activity.

MethodsMiceTargeting of Pld4 was carried out in 129/J strain embryonic stem cells by the TSRI Mouse Genetics Core. The neor cassette was removed by breeding to mice expressing Flp recombinase to generate Pld4fl/+ mice. Subsequent breeding to EIIa-Cre mice generated Pld4+/− mice, and both Pld4fl/+ and Pld4+/− were subsequently backcrossed to C57BL/6J for >20 generations. Pld4-mutant mice were genotyped by PCR using the following primers #32- TTTGCCATACCACCATAGCA, #33- GTTGGGCAGCAAATCTTAGG, #34- CTGGATTCCCCAAGAGAACA, and generating product sizes of 300 bp (wild-type), 350 bp (Flox), and 420 bp (ablated) for indicated alleles. The following mouse strains bred with Pld4−/− mice including B6.129S7-Rag1tm1Mom/J, B6.129P2-Lyz2tm1(cre)Ifo/J, B6.Cg-Tg(Itgax-cre)1–1Reiz/J, B6.129S7-Ifngtm1Ts/J, C57BL/6J-Tlr9M7Btlr/Mmjax (CpG11) were from the Jackson Laboratory. Unc93b3d/3d mice were kindly provided by B. Beutler, Univ. Texas Southwestern. Mutation of Pld3 was carried out by nuclear injection of C57BL/6J zygotes with plasmid eSpCas9(1.1)51 (a gift from Y. Doyon, Centre Hospitalier Universitaire de Québec, Addgene plasmid 79877) engineered to express the guide sequence GGACTGGCGGTCGCTGACCC targeting Pld3 exon 9, which encodes the first HKD motif. DNA (5 ng/μl) in Tris-HCl, pH 7.5, 10 mM, EDTA 0.1 mM, NaCl 50 mM was microinjected by the TSRI Mouse Genetics Core staff. Pld3−/− mice were genotyped by PCR using P3Ex5F-GCACATGCACACACACACAAAG and P3Ex6R- TACAATGAGGGCCAGGTAAGTG and the PCR product was digested with PshA1 restriction enzyme as the PshA1 site is only present in the wild-type allele. Experiments were performed as approved by the TSRI IACUC Committee.Dendritic cell generation, isolation, and stimulationDCs were expanded in vitro from bone marrow precursors using FLT3L as described52. Sorted splenic DCs were isolated essentially as described52. CD8α+ cDC (CD45RA−, CD11c+, CD11b+ and CD8α+) cells were plated at a density of 106/ml in round-bottom plates in media containing GM-CSF (10 ng/ml), mouse IL-4 (10 ng/ml) and rat IFN-γ (10 ng/ml). Stimuli were added and supernatant was collected 18–24 h later for cytokine analyses. VACV70 was purchased from InvivoGen and added to the culture media after being complexed with Lyovec at a concentration of 1 μg/ml. Oligonucleotides purchased from IDT or InvivoGen were also used directly; 2216PS (1 μM), 2216 (1 μM) 1668PS (0.25 μM) and 1668 (1 μM). 2216 subfragments were used at 1 μM. ELISA kits were used to measure the following cytokines: IL-12p70 (Duoset, R&D Systems), IL-6 (BD OptEIA, BD Biosciences), IFN-α (Lumikine, InvivoGen), IFN-λ (mouse IL-28A/B, R&D Systems).Macrophage analysisMacrophages were elicited into the peritoneal cavity by 2% thioglycolate broth treatment as described53 and stimulated essentially as described above. Ultrapure LPS from Salmonella minnesota R595 (InvivoGen) was used at 100 ng/ml. For the experiments described in Fig. 5h thioglycolate-elicited macrophages (2 × 106) were stimulated for either 3.5 or eight h with CpG-containing stimuli 2216-3′-biotin at 5 μM. Cells were washed twice with cold PBS before cell lysis with 0.5% NP40, 150 mM NaCl and 1 mM EDTA. Nuclei and cell debris were removed by centrifugation and the supernatant was incubated with 20 μl streptavidin magnetic beads (Invitrogen) for 40 min at 4°C with rotation. Beads were isolated using a magnet, washed twice with lysis buffer, and twice with T4 Polynucleotide Kinase (PNK) buffer (NEB). Bound DNA was labeled with γ32P-ATP (Perkin Elmer) with PNK for 30 min at 37°C and beads were washed twice with lysis buffer. Labeled DNA was then eluted with 95% formamide, 10 mM EDTA, heated to 70°C for 10 min and loaded onto 20% TBE/Urea Acrylamide gels. Radiolabeled DNA was exposed to a phosphorimager screen and the signal visualized using a Molecular Dynamics Typhoon phosphorimager and quantitated using ImageQuant software. In some cases, bone marrow derived macrophages were grown in 20% L929 supernatant containing medium over 7 days and stimulated as above.Protein purification and nuclease assayThe N-terminal and transmembrane domain-deleted forms of mouse and human PLD3, PLD3-AA, PLD4, and PLD4-AA were cloned into vector pCEP4 (Thermo) and transfected into HEK293-EBNA cells (Invitrogen). The stable cell lines expressing appropriate proteins were generated by selection with G418 (0.25 mg/ml) and puromycin (0.5 μg/ml) in DMEM complete medium. After selection, cells were expanded in suspension culture in 293 Freestyle media (Life Technologies) containing G418 (0.25 mg/ml) and puromycin (0.5 μg/ml) in a humidified 37°C CO2 incubator (8%), rotating at 135 rpm at 1–2 × 106 cells/ml. Cells were harvested, centrifuged at 5,000g for 20 min and supernatant filtered using 0.2 μm filter (Millipore). Protino Ni-NTA Agarose beads (Macherey-Nagel) were added to the supernatants and rotated for 4 h at 4°C. After incubation, the beads were washed three times in buffer (Tris-HCl 20 mM, pH 7.5, NaCl 250 mM, Imidazole 10 mM) and eluted (Tris-HCl 20 mM, pH 7.5, NaCl 250 mM, Imidazole 200 mM). The eluted proteins were dialyzed overnight in PBS at 4°C and protein concentrations were determined using the BCA protein assay kit (Thermo). Proteins were analyzed for purity by SDS-PAGE stained with Coomassie Brilliant Blue. Nuclease assays for DNA and RNA digestion were carried out for 2 h at 37°C in 50 mM 2-(N-morpholino)ethanesulfonic acid (MES) pH 5.5, 125 mM NaCl, with substrates at 2.5 μM and enzyme at 100 nM. For phosphorothioate-modified substrates, pH was pH 5.0, enzyme concentration 500 nM, and incubation time 6 h unless otherwise noted.Dinucleotide substrate assayEnzymatic activity assay on dinucleotide substrates was performed as described34. Lyophilized bovine spleen phosphodiesterase II (PDII) and snake venom phosphodiesterase I (PDI) were diluted to 2.5 U/ml and 100 U/ml in water, respectively. Adenosine deaminase (ADA) was resuspended in 1 ml water then diluted to 20 μg/ml in 0.5× PBS. All enzymes were purchased from Worthington. The dinucleotide substrates GpA and ApG were purchased from TriLink Biotechnologies. Final reaction condition for PLD3, PLD4, and bovine spleen PDII was 50 mM MES pH 6.1, 20 mM NaCl. Snake venom PDI enzyme reaction was performed in 50 mM Tris, pH 7.9, 20 mM NaCl. Each reaction consisted of dinucleotide substrate (60 μM), ADA enzyme (2 μg/ml), and either PLD3 (50 nM), PLD4 (50 nM), PDII (0.25 U/ml), or PDI (10 U/ml) in a total volume of 250 μl. The reaction was performed with the Nanodrop 3000C in a heated quartz cuvette (37 °C) and absorbance at 265 nm measured every 15 s. The tested phosphodiesterase was added after four initial absorbance readings (1 min) and absorbance measured for another 5 min. In the absence of an added phosphodiesterase, such as substrate alone, or ADA alone, no change in absorbance was observed throughout the 6 min of recording.Studies with 293HEK cellsHEK293-BluehTLR9 cells (InvivoGen) were cultured and stimulated according to the manufacturer’s instructions. Briefly, 100 μl cells/well at a density of 106/ml were cultured in 96-well plates. Stimuli were added and culture supernatants were harvested 16 h later for SEAP1 detection using a kit (InvivoGen). For PLD3-deficient cells, HEK293-BluehTLR9 cells were transiently transfected with a CRISPR/Cas9 construct previously validated for efficacy that carried the guide RNA sequence TCCTCATTCTGGCGGTTGT. Single-cell clones were then isolated and individually screened by sequencing for homozygous mutation of PLD3. PLD3-deficient cells were transduced with lentiviruses encoding human PLD4, human PLD4-AA, mouse PLD3, and mouse PLD3-AA, respectively and cultured for several days before stimulation. ODN stimuli were purchased from InvivoGen or Integrated DNA Technologies. Cells were cultured with 2216-3′-biotin for 6 h at 37 °C then washed and lysed as for macrophages. Streptavidin precipitated DNA was 5′ radiolabeled using polynucleotide kinase and γ32P-ATP. To achieve higher resolution of smaller oligonucleotide fragments, samples in formamide were loaded onto a 20% acrylamide (19:1 Accugel acrylamide (National Diagnostics)) denaturing gel in 50 mM histidine as described50.Blood analysisMonocyte, platelet, reticulocyte, red blood cell, and white blood cell counts in whole blood from age-matched male and female animals were determined using an IDEX Procyte DX Machine for blood subset analysis. Plasma concentrations of IFN-γ and CXCL10 of adult Pld4fl/fl and Pld4−/− mice were determined using FlowCytomix beads (eBioscience). Serum from 16–19-day-old animals was analyzed for cytokine concentrations using a kit (Flowcytomix, eBiosciences)FerritinMouse Ferritin concentrations were determined from sera isolated from 12–21 day old animals as outlined by the manufacturer of the ELISA kit (Alpco).Generation of Bone Marrow ChimerasBone marrow from 16-day-old littermates of CD45.2+
Pld3−/−
Pld4+/−, Pld3+/−
Pld4−/−, or doubly deficient Pld3−/−Pld4−/− genotypes were injected intravenously into lethally irradiated (1000 rad) 8 week old sex matched B6.SJL-CD45.1+ recipients. Eight weeks after transfer, blood and spleen populations and serum cytokines were analyzed.Flow cytometry analysis of leukocyte compartmentsTissue suspensions were generated by mashing tissue between frosted glass slides. Bone marrow was released from tibia and femurs by a mortar and pestle crushing method. Red blood cells were lysed with ammonium chloride (0.83%) and cells filtered to generate single cell suspensions. Peritoneal cavity cells were isolated by lavage. Cells were pre-incubated with 1 μg/106 cells of Fc Block (2.4G2) prior to staining with antibodies. Splenic NK cell proportions were determined by flow cytometry of splenocytes from age- and sex-matched animals using the following gates (TCRβ− NK1.1+CD49b+). Flow cytometry of DCs was performed from spleens treated with collagenase and DNase I treatment prior to separation on light density gradients (Nycodenz). The following gating strategies were used to identify and quantify different leukocyte subsets. pDCs (CD11c+CD45RA+SiglecH+CD19−TCRβ−); CD8+ cDCs (CD19−TCRβ−CD11c+CD45RA−CD11bloCD8α+); CD11b+ cDCs (CD19−TCRβ−CD11c+CD45RA−CD11bhiCD8α+); CD4+ (TCRβ+CD4+CD8α−); CD8+ (TCRβ+CD4−CD8α+); CD19+ B cells (CD19+B220+); Transitional 1 (TI) B cells (B220+CD93+IgM+CD23−); Transitional 2 (T2) B cell (B220+CD93+IgM+CD23+); Follicular B2 cells (B2) (CD19+B220+CD93−IgM+CD23+); Marginal zone B cells (MZ) (CD19+ B220+CD93−CD23loIgM+CD21hi); Germinal Center (GC) (TCRβ−B220+FAS+ GL7+); Plasma cell (PC) (TCRβ−F480−IgD−CD138+); resident peritoneal B1 cells (B220loCD19+CD43+IgM+); resident peritoneal B2 (B220hiCD19+CD43−IgM+); resident peritoneal macrophages (size, F4/80+CD11b+). Antibodies used in this study are listed in Supplementary Table 3. Gating strategies used for dendritic cells, peritoneal macrophages, and bone marrow chimera B and T cells are shown in Supplementary Figure 8.Experimental autoimmune encephalomyelitis induction and monitoringEAE was induced by immunization with MOG35–55 peptide emulsified in complete Freund’s adjuvant followed by Pertussis toxin administration as outlined by the manufacturer (Hooke Laboratories) and scored according to the manufacturer’s instructions. Female animals over 8 weeks old were used. In some cases, C57BL/6 mice congenic for CD45.1 (B6.CD45.1) were lethally irradiated with two doses of 500 rads and were reconstituted with bone marrow from either Pld4fl/fl or Pld4−/− mice for 10 weeks prior to immunization.CD68+ liver stainsLivers from age and sex matched mice were frozen in optimal cutting temperature (O.C.T., Sakura). Then 7 μm sections were cut, fixed with acetone for 10 min, and stained with anti-CD68 Alexa Fluor 647 (FA-11, BioLegend). After washing in PBS, nuclei were stained with Hoechst 33258 for 5 min prior to mounting in Prolong Diamond Antifade mounting medium (Invitrogen). Sections were imaged using the Keyence BZ-X710 fluorescence microscope. CD68+% and area was determined using FIJI software54. Briefly, color images were split into channels and a threshold set for each channel. The analyze particles tool was used to estimate number of nuclei in image (Hoechst blue channel). The analyze particle tool was then applied to the CD68+ channel (Red channel). The percentage was then calculated. The measure tool was used to calculate the average area of the particles above the threshold in the CD68+ channel.Mass spectrometryPhosphodiesterase II (Worthington Biochemical Corp. Cat#: LS003602, LOT #: 36m16915) was fractionated on a Superdex 200 column in PBS and two adjacent fractions with peak enzymatic activity were pooled. The solution sample was reduced (200 mM DTT), alkylated (200 mM iodoacetamide) and digested with trypsin overnight using an estimated 1:30 enzyme to substrate ratio before being analyzed by nano-LC-MS/MS at the TSRI core facility. The MS/MS raw data were searched against the custom sequence database, which included the sequence [NP_001071509.1 phospholipase D3 [Bos taurus]] and the NCBInr database. Bovine PLD3 protein was identified with 7 peptides and 17% sequence coverage and the sequence [XP_015324054.1 PREDICTED: phospholipase D4 isoform X2 [Bos taurus]] was not identified. Other proteins also identified with at least five different peptides are listed in Supplementary Fig. 4.RNA expression analysis and sequencingRNA expression data in Fig. 4c was from the BioGPS MOE430 Gene Atlas Data set55, 56. For the sequencing analysis in Fig. 1k, total RNA was isolated from splenocytes using a kit (RNA easy, Qiagen). RNA was prepared into RNAseq libraries using the NEBNext Ultra Directional RNA Library Prep Kit for Illumina following the manufacturer’s recommended protocol. The libraries were then PCR amplified 15 cycles using barcoded PCR primers, purified and size selected using AMPure XP Beads before loading onto an Illumina NextSeq500 for 75 base single read sequencing. A total of 16.3–19.6 million “passed filter” reads were obtained for each of the 6 samples. Reads were compared to the mouse genome build mm10 and quantitated using the program Salmon. DC RNAseq data described in Fig. 4d were generated as follows. DC subsets were purified from spleens of 10–14 C57BL/6 mice/sample as described57. Briefly, cell sorting was performed on a BD Influx™ machine. RNA was extracted using RNeasy Mini Kit (QIAGEN) and residual genomic DNA removed using RNase-free DNase (QIAGEN). Sequencing libraries were prepared using the Illumina TruSeq Stranded mRNA kit and sequencing was carried out by Micromon using High-Output SBS on the Illumina NextSeq 500. RNAseq data analyses were performed in Degust (http://degust.erc.monash.edu/ Version: 3.1.0) by Monash Bioinformatics platform, David R Powell & Adele Barugahare.Statistical analysesAnalysis of differential gene expression was carried out using the program Edge. Other data comparisons used the program Prism (Graph Pad), with correction for multiple comparisons. When two groups were compared, an unpaired, two-tailed T test was used. Actual P values are provided in Supplementary Table 4. Additional information is provided in the Life Sciences Reporting Summary.

MiceTargeting of Pld4 was carried out in 129/J strain embryonic stem cells by the TSRI Mouse Genetics Core. The neor cassette was removed by breeding to mice expressing Flp recombinase to generate Pld4fl/+ mice. Subsequent breeding to EIIa-Cre mice generated Pld4+/− mice, and both Pld4fl/+ and Pld4+/− were subsequently backcrossed to C57BL/6J for >20 generations. Pld4-mutant mice were genotyped by PCR using the following primers #32- TTTGCCATACCACCATAGCA, #33- GTTGGGCAGCAAATCTTAGG, #34- CTGGATTCCCCAAGAGAACA, and generating product sizes of 300 bp (wild-type), 350 bp (Flox), and 420 bp (ablated) for indicated alleles. The following mouse strains bred with Pld4−/− mice including B6.129S7-Rag1tm1Mom/J, B6.129P2-Lyz2tm1(cre)Ifo/J, B6.Cg-Tg(Itgax-cre)1–1Reiz/J, B6.129S7-Ifngtm1Ts/J, C57BL/6J-Tlr9M7Btlr/Mmjax (CpG11) were from the Jackson Laboratory. Unc93b3d/3d mice were kindly provided by B. Beutler, Univ. Texas Southwestern. Mutation of Pld3 was carried out by nuclear injection of C57BL/6J zygotes with plasmid eSpCas9(1.1)51 (a gift from Y. Doyon, Centre Hospitalier Universitaire de Québec, Addgene plasmid 79877) engineered to express the guide sequence GGACTGGCGGTCGCTGACCC targeting Pld3 exon 9, which encodes the first HKD motif. DNA (5 ng/μl) in Tris-HCl, pH 7.5, 10 mM, EDTA 0.1 mM, NaCl 50 mM was microinjected by the TSRI Mouse Genetics Core staff. Pld3−/− mice were genotyped by PCR using P3Ex5F-GCACATGCACACACACACAAAG and P3Ex6R- TACAATGAGGGCCAGGTAAGTG and the PCR product was digested with PshA1 restriction enzyme as the PshA1 site is only present in the wild-type allele. Experiments were performed as approved by the TSRI IACUC Committee.

Targeting of Pld4 was carried out in 129/J strain embryonic stem cells by the TSRI Mouse Genetics Core. The neor cassette was removed by breeding to mice expressing Flp recombinase to generate Pld4fl/+ mice. Subsequent breeding to EIIa-Cre mice generated Pld4+/− mice, and both Pld4fl/+ and Pld4+/− were subsequently backcrossed to C57BL/6J for >20 generations. Pld4-mutant mice were genotyped by PCR using the following primers #32- TTTGCCATACCACCATAGCA, #33- GTTGGGCAGCAAATCTTAGG, #34- CTGGATTCCCCAAGAGAACA, and generating product sizes of 300 bp (wild-type), 350 bp (Flox), and 420 bp (ablated) for indicated alleles. The following mouse strains bred with Pld4−/− mice including B6.129S7-Rag1tm1Mom/J, B6.129P2-Lyz2tm1(cre)Ifo/J, B6.Cg-Tg(Itgax-cre)1–1Reiz/J, B6.129S7-Ifngtm1Ts/J, C57BL/6J-Tlr9M7Btlr/Mmjax (CpG11) were from the Jackson Laboratory. Unc93b3d/3d mice were kindly provided by B. Beutler, Univ. Texas Southwestern. Mutation of Pld3 was carried out by nuclear injection of C57BL/6J zygotes with plasmid eSpCas9(1.1)51 (a gift from Y. Doyon, Centre Hospitalier Universitaire de Québec, Addgene plasmid 79877) engineered to express the guide sequence GGACTGGCGGTCGCTGACCC targeting Pld3 exon 9, which encodes the first HKD motif. DNA (5 ng/μl) in Tris-HCl, pH 7.5, 10 mM, EDTA 0.1 mM, NaCl 50 mM was microinjected by the TSRI Mouse Genetics Core staff. Pld3−/− mice were genotyped by PCR using P3Ex5F-GCACATGCACACACACACAAAG and P3Ex6R- TACAATGAGGGCCAGGTAAGTG and the PCR product was digested with PshA1 restriction enzyme as the PshA1 site is only present in the wild-type allele. Experiments were performed as approved by the TSRI IACUC Committee.

Dendritic cell generation, isolation, and stimulationDCs were expanded in vitro from bone marrow precursors using FLT3L as described52. Sorted splenic DCs were isolated essentially as described52. CD8α+ cDC (CD45RA−, CD11c+, CD11b+ and CD8α+) cells were plated at a density of 106/ml in round-bottom plates in media containing GM-CSF (10 ng/ml), mouse IL-4 (10 ng/ml) and rat IFN-γ (10 ng/ml). Stimuli were added and supernatant was collected 18–24 h later for cytokine analyses. VACV70 was purchased from InvivoGen and added to the culture media after being complexed with Lyovec at a concentration of 1 μg/ml. Oligonucleotides purchased from IDT or InvivoGen were also used directly; 2216PS (1 μM), 2216 (1 μM) 1668PS (0.25 μM) and 1668 (1 μM). 2216 subfragments were used at 1 μM. ELISA kits were used to measure the following cytokines: IL-12p70 (Duoset, R&D Systems), IL-6 (BD OptEIA, BD Biosciences), IFN-α (Lumikine, InvivoGen), IFN-λ (mouse IL-28A/B, R&D Systems).

Dendritic cell generation, isolation, and stimulation

DCs were expanded in vitro from bone marrow precursors using FLT3L as described52. Sorted splenic DCs were isolated essentially as described52. CD8α+ cDC (CD45RA−, CD11c+, CD11b+ and CD8α+) cells were plated at a density of 106/ml in round-bottom plates in media containing GM-CSF (10 ng/ml), mouse IL-4 (10 ng/ml) and rat IFN-γ (10 ng/ml). Stimuli were added and supernatant was collected 18–24 h later for cytokine analyses. VACV70 was purchased from InvivoGen and added to the culture media after being complexed with Lyovec at a concentration of 1 μg/ml. Oligonucleotides purchased from IDT or InvivoGen were also used directly; 2216PS (1 μM), 2216 (1 μM) 1668PS (0.25 μM) and 1668 (1 μM). 2216 subfragments were used at 1 μM. ELISA kits were used to measure the following cytokines: IL-12p70 (Duoset, R&D Systems), IL-6 (BD OptEIA, BD Biosciences), IFN-α (Lumikine, InvivoGen), IFN-λ (mouse IL-28A/B, R&D Systems).

Macrophage analysisMacrophages were elicited into the peritoneal cavity by 2% thioglycolate broth treatment as described53 and stimulated essentially as described above. Ultrapure LPS from Salmonella minnesota R595 (InvivoGen) was used at 100 ng/ml. For the experiments described in Fig. 5h thioglycolate-elicited macrophages (2 × 106) were stimulated for either 3.5 or eight h with CpG-containing stimuli 2216-3′-biotin at 5 μM. Cells were washed twice with cold PBS before cell lysis with 0.5% NP40, 150 mM NaCl and 1 mM EDTA. Nuclei and cell debris were removed by centrifugation and the supernatant was incubated with 20 μl streptavidin magnetic beads (Invitrogen) for 40 min at 4°C with rotation. Beads were isolated using a magnet, washed twice with lysis buffer, and twice with T4 Polynucleotide Kinase (PNK) buffer (NEB). Bound DNA was labeled with γ32P-ATP (Perkin Elmer) with PNK for 30 min at 37°C and beads were washed twice with lysis buffer. Labeled DNA was then eluted with 95% formamide, 10 mM EDTA, heated to 70°C for 10 min and loaded onto 20% TBE/Urea Acrylamide gels. Radiolabeled DNA was exposed to a phosphorimager screen and the signal visualized using a Molecular Dynamics Typhoon phosphorimager and quantitated using ImageQuant software. In some cases, bone marrow derived macrophages were grown in 20% L929 supernatant containing medium over 7 days and stimulated as above.

Macrophage analysis

Macrophages were elicited into the peritoneal cavity by 2% thioglycolate broth treatment as described53 and stimulated essentially as described above. Ultrapure LPS from Salmonella minnesota R595 (InvivoGen) was used at 100 ng/ml. For the experiments described in Fig. 5h thioglycolate-elicited macrophages (2 × 106) were stimulated for either 3.5 or eight h with CpG-containing stimuli 2216-3′-biotin at 5 μM. Cells were washed twice with cold PBS before cell lysis with 0.5% NP40, 150 mM NaCl and 1 mM EDTA. Nuclei and cell debris were removed by centrifugation and the supernatant was incubated with 20 μl streptavidin magnetic beads (Invitrogen) for 40 min at 4°C with rotation. Beads were isolated using a magnet, washed twice with lysis buffer, and twice with T4 Polynucleotide Kinase (PNK) buffer (NEB). Bound DNA was labeled with γ32P-ATP (Perkin Elmer) with PNK for 30 min at 37°C and beads were washed twice with lysis buffer. Labeled DNA was then eluted with 95% formamide, 10 mM EDTA, heated to 70°C for 10 min and loaded onto 20% TBE/Urea Acrylamide gels. Radiolabeled DNA was exposed to a phosphorimager screen and the signal visualized using a Molecular Dynamics Typhoon phosphorimager and quantitated using ImageQuant software. In some cases, bone marrow derived macrophages were grown in 20% L929 supernatant containing medium over 7 days and stimulated as above.

Protein purification and nuclease assayThe N-terminal and transmembrane domain-deleted forms of mouse and human PLD3, PLD3-AA, PLD4, and PLD4-AA were cloned into vector pCEP4 (Thermo) and transfected into HEK293-EBNA cells (Invitrogen). The stable cell lines expressing appropriate proteins were generated by selection with G418 (0.25 mg/ml) and puromycin (0.5 μg/ml) in DMEM complete medium. After selection, cells were expanded in suspension culture in 293 Freestyle media (Life Technologies) containing G418 (0.25 mg/ml) and puromycin (0.5 μg/ml) in a humidified 37°C CO2 incubator (8%), rotating at 135 rpm at 1–2 × 106 cells/ml. Cells were harvested, centrifuged at 5,000g for 20 min and supernatant filtered using 0.2 μm filter (Millipore). Protino Ni-NTA Agarose beads (Macherey-Nagel) were added to the supernatants and rotated for 4 h at 4°C. After incubation, the beads were washed three times in buffer (Tris-HCl 20 mM, pH 7.5, NaCl 250 mM, Imidazole 10 mM) and eluted (Tris-HCl 20 mM, pH 7.5, NaCl 250 mM, Imidazole 200 mM). The eluted proteins were dialyzed overnight in PBS at 4°C and protein concentrations were determined using the BCA protein assay kit (Thermo). Proteins were analyzed for purity by SDS-PAGE stained with Coomassie Brilliant Blue. Nuclease assays for DNA and RNA digestion were carried out for 2 h at 37°C in 50 mM 2-(N-morpholino)ethanesulfonic acid (MES) pH 5.5, 125 mM NaCl, with substrates at 2.5 μM and enzyme at 100 nM. For phosphorothioate-modified substrates, pH was pH 5.0, enzyme concentration 500 nM, and incubation time 6 h unless otherwise noted.

Protein purification and nuclease assay

The N-terminal and transmembrane domain-deleted forms of mouse and human PLD3, PLD3-AA, PLD4, and PLD4-AA were cloned into vector pCEP4 (Thermo) and transfected into HEK293-EBNA cells (Invitrogen). The stable cell lines expressing appropriate proteins were generated by selection with G418 (0.25 mg/ml) and puromycin (0.5 μg/ml) in DMEM complete medium. After selection, cells were expanded in suspension culture in 293 Freestyle media (Life Technologies) containing G418 (0.25 mg/ml) and puromycin (0.5 μg/ml) in a humidified 37°C CO2 incubator (8%), rotating at 135 rpm at 1–2 × 106 cells/ml. Cells were harvested, centrifuged at 5,000g for 20 min and supernatant filtered using 0.2 μm filter (Millipore). Protino Ni-NTA Agarose beads (Macherey-Nagel) were added to the supernatants and rotated for 4 h at 4°C. After incubation, the beads were washed three times in buffer (Tris-HCl 20 mM, pH 7.5, NaCl 250 mM, Imidazole 10 mM) and eluted (Tris-HCl 20 mM, pH 7.5, NaCl 250 mM, Imidazole 200 mM). The eluted proteins were dialyzed overnight in PBS at 4°C and protein concentrations were determined using the BCA protein assay kit (Thermo). Proteins were analyzed for purity by SDS-PAGE stained with Coomassie Brilliant Blue. Nuclease assays for DNA and RNA digestion were carried out for 2 h at 37°C in 50 mM 2-(N-morpholino)ethanesulfonic acid (MES) pH 5.5, 125 mM NaCl, with substrates at 2.5 μM and enzyme at 100 nM. For phosphorothioate-modified substrates, pH was pH 5.0, enzyme concentration 500 nM, and incubation time 6 h unless otherwise noted.

Dinucleotide substrate assayEnzymatic activity assay on dinucleotide substrates was performed as described34. Lyophilized bovine spleen phosphodiesterase II (PDII) and snake venom phosphodiesterase I (PDI) were diluted to 2.5 U/ml and 100 U/ml in water, respectively. Adenosine deaminase (ADA) was resuspended in 1 ml water then diluted to 20 μg/ml in 0.5× PBS. All enzymes were purchased from Worthington. The dinucleotide substrates GpA and ApG were purchased from TriLink Biotechnologies. Final reaction condition for PLD3, PLD4, and bovine spleen PDII was 50 mM MES pH 6.1, 20 mM NaCl. Snake venom PDI enzyme reaction was performed in 50 mM Tris, pH 7.9, 20 mM NaCl. Each reaction consisted of dinucleotide substrate (60 μM), ADA enzyme (2 μg/ml), and either PLD3 (50 nM), PLD4 (50 nM), PDII (0.25 U/ml), or PDI (10 U/ml) in a total volume of 250 μl. The reaction was performed with the Nanodrop 3000C in a heated quartz cuvette (37 °C) and absorbance at 265 nm measured every 15 s. The tested phosphodiesterase was added after four initial absorbance readings (1 min) and absorbance measured for another 5 min. In the absence of an added phosphodiesterase, such as substrate alone, or ADA alone, no change in absorbance was observed throughout the 6 min of recording.

Dinucleotide substrate assay

Enzymatic activity assay on dinucleotide substrates was performed as described34. Lyophilized bovine spleen phosphodiesterase II (PDII) and snake venom phosphodiesterase I (PDI) were diluted to 2.5 U/ml and 100 U/ml in water, respectively. Adenosine deaminase (ADA) was resuspended in 1 ml water then diluted to 20 μg/ml in 0.5× PBS. All enzymes were purchased from Worthington. The dinucleotide substrates GpA and ApG were purchased from TriLink Biotechnologies. Final reaction condition for PLD3, PLD4, and bovine spleen PDII was 50 mM MES pH 6.1, 20 mM NaCl. Snake venom PDI enzyme reaction was performed in 50 mM Tris, pH 7.9, 20 mM NaCl. Each reaction consisted of dinucleotide substrate (60 μM), ADA enzyme (2 μg/ml), and either PLD3 (50 nM), PLD4 (50 nM), PDII (0.25 U/ml), or PDI (10 U/ml) in a total volume of 250 μl. The reaction was performed with the Nanodrop 3000C in a heated quartz cuvette (37 °C) and absorbance at 265 nm measured every 15 s. The tested phosphodiesterase was added after four initial absorbance readings (1 min) and absorbance measured for another 5 min. In the absence of an added phosphodiesterase, such as substrate alone, or ADA alone, no change in absorbance was observed throughout the 6 min of recording.

Studies with 293HEK cellsHEK293-BluehTLR9 cells (InvivoGen) were cultured and stimulated according to the manufacturer’s instructions. Briefly, 100 μl cells/well at a density of 106/ml were cultured in 96-well plates. Stimuli were added and culture supernatants were harvested 16 h later for SEAP1 detection using a kit (InvivoGen). For PLD3-deficient cells, HEK293-BluehTLR9 cells were transiently transfected with a CRISPR/Cas9 construct previously validated for efficacy that carried the guide RNA sequence TCCTCATTCTGGCGGTTGT. Single-cell clones were then isolated and individually screened by sequencing for homozygous mutation of PLD3. PLD3-deficient cells were transduced with lentiviruses encoding human PLD4, human PLD4-AA, mouse PLD3, and mouse PLD3-AA, respectively and cultured for several days before stimulation. ODN stimuli were purchased from InvivoGen or Integrated DNA Technologies. Cells were cultured with 2216-3′-biotin for 6 h at 37 °C then washed and lysed as for macrophages. Streptavidin precipitated DNA was 5′ radiolabeled using polynucleotide kinase and γ32P-ATP. To achieve higher resolution of smaller oligonucleotide fragments, samples in formamide were loaded onto a 20% acrylamide (19:1 Accugel acrylamide (National Diagnostics)) denaturing gel in 50 mM histidine as described50.

Studies with 293HEK cells

HEK293-BluehTLR9 cells (InvivoGen) were cultured and stimulated according to the manufacturer’s instructions. Briefly, 100 μl cells/well at a density of 106/ml were cultured in 96-well plates. Stimuli were added and culture supernatants were harvested 16 h later for SEAP1 detection using a kit (InvivoGen). For PLD3-deficient cells, HEK293-BluehTLR9 cells were transiently transfected with a CRISPR/Cas9 construct previously validated for efficacy that carried the guide RNA sequence TCCTCATTCTGGCGGTTGT. Single-cell clones were then isolated and individually screened by sequencing for homozygous mutation of PLD3. PLD3-deficient cells were transduced with lentiviruses encoding human PLD4, human PLD4-AA, mouse PLD3, and mouse PLD3-AA, respectively and cultured for several days before stimulation. ODN stimuli were purchased from InvivoGen or Integrated DNA Technologies. Cells were cultured with 2216-3′-biotin for 6 h at 37 °C then washed and lysed as for macrophages. Streptavidin precipitated DNA was 5′ radiolabeled using polynucleotide kinase and γ32P-ATP. To achieve higher resolution of smaller oligonucleotide fragments, samples in formamide were loaded onto a 20% acrylamide (19:1 Accugel acrylamide (National Diagnostics)) denaturing gel in 50 mM histidine as described50.

Blood analysisMonocyte, platelet, reticulocyte, red blood cell, and white blood cell counts in whole blood from age-matched male and female animals were determined using an IDEX Procyte DX Machine for blood subset analysis. Plasma concentrations of IFN-γ and CXCL10 of adult Pld4fl/fl and Pld4−/− mice were determined using FlowCytomix beads (eBioscience). Serum from 16–19-day-old animals was analyzed for cytokine concentrations using a kit (Flowcytomix, eBiosciences)

Blood analysis

Monocyte, platelet, reticulocyte, red blood cell, and white blood cell counts in whole blood from age-matched male and female animals were determined using an IDEX Procyte DX Machine for blood subset analysis. Plasma concentrations of IFN-γ and CXCL10 of adult Pld4fl/fl and Pld4−/− mice were determined using FlowCytomix beads (eBioscience). Serum from 16–19-day-old animals was analyzed for cytokine concentrations using a kit (Flowcytomix, eBiosciences)

FerritinMouse Ferritin concentrations were determined from sera isolated from 12–21 day old animals as outlined by the manufacturer of the ELISA kit (Alpco).

Mouse Ferritin concentrations were determined from sera isolated from 12–21 day old animals as outlined by the manufacturer of the ELISA kit (Alpco).

Generation of Bone Marrow ChimerasBone marrow from 16-day-old littermates of CD45.2+
Pld3−/−
Pld4+/−, Pld3+/−
Pld4−/−, or doubly deficient Pld3−/−Pld4−/− genotypes were injected intravenously into lethally irradiated (1000 rad) 8 week old sex matched B6.SJL-CD45.1+ recipients. Eight weeks after transfer, blood and spleen populations and serum cytokines were analyzed.

Generation of Bone Marrow Chimeras

Bone marrow from 16-day-old littermates of CD45.2+
Pld3−/−
Pld4+/−, Pld3+/−
Pld4−/−, or doubly deficient Pld3−/−Pld4−/− genotypes were injected intravenously into lethally irradiated (1000 rad) 8 week old sex matched B6.SJL-CD45.1+ recipients. Eight weeks after transfer, blood and spleen populations and serum cytokines were analyzed.

Flow cytometry analysis of leukocyte compartmentsTissue suspensions were generated by mashing tissue between frosted glass slides. Bone marrow was released from tibia and femurs by a mortar and pestle crushing method. Red blood cells were lysed with ammonium chloride (0.83%) and cells filtered to generate single cell suspensions. Peritoneal cavity cells were isolated by lavage. Cells were pre-incubated with 1 μg/106 cells of Fc Block (2.4G2) prior to staining with antibodies. Splenic NK cell proportions were determined by flow cytometry of splenocytes from age- and sex-matched animals using the following gates (TCRβ− NK1.1+CD49b+). Flow cytometry of DCs was performed from spleens treated with collagenase and DNase I treatment prior to separation on light density gradients (Nycodenz). The following gating strategies were used to identify and quantify different leukocyte subsets. pDCs (CD11c+CD45RA+SiglecH+CD19−TCRβ−); CD8+ cDCs (CD19−TCRβ−CD11c+CD45RA−CD11bloCD8α+); CD11b+ cDCs (CD19−TCRβ−CD11c+CD45RA−CD11bhiCD8α+); CD4+ (TCRβ+CD4+CD8α−); CD8+ (TCRβ+CD4−CD8α+); CD19+ B cells (CD19+B220+); Transitional 1 (TI) B cells (B220+CD93+IgM+CD23−); Transitional 2 (T2) B cell (B220+CD93+IgM+CD23+); Follicular B2 cells (B2) (CD19+B220+CD93−IgM+CD23+); Marginal zone B cells (MZ) (CD19+ B220+CD93−CD23loIgM+CD21hi); Germinal Center (GC) (TCRβ−B220+FAS+ GL7+); Plasma cell (PC) (TCRβ−F480−IgD−CD138+); resident peritoneal B1 cells (B220loCD19+CD43+IgM+); resident peritoneal B2 (B220hiCD19+CD43−IgM+); resident peritoneal macrophages (size, F4/80+CD11b+). Antibodies used in this study are listed in Supplementary Table 3. Gating strategies used for dendritic cells, peritoneal macrophages, and bone marrow chimera B and T cells are shown in Supplementary Figure 8.

Flow cytometry analysis of leukocyte compartments

Tissue suspensions were generated by mashing tissue between frosted glass slides. Bone marrow was released from tibia and femurs by a mortar and pestle crushing method. Red blood cells were lysed with ammonium chloride (0.83%) and cells filtered to generate single cell suspensions. Peritoneal cavity cells were isolated by lavage. Cells were pre-incubated with 1 μg/106 cells of Fc Block (2.4G2) prior to staining with antibodies. Splenic NK cell proportions were determined by flow cytometry of splenocytes from age- and sex-matched animals using the following gates (TCRβ− NK1.1+CD49b+). Flow cytometry of DCs was performed from spleens treated with collagenase and DNase I treatment prior to separation on light density gradients (Nycodenz). The following gating strategies were used to identify and quantify different leukocyte subsets. pDCs (CD11c+CD45RA+SiglecH+CD19−TCRβ−); CD8+ cDCs (CD19−TCRβ−CD11c+CD45RA−CD11bloCD8α+); CD11b+ cDCs (CD19−TCRβ−CD11c+CD45RA−CD11bhiCD8α+); CD4+ (TCRβ+CD4+CD8α−); CD8+ (TCRβ+CD4−CD8α+); CD19+ B cells (CD19+B220+); Transitional 1 (TI) B cells (B220+CD93+IgM+CD23−); Transitional 2 (T2) B cell (B220+CD93+IgM+CD23+); Follicular B2 cells (B2) (CD19+B220+CD93−IgM+CD23+); Marginal zone B cells (MZ) (CD19+ B220+CD93−CD23loIgM+CD21hi); Germinal Center (GC) (TCRβ−B220+FAS+ GL7+); Plasma cell (PC) (TCRβ−F480−IgD−CD138+); resident peritoneal B1 cells (B220loCD19+CD43+IgM+); resident peritoneal B2 (B220hiCD19+CD43−IgM+); resident peritoneal macrophages (size, F4/80+CD11b+). Antibodies used in this study are listed in Supplementary Table 3. Gating strategies used for dendritic cells, peritoneal macrophages, and bone marrow chimera B and T cells are shown in Supplementary Figure 8.

Experimental autoimmune encephalomyelitis induction and monitoringEAE was induced by immunization with MOG35–55 peptide emulsified in complete Freund’s adjuvant followed by Pertussis toxin administration as outlined by the manufacturer (Hooke Laboratories) and scored according to the manufacturer’s instructions. Female animals over 8 weeks old were used. In some cases, C57BL/6 mice congenic for CD45.1 (B6.CD45.1) were lethally irradiated with two doses of 500 rads and were reconstituted with bone marrow from either Pld4fl/fl or Pld4−/− mice for 10 weeks prior to immunization.

Experimental autoimmune encephalomyelitis induction and monitoring

EAE was induced by immunization with MOG35–55 peptide emulsified in complete Freund’s adjuvant followed by Pertussis toxin administration as outlined by the manufacturer (Hooke Laboratories) and scored according to the manufacturer’s instructions. Female animals over 8 weeks old were used. In some cases, C57BL/6 mice congenic for CD45.1 (B6.CD45.1) were lethally irradiated with two doses of 500 rads and were reconstituted with bone marrow from either Pld4fl/fl or Pld4−/− mice for 10 weeks prior to immunization.

CD68+ liver stainsLivers from age and sex matched mice were frozen in optimal cutting temperature (O.C.T., Sakura). Then 7 μm sections were cut, fixed with acetone for 10 min, and stained with anti-CD68 Alexa Fluor 647 (FA-11, BioLegend). After washing in PBS, nuclei were stained with Hoechst 33258 for 5 min prior to mounting in Prolong Diamond Antifade mounting medium (Invitrogen). Sections were imaged using the Keyence BZ-X710 fluorescence microscope. CD68+% and area was determined using FIJI software54. Briefly, color images were split into channels and a threshold set for each channel. The analyze particles tool was used to estimate number of nuclei in image (Hoechst blue channel). The analyze particle tool was then applied to the CD68+ channel (Red channel). The percentage was then calculated. The measure tool was used to calculate the average area of the particles above the threshold in the CD68+ channel.

CD68+ liver stains

Livers from age and sex matched mice were frozen in optimal cutting temperature (O.C.T., Sakura). Then 7 μm sections were cut, fixed with acetone for 10 min, and stained with anti-CD68 Alexa Fluor 647 (FA-11, BioLegend). After washing in PBS, nuclei were stained with Hoechst 33258 for 5 min prior to mounting in Prolong Diamond Antifade mounting medium (Invitrogen). Sections were imaged using the Keyence BZ-X710 fluorescence microscope. CD68+% and area was determined using FIJI software54. Briefly, color images were split into channels and a threshold set for each channel. The analyze particles tool was used to estimate number of nuclei in image (Hoechst blue channel). The analyze particle tool was then applied to the CD68+ channel (Red channel). The percentage was then calculated. The measure tool was used to calculate the average area of the particles above the threshold in the CD68+ channel.

Mass spectrometryPhosphodiesterase II (Worthington Biochemical Corp. Cat#: LS003602, LOT #: 36m16915) was fractionated on a Superdex 200 column in PBS and two adjacent fractions with peak enzymatic activity were pooled. The solution sample was reduced (200 mM DTT), alkylated (200 mM iodoacetamide) and digested with trypsin overnight using an estimated 1:30 enzyme to substrate ratio before being analyzed by nano-LC-MS/MS at the TSRI core facility. The MS/MS raw data were searched against the custom sequence database, which included the sequence [NP_001071509.1 phospholipase D3 [Bos taurus]] and the NCBInr database. Bovine PLD3 protein was identified with 7 peptides and 17% sequence coverage and the sequence [XP_015324054.1 PREDICTED: phospholipase D4 isoform X2 [Bos taurus]] was not identified. Other proteins also identified with at least five different peptides are listed in Supplementary Fig. 4.

Mass spectrometry

Phosphodiesterase II (Worthington Biochemical Corp. Cat#: LS003602, LOT #: 36m16915) was fractionated on a Superdex 200 column in PBS and two adjacent fractions with peak enzymatic activity were pooled. The solution sample was reduced (200 mM DTT), alkylated (200 mM iodoacetamide) and digested with trypsin overnight using an estimated 1:30 enzyme to substrate ratio before being analyzed by nano-LC-MS/MS at the TSRI core facility. The MS/MS raw data were searched against the custom sequence database, which included the sequence [NP_001071509.1 phospholipase D3 [Bos taurus]] and the NCBInr database. Bovine PLD3 protein was identified with 7 peptides and 17% sequence coverage and the sequence [XP_015324054.1 PREDICTED: phospholipase D4 isoform X2 [Bos taurus]] was not identified. Other proteins also identified with at least five different peptides are listed in Supplementary Fig. 4.

RNA expression analysis and sequencingRNA expression data in Fig. 4c was from the BioGPS MOE430 Gene Atlas Data set55, 56. For the sequencing analysis in Fig. 1k, total RNA was isolated from splenocytes using a kit (RNA easy, Qiagen). RNA was prepared into RNAseq libraries using the NEBNext Ultra Directional RNA Library Prep Kit for Illumina following the manufacturer’s recommended protocol. The libraries were then PCR amplified 15 cycles using barcoded PCR primers, purified and size selected using AMPure XP Beads before loading onto an Illumina NextSeq500 for 75 base single read sequencing. A total of 16.3–19.6 million “passed filter” reads were obtained for each of the 6 samples. Reads were compared to the mouse genome build mm10 and quantitated using the program Salmon. DC RNAseq data described in Fig. 4d were generated as follows. DC subsets were purified from spleens of 10–14 C57BL/6 mice/sample as described57. Briefly, cell sorting was performed on a BD Influx™ machine. RNA was extracted using RNeasy Mini Kit (QIAGEN) and residual genomic DNA removed using RNase-free DNase (QIAGEN). Sequencing libraries were prepared using the Illumina TruSeq Stranded mRNA kit and sequencing was carried out by Micromon using High-Output SBS on the Illumina NextSeq 500. RNAseq data analyses were performed in Degust (http://degust.erc.monash.edu/ Version: 3.1.0) by Monash Bioinformatics platform, David R Powell & Adele Barugahare.

RNA expression analysis and sequencing

RNA expression data in Fig. 4c was from the BioGPS MOE430 Gene Atlas Data set55, 56. For the sequencing analysis in Fig. 1k, total RNA was isolated from splenocytes using a kit (RNA easy, Qiagen). RNA was prepared into RNAseq libraries using the NEBNext Ultra Directional RNA Library Prep Kit for Illumina following the manufacturer’s recommended protocol. The libraries were then PCR amplified 15 cycles using barcoded PCR primers, purified and size selected using AMPure XP Beads before loading onto an Illumina NextSeq500 for 75 base single read sequencing. A total of 16.3–19.6 million “passed filter” reads were obtained for each of the 6 samples. Reads were compared to the mouse genome build mm10 and quantitated using the program Salmon. DC RNAseq data described in Fig. 4d were generated as follows. DC subsets were purified from spleens of 10–14 C57BL/6 mice/sample as described57. Briefly, cell sorting was performed on a BD Influx™ machine. RNA was extracted using RNeasy Mini Kit (QIAGEN) and residual genomic DNA removed using RNase-free DNase (QIAGEN). Sequencing libraries were prepared using the Illumina TruSeq Stranded mRNA kit and sequencing was carried out by Micromon using High-Output SBS on the Illumina NextSeq 500. RNAseq data analyses were performed in Degust (http://degust.erc.monash.edu/ Version: 3.1.0) by Monash Bioinformatics platform, David R Powell & Adele Barugahare.

Statistical analysesAnalysis of differential gene expression was carried out using the program Edge. Other data comparisons used the program Prism (Graph Pad), with correction for multiple comparisons. When two groups were compared, an unpaired, two-tailed T test was used. Actual P values are provided in Supplementary Table 4. Additional information is provided in the Life Sciences Reporting Summary.

Statistical analyses

Analysis of differential gene expression was carried out using the program Edge. Other data comparisons used the program Prism (Graph Pad), with correction for multiple comparisons. When two groups were compared, an unpaired, two-tailed T test was used. Actual P values are provided in Supplementary Table 4. Additional information is provided in the Life Sciences Reporting Summary.

Supplementary Material123

Supplementary Material
