---
pmid: '35976721'
title: The adaptor protein chaperone AAGAB stabilizes AP-4 complex subunits.
authors:
- Mattera R
- De Pace R
- Bonifacino JS
journal: Mol Biol Cell
year: '2022'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC9635299
doi: 10.1091/mbc.E22-05-0177
---

# The adaptor protein chaperone AAGAB stabilizes AP-4 complex subunits.
**Authors:** Mattera R, De Pace R, Bonifacino JS
**Journal:** Mol Biol Cell (2022)
**DOI:** [10.1091/mbc.E22-05-0177](https://doi.org/10.1091/mbc.E22-05-0177)
**PMC:** [PMC9635299](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9635299/)

## Abstract

1. Mol Biol Cell. 2022 Oct 1;33(12):ar109. doi: 10.1091/mbc.E22-05-0177. Epub
2022  Aug 17.

The adaptor protein chaperone AAGAB stabilizes AP-4 complex subunits.

Mattera R(1), De Pace R(1), Bonifacino JS(1).

Author information:
(1)Neurosciences and Cellular and Structural Biology Division, Eunice Kennedy 
Shriver National Institute of Child Health and Human Development, National 
Institutes of Health, Bethesda, MD 20892.

Adaptor protein 4 (AP-4) is a heterotetrameric complex composed of ε, β4, μ4, 
and σ4 subunits that mediates export of a subset of transmembrane cargos, 
including autophagy protein 9A (ATG9A), from the trans-Golgi network (TGN). AP-4 
has received particular attention in recent years because mutations in any of 
its subunits cause a complicated form of hereditary spastic paraplegia referred 
to as "AP-4-deficiency syndrome." The identification of proteins that interact 
with AP-4 has shed light on the mechanisms of AP-4-dependent cargo sorting and 
distribution within the cell. However, the mechanisms by which the AP-4 complex 
itself is assembled have remained unknown. Here, we report that the alpha- and 
gamma-adaptin-binding protein (AAGAB, also known as p34) binds to and stabilizes 
the AP-4 ε and σ4 subunits, thus promoting complex assembly. The physiological 
importance of these interactions is underscored by the observation that 
AAGAB-knockout cells exhibit reduced levels of AP-4 subunits and accumulation of 
ATG9A at the TGN like those in cells with mutations in AP-4-subunit genes. These 
findings demonstrate that AP-4 assembly is not spontaneous but AAGAB-assisted, 
further contributing to the understanding of an adaptor protein complex that is 
critically involved in development of the central nervous system.

DOI: 10.1091/mbc.E22-05-0177
PMCID: PMC9635299
PMID: 35976721 [Indexed for MEDLINE]

## Full Text

INTRODUCTIONHeterotetrameric adaptor protein (AP) complexes are components of protein coats that mediate sorting of transmembrane proteins in the endomembrane system of eukaryotic cells (reviewed by Dell’Angelica and Bonifacino, 2019; Sanger et al., 2019). Mammals have five distinct AP complexes named AP-1, AP-2, AP-3, AP-4, and AP-5. Each complex comprises two large subunits (γ, α, δ, ε, or ζ, and β1, β2, β3, β4, or β5, respectively), one medium-sized subunit (μ1, μ2, μ3, μ4, or μ5), and one small subunit (σ1, σ2, σ3, σ4, or σ5). Some of these subunits occur as multiple isoforms encoded by different genes. AP complexes localize to different cellular compartments, including the trans-Golgi network (TGN; AP-1 and AP-4), the plasma membrane (AP-2), tubular/recycling endosomes (AP-1 and AP-3), and late endosomes (AP-5) (Dell’Angelica and Bonifacino, 2019; Sanger et al., 2019).The AP-4 complex is composed of ε, β4, μ4, and σ4 subunits (Dell’Angelica et al., 1999a; Hirst et al., 1999), products of the AP4E1, AP4B1, AP4M1, and AP4S1 genes, respectively (Figure 1A). This complex has received particular attention in recent years because mutations in the genes encoding each of its subunits cause a subset of autosomal-recessive complicated hereditary spastic paraplegias (abbreviated HSP or SPG) collectively referred to as “AP-4-deficiency syndrome” (Verkerk et al., 2009; Abou Jamra et al., 2011; reviewed by Ebrahimi-Fakhari et al., 2018; Dell’Angelica and Bonifacino, 2019; Sanger et al., 2019; Mattera et al., 2020a). This syndrome features functional alterations such as upper and lower limb spasticity, intellectual disability and seizures, and neuroanatomical defects such as microcephaly, thin corpus callosum, and ventriculomegaly (Verkerk et al., 2009; Abou Jamra et al., 2011; Ebrahimi-Fakhari et al., 2018; Mattera et al., 2020a). AP-4 localizes to the TGN in an ARF-dependent manner (Boehm et al., 2001) and mediates export of transmembrane proteins (i.e., “cargos”), including ATG9A (Mattera et al., 2017; De Pace et al., 2018; Davies et al., 2018; Ivankovic et al., 2020; Behne et al., 2020), SERINC1 and SERINC3 (Davies et al., 2018), and DAGLB (Davies et al., 2022), from the TGN toward the cell periphery. In addition, AP-4 interacts with several accessory proteins, including tepsin (Borner et al., 2012), RUSC1 and RUSC2 (Davies et al., 2018), and the FTS-Hook-FHIP (FHF) complex (Mattera et al., 2020b). The function of tepsin is currently unknown, although it has the potential to cross-link AP-4 through multivalent interactions with several of its subunits (Mattera et al., 2015, 2020a; Frazier et al., 2016). The interactions of AP-4 with RUSC2 and the FHF complex promote anterograde and retrograde transport of ATG9A-containing vesicles by coupling to the microtubule motors kinesin-1 and dynein-dynactin, respectively (Davies et al., 2018; Mattera et al., 2020b; Guardia et al., 2021). Together, these interactions allow AP-4 to mediate both packaging of cargos into TGN-derived transport vesicles and distribution of these vesicles throughout the cytoplasm.FIGURE 1:Interaction of AAGAB with AP-4 subunits. (A) Schematic representation of the AP-4 complex depicting its ε, β4, μ4, and σ4 subunits, and the trunk, hinge, and ear domains of ε and β4. The HGNC names for the genes encoding the four subunits are indicated in parentheses. The trunk domains of ε and β4, together with the μ4 and σ4 subunits, form the core of the AP-4 complex. Also shown are the predicted sandwich and platform subdomains of the ε ear and the platform subdomain of β4 (Mattera et al., 2011, 2015; Eletsky et al., 2013; Frazier et al., 2016). (B) HEK293T cells were transiently transfected with plasmids encoding either TSF-myrlysin (negative control) or TSF-AP-4 ε. The TSF-tagged proteins in the cell lysates were pulled down with Strep-Tactin beads and eluted. Samples of cell lysates and pull-down eluates were subjected to SDS–PAGE and immunoblotting with rabbit anti-AAGAB (Top panels) or mouse anti-FLAG (for the TSF tag; Bottom panels). The left panels show the expression of endogenous AAGAB, Top, and recombinant TSF tagged proteins, Bottom, in lysates of transfected cells (note the higher expression of TSF-myrlysin compared with TSF-AP-4 ε). The right panels show the pulldown of AAGAB, Top, and TSF-tagged proteins, Bottom, by Strep-Tactin beads. The positions of molecular mass markers (in kDa) are shown on the left. This experiment demonstrated that AAGAB binds to TSF-AP-4 ε but not to TSF-myrlysin. (C) Y2H analysis of the interaction of AAGAB with subunits of different AP complexes. The interaction of tepsin with the ε, β4, and μ4 subunits (Borner et al., 2012; Mattera et al, 2015, 2020b; Frazier et al., 2016) was used as a positive control. AP subunits were subcloned into GAL4 transcriptional activation domain (AD) Y2H plasmids, whereas AAGAB and tepsin were subcloned into GAL4 DNA-binding domain (BD) Y2H plasmids. Yeast double transformants coexpressing AD-AP subunit fusions with BD-p53 and BD-AAGAB or BD-tepsin fusions with AD-SV40 large T antigen (T-Ag) were used as negative controls. Double transformants expressing BD-p53 and AD-T-Ag fusions provided an additional positive control. Double transformants were plated on medium lacking histidine, leucine, and tryptophan (–His) to detect interactions, and in medium lacking leucine and tryptophan (+His) to control for viability and seeding. The –His plates were supplemented with 1 mM 3-amino-1,2,4 triazole (3AT), a competitive inhibitor of the His3 protein, to minimize background growth due to non-specific interactions. The results demonstrate that AAGAB binds not only AP-1-γ and AP-2-α, as previously reported (Gulbranson et al., 2019; Wan et al., 2021), but also AP-4 ε and σ4.In a previous tandem affinity purification and mass spectrometry (TAP-MS) screen for AP-4 interactors (Mattera et al., 2015; Mattera et al., 2017), we identified the alpha- and gamma-adaptin-binding (AAGAB) protein (also known as p34 in mammals and Irc6p in yeast; Page et al., 1999; Gorynia et al., 2012) as a potential AP-4 partner. The significance of this interaction, however, was not examined. AAGAB is a protein originally identified in a yeast two-hybrid (Y2H) screen using the AP-1 γ subunit as a bait (Page et al., 1999) (all specific references to AP-1 γ and AP-2 α in this article correspond to the γ1 and αC isoforms, respectively). AAGAB comprises an N-terminal low–molecular weight (LMW) GTPase fold lacking signature motifs for high-affinity GTP binding and GTPase activity, and a C-terminal adaptin-binding domain (Gorynia et al., 2012; Pöhler et al., 2012; Zhou et al., 2019). Recently, AAGAB was identified in a CRISPR-Cas9 genetic screen for altered clathrin-mediated endocytosis and subsequently shown to function as a chaperone for the assembly of the AP-1 and AP-2 complexes (Gulbranson et al., 2019; Wan et al., 2021). Furthermore, heterozygous, nonsense or frameshift mutations in the AAGAB gene were shown to cause an autosomal-dominant keratinization disorder known as punctate palmoplantar keratoderma type 1 (PPKP1) or Buschke–Fischer–Brauer disease (Giehl et al., 2012; Pöhler et al., 2012), probably resulting from impaired assembly of AP-1 and/or AP-2.We have now investigated the significance of the AP-4–AAGAB interaction identified in our original TAP-MS screen. We find that (i) AAGAB directly binds to the ε and σ4 subunits of AP-4, (ii) AAGAB stabilizes both endogenous and recombinant AP-4 subunits by preventing their proteasomal degradation, and (iii) AAGAB-knockout (KO) cells exhibit phenotypic alterations mimicking AP-4 deficiency. Our study thus extends the chaperone role of AAGAB to AP-4, and provides new insights into the assembly of AP-4 that may be relevant to the understanding of AP-4 deficiency.

INTRODUCTION

Heterotetrameric adaptor protein (AP) complexes are components of protein coats that mediate sorting of transmembrane proteins in the endomembrane system of eukaryotic cells (reviewed by Dell’Angelica and Bonifacino, 2019; Sanger et al., 2019). Mammals have five distinct AP complexes named AP-1, AP-2, AP-3, AP-4, and AP-5. Each complex comprises two large subunits (γ, α, δ, ε, or ζ, and β1, β2, β3, β4, or β5, respectively), one medium-sized subunit (μ1, μ2, μ3, μ4, or μ5), and one small subunit (σ1, σ2, σ3, σ4, or σ5). Some of these subunits occur as multiple isoforms encoded by different genes. AP complexes localize to different cellular compartments, including the trans-Golgi network (TGN; AP-1 and AP-4), the plasma membrane (AP-2), tubular/recycling endosomes (AP-1 and AP-3), and late endosomes (AP-5) (Dell’Angelica and Bonifacino, 2019; Sanger et al., 2019).

The AP-4 complex is composed of ε, β4, μ4, and σ4 subunits (Dell’Angelica et al., 1999a; Hirst et al., 1999), products of the AP4E1, AP4B1, AP4M1, and AP4S1 genes, respectively (Figure 1A). This complex has received particular attention in recent years because mutations in the genes encoding each of its subunits cause a subset of autosomal-recessive complicated hereditary spastic paraplegias (abbreviated HSP or SPG) collectively referred to as “AP-4-deficiency syndrome” (Verkerk et al., 2009; Abou Jamra et al., 2011; reviewed by Ebrahimi-Fakhari et al., 2018; Dell’Angelica and Bonifacino, 2019; Sanger et al., 2019; Mattera et al., 2020a). This syndrome features functional alterations such as upper and lower limb spasticity, intellectual disability and seizures, and neuroanatomical defects such as microcephaly, thin corpus callosum, and ventriculomegaly (Verkerk et al., 2009; Abou Jamra et al., 2011; Ebrahimi-Fakhari et al., 2018; Mattera et al., 2020a). AP-4 localizes to the TGN in an ARF-dependent manner (Boehm et al., 2001) and mediates export of transmembrane proteins (i.e., “cargos”), including ATG9A (Mattera et al., 2017; De Pace et al., 2018; Davies et al., 2018; Ivankovic et al., 2020; Behne et al., 2020), SERINC1 and SERINC3 (Davies et al., 2018), and DAGLB (Davies et al., 2022), from the TGN toward the cell periphery. In addition, AP-4 interacts with several accessory proteins, including tepsin (Borner et al., 2012), RUSC1 and RUSC2 (Davies et al., 2018), and the FTS-Hook-FHIP (FHF) complex (Mattera et al., 2020b). The function of tepsin is currently unknown, although it has the potential to cross-link AP-4 through multivalent interactions with several of its subunits (Mattera et al., 2015, 2020a; Frazier et al., 2016). The interactions of AP-4 with RUSC2 and the FHF complex promote anterograde and retrograde transport of ATG9A-containing vesicles by coupling to the microtubule motors kinesin-1 and dynein-dynactin, respectively (Davies et al., 2018; Mattera et al., 2020b; Guardia et al., 2021). Together, these interactions allow AP-4 to mediate both packaging of cargos into TGN-derived transport vesicles and distribution of these vesicles throughout the cytoplasm.

Interaction of AAGAB with AP-4 subunits. (A) Schematic representation of the AP-4 complex depicting its ε, β4, μ4, and σ4 subunits, and the trunk, hinge, and ear domains of ε and β4. The HGNC names for the genes encoding the four subunits are indicated in parentheses. The trunk domains of ε and β4, together with the μ4 and σ4 subunits, form the core of the AP-4 complex. Also shown are the predicted sandwich and platform subdomains of the ε ear and the platform subdomain of β4 (Mattera et al., 2011, 2015; Eletsky et al., 2013; Frazier et al., 2016). (B) HEK293T cells were transiently transfected with plasmids encoding either TSF-myrlysin (negative control) or TSF-AP-4 ε. The TSF-tagged proteins in the cell lysates were pulled down with Strep-Tactin beads and eluted. Samples of cell lysates and pull-down eluates were subjected to SDS–PAGE and immunoblotting with rabbit anti-AAGAB (Top panels) or mouse anti-FLAG (for the TSF tag; Bottom panels). The left panels show the expression of endogenous AAGAB, Top, and recombinant TSF tagged proteins, Bottom, in lysates of transfected cells (note the higher expression of TSF-myrlysin compared with TSF-AP-4 ε). The right panels show the pulldown of AAGAB, Top, and TSF-tagged proteins, Bottom, by Strep-Tactin beads. The positions of molecular mass markers (in kDa) are shown on the left. This experiment demonstrated that AAGAB binds to TSF-AP-4 ε but not to TSF-myrlysin. (C) Y2H analysis of the interaction of AAGAB with subunits of different AP complexes. The interaction of tepsin with the ε, β4, and μ4 subunits (Borner et al., 2012; Mattera et al, 2015, 2020b; Frazier et al., 2016) was used as a positive control. AP subunits were subcloned into GAL4 transcriptional activation domain (AD) Y2H plasmids, whereas AAGAB and tepsin were subcloned into GAL4 DNA-binding domain (BD) Y2H plasmids. Yeast double transformants coexpressing AD-AP subunit fusions with BD-p53 and BD-AAGAB or BD-tepsin fusions with AD-SV40 large T antigen (T-Ag) were used as negative controls. Double transformants expressing BD-p53 and AD-T-Ag fusions provided an additional positive control. Double transformants were plated on medium lacking histidine, leucine, and tryptophan (–His) to detect interactions, and in medium lacking leucine and tryptophan (+His) to control for viability and seeding. The –His plates were supplemented with 1 mM 3-amino-1,2,4 triazole (3AT), a competitive inhibitor of the His3 protein, to minimize background growth due to non-specific interactions. The results demonstrate that AAGAB binds not only AP-1-γ and AP-2-α, as previously reported (Gulbranson et al., 2019; Wan et al., 2021), but also AP-4 ε and σ4.

In a previous tandem affinity purification and mass spectrometry (TAP-MS) screen for AP-4 interactors (Mattera et al., 2015; Mattera et al., 2017), we identified the alpha- and gamma-adaptin-binding (AAGAB) protein (also known as p34 in mammals and Irc6p in yeast; Page et al., 1999; Gorynia et al., 2012) as a potential AP-4 partner. The significance of this interaction, however, was not examined. AAGAB is a protein originally identified in a yeast two-hybrid (Y2H) screen using the AP-1 γ subunit as a bait (Page et al., 1999) (all specific references to AP-1 γ and AP-2 α in this article correspond to the γ1 and αC isoforms, respectively). AAGAB comprises an N-terminal low–molecular weight (LMW) GTPase fold lacking signature motifs for high-affinity GTP binding and GTPase activity, and a C-terminal adaptin-binding domain (Gorynia et al., 2012; Pöhler et al., 2012; Zhou et al., 2019). Recently, AAGAB was identified in a CRISPR-Cas9 genetic screen for altered clathrin-mediated endocytosis and subsequently shown to function as a chaperone for the assembly of the AP-1 and AP-2 complexes (Gulbranson et al., 2019; Wan et al., 2021). Furthermore, heterozygous, nonsense or frameshift mutations in the AAGAB gene were shown to cause an autosomal-dominant keratinization disorder known as punctate palmoplantar keratoderma type 1 (PPKP1) or Buschke–Fischer–Brauer disease (Giehl et al., 2012; Pöhler et al., 2012), probably resulting from impaired assembly of AP-1 and/or AP-2.

We have now investigated the significance of the AP-4–AAGAB interaction identified in our original TAP-MS screen. We find that (i) AAGAB directly binds to the ε and σ4 subunits of AP-4, (ii) AAGAB stabilizes both endogenous and recombinant AP-4 subunits by preventing their proteasomal degradation, and (iii) AAGAB-knockout (KO) cells exhibit phenotypic alterations mimicking AP-4 deficiency. Our study thus extends the chaperone role of AAGAB to AP-4, and provides new insights into the assembly of AP-4 that may be relevant to the understanding of AP-4 deficiency.

RESULTSAAGAB directly binds to AP-4 ε and σ4 subunitsIn previous work (Mattera et al., 2015; Mattera et al., 2017), we performed a TAP-MS identification of AP-4 interactors using two Strep/one FLAG (TSF)-tagged AP-4 ε (TSF-AP-4 ε) stably expressed in H4 human neuroglioma cells (see Supplemental Table S1). We subsequently confirmed and characterized the interactions of AP-4 with top hits in that analysis, including tepsin (Mattera et al., 2015), ATG9A (Mattera et al., 2017; De Pace et al., 2018), and subunits of the FHF complex (Mattera et al, 2020b). Going down the list of hits, we noticed the presence of AAGAB, albeit with a low peptide count (Supplemental Table S1). The previously demonstrated role of AAGAB as a chaperone in the assembly of AP-1 and AP-2 (Gulbranson et al., 2019; Wan et al., 2021) prompted us to investigate the functional significance of the AP-4–AAGAB interaction.We confirmed the AP-4–AAGAB interaction in pull-down experiments using lysates of HEK293T cells transiently transfected with plasmids encoding either TSF-AP-4 ε or TSF-myrlysin (a subunit of the BORC complex, also termed BORCS5 or LOH12CR1; Pu et al., 2015), as a negative control. We observed that TSF-AP-4 ε, but not TSF-myrlysin, pulled down endogenous AAGAB (Figure 1B). This result demonstrated the specificity of the interaction between AAGAB and AP-4 ε.Next, we asked if this interaction represented direct binding of AAGAB to AP-4 subunits. To address this question, we carried out Y2H analyses using constructs encoding (i) the GAL4 transcriptional activation domain (AD) fused to each of the four AP-4 subunits and (ii) the GAL4 DNA-binding domain (BD) fused to AAGAB (Figure 1C). Given the previously reported interactions of AAGAB with AP-1 γ and AP-2 α (Page et al., 1999; Gulbranson et al., 2019; Wan et al., 2021), we included AD fusions to the AP-1 γ, AP-2 α and AP-3 δ subunits as controls. We also used as an additional control a BD fusion to tepsin, which binds to the ε, β4, and μ4 subunits of AP-4 (Borner et al., 2012; Mattera et al., 2015; Frazier et al., 2016; Mattera et al., 2020b). The results of these experiments showed that AAGAB binds to the ε and σ4 subunits, but not the β4 and μ4 subunits, of AP-4 (Figure 1C). The control samples confirmed interactions of AAGAB with AP-1 γ and AP-2 α, but not AP-3 δ, and of tepsin with AP-4 ε, β4, and μ4 (Figure 1C). Thus, AAGAB binds directly to a subset of AP-4 subunits, ε and σ4, which constitute an AP-4 ε-σ4 hemicomplex (Mattera et al., 2011).Structural determinants of the AP-4–AAGAB interactionTo analyze the structural determinants of the interaction of AP-4 ε with AAGAB, we performed additional Y2H analyses with AD fusions to AP-4 ε deletion mutants (Figure 2, A and B). We found that the minimal fragment of AP-4 ε that binds to AAGAB spans amino acid residues 1–138, corresponding to the N-terminal part of the “trunk” domain (Figure 1A; Figure 2, A–C). This mode of interaction contrasted with that of tepsin, which binds to the C-terminal “ear” domain of AP-4 ε (residues 883–1137; Figure 1A; Figure 2, A and B; Mattera et al., 2015).FIGURE 2:Determinants of AP-4 ε and σ4 interactions with AAGAB. (Α) Consensus secondary structure prediction of human AP-4 ε according to the NPS@ server (https://npsa-prabi.ibcp.fr/cgi-bin/npsa_automat.pl?page = /NPSA/npsa_seccons.html; Combet et al., 2000) indicating its trunk, hinge, and ear domains. Blue, red, and magenta coloring represents α-helix, β-sheet, and random coil structure, respectively. The deletion mutants used in the Y2H analysis shown in panel B are also depicted. (B) Y2H analysis of the interaction of AAGAB with different AP-4 ε deletion constructs. Experiments were performed as explained for Figure 1C. The results demonstrate that AAGAB binds to the N-terminal portion (residues 1–138) of the AP-4 ε trunk. In contrast, the tepsin control interacts with the AP-4 ε 727–1137 (part of hinge plus ear domains) and 883–1137 (ear domain) constructs, as previously reported (Mattera et al., 2015). (C) AlphaFold (https://alphafold.ebi.ac.uk; Jumper et al., 2021) prediction of the tertiary structure of the human AP-4 ε trunk domain (human AP-4 ε residues 1–599). The AP-4 ε 1–138 region containing the AAGAB binding site is depicted in gray. Also shown are the N-and C-termini of the trunk domain. The image was rendered using PyMol (https://pymol.org/). (D) AlphaFold prediction of the tertiary structure of mouse AP-4 σ4 showing the position of the conserved V88 and I103 residues that were mutated to D and S, respectively. Also shown are the N-and C-termini. The image was rendered using PyMol. (E) Y2H analysis of the interaction of AAGAB with AP-4 σ4 mutants performed as explained for Figure 1C. Notice that the V88D and I03S substitutions abrogate binding of AP-4 σ4 to both AAGAB and AP-4 ε. (F) Y2H analysis of the interaction of AAGAB with σ subunits (including isoforms) of AP-1, AP-2, AP-3, and AP-4. Notice the interaction of AAGAB with all σ subunits tested except AP-3 σ3A and AP-3 σ3B (Top panel). The AD-fusions of AP-3 σ3A and AP-3 σ3B are able to bind to the BD-AP-3 δ control (Bottom panel), indicating that their lack of interaction with AAGAB is not due to problems with their expression, stability or targeting to the yeast nucleus.We also used the Y2H system to examine structural determinants of the interaction of AP-4 σ4 with AAGAB (Figure 2, D and E). A previous study (Gulbranson et al., 2019) showed that assembly of AP-2 σ2 with an AAGAB-AP-2 α-complex requires AP-2 σ2 residues involved in the recognition of acidic-dileucine signals fitting the consensus motif (D/E)XXXL(L/I), where X signifies any amino acid. In general, these signals bind to the AP-1 γ-σ1, AP-2 α-σ2, and AP-3 δ-σ3 hemicomplexes (Janvier et al., 2003; Chaudhuri et al., 2007; Doray et al., 2007) mainly through conserved residues on σ1, σ2, and σ3 (Kelly et al., 2008; Mattera et al., 2011; Jia et al., 2014, Ren et al., 2014). An additional contribution is made by an R residue near the N-terminus of the γ, α, and δ subunits (Kelly et al., 2008; Mattera et al, 2011; Jia et al., 2014; Ren et al., 2014). AP-4 ε-σ4 has the conserved residues on σ4, including V88 and I103 (Figure 2D), but lacks the corresponding R residue in AP-4 ε, likely explaining why the AP-4 ε-σ4 hemicomplex has not been found to bind acidic-dileucine signals (Mattera et al., 2011). This difference notwithstanding, we tested whether the conserved residues in AP-4 σ4 are required for interaction with AAGAB. In line with previous findings on AP-2 σ2 (Gulbranson et al., 2019), V88D, I103S, or double V88D-I103S mutations in AP-4 σ4 abrogated the interaction with AAGAB (Figure 2, D and E). However, these mutations also abolished the interaction of AP-4 σ4 with AP-4 ε (Figure 2E), suggesting that they render the mutant proteins unstable, either intrinsically or because they cannot interact with the yeast AAGAB ortholog Irc6p. Furthermore, we cannot exclude that the corresponding mutant fusion proteins are not targeted to the yeast nucleus where the two-hybrid interactions activate GAL4-driven transcription. These uncertainties prevented us from concluding whether there is a conserved mode of interaction of AAGAB with AP-2 σ2 and AP-4 σ4.The conservation of the putative acidic dileucine–binding residues on the σ1-4 subunits prompted us to analyze whether all these subunits bind AAGAB. We found that the σ1A, σ1B, and σ1C isoforms of AP-1 σ1, as well as the σ2 subunit of AP-2, bind to AAGAB, whereas the σ3A and σ3B subunit isoforms of AP-3 do not (Figure 2F). In this case, the activity of the σ3A and σ3B constructs was confirmed by their interaction with AP-3 δ (Figure 2F). These findings indicated that AAGAB can bind not only the large AP-1 γ, AP-2 α, and AP-4 ε subunits, but also the cognate σ1, σ2, and σ4 subunits. In contrast, AAGAB does not bind either the δ or σ3 subunits of AP-3.Finally, we used the Y2H system to analyze the AAGAB domains involved in binding to AP-4 ε and σ4 (Figure 3, A–C). AAGAB comprises an N-terminal LMW GTPase-like fold (residues 1–155) and a C-terminal adaptin-binding domain (residues 156–315; Figure 3, A and B; Gorynia et al., 2012; Pöhler et al., 2012; Zhou et al., 2019). The Y2H assays showed that the AAGAB 156–315 fragment was sufficient for binding to AP-4 ε, as well as AP-1 γ and AP-2 α (Figure 3C). In contrast, neither the 1–155 fragment nor the 156–315 fragment was sufficient for binding to AP-4 σ4 (Figure 3C), indicating that this subunit requires both AAGAB domains for interaction.FIGURE 3:Identification of AAGAB domains that bind to AP-4 ε and σ4. (A) Consensus secondary structure prediction of human AAGAB according to the NPS@ server. The N-terminal low-molecular-weight (LMW) GTPase fold domain and C-terminal adaptin-binding domain are shown, along with the two deletion constructs used in Y2H analysis. Predictions are depicted as indicated in the legend to Figure 2A. (B) AlphaFold prediction of the human AAGAB tertiary structure. Depicted in blue, cyan, yellow, and orange are regions with very high (>90), confident (>70 and <90), low (>50 and <70), and very low (<50) residue confidence scores (pLDDT), respectively. N- and C-termini are indicated. (C) Y2H analysis showing that AP-4 ε binds to the AAGAB adaptin-binding domain (156–315 fragment), whereas AP-4 σ4 requires residues on both the N-terminal LMW GTPase fold and the C-terminal adaptin-binding domains. Y2H assays were performed as described for Figure 1C.AAGAB KO decreases the levels of AP-4 subunitsHaving established that AAGAB directly binds to AP-4 subunits, we analyzed the consequences of ablating the AAGAB gene on AP-4 levels and function. To this end, we generated AAGAB-KO HeLa cell lines using the CRISPR-Cas9 system. We alternatively targeted exons 1 and 2 of the AAGAB gene (Supplemental Figure S1A) and obtained several clones of AAGAB-KO cells that were subsequently analyzed by immunoblotting (Figure 4 and Supplemental Figure S1B). We found that targeting either exon completely abrogated AAGAB protein expression in all tested clones (Figure 4 and Supplemental Figure S1B). Importantly, all AAGAB-KO clones exhibited decreased levels of AP-4 ε and AP-4 β4 (Figure 4, Supplemental Figure S1B). We could not assess the levels of AP-4 μ4 and σ4 because of the lack of suitable antibodies, but we expect them to be reduced, given the need of all subunits for stability of AP complexes (Dell’Angelica et al., 1999b; Davies et al., 2018; De Pace et al., 2018). Despite some variability in the different clones, we also observed an increase in the levels of the AP-4 cargo ATG9A in AAGAB-KO cells (Figure 4, Supplemental Figure S1B), similar to that previously observed in AP-4-deficient cells (Mattera et al., 2017; Davies et al., 2018; De Pace et al., 2018; Behne et al., 2020). In agreement with previous findings (Gulbranson et al., 2019; Wan et al., 2021), the AAGAB-KO HeLa cells also displayed reduced levels of AP-1 γ (Figure 4; Supplemental Figure S1B). These findings demonstrated that AAGAB is required for maintenance of normal levels of AP-4 ε and β4.FIGURE 4:Reduced levels of AP-4 subunits and increased levels of ATG9A in AAGAB-KO cells. Immunoblot analysis of WT and several clones of AAGAB-KO HeLa cells generated by CRISPR-Cas9 targeting either exon1 or exon 2 of the human AAGAB gene (see Supplemental Figure S1A). Cell lysates were subjected to SDS–PAGE and immunoblotting with rabbit anti-AAGAB, mouse anti-AP-4 ε, rabbit anti-AP-4 β4, rabbit anti-ATG9A, and mouse anti-AP-1 γ. Ponceau S staining of the PVDF membranes is included as a loading control. The positions of molecular mass markers (in kDa) are shown at the left of the blots. The anti-AP-4 β4 and anti-AP-1 γ immunoblots display additional bands of unknown identity. Notice the decrease in AP-4 ε, AP-4 β4, and AP-1 γ, and the increase in ATG9A in AAGAB-KO cells.Effects of AAGAB on the stability and assembly of AP-4 subunitsNext, we carried out cycloheximide (CHX) chase experiments to determine if AAGAB KO causes degradation of endogenous AP-4 subunits. Indeed, we observed a faster decrease of AP-4 ε and AP-4 β4 levels in AAGAB-KO cells relative to WT cells over 20 h of chase (Figure 5, A–C). Interestingly, these experiments also showed that AAGAB itself is degraded in WT cells with a half-time of approximately 5 h (Figure 5, A and D). Incubation with the proteasomal inhibitor MG132 blocked the decrease in AP-4 ε levels during the CHX chase in AAGAB-KO cells, whereas no changes were detected in WT cells (Figure 5, E and F). MG132 treatment also blocked degradation of AAGAB in WT cells (Figure 5, E and F). From these experiments, we concluded that the lower levels of AP-4 ε in AAGAB-KO cells, as well as the normal turnover of AAGAB in WT cells, are due to proteasomal degradation.FIGURE 5:Decreased stability of endogenous AP-4 subunits in AAGAB-KO cells. (A) WT and AAGAB-KO HeLa cells (clone #19) were incubated for 0–20 h in the presence of the protein synthesis inhibitor cycloheximide (CHX; 0.1 mg/ml). Cells were lysed at the indicated times and the lysates subjected to SDS–PAGE and immunoblotting with mouse anti-AP-4 ε, rabbit anti-AP-4 β4, and rabbit anti-AAGAB. In this panel and in panel E, the positions of molecular mass markers (in kDa) are indicated at the left of the blots. Ponceau S staining of membranes is shown as a loading control. (B–D) Densitometric analysis (using ImageJ) of the AP-4 ε, AP-4 β4, and AAGAB signals from experiments such as that in panel A. Enhanced chemiluminescence (ECL) images that did not exhibit signal saturation were analyzed by densitometry. Some immunoblots were also developed with fluorescent secondary antibodies and yielded results similar to those generated by ECL. Values shown are the percentage of signal relative to that at time 0 (mean ± SD of 2–4 measurements of samples from two independent experiments with similar results). Statistical significance of differences at each time point was analyzed by unpaired one-tailed Student’s t test (*p < 0.05; **p < 0.01). Notice the faster degradation of AP-4 ε and AP-4 β4 in AAGAB-KO vs. WT cells, and the degradation of AAGAB in WT cells. (E) WT and AAGAB-KO cells were incubated for 4.5–6.5 h in the presence or absence of 0.1 mg/ml CHX and 30 μM MG132, as indicated at the tops of blots. Cell lysates were subsequently analyzed by SDS–PAGE and immunoblotting with mouse anti-AP-4 ε and rabbit anti-AAGAB. Two anti-AP-4 ε blots with different exposures times (short and long) are shown. Ponceau S staining of membranes is shown as a loading control. (F) Densitometric analysis (using ImageJ) of the AP-4 ε and AAGAB signals from experiments such as that in panel E. Values are the percentages of signal relative to control (no CHX chase, no MG132 added; mean ± SD of 3–4 measurements of samples from two independent experiments with similar results). Statistical significance of experimental groups compared with their corresponding controls (100% signal) was analyzed by one-way ANOVA followed by Tukey’s test (**p < 0.01). Notice that MG132 prevents the decrease in AP-4 ε in AAGAB-KO cells as well as AAGAB turnover in WT cells during the CHX chase, indicating that both are degraded by the proteasome.We also examined the effect of AAGAB overexpression on the expression levels of recombinant AP-4 subunits. To this end, we transiently transfected WT and AAGAB-KO HeLa cells with plasmids expressing TSF-tagged AP-4 ε and Myc-tagged AP-4 σ4, with or without AAGAB-GFP. The results shown in Figure 6A demonstrate that coexpression with AAGAB-GFP increases the expression levels of recombinant AP-4 ε and σ4 subunits in both WT and AAGAB-KO cells, emphasizing the role of AAGAB in the stabilization of AP-4.FIGURE 6:Expression of recombinant AP-4 subunits and assembly of AP-4 complex in WT and AAGAB-KO cells. (A) WT and AAGAB-KO HeLa cells (clone #19) were transfected at ∼50% confluency with plasmids encoding TSF-AP-4 ε and AP-4 (Myc)3-σ4, with or without AAGAB-GFP. At 19 h after transfection, cells were lysed and extracts subjected to SDS–PAGE and immunoblotting with mouse anti-AP-4 ε, rabbit anti-Myc, or rabbit anti-AAGAB. Notice the increased expression of TSF-ε and (Myc)3-σ4 upon coexpression with AAGAB-GFP in both WT and AAGAB-KO cells. Most of the anti-AP-4 ε signal in lysates of TSF-AP-4 ε-transfected cells corresponds to recombinant AP-4 ε. (B) WT and AAGAB-KO HeLa cells (clone #19) were plated on triplicate 100-mm dishes and grown to near confluency. Cells were subsequently lysed and the cell extracts subjected to immunoprecipitation with mouse anti-GFP (negative control) or rabbit anti-AP-4 β4. The immunoprecipitated complexes were eluted from the antibody-bound beads and subjected to SDS–PAGE and immunoblotting with mouse anti-AP-4 ε or rabbit anti-AP-4 β4. Notice that, even though AAGAB-KO cells exhibited a marked reduction in the levels of AP-4 ε (lysate blot), the remaining AP-4 ε co-immunoprecipitated with anti-AP-4 β4 in both WT and AAGAB-KO cell lysates. Ponceau S staining of membranes is provided as a loading control in both panels. The positions of molecular mass markers (in kDa) are shown at left of blots.Finally, we compared the assembly of endogenous AP-4 subunits by immunoprecipitation of lysates from WT and AAGAB-KO cells with anti-AP-4 β4 followed by immunoblotting of the immunoprecipitated material with anti-AP-4 ε. The results in Figure 6B show that, although AAGAB-KO cells exhibit markedly reduced levels of AP-4 ε (lysates blot), the remaining AP-4 ε can still be coimmunoprecipitated with anti-AP-4 β4 in the absence of AAGAB. These results indicate that the small amount of AP-4 ε that survives under these conditions can assemble into the complex.Abnormal distribution of ATG9A in AAGAB-KO cellsGiven that AAGAB-KO cells exhibit both a marked reduction in AP-4 levels and an increase in ATG9A levels (Figure 4; Supplemental Figure S1B) reminiscent of those observed in AP-4 deficiency (Mattera et al., 2017; Davies et al., 2018; De Pace et al., 2018; Ivankovic et al., 2020; Behne et al., 2020), we asked whether the cellular distribution of ATG9A is also altered in these cells. Immunofluorescence microscopy analysis showed that, consistent with the results of the immunoblotting experiments, inactivation of the AAGAB gene caused a marked reduction in AP-4 ε (Figure 7, A and B; quantitation in panel E) as well as in AP-1 γ and AP-2 α signals (Supplemental Figure S2, A–D; Wan et al., 2021). Importantly, AAGAB-KO cells also displayed increased accumulation of ATG9A at the TGN (Figure 7, A and B; quantification in panel E), similar to that observed in various types of AP-4-deficient cells, including human HeLa, HAP1, and SH-SY5Y cells lines, mouse embryonic and human fibroblasts in primary culture, mouse hippocampal neurons in primary culture, and human iPSC-derived neurons (Mattera et al., 2017; Davies et al., 2018; De Pace et al., 2018; Ivankovic et al., 2020; Behne et al., 2020). This phenotype was reproducibly observed in different AAGAB-KO HeLa clones (Figure 7, A and B; Supplemental Figure S3, A and B), confirming that reduction of AP-4 levels in AAGAB-KO cells causes retention of ATG9A at the TGN. Expression of AAGAB-GFP in different clones of AAGAB-KO cells rescued both the expression of AP-4 ε and the normal distribution of ATG9A between peripheral and TGN compartments (Figure 7, C and D; Supplemental Figure S3, C and D; compare AAGAB-GFP-expressing and -nonexpressing cells in the same fields). Quantification of the rescue is shown in Figure 7E. This rescue demonstrated that the altered staining for AP-4 ε and ATG9A is a bona fide phenotype of AAGAB KO and not due to off-target effects of CRISPR-Cas9 editing.FIGURE 7:Altered immunostaining of AP-4 ε and ATG9A in AAGAB-KO HeLa cells. (A, B) WT and AAGAB-KO HeLa cells (clone #10) were fixed for 10 min with methanol at –20°C and incubated with mouse anti-AP-4 ε, rabbit anti-ATG9A, and sheep anti-TGN46 followed by Alexa Fluor 555-conjugated anti-mouse IgG, Alexa Fluor 488-conjugated anti-rabbit IgG, and Alexa Fluor 647-conjugated anti-sheep IgG. Nuclei were stained with DAPI. Stained cells were imaged by confocal microscopy. Single-channel images are shown in inverted gray scale and merge panels display AP-4 ε, ATG9A, and TGN46 in red, green, and magenta, respectively. Cells are outlined by dashed lines. Notice that in WT cells, AP-4 ε immunofluorescence exhibits a predominantly perinuclear distribution, along with scattered peripheral puncta that mostly represent nonspecific staining (Mattera et al., 2017, 2020b). This typical AP-4 ε immunostaining was observed in 86.1 ± 3.0% of WT cells compared with only 7.4 ± 2.6% of AAGAB-KO cells (see graph in E and corresponding legend for statistical analysis). ATG9A staining in WT cells is distributed between peripheral and juxtanuclear (TGN) pools, and AP-4 deficiency results in a marked concentration of the signal at the TGN (Mattera et al., 2017; Davies et al., 2018; De Pace et al., 2018; Behne et al., 2020). Importantly, the majority of AAGAB-KO cells exhibit ATG9A concentration at the TGN that mimics that in AP-4 KO cells. The typical distribution of ATG9A (peripheral plus perinuclear) was observed in 94.2 ± 4.2% of WT cells compared with only 14.4 ± 7.0% of AAGAB-KO cells (graph in panel E). (C, D) AAGAB-KO cells (clones #3 and 5 in panels C and D, respectively) were transfected with pEGFP-N1-AAGAB. Forty-eight h after transfection, cells were fixed for 10 min with methanol at –20°C and incubated with mouse anti-AP-4 ε (C) or rabbit anti-ATG9A (D) and sheep anti-TGN46 followed by Alexa Fluor 555-conjugated anti-mouse IgG (C) or Alexa Fluor 555-conjugated anti-rabbit IgG (D) Alexa Fluor 647-conjugated anti-sheep IgG, and GFP-Booster Atto488. Nuclei were stained with DAPI. Single-channel images are in inverted gray scale, whereas merge panels display AAGAB-GFP in green, AP-4 ε, or ATG9A in red—C and D, respectively—and TGN46 in magenta. Cells are outlined by dashed lines. We observed that 83.1 ± 6.7% and 90.8 ± 5.7% of AAGAB-GFP-transfected cells exhibited rescue of the AP-4 ε and ATG9A staining phenotypes, respectively (see graph in panel E and corresponding legend for statistical analysis). Scale bars: 20 μm. (E) Values represent the percentages of total cells exhibiting normal staining of AP-4 ε and ATG9A in WT, AAGAB-KO and AAGAB-KO cells transfected with AAGAB-GFP plasmid (mean ± SD of 4–6 measurements; **p < 0.01, one-way ANOVA followed by Dunnett’s test compared with WT). The total numbers of cells analyzed for AP-4 ε distribution were 950, 567, and 500 for WT, AAGAB-KO, and AAGAB-GFP rescue, respectively. The total numbers of cells scored for distribution of ATG9A were 977, 757, and 381 for WT, AAGAB-KO, and AAGAB-GFP rescue, respectively. “Normal distribution” means AP-4 ε at the TGN and ATG9A at both the TGN and the peripheral cytoplasm.It is worth noting that the AAGAB KO did not alter the localization of TGN46 to the TGN (Figure 7, Supplemental Figure S2, and Supplemental Figure S3). This is despite the facts that (1) the cytosolic tail of TGN46 has a tyrosine-based sorting signal that interacts with the AP-1 μ1 and AP-2 μ2 subunits (Ohno et al., 1995), (2) AAGAB KO markedly reduces AP-1 and AP-2 levels and increases the localization of the AP-2 cargo transferrin receptor at the plasma membrane (Figure 4; Supplemental Figure S1B; Supplemental Figure S2; Gulbranson et al., 2019; Wan et al., 2021), and (3) TGN46 cycles between the TGN, plasma membrane, and endosomes, albeit with predominant localization to the TGN at steady state (Reaves et al., 1993). The independence of TGN46 localization on AAGAB may be due to the presence of a TGN-retention determinant in the transmembrane domain (Ponnambalam et al., 1994) such that the population that exits the TGN is small, and to the sufficiency of residual levels of AP-1 and AP-2 in AAGAB-KO cells for retrieval of this small population to the TGN.

AAGAB directly binds to AP-4 ε and σ4 subunitsIn previous work (Mattera et al., 2015; Mattera et al., 2017), we performed a TAP-MS identification of AP-4 interactors using two Strep/one FLAG (TSF)-tagged AP-4 ε (TSF-AP-4 ε) stably expressed in H4 human neuroglioma cells (see Supplemental Table S1). We subsequently confirmed and characterized the interactions of AP-4 with top hits in that analysis, including tepsin (Mattera et al., 2015), ATG9A (Mattera et al., 2017; De Pace et al., 2018), and subunits of the FHF complex (Mattera et al, 2020b). Going down the list of hits, we noticed the presence of AAGAB, albeit with a low peptide count (Supplemental Table S1). The previously demonstrated role of AAGAB as a chaperone in the assembly of AP-1 and AP-2 (Gulbranson et al., 2019; Wan et al., 2021) prompted us to investigate the functional significance of the AP-4–AAGAB interaction.We confirmed the AP-4–AAGAB interaction in pull-down experiments using lysates of HEK293T cells transiently transfected with plasmids encoding either TSF-AP-4 ε or TSF-myrlysin (a subunit of the BORC complex, also termed BORCS5 or LOH12CR1; Pu et al., 2015), as a negative control. We observed that TSF-AP-4 ε, but not TSF-myrlysin, pulled down endogenous AAGAB (Figure 1B). This result demonstrated the specificity of the interaction between AAGAB and AP-4 ε.Next, we asked if this interaction represented direct binding of AAGAB to AP-4 subunits. To address this question, we carried out Y2H analyses using constructs encoding (i) the GAL4 transcriptional activation domain (AD) fused to each of the four AP-4 subunits and (ii) the GAL4 DNA-binding domain (BD) fused to AAGAB (Figure 1C). Given the previously reported interactions of AAGAB with AP-1 γ and AP-2 α (Page et al., 1999; Gulbranson et al., 2019; Wan et al., 2021), we included AD fusions to the AP-1 γ, AP-2 α and AP-3 δ subunits as controls. We also used as an additional control a BD fusion to tepsin, which binds to the ε, β4, and μ4 subunits of AP-4 (Borner et al., 2012; Mattera et al., 2015; Frazier et al., 2016; Mattera et al., 2020b). The results of these experiments showed that AAGAB binds to the ε and σ4 subunits, but not the β4 and μ4 subunits, of AP-4 (Figure 1C). The control samples confirmed interactions of AAGAB with AP-1 γ and AP-2 α, but not AP-3 δ, and of tepsin with AP-4 ε, β4, and μ4 (Figure 1C). Thus, AAGAB binds directly to a subset of AP-4 subunits, ε and σ4, which constitute an AP-4 ε-σ4 hemicomplex (Mattera et al., 2011).

AAGAB directly binds to AP-4 ε and σ4 subunits

In previous work (Mattera et al., 2015; Mattera et al., 2017), we performed a TAP-MS identification of AP-4 interactors using two Strep/one FLAG (TSF)-tagged AP-4 ε (TSF-AP-4 ε) stably expressed in H4 human neuroglioma cells (see Supplemental Table S1). We subsequently confirmed and characterized the interactions of AP-4 with top hits in that analysis, including tepsin (Mattera et al., 2015), ATG9A (Mattera et al., 2017; De Pace et al., 2018), and subunits of the FHF complex (Mattera et al, 2020b). Going down the list of hits, we noticed the presence of AAGAB, albeit with a low peptide count (Supplemental Table S1). The previously demonstrated role of AAGAB as a chaperone in the assembly of AP-1 and AP-2 (Gulbranson et al., 2019; Wan et al., 2021) prompted us to investigate the functional significance of the AP-4–AAGAB interaction.

We confirmed the AP-4–AAGAB interaction in pull-down experiments using lysates of HEK293T cells transiently transfected with plasmids encoding either TSF-AP-4 ε or TSF-myrlysin (a subunit of the BORC complex, also termed BORCS5 or LOH12CR1; Pu et al., 2015), as a negative control. We observed that TSF-AP-4 ε, but not TSF-myrlysin, pulled down endogenous AAGAB (Figure 1B). This result demonstrated the specificity of the interaction between AAGAB and AP-4 ε.

Next, we asked if this interaction represented direct binding of AAGAB to AP-4 subunits. To address this question, we carried out Y2H analyses using constructs encoding (i) the GAL4 transcriptional activation domain (AD) fused to each of the four AP-4 subunits and (ii) the GAL4 DNA-binding domain (BD) fused to AAGAB (Figure 1C). Given the previously reported interactions of AAGAB with AP-1 γ and AP-2 α (Page et al., 1999; Gulbranson et al., 2019; Wan et al., 2021), we included AD fusions to the AP-1 γ, AP-2 α and AP-3 δ subunits as controls. We also used as an additional control a BD fusion to tepsin, which binds to the ε, β4, and μ4 subunits of AP-4 (Borner et al., 2012; Mattera et al., 2015; Frazier et al., 2016; Mattera et al., 2020b). The results of these experiments showed that AAGAB binds to the ε and σ4 subunits, but not the β4 and μ4 subunits, of AP-4 (Figure 1C). The control samples confirmed interactions of AAGAB with AP-1 γ and AP-2 α, but not AP-3 δ, and of tepsin with AP-4 ε, β4, and μ4 (Figure 1C). Thus, AAGAB binds directly to a subset of AP-4 subunits, ε and σ4, which constitute an AP-4 ε-σ4 hemicomplex (Mattera et al., 2011).

Structural determinants of the AP-4–AAGAB interactionTo analyze the structural determinants of the interaction of AP-4 ε with AAGAB, we performed additional Y2H analyses with AD fusions to AP-4 ε deletion mutants (Figure 2, A and B). We found that the minimal fragment of AP-4 ε that binds to AAGAB spans amino acid residues 1–138, corresponding to the N-terminal part of the “trunk” domain (Figure 1A; Figure 2, A–C). This mode of interaction contrasted with that of tepsin, which binds to the C-terminal “ear” domain of AP-4 ε (residues 883–1137; Figure 1A; Figure 2, A and B; Mattera et al., 2015).FIGURE 2:Determinants of AP-4 ε and σ4 interactions with AAGAB. (Α) Consensus secondary structure prediction of human AP-4 ε according to the NPS@ server (https://npsa-prabi.ibcp.fr/cgi-bin/npsa_automat.pl?page = /NPSA/npsa_seccons.html; Combet et al., 2000) indicating its trunk, hinge, and ear domains. Blue, red, and magenta coloring represents α-helix, β-sheet, and random coil structure, respectively. The deletion mutants used in the Y2H analysis shown in panel B are also depicted. (B) Y2H analysis of the interaction of AAGAB with different AP-4 ε deletion constructs. Experiments were performed as explained for Figure 1C. The results demonstrate that AAGAB binds to the N-terminal portion (residues 1–138) of the AP-4 ε trunk. In contrast, the tepsin control interacts with the AP-4 ε 727–1137 (part of hinge plus ear domains) and 883–1137 (ear domain) constructs, as previously reported (Mattera et al., 2015). (C) AlphaFold (https://alphafold.ebi.ac.uk; Jumper et al., 2021) prediction of the tertiary structure of the human AP-4 ε trunk domain (human AP-4 ε residues 1–599). The AP-4 ε 1–138 region containing the AAGAB binding site is depicted in gray. Also shown are the N-and C-termini of the trunk domain. The image was rendered using PyMol (https://pymol.org/). (D) AlphaFold prediction of the tertiary structure of mouse AP-4 σ4 showing the position of the conserved V88 and I103 residues that were mutated to D and S, respectively. Also shown are the N-and C-termini. The image was rendered using PyMol. (E) Y2H analysis of the interaction of AAGAB with AP-4 σ4 mutants performed as explained for Figure 1C. Notice that the V88D and I03S substitutions abrogate binding of AP-4 σ4 to both AAGAB and AP-4 ε. (F) Y2H analysis of the interaction of AAGAB with σ subunits (including isoforms) of AP-1, AP-2, AP-3, and AP-4. Notice the interaction of AAGAB with all σ subunits tested except AP-3 σ3A and AP-3 σ3B (Top panel). The AD-fusions of AP-3 σ3A and AP-3 σ3B are able to bind to the BD-AP-3 δ control (Bottom panel), indicating that their lack of interaction with AAGAB is not due to problems with their expression, stability or targeting to the yeast nucleus.We also used the Y2H system to examine structural determinants of the interaction of AP-4 σ4 with AAGAB (Figure 2, D and E). A previous study (Gulbranson et al., 2019) showed that assembly of AP-2 σ2 with an AAGAB-AP-2 α-complex requires AP-2 σ2 residues involved in the recognition of acidic-dileucine signals fitting the consensus motif (D/E)XXXL(L/I), where X signifies any amino acid. In general, these signals bind to the AP-1 γ-σ1, AP-2 α-σ2, and AP-3 δ-σ3 hemicomplexes (Janvier et al., 2003; Chaudhuri et al., 2007; Doray et al., 2007) mainly through conserved residues on σ1, σ2, and σ3 (Kelly et al., 2008; Mattera et al., 2011; Jia et al., 2014, Ren et al., 2014). An additional contribution is made by an R residue near the N-terminus of the γ, α, and δ subunits (Kelly et al., 2008; Mattera et al, 2011; Jia et al., 2014; Ren et al., 2014). AP-4 ε-σ4 has the conserved residues on σ4, including V88 and I103 (Figure 2D), but lacks the corresponding R residue in AP-4 ε, likely explaining why the AP-4 ε-σ4 hemicomplex has not been found to bind acidic-dileucine signals (Mattera et al., 2011). This difference notwithstanding, we tested whether the conserved residues in AP-4 σ4 are required for interaction with AAGAB. In line with previous findings on AP-2 σ2 (Gulbranson et al., 2019), V88D, I103S, or double V88D-I103S mutations in AP-4 σ4 abrogated the interaction with AAGAB (Figure 2, D and E). However, these mutations also abolished the interaction of AP-4 σ4 with AP-4 ε (Figure 2E), suggesting that they render the mutant proteins unstable, either intrinsically or because they cannot interact with the yeast AAGAB ortholog Irc6p. Furthermore, we cannot exclude that the corresponding mutant fusion proteins are not targeted to the yeast nucleus where the two-hybrid interactions activate GAL4-driven transcription. These uncertainties prevented us from concluding whether there is a conserved mode of interaction of AAGAB with AP-2 σ2 and AP-4 σ4.The conservation of the putative acidic dileucine–binding residues on the σ1-4 subunits prompted us to analyze whether all these subunits bind AAGAB. We found that the σ1A, σ1B, and σ1C isoforms of AP-1 σ1, as well as the σ2 subunit of AP-2, bind to AAGAB, whereas the σ3A and σ3B subunit isoforms of AP-3 do not (Figure 2F). In this case, the activity of the σ3A and σ3B constructs was confirmed by their interaction with AP-3 δ (Figure 2F). These findings indicated that AAGAB can bind not only the large AP-1 γ, AP-2 α, and AP-4 ε subunits, but also the cognate σ1, σ2, and σ4 subunits. In contrast, AAGAB does not bind either the δ or σ3 subunits of AP-3.Finally, we used the Y2H system to analyze the AAGAB domains involved in binding to AP-4 ε and σ4 (Figure 3, A–C). AAGAB comprises an N-terminal LMW GTPase-like fold (residues 1–155) and a C-terminal adaptin-binding domain (residues 156–315; Figure 3, A and B; Gorynia et al., 2012; Pöhler et al., 2012; Zhou et al., 2019). The Y2H assays showed that the AAGAB 156–315 fragment was sufficient for binding to AP-4 ε, as well as AP-1 γ and AP-2 α (Figure 3C). In contrast, neither the 1–155 fragment nor the 156–315 fragment was sufficient for binding to AP-4 σ4 (Figure 3C), indicating that this subunit requires both AAGAB domains for interaction.FIGURE 3:Identification of AAGAB domains that bind to AP-4 ε and σ4. (A) Consensus secondary structure prediction of human AAGAB according to the NPS@ server. The N-terminal low-molecular-weight (LMW) GTPase fold domain and C-terminal adaptin-binding domain are shown, along with the two deletion constructs used in Y2H analysis. Predictions are depicted as indicated in the legend to Figure 2A. (B) AlphaFold prediction of the human AAGAB tertiary structure. Depicted in blue, cyan, yellow, and orange are regions with very high (>90), confident (>70 and <90), low (>50 and <70), and very low (<50) residue confidence scores (pLDDT), respectively. N- and C-termini are indicated. (C) Y2H analysis showing that AP-4 ε binds to the AAGAB adaptin-binding domain (156–315 fragment), whereas AP-4 σ4 requires residues on both the N-terminal LMW GTPase fold and the C-terminal adaptin-binding domains. Y2H assays were performed as described for Figure 1C.

Structural determinants of the AP-4–AAGAB interaction

To analyze the structural determinants of the interaction of AP-4 ε with AAGAB, we performed additional Y2H analyses with AD fusions to AP-4 ε deletion mutants (Figure 2, A and B). We found that the minimal fragment of AP-4 ε that binds to AAGAB spans amino acid residues 1–138, corresponding to the N-terminal part of the “trunk” domain (Figure 1A; Figure 2, A–C). This mode of interaction contrasted with that of tepsin, which binds to the C-terminal “ear” domain of AP-4 ε (residues 883–1137; Figure 1A; Figure 2, A and B; Mattera et al., 2015).

Determinants of AP-4 ε and σ4 interactions with AAGAB. (Α) Consensus secondary structure prediction of human AP-4 ε according to the NPS@ server (https://npsa-prabi.ibcp.fr/cgi-bin/npsa_automat.pl?page = /NPSA/npsa_seccons.html; Combet et al., 2000) indicating its trunk, hinge, and ear domains. Blue, red, and magenta coloring represents α-helix, β-sheet, and random coil structure, respectively. The deletion mutants used in the Y2H analysis shown in panel B are also depicted. (B) Y2H analysis of the interaction of AAGAB with different AP-4 ε deletion constructs. Experiments were performed as explained for Figure 1C. The results demonstrate that AAGAB binds to the N-terminal portion (residues 1–138) of the AP-4 ε trunk. In contrast, the tepsin control interacts with the AP-4 ε 727–1137 (part of hinge plus ear domains) and 883–1137 (ear domain) constructs, as previously reported (Mattera et al., 2015). (C) AlphaFold (https://alphafold.ebi.ac.uk; Jumper et al., 2021) prediction of the tertiary structure of the human AP-4 ε trunk domain (human AP-4 ε residues 1–599). The AP-4 ε 1–138 region containing the AAGAB binding site is depicted in gray. Also shown are the N-and C-termini of the trunk domain. The image was rendered using PyMol (https://pymol.org/). (D) AlphaFold prediction of the tertiary structure of mouse AP-4 σ4 showing the position of the conserved V88 and I103 residues that were mutated to D and S, respectively. Also shown are the N-and C-termini. The image was rendered using PyMol. (E) Y2H analysis of the interaction of AAGAB with AP-4 σ4 mutants performed as explained for Figure 1C. Notice that the V88D and I03S substitutions abrogate binding of AP-4 σ4 to both AAGAB and AP-4 ε. (F) Y2H analysis of the interaction of AAGAB with σ subunits (including isoforms) of AP-1, AP-2, AP-3, and AP-4. Notice the interaction of AAGAB with all σ subunits tested except AP-3 σ3A and AP-3 σ3B (Top panel). The AD-fusions of AP-3 σ3A and AP-3 σ3B are able to bind to the BD-AP-3 δ control (Bottom panel), indicating that their lack of interaction with AAGAB is not due to problems with their expression, stability or targeting to the yeast nucleus.

We also used the Y2H system to examine structural determinants of the interaction of AP-4 σ4 with AAGAB (Figure 2, D and E). A previous study (Gulbranson et al., 2019) showed that assembly of AP-2 σ2 with an AAGAB-AP-2 α-complex requires AP-2 σ2 residues involved in the recognition of acidic-dileucine signals fitting the consensus motif (D/E)XXXL(L/I), where X signifies any amino acid. In general, these signals bind to the AP-1 γ-σ1, AP-2 α-σ2, and AP-3 δ-σ3 hemicomplexes (Janvier et al., 2003; Chaudhuri et al., 2007; Doray et al., 2007) mainly through conserved residues on σ1, σ2, and σ3 (Kelly et al., 2008; Mattera et al., 2011; Jia et al., 2014, Ren et al., 2014). An additional contribution is made by an R residue near the N-terminus of the γ, α, and δ subunits (Kelly et al., 2008; Mattera et al, 2011; Jia et al., 2014; Ren et al., 2014). AP-4 ε-σ4 has the conserved residues on σ4, including V88 and I103 (Figure 2D), but lacks the corresponding R residue in AP-4 ε, likely explaining why the AP-4 ε-σ4 hemicomplex has not been found to bind acidic-dileucine signals (Mattera et al., 2011). This difference notwithstanding, we tested whether the conserved residues in AP-4 σ4 are required for interaction with AAGAB. In line with previous findings on AP-2 σ2 (Gulbranson et al., 2019), V88D, I103S, or double V88D-I103S mutations in AP-4 σ4 abrogated the interaction with AAGAB (Figure 2, D and E). However, these mutations also abolished the interaction of AP-4 σ4 with AP-4 ε (Figure 2E), suggesting that they render the mutant proteins unstable, either intrinsically or because they cannot interact with the yeast AAGAB ortholog Irc6p. Furthermore, we cannot exclude that the corresponding mutant fusion proteins are not targeted to the yeast nucleus where the two-hybrid interactions activate GAL4-driven transcription. These uncertainties prevented us from concluding whether there is a conserved mode of interaction of AAGAB with AP-2 σ2 and AP-4 σ4.

The conservation of the putative acidic dileucine–binding residues on the σ1-4 subunits prompted us to analyze whether all these subunits bind AAGAB. We found that the σ1A, σ1B, and σ1C isoforms of AP-1 σ1, as well as the σ2 subunit of AP-2, bind to AAGAB, whereas the σ3A and σ3B subunit isoforms of AP-3 do not (Figure 2F). In this case, the activity of the σ3A and σ3B constructs was confirmed by their interaction with AP-3 δ (Figure 2F). These findings indicated that AAGAB can bind not only the large AP-1 γ, AP-2 α, and AP-4 ε subunits, but also the cognate σ1, σ2, and σ4 subunits. In contrast, AAGAB does not bind either the δ or σ3 subunits of AP-3.

Finally, we used the Y2H system to analyze the AAGAB domains involved in binding to AP-4 ε and σ4 (Figure 3, A–C). AAGAB comprises an N-terminal LMW GTPase-like fold (residues 1–155) and a C-terminal adaptin-binding domain (residues 156–315; Figure 3, A and B; Gorynia et al., 2012; Pöhler et al., 2012; Zhou et al., 2019). The Y2H assays showed that the AAGAB 156–315 fragment was sufficient for binding to AP-4 ε, as well as AP-1 γ and AP-2 α (Figure 3C). In contrast, neither the 1–155 fragment nor the 156–315 fragment was sufficient for binding to AP-4 σ4 (Figure 3C), indicating that this subunit requires both AAGAB domains for interaction.

Identification of AAGAB domains that bind to AP-4 ε and σ4. (A) Consensus secondary structure prediction of human AAGAB according to the NPS@ server. The N-terminal low-molecular-weight (LMW) GTPase fold domain and C-terminal adaptin-binding domain are shown, along with the two deletion constructs used in Y2H analysis. Predictions are depicted as indicated in the legend to Figure 2A. (B) AlphaFold prediction of the human AAGAB tertiary structure. Depicted in blue, cyan, yellow, and orange are regions with very high (>90), confident (>70 and <90), low (>50 and <70), and very low (<50) residue confidence scores (pLDDT), respectively. N- and C-termini are indicated. (C) Y2H analysis showing that AP-4 ε binds to the AAGAB adaptin-binding domain (156–315 fragment), whereas AP-4 σ4 requires residues on both the N-terminal LMW GTPase fold and the C-terminal adaptin-binding domains. Y2H assays were performed as described for Figure 1C.

AAGAB KO decreases the levels of AP-4 subunitsHaving established that AAGAB directly binds to AP-4 subunits, we analyzed the consequences of ablating the AAGAB gene on AP-4 levels and function. To this end, we generated AAGAB-KO HeLa cell lines using the CRISPR-Cas9 system. We alternatively targeted exons 1 and 2 of the AAGAB gene (Supplemental Figure S1A) and obtained several clones of AAGAB-KO cells that were subsequently analyzed by immunoblotting (Figure 4 and Supplemental Figure S1B). We found that targeting either exon completely abrogated AAGAB protein expression in all tested clones (Figure 4 and Supplemental Figure S1B). Importantly, all AAGAB-KO clones exhibited decreased levels of AP-4 ε and AP-4 β4 (Figure 4, Supplemental Figure S1B). We could not assess the levels of AP-4 μ4 and σ4 because of the lack of suitable antibodies, but we expect them to be reduced, given the need of all subunits for stability of AP complexes (Dell’Angelica et al., 1999b; Davies et al., 2018; De Pace et al., 2018). Despite some variability in the different clones, we also observed an increase in the levels of the AP-4 cargo ATG9A in AAGAB-KO cells (Figure 4, Supplemental Figure S1B), similar to that previously observed in AP-4-deficient cells (Mattera et al., 2017; Davies et al., 2018; De Pace et al., 2018; Behne et al., 2020). In agreement with previous findings (Gulbranson et al., 2019; Wan et al., 2021), the AAGAB-KO HeLa cells also displayed reduced levels of AP-1 γ (Figure 4; Supplemental Figure S1B). These findings demonstrated that AAGAB is required for maintenance of normal levels of AP-4 ε and β4.FIGURE 4:Reduced levels of AP-4 subunits and increased levels of ATG9A in AAGAB-KO cells. Immunoblot analysis of WT and several clones of AAGAB-KO HeLa cells generated by CRISPR-Cas9 targeting either exon1 or exon 2 of the human AAGAB gene (see Supplemental Figure S1A). Cell lysates were subjected to SDS–PAGE and immunoblotting with rabbit anti-AAGAB, mouse anti-AP-4 ε, rabbit anti-AP-4 β4, rabbit anti-ATG9A, and mouse anti-AP-1 γ. Ponceau S staining of the PVDF membranes is included as a loading control. The positions of molecular mass markers (in kDa) are shown at the left of the blots. The anti-AP-4 β4 and anti-AP-1 γ immunoblots display additional bands of unknown identity. Notice the decrease in AP-4 ε, AP-4 β4, and AP-1 γ, and the increase in ATG9A in AAGAB-KO cells.

AAGAB KO decreases the levels of AP-4 subunits

Having established that AAGAB directly binds to AP-4 subunits, we analyzed the consequences of ablating the AAGAB gene on AP-4 levels and function. To this end, we generated AAGAB-KO HeLa cell lines using the CRISPR-Cas9 system. We alternatively targeted exons 1 and 2 of the AAGAB gene (Supplemental Figure S1A) and obtained several clones of AAGAB-KO cells that were subsequently analyzed by immunoblotting (Figure 4 and Supplemental Figure S1B). We found that targeting either exon completely abrogated AAGAB protein expression in all tested clones (Figure 4 and Supplemental Figure S1B). Importantly, all AAGAB-KO clones exhibited decreased levels of AP-4 ε and AP-4 β4 (Figure 4, Supplemental Figure S1B). We could not assess the levels of AP-4 μ4 and σ4 because of the lack of suitable antibodies, but we expect them to be reduced, given the need of all subunits for stability of AP complexes (Dell’Angelica et al., 1999b; Davies et al., 2018; De Pace et al., 2018). Despite some variability in the different clones, we also observed an increase in the levels of the AP-4 cargo ATG9A in AAGAB-KO cells (Figure 4, Supplemental Figure S1B), similar to that previously observed in AP-4-deficient cells (Mattera et al., 2017; Davies et al., 2018; De Pace et al., 2018; Behne et al., 2020). In agreement with previous findings (Gulbranson et al., 2019; Wan et al., 2021), the AAGAB-KO HeLa cells also displayed reduced levels of AP-1 γ (Figure 4; Supplemental Figure S1B). These findings demonstrated that AAGAB is required for maintenance of normal levels of AP-4 ε and β4.

Reduced levels of AP-4 subunits and increased levels of ATG9A in AAGAB-KO cells. Immunoblot analysis of WT and several clones of AAGAB-KO HeLa cells generated by CRISPR-Cas9 targeting either exon1 or exon 2 of the human AAGAB gene (see Supplemental Figure S1A). Cell lysates were subjected to SDS–PAGE and immunoblotting with rabbit anti-AAGAB, mouse anti-AP-4 ε, rabbit anti-AP-4 β4, rabbit anti-ATG9A, and mouse anti-AP-1 γ. Ponceau S staining of the PVDF membranes is included as a loading control. The positions of molecular mass markers (in kDa) are shown at the left of the blots. The anti-AP-4 β4 and anti-AP-1 γ immunoblots display additional bands of unknown identity. Notice the decrease in AP-4 ε, AP-4 β4, and AP-1 γ, and the increase in ATG9A in AAGAB-KO cells.

Effects of AAGAB on the stability and assembly of AP-4 subunitsNext, we carried out cycloheximide (CHX) chase experiments to determine if AAGAB KO causes degradation of endogenous AP-4 subunits. Indeed, we observed a faster decrease of AP-4 ε and AP-4 β4 levels in AAGAB-KO cells relative to WT cells over 20 h of chase (Figure 5, A–C). Interestingly, these experiments also showed that AAGAB itself is degraded in WT cells with a half-time of approximately 5 h (Figure 5, A and D). Incubation with the proteasomal inhibitor MG132 blocked the decrease in AP-4 ε levels during the CHX chase in AAGAB-KO cells, whereas no changes were detected in WT cells (Figure 5, E and F). MG132 treatment also blocked degradation of AAGAB in WT cells (Figure 5, E and F). From these experiments, we concluded that the lower levels of AP-4 ε in AAGAB-KO cells, as well as the normal turnover of AAGAB in WT cells, are due to proteasomal degradation.FIGURE 5:Decreased stability of endogenous AP-4 subunits in AAGAB-KO cells. (A) WT and AAGAB-KO HeLa cells (clone #19) were incubated for 0–20 h in the presence of the protein synthesis inhibitor cycloheximide (CHX; 0.1 mg/ml). Cells were lysed at the indicated times and the lysates subjected to SDS–PAGE and immunoblotting with mouse anti-AP-4 ε, rabbit anti-AP-4 β4, and rabbit anti-AAGAB. In this panel and in panel E, the positions of molecular mass markers (in kDa) are indicated at the left of the blots. Ponceau S staining of membranes is shown as a loading control. (B–D) Densitometric analysis (using ImageJ) of the AP-4 ε, AP-4 β4, and AAGAB signals from experiments such as that in panel A. Enhanced chemiluminescence (ECL) images that did not exhibit signal saturation were analyzed by densitometry. Some immunoblots were also developed with fluorescent secondary antibodies and yielded results similar to those generated by ECL. Values shown are the percentage of signal relative to that at time 0 (mean ± SD of 2–4 measurements of samples from two independent experiments with similar results). Statistical significance of differences at each time point was analyzed by unpaired one-tailed Student’s t test (*p < 0.05; **p < 0.01). Notice the faster degradation of AP-4 ε and AP-4 β4 in AAGAB-KO vs. WT cells, and the degradation of AAGAB in WT cells. (E) WT and AAGAB-KO cells were incubated for 4.5–6.5 h in the presence or absence of 0.1 mg/ml CHX and 30 μM MG132, as indicated at the tops of blots. Cell lysates were subsequently analyzed by SDS–PAGE and immunoblotting with mouse anti-AP-4 ε and rabbit anti-AAGAB. Two anti-AP-4 ε blots with different exposures times (short and long) are shown. Ponceau S staining of membranes is shown as a loading control. (F) Densitometric analysis (using ImageJ) of the AP-4 ε and AAGAB signals from experiments such as that in panel E. Values are the percentages of signal relative to control (no CHX chase, no MG132 added; mean ± SD of 3–4 measurements of samples from two independent experiments with similar results). Statistical significance of experimental groups compared with their corresponding controls (100% signal) was analyzed by one-way ANOVA followed by Tukey’s test (**p < 0.01). Notice that MG132 prevents the decrease in AP-4 ε in AAGAB-KO cells as well as AAGAB turnover in WT cells during the CHX chase, indicating that both are degraded by the proteasome.We also examined the effect of AAGAB overexpression on the expression levels of recombinant AP-4 subunits. To this end, we transiently transfected WT and AAGAB-KO HeLa cells with plasmids expressing TSF-tagged AP-4 ε and Myc-tagged AP-4 σ4, with or without AAGAB-GFP. The results shown in Figure 6A demonstrate that coexpression with AAGAB-GFP increases the expression levels of recombinant AP-4 ε and σ4 subunits in both WT and AAGAB-KO cells, emphasizing the role of AAGAB in the stabilization of AP-4.FIGURE 6:Expression of recombinant AP-4 subunits and assembly of AP-4 complex in WT and AAGAB-KO cells. (A) WT and AAGAB-KO HeLa cells (clone #19) were transfected at ∼50% confluency with plasmids encoding TSF-AP-4 ε and AP-4 (Myc)3-σ4, with or without AAGAB-GFP. At 19 h after transfection, cells were lysed and extracts subjected to SDS–PAGE and immunoblotting with mouse anti-AP-4 ε, rabbit anti-Myc, or rabbit anti-AAGAB. Notice the increased expression of TSF-ε and (Myc)3-σ4 upon coexpression with AAGAB-GFP in both WT and AAGAB-KO cells. Most of the anti-AP-4 ε signal in lysates of TSF-AP-4 ε-transfected cells corresponds to recombinant AP-4 ε. (B) WT and AAGAB-KO HeLa cells (clone #19) were plated on triplicate 100-mm dishes and grown to near confluency. Cells were subsequently lysed and the cell extracts subjected to immunoprecipitation with mouse anti-GFP (negative control) or rabbit anti-AP-4 β4. The immunoprecipitated complexes were eluted from the antibody-bound beads and subjected to SDS–PAGE and immunoblotting with mouse anti-AP-4 ε or rabbit anti-AP-4 β4. Notice that, even though AAGAB-KO cells exhibited a marked reduction in the levels of AP-4 ε (lysate blot), the remaining AP-4 ε co-immunoprecipitated with anti-AP-4 β4 in both WT and AAGAB-KO cell lysates. Ponceau S staining of membranes is provided as a loading control in both panels. The positions of molecular mass markers (in kDa) are shown at left of blots.Finally, we compared the assembly of endogenous AP-4 subunits by immunoprecipitation of lysates from WT and AAGAB-KO cells with anti-AP-4 β4 followed by immunoblotting of the immunoprecipitated material with anti-AP-4 ε. The results in Figure 6B show that, although AAGAB-KO cells exhibit markedly reduced levels of AP-4 ε (lysates blot), the remaining AP-4 ε can still be coimmunoprecipitated with anti-AP-4 β4 in the absence of AAGAB. These results indicate that the small amount of AP-4 ε that survives under these conditions can assemble into the complex.

Effects of AAGAB on the stability and assembly of AP-4 subunits

Next, we carried out cycloheximide (CHX) chase experiments to determine if AAGAB KO causes degradation of endogenous AP-4 subunits. Indeed, we observed a faster decrease of AP-4 ε and AP-4 β4 levels in AAGAB-KO cells relative to WT cells over 20 h of chase (Figure 5, A–C). Interestingly, these experiments also showed that AAGAB itself is degraded in WT cells with a half-time of approximately 5 h (Figure 5, A and D). Incubation with the proteasomal inhibitor MG132 blocked the decrease in AP-4 ε levels during the CHX chase in AAGAB-KO cells, whereas no changes were detected in WT cells (Figure 5, E and F). MG132 treatment also blocked degradation of AAGAB in WT cells (Figure 5, E and F). From these experiments, we concluded that the lower levels of AP-4 ε in AAGAB-KO cells, as well as the normal turnover of AAGAB in WT cells, are due to proteasomal degradation.

Decreased stability of endogenous AP-4 subunits in AAGAB-KO cells. (A) WT and AAGAB-KO HeLa cells (clone #19) were incubated for 0–20 h in the presence of the protein synthesis inhibitor cycloheximide (CHX; 0.1 mg/ml). Cells were lysed at the indicated times and the lysates subjected to SDS–PAGE and immunoblotting with mouse anti-AP-4 ε, rabbit anti-AP-4 β4, and rabbit anti-AAGAB. In this panel and in panel E, the positions of molecular mass markers (in kDa) are indicated at the left of the blots. Ponceau S staining of membranes is shown as a loading control. (B–D) Densitometric analysis (using ImageJ) of the AP-4 ε, AP-4 β4, and AAGAB signals from experiments such as that in panel A. Enhanced chemiluminescence (ECL) images that did not exhibit signal saturation were analyzed by densitometry. Some immunoblots were also developed with fluorescent secondary antibodies and yielded results similar to those generated by ECL. Values shown are the percentage of signal relative to that at time 0 (mean ± SD of 2–4 measurements of samples from two independent experiments with similar results). Statistical significance of differences at each time point was analyzed by unpaired one-tailed Student’s t test (*p < 0.05; **p < 0.01). Notice the faster degradation of AP-4 ε and AP-4 β4 in AAGAB-KO vs. WT cells, and the degradation of AAGAB in WT cells. (E) WT and AAGAB-KO cells were incubated for 4.5–6.5 h in the presence or absence of 0.1 mg/ml CHX and 30 μM MG132, as indicated at the tops of blots. Cell lysates were subsequently analyzed by SDS–PAGE and immunoblotting with mouse anti-AP-4 ε and rabbit anti-AAGAB. Two anti-AP-4 ε blots with different exposures times (short and long) are shown. Ponceau S staining of membranes is shown as a loading control. (F) Densitometric analysis (using ImageJ) of the AP-4 ε and AAGAB signals from experiments such as that in panel E. Values are the percentages of signal relative to control (no CHX chase, no MG132 added; mean ± SD of 3–4 measurements of samples from two independent experiments with similar results). Statistical significance of experimental groups compared with their corresponding controls (100% signal) was analyzed by one-way ANOVA followed by Tukey’s test (**p < 0.01). Notice that MG132 prevents the decrease in AP-4 ε in AAGAB-KO cells as well as AAGAB turnover in WT cells during the CHX chase, indicating that both are degraded by the proteasome.

We also examined the effect of AAGAB overexpression on the expression levels of recombinant AP-4 subunits. To this end, we transiently transfected WT and AAGAB-KO HeLa cells with plasmids expressing TSF-tagged AP-4 ε and Myc-tagged AP-4 σ4, with or without AAGAB-GFP. The results shown in Figure 6A demonstrate that coexpression with AAGAB-GFP increases the expression levels of recombinant AP-4 ε and σ4 subunits in both WT and AAGAB-KO cells, emphasizing the role of AAGAB in the stabilization of AP-4.

Expression of recombinant AP-4 subunits and assembly of AP-4 complex in WT and AAGAB-KO cells. (A) WT and AAGAB-KO HeLa cells (clone #19) were transfected at ∼50% confluency with plasmids encoding TSF-AP-4 ε and AP-4 (Myc)3-σ4, with or without AAGAB-GFP. At 19 h after transfection, cells were lysed and extracts subjected to SDS–PAGE and immunoblotting with mouse anti-AP-4 ε, rabbit anti-Myc, or rabbit anti-AAGAB. Notice the increased expression of TSF-ε and (Myc)3-σ4 upon coexpression with AAGAB-GFP in both WT and AAGAB-KO cells. Most of the anti-AP-4 ε signal in lysates of TSF-AP-4 ε-transfected cells corresponds to recombinant AP-4 ε. (B) WT and AAGAB-KO HeLa cells (clone #19) were plated on triplicate 100-mm dishes and grown to near confluency. Cells were subsequently lysed and the cell extracts subjected to immunoprecipitation with mouse anti-GFP (negative control) or rabbit anti-AP-4 β4. The immunoprecipitated complexes were eluted from the antibody-bound beads and subjected to SDS–PAGE and immunoblotting with mouse anti-AP-4 ε or rabbit anti-AP-4 β4. Notice that, even though AAGAB-KO cells exhibited a marked reduction in the levels of AP-4 ε (lysate blot), the remaining AP-4 ε co-immunoprecipitated with anti-AP-4 β4 in both WT and AAGAB-KO cell lysates. Ponceau S staining of membranes is provided as a loading control in both panels. The positions of molecular mass markers (in kDa) are shown at left of blots.

Finally, we compared the assembly of endogenous AP-4 subunits by immunoprecipitation of lysates from WT and AAGAB-KO cells with anti-AP-4 β4 followed by immunoblotting of the immunoprecipitated material with anti-AP-4 ε. The results in Figure 6B show that, although AAGAB-KO cells exhibit markedly reduced levels of AP-4 ε (lysates blot), the remaining AP-4 ε can still be coimmunoprecipitated with anti-AP-4 β4 in the absence of AAGAB. These results indicate that the small amount of AP-4 ε that survives under these conditions can assemble into the complex.

Abnormal distribution of ATG9A in AAGAB-KO cellsGiven that AAGAB-KO cells exhibit both a marked reduction in AP-4 levels and an increase in ATG9A levels (Figure 4; Supplemental Figure S1B) reminiscent of those observed in AP-4 deficiency (Mattera et al., 2017; Davies et al., 2018; De Pace et al., 2018; Ivankovic et al., 2020; Behne et al., 2020), we asked whether the cellular distribution of ATG9A is also altered in these cells. Immunofluorescence microscopy analysis showed that, consistent with the results of the immunoblotting experiments, inactivation of the AAGAB gene caused a marked reduction in AP-4 ε (Figure 7, A and B; quantitation in panel E) as well as in AP-1 γ and AP-2 α signals (Supplemental Figure S2, A–D; Wan et al., 2021). Importantly, AAGAB-KO cells also displayed increased accumulation of ATG9A at the TGN (Figure 7, A and B; quantification in panel E), similar to that observed in various types of AP-4-deficient cells, including human HeLa, HAP1, and SH-SY5Y cells lines, mouse embryonic and human fibroblasts in primary culture, mouse hippocampal neurons in primary culture, and human iPSC-derived neurons (Mattera et al., 2017; Davies et al., 2018; De Pace et al., 2018; Ivankovic et al., 2020; Behne et al., 2020). This phenotype was reproducibly observed in different AAGAB-KO HeLa clones (Figure 7, A and B; Supplemental Figure S3, A and B), confirming that reduction of AP-4 levels in AAGAB-KO cells causes retention of ATG9A at the TGN. Expression of AAGAB-GFP in different clones of AAGAB-KO cells rescued both the expression of AP-4 ε and the normal distribution of ATG9A between peripheral and TGN compartments (Figure 7, C and D; Supplemental Figure S3, C and D; compare AAGAB-GFP-expressing and -nonexpressing cells in the same fields). Quantification of the rescue is shown in Figure 7E. This rescue demonstrated that the altered staining for AP-4 ε and ATG9A is a bona fide phenotype of AAGAB KO and not due to off-target effects of CRISPR-Cas9 editing.FIGURE 7:Altered immunostaining of AP-4 ε and ATG9A in AAGAB-KO HeLa cells. (A, B) WT and AAGAB-KO HeLa cells (clone #10) were fixed for 10 min with methanol at –20°C and incubated with mouse anti-AP-4 ε, rabbit anti-ATG9A, and sheep anti-TGN46 followed by Alexa Fluor 555-conjugated anti-mouse IgG, Alexa Fluor 488-conjugated anti-rabbit IgG, and Alexa Fluor 647-conjugated anti-sheep IgG. Nuclei were stained with DAPI. Stained cells were imaged by confocal microscopy. Single-channel images are shown in inverted gray scale and merge panels display AP-4 ε, ATG9A, and TGN46 in red, green, and magenta, respectively. Cells are outlined by dashed lines. Notice that in WT cells, AP-4 ε immunofluorescence exhibits a predominantly perinuclear distribution, along with scattered peripheral puncta that mostly represent nonspecific staining (Mattera et al., 2017, 2020b). This typical AP-4 ε immunostaining was observed in 86.1 ± 3.0% of WT cells compared with only 7.4 ± 2.6% of AAGAB-KO cells (see graph in E and corresponding legend for statistical analysis). ATG9A staining in WT cells is distributed between peripheral and juxtanuclear (TGN) pools, and AP-4 deficiency results in a marked concentration of the signal at the TGN (Mattera et al., 2017; Davies et al., 2018; De Pace et al., 2018; Behne et al., 2020). Importantly, the majority of AAGAB-KO cells exhibit ATG9A concentration at the TGN that mimics that in AP-4 KO cells. The typical distribution of ATG9A (peripheral plus perinuclear) was observed in 94.2 ± 4.2% of WT cells compared with only 14.4 ± 7.0% of AAGAB-KO cells (graph in panel E). (C, D) AAGAB-KO cells (clones #3 and 5 in panels C and D, respectively) were transfected with pEGFP-N1-AAGAB. Forty-eight h after transfection, cells were fixed for 10 min with methanol at –20°C and incubated with mouse anti-AP-4 ε (C) or rabbit anti-ATG9A (D) and sheep anti-TGN46 followed by Alexa Fluor 555-conjugated anti-mouse IgG (C) or Alexa Fluor 555-conjugated anti-rabbit IgG (D) Alexa Fluor 647-conjugated anti-sheep IgG, and GFP-Booster Atto488. Nuclei were stained with DAPI. Single-channel images are in inverted gray scale, whereas merge panels display AAGAB-GFP in green, AP-4 ε, or ATG9A in red—C and D, respectively—and TGN46 in magenta. Cells are outlined by dashed lines. We observed that 83.1 ± 6.7% and 90.8 ± 5.7% of AAGAB-GFP-transfected cells exhibited rescue of the AP-4 ε and ATG9A staining phenotypes, respectively (see graph in panel E and corresponding legend for statistical analysis). Scale bars: 20 μm. (E) Values represent the percentages of total cells exhibiting normal staining of AP-4 ε and ATG9A in WT, AAGAB-KO and AAGAB-KO cells transfected with AAGAB-GFP plasmid (mean ± SD of 4–6 measurements; **p < 0.01, one-way ANOVA followed by Dunnett’s test compared with WT). The total numbers of cells analyzed for AP-4 ε distribution were 950, 567, and 500 for WT, AAGAB-KO, and AAGAB-GFP rescue, respectively. The total numbers of cells scored for distribution of ATG9A were 977, 757, and 381 for WT, AAGAB-KO, and AAGAB-GFP rescue, respectively. “Normal distribution” means AP-4 ε at the TGN and ATG9A at both the TGN and the peripheral cytoplasm.It is worth noting that the AAGAB KO did not alter the localization of TGN46 to the TGN (Figure 7, Supplemental Figure S2, and Supplemental Figure S3). This is despite the facts that (1) the cytosolic tail of TGN46 has a tyrosine-based sorting signal that interacts with the AP-1 μ1 and AP-2 μ2 subunits (Ohno et al., 1995), (2) AAGAB KO markedly reduces AP-1 and AP-2 levels and increases the localization of the AP-2 cargo transferrin receptor at the plasma membrane (Figure 4; Supplemental Figure S1B; Supplemental Figure S2; Gulbranson et al., 2019; Wan et al., 2021), and (3) TGN46 cycles between the TGN, plasma membrane, and endosomes, albeit with predominant localization to the TGN at steady state (Reaves et al., 1993). The independence of TGN46 localization on AAGAB may be due to the presence of a TGN-retention determinant in the transmembrane domain (Ponnambalam et al., 1994) such that the population that exits the TGN is small, and to the sufficiency of residual levels of AP-1 and AP-2 in AAGAB-KO cells for retrieval of this small population to the TGN.

Abnormal distribution of ATG9A in AAGAB-KO cells

Given that AAGAB-KO cells exhibit both a marked reduction in AP-4 levels and an increase in ATG9A levels (Figure 4; Supplemental Figure S1B) reminiscent of those observed in AP-4 deficiency (Mattera et al., 2017; Davies et al., 2018; De Pace et al., 2018; Ivankovic et al., 2020; Behne et al., 2020), we asked whether the cellular distribution of ATG9A is also altered in these cells. Immunofluorescence microscopy analysis showed that, consistent with the results of the immunoblotting experiments, inactivation of the AAGAB gene caused a marked reduction in AP-4 ε (Figure 7, A and B; quantitation in panel E) as well as in AP-1 γ and AP-2 α signals (Supplemental Figure S2, A–D; Wan et al., 2021). Importantly, AAGAB-KO cells also displayed increased accumulation of ATG9A at the TGN (Figure 7, A and B; quantification in panel E), similar to that observed in various types of AP-4-deficient cells, including human HeLa, HAP1, and SH-SY5Y cells lines, mouse embryonic and human fibroblasts in primary culture, mouse hippocampal neurons in primary culture, and human iPSC-derived neurons (Mattera et al., 2017; Davies et al., 2018; De Pace et al., 2018; Ivankovic et al., 2020; Behne et al., 2020). This phenotype was reproducibly observed in different AAGAB-KO HeLa clones (Figure 7, A and B; Supplemental Figure S3, A and B), confirming that reduction of AP-4 levels in AAGAB-KO cells causes retention of ATG9A at the TGN. Expression of AAGAB-GFP in different clones of AAGAB-KO cells rescued both the expression of AP-4 ε and the normal distribution of ATG9A between peripheral and TGN compartments (Figure 7, C and D; Supplemental Figure S3, C and D; compare AAGAB-GFP-expressing and -nonexpressing cells in the same fields). Quantification of the rescue is shown in Figure 7E. This rescue demonstrated that the altered staining for AP-4 ε and ATG9A is a bona fide phenotype of AAGAB KO and not due to off-target effects of CRISPR-Cas9 editing.

Altered immunostaining of AP-4 ε and ATG9A in AAGAB-KO HeLa cells. (A, B) WT and AAGAB-KO HeLa cells (clone #10) were fixed for 10 min with methanol at –20°C and incubated with mouse anti-AP-4 ε, rabbit anti-ATG9A, and sheep anti-TGN46 followed by Alexa Fluor 555-conjugated anti-mouse IgG, Alexa Fluor 488-conjugated anti-rabbit IgG, and Alexa Fluor 647-conjugated anti-sheep IgG. Nuclei were stained with DAPI. Stained cells were imaged by confocal microscopy. Single-channel images are shown in inverted gray scale and merge panels display AP-4 ε, ATG9A, and TGN46 in red, green, and magenta, respectively. Cells are outlined by dashed lines. Notice that in WT cells, AP-4 ε immunofluorescence exhibits a predominantly perinuclear distribution, along with scattered peripheral puncta that mostly represent nonspecific staining (Mattera et al., 2017, 2020b). This typical AP-4 ε immunostaining was observed in 86.1 ± 3.0% of WT cells compared with only 7.4 ± 2.6% of AAGAB-KO cells (see graph in E and corresponding legend for statistical analysis). ATG9A staining in WT cells is distributed between peripheral and juxtanuclear (TGN) pools, and AP-4 deficiency results in a marked concentration of the signal at the TGN (Mattera et al., 2017; Davies et al., 2018; De Pace et al., 2018; Behne et al., 2020). Importantly, the majority of AAGAB-KO cells exhibit ATG9A concentration at the TGN that mimics that in AP-4 KO cells. The typical distribution of ATG9A (peripheral plus perinuclear) was observed in 94.2 ± 4.2% of WT cells compared with only 14.4 ± 7.0% of AAGAB-KO cells (graph in panel E). (C, D) AAGAB-KO cells (clones #3 and 5 in panels C and D, respectively) were transfected with pEGFP-N1-AAGAB. Forty-eight h after transfection, cells were fixed for 10 min with methanol at –20°C and incubated with mouse anti-AP-4 ε (C) or rabbit anti-ATG9A (D) and sheep anti-TGN46 followed by Alexa Fluor 555-conjugated anti-mouse IgG (C) or Alexa Fluor 555-conjugated anti-rabbit IgG (D) Alexa Fluor 647-conjugated anti-sheep IgG, and GFP-Booster Atto488. Nuclei were stained with DAPI. Single-channel images are in inverted gray scale, whereas merge panels display AAGAB-GFP in green, AP-4 ε, or ATG9A in red—C and D, respectively—and TGN46 in magenta. Cells are outlined by dashed lines. We observed that 83.1 ± 6.7% and 90.8 ± 5.7% of AAGAB-GFP-transfected cells exhibited rescue of the AP-4 ε and ATG9A staining phenotypes, respectively (see graph in panel E and corresponding legend for statistical analysis). Scale bars: 20 μm. (E) Values represent the percentages of total cells exhibiting normal staining of AP-4 ε and ATG9A in WT, AAGAB-KO and AAGAB-KO cells transfected with AAGAB-GFP plasmid (mean ± SD of 4–6 measurements; **p < 0.01, one-way ANOVA followed by Dunnett’s test compared with WT). The total numbers of cells analyzed for AP-4 ε distribution were 950, 567, and 500 for WT, AAGAB-KO, and AAGAB-GFP rescue, respectively. The total numbers of cells scored for distribution of ATG9A were 977, 757, and 381 for WT, AAGAB-KO, and AAGAB-GFP rescue, respectively. “Normal distribution” means AP-4 ε at the TGN and ATG9A at both the TGN and the peripheral cytoplasm.

It is worth noting that the AAGAB KO did not alter the localization of TGN46 to the TGN (Figure 7, Supplemental Figure S2, and Supplemental Figure S3). This is despite the facts that (1) the cytosolic tail of TGN46 has a tyrosine-based sorting signal that interacts with the AP-1 μ1 and AP-2 μ2 subunits (Ohno et al., 1995), (2) AAGAB KO markedly reduces AP-1 and AP-2 levels and increases the localization of the AP-2 cargo transferrin receptor at the plasma membrane (Figure 4; Supplemental Figure S1B; Supplemental Figure S2; Gulbranson et al., 2019; Wan et al., 2021), and (3) TGN46 cycles between the TGN, plasma membrane, and endosomes, albeit with predominant localization to the TGN at steady state (Reaves et al., 1993). The independence of TGN46 localization on AAGAB may be due to the presence of a TGN-retention determinant in the transmembrane domain (Ponnambalam et al., 1994) such that the population that exits the TGN is small, and to the sufficiency of residual levels of AP-1 and AP-2 in AAGAB-KO cells for retrieval of this small population to the TGN.

DISCUSSIONAAGAB/p34 was originally identified as a cytosolic protein that binds AP-1 γ and AP-2 α but is not enriched in clathrin-coated vesicles, leading to the proposal that AAGAB could function as a chaperone to prevent binding of soluble AP-1 and AP-2 complexes to soluble clathrin, aid in the recruitment of these complexes to membranes, or remove the assembled complexes from clathrin coats (Page et al., 1999). More recent studies demonstrated that the actual role of AAGAB is to stabilize the AP-1 γ and AP-2 α subunits during assembly of the corresponding complexes (Gulbranson et al., 2019; Wan et al., 2021). Our study extends these findings by demonstrating that AAGAB also binds to and stabilizes AP-4 ε and σ4, thus contributing to the assembly of the AP-4 complex.AAGAB was shown to participate in the sequential assembly of the AP-1 and AP-2 complexes, through an initial interaction with AP-1 γ or AP-2 α, followed by addition of AP-1 σ1 or AP-2 σ2, dissociation of AAGAB, and binding of β1-μ1 or β2-μ2 hemicomplexes, respectively, to generate the corresponding heterotetramers (Gulbranson et al., 2019; Wan et al., 2021). Our results indicate that a similar series of events might take place in the assembly of the AP-4 complex. The fact that AAGAB binds both AP-4 ε and AP-4 σ4 suggests that AAGAB does not only stabilize both AP-4 subunits but also serves as platform for assembly of the AP-4 ε-σ4 hemicomplex. In this regard, AP-4 behaves like AP-1, for which AAGAB was shown to interact with both AP-1 γ and AP-1 σ1 (Wan et al., 2021). In contrast, AAGAB was found to bind AP-2 α but not AP-2 σ2 (Gulbranson et al., 2019). Our Y2H analyses, however, show that AAGAB does bind AP-2 σ2, suggesting that the assembly of all three complexes (AP-1, AP-2 and AP-4) follows a similar pathway.AAGAB KO disrupts this pathway, leading to proteasomal degradation of unassembled AP-4 subunits. This outcome is similar to that resulting from KO or mutations in AP-4 subunits, in which the other subunits are degraded (Matsuda et al., 2008; Hirst et al., 2013; Davies et al., 2018; De Pace et al., 2018; Behne et al., 2020). It is noteworthy that the small remnant of AP-4 ε in AAGAB-KO cells coimmunoprecipitates with AP-4 β4, suggesting that AAGAB may increase the efficiency of assembly primarily through stabilization of the subunits.At present, it is unclear why the AP-1 β1, AP-2 β2, and AP-4 β4 subunits do not interact with AAGAB, even though their stability depends on AAGAB expression (Gulbranson et al., 2019; Wan et al., 2021; this study). Our analyses also show that AP-3 δ and σ3, which are homologous to AP-4 ε and σ4, respectively, fail to interact with AAGAB. This finding is in line with the previous report that levels of AP-3 subunits are unaltered in AAGAB-KO cells (Wan et al., 2021). We speculate that additional chaperones might participate in the stabilization of other AP-complex subunits during assembly.The physiological role of AAGAB in the assembly of functional AP-4 is underscored by our observation that AAGAB-KO cells exhibit phenotypic defects typical of AP-4 deficiency, including the lack of AP-4 staining and accumulation of ATG9A at the TGN. Because AAGAB KO also affects AP-1 and AP-2 (Gulbranson et al., 2019; Wan et al., 2021), it could be argued that the ATG9A phenotype results from defects in these complexes. However, AP-1 KO and AP-2 KO do not cause ATG9A retention at the TGN, at least in the nutrient-replete medium used in our experiments (Popovic and Dikic, 2014; Mattera et al., 2017), supporting the conclusion that the ATG9A phenotype in AAGAB-KO cells is specifically caused by the reduction in AP-4 levels.Despite the critical role of AAGAB in the assembly of AP-1, AP-2, and AP-4, mutations in AAGAB and AP complex subunits cause different diseases in humans. Monoallelic loss-of-function mutations in AAGAB cause the autosomal-dominant disorder PPKP1, characterized by the development of focal skin lesions of variable severity in the palms and soles (Giehl et al., 2012; Pöhler et al., 2012, 2013). Given that AAGAB is ubiquitously expressed, it is unclear why the manifestations of AAGAB deficiency are limited to the skin. Moreover, foci appear in the second or third decade of life and coalesce into larger lesions in an age-dependent manner. These features, together with the lack of evidence for loss of AAGAB heterozygosity or acquisition of compound heterozygous AAGAB mutations in PPKP1 skin lesions, suggest that mutations in a second gene specifically expressed in the skin might promote disease development (Pöhler et al., 2012). Biallelic loss-of-function mutations in AAGAB are likely lethal, as no individuals homozygous for inactive AAGAB alleles were identified in highly inbred families with PPKP1 (Charfeddine et al., 2016).In contrast, biallelic mutations in AP-1, AP-2, and AP-4 subunits cause autosomal-recessive “coatopathies” that most often affect the central nervous system (Dell’Angelica and Bonifacino, 2019). Individuals with monoallelic mutations in AP subunits do not exhibit any symptoms, except for the reported association of heterozygous variants of AP-4 ε (AP4E1 gene) with persistent developmental stuttering in some individuals (Raza et al., 2015) and of AP-2 σ2 (AP2S1 gene) with familial hypocalciuric hypercalcemia type 3 (FHH3; Nesbit et al., 2013). Diseases caused by biallelic mutations in the AP-1 σ1A (AP1S1 gene; Montpetit et al., 2008), σ1C (AP1S3 gene; Setta-Kaffetzi et al., 2014) and β1 (AP1B1 gene; Alsaif et al., 2019; Boyden et al., 2019; Ito et al., 2021) subunits, however, do have skin manifestations, probably owing to the requirement of AP-1 for polarized sorting in epithelial cells (Fölsch et al., 1999; Gravotta et al., 2012). It is thus possible that skin lesions in PPKP1 arise from reduction in AP-1 levels in conjunction with a second-gene mutation. AP-2 deficiency specifically in the skin has also been proposed to underlie PPKP1 by impairing clathrin-mediated endocytosis of growth factor receptors and thus causing increased receptor signaling and keratinocyte proliferation (Pöhler et al., 2012, 2013). We think that partial AP-4 deficiency in the skin could also contribute to PPKP1 through missorting of ATG9A and DAGLB, with consequent defects in autophagy (Mattera et al., 2017; Davies et al., 2018; De Pace et al., 2018; Ivankovic et al., 2020; Behne et al., 2020) and endocannabinoid signaling (Davies et al., 2022), respectively—both processes that are important for normal skin physiology (Klapan et al., 2022; Río et al., 2018).The identification of AAGAB as an AP-4 chaperone sheds light on the assembly of the AP-4 complex that might be relevant to AP-4-deficiency syndrome. Most of the mutations reported in patients with this syndrome cause frameshifts or premature termination, with loss of function of the corresponding proteins (Ebrahimi-Fakhari et al., 2018; Mattera et al., 2020a). However, there are missense variants of AP-4 subunits of uncertain significance recorded in the gnomAD database (https://gnomad .broadinstitute.org/), which could be pathogenic under conditions of homozygosity or compound heterozygosity. Knowledge that AP-4 assembly is chaperone-assisted raises the prospect of treatment of such cases by overexpression of AAGAB. Overexpression of AAGAB could also be used to increase the yield of assembled recombinant AP-4 complex in bacterial or eukaryotic host cells for structural and functional studies, as recently done for AP-1 and AP-2 (Wang et al., 2022), as well as for diagnostic applications.

AAGAB/p34 was originally identified as a cytosolic protein that binds AP-1 γ and AP-2 α but is not enriched in clathrin-coated vesicles, leading to the proposal that AAGAB could function as a chaperone to prevent binding of soluble AP-1 and AP-2 complexes to soluble clathrin, aid in the recruitment of these complexes to membranes, or remove the assembled complexes from clathrin coats (Page et al., 1999). More recent studies demonstrated that the actual role of AAGAB is to stabilize the AP-1 γ and AP-2 α subunits during assembly of the corresponding complexes (Gulbranson et al., 2019; Wan et al., 2021). Our study extends these findings by demonstrating that AAGAB also binds to and stabilizes AP-4 ε and σ4, thus contributing to the assembly of the AP-4 complex.

AAGAB was shown to participate in the sequential assembly of the AP-1 and AP-2 complexes, through an initial interaction with AP-1 γ or AP-2 α, followed by addition of AP-1 σ1 or AP-2 σ2, dissociation of AAGAB, and binding of β1-μ1 or β2-μ2 hemicomplexes, respectively, to generate the corresponding heterotetramers (Gulbranson et al., 2019; Wan et al., 2021). Our results indicate that a similar series of events might take place in the assembly of the AP-4 complex. The fact that AAGAB binds both AP-4 ε and AP-4 σ4 suggests that AAGAB does not only stabilize both AP-4 subunits but also serves as platform for assembly of the AP-4 ε-σ4 hemicomplex. In this regard, AP-4 behaves like AP-1, for which AAGAB was shown to interact with both AP-1 γ and AP-1 σ1 (Wan et al., 2021). In contrast, AAGAB was found to bind AP-2 α but not AP-2 σ2 (Gulbranson et al., 2019). Our Y2H analyses, however, show that AAGAB does bind AP-2 σ2, suggesting that the assembly of all three complexes (AP-1, AP-2 and AP-4) follows a similar pathway.

AAGAB KO disrupts this pathway, leading to proteasomal degradation of unassembled AP-4 subunits. This outcome is similar to that resulting from KO or mutations in AP-4 subunits, in which the other subunits are degraded (Matsuda et al., 2008; Hirst et al., 2013; Davies et al., 2018; De Pace et al., 2018; Behne et al., 2020). It is noteworthy that the small remnant of AP-4 ε in AAGAB-KO cells coimmunoprecipitates with AP-4 β4, suggesting that AAGAB may increase the efficiency of assembly primarily through stabilization of the subunits.

At present, it is unclear why the AP-1 β1, AP-2 β2, and AP-4 β4 subunits do not interact with AAGAB, even though their stability depends on AAGAB expression (Gulbranson et al., 2019; Wan et al., 2021; this study). Our analyses also show that AP-3 δ and σ3, which are homologous to AP-4 ε and σ4, respectively, fail to interact with AAGAB. This finding is in line with the previous report that levels of AP-3 subunits are unaltered in AAGAB-KO cells (Wan et al., 2021). We speculate that additional chaperones might participate in the stabilization of other AP-complex subunits during assembly.

The physiological role of AAGAB in the assembly of functional AP-4 is underscored by our observation that AAGAB-KO cells exhibit phenotypic defects typical of AP-4 deficiency, including the lack of AP-4 staining and accumulation of ATG9A at the TGN. Because AAGAB KO also affects AP-1 and AP-2 (Gulbranson et al., 2019; Wan et al., 2021), it could be argued that the ATG9A phenotype results from defects in these complexes. However, AP-1 KO and AP-2 KO do not cause ATG9A retention at the TGN, at least in the nutrient-replete medium used in our experiments (Popovic and Dikic, 2014; Mattera et al., 2017), supporting the conclusion that the ATG9A phenotype in AAGAB-KO cells is specifically caused by the reduction in AP-4 levels.

Despite the critical role of AAGAB in the assembly of AP-1, AP-2, and AP-4, mutations in AAGAB and AP complex subunits cause different diseases in humans. Monoallelic loss-of-function mutations in AAGAB cause the autosomal-dominant disorder PPKP1, characterized by the development of focal skin lesions of variable severity in the palms and soles (Giehl et al., 2012; Pöhler et al., 2012, 2013). Given that AAGAB is ubiquitously expressed, it is unclear why the manifestations of AAGAB deficiency are limited to the skin. Moreover, foci appear in the second or third decade of life and coalesce into larger lesions in an age-dependent manner. These features, together with the lack of evidence for loss of AAGAB heterozygosity or acquisition of compound heterozygous AAGAB mutations in PPKP1 skin lesions, suggest that mutations in a second gene specifically expressed in the skin might promote disease development (Pöhler et al., 2012). Biallelic loss-of-function mutations in AAGAB are likely lethal, as no individuals homozygous for inactive AAGAB alleles were identified in highly inbred families with PPKP1 (Charfeddine et al., 2016).

In contrast, biallelic mutations in AP-1, AP-2, and AP-4 subunits cause autosomal-recessive “coatopathies” that most often affect the central nervous system (Dell’Angelica and Bonifacino, 2019). Individuals with monoallelic mutations in AP subunits do not exhibit any symptoms, except for the reported association of heterozygous variants of AP-4 ε (AP4E1 gene) with persistent developmental stuttering in some individuals (Raza et al., 2015) and of AP-2 σ2 (AP2S1 gene) with familial hypocalciuric hypercalcemia type 3 (FHH3; Nesbit et al., 2013). Diseases caused by biallelic mutations in the AP-1 σ1A (AP1S1 gene; Montpetit et al., 2008), σ1C (AP1S3 gene; Setta-Kaffetzi et al., 2014) and β1 (AP1B1 gene; Alsaif et al., 2019; Boyden et al., 2019; Ito et al., 2021) subunits, however, do have skin manifestations, probably owing to the requirement of AP-1 for polarized sorting in epithelial cells (Fölsch et al., 1999; Gravotta et al., 2012). It is thus possible that skin lesions in PPKP1 arise from reduction in AP-1 levels in conjunction with a second-gene mutation. AP-2 deficiency specifically in the skin has also been proposed to underlie PPKP1 by impairing clathrin-mediated endocytosis of growth factor receptors and thus causing increased receptor signaling and keratinocyte proliferation (Pöhler et al., 2012, 2013). We think that partial AP-4 deficiency in the skin could also contribute to PPKP1 through missorting of ATG9A and DAGLB, with consequent defects in autophagy (Mattera et al., 2017; Davies et al., 2018; De Pace et al., 2018; Ivankovic et al., 2020; Behne et al., 2020) and endocannabinoid signaling (Davies et al., 2022), respectively—both processes that are important for normal skin physiology (Klapan et al., 2022; Río et al., 2018).

The identification of AAGAB as an AP-4 chaperone sheds light on the assembly of the AP-4 complex that might be relevant to AP-4-deficiency syndrome. Most of the mutations reported in patients with this syndrome cause frameshifts or premature termination, with loss of function of the corresponding proteins (Ebrahimi-Fakhari et al., 2018; Mattera et al., 2020a). However, there are missense variants of AP-4 subunits of uncertain significance recorded in the gnomAD database (https://gnomad .broadinstitute.org/), which could be pathogenic under conditions of homozygosity or compound heterozygosity. Knowledge that AP-4 assembly is chaperone-assisted raises the prospect of treatment of such cases by overexpression of AAGAB. Overexpression of AAGAB could also be used to increase the yield of assembled recombinant AP-4 complex in bacterial or eukaryotic host cells for structural and functional studies, as recently done for AP-1 and AP-2 (Wang et al., 2022), as well as for diagnostic applications.

MATERIALS AND METHODSRecombinant DNA constructsFull-length human AAGAB cDNA (NM_024666.5) was PCR-amplified from the Mammalian Gene Collection human ORFeome v8.1 library (Transomic Technologies) with primers including EcoRI (forward) and XhoI (reverse) restriction sites. Human AAGAB EcoRI/XhoI fragments were subcloned in the EcoRI/SalI sites of the GAL4 DNA-binding domain vector pGBT9 and of pEGFP-N1-A206K (Clontech). The pGBT9-human AAGAB 1-155 and 155-315 constructs were generated by site-directed mutagenesis of the full-length template (QuikChange, Agilent). The human AP-4 ε construct (NM_007347.4) with an N-terminal two Strep/one FLAG (TSF) tag subcloned in pcDNA 3.1 and the pGBT9-human tepsin construct were previously described (Mattera et al., 2015). The pcDNA 3.1-TSF-myrlysin (a subunit of the BORC complex also termed BORCS5 or LOH12CR1) was described by Pu et al. (2015). The cDNAs encoding mouse AP-1 γ1 (abbreviated as γ, NM_001301211.1), rat AP-2 αC (abbreviated as α), and human AP-3 δ (AF002163.1) were subcloned in the GAL4 activation domain vector pGADT7 (Clontech; Mattera et al., 2011). The human AP-3 δ cDNA was PCR-amplified with primers containing EcoRI and XhoI sites, and subcloned at the EcoRI/SalI sites of pGBT9 (Clontech). Complementary DNAs encoding full-length human AP-4 ε, human ε deletion mutants 1–138, 1–260, 1–368, 262–368, and 727–1137 (part of hinge plus ear domain) and human β4 (NM_001253852.2) subcloned in pGADT7 were described in Boehm et al. (2001). The pGADT7 human AP-4 ε 600–839 (hinge domain) and 883–1137 (ear domain) constructs were prepared by site-directed mutagenesis. The human AP-4 μ4 construct (NM_004722.4) in the GAL4 activation domain vector pACT2 (Clontech) was previously described (Guo et al., 2013). Complementary DNAs encoding human AP-1 σ1A, (NM_001283.3), AP-1 σ1B (NM_003916.4), and AP-1 σ1C (AF393369) were subcloned into the EcoRI/XhoI sites of pGADT7 (both the σ1A and σ1B constructs in pGADT7 contain a LELQMNRRY nonapeptide extension at their C-termini). The rat AP-2 σ2 cDNA (NM_022952.2) was PCR-amplified from pBridge-mouse presenilin2 cytosolic tail.rat AP-2 σ2 (Sannerud et al., 2016) with primers containing EcoRI and XhoI sites and subcloned into the corresponding sites of pGADT7. The human AP-3 σ3A (NM_001284.3) and AP-3 σ3B (NM_005829.5) constructs in pACT2 have been described (Dell’Angelica et al., 1997). The mouse AP-4 σ4 cDNA (NM_021710.4) subcloned into the BamH1/SalI sites of the GAL4 activation domain vector pGAD424 (Clontech) was used as a template to generate the corresponding V88D, I103S, and V88D-I103S mutants by site-directed mutagenesis. The pGAD424-mouse σ4 was subjected to site-directed mutagenesis to eliminate the internal EcoR1 site. The mouse σ4 cDNA without an internal EcoRI site was PCR-amplified with primers including EcoRI (forward) and NotI (reverse) sites and subcloned into the large restriction fragment obtained after digestion of pCI-neo-(Myc)3-mouse μ1B (Guo et al., 2013) with EcoRI and NotI. This ligation generated a mouse σ4 construct appended with an N-terminal triple Myc tag and the 10-amino acid spacer sequence GSGSGGSGSG (pCIneo-AP-4 (Myc)3-σ4).AntibodiesThe following antibodies were used in this study: rabbit anti-AAGAB (Sigma cat. HPA040174, 1:500 for IB); mouse anti-AP-1 γ (BD Biosciences cat. 610385, 1:400 for IB, 1:500 for IF); mouse anti-AP-2 α (clone AP6, ThermoFisher cat. MA1-064, 1:200 for IF); mouse anti-AP-4 ε (BD Biosciences cat. 612018, 1:400 for IB, 1:75 for IF); rabbit anti-AP-4 β4 (C-terminus) generated in our laboratory (anti-AP-4 β4C in Dell’Angelica et al., 1999a, 1: 500 for IB, 2 μg for IP); rabbit anti-ATG9A (Abcam cat. ab108338, 1:1,000 for IB, 1:200 for IF); mouse anti-FLAG (M2, Sigma cat. F3165, 1: 1,000 for IB); mouse anti-GFP (Roche cat. 11814460001, 1:750 for IB, 2 μg for IP); sheep anti-TGN46 (Bio-Rad cat. AHP500G, 1:1,000 for IF); rabbit anti-Myc tag (Cell Signaling cat. 2272, 1:1,000 for IB); mouse anti-TfR (clone H68.4, ThermoFisher cat. 13-6800, 1:1,000 for IF; mouse anti-α-tubulin (Sigma cat. T9026, 1:2,000 for IB); Alexa Fluor 488-conjugated donkey anti-rabbit IgG (ThermoFisher cat. A-21206, 1:1,000 for IF); Alexa Fluor 555-conjugated donkey anti-mouse IgG (ThermoFisher cat. A-31570, 1:1,000 for IF); Alexa Fluor 555-conjugated anti-rabbit IgG (ThermoFisher cat. A-31572, 1:1,000 for IF); Alexa Fluor 647-conjugated donkey anti-sheep IgG (ThermoFisher cat. A-21448, 1:1,000 for IF); GFP-Booster Atto488 (Chromotek cat. Gb2AF488. 1:400 for IF); HRP-conjugated sheep anti-mouse IgG (GE Healthcare cat. NXA931, 1:5,000 for IB); HRP-conjugated donkey anti-rabbit IgG (GE Healthcare cat. NA934V, 1:5,000 for IB); IRDye 680RD goat anti-mouse IgG (LI-COR cat. 926-68070; 1:10,000 for IB); and IRDye 800CW donkey anti-rabbit IgG (LI-COR cat. 926-32213, 1:10,000 for IB).Tandem affinity purification and mass spectrometryThe identification of copurifying proteins by TAP-MS from lysates of H4 human neuroglioma cells stably transfected with TSF-tagged human AP-4 ε was described previously (Mattera et al., 2015). A spreadsheet with all the proteins identified by MS and their filtering according to CRAPome scores (www.crapome.org; Mellacheruvu et al., 2013) can be found in Dataset S1 of Mattera et al. (2017).Yeast two-hybrid assaysAssays were performed following the guidelines in the Matchmaker two-hybrid system (Takara) as described by Mattera et al. (2003). The AH109 reporter strain used in the assays was maintained on YPD plates containing 30 μg/ml adenine hemisulfate (MP Biomedicals). The plates with medium lacking His, Leu, and Trp (labeled “–His” in the figures) and medium lacking only Leu and Trp (labeled “+His” in the figures) were prepared with dropout base agar (MP Biomedicals) using the corresponding supplement mixtures (MP Biomedicals and Takara) and supplemented with 30 μg/ml adenine hemisulfate. The –His medium was also supplemented with 1 mM 3-amino-1,2,4 triazole (3AT) (Sigma-Aldrich) after autoclaving and immediately preceding pouring of plates.Generation of AAGAB-KO HeLa cellsKO clones targeting either exon 1 or exon 2 of the human AAGAB gene (NCBI Gene ID 79719) were generated in HeLa cells using CRISPR-Cas9 (Cong et al., 2013, Ran et al., 2013). We used 20-nucleotide long guide RNA (gRNA) antisense strand 5′ CTGGTCTCCTGAGAAGACGG or sense strand 5′ CCGTCTTCTCAGGAGACCAG to target exon 1 (nucleotides 44–63 in the cDNA sequence; the first gRNA is immediately adjacent to an NGG protospacer adjacent motif, whereas the second has a single nucleotide spacing) (Supplemental Figure S1A). We also targeted AAGAB gene exon 2 with gRNA antisense 5′ CAAAGTAAACCACAAATGCT (nucleotides 234–253 in the cDNA sequence; Supplemental Figure S1A). The targeting complementary oligonucleotides including CACCG and AAAC overhang sequences (forward and reverse oligonucleotides, respectively) were annealed and subcloned in BbsI-digested PX458 (pSpCas9(BB)-2A-GFP; Addgene plasmid cat. 48138). Constructs were verified by sequencing with primer 5′ GAGGGCCTATTTCCCATGATTC. HeLa cells were transfected with the targeting constructs using Lipofectamine 2000 (ThermoFisher). Twenty-four h after transfection, cells were trypsinized and GFP-positive cells were subsequently sorted into individual clones in 96-well plates at the Flow Cytometry Core, National Heart Lung and Blood Institute, NIH. Cell clones were subsequently split, plated on replica plates for maintenance and screening, and analyzed by immunoblotting using rabbit anti-AAGAB. Cell clones lacking expression of AAGAB were subsequently genotyped by sequencing of PCR fragments amplified from genomic DNA using forward primers 5′ GAAGGTCTGGGTGGGCATTT or 5′ CCTGGTCATTCCGAGAACGC and reverse primer 5′ TCCCGGAGCAGCAAGGATA when targeting exon 1, or forward primer 5′ GGAACAGAAGATCTTATTGTGGAAGTG and reverse primer 5′ CAGACACTCTATCGCAGACCA when targeting exon 2.Cell culture, transfection, immunoprecipitation, and immunoblottingHeLa cells were cultured on 100-mm dishes or multiwell plates in DMEM (Quality Biological) supplemented with 10% fetal bovine serum (Corning), 100 units/ml penicillin, and 100 μg/ml streptomycin (Corning) at 37°C under a humidified atmosphere (95:5 air:CO2). Cells plated on 100-mm dishes or 6-well plates were transfected with 7 μg or 1.4 μg of plasmid DNA, respectively, using the X-tremeGENE9 DNA transfection reagent (Roche) as described previously (Mattera et al., 2020b). Cells were washed twice with PBS and lysed in 50 mM Tris-HCl pH 7.4, 0.8% (vol/vol) Triton X-100 and 75 mM NaCl supplemented with a protease inhibitor cocktail (EDTA-free Complete, Roche). Lysates from cells cultured on 100-mm dishes were subjected to immunoprecipitation using 2 μg of antibodies immobilized onto 25 μl of Protein G-Sepharose beads (GE Healthcare) as described previously (Mattera et al., 2003). Lysates or immunoprecipitated complexes were subjected to SDS–PAGE in 10% acrylamide gels and transferred onto nylon membranes (Immobilon-P, Millipore). Membranes were subsequently stained with Ponceau S (Sigma Aldrich) and immunoblotted with primary and secondary antibodies using buffers containing 0.05% Tween (Sigma Aldrich) and 3% dry milk (BioRad). Blots were developed with Western Lighting Plus-ECL (PerkinElmer) or SuperSignal West Pico Plus (ThermoFisher) reagents.Cycloheximide chaseWT and AAGAB-KO cells were plated on 6-well plates and cultured to approximately 80% confluency. Cells were subsequently incubated with 0.1 mg/ml cycloheximide (CHX; Sigma-Aldrich) added to regular culture medium for 0–20 h. At the indicated times, cells were washed twice with 2 ml of PBS and lysed in 250 μl of 15 mM HEPES pH 7.4, 0.5% (vol/vol) Triton X-100,and 150 mM NaCl supplemented with a protease inhibitor cocktail (EDTA-free Complete, Roche). Lysates were incubated for 30 min at 4°C with end-over-end rotation, followed by centrifugation for 15 min at 21,000 × g and 4°C. Supernatants were subsequently supplemented with 1X Laemmli buffer and subjected to SDS–PAGE and immunoblotting as described in the previous section. In some experiments, cells with or without 0.1 mg/ml CHX were incubated for 4.5–6.5 h in the presence or absence of 30 μM MG132 (Sigma-Aldrich). At the end of this period, cells were processed as described above.Immunofluorescence microscopyCells were fixed for 10 min with methanol at –20°C or 15 min with 4% paraformaldehyde at room temperature and incubated with primary (45–60 min at room temperature) and secondary (30–45 min at room temperature) antibodies diluted in PBS supplemented with 0.1% saponin, 0.1% bovine serum albumin, and 0.02% sodium azide as described in the figure legends. DAPI (ThermoFisher) at 300 nM, and GFP-Booster Atto488 (Chromotek) at 1:400 dilution were added during incubation with secondary antibodies. Relative changes in AP-4 ε staining and ATG9A distribution in AAGAB-KO cells, as well as rescue following transfection with AAGAB-GFP, were assessed by manual scoring of cells using a Zeiss Axio Imager.A1 fluorescence microscope fitted with a Plan Apochromat 63×/1.4 Oil DIC M27 objective. Confocal microscopy images were obtained using a Zeiss LSM 780 microscope with a 63×, 1.4 NA Plan Apochromat 63× objective.

MATERIALS AND METHODS

Recombinant DNA constructsFull-length human AAGAB cDNA (NM_024666.5) was PCR-amplified from the Mammalian Gene Collection human ORFeome v8.1 library (Transomic Technologies) with primers including EcoRI (forward) and XhoI (reverse) restriction sites. Human AAGAB EcoRI/XhoI fragments were subcloned in the EcoRI/SalI sites of the GAL4 DNA-binding domain vector pGBT9 and of pEGFP-N1-A206K (Clontech). The pGBT9-human AAGAB 1-155 and 155-315 constructs were generated by site-directed mutagenesis of the full-length template (QuikChange, Agilent). The human AP-4 ε construct (NM_007347.4) with an N-terminal two Strep/one FLAG (TSF) tag subcloned in pcDNA 3.1 and the pGBT9-human tepsin construct were previously described (Mattera et al., 2015). The pcDNA 3.1-TSF-myrlysin (a subunit of the BORC complex also termed BORCS5 or LOH12CR1) was described by Pu et al. (2015). The cDNAs encoding mouse AP-1 γ1 (abbreviated as γ, NM_001301211.1), rat AP-2 αC (abbreviated as α), and human AP-3 δ (AF002163.1) were subcloned in the GAL4 activation domain vector pGADT7 (Clontech; Mattera et al., 2011). The human AP-3 δ cDNA was PCR-amplified with primers containing EcoRI and XhoI sites, and subcloned at the EcoRI/SalI sites of pGBT9 (Clontech). Complementary DNAs encoding full-length human AP-4 ε, human ε deletion mutants 1–138, 1–260, 1–368, 262–368, and 727–1137 (part of hinge plus ear domain) and human β4 (NM_001253852.2) subcloned in pGADT7 were described in Boehm et al. (2001). The pGADT7 human AP-4 ε 600–839 (hinge domain) and 883–1137 (ear domain) constructs were prepared by site-directed mutagenesis. The human AP-4 μ4 construct (NM_004722.4) in the GAL4 activation domain vector pACT2 (Clontech) was previously described (Guo et al., 2013). Complementary DNAs encoding human AP-1 σ1A, (NM_001283.3), AP-1 σ1B (NM_003916.4), and AP-1 σ1C (AF393369) were subcloned into the EcoRI/XhoI sites of pGADT7 (both the σ1A and σ1B constructs in pGADT7 contain a LELQMNRRY nonapeptide extension at their C-termini). The rat AP-2 σ2 cDNA (NM_022952.2) was PCR-amplified from pBridge-mouse presenilin2 cytosolic tail.rat AP-2 σ2 (Sannerud et al., 2016) with primers containing EcoRI and XhoI sites and subcloned into the corresponding sites of pGADT7. The human AP-3 σ3A (NM_001284.3) and AP-3 σ3B (NM_005829.5) constructs in pACT2 have been described (Dell’Angelica et al., 1997). The mouse AP-4 σ4 cDNA (NM_021710.4) subcloned into the BamH1/SalI sites of the GAL4 activation domain vector pGAD424 (Clontech) was used as a template to generate the corresponding V88D, I103S, and V88D-I103S mutants by site-directed mutagenesis. The pGAD424-mouse σ4 was subjected to site-directed mutagenesis to eliminate the internal EcoR1 site. The mouse σ4 cDNA without an internal EcoRI site was PCR-amplified with primers including EcoRI (forward) and NotI (reverse) sites and subcloned into the large restriction fragment obtained after digestion of pCI-neo-(Myc)3-mouse μ1B (Guo et al., 2013) with EcoRI and NotI. This ligation generated a mouse σ4 construct appended with an N-terminal triple Myc tag and the 10-amino acid spacer sequence GSGSGGSGSG (pCIneo-AP-4 (Myc)3-σ4).

Recombinant DNA constructs

Full-length human AAGAB cDNA (NM_024666.5) was PCR-amplified from the Mammalian Gene Collection human ORFeome v8.1 library (Transomic Technologies) with primers including EcoRI (forward) and XhoI (reverse) restriction sites. Human AAGAB EcoRI/XhoI fragments were subcloned in the EcoRI/SalI sites of the GAL4 DNA-binding domain vector pGBT9 and of pEGFP-N1-A206K (Clontech). The pGBT9-human AAGAB 1-155 and 155-315 constructs were generated by site-directed mutagenesis of the full-length template (QuikChange, Agilent). The human AP-4 ε construct (NM_007347.4) with an N-terminal two Strep/one FLAG (TSF) tag subcloned in pcDNA 3.1 and the pGBT9-human tepsin construct were previously described (Mattera et al., 2015). The pcDNA 3.1-TSF-myrlysin (a subunit of the BORC complex also termed BORCS5 or LOH12CR1) was described by Pu et al. (2015). The cDNAs encoding mouse AP-1 γ1 (abbreviated as γ, NM_001301211.1), rat AP-2 αC (abbreviated as α), and human AP-3 δ (AF002163.1) were subcloned in the GAL4 activation domain vector pGADT7 (Clontech; Mattera et al., 2011). The human AP-3 δ cDNA was PCR-amplified with primers containing EcoRI and XhoI sites, and subcloned at the EcoRI/SalI sites of pGBT9 (Clontech). Complementary DNAs encoding full-length human AP-4 ε, human ε deletion mutants 1–138, 1–260, 1–368, 262–368, and 727–1137 (part of hinge plus ear domain) and human β4 (NM_001253852.2) subcloned in pGADT7 were described in Boehm et al. (2001). The pGADT7 human AP-4 ε 600–839 (hinge domain) and 883–1137 (ear domain) constructs were prepared by site-directed mutagenesis. The human AP-4 μ4 construct (NM_004722.4) in the GAL4 activation domain vector pACT2 (Clontech) was previously described (Guo et al., 2013). Complementary DNAs encoding human AP-1 σ1A, (NM_001283.3), AP-1 σ1B (NM_003916.4), and AP-1 σ1C (AF393369) were subcloned into the EcoRI/XhoI sites of pGADT7 (both the σ1A and σ1B constructs in pGADT7 contain a LELQMNRRY nonapeptide extension at their C-termini). The rat AP-2 σ2 cDNA (NM_022952.2) was PCR-amplified from pBridge-mouse presenilin2 cytosolic tail.rat AP-2 σ2 (Sannerud et al., 2016) with primers containing EcoRI and XhoI sites and subcloned into the corresponding sites of pGADT7. The human AP-3 σ3A (NM_001284.3) and AP-3 σ3B (NM_005829.5) constructs in pACT2 have been described (Dell’Angelica et al., 1997). The mouse AP-4 σ4 cDNA (NM_021710.4) subcloned into the BamH1/SalI sites of the GAL4 activation domain vector pGAD424 (Clontech) was used as a template to generate the corresponding V88D, I103S, and V88D-I103S mutants by site-directed mutagenesis. The pGAD424-mouse σ4 was subjected to site-directed mutagenesis to eliminate the internal EcoR1 site. The mouse σ4 cDNA without an internal EcoRI site was PCR-amplified with primers including EcoRI (forward) and NotI (reverse) sites and subcloned into the large restriction fragment obtained after digestion of pCI-neo-(Myc)3-mouse μ1B (Guo et al., 2013) with EcoRI and NotI. This ligation generated a mouse σ4 construct appended with an N-terminal triple Myc tag and the 10-amino acid spacer sequence GSGSGGSGSG (pCIneo-AP-4 (Myc)3-σ4).

AntibodiesThe following antibodies were used in this study: rabbit anti-AAGAB (Sigma cat. HPA040174, 1:500 for IB); mouse anti-AP-1 γ (BD Biosciences cat. 610385, 1:400 for IB, 1:500 for IF); mouse anti-AP-2 α (clone AP6, ThermoFisher cat. MA1-064, 1:200 for IF); mouse anti-AP-4 ε (BD Biosciences cat. 612018, 1:400 for IB, 1:75 for IF); rabbit anti-AP-4 β4 (C-terminus) generated in our laboratory (anti-AP-4 β4C in Dell’Angelica et al., 1999a, 1: 500 for IB, 2 μg for IP); rabbit anti-ATG9A (Abcam cat. ab108338, 1:1,000 for IB, 1:200 for IF); mouse anti-FLAG (M2, Sigma cat. F3165, 1: 1,000 for IB); mouse anti-GFP (Roche cat. 11814460001, 1:750 for IB, 2 μg for IP); sheep anti-TGN46 (Bio-Rad cat. AHP500G, 1:1,000 for IF); rabbit anti-Myc tag (Cell Signaling cat. 2272, 1:1,000 for IB); mouse anti-TfR (clone H68.4, ThermoFisher cat. 13-6800, 1:1,000 for IF; mouse anti-α-tubulin (Sigma cat. T9026, 1:2,000 for IB); Alexa Fluor 488-conjugated donkey anti-rabbit IgG (ThermoFisher cat. A-21206, 1:1,000 for IF); Alexa Fluor 555-conjugated donkey anti-mouse IgG (ThermoFisher cat. A-31570, 1:1,000 for IF); Alexa Fluor 555-conjugated anti-rabbit IgG (ThermoFisher cat. A-31572, 1:1,000 for IF); Alexa Fluor 647-conjugated donkey anti-sheep IgG (ThermoFisher cat. A-21448, 1:1,000 for IF); GFP-Booster Atto488 (Chromotek cat. Gb2AF488. 1:400 for IF); HRP-conjugated sheep anti-mouse IgG (GE Healthcare cat. NXA931, 1:5,000 for IB); HRP-conjugated donkey anti-rabbit IgG (GE Healthcare cat. NA934V, 1:5,000 for IB); IRDye 680RD goat anti-mouse IgG (LI-COR cat. 926-68070; 1:10,000 for IB); and IRDye 800CW donkey anti-rabbit IgG (LI-COR cat. 926-32213, 1:10,000 for IB).

The following antibodies were used in this study: rabbit anti-AAGAB (Sigma cat. HPA040174, 1:500 for IB); mouse anti-AP-1 γ (BD Biosciences cat. 610385, 1:400 for IB, 1:500 for IF); mouse anti-AP-2 α (clone AP6, ThermoFisher cat. MA1-064, 1:200 for IF); mouse anti-AP-4 ε (BD Biosciences cat. 612018, 1:400 for IB, 1:75 for IF); rabbit anti-AP-4 β4 (C-terminus) generated in our laboratory (anti-AP-4 β4C in Dell’Angelica et al., 1999a, 1: 500 for IB, 2 μg for IP); rabbit anti-ATG9A (Abcam cat. ab108338, 1:1,000 for IB, 1:200 for IF); mouse anti-FLAG (M2, Sigma cat. F3165, 1: 1,000 for IB); mouse anti-GFP (Roche cat. 11814460001, 1:750 for IB, 2 μg for IP); sheep anti-TGN46 (Bio-Rad cat. AHP500G, 1:1,000 for IF); rabbit anti-Myc tag (Cell Signaling cat. 2272, 1:1,000 for IB); mouse anti-TfR (clone H68.4, ThermoFisher cat. 13-6800, 1:1,000 for IF; mouse anti-α-tubulin (Sigma cat. T9026, 1:2,000 for IB); Alexa Fluor 488-conjugated donkey anti-rabbit IgG (ThermoFisher cat. A-21206, 1:1,000 for IF); Alexa Fluor 555-conjugated donkey anti-mouse IgG (ThermoFisher cat. A-31570, 1:1,000 for IF); Alexa Fluor 555-conjugated anti-rabbit IgG (ThermoFisher cat. A-31572, 1:1,000 for IF); Alexa Fluor 647-conjugated donkey anti-sheep IgG (ThermoFisher cat. A-21448, 1:1,000 for IF); GFP-Booster Atto488 (Chromotek cat. Gb2AF488. 1:400 for IF); HRP-conjugated sheep anti-mouse IgG (GE Healthcare cat. NXA931, 1:5,000 for IB); HRP-conjugated donkey anti-rabbit IgG (GE Healthcare cat. NA934V, 1:5,000 for IB); IRDye 680RD goat anti-mouse IgG (LI-COR cat. 926-68070; 1:10,000 for IB); and IRDye 800CW donkey anti-rabbit IgG (LI-COR cat. 926-32213, 1:10,000 for IB).

Tandem affinity purification and mass spectrometryThe identification of copurifying proteins by TAP-MS from lysates of H4 human neuroglioma cells stably transfected with TSF-tagged human AP-4 ε was described previously (Mattera et al., 2015). A spreadsheet with all the proteins identified by MS and their filtering according to CRAPome scores (www.crapome.org; Mellacheruvu et al., 2013) can be found in Dataset S1 of Mattera et al. (2017).

Tandem affinity purification and mass spectrometry

The identification of copurifying proteins by TAP-MS from lysates of H4 human neuroglioma cells stably transfected with TSF-tagged human AP-4 ε was described previously (Mattera et al., 2015). A spreadsheet with all the proteins identified by MS and their filtering according to CRAPome scores (www.crapome.org; Mellacheruvu et al., 2013) can be found in Dataset S1 of Mattera et al. (2017).

Yeast two-hybrid assaysAssays were performed following the guidelines in the Matchmaker two-hybrid system (Takara) as described by Mattera et al. (2003). The AH109 reporter strain used in the assays was maintained on YPD plates containing 30 μg/ml adenine hemisulfate (MP Biomedicals). The plates with medium lacking His, Leu, and Trp (labeled “–His” in the figures) and medium lacking only Leu and Trp (labeled “+His” in the figures) were prepared with dropout base agar (MP Biomedicals) using the corresponding supplement mixtures (MP Biomedicals and Takara) and supplemented with 30 μg/ml adenine hemisulfate. The –His medium was also supplemented with 1 mM 3-amino-1,2,4 triazole (3AT) (Sigma-Aldrich) after autoclaving and immediately preceding pouring of plates.

Yeast two-hybrid assays

Assays were performed following the guidelines in the Matchmaker two-hybrid system (Takara) as described by Mattera et al. (2003). The AH109 reporter strain used in the assays was maintained on YPD plates containing 30 μg/ml adenine hemisulfate (MP Biomedicals). The plates with medium lacking His, Leu, and Trp (labeled “–His” in the figures) and medium lacking only Leu and Trp (labeled “+His” in the figures) were prepared with dropout base agar (MP Biomedicals) using the corresponding supplement mixtures (MP Biomedicals and Takara) and supplemented with 30 μg/ml adenine hemisulfate. The –His medium was also supplemented with 1 mM 3-amino-1,2,4 triazole (3AT) (Sigma-Aldrich) after autoclaving and immediately preceding pouring of plates.

Generation of AAGAB-KO HeLa cellsKO clones targeting either exon 1 or exon 2 of the human AAGAB gene (NCBI Gene ID 79719) were generated in HeLa cells using CRISPR-Cas9 (Cong et al., 2013, Ran et al., 2013). We used 20-nucleotide long guide RNA (gRNA) antisense strand 5′ CTGGTCTCCTGAGAAGACGG or sense strand 5′ CCGTCTTCTCAGGAGACCAG to target exon 1 (nucleotides 44–63 in the cDNA sequence; the first gRNA is immediately adjacent to an NGG protospacer adjacent motif, whereas the second has a single nucleotide spacing) (Supplemental Figure S1A). We also targeted AAGAB gene exon 2 with gRNA antisense 5′ CAAAGTAAACCACAAATGCT (nucleotides 234–253 in the cDNA sequence; Supplemental Figure S1A). The targeting complementary oligonucleotides including CACCG and AAAC overhang sequences (forward and reverse oligonucleotides, respectively) were annealed and subcloned in BbsI-digested PX458 (pSpCas9(BB)-2A-GFP; Addgene plasmid cat. 48138). Constructs were verified by sequencing with primer 5′ GAGGGCCTATTTCCCATGATTC. HeLa cells were transfected with the targeting constructs using Lipofectamine 2000 (ThermoFisher). Twenty-four h after transfection, cells were trypsinized and GFP-positive cells were subsequently sorted into individual clones in 96-well plates at the Flow Cytometry Core, National Heart Lung and Blood Institute, NIH. Cell clones were subsequently split, plated on replica plates for maintenance and screening, and analyzed by immunoblotting using rabbit anti-AAGAB. Cell clones lacking expression of AAGAB were subsequently genotyped by sequencing of PCR fragments amplified from genomic DNA using forward primers 5′ GAAGGTCTGGGTGGGCATTT or 5′ CCTGGTCATTCCGAGAACGC and reverse primer 5′ TCCCGGAGCAGCAAGGATA when targeting exon 1, or forward primer 5′ GGAACAGAAGATCTTATTGTGGAAGTG and reverse primer 5′ CAGACACTCTATCGCAGACCA when targeting exon 2.

Generation of AAGAB-KO HeLa cells

KO clones targeting either exon 1 or exon 2 of the human AAGAB gene (NCBI Gene ID 79719) were generated in HeLa cells using CRISPR-Cas9 (Cong et al., 2013, Ran et al., 2013). We used 20-nucleotide long guide RNA (gRNA) antisense strand 5′ CTGGTCTCCTGAGAAGACGG or sense strand 5′ CCGTCTTCTCAGGAGACCAG to target exon 1 (nucleotides 44–63 in the cDNA sequence; the first gRNA is immediately adjacent to an NGG protospacer adjacent motif, whereas the second has a single nucleotide spacing) (Supplemental Figure S1A). We also targeted AAGAB gene exon 2 with gRNA antisense 5′ CAAAGTAAACCACAAATGCT (nucleotides 234–253 in the cDNA sequence; Supplemental Figure S1A). The targeting complementary oligonucleotides including CACCG and AAAC overhang sequences (forward and reverse oligonucleotides, respectively) were annealed and subcloned in BbsI-digested PX458 (pSpCas9(BB)-2A-GFP; Addgene plasmid cat. 48138). Constructs were verified by sequencing with primer 5′ GAGGGCCTATTTCCCATGATTC. HeLa cells were transfected with the targeting constructs using Lipofectamine 2000 (ThermoFisher). Twenty-four h after transfection, cells were trypsinized and GFP-positive cells were subsequently sorted into individual clones in 96-well plates at the Flow Cytometry Core, National Heart Lung and Blood Institute, NIH. Cell clones were subsequently split, plated on replica plates for maintenance and screening, and analyzed by immunoblotting using rabbit anti-AAGAB. Cell clones lacking expression of AAGAB were subsequently genotyped by sequencing of PCR fragments amplified from genomic DNA using forward primers 5′ GAAGGTCTGGGTGGGCATTT or 5′ CCTGGTCATTCCGAGAACGC and reverse primer 5′ TCCCGGAGCAGCAAGGATA when targeting exon 1, or forward primer 5′ GGAACAGAAGATCTTATTGTGGAAGTG and reverse primer 5′ CAGACACTCTATCGCAGACCA when targeting exon 2.

Cell culture, transfection, immunoprecipitation, and immunoblottingHeLa cells were cultured on 100-mm dishes or multiwell plates in DMEM (Quality Biological) supplemented with 10% fetal bovine serum (Corning), 100 units/ml penicillin, and 100 μg/ml streptomycin (Corning) at 37°C under a humidified atmosphere (95:5 air:CO2). Cells plated on 100-mm dishes or 6-well plates were transfected with 7 μg or 1.4 μg of plasmid DNA, respectively, using the X-tremeGENE9 DNA transfection reagent (Roche) as described previously (Mattera et al., 2020b). Cells were washed twice with PBS and lysed in 50 mM Tris-HCl pH 7.4, 0.8% (vol/vol) Triton X-100 and 75 mM NaCl supplemented with a protease inhibitor cocktail (EDTA-free Complete, Roche). Lysates from cells cultured on 100-mm dishes were subjected to immunoprecipitation using 2 μg of antibodies immobilized onto 25 μl of Protein G-Sepharose beads (GE Healthcare) as described previously (Mattera et al., 2003). Lysates or immunoprecipitated complexes were subjected to SDS–PAGE in 10% acrylamide gels and transferred onto nylon membranes (Immobilon-P, Millipore). Membranes were subsequently stained with Ponceau S (Sigma Aldrich) and immunoblotted with primary and secondary antibodies using buffers containing 0.05% Tween (Sigma Aldrich) and 3% dry milk (BioRad). Blots were developed with Western Lighting Plus-ECL (PerkinElmer) or SuperSignal West Pico Plus (ThermoFisher) reagents.

Cell culture, transfection, immunoprecipitation, and immunoblotting

HeLa cells were cultured on 100-mm dishes or multiwell plates in DMEM (Quality Biological) supplemented with 10% fetal bovine serum (Corning), 100 units/ml penicillin, and 100 μg/ml streptomycin (Corning) at 37°C under a humidified atmosphere (95:5 air:CO2). Cells plated on 100-mm dishes or 6-well plates were transfected with 7 μg or 1.4 μg of plasmid DNA, respectively, using the X-tremeGENE9 DNA transfection reagent (Roche) as described previously (Mattera et al., 2020b). Cells were washed twice with PBS and lysed in 50 mM Tris-HCl pH 7.4, 0.8% (vol/vol) Triton X-100 and 75 mM NaCl supplemented with a protease inhibitor cocktail (EDTA-free Complete, Roche). Lysates from cells cultured on 100-mm dishes were subjected to immunoprecipitation using 2 μg of antibodies immobilized onto 25 μl of Protein G-Sepharose beads (GE Healthcare) as described previously (Mattera et al., 2003). Lysates or immunoprecipitated complexes were subjected to SDS–PAGE in 10% acrylamide gels and transferred onto nylon membranes (Immobilon-P, Millipore). Membranes were subsequently stained with Ponceau S (Sigma Aldrich) and immunoblotted with primary and secondary antibodies using buffers containing 0.05% Tween (Sigma Aldrich) and 3% dry milk (BioRad). Blots were developed with Western Lighting Plus-ECL (PerkinElmer) or SuperSignal West Pico Plus (ThermoFisher) reagents.

Cycloheximide chaseWT and AAGAB-KO cells were plated on 6-well plates and cultured to approximately 80% confluency. Cells were subsequently incubated with 0.1 mg/ml cycloheximide (CHX; Sigma-Aldrich) added to regular culture medium for 0–20 h. At the indicated times, cells were washed twice with 2 ml of PBS and lysed in 250 μl of 15 mM HEPES pH 7.4, 0.5% (vol/vol) Triton X-100,and 150 mM NaCl supplemented with a protease inhibitor cocktail (EDTA-free Complete, Roche). Lysates were incubated for 30 min at 4°C with end-over-end rotation, followed by centrifugation for 15 min at 21,000 × g and 4°C. Supernatants were subsequently supplemented with 1X Laemmli buffer and subjected to SDS–PAGE and immunoblotting as described in the previous section. In some experiments, cells with or without 0.1 mg/ml CHX were incubated for 4.5–6.5 h in the presence or absence of 30 μM MG132 (Sigma-Aldrich). At the end of this period, cells were processed as described above.

Cycloheximide chase

WT and AAGAB-KO cells were plated on 6-well plates and cultured to approximately 80% confluency. Cells were subsequently incubated with 0.1 mg/ml cycloheximide (CHX; Sigma-Aldrich) added to regular culture medium for 0–20 h. At the indicated times, cells were washed twice with 2 ml of PBS and lysed in 250 μl of 15 mM HEPES pH 7.4, 0.5% (vol/vol) Triton X-100,and 150 mM NaCl supplemented with a protease inhibitor cocktail (EDTA-free Complete, Roche). Lysates were incubated for 30 min at 4°C with end-over-end rotation, followed by centrifugation for 15 min at 21,000 × g and 4°C. Supernatants were subsequently supplemented with 1X Laemmli buffer and subjected to SDS–PAGE and immunoblotting as described in the previous section. In some experiments, cells with or without 0.1 mg/ml CHX were incubated for 4.5–6.5 h in the presence or absence of 30 μM MG132 (Sigma-Aldrich). At the end of this period, cells were processed as described above.

Immunofluorescence microscopyCells were fixed for 10 min with methanol at –20°C or 15 min with 4% paraformaldehyde at room temperature and incubated with primary (45–60 min at room temperature) and secondary (30–45 min at room temperature) antibodies diluted in PBS supplemented with 0.1% saponin, 0.1% bovine serum albumin, and 0.02% sodium azide as described in the figure legends. DAPI (ThermoFisher) at 300 nM, and GFP-Booster Atto488 (Chromotek) at 1:400 dilution were added during incubation with secondary antibodies. Relative changes in AP-4 ε staining and ATG9A distribution in AAGAB-KO cells, as well as rescue following transfection with AAGAB-GFP, were assessed by manual scoring of cells using a Zeiss Axio Imager.A1 fluorescence microscope fitted with a Plan Apochromat 63×/1.4 Oil DIC M27 objective. Confocal microscopy images were obtained using a Zeiss LSM 780 microscope with a 63×, 1.4 NA Plan Apochromat 63× objective.

Immunofluorescence microscopy

Cells were fixed for 10 min with methanol at –20°C or 15 min with 4% paraformaldehyde at room temperature and incubated with primary (45–60 min at room temperature) and secondary (30–45 min at room temperature) antibodies diluted in PBS supplemented with 0.1% saponin, 0.1% bovine serum albumin, and 0.02% sodium azide as described in the figure legends. DAPI (ThermoFisher) at 300 nM, and GFP-Booster Atto488 (Chromotek) at 1:400 dilution were added during incubation with secondary antibodies. Relative changes in AP-4 ε staining and ATG9A distribution in AAGAB-KO cells, as well as rescue following transfection with AAGAB-GFP, were assessed by manual scoring of cells using a Zeiss Axio Imager.A1 fluorescence microscope fitted with a Plan Apochromat 63×/1.4 Oil DIC M27 objective. Confocal microscopy images were obtained using a Zeiss LSM 780 microscope with a 63×, 1.4 NA Plan Apochromat 63× objective.

Supplementary MaterialClick here for additional data file.

Supplementary Material

Click here for additional data file.
