---
pmid: '20388662'
title: Cytosol-localized heat shock factor-binding protein, AtHSBP, functions as a
  negative regulator of heat shock response by translocation to the nucleus and is
  required for seed development in Arabidopsis.
authors:
- Hsu SF
- Lai HC
- Jinn TL
journal: Plant Physiol
year: '2010'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC2879799
doi: 10.1104/pp.109.151225
---

# Cytosol-localized heat shock factor-binding protein, AtHSBP, functions as a negative regulator of heat shock response by translocation to the nucleus and is required for seed development in Arabidopsis.
**Authors:** Hsu SF, Lai HC, Jinn TL
**Journal:** Plant Physiol (2010)
**DOI:** [10.1104/pp.109.151225](https://doi.org/10.1104/pp.109.151225)
**PMC:** [PMC2879799](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2879799/)

## Abstract

1. Plant Physiol. 2010 Jun;153(2):773-84. doi: 10.1104/pp.109.151225. Epub 2010
Apr  13.

Cytosol-localized heat shock factor-binding protein, AtHSBP, functions as a 
negative regulator of heat shock response by translocation to the nucleus and is 
required for seed development in Arabidopsis.

Hsu SF(1), Lai HC, Jinn TL.

Author information:
(1)Institute of Plant Biology and Department of Life Science, National Taiwan 
University, Taipei 10617, Taiwan.

Heat shock response (HSR) is a universal mechanism in all organisms. It is under 
tight regulation by heat shock factors (HSFs) and heat shock proteins (HSPs) 
after heat shock (HS) to prevent stress damage. On the attenuation of HSR, HSP70 
and HSF Binding Protein1 (HSBP1) interact with HSF1 and thus dissociate trimeric 
HSF1 into an inert monomeric form in humans. However, little is known about the 
effect of HSBP with thermal stress in plants. This report describes our 
investigation of the role of AtHSBP in Arabidopsis (Arabidopsis thaliana) by 
genetic and molecular approaches. AtHSBP was heat inducible and ubiquitously 
expressed in all tissues; AtHSBP was also crucial for seed development, as 
demonstrated by AtHSBP-knockout lines showing seed abortion. Thermotolerance 
results showed that AtHSBP participates in acquired thermotolerance but not 
basal thermotolerance and is a negative regulator of HSR. Subcellular 
localization revealed that the cytosol-localized AtHSBP translocated to the 
nucleus in response to HS. Protoplast two-hybrid assay results confirmed that 
AtHSBP interacts with itself and with the HSFs, AtHSFA1a, AtHSFA1b, and AtHSFA2. 
AtHSBP also negatively affected AtHSFA1b DNA-binding capacity in vitro. 
Quantitative polymerase chain reaction and western-blot analysis demonstrated 
that altered levels of AtHSBP lead to differential HSP expression, mainly during 
the recovery from HS. These studies provide a new insight into HSBP in plants 
and reveal that AtHSBP is a negative regulator of HSR and required for seed 
development.

DOI: 10.1104/pp.109.151225
PMCID: PMC2879799
PMID: 20388662 [Indexed for MEDLINE]

## Full Text

Abstract

Heat shock response (HSR) is a universal mechanism in all organisms. It is under tight regulation by heat shock factors (HSFs) and heat shock proteins (HSPs) after heat shock (HS) to prevent stress damage. On the attenuation of HSR, HSP70 and HSF Binding Protein1 (HSBP1) interact with HSF1 and thus dissociate trimeric HSF1 into an inert monomeric form in humans. However, little is known about the effect of HSBP with thermal stress in plants. This report describes our investigation of the role of AtHSBP in Arabidopsis ( Arabidopsis thaliana ) by genetic and molecular approaches. AtHSBP was heat inducible and ubiquitously expressed in all tissues; AtHSBP was also crucial for seed development, as demonstrated by AtHSBP -knockout lines showing seed abortion. Thermotolerance results showed that AtHSBP participates in acquired thermotolerance but not basal thermotolerance and is a negative regulator of HSR. Subcellular localization revealed that the cytosol-localized AtHSBP translocated to the nucleus in response to HS. Protoplast two-hybrid assay results confirmed that AtHSBP interacts with itself and with the HSFs, AtHSFA1a, AtHSFA1b, and AtHSFA2. AtHSBP also negatively affected AtHSFA1b DNA-binding capacity in vitro. Quantitative polymerase chain reaction and western-blot analysis demonstrated that altered levels of AtHSBP lead to differential HSP expression, mainly during the recovery from HS. These studies provide a new insight into HSBP in plants and reveal that AtHSBP is a negative regulator of HSR and required for seed development.

DISCUSSION

In this report, we characterized AtHSBP in Arabidopsis in response to HS and showed that it is heat inducible, ubiquitously expressed, required for seed development, and acts as a negative regulator for the attenuation of HSR. Subcellular localization assay revealed that AtHSBP is a predominantly cytosol-localized protein and that its translocation to the nucleus depends on thermal stress. We also demonstrated that AtHSBP is sufficient to interact with AtHSFs to reduce HSF transactivation and the protein levels of HSP genes.

AtHSBP contains a single α -helix (residues 15–59) with two potential coiled-coil regions (residues 20–34 and 41–65) that show high similarity across all HSBP homologs ( Fig. 1 ; Supplemental Table S1). Particularly, nearly all the heptad-repeat residues in the trimerization region, AtHSBP residues 19 to 53 corresponding to HsHSBP1 residues 15 to 49 ( Liu et al., 2009 ), are almost conserved ( Fig. 1 ). The GST-AtHSBP 39-86 , containing a potential coiled-coil region, showed decreased AtHSFA1b DNA-binding activity in vitro ( Fig. 7B ). This C-terminal coiled-coil domain could contribute to the interaction with HSFs. The heptad-repeat regions form coiled coils, as long as similarly sized hydrophobic residues are located at the a and d positions to ensure packing of the interhelical space, even if the overall sequence diverges. Likewise, AtHSBP could be cross-interacted with itself ( Fig. 6 ), which suggested that the HSBP helical region may function to self-associate and may interact with proteins other than the trimerization domain of HSFs in the cell.

HSBPs exist as two isoforms in monocots but only one copy in dicots. Maize EMP2 required for shoot development has been demonstrated by Fu and Scanlon (2004) ; embryogenesis in emp2 mutant kernels is severely retarded at the time, 12 d after pollination, well before maize kernels become competent to invoke the HSR ( Fu et al., 2002 ). These results suggested that EMP2 performs an important developmental function in embryogenesis; in addition, it serves as a positive regulator of HSR attenuation in seeds. We found that the expression of AtHSBP in flowers and siliques was higher than that in other tissues ( Fig. 3B ); meanwhile, AtHSBP -knockout lines also showed seed abortion ( Fig. 3, C and D ), which suggested that the defective AtHSBP may lead unattenuated HSR to trigger embryo abortion, similar to emp2 mutant kernels ( Fu et al., 2002 ). However, which embryonic stage was affected to result in seed abortion of AtHSBP -knockout lines remains unclear. Besides, the expression of heat-inducible AtHSBP ( Fig. 3A ) was similar to that of ZmHSBP2 under thermal stress ( Fu and Scanlon, 2004 ), and the roles of EMP2 and ZmHSBP2 in HSR were suggested to be nonredundant ( Fu et al., 2006 ). These findings may support that one AtHSBP confers multiple functions required for embryo viability and regulation of the HSR.

HsHSBP1 has been demonstrated to be a nucleus-localized protein, regardless of HS ( Satyal et al., 1998 ). EMP2 mainly localizes within the nucleus, and the signals are also detected in the cytoplasm of embryonic cells; nevertheless, the localization and function of ZmHSBP2 are still unclear ( Fu and Scanlon, 2004 ). We found AtHSBP-GFP localized in the cytoplasm, a finding different from previous reports for maize and human ( Satyal et al., 1998 ; Fu and Scanlon, 2004 ). Although AtHSBP was detected in the nucleus during a 1-h recovery from HS, the localization of the AtHSBP-S35A mutant and GUS-GFP control were unaffected by thermal stress ( Fig. 5 ; Supplemental Fig. S2). These results demonstrate that the translocation of AtHSBP-GFP indeed depends on thermal stress. However, the nucleus-localized AtHSBP signals were undetectable during a 2-h recovery from HS ( Fig. 5A ). The cytoplasm showed highly expressed AtHSBP-GFP, so judging whether AtHSBP-GFP was relocated to the cytoplasm or decreased in level by a degradation process during a 2-h recovery from HS was difficult. Above all, the nuclear localization of AtHSBP occurred during the recovery from HS, which was associated with the attenuation of HSR.

As compared with AtHSBP-ox1 , AtHSBP-ox2 showed approximately 2-fold higher AtHSBP protein accumulation ( Fig. 2B ), but the seedling survival rate in response to HS showed no statistically significant differences between them ( Table II ). The highly cytosol-expressed AtHSBP-GFP showed partial but not complete translocation to the nucleus during the HSR. The nucleus-localized signal of AtHSBP-GFP expressed in Athsbp-1 protoplasts also showed a similar intensity, as shown in Figure 5A during HSR (Supplemental Fig. S3, H1R1). Studies of HsHSBP1 structure demonstrate that HsHSBP1 has buried its polar side chain in the hydrophobic interior of the helix bundle; however, an exception, residue Ser-31, occurs abnormally at the a position of the heptad repeat, which implies that the residue Ser-31 may have important functions ( Liu et al., 2009 ). We found the localization of AtHSBP-S35A mutant to be unaffected by HS ( Fig. 5B ), which suggested that AtHSBP Ser-35 is important for its function. These findings may imply that (1) a limited amount of the nucleus-localized AtHSBP is required for its function, and (2) the translocation of AtHSBP (without nuclear localization signal) may be mediated through HS-related factors that function during the attenuation of HSR. The S35A mutation may result in a loss of interacting ability, which is required for its nuclear localization, but additional studies are necessary to address this suggestion.

In animals, current research suggests that the inactive HSF1 monomer is in the cytoplasm, interacting with HSP70 chaperone in the control state. Following the activation of HSR, the nucleus-localized and activated HSF1 trimer acquires DNA-binding capacity and induces HSP expression. On the recovery of HS, HSBP1 and HSP70 bind directly to HSF1, which HSF1 dissociates into monomers and then returns to the cytoplasm (for review, see Wu, 1995 ; Morimoto, 1998 ; Satyal et al., 1998 ; Pirkkala et al., 2001 ). We confirmed that AtHSBP interacts with AtHSFs ( Fig. 6 ) and decreases the HSF DNA-binding ability ( Fig. 7B ), and these findings were similar to HsHSBP1 functions in humans ( Satyal et al., 1998 ). In plants, interactions of AtHSP70s and AtHSFA1a are demonstrated by yeast two-hybrid assay ( Kim and Schöffl, 2002 ). HSP70 antisense assay reveals the requirement of an extended time for inactivation of HSF transcriptional activity, a longer shutoff time, which suggests that HSP70 family proteins may mediate HSF transcriptional activity ( Lee and Schöffl, 1996 ). Although the defective AtHSBP led to unattenuated HSP expression during the recovery from HS ( Fig. 8A ), similar to the HSP70 antisense lines, whether AtHSBP coordinates with HSP70 in the attenuation of HSR in Arabidopsis remains unclear.

Modified protoplast two-hybrid assay investigates interacting proteins invoked in transcriptional regulation. HsHSBP1 is suggested to directly bind to the HSF1 trimerization domain for inactivation of HSF1 activity ( Satyal et al., 1998 ). We found that AtHSBP significantly interacted with AtHSFA1a, AtHSFA1b, and AtHSFA2, although the interaction was weaker than that of positive controls ( Fig. 6 ). We confirmed that AtHSBP decreased AtHSFA1b DNA-binding capacity in vitro ( Fig. 7B ); therefore, the interaction of AtHSBP and AtHSFs may be weak and transient in vivo. Furthermore, a high protein molar ratio of AtHSBP to AtHSFA1b (approximately 100-fold GST-AtHSBP higher than MBP-AtHSFA1b) did not totally reduce AtHSFA1b DNA-binding capacity in vitro, and these results also could not accurately reflect the AtHSBP activity in both the transcript and protein levels relative to AtHSFs in vivo, which suggested that AtHSBP may require other factors, such as HSP70, as reported in animals, to coordinate its activity.

In summary, we propose that the cytosol-localized AtHSBP translocates to the nucleus during recovery from HS ( Fig. 5A , H1R1; localization data), where it negatively affects AtHSF DNA-binding capacity ( Fig. 7B ; EMSA data) and decreases the programmed HSP expression ( Fig. 8 ; western-blot analysis) to attenuate HSR. In concert, AtHSBP expression was slightly up-regulated by HS (approximately 1.5-fold higher) and peaked during the 1-h recovery period (approximately 3-fold higher); the expression then returned to the basal level after a 4-h recovery from HS ( Fig. 3A ), which was in agreement with the AtHSBP behavior during the recovery from HS.

Numerous questions remain regarding the roles of the molecular chaperones HSP, HSF, and HSBP in regulating HSR. All of the HSF-interacting proteins reported to date act as negative regulators for HSF transcriptional activity (for review, see Morimoto, 1998 ; Pirkkala et al., 2001 ). The consequence of forming a molecular complex requires that HSF be kept in a state that can be readily activated, but how and when the proteins interact with HSF remain to be answered. Further study could evaluate any positive regulators of HSF or whether the fast activation mechanism of HSF is an intrinsic property that has been evolutionarily conserved to allow for the rapid activation of HSF with stress stimuli.
