---
pmid: '24947832'
title: Differential protein-protein interactions of LRRK1 and LRRK2 indicate roles
  in distinct cellular signaling pathways.
authors:
- Reyniers L
- Del Giudice MG
- Civiero L
- Belluzzi E
- Lobbestael E
- Beilina A
- Arrigoni G
- Derua R
- Waelkens E
- Li Y
- Crosio C
- Iaccarino C
- Cookson MR
- Baekelandt V
- Greggio E
- Taymans JM
journal: J Neurochem
year: '2014'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC4272680
doi: 10.1111/jnc.12798
---

# Differential protein-protein interactions of LRRK1 and LRRK2 indicate roles in distinct cellular signaling pathways.
**Authors:** Reyniers L, Del Giudice MG, Civiero L, Belluzzi E, Lobbestael E, Beilina A, Arrigoni G, Derua R, Waelkens E, Li Y, Crosio C, Iaccarino C, Cookson MR, Baekelandt V, Greggio E, Taymans JM
**Journal:** J Neurochem (2014)
**DOI:** [10.1111/jnc.12798](https://doi.org/10.1111/jnc.12798)
**PMC:** [PMC4272680](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4272680/)

## Abstract

1. J Neurochem. 2014 Oct;131(2):239-50. doi: 10.1111/jnc.12798. Epub 2014 Jul 14.

Differential protein-protein interactions of LRRK1 and LRRK2 indicate roles in 
distinct cellular signaling pathways.

Reyniers L(1), Del Giudice MG(2), Civiero L(3), Belluzzi E(3), Lobbestael E(1), 
Beilina A(4), Arrigoni G(5)(6), Derua R(7), Waelkens E(7), Li Y(8), Crosio C(2), 
Iaccarino C(2), Cookson MR(4), Baekelandt V(1), Greggio E(3), Taymans JM(1).

Author information:
(1)Laboratory for Neurobiology and Gene Therapy, Department of Neurosciences, KU 
Leuven, Leuven, Belgium.
(2)Department of Biomedical Sciences, University of Sassari, Sassari, Italy.
(3)Department of Biology, University of Padova, Padova, Italy.
(4)Laboratory of Neurogenetics, National Institute on Aging, NIH, Bethesda, 
Maryland, USA.
(5)Department of Biomedical Sciences, University of Padova, Padova, Italy.
(6)Proteomics Center of Padova University, Padova, Italy.
(7)Department of Cellular and Molecular Medicine, KU Leuven, Laboratory of 
protein phosphorylation and proteomics, Leuven, Belgium.
(8)Protein/Peptide Sequencing Facility, National Institute of Neurological 
Disorders and Stroke, National Institutes of Health, Bethesda, Maryland, USA.

Genetic studies show that LRRK2, and not its closest paralogue LRRK1, is linked 
to Parkinson's disease. To gain insight into the molecular and cellular basis of 
this discrepancy, we searched for LRRK1- and LRRK2-specific cellular processes 
by identifying their distinct interacting proteins. A protein microarray-based 
interaction screen was performed with recombinant 3xFlag-LRRK1 and 3xFlag-LRRK2 
and, in parallel, co-immunoprecipitation followed by mass spectrometry was 
performed from SH-SY5Y neuroblastoma cell lines stably expressing 3xFlag-LRRK1 
or 3xFlag-LRRK2. We identified a set of LRRK1- and LRRK2-specific as well as 
common interactors. One of our most prominent findings was that both screens 
pointed to epidermal growth factor receptor (EGF-R) as a LRRK1-specific 
interactor, while 14-3-3 proteins were LRRK2-specific. This is consistent with 
phosphosite mapping of LRRK1, revealing phosphosites outside of 14-3-3 consensus 
binding motifs. To assess the functional relevance of these interactions, 
SH-SY5Y-LRRK1 and -LRRK2 cell lines were treated with LRRK2 kinase inhibitors 
that disrupt 14-3-3 binding, or with EGF, an EGF-R agonist. Redistribution of 
LRRK2, not LRRK1, from diffuse cytoplasmic to filamentous aggregates was 
observed after inhibitor treatment. Similarly, EGF induced translocation of 
LRRK1, but not of LRRK2, to endosomes. Our study confirms that LRRK1 and LRRK2 
can carry out distinct functions by interacting with different cellular 
proteins. LRRK1 and LRRK2 (leucine-rich repeat kinase) interaction partners were 
identified by two different protein-protein interaction screens. These confirmed 
epidermal growth factor receptor (EGR-R) as a LRRK1-specific interactor, while 
14-3-3 proteins were LRRK2-specific. Functional analysis of these interactions 
and the pathways they mediate shows that LRRK1 and LRRK2 signaling do not 
intersect, reflective of the differential role of both LRRKs in Parkinson's 
disease.

© 2014 International Society for Neurochemistry.

DOI: 10.1111/jnc.12798
PMCID: PMC4272680
PMID: 24947832 [Indexed for MEDLINE]

Conflict of interest statement: The authors declare no conflicts of interest.

## Full Text

Abstract

Genetic studies show that LRRK2, and not its closest paralogue LRRK1, is linked to Parkinson’s disease. To gain insight into the molecular and cellular basis of this discrepancy, we searched for LRRK1- and LRRK2-specific cellular processes by identifying their distinct interacting proteins. A protein microarray-based interaction screen was performed with recombinant 3xFlag-LRRK1 and 3xFlag-LRRK2 and, in parallel, co-immunoprecipitation followed by mass spectrometry was performed from SH-SY5Y neuroblastoma cell lines stably expressing 3xFlag-LRRK1 or 3xFlag-LRRK2. We identified a set of LRRK1- and LRRK2-specific as well as common interactors. One of our most prominent findings was that both screens pointed to epidermal growth factor receptor (EGF-R) as a LRRK1-specific interactor, while 14-3-3 proteins were LRRK2-specific. This is consistent with phosphosite mapping of LRRK1, revealing phosphosites outside of 14-3-3 consensus binding motifs. To assess the functional relevance of these interactions, SH-SY5Y-LRRK1 and -LRRK2 cell lines were treated with LRRK2 kinase inhibitors that disrupt 14-3-3 binding, or with EGF, an EGF-R agonist. Redistribution of LRRK2, not LRRK1, from diffuse cytoplasmic to filamentous aggregates was observed after inhibitor treatment. Similarly, EGF induced translocation of LRRK1, but not of LRRK2, to endosomes. Our study confirms that LRRK1 and LRRK2 can carry out distinct functions by interacting with different cellular proteins.

Introduction

Since the identification of mutations in leucine-rich repeat kinase 2 (LRRK2) linked to familial forms of Parkinson’s disease (PD) ( Paisan-Ruiz et al., 2004 ; Zimprich et al., 2004 ), LRRK2 has been shown to be an important contributor to PD pathogenesis. Mutations in LRRK2 are a common cause of familial forms of PD where the phenotype is clinically similar to that of sporadic PD ( Haugarvoll et al., 2008 ; Healy et al., 2008 ). In addition, genetic variation at the LRRK2 locus is also associated with sporadic PD ( Satake et al., 2009 ; Simón-Sánchez et al., 2009 ; Ross et al., 2011 ). However, the precise cellular processes regulated by LRRK2 and the exact nature of its dysfunction in the etiology of disease is incompletely understood.

LRRK2 is a member of the ROCO family of multidomain proteins which are characterized by the presence of a Ras of complex (ROC) protein domain and an adjacent C-terminal of ROC domain (COR) ( Lewis, 2009 ). LRRK2’s closest paralogue is LRRK1, which displays a similar domain organization: besides the signature ROC-COR domains both proteins also encode a serine-threonine kinase domain C-terminal of ROC-COR, and N-terminal leucine-rich and ankyrin-like repeats ( Marín, 2006 ; 2008 ; Civiero et al., 2012 ). Differences between the two proteins are that LRRK2 additionally displays an N-terminal segment encoding armadillo-like repeats, while the WD40 domain present in the LRRK2 C-terminus is absent in the equivalent section of LRRK1 ( Marín, 2006 ; 2008 ; Civiero et al., 2012 ) ( Figure S1B ). There is no genetic support for the involvement of LRRK1 in the pathogenesis of PD, suggesting that LRRK2 has unique properties that are relevant to the disease process ( Haugarvoll et al., 2007 ; Taylor et al., 2007 ). We have previously shown that the detrimental effects of mutations in LRRK2 are not reproduced by introducing equivalent mutations in LRRK1 ( Greggio et al., 2007 ). Collectively, these observations suggest that although the two proteins have a similar modular organization, LRRK1 and LRRK2 have distinct properties which may be important for disease pathogenesis.

However, LRRK1 has also been shown to interact with LRRK2 ( Paisan-Ruiz et al., 2004 ; Zimprich et al., 2004 ; Klein et al., 2009 ; Dächsel et al., 2010 ). This interaction, and the high degree of homology between the two proteins, indicates that LRRK1 may be involved in the LRRK2 signaling network. In order to better understand the relationship between cellular signaling functions of LRRK1 and LRRK2, we have screened for protein-protein interaction partners of LRRK1 and LRRK2. These screens pointed to a LRRK1-specific interaction with EGF-R and a LRRK2-specific interaction with 14-3-3 proteins, confirming previous reports ( Haugarvoll et al., 2008 ; Healy et al., 2008 ; Nichols et al., 2010 ; Hanafusa et al., 2011 ). In this study, we further characterize these two interactions, in particular the crosstalk between the two pathways. We show that these interactions are specific for each LRRK and that they mediate specific, independently occurring cellular translocation processes for each LRRK. In turn, these results point to segregated functional complexes in the LRRK1 and LRRK2 signaling networks in mammalian cells.

Discussion

In this study, we compared the cellular functions of LRRK1 and LRRK2 by identifying their interaction partners using two different proteomic screening approaches, protein microarray and AP-MS ( Figure 1 ). Given that the nature of the analyzed complexes is different in the two approaches, the protein microarray interactions are assessed in vitro while AP-MS complexes are isolated from living cells, the combination of both approaches in parallel allows us to obtain a more complete picture of the proteins’ interactomes.

It should be noted though that isolated complexes are strongly dependent on the cell type or tissue that is used and this should also be taken into account for the interpretation of the results. Also, as for any screening methodology, results will include true hits as well as a certain proportion of false-positives and false-negatives. Therefore, results obtained with these screening experiments should be carefully interpreted and thoroughly validated. As a first validation step, we overexpressed LRRK1 or LRRK2 in HEK293T cell lines and tested their interaction (either on endogenous protein or after overexpression depending on the endogenous protein levels of the interactor tested) by CoIP. Our results confirm the LRRK1:EGF-R, LRRK2-14-3-3 interactions which were specific for each LRRK and Hsc70, BAG5 and Hsp90 as common interactors. Since the previously reported interactions between LRRK1:EGF-R and LRRK2:14-3-3 ( Dzamko et al., 2010 ; Nichols et al., 2010 ; Titz et al., 2010 ; Hanafusa et al., 2011 ; Li et al., 2011 ) were the only ones identified with the two screening methodologies, we decided to focus on these two and study the potential crosstalk between LRRK1 and LRRK2 for these interactions.

Previous studies have reported that the phosphorylation status of LRRK2 is an important regulatory mechanism. LRRK2 is basally phosphorylated in cells ( Greggio et al., 2007 ; Ito et al., 2007 ; Berger et al., 2010 ; Taymans et al., 2011 ; 2013 ) with several phosphosites clustering in a region between the ankyrin and leucine-rich repeat domains (S910, S935, S955 and S973) ( Gloeckner et al., 2010 ; Nichols et al., 2010 ). These sites have generated much interest as their phosphorylation is reduced for several disease mutant forms of LRRK2 ( Dzamko et al., 2010 ; Nichols et al., 2010 ; Li et al., 2011 ). Since the LRRK2:14-3-3 interaction is dependent on phosphorylation of LRRK2 at S910 and S935 ( Nichols et al., 2010 ) we mapped LRRK1 and LRRK2 phosphosites in parallel and found that these map to distinct regions of the respective proteins: i.e. in the ANK, COR or kinase domains for LRRK1 and in the region linking the ankyrin repeat domain with the leucine-rich repeat domain for LRRK2 ( Figure S1B ). In particular, LRRK1 is not phosphorylated in the region between the ANK and LRR domains that mediates 14-3-3 binding in LRRK2, consistent with the lack of LRRK1 binding to 14-3-3. A further indication that LRRK1 and LRRK2 phosphosites are differentially regulated is their sensitivity to the LRRK2 IN-1 inhibitor ( Deng et al., 2011 ). Treatment of cells in vitro with LRRK2-IN1 results in dephosphorylation of LRRK2 at the 14-3-3 binding sites of S910 and S935 ( Dzamko et al., 2010 ; Deng et al., 2011 ). Here we show via metabolic labeling that LRRK1 phosphorylation is not significantly influenced by LRRK2-IN1 treatment, while LRRK2 was dephosphorylated under the same conditions ( Figure 3 ). These observed differences between LRRK1 and LRRK2 suggest low LRRK1 off-target risks of LRRK2 inhibition. Inhibition of the kinase function of LRRK2 is considered to have therapeutic potential (reviewed in ( Greggio and Singleton, 2007 ; Vancraenenbroeck et al., 2011 )), and the fact that dephosphorylation following LRRK2 kinase inhibition occurs for LRRK2 but not LRRK1 might suggest that targeting LRRK2 by kinase inhibitors does not interfere with the normal function of LRRK1.

Formation of both LRRK1:EGF-R and LRRK2:14-3-3 complexes result in LRRK cellular translocation: LRRK1 traffics together with EGF-R to endosomes upon stimulation of the receptor with EGF ( Hanafusa et al., 2011 ) while LRRK2 translocates to filamentous structures upon inhibitor-induced dephosphorylation and loss of 14-3-3 binding ( Dzamko et al., 2010 ; Deng et al., 2011 ; Lobbestael et al., 2013 ). We exploited these cellular translocation properties to assess potential crosstalk between LRRK1 and LRRK2 cellular functioning. We first tested for overlap in the LRRK1 and LRRK2 translocation phenotypes after treatment with LRRK2-IN1 or EGF. We saw that only LRRK2, and not LRRK1, translocated after LRRK2-IN1 treatment. Similarly, treatment with EGF did not affect the localization of LRRK2 while it was sufficient to translocate LRRK1 to endosomes. Furthermore, to exclude that the partner LRRK co-regulates the translocation phenotype or that one LRRK may functionally take over the role of the partnering LRRK upon its depletion from the cell, we tested LRRK1 and LRRK2 translocation after silencing of the partnering LRRK by RNAi. Here also, LRRK1 and LRRK2 translocation patterns remained unchanged, allowing us to conclude that the LRRK1:EGF-R and LRRK2:14-3-3 interactions mediate distinct cellular processes for each LRRK.

Despite the absence of crosstalk between LRRK1 and LRRK2 for these interactions, our proteomic screenings revealed also a number of shared interactors ( Figure 1C , Tables 1C and 1F). Out of these, we found several chaperone proteins such as Hsc70, Hsp90 (identified via AP-MS) and BAG proteins (identified using protein microarray) suggesting that the mechanisms regulating folding and stability of LRRK1 and LRRK2 are similar. Furthermore, several studies have shown the importance of proper folding for ROCO proteins, which engage in intramolecular interactions with domains with distinct folds ( Greggio et al., 2008 ; Marín et al., 2008 ; Daniëls et al., 2011 ). Hsp90 is reported to assist up to 10% of all cellular proteins and is heavily involved in conformational regulation of signaling proteins, including kinases such as Raf1, Akt, cyclin dependent kinase 4 (Cdk4) ( Pearl and Prodromou, 2006 ; Li et al., 2012 ) and LRRK2 ( Wang et al., 2008 ).

Other shared interactors include proteins related to cytoskeleton such as tubulins (AP/MS) or to vesicular physiology such as the cyclin G activated kinase (GAK), the secretory protein chromogranin B or neurosecretory protein VGF. Both LRRKs have already been functionally linked to vesicular physiology however so far these appear to be in different vesicular processes, such as receptor trafficking in endosomes for LRRK1 and autophagy or Golgi-related functions for LRRK2 ( Beilina et al., 2014 ). Therefore, although these data indicate that there is no link between LRRK2 and endosomal trafficking of EGFR, LRRK2 may be involved in other vesicular processes as it was shown to co-localize with synaptic vesicles in cell culture and animal models, interact with Rab5, an early endosomal marker, and functionally associate with the synaptic vesicle endocytosis protein Endophilin A ( Moore, 2008 ; Shin et al., 2008 ; Higashi et al., 2009 ; Dodson et al., 2012 ; Matta et al., 2012 ). We also show that LRRK2 does not respond to EGF treatment. However, LRRK2 is reported to play a role in mitogen-activated protein kinase (MAPK) signaling cascades ( Gloeckner et al., 2009 ; Hsu, Chan, Greggio, et al., 2010 ; Hsu, Chan, and Wolozin, 2010 ) which are downstream of growth factor receptors. Further work will be required to confirm these other interactions reported here and explore any functional links between LRRK1 and LRRK2 through these interactions.

In conclusion, our comparative cellular study shows that, although LRRK1 and LRRK2 have a number of common interactors, there is no evidence of signaling crosstalk between the LRRK2 proteins at the level of two LRRK-specific cellular interactions: LRRK1:EGF-R and LRRK2:14-3-3. Our work should stimulate further studies aiming at identifying similarities and differences in the cellular processes mediated by LRRK proteins, as these will provide further clues on the molecular mechanisms by which LRRK2 causes Parkinson’s disease and LRRK1 does not.
