---
pmid: '23436906'
title: Parallel visualization of multiple protein complexes in individual cells in
  tumor tissue.
authors:
- Leuchowius KJ
- Clausson CM
- Grannas K
- Erbilgin Y
- Botling J
- Zieba A
- Landegren U
- Söderberg O
journal: Mol Cell Proteomics
year: '2013'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC3675814
doi: 10.1074/mcp.O112.023374
---

# Parallel visualization of multiple protein complexes in individual cells in tumor tissue.
**Authors:** Leuchowius KJ, Clausson CM, Grannas K, Erbilgin Y, Botling J, Zieba A, Landegren U, Söderberg O
**Journal:** Mol Cell Proteomics (2013)
**DOI:** [10.1074/mcp.O112.023374](https://doi.org/10.1074/mcp.O112.023374)
**PMC:** [PMC3675814](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3675814/)

## Abstract

1. Mol Cell Proteomics. 2013 Jun;12(6):1563-71. doi: 10.1074/mcp.O112.023374.
Epub  2013 Feb 22.

Parallel visualization of multiple protein complexes in individual cells in 
tumor tissue.

Leuchowius KJ(1), Clausson CM, Grannas K, Erbilgin Y, Botling J, Zieba A, 
Landegren U, Söderberg O.

Author information:
(1)Department of Immunology, Genetics and Pathology, Science for Life 
Laboratory, Rudbeck Laboratory, University of Uppsala, Uppsala, Sweden. 
leuchowius@wehi.edu.au

Comment in
    Expert Rev Proteomics. 2013 Jun;10(3):219-21. doi: 10.1586/epr.13.22.

Cellular functions are regulated and executed by complex protein interaction 
networks. Accordingly, it is essential to understand the interplay between 
proteins in determining the activity status of signaling cascades. New methods 
are therefore required to provide information on different protein interaction 
events at the single cell level in heterogeneous cell populations such as in 
tissue sections. Here, we describe a multiplex proximity ligation assay for 
simultaneous visualization of multiple protein complexes in situ. The assay is 
an enhancement of the original proximity ligation assay, and it is based on 
using proximity probes labeled with unique tag sequences that can be used to 
read out which probes, from a pool of probes, have bound a certain protein 
complex. Using this approach, it is possible to gain information on the 
constituents of different protein complexes, the subcellular location of the 
complexes, and how the balance between different complex constituents can change 
between normal and malignant cells, for example. As a proof of concept, we used 
the assay to simultaneously visualize multiple protein complexes involving EGFR, 
HER2, and HER3 homo- and heterodimers on a single-cell level in breast cancer 
tissue sections. The ability to study several protein complex formations 
concurrently at single cell resolution could be of great potential for a systems 
understanding, paving the way for improved disease diagnostics and possibilities 
for drug development.

DOI: 10.1074/mcp.O112.023374
PMCID: PMC3675814
PMID: 23436906 [Indexed for MEDLINE]

## Full Text

Abstract

Cellular functions are regulated and executed by complex protein interaction networks. Accordingly, it is essential to understand the interplay between proteins in determining the activity status of signaling cascades. New methods are therefore required to provide information on different protein interaction events at the single cell level in heterogeneous cell populations such as in tissue sections. Here, we describe a multiplex proximity ligation assay for simultaneous visualization of multiple protein complexes in situ . The assay is an enhancement of the original proximity ligation assay, and it is based on using proximity probes labeled with unique tag sequences that can be used to read out which probes, from a pool of probes, have bound a certain protein complex. Using this approach, it is possible to gain information on the constituents of different protein complexes, the subcellular location of the complexes, and how the balance between different complex constituents can change between normal and malignant cells, for example. As a proof of concept, we used the assay to simultaneously visualize multiple protein complexes involving EGFR, HER2, and HER3 homo- and heterodimers on a single-cell level in breast cancer tissue sections. The ability to study several protein complex formations concurrently at single cell resolution could be of great potential for a systems understanding, paving the way for improved disease diagnostics and possibilities for drug development.

RESULTS

To enable multiplex combinatorial in situ PLA, we included a tag-specific sequence within the PLA probe. This design utilizes the same oligonucleotides as the version published previously of in situ PLA, with the added requirement of a tag oligonucleotide to create a circular ligation product. By introducing a specific tag for each PLA probe, the ligation product will contain information of the identity of the PLA probe ( Fig. 1 ). To test the new oligonucleotide design, we chose to detect proximity between HER family receptors, visualizing complexes of EGFR, HER2, and HER3. Antibodies against EGFR, HER2, and HER3 were conjugated with oligonucleotides to convert them into PLA probes. To verify the selectivity of the PLA probes, stably transfected PAE cells expressing different combinations of EGFR, HER2, and HER3 were investigated. The three PLA probes were used one at a time to detect the expression levels of all three proteins in the different cells. The binding of the PLA probes was in accordance with results obtained by Western blotting ( 18 ; supplemental Fig. 7 ), demonstrating selective binding of the probes ( Fig. 2 ). To determine how the probe concentrations influence the ratio of observed protein complexes, we then analyzed SK-BR-3 cells while varying the PLA probe concentrations ( Fig. 3 ). As expected, the amount of RCA products for each protein complex varied accordingly. We continued to investigate the reproducibility of the assay by analyzing the mean number of signals per cell obtained when staining three different wells of SK-BR-3 cells using the same master mix (intra-experimental set), compared with three consecutive experiments with individual reagent mixes (inter-experimental set). No significant difference between the intra-experimental and the inter-experimental means of signals per cell was detected ( supplemental Fig. 8 ).

After the validation steps of the PLA probes on cells, we continued to demonstrate the multiplex combinatorial in situ PLA on fresh frozen tissue. We chose to detect proximity between the EGFR family members EGFR, HER2, and HER3 in fresh-frozen human breast cancer tissue. Breast cancer tissues are routinely characterized by immunohistochemistry to assess HER2 expression and are scored from 0+, indicating no visible staining, to 3+, indicating strong staining. Breast cancer tissue sections are heterogeneous, and the staining patterns will vary depending on the fraction of cancer cells present in the image. We performed the assay multiple times in different tissue sections to confirm reproducibility. We present representative images of the different tissues from patients of the two extremes (0+ and 3+) and analyzed for all possible EGFR, HER2, and HER3 combinations, together with quantification of these selected images. First, we used a general HER2-specific PLA probe in combination with HER2-, HER3-, and EGFR-specific probes to evaluate the level of complex formation of HER2 with all three receptors. We observed high levels of HER2 complexes, especially HER2 homodimers, in the 3+ tissue but not in the 0+ tissues ( Fig. 4 , a and b ). A specific staining pattern was observed, with tumor areas displaying very high levels of complexes, whereas much fewer complexes were observed in the surrounding stroma ( supplemental Figs. 9–14 ). We then replaced the HER2-specific general PLA probe with probes recognizing EGFR or HER3 to detect all pairwise associations between EGFR and HER3 with EGFR, HER2, and HER3 ( Fig. 4 , c and d and e and f ). The numbers of observed complexes involving EGFR and HER3 were generally moderate to low in both tissues. Distinct staining patterns across different cells were discerned, even though differences were not as clearly defined as for complexes involving HER2.

DISCUSSION

The ability of in situ PLA to visualize endogenous protein complexes in clinical tissue specimens provides unique opportunities to characterize molecular processes at cellular resolution to explain cellular reactions and for diagnosis of disease states. As is true for other proximity-based methods used to monitor protein complexes in situ , e.g. FRET, the VeraTag assay, or protein fragment complementation assays such as BiFC, the detected proximity between two proteins is not proof of direct physical interactions between the molecules. Instead, the assays may provide information that the molecules participate in the same protein complexes and thus contribute to the actions by these protein complexes. So-called scaffold proteins can coordinate binding in single complexes by many proteins whose co-location could be measured by methods like in situ PLA ( 21 ). However, highly expressed proteins should have an increased risk of false positive PLA signal production as more copies will reside close to the interaction partner by chance. This is a fate shared with the other mentioned proximity-based detection methods. The distance that can separate two molecules recognized in a proximity ligation reaction depends on a number of factors such as the location of the recognized epitopes on the protein surfaces, the size of the affinity reagents used, the site of the attached oligonucleotides, and the distance over which the oligonucleotides can give rise to a DNA circle and hence detect a signal via RCA. Under the conditions reported herein, we estimate that epitope distances of a few tens of nanometers permit production of detection signals.

Just as with immunohistochemistry, the detection efficiency of in situ PLA depends on factors such as choice of fixation and pretreatment in combination with the antibodies used, as well as the concentration of each probe. It is therefore important to optimize all of these parameters for each probe set to ensure optimal conditions. To be able to use in situ PLA for formalin-fixed paraffin embedded (FFPE), the antibodies used need to recognize the fixed epitopes, and antigen retrieval may be required ( 22 ). Another concern with PLA is that at higher concentrations of recorded PLA signals, these RCA products will start to coalesce, preventing counting of individual signals. Recent developments have made possible the detection of RCA products with adjustable sensitivity to expand the dynamic range of the assay ( 23 ).

Here, we describe a novel oligonucleotide design that facilitates multiplexed analyses of combinatorial protein interactions in fixed cells and tissue, using in situ PLA. The level of multiplexing in this advancement of in situ PLA could be increased substantially from the three simultaneous detection reactions that we demonstrate herein, simply by designing larger sets of orthogonal oligonucleotides. By using a serial hybridization approach, as has been described previously for RCA products deposited on slides ( 24 ), the number of recorded protein complex identities can be increased beyond the limit set by the number of fluorophores that can be spectrally distinguished by microscopy. We have used an oligonucleotide design suitable to detect complexes of one protein and each of three other proteins ( Fig. 1 ), but other designs could be used that would allow detection of all pairwise combinations by including a detection tag in both PLA probes ( supplemental Fig. 15 ) and assay for double-labeled RCA products ( 25 ). Multiplex assays generally save time and effort and, where applicable, precious clinical material, while providing opportunities for internally controlled reactions. More importantly, however, the method demonstrated herein allows simultaneous assessment of several concurrent molecular events at cellular and subcellular resolution in cells and tissues. This provides entirely new opportunities for snapshot analyses of molecular events that will help elucidate intricate protein networks, revealing protein complexes that jointly exert cellular responses. Multiplex in situ PLA can measure the balance between protein complex formations along a pathway or between different pathways, and it can provide a foundation for a systems understanding of cellular functions. Multiplex in situ PLA also has the potential to reveal entirely new biomarkers that reflect activity states rather than the mere presence or absence of individual proteins. This promises to result in improved disease diagnostics and efficient prediction of responses to molecularly targeted treatment. The method could also prove of great importance in drug discovery, for example in network pharmacology ( 26 ) where drugs are designed to act on several targets simultaneously. The ability to simultaneously assess several protein complexes may prove important in approaches to perturb robust underlying networks ( 27 ) and to avoid escape mutations in malignancy ( 26 , 28 ).
