---
pmid: '11932238'
title: The genome of M. acetivorans reveals extensive metabolic and physiological
  diversity.
authors:
- Galagan JE
- Nusbaum C
- Roy A
- Endrizzi MG
- Macdonald P
- FitzHugh W
- Calvo S
- Engels R
- Smirnov S
- Atnoor D
- Brown A
- Allen N
- Naylor J
- Stange-Thomann N
- DeArellano K
- Johnson R
- Linton L
- McEwan P
- McKernan K
- Talamas J
- Tirrell A
- Ye W
- Zimmer A
- Barber RD
- Cann I
- Graham DE
- Grahame DA
- Guss AM
- Hedderich R
- Ingram-Smith C
- Kuettner HC
- Krzycki JA
- Leigh JA
- Li W
- Liu J
- Mukhopadhyay B
- Reeve JN
- Smith K
- Springer TA
- Umayam LA
- White O
- White RH
- Conway de Macario E
- Ferry JG
- Jarrell KF
- Jing H
- Macario AJ
- Paulsen I
- Pritchett M
- Sowers KR
- Swanson RV
- Zinder SH
- Lander E
- Metcalf WW
- Birren B
journal: Genome Res
year: '2002'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC187521
doi: 10.1101/gr.223902
---

# The genome of M. acetivorans reveals extensive metabolic and physiological diversity.
**Authors:** Galagan JE, Nusbaum C, Roy A, Endrizzi MG, Macdonald P, FitzHugh W, Calvo S, Engels R, Smirnov S, Atnoor D, Brown A, Allen N, Naylor J, Stange-Thomann N, DeArellano K, Johnson R, Linton L, McEwan P, McKernan K, Talamas J, Tirrell A, Ye W, Zimmer A, Barber RD, Cann I, Graham DE, Grahame DA, Guss AM, Hedderich R, Ingram-Smith C, Kuettner HC, Krzycki JA, Leigh JA, Li W, Liu J, Mukhopadhyay B, Reeve JN, Smith K, Springer TA, Umayam LA, White O, White RH, Conway de Macario E, Ferry JG, Jarrell KF, Jing H, Macario AJ, Paulsen I, Pritchett M, Sowers KR, Swanson RV, Zinder SH, Lander E, Metcalf WW, Birren B
**Journal:** Genome Res (2002)
**DOI:** [10.1101/gr.223902](https://doi.org/10.1101/gr.223902)
**PMC:** [PMC187521](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC187521/)

## Abstract

1. Genome Res. 2002 Apr;12(4):532-42. doi: 10.1101/gr.223902.

The genome of M. acetivorans reveals extensive metabolic and physiological 
diversity.

Galagan JE(1), Nusbaum C, Roy A, Endrizzi MG, Macdonald P, FitzHugh W, Calvo S, 
Engels R, Smirnov S, Atnoor D, Brown A, Allen N, Naylor J, Stange-Thomann N, 
DeArellano K, Johnson R, Linton L, McEwan P, McKernan K, Talamas J, Tirrell A, 
Ye W, Zimmer A, Barber RD, Cann I, Graham DE, Grahame DA, Guss AM, Hedderich R, 
Ingram-Smith C, Kuettner HC, Krzycki JA, Leigh JA, Li W, Liu J, Mukhopadhyay B, 
Reeve JN, Smith K, Springer TA, Umayam LA, White O, White RH, Conway de Macario 
E, Ferry JG, Jarrell KF, Jing H, Macario AJ, Paulsen I, Pritchett M, Sowers KR, 
Swanson RV, Zinder SH, Lander E, Metcalf WW, Birren B.

Author information:
(1)Whitehead Institute Center for Genome Research, Cambridge, Massachusetts 
02141, USA.

Methanogenesis, the biological production of methane, plays a pivotal role in 
the global carbon cycle and contributes significantly to global warming. The 
majority of methane in nature is derived from acetate. Here we report the 
complete genome sequence of an acetate-utilizing methanogen, Methanosarcina 
acetivorans C2A. Methanosarcineae are the most metabolically diverse 
methanogens, thrive in a broad range of environments, and are unique among the 
Archaea in forming complex multicellular structures. This diversity is reflected 
in the genome of M. acetivorans. At 5,751,492 base pairs it is by far the 
largest known archaeal genome. The 4524 open reading frames code for a 
strikingly wide and unanticipated variety of metabolic and cellular 
capabilities. The presence of novel methyltransferases indicates the likelihood 
of undiscovered natural energy sources for methanogenesis, whereas the presence 
of single-subunit carbon monoxide dehydrogenases raises the possibility of 
nonmethanogenic growth. Although motility has not been observed in any 
Methanosarcineae, a flagellin gene cluster and two complete chemotaxis gene 
clusters were identified. The availability of genetic methods, coupled with its 
physiological and metabolic diversity, makes M. acetivorans a powerful model 
organism for the study of archaeal biology. [Sequence, data, annotations and 
analyses are available at http://www-genome.wi.mit.edu/.]

DOI: 10.1101/gr.223902
PMCID: PMC187521
PMID: 11932238 [Indexed for MEDLINE]

## Full Text

Abstract

Methanogenesis, the biological production of methane, plays a pivotal role in the global carbon cycle and contributes significantly to global warming. The majority of methane in nature is derived from acetate. Here we report the complete genome sequence of an acetate-utilizing methanogen, Methanosarcina acetivorans C2A. Methanosarcineae are the most metabolically diverse methanogens, thrive in a broad range of environments, and are unique among the Archaea in forming complex multicellular structures. This diversity is reflected in the genome of M. acetivorans . At 5,751,492 base pairs it is by far the largest known archaeal genome. The 4524 open reading frames code for a strikingly wide and unanticipated variety of metabolic and cellular capabilities. The presence of novel methyltransferases indicates the likelihood of undiscovered natural energy sources for methanogenesis, whereas the presence of single-subunit carbon monoxide dehydrogenases raises the possibility of nonmethanogenic growth. Although motility has not been observed in any Methanosarcineae , a flagellin gene cluster and two complete chemotaxis gene clusters were identified. The availability of genetic methods, coupled with its physiological and metabolic diversity, makes M. acetivorans a powerful model organism for the study of archaeal biology.

[Sequence, data, annotations, and analyses are available at http://www-genome.wi.mit.edu/ . The sequence data described in this paper have been submitted to the GenBank data library under accession no. AE010299 .]

DISCUSSION

Methanosarcineae are unsurpassed among methanogens in terms of metabolic, physiological, and environmental versatility and are unique among archaea in forming complex multicellular structures. These capabilities are reflected in the genome of M. acetivorans , the largest known for an archaeon. The genome reveals extensive genetic diversity and redundancy underlying the ability of M. acetivorans to adapt to varied environmental conditions. In addition, the genome sequence indicates the potential for entirely unexpected metabolic capabilities. These include the possibilities of methanogenesis from as yet undiscovered substrates, nonmethanogenic growth-utilizing CO, and perhaps even O 2 -dependent respiration.

The powerful genetic methods available for M. acetivorans allow the immediate exploitation of the complete genome sequence for understanding the organism's biochemical and environmental diversity and the regulatory mechanisms that underlie its adaptive abilities. In addition, these methods permit functional studies of the over 1300 predicted proteins lacking homology to known proteins. Moreover, the existence of ongoing sequencing efforts for M. mazei (GÃ¶ttingen Genomics Laboratory at http://www.g2l.bio.uni-goettingen.de/ ), Methanosarcina thermophila , and Methanosarcina barkeri (DOE Joint Genome Institute [JGI] at http://www.jgi.doe.gov/JGI_microbial/html/ ) promises unparalleled opportunity for comparative studies. Such studies offer the potential for elucidating the evolutionary basis of the extraordinary genetic diversity of the Methanosarcineae , leading in turn to a greater understanding of the archaeal domain of life.
