---
pmid: '23509263'
title: Activity-enhancing mutations in an E3 ubiquitin ligase identified by high-throughput
  mutagenesis.
authors:
- Starita LM
- Pruneda JN
- Lo RS
- Fowler DM
- Kim HJ
- Hiatt JB
- Shendure J
- Brzovic PS
- Fields S
- Klevit RE
journal: Proc Natl Acad Sci U S A
year: '2013'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC3619334
doi: 10.1073/pnas.1303309110
---

# Activity-enhancing mutations in an E3 ubiquitin ligase identified by high-throughput mutagenesis.
**Authors:** Starita LM, Pruneda JN, Lo RS, Fowler DM, Kim HJ, Hiatt JB, Shendure J, Brzovic PS, Fields S, Klevit RE
**Journal:** Proc Natl Acad Sci U S A (2013)
**DOI:** [10.1073/pnas.1303309110](https://doi.org/10.1073/pnas.1303309110)
**PMC:** [PMC3619334](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3619334/)

## Abstract

1. Proc Natl Acad Sci U S A. 2013 Apr 2;110(14):E1263-72. doi: 
10.1073/pnas.1303309110. Epub 2013 Mar 18.

Activity-enhancing mutations in an E3 ubiquitin ligase identified by 
high-throughput mutagenesis.

Starita LM(1), Pruneda JN, Lo RS, Fowler DM, Kim HJ, Hiatt JB, Shendure J, 
Brzovic PS, Fields S, Klevit RE.

Author information:
(1)Howard Hughes Medical Institute, Department of Genome Sciences, University of 
Washington, Seattle, WA 98195, USA.

Although ubiquitination plays a critical role in virtually all cellular 
processes, mechanistic details of ubiquitin (Ub) transfer are still being 
defined. To identify the molecular determinants within E3 ligases that modulate 
activity, we scored each member of a library of nearly 100,000 protein variants 
of the murine ubiquitination factor E4B (Ube4b) U-box domain for 
auto-ubiquitination activity in the presence of the E2 UbcH5c. This assay 
identified mutations that enhance activity both in vitro and in cellular p53 
degradation assays. The activity-enhancing mutations fall into two distinct 
mechanistic classes: One increases the U-box:E2-binding affinity, and the other 
allosterically stimulates the formation of catalytically active conformations of 
the E2∼Ub conjugate. The same mutations enhance E3 activity in the presence of 
another E2, Ube2w, implying a common allosteric mechanism, and therefore the 
general applicability of our observations to other E3s. A comparison of the E3 
activity with the two different E2s identified an additional variant that 
exhibits E3:E2 specificity. Our results highlight the general utility of 
high-throughput mutagenesis in delineating the molecular basis of enzyme 
activity.

DOI: 10.1073/pnas.1303309110
PMCID: PMC3619334
PMID: 23509263 [Indexed for MEDLINE]

Conflict of interest statement: The authors declare no conflict of interest.

## Full Text

Abstract

Although ubiquitination plays a critical role in virtually all cellular processes, mechanistic details of ubiquitin (Ub) transfer are still being defined. To identify the molecular determinants within E3 ligases that modulate activity, we scored each member of a library of nearly 100,000 protein variants of the murine ubiquitination factor E4B (Ube4b) U-box domain for auto-ubiquitination activity in the presence of the E2 UbcH5c. This assay identified mutations that enhance activity both in vitro and in cellular p53 degradation assays. The activity-enhancing mutations fall into two distinct mechanistic classes: One increases the U-box:E2-binding affinity, and the other allosterically stimulates the formation of catalytically active conformations of the E2∼Ub conjugate. The same mutations enhance E3 activity in the presence of another E2, Ube2w, implying a common allosteric mechanism, and therefore the general applicability of our observations to other E3s. A comparison of the E3 activity with the two different E2s identified an additional variant that exhibits E3:E2 specificity. Our results highlight the general utility of high-throughput mutagenesis in delineating the molecular basis of enzyme activity.

Discussion

Our deep mutational scan of Ube4b has provided the most in-depth analysis to date of U-box sequence–function relationships. The activating mutations identified from this approach represent changes in RING/U-box E3s that broadly enhance ligase activity in a substrate-independent fashion. The selection strategy was predicated on the hypothesis that the ability of E3s to activate Ub transfer by E2∼Ubs can be separated into substrate-dependent and substrate-independent contributions, and that mutations that increase substrate-independent processes will also be enhanced for substrate ubiquitination activity. The hypothesis was borne out because mutations identified using Ube4b auto-ubiquitination as the readout also exhibited enhanced activity in cellular degradation of the substrate p53 and in biochemical assays monitoring transfer to the free amino acid lysine. Many of the most enriched variants contain a conservative mutation, suggesting that the weak interactions and catalytical functions of U-box ligases can be specifically tuned to suit their biological roles. Despite the capacity of a single point mutation to enhance ligase activity greatly, Ube4b has evolved in mammals to function with more moderate ubiquitination activity. The importance of human UBE4B ligase activity in the turnover of p53 ( 16 ) argues that precise levels of ubiquitination activity are critical for cellular function. Overactive ubiquitination by UBE4B would offer an opportunity for tumor cells to suppress the p53-dependent cell cycle arrest and/or apoptotic response. Because other U-boxes contain some of the amino acids in their native sequences that activated Ube4b, their activities may also be subject to fine-tuning.

Gain-of-function mutations that aid in the elucidation of enzymatic mechanisms are commonly sought but rarely discovered using conventional mutagenesis strategies. The rare nature of ligase-activating mutations emphasizes the power of the deep mutational scanning method: Only 25 single mutants (2.7% of the total) exhibited threefold or greater enrichment over WT after three rounds of selection. Many of the activity-enhancing mutations are chemically similar, evolutionarily conserved, and physically distant from known protein/protein interfaces, making it unlikely that they would have been predicted even with available high-resolution structural information. As mutations that significantly activate RING-type E3s, this set provides a unique tool to further the understanding of ligase function. Our NMR-based analyses illustrate that the E4BU-activating mutations fall into two classes: those that predominantly affect the binding affinity of E4BU for the E2 (L1107I and M1124V) and those that primarily affect the ability of E4BU to activate E2∼Ub conjugates (D1139N and N1142T) allosterically.

Because E4BU binds both the free and Ub-conjugated forms of the E2 with relatively similar affinities ( 3 ), it was unclear whether tighter binding would be beneficial or detrimental for Ub transfer. On the one hand, an extremely tight-binding E2 would transfer only the first Ub before acting as an inhibitor because it would be unable to be recharged by the E1 Ub-activating enzyme. On the other hand, a longer-lived E3:E2∼Ub complex (as is observed with the L1107I and M1124V variants) would allow more time for the formation of active E2∼Ub conformations and the approach of an incoming substrate lysine. We postulate that along the continuum of E3:E2-binding affinities, a point must exist at which Ub transfer reactivity is maximized before additional binding energy inhibits the reaction; at that point, the E3:E2 dissociation step becomes rate-limiting. WT Ube4b must be low enough on this binding energy vs. catalysis curve such that tighter binding can lead to enhanced Ub transfer; the hyperactive L1107I and M1124V variants showed as much as a sixfold enhancement in binding affinity.

Although the D1139N and N1142T variants showed no appreciable effect on E2 binding compared with WT, they demonstrated an enhanced ability to promote the active, closed conformations of the E2∼Ub conjugate allosterically. Neither the D1139N mutation nor the N1142T mutation could rescue an R1143 mutation in the phage assay ( Dataset S1 ), consistent with the identification of R1143 as the “linchpin” for the allosteric activation of Ub transfer ( 3 ). D1139 and N1142 contribute significantly to the interactions that stabilize E4BU loop 2. We propose that the D1139N and N1142T mutations alter the structure and/or dynamics of loop 2, thereby enhancing the probability and/or longevity of the E4BU:UbcH5c intermolecular hydrogen bond and augmenting the allosteric role of the E3. Additionally, D1139 and N1142 are analogous to conserved zinc-binding cysteines in loop 2 of RING domains [e.g., C61 and C64 in breast cancer type 1 susceptibility protein (BRCA1)], suggesting that U-boxes may have evolved as a class of E3 ligase distinct from the RING cross-brace structure to enable additional modes of modulating their ligase activities.

The phage selection revealed thousands of sequence–function relationships for Ube4b:UbcH5c activity. Although this is likely the most relevant E3:E2 pair for the degradation of p53 ( 16 ), little is known regarding other roles Ube4b may play in the cell that potentially use different E2s. We showed that Ube4b is also active with Ube2w, an E2 linked to Fanconi anemia due to its ubiquitination of Fanconi anemia complementation group D2 (FANCD2) ( 23 ), and that hyperactivating mutations in Ube4b identified using this E2 are similar to those obtained with UbcH5c. However, the L1107I, D1139N, and N1142T mutations enhanced Ub transfer activity with Ube2w more than the M1124V mutation did. Based on the critical E3:E2 intermolecular hydrogen bond conserved in all structures solved to date, we predict that allosteric mutations will be more universal among E2s, whereas binding mutations may be more E2-specific. By comparing the enrichment values from the UbcH5c and Ube2w selections, we identified the R1104E mutation as highly detrimental for UbcH5c-dependent activity but with little effect on Ube2w-dependent activity. Therefore, future application of the deep mutational scanning technique might be used to design nonnative E3:E2 pairs or to examine biological roles of E3:E2 combinations through the engineering of variants specific for a given E2.

Beyond the practical applications for future studies of E3 ligases, the identification and characterization of Ube4b ligase-activating mutations have added molecular detail to the separable binding and allosteric roles of E3 ligases in facilitating Ub transfer. The insights into E3 ligase function provided by the deep mutational scans of Ube4b suggest that the mutational analysis of enzymes on a high-throughput basis has promise as a general means to identify key features of enzymatic mechanisms.

Materials and Methods

Full experimental and analytical details can be found in SI Materials and Methods .
