---
pmid: '12853576'
title: Human inhibitory receptors Ig-like transcript 2 (ILT2) and ILT4 compete with
  CD8 for MHC class I binding and bind preferentially to HLA-G.
authors:
- Shiroishi M
- Tsumoto K
- Amano K
- Shirakihara Y
- Colonna M
- Braud VM
- Allan DS
- Makadzange A
- Rowland-Jones S
- Willcox B
- Jones EY
- van der Merwe PA
- Kumagai I
- Maenaka K
journal: Proc Natl Acad Sci U S A
year: '2003'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC166403
doi: 10.1073/pnas.1431057100
---

# Human inhibitory receptors Ig-like transcript 2 (ILT2) and ILT4 compete with CD8 for MHC class I binding and bind preferentially to HLA-G.
**Authors:** Shiroishi M, Tsumoto K, Amano K, Shirakihara Y, Colonna M, Braud VM, Allan DS, Makadzange A, Rowland-Jones S, Willcox B, Jones EY, van der Merwe PA, Kumagai I, Maenaka K
**Journal:** Proc Natl Acad Sci U S A (2003)
**DOI:** [10.1073/pnas.1431057100](https://doi.org/10.1073/pnas.1431057100)
**PMC:** [PMC166403](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC166403/)

## Abstract

1. Proc Natl Acad Sci U S A. 2003 Jul 22;100(15):8856-61. doi: 
10.1073/pnas.1431057100. Epub 2003 Jul 9.

Human inhibitory receptors Ig-like transcript 2 (ILT2) and ILT4 compete with CD8 
for MHC class I binding and bind preferentially to HLA-G.

Shiroishi M(1), Tsumoto K, Amano K, Shirakihara Y, Colonna M, Braud VM, Allan 
DS, Makadzange A, Rowland-Jones S, Willcox B, Jones EY, van der Merwe PA, 
Kumagai I, Maenaka K.

Author information:
(1)Division of Structural Biology, Medical Institute of Bioregulation, Kyushu 
University, 3-1-1 Maidashi, Higashi-ku, Fukuoka 812-8582, Japan.

Ig-like transcript 4 (ILT4) (also known as leukocyte Ig-like receptor 2, CD85d, 
and LILRB2) is a cell surface receptor expressed mainly on myelomonocytic cells, 
whereas ILT2 (also known as leukocyte Ig-like receptor 1, CD85j, and LILRB1) is 
expressed on a wider range of immune cells including subsets of natural killer 
and T cells. Both ILTs contain immunoreceptor tyrosine-based inhibitory receptor 
motifs in their cytoplasmic tails that inhibit cellular responses by recruiting 
phosphatases such as SHP-1 (Src homology 2 domain containing tyrosine 
phosphatase 1). Although these ILTs have been shown to recognize a broad range 
of classical and nonclassical human MHC class I molecules (MHCIs), their precise 
binding properties remain controversial. We have used surface plasmon resonance 
to analyze the interaction of soluble forms of ILT4 and ILT2 with several MHCIs. 
Although the range of affinities measured was quite broad (Kd = 2-45 microM), 
some interesting differences were observed. ILT2 generally bound with a 2- to 
3-fold higher affinity than ILT4 to the same MHCI. Furthermore, ILT2 and ILT4 
bound to HLA-G with a 3- to 4-fold higher affinity than to classical MHCIs, 
suggesting that ILT/HLA-G recognition may play a dominant role in the regulation 
of natural killer, T, and myelomonocytic cell activation. Finally, we show that 
ILT2 and ILT4 effectively compete with CD8 for MHCI binding, raising the 
possibility that ILT2 modulates CD8+ T cell activation by blocking the CD8 
binding as well as by recruiting inhibitory molecules through its immunoreceptor 
tyrosine-based inhibitory receptor motif.

DOI: 10.1073/pnas.1431057100
PMCID: PMC166403
PMID: 12853576 [Indexed for MEDLINE]

## Full Text

Abstract

Ig-like transcript 4 (ILT4) (also known as leukocyte Ig-like receptor 2,
 CD85d, and LILRB2) is a cell surface receptor expressed mainly on
 myelomonocytic cells, whereas ILT2 (also known as leukocyte Ig-like receptor
 1, CD85j, and LILRB1) is expressed on a wider range of immune cells including
 subsets of natural killer and T cells. Both ILTs contain immunoreceptor
 tyrosine-based inhibitory receptor motifs in their cytoplasmic tails that
 inhibit cellular responses by recruiting phosphatases such as SHP-1 (Src
 homology 2 domain containing tyrosine phosphatase 1). Although these ILTs have
 been shown to recognize a broad range of classical and nonclassical human MHC
 class I molecules (MHCIs), their precise binding properties remain
 controversial. We have used surface plasmon resonance to analyze the
 interaction of soluble forms of ILT4 and ILT2 with several MHCIs. Although the
 range of affinities measured was quite broad ( K d =
 2–45 μM), some interesting differences were observed. ILT2 generally
 bound with a 2- to 3-fold higher affinity than ILT4 to the same MHCI.
 Furthermore, ILT2 and ILT4 bound to HLA-G with a 3- to 4-fold higher affinity
 than to classical MHCIs, suggesting that ILT/HLA-G recognition may play a
 dominant role in the regulation of natural killer, T, and myelomonocytic cell
 activation. Finally, we show that ILT2 and ILT4 effectively compete with CD8
 for MHCI binding, raising the possibility that ILT2 modulates CD8 + T cell activation by blocking the CD8 binding as well as by recruiting
 inhibitory molecules through its immunoreceptor tyrosine-based inhibitory
 receptor motif.

Materials and Methods

Production of ILT2 and ILT4 Ectodomains. DNA encoding the first two
 extracellular domains (residues 1–197) of ILT4 was amplified from cDNA
 ( 6 ) by using 5′-G GAA CAT
 ATG GGG ACC ATC CCC AAG CCC-3′ as forward primer and 5′-CC CAA GCT
 TAC TAT GGG ACC AGG AAG CTC CAG G-3′ as reverse primer. The resultant
 fragments were digested with the restriction enzymes Nde I and Hind III and ligated into the pGMT7 vector
 ( 27 ) (designated pGMILT4D1D2). Escherichia coli strain BL21(DE3)pLysS cells (Novagen) harboring
 pGMILT4D1D2 produced ILT4D1D2 inclusion bodies. They were isolated from cell
 pellet by sonication and washed repeatedly with wash solution including 0.5%
 Triton X-100. The DNA encoding ILT2D1D2 region (residues 1–197) designed
 by using E. coli -favored codons was constructed by PCR with 10
 chemically synthesized DNAs (see Table 2, which is published as supporting
 information on the PNAS web site, www.pnas.org ).
 The fragment was inserted into the pGMT7 vector (designated pGMILT2D1D2), and
 inclusion bodies containing ILT2D1D2 were obtained by using the same method as
 described above.

The purified ILT2D1D2 inclusion bodies were solubilized in denaturant
 solution including 6 M guanidine hydrochloride. By using the refolding buffer
 (0.1 M Tris·HCl, pH 8.0/0.4 M l -arginine/2 mM EDTA/5 mM
 reduced glutathione/0.5 mM oxidized glutathione/0.1 mM PMSF), the solubilized
 protein solution was diluted slowly to the final protein concentration of
 1–2 μM and stirred for 48 h at 4°C. Then the refolding mixture of
 ILT2D1D2 or ILT4D1D2 was concentrated with a VIVAFLOW50 system (Sartorius).
 ILT2D1D2 was purified by gel filtration with Superdex 75 (Amersham Pharmacia).
 In the case of ILT4D1D2, after concentrating to 5 μM, the buffer was
 exchanged gradually to 50 mM phosphate buffer, pH 6.0, with the VIVAFLOW50
 system. ILT4D1D2 was purified by anion-exchange chromatography (SP Sepharose,
 Amersham Pharmacia) followed by gel filtration (Superdex 75).

The whole ectodomain of ILT2 was obtained from transfected J558L cells
 producing chimeric ILT2 molecule fused with the Fc portion of human IgG1
 ( 6 ). On the other hand, for the
 whole ectodomain of ILT4, an expression system producing soluble extracellular
 domains of ILT4 tagged at the C terminus with a c-Myc tag and poly-His tail
 was constructed.

Production of Soluble and Biotinylated MHC Molecules. Soluble
 biotinylated HLA-A*1101 (with peptide AIFQSSMTK), HLA-B*3501 (with peptide
 IPLTEEAEL), HLA-Cw*0401 (with peptide QYDDAVYKL), and HLA-G1 (with peptide
 RIIPRHLQL) with C-terminal biotin ligase (BirA) recognition sequence
 (GSLHHILDAQKMVWNHR) were prepared as described
 ( 27 , 28 ). Each MHCI was
 biotinylated with 50 mM d -biotin/100 mM ATP/15 μM BirA for 15
 μM MHCI in the reaction buffer. After biotinylation, MHCIs were separated
 from the reaction mixture by gel filtration (Superdex 75). Biotinylated
 HLA-Cw*0702 was refolded with peptide [DS11 (RYRPGTVAL) or DS12 (NKADVILKY)]
 and chemically biotinylated β 2 -microglobulin
 (β 2 m) in the same way as other MHCI complexes. Purification
 was accomplished by using Superdex 200 and MonoQ columns.

Surface Plasmon Resonance (SPR). SPR experiments were performed by
 using a BIAcore2000 (BIAcore, St. Albans, U.K.). The biotinylated MHCIs and
 control protein (biotinylated OX48 antibody or BSA) were immobilized on the
 research-grade CM5 chip (BIAcore), onto which streptavidin was covalently
 coupled. ILT2D1D2 and ILT4D1D2, after buffer exchange to HBS-EP (10 mM Hepes,
 pH7.4/150 mM NaCl/3.4 mM EDTA/0.005% Surfactant P20), were injected over the
 immobilized MHCIs. The binding response at each concentration was calculated
 by subtracting the equilibrium response measured in the control flow cell from
 the response in the MHCIs flow cell. Kinetic constants were derived by using
 the curve-fitting facility of the biaevaluation 3.0 program
 (BIAcore) to fit rate equations derived from the simple 1:1 Langmuir binding
 model ( A + B ↔ AB ). Other curve fitting was
 performed by origin 3 (Microcal Software, Northampton, MA).
 Affinity constants ( K d ) were derived by Scatchard analysis
 or nonlinear curve fitting of the standard Langmuir binding isotherm.

The whole ectodomain of ILT2 fused with Fc was indirectly coupled to the
 SPR sensor surface by using an anti-human-Fc mAb. Sensor flow-cell surfaces
 coated with mAb alone served as paired controls. With regards to ILT4, the
 whole ectodomain was coupled indirectly to the SPR sensor chip by using the
 anti-ILT 40H2 mAb. MHCIs were injected over the immobilized ILTs in
 HBS-EP.

Competitive Binding Assays. The ectodomain of KIR2DL1 (residues
 1–224) and CD8α homodimer (CD8αα) (residues
 1–120) were produced as described
 ( 27 , 29 ). ILT2D1D2 and ILT4D1D2
 with or without a fixed amount of KIR2DL1(38 μM), which was almost
 saturated, were flowed over the immobilized HLA-Cw*0401. ILT2D1D2 and
 ILT4D1D2, with or without a fixed concentration of CD8αα (92
 μM), were flowed over the immobilized HLA-B*3501, -Cw*0401, and -G1.

Results and Discussion

Production of Soluble ILTs and MHCIs. We initially attempted to
 express three forms of the ILT4 ectodomain: domain 1 alone (D1, residues
 1–98); domains 1 and 2 (D1D2, residues 1–197); and the entire
 extracellular domains (D1–D4, residues 1–435). These constructs
 were expressed in E. coli as inclusion bodies and subsequently
 refolded in vitro . ILT4D1 and ILT4D1–D4 were difficult to
 refold and aggregated readily, making it impossible to produce enough material
 for further study. However, ILT4D1D2 was successfully refolded and purified as
 described in Materials and Methods . Because stored ILT4D1D2 tended to
 aggregate, we prepared freshly refolded and purified ILT4D1D2 for each
 experiment. The equivalent ILT2 fragment (ILT2D1D2) was prepared by using a
 very similar method (see Materials and Methods ). The far-UV circular
 dichroism spectrum of ILT4D1D2 indicated that it had mainly β-sheet
 secondary structure, similar to ILT2D1D2 and as expected for a protein with
 two Ig-like domains (ref. 18 and data not shown). For comparison, soluble fragments comprising the entire
 ectodomain of ILT2 and ILT4 (ILT4D1–D4 and ILT2D1–D4) were
 produced by using the eukaryotic expression systems (see Materials and
 Methods ).

All the MHCIs were expressed in bacterial inclusion bodies, refolded in
 vitro with bacterially expressed β 2 m and appropriate
 peptides, and purified as described in Material and Methods . With the
 exception of HLA-Cw*0702, all MHC heavy chains incorporated a C-terminal
 biotinylation tag (LHHILDAQKMVWNHR). After refolding and purification they
 were biotinylated with biotin ligase, which specifically biotinylates a lysine
 in the tag (underlined). HLA-Cw*0702 was refolded with chemically biotinylated
 β 2 m. Biotin-HLA-Cw*0702 produced in this way has been used
 successfully for KIR2DL3-binding studies
 ( 30 ).

Affinities of ILT2 and ILT4 Binding Toward MHCIs. Affinity
 measurements were performed by using SPR as implemented in the BIAcore2000
 instrument. Soluble ILT4D1D2 or ILT2D1D2 was injected over sensor surfaces to
 which biotinylated MHCI complexes or (as negative controls) biotinylated OX48
 antibody or BSA had been immobilized. The affinities of ILT4D1D2 and ILT2D1D2
 binding to MHCIs were measured by equilibrium binding analysis. A range of
 concentrations of ILT4D1D2 or ILT2D1D2 was injected through flow cells with
 MHCIs immobilized.

Representative data for equilibrium binding of ILT4D1D2 to various MHCIs
 are shown in Fig. 1 A .
 ILT4D1D2 binding reached equilibrium and dissociated rapidly, indicating that
 it bound MHCI with the fast kinetics that are typical of interactions
 mediating cell–cell recognition [for example, see Maenaka et
 al. ( 30 , 31 )]. ILT2D1D2 binding also
 displayed fast kinetics ( Fig.
 1 C and see below). Fitting of conventional and Scatchard
 plots of ILT4D1D2 ( Fig.
 1 B and Inset ) and ILT2D1D2
 ( Fig. 1 D and Inset ) binding data indicated that binding conformed to the simple
 1:1 (Langmuir) binding model.

The results of multiple independent measurements for both ILT4D1D2 and
 ILT2D1D2 are summarized in Table
 1 and Figs. 4 and 5, which are published as supporting information
 on the PNAS web site. ILT4D1D2 binds to a broad range of classical MHCIs
 (HLA-A*1101, -B*3501, -Cw*0401, and -Cw*0702) with affinities
 ( K d ) ranging from ≈14 to ≈45 μM (Fig. 5 and Table 1 ). Interestingly,
 ILT4D1D2 binds with an even higher affinity ( K d ≈ 5
 μM) to the nonclassical MHCI HLA-G1. A previous cellular binding study
 ( 7 ) reported that ILT4 binds to
 HLA-A2, -A3, -B8, -B27, -B35, and -G1 but not to HLA-Cw3 or -Cw5. This result
 is surprising given the small differences (one or two residue changes) in the
 relevant (α3) portions of HLA-Cw4, which we show binds ILT4 and HLA-Cw3
 and -Cw5 (see Fig. 5 C ). However, there have been previous reports of
 discrepancies between cell-based and direct binding studies when measuring
 ILT2–HLA-C interactions
 ( 6 – 8 ).
 The reason for these discrepancies remains unclear, but it may be related to
 avidity effects (cell-based assays rely on multivalent binding) and/or the low
 level of cell surface expression of HLA-C when compared with other MHC
 alleles.

ILT2D1D2 binds to HLA-B35 and HLA-Cw4 with affinities
 ( K d ≈ 8.8 and 6.5 μM, respectively) similar to those
 measured for KIR–MHCI interactions ( K d ≈
 7–10 μM) ( 30 , 32 ). As found with ILT4D1D2,
 the ILT2D1D2 bound with the highest affinity to HLA-G1 ( K d ≈ 2.0 μM). As with ILT4, these results are consistent with the previous
 cellular-based assays with the exception of HLA-Cs. HLA-C transfectants do not
 bind efficiently to ILT2 Fc fusion protein or inhibit the killing activity of
 ILT2 + NK cells ( 6 , 15 ).

The higher affinity of ILT2 and ILT4 for HLA-G1 versus other MHCIs (HLA-A2,
 -A68, -B8, and -E) was confirmed by using entire ectodomains (D1–D4) of
 both ILT2 and ILT4 produced by eukaryotic cells (data not shown). In contrast,
 Chapman et al. ( 17 )
 reported that ILT2 binds HLA-G1 with a lower affinity ( K d ≈ 100 μM) than it binds other MHCIs. The reason for this discrepancy is
 uncertain, but it may be the result of differences in the way MHCIs were
 immobilized. In our study we typically used the site-specific biotinylated tag
 at the C terminus of the MHCI heavy chain, whereas Chapman et al. ( 17 ) randomly coupled the MHCI
 via amines. The former method presents MHCI in a manner optimal for ligand
 binding, whereas it is possible that the random coupling of MHC disrupts or
 interferes with ILT binding.

ILTs Compete with CD8 for Binding to MHCIs. Although there is
 evidence from domain-swapping experiments that ILT2 binds the α3 domain
 of MHCI heavy chains ( 17 ), it
 is not known whether ILT binding interferes with the binding of other MHCI
 ligands. We therefore used SPR to examine whether ILT binding to MHCI
 influenced the binding of the MHCI ligands KIR2DL1 and CD8 and the mAb BBM.1,
 which binds β 2 m. KIR2DL1 binds the region adjacent to the
 peptide C terminus on the peptide-binding platform of HLA-Cw4 and related
 alleles [ Fig. 2 A and
 Fan et al. ( 20 )].

Having established that KIR2DL1 binds HLA-Cw4 with an affinity of K d ≈ 3.3 μM ( Fig.
 2 A ), we examined the binding response when increasing
 concentrations of ILT4 ( Fig.
 2 B ) or ILT2 ( Fig.
 2 C ) were injected over HLA-Cw4 with (squares) or without
 (circles) a fixed, high concentration (38 μM) of KIR2DL1. The difference
 between the responses remained the same (crosses) whatever the concentration
 of ILTs, indicating that the binding of ILTs and KIR2DL1 to HLA-Cw4 was purely
 additive. This demonstrates that the binding of ILTs does not affect the
 binding of KIR2DL1 to the same molecule, consistent with previous data showing
 that they bind to different regions of the MHCI.

We next examined whether ILTs interfered with CD8 binding to MHCIs
 ( Fig. 2 E – I ). Having showed that soluble
 recombinant CD8αα bound to HLA-B35, -Cw4, and -G1 with affinities
 of K d ≈ 126, 210, and 72 μM, respectively
 ( Fig. 2 D and Table 1 ), we injected
 increasing concentrations of ILT2 and ILT4 in the presence (squares) or
 absence (circles) of CD8αα (92 μM) over different MHCIs
 ( Fig. 2 E – I ). The difference between the responses
 with or without CD8αα (crosses) decreased as the concentration of
 ILTs increased, indicating that binding was not additive and that ILTs
 inhibited CD8αα binding to these MHCIs. In assays in which the
 concentration of ILT used was saturating, this inhibition was complete
 ( Fig. 2 F – I ), which is typical of competitive
 inhibition and indicates that the ILT and CD8αα-binding sites on
 MHCI overlap or are close enough to each other to block the binding. This is
 consistent with the demonstration that ILT binding requires the α3
 domain of MHCIs ( 17 ).

We next examined the effect of the anti-β 2 m mAb BBM.1 on
 the binding of ILTs and CD8αα
 ( Fig. 2 J ). The binding
 responses observed for ILT2, ILT4, and CD8αα to three different
 MHCIs were the same as before and after saturating the MHCI with BBM.1 mAb
 ( Fig. 2 J ), indicating
 that the BBM.1-binding site does not overlap with either the
 CD8αα- or ILT-binding sites. There is some controversy as to the
 exact position of the BBM.1-binding site on β 2 m; one putative
 site overlaps with the CD8αα site (residues 58–63 and
 91–95; shown in cyan on β 2 m in Fig. 3 C )
 ( 33 ), whereas the other
 proposed site (residues 38, 44, and 45; blue surfaces in Fig. 3 C ) does not
 ( 34 ). Our results suggest that
 the latter site is the BBM.1 epitope.

The binding sites of KIR2DL1
 ( 20 ), BBM.1 mAb
 ( 34 ), and CD8αα
 ( 35 ), as defined by
 mutagenesis and structural studies, are shown in Fig. 3 C . Our results
 show that the ILT-binding site overlaps the CD8αα-binding site
 (cyan surfaces in Fig.
 3 C ). A previous domain-swapping study demonstrated that
 the α3 domain is required for ILT binding
 ( 17 ). Taken together these
 data suggest that the ILT-binding site overlaps with the portion of the
 CD8αα-binding site located on the α3 domain. Consistent with
 this observation, ILT4 binding to HLA-Cw7 and -G1 was not affected by changing
 the peptide in the peptide groove (data not shown).

To localize the ILT-binding site on MHCIs further, we took advantage of our
 finding that ILTs bind to HLA-G1 with a significantly higher affinity than to
 classical MHCIs and the fact that these differences are likely to result from
 a relatively small number of sequence differences
 ( Fig. 3 A ). The regions
 on the surface of MHCIs that differ between HLA-G1 and the classical MHCIs are
 shown in Fig. 3 C (orange and green surfaces on Upper Left and Upper Right ,
 respectively). They cluster in two sites: ( i ) the G strand residues
 268, 271, 275, and 276 (green in Fig.
 3 C Upper Right ) and ( ii ) the CC′E
 β-strand residues 195, 197, 214, and 228 (orange in Fig. 3 C Upper
 Left ). Given that the CD8αα- and ILT-binding sites overlap,
 it is unlikely that site 1 residues (green) contribute to ILT binding.
 However, site 2 residues (orange) are well positioned to contribute to ILT
 binding ( Fig. 3 C ).

There is some evidence that the ILT2- and ILT4-binding sites are not
 identical. First, as shown in Table
 1 , HLA-Cw7 produced by using the chemically biotinylated
 β 2 m could bind to ILT4 but not to ILT2. Second, Allen et
 al. ( 36 ) reported that
 the free form of HLA-B27 lacking β 2 m bound ILT4-transfected
 but not ILT2-transfected cells. These results suggest that ILT2 binding
 depends more on β 2 m than ILT4 binding. Perhaps the
 ILT2-binding site incorporates the region around the
 α3–β 2 m interface.

Our results indicate that, although both ILT2 and ILT4 bind to a range of
 MHCIs, there are significant variations in the binding affinities. On the one
 hand, ILTs bind HLA-G with a higher affinity than they bind classic MHCIs. One
 notable difference in the putative ILT-binding site is the relatively
 hydrophobic patch formed by F195/Y197 on HLA-G, which is absent in other MHCs
 where the corresponding residues are S195/H197. On the other hand, ILT2 has a
 higher affinity and narrower specificity than ILT4. What might be the
 structural basis for this difference? Comparison of sequences in the putative
 MHCI-binding site of ILT molecules indicates notable differences
 ( Fig. 3 B ). For example
 the putative binding region on ILT2D1D2
 ( 18 ) includes residues (Y76,
 D80, and R84) that are not conserved in ILT4 (Q76, R80, and R84)
 ( Fig. 3 C ).
 Furthermore, there is evidence, noted above, that ILT2 but not ILT4 binding to
 MHCI depends on β 2 m, suggesting a difference in the
 ILT2-versus ILT4-binding sites on MHCI. Thus there is significant variation in
 the ILT–MHCI binding interfaces that could easily account for the
 observed variation in binding affinities. Whether these differences are
 physiologically important remains to be shown.

Functional Implications. Our finding that ILT2 and ILT4 bind with a
 higher affinity to HLA-G than to classical MHCIs raises the possibility that
 both the ILTs may contribute to functional interactions between leukocytes
 expressing ILTs (T, NK, and myelomonocytic cells) and cells expressing HLA-G.
 The latter include thymic epithelial cells, fetal trophoblast tissue, some
 cancers, and the cells infected by human cytomegalovirus
 ( 37 ).

ILT2 may influence thymocyte development by interacting with HLA-G and
 classical MHCIs on thymic epithelial cells, thereby modulating the threshold
 of TCR triggering. It is noteworthy that the binding affinity of the
 ILT2–HLA-G interaction is at the high end of the range of affinities
 measured for TCR–MHCs interactions ( K d = 1–50
 μM).

We also show that ILTs compete directly with CD8αα for binding
 to MHCIs. Whereas intraepithelial T cells express CD8αα, most T
 cells express CD8αβ. However, our results are likely to be relevant
 for all T cells, because CD8αα and CD8αβ have
 comparable affinities for MHC, and these affinities are considerably lower
 than the affinity of ILT2 binding to MHCI. The higher affinity of ILT versus
 CD8 binding suggest that ILTs may effectively block CD8 binding at the
 cell–cell interface. Interestingly, Dietrich et al. ( 9 ) showed that ILT2 and TCR
 colocalize at the immunological synapse formed between T cells and
 antigen-presenting cells expressing ligands for ILT2 (HLA-B27) and TCR
 (superantigen) on their surface. These data suggested that ILT2 could
 potentially function as an “inhibitory” coreceptor, first by
 blocking binding of CD8 and second by bringing immunoreceptor tyrosine-based
 inhibitory receptor motifs with their associated tyrosine phosphatases into
 proximity with the TCR engaging the same peptide-MHCI. There is at present no
 direct evidence for a “competitive” inhibitory effect, and the
 fact that CD8 + T cells that express ILT2 can be activated by
 recognition of MHCIs suggests that this competition may be modulated in some
 way. One possibility is that CD8 is effectively recruited to the TCR–CD3
 complex, thereby enhancing its ability to engage MHCIs.

The relatively high affinity of ILT2 binding to HLA-G is consistent with
 previous observations that the effects of ILT2-mediated inhibition on
 peripheral blood NK cells can be largely attributed to HLA-G1 recognition
 ( 38 ). Similarly, the
 ILT2–HLA-G interaction may also effectively inhibit NK cell recognition
 of trophoblast and HLA-G-expressing tumor cells, thereby contributing to
 materno-fetal stolerance and escape of tumor cells.

Although the affinity of ILT4 binding to HLA-G is slightly lower than ILT2,
 ILT4 shows a much stronger preference for HLA-G versus classical MHCIs than
 does ILT2, which suggests that the ILT4–HLA-G interaction may be of
 considerable significance for regulating the maturation and/or function of
 cells in the myelomonocytic lineage. Indeed, it has been suggested that ILT4
 also may regulate the threshold of myelomonocytic activation in materno-fetal
 tolerance ( 12 ), inflammatory
 responses ( 11 ), and
 tolerogenic antigen-presenting cells induced by regulatory T cells
 ( 13 ).

In conclusion, we report here that ILT2 and ILT4 bind more strongly to
 HLA-G than to classical MHCIs, that ILT2 binds with a higher affinity than
 ILT4, and that ILTs compete with CD8 for binding to MHCIs. These observations
 provide insights into the possible role of ILT–MHCI interactions in
 regulating immunological recognition.
