---
pmid: '37703846'
title: CLIPB4 Is a Central Node in the Protease Network that Regulates Humoral Immunity
  in Anopheles gambiae Mosquitoes.
authors:
- Zhang X
- Zhang S
- Kuang J
- Sellens KA
- Morejon B
- Saab SA
- Li M
- Metto EC
- An C
- Culbertson CT
- Osta MA
- Scoglio C
- Michel K
journal: J Innate Immun
year: '2023'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC10603620
doi: 10.1159/000533898
---

# CLIPB4 Is a Central Node in the Protease Network that Regulates Humoral Immunity in Anopheles gambiae Mosquitoes.
**Authors:** Zhang X, Zhang S, Kuang J, Sellens KA, Morejon B, Saab SA, Li M, Metto EC, An C, Culbertson CT, Osta MA, Scoglio C, Michel K
**Journal:** J Innate Immun (2023)
**DOI:** [10.1159/000533898](https://doi.org/10.1159/000533898)
**PMC:** [PMC10603620](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10603620/)

## Abstract

1. J Innate Immun. 2023;15(1):680-696. doi: 10.1159/000533898. Epub 2023 Sep 13.

CLIPB4 Is a Central Node in the Protease Network that Regulates Humoral Immunity 
in Anopheles gambiae Mosquitoes.

Zhang X(1), Zhang S(1)(2), Kuang J(3), Sellens KA(4), Morejon B(1), Saab SA(5), 
Li M(1), Metto EC(4), An C(2), Culbertson CT(4), Osta MA(5), Scoglio C(3), 
Michel K(1).

Author information:
(1)Division of Biology, Kansas State University, Manhattan, Kansas, USA.
(2)Department of Entomology, College of Plant Protection, China Agricultural 
University, Beijing, China.
(3)Department of Electrical and Computer Engineering, Kansas State University, 
Manhattan, Kansas, USA.
(4)Department of Chemistry, Kansas State University, Manhattan, Kansas, USA.
(5)Department of Biology, American University of Beirut, Beirut, Lebanon.

Update of
    bioRxiv. 2023 Jul 16:2023.07.07.545904. doi: 10.1101/2023.07.07.545904.

Insect humoral immune responses are regulated in part by protease cascades, 
whose components circulate as zymogens in the hemolymph. In mosquitoes, these 
cascades consist of clip-domain serine proteases (cSPs) and/or their 
non-catalytic homologs, which form a complex network, whose molecular make-up is 
not fully understood. Using a systems biology approach, based on a co-expression 
network of gene family members that function in melanization and 
co-immunoprecipitation using the serine protease inhibitor (SRPN)2, a key 
negative regulator of the melanization response in mosquitoes, we identify the 
cSP CLIPB4 from the African malaria mosquito Anopheles gambiae as a central node 
in this protease network. CLIPB4 is tightly co-expressed with SRPN2 and forms 
protein complexes with SRPN2 in the hemolymph of immune-challenged female 
mosquitoes. Genetic and biochemical approaches validate our network analysis and 
show that CLIPB4 is required for melanization and antibacterial immunity, acting 
as a prophenoloxidase (proPO)-activating protease, which is inhibited by SRPN2. 
In addition, we provide novel insight into the structural organization of the 
cSP network in An. gambiae, by demonstrating that CLIPB4 is able to activate 
proCLIPB8, a cSP upstream of the proPO-activating protease CLIPB9. These data 
provide the first evidence that, in mosquitoes, cSPs provide branching points in 
immune protease networks and deliver positive reinforcement in proPO activation 
cascades.

© 2023 The Author(s). Published by S. Karger AG, Basel.

DOI: 10.1159/000533898
PMCID: PMC10603620
PMID: 37703846 [Indexed for MEDLINE]

Conflict of interest statement: The authors disclosed all potential conflicts of 
interest, and none existed for the presented work.

## Full Text

IntroductionThe African malaria mosquito, Anopheles gambiae, is one of the efficient vectors of life-threatening human malaria parasites. In 2021, approximately 247 million cases and 619,000 deaths were reported from malaria worldwide [1]. Despite the availability of an effective, albeit non-sterilizing, vaccine, vector population control remains the most common and effective malaria prevention tool, which is threatened by insecticide resistance in target mosquito populations. Novel vector population control schemes, utilizing gene drive, male sterility, and Wolbachia-based methods [2–4] in part rely on detailed molecular understanding of mosquito physiology. For this reason, mosquito innate immunity has received much attention, as it is a major determinant of vector competence and survival of mosquitoes within and across their life stages [5–7].Protease cascades play a central role in innate immunity, regulating blood clotting and complement activation in mammals, hemolymph clotting in horseshoe crabs, and melanization and Toll signaling pathway activation in insects [8–10]. Their activation leads to proteolytic cleavage of a series of inactive protease zymogens between a prodomain and the protease domain, which then cleaves downstream key immune factors, ensuring a fast, often within minutes, activation of antimicrobial immunity. In insects, these cascades consist of so-called clip-containing serine proteases (cSPs), a large protein family (consisting of 5 subfamilies, A-E), which is characterized by one or more clip prodomains and one or more protease domains with trypsin-like specificity, and their homologs (cSPHs), which have lost one or more residues of the catalytic triad required for proteolytic activity. Our knowledge of the molecular makeup of these cascades is mainly derived from insect model species that are either large in size, enabling biochemical investigations of their hemolymph (e.g., Manduca sexta [11] and Tenebrio molitor [12]), or hold an excellent genetic and molecular toolbox (in case of Drosophila melanogaster [13, 14]).At their core, canonical insect immune protease cascades consist of a modular serine protease (ModSP), which upon recognition of foreign molecular patterns, self-activates and then cleaves and activates a cSP in the subfamily C that in turn cleaves and activates a terminal cSP in the subfamily B (cSP-B) [15–17]. Activation of melanization, an insect-specific immune response proceeds through the terminal cSP-B prophenoloxidase activating protease (PAP) that cleaves the zymogen prophenoloxidase (proPO) to active phenoloxidase, a key enzyme in the conversion of aromatic amino acids to quinones that ultimately polymerize to eumelanin on foreign surfaces. Clip serine protease homologs support the proPO cascade, as they are required for effective activation of PO on foreign surfaces [18–20]. To prevent uncontrolled spread and over-activation of responses, serine protease inhibitors in the serpin (SRPN) family tightly control mammalian and arthropod immune protease cascades.In An. gambiae, melanization is a readily selectable parasite resistance mechanism that has garnered much attention in hopes of exploiting the molecular underpinnings of this resistance mechanism for malaria control purposes [21, 22]. In addition, the gene families encoding cSPs and cSPHs in An. gambiae, known as CLIPs, and proPOs have undergone large expansions, raising the question whether regulation of melanization in this mosquito species is more complex than in other insects [23–26]. To date, the melanization response in An. gambiae has been analyzed genetically using RNAi [27] combined with at least five experimental readouts, often with the implicit or explicit assumption that identified CLIPs contribute biochemically to proPO activation. Thus far, two An. gambiae cSPs, CLIPB9 and B10 have been identified as PAPs, based on their ability to revert increased systemic melanization induced genetically by knockdown (kd) of An. gambiae serpin-2 (SRPN2) and to cleave purified M. sexta proPO in vitro [28, 29]. These data indicate that, based on the canonical model of proPO activation cascades, melanization in An. gambiae depends on at least two such cascades, characterized by CLIPB9 and B10, respectively. Based on genetic and biochemical data, CLIPB8 is located further upstream of CLIPB9 [30]. All three CLIPB proteases required for proteolytic activation of proPO are inhibited by SRPNs. Of the 18 SRPNs annotated in the An. gambiae genome, only SRPN2 has been shown to control systemic melanization [7, 31]. It effectively inhibits the two known PAPs (CLIPB9 and B10) and also CLIPB8, albeit to a lesser degree [28–30]. Several additional CLIPBs, including CLIPB3, 4, 14, and 17 also contribute to melanization; however, the molecular function and placement of these additional CLIPBs in proPO activation cascades are unknown [32, 33]. The only cSP in the CLIPC family to contribute to melanization is CLIPC9, presumably through the action of cleavage of either PAP, which however remains to be demonstrated [34].In addition to cSPs, melanization in An. gambiae is regulated positively by a core module of cSPHs, consisting, in hierarchical order, of SPCLIP1, CLIPA8, and CLIPA28. This module is located genetically upstream of CLIPC9 and downstream of the thioester-containing protein (TEP1) [6, 32, 34–36]. Melanization is also regulated negatively by cSPHs, including CLIPA2 and A14 [32, 37]. While the precise mechanism(s) by which An. gambiae cSPHs regulate melanization (positively or negatively) are unknown, observations in An. gambiae suggest that cSPHs contribute to the opsonization of microbial surfaces by TEP1 and the leucine-rich repeat proteins (LRIM) LRIM1 and APLC1 [35, 38] and, based on data from other insect species, to PO activity by mediating proPO cleavage by PAPs [19, 39, 40]. In addition, melanization of rodent and human malaria parasites in An. gambiae is inhibited by two C-type lectins (CTL), CTL4 and CTLMA2 [41, 42], which presumably bind to sugar moieties on the parasite surface and prevent melanization locally. The rodent malaria parasite melanization phenotype induced by CTL4 kd has been used successfully to identify some of the cSPs and cSPHs mentioned above (e.g., [32]).However, despite the significant number of studies summarized above, our understanding of cSP cascades that control melanization in An. gambiae remains incomplete, and more than 80% of the 110 annotated CLIPs await experimental exploration. To guide the experimental analysis of An. gambiae cSPs and to identify putative key regulators of the melanization response, we describe here a novel systems biology approach to study melanization in insects, combining transcriptomic network and proteomic analyses. Using this approach, we identify the cSP CLIPB4 as a central node in the protease network that regulates melanization in An. gambiae and characterize its placement in the proPO activation cascade using a combination of genetic and biochemical approaches.

Introduction

The African malaria mosquito, Anopheles gambiae, is one of the efficient vectors of life-threatening human malaria parasites. In 2021, approximately 247 million cases and 619,000 deaths were reported from malaria worldwide [1]. Despite the availability of an effective, albeit non-sterilizing, vaccine, vector population control remains the most common and effective malaria prevention tool, which is threatened by insecticide resistance in target mosquito populations. Novel vector population control schemes, utilizing gene drive, male sterility, and Wolbachia-based methods [2–4] in part rely on detailed molecular understanding of mosquito physiology. For this reason, mosquito innate immunity has received much attention, as it is a major determinant of vector competence and survival of mosquitoes within and across their life stages [5–7].

Protease cascades play a central role in innate immunity, regulating blood clotting and complement activation in mammals, hemolymph clotting in horseshoe crabs, and melanization and Toll signaling pathway activation in insects [8–10]. Their activation leads to proteolytic cleavage of a series of inactive protease zymogens between a prodomain and the protease domain, which then cleaves downstream key immune factors, ensuring a fast, often within minutes, activation of antimicrobial immunity. In insects, these cascades consist of so-called clip-containing serine proteases (cSPs), a large protein family (consisting of 5 subfamilies, A-E), which is characterized by one or more clip prodomains and one or more protease domains with trypsin-like specificity, and their homologs (cSPHs), which have lost one or more residues of the catalytic triad required for proteolytic activity. Our knowledge of the molecular makeup of these cascades is mainly derived from insect model species that are either large in size, enabling biochemical investigations of their hemolymph (e.g., Manduca sexta [11] and Tenebrio molitor [12]), or hold an excellent genetic and molecular toolbox (in case of Drosophila melanogaster [13, 14]).

At their core, canonical insect immune protease cascades consist of a modular serine protease (ModSP), which upon recognition of foreign molecular patterns, self-activates and then cleaves and activates a cSP in the subfamily C that in turn cleaves and activates a terminal cSP in the subfamily B (cSP-B) [15–17]. Activation of melanization, an insect-specific immune response proceeds through the terminal cSP-B prophenoloxidase activating protease (PAP) that cleaves the zymogen prophenoloxidase (proPO) to active phenoloxidase, a key enzyme in the conversion of aromatic amino acids to quinones that ultimately polymerize to eumelanin on foreign surfaces. Clip serine protease homologs support the proPO cascade, as they are required for effective activation of PO on foreign surfaces [18–20]. To prevent uncontrolled spread and over-activation of responses, serine protease inhibitors in the serpin (SRPN) family tightly control mammalian and arthropod immune protease cascades.

In An. gambiae, melanization is a readily selectable parasite resistance mechanism that has garnered much attention in hopes of exploiting the molecular underpinnings of this resistance mechanism for malaria control purposes [21, 22]. In addition, the gene families encoding cSPs and cSPHs in An. gambiae, known as CLIPs, and proPOs have undergone large expansions, raising the question whether regulation of melanization in this mosquito species is more complex than in other insects [23–26]. To date, the melanization response in An. gambiae has been analyzed genetically using RNAi [27] combined with at least five experimental readouts, often with the implicit or explicit assumption that identified CLIPs contribute biochemically to proPO activation. Thus far, two An. gambiae cSPs, CLIPB9 and B10 have been identified as PAPs, based on their ability to revert increased systemic melanization induced genetically by knockdown (kd) of An. gambiae serpin-2 (SRPN2) and to cleave purified M. sexta proPO in vitro [28, 29]. These data indicate that, based on the canonical model of proPO activation cascades, melanization in An. gambiae depends on at least two such cascades, characterized by CLIPB9 and B10, respectively. Based on genetic and biochemical data, CLIPB8 is located further upstream of CLIPB9 [30]. All three CLIPB proteases required for proteolytic activation of proPO are inhibited by SRPNs. Of the 18 SRPNs annotated in the An. gambiae genome, only SRPN2 has been shown to control systemic melanization [7, 31]. It effectively inhibits the two known PAPs (CLIPB9 and B10) and also CLIPB8, albeit to a lesser degree [28–30]. Several additional CLIPBs, including CLIPB3, 4, 14, and 17 also contribute to melanization; however, the molecular function and placement of these additional CLIPBs in proPO activation cascades are unknown [32, 33]. The only cSP in the CLIPC family to contribute to melanization is CLIPC9, presumably through the action of cleavage of either PAP, which however remains to be demonstrated [34].

In addition to cSPs, melanization in An. gambiae is regulated positively by a core module of cSPHs, consisting, in hierarchical order, of SPCLIP1, CLIPA8, and CLIPA28. This module is located genetically upstream of CLIPC9 and downstream of the thioester-containing protein (TEP1) [6, 32, 34–36]. Melanization is also regulated negatively by cSPHs, including CLIPA2 and A14 [32, 37]. While the precise mechanism(s) by which An. gambiae cSPHs regulate melanization (positively or negatively) are unknown, observations in An. gambiae suggest that cSPHs contribute to the opsonization of microbial surfaces by TEP1 and the leucine-rich repeat proteins (LRIM) LRIM1 and APLC1 [35, 38] and, based on data from other insect species, to PO activity by mediating proPO cleavage by PAPs [19, 39, 40]. In addition, melanization of rodent and human malaria parasites in An. gambiae is inhibited by two C-type lectins (CTL), CTL4 and CTLMA2 [41, 42], which presumably bind to sugar moieties on the parasite surface and prevent melanization locally. The rodent malaria parasite melanization phenotype induced by CTL4 kd has been used successfully to identify some of the cSPs and cSPHs mentioned above (e.g., [32]).

However, despite the significant number of studies summarized above, our understanding of cSP cascades that control melanization in An. gambiae remains incomplete, and more than 80% of the 110 annotated CLIPs await experimental exploration. To guide the experimental analysis of An. gambiae cSPs and to identify putative key regulators of the melanization response, we describe here a novel systems biology approach to study melanization in insects, combining transcriptomic network and proteomic analyses. Using this approach, we identify the cSP CLIPB4 as a central node in the protease network that regulates melanization in An. gambiae and characterize its placement in the proPO activation cascade using a combination of genetic and biochemical approaches.

Materials and MethodsMosquito CulturesThe An. gambiae G3 strain was obtained from the Malaria Research and Reference Reagent Resource Center (MR4) at CDC and reared as described previously [29]. For egg production, females were provided a blood meal of commercially heparinized horse blood (Plasvacc, USA) using parafilm as a membrane stretched across glass feeders connected to a 37°C circulating water bath.
Anopheles gambiae Melanization Gene Co-Expression NetworkThe Anopheles gambiae melanization gene co-expression network (AgMelGCN) was constructed according to [43]. To construct the network, genes of interest (to be represented as nodes in AgMelGCN), were selected based on published evidence confirming either (1) their role in melanization experimentally or (2) their phylogenetic membership in protein families known to contribute to melanization in mosquitoes. The genes of interest list contained 191 genes, including 9 prophenoloxidases (PPO), 14 TEPs, 23 CTL, 16 LRIM proteins, 18 SRPNs, 4 putative ModSPs (including SP24D), 20 clip-domain serine protease A family proteins (CLIPAs), 29 CLIPBs, 12 CLIPCs, 13 CLIPDs, and 32 CLIPEs. Their raw expression values were collated from 257 unique physiological conditions assessed in 30 separate transcriptome studies, and z-score conversion was performed for normalization under each condition. The detailed list of conditions and transcriptome studies is available in the study by Kuang et al. [43]. Two genes were considered co-expressed and connected by edges if the Pearson correlation coefficient (PCC) of their expression vectors was above the 3% fitted sliding threshold. PCCs were rescaled based on the reversed curve of the fitted sliding thresholds to assign edge weights. The network was visualized using Gephi 0.9.2 [44] using the Fruchterman-Reingold force-directed algorithm [45]. Network communities were assigned using the Girvan-Newman algorithm [46]. Four measures were applied for node centrality, evaluating the edge abundance connected to a single node: node strength (sum of connected edge weights), eigenvector centrality, betweenness centrality, and harmonic closeness centrality [47]. In addition, s-core decomposition analysis was performed. In the case of a weighted network, the s-core subgraph consists of all nodes i with node strengths s(i) > s, where s is a threshold value. We define the threshold value of the sn-core as s(n- 1) = min s(i), among all nodes i belonging to the s(n-1)-core network. The sn-core is found by the iterative removal of all nodes with strengths s(i) ≤ s(n-1) [48].SRPN2 Co-Immunoprecipitation and Mass Spectrometric AnalysisFemale mosquitoes were challenged by microinjection of 69 nL lyophilized Micrococcus luteus in water (OD600 = 0.55, Sigma-Aldrich) per mosquito, and hemolymph was collected from 420 female mosquitoes into 1× Roche protease inhibitor in PBS by proboscis clipping 19 h post-infection. The rabbit SRPN2 antiserum, produced previously [7, 29], was affinity purified using the pH 7.2 coupling buffer with AminoLink™ Plus Immobilization Kit (Thermo Fisher Scientific), and stored in PBS. Co-immunoprecipitation (Co-IP) was performed with Pierce™ Protein A Magnetic Beads (Thermo Fisher Scientific), and beads were rinsed with the buffer PBSOg (PBS with 0.7% w/v n-Octyl-BD-glucopyranoside). Mosquito hemolymph was equally split for two Co-IP reactions: one with 40 μL 1:1 slurry of PBSOg-containing magnetic beads cross-linked with 30 μg of purified SRPN2 antibody, and the other with 40 μL 1:1 slurry of PBSOg-containing control magnetic beads. Hemolymph was incubated with the beads for 1.5 h at room temperature (RT) with constant rotation. The unbound fractions were removed, and magnetic beads were washed twice with PBSOg followed by four washes with UltraPure™ water (Invitrogen). Bound proteins were eluted into 20 μL 0.1% TFA (w/v, trifluoroacetic acid in water) in each Co-IP. Samples were shipped to the Nevada Proteomics Center (University of Nevada, Reno) for identification via LC-Orbitrap-MS. Contaminants were removed with 2-D Clean-up kit (GE Healthcare), and in-tube trypsin digestion was performed. Resultant peptides were run through a Michrom Paradigm Microdialysis Liquid Chromatography and Michrom CaptiveSpray coupled to a Thermo LTQ Orbitrap XL with electron-transfer dissociation (ETD). MS spectra were analyzed using Sequest (IseNode in Proteome Discoverer version 2.2.0.388, Thermo Fisher Scientific). Sequest searches of ANOGA_uniprot_20210804.fasta (25,565 entries) were performed with the following settings: digestion enzyme trypsin, maximal missed cleavage = 2, fragment ion mass tolerance = 0.60 Da, and a parent ion tolerance = 50 PPM. Carbamidomethyl of cysteine was specified as a fixed modification, while oxidation of methionine, acetylation of the N-terminus, and phosphorylation of serine were specified as variable modifications. Scaffold (version 5.1.0, Proteome Software Inc.) was used to validate peptide and protein identifications. Peptide probability was calculated by the PeptideProphet algorithm [49] with Scaffold delta-mass correction, and a ≥95.0% identification cut-off. Protein probabilities were assigned by the ProteinProphet algorithm [50]. Proteins that contained similar peptides and could not be differentiated based on MS analysis alone were grouped to satisfy the principles of parsimony. In the SRPN2 Co-IP sample, a protein was considered present if protein probability was greater than 99.0%, and 2 minimally unique peptides were identified in the sequence. A protein was removed from the SRPN2 Co-IP list if at least one minimal peptide was detected in the control sample. Proteome Discoverer Sequest HT scores were listed at the protein level to rank the final identification list.Expression and Purification of Recombinant Proteins proCLIPB4 and proCLIPB4XaTo produce recombinant proCLIPB4, its full coding region, including the signal peptide, was amplified by PCR using gene-specific primers (online suppl. Table S1; for all online suppl. material, see https://doi.org/10.1159/000533898) and An. gambiae adult cDNA as template. The forward primer contained a NotI site at 5′-end, and the reverse primer included three glycine codons, six histidine codons, a stop codon, and a HindIII site at the 3′-end. The PCR product was digested by NotI and HindIII and inserted into the same restriction sites in the transfer vector pFastBac1 (Invitrogen). The resulting proCLIPB4-6His-pFastBac1 plasmid was used as a template to produce the mutant proCLIPB4Xa-6His-pFastBac1. To utilize commercial factor Xa (New England Biolabs), the predicted activation site LTDR107 of proCLIPB4 was replaced with IEGR107. Site-directed mutagenesis was introduced to proCLIPB4-6His-pFastBac1 by Vent DNA polymerase (New England Biolabs) with mutagenic primers (online suppl. Table S1), DpnI (New England Biolabs) digestion and NEB10β transformation. The resulting proCLIPB4Xa-6His-pFastBac1 was confirmed by Sanger sequencing. The DNA segments containing proCLIPB4-6His and proCLIPB4Xa-6His, respectively, were transferred from pFastBac1 to pOET3 (Oxford Expression Technologies) by BamHI and HindIII sites, creating proCLIPB4-6His-pOET3 and proCLIPB4Xa-6His-pOET3, respectively. The pFastBac1 and pOET3 transfer vectors were used to produce corresponding CLIPB4 baculoviruses using the Sf9 cell line by means of Bac-to-Bac system (Invitrogen) or flashBAC-Baculovirus Expression system (Genway Biotech). Large-scale expression was performed using Sf9 suspension cultures at 2 × 106 cells/mL cell density in Sf-900 II serum-free medium (Invitrogen). Baculoviruses were inoculated at multiplicity of infection of 1, and expression media were harvested 48 h postinfection. Recombinant proteins were purified as described previously [28–30]. In brief, cells were cleared by centrifugation, and expression medium was dialyzed and subjected to column chromatography with Ni-NTA agarose (Qiagen) and Q Sepharose Fast Flow (Cytiva). Eluted fractions were screened by immunoblot with THE™ His Tag Antibody (for antibody source and dilution, see online suppl. Table S2). Combined fractions were concentrated with buffer exchange (20 mm Tris, 50 mm NaCl, pH 8.0) in Amicon™ Ultra-10K centrifugal units. Recombinant proteins were stored at −80°C until future use. Recombinant proCLIPB8Xa, proCLIPB8, proCLIPB9Xa, proCLIPB9, and SRPN2 were expressed and purified as described previously [29, 30].SDS-PAGE and ImmunoblotProtein samples were mixed with SDS sample buffer containing β-mercaptoethanol and denatured at 95°C for 5 min. Proteins were separated on SDS-PAGE gels (Bio-Rad) with constant voltage of 120 V followed by 180 V. SDS-PAGE gels were stained with EZ-Run Protein Gel Staining Solution (Fisher) and de-stained with Milli-Q water. For immunoblot, proteins were transferred onto nitrocellulose membranes (GE Healthcare) at 10 V constant voltage using Owl™ Semidry Electroblotting Systems (Thermo Fisher Scientific). Membranes were incubated with 5% dry milk in 1× TBST (0.05% Tween-20) at RT for 1 h, followed by primary antibody incubation diluted in 0.5% dry milk in 1× TBST (4°C overnight or RT for 2 h). All antibodies used in this study, including their origin and dilutions used for immunoblots are listed in online supplementary Table S2. Blots were washed 3 times with 1× TBST for 20 min each at RT. Membranes were further incubated with diluted secondary antibodies in 0.5% dry milk and 1× TBST at RT for 1 h. Protein signals were developed by Alkaline Phosphatase Conjugate Substrate Kit (Bio-Rad) and Amersham™ ECL™ Western Blotting Detection Reagents (GE Healthcare) for alkaline phosphatase-conjugated and HRP-conjugated secondary antibodies, respectively. Images were captured by CCD camera of Azure Biosystems 300 for ECL development.Factor Xa Activation of IEGR-Mutated Recombinant ProteinsFactor Xa activation of recombinant proCLIPB4Xa is described in the legend of online supplementary Figure S1. Activation of proCLIPB8Xa and proCLIPB9Xa was performed as described previously [29, 30].SRPN2 Inhibition of CLIPB4To produce active CLIPB4, 630 ng of recombinant proCLIPB4Xa was incubated with 1 μg of factor Xa in a total volume of 11 μL with factor Xa activation buffer at 37°C overnight. Two negative controls were set up in parallel, where proCLIPB4Xa or factor Xa was replaced with the same volume of factor Xa activation buffer. Activated CLIPB4Xa (154 ng/per reaction) was mixed with recombinant (r) SRPN2 at molar ratios of 1:1 or 1:10 (proCLIPB4Xa:SRPN2). After incubation at RT for 15 min, the reaction mixtures were subjected to immunoblot analysis with mouse anti-His, rabbit anti-SRPN2, or rabbit anti-CLIPB4 as primary antibodies (online suppl. Table S2). For mass spectrometry identification, reactions were set up as described above, and subjected to SDS-PAGE. After Coomassie brilliant blue staining, the band of interest (∼72 kDa) was excised. In-gel trypsin digestion and electrospray ionization mass spectrometry (Bruker Daltonics HCT Ultra) were performed at the Biotechnology/Proteomics Core Facility (Kansas State University). MS results were analyzed in Scaffold 4.0. To determine the stoichiometry of inhibition, 90 ng of factor Xa-activated rCLIPB4Xa was mixed with rSRPN2 at molar ratios of 0:1, 0.5:1, 1:1, 2:1, 4:1, and 6:1 (rSRPN2:rCLIPB4Xa) in reaction buffer 20 mm Tris, 150 mm NaCl, pH 8.0 supplemented with 3 μg BSA. Reactions were incubated at RT for 15 min. The residual amidase activity was determined with IEARpNA substrate at OD405 as described above. IEARase activity of factor Xa was measured in parallel and subtracted from the above reactions of rSRPN2:rCLIPB4Xa. Amidase activity at molar ratio of 0:1 (rSRPN2:rCLIPB4Xa) was defined as 100%. All assays were performed in triplicate.Purification of Prophenoloxidase 1 and 2 from M. sexta Larval HemolymphProphenoloxidase 1 and 2 (MsPPO1&2) were purified from M. sexta hemolymph similarly to published methodology [51]. Briefly, hemolymph was collected into 100% ammonium sulfate (AS) at 0 °C from 5th instar day 2 M sexta larvae. Fraction of 38–48% AS precipitation was dissolved in buffer A (10 mm KPi, pH 6.8, 500 mm NaCl, 0.5 mm reduced l-glutathione), and dialyzed against buffer A. Bio-Gel HT Hydroxyapatite column (Bio-Rad) was utilized to separate proteins via a linear gradient of 10–100 mm KPi in buffer A. Two microliters per fraction were assayed with 200 μL 2 mm dopamine hydrochloride in 50 mm NaPi, pH 6.5 with and without 0.1% cetylpyridinium chloride (CPC, w/v) in 96-well plates, and absorbance change was monitored at 470 nm in an EPOCH 2 microplate reader (BioTek). Fractions containing CPC-activated PO activity were combined, and concentrations of MgCl2 and CaCl2 were adjusted to 1 mm each. The protein solution was applied twice through a Concanavalin A Sepharose 4B column (GE Healthcare), equilibrated in 20 mm Tris, pH 7.4, 0.5 m NaCl, 1 mm MgCl2, 1 mm CaCl2. Column flow-through fraction was adjusted to 1 m AS and loaded onto a Phenyl Sepharose 6 Fast Flow low sub column (Cytiva), equilibrated in 0.1 m KPi, pH 7.1, 1 m AS. Elution was performed with a descending linear gradient of 0.1 m KPi, pH 7.1, 1 m AS to 0.01 m KPi, pH 7.1. Fractions with CPC-activated PO activity were pooled and concentrated to 1.1 mL using Amicon™ Ultra-15 30K centrifugal units (Merck Millipore). Proteins were separated by Sephacryl S-300 column (Cytiva), equilibrated in 10 mm MOPS, pH 7.2, 0.5 m NaCl. Fractions containing CPC-activated PO activity were pooled and concentrated to 4.6 mg/mL by Amicon™ Ultra-15 30K units. Equal volume of glycerol was mixed with purified MsPPO1&2, and the purified zymogen was stored at −80°C at 2.3 mg/mL in 5 mm MOPS, pH 7.2, 0.25 m NaCl, 50% glycerol.ProPO Activation by CLIPB4Xa Using M. sexta Plasma and Purified MsPPO1&2To evaluate the contribution of CLIPB4 on proPO activity, M. sexta plasma and purified MsPPO1&2 were utilized as described previously [29]. Naïve hemolymph was collected from a cut proleg of 5th instar day 3 M sexta larvae, and centrifuged at 5,000 g, 4°C for 5 min to remove hemocytes and cell debris. Factor Xa was used to activate proCLIPB4Xa as described in online supplementary Figure S1. For PO enzymatic activity plate assay, in each well, 1 μL plasma was mixed with 3 μL factor Xa-proCLIPB4Xa activation mixture per well, supplemented to 10 μL with buffer B (20 mm Tris, 20 mm NaCl, 6.7 mm CaCl2, pH 7.5). Three negative controls were set up, replacing factor Xa, proCLIPB4Xa, and both with buffer B, respectively, for a final reaction volume of 10 μL. Three technical replicates were set up per reaction, and incubated at RT for 15 min. Per well, 200 μL 2 mm dopamine hydrochloride was added in the buffer 50 mm NaPi, pH 6.5. Absorbance was monitored at 470 nm at RT for 1 h. For PO activity assay with purified MsPPO1&2, per well, 1 μg purified MsPPO1&2 (4 μL at 250 ng/μL) was mixed with 5 μL of activation mixture and 3 μL of buffer B for a total reaction volume of 12 μL. One additional negative control was included by mixing 1 μg purified MsPPO1&2 with 8 μL buffer B. Three technical replicates were set up for each reaction and incubated at RT for 1 h. Dopamine hydrochloride was added as described above, and absorbance was monitored at 470 nm at RT for 1 h. One unit of PO activity was defined as ΔA470/min = 0.001.ProPO Cleavage by CLIPB4Xa Using M. sexta Plasma and Purified MsPPO1&2To detect cleavage of MsPPO1&2, we performed immunoblots with rabbit anti-MsPPO1&2 primary antibody (online suppl. Table S2). Naïve M. sexta plasma was first diluted to 1:20 in 20 mm Tris, 150 mm NaCl, pH 8.0. Reactions were set up by mixing 1 μL diluted plasma, 3 μL factor Xa activation buffer or 3 μL each from factor Xa-proCLIPB4Xa RT activation mixtures (including negative controls), and 6 μL buffer B for a 10 μL total reaction volume. Three additional samples were prepared by mixing 3 μL each from factor Xa-proCLIPB4Xa RT activation mixtures and 7 μL buffer B. Reactions were incubated at RT for 20 min. For purified MsPPO1&2, reactions were set up by mixing 2 μL purified MsPPO1&2 (250 ng/μL, 500 ng) with 6 μL factor Xa activation buffer or 6 μL each from factor Xa-proCLIPB4Xa activation mixtures (see online suppl. Fig. 1). Reactions were incubated at RT for 20 min. For immunoblot, 1.2 μL of each reaction (containing 75 ng of MsPPO1&2) was mixed with 8.8 μL water and boiled at 95°C for 5 min with SDS sample buffer. Samples were separated on 7.5% SDS-PAGE gels and probed with MsPPO1&2 antibody. In addition, CLIPB9Xa-MsPPO1&2 reactions were included as a positive control. Briefly, to obtain activated CLIPB9Xa, 2.5 μg of purified recombinant proCLIPB9Xa was mixed with 1 μg factor Xa, and factor Xa activation buffer in a total reaction volume of 25 μL and incubated at 37°C overnight. Two negative control reactions were set up and incubated in parallel, in which recombinant proCLIPB9Xa and factor Xa were replaced with the same volume of factor Xa activation buffer, respectively.CLIPB4 Activation of proCLIPB8 and proCLIPB9To test whether CLIPB4 cleaves recombinant proCLIPB8 and/or proCLIPB9, proCLIPB4Xa was activated by factor Xa at 37°C. 154 ng of factor Xa-activated CLIPB4Xa was incubated with 100 ng proCLIPB8 or 100 ng proCLIPB9 at RT for 15 min. Recombinant proCLIPB8Xa or proCLIPB9Xa was activated by factor Xa similarly as described previously [29, 30], and 100 ng of activated CLIPB8Xa or CLIPB9Xa was used as a positive control. Samples were subjected to SDS-PAGE and immunoblot using mouse anti-His, rabbit anti-CLIPB8, or rabbit anti-CLIPB9 antibodies (online suppl. Table S2). To measure the enzymatic activity of cleaved CLIPB8, 610 ng of recombinant proCLIPB4Xa (102 ng/μL) was incubated at 37°C for 10 h with 1 μg (1 μg/µL) factor Xa, supplemented to 21 μL with factor Xa activation buffer. 72 ng of activated CLIPB4Xa was incubated with 450 ng proCLIPB8 (225 ng/μL) at RT for 1 h. Reactions were supplemented with 200 μL of 100 μm N-benzoyl-Phe-Val-Arg-p-nitroanilide (FVR-pNA, EMD Chemicals) in 0.1 m Tris-HCl, pH 8.0, 0.1 m NaCl, 5 mm CaCl2, and amidase activity was measured continuously at OD405 for 30 min. 450 ng of activated CLIPB8Xa was used as a positive control. One unit of CLIPB8 activity was defined as ΔA405/min = 0.001. All activity assays were performed in triplicate.Double-Stranded RNA Synthesis and RNAi ExperimentsDouble-Stranded RNA (dsRNA) synthesis and adult female mosquito injections were performed as described previously [29]. Primers are listed in online supplementary Table S1 for T7-tagged gene-specific amplification of CLIPB4. Primers were adapted from previous studies for dsRNA synthesis of SRPN2, GFP, CLIPB8, and LacZ [29, 30, 52]. In single gene kd experiments, 3–4-day-old adult female mosquitoes were injected each with 207 ng of dsRNA in a total volume of 69 nL. For kd control, mosquitoes were injected with the same quantity of the non-related dsRNA of either Green Fluorescent Protein (dsGFP) or β-galactosidase (dsLacZ). For double knockdowns (dkd), mosquitoes were injected with 138 nL of a 1:1 dsRNA mixture containing 1.5 μg/µL of each dsRNA. For dkd controls, dsGFP was added to compensate the total dsRNA dose of 414 ng/mosquito.Melanization Phenotype AnalysisTo assess CLIPB4’s impact on melanization, two assays were utilized, melanization-associated spot assay (MelASA) and SRPN2 depletion-induced melanotic pseudotumor formation. To quantify melanotic excreta, MelASA was performed in the standard format (50 mosquitoes/experimental cup) as described previously [34], with or without bacterial challenge. For MelASAs after bacterial challenge, dsRNA-injected mosquitoes were allowed to recover for 60 h and injected each with 50.6 nL of ×1 PBS, OD600 = 1.5 Escherichia coli (resuspended in 1x PBS, same below), OD600 = 1.5 Staphylococcus aureus, and OD600 = 0.4 M. luteus (Micrococcus lysodeikticus lyophilized cells ATCC, No. 4698, Sigma-Aldrich), respectively. Clean P8-grade white filter papers (Fisher) were inserted into the bottom of the experimental cups after bacterial challenge and removed 12 h later. For MelASAs without bacterial challenge, clean filter papers were inserted immediately after dsRNA injection, and removed 60 h later. All MelASAs were performed with five independent biological replicates using different mosquito generations. Collected filter papers were imaged under white epi-illumination without filters in an Alpha Imager 2200 system (Alpha Innotec). Images were processed and analyzed by Fiji is Just ImageJ (FIJI, Version 2.1.0) as described by Sousa et al. [53] with the following changes. We extended the workflow to include 4 macros, which enable automatic sequential processing and analysis of all images. The first two macros, published in [53], threshold the filter images, the third macro (setOption[“BlackBackground”, true]; run[“Convert to Mask”];) performs a “convert to mask” step, and the fourth and final macro (run[“Analyze Particles...”, “show = Masks display exclude summarize”];) measures “the total spot area” on each filter paper. To quantify melanotic pseudotumor formation, mosquitoes were dissected when mortality reached 70% in the dsSRPN2/dsGFP group. Abdominal walls were prepared as described previously [7] and examined under ×40 magnification with Axio Imager A1 microscope (Zeiss) equipped with AxioCam MR5 (Zeiss). Image processing and abdomen melanized area calculation were performed in ImageJ software as described previously [29, 54]. Five independent biological replicates were performed using 20 mosquitoes per treatment. All statistical analyses were performed using GraphPad Prism (ver. 9.5.1, GraphPad Software).

Materials and Methods

Mosquito CulturesThe An. gambiae G3 strain was obtained from the Malaria Research and Reference Reagent Resource Center (MR4) at CDC and reared as described previously [29]. For egg production, females were provided a blood meal of commercially heparinized horse blood (Plasvacc, USA) using parafilm as a membrane stretched across glass feeders connected to a 37°C circulating water bath.

Mosquito Cultures

The An. gambiae G3 strain was obtained from the Malaria Research and Reference Reagent Resource Center (MR4) at CDC and reared as described previously [29]. For egg production, females were provided a blood meal of commercially heparinized horse blood (Plasvacc, USA) using parafilm as a membrane stretched across glass feeders connected to a 37°C circulating water bath.

Anopheles gambiae Melanization Gene Co-Expression NetworkThe Anopheles gambiae melanization gene co-expression network (AgMelGCN) was constructed according to [43]. To construct the network, genes of interest (to be represented as nodes in AgMelGCN), were selected based on published evidence confirming either (1) their role in melanization experimentally or (2) their phylogenetic membership in protein families known to contribute to melanization in mosquitoes. The genes of interest list contained 191 genes, including 9 prophenoloxidases (PPO), 14 TEPs, 23 CTL, 16 LRIM proteins, 18 SRPNs, 4 putative ModSPs (including SP24D), 20 clip-domain serine protease A family proteins (CLIPAs), 29 CLIPBs, 12 CLIPCs, 13 CLIPDs, and 32 CLIPEs. Their raw expression values were collated from 257 unique physiological conditions assessed in 30 separate transcriptome studies, and z-score conversion was performed for normalization under each condition. The detailed list of conditions and transcriptome studies is available in the study by Kuang et al. [43]. Two genes were considered co-expressed and connected by edges if the Pearson correlation coefficient (PCC) of their expression vectors was above the 3% fitted sliding threshold. PCCs were rescaled based on the reversed curve of the fitted sliding thresholds to assign edge weights. The network was visualized using Gephi 0.9.2 [44] using the Fruchterman-Reingold force-directed algorithm [45]. Network communities were assigned using the Girvan-Newman algorithm [46]. Four measures were applied for node centrality, evaluating the edge abundance connected to a single node: node strength (sum of connected edge weights), eigenvector centrality, betweenness centrality, and harmonic closeness centrality [47]. In addition, s-core decomposition analysis was performed. In the case of a weighted network, the s-core subgraph consists of all nodes i with node strengths s(i) > s, where s is a threshold value. We define the threshold value of the sn-core as s(n- 1) = min s(i), among all nodes i belonging to the s(n-1)-core network. The sn-core is found by the iterative removal of all nodes with strengths s(i) ≤ s(n-1) [48].

Anopheles gambiae Melanization Gene Co-Expression Network

The Anopheles gambiae melanization gene co-expression network (AgMelGCN) was constructed according to [43]. To construct the network, genes of interest (to be represented as nodes in AgMelGCN), were selected based on published evidence confirming either (1) their role in melanization experimentally or (2) their phylogenetic membership in protein families known to contribute to melanization in mosquitoes. The genes of interest list contained 191 genes, including 9 prophenoloxidases (PPO), 14 TEPs, 23 CTL, 16 LRIM proteins, 18 SRPNs, 4 putative ModSPs (including SP24D), 20 clip-domain serine protease A family proteins (CLIPAs), 29 CLIPBs, 12 CLIPCs, 13 CLIPDs, and 32 CLIPEs. Their raw expression values were collated from 257 unique physiological conditions assessed in 30 separate transcriptome studies, and z-score conversion was performed for normalization under each condition. The detailed list of conditions and transcriptome studies is available in the study by Kuang et al. [43]. Two genes were considered co-expressed and connected by edges if the Pearson correlation coefficient (PCC) of their expression vectors was above the 3% fitted sliding threshold. PCCs were rescaled based on the reversed curve of the fitted sliding thresholds to assign edge weights. The network was visualized using Gephi 0.9.2 [44] using the Fruchterman-Reingold force-directed algorithm [45]. Network communities were assigned using the Girvan-Newman algorithm [46]. Four measures were applied for node centrality, evaluating the edge abundance connected to a single node: node strength (sum of connected edge weights), eigenvector centrality, betweenness centrality, and harmonic closeness centrality [47]. In addition, s-core decomposition analysis was performed. In the case of a weighted network, the s-core subgraph consists of all nodes i with node strengths s(i) > s, where s is a threshold value. We define the threshold value of the sn-core as s(n- 1) = min s(i), among all nodes i belonging to the s(n-1)-core network. The sn-core is found by the iterative removal of all nodes with strengths s(i) ≤ s(n-1) [48].

SRPN2 Co-Immunoprecipitation and Mass Spectrometric AnalysisFemale mosquitoes were challenged by microinjection of 69 nL lyophilized Micrococcus luteus in water (OD600 = 0.55, Sigma-Aldrich) per mosquito, and hemolymph was collected from 420 female mosquitoes into 1× Roche protease inhibitor in PBS by proboscis clipping 19 h post-infection. The rabbit SRPN2 antiserum, produced previously [7, 29], was affinity purified using the pH 7.2 coupling buffer with AminoLink™ Plus Immobilization Kit (Thermo Fisher Scientific), and stored in PBS. Co-immunoprecipitation (Co-IP) was performed with Pierce™ Protein A Magnetic Beads (Thermo Fisher Scientific), and beads were rinsed with the buffer PBSOg (PBS with 0.7% w/v n-Octyl-BD-glucopyranoside). Mosquito hemolymph was equally split for two Co-IP reactions: one with 40 μL 1:1 slurry of PBSOg-containing magnetic beads cross-linked with 30 μg of purified SRPN2 antibody, and the other with 40 μL 1:1 slurry of PBSOg-containing control magnetic beads. Hemolymph was incubated with the beads for 1.5 h at room temperature (RT) with constant rotation. The unbound fractions were removed, and magnetic beads were washed twice with PBSOg followed by four washes with UltraPure™ water (Invitrogen). Bound proteins were eluted into 20 μL 0.1% TFA (w/v, trifluoroacetic acid in water) in each Co-IP. Samples were shipped to the Nevada Proteomics Center (University of Nevada, Reno) for identification via LC-Orbitrap-MS. Contaminants were removed with 2-D Clean-up kit (GE Healthcare), and in-tube trypsin digestion was performed. Resultant peptides were run through a Michrom Paradigm Microdialysis Liquid Chromatography and Michrom CaptiveSpray coupled to a Thermo LTQ Orbitrap XL with electron-transfer dissociation (ETD). MS spectra were analyzed using Sequest (IseNode in Proteome Discoverer version 2.2.0.388, Thermo Fisher Scientific). Sequest searches of ANOGA_uniprot_20210804.fasta (25,565 entries) were performed with the following settings: digestion enzyme trypsin, maximal missed cleavage = 2, fragment ion mass tolerance = 0.60 Da, and a parent ion tolerance = 50 PPM. Carbamidomethyl of cysteine was specified as a fixed modification, while oxidation of methionine, acetylation of the N-terminus, and phosphorylation of serine were specified as variable modifications. Scaffold (version 5.1.0, Proteome Software Inc.) was used to validate peptide and protein identifications. Peptide probability was calculated by the PeptideProphet algorithm [49] with Scaffold delta-mass correction, and a ≥95.0% identification cut-off. Protein probabilities were assigned by the ProteinProphet algorithm [50]. Proteins that contained similar peptides and could not be differentiated based on MS analysis alone were grouped to satisfy the principles of parsimony. In the SRPN2 Co-IP sample, a protein was considered present if protein probability was greater than 99.0%, and 2 minimally unique peptides were identified in the sequence. A protein was removed from the SRPN2 Co-IP list if at least one minimal peptide was detected in the control sample. Proteome Discoverer Sequest HT scores were listed at the protein level to rank the final identification list.

SRPN2 Co-Immunoprecipitation and Mass Spectrometric Analysis

Female mosquitoes were challenged by microinjection of 69 nL lyophilized Micrococcus luteus in water (OD600 = 0.55, Sigma-Aldrich) per mosquito, and hemolymph was collected from 420 female mosquitoes into 1× Roche protease inhibitor in PBS by proboscis clipping 19 h post-infection. The rabbit SRPN2 antiserum, produced previously [7, 29], was affinity purified using the pH 7.2 coupling buffer with AminoLink™ Plus Immobilization Kit (Thermo Fisher Scientific), and stored in PBS. Co-immunoprecipitation (Co-IP) was performed with Pierce™ Protein A Magnetic Beads (Thermo Fisher Scientific), and beads were rinsed with the buffer PBSOg (PBS with 0.7% w/v n-Octyl-BD-glucopyranoside). Mosquito hemolymph was equally split for two Co-IP reactions: one with 40 μL 1:1 slurry of PBSOg-containing magnetic beads cross-linked with 30 μg of purified SRPN2 antibody, and the other with 40 μL 1:1 slurry of PBSOg-containing control magnetic beads. Hemolymph was incubated with the beads for 1.5 h at room temperature (RT) with constant rotation. The unbound fractions were removed, and magnetic beads were washed twice with PBSOg followed by four washes with UltraPure™ water (Invitrogen). Bound proteins were eluted into 20 μL 0.1% TFA (w/v, trifluoroacetic acid in water) in each Co-IP. Samples were shipped to the Nevada Proteomics Center (University of Nevada, Reno) for identification via LC-Orbitrap-MS. Contaminants were removed with 2-D Clean-up kit (GE Healthcare), and in-tube trypsin digestion was performed. Resultant peptides were run through a Michrom Paradigm Microdialysis Liquid Chromatography and Michrom CaptiveSpray coupled to a Thermo LTQ Orbitrap XL with electron-transfer dissociation (ETD). MS spectra were analyzed using Sequest (IseNode in Proteome Discoverer version 2.2.0.388, Thermo Fisher Scientific). Sequest searches of ANOGA_uniprot_20210804.fasta (25,565 entries) were performed with the following settings: digestion enzyme trypsin, maximal missed cleavage = 2, fragment ion mass tolerance = 0.60 Da, and a parent ion tolerance = 50 PPM. Carbamidomethyl of cysteine was specified as a fixed modification, while oxidation of methionine, acetylation of the N-terminus, and phosphorylation of serine were specified as variable modifications. Scaffold (version 5.1.0, Proteome Software Inc.) was used to validate peptide and protein identifications. Peptide probability was calculated by the PeptideProphet algorithm [49] with Scaffold delta-mass correction, and a ≥95.0% identification cut-off. Protein probabilities were assigned by the ProteinProphet algorithm [50]. Proteins that contained similar peptides and could not be differentiated based on MS analysis alone were grouped to satisfy the principles of parsimony. In the SRPN2 Co-IP sample, a protein was considered present if protein probability was greater than 99.0%, and 2 minimally unique peptides were identified in the sequence. A protein was removed from the SRPN2 Co-IP list if at least one minimal peptide was detected in the control sample. Proteome Discoverer Sequest HT scores were listed at the protein level to rank the final identification list.

Expression and Purification of Recombinant Proteins proCLIPB4 and proCLIPB4XaTo produce recombinant proCLIPB4, its full coding region, including the signal peptide, was amplified by PCR using gene-specific primers (online suppl. Table S1; for all online suppl. material, see https://doi.org/10.1159/000533898) and An. gambiae adult cDNA as template. The forward primer contained a NotI site at 5′-end, and the reverse primer included three glycine codons, six histidine codons, a stop codon, and a HindIII site at the 3′-end. The PCR product was digested by NotI and HindIII and inserted into the same restriction sites in the transfer vector pFastBac1 (Invitrogen). The resulting proCLIPB4-6His-pFastBac1 plasmid was used as a template to produce the mutant proCLIPB4Xa-6His-pFastBac1. To utilize commercial factor Xa (New England Biolabs), the predicted activation site LTDR107 of proCLIPB4 was replaced with IEGR107. Site-directed mutagenesis was introduced to proCLIPB4-6His-pFastBac1 by Vent DNA polymerase (New England Biolabs) with mutagenic primers (online suppl. Table S1), DpnI (New England Biolabs) digestion and NEB10β transformation. The resulting proCLIPB4Xa-6His-pFastBac1 was confirmed by Sanger sequencing. The DNA segments containing proCLIPB4-6His and proCLIPB4Xa-6His, respectively, were transferred from pFastBac1 to pOET3 (Oxford Expression Technologies) by BamHI and HindIII sites, creating proCLIPB4-6His-pOET3 and proCLIPB4Xa-6His-pOET3, respectively. The pFastBac1 and pOET3 transfer vectors were used to produce corresponding CLIPB4 baculoviruses using the Sf9 cell line by means of Bac-to-Bac system (Invitrogen) or flashBAC-Baculovirus Expression system (Genway Biotech). Large-scale expression was performed using Sf9 suspension cultures at 2 × 106 cells/mL cell density in Sf-900 II serum-free medium (Invitrogen). Baculoviruses were inoculated at multiplicity of infection of 1, and expression media were harvested 48 h postinfection. Recombinant proteins were purified as described previously [28–30]. In brief, cells were cleared by centrifugation, and expression medium was dialyzed and subjected to column chromatography with Ni-NTA agarose (Qiagen) and Q Sepharose Fast Flow (Cytiva). Eluted fractions were screened by immunoblot with THE™ His Tag Antibody (for antibody source and dilution, see online suppl. Table S2). Combined fractions were concentrated with buffer exchange (20 mm Tris, 50 mm NaCl, pH 8.0) in Amicon™ Ultra-10K centrifugal units. Recombinant proteins were stored at −80°C until future use. Recombinant proCLIPB8Xa, proCLIPB8, proCLIPB9Xa, proCLIPB9, and SRPN2 were expressed and purified as described previously [29, 30].

Expression and Purification of Recombinant Proteins proCLIPB4 and proCLIPB4Xa

To produce recombinant proCLIPB4, its full coding region, including the signal peptide, was amplified by PCR using gene-specific primers (online suppl. Table S1; for all online suppl. material, see https://doi.org/10.1159/000533898) and An. gambiae adult cDNA as template. The forward primer contained a NotI site at 5′-end, and the reverse primer included three glycine codons, six histidine codons, a stop codon, and a HindIII site at the 3′-end. The PCR product was digested by NotI and HindIII and inserted into the same restriction sites in the transfer vector pFastBac1 (Invitrogen). The resulting proCLIPB4-6His-pFastBac1 plasmid was used as a template to produce the mutant proCLIPB4Xa-6His-pFastBac1. To utilize commercial factor Xa (New England Biolabs), the predicted activation site LTDR107 of proCLIPB4 was replaced with IEGR107. Site-directed mutagenesis was introduced to proCLIPB4-6His-pFastBac1 by Vent DNA polymerase (New England Biolabs) with mutagenic primers (online suppl. Table S1), DpnI (New England Biolabs) digestion and NEB10β transformation. The resulting proCLIPB4Xa-6His-pFastBac1 was confirmed by Sanger sequencing. The DNA segments containing proCLIPB4-6His and proCLIPB4Xa-6His, respectively, were transferred from pFastBac1 to pOET3 (Oxford Expression Technologies) by BamHI and HindIII sites, creating proCLIPB4-6His-pOET3 and proCLIPB4Xa-6His-pOET3, respectively. The pFastBac1 and pOET3 transfer vectors were used to produce corresponding CLIPB4 baculoviruses using the Sf9 cell line by means of Bac-to-Bac system (Invitrogen) or flashBAC-Baculovirus Expression system (Genway Biotech). Large-scale expression was performed using Sf9 suspension cultures at 2 × 106 cells/mL cell density in Sf-900 II serum-free medium (Invitrogen). Baculoviruses were inoculated at multiplicity of infection of 1, and expression media were harvested 48 h postinfection. Recombinant proteins were purified as described previously [28–30]. In brief, cells were cleared by centrifugation, and expression medium was dialyzed and subjected to column chromatography with Ni-NTA agarose (Qiagen) and Q Sepharose Fast Flow (Cytiva). Eluted fractions were screened by immunoblot with THE™ His Tag Antibody (for antibody source and dilution, see online suppl. Table S2). Combined fractions were concentrated with buffer exchange (20 mm Tris, 50 mm NaCl, pH 8.0) in Amicon™ Ultra-10K centrifugal units. Recombinant proteins were stored at −80°C until future use. Recombinant proCLIPB8Xa, proCLIPB8, proCLIPB9Xa, proCLIPB9, and SRPN2 were expressed and purified as described previously [29, 30].

SDS-PAGE and ImmunoblotProtein samples were mixed with SDS sample buffer containing β-mercaptoethanol and denatured at 95°C for 5 min. Proteins were separated on SDS-PAGE gels (Bio-Rad) with constant voltage of 120 V followed by 180 V. SDS-PAGE gels were stained with EZ-Run Protein Gel Staining Solution (Fisher) and de-stained with Milli-Q water. For immunoblot, proteins were transferred onto nitrocellulose membranes (GE Healthcare) at 10 V constant voltage using Owl™ Semidry Electroblotting Systems (Thermo Fisher Scientific). Membranes were incubated with 5% dry milk in 1× TBST (0.05% Tween-20) at RT for 1 h, followed by primary antibody incubation diluted in 0.5% dry milk in 1× TBST (4°C overnight or RT for 2 h). All antibodies used in this study, including their origin and dilutions used for immunoblots are listed in online supplementary Table S2. Blots were washed 3 times with 1× TBST for 20 min each at RT. Membranes were further incubated with diluted secondary antibodies in 0.5% dry milk and 1× TBST at RT for 1 h. Protein signals were developed by Alkaline Phosphatase Conjugate Substrate Kit (Bio-Rad) and Amersham™ ECL™ Western Blotting Detection Reagents (GE Healthcare) for alkaline phosphatase-conjugated and HRP-conjugated secondary antibodies, respectively. Images were captured by CCD camera of Azure Biosystems 300 for ECL development.

SDS-PAGE and Immunoblot

Protein samples were mixed with SDS sample buffer containing β-mercaptoethanol and denatured at 95°C for 5 min. Proteins were separated on SDS-PAGE gels (Bio-Rad) with constant voltage of 120 V followed by 180 V. SDS-PAGE gels were stained with EZ-Run Protein Gel Staining Solution (Fisher) and de-stained with Milli-Q water. For immunoblot, proteins were transferred onto nitrocellulose membranes (GE Healthcare) at 10 V constant voltage using Owl™ Semidry Electroblotting Systems (Thermo Fisher Scientific). Membranes were incubated with 5% dry milk in 1× TBST (0.05% Tween-20) at RT for 1 h, followed by primary antibody incubation diluted in 0.5% dry milk in 1× TBST (4°C overnight or RT for 2 h). All antibodies used in this study, including their origin and dilutions used for immunoblots are listed in online supplementary Table S2. Blots were washed 3 times with 1× TBST for 20 min each at RT. Membranes were further incubated with diluted secondary antibodies in 0.5% dry milk and 1× TBST at RT for 1 h. Protein signals were developed by Alkaline Phosphatase Conjugate Substrate Kit (Bio-Rad) and Amersham™ ECL™ Western Blotting Detection Reagents (GE Healthcare) for alkaline phosphatase-conjugated and HRP-conjugated secondary antibodies, respectively. Images were captured by CCD camera of Azure Biosystems 300 for ECL development.

Factor Xa Activation of IEGR-Mutated Recombinant ProteinsFactor Xa activation of recombinant proCLIPB4Xa is described in the legend of online supplementary Figure S1. Activation of proCLIPB8Xa and proCLIPB9Xa was performed as described previously [29, 30].

Factor Xa Activation of IEGR-Mutated Recombinant Proteins

Factor Xa activation of recombinant proCLIPB4Xa is described in the legend of online supplementary Figure S1. Activation of proCLIPB8Xa and proCLIPB9Xa was performed as described previously [29, 30].

SRPN2 Inhibition of CLIPB4To produce active CLIPB4, 630 ng of recombinant proCLIPB4Xa was incubated with 1 μg of factor Xa in a total volume of 11 μL with factor Xa activation buffer at 37°C overnight. Two negative controls were set up in parallel, where proCLIPB4Xa or factor Xa was replaced with the same volume of factor Xa activation buffer. Activated CLIPB4Xa (154 ng/per reaction) was mixed with recombinant (r) SRPN2 at molar ratios of 1:1 or 1:10 (proCLIPB4Xa:SRPN2). After incubation at RT for 15 min, the reaction mixtures were subjected to immunoblot analysis with mouse anti-His, rabbit anti-SRPN2, or rabbit anti-CLIPB4 as primary antibodies (online suppl. Table S2). For mass spectrometry identification, reactions were set up as described above, and subjected to SDS-PAGE. After Coomassie brilliant blue staining, the band of interest (∼72 kDa) was excised. In-gel trypsin digestion and electrospray ionization mass spectrometry (Bruker Daltonics HCT Ultra) were performed at the Biotechnology/Proteomics Core Facility (Kansas State University). MS results were analyzed in Scaffold 4.0. To determine the stoichiometry of inhibition, 90 ng of factor Xa-activated rCLIPB4Xa was mixed with rSRPN2 at molar ratios of 0:1, 0.5:1, 1:1, 2:1, 4:1, and 6:1 (rSRPN2:rCLIPB4Xa) in reaction buffer 20 mm Tris, 150 mm NaCl, pH 8.0 supplemented with 3 μg BSA. Reactions were incubated at RT for 15 min. The residual amidase activity was determined with IEARpNA substrate at OD405 as described above. IEARase activity of factor Xa was measured in parallel and subtracted from the above reactions of rSRPN2:rCLIPB4Xa. Amidase activity at molar ratio of 0:1 (rSRPN2:rCLIPB4Xa) was defined as 100%. All assays were performed in triplicate.

SRPN2 Inhibition of CLIPB4

To produce active CLIPB4, 630 ng of recombinant proCLIPB4Xa was incubated with 1 μg of factor Xa in a total volume of 11 μL with factor Xa activation buffer at 37°C overnight. Two negative controls were set up in parallel, where proCLIPB4Xa or factor Xa was replaced with the same volume of factor Xa activation buffer. Activated CLIPB4Xa (154 ng/per reaction) was mixed with recombinant (r) SRPN2 at molar ratios of 1:1 or 1:10 (proCLIPB4Xa:SRPN2). After incubation at RT for 15 min, the reaction mixtures were subjected to immunoblot analysis with mouse anti-His, rabbit anti-SRPN2, or rabbit anti-CLIPB4 as primary antibodies (online suppl. Table S2). For mass spectrometry identification, reactions were set up as described above, and subjected to SDS-PAGE. After Coomassie brilliant blue staining, the band of interest (∼72 kDa) was excised. In-gel trypsin digestion and electrospray ionization mass spectrometry (Bruker Daltonics HCT Ultra) were performed at the Biotechnology/Proteomics Core Facility (Kansas State University). MS results were analyzed in Scaffold 4.0. To determine the stoichiometry of inhibition, 90 ng of factor Xa-activated rCLIPB4Xa was mixed with rSRPN2 at molar ratios of 0:1, 0.5:1, 1:1, 2:1, 4:1, and 6:1 (rSRPN2:rCLIPB4Xa) in reaction buffer 20 mm Tris, 150 mm NaCl, pH 8.0 supplemented with 3 μg BSA. Reactions were incubated at RT for 15 min. The residual amidase activity was determined with IEARpNA substrate at OD405 as described above. IEARase activity of factor Xa was measured in parallel and subtracted from the above reactions of rSRPN2:rCLIPB4Xa. Amidase activity at molar ratio of 0:1 (rSRPN2:rCLIPB4Xa) was defined as 100%. All assays were performed in triplicate.

Purification of Prophenoloxidase 1 and 2 from M. sexta Larval HemolymphProphenoloxidase 1 and 2 (MsPPO1&2) were purified from M. sexta hemolymph similarly to published methodology [51]. Briefly, hemolymph was collected into 100% ammonium sulfate (AS) at 0 °C from 5th instar day 2 M sexta larvae. Fraction of 38–48% AS precipitation was dissolved in buffer A (10 mm KPi, pH 6.8, 500 mm NaCl, 0.5 mm reduced l-glutathione), and dialyzed against buffer A. Bio-Gel HT Hydroxyapatite column (Bio-Rad) was utilized to separate proteins via a linear gradient of 10–100 mm KPi in buffer A. Two microliters per fraction were assayed with 200 μL 2 mm dopamine hydrochloride in 50 mm NaPi, pH 6.5 with and without 0.1% cetylpyridinium chloride (CPC, w/v) in 96-well plates, and absorbance change was monitored at 470 nm in an EPOCH 2 microplate reader (BioTek). Fractions containing CPC-activated PO activity were combined, and concentrations of MgCl2 and CaCl2 were adjusted to 1 mm each. The protein solution was applied twice through a Concanavalin A Sepharose 4B column (GE Healthcare), equilibrated in 20 mm Tris, pH 7.4, 0.5 m NaCl, 1 mm MgCl2, 1 mm CaCl2. Column flow-through fraction was adjusted to 1 m AS and loaded onto a Phenyl Sepharose 6 Fast Flow low sub column (Cytiva), equilibrated in 0.1 m KPi, pH 7.1, 1 m AS. Elution was performed with a descending linear gradient of 0.1 m KPi, pH 7.1, 1 m AS to 0.01 m KPi, pH 7.1. Fractions with CPC-activated PO activity were pooled and concentrated to 1.1 mL using Amicon™ Ultra-15 30K centrifugal units (Merck Millipore). Proteins were separated by Sephacryl S-300 column (Cytiva), equilibrated in 10 mm MOPS, pH 7.2, 0.5 m NaCl. Fractions containing CPC-activated PO activity were pooled and concentrated to 4.6 mg/mL by Amicon™ Ultra-15 30K units. Equal volume of glycerol was mixed with purified MsPPO1&2, and the purified zymogen was stored at −80°C at 2.3 mg/mL in 5 mm MOPS, pH 7.2, 0.25 m NaCl, 50% glycerol.

Purification of Prophenoloxidase 1 and 2 from M. sexta Larval Hemolymph

Prophenoloxidase 1 and 2 (MsPPO1&2) were purified from M. sexta hemolymph similarly to published methodology [51]. Briefly, hemolymph was collected into 100% ammonium sulfate (AS) at 0 °C from 5th instar day 2 M sexta larvae. Fraction of 38–48% AS precipitation was dissolved in buffer A (10 mm KPi, pH 6.8, 500 mm NaCl, 0.5 mm reduced l-glutathione), and dialyzed against buffer A. Bio-Gel HT Hydroxyapatite column (Bio-Rad) was utilized to separate proteins via a linear gradient of 10–100 mm KPi in buffer A. Two microliters per fraction were assayed with 200 μL 2 mm dopamine hydrochloride in 50 mm NaPi, pH 6.5 with and without 0.1% cetylpyridinium chloride (CPC, w/v) in 96-well plates, and absorbance change was monitored at 470 nm in an EPOCH 2 microplate reader (BioTek). Fractions containing CPC-activated PO activity were combined, and concentrations of MgCl2 and CaCl2 were adjusted to 1 mm each. The protein solution was applied twice through a Concanavalin A Sepharose 4B column (GE Healthcare), equilibrated in 20 mm Tris, pH 7.4, 0.5 m NaCl, 1 mm MgCl2, 1 mm CaCl2. Column flow-through fraction was adjusted to 1 m AS and loaded onto a Phenyl Sepharose 6 Fast Flow low sub column (Cytiva), equilibrated in 0.1 m KPi, pH 7.1, 1 m AS. Elution was performed with a descending linear gradient of 0.1 m KPi, pH 7.1, 1 m AS to 0.01 m KPi, pH 7.1. Fractions with CPC-activated PO activity were pooled and concentrated to 1.1 mL using Amicon™ Ultra-15 30K centrifugal units (Merck Millipore). Proteins were separated by Sephacryl S-300 column (Cytiva), equilibrated in 10 mm MOPS, pH 7.2, 0.5 m NaCl. Fractions containing CPC-activated PO activity were pooled and concentrated to 4.6 mg/mL by Amicon™ Ultra-15 30K units. Equal volume of glycerol was mixed with purified MsPPO1&2, and the purified zymogen was stored at −80°C at 2.3 mg/mL in 5 mm MOPS, pH 7.2, 0.25 m NaCl, 50% glycerol.

ProPO Activation by CLIPB4Xa Using M. sexta Plasma and Purified MsPPO1&2To evaluate the contribution of CLIPB4 on proPO activity, M. sexta plasma and purified MsPPO1&2 were utilized as described previously [29]. Naïve hemolymph was collected from a cut proleg of 5th instar day 3 M sexta larvae, and centrifuged at 5,000 g, 4°C for 5 min to remove hemocytes and cell debris. Factor Xa was used to activate proCLIPB4Xa as described in online supplementary Figure S1. For PO enzymatic activity plate assay, in each well, 1 μL plasma was mixed with 3 μL factor Xa-proCLIPB4Xa activation mixture per well, supplemented to 10 μL with buffer B (20 mm Tris, 20 mm NaCl, 6.7 mm CaCl2, pH 7.5). Three negative controls were set up, replacing factor Xa, proCLIPB4Xa, and both with buffer B, respectively, for a final reaction volume of 10 μL. Three technical replicates were set up per reaction, and incubated at RT for 15 min. Per well, 200 μL 2 mm dopamine hydrochloride was added in the buffer 50 mm NaPi, pH 6.5. Absorbance was monitored at 470 nm at RT for 1 h. For PO activity assay with purified MsPPO1&2, per well, 1 μg purified MsPPO1&2 (4 μL at 250 ng/μL) was mixed with 5 μL of activation mixture and 3 μL of buffer B for a total reaction volume of 12 μL. One additional negative control was included by mixing 1 μg purified MsPPO1&2 with 8 μL buffer B. Three technical replicates were set up for each reaction and incubated at RT for 1 h. Dopamine hydrochloride was added as described above, and absorbance was monitored at 470 nm at RT for 1 h. One unit of PO activity was defined as ΔA470/min = 0.001.

ProPO Activation by CLIPB4Xa Using M. sexta Plasma and Purified MsPPO1&2

To evaluate the contribution of CLIPB4 on proPO activity, M. sexta plasma and purified MsPPO1&2 were utilized as described previously [29]. Naïve hemolymph was collected from a cut proleg of 5th instar day 3 M sexta larvae, and centrifuged at 5,000 g, 4°C for 5 min to remove hemocytes and cell debris. Factor Xa was used to activate proCLIPB4Xa as described in online supplementary Figure S1. For PO enzymatic activity plate assay, in each well, 1 μL plasma was mixed with 3 μL factor Xa-proCLIPB4Xa activation mixture per well, supplemented to 10 μL with buffer B (20 mm Tris, 20 mm NaCl, 6.7 mm CaCl2, pH 7.5). Three negative controls were set up, replacing factor Xa, proCLIPB4Xa, and both with buffer B, respectively, for a final reaction volume of 10 μL. Three technical replicates were set up per reaction, and incubated at RT for 15 min. Per well, 200 μL 2 mm dopamine hydrochloride was added in the buffer 50 mm NaPi, pH 6.5. Absorbance was monitored at 470 nm at RT for 1 h. For PO activity assay with purified MsPPO1&2, per well, 1 μg purified MsPPO1&2 (4 μL at 250 ng/μL) was mixed with 5 μL of activation mixture and 3 μL of buffer B for a total reaction volume of 12 μL. One additional negative control was included by mixing 1 μg purified MsPPO1&2 with 8 μL buffer B. Three technical replicates were set up for each reaction and incubated at RT for 1 h. Dopamine hydrochloride was added as described above, and absorbance was monitored at 470 nm at RT for 1 h. One unit of PO activity was defined as ΔA470/min = 0.001.

ProPO Cleavage by CLIPB4Xa Using M. sexta Plasma and Purified MsPPO1&2To detect cleavage of MsPPO1&2, we performed immunoblots with rabbit anti-MsPPO1&2 primary antibody (online suppl. Table S2). Naïve M. sexta plasma was first diluted to 1:20 in 20 mm Tris, 150 mm NaCl, pH 8.0. Reactions were set up by mixing 1 μL diluted plasma, 3 μL factor Xa activation buffer or 3 μL each from factor Xa-proCLIPB4Xa RT activation mixtures (including negative controls), and 6 μL buffer B for a 10 μL total reaction volume. Three additional samples were prepared by mixing 3 μL each from factor Xa-proCLIPB4Xa RT activation mixtures and 7 μL buffer B. Reactions were incubated at RT for 20 min. For purified MsPPO1&2, reactions were set up by mixing 2 μL purified MsPPO1&2 (250 ng/μL, 500 ng) with 6 μL factor Xa activation buffer or 6 μL each from factor Xa-proCLIPB4Xa activation mixtures (see online suppl. Fig. 1). Reactions were incubated at RT for 20 min. For immunoblot, 1.2 μL of each reaction (containing 75 ng of MsPPO1&2) was mixed with 8.8 μL water and boiled at 95°C for 5 min with SDS sample buffer. Samples were separated on 7.5% SDS-PAGE gels and probed with MsPPO1&2 antibody. In addition, CLIPB9Xa-MsPPO1&2 reactions were included as a positive control. Briefly, to obtain activated CLIPB9Xa, 2.5 μg of purified recombinant proCLIPB9Xa was mixed with 1 μg factor Xa, and factor Xa activation buffer in a total reaction volume of 25 μL and incubated at 37°C overnight. Two negative control reactions were set up and incubated in parallel, in which recombinant proCLIPB9Xa and factor Xa were replaced with the same volume of factor Xa activation buffer, respectively.

ProPO Cleavage by CLIPB4Xa Using M. sexta Plasma and Purified MsPPO1&2

To detect cleavage of MsPPO1&2, we performed immunoblots with rabbit anti-MsPPO1&2 primary antibody (online suppl. Table S2). Naïve M. sexta plasma was first diluted to 1:20 in 20 mm Tris, 150 mm NaCl, pH 8.0. Reactions were set up by mixing 1 μL diluted plasma, 3 μL factor Xa activation buffer or 3 μL each from factor Xa-proCLIPB4Xa RT activation mixtures (including negative controls), and 6 μL buffer B for a 10 μL total reaction volume. Three additional samples were prepared by mixing 3 μL each from factor Xa-proCLIPB4Xa RT activation mixtures and 7 μL buffer B. Reactions were incubated at RT for 20 min. For purified MsPPO1&2, reactions were set up by mixing 2 μL purified MsPPO1&2 (250 ng/μL, 500 ng) with 6 μL factor Xa activation buffer or 6 μL each from factor Xa-proCLIPB4Xa activation mixtures (see online suppl. Fig. 1). Reactions were incubated at RT for 20 min. For immunoblot, 1.2 μL of each reaction (containing 75 ng of MsPPO1&2) was mixed with 8.8 μL water and boiled at 95°C for 5 min with SDS sample buffer. Samples were separated on 7.5% SDS-PAGE gels and probed with MsPPO1&2 antibody. In addition, CLIPB9Xa-MsPPO1&2 reactions were included as a positive control. Briefly, to obtain activated CLIPB9Xa, 2.5 μg of purified recombinant proCLIPB9Xa was mixed with 1 μg factor Xa, and factor Xa activation buffer in a total reaction volume of 25 μL and incubated at 37°C overnight. Two negative control reactions were set up and incubated in parallel, in which recombinant proCLIPB9Xa and factor Xa were replaced with the same volume of factor Xa activation buffer, respectively.

CLIPB4 Activation of proCLIPB8 and proCLIPB9To test whether CLIPB4 cleaves recombinant proCLIPB8 and/or proCLIPB9, proCLIPB4Xa was activated by factor Xa at 37°C. 154 ng of factor Xa-activated CLIPB4Xa was incubated with 100 ng proCLIPB8 or 100 ng proCLIPB9 at RT for 15 min. Recombinant proCLIPB8Xa or proCLIPB9Xa was activated by factor Xa similarly as described previously [29, 30], and 100 ng of activated CLIPB8Xa or CLIPB9Xa was used as a positive control. Samples were subjected to SDS-PAGE and immunoblot using mouse anti-His, rabbit anti-CLIPB8, or rabbit anti-CLIPB9 antibodies (online suppl. Table S2). To measure the enzymatic activity of cleaved CLIPB8, 610 ng of recombinant proCLIPB4Xa (102 ng/μL) was incubated at 37°C for 10 h with 1 μg (1 μg/µL) factor Xa, supplemented to 21 μL with factor Xa activation buffer. 72 ng of activated CLIPB4Xa was incubated with 450 ng proCLIPB8 (225 ng/μL) at RT for 1 h. Reactions were supplemented with 200 μL of 100 μm N-benzoyl-Phe-Val-Arg-p-nitroanilide (FVR-pNA, EMD Chemicals) in 0.1 m Tris-HCl, pH 8.0, 0.1 m NaCl, 5 mm CaCl2, and amidase activity was measured continuously at OD405 for 30 min. 450 ng of activated CLIPB8Xa was used as a positive control. One unit of CLIPB8 activity was defined as ΔA405/min = 0.001. All activity assays were performed in triplicate.

CLIPB4 Activation of proCLIPB8 and proCLIPB9

To test whether CLIPB4 cleaves recombinant proCLIPB8 and/or proCLIPB9, proCLIPB4Xa was activated by factor Xa at 37°C. 154 ng of factor Xa-activated CLIPB4Xa was incubated with 100 ng proCLIPB8 or 100 ng proCLIPB9 at RT for 15 min. Recombinant proCLIPB8Xa or proCLIPB9Xa was activated by factor Xa similarly as described previously [29, 30], and 100 ng of activated CLIPB8Xa or CLIPB9Xa was used as a positive control. Samples were subjected to SDS-PAGE and immunoblot using mouse anti-His, rabbit anti-CLIPB8, or rabbit anti-CLIPB9 antibodies (online suppl. Table S2). To measure the enzymatic activity of cleaved CLIPB8, 610 ng of recombinant proCLIPB4Xa (102 ng/μL) was incubated at 37°C for 10 h with 1 μg (1 μg/µL) factor Xa, supplemented to 21 μL with factor Xa activation buffer. 72 ng of activated CLIPB4Xa was incubated with 450 ng proCLIPB8 (225 ng/μL) at RT for 1 h. Reactions were supplemented with 200 μL of 100 μm N-benzoyl-Phe-Val-Arg-p-nitroanilide (FVR-pNA, EMD Chemicals) in 0.1 m Tris-HCl, pH 8.0, 0.1 m NaCl, 5 mm CaCl2, and amidase activity was measured continuously at OD405 for 30 min. 450 ng of activated CLIPB8Xa was used as a positive control. One unit of CLIPB8 activity was defined as ΔA405/min = 0.001. All activity assays were performed in triplicate.

Double-Stranded RNA Synthesis and RNAi ExperimentsDouble-Stranded RNA (dsRNA) synthesis and adult female mosquito injections were performed as described previously [29]. Primers are listed in online supplementary Table S1 for T7-tagged gene-specific amplification of CLIPB4. Primers were adapted from previous studies for dsRNA synthesis of SRPN2, GFP, CLIPB8, and LacZ [29, 30, 52]. In single gene kd experiments, 3–4-day-old adult female mosquitoes were injected each with 207 ng of dsRNA in a total volume of 69 nL. For kd control, mosquitoes were injected with the same quantity of the non-related dsRNA of either Green Fluorescent Protein (dsGFP) or β-galactosidase (dsLacZ). For double knockdowns (dkd), mosquitoes were injected with 138 nL of a 1:1 dsRNA mixture containing 1.5 μg/µL of each dsRNA. For dkd controls, dsGFP was added to compensate the total dsRNA dose of 414 ng/mosquito.

Double-Stranded RNA Synthesis and RNAi Experiments

Double-Stranded RNA (dsRNA) synthesis and adult female mosquito injections were performed as described previously [29]. Primers are listed in online supplementary Table S1 for T7-tagged gene-specific amplification of CLIPB4. Primers were adapted from previous studies for dsRNA synthesis of SRPN2, GFP, CLIPB8, and LacZ [29, 30, 52]. In single gene kd experiments, 3–4-day-old adult female mosquitoes were injected each with 207 ng of dsRNA in a total volume of 69 nL. For kd control, mosquitoes were injected with the same quantity of the non-related dsRNA of either Green Fluorescent Protein (dsGFP) or β-galactosidase (dsLacZ). For double knockdowns (dkd), mosquitoes were injected with 138 nL of a 1:1 dsRNA mixture containing 1.5 μg/µL of each dsRNA. For dkd controls, dsGFP was added to compensate the total dsRNA dose of 414 ng/mosquito.

Melanization Phenotype AnalysisTo assess CLIPB4’s impact on melanization, two assays were utilized, melanization-associated spot assay (MelASA) and SRPN2 depletion-induced melanotic pseudotumor formation. To quantify melanotic excreta, MelASA was performed in the standard format (50 mosquitoes/experimental cup) as described previously [34], with or without bacterial challenge. For MelASAs after bacterial challenge, dsRNA-injected mosquitoes were allowed to recover for 60 h and injected each with 50.6 nL of ×1 PBS, OD600 = 1.5 Escherichia coli (resuspended in 1x PBS, same below), OD600 = 1.5 Staphylococcus aureus, and OD600 = 0.4 M. luteus (Micrococcus lysodeikticus lyophilized cells ATCC, No. 4698, Sigma-Aldrich), respectively. Clean P8-grade white filter papers (Fisher) were inserted into the bottom of the experimental cups after bacterial challenge and removed 12 h later. For MelASAs without bacterial challenge, clean filter papers were inserted immediately after dsRNA injection, and removed 60 h later. All MelASAs were performed with five independent biological replicates using different mosquito generations. Collected filter papers were imaged under white epi-illumination without filters in an Alpha Imager 2200 system (Alpha Innotec). Images were processed and analyzed by Fiji is Just ImageJ (FIJI, Version 2.1.0) as described by Sousa et al. [53] with the following changes. We extended the workflow to include 4 macros, which enable automatic sequential processing and analysis of all images. The first two macros, published in [53], threshold the filter images, the third macro (setOption[“BlackBackground”, true]; run[“Convert to Mask”];) performs a “convert to mask” step, and the fourth and final macro (run[“Analyze Particles...”, “show = Masks display exclude summarize”];) measures “the total spot area” on each filter paper. To quantify melanotic pseudotumor formation, mosquitoes were dissected when mortality reached 70% in the dsSRPN2/dsGFP group. Abdominal walls were prepared as described previously [7] and examined under ×40 magnification with Axio Imager A1 microscope (Zeiss) equipped with AxioCam MR5 (Zeiss). Image processing and abdomen melanized area calculation were performed in ImageJ software as described previously [29, 54]. Five independent biological replicates were performed using 20 mosquitoes per treatment. All statistical analyses were performed using GraphPad Prism (ver. 9.5.1, GraphPad Software).

Melanization Phenotype Analysis

To assess CLIPB4’s impact on melanization, two assays were utilized, melanization-associated spot assay (MelASA) and SRPN2 depletion-induced melanotic pseudotumor formation. To quantify melanotic excreta, MelASA was performed in the standard format (50 mosquitoes/experimental cup) as described previously [34], with or without bacterial challenge. For MelASAs after bacterial challenge, dsRNA-injected mosquitoes were allowed to recover for 60 h and injected each with 50.6 nL of ×1 PBS, OD600 = 1.5 Escherichia coli (resuspended in 1x PBS, same below), OD600 = 1.5 Staphylococcus aureus, and OD600 = 0.4 M. luteus (Micrococcus lysodeikticus lyophilized cells ATCC, No. 4698, Sigma-Aldrich), respectively. Clean P8-grade white filter papers (Fisher) were inserted into the bottom of the experimental cups after bacterial challenge and removed 12 h later. For MelASAs without bacterial challenge, clean filter papers were inserted immediately after dsRNA injection, and removed 60 h later. All MelASAs were performed with five independent biological replicates using different mosquito generations. Collected filter papers were imaged under white epi-illumination without filters in an Alpha Imager 2200 system (Alpha Innotec). Images were processed and analyzed by Fiji is Just ImageJ (FIJI, Version 2.1.0) as described by Sousa et al. [53] with the following changes. We extended the workflow to include 4 macros, which enable automatic sequential processing and analysis of all images. The first two macros, published in [53], threshold the filter images, the third macro (setOption[“BlackBackground”, true]; run[“Convert to Mask”];) performs a “convert to mask” step, and the fourth and final macro (run[“Analyze Particles...”, “show = Masks display exclude summarize”];) measures “the total spot area” on each filter paper. To quantify melanotic pseudotumor formation, mosquitoes were dissected when mortality reached 70% in the dsSRPN2/dsGFP group. Abdominal walls were prepared as described previously [7] and examined under ×40 magnification with Axio Imager A1 microscope (Zeiss) equipped with AxioCam MR5 (Zeiss). Image processing and abdomen melanized area calculation were performed in ImageJ software as described previously [29, 54]. Five independent biological replicates were performed using 20 mosquitoes per treatment. All statistical analyses were performed using GraphPad Prism (ver. 9.5.1, GraphPad Software).

ResultsCo-Expression Analysis of Putative Melanization Genes Identifies CLIPB4 as a Central NodeIn the absence of a genome-wide protein-protein interaction network for An. gambiae, we used gene co-expression as a proxy to identify proteins that are likely to inhabit the same space at the same time. To facilitate the identification of possible protein-protein interactions, and specifically SRPN2-cSP interactions controlling melanization, we therefore constructed an undirected gene co-expression network of all An. gambiae genes belonging to gene families experimentally shown to contribute to melanization, which we named the AgMelGCN (Fig. 1a). In total, AgMelGCN contains 178 nodes (genes) and 1,229 edges (co-expression links between gene pairs) (online suppl. Tables S2, S3). To reveal the central nodes of AgMelGCN, we performed an s-core analysis (s = 11.14). The core contained 25 genes, of which nine play a role in melanization (APL1C, CLIPA2, A8, B4, B10, CLIPC9, CTL4, CTLMA2, LRIM1; online suppl. Table S3). To identify key cSPs in the network, we calculated two centrality measures: node strength (weight sum of all node’s edges) and harmonic closeness centrality (sum of shortest paths to all other nodes within the full network) [47]. Of all annotated cSPs, CLIPB4 was the highest ranked node in either centrality measure (online suppl. Table S3; Fig. 1b). In AgMelGCN, CLIPB4 was connected by a weighted edge to 44 other nodes, including seven SRPNs; notably, the SRPN2-CLIPB4 edge had the highest weight among all SRPN nodes connected to CLIPB4 (Fig. 1b; online suppl. Table S4). SRPN2 was connected to 18 nodes, including five cSPs, all in the CLIPB subfamily (Fig. 1c; online suppl. Table S4), of which the SRPN2-CLIPB4 edge possessed the highest weight. Overall, these results suggested that CLIPB4 is a central node in the AgMelGCN whose expression most closely resembled that of SRPN2.Fig. 1.Gene co-expression network of genes with putative roles in melanization (AgMelGCN). a The network topography of AgMelGCN contains 178 nodes and 1,229 edges. Of all annotated putative CLIP serine proteases, CLIPB4 has the highest node degree in the network and is co-expressed with SRPN2. b Genes in the network that are co-expressed with CLIPB4, which includes SRPN2. c Genes in the network that are co-expressed with SRPN2, which includes CLIPB4. The position of every node in the network visualization is identical in all three panels.SRPN2 and CLIPB4 Interact in Adult Mosquito HemolymphSince SRPN2 is a critical factor affecting the melanization process, we aimed to identify its closely associated proteins in the hemolymph of adult mosquitoes. Using Co-IP, we identified 21 proteins bound directly or indirectly to SRPN2 in the hemolymph of adult female mosquitoes 24 h post-challenge with lyophilized M. luteus bacteria (Table 1). Among these, six genes were co-expressed with SRPN2 in AgMelGCN, including TEP15, CLIPB4, CLIPA14, SRPN11, CLIPA9, and CLIPA6 (Fig. 1c; Table 1 and online suppl. Table S4). Among annotated cSPs and cSPHs, CLIPB4 was ranked the highest in SRPN2 Co-IP MS protein scores, warranting further examination of the interaction between SRPN2 and CLIPB4.Table 1.List of proteins that co-immunoprecipitate with SRPN2AGAP number*NCBI Acc. #Gene nameMolecular weight, kDaScaffoldProteome discovererpercent coverageunique peptide counttotal spectrum countscorepercent coveragetotal peptide count# PSMs
AGAP006911
Q7QIJ8SRPN24752.8132699.29531328
AGAP008364
A0A453YTR1TEP1516419.4212797.21212431
AGAP000376
Q7QF98Transferrin6926.9141973.08341924
AGAP003250
C0L2J9CLIPB43933.181145.7636914
AGAP006258
O44250PPO1**7816.471243.816712
AGAP009217
Q7PVQ0CLIPB123822.35632.934811
AGAP004977
O96753PPO67914.16930.8716710
AGAP011788
A0A1S4H7K2CLIPA144625.87829.892679
AGAP028138
Q5TPW1Gelsolin838.195726.841179
AGAP008060
Q6JEK4Imaginal disc growth factor 2488.663316.121666
AGAP003095
Q8MZM5Dopachrome conversion enzyme539.093314.41655
AGAP001377
Q005M5SRPN1157145513.851455
AGAP003139
A0A1S4GHQ9SRPN95010.33313.671034
AGAP002564
F5HKV6Fructose bisphosphate aldolase397.712310.561034
AGAP010968
A0A1S4H5E1CLIPA9407.18239.22723
AGAP011789
Q7PV63CLIPA6467.03229.2723
AGAP007406
Q7PT29Elongation factor 1-alpha507.56228.921133
AGAP010815
A0A2P1E488TEP11521.87228.4333
AGAP007037
F2Y0G4APL1C825.7337.95633
AGAP003689
A0A1S4GIW4CLIPC7674.19225.82422
AGAP010724
Q7PCX0Histon H41123.3225.192322
AGAP003691
A0A1S4GJE4CLIPE12983.13224.88322*AGAP number based on AgamP4.13.**Named PPO2 in VectorBase.CLIPB4 Is a Serine Protease with Trypsin-Like SpecificityTo investigate its biochemical function(s), we expressed recombinant An. gambiae proCLIPB4 zymogen in vitro. The determinants of enzyme specificity were predicted to be D305, G332, and G343 (online suppl. Fig. S2a), suggesting that CLIPB4 is a trypsin-like protease that cleaves its substrate after arginine or lysine residues [55]. As the endogenous activating enzyme of CLIPB4 is unknown, a mutant (proCLIPB4Xa) was produced by replacing its putative activation site 104LTDR107 [56] with 104IEGR107 (online suppl. Fig. S2b), enabling recognition and cleavage by commercially available bovine factor Xa. We produced and purified recombinant proCLIPB4Xa zymogen (rproCLIPB4Xa) tagged with 6His at its C-terminus (online suppl. Fig. S3a). Coomassie stain, His antibody, and anti-CLIPB4 antibody detected rproCLIPB4Xa (37.9 kDa) at the expected molecular weight. Recombinant proCLIPB4Xa was cleaved by factor Xa after incubation at 37°C and RT, generating a fragment corresponding to the expected size of the CLIPB4 catalytic domain (rCLIPB4Xa, online suppl. Fig. S1a and c). Recombinant CLIPB4Xa exhibited amidase activity, using IEARpNA substrate, while rproCLIPB4Xa did not (online suppl. Fig. S1b and d). As noted in previous works, factor Xa also utilized IEARpNA as a substrate [28–30]. These results together support that, consistent with our construct design, factor Xa can cleave and activate rCLIPB4Xa-6His.SRPN2 Irreversibly Inhibits CLIPB4 in vitroBased on the AgMelGCN and Co-IP results, we hypothesized that SRPN2 is a direct inhibitor of CLIPB4. To test this hypothesis, we determined (i) if SRPN2 and CLIPB4 form inhibitory protein complexes and (ii) if SRPN2 decreases CLIPB4 amidase activity in a concentration-dependent manner. Purified rSRPN2, tagged N-terminally with 6-His tag, was incubated with rCLIPB4Xa at RT for 15 min at two different molar ratios, 1:1 or 10:1 (rSRPN2:rCLIPB4Xa). In SDS-PAGE and immunoblot analyses of these reactions, a higher molecular weight band appeared at 72 kDa, which was detected with anti-His, anti-SRPN2, and anti-CLIPB4 antibodies (Fig. 2a, b, c). Electrospray ionization mass spectrometry analysis on the 72 kDa band (online suppl. Table S5) detected 15 and 2 peptides for SRPN2 and CLIPB4, respectively, corresponding to 11 and 2 unique peptides for each, confirming that SRPN2 and CLIPB4 form a SDS-stable protein complex. CLIPB4Xa amidase activity against IEARpNA decreased with increasing amount of rSRPN2, with a stoichiometry of inhibition (SI) of 1.4 (Fig. 2d), demonstrating that SRPN2 functions as an efficient inhibitor of CLIPB4.Fig. 2.SRPN2 forms SDS-stable complex with CLIPB4 and inhibits its activity. a, b, c Detection of CLIPB4-SRPN2 complex by immunoblot analysis. Recombinant proCLIPB4Xa was activated by factor Xa, and then incubated with recombinant SRPN2 with a molar ratio of 1:1 or 10:1 (rSRPN2:proCLIPB4Xa) at RT for 15 min. The mixtures were subjected to 12% SDS-PAGE followed by immunoblot analysis with mouse anti-His (a) or rabbit anti-SRPN2 (b) or rabbit anti-CLIPB4 (c) antibodies. Circles, proCLIPB4Xa zymogen; asterisks, catalytic domain of CLIPB4Xa; solid triangles, recombinant SRPN2; hollow triangles, truncated SRPN2; arrows, CLIPB4/SRPN2 complex. d Stoichiometry of SRPN2 inhibition of CLIPB4. Factor Xa-activated CLIPB4 was incubated with recombinant SRPN2 at various molar ratios at RT for 15 min. Residual activities of CLIPB4 were measured with IEARpNA as substrate and plotted as mean ± 1 SD (n = 3) against the corresponding molar ratios of rSRPN2 and proCLIPB4Xa.CLIPB4kd Is Required for the Phenotypes Induced by SRPN2 DepletionPreviously, SRPN2kd was shown to reduce the longevity of female mosquitoes and to induce melanotic pseudotumor formation [7]. Our biochemical data support that SRPN2 directly inhibits the activity of CLIPB4, and we hypothesized that CLIPB4 is required for the phenotypes induced by SRPN2 depletion. To test this hypothesis, we performed genetic epistasis experiments, inducing gene kd by long dsRNA injection. Adult female An. gambiae were injected with dsRNAs targeting CLIPB4, SRPN2, and both CLIPB4 and SRPN2, respectively. Injection of dsGFP was used as a control. CLIPB4kd was significant and specific as determined by immunoblot (online suppl. Fig. S4). Injection of dsSRPN2 did not affect CLIPB4 protein levels.We first evaluated the effect of CLIPB4kd on survival by comparing the survival curves between the four different treatment groups, and we also monitored the survival of uninjected mosquitoes. SRPN2kd significantly shortened longevity as compared to the uninjected controls and dsGFP-control treatment (Fig. 3a, b). In contrast, CLIPB4kd had no effect on lifespan as compared to the controls. Importantly, dsCLIPB4/SRPN2 treatment substantially and significantly increased mosquito longevity as compared to SRPN2kd mosquitoes, returning longevity to that of uninjected and dsGFP-control treated mosquitoes.Fig. 3.dsCLIPB4 partially recovers the phenotype of SRPN2 depletion. aCLIPB4kd alone did not affect survival rates but reverted the increased mortality induced by SRPN2 depletion. Survival curves were plotted as means with 95% confidence intervals (n = 3). b Median survivals in days were plotted (mean ± 1 SEM) based on results shown in (a); statistically significant differences are indicated by different letters (one-way ANOVA followed by Newman-Keuls test, p < 0.01). c Total melanotic area per abdomen decreased due to dsCLIPB4 in SRPN2-depleted genetic background. Medians and interquartile ranges are marked (n = 104, 5 biological replicates), and statistical analysis was performed with the Mann-Whitney U test. d SRPN2 depletion increased melanotic excreta, which was tempered partially by dsCLIPB4 treatment. Left panel shows examples for processed (thresholded) filter paper images. Data are shown as dots for individual replicates, bars represent means ± 1 SD (n = 5); data were distributed normally (Shapiro-Wilk test, p < 0.0001), and statistically significant differences are indicated by different letters (one-way ANOVA followed by Newman-Keuls test, p < 0.01).We then evaluated the effect of CLIPB4kd on melanization, using the appearance of melanotic tumors as a readout (Fig. 3c). Injection of dsGFP or dsCLIPB4 alone did not cause melanization (data not shown). In contrast, CLIPB4kd in a SRPN2-depleted background, significantly affected the SRPN2kd melanization phenotype, reducing the average total melanotic area for each dissected mosquito abdomen by 54%, however, not completely reverting the SRPN2 depletion phenotype. We also used the recently established MelASA to evaluate the contribution of CLIPB4 on the SRPN2 depletion phenotype [34]. The standard format (50 mosquitoes/cup) was adopted and melanotic excreta were recorded for the four treatment groups. CLIPB4kd alone exhibited a similar area of melanotic excreta as compared to that of the negative control, dsGFP. SRPN2 depletion, however, significantly increased melanotic excreta of female mosquitoes. Dkd of SRPN2 and CLIPB4 significantly reduced the excreta compared to SRPN2 single kd but did not completely abolish the SRPN2 depletion phenotype (Fig. 3d). Taken together, the silencing experiments strongly suggest that CLIPB4 is epistatic to SRPN2 and therefore downstream of this SRPN in the proPO activation cascade.CLIPB4 Is Required for Melanization after Microbial ChallengeSince CLIPB4 is required for melanization induced by SRPN2 depletion, we hypothesized that CLIPB4 is broadly required for melanization in antimicrobial immunity. To test the hypothesis, we first conducted MelASAs after challenge with bacteria (S. aureus, M. luteus, and E. coli) (online suppl. Fig. S5). Without bacterial challenge, only small amounts of melanotic excreta were observed in the dsGFP-injected and CLIPB4kd treatment groups, consistent with our previous observation in Figure 3d. Each bacterial challenge significantly increased melanotic excreta in the dsGFP-injected group as compared to dsGFP-injected mosquitoes treated with PBS, supporting that bacterial treatment indeed induces the melanization immune response. The induction of melanotic excreta by bacterial challenge was dampened in the CLIPB4kd treatment groups. Independent of bacterial species, melanotic excreta were reduced by about 50% in CLIPB4kd mosquitoes as compared to dsGFP-injected mosquitoes, demonstrating that melanization in response to bacterial infection is broadly dependent on CLIPB4.CLIPB4 Functions as a proPO-Activating EnzymeThe shortage of hemolymph collectable from An. gambiae mosquitoes can be overcome by using the M. sexta model system as a source of proPO, as M. sexta PPO1&2 share the conserved cleavage site with eight of the nine proPOs encoded in the An. gambiae genome [7]. Activated CLIPB4Xa was incubated with M. sexta plasma, followed by immunoblot analysis using anti-M. sexta PPO antibody. A doublet band around 80 kDa in M. sexta plasma represents heterodimeric proPO consisting of 79-kDa PPO1 and 80-kDa PPO2 [51]. Addition of activated CLIPB4Xa to M. sexta plasma resulted in the appearance of a 70-kDa doublet band corresponding to M. sexta active PO, of which the upper active PO band almost co-migrated with the smaller proPO isoform (Fig. 4a). This proPO and active PO isoform banding pattern is commonly observed in immunoblots of M. sexta hemolymph probed with anti-MsPPO1&2 antibody due to co-migration with storage hexamerins that lead to compression of the bands [51]. The same doublet band was observed in the plasma supplemented with activated CLIPB9Xa, which we identified previously as a functional PAP in An. gambiae [29]. PO activity of plasma increased in the presence of active CLIPB4Xa (Fig. 4c). These results confirm that the proteolytic activity of CLIPB4 promotes proPO cleavage and PO activity. To determine the placement of CLIPB4 in the proPO activation cascade in An. gambiae, we next tested whether CLIPB4 can directly cleave and activate proPO. PPO1&2 was purified from 5th instar M. sexta larvae (Fig. S3c). By Coomassie Blue staining and immunoblot analysis with MsPPO antibody, a dimer was observed around 80 kDa, consistent with previous observations of purified full-length M. sexta PPO1&2 [51]. We then incubated factor Xa-activated rCLIPB4Xa with purified MsPPO1&2, again using rCLIPB9Xa as a positive control. Neither rproCLIPB9Xa zymogen nor factor Xa alone were able to cleave MsPPO1&2, while active rCLIPB9Xa cleaved purified MsPPO1&2 to produce the PO1&2 band at 70 kDa (Fig. 4b). Similarly, neither rproCLIPB4Xa zymogen nor factor Xa alone cleaved MsPPO1&2, but active rCLIPB4Xa cleaved proPO1&2 (Fig. 4b), leading to an increase in PO activity (Fig. 4d), demonstrating that CLIPB4 functions as a PAP.Fig. 4.CLIPB4 cleaves and activates M. sexta PPO1&2. Factor Xa-activated proCLIPB4Xa cleaves MsPPO1&2 in plasma (a) and purified protein (b), detected by immunoblot using rabbit anti-MsPPO1&2 antibody. The light and dark gray arrows indicate the larger isoforms of full and cleaved MsPPO1&2, respectively. CLIPB9 cleavage of M. sexta PPO was used as a positive control. The cleavage by CLIPB4 leads to increased PO activity in plasma (c) and with purified PPO protein (d). Bars represent mean ± 1 SD (n = 3); statistically significant differences are indicated by different letters (one-way ANOVA followed by Newman-Keuls test, p < 0.05).CLIPB4 Cleaves and Activates CLIPB8We next explored whether CLIPB4 also promotes PO activity indirectly by activating the PAP zymogen proCLIPB9 [29] and/or proCLIPB8, a protease upstream of CLIPB9 in the melanization cascade in An. gambiae [30]. To test whether CLIPB4 can cleave proCLIPB8 and/or B9 in vitro, active CLIPB4Xa was incubated with wild-type recombinant proCLIPB8 and proCLIPB9, respectively, followed by immunoblot analysis using either anti-His antibody or anti-CLIPB8 and anti-CLIPB9 antibodies, respectively. In the reaction of active CLIPB4Xa with proCLIPB8 zymogen, anti-His and anti-CLIPB8 antibodies detected a 36 kDa band, corresponding to the CLIPB8 protease domain, which was also detected in the control reaction of proCLIPB8Xa incubated with factor Xa (Fig. 5a, b). In contrast, addition of active CLIPB4Xa (or factor Xa) did not result in the cleavage of proCLIPB9 (Fig. 5c,d), suggesting that CLIPB4 is not directly upstream of CLIPB9. To evaluate whether cleavage of recombinant proCLIPB8 by CLIPB4Xa results in CLIPB8 activation, we measured the amidase activity of the reaction mixtures using FVR as a substrate [30]. Each of the five control reactions, containing proCLIPB8, proCLIPB8Xa, factor Xa, proCLIPB4Xa, and proCLIPB4Xa/proCLIPB8, respectively, did not exhibit amidase activity. Active CLIPB4Xa showed limited amidase activity, which was increased three-fold when active CLIPB4Xa was incubated with proCLIPB8 prior to the addition of substrate (Fig. 5e). Together with the immunoblot results (Fig. 5a, b), FVRase activity assay supports that CLIPB4 specifically cleaves and activates CLIPB8.Fig. 5.CLIPB4 activates B8, not B9. Recombinant proCLIPB4Xa was activated by factor Xa, and subsequently incubated with 100 ng of recombinant proCLIPB8 or proCLIPB9 at RT for 15 min. Samples were subjected to 12% SDS-PAGE followed by immunoblot with mouse anti-His (a, c), rabbit anti-CLIPB8 (b), and rabbit anti-CLIPB9 (d) antibodies, respectively. In parallel, 100 ng of factor Xa-activated proCLIPB8Xa or proCLIPB9Xa were used as positive controls. Circles, proCLIPB4Xa zymogen; asterisks, catalytic domain of CLIPB4; hollow squares, recombinant proCLIPB8 or proCLIPB9 zymogen; solid squares, catalytic domain of proCLIPB8; hollow triangles, proCLIPB8Xa or proCLIPB9Xa zymogen; solid triangles, catalytic domain of proCLIPB8Xa or proCLIPB9Xa. e 72 ng of factor Xa-activated proCLIPB4Xa was incubated with 450 ng of recombinant proCLIPB8 at RT for 1 h, and the catalytic activity of CLIPB8 was determined using FVR as substrate; 450 ng of factor Xa-activated proCLIPB8Xa was used as a positive control. The bars represent mean ± 1 SD (n = 3); statistically significant differences are indicated by different letters (one-way ANOVA followed by Newman-Keuls test, p < 0.05).To examine this phenomenon in vivo, dsLacZ, dsCLIPB4, and dsCLIPB8 were injected into adult female mosquitoes that were challenged by S. aureus after the dsRNA injection recovery. Hemolymph was collected 10 h after challenge and subjected to SDS-PAGE and immunoblot detection (Fig. 6). Apolipophorin-II (APOII) was utilized as a loading control. CLIPB8 was depleted with dsCLIPB8 treatment, serving as a positive control for CLIPB8 antibody detection. Ten hours after bacterial challenge, a clear reduction of CLIPB8 was observed in dsLacZ compared to the naïve control in two independent experiments, suggesting that CLIPB8 in the hemolymph was rapidly utilized in the acute immune response [57]. The CLIPB8 reduction, mediated by S. aureus challenge, was rescued with dsCLIPB4 treatment in two independent experiments, providing additional support that CLIPB4 acts upstream of CLIPB8 in the hierarchical cSP cascade regulating melanization in An. gambiae.Fig. 6.CLIPB4 is required for infection-induced depletion of CLIPB8 from the hemolymph of An. gambiae. Female mosquitoes were treated by dsLacZ, dsCLIPB4, and dsCLIPB8. Post-recovery, mosquitoes were challenged by S. aureus (OD600 = 0.8). Ten hours postinfection, hemolymph was collected by proboscis clipping, separated by SDS-PAGE, and subjected to immunoblot using rabbit anti-CLIPB8 and rabbit anti-APOII antibodies. Results were shown from two independent experiments.

Co-Expression Analysis of Putative Melanization Genes Identifies CLIPB4 as a Central NodeIn the absence of a genome-wide protein-protein interaction network for An. gambiae, we used gene co-expression as a proxy to identify proteins that are likely to inhabit the same space at the same time. To facilitate the identification of possible protein-protein interactions, and specifically SRPN2-cSP interactions controlling melanization, we therefore constructed an undirected gene co-expression network of all An. gambiae genes belonging to gene families experimentally shown to contribute to melanization, which we named the AgMelGCN (Fig. 1a). In total, AgMelGCN contains 178 nodes (genes) and 1,229 edges (co-expression links between gene pairs) (online suppl. Tables S2, S3). To reveal the central nodes of AgMelGCN, we performed an s-core analysis (s = 11.14). The core contained 25 genes, of which nine play a role in melanization (APL1C, CLIPA2, A8, B4, B10, CLIPC9, CTL4, CTLMA2, LRIM1; online suppl. Table S3). To identify key cSPs in the network, we calculated two centrality measures: node strength (weight sum of all node’s edges) and harmonic closeness centrality (sum of shortest paths to all other nodes within the full network) [47]. Of all annotated cSPs, CLIPB4 was the highest ranked node in either centrality measure (online suppl. Table S3; Fig. 1b). In AgMelGCN, CLIPB4 was connected by a weighted edge to 44 other nodes, including seven SRPNs; notably, the SRPN2-CLIPB4 edge had the highest weight among all SRPN nodes connected to CLIPB4 (Fig. 1b; online suppl. Table S4). SRPN2 was connected to 18 nodes, including five cSPs, all in the CLIPB subfamily (Fig. 1c; online suppl. Table S4), of which the SRPN2-CLIPB4 edge possessed the highest weight. Overall, these results suggested that CLIPB4 is a central node in the AgMelGCN whose expression most closely resembled that of SRPN2.Fig. 1.Gene co-expression network of genes with putative roles in melanization (AgMelGCN). a The network topography of AgMelGCN contains 178 nodes and 1,229 edges. Of all annotated putative CLIP serine proteases, CLIPB4 has the highest node degree in the network and is co-expressed with SRPN2. b Genes in the network that are co-expressed with CLIPB4, which includes SRPN2. c Genes in the network that are co-expressed with SRPN2, which includes CLIPB4. The position of every node in the network visualization is identical in all three panels.

Co-Expression Analysis of Putative Melanization Genes Identifies CLIPB4 as a Central Node

In the absence of a genome-wide protein-protein interaction network for An. gambiae, we used gene co-expression as a proxy to identify proteins that are likely to inhabit the same space at the same time. To facilitate the identification of possible protein-protein interactions, and specifically SRPN2-cSP interactions controlling melanization, we therefore constructed an undirected gene co-expression network of all An. gambiae genes belonging to gene families experimentally shown to contribute to melanization, which we named the AgMelGCN (Fig. 1a). In total, AgMelGCN contains 178 nodes (genes) and 1,229 edges (co-expression links between gene pairs) (online suppl. Tables S2, S3). To reveal the central nodes of AgMelGCN, we performed an s-core analysis (s = 11.14). The core contained 25 genes, of which nine play a role in melanization (APL1C, CLIPA2, A8, B4, B10, CLIPC9, CTL4, CTLMA2, LRIM1; online suppl. Table S3). To identify key cSPs in the network, we calculated two centrality measures: node strength (weight sum of all node’s edges) and harmonic closeness centrality (sum of shortest paths to all other nodes within the full network) [47]. Of all annotated cSPs, CLIPB4 was the highest ranked node in either centrality measure (online suppl. Table S3; Fig. 1b). In AgMelGCN, CLIPB4 was connected by a weighted edge to 44 other nodes, including seven SRPNs; notably, the SRPN2-CLIPB4 edge had the highest weight among all SRPN nodes connected to CLIPB4 (Fig. 1b; online suppl. Table S4). SRPN2 was connected to 18 nodes, including five cSPs, all in the CLIPB subfamily (Fig. 1c; online suppl. Table S4), of which the SRPN2-CLIPB4 edge possessed the highest weight. Overall, these results suggested that CLIPB4 is a central node in the AgMelGCN whose expression most closely resembled that of SRPN2.

Gene co-expression network of genes with putative roles in melanization (AgMelGCN). a The network topography of AgMelGCN contains 178 nodes and 1,229 edges. Of all annotated putative CLIP serine proteases, CLIPB4 has the highest node degree in the network and is co-expressed with SRPN2. b Genes in the network that are co-expressed with CLIPB4, which includes SRPN2. c Genes in the network that are co-expressed with SRPN2, which includes CLIPB4. The position of every node in the network visualization is identical in all three panels.

SRPN2 and CLIPB4 Interact in Adult Mosquito HemolymphSince SRPN2 is a critical factor affecting the melanization process, we aimed to identify its closely associated proteins in the hemolymph of adult mosquitoes. Using Co-IP, we identified 21 proteins bound directly or indirectly to SRPN2 in the hemolymph of adult female mosquitoes 24 h post-challenge with lyophilized M. luteus bacteria (Table 1). Among these, six genes were co-expressed with SRPN2 in AgMelGCN, including TEP15, CLIPB4, CLIPA14, SRPN11, CLIPA9, and CLIPA6 (Fig. 1c; Table 1 and online suppl. Table S4). Among annotated cSPs and cSPHs, CLIPB4 was ranked the highest in SRPN2 Co-IP MS protein scores, warranting further examination of the interaction between SRPN2 and CLIPB4.Table 1.List of proteins that co-immunoprecipitate with SRPN2AGAP number*NCBI Acc. #Gene nameMolecular weight, kDaScaffoldProteome discovererpercent coverageunique peptide counttotal spectrum countscorepercent coveragetotal peptide count# PSMs
AGAP006911
Q7QIJ8SRPN24752.8132699.29531328
AGAP008364
A0A453YTR1TEP1516419.4212797.21212431
AGAP000376
Q7QF98Transferrin6926.9141973.08341924
AGAP003250
C0L2J9CLIPB43933.181145.7636914
AGAP006258
O44250PPO1**7816.471243.816712
AGAP009217
Q7PVQ0CLIPB123822.35632.934811
AGAP004977
O96753PPO67914.16930.8716710
AGAP011788
A0A1S4H7K2CLIPA144625.87829.892679
AGAP028138
Q5TPW1Gelsolin838.195726.841179
AGAP008060
Q6JEK4Imaginal disc growth factor 2488.663316.121666
AGAP003095
Q8MZM5Dopachrome conversion enzyme539.093314.41655
AGAP001377
Q005M5SRPN1157145513.851455
AGAP003139
A0A1S4GHQ9SRPN95010.33313.671034
AGAP002564
F5HKV6Fructose bisphosphate aldolase397.712310.561034
AGAP010968
A0A1S4H5E1CLIPA9407.18239.22723
AGAP011789
Q7PV63CLIPA6467.03229.2723
AGAP007406
Q7PT29Elongation factor 1-alpha507.56228.921133
AGAP010815
A0A2P1E488TEP11521.87228.4333
AGAP007037
F2Y0G4APL1C825.7337.95633
AGAP003689
A0A1S4GIW4CLIPC7674.19225.82422
AGAP010724
Q7PCX0Histon H41123.3225.192322
AGAP003691
A0A1S4GJE4CLIPE12983.13224.88322*AGAP number based on AgamP4.13.**Named PPO2 in VectorBase.

SRPN2 and CLIPB4 Interact in Adult Mosquito Hemolymph

Since SRPN2 is a critical factor affecting the melanization process, we aimed to identify its closely associated proteins in the hemolymph of adult mosquitoes. Using Co-IP, we identified 21 proteins bound directly or indirectly to SRPN2 in the hemolymph of adult female mosquitoes 24 h post-challenge with lyophilized M. luteus bacteria (Table 1). Among these, six genes were co-expressed with SRPN2 in AgMelGCN, including TEP15, CLIPB4, CLIPA14, SRPN11, CLIPA9, and CLIPA6 (Fig. 1c; Table 1 and online suppl. Table S4). Among annotated cSPs and cSPHs, CLIPB4 was ranked the highest in SRPN2 Co-IP MS protein scores, warranting further examination of the interaction between SRPN2 and CLIPB4.

List of proteins that co-immunoprecipitate with SRPN2

*AGAP number based on AgamP4.13.

**Named PPO2 in VectorBase.

CLIPB4 Is a Serine Protease with Trypsin-Like SpecificityTo investigate its biochemical function(s), we expressed recombinant An. gambiae proCLIPB4 zymogen in vitro. The determinants of enzyme specificity were predicted to be D305, G332, and G343 (online suppl. Fig. S2a), suggesting that CLIPB4 is a trypsin-like protease that cleaves its substrate after arginine or lysine residues [55]. As the endogenous activating enzyme of CLIPB4 is unknown, a mutant (proCLIPB4Xa) was produced by replacing its putative activation site 104LTDR107 [56] with 104IEGR107 (online suppl. Fig. S2b), enabling recognition and cleavage by commercially available bovine factor Xa. We produced and purified recombinant proCLIPB4Xa zymogen (rproCLIPB4Xa) tagged with 6His at its C-terminus (online suppl. Fig. S3a). Coomassie stain, His antibody, and anti-CLIPB4 antibody detected rproCLIPB4Xa (37.9 kDa) at the expected molecular weight. Recombinant proCLIPB4Xa was cleaved by factor Xa after incubation at 37°C and RT, generating a fragment corresponding to the expected size of the CLIPB4 catalytic domain (rCLIPB4Xa, online suppl. Fig. S1a and c). Recombinant CLIPB4Xa exhibited amidase activity, using IEARpNA substrate, while rproCLIPB4Xa did not (online suppl. Fig. S1b and d). As noted in previous works, factor Xa also utilized IEARpNA as a substrate [28–30]. These results together support that, consistent with our construct design, factor Xa can cleave and activate rCLIPB4Xa-6His.

CLIPB4 Is a Serine Protease with Trypsin-Like Specificity

To investigate its biochemical function(s), we expressed recombinant An. gambiae proCLIPB4 zymogen in vitro. The determinants of enzyme specificity were predicted to be D305, G332, and G343 (online suppl. Fig. S2a), suggesting that CLIPB4 is a trypsin-like protease that cleaves its substrate after arginine or lysine residues [55]. As the endogenous activating enzyme of CLIPB4 is unknown, a mutant (proCLIPB4Xa) was produced by replacing its putative activation site 104LTDR107 [56] with 104IEGR107 (online suppl. Fig. S2b), enabling recognition and cleavage by commercially available bovine factor Xa. We produced and purified recombinant proCLIPB4Xa zymogen (rproCLIPB4Xa) tagged with 6His at its C-terminus (online suppl. Fig. S3a). Coomassie stain, His antibody, and anti-CLIPB4 antibody detected rproCLIPB4Xa (37.9 kDa) at the expected molecular weight. Recombinant proCLIPB4Xa was cleaved by factor Xa after incubation at 37°C and RT, generating a fragment corresponding to the expected size of the CLIPB4 catalytic domain (rCLIPB4Xa, online suppl. Fig. S1a and c). Recombinant CLIPB4Xa exhibited amidase activity, using IEARpNA substrate, while rproCLIPB4Xa did not (online suppl. Fig. S1b and d). As noted in previous works, factor Xa also utilized IEARpNA as a substrate [28–30]. These results together support that, consistent with our construct design, factor Xa can cleave and activate rCLIPB4Xa-6His.

SRPN2 Irreversibly Inhibits CLIPB4 in vitroBased on the AgMelGCN and Co-IP results, we hypothesized that SRPN2 is a direct inhibitor of CLIPB4. To test this hypothesis, we determined (i) if SRPN2 and CLIPB4 form inhibitory protein complexes and (ii) if SRPN2 decreases CLIPB4 amidase activity in a concentration-dependent manner. Purified rSRPN2, tagged N-terminally with 6-His tag, was incubated with rCLIPB4Xa at RT for 15 min at two different molar ratios, 1:1 or 10:1 (rSRPN2:rCLIPB4Xa). In SDS-PAGE and immunoblot analyses of these reactions, a higher molecular weight band appeared at 72 kDa, which was detected with anti-His, anti-SRPN2, and anti-CLIPB4 antibodies (Fig. 2a, b, c). Electrospray ionization mass spectrometry analysis on the 72 kDa band (online suppl. Table S5) detected 15 and 2 peptides for SRPN2 and CLIPB4, respectively, corresponding to 11 and 2 unique peptides for each, confirming that SRPN2 and CLIPB4 form a SDS-stable protein complex. CLIPB4Xa amidase activity against IEARpNA decreased with increasing amount of rSRPN2, with a stoichiometry of inhibition (SI) of 1.4 (Fig. 2d), demonstrating that SRPN2 functions as an efficient inhibitor of CLIPB4.Fig. 2.SRPN2 forms SDS-stable complex with CLIPB4 and inhibits its activity. a, b, c Detection of CLIPB4-SRPN2 complex by immunoblot analysis. Recombinant proCLIPB4Xa was activated by factor Xa, and then incubated with recombinant SRPN2 with a molar ratio of 1:1 or 10:1 (rSRPN2:proCLIPB4Xa) at RT for 15 min. The mixtures were subjected to 12% SDS-PAGE followed by immunoblot analysis with mouse anti-His (a) or rabbit anti-SRPN2 (b) or rabbit anti-CLIPB4 (c) antibodies. Circles, proCLIPB4Xa zymogen; asterisks, catalytic domain of CLIPB4Xa; solid triangles, recombinant SRPN2; hollow triangles, truncated SRPN2; arrows, CLIPB4/SRPN2 complex. d Stoichiometry of SRPN2 inhibition of CLIPB4. Factor Xa-activated CLIPB4 was incubated with recombinant SRPN2 at various molar ratios at RT for 15 min. Residual activities of CLIPB4 were measured with IEARpNA as substrate and plotted as mean ± 1 SD (n = 3) against the corresponding molar ratios of rSRPN2 and proCLIPB4Xa.

SRPN2 Irreversibly Inhibits CLIPB4 in vitro

Based on the AgMelGCN and Co-IP results, we hypothesized that SRPN2 is a direct inhibitor of CLIPB4. To test this hypothesis, we determined (i) if SRPN2 and CLIPB4 form inhibitory protein complexes and (ii) if SRPN2 decreases CLIPB4 amidase activity in a concentration-dependent manner. Purified rSRPN2, tagged N-terminally with 6-His tag, was incubated with rCLIPB4Xa at RT for 15 min at two different molar ratios, 1:1 or 10:1 (rSRPN2:rCLIPB4Xa). In SDS-PAGE and immunoblot analyses of these reactions, a higher molecular weight band appeared at 72 kDa, which was detected with anti-His, anti-SRPN2, and anti-CLIPB4 antibodies (Fig. 2a, b, c). Electrospray ionization mass spectrometry analysis on the 72 kDa band (online suppl. Table S5) detected 15 and 2 peptides for SRPN2 and CLIPB4, respectively, corresponding to 11 and 2 unique peptides for each, confirming that SRPN2 and CLIPB4 form a SDS-stable protein complex. CLIPB4Xa amidase activity against IEARpNA decreased with increasing amount of rSRPN2, with a stoichiometry of inhibition (SI) of 1.4 (Fig. 2d), demonstrating that SRPN2 functions as an efficient inhibitor of CLIPB4.

SRPN2 forms SDS-stable complex with CLIPB4 and inhibits its activity. a, b, c Detection of CLIPB4-SRPN2 complex by immunoblot analysis. Recombinant proCLIPB4Xa was activated by factor Xa, and then incubated with recombinant SRPN2 with a molar ratio of 1:1 or 10:1 (rSRPN2:proCLIPB4Xa) at RT for 15 min. The mixtures were subjected to 12% SDS-PAGE followed by immunoblot analysis with mouse anti-His (a) or rabbit anti-SRPN2 (b) or rabbit anti-CLIPB4 (c) antibodies. Circles, proCLIPB4Xa zymogen; asterisks, catalytic domain of CLIPB4Xa; solid triangles, recombinant SRPN2; hollow triangles, truncated SRPN2; arrows, CLIPB4/SRPN2 complex. d Stoichiometry of SRPN2 inhibition of CLIPB4. Factor Xa-activated CLIPB4 was incubated with recombinant SRPN2 at various molar ratios at RT for 15 min. Residual activities of CLIPB4 were measured with IEARpNA as substrate and plotted as mean ± 1 SD (n = 3) against the corresponding molar ratios of rSRPN2 and proCLIPB4Xa.

CLIPB4kd Is Required for the Phenotypes Induced by SRPN2 DepletionPreviously, SRPN2kd was shown to reduce the longevity of female mosquitoes and to induce melanotic pseudotumor formation [7]. Our biochemical data support that SRPN2 directly inhibits the activity of CLIPB4, and we hypothesized that CLIPB4 is required for the phenotypes induced by SRPN2 depletion. To test this hypothesis, we performed genetic epistasis experiments, inducing gene kd by long dsRNA injection. Adult female An. gambiae were injected with dsRNAs targeting CLIPB4, SRPN2, and both CLIPB4 and SRPN2, respectively. Injection of dsGFP was used as a control. CLIPB4kd was significant and specific as determined by immunoblot (online suppl. Fig. S4). Injection of dsSRPN2 did not affect CLIPB4 protein levels.We first evaluated the effect of CLIPB4kd on survival by comparing the survival curves between the four different treatment groups, and we also monitored the survival of uninjected mosquitoes. SRPN2kd significantly shortened longevity as compared to the uninjected controls and dsGFP-control treatment (Fig. 3a, b). In contrast, CLIPB4kd had no effect on lifespan as compared to the controls. Importantly, dsCLIPB4/SRPN2 treatment substantially and significantly increased mosquito longevity as compared to SRPN2kd mosquitoes, returning longevity to that of uninjected and dsGFP-control treated mosquitoes.Fig. 3.dsCLIPB4 partially recovers the phenotype of SRPN2 depletion. aCLIPB4kd alone did not affect survival rates but reverted the increased mortality induced by SRPN2 depletion. Survival curves were plotted as means with 95% confidence intervals (n = 3). b Median survivals in days were plotted (mean ± 1 SEM) based on results shown in (a); statistically significant differences are indicated by different letters (one-way ANOVA followed by Newman-Keuls test, p < 0.01). c Total melanotic area per abdomen decreased due to dsCLIPB4 in SRPN2-depleted genetic background. Medians and interquartile ranges are marked (n = 104, 5 biological replicates), and statistical analysis was performed with the Mann-Whitney U test. d SRPN2 depletion increased melanotic excreta, which was tempered partially by dsCLIPB4 treatment. Left panel shows examples for processed (thresholded) filter paper images. Data are shown as dots for individual replicates, bars represent means ± 1 SD (n = 5); data were distributed normally (Shapiro-Wilk test, p < 0.0001), and statistically significant differences are indicated by different letters (one-way ANOVA followed by Newman-Keuls test, p < 0.01).We then evaluated the effect of CLIPB4kd on melanization, using the appearance of melanotic tumors as a readout (Fig. 3c). Injection of dsGFP or dsCLIPB4 alone did not cause melanization (data not shown). In contrast, CLIPB4kd in a SRPN2-depleted background, significantly affected the SRPN2kd melanization phenotype, reducing the average total melanotic area for each dissected mosquito abdomen by 54%, however, not completely reverting the SRPN2 depletion phenotype. We also used the recently established MelASA to evaluate the contribution of CLIPB4 on the SRPN2 depletion phenotype [34]. The standard format (50 mosquitoes/cup) was adopted and melanotic excreta were recorded for the four treatment groups. CLIPB4kd alone exhibited a similar area of melanotic excreta as compared to that of the negative control, dsGFP. SRPN2 depletion, however, significantly increased melanotic excreta of female mosquitoes. Dkd of SRPN2 and CLIPB4 significantly reduced the excreta compared to SRPN2 single kd but did not completely abolish the SRPN2 depletion phenotype (Fig. 3d). Taken together, the silencing experiments strongly suggest that CLIPB4 is epistatic to SRPN2 and therefore downstream of this SRPN in the proPO activation cascade.

CLIPB4kd Is Required for the Phenotypes Induced by SRPN2 Depletion

Previously, SRPN2kd was shown to reduce the longevity of female mosquitoes and to induce melanotic pseudotumor formation [7]. Our biochemical data support that SRPN2 directly inhibits the activity of CLIPB4, and we hypothesized that CLIPB4 is required for the phenotypes induced by SRPN2 depletion. To test this hypothesis, we performed genetic epistasis experiments, inducing gene kd by long dsRNA injection. Adult female An. gambiae were injected with dsRNAs targeting CLIPB4, SRPN2, and both CLIPB4 and SRPN2, respectively. Injection of dsGFP was used as a control. CLIPB4kd was significant and specific as determined by immunoblot (online suppl. Fig. S4). Injection of dsSRPN2 did not affect CLIPB4 protein levels.

We first evaluated the effect of CLIPB4kd on survival by comparing the survival curves between the four different treatment groups, and we also monitored the survival of uninjected mosquitoes. SRPN2kd significantly shortened longevity as compared to the uninjected controls and dsGFP-control treatment (Fig. 3a, b). In contrast, CLIPB4kd had no effect on lifespan as compared to the controls. Importantly, dsCLIPB4/SRPN2 treatment substantially and significantly increased mosquito longevity as compared to SRPN2kd mosquitoes, returning longevity to that of uninjected and dsGFP-control treated mosquitoes.

dsCLIPB4 partially recovers the phenotype of SRPN2 depletion. aCLIPB4kd alone did not affect survival rates but reverted the increased mortality induced by SRPN2 depletion. Survival curves were plotted as means with 95% confidence intervals (n = 3). b Median survivals in days were plotted (mean ± 1 SEM) based on results shown in (a); statistically significant differences are indicated by different letters (one-way ANOVA followed by Newman-Keuls test, p < 0.01). c Total melanotic area per abdomen decreased due to dsCLIPB4 in SRPN2-depleted genetic background. Medians and interquartile ranges are marked (n = 104, 5 biological replicates), and statistical analysis was performed with the Mann-Whitney U test. d SRPN2 depletion increased melanotic excreta, which was tempered partially by dsCLIPB4 treatment. Left panel shows examples for processed (thresholded) filter paper images. Data are shown as dots for individual replicates, bars represent means ± 1 SD (n = 5); data were distributed normally (Shapiro-Wilk test, p < 0.0001), and statistically significant differences are indicated by different letters (one-way ANOVA followed by Newman-Keuls test, p < 0.01).

We then evaluated the effect of CLIPB4kd on melanization, using the appearance of melanotic tumors as a readout (Fig. 3c). Injection of dsGFP or dsCLIPB4 alone did not cause melanization (data not shown). In contrast, CLIPB4kd in a SRPN2-depleted background, significantly affected the SRPN2kd melanization phenotype, reducing the average total melanotic area for each dissected mosquito abdomen by 54%, however, not completely reverting the SRPN2 depletion phenotype. We also used the recently established MelASA to evaluate the contribution of CLIPB4 on the SRPN2 depletion phenotype [34]. The standard format (50 mosquitoes/cup) was adopted and melanotic excreta were recorded for the four treatment groups. CLIPB4kd alone exhibited a similar area of melanotic excreta as compared to that of the negative control, dsGFP. SRPN2 depletion, however, significantly increased melanotic excreta of female mosquitoes. Dkd of SRPN2 and CLIPB4 significantly reduced the excreta compared to SRPN2 single kd but did not completely abolish the SRPN2 depletion phenotype (Fig. 3d). Taken together, the silencing experiments strongly suggest that CLIPB4 is epistatic to SRPN2 and therefore downstream of this SRPN in the proPO activation cascade.

CLIPB4 Is Required for Melanization after Microbial ChallengeSince CLIPB4 is required for melanization induced by SRPN2 depletion, we hypothesized that CLIPB4 is broadly required for melanization in antimicrobial immunity. To test the hypothesis, we first conducted MelASAs after challenge with bacteria (S. aureus, M. luteus, and E. coli) (online suppl. Fig. S5). Without bacterial challenge, only small amounts of melanotic excreta were observed in the dsGFP-injected and CLIPB4kd treatment groups, consistent with our previous observation in Figure 3d. Each bacterial challenge significantly increased melanotic excreta in the dsGFP-injected group as compared to dsGFP-injected mosquitoes treated with PBS, supporting that bacterial treatment indeed induces the melanization immune response. The induction of melanotic excreta by bacterial challenge was dampened in the CLIPB4kd treatment groups. Independent of bacterial species, melanotic excreta were reduced by about 50% in CLIPB4kd mosquitoes as compared to dsGFP-injected mosquitoes, demonstrating that melanization in response to bacterial infection is broadly dependent on CLIPB4.

CLIPB4 Is Required for Melanization after Microbial Challenge

Since CLIPB4 is required for melanization induced by SRPN2 depletion, we hypothesized that CLIPB4 is broadly required for melanization in antimicrobial immunity. To test the hypothesis, we first conducted MelASAs after challenge with bacteria (S. aureus, M. luteus, and E. coli) (online suppl. Fig. S5). Without bacterial challenge, only small amounts of melanotic excreta were observed in the dsGFP-injected and CLIPB4kd treatment groups, consistent with our previous observation in Figure 3d. Each bacterial challenge significantly increased melanotic excreta in the dsGFP-injected group as compared to dsGFP-injected mosquitoes treated with PBS, supporting that bacterial treatment indeed induces the melanization immune response. The induction of melanotic excreta by bacterial challenge was dampened in the CLIPB4kd treatment groups. Independent of bacterial species, melanotic excreta were reduced by about 50% in CLIPB4kd mosquitoes as compared to dsGFP-injected mosquitoes, demonstrating that melanization in response to bacterial infection is broadly dependent on CLIPB4.

CLIPB4 Functions as a proPO-Activating EnzymeThe shortage of hemolymph collectable from An. gambiae mosquitoes can be overcome by using the M. sexta model system as a source of proPO, as M. sexta PPO1&2 share the conserved cleavage site with eight of the nine proPOs encoded in the An. gambiae genome [7]. Activated CLIPB4Xa was incubated with M. sexta plasma, followed by immunoblot analysis using anti-M. sexta PPO antibody. A doublet band around 80 kDa in M. sexta plasma represents heterodimeric proPO consisting of 79-kDa PPO1 and 80-kDa PPO2 [51]. Addition of activated CLIPB4Xa to M. sexta plasma resulted in the appearance of a 70-kDa doublet band corresponding to M. sexta active PO, of which the upper active PO band almost co-migrated with the smaller proPO isoform (Fig. 4a). This proPO and active PO isoform banding pattern is commonly observed in immunoblots of M. sexta hemolymph probed with anti-MsPPO1&2 antibody due to co-migration with storage hexamerins that lead to compression of the bands [51]. The same doublet band was observed in the plasma supplemented with activated CLIPB9Xa, which we identified previously as a functional PAP in An. gambiae [29]. PO activity of plasma increased in the presence of active CLIPB4Xa (Fig. 4c). These results confirm that the proteolytic activity of CLIPB4 promotes proPO cleavage and PO activity. To determine the placement of CLIPB4 in the proPO activation cascade in An. gambiae, we next tested whether CLIPB4 can directly cleave and activate proPO. PPO1&2 was purified from 5th instar M. sexta larvae (Fig. S3c). By Coomassie Blue staining and immunoblot analysis with MsPPO antibody, a dimer was observed around 80 kDa, consistent with previous observations of purified full-length M. sexta PPO1&2 [51]. We then incubated factor Xa-activated rCLIPB4Xa with purified MsPPO1&2, again using rCLIPB9Xa as a positive control. Neither rproCLIPB9Xa zymogen nor factor Xa alone were able to cleave MsPPO1&2, while active rCLIPB9Xa cleaved purified MsPPO1&2 to produce the PO1&2 band at 70 kDa (Fig. 4b). Similarly, neither rproCLIPB4Xa zymogen nor factor Xa alone cleaved MsPPO1&2, but active rCLIPB4Xa cleaved proPO1&2 (Fig. 4b), leading to an increase in PO activity (Fig. 4d), demonstrating that CLIPB4 functions as a PAP.Fig. 4.CLIPB4 cleaves and activates M. sexta PPO1&2. Factor Xa-activated proCLIPB4Xa cleaves MsPPO1&2 in plasma (a) and purified protein (b), detected by immunoblot using rabbit anti-MsPPO1&2 antibody. The light and dark gray arrows indicate the larger isoforms of full and cleaved MsPPO1&2, respectively. CLIPB9 cleavage of M. sexta PPO was used as a positive control. The cleavage by CLIPB4 leads to increased PO activity in plasma (c) and with purified PPO protein (d). Bars represent mean ± 1 SD (n = 3); statistically significant differences are indicated by different letters (one-way ANOVA followed by Newman-Keuls test, p < 0.05).

CLIPB4 Functions as a proPO-Activating Enzyme

The shortage of hemolymph collectable from An. gambiae mosquitoes can be overcome by using the M. sexta model system as a source of proPO, as M. sexta PPO1&2 share the conserved cleavage site with eight of the nine proPOs encoded in the An. gambiae genome [7]. Activated CLIPB4Xa was incubated with M. sexta plasma, followed by immunoblot analysis using anti-M. sexta PPO antibody. A doublet band around 80 kDa in M. sexta plasma represents heterodimeric proPO consisting of 79-kDa PPO1 and 80-kDa PPO2 [51]. Addition of activated CLIPB4Xa to M. sexta plasma resulted in the appearance of a 70-kDa doublet band corresponding to M. sexta active PO, of which the upper active PO band almost co-migrated with the smaller proPO isoform (Fig. 4a). This proPO and active PO isoform banding pattern is commonly observed in immunoblots of M. sexta hemolymph probed with anti-MsPPO1&2 antibody due to co-migration with storage hexamerins that lead to compression of the bands [51]. The same doublet band was observed in the plasma supplemented with activated CLIPB9Xa, which we identified previously as a functional PAP in An. gambiae [29]. PO activity of plasma increased in the presence of active CLIPB4Xa (Fig. 4c). These results confirm that the proteolytic activity of CLIPB4 promotes proPO cleavage and PO activity. To determine the placement of CLIPB4 in the proPO activation cascade in An. gambiae, we next tested whether CLIPB4 can directly cleave and activate proPO. PPO1&2 was purified from 5th instar M. sexta larvae (Fig. S3c). By Coomassie Blue staining and immunoblot analysis with MsPPO antibody, a dimer was observed around 80 kDa, consistent with previous observations of purified full-length M. sexta PPO1&2 [51]. We then incubated factor Xa-activated rCLIPB4Xa with purified MsPPO1&2, again using rCLIPB9Xa as a positive control. Neither rproCLIPB9Xa zymogen nor factor Xa alone were able to cleave MsPPO1&2, while active rCLIPB9Xa cleaved purified MsPPO1&2 to produce the PO1&2 band at 70 kDa (Fig. 4b). Similarly, neither rproCLIPB4Xa zymogen nor factor Xa alone cleaved MsPPO1&2, but active rCLIPB4Xa cleaved proPO1&2 (Fig. 4b), leading to an increase in PO activity (Fig. 4d), demonstrating that CLIPB4 functions as a PAP.

CLIPB4 cleaves and activates M. sexta PPO1&2. Factor Xa-activated proCLIPB4Xa cleaves MsPPO1&2 in plasma (a) and purified protein (b), detected by immunoblot using rabbit anti-MsPPO1&2 antibody. The light and dark gray arrows indicate the larger isoforms of full and cleaved MsPPO1&2, respectively. CLIPB9 cleavage of M. sexta PPO was used as a positive control. The cleavage by CLIPB4 leads to increased PO activity in plasma (c) and with purified PPO protein (d). Bars represent mean ± 1 SD (n = 3); statistically significant differences are indicated by different letters (one-way ANOVA followed by Newman-Keuls test, p < 0.05).

CLIPB4 Cleaves and Activates CLIPB8We next explored whether CLIPB4 also promotes PO activity indirectly by activating the PAP zymogen proCLIPB9 [29] and/or proCLIPB8, a protease upstream of CLIPB9 in the melanization cascade in An. gambiae [30]. To test whether CLIPB4 can cleave proCLIPB8 and/or B9 in vitro, active CLIPB4Xa was incubated with wild-type recombinant proCLIPB8 and proCLIPB9, respectively, followed by immunoblot analysis using either anti-His antibody or anti-CLIPB8 and anti-CLIPB9 antibodies, respectively. In the reaction of active CLIPB4Xa with proCLIPB8 zymogen, anti-His and anti-CLIPB8 antibodies detected a 36 kDa band, corresponding to the CLIPB8 protease domain, which was also detected in the control reaction of proCLIPB8Xa incubated with factor Xa (Fig. 5a, b). In contrast, addition of active CLIPB4Xa (or factor Xa) did not result in the cleavage of proCLIPB9 (Fig. 5c,d), suggesting that CLIPB4 is not directly upstream of CLIPB9. To evaluate whether cleavage of recombinant proCLIPB8 by CLIPB4Xa results in CLIPB8 activation, we measured the amidase activity of the reaction mixtures using FVR as a substrate [30]. Each of the five control reactions, containing proCLIPB8, proCLIPB8Xa, factor Xa, proCLIPB4Xa, and proCLIPB4Xa/proCLIPB8, respectively, did not exhibit amidase activity. Active CLIPB4Xa showed limited amidase activity, which was increased three-fold when active CLIPB4Xa was incubated with proCLIPB8 prior to the addition of substrate (Fig. 5e). Together with the immunoblot results (Fig. 5a, b), FVRase activity assay supports that CLIPB4 specifically cleaves and activates CLIPB8.Fig. 5.CLIPB4 activates B8, not B9. Recombinant proCLIPB4Xa was activated by factor Xa, and subsequently incubated with 100 ng of recombinant proCLIPB8 or proCLIPB9 at RT for 15 min. Samples were subjected to 12% SDS-PAGE followed by immunoblot with mouse anti-His (a, c), rabbit anti-CLIPB8 (b), and rabbit anti-CLIPB9 (d) antibodies, respectively. In parallel, 100 ng of factor Xa-activated proCLIPB8Xa or proCLIPB9Xa were used as positive controls. Circles, proCLIPB4Xa zymogen; asterisks, catalytic domain of CLIPB4; hollow squares, recombinant proCLIPB8 or proCLIPB9 zymogen; solid squares, catalytic domain of proCLIPB8; hollow triangles, proCLIPB8Xa or proCLIPB9Xa zymogen; solid triangles, catalytic domain of proCLIPB8Xa or proCLIPB9Xa. e 72 ng of factor Xa-activated proCLIPB4Xa was incubated with 450 ng of recombinant proCLIPB8 at RT for 1 h, and the catalytic activity of CLIPB8 was determined using FVR as substrate; 450 ng of factor Xa-activated proCLIPB8Xa was used as a positive control. The bars represent mean ± 1 SD (n = 3); statistically significant differences are indicated by different letters (one-way ANOVA followed by Newman-Keuls test, p < 0.05).To examine this phenomenon in vivo, dsLacZ, dsCLIPB4, and dsCLIPB8 were injected into adult female mosquitoes that were challenged by S. aureus after the dsRNA injection recovery. Hemolymph was collected 10 h after challenge and subjected to SDS-PAGE and immunoblot detection (Fig. 6). Apolipophorin-II (APOII) was utilized as a loading control. CLIPB8 was depleted with dsCLIPB8 treatment, serving as a positive control for CLIPB8 antibody detection. Ten hours after bacterial challenge, a clear reduction of CLIPB8 was observed in dsLacZ compared to the naïve control in two independent experiments, suggesting that CLIPB8 in the hemolymph was rapidly utilized in the acute immune response [57]. The CLIPB8 reduction, mediated by S. aureus challenge, was rescued with dsCLIPB4 treatment in two independent experiments, providing additional support that CLIPB4 acts upstream of CLIPB8 in the hierarchical cSP cascade regulating melanization in An. gambiae.Fig. 6.CLIPB4 is required for infection-induced depletion of CLIPB8 from the hemolymph of An. gambiae. Female mosquitoes were treated by dsLacZ, dsCLIPB4, and dsCLIPB8. Post-recovery, mosquitoes were challenged by S. aureus (OD600 = 0.8). Ten hours postinfection, hemolymph was collected by proboscis clipping, separated by SDS-PAGE, and subjected to immunoblot using rabbit anti-CLIPB8 and rabbit anti-APOII antibodies. Results were shown from two independent experiments.

CLIPB4 Cleaves and Activates CLIPB8

We next explored whether CLIPB4 also promotes PO activity indirectly by activating the PAP zymogen proCLIPB9 [29] and/or proCLIPB8, a protease upstream of CLIPB9 in the melanization cascade in An. gambiae [30]. To test whether CLIPB4 can cleave proCLIPB8 and/or B9 in vitro, active CLIPB4Xa was incubated with wild-type recombinant proCLIPB8 and proCLIPB9, respectively, followed by immunoblot analysis using either anti-His antibody or anti-CLIPB8 and anti-CLIPB9 antibodies, respectively. In the reaction of active CLIPB4Xa with proCLIPB8 zymogen, anti-His and anti-CLIPB8 antibodies detected a 36 kDa band, corresponding to the CLIPB8 protease domain, which was also detected in the control reaction of proCLIPB8Xa incubated with factor Xa (Fig. 5a, b). In contrast, addition of active CLIPB4Xa (or factor Xa) did not result in the cleavage of proCLIPB9 (Fig. 5c,d), suggesting that CLIPB4 is not directly upstream of CLIPB9. To evaluate whether cleavage of recombinant proCLIPB8 by CLIPB4Xa results in CLIPB8 activation, we measured the amidase activity of the reaction mixtures using FVR as a substrate [30]. Each of the five control reactions, containing proCLIPB8, proCLIPB8Xa, factor Xa, proCLIPB4Xa, and proCLIPB4Xa/proCLIPB8, respectively, did not exhibit amidase activity. Active CLIPB4Xa showed limited amidase activity, which was increased three-fold when active CLIPB4Xa was incubated with proCLIPB8 prior to the addition of substrate (Fig. 5e). Together with the immunoblot results (Fig. 5a, b), FVRase activity assay supports that CLIPB4 specifically cleaves and activates CLIPB8.

CLIPB4 activates B8, not B9. Recombinant proCLIPB4Xa was activated by factor Xa, and subsequently incubated with 100 ng of recombinant proCLIPB8 or proCLIPB9 at RT for 15 min. Samples were subjected to 12% SDS-PAGE followed by immunoblot with mouse anti-His (a, c), rabbit anti-CLIPB8 (b), and rabbit anti-CLIPB9 (d) antibodies, respectively. In parallel, 100 ng of factor Xa-activated proCLIPB8Xa or proCLIPB9Xa were used as positive controls. Circles, proCLIPB4Xa zymogen; asterisks, catalytic domain of CLIPB4; hollow squares, recombinant proCLIPB8 or proCLIPB9 zymogen; solid squares, catalytic domain of proCLIPB8; hollow triangles, proCLIPB8Xa or proCLIPB9Xa zymogen; solid triangles, catalytic domain of proCLIPB8Xa or proCLIPB9Xa. e 72 ng of factor Xa-activated proCLIPB4Xa was incubated with 450 ng of recombinant proCLIPB8 at RT for 1 h, and the catalytic activity of CLIPB8 was determined using FVR as substrate; 450 ng of factor Xa-activated proCLIPB8Xa was used as a positive control. The bars represent mean ± 1 SD (n = 3); statistically significant differences are indicated by different letters (one-way ANOVA followed by Newman-Keuls test, p < 0.05).

To examine this phenomenon in vivo, dsLacZ, dsCLIPB4, and dsCLIPB8 were injected into adult female mosquitoes that were challenged by S. aureus after the dsRNA injection recovery. Hemolymph was collected 10 h after challenge and subjected to SDS-PAGE and immunoblot detection (Fig. 6). Apolipophorin-II (APOII) was utilized as a loading control. CLIPB8 was depleted with dsCLIPB8 treatment, serving as a positive control for CLIPB8 antibody detection. Ten hours after bacterial challenge, a clear reduction of CLIPB8 was observed in dsLacZ compared to the naïve control in two independent experiments, suggesting that CLIPB8 in the hemolymph was rapidly utilized in the acute immune response [57]. The CLIPB8 reduction, mediated by S. aureus challenge, was rescued with dsCLIPB4 treatment in two independent experiments, providing additional support that CLIPB4 acts upstream of CLIPB8 in the hierarchical cSP cascade regulating melanization in An. gambiae.

CLIPB4 is required for infection-induced depletion of CLIPB8 from the hemolymph of An. gambiae. Female mosquitoes were treated by dsLacZ, dsCLIPB4, and dsCLIPB8. Post-recovery, mosquitoes were challenged by S. aureus (OD600 = 0.8). Ten hours postinfection, hemolymph was collected by proboscis clipping, separated by SDS-PAGE, and subjected to immunoblot using rabbit anti-CLIPB8 and rabbit anti-APOII antibodies. Results were shown from two independent experiments.

DiscussionMosquitoes interact with a broad range of organisms across the symbiosis continuum and deploy melanization as a broad-spectrum immune defense against many organisms they encounter [58]. The melanization of microbial surfaces in An. gambiae requires opsonization through TEP1 and the activity of PO to produce eumelanin, which depends on the action of cascades made up of cSPs and cSPHs. This study identifies CLIPB4 as an important factor required for melanization in response to microbial infection. CLIPB4 was previously shown to contribute to melanization of glass beads and malaria parasites in CTL4 kd mosquitoes [32, 33]. Here, we show that CLIPB4 is also required for melanotic tumor formation exerted by the depletion of SRPN2 in adult female mosquitoes, demonstrating that its contribution to melanization is not limited to specific surfaces, and suggesting that CLIPB4 is fundamental to eumelanin production. Indeed, our data demonstrate that CLIPB4 functions as a PAP, mediating proPO activation ex vivo and in vitro. In addition to CLIPB9 and B10, CLIPB4 is the third PAP identified in An. gambiae [28, 29]. In all 3 cases, their ability to cleave and activate proPO in vitro was demonstrated using purified proPO1&2 from the hemolymph of M. sexta. Given that An. gambiae PPO5 and 6, the most abundant proPOs in the hemolymph of adult female An. gambiae, share the same activation cleavage site with M. sexta proPO1&2, it is highly likely that all three CLIPBs are bona fide PAPs in An. gambiae [7, 59].The existence of three PAPs, each constituting a terminal protease in a cSP cascade, suggests that melanization in adult female An. gambiae is regulated by at least three proPO activation cleavage cascades. Similarly, M. sexta proPO activation is regulated by at least two proPO activation cascades, utilizing either PAP3 or PAP1/PAP2 as the terminal proteases to cleave proPO [60, 61]. Likewise, two distinct proPO activation cascades regulate melanization in the lepidopteran species, Helicoverpa armigera [15, 62]. In An. gambiae, possibly not all three PAPs contribute equally to activate proPO. Expression of CLIPB4 in adult female An. gambiae, as measured by RNA-seq is much higher than that of CLIPB9, B10, and any other CLIPB whose kd causes a melanization phenotype [63], suggesting that CLIPB4 protein, based on its abundance, contributes more to proPO activation than other PAPs. This observation may also explain why neither CLIPB8, B9, nor B10 were identified in the SRPN2 Co-IP analyses, despite previous data demonstrating that CLIPB9 and SRPN2 form inhibitory complexes in An. gambiae hemolymph [29]. Whether, in addition to protein concentration, the activation of the three An. gambiae PAPs depends on the type of microbial challenge, as observed in H. armigera [62], is currently under investigation.In addition to its PAP function, CLIPB4’s central role in melanization in female An. gambiae is further manifested by its ability to cleave and activate proCLIPB8. This function was demonstrated in vitro by biochemical studies and in vivo, as CLIPB4kd reverts the consumption of CLIPB8 in the hemolymph of adult female An. gambiae upon microbial infection. This is not only the first observation of multiple substrates for a single An. gambiae CLIP protease but also the first demonstration of a CLIPB protease activating another proCLIPB zymogen in mosquitoes, breaking the mold of the classical hierarchy of CLIPC-CLIPB in insect immune protease cascades [16]. The observation of CLIPB4 cleavage of CLIPB8 is paralleled in M. sexta, where PAP3 cleaves proHP5, a cSP-B family member [64]. This cleavage of HP5 provides positive reinforcement of proPO activation, as HP5 in turn cleaves HP6, a cSP in the C family, upstream of PAP1, linking the two proPO activation cascades in M. sexta [64]. Based on these similar patterns in two evolutionarily distant insect species, it is tempting to speculate that positive reinforcement of proPO activation by cSP-B-mediated activation of other cSP-Bs is a common theme in the melanization immune response.Given that the regulation of melanization through proPO activation requires cSPs and cSPHs circulating in mosquito hemolymph prior to immune challenge, we posited that identification of important members in the cSP and cSPH cascades could be identified based on their co-expression. Indeed, the topology of the AgMelGCN, the co-expression network of members of gene families known to contribute to melanization, clustered genes with known immune phenotypes into a single community. Using network science approaches, we then identified highly connected nodes, referred to as hubs, which are integral to scale-free network stability [65]. In gene co-expression networks, hub genes are often relevant for network functionality, either as regulators or because they encode proteins with essential functions [66, 67]. The nodes with highest connectivity in the AgMelGCN encoded two cSPHs (CLIPA9 and 8), APLC1, and CLIPB4. All four genes encode proteins with known functions in innate immunity. CLIPA9 kd in An. gambiae reduces Plasmodium falciparum oocysts, a phenotype that seems dependent on the presence of an unperturbed microbiota through an unknown mechanism [68]. CLIPA8 is a key positive regulator of melanization and part of the cSPH cascade upstream of CLIPC9 [34, 36]. CLIPA8 is required for melanization of malaria parasites after CTL4 kd, filamentous fungi, and bacteria [32, 69, 70]. The leucine-rich repeat protein APLC1 contributes to antimicrobial immunity in mosquitoes by stabilizing TEP1 in the hemolymph, preventing its premature activation, and facilitating its delivery to microbial surfaces for opsonization [38, 71, 72]. Likewise, CLIPB4 plays an important role in melanization through its integral roles in the cSP cascade that activates proPO, as described in this current study. Combined, these data support that gene co-expression networks can indeed be used to identify genes with important functions in mosquito innate immunity and provide a blueprint for analyses of innate immunity in other insect species and likely other complex physiologies in mosquitoes.The topology of the AgMelGCN also demonstrates the integration of the cSPH and cSP cascades through co-expression of members of each cascade. Our Co-IP data indicate that these cascades also at least partially form protein complexes in the hemolymph of An. gambiae. We recently showed that CLIPB4 also cleaves CLIPA8 in vitro, which is required for CLIPA8 function [57], suggesting that the activation of cSPHs required for melanization is mediated by cSPs and potentially integrated with proPO activation through the formation of higher molecular weight protein complexes. Indeed, higher molecular weight complexes between cSPs and cSPHs and proPO have been described in other insect species, including M. sexta [73] and Bombyx mori [74], indicating that the formation of such immune complexes is likely an evolutionarily conserved process that is used to optimize proPO activity on microbial surfaces.Based on our results presented herein and published data, we propose the following model of proPO activation in the melanization response of An. gambiae (Fig. S6). In adult female mosquitoes, circulating proPO zymogen is activated through the proteolytic activity of three CLIPB proteases, CLIPB4, B9, and B10 [28, 29]. CLIPB4 also activates proCLIPB8, which indirectly activates proCLIPB9 [30], providing positive reinforcement of proPO activation. Each of these CLIPBs provides an intervention point, as their activity is inhibited directly by SRPN2, reinforcing its role as the main negative regulator of melanization and survival in An. gambiae [7, 31]. This observation is paralleled in M. sexta, where serpin-3, the ortholog of SRPN2 inhibits all three identified PAPs [75, 76]. CLIPC9, currently the only known CLIPC protease required for melanization, is likely located upstream of one or more of the three PAPs and responsible for their proteolytic activation. In turn, based on the canonical proPO activation cascades in other insects, we hypothesize that proCLIPC9 is activated proteolytically through the action of an unknown ModSP. Additionally, melanization of microbial surfaces is further regulated through the activity of a cSPH cascade upstream of CLIPC9 [34, 36]. Whether cSPHs also support melanization downstream of PAPs, as observed in other insect species [20, 40, 62] is currently under investigation. However, despite the existing complexity of this immune protease network, the presence of 110 CLIP genes in the An. gambiae genome suggests that additional CLIPB and CLIPC proteases may be involved [23, 25, 56]. We are currently performing a reverse genetic screen to identify the contribution of all annotated An. gambiae cSPs and cSPHs to melanization and antimicrobial activity, leveraging the AgMelGCN and recently established in vivo bioassays [34, 77]. Completion of this screen will describe the entire molecular make-up of the An. gambiae immune protease network and enable us to identify the molecular interactions within that are critical for vector competence and survival in this important vector species.

Mosquitoes interact with a broad range of organisms across the symbiosis continuum and deploy melanization as a broad-spectrum immune defense against many organisms they encounter [58]. The melanization of microbial surfaces in An. gambiae requires opsonization through TEP1 and the activity of PO to produce eumelanin, which depends on the action of cascades made up of cSPs and cSPHs. This study identifies CLIPB4 as an important factor required for melanization in response to microbial infection. CLIPB4 was previously shown to contribute to melanization of glass beads and malaria parasites in CTL4 kd mosquitoes [32, 33]. Here, we show that CLIPB4 is also required for melanotic tumor formation exerted by the depletion of SRPN2 in adult female mosquitoes, demonstrating that its contribution to melanization is not limited to specific surfaces, and suggesting that CLIPB4 is fundamental to eumelanin production. Indeed, our data demonstrate that CLIPB4 functions as a PAP, mediating proPO activation ex vivo and in vitro. In addition to CLIPB9 and B10, CLIPB4 is the third PAP identified in An. gambiae [28, 29]. In all 3 cases, their ability to cleave and activate proPO in vitro was demonstrated using purified proPO1&2 from the hemolymph of M. sexta. Given that An. gambiae PPO5 and 6, the most abundant proPOs in the hemolymph of adult female An. gambiae, share the same activation cleavage site with M. sexta proPO1&2, it is highly likely that all three CLIPBs are bona fide PAPs in An. gambiae [7, 59].

The existence of three PAPs, each constituting a terminal protease in a cSP cascade, suggests that melanization in adult female An. gambiae is regulated by at least three proPO activation cleavage cascades. Similarly, M. sexta proPO activation is regulated by at least two proPO activation cascades, utilizing either PAP3 or PAP1/PAP2 as the terminal proteases to cleave proPO [60, 61]. Likewise, two distinct proPO activation cascades regulate melanization in the lepidopteran species, Helicoverpa armigera [15, 62]. In An. gambiae, possibly not all three PAPs contribute equally to activate proPO. Expression of CLIPB4 in adult female An. gambiae, as measured by RNA-seq is much higher than that of CLIPB9, B10, and any other CLIPB whose kd causes a melanization phenotype [63], suggesting that CLIPB4 protein, based on its abundance, contributes more to proPO activation than other PAPs. This observation may also explain why neither CLIPB8, B9, nor B10 were identified in the SRPN2 Co-IP analyses, despite previous data demonstrating that CLIPB9 and SRPN2 form inhibitory complexes in An. gambiae hemolymph [29]. Whether, in addition to protein concentration, the activation of the three An. gambiae PAPs depends on the type of microbial challenge, as observed in H. armigera [62], is currently under investigation.

In addition to its PAP function, CLIPB4’s central role in melanization in female An. gambiae is further manifested by its ability to cleave and activate proCLIPB8. This function was demonstrated in vitro by biochemical studies and in vivo, as CLIPB4kd reverts the consumption of CLIPB8 in the hemolymph of adult female An. gambiae upon microbial infection. This is not only the first observation of multiple substrates for a single An. gambiae CLIP protease but also the first demonstration of a CLIPB protease activating another proCLIPB zymogen in mosquitoes, breaking the mold of the classical hierarchy of CLIPC-CLIPB in insect immune protease cascades [16]. The observation of CLIPB4 cleavage of CLIPB8 is paralleled in M. sexta, where PAP3 cleaves proHP5, a cSP-B family member [64]. This cleavage of HP5 provides positive reinforcement of proPO activation, as HP5 in turn cleaves HP6, a cSP in the C family, upstream of PAP1, linking the two proPO activation cascades in M. sexta [64]. Based on these similar patterns in two evolutionarily distant insect species, it is tempting to speculate that positive reinforcement of proPO activation by cSP-B-mediated activation of other cSP-Bs is a common theme in the melanization immune response.

Given that the regulation of melanization through proPO activation requires cSPs and cSPHs circulating in mosquito hemolymph prior to immune challenge, we posited that identification of important members in the cSP and cSPH cascades could be identified based on their co-expression. Indeed, the topology of the AgMelGCN, the co-expression network of members of gene families known to contribute to melanization, clustered genes with known immune phenotypes into a single community. Using network science approaches, we then identified highly connected nodes, referred to as hubs, which are integral to scale-free network stability [65]. In gene co-expression networks, hub genes are often relevant for network functionality, either as regulators or because they encode proteins with essential functions [66, 67]. The nodes with highest connectivity in the AgMelGCN encoded two cSPHs (CLIPA9 and 8), APLC1, and CLIPB4. All four genes encode proteins with known functions in innate immunity. CLIPA9 kd in An. gambiae reduces Plasmodium falciparum oocysts, a phenotype that seems dependent on the presence of an unperturbed microbiota through an unknown mechanism [68]. CLIPA8 is a key positive regulator of melanization and part of the cSPH cascade upstream of CLIPC9 [34, 36]. CLIPA8 is required for melanization of malaria parasites after CTL4 kd, filamentous fungi, and bacteria [32, 69, 70]. The leucine-rich repeat protein APLC1 contributes to antimicrobial immunity in mosquitoes by stabilizing TEP1 in the hemolymph, preventing its premature activation, and facilitating its delivery to microbial surfaces for opsonization [38, 71, 72]. Likewise, CLIPB4 plays an important role in melanization through its integral roles in the cSP cascade that activates proPO, as described in this current study. Combined, these data support that gene co-expression networks can indeed be used to identify genes with important functions in mosquito innate immunity and provide a blueprint for analyses of innate immunity in other insect species and likely other complex physiologies in mosquitoes.

The topology of the AgMelGCN also demonstrates the integration of the cSPH and cSP cascades through co-expression of members of each cascade. Our Co-IP data indicate that these cascades also at least partially form protein complexes in the hemolymph of An. gambiae. We recently showed that CLIPB4 also cleaves CLIPA8 in vitro, which is required for CLIPA8 function [57], suggesting that the activation of cSPHs required for melanization is mediated by cSPs and potentially integrated with proPO activation through the formation of higher molecular weight protein complexes. Indeed, higher molecular weight complexes between cSPs and cSPHs and proPO have been described in other insect species, including M. sexta [73] and Bombyx mori [74], indicating that the formation of such immune complexes is likely an evolutionarily conserved process that is used to optimize proPO activity on microbial surfaces.

Based on our results presented herein and published data, we propose the following model of proPO activation in the melanization response of An. gambiae (Fig. S6). In adult female mosquitoes, circulating proPO zymogen is activated through the proteolytic activity of three CLIPB proteases, CLIPB4, B9, and B10 [28, 29]. CLIPB4 also activates proCLIPB8, which indirectly activates proCLIPB9 [30], providing positive reinforcement of proPO activation. Each of these CLIPBs provides an intervention point, as their activity is inhibited directly by SRPN2, reinforcing its role as the main negative regulator of melanization and survival in An. gambiae [7, 31]. This observation is paralleled in M. sexta, where serpin-3, the ortholog of SRPN2 inhibits all three identified PAPs [75, 76]. CLIPC9, currently the only known CLIPC protease required for melanization, is likely located upstream of one or more of the three PAPs and responsible for their proteolytic activation. In turn, based on the canonical proPO activation cascades in other insects, we hypothesize that proCLIPC9 is activated proteolytically through the action of an unknown ModSP. Additionally, melanization of microbial surfaces is further regulated through the activity of a cSPH cascade upstream of CLIPC9 [34, 36]. Whether cSPHs also support melanization downstream of PAPs, as observed in other insect species [20, 40, 62] is currently under investigation. However, despite the existing complexity of this immune protease network, the presence of 110 CLIP genes in the An. gambiae genome suggests that additional CLIPB and CLIPC proteases may be involved [23, 25, 56]. We are currently performing a reverse genetic screen to identify the contribution of all annotated An. gambiae cSPs and cSPHs to melanization and antimicrobial activity, leveraging the AgMelGCN and recently established in vivo bioassays [34, 77]. Completion of this screen will describe the entire molecular make-up of the An. gambiae immune protease network and enable us to identify the molecular interactions within that are critical for vector competence and survival in this important vector species.

We would like to thank all members of the Michel laboratory for mosquito rearing. We thank the former Michel laboratory members, Ms. Erin Peel for helping with the reverse genetic experiments, and Dr. Xin Zhang for initial plasmid construction and expression of recombinant proteins. We are grateful to Dr. John Tomich and the members of the Biotechnology/Proteomics Core Facility, Kansas State University for the MS analysis of the SRPN2/CLIPB4 complexes. We thank Dr. Susan Paskewitz, University of Wisconsin for the kind gift of the CLIPB4 antibody. We gratefully acknowledge Ms. Rebekah J. Woolsey and Dr. David R. Quilici at the Mick Hitchcock, Ph.D. Nevada Proteomics Center, University of Nevada, Reno, for providing the MS analysis of the Co-IP experiments. Finally, sincere thanks go to Dr. Michael R. Kanost, Kansas State University, for his continued support and mentorship.

Statement of EthicsAn ethics statement was not required for this study type, no human or animal subjects or materials were used.

Statement of Ethics

An ethics statement was not required for this study type, no human or animal subjects or materials were used.

Conflict of Interest StatementThe authors disclosed all potential conflicts of interest, and none existed for the presented work.

Conflict of Interest Statement

The authors disclosed all potential conflicts of interest, and none existed for the presented work.

K.M. conceived the study, and C.A., C.T.C., M.A.O., C.S., and K.M. designed the experiments and supervised their execution. X.Z., S.Z., J.K., K.A.S., B.M. V., S.A.S., M.L., E.C.M., and K.M. performed the experiments and analyzed the data. X.Z., S.Z., and K.M. prepared the manuscript, and X.Z., S.Z., M.A.O., C.S., and K.M. revised the manuscript. All authors approved the submitted version.
