---
pmid: '22447446'
title: Interactions between DNA, transcriptional regulator Dreb2a and the Med25 mediator
  subunit from Arabidopsis thaliana involve conformational changes.
authors:
- Blomberg J
- Aguilar X
- Brännström K
- Rautio L
- Olofsson A
- Wittung-Stafshede P
- Björklund S
journal: Nucleic Acids Res
year: '2012'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC3401450
doi: 10.1093/nar/gks265
---

# Interactions between DNA, transcriptional regulator Dreb2a and the Med25 mediator subunit from Arabidopsis thaliana involve conformational changes.
**Authors:** Blomberg J, Aguilar X, Brännström K, Rautio L, Olofsson A, Wittung-Stafshede P, Björklund S
**Journal:** Nucleic Acids Res (2012)
**DOI:** [10.1093/nar/gks265](https://doi.org/10.1093/nar/gks265)
**PMC:** [PMC3401450](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3401450/)

## Abstract

1. Nucleic Acids Res. 2012 Jul;40(13):5938-50. doi: 10.1093/nar/gks265. Epub 2012
 Mar 24.

Interactions between DNA, transcriptional regulator Dreb2a and the Med25 
mediator subunit from Arabidopsis thaliana involve conformational changes.

Blomberg J(1), Aguilar X, Brännström K, Rautio L, Olofsson A, Wittung-Stafshede 
P, Björklund S.

Author information:
(1)Department of Medical Biochemistry and Biophysics, Umeå University, SE-901 87 
Umeå, Sweden.

Mediator is a multiprotein coregulatory complex that conveys signals from 
DNA-bound transcriptional regulators to the RNA polymerase II transcription 
machinery in eukaryotes. The molecular mechanisms for how these signals are 
transmitted are still elusive. By using purified transcription factor Dreb2a, 
mediator subunit Med25 from Arabidopsis thaliana, and a combination of 
biochemical and biophysical methods, we show that binding of Dreb2a to its 
canonical DNA sequence leads to an increase in secondary structure of the 
transcription factor. Similarly, interaction between the Dreb2a and Med25 in the 
absence of DNA results in conformational changes. However, the presence of the 
canonical Dreb2a DNA-binding site reduces the affinity between Dreb2a and Med25. 
We conclude that transcription regulation is facilitated by small but distinct 
changes in energetic and structural parameters of the involved proteins.

DOI: 10.1093/nar/gks265
PMCID: PMC3401450
PMID: 22447446 [Indexed for MEDLINE]

## Full Text

INTRODUCTIONEukaryotic cells express a set of general transcription factors (GTFs: TBP, TFIIB, TFIIE, TFIIF and TFIIH) and RNA polymerase II, which are required for transcription of all protein-encoding genes. Together, the GTFs form a so called pre-initiation complex (PIC) that assembles close to the transcription start site. Transcription is regulated at several levels; most importantly by changes in chromatin structure and by transcriptional regulators (activators or repressors) that bind to specific DNA sequences. In the latter process, transcriptional regulators execute their function by affecting different steps in transcription such as formation of the PIC (1,2), promoter escape (3) and elongation (4–6). There are several examples of direct interactions between DNA-bound transcriptional regulators and components of the general transcription machinery, but these interactions are not sufficient to support regulated transcription. Instead, transcriptional regulators operate through interactions with different types of coregulators (coactivators and corepressors), which transmit signals to the general transcription machinery. Coregulators are not believed to interact with DNA, but rather to form bridges between the DNA-bound activators/repressors and the GTFs. The most important coregulator is mediator, a multisubunit protein complex which is required for proper expression of nearly all genes in all eukaryotes (7). Mediator was initially identified in yeast (8,9), but it is now clear that mediator is present in all eukarya and a common terminology for mediator subunits in all species has been proposed (10,11)Even though the subunit composition and the structure of mediator are well described, less is known about how mediator operates to integrate and convey signals from DNA-bound transcriptional regulators. This is in part due to its complexity and size. Direct interactions between transcriptional regulators and different mediator subunits are believed to recruit and stabilize mediator at the promoter, and there are indications that interactions with transcriptional regulators induce conformational changes or structural shifts in mediator (12–14). It is possible that such conformational changes in mediator are propagated to induce further structural shifts in components of the general transcription machinery. These changes, combined with other signal pathways that cause posttranslational modifications of different transcription factors, might therefore determine the level of transcription from each gene.We recently identified mediator in Arabidopsis thaliana and we have also reported that the Med25 subunit in Arabidopsis interacts with three different transcription factors, Dreb2a, Zfhd1 and myb-like which all are involved in different stress response pathways (15,16) In this article, we use a set of biochemical and biophysical methods to study interaction between the Med25 mediator subunit and the transcription factor Dreb2a from A.
thaliana. We also investigate how this interaction is affected by the presence of the cognate DNA-binding site for Dreb2a. Our results show that functional interactions between Dreb2a and Med25 induce conformational changes which in turn cause changes in their binding affinity.

INTRODUCTION

Eukaryotic cells express a set of general transcription factors (GTFs: TBP, TFIIB, TFIIE, TFIIF and TFIIH) and RNA polymerase II, which are required for transcription of all protein-encoding genes. Together, the GTFs form a so called pre-initiation complex (PIC) that assembles close to the transcription start site. Transcription is regulated at several levels; most importantly by changes in chromatin structure and by transcriptional regulators (activators or repressors) that bind to specific DNA sequences. In the latter process, transcriptional regulators execute their function by affecting different steps in transcription such as formation of the PIC (1,2), promoter escape (3) and elongation (4–6). There are several examples of direct interactions between DNA-bound transcriptional regulators and components of the general transcription machinery, but these interactions are not sufficient to support regulated transcription. Instead, transcriptional regulators operate through interactions with different types of coregulators (coactivators and corepressors), which transmit signals to the general transcription machinery. Coregulators are not believed to interact with DNA, but rather to form bridges between the DNA-bound activators/repressors and the GTFs. The most important coregulator is mediator, a multisubunit protein complex which is required for proper expression of nearly all genes in all eukaryotes (7). Mediator was initially identified in yeast (8,9), but it is now clear that mediator is present in all eukarya and a common terminology for mediator subunits in all species has been proposed (10,11)

Even though the subunit composition and the structure of mediator are well described, less is known about how mediator operates to integrate and convey signals from DNA-bound transcriptional regulators. This is in part due to its complexity and size. Direct interactions between transcriptional regulators and different mediator subunits are believed to recruit and stabilize mediator at the promoter, and there are indications that interactions with transcriptional regulators induce conformational changes or structural shifts in mediator (12–14). It is possible that such conformational changes in mediator are propagated to induce further structural shifts in components of the general transcription machinery. These changes, combined with other signal pathways that cause posttranslational modifications of different transcription factors, might therefore determine the level of transcription from each gene.

We recently identified mediator in Arabidopsis thaliana and we have also reported that the Med25 subunit in Arabidopsis interacts with three different transcription factors, Dreb2a, Zfhd1 and myb-like which all are involved in different stress response pathways (15,16) In this article, we use a set of biochemical and biophysical methods to study interaction between the Med25 mediator subunit and the transcription factor Dreb2a from A.
thaliana. We also investigate how this interaction is affected by the presence of the cognate DNA-binding site for Dreb2a. Our results show that functional interactions between Dreb2a and Med25 induce conformational changes which in turn cause changes in their binding affinity.

MATERIALS AND METHODSProtein purification and expressionPurification of Med25551–680 and Dreb2a168–335 was carried out as described previously (16). The GST-tag of Med25551–680 that is removed by cleavage with Prescission Enzyme (GE Healthcare) was collected and stored in −20°C before use as a control for the experiments described in Figure 7. The nucleotide sequence corresponding to amino acids 168–253 (Dreb2a168–253) and 254–335 (Dreb2a254–335) was amplified from cDNA with polymerase chain reaction and cloned into the NcoI/NotI sites of the pETM-6×his vector (kindly provided by Günter Stier, EMBL, Germany). The primers used were: 5′-GCTACCATGGATTGTGAATCTAAACCCTTCT-3′and 5′-GCTTGCGGCCGCTTACAAGTGACTCTGATCCACATG-3′ for Dreb2a168–253 and 5′-GCTACCATGGATTCTTCAGACATGTTTGATG-3′ and 5′-GCTTGCGGCCGCTTAGTTCTCCAGATCCAAGTAACT-3′ for Dreb2a254–335. Dreb2a168–253 and Dreb2a245–335 were cloned into the pETM-6×his vector and produced as described for Dreb2a168–335 (16). Final purification of Dreb2a168–253 and Dreb2a254–335 was made with ion exchange chromatography using a Mono Q 5/50 GL (GE Healthcare). The proteins were applied to the columns in 25 mM sodium phosphate buffer pH 6.2 and sodium acetate buffer pH 5.2, respectively. Both buffers contained 1 mM dithiothreitol (DTT) and a starting concentration of 20 mM NaCl. Adsorbed proteins were eluted with a linear gradient of 18 column volumes to 0.8 M NaCl on an ÄKTAexplorer (GE Healthcare). Full-length Dreb2a, Dreb2af.1., was cloned into a pETM-6×his-GB1 vector at the NcoI/NotI sites and the resulting plasmid was transfected into the Escherichia coli Rosetta (DE3) pLysS strain. The bacteria were grown at 30°C in Terrific Broth supplemented with 100 µg/ml kanamycin and 34 µg/ml chloramphenicol. Protein expression was induced by addition of isopropyl β-d-thiogalactopyranoside (IPTG) to 1 mM at OD600 = 1.5. After 6 h of induction, cells were harvested by centrifugation and pellets were lysed in 20 mM Tris pH 8.0, 300 mM NaCl, 0.2% NP40 and 2 mM β-mercaptoethanol. The lysates were sonicated on ice with a Branson Sonifier 450 equipped with a microtip (VWR, Sweden). The following settings were applied: duty cycle 50%, output control 3.5, time 8 min. The lysates were then cleared by centrifugation at 35 000 g for 1 h using a 45Ti rotor (Beckman Coulter AB, Sweden), and Dreb2af.1. was purified from the supernatant by ammonium sulfate precipitation (20%). Precipitated proteins were dissolved in 20 mM Tris pH 7.5, 100 mM NaCl and 2 mM β-mercaptoethanol and loaded onto a Mono S 5/50 GL (GE Healthcare). Adsorbed proteins were eluted with a linear gradient to 800 mM NaCl. Fractions containing Dreb2af.1. were pooled, and dialyzed against 20 mM Tris pH 7.5, 150 mM NaCl and 2 mM β-mercaptoethanol.The pET21b vector containing the human Med25–ACID domain, hMed25394–543, was a generous gift from Patrick Cramer and Michael Sattler. The vector was transformed into E. coli BL21 pLysS and the bacteria were grown at 20°C in Terrific Broth supplemented with 100 µg/ml carbenicillin and 34 µg/ml chloramphenicol. Protein expression was induced at OD600 = 1.5 with 0.1 mM IPTG for 16 h. Cell pellets were lysed in lysis buffer (50 mM sodium phosphate pH 8.0, 300 mM NaCl, 20 mM Imidazole, 0.2% NP40, 2 mM β-mercaptoethanol). A cleared supernatant was obtained by sonication (15 s on, 30 s off, output 3.5, 2 × 8 min) and centrifugation (as described above). Protein purification was made on a Ni–NTA column (Qiagen). The column was washed twice with 10 column volumes lysis buffer and three times with wash buffer (50 mM sodium phosphate pH 8.0, 300 mM NaCl, 30 mM imidazole, 2 mM β-mercaptoethanol). Elution was made in wash buffer supplemented with 300 mM imidazole. Eluted proteins were dialyzed against 20 mM Tris pH 7.0, 50 mM NaCl, 1 mM DTT and further purified on a Mono S 5/50 GL as described above. The purified protein eluted in ∼270 mM NaCl. Fractions with purified hMed25394–543 were pooled and aliquots were stored in −20°C before analysis. The purity of the proteins used was assayed with SDS–PAGE and Coomassie stain using Page Blue (Fermentas). The protein concentrations were initially determined with Bradford protein reagent (Fermentas) and during time of experiments confirmed by photospectrometry at A 280 nm or with a NanoDrop ND-1000 spectrophotometer (Saveen Werner).Surface plasmon resonance analysisBinding experiments were performed at 25°C on a Biacore3000 (GE healthcare) and analyzed with the Scrubber 2 software (BioLogic Software). Attachment of Med25551–680 and streptavidine to a Biacore CM5 sensor chip was made with an amine coupling kit according to the recommendations from GE Healthcare. Approximately 1500 resonance units (RU) of Med25551–680 were immobilized in 10 mM potassium phosphate buffer pH 7.0 with HBS-EP buffer as running buffer [10 mM HEPES pH 7.5, 150 mM NaCl, 3.4 mM EDTA and 0.005% non-ionic surfactant polyoxyethylenesorbitan (P2O) (GE Healthcare)]. Triplicates of decreasing concentrations of Dreb2a168–335, Dreb2a168–253 and Dreb2a254–335 (8 µM with 2-fold dilutions down to 0.031 µM) were injected at 20 µl/min for 3 min in running buffer (20 mM Tris pH 7.5, 150 mM NaCl and 1 mM DTT) and the chip was regenerated with 2 M MgCl2 followed by running buffer (flow 100 µl/min, 20 µl each) between each sample. Duplicates of Dreb2af.l. was injected in the same Tris-running buffer supplemented with 2 mM MgCl2. Between different concentrations (3 µM down to 0.047 µM with 2-fold dilution), Dreb2af.l. was washed away from the chip with a quick injection (100 µl/min) of 20 µl of regeneration buffer (50 mM NaOH, 1 M NaCl) followed by a quick inject of 20 µl running buffer. Two 29-nt long oligonucleotides, 5′-AAAGATATACTACCGACATGAGTTCCAAA-3′ and 5′-AAAGATATACTACTTTTATGAGTTCCAAA-3′, that contain the Dreb2a-binding sequence DRE or a mutated version of the DRE sequence from the rd29A promoter was used for experiments with Dreb2af.l. in conjugation with DNA (16). An amount of 2, 1 and 0.5 µM Dreb2af.l. were incubated with 4 µM of annealed oligonucleotides in running buffer containing 2 mM MgCl2 for 10 min prior to injection. Washes were made between each injection as described for Dreb2af.l. alone.For immobilization of DNA to the sensor chip, 50 µg/ml streptavidine (Sigma Aldrich) was injected at 5 µl/min until approximately 1000 RU was reached. After blockage with ethanolamine, biotin-labeled annealed oligonucleotide (10 µM) was injected at 20 µl/min in HBS-EP buffer until 350 RU was reached. A 71-nt long oligonucleotide from the promoter region of rd29A (described above), labeled at the 5′-end with biotin-TEG (5′ [Btn-Tg]-CAGTTTGAAAGAAAAGGGAAAAAAAGAAAAAATAAATAAAAGATATACTACCGACATGAGTTCCAAAAAGC-3′) was used (synthesized by Sigma Aldrich). Biotin–TEG-labeled oligonucleotides have an extra spacer with mixed polarity based on triethylene glycol that confers better binding to streptavidine than unmodified biotin. The complementary unlabeled oligonucleotide was annealed to the biotinylated oligonucleotide in 2-fold excess to achieve maximal dimerization of the biotin-labeled oligonucleotide. Dreb2af.l. was injected and the chip was washed as described in the previous paragraph.The nitrilotriacetic acid (NTA) sensor chip was subjected to 100 mM NiCl2 in HBS-buffer without EDTA at 5 µl/min for 5 min to obtain a metal chelating sensor surface. After activation, Df.l. (6.5 µM) in HBS buffer without EDTA was injected at 5 µl/min until 1100 RU was obtained. The chip was blocked with ethanolamine and unbound Ni2+ was subsequently removed with 250 mM EDTA at 5 µl/min for 7 min. HumMed25394–543 was injected from 3 µM down to 0.047 µM with 2-fold dilution in Tris running buffer supplemented with 5 mM MgCl2. Between the samples, the chip was washed at 100 µl/min with 35 µl of regeneration buffer (50 mM NaOH, 1 M NaCl), 35 µl of 250 mM EDTA and finally 35 µl running buffer.In all experiments, a reference surface that had been activated and subsequently blocked was used. As an additional reference, injection of buffer controls was made in all experiments.Far-UV circular dichroism spectroscopyFar-UV circular dichroism (CD) spectra were acquired on a Jasco J-720 spectropolarimeter using a quartz cuvette with a 1-mm path length. The spectra were recorded at a wavelength range of 200–300 nm with 0.5 nm data interval using a scan rate of 50 nm/min. Each spectrum was an average of five scans. Each spectrum was corrected by subtracting buffer and buffer–DNA signals, respectively, and all experiments were executed at least twice. The experiments were conducted with fixed concentrations of proteins (10 and 20 µM) in the absence and presence of specific and mutated oligonucleotides (10 µM). The molar ratio of DNA:protein was always 1:1. For protein mixture experiments with DNA, Dreb2af.1. was first pre-incubated with specific or mutated DNA for 1 h at room temperature before addition of Med25551–680. All samples were incubated in a 20 mM Tris, 150 mM NaCl, 2 mM β-mercaptoethanol, 5 mM MgCl2, pH 7.5 buffer at 20°C and for CD measurements as a function of time at 4°C. For the ethylene glycol (Sigma Aldrich) samples, the final concentration was fixed to 300 mg/ml.Gel electrophoresis mobility shift assaysThe 29-nt oligonucleotides described above were annealed and the probes were labeled with 32P-ATP using T4 polynucleotide kinase (Fermentas). An amount of 1 µM Dreb2af.l. was incubated with the labeled probes (∼20 000 cpm or 20 fmol /reaction) in 25 mM HEPES pH 7.5, 50 mM KCl, 5 mM MgCl2, 10% glycerol, 1 µg/ml poly dIdC (GE Healthcare), 1 µg/µl BSA and 1 mM DTT. After incubation for 10 min at room temperature, 1 µM or 4 µM of Med25551–680 was added and the reaction was incubated for an additional 10 min at room temperature. For competition with unlabeled probe, 1.3 µM cold probe was added at the start of the reaction. The samples were separated on a 3.5% polyacrylamide Tris–Borate–EDTA gel and the dried gel was exposed to a phosphor storage screen and imaged with a Typhoon 9400 phosphorimager (GE Healthcare).Size exclusion chromatographyA HiLoad 16/60 Superdex 200 (GE Healthcare) was calibrated with ribonuclease A (13.7 kDa), ovalbumin (44 kDa), conalbumin (75 kDa) and aldolase (158 kDa) from the gel-calibration kits supplied by the same manufacturer. The void volume (Vo) of the column was also determined to 40 ml with blue dextran 2000. An amount of 12 µM of Dreb2af.l., Med25551–680 and Med25551–680–GST were loaded in 500 µl. When Dreb2af.l. and Med25551–680 or Med25551–680–GST were mixed, samples were incubated for 15 min at room temperature followed by centrifugation for 15 min at 20 000 g before loaded onto the column. All samples were run in 20 mM Tris pH 7.5, 150 mM NaCl and 1 mM DTT at 4°C and the proteins were separated in 0.5 ml fractions. The presence of Dreb2af.l., Med25551–680 and Med25551–680–GST in the eluted fractions was detected using immunoblotting. The proteins were concentrated through precipitation with a 4-fold excess of acetone in −20°C over night. Precipitated proteins were collected through centrifugation at 20 000 g for 15 min. Air dried pellets were dissolved in SDS-loading buffer to a final concentration of 75 mM Tris pH 6.8, 1% SDS, 10% glycerol, 0.7 M β-mercaptoethanol and 0.00125% Bromophenol blue.To validate the interaction between Dreb2af.l. and Med25551–680–GST, samples were separated on a 10% SDS–PAGE and subjected to immunoblotting. Since the GB1-tag on Dreb2af.l. is a strong immunoglobulin-binding protein, secondary antibody can be used to directly identify the GB1-Dreb2af.l. protein. Thus, the 50–100 kDa part of the PVDF membrane was incubated with anti-mouse secondary antibody (1:5000) for 3 h in room temperature, washed and developed with ECL solution (all reagents from GE Healthcare). The 25–50 kDa part of the membrane was incubated with anti-GST antibody (1:2000, a generous gift from professor Sven Carlsson, Umea University, Sweden) at 4°C over night, washed and incubated with anti-rabbit antibody (1:5000, GE Healthcare, Sweden) for 1 h in room temperature. All membranes were blocked in TBST (20 mM Tris pH 7.5, 150 mM NaCl and 0.1% Tween-20) containing 5% milk for 1 h before addition of antibody. All antibodies were diluted in TBST. To verify that Med25551–680 eluted in complex with Dreb2af.l., samples were loaded on a 4–12% NuPAGE® Bis–Tris gel (Invitrogen) and run in MES SDS running buffer (50 mM Tris, 50 mM MES, 0.1% SDS and 1 mM EDTA). Proteins were visualized with Coomassie stain using PageBlue.Formaldehyde fixationAn amount of 1 µM Dreb2af.l. was mixed with Med25551–680, Med25551–680–GST or hMed25394–543 (1 µM or 4 µM) for 20 min in room temperature in the same buffer as the one used for gel electrophoresis mobility shift assays (GEMSA) (25 mM HEPES pH 7.5, 50 mM KCl, 5 mM MgCl2, 10% glycerol, 1 µg/ml poly dIdC 1 µg/µl BSA and 1 mM DTT). Formed complexes were cross-linked by the addition of formaldehyde to a final concentration of 0.5% for 30 min in room temperature. The cross-linked proteins were mixed with SDS-loading buffer, loaded onto a 3–8% NuPAGE® Novex Tris–Acetate gel (Invitrogen) and run in Tris–Acetate SDS-running buffer (50 mM Tris–base, 50 mM Tricine, 0.1% SDS pH 8.24). Dreb2af.l. boiled for 5 min was included as a denatured control. The migration of Dreb2af.l. was visualized by immunoblotting as described above.Pull-down assaysAn amount of 6 µM Dreb2af.l. was bond to 15 µl IgG sepharose 6 fast flow (GE Healthcare) for 30 min in binding buffer (20 mM Tris pH 7.5, 150 mM NaCl, 2 mM β-mercaptoethanol) at room temperature. hMed25394–543 was added to a final concentration of 6 µM and the samples were incubated for 30 min. Unbound protein were collected in flow through and the beads were washed two times in 1 ml of binding buffer containing 0.05% Tween-20. Bound proteins were extracted by addition of 1× SDS-loading buffer and boiled for 5 min. The samples were loaded on a 4–12% NuPAGE® Bis–Tris gel (Invitrogen) and run in MOPS SDS-running buffer (50 mM Tris pH 7.7, 50 mM MES, 0.1% SDS and 1 mM EDTA). The proteins were detected as described for size exclusion chromatography except that hMed25394–543 was visualized by incubation with anti-poly His antibody (1:2000, H1029, Sigma Aldrich).Prediction of secondary structureThe prediction of secondary structures was made with the Jpred 3 server, available at www.compbio.dundee.ac.uk/jpred. It has been reported to predict secondary structure of proteins at an accuracy of 81.5% (17).

MATERIALS AND METHODS

Protein purification and expressionPurification of Med25551–680 and Dreb2a168–335 was carried out as described previously (16). The GST-tag of Med25551–680 that is removed by cleavage with Prescission Enzyme (GE Healthcare) was collected and stored in −20°C before use as a control for the experiments described in Figure 7. The nucleotide sequence corresponding to amino acids 168–253 (Dreb2a168–253) and 254–335 (Dreb2a254–335) was amplified from cDNA with polymerase chain reaction and cloned into the NcoI/NotI sites of the pETM-6×his vector (kindly provided by Günter Stier, EMBL, Germany). The primers used were: 5′-GCTACCATGGATTGTGAATCTAAACCCTTCT-3′and 5′-GCTTGCGGCCGCTTACAAGTGACTCTGATCCACATG-3′ for Dreb2a168–253 and 5′-GCTACCATGGATTCTTCAGACATGTTTGATG-3′ and 5′-GCTTGCGGCCGCTTAGTTCTCCAGATCCAAGTAACT-3′ for Dreb2a254–335. Dreb2a168–253 and Dreb2a245–335 were cloned into the pETM-6×his vector and produced as described for Dreb2a168–335 (16). Final purification of Dreb2a168–253 and Dreb2a254–335 was made with ion exchange chromatography using a Mono Q 5/50 GL (GE Healthcare). The proteins were applied to the columns in 25 mM sodium phosphate buffer pH 6.2 and sodium acetate buffer pH 5.2, respectively. Both buffers contained 1 mM dithiothreitol (DTT) and a starting concentration of 20 mM NaCl. Adsorbed proteins were eluted with a linear gradient of 18 column volumes to 0.8 M NaCl on an ÄKTAexplorer (GE Healthcare). Full-length Dreb2a, Dreb2af.1., was cloned into a pETM-6×his-GB1 vector at the NcoI/NotI sites and the resulting plasmid was transfected into the Escherichia coli Rosetta (DE3) pLysS strain. The bacteria were grown at 30°C in Terrific Broth supplemented with 100 µg/ml kanamycin and 34 µg/ml chloramphenicol. Protein expression was induced by addition of isopropyl β-d-thiogalactopyranoside (IPTG) to 1 mM at OD600 = 1.5. After 6 h of induction, cells were harvested by centrifugation and pellets were lysed in 20 mM Tris pH 8.0, 300 mM NaCl, 0.2% NP40 and 2 mM β-mercaptoethanol. The lysates were sonicated on ice with a Branson Sonifier 450 equipped with a microtip (VWR, Sweden). The following settings were applied: duty cycle 50%, output control 3.5, time 8 min. The lysates were then cleared by centrifugation at 35 000 g for 1 h using a 45Ti rotor (Beckman Coulter AB, Sweden), and Dreb2af.1. was purified from the supernatant by ammonium sulfate precipitation (20%). Precipitated proteins were dissolved in 20 mM Tris pH 7.5, 100 mM NaCl and 2 mM β-mercaptoethanol and loaded onto a Mono S 5/50 GL (GE Healthcare). Adsorbed proteins were eluted with a linear gradient to 800 mM NaCl. Fractions containing Dreb2af.1. were pooled, and dialyzed against 20 mM Tris pH 7.5, 150 mM NaCl and 2 mM β-mercaptoethanol.The pET21b vector containing the human Med25–ACID domain, hMed25394–543, was a generous gift from Patrick Cramer and Michael Sattler. The vector was transformed into E. coli BL21 pLysS and the bacteria were grown at 20°C in Terrific Broth supplemented with 100 µg/ml carbenicillin and 34 µg/ml chloramphenicol. Protein expression was induced at OD600 = 1.5 with 0.1 mM IPTG for 16 h. Cell pellets were lysed in lysis buffer (50 mM sodium phosphate pH 8.0, 300 mM NaCl, 20 mM Imidazole, 0.2% NP40, 2 mM β-mercaptoethanol). A cleared supernatant was obtained by sonication (15 s on, 30 s off, output 3.5, 2 × 8 min) and centrifugation (as described above). Protein purification was made on a Ni–NTA column (Qiagen). The column was washed twice with 10 column volumes lysis buffer and three times with wash buffer (50 mM sodium phosphate pH 8.0, 300 mM NaCl, 30 mM imidazole, 2 mM β-mercaptoethanol). Elution was made in wash buffer supplemented with 300 mM imidazole. Eluted proteins were dialyzed against 20 mM Tris pH 7.0, 50 mM NaCl, 1 mM DTT and further purified on a Mono S 5/50 GL as described above. The purified protein eluted in ∼270 mM NaCl. Fractions with purified hMed25394–543 were pooled and aliquots were stored in −20°C before analysis. The purity of the proteins used was assayed with SDS–PAGE and Coomassie stain using Page Blue (Fermentas). The protein concentrations were initially determined with Bradford protein reagent (Fermentas) and during time of experiments confirmed by photospectrometry at A 280 nm or with a NanoDrop ND-1000 spectrophotometer (Saveen Werner).

Protein purification and expression

Purification of Med25551–680 and Dreb2a168–335 was carried out as described previously (16). The GST-tag of Med25551–680 that is removed by cleavage with Prescission Enzyme (GE Healthcare) was collected and stored in −20°C before use as a control for the experiments described in Figure 7. The nucleotide sequence corresponding to amino acids 168–253 (Dreb2a168–253) and 254–335 (Dreb2a254–335) was amplified from cDNA with polymerase chain reaction and cloned into the NcoI/NotI sites of the pETM-6×his vector (kindly provided by Günter Stier, EMBL, Germany). The primers used were: 5′-GCTACCATGGATTGTGAATCTAAACCCTTCT-3′and 5′-GCTTGCGGCCGCTTACAAGTGACTCTGATCCACATG-3′ for Dreb2a168–253 and 5′-GCTACCATGGATTCTTCAGACATGTTTGATG-3′ and 5′-GCTTGCGGCCGCTTAGTTCTCCAGATCCAAGTAACT-3′ for Dreb2a254–335. Dreb2a168–253 and Dreb2a245–335 were cloned into the pETM-6×his vector and produced as described for Dreb2a168–335 (16). Final purification of Dreb2a168–253 and Dreb2a254–335 was made with ion exchange chromatography using a Mono Q 5/50 GL (GE Healthcare). The proteins were applied to the columns in 25 mM sodium phosphate buffer pH 6.2 and sodium acetate buffer pH 5.2, respectively. Both buffers contained 1 mM dithiothreitol (DTT) and a starting concentration of 20 mM NaCl. Adsorbed proteins were eluted with a linear gradient of 18 column volumes to 0.8 M NaCl on an ÄKTAexplorer (GE Healthcare). Full-length Dreb2a, Dreb2af.1., was cloned into a pETM-6×his-GB1 vector at the NcoI/NotI sites and the resulting plasmid was transfected into the Escherichia coli Rosetta (DE3) pLysS strain. The bacteria were grown at 30°C in Terrific Broth supplemented with 100 µg/ml kanamycin and 34 µg/ml chloramphenicol. Protein expression was induced by addition of isopropyl β-d-thiogalactopyranoside (IPTG) to 1 mM at OD600 = 1.5. After 6 h of induction, cells were harvested by centrifugation and pellets were lysed in 20 mM Tris pH 8.0, 300 mM NaCl, 0.2% NP40 and 2 mM β-mercaptoethanol. The lysates were sonicated on ice with a Branson Sonifier 450 equipped with a microtip (VWR, Sweden). The following settings were applied: duty cycle 50%, output control 3.5, time 8 min. The lysates were then cleared by centrifugation at 35 000 g for 1 h using a 45Ti rotor (Beckman Coulter AB, Sweden), and Dreb2af.1. was purified from the supernatant by ammonium sulfate precipitation (20%). Precipitated proteins were dissolved in 20 mM Tris pH 7.5, 100 mM NaCl and 2 mM β-mercaptoethanol and loaded onto a Mono S 5/50 GL (GE Healthcare). Adsorbed proteins were eluted with a linear gradient to 800 mM NaCl. Fractions containing Dreb2af.1. were pooled, and dialyzed against 20 mM Tris pH 7.5, 150 mM NaCl and 2 mM β-mercaptoethanol.

The pET21b vector containing the human Med25–ACID domain, hMed25394–543, was a generous gift from Patrick Cramer and Michael Sattler. The vector was transformed into E. coli BL21 pLysS and the bacteria were grown at 20°C in Terrific Broth supplemented with 100 µg/ml carbenicillin and 34 µg/ml chloramphenicol. Protein expression was induced at OD600 = 1.5 with 0.1 mM IPTG for 16 h. Cell pellets were lysed in lysis buffer (50 mM sodium phosphate pH 8.0, 300 mM NaCl, 20 mM Imidazole, 0.2% NP40, 2 mM β-mercaptoethanol). A cleared supernatant was obtained by sonication (15 s on, 30 s off, output 3.5, 2 × 8 min) and centrifugation (as described above). Protein purification was made on a Ni–NTA column (Qiagen). The column was washed twice with 10 column volumes lysis buffer and three times with wash buffer (50 mM sodium phosphate pH 8.0, 300 mM NaCl, 30 mM imidazole, 2 mM β-mercaptoethanol). Elution was made in wash buffer supplemented with 300 mM imidazole. Eluted proteins were dialyzed against 20 mM Tris pH 7.0, 50 mM NaCl, 1 mM DTT and further purified on a Mono S 5/50 GL as described above. The purified protein eluted in ∼270 mM NaCl. Fractions with purified hMed25394–543 were pooled and aliquots were stored in −20°C before analysis. The purity of the proteins used was assayed with SDS–PAGE and Coomassie stain using Page Blue (Fermentas). The protein concentrations were initially determined with Bradford protein reagent (Fermentas) and during time of experiments confirmed by photospectrometry at A 280 nm or with a NanoDrop ND-1000 spectrophotometer (Saveen Werner).

Surface plasmon resonance analysisBinding experiments were performed at 25°C on a Biacore3000 (GE healthcare) and analyzed with the Scrubber 2 software (BioLogic Software). Attachment of Med25551–680 and streptavidine to a Biacore CM5 sensor chip was made with an amine coupling kit according to the recommendations from GE Healthcare. Approximately 1500 resonance units (RU) of Med25551–680 were immobilized in 10 mM potassium phosphate buffer pH 7.0 with HBS-EP buffer as running buffer [10 mM HEPES pH 7.5, 150 mM NaCl, 3.4 mM EDTA and 0.005% non-ionic surfactant polyoxyethylenesorbitan (P2O) (GE Healthcare)]. Triplicates of decreasing concentrations of Dreb2a168–335, Dreb2a168–253 and Dreb2a254–335 (8 µM with 2-fold dilutions down to 0.031 µM) were injected at 20 µl/min for 3 min in running buffer (20 mM Tris pH 7.5, 150 mM NaCl and 1 mM DTT) and the chip was regenerated with 2 M MgCl2 followed by running buffer (flow 100 µl/min, 20 µl each) between each sample. Duplicates of Dreb2af.l. was injected in the same Tris-running buffer supplemented with 2 mM MgCl2. Between different concentrations (3 µM down to 0.047 µM with 2-fold dilution), Dreb2af.l. was washed away from the chip with a quick injection (100 µl/min) of 20 µl of regeneration buffer (50 mM NaOH, 1 M NaCl) followed by a quick inject of 20 µl running buffer. Two 29-nt long oligonucleotides, 5′-AAAGATATACTACCGACATGAGTTCCAAA-3′ and 5′-AAAGATATACTACTTTTATGAGTTCCAAA-3′, that contain the Dreb2a-binding sequence DRE or a mutated version of the DRE sequence from the rd29A promoter was used for experiments with Dreb2af.l. in conjugation with DNA (16). An amount of 2, 1 and 0.5 µM Dreb2af.l. were incubated with 4 µM of annealed oligonucleotides in running buffer containing 2 mM MgCl2 for 10 min prior to injection. Washes were made between each injection as described for Dreb2af.l. alone.For immobilization of DNA to the sensor chip, 50 µg/ml streptavidine (Sigma Aldrich) was injected at 5 µl/min until approximately 1000 RU was reached. After blockage with ethanolamine, biotin-labeled annealed oligonucleotide (10 µM) was injected at 20 µl/min in HBS-EP buffer until 350 RU was reached. A 71-nt long oligonucleotide from the promoter region of rd29A (described above), labeled at the 5′-end with biotin-TEG (5′ [Btn-Tg]-CAGTTTGAAAGAAAAGGGAAAAAAAGAAAAAATAAATAAAAGATATACTACCGACATGAGTTCCAAAAAGC-3′) was used (synthesized by Sigma Aldrich). Biotin–TEG-labeled oligonucleotides have an extra spacer with mixed polarity based on triethylene glycol that confers better binding to streptavidine than unmodified biotin. The complementary unlabeled oligonucleotide was annealed to the biotinylated oligonucleotide in 2-fold excess to achieve maximal dimerization of the biotin-labeled oligonucleotide. Dreb2af.l. was injected and the chip was washed as described in the previous paragraph.The nitrilotriacetic acid (NTA) sensor chip was subjected to 100 mM NiCl2 in HBS-buffer without EDTA at 5 µl/min for 5 min to obtain a metal chelating sensor surface. After activation, Df.l. (6.5 µM) in HBS buffer without EDTA was injected at 5 µl/min until 1100 RU was obtained. The chip was blocked with ethanolamine and unbound Ni2+ was subsequently removed with 250 mM EDTA at 5 µl/min for 7 min. HumMed25394–543 was injected from 3 µM down to 0.047 µM with 2-fold dilution in Tris running buffer supplemented with 5 mM MgCl2. Between the samples, the chip was washed at 100 µl/min with 35 µl of regeneration buffer (50 mM NaOH, 1 M NaCl), 35 µl of 250 mM EDTA and finally 35 µl running buffer.In all experiments, a reference surface that had been activated and subsequently blocked was used. As an additional reference, injection of buffer controls was made in all experiments.

Surface plasmon resonance analysis

Binding experiments were performed at 25°C on a Biacore3000 (GE healthcare) and analyzed with the Scrubber 2 software (BioLogic Software). Attachment of Med25551–680 and streptavidine to a Biacore CM5 sensor chip was made with an amine coupling kit according to the recommendations from GE Healthcare. Approximately 1500 resonance units (RU) of Med25551–680 were immobilized in 10 mM potassium phosphate buffer pH 7.0 with HBS-EP buffer as running buffer [10 mM HEPES pH 7.5, 150 mM NaCl, 3.4 mM EDTA and 0.005% non-ionic surfactant polyoxyethylenesorbitan (P2O) (GE Healthcare)]. Triplicates of decreasing concentrations of Dreb2a168–335, Dreb2a168–253 and Dreb2a254–335 (8 µM with 2-fold dilutions down to 0.031 µM) were injected at 20 µl/min for 3 min in running buffer (20 mM Tris pH 7.5, 150 mM NaCl and 1 mM DTT) and the chip was regenerated with 2 M MgCl2 followed by running buffer (flow 100 µl/min, 20 µl each) between each sample. Duplicates of Dreb2af.l. was injected in the same Tris-running buffer supplemented with 2 mM MgCl2. Between different concentrations (3 µM down to 0.047 µM with 2-fold dilution), Dreb2af.l. was washed away from the chip with a quick injection (100 µl/min) of 20 µl of regeneration buffer (50 mM NaOH, 1 M NaCl) followed by a quick inject of 20 µl running buffer. Two 29-nt long oligonucleotides, 5′-AAAGATATACTACCGACATGAGTTCCAAA-3′ and 5′-AAAGATATACTACTTTTATGAGTTCCAAA-3′, that contain the Dreb2a-binding sequence DRE or a mutated version of the DRE sequence from the rd29A promoter was used for experiments with Dreb2af.l. in conjugation with DNA (16). An amount of 2, 1 and 0.5 µM Dreb2af.l. were incubated with 4 µM of annealed oligonucleotides in running buffer containing 2 mM MgCl2 for 10 min prior to injection. Washes were made between each injection as described for Dreb2af.l. alone.

For immobilization of DNA to the sensor chip, 50 µg/ml streptavidine (Sigma Aldrich) was injected at 5 µl/min until approximately 1000 RU was reached. After blockage with ethanolamine, biotin-labeled annealed oligonucleotide (10 µM) was injected at 20 µl/min in HBS-EP buffer until 350 RU was reached. A 71-nt long oligonucleotide from the promoter region of rd29A (described above), labeled at the 5′-end with biotin-TEG (5′ [Btn-Tg]-CAGTTTGAAAGAAAAGGGAAAAAAAGAAAAAATAAATAAAAGATATACTACCGACATGAGTTCCAAAAAGC-3′) was used (synthesized by Sigma Aldrich). Biotin–TEG-labeled oligonucleotides have an extra spacer with mixed polarity based on triethylene glycol that confers better binding to streptavidine than unmodified biotin. The complementary unlabeled oligonucleotide was annealed to the biotinylated oligonucleotide in 2-fold excess to achieve maximal dimerization of the biotin-labeled oligonucleotide. Dreb2af.l. was injected and the chip was washed as described in the previous paragraph.

The nitrilotriacetic acid (NTA) sensor chip was subjected to 100 mM NiCl2 in HBS-buffer without EDTA at 5 µl/min for 5 min to obtain a metal chelating sensor surface. After activation, Df.l. (6.5 µM) in HBS buffer without EDTA was injected at 5 µl/min until 1100 RU was obtained. The chip was blocked with ethanolamine and unbound Ni2+ was subsequently removed with 250 mM EDTA at 5 µl/min for 7 min. HumMed25394–543 was injected from 3 µM down to 0.047 µM with 2-fold dilution in Tris running buffer supplemented with 5 mM MgCl2. Between the samples, the chip was washed at 100 µl/min with 35 µl of regeneration buffer (50 mM NaOH, 1 M NaCl), 35 µl of 250 mM EDTA and finally 35 µl running buffer.

In all experiments, a reference surface that had been activated and subsequently blocked was used. As an additional reference, injection of buffer controls was made in all experiments.

Far-UV circular dichroism spectroscopyFar-UV circular dichroism (CD) spectra were acquired on a Jasco J-720 spectropolarimeter using a quartz cuvette with a 1-mm path length. The spectra were recorded at a wavelength range of 200–300 nm with 0.5 nm data interval using a scan rate of 50 nm/min. Each spectrum was an average of five scans. Each spectrum was corrected by subtracting buffer and buffer–DNA signals, respectively, and all experiments were executed at least twice. The experiments were conducted with fixed concentrations of proteins (10 and 20 µM) in the absence and presence of specific and mutated oligonucleotides (10 µM). The molar ratio of DNA:protein was always 1:1. For protein mixture experiments with DNA, Dreb2af.1. was first pre-incubated with specific or mutated DNA for 1 h at room temperature before addition of Med25551–680. All samples were incubated in a 20 mM Tris, 150 mM NaCl, 2 mM β-mercaptoethanol, 5 mM MgCl2, pH 7.5 buffer at 20°C and for CD measurements as a function of time at 4°C. For the ethylene glycol (Sigma Aldrich) samples, the final concentration was fixed to 300 mg/ml.

Far-UV circular dichroism spectroscopy

Far-UV circular dichroism (CD) spectra were acquired on a Jasco J-720 spectropolarimeter using a quartz cuvette with a 1-mm path length. The spectra were recorded at a wavelength range of 200–300 nm with 0.5 nm data interval using a scan rate of 50 nm/min. Each spectrum was an average of five scans. Each spectrum was corrected by subtracting buffer and buffer–DNA signals, respectively, and all experiments were executed at least twice. The experiments were conducted with fixed concentrations of proteins (10 and 20 µM) in the absence and presence of specific and mutated oligonucleotides (10 µM). The molar ratio of DNA:protein was always 1:1. For protein mixture experiments with DNA, Dreb2af.1. was first pre-incubated with specific or mutated DNA for 1 h at room temperature before addition of Med25551–680. All samples were incubated in a 20 mM Tris, 150 mM NaCl, 2 mM β-mercaptoethanol, 5 mM MgCl2, pH 7.5 buffer at 20°C and for CD measurements as a function of time at 4°C. For the ethylene glycol (Sigma Aldrich) samples, the final concentration was fixed to 300 mg/ml.

Gel electrophoresis mobility shift assaysThe 29-nt oligonucleotides described above were annealed and the probes were labeled with 32P-ATP using T4 polynucleotide kinase (Fermentas). An amount of 1 µM Dreb2af.l. was incubated with the labeled probes (∼20 000 cpm or 20 fmol /reaction) in 25 mM HEPES pH 7.5, 50 mM KCl, 5 mM MgCl2, 10% glycerol, 1 µg/ml poly dIdC (GE Healthcare), 1 µg/µl BSA and 1 mM DTT. After incubation for 10 min at room temperature, 1 µM or 4 µM of Med25551–680 was added and the reaction was incubated for an additional 10 min at room temperature. For competition with unlabeled probe, 1.3 µM cold probe was added at the start of the reaction. The samples were separated on a 3.5% polyacrylamide Tris–Borate–EDTA gel and the dried gel was exposed to a phosphor storage screen and imaged with a Typhoon 9400 phosphorimager (GE Healthcare).

Gel electrophoresis mobility shift assays

The 29-nt oligonucleotides described above were annealed and the probes were labeled with 32P-ATP using T4 polynucleotide kinase (Fermentas). An amount of 1 µM Dreb2af.l. was incubated with the labeled probes (∼20 000 cpm or 20 fmol /reaction) in 25 mM HEPES pH 7.5, 50 mM KCl, 5 mM MgCl2, 10% glycerol, 1 µg/ml poly dIdC (GE Healthcare), 1 µg/µl BSA and 1 mM DTT. After incubation for 10 min at room temperature, 1 µM or 4 µM of Med25551–680 was added and the reaction was incubated for an additional 10 min at room temperature. For competition with unlabeled probe, 1.3 µM cold probe was added at the start of the reaction. The samples were separated on a 3.5% polyacrylamide Tris–Borate–EDTA gel and the dried gel was exposed to a phosphor storage screen and imaged with a Typhoon 9400 phosphorimager (GE Healthcare).

Size exclusion chromatographyA HiLoad 16/60 Superdex 200 (GE Healthcare) was calibrated with ribonuclease A (13.7 kDa), ovalbumin (44 kDa), conalbumin (75 kDa) and aldolase (158 kDa) from the gel-calibration kits supplied by the same manufacturer. The void volume (Vo) of the column was also determined to 40 ml with blue dextran 2000. An amount of 12 µM of Dreb2af.l., Med25551–680 and Med25551–680–GST were loaded in 500 µl. When Dreb2af.l. and Med25551–680 or Med25551–680–GST were mixed, samples were incubated for 15 min at room temperature followed by centrifugation for 15 min at 20 000 g before loaded onto the column. All samples were run in 20 mM Tris pH 7.5, 150 mM NaCl and 1 mM DTT at 4°C and the proteins were separated in 0.5 ml fractions. The presence of Dreb2af.l., Med25551–680 and Med25551–680–GST in the eluted fractions was detected using immunoblotting. The proteins were concentrated through precipitation with a 4-fold excess of acetone in −20°C over night. Precipitated proteins were collected through centrifugation at 20 000 g for 15 min. Air dried pellets were dissolved in SDS-loading buffer to a final concentration of 75 mM Tris pH 6.8, 1% SDS, 10% glycerol, 0.7 M β-mercaptoethanol and 0.00125% Bromophenol blue.To validate the interaction between Dreb2af.l. and Med25551–680–GST, samples were separated on a 10% SDS–PAGE and subjected to immunoblotting. Since the GB1-tag on Dreb2af.l. is a strong immunoglobulin-binding protein, secondary antibody can be used to directly identify the GB1-Dreb2af.l. protein. Thus, the 50–100 kDa part of the PVDF membrane was incubated with anti-mouse secondary antibody (1:5000) for 3 h in room temperature, washed and developed with ECL solution (all reagents from GE Healthcare). The 25–50 kDa part of the membrane was incubated with anti-GST antibody (1:2000, a generous gift from professor Sven Carlsson, Umea University, Sweden) at 4°C over night, washed and incubated with anti-rabbit antibody (1:5000, GE Healthcare, Sweden) for 1 h in room temperature. All membranes were blocked in TBST (20 mM Tris pH 7.5, 150 mM NaCl and 0.1% Tween-20) containing 5% milk for 1 h before addition of antibody. All antibodies were diluted in TBST. To verify that Med25551–680 eluted in complex with Dreb2af.l., samples were loaded on a 4–12% NuPAGE® Bis–Tris gel (Invitrogen) and run in MES SDS running buffer (50 mM Tris, 50 mM MES, 0.1% SDS and 1 mM EDTA). Proteins were visualized with Coomassie stain using PageBlue.

Size exclusion chromatography

A HiLoad 16/60 Superdex 200 (GE Healthcare) was calibrated with ribonuclease A (13.7 kDa), ovalbumin (44 kDa), conalbumin (75 kDa) and aldolase (158 kDa) from the gel-calibration kits supplied by the same manufacturer. The void volume (Vo) of the column was also determined to 40 ml with blue dextran 2000. An amount of 12 µM of Dreb2af.l., Med25551–680 and Med25551–680–GST were loaded in 500 µl. When Dreb2af.l. and Med25551–680 or Med25551–680–GST were mixed, samples were incubated for 15 min at room temperature followed by centrifugation for 15 min at 20 000 g before loaded onto the column. All samples were run in 20 mM Tris pH 7.5, 150 mM NaCl and 1 mM DTT at 4°C and the proteins were separated in 0.5 ml fractions. The presence of Dreb2af.l., Med25551–680 and Med25551–680–GST in the eluted fractions was detected using immunoblotting. The proteins were concentrated through precipitation with a 4-fold excess of acetone in −20°C over night. Precipitated proteins were collected through centrifugation at 20 000 g for 15 min. Air dried pellets were dissolved in SDS-loading buffer to a final concentration of 75 mM Tris pH 6.8, 1% SDS, 10% glycerol, 0.7 M β-mercaptoethanol and 0.00125% Bromophenol blue.

To validate the interaction between Dreb2af.l. and Med25551–680–GST, samples were separated on a 10% SDS–PAGE and subjected to immunoblotting. Since the GB1-tag on Dreb2af.l. is a strong immunoglobulin-binding protein, secondary antibody can be used to directly identify the GB1-Dreb2af.l. protein. Thus, the 50–100 kDa part of the PVDF membrane was incubated with anti-mouse secondary antibody (1:5000) for 3 h in room temperature, washed and developed with ECL solution (all reagents from GE Healthcare). The 25–50 kDa part of the membrane was incubated with anti-GST antibody (1:2000, a generous gift from professor Sven Carlsson, Umea University, Sweden) at 4°C over night, washed and incubated with anti-rabbit antibody (1:5000, GE Healthcare, Sweden) for 1 h in room temperature. All membranes were blocked in TBST (20 mM Tris pH 7.5, 150 mM NaCl and 0.1% Tween-20) containing 5% milk for 1 h before addition of antibody. All antibodies were diluted in TBST. To verify that Med25551–680 eluted in complex with Dreb2af.l., samples were loaded on a 4–12% NuPAGE® Bis–Tris gel (Invitrogen) and run in MES SDS running buffer (50 mM Tris, 50 mM MES, 0.1% SDS and 1 mM EDTA). Proteins were visualized with Coomassie stain using PageBlue.

Formaldehyde fixationAn amount of 1 µM Dreb2af.l. was mixed with Med25551–680, Med25551–680–GST or hMed25394–543 (1 µM or 4 µM) for 20 min in room temperature in the same buffer as the one used for gel electrophoresis mobility shift assays (GEMSA) (25 mM HEPES pH 7.5, 50 mM KCl, 5 mM MgCl2, 10% glycerol, 1 µg/ml poly dIdC 1 µg/µl BSA and 1 mM DTT). Formed complexes were cross-linked by the addition of formaldehyde to a final concentration of 0.5% for 30 min in room temperature. The cross-linked proteins were mixed with SDS-loading buffer, loaded onto a 3–8% NuPAGE® Novex Tris–Acetate gel (Invitrogen) and run in Tris–Acetate SDS-running buffer (50 mM Tris–base, 50 mM Tricine, 0.1% SDS pH 8.24). Dreb2af.l. boiled for 5 min was included as a denatured control. The migration of Dreb2af.l. was visualized by immunoblotting as described above.

Formaldehyde fixation

An amount of 1 µM Dreb2af.l. was mixed with Med25551–680, Med25551–680–GST or hMed25394–543 (1 µM or 4 µM) for 20 min in room temperature in the same buffer as the one used for gel electrophoresis mobility shift assays (GEMSA) (25 mM HEPES pH 7.5, 50 mM KCl, 5 mM MgCl2, 10% glycerol, 1 µg/ml poly dIdC 1 µg/µl BSA and 1 mM DTT). Formed complexes were cross-linked by the addition of formaldehyde to a final concentration of 0.5% for 30 min in room temperature. The cross-linked proteins were mixed with SDS-loading buffer, loaded onto a 3–8% NuPAGE® Novex Tris–Acetate gel (Invitrogen) and run in Tris–Acetate SDS-running buffer (50 mM Tris–base, 50 mM Tricine, 0.1% SDS pH 8.24). Dreb2af.l. boiled for 5 min was included as a denatured control. The migration of Dreb2af.l. was visualized by immunoblotting as described above.

Pull-down assaysAn amount of 6 µM Dreb2af.l. was bond to 15 µl IgG sepharose 6 fast flow (GE Healthcare) for 30 min in binding buffer (20 mM Tris pH 7.5, 150 mM NaCl, 2 mM β-mercaptoethanol) at room temperature. hMed25394–543 was added to a final concentration of 6 µM and the samples were incubated for 30 min. Unbound protein were collected in flow through and the beads were washed two times in 1 ml of binding buffer containing 0.05% Tween-20. Bound proteins were extracted by addition of 1× SDS-loading buffer and boiled for 5 min. The samples were loaded on a 4–12% NuPAGE® Bis–Tris gel (Invitrogen) and run in MOPS SDS-running buffer (50 mM Tris pH 7.7, 50 mM MES, 0.1% SDS and 1 mM EDTA). The proteins were detected as described for size exclusion chromatography except that hMed25394–543 was visualized by incubation with anti-poly His antibody (1:2000, H1029, Sigma Aldrich).

Pull-down assays

An amount of 6 µM Dreb2af.l. was bond to 15 µl IgG sepharose 6 fast flow (GE Healthcare) for 30 min in binding buffer (20 mM Tris pH 7.5, 150 mM NaCl, 2 mM β-mercaptoethanol) at room temperature. hMed25394–543 was added to a final concentration of 6 µM and the samples were incubated for 30 min. Unbound protein were collected in flow through and the beads were washed two times in 1 ml of binding buffer containing 0.05% Tween-20. Bound proteins were extracted by addition of 1× SDS-loading buffer and boiled for 5 min. The samples were loaded on a 4–12% NuPAGE® Bis–Tris gel (Invitrogen) and run in MOPS SDS-running buffer (50 mM Tris pH 7.7, 50 mM MES, 0.1% SDS and 1 mM EDTA). The proteins were detected as described for size exclusion chromatography except that hMed25394–543 was visualized by incubation with anti-poly His antibody (1:2000, H1029, Sigma Aldrich).

Prediction of secondary structureThe prediction of secondary structures was made with the Jpred 3 server, available at www.compbio.dundee.ac.uk/jpred. It has been reported to predict secondary structure of proteins at an accuracy of 81.5% (17).

Prediction of secondary structure

The prediction of secondary structures was made with the Jpred 3 server, available at www.compbio.dundee.ac.uk/jpred. It has been reported to predict secondary structure of proteins at an accuracy of 81.5% (17).

RESULTSThe structures of the Med25–ACID domain and the C-terminal domain of Dreb2a do not change upon protein–protein interactionWe previously mapped the Med25–ACID-binding domain (BD; amino acids 551–680) of Dreb2a to the region in between the activation domain (AD) and the negative regulatory domain (NRD), (Figure 1A) (16). To investigate the binding properties of Dreb2a to Med25551–680, three different his-tagged domains of Dreb2a (Dreb2a168–335, Dreb2a168–253 and Dreb2a254–335) and GST-tagged Med25551–680 were purified to near homogeneity as described in ‘Materials and Methods’ section (Figure 1A). The GST-tag of Med25551–680 was removed by protease cleavage. A typical purification result for all proteins is shown in Supplementary Figure S1.
Figure 1.The C-terminal part of Dreb2a and Med25551–680 interacts in vitro without affecting the protein conformations. (A) Schematic picture of the different domains of Dreb2a. Three truncated versions; the BD and AD (Dreb2a168–335), the BD (Dreb2a168–253) or AD (Dreb2a254–335) as well as full-length (Dreb2af.l.) Dreb2a were used in the studies. (B) Increasing concentrations of the different Dreb2a domains were injected over a sensor chip to which Med25551–680 was pre-bound. The mean of three independent plateau values are plotted. (C) Sensogram for the increasing concentrations of Dreb2a168–335, showing its binding kinetics to Med25551–680. (D) Far-UV CD spectra of Med25551–680 (blue), Dreb2a168–335 (green) and the mixture of both proteins (purple) mixed in a 1:1 molar ratio (20 µM each). Also shown is the CD sum of the individual protein signals (black dotted).To determine which part of Dreb2a that is needed for binding to Med25551–680, we performed binding studies using surface plasmon resonance (SPR). We found that only the extended C-terminal domain, Dreb2a168–335 which includes both the BD and the AD, interacted with Med25551–680 with significant affinity. Affinity was determined from the plateau values of binding curves for injections of various concentrations of the different domains (Figure 1B). From the analysis, the dissociation constant (KD) was estimated to 1 µM (pH 7.5, 20°C) for the Dreb2a168–335–Med25551–680 interaction. The Dreb2a168–335–Med25551–680 interaction displayed fast kinetics, thus indicating a dynamic complex (Figure 1C). This is very similar to the KD of 1.6 µM that was determined by isothermal titration calorimetric (ITC) assays for the human Med25 ACID domain and different domains of the VP16 transcriptional activator (18).Using Far-UV CD, we found that the spectrum of the Dreb2a168–335 domain indicated a lack of secondary structure for this domain in solution. The negative maximum ∼200 nm implies an unstructured random coil conformation (Figure 1D). In contrast, the negative peak at 230 nm in the CD spectrum of Med25551–680 represents contribution from aromatic residues that are in an ordered conformation (19). Notably, the interaction between Dreb2a168–335 and Med25551–680 did not result in any change in the total secondary structure content of the two proteins. The CD signal for the proteins after mixing at concentrations well above the KD overlapped with the calculated sum of the individual CD signals for each protein (Figure 1D).Binding of full-length Dreb2a to Med25551–680 results in a small gain of structureTo obtain sufficient expression levels of Dreb2af.l. in E. coli, the protein was fused to a GB1-tag along with the His-tag. The B1 IgG-BD of protein G (GB1) from Streptococcus is a 56 amino acids long single domain protein with one α-helix and a four-stranded β-sheet which is used as a fusion partner to increase solubility and expression of instable proteins (20,21). Due to its small size (8757.6 Da) it is regarded as one of the most optimal tags for NMR studies of instable proteins (22). Since removal of the GB1-tag from our recombinantly expressed Dreb2af.l. resulted in precipitation after purification, the GB1-Dreb2af.l. fusion protein of 47 kDa was purified to near homogeneity (Supplementary Figure S1B).The interaction between Med25551–680 and Dreb2af.l. was studied using SPR, where Med25551–680 was immobilized on a sensor chip using amine coupling. The sensograms in Figure 2A show that the interaction between Med25551–680 and Dreb2af.l. displayed complex kinetics: the dissociation was bi-phasic with a fast and a slow dissociation rate. Due to the complexity of the kinetics, an accurate KD is difficult to determine. However, at low concentrations Dreb2af.1 the interaction displayed less complexity and an on rate of 22 000 M−1 S−1 could be approximated. The slow part of the dissociation rate could be approximated to 0.0023 S−1. Combining the on rate and the dissociation rate gives an approximated KD of 0.1 µM.
Figure 2.The interaction between full-length Dreb2a and Med25551–680 shows complex kinetics and involves changes in secondary structure. (A) SRP sensograms showing the binding of increasing concentrations of Dreb2af.l. to Med25551–680. An amount of 3 µM Dreb2af.l. was diluted 2-fold to 0.047 µM and duplicate samples were injected over immobilized Med25551–680. (B) CD spectra of full-length Dreb2a (dark grey) and Dreb2a168–335 domain (light grey dashed). The signals are shown as mean residue ellipticity, i.e. CD per amino acid. (C) CD spectra of Med25551–680 (blue), Dreb2af.l. (green) and the mixture of both proteins (purple) in a 1:1 molar ratio (10 µM). The CD spectrum of the sum of the individual proteins CD signals is also shown (black dashed). (D) Far-UV CD spectra of Med25551–680 (light grey) and Dreb2af.l. (dark grey) in the absence (solid) and presence (dashed) of 300 mg/ml ethylene glycol (EG).This dissociation rate for Dreb2af.1., is significantly slower than the dissociation rate for Dreb2a168–335, which thus indicate that Dreb2af.l., forms a more stable complex with Med25551–680 compared to Dreb2a168–335. Again, this KD for the full-length Dreb2a transcription activation domain (TAD) is comparable to the KD reported for the full-length VP16 TAD interaction with the human Med25 ACID domain (18).Furthermore, Dreb2af.l. contains more secondary structure per amino acid compared to Dreb2a168–335, as probed by far-UV CD (Figure 2B). This is in agreement with a prediction of secondary structure content, which suggests that the N-terminal part of Dreb2a contains more α-helix and β-sheet structures (Supplementary Figure S2). Most of the predicted ordered structures are found in the DNA-BD (amino acids 78–135) that contains three β-strands and one α-helix. Moreover, mixing of Med25551–680 and Dreb2af.l. at concentrations favoring complex formation based on SPR resulted in an increased negative far-UV CD signal (∼10% at 220 nm) as compared to the sum of the CD signals for the individual proteins (Figure 2C). Addition of the stabilizing osmolyte ethylene glycol to Dreb2af.1. resulted in increased secondary structure based on CD measurements (∼11% at 220 nm). Med25 was, however, not affected by the addition of ethylene glycol (EG) (Figure 2D). This implies that Dreb2af.1. is prone to acquire a more ordered conformation under certain conditions.Both Dreb2af.1. and Med25551–680 is present as monomers and the stoichiometry of binding to each other is 1:1The size of Med25551–680, Med25551–680 –GST, Dreb2af.l. and the Med25551–680 /GST– Dreb2af.l. proteins were investigated with size exclusion chromatography. Med25551–680 –GST (42 kDa) eluted at 71 ml which corresponds to an 80-kDa globular protein when compared to a standard curve generated from separation of various standard proteins on the same gel-filtration column (Figure 3A and E). This is in agreement with other studies that indicate dimerization of GST monomers (23–25). However, GST-fusion proteins are sometimes detected as monomers (26,27), a feature that can be concentration dependent (28). Dreb2af.l. that has a molecular weight of 47 kDa eluted close to 71 ml, suggesting that it either is present as an extended, non-globular monomer or as a dimer (Figure 3A and C). Complex formation of Med25551–680 –GST and Dreb2af.l. generated a shift of the peak corresponding to the elution profile of a >120 kDa protein (Figure 3A and E). This shift indicates that Med25551–680 –GST and Dreb2af.l. interact and the elution profile is most compatible with a complex composed of one Med25551–680–GST dimer (84 kDa) and one Dreb2af.l. monomer (47 kDa) when mixed in a 1:1 ratio. However, the resolution in this region is low and it is thus difficult to draw any further conclusions other than that Med25551–680–GST and Dreb2af.l form a complex. The presence of Med25551–680 –GST and Dreb2af.l. in the fractions was further confirmed with immunoblotting (Figure 3B). Moreover, when Med25551–680 without the GST-tag was fractionated it eluted at 90.4 ml, which corresponds to the elution profile of a 16.4 kDa protein (Figure 3C and E). Med25551–680 has a theoretical molecular weight of 14.7 kDa, thus indicating that Med25551–680 is present as a monomer when the GST-tag is removed. Complex formation of Dreb2af.l. with untagged Med25551–680 resulted in a small shift in migration, representing an increased molecular weight of 10 kDa (Figure 3C and E). This suggests that the binding of Med25551–680 to Dreb2af.l. is 1:1 and that the Med25551–680 interaction induces a minor change in the structure of Dreb2af.l. which affects its migration on the column. The elution of Med25551–680 at 70 ml when bound to Dreb2af.l. was confirmed with immunoblotting (Figure 3D).
Figure 3.Full-length Dreb2a and Med25551–680 bind each other mainly with a 1:1 ratio. (A) Size exclusion chromatogram for Dreb2af.l. (dark grey) and Med25551–680–GST (light grey) alone and in complex (black). The molecular weights of the standards used are marked with stars on the x-axis. (B) The presence of Dreb2af.l. (D) and Med25551–680–GST (Mg) in the eluted fractions (ml) was confirmed with immunoblot. Input (I) controls are shown in lane 1 and 2. (C) Size exclusion chromatogram for Dreb2af.l. (dark grey) and Med25551–680 (light grey) alone and in complex (black). (D) Coomassie-stained SDS–PAGE on eluted fractions (ml) show in which fractions Dreb2af.l. (D) and Med25551–680 (M) are found. (E) List of the observed molecular weights in A and C (calculated from the standard) compared to the theoretical molecular weights of the single proteins and possible complex formations of Dreb2af.l. and Med25. Ratios of 1:1, 1:2 and 2:1 of Dreb2af.l. versus Med25 are listed. (F) Formaldehyde cross-linked (0.5%) Dreb2af.l., (D), Med25551–680–GST (Mg) and Med25551–680 (M) alone or in complex was separated on a 4–12% gradient gel and the migration Dreb2af.l. was visualized by immunoblot. Complex formation of Dreb2af.l., and Med25551–680 was induced at 1:1 (1 µM each) and 1:4 (1 µM Dreb2af.l. and 4 µM Med25551–680) ratios. Denatured Dreb2af.l., in the first lane was included as a control.The size of the Med25551–680 and Dreb2af.l. complex was further investigated by formaldehyde cross-linking and SDS–PAGE. As illustrated in Figure 3F, cross-linked Dreb2af.l. did not differ in size compared to non-crosslinked and denatured Dreb2af.l., showing that it exists as a monomer. Cross-linking of the Dreb2af.l.–Med25551–680 complex reduced the migration of Dreb2af.l. with ∼10 kDa, when the proteins were mixed in a 1:1 ratio. A 4-fold excess of Med25551–680 mixed with Dreb2af.l. revealed that Dreb2af.l. can bind two monomers of Med25551–680
in vitro when Med25551–680 is present in excess. Cross-linking of Dreb2af.l. and Med25551–680–GST resulted in a complex at 130 kDa, showing that Dreb2af.l. binds Med25551–680–GST dimers. However, most of this mixture resulted in an elution profile corresponding to a size of 90 kDa, which illustrate that the Dreb2af.l. and Med25551–680–GST complex mainly consist of one Dreb2af.l. monomer and one Med25551–680–GST monomer. Addition of a 4-fold excess of Med25551–680–GST did not result in an increased amount of the larger complex of 130 kDa. In summary this illustrates that Dreb2af.l. and Med25551–680 primarily binds each other in a 1:1 ratio.The binding of Dreb2af.l. to Med25551–680 is reduced when Dreb2af.l. is pre-bound to DNAThe conserved sequence TACCGACAT, named the dehydration-responsive element (DRE), is an essential cis-acting element for induction of RD29A expression in the ABA-independent response to dehydration, high salinity and cold (29). Dreb2af.l. has been shown to bind sequence-specifically to the TACCGACAT sequence and not to the mutated TACTTTTAT sequence (30). We confirmed this by GEMSA and addition of Med25551–680 resulted in a super-shift of the Dreb2af.l.–DNA complex (Figure 4A). This shows that DNA-bound Dreb2af.l. binds Med25551–680. Since the super-shift was very small, most likely due to the small size (14.7 kDa) and high pI (9.64) of Med25551–680, Dreb2af.l. alone was loaded both in lane 4 and 7 as controls. The binding property of Dreb2af.l. to DNA was monitored with SPR where biotinylated oligonucleotides were attached to a streptavidine-coated chip. The binding curves showed that the Dreb2af.l.–DNA binding was dynamic, with fast on and off rates (Figure 4B). When the plateau values of the curves in Figure 4B were plotted, the KD was determined to 0.60 µM (Figure 4C). This KD value is slightly higher than what was detected for the Arabidopsis cold-responsive transcription factor RAV1 and the rice bZIP transcriptional activator RF2b (0.15 and 0.457 µM, respectively) (31,32). As noticed by Yamasaki et al., the DNA interaction is abolished by high salt concentrations and thus, the use of different buffers can explain the difference. We found that a lower salt concentration resulted in stronger binding of Dreb2af.l. to its DNA (data not shown).
Figure 4.Dreb2a binds its specific DNA with fast kinetics. (A) The binding of Dreb2af.l. to 29-nt oligonucleotides with the TACCGACAT sequence (Wt-probe) or the mutated TACTTTTAT sequence (mut-probe) was monitored with GEMSA. Lane 1:Labeled Wt-probe, Lane 2: Labeled Wt-probe, Dreb2af.l. (1 μM), a 1000-fold molar excess of unlabeled Wt-probe, Lane 3- Dreb2af.l. induced shift using labeled mut-probe, Lanes 4 and 7: Labeled Wt-probe and Dreb2af.l. (1 μM), Lane 5: Labeled Wt-probe, Dreb2af.l. (1 μM) and Med25551–680 (4 μM), Lane 6: Labeled Wt-probe and Med25551–680 (4 μM) (B) A biotin label oligonucleotide of 71 nt that contain the binding site TACCGACAT was immobilized on a streptavidine coated sensor chip and graded concentrations of Dreb2af.l. was injected. (C) A streptavidine coated surface was used as reference, the plateau values were plotted and the dissociation constant was determined.A gain in secondary structure was detected when Dreb2af.l. was mixed with its specific DNA but not when mixed with the mutated oligonucleotide (Figure 5A). The increase in the negative far-UV CD signal of Dreb2af.l. in presence of DNA was ∼17% at 205 and 220 nm of the total signal of the protein in the absence of DNA. This indicates that structural changes that increase the overall secondary structure content occure in Dreb2af.l. upon DNA binding. We assume that there are no structural changes in the DNA that affects the CD signal upon protein binding. In support of this, there are no changes in the DNA signal >260 nm (where protein has no contribution) with and without protein (not shown). Binding to DNA increased the stability of Dreb2af.l. as a function of time, based on CD measurements at 4°C (Figure 5B). The negative CD amplitude of Dreb2af.l. in presence of its specific DNA decreased by 14% at 205 and 220 nm after 8 days incubation, while the protein in presence of mutated DNA had a CD signal decrease of 33% at 205 and 220 nm at the same time point. Decreased CD intensity may correspond to protein unfolding and/or precipitation. In any case, the results show that Dreb2a adopts a more stable conformation when it is bound to its consensus DNA-binding site. Although comparing data after 8 days of incubation does not appear biologically relevant, the low temperature used here (4°C) slows down the reactions and thus these processes would be faster at physiological conditions. Nonetheless, this experiment is only used to emphasize a relative stability difference for Dreb2af.l. when in the presence of the two different DNAs.
Figure 5.Interaction with DNA induces conformational changes and increased integrity of Dreb2a. (A) CD spectra of Dreb2af.l. in the absence (dark grey) and presence of DNA (black) or mutated DNA (light grey) in a 1:1 molar ratio (10 µM). The CD signal of DNA is subtracted in both cases. (B) Comparison of CD spectra (DNA signal subtracted) of Dreb2af.l. in the presence of specific DNA (upper) and mutated DNA (lower) at Day 1 (solid lines) and Day 8 (dashed lines) upon continuous incubation of the mixtures at 4°C.The complex formation of Med25551–680 and Dreb2af.l. in SPR, gel filtration and CD experiments (Figures 2 and 3) clearly shows that the interaction of Med25551–680 and Dreb2af.l. is not dependent on the binding of Dreb2af.l. to DNA. To investigate what impact DNA has on the Med25551–680–Dreb2af.l. binding, additional SPR and CD measurements were performed. A reduced binding of Dreb2af.l. in conjugation with DNA was observed when Med25551–680 was attached to a sensor chip with amine coupling and Dreb2af.l. was injected alone or pre-bound to DNA (Figure 6A–C). This suggests that DNA-bound Dreb2af.l. has lower affinity to Med25551–680 compared to free Dreb2af.l.. Binding of Med25551–680 to Dreb2af.l. pre-bound to DNA did not result in any additional structural changes since the far-UV CD spectrum of the mixture and the sum of the individual protein signals of Med25551–680 and Dreb2af.l. bound to DNA overlapped (Figure 6D).
Figure 6.Dreb2a in complex with DNA show reduced binding to Med25551–680. Med25551–680 was attached to a sensor chip with amine coupling. (A) Sensogram showing dose-dependent response for Dreb2af.l.. (B) Sensogram showing dose-dependent response for Dreb2af.l. pre-bound to its specific DNA and (C) Dreb2af.l. pre-bound to the mutated DNA. (D) Far-UV CD spectra of a mixture of Med25 and pre-incubated Dreb2a with specific DNA (black) in a 1:1:1 molar ratio (10 µM), DNA signal subtracted. The sum of the individual CD signals of Dreb2a in presence of DNA and that of Med25 are also shown (dash).The ACID domain of human Med25394–543 show structural similarities to Med25551–680 from A. thaliana and forms a complex with Dreb2af.l.We have previously reported that there is only a week homology between the amino acid sequence of Med25 from A. thaliana and human (15). Interestingly, prediction of the secondary structure of the ACID domains of human Med25 (humMed25394–543) and A. thaliana (Med25551–680) revealed that some similarities exists in the distribution of α-helixes and β-sheets (Figure 7A). The NMR-structure of the human Med25 ACID domain was recently solved (18,33,34), and it was shown to contain a closed β-barrel with seven strands and three α-helixes. This is in line with the prediction of human Med25394–543 in Figure 7A, with the expectation that the third β-sheet (B3) is predicted to be α-helical in the beginning (amino acids 447–450). The human Med25394–543 is predicted to contain one α-helix more than Med25551–680 and CD measurement on the two different Med25 ACIDs did indeed revealed that human Med25394–543 contain more secondary structure (Figure 7B), Pull-down experiments with Dreb2af.l. and human Med25394–543 showed that when the proteins were mixed, a small fraction eluted together (bound) from the beads to which Dreb2af.l. had been attached (Figure 7C). Most of the humMed25394–543 did however not bind Dreb2af.l. but was found in flow through. Notably, since Dreb2af.l. is a plant-specific transcription factor with no apparent homolog in humans, the interaction was expected to be weak. To confirm that humMed25394–543 interacts with Dreb2af.l., cross-linking were made prior to SDS–PAGE and immunoblotting (Figure 7D). In this assay, Dreb2af.l. formed a complex together with humMed25394–543 in a similar dose-dependent manner as that detected for Med25551–680 when mixed in 1:1 and 1:4 ratios (Figures 7D and 3F). Importantly, addition of GST did not result in complex formation with Dreb2af.l.. The interaction between humMed25394–543 and Df.l. was studied using SPR, where His-tagged Df.l. was immobilized on a NTA sensor chip. The sensograms show that the interaction between humMed25394–543 and Df.l. displayed similar complex kinetics as the A. thaliana Med25551–680 and Df.l interaction (Figure 7E) and subsequently a similar dissociation constant. Due to the complexity of the kinetics, an accurate dissociation constant could not be determined but is in the same range as of Med25ACID. Detection of a humMed25394–543–Dreb2af.l. complex with three independent assays suggests that structural features that are important for interaction with activators must be conserved between Med25ACID from A. thaliana and humans.
Figure 7.Structural similarity between the ACID-domains of Med25 from human and A. thaliana mediates similar binding propensity to Dreb2af.l.. (A) Prediction of secondary structure in the ACID-domain of Med25 from human (humMed25394–543) and A. thaliana (aMed25551–680) was made utilizing the jpred 3 server. The humMed25 contains somewhat more α-helixes (H) in dark grey and β-strands (E) in light grey than aMed25. (B) CD spectra of humMed25394–543 and aMed25551–680 confirmed that humMed25394–543 contains more ordered structures. (C) Dreb2af.l. (D) was bound to IgG sepharose through its GB-1 tag. Addition of humMed25394–543 (hM) resulted in a Dreb2af.l.–humMed25394–543 complex that was visualized by SDS–PAGE and immunoblotting. (D) Complex formation of Dreb2af.l. and humMed25394–543 (hM) was investigated with formaldehyde cross-linking (0.5%) and immunoblot. Dreb2af.l., and humMed25394–543 was mixed at 1:1 (1 µM each) and 1:4 (1 µM Dreb2af.l. and 4 µM Med25) ratios. An amount of 4 µM humMed25394–543 alone, and Dreb2af.l. incubation with GST (1:4) severed as negative controls. Complex formation between 1 µM aMed25551–680 (M) and 1 µM Dreb2af.l. was included as a positive control. (E) SRP sensograms showing the binding of increasing concentrations of humMed25394–543 to Df.l.. An amount of 3 µM of humMed25394–543 was diluted 1:1 down to 0.047 µM and samples were passed over a NTA sensor chip sensorchip where Df.l. was immobilized via its His-tag.

The structures of the Med25–ACID domain and the C-terminal domain of Dreb2a do not change upon protein–protein interactionWe previously mapped the Med25–ACID-binding domain (BD; amino acids 551–680) of Dreb2a to the region in between the activation domain (AD) and the negative regulatory domain (NRD), (Figure 1A) (16). To investigate the binding properties of Dreb2a to Med25551–680, three different his-tagged domains of Dreb2a (Dreb2a168–335, Dreb2a168–253 and Dreb2a254–335) and GST-tagged Med25551–680 were purified to near homogeneity as described in ‘Materials and Methods’ section (Figure 1A). The GST-tag of Med25551–680 was removed by protease cleavage. A typical purification result for all proteins is shown in Supplementary Figure S1.
Figure 1.The C-terminal part of Dreb2a and Med25551–680 interacts in vitro without affecting the protein conformations. (A) Schematic picture of the different domains of Dreb2a. Three truncated versions; the BD and AD (Dreb2a168–335), the BD (Dreb2a168–253) or AD (Dreb2a254–335) as well as full-length (Dreb2af.l.) Dreb2a were used in the studies. (B) Increasing concentrations of the different Dreb2a domains were injected over a sensor chip to which Med25551–680 was pre-bound. The mean of three independent plateau values are plotted. (C) Sensogram for the increasing concentrations of Dreb2a168–335, showing its binding kinetics to Med25551–680. (D) Far-UV CD spectra of Med25551–680 (blue), Dreb2a168–335 (green) and the mixture of both proteins (purple) mixed in a 1:1 molar ratio (20 µM each). Also shown is the CD sum of the individual protein signals (black dotted).To determine which part of Dreb2a that is needed for binding to Med25551–680, we performed binding studies using surface plasmon resonance (SPR). We found that only the extended C-terminal domain, Dreb2a168–335 which includes both the BD and the AD, interacted with Med25551–680 with significant affinity. Affinity was determined from the plateau values of binding curves for injections of various concentrations of the different domains (Figure 1B). From the analysis, the dissociation constant (KD) was estimated to 1 µM (pH 7.5, 20°C) for the Dreb2a168–335–Med25551–680 interaction. The Dreb2a168–335–Med25551–680 interaction displayed fast kinetics, thus indicating a dynamic complex (Figure 1C). This is very similar to the KD of 1.6 µM that was determined by isothermal titration calorimetric (ITC) assays for the human Med25 ACID domain and different domains of the VP16 transcriptional activator (18).Using Far-UV CD, we found that the spectrum of the Dreb2a168–335 domain indicated a lack of secondary structure for this domain in solution. The negative maximum ∼200 nm implies an unstructured random coil conformation (Figure 1D). In contrast, the negative peak at 230 nm in the CD spectrum of Med25551–680 represents contribution from aromatic residues that are in an ordered conformation (19). Notably, the interaction between Dreb2a168–335 and Med25551–680 did not result in any change in the total secondary structure content of the two proteins. The CD signal for the proteins after mixing at concentrations well above the KD overlapped with the calculated sum of the individual CD signals for each protein (Figure 1D).

The structures of the Med25–ACID domain and the C-terminal domain of Dreb2a do not change upon protein–protein interaction

We previously mapped the Med25–ACID-binding domain (BD; amino acids 551–680) of Dreb2a to the region in between the activation domain (AD) and the negative regulatory domain (NRD), (Figure 1A) (16). To investigate the binding properties of Dreb2a to Med25551–680, three different his-tagged domains of Dreb2a (Dreb2a168–335, Dreb2a168–253 and Dreb2a254–335) and GST-tagged Med25551–680 were purified to near homogeneity as described in ‘Materials and Methods’ section (Figure 1A). The GST-tag of Med25551–680 was removed by protease cleavage. A typical purification result for all proteins is shown in Supplementary Figure S1.
Figure 1.The C-terminal part of Dreb2a and Med25551–680 interacts in vitro without affecting the protein conformations. (A) Schematic picture of the different domains of Dreb2a. Three truncated versions; the BD and AD (Dreb2a168–335), the BD (Dreb2a168–253) or AD (Dreb2a254–335) as well as full-length (Dreb2af.l.) Dreb2a were used in the studies. (B) Increasing concentrations of the different Dreb2a domains were injected over a sensor chip to which Med25551–680 was pre-bound. The mean of three independent plateau values are plotted. (C) Sensogram for the increasing concentrations of Dreb2a168–335, showing its binding kinetics to Med25551–680. (D) Far-UV CD spectra of Med25551–680 (blue), Dreb2a168–335 (green) and the mixture of both proteins (purple) mixed in a 1:1 molar ratio (20 µM each). Also shown is the CD sum of the individual protein signals (black dotted).

The C-terminal part of Dreb2a and Med25551–680 interacts in vitro without affecting the protein conformations. (A) Schematic picture of the different domains of Dreb2a. Three truncated versions; the BD and AD (Dreb2a168–335), the BD (Dreb2a168–253) or AD (Dreb2a254–335) as well as full-length (Dreb2af.l.) Dreb2a were used in the studies. (B) Increasing concentrations of the different Dreb2a domains were injected over a sensor chip to which Med25551–680 was pre-bound. The mean of three independent plateau values are plotted. (C) Sensogram for the increasing concentrations of Dreb2a168–335, showing its binding kinetics to Med25551–680. (D) Far-UV CD spectra of Med25551–680 (blue), Dreb2a168–335 (green) and the mixture of both proteins (purple) mixed in a 1:1 molar ratio (20 µM each). Also shown is the CD sum of the individual protein signals (black dotted).

To determine which part of Dreb2a that is needed for binding to Med25551–680, we performed binding studies using surface plasmon resonance (SPR). We found that only the extended C-terminal domain, Dreb2a168–335 which includes both the BD and the AD, interacted with Med25551–680 with significant affinity. Affinity was determined from the plateau values of binding curves for injections of various concentrations of the different domains (Figure 1B). From the analysis, the dissociation constant (KD) was estimated to 1 µM (pH 7.5, 20°C) for the Dreb2a168–335–Med25551–680 interaction. The Dreb2a168–335–Med25551–680 interaction displayed fast kinetics, thus indicating a dynamic complex (Figure 1C). This is very similar to the KD of 1.6 µM that was determined by isothermal titration calorimetric (ITC) assays for the human Med25 ACID domain and different domains of the VP16 transcriptional activator (18).

Using Far-UV CD, we found that the spectrum of the Dreb2a168–335 domain indicated a lack of secondary structure for this domain in solution. The negative maximum ∼200 nm implies an unstructured random coil conformation (Figure 1D). In contrast, the negative peak at 230 nm in the CD spectrum of Med25551–680 represents contribution from aromatic residues that are in an ordered conformation (19). Notably, the interaction between Dreb2a168–335 and Med25551–680 did not result in any change in the total secondary structure content of the two proteins. The CD signal for the proteins after mixing at concentrations well above the KD overlapped with the calculated sum of the individual CD signals for each protein (Figure 1D).

Binding of full-length Dreb2a to Med25551–680 results in a small gain of structureTo obtain sufficient expression levels of Dreb2af.l. in E. coli, the protein was fused to a GB1-tag along with the His-tag. The B1 IgG-BD of protein G (GB1) from Streptococcus is a 56 amino acids long single domain protein with one α-helix and a four-stranded β-sheet which is used as a fusion partner to increase solubility and expression of instable proteins (20,21). Due to its small size (8757.6 Da) it is regarded as one of the most optimal tags for NMR studies of instable proteins (22). Since removal of the GB1-tag from our recombinantly expressed Dreb2af.l. resulted in precipitation after purification, the GB1-Dreb2af.l. fusion protein of 47 kDa was purified to near homogeneity (Supplementary Figure S1B).The interaction between Med25551–680 and Dreb2af.l. was studied using SPR, where Med25551–680 was immobilized on a sensor chip using amine coupling. The sensograms in Figure 2A show that the interaction between Med25551–680 and Dreb2af.l. displayed complex kinetics: the dissociation was bi-phasic with a fast and a slow dissociation rate. Due to the complexity of the kinetics, an accurate KD is difficult to determine. However, at low concentrations Dreb2af.1 the interaction displayed less complexity and an on rate of 22 000 M−1 S−1 could be approximated. The slow part of the dissociation rate could be approximated to 0.0023 S−1. Combining the on rate and the dissociation rate gives an approximated KD of 0.1 µM.
Figure 2.The interaction between full-length Dreb2a and Med25551–680 shows complex kinetics and involves changes in secondary structure. (A) SRP sensograms showing the binding of increasing concentrations of Dreb2af.l. to Med25551–680. An amount of 3 µM Dreb2af.l. was diluted 2-fold to 0.047 µM and duplicate samples were injected over immobilized Med25551–680. (B) CD spectra of full-length Dreb2a (dark grey) and Dreb2a168–335 domain (light grey dashed). The signals are shown as mean residue ellipticity, i.e. CD per amino acid. (C) CD spectra of Med25551–680 (blue), Dreb2af.l. (green) and the mixture of both proteins (purple) in a 1:1 molar ratio (10 µM). The CD spectrum of the sum of the individual proteins CD signals is also shown (black dashed). (D) Far-UV CD spectra of Med25551–680 (light grey) and Dreb2af.l. (dark grey) in the absence (solid) and presence (dashed) of 300 mg/ml ethylene glycol (EG).This dissociation rate for Dreb2af.1., is significantly slower than the dissociation rate for Dreb2a168–335, which thus indicate that Dreb2af.l., forms a more stable complex with Med25551–680 compared to Dreb2a168–335. Again, this KD for the full-length Dreb2a transcription activation domain (TAD) is comparable to the KD reported for the full-length VP16 TAD interaction with the human Med25 ACID domain (18).Furthermore, Dreb2af.l. contains more secondary structure per amino acid compared to Dreb2a168–335, as probed by far-UV CD (Figure 2B). This is in agreement with a prediction of secondary structure content, which suggests that the N-terminal part of Dreb2a contains more α-helix and β-sheet structures (Supplementary Figure S2). Most of the predicted ordered structures are found in the DNA-BD (amino acids 78–135) that contains three β-strands and one α-helix. Moreover, mixing of Med25551–680 and Dreb2af.l. at concentrations favoring complex formation based on SPR resulted in an increased negative far-UV CD signal (∼10% at 220 nm) as compared to the sum of the CD signals for the individual proteins (Figure 2C). Addition of the stabilizing osmolyte ethylene glycol to Dreb2af.1. resulted in increased secondary structure based on CD measurements (∼11% at 220 nm). Med25 was, however, not affected by the addition of ethylene glycol (EG) (Figure 2D). This implies that Dreb2af.1. is prone to acquire a more ordered conformation under certain conditions.

Binding of full-length Dreb2a to Med25551–680 results in a small gain of structure

To obtain sufficient expression levels of Dreb2af.l. in E. coli, the protein was fused to a GB1-tag along with the His-tag. The B1 IgG-BD of protein G (GB1) from Streptococcus is a 56 amino acids long single domain protein with one α-helix and a four-stranded β-sheet which is used as a fusion partner to increase solubility and expression of instable proteins (20,21). Due to its small size (8757.6 Da) it is regarded as one of the most optimal tags for NMR studies of instable proteins (22). Since removal of the GB1-tag from our recombinantly expressed Dreb2af.l. resulted in precipitation after purification, the GB1-Dreb2af.l. fusion protein of 47 kDa was purified to near homogeneity (Supplementary Figure S1B).

The interaction between Med25551–680 and Dreb2af.l. was studied using SPR, where Med25551–680 was immobilized on a sensor chip using amine coupling. The sensograms in Figure 2A show that the interaction between Med25551–680 and Dreb2af.l. displayed complex kinetics: the dissociation was bi-phasic with a fast and a slow dissociation rate. Due to the complexity of the kinetics, an accurate KD is difficult to determine. However, at low concentrations Dreb2af.1 the interaction displayed less complexity and an on rate of 22 000 M−1 S−1 could be approximated. The slow part of the dissociation rate could be approximated to 0.0023 S−1. Combining the on rate and the dissociation rate gives an approximated KD of 0.1 µM.
Figure 2.The interaction between full-length Dreb2a and Med25551–680 shows complex kinetics and involves changes in secondary structure. (A) SRP sensograms showing the binding of increasing concentrations of Dreb2af.l. to Med25551–680. An amount of 3 µM Dreb2af.l. was diluted 2-fold to 0.047 µM and duplicate samples were injected over immobilized Med25551–680. (B) CD spectra of full-length Dreb2a (dark grey) and Dreb2a168–335 domain (light grey dashed). The signals are shown as mean residue ellipticity, i.e. CD per amino acid. (C) CD spectra of Med25551–680 (blue), Dreb2af.l. (green) and the mixture of both proteins (purple) in a 1:1 molar ratio (10 µM). The CD spectrum of the sum of the individual proteins CD signals is also shown (black dashed). (D) Far-UV CD spectra of Med25551–680 (light grey) and Dreb2af.l. (dark grey) in the absence (solid) and presence (dashed) of 300 mg/ml ethylene glycol (EG).

The interaction between full-length Dreb2a and Med25551–680 shows complex kinetics and involves changes in secondary structure. (A) SRP sensograms showing the binding of increasing concentrations of Dreb2af.l. to Med25551–680. An amount of 3 µM Dreb2af.l. was diluted 2-fold to 0.047 µM and duplicate samples were injected over immobilized Med25551–680. (B) CD spectra of full-length Dreb2a (dark grey) and Dreb2a168–335 domain (light grey dashed). The signals are shown as mean residue ellipticity, i.e. CD per amino acid. (C) CD spectra of Med25551–680 (blue), Dreb2af.l. (green) and the mixture of both proteins (purple) in a 1:1 molar ratio (10 µM). The CD spectrum of the sum of the individual proteins CD signals is also shown (black dashed). (D) Far-UV CD spectra of Med25551–680 (light grey) and Dreb2af.l. (dark grey) in the absence (solid) and presence (dashed) of 300 mg/ml ethylene glycol (EG).

This dissociation rate for Dreb2af.1., is significantly slower than the dissociation rate for Dreb2a168–335, which thus indicate that Dreb2af.l., forms a more stable complex with Med25551–680 compared to Dreb2a168–335. Again, this KD for the full-length Dreb2a transcription activation domain (TAD) is comparable to the KD reported for the full-length VP16 TAD interaction with the human Med25 ACID domain (18).

Furthermore, Dreb2af.l. contains more secondary structure per amino acid compared to Dreb2a168–335, as probed by far-UV CD (Figure 2B). This is in agreement with a prediction of secondary structure content, which suggests that the N-terminal part of Dreb2a contains more α-helix and β-sheet structures (Supplementary Figure S2). Most of the predicted ordered structures are found in the DNA-BD (amino acids 78–135) that contains three β-strands and one α-helix. Moreover, mixing of Med25551–680 and Dreb2af.l. at concentrations favoring complex formation based on SPR resulted in an increased negative far-UV CD signal (∼10% at 220 nm) as compared to the sum of the CD signals for the individual proteins (Figure 2C). Addition of the stabilizing osmolyte ethylene glycol to Dreb2af.1. resulted in increased secondary structure based on CD measurements (∼11% at 220 nm). Med25 was, however, not affected by the addition of ethylene glycol (EG) (Figure 2D). This implies that Dreb2af.1. is prone to acquire a more ordered conformation under certain conditions.

Both Dreb2af.1. and Med25551–680 is present as monomers and the stoichiometry of binding to each other is 1:1The size of Med25551–680, Med25551–680 –GST, Dreb2af.l. and the Med25551–680 /GST– Dreb2af.l. proteins were investigated with size exclusion chromatography. Med25551–680 –GST (42 kDa) eluted at 71 ml which corresponds to an 80-kDa globular protein when compared to a standard curve generated from separation of various standard proteins on the same gel-filtration column (Figure 3A and E). This is in agreement with other studies that indicate dimerization of GST monomers (23–25). However, GST-fusion proteins are sometimes detected as monomers (26,27), a feature that can be concentration dependent (28). Dreb2af.l. that has a molecular weight of 47 kDa eluted close to 71 ml, suggesting that it either is present as an extended, non-globular monomer or as a dimer (Figure 3A and C). Complex formation of Med25551–680 –GST and Dreb2af.l. generated a shift of the peak corresponding to the elution profile of a >120 kDa protein (Figure 3A and E). This shift indicates that Med25551–680 –GST and Dreb2af.l. interact and the elution profile is most compatible with a complex composed of one Med25551–680–GST dimer (84 kDa) and one Dreb2af.l. monomer (47 kDa) when mixed in a 1:1 ratio. However, the resolution in this region is low and it is thus difficult to draw any further conclusions other than that Med25551–680–GST and Dreb2af.l form a complex. The presence of Med25551–680 –GST and Dreb2af.l. in the fractions was further confirmed with immunoblotting (Figure 3B). Moreover, when Med25551–680 without the GST-tag was fractionated it eluted at 90.4 ml, which corresponds to the elution profile of a 16.4 kDa protein (Figure 3C and E). Med25551–680 has a theoretical molecular weight of 14.7 kDa, thus indicating that Med25551–680 is present as a monomer when the GST-tag is removed. Complex formation of Dreb2af.l. with untagged Med25551–680 resulted in a small shift in migration, representing an increased molecular weight of 10 kDa (Figure 3C and E). This suggests that the binding of Med25551–680 to Dreb2af.l. is 1:1 and that the Med25551–680 interaction induces a minor change in the structure of Dreb2af.l. which affects its migration on the column. The elution of Med25551–680 at 70 ml when bound to Dreb2af.l. was confirmed with immunoblotting (Figure 3D).
Figure 3.Full-length Dreb2a and Med25551–680 bind each other mainly with a 1:1 ratio. (A) Size exclusion chromatogram for Dreb2af.l. (dark grey) and Med25551–680–GST (light grey) alone and in complex (black). The molecular weights of the standards used are marked with stars on the x-axis. (B) The presence of Dreb2af.l. (D) and Med25551–680–GST (Mg) in the eluted fractions (ml) was confirmed with immunoblot. Input (I) controls are shown in lane 1 and 2. (C) Size exclusion chromatogram for Dreb2af.l. (dark grey) and Med25551–680 (light grey) alone and in complex (black). (D) Coomassie-stained SDS–PAGE on eluted fractions (ml) show in which fractions Dreb2af.l. (D) and Med25551–680 (M) are found. (E) List of the observed molecular weights in A and C (calculated from the standard) compared to the theoretical molecular weights of the single proteins and possible complex formations of Dreb2af.l. and Med25. Ratios of 1:1, 1:2 and 2:1 of Dreb2af.l. versus Med25 are listed. (F) Formaldehyde cross-linked (0.5%) Dreb2af.l., (D), Med25551–680–GST (Mg) and Med25551–680 (M) alone or in complex was separated on a 4–12% gradient gel and the migration Dreb2af.l. was visualized by immunoblot. Complex formation of Dreb2af.l., and Med25551–680 was induced at 1:1 (1 µM each) and 1:4 (1 µM Dreb2af.l. and 4 µM Med25551–680) ratios. Denatured Dreb2af.l., in the first lane was included as a control.The size of the Med25551–680 and Dreb2af.l. complex was further investigated by formaldehyde cross-linking and SDS–PAGE. As illustrated in Figure 3F, cross-linked Dreb2af.l. did not differ in size compared to non-crosslinked and denatured Dreb2af.l., showing that it exists as a monomer. Cross-linking of the Dreb2af.l.–Med25551–680 complex reduced the migration of Dreb2af.l. with ∼10 kDa, when the proteins were mixed in a 1:1 ratio. A 4-fold excess of Med25551–680 mixed with Dreb2af.l. revealed that Dreb2af.l. can bind two monomers of Med25551–680
in vitro when Med25551–680 is present in excess. Cross-linking of Dreb2af.l. and Med25551–680–GST resulted in a complex at 130 kDa, showing that Dreb2af.l. binds Med25551–680–GST dimers. However, most of this mixture resulted in an elution profile corresponding to a size of 90 kDa, which illustrate that the Dreb2af.l. and Med25551–680–GST complex mainly consist of one Dreb2af.l. monomer and one Med25551–680–GST monomer. Addition of a 4-fold excess of Med25551–680–GST did not result in an increased amount of the larger complex of 130 kDa. In summary this illustrates that Dreb2af.l. and Med25551–680 primarily binds each other in a 1:1 ratio.

Both Dreb2af.1. and Med25551–680 is present as monomers and the stoichiometry of binding to each other is 1:1

The size of Med25551–680, Med25551–680 –GST, Dreb2af.l. and the Med25551–680 /GST– Dreb2af.l. proteins were investigated with size exclusion chromatography. Med25551–680 –GST (42 kDa) eluted at 71 ml which corresponds to an 80-kDa globular protein when compared to a standard curve generated from separation of various standard proteins on the same gel-filtration column (Figure 3A and E). This is in agreement with other studies that indicate dimerization of GST monomers (23–25). However, GST-fusion proteins are sometimes detected as monomers (26,27), a feature that can be concentration dependent (28). Dreb2af.l. that has a molecular weight of 47 kDa eluted close to 71 ml, suggesting that it either is present as an extended, non-globular monomer or as a dimer (Figure 3A and C). Complex formation of Med25551–680 –GST and Dreb2af.l. generated a shift of the peak corresponding to the elution profile of a >120 kDa protein (Figure 3A and E). This shift indicates that Med25551–680 –GST and Dreb2af.l. interact and the elution profile is most compatible with a complex composed of one Med25551–680–GST dimer (84 kDa) and one Dreb2af.l. monomer (47 kDa) when mixed in a 1:1 ratio. However, the resolution in this region is low and it is thus difficult to draw any further conclusions other than that Med25551–680–GST and Dreb2af.l form a complex. The presence of Med25551–680 –GST and Dreb2af.l. in the fractions was further confirmed with immunoblotting (Figure 3B). Moreover, when Med25551–680 without the GST-tag was fractionated it eluted at 90.4 ml, which corresponds to the elution profile of a 16.4 kDa protein (Figure 3C and E). Med25551–680 has a theoretical molecular weight of 14.7 kDa, thus indicating that Med25551–680 is present as a monomer when the GST-tag is removed. Complex formation of Dreb2af.l. with untagged Med25551–680 resulted in a small shift in migration, representing an increased molecular weight of 10 kDa (Figure 3C and E). This suggests that the binding of Med25551–680 to Dreb2af.l. is 1:1 and that the Med25551–680 interaction induces a minor change in the structure of Dreb2af.l. which affects its migration on the column. The elution of Med25551–680 at 70 ml when bound to Dreb2af.l. was confirmed with immunoblotting (Figure 3D).
Figure 3.Full-length Dreb2a and Med25551–680 bind each other mainly with a 1:1 ratio. (A) Size exclusion chromatogram for Dreb2af.l. (dark grey) and Med25551–680–GST (light grey) alone and in complex (black). The molecular weights of the standards used are marked with stars on the x-axis. (B) The presence of Dreb2af.l. (D) and Med25551–680–GST (Mg) in the eluted fractions (ml) was confirmed with immunoblot. Input (I) controls are shown in lane 1 and 2. (C) Size exclusion chromatogram for Dreb2af.l. (dark grey) and Med25551–680 (light grey) alone and in complex (black). (D) Coomassie-stained SDS–PAGE on eluted fractions (ml) show in which fractions Dreb2af.l. (D) and Med25551–680 (M) are found. (E) List of the observed molecular weights in A and C (calculated from the standard) compared to the theoretical molecular weights of the single proteins and possible complex formations of Dreb2af.l. and Med25. Ratios of 1:1, 1:2 and 2:1 of Dreb2af.l. versus Med25 are listed. (F) Formaldehyde cross-linked (0.5%) Dreb2af.l., (D), Med25551–680–GST (Mg) and Med25551–680 (M) alone or in complex was separated on a 4–12% gradient gel and the migration Dreb2af.l. was visualized by immunoblot. Complex formation of Dreb2af.l., and Med25551–680 was induced at 1:1 (1 µM each) and 1:4 (1 µM Dreb2af.l. and 4 µM Med25551–680) ratios. Denatured Dreb2af.l., in the first lane was included as a control.

Full-length Dreb2a and Med25551–680 bind each other mainly with a 1:1 ratio. (A) Size exclusion chromatogram for Dreb2af.l. (dark grey) and Med25551–680–GST (light grey) alone and in complex (black). The molecular weights of the standards used are marked with stars on the x-axis. (B) The presence of Dreb2af.l. (D) and Med25551–680–GST (Mg) in the eluted fractions (ml) was confirmed with immunoblot. Input (I) controls are shown in lane 1 and 2. (C) Size exclusion chromatogram for Dreb2af.l. (dark grey) and Med25551–680 (light grey) alone and in complex (black). (D) Coomassie-stained SDS–PAGE on eluted fractions (ml) show in which fractions Dreb2af.l. (D) and Med25551–680 (M) are found. (E) List of the observed molecular weights in A and C (calculated from the standard) compared to the theoretical molecular weights of the single proteins and possible complex formations of Dreb2af.l. and Med25. Ratios of 1:1, 1:2 and 2:1 of Dreb2af.l. versus Med25 are listed. (F) Formaldehyde cross-linked (0.5%) Dreb2af.l., (D), Med25551–680–GST (Mg) and Med25551–680 (M) alone or in complex was separated on a 4–12% gradient gel and the migration Dreb2af.l. was visualized by immunoblot. Complex formation of Dreb2af.l., and Med25551–680 was induced at 1:1 (1 µM each) and 1:4 (1 µM Dreb2af.l. and 4 µM Med25551–680) ratios. Denatured Dreb2af.l., in the first lane was included as a control.

The size of the Med25551–680 and Dreb2af.l. complex was further investigated by formaldehyde cross-linking and SDS–PAGE. As illustrated in Figure 3F, cross-linked Dreb2af.l. did not differ in size compared to non-crosslinked and denatured Dreb2af.l., showing that it exists as a monomer. Cross-linking of the Dreb2af.l.–Med25551–680 complex reduced the migration of Dreb2af.l. with ∼10 kDa, when the proteins were mixed in a 1:1 ratio. A 4-fold excess of Med25551–680 mixed with Dreb2af.l. revealed that Dreb2af.l. can bind two monomers of Med25551–680
in vitro when Med25551–680 is present in excess. Cross-linking of Dreb2af.l. and Med25551–680–GST resulted in a complex at 130 kDa, showing that Dreb2af.l. binds Med25551–680–GST dimers. However, most of this mixture resulted in an elution profile corresponding to a size of 90 kDa, which illustrate that the Dreb2af.l. and Med25551–680–GST complex mainly consist of one Dreb2af.l. monomer and one Med25551–680–GST monomer. Addition of a 4-fold excess of Med25551–680–GST did not result in an increased amount of the larger complex of 130 kDa. In summary this illustrates that Dreb2af.l. and Med25551–680 primarily binds each other in a 1:1 ratio.

The binding of Dreb2af.l. to Med25551–680 is reduced when Dreb2af.l. is pre-bound to DNAThe conserved sequence TACCGACAT, named the dehydration-responsive element (DRE), is an essential cis-acting element for induction of RD29A expression in the ABA-independent response to dehydration, high salinity and cold (29). Dreb2af.l. has been shown to bind sequence-specifically to the TACCGACAT sequence and not to the mutated TACTTTTAT sequence (30). We confirmed this by GEMSA and addition of Med25551–680 resulted in a super-shift of the Dreb2af.l.–DNA complex (Figure 4A). This shows that DNA-bound Dreb2af.l. binds Med25551–680. Since the super-shift was very small, most likely due to the small size (14.7 kDa) and high pI (9.64) of Med25551–680, Dreb2af.l. alone was loaded both in lane 4 and 7 as controls. The binding property of Dreb2af.l. to DNA was monitored with SPR where biotinylated oligonucleotides were attached to a streptavidine-coated chip. The binding curves showed that the Dreb2af.l.–DNA binding was dynamic, with fast on and off rates (Figure 4B). When the plateau values of the curves in Figure 4B were plotted, the KD was determined to 0.60 µM (Figure 4C). This KD value is slightly higher than what was detected for the Arabidopsis cold-responsive transcription factor RAV1 and the rice bZIP transcriptional activator RF2b (0.15 and 0.457 µM, respectively) (31,32). As noticed by Yamasaki et al., the DNA interaction is abolished by high salt concentrations and thus, the use of different buffers can explain the difference. We found that a lower salt concentration resulted in stronger binding of Dreb2af.l. to its DNA (data not shown).
Figure 4.Dreb2a binds its specific DNA with fast kinetics. (A) The binding of Dreb2af.l. to 29-nt oligonucleotides with the TACCGACAT sequence (Wt-probe) or the mutated TACTTTTAT sequence (mut-probe) was monitored with GEMSA. Lane 1:Labeled Wt-probe, Lane 2: Labeled Wt-probe, Dreb2af.l. (1 μM), a 1000-fold molar excess of unlabeled Wt-probe, Lane 3- Dreb2af.l. induced shift using labeled mut-probe, Lanes 4 and 7: Labeled Wt-probe and Dreb2af.l. (1 μM), Lane 5: Labeled Wt-probe, Dreb2af.l. (1 μM) and Med25551–680 (4 μM), Lane 6: Labeled Wt-probe and Med25551–680 (4 μM) (B) A biotin label oligonucleotide of 71 nt that contain the binding site TACCGACAT was immobilized on a streptavidine coated sensor chip and graded concentrations of Dreb2af.l. was injected. (C) A streptavidine coated surface was used as reference, the plateau values were plotted and the dissociation constant was determined.A gain in secondary structure was detected when Dreb2af.l. was mixed with its specific DNA but not when mixed with the mutated oligonucleotide (Figure 5A). The increase in the negative far-UV CD signal of Dreb2af.l. in presence of DNA was ∼17% at 205 and 220 nm of the total signal of the protein in the absence of DNA. This indicates that structural changes that increase the overall secondary structure content occure in Dreb2af.l. upon DNA binding. We assume that there are no structural changes in the DNA that affects the CD signal upon protein binding. In support of this, there are no changes in the DNA signal >260 nm (where protein has no contribution) with and without protein (not shown). Binding to DNA increased the stability of Dreb2af.l. as a function of time, based on CD measurements at 4°C (Figure 5B). The negative CD amplitude of Dreb2af.l. in presence of its specific DNA decreased by 14% at 205 and 220 nm after 8 days incubation, while the protein in presence of mutated DNA had a CD signal decrease of 33% at 205 and 220 nm at the same time point. Decreased CD intensity may correspond to protein unfolding and/or precipitation. In any case, the results show that Dreb2a adopts a more stable conformation when it is bound to its consensus DNA-binding site. Although comparing data after 8 days of incubation does not appear biologically relevant, the low temperature used here (4°C) slows down the reactions and thus these processes would be faster at physiological conditions. Nonetheless, this experiment is only used to emphasize a relative stability difference for Dreb2af.l. when in the presence of the two different DNAs.
Figure 5.Interaction with DNA induces conformational changes and increased integrity of Dreb2a. (A) CD spectra of Dreb2af.l. in the absence (dark grey) and presence of DNA (black) or mutated DNA (light grey) in a 1:1 molar ratio (10 µM). The CD signal of DNA is subtracted in both cases. (B) Comparison of CD spectra (DNA signal subtracted) of Dreb2af.l. in the presence of specific DNA (upper) and mutated DNA (lower) at Day 1 (solid lines) and Day 8 (dashed lines) upon continuous incubation of the mixtures at 4°C.The complex formation of Med25551–680 and Dreb2af.l. in SPR, gel filtration and CD experiments (Figures 2 and 3) clearly shows that the interaction of Med25551–680 and Dreb2af.l. is not dependent on the binding of Dreb2af.l. to DNA. To investigate what impact DNA has on the Med25551–680–Dreb2af.l. binding, additional SPR and CD measurements were performed. A reduced binding of Dreb2af.l. in conjugation with DNA was observed when Med25551–680 was attached to a sensor chip with amine coupling and Dreb2af.l. was injected alone or pre-bound to DNA (Figure 6A–C). This suggests that DNA-bound Dreb2af.l. has lower affinity to Med25551–680 compared to free Dreb2af.l.. Binding of Med25551–680 to Dreb2af.l. pre-bound to DNA did not result in any additional structural changes since the far-UV CD spectrum of the mixture and the sum of the individual protein signals of Med25551–680 and Dreb2af.l. bound to DNA overlapped (Figure 6D).
Figure 6.Dreb2a in complex with DNA show reduced binding to Med25551–680. Med25551–680 was attached to a sensor chip with amine coupling. (A) Sensogram showing dose-dependent response for Dreb2af.l.. (B) Sensogram showing dose-dependent response for Dreb2af.l. pre-bound to its specific DNA and (C) Dreb2af.l. pre-bound to the mutated DNA. (D) Far-UV CD spectra of a mixture of Med25 and pre-incubated Dreb2a with specific DNA (black) in a 1:1:1 molar ratio (10 µM), DNA signal subtracted. The sum of the individual CD signals of Dreb2a in presence of DNA and that of Med25 are also shown (dash).

The binding of Dreb2af.l. to Med25551–680 is reduced when Dreb2af.l. is pre-bound to DNA

The conserved sequence TACCGACAT, named the dehydration-responsive element (DRE), is an essential cis-acting element for induction of RD29A expression in the ABA-independent response to dehydration, high salinity and cold (29). Dreb2af.l. has been shown to bind sequence-specifically to the TACCGACAT sequence and not to the mutated TACTTTTAT sequence (30). We confirmed this by GEMSA and addition of Med25551–680 resulted in a super-shift of the Dreb2af.l.–DNA complex (Figure 4A). This shows that DNA-bound Dreb2af.l. binds Med25551–680. Since the super-shift was very small, most likely due to the small size (14.7 kDa) and high pI (9.64) of Med25551–680, Dreb2af.l. alone was loaded both in lane 4 and 7 as controls. The binding property of Dreb2af.l. to DNA was monitored with SPR where biotinylated oligonucleotides were attached to a streptavidine-coated chip. The binding curves showed that the Dreb2af.l.–DNA binding was dynamic, with fast on and off rates (Figure 4B). When the plateau values of the curves in Figure 4B were plotted, the KD was determined to 0.60 µM (Figure 4C). This KD value is slightly higher than what was detected for the Arabidopsis cold-responsive transcription factor RAV1 and the rice bZIP transcriptional activator RF2b (0.15 and 0.457 µM, respectively) (31,32). As noticed by Yamasaki et al., the DNA interaction is abolished by high salt concentrations and thus, the use of different buffers can explain the difference. We found that a lower salt concentration resulted in stronger binding of Dreb2af.l. to its DNA (data not shown).
Figure 4.Dreb2a binds its specific DNA with fast kinetics. (A) The binding of Dreb2af.l. to 29-nt oligonucleotides with the TACCGACAT sequence (Wt-probe) or the mutated TACTTTTAT sequence (mut-probe) was monitored with GEMSA. Lane 1:Labeled Wt-probe, Lane 2: Labeled Wt-probe, Dreb2af.l. (1 μM), a 1000-fold molar excess of unlabeled Wt-probe, Lane 3- Dreb2af.l. induced shift using labeled mut-probe, Lanes 4 and 7: Labeled Wt-probe and Dreb2af.l. (1 μM), Lane 5: Labeled Wt-probe, Dreb2af.l. (1 μM) and Med25551–680 (4 μM), Lane 6: Labeled Wt-probe and Med25551–680 (4 μM) (B) A biotin label oligonucleotide of 71 nt that contain the binding site TACCGACAT was immobilized on a streptavidine coated sensor chip and graded concentrations of Dreb2af.l. was injected. (C) A streptavidine coated surface was used as reference, the plateau values were plotted and the dissociation constant was determined.

Dreb2a binds its specific DNA with fast kinetics. (A) The binding of Dreb2af.l. to 29-nt oligonucleotides with the TACCGACAT sequence (Wt-probe) or the mutated TACTTTTAT sequence (mut-probe) was monitored with GEMSA. Lane 1:Labeled Wt-probe, Lane 2: Labeled Wt-probe, Dreb2af.l. (1 μM), a 1000-fold molar excess of unlabeled Wt-probe, Lane 3- Dreb2af.l. induced shift using labeled mut-probe, Lanes 4 and 7: Labeled Wt-probe and Dreb2af.l. (1 μM), Lane 5: Labeled Wt-probe, Dreb2af.l. (1 μM) and Med25551–680 (4 μM), Lane 6: Labeled Wt-probe and Med25551–680 (4 μM) (B) A biotin label oligonucleotide of 71 nt that contain the binding site TACCGACAT was immobilized on a streptavidine coated sensor chip and graded concentrations of Dreb2af.l. was injected. (C) A streptavidine coated surface was used as reference, the plateau values were plotted and the dissociation constant was determined.

A gain in secondary structure was detected when Dreb2af.l. was mixed with its specific DNA but not when mixed with the mutated oligonucleotide (Figure 5A). The increase in the negative far-UV CD signal of Dreb2af.l. in presence of DNA was ∼17% at 205 and 220 nm of the total signal of the protein in the absence of DNA. This indicates that structural changes that increase the overall secondary structure content occure in Dreb2af.l. upon DNA binding. We assume that there are no structural changes in the DNA that affects the CD signal upon protein binding. In support of this, there are no changes in the DNA signal >260 nm (where protein has no contribution) with and without protein (not shown). Binding to DNA increased the stability of Dreb2af.l. as a function of time, based on CD measurements at 4°C (Figure 5B). The negative CD amplitude of Dreb2af.l. in presence of its specific DNA decreased by 14% at 205 and 220 nm after 8 days incubation, while the protein in presence of mutated DNA had a CD signal decrease of 33% at 205 and 220 nm at the same time point. Decreased CD intensity may correspond to protein unfolding and/or precipitation. In any case, the results show that Dreb2a adopts a more stable conformation when it is bound to its consensus DNA-binding site. Although comparing data after 8 days of incubation does not appear biologically relevant, the low temperature used here (4°C) slows down the reactions and thus these processes would be faster at physiological conditions. Nonetheless, this experiment is only used to emphasize a relative stability difference for Dreb2af.l. when in the presence of the two different DNAs.
Figure 5.Interaction with DNA induces conformational changes and increased integrity of Dreb2a. (A) CD spectra of Dreb2af.l. in the absence (dark grey) and presence of DNA (black) or mutated DNA (light grey) in a 1:1 molar ratio (10 µM). The CD signal of DNA is subtracted in both cases. (B) Comparison of CD spectra (DNA signal subtracted) of Dreb2af.l. in the presence of specific DNA (upper) and mutated DNA (lower) at Day 1 (solid lines) and Day 8 (dashed lines) upon continuous incubation of the mixtures at 4°C.

Interaction with DNA induces conformational changes and increased integrity of Dreb2a. (A) CD spectra of Dreb2af.l. in the absence (dark grey) and presence of DNA (black) or mutated DNA (light grey) in a 1:1 molar ratio (10 µM). The CD signal of DNA is subtracted in both cases. (B) Comparison of CD spectra (DNA signal subtracted) of Dreb2af.l. in the presence of specific DNA (upper) and mutated DNA (lower) at Day 1 (solid lines) and Day 8 (dashed lines) upon continuous incubation of the mixtures at 4°C.

The complex formation of Med25551–680 and Dreb2af.l. in SPR, gel filtration and CD experiments (Figures 2 and 3) clearly shows that the interaction of Med25551–680 and Dreb2af.l. is not dependent on the binding of Dreb2af.l. to DNA. To investigate what impact DNA has on the Med25551–680–Dreb2af.l. binding, additional SPR and CD measurements were performed. A reduced binding of Dreb2af.l. in conjugation with DNA was observed when Med25551–680 was attached to a sensor chip with amine coupling and Dreb2af.l. was injected alone or pre-bound to DNA (Figure 6A–C). This suggests that DNA-bound Dreb2af.l. has lower affinity to Med25551–680 compared to free Dreb2af.l.. Binding of Med25551–680 to Dreb2af.l. pre-bound to DNA did not result in any additional structural changes since the far-UV CD spectrum of the mixture and the sum of the individual protein signals of Med25551–680 and Dreb2af.l. bound to DNA overlapped (Figure 6D).
Figure 6.Dreb2a in complex with DNA show reduced binding to Med25551–680. Med25551–680 was attached to a sensor chip with amine coupling. (A) Sensogram showing dose-dependent response for Dreb2af.l.. (B) Sensogram showing dose-dependent response for Dreb2af.l. pre-bound to its specific DNA and (C) Dreb2af.l. pre-bound to the mutated DNA. (D) Far-UV CD spectra of a mixture of Med25 and pre-incubated Dreb2a with specific DNA (black) in a 1:1:1 molar ratio (10 µM), DNA signal subtracted. The sum of the individual CD signals of Dreb2a in presence of DNA and that of Med25 are also shown (dash).

Dreb2a in complex with DNA show reduced binding to Med25551–680. Med25551–680 was attached to a sensor chip with amine coupling. (A) Sensogram showing dose-dependent response for Dreb2af.l.. (B) Sensogram showing dose-dependent response for Dreb2af.l. pre-bound to its specific DNA and (C) Dreb2af.l. pre-bound to the mutated DNA. (D) Far-UV CD spectra of a mixture of Med25 and pre-incubated Dreb2a with specific DNA (black) in a 1:1:1 molar ratio (10 µM), DNA signal subtracted. The sum of the individual CD signals of Dreb2a in presence of DNA and that of Med25 are also shown (dash).

The ACID domain of human Med25394–543 show structural similarities to Med25551–680 from A. thaliana and forms a complex with Dreb2af.l.We have previously reported that there is only a week homology between the amino acid sequence of Med25 from A. thaliana and human (15). Interestingly, prediction of the secondary structure of the ACID domains of human Med25 (humMed25394–543) and A. thaliana (Med25551–680) revealed that some similarities exists in the distribution of α-helixes and β-sheets (Figure 7A). The NMR-structure of the human Med25 ACID domain was recently solved (18,33,34), and it was shown to contain a closed β-barrel with seven strands and three α-helixes. This is in line with the prediction of human Med25394–543 in Figure 7A, with the expectation that the third β-sheet (B3) is predicted to be α-helical in the beginning (amino acids 447–450). The human Med25394–543 is predicted to contain one α-helix more than Med25551–680 and CD measurement on the two different Med25 ACIDs did indeed revealed that human Med25394–543 contain more secondary structure (Figure 7B), Pull-down experiments with Dreb2af.l. and human Med25394–543 showed that when the proteins were mixed, a small fraction eluted together (bound) from the beads to which Dreb2af.l. had been attached (Figure 7C). Most of the humMed25394–543 did however not bind Dreb2af.l. but was found in flow through. Notably, since Dreb2af.l. is a plant-specific transcription factor with no apparent homolog in humans, the interaction was expected to be weak. To confirm that humMed25394–543 interacts with Dreb2af.l., cross-linking were made prior to SDS–PAGE and immunoblotting (Figure 7D). In this assay, Dreb2af.l. formed a complex together with humMed25394–543 in a similar dose-dependent manner as that detected for Med25551–680 when mixed in 1:1 and 1:4 ratios (Figures 7D and 3F). Importantly, addition of GST did not result in complex formation with Dreb2af.l.. The interaction between humMed25394–543 and Df.l. was studied using SPR, where His-tagged Df.l. was immobilized on a NTA sensor chip. The sensograms show that the interaction between humMed25394–543 and Df.l. displayed similar complex kinetics as the A. thaliana Med25551–680 and Df.l interaction (Figure 7E) and subsequently a similar dissociation constant. Due to the complexity of the kinetics, an accurate dissociation constant could not be determined but is in the same range as of Med25ACID. Detection of a humMed25394–543–Dreb2af.l. complex with three independent assays suggests that structural features that are important for interaction with activators must be conserved between Med25ACID from A. thaliana and humans.
Figure 7.Structural similarity between the ACID-domains of Med25 from human and A. thaliana mediates similar binding propensity to Dreb2af.l.. (A) Prediction of secondary structure in the ACID-domain of Med25 from human (humMed25394–543) and A. thaliana (aMed25551–680) was made utilizing the jpred 3 server. The humMed25 contains somewhat more α-helixes (H) in dark grey and β-strands (E) in light grey than aMed25. (B) CD spectra of humMed25394–543 and aMed25551–680 confirmed that humMed25394–543 contains more ordered structures. (C) Dreb2af.l. (D) was bound to IgG sepharose through its GB-1 tag. Addition of humMed25394–543 (hM) resulted in a Dreb2af.l.–humMed25394–543 complex that was visualized by SDS–PAGE and immunoblotting. (D) Complex formation of Dreb2af.l. and humMed25394–543 (hM) was investigated with formaldehyde cross-linking (0.5%) and immunoblot. Dreb2af.l., and humMed25394–543 was mixed at 1:1 (1 µM each) and 1:4 (1 µM Dreb2af.l. and 4 µM Med25) ratios. An amount of 4 µM humMed25394–543 alone, and Dreb2af.l. incubation with GST (1:4) severed as negative controls. Complex formation between 1 µM aMed25551–680 (M) and 1 µM Dreb2af.l. was included as a positive control. (E) SRP sensograms showing the binding of increasing concentrations of humMed25394–543 to Df.l.. An amount of 3 µM of humMed25394–543 was diluted 1:1 down to 0.047 µM and samples were passed over a NTA sensor chip sensorchip where Df.l. was immobilized via its His-tag.

The ACID domain of human Med25394–543 show structural similarities to Med25551–680 from A. thaliana and forms a complex with Dreb2af.l.

We have previously reported that there is only a week homology between the amino acid sequence of Med25 from A. thaliana and human (15). Interestingly, prediction of the secondary structure of the ACID domains of human Med25 (humMed25394–543) and A. thaliana (Med25551–680) revealed that some similarities exists in the distribution of α-helixes and β-sheets (Figure 7A). The NMR-structure of the human Med25 ACID domain was recently solved (18,33,34), and it was shown to contain a closed β-barrel with seven strands and three α-helixes. This is in line with the prediction of human Med25394–543 in Figure 7A, with the expectation that the third β-sheet (B3) is predicted to be α-helical in the beginning (amino acids 447–450). The human Med25394–543 is predicted to contain one α-helix more than Med25551–680 and CD measurement on the two different Med25 ACIDs did indeed revealed that human Med25394–543 contain more secondary structure (Figure 7B), Pull-down experiments with Dreb2af.l. and human Med25394–543 showed that when the proteins were mixed, a small fraction eluted together (bound) from the beads to which Dreb2af.l. had been attached (Figure 7C). Most of the humMed25394–543 did however not bind Dreb2af.l. but was found in flow through. Notably, since Dreb2af.l. is a plant-specific transcription factor with no apparent homolog in humans, the interaction was expected to be weak. To confirm that humMed25394–543 interacts with Dreb2af.l., cross-linking were made prior to SDS–PAGE and immunoblotting (Figure 7D). In this assay, Dreb2af.l. formed a complex together with humMed25394–543 in a similar dose-dependent manner as that detected for Med25551–680 when mixed in 1:1 and 1:4 ratios (Figures 7D and 3F). Importantly, addition of GST did not result in complex formation with Dreb2af.l.. The interaction between humMed25394–543 and Df.l. was studied using SPR, where His-tagged Df.l. was immobilized on a NTA sensor chip. The sensograms show that the interaction between humMed25394–543 and Df.l. displayed similar complex kinetics as the A. thaliana Med25551–680 and Df.l interaction (Figure 7E) and subsequently a similar dissociation constant. Due to the complexity of the kinetics, an accurate dissociation constant could not be determined but is in the same range as of Med25ACID. Detection of a humMed25394–543–Dreb2af.l. complex with three independent assays suggests that structural features that are important for interaction with activators must be conserved between Med25ACID from A. thaliana and humans.
Figure 7.Structural similarity between the ACID-domains of Med25 from human and A. thaliana mediates similar binding propensity to Dreb2af.l.. (A) Prediction of secondary structure in the ACID-domain of Med25 from human (humMed25394–543) and A. thaliana (aMed25551–680) was made utilizing the jpred 3 server. The humMed25 contains somewhat more α-helixes (H) in dark grey and β-strands (E) in light grey than aMed25. (B) CD spectra of humMed25394–543 and aMed25551–680 confirmed that humMed25394–543 contains more ordered structures. (C) Dreb2af.l. (D) was bound to IgG sepharose through its GB-1 tag. Addition of humMed25394–543 (hM) resulted in a Dreb2af.l.–humMed25394–543 complex that was visualized by SDS–PAGE and immunoblotting. (D) Complex formation of Dreb2af.l. and humMed25394–543 (hM) was investigated with formaldehyde cross-linking (0.5%) and immunoblot. Dreb2af.l., and humMed25394–543 was mixed at 1:1 (1 µM each) and 1:4 (1 µM Dreb2af.l. and 4 µM Med25) ratios. An amount of 4 µM humMed25394–543 alone, and Dreb2af.l. incubation with GST (1:4) severed as negative controls. Complex formation between 1 µM aMed25551–680 (M) and 1 µM Dreb2af.l. was included as a positive control. (E) SRP sensograms showing the binding of increasing concentrations of humMed25394–543 to Df.l.. An amount of 3 µM of humMed25394–543 was diluted 1:1 down to 0.047 µM and samples were passed over a NTA sensor chip sensorchip where Df.l. was immobilized via its His-tag.

Structural similarity between the ACID-domains of Med25 from human and A. thaliana mediates similar binding propensity to Dreb2af.l.. (A) Prediction of secondary structure in the ACID-domain of Med25 from human (humMed25394–543) and A. thaliana (aMed25551–680) was made utilizing the jpred 3 server. The humMed25 contains somewhat more α-helixes (H) in dark grey and β-strands (E) in light grey than aMed25. (B) CD spectra of humMed25394–543 and aMed25551–680 confirmed that humMed25394–543 contains more ordered structures. (C) Dreb2af.l. (D) was bound to IgG sepharose through its GB-1 tag. Addition of humMed25394–543 (hM) resulted in a Dreb2af.l.–humMed25394–543 complex that was visualized by SDS–PAGE and immunoblotting. (D) Complex formation of Dreb2af.l. and humMed25394–543 (hM) was investigated with formaldehyde cross-linking (0.5%) and immunoblot. Dreb2af.l., and humMed25394–543 was mixed at 1:1 (1 µM each) and 1:4 (1 µM Dreb2af.l. and 4 µM Med25) ratios. An amount of 4 µM humMed25394–543 alone, and Dreb2af.l. incubation with GST (1:4) severed as negative controls. Complex formation between 1 µM aMed25551–680 (M) and 1 µM Dreb2af.l. was included as a positive control. (E) SRP sensograms showing the binding of increasing concentrations of humMed25394–543 to Df.l.. An amount of 3 µM of humMed25394–543 was diluted 1:1 down to 0.047 µM and samples were passed over a NTA sensor chip sensorchip where Df.l. was immobilized via its His-tag.

DISCUSSIONConformational disorder is found in many regulatory proteins mediating signal transduction and transcription. Disordered regions may confer advantages over folded proteins in binding and the capability of disordered protein regions to interact with multiple partners is beneficial in organizing multi-protein complexes. We had previously identified the Med25-BD in Dreb2a to amino acids 169–254 using the yeast two hybrid system (16). This domain is located between a NRD (amino acids 136–165) and an AD (amino acids 254–335) which were identified by others (35). Surprisingly, we now find that both the BD and the AD are required for interaction using purified proteins in SPR assays. A difference between the two hybrid assays and the interaction studies in this article is that the domains that are used in the two hybrid assays are expressed as fusion proteins with the Gal4 AD. It is possible that the Dreb2a BD by itself is too short to adopt any secondary structure, but that addition of the Gal4 AD helps to stabilize the Dreb2a BD such that it can interact with Med25. Many activator domains are known to be intrinsically unstructured in solution, which has led to the proposal of an induced fit model where flexible domains become structured upon binding to the appropriate target proteins (36). Initially we thought that this was not the case for Dreb2a, since the CD signal for the Dreb2a C-terminal domain (BD + AD) and the Med25 ACID domain after mixing overlapped with the calculated sum of the individual CD signals for each protein, thus indicating that no conformational changes occur upon binding. However, when Dreb2af.l protein was used we detected an increased negative far-UV CD signal for the Dreb2af.l.–Med25 mixture compared to the sum of the CD signals for the individual proteins. We also found that the dissociation rate for the complex with the Med25 ACID domain was slower when using the Dreb2af.l. protein relative to the Dreb2a168–335, which indicate a more stable complex. Thus, there are structural changes in the proteins that take place upon binding of Dreb2af.l. and the Med25 ACID domain. These conformational changes may occur in both proteins or only in one of them; the CD experiments cannot resolve this issue. However, since addition of the stabilizing osmolyte ethylene glycol resulted in a gain of secondary structure of Dreb2af.l. but not Med25, we speculate that the major structural changes that occur upon Dreb2af.l.–Med25 interaction represent changes in Dreb2af.l..The fast kinetics that we report for the Dreb2a–Med25 interaction is similar to results reported for other transcriptional activators and their interaction with different target proteins. Activator domains are typically intrinsically unstructured, such as the domains of p53, c-Myc, VP16 and the glucocorticoid receptor (37–40). Several reports show that target binding mediates folding of the unstructured domain (41,42). When target induced folding of Gal-4 and VP16 was tested with surface Plasmon resonance (43), the interaction to a set of target proteins displayed bi-phasic kinetics with a fast initial binding followed by a slower phase. This is in line with our results for the interaction between the Dreb2af.l. protein and the Med25 ACID domain.Our results using Dreb2a that has been pre bound to its cognate DNA sequence might seem counterintuitive. Figure 6 shows that Dreb2a has a lower affinity for the Med25 ACID domain in the presence of the consensus Dreb2a-binding sequence compared to when a mutated version of the sequence is present. Thus, the conformational changes that occur in Dreb2a when it binds to its consensus DNA sequence, result in a reduced binding affinity to the Med25 ACID domain. This is however what we predicted in our recently published hypothesis (16). We suggested that the conformational changes that are induced in Dreb2a when it binds to DNA leads to a reduced binding to the Med25 ACID domain, but an increased interaction to another mediator subunit located in the vicinity of Med25. Our hypothesis takes into consideration that Dreb2a can function both as a transcriptional activator and a transcriptional repressor based on our own functional assays in A. thaliana mutant lines that lack Dreb2a, on results from two hybrid experiments, and with results reported by others concerning A. thaliana lines where Dreb2a was overexpressed (16,30). We proposed that when interaction between the NRD of Dreb2a and Med25 is disrupted, either by deletion of the NRD of Dreb2a or by deletion of Med25, then Dreb2a functions as an activator, both for induction of genes involved in drought and in development. In contrast, we suggested that Dreb2a can execute its repressing effect of by placing its NRD in the vicinity of another mediator subunit [see the discussion in ref. (16)]. It is possible that the conformational changes in Dreb2a that would place the NRD closer to another mediator subunit, but which reduces the interaction between Dreb2a and the Med25 ACID domain are caused by interactions between Dreb2a and DNA. Further studies to identify other mediator subunits that can interact with different conformational forms of Dreb2a are required in order to verify that our hypothesis is valid. We have already identified A. thaliana mutants that lack other mediator subunits than Med25, but show phenotypes indicating that they are involved in the same signal pathways as Med25 (data not shown).Finally, it is interesting to notice that the ACID domain in the human Med25 protein shows structural similarities to the Med25 ACID domain from A. thaliana, even if the amino acid sequences differ considerably between the two species. This structural similarity is also manifested by the interaction between Dreb2a and the human Med25 ACID domain that we detect using pull down, cross-linking and SRP assays. The ACID domain was initially identified in human Med25 due to its interaction with the herpes simplex VP16 AD. It would however be interesting to identify the natural human transcriptional regulator(s) that interacts with human Med25. Work along these lines is in progress.

Conformational disorder is found in many regulatory proteins mediating signal transduction and transcription. Disordered regions may confer advantages over folded proteins in binding and the capability of disordered protein regions to interact with multiple partners is beneficial in organizing multi-protein complexes. We had previously identified the Med25-BD in Dreb2a to amino acids 169–254 using the yeast two hybrid system (16). This domain is located between a NRD (amino acids 136–165) and an AD (amino acids 254–335) which were identified by others (35). Surprisingly, we now find that both the BD and the AD are required for interaction using purified proteins in SPR assays. A difference between the two hybrid assays and the interaction studies in this article is that the domains that are used in the two hybrid assays are expressed as fusion proteins with the Gal4 AD. It is possible that the Dreb2a BD by itself is too short to adopt any secondary structure, but that addition of the Gal4 AD helps to stabilize the Dreb2a BD such that it can interact with Med25. Many activator domains are known to be intrinsically unstructured in solution, which has led to the proposal of an induced fit model where flexible domains become structured upon binding to the appropriate target proteins (36). Initially we thought that this was not the case for Dreb2a, since the CD signal for the Dreb2a C-terminal domain (BD + AD) and the Med25 ACID domain after mixing overlapped with the calculated sum of the individual CD signals for each protein, thus indicating that no conformational changes occur upon binding. However, when Dreb2af.l protein was used we detected an increased negative far-UV CD signal for the Dreb2af.l.–Med25 mixture compared to the sum of the CD signals for the individual proteins. We also found that the dissociation rate for the complex with the Med25 ACID domain was slower when using the Dreb2af.l. protein relative to the Dreb2a168–335, which indicate a more stable complex. Thus, there are structural changes in the proteins that take place upon binding of Dreb2af.l. and the Med25 ACID domain. These conformational changes may occur in both proteins or only in one of them; the CD experiments cannot resolve this issue. However, since addition of the stabilizing osmolyte ethylene glycol resulted in a gain of secondary structure of Dreb2af.l. but not Med25, we speculate that the major structural changes that occur upon Dreb2af.l.–Med25 interaction represent changes in Dreb2af.l..

The fast kinetics that we report for the Dreb2a–Med25 interaction is similar to results reported for other transcriptional activators and their interaction with different target proteins. Activator domains are typically intrinsically unstructured, such as the domains of p53, c-Myc, VP16 and the glucocorticoid receptor (37–40). Several reports show that target binding mediates folding of the unstructured domain (41,42). When target induced folding of Gal-4 and VP16 was tested with surface Plasmon resonance (43), the interaction to a set of target proteins displayed bi-phasic kinetics with a fast initial binding followed by a slower phase. This is in line with our results for the interaction between the Dreb2af.l. protein and the Med25 ACID domain.

Our results using Dreb2a that has been pre bound to its cognate DNA sequence might seem counterintuitive. Figure 6 shows that Dreb2a has a lower affinity for the Med25 ACID domain in the presence of the consensus Dreb2a-binding sequence compared to when a mutated version of the sequence is present. Thus, the conformational changes that occur in Dreb2a when it binds to its consensus DNA sequence, result in a reduced binding affinity to the Med25 ACID domain. This is however what we predicted in our recently published hypothesis (16). We suggested that the conformational changes that are induced in Dreb2a when it binds to DNA leads to a reduced binding to the Med25 ACID domain, but an increased interaction to another mediator subunit located in the vicinity of Med25. Our hypothesis takes into consideration that Dreb2a can function both as a transcriptional activator and a transcriptional repressor based on our own functional assays in A. thaliana mutant lines that lack Dreb2a, on results from two hybrid experiments, and with results reported by others concerning A. thaliana lines where Dreb2a was overexpressed (16,30). We proposed that when interaction between the NRD of Dreb2a and Med25 is disrupted, either by deletion of the NRD of Dreb2a or by deletion of Med25, then Dreb2a functions as an activator, both for induction of genes involved in drought and in development. In contrast, we suggested that Dreb2a can execute its repressing effect of by placing its NRD in the vicinity of another mediator subunit [see the discussion in ref. (16)]. It is possible that the conformational changes in Dreb2a that would place the NRD closer to another mediator subunit, but which reduces the interaction between Dreb2a and the Med25 ACID domain are caused by interactions between Dreb2a and DNA. Further studies to identify other mediator subunits that can interact with different conformational forms of Dreb2a are required in order to verify that our hypothesis is valid. We have already identified A. thaliana mutants that lack other mediator subunits than Med25, but show phenotypes indicating that they are involved in the same signal pathways as Med25 (data not shown).

Finally, it is interesting to notice that the ACID domain in the human Med25 protein shows structural similarities to the Med25 ACID domain from A. thaliana, even if the amino acid sequences differ considerably between the two species. This structural similarity is also manifested by the interaction between Dreb2a and the human Med25 ACID domain that we detect using pull down, cross-linking and SRP assays. The ACID domain was initially identified in human Med25 due to its interaction with the herpes simplex VP16 AD. It would however be interesting to identify the natural human transcriptional regulator(s) that interacts with human Med25. Work along these lines is in progress.

SUPPLEMENTARY DATASupplementary Data are available at NAR Online: Supplementary Figures S1 and S2.

SUPPLEMENTARY DATA

Supplementary Data are available at NAR Online: Supplementary Figures S1 and S2.

Conflict of interest statement. None declared.

Supplementary MaterialSupplementary Data

Supplementary Material

Supplementary Data
