---
pmid: '11427894'
title: Structure of the histone deacetylase SIRT2.
authors:
- Finnin MS
- Donigian JR
- Pavletich NP
journal: Nat Struct Biol
year: '2001'
full_text_available: false
doi: 10.1038/89668
---

# Structure of the histone deacetylase SIRT2.
**Authors:** Finnin MS, Donigian JR, Pavletich NP
**Journal:** Nat Struct Biol (2001)
**DOI:** [10.1038/89668](https://doi.org/10.1038/89668)

## Abstract

1. Nat Struct Biol. 2001 Jul;8(7):621-5. doi: 10.1038/89668.

Structure of the histone deacetylase SIRT2.

Finnin MS(1), Donigian JR, Pavletich NP.

Author information:
(1)Cellular Biochemistry and Biophysics Program, and Howard Hughes Medical 
Institute, Memorial Sloan-Kettering Cancer Center, New York, New York 10021, 
USA.

Sir2 is an NAD-dependent histone deacetylase that mediates transcriptional 
silencing at mating-type loci, telomeres and ribosomal gene clusters, and has a 
critical role in the determination of life span in yeast and Caenorhabditis 
elegans. The 1.7 A crystal structure of the 323 amino acid catalytic core of 
human SIRT2, a homolog of yeast Sir2, reveals an NAD-binding domain, which is a 
variant of the Rossmann fold, and a smaller domain composed of a helical module 
and a zinc-binding module. A conserved large groove at the interface of the two 
domains is the likely site of catalysis based on mutagenesis. Intersecting this 
large groove, there is a pocket formed by the helical module. The pocket is 
lined with hydrophobic residues conserved within each of the five Sir2 classes, 
suggesting that it is a class-specific protein-binding site.

DOI: 10.1038/89668
PMID: 11427894 [Indexed for MEDLINE]
