---
pmid: '28065597'
title: A Global Analysis of the Receptor Tyrosine Kinase-Protein Phosphatase Interactome.
authors:
- Yao Z
- Darowski K
- St-Denis N
- Wong V
- Offensperger F
- Villedieu A
- Amin S
- Malty R
- Aoki H
- Guo H
- Xu Y
- Iorio C
- Kotlyar M
- Emili A
- Jurisica I
- Neel BG
- Babu M
- Gingras AC
- Stagljar I
journal: Mol Cell
year: '2017'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC5663465
doi: 10.1016/j.molcel.2016.12.004
---

# A Global Analysis of the Receptor Tyrosine Kinase-Protein Phosphatase Interactome.
**Authors:** Yao Z, Darowski K, St-Denis N, Wong V, Offensperger F, Villedieu A, Amin S, Malty R, Aoki H, Guo H, Xu Y, Iorio C, Kotlyar M, Emili A, Jurisica I, Neel BG, Babu M, Gingras AC, Stagljar I
**Journal:** Mol Cell (2017)
**DOI:** [10.1016/j.molcel.2016.12.004](https://doi.org/10.1016/j.molcel.2016.12.004)
**PMC:** [PMC5663465](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5663465/)

## Abstract

1. Mol Cell. 2017 Jan 19;65(2):347-360. doi: 10.1016/j.molcel.2016.12.004. Epub 
2017 Jan 5.

A Global Analysis of the Receptor Tyrosine Kinase-Protein Phosphatase 
Interactome.

Yao Z(1), Darowski K(1), St-Denis N(2), Wong V(1), Offensperger F(1), Villedieu 
A(1), Amin S(3), Malty R(3), Aoki H(3), Guo H(1), Xu Y(4), Iorio C(4), Kotlyar 
M(4), Emili A(5), Jurisica I(6), Neel BG(7), Babu M(3), Gingras AC(8), Stagljar 
I(9).

Author information:
(1)Donnelly Centre, University of Toronto, Toronto, ON M5S 3E1, Canada.
(2)Lunenfeld-Tanenbaum Research Institute at Mount Sinai, Toronto, ON M5G 1X5, 
Canada.
(3)Department of Biochemistry, University of Regina, Regina, SK S4S 0A2, Canada.
(4)Princess Margaret Cancer Centre, University Health Network, Toronto, ON M5G 
1L7, Canada.
(5)Donnelly Centre, University of Toronto, Toronto, ON M5S 3E1, Canada; 
Department of Molecular Genetics, University of Toronto, Toronto, ON M5S 1A8, 
Canada.
(6)Princess Margaret Cancer Centre, University Health Network, Toronto, ON M5G 
1L7, Canada; Department of Medical Biophysics, University of Toronto, Toronto, 
ON M5G 1L7, Canada; Department of Computer Science, University of Toronto, 
Toronto, ON M5S 3G4, Canada; Institute of Neuroimmunology, Slovak Academy of 
Sciences, 845 10 Bratislava, Slovak Republic.
(7)Princess Margaret Cancer Centre, University Health Network, Toronto, ON M5G 
1L7, Canada; Department of Medical Biophysics, University of Toronto, Toronto, 
ON M5G 1L7, Canada; Department of Biochemistry, University of Toronto, Toronto, 
ON M5S 1A8, Canada.
(8)Lunenfeld-Tanenbaum Research Institute at Mount Sinai, Toronto, ON M5G 1X5, 
Canada; Department of Molecular Genetics, University of Toronto, Toronto, ON M5S 
1A8, Canada.
(9)Donnelly Centre, University of Toronto, Toronto, ON M5S 3E1, Canada; 
Department of Molecular Genetics, University of Toronto, Toronto, ON M5S 1A8, 
Canada; Department of Biochemistry, University of Toronto, Toronto, ON M5S 1A8, 
Canada. Electronic address: igor.stagljar@utoronto.ca.

Receptor tyrosine kinases (RTKs) and protein phosphatases comprise protein 
families that play crucial roles in cell signaling. We used two protein-protein 
interaction (PPI) approaches, the membrane yeast two-hybrid (MYTH) and the 
mammalian membrane two-hybrid (MaMTH), to map the PPIs between human RTKs and 
phosphatases. The resulting RTK-phosphatase interactome reveals a considerable 
number of previously unidentified interactions and suggests specific roles for 
different phosphatase families. Additionally, the differential PPIs of some 
protein tyrosine phosphatases (PTPs) and their mutants suggest diverse 
mechanisms of these PTPs in the regulation of RTK signaling. We further found 
that PTPRH and PTPRB directly dephosphorylate EGFR and repress its downstream 
signaling. By contrast, PTPRA plays a dual role in EGFR signaling: besides 
facilitating EGFR dephosphorylation, it enhances downstream ERK signaling by 
activating SRC. This comprehensive RTK-phosphatase interactome study provides a 
broad and deep view of RTK signaling.

Copyright © 2017 Elsevier Inc. All rights reserved.

DOI: 10.1016/j.molcel.2016.12.004
PMCID: PMC5663465
PMID: 28065597 [Indexed for MEDLINE]

## Full Text

INTRODUCTION

Receptor tyrosine kinases (RTKs) receive varied extracellular chemical signals and process and relay the information to the intracellular space ( Lemmon and Schlessinger, 2010 ). Consequently, RTKs play essential roles in the regulation of multiple cellular processes. Aberrant regulation of these signaling processes can cause various disorders. Therefore, delineating the detailed molecular mechanisms of RTK signaling is critical to both completely understanding the pathophysiology of many diseases and the development of new preventative and treatment measures.

The human genome harbors 58 RTKs ( Manning et al., 2002 ). Their activation leads to tyrosine phosphorylation. The phosphorylated tyrosines recruit downstream proteins that contain SRC homology 2 (SH2) or phosphotyrosine-binding (PTB) domains, which can be enzymes, regulatory proteins and adaptor proteins that are responsible for activation of multiple downstream cascades. Some recruited proteins are also involved in negative regulation of RTK signaling. RTKs also undergo Ser/Thr phosphorylation although the function of most Ser/Thr phosphorylation is still not fully understood.

Dephosphorylation of RTKs is carried out by protein phosphatases. Approximately 140 protein phosphatases (if one counts only catalytic subunits for multi-subunit phosphatases) have been identified in the human genome. Unlike kinases, protein phosphatases evolved from distinct genes and employ different enzymatic mechanisms ( Li et al., 2013 ). They are traditionally divided into two classes, protein Ser/Thr phosphatases (PSPs) and protein tyrosine phosphatases (PTPs). PSPs include the PPP, PPM, and FCP/SCP families ( Li et al., 2013 ). The cysteine-based PTP superfamily includes about 100 members ( Alonso et al., 2004 ; Tonks, 2006 ), grouped into classical PTPs and dual specificity phosphatases (DUSPs). Classical PTPs play critical roles in tyrosine kinase signaling ( Neel and Tonks, 1997 ), whereas DUSPs can dephosphorylate Tyr and Ser/Thr residues. Some DUSPs function as lipid or glycogen phosphatases. The EYA family comprises a small set with an aspartate-based catalytic domain ( Alonso et al., 2004 ; Jemc and Rebay, 2007 ).

As RTK signaling is reversible, the removal of phosphate is as important as its addition. However, the role of phosphatases is less appreciated than that of kinases, and they have long been considered simply and erroneously as signal erasers. Such an over-simplified model ignores the complexities and precision of RTK signaling. To fully understand the complex picture of RTK signaling, comprehensive studies of phosphatase involvement are needed. We sought to address this question at a systems level from the perspective of RTK-phosphatase protein-protein interactions (PPIs). We previously developed the MYTH system ( Mak et al., 2012 ; Snider et al., 2013 , 2010 ; Stagljar et al., 1998 ; Thaminy et al., 2003 ) and its mammalian version, MaMTH ( Petschnigg et al., 2014 ), to study membrane PPIs. In this study, we used both systems to map the genome-wide RTK-phosphatase interactome. Our comprehensive screens identified a large number of PPIs, most of which have not been reported previously, suggesting distinct roles of individual phosphatases in RTK signaling. Furthermore, we revealed molecular details on how PTPRH, PTPRB and PTPRA help regulate EGFR signaling.

DISCUSSION

RTK signaling is a complex process in which phosphorylation and PPIs are two elementary and critical steps. The biophysical features of membrane proteins make it difficult to study the PPIs involved in RTK signaling in their natural membrane environment at the systems level. We used two complementary interaction proteomics approaches, MYTH and MaMTH ( Snider et al., 2015 ; Yao et al., 2015 ) to characterize the global RTK-phosphatase interactome. As MYTH is performed in yeast, in which mammalian proteins beyond the introduced RTK bait and phosphatase prey are absent, it can largely reduce the complexity and the bias caused by factors such as indirect PPIs, competition and antagonism of endogenous signaling molecules as well as the complex behaviors of RTKs in mammalian cells derived from recycling, degradation and feedback regulation. However, some RTKs are inactive when expressed in yeast and yeast contains a different spectrum of phosphatases, which together could cause underestimation of phosphorylation-dependent interactions. Phosphorylation-independent interactions should be less biased in a MYTH screen. By contrast, MaMTH works in virtually all mammalian cells, and therefore provides a more physiological context for PPI detection between integral membrane proteins.

Our MYTH and MaMTH screens revealed more than 300 RTK-phosphatase interactions, most of which have not been previously reported. All of the phosphatase families tested, although not each member, show some interactions, indicating their potential roles in RTK signaling. However, it should be noted that these interactions are not equivalent to enzyme-substrate interactions, and therefore do not necessarily suggest that the involved phosphatase directly dephosphorylates a given RTK or, conversely, that the RTK can phosphorylate the phosphatase. The functions of these interactions can be diverse and need further study.

We paid particular attention to PTPs, as they play critical roles in RTK signaling. Several PTPs showed high levels of interaction with the RTKs in our study. The substrate-trapping behaviors of some mutants support that they are indeed enzymes capable of dephosphorylating the interacting RTKs. However, the “non-trapping” behavior of other PTP inactive mutants suggests different modes of interaction. For example, the PTPs with stable interactions could serve other regulatory functions such as scaffolding, dephosphorylating other associated proteins and more. Moreover, several PTP mutants displayed a lower strength of interaction, suggesting a more complex mode of interaction that might involve mutual inhibition.

Although numerous studies have demonstrated that multiple PTPs play inhibitory roles in RTK signaling, several PTPs such as PTPN11 are also recognized as positive regulators ( Neel et al., 2003 ). Our global RTK-phosphatase interactome provides a basis and guidance for further characterization of the functions of interacting PTPs as exemplified by our in-depth studies. PTPRH and PTPRB are similar RPTPs belonging to subgroup “R3” ( Andersen et al., 2004 ) and exhibit similar inhibitory functions toward EGFR in our study. This similarity in function demonstrates the redundancy in the regulation of RTK signaling. Interestingly, another member of the “R3” subgroup, PTPRJ, has also been shown to be a strong inhibitory EGFR phosphatase ( Tarcic et al., 2009 ). The fact that PTPRJ/EGFR interaction was not detected in our MaMTH assay might reflect a more transient interaction. Another interesting example is given by the comparison of PTPRA and PTPRE, which are highly similar in sequence and in their capabilities to dephosphorylate and activate SRC ( Roskoski, 2005 ). However, we found that only PTPRA facilitates EGFR downstream signaling. This function correlates with PTPRA’s capability of interaction with EGFR, suggesting this interaction plays an essential role in its regulation of EGFR signaling. This example demonstrates the complexity and specificity of RTK regulation. Besides, multiple dual specificity phosphatase hits were detected in our screens. Whether these phosphatases dephosphorylate RTKs or play other roles in RTK signalling awaits future studies.

Serine/threonine phosphorylation also has important effects on RTK signaling. We consistently found numerous PSP hits in our screens, although most of their functions remain unknown. It should be noted that PPPs are typically multi-subunit proteins, composed of a catalytic subunit and scaffold/regulatory subunits that serve to determine their substrate specificities. Since only catalytic subunits were screened, PPP family hits were probably under-represented by our assay and could be improved by screening scaffolding/regulatory subunits.

In conclusion, through comprehensive investigation of the RTK-phosphatase interactome as well as initial functional studies, we provide a number of new insights into the relationships between RTKs and phosphatases. The RTK-phosphatase interactome reported here represents a rich resource for elucidation of further biological functions of RTKs and phosphatases, and might also serve as guide and a reference for the detailed characterization of individual phosphatases. Furthermore, this study demonstrates the principles of redundancy, specificity, and complexity that govern RTK signaling, and broadens our understanding of RTK signaling.
