---
pmid: '21399639'
title: IKKβ phosphorylation regulates RPS3 nuclear translocation and NF-κB function
  during infection with Escherichia coli strain O157:H7.
authors:
- Wan F
- Weaver A
- Gao X
- Bern M
- Hardwidge PR
- Lenardo MJ
journal: Nat Immunol
year: '2011'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC3062687
doi: 10.1038/ni.2007
---

# IKKβ phosphorylation regulates RPS3 nuclear translocation and NF-κB function during infection with Escherichia coli strain O157:H7.
**Authors:** Wan F, Weaver A, Gao X, Bern M, Hardwidge PR, Lenardo MJ
**Journal:** Nat Immunol (2011)
**DOI:** [10.1038/ni.2007](https://doi.org/10.1038/ni.2007)
**PMC:** [PMC3062687](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3062687/)

## Abstract

1. Nat Immunol. 2011 Apr;12(4):335-43. doi: 10.1038/ni.2007. Epub 2011 Mar 13.

IKKβ phosphorylation regulates RPS3 nuclear translocation and NF-κB function 
during infection with Escherichia coli strain O157:H7.

Wan F(1), Weaver A, Gao X, Bern M, Hardwidge PR, Lenardo MJ.

Author information:
(1)Laboratory of Immunology, National Institute of Allergy and Infectious 
Diseases, National Institutes of Health, Bethesda, Maryland, USA.

NF-κB is a major gene regulator in immune responses, and ribosomal protein S3 
(RPS3) is an NF-κB subunit that directs specific gene transcription. However, it 
is unknown how nuclear translocation of RPS3 is regulated. Here we report that 
phosphorylation of RPS3 Ser209 by the kinase IKKβ was crucial for nuclear 
localization of RPS3 in response to activating stimuli. Moreover, virulence 
protein NleH1 of the foodborne pathogen Escherichia coli strain O157:H7 
specifically inhibited phosphorylation of RPS3 Ser209 and blocked RPS3 function, 
thereby promoting bacterial colonization and diarrhea but resulting in less 
mortality in a gnotobiotic piglet-infection model. Thus, the IKKβ-dependent 
modification of a specific amino acid in RPS3 promoted specific NF-κB functions 
that underlie the molecular pathogenetic mechanisms of E. coli O157:H7.

DOI: 10.1038/ni.2007
PMCID: PMC3062687
PMID: 21399639 [Indexed for MEDLINE]

## Full Text

Nuclear Factor-kappa B (NF-κB) regulates crucial cellular functions and diverse stimuli activate this pleiotropic transcription factor, which in turn regulates a vast array of genetic targets1-3. The best-known mammalian NF-κB subunits are Rel proteins, including RelA (p65), RelB, c-Rel, p50, and p52 (refs. 4, 5). However, we recently demonstrated that ribosomal protein S3 (RPS3) is a key non-Rel subunit of certain native NF-κB complexes6. RPS3 is defined as a “specifier” subunit of NF-κB, because it facilitates high affinity DNA binding thus determining the regulatory specificity of NF-κB for selected target genes7. RPS3 regulation of NF-κB governs key physiological processes, including immunoglobulin κ light chain gene expression and receptor editing in B cells6, 8, cytokine production in T cells6, and in host defense against enterohemorrhagic Escherichia coli (EHEC)9. In particular, the E. coli O157:H7 type III secretion system (T3SS) effector protein NleH1 selectively blocks NF-κB target gene transcription by attenuating RPS3 nuclear translocation, without affecting p65 localization9. Nonetheless, how specific NF-κB activating signals induce RPS3 nuclear translocation is unknown.

Extra-ribosomal functions have been ascribed to ribosomal proteins10. Besides binding RNA within the 40S ribosomal subunit, RPS3 participates in transcription6, DNA repair11, 12, and apoptosis13. Whether or not RPS3 is phosphorylated had been controversial14-18. Since kinase cascades play a critical role in NF-κB regulation, we tested whether RPS3 is phosphorylated in the context of NF-κB activation and sought to identify the responsible kinase19. Additionally, we aimed to define a regulatory role for the C-terminal tail of RPS3 whose function was unknown.

Here we show that the Inhibitor of κB (IκB) kinase beta (IKKβ) phosphorylated RPS3 at serine 209 (S209). RPS3 S209 phosphorylation enhanced its association with importin-α, mediating RPS3 entry into the karyopherin pathway for nuclear translocation. Furthermore, the E. coli NleH1 effector specifically inhibited RPS3 S209 revealing how E. coli O157:H7 inhibits this important innate immune response mechanism.

ResultsRPS3 phosphorylation in response to NF-κB activationTo test whether RPS3 is phosphorylated during NF-κB activation, we performed 32P-labeling experiments in tumor necrosis factor (TNF)-stimulated HEK 293T cells. While RPS3 was scarcely phosphorylated in unstimulated cells, we observed a marked increase in 32P-incorporation after TNF stimulation despite no increase in RPS3 protein (Fig. 1a). To determine which RPS3 residues were phosphorylated, we immunoprecipitated RPS3 from either resting or stimulated cells and performed immunoblotting with phosphorylation-specific antibodies. Both TNF and phorbol myristate acetate/ionomycin (PMA+I) stimulated rapid phosphorylation and degradaion of IκBα within 5 min which was accompanied by RPS3 phosphorylation on serine residues (Fig. 1b
and data not shown), similar to the in vivo labeling. We did not detect tyrosine- or threonine-phosphorylation of RPS3 (Fig. 1b).RPS3 and IKKβ interactionThe activation of the inhibitor of κB kinase (IKK), consisting of a regulatory subunit IKKγ and two catalytic subunits, IKKα and IKKβ, is critical for the phosphorylation and dispatch of the inhibitory IκBs and the liberation of NF-κB20-22. Given that RPS3 can be found in the cytoplasmic p65-p50-IκBα inhibitory complex in resting cells6, we hypothesized that activated IKKβ might also bind to and phosphorylate RPS3. First, we found that ectopically expressed IKKβ and RPS3 interacted (Fig. 1c). We next examined resting Jurkat cells and detected a modest endogenous IKKβ-RPS3 interaction (Fig. 1d), potentially accounting for the basal NF-κB transcription required for cell proliferation and survival. RPS3-IKKβ association was clearly augmented upon TNF stimulation, peaking at 10 min. (Fig. 1d), following similar kinetics to RPS3 serine phosphorylation (Fig. 1b). By contrast, there was no detectable interaction between RPS3 and IKKα (Fig. 1d).IKKβ is required for RPS3 nuclear translocationTo examine whether the RPS3-IKKβ interaction is required for RPS3 nuclear translocation, we knocked down IKKα or IKKβ expression with siRNAs (Supplementary Fig. 1) and then observed stimulation-induced RPS3 nuclear migration by confocal microscopy. Both TNF and PMA+I triggered RPS3 nuclear translocation in Jurkat cells transfected with a scrambled nonspecific (NS) siRNA (Fig. 2a)6. RPS3 nuclear translocation was only slightly, if at all, impaired by IKKα-silencing. Conversely, knockdown of IKKβ attenuated 60-70% of RPS3 nuclear accumulation following stimulation (Fig. 2a). Immunoblotting of nuclear fractions confirmed that full expression of IKKβ, but not IKKα, was necessary for activation-induced RPS3 nuclear translocation (Fig. 2b). Control immunoblots revealed that p65 nuclear translocation was blocked under the same conditions (Fig. 2b).We next examined the nuclear translocation of RPS3 in cells ectopically expressing either kinase-dead (SSAA) or constitutively-active (SSEE) mutant IKKβ proteins. As expected, the SSEE, but not SSAA, mutant of IKKβ induced NF-κB-dependent luciferase reporter activity (Fig. 2c, left). Whereas RPS3 remained cytosolic in IKKβ (SSAA)-expressing cells (Fig. 2c, right), a substantial proportion of RPS3 translocated to the nucleus in cells expressing IKKβ (SSEE) (Fig. 2c, right). The percentage of cells containing detectable nuclear RPS3 increased 5-fold in IKKβ (SSEE)-expressing cells, but not in IKKβ (SSAA)-expressing ones (Fig. 2d and Supplementary Fig. 2). Thus, IKKβ activity is necessary and sufficient for RPS3 nuclear translocation in response to NF-κB activating stimuli.IκBα degradation and RPS3 nuclear translocationImportin-α regulates the nuclear import of NF-κB Rel subunits23, 24. RPS3 harbors a nuclear localization signal (NLS) sequence and its nuclear translocation occurs in parallel to, but independently of, p65 translocation6. We envisioned that RPS3 could also utilize the importin-α/β pathway. Consistent with this notion, RPS3 association with importin-α, but not importin-β, was enhanced in TNF–stimulated cells (Fig. 3a). Therefore, we examined whether RPS3 binding to importin-α is essential for nuclear translocation during NF-κB activation.Since IκBα degradation is a prerequisite to unmask the NLS of p65, and both RPS3 and IκBα bind to p65 in the cytoplasmic inhibitory complex, we tested whether IκBα degradation is required for the liberation of RPS3. We measured the association of RPS3 with importin-α in 293T cells overexpressing wild-type IκBα or an IκBα mutant (SSAA) resistant to IKKβ-induced phosphorylation and degradation. In cells transfected with wild-type IκBα, TNF stimulation augmented the interaction of RPS3 and importin-α to a similar degree as in non-transfected cells. By contrast, we observed that the RPS3-importin-α association was abolished by the presence of non-degradable IκBα (Fig. 3b).To examine whether IκBα is the only cytoplasmic barrier precluding RPS3 nuclear translocation, we measured both RPS3-importin-α association and nuclear RPS3 after reducing IκBα expression. Compared with nonspecific siRNA, siRNA targeting of IκBα completely depleted IκBα in Jurkat cells (Fig. 3c, input). Nevertheless, the RPS3-importin-α association was not augmented (Fig. 3c), nor was significant nuclear RPS3 detected (Fig. 3d). Moreover, cells treated with sodium pervanadate (Pv) to induce IκBα degradation through an IKK-independent mechanism25-27 did not show increased association between RPS3 and importin-α (Fig. 3e and Supplementary Fig. 3b) or nuclear accumulation of RPS3, despite complete IκBα degradation (Supplementary Fig. 3c). We further examined whether a subsequent NF-κB activation signal independently promotes the importin-α association and nuclear transport of RPS3 after IκBα degradation. We found that TNF stimulation following Pv treatment was required for the RPS3-importin-α association, comparable to TNF stimulation alone (Fig. 3e). Thus, IκBα phosphorylation and degradation itself is required but not sufficient to cause RPS3 association with importin-α followed by nuclear translocation. Rather, an additional signal, potentially IKKβ phosphorylation of RPS3, is required.IKKβ phosphorylates RPS3 at serine 209Although originally defined as the kinase that phosphorylates IκB19, IKKβ also phosphorylates unrelated substrates including 14-3-3β and Bcl10, which lack the IKK consensus motif (DpSGYXpS/T)28. We therefore hypothesized that IKKβ could directly phosphorylate RPS3. By in vitro kinase assays using recombinant IKK and RPS3 proteins, we observed strong incorporation of 32P in autophosphorylatd IKKα and IKKβ (Fig. 4a, lanes 2-7) as well as phosporylated GST-IκBα (1-54) (Supplementary Fig. 4), but not the GST protein alone (Fig. 4a, lanes 3 and 6), when either IKKα or IKKβ was used. We discovered that GST-RPS3 could be phosphorylated by IKKβ, but not IKKα, in vitro (Fig. 4a, compare lanes 4 and 7).To identify the RPS3 amino acid residue(s) phosphorylated by IKKβ, we performed liquid chromatography-tandem mass spectrometry analyses using in vitro phosphorylated RPS3. The results indicated that IKKβ phosphorylated S209, located in the RPS3 C-terminus (Fig. 4b). RPS3 amino acid sequence alignment revealed that S209 is conserved in many species throughout phylogeny with the exception of Caenorhabditis elegans and Schizosaccharomyces pombe, two organisms that do not possess the NF-κB signal pathway (Supplementary Fig. 5).To verify biochemically that S209 is an IKKβ substrate, we performed 32P-labeling in vitro kinase assays with recombinant wild-type or S209A mutant RPS3 proteins. Compared with the wild-type protein, the S209A mutation reduced IKKβ–mediated RPS3 phosphorylation (Fig. 4c). There might be alternative phosphorylation site(s) under these conditions given modest residual phosphorylated RPS3 (Fig. 4c). RPS3 S209 does not fall within a conventional IKK recognition motif, but rather resides in a sequence motif (XXXpS/TXXE), potentially recognized by casein kinase II (CK2). Although IKKβ kinase can display a CK2-like phosphorylation specificity29, no CK2 protein was detectable in our recombinant IKK proteins (Supplementary Fig. 6). Thus, RPS3 S209 phosphorylation was due to the alternate specificity of the IKKβ kinase rather than any trace amount of CK2 bound to IKKs. To determine whether S209 is the critical site at which IKKβ phosphorylates RPS3 in living cells, we transfected the wild-type or S209A mutant Flag-RPS3 alone, or together with IKKβ into cells. Indeed, we observed that overexpressing IKKβ enhanced Flag-RPS3 phosphorylation, but phosphorylation was effectively eliminated by alanine substitution indicating that S209 is the predominant target site for IKKβ phosphorylation (Fig. 4d). We next generated a phospho-S209 RPS3 antibody and confirmed that endogenous RPS3 was phosphorylated at S209 in a time-dependent manner upon TNF stimulation (Fig. 4e). Thus, the RPS3 C-terminal tail potentially contains an important regulatory site.Phosphorylation of RPS3 and its NF-κB functionWe next examined whether S209 phosphorylation plays a role in the nuclear translocation of RPS3 during NF-κB activation. Subcellular fractions from either wild-type or S209A mutant RPS3-transfected cells were prepared and blotted for heat-shock protein 90 (hsp90), a cytoplasmic protein, and poly (ADP-ribose) polymerase (PARP), a nuclear protein, confirming a clean separation (Fig. 5a). As expected, PMA+I stimulation triggered wild-type Flag-RPS3 nuclear translocation (Fig. 5a). However, RPS3 (S209A) nuclear translocation was attenuated (Fig. 5a). We also tested the impact of activating NF-κB by overexpressing IKKβ on RPS3 nuclear translocation. IKKβ overexpression activated NF-κB measured by luciferase assays (Supplementary Fig. 7), and also induced the nuclear translocation of wild-type, but not S209A, RPS3 (Fig. 5b). These data suggest that S209 phosphorylation is critical for the NF-κB activation-induced RPS3 nuclear translocation.To examine the role of S209 phosphorylation of RPS3 to its NF-κB function6, 7, 30, we silenced endogenous RPS3 expression using an siRNA that targets the 3′ untranslated region (3′ UTR) of RPS3 mRNA, followed by complementation with either wild-type or S209A mutant RPS3 via transfection. As expected, RPS3 siRNA severely reduced endogenous RPS3 abundance compared to NS siRNA, but did not affect the robust expression of Flag-tagged RPS3 from a transfected construct lacking the 3′ UTR (Fig. 5c). We also found that RPS3 knockdown reduced TNF-induced expression of an Ig κB-driven luciferase construct6 (Fig. 5d). The impaired luciferase signal caused by RPS3 deficiency was completely restored by transfecting wild-type, but not by S209A RPS3 (Fig. 5d), despite equivalent expression (Fig. 5c). Moreover, the failure of S209A RPS3 to restore luciferase activity did not result from defective translation because the transient overexpression of green fluorescent protein (GFP) was comparable in cells complemented with wild-type or S029A RPS3 (Supplementary Fig. 8). Taken together, these data suggest that RPS3 S209 phosphorylation is critical for NF-κB activity involving the canonical Ig κB site.We next used chromatin immunoprecipitation to determine whether S209 phosphorylation affects RPS3 and p65 recruitment to specific κB sites in intact chromatin during NF-κB activation. In RPS3 knockdown cells, PMA+I stimulated the recruitment of ectopically expressed, Flag-tagged wild-type, but not S209A RPS3 to the κB sites of the NFKBIA and IL8 promoters (Fig. 5e). While expressing RPS3 S209A had no impact on p65 nuclear translocation, it substantially attenuated p65 recruitment (Fig. 5e). Additional experiments revealed that p65 attraction to RPS3-independent NF-κB target gene promoters such as CD25 was increased (Supplementary Fig. 9), consistent with our previous observations6. There was no significant Flag-RPS3 or p65 recruitment to ACTB promoter lacking κB sites (Fig. 5e), suggesting the recruitment was κB site-specific. Thus, the recruitment of RPS3 as well as the contingent recruitment of p65 to key promoters depended on S209.Interleukin 8 (IL-8) secretion induced by either T cell receptor (TCR) agonist stimulation or PMA+I was decreased as a consequence of reduced RPS3/p65 recruitment to the IL8 κB sites in the presence of S209A mutant compared to wild-type RPS3 (Supplementary Fig. 10). However, cell surface CD25 expression was comparable between the wild-type and S209A RPS3 transfected cells (Supplementary Fig. 11). Therefore, RPS3 S209 phosphorylation by IKKβ is apparently required for RPS3 in directing NF-κB to a specific subset of target genes.NleH1 inhibits RPS3 phosphorylation in vitroEHEC pathogens are important causative agents of both foodborne disease and pediatric renal failure31. EHEC utilize T3SS to inject effector proteins directly into intestinal epithelial cells32, a subset of which inhibit NF-κB-dependent innate responses9, 33-38. The E. coli O157:H7 EDL933 effector protein NleH1 binds to and attenuates RPS3 nuclear translocation, thus impairing RPS3-dependent NF-κB signaling9. We therefore hypothesized that NleH1 may function by inhibiting RPS3 S209 phosphorylation. As expected, transfecting increasing amounts of NleH1-HA plasmid blocked TNFα-induced NF-κB activation in a dose-dependent manner (Fig. 6a-b)9. Remarkably, NleH1 reduced both TNF-induced, as well as basal RPS3 phosphorylation to roughly 20% of vehicle control (Fig. 6c). Expressing NleH1 does not interfere with either TNF-induced IKK activation or IκBα degradation, consistent with the lack of NleH1 impact on p65 nuclear translocation9 (Fig. 6c).To determine if NleH1 inhibits RPS3 phosphorylation, we infected HeLa cells with E. coli O157:H7 strains possessing or lacking either nleH1 (ΔnleH1) or with a strain lacking a functional T3SS unable to inject NleH1 into mammalian cells (ΔescN). In uninfected cells, TNF-treatment stimulated a ∼7-fold increase in RPS3 S209 phosphorylation, peaking at 30 minutes (Fig. 6d). By contrast, RPS3 S209 phosphorylation was substantially impaired in cells infected with wild-type E. coli O157:H7 (Fig. 6d). However, TNF-induced RPS3 S209 phosphorylation was unimpaired in cells infected with either ΔnleH1 or ΔescN (Fig. 6d). We showed previously that wild-type, but not ΔnleH1 or ΔescN E. coli O157:H7 significantly attenuated TNF-induced RPS3 nuclear translocation9. The parallel between RPS3 phosphorylation and its nuclear translocation during E. coli infection provides evidence in the context of an NF-κB-dependent disease process that RPS3 S209 phosphorylation is important for nuclear translocation.Our discovery that NleH1 inhibits RPS3 S209 phosphorylation suggested that it should also blocks RPS3-dependent NF-κB target gene transcription (e.g. IL8, NFKBIA, and TNFAIP3). Indeed, these genes were only modestly upregulated in cells infected with wild-type E. coli O157:H7, but significantly induced in cells infected with either ΔnleH1 or ΔescN strains (Fig. 6e). In contrast, deleting nleH1 had no impact on the expression of RPS3-independent genes, including CD25 and TNFSF13B (Supplementary Fig. 12). Together these results demonstrate that NleH1 specifically inhibits the protective immune response by directly blocking RPS3 S209 phosphorylation and thereby impairing critical RPS3-dependent NF-κB target genes.NleH1 inhibits RPS3 S209 phosphorylation in vivoWe previously utilized a gnotobiotic piglet infection model to determine that piglets infected with ΔnleH1 mutant died more rapidly than those infected with wild-type E. coli O157:H7. Piglets infected with ΔnleH1 displayed clinical disease consistent with a robust inflammatory response, but with reduced bacterial colonization and little diarrhea9. While seemingly paradoxical, based on our cell culture data (Fig. 6d), we hypothesized that NleH1 blocked RPS3 S209 phosphorylation in vivo, thereby preventing RPS3 nuclear translocation in infected piglets. We isolated piglet colons at necropsy, and subjected them to immunohistochemistry using our phospho-RPS3 antibody. Consistent with in vitro data, piglets infected with wild-type E. coli O157:H7 exhibited diffuse and low intensity phospho-RPS3 staining, whereas in piglets infected with ΔnleH1 mutant, phospho-RPS3 expression was florid and intense (Fig. 6f). These data demonstrate that NleH1 inhibits RPS3 S209 phosphorylation both in vitro and in vivo, which might benefit the bacterium in colonization and transmission.NleH1 steers the IKKβ substrate specificitiesNleH1 is an autophosphorylated serine-threonine kinase, which depends on the lysine 159 (K159)9. To explore the mechanism by which NleH1 inhibits RPS3 S209 phosphorylation, we first performed an in vitro kinase assay with purified wild-type His-NleH1 protein and a mutant His-NleH1 (K159A) protein, confirming that NleH1 is autophosphorylated and the K159A is an NleH1 kinase-dead mutant (Fig. 7a). To examine whether the kinase activity is required for NleH1 to inhibit IKKβ phosphorlyation of RPS3 on S209, we ectopically expressing either wild-type or K159A NleH1 in 293T cells. Wild-type NleH1 expression significantly reduced TNF-induced RPS3 S209 phosphorylation, whereas the K159A mutant failed to do so (Fig. 7b). Thus NleH1 kinase activity is required to protect RPS3 from IKKβ-mediated phosphorylation.Citrobacter rodentium is a mouse pathogen that shares pathogenic strategies with E. coli
39, most notably for our investigation, C. rodentium NleH inhibited RPS3 nuclear translocation and RPS3-dependent NF-κB luciferase activity to an extent equivalent to E. coli NleH1 (ref. 9). We assayed RPS3 S209 phosphorylation in HeLa cells infected with different C. rodentium strains. In uninfected cells, TNF-treatment stimulated a ∼3.5-fold increase in RPS3 S209 phosphorylation (Fig. 7c). Such augmentation of RPS3 phosphorylation was reduced by about 60% by wild-type C. rodentium infection (Fig. 7c). However, RPS3 phosphorylation was enhanced when cells were infected with a C. rodentium strain lacking NleH (Fig. 7c, ΔnleH). We further examined the role of NleH1 kinase activity using the C. rodentium ΔnleH strain as a background on which to express either wild-type or K159A E. coli NleH1. Complementing ΔnleH mutant with wild-type NleH1 almost abolished TNF-induced RPS3 S209 phosphorylation whereas complementing with K159A failed to inhibit RPS3 phosphorylation (Fig. 7c). Collectively, these results demonstrate that NleH1 kinase activity is required to block RPS3 S209 phosphorylation.We next examined whether the inhibitory activity of NleH1 is sufficiently robust to impair the strong nuclear translocation of RPS3 trigged by the constitutively-active IKKβ (IKKβ [SSEE]) (Fig. 2d). We found that ectopically expressing either wild-type or SSEE IKKβ proteins, triggered more RPS3 nuclear translocation than the kinase-dead IKKβ (SSAA) protein (Fig. 7d). RPS3 nuclear accumulation was substantially retarded by infecting cells with wild-type E. coli O157:H7 (Fig. 7d). In contrast, infecting with either ΔnleH1 or ΔescN strains only slightly impaired RPS3 nuclear translocation in either IKKβ- or IKKβ (SSEE)-expressing cells (Fig. 7d). As expected, E. coli infections did not affect the RPS3 nuclear translocation in IKKβ (SSAA)-expressing cells, where NF-κB signaling was low (Fig. 7d). Thus, during infection NleH1 is sufficiently potent to inhibit RPS3 nuclear translocation even in cells expressing constitutively-activated IKKβ.We examined whether NleH1 could directly phosphorylate IKKβ thus inhibiting IKKβ-mediated RPS3 S209 phosphorylation. We performed in vitro kinase assays using immunoprecipitated Flag-IKKβ (K44A) as substrate and recombinant His-NleH1 as kinase, so that IKKβ autophosphorylation would not obscure NleH1-induced phosphorylation. However, we did not observe any detectable 32P incorporation in IKKβ (Supplementary Fig. 13), thus ruling out this possibility.We then tested the hypothesis that NleH1 could alter the IKKβ substrate specificities. To this end, we performed in vitro kinase assays using both CK2 and IKK substrates for IKKβ. As expected, IKKβ phosphorylated RPS3 (Fig. 7e, lane 7) and GST-IκBα (1-54) protein (Fig. 7e, lane 9), demonstrating it harbors either CK2 or IKK substrate specificity. Preincubation of IKKβ with NleH1 reduced IKKβ-mediated RPS3 phosphorylation, i.e. the CK2 kinase specificity, but not IKKβ-mediated GST-IκBα phosphorylation, i.e. the IKK kinase specificity (Fig. 7e). Control experiments revealed no NleH1-mediated phosphorylation or autophosphorylation of RPS3 or GST-IκBα (Fig. 7e). Taken together, NleH1 blocks the CK2 substrate specificity of IKKβ thus inhibiting the IKKβ-mediated RPS3 S209 phosphorylation thus representing a novel strategy by E. coli O157:H7 to alter the host innate immune response.

RPS3 phosphorylation in response to NF-κB activationTo test whether RPS3 is phosphorylated during NF-κB activation, we performed 32P-labeling experiments in tumor necrosis factor (TNF)-stimulated HEK 293T cells. While RPS3 was scarcely phosphorylated in unstimulated cells, we observed a marked increase in 32P-incorporation after TNF stimulation despite no increase in RPS3 protein (Fig. 1a). To determine which RPS3 residues were phosphorylated, we immunoprecipitated RPS3 from either resting or stimulated cells and performed immunoblotting with phosphorylation-specific antibodies. Both TNF and phorbol myristate acetate/ionomycin (PMA+I) stimulated rapid phosphorylation and degradaion of IκBα within 5 min which was accompanied by RPS3 phosphorylation on serine residues (Fig. 1b
and data not shown), similar to the in vivo labeling. We did not detect tyrosine- or threonine-phosphorylation of RPS3 (Fig. 1b).

RPS3 phosphorylation in response to NF-κB activation

To test whether RPS3 is phosphorylated during NF-κB activation, we performed 32P-labeling experiments in tumor necrosis factor (TNF)-stimulated HEK 293T cells. While RPS3 was scarcely phosphorylated in unstimulated cells, we observed a marked increase in 32P-incorporation after TNF stimulation despite no increase in RPS3 protein (Fig. 1a). To determine which RPS3 residues were phosphorylated, we immunoprecipitated RPS3 from either resting or stimulated cells and performed immunoblotting with phosphorylation-specific antibodies. Both TNF and phorbol myristate acetate/ionomycin (PMA+I) stimulated rapid phosphorylation and degradaion of IκBα within 5 min which was accompanied by RPS3 phosphorylation on serine residues (Fig. 1b
and data not shown), similar to the in vivo labeling. We did not detect tyrosine- or threonine-phosphorylation of RPS3 (Fig. 1b).

RPS3 and IKKβ interactionThe activation of the inhibitor of κB kinase (IKK), consisting of a regulatory subunit IKKγ and two catalytic subunits, IKKα and IKKβ, is critical for the phosphorylation and dispatch of the inhibitory IκBs and the liberation of NF-κB20-22. Given that RPS3 can be found in the cytoplasmic p65-p50-IκBα inhibitory complex in resting cells6, we hypothesized that activated IKKβ might also bind to and phosphorylate RPS3. First, we found that ectopically expressed IKKβ and RPS3 interacted (Fig. 1c). We next examined resting Jurkat cells and detected a modest endogenous IKKβ-RPS3 interaction (Fig. 1d), potentially accounting for the basal NF-κB transcription required for cell proliferation and survival. RPS3-IKKβ association was clearly augmented upon TNF stimulation, peaking at 10 min. (Fig. 1d), following similar kinetics to RPS3 serine phosphorylation (Fig. 1b). By contrast, there was no detectable interaction between RPS3 and IKKα (Fig. 1d).

RPS3 and IKKβ interaction

The activation of the inhibitor of κB kinase (IKK), consisting of a regulatory subunit IKKγ and two catalytic subunits, IKKα and IKKβ, is critical for the phosphorylation and dispatch of the inhibitory IκBs and the liberation of NF-κB20-22. Given that RPS3 can be found in the cytoplasmic p65-p50-IκBα inhibitory complex in resting cells6, we hypothesized that activated IKKβ might also bind to and phosphorylate RPS3. First, we found that ectopically expressed IKKβ and RPS3 interacted (Fig. 1c). We next examined resting Jurkat cells and detected a modest endogenous IKKβ-RPS3 interaction (Fig. 1d), potentially accounting for the basal NF-κB transcription required for cell proliferation and survival. RPS3-IKKβ association was clearly augmented upon TNF stimulation, peaking at 10 min. (Fig. 1d), following similar kinetics to RPS3 serine phosphorylation (Fig. 1b). By contrast, there was no detectable interaction between RPS3 and IKKα (Fig. 1d).

IKKβ is required for RPS3 nuclear translocationTo examine whether the RPS3-IKKβ interaction is required for RPS3 nuclear translocation, we knocked down IKKα or IKKβ expression with siRNAs (Supplementary Fig. 1) and then observed stimulation-induced RPS3 nuclear migration by confocal microscopy. Both TNF and PMA+I triggered RPS3 nuclear translocation in Jurkat cells transfected with a scrambled nonspecific (NS) siRNA (Fig. 2a)6. RPS3 nuclear translocation was only slightly, if at all, impaired by IKKα-silencing. Conversely, knockdown of IKKβ attenuated 60-70% of RPS3 nuclear accumulation following stimulation (Fig. 2a). Immunoblotting of nuclear fractions confirmed that full expression of IKKβ, but not IKKα, was necessary for activation-induced RPS3 nuclear translocation (Fig. 2b). Control immunoblots revealed that p65 nuclear translocation was blocked under the same conditions (Fig. 2b).We next examined the nuclear translocation of RPS3 in cells ectopically expressing either kinase-dead (SSAA) or constitutively-active (SSEE) mutant IKKβ proteins. As expected, the SSEE, but not SSAA, mutant of IKKβ induced NF-κB-dependent luciferase reporter activity (Fig. 2c, left). Whereas RPS3 remained cytosolic in IKKβ (SSAA)-expressing cells (Fig. 2c, right), a substantial proportion of RPS3 translocated to the nucleus in cells expressing IKKβ (SSEE) (Fig. 2c, right). The percentage of cells containing detectable nuclear RPS3 increased 5-fold in IKKβ (SSEE)-expressing cells, but not in IKKβ (SSAA)-expressing ones (Fig. 2d and Supplementary Fig. 2). Thus, IKKβ activity is necessary and sufficient for RPS3 nuclear translocation in response to NF-κB activating stimuli.

IKKβ is required for RPS3 nuclear translocation

To examine whether the RPS3-IKKβ interaction is required for RPS3 nuclear translocation, we knocked down IKKα or IKKβ expression with siRNAs (Supplementary Fig. 1) and then observed stimulation-induced RPS3 nuclear migration by confocal microscopy. Both TNF and PMA+I triggered RPS3 nuclear translocation in Jurkat cells transfected with a scrambled nonspecific (NS) siRNA (Fig. 2a)6. RPS3 nuclear translocation was only slightly, if at all, impaired by IKKα-silencing. Conversely, knockdown of IKKβ attenuated 60-70% of RPS3 nuclear accumulation following stimulation (Fig. 2a). Immunoblotting of nuclear fractions confirmed that full expression of IKKβ, but not IKKα, was necessary for activation-induced RPS3 nuclear translocation (Fig. 2b). Control immunoblots revealed that p65 nuclear translocation was blocked under the same conditions (Fig. 2b).

We next examined the nuclear translocation of RPS3 in cells ectopically expressing either kinase-dead (SSAA) or constitutively-active (SSEE) mutant IKKβ proteins. As expected, the SSEE, but not SSAA, mutant of IKKβ induced NF-κB-dependent luciferase reporter activity (Fig. 2c, left). Whereas RPS3 remained cytosolic in IKKβ (SSAA)-expressing cells (Fig. 2c, right), a substantial proportion of RPS3 translocated to the nucleus in cells expressing IKKβ (SSEE) (Fig. 2c, right). The percentage of cells containing detectable nuclear RPS3 increased 5-fold in IKKβ (SSEE)-expressing cells, but not in IKKβ (SSAA)-expressing ones (Fig. 2d and Supplementary Fig. 2). Thus, IKKβ activity is necessary and sufficient for RPS3 nuclear translocation in response to NF-κB activating stimuli.

IκBα degradation and RPS3 nuclear translocationImportin-α regulates the nuclear import of NF-κB Rel subunits23, 24. RPS3 harbors a nuclear localization signal (NLS) sequence and its nuclear translocation occurs in parallel to, but independently of, p65 translocation6. We envisioned that RPS3 could also utilize the importin-α/β pathway. Consistent with this notion, RPS3 association with importin-α, but not importin-β, was enhanced in TNF–stimulated cells (Fig. 3a). Therefore, we examined whether RPS3 binding to importin-α is essential for nuclear translocation during NF-κB activation.Since IκBα degradation is a prerequisite to unmask the NLS of p65, and both RPS3 and IκBα bind to p65 in the cytoplasmic inhibitory complex, we tested whether IκBα degradation is required for the liberation of RPS3. We measured the association of RPS3 with importin-α in 293T cells overexpressing wild-type IκBα or an IκBα mutant (SSAA) resistant to IKKβ-induced phosphorylation and degradation. In cells transfected with wild-type IκBα, TNF stimulation augmented the interaction of RPS3 and importin-α to a similar degree as in non-transfected cells. By contrast, we observed that the RPS3-importin-α association was abolished by the presence of non-degradable IκBα (Fig. 3b).To examine whether IκBα is the only cytoplasmic barrier precluding RPS3 nuclear translocation, we measured both RPS3-importin-α association and nuclear RPS3 after reducing IκBα expression. Compared with nonspecific siRNA, siRNA targeting of IκBα completely depleted IκBα in Jurkat cells (Fig. 3c, input). Nevertheless, the RPS3-importin-α association was not augmented (Fig. 3c), nor was significant nuclear RPS3 detected (Fig. 3d). Moreover, cells treated with sodium pervanadate (Pv) to induce IκBα degradation through an IKK-independent mechanism25-27 did not show increased association between RPS3 and importin-α (Fig. 3e and Supplementary Fig. 3b) or nuclear accumulation of RPS3, despite complete IκBα degradation (Supplementary Fig. 3c). We further examined whether a subsequent NF-κB activation signal independently promotes the importin-α association and nuclear transport of RPS3 after IκBα degradation. We found that TNF stimulation following Pv treatment was required for the RPS3-importin-α association, comparable to TNF stimulation alone (Fig. 3e). Thus, IκBα phosphorylation and degradation itself is required but not sufficient to cause RPS3 association with importin-α followed by nuclear translocation. Rather, an additional signal, potentially IKKβ phosphorylation of RPS3, is required.

IκBα degradation and RPS3 nuclear translocation

Importin-α regulates the nuclear import of NF-κB Rel subunits23, 24. RPS3 harbors a nuclear localization signal (NLS) sequence and its nuclear translocation occurs in parallel to, but independently of, p65 translocation6. We envisioned that RPS3 could also utilize the importin-α/β pathway. Consistent with this notion, RPS3 association with importin-α, but not importin-β, was enhanced in TNF–stimulated cells (Fig. 3a). Therefore, we examined whether RPS3 binding to importin-α is essential for nuclear translocation during NF-κB activation.

Since IκBα degradation is a prerequisite to unmask the NLS of p65, and both RPS3 and IκBα bind to p65 in the cytoplasmic inhibitory complex, we tested whether IκBα degradation is required for the liberation of RPS3. We measured the association of RPS3 with importin-α in 293T cells overexpressing wild-type IκBα or an IκBα mutant (SSAA) resistant to IKKβ-induced phosphorylation and degradation. In cells transfected with wild-type IκBα, TNF stimulation augmented the interaction of RPS3 and importin-α to a similar degree as in non-transfected cells. By contrast, we observed that the RPS3-importin-α association was abolished by the presence of non-degradable IκBα (Fig. 3b).

To examine whether IκBα is the only cytoplasmic barrier precluding RPS3 nuclear translocation, we measured both RPS3-importin-α association and nuclear RPS3 after reducing IκBα expression. Compared with nonspecific siRNA, siRNA targeting of IκBα completely depleted IκBα in Jurkat cells (Fig. 3c, input). Nevertheless, the RPS3-importin-α association was not augmented (Fig. 3c), nor was significant nuclear RPS3 detected (Fig. 3d). Moreover, cells treated with sodium pervanadate (Pv) to induce IκBα degradation through an IKK-independent mechanism25-27 did not show increased association between RPS3 and importin-α (Fig. 3e and Supplementary Fig. 3b) or nuclear accumulation of RPS3, despite complete IκBα degradation (Supplementary Fig. 3c). We further examined whether a subsequent NF-κB activation signal independently promotes the importin-α association and nuclear transport of RPS3 after IκBα degradation. We found that TNF stimulation following Pv treatment was required for the RPS3-importin-α association, comparable to TNF stimulation alone (Fig. 3e). Thus, IκBα phosphorylation and degradation itself is required but not sufficient to cause RPS3 association with importin-α followed by nuclear translocation. Rather, an additional signal, potentially IKKβ phosphorylation of RPS3, is required.

IKKβ phosphorylates RPS3 at serine 209Although originally defined as the kinase that phosphorylates IκB19, IKKβ also phosphorylates unrelated substrates including 14-3-3β and Bcl10, which lack the IKK consensus motif (DpSGYXpS/T)28. We therefore hypothesized that IKKβ could directly phosphorylate RPS3. By in vitro kinase assays using recombinant IKK and RPS3 proteins, we observed strong incorporation of 32P in autophosphorylatd IKKα and IKKβ (Fig. 4a, lanes 2-7) as well as phosporylated GST-IκBα (1-54) (Supplementary Fig. 4), but not the GST protein alone (Fig. 4a, lanes 3 and 6), when either IKKα or IKKβ was used. We discovered that GST-RPS3 could be phosphorylated by IKKβ, but not IKKα, in vitro (Fig. 4a, compare lanes 4 and 7).To identify the RPS3 amino acid residue(s) phosphorylated by IKKβ, we performed liquid chromatography-tandem mass spectrometry analyses using in vitro phosphorylated RPS3. The results indicated that IKKβ phosphorylated S209, located in the RPS3 C-terminus (Fig. 4b). RPS3 amino acid sequence alignment revealed that S209 is conserved in many species throughout phylogeny with the exception of Caenorhabditis elegans and Schizosaccharomyces pombe, two organisms that do not possess the NF-κB signal pathway (Supplementary Fig. 5).To verify biochemically that S209 is an IKKβ substrate, we performed 32P-labeling in vitro kinase assays with recombinant wild-type or S209A mutant RPS3 proteins. Compared with the wild-type protein, the S209A mutation reduced IKKβ–mediated RPS3 phosphorylation (Fig. 4c). There might be alternative phosphorylation site(s) under these conditions given modest residual phosphorylated RPS3 (Fig. 4c). RPS3 S209 does not fall within a conventional IKK recognition motif, but rather resides in a sequence motif (XXXpS/TXXE), potentially recognized by casein kinase II (CK2). Although IKKβ kinase can display a CK2-like phosphorylation specificity29, no CK2 protein was detectable in our recombinant IKK proteins (Supplementary Fig. 6). Thus, RPS3 S209 phosphorylation was due to the alternate specificity of the IKKβ kinase rather than any trace amount of CK2 bound to IKKs. To determine whether S209 is the critical site at which IKKβ phosphorylates RPS3 in living cells, we transfected the wild-type or S209A mutant Flag-RPS3 alone, or together with IKKβ into cells. Indeed, we observed that overexpressing IKKβ enhanced Flag-RPS3 phosphorylation, but phosphorylation was effectively eliminated by alanine substitution indicating that S209 is the predominant target site for IKKβ phosphorylation (Fig. 4d). We next generated a phospho-S209 RPS3 antibody and confirmed that endogenous RPS3 was phosphorylated at S209 in a time-dependent manner upon TNF stimulation (Fig. 4e). Thus, the RPS3 C-terminal tail potentially contains an important regulatory site.

IKKβ phosphorylates RPS3 at serine 209

Although originally defined as the kinase that phosphorylates IκB19, IKKβ also phosphorylates unrelated substrates including 14-3-3β and Bcl10, which lack the IKK consensus motif (DpSGYXpS/T)28. We therefore hypothesized that IKKβ could directly phosphorylate RPS3. By in vitro kinase assays using recombinant IKK and RPS3 proteins, we observed strong incorporation of 32P in autophosphorylatd IKKα and IKKβ (Fig. 4a, lanes 2-7) as well as phosporylated GST-IκBα (1-54) (Supplementary Fig. 4), but not the GST protein alone (Fig. 4a, lanes 3 and 6), when either IKKα or IKKβ was used. We discovered that GST-RPS3 could be phosphorylated by IKKβ, but not IKKα, in vitro (Fig. 4a, compare lanes 4 and 7).

To identify the RPS3 amino acid residue(s) phosphorylated by IKKβ, we performed liquid chromatography-tandem mass spectrometry analyses using in vitro phosphorylated RPS3. The results indicated that IKKβ phosphorylated S209, located in the RPS3 C-terminus (Fig. 4b). RPS3 amino acid sequence alignment revealed that S209 is conserved in many species throughout phylogeny with the exception of Caenorhabditis elegans and Schizosaccharomyces pombe, two organisms that do not possess the NF-κB signal pathway (Supplementary Fig. 5).

To verify biochemically that S209 is an IKKβ substrate, we performed 32P-labeling in vitro kinase assays with recombinant wild-type or S209A mutant RPS3 proteins. Compared with the wild-type protein, the S209A mutation reduced IKKβ–mediated RPS3 phosphorylation (Fig. 4c). There might be alternative phosphorylation site(s) under these conditions given modest residual phosphorylated RPS3 (Fig. 4c). RPS3 S209 does not fall within a conventional IKK recognition motif, but rather resides in a sequence motif (XXXpS/TXXE), potentially recognized by casein kinase II (CK2). Although IKKβ kinase can display a CK2-like phosphorylation specificity29, no CK2 protein was detectable in our recombinant IKK proteins (Supplementary Fig. 6). Thus, RPS3 S209 phosphorylation was due to the alternate specificity of the IKKβ kinase rather than any trace amount of CK2 bound to IKKs. To determine whether S209 is the critical site at which IKKβ phosphorylates RPS3 in living cells, we transfected the wild-type or S209A mutant Flag-RPS3 alone, or together with IKKβ into cells. Indeed, we observed that overexpressing IKKβ enhanced Flag-RPS3 phosphorylation, but phosphorylation was effectively eliminated by alanine substitution indicating that S209 is the predominant target site for IKKβ phosphorylation (Fig. 4d). We next generated a phospho-S209 RPS3 antibody and confirmed that endogenous RPS3 was phosphorylated at S209 in a time-dependent manner upon TNF stimulation (Fig. 4e). Thus, the RPS3 C-terminal tail potentially contains an important regulatory site.

Phosphorylation of RPS3 and its NF-κB functionWe next examined whether S209 phosphorylation plays a role in the nuclear translocation of RPS3 during NF-κB activation. Subcellular fractions from either wild-type or S209A mutant RPS3-transfected cells were prepared and blotted for heat-shock protein 90 (hsp90), a cytoplasmic protein, and poly (ADP-ribose) polymerase (PARP), a nuclear protein, confirming a clean separation (Fig. 5a). As expected, PMA+I stimulation triggered wild-type Flag-RPS3 nuclear translocation (Fig. 5a). However, RPS3 (S209A) nuclear translocation was attenuated (Fig. 5a). We also tested the impact of activating NF-κB by overexpressing IKKβ on RPS3 nuclear translocation. IKKβ overexpression activated NF-κB measured by luciferase assays (Supplementary Fig. 7), and also induced the nuclear translocation of wild-type, but not S209A, RPS3 (Fig. 5b). These data suggest that S209 phosphorylation is critical for the NF-κB activation-induced RPS3 nuclear translocation.To examine the role of S209 phosphorylation of RPS3 to its NF-κB function6, 7, 30, we silenced endogenous RPS3 expression using an siRNA that targets the 3′ untranslated region (3′ UTR) of RPS3 mRNA, followed by complementation with either wild-type or S209A mutant RPS3 via transfection. As expected, RPS3 siRNA severely reduced endogenous RPS3 abundance compared to NS siRNA, but did not affect the robust expression of Flag-tagged RPS3 from a transfected construct lacking the 3′ UTR (Fig. 5c). We also found that RPS3 knockdown reduced TNF-induced expression of an Ig κB-driven luciferase construct6 (Fig. 5d). The impaired luciferase signal caused by RPS3 deficiency was completely restored by transfecting wild-type, but not by S209A RPS3 (Fig. 5d), despite equivalent expression (Fig. 5c). Moreover, the failure of S209A RPS3 to restore luciferase activity did not result from defective translation because the transient overexpression of green fluorescent protein (GFP) was comparable in cells complemented with wild-type or S029A RPS3 (Supplementary Fig. 8). Taken together, these data suggest that RPS3 S209 phosphorylation is critical for NF-κB activity involving the canonical Ig κB site.We next used chromatin immunoprecipitation to determine whether S209 phosphorylation affects RPS3 and p65 recruitment to specific κB sites in intact chromatin during NF-κB activation. In RPS3 knockdown cells, PMA+I stimulated the recruitment of ectopically expressed, Flag-tagged wild-type, but not S209A RPS3 to the κB sites of the NFKBIA and IL8 promoters (Fig. 5e). While expressing RPS3 S209A had no impact on p65 nuclear translocation, it substantially attenuated p65 recruitment (Fig. 5e). Additional experiments revealed that p65 attraction to RPS3-independent NF-κB target gene promoters such as CD25 was increased (Supplementary Fig. 9), consistent with our previous observations6. There was no significant Flag-RPS3 or p65 recruitment to ACTB promoter lacking κB sites (Fig. 5e), suggesting the recruitment was κB site-specific. Thus, the recruitment of RPS3 as well as the contingent recruitment of p65 to key promoters depended on S209.Interleukin 8 (IL-8) secretion induced by either T cell receptor (TCR) agonist stimulation or PMA+I was decreased as a consequence of reduced RPS3/p65 recruitment to the IL8 κB sites in the presence of S209A mutant compared to wild-type RPS3 (Supplementary Fig. 10). However, cell surface CD25 expression was comparable between the wild-type and S209A RPS3 transfected cells (Supplementary Fig. 11). Therefore, RPS3 S209 phosphorylation by IKKβ is apparently required for RPS3 in directing NF-κB to a specific subset of target genes.

Phosphorylation of RPS3 and its NF-κB function

We next examined whether S209 phosphorylation plays a role in the nuclear translocation of RPS3 during NF-κB activation. Subcellular fractions from either wild-type or S209A mutant RPS3-transfected cells were prepared and blotted for heat-shock protein 90 (hsp90), a cytoplasmic protein, and poly (ADP-ribose) polymerase (PARP), a nuclear protein, confirming a clean separation (Fig. 5a). As expected, PMA+I stimulation triggered wild-type Flag-RPS3 nuclear translocation (Fig. 5a). However, RPS3 (S209A) nuclear translocation was attenuated (Fig. 5a). We also tested the impact of activating NF-κB by overexpressing IKKβ on RPS3 nuclear translocation. IKKβ overexpression activated NF-κB measured by luciferase assays (Supplementary Fig. 7), and also induced the nuclear translocation of wild-type, but not S209A, RPS3 (Fig. 5b). These data suggest that S209 phosphorylation is critical for the NF-κB activation-induced RPS3 nuclear translocation.

To examine the role of S209 phosphorylation of RPS3 to its NF-κB function6, 7, 30, we silenced endogenous RPS3 expression using an siRNA that targets the 3′ untranslated region (3′ UTR) of RPS3 mRNA, followed by complementation with either wild-type or S209A mutant RPS3 via transfection. As expected, RPS3 siRNA severely reduced endogenous RPS3 abundance compared to NS siRNA, but did not affect the robust expression of Flag-tagged RPS3 from a transfected construct lacking the 3′ UTR (Fig. 5c). We also found that RPS3 knockdown reduced TNF-induced expression of an Ig κB-driven luciferase construct6 (Fig. 5d). The impaired luciferase signal caused by RPS3 deficiency was completely restored by transfecting wild-type, but not by S209A RPS3 (Fig. 5d), despite equivalent expression (Fig. 5c). Moreover, the failure of S209A RPS3 to restore luciferase activity did not result from defective translation because the transient overexpression of green fluorescent protein (GFP) was comparable in cells complemented with wild-type or S029A RPS3 (Supplementary Fig. 8). Taken together, these data suggest that RPS3 S209 phosphorylation is critical for NF-κB activity involving the canonical Ig κB site.

We next used chromatin immunoprecipitation to determine whether S209 phosphorylation affects RPS3 and p65 recruitment to specific κB sites in intact chromatin during NF-κB activation. In RPS3 knockdown cells, PMA+I stimulated the recruitment of ectopically expressed, Flag-tagged wild-type, but not S209A RPS3 to the κB sites of the NFKBIA and IL8 promoters (Fig. 5e). While expressing RPS3 S209A had no impact on p65 nuclear translocation, it substantially attenuated p65 recruitment (Fig. 5e). Additional experiments revealed that p65 attraction to RPS3-independent NF-κB target gene promoters such as CD25 was increased (Supplementary Fig. 9), consistent with our previous observations6. There was no significant Flag-RPS3 or p65 recruitment to ACTB promoter lacking κB sites (Fig. 5e), suggesting the recruitment was κB site-specific. Thus, the recruitment of RPS3 as well as the contingent recruitment of p65 to key promoters depended on S209.

Interleukin 8 (IL-8) secretion induced by either T cell receptor (TCR) agonist stimulation or PMA+I was decreased as a consequence of reduced RPS3/p65 recruitment to the IL8 κB sites in the presence of S209A mutant compared to wild-type RPS3 (Supplementary Fig. 10). However, cell surface CD25 expression was comparable between the wild-type and S209A RPS3 transfected cells (Supplementary Fig. 11). Therefore, RPS3 S209 phosphorylation by IKKβ is apparently required for RPS3 in directing NF-κB to a specific subset of target genes.

NleH1 inhibits RPS3 phosphorylation in vitroEHEC pathogens are important causative agents of both foodborne disease and pediatric renal failure31. EHEC utilize T3SS to inject effector proteins directly into intestinal epithelial cells32, a subset of which inhibit NF-κB-dependent innate responses9, 33-38. The E. coli O157:H7 EDL933 effector protein NleH1 binds to and attenuates RPS3 nuclear translocation, thus impairing RPS3-dependent NF-κB signaling9. We therefore hypothesized that NleH1 may function by inhibiting RPS3 S209 phosphorylation. As expected, transfecting increasing amounts of NleH1-HA plasmid blocked TNFα-induced NF-κB activation in a dose-dependent manner (Fig. 6a-b)9. Remarkably, NleH1 reduced both TNF-induced, as well as basal RPS3 phosphorylation to roughly 20% of vehicle control (Fig. 6c). Expressing NleH1 does not interfere with either TNF-induced IKK activation or IκBα degradation, consistent with the lack of NleH1 impact on p65 nuclear translocation9 (Fig. 6c).To determine if NleH1 inhibits RPS3 phosphorylation, we infected HeLa cells with E. coli O157:H7 strains possessing or lacking either nleH1 (ΔnleH1) or with a strain lacking a functional T3SS unable to inject NleH1 into mammalian cells (ΔescN). In uninfected cells, TNF-treatment stimulated a ∼7-fold increase in RPS3 S209 phosphorylation, peaking at 30 minutes (Fig. 6d). By contrast, RPS3 S209 phosphorylation was substantially impaired in cells infected with wild-type E. coli O157:H7 (Fig. 6d). However, TNF-induced RPS3 S209 phosphorylation was unimpaired in cells infected with either ΔnleH1 or ΔescN (Fig. 6d). We showed previously that wild-type, but not ΔnleH1 or ΔescN E. coli O157:H7 significantly attenuated TNF-induced RPS3 nuclear translocation9. The parallel between RPS3 phosphorylation and its nuclear translocation during E. coli infection provides evidence in the context of an NF-κB-dependent disease process that RPS3 S209 phosphorylation is important for nuclear translocation.Our discovery that NleH1 inhibits RPS3 S209 phosphorylation suggested that it should also blocks RPS3-dependent NF-κB target gene transcription (e.g. IL8, NFKBIA, and TNFAIP3). Indeed, these genes were only modestly upregulated in cells infected with wild-type E. coli O157:H7, but significantly induced in cells infected with either ΔnleH1 or ΔescN strains (Fig. 6e). In contrast, deleting nleH1 had no impact on the expression of RPS3-independent genes, including CD25 and TNFSF13B (Supplementary Fig. 12). Together these results demonstrate that NleH1 specifically inhibits the protective immune response by directly blocking RPS3 S209 phosphorylation and thereby impairing critical RPS3-dependent NF-κB target genes.

NleH1 inhibits RPS3 phosphorylation in vitro

EHEC pathogens are important causative agents of both foodborne disease and pediatric renal failure31. EHEC utilize T3SS to inject effector proteins directly into intestinal epithelial cells32, a subset of which inhibit NF-κB-dependent innate responses9, 33-38. The E. coli O157:H7 EDL933 effector protein NleH1 binds to and attenuates RPS3 nuclear translocation, thus impairing RPS3-dependent NF-κB signaling9. We therefore hypothesized that NleH1 may function by inhibiting RPS3 S209 phosphorylation. As expected, transfecting increasing amounts of NleH1-HA plasmid blocked TNFα-induced NF-κB activation in a dose-dependent manner (Fig. 6a-b)9. Remarkably, NleH1 reduced both TNF-induced, as well as basal RPS3 phosphorylation to roughly 20% of vehicle control (Fig. 6c). Expressing NleH1 does not interfere with either TNF-induced IKK activation or IκBα degradation, consistent with the lack of NleH1 impact on p65 nuclear translocation9 (Fig. 6c).

To determine if NleH1 inhibits RPS3 phosphorylation, we infected HeLa cells with E. coli O157:H7 strains possessing or lacking either nleH1 (ΔnleH1) or with a strain lacking a functional T3SS unable to inject NleH1 into mammalian cells (ΔescN). In uninfected cells, TNF-treatment stimulated a ∼7-fold increase in RPS3 S209 phosphorylation, peaking at 30 minutes (Fig. 6d). By contrast, RPS3 S209 phosphorylation was substantially impaired in cells infected with wild-type E. coli O157:H7 (Fig. 6d). However, TNF-induced RPS3 S209 phosphorylation was unimpaired in cells infected with either ΔnleH1 or ΔescN (Fig. 6d). We showed previously that wild-type, but not ΔnleH1 or ΔescN E. coli O157:H7 significantly attenuated TNF-induced RPS3 nuclear translocation9. The parallel between RPS3 phosphorylation and its nuclear translocation during E. coli infection provides evidence in the context of an NF-κB-dependent disease process that RPS3 S209 phosphorylation is important for nuclear translocation.

Our discovery that NleH1 inhibits RPS3 S209 phosphorylation suggested that it should also blocks RPS3-dependent NF-κB target gene transcription (e.g. IL8, NFKBIA, and TNFAIP3). Indeed, these genes were only modestly upregulated in cells infected with wild-type E. coli O157:H7, but significantly induced in cells infected with either ΔnleH1 or ΔescN strains (Fig. 6e). In contrast, deleting nleH1 had no impact on the expression of RPS3-independent genes, including CD25 and TNFSF13B (Supplementary Fig. 12). Together these results demonstrate that NleH1 specifically inhibits the protective immune response by directly blocking RPS3 S209 phosphorylation and thereby impairing critical RPS3-dependent NF-κB target genes.

NleH1 inhibits RPS3 S209 phosphorylation in vivoWe previously utilized a gnotobiotic piglet infection model to determine that piglets infected with ΔnleH1 mutant died more rapidly than those infected with wild-type E. coli O157:H7. Piglets infected with ΔnleH1 displayed clinical disease consistent with a robust inflammatory response, but with reduced bacterial colonization and little diarrhea9. While seemingly paradoxical, based on our cell culture data (Fig. 6d), we hypothesized that NleH1 blocked RPS3 S209 phosphorylation in vivo, thereby preventing RPS3 nuclear translocation in infected piglets. We isolated piglet colons at necropsy, and subjected them to immunohistochemistry using our phospho-RPS3 antibody. Consistent with in vitro data, piglets infected with wild-type E. coli O157:H7 exhibited diffuse and low intensity phospho-RPS3 staining, whereas in piglets infected with ΔnleH1 mutant, phospho-RPS3 expression was florid and intense (Fig. 6f). These data demonstrate that NleH1 inhibits RPS3 S209 phosphorylation both in vitro and in vivo, which might benefit the bacterium in colonization and transmission.

NleH1 inhibits RPS3 S209 phosphorylation in vivo

We previously utilized a gnotobiotic piglet infection model to determine that piglets infected with ΔnleH1 mutant died more rapidly than those infected with wild-type E. coli O157:H7. Piglets infected with ΔnleH1 displayed clinical disease consistent with a robust inflammatory response, but with reduced bacterial colonization and little diarrhea9. While seemingly paradoxical, based on our cell culture data (Fig. 6d), we hypothesized that NleH1 blocked RPS3 S209 phosphorylation in vivo, thereby preventing RPS3 nuclear translocation in infected piglets. We isolated piglet colons at necropsy, and subjected them to immunohistochemistry using our phospho-RPS3 antibody. Consistent with in vitro data, piglets infected with wild-type E. coli O157:H7 exhibited diffuse and low intensity phospho-RPS3 staining, whereas in piglets infected with ΔnleH1 mutant, phospho-RPS3 expression was florid and intense (Fig. 6f). These data demonstrate that NleH1 inhibits RPS3 S209 phosphorylation both in vitro and in vivo, which might benefit the bacterium in colonization and transmission.

NleH1 steers the IKKβ substrate specificitiesNleH1 is an autophosphorylated serine-threonine kinase, which depends on the lysine 159 (K159)9. To explore the mechanism by which NleH1 inhibits RPS3 S209 phosphorylation, we first performed an in vitro kinase assay with purified wild-type His-NleH1 protein and a mutant His-NleH1 (K159A) protein, confirming that NleH1 is autophosphorylated and the K159A is an NleH1 kinase-dead mutant (Fig. 7a). To examine whether the kinase activity is required for NleH1 to inhibit IKKβ phosphorlyation of RPS3 on S209, we ectopically expressing either wild-type or K159A NleH1 in 293T cells. Wild-type NleH1 expression significantly reduced TNF-induced RPS3 S209 phosphorylation, whereas the K159A mutant failed to do so (Fig. 7b). Thus NleH1 kinase activity is required to protect RPS3 from IKKβ-mediated phosphorylation.Citrobacter rodentium is a mouse pathogen that shares pathogenic strategies with E. coli
39, most notably for our investigation, C. rodentium NleH inhibited RPS3 nuclear translocation and RPS3-dependent NF-κB luciferase activity to an extent equivalent to E. coli NleH1 (ref. 9). We assayed RPS3 S209 phosphorylation in HeLa cells infected with different C. rodentium strains. In uninfected cells, TNF-treatment stimulated a ∼3.5-fold increase in RPS3 S209 phosphorylation (Fig. 7c). Such augmentation of RPS3 phosphorylation was reduced by about 60% by wild-type C. rodentium infection (Fig. 7c). However, RPS3 phosphorylation was enhanced when cells were infected with a C. rodentium strain lacking NleH (Fig. 7c, ΔnleH). We further examined the role of NleH1 kinase activity using the C. rodentium ΔnleH strain as a background on which to express either wild-type or K159A E. coli NleH1. Complementing ΔnleH mutant with wild-type NleH1 almost abolished TNF-induced RPS3 S209 phosphorylation whereas complementing with K159A failed to inhibit RPS3 phosphorylation (Fig. 7c). Collectively, these results demonstrate that NleH1 kinase activity is required to block RPS3 S209 phosphorylation.We next examined whether the inhibitory activity of NleH1 is sufficiently robust to impair the strong nuclear translocation of RPS3 trigged by the constitutively-active IKKβ (IKKβ [SSEE]) (Fig. 2d). We found that ectopically expressing either wild-type or SSEE IKKβ proteins, triggered more RPS3 nuclear translocation than the kinase-dead IKKβ (SSAA) protein (Fig. 7d). RPS3 nuclear accumulation was substantially retarded by infecting cells with wild-type E. coli O157:H7 (Fig. 7d). In contrast, infecting with either ΔnleH1 or ΔescN strains only slightly impaired RPS3 nuclear translocation in either IKKβ- or IKKβ (SSEE)-expressing cells (Fig. 7d). As expected, E. coli infections did not affect the RPS3 nuclear translocation in IKKβ (SSAA)-expressing cells, where NF-κB signaling was low (Fig. 7d). Thus, during infection NleH1 is sufficiently potent to inhibit RPS3 nuclear translocation even in cells expressing constitutively-activated IKKβ.We examined whether NleH1 could directly phosphorylate IKKβ thus inhibiting IKKβ-mediated RPS3 S209 phosphorylation. We performed in vitro kinase assays using immunoprecipitated Flag-IKKβ (K44A) as substrate and recombinant His-NleH1 as kinase, so that IKKβ autophosphorylation would not obscure NleH1-induced phosphorylation. However, we did not observe any detectable 32P incorporation in IKKβ (Supplementary Fig. 13), thus ruling out this possibility.We then tested the hypothesis that NleH1 could alter the IKKβ substrate specificities. To this end, we performed in vitro kinase assays using both CK2 and IKK substrates for IKKβ. As expected, IKKβ phosphorylated RPS3 (Fig. 7e, lane 7) and GST-IκBα (1-54) protein (Fig. 7e, lane 9), demonstrating it harbors either CK2 or IKK substrate specificity. Preincubation of IKKβ with NleH1 reduced IKKβ-mediated RPS3 phosphorylation, i.e. the CK2 kinase specificity, but not IKKβ-mediated GST-IκBα phosphorylation, i.e. the IKK kinase specificity (Fig. 7e). Control experiments revealed no NleH1-mediated phosphorylation or autophosphorylation of RPS3 or GST-IκBα (Fig. 7e). Taken together, NleH1 blocks the CK2 substrate specificity of IKKβ thus inhibiting the IKKβ-mediated RPS3 S209 phosphorylation thus representing a novel strategy by E. coli O157:H7 to alter the host innate immune response.

NleH1 steers the IKKβ substrate specificities

NleH1 is an autophosphorylated serine-threonine kinase, which depends on the lysine 159 (K159)9. To explore the mechanism by which NleH1 inhibits RPS3 S209 phosphorylation, we first performed an in vitro kinase assay with purified wild-type His-NleH1 protein and a mutant His-NleH1 (K159A) protein, confirming that NleH1 is autophosphorylated and the K159A is an NleH1 kinase-dead mutant (Fig. 7a). To examine whether the kinase activity is required for NleH1 to inhibit IKKβ phosphorlyation of RPS3 on S209, we ectopically expressing either wild-type or K159A NleH1 in 293T cells. Wild-type NleH1 expression significantly reduced TNF-induced RPS3 S209 phosphorylation, whereas the K159A mutant failed to do so (Fig. 7b). Thus NleH1 kinase activity is required to protect RPS3 from IKKβ-mediated phosphorylation.

Citrobacter rodentium is a mouse pathogen that shares pathogenic strategies with E. coli
39, most notably for our investigation, C. rodentium NleH inhibited RPS3 nuclear translocation and RPS3-dependent NF-κB luciferase activity to an extent equivalent to E. coli NleH1 (ref. 9). We assayed RPS3 S209 phosphorylation in HeLa cells infected with different C. rodentium strains. In uninfected cells, TNF-treatment stimulated a ∼3.5-fold increase in RPS3 S209 phosphorylation (Fig. 7c). Such augmentation of RPS3 phosphorylation was reduced by about 60% by wild-type C. rodentium infection (Fig. 7c). However, RPS3 phosphorylation was enhanced when cells were infected with a C. rodentium strain lacking NleH (Fig. 7c, ΔnleH). We further examined the role of NleH1 kinase activity using the C. rodentium ΔnleH strain as a background on which to express either wild-type or K159A E. coli NleH1. Complementing ΔnleH mutant with wild-type NleH1 almost abolished TNF-induced RPS3 S209 phosphorylation whereas complementing with K159A failed to inhibit RPS3 phosphorylation (Fig. 7c). Collectively, these results demonstrate that NleH1 kinase activity is required to block RPS3 S209 phosphorylation.

We next examined whether the inhibitory activity of NleH1 is sufficiently robust to impair the strong nuclear translocation of RPS3 trigged by the constitutively-active IKKβ (IKKβ [SSEE]) (Fig. 2d). We found that ectopically expressing either wild-type or SSEE IKKβ proteins, triggered more RPS3 nuclear translocation than the kinase-dead IKKβ (SSAA) protein (Fig. 7d). RPS3 nuclear accumulation was substantially retarded by infecting cells with wild-type E. coli O157:H7 (Fig. 7d). In contrast, infecting with either ΔnleH1 or ΔescN strains only slightly impaired RPS3 nuclear translocation in either IKKβ- or IKKβ (SSEE)-expressing cells (Fig. 7d). As expected, E. coli infections did not affect the RPS3 nuclear translocation in IKKβ (SSAA)-expressing cells, where NF-κB signaling was low (Fig. 7d). Thus, during infection NleH1 is sufficiently potent to inhibit RPS3 nuclear translocation even in cells expressing constitutively-activated IKKβ.

We examined whether NleH1 could directly phosphorylate IKKβ thus inhibiting IKKβ-mediated RPS3 S209 phosphorylation. We performed in vitro kinase assays using immunoprecipitated Flag-IKKβ (K44A) as substrate and recombinant His-NleH1 as kinase, so that IKKβ autophosphorylation would not obscure NleH1-induced phosphorylation. However, we did not observe any detectable 32P incorporation in IKKβ (Supplementary Fig. 13), thus ruling out this possibility.

We then tested the hypothesis that NleH1 could alter the IKKβ substrate specificities. To this end, we performed in vitro kinase assays using both CK2 and IKK substrates for IKKβ. As expected, IKKβ phosphorylated RPS3 (Fig. 7e, lane 7) and GST-IκBα (1-54) protein (Fig. 7e, lane 9), demonstrating it harbors either CK2 or IKK substrate specificity. Preincubation of IKKβ with NleH1 reduced IKKβ-mediated RPS3 phosphorylation, i.e. the CK2 kinase specificity, but not IKKβ-mediated GST-IκBα phosphorylation, i.e. the IKK kinase specificity (Fig. 7e). Control experiments revealed no NleH1-mediated phosphorylation or autophosphorylation of RPS3 or GST-IκBα (Fig. 7e). Taken together, NleH1 blocks the CK2 substrate specificity of IKKβ thus inhibiting the IKKβ-mediated RPS3 S209 phosphorylation thus representing a novel strategy by E. coli O157:H7 to alter the host innate immune response.

DiscussionRPS3 was previously demonstrated to function as an integral subunit conferring NF-κB regulatory specificity6. Here we sought to elucidate how NF-κB activation signaling triggers RPS3 to translocate and participate in NF-κB function in the nucleus. We demonstrate that IKKβ-mediated RPS3 S209 phosphorylation represents a critical determinant in governing its nuclear import thus unveiling a novel mechanism behind NF-κB regulatory specificity. IKKβ is the major kinase that phosphorylates IκBs in the classical NF-κB pathway, leading to their subsequent degradation40. Strikingly, RPS3 possesses not any consensus IKK motif; instead, S209 is centered in a consensus CK2 motif. The recent observation that human IKKβ displayed CK2-like phosphorylation specificity29 coincides with our evidence that recombinant IKKβ, but not IKKα, phosphorylated RPS3. We found this phosphorylation is a critical modulation for RPS3 nuclear translocation (via importin-α) and engagement in specific NF-κB transcription. CK2 was previously shown to phosphorylate p65 and to bind to and phosphorylate IKKβ41-43, however, we ruled out the possibility that the IKKβ-bound CK2 could account for the observed RPS3 phosphorylation because no CK2 was detected in the IKKβ preparations used for the in vitro kinase assay. Because RPS3 only harbors the CK2 motif and not a traditional IKK motif, this RPS3 regulatory function could explain why IKK harbors the alternative substrate phosphorylation capability.More importantly, our study has elucidated how RPS3 is biochemically integrated into NF-κB activation signaling in a manner that is pivotal for the pathogenesis of foodborne pathogen E. coli O157:H7. IKKβ-mediated RPS3 S209 phosphorylation is a critical target modulated by this pathogen to subvert host NF-κB signaling. The bacterial effector NleH1 specifically binds to RPS3 once injected into host cells and profoundly suppresses NF-κB and its attendant protective immune responses9. Our data now show that NleH1 selectively inhibits RPS3 phosphorylation, thus retarding its nuclear translocation and subsequent NF-κB function, without altering other NF-κB signaling. Although NleH1 did not directly phosphorylate IKKβ, its kinase activity was required to inhibit IKKβ-mediated RPS3 S209 phosphorylation. Many bacteria pathogens have products that target key kinases to inactivate them in host cells, whereas E. coli O157:H7 employed NleH1 to steer the substrate specificity of IKKβ thus specifically fine-tuning host NF-κB signaling. This could represent a novel strategy to fine-tune host NF-κB signaling that could be shared by other pathogens. These data provide new insights into the poorly understood action mechanism for most T3SS effectors.NleH1 attenuates the transcription of RPS3-dependent, but not all, NF-κB target genes, in particular those genes associated with acute proinflammatory responses, including IL8 and TNF. In contrast, NleH1 does not block NF-κB p65 nuclear translocation, which suggests that certain p65-dependent but RPS3-independent NF-κB target genes might thus be beneficial for E. coli O157:H7 to replicate and disseminate in the host. By selectively inhibiting RPS3 and its attendant NF-κB function with NleH1, the pathogen achieves the ability to increase colonization and diarrhea yet limiting the mortality of the host. This seemingly paradoxical combination of effects make sense when one considers that increased bacterial load and diarrhea together with survival of the infected host would promote the spreading of the bacteria among a population of susceptible individuals. Such complex and paradoxical pathological effects that influence the spread of disease are often poorly understood at the molecular level. Our data elucidate how alterations in selective NF-κB function, achieved by impeding RPS3, but not altering p65 nuclear translocation, can influence specific cytokines that affect bacterial colonization, diarrhea diseases and mortality. It may be fruitful in attempting to understand other infectious and autoimmune diseases involving NF-κB to consider selective effects of subunits such as RPS3 in addition to global NF-κB inhibition.

RPS3 was previously demonstrated to function as an integral subunit conferring NF-κB regulatory specificity6. Here we sought to elucidate how NF-κB activation signaling triggers RPS3 to translocate and participate in NF-κB function in the nucleus. We demonstrate that IKKβ-mediated RPS3 S209 phosphorylation represents a critical determinant in governing its nuclear import thus unveiling a novel mechanism behind NF-κB regulatory specificity. IKKβ is the major kinase that phosphorylates IκBs in the classical NF-κB pathway, leading to their subsequent degradation40. Strikingly, RPS3 possesses not any consensus IKK motif; instead, S209 is centered in a consensus CK2 motif. The recent observation that human IKKβ displayed CK2-like phosphorylation specificity29 coincides with our evidence that recombinant IKKβ, but not IKKα, phosphorylated RPS3. We found this phosphorylation is a critical modulation for RPS3 nuclear translocation (via importin-α) and engagement in specific NF-κB transcription. CK2 was previously shown to phosphorylate p65 and to bind to and phosphorylate IKKβ41-43, however, we ruled out the possibility that the IKKβ-bound CK2 could account for the observed RPS3 phosphorylation because no CK2 was detected in the IKKβ preparations used for the in vitro kinase assay. Because RPS3 only harbors the CK2 motif and not a traditional IKK motif, this RPS3 regulatory function could explain why IKK harbors the alternative substrate phosphorylation capability.

More importantly, our study has elucidated how RPS3 is biochemically integrated into NF-κB activation signaling in a manner that is pivotal for the pathogenesis of foodborne pathogen E. coli O157:H7. IKKβ-mediated RPS3 S209 phosphorylation is a critical target modulated by this pathogen to subvert host NF-κB signaling. The bacterial effector NleH1 specifically binds to RPS3 once injected into host cells and profoundly suppresses NF-κB and its attendant protective immune responses9. Our data now show that NleH1 selectively inhibits RPS3 phosphorylation, thus retarding its nuclear translocation and subsequent NF-κB function, without altering other NF-κB signaling. Although NleH1 did not directly phosphorylate IKKβ, its kinase activity was required to inhibit IKKβ-mediated RPS3 S209 phosphorylation. Many bacteria pathogens have products that target key kinases to inactivate them in host cells, whereas E. coli O157:H7 employed NleH1 to steer the substrate specificity of IKKβ thus specifically fine-tuning host NF-κB signaling. This could represent a novel strategy to fine-tune host NF-κB signaling that could be shared by other pathogens. These data provide new insights into the poorly understood action mechanism for most T3SS effectors.

NleH1 attenuates the transcription of RPS3-dependent, but not all, NF-κB target genes, in particular those genes associated with acute proinflammatory responses, including IL8 and TNF. In contrast, NleH1 does not block NF-κB p65 nuclear translocation, which suggests that certain p65-dependent but RPS3-independent NF-κB target genes might thus be beneficial for E. coli O157:H7 to replicate and disseminate in the host. By selectively inhibiting RPS3 and its attendant NF-κB function with NleH1, the pathogen achieves the ability to increase colonization and diarrhea yet limiting the mortality of the host. This seemingly paradoxical combination of effects make sense when one considers that increased bacterial load and diarrhea together with survival of the infected host would promote the spreading of the bacteria among a population of susceptible individuals. Such complex and paradoxical pathological effects that influence the spread of disease are often poorly understood at the molecular level. Our data elucidate how alterations in selective NF-κB function, achieved by impeding RPS3, but not altering p65 nuclear translocation, can influence specific cytokines that affect bacterial colonization, diarrhea diseases and mortality. It may be fruitful in attempting to understand other infectious and autoimmune diseases involving NF-κB to consider selective effects of subunits such as RPS3 in addition to global NF-κB inhibition.

MethodsCells and ReagentsJurkat E6.1, HEK293T and HeLa cells were cultured in RPMI 1640 and DMEM supplemented with 10% fetal calf serum, 2 mM glutamine, and 100 U/ml each of penicillin and streptomycin, respectively. IκBα (C-21, sc-371), p65 (C-20, sc-372), and phospho-threonine (H-2, sc-5267) antibodies were from Santa Cruz Biotechnology; β-actin (AC-15, A5441), Flag (M2, F3165), HA (HA-7, H3663), importin-α (IM-75, I1784), and importin-β (31H4, I2534) antibodies were from Sigma; PARP (C2-10, 556362), IKKα (B78-1, 556532), and IKKβ (24, 611254) antibodies were from BD Pharmingen; CK2α (31, 611610) and Hsp90 (68, 610418) antibodies were from BD Transduction Laboratories; phospho-IκBα (5A5, 9246S) and phospho-IKKα/β (16A6, 2697S) antibodies were from Cell Signaling Technology; phospho-serine (AB1603) and phospho-tyrosine (4G10, 05-777) antibodies were from Millipore. The rabbit polyclonal RPS3 antiserum was as described previously6. The rabbit polyclonal antibody specific for S209 phosphorylated RPS3 was generated and affinity purified by Primm Biotech using the peptide NH2-CKPLPDHV(Sp)IVE-COOH.Plasmid ConstructsThe Flag-IKKβ (SSEE), Flag-IKKβ (SSAA), and HA-IκBα (SSAA) constructs were provided by C. Wu (NCI, Bethesda) and U. Siebenlist (NIAID, Bethesda), respectively. The HA-IκBα and IKKβ (K44A)-Flag plasmids were purchased from Addgene44, 45. The Flag-RPS3, GST-RPS3, HA-RPS3, VN-HA, NleH1-HA plasmids were described previously6, 9. The point mutants of RPS3 were generated by site-directed mutagenesis using the Quick Change Kit (Stratagene) with primers forward 5′-CTGCCTGACCACGTGGCCATTGTGGAACCCAAA-3′ and reverse 5′-TTTGGGTTCCACAATGGCCACGTGGTCAGGCAG-3′ for S209A. All mutants were verified by DNA sequencing.32P in vivo LabelingHEK 293T cells were labeled with 2 mCi/ml 32P-orthophosphate (Perkin Elmer) in phosphate-free medium (Invitrogen) for 2 h. Cells were then left untreated or treated with TNF (50 ng/ml, R&D Systems) for indicated periods. Cell lysates were prepared and used for immunoprecipitations with RPS3 antibody.In Vitro Kinase AssayKinase-active recombinant IKKβ and IKKα proteins were purchased from Active Motif and Millipore, respectively. Bacterially purified glutathione S-transferase (GST), GST-IκBα (1-54), wild type, mutant GST-RPS3, or RPS3 proteins were used as substrates. The in vitro kinase assay was performed as previously described29. Briefly, enzyme (100 ng) and substrate (2 μg) were co-incubated in IKK reaction buffer (25 mM Tris–HCl [pH 8.0], 50 mM KCl, 10 mM MgCl2, 1 mM DTT, 1 mM Na3VO4, 1 mM ATP) or NleH1 reaction buffer (50 mM Tris-HCl [pH 7.6], 5 mM MgCl2, 1 mM DTT, 1 mM ATP) with 0.5 μCi 32P-γ-ATP (GE Healthcare) added at 37 °C for 30 min. The reactions were resolved by SDS–PAGE and visualized by autoradiography.LC-MS/MS AnalysisGST or GST-RPS3 was incubated with recombinant IKKβ protein as described above in an in vitro kinase assay reaction conducted without 32P-γ-ATP labeling. The reaction was separated by SDS-PAGE, and the protein gel was stained with Colloidal Blue (Invitrogen). The corresponding protein fragments were excised and subjected to trypsin digestion and LC-MS/MS at the Yale Cancer Center Mass Spectrometry Resource (New Haven, CT).RNAi and TransfectionThe siRNA (sense-strand sequence) IKKα, 5′-AUGACAGAGAAUGAUCAUGUUCUGC -3′; IKKβ, 5′-GCAGCAAGGAGAACAGAGGUUAAUA -3′; IκBα, 5′-GAGCUCCGAGACUUUCGAGGAAAUA -3′; RPS3-3′ UTR, 5′-GGAUGUUGCUCUCUAAAGACC -3′ (Invitrogen). Transient transfection of siRNA and DNA constructs into Jurkat cells and 293T cells was described previously6.Subcellular FractionationSubcellular fractionation was performed by differential centrifugation as previously described6. Briefly, cells were resuspended in ice-cold Buffer A (10 mM HEPES pH 7.9, 10 mM KCl, 1.5 mM MgCl2, 0.1 mM EDTA, 0.5 mM DTT, 0.4 % NP-40, 0.5 mM PMSF, complete protease inhibitor cocktail) at 4 °C for 5 min. Lysates were centrifuged at 4 °C, 500 × g for 3 min, and supernatants were collected as cytosolic fractions. Pellets were incubated in Buffer C (20 mM HEPES pH7.9, 420 mM NaCl, 1.5 mM MgCl2, 25% glycerol, 0.5 mM PMSF, 0.2 mM EDTA, 0.5 mM DTT, complete protease inhibitor cocktail) at 4 °C for 10 min. Supernatants were collected as nuclear fractions following a centrifuge at 4 °C, 13,800 × g for 10 min.Luciferase Reporter Gene AssaysLuciferase reporter gene assays were performed as previously described6. Briefly, cells were cotransfected at a ratio of 10:1 with various promoter-driven firefly luciferase constructs to the Renilla luciferase pTKRL plasmid, together with indicated plasmids. Cells were cultured for 1–2 days and then stimulated in triplicate before harvest. Lysates were analyzed using the Dual-Luciferase Kit (Promega).Chromatin Immunoprecipitation (ChIP)ChIP assays was performed as previously described6. The primers used to amplify the promoter region adjacent to the κB sites of IL8 and NFKBIA, as well as ACTB have been described6.Immunofluorescence MicroscopyConfocal microscopy was performed as previously described6. Briefly, cells were fixed with 4 % paraformaldehyde in PBS and then Cellspin mounted onto slides. The fixed cells were then permeabilized with 0.05 % Triton X-100 in PBS and stained with FITC-conjugated rabbit anti-RPS3 antibodies (Primm Biotech), or AlexaFluor 594-conjugated rat anti-Flag antibodies (BD) for 40 min together with 1 μg/ml of Hoechst 33342 (Sigma) for 5 min at 25 °C. The slides were then rinsed with PBS three times and cover mounted for fluorescence microscopy.Immunoprecipitation and immunoblotThe cells were harvested and lysed on ice by 0.4 ml of the modified RIPA buffer (50 mM Tris-HCl [pH 7.4], 1% NP-40, 0.25% Na-deoxycholate, 150 mM NaCl, 1 mM EDTA, 1 mM PMSF, 1 mM Na3VO4, 1 mM NaF) supplemented with 1 × protease inhibitor cocktail (Roche) and 1 × phosphatase inhibitor cocktail set I (EMD Biosciences) for 30 min. The lysates were centrifuged at 10,000 × g at 4 °C for 10 min to remove insoluble material. After normalizing protein concentrations, lysates were subjected to immunoprecipitation by adding 10 mg/ml appropriate antibody plus 30 ml of protein G-agarose (Roche), and rotated for at least 2 h at 4°C. The precipitates were washed at least five times with cold lysis buffer followed by separation by SDS-PAGE under reduced and denaturing conditions. Nitrocellulose membranes were blocked in 5 % nonfat milk in 0.1 % PBS-Tween 20 (PBS-T), probed with specific antibodies as described previously6. For immunoblotting of phosphorylated proteins, gels were transferred to methanol-treated polyvinylidene chloride membranes, retreated with methanol, and dried for 30 min. Blots were blocked in 5 % bovine serum albumin in 0.1 % Tris buffered saline-Tween 20 (TBS-T), and probed with specific antibodies as described previously46. Bands were imaged by the Super Signaling system (Pierce) according to the manufacturer's instructions.ELISAThe amount of IL-8 present in supernatants collected from Jurkat cell culture was measured using a Human Interleukin-8 ELISA Ready-SET-Go kit (eBioscience) according to the manufacturer's instructions.Cell InfectionsHeLa cells were infected with E. coli O157:H7 or C. rodentium strains as described previously9.ImmunohistochemistryGnotobiotic piglets were infected with E. coli O157:H7 strains as described previously9. Spiral colon specimens were collected at necropsy and embedded in paraffin. Paraffin sectioning and immunohistochemical staining using phospho-RPS3 antibody were performed by Histoserv Inc.

Cells and ReagentsJurkat E6.1, HEK293T and HeLa cells were cultured in RPMI 1640 and DMEM supplemented with 10% fetal calf serum, 2 mM glutamine, and 100 U/ml each of penicillin and streptomycin, respectively. IκBα (C-21, sc-371), p65 (C-20, sc-372), and phospho-threonine (H-2, sc-5267) antibodies were from Santa Cruz Biotechnology; β-actin (AC-15, A5441), Flag (M2, F3165), HA (HA-7, H3663), importin-α (IM-75, I1784), and importin-β (31H4, I2534) antibodies were from Sigma; PARP (C2-10, 556362), IKKα (B78-1, 556532), and IKKβ (24, 611254) antibodies were from BD Pharmingen; CK2α (31, 611610) and Hsp90 (68, 610418) antibodies were from BD Transduction Laboratories; phospho-IκBα (5A5, 9246S) and phospho-IKKα/β (16A6, 2697S) antibodies were from Cell Signaling Technology; phospho-serine (AB1603) and phospho-tyrosine (4G10, 05-777) antibodies were from Millipore. The rabbit polyclonal RPS3 antiserum was as described previously6. The rabbit polyclonal antibody specific for S209 phosphorylated RPS3 was generated and affinity purified by Primm Biotech using the peptide NH2-CKPLPDHV(Sp)IVE-COOH.

Cells and Reagents

Jurkat E6.1, HEK293T and HeLa cells were cultured in RPMI 1640 and DMEM supplemented with 10% fetal calf serum, 2 mM glutamine, and 100 U/ml each of penicillin and streptomycin, respectively. IκBα (C-21, sc-371), p65 (C-20, sc-372), and phospho-threonine (H-2, sc-5267) antibodies were from Santa Cruz Biotechnology; β-actin (AC-15, A5441), Flag (M2, F3165), HA (HA-7, H3663), importin-α (IM-75, I1784), and importin-β (31H4, I2534) antibodies were from Sigma; PARP (C2-10, 556362), IKKα (B78-1, 556532), and IKKβ (24, 611254) antibodies were from BD Pharmingen; CK2α (31, 611610) and Hsp90 (68, 610418) antibodies were from BD Transduction Laboratories; phospho-IκBα (5A5, 9246S) and phospho-IKKα/β (16A6, 2697S) antibodies were from Cell Signaling Technology; phospho-serine (AB1603) and phospho-tyrosine (4G10, 05-777) antibodies were from Millipore. The rabbit polyclonal RPS3 antiserum was as described previously6. The rabbit polyclonal antibody specific for S209 phosphorylated RPS3 was generated and affinity purified by Primm Biotech using the peptide NH2-CKPLPDHV(Sp)IVE-COOH.

Plasmid ConstructsThe Flag-IKKβ (SSEE), Flag-IKKβ (SSAA), and HA-IκBα (SSAA) constructs were provided by C. Wu (NCI, Bethesda) and U. Siebenlist (NIAID, Bethesda), respectively. The HA-IκBα and IKKβ (K44A)-Flag plasmids were purchased from Addgene44, 45. The Flag-RPS3, GST-RPS3, HA-RPS3, VN-HA, NleH1-HA plasmids were described previously6, 9. The point mutants of RPS3 were generated by site-directed mutagenesis using the Quick Change Kit (Stratagene) with primers forward 5′-CTGCCTGACCACGTGGCCATTGTGGAACCCAAA-3′ and reverse 5′-TTTGGGTTCCACAATGGCCACGTGGTCAGGCAG-3′ for S209A. All mutants were verified by DNA sequencing.

Plasmid Constructs

The Flag-IKKβ (SSEE), Flag-IKKβ (SSAA), and HA-IκBα (SSAA) constructs were provided by C. Wu (NCI, Bethesda) and U. Siebenlist (NIAID, Bethesda), respectively. The HA-IκBα and IKKβ (K44A)-Flag plasmids were purchased from Addgene44, 45. The Flag-RPS3, GST-RPS3, HA-RPS3, VN-HA, NleH1-HA plasmids were described previously6, 9. The point mutants of RPS3 were generated by site-directed mutagenesis using the Quick Change Kit (Stratagene) with primers forward 5′-CTGCCTGACCACGTGGCCATTGTGGAACCCAAA-3′ and reverse 5′-TTTGGGTTCCACAATGGCCACGTGGTCAGGCAG-3′ for S209A. All mutants were verified by DNA sequencing.

32P in vivo LabelingHEK 293T cells were labeled with 2 mCi/ml 32P-orthophosphate (Perkin Elmer) in phosphate-free medium (Invitrogen) for 2 h. Cells were then left untreated or treated with TNF (50 ng/ml, R&D Systems) for indicated periods. Cell lysates were prepared and used for immunoprecipitations with RPS3 antibody.

32P in vivo Labeling

HEK 293T cells were labeled with 2 mCi/ml 32P-orthophosphate (Perkin Elmer) in phosphate-free medium (Invitrogen) for 2 h. Cells were then left untreated or treated with TNF (50 ng/ml, R&D Systems) for indicated periods. Cell lysates were prepared and used for immunoprecipitations with RPS3 antibody.

In Vitro Kinase AssayKinase-active recombinant IKKβ and IKKα proteins were purchased from Active Motif and Millipore, respectively. Bacterially purified glutathione S-transferase (GST), GST-IκBα (1-54), wild type, mutant GST-RPS3, or RPS3 proteins were used as substrates. The in vitro kinase assay was performed as previously described29. Briefly, enzyme (100 ng) and substrate (2 μg) were co-incubated in IKK reaction buffer (25 mM Tris–HCl [pH 8.0], 50 mM KCl, 10 mM MgCl2, 1 mM DTT, 1 mM Na3VO4, 1 mM ATP) or NleH1 reaction buffer (50 mM Tris-HCl [pH 7.6], 5 mM MgCl2, 1 mM DTT, 1 mM ATP) with 0.5 μCi 32P-γ-ATP (GE Healthcare) added at 37 °C for 30 min. The reactions were resolved by SDS–PAGE and visualized by autoradiography.

In Vitro Kinase Assay

Kinase-active recombinant IKKβ and IKKα proteins were purchased from Active Motif and Millipore, respectively. Bacterially purified glutathione S-transferase (GST), GST-IκBα (1-54), wild type, mutant GST-RPS3, or RPS3 proteins were used as substrates. The in vitro kinase assay was performed as previously described29. Briefly, enzyme (100 ng) and substrate (2 μg) were co-incubated in IKK reaction buffer (25 mM Tris–HCl [pH 8.0], 50 mM KCl, 10 mM MgCl2, 1 mM DTT, 1 mM Na3VO4, 1 mM ATP) or NleH1 reaction buffer (50 mM Tris-HCl [pH 7.6], 5 mM MgCl2, 1 mM DTT, 1 mM ATP) with 0.5 μCi 32P-γ-ATP (GE Healthcare) added at 37 °C for 30 min. The reactions were resolved by SDS–PAGE and visualized by autoradiography.

LC-MS/MS AnalysisGST or GST-RPS3 was incubated with recombinant IKKβ protein as described above in an in vitro kinase assay reaction conducted without 32P-γ-ATP labeling. The reaction was separated by SDS-PAGE, and the protein gel was stained with Colloidal Blue (Invitrogen). The corresponding protein fragments were excised and subjected to trypsin digestion and LC-MS/MS at the Yale Cancer Center Mass Spectrometry Resource (New Haven, CT).

LC-MS/MS Analysis

GST or GST-RPS3 was incubated with recombinant IKKβ protein as described above in an in vitro kinase assay reaction conducted without 32P-γ-ATP labeling. The reaction was separated by SDS-PAGE, and the protein gel was stained with Colloidal Blue (Invitrogen). The corresponding protein fragments were excised and subjected to trypsin digestion and LC-MS/MS at the Yale Cancer Center Mass Spectrometry Resource (New Haven, CT).

RNAi and TransfectionThe siRNA (sense-strand sequence) IKKα, 5′-AUGACAGAGAAUGAUCAUGUUCUGC -3′; IKKβ, 5′-GCAGCAAGGAGAACAGAGGUUAAUA -3′; IκBα, 5′-GAGCUCCGAGACUUUCGAGGAAAUA -3′; RPS3-3′ UTR, 5′-GGAUGUUGCUCUCUAAAGACC -3′ (Invitrogen). Transient transfection of siRNA and DNA constructs into Jurkat cells and 293T cells was described previously6.

RNAi and Transfection

The siRNA (sense-strand sequence) IKKα, 5′-AUGACAGAGAAUGAUCAUGUUCUGC -3′; IKKβ, 5′-GCAGCAAGGAGAACAGAGGUUAAUA -3′; IκBα, 5′-GAGCUCCGAGACUUUCGAGGAAAUA -3′; RPS3-3′ UTR, 5′-GGAUGUUGCUCUCUAAAGACC -3′ (Invitrogen). Transient transfection of siRNA and DNA constructs into Jurkat cells and 293T cells was described previously6.

Subcellular FractionationSubcellular fractionation was performed by differential centrifugation as previously described6. Briefly, cells were resuspended in ice-cold Buffer A (10 mM HEPES pH 7.9, 10 mM KCl, 1.5 mM MgCl2, 0.1 mM EDTA, 0.5 mM DTT, 0.4 % NP-40, 0.5 mM PMSF, complete protease inhibitor cocktail) at 4 °C for 5 min. Lysates were centrifuged at 4 °C, 500 × g for 3 min, and supernatants were collected as cytosolic fractions. Pellets were incubated in Buffer C (20 mM HEPES pH7.9, 420 mM NaCl, 1.5 mM MgCl2, 25% glycerol, 0.5 mM PMSF, 0.2 mM EDTA, 0.5 mM DTT, complete protease inhibitor cocktail) at 4 °C for 10 min. Supernatants were collected as nuclear fractions following a centrifuge at 4 °C, 13,800 × g for 10 min.

Subcellular Fractionation

Subcellular fractionation was performed by differential centrifugation as previously described6. Briefly, cells were resuspended in ice-cold Buffer A (10 mM HEPES pH 7.9, 10 mM KCl, 1.5 mM MgCl2, 0.1 mM EDTA, 0.5 mM DTT, 0.4 % NP-40, 0.5 mM PMSF, complete protease inhibitor cocktail) at 4 °C for 5 min. Lysates were centrifuged at 4 °C, 500 × g for 3 min, and supernatants were collected as cytosolic fractions. Pellets were incubated in Buffer C (20 mM HEPES pH7.9, 420 mM NaCl, 1.5 mM MgCl2, 25% glycerol, 0.5 mM PMSF, 0.2 mM EDTA, 0.5 mM DTT, complete protease inhibitor cocktail) at 4 °C for 10 min. Supernatants were collected as nuclear fractions following a centrifuge at 4 °C, 13,800 × g for 10 min.

Luciferase Reporter Gene AssaysLuciferase reporter gene assays were performed as previously described6. Briefly, cells were cotransfected at a ratio of 10:1 with various promoter-driven firefly luciferase constructs to the Renilla luciferase pTKRL plasmid, together with indicated plasmids. Cells were cultured for 1–2 days and then stimulated in triplicate before harvest. Lysates were analyzed using the Dual-Luciferase Kit (Promega).

Luciferase Reporter Gene Assays

Luciferase reporter gene assays were performed as previously described6. Briefly, cells were cotransfected at a ratio of 10:1 with various promoter-driven firefly luciferase constructs to the Renilla luciferase pTKRL plasmid, together with indicated plasmids. Cells were cultured for 1–2 days and then stimulated in triplicate before harvest. Lysates were analyzed using the Dual-Luciferase Kit (Promega).

Chromatin Immunoprecipitation (ChIP)ChIP assays was performed as previously described6. The primers used to amplify the promoter region adjacent to the κB sites of IL8 and NFKBIA, as well as ACTB have been described6.

Chromatin Immunoprecipitation (ChIP)

ChIP assays was performed as previously described6. The primers used to amplify the promoter region adjacent to the κB sites of IL8 and NFKBIA, as well as ACTB have been described6.

Immunofluorescence MicroscopyConfocal microscopy was performed as previously described6. Briefly, cells were fixed with 4 % paraformaldehyde in PBS and then Cellspin mounted onto slides. The fixed cells were then permeabilized with 0.05 % Triton X-100 in PBS and stained with FITC-conjugated rabbit anti-RPS3 antibodies (Primm Biotech), or AlexaFluor 594-conjugated rat anti-Flag antibodies (BD) for 40 min together with 1 μg/ml of Hoechst 33342 (Sigma) for 5 min at 25 °C. The slides were then rinsed with PBS three times and cover mounted for fluorescence microscopy.

Immunofluorescence Microscopy

Confocal microscopy was performed as previously described6. Briefly, cells were fixed with 4 % paraformaldehyde in PBS and then Cellspin mounted onto slides. The fixed cells were then permeabilized with 0.05 % Triton X-100 in PBS and stained with FITC-conjugated rabbit anti-RPS3 antibodies (Primm Biotech), or AlexaFluor 594-conjugated rat anti-Flag antibodies (BD) for 40 min together with 1 μg/ml of Hoechst 33342 (Sigma) for 5 min at 25 °C. The slides were then rinsed with PBS three times and cover mounted for fluorescence microscopy.

Immunoprecipitation and immunoblotThe cells were harvested and lysed on ice by 0.4 ml of the modified RIPA buffer (50 mM Tris-HCl [pH 7.4], 1% NP-40, 0.25% Na-deoxycholate, 150 mM NaCl, 1 mM EDTA, 1 mM PMSF, 1 mM Na3VO4, 1 mM NaF) supplemented with 1 × protease inhibitor cocktail (Roche) and 1 × phosphatase inhibitor cocktail set I (EMD Biosciences) for 30 min. The lysates were centrifuged at 10,000 × g at 4 °C for 10 min to remove insoluble material. After normalizing protein concentrations, lysates were subjected to immunoprecipitation by adding 10 mg/ml appropriate antibody plus 30 ml of protein G-agarose (Roche), and rotated for at least 2 h at 4°C. The precipitates were washed at least five times with cold lysis buffer followed by separation by SDS-PAGE under reduced and denaturing conditions. Nitrocellulose membranes were blocked in 5 % nonfat milk in 0.1 % PBS-Tween 20 (PBS-T), probed with specific antibodies as described previously6. For immunoblotting of phosphorylated proteins, gels were transferred to methanol-treated polyvinylidene chloride membranes, retreated with methanol, and dried for 30 min. Blots were blocked in 5 % bovine serum albumin in 0.1 % Tris buffered saline-Tween 20 (TBS-T), and probed with specific antibodies as described previously46. Bands were imaged by the Super Signaling system (Pierce) according to the manufacturer's instructions.

Immunoprecipitation and immunoblot

The cells were harvested and lysed on ice by 0.4 ml of the modified RIPA buffer (50 mM Tris-HCl [pH 7.4], 1% NP-40, 0.25% Na-deoxycholate, 150 mM NaCl, 1 mM EDTA, 1 mM PMSF, 1 mM Na3VO4, 1 mM NaF) supplemented with 1 × protease inhibitor cocktail (Roche) and 1 × phosphatase inhibitor cocktail set I (EMD Biosciences) for 30 min. The lysates were centrifuged at 10,000 × g at 4 °C for 10 min to remove insoluble material. After normalizing protein concentrations, lysates were subjected to immunoprecipitation by adding 10 mg/ml appropriate antibody plus 30 ml of protein G-agarose (Roche), and rotated for at least 2 h at 4°C. The precipitates were washed at least five times with cold lysis buffer followed by separation by SDS-PAGE under reduced and denaturing conditions. Nitrocellulose membranes were blocked in 5 % nonfat milk in 0.1 % PBS-Tween 20 (PBS-T), probed with specific antibodies as described previously6. For immunoblotting of phosphorylated proteins, gels were transferred to methanol-treated polyvinylidene chloride membranes, retreated with methanol, and dried for 30 min. Blots were blocked in 5 % bovine serum albumin in 0.1 % Tris buffered saline-Tween 20 (TBS-T), and probed with specific antibodies as described previously46. Bands were imaged by the Super Signaling system (Pierce) according to the manufacturer's instructions.

ELISAThe amount of IL-8 present in supernatants collected from Jurkat cell culture was measured using a Human Interleukin-8 ELISA Ready-SET-Go kit (eBioscience) according to the manufacturer's instructions.

The amount of IL-8 present in supernatants collected from Jurkat cell culture was measured using a Human Interleukin-8 ELISA Ready-SET-Go kit (eBioscience) according to the manufacturer's instructions.

Cell InfectionsHeLa cells were infected with E. coli O157:H7 or C. rodentium strains as described previously9.

Cell Infections

HeLa cells were infected with E. coli O157:H7 or C. rodentium strains as described previously9.

ImmunohistochemistryGnotobiotic piglets were infected with E. coli O157:H7 strains as described previously9. Spiral colon specimens were collected at necropsy and embedded in paraffin. Paraffin sectioning and immunohistochemical staining using phospho-RPS3 antibody were performed by Histoserv Inc.

Immunohistochemistry

Gnotobiotic piglets were infected with E. coli O157:H7 strains as described previously9. Spiral colon specimens were collected at necropsy and embedded in paraffin. Paraffin sectioning and immunohistochemical staining using phospho-RPS3 antibody were performed by Histoserv Inc.

Supplementary Material1

Supplementary Material
