---
pmid: '17537789'
title: Wnt/beta-catenin signaling regulates nephron induction during mouse kidney
  development.
authors:
- Park JS
- Valerius MT
- McMahon AP
journal: Development
year: '2007'
full_text_available: true
pmcid: PMC12378450
doi: 10.1242/dev.006155
---

# Wnt/beta-catenin signaling regulates nephron induction during mouse kidney development.
**Authors:** Park JS, Valerius MT, McMahon AP
**Journal:** Development (2007)
**DOI:** [10.1242/dev.006155](https://doi.org/10.1242/dev.006155)
**PMC:** [PMC12378450](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC12378450/)

## Abstract

1. Development. 2007 Jul;134(13):2533-9. doi: 10.1242/dev.006155. Epub 2007 May
30.

Wnt/beta-catenin signaling regulates nephron induction during mouse kidney 
development.

Park JS(1), Valerius MT, McMahon AP.

Author information:
(1)Department of Molecular and Cellular Biology, Harvard University, 16 Divinity 
Avenue, Cambridge, MA 02138, USA.

Mammalian nephrons form as a result of a complex morphogenesis and patterning of 
a simple epithelial precursor, the renal vesicle. Renal vesicles are established 
from a mesenchymal progenitor population in response to inductive signals. 
Several lines of evidence support the sequential roles of two Wnt family 
members, Wnt9b and Wnt4, in renal vesicle induction. Using genetic approaches to 
specifically manipulate the activity of beta-catenin within the mesenchymal 
progenitor pool in mice, we investigated the potential role of the canonical Wnt 
pathway in these inductive events. Progenitor-cell-specific removal of 
beta-catenin activity completely blocked both the formation of renal vesicles 
and the expected molecular signature of an earlier inductive response. By 
contrast, activation of stabilized beta-catenin in the same cell population 
causes ectopic expression of mesenchymal induction markers in vitro and 
functionally replaces the requirement for Wnt9b and Wnt4 in their inductive 
roles in vivo. Thus, canonical Wnt signaling is both necessary and sufficient 
for initiating and maintaining inductive pathways mediated by Wnt9b and Wnt4. 
However, the failure of induced mesenchyme with high levels of beta-catenin 
activity to form epithelial structures suggests that modulating canonical 
signaling may be crucial for the cellular transition to the renal vesicle.

DOI: 10.1242/dev.006155
PMID: 17537789 [Indexed for MEDLINE]

## Full Text

IntroductionEach nephron that forms during embryonic and fetal development follows a deeply conserved developmental program that, by birth, ensures fluid bodily homeostasis1. To generate a nephron, mesenchymal nephron progenitors (NPs) gradually cluster together into pretubular aggregates (PTAs), which in turn epithelialize into renal vesicles (RVs), later giving rise to the tubular Comma and S-shaped body nephron stages (CSB and SSB)2,3. It is during this process that the early nephron generates a proximal-to-distal (PD) axis, a symmetry-breaking event that over time produces ~24 functionally specialized cell-types spatially distributed along the adult nephron PD axis1,4. The proximal-most end of the adult nephron consists of podocytes that generate a physical filter interacting with capillaries in the glomerular tuft, while the remainder of the nephron is a tubular structure consisting of several broad segments—proximal tubule, descending loop of Henle, ascending Loop of Henle, macula densa, distal convoluted tubule (DCT), and connecting tubule (CNT), each divisible into subtypes and by function1,4–6. At the distal-most end, the nephron forms a luminal connection to drain urine into the collecting duct. While fate-mapping has shown all adult mouse nephron cells form from NPs7, the mechanisms underpinning the gradual specification of PD identities are poorly understood, particularly in humans.Identifying mechanisms underlying human nephron patterning requires in vitro systems that replicate the development of the nephron PD axis. Many studies have produced kidney organoids with nephron-like structures (nephroids), generated by directed differentiation from pluripotent stem cells. However, resulting nephroids primarily reflect developmental rather than mature cell states, and distal nephron segments are poorly represented8–10. To overcome these limitations, it is essential to understand and mimic nephron axial patterning in vitro.In vivo, distal and proximal nephron identities form sequentially as NPs are recruited into the PTA, RV, and CSB3. Several signal inputs regulate distal nephron development11–13. The distal domain is marked by transcription factors HNF1B and POU3F33,14, and their mouse orthologs regulate development of nascent nephron epithelia and maturation of distal segments of the loop of Henle, macula densa, and distal tubule15–17. Distal cell identities require an integration of dosed Wnt, Bmp, Notch, and Fgf signaling18–21 and perturbations to these pathways generate distinct phenotypes. However, interpretations of these phenotypes are hindered by the nephron’s complex morphogenetic program and dynamic gene expression patterns. Wnt9b likely initiates nephrogenesis by driving β-catenin-mediated differentiation and cell aggregation to form PTAs18,22–26. Thereafter, distal and proximal positional identities are sensitive to the dosage of β-catenin, as increasing or decreasing the transcriptional output of β-catenin distalizes and proximalizes nephrons, respectively27. While it is clear that β-catenin is central to nephron PD patterning, our previous work in mice shows that nephron cells can be further sensitized to β-catenin by simultaneously modulating Bmp or Pi3k signaling, with each combination generating distinct axial patterning outcomes27.In this study, using the WNT and BMP signal transduction pathways as a starting point, we develop an approach to generate hundreds of nephroids that reproducibly break their PD axis symmetry and generate defined nephron cell identities. Using a combination of single-cell sequencing, small-scale organoid screening, spatial transcriptomics, and pathway perturbations, we show that distal identities are dependent on conserved patterning programs in the human iPSC-derived nephron. By examining the expression dynamics of FGF8 in vivo during early nephron development, we link the ligand’s expression to the forming ascending loop of Henle and demonstrate that these cell fates depend on FGF signaling. Strikingly, distal fated nephron cells devoid of FGF signaling revert to generate proximal cell identities, a program that we in turn show is conditional on BMP signal transduction. The system as whole demonstrates considerable cell fate plasticity and provides a model for switching human nephron cell fates at will, while defining the spatiotemporal requirements for WNT, FGF, and BMP signaling during axial nephron patterning.

Introduction

Each nephron that forms during embryonic and fetal development follows a deeply conserved developmental program that, by birth, ensures fluid bodily homeostasis1. To generate a nephron, mesenchymal nephron progenitors (NPs) gradually cluster together into pretubular aggregates (PTAs), which in turn epithelialize into renal vesicles (RVs), later giving rise to the tubular Comma and S-shaped body nephron stages (CSB and SSB)2,3. It is during this process that the early nephron generates a proximal-to-distal (PD) axis, a symmetry-breaking event that over time produces ~24 functionally specialized cell-types spatially distributed along the adult nephron PD axis1,4. The proximal-most end of the adult nephron consists of podocytes that generate a physical filter interacting with capillaries in the glomerular tuft, while the remainder of the nephron is a tubular structure consisting of several broad segments—proximal tubule, descending loop of Henle, ascending Loop of Henle, macula densa, distal convoluted tubule (DCT), and connecting tubule (CNT), each divisible into subtypes and by function1,4–6. At the distal-most end, the nephron forms a luminal connection to drain urine into the collecting duct. While fate-mapping has shown all adult mouse nephron cells form from NPs7, the mechanisms underpinning the gradual specification of PD identities are poorly understood, particularly in humans.

Identifying mechanisms underlying human nephron patterning requires in vitro systems that replicate the development of the nephron PD axis. Many studies have produced kidney organoids with nephron-like structures (nephroids), generated by directed differentiation from pluripotent stem cells. However, resulting nephroids primarily reflect developmental rather than mature cell states, and distal nephron segments are poorly represented8–10. To overcome these limitations, it is essential to understand and mimic nephron axial patterning in vitro.

In vivo, distal and proximal nephron identities form sequentially as NPs are recruited into the PTA, RV, and CSB3. Several signal inputs regulate distal nephron development11–13. The distal domain is marked by transcription factors HNF1B and POU3F33,14, and their mouse orthologs regulate development of nascent nephron epithelia and maturation of distal segments of the loop of Henle, macula densa, and distal tubule15–17. Distal cell identities require an integration of dosed Wnt, Bmp, Notch, and Fgf signaling18–21 and perturbations to these pathways generate distinct phenotypes. However, interpretations of these phenotypes are hindered by the nephron’s complex morphogenetic program and dynamic gene expression patterns. Wnt9b likely initiates nephrogenesis by driving β-catenin-mediated differentiation and cell aggregation to form PTAs18,22–26. Thereafter, distal and proximal positional identities are sensitive to the dosage of β-catenin, as increasing or decreasing the transcriptional output of β-catenin distalizes and proximalizes nephrons, respectively27. While it is clear that β-catenin is central to nephron PD patterning, our previous work in mice shows that nephron cells can be further sensitized to β-catenin by simultaneously modulating Bmp or Pi3k signaling, with each combination generating distinct axial patterning outcomes27.

In this study, using the WNT and BMP signal transduction pathways as a starting point, we develop an approach to generate hundreds of nephroids that reproducibly break their PD axis symmetry and generate defined nephron cell identities. Using a combination of single-cell sequencing, small-scale organoid screening, spatial transcriptomics, and pathway perturbations, we show that distal identities are dependent on conserved patterning programs in the human iPSC-derived nephron. By examining the expression dynamics of FGF8 in vivo during early nephron development, we link the ligand’s expression to the forming ascending loop of Henle and demonstrate that these cell fates depend on FGF signaling. Strikingly, distal fated nephron cells devoid of FGF signaling revert to generate proximal cell identities, a program that we in turn show is conditional on BMP signal transduction. The system as whole demonstrates considerable cell fate plasticity and provides a model for switching human nephron cell fates at will, while defining the spatiotemporal requirements for WNT, FGF, and BMP signaling during axial nephron patterning.

ResultsSpontaneous nephron axial patterning in a malleable windowTo establish a system for controlling human nephron cell identities in vitro, we modified a kidney organoid protocol28 and identified a differentiation window for manipulating PD axial symmetry breaking during the PTA to RV transition. Briefly, human iPSCs are differentiated in monolayer culture for 7 days to mimic WNT-mediated posterior paraxial mesoderm induction and FGF-mediated intermediate mesoderm induction, before being transitioned into sheet-like 3D organoid cultures (Fig. 1a). At differentiation day 10 (DD10), these WT1+ organoids display synchronously epithelializing PTA-like structures forming apical-basal polarities (Fig. S1a–c). An apical surface positive for tight-junction protein ZO-1 faces inwards, while the outside of each PTA deposits a basal β-laminin+ basement membrane (Figs. 1b and S1b). At DD10, PTAs are positive for NP marker SIX1 and PAX2; in vivo SIX1 is expected to be broadly detected in NPs, PTAs, and distally polarized in RVs away from the point of NP recruitment, while PAX2 follows a similar pattern in NPs and PTAs but is strongly detected throughout RVs and then downregulated in forming podocytes3,14. These in vitro PTAs display uniform distribution of nephron induction transcription factors PAX8 and LEF1 (in vivo detected throughout the PTA and RV, with dynamics of being upregulated distally first3,14), and Notch ligand JAG1 (in vivo first detected in the distal PTA and thereafter following a proximal moving direction3,14). The in vitro PTAs are surrounded by MEIS1+ interstitium (Fig. S1a–c). Over a 2-day period, each nephroid thereafter elongates and morphologically begins to resemble RVs, displaying upregulation of JAG1 and epithelial cadherin, CDH1 (Fig. 1b). At this stage, a portion of nephrons break synchronicity and mimic the polarized proximal downregulation of WT1 observed in vivo, while maintaining JAG1 and CDH1. Compared to DD10 when 482/482 (100%) of nephroids are symmetric and do not yet display any evidence of localized downregulation of markers, on DD12, 36/368 (10%) nephroids display polarized downregulation of WT1, highlighting the first timepoint an emerging PD axis and asymmetry is observed in vitro (Fig. 1b). In this early mosaic state, the majority (90%) of nephroids maintain uniform levels of WT1 protein, but by DD14, 88/383 (23%) of nephroids have WT1- domains (Fig. 1b, c). When the PD axis develops in vivo, distinct CDH1+ distal, JAG1+ medial, and WT1+ proximal domains form (Fig. 1b); CDH1 marks precursors of the distal nephron, JAG1 marks proximal/medial nephron cells, and WT1 is found in podocytes and parietal epithelium, respectively. In vitro, however, DD14 nephroids only partially recapitulate this marker segregation, and the majority of nephrons— 295/383 (77%)—are positive for WT1 along the whole structure nephron structure. Additionally, while human SSBs establish sharp boundaries between each PD domain, CDH1, JAG1, and WT1 boundaries in organoid nephroids overlap. Consistent with the broad detection of WT1 at precursor stages (DD14), nephroids are broadly positive for podocyte marker MAFB and differentiate into large MAFB+ renal corpuscles (DD28), surrounded by a CD31+ vasculature-like networks; few GATA3+ distal tubules form (Fig. S1d–e). This indicates that the nephroids in this system display a bias towards podocyte-forming programs (Fig. 1d).Fig. 1Directed differentiation of kidney organoids for a proximal-to-distal fate switch.a (left) Schematic of modified Takasato protocol for iPSC-derived kidney organoids. (right) Brightfield images of DD10, DD12, and DD14 organoids. Scale bar: 500 microns. b Human kidney immunofluorescence stains for proximal, medial, and distal domain markers at the PTA, RV, and SSB stages. Corresponding stage-matched kidney organoids at DD10, DD12, and DD14 are stained for the same markers. Yellow arrows denote polarized nephrons. Biological replicates (n > 4) showed similar results. Scale bar: 50 microns. c Bar graph showing percentage of total nephrons per organoid symmetrical vs asymmetrical WT1 expression on DD10, DD12, and DD14 (n = 2). d Schematic of kidney organoid differentiation trajectory from DD10-DD28. e Immunofluorescence stains of human kidneys at the RV, CSB, SSB, and CLSN stages showing transcription factors marking the distal nephron domains. The SSB includes labeled positional relationships of precursors of the CNT (pCNT), DCT (pDT), loop of Henle (pLOH), proximal tubule (pPT), and podocyte (pPod). Scale bar: 50 microns. f Immunofluorescence stains of DD12 organoids showing distal transcription factor expression in polarized vs unpolarized nephrons. Biological replicates (n = 3) showed similar results. Scale bar: 50 microns. g Heatmaps of TPM value z-scores of nephron cell type markers at the SSB stage, expressed across 10 differentiation conditions in DD14 organoids (n = 2). Blue arrow denotes condition 8. h, i Immunofluorescence stains of DD14 control (condition 1) vs DE (condition 8) organoids showing proximal-distal marker expression. Biological replicates (n > 4) showed similar results. Scale bar: 50 microns. j Volcano plot of DESeq analysis comparing DD14 DMSO control and DE organoid total mRNA sequencing data (Wald test p-value, adjusted by FDR correction). Dashed lines show 0.05 p-value cutoff (horizontal) and ±1.5-fold change cutoff (vertical).In human nephrogenesis, distal nephron precursors in the early renal vesicle are marked by transcription factors HNF1B, POU3F3, and TFAP2A. HNF1B is detected at the distal-most end of the early renal vesicle and, together with POU3F3 and TFAP2A, is thought to mark early to late distal segments (Fig. 1e). As the SSB elongates, domains of these transcription factors shift, forming 3 distinct segments from the distal-most to medial SSB: HNF1B+/POU3F3+/TFAP2A+, HNF1B+/POU3F3+/TFAP2A- and HNF1Bhigh/POU3F3-/TFAP2A- regions. These segments correspond to putative precursors of the distal nephron, segment 3 of the proximal tubule, and segments 1 and 2 of the proximal tubule, respectively1. In contrast, as most nephroids have not broken symmetry at the RV stage (DD12; see Fig. 1c), few nephroids express these 3 distal transcription factors and rare triple HNF1B+/POU3F3+/TFAP2A+ domains are observed (Fig. 1f). Detection of TFAP2A precisely coincides with the early downregulation of WT1, where TFAP2A is specifically detected in WT1off cells, indicating the emergence of a distal pole, albeit infrequently, and use of WT1 as an axial patterning marker (Fig. S1f). This is consistent with the infrequent distal nephron phenotype seen at DD28. Overall, these views suggest that cell type outcomes are established during the initial stages of differentiation and highlight a time frame (DD10-DD12) when axial symmetry breaking occurs - generating polarized patterns of WT1, HNF1B, JAG1, POU3F3, TFAP2A along the PD axis - which is suitable for manipulating patterning and reading outcomes at DD14.Altering WNT and BMP signals shifts nephron axial patterning to distal programsTo generate distal axial nephron cell fates, we constructed a dosage-duration-timing matrix, increasing canonical WNT signaling and reducing BMP signaling during the PTA to RV transition. We have previously linked β-catenin signaling to positively regulating distal axial identities in the mouse nephron27, while BMP/SMAD reporters are active in proximal axial identities27,29.We activated WNT signaling using GSK3β inhibitor, CHIR99021 (CHIR), a well-studied WNT agonist, modulating dosages and duration, and timing dosing to avoid disrupting the mesenchymal-to-epithelial transition (MET)18,22,30. We inhibited BMP signaling using LDN 193189 dihydrochloride (LDN), a selective inhibitor of BMP receptors ALK2 and ALK3 that prevents phosphorylation of SMAD1/5/831. To promote NP epithelialization during the PTA-to-RV transition, we incorporated PI3K inhibitor, LY2940021 (LY29), from DD10 to DD1227,32. We assayed patterning outcomes at DD14 by whole-mount immunofluorescent stains for PD markers (in PD order: WT1, PODXL (also marks apical surfaces together with ZO1), JAG1, HNF1B, CDH1, POU3F3, TFAP2A) and by bulk RNA sequencing at DD14 and DD20 to determine long-term patterning, across these 2- and 3-inhibitor combinations (Fig. S1g).In combination, conditions 4, 6, 7, 8, 9, and 10 (Fig. S1g) displayed evidently larger TFAP2A+ domains relative to control (cond. 1), CHIR-only (cond. 2), LDN-only (cond. 3), and LY29-only (cond. 5) treatments (Fig. S1h). This agrees with bulk RNA sequencing data where, on their own, each small molecule inhibitor (SMI) only negligibly affected axial domain shifts (Fig. S1i). WNT activation from DD11-DD12 yielded a modest increase in distal domain emergence relative to control organoids (slight upregulation of POU3F3, HNF1B and TFAP2A), but this ineffectively suppressed podocyte (WT1+) differentiation (Fig. S1i). This is consistent with our previous ex vivo data demonstrating a transient and reversible effect of SMI pulse treatments when used solitarily27. To increase resolution to the precursor populations generated in each SMI condition, we assessed DD14 and DD20 organoid expression of genes (top 50) that are normally enriched in axially distributed PD nephron precursors in in vivo SSBs, as described in Lindström et al. 1 (Figs. 1g, and S1j, k). While no in vivo SSB precursor gene list was fully represented, meeting the cutoff criteria of TPM > 10, podocytes (37/50 genes), loop of Henle (38/50 genes) and distal tubule (37/50 genes) precursors had the highest representation in our organoids, while fewer parietal epithelium-, proximal tubule- and connecting tubule-enriched genes were expressed (31/50, 22/50 and 25/50 genes, respectively). Condition 8 (DD10-11 PI3K inhibition, DD11-12 PI3K inhibition, WNT activation, BMP inhibition), henceforth referred to as distal-enriched (DE), generated nephron-like precursors resembling those of the DCT, loop of Henle, and CNT at the SSB stage (DD14), and maintained expression of genes while simultaneously and suppressing podocyte-enriched genes until DD20 (Figs. 1g and S1j, k). Consistent with these findings, DE organoids expressed the highest level of TFAP2A and POU3F3 (Fig. S1l). Integration of PI3K inhibition, WNT activation, and BMP inhibition was thus identified as a strategy for generating DE nephroids.Direct comparisons between control and DE nephroids showed a remarkable switch from proximal to distal axial identities. DE nephroids lost their proximal/podocyte (WT1+) and gained distal (TFAP2A+/POU3F3+/HNF1B+) cell profiles (Fig. 1h–i). TFAP2A activation was systemic in DE organoids and was strongly co-detected with POU3F3 and HNF1B (Fig. 1h). While WT1 expression was prominent in control organoids, it was suppressed in DE organoids, which instead formed elongated CDH1+/JAG1low nephroid tubules (Fig. 1i). An equivalent direct comparison of the total mRNA signature of DD14 control and DE organoids (Supplementary Data 1) further illustrates this dramatic shift in cell identities. Control organoids express podocyte-enriched genes such as OLFM3, PODXL, MAFB, and IL1R1, while DE organoids are enriched for distal nephron markers, TFAP2A, MAL, IRX1, SIM2, BMP7, and SPON1 (Fig. 1j)1,33. Together, these data provide strong evidence that simultaneous activation of WNT/β-catenin and suppression of BMP/SMAD signal transduction synergistically drives nephroid distalization.A transient BMPOFF and WNTON state primes nephroid cells for distal nephron formationTo assess the axial shift of our DE nephroid model, we performed single-cell RNA sequencing to analyze the cellular composition of DD14 DE and control organoids. As expected, both control (8219 cells) and DE (9301 cells) organoids consisted of nephron-like (PAX2+) or interstitial-like (PDGFRA+) cells, with few off-target cell types (no detection of muscle marker, MYOD1) (Fig. S2a, b and Supplementary Data 2). Both nephron and interstitial progenitors are derived from the intermediate mesoderm34–36, indicating our organoid cell composition efficiently generates intermediate mesoderm and nephron cells. Control organoids contained a small subset of CDH5+ endothelial progenitors (Fig. S2a), which likely give rise to the CD31+ vasculature-like cells observed in DD28. Nephron-like cells were the dominant cell population in the organoids, with 80% (6650 cells) and 77% (7158 cells) of cells from control and DE organoids, respectively, being classified as nephron lineage (Fig. 2a).Fig. 2Single-cell RNA sequencing data shows nephron lineage divergence between control and DE organoids.a Annotated unsupervised clustering of PAX2+ nephron-like DD14 control and DE organoid cells represented in a UMAP. (Inset) Sample contribution of each condition. b Violin plot with annotations of select gene markers (top x-axis) showing expression levels (bottom x-axis) in the DD14 organoid dataset. The y-axis shows cluster numbers and sample origin (gray square – control organoids, pink square – DE organoids). c Volcano plot of DESeq analysis comparing DD14 DMSO control and DE organoid nephron scRNA sequencing data (Wald test p-value, adjusted by FDR correction). Dashed lines show 0.05 p-value cutoff (horizontal) and ±1.5-fold change cutoff (vertical). d Bar plot showing normalized percent contribution of cells expressing select nephron markers (log2FC > 0) separated by sample origin. Numbers in bars show raw cell counts. e Feature plots of select genes in the DD14 DE organoid dataset. f Immunofluorescence stain of week 16 human kidney from SSB-CLSN stage marking DCT/CNT precursor and loop of Henle/macula densa precursor segments. g Immunofluorescence stains of DD14 control and DE organoids showing expression of DCT/CNT precursor and loop of Henle/macula densa precursor markers. Biological replicates (n > 4) showed similar results. Yellow arrow – autofluorescence. Scale bar: 50 microns.When control and DE nephroid cells were merged and compared, few cells co-clustered with the other condition, suggesting minimal overlap in equivalent cell states between the 2 conditions (Fig. 2a, top right). Cell states clustered based on differentiation states and cellular processes defined by well-studied nephron patterning markers (Fig. 2a, b and Supplementary Data 2). Consistent with the almost binary switch observed by immunofluorescent stains, control and DE organoids are demarcated by WT1 and TFAP2A detection, respectively, underpinned by otherwise broad detection of genes expressed throughout the early nephron PD axis, such as PAX8, PAX2, and LHX1 (Figs. 2b and S1e). Clusters 3, 8, 12 and 17, comprised of control nephroid cells, displayed expression of induced nephron progenitor markers, BMPER, MEOX1 and SIX1 (Fig. 2b). Bona fide differentiation of podocytes (NPHS2+, PODXL+, SOST+, TCF21+) was only observed in controls; WT1+ DE cells did not express these markers, indicating podocyte differentiation was suppressed, alluding to rudimentary remnants of proximal-like cells. Instead, DE nephroid cells were strongly biased towards distal axial identities expressing TFAP2A (Fig. 2b), consistent with our immunofluorescence data. These TFAP2A+ cells co-expressed a range of early distal RV markers like EMX2, LHX1, and POU3F3, and late distal RV markers, GATA3, MAL, and CXCR41,3,5,6,14. A portion of DE cells also expressed differentiation markers (CLDN10, SIM2, and PAPPA2) normally enriched in the loop of Henle and macula densa of the in vivo nephron5,6 (clusters 13, 0, and 4; Fig. 2b).To scrutinize differences in gene expression of control and DE nephroid cells, we performed comparative differential gene expression analyses, identifying 480 control and 629 DE enriched genes (p < 0.05, log2FC > 1.5; Fig. 2c and Supplementary Data 1). These data corroborate conclusions that control organoids generate a podocyte-like gene signature positive for WT1, IL1R1, KIRREL3, OLFM3, PTPRO, PLA2R1, SYNPO, while DE nephroids adopt a broad and distinct distal-biased profile exhibiting expression of markers indicating early distal nephron development (LGR5, CALCR, MAL, TFAP2A, HNF1B, MECOM, GATA3) and precursor specification and maturation (TMEM52B, ERBB4, ESRRγ, PAPPA2) (Figs. 2c and S2f and Supplementary Data 1). The overt shift in differentiation towards distal nephron identities was further emphasized by quantifying cells expressing PD axial markers. When normalized for cell numbers (6650 control and 7158 DE cells), we examined the cells expressing each marker and quantified what percentage of cells expressing each gene came from control or DE nephroids; e.g., HNF1B is detected in 1610 out of 6650 control organoid cells, and in 6031 out of 7158 DE organoid cells, thus, when normalized to account for each condition’s total cell count, total HNF1B+ cells consist of 21% control and 79% DE cells. DE nephroid cells were strongly biased to develop distal nephron identities as they largely contributed to cells expressing distal nephron genes, while control nephroid cells generate proximal-like cells (Fig. 2d). At DD14, neither condition contained many HNF4A+ cells, but control nephroids had ~4x more HNF4A+ cells than DE organoids.Control nephroids displayed enrichment of BMP/SMAD pathway effectors ID1, ID2, ID3, and ID437 (Figs. 2c and S3a and Supplementary Data 1) compared to DE nephroids. To evaluate BMP activity, we assayed active SMAD signaling by immunolabelling dual phospho-SMAD1/5. Strong phospho-SMAD labeling was detected in control nephroids and in their surrounding interstitium, but this was absent in the DE organoids, consistent with LDN efficiently suppressing BMP/SMAD signaling in DE conditions (Fig. S3b). While the role of BMP/SMAD signaling is not fully established in the human nephron, labeling of developing human nephrons indicated phospho-SMAD activity in proximal PTAs, RVs, CSB, and in the proximal precursors of the SSB and CLSN (Fig. S3c), thus further correlating BMP signaling with human proximal axial nephron identities as suggested in mice38. Conversely, DE organoids showed upregulation of canonical WNT targets JUN, DLL1, CDH1, EMX2, SALL4, TFCP2L1, and WNT439–42, as expected from the addition of CHIR increasing WNT/β-catenin signaling (Fig. S3d).DE nephroids expressed genes indicating specification of maturing distal axial identities. Within HNF1B+, TFAP2A+, ERBB4+, MECOM+ DE nephroids cells, select cells were also positive for PAPPA2, ESRRγ, GATA3, and TMEM52B (Figs. 2e and S2f). In developing human nephrons, MECOM marks the entire distal nephron domain in the SSB, GATA3 and TMEM52B mark distal precursors of the CNT and DCT at the distal-most parts of the nephron, while the maturing ascending loop of Henle and macula densa precursors are marked by PAPPA2, ESRRγ and ERBB4; HNF1B and TFAP2A are included for reference (Fig. 2f). ESRRγ, a receptor that acts as a transcriptional activator in the absence of a bound ligand, marks loop of Henle/macula densa precursors43 and is detected in their nuclei in the SSB in a mosaic pattern. The receptor ERBB4 is first detected at the SSB stage in the distal limb. Its detection coincides with that of metalloprotease, PAPPA2, a putative marker of loop of Henle and macula densa precursors1,44. Using these as anchors to scrutinize the DE nephroid differentiation, immunolabeling demonstrated that DE nephroids are shifted to a distal MECOM+/TMEM52B+ cell state that is broadly co-labeled with loop of Henle receptor ERBB4, and mosaic detection of maturation transcription factor ESRRγ and macula densa/loop of Henle marker, PAPPA2 (Fig. 2g). These markers are rare in control nephroids and suggest that the DE nephroids are driven towards a distal tubule fate with a further sub-specification towards the distal tubule, macula densa, and thick ascending limb of the loop of Henle. Interestingly, both organoid conditions display sporadic GATA3+ domains. This specification agrees with unbiased GO term analyses of genes upregulated in DD14 DE organoids (log2FC > 1.5) (Fig. S3e and Supplementary Data 1); DE genes include those associated with nephron epithelium development, calcium ion binding, and ion transmembrane transporter activity, consistent with functions associated with the distal nephron epithelia. Of note, FGF receptor binding was highlighted as a predicted enriched molecular function in DE organoids, attributed to significant upregulation of FGF8, FGF10, and FGF18 (Log2FC = 3.421, 3.128, and 5.288, respectively), indicating elevated FGF signaling in DE organoids relative to controls.To ensure the DE protocol is robustly reproducible, we performed experiments replacing either the PI3K inhibitor LY294002 with GDC-0941, a structurally different and potent PI3K inhibitor, or replaced SMAD inhibitor LDN-193189 with Dorsomorphin, a BMP inhibitor that targets BMP receptors, ALK2, ALK3, and ALK645–47. Both conditions replicated results of the DE protocol, displaying a global shift to distal axial identities marked by elongated distal TFAP2A+, CDH1+ tubules with apical PODXL+ surfaces, positive for ERBB4, and increased abundance of PAPPA2+ cells (Fig. S4a). To further benchmark the DE protocol, we generated and distalized kidney organoids from a different iPSC HNF4A-YFP reporter line48. These nephroids also distalized and displayed near-global expansion of HNF1B, POU3F3, and TFAP2A axial identities with co-activation of β-catenin target LEF1, and mosaic expression of GATA3, while downregulating podocyte (WT1, MAFB), and proximal tubule (HNF4A) precursor markers (Fig. S4b–e). In contrast, controls were bias towards forming WT1+/MAFB+/ZO1high podocyte-like cells and few HNF1B+/POU3F3+/TFAP2A+ domains. Our validation experiments show that the DE protocol is adaptable to different cell lines and SMIs.These observations support the hypothesis that driving distal cell identities (WNT/β-catenin activation) and suppressing proximal cell differentiation (BMP/SMAD inhibition) generates distal nephron-like cells. Moreover, these data present a distal nephron-like organoid model enriched with loop of Henle and DCT precursor-like cells.Distalized nephrons differentiate to loop of Henle fatesTo directly identify the closest in vivo equivalents of DE nephroid cell states, we generated an in vivo transcriptome framework by reanalyzing previous single-cell RNA sequencing capturing human nephrogenesis data from week 14 (11,992 cells) and week 17 (4,772 cells) human kidneys1,49, processed as described in Schnell et al32. In these data, NPs (MEOX1+/CITED1+) develop into podocyte (MAFB+), parietal epithelium (PLAT+), proximal tubule (HNF4A+/SLC3A1+), and early distal tubule precursors (HNF1B+/POU3F3+/TFAP2A+) that differentiate into loop of Henle (SLC12A1+/PAPPA2+/CLDN10+) and a combined cluster consisting of DCT and CNT cells (TMEM52B+/GATA3+/EMX1+) (Figs. 3a and S5a, b and Supplementary Data 2). Using these as a reference, we mapped control and DE nephroid transcriptomes to the most similar in vivo cell RNA profiles (Fig. 3a). DE nephroids closely resembled distal precursor cells, with a minor WT1+ NP-like cell population remaining in these organoids. Strikingly, DE nephroid cells also mapped to loop of Henle precursors, but not DCT and CNT precursors (Figs. 3a and S5c). In contrast, control nephroid cells closely corresponded to PTA and early podocyte cells, and to a lesser degree, the earliest PT precursors, where proximal and distal cell fates have not fully diverged (Figs. 3a and S5c). Cell type prediction scores (see ‘Methods’) between nephroid and in vivo counterparts showed that DE organoids most closely resemble the distal nephron precursor state and loop of Henle precursors, while control nephroids resemble the PTA-like state and early podocytes (Fig. 3b). There is therefore a strong prediction that DE nephroid cells are in an early distal nephron identity programmed towards a loop of Henle-like state.Fig. 3Characterizations of distal nephron segmentation in distal-enriched organoids.a Schematic showing reference mapping of DD14 organoid scRNA sequencing dataset to reanalyzed week 14 and week 17 (4772 cells) merged human nephron and ureteric scRNA sequencing data1,49. (Right) UMAP of human kidney (reference) overlaid with DD14 control and DE organoid mapping. b Bar plot showing number of organoid cells mapping to each predicted cell type, split by condition. Numbers above bars denote prediction score for each cell type. c Immunofluorescence stain of DD14 control and DE organoids showing detection of IRX1. Biological replicates (n = 3) showed similar results. Scale bar: 50 microns. d–e Immunofluorescent stains of DD20 control and DE organoids showing detection of select loop of Henle/macula densa precursor and podocyte precursor markers. Biological replicates (n = 3) showed similar results. Scale bar: 50 microns. Abbreviations: NP nephron progenitors, IP induced progenitors, PTA pretubular aggregate, pod podocytes, PEC parietal epithelial cells, dist prec distal nephron precursors, PT prec proximal tubule precursors, PT proximal tubule, LOH loop of Henle, DCT/CNT distal convoluted tubule/connecting tubule.To increase resolution to primed differentiation programs in the DE model and their transcriptional drivers, we analyzed expression of all known human transcription factors in in vivo nephrons (approach described in Kim et al.5). We identified 39 distal nephron-enriched transcription factors that have either very low or no expression in other nephron cell types (see 'Methods') (Fig. S5d); IRX1, IRX2, TFAP2B, SIM2, IRX6, TFEB and NRK are enriched in loop of Henle/macula densa precursors, EMX1, GATA3, PRDM16 and NR0B2 are enriched in distal convoluted tubule and connecting tubule precursors, and GATA2, ELF5 and EHF are enriched in the ureteric epithelium (UE). Several of these transcription factors are known to be distal nephron-enriched, and co-expression of Irx1, Irx2, and Sim2 in loop of Henle/macula densa precursors is conserved in P0 mouse nephrons5, indicating function. Analysis of transcription factors in our DE nephroids showed that loop of Henle-enriched transcription factors, IRX1, IRX2, TFAP2B, and SIM2, are restricted to PAPPA2+ DE nephroid clusters 13, 0, and 4 (Fig. S5e). Of the DCT-enriched transcription factors, only GATA3 was detected. EMX1, which is normally expressed in the DCT and CNT precursors of the CLSN, was absent. Transcription factors that are predicted to be expressed in more mature loop of Henle segments, IRX6, TFEB, and NRK, and DCT segments, NR0B2, PRDM16, and EMX1, were not detected. Their lack of detection in DD14 nephroids, which resemble the SSB stage, was expected. We observed strong co-expression of known loop of Henle/macula densa precursor-enriched markers, IRX1 and SIM2, in a subset of DE nephroid cells (Fig. S5f). Validation experiments show IRX1 is broadly present in DD14 DE nephroids (Fig. 3c), indicating these cells are initially driven to a distal axial state primed to thereafter generate a subset of more mature nephron cell types.Our bulk RNA sequencing of DD14 and DD20 DE organoids show that the gene signatures of loop of Henle and macula densa lineages are maintained and reinforced, indicated by large increases in expression of PAPPA2, ERBB4, and ascending loop of Henle/macula densa solute carrier, SLC12A1 (Fig. S5g), while transcription factors IRX1 and SIM2 expression were maintained (Fig. S5g). Validation by immunostaining in DD20 organoids shows strong detection of IRX1, PAPPA2, and SLC12A1 in DE organoids (Fig. 3d–e). In contrast, maturation markers of the DCT, such as EMX1, and calcium transport-associated TRPV5 and CALB1 were undetected in both DD14 and DD20 DE organoids, while GATA3 expression remained the same between the two time-points, and SALL3 expression was downregulated (Fig. S5g). Podocyte signatures did not recover by DD20, where DE organoids displayed few MAFB+ cells, relative to controls (Fig. 3e).To identify differentiation endpoints, we performed single-nucleus RNA sequencing of DD28 control and DE organoids. Consistent with DD14 analyses, control and DE organoids largely consisted of PAX8+ nephron- and PDGFRA+ interstitial-like cells (Fig. S6a, b) Control organoids contained rare CDH5 (endothelium) and MYOD1 (muscle) transcripts, and RENIN+ granular-like cell populations, the latter consistent juxtaglomerular cells, while DE organoids displayed a minor population of NEUROD1+ neuron-like cells, common in long-term cultures of kidney organoids9 (Fig. S6a, b). Direct comparison of control (3,947 nuclei) and DE (1,457 nuclei) PAX8+ nephroid lineages revealed a similar pattern to DD14 organoids, where both conditions have distinct spatial clustering (Fig. 4a). Control organoids were enriched with MAFB+ podocytes, CLDN1+ parietal epithelial cells and HNF4A+ proximal tubule cells, whereas DE nephroids have a strong enrichment of PAPPA2+ loop of Henle/macula densa cells, TMEM52B+ distal tubules and a small population of GATA3+ nephron connecting tubules (Fig. 4b). Immunofluorescent labeling confirmed that DE organoids consisted primarily of TFAP2A+/SLC12A1+/ERBB4+/CDH1+ cells that were mosaically positive for PAPPA2 and IRX1 (Figs. 4c, d and S6c). Markers of other nephron axial identities, e.g., proximal tubule (HNF4A+), podocyte (MAFB+), and CNT (GATA3+), were rare (Fig. 4d, e). In contrast, control organoids were enriched with MAFB+ glomeruli-like cells abutting CD31+ vasculature and RENIN+/GATA3+ juxtaglomerular-like cells, contained HNF1B+/HNF4A+ proximal tubule-like cells, and displayed sparse PAPPA2 and IRX1 labeling (Fig. 4d, e); RENIN+ cells were not observed in DE organoids.Fig. 4Single-nucleus RNA sequencing shows endpoint differentiation cell-types in DD28 control and DE organoids.a Annotated unsupervised clustering of integrated PAX2+ nephron-like DD28 control and DE organoid cells represented in a UMAP. (Inset) Sample contribution of each condition. b Feature plots of nephron marker expression in DD28 organoids showing podocyte-, PEC-, proximal tubule-, loop of Henle-, and distal tubule-like cells. c–e Immunofluorescent stains of DD28 control and DE organoids showing detection of glomerular, proximal tubule, loop of Henle/macula densa, and DCT precursor markers. Biological replicates (n > 4) showed similar results. Scale bar: 50 microns. f UMAP of adult human kidney snRNAseq data (44,774 cells) with ‘subclass.l1’ annotations, including nephron (pink border) and ureteric epithelia (gray border). g UMAP of adult human kidney (reference) overlaid with DD28 control and DE organoid reference mapping. h Bar plot showing number of organoid cells mapping to each predicted cell type in control (top) and DE (bottom) organoids. Numbers above bars denote prediction score for each cell type. i Feature plots showing detection of select TAL-enriched ion transporters in DD28 organoids. j Illustration of cell type enrichment in DD28 DE organoids.To determine how endpoint differentiation nephroids relate to the in vivo human kidney, we compared them to published adult human kidney snRNAseq data from the Kidney Precision Medicine project (KPMP; 44,774 cells)50, which contains annotated cell types, including podocytes (POD), parietal epithelial cells (PEC), proximal tubule (PT), the descending thin limb (DTL), ascending thin limb (ATL), and thick ascending limb (TAL) of the loop of Henle, distal convoluted tubule (DCT), nephron connecting tubule (CNT), principle cells (PC), intercalated cells (IC), and papillary tip epithelial cells (PapE) (Fig. 4f). Multimodal reference mapping with control and DE nephroid cells predicted the majority of control organoid cells map to adult human podocytes, PT, PEC, and DTL (Fig. 4g, h). DE organoids, however, mainly map to TAL cells—note that the authors’ system of annotation groups the macula densa into this classification50—and DTL. DESeq analysis of control and DE organoids further corroborate that DE organoids are enriched with distal nephron markers (MECOM, TFCP2L1, and TFAP2B), loop of Henle/macula densa specific markers (SLC12A1, CASR, SIM2, and PAPPA2), while control organoids displayed sustained expression of genes associated with mature podocyte markers, ITIH5 and THSD7A43,49,50 (Fig. S6d). As expected for cells mapping to adult functional cell populations, DE organoids displayed robust expression of physiology-imparting genes encoding distal-nephron-specific transporters enriched in adult human TAL cells, such as SLC12A1 (NKCC2), CLDN16, KCNJ1 (ROMK), CASR, SCNN1A (ENaC), CLNCKA (ClC-Ka), CLCNKB (ClC-Kb), BSND (Barttin), SLC9A3 (NHE3), and glucagon receptor, GCGR (Figs. 4i and S6e). Loss-of-function mutations to proteins NKCC2, ROMK, ClC-Kb, Barttin, and CASR are linked to Bartter Syndrome, a genetic condition that causes ion imbalance and an inability to reabsorb salt51. Multimodal reference mapping is a powerful predictive tool and can be limited by reference dependency and differences in sequencing platforms used. To mitigate this, we matched snRNA-to-snRNA-seq in vitro/in vitro mapping to minimally bias conditions for these comparisons. GO term analyses of top differentially expressed genes in whole DD28 DE organoids further underscore distal nephron-linked functions of ionic exchange (Fig. S6f).Together, these data demonstrate a model for a nearly systemic axial switch in nephron cell type identities.Modulation of FGF and BMP signaling shifts domain identities along the proximal-distal nephron axisThe DE organoid approach demonstrates that simultaneous activation of WNT and inhibition of BMP/SMAD signaling generates cells primed for distal nephron fates. DE organoids significantly increased FGFR1, MAP7, and MAP2 expression at DD14 (Fig. 2c and Supplementary Data 1), highlighting a potential role of FGF/MAPK signaling in distalization. Mouse studies have shown that Fgf8 is required for distal nephron cell survival and differentiation, it is a direct target of Wnt/β-catenin, and Fgf8 hypomorphs fail to form Slc12a1+ loop of Henle cells, indicating a non-redundant role, thus raising the possibility that FGF signaling is responsible for distalization and formation of TAL-like cells in DE organoids22,52. Consistent with this view, FGF8 was strongly upregulated in DD14 DE nephroids (log2FC ~ 3) and thereafter downregulated by DD28 (Fig. 5a). In vivo, single-cell RNA sequencing data showed FGF8 is transiently upregulated during distal nephron development in a sequence closely matching TFAP2A expression, but FGF8 is turned off soon after SSB stages (Fig. 5b). To resolve the exact normal sequence of FGF8 up and downregulation in vivo, we generated spatial transcriptomic data using seqFISH53, capturing human kidney and nephron development54. In vivo, FGF8 is detected in TFAP2A+ distal cells in the early RV, remains there into the late RV, and then expands within the TFAP2A+ domain (Fig. 5c). By SSB stages, FGF8 is detected in the forming IRX1+ domain and is downregulated in distal-most GATA3+ cells (Fig. S7a). By late SSB stages, FGF8 is downregulated throughout distal TFAP2A+ cells aside from the IRX1+ and PAPPA2+ loop of Henle precursors, indicating a shift of expression within the TFAP2A+ domain. FGF8 transcripts are not detected in HNF4A+ proximal and MAFB+ podocyte precursors (Fig. S7a). By the CLSN stage, FGF8 is turned off in the distal nephron (Fig. 5c) and is thereafter not detected in adult stages (Figure S7b).Fig. 5Dynamics of FGF, WNT, and BMP signaling in nephron segmentation.a Feature plot of FGF8 expression in DD14 and DD28 organoids, split by condition. b Feature plot of FGF8 in the human kidney dataset. Inset: TFAP2A detection in the distal nephron. c SeqFISH data of week 16 human kidney showing detection of FGF8 and TFAP2A from early RV to CLSN. d–g (left) Schematics of SMI treatment and nephron segmentation in DD14 organoids and (right) immunofluorescent stains showing detection of DCT/CNT, proximal tubule, loop of Henle/macula densa, and podocyte precursor markers. Biological replicates (n > 4) showed similar results. Scale bars: 50 microns. h Bar plots of percent quantification of fluorescence detection of podocyte, proximal tubule, and distal nephron markers in select nephroids of DD14 control, DE, DE + FGFR1i + BMPi organoids (n = 2). i Schematic of nephron segments and the corresponding pathways that drive their development. j Proposed schematic of select signaling pathway interactions in the developing mammalian nephron.Other FGF ligands, FGF9 and FGF10, which belong to FGF9 and FGF7 subfamilies, respectively55, are also expressed during in vivo nephron patterning. In the developing nephron, FGF9 is upregulated distally as FGF8 is being downregulated, forming a largely mutually exclusive expression pattern. FGF9 is also strongly upregulated in podocytes and parietal epithelium but is absent from proximal tubule precursors. FGF10 follows a similar onset of expression to FGF8 but then persists in the DCT and CNT precursors; FGF10 is also expressed in nephron progenitors (Figs. 5b and S7c). In the adult in vivo nephron, FGF9 expression is detected in distal tubular nephron cell-types, in podocytes, and in the parietal epithelium, but FGF10 expression is downregulated. In vitro, FGF9 and FGF10 follow similar expression patterns to in vivo. FGF10 is upregulated in TFAP2A+ cells in DE nephroids in a pattern closely mimicking FGF8 but is, as expected, infrequently detected in control nephroids that are primarily comprised of early podocytes and proximal fated cells (Fig. S7c). Mirroring the expression of FGF10, FGF9 is enriched in podocytes in control nephroids following an in vivo-like pattern and is detected at lower levels in distalized DE nephroids. By DD28, FGF10 is detected in very few cells, but FGF9 remains detected in control organoid podocyte-like cells and DE organoid distal nephroid cells, suggesting it is eventually upregulated in distal nephroids (Fig. S7d). These data suggest the control and DE nephroids therefore largely mimic in vivo adult nephron expression, where FGF9 is detected, and FGF10 is not detected.Distalized organoid nephrons also express FGF18 at DD14, but this FGF ligand, belonging to the FGF8 ligand subfamily55, is not normally detected in vivo during early nephrogenesis. FGF18 is expressed in the adult kidney in the ascending thin limb (ATL) and papillary tip epithelial cells (PapE) of the nephron (Fig. S7d). Unlike the in vivo pattern, where FGF18 is expressed only after early nephrogenesis, in nephroids, FGF18 is downregulated after DD14 and is not observed at DD28.In mice, Fgf-receptor 1 (Fgfr1) is described as the likely receptor for Fgf8, as it is expressed in the nascent nephron and tubular epithelia of the nephron52. In human nephrons, our data suggest FGFR1 and FGFR2 are broadly expressed from progenitor stages (FGFR2 is infrequently detected in podocytes), while FGFR3 is primarily detected in the maturing distal and proximal tubule epithelium – after the proposed action of FGF8 (Fig. S7e). In vitro, in DD14 organoids, FGFR1 expression is detected in both control and DE organoids, though at higher levels in the latter (Fig. S7e and 2c Supplementary Data 1). Lower levels of FGFR2 and FGFR3 are also detected in all clusters of both conditions, and FGFR3 is infrequently detected in the nephron progenitor-like cells (Fig. S7e). Given the overall dynamics of FGF-receptor expression, FGFR1 being detected in TFAP2A+ distalizing cells matching expression of FGF8, it being the preferred receptor for FGF852,56, these lend credibility to FGFR1 being a candidate receptor for FGF signaling in the early distal nephron.To determine whether FGF signaling is required for human distal nephron development, we inhibited FGFR1 between early RV- to SSB-like stages (DD11-DD14) using SU-5402, a potent FGFR1 inhibitor57,58, and compared forming nephroids with control and DE conditions (Fig. 5d–f). This condition generated a Wnt/β-catenin active state unable to signal via FGFR1. Strikingly, inhibition of FGFR1 in DE organoids reversed the DE phenotype (Fig. 5f, h). Distal nephron marker TFAP2A was reduced to control organoid levels while HNF4A+ proximal tubule domains formed - and did so more frequently than in controls - next to MAFB+/WT1+ podocytes (Fig. 5f, h; compare to d and e) while IRX1 cell populations were visually diminished. The HNF4A+ cells developed within HNF1Bhigh domains, strongly reminiscent of normal in vivo development, where high HNF1B expression in the medial SSB precedes and is required for HNF4A expression16,32,59. These data show that inhibition of FGFR1 in DE organoids results in a next-to-complete reversal of the DE state back to a proximal axis-biased differentiation outcome.Given that HNF4A+ proximal tubule precursors and MAFB+ podocytes normally form in cells positive for phosphorylated SMAD (Fig. S3c) and these developed in the DE + FGFR1-inhibitor condition, we asked whether the switch is dependent on SMAD signaling driving proximal axial cell-fates in a setting where FGF8 is rendered incapable of distalizing nephroids due to inhibition of FGFR1. To test this, we extended the LDN-mediated SMAD inhibitor treatment so that DE organoids simultaneously received the FGFR1 inhibitor and the SMAD inhibitor until DD14 (Fig. 5g). Remarkably, this again flipped the differentiation outcome so that TFAP2A and IRX1 were detected throughout nephroids, WT1 and MAFB were lost, and HNF1B was again detected at homogenous levels (as in DE conditions) in the absence of HNF4A (GATA3 expression was relatively low in all conditions) (Fig. 5g, h). These observations suggest that inhibition of FGF in our DE organoids reenables a BMP/SMAD-dependent proximal-driving program that, if blocked, reverts to a distal program.Of note, we observed only a modest expansion of the HNF4A domain in control organoids treated with SU-5402 for the same duration as the other experiments, despite expressing FGFR1 (Fig. S8a–c), indicating that it is the DE condition that generates a FGF8high state driving a distal fate. When analyzing differentiation endpoints (DD28) of DE and DE + FGFR1 inhibitor (SU-5402) treated organoids, the latter developed podocytes as well as loop of Henle identities, showing that the axial shifts imposed by DD14 generate nephroids that thereafter differentiate towards their respective primed states (Fig. S8d–e).To validate phenotypes caused by SU-5402 treatment, we used another selective and potent inhibitor of FGFR1(PD-166866)60. As expected, PD-166866-treated DE organoids (DD11-DD14) formed few TFAP2A+ distal cells while proximal tubule-like cell and podocyte identities reemerged (Fig. S8f). Control organoids treated with PD-166866 for the same duration behaved similarly to controls administered SU-5402, with the exception of a more prominent loss of IRX1+ cells in the former, alluding to lack of loop of Henle-like precursors (Fig. S8b and S8g).Our and others’ work has demonstrated proximal axial cell specification requires Notch signaling20,27,61–63. To determine if the distal-enhanced (β-cateninON/FGFON + BMPOFF) state requires normal Notch signaling, we blocked Notch receptor activity through γ-secretase inhibition (adding DAPT), from DD11-DD14. Inhibition of Notch in vivo is detrimental to the development of all nephron cell types20 and we made similar observations in our DD14 organoids, where the morphology of DD14 control organoids treated with DAPT resembled that of RV-like DD12 control organoids, suggesting arrested differentiation, and nephroids were small, round, and displayed WT1+ cells devoid of MAFB, no detectable HNF4A, and lack of IRX1 expression—normally detectable at DD14 (Fig S8h). DE organoids supplemented with DAPT still developed into TFAP2A+ distalized nephrons with GATA3, devoid of HNF4A and MAFB, and as expected, a JAG1high/HNF1Bhigh state failed to form (Fig S8i). Overall, these data indicate that our organoid model faithfully recapitulates the Notch loss-of-function phenotypes and that the distal program can be directed in the absence of Notch activity.Here, we show that the nephron PD polarity is a highly tunable epithelial axis, and molecular functions required for nephron cell type specification can be disentangled in organoid models, leading to the production of reproducible protocols for differentiating cell fates along the PD axis (Fig. 5i, j). Our model proposes pathway-mediated differentiation to form the different nephron segments and maintain boundaries of their domains (Fig. 5j).

Spontaneous nephron axial patterning in a malleable windowTo establish a system for controlling human nephron cell identities in vitro, we modified a kidney organoid protocol28 and identified a differentiation window for manipulating PD axial symmetry breaking during the PTA to RV transition. Briefly, human iPSCs are differentiated in monolayer culture for 7 days to mimic WNT-mediated posterior paraxial mesoderm induction and FGF-mediated intermediate mesoderm induction, before being transitioned into sheet-like 3D organoid cultures (Fig. 1a). At differentiation day 10 (DD10), these WT1+ organoids display synchronously epithelializing PTA-like structures forming apical-basal polarities (Fig. S1a–c). An apical surface positive for tight-junction protein ZO-1 faces inwards, while the outside of each PTA deposits a basal β-laminin+ basement membrane (Figs. 1b and S1b). At DD10, PTAs are positive for NP marker SIX1 and PAX2; in vivo SIX1 is expected to be broadly detected in NPs, PTAs, and distally polarized in RVs away from the point of NP recruitment, while PAX2 follows a similar pattern in NPs and PTAs but is strongly detected throughout RVs and then downregulated in forming podocytes3,14. These in vitro PTAs display uniform distribution of nephron induction transcription factors PAX8 and LEF1 (in vivo detected throughout the PTA and RV, with dynamics of being upregulated distally first3,14), and Notch ligand JAG1 (in vivo first detected in the distal PTA and thereafter following a proximal moving direction3,14). The in vitro PTAs are surrounded by MEIS1+ interstitium (Fig. S1a–c). Over a 2-day period, each nephroid thereafter elongates and morphologically begins to resemble RVs, displaying upregulation of JAG1 and epithelial cadherin, CDH1 (Fig. 1b). At this stage, a portion of nephrons break synchronicity and mimic the polarized proximal downregulation of WT1 observed in vivo, while maintaining JAG1 and CDH1. Compared to DD10 when 482/482 (100%) of nephroids are symmetric and do not yet display any evidence of localized downregulation of markers, on DD12, 36/368 (10%) nephroids display polarized downregulation of WT1, highlighting the first timepoint an emerging PD axis and asymmetry is observed in vitro (Fig. 1b). In this early mosaic state, the majority (90%) of nephroids maintain uniform levels of WT1 protein, but by DD14, 88/383 (23%) of nephroids have WT1- domains (Fig. 1b, c). When the PD axis develops in vivo, distinct CDH1+ distal, JAG1+ medial, and WT1+ proximal domains form (Fig. 1b); CDH1 marks precursors of the distal nephron, JAG1 marks proximal/medial nephron cells, and WT1 is found in podocytes and parietal epithelium, respectively. In vitro, however, DD14 nephroids only partially recapitulate this marker segregation, and the majority of nephrons— 295/383 (77%)—are positive for WT1 along the whole structure nephron structure. Additionally, while human SSBs establish sharp boundaries between each PD domain, CDH1, JAG1, and WT1 boundaries in organoid nephroids overlap. Consistent with the broad detection of WT1 at precursor stages (DD14), nephroids are broadly positive for podocyte marker MAFB and differentiate into large MAFB+ renal corpuscles (DD28), surrounded by a CD31+ vasculature-like networks; few GATA3+ distal tubules form (Fig. S1d–e). This indicates that the nephroids in this system display a bias towards podocyte-forming programs (Fig. 1d).Fig. 1Directed differentiation of kidney organoids for a proximal-to-distal fate switch.a (left) Schematic of modified Takasato protocol for iPSC-derived kidney organoids. (right) Brightfield images of DD10, DD12, and DD14 organoids. Scale bar: 500 microns. b Human kidney immunofluorescence stains for proximal, medial, and distal domain markers at the PTA, RV, and SSB stages. Corresponding stage-matched kidney organoids at DD10, DD12, and DD14 are stained for the same markers. Yellow arrows denote polarized nephrons. Biological replicates (n > 4) showed similar results. Scale bar: 50 microns. c Bar graph showing percentage of total nephrons per organoid symmetrical vs asymmetrical WT1 expression on DD10, DD12, and DD14 (n = 2). d Schematic of kidney organoid differentiation trajectory from DD10-DD28. e Immunofluorescence stains of human kidneys at the RV, CSB, SSB, and CLSN stages showing transcription factors marking the distal nephron domains. The SSB includes labeled positional relationships of precursors of the CNT (pCNT), DCT (pDT), loop of Henle (pLOH), proximal tubule (pPT), and podocyte (pPod). Scale bar: 50 microns. f Immunofluorescence stains of DD12 organoids showing distal transcription factor expression in polarized vs unpolarized nephrons. Biological replicates (n = 3) showed similar results. Scale bar: 50 microns. g Heatmaps of TPM value z-scores of nephron cell type markers at the SSB stage, expressed across 10 differentiation conditions in DD14 organoids (n = 2). Blue arrow denotes condition 8. h, i Immunofluorescence stains of DD14 control (condition 1) vs DE (condition 8) organoids showing proximal-distal marker expression. Biological replicates (n > 4) showed similar results. Scale bar: 50 microns. j Volcano plot of DESeq analysis comparing DD14 DMSO control and DE organoid total mRNA sequencing data (Wald test p-value, adjusted by FDR correction). Dashed lines show 0.05 p-value cutoff (horizontal) and ±1.5-fold change cutoff (vertical).In human nephrogenesis, distal nephron precursors in the early renal vesicle are marked by transcription factors HNF1B, POU3F3, and TFAP2A. HNF1B is detected at the distal-most end of the early renal vesicle and, together with POU3F3 and TFAP2A, is thought to mark early to late distal segments (Fig. 1e). As the SSB elongates, domains of these transcription factors shift, forming 3 distinct segments from the distal-most to medial SSB: HNF1B+/POU3F3+/TFAP2A+, HNF1B+/POU3F3+/TFAP2A- and HNF1Bhigh/POU3F3-/TFAP2A- regions. These segments correspond to putative precursors of the distal nephron, segment 3 of the proximal tubule, and segments 1 and 2 of the proximal tubule, respectively1. In contrast, as most nephroids have not broken symmetry at the RV stage (DD12; see Fig. 1c), few nephroids express these 3 distal transcription factors and rare triple HNF1B+/POU3F3+/TFAP2A+ domains are observed (Fig. 1f). Detection of TFAP2A precisely coincides with the early downregulation of WT1, where TFAP2A is specifically detected in WT1off cells, indicating the emergence of a distal pole, albeit infrequently, and use of WT1 as an axial patterning marker (Fig. S1f). This is consistent with the infrequent distal nephron phenotype seen at DD28. Overall, these views suggest that cell type outcomes are established during the initial stages of differentiation and highlight a time frame (DD10-DD12) when axial symmetry breaking occurs - generating polarized patterns of WT1, HNF1B, JAG1, POU3F3, TFAP2A along the PD axis - which is suitable for manipulating patterning and reading outcomes at DD14.

Spontaneous nephron axial patterning in a malleable window

To establish a system for controlling human nephron cell identities in vitro, we modified a kidney organoid protocol28 and identified a differentiation window for manipulating PD axial symmetry breaking during the PTA to RV transition. Briefly, human iPSCs are differentiated in monolayer culture for 7 days to mimic WNT-mediated posterior paraxial mesoderm induction and FGF-mediated intermediate mesoderm induction, before being transitioned into sheet-like 3D organoid cultures (Fig. 1a). At differentiation day 10 (DD10), these WT1+ organoids display synchronously epithelializing PTA-like structures forming apical-basal polarities (Fig. S1a–c). An apical surface positive for tight-junction protein ZO-1 faces inwards, while the outside of each PTA deposits a basal β-laminin+ basement membrane (Figs. 1b and S1b). At DD10, PTAs are positive for NP marker SIX1 and PAX2; in vivo SIX1 is expected to be broadly detected in NPs, PTAs, and distally polarized in RVs away from the point of NP recruitment, while PAX2 follows a similar pattern in NPs and PTAs but is strongly detected throughout RVs and then downregulated in forming podocytes3,14. These in vitro PTAs display uniform distribution of nephron induction transcription factors PAX8 and LEF1 (in vivo detected throughout the PTA and RV, with dynamics of being upregulated distally first3,14), and Notch ligand JAG1 (in vivo first detected in the distal PTA and thereafter following a proximal moving direction3,14). The in vitro PTAs are surrounded by MEIS1+ interstitium (Fig. S1a–c). Over a 2-day period, each nephroid thereafter elongates and morphologically begins to resemble RVs, displaying upregulation of JAG1 and epithelial cadherin, CDH1 (Fig. 1b). At this stage, a portion of nephrons break synchronicity and mimic the polarized proximal downregulation of WT1 observed in vivo, while maintaining JAG1 and CDH1. Compared to DD10 when 482/482 (100%) of nephroids are symmetric and do not yet display any evidence of localized downregulation of markers, on DD12, 36/368 (10%) nephroids display polarized downregulation of WT1, highlighting the first timepoint an emerging PD axis and asymmetry is observed in vitro (Fig. 1b). In this early mosaic state, the majority (90%) of nephroids maintain uniform levels of WT1 protein, but by DD14, 88/383 (23%) of nephroids have WT1- domains (Fig. 1b, c). When the PD axis develops in vivo, distinct CDH1+ distal, JAG1+ medial, and WT1+ proximal domains form (Fig. 1b); CDH1 marks precursors of the distal nephron, JAG1 marks proximal/medial nephron cells, and WT1 is found in podocytes and parietal epithelium, respectively. In vitro, however, DD14 nephroids only partially recapitulate this marker segregation, and the majority of nephrons— 295/383 (77%)—are positive for WT1 along the whole structure nephron structure. Additionally, while human SSBs establish sharp boundaries between each PD domain, CDH1, JAG1, and WT1 boundaries in organoid nephroids overlap. Consistent with the broad detection of WT1 at precursor stages (DD14), nephroids are broadly positive for podocyte marker MAFB and differentiate into large MAFB+ renal corpuscles (DD28), surrounded by a CD31+ vasculature-like networks; few GATA3+ distal tubules form (Fig. S1d–e). This indicates that the nephroids in this system display a bias towards podocyte-forming programs (Fig. 1d).Fig. 1Directed differentiation of kidney organoids for a proximal-to-distal fate switch.a (left) Schematic of modified Takasato protocol for iPSC-derived kidney organoids. (right) Brightfield images of DD10, DD12, and DD14 organoids. Scale bar: 500 microns. b Human kidney immunofluorescence stains for proximal, medial, and distal domain markers at the PTA, RV, and SSB stages. Corresponding stage-matched kidney organoids at DD10, DD12, and DD14 are stained for the same markers. Yellow arrows denote polarized nephrons. Biological replicates (n > 4) showed similar results. Scale bar: 50 microns. c Bar graph showing percentage of total nephrons per organoid symmetrical vs asymmetrical WT1 expression on DD10, DD12, and DD14 (n = 2). d Schematic of kidney organoid differentiation trajectory from DD10-DD28. e Immunofluorescence stains of human kidneys at the RV, CSB, SSB, and CLSN stages showing transcription factors marking the distal nephron domains. The SSB includes labeled positional relationships of precursors of the CNT (pCNT), DCT (pDT), loop of Henle (pLOH), proximal tubule (pPT), and podocyte (pPod). Scale bar: 50 microns. f Immunofluorescence stains of DD12 organoids showing distal transcription factor expression in polarized vs unpolarized nephrons. Biological replicates (n = 3) showed similar results. Scale bar: 50 microns. g Heatmaps of TPM value z-scores of nephron cell type markers at the SSB stage, expressed across 10 differentiation conditions in DD14 organoids (n = 2). Blue arrow denotes condition 8. h, i Immunofluorescence stains of DD14 control (condition 1) vs DE (condition 8) organoids showing proximal-distal marker expression. Biological replicates (n > 4) showed similar results. Scale bar: 50 microns. j Volcano plot of DESeq analysis comparing DD14 DMSO control and DE organoid total mRNA sequencing data (Wald test p-value, adjusted by FDR correction). Dashed lines show 0.05 p-value cutoff (horizontal) and ±1.5-fold change cutoff (vertical).

Directed differentiation of kidney organoids for a proximal-to-distal fate switch.

a (left) Schematic of modified Takasato protocol for iPSC-derived kidney organoids. (right) Brightfield images of DD10, DD12, and DD14 organoids. Scale bar: 500 microns. b Human kidney immunofluorescence stains for proximal, medial, and distal domain markers at the PTA, RV, and SSB stages. Corresponding stage-matched kidney organoids at DD10, DD12, and DD14 are stained for the same markers. Yellow arrows denote polarized nephrons. Biological replicates (n > 4) showed similar results. Scale bar: 50 microns. c Bar graph showing percentage of total nephrons per organoid symmetrical vs asymmetrical WT1 expression on DD10, DD12, and DD14 (n = 2). d Schematic of kidney organoid differentiation trajectory from DD10-DD28. e Immunofluorescence stains of human kidneys at the RV, CSB, SSB, and CLSN stages showing transcription factors marking the distal nephron domains. The SSB includes labeled positional relationships of precursors of the CNT (pCNT), DCT (pDT), loop of Henle (pLOH), proximal tubule (pPT), and podocyte (pPod). Scale bar: 50 microns. f Immunofluorescence stains of DD12 organoids showing distal transcription factor expression in polarized vs unpolarized nephrons. Biological replicates (n = 3) showed similar results. Scale bar: 50 microns. g Heatmaps of TPM value z-scores of nephron cell type markers at the SSB stage, expressed across 10 differentiation conditions in DD14 organoids (n = 2). Blue arrow denotes condition 8. h, i Immunofluorescence stains of DD14 control (condition 1) vs DE (condition 8) organoids showing proximal-distal marker expression. Biological replicates (n > 4) showed similar results. Scale bar: 50 microns. j Volcano plot of DESeq analysis comparing DD14 DMSO control and DE organoid total mRNA sequencing data (Wald test p-value, adjusted by FDR correction). Dashed lines show 0.05 p-value cutoff (horizontal) and ±1.5-fold change cutoff (vertical).

In human nephrogenesis, distal nephron precursors in the early renal vesicle are marked by transcription factors HNF1B, POU3F3, and TFAP2A. HNF1B is detected at the distal-most end of the early renal vesicle and, together with POU3F3 and TFAP2A, is thought to mark early to late distal segments (Fig. 1e). As the SSB elongates, domains of these transcription factors shift, forming 3 distinct segments from the distal-most to medial SSB: HNF1B+/POU3F3+/TFAP2A+, HNF1B+/POU3F3+/TFAP2A- and HNF1Bhigh/POU3F3-/TFAP2A- regions. These segments correspond to putative precursors of the distal nephron, segment 3 of the proximal tubule, and segments 1 and 2 of the proximal tubule, respectively1. In contrast, as most nephroids have not broken symmetry at the RV stage (DD12; see Fig. 1c), few nephroids express these 3 distal transcription factors and rare triple HNF1B+/POU3F3+/TFAP2A+ domains are observed (Fig. 1f). Detection of TFAP2A precisely coincides with the early downregulation of WT1, where TFAP2A is specifically detected in WT1off cells, indicating the emergence of a distal pole, albeit infrequently, and use of WT1 as an axial patterning marker (Fig. S1f). This is consistent with the infrequent distal nephron phenotype seen at DD28. Overall, these views suggest that cell type outcomes are established during the initial stages of differentiation and highlight a time frame (DD10-DD12) when axial symmetry breaking occurs - generating polarized patterns of WT1, HNF1B, JAG1, POU3F3, TFAP2A along the PD axis - which is suitable for manipulating patterning and reading outcomes at DD14.

Altering WNT and BMP signals shifts nephron axial patterning to distal programsTo generate distal axial nephron cell fates, we constructed a dosage-duration-timing matrix, increasing canonical WNT signaling and reducing BMP signaling during the PTA to RV transition. We have previously linked β-catenin signaling to positively regulating distal axial identities in the mouse nephron27, while BMP/SMAD reporters are active in proximal axial identities27,29.We activated WNT signaling using GSK3β inhibitor, CHIR99021 (CHIR), a well-studied WNT agonist, modulating dosages and duration, and timing dosing to avoid disrupting the mesenchymal-to-epithelial transition (MET)18,22,30. We inhibited BMP signaling using LDN 193189 dihydrochloride (LDN), a selective inhibitor of BMP receptors ALK2 and ALK3 that prevents phosphorylation of SMAD1/5/831. To promote NP epithelialization during the PTA-to-RV transition, we incorporated PI3K inhibitor, LY2940021 (LY29), from DD10 to DD1227,32. We assayed patterning outcomes at DD14 by whole-mount immunofluorescent stains for PD markers (in PD order: WT1, PODXL (also marks apical surfaces together with ZO1), JAG1, HNF1B, CDH1, POU3F3, TFAP2A) and by bulk RNA sequencing at DD14 and DD20 to determine long-term patterning, across these 2- and 3-inhibitor combinations (Fig. S1g).In combination, conditions 4, 6, 7, 8, 9, and 10 (Fig. S1g) displayed evidently larger TFAP2A+ domains relative to control (cond. 1), CHIR-only (cond. 2), LDN-only (cond. 3), and LY29-only (cond. 5) treatments (Fig. S1h). This agrees with bulk RNA sequencing data where, on their own, each small molecule inhibitor (SMI) only negligibly affected axial domain shifts (Fig. S1i). WNT activation from DD11-DD12 yielded a modest increase in distal domain emergence relative to control organoids (slight upregulation of POU3F3, HNF1B and TFAP2A), but this ineffectively suppressed podocyte (WT1+) differentiation (Fig. S1i). This is consistent with our previous ex vivo data demonstrating a transient and reversible effect of SMI pulse treatments when used solitarily27. To increase resolution to the precursor populations generated in each SMI condition, we assessed DD14 and DD20 organoid expression of genes (top 50) that are normally enriched in axially distributed PD nephron precursors in in vivo SSBs, as described in Lindström et al. 1 (Figs. 1g, and S1j, k). While no in vivo SSB precursor gene list was fully represented, meeting the cutoff criteria of TPM > 10, podocytes (37/50 genes), loop of Henle (38/50 genes) and distal tubule (37/50 genes) precursors had the highest representation in our organoids, while fewer parietal epithelium-, proximal tubule- and connecting tubule-enriched genes were expressed (31/50, 22/50 and 25/50 genes, respectively). Condition 8 (DD10-11 PI3K inhibition, DD11-12 PI3K inhibition, WNT activation, BMP inhibition), henceforth referred to as distal-enriched (DE), generated nephron-like precursors resembling those of the DCT, loop of Henle, and CNT at the SSB stage (DD14), and maintained expression of genes while simultaneously and suppressing podocyte-enriched genes until DD20 (Figs. 1g and S1j, k). Consistent with these findings, DE organoids expressed the highest level of TFAP2A and POU3F3 (Fig. S1l). Integration of PI3K inhibition, WNT activation, and BMP inhibition was thus identified as a strategy for generating DE nephroids.Direct comparisons between control and DE nephroids showed a remarkable switch from proximal to distal axial identities. DE nephroids lost their proximal/podocyte (WT1+) and gained distal (TFAP2A+/POU3F3+/HNF1B+) cell profiles (Fig. 1h–i). TFAP2A activation was systemic in DE organoids and was strongly co-detected with POU3F3 and HNF1B (Fig. 1h). While WT1 expression was prominent in control organoids, it was suppressed in DE organoids, which instead formed elongated CDH1+/JAG1low nephroid tubules (Fig. 1i). An equivalent direct comparison of the total mRNA signature of DD14 control and DE organoids (Supplementary Data 1) further illustrates this dramatic shift in cell identities. Control organoids express podocyte-enriched genes such as OLFM3, PODXL, MAFB, and IL1R1, while DE organoids are enriched for distal nephron markers, TFAP2A, MAL, IRX1, SIM2, BMP7, and SPON1 (Fig. 1j)1,33. Together, these data provide strong evidence that simultaneous activation of WNT/β-catenin and suppression of BMP/SMAD signal transduction synergistically drives nephroid distalization.

Altering WNT and BMP signals shifts nephron axial patterning to distal programs

To generate distal axial nephron cell fates, we constructed a dosage-duration-timing matrix, increasing canonical WNT signaling and reducing BMP signaling during the PTA to RV transition. We have previously linked β-catenin signaling to positively regulating distal axial identities in the mouse nephron27, while BMP/SMAD reporters are active in proximal axial identities27,29.

We activated WNT signaling using GSK3β inhibitor, CHIR99021 (CHIR), a well-studied WNT agonist, modulating dosages and duration, and timing dosing to avoid disrupting the mesenchymal-to-epithelial transition (MET)18,22,30. We inhibited BMP signaling using LDN 193189 dihydrochloride (LDN), a selective inhibitor of BMP receptors ALK2 and ALK3 that prevents phosphorylation of SMAD1/5/831. To promote NP epithelialization during the PTA-to-RV transition, we incorporated PI3K inhibitor, LY2940021 (LY29), from DD10 to DD1227,32. We assayed patterning outcomes at DD14 by whole-mount immunofluorescent stains for PD markers (in PD order: WT1, PODXL (also marks apical surfaces together with ZO1), JAG1, HNF1B, CDH1, POU3F3, TFAP2A) and by bulk RNA sequencing at DD14 and DD20 to determine long-term patterning, across these 2- and 3-inhibitor combinations (Fig. S1g).

In combination, conditions 4, 6, 7, 8, 9, and 10 (Fig. S1g) displayed evidently larger TFAP2A+ domains relative to control (cond. 1), CHIR-only (cond. 2), LDN-only (cond. 3), and LY29-only (cond. 5) treatments (Fig. S1h). This agrees with bulk RNA sequencing data where, on their own, each small molecule inhibitor (SMI) only negligibly affected axial domain shifts (Fig. S1i). WNT activation from DD11-DD12 yielded a modest increase in distal domain emergence relative to control organoids (slight upregulation of POU3F3, HNF1B and TFAP2A), but this ineffectively suppressed podocyte (WT1+) differentiation (Fig. S1i). This is consistent with our previous ex vivo data demonstrating a transient and reversible effect of SMI pulse treatments when used solitarily27. To increase resolution to the precursor populations generated in each SMI condition, we assessed DD14 and DD20 organoid expression of genes (top 50) that are normally enriched in axially distributed PD nephron precursors in in vivo SSBs, as described in Lindström et al. 1 (Figs. 1g, and S1j, k). While no in vivo SSB precursor gene list was fully represented, meeting the cutoff criteria of TPM > 10, podocytes (37/50 genes), loop of Henle (38/50 genes) and distal tubule (37/50 genes) precursors had the highest representation in our organoids, while fewer parietal epithelium-, proximal tubule- and connecting tubule-enriched genes were expressed (31/50, 22/50 and 25/50 genes, respectively). Condition 8 (DD10-11 PI3K inhibition, DD11-12 PI3K inhibition, WNT activation, BMP inhibition), henceforth referred to as distal-enriched (DE), generated nephron-like precursors resembling those of the DCT, loop of Henle, and CNT at the SSB stage (DD14), and maintained expression of genes while simultaneously and suppressing podocyte-enriched genes until DD20 (Figs. 1g and S1j, k). Consistent with these findings, DE organoids expressed the highest level of TFAP2A and POU3F3 (Fig. S1l). Integration of PI3K inhibition, WNT activation, and BMP inhibition was thus identified as a strategy for generating DE nephroids.

Direct comparisons between control and DE nephroids showed a remarkable switch from proximal to distal axial identities. DE nephroids lost their proximal/podocyte (WT1+) and gained distal (TFAP2A+/POU3F3+/HNF1B+) cell profiles (Fig. 1h–i). TFAP2A activation was systemic in DE organoids and was strongly co-detected with POU3F3 and HNF1B (Fig. 1h). While WT1 expression was prominent in control organoids, it was suppressed in DE organoids, which instead formed elongated CDH1+/JAG1low nephroid tubules (Fig. 1i). An equivalent direct comparison of the total mRNA signature of DD14 control and DE organoids (Supplementary Data 1) further illustrates this dramatic shift in cell identities. Control organoids express podocyte-enriched genes such as OLFM3, PODXL, MAFB, and IL1R1, while DE organoids are enriched for distal nephron markers, TFAP2A, MAL, IRX1, SIM2, BMP7, and SPON1 (Fig. 1j)1,33. Together, these data provide strong evidence that simultaneous activation of WNT/β-catenin and suppression of BMP/SMAD signal transduction synergistically drives nephroid distalization.

A transient BMPOFF and WNTON state primes nephroid cells for distal nephron formationTo assess the axial shift of our DE nephroid model, we performed single-cell RNA sequencing to analyze the cellular composition of DD14 DE and control organoids. As expected, both control (8219 cells) and DE (9301 cells) organoids consisted of nephron-like (PAX2+) or interstitial-like (PDGFRA+) cells, with few off-target cell types (no detection of muscle marker, MYOD1) (Fig. S2a, b and Supplementary Data 2). Both nephron and interstitial progenitors are derived from the intermediate mesoderm34–36, indicating our organoid cell composition efficiently generates intermediate mesoderm and nephron cells. Control organoids contained a small subset of CDH5+ endothelial progenitors (Fig. S2a), which likely give rise to the CD31+ vasculature-like cells observed in DD28. Nephron-like cells were the dominant cell population in the organoids, with 80% (6650 cells) and 77% (7158 cells) of cells from control and DE organoids, respectively, being classified as nephron lineage (Fig. 2a).Fig. 2Single-cell RNA sequencing data shows nephron lineage divergence between control and DE organoids.a Annotated unsupervised clustering of PAX2+ nephron-like DD14 control and DE organoid cells represented in a UMAP. (Inset) Sample contribution of each condition. b Violin plot with annotations of select gene markers (top x-axis) showing expression levels (bottom x-axis) in the DD14 organoid dataset. The y-axis shows cluster numbers and sample origin (gray square – control organoids, pink square – DE organoids). c Volcano plot of DESeq analysis comparing DD14 DMSO control and DE organoid nephron scRNA sequencing data (Wald test p-value, adjusted by FDR correction). Dashed lines show 0.05 p-value cutoff (horizontal) and ±1.5-fold change cutoff (vertical). d Bar plot showing normalized percent contribution of cells expressing select nephron markers (log2FC > 0) separated by sample origin. Numbers in bars show raw cell counts. e Feature plots of select genes in the DD14 DE organoid dataset. f Immunofluorescence stain of week 16 human kidney from SSB-CLSN stage marking DCT/CNT precursor and loop of Henle/macula densa precursor segments. g Immunofluorescence stains of DD14 control and DE organoids showing expression of DCT/CNT precursor and loop of Henle/macula densa precursor markers. Biological replicates (n > 4) showed similar results. Yellow arrow – autofluorescence. Scale bar: 50 microns.When control and DE nephroid cells were merged and compared, few cells co-clustered with the other condition, suggesting minimal overlap in equivalent cell states between the 2 conditions (Fig. 2a, top right). Cell states clustered based on differentiation states and cellular processes defined by well-studied nephron patterning markers (Fig. 2a, b and Supplementary Data 2). Consistent with the almost binary switch observed by immunofluorescent stains, control and DE organoids are demarcated by WT1 and TFAP2A detection, respectively, underpinned by otherwise broad detection of genes expressed throughout the early nephron PD axis, such as PAX8, PAX2, and LHX1 (Figs. 2b and S1e). Clusters 3, 8, 12 and 17, comprised of control nephroid cells, displayed expression of induced nephron progenitor markers, BMPER, MEOX1 and SIX1 (Fig. 2b). Bona fide differentiation of podocytes (NPHS2+, PODXL+, SOST+, TCF21+) was only observed in controls; WT1+ DE cells did not express these markers, indicating podocyte differentiation was suppressed, alluding to rudimentary remnants of proximal-like cells. Instead, DE nephroid cells were strongly biased towards distal axial identities expressing TFAP2A (Fig. 2b), consistent with our immunofluorescence data. These TFAP2A+ cells co-expressed a range of early distal RV markers like EMX2, LHX1, and POU3F3, and late distal RV markers, GATA3, MAL, and CXCR41,3,5,6,14. A portion of DE cells also expressed differentiation markers (CLDN10, SIM2, and PAPPA2) normally enriched in the loop of Henle and macula densa of the in vivo nephron5,6 (clusters 13, 0, and 4; Fig. 2b).To scrutinize differences in gene expression of control and DE nephroid cells, we performed comparative differential gene expression analyses, identifying 480 control and 629 DE enriched genes (p < 0.05, log2FC > 1.5; Fig. 2c and Supplementary Data 1). These data corroborate conclusions that control organoids generate a podocyte-like gene signature positive for WT1, IL1R1, KIRREL3, OLFM3, PTPRO, PLA2R1, SYNPO, while DE nephroids adopt a broad and distinct distal-biased profile exhibiting expression of markers indicating early distal nephron development (LGR5, CALCR, MAL, TFAP2A, HNF1B, MECOM, GATA3) and precursor specification and maturation (TMEM52B, ERBB4, ESRRγ, PAPPA2) (Figs. 2c and S2f and Supplementary Data 1). The overt shift in differentiation towards distal nephron identities was further emphasized by quantifying cells expressing PD axial markers. When normalized for cell numbers (6650 control and 7158 DE cells), we examined the cells expressing each marker and quantified what percentage of cells expressing each gene came from control or DE nephroids; e.g., HNF1B is detected in 1610 out of 6650 control organoid cells, and in 6031 out of 7158 DE organoid cells, thus, when normalized to account for each condition’s total cell count, total HNF1B+ cells consist of 21% control and 79% DE cells. DE nephroid cells were strongly biased to develop distal nephron identities as they largely contributed to cells expressing distal nephron genes, while control nephroid cells generate proximal-like cells (Fig. 2d). At DD14, neither condition contained many HNF4A+ cells, but control nephroids had ~4x more HNF4A+ cells than DE organoids.Control nephroids displayed enrichment of BMP/SMAD pathway effectors ID1, ID2, ID3, and ID437 (Figs. 2c and S3a and Supplementary Data 1) compared to DE nephroids. To evaluate BMP activity, we assayed active SMAD signaling by immunolabelling dual phospho-SMAD1/5. Strong phospho-SMAD labeling was detected in control nephroids and in their surrounding interstitium, but this was absent in the DE organoids, consistent with LDN efficiently suppressing BMP/SMAD signaling in DE conditions (Fig. S3b). While the role of BMP/SMAD signaling is not fully established in the human nephron, labeling of developing human nephrons indicated phospho-SMAD activity in proximal PTAs, RVs, CSB, and in the proximal precursors of the SSB and CLSN (Fig. S3c), thus further correlating BMP signaling with human proximal axial nephron identities as suggested in mice38. Conversely, DE organoids showed upregulation of canonical WNT targets JUN, DLL1, CDH1, EMX2, SALL4, TFCP2L1, and WNT439–42, as expected from the addition of CHIR increasing WNT/β-catenin signaling (Fig. S3d).DE nephroids expressed genes indicating specification of maturing distal axial identities. Within HNF1B+, TFAP2A+, ERBB4+, MECOM+ DE nephroids cells, select cells were also positive for PAPPA2, ESRRγ, GATA3, and TMEM52B (Figs. 2e and S2f). In developing human nephrons, MECOM marks the entire distal nephron domain in the SSB, GATA3 and TMEM52B mark distal precursors of the CNT and DCT at the distal-most parts of the nephron, while the maturing ascending loop of Henle and macula densa precursors are marked by PAPPA2, ESRRγ and ERBB4; HNF1B and TFAP2A are included for reference (Fig. 2f). ESRRγ, a receptor that acts as a transcriptional activator in the absence of a bound ligand, marks loop of Henle/macula densa precursors43 and is detected in their nuclei in the SSB in a mosaic pattern. The receptor ERBB4 is first detected at the SSB stage in the distal limb. Its detection coincides with that of metalloprotease, PAPPA2, a putative marker of loop of Henle and macula densa precursors1,44. Using these as anchors to scrutinize the DE nephroid differentiation, immunolabeling demonstrated that DE nephroids are shifted to a distal MECOM+/TMEM52B+ cell state that is broadly co-labeled with loop of Henle receptor ERBB4, and mosaic detection of maturation transcription factor ESRRγ and macula densa/loop of Henle marker, PAPPA2 (Fig. 2g). These markers are rare in control nephroids and suggest that the DE nephroids are driven towards a distal tubule fate with a further sub-specification towards the distal tubule, macula densa, and thick ascending limb of the loop of Henle. Interestingly, both organoid conditions display sporadic GATA3+ domains. This specification agrees with unbiased GO term analyses of genes upregulated in DD14 DE organoids (log2FC > 1.5) (Fig. S3e and Supplementary Data 1); DE genes include those associated with nephron epithelium development, calcium ion binding, and ion transmembrane transporter activity, consistent with functions associated with the distal nephron epithelia. Of note, FGF receptor binding was highlighted as a predicted enriched molecular function in DE organoids, attributed to significant upregulation of FGF8, FGF10, and FGF18 (Log2FC = 3.421, 3.128, and 5.288, respectively), indicating elevated FGF signaling in DE organoids relative to controls.To ensure the DE protocol is robustly reproducible, we performed experiments replacing either the PI3K inhibitor LY294002 with GDC-0941, a structurally different and potent PI3K inhibitor, or replaced SMAD inhibitor LDN-193189 with Dorsomorphin, a BMP inhibitor that targets BMP receptors, ALK2, ALK3, and ALK645–47. Both conditions replicated results of the DE protocol, displaying a global shift to distal axial identities marked by elongated distal TFAP2A+, CDH1+ tubules with apical PODXL+ surfaces, positive for ERBB4, and increased abundance of PAPPA2+ cells (Fig. S4a). To further benchmark the DE protocol, we generated and distalized kidney organoids from a different iPSC HNF4A-YFP reporter line48. These nephroids also distalized and displayed near-global expansion of HNF1B, POU3F3, and TFAP2A axial identities with co-activation of β-catenin target LEF1, and mosaic expression of GATA3, while downregulating podocyte (WT1, MAFB), and proximal tubule (HNF4A) precursor markers (Fig. S4b–e). In contrast, controls were bias towards forming WT1+/MAFB+/ZO1high podocyte-like cells and few HNF1B+/POU3F3+/TFAP2A+ domains. Our validation experiments show that the DE protocol is adaptable to different cell lines and SMIs.These observations support the hypothesis that driving distal cell identities (WNT/β-catenin activation) and suppressing proximal cell differentiation (BMP/SMAD inhibition) generates distal nephron-like cells. Moreover, these data present a distal nephron-like organoid model enriched with loop of Henle and DCT precursor-like cells.

A transient BMPOFF and WNTON state primes nephroid cells for distal nephron formation

To assess the axial shift of our DE nephroid model, we performed single-cell RNA sequencing to analyze the cellular composition of DD14 DE and control organoids. As expected, both control (8219 cells) and DE (9301 cells) organoids consisted of nephron-like (PAX2+) or interstitial-like (PDGFRA+) cells, with few off-target cell types (no detection of muscle marker, MYOD1) (Fig. S2a, b and Supplementary Data 2). Both nephron and interstitial progenitors are derived from the intermediate mesoderm34–36, indicating our organoid cell composition efficiently generates intermediate mesoderm and nephron cells. Control organoids contained a small subset of CDH5+ endothelial progenitors (Fig. S2a), which likely give rise to the CD31+ vasculature-like cells observed in DD28. Nephron-like cells were the dominant cell population in the organoids, with 80% (6650 cells) and 77% (7158 cells) of cells from control and DE organoids, respectively, being classified as nephron lineage (Fig. 2a).Fig. 2Single-cell RNA sequencing data shows nephron lineage divergence between control and DE organoids.a Annotated unsupervised clustering of PAX2+ nephron-like DD14 control and DE organoid cells represented in a UMAP. (Inset) Sample contribution of each condition. b Violin plot with annotations of select gene markers (top x-axis) showing expression levels (bottom x-axis) in the DD14 organoid dataset. The y-axis shows cluster numbers and sample origin (gray square – control organoids, pink square – DE organoids). c Volcano plot of DESeq analysis comparing DD14 DMSO control and DE organoid nephron scRNA sequencing data (Wald test p-value, adjusted by FDR correction). Dashed lines show 0.05 p-value cutoff (horizontal) and ±1.5-fold change cutoff (vertical). d Bar plot showing normalized percent contribution of cells expressing select nephron markers (log2FC > 0) separated by sample origin. Numbers in bars show raw cell counts. e Feature plots of select genes in the DD14 DE organoid dataset. f Immunofluorescence stain of week 16 human kidney from SSB-CLSN stage marking DCT/CNT precursor and loop of Henle/macula densa precursor segments. g Immunofluorescence stains of DD14 control and DE organoids showing expression of DCT/CNT precursor and loop of Henle/macula densa precursor markers. Biological replicates (n > 4) showed similar results. Yellow arrow – autofluorescence. Scale bar: 50 microns.

Single-cell RNA sequencing data shows nephron lineage divergence between control and DE organoids.

a Annotated unsupervised clustering of PAX2+ nephron-like DD14 control and DE organoid cells represented in a UMAP. (Inset) Sample contribution of each condition. b Violin plot with annotations of select gene markers (top x-axis) showing expression levels (bottom x-axis) in the DD14 organoid dataset. The y-axis shows cluster numbers and sample origin (gray square – control organoids, pink square – DE organoids). c Volcano plot of DESeq analysis comparing DD14 DMSO control and DE organoid nephron scRNA sequencing data (Wald test p-value, adjusted by FDR correction). Dashed lines show 0.05 p-value cutoff (horizontal) and ±1.5-fold change cutoff (vertical). d Bar plot showing normalized percent contribution of cells expressing select nephron markers (log2FC > 0) separated by sample origin. Numbers in bars show raw cell counts. e Feature plots of select genes in the DD14 DE organoid dataset. f Immunofluorescence stain of week 16 human kidney from SSB-CLSN stage marking DCT/CNT precursor and loop of Henle/macula densa precursor segments. g Immunofluorescence stains of DD14 control and DE organoids showing expression of DCT/CNT precursor and loop of Henle/macula densa precursor markers. Biological replicates (n > 4) showed similar results. Yellow arrow – autofluorescence. Scale bar: 50 microns.

When control and DE nephroid cells were merged and compared, few cells co-clustered with the other condition, suggesting minimal overlap in equivalent cell states between the 2 conditions (Fig. 2a, top right). Cell states clustered based on differentiation states and cellular processes defined by well-studied nephron patterning markers (Fig. 2a, b and Supplementary Data 2). Consistent with the almost binary switch observed by immunofluorescent stains, control and DE organoids are demarcated by WT1 and TFAP2A detection, respectively, underpinned by otherwise broad detection of genes expressed throughout the early nephron PD axis, such as PAX8, PAX2, and LHX1 (Figs. 2b and S1e). Clusters 3, 8, 12 and 17, comprised of control nephroid cells, displayed expression of induced nephron progenitor markers, BMPER, MEOX1 and SIX1 (Fig. 2b). Bona fide differentiation of podocytes (NPHS2+, PODXL+, SOST+, TCF21+) was only observed in controls; WT1+ DE cells did not express these markers, indicating podocyte differentiation was suppressed, alluding to rudimentary remnants of proximal-like cells. Instead, DE nephroid cells were strongly biased towards distal axial identities expressing TFAP2A (Fig. 2b), consistent with our immunofluorescence data. These TFAP2A+ cells co-expressed a range of early distal RV markers like EMX2, LHX1, and POU3F3, and late distal RV markers, GATA3, MAL, and CXCR41,3,5,6,14. A portion of DE cells also expressed differentiation markers (CLDN10, SIM2, and PAPPA2) normally enriched in the loop of Henle and macula densa of the in vivo nephron5,6 (clusters 13, 0, and 4; Fig. 2b).

To scrutinize differences in gene expression of control and DE nephroid cells, we performed comparative differential gene expression analyses, identifying 480 control and 629 DE enriched genes (p < 0.05, log2FC > 1.5; Fig. 2c and Supplementary Data 1). These data corroborate conclusions that control organoids generate a podocyte-like gene signature positive for WT1, IL1R1, KIRREL3, OLFM3, PTPRO, PLA2R1, SYNPO, while DE nephroids adopt a broad and distinct distal-biased profile exhibiting expression of markers indicating early distal nephron development (LGR5, CALCR, MAL, TFAP2A, HNF1B, MECOM, GATA3) and precursor specification and maturation (TMEM52B, ERBB4, ESRRγ, PAPPA2) (Figs. 2c and S2f and Supplementary Data 1). The overt shift in differentiation towards distal nephron identities was further emphasized by quantifying cells expressing PD axial markers. When normalized for cell numbers (6650 control and 7158 DE cells), we examined the cells expressing each marker and quantified what percentage of cells expressing each gene came from control or DE nephroids; e.g., HNF1B is detected in 1610 out of 6650 control organoid cells, and in 6031 out of 7158 DE organoid cells, thus, when normalized to account for each condition’s total cell count, total HNF1B+ cells consist of 21% control and 79% DE cells. DE nephroid cells were strongly biased to develop distal nephron identities as they largely contributed to cells expressing distal nephron genes, while control nephroid cells generate proximal-like cells (Fig. 2d). At DD14, neither condition contained many HNF4A+ cells, but control nephroids had ~4x more HNF4A+ cells than DE organoids.

Control nephroids displayed enrichment of BMP/SMAD pathway effectors ID1, ID2, ID3, and ID437 (Figs. 2c and S3a and Supplementary Data 1) compared to DE nephroids. To evaluate BMP activity, we assayed active SMAD signaling by immunolabelling dual phospho-SMAD1/5. Strong phospho-SMAD labeling was detected in control nephroids and in their surrounding interstitium, but this was absent in the DE organoids, consistent with LDN efficiently suppressing BMP/SMAD signaling in DE conditions (Fig. S3b). While the role of BMP/SMAD signaling is not fully established in the human nephron, labeling of developing human nephrons indicated phospho-SMAD activity in proximal PTAs, RVs, CSB, and in the proximal precursors of the SSB and CLSN (Fig. S3c), thus further correlating BMP signaling with human proximal axial nephron identities as suggested in mice38. Conversely, DE organoids showed upregulation of canonical WNT targets JUN, DLL1, CDH1, EMX2, SALL4, TFCP2L1, and WNT439–42, as expected from the addition of CHIR increasing WNT/β-catenin signaling (Fig. S3d).

DE nephroids expressed genes indicating specification of maturing distal axial identities. Within HNF1B+, TFAP2A+, ERBB4+, MECOM+ DE nephroids cells, select cells were also positive for PAPPA2, ESRRγ, GATA3, and TMEM52B (Figs. 2e and S2f). In developing human nephrons, MECOM marks the entire distal nephron domain in the SSB, GATA3 and TMEM52B mark distal precursors of the CNT and DCT at the distal-most parts of the nephron, while the maturing ascending loop of Henle and macula densa precursors are marked by PAPPA2, ESRRγ and ERBB4; HNF1B and TFAP2A are included for reference (Fig. 2f). ESRRγ, a receptor that acts as a transcriptional activator in the absence of a bound ligand, marks loop of Henle/macula densa precursors43 and is detected in their nuclei in the SSB in a mosaic pattern. The receptor ERBB4 is first detected at the SSB stage in the distal limb. Its detection coincides with that of metalloprotease, PAPPA2, a putative marker of loop of Henle and macula densa precursors1,44. Using these as anchors to scrutinize the DE nephroid differentiation, immunolabeling demonstrated that DE nephroids are shifted to a distal MECOM+/TMEM52B+ cell state that is broadly co-labeled with loop of Henle receptor ERBB4, and mosaic detection of maturation transcription factor ESRRγ and macula densa/loop of Henle marker, PAPPA2 (Fig. 2g). These markers are rare in control nephroids and suggest that the DE nephroids are driven towards a distal tubule fate with a further sub-specification towards the distal tubule, macula densa, and thick ascending limb of the loop of Henle. Interestingly, both organoid conditions display sporadic GATA3+ domains. This specification agrees with unbiased GO term analyses of genes upregulated in DD14 DE organoids (log2FC > 1.5) (Fig. S3e and Supplementary Data 1); DE genes include those associated with nephron epithelium development, calcium ion binding, and ion transmembrane transporter activity, consistent with functions associated with the distal nephron epithelia. Of note, FGF receptor binding was highlighted as a predicted enriched molecular function in DE organoids, attributed to significant upregulation of FGF8, FGF10, and FGF18 (Log2FC = 3.421, 3.128, and 5.288, respectively), indicating elevated FGF signaling in DE organoids relative to controls.

To ensure the DE protocol is robustly reproducible, we performed experiments replacing either the PI3K inhibitor LY294002 with GDC-0941, a structurally different and potent PI3K inhibitor, or replaced SMAD inhibitor LDN-193189 with Dorsomorphin, a BMP inhibitor that targets BMP receptors, ALK2, ALK3, and ALK645–47. Both conditions replicated results of the DE protocol, displaying a global shift to distal axial identities marked by elongated distal TFAP2A+, CDH1+ tubules with apical PODXL+ surfaces, positive for ERBB4, and increased abundance of PAPPA2+ cells (Fig. S4a). To further benchmark the DE protocol, we generated and distalized kidney organoids from a different iPSC HNF4A-YFP reporter line48. These nephroids also distalized and displayed near-global expansion of HNF1B, POU3F3, and TFAP2A axial identities with co-activation of β-catenin target LEF1, and mosaic expression of GATA3, while downregulating podocyte (WT1, MAFB), and proximal tubule (HNF4A) precursor markers (Fig. S4b–e). In contrast, controls were bias towards forming WT1+/MAFB+/ZO1high podocyte-like cells and few HNF1B+/POU3F3+/TFAP2A+ domains. Our validation experiments show that the DE protocol is adaptable to different cell lines and SMIs.

These observations support the hypothesis that driving distal cell identities (WNT/β-catenin activation) and suppressing proximal cell differentiation (BMP/SMAD inhibition) generates distal nephron-like cells. Moreover, these data present a distal nephron-like organoid model enriched with loop of Henle and DCT precursor-like cells.

Distalized nephrons differentiate to loop of Henle fatesTo directly identify the closest in vivo equivalents of DE nephroid cell states, we generated an in vivo transcriptome framework by reanalyzing previous single-cell RNA sequencing capturing human nephrogenesis data from week 14 (11,992 cells) and week 17 (4,772 cells) human kidneys1,49, processed as described in Schnell et al32. In these data, NPs (MEOX1+/CITED1+) develop into podocyte (MAFB+), parietal epithelium (PLAT+), proximal tubule (HNF4A+/SLC3A1+), and early distal tubule precursors (HNF1B+/POU3F3+/TFAP2A+) that differentiate into loop of Henle (SLC12A1+/PAPPA2+/CLDN10+) and a combined cluster consisting of DCT and CNT cells (TMEM52B+/GATA3+/EMX1+) (Figs. 3a and S5a, b and Supplementary Data 2). Using these as a reference, we mapped control and DE nephroid transcriptomes to the most similar in vivo cell RNA profiles (Fig. 3a). DE nephroids closely resembled distal precursor cells, with a minor WT1+ NP-like cell population remaining in these organoids. Strikingly, DE nephroid cells also mapped to loop of Henle precursors, but not DCT and CNT precursors (Figs. 3a and S5c). In contrast, control nephroid cells closely corresponded to PTA and early podocyte cells, and to a lesser degree, the earliest PT precursors, where proximal and distal cell fates have not fully diverged (Figs. 3a and S5c). Cell type prediction scores (see ‘Methods’) between nephroid and in vivo counterparts showed that DE organoids most closely resemble the distal nephron precursor state and loop of Henle precursors, while control nephroids resemble the PTA-like state and early podocytes (Fig. 3b). There is therefore a strong prediction that DE nephroid cells are in an early distal nephron identity programmed towards a loop of Henle-like state.Fig. 3Characterizations of distal nephron segmentation in distal-enriched organoids.a Schematic showing reference mapping of DD14 organoid scRNA sequencing dataset to reanalyzed week 14 and week 17 (4772 cells) merged human nephron and ureteric scRNA sequencing data1,49. (Right) UMAP of human kidney (reference) overlaid with DD14 control and DE organoid mapping. b Bar plot showing number of organoid cells mapping to each predicted cell type, split by condition. Numbers above bars denote prediction score for each cell type. c Immunofluorescence stain of DD14 control and DE organoids showing detection of IRX1. Biological replicates (n = 3) showed similar results. Scale bar: 50 microns. d–e Immunofluorescent stains of DD20 control and DE organoids showing detection of select loop of Henle/macula densa precursor and podocyte precursor markers. Biological replicates (n = 3) showed similar results. Scale bar: 50 microns. Abbreviations: NP nephron progenitors, IP induced progenitors, PTA pretubular aggregate, pod podocytes, PEC parietal epithelial cells, dist prec distal nephron precursors, PT prec proximal tubule precursors, PT proximal tubule, LOH loop of Henle, DCT/CNT distal convoluted tubule/connecting tubule.To increase resolution to primed differentiation programs in the DE model and their transcriptional drivers, we analyzed expression of all known human transcription factors in in vivo nephrons (approach described in Kim et al.5). We identified 39 distal nephron-enriched transcription factors that have either very low or no expression in other nephron cell types (see 'Methods') (Fig. S5d); IRX1, IRX2, TFAP2B, SIM2, IRX6, TFEB and NRK are enriched in loop of Henle/macula densa precursors, EMX1, GATA3, PRDM16 and NR0B2 are enriched in distal convoluted tubule and connecting tubule precursors, and GATA2, ELF5 and EHF are enriched in the ureteric epithelium (UE). Several of these transcription factors are known to be distal nephron-enriched, and co-expression of Irx1, Irx2, and Sim2 in loop of Henle/macula densa precursors is conserved in P0 mouse nephrons5, indicating function. Analysis of transcription factors in our DE nephroids showed that loop of Henle-enriched transcription factors, IRX1, IRX2, TFAP2B, and SIM2, are restricted to PAPPA2+ DE nephroid clusters 13, 0, and 4 (Fig. S5e). Of the DCT-enriched transcription factors, only GATA3 was detected. EMX1, which is normally expressed in the DCT and CNT precursors of the CLSN, was absent. Transcription factors that are predicted to be expressed in more mature loop of Henle segments, IRX6, TFEB, and NRK, and DCT segments, NR0B2, PRDM16, and EMX1, were not detected. Their lack of detection in DD14 nephroids, which resemble the SSB stage, was expected. We observed strong co-expression of known loop of Henle/macula densa precursor-enriched markers, IRX1 and SIM2, in a subset of DE nephroid cells (Fig. S5f). Validation experiments show IRX1 is broadly present in DD14 DE nephroids (Fig. 3c), indicating these cells are initially driven to a distal axial state primed to thereafter generate a subset of more mature nephron cell types.Our bulk RNA sequencing of DD14 and DD20 DE organoids show that the gene signatures of loop of Henle and macula densa lineages are maintained and reinforced, indicated by large increases in expression of PAPPA2, ERBB4, and ascending loop of Henle/macula densa solute carrier, SLC12A1 (Fig. S5g), while transcription factors IRX1 and SIM2 expression were maintained (Fig. S5g). Validation by immunostaining in DD20 organoids shows strong detection of IRX1, PAPPA2, and SLC12A1 in DE organoids (Fig. 3d–e). In contrast, maturation markers of the DCT, such as EMX1, and calcium transport-associated TRPV5 and CALB1 were undetected in both DD14 and DD20 DE organoids, while GATA3 expression remained the same between the two time-points, and SALL3 expression was downregulated (Fig. S5g). Podocyte signatures did not recover by DD20, where DE organoids displayed few MAFB+ cells, relative to controls (Fig. 3e).To identify differentiation endpoints, we performed single-nucleus RNA sequencing of DD28 control and DE organoids. Consistent with DD14 analyses, control and DE organoids largely consisted of PAX8+ nephron- and PDGFRA+ interstitial-like cells (Fig. S6a, b) Control organoids contained rare CDH5 (endothelium) and MYOD1 (muscle) transcripts, and RENIN+ granular-like cell populations, the latter consistent juxtaglomerular cells, while DE organoids displayed a minor population of NEUROD1+ neuron-like cells, common in long-term cultures of kidney organoids9 (Fig. S6a, b). Direct comparison of control (3,947 nuclei) and DE (1,457 nuclei) PAX8+ nephroid lineages revealed a similar pattern to DD14 organoids, where both conditions have distinct spatial clustering (Fig. 4a). Control organoids were enriched with MAFB+ podocytes, CLDN1+ parietal epithelial cells and HNF4A+ proximal tubule cells, whereas DE nephroids have a strong enrichment of PAPPA2+ loop of Henle/macula densa cells, TMEM52B+ distal tubules and a small population of GATA3+ nephron connecting tubules (Fig. 4b). Immunofluorescent labeling confirmed that DE organoids consisted primarily of TFAP2A+/SLC12A1+/ERBB4+/CDH1+ cells that were mosaically positive for PAPPA2 and IRX1 (Figs. 4c, d and S6c). Markers of other nephron axial identities, e.g., proximal tubule (HNF4A+), podocyte (MAFB+), and CNT (GATA3+), were rare (Fig. 4d, e). In contrast, control organoids were enriched with MAFB+ glomeruli-like cells abutting CD31+ vasculature and RENIN+/GATA3+ juxtaglomerular-like cells, contained HNF1B+/HNF4A+ proximal tubule-like cells, and displayed sparse PAPPA2 and IRX1 labeling (Fig. 4d, e); RENIN+ cells were not observed in DE organoids.Fig. 4Single-nucleus RNA sequencing shows endpoint differentiation cell-types in DD28 control and DE organoids.a Annotated unsupervised clustering of integrated PAX2+ nephron-like DD28 control and DE organoid cells represented in a UMAP. (Inset) Sample contribution of each condition. b Feature plots of nephron marker expression in DD28 organoids showing podocyte-, PEC-, proximal tubule-, loop of Henle-, and distal tubule-like cells. c–e Immunofluorescent stains of DD28 control and DE organoids showing detection of glomerular, proximal tubule, loop of Henle/macula densa, and DCT precursor markers. Biological replicates (n > 4) showed similar results. Scale bar: 50 microns. f UMAP of adult human kidney snRNAseq data (44,774 cells) with ‘subclass.l1’ annotations, including nephron (pink border) and ureteric epithelia (gray border). g UMAP of adult human kidney (reference) overlaid with DD28 control and DE organoid reference mapping. h Bar plot showing number of organoid cells mapping to each predicted cell type in control (top) and DE (bottom) organoids. Numbers above bars denote prediction score for each cell type. i Feature plots showing detection of select TAL-enriched ion transporters in DD28 organoids. j Illustration of cell type enrichment in DD28 DE organoids.To determine how endpoint differentiation nephroids relate to the in vivo human kidney, we compared them to published adult human kidney snRNAseq data from the Kidney Precision Medicine project (KPMP; 44,774 cells)50, which contains annotated cell types, including podocytes (POD), parietal epithelial cells (PEC), proximal tubule (PT), the descending thin limb (DTL), ascending thin limb (ATL), and thick ascending limb (TAL) of the loop of Henle, distal convoluted tubule (DCT), nephron connecting tubule (CNT), principle cells (PC), intercalated cells (IC), and papillary tip epithelial cells (PapE) (Fig. 4f). Multimodal reference mapping with control and DE nephroid cells predicted the majority of control organoid cells map to adult human podocytes, PT, PEC, and DTL (Fig. 4g, h). DE organoids, however, mainly map to TAL cells—note that the authors’ system of annotation groups the macula densa into this classification50—and DTL. DESeq analysis of control and DE organoids further corroborate that DE organoids are enriched with distal nephron markers (MECOM, TFCP2L1, and TFAP2B), loop of Henle/macula densa specific markers (SLC12A1, CASR, SIM2, and PAPPA2), while control organoids displayed sustained expression of genes associated with mature podocyte markers, ITIH5 and THSD7A43,49,50 (Fig. S6d). As expected for cells mapping to adult functional cell populations, DE organoids displayed robust expression of physiology-imparting genes encoding distal-nephron-specific transporters enriched in adult human TAL cells, such as SLC12A1 (NKCC2), CLDN16, KCNJ1 (ROMK), CASR, SCNN1A (ENaC), CLNCKA (ClC-Ka), CLCNKB (ClC-Kb), BSND (Barttin), SLC9A3 (NHE3), and glucagon receptor, GCGR (Figs. 4i and S6e). Loss-of-function mutations to proteins NKCC2, ROMK, ClC-Kb, Barttin, and CASR are linked to Bartter Syndrome, a genetic condition that causes ion imbalance and an inability to reabsorb salt51. Multimodal reference mapping is a powerful predictive tool and can be limited by reference dependency and differences in sequencing platforms used. To mitigate this, we matched snRNA-to-snRNA-seq in vitro/in vitro mapping to minimally bias conditions for these comparisons. GO term analyses of top differentially expressed genes in whole DD28 DE organoids further underscore distal nephron-linked functions of ionic exchange (Fig. S6f).Together, these data demonstrate a model for a nearly systemic axial switch in nephron cell type identities.

Distalized nephrons differentiate to loop of Henle fates

To directly identify the closest in vivo equivalents of DE nephroid cell states, we generated an in vivo transcriptome framework by reanalyzing previous single-cell RNA sequencing capturing human nephrogenesis data from week 14 (11,992 cells) and week 17 (4,772 cells) human kidneys1,49, processed as described in Schnell et al32. In these data, NPs (MEOX1+/CITED1+) develop into podocyte (MAFB+), parietal epithelium (PLAT+), proximal tubule (HNF4A+/SLC3A1+), and early distal tubule precursors (HNF1B+/POU3F3+/TFAP2A+) that differentiate into loop of Henle (SLC12A1+/PAPPA2+/CLDN10+) and a combined cluster consisting of DCT and CNT cells (TMEM52B+/GATA3+/EMX1+) (Figs. 3a and S5a, b and Supplementary Data 2). Using these as a reference, we mapped control and DE nephroid transcriptomes to the most similar in vivo cell RNA profiles (Fig. 3a). DE nephroids closely resembled distal precursor cells, with a minor WT1+ NP-like cell population remaining in these organoids. Strikingly, DE nephroid cells also mapped to loop of Henle precursors, but not DCT and CNT precursors (Figs. 3a and S5c). In contrast, control nephroid cells closely corresponded to PTA and early podocyte cells, and to a lesser degree, the earliest PT precursors, where proximal and distal cell fates have not fully diverged (Figs. 3a and S5c). Cell type prediction scores (see ‘Methods’) between nephroid and in vivo counterparts showed that DE organoids most closely resemble the distal nephron precursor state and loop of Henle precursors, while control nephroids resemble the PTA-like state and early podocytes (Fig. 3b). There is therefore a strong prediction that DE nephroid cells are in an early distal nephron identity programmed towards a loop of Henle-like state.Fig. 3Characterizations of distal nephron segmentation in distal-enriched organoids.a Schematic showing reference mapping of DD14 organoid scRNA sequencing dataset to reanalyzed week 14 and week 17 (4772 cells) merged human nephron and ureteric scRNA sequencing data1,49. (Right) UMAP of human kidney (reference) overlaid with DD14 control and DE organoid mapping. b Bar plot showing number of organoid cells mapping to each predicted cell type, split by condition. Numbers above bars denote prediction score for each cell type. c Immunofluorescence stain of DD14 control and DE organoids showing detection of IRX1. Biological replicates (n = 3) showed similar results. Scale bar: 50 microns. d–e Immunofluorescent stains of DD20 control and DE organoids showing detection of select loop of Henle/macula densa precursor and podocyte precursor markers. Biological replicates (n = 3) showed similar results. Scale bar: 50 microns. Abbreviations: NP nephron progenitors, IP induced progenitors, PTA pretubular aggregate, pod podocytes, PEC parietal epithelial cells, dist prec distal nephron precursors, PT prec proximal tubule precursors, PT proximal tubule, LOH loop of Henle, DCT/CNT distal convoluted tubule/connecting tubule.

Characterizations of distal nephron segmentation in distal-enriched organoids.

a Schematic showing reference mapping of DD14 organoid scRNA sequencing dataset to reanalyzed week 14 and week 17 (4772 cells) merged human nephron and ureteric scRNA sequencing data1,49. (Right) UMAP of human kidney (reference) overlaid with DD14 control and DE organoid mapping. b Bar plot showing number of organoid cells mapping to each predicted cell type, split by condition. Numbers above bars denote prediction score for each cell type. c Immunofluorescence stain of DD14 control and DE organoids showing detection of IRX1. Biological replicates (n = 3) showed similar results. Scale bar: 50 microns. d–e Immunofluorescent stains of DD20 control and DE organoids showing detection of select loop of Henle/macula densa precursor and podocyte precursor markers. Biological replicates (n = 3) showed similar results. Scale bar: 50 microns. Abbreviations: NP nephron progenitors, IP induced progenitors, PTA pretubular aggregate, pod podocytes, PEC parietal epithelial cells, dist prec distal nephron precursors, PT prec proximal tubule precursors, PT proximal tubule, LOH loop of Henle, DCT/CNT distal convoluted tubule/connecting tubule.

To increase resolution to primed differentiation programs in the DE model and their transcriptional drivers, we analyzed expression of all known human transcription factors in in vivo nephrons (approach described in Kim et al.5). We identified 39 distal nephron-enriched transcription factors that have either very low or no expression in other nephron cell types (see 'Methods') (Fig. S5d); IRX1, IRX2, TFAP2B, SIM2, IRX6, TFEB and NRK are enriched in loop of Henle/macula densa precursors, EMX1, GATA3, PRDM16 and NR0B2 are enriched in distal convoluted tubule and connecting tubule precursors, and GATA2, ELF5 and EHF are enriched in the ureteric epithelium (UE). Several of these transcription factors are known to be distal nephron-enriched, and co-expression of Irx1, Irx2, and Sim2 in loop of Henle/macula densa precursors is conserved in P0 mouse nephrons5, indicating function. Analysis of transcription factors in our DE nephroids showed that loop of Henle-enriched transcription factors, IRX1, IRX2, TFAP2B, and SIM2, are restricted to PAPPA2+ DE nephroid clusters 13, 0, and 4 (Fig. S5e). Of the DCT-enriched transcription factors, only GATA3 was detected. EMX1, which is normally expressed in the DCT and CNT precursors of the CLSN, was absent. Transcription factors that are predicted to be expressed in more mature loop of Henle segments, IRX6, TFEB, and NRK, and DCT segments, NR0B2, PRDM16, and EMX1, were not detected. Their lack of detection in DD14 nephroids, which resemble the SSB stage, was expected. We observed strong co-expression of known loop of Henle/macula densa precursor-enriched markers, IRX1 and SIM2, in a subset of DE nephroid cells (Fig. S5f). Validation experiments show IRX1 is broadly present in DD14 DE nephroids (Fig. 3c), indicating these cells are initially driven to a distal axial state primed to thereafter generate a subset of more mature nephron cell types.

Our bulk RNA sequencing of DD14 and DD20 DE organoids show that the gene signatures of loop of Henle and macula densa lineages are maintained and reinforced, indicated by large increases in expression of PAPPA2, ERBB4, and ascending loop of Henle/macula densa solute carrier, SLC12A1 (Fig. S5g), while transcription factors IRX1 and SIM2 expression were maintained (Fig. S5g). Validation by immunostaining in DD20 organoids shows strong detection of IRX1, PAPPA2, and SLC12A1 in DE organoids (Fig. 3d–e). In contrast, maturation markers of the DCT, such as EMX1, and calcium transport-associated TRPV5 and CALB1 were undetected in both DD14 and DD20 DE organoids, while GATA3 expression remained the same between the two time-points, and SALL3 expression was downregulated (Fig. S5g). Podocyte signatures did not recover by DD20, where DE organoids displayed few MAFB+ cells, relative to controls (Fig. 3e).

To identify differentiation endpoints, we performed single-nucleus RNA sequencing of DD28 control and DE organoids. Consistent with DD14 analyses, control and DE organoids largely consisted of PAX8+ nephron- and PDGFRA+ interstitial-like cells (Fig. S6a, b) Control organoids contained rare CDH5 (endothelium) and MYOD1 (muscle) transcripts, and RENIN+ granular-like cell populations, the latter consistent juxtaglomerular cells, while DE organoids displayed a minor population of NEUROD1+ neuron-like cells, common in long-term cultures of kidney organoids9 (Fig. S6a, b). Direct comparison of control (3,947 nuclei) and DE (1,457 nuclei) PAX8+ nephroid lineages revealed a similar pattern to DD14 organoids, where both conditions have distinct spatial clustering (Fig. 4a). Control organoids were enriched with MAFB+ podocytes, CLDN1+ parietal epithelial cells and HNF4A+ proximal tubule cells, whereas DE nephroids have a strong enrichment of PAPPA2+ loop of Henle/macula densa cells, TMEM52B+ distal tubules and a small population of GATA3+ nephron connecting tubules (Fig. 4b). Immunofluorescent labeling confirmed that DE organoids consisted primarily of TFAP2A+/SLC12A1+/ERBB4+/CDH1+ cells that were mosaically positive for PAPPA2 and IRX1 (Figs. 4c, d and S6c). Markers of other nephron axial identities, e.g., proximal tubule (HNF4A+), podocyte (MAFB+), and CNT (GATA3+), were rare (Fig. 4d, e). In contrast, control organoids were enriched with MAFB+ glomeruli-like cells abutting CD31+ vasculature and RENIN+/GATA3+ juxtaglomerular-like cells, contained HNF1B+/HNF4A+ proximal tubule-like cells, and displayed sparse PAPPA2 and IRX1 labeling (Fig. 4d, e); RENIN+ cells were not observed in DE organoids.Fig. 4Single-nucleus RNA sequencing shows endpoint differentiation cell-types in DD28 control and DE organoids.a Annotated unsupervised clustering of integrated PAX2+ nephron-like DD28 control and DE organoid cells represented in a UMAP. (Inset) Sample contribution of each condition. b Feature plots of nephron marker expression in DD28 organoids showing podocyte-, PEC-, proximal tubule-, loop of Henle-, and distal tubule-like cells. c–e Immunofluorescent stains of DD28 control and DE organoids showing detection of glomerular, proximal tubule, loop of Henle/macula densa, and DCT precursor markers. Biological replicates (n > 4) showed similar results. Scale bar: 50 microns. f UMAP of adult human kidney snRNAseq data (44,774 cells) with ‘subclass.l1’ annotations, including nephron (pink border) and ureteric epithelia (gray border). g UMAP of adult human kidney (reference) overlaid with DD28 control and DE organoid reference mapping. h Bar plot showing number of organoid cells mapping to each predicted cell type in control (top) and DE (bottom) organoids. Numbers above bars denote prediction score for each cell type. i Feature plots showing detection of select TAL-enriched ion transporters in DD28 organoids. j Illustration of cell type enrichment in DD28 DE organoids.

Single-nucleus RNA sequencing shows endpoint differentiation cell-types in DD28 control and DE organoids.

a Annotated unsupervised clustering of integrated PAX2+ nephron-like DD28 control and DE organoid cells represented in a UMAP. (Inset) Sample contribution of each condition. b Feature plots of nephron marker expression in DD28 organoids showing podocyte-, PEC-, proximal tubule-, loop of Henle-, and distal tubule-like cells. c–e Immunofluorescent stains of DD28 control and DE organoids showing detection of glomerular, proximal tubule, loop of Henle/macula densa, and DCT precursor markers. Biological replicates (n > 4) showed similar results. Scale bar: 50 microns. f UMAP of adult human kidney snRNAseq data (44,774 cells) with ‘subclass.l1’ annotations, including nephron (pink border) and ureteric epithelia (gray border). g UMAP of adult human kidney (reference) overlaid with DD28 control and DE organoid reference mapping. h Bar plot showing number of organoid cells mapping to each predicted cell type in control (top) and DE (bottom) organoids. Numbers above bars denote prediction score for each cell type. i Feature plots showing detection of select TAL-enriched ion transporters in DD28 organoids. j Illustration of cell type enrichment in DD28 DE organoids.

To determine how endpoint differentiation nephroids relate to the in vivo human kidney, we compared them to published adult human kidney snRNAseq data from the Kidney Precision Medicine project (KPMP; 44,774 cells)50, which contains annotated cell types, including podocytes (POD), parietal epithelial cells (PEC), proximal tubule (PT), the descending thin limb (DTL), ascending thin limb (ATL), and thick ascending limb (TAL) of the loop of Henle, distal convoluted tubule (DCT), nephron connecting tubule (CNT), principle cells (PC), intercalated cells (IC), and papillary tip epithelial cells (PapE) (Fig. 4f). Multimodal reference mapping with control and DE nephroid cells predicted the majority of control organoid cells map to adult human podocytes, PT, PEC, and DTL (Fig. 4g, h). DE organoids, however, mainly map to TAL cells—note that the authors’ system of annotation groups the macula densa into this classification50—and DTL. DESeq analysis of control and DE organoids further corroborate that DE organoids are enriched with distal nephron markers (MECOM, TFCP2L1, and TFAP2B), loop of Henle/macula densa specific markers (SLC12A1, CASR, SIM2, and PAPPA2), while control organoids displayed sustained expression of genes associated with mature podocyte markers, ITIH5 and THSD7A43,49,50 (Fig. S6d). As expected for cells mapping to adult functional cell populations, DE organoids displayed robust expression of physiology-imparting genes encoding distal-nephron-specific transporters enriched in adult human TAL cells, such as SLC12A1 (NKCC2), CLDN16, KCNJ1 (ROMK), CASR, SCNN1A (ENaC), CLNCKA (ClC-Ka), CLCNKB (ClC-Kb), BSND (Barttin), SLC9A3 (NHE3), and glucagon receptor, GCGR (Figs. 4i and S6e). Loss-of-function mutations to proteins NKCC2, ROMK, ClC-Kb, Barttin, and CASR are linked to Bartter Syndrome, a genetic condition that causes ion imbalance and an inability to reabsorb salt51. Multimodal reference mapping is a powerful predictive tool and can be limited by reference dependency and differences in sequencing platforms used. To mitigate this, we matched snRNA-to-snRNA-seq in vitro/in vitro mapping to minimally bias conditions for these comparisons. GO term analyses of top differentially expressed genes in whole DD28 DE organoids further underscore distal nephron-linked functions of ionic exchange (Fig. S6f).

Together, these data demonstrate a model for a nearly systemic axial switch in nephron cell type identities.

Modulation of FGF and BMP signaling shifts domain identities along the proximal-distal nephron axisThe DE organoid approach demonstrates that simultaneous activation of WNT and inhibition of BMP/SMAD signaling generates cells primed for distal nephron fates. DE organoids significantly increased FGFR1, MAP7, and MAP2 expression at DD14 (Fig. 2c and Supplementary Data 1), highlighting a potential role of FGF/MAPK signaling in distalization. Mouse studies have shown that Fgf8 is required for distal nephron cell survival and differentiation, it is a direct target of Wnt/β-catenin, and Fgf8 hypomorphs fail to form Slc12a1+ loop of Henle cells, indicating a non-redundant role, thus raising the possibility that FGF signaling is responsible for distalization and formation of TAL-like cells in DE organoids22,52. Consistent with this view, FGF8 was strongly upregulated in DD14 DE nephroids (log2FC ~ 3) and thereafter downregulated by DD28 (Fig. 5a). In vivo, single-cell RNA sequencing data showed FGF8 is transiently upregulated during distal nephron development in a sequence closely matching TFAP2A expression, but FGF8 is turned off soon after SSB stages (Fig. 5b). To resolve the exact normal sequence of FGF8 up and downregulation in vivo, we generated spatial transcriptomic data using seqFISH53, capturing human kidney and nephron development54. In vivo, FGF8 is detected in TFAP2A+ distal cells in the early RV, remains there into the late RV, and then expands within the TFAP2A+ domain (Fig. 5c). By SSB stages, FGF8 is detected in the forming IRX1+ domain and is downregulated in distal-most GATA3+ cells (Fig. S7a). By late SSB stages, FGF8 is downregulated throughout distal TFAP2A+ cells aside from the IRX1+ and PAPPA2+ loop of Henle precursors, indicating a shift of expression within the TFAP2A+ domain. FGF8 transcripts are not detected in HNF4A+ proximal and MAFB+ podocyte precursors (Fig. S7a). By the CLSN stage, FGF8 is turned off in the distal nephron (Fig. 5c) and is thereafter not detected in adult stages (Figure S7b).Fig. 5Dynamics of FGF, WNT, and BMP signaling in nephron segmentation.a Feature plot of FGF8 expression in DD14 and DD28 organoids, split by condition. b Feature plot of FGF8 in the human kidney dataset. Inset: TFAP2A detection in the distal nephron. c SeqFISH data of week 16 human kidney showing detection of FGF8 and TFAP2A from early RV to CLSN. d–g (left) Schematics of SMI treatment and nephron segmentation in DD14 organoids and (right) immunofluorescent stains showing detection of DCT/CNT, proximal tubule, loop of Henle/macula densa, and podocyte precursor markers. Biological replicates (n > 4) showed similar results. Scale bars: 50 microns. h Bar plots of percent quantification of fluorescence detection of podocyte, proximal tubule, and distal nephron markers in select nephroids of DD14 control, DE, DE + FGFR1i + BMPi organoids (n = 2). i Schematic of nephron segments and the corresponding pathways that drive their development. j Proposed schematic of select signaling pathway interactions in the developing mammalian nephron.Other FGF ligands, FGF9 and FGF10, which belong to FGF9 and FGF7 subfamilies, respectively55, are also expressed during in vivo nephron patterning. In the developing nephron, FGF9 is upregulated distally as FGF8 is being downregulated, forming a largely mutually exclusive expression pattern. FGF9 is also strongly upregulated in podocytes and parietal epithelium but is absent from proximal tubule precursors. FGF10 follows a similar onset of expression to FGF8 but then persists in the DCT and CNT precursors; FGF10 is also expressed in nephron progenitors (Figs. 5b and S7c). In the adult in vivo nephron, FGF9 expression is detected in distal tubular nephron cell-types, in podocytes, and in the parietal epithelium, but FGF10 expression is downregulated. In vitro, FGF9 and FGF10 follow similar expression patterns to in vivo. FGF10 is upregulated in TFAP2A+ cells in DE nephroids in a pattern closely mimicking FGF8 but is, as expected, infrequently detected in control nephroids that are primarily comprised of early podocytes and proximal fated cells (Fig. S7c). Mirroring the expression of FGF10, FGF9 is enriched in podocytes in control nephroids following an in vivo-like pattern and is detected at lower levels in distalized DE nephroids. By DD28, FGF10 is detected in very few cells, but FGF9 remains detected in control organoid podocyte-like cells and DE organoid distal nephroid cells, suggesting it is eventually upregulated in distal nephroids (Fig. S7d). These data suggest the control and DE nephroids therefore largely mimic in vivo adult nephron expression, where FGF9 is detected, and FGF10 is not detected.Distalized organoid nephrons also express FGF18 at DD14, but this FGF ligand, belonging to the FGF8 ligand subfamily55, is not normally detected in vivo during early nephrogenesis. FGF18 is expressed in the adult kidney in the ascending thin limb (ATL) and papillary tip epithelial cells (PapE) of the nephron (Fig. S7d). Unlike the in vivo pattern, where FGF18 is expressed only after early nephrogenesis, in nephroids, FGF18 is downregulated after DD14 and is not observed at DD28.In mice, Fgf-receptor 1 (Fgfr1) is described as the likely receptor for Fgf8, as it is expressed in the nascent nephron and tubular epithelia of the nephron52. In human nephrons, our data suggest FGFR1 and FGFR2 are broadly expressed from progenitor stages (FGFR2 is infrequently detected in podocytes), while FGFR3 is primarily detected in the maturing distal and proximal tubule epithelium – after the proposed action of FGF8 (Fig. S7e). In vitro, in DD14 organoids, FGFR1 expression is detected in both control and DE organoids, though at higher levels in the latter (Fig. S7e and 2c Supplementary Data 1). Lower levels of FGFR2 and FGFR3 are also detected in all clusters of both conditions, and FGFR3 is infrequently detected in the nephron progenitor-like cells (Fig. S7e). Given the overall dynamics of FGF-receptor expression, FGFR1 being detected in TFAP2A+ distalizing cells matching expression of FGF8, it being the preferred receptor for FGF852,56, these lend credibility to FGFR1 being a candidate receptor for FGF signaling in the early distal nephron.To determine whether FGF signaling is required for human distal nephron development, we inhibited FGFR1 between early RV- to SSB-like stages (DD11-DD14) using SU-5402, a potent FGFR1 inhibitor57,58, and compared forming nephroids with control and DE conditions (Fig. 5d–f). This condition generated a Wnt/β-catenin active state unable to signal via FGFR1. Strikingly, inhibition of FGFR1 in DE organoids reversed the DE phenotype (Fig. 5f, h). Distal nephron marker TFAP2A was reduced to control organoid levels while HNF4A+ proximal tubule domains formed - and did so more frequently than in controls - next to MAFB+/WT1+ podocytes (Fig. 5f, h; compare to d and e) while IRX1 cell populations were visually diminished. The HNF4A+ cells developed within HNF1Bhigh domains, strongly reminiscent of normal in vivo development, where high HNF1B expression in the medial SSB precedes and is required for HNF4A expression16,32,59. These data show that inhibition of FGFR1 in DE organoids results in a next-to-complete reversal of the DE state back to a proximal axis-biased differentiation outcome.Given that HNF4A+ proximal tubule precursors and MAFB+ podocytes normally form in cells positive for phosphorylated SMAD (Fig. S3c) and these developed in the DE + FGFR1-inhibitor condition, we asked whether the switch is dependent on SMAD signaling driving proximal axial cell-fates in a setting where FGF8 is rendered incapable of distalizing nephroids due to inhibition of FGFR1. To test this, we extended the LDN-mediated SMAD inhibitor treatment so that DE organoids simultaneously received the FGFR1 inhibitor and the SMAD inhibitor until DD14 (Fig. 5g). Remarkably, this again flipped the differentiation outcome so that TFAP2A and IRX1 were detected throughout nephroids, WT1 and MAFB were lost, and HNF1B was again detected at homogenous levels (as in DE conditions) in the absence of HNF4A (GATA3 expression was relatively low in all conditions) (Fig. 5g, h). These observations suggest that inhibition of FGF in our DE organoids reenables a BMP/SMAD-dependent proximal-driving program that, if blocked, reverts to a distal program.Of note, we observed only a modest expansion of the HNF4A domain in control organoids treated with SU-5402 for the same duration as the other experiments, despite expressing FGFR1 (Fig. S8a–c), indicating that it is the DE condition that generates a FGF8high state driving a distal fate. When analyzing differentiation endpoints (DD28) of DE and DE + FGFR1 inhibitor (SU-5402) treated organoids, the latter developed podocytes as well as loop of Henle identities, showing that the axial shifts imposed by DD14 generate nephroids that thereafter differentiate towards their respective primed states (Fig. S8d–e).To validate phenotypes caused by SU-5402 treatment, we used another selective and potent inhibitor of FGFR1(PD-166866)60. As expected, PD-166866-treated DE organoids (DD11-DD14) formed few TFAP2A+ distal cells while proximal tubule-like cell and podocyte identities reemerged (Fig. S8f). Control organoids treated with PD-166866 for the same duration behaved similarly to controls administered SU-5402, with the exception of a more prominent loss of IRX1+ cells in the former, alluding to lack of loop of Henle-like precursors (Fig. S8b and S8g).Our and others’ work has demonstrated proximal axial cell specification requires Notch signaling20,27,61–63. To determine if the distal-enhanced (β-cateninON/FGFON + BMPOFF) state requires normal Notch signaling, we blocked Notch receptor activity through γ-secretase inhibition (adding DAPT), from DD11-DD14. Inhibition of Notch in vivo is detrimental to the development of all nephron cell types20 and we made similar observations in our DD14 organoids, where the morphology of DD14 control organoids treated with DAPT resembled that of RV-like DD12 control organoids, suggesting arrested differentiation, and nephroids were small, round, and displayed WT1+ cells devoid of MAFB, no detectable HNF4A, and lack of IRX1 expression—normally detectable at DD14 (Fig S8h). DE organoids supplemented with DAPT still developed into TFAP2A+ distalized nephrons with GATA3, devoid of HNF4A and MAFB, and as expected, a JAG1high/HNF1Bhigh state failed to form (Fig S8i). Overall, these data indicate that our organoid model faithfully recapitulates the Notch loss-of-function phenotypes and that the distal program can be directed in the absence of Notch activity.Here, we show that the nephron PD polarity is a highly tunable epithelial axis, and molecular functions required for nephron cell type specification can be disentangled in organoid models, leading to the production of reproducible protocols for differentiating cell fates along the PD axis (Fig. 5i, j). Our model proposes pathway-mediated differentiation to form the different nephron segments and maintain boundaries of their domains (Fig. 5j).

Modulation of FGF and BMP signaling shifts domain identities along the proximal-distal nephron axis

The DE organoid approach demonstrates that simultaneous activation of WNT and inhibition of BMP/SMAD signaling generates cells primed for distal nephron fates. DE organoids significantly increased FGFR1, MAP7, and MAP2 expression at DD14 (Fig. 2c and Supplementary Data 1), highlighting a potential role of FGF/MAPK signaling in distalization. Mouse studies have shown that Fgf8 is required for distal nephron cell survival and differentiation, it is a direct target of Wnt/β-catenin, and Fgf8 hypomorphs fail to form Slc12a1+ loop of Henle cells, indicating a non-redundant role, thus raising the possibility that FGF signaling is responsible for distalization and formation of TAL-like cells in DE organoids22,52. Consistent with this view, FGF8 was strongly upregulated in DD14 DE nephroids (log2FC ~ 3) and thereafter downregulated by DD28 (Fig. 5a). In vivo, single-cell RNA sequencing data showed FGF8 is transiently upregulated during distal nephron development in a sequence closely matching TFAP2A expression, but FGF8 is turned off soon after SSB stages (Fig. 5b). To resolve the exact normal sequence of FGF8 up and downregulation in vivo, we generated spatial transcriptomic data using seqFISH53, capturing human kidney and nephron development54. In vivo, FGF8 is detected in TFAP2A+ distal cells in the early RV, remains there into the late RV, and then expands within the TFAP2A+ domain (Fig. 5c). By SSB stages, FGF8 is detected in the forming IRX1+ domain and is downregulated in distal-most GATA3+ cells (Fig. S7a). By late SSB stages, FGF8 is downregulated throughout distal TFAP2A+ cells aside from the IRX1+ and PAPPA2+ loop of Henle precursors, indicating a shift of expression within the TFAP2A+ domain. FGF8 transcripts are not detected in HNF4A+ proximal and MAFB+ podocyte precursors (Fig. S7a). By the CLSN stage, FGF8 is turned off in the distal nephron (Fig. 5c) and is thereafter not detected in adult stages (Figure S7b).Fig. 5Dynamics of FGF, WNT, and BMP signaling in nephron segmentation.a Feature plot of FGF8 expression in DD14 and DD28 organoids, split by condition. b Feature plot of FGF8 in the human kidney dataset. Inset: TFAP2A detection in the distal nephron. c SeqFISH data of week 16 human kidney showing detection of FGF8 and TFAP2A from early RV to CLSN. d–g (left) Schematics of SMI treatment and nephron segmentation in DD14 organoids and (right) immunofluorescent stains showing detection of DCT/CNT, proximal tubule, loop of Henle/macula densa, and podocyte precursor markers. Biological replicates (n > 4) showed similar results. Scale bars: 50 microns. h Bar plots of percent quantification of fluorescence detection of podocyte, proximal tubule, and distal nephron markers in select nephroids of DD14 control, DE, DE + FGFR1i + BMPi organoids (n = 2). i Schematic of nephron segments and the corresponding pathways that drive their development. j Proposed schematic of select signaling pathway interactions in the developing mammalian nephron.

Dynamics of FGF, WNT, and BMP signaling in nephron segmentation.

a Feature plot of FGF8 expression in DD14 and DD28 organoids, split by condition. b Feature plot of FGF8 in the human kidney dataset. Inset: TFAP2A detection in the distal nephron. c SeqFISH data of week 16 human kidney showing detection of FGF8 and TFAP2A from early RV to CLSN. d–g (left) Schematics of SMI treatment and nephron segmentation in DD14 organoids and (right) immunofluorescent stains showing detection of DCT/CNT, proximal tubule, loop of Henle/macula densa, and podocyte precursor markers. Biological replicates (n > 4) showed similar results. Scale bars: 50 microns. h Bar plots of percent quantification of fluorescence detection of podocyte, proximal tubule, and distal nephron markers in select nephroids of DD14 control, DE, DE + FGFR1i + BMPi organoids (n = 2). i Schematic of nephron segments and the corresponding pathways that drive their development. j Proposed schematic of select signaling pathway interactions in the developing mammalian nephron.

Other FGF ligands, FGF9 and FGF10, which belong to FGF9 and FGF7 subfamilies, respectively55, are also expressed during in vivo nephron patterning. In the developing nephron, FGF9 is upregulated distally as FGF8 is being downregulated, forming a largely mutually exclusive expression pattern. FGF9 is also strongly upregulated in podocytes and parietal epithelium but is absent from proximal tubule precursors. FGF10 follows a similar onset of expression to FGF8 but then persists in the DCT and CNT precursors; FGF10 is also expressed in nephron progenitors (Figs. 5b and S7c). In the adult in vivo nephron, FGF9 expression is detected in distal tubular nephron cell-types, in podocytes, and in the parietal epithelium, but FGF10 expression is downregulated. In vitro, FGF9 and FGF10 follow similar expression patterns to in vivo. FGF10 is upregulated in TFAP2A+ cells in DE nephroids in a pattern closely mimicking FGF8 but is, as expected, infrequently detected in control nephroids that are primarily comprised of early podocytes and proximal fated cells (Fig. S7c). Mirroring the expression of FGF10, FGF9 is enriched in podocytes in control nephroids following an in vivo-like pattern and is detected at lower levels in distalized DE nephroids. By DD28, FGF10 is detected in very few cells, but FGF9 remains detected in control organoid podocyte-like cells and DE organoid distal nephroid cells, suggesting it is eventually upregulated in distal nephroids (Fig. S7d). These data suggest the control and DE nephroids therefore largely mimic in vivo adult nephron expression, where FGF9 is detected, and FGF10 is not detected.

Distalized organoid nephrons also express FGF18 at DD14, but this FGF ligand, belonging to the FGF8 ligand subfamily55, is not normally detected in vivo during early nephrogenesis. FGF18 is expressed in the adult kidney in the ascending thin limb (ATL) and papillary tip epithelial cells (PapE) of the nephron (Fig. S7d). Unlike the in vivo pattern, where FGF18 is expressed only after early nephrogenesis, in nephroids, FGF18 is downregulated after DD14 and is not observed at DD28.

In mice, Fgf-receptor 1 (Fgfr1) is described as the likely receptor for Fgf8, as it is expressed in the nascent nephron and tubular epithelia of the nephron52. In human nephrons, our data suggest FGFR1 and FGFR2 are broadly expressed from progenitor stages (FGFR2 is infrequently detected in podocytes), while FGFR3 is primarily detected in the maturing distal and proximal tubule epithelium – after the proposed action of FGF8 (Fig. S7e). In vitro, in DD14 organoids, FGFR1 expression is detected in both control and DE organoids, though at higher levels in the latter (Fig. S7e and 2c Supplementary Data 1). Lower levels of FGFR2 and FGFR3 are also detected in all clusters of both conditions, and FGFR3 is infrequently detected in the nephron progenitor-like cells (Fig. S7e). Given the overall dynamics of FGF-receptor expression, FGFR1 being detected in TFAP2A+ distalizing cells matching expression of FGF8, it being the preferred receptor for FGF852,56, these lend credibility to FGFR1 being a candidate receptor for FGF signaling in the early distal nephron.

To determine whether FGF signaling is required for human distal nephron development, we inhibited FGFR1 between early RV- to SSB-like stages (DD11-DD14) using SU-5402, a potent FGFR1 inhibitor57,58, and compared forming nephroids with control and DE conditions (Fig. 5d–f). This condition generated a Wnt/β-catenin active state unable to signal via FGFR1. Strikingly, inhibition of FGFR1 in DE organoids reversed the DE phenotype (Fig. 5f, h). Distal nephron marker TFAP2A was reduced to control organoid levels while HNF4A+ proximal tubule domains formed - and did so more frequently than in controls - next to MAFB+/WT1+ podocytes (Fig. 5f, h; compare to d and e) while IRX1 cell populations were visually diminished. The HNF4A+ cells developed within HNF1Bhigh domains, strongly reminiscent of normal in vivo development, where high HNF1B expression in the medial SSB precedes and is required for HNF4A expression16,32,59. These data show that inhibition of FGFR1 in DE organoids results in a next-to-complete reversal of the DE state back to a proximal axis-biased differentiation outcome.

Given that HNF4A+ proximal tubule precursors and MAFB+ podocytes normally form in cells positive for phosphorylated SMAD (Fig. S3c) and these developed in the DE + FGFR1-inhibitor condition, we asked whether the switch is dependent on SMAD signaling driving proximal axial cell-fates in a setting where FGF8 is rendered incapable of distalizing nephroids due to inhibition of FGFR1. To test this, we extended the LDN-mediated SMAD inhibitor treatment so that DE organoids simultaneously received the FGFR1 inhibitor and the SMAD inhibitor until DD14 (Fig. 5g). Remarkably, this again flipped the differentiation outcome so that TFAP2A and IRX1 were detected throughout nephroids, WT1 and MAFB were lost, and HNF1B was again detected at homogenous levels (as in DE conditions) in the absence of HNF4A (GATA3 expression was relatively low in all conditions) (Fig. 5g, h). These observations suggest that inhibition of FGF in our DE organoids reenables a BMP/SMAD-dependent proximal-driving program that, if blocked, reverts to a distal program.

Of note, we observed only a modest expansion of the HNF4A domain in control organoids treated with SU-5402 for the same duration as the other experiments, despite expressing FGFR1 (Fig. S8a–c), indicating that it is the DE condition that generates a FGF8high state driving a distal fate. When analyzing differentiation endpoints (DD28) of DE and DE + FGFR1 inhibitor (SU-5402) treated organoids, the latter developed podocytes as well as loop of Henle identities, showing that the axial shifts imposed by DD14 generate nephroids that thereafter differentiate towards their respective primed states (Fig. S8d–e).

To validate phenotypes caused by SU-5402 treatment, we used another selective and potent inhibitor of FGFR1(PD-166866)60. As expected, PD-166866-treated DE organoids (DD11-DD14) formed few TFAP2A+ distal cells while proximal tubule-like cell and podocyte identities reemerged (Fig. S8f). Control organoids treated with PD-166866 for the same duration behaved similarly to controls administered SU-5402, with the exception of a more prominent loss of IRX1+ cells in the former, alluding to lack of loop of Henle-like precursors (Fig. S8b and S8g).

Our and others’ work has demonstrated proximal axial cell specification requires Notch signaling20,27,61–63. To determine if the distal-enhanced (β-cateninON/FGFON + BMPOFF) state requires normal Notch signaling, we blocked Notch receptor activity through γ-secretase inhibition (adding DAPT), from DD11-DD14. Inhibition of Notch in vivo is detrimental to the development of all nephron cell types20 and we made similar observations in our DD14 organoids, where the morphology of DD14 control organoids treated with DAPT resembled that of RV-like DD12 control organoids, suggesting arrested differentiation, and nephroids were small, round, and displayed WT1+ cells devoid of MAFB, no detectable HNF4A, and lack of IRX1 expression—normally detectable at DD14 (Fig S8h). DE organoids supplemented with DAPT still developed into TFAP2A+ distalized nephrons with GATA3, devoid of HNF4A and MAFB, and as expected, a JAG1high/HNF1Bhigh state failed to form (Fig S8i). Overall, these data indicate that our organoid model faithfully recapitulates the Notch loss-of-function phenotypes and that the distal program can be directed in the absence of Notch activity.

Here, we show that the nephron PD polarity is a highly tunable epithelial axis, and molecular functions required for nephron cell type specification can be disentangled in organoid models, leading to the production of reproducible protocols for differentiating cell fates along the PD axis (Fig. 5i, j). Our model proposes pathway-mediated differentiation to form the different nephron segments and maintain boundaries of their domains (Fig. 5j).

DiscussionGenerating functional iPSC-derived nephrons requires optimized self-organization and identifying external cues that control formation of physiology-imparting cell fates. Here, we investigate the nephron axial polarity and signals that tune its development, and we demonstrate its considerable plasticity. A common approach to generate symmetry breaks along an axis is by modifying developmental pathways (WNT, FGF, BMP, Notch). For instance, WNT and FGF agonists tune axial patterning in embryo models64,65, while changes to BMP, WNT, and FGF signaling define midbrain-hindbrain boundaries66. In the nephron, WNT agonists and antagonists have been used in ex vivo and in vitro studies to control nephron patterning27,67,68, while small molecule inhibitors added to kidney organoid cultures can expand specific domains32,67,69,70. Our recent work demonstrated that providing signal inputs in the form of ligand-secreting synthetic cellular organizers can locally control axial identities and axial morphogenesis54. Here, we show that the nephron axis is controlled by WNT, FGF, and BMP signaling, which can be tuned to generate cell identities on demand.Given its rapid and reproducible patterning, the nephron tubule is ideal for understanding how signal transduction regulates cell type specification in mammalian development. We adopted a kidney organoid model28, developed a protocol to direct differentiation towards distal nephron cell fates, and demonstrate that WNT and FGF pathways cooperatively drive distal nephron identities. Our distal-enriched (DE) protocol differentiated distal axial identities in an imposed transient WNTON and BMPOFF state that autonomously activated FGF8, and by auto- or paracrine FGF signaling, drove early distalization, priming thick ascending loop of Henle-like cell fates. In DE organoids where FGF signaling was blocked (DE + FGFOFF), the nephroids reverted their distal axial programming to instead form proximal tubule and podocyte precursors, fates otherwise not found in DE nephroids. These findings show that FGF signaling acts as a dominant pathway in DE conditions and promotes distal identities by outcompeting or actively suppressing proximal axial fates. Taking this one step further, in DE and FGFOFF organoids where BMP/SMAD signaling is also blocked (i.e., DE and sustained FGFOFF + BMPOFF), cells reverted to distal axial identities, thus highlighting a BMP/SMAD-mediated role in forming proximal cell fates. Our findings illustrate a model where nephron patterning is set up by opposing distalizing WNT/FGF and proximalizing BMP signaling, supporting the hypothesis that FGF and BMP compete to drive nephron fates.It is not known how these signaling pathways interact to define nephron segment boundaries and specify axial nephron identities. Mouse models show that WNT/β-catenin, BMP, FGF, and Notch signaling are all required for nephron patterning18–21. We know that nephrons develop their distal-to-proximal identity against the collecting duct2,71, and recent human spatial transcriptomic data show that WNT/β-catenin target genes are activated in the earliest stages of nephrogenesis in those nephron cells closest to the WNT9B expressing collecting duct, and that iPSC-derived nephroids respond to canonical WNTs by driving distal axial cell fates54. The initial PD symmetry break is therefore likely driven by canonical WNT ligands. At the other end of the early nephron, we show that cells display active BMP/SMAD signaling, which is absent in distal cells. These two poles set up an early axial asymmetry with distal β-catenin and proximal BMP/SMAD signaling. Consistent with our data, dividing the axis into WNTON/BMPOFF and WNTOFF/BMPON poles, are mouse experiments demonstrating that activation of WNT/β-catenin and simultaneous inhibition of BMP/SMAD suppresses podocyte cell fates while driving a Cdh1+ distal fate27.A WNT-BMP axial division is reflected in control organoids, which display high pSMAD throughout nephroids and infrequently form distal domains. Shifting these cells to a WNTON/ BMPOFF state between days 11 and 12 is sufficient to enforce a distal-dominant program leading to cells adopting a distal axial identity. What remains unclear is why mouse nephron cells in WNTON/BMPOFF states upregulate Tcf/Lef reporters to supraphysiological levels27, which would indicate that BMP not only specifies proximal states but also more broadly controls WNT/β-catenin signaling levels.FGF8/Fgf8 and FGFR1/Fgfr1 expression patterns appear conserved in developing mouse and human nephrons, suggesting that FGF8 binds to and signals through FGFR1 in the developing mammalian nephron, and in this study. In mouse perturbation experiments, Fgf8 is necessary for the survival of tubular nephron cells and is required for loop of Henle cells to form52. This function is underscored by localization of human FGF8 to the distal/medial boundary in the late SSB -where the loop of Henle is thought to originate—which may account for the bias of our DE organoids towards loop of Henle/macula densa precursors, and reduction of SLC12A1+ tubules upon suppression of FGF signaling (Fig. S8e). In mice, Fgf8 is a direct target of β-catenin72, and it is one of the first genes upregulated in the human PTA, adjacent to the WNT9B-expressing collecting duct54,72. Given that the WNT-producing collecting duct becomes further distanced to the distal nephron tubule during nephrogenesis (separated by the forming connecting tubule), it is therefore unlikely that the FGF8/Fgf8 expression, which propagates along the nephron axis in a proximal direction, is directly driven by WNT ligands. The proximal moving expression of FGF8 is also reminiscent of the proximal wave displayed by Notch ligand JAG13,73, suggesting either interplay or additional factors controlling signaling along the growing nephron tubule. While not the focus of this study, Notch is broadly required for nephron differentiation61,63 and, in particular, proximal specification defined by the Jag1high/Hnf1bhigh cell state preceding expression of HNF4A/Hnf4a in mouse and human SSBs32,63. Inhibition of Notch in our in vitro kidney organoid system replicated phenotypes observed in murine kidneys with deleted Notch1 and Notch2 function63—control organoids exhibited developmental arrest at the RV-like stage, and in DE organoids, TFAP2A expression was maintained, but further nephron segmentation was not evident. In mice, Notch2 signaling is absent in the distal limb of the SSB and is not required for distal tubule formation16,33,63,74, which can explain why TFAP2A expression was unaffected in DAPT-treated DE organoids.Together, these experiments offer a layered view of the complex dynamics of signaling at play, determining cell fates in the mammalian nephron. However, further analyses demonstrating the mechanisms by which both FGF and NOTCH signaling shift proximally, and their interactions with each other and WNT and BMP pathways are therefore required. Roles of other FGF receptors expressed during nephron development, i.e., FGFR2 and 3, are also worth further investigation as their interactions with FGF ligands in this context remain largely unknown.Limitations of studyThe kidney organoid models presented here show an effective and reproducible approach to pathway modulations and axial tuning using small-molecule inhibitors. A challenging aspect of this study is characterizing the molecular and cellular targets affected by our pathway changes, which are compounded by the interplay between different pathways as well as the inherent possibility of off-target effects by small molecule inhibitors, e.g., potential simultaneous targeting of FGFR2 and FGFR3 with SU-402. Future genetic studies detailing FGF ligand and receptor pairings during nephron patterning would help resolve this. Our model consists primarily of nephron and interstitial cells, and the inhibitors can therefore target both populations simultaneously. Developing new approaches, such as those described by Huang et al. 75, where iPSC-derived nephron progenitors are propagated in isolation and then induced to form nephrons, could be a powerful future approach to directly target only nephron cells. However, in in vivo environments, the constant crosstalk between the developing nephron and surrounding non-nephron cells is likely critical for normal patterning. A strength of protocols that generate more homogenous cell populations is the ability to interrogate and use cells for downstream applications, such as scrutiny of physiologies of SLC12A1-expressing cells or detailing genetics of disease-associated mutations, such as those driving polycystic kidney disease76. The flip-side of this is that the homogeneity carries a reduced potential to detail communication between cells of different types—as is evident from loss of RENIN-producing cells in the DE-organoid, cells common in controls. While our models provide mechanisms for axial nephron patterning and targeted precursor enrichment, more work is required to delineate the complex cell-to-cell communication that is expected in the in vivo kidney, as well as determining the maturation cues necessary for emerging physiologies.

Generating functional iPSC-derived nephrons requires optimized self-organization and identifying external cues that control formation of physiology-imparting cell fates. Here, we investigate the nephron axial polarity and signals that tune its development, and we demonstrate its considerable plasticity. A common approach to generate symmetry breaks along an axis is by modifying developmental pathways (WNT, FGF, BMP, Notch). For instance, WNT and FGF agonists tune axial patterning in embryo models64,65, while changes to BMP, WNT, and FGF signaling define midbrain-hindbrain boundaries66. In the nephron, WNT agonists and antagonists have been used in ex vivo and in vitro studies to control nephron patterning27,67,68, while small molecule inhibitors added to kidney organoid cultures can expand specific domains32,67,69,70. Our recent work demonstrated that providing signal inputs in the form of ligand-secreting synthetic cellular organizers can locally control axial identities and axial morphogenesis54. Here, we show that the nephron axis is controlled by WNT, FGF, and BMP signaling, which can be tuned to generate cell identities on demand.

Given its rapid and reproducible patterning, the nephron tubule is ideal for understanding how signal transduction regulates cell type specification in mammalian development. We adopted a kidney organoid model28, developed a protocol to direct differentiation towards distal nephron cell fates, and demonstrate that WNT and FGF pathways cooperatively drive distal nephron identities. Our distal-enriched (DE) protocol differentiated distal axial identities in an imposed transient WNTON and BMPOFF state that autonomously activated FGF8, and by auto- or paracrine FGF signaling, drove early distalization, priming thick ascending loop of Henle-like cell fates. In DE organoids where FGF signaling was blocked (DE + FGFOFF), the nephroids reverted their distal axial programming to instead form proximal tubule and podocyte precursors, fates otherwise not found in DE nephroids. These findings show that FGF signaling acts as a dominant pathway in DE conditions and promotes distal identities by outcompeting or actively suppressing proximal axial fates. Taking this one step further, in DE and FGFOFF organoids where BMP/SMAD signaling is also blocked (i.e., DE and sustained FGFOFF + BMPOFF), cells reverted to distal axial identities, thus highlighting a BMP/SMAD-mediated role in forming proximal cell fates. Our findings illustrate a model where nephron patterning is set up by opposing distalizing WNT/FGF and proximalizing BMP signaling, supporting the hypothesis that FGF and BMP compete to drive nephron fates.

It is not known how these signaling pathways interact to define nephron segment boundaries and specify axial nephron identities. Mouse models show that WNT/β-catenin, BMP, FGF, and Notch signaling are all required for nephron patterning18–21. We know that nephrons develop their distal-to-proximal identity against the collecting duct2,71, and recent human spatial transcriptomic data show that WNT/β-catenin target genes are activated in the earliest stages of nephrogenesis in those nephron cells closest to the WNT9B expressing collecting duct, and that iPSC-derived nephroids respond to canonical WNTs by driving distal axial cell fates54. The initial PD symmetry break is therefore likely driven by canonical WNT ligands. At the other end of the early nephron, we show that cells display active BMP/SMAD signaling, which is absent in distal cells. These two poles set up an early axial asymmetry with distal β-catenin and proximal BMP/SMAD signaling. Consistent with our data, dividing the axis into WNTON/BMPOFF and WNTOFF/BMPON poles, are mouse experiments demonstrating that activation of WNT/β-catenin and simultaneous inhibition of BMP/SMAD suppresses podocyte cell fates while driving a Cdh1+ distal fate27.

A WNT-BMP axial division is reflected in control organoids, which display high pSMAD throughout nephroids and infrequently form distal domains. Shifting these cells to a WNTON/ BMPOFF state between days 11 and 12 is sufficient to enforce a distal-dominant program leading to cells adopting a distal axial identity. What remains unclear is why mouse nephron cells in WNTON/BMPOFF states upregulate Tcf/Lef reporters to supraphysiological levels27, which would indicate that BMP not only specifies proximal states but also more broadly controls WNT/β-catenin signaling levels.

FGF8/Fgf8 and FGFR1/Fgfr1 expression patterns appear conserved in developing mouse and human nephrons, suggesting that FGF8 binds to and signals through FGFR1 in the developing mammalian nephron, and in this study. In mouse perturbation experiments, Fgf8 is necessary for the survival of tubular nephron cells and is required for loop of Henle cells to form52. This function is underscored by localization of human FGF8 to the distal/medial boundary in the late SSB -where the loop of Henle is thought to originate—which may account for the bias of our DE organoids towards loop of Henle/macula densa precursors, and reduction of SLC12A1+ tubules upon suppression of FGF signaling (Fig. S8e). In mice, Fgf8 is a direct target of β-catenin72, and it is one of the first genes upregulated in the human PTA, adjacent to the WNT9B-expressing collecting duct54,72. Given that the WNT-producing collecting duct becomes further distanced to the distal nephron tubule during nephrogenesis (separated by the forming connecting tubule), it is therefore unlikely that the FGF8/Fgf8 expression, which propagates along the nephron axis in a proximal direction, is directly driven by WNT ligands. The proximal moving expression of FGF8 is also reminiscent of the proximal wave displayed by Notch ligand JAG13,73, suggesting either interplay or additional factors controlling signaling along the growing nephron tubule. While not the focus of this study, Notch is broadly required for nephron differentiation61,63 and, in particular, proximal specification defined by the Jag1high/Hnf1bhigh cell state preceding expression of HNF4A/Hnf4a in mouse and human SSBs32,63. Inhibition of Notch in our in vitro kidney organoid system replicated phenotypes observed in murine kidneys with deleted Notch1 and Notch2 function63—control organoids exhibited developmental arrest at the RV-like stage, and in DE organoids, TFAP2A expression was maintained, but further nephron segmentation was not evident. In mice, Notch2 signaling is absent in the distal limb of the SSB and is not required for distal tubule formation16,33,63,74, which can explain why TFAP2A expression was unaffected in DAPT-treated DE organoids.

Together, these experiments offer a layered view of the complex dynamics of signaling at play, determining cell fates in the mammalian nephron. However, further analyses demonstrating the mechanisms by which both FGF and NOTCH signaling shift proximally, and their interactions with each other and WNT and BMP pathways are therefore required. Roles of other FGF receptors expressed during nephron development, i.e., FGFR2 and 3, are also worth further investigation as their interactions with FGF ligands in this context remain largely unknown.

Limitations of studyThe kidney organoid models presented here show an effective and reproducible approach to pathway modulations and axial tuning using small-molecule inhibitors. A challenging aspect of this study is characterizing the molecular and cellular targets affected by our pathway changes, which are compounded by the interplay between different pathways as well as the inherent possibility of off-target effects by small molecule inhibitors, e.g., potential simultaneous targeting of FGFR2 and FGFR3 with SU-402. Future genetic studies detailing FGF ligand and receptor pairings during nephron patterning would help resolve this. Our model consists primarily of nephron and interstitial cells, and the inhibitors can therefore target both populations simultaneously. Developing new approaches, such as those described by Huang et al. 75, where iPSC-derived nephron progenitors are propagated in isolation and then induced to form nephrons, could be a powerful future approach to directly target only nephron cells. However, in in vivo environments, the constant crosstalk between the developing nephron and surrounding non-nephron cells is likely critical for normal patterning. A strength of protocols that generate more homogenous cell populations is the ability to interrogate and use cells for downstream applications, such as scrutiny of physiologies of SLC12A1-expressing cells or detailing genetics of disease-associated mutations, such as those driving polycystic kidney disease76. The flip-side of this is that the homogeneity carries a reduced potential to detail communication between cells of different types—as is evident from loss of RENIN-producing cells in the DE-organoid, cells common in controls. While our models provide mechanisms for axial nephron patterning and targeted precursor enrichment, more work is required to delineate the complex cell-to-cell communication that is expected in the in vivo kidney, as well as determining the maturation cues necessary for emerging physiologies.

Limitations of study

The kidney organoid models presented here show an effective and reproducible approach to pathway modulations and axial tuning using small-molecule inhibitors. A challenging aspect of this study is characterizing the molecular and cellular targets affected by our pathway changes, which are compounded by the interplay between different pathways as well as the inherent possibility of off-target effects by small molecule inhibitors, e.g., potential simultaneous targeting of FGFR2 and FGFR3 with SU-402. Future genetic studies detailing FGF ligand and receptor pairings during nephron patterning would help resolve this. Our model consists primarily of nephron and interstitial cells, and the inhibitors can therefore target both populations simultaneously. Developing new approaches, such as those described by Huang et al. 75, where iPSC-derived nephron progenitors are propagated in isolation and then induced to form nephrons, could be a powerful future approach to directly target only nephron cells. However, in in vivo environments, the constant crosstalk between the developing nephron and surrounding non-nephron cells is likely critical for normal patterning. A strength of protocols that generate more homogenous cell populations is the ability to interrogate and use cells for downstream applications, such as scrutiny of physiologies of SLC12A1-expressing cells or detailing genetics of disease-associated mutations, such as those driving polycystic kidney disease76. The flip-side of this is that the homogeneity carries a reduced potential to detail communication between cells of different types—as is evident from loss of RENIN-producing cells in the DE-organoid, cells common in controls. While our models provide mechanisms for axial nephron patterning and targeted precursor enrichment, more work is required to delineate the complex cell-to-cell communication that is expected in the in vivo kidney, as well as determining the maturation cues necessary for emerging physiologies.

MethodsHuman kidney studiesKidney samples were collected under Institutional Review Board-approved protocols (USC-HS-13-0399 and CHLA-14-2211) for previous studies14,77,78. Following patient decision for termination, informed consent for donation of the products of conception for research purposes was obtained, and samples were collected without patient identifiers. The person obtaining informed consent was different than the physician performing the termination procedure, and the decision to donate tissue did not impact the method of termination. Developmental age was determined according to the American College of Obstetrics and Gynecology guidelines79–81. Whole intact kidneys were delivered for analysis on ice at 4 °C in high glucose DMEM (Corning, 10-017-CV) supplemented with 10% fetal bovine serum (GenClone) and 25 mM HEPES (Cytiva, SH30237.01).Human induced pluripotent stem cells (HiPSCs)iPSCs derived from human foreskin fibroblasts (FiPSC #3C15, ATCC #CRL-2522) were generously gifted by Dr Amy L. Ryan (University of Iowa). The fibroblast-derived HNF4A-YFP iPSCs (male) were obtained from the ReBuilding a Kidney (RBK) Consortium (ATCC # PCS-201-010).HiPSC-derived kidney organoid differentiationiPSCs are cultured in Matrigel (Corning, 354277) -coated 12-well plates in Essential 8 medium (Gibco, A1517001) supplemented with Y-27632 (Tocris, 1254) for 24 h, after which medium is replaced with Essential 8 medium. When the cells are about 60% confluent (~48 h), iPSCs are passaged and seeded for differentiation into kidney organoids using a modified Takasato protocol82. Briefly, on DD0, iPSCs are seeded at a density of 17.5k cells/well in a Matrigel-coated 12-well plate in Essential 8 medium supplemented with 5 μM Y-27632. After 6 h, 9 μM CHIR99021 (Tocris, 4423) in TeSR-E6 (STEMCELL Technologies, 05946) is added and replenished every other day for 5 days, followed by TeSR-E6 containing 200 ng/ml FGF9 (Bio-techne, 273-F9-025) and 100 ng/ml Heparin (Sigma-Aldrich, H4784) between DD5 and DD7. On DD7, cell aggregation is achieved by adding 30 μl of 100,000 cells in TeSR-E6 containing 10 μM Y-27632 into a round-bottom low-attachment 96-well plate. The plate is spun at 100 × g for 2 min in a swing bucket centrifuge and incubated at 37 °C for 2 h. The aggregates are then transferred into 6-well Transwell plates (STEMCELL Technologies, 100-1026), following the Takasato protocol. For our distal-enrichment protocol, 10 μM LY294002 (Tocris, 1130) is added to the TeSR-E6 medium supplemented with FGF9 and Heparin from DD10-11. On DD11, TeSR-E6 medium containing FGF9 and Heparin is supplemented with 10 μM LY294002, 1.5 μM CHIR99021, and 1 μM LDN 193189 dihydrochloride (Tocris, 6053) and added to these organoids. On DD12, medium is changed to TeSR-E6 with no growth factors for both control and treated groups. Organoids are cultured until DD28.For validation experiments of PI3K inhibition, 10 μM LY29 is replaced with 0.75 μM GDC-0941 (Selleck, S1065) from DD10-DD12, with 1.5 μM CHIR and 1 μM LDN added to this media from DD11-DD12. For our BMP inhibition validation in our DE protocol, organoids were treated with 10 μM LY29 (DD10-DD12), to which 1.5 μM CHIR and 2.5 μM Dorsomorphin (Sigma-Aldrich, P5499) were added from DD11-DD12. For our FGFR1 inhibition experiments, 1 µM SU-5402 (Sigma-Aldrich, SML0443) is added from DD11-DD12 to TeSR-E6 supplemented with FGF9, Heparin, LY29, CHIR, and LDN in the amounts specified above, then TeSR-E6 with no growth factors from DD12-DD14. FGFR1 inhibition experiments were repeated with 100 nM PD-166866 (Sigma-Aldrich, PZ0114).HNF4A-YFP iPSCs48 were differentiated with the same differentiation protocol described above, with the exception of DD0-DD5, where they were treated with 7 μM CHIR99021, based on the differentiation protocol outlined in Schnell et al.32.Immunofluorescence analysisImmunostaining of human kidney cryosectionsFrozen human week 18.6 kidney cryosections were thawed at room temperature (RT) for ~5 min. Antigen retrieval was performed using 1X Citrate buffer (Sigma-Aldrich) on high pressure for 1 min in a pressure cooker. The slides were rinsed with deionized water and incubated for 1 h in blocking buffer solution (1.5% SEA BLOCK blocking buffer (Thermo Scientific, 37527 × 3) + 0.1% Triton X-100 (MD Millipore, 108643) in 1X PBS (Thermo Scientific, 10010049)). The slides were incubated overnight at 4 °C in primary antibodies diluted in blocking buffer solution (see ‘Immunostaining of Whole Kidney Organoids’ section below for dilutions used), rinsed in PBS-T (0.1% Triton X-100 in 1X PBS) and incubated in secondary antibodies diluted in blocking buffer solution for 1 h at room temperature; secondary antibodies conjugated with Alexa Fluor 488, 555, 594, and 647 were all diluted to 1:500. Once rinsed, the slides were incubated in 1 μg/ml Hoechst 33342 (Thermo Scientific, H3570) in PBS-T for 5 min at RT for nuclei staining, followed by a final rinse in 1X PBS. Sections were then mounted with Immu Mount (Thermo Scientific, 9990402).Immunostaining of whole kidney organoidsWhole organoids were fixed with 4% paraformaldehyde (Electron Microscopy Sciences, 15710) in 1X PBS—1.2 ml of 4% paraformaldehyde was added under the transwell and incubated for 10 min on ice, after which an additional 1 ml of 4% paraformaldehyde was added to the top of the organoids and incubated for 10 min on ice. After incubation, the 4% paraformaldehyde was replaced with cold 1X PBS. Fixed organoids were cut out of the transwells and transferred into blocking buffer solution. The organoids were incubated on a nutator at 4 °C for 1 h, after which they were incubated overnight on a nutator at 4 °C in primary antibodies diluted in blocking buffer solution. The primary antibodies used were as follows: Laminin β1 (Santa Cruz Biotechnology, sc-33709, 1:250), HNF4A (R&D Systems, MAB4605, 1:200), POU3F3 (Thermo Scientific, PA564311, 1:100), Mafb (Novus (R&D), MAB3810, 1:500), GATA3 (Novus (R&D), AF2605, 1:200), Jag1 (Novus (R&D), AF599, 1:100), SLC12A1 (Sigma-Aldrich, HPA018107, 1:200), PAX8 (abcam, ab189249, 1:100), WT1 (abcam, ab89901, 1:500), LEF1 (Cell signaling, 2286S, 1:200), CDH1 (BD Laboratories, 610181, 1:200), PAPPA2 (R&D Systems, AF1668, 1:100), HNF1B (Thermo Scientific, MA5-24605, 1:500), Pax2 (Thermo Scientific, 716000, 1:100), TFAP2A (Santa Cruz Biotechnology, sc-12726, 1:200), IRX1 (Sigma-Aldrich, HPA043160, 1:200), SIX1 (Cell signaling, 12891S, 1:300), Pax2 (R&D Systems, AF3364, 1:50), TMEM52B (Novus (R&D), NBP2-49272, 1:100), MEIS1/2/3 (Active Motif, 39096, 1:1000), ZO-1 (Thermo Scientific, 33-9100, 1:200), PODXL (R&D Systems, AF1658, 1:300), Renin (R&D Systems, AF4090, 1:100), MECOM (R&D Systems, MAB75061, 1:100), hErbB4 (R&D Systems, MAB1131, 1:100), LEF1 (Santa Cruz Biotechnology, sc-374412, 1:200), CD31 (abcam, ab9498, 1:250), ESRRγ (Sigma-Aldrich, HPA044678, 1:100), GATA3 (Cell Signaling Technology, 5852 T, 1:100), and acetyl-alpha tubulin (Sigma-Aldrich, MABT868, 1:500). After a 3x wash in PBS-T (1 hr each), organoids were incubated overnight on a nutator at 4 °C in secondary antibodies diluted in blocking buffer solution. Secondary antibodies conjugated with Alexa Fluor 488, 555, 594, and 647 were all diluted to 1:500. Following incubation, the organoids were washed 3x in PBS-T (1 h each), followed by a 35-min incubation on a nutator at 4 °C in 1 μg/ml Hoechst 33342 in PBS-T. After a final rinse in 1X PBS, organoids were serially dehydrated in 50%, 75% and 100% methanol in 1X PBS for 30 min each, then incubated at RT in 50% methanol in a BABB solution (benzyl alcohol (Sigma-Aldrich, 108006) and benzyl benzoate (Sigma-Aldrich, B6630) in a 1:2 ratio). The organoids were then fully cleared in 100% BABB and stored at 4 °C.Immunohistochemistry analysisParaffin embedding and sectioningWeek 16 human kidneys were fixed in 4% paraformaldehyde in 1X PBS for 1 h at 4 °C and washed 2x in 1X PBS for 5 min each. They were transferred to cassettes and stored in 70% ethanol in 4 °C and transferred to the USC Translational Research lab histology core for paraffin processing and sectioning in 5-μm sections. DD14 kidney organoids were fixed in 4% paraformaldehyde following the same protocol described in the “Immunofluorescence Analysis” section. The paraffin embedding protocol for organoids was adapted from Wörsdörfer et al.83. Briefly, fixed organoids were detached from Transwell membranes and embedded in 300 μL of 1% agarose (Avantor, 0710) in 1X PBS at 60 °C in 10 × 10 x 5 mm cryomolds (Tissue Tek, 4565) and allowed to set. The agarose blocks were transferred to tissue cassettes (Simport, M511-10) and stored in 70% ethanol at 4 °C overnight. Paraffin processing was done using the Tissue Tek VIP tissue processor (Sakura), and the agarose blocks were embedded into a paraffin block. The organoid blocks were sectioned into 5 μm-thick sections using a sliding microtome and dried overnight at 60 °C in a lab oven.Antibody staining and hematoxylin counterstainingParaffin sections were dewaxed with the following solution series: 100% Xylene (Sigma-Aldrich, 247642) for 10 min (2x), 100% ethanol for 1 min, 95% ethanol for 1 min, 80% ethanol for 1 min, 60% ethanol for 1 min, 30% ethanol for 1 min, and 2x rinse in deionized water. Antigen retrieval was performed as described in the ‘Immunofluorescence Analysis’ section, and the sections were incubated in 0.1% TBS-T (0.1% Tween-20 in 1X TBS (Cell Signaling Technology, 12498S)) + 1.5% SEA BLOCK blocking buffer for 1 h at RT. Sections were incubated in pSMAD1/5 (Cell Signaling, 9516 T, 1:100) diluted in 0.1% TBS-T + 1.5% SEA BLOCK blocking solution overnight at 4 °C. The slides were washed 2x in 0.1% TBS-T for 5 min each. Endogenous peroxidase activity was blocked with 3% hydrogen peroxide (Sigma-Aldrich, 216763) for 10 min at RT and rinsed 2x with deionized water. The slides were incubated in biotinylated secondary antibody in 0.1% TBS-T + 1.5% SEA BLOCK blocking solution for 30 min, then washed 3x with 0.1% TBS-T for 5 min each. An avidin-biotin solution was prepared from the VECTASTAIN ABC kits (Vector Laboratories, PK-6100) using manufacturer instructions to visualize biotinylated secondary antibodies, and slides were incubated in ABC solution for 30 min at RT. Slides were washed again 3x with 0.1% TBS-T for 5 min each. The slides were developed using the ImmPACT DAB substrate kit (Vector Laboratories, SK-4105) for 30 min, and development was stopped by plunging the slide into deionized water. The slides were counterstained with hematoxylin (Sigma-Aldrich, 51275) for 10 s, rinsed with deionized water, and mounted using Immu Mount.Image acquisition and analysisFluorescent immunostains were imaged on the Leica SP8 with DLSM (Leica) and Stellaris 8 (Leica) confocal microscopes. Z-stack images were acquired at 3 μm intervals. Image processing for images taken on Leica microscopes was done on LAS X image analysis software (Leica). The immunohistochemistry images were taken on the ZEISS Axioscan 7 (ZEISS) and processed on ZEISS ZEN lite image analysis software (ZEISS v3.4).Spatial genomics analysis on human kidney samplesWeek 16 human kidneys embedded in OCT (Electron Microscopy Sciences, 4583) were cryosectioned at 10 μm thickness and placed on coverslips provided by Spatial Genomics. They were fixed in 4% paraformaldehyde in PBS for 15 min at RT. Sections were washed 4 times in 1x PBS for 5 min at RT and dehydrated in 100% isopropanol for 30 s at RT. The sections were air dried, stored at −80 °C, and shipped to Spatial Genomics, Inc. in dry ice for sequencing. The custom probe set used consisted of 250 cell-state and/or lineage-defining genes that represented all 5 nephron cell types54. These genes were derived from differential gene expression analysis of human single-nucleus RNA sequencing data. Data collection and imaging were performed by Spatial Genomics, and visualization was done using the Spatial Genomics Viewer software, with screengrabs obtained at 0.33x zoom.Bulk mRNA sequencing and analyses of human kidney organoidsSample collection and preparationAll bulk RNA samples were collected in duplicate (2 biological replicates). Kidney organoids were collected at select timepoints (2–3 organoids pooled per condition) into a 1:100 solution of lysis buffer RLT (Qiagen, 79216) and β-Mercaptoethanol (Sigma-Aldrich, M3148) and homogenized by pipetting for 30 s. RNA was isolated from samples using the Qiagen RNeasy kit for RNA purification (Qiagen, 74104), and total RNA was quantified on a NanoDrop spectrophotometer (Thermo Scientific). mRNA from the samples was sequenced by Novogene using the Illumina NovoSeq X Plus PE153 platform (Illumina).Data processingRaw FASTQ files were aligned to the human genome using STAR alignment84 (v2.7.10a), and feature counts were generated from BAM files using the featureCounts function of the Rsubread package85. TPM normalization was performed, and genes were annotated on Partek Flow, where .txt files were generated with TPM values for each gene in each sample. For generating heatmaps of bulk mRNA sequencing data, a matrix of TPM values was loaded on R, and mitochondrial and ribosomal genes and gene rows with TPM values of <10 were excluded. The pheatmap function of the pheatmap package (v1.0.12) was used to generate heatmaps that used hierarchical clustering. The DESeqDataSetFromMatrix function from the DESeq2 package86 (v1.42.1) was used to perform DESeq analysis on paired samples, and EnhancedVolcano function (v1.20.0) generated volcano plots, where a p value cutoff of 0.05 and a log2 fold change cutoff of 1.5 was set.Single-cell RNA sequencing and analyses of DD14 human kidney organoidsSample collection and preparationKidney organoids were collected at select time-points (9 organoids each for DD14 control and DE organoids) and dissociated into single cells using Accumax (STEMCELL Technologies, 07921) for 45 min at 37 °C, with the organoids pipetted at 5-min intervals with a P1000 tip 10 times. Once the cells were fully dissociated, the Accumax enzyme was neutralized using twice the volume of cold AutoMacs buffer (Miltenyi, 130-091-221). The tubes were centrifuged at 300 × g for 3 min at 4 °C, the supernatant was aspirated, and the dissociated cells were resuspended in 2 ml of cold AutoMacs buffer. The cells were strained using a 40 μm cell strainer and chased with 1 ml of cold AutoMacs buffer, then centrifuged again at 300 × g for 3 min at 4 °C. The supernatant was aspirated, and the cells were resuspended with 500 μl of cold AutoMacs buffer containing nuclear dyes DRAQ5 (Thermo Scientific, 62254) (1:1000) and DAPI (Thermo Scientific, D1306) (1:1000) and FAC-sorted for live cells on the ARIA II FACS (BD Biosciences). The resuspended cells were transferred to round-bottom 15-ml tubes. Using dissociated cells resuspended in AutoMacs buffer as a negative control for gating, the dissociated cells resuspended in the nuclear dyes were sorted for single-live cells (DRAQ5+, DAPI-) for collection. These cells were then fixed using the Evercode Cell Fixation kit (Parse Biosciences, ECF2001) and provided protocol. For each datapoint, 7 sublibraries were prepared for SPLiT-seq (v1.2.1) using the Evercode WT v2 kit (Parse Biosciences, ECW02030) and protocol and sequenced separately by Novogene using the Illumina NovaSeq X Plus 25B PE150 platform (Illumina) with a depth of ~63k reads/cell. Raw FASTQ files of each sublibrary were demultiplexed and aligned to the human genome using split-pipe from Parse Biosciences, where corresponding conditions were assigned to each sublibrary.Seurat object processingSeurat objects were made for each sublibrary using the Seurat package (V4) in R87, and initial quality control metrics were run to cutoff low quality cells (1000–8000 genes per cell, percent mitochondria <20). For each condition, the 7 Seurat objects (1 object per sublibrary) were merged using merge function, and no batch effects were observed. However, cycling genes were significantly contributing to cell heterogeneity as cells separated by cell cycling phases, thus we log-normalized the merged objects and regressed out cell cycle scores88 while scaling features in our datasets using the ScaleData function. Non-linear dimensionality reduction was performed using RunUMAP and FindNeighbors using 30 principal components. The objects were clustered at a resolution of 1.5. For objects from each condition, differential gene expression analysis was performed on log-normalized transcripts using FindAllMarkers function, where only genes detected in a minimum of 25% of cells in either population were tested, and the cutoff for log fold change was 0.25. Post-quality control, the Seurat objects contained 8219 control cells and 9301 DE cells. From the control organoid Seurat object, cells from clusters 21, 25, 15, 17, 6, 24, and 12 were identified as interstitial and subset out to retain nephron-only clusters. Similarly, from the DE organoid Seurat object, clusters 24, 4, 19, 18, 12, and 20 were subset out. In each instance, Nephron-only subset objects were reclustered at a resolution of 1.5.Nephron-only objects from control and DE organoids were merged using the merge function, where cell cycle regression was performed again during scaling. Differential gene expression analysis was performed on log-normalized transcripts using the same criteria described above. DESeq-derived differential gene expression between control and DE nephron cells was determined using the FindMarkers function, and the results were plotted on a volcano plot using the EnhancedVolcano function, where a p value cutoff of 0.5 and log2 fold change cutoff of 1.5 were set.Reanalysis of published developing human single-cell RNA sequencing dataSingle cell RNA sequencing data of week 1449 and week 171 human kidney were downloaded from the Gene Expression Omnibus (accession numbers: GSE139280, GSE127344). The Seurat package (v3) was used for quality control analyses, and SCTransform data integration89 of the 2 datasets due to batch effects. Based on differential gene expression of known kidney cell type markers, clusters were annotated with corresponding cell types, and differential gene expression analysis was performed using FindAllMarkers.Transcription factor analysisFor analysis of distal nephron-enriched transcription regulators, we examined gene expression levels of transcription factors from the human90 and mouse91 transcription factor atlases—the mouse atlas was converted into its human ortholog using Ensembl BioMart92 as described in Kim et al.5. The enrichment of transcription factor detection in TFAP2A+ clusters was shortlisted by feature plot visualization and represented in Fig S5.Single-nucleus RNA sequencing and analyses of DD28 human kidney organoidsSample collection and preparationKidney organoids were collected at DD28 (6 organoids each for DD28 control and DE organoids), pooled, and immediately snap-frozen in liquid nitrogen. Nuclei isolation was conducted as previously described5, and sequencing was performed using 10X ChromiumTM GEM-X technology. Raw FASTQ files of each condition were demultiplexed and aligned to the NCBI human reference genome (GRCh38-2024-A) using Cell Ranger (v9.0.0).Seurat object processingSeurat objects were made for each condition using Seurat’s Read10X function, and initial quality control metrics were run to cut off low-quality cells (1000–8000 genes per cell, percent mitochondria <10). For each object, log-normalization was used to normalize the count data with a scale factor of 10,000. The ScaleData function was used to scale features in our datasets, and non-linear dimensionality reduction was performed using RunUMAP and FindNeighbors using 30 principal components. The objects were initially clustered at a resolution of 3 to identify clusters with outlying feature counts (low-quality cells or doublets), which were then subset out to generate the final objects. For objects from each condition, differential gene expression analysis was performed on log-normalized transcripts using FindAllMarkers function, where only genes detected in a minimum of 25% of cells in either population were tested, and the cutoff for log fold change was 0.25. Post-quality control, the Seurat objects contained 5374 control cells and 3153 DE cells. In each object, Nephron-only subset objects were reclustered at a resolution of 1.Nephron-only objects from control and DE organoids were merged using the merge function. However, clustering revealed batch effects, thus CCA integration was performed using the IntegrateLayers function. The layers were re-joined post-integration, and the normalization analysis workflow was performed with the “integrated.cca” reduction used as the input for building the shared nearest neighbors (SNN) network. Differential gene expression analysis was performed on log-normalized transcripts using the same criteria described above. DESeq-derived differential gene expression between control and DE nephron cells was determined using the FindMarkers function, and the results were plotted on a volcano plot using the EnhancedVolcano function, where a p-value cutoff of 0.5 and log2 fold change cutoff of 1.5 were set.Reanalysis of published adult kidney single-nucleus RNA sequencing dataThis study features single-nucleus RNA sequencing of 9 healthy adult human kidneys generated as part of the Kidney Precision Medicine Project [https://atlas.kpmp.org/explorer/] – this data is available for download on the Gene Expression Omnibus (accession number: GSE183279) and features datasets with the following experimental IDs: BUKMAP_20190529L_10X-R, BUKMAP_20190607L_10X-R, BUKMAP_20190822F_10X-R, BUKMAP_20190829B_10X-R, BUKMAP_20191104A-10X-R, BUKMAP_20200205A_10X-R, BUKMAP_20200205D_10X-R, BUKMAP_20200205F_10X-R, BUKMAP_20200304A_10X-R, BUKMAP_20200304B_10X-R, BUKMAP_20200304F_10X-R, KPMP_20190607I-10X-R, KPMP_20190607J-10X-R, KPMP_20190607K-10X-R.The Seurat package (v5) was used for SC Transformation normalization. The metadata contains different groups of annotated cell types and subtypes (described in Lake et al.50)—these annotations were maintained for reanalysis. The grouping of ‘class’ was used to subset kidney epithelia from stromal and endothelial cells, such that only the nephrogenic and ureteric lineages remained (44,774 cells).Multimodal reference mapping of human and organoid RNA sequencing dataDD14 organoids and developing human single-cell RNA sequencing dataThe annotated clusters of the single-cell human kidney Seurat object were used to predict cell types in the DD14 control and DE nephroid datasets following the Seurat v4 reference mapping vignette developed by the Satija Lab93. Briefly, the merged organoid dataset was split by condition and treated as separate query samples for finding anchors to map them to the human kidney reference. Mapping each organoid (query) cell to the kidney organoid (reference) dataset is based on its k-nearest neighbors, for which the default, 5, was used. Seurat determines a prediction score for each assigned label to represent the confidence score (from 0 to 1) of how well organoid cells’ features (genes) align with the human kidney reference datasets. When determining transfer anchors between the reference and query, no anchors were filtered, and 50 dimensions were used from principal component analysis (PCA) reduction. The ggplot package was used for visualization of the number of cells in each condition that were represented in the predicted cell types.DD28 organoids and adult human single-nucleus RNA sequencing dataThe “subclass.l1” cell-type annotation in the adult human dataset was used to predict cell types in the DD28 control and DE nephroid datasets following the Satija lab’s Seurat v4 reference mapping vignette. The same reference mapping script and cell-type prediction scoring parameters were used as described above for the DD14 organoid and developing human dataset.Quantification and statistical analysisQuantification of symmetric and asymmetric nephroids was performed on ImageJ (Fiji) using the Cell Counter plugin. Statistical analysis shown on all bar plots was performed using unpaired parametric t-test (GraphPad Prism v10.0.0) for biological replicates (n = 2), where the p value cut-off is 0.05. All p values calculated are included on each graph.Quantification of fluorescence detection of MAFB, HNF4A, TFAP2A, and GATA3 was performed on ImageJ (Fiji) using the Cell Counter plugin. Single-plane organoid quadrant images were selected for this analysis, and for each image, 12 visible nephroids were selected with a DAPI background. Fluorescent cells and total cells within each nephroid were counted and aggregated to calculate percentage fluorescent cells. For each condition, images from 2 organoids (biological replicates) were quantified.Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Nils O. Lindström (nils.lindstrom@med.usc.edu).Reporting summaryFurther information on research design is available in the Nature Portfolio Reporting Summary linked to this article.

Human kidney studiesKidney samples were collected under Institutional Review Board-approved protocols (USC-HS-13-0399 and CHLA-14-2211) for previous studies14,77,78. Following patient decision for termination, informed consent for donation of the products of conception for research purposes was obtained, and samples were collected without patient identifiers. The person obtaining informed consent was different than the physician performing the termination procedure, and the decision to donate tissue did not impact the method of termination. Developmental age was determined according to the American College of Obstetrics and Gynecology guidelines79–81. Whole intact kidneys were delivered for analysis on ice at 4 °C in high glucose DMEM (Corning, 10-017-CV) supplemented with 10% fetal bovine serum (GenClone) and 25 mM HEPES (Cytiva, SH30237.01).

Human kidney studies

Kidney samples were collected under Institutional Review Board-approved protocols (USC-HS-13-0399 and CHLA-14-2211) for previous studies14,77,78. Following patient decision for termination, informed consent for donation of the products of conception for research purposes was obtained, and samples were collected without patient identifiers. The person obtaining informed consent was different than the physician performing the termination procedure, and the decision to donate tissue did not impact the method of termination. Developmental age was determined according to the American College of Obstetrics and Gynecology guidelines79–81. Whole intact kidneys were delivered for analysis on ice at 4 °C in high glucose DMEM (Corning, 10-017-CV) supplemented with 10% fetal bovine serum (GenClone) and 25 mM HEPES (Cytiva, SH30237.01).

Human induced pluripotent stem cells (HiPSCs)iPSCs derived from human foreskin fibroblasts (FiPSC #3C15, ATCC #CRL-2522) were generously gifted by Dr Amy L. Ryan (University of Iowa). The fibroblast-derived HNF4A-YFP iPSCs (male) were obtained from the ReBuilding a Kidney (RBK) Consortium (ATCC # PCS-201-010).

Human induced pluripotent stem cells (HiPSCs)

iPSCs derived from human foreskin fibroblasts (FiPSC #3C15, ATCC #CRL-2522) were generously gifted by Dr Amy L. Ryan (University of Iowa). The fibroblast-derived HNF4A-YFP iPSCs (male) were obtained from the ReBuilding a Kidney (RBK) Consortium (ATCC # PCS-201-010).

HiPSC-derived kidney organoid differentiationiPSCs are cultured in Matrigel (Corning, 354277) -coated 12-well plates in Essential 8 medium (Gibco, A1517001) supplemented with Y-27632 (Tocris, 1254) for 24 h, after which medium is replaced with Essential 8 medium. When the cells are about 60% confluent (~48 h), iPSCs are passaged and seeded for differentiation into kidney organoids using a modified Takasato protocol82. Briefly, on DD0, iPSCs are seeded at a density of 17.5k cells/well in a Matrigel-coated 12-well plate in Essential 8 medium supplemented with 5 μM Y-27632. After 6 h, 9 μM CHIR99021 (Tocris, 4423) in TeSR-E6 (STEMCELL Technologies, 05946) is added and replenished every other day for 5 days, followed by TeSR-E6 containing 200 ng/ml FGF9 (Bio-techne, 273-F9-025) and 100 ng/ml Heparin (Sigma-Aldrich, H4784) between DD5 and DD7. On DD7, cell aggregation is achieved by adding 30 μl of 100,000 cells in TeSR-E6 containing 10 μM Y-27632 into a round-bottom low-attachment 96-well plate. The plate is spun at 100 × g for 2 min in a swing bucket centrifuge and incubated at 37 °C for 2 h. The aggregates are then transferred into 6-well Transwell plates (STEMCELL Technologies, 100-1026), following the Takasato protocol. For our distal-enrichment protocol, 10 μM LY294002 (Tocris, 1130) is added to the TeSR-E6 medium supplemented with FGF9 and Heparin from DD10-11. On DD11, TeSR-E6 medium containing FGF9 and Heparin is supplemented with 10 μM LY294002, 1.5 μM CHIR99021, and 1 μM LDN 193189 dihydrochloride (Tocris, 6053) and added to these organoids. On DD12, medium is changed to TeSR-E6 with no growth factors for both control and treated groups. Organoids are cultured until DD28.For validation experiments of PI3K inhibition, 10 μM LY29 is replaced with 0.75 μM GDC-0941 (Selleck, S1065) from DD10-DD12, with 1.5 μM CHIR and 1 μM LDN added to this media from DD11-DD12. For our BMP inhibition validation in our DE protocol, organoids were treated with 10 μM LY29 (DD10-DD12), to which 1.5 μM CHIR and 2.5 μM Dorsomorphin (Sigma-Aldrich, P5499) were added from DD11-DD12. For our FGFR1 inhibition experiments, 1 µM SU-5402 (Sigma-Aldrich, SML0443) is added from DD11-DD12 to TeSR-E6 supplemented with FGF9, Heparin, LY29, CHIR, and LDN in the amounts specified above, then TeSR-E6 with no growth factors from DD12-DD14. FGFR1 inhibition experiments were repeated with 100 nM PD-166866 (Sigma-Aldrich, PZ0114).HNF4A-YFP iPSCs48 were differentiated with the same differentiation protocol described above, with the exception of DD0-DD5, where they were treated with 7 μM CHIR99021, based on the differentiation protocol outlined in Schnell et al.32.

HiPSC-derived kidney organoid differentiation

iPSCs are cultured in Matrigel (Corning, 354277) -coated 12-well plates in Essential 8 medium (Gibco, A1517001) supplemented with Y-27632 (Tocris, 1254) for 24 h, after which medium is replaced with Essential 8 medium. When the cells are about 60% confluent (~48 h), iPSCs are passaged and seeded for differentiation into kidney organoids using a modified Takasato protocol82. Briefly, on DD0, iPSCs are seeded at a density of 17.5k cells/well in a Matrigel-coated 12-well plate in Essential 8 medium supplemented with 5 μM Y-27632. After 6 h, 9 μM CHIR99021 (Tocris, 4423) in TeSR-E6 (STEMCELL Technologies, 05946) is added and replenished every other day for 5 days, followed by TeSR-E6 containing 200 ng/ml FGF9 (Bio-techne, 273-F9-025) and 100 ng/ml Heparin (Sigma-Aldrich, H4784) between DD5 and DD7. On DD7, cell aggregation is achieved by adding 30 μl of 100,000 cells in TeSR-E6 containing 10 μM Y-27632 into a round-bottom low-attachment 96-well plate. The plate is spun at 100 × g for 2 min in a swing bucket centrifuge and incubated at 37 °C for 2 h. The aggregates are then transferred into 6-well Transwell plates (STEMCELL Technologies, 100-1026), following the Takasato protocol. For our distal-enrichment protocol, 10 μM LY294002 (Tocris, 1130) is added to the TeSR-E6 medium supplemented with FGF9 and Heparin from DD10-11. On DD11, TeSR-E6 medium containing FGF9 and Heparin is supplemented with 10 μM LY294002, 1.5 μM CHIR99021, and 1 μM LDN 193189 dihydrochloride (Tocris, 6053) and added to these organoids. On DD12, medium is changed to TeSR-E6 with no growth factors for both control and treated groups. Organoids are cultured until DD28.

For validation experiments of PI3K inhibition, 10 μM LY29 is replaced with 0.75 μM GDC-0941 (Selleck, S1065) from DD10-DD12, with 1.5 μM CHIR and 1 μM LDN added to this media from DD11-DD12. For our BMP inhibition validation in our DE protocol, organoids were treated with 10 μM LY29 (DD10-DD12), to which 1.5 μM CHIR and 2.5 μM Dorsomorphin (Sigma-Aldrich, P5499) were added from DD11-DD12. For our FGFR1 inhibition experiments, 1 µM SU-5402 (Sigma-Aldrich, SML0443) is added from DD11-DD12 to TeSR-E6 supplemented with FGF9, Heparin, LY29, CHIR, and LDN in the amounts specified above, then TeSR-E6 with no growth factors from DD12-DD14. FGFR1 inhibition experiments were repeated with 100 nM PD-166866 (Sigma-Aldrich, PZ0114).

HNF4A-YFP iPSCs48 were differentiated with the same differentiation protocol described above, with the exception of DD0-DD5, where they were treated with 7 μM CHIR99021, based on the differentiation protocol outlined in Schnell et al.32.

Immunofluorescence analysisImmunostaining of human kidney cryosectionsFrozen human week 18.6 kidney cryosections were thawed at room temperature (RT) for ~5 min. Antigen retrieval was performed using 1X Citrate buffer (Sigma-Aldrich) on high pressure for 1 min in a pressure cooker. The slides were rinsed with deionized water and incubated for 1 h in blocking buffer solution (1.5% SEA BLOCK blocking buffer (Thermo Scientific, 37527 × 3) + 0.1% Triton X-100 (MD Millipore, 108643) in 1X PBS (Thermo Scientific, 10010049)). The slides were incubated overnight at 4 °C in primary antibodies diluted in blocking buffer solution (see ‘Immunostaining of Whole Kidney Organoids’ section below for dilutions used), rinsed in PBS-T (0.1% Triton X-100 in 1X PBS) and incubated in secondary antibodies diluted in blocking buffer solution for 1 h at room temperature; secondary antibodies conjugated with Alexa Fluor 488, 555, 594, and 647 were all diluted to 1:500. Once rinsed, the slides were incubated in 1 μg/ml Hoechst 33342 (Thermo Scientific, H3570) in PBS-T for 5 min at RT for nuclei staining, followed by a final rinse in 1X PBS. Sections were then mounted with Immu Mount (Thermo Scientific, 9990402).Immunostaining of whole kidney organoidsWhole organoids were fixed with 4% paraformaldehyde (Electron Microscopy Sciences, 15710) in 1X PBS—1.2 ml of 4% paraformaldehyde was added under the transwell and incubated for 10 min on ice, after which an additional 1 ml of 4% paraformaldehyde was added to the top of the organoids and incubated for 10 min on ice. After incubation, the 4% paraformaldehyde was replaced with cold 1X PBS. Fixed organoids were cut out of the transwells and transferred into blocking buffer solution. The organoids were incubated on a nutator at 4 °C for 1 h, after which they were incubated overnight on a nutator at 4 °C in primary antibodies diluted in blocking buffer solution. The primary antibodies used were as follows: Laminin β1 (Santa Cruz Biotechnology, sc-33709, 1:250), HNF4A (R&D Systems, MAB4605, 1:200), POU3F3 (Thermo Scientific, PA564311, 1:100), Mafb (Novus (R&D), MAB3810, 1:500), GATA3 (Novus (R&D), AF2605, 1:200), Jag1 (Novus (R&D), AF599, 1:100), SLC12A1 (Sigma-Aldrich, HPA018107, 1:200), PAX8 (abcam, ab189249, 1:100), WT1 (abcam, ab89901, 1:500), LEF1 (Cell signaling, 2286S, 1:200), CDH1 (BD Laboratories, 610181, 1:200), PAPPA2 (R&D Systems, AF1668, 1:100), HNF1B (Thermo Scientific, MA5-24605, 1:500), Pax2 (Thermo Scientific, 716000, 1:100), TFAP2A (Santa Cruz Biotechnology, sc-12726, 1:200), IRX1 (Sigma-Aldrich, HPA043160, 1:200), SIX1 (Cell signaling, 12891S, 1:300), Pax2 (R&D Systems, AF3364, 1:50), TMEM52B (Novus (R&D), NBP2-49272, 1:100), MEIS1/2/3 (Active Motif, 39096, 1:1000), ZO-1 (Thermo Scientific, 33-9100, 1:200), PODXL (R&D Systems, AF1658, 1:300), Renin (R&D Systems, AF4090, 1:100), MECOM (R&D Systems, MAB75061, 1:100), hErbB4 (R&D Systems, MAB1131, 1:100), LEF1 (Santa Cruz Biotechnology, sc-374412, 1:200), CD31 (abcam, ab9498, 1:250), ESRRγ (Sigma-Aldrich, HPA044678, 1:100), GATA3 (Cell Signaling Technology, 5852 T, 1:100), and acetyl-alpha tubulin (Sigma-Aldrich, MABT868, 1:500). After a 3x wash in PBS-T (1 hr each), organoids were incubated overnight on a nutator at 4 °C in secondary antibodies diluted in blocking buffer solution. Secondary antibodies conjugated with Alexa Fluor 488, 555, 594, and 647 were all diluted to 1:500. Following incubation, the organoids were washed 3x in PBS-T (1 h each), followed by a 35-min incubation on a nutator at 4 °C in 1 μg/ml Hoechst 33342 in PBS-T. After a final rinse in 1X PBS, organoids were serially dehydrated in 50%, 75% and 100% methanol in 1X PBS for 30 min each, then incubated at RT in 50% methanol in a BABB solution (benzyl alcohol (Sigma-Aldrich, 108006) and benzyl benzoate (Sigma-Aldrich, B6630) in a 1:2 ratio). The organoids were then fully cleared in 100% BABB and stored at 4 °C.

Immunofluorescence analysis

Immunostaining of human kidney cryosectionsFrozen human week 18.6 kidney cryosections were thawed at room temperature (RT) for ~5 min. Antigen retrieval was performed using 1X Citrate buffer (Sigma-Aldrich) on high pressure for 1 min in a pressure cooker. The slides were rinsed with deionized water and incubated for 1 h in blocking buffer solution (1.5% SEA BLOCK blocking buffer (Thermo Scientific, 37527 × 3) + 0.1% Triton X-100 (MD Millipore, 108643) in 1X PBS (Thermo Scientific, 10010049)). The slides were incubated overnight at 4 °C in primary antibodies diluted in blocking buffer solution (see ‘Immunostaining of Whole Kidney Organoids’ section below for dilutions used), rinsed in PBS-T (0.1% Triton X-100 in 1X PBS) and incubated in secondary antibodies diluted in blocking buffer solution for 1 h at room temperature; secondary antibodies conjugated with Alexa Fluor 488, 555, 594, and 647 were all diluted to 1:500. Once rinsed, the slides were incubated in 1 μg/ml Hoechst 33342 (Thermo Scientific, H3570) in PBS-T for 5 min at RT for nuclei staining, followed by a final rinse in 1X PBS. Sections were then mounted with Immu Mount (Thermo Scientific, 9990402).

Immunostaining of human kidney cryosections

Frozen human week 18.6 kidney cryosections were thawed at room temperature (RT) for ~5 min. Antigen retrieval was performed using 1X Citrate buffer (Sigma-Aldrich) on high pressure for 1 min in a pressure cooker. The slides were rinsed with deionized water and incubated for 1 h in blocking buffer solution (1.5% SEA BLOCK blocking buffer (Thermo Scientific, 37527 × 3) + 0.1% Triton X-100 (MD Millipore, 108643) in 1X PBS (Thermo Scientific, 10010049)). The slides were incubated overnight at 4 °C in primary antibodies diluted in blocking buffer solution (see ‘Immunostaining of Whole Kidney Organoids’ section below for dilutions used), rinsed in PBS-T (0.1% Triton X-100 in 1X PBS) and incubated in secondary antibodies diluted in blocking buffer solution for 1 h at room temperature; secondary antibodies conjugated with Alexa Fluor 488, 555, 594, and 647 were all diluted to 1:500. Once rinsed, the slides were incubated in 1 μg/ml Hoechst 33342 (Thermo Scientific, H3570) in PBS-T for 5 min at RT for nuclei staining, followed by a final rinse in 1X PBS. Sections were then mounted with Immu Mount (Thermo Scientific, 9990402).

Immunostaining of whole kidney organoidsWhole organoids were fixed with 4% paraformaldehyde (Electron Microscopy Sciences, 15710) in 1X PBS—1.2 ml of 4% paraformaldehyde was added under the transwell and incubated for 10 min on ice, after which an additional 1 ml of 4% paraformaldehyde was added to the top of the organoids and incubated for 10 min on ice. After incubation, the 4% paraformaldehyde was replaced with cold 1X PBS. Fixed organoids were cut out of the transwells and transferred into blocking buffer solution. The organoids were incubated on a nutator at 4 °C for 1 h, after which they were incubated overnight on a nutator at 4 °C in primary antibodies diluted in blocking buffer solution. The primary antibodies used were as follows: Laminin β1 (Santa Cruz Biotechnology, sc-33709, 1:250), HNF4A (R&D Systems, MAB4605, 1:200), POU3F3 (Thermo Scientific, PA564311, 1:100), Mafb (Novus (R&D), MAB3810, 1:500), GATA3 (Novus (R&D), AF2605, 1:200), Jag1 (Novus (R&D), AF599, 1:100), SLC12A1 (Sigma-Aldrich, HPA018107, 1:200), PAX8 (abcam, ab189249, 1:100), WT1 (abcam, ab89901, 1:500), LEF1 (Cell signaling, 2286S, 1:200), CDH1 (BD Laboratories, 610181, 1:200), PAPPA2 (R&D Systems, AF1668, 1:100), HNF1B (Thermo Scientific, MA5-24605, 1:500), Pax2 (Thermo Scientific, 716000, 1:100), TFAP2A (Santa Cruz Biotechnology, sc-12726, 1:200), IRX1 (Sigma-Aldrich, HPA043160, 1:200), SIX1 (Cell signaling, 12891S, 1:300), Pax2 (R&D Systems, AF3364, 1:50), TMEM52B (Novus (R&D), NBP2-49272, 1:100), MEIS1/2/3 (Active Motif, 39096, 1:1000), ZO-1 (Thermo Scientific, 33-9100, 1:200), PODXL (R&D Systems, AF1658, 1:300), Renin (R&D Systems, AF4090, 1:100), MECOM (R&D Systems, MAB75061, 1:100), hErbB4 (R&D Systems, MAB1131, 1:100), LEF1 (Santa Cruz Biotechnology, sc-374412, 1:200), CD31 (abcam, ab9498, 1:250), ESRRγ (Sigma-Aldrich, HPA044678, 1:100), GATA3 (Cell Signaling Technology, 5852 T, 1:100), and acetyl-alpha tubulin (Sigma-Aldrich, MABT868, 1:500). After a 3x wash in PBS-T (1 hr each), organoids were incubated overnight on a nutator at 4 °C in secondary antibodies diluted in blocking buffer solution. Secondary antibodies conjugated with Alexa Fluor 488, 555, 594, and 647 were all diluted to 1:500. Following incubation, the organoids were washed 3x in PBS-T (1 h each), followed by a 35-min incubation on a nutator at 4 °C in 1 μg/ml Hoechst 33342 in PBS-T. After a final rinse in 1X PBS, organoids were serially dehydrated in 50%, 75% and 100% methanol in 1X PBS for 30 min each, then incubated at RT in 50% methanol in a BABB solution (benzyl alcohol (Sigma-Aldrich, 108006) and benzyl benzoate (Sigma-Aldrich, B6630) in a 1:2 ratio). The organoids were then fully cleared in 100% BABB and stored at 4 °C.

Immunostaining of whole kidney organoids

Whole organoids were fixed with 4% paraformaldehyde (Electron Microscopy Sciences, 15710) in 1X PBS—1.2 ml of 4% paraformaldehyde was added under the transwell and incubated for 10 min on ice, after which an additional 1 ml of 4% paraformaldehyde was added to the top of the organoids and incubated for 10 min on ice. After incubation, the 4% paraformaldehyde was replaced with cold 1X PBS. Fixed organoids were cut out of the transwells and transferred into blocking buffer solution. The organoids were incubated on a nutator at 4 °C for 1 h, after which they were incubated overnight on a nutator at 4 °C in primary antibodies diluted in blocking buffer solution. The primary antibodies used were as follows: Laminin β1 (Santa Cruz Biotechnology, sc-33709, 1:250), HNF4A (R&D Systems, MAB4605, 1:200), POU3F3 (Thermo Scientific, PA564311, 1:100), Mafb (Novus (R&D), MAB3810, 1:500), GATA3 (Novus (R&D), AF2605, 1:200), Jag1 (Novus (R&D), AF599, 1:100), SLC12A1 (Sigma-Aldrich, HPA018107, 1:200), PAX8 (abcam, ab189249, 1:100), WT1 (abcam, ab89901, 1:500), LEF1 (Cell signaling, 2286S, 1:200), CDH1 (BD Laboratories, 610181, 1:200), PAPPA2 (R&D Systems, AF1668, 1:100), HNF1B (Thermo Scientific, MA5-24605, 1:500), Pax2 (Thermo Scientific, 716000, 1:100), TFAP2A (Santa Cruz Biotechnology, sc-12726, 1:200), IRX1 (Sigma-Aldrich, HPA043160, 1:200), SIX1 (Cell signaling, 12891S, 1:300), Pax2 (R&D Systems, AF3364, 1:50), TMEM52B (Novus (R&D), NBP2-49272, 1:100), MEIS1/2/3 (Active Motif, 39096, 1:1000), ZO-1 (Thermo Scientific, 33-9100, 1:200), PODXL (R&D Systems, AF1658, 1:300), Renin (R&D Systems, AF4090, 1:100), MECOM (R&D Systems, MAB75061, 1:100), hErbB4 (R&D Systems, MAB1131, 1:100), LEF1 (Santa Cruz Biotechnology, sc-374412, 1:200), CD31 (abcam, ab9498, 1:250), ESRRγ (Sigma-Aldrich, HPA044678, 1:100), GATA3 (Cell Signaling Technology, 5852 T, 1:100), and acetyl-alpha tubulin (Sigma-Aldrich, MABT868, 1:500). After a 3x wash in PBS-T (1 hr each), organoids were incubated overnight on a nutator at 4 °C in secondary antibodies diluted in blocking buffer solution. Secondary antibodies conjugated with Alexa Fluor 488, 555, 594, and 647 were all diluted to 1:500. Following incubation, the organoids were washed 3x in PBS-T (1 h each), followed by a 35-min incubation on a nutator at 4 °C in 1 μg/ml Hoechst 33342 in PBS-T. After a final rinse in 1X PBS, organoids were serially dehydrated in 50%, 75% and 100% methanol in 1X PBS for 30 min each, then incubated at RT in 50% methanol in a BABB solution (benzyl alcohol (Sigma-Aldrich, 108006) and benzyl benzoate (Sigma-Aldrich, B6630) in a 1:2 ratio). The organoids were then fully cleared in 100% BABB and stored at 4 °C.

Immunohistochemistry analysisParaffin embedding and sectioningWeek 16 human kidneys were fixed in 4% paraformaldehyde in 1X PBS for 1 h at 4 °C and washed 2x in 1X PBS for 5 min each. They were transferred to cassettes and stored in 70% ethanol in 4 °C and transferred to the USC Translational Research lab histology core for paraffin processing and sectioning in 5-μm sections. DD14 kidney organoids were fixed in 4% paraformaldehyde following the same protocol described in the “Immunofluorescence Analysis” section. The paraffin embedding protocol for organoids was adapted from Wörsdörfer et al.83. Briefly, fixed organoids were detached from Transwell membranes and embedded in 300 μL of 1% agarose (Avantor, 0710) in 1X PBS at 60 °C in 10 × 10 x 5 mm cryomolds (Tissue Tek, 4565) and allowed to set. The agarose blocks were transferred to tissue cassettes (Simport, M511-10) and stored in 70% ethanol at 4 °C overnight. Paraffin processing was done using the Tissue Tek VIP tissue processor (Sakura), and the agarose blocks were embedded into a paraffin block. The organoid blocks were sectioned into 5 μm-thick sections using a sliding microtome and dried overnight at 60 °C in a lab oven.Antibody staining and hematoxylin counterstainingParaffin sections were dewaxed with the following solution series: 100% Xylene (Sigma-Aldrich, 247642) for 10 min (2x), 100% ethanol for 1 min, 95% ethanol for 1 min, 80% ethanol for 1 min, 60% ethanol for 1 min, 30% ethanol for 1 min, and 2x rinse in deionized water. Antigen retrieval was performed as described in the ‘Immunofluorescence Analysis’ section, and the sections were incubated in 0.1% TBS-T (0.1% Tween-20 in 1X TBS (Cell Signaling Technology, 12498S)) + 1.5% SEA BLOCK blocking buffer for 1 h at RT. Sections were incubated in pSMAD1/5 (Cell Signaling, 9516 T, 1:100) diluted in 0.1% TBS-T + 1.5% SEA BLOCK blocking solution overnight at 4 °C. The slides were washed 2x in 0.1% TBS-T for 5 min each. Endogenous peroxidase activity was blocked with 3% hydrogen peroxide (Sigma-Aldrich, 216763) for 10 min at RT and rinsed 2x with deionized water. The slides were incubated in biotinylated secondary antibody in 0.1% TBS-T + 1.5% SEA BLOCK blocking solution for 30 min, then washed 3x with 0.1% TBS-T for 5 min each. An avidin-biotin solution was prepared from the VECTASTAIN ABC kits (Vector Laboratories, PK-6100) using manufacturer instructions to visualize biotinylated secondary antibodies, and slides were incubated in ABC solution for 30 min at RT. Slides were washed again 3x with 0.1% TBS-T for 5 min each. The slides were developed using the ImmPACT DAB substrate kit (Vector Laboratories, SK-4105) for 30 min, and development was stopped by plunging the slide into deionized water. The slides were counterstained with hematoxylin (Sigma-Aldrich, 51275) for 10 s, rinsed with deionized water, and mounted using Immu Mount.

Immunohistochemistry analysis

Paraffin embedding and sectioningWeek 16 human kidneys were fixed in 4% paraformaldehyde in 1X PBS for 1 h at 4 °C and washed 2x in 1X PBS for 5 min each. They were transferred to cassettes and stored in 70% ethanol in 4 °C and transferred to the USC Translational Research lab histology core for paraffin processing and sectioning in 5-μm sections. DD14 kidney organoids were fixed in 4% paraformaldehyde following the same protocol described in the “Immunofluorescence Analysis” section. The paraffin embedding protocol for organoids was adapted from Wörsdörfer et al.83. Briefly, fixed organoids were detached from Transwell membranes and embedded in 300 μL of 1% agarose (Avantor, 0710) in 1X PBS at 60 °C in 10 × 10 x 5 mm cryomolds (Tissue Tek, 4565) and allowed to set. The agarose blocks were transferred to tissue cassettes (Simport, M511-10) and stored in 70% ethanol at 4 °C overnight. Paraffin processing was done using the Tissue Tek VIP tissue processor (Sakura), and the agarose blocks were embedded into a paraffin block. The organoid blocks were sectioned into 5 μm-thick sections using a sliding microtome and dried overnight at 60 °C in a lab oven.

Paraffin embedding and sectioning

Week 16 human kidneys were fixed in 4% paraformaldehyde in 1X PBS for 1 h at 4 °C and washed 2x in 1X PBS for 5 min each. They were transferred to cassettes and stored in 70% ethanol in 4 °C and transferred to the USC Translational Research lab histology core for paraffin processing and sectioning in 5-μm sections. DD14 kidney organoids were fixed in 4% paraformaldehyde following the same protocol described in the “Immunofluorescence Analysis” section. The paraffin embedding protocol for organoids was adapted from Wörsdörfer et al.83. Briefly, fixed organoids were detached from Transwell membranes and embedded in 300 μL of 1% agarose (Avantor, 0710) in 1X PBS at 60 °C in 10 × 10 x 5 mm cryomolds (Tissue Tek, 4565) and allowed to set. The agarose blocks were transferred to tissue cassettes (Simport, M511-10) and stored in 70% ethanol at 4 °C overnight. Paraffin processing was done using the Tissue Tek VIP tissue processor (Sakura), and the agarose blocks were embedded into a paraffin block. The organoid blocks were sectioned into 5 μm-thick sections using a sliding microtome and dried overnight at 60 °C in a lab oven.

Antibody staining and hematoxylin counterstainingParaffin sections were dewaxed with the following solution series: 100% Xylene (Sigma-Aldrich, 247642) for 10 min (2x), 100% ethanol for 1 min, 95% ethanol for 1 min, 80% ethanol for 1 min, 60% ethanol for 1 min, 30% ethanol for 1 min, and 2x rinse in deionized water. Antigen retrieval was performed as described in the ‘Immunofluorescence Analysis’ section, and the sections were incubated in 0.1% TBS-T (0.1% Tween-20 in 1X TBS (Cell Signaling Technology, 12498S)) + 1.5% SEA BLOCK blocking buffer for 1 h at RT. Sections were incubated in pSMAD1/5 (Cell Signaling, 9516 T, 1:100) diluted in 0.1% TBS-T + 1.5% SEA BLOCK blocking solution overnight at 4 °C. The slides were washed 2x in 0.1% TBS-T for 5 min each. Endogenous peroxidase activity was blocked with 3% hydrogen peroxide (Sigma-Aldrich, 216763) for 10 min at RT and rinsed 2x with deionized water. The slides were incubated in biotinylated secondary antibody in 0.1% TBS-T + 1.5% SEA BLOCK blocking solution for 30 min, then washed 3x with 0.1% TBS-T for 5 min each. An avidin-biotin solution was prepared from the VECTASTAIN ABC kits (Vector Laboratories, PK-6100) using manufacturer instructions to visualize biotinylated secondary antibodies, and slides were incubated in ABC solution for 30 min at RT. Slides were washed again 3x with 0.1% TBS-T for 5 min each. The slides were developed using the ImmPACT DAB substrate kit (Vector Laboratories, SK-4105) for 30 min, and development was stopped by plunging the slide into deionized water. The slides were counterstained with hematoxylin (Sigma-Aldrich, 51275) for 10 s, rinsed with deionized water, and mounted using Immu Mount.

Antibody staining and hematoxylin counterstaining

Paraffin sections were dewaxed with the following solution series: 100% Xylene (Sigma-Aldrich, 247642) for 10 min (2x), 100% ethanol for 1 min, 95% ethanol for 1 min, 80% ethanol for 1 min, 60% ethanol for 1 min, 30% ethanol for 1 min, and 2x rinse in deionized water. Antigen retrieval was performed as described in the ‘Immunofluorescence Analysis’ section, and the sections were incubated in 0.1% TBS-T (0.1% Tween-20 in 1X TBS (Cell Signaling Technology, 12498S)) + 1.5% SEA BLOCK blocking buffer for 1 h at RT. Sections were incubated in pSMAD1/5 (Cell Signaling, 9516 T, 1:100) diluted in 0.1% TBS-T + 1.5% SEA BLOCK blocking solution overnight at 4 °C. The slides were washed 2x in 0.1% TBS-T for 5 min each. Endogenous peroxidase activity was blocked with 3% hydrogen peroxide (Sigma-Aldrich, 216763) for 10 min at RT and rinsed 2x with deionized water. The slides were incubated in biotinylated secondary antibody in 0.1% TBS-T + 1.5% SEA BLOCK blocking solution for 30 min, then washed 3x with 0.1% TBS-T for 5 min each. An avidin-biotin solution was prepared from the VECTASTAIN ABC kits (Vector Laboratories, PK-6100) using manufacturer instructions to visualize biotinylated secondary antibodies, and slides were incubated in ABC solution for 30 min at RT. Slides were washed again 3x with 0.1% TBS-T for 5 min each. The slides were developed using the ImmPACT DAB substrate kit (Vector Laboratories, SK-4105) for 30 min, and development was stopped by plunging the slide into deionized water. The slides were counterstained with hematoxylin (Sigma-Aldrich, 51275) for 10 s, rinsed with deionized water, and mounted using Immu Mount.

Image acquisition and analysisFluorescent immunostains were imaged on the Leica SP8 with DLSM (Leica) and Stellaris 8 (Leica) confocal microscopes. Z-stack images were acquired at 3 μm intervals. Image processing for images taken on Leica microscopes was done on LAS X image analysis software (Leica). The immunohistochemistry images were taken on the ZEISS Axioscan 7 (ZEISS) and processed on ZEISS ZEN lite image analysis software (ZEISS v3.4).

Image acquisition and analysis

Fluorescent immunostains were imaged on the Leica SP8 with DLSM (Leica) and Stellaris 8 (Leica) confocal microscopes. Z-stack images were acquired at 3 μm intervals. Image processing for images taken on Leica microscopes was done on LAS X image analysis software (Leica). The immunohistochemistry images were taken on the ZEISS Axioscan 7 (ZEISS) and processed on ZEISS ZEN lite image analysis software (ZEISS v3.4).

Spatial genomics analysis on human kidney samplesWeek 16 human kidneys embedded in OCT (Electron Microscopy Sciences, 4583) were cryosectioned at 10 μm thickness and placed on coverslips provided by Spatial Genomics. They were fixed in 4% paraformaldehyde in PBS for 15 min at RT. Sections were washed 4 times in 1x PBS for 5 min at RT and dehydrated in 100% isopropanol for 30 s at RT. The sections were air dried, stored at −80 °C, and shipped to Spatial Genomics, Inc. in dry ice for sequencing. The custom probe set used consisted of 250 cell-state and/or lineage-defining genes that represented all 5 nephron cell types54. These genes were derived from differential gene expression analysis of human single-nucleus RNA sequencing data. Data collection and imaging were performed by Spatial Genomics, and visualization was done using the Spatial Genomics Viewer software, with screengrabs obtained at 0.33x zoom.

Spatial genomics analysis on human kidney samples

Week 16 human kidneys embedded in OCT (Electron Microscopy Sciences, 4583) were cryosectioned at 10 μm thickness and placed on coverslips provided by Spatial Genomics. They were fixed in 4% paraformaldehyde in PBS for 15 min at RT. Sections were washed 4 times in 1x PBS for 5 min at RT and dehydrated in 100% isopropanol for 30 s at RT. The sections were air dried, stored at −80 °C, and shipped to Spatial Genomics, Inc. in dry ice for sequencing. The custom probe set used consisted of 250 cell-state and/or lineage-defining genes that represented all 5 nephron cell types54. These genes were derived from differential gene expression analysis of human single-nucleus RNA sequencing data. Data collection and imaging were performed by Spatial Genomics, and visualization was done using the Spatial Genomics Viewer software, with screengrabs obtained at 0.33x zoom.

Bulk mRNA sequencing and analyses of human kidney organoidsSample collection and preparationAll bulk RNA samples were collected in duplicate (2 biological replicates). Kidney organoids were collected at select timepoints (2–3 organoids pooled per condition) into a 1:100 solution of lysis buffer RLT (Qiagen, 79216) and β-Mercaptoethanol (Sigma-Aldrich, M3148) and homogenized by pipetting for 30 s. RNA was isolated from samples using the Qiagen RNeasy kit for RNA purification (Qiagen, 74104), and total RNA was quantified on a NanoDrop spectrophotometer (Thermo Scientific). mRNA from the samples was sequenced by Novogene using the Illumina NovoSeq X Plus PE153 platform (Illumina).Data processingRaw FASTQ files were aligned to the human genome using STAR alignment84 (v2.7.10a), and feature counts were generated from BAM files using the featureCounts function of the Rsubread package85. TPM normalization was performed, and genes were annotated on Partek Flow, where .txt files were generated with TPM values for each gene in each sample. For generating heatmaps of bulk mRNA sequencing data, a matrix of TPM values was loaded on R, and mitochondrial and ribosomal genes and gene rows with TPM values of <10 were excluded. The pheatmap function of the pheatmap package (v1.0.12) was used to generate heatmaps that used hierarchical clustering. The DESeqDataSetFromMatrix function from the DESeq2 package86 (v1.42.1) was used to perform DESeq analysis on paired samples, and EnhancedVolcano function (v1.20.0) generated volcano plots, where a p value cutoff of 0.05 and a log2 fold change cutoff of 1.5 was set.

Bulk mRNA sequencing and analyses of human kidney organoids

Sample collection and preparationAll bulk RNA samples were collected in duplicate (2 biological replicates). Kidney organoids were collected at select timepoints (2–3 organoids pooled per condition) into a 1:100 solution of lysis buffer RLT (Qiagen, 79216) and β-Mercaptoethanol (Sigma-Aldrich, M3148) and homogenized by pipetting for 30 s. RNA was isolated from samples using the Qiagen RNeasy kit for RNA purification (Qiagen, 74104), and total RNA was quantified on a NanoDrop spectrophotometer (Thermo Scientific). mRNA from the samples was sequenced by Novogene using the Illumina NovoSeq X Plus PE153 platform (Illumina).

Sample collection and preparation

All bulk RNA samples were collected in duplicate (2 biological replicates). Kidney organoids were collected at select timepoints (2–3 organoids pooled per condition) into a 1:100 solution of lysis buffer RLT (Qiagen, 79216) and β-Mercaptoethanol (Sigma-Aldrich, M3148) and homogenized by pipetting for 30 s. RNA was isolated from samples using the Qiagen RNeasy kit for RNA purification (Qiagen, 74104), and total RNA was quantified on a NanoDrop spectrophotometer (Thermo Scientific). mRNA from the samples was sequenced by Novogene using the Illumina NovoSeq X Plus PE153 platform (Illumina).

Data processingRaw FASTQ files were aligned to the human genome using STAR alignment84 (v2.7.10a), and feature counts were generated from BAM files using the featureCounts function of the Rsubread package85. TPM normalization was performed, and genes were annotated on Partek Flow, where .txt files were generated with TPM values for each gene in each sample. For generating heatmaps of bulk mRNA sequencing data, a matrix of TPM values was loaded on R, and mitochondrial and ribosomal genes and gene rows with TPM values of <10 were excluded. The pheatmap function of the pheatmap package (v1.0.12) was used to generate heatmaps that used hierarchical clustering. The DESeqDataSetFromMatrix function from the DESeq2 package86 (v1.42.1) was used to perform DESeq analysis on paired samples, and EnhancedVolcano function (v1.20.0) generated volcano plots, where a p value cutoff of 0.05 and a log2 fold change cutoff of 1.5 was set.

Data processing

Raw FASTQ files were aligned to the human genome using STAR alignment84 (v2.7.10a), and feature counts were generated from BAM files using the featureCounts function of the Rsubread package85. TPM normalization was performed, and genes were annotated on Partek Flow, where .txt files were generated with TPM values for each gene in each sample. For generating heatmaps of bulk mRNA sequencing data, a matrix of TPM values was loaded on R, and mitochondrial and ribosomal genes and gene rows with TPM values of <10 were excluded. The pheatmap function of the pheatmap package (v1.0.12) was used to generate heatmaps that used hierarchical clustering. The DESeqDataSetFromMatrix function from the DESeq2 package86 (v1.42.1) was used to perform DESeq analysis on paired samples, and EnhancedVolcano function (v1.20.0) generated volcano plots, where a p value cutoff of 0.05 and a log2 fold change cutoff of 1.5 was set.

Single-cell RNA sequencing and analyses of DD14 human kidney organoidsSample collection and preparationKidney organoids were collected at select time-points (9 organoids each for DD14 control and DE organoids) and dissociated into single cells using Accumax (STEMCELL Technologies, 07921) for 45 min at 37 °C, with the organoids pipetted at 5-min intervals with a P1000 tip 10 times. Once the cells were fully dissociated, the Accumax enzyme was neutralized using twice the volume of cold AutoMacs buffer (Miltenyi, 130-091-221). The tubes were centrifuged at 300 × g for 3 min at 4 °C, the supernatant was aspirated, and the dissociated cells were resuspended in 2 ml of cold AutoMacs buffer. The cells were strained using a 40 μm cell strainer and chased with 1 ml of cold AutoMacs buffer, then centrifuged again at 300 × g for 3 min at 4 °C. The supernatant was aspirated, and the cells were resuspended with 500 μl of cold AutoMacs buffer containing nuclear dyes DRAQ5 (Thermo Scientific, 62254) (1:1000) and DAPI (Thermo Scientific, D1306) (1:1000) and FAC-sorted for live cells on the ARIA II FACS (BD Biosciences). The resuspended cells were transferred to round-bottom 15-ml tubes. Using dissociated cells resuspended in AutoMacs buffer as a negative control for gating, the dissociated cells resuspended in the nuclear dyes were sorted for single-live cells (DRAQ5+, DAPI-) for collection. These cells were then fixed using the Evercode Cell Fixation kit (Parse Biosciences, ECF2001) and provided protocol. For each datapoint, 7 sublibraries were prepared for SPLiT-seq (v1.2.1) using the Evercode WT v2 kit (Parse Biosciences, ECW02030) and protocol and sequenced separately by Novogene using the Illumina NovaSeq X Plus 25B PE150 platform (Illumina) with a depth of ~63k reads/cell. Raw FASTQ files of each sublibrary were demultiplexed and aligned to the human genome using split-pipe from Parse Biosciences, where corresponding conditions were assigned to each sublibrary.Seurat object processingSeurat objects were made for each sublibrary using the Seurat package (V4) in R87, and initial quality control metrics were run to cutoff low quality cells (1000–8000 genes per cell, percent mitochondria <20). For each condition, the 7 Seurat objects (1 object per sublibrary) were merged using merge function, and no batch effects were observed. However, cycling genes were significantly contributing to cell heterogeneity as cells separated by cell cycling phases, thus we log-normalized the merged objects and regressed out cell cycle scores88 while scaling features in our datasets using the ScaleData function. Non-linear dimensionality reduction was performed using RunUMAP and FindNeighbors using 30 principal components. The objects were clustered at a resolution of 1.5. For objects from each condition, differential gene expression analysis was performed on log-normalized transcripts using FindAllMarkers function, where only genes detected in a minimum of 25% of cells in either population were tested, and the cutoff for log fold change was 0.25. Post-quality control, the Seurat objects contained 8219 control cells and 9301 DE cells. From the control organoid Seurat object, cells from clusters 21, 25, 15, 17, 6, 24, and 12 were identified as interstitial and subset out to retain nephron-only clusters. Similarly, from the DE organoid Seurat object, clusters 24, 4, 19, 18, 12, and 20 were subset out. In each instance, Nephron-only subset objects were reclustered at a resolution of 1.5.Nephron-only objects from control and DE organoids were merged using the merge function, where cell cycle regression was performed again during scaling. Differential gene expression analysis was performed on log-normalized transcripts using the same criteria described above. DESeq-derived differential gene expression between control and DE nephron cells was determined using the FindMarkers function, and the results were plotted on a volcano plot using the EnhancedVolcano function, where a p value cutoff of 0.5 and log2 fold change cutoff of 1.5 were set.

Single-cell RNA sequencing and analyses of DD14 human kidney organoids

Sample collection and preparationKidney organoids were collected at select time-points (9 organoids each for DD14 control and DE organoids) and dissociated into single cells using Accumax (STEMCELL Technologies, 07921) for 45 min at 37 °C, with the organoids pipetted at 5-min intervals with a P1000 tip 10 times. Once the cells were fully dissociated, the Accumax enzyme was neutralized using twice the volume of cold AutoMacs buffer (Miltenyi, 130-091-221). The tubes were centrifuged at 300 × g for 3 min at 4 °C, the supernatant was aspirated, and the dissociated cells were resuspended in 2 ml of cold AutoMacs buffer. The cells were strained using a 40 μm cell strainer and chased with 1 ml of cold AutoMacs buffer, then centrifuged again at 300 × g for 3 min at 4 °C. The supernatant was aspirated, and the cells were resuspended with 500 μl of cold AutoMacs buffer containing nuclear dyes DRAQ5 (Thermo Scientific, 62254) (1:1000) and DAPI (Thermo Scientific, D1306) (1:1000) and FAC-sorted for live cells on the ARIA II FACS (BD Biosciences). The resuspended cells were transferred to round-bottom 15-ml tubes. Using dissociated cells resuspended in AutoMacs buffer as a negative control for gating, the dissociated cells resuspended in the nuclear dyes were sorted for single-live cells (DRAQ5+, DAPI-) for collection. These cells were then fixed using the Evercode Cell Fixation kit (Parse Biosciences, ECF2001) and provided protocol. For each datapoint, 7 sublibraries were prepared for SPLiT-seq (v1.2.1) using the Evercode WT v2 kit (Parse Biosciences, ECW02030) and protocol and sequenced separately by Novogene using the Illumina NovaSeq X Plus 25B PE150 platform (Illumina) with a depth of ~63k reads/cell. Raw FASTQ files of each sublibrary were demultiplexed and aligned to the human genome using split-pipe from Parse Biosciences, where corresponding conditions were assigned to each sublibrary.

Sample collection and preparation

Kidney organoids were collected at select time-points (9 organoids each for DD14 control and DE organoids) and dissociated into single cells using Accumax (STEMCELL Technologies, 07921) for 45 min at 37 °C, with the organoids pipetted at 5-min intervals with a P1000 tip 10 times. Once the cells were fully dissociated, the Accumax enzyme was neutralized using twice the volume of cold AutoMacs buffer (Miltenyi, 130-091-221). The tubes were centrifuged at 300 × g for 3 min at 4 °C, the supernatant was aspirated, and the dissociated cells were resuspended in 2 ml of cold AutoMacs buffer. The cells were strained using a 40 μm cell strainer and chased with 1 ml of cold AutoMacs buffer, then centrifuged again at 300 × g for 3 min at 4 °C. The supernatant was aspirated, and the cells were resuspended with 500 μl of cold AutoMacs buffer containing nuclear dyes DRAQ5 (Thermo Scientific, 62254) (1:1000) and DAPI (Thermo Scientific, D1306) (1:1000) and FAC-sorted for live cells on the ARIA II FACS (BD Biosciences). The resuspended cells were transferred to round-bottom 15-ml tubes. Using dissociated cells resuspended in AutoMacs buffer as a negative control for gating, the dissociated cells resuspended in the nuclear dyes were sorted for single-live cells (DRAQ5+, DAPI-) for collection. These cells were then fixed using the Evercode Cell Fixation kit (Parse Biosciences, ECF2001) and provided protocol. For each datapoint, 7 sublibraries were prepared for SPLiT-seq (v1.2.1) using the Evercode WT v2 kit (Parse Biosciences, ECW02030) and protocol and sequenced separately by Novogene using the Illumina NovaSeq X Plus 25B PE150 platform (Illumina) with a depth of ~63k reads/cell. Raw FASTQ files of each sublibrary were demultiplexed and aligned to the human genome using split-pipe from Parse Biosciences, where corresponding conditions were assigned to each sublibrary.

Seurat object processingSeurat objects were made for each sublibrary using the Seurat package (V4) in R87, and initial quality control metrics were run to cutoff low quality cells (1000–8000 genes per cell, percent mitochondria <20). For each condition, the 7 Seurat objects (1 object per sublibrary) were merged using merge function, and no batch effects were observed. However, cycling genes were significantly contributing to cell heterogeneity as cells separated by cell cycling phases, thus we log-normalized the merged objects and regressed out cell cycle scores88 while scaling features in our datasets using the ScaleData function. Non-linear dimensionality reduction was performed using RunUMAP and FindNeighbors using 30 principal components. The objects were clustered at a resolution of 1.5. For objects from each condition, differential gene expression analysis was performed on log-normalized transcripts using FindAllMarkers function, where only genes detected in a minimum of 25% of cells in either population were tested, and the cutoff for log fold change was 0.25. Post-quality control, the Seurat objects contained 8219 control cells and 9301 DE cells. From the control organoid Seurat object, cells from clusters 21, 25, 15, 17, 6, 24, and 12 were identified as interstitial and subset out to retain nephron-only clusters. Similarly, from the DE organoid Seurat object, clusters 24, 4, 19, 18, 12, and 20 were subset out. In each instance, Nephron-only subset objects were reclustered at a resolution of 1.5.Nephron-only objects from control and DE organoids were merged using the merge function, where cell cycle regression was performed again during scaling. Differential gene expression analysis was performed on log-normalized transcripts using the same criteria described above. DESeq-derived differential gene expression between control and DE nephron cells was determined using the FindMarkers function, and the results were plotted on a volcano plot using the EnhancedVolcano function, where a p value cutoff of 0.5 and log2 fold change cutoff of 1.5 were set.

Seurat object processing

Seurat objects were made for each sublibrary using the Seurat package (V4) in R87, and initial quality control metrics were run to cutoff low quality cells (1000–8000 genes per cell, percent mitochondria <20). For each condition, the 7 Seurat objects (1 object per sublibrary) were merged using merge function, and no batch effects were observed. However, cycling genes were significantly contributing to cell heterogeneity as cells separated by cell cycling phases, thus we log-normalized the merged objects and regressed out cell cycle scores88 while scaling features in our datasets using the ScaleData function. Non-linear dimensionality reduction was performed using RunUMAP and FindNeighbors using 30 principal components. The objects were clustered at a resolution of 1.5. For objects from each condition, differential gene expression analysis was performed on log-normalized transcripts using FindAllMarkers function, where only genes detected in a minimum of 25% of cells in either population were tested, and the cutoff for log fold change was 0.25. Post-quality control, the Seurat objects contained 8219 control cells and 9301 DE cells. From the control organoid Seurat object, cells from clusters 21, 25, 15, 17, 6, 24, and 12 were identified as interstitial and subset out to retain nephron-only clusters. Similarly, from the DE organoid Seurat object, clusters 24, 4, 19, 18, 12, and 20 were subset out. In each instance, Nephron-only subset objects were reclustered at a resolution of 1.5.

Nephron-only objects from control and DE organoids were merged using the merge function, where cell cycle regression was performed again during scaling. Differential gene expression analysis was performed on log-normalized transcripts using the same criteria described above. DESeq-derived differential gene expression between control and DE nephron cells was determined using the FindMarkers function, and the results were plotted on a volcano plot using the EnhancedVolcano function, where a p value cutoff of 0.5 and log2 fold change cutoff of 1.5 were set.

Reanalysis of published developing human single-cell RNA sequencing dataSingle cell RNA sequencing data of week 1449 and week 171 human kidney were downloaded from the Gene Expression Omnibus (accession numbers: GSE139280, GSE127344). The Seurat package (v3) was used for quality control analyses, and SCTransform data integration89 of the 2 datasets due to batch effects. Based on differential gene expression of known kidney cell type markers, clusters were annotated with corresponding cell types, and differential gene expression analysis was performed using FindAllMarkers.Transcription factor analysisFor analysis of distal nephron-enriched transcription regulators, we examined gene expression levels of transcription factors from the human90 and mouse91 transcription factor atlases—the mouse atlas was converted into its human ortholog using Ensembl BioMart92 as described in Kim et al.5. The enrichment of transcription factor detection in TFAP2A+ clusters was shortlisted by feature plot visualization and represented in Fig S5.

Reanalysis of published developing human single-cell RNA sequencing data

Single cell RNA sequencing data of week 1449 and week 171 human kidney were downloaded from the Gene Expression Omnibus (accession numbers: GSE139280, GSE127344). The Seurat package (v3) was used for quality control analyses, and SCTransform data integration89 of the 2 datasets due to batch effects. Based on differential gene expression of known kidney cell type markers, clusters were annotated with corresponding cell types, and differential gene expression analysis was performed using FindAllMarkers.

Transcription factor analysisFor analysis of distal nephron-enriched transcription regulators, we examined gene expression levels of transcription factors from the human90 and mouse91 transcription factor atlases—the mouse atlas was converted into its human ortholog using Ensembl BioMart92 as described in Kim et al.5. The enrichment of transcription factor detection in TFAP2A+ clusters was shortlisted by feature plot visualization and represented in Fig S5.

Transcription factor analysis

For analysis of distal nephron-enriched transcription regulators, we examined gene expression levels of transcription factors from the human90 and mouse91 transcription factor atlases—the mouse atlas was converted into its human ortholog using Ensembl BioMart92 as described in Kim et al.5. The enrichment of transcription factor detection in TFAP2A+ clusters was shortlisted by feature plot visualization and represented in Fig S5.

Single-nucleus RNA sequencing and analyses of DD28 human kidney organoidsSample collection and preparationKidney organoids were collected at DD28 (6 organoids each for DD28 control and DE organoids), pooled, and immediately snap-frozen in liquid nitrogen. Nuclei isolation was conducted as previously described5, and sequencing was performed using 10X ChromiumTM GEM-X technology. Raw FASTQ files of each condition were demultiplexed and aligned to the NCBI human reference genome (GRCh38-2024-A) using Cell Ranger (v9.0.0).Seurat object processingSeurat objects were made for each condition using Seurat’s Read10X function, and initial quality control metrics were run to cut off low-quality cells (1000–8000 genes per cell, percent mitochondria <10). For each object, log-normalization was used to normalize the count data with a scale factor of 10,000. The ScaleData function was used to scale features in our datasets, and non-linear dimensionality reduction was performed using RunUMAP and FindNeighbors using 30 principal components. The objects were initially clustered at a resolution of 3 to identify clusters with outlying feature counts (low-quality cells or doublets), which were then subset out to generate the final objects. For objects from each condition, differential gene expression analysis was performed on log-normalized transcripts using FindAllMarkers function, where only genes detected in a minimum of 25% of cells in either population were tested, and the cutoff for log fold change was 0.25. Post-quality control, the Seurat objects contained 5374 control cells and 3153 DE cells. In each object, Nephron-only subset objects were reclustered at a resolution of 1.Nephron-only objects from control and DE organoids were merged using the merge function. However, clustering revealed batch effects, thus CCA integration was performed using the IntegrateLayers function. The layers were re-joined post-integration, and the normalization analysis workflow was performed with the “integrated.cca” reduction used as the input for building the shared nearest neighbors (SNN) network. Differential gene expression analysis was performed on log-normalized transcripts using the same criteria described above. DESeq-derived differential gene expression between control and DE nephron cells was determined using the FindMarkers function, and the results were plotted on a volcano plot using the EnhancedVolcano function, where a p-value cutoff of 0.5 and log2 fold change cutoff of 1.5 were set.

Single-nucleus RNA sequencing and analyses of DD28 human kidney organoids

Sample collection and preparationKidney organoids were collected at DD28 (6 organoids each for DD28 control and DE organoids), pooled, and immediately snap-frozen in liquid nitrogen. Nuclei isolation was conducted as previously described5, and sequencing was performed using 10X ChromiumTM GEM-X technology. Raw FASTQ files of each condition were demultiplexed and aligned to the NCBI human reference genome (GRCh38-2024-A) using Cell Ranger (v9.0.0).

Sample collection and preparation

Kidney organoids were collected at DD28 (6 organoids each for DD28 control and DE organoids), pooled, and immediately snap-frozen in liquid nitrogen. Nuclei isolation was conducted as previously described5, and sequencing was performed using 10X ChromiumTM GEM-X technology. Raw FASTQ files of each condition were demultiplexed and aligned to the NCBI human reference genome (GRCh38-2024-A) using Cell Ranger (v9.0.0).

Seurat object processingSeurat objects were made for each condition using Seurat’s Read10X function, and initial quality control metrics were run to cut off low-quality cells (1000–8000 genes per cell, percent mitochondria <10). For each object, log-normalization was used to normalize the count data with a scale factor of 10,000. The ScaleData function was used to scale features in our datasets, and non-linear dimensionality reduction was performed using RunUMAP and FindNeighbors using 30 principal components. The objects were initially clustered at a resolution of 3 to identify clusters with outlying feature counts (low-quality cells or doublets), which were then subset out to generate the final objects. For objects from each condition, differential gene expression analysis was performed on log-normalized transcripts using FindAllMarkers function, where only genes detected in a minimum of 25% of cells in either population were tested, and the cutoff for log fold change was 0.25. Post-quality control, the Seurat objects contained 5374 control cells and 3153 DE cells. In each object, Nephron-only subset objects were reclustered at a resolution of 1.Nephron-only objects from control and DE organoids were merged using the merge function. However, clustering revealed batch effects, thus CCA integration was performed using the IntegrateLayers function. The layers were re-joined post-integration, and the normalization analysis workflow was performed with the “integrated.cca” reduction used as the input for building the shared nearest neighbors (SNN) network. Differential gene expression analysis was performed on log-normalized transcripts using the same criteria described above. DESeq-derived differential gene expression between control and DE nephron cells was determined using the FindMarkers function, and the results were plotted on a volcano plot using the EnhancedVolcano function, where a p-value cutoff of 0.5 and log2 fold change cutoff of 1.5 were set.

Seurat object processing

Seurat objects were made for each condition using Seurat’s Read10X function, and initial quality control metrics were run to cut off low-quality cells (1000–8000 genes per cell, percent mitochondria <10). For each object, log-normalization was used to normalize the count data with a scale factor of 10,000. The ScaleData function was used to scale features in our datasets, and non-linear dimensionality reduction was performed using RunUMAP and FindNeighbors using 30 principal components. The objects were initially clustered at a resolution of 3 to identify clusters with outlying feature counts (low-quality cells or doublets), which were then subset out to generate the final objects. For objects from each condition, differential gene expression analysis was performed on log-normalized transcripts using FindAllMarkers function, where only genes detected in a minimum of 25% of cells in either population were tested, and the cutoff for log fold change was 0.25. Post-quality control, the Seurat objects contained 5374 control cells and 3153 DE cells. In each object, Nephron-only subset objects were reclustered at a resolution of 1.

Nephron-only objects from control and DE organoids were merged using the merge function. However, clustering revealed batch effects, thus CCA integration was performed using the IntegrateLayers function. The layers were re-joined post-integration, and the normalization analysis workflow was performed with the “integrated.cca” reduction used as the input for building the shared nearest neighbors (SNN) network. Differential gene expression analysis was performed on log-normalized transcripts using the same criteria described above. DESeq-derived differential gene expression between control and DE nephron cells was determined using the FindMarkers function, and the results were plotted on a volcano plot using the EnhancedVolcano function, where a p-value cutoff of 0.5 and log2 fold change cutoff of 1.5 were set.

Reanalysis of published adult kidney single-nucleus RNA sequencing dataThis study features single-nucleus RNA sequencing of 9 healthy adult human kidneys generated as part of the Kidney Precision Medicine Project [https://atlas.kpmp.org/explorer/] – this data is available for download on the Gene Expression Omnibus (accession number: GSE183279) and features datasets with the following experimental IDs: BUKMAP_20190529L_10X-R, BUKMAP_20190607L_10X-R, BUKMAP_20190822F_10X-R, BUKMAP_20190829B_10X-R, BUKMAP_20191104A-10X-R, BUKMAP_20200205A_10X-R, BUKMAP_20200205D_10X-R, BUKMAP_20200205F_10X-R, BUKMAP_20200304A_10X-R, BUKMAP_20200304B_10X-R, BUKMAP_20200304F_10X-R, KPMP_20190607I-10X-R, KPMP_20190607J-10X-R, KPMP_20190607K-10X-R.The Seurat package (v5) was used for SC Transformation normalization. The metadata contains different groups of annotated cell types and subtypes (described in Lake et al.50)—these annotations were maintained for reanalysis. The grouping of ‘class’ was used to subset kidney epithelia from stromal and endothelial cells, such that only the nephrogenic and ureteric lineages remained (44,774 cells).

Reanalysis of published adult kidney single-nucleus RNA sequencing data

This study features single-nucleus RNA sequencing of 9 healthy adult human kidneys generated as part of the Kidney Precision Medicine Project [https://atlas.kpmp.org/explorer/] – this data is available for download on the Gene Expression Omnibus (accession number: GSE183279) and features datasets with the following experimental IDs: BUKMAP_20190529L_10X-R, BUKMAP_20190607L_10X-R, BUKMAP_20190822F_10X-R, BUKMAP_20190829B_10X-R, BUKMAP_20191104A-10X-R, BUKMAP_20200205A_10X-R, BUKMAP_20200205D_10X-R, BUKMAP_20200205F_10X-R, BUKMAP_20200304A_10X-R, BUKMAP_20200304B_10X-R, BUKMAP_20200304F_10X-R, KPMP_20190607I-10X-R, KPMP_20190607J-10X-R, KPMP_20190607K-10X-R.

The Seurat package (v5) was used for SC Transformation normalization. The metadata contains different groups of annotated cell types and subtypes (described in Lake et al.50)—these annotations were maintained for reanalysis. The grouping of ‘class’ was used to subset kidney epithelia from stromal and endothelial cells, such that only the nephrogenic and ureteric lineages remained (44,774 cells).

Multimodal reference mapping of human and organoid RNA sequencing dataDD14 organoids and developing human single-cell RNA sequencing dataThe annotated clusters of the single-cell human kidney Seurat object were used to predict cell types in the DD14 control and DE nephroid datasets following the Seurat v4 reference mapping vignette developed by the Satija Lab93. Briefly, the merged organoid dataset was split by condition and treated as separate query samples for finding anchors to map them to the human kidney reference. Mapping each organoid (query) cell to the kidney organoid (reference) dataset is based on its k-nearest neighbors, for which the default, 5, was used. Seurat determines a prediction score for each assigned label to represent the confidence score (from 0 to 1) of how well organoid cells’ features (genes) align with the human kidney reference datasets. When determining transfer anchors between the reference and query, no anchors were filtered, and 50 dimensions were used from principal component analysis (PCA) reduction. The ggplot package was used for visualization of the number of cells in each condition that were represented in the predicted cell types.DD28 organoids and adult human single-nucleus RNA sequencing dataThe “subclass.l1” cell-type annotation in the adult human dataset was used to predict cell types in the DD28 control and DE nephroid datasets following the Satija lab’s Seurat v4 reference mapping vignette. The same reference mapping script and cell-type prediction scoring parameters were used as described above for the DD14 organoid and developing human dataset.

Multimodal reference mapping of human and organoid RNA sequencing data

DD14 organoids and developing human single-cell RNA sequencing dataThe annotated clusters of the single-cell human kidney Seurat object were used to predict cell types in the DD14 control and DE nephroid datasets following the Seurat v4 reference mapping vignette developed by the Satija Lab93. Briefly, the merged organoid dataset was split by condition and treated as separate query samples for finding anchors to map them to the human kidney reference. Mapping each organoid (query) cell to the kidney organoid (reference) dataset is based on its k-nearest neighbors, for which the default, 5, was used. Seurat determines a prediction score for each assigned label to represent the confidence score (from 0 to 1) of how well organoid cells’ features (genes) align with the human kidney reference datasets. When determining transfer anchors between the reference and query, no anchors were filtered, and 50 dimensions were used from principal component analysis (PCA) reduction. The ggplot package was used for visualization of the number of cells in each condition that were represented in the predicted cell types.

DD14 organoids and developing human single-cell RNA sequencing data

The annotated clusters of the single-cell human kidney Seurat object were used to predict cell types in the DD14 control and DE nephroid datasets following the Seurat v4 reference mapping vignette developed by the Satija Lab93. Briefly, the merged organoid dataset was split by condition and treated as separate query samples for finding anchors to map them to the human kidney reference. Mapping each organoid (query) cell to the kidney organoid (reference) dataset is based on its k-nearest neighbors, for which the default, 5, was used. Seurat determines a prediction score for each assigned label to represent the confidence score (from 0 to 1) of how well organoid cells’ features (genes) align with the human kidney reference datasets. When determining transfer anchors between the reference and query, no anchors were filtered, and 50 dimensions were used from principal component analysis (PCA) reduction. The ggplot package was used for visualization of the number of cells in each condition that were represented in the predicted cell types.

DD28 organoids and adult human single-nucleus RNA sequencing dataThe “subclass.l1” cell-type annotation in the adult human dataset was used to predict cell types in the DD28 control and DE nephroid datasets following the Satija lab’s Seurat v4 reference mapping vignette. The same reference mapping script and cell-type prediction scoring parameters were used as described above for the DD14 organoid and developing human dataset.

DD28 organoids and adult human single-nucleus RNA sequencing data

The “subclass.l1” cell-type annotation in the adult human dataset was used to predict cell types in the DD28 control and DE nephroid datasets following the Satija lab’s Seurat v4 reference mapping vignette. The same reference mapping script and cell-type prediction scoring parameters were used as described above for the DD14 organoid and developing human dataset.

Quantification and statistical analysisQuantification of symmetric and asymmetric nephroids was performed on ImageJ (Fiji) using the Cell Counter plugin. Statistical analysis shown on all bar plots was performed using unpaired parametric t-test (GraphPad Prism v10.0.0) for biological replicates (n = 2), where the p value cut-off is 0.05. All p values calculated are included on each graph.Quantification of fluorescence detection of MAFB, HNF4A, TFAP2A, and GATA3 was performed on ImageJ (Fiji) using the Cell Counter plugin. Single-plane organoid quadrant images were selected for this analysis, and for each image, 12 visible nephroids were selected with a DAPI background. Fluorescent cells and total cells within each nephroid were counted and aggregated to calculate percentage fluorescent cells. For each condition, images from 2 organoids (biological replicates) were quantified.Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Nils O. Lindström (nils.lindstrom@med.usc.edu).

Quantification and statistical analysis

Quantification of symmetric and asymmetric nephroids was performed on ImageJ (Fiji) using the Cell Counter plugin. Statistical analysis shown on all bar plots was performed using unpaired parametric t-test (GraphPad Prism v10.0.0) for biological replicates (n = 2), where the p value cut-off is 0.05. All p values calculated are included on each graph.

Quantification of fluorescence detection of MAFB, HNF4A, TFAP2A, and GATA3 was performed on ImageJ (Fiji) using the Cell Counter plugin. Single-plane organoid quadrant images were selected for this analysis, and for each image, 12 visible nephroids were selected with a DAPI background. Fluorescent cells and total cells within each nephroid were counted and aggregated to calculate percentage fluorescent cells. For each condition, images from 2 organoids (biological replicates) were quantified.

Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Nils O. Lindström (nils.lindstrom@med.usc.edu).

Reporting summaryFurther information on research design is available in the Nature Portfolio Reporting Summary linked to this article.

Reporting summary

Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article.

Supplementary information
Supplementary Information
Peer Review file
Description of Additional Supplementary Files
Supplementary Data 1
Supplementary Data 2
Reporting Summary

Supplementary information

Supplementary Information
Peer Review file
Description of Additional Supplementary Files
Supplementary Data 1
Supplementary Data 2
Reporting Summary

Supplementary Information

Peer Review file

Description of Additional Supplementary Files

Supplementary Data 1

Supplementary Data 2

Reporting Summary
