---
pmid: '25173106'
title: Genome-wide analysis of multi-ancestry cohorts identifies new loci influencing
  intraocular pressure and susceptibility to glaucoma.
authors:
- Hysi PG
- Cheng CY
- Springelkamp H
- Macgregor S
- Bailey JNC
- Wojciechowski R
- Vitart V
- Nag A
- Hewitt AW
- Höhn R
- Venturini C
- Mirshahi A
- Ramdas WD
- Thorleifsson G
- Vithana E
- Khor CC
- Stefansson AB
- Liao J
- Haines JL
- Amin N
- Wang YX
- Wild PS
- Ozel AB
- Li JZ
- Fleck BW
- Zeller T
- Staffieri SE
- Teo YY
- Cuellar-Partida G
- Luo X
- Allingham RR
- Richards JE
- Senft A
- Karssen LC
- Zheng Y
- Bellenguez C
- Xu L
- Iglesias AI
- Wilson JF
- Kang JH
- van Leeuwen EM
- Jonsson V
- Thorsteinsdottir U
- Despriet DDG
- Ennis S
- Moroi SE
- Martin NG
- Jansonius NM
- Yazar S
- Tai ES
- Amouyel P
- Kirwan J
- van Koolwijk LME
- Hauser MA
- Jonasson F
- Leo P
- Loomis SJ
- Fogarty R
- Rivadeneira F
- Kearns L
- Lackner KJ
- de Jong PTVM
- Simpson CL
- Pennell CE
- Oostra BA
- Uitterlinden AG
- Saw SM
- Lotery AJ
- Bailey-Wilson JE
- Hofman A
- Vingerling JR
- Maubaret C
- Pfeiffer N
- Wolfs RCW
- Lemij HG
- Young TL
- Pasquale LR
- Delcourt C
- Spector TD
- Klaver CCW
- Small KS
- Burdon KP
- Stefansson K
- Wong TY
- BMES GWAS Group
- NEIGHBORHOOD Consortium
- Wellcome Trust Case Control Consortium 2
- Viswanathan A
- Mackey DA
- Craig JE
- Wiggs JL
- van Duijn CM
- Hammond CJ
- Aung T
journal: Nat Genet
year: '2014'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC4177225
doi: 10.1038/ng.3087
---

# Genome-wide analysis of multi-ancestry cohorts identifies new loci influencing intraocular pressure and susceptibility to glaucoma.
**Authors:** Hysi PG, Cheng CY, Springelkamp H, Macgregor S, Bailey JNC, Wojciechowski R, Vitart V, Nag A, Hewitt AW, Höhn R, Venturini C, Mirshahi A, Ramdas WD, Thorleifsson G, Vithana E, Khor CC, Stefansson AB, Liao J, Haines JL, Amin N, Wang YX, Wild PS, Ozel AB, Li JZ, Fleck BW, Zeller T, Staffieri SE, Teo YY, Cuellar-Partida G, Luo X, Allingham RR, Richards JE, Senft A, Karssen LC, Zheng Y, Bellenguez C, Xu L, Iglesias AI, Wilson JF, Kang JH, van Leeuwen EM, Jonsson V, Thorsteinsdottir U, Despriet DDG, Ennis S, Moroi SE, Martin NG, Jansonius NM, Yazar S, Tai ES, Amouyel P, Kirwan J, van Koolwijk LME, Hauser MA, Jonasson F, Leo P, Loomis SJ, Fogarty R, Rivadeneira F, Kearns L, Lackner KJ, de Jong PTVM, Simpson CL, Pennell CE, Oostra BA, Uitterlinden AG, Saw SM, Lotery AJ, Bailey-Wilson JE, Hofman A, Vingerling JR, Maubaret C, Pfeiffer N, Wolfs RCW, Lemij HG, Young TL, Pasquale LR, Delcourt C, Spector TD, Klaver CCW, Small KS, Burdon KP, Stefansson K, Wong TY, BMES GWAS Group, NEIGHBORHOOD Consortium, Wellcome Trust Case Control Consortium 2, Viswanathan A, Mackey DA, Craig JE, Wiggs JL, van Duijn CM, Hammond CJ, Aung T
**Journal:** Nat Genet (2014)
**DOI:** [10.1038/ng.3087](https://doi.org/10.1038/ng.3087)
**PMC:** [PMC4177225](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4177225/)

## Abstract

1. Nat Genet. 2014 Oct;46(10):1126-1130. doi: 10.1038/ng.3087. Epub 2014 Aug 31.

Genome-wide analysis of multi-ancestry cohorts identifies new loci influencing 
intraocular pressure and susceptibility to glaucoma.

Hysi PG(#)(1), Cheng CY(#)(2)(3)(4)(5), Springelkamp H(#)(6)(7), Macgregor 
S(#)(8), Bailey JNC(#)(9), Wojciechowski R(#)(10)(11), Vitart V(12), Nag A(1), 
Hewitt AW(13), Höhn R(14), Venturini C(1)(15), Mirshahi A(14), Ramdas WD(6)(7), 
Thorleifsson G(16), Vithana E(2)(3)(5), Khor CC(3)(17), Stefansson AB(18), Liao 
J(2)(3), Haines JL(9), Amin N(7), Wang YX(19), Wild PS(20), Ozel AB(21), Li 
JZ(21), Fleck BW(22), Zeller T(23), Staffieri SE(13), Teo YY(4)(24), 
Cuellar-Partida G(8), Luo X(25), Allingham RR(26), Richards JE(6), Senft A(27), 
Karssen LC(7), Zheng Y(2)(4), Bellenguez C(28)(29)(30), Xu L(19), Iglesias 
AI(7), Wilson JF(31), Kang JH(32), van Leeuwen EM(7), Jonsson V(33), 
Thorsteinsdottir U(16), Despriet DDG(6), Ennis S(34), Moroi SE(35), Martin 
NG(36), Jansonius NM(37), Yazar S(38), Tai ES(4)(5)(39), Amouyel 
P(27)(28)(29)(40), Kirwan J(41), van Koolwijk LME(7), Hauser MA(26)(42), 
Jonasson F(43), Leo P(44), Loomis SJ(45), Fogarty R(46), Rivadeneira 
F(7)(47)(48), Kearns L(13), Lackner KJ(49), de Jong PTVM(50)(51)(52), Simpson 
CL(11), Pennell CE(53), Oostra BA(54), Uitterlinden AG(7)(47)(48), Saw 
SM(2)(3)(4)(5), Lotery AJ(55), Bailey-Wilson JE(11), Hofman A(7)(48), Vingerling 
JR(6)(7), Maubaret C(56)(57), Pfeiffer N(14), Wolfs RCW(6), Lemij HG(58), Young 
TL(25), Pasquale LR(32)(45), Delcourt C(56)(57), Spector TD(1), Klaver 
CCW(6)(7), Small KS(1), Burdon KP(46), Stefansson K(16), Wong TY(2)(3)(4); BMES 
GWAS Group; NEIGHBORHOOD Consortium; Wellcome Trust Case Control Consortium 2; 
Viswanathan A(59), Mackey DA(13)(38), Craig JE(46), Wiggs JL(45), van Duijn 
CM(7), Hammond CJ(1), Aung T(2)(3).

Author information:
(1)Department of Twin Research and Genetic Epidemiology, King's College London, 
UK.
(2)Singapore Eye Research Institute, Singapore National Eye Centre, Singapore.
(3)Department of Ophthalmology, National University of Singapore and National 
University Health System, Singapore.
(4)Saw Swee Hock School of Public Health, National University of Singapore and 
National University Health System, Singapore.
(5)Duke-National University of Singapore, Graduate Medical School, Singapore.
(6)Department of Ophthalmology, Erasmus Medical Center, Rotterdam, the 
Netherlands.
(7)Department of Epidemiology, Erasmus Medical Center, Rotterdam, the 
Netherlands.
(8)Statistical Genetics, QIMR Berghofer Medical Research Institute Royal 
Brisbane Hospital, Brisbane, Australia 4029.
(9)Department of Epidemiology and Biostatistics, Case Western Reserve 
University, Cleveland, OH, USA.
(10)Department of Epidemiology, Johns Hopkins Bloomberg School of Public Health 
and, Baltimore, MD, USA.
(11)National Human Genome Research Institute, National Institutes of Health, 
Baltimore, MD 21224, USA.
(12)MRC Human Genetics Unit, Institute of Genetics and Molecular Medicine, 
University of Edinburgh, Edinburgh.
(13)Centre for Eye Research Australia, University of Melbourne, Royal Victorian 
Eye and Ear Hospital, Australia.
(14)Department of Ophthalmology, University Medical Center Mainz, Mainz, 
Germany.
(15)Institute of Ophthalmology, University College London, UK.
(16)deCODE/Amgen, 101 Reykjavik, Iceland.
(17)Division of Human Genetics, Genome Institute of Singapore, Singapore.
(18)Eye Clinic, 105 Reykjavik, Iceland.
(19)Beijing Institute of Ophthalmology, Beijing Tongren Hospital, Capital 
University of Medical Science, Beijing, China 100730.
(20)Department of Internal Medicine II, University Medical Center Mainz, Mainz, 
Germany.
(21)Department of Human Genetics, University of Michigan, Ann Arbor, MI, USA.
(22)NHS Princess Alexandra Eye Pavilion, Edinburgh, UK.
(23)Clinic for General and Interventional Cardiology, University Heart Center 
Hamburg, Germany.
(24)Department of Statistics and Applied Probability, National University of 
Singapore, Singapore.
(25)Duke University, Duke Eye Center, Durham, NC, USA.
(26)Department of Ophthalmology, Duke University Medical Center, Durham, NC, 
USA.
(27)Institute of Medical Biometry and Statistics, University Hospital 
Schleswig-Holstein, Lübeck, Germany.
(28)Inserm, UMR774, Lille, France.
(29)Université Lille 2, Lille, France.
(30)Institut Pasteur de Lille, France.
(31)Centre for Population Health Sciences, University of Edinburgh Medical 
School, Edinburgh, UK.
(32)Channing Division of Network Medicine, Harvard Medical School, Boston, MA, 
USA.
(33)Department of Ophthalmology, Landspitali National University Hospital, 
Reykjavik, Iceland.
(34)Human Genetics & Genomic Medicine, Faulty of Medicine, University of 
Southampton, UK.
(35)Department of Ophthalmology and Visual Sciences, University of Michigan, Ann 
Arbor, MI, USA.
(36)Genetic Epidemiology, QIMR Berghofer Medical Research Institute Royal 
Brisbane Hospital, Brisbane, Australia.
(37)Department of Ophthalmology, University Medical Center Groningen, the 
Netherlands.
(38)Centre for Ophthalmology and Visual Science, Lions Eye Institute, University 
of Western Australia, Perth, Australia.
(39)Department of Medicine, National University of Singapore and National 
University Health System, Singapore.
(40)Centre Hospitalier Régional Universitaire de Lille, France.
(41)Department of Ophthalmology, Portsmouth Hospitals NHS Trust, Portsmouth, UK.
(42)Department of Medicine, Duke University Medical Center, Durham, NC, USA.
(43)Faculty of Medicine, University of Iceland, Reykjavik, Iceland.
(44)The University of Queensland, Diamantina Institute, Woollongabba, Australia.
(45)Dept. Ophthalmology Harvard Medical School and Massachusetts Eye and Ear 
Infirmary, Boston, MA, USA.
(46)Department of Ophthalmology, Flinders University, Adelaide, Australia.
(47)Department of Internal Medicine, Erasmus Medical Center, Rotterdam, the 
Netherlands.
(48)Netherlands Consortium for Healthy Ageing, Netherlands Genomics Initiative, 
The Hague, the Netherlands.
(49)Institute of Clinical Chemistry and Laboratory Medicine, Mainz, Germany.
(50)Netherlands Institute for Neuroscience, Amsterdam, the Netherlands.
(51)Department of Ophthalmology, Leiden University Medical Center, Leiden, the 
Netherlands.
(52)Department of Ophthalmology, Academic Medical Center, Amsterdam, the 
Netherlands.
(53)School of Women's and Infants' Health, The University of Western Australia, 
Perth, Australia.
(54)Department of Clinical Genetics, Erasmus Medical Center, Rotterdam, the 
Netherlands.
(55)Clinical and Experimental Sciences, Faculty of Medicine, University of 
Southampton, UK.
(56)Univ. Bordeaux, ISPED, Bordeaux, France.
(57)Inserm, Centre INSERM U897-Epidemiologie-Biostatistique Bordeaux, France.
(58)Glaucoma Service, the Rotterdam Eye Hospital, Rotterdam, the Netherlands.
(59)NIHR Biomedical Research Centre, Moorfields Eye Hospital NHS Foundation 
Trust and UCL Institute of Ophthalmology, London, UK.
(#)Contributed equally

Elevated intraocular pressure (IOP) is an important risk factor in developing 
glaucoma, and variability in IOP might herald glaucomatous development or 
progression. We report the results of a genome-wide association study 
meta-analysis of 18 population cohorts from the International Glaucoma Genetics 
Consortium (IGGC), comprising 35,296 multi-ancestry participants for IOP. We 
confirm genetic association of known loci for IOP and primary open-angle 
glaucoma (POAG) and identify four new IOP-associated loci located on chromosome 
3q25.31 within the FNDC3B gene (P = 4.19 × 10(-8) for rs6445055), two on 
chromosome 9 (P = 2.80 × 10(-11) for rs2472493 near ABCA1 and P = 6.39 × 10(-11) 
for rs8176693 within ABO) and one on chromosome 11p11.2 (best P = 1.04 × 10(-11) 
for rs747782). Separate meta-analyses of 4 independent POAG cohorts, totaling 
4,284 cases and 95,560 controls, showed that 3 of these loci for IOP were also 
associated with POAG.

DOI: 10.1038/ng.3087
PMCID: PMC4177225
PMID: 25173106 [Indexed for MEDLINE]

## Full Text

Primary open angle glaucoma (POAG) is the leading cause of irreversible blindness in the world1. The only modifiable risk factor for the development and progression of glaucoma is high intraocular pressure (IOP)2, and lowering IOP is currently the only therapy that can reduce glaucomatous progression, even in forms of glaucoma that have IOP close to the statistical norm for the population (normal tension glaucoma or NTG)34. POAG and IOP are highly heritable; the lifetime risk of developing POAG is 22% among first degree relatives of patients5, which is approximately 10 times higher than the rest of the population1. The IOP heritability is estimated to be approximately 55%6. Genetic studies have shown that the genetic risk of POAG and IOP are partly shared; polymorphisms within the TMCO1 gene are associated with both POAG risk 7 and IOP8. Studying genetic determinants of IOP is therefore likely to provide critical insights into the genetic architecture of POAG and open new avenues for therapeutic intervention.

In this study we present results from a meta-analysis of genome-wide association studies (GWAS) of IOP from 18 studies participating in the International Glaucoma Genetics Consortium (IGGC) and an assessment of the importance of the genetic findings for susceptibility to POAG (Figure 1). The IOP meta-analysis included 35,296 subjects (7,738 Asians and 27,558 of European descent) drawn from the general populations of seven countries. Demographic characteristics of these population cohorts are given in the Supplementary Table 1. Genotyping assays and imputation to HapMap2 haplotypes were performed at individual sites. Association analyses were performed using an additive model, with IOP as outcome, number of alleles at each polymorphic site as predictors, adjusting for age and sex. IOP levels for participants who were receiving IOP-lowering therapy at the time of the study and whose baseline, pre-treatment levels were not available were imputed as previously described8. Subjects who had undergone surgery or had other eye diseases that could affect IOP were removed (Supplementary Note). Secondary analyses were carried out adjusting for central corneal thickness (CCT), which is known to influence IOP measurements 9.

After applying conventional quality control filters, a fixed effects meta-analysis of 22 autosomes across the cohorts was performed with approximately 2.5 million markers. Within-study genomic inflation factors10 ranged between 0.992 and 1.043 (Supplementary Table 2 and Supplementary Figure 1), indicating a lack of major population stratification bias within each study. SNPs available in fewer than 16 cohorts or showing large heterogeneity (defined as I2>75% 11) were removed. We found 145 SNPs (Supplementary Table 3) whose association crossed the conventional genome-wide significance threshold (p<5×10−08) 12. All of these SNPs clustered around seven separate regions of the genome (Figure 2, Supplementary Figures 2 and 3). Two of the regions associated with IOP in our meta-analysis had previously been implicated in IOP variability: the region near the TMCO1 locus7,8 (p=2.19×10−09 for rs7555523), and near GAS7 gene8 (p=1.03×10−11 for rs9913911). A third locus, novel for IOP, was near the CAV1 and CAV2 genes (p=1.87×10−11 for rs10258482) which had previously been associated with POAG13.

Novel associations were identified within large linkage disequilibrium (LD) block on chromosome 11 encompassing, among other genes, AGBL2, SPI1 and PTPRJ (best p=1.04×10−11 for rs747782), (Supplementary Figure 1). Two additional loci were mapped on chromosome 9: one at 9q31.1 upstream the ABCA1 gene (p=2.80×10−11 for rs2472493) and the other at 9q34.2 within the ABO blood group gene (p=3.08×10−11 for rs8176743). A fourth region was detected on chr3q25.31, within the FNDC3B gene (p=4.19×10−08 for rs6445055).

Interestingly, of all the loci previously associated with glaucoma or related quantitative traits14, CDKN2BAS and SIX1/SIX6 are not associated with IOP in the meta-analysis. It is possible these two loci exert their influence on POAG through mechanisms unrelated to intraocular pressure.

Genome-wide significant SNPs from the IOP meta-analysis were then investigated for their effect on the clinical outcome of POAG in four independent cohorts representing a combined 4,284 POAG cases (normal and high tension glaucoma) and 95,560 controls (details about these cohorts in the Supplementary Note). Associations to POAG were found for the novel regions near ABCA1 gene (p=4.15×10−09 for rs2472493), near the FNDC3B gene (p=0.03 for rs6445055) and on the chromosome 11 cluster (p=0.008 for rs12419342). We did not find significant statistical evidence of association of POAG to the ABO locus. The case-control analyses reinforced association evidence at the previously identified loci on TMCO1 (p=1.34×10−16 for rs7555523), CAV1/CAV2 (p=6.27×10−09 for rs10258482) and GAS7 (p=5.22×10−13 for rs12150284). All alleles associated with increased IOP levels also raised glaucoma risk (Table 1).

We then examined whether the effect sizes of SNPs on IOP levels (BetaIOP) were linearly related to their effect sizes on POAG (BetaPOAG) using a causal inference framework as previously described15. In a linear regression analysis, we observed a significant association between BetaIOP and BetaPOAG (p=0.03, Supplementary Table 4), suggesting the strength of SNPs’ effect on IOP levels is correlated with their effects on the risk of POAG.

We subsequently investigated the relationship between variants within the seven regions associated with IOP and cis- regulation of mRNA expression in three tissues (adipose, lymphoblastoid cell lines [LCLs] and skin) from a sample of 856 British subjects16. The most significant eQTL associations were generally observed in LCLs for most loci, except for CAV1 where effects were strongest in adipose and skin tissues (Table 2). Significant eQTL association was observed for rs4656461 and rs7555523 (p=0.003 and p=0.0001 with TMCO1 and ALDH9A1 transcript expressions in skin and LCLs respectively), rs2024211 (p=5.43×10−16 and 3.84×10−13 with CAV1 in adipose and skin tissues respectively), rs2472493 with ABCA1 (p=3.67×10−05 in LCLs) and rs1681630 with the SPI1 expression on chromosome 11 (p=2.72×10−10 in LCLs) among others (Table 2, Supplementary Table 5A). These SNPs also had the strongest eQTL effects for their respective transcripts (Supplementary Table 5B).

We measured the mRNA expression levels of the identified genes in adult ocular tissues using reverse-transcriptase PCR. We found that most of the identified genes, including TMCO1, FDNC3B, CAV1/CAV2, ABCA1 and GAS7 were expressed in most ocular tissues (Supplementary Table 6). The expression of genes within the chromosome 11 locus showed varied expression levels across ocular tissues.

Gene-based tests or Gene Ontology enrichment analyses failed to identify any new genes or pathways after multiple testing correction (Supplementary Tables 7 and 8).

Altogether, these SNPs explain approximately 1.2% of the IOP heritability in the TwinsUK cohort17, 1.5% of the IOP phenotypic variability in the Rotterdam study18 and between 0.6 and 1.2% in Asians. FNDC3B has been associated with central corneal thickness (CCT) 19 and as CCT has a significant effect on IOP measurements20, we performed an additional meta-analysis of IOP adjusted for age, sex and CCT in a smaller subsample which had CCT measures (19,563 subjects from 13 population cohorts). The association for rs6445055 remained nominally significant, albeit reduced (p= 9.87×10−04, β= −0.121 compared to −0.177 prior to adjustment for CCT). This suggests that this locus has at least some CCT-independent effect over the IOP levels. The association evidence remained consistent, although slightly reduced, for the other loci (Supplementary Table 9).

We report association of variants within the ABCA1 gene and IOP and POAG. A strong eQTL effect was observed in LCLs (p=3.67×10−05) for the most highly associated SNP (rs2472493) in our analyses. The ABCA1 gene is expressed in many tissues21 and its expression in leukocytes is significantly up-regulated in glaucoma patients22.

Associations for a number of SNPs within the ABO blood group gene and IOP, although statistically significant and homogeneous across the participating cohorts, were not observed in the glaucoma case-control meta-analysis. This might be due to type I error in the initial meta-analysis or insufficient power to detect a primarily IOP-led effect in cases that included NTG patients, resulting in a type II error in the latter. Four of the nine GWAS polymorphisms associated at genome-wide significance in the ABO locus are non-synonymous variants, determining B blood group23. This may be relevant given previous observations that the B blood group is epidemiologically associated with glaucoma, including POAG24, although the mechanisms remain unclear.

Association was found between IOP and variants lying over a large region on chromosome 11. Of the many genes in that region, eQTL analyses singled out SPI1 and AGBL2 as possible candidates for prioritization in future studies. eQTL analyses also raised the possibility of ALDH9A1 as a candidate for IOP regulation, given its strong expression in ciliary body25 and location just downstream of the TMCO1-associated variant. The eQTL results also suggest that CAV1 is a stronger candidate than CAV2, although transcription regulation may not be the only mechanism influencing IOP at this locus.

Although IOP and POAG are strongly genetically correlated26, we explored further their shared genetic backgrounds. Using independent (i.e. not in LD) SNPs with p< 10−06 in the IOP GWAS meta-analysis as described elsewhere27, we found a statistically significant polygenic overlap between IOP and POAG in the ANZRAG cohort (p=4.33×10−05). The variance explained in POAG was 0.7%, which changed little if less significant SNPs were progressively included in the model (Supplementary Table 10).

There are potential limitations to this study. First, there is variability across the studies in terms of IOP measurement methods, although the differences are likely to be small28. In addition, we maximized power to discover genetic variants of small effect size by including multiethnic cohorts, at the risk of introducing heterogeneity into the study. Heterogeneity was however generally low (Table 1) for most of the loci reported, so we consider our results to be conservative. Second, assessment of clinical importance using panels of POAG cases is not equivalent to a formal replication. Even in this case, we expect our results to be over-conservative at the price of reduced sensitivity, which could be a possible reason for non-validation of our association with IOP in the ABO blood group locus. Finally, we based our eQTL analysis on sample tissue availability rather than ideal ocular tissue types. Tissues such as trabecular meshwork would have been preferable, but they are impractical because they are generally less accessible. We tried to circumvent this limitation by studying three different tissues, but caution is required when interpreting eQTL results.

Despite these considerations, our report of seven loci associated with IOP and glaucoma, of which four are novel, is an important step forward to better understanding the mechanisms of IOP regulation-currently the only modifiable risk factor for POAG.

Online MethodsIGGC participantsAll studies participating in this meta-analysis are part of the International Glaucoma Genetics Consortium. The discovery cohorts included 27,558 individuals of European ancestry from 14 studies (ALIENOR, BATS, BMES,29,30 ERF,31,32 Framingham Family Study,33 GHS1, GHS2, ORCADES,34 RAINE, 35-37 RS-I, RS-II, RS-III,38 TEST39 and TwinsUK40). In addition, 7,738 individuals of Asian ancestry from four cohorts (BES,41 SCES,42 SiMES,43 SINDI,42) were included. In addition four case-control population panels were used, all of European ancestry: ANZRAG,7 MEEI, NEIGHBOR and deCode. General methods, demographics and phenotyping of the study cohorts have previously been described extensively and are provided in the Supplementary Note and Supplementary Table 1. All studies were performed with the approval of their local Medical Ethics Committees and written informed consent was obtained from all participants in accordance with the Declaration of Helsinki.Phenotype measurementsEligible participants underwent an ophthalmologic examination including measurements of IOP and, for most but not all studies, measurements of central corneal thickness (CCT). Each participating cohort was phenotyped separately and IOP measurement methods used by each are described in the Supplementary Table 1.Genotyping & imputationThe study samples were genotyped on either the Illumina (San Diego, CA, USA) or Affymetrix (Santa Clara, CA, USA) platforms. Each study performed single nucleotide polymorphism (SNP) imputation using the genotype data, together with the HapMap Phase II ethnically matched reference panels (CEU, JPT+CHB, or the 4 HapMap populations) on the basis of build 36 databases (release 22 or 24). The Markov Chain Haplotyping software, IMPUTE44,45 or MACH,46 were adopted for imputation. A detailed description regarding genotyping platforms and imputation procedures for each study is provided on Supplementary Table 1.Stringent quality control of genotype data was applied in each cohort. Samples with low call rates (<95%) or with gender discrepancies were excluded. Cryptically related samples and outliers in population structure from principal component analyses were also excluded. SNPs flagged with missingness >5%, gross departure from Hardy Weinberg Equilibrium (P value <10−6) and minor allele frequency (MAF) <1% were removed from further analyses.Statistical AnalysisFor each study, an allele-dosage regression model at each directly genotyped or imputed SNP was conducted to determine its association with IOP. Eyes with prior glaucoma surgery or laser were excluded. For subject receiving IOP-lowering medication, we added 25% to the measured IOP levels to estimate pre-treatment IOP, based on a reported average of 17% to 33% IOP reduction caused by IOP lowering medication in a meta-analysis of clinical trails.47 The mean of the right and left IOP measurements was used. When data from only one eye were available, the IOP measurement from the available eye was used.For the analyses, we assumed an additive genetic model where the dosage of each SNP is a continuous variable ranging from 0 to 2 for minor alleles carried. Primary analysis for IOP was adjusted for age and sex. Additional adjustment for principal components was carried out by a few participating cohorts to correct for subtle population substructure.The per-SNP meta-analyses were performed using the GWAMA software with weighted inverse-variance approach, assuming fixed effects, as for initial discovery purposes the fixed-effects model is preferred for increased statistical power.48 A Cochran’s Q test and I2 were used to assess heterogeneity across studies49 . For each participating cohort, only SNPs with sufficient imputation quality scores (proper-info of IMPUTE or R2 of MACH >0.3) were included into the meta-analysis.Gene-based testing was conducted using VEGAS software50 on the European ancestry and Asian ancestry meta-analysis results separately. VEGAS incorporates information from the full set of markers within a gene and accounts for LD between markers by using simulations from the multivariate normal distribution. For samples of European descent, we used the HapMap 2 CEU population as the reference to estimate patterns of LD. For Asian ancestry groups, we used the combined HapMap 2 JPT and CHB populations as the reference population to approximate LD patterns. To include gene regulatory regions, SNPs were included if they fell within 50 kb of a gene. We performed meta-analysis on the two sets of gene-based P-values using Fisher’s method.VEGAS-Pathway analysis50,51 was carried out using prespecified pathways from Gene Ontology. Pathways of with 10 to 1,000 components were selected, yielding 4,628 pathways. Pathway analysis was based on combining gene-based test results from VEGAS. Pathway P-values were computed by summing χ2 test statistics derived from VEGAS P-values. Empirical VEGAS-Pathway P values for each pathway were computed by comparing the summed χ2 test statistics from real data with those generated in 500,000 simulations where the relevant number (according to the size of the pathway) of randomly drawn χ2 test statistics was summed. To ensure that clusters of genes did not adversely affect results, within each pathway, gene sets were pruned such that each gene was >500 kb away from all other genes in the pathway. Where required, all but one of the clustered genes was dropped at random when genes were clustered. We performed meta-analysis on the two sets of pathway p-values using Fisher’s method.To investigate shared genetic background by using a large number of autosomal SNPs we made a systematic evaluation of the overlap between the IOP and POAG based on profile scores following approaches previously described27. We estimated the relative risk for each SNP of interest based on a discovery set (IOP), with a profile score computed for every individual in a target set of interest (POAG). For each target set individual the profile score was computed as the number of risk alleles weighted by the effect size estimated in the discovery set. The discovery set comprised the European ancestry derived samples from our meta-analysis and the target set was was a set of 590 glaucoma cases and 3956 controls, as previously described7. To ensure that there was not a high degree of dependence between the SNPs included in the profile score, we filtered the set of SNPs used in the profile score so that only a set of 149,571 SNPs in low linkage disequilibrium (r2<0.5) were used. We constructed models including more SNPs progressively lowering the threshold of inclusion (i.e. p<0.000001, p<0.00001, p<0.0001, p<0.001, p<0.01, p<0.1, p<0.5). Profiles derived from IOP SNP effects were tested for association with the phenotype (POAG here) using a logistic regression. Variance explained was assessed using Nagelkerke’s pseudoR2 measure52.To assess whether and to what degree IOP levels confer POAG risk, we performed a causal inference analysis using an instrumental variable framework as previously described15. In brief, we obtained estimates of effect size (BetaIOP) for the association of a given SNP with IOP from the meta-analysis of the 18 discovery cohorts. For the association of a given SNP with POAG, we obtained estimates of the effect size (BetaPOAG) from the four case-control panels as described above. We selected the the strongest associated SNP from each of the genome-wide significant IOP loci that we identified. To assess whether the strength of SNPs’ association with IOP predicts the risk of POAG, we conducted linear regression analysis using the effect sizes of each SNP for IOP (BetaIOP) as independent variables and the effect sizes of POAG (BetaPOAG) as dependent outcome variables . A total of eight independent IOP-associated SNPs were used for this analysis, including rs7555523 (TMCO1), rs6445055 (FNDC3B), rs10258482 (CAV1), rs2472493 (ABCA1), rs8176743 (ABO), rs747782 (NUP160-PTPRJ), and rs9913911 (GAS7).Gene expression in human tissuesAdult ocular samples were obtained from normal eyes of an 82-year-old European ancestry female from the North Carolina Eye Bank, Winston-Salem, North Carolina, USA. All adult ocular samples were stored in Qiagen’s RNAlater within 6.5 hours of collection and shipped on dry ice overnight to the lab. Isolated tissues were snap-frozen and stored at −280 °C until RNA extraction. RNA was extracted from each tissue sample independently using the Ambion mirVana total RNA extraction kit. The tissue samples were homogenized in Ambion lysis buffer using an Omni Bead Ruptor Tissue Homogenizer per protocol. Reverse transcription reactions were performed with Invitrogen SuperScript III First-Strand Synthesis kit. The expression of the identified genes were assessed by running 10 ul reactions with Qiagen’s PCR products consisting of 1.26 ul H2O, 1.0 ul 10X buffer, 1.0 ul dNTPs, 0.3 ul MgCl, 2.0 ul Q-Solution, 0.06 ul taq polymerase, 1.0 ul forward primer, 1.0 ul reverse primer and 1.5.0 ul cDNA. The reactions were run on a Eppendorf Mastercycler Pro S thermocycler with touchdown PCR ramping down 1°C per cycle from 72 °C to 55 °C followed by 50 cycles of 94 °C for 30 seconds, 55 °C for 30 seconds and 72 °C for 30 seconds with a final elongation of 7 minutes at 72 °C. All primer sets were designed using Primer3.53 Products were run on a 2% agarose gel at 70 volts for 35 minutes. Primer sets were run on a custom tissue panel including Clontech’s Human MTC Panel I, Fetal MTC Panel I and an ocular tissue panel.

Online Methods

IGGC participantsAll studies participating in this meta-analysis are part of the International Glaucoma Genetics Consortium. The discovery cohorts included 27,558 individuals of European ancestry from 14 studies (ALIENOR, BATS, BMES,29,30 ERF,31,32 Framingham Family Study,33 GHS1, GHS2, ORCADES,34 RAINE, 35-37 RS-I, RS-II, RS-III,38 TEST39 and TwinsUK40). In addition, 7,738 individuals of Asian ancestry from four cohorts (BES,41 SCES,42 SiMES,43 SINDI,42) were included. In addition four case-control population panels were used, all of European ancestry: ANZRAG,7 MEEI, NEIGHBOR and deCode. General methods, demographics and phenotyping of the study cohorts have previously been described extensively and are provided in the Supplementary Note and Supplementary Table 1. All studies were performed with the approval of their local Medical Ethics Committees and written informed consent was obtained from all participants in accordance with the Declaration of Helsinki.

IGGC participants

All studies participating in this meta-analysis are part of the International Glaucoma Genetics Consortium. The discovery cohorts included 27,558 individuals of European ancestry from 14 studies (ALIENOR, BATS, BMES,29,30 ERF,31,32 Framingham Family Study,33 GHS1, GHS2, ORCADES,34 RAINE, 35-37 RS-I, RS-II, RS-III,38 TEST39 and TwinsUK40). In addition, 7,738 individuals of Asian ancestry from four cohorts (BES,41 SCES,42 SiMES,43 SINDI,42) were included. In addition four case-control population panels were used, all of European ancestry: ANZRAG,7 MEEI, NEIGHBOR and deCode. General methods, demographics and phenotyping of the study cohorts have previously been described extensively and are provided in the Supplementary Note and Supplementary Table 1. All studies were performed with the approval of their local Medical Ethics Committees and written informed consent was obtained from all participants in accordance with the Declaration of Helsinki.

Phenotype measurementsEligible participants underwent an ophthalmologic examination including measurements of IOP and, for most but not all studies, measurements of central corneal thickness (CCT). Each participating cohort was phenotyped separately and IOP measurement methods used by each are described in the Supplementary Table 1.

Phenotype measurements

Eligible participants underwent an ophthalmologic examination including measurements of IOP and, for most but not all studies, measurements of central corneal thickness (CCT). Each participating cohort was phenotyped separately and IOP measurement methods used by each are described in the Supplementary Table 1.

Genotyping & imputationThe study samples were genotyped on either the Illumina (San Diego, CA, USA) or Affymetrix (Santa Clara, CA, USA) platforms. Each study performed single nucleotide polymorphism (SNP) imputation using the genotype data, together with the HapMap Phase II ethnically matched reference panels (CEU, JPT+CHB, or the 4 HapMap populations) on the basis of build 36 databases (release 22 or 24). The Markov Chain Haplotyping software, IMPUTE44,45 or MACH,46 were adopted for imputation. A detailed description regarding genotyping platforms and imputation procedures for each study is provided on Supplementary Table 1.Stringent quality control of genotype data was applied in each cohort. Samples with low call rates (<95%) or with gender discrepancies were excluded. Cryptically related samples and outliers in population structure from principal component analyses were also excluded. SNPs flagged with missingness >5%, gross departure from Hardy Weinberg Equilibrium (P value <10−6) and minor allele frequency (MAF) <1% were removed from further analyses.

Genotyping & imputation

The study samples were genotyped on either the Illumina (San Diego, CA, USA) or Affymetrix (Santa Clara, CA, USA) platforms. Each study performed single nucleotide polymorphism (SNP) imputation using the genotype data, together with the HapMap Phase II ethnically matched reference panels (CEU, JPT+CHB, or the 4 HapMap populations) on the basis of build 36 databases (release 22 or 24). The Markov Chain Haplotyping software, IMPUTE44,45 or MACH,46 were adopted for imputation. A detailed description regarding genotyping platforms and imputation procedures for each study is provided on Supplementary Table 1.

Stringent quality control of genotype data was applied in each cohort. Samples with low call rates (<95%) or with gender discrepancies were excluded. Cryptically related samples and outliers in population structure from principal component analyses were also excluded. SNPs flagged with missingness >5%, gross departure from Hardy Weinberg Equilibrium (P value <10−6) and minor allele frequency (MAF) <1% were removed from further analyses.

Statistical AnalysisFor each study, an allele-dosage regression model at each directly genotyped or imputed SNP was conducted to determine its association with IOP. Eyes with prior glaucoma surgery or laser were excluded. For subject receiving IOP-lowering medication, we added 25% to the measured IOP levels to estimate pre-treatment IOP, based on a reported average of 17% to 33% IOP reduction caused by IOP lowering medication in a meta-analysis of clinical trails.47 The mean of the right and left IOP measurements was used. When data from only one eye were available, the IOP measurement from the available eye was used.For the analyses, we assumed an additive genetic model where the dosage of each SNP is a continuous variable ranging from 0 to 2 for minor alleles carried. Primary analysis for IOP was adjusted for age and sex. Additional adjustment for principal components was carried out by a few participating cohorts to correct for subtle population substructure.The per-SNP meta-analyses were performed using the GWAMA software with weighted inverse-variance approach, assuming fixed effects, as for initial discovery purposes the fixed-effects model is preferred for increased statistical power.48 A Cochran’s Q test and I2 were used to assess heterogeneity across studies49 . For each participating cohort, only SNPs with sufficient imputation quality scores (proper-info of IMPUTE or R2 of MACH >0.3) were included into the meta-analysis.Gene-based testing was conducted using VEGAS software50 on the European ancestry and Asian ancestry meta-analysis results separately. VEGAS incorporates information from the full set of markers within a gene and accounts for LD between markers by using simulations from the multivariate normal distribution. For samples of European descent, we used the HapMap 2 CEU population as the reference to estimate patterns of LD. For Asian ancestry groups, we used the combined HapMap 2 JPT and CHB populations as the reference population to approximate LD patterns. To include gene regulatory regions, SNPs were included if they fell within 50 kb of a gene. We performed meta-analysis on the two sets of gene-based P-values using Fisher’s method.VEGAS-Pathway analysis50,51 was carried out using prespecified pathways from Gene Ontology. Pathways of with 10 to 1,000 components were selected, yielding 4,628 pathways. Pathway analysis was based on combining gene-based test results from VEGAS. Pathway P-values were computed by summing χ2 test statistics derived from VEGAS P-values. Empirical VEGAS-Pathway P values for each pathway were computed by comparing the summed χ2 test statistics from real data with those generated in 500,000 simulations where the relevant number (according to the size of the pathway) of randomly drawn χ2 test statistics was summed. To ensure that clusters of genes did not adversely affect results, within each pathway, gene sets were pruned such that each gene was >500 kb away from all other genes in the pathway. Where required, all but one of the clustered genes was dropped at random when genes were clustered. We performed meta-analysis on the two sets of pathway p-values using Fisher’s method.To investigate shared genetic background by using a large number of autosomal SNPs we made a systematic evaluation of the overlap between the IOP and POAG based on profile scores following approaches previously described27. We estimated the relative risk for each SNP of interest based on a discovery set (IOP), with a profile score computed for every individual in a target set of interest (POAG). For each target set individual the profile score was computed as the number of risk alleles weighted by the effect size estimated in the discovery set. The discovery set comprised the European ancestry derived samples from our meta-analysis and the target set was was a set of 590 glaucoma cases and 3956 controls, as previously described7. To ensure that there was not a high degree of dependence between the SNPs included in the profile score, we filtered the set of SNPs used in the profile score so that only a set of 149,571 SNPs in low linkage disequilibrium (r2<0.5) were used. We constructed models including more SNPs progressively lowering the threshold of inclusion (i.e. p<0.000001, p<0.00001, p<0.0001, p<0.001, p<0.01, p<0.1, p<0.5). Profiles derived from IOP SNP effects were tested for association with the phenotype (POAG here) using a logistic regression. Variance explained was assessed using Nagelkerke’s pseudoR2 measure52.To assess whether and to what degree IOP levels confer POAG risk, we performed a causal inference analysis using an instrumental variable framework as previously described15. In brief, we obtained estimates of effect size (BetaIOP) for the association of a given SNP with IOP from the meta-analysis of the 18 discovery cohorts. For the association of a given SNP with POAG, we obtained estimates of the effect size (BetaPOAG) from the four case-control panels as described above. We selected the the strongest associated SNP from each of the genome-wide significant IOP loci that we identified. To assess whether the strength of SNPs’ association with IOP predicts the risk of POAG, we conducted linear regression analysis using the effect sizes of each SNP for IOP (BetaIOP) as independent variables and the effect sizes of POAG (BetaPOAG) as dependent outcome variables . A total of eight independent IOP-associated SNPs were used for this analysis, including rs7555523 (TMCO1), rs6445055 (FNDC3B), rs10258482 (CAV1), rs2472493 (ABCA1), rs8176743 (ABO), rs747782 (NUP160-PTPRJ), and rs9913911 (GAS7).

Statistical Analysis

For each study, an allele-dosage regression model at each directly genotyped or imputed SNP was conducted to determine its association with IOP. Eyes with prior glaucoma surgery or laser were excluded. For subject receiving IOP-lowering medication, we added 25% to the measured IOP levels to estimate pre-treatment IOP, based on a reported average of 17% to 33% IOP reduction caused by IOP lowering medication in a meta-analysis of clinical trails.47 The mean of the right and left IOP measurements was used. When data from only one eye were available, the IOP measurement from the available eye was used.

For the analyses, we assumed an additive genetic model where the dosage of each SNP is a continuous variable ranging from 0 to 2 for minor alleles carried. Primary analysis for IOP was adjusted for age and sex. Additional adjustment for principal components was carried out by a few participating cohorts to correct for subtle population substructure.

The per-SNP meta-analyses were performed using the GWAMA software with weighted inverse-variance approach, assuming fixed effects, as for initial discovery purposes the fixed-effects model is preferred for increased statistical power.48 A Cochran’s Q test and I2 were used to assess heterogeneity across studies49 . For each participating cohort, only SNPs with sufficient imputation quality scores (proper-info of IMPUTE or R2 of MACH >0.3) were included into the meta-analysis.

Gene-based testing was conducted using VEGAS software50 on the European ancestry and Asian ancestry meta-analysis results separately. VEGAS incorporates information from the full set of markers within a gene and accounts for LD between markers by using simulations from the multivariate normal distribution. For samples of European descent, we used the HapMap 2 CEU population as the reference to estimate patterns of LD. For Asian ancestry groups, we used the combined HapMap 2 JPT and CHB populations as the reference population to approximate LD patterns. To include gene regulatory regions, SNPs were included if they fell within 50 kb of a gene. We performed meta-analysis on the two sets of gene-based P-values using Fisher’s method.

VEGAS-Pathway analysis50,51 was carried out using prespecified pathways from Gene Ontology. Pathways of with 10 to 1,000 components were selected, yielding 4,628 pathways. Pathway analysis was based on combining gene-based test results from VEGAS. Pathway P-values were computed by summing χ2 test statistics derived from VEGAS P-values. Empirical VEGAS-Pathway P values for each pathway were computed by comparing the summed χ2 test statistics from real data with those generated in 500,000 simulations where the relevant number (according to the size of the pathway) of randomly drawn χ2 test statistics was summed. To ensure that clusters of genes did not adversely affect results, within each pathway, gene sets were pruned such that each gene was >500 kb away from all other genes in the pathway. Where required, all but one of the clustered genes was dropped at random when genes were clustered. We performed meta-analysis on the two sets of pathway p-values using Fisher’s method.

To investigate shared genetic background by using a large number of autosomal SNPs we made a systematic evaluation of the overlap between the IOP and POAG based on profile scores following approaches previously described27. We estimated the relative risk for each SNP of interest based on a discovery set (IOP), with a profile score computed for every individual in a target set of interest (POAG). For each target set individual the profile score was computed as the number of risk alleles weighted by the effect size estimated in the discovery set. The discovery set comprised the European ancestry derived samples from our meta-analysis and the target set was was a set of 590 glaucoma cases and 3956 controls, as previously described7. To ensure that there was not a high degree of dependence between the SNPs included in the profile score, we filtered the set of SNPs used in the profile score so that only a set of 149,571 SNPs in low linkage disequilibrium (r2<0.5) were used. We constructed models including more SNPs progressively lowering the threshold of inclusion (i.e. p<0.000001, p<0.00001, p<0.0001, p<0.001, p<0.01, p<0.1, p<0.5). Profiles derived from IOP SNP effects were tested for association with the phenotype (POAG here) using a logistic regression. Variance explained was assessed using Nagelkerke’s pseudoR2 measure52.

To assess whether and to what degree IOP levels confer POAG risk, we performed a causal inference analysis using an instrumental variable framework as previously described15. In brief, we obtained estimates of effect size (BetaIOP) for the association of a given SNP with IOP from the meta-analysis of the 18 discovery cohorts. For the association of a given SNP with POAG, we obtained estimates of the effect size (BetaPOAG) from the four case-control panels as described above. We selected the the strongest associated SNP from each of the genome-wide significant IOP loci that we identified. To assess whether the strength of SNPs’ association with IOP predicts the risk of POAG, we conducted linear regression analysis using the effect sizes of each SNP for IOP (BetaIOP) as independent variables and the effect sizes of POAG (BetaPOAG) as dependent outcome variables . A total of eight independent IOP-associated SNPs were used for this analysis, including rs7555523 (TMCO1), rs6445055 (FNDC3B), rs10258482 (CAV1), rs2472493 (ABCA1), rs8176743 (ABO), rs747782 (NUP160-PTPRJ), and rs9913911 (GAS7).

Gene expression in human tissuesAdult ocular samples were obtained from normal eyes of an 82-year-old European ancestry female from the North Carolina Eye Bank, Winston-Salem, North Carolina, USA. All adult ocular samples were stored in Qiagen’s RNAlater within 6.5 hours of collection and shipped on dry ice overnight to the lab. Isolated tissues were snap-frozen and stored at −280 °C until RNA extraction. RNA was extracted from each tissue sample independently using the Ambion mirVana total RNA extraction kit. The tissue samples were homogenized in Ambion lysis buffer using an Omni Bead Ruptor Tissue Homogenizer per protocol. Reverse transcription reactions were performed with Invitrogen SuperScript III First-Strand Synthesis kit. The expression of the identified genes were assessed by running 10 ul reactions with Qiagen’s PCR products consisting of 1.26 ul H2O, 1.0 ul 10X buffer, 1.0 ul dNTPs, 0.3 ul MgCl, 2.0 ul Q-Solution, 0.06 ul taq polymerase, 1.0 ul forward primer, 1.0 ul reverse primer and 1.5.0 ul cDNA. The reactions were run on a Eppendorf Mastercycler Pro S thermocycler with touchdown PCR ramping down 1°C per cycle from 72 °C to 55 °C followed by 50 cycles of 94 °C for 30 seconds, 55 °C for 30 seconds and 72 °C for 30 seconds with a final elongation of 7 minutes at 72 °C. All primer sets were designed using Primer3.53 Products were run on a 2% agarose gel at 70 volts for 35 minutes. Primer sets were run on a custom tissue panel including Clontech’s Human MTC Panel I, Fetal MTC Panel I and an ocular tissue panel.

Gene expression in human tissues

Adult ocular samples were obtained from normal eyes of an 82-year-old European ancestry female from the North Carolina Eye Bank, Winston-Salem, North Carolina, USA. All adult ocular samples were stored in Qiagen’s RNAlater within 6.5 hours of collection and shipped on dry ice overnight to the lab. Isolated tissues were snap-frozen and stored at −280 °C until RNA extraction. RNA was extracted from each tissue sample independently using the Ambion mirVana total RNA extraction kit. The tissue samples were homogenized in Ambion lysis buffer using an Omni Bead Ruptor Tissue Homogenizer per protocol. Reverse transcription reactions were performed with Invitrogen SuperScript III First-Strand Synthesis kit. The expression of the identified genes were assessed by running 10 ul reactions with Qiagen’s PCR products consisting of 1.26 ul H2O, 1.0 ul 10X buffer, 1.0 ul dNTPs, 0.3 ul MgCl, 2.0 ul Q-Solution, 0.06 ul taq polymerase, 1.0 ul forward primer, 1.0 ul reverse primer and 1.5.0 ul cDNA. The reactions were run on a Eppendorf Mastercycler Pro S thermocycler with touchdown PCR ramping down 1°C per cycle from 72 °C to 55 °C followed by 50 cycles of 94 °C for 30 seconds, 55 °C for 30 seconds and 72 °C for 30 seconds with a final elongation of 7 minutes at 72 °C. All primer sets were designed using Primer3.53 Products were run on a 2% agarose gel at 70 volts for 35 minutes. Primer sets were run on a custom tissue panel including Clontech’s Human MTC Panel I, Fetal MTC Panel I and an ocular tissue panel.

Supplementary MaterialSupplementary Material

Supplementary Material
