---
pmid: '33657378'
title: Functional genetics of human gut commensal Bacteroides thetaiotaomicron reveals
  metabolic requirements for growth across environments.
authors:
- Liu H
- Shiver AL
- Price MN
- Carlson HK
- Trotter VV
- Chen Y
- Escalante V
- Ray J
- Hern KE
- Petzold CJ
- Turnbaugh PJ
- Huang KC
- Arkin AP
- Deutschbauer AM
journal: Cell Rep
year: '2021'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC8121099
doi: 10.1016/j.celrep.2021.108789
---

# Functional genetics of human gut commensal Bacteroides thetaiotaomicron reveals metabolic requirements for growth across environments.
**Authors:** Liu H, Shiver AL, Price MN, Carlson HK, Trotter VV, Chen Y, Escalante V, Ray J, Hern KE, Petzold CJ, Turnbaugh PJ, Huang KC, Arkin AP, Deutschbauer AM
**Journal:** Cell Rep (2021)
**DOI:** [10.1016/j.celrep.2021.108789](https://doi.org/10.1016/j.celrep.2021.108789)
**PMC:** [PMC8121099](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8121099/)

## Abstract

1. Cell Rep. 2021 Mar 2;34(9):108789. doi: 10.1016/j.celrep.2021.108789.

Functional genetics of human gut commensal Bacteroides thetaiotaomicron reveals 
metabolic requirements for growth across environments.

Liu H(1), Shiver AL(2), Price MN(1), Carlson HK(1), Trotter VV(1), Chen Y(3), 
Escalante V(4), Ray J(1), Hern KE(5), Petzold CJ(3), Turnbaugh PJ(6), Huang 
KC(7), Arkin AP(8), Deutschbauer AM(9).

Author information:
(1)Environmental Genomics and Systems Biology Division, Lawrence Berkeley 
National Laboratory, Berkeley, CA 94720, USA.
(2)Department of Bioengineering, Stanford University, Stanford, CA 94305, USA.
(3)Biological Systems and Engineering Division, Lawrence Berkeley National 
Laboratory, Berkeley, CA 94720, USA.
(4)Department of Microbiology and Immunology, University of California, San 
Francisco, San Francisco, CA 94143, USA.
(5)Department of Plant and Microbial Biology, University of California, 
Berkeley, Berkeley, CA 94720, USA.
(6)Department of Microbiology and Immunology, University of California, San 
Francisco, San Francisco, CA 94143, USA; Chan Zuckerberg Biohub, San Francisco, 
CA 94158, USA.
(7)Department of Bioengineering, Stanford University, Stanford, CA 94305, USA; 
Chan Zuckerberg Biohub, San Francisco, CA 94158, USA; Department of Microbiology 
and Immunology, Stanford University School of Medicine, Stanford, CA 94305, USA.
(8)Environmental Genomics and Systems Biology Division, Lawrence Berkeley 
National Laboratory, Berkeley, CA 94720, USA; Department of Bioengineering, 
University of California, Berkeley, Berkeley, CA 94720, USA.
(9)Environmental Genomics and Systems Biology Division, Lawrence Berkeley 
National Laboratory, Berkeley, CA 94720, USA; Department of Plant and Microbial 
Biology, University of California, Berkeley, Berkeley, CA 94720, USA. Electronic 
address: amdeutschbauer@lbl.gov.

Harnessing the microbiota for beneficial outcomes is limited by our poor 
understanding of the constituent bacteria, as the functions of most of their 
genes are unknown. Here, we measure the growth of a barcoded transposon mutant 
library of the gut commensal Bacteroides thetaiotaomicron on 48 carbon sources, 
in the presence of 56 stress-inducing compounds, and during mono-colonization of 
gnotobiotic mice. We identify 516 genes with a specific phenotype under only one 
or a few conditions, enabling informed predictions of gene function. For 
example, we identify a glycoside hydrolase important for growth on type I 
rhamnogalacturonan, a DUF4861 protein for glycosaminoglycan utilization, a 
3-keto-glucoside hydrolase for disaccharide utilization, and a tripartite 
multidrug resistance system specifically for bile salt tolerance. Furthermore, 
we show that B. thetaiotaomicron uses alternative enzymes for synthesizing 
nitrogen-containing metabolic precursors based on ammonium availability and that 
these enzymes are used differentially in vivo in a diet-dependent manner.

Copyright © 2021 The Author(s). Published by Elsevier Inc. All rights reserved.

DOI: 10.1016/j.celrep.2021.108789
PMCID: PMC8121099
PMID: 33657378 [Indexed for MEDLINE]

Conflict of interest statement: Declaration of interests The authors declare no 
competing interests.

## Full Text

INTRODUCTIONThe human gut microbiota plays important roles in health and disease (Cho and Blaser, 2012; Zitvogel et al., 2017). It is anticipated that the human gut microbiota can be manipulated to treat a number of diseases (Guinane and Cotter, 2013). However, these efforts will be limited without a greater understanding of the functional potential of the bacteria that comprise the human microbiota. A primary bottleneck is that approximately half of protein-coding genes do not yet have a known molecular function (Heintz-Buschart and Wilmes, 2018).One of the most well-studied and prevalent members of the human gut microbiota is the anaerobe Bacteroides thetaiotaomicron VPI-5482 (Wexler and Goodman, 2017). Pioneering work revealed the broad range of complex polysaccharides and simple carbohydrates that B. thetaiotaomicron can degrade and ferment in the gut (Kotarski and Salyers, 1984; Salyers and O’Brien, 1980; Salyers et al., 1977). Genome analysis found that many carbon degradation systems are encoded within polysaccharide utilization loci (PULs) (Xu et al., 2003), which typically contain a cluster of genes involved in the regulation, transport, and catabolism of polysaccharides (Bjursell et al., 2006). The most well-studied PUL in B. thetaiotaomicron is the starch utilization system (Sus), encoded by susRABCDEFG (Martens et al., 2009). SusR is the sensor/regulator, SusCDEF are membrane proteins responsible for starch binding and transport, and SusABG are hydrolytic enzymes. With the minimal requirement of a susCD gene pair, computational analyses have identified >80 predicted PULs in the B. thetaiotaomicron genome (Martens et al., 2008) but the majority are experimentally uncharacterized.Although traditional single-gene genetics and enzymology are useful for elucidating protein function, such approaches are often time-consuming and laborious. Thus, large-scale functional-genetic methods are attractive for the characterization of genes of unknown function (Gray et al., 2015). Methods based on transposon-site sequencing (TnSeq) are useful for assaying the importance of many genes in parallel under multiple conditions (van Opijnen and Camilli, 2013). For example, transposon insertion sequencing (INSeq) applied to B. thetaiotaomicron revealed genes required for colonizing mice (Goodman et al., 2009). Subsequent application of INSeq to multiple Bacteroides species identified species-specific genes important in vivo (Wu et al., 2015). The development of large, multi-condition gene-phenotype datasets has proven useful for inferring gene function in a broad range of organisms (Deutschbauer et al., 2011; Hillenmeyer et al., 2010; Nichols et al., 2011), although this approach has not been applied to gut commensals.Using a barcoded variant of TnSeq (RB-TnSeq) (Wetmore et al., 2015), we performed hundreds of in vitro genome-wide fitness assays in B. thetaiotaomicron, including in the presence of simple sugars, complex polysaccharides, and antibiotics. We further profiled fitness during mono-colonization of germ-free mice. This rich dataset provides experimental evidence for many predicted gene functions in B. thetaiotaomicron and links genes that previously lacked informative annotations to specific processes including polysaccharide degradation, disaccharide catabolism, and bile salt tolerance. Our data also revealed that alternative enzymes are used to synthesize nitrogen-containing precursor metabolites in response to diet-dependent availability of ammonium in the gut.

INTRODUCTION

The human gut microbiota plays important roles in health and disease (Cho and Blaser, 2012; Zitvogel et al., 2017). It is anticipated that the human gut microbiota can be manipulated to treat a number of diseases (Guinane and Cotter, 2013). However, these efforts will be limited without a greater understanding of the functional potential of the bacteria that comprise the human microbiota. A primary bottleneck is that approximately half of protein-coding genes do not yet have a known molecular function (Heintz-Buschart and Wilmes, 2018).

One of the most well-studied and prevalent members of the human gut microbiota is the anaerobe Bacteroides thetaiotaomicron VPI-5482 (Wexler and Goodman, 2017). Pioneering work revealed the broad range of complex polysaccharides and simple carbohydrates that B. thetaiotaomicron can degrade and ferment in the gut (Kotarski and Salyers, 1984; Salyers and O’Brien, 1980; Salyers et al., 1977). Genome analysis found that many carbon degradation systems are encoded within polysaccharide utilization loci (PULs) (Xu et al., 2003), which typically contain a cluster of genes involved in the regulation, transport, and catabolism of polysaccharides (Bjursell et al., 2006). The most well-studied PUL in B. thetaiotaomicron is the starch utilization system (Sus), encoded by susRABCDEFG (Martens et al., 2009). SusR is the sensor/regulator, SusCDEF are membrane proteins responsible for starch binding and transport, and SusABG are hydrolytic enzymes. With the minimal requirement of a susCD gene pair, computational analyses have identified >80 predicted PULs in the B. thetaiotaomicron genome (Martens et al., 2008) but the majority are experimentally uncharacterized.

Although traditional single-gene genetics and enzymology are useful for elucidating protein function, such approaches are often time-consuming and laborious. Thus, large-scale functional-genetic methods are attractive for the characterization of genes of unknown function (Gray et al., 2015). Methods based on transposon-site sequencing (TnSeq) are useful for assaying the importance of many genes in parallel under multiple conditions (van Opijnen and Camilli, 2013). For example, transposon insertion sequencing (INSeq) applied to B. thetaiotaomicron revealed genes required for colonizing mice (Goodman et al., 2009). Subsequent application of INSeq to multiple Bacteroides species identified species-specific genes important in vivo (Wu et al., 2015). The development of large, multi-condition gene-phenotype datasets has proven useful for inferring gene function in a broad range of organisms (Deutschbauer et al., 2011; Hillenmeyer et al., 2010; Nichols et al., 2011), although this approach has not been applied to gut commensals.

Using a barcoded variant of TnSeq (RB-TnSeq) (Wetmore et al., 2015), we performed hundreds of in vitro genome-wide fitness assays in B. thetaiotaomicron, including in the presence of simple sugars, complex polysaccharides, and antibiotics. We further profiled fitness during mono-colonization of germ-free mice. This rich dataset provides experimental evidence for many predicted gene functions in B. thetaiotaomicron and links genes that previously lacked informative annotations to specific processes including polysaccharide degradation, disaccharide catabolism, and bile salt tolerance. Our data also revealed that alternative enzymes are used to synthesize nitrogen-containing precursor metabolites in response to diet-dependent availability of ammonium in the gut.

RESULTSA gene-phenotype map of B. thetaiotaomicronTo enable high-throughput genetics in B. thetaiotaomicron VPI-5482, we generated a genome-wide RB-TnSeq library in this bacterium. The final library contains 315,668 uniquely barcoded mutants with insertions that we could confidently map to a single location in the genome (Figure 1A). To enable follow-up studies, we generated a 40-plate archived collection of individual transposon-insertion mutants in a 96-well microplate format by sorting cells using flow cytometry (Shiver et al., 2019).To validate the utility of the mutant library, we grew it in a defined medium with either glucose or xylose as the sole carbon source. For each experiment, we measured the abundance of the DNA barcodes by sequencing (BarSeq) before and after selection, and we defined the fitness of a gene as the average log2 change in relative abundance of its mutants (Figure 1B). Negative and positive values mean that the mutants in that gene were less and more fit, respectively, than the average strain. Many genes involved in amino acid biosynthesis were important for fitness in either carbon source because the medium lacked most amino acids. Genes from the xylose utilization pathway (BT0792, xylulose kinase; BT0793, xylose isomerase; BT0794, xylose transporter) were important for growth on xylose only (Figure 1B). This example illustrates the biological consistency of fitness assays using our mutant library.To create a gene-phenotype compendium, we performed 523 genome-wide fitness assays that passed metrics for internal consistency (Wetmore et al., 2015; STAR methods), and all analyses in this study are derived from these experiments. Many experiments were in defined Varel-Bryant (VB) media with a single carbon source. In addition, we performed hundreds of experiments with an inhibitory concentration of a stressor such as an antibiotic, biocide, or bile salt. Finally, we measured the in vivo fitness of each mutant during mono-colonization of germ-free mice. We successfully quantified fitness for 4,055 protein-coding genes in each experiment. Among the genes with no fitness data, we estimate that 378 are likely essential for viability in the brain heart infusion supplemented (BHIS) medium used to generate the library (Table S1). Of these genes, 202 (53%) were also identified as candidate essential genes in a previous study of transposon insertion mutants grown in a rich medium similar to BHIS medium (Goodman et al., 2009).We identified 516 genes with a specific phenotype (Table S2), which we defined as a gene-condition combination with ∣fitness∣ > 1 that was statistically significant and for which the gene lacked a significant fitness change in most other experiments (STAR methods). A specific phenotype often links a gene to a condition that is directly related to its function (Price et al., 2018). For example, BT3788 is a SusC homolog located in the PUL for mannan utilization (Cuskin et al., 2015); across all 48 carbon sources profiled, BT3788 was important only for fitness on mannan (Figure 1C). BT4507 encodes a class A serine β-lactamase (Stentz et al., 2015); mutants in this gene were impaired on two β-lactam antibiotics (ceftriaxone and cephalothin) but not under any of the other conditions profiled, including other antibiotics (Figure 1D).A second method for inferring gene function is co-fitness (Price et al., 2018), a measure of the degree to which pairs of genes share similar fitness profiles across a large genetic dataset (Table S3). To illustrate this approach, we used co-fitness to identify the enzyme responsible for the first step of arginine biosynthesis in B. thetaiotaomicron. Bacteroides species use a non-canonical pathway for synthesizing arginine, in which intermediates are succinylated instead of acetylated (Cabrera-Luque, 2010; Shi et al., 2006). A majority of the enzymes known to be involved in arginine synthesis in Escherichia coli (Caldara et al., 2008) have clear homology to enzymes in B. thetaiotaomicron; a notable exception is the first step, in which N-acetylglutamate synthase appears to have been replaced by an evolutionarily unrelated N-succinylglutamate synthase. BT3761 has been predicted to fill this role, in part because it is in a 5-gene operon of arginine biosynthetic genes. In addition, BT3761 belongs to the GNAT family of N-acyltransferases and is 59% identical (protein sequence) to an unusual N-acetyl-glutamate synthase from Caldithrix abyssi (Cabrera-Luque, 2010; Kublanov et al., 2017). The four most co-fit genes with BT3761 are involved in arginine biosynthesis (BT3759, homologous to argC in E. coli; BT3733, argH; BT3760, argG; and BT3395, argB). Furthermore, BT3761 was critical for growth on minimal media without amino acids, and its fitness defect was rescued through addition of arginine (Figure 1E). Additionally, quantitative real-time reverse-transcriptase PCR (RT-PCR) revealed that BT3761 RNA levels in wild-type cells were repressed 6.4-fold upon addition of 3 mM arginine (95% confidence interval: 5.3- to 7.8-fold repression). These results support the hypothesis that BT3761 participates in arginine biosynthesis as an N-succinylglutamate synthase.Overall, we identified a functional association (a specific phenotype, or co-fitness of ≥0.8) for 633 of the 4,055 protein-coding genes for which we collected data. Genes with a functional association were more likely to have a specific Clusters of Orthologous Groups (COG) functional category (Tatusov et al., 2003) assigned than genes with no functional association (55% versus 39%; p = 10−15, Fisher’s exact test). Nonetheless, we were able to identify a functional association for 270 genes without a specific COG function category. According to the PaperBLAST database (Price and Arkin, 2017), 174 of these genes have not been previously discussed in the literature, nor do they have close homologs (>70% amino acid identity) to other Bacteroides species.Functional-genetic screen links PULs to specific substratesFitness profiling across multiple substrates provided a broad view of the importance of each PUL component to carbon catabolism. The 48 carbon sources we assayed represent a range of substrate complexity, including 14 monosaccharides and their derivatives, 12 disaccharides, 1 trisaccharide, 2 tetrasaccharides, 3 oligosaccharides (>4 monomers), 2 sugar alcohols, and 14 complex polysaccharides. As an illustration, we re-examined the Sus system by comparing its phenotypes during growth on starch; the component polysaccharides of starch (amylopectin and amylose); the oligosaccharide α-cyclodextrin; and the simpler break-down products glucose, maltose, maltotetraose, and maltohexaose (Figure 1F). The SusCD transporter proteins were important on all saccharides of a length of ≥4 but were largely dispensable for growth on maltose and glucose, which is in agreement with previous findings that SusD importance varies depending on the size of glucose polymers (Koropatkin et al., 2008).The three glycoside hydrolases (GHs; SusA, SusB, and SusG) were variably important across substrates. SusA was previously characterized as a neopullulanase that cleaves α-1,4 linkages in pullulan, a polymer of maltotriose subunits, and was shown to be mildly important for growth on starch (D’Elia and Salyers, 1996; Reeves et al., 1996). In our data, SusA was only important for growth on α-cyclodextrin, a cyclic oligosaccharide of 6 glucose monomers linked by α-1,4 glycosidic bonds (Figure 1F). SusA homologs in other species are often annotated as a cyclomaltodextrinase (enzyme that linearizes circular oligosaccharides), and homologs are required for growth on α-cyclodextrin in Caulobacter crescentus and Echinocola vietnamensis KMM 6221 (Price et al., 2018). These data suggest that SusA also has cyclomaltodextrinase activity, as previously proposed (Smith and Salyers, 1991).SusB is a GH97 family member that hydrolyzes multiple linkages, including α-1,4 bonds (Kitamura et al., 2008). In previous work, a ΔsusB mutant had negligible growth defects on starch, raising doubts about the role of SusB in the use of starch and related compounds (D’Elia and Salyers, 1996; Reeves et al., 1996). We found that insertions in susB had fitness defects on several forms of starch (Figure 1F), motivating us to re-examine the role of SusB in polysaccharide utilization. To confirm that we were measuring a bona fide phenotype of susB and not polar effects on downstream genes like susCD, we generated a clean deletion. The strongest fitness defects of susB insertions were during growth on α-cyclodextrin, and we found that ΔsusB indeed had a growth defect on this substrate compared to glucose (Figures 1G and 1H). We used quantitative real-time RT-PCR to test if the susB mutation disrupted the expression of susCD during growth on α-cyclodextrin. Instead, we found that expression levels of susC and susD were 2.7-fold higher in ΔsusB cells than in wild-type cells (susC: 2.7-fold, 95% confidence interval: 2.0–3.7; susD: 2.7-fold, 95% confidence interval: 2.1–3.4).SusG breaks down extracellular starch into simpler oligosaccharides (Koropatkin and Smith, 2010). Consistent with this function, in our fitness data, susG was highly important for growth on starch and amylopectin and to a lesser extent on amylose and α-cyclodextrin (Figure 1F). Although the outer membrane proteins SusE and SusF have been demonstrated to bind starch (Cameron et al., 2012), their precise function remains unclear. We found that these proteins were only important for growth on amylopectin, suggesting that SusEF preferentially bind this component of starch or they are important for stabilization of the SusG-starch complex, as previously proposed (Foley et al., 2018; Tuson et al., 2018). Taken together, our re-evaluation of the Sus based on genetics data across multiple substrates adds insight into this well-studied system.To globally associate PULs to carbon substrates, we identified instances in which at least one gene in a given PUL had a specific and important phenotype (fitness, <−1). Based on these criteria, we associated 20 of the 93 known PULs (Terrapon et al., 2018) to 32 substrates (Table S4; Figure 2). We identified specific and important phenotypes for ≥3 genes from multiple PULs on their predicted or known polysaccharides (Table S4; Figure 2), as follows: PUL5 on arabinogalactan (Cartmell et al., 2018), PUL22 on fructans (Sonnenburg et al., 2010), PUL48 on dextran (Rogers et al., 2013), PUL57 on chondroitin sulfate (CS) and hyaluronic acid (HA) (Raghavan et al., 2014), PUL68 on mannan (Cuskin et al., 2015), PUL75 on pectic glycans (Luis et al., 2018), PUL85 on heparin (Cartmell et al., 2017), and PUL86 on type I rhamnogalacturonan (Luis et al., 2018).In addition to confirming the results of previous studies, our screen provided genetic evidence for the prediction that, besides PUL5, PUL47 and PUL80 are also important for arabinogalactan utilization (Martens et al., 2011). The arabinogalactan phenotypes for genes in PUL47 and PUL80 were mild (fitness values ranged from −0.7 to −1.4), but similar phenotypes were observed for ≥3 genes in each PUL, and the genes in these clusters did not have clear phenotypes under any other conditions profiled (Figures 3A and 3B). Even though these genes did not meet the criteria for a specific phenotype, mutants in these genes had statistically significant defects, all with ∣t∣ > 4, and an estimated false discovery rate of <5% (STAR methods). These data demonstrate that efficient breakdown of certain complex polysaccharides like arabinogalactan requires multiple PULs. Additional genes (PUL65 and BT3674–BT3687) are involved in removing glucuronic acid units from gum Arabic arabinogalactan (Cartmell et al., 2018), but these genes had weak phenotypes (fitness, >−0.5) during growth on larch wood arabinogalactan, consistent with the low glucuronic acid content of larch wood arabinogalactan (Willför et al., 2002).Our analysis also linked PULs to specific monosaccharides and disaccharides. For example, the SusCD homologs of PUL15 (BT1118 and BT1119) were important on galacturonic acid and 1,4-β-D-galactobiose (Figure 3C). Consistent with Glowacki et al. (2020) results, components of PUL37 were only important for growth on D-ribose, in particular a hypothetical protein (BT2802; fitness, −4.3), two predicted ribokinases (BT2803 and BT2804), and a sugar transporter (BT2809) (Figure 3D). The SusCD homologs in this cluster were also mildly important on D-ribose (BT2805 fitness, −1.3; BT2806 fitness, −1.4). Finally, a putative glucosidase (BT3567), SusD-like protein (BT3568), and SusC-like protein (BT3569) from PUL61 were required for optimal growth on the disaccharide laminaribiose (Figure 3E). Although the PULs of B. thetaiotaomicron have typically been studied in the context of polysaccharides, our data suggest that some PULs evolved to catabolize simpler substrates. In support of this view, PUL37 and PUL61 contain the core SusCD proteins but do not have homologs of SusEF or hydrolases expected to act on complex substrates.Identification of polysaccharide degradation genesAlthough the degradation pathways for some polysaccharides are well described in B. thetaiotaomicron, prior work found that genes important in glycan utilization can be located outside the corresponding PUL (Benjdia et al., 2011; Raghavan et al., 2014). By examining genes with specific and important phenotypes, we identified three genes for polysaccharide utilization. The first example is in metabolism of type I rhamnogalacturonan (RG-I), a polysaccharide containing repeats of a disaccharide of galacturonic acid and rhamnose. A recent study proposed the involvement of five GH28-family glycosyl hydrolases (BT4123, BT4146, BT4149, BT4153, and BT4155) in breaking down RG-I (Luis et al., 2018). In our fitness data, only BT4149 was important for utilizing RG-I, and this phenotype was mild (fitness, −1.3). Rather, we found that another GH28-family member, BT4187, was more important for growth on RG-I (fitness, −2.2) (Figure 3F). In support of the importance of BT4187 in RG-I degradation, the operon containing BT4187 is predicted to be in the same regulon as PUL75 (Ravcheev et al., 2013), which is both important for fitness (Figure 2) and induced during growth on RG-I (Martens et al., 2011). Although it is possible that the other hydrolases are functionally redundant, our results demonstrate that the loss of BT4187 alone results in a strong fitness defect during growth on RG-I.The next two examples associate genes of previously unknown function with utilization of host-derived glycosaminoglycans (GAGs), which are structural components of animal tissues and are composed of repeating disaccharide units of a hexuronic acid linked to an amino sugar. B. thetaiotaomicron is capable of using GAGs as nutrients, including CS, HA, and heparin (Cartmell et al., 2017; Ndeh and Gilbert, 2018; Raghavan and Groisman, 2015). CS and HA differ by their repeating disaccharide; CS contains sulfated N-acetylgalactosamine and glucuronic acid, and HA contains N-acetylglucosamine and glucuronic acid. Previous studies proposed a model for CS and HA catabolism by B. thetaiotaomicron involving PUL57 and the CS lyase BT4410 (Martens et al., 2008; Raghavan et al., 2014). In our data, components of PUL57 were indeed required for optimal growth on CS and HA (Figure 3G), whereas other components of PUL57 were important on one substrate only. BT3349 (a sulfatase) was only important on CS, as expected given that only CS contains sulfated sugars. The polysaccharide lyase family PL29 enzyme encoded by BT3328 was only important for fitness on HA, despite the activity of this enzyme against both HA and CS in vitro (Ndeh et al., 2018).The first GAG utilization gene we identified was BT4411, which was important for growth on HA but not on CS (Figure 3G). BT4411 is predicted to be in the same operon as the CS lyase BT4410, which was also more important for fitness on HA than on CS. BT4411 contains a domain of unknown function (DUF4627) that has significant similarity to CBM_4_9 (carbohydrate binding domain PF02018) (El-Gebali et al., 2019). The BT4410–BT4411 operon is conserved in several other Bacteroides species, and both proteins have putative signal peptides (Yu et al., 2010), suggesting that BT4410 and BT4411 function together in HA catabolism.Finally, we found that BT3233 was important for growth on the following 3 GAGs: CS (mean fitness, −1.6), HA (mean fitness, −0.7), and heparin (mean fitness, −2.2) (Figure 3G). BT3233 is a member of the uncharacterized protein family DUF4861 (El-Gebali et al., 2019) and belongs to a superfamily of putative carbohydrate binding domains (galactose mutarotase-like domain, InterPro IPR011013) (Mitchell et al., 2019). Homologs of BT3233 are mostly confined to Bacteroides species, including a number of human commensals. Using an archived transposon mutant, we confirmed that BT3233 is important for growth on CS (Figure 3H,I). Although its precise function is unknown, our data suggest that BT3233 plays an important role in GAG catabolism.Disaccharide catabolism through a putative 3-keto intermediateOur fitness assays revealed that components of a gene cluster (BT2157, BT2159, and BT2160) were important for catabolizing the disaccharides trehalose (glucose-α-1,1-glucose), leucrose (glucose-α-1,5-fructose), palatinose (glucose-α-1,6-fructose), and maltitol (glucose-α-1,4-glucitol) and the trisaccharide raffinose (galactose-α-1,6-glucose-β-1,2-fructose) (Figure 4A). BT2160 encodes a SusR-family transcription factor that likely regulates the cluster (Ravcheev et al., 2013). BT2156 did not have a significant phenotype under any assayed conditions. B. thetaiotaomicron contains a second gene cluster (BT4446–BT4449) that is highly similar to BT2156–BT2159 (Figure S1A), except that BT4446–BT4449 does not contain a SusR-family regulator nor did it have a significant phenotype in our data. BT2158 is 98% identical at the nucleotide level to BT4448; hence, we were not able to unambiguously map transposon insertions within these genes by using our standard parameters. To determine if BT2158 was important for fitness, we re-ran our transposon insertion mapping with less stringency by only requiring one nucleotide difference to assign insertions to BT2158 and BT4448, which successfully generated fitness data for both genes. Although BT4448 did not have strong phenotypes, BT2158 was important for growth on trehalose, leucrose, palatinose, maltitol, and raffinose (fitness, ≥ −1.3 in all experiments on these substrates). Furthermore, BT2158 had the highest co-fitness with BT2160 (r = 0.67). To confirm these results, we generated individual deletions of BT2158 and BT4448 and assayed growth of these mutants on glucose or trehalose. As expected, ΔBT4448 did not exhibit a growth defect on either substrate (Figures S1E and S1F), whereas ΔBT2158 did not grow on trehalose (Figure S1F). These results demonstrate that BT2157–BT2160 is important for fitness on similar substrates and likely act in the same pathway. A recent genetic and biochemical study found that components of BT2156–BT2160 are also capable of transforming glucosinolates (which are thioglycosides, so there is some chemical similarity to glycosides) to produce isothiocyanates (Liou et al., 2020).To explore this operon, we examined the protein sequences. BT2156 encodes a predicted lolE-like epimerase, BT2157 encodes a DUF1080-containing protein, and BT2158 and BT2159 encode MviM-like dehydrogenases. DUF1080 has structural similarity to endo-1,3-1,4-β-glucanases (El-Gebali et al., 2019) and is likely a family of hydrolases. Interestingly, homologs of BT2156–BT2159 are chromosomally clustered and are important for catabolizing disaccharides and other glycosides in C. crescentus, Pedobacter sp. GW460-11-11-14-LB5, and E. vietnamensis KMM 6221 (Figures S1 and S2) (Price et al., 2018). Thus, we considered that B. thetaiotaomicron and these other species might catabolize glycosides through similar mechanisms. In C. crescentus, homologs of BT2156–BT2159 are in a genome cluster with the glucoside 3-dehydrogenase LacABC (Figure S1B), and these homologs share similar phenotypes on glycosides as LacABC (Figure S2A). LacABC is the first step in lactose catabolism by a 3-keto-lactose intermediate (Arellano et al., 2010). Similarly, sucrose degradation in Agrobacterium tumefaciens proceeds through a 3-keto-sucrose intermediate (Bernaerts and De Ley, 1960; Schuerman et al., 1997), and in Sinorhizobium meliloti, degradation of various disaccharides occurs through 3-keto intermediates (Ampomah et al., 2013). A. tumefaciens and S. meliloti use the dehydrogenase ThuAB (unrelated to LacABC) to form 3-keto-disaccharides (Bernaerts and De Ley, 1960; Schuerman et al., 1997). The 3-keto-disaccharides are thought to be hydrolyzed to a hexose and a 3-keto-hexose (Hayano and Fukui, 1970), followed by reduction of the 3-keto-hexose (Hayano et al., 1973), but the genes involved are not known.We hypothesized that B. thetaiotaomicron degrades glycosides by 3-keto intermediates through activity of BT2157–BT2160, even though B. thetaiotaomicron does not have LacABC or ThuAB. We focused on the utilization of trehalose; BT2157–BT2160 had relatively strong phenotypes on trehalose, and homologs of some of these proteins are important for trehalose utilization in Pedobacter sp. GW460-11-11-14-LB5 and E. vietnamensis KMM 6221 (Figures S2B and S2C). First, we investigated whether expression of this operon was induced on trehalose by using proteomics of stationary-phase cultures. We observed a ≥9-fold increase in abundance for three of the target proteins (BT2157–BT2159) during growth on trehalose compared to glucose, suggesting that this system is regulated by trehalose availability (Figure 4B). Next, we generated a deletion of BT2160 and assayed its ability to grow on trehalose. Similar to ΔBT2158, BT2160 was required for growth on trehalose (Figure 4C). Conversely, neither gene was required for growth on glucose (Figure 4D). Lastly, to provide evidence that the degradation pathway proceeds by a 3-keto-trehalose intermediate, we determined whether the DUF1080 hydrolase encoded by BT2157 was specific to 3-keto-disaccharides. We purified BT2157 and used Agrobacterium rhizogenes to produce a supernatant containing 3-keto-trehalose. BT2157 hydrolyzed >70% of the 3-keto-trehalose in the supernatant within 1 min (Figure 4E). In contrast, the concentration of trehalose in the supernatant only decreased by ~14% after 10 min of incubation with BT2157. BT2157 appears to hydrolyze 3-keto-trehalose in vitro in the absence of other products of the BT2156–BT2160 operon and more rapidly than the glucosinolates glucoraphanin or glucobrassicin (Liou et al., 2020). Our data demonstrate that BT2157–BT2160 is the primary route of trehalose catabolism in B. thetaiotaomicron, that the expression of these proteins is induced by trehalose, and that BT2157 preferentially hydrolyzes 3-keto-trehalose, thereby supporting a model that trehalose degradation proceeds through a 3-keto-glycoside intermediate.Identification of an efflux system for bile saltsThe human gut involves exposure to growth-promoting complex polysaccharides and also to bile salts, which can inhibit bacterial growth. Consequently, commensal bacteria have evolved strategies to cope with bile salts, including removal by efflux pumps, deconjugation by bile salt hydrolases (BSHs), and oxidation and epimerization of hydroxyl groups by hydroxysteroid dehydrogenases (HSDHs) (Begley et al., 2005). BSH and HSDH homologs have been identified in B. thetaiotaomicron (Ridlon et al., 2006), and BT2086 has BSH activity in vitro and in vivo (Yao et al., 2018).To identify B. thetaiotaomicron genes involved in bile salt tolerance, we examined fitness in the presence of inhibitory concentrations of the primary bile acid chenodeoxycholic acid, the secondary bile acids lithocholic acid or deoxycholic acid, or cholic-acid-derived bile salts (a mixture of sodium cholate and sodium deoxycholate, which we refer to generically as “bile salts”). From these data, we identified a four-gene operon, BT2792–BT2795, important for fitness under deoxycholic acid, chenodeoxycholic acid, and bile salt stress (Figure 5A). These genes were not important when stressed with lithocholic acid, at least under the concentrations assayed. Based on homology (Overbeek et al., 2005), these proteins form a tripartite multidrug efflux system. Interestingly, these genes did not have a significant phenotype under any other conditions profiled (Figure 5A), including dozens of diverse antibiotics, suggesting that this efflux system is specific to bile acids and bile salts. Closely related efflux systems are present in other Bacteroides species (Dehal et al., 2010), although the importance of those genes for bile salt tolerance remains to be determined.To confirm the involvement of this operon in bile salt tolerance, we assayed the growth of archived transposon mutants. BT2793, BT2794, and BT2795 mutants all had longer lag phases and reduced growth rates relative to controls when stressed with 0.5 mg/mL bile salts (Figures 5B and 5C). To determine if this efflux system is induced by the stressor, we performed targeted proteomic analysis by using wild-type cells grown in the presence of various concentrations of bile salts. BT2792, BT2794, and BT2795 were all induced in the presence of bile salts, and the amount of induction increased with higher bile salt concentrations (Figures 5D and 5E), consistent with previous observations of bile-salt-mediated upregulation of efflux pumps in other human gut commensals (Lin et al., 2005; Prouty et al., 2004; Pumbwe et al., 2007; Ruiz et al., 2013). Taken together, our data show that the efflux system encoded by BT2792–BT2795 is induced by and important for resistance to bile salts.Association of antibiotics with transport systemsThe use of antimicrobial compounds to treat infection means that gut commensals are regularly exposed to these inhibitors; thus, we examined fitness with different classes of inhibitors including antibiotics and biocides. Given the prevalence of efflux as a mechanism of antibiotic resistance in bacteria (Webber and Piddock, 2003), we focused on transport-related proteins with strong phenotypes. For example, BexA (BT0588) belongs to the multidrug and toxic compound extrusion (MATE) family and has been shown to be responsible for efflux of the fluoroquinolone antibiotics norfloxacin and ciprofloxacin in B. thetaiotaomicron (Miyamae et al., 2001). In our dataset, BT0588 mutants exhibited a mild fitness defect with ciprofloxacin (minimum fitness, −1.6). Insertions in BT0588 resulted in a more pronounced phenotype with another fluoroquinolone (lomefloxacin) or with paraquat (Figure 6A). In another example, we found that the RND transporter system encoded by BT3337–BT3339 was important for fusidic acid tolerance (Figure 6B) and to a lesser extent for tolerating cefoxitin and the antipsychotic drug chlorpromazine.Lastly, we identified a two-gene operon, BT1439–BT1440, with high positive fitness when grown in the presence of vancomycin (Figure 6C), a glycopeptide antibiotic that inhibits peptidoglycan biosynthesis in the periplasm. Although vancomycin is usually thought to target Gram-positive bacteria, it will inhibit Gram-negative bacteria if it passes through the outer membrane (Antonoplis et al., 2019). Vancomycin is commonly used for treating Clostridium difficile infections (CDIs) and is often taken orally. Fecal levels of vancomycin in CDI patients can exceed 2 mg/mL during treatment (Gonzales et al., 2010), which is above the concentrations used in our screen (0.009–0.019 mg/mL). Thus, our data suggest that vancomycin use in humans can impact B. thetaiotaomicron in the gut (Basolo et al., 2020). BT1439 is a SusD homolog and BT1440 is a SusC homolog, and these genes had high co-fitness (r = 0.75), indicating that they likely act together. Although SusCD complexes are best known for translocating carbohydrate substrates, no growth defect was found on any of the carbohydrates assayed. To validate our data, we constructed a BT1439 deletion mutant and found that it was more resistant to vancomycin than the parent (Figures 6D, 6E, and S3). Interestingly, the vancomycin phenotype of BT1439–BT1440 was only observed in BHIS medium and not in VB medium (Figure 6C). This medium-specific phenotype could be due to altered regulation or to vancomycin binding to another compound in the growth medium (such as the peptides in BHIS).B. thetaiotaomicron uses alternative biosynthetic enzymes to adapt to ammonium availability in vivoWe were interested in whether data from our in vitro chemical-genetic screen could illuminate fitness phenotypes in vivo. To generate in vivo fitness data, we monocolonized germ-free mice with our barcoded library; collected fecal samples at days 2, 4, and 7 post-gavage; and quantified gene fitness using BarSeq. We first compared our in vivo data to those of a previous study that colonized mice with a set of Bacteroides species including B. thetaiotaomicron VPI-5482, in both a low-fat, high-plant polysaccharide (LF/HPP) and a high-fat, high-sugar (HF/HS) diet (Wu et al., 2015). Our fitness data were more correlated with the LF/HPP (r = 0.52, 95% confidence interval: 0.46–0.58) (Figure 7A) than the HF/HS diet (r = 0.33, 95% confidence interval: 0.25–0.40), as expected based on the mouse diet in our in vivo experiment (STAR methods). This correlation in part reflects the nutrients available in vivo to B. thetaiotaomicron, as there was significant overlap in genes related to amino acid biosynthesis (odds ratio = 28, 95% confidence interval: 9–92) and carbohydrate-active enzymes (CAZymes) (odds ratio = 40, 95% confidence interval: 15–102) (Figure 7B). Thus, our data are consistent with the previous study of B. thetaiotaomicron fitness in vivo in the context of a high-plant-polysaccharide diet.Although insertions in amino acid biosynthesis genes had immediate deleterious effects on growth in our dataset, the fitness defects of CAZymes tended to be smaller at first and increased in magnitude over time (Figure 7C, top), likely reflecting the foraging of B. thetaiotaomicron on a diverse set of carbohydrates in the gut (Sonnenburg et al., 2005). Nonetheless, we identified significant growth defects in vivo for genes in multiple PULs, including the arabinan-activated PUL7 (BT0350, BT0351, and BT0367), and predicted PUL27 (BT2262–BT2264) (Figure 7C, bottom); PUL27 is constitutively expressed in vitro (Glenwright et al., 2017) but has no clearly defined substrates and was not important for growth under any conditions from our in vitro screen. The in vivo phenotypes of PUL27 suggest that it functions in the uptake of a diet- or host-derived nutrient that has yet to be identified.The biosynthetic pathways for arginine and lysine were critical for growth in vivo, as previously observed (Goodman et al., 2009; Wu et al., 2015). Our in vitro data suggested that for both pathways, alternative enzymes are used depending on ammonium availability. The first example is synthesis of meso-diaminopimelate (mDAP), a precursor to lysine and peptidoglycan. Two alternative pathways for producing mDAP from 2,3,4,5-tetrahydropicolinate are conserved among the Bacteroides (Figure 7D). The dehydrogenase branch, catalyzed in a single step by Ddh (BT1979), uses free ammonium as a nitrogen source. The transaminase branch, catalyzed in two steps by DapL (BT0547) and DapF (BT0548), uses glutamate as a nitrogen source. The functions of DapL and Ddh have been verified biochemically using enzymes from B. fragilis (Hudson et al., 2008, 2011), which have 90% and 95% amino acid identity to the B. thetaiotaomicron enzymes, respectively. In Corynebacterium glutamicum, the dehydrogenase branch preferentially operates when ammonium is abundant; flux through the dehydrogenase branch is correlated with ammonium levels (Sonntag et al., 1993), and mutants in the transaminase branch require high levels of ammonium (Wehrmann et al., 1998). In our B. thetaiotaomicron library, ddh was important for fitness during growth in minimal media with ammonium but had no phenotype when ammonium was replaced with amino acids or other organic nitrogen sources (Figure 7E). Instead, insertions in dapLF were deleterious under these low ammonium conditions (Figure 7E), suggesting that relative flux through the two branches depends on availability of free ammonium in B. thetaiotaomicron as well (Figure 7D).The second example of use of alternative enzymes is carbamoyl phosphate synthase (CPS), which synthesizes a nitrogen-containing precursor (carbamoyl phosphate) to arginine and pyrimidine nucleotides. In eubacteria and eukaryotes, CPS is composed of two subunits; the glutaminase subunit (CarA) liberates ammonium from glutamine and funnels it to the synthetase subunit (CarB), where the ammonium combines with other reactants to form carbamoyl phosphate (Figure 7D; Anderson and Meister, 1965). To date, all eubacterial CPSs have been found to be glutamine dependent. However, eubacterial CarB functions as an ammonium-dependent CPS when purified alone (Trotta et al., 1971), and mutations in the glutaminase subunit of the mitochondrial CPS that participate in the urea cycle make the enzyme ammonium dependent (Saeed-Kothe and Powers-Lee, 2003). B. thetaiotaomicron carries a two-gene locus, BT0556–BT0557, encoding a full CPS (CarAB) as well as a lone synthetase subunit BT3866 (CarB2) with no clear glutaminase partner (Figure 7D). Unexpectedly, the fitness defects of insertions in carAB were limited to minimal media without added ammonium (Figure 7E). With ammonium present, insertions in carAB had no phenotype, whereas carB2 was very important for fitness (Figure 7E). There were no genes that were both co-fit with carB2 and had clear sequence homology to a CPS glutaminase subunit. In particular, carA (BT0556) was co-fit with carB (BT0557) but not carB2 (r = 0.85 and r = −0.02, respectively). In other bacteria with two CPSs, one enzyme supplies carbamoyl phosphate for arginine biosynthesis, whereas the second is responsive to needs for pyrimidine biosynthesis (Paulus and Switzer, 1979), regardless of the nitrogen source. We propose that in B. thetaiotaomicron, the use of CarAB or CarB2 depends primarily on ammonium availability, with CarAB acting as a prototypical glutamine-dependent enzyme and CarB2 using ammonium to directly synthesize carbamoyl phosphate (Figure 7D).In our mouse experiment, both carAB and dapLF were critical for growth in vivo (Figure 7E). This observation is consistent with results of a previous study, in which carAB and dapL insertions in B. thetaiotaomicron VPI-5482 were less fit in a LF/HPP diet (similar to our in vivo experiment), whereas carB2 and ddh insertions were less fit in a HF/HS diet (Wu et al., 2015). Likewise, homologs of carB2 and ddh were important for fitness in a HF/HS diet but not in a LF/HPP diet in B. thetaiotaomicron 7330 (Wu et al., 2015). The phenotypes of carB2 and ddh were also greatly diminished in B. ovatus ATCC8483 when an arabinoxylan supplement was added to the diet (Wu et al., 2015). Because the diet was otherwise held constant, this last result shows that addition of microbiota-accessible carbohydrates is sufficient to switch Bacteroides ovatus from use of Ddh and CarB2 to DapLF and CarAB and suggests that it is the presence of microbiota-accessible carbohydrates in a LF/HPP diet (in contrast to a HF/HS diet) that has the same effect on other Bacteroides species.We propose the following model to synthesize these observations (Figure 7F). The fermentation of protein by the colonic microbiota is a major source of ammonium in the gut (Wrong et al., 1985), but the presence of fermentable carbohydrates significantly reduces the amount of ammonium produced (Birkett et al., 1996; Mortensen, 1992). Thus, when B. thetaiotaomicron grows in the context of a diet high in plant polysaccharides, it ferments the undigested carbohydrates and ammonium concentration in the gut remains low. In this environment, DapLF and CarAB are the preferred enzymes. Conversely, when B. thetaiotaomicron grows in a low-plant-polysaccharide environment, protein fermentation plays a larger role in energy generation and excess ammonium is liberated from the catabolic deamination of amino acids (Allison and Macfarlane, 1989; Dai et al., 2011; Smith and Macfarlane, 1996). B. thetaiotaomicron then takes advantage of the excess ammonium by directly incorporating it into nitrogen-containing compounds using Ddh and CarB2. Both carAB and dapLF are found in the high-affinity ammonium locus of B. thetaiotaomicron VPI-5482, and their expression is induced in vitro when ammonium is limiting (Iakiviak, 2017), suggesting that transcriptional control in response to ammonium levels is a critical regulatory mechanism controlling flux through the alternative pathways.

A gene-phenotype map of B. thetaiotaomicronTo enable high-throughput genetics in B. thetaiotaomicron VPI-5482, we generated a genome-wide RB-TnSeq library in this bacterium. The final library contains 315,668 uniquely barcoded mutants with insertions that we could confidently map to a single location in the genome (Figure 1A). To enable follow-up studies, we generated a 40-plate archived collection of individual transposon-insertion mutants in a 96-well microplate format by sorting cells using flow cytometry (Shiver et al., 2019).To validate the utility of the mutant library, we grew it in a defined medium with either glucose or xylose as the sole carbon source. For each experiment, we measured the abundance of the DNA barcodes by sequencing (BarSeq) before and after selection, and we defined the fitness of a gene as the average log2 change in relative abundance of its mutants (Figure 1B). Negative and positive values mean that the mutants in that gene were less and more fit, respectively, than the average strain. Many genes involved in amino acid biosynthesis were important for fitness in either carbon source because the medium lacked most amino acids. Genes from the xylose utilization pathway (BT0792, xylulose kinase; BT0793, xylose isomerase; BT0794, xylose transporter) were important for growth on xylose only (Figure 1B). This example illustrates the biological consistency of fitness assays using our mutant library.To create a gene-phenotype compendium, we performed 523 genome-wide fitness assays that passed metrics for internal consistency (Wetmore et al., 2015; STAR methods), and all analyses in this study are derived from these experiments. Many experiments were in defined Varel-Bryant (VB) media with a single carbon source. In addition, we performed hundreds of experiments with an inhibitory concentration of a stressor such as an antibiotic, biocide, or bile salt. Finally, we measured the in vivo fitness of each mutant during mono-colonization of germ-free mice. We successfully quantified fitness for 4,055 protein-coding genes in each experiment. Among the genes with no fitness data, we estimate that 378 are likely essential for viability in the brain heart infusion supplemented (BHIS) medium used to generate the library (Table S1). Of these genes, 202 (53%) were also identified as candidate essential genes in a previous study of transposon insertion mutants grown in a rich medium similar to BHIS medium (Goodman et al., 2009).We identified 516 genes with a specific phenotype (Table S2), which we defined as a gene-condition combination with ∣fitness∣ > 1 that was statistically significant and for which the gene lacked a significant fitness change in most other experiments (STAR methods). A specific phenotype often links a gene to a condition that is directly related to its function (Price et al., 2018). For example, BT3788 is a SusC homolog located in the PUL for mannan utilization (Cuskin et al., 2015); across all 48 carbon sources profiled, BT3788 was important only for fitness on mannan (Figure 1C). BT4507 encodes a class A serine β-lactamase (Stentz et al., 2015); mutants in this gene were impaired on two β-lactam antibiotics (ceftriaxone and cephalothin) but not under any of the other conditions profiled, including other antibiotics (Figure 1D).A second method for inferring gene function is co-fitness (Price et al., 2018), a measure of the degree to which pairs of genes share similar fitness profiles across a large genetic dataset (Table S3). To illustrate this approach, we used co-fitness to identify the enzyme responsible for the first step of arginine biosynthesis in B. thetaiotaomicron. Bacteroides species use a non-canonical pathway for synthesizing arginine, in which intermediates are succinylated instead of acetylated (Cabrera-Luque, 2010; Shi et al., 2006). A majority of the enzymes known to be involved in arginine synthesis in Escherichia coli (Caldara et al., 2008) have clear homology to enzymes in B. thetaiotaomicron; a notable exception is the first step, in which N-acetylglutamate synthase appears to have been replaced by an evolutionarily unrelated N-succinylglutamate synthase. BT3761 has been predicted to fill this role, in part because it is in a 5-gene operon of arginine biosynthetic genes. In addition, BT3761 belongs to the GNAT family of N-acyltransferases and is 59% identical (protein sequence) to an unusual N-acetyl-glutamate synthase from Caldithrix abyssi (Cabrera-Luque, 2010; Kublanov et al., 2017). The four most co-fit genes with BT3761 are involved in arginine biosynthesis (BT3759, homologous to argC in E. coli; BT3733, argH; BT3760, argG; and BT3395, argB). Furthermore, BT3761 was critical for growth on minimal media without amino acids, and its fitness defect was rescued through addition of arginine (Figure 1E). Additionally, quantitative real-time reverse-transcriptase PCR (RT-PCR) revealed that BT3761 RNA levels in wild-type cells were repressed 6.4-fold upon addition of 3 mM arginine (95% confidence interval: 5.3- to 7.8-fold repression). These results support the hypothesis that BT3761 participates in arginine biosynthesis as an N-succinylglutamate synthase.Overall, we identified a functional association (a specific phenotype, or co-fitness of ≥0.8) for 633 of the 4,055 protein-coding genes for which we collected data. Genes with a functional association were more likely to have a specific Clusters of Orthologous Groups (COG) functional category (Tatusov et al., 2003) assigned than genes with no functional association (55% versus 39%; p = 10−15, Fisher’s exact test). Nonetheless, we were able to identify a functional association for 270 genes without a specific COG function category. According to the PaperBLAST database (Price and Arkin, 2017), 174 of these genes have not been previously discussed in the literature, nor do they have close homologs (>70% amino acid identity) to other Bacteroides species.

A gene-phenotype map of B. thetaiotaomicron

To enable high-throughput genetics in B. thetaiotaomicron VPI-5482, we generated a genome-wide RB-TnSeq library in this bacterium. The final library contains 315,668 uniquely barcoded mutants with insertions that we could confidently map to a single location in the genome (Figure 1A). To enable follow-up studies, we generated a 40-plate archived collection of individual transposon-insertion mutants in a 96-well microplate format by sorting cells using flow cytometry (Shiver et al., 2019).

To validate the utility of the mutant library, we grew it in a defined medium with either glucose or xylose as the sole carbon source. For each experiment, we measured the abundance of the DNA barcodes by sequencing (BarSeq) before and after selection, and we defined the fitness of a gene as the average log2 change in relative abundance of its mutants (Figure 1B). Negative and positive values mean that the mutants in that gene were less and more fit, respectively, than the average strain. Many genes involved in amino acid biosynthesis were important for fitness in either carbon source because the medium lacked most amino acids. Genes from the xylose utilization pathway (BT0792, xylulose kinase; BT0793, xylose isomerase; BT0794, xylose transporter) were important for growth on xylose only (Figure 1B). This example illustrates the biological consistency of fitness assays using our mutant library.

To create a gene-phenotype compendium, we performed 523 genome-wide fitness assays that passed metrics for internal consistency (Wetmore et al., 2015; STAR methods), and all analyses in this study are derived from these experiments. Many experiments were in defined Varel-Bryant (VB) media with a single carbon source. In addition, we performed hundreds of experiments with an inhibitory concentration of a stressor such as an antibiotic, biocide, or bile salt. Finally, we measured the in vivo fitness of each mutant during mono-colonization of germ-free mice. We successfully quantified fitness for 4,055 protein-coding genes in each experiment. Among the genes with no fitness data, we estimate that 378 are likely essential for viability in the brain heart infusion supplemented (BHIS) medium used to generate the library (Table S1). Of these genes, 202 (53%) were also identified as candidate essential genes in a previous study of transposon insertion mutants grown in a rich medium similar to BHIS medium (Goodman et al., 2009).

We identified 516 genes with a specific phenotype (Table S2), which we defined as a gene-condition combination with ∣fitness∣ > 1 that was statistically significant and for which the gene lacked a significant fitness change in most other experiments (STAR methods). A specific phenotype often links a gene to a condition that is directly related to its function (Price et al., 2018). For example, BT3788 is a SusC homolog located in the PUL for mannan utilization (Cuskin et al., 2015); across all 48 carbon sources profiled, BT3788 was important only for fitness on mannan (Figure 1C). BT4507 encodes a class A serine β-lactamase (Stentz et al., 2015); mutants in this gene were impaired on two β-lactam antibiotics (ceftriaxone and cephalothin) but not under any of the other conditions profiled, including other antibiotics (Figure 1D).

A second method for inferring gene function is co-fitness (Price et al., 2018), a measure of the degree to which pairs of genes share similar fitness profiles across a large genetic dataset (Table S3). To illustrate this approach, we used co-fitness to identify the enzyme responsible for the first step of arginine biosynthesis in B. thetaiotaomicron. Bacteroides species use a non-canonical pathway for synthesizing arginine, in which intermediates are succinylated instead of acetylated (Cabrera-Luque, 2010; Shi et al., 2006). A majority of the enzymes known to be involved in arginine synthesis in Escherichia coli (Caldara et al., 2008) have clear homology to enzymes in B. thetaiotaomicron; a notable exception is the first step, in which N-acetylglutamate synthase appears to have been replaced by an evolutionarily unrelated N-succinylglutamate synthase. BT3761 has been predicted to fill this role, in part because it is in a 5-gene operon of arginine biosynthetic genes. In addition, BT3761 belongs to the GNAT family of N-acyltransferases and is 59% identical (protein sequence) to an unusual N-acetyl-glutamate synthase from Caldithrix abyssi (Cabrera-Luque, 2010; Kublanov et al., 2017). The four most co-fit genes with BT3761 are involved in arginine biosynthesis (BT3759, homologous to argC in E. coli; BT3733, argH; BT3760, argG; and BT3395, argB). Furthermore, BT3761 was critical for growth on minimal media without amino acids, and its fitness defect was rescued through addition of arginine (Figure 1E). Additionally, quantitative real-time reverse-transcriptase PCR (RT-PCR) revealed that BT3761 RNA levels in wild-type cells were repressed 6.4-fold upon addition of 3 mM arginine (95% confidence interval: 5.3- to 7.8-fold repression). These results support the hypothesis that BT3761 participates in arginine biosynthesis as an N-succinylglutamate synthase.

Overall, we identified a functional association (a specific phenotype, or co-fitness of ≥0.8) for 633 of the 4,055 protein-coding genes for which we collected data. Genes with a functional association were more likely to have a specific Clusters of Orthologous Groups (COG) functional category (Tatusov et al., 2003) assigned than genes with no functional association (55% versus 39%; p = 10−15, Fisher’s exact test). Nonetheless, we were able to identify a functional association for 270 genes without a specific COG function category. According to the PaperBLAST database (Price and Arkin, 2017), 174 of these genes have not been previously discussed in the literature, nor do they have close homologs (>70% amino acid identity) to other Bacteroides species.

Functional-genetic screen links PULs to specific substratesFitness profiling across multiple substrates provided a broad view of the importance of each PUL component to carbon catabolism. The 48 carbon sources we assayed represent a range of substrate complexity, including 14 monosaccharides and their derivatives, 12 disaccharides, 1 trisaccharide, 2 tetrasaccharides, 3 oligosaccharides (>4 monomers), 2 sugar alcohols, and 14 complex polysaccharides. As an illustration, we re-examined the Sus system by comparing its phenotypes during growth on starch; the component polysaccharides of starch (amylopectin and amylose); the oligosaccharide α-cyclodextrin; and the simpler break-down products glucose, maltose, maltotetraose, and maltohexaose (Figure 1F). The SusCD transporter proteins were important on all saccharides of a length of ≥4 but were largely dispensable for growth on maltose and glucose, which is in agreement with previous findings that SusD importance varies depending on the size of glucose polymers (Koropatkin et al., 2008).The three glycoside hydrolases (GHs; SusA, SusB, and SusG) were variably important across substrates. SusA was previously characterized as a neopullulanase that cleaves α-1,4 linkages in pullulan, a polymer of maltotriose subunits, and was shown to be mildly important for growth on starch (D’Elia and Salyers, 1996; Reeves et al., 1996). In our data, SusA was only important for growth on α-cyclodextrin, a cyclic oligosaccharide of 6 glucose monomers linked by α-1,4 glycosidic bonds (Figure 1F). SusA homologs in other species are often annotated as a cyclomaltodextrinase (enzyme that linearizes circular oligosaccharides), and homologs are required for growth on α-cyclodextrin in Caulobacter crescentus and Echinocola vietnamensis KMM 6221 (Price et al., 2018). These data suggest that SusA also has cyclomaltodextrinase activity, as previously proposed (Smith and Salyers, 1991).SusB is a GH97 family member that hydrolyzes multiple linkages, including α-1,4 bonds (Kitamura et al., 2008). In previous work, a ΔsusB mutant had negligible growth defects on starch, raising doubts about the role of SusB in the use of starch and related compounds (D’Elia and Salyers, 1996; Reeves et al., 1996). We found that insertions in susB had fitness defects on several forms of starch (Figure 1F), motivating us to re-examine the role of SusB in polysaccharide utilization. To confirm that we were measuring a bona fide phenotype of susB and not polar effects on downstream genes like susCD, we generated a clean deletion. The strongest fitness defects of susB insertions were during growth on α-cyclodextrin, and we found that ΔsusB indeed had a growth defect on this substrate compared to glucose (Figures 1G and 1H). We used quantitative real-time RT-PCR to test if the susB mutation disrupted the expression of susCD during growth on α-cyclodextrin. Instead, we found that expression levels of susC and susD were 2.7-fold higher in ΔsusB cells than in wild-type cells (susC: 2.7-fold, 95% confidence interval: 2.0–3.7; susD: 2.7-fold, 95% confidence interval: 2.1–3.4).SusG breaks down extracellular starch into simpler oligosaccharides (Koropatkin and Smith, 2010). Consistent with this function, in our fitness data, susG was highly important for growth on starch and amylopectin and to a lesser extent on amylose and α-cyclodextrin (Figure 1F). Although the outer membrane proteins SusE and SusF have been demonstrated to bind starch (Cameron et al., 2012), their precise function remains unclear. We found that these proteins were only important for growth on amylopectin, suggesting that SusEF preferentially bind this component of starch or they are important for stabilization of the SusG-starch complex, as previously proposed (Foley et al., 2018; Tuson et al., 2018). Taken together, our re-evaluation of the Sus based on genetics data across multiple substrates adds insight into this well-studied system.To globally associate PULs to carbon substrates, we identified instances in which at least one gene in a given PUL had a specific and important phenotype (fitness, <−1). Based on these criteria, we associated 20 of the 93 known PULs (Terrapon et al., 2018) to 32 substrates (Table S4; Figure 2). We identified specific and important phenotypes for ≥3 genes from multiple PULs on their predicted or known polysaccharides (Table S4; Figure 2), as follows: PUL5 on arabinogalactan (Cartmell et al., 2018), PUL22 on fructans (Sonnenburg et al., 2010), PUL48 on dextran (Rogers et al., 2013), PUL57 on chondroitin sulfate (CS) and hyaluronic acid (HA) (Raghavan et al., 2014), PUL68 on mannan (Cuskin et al., 2015), PUL75 on pectic glycans (Luis et al., 2018), PUL85 on heparin (Cartmell et al., 2017), and PUL86 on type I rhamnogalacturonan (Luis et al., 2018).In addition to confirming the results of previous studies, our screen provided genetic evidence for the prediction that, besides PUL5, PUL47 and PUL80 are also important for arabinogalactan utilization (Martens et al., 2011). The arabinogalactan phenotypes for genes in PUL47 and PUL80 were mild (fitness values ranged from −0.7 to −1.4), but similar phenotypes were observed for ≥3 genes in each PUL, and the genes in these clusters did not have clear phenotypes under any other conditions profiled (Figures 3A and 3B). Even though these genes did not meet the criteria for a specific phenotype, mutants in these genes had statistically significant defects, all with ∣t∣ > 4, and an estimated false discovery rate of <5% (STAR methods). These data demonstrate that efficient breakdown of certain complex polysaccharides like arabinogalactan requires multiple PULs. Additional genes (PUL65 and BT3674–BT3687) are involved in removing glucuronic acid units from gum Arabic arabinogalactan (Cartmell et al., 2018), but these genes had weak phenotypes (fitness, >−0.5) during growth on larch wood arabinogalactan, consistent with the low glucuronic acid content of larch wood arabinogalactan (Willför et al., 2002).Our analysis also linked PULs to specific monosaccharides and disaccharides. For example, the SusCD homologs of PUL15 (BT1118 and BT1119) were important on galacturonic acid and 1,4-β-D-galactobiose (Figure 3C). Consistent with Glowacki et al. (2020) results, components of PUL37 were only important for growth on D-ribose, in particular a hypothetical protein (BT2802; fitness, −4.3), two predicted ribokinases (BT2803 and BT2804), and a sugar transporter (BT2809) (Figure 3D). The SusCD homologs in this cluster were also mildly important on D-ribose (BT2805 fitness, −1.3; BT2806 fitness, −1.4). Finally, a putative glucosidase (BT3567), SusD-like protein (BT3568), and SusC-like protein (BT3569) from PUL61 were required for optimal growth on the disaccharide laminaribiose (Figure 3E). Although the PULs of B. thetaiotaomicron have typically been studied in the context of polysaccharides, our data suggest that some PULs evolved to catabolize simpler substrates. In support of this view, PUL37 and PUL61 contain the core SusCD proteins but do not have homologs of SusEF or hydrolases expected to act on complex substrates.

Functional-genetic screen links PULs to specific substrates

Fitness profiling across multiple substrates provided a broad view of the importance of each PUL component to carbon catabolism. The 48 carbon sources we assayed represent a range of substrate complexity, including 14 monosaccharides and their derivatives, 12 disaccharides, 1 trisaccharide, 2 tetrasaccharides, 3 oligosaccharides (>4 monomers), 2 sugar alcohols, and 14 complex polysaccharides. As an illustration, we re-examined the Sus system by comparing its phenotypes during growth on starch; the component polysaccharides of starch (amylopectin and amylose); the oligosaccharide α-cyclodextrin; and the simpler break-down products glucose, maltose, maltotetraose, and maltohexaose (Figure 1F). The SusCD transporter proteins were important on all saccharides of a length of ≥4 but were largely dispensable for growth on maltose and glucose, which is in agreement with previous findings that SusD importance varies depending on the size of glucose polymers (Koropatkin et al., 2008).

The three glycoside hydrolases (GHs; SusA, SusB, and SusG) were variably important across substrates. SusA was previously characterized as a neopullulanase that cleaves α-1,4 linkages in pullulan, a polymer of maltotriose subunits, and was shown to be mildly important for growth on starch (D’Elia and Salyers, 1996; Reeves et al., 1996). In our data, SusA was only important for growth on α-cyclodextrin, a cyclic oligosaccharide of 6 glucose monomers linked by α-1,4 glycosidic bonds (Figure 1F). SusA homologs in other species are often annotated as a cyclomaltodextrinase (enzyme that linearizes circular oligosaccharides), and homologs are required for growth on α-cyclodextrin in Caulobacter crescentus and Echinocola vietnamensis KMM 6221 (Price et al., 2018). These data suggest that SusA also has cyclomaltodextrinase activity, as previously proposed (Smith and Salyers, 1991).

SusB is a GH97 family member that hydrolyzes multiple linkages, including α-1,4 bonds (Kitamura et al., 2008). In previous work, a ΔsusB mutant had negligible growth defects on starch, raising doubts about the role of SusB in the use of starch and related compounds (D’Elia and Salyers, 1996; Reeves et al., 1996). We found that insertions in susB had fitness defects on several forms of starch (Figure 1F), motivating us to re-examine the role of SusB in polysaccharide utilization. To confirm that we were measuring a bona fide phenotype of susB and not polar effects on downstream genes like susCD, we generated a clean deletion. The strongest fitness defects of susB insertions were during growth on α-cyclodextrin, and we found that ΔsusB indeed had a growth defect on this substrate compared to glucose (Figures 1G and 1H). We used quantitative real-time RT-PCR to test if the susB mutation disrupted the expression of susCD during growth on α-cyclodextrin. Instead, we found that expression levels of susC and susD were 2.7-fold higher in ΔsusB cells than in wild-type cells (susC: 2.7-fold, 95% confidence interval: 2.0–3.7; susD: 2.7-fold, 95% confidence interval: 2.1–3.4).

SusG breaks down extracellular starch into simpler oligosaccharides (Koropatkin and Smith, 2010). Consistent with this function, in our fitness data, susG was highly important for growth on starch and amylopectin and to a lesser extent on amylose and α-cyclodextrin (Figure 1F). Although the outer membrane proteins SusE and SusF have been demonstrated to bind starch (Cameron et al., 2012), their precise function remains unclear. We found that these proteins were only important for growth on amylopectin, suggesting that SusEF preferentially bind this component of starch or they are important for stabilization of the SusG-starch complex, as previously proposed (Foley et al., 2018; Tuson et al., 2018). Taken together, our re-evaluation of the Sus based on genetics data across multiple substrates adds insight into this well-studied system.

To globally associate PULs to carbon substrates, we identified instances in which at least one gene in a given PUL had a specific and important phenotype (fitness, <−1). Based on these criteria, we associated 20 of the 93 known PULs (Terrapon et al., 2018) to 32 substrates (Table S4; Figure 2). We identified specific and important phenotypes for ≥3 genes from multiple PULs on their predicted or known polysaccharides (Table S4; Figure 2), as follows: PUL5 on arabinogalactan (Cartmell et al., 2018), PUL22 on fructans (Sonnenburg et al., 2010), PUL48 on dextran (Rogers et al., 2013), PUL57 on chondroitin sulfate (CS) and hyaluronic acid (HA) (Raghavan et al., 2014), PUL68 on mannan (Cuskin et al., 2015), PUL75 on pectic glycans (Luis et al., 2018), PUL85 on heparin (Cartmell et al., 2017), and PUL86 on type I rhamnogalacturonan (Luis et al., 2018).

In addition to confirming the results of previous studies, our screen provided genetic evidence for the prediction that, besides PUL5, PUL47 and PUL80 are also important for arabinogalactan utilization (Martens et al., 2011). The arabinogalactan phenotypes for genes in PUL47 and PUL80 were mild (fitness values ranged from −0.7 to −1.4), but similar phenotypes were observed for ≥3 genes in each PUL, and the genes in these clusters did not have clear phenotypes under any other conditions profiled (Figures 3A and 3B). Even though these genes did not meet the criteria for a specific phenotype, mutants in these genes had statistically significant defects, all with ∣t∣ > 4, and an estimated false discovery rate of <5% (STAR methods). These data demonstrate that efficient breakdown of certain complex polysaccharides like arabinogalactan requires multiple PULs. Additional genes (PUL65 and BT3674–BT3687) are involved in removing glucuronic acid units from gum Arabic arabinogalactan (Cartmell et al., 2018), but these genes had weak phenotypes (fitness, >−0.5) during growth on larch wood arabinogalactan, consistent with the low glucuronic acid content of larch wood arabinogalactan (Willför et al., 2002).

Our analysis also linked PULs to specific monosaccharides and disaccharides. For example, the SusCD homologs of PUL15 (BT1118 and BT1119) were important on galacturonic acid and 1,4-β-D-galactobiose (Figure 3C). Consistent with Glowacki et al. (2020) results, components of PUL37 were only important for growth on D-ribose, in particular a hypothetical protein (BT2802; fitness, −4.3), two predicted ribokinases (BT2803 and BT2804), and a sugar transporter (BT2809) (Figure 3D). The SusCD homologs in this cluster were also mildly important on D-ribose (BT2805 fitness, −1.3; BT2806 fitness, −1.4). Finally, a putative glucosidase (BT3567), SusD-like protein (BT3568), and SusC-like protein (BT3569) from PUL61 were required for optimal growth on the disaccharide laminaribiose (Figure 3E). Although the PULs of B. thetaiotaomicron have typically been studied in the context of polysaccharides, our data suggest that some PULs evolved to catabolize simpler substrates. In support of this view, PUL37 and PUL61 contain the core SusCD proteins but do not have homologs of SusEF or hydrolases expected to act on complex substrates.

Identification of polysaccharide degradation genesAlthough the degradation pathways for some polysaccharides are well described in B. thetaiotaomicron, prior work found that genes important in glycan utilization can be located outside the corresponding PUL (Benjdia et al., 2011; Raghavan et al., 2014). By examining genes with specific and important phenotypes, we identified three genes for polysaccharide utilization. The first example is in metabolism of type I rhamnogalacturonan (RG-I), a polysaccharide containing repeats of a disaccharide of galacturonic acid and rhamnose. A recent study proposed the involvement of five GH28-family glycosyl hydrolases (BT4123, BT4146, BT4149, BT4153, and BT4155) in breaking down RG-I (Luis et al., 2018). In our fitness data, only BT4149 was important for utilizing RG-I, and this phenotype was mild (fitness, −1.3). Rather, we found that another GH28-family member, BT4187, was more important for growth on RG-I (fitness, −2.2) (Figure 3F). In support of the importance of BT4187 in RG-I degradation, the operon containing BT4187 is predicted to be in the same regulon as PUL75 (Ravcheev et al., 2013), which is both important for fitness (Figure 2) and induced during growth on RG-I (Martens et al., 2011). Although it is possible that the other hydrolases are functionally redundant, our results demonstrate that the loss of BT4187 alone results in a strong fitness defect during growth on RG-I.The next two examples associate genes of previously unknown function with utilization of host-derived glycosaminoglycans (GAGs), which are structural components of animal tissues and are composed of repeating disaccharide units of a hexuronic acid linked to an amino sugar. B. thetaiotaomicron is capable of using GAGs as nutrients, including CS, HA, and heparin (Cartmell et al., 2017; Ndeh and Gilbert, 2018; Raghavan and Groisman, 2015). CS and HA differ by their repeating disaccharide; CS contains sulfated N-acetylgalactosamine and glucuronic acid, and HA contains N-acetylglucosamine and glucuronic acid. Previous studies proposed a model for CS and HA catabolism by B. thetaiotaomicron involving PUL57 and the CS lyase BT4410 (Martens et al., 2008; Raghavan et al., 2014). In our data, components of PUL57 were indeed required for optimal growth on CS and HA (Figure 3G), whereas other components of PUL57 were important on one substrate only. BT3349 (a sulfatase) was only important on CS, as expected given that only CS contains sulfated sugars. The polysaccharide lyase family PL29 enzyme encoded by BT3328 was only important for fitness on HA, despite the activity of this enzyme against both HA and CS in vitro (Ndeh et al., 2018).The first GAG utilization gene we identified was BT4411, which was important for growth on HA but not on CS (Figure 3G). BT4411 is predicted to be in the same operon as the CS lyase BT4410, which was also more important for fitness on HA than on CS. BT4411 contains a domain of unknown function (DUF4627) that has significant similarity to CBM_4_9 (carbohydrate binding domain PF02018) (El-Gebali et al., 2019). The BT4410–BT4411 operon is conserved in several other Bacteroides species, and both proteins have putative signal peptides (Yu et al., 2010), suggesting that BT4410 and BT4411 function together in HA catabolism.Finally, we found that BT3233 was important for growth on the following 3 GAGs: CS (mean fitness, −1.6), HA (mean fitness, −0.7), and heparin (mean fitness, −2.2) (Figure 3G). BT3233 is a member of the uncharacterized protein family DUF4861 (El-Gebali et al., 2019) and belongs to a superfamily of putative carbohydrate binding domains (galactose mutarotase-like domain, InterPro IPR011013) (Mitchell et al., 2019). Homologs of BT3233 are mostly confined to Bacteroides species, including a number of human commensals. Using an archived transposon mutant, we confirmed that BT3233 is important for growth on CS (Figure 3H,I). Although its precise function is unknown, our data suggest that BT3233 plays an important role in GAG catabolism.

Identification of polysaccharide degradation genes

Although the degradation pathways for some polysaccharides are well described in B. thetaiotaomicron, prior work found that genes important in glycan utilization can be located outside the corresponding PUL (Benjdia et al., 2011; Raghavan et al., 2014). By examining genes with specific and important phenotypes, we identified three genes for polysaccharide utilization. The first example is in metabolism of type I rhamnogalacturonan (RG-I), a polysaccharide containing repeats of a disaccharide of galacturonic acid and rhamnose. A recent study proposed the involvement of five GH28-family glycosyl hydrolases (BT4123, BT4146, BT4149, BT4153, and BT4155) in breaking down RG-I (Luis et al., 2018). In our fitness data, only BT4149 was important for utilizing RG-I, and this phenotype was mild (fitness, −1.3). Rather, we found that another GH28-family member, BT4187, was more important for growth on RG-I (fitness, −2.2) (Figure 3F). In support of the importance of BT4187 in RG-I degradation, the operon containing BT4187 is predicted to be in the same regulon as PUL75 (Ravcheev et al., 2013), which is both important for fitness (Figure 2) and induced during growth on RG-I (Martens et al., 2011). Although it is possible that the other hydrolases are functionally redundant, our results demonstrate that the loss of BT4187 alone results in a strong fitness defect during growth on RG-I.

The next two examples associate genes of previously unknown function with utilization of host-derived glycosaminoglycans (GAGs), which are structural components of animal tissues and are composed of repeating disaccharide units of a hexuronic acid linked to an amino sugar. B. thetaiotaomicron is capable of using GAGs as nutrients, including CS, HA, and heparin (Cartmell et al., 2017; Ndeh and Gilbert, 2018; Raghavan and Groisman, 2015). CS and HA differ by their repeating disaccharide; CS contains sulfated N-acetylgalactosamine and glucuronic acid, and HA contains N-acetylglucosamine and glucuronic acid. Previous studies proposed a model for CS and HA catabolism by B. thetaiotaomicron involving PUL57 and the CS lyase BT4410 (Martens et al., 2008; Raghavan et al., 2014). In our data, components of PUL57 were indeed required for optimal growth on CS and HA (Figure 3G), whereas other components of PUL57 were important on one substrate only. BT3349 (a sulfatase) was only important on CS, as expected given that only CS contains sulfated sugars. The polysaccharide lyase family PL29 enzyme encoded by BT3328 was only important for fitness on HA, despite the activity of this enzyme against both HA and CS in vitro (Ndeh et al., 2018).

The first GAG utilization gene we identified was BT4411, which was important for growth on HA but not on CS (Figure 3G). BT4411 is predicted to be in the same operon as the CS lyase BT4410, which was also more important for fitness on HA than on CS. BT4411 contains a domain of unknown function (DUF4627) that has significant similarity to CBM_4_9 (carbohydrate binding domain PF02018) (El-Gebali et al., 2019). The BT4410–BT4411 operon is conserved in several other Bacteroides species, and both proteins have putative signal peptides (Yu et al., 2010), suggesting that BT4410 and BT4411 function together in HA catabolism.

Finally, we found that BT3233 was important for growth on the following 3 GAGs: CS (mean fitness, −1.6), HA (mean fitness, −0.7), and heparin (mean fitness, −2.2) (Figure 3G). BT3233 is a member of the uncharacterized protein family DUF4861 (El-Gebali et al., 2019) and belongs to a superfamily of putative carbohydrate binding domains (galactose mutarotase-like domain, InterPro IPR011013) (Mitchell et al., 2019). Homologs of BT3233 are mostly confined to Bacteroides species, including a number of human commensals. Using an archived transposon mutant, we confirmed that BT3233 is important for growth on CS (Figure 3H,I). Although its precise function is unknown, our data suggest that BT3233 plays an important role in GAG catabolism.

Disaccharide catabolism through a putative 3-keto intermediateOur fitness assays revealed that components of a gene cluster (BT2157, BT2159, and BT2160) were important for catabolizing the disaccharides trehalose (glucose-α-1,1-glucose), leucrose (glucose-α-1,5-fructose), palatinose (glucose-α-1,6-fructose), and maltitol (glucose-α-1,4-glucitol) and the trisaccharide raffinose (galactose-α-1,6-glucose-β-1,2-fructose) (Figure 4A). BT2160 encodes a SusR-family transcription factor that likely regulates the cluster (Ravcheev et al., 2013). BT2156 did not have a significant phenotype under any assayed conditions. B. thetaiotaomicron contains a second gene cluster (BT4446–BT4449) that is highly similar to BT2156–BT2159 (Figure S1A), except that BT4446–BT4449 does not contain a SusR-family regulator nor did it have a significant phenotype in our data. BT2158 is 98% identical at the nucleotide level to BT4448; hence, we were not able to unambiguously map transposon insertions within these genes by using our standard parameters. To determine if BT2158 was important for fitness, we re-ran our transposon insertion mapping with less stringency by only requiring one nucleotide difference to assign insertions to BT2158 and BT4448, which successfully generated fitness data for both genes. Although BT4448 did not have strong phenotypes, BT2158 was important for growth on trehalose, leucrose, palatinose, maltitol, and raffinose (fitness, ≥ −1.3 in all experiments on these substrates). Furthermore, BT2158 had the highest co-fitness with BT2160 (r = 0.67). To confirm these results, we generated individual deletions of BT2158 and BT4448 and assayed growth of these mutants on glucose or trehalose. As expected, ΔBT4448 did not exhibit a growth defect on either substrate (Figures S1E and S1F), whereas ΔBT2158 did not grow on trehalose (Figure S1F). These results demonstrate that BT2157–BT2160 is important for fitness on similar substrates and likely act in the same pathway. A recent genetic and biochemical study found that components of BT2156–BT2160 are also capable of transforming glucosinolates (which are thioglycosides, so there is some chemical similarity to glycosides) to produce isothiocyanates (Liou et al., 2020).To explore this operon, we examined the protein sequences. BT2156 encodes a predicted lolE-like epimerase, BT2157 encodes a DUF1080-containing protein, and BT2158 and BT2159 encode MviM-like dehydrogenases. DUF1080 has structural similarity to endo-1,3-1,4-β-glucanases (El-Gebali et al., 2019) and is likely a family of hydrolases. Interestingly, homologs of BT2156–BT2159 are chromosomally clustered and are important for catabolizing disaccharides and other glycosides in C. crescentus, Pedobacter sp. GW460-11-11-14-LB5, and E. vietnamensis KMM 6221 (Figures S1 and S2) (Price et al., 2018). Thus, we considered that B. thetaiotaomicron and these other species might catabolize glycosides through similar mechanisms. In C. crescentus, homologs of BT2156–BT2159 are in a genome cluster with the glucoside 3-dehydrogenase LacABC (Figure S1B), and these homologs share similar phenotypes on glycosides as LacABC (Figure S2A). LacABC is the first step in lactose catabolism by a 3-keto-lactose intermediate (Arellano et al., 2010). Similarly, sucrose degradation in Agrobacterium tumefaciens proceeds through a 3-keto-sucrose intermediate (Bernaerts and De Ley, 1960; Schuerman et al., 1997), and in Sinorhizobium meliloti, degradation of various disaccharides occurs through 3-keto intermediates (Ampomah et al., 2013). A. tumefaciens and S. meliloti use the dehydrogenase ThuAB (unrelated to LacABC) to form 3-keto-disaccharides (Bernaerts and De Ley, 1960; Schuerman et al., 1997). The 3-keto-disaccharides are thought to be hydrolyzed to a hexose and a 3-keto-hexose (Hayano and Fukui, 1970), followed by reduction of the 3-keto-hexose (Hayano et al., 1973), but the genes involved are not known.We hypothesized that B. thetaiotaomicron degrades glycosides by 3-keto intermediates through activity of BT2157–BT2160, even though B. thetaiotaomicron does not have LacABC or ThuAB. We focused on the utilization of trehalose; BT2157–BT2160 had relatively strong phenotypes on trehalose, and homologs of some of these proteins are important for trehalose utilization in Pedobacter sp. GW460-11-11-14-LB5 and E. vietnamensis KMM 6221 (Figures S2B and S2C). First, we investigated whether expression of this operon was induced on trehalose by using proteomics of stationary-phase cultures. We observed a ≥9-fold increase in abundance for three of the target proteins (BT2157–BT2159) during growth on trehalose compared to glucose, suggesting that this system is regulated by trehalose availability (Figure 4B). Next, we generated a deletion of BT2160 and assayed its ability to grow on trehalose. Similar to ΔBT2158, BT2160 was required for growth on trehalose (Figure 4C). Conversely, neither gene was required for growth on glucose (Figure 4D). Lastly, to provide evidence that the degradation pathway proceeds by a 3-keto-trehalose intermediate, we determined whether the DUF1080 hydrolase encoded by BT2157 was specific to 3-keto-disaccharides. We purified BT2157 and used Agrobacterium rhizogenes to produce a supernatant containing 3-keto-trehalose. BT2157 hydrolyzed >70% of the 3-keto-trehalose in the supernatant within 1 min (Figure 4E). In contrast, the concentration of trehalose in the supernatant only decreased by ~14% after 10 min of incubation with BT2157. BT2157 appears to hydrolyze 3-keto-trehalose in vitro in the absence of other products of the BT2156–BT2160 operon and more rapidly than the glucosinolates glucoraphanin or glucobrassicin (Liou et al., 2020). Our data demonstrate that BT2157–BT2160 is the primary route of trehalose catabolism in B. thetaiotaomicron, that the expression of these proteins is induced by trehalose, and that BT2157 preferentially hydrolyzes 3-keto-trehalose, thereby supporting a model that trehalose degradation proceeds through a 3-keto-glycoside intermediate.

Disaccharide catabolism through a putative 3-keto intermediate

Our fitness assays revealed that components of a gene cluster (BT2157, BT2159, and BT2160) were important for catabolizing the disaccharides trehalose (glucose-α-1,1-glucose), leucrose (glucose-α-1,5-fructose), palatinose (glucose-α-1,6-fructose), and maltitol (glucose-α-1,4-glucitol) and the trisaccharide raffinose (galactose-α-1,6-glucose-β-1,2-fructose) (Figure 4A). BT2160 encodes a SusR-family transcription factor that likely regulates the cluster (Ravcheev et al., 2013). BT2156 did not have a significant phenotype under any assayed conditions. B. thetaiotaomicron contains a second gene cluster (BT4446–BT4449) that is highly similar to BT2156–BT2159 (Figure S1A), except that BT4446–BT4449 does not contain a SusR-family regulator nor did it have a significant phenotype in our data. BT2158 is 98% identical at the nucleotide level to BT4448; hence, we were not able to unambiguously map transposon insertions within these genes by using our standard parameters. To determine if BT2158 was important for fitness, we re-ran our transposon insertion mapping with less stringency by only requiring one nucleotide difference to assign insertions to BT2158 and BT4448, which successfully generated fitness data for both genes. Although BT4448 did not have strong phenotypes, BT2158 was important for growth on trehalose, leucrose, palatinose, maltitol, and raffinose (fitness, ≥ −1.3 in all experiments on these substrates). Furthermore, BT2158 had the highest co-fitness with BT2160 (r = 0.67). To confirm these results, we generated individual deletions of BT2158 and BT4448 and assayed growth of these mutants on glucose or trehalose. As expected, ΔBT4448 did not exhibit a growth defect on either substrate (Figures S1E and S1F), whereas ΔBT2158 did not grow on trehalose (Figure S1F). These results demonstrate that BT2157–BT2160 is important for fitness on similar substrates and likely act in the same pathway. A recent genetic and biochemical study found that components of BT2156–BT2160 are also capable of transforming glucosinolates (which are thioglycosides, so there is some chemical similarity to glycosides) to produce isothiocyanates (Liou et al., 2020).

To explore this operon, we examined the protein sequences. BT2156 encodes a predicted lolE-like epimerase, BT2157 encodes a DUF1080-containing protein, and BT2158 and BT2159 encode MviM-like dehydrogenases. DUF1080 has structural similarity to endo-1,3-1,4-β-glucanases (El-Gebali et al., 2019) and is likely a family of hydrolases. Interestingly, homologs of BT2156–BT2159 are chromosomally clustered and are important for catabolizing disaccharides and other glycosides in C. crescentus, Pedobacter sp. GW460-11-11-14-LB5, and E. vietnamensis KMM 6221 (Figures S1 and S2) (Price et al., 2018). Thus, we considered that B. thetaiotaomicron and these other species might catabolize glycosides through similar mechanisms. In C. crescentus, homologs of BT2156–BT2159 are in a genome cluster with the glucoside 3-dehydrogenase LacABC (Figure S1B), and these homologs share similar phenotypes on glycosides as LacABC (Figure S2A). LacABC is the first step in lactose catabolism by a 3-keto-lactose intermediate (Arellano et al., 2010). Similarly, sucrose degradation in Agrobacterium tumefaciens proceeds through a 3-keto-sucrose intermediate (Bernaerts and De Ley, 1960; Schuerman et al., 1997), and in Sinorhizobium meliloti, degradation of various disaccharides occurs through 3-keto intermediates (Ampomah et al., 2013). A. tumefaciens and S. meliloti use the dehydrogenase ThuAB (unrelated to LacABC) to form 3-keto-disaccharides (Bernaerts and De Ley, 1960; Schuerman et al., 1997). The 3-keto-disaccharides are thought to be hydrolyzed to a hexose and a 3-keto-hexose (Hayano and Fukui, 1970), followed by reduction of the 3-keto-hexose (Hayano et al., 1973), but the genes involved are not known.

We hypothesized that B. thetaiotaomicron degrades glycosides by 3-keto intermediates through activity of BT2157–BT2160, even though B. thetaiotaomicron does not have LacABC or ThuAB. We focused on the utilization of trehalose; BT2157–BT2160 had relatively strong phenotypes on trehalose, and homologs of some of these proteins are important for trehalose utilization in Pedobacter sp. GW460-11-11-14-LB5 and E. vietnamensis KMM 6221 (Figures S2B and S2C). First, we investigated whether expression of this operon was induced on trehalose by using proteomics of stationary-phase cultures. We observed a ≥9-fold increase in abundance for three of the target proteins (BT2157–BT2159) during growth on trehalose compared to glucose, suggesting that this system is regulated by trehalose availability (Figure 4B). Next, we generated a deletion of BT2160 and assayed its ability to grow on trehalose. Similar to ΔBT2158, BT2160 was required for growth on trehalose (Figure 4C). Conversely, neither gene was required for growth on glucose (Figure 4D). Lastly, to provide evidence that the degradation pathway proceeds by a 3-keto-trehalose intermediate, we determined whether the DUF1080 hydrolase encoded by BT2157 was specific to 3-keto-disaccharides. We purified BT2157 and used Agrobacterium rhizogenes to produce a supernatant containing 3-keto-trehalose. BT2157 hydrolyzed >70% of the 3-keto-trehalose in the supernatant within 1 min (Figure 4E). In contrast, the concentration of trehalose in the supernatant only decreased by ~14% after 10 min of incubation with BT2157. BT2157 appears to hydrolyze 3-keto-trehalose in vitro in the absence of other products of the BT2156–BT2160 operon and more rapidly than the glucosinolates glucoraphanin or glucobrassicin (Liou et al., 2020). Our data demonstrate that BT2157–BT2160 is the primary route of trehalose catabolism in B. thetaiotaomicron, that the expression of these proteins is induced by trehalose, and that BT2157 preferentially hydrolyzes 3-keto-trehalose, thereby supporting a model that trehalose degradation proceeds through a 3-keto-glycoside intermediate.

Identification of an efflux system for bile saltsThe human gut involves exposure to growth-promoting complex polysaccharides and also to bile salts, which can inhibit bacterial growth. Consequently, commensal bacteria have evolved strategies to cope with bile salts, including removal by efflux pumps, deconjugation by bile salt hydrolases (BSHs), and oxidation and epimerization of hydroxyl groups by hydroxysteroid dehydrogenases (HSDHs) (Begley et al., 2005). BSH and HSDH homologs have been identified in B. thetaiotaomicron (Ridlon et al., 2006), and BT2086 has BSH activity in vitro and in vivo (Yao et al., 2018).To identify B. thetaiotaomicron genes involved in bile salt tolerance, we examined fitness in the presence of inhibitory concentrations of the primary bile acid chenodeoxycholic acid, the secondary bile acids lithocholic acid or deoxycholic acid, or cholic-acid-derived bile salts (a mixture of sodium cholate and sodium deoxycholate, which we refer to generically as “bile salts”). From these data, we identified a four-gene operon, BT2792–BT2795, important for fitness under deoxycholic acid, chenodeoxycholic acid, and bile salt stress (Figure 5A). These genes were not important when stressed with lithocholic acid, at least under the concentrations assayed. Based on homology (Overbeek et al., 2005), these proteins form a tripartite multidrug efflux system. Interestingly, these genes did not have a significant phenotype under any other conditions profiled (Figure 5A), including dozens of diverse antibiotics, suggesting that this efflux system is specific to bile acids and bile salts. Closely related efflux systems are present in other Bacteroides species (Dehal et al., 2010), although the importance of those genes for bile salt tolerance remains to be determined.To confirm the involvement of this operon in bile salt tolerance, we assayed the growth of archived transposon mutants. BT2793, BT2794, and BT2795 mutants all had longer lag phases and reduced growth rates relative to controls when stressed with 0.5 mg/mL bile salts (Figures 5B and 5C). To determine if this efflux system is induced by the stressor, we performed targeted proteomic analysis by using wild-type cells grown in the presence of various concentrations of bile salts. BT2792, BT2794, and BT2795 were all induced in the presence of bile salts, and the amount of induction increased with higher bile salt concentrations (Figures 5D and 5E), consistent with previous observations of bile-salt-mediated upregulation of efflux pumps in other human gut commensals (Lin et al., 2005; Prouty et al., 2004; Pumbwe et al., 2007; Ruiz et al., 2013). Taken together, our data show that the efflux system encoded by BT2792–BT2795 is induced by and important for resistance to bile salts.

Identification of an efflux system for bile salts

The human gut involves exposure to growth-promoting complex polysaccharides and also to bile salts, which can inhibit bacterial growth. Consequently, commensal bacteria have evolved strategies to cope with bile salts, including removal by efflux pumps, deconjugation by bile salt hydrolases (BSHs), and oxidation and epimerization of hydroxyl groups by hydroxysteroid dehydrogenases (HSDHs) (Begley et al., 2005). BSH and HSDH homologs have been identified in B. thetaiotaomicron (Ridlon et al., 2006), and BT2086 has BSH activity in vitro and in vivo (Yao et al., 2018).

To identify B. thetaiotaomicron genes involved in bile salt tolerance, we examined fitness in the presence of inhibitory concentrations of the primary bile acid chenodeoxycholic acid, the secondary bile acids lithocholic acid or deoxycholic acid, or cholic-acid-derived bile salts (a mixture of sodium cholate and sodium deoxycholate, which we refer to generically as “bile salts”). From these data, we identified a four-gene operon, BT2792–BT2795, important for fitness under deoxycholic acid, chenodeoxycholic acid, and bile salt stress (Figure 5A). These genes were not important when stressed with lithocholic acid, at least under the concentrations assayed. Based on homology (Overbeek et al., 2005), these proteins form a tripartite multidrug efflux system. Interestingly, these genes did not have a significant phenotype under any other conditions profiled (Figure 5A), including dozens of diverse antibiotics, suggesting that this efflux system is specific to bile acids and bile salts. Closely related efflux systems are present in other Bacteroides species (Dehal et al., 2010), although the importance of those genes for bile salt tolerance remains to be determined.

To confirm the involvement of this operon in bile salt tolerance, we assayed the growth of archived transposon mutants. BT2793, BT2794, and BT2795 mutants all had longer lag phases and reduced growth rates relative to controls when stressed with 0.5 mg/mL bile salts (Figures 5B and 5C). To determine if this efflux system is induced by the stressor, we performed targeted proteomic analysis by using wild-type cells grown in the presence of various concentrations of bile salts. BT2792, BT2794, and BT2795 were all induced in the presence of bile salts, and the amount of induction increased with higher bile salt concentrations (Figures 5D and 5E), consistent with previous observations of bile-salt-mediated upregulation of efflux pumps in other human gut commensals (Lin et al., 2005; Prouty et al., 2004; Pumbwe et al., 2007; Ruiz et al., 2013). Taken together, our data show that the efflux system encoded by BT2792–BT2795 is induced by and important for resistance to bile salts.

Association of antibiotics with transport systemsThe use of antimicrobial compounds to treat infection means that gut commensals are regularly exposed to these inhibitors; thus, we examined fitness with different classes of inhibitors including antibiotics and biocides. Given the prevalence of efflux as a mechanism of antibiotic resistance in bacteria (Webber and Piddock, 2003), we focused on transport-related proteins with strong phenotypes. For example, BexA (BT0588) belongs to the multidrug and toxic compound extrusion (MATE) family and has been shown to be responsible for efflux of the fluoroquinolone antibiotics norfloxacin and ciprofloxacin in B. thetaiotaomicron (Miyamae et al., 2001). In our dataset, BT0588 mutants exhibited a mild fitness defect with ciprofloxacin (minimum fitness, −1.6). Insertions in BT0588 resulted in a more pronounced phenotype with another fluoroquinolone (lomefloxacin) or with paraquat (Figure 6A). In another example, we found that the RND transporter system encoded by BT3337–BT3339 was important for fusidic acid tolerance (Figure 6B) and to a lesser extent for tolerating cefoxitin and the antipsychotic drug chlorpromazine.Lastly, we identified a two-gene operon, BT1439–BT1440, with high positive fitness when grown in the presence of vancomycin (Figure 6C), a glycopeptide antibiotic that inhibits peptidoglycan biosynthesis in the periplasm. Although vancomycin is usually thought to target Gram-positive bacteria, it will inhibit Gram-negative bacteria if it passes through the outer membrane (Antonoplis et al., 2019). Vancomycin is commonly used for treating Clostridium difficile infections (CDIs) and is often taken orally. Fecal levels of vancomycin in CDI patients can exceed 2 mg/mL during treatment (Gonzales et al., 2010), which is above the concentrations used in our screen (0.009–0.019 mg/mL). Thus, our data suggest that vancomycin use in humans can impact B. thetaiotaomicron in the gut (Basolo et al., 2020). BT1439 is a SusD homolog and BT1440 is a SusC homolog, and these genes had high co-fitness (r = 0.75), indicating that they likely act together. Although SusCD complexes are best known for translocating carbohydrate substrates, no growth defect was found on any of the carbohydrates assayed. To validate our data, we constructed a BT1439 deletion mutant and found that it was more resistant to vancomycin than the parent (Figures 6D, 6E, and S3). Interestingly, the vancomycin phenotype of BT1439–BT1440 was only observed in BHIS medium and not in VB medium (Figure 6C). This medium-specific phenotype could be due to altered regulation or to vancomycin binding to another compound in the growth medium (such as the peptides in BHIS).

Association of antibiotics with transport systems

The use of antimicrobial compounds to treat infection means that gut commensals are regularly exposed to these inhibitors; thus, we examined fitness with different classes of inhibitors including antibiotics and biocides. Given the prevalence of efflux as a mechanism of antibiotic resistance in bacteria (Webber and Piddock, 2003), we focused on transport-related proteins with strong phenotypes. For example, BexA (BT0588) belongs to the multidrug and toxic compound extrusion (MATE) family and has been shown to be responsible for efflux of the fluoroquinolone antibiotics norfloxacin and ciprofloxacin in B. thetaiotaomicron (Miyamae et al., 2001). In our dataset, BT0588 mutants exhibited a mild fitness defect with ciprofloxacin (minimum fitness, −1.6). Insertions in BT0588 resulted in a more pronounced phenotype with another fluoroquinolone (lomefloxacin) or with paraquat (Figure 6A). In another example, we found that the RND transporter system encoded by BT3337–BT3339 was important for fusidic acid tolerance (Figure 6B) and to a lesser extent for tolerating cefoxitin and the antipsychotic drug chlorpromazine.

Lastly, we identified a two-gene operon, BT1439–BT1440, with high positive fitness when grown in the presence of vancomycin (Figure 6C), a glycopeptide antibiotic that inhibits peptidoglycan biosynthesis in the periplasm. Although vancomycin is usually thought to target Gram-positive bacteria, it will inhibit Gram-negative bacteria if it passes through the outer membrane (Antonoplis et al., 2019). Vancomycin is commonly used for treating Clostridium difficile infections (CDIs) and is often taken orally. Fecal levels of vancomycin in CDI patients can exceed 2 mg/mL during treatment (Gonzales et al., 2010), which is above the concentrations used in our screen (0.009–0.019 mg/mL). Thus, our data suggest that vancomycin use in humans can impact B. thetaiotaomicron in the gut (Basolo et al., 2020). BT1439 is a SusD homolog and BT1440 is a SusC homolog, and these genes had high co-fitness (r = 0.75), indicating that they likely act together. Although SusCD complexes are best known for translocating carbohydrate substrates, no growth defect was found on any of the carbohydrates assayed. To validate our data, we constructed a BT1439 deletion mutant and found that it was more resistant to vancomycin than the parent (Figures 6D, 6E, and S3). Interestingly, the vancomycin phenotype of BT1439–BT1440 was only observed in BHIS medium and not in VB medium (Figure 6C). This medium-specific phenotype could be due to altered regulation or to vancomycin binding to another compound in the growth medium (such as the peptides in BHIS).

B. thetaiotaomicron uses alternative biosynthetic enzymes to adapt to ammonium availability in vivoWe were interested in whether data from our in vitro chemical-genetic screen could illuminate fitness phenotypes in vivo. To generate in vivo fitness data, we monocolonized germ-free mice with our barcoded library; collected fecal samples at days 2, 4, and 7 post-gavage; and quantified gene fitness using BarSeq. We first compared our in vivo data to those of a previous study that colonized mice with a set of Bacteroides species including B. thetaiotaomicron VPI-5482, in both a low-fat, high-plant polysaccharide (LF/HPP) and a high-fat, high-sugar (HF/HS) diet (Wu et al., 2015). Our fitness data were more correlated with the LF/HPP (r = 0.52, 95% confidence interval: 0.46–0.58) (Figure 7A) than the HF/HS diet (r = 0.33, 95% confidence interval: 0.25–0.40), as expected based on the mouse diet in our in vivo experiment (STAR methods). This correlation in part reflects the nutrients available in vivo to B. thetaiotaomicron, as there was significant overlap in genes related to amino acid biosynthesis (odds ratio = 28, 95% confidence interval: 9–92) and carbohydrate-active enzymes (CAZymes) (odds ratio = 40, 95% confidence interval: 15–102) (Figure 7B). Thus, our data are consistent with the previous study of B. thetaiotaomicron fitness in vivo in the context of a high-plant-polysaccharide diet.Although insertions in amino acid biosynthesis genes had immediate deleterious effects on growth in our dataset, the fitness defects of CAZymes tended to be smaller at first and increased in magnitude over time (Figure 7C, top), likely reflecting the foraging of B. thetaiotaomicron on a diverse set of carbohydrates in the gut (Sonnenburg et al., 2005). Nonetheless, we identified significant growth defects in vivo for genes in multiple PULs, including the arabinan-activated PUL7 (BT0350, BT0351, and BT0367), and predicted PUL27 (BT2262–BT2264) (Figure 7C, bottom); PUL27 is constitutively expressed in vitro (Glenwright et al., 2017) but has no clearly defined substrates and was not important for growth under any conditions from our in vitro screen. The in vivo phenotypes of PUL27 suggest that it functions in the uptake of a diet- or host-derived nutrient that has yet to be identified.The biosynthetic pathways for arginine and lysine were critical for growth in vivo, as previously observed (Goodman et al., 2009; Wu et al., 2015). Our in vitro data suggested that for both pathways, alternative enzymes are used depending on ammonium availability. The first example is synthesis of meso-diaminopimelate (mDAP), a precursor to lysine and peptidoglycan. Two alternative pathways for producing mDAP from 2,3,4,5-tetrahydropicolinate are conserved among the Bacteroides (Figure 7D). The dehydrogenase branch, catalyzed in a single step by Ddh (BT1979), uses free ammonium as a nitrogen source. The transaminase branch, catalyzed in two steps by DapL (BT0547) and DapF (BT0548), uses glutamate as a nitrogen source. The functions of DapL and Ddh have been verified biochemically using enzymes from B. fragilis (Hudson et al., 2008, 2011), which have 90% and 95% amino acid identity to the B. thetaiotaomicron enzymes, respectively. In Corynebacterium glutamicum, the dehydrogenase branch preferentially operates when ammonium is abundant; flux through the dehydrogenase branch is correlated with ammonium levels (Sonntag et al., 1993), and mutants in the transaminase branch require high levels of ammonium (Wehrmann et al., 1998). In our B. thetaiotaomicron library, ddh was important for fitness during growth in minimal media with ammonium but had no phenotype when ammonium was replaced with amino acids or other organic nitrogen sources (Figure 7E). Instead, insertions in dapLF were deleterious under these low ammonium conditions (Figure 7E), suggesting that relative flux through the two branches depends on availability of free ammonium in B. thetaiotaomicron as well (Figure 7D).The second example of use of alternative enzymes is carbamoyl phosphate synthase (CPS), which synthesizes a nitrogen-containing precursor (carbamoyl phosphate) to arginine and pyrimidine nucleotides. In eubacteria and eukaryotes, CPS is composed of two subunits; the glutaminase subunit (CarA) liberates ammonium from glutamine and funnels it to the synthetase subunit (CarB), where the ammonium combines with other reactants to form carbamoyl phosphate (Figure 7D; Anderson and Meister, 1965). To date, all eubacterial CPSs have been found to be glutamine dependent. However, eubacterial CarB functions as an ammonium-dependent CPS when purified alone (Trotta et al., 1971), and mutations in the glutaminase subunit of the mitochondrial CPS that participate in the urea cycle make the enzyme ammonium dependent (Saeed-Kothe and Powers-Lee, 2003). B. thetaiotaomicron carries a two-gene locus, BT0556–BT0557, encoding a full CPS (CarAB) as well as a lone synthetase subunit BT3866 (CarB2) with no clear glutaminase partner (Figure 7D). Unexpectedly, the fitness defects of insertions in carAB were limited to minimal media without added ammonium (Figure 7E). With ammonium present, insertions in carAB had no phenotype, whereas carB2 was very important for fitness (Figure 7E). There were no genes that were both co-fit with carB2 and had clear sequence homology to a CPS glutaminase subunit. In particular, carA (BT0556) was co-fit with carB (BT0557) but not carB2 (r = 0.85 and r = −0.02, respectively). In other bacteria with two CPSs, one enzyme supplies carbamoyl phosphate for arginine biosynthesis, whereas the second is responsive to needs for pyrimidine biosynthesis (Paulus and Switzer, 1979), regardless of the nitrogen source. We propose that in B. thetaiotaomicron, the use of CarAB or CarB2 depends primarily on ammonium availability, with CarAB acting as a prototypical glutamine-dependent enzyme and CarB2 using ammonium to directly synthesize carbamoyl phosphate (Figure 7D).In our mouse experiment, both carAB and dapLF were critical for growth in vivo (Figure 7E). This observation is consistent with results of a previous study, in which carAB and dapL insertions in B. thetaiotaomicron VPI-5482 were less fit in a LF/HPP diet (similar to our in vivo experiment), whereas carB2 and ddh insertions were less fit in a HF/HS diet (Wu et al., 2015). Likewise, homologs of carB2 and ddh were important for fitness in a HF/HS diet but not in a LF/HPP diet in B. thetaiotaomicron 7330 (Wu et al., 2015). The phenotypes of carB2 and ddh were also greatly diminished in B. ovatus ATCC8483 when an arabinoxylan supplement was added to the diet (Wu et al., 2015). Because the diet was otherwise held constant, this last result shows that addition of microbiota-accessible carbohydrates is sufficient to switch Bacteroides ovatus from use of Ddh and CarB2 to DapLF and CarAB and suggests that it is the presence of microbiota-accessible carbohydrates in a LF/HPP diet (in contrast to a HF/HS diet) that has the same effect on other Bacteroides species.We propose the following model to synthesize these observations (Figure 7F). The fermentation of protein by the colonic microbiota is a major source of ammonium in the gut (Wrong et al., 1985), but the presence of fermentable carbohydrates significantly reduces the amount of ammonium produced (Birkett et al., 1996; Mortensen, 1992). Thus, when B. thetaiotaomicron grows in the context of a diet high in plant polysaccharides, it ferments the undigested carbohydrates and ammonium concentration in the gut remains low. In this environment, DapLF and CarAB are the preferred enzymes. Conversely, when B. thetaiotaomicron grows in a low-plant-polysaccharide environment, protein fermentation plays a larger role in energy generation and excess ammonium is liberated from the catabolic deamination of amino acids (Allison and Macfarlane, 1989; Dai et al., 2011; Smith and Macfarlane, 1996). B. thetaiotaomicron then takes advantage of the excess ammonium by directly incorporating it into nitrogen-containing compounds using Ddh and CarB2. Both carAB and dapLF are found in the high-affinity ammonium locus of B. thetaiotaomicron VPI-5482, and their expression is induced in vitro when ammonium is limiting (Iakiviak, 2017), suggesting that transcriptional control in response to ammonium levels is a critical regulatory mechanism controlling flux through the alternative pathways.

B. thetaiotaomicron uses alternative biosynthetic enzymes to adapt to ammonium availability in vivo

We were interested in whether data from our in vitro chemical-genetic screen could illuminate fitness phenotypes in vivo. To generate in vivo fitness data, we monocolonized germ-free mice with our barcoded library; collected fecal samples at days 2, 4, and 7 post-gavage; and quantified gene fitness using BarSeq. We first compared our in vivo data to those of a previous study that colonized mice with a set of Bacteroides species including B. thetaiotaomicron VPI-5482, in both a low-fat, high-plant polysaccharide (LF/HPP) and a high-fat, high-sugar (HF/HS) diet (Wu et al., 2015). Our fitness data were more correlated with the LF/HPP (r = 0.52, 95% confidence interval: 0.46–0.58) (Figure 7A) than the HF/HS diet (r = 0.33, 95% confidence interval: 0.25–0.40), as expected based on the mouse diet in our in vivo experiment (STAR methods). This correlation in part reflects the nutrients available in vivo to B. thetaiotaomicron, as there was significant overlap in genes related to amino acid biosynthesis (odds ratio = 28, 95% confidence interval: 9–92) and carbohydrate-active enzymes (CAZymes) (odds ratio = 40, 95% confidence interval: 15–102) (Figure 7B). Thus, our data are consistent with the previous study of B. thetaiotaomicron fitness in vivo in the context of a high-plant-polysaccharide diet.

Although insertions in amino acid biosynthesis genes had immediate deleterious effects on growth in our dataset, the fitness defects of CAZymes tended to be smaller at first and increased in magnitude over time (Figure 7C, top), likely reflecting the foraging of B. thetaiotaomicron on a diverse set of carbohydrates in the gut (Sonnenburg et al., 2005). Nonetheless, we identified significant growth defects in vivo for genes in multiple PULs, including the arabinan-activated PUL7 (BT0350, BT0351, and BT0367), and predicted PUL27 (BT2262–BT2264) (Figure 7C, bottom); PUL27 is constitutively expressed in vitro (Glenwright et al., 2017) but has no clearly defined substrates and was not important for growth under any conditions from our in vitro screen. The in vivo phenotypes of PUL27 suggest that it functions in the uptake of a diet- or host-derived nutrient that has yet to be identified.

The biosynthetic pathways for arginine and lysine were critical for growth in vivo, as previously observed (Goodman et al., 2009; Wu et al., 2015). Our in vitro data suggested that for both pathways, alternative enzymes are used depending on ammonium availability. The first example is synthesis of meso-diaminopimelate (mDAP), a precursor to lysine and peptidoglycan. Two alternative pathways for producing mDAP from 2,3,4,5-tetrahydropicolinate are conserved among the Bacteroides (Figure 7D). The dehydrogenase branch, catalyzed in a single step by Ddh (BT1979), uses free ammonium as a nitrogen source. The transaminase branch, catalyzed in two steps by DapL (BT0547) and DapF (BT0548), uses glutamate as a nitrogen source. The functions of DapL and Ddh have been verified biochemically using enzymes from B. fragilis (Hudson et al., 2008, 2011), which have 90% and 95% amino acid identity to the B. thetaiotaomicron enzymes, respectively. In Corynebacterium glutamicum, the dehydrogenase branch preferentially operates when ammonium is abundant; flux through the dehydrogenase branch is correlated with ammonium levels (Sonntag et al., 1993), and mutants in the transaminase branch require high levels of ammonium (Wehrmann et al., 1998). In our B. thetaiotaomicron library, ddh was important for fitness during growth in minimal media with ammonium but had no phenotype when ammonium was replaced with amino acids or other organic nitrogen sources (Figure 7E). Instead, insertions in dapLF were deleterious under these low ammonium conditions (Figure 7E), suggesting that relative flux through the two branches depends on availability of free ammonium in B. thetaiotaomicron as well (Figure 7D).

The second example of use of alternative enzymes is carbamoyl phosphate synthase (CPS), which synthesizes a nitrogen-containing precursor (carbamoyl phosphate) to arginine and pyrimidine nucleotides. In eubacteria and eukaryotes, CPS is composed of two subunits; the glutaminase subunit (CarA) liberates ammonium from glutamine and funnels it to the synthetase subunit (CarB), where the ammonium combines with other reactants to form carbamoyl phosphate (Figure 7D; Anderson and Meister, 1965). To date, all eubacterial CPSs have been found to be glutamine dependent. However, eubacterial CarB functions as an ammonium-dependent CPS when purified alone (Trotta et al., 1971), and mutations in the glutaminase subunit of the mitochondrial CPS that participate in the urea cycle make the enzyme ammonium dependent (Saeed-Kothe and Powers-Lee, 2003). B. thetaiotaomicron carries a two-gene locus, BT0556–BT0557, encoding a full CPS (CarAB) as well as a lone synthetase subunit BT3866 (CarB2) with no clear glutaminase partner (Figure 7D). Unexpectedly, the fitness defects of insertions in carAB were limited to minimal media without added ammonium (Figure 7E). With ammonium present, insertions in carAB had no phenotype, whereas carB2 was very important for fitness (Figure 7E). There were no genes that were both co-fit with carB2 and had clear sequence homology to a CPS glutaminase subunit. In particular, carA (BT0556) was co-fit with carB (BT0557) but not carB2 (r = 0.85 and r = −0.02, respectively). In other bacteria with two CPSs, one enzyme supplies carbamoyl phosphate for arginine biosynthesis, whereas the second is responsive to needs for pyrimidine biosynthesis (Paulus and Switzer, 1979), regardless of the nitrogen source. We propose that in B. thetaiotaomicron, the use of CarAB or CarB2 depends primarily on ammonium availability, with CarAB acting as a prototypical glutamine-dependent enzyme and CarB2 using ammonium to directly synthesize carbamoyl phosphate (Figure 7D).

In our mouse experiment, both carAB and dapLF were critical for growth in vivo (Figure 7E). This observation is consistent with results of a previous study, in which carAB and dapL insertions in B. thetaiotaomicron VPI-5482 were less fit in a LF/HPP diet (similar to our in vivo experiment), whereas carB2 and ddh insertions were less fit in a HF/HS diet (Wu et al., 2015). Likewise, homologs of carB2 and ddh were important for fitness in a HF/HS diet but not in a LF/HPP diet in B. thetaiotaomicron 7330 (Wu et al., 2015). The phenotypes of carB2 and ddh were also greatly diminished in B. ovatus ATCC8483 when an arabinoxylan supplement was added to the diet (Wu et al., 2015). Because the diet was otherwise held constant, this last result shows that addition of microbiota-accessible carbohydrates is sufficient to switch Bacteroides ovatus from use of Ddh and CarB2 to DapLF and CarAB and suggests that it is the presence of microbiota-accessible carbohydrates in a LF/HPP diet (in contrast to a HF/HS diet) that has the same effect on other Bacteroides species.

We propose the following model to synthesize these observations (Figure 7F). The fermentation of protein by the colonic microbiota is a major source of ammonium in the gut (Wrong et al., 1985), but the presence of fermentable carbohydrates significantly reduces the amount of ammonium produced (Birkett et al., 1996; Mortensen, 1992). Thus, when B. thetaiotaomicron grows in the context of a diet high in plant polysaccharides, it ferments the undigested carbohydrates and ammonium concentration in the gut remains low. In this environment, DapLF and CarAB are the preferred enzymes. Conversely, when B. thetaiotaomicron grows in a low-plant-polysaccharide environment, protein fermentation plays a larger role in energy generation and excess ammonium is liberated from the catabolic deamination of amino acids (Allison and Macfarlane, 1989; Dai et al., 2011; Smith and Macfarlane, 1996). B. thetaiotaomicron then takes advantage of the excess ammonium by directly incorporating it into nitrogen-containing compounds using Ddh and CarB2. Both carAB and dapLF are found in the high-affinity ammonium locus of B. thetaiotaomicron VPI-5482, and their expression is induced in vitro when ammonium is limiting (Iakiviak, 2017), suggesting that transcriptional control in response to ammonium levels is a critical regulatory mechanism controlling flux through the alternative pathways.

DISCUSSIONA small fraction of the gene content of the human microbiome has been experimentally characterized, and there are large knowledge gaps in our molecular understanding of these health-relevant bacteria. To address this knowledge gap, we performed hundreds of genome-wide fitness experiments in B. thetaiotaomicron and used the data to reannotate the functions of 40 proteins (Table S5). Nineteen of these proteins were annotated as “hypothetical protein” in RefSeq (as of April 2019), and many of the other proteins had vague annotations such as “MFS transporter.” These reannotations are also available in the Fitness Browser (Price et al., 2018) and PaperBLAST (Price and Arkin, 2017). Our large dataset contains phenotypes for hundreds of additional genes and should be a valuable resource for gene function inference and hypothesis generation. To facilitate these future studies, the B. thetaiotaomicron fitness data are publicly available for comparative analyses in the Fitness Browser at https://fit.genomics.lbl.gov.Much of the research to date on B. thetaiotaomicron has focused on polysaccharide degradation by PULs. Although our analysis was able to link 20 PULs to carbon sources, these findings represent <25% of the predicted PULs in the genome. Why so few? First, B. thetaiotaomicron may contain two or more PULs that act on the same substrate, and this redundancy would mask the impact of any single-gene mutation. Second, the PUL component(s) may perform its activity extracellularly, such that other mutants in the pooled library can complement any growth deficiency in trans. In addition to assaying mutants individually, approaches for performing genome-wide fitness assays within droplets hold the potential for eliminating complementation by other mutants (Thibault et al., 2019). Third, some PULs may not be expressed under our growth conditions or are involved in the breakdown of polysaccharides that we did not profile. For example, transcriptional profiling connected some PULs with N-linked glycans (Briliūtė et al., 2019; Cao et al., 2014). Some of these substrates may be activated only within the host; interestingly, we identified a phenotype for components of PUL27 within mice but not in any in vitro assays. Fourth, some PULs may have evolved to consume simpler sugars and oligosaccharides. For example, we identified a PUL (BT3567–BT3569) that was important only for fitness on the disaccharide laminaribiose, suggesting that chemical-genetic profiling with additional simple carbon sources may reveal functions for some PULs. Lastly, the definition of a PUL is very minimal and only requires a SusCD gene pair. Some of these minimal PULs may have other cellular roles. For instance, we showed that a two-gene PUL (BT1439–BT1440) confers susceptibility to vancomycin in BHIS (Figure 6D).We showed that growth on trehalose and other disaccharides required a 3-keto-disaccharide hydrolase (BT2157). This enzyme has a predicted signal peptide (Almagro Armenteros et al., 2019), implying that these disaccharides are oxidized in the periplasm. However, B. thetaiotaomicron does not encode the periplasmic 3-ketoglycoside dehydrogenases LacABC or ThuAB (Ampomah et al., 2013; Arellano et al., 2010; Miyazaki et al., 2018). Moreover, B. thetaiotaomicron has a limited electron transport chain that ends at menaquinol:fumarate oxidoreductase (Fischbach and Sonnenburg, 2011), yielding succinate as the reduced end product, which probably precludes the operation of cytochrome c-dependent dehydrogenases such as LacABC or ThuAB. Instead, oxidation to a 3-keto-disaccharide is probably performed by BT2158, which was important for growth on trehalose and several other disaccharides but not for growth on 3-keto-trehalose (Figure 4A). BT2158 has a putative Tat export signal (Almagro Armenteros et al., 2019) and is distantly related (Pfam: PF01408) to periplasmic glucose-fructose oxidoreductase (Gfo). Gfo has a tightly bound NADP cofactor that is exported together with the folded protein by the Tat system (Halbig et al., 1999), and it oxidizes glucose to gluconolactone while reducing fructose to sorbitol (Kingston et al., 1996). Hence, we propose that BT2158 uses a tightly bound NAD(P) cofactor to oxidize disaccharides to 3-keto-disaccharides while reducing another as yet unidentified sugar.Integrating data from gnotobiotic mouse models and our chemical-genetic screen, we presented evidence that B. thetaiotaomicron remodels its biosynthetic pathways to directly incorporate ammonium into nitrogen-rich compounds when ammonium is plentiful and uses alternative nitrogen sources when ammonium is scarce. Furthermore, we showed that this capacity is likely an adaptation to diet-dependent ammonium fluctuations in the colon. Interestingly, the genes encoding the ammonium-independent enzymes are all located in the high-affinity ammonium (HAA) locus. Expression of genes in the HAA locus is induced under low-ammonium conditions (Iakiviak, 2017), providing a simple regulatory mechanism for the transition between ammonium-dependent and -independent pathways. We found that the HAA locus carries the genes for a canonical glutamine-dependent carbamoyl phosphate synthetase (CPS). Meanwhile, an orphan CPS synthetase subunit (CarB2) is critical for growth in high-ammonium environments. The lack of support for any glutaminase partner for CarB2 in our dataset suggests that CarB2 functions as an ammonium-dependent CPS (Figure 7D). Prokaryotic ammonium-dependent CPSs have been described previously, but these proteins are restricted to Archaea (Legrain et al., 1995; Popa et al., 2012), leading to a long-standing hypothesis that eubacteria only encode glutamine-dependent CPSs. Further experimental confirmation that CarB2 is an ammonium-dependent CPS will be an important future step in filling this knowledge gap.Our approach can be applied generally to uncover gene functions in the human microbiome. In particular, barcoding of mutant libraries enables a rapid assessment of gene importance across many conditions at low cost (Price et al., 2018). Nevertheless, challenges still need to be overcome, including the accelerated development of genetic tools for non-model species in the human microbiota (Liu and Deutschbauer, 2018; Peters et al., 2019) and of scalable phenotypic assays that more accurately reflect the natural ecology of these species. In addition, conservation of a phenotype in multiple species is a powerful indicator of gene function (Price et al., 2018), and the generation of similar gene-phenotype maps in related bacteria will expand the scope of our analysis.

A small fraction of the gene content of the human microbiome has been experimentally characterized, and there are large knowledge gaps in our molecular understanding of these health-relevant bacteria. To address this knowledge gap, we performed hundreds of genome-wide fitness experiments in B. thetaiotaomicron and used the data to reannotate the functions of 40 proteins (Table S5). Nineteen of these proteins were annotated as “hypothetical protein” in RefSeq (as of April 2019), and many of the other proteins had vague annotations such as “MFS transporter.” These reannotations are also available in the Fitness Browser (Price et al., 2018) and PaperBLAST (Price and Arkin, 2017). Our large dataset contains phenotypes for hundreds of additional genes and should be a valuable resource for gene function inference and hypothesis generation. To facilitate these future studies, the B. thetaiotaomicron fitness data are publicly available for comparative analyses in the Fitness Browser at https://fit.genomics.lbl.gov.

Much of the research to date on B. thetaiotaomicron has focused on polysaccharide degradation by PULs. Although our analysis was able to link 20 PULs to carbon sources, these findings represent <25% of the predicted PULs in the genome. Why so few? First, B. thetaiotaomicron may contain two or more PULs that act on the same substrate, and this redundancy would mask the impact of any single-gene mutation. Second, the PUL component(s) may perform its activity extracellularly, such that other mutants in the pooled library can complement any growth deficiency in trans. In addition to assaying mutants individually, approaches for performing genome-wide fitness assays within droplets hold the potential for eliminating complementation by other mutants (Thibault et al., 2019). Third, some PULs may not be expressed under our growth conditions or are involved in the breakdown of polysaccharides that we did not profile. For example, transcriptional profiling connected some PULs with N-linked glycans (Briliūtė et al., 2019; Cao et al., 2014). Some of these substrates may be activated only within the host; interestingly, we identified a phenotype for components of PUL27 within mice but not in any in vitro assays. Fourth, some PULs may have evolved to consume simpler sugars and oligosaccharides. For example, we identified a PUL (BT3567–BT3569) that was important only for fitness on the disaccharide laminaribiose, suggesting that chemical-genetic profiling with additional simple carbon sources may reveal functions for some PULs. Lastly, the definition of a PUL is very minimal and only requires a SusCD gene pair. Some of these minimal PULs may have other cellular roles. For instance, we showed that a two-gene PUL (BT1439–BT1440) confers susceptibility to vancomycin in BHIS (Figure 6D).

We showed that growth on trehalose and other disaccharides required a 3-keto-disaccharide hydrolase (BT2157). This enzyme has a predicted signal peptide (Almagro Armenteros et al., 2019), implying that these disaccharides are oxidized in the periplasm. However, B. thetaiotaomicron does not encode the periplasmic 3-ketoglycoside dehydrogenases LacABC or ThuAB (Ampomah et al., 2013; Arellano et al., 2010; Miyazaki et al., 2018). Moreover, B. thetaiotaomicron has a limited electron transport chain that ends at menaquinol:fumarate oxidoreductase (Fischbach and Sonnenburg, 2011), yielding succinate as the reduced end product, which probably precludes the operation of cytochrome c-dependent dehydrogenases such as LacABC or ThuAB. Instead, oxidation to a 3-keto-disaccharide is probably performed by BT2158, which was important for growth on trehalose and several other disaccharides but not for growth on 3-keto-trehalose (Figure 4A). BT2158 has a putative Tat export signal (Almagro Armenteros et al., 2019) and is distantly related (Pfam: PF01408) to periplasmic glucose-fructose oxidoreductase (Gfo). Gfo has a tightly bound NADP cofactor that is exported together with the folded protein by the Tat system (Halbig et al., 1999), and it oxidizes glucose to gluconolactone while reducing fructose to sorbitol (Kingston et al., 1996). Hence, we propose that BT2158 uses a tightly bound NAD(P) cofactor to oxidize disaccharides to 3-keto-disaccharides while reducing another as yet unidentified sugar.

Integrating data from gnotobiotic mouse models and our chemical-genetic screen, we presented evidence that B. thetaiotaomicron remodels its biosynthetic pathways to directly incorporate ammonium into nitrogen-rich compounds when ammonium is plentiful and uses alternative nitrogen sources when ammonium is scarce. Furthermore, we showed that this capacity is likely an adaptation to diet-dependent ammonium fluctuations in the colon. Interestingly, the genes encoding the ammonium-independent enzymes are all located in the high-affinity ammonium (HAA) locus. Expression of genes in the HAA locus is induced under low-ammonium conditions (Iakiviak, 2017), providing a simple regulatory mechanism for the transition between ammonium-dependent and -independent pathways. We found that the HAA locus carries the genes for a canonical glutamine-dependent carbamoyl phosphate synthetase (CPS). Meanwhile, an orphan CPS synthetase subunit (CarB2) is critical for growth in high-ammonium environments. The lack of support for any glutaminase partner for CarB2 in our dataset suggests that CarB2 functions as an ammonium-dependent CPS (Figure 7D). Prokaryotic ammonium-dependent CPSs have been described previously, but these proteins are restricted to Archaea (Legrain et al., 1995; Popa et al., 2012), leading to a long-standing hypothesis that eubacteria only encode glutamine-dependent CPSs. Further experimental confirmation that CarB2 is an ammonium-dependent CPS will be an important future step in filling this knowledge gap.

Our approach can be applied generally to uncover gene functions in the human microbiome. In particular, barcoding of mutant libraries enables a rapid assessment of gene importance across many conditions at low cost (Price et al., 2018). Nevertheless, challenges still need to be overcome, including the accelerated development of genetic tools for non-model species in the human microbiota (Liu and Deutschbauer, 2018; Peters et al., 2019) and of scalable phenotypic assays that more accurately reflect the natural ecology of these species. In addition, conservation of a phenotype in multiple species is a powerful indicator of gene function (Price et al., 2018), and the generation of similar gene-phenotype maps in related bacteria will expand the scope of our analysis.

STAR★METHODSDetailed methods are provided in the online version of this paper and include the following:RESOURCE AVAILABILITYLead contactFurther information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Adam Deutschbauer (amdeutschbauer@lbl.gov).Materials availabilityPlasmids, strains, and mutant libraries generated in this study are available from the lead contact with a completed Material Transfer Agreement (MTA).Data and code availabilityFitness data from the 523 successful experiments is available for comparative analysis in the Fitness Browser at https://fit.genomics.lbl.gov (archived at https://figshare.com/articles/dataset/The_Fitness_Browser_Genome-wide_mutant_fitness_data_for_diverse_bacteria_November_2020_release_/13172087/1). In addition, original fitness data (and all associated metadata) have been deposited to figshare: https://figshare.com/articles/dataset/RB-TnSeq_data_for_Bacteroides_thetaiotaomicron_VPI-5482/13171766/1. The software for RB-TnSeq is available at https://bitbucket.org/berkeleylab/feba. The code for generating the figure panels of Figures 7A-7C and 7E and associated statistics in the text is available as a part of a Code Ocean compute capsule (https://codeocean.com/capsule/3569674/tree/v1).EXPERIMENTAL MODEL AND SUBJECT DETAILSBacterial strains, plasmids, and oligonucleotides used in this study are listed in the Key Resources Table.Male germ-free (GF) C57BL/6J mice were maintained within the UCSF Gnotobiotic Core Facility (https://gnotobiotics.ucsf.edu) and fed ad libitum autoclaved standard chow diet (Lab Diet 5021). The mice were housed in a single gnotobiotic isolator for the duration of the bacterial mutant library colonization experiment. All mouse experiments were approved by the University of California, San Francisco (UCSF) Institutional Animal Care and Use Committee and performed accordingly.METHOD DETAILSMedia, compounds and growth conditionsGrowth media were purchased from BD. In this study, we used many of the compounds from a previous investigation of bacterial mutant fitness (Price et al., 2018); most were purchased from Sigma-Aldrich. We also purchased and assayed additional compounds specifically relevant to B. thetaiotaomicron, such as polysaccharides and bile salts. The source for some compounds is listed in Table S7.We typically grew B. thetaiotaomicron in an anaerobic chamber at 37°C in one of two growth media: Brain Heart Infusion Supplemented (BHIS) medium using Bacto Brain Heart Infusion (BHI) as the base, or Varel-Bryant (VB) defined medium (Varel and Bryant, 1974). Media recipes are provided in Table S7. VB medium contains methionine, so vitamin B12 is not required.Construction of a barcoded transposon delivery vector for B. thetaiotaomicronWe previously described a mariner transposon delivery vector library pHLL255 (magic pool) containing hundreds of combinations of promoters and erythromycin drug markers (Liu et al., 2018). To identify a suitable vector for mutagenizing B. thetaiotaomicron, we first constructed a small mutant library by conjugating with an Escherichia coli strain carrying pHLL255. We performed TnSeq on this library to map transposon insertion locations and their associated DNA barcodes, enabling us to infer which vector design in the magic pool resulted in the mutation (Liu et al., 2018). Analysis of this library revealed that the optimal drug marker for mutagenizing B. thetaiotaomicron was the erythromycin resistance gene ermG (Goodman et al., 2009; Shoemaker et al., 2001), the optimal promoter for expressing the mariner transposase was the native rpoD (BT1311) promoter (Goodman et al., 2009), and the optimal promoter for expressing ermG was the native dnaE (BT2230) promoter.Although this vector design had the highest insertion efficiency relative to the other designs in the magic pool, it still resulted in strand bias whereby the insertion orientation of the transposon was more often in the same transcriptional orientation as the mutated gene. Thus, in addition to constructing the best but potentially suboptimal vector design from the magic pool results (pTGG45), we also constructed another vector (pTGG46) by replacing the dnaE promoter in pTGG45 with a cepA hybrid promoter from B. fragilis RBF-103 (Bayley et al., 2000). To determine which vector exhibited the best combination of high efficiency and low strand bias, we constructed two small mutant libraries with either pTGG45 or pTGG46. TnSeq analysis of these libraries revealed that both had high insertion efficiency, but while the strand bias was again significant for pTGG45 (60% of insertions were in the same orientation as transcription of the mutated gene), there was essentially no strand bias for pTGG46 (49.4%). Thus, we barcoded pTGG46 (pTGG46_NN1) with millions of random 20-nucleotide barcodes using Golden Gate assembly, as previously described (Liu et al., 2018).Transposon mutant library constructionWe carried out transposon mutagenesis by conjugating B. thetaiotaomicron recipient cells with E. coli donor cells carrying the bar-coded transposon vector library pTGG46_NN1 at 1:1 ratio. Specifically, we grew 10 mL of wild-type B. thetaiotaomicron VPI-5482 in BHIS overnight at 37°C. The next morning, we recovered a 2 mL freezer stock of strain AMD776 (E. coli WM3064 with pTGG46_NN1) in 50 mL LB supplemented with 50 μg/mL carbenicillin and 300 μM diaminopimelic acid (DAP) at 37°C. When the OD600 of the E. coli donor strain reached ~1.0, we harvested 15 OD600 units of the culture and washed the cells three times with fresh BHIS supplemented with DAP. Then, 15 OD600 units of B. thetaiotaomicron wild-type cells were harvested, mixed with the washed donor cells, and resuspended in a final volume of 60 μL with BHIS supplemented with DAP. The resuspension was spotted onto 0.45-μm gravimetric analysis membrane filters (Millipore) and incubated anaerobically overnight on BHIS agar plates supplemented with DAP at 37°C. The next day, the conjugation mixture was scraped from the membrane and resuspended in 6 mL fresh BHIS medium supplemented with 10 μg/mL erythromycin and plated at different dilutions on BHIS plates supplemented with 10 μg/mL erythromycin. Plates were incubated at 37°C anaerobically for 48 h to let visible colonies develop. We then pooled ~250,000 colonies and grew the pooled library anaerobically in liquid BHIS supplemented with 10 μg/mL erythromycin and 200 μg/mL gentamicin (to further select against the E. coli conjugation donor strain as B. thetaiotaomicron is naturally resistant to gentamicin). When the culture reached saturation, we made multiple, single-use glycerol stocks of the final library and extracted genomic DNA for TnSeq analysis. To map the genomic locations of the transposon insertions and link these insertions to their associated DNA barcodes, we used our previously described TnSeq protocol (Wetmore et al., 2015). To map insertions in BT2158, we relaxed our mapping stringency and only required one nucleotide difference relative to BT4448 (BT2158 and BT4448 are 98% identical with only 30 mismatches over 1,476 nucleotides). The final mutant library was named Btheta_ML6.In vitro genome-wide mutant fitness assaysWe performed genome-wide mutant fitness assays as described previously (Price et al., 2018). For B. thetaiotaomicron, we thawed an aliquot of the full transposon mutant library, inoculated the entire aliquot into 50 mL of BHIS supplemented with 10 μg/mL erythromycin, and grew the library to mid-log phase. We then collected 6 cell pellets of ~1.0 OD600 unit each (the “Time0” sample). We used the remaining cells to inoculate competitive growth assays. Nearly all fitness assays were performed in 1.2 mL of growth medium in either a 24-well transparent microplate (Greiner) or in a 96-deepwell plate (Costar) at a starting OD600 of 0.02. For carbon and nitrogen source assays in VB, we washed the cells in VB lacking these nutrients two times. We grew cultures until they reached stationary phase, and then collected cell pellets (the “Condition” sample). We also performed a few “survival” experiments in which we incubated the mutant library aerobically for different amounts of time. After this incubation, we inoculated cells into fresh BHIS to select for cells that survived the stress. We extracted genomic DNA from the TimeO and Condition samples in a 96-well microplate format with a QIAamp 96 DNA Qiacube HT kit (QIAGEN). In addition, we performed genome-wide fitness assays using previously described E. vietnamensis KMM6221 and Pedobacter sp. GW460-11-11-14-LB5 barcoded transposon mutant libraries (Price et al., 2018) with the carbon sources substrates lactulose, maltitol, methyl-β-D-galactopyranoside, and lactitol.We performed barcode sequencing (BarSeq) as previously described (Price et al., 2018, 2019). For most experiments, we performed BarSeq with indexed P2 oligos and a mix of non-indexed P1 oligos with variable lengths of Ns (2-5) to stagger the reads. For some experiments, we used BarSeq oligos with both P1 and P2 indexed to minimize the impact of incorrectly assigned indexes in Illumina HiSeq4000 runs (Sinha et al., 2017). Strain and gene fitness scores were calculated as previously described (Wetmore et al., 2015).Construction of deletion strains and individual mutant growth assaysWe constructed unmarked, in-frame gene deletions using a previously established approach (Mishra and Imlay, 2013). Growth curves of individual mutants were carried out in a 96-well plate with either an Epoch2 (BioTek) or Infinite F200 (Tecan) plate reader housed in an anaerobic chamber. Note that optical densities in microplates are typically lower than those in a standard cuvette. To calculate IC50 values for ΔBT1439 and the parent on vancomycin, we used the drc package in R (Ritz et al., 2015). Specifically, we used drm() with the LL.4() model and defined the response as the average OD600 at the time that uninhibited cells (no vancomycin) reached their maximum OD600 value.Growth and cell harvesting for RT-qPCR assay of BT3761 repression by arginineAn overnight culture of B. thetaiotaomicron VPI-5482 Δtdk in 5 mL of tryptone-yeast extract-glucose (TYG) broth was grown at 37°C in a temperature-controlled Model 2000 incubator (COY Lab Products) in an anaerobic chamber (custom build, COY Lab Products). This overnight culture was pelleted and washed in VB medium three times before being inoculated into several wells of VB (the control condition) and VB supplemented with 3 mM arginine (the experimental condition). Biological replicates were grown in a 2-mL 96-deepwell plate (PlateOne, USA Scientific, Cat. No. 1896-2110). The atmospheric gas was a blend of 85% nitrogen, 10% carbon dioxide, and 5% hydrogen (Praxair, Cat. No. BI-NICDHYC4). OD600 was monitored using an Epoch2 (BioTek) microplate reader inside the anaerobic chamber, and cultures were periodically diluted into fresh media to keep the OD600 below 0.05 so that cells remained in log phase. After ~24 h of growth, 1.5 mL were transferred into 3 mL of RNAprotect® Bacterial Reagent. The cell suspension was centrifuged for 30 min at 4,000g and 4°C in a Sorvall Legend XTR centrifuge (Thermo Scientific). The supernatant was removed, cell pellets were air-dried at room temperature for 5 min, and pellets were flash frozen in liquid nitrogen and stored at −80°C until RNA was extracted the next day.Growth and cell harvesting for RT-qPCR assay of potential polar effects of ΔsusBOvernight cultures of B. thetaiotaomicron VPI-5482 Δtdk (the parental control) and B. thetaiotaomicron VPI-5482 Δtdk ΔBT3703 (ΔsusB, the experimental strain) were grown in 5 mL VB with α-cyclodextrin as the sole carbon source at 37°C in a temperature-controlled incubator inside an anaerobic chamber. Cultures were then inoculated into separate wells of VB with α-cyclodextrin and grown in log phase via periodic dilution into fresh medium to keep OD600 < 0.05. Details of growth conditions (plasticware and other equipment) are the same as the arginine repression experiment above. After ~24 h of growth, 1 mL was transferred into 2 mL of RNAprotect® Bacterial Reagent and the cell suspension was centrifuged for 30 min at 4°C and 3,428g in a 5920R centrifuge (Eppendorf). The supernatant was removed, cell pellets were dried at room temperature for 5 min, and pellets were stored at −80°C (without flash freezing) until the RNA was extracted (within 1 week).RNA isolation and quality control for RT-qPCR experimentsRNA was isolated using the RNeasy® Mini Kit (QIAGEN, Cat. No. 74104) with an on-column DNase I digestion using the RNase-free DNase Set (QIAGEN, Cat. No. 79254). RNA isolation was performed following the kit handbook, with no modifications. We used 2-mercaptoethanol from Sigma-Aldrich (Cat. No. M3148-100ML)to make fresh RLT buffer (from the QIAGEN kit) for every RNA isolation experiment. Every sample was eluted into 50 μL of RNase-free water from the kit. RNA concentration and purity were quantified using UV-vis absorbance spectra measured on a NanoDrop One C (Thermo Scientific, software version 1.4.2). The yield (concentration of RNA) and purity (A260/A280, A260/A230) are reported for every sample in Table S6. After quantification, RNA samples were diluted to their final concentration using values reported from the UV-vis spectrophotometer and stored at −20°C. After using the RNA in RT-qPCR experiments, the integrity of RNA in the remaining volume of samples was quantified using a Fragment Analyzer (Advanced Analytical Technologies, Inc.) and a high-sensitivity RNA kit (15 nt) (Agilent, Cat. No. DNF-472-0500). The RNA Quality Number, as determined by PROsize v. 3.0.16, is reported in Table S6.Reaction conditions for RT-qPCR experimentsAll RT-qPCR experiments were performed using the iTaq Universal SYBR® Green One-Step Kit (Bio-Rad, Cat. No. 1725151). In every experiment, we used a 10 μL final volume with 4 μL of input RNA and 0.3 μM primer. For the arginine-repression experiment, 22 ng of total RNA were used. For ΔsusB polarity experiments, 100 ng of total RNA were used. The RT-qPCR reaction was performed according to the kit handbook instructions for Bio-Rad CFX96 machines, without modification. Briefly, a single-step incubation for 10 min at 50°C allowed for reverse transcription to proceed, followed by a single-step incubation at 95°C for 1 min to denature the DNA and activate the polymerase. These steps were followed by 40 cycles of a 10 s denaturation step (at 95°C) and a 20 s annealing and extension hybrid step (at 60°C). Finally, we performed a melt-curve analysis by incrementing from 65°C to 95°C in 0.5°C steps, at 5 s per step. Samples were analyzed in Hard-Shell PCR Plates, white shell/white well (Bio-Rad, Cat. No. HSP9655) and run on a Bio-RAD® CFX96 machine using Bio-Rad CFX Manager v. 3.1. In both experiments, three biological replicates were analyzed, with one technical replicate per sample (n = 6). At least one no-template control was run on the same plate for each primer set. A no-reverse transcriptase experiment was run for each sample using the experimental target primers (BT3761 for the arginine expression experiment, and susC and susD for the susB polarity experiment). We designed primers for the amplification of 15 reference target genes in B. thetaiotaomicron in addition to the 3 experimental targets and the traditional reference target of 16S rDNA (Table S6). We determined the amplification efficiency of all of these primer pairs using 10-fold RNA dilution experiments (Table S6). We also found the best combination of reference genes for both experiments by running preliminary experiments using a large set of reference genes. A set of reference genes were chosen that exhibited high stability between experiments and controls and that were distributed across multiple biological processes.Targeted proteomicsFor carbon source experiments, we grew cultures of wild-type B. thetaiotaomicron in VB with 20 mM D-trehalose, D-raffinose, or D-glucose overnight to OD600 values between 1.0 and 1.5. For bile salt experiments, we grew wild-type B. thetaiotaomicron in VB with 20 mM D-glucose supplemented with 0, 0.125, 0.25, or 0.5 mg/mL bile salts for 20 h to OD600 values between 0.5 and 0.9. For all experiments, we harvested 2 OD600 units of cells and washed two times with 10 mM NH4HCO3 before extracting total protein using a methanol-chloroform precipitation method (Chen et al., 2019a). Protein concentration was determined using the Bio-Rad DC protein assay (Lot #5000112) following the manufacturer’s protocol, and the same amount of proteins was subjected to standard reduction, alkylation, and trypsin digestion procedures. A targeted selected reaction monitoring (SRM) method was developed to quantify unique peptides of proteins using an in-house constructed B. thetaiotaomicron spectral library (Chen et al., 2019b).SRM targeted proteomic assays were performed on an Agilent 6460 QQQ mass spectrometer system coupled to an Agilent 1290 UHPLC system (Agilent Technologies, Santa Clara, CA). Twenty micrograms of peptides from each sample were separated on an Ascentis Express Peptide C18 column (2.7-mm particle size, 160-Å pore size, 5-cm length × 2.1-mm inside diameter (ID), coupled to a 5-mm × 2.1-mm ID guard column with the same particle and pore size, operating at 60°C; Sigma-Aldrich) operating at a flow rate of 0.4 mL/min via the following gradient: initial conditions were 98% solvent A (0.1% formic acid), 2% solvent B (99.9% acetonitrile, 0.1% formic acid). Solvent B was increased to 40% over 5 min, and then to 80% over 0.5 min, and held for 2 min at a flow rate of 0.6 mL/min, followed by a ramp back down to 2% solvent B over 0.5 min where it was held for 1 min to re-equilibrate the column to original conditions. The eluted peptides were ionized via an Agilent Jet Stream ESI source operating in positive-ion mode with the following source parameters: gas temperature = 250°C, gas flow = 13 L/min, nebulizer pressure = 35 psi, sheath gas temperature = 250°C, sheath gas flow = 11 L/min, capillary voltage = 3500 V, nozzle voltage = 0 V. Data were acquired using Agilent MassHunter v. B.08.02. Acquired SRM data were analyzed by Skyline v. 3.70 (MacCoss Lab Software).Production of 3-keto-trehaloseSingle colonies of Agrobacterium rhizogenes K599 were inoculated in a minimal medium (MM) from Kurowski and Darbyshire (1978) with 2% (w/v) trehalose as a carbon source and supplemented with 0.5 g/L yeast extract, and grown overnight at 30°C shaking at 250 rpm. Cells were then harvested by centrifugation, washed twice with 0.1 M phosphate-buffer (pH 7.0) and inoculated into 50 mL of MM+2% trehalose without yeast extract. The next day, an aliquot of this culture in MM was used to inoculate 200 mL of MM+2% trehalose (1:20 ratio). Samples were removed periodically, centrifuged, and filter-sterilized to obtain cell-free supernatants. Consumption of trehalose was monitored using the Trehalose Assay kit (Megazyme) and the formation of keto-sugar was determined by the alkaline method from Fukui and Hayano (1969). The cell-free 3-keto-trehalose-containing culture supernatant used for fitness and enzymatic assays was obtained after 29 h of growth and contained 17.4 mM of trehalose and 7.9 mM 3-keto-trehalose.Expression and purification of recombinant BT2157The sequence encoding BT2157 was PCR-amplified from genomic DNA, cloned into a linearized pET32a plasmid using Gibson Assembly, and transformed into E. coli chemically competent DH5-α cells (New England Biolabs). The plasmid sequence was verified by Sanger sequencing and used to transform the production strain E. coli BL21 DE3. For protein production, E. coli BL21 DE3 cells expressing N-terminal His-tagged BT2157 were inoculated from single colonies into LB with 100 μg/mL ampicillin and grown at 37°C overnight. Production cultures were started by inoculating 10 mL of pre-cultures into 1 L of LB with 100 μg/mL ampicillin. After growth at 37°C with agitation until OD600~0.5, protein expression was induced by the addition of 0.15 mM isopropyl β-D-1 thiogalactopyranoside (IPTG) and cultures were incubated at 37°C for 3 h. Cells were pelleted by centrifugation at 8000g for 15 min, resuspended in 60 mL of lysis buffer (50 mM potassium phosphate, 300 mM NaCl, 10 mM imidazole, pH 7.8), and stored at −80°C. For protein purification, resuspended cells were thawed on ice and lysed by sonication. Cell debris and unbroken cells were removed by centrifugation at 10,000 rcf. for 20 min at 6°C. All purification steps were performed using ice-cold buffers and reagents. Protein purification was performed as previously described (Liou et al., 2020), except with a HisTrap FF column (GE Healthcare) replacing the free resin.In vitro endpoint assays with recombinant BT2157In vitro endpoint assays with purified BT2157 consisted of 50 μL of cell-free A. rhizogenes culture supernatant containing 7.9 mM 3-keto-trehalose, 50 μL of purified protein, and reaction buffer (50 mM potassium buffer, pH 7) up to a total volume of 1 mL. Each reaction contained final concentrations of 395 μM 3-keto-trehalose and 28.8 μM his-tagged BT2157. Reactions were initiated by the addition of substrate and incubated at 30°C. Reactions were interrupted by the addition of 3 mL of 0.1 mM NaOH, and after 3 min of incubation, the absorbance at 340 nm was measured for each sample to determine keto-trehalose concentration (Fukui and Hayano, 1969).Gnotobiotic mouse experimentsEight GF male 7-week-old C57BL/6J mice were housed in a single gnotobiotic isolator for the duration of the experiment, fed a standard irradiated diet ad libitum, split into 3 cages (n = 2-3 mice/cage), and allowed to acclimate for 3 days prior to colonization. The inoculum was prepared by thawing a 1 mL aliquot of the B. thetaiotaomicron transposon library, transferring the entire aliquot into 50 mL of BHIS supplemented with 10 μg/mL erythromycin, and incubating anaerobically at 37°C until the cells reached mid-log phase (16 h). Six bacterial cell pellets were collected as Time0 sample references. Each mouse was colonized by oral gavage with 200 μL of the B. thetaiotaomicron transposon library. Stool samples were collected on days 2, 4, and 7 post-colonization to assess longitudinal shifts in mutant abundance using BarSeq. Mice were monitored and weighed daily. Genomic DNA from the Time0 cells and stool samples collected over time was extracted using the ZymoBIOMICS 96 MagBead DNA Kit (D4302) and followed by BarSeq, as described above.QUANTIFICATION AND STATISTICAL ANALYSISFitness data analysisOverall, we attempted 812 genome-wide fitness assays in this study, of which 523 passed our metrics for internal consistency (Wetmore et al., 2015). We considered a gene fitness value in a single experiment to be statistically significant if ∣fitness∣ > 1 and ∣t∣ > 4, where t is fitness/estimated standard error. Across all 523 experiments × 4,055 genes, 31,717 gene fitness values (1.5%) had ~fitness~ > 1 and ∣t∣ > 4. By contrast, across 68 control comparisons of a Time0 sample to other Time0 samples from the same day, 135 gene fitness values (0.05%) met these thresholds. The estimated false discovery rate for a significant phenotype was 0.05%/1.5% = 3.3%. From these 523 experiments, we identified genes with a specific phenotype in only one or a handful of conditions using previously described criteria (Price et al., 2018): ∣fitness∣ > 1 and ∣t∣ > 5 and 95th percentile(∣fitness∣) < 1 and ∣fitness∣ > 95th percentile(∣fitness∣)+0.5. For the analysis of the total number of genes with a specific phenotype, we excluded 1 gene that had a specific phenotype in BHIS medium alone. A specific important phenotype has the same criteria as above except that the fitness score has to be negative. For the specific phenotype analysis of PUL genes on carbon sources (Table S4; Figure 2), we used a subset of 492 experiments (we excluded the mouse experiment, and carbon source experiments with lactulose, maltitol, and 3-keto-trehalose) (https://figshare.com/articles/Large-scale_genetic_characterization_of_the_human_gut_commensal_bacterium_Bacteroides_thetaiotaomicron/7742855/1). We identified pairs of genes with highly correlated patterns of phenotypes across all 523 successful experiments (cofitness) using the linear (Pearson) correlation and a threshold of 0.8 (Price et al., 2018). Most data analyses to infer the putative functions of genes were performed with the Fitness Browser (https://fit.genomics.lbl.gov).One caveat in most transposon mutagenesis studies is complications due to polarity, whereby insertion in upstream genes of an operon can disrupt the expression and function of downstream genes. Relatedly, it is also possible for the promoter of the drug resistance gene used to select mutants to drive expression of downstream genes (if the transposon is oriented in the same direction as the operon). The transposon that we used to construct our library only contains the drug marker cassette (promoter and gene) and the barcode region; there is no obvious transcriptional terminator. Thus, we expect that if a transposon lands in the same transcriptional orientation as the operon, then downstream genes will be overexpressed. To quantify the potential for polarity in our dataset, we identified instances in which fitness values calculated from plus-strand transposon insertions differed significantly from those calculated from minus-strand insertions. Specifically, we considered gene-condition phenotypes to potentially be due to polar effects if: (1) the absolute value of the fitness calculated using the standard approach with all insertions was > 1, (2) there were at least two strains in each orientation with sufficient data, (3) the average fitness on the two strands differed by ≥ 1, and (4) an unpaired t test on the two sets of fitness values gave p < 0.01. Among 80,737 gene-experiment pairs with ∣fitness∣ > 1 and ≥ 2 insertions on both strands, we identified 2,248 instances (3%) of potential polarity (Table S9), including 208 genes covered by ≥ 2 separate experiments. Thus, while polarity does appear to occur, the majority of significant phenotypes in our dataset are consistent regardless of transposon orientation.[REMOVED HYPERLINK FIELD]Data analysis for RT-qPCR experimentsCq values were determined using the Bio-Rad CFX Manager software running the Bio-RAD® CFX96 Real-Time System. Csv files exported by Bio-Rad CFX Manager were manually converted into qbase-format files and input into qbase+ v. 3.2 for analysis. All biological and technical replicates were treated as equivalent in the analysis. Amplification efficiencies of reference and target genes were calculated for every primer pair and used as part of the analysis. Final fold-changes and 95% confidence intervals were calculated from the CNRQ values and standard errors output by qbase+, with the assumption that measurement errors were log-normal. M and CV values for reference gene stability, along with Cq values of no-template controls and the difference in Cq values of target genes with and without reverse-transcriptase, are reported in Table S6.Data sources for enrichment analysisData on predicted amino acid biosynthesis genes in the B. thetaiotaomicron VPI-5482 genome were downloaded from the BioCyc database (https://biocyc.org/; Karp et al., 2019). The genes for predicted carbohydrate active enzyme (CAZymes) in B. thetaiotaomicron VPI-5482 were downloaded from the Carbohydrate Active Enzymes database (http://www.cazy.org/; Lombard et al., 2014).

STAR★METHODS

Detailed methods are provided in the online version of this paper and include the following:

RESOURCE AVAILABILITYLead contactFurther information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Adam Deutschbauer (amdeutschbauer@lbl.gov).Materials availabilityPlasmids, strains, and mutant libraries generated in this study are available from the lead contact with a completed Material Transfer Agreement (MTA).Data and code availabilityFitness data from the 523 successful experiments is available for comparative analysis in the Fitness Browser at https://fit.genomics.lbl.gov (archived at https://figshare.com/articles/dataset/The_Fitness_Browser_Genome-wide_mutant_fitness_data_for_diverse_bacteria_November_2020_release_/13172087/1). In addition, original fitness data (and all associated metadata) have been deposited to figshare: https://figshare.com/articles/dataset/RB-TnSeq_data_for_Bacteroides_thetaiotaomicron_VPI-5482/13171766/1. The software for RB-TnSeq is available at https://bitbucket.org/berkeleylab/feba. The code for generating the figure panels of Figures 7A-7C and 7E and associated statistics in the text is available as a part of a Code Ocean compute capsule (https://codeocean.com/capsule/3569674/tree/v1).

RESOURCE AVAILABILITY

Lead contactFurther information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Adam Deutschbauer (amdeutschbauer@lbl.gov).

Lead contact

Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Adam Deutschbauer (amdeutschbauer@lbl.gov).

Materials availabilityPlasmids, strains, and mutant libraries generated in this study are available from the lead contact with a completed Material Transfer Agreement (MTA).

Materials availability

Plasmids, strains, and mutant libraries generated in this study are available from the lead contact with a completed Material Transfer Agreement (MTA).

Data and code availabilityFitness data from the 523 successful experiments is available for comparative analysis in the Fitness Browser at https://fit.genomics.lbl.gov (archived at https://figshare.com/articles/dataset/The_Fitness_Browser_Genome-wide_mutant_fitness_data_for_diverse_bacteria_November_2020_release_/13172087/1). In addition, original fitness data (and all associated metadata) have been deposited to figshare: https://figshare.com/articles/dataset/RB-TnSeq_data_for_Bacteroides_thetaiotaomicron_VPI-5482/13171766/1. The software for RB-TnSeq is available at https://bitbucket.org/berkeleylab/feba. The code for generating the figure panels of Figures 7A-7C and 7E and associated statistics in the text is available as a part of a Code Ocean compute capsule (https://codeocean.com/capsule/3569674/tree/v1).

Data and code availability

Fitness data from the 523 successful experiments is available for comparative analysis in the Fitness Browser at https://fit.genomics.lbl.gov (archived at https://figshare.com/articles/dataset/The_Fitness_Browser_Genome-wide_mutant_fitness_data_for_diverse_bacteria_November_2020_release_/13172087/1). In addition, original fitness data (and all associated metadata) have been deposited to figshare: https://figshare.com/articles/dataset/RB-TnSeq_data_for_Bacteroides_thetaiotaomicron_VPI-5482/13171766/1. The software for RB-TnSeq is available at https://bitbucket.org/berkeleylab/feba. The code for generating the figure panels of Figures 7A-7C and 7E and associated statistics in the text is available as a part of a Code Ocean compute capsule (https://codeocean.com/capsule/3569674/tree/v1).

EXPERIMENTAL MODEL AND SUBJECT DETAILSBacterial strains, plasmids, and oligonucleotides used in this study are listed in the Key Resources Table.Male germ-free (GF) C57BL/6J mice were maintained within the UCSF Gnotobiotic Core Facility (https://gnotobiotics.ucsf.edu) and fed ad libitum autoclaved standard chow diet (Lab Diet 5021). The mice were housed in a single gnotobiotic isolator for the duration of the bacterial mutant library colonization experiment. All mouse experiments were approved by the University of California, San Francisco (UCSF) Institutional Animal Care and Use Committee and performed accordingly.

EXPERIMENTAL MODEL AND SUBJECT DETAILS

Bacterial strains, plasmids, and oligonucleotides used in this study are listed in the Key Resources Table.

Male germ-free (GF) C57BL/6J mice were maintained within the UCSF Gnotobiotic Core Facility (https://gnotobiotics.ucsf.edu) and fed ad libitum autoclaved standard chow diet (Lab Diet 5021). The mice were housed in a single gnotobiotic isolator for the duration of the bacterial mutant library colonization experiment. All mouse experiments were approved by the University of California, San Francisco (UCSF) Institutional Animal Care and Use Committee and performed accordingly.

METHOD DETAILSMedia, compounds and growth conditionsGrowth media were purchased from BD. In this study, we used many of the compounds from a previous investigation of bacterial mutant fitness (Price et al., 2018); most were purchased from Sigma-Aldrich. We also purchased and assayed additional compounds specifically relevant to B. thetaiotaomicron, such as polysaccharides and bile salts. The source for some compounds is listed in Table S7.We typically grew B. thetaiotaomicron in an anaerobic chamber at 37°C in one of two growth media: Brain Heart Infusion Supplemented (BHIS) medium using Bacto Brain Heart Infusion (BHI) as the base, or Varel-Bryant (VB) defined medium (Varel and Bryant, 1974). Media recipes are provided in Table S7. VB medium contains methionine, so vitamin B12 is not required.Construction of a barcoded transposon delivery vector for B. thetaiotaomicronWe previously described a mariner transposon delivery vector library pHLL255 (magic pool) containing hundreds of combinations of promoters and erythromycin drug markers (Liu et al., 2018). To identify a suitable vector for mutagenizing B. thetaiotaomicron, we first constructed a small mutant library by conjugating with an Escherichia coli strain carrying pHLL255. We performed TnSeq on this library to map transposon insertion locations and their associated DNA barcodes, enabling us to infer which vector design in the magic pool resulted in the mutation (Liu et al., 2018). Analysis of this library revealed that the optimal drug marker for mutagenizing B. thetaiotaomicron was the erythromycin resistance gene ermG (Goodman et al., 2009; Shoemaker et al., 2001), the optimal promoter for expressing the mariner transposase was the native rpoD (BT1311) promoter (Goodman et al., 2009), and the optimal promoter for expressing ermG was the native dnaE (BT2230) promoter.Although this vector design had the highest insertion efficiency relative to the other designs in the magic pool, it still resulted in strand bias whereby the insertion orientation of the transposon was more often in the same transcriptional orientation as the mutated gene. Thus, in addition to constructing the best but potentially suboptimal vector design from the magic pool results (pTGG45), we also constructed another vector (pTGG46) by replacing the dnaE promoter in pTGG45 with a cepA hybrid promoter from B. fragilis RBF-103 (Bayley et al., 2000). To determine which vector exhibited the best combination of high efficiency and low strand bias, we constructed two small mutant libraries with either pTGG45 or pTGG46. TnSeq analysis of these libraries revealed that both had high insertion efficiency, but while the strand bias was again significant for pTGG45 (60% of insertions were in the same orientation as transcription of the mutated gene), there was essentially no strand bias for pTGG46 (49.4%). Thus, we barcoded pTGG46 (pTGG46_NN1) with millions of random 20-nucleotide barcodes using Golden Gate assembly, as previously described (Liu et al., 2018).Transposon mutant library constructionWe carried out transposon mutagenesis by conjugating B. thetaiotaomicron recipient cells with E. coli donor cells carrying the bar-coded transposon vector library pTGG46_NN1 at 1:1 ratio. Specifically, we grew 10 mL of wild-type B. thetaiotaomicron VPI-5482 in BHIS overnight at 37°C. The next morning, we recovered a 2 mL freezer stock of strain AMD776 (E. coli WM3064 with pTGG46_NN1) in 50 mL LB supplemented with 50 μg/mL carbenicillin and 300 μM diaminopimelic acid (DAP) at 37°C. When the OD600 of the E. coli donor strain reached ~1.0, we harvested 15 OD600 units of the culture and washed the cells three times with fresh BHIS supplemented with DAP. Then, 15 OD600 units of B. thetaiotaomicron wild-type cells were harvested, mixed with the washed donor cells, and resuspended in a final volume of 60 μL with BHIS supplemented with DAP. The resuspension was spotted onto 0.45-μm gravimetric analysis membrane filters (Millipore) and incubated anaerobically overnight on BHIS agar plates supplemented with DAP at 37°C. The next day, the conjugation mixture was scraped from the membrane and resuspended in 6 mL fresh BHIS medium supplemented with 10 μg/mL erythromycin and plated at different dilutions on BHIS plates supplemented with 10 μg/mL erythromycin. Plates were incubated at 37°C anaerobically for 48 h to let visible colonies develop. We then pooled ~250,000 colonies and grew the pooled library anaerobically in liquid BHIS supplemented with 10 μg/mL erythromycin and 200 μg/mL gentamicin (to further select against the E. coli conjugation donor strain as B. thetaiotaomicron is naturally resistant to gentamicin). When the culture reached saturation, we made multiple, single-use glycerol stocks of the final library and extracted genomic DNA for TnSeq analysis. To map the genomic locations of the transposon insertions and link these insertions to their associated DNA barcodes, we used our previously described TnSeq protocol (Wetmore et al., 2015). To map insertions in BT2158, we relaxed our mapping stringency and only required one nucleotide difference relative to BT4448 (BT2158 and BT4448 are 98% identical with only 30 mismatches over 1,476 nucleotides). The final mutant library was named Btheta_ML6.In vitro genome-wide mutant fitness assaysWe performed genome-wide mutant fitness assays as described previously (Price et al., 2018). For B. thetaiotaomicron, we thawed an aliquot of the full transposon mutant library, inoculated the entire aliquot into 50 mL of BHIS supplemented with 10 μg/mL erythromycin, and grew the library to mid-log phase. We then collected 6 cell pellets of ~1.0 OD600 unit each (the “Time0” sample). We used the remaining cells to inoculate competitive growth assays. Nearly all fitness assays were performed in 1.2 mL of growth medium in either a 24-well transparent microplate (Greiner) or in a 96-deepwell plate (Costar) at a starting OD600 of 0.02. For carbon and nitrogen source assays in VB, we washed the cells in VB lacking these nutrients two times. We grew cultures until they reached stationary phase, and then collected cell pellets (the “Condition” sample). We also performed a few “survival” experiments in which we incubated the mutant library aerobically for different amounts of time. After this incubation, we inoculated cells into fresh BHIS to select for cells that survived the stress. We extracted genomic DNA from the TimeO and Condition samples in a 96-well microplate format with a QIAamp 96 DNA Qiacube HT kit (QIAGEN). In addition, we performed genome-wide fitness assays using previously described E. vietnamensis KMM6221 and Pedobacter sp. GW460-11-11-14-LB5 barcoded transposon mutant libraries (Price et al., 2018) with the carbon sources substrates lactulose, maltitol, methyl-β-D-galactopyranoside, and lactitol.We performed barcode sequencing (BarSeq) as previously described (Price et al., 2018, 2019). For most experiments, we performed BarSeq with indexed P2 oligos and a mix of non-indexed P1 oligos with variable lengths of Ns (2-5) to stagger the reads. For some experiments, we used BarSeq oligos with both P1 and P2 indexed to minimize the impact of incorrectly assigned indexes in Illumina HiSeq4000 runs (Sinha et al., 2017). Strain and gene fitness scores were calculated as previously described (Wetmore et al., 2015).Construction of deletion strains and individual mutant growth assaysWe constructed unmarked, in-frame gene deletions using a previously established approach (Mishra and Imlay, 2013). Growth curves of individual mutants were carried out in a 96-well plate with either an Epoch2 (BioTek) or Infinite F200 (Tecan) plate reader housed in an anaerobic chamber. Note that optical densities in microplates are typically lower than those in a standard cuvette. To calculate IC50 values for ΔBT1439 and the parent on vancomycin, we used the drc package in R (Ritz et al., 2015). Specifically, we used drm() with the LL.4() model and defined the response as the average OD600 at the time that uninhibited cells (no vancomycin) reached their maximum OD600 value.Growth and cell harvesting for RT-qPCR assay of BT3761 repression by arginineAn overnight culture of B. thetaiotaomicron VPI-5482 Δtdk in 5 mL of tryptone-yeast extract-glucose (TYG) broth was grown at 37°C in a temperature-controlled Model 2000 incubator (COY Lab Products) in an anaerobic chamber (custom build, COY Lab Products). This overnight culture was pelleted and washed in VB medium three times before being inoculated into several wells of VB (the control condition) and VB supplemented with 3 mM arginine (the experimental condition). Biological replicates were grown in a 2-mL 96-deepwell plate (PlateOne, USA Scientific, Cat. No. 1896-2110). The atmospheric gas was a blend of 85% nitrogen, 10% carbon dioxide, and 5% hydrogen (Praxair, Cat. No. BI-NICDHYC4). OD600 was monitored using an Epoch2 (BioTek) microplate reader inside the anaerobic chamber, and cultures were periodically diluted into fresh media to keep the OD600 below 0.05 so that cells remained in log phase. After ~24 h of growth, 1.5 mL were transferred into 3 mL of RNAprotect® Bacterial Reagent. The cell suspension was centrifuged for 30 min at 4,000g and 4°C in a Sorvall Legend XTR centrifuge (Thermo Scientific). The supernatant was removed, cell pellets were air-dried at room temperature for 5 min, and pellets were flash frozen in liquid nitrogen and stored at −80°C until RNA was extracted the next day.Growth and cell harvesting for RT-qPCR assay of potential polar effects of ΔsusBOvernight cultures of B. thetaiotaomicron VPI-5482 Δtdk (the parental control) and B. thetaiotaomicron VPI-5482 Δtdk ΔBT3703 (ΔsusB, the experimental strain) were grown in 5 mL VB with α-cyclodextrin as the sole carbon source at 37°C in a temperature-controlled incubator inside an anaerobic chamber. Cultures were then inoculated into separate wells of VB with α-cyclodextrin and grown in log phase via periodic dilution into fresh medium to keep OD600 < 0.05. Details of growth conditions (plasticware and other equipment) are the same as the arginine repression experiment above. After ~24 h of growth, 1 mL was transferred into 2 mL of RNAprotect® Bacterial Reagent and the cell suspension was centrifuged for 30 min at 4°C and 3,428g in a 5920R centrifuge (Eppendorf). The supernatant was removed, cell pellets were dried at room temperature for 5 min, and pellets were stored at −80°C (without flash freezing) until the RNA was extracted (within 1 week).RNA isolation and quality control for RT-qPCR experimentsRNA was isolated using the RNeasy® Mini Kit (QIAGEN, Cat. No. 74104) with an on-column DNase I digestion using the RNase-free DNase Set (QIAGEN, Cat. No. 79254). RNA isolation was performed following the kit handbook, with no modifications. We used 2-mercaptoethanol from Sigma-Aldrich (Cat. No. M3148-100ML)to make fresh RLT buffer (from the QIAGEN kit) for every RNA isolation experiment. Every sample was eluted into 50 μL of RNase-free water from the kit. RNA concentration and purity were quantified using UV-vis absorbance spectra measured on a NanoDrop One C (Thermo Scientific, software version 1.4.2). The yield (concentration of RNA) and purity (A260/A280, A260/A230) are reported for every sample in Table S6. After quantification, RNA samples were diluted to their final concentration using values reported from the UV-vis spectrophotometer and stored at −20°C. After using the RNA in RT-qPCR experiments, the integrity of RNA in the remaining volume of samples was quantified using a Fragment Analyzer (Advanced Analytical Technologies, Inc.) and a high-sensitivity RNA kit (15 nt) (Agilent, Cat. No. DNF-472-0500). The RNA Quality Number, as determined by PROsize v. 3.0.16, is reported in Table S6.Reaction conditions for RT-qPCR experimentsAll RT-qPCR experiments were performed using the iTaq Universal SYBR® Green One-Step Kit (Bio-Rad, Cat. No. 1725151). In every experiment, we used a 10 μL final volume with 4 μL of input RNA and 0.3 μM primer. For the arginine-repression experiment, 22 ng of total RNA were used. For ΔsusB polarity experiments, 100 ng of total RNA were used. The RT-qPCR reaction was performed according to the kit handbook instructions for Bio-Rad CFX96 machines, without modification. Briefly, a single-step incubation for 10 min at 50°C allowed for reverse transcription to proceed, followed by a single-step incubation at 95°C for 1 min to denature the DNA and activate the polymerase. These steps were followed by 40 cycles of a 10 s denaturation step (at 95°C) and a 20 s annealing and extension hybrid step (at 60°C). Finally, we performed a melt-curve analysis by incrementing from 65°C to 95°C in 0.5°C steps, at 5 s per step. Samples were analyzed in Hard-Shell PCR Plates, white shell/white well (Bio-Rad, Cat. No. HSP9655) and run on a Bio-RAD® CFX96 machine using Bio-Rad CFX Manager v. 3.1. In both experiments, three biological replicates were analyzed, with one technical replicate per sample (n = 6). At least one no-template control was run on the same plate for each primer set. A no-reverse transcriptase experiment was run for each sample using the experimental target primers (BT3761 for the arginine expression experiment, and susC and susD for the susB polarity experiment). We designed primers for the amplification of 15 reference target genes in B. thetaiotaomicron in addition to the 3 experimental targets and the traditional reference target of 16S rDNA (Table S6). We determined the amplification efficiency of all of these primer pairs using 10-fold RNA dilution experiments (Table S6). We also found the best combination of reference genes for both experiments by running preliminary experiments using a large set of reference genes. A set of reference genes were chosen that exhibited high stability between experiments and controls and that were distributed across multiple biological processes.Targeted proteomicsFor carbon source experiments, we grew cultures of wild-type B. thetaiotaomicron in VB with 20 mM D-trehalose, D-raffinose, or D-glucose overnight to OD600 values between 1.0 and 1.5. For bile salt experiments, we grew wild-type B. thetaiotaomicron in VB with 20 mM D-glucose supplemented with 0, 0.125, 0.25, or 0.5 mg/mL bile salts for 20 h to OD600 values between 0.5 and 0.9. For all experiments, we harvested 2 OD600 units of cells and washed two times with 10 mM NH4HCO3 before extracting total protein using a methanol-chloroform precipitation method (Chen et al., 2019a). Protein concentration was determined using the Bio-Rad DC protein assay (Lot #5000112) following the manufacturer’s protocol, and the same amount of proteins was subjected to standard reduction, alkylation, and trypsin digestion procedures. A targeted selected reaction monitoring (SRM) method was developed to quantify unique peptides of proteins using an in-house constructed B. thetaiotaomicron spectral library (Chen et al., 2019b).SRM targeted proteomic assays were performed on an Agilent 6460 QQQ mass spectrometer system coupled to an Agilent 1290 UHPLC system (Agilent Technologies, Santa Clara, CA). Twenty micrograms of peptides from each sample were separated on an Ascentis Express Peptide C18 column (2.7-mm particle size, 160-Å pore size, 5-cm length × 2.1-mm inside diameter (ID), coupled to a 5-mm × 2.1-mm ID guard column with the same particle and pore size, operating at 60°C; Sigma-Aldrich) operating at a flow rate of 0.4 mL/min via the following gradient: initial conditions were 98% solvent A (0.1% formic acid), 2% solvent B (99.9% acetonitrile, 0.1% formic acid). Solvent B was increased to 40% over 5 min, and then to 80% over 0.5 min, and held for 2 min at a flow rate of 0.6 mL/min, followed by a ramp back down to 2% solvent B over 0.5 min where it was held for 1 min to re-equilibrate the column to original conditions. The eluted peptides were ionized via an Agilent Jet Stream ESI source operating in positive-ion mode with the following source parameters: gas temperature = 250°C, gas flow = 13 L/min, nebulizer pressure = 35 psi, sheath gas temperature = 250°C, sheath gas flow = 11 L/min, capillary voltage = 3500 V, nozzle voltage = 0 V. Data were acquired using Agilent MassHunter v. B.08.02. Acquired SRM data were analyzed by Skyline v. 3.70 (MacCoss Lab Software).Production of 3-keto-trehaloseSingle colonies of Agrobacterium rhizogenes K599 were inoculated in a minimal medium (MM) from Kurowski and Darbyshire (1978) with 2% (w/v) trehalose as a carbon source and supplemented with 0.5 g/L yeast extract, and grown overnight at 30°C shaking at 250 rpm. Cells were then harvested by centrifugation, washed twice with 0.1 M phosphate-buffer (pH 7.0) and inoculated into 50 mL of MM+2% trehalose without yeast extract. The next day, an aliquot of this culture in MM was used to inoculate 200 mL of MM+2% trehalose (1:20 ratio). Samples were removed periodically, centrifuged, and filter-sterilized to obtain cell-free supernatants. Consumption of trehalose was monitored using the Trehalose Assay kit (Megazyme) and the formation of keto-sugar was determined by the alkaline method from Fukui and Hayano (1969). The cell-free 3-keto-trehalose-containing culture supernatant used for fitness and enzymatic assays was obtained after 29 h of growth and contained 17.4 mM of trehalose and 7.9 mM 3-keto-trehalose.Expression and purification of recombinant BT2157The sequence encoding BT2157 was PCR-amplified from genomic DNA, cloned into a linearized pET32a plasmid using Gibson Assembly, and transformed into E. coli chemically competent DH5-α cells (New England Biolabs). The plasmid sequence was verified by Sanger sequencing and used to transform the production strain E. coli BL21 DE3. For protein production, E. coli BL21 DE3 cells expressing N-terminal His-tagged BT2157 were inoculated from single colonies into LB with 100 μg/mL ampicillin and grown at 37°C overnight. Production cultures were started by inoculating 10 mL of pre-cultures into 1 L of LB with 100 μg/mL ampicillin. After growth at 37°C with agitation until OD600~0.5, protein expression was induced by the addition of 0.15 mM isopropyl β-D-1 thiogalactopyranoside (IPTG) and cultures were incubated at 37°C for 3 h. Cells were pelleted by centrifugation at 8000g for 15 min, resuspended in 60 mL of lysis buffer (50 mM potassium phosphate, 300 mM NaCl, 10 mM imidazole, pH 7.8), and stored at −80°C. For protein purification, resuspended cells were thawed on ice and lysed by sonication. Cell debris and unbroken cells were removed by centrifugation at 10,000 rcf. for 20 min at 6°C. All purification steps were performed using ice-cold buffers and reagents. Protein purification was performed as previously described (Liou et al., 2020), except with a HisTrap FF column (GE Healthcare) replacing the free resin.In vitro endpoint assays with recombinant BT2157In vitro endpoint assays with purified BT2157 consisted of 50 μL of cell-free A. rhizogenes culture supernatant containing 7.9 mM 3-keto-trehalose, 50 μL of purified protein, and reaction buffer (50 mM potassium buffer, pH 7) up to a total volume of 1 mL. Each reaction contained final concentrations of 395 μM 3-keto-trehalose and 28.8 μM his-tagged BT2157. Reactions were initiated by the addition of substrate and incubated at 30°C. Reactions were interrupted by the addition of 3 mL of 0.1 mM NaOH, and after 3 min of incubation, the absorbance at 340 nm was measured for each sample to determine keto-trehalose concentration (Fukui and Hayano, 1969).Gnotobiotic mouse experimentsEight GF male 7-week-old C57BL/6J mice were housed in a single gnotobiotic isolator for the duration of the experiment, fed a standard irradiated diet ad libitum, split into 3 cages (n = 2-3 mice/cage), and allowed to acclimate for 3 days prior to colonization. The inoculum was prepared by thawing a 1 mL aliquot of the B. thetaiotaomicron transposon library, transferring the entire aliquot into 50 mL of BHIS supplemented with 10 μg/mL erythromycin, and incubating anaerobically at 37°C until the cells reached mid-log phase (16 h). Six bacterial cell pellets were collected as Time0 sample references. Each mouse was colonized by oral gavage with 200 μL of the B. thetaiotaomicron transposon library. Stool samples were collected on days 2, 4, and 7 post-colonization to assess longitudinal shifts in mutant abundance using BarSeq. Mice were monitored and weighed daily. Genomic DNA from the Time0 cells and stool samples collected over time was extracted using the ZymoBIOMICS 96 MagBead DNA Kit (D4302) and followed by BarSeq, as described above.

METHOD DETAILS

Media, compounds and growth conditionsGrowth media were purchased from BD. In this study, we used many of the compounds from a previous investigation of bacterial mutant fitness (Price et al., 2018); most were purchased from Sigma-Aldrich. We also purchased and assayed additional compounds specifically relevant to B. thetaiotaomicron, such as polysaccharides and bile salts. The source for some compounds is listed in Table S7.We typically grew B. thetaiotaomicron in an anaerobic chamber at 37°C in one of two growth media: Brain Heart Infusion Supplemented (BHIS) medium using Bacto Brain Heart Infusion (BHI) as the base, or Varel-Bryant (VB) defined medium (Varel and Bryant, 1974). Media recipes are provided in Table S7. VB medium contains methionine, so vitamin B12 is not required.

Media, compounds and growth conditions

Growth media were purchased from BD. In this study, we used many of the compounds from a previous investigation of bacterial mutant fitness (Price et al., 2018); most were purchased from Sigma-Aldrich. We also purchased and assayed additional compounds specifically relevant to B. thetaiotaomicron, such as polysaccharides and bile salts. The source for some compounds is listed in Table S7.

We typically grew B. thetaiotaomicron in an anaerobic chamber at 37°C in one of two growth media: Brain Heart Infusion Supplemented (BHIS) medium using Bacto Brain Heart Infusion (BHI) as the base, or Varel-Bryant (VB) defined medium (Varel and Bryant, 1974). Media recipes are provided in Table S7. VB medium contains methionine, so vitamin B12 is not required.

Construction of a barcoded transposon delivery vector for B. thetaiotaomicronWe previously described a mariner transposon delivery vector library pHLL255 (magic pool) containing hundreds of combinations of promoters and erythromycin drug markers (Liu et al., 2018). To identify a suitable vector for mutagenizing B. thetaiotaomicron, we first constructed a small mutant library by conjugating with an Escherichia coli strain carrying pHLL255. We performed TnSeq on this library to map transposon insertion locations and their associated DNA barcodes, enabling us to infer which vector design in the magic pool resulted in the mutation (Liu et al., 2018). Analysis of this library revealed that the optimal drug marker for mutagenizing B. thetaiotaomicron was the erythromycin resistance gene ermG (Goodman et al., 2009; Shoemaker et al., 2001), the optimal promoter for expressing the mariner transposase was the native rpoD (BT1311) promoter (Goodman et al., 2009), and the optimal promoter for expressing ermG was the native dnaE (BT2230) promoter.Although this vector design had the highest insertion efficiency relative to the other designs in the magic pool, it still resulted in strand bias whereby the insertion orientation of the transposon was more often in the same transcriptional orientation as the mutated gene. Thus, in addition to constructing the best but potentially suboptimal vector design from the magic pool results (pTGG45), we also constructed another vector (pTGG46) by replacing the dnaE promoter in pTGG45 with a cepA hybrid promoter from B. fragilis RBF-103 (Bayley et al., 2000). To determine which vector exhibited the best combination of high efficiency and low strand bias, we constructed two small mutant libraries with either pTGG45 or pTGG46. TnSeq analysis of these libraries revealed that both had high insertion efficiency, but while the strand bias was again significant for pTGG45 (60% of insertions were in the same orientation as transcription of the mutated gene), there was essentially no strand bias for pTGG46 (49.4%). Thus, we barcoded pTGG46 (pTGG46_NN1) with millions of random 20-nucleotide barcodes using Golden Gate assembly, as previously described (Liu et al., 2018).

Construction of a barcoded transposon delivery vector for B. thetaiotaomicron

We previously described a mariner transposon delivery vector library pHLL255 (magic pool) containing hundreds of combinations of promoters and erythromycin drug markers (Liu et al., 2018). To identify a suitable vector for mutagenizing B. thetaiotaomicron, we first constructed a small mutant library by conjugating with an Escherichia coli strain carrying pHLL255. We performed TnSeq on this library to map transposon insertion locations and their associated DNA barcodes, enabling us to infer which vector design in the magic pool resulted in the mutation (Liu et al., 2018). Analysis of this library revealed that the optimal drug marker for mutagenizing B. thetaiotaomicron was the erythromycin resistance gene ermG (Goodman et al., 2009; Shoemaker et al., 2001), the optimal promoter for expressing the mariner transposase was the native rpoD (BT1311) promoter (Goodman et al., 2009), and the optimal promoter for expressing ermG was the native dnaE (BT2230) promoter.

Although this vector design had the highest insertion efficiency relative to the other designs in the magic pool, it still resulted in strand bias whereby the insertion orientation of the transposon was more often in the same transcriptional orientation as the mutated gene. Thus, in addition to constructing the best but potentially suboptimal vector design from the magic pool results (pTGG45), we also constructed another vector (pTGG46) by replacing the dnaE promoter in pTGG45 with a cepA hybrid promoter from B. fragilis RBF-103 (Bayley et al., 2000). To determine which vector exhibited the best combination of high efficiency and low strand bias, we constructed two small mutant libraries with either pTGG45 or pTGG46. TnSeq analysis of these libraries revealed that both had high insertion efficiency, but while the strand bias was again significant for pTGG45 (60% of insertions were in the same orientation as transcription of the mutated gene), there was essentially no strand bias for pTGG46 (49.4%). Thus, we barcoded pTGG46 (pTGG46_NN1) with millions of random 20-nucleotide barcodes using Golden Gate assembly, as previously described (Liu et al., 2018).

Transposon mutant library constructionWe carried out transposon mutagenesis by conjugating B. thetaiotaomicron recipient cells with E. coli donor cells carrying the bar-coded transposon vector library pTGG46_NN1 at 1:1 ratio. Specifically, we grew 10 mL of wild-type B. thetaiotaomicron VPI-5482 in BHIS overnight at 37°C. The next morning, we recovered a 2 mL freezer stock of strain AMD776 (E. coli WM3064 with pTGG46_NN1) in 50 mL LB supplemented with 50 μg/mL carbenicillin and 300 μM diaminopimelic acid (DAP) at 37°C. When the OD600 of the E. coli donor strain reached ~1.0, we harvested 15 OD600 units of the culture and washed the cells three times with fresh BHIS supplemented with DAP. Then, 15 OD600 units of B. thetaiotaomicron wild-type cells were harvested, mixed with the washed donor cells, and resuspended in a final volume of 60 μL with BHIS supplemented with DAP. The resuspension was spotted onto 0.45-μm gravimetric analysis membrane filters (Millipore) and incubated anaerobically overnight on BHIS agar plates supplemented with DAP at 37°C. The next day, the conjugation mixture was scraped from the membrane and resuspended in 6 mL fresh BHIS medium supplemented with 10 μg/mL erythromycin and plated at different dilutions on BHIS plates supplemented with 10 μg/mL erythromycin. Plates were incubated at 37°C anaerobically for 48 h to let visible colonies develop. We then pooled ~250,000 colonies and grew the pooled library anaerobically in liquid BHIS supplemented with 10 μg/mL erythromycin and 200 μg/mL gentamicin (to further select against the E. coli conjugation donor strain as B. thetaiotaomicron is naturally resistant to gentamicin). When the culture reached saturation, we made multiple, single-use glycerol stocks of the final library and extracted genomic DNA for TnSeq analysis. To map the genomic locations of the transposon insertions and link these insertions to their associated DNA barcodes, we used our previously described TnSeq protocol (Wetmore et al., 2015). To map insertions in BT2158, we relaxed our mapping stringency and only required one nucleotide difference relative to BT4448 (BT2158 and BT4448 are 98% identical with only 30 mismatches over 1,476 nucleotides). The final mutant library was named Btheta_ML6.

Transposon mutant library construction

We carried out transposon mutagenesis by conjugating B. thetaiotaomicron recipient cells with E. coli donor cells carrying the bar-coded transposon vector library pTGG46_NN1 at 1:1 ratio. Specifically, we grew 10 mL of wild-type B. thetaiotaomicron VPI-5482 in BHIS overnight at 37°C. The next morning, we recovered a 2 mL freezer stock of strain AMD776 (E. coli WM3064 with pTGG46_NN1) in 50 mL LB supplemented with 50 μg/mL carbenicillin and 300 μM diaminopimelic acid (DAP) at 37°C. When the OD600 of the E. coli donor strain reached ~1.0, we harvested 15 OD600 units of the culture and washed the cells three times with fresh BHIS supplemented with DAP. Then, 15 OD600 units of B. thetaiotaomicron wild-type cells were harvested, mixed with the washed donor cells, and resuspended in a final volume of 60 μL with BHIS supplemented with DAP. The resuspension was spotted onto 0.45-μm gravimetric analysis membrane filters (Millipore) and incubated anaerobically overnight on BHIS agar plates supplemented with DAP at 37°C. The next day, the conjugation mixture was scraped from the membrane and resuspended in 6 mL fresh BHIS medium supplemented with 10 μg/mL erythromycin and plated at different dilutions on BHIS plates supplemented with 10 μg/mL erythromycin. Plates were incubated at 37°C anaerobically for 48 h to let visible colonies develop. We then pooled ~250,000 colonies and grew the pooled library anaerobically in liquid BHIS supplemented with 10 μg/mL erythromycin and 200 μg/mL gentamicin (to further select against the E. coli conjugation donor strain as B. thetaiotaomicron is naturally resistant to gentamicin). When the culture reached saturation, we made multiple, single-use glycerol stocks of the final library and extracted genomic DNA for TnSeq analysis. To map the genomic locations of the transposon insertions and link these insertions to their associated DNA barcodes, we used our previously described TnSeq protocol (Wetmore et al., 2015). To map insertions in BT2158, we relaxed our mapping stringency and only required one nucleotide difference relative to BT4448 (BT2158 and BT4448 are 98% identical with only 30 mismatches over 1,476 nucleotides). The final mutant library was named Btheta_ML6.

In vitro genome-wide mutant fitness assaysWe performed genome-wide mutant fitness assays as described previously (Price et al., 2018). For B. thetaiotaomicron, we thawed an aliquot of the full transposon mutant library, inoculated the entire aliquot into 50 mL of BHIS supplemented with 10 μg/mL erythromycin, and grew the library to mid-log phase. We then collected 6 cell pellets of ~1.0 OD600 unit each (the “Time0” sample). We used the remaining cells to inoculate competitive growth assays. Nearly all fitness assays were performed in 1.2 mL of growth medium in either a 24-well transparent microplate (Greiner) or in a 96-deepwell plate (Costar) at a starting OD600 of 0.02. For carbon and nitrogen source assays in VB, we washed the cells in VB lacking these nutrients two times. We grew cultures until they reached stationary phase, and then collected cell pellets (the “Condition” sample). We also performed a few “survival” experiments in which we incubated the mutant library aerobically for different amounts of time. After this incubation, we inoculated cells into fresh BHIS to select for cells that survived the stress. We extracted genomic DNA from the TimeO and Condition samples in a 96-well microplate format with a QIAamp 96 DNA Qiacube HT kit (QIAGEN). In addition, we performed genome-wide fitness assays using previously described E. vietnamensis KMM6221 and Pedobacter sp. GW460-11-11-14-LB5 barcoded transposon mutant libraries (Price et al., 2018) with the carbon sources substrates lactulose, maltitol, methyl-β-D-galactopyranoside, and lactitol.We performed barcode sequencing (BarSeq) as previously described (Price et al., 2018, 2019). For most experiments, we performed BarSeq with indexed P2 oligos and a mix of non-indexed P1 oligos with variable lengths of Ns (2-5) to stagger the reads. For some experiments, we used BarSeq oligos with both P1 and P2 indexed to minimize the impact of incorrectly assigned indexes in Illumina HiSeq4000 runs (Sinha et al., 2017). Strain and gene fitness scores were calculated as previously described (Wetmore et al., 2015).

In vitro genome-wide mutant fitness assays

We performed genome-wide mutant fitness assays as described previously (Price et al., 2018). For B. thetaiotaomicron, we thawed an aliquot of the full transposon mutant library, inoculated the entire aliquot into 50 mL of BHIS supplemented with 10 μg/mL erythromycin, and grew the library to mid-log phase. We then collected 6 cell pellets of ~1.0 OD600 unit each (the “Time0” sample). We used the remaining cells to inoculate competitive growth assays. Nearly all fitness assays were performed in 1.2 mL of growth medium in either a 24-well transparent microplate (Greiner) or in a 96-deepwell plate (Costar) at a starting OD600 of 0.02. For carbon and nitrogen source assays in VB, we washed the cells in VB lacking these nutrients two times. We grew cultures until they reached stationary phase, and then collected cell pellets (the “Condition” sample). We also performed a few “survival” experiments in which we incubated the mutant library aerobically for different amounts of time. After this incubation, we inoculated cells into fresh BHIS to select for cells that survived the stress. We extracted genomic DNA from the TimeO and Condition samples in a 96-well microplate format with a QIAamp 96 DNA Qiacube HT kit (QIAGEN). In addition, we performed genome-wide fitness assays using previously described E. vietnamensis KMM6221 and Pedobacter sp. GW460-11-11-14-LB5 barcoded transposon mutant libraries (Price et al., 2018) with the carbon sources substrates lactulose, maltitol, methyl-β-D-galactopyranoside, and lactitol.

We performed barcode sequencing (BarSeq) as previously described (Price et al., 2018, 2019). For most experiments, we performed BarSeq with indexed P2 oligos and a mix of non-indexed P1 oligos with variable lengths of Ns (2-5) to stagger the reads. For some experiments, we used BarSeq oligos with both P1 and P2 indexed to minimize the impact of incorrectly assigned indexes in Illumina HiSeq4000 runs (Sinha et al., 2017). Strain and gene fitness scores were calculated as previously described (Wetmore et al., 2015).

Construction of deletion strains and individual mutant growth assaysWe constructed unmarked, in-frame gene deletions using a previously established approach (Mishra and Imlay, 2013). Growth curves of individual mutants were carried out in a 96-well plate with either an Epoch2 (BioTek) or Infinite F200 (Tecan) plate reader housed in an anaerobic chamber. Note that optical densities in microplates are typically lower than those in a standard cuvette. To calculate IC50 values for ΔBT1439 and the parent on vancomycin, we used the drc package in R (Ritz et al., 2015). Specifically, we used drm() with the LL.4() model and defined the response as the average OD600 at the time that uninhibited cells (no vancomycin) reached their maximum OD600 value.

Construction of deletion strains and individual mutant growth assays

We constructed unmarked, in-frame gene deletions using a previously established approach (Mishra and Imlay, 2013). Growth curves of individual mutants were carried out in a 96-well plate with either an Epoch2 (BioTek) or Infinite F200 (Tecan) plate reader housed in an anaerobic chamber. Note that optical densities in microplates are typically lower than those in a standard cuvette. To calculate IC50 values for ΔBT1439 and the parent on vancomycin, we used the drc package in R (Ritz et al., 2015). Specifically, we used drm() with the LL.4() model and defined the response as the average OD600 at the time that uninhibited cells (no vancomycin) reached their maximum OD600 value.

Growth and cell harvesting for RT-qPCR assay of BT3761 repression by arginineAn overnight culture of B. thetaiotaomicron VPI-5482 Δtdk in 5 mL of tryptone-yeast extract-glucose (TYG) broth was grown at 37°C in a temperature-controlled Model 2000 incubator (COY Lab Products) in an anaerobic chamber (custom build, COY Lab Products). This overnight culture was pelleted and washed in VB medium three times before being inoculated into several wells of VB (the control condition) and VB supplemented with 3 mM arginine (the experimental condition). Biological replicates were grown in a 2-mL 96-deepwell plate (PlateOne, USA Scientific, Cat. No. 1896-2110). The atmospheric gas was a blend of 85% nitrogen, 10% carbon dioxide, and 5% hydrogen (Praxair, Cat. No. BI-NICDHYC4). OD600 was monitored using an Epoch2 (BioTek) microplate reader inside the anaerobic chamber, and cultures were periodically diluted into fresh media to keep the OD600 below 0.05 so that cells remained in log phase. After ~24 h of growth, 1.5 mL were transferred into 3 mL of RNAprotect® Bacterial Reagent. The cell suspension was centrifuged for 30 min at 4,000g and 4°C in a Sorvall Legend XTR centrifuge (Thermo Scientific). The supernatant was removed, cell pellets were air-dried at room temperature for 5 min, and pellets were flash frozen in liquid nitrogen and stored at −80°C until RNA was extracted the next day.

Growth and cell harvesting for RT-qPCR assay of BT3761 repression by arginine

An overnight culture of B. thetaiotaomicron VPI-5482 Δtdk in 5 mL of tryptone-yeast extract-glucose (TYG) broth was grown at 37°C in a temperature-controlled Model 2000 incubator (COY Lab Products) in an anaerobic chamber (custom build, COY Lab Products). This overnight culture was pelleted and washed in VB medium three times before being inoculated into several wells of VB (the control condition) and VB supplemented with 3 mM arginine (the experimental condition). Biological replicates were grown in a 2-mL 96-deepwell plate (PlateOne, USA Scientific, Cat. No. 1896-2110). The atmospheric gas was a blend of 85% nitrogen, 10% carbon dioxide, and 5% hydrogen (Praxair, Cat. No. BI-NICDHYC4). OD600 was monitored using an Epoch2 (BioTek) microplate reader inside the anaerobic chamber, and cultures were periodically diluted into fresh media to keep the OD600 below 0.05 so that cells remained in log phase. After ~24 h of growth, 1.5 mL were transferred into 3 mL of RNAprotect® Bacterial Reagent. The cell suspension was centrifuged for 30 min at 4,000g and 4°C in a Sorvall Legend XTR centrifuge (Thermo Scientific). The supernatant was removed, cell pellets were air-dried at room temperature for 5 min, and pellets were flash frozen in liquid nitrogen and stored at −80°C until RNA was extracted the next day.

Growth and cell harvesting for RT-qPCR assay of potential polar effects of ΔsusBOvernight cultures of B. thetaiotaomicron VPI-5482 Δtdk (the parental control) and B. thetaiotaomicron VPI-5482 Δtdk ΔBT3703 (ΔsusB, the experimental strain) were grown in 5 mL VB with α-cyclodextrin as the sole carbon source at 37°C in a temperature-controlled incubator inside an anaerobic chamber. Cultures were then inoculated into separate wells of VB with α-cyclodextrin and grown in log phase via periodic dilution into fresh medium to keep OD600 < 0.05. Details of growth conditions (plasticware and other equipment) are the same as the arginine repression experiment above. After ~24 h of growth, 1 mL was transferred into 2 mL of RNAprotect® Bacterial Reagent and the cell suspension was centrifuged for 30 min at 4°C and 3,428g in a 5920R centrifuge (Eppendorf). The supernatant was removed, cell pellets were dried at room temperature for 5 min, and pellets were stored at −80°C (without flash freezing) until the RNA was extracted (within 1 week).

Growth and cell harvesting for RT-qPCR assay of potential polar effects of ΔsusB

Overnight cultures of B. thetaiotaomicron VPI-5482 Δtdk (the parental control) and B. thetaiotaomicron VPI-5482 Δtdk ΔBT3703 (ΔsusB, the experimental strain) were grown in 5 mL VB with α-cyclodextrin as the sole carbon source at 37°C in a temperature-controlled incubator inside an anaerobic chamber. Cultures were then inoculated into separate wells of VB with α-cyclodextrin and grown in log phase via periodic dilution into fresh medium to keep OD600 < 0.05. Details of growth conditions (plasticware and other equipment) are the same as the arginine repression experiment above. After ~24 h of growth, 1 mL was transferred into 2 mL of RNAprotect® Bacterial Reagent and the cell suspension was centrifuged for 30 min at 4°C and 3,428g in a 5920R centrifuge (Eppendorf). The supernatant was removed, cell pellets were dried at room temperature for 5 min, and pellets were stored at −80°C (without flash freezing) until the RNA was extracted (within 1 week).

RNA isolation and quality control for RT-qPCR experimentsRNA was isolated using the RNeasy® Mini Kit (QIAGEN, Cat. No. 74104) with an on-column DNase I digestion using the RNase-free DNase Set (QIAGEN, Cat. No. 79254). RNA isolation was performed following the kit handbook, with no modifications. We used 2-mercaptoethanol from Sigma-Aldrich (Cat. No. M3148-100ML)to make fresh RLT buffer (from the QIAGEN kit) for every RNA isolation experiment. Every sample was eluted into 50 μL of RNase-free water from the kit. RNA concentration and purity were quantified using UV-vis absorbance spectra measured on a NanoDrop One C (Thermo Scientific, software version 1.4.2). The yield (concentration of RNA) and purity (A260/A280, A260/A230) are reported for every sample in Table S6. After quantification, RNA samples were diluted to their final concentration using values reported from the UV-vis spectrophotometer and stored at −20°C. After using the RNA in RT-qPCR experiments, the integrity of RNA in the remaining volume of samples was quantified using a Fragment Analyzer (Advanced Analytical Technologies, Inc.) and a high-sensitivity RNA kit (15 nt) (Agilent, Cat. No. DNF-472-0500). The RNA Quality Number, as determined by PROsize v. 3.0.16, is reported in Table S6.

RNA isolation and quality control for RT-qPCR experiments

RNA was isolated using the RNeasy® Mini Kit (QIAGEN, Cat. No. 74104) with an on-column DNase I digestion using the RNase-free DNase Set (QIAGEN, Cat. No. 79254). RNA isolation was performed following the kit handbook, with no modifications. We used 2-mercaptoethanol from Sigma-Aldrich (Cat. No. M3148-100ML)to make fresh RLT buffer (from the QIAGEN kit) for every RNA isolation experiment. Every sample was eluted into 50 μL of RNase-free water from the kit. RNA concentration and purity were quantified using UV-vis absorbance spectra measured on a NanoDrop One C (Thermo Scientific, software version 1.4.2). The yield (concentration of RNA) and purity (A260/A280, A260/A230) are reported for every sample in Table S6. After quantification, RNA samples were diluted to their final concentration using values reported from the UV-vis spectrophotometer and stored at −20°C. After using the RNA in RT-qPCR experiments, the integrity of RNA in the remaining volume of samples was quantified using a Fragment Analyzer (Advanced Analytical Technologies, Inc.) and a high-sensitivity RNA kit (15 nt) (Agilent, Cat. No. DNF-472-0500). The RNA Quality Number, as determined by PROsize v. 3.0.16, is reported in Table S6.

Reaction conditions for RT-qPCR experimentsAll RT-qPCR experiments were performed using the iTaq Universal SYBR® Green One-Step Kit (Bio-Rad, Cat. No. 1725151). In every experiment, we used a 10 μL final volume with 4 μL of input RNA and 0.3 μM primer. For the arginine-repression experiment, 22 ng of total RNA were used. For ΔsusB polarity experiments, 100 ng of total RNA were used. The RT-qPCR reaction was performed according to the kit handbook instructions for Bio-Rad CFX96 machines, without modification. Briefly, a single-step incubation for 10 min at 50°C allowed for reverse transcription to proceed, followed by a single-step incubation at 95°C for 1 min to denature the DNA and activate the polymerase. These steps were followed by 40 cycles of a 10 s denaturation step (at 95°C) and a 20 s annealing and extension hybrid step (at 60°C). Finally, we performed a melt-curve analysis by incrementing from 65°C to 95°C in 0.5°C steps, at 5 s per step. Samples were analyzed in Hard-Shell PCR Plates, white shell/white well (Bio-Rad, Cat. No. HSP9655) and run on a Bio-RAD® CFX96 machine using Bio-Rad CFX Manager v. 3.1. In both experiments, three biological replicates were analyzed, with one technical replicate per sample (n = 6). At least one no-template control was run on the same plate for each primer set. A no-reverse transcriptase experiment was run for each sample using the experimental target primers (BT3761 for the arginine expression experiment, and susC and susD for the susB polarity experiment). We designed primers for the amplification of 15 reference target genes in B. thetaiotaomicron in addition to the 3 experimental targets and the traditional reference target of 16S rDNA (Table S6). We determined the amplification efficiency of all of these primer pairs using 10-fold RNA dilution experiments (Table S6). We also found the best combination of reference genes for both experiments by running preliminary experiments using a large set of reference genes. A set of reference genes were chosen that exhibited high stability between experiments and controls and that were distributed across multiple biological processes.

Reaction conditions for RT-qPCR experiments

All RT-qPCR experiments were performed using the iTaq Universal SYBR® Green One-Step Kit (Bio-Rad, Cat. No. 1725151). In every experiment, we used a 10 μL final volume with 4 μL of input RNA and 0.3 μM primer. For the arginine-repression experiment, 22 ng of total RNA were used. For ΔsusB polarity experiments, 100 ng of total RNA were used. The RT-qPCR reaction was performed according to the kit handbook instructions for Bio-Rad CFX96 machines, without modification. Briefly, a single-step incubation for 10 min at 50°C allowed for reverse transcription to proceed, followed by a single-step incubation at 95°C for 1 min to denature the DNA and activate the polymerase. These steps were followed by 40 cycles of a 10 s denaturation step (at 95°C) and a 20 s annealing and extension hybrid step (at 60°C). Finally, we performed a melt-curve analysis by incrementing from 65°C to 95°C in 0.5°C steps, at 5 s per step. Samples were analyzed in Hard-Shell PCR Plates, white shell/white well (Bio-Rad, Cat. No. HSP9655) and run on a Bio-RAD® CFX96 machine using Bio-Rad CFX Manager v. 3.1. In both experiments, three biological replicates were analyzed, with one technical replicate per sample (n = 6). At least one no-template control was run on the same plate for each primer set. A no-reverse transcriptase experiment was run for each sample using the experimental target primers (BT3761 for the arginine expression experiment, and susC and susD for the susB polarity experiment). We designed primers for the amplification of 15 reference target genes in B. thetaiotaomicron in addition to the 3 experimental targets and the traditional reference target of 16S rDNA (Table S6). We determined the amplification efficiency of all of these primer pairs using 10-fold RNA dilution experiments (Table S6). We also found the best combination of reference genes for both experiments by running preliminary experiments using a large set of reference genes. A set of reference genes were chosen that exhibited high stability between experiments and controls and that were distributed across multiple biological processes.

Targeted proteomicsFor carbon source experiments, we grew cultures of wild-type B. thetaiotaomicron in VB with 20 mM D-trehalose, D-raffinose, or D-glucose overnight to OD600 values between 1.0 and 1.5. For bile salt experiments, we grew wild-type B. thetaiotaomicron in VB with 20 mM D-glucose supplemented with 0, 0.125, 0.25, or 0.5 mg/mL bile salts for 20 h to OD600 values between 0.5 and 0.9. For all experiments, we harvested 2 OD600 units of cells and washed two times with 10 mM NH4HCO3 before extracting total protein using a methanol-chloroform precipitation method (Chen et al., 2019a). Protein concentration was determined using the Bio-Rad DC protein assay (Lot #5000112) following the manufacturer’s protocol, and the same amount of proteins was subjected to standard reduction, alkylation, and trypsin digestion procedures. A targeted selected reaction monitoring (SRM) method was developed to quantify unique peptides of proteins using an in-house constructed B. thetaiotaomicron spectral library (Chen et al., 2019b).SRM targeted proteomic assays were performed on an Agilent 6460 QQQ mass spectrometer system coupled to an Agilent 1290 UHPLC system (Agilent Technologies, Santa Clara, CA). Twenty micrograms of peptides from each sample were separated on an Ascentis Express Peptide C18 column (2.7-mm particle size, 160-Å pore size, 5-cm length × 2.1-mm inside diameter (ID), coupled to a 5-mm × 2.1-mm ID guard column with the same particle and pore size, operating at 60°C; Sigma-Aldrich) operating at a flow rate of 0.4 mL/min via the following gradient: initial conditions were 98% solvent A (0.1% formic acid), 2% solvent B (99.9% acetonitrile, 0.1% formic acid). Solvent B was increased to 40% over 5 min, and then to 80% over 0.5 min, and held for 2 min at a flow rate of 0.6 mL/min, followed by a ramp back down to 2% solvent B over 0.5 min where it was held for 1 min to re-equilibrate the column to original conditions. The eluted peptides were ionized via an Agilent Jet Stream ESI source operating in positive-ion mode with the following source parameters: gas temperature = 250°C, gas flow = 13 L/min, nebulizer pressure = 35 psi, sheath gas temperature = 250°C, sheath gas flow = 11 L/min, capillary voltage = 3500 V, nozzle voltage = 0 V. Data were acquired using Agilent MassHunter v. B.08.02. Acquired SRM data were analyzed by Skyline v. 3.70 (MacCoss Lab Software).

Targeted proteomics

For carbon source experiments, we grew cultures of wild-type B. thetaiotaomicron in VB with 20 mM D-trehalose, D-raffinose, or D-glucose overnight to OD600 values between 1.0 and 1.5. For bile salt experiments, we grew wild-type B. thetaiotaomicron in VB with 20 mM D-glucose supplemented with 0, 0.125, 0.25, or 0.5 mg/mL bile salts for 20 h to OD600 values between 0.5 and 0.9. For all experiments, we harvested 2 OD600 units of cells and washed two times with 10 mM NH4HCO3 before extracting total protein using a methanol-chloroform precipitation method (Chen et al., 2019a). Protein concentration was determined using the Bio-Rad DC protein assay (Lot #5000112) following the manufacturer’s protocol, and the same amount of proteins was subjected to standard reduction, alkylation, and trypsin digestion procedures. A targeted selected reaction monitoring (SRM) method was developed to quantify unique peptides of proteins using an in-house constructed B. thetaiotaomicron spectral library (Chen et al., 2019b).

SRM targeted proteomic assays were performed on an Agilent 6460 QQQ mass spectrometer system coupled to an Agilent 1290 UHPLC system (Agilent Technologies, Santa Clara, CA). Twenty micrograms of peptides from each sample were separated on an Ascentis Express Peptide C18 column (2.7-mm particle size, 160-Å pore size, 5-cm length × 2.1-mm inside diameter (ID), coupled to a 5-mm × 2.1-mm ID guard column with the same particle and pore size, operating at 60°C; Sigma-Aldrich) operating at a flow rate of 0.4 mL/min via the following gradient: initial conditions were 98% solvent A (0.1% formic acid), 2% solvent B (99.9% acetonitrile, 0.1% formic acid). Solvent B was increased to 40% over 5 min, and then to 80% over 0.5 min, and held for 2 min at a flow rate of 0.6 mL/min, followed by a ramp back down to 2% solvent B over 0.5 min where it was held for 1 min to re-equilibrate the column to original conditions. The eluted peptides were ionized via an Agilent Jet Stream ESI source operating in positive-ion mode with the following source parameters: gas temperature = 250°C, gas flow = 13 L/min, nebulizer pressure = 35 psi, sheath gas temperature = 250°C, sheath gas flow = 11 L/min, capillary voltage = 3500 V, nozzle voltage = 0 V. Data were acquired using Agilent MassHunter v. B.08.02. Acquired SRM data were analyzed by Skyline v. 3.70 (MacCoss Lab Software).

Production of 3-keto-trehaloseSingle colonies of Agrobacterium rhizogenes K599 were inoculated in a minimal medium (MM) from Kurowski and Darbyshire (1978) with 2% (w/v) trehalose as a carbon source and supplemented with 0.5 g/L yeast extract, and grown overnight at 30°C shaking at 250 rpm. Cells were then harvested by centrifugation, washed twice with 0.1 M phosphate-buffer (pH 7.0) and inoculated into 50 mL of MM+2% trehalose without yeast extract. The next day, an aliquot of this culture in MM was used to inoculate 200 mL of MM+2% trehalose (1:20 ratio). Samples were removed periodically, centrifuged, and filter-sterilized to obtain cell-free supernatants. Consumption of trehalose was monitored using the Trehalose Assay kit (Megazyme) and the formation of keto-sugar was determined by the alkaline method from Fukui and Hayano (1969). The cell-free 3-keto-trehalose-containing culture supernatant used for fitness and enzymatic assays was obtained after 29 h of growth and contained 17.4 mM of trehalose and 7.9 mM 3-keto-trehalose.

Production of 3-keto-trehalose

Single colonies of Agrobacterium rhizogenes K599 were inoculated in a minimal medium (MM) from Kurowski and Darbyshire (1978) with 2% (w/v) trehalose as a carbon source and supplemented with 0.5 g/L yeast extract, and grown overnight at 30°C shaking at 250 rpm. Cells were then harvested by centrifugation, washed twice with 0.1 M phosphate-buffer (pH 7.0) and inoculated into 50 mL of MM+2% trehalose without yeast extract. The next day, an aliquot of this culture in MM was used to inoculate 200 mL of MM+2% trehalose (1:20 ratio). Samples were removed periodically, centrifuged, and filter-sterilized to obtain cell-free supernatants. Consumption of trehalose was monitored using the Trehalose Assay kit (Megazyme) and the formation of keto-sugar was determined by the alkaline method from Fukui and Hayano (1969). The cell-free 3-keto-trehalose-containing culture supernatant used for fitness and enzymatic assays was obtained after 29 h of growth and contained 17.4 mM of trehalose and 7.9 mM 3-keto-trehalose.

Expression and purification of recombinant BT2157The sequence encoding BT2157 was PCR-amplified from genomic DNA, cloned into a linearized pET32a plasmid using Gibson Assembly, and transformed into E. coli chemically competent DH5-α cells (New England Biolabs). The plasmid sequence was verified by Sanger sequencing and used to transform the production strain E. coli BL21 DE3. For protein production, E. coli BL21 DE3 cells expressing N-terminal His-tagged BT2157 were inoculated from single colonies into LB with 100 μg/mL ampicillin and grown at 37°C overnight. Production cultures were started by inoculating 10 mL of pre-cultures into 1 L of LB with 100 μg/mL ampicillin. After growth at 37°C with agitation until OD600~0.5, protein expression was induced by the addition of 0.15 mM isopropyl β-D-1 thiogalactopyranoside (IPTG) and cultures were incubated at 37°C for 3 h. Cells were pelleted by centrifugation at 8000g for 15 min, resuspended in 60 mL of lysis buffer (50 mM potassium phosphate, 300 mM NaCl, 10 mM imidazole, pH 7.8), and stored at −80°C. For protein purification, resuspended cells were thawed on ice and lysed by sonication. Cell debris and unbroken cells were removed by centrifugation at 10,000 rcf. for 20 min at 6°C. All purification steps were performed using ice-cold buffers and reagents. Protein purification was performed as previously described (Liou et al., 2020), except with a HisTrap FF column (GE Healthcare) replacing the free resin.

Expression and purification of recombinant BT2157

The sequence encoding BT2157 was PCR-amplified from genomic DNA, cloned into a linearized pET32a plasmid using Gibson Assembly, and transformed into E. coli chemically competent DH5-α cells (New England Biolabs). The plasmid sequence was verified by Sanger sequencing and used to transform the production strain E. coli BL21 DE3. For protein production, E. coli BL21 DE3 cells expressing N-terminal His-tagged BT2157 were inoculated from single colonies into LB with 100 μg/mL ampicillin and grown at 37°C overnight. Production cultures were started by inoculating 10 mL of pre-cultures into 1 L of LB with 100 μg/mL ampicillin. After growth at 37°C with agitation until OD600~0.5, protein expression was induced by the addition of 0.15 mM isopropyl β-D-1 thiogalactopyranoside (IPTG) and cultures were incubated at 37°C for 3 h. Cells were pelleted by centrifugation at 8000g for 15 min, resuspended in 60 mL of lysis buffer (50 mM potassium phosphate, 300 mM NaCl, 10 mM imidazole, pH 7.8), and stored at −80°C. For protein purification, resuspended cells were thawed on ice and lysed by sonication. Cell debris and unbroken cells were removed by centrifugation at 10,000 rcf. for 20 min at 6°C. All purification steps were performed using ice-cold buffers and reagents. Protein purification was performed as previously described (Liou et al., 2020), except with a HisTrap FF column (GE Healthcare) replacing the free resin.

In vitro endpoint assays with recombinant BT2157In vitro endpoint assays with purified BT2157 consisted of 50 μL of cell-free A. rhizogenes culture supernatant containing 7.9 mM 3-keto-trehalose, 50 μL of purified protein, and reaction buffer (50 mM potassium buffer, pH 7) up to a total volume of 1 mL. Each reaction contained final concentrations of 395 μM 3-keto-trehalose and 28.8 μM his-tagged BT2157. Reactions were initiated by the addition of substrate and incubated at 30°C. Reactions were interrupted by the addition of 3 mL of 0.1 mM NaOH, and after 3 min of incubation, the absorbance at 340 nm was measured for each sample to determine keto-trehalose concentration (Fukui and Hayano, 1969).

In vitro endpoint assays with recombinant BT2157

In vitro endpoint assays with purified BT2157 consisted of 50 μL of cell-free A. rhizogenes culture supernatant containing 7.9 mM 3-keto-trehalose, 50 μL of purified protein, and reaction buffer (50 mM potassium buffer, pH 7) up to a total volume of 1 mL. Each reaction contained final concentrations of 395 μM 3-keto-trehalose and 28.8 μM his-tagged BT2157. Reactions were initiated by the addition of substrate and incubated at 30°C. Reactions were interrupted by the addition of 3 mL of 0.1 mM NaOH, and after 3 min of incubation, the absorbance at 340 nm was measured for each sample to determine keto-trehalose concentration (Fukui and Hayano, 1969).

Gnotobiotic mouse experimentsEight GF male 7-week-old C57BL/6J mice were housed in a single gnotobiotic isolator for the duration of the experiment, fed a standard irradiated diet ad libitum, split into 3 cages (n = 2-3 mice/cage), and allowed to acclimate for 3 days prior to colonization. The inoculum was prepared by thawing a 1 mL aliquot of the B. thetaiotaomicron transposon library, transferring the entire aliquot into 50 mL of BHIS supplemented with 10 μg/mL erythromycin, and incubating anaerobically at 37°C until the cells reached mid-log phase (16 h). Six bacterial cell pellets were collected as Time0 sample references. Each mouse was colonized by oral gavage with 200 μL of the B. thetaiotaomicron transposon library. Stool samples were collected on days 2, 4, and 7 post-colonization to assess longitudinal shifts in mutant abundance using BarSeq. Mice were monitored and weighed daily. Genomic DNA from the Time0 cells and stool samples collected over time was extracted using the ZymoBIOMICS 96 MagBead DNA Kit (D4302) and followed by BarSeq, as described above.

Gnotobiotic mouse experiments

Eight GF male 7-week-old C57BL/6J mice were housed in a single gnotobiotic isolator for the duration of the experiment, fed a standard irradiated diet ad libitum, split into 3 cages (n = 2-3 mice/cage), and allowed to acclimate for 3 days prior to colonization. The inoculum was prepared by thawing a 1 mL aliquot of the B. thetaiotaomicron transposon library, transferring the entire aliquot into 50 mL of BHIS supplemented with 10 μg/mL erythromycin, and incubating anaerobically at 37°C until the cells reached mid-log phase (16 h). Six bacterial cell pellets were collected as Time0 sample references. Each mouse was colonized by oral gavage with 200 μL of the B. thetaiotaomicron transposon library. Stool samples were collected on days 2, 4, and 7 post-colonization to assess longitudinal shifts in mutant abundance using BarSeq. Mice were monitored and weighed daily. Genomic DNA from the Time0 cells and stool samples collected over time was extracted using the ZymoBIOMICS 96 MagBead DNA Kit (D4302) and followed by BarSeq, as described above.

QUANTIFICATION AND STATISTICAL ANALYSISFitness data analysisOverall, we attempted 812 genome-wide fitness assays in this study, of which 523 passed our metrics for internal consistency (Wetmore et al., 2015). We considered a gene fitness value in a single experiment to be statistically significant if ∣fitness∣ > 1 and ∣t∣ > 4, where t is fitness/estimated standard error. Across all 523 experiments × 4,055 genes, 31,717 gene fitness values (1.5%) had ~fitness~ > 1 and ∣t∣ > 4. By contrast, across 68 control comparisons of a Time0 sample to other Time0 samples from the same day, 135 gene fitness values (0.05%) met these thresholds. The estimated false discovery rate for a significant phenotype was 0.05%/1.5% = 3.3%. From these 523 experiments, we identified genes with a specific phenotype in only one or a handful of conditions using previously described criteria (Price et al., 2018): ∣fitness∣ > 1 and ∣t∣ > 5 and 95th percentile(∣fitness∣) < 1 and ∣fitness∣ > 95th percentile(∣fitness∣)+0.5. For the analysis of the total number of genes with a specific phenotype, we excluded 1 gene that had a specific phenotype in BHIS medium alone. A specific important phenotype has the same criteria as above except that the fitness score has to be negative. For the specific phenotype analysis of PUL genes on carbon sources (Table S4; Figure 2), we used a subset of 492 experiments (we excluded the mouse experiment, and carbon source experiments with lactulose, maltitol, and 3-keto-trehalose) (https://figshare.com/articles/Large-scale_genetic_characterization_of_the_human_gut_commensal_bacterium_Bacteroides_thetaiotaomicron/7742855/1). We identified pairs of genes with highly correlated patterns of phenotypes across all 523 successful experiments (cofitness) using the linear (Pearson) correlation and a threshold of 0.8 (Price et al., 2018). Most data analyses to infer the putative functions of genes were performed with the Fitness Browser (https://fit.genomics.lbl.gov).One caveat in most transposon mutagenesis studies is complications due to polarity, whereby insertion in upstream genes of an operon can disrupt the expression and function of downstream genes. Relatedly, it is also possible for the promoter of the drug resistance gene used to select mutants to drive expression of downstream genes (if the transposon is oriented in the same direction as the operon). The transposon that we used to construct our library only contains the drug marker cassette (promoter and gene) and the barcode region; there is no obvious transcriptional terminator. Thus, we expect that if a transposon lands in the same transcriptional orientation as the operon, then downstream genes will be overexpressed. To quantify the potential for polarity in our dataset, we identified instances in which fitness values calculated from plus-strand transposon insertions differed significantly from those calculated from minus-strand insertions. Specifically, we considered gene-condition phenotypes to potentially be due to polar effects if: (1) the absolute value of the fitness calculated using the standard approach with all insertions was > 1, (2) there were at least two strains in each orientation with sufficient data, (3) the average fitness on the two strands differed by ≥ 1, and (4) an unpaired t test on the two sets of fitness values gave p < 0.01. Among 80,737 gene-experiment pairs with ∣fitness∣ > 1 and ≥ 2 insertions on both strands, we identified 2,248 instances (3%) of potential polarity (Table S9), including 208 genes covered by ≥ 2 separate experiments. Thus, while polarity does appear to occur, the majority of significant phenotypes in our dataset are consistent regardless of transposon orientation.[REMOVED HYPERLINK FIELD]Data analysis for RT-qPCR experimentsCq values were determined using the Bio-Rad CFX Manager software running the Bio-RAD® CFX96 Real-Time System. Csv files exported by Bio-Rad CFX Manager were manually converted into qbase-format files and input into qbase+ v. 3.2 for analysis. All biological and technical replicates were treated as equivalent in the analysis. Amplification efficiencies of reference and target genes were calculated for every primer pair and used as part of the analysis. Final fold-changes and 95% confidence intervals were calculated from the CNRQ values and standard errors output by qbase+, with the assumption that measurement errors were log-normal. M and CV values for reference gene stability, along with Cq values of no-template controls and the difference in Cq values of target genes with and without reverse-transcriptase, are reported in Table S6.Data sources for enrichment analysisData on predicted amino acid biosynthesis genes in the B. thetaiotaomicron VPI-5482 genome were downloaded from the BioCyc database (https://biocyc.org/; Karp et al., 2019). The genes for predicted carbohydrate active enzyme (CAZymes) in B. thetaiotaomicron VPI-5482 were downloaded from the Carbohydrate Active Enzymes database (http://www.cazy.org/; Lombard et al., 2014).

QUANTIFICATION AND STATISTICAL ANALYSIS

Fitness data analysisOverall, we attempted 812 genome-wide fitness assays in this study, of which 523 passed our metrics for internal consistency (Wetmore et al., 2015). We considered a gene fitness value in a single experiment to be statistically significant if ∣fitness∣ > 1 and ∣t∣ > 4, where t is fitness/estimated standard error. Across all 523 experiments × 4,055 genes, 31,717 gene fitness values (1.5%) had ~fitness~ > 1 and ∣t∣ > 4. By contrast, across 68 control comparisons of a Time0 sample to other Time0 samples from the same day, 135 gene fitness values (0.05%) met these thresholds. The estimated false discovery rate for a significant phenotype was 0.05%/1.5% = 3.3%. From these 523 experiments, we identified genes with a specific phenotype in only one or a handful of conditions using previously described criteria (Price et al., 2018): ∣fitness∣ > 1 and ∣t∣ > 5 and 95th percentile(∣fitness∣) < 1 and ∣fitness∣ > 95th percentile(∣fitness∣)+0.5. For the analysis of the total number of genes with a specific phenotype, we excluded 1 gene that had a specific phenotype in BHIS medium alone. A specific important phenotype has the same criteria as above except that the fitness score has to be negative. For the specific phenotype analysis of PUL genes on carbon sources (Table S4; Figure 2), we used a subset of 492 experiments (we excluded the mouse experiment, and carbon source experiments with lactulose, maltitol, and 3-keto-trehalose) (https://figshare.com/articles/Large-scale_genetic_characterization_of_the_human_gut_commensal_bacterium_Bacteroides_thetaiotaomicron/7742855/1). We identified pairs of genes with highly correlated patterns of phenotypes across all 523 successful experiments (cofitness) using the linear (Pearson) correlation and a threshold of 0.8 (Price et al., 2018). Most data analyses to infer the putative functions of genes were performed with the Fitness Browser (https://fit.genomics.lbl.gov).One caveat in most transposon mutagenesis studies is complications due to polarity, whereby insertion in upstream genes of an operon can disrupt the expression and function of downstream genes. Relatedly, it is also possible for the promoter of the drug resistance gene used to select mutants to drive expression of downstream genes (if the transposon is oriented in the same direction as the operon). The transposon that we used to construct our library only contains the drug marker cassette (promoter and gene) and the barcode region; there is no obvious transcriptional terminator. Thus, we expect that if a transposon lands in the same transcriptional orientation as the operon, then downstream genes will be overexpressed. To quantify the potential for polarity in our dataset, we identified instances in which fitness values calculated from plus-strand transposon insertions differed significantly from those calculated from minus-strand insertions. Specifically, we considered gene-condition phenotypes to potentially be due to polar effects if: (1) the absolute value of the fitness calculated using the standard approach with all insertions was > 1, (2) there were at least two strains in each orientation with sufficient data, (3) the average fitness on the two strands differed by ≥ 1, and (4) an unpaired t test on the two sets of fitness values gave p < 0.01. Among 80,737 gene-experiment pairs with ∣fitness∣ > 1 and ≥ 2 insertions on both strands, we identified 2,248 instances (3%) of potential polarity (Table S9), including 208 genes covered by ≥ 2 separate experiments. Thus, while polarity does appear to occur, the majority of significant phenotypes in our dataset are consistent regardless of transposon orientation.[REMOVED HYPERLINK FIELD]

Fitness data analysis

Overall, we attempted 812 genome-wide fitness assays in this study, of which 523 passed our metrics for internal consistency (Wetmore et al., 2015). We considered a gene fitness value in a single experiment to be statistically significant if ∣fitness∣ > 1 and ∣t∣ > 4, where t is fitness/estimated standard error. Across all 523 experiments × 4,055 genes, 31,717 gene fitness values (1.5%) had ~fitness~ > 1 and ∣t∣ > 4. By contrast, across 68 control comparisons of a Time0 sample to other Time0 samples from the same day, 135 gene fitness values (0.05%) met these thresholds. The estimated false discovery rate for a significant phenotype was 0.05%/1.5% = 3.3%. From these 523 experiments, we identified genes with a specific phenotype in only one or a handful of conditions using previously described criteria (Price et al., 2018): ∣fitness∣ > 1 and ∣t∣ > 5 and 95th percentile(∣fitness∣) < 1 and ∣fitness∣ > 95th percentile(∣fitness∣)+0.5. For the analysis of the total number of genes with a specific phenotype, we excluded 1 gene that had a specific phenotype in BHIS medium alone. A specific important phenotype has the same criteria as above except that the fitness score has to be negative. For the specific phenotype analysis of PUL genes on carbon sources (Table S4; Figure 2), we used a subset of 492 experiments (we excluded the mouse experiment, and carbon source experiments with lactulose, maltitol, and 3-keto-trehalose) (https://figshare.com/articles/Large-scale_genetic_characterization_of_the_human_gut_commensal_bacterium_Bacteroides_thetaiotaomicron/7742855/1). We identified pairs of genes with highly correlated patterns of phenotypes across all 523 successful experiments (cofitness) using the linear (Pearson) correlation and a threshold of 0.8 (Price et al., 2018). Most data analyses to infer the putative functions of genes were performed with the Fitness Browser (https://fit.genomics.lbl.gov).

One caveat in most transposon mutagenesis studies is complications due to polarity, whereby insertion in upstream genes of an operon can disrupt the expression and function of downstream genes. Relatedly, it is also possible for the promoter of the drug resistance gene used to select mutants to drive expression of downstream genes (if the transposon is oriented in the same direction as the operon). The transposon that we used to construct our library only contains the drug marker cassette (promoter and gene) and the barcode region; there is no obvious transcriptional terminator. Thus, we expect that if a transposon lands in the same transcriptional orientation as the operon, then downstream genes will be overexpressed. To quantify the potential for polarity in our dataset, we identified instances in which fitness values calculated from plus-strand transposon insertions differed significantly from those calculated from minus-strand insertions. Specifically, we considered gene-condition phenotypes to potentially be due to polar effects if: (1) the absolute value of the fitness calculated using the standard approach with all insertions was > 1, (2) there were at least two strains in each orientation with sufficient data, (3) the average fitness on the two strands differed by ≥ 1, and (4) an unpaired t test on the two sets of fitness values gave p < 0.01. Among 80,737 gene-experiment pairs with ∣fitness∣ > 1 and ≥ 2 insertions on both strands, we identified 2,248 instances (3%) of potential polarity (Table S9), including 208 genes covered by ≥ 2 separate experiments. Thus, while polarity does appear to occur, the majority of significant phenotypes in our dataset are consistent regardless of transposon orientation.[REMOVED HYPERLINK FIELD]

Data analysis for RT-qPCR experimentsCq values were determined using the Bio-Rad CFX Manager software running the Bio-RAD® CFX96 Real-Time System. Csv files exported by Bio-Rad CFX Manager were manually converted into qbase-format files and input into qbase+ v. 3.2 for analysis. All biological and technical replicates were treated as equivalent in the analysis. Amplification efficiencies of reference and target genes were calculated for every primer pair and used as part of the analysis. Final fold-changes and 95% confidence intervals were calculated from the CNRQ values and standard errors output by qbase+, with the assumption that measurement errors were log-normal. M and CV values for reference gene stability, along with Cq values of no-template controls and the difference in Cq values of target genes with and without reverse-transcriptase, are reported in Table S6.

Data analysis for RT-qPCR experiments

Cq values were determined using the Bio-Rad CFX Manager software running the Bio-RAD® CFX96 Real-Time System. Csv files exported by Bio-Rad CFX Manager were manually converted into qbase-format files and input into qbase+ v. 3.2 for analysis. All biological and technical replicates were treated as equivalent in the analysis. Amplification efficiencies of reference and target genes were calculated for every primer pair and used as part of the analysis. Final fold-changes and 95% confidence intervals were calculated from the CNRQ values and standard errors output by qbase+, with the assumption that measurement errors were log-normal. M and CV values for reference gene stability, along with Cq values of no-template controls and the difference in Cq values of target genes with and without reverse-transcriptase, are reported in Table S6.

Data sources for enrichment analysisData on predicted amino acid biosynthesis genes in the B. thetaiotaomicron VPI-5482 genome were downloaded from the BioCyc database (https://biocyc.org/; Karp et al., 2019). The genes for predicted carbohydrate active enzyme (CAZymes) in B. thetaiotaomicron VPI-5482 were downloaded from the Carbohydrate Active Enzymes database (http://www.cazy.org/; Lombard et al., 2014).

Data sources for enrichment analysis

Data on predicted amino acid biosynthesis genes in the B. thetaiotaomicron VPI-5482 genome were downloaded from the BioCyc database (https://biocyc.org/; Karp et al., 2019). The genes for predicted carbohydrate active enzyme (CAZymes) in B. thetaiotaomicron VPI-5482 were downloaded from the Carbohydrate Active Enzymes database (http://www.cazy.org/; Lombard et al., 2014).

Supplementary Material1345678910

Supplementary Material
