---
pmid: '18354501'
title: 'Structural insight into the TFIIE-TFIIH interaction: TFIIE and p53 share the
  binding region on TFIIH.'
authors:
- Okuda M
- Tanaka A
- Satoh M
- Mizuta S
- Takazawa M
- Ohkuma Y
- Nishimura Y
journal: EMBO J
year: '2008'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC2275666
doi: 10.1038/emboj.2008.47
---

# Structural insight into the TFIIE-TFIIH interaction: TFIIE and p53 share the binding region on TFIIH.
**Authors:** Okuda M, Tanaka A, Satoh M, Mizuta S, Takazawa M, Ohkuma Y, Nishimura Y
**Journal:** EMBO J (2008)
**DOI:** [10.1038/emboj.2008.47](https://doi.org/10.1038/emboj.2008.47)
**PMC:** [PMC2275666](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2275666/)

## Abstract

1. EMBO J. 2008 Apr 9;27(7):1161-71. doi: 10.1038/emboj.2008.47. Epub 2008 Mar
20.

Structural insight into the TFIIE-TFIIH interaction: TFIIE and p53 share the 
binding region on TFIIH.

Okuda M(1), Tanaka A, Satoh M, Mizuta S, Takazawa M, Ohkuma Y, Nishimura Y.

Author information:
(1)Laboratory of Structural Biology, Graduate School of Supramolecular Biology, 
Yokohama City University, Yokohama, Japan.

RNA polymerase II and general transcription factors (GTFs) assemble on a 
promoter to form a transcription preinitiation complex (PIC). Among the GTFs, 
TFIIE recruits TFIIH to complete the PIC formation and regulates enzymatic 
activities of TFIIH. However, the mode of binding between TFIIE and TFIIH is 
poorly understood. Here, we demonstrate the specific binding of the C-terminal 
acidic domain (AC-D) of the human TFIIEalpha subunit to the pleckstrin homology 
domain (PH-D) of the human TFIIH p62 subunit and describe the solution 
structures of the free and PH-D-bound forms of AC-D. Although the flexible 
N-terminal acidic tail from AC-D wraps around PH-D, the core domain of AC-D also 
interacts with PH-D. AC-D employs an entirely novel binding mode, which differs 
from the amphipathic helix method used by many transcriptional activators. So 
the binding surface between PH-D and AC-D is much broader than the specific 
binding surface between PH-D and the p53 acidic fragments. From our in vitro 
studies, we demonstrate that this interaction could be a switch to replace p53 
with TFIIE on TFIIH in transcription.

DOI: 10.1038/emboj.2008.47
PMCID: PMC2275666
PMID: 18354501 [Indexed for MEDLINE]

## Full Text

IntroductionIn eukaryotes, transcription of protein-encoding genes is performed by RNA polymerase II (Pol II). Although it is a complex enzyme comprised of 12 subunits, Pol II alone is unable to accurately recognize promoters to initiate transcription. Transcription initiation by Pol II requires five proteins; TFIIB, TFIID, TFIIE, TFIIF and TFIIH collectively known as ‘general transcription factors (GTFs)' (Orphanides et al, 1996; Roeder, 1996). Pol II and GTFs converge on a promoter in a highly ordered manner to form the preinitiation complex (PIC). After the binding of TFIID to the TATA element located 30 nt upstream of the transcription initiation site (+1), TFIIB and TFIIF together with Pol II are recruited. TFIIE then joins the PIC, and finally TFIIE recruits TFIIH. After these steps, double-stranded DNA around the initiation site is melted to the single-stranded form by TFIIH (Dvir et al, 1996). TFIIE binds to the region between −10 and +2 (Douziech et al, 2000), where it is required to initiate melting and assist in the formation of the open complex (Holstege et al, 1996; Okamoto et al, 1998). Following extensive phosphorylation of the C-terminal domain (CTD) of the largest subunit Rpb1 of Pol II by TFIIH, activated Pol II releases all GTFs except for TFIIF and proceeds to transcription elongation (Lu et al, 1992). Upon promoter clearance, TFIIE increases the CTD kinase activity of TFIIH (Drapkin et al, 1994; Ohkuma and Roeder, 1994). Thus, it is now known that both TFIIE and TFIIH have significant functions in transcription initiation and the transition to elongation.In relation to transcriptional machinery, so far the structures of TBP (TATA box-binding protein) subunit (Nikolov et al, 1992) from TFIID, TBP–DNA (Kim et al, 1993a, 1993b), TBP–DNA–TFIIB (Nikolov et al, 1995), Pol II (Cramer et al, 2000, 2001) and Pol II–TFIIB (Bushnell et al, 2004) have been determined. On the basis of these studies, we can see the detailed structural model for the TBP–DNA–TFIIB–Pol II complex. However, for TFIIF-, TFIIE- and TFIIH-associated complex, only their several domain structures have been determined and their interaction mode has not yet been available. For the structural modelling of the PIC, many structural insights into the interactions among Pol II and GTFs, in particular the interactions between the late entries, TFIIE and TFIIIH, are required.Human TFIIE (hTFIIE) is a heterodimer, consisting of an α subunit (hTFIIEα, 57 kDa) and a β subunit (hTFIIEβ, 34 kDa) (Ohkuma et al, 1990, 1991; Peterson et al, 1991; Sumimoto et al, 1991; Itoh et al, 2005; Jawhari et al, 2006). Both subunits possess several characteristic sequences and structural/functional domains; for example, a Ser, Thr, Asp and Glu-rich (STDE) sequence and an acidic amino-acid-rich sequence are found in the C-terminal region of hTFIIEα, whereas a Ser-rich sequence is found in the N-terminal region of hTFIIEβ. Furthermore, two tertiary structures of a zinc-finger domain of hTFIIEα (Okuda et al, 2004) and a winged helix/forkhead domain of hTFIIEβ (Okuda et al, 2000) have been solved by NMR spectroscopy.Human TFIIH (hTFIIH) is a much larger molecule (480 kDa) consisting of 10 subunits (Giglia-Mari et al, 2004). This is divided into two subcomplexes, a core complex (XPB, p34, p44, p52, p62 and p8/TTDA subunits) and a CDK-activating kinase complex (CAK) (CDK7, cyclin H and MAT1 subunits). The two subcomplexes are linked by the XPD subunit (Schultz et al, 2000). The intriguing feature is that this factor has three enzymatic activities: ATP-dependent DNA helicase, DNA-dependent ATPase and CTD kinase activities, and participates not only in transcription but also in DNA repair and cell cycle control. Of these activities of hTFIIH, hTFIIE stimulates the CTD kinase and ATPase activities, and represses the helicase activity (Lu et al, 1992; Drapkin et al, 1994; Ohkuma and Roeder, 1994).With regard to the interaction between hTFIIE and hTFIIH, it has been shown that the C-terminal acidic region of hTFIIEα is necessary for native hTFIIH binding (Ohkuma et al, 1995) and hTFIIEα strongly binds to the p62 subunit of hTFIIH (Yamamoto et al, 2001; Okuda et al, 2004). Besides these findings, however, little was known about this fundamental mechanism. Insight into the mechanism is gained from the work described here, which demonstrates that the C-terminal acidic domain (AC-D) of hTFIIEα containing the acidic region specifically binds to the N-terminal pleckstrin homology domain (PH-D) of the p62 subunit of hTFIIH. We have determined structures of both free hTFIIEα AC-D and its form bound to the PH-D of hTFIIH p62 by using NMR spectroscopy. The structures reveal that hTFIIEα AC-D recognizes p62 PH-D tightly through a combination of hydrophobic and electrostatic interactions. hTFIIEα AC-D is found to share its binding surface on p62 PH-D with the acidic transactivation domains (TADs) of tumour supressor protein p53 (Di Lello et al, 2006) and herpes simplex virus protein VP16 (Di Lello et al, 2005). However, hTFIIEα AC-D employs an entirely novel binding mode, which differs from the amphipathic helix method used by many transcriptional activators. Our structural and functional studies are informative with regard to the roles of these proteins in the transcription initiation mechanism.

Introduction

In eukaryotes, transcription of protein-encoding genes is performed by RNA polymerase II (Pol II). Although it is a complex enzyme comprised of 12 subunits, Pol II alone is unable to accurately recognize promoters to initiate transcription. Transcription initiation by Pol II requires five proteins; TFIIB, TFIID, TFIIE, TFIIF and TFIIH collectively known as ‘general transcription factors (GTFs)' (Orphanides et al, 1996; Roeder, 1996). Pol II and GTFs converge on a promoter in a highly ordered manner to form the preinitiation complex (PIC). After the binding of TFIID to the TATA element located 30 nt upstream of the transcription initiation site (+1), TFIIB and TFIIF together with Pol II are recruited. TFIIE then joins the PIC, and finally TFIIE recruits TFIIH. After these steps, double-stranded DNA around the initiation site is melted to the single-stranded form by TFIIH (Dvir et al, 1996). TFIIE binds to the region between −10 and +2 (Douziech et al, 2000), where it is required to initiate melting and assist in the formation of the open complex (Holstege et al, 1996; Okamoto et al, 1998). Following extensive phosphorylation of the C-terminal domain (CTD) of the largest subunit Rpb1 of Pol II by TFIIH, activated Pol II releases all GTFs except for TFIIF and proceeds to transcription elongation (Lu et al, 1992). Upon promoter clearance, TFIIE increases the CTD kinase activity of TFIIH (Drapkin et al, 1994; Ohkuma and Roeder, 1994). Thus, it is now known that both TFIIE and TFIIH have significant functions in transcription initiation and the transition to elongation.

In relation to transcriptional machinery, so far the structures of TBP (TATA box-binding protein) subunit (Nikolov et al, 1992) from TFIID, TBP–DNA (Kim et al, 1993a, 1993b), TBP–DNA–TFIIB (Nikolov et al, 1995), Pol II (Cramer et al, 2000, 2001) and Pol II–TFIIB (Bushnell et al, 2004) have been determined. On the basis of these studies, we can see the detailed structural model for the TBP–DNA–TFIIB–Pol II complex. However, for TFIIF-, TFIIE- and TFIIH-associated complex, only their several domain structures have been determined and their interaction mode has not yet been available. For the structural modelling of the PIC, many structural insights into the interactions among Pol II and GTFs, in particular the interactions between the late entries, TFIIE and TFIIIH, are required.

Human TFIIE (hTFIIE) is a heterodimer, consisting of an α subunit (hTFIIEα, 57 kDa) and a β subunit (hTFIIEβ, 34 kDa) (Ohkuma et al, 1990, 1991; Peterson et al, 1991; Sumimoto et al, 1991; Itoh et al, 2005; Jawhari et al, 2006). Both subunits possess several characteristic sequences and structural/functional domains; for example, a Ser, Thr, Asp and Glu-rich (STDE) sequence and an acidic amino-acid-rich sequence are found in the C-terminal region of hTFIIEα, whereas a Ser-rich sequence is found in the N-terminal region of hTFIIEβ. Furthermore, two tertiary structures of a zinc-finger domain of hTFIIEα (Okuda et al, 2004) and a winged helix/forkhead domain of hTFIIEβ (Okuda et al, 2000) have been solved by NMR spectroscopy.

Human TFIIH (hTFIIH) is a much larger molecule (480 kDa) consisting of 10 subunits (Giglia-Mari et al, 2004). This is divided into two subcomplexes, a core complex (XPB, p34, p44, p52, p62 and p8/TTDA subunits) and a CDK-activating kinase complex (CAK) (CDK7, cyclin H and MAT1 subunits). The two subcomplexes are linked by the XPD subunit (Schultz et al, 2000). The intriguing feature is that this factor has three enzymatic activities: ATP-dependent DNA helicase, DNA-dependent ATPase and CTD kinase activities, and participates not only in transcription but also in DNA repair and cell cycle control. Of these activities of hTFIIH, hTFIIE stimulates the CTD kinase and ATPase activities, and represses the helicase activity (Lu et al, 1992; Drapkin et al, 1994; Ohkuma and Roeder, 1994).

With regard to the interaction between hTFIIE and hTFIIH, it has been shown that the C-terminal acidic region of hTFIIEα is necessary for native hTFIIH binding (Ohkuma et al, 1995) and hTFIIEα strongly binds to the p62 subunit of hTFIIH (Yamamoto et al, 2001; Okuda et al, 2004). Besides these findings, however, little was known about this fundamental mechanism. Insight into the mechanism is gained from the work described here, which demonstrates that the C-terminal acidic domain (AC-D) of hTFIIEα containing the acidic region specifically binds to the N-terminal pleckstrin homology domain (PH-D) of the p62 subunit of hTFIIH. We have determined structures of both free hTFIIEα AC-D and its form bound to the PH-D of hTFIIH p62 by using NMR spectroscopy. The structures reveal that hTFIIEα AC-D recognizes p62 PH-D tightly through a combination of hydrophobic and electrostatic interactions. hTFIIEα AC-D is found to share its binding surface on p62 PH-D with the acidic transactivation domains (TADs) of tumour supressor protein p53 (Di Lello et al, 2006) and herpes simplex virus protein VP16 (Di Lello et al, 2005). However, hTFIIEα AC-D employs an entirely novel binding mode, which differs from the amphipathic helix method used by many transcriptional activators. Our structural and functional studies are informative with regard to the roles of these proteins in the transcription initiation mechanism.

ResultsStructure of hTFIIEα AC-DIt has been reported that the C-terminal acidic region of hTFIIEα is necessary for hTFIIH binding (Figure 1A) (Ohkuma et al, 1995). To characterize the precise interaction at the molecular level, we first solved a solution structure of the AC-D of hTFIIEα using NMR spectroscopy (Figure 1B and Table I). The protein has a globular structure with flexible and disordered tails, consisting of the 16 N-terminal residues (amino acids 378–393) and the 5 C-terminal residues (amino acids 435–439). The core region forms a compact structure; the β-turn (S1–S2) is followed by three α-helices (H1, H2 and H3). These structural elements interact with each other and are maintained by a small but rigid hydrophobic core formed by P394, V396, V398, F403, Y405, V408, L414, V415, M418, E422, K423, Y426, I427, M429 and M433 residues. As the hydrophobic core residues as well as consecutive acidic amino acids found in the N-terminal regions of the AC-Ds are highly conserved in metazoans, they would all be expected to have similar structural features to hTFIIEα AC-D (Figure 1A). The structure seems to be a novel fold; similar structures with a Z score over 2.0 could not be detected by the DALI server.hTFIIEα AC-D specifically binds to PH-D of hTFIIH p62Previous studies showed that hTFIIEα specifically bound to the p62 subunits of hTFIIH (Yamamoto et al, 2001; Okuda et al, 2004). Given that the C-terminal acidic region of hTFIIEα (residues 378–393) is essential for hTFIIH binding (Ohkuma et al, 1995), hTFIIEα AC-D is likely to be responsible for hTFIIH recognition. To confirm this and to identify the AC-D-binding region in p62, we performed a GST pull-down assay using hTFIIEα AC-D and GST-fused p62 deletion mutants (Figure 2A). After purification by glutathione-Sepharose column chromatography, all samples containing the C-terminal region, namely full-length GST–p621−548, GST–p62109−548, GST–p62238−548 and GST–p62333−548, were considerably degraded or incompletely translated (data not shown). Though such instability of the C-terminal half of p62 has previously been reported (Jawhari et al, 2004), we found that full-length GST–p621−548, GST–p621−108, GST–p621−238 and GST–p621−333 bound to hTFIIEα AC-D, whereas no binding was observed with GST–p62109−548, GST–p62238−548, GST–p62333−548 and GST alone (Figure 2B). p62 contains the structurally stable PH-D (residues 1–108) (Gervais et al, 2004) and double BSD domains (residues 109–232) (Doerks et al, 2002) within the N-terminal half. The truncation variants with hTFIIEα-binding ability all possess the PH-D. We therefore asked whether p62 PH-D could interact with full-length hTFIIEα (Figure 2B). The results showed that GST-fused p621−548, p621−108, p621−238 and p621−333, all of which contain N-terminal 108 residues, bound to hTFIIEα, whereas the mutants p62109−548, p62238−548 and p62333−548, which lack those 108 residues, could not bind to hTFIIEα (Figure 2B, lanes 2–5 versus lanes 6–8). Thus, we concluded that hTFIIEα binds specifically to the PH-D of hTFIIH p62 (p62 PH-D) through its AC-D.Structure of complex between hTFIIEα AC-D and hTFIIH p62 PH-DTo obtain the p62 PH-D-bound structure of hTFIIEα AC-D using NMR, we performed NMR titration experiments in buffers both with and without 100 mM NaCl for both domains (Supplementary Figures 1–3). Although in the 100 mM NaCl buffer the dissociation constant (Kd) between AC-D and p62 PH-D was estimated from the titration plots as 376±81 nM (Supplementary Figure 1C) or 237±82 nM (Supplementary Figure 2C), the NaCl-free buffer NMR titration experiment showed much stronger binding affinity between AC-D and p62 PH-D because of the slow exchange timescale with a Kd below about 150 nM (Supplementary Figure 3). On the basis of these results, we determined the complex structure in 20 mM potassium phosphate buffer without NaCl.In total, 4489 NOE-derived distance restraints, including 371 intermolecular NOEs, 120 hydrogen bond restraints and 282 dihedral angle restraints were used to determine the complex structure (Table I). The complex structure is shown in Figure 3A and B. The occluded solvent-accessible surface between hTFIIEα AC-D and p62 PH-D was ∼2300 Å2. The core structure of hTFIIEα AC-D and p62 PH-D in the complex was essentially the same as seen in each isolated free structure except for the extended and highly acidic N-terminal tail of hTFIIEα AC-D. In the free form, it is disordered (Figure 1B) but upon complex formation becomes fixed, forming a new S0 strand that extensively overlays the positively charged surface of p62 PH-D formed by K18, K19, K54, K60, K62 and K93 (Figure 3C). Nine consecutive acidic residues from the N-terminal tail of hTFIIEα AC-D run across the second β-sheet (S5, S6 and S7) of p62 PH-D, electrostatically interacting with K18 and K19 on the loop between the S1 and S2 strands and K60 and K62 on the loop between S5 and S6 strands. The extended tail of hTFIIEα AC-D curves at E386 (Figure 3B). Polypeptides from F387 to A391 of hTFIIEα AC-D align along the S5 strand of p62 PH-D, forming an antiparallel β-sheet structure with it. It is noteworthy that F387 in the sequence of acidic residues is accommodated in a shallow pocket on the second β-sheet of p62 PH-D formed by K54, I55, S56, K60, Q64, L65, Q66 and N76 (Figure 3D). In the pocket, hydrophobic interactions between the aromatic side chain of F387 (hTFIIEα) and the aliphatic portions of K54 and K60 (p62) and amino-aromatic interactions (Burley and Petsko, 1986) between the side chains of F387 and Q64, Q66 and N76 (p62) were observed. E388 (hTFIIEα) interacts through van der Waals contacts with I55 and P57 (p62), and E389 (hTFIIEα) forms a salt bridge with K54 (p62). Similar to as seen for F387 (hTFIIEα), V390 was inserted into a shallow pocket between the S5 strand and the C-terminal H1 helix of p62 PH-D, formed by Q53 and I55 on the S5 strand and K93 and Q97 on the H1 helix (Figure 3E). V390 makes extensive hydrophobic contacts with I55 and the aliphatic regions of Q53, K93 and Q97. The N-terminal tail of hTFIIEα AC-D bends further at D392 (Figure 3B), which causes van der Waals contacts between L100 and P101 (p62) (Figure 3F). D393 (hTFIIEα) at the end of the tail lies in close proximity to Q97 (p62) making van der Waals contact with it.These interacting amino acids were also observed in the NMR titration experiments. In hTFIIEα AC-D, the NMR signals of E386, F387, E388, E389, V390, A391 and D392 were changed significantly upon addition of p62 PH-D (Supplementary Figure 1B) and also in p62 PH-D the NMR signals of K19, Q53, K54, I55, S56, E58, K60, A61, I63, Q64, L65, Q66, T74, T75 and F77 were changed by adding hTFIIEα AC-D (Supplementary Figure 2B).It is remarkable that in addition to the interaction involving the N-terminal flexible tail of hTFIIEα AC-D, its core structure also participates in the binding to p62 PH-D (Figure 3F). Several residues in the N- and C-terminal regions of the hTFIIEα AC-D core structure interact with residues located in the C-terminal region of p62 PH-D. P394 (hTFIIEα) at the N terminus of the core structure contributes to the formation of the hydrophobic core of hTFIIEα AC-D and simultaneously makes intimate van der Waals contacts with P101 (p62) and also with the aliphatic portion of Q98 (p62). I395 (hTFIIEα), which is exposed to the surface in the free form, now makes hydrophobic contact with the aliphatic segment of Q98 (p62). R432 (hTFIIEα), which is at the end of the H3 helix, makes van der Waals contacts with P101 and the hydrophobic portion of K102 (p62). M433 (TFIIEα) makes van der Waals contact with the aliphatic region of K104 (p62), which is also able to make an electrostatic interaction with D392 or D436. These interactions allow the core structure of hTFIIEα AC-D to take up a position to the side of p62 PH-D, such that the whole complex structure is well defined as shown in Figure 3A.Effects of hTFIIEα AC-D mutations on binding to p62 PH-DIn functional studies of hTFIIEα, we made several mutants of hTFIIEα by changing S365, V372, D380, E383, F387, V390 and D393 to alanine (S365A, V372A, D380A, E383A, F387A, V390A and D393A) as well as S365E, V372D, F387E and V390K. These mutants were expressed in Escherichia coli with hexa-histidine (6H) at the N terminus. All were soluble and could therefore be easily purified using a Ni-nitrilotriacetic acid (NTA) agarose column (Figure 4A).The ability of hTFIIEα mutants to bind to GST-tagged p62 PH-D was examined by in vitro binding assay (Figure 4B). The p62 PH-D-binding activity of the hTFIIEα mutants was severely reduced when the AC-D residues, F387 and V390, which fit into shallow pockets of p62 PH-D, were changed to F387A, F387E, V390A and V390K (Figure 4B, second column p62 PH-D, lanes 9–12). We have shown previously that the N terminus of hTFIIEα is essential for binding to hTFIIEβ (Ohkuma et al, 1995). Consistent with these observations is the fact that none of the hTFIIEα mutations affected the binding of hTFIIEα to hTFIIEβ (Figure 4B, third column IIEβ, lanes 3–13).Functional roles of hTFIIEα AC-D during transcriptionTo further investigate the functional roles of hTFIIEα AC-D, we first checked the effects of the hTFIIEα mutants on basal transcription using the adenovirus major late pML(C2AT)Δ-50 template (Figure 4C). All mutants exhibited defects in transcription. Intriguingly, however, the mutants of S365A, S365E, V372A, V372D, D380A and E383A showed more defects than the p62 PH-D-binding defective mutants (Figure 4C, lanes 3–8 versus lanes 9–13).We next tested the effects on CTD phosphorylation. Each hTFIIEα mutant was mixed with hTFIIH and Pol II. The mixture was analysed by SDS–PAGE and phosphorylated Rpb1 from Pol II was detected by autoradiography (Figure 4D). All mutants failed to stimulate CTD phosphorylation properly compared with wild type (Figure 4D, lanes 3–13 versus lane 2). Phosphorylation profiles of the above-described mutants (S365A, S365E, V372A, V372D, D380A and E383A) phosphorylated CTD but most of Rpb1 was detected at the hypo-phosphorylated IIa position (Figure 4D, lanes 3–8). In contrast, the p62 PH-D-binding defective mutants phosphorylated CTD only weakly (Figure 4D, lanes 9–12).A part of the hTFIIE-binding surface on hTFIIH p62 is shared with transcriptional activator p53Recently, the structure of a complex of the PH-D of Saccharomyces cerevisiae Tfb1, a homologue of human p62, with a TAD2 of activator p53 was determined by NMR spectroscopy (Di Lello et al, 2006). The structure of the Tfb1 PH-D closely resembles that of p62, except for its longer connecting loop between S6 and S7 (Figure 5A and B). Furthermore, the herpes simplex virus protein 16 (VP16) TAD also interacts with virtually identical sites of Tfb1 and p62 PH-D (Di Lello et al, 2005). Interestingly, the binding sites of p62 for p53 TAD2 and VP16 TAD significantly overlap with a part of the binding site for hTFIIEα AC-D. However, their binding mode is entirely different. The binding site of p53 TAD2 peptide is disordered in an unbound state, but it forms a nine-residue amphipathic α-helix upon binding to Tfb1 PH-D and p62 PH-D. The p53 helix contacts the second β-sheet of Tfb1 PH-D through the interactions of I50 (p53)—M59, M88 (Tfb1); E51 (p53)—R61 or R86 (Tfb1); W53 (p53)—K57, M59 (Tfb1); F54 (p53)—Q49, A50, T51, M59, L60, R61 (Tfb1); and E56 (p53)—K57 (Tfb1). Although the N-terminal tail of hTFIIEα AC-D also becomes ordered upon binding to p62 PH-D, it forms a bent extended structure containing a S0 strand, but not α-helix. In spite of such great structural differences, both hTFIIEα AC-D and p53 TAD2 peptide insert phenylalanine residues, F387 of hTFIIEα AC-D and F54 of p53 TAD2, into the equivalent pocket on the second β-sheet of p62 PH-D. Although the p53 TAD2 peptide consisting of residues 20–73 forms no contacts beside this limited area, hTFIIEα AC-D further interacts with p62 PH-D as mentioned above. The binding surface area of hTFIIEα AC-D and p62 PH-D is calculated as ∼2300 Å2, which is much larger than the binding area of p53 TAD2 for Tfb1 calculated as ∼800 Å2.To analyse this interaction biochemically, several point mutants of p62 PH-D were created, bacterially expressed and used in binding studies with hTFIIEα AC-D and p531−73 (Figure 5C). As controls, AC-D-containing hTFIIEα wild type (IIEα wt) and hTFIIEα351−439 (IIEα351−439) were also examined in parallel. As shown, K54, which forms the shallow pocket for F387 of hTFIIEα AC-D with its side chain interacting electrostatically with E389 of hTFIIEα AC-D, was the residue for which mutation to alanine had the largest effect as it prevented binding of all three hTFIIEα proteins tested (Figure 5C, lane 5). In addition, Q66, which also forms the same shallow pocket for F387 of hTFIIEα AC-D and the side chain of Q66 makes contact with F387 through amino-aromatic interaction, was shown to be essential for binding to hTFIIEα AC-D (Figure 5C, lane 6). The adjacent residues, V68, T74 and N76 of p62 PH-D as well as the N-terminal basic residues, K18 and K19, also affected binding but to a lesser extent (Figure 5C, lanes 3, 4, 7–9). A similar but distinct inhibition profile was observed for the N-terminal TAD of p53 (p531−73). In this case, Q66 was also central to the interaction but the essential binding residues were more widespread (Figure 5C, the bottom column, lanes 3 and 6–9).Replacement of p53 bound to hTFIIH p62 with hTFIIEαAs the overlap of the p62 PH-D-binding region of hTFIIEα AC-D with that of p53 TAD2 was observed and judging from the functional context that p53 may recruit TFIIH at transcriptional activation but at some point TFIIE should take over to recruit TFIIH into the PIC, we then asked whether p53 can be replaced on p62 with hTFIIEα. As shown in Figure 5D, p62 PH-D bound to GST–p53 TAD (p531−73) was removed upon addition of 6H–hTFIIEα wild type (Figure 5D, lanes 2–4). Unbound fractions were then mixed with Ni-NTA resin and FLAG-p62 PH-D bound to 6H–hTFIIEα was detected by western blotting (Figure 5D, lanes 7–9). This clearly demonstrates that p53 binding to p62 can be replaced with hTFIIEα.

Structure of hTFIIEα AC-DIt has been reported that the C-terminal acidic region of hTFIIEα is necessary for hTFIIH binding (Figure 1A) (Ohkuma et al, 1995). To characterize the precise interaction at the molecular level, we first solved a solution structure of the AC-D of hTFIIEα using NMR spectroscopy (Figure 1B and Table I). The protein has a globular structure with flexible and disordered tails, consisting of the 16 N-terminal residues (amino acids 378–393) and the 5 C-terminal residues (amino acids 435–439). The core region forms a compact structure; the β-turn (S1–S2) is followed by three α-helices (H1, H2 and H3). These structural elements interact with each other and are maintained by a small but rigid hydrophobic core formed by P394, V396, V398, F403, Y405, V408, L414, V415, M418, E422, K423, Y426, I427, M429 and M433 residues. As the hydrophobic core residues as well as consecutive acidic amino acids found in the N-terminal regions of the AC-Ds are highly conserved in metazoans, they would all be expected to have similar structural features to hTFIIEα AC-D (Figure 1A). The structure seems to be a novel fold; similar structures with a Z score over 2.0 could not be detected by the DALI server.

Structure of hTFIIEα AC-D

It has been reported that the C-terminal acidic region of hTFIIEα is necessary for hTFIIH binding (Figure 1A) (Ohkuma et al, 1995). To characterize the precise interaction at the molecular level, we first solved a solution structure of the AC-D of hTFIIEα using NMR spectroscopy (Figure 1B and Table I). The protein has a globular structure with flexible and disordered tails, consisting of the 16 N-terminal residues (amino acids 378–393) and the 5 C-terminal residues (amino acids 435–439). The core region forms a compact structure; the β-turn (S1–S2) is followed by three α-helices (H1, H2 and H3). These structural elements interact with each other and are maintained by a small but rigid hydrophobic core formed by P394, V396, V398, F403, Y405, V408, L414, V415, M418, E422, K423, Y426, I427, M429 and M433 residues. As the hydrophobic core residues as well as consecutive acidic amino acids found in the N-terminal regions of the AC-Ds are highly conserved in metazoans, they would all be expected to have similar structural features to hTFIIEα AC-D (Figure 1A). The structure seems to be a novel fold; similar structures with a Z score over 2.0 could not be detected by the DALI server.

hTFIIEα AC-D specifically binds to PH-D of hTFIIH p62Previous studies showed that hTFIIEα specifically bound to the p62 subunits of hTFIIH (Yamamoto et al, 2001; Okuda et al, 2004). Given that the C-terminal acidic region of hTFIIEα (residues 378–393) is essential for hTFIIH binding (Ohkuma et al, 1995), hTFIIEα AC-D is likely to be responsible for hTFIIH recognition. To confirm this and to identify the AC-D-binding region in p62, we performed a GST pull-down assay using hTFIIEα AC-D and GST-fused p62 deletion mutants (Figure 2A). After purification by glutathione-Sepharose column chromatography, all samples containing the C-terminal region, namely full-length GST–p621−548, GST–p62109−548, GST–p62238−548 and GST–p62333−548, were considerably degraded or incompletely translated (data not shown). Though such instability of the C-terminal half of p62 has previously been reported (Jawhari et al, 2004), we found that full-length GST–p621−548, GST–p621−108, GST–p621−238 and GST–p621−333 bound to hTFIIEα AC-D, whereas no binding was observed with GST–p62109−548, GST–p62238−548, GST–p62333−548 and GST alone (Figure 2B). p62 contains the structurally stable PH-D (residues 1–108) (Gervais et al, 2004) and double BSD domains (residues 109–232) (Doerks et al, 2002) within the N-terminal half. The truncation variants with hTFIIEα-binding ability all possess the PH-D. We therefore asked whether p62 PH-D could interact with full-length hTFIIEα (Figure 2B). The results showed that GST-fused p621−548, p621−108, p621−238 and p621−333, all of which contain N-terminal 108 residues, bound to hTFIIEα, whereas the mutants p62109−548, p62238−548 and p62333−548, which lack those 108 residues, could not bind to hTFIIEα (Figure 2B, lanes 2–5 versus lanes 6–8). Thus, we concluded that hTFIIEα binds specifically to the PH-D of hTFIIH p62 (p62 PH-D) through its AC-D.

hTFIIEα AC-D specifically binds to PH-D of hTFIIH p62

Previous studies showed that hTFIIEα specifically bound to the p62 subunits of hTFIIH (Yamamoto et al, 2001; Okuda et al, 2004). Given that the C-terminal acidic region of hTFIIEα (residues 378–393) is essential for hTFIIH binding (Ohkuma et al, 1995), hTFIIEα AC-D is likely to be responsible for hTFIIH recognition. To confirm this and to identify the AC-D-binding region in p62, we performed a GST pull-down assay using hTFIIEα AC-D and GST-fused p62 deletion mutants (Figure 2A). After purification by glutathione-Sepharose column chromatography, all samples containing the C-terminal region, namely full-length GST–p621−548, GST–p62109−548, GST–p62238−548 and GST–p62333−548, were considerably degraded or incompletely translated (data not shown). Though such instability of the C-terminal half of p62 has previously been reported (Jawhari et al, 2004), we found that full-length GST–p621−548, GST–p621−108, GST–p621−238 and GST–p621−333 bound to hTFIIEα AC-D, whereas no binding was observed with GST–p62109−548, GST–p62238−548, GST–p62333−548 and GST alone (Figure 2B). p62 contains the structurally stable PH-D (residues 1–108) (Gervais et al, 2004) and double BSD domains (residues 109–232) (Doerks et al, 2002) within the N-terminal half. The truncation variants with hTFIIEα-binding ability all possess the PH-D. We therefore asked whether p62 PH-D could interact with full-length hTFIIEα (Figure 2B). The results showed that GST-fused p621−548, p621−108, p621−238 and p621−333, all of which contain N-terminal 108 residues, bound to hTFIIEα, whereas the mutants p62109−548, p62238−548 and p62333−548, which lack those 108 residues, could not bind to hTFIIEα (Figure 2B, lanes 2–5 versus lanes 6–8). Thus, we concluded that hTFIIEα binds specifically to the PH-D of hTFIIH p62 (p62 PH-D) through its AC-D.

Structure of complex between hTFIIEα AC-D and hTFIIH p62 PH-DTo obtain the p62 PH-D-bound structure of hTFIIEα AC-D using NMR, we performed NMR titration experiments in buffers both with and without 100 mM NaCl for both domains (Supplementary Figures 1–3). Although in the 100 mM NaCl buffer the dissociation constant (Kd) between AC-D and p62 PH-D was estimated from the titration plots as 376±81 nM (Supplementary Figure 1C) or 237±82 nM (Supplementary Figure 2C), the NaCl-free buffer NMR titration experiment showed much stronger binding affinity between AC-D and p62 PH-D because of the slow exchange timescale with a Kd below about 150 nM (Supplementary Figure 3). On the basis of these results, we determined the complex structure in 20 mM potassium phosphate buffer without NaCl.In total, 4489 NOE-derived distance restraints, including 371 intermolecular NOEs, 120 hydrogen bond restraints and 282 dihedral angle restraints were used to determine the complex structure (Table I). The complex structure is shown in Figure 3A and B. The occluded solvent-accessible surface between hTFIIEα AC-D and p62 PH-D was ∼2300 Å2. The core structure of hTFIIEα AC-D and p62 PH-D in the complex was essentially the same as seen in each isolated free structure except for the extended and highly acidic N-terminal tail of hTFIIEα AC-D. In the free form, it is disordered (Figure 1B) but upon complex formation becomes fixed, forming a new S0 strand that extensively overlays the positively charged surface of p62 PH-D formed by K18, K19, K54, K60, K62 and K93 (Figure 3C). Nine consecutive acidic residues from the N-terminal tail of hTFIIEα AC-D run across the second β-sheet (S5, S6 and S7) of p62 PH-D, electrostatically interacting with K18 and K19 on the loop between the S1 and S2 strands and K60 and K62 on the loop between S5 and S6 strands. The extended tail of hTFIIEα AC-D curves at E386 (Figure 3B). Polypeptides from F387 to A391 of hTFIIEα AC-D align along the S5 strand of p62 PH-D, forming an antiparallel β-sheet structure with it. It is noteworthy that F387 in the sequence of acidic residues is accommodated in a shallow pocket on the second β-sheet of p62 PH-D formed by K54, I55, S56, K60, Q64, L65, Q66 and N76 (Figure 3D). In the pocket, hydrophobic interactions between the aromatic side chain of F387 (hTFIIEα) and the aliphatic portions of K54 and K60 (p62) and amino-aromatic interactions (Burley and Petsko, 1986) between the side chains of F387 and Q64, Q66 and N76 (p62) were observed. E388 (hTFIIEα) interacts through van der Waals contacts with I55 and P57 (p62), and E389 (hTFIIEα) forms a salt bridge with K54 (p62). Similar to as seen for F387 (hTFIIEα), V390 was inserted into a shallow pocket between the S5 strand and the C-terminal H1 helix of p62 PH-D, formed by Q53 and I55 on the S5 strand and K93 and Q97 on the H1 helix (Figure 3E). V390 makes extensive hydrophobic contacts with I55 and the aliphatic regions of Q53, K93 and Q97. The N-terminal tail of hTFIIEα AC-D bends further at D392 (Figure 3B), which causes van der Waals contacts between L100 and P101 (p62) (Figure 3F). D393 (hTFIIEα) at the end of the tail lies in close proximity to Q97 (p62) making van der Waals contact with it.These interacting amino acids were also observed in the NMR titration experiments. In hTFIIEα AC-D, the NMR signals of E386, F387, E388, E389, V390, A391 and D392 were changed significantly upon addition of p62 PH-D (Supplementary Figure 1B) and also in p62 PH-D the NMR signals of K19, Q53, K54, I55, S56, E58, K60, A61, I63, Q64, L65, Q66, T74, T75 and F77 were changed by adding hTFIIEα AC-D (Supplementary Figure 2B).It is remarkable that in addition to the interaction involving the N-terminal flexible tail of hTFIIEα AC-D, its core structure also participates in the binding to p62 PH-D (Figure 3F). Several residues in the N- and C-terminal regions of the hTFIIEα AC-D core structure interact with residues located in the C-terminal region of p62 PH-D. P394 (hTFIIEα) at the N terminus of the core structure contributes to the formation of the hydrophobic core of hTFIIEα AC-D and simultaneously makes intimate van der Waals contacts with P101 (p62) and also with the aliphatic portion of Q98 (p62). I395 (hTFIIEα), which is exposed to the surface in the free form, now makes hydrophobic contact with the aliphatic segment of Q98 (p62). R432 (hTFIIEα), which is at the end of the H3 helix, makes van der Waals contacts with P101 and the hydrophobic portion of K102 (p62). M433 (TFIIEα) makes van der Waals contact with the aliphatic region of K104 (p62), which is also able to make an electrostatic interaction with D392 or D436. These interactions allow the core structure of hTFIIEα AC-D to take up a position to the side of p62 PH-D, such that the whole complex structure is well defined as shown in Figure 3A.

Structure of complex between hTFIIEα AC-D and hTFIIH p62 PH-D

To obtain the p62 PH-D-bound structure of hTFIIEα AC-D using NMR, we performed NMR titration experiments in buffers both with and without 100 mM NaCl for both domains (Supplementary Figures 1–3). Although in the 100 mM NaCl buffer the dissociation constant (Kd) between AC-D and p62 PH-D was estimated from the titration plots as 376±81 nM (Supplementary Figure 1C) or 237±82 nM (Supplementary Figure 2C), the NaCl-free buffer NMR titration experiment showed much stronger binding affinity between AC-D and p62 PH-D because of the slow exchange timescale with a Kd below about 150 nM (Supplementary Figure 3). On the basis of these results, we determined the complex structure in 20 mM potassium phosphate buffer without NaCl.

In total, 4489 NOE-derived distance restraints, including 371 intermolecular NOEs, 120 hydrogen bond restraints and 282 dihedral angle restraints were used to determine the complex structure (Table I). The complex structure is shown in Figure 3A and B. The occluded solvent-accessible surface between hTFIIEα AC-D and p62 PH-D was ∼2300 Å2. The core structure of hTFIIEα AC-D and p62 PH-D in the complex was essentially the same as seen in each isolated free structure except for the extended and highly acidic N-terminal tail of hTFIIEα AC-D. In the free form, it is disordered (Figure 1B) but upon complex formation becomes fixed, forming a new S0 strand that extensively overlays the positively charged surface of p62 PH-D formed by K18, K19, K54, K60, K62 and K93 (Figure 3C). Nine consecutive acidic residues from the N-terminal tail of hTFIIEα AC-D run across the second β-sheet (S5, S6 and S7) of p62 PH-D, electrostatically interacting with K18 and K19 on the loop between the S1 and S2 strands and K60 and K62 on the loop between S5 and S6 strands. The extended tail of hTFIIEα AC-D curves at E386 (Figure 3B). Polypeptides from F387 to A391 of hTFIIEα AC-D align along the S5 strand of p62 PH-D, forming an antiparallel β-sheet structure with it. It is noteworthy that F387 in the sequence of acidic residues is accommodated in a shallow pocket on the second β-sheet of p62 PH-D formed by K54, I55, S56, K60, Q64, L65, Q66 and N76 (Figure 3D). In the pocket, hydrophobic interactions between the aromatic side chain of F387 (hTFIIEα) and the aliphatic portions of K54 and K60 (p62) and amino-aromatic interactions (Burley and Petsko, 1986) between the side chains of F387 and Q64, Q66 and N76 (p62) were observed. E388 (hTFIIEα) interacts through van der Waals contacts with I55 and P57 (p62), and E389 (hTFIIEα) forms a salt bridge with K54 (p62). Similar to as seen for F387 (hTFIIEα), V390 was inserted into a shallow pocket between the S5 strand and the C-terminal H1 helix of p62 PH-D, formed by Q53 and I55 on the S5 strand and K93 and Q97 on the H1 helix (Figure 3E). V390 makes extensive hydrophobic contacts with I55 and the aliphatic regions of Q53, K93 and Q97. The N-terminal tail of hTFIIEα AC-D bends further at D392 (Figure 3B), which causes van der Waals contacts between L100 and P101 (p62) (Figure 3F). D393 (hTFIIEα) at the end of the tail lies in close proximity to Q97 (p62) making van der Waals contact with it.

These interacting amino acids were also observed in the NMR titration experiments. In hTFIIEα AC-D, the NMR signals of E386, F387, E388, E389, V390, A391 and D392 were changed significantly upon addition of p62 PH-D (Supplementary Figure 1B) and also in p62 PH-D the NMR signals of K19, Q53, K54, I55, S56, E58, K60, A61, I63, Q64, L65, Q66, T74, T75 and F77 were changed by adding hTFIIEα AC-D (Supplementary Figure 2B).

It is remarkable that in addition to the interaction involving the N-terminal flexible tail of hTFIIEα AC-D, its core structure also participates in the binding to p62 PH-D (Figure 3F). Several residues in the N- and C-terminal regions of the hTFIIEα AC-D core structure interact with residues located in the C-terminal region of p62 PH-D. P394 (hTFIIEα) at the N terminus of the core structure contributes to the formation of the hydrophobic core of hTFIIEα AC-D and simultaneously makes intimate van der Waals contacts with P101 (p62) and also with the aliphatic portion of Q98 (p62). I395 (hTFIIEα), which is exposed to the surface in the free form, now makes hydrophobic contact with the aliphatic segment of Q98 (p62). R432 (hTFIIEα), which is at the end of the H3 helix, makes van der Waals contacts with P101 and the hydrophobic portion of K102 (p62). M433 (TFIIEα) makes van der Waals contact with the aliphatic region of K104 (p62), which is also able to make an electrostatic interaction with D392 or D436. These interactions allow the core structure of hTFIIEα AC-D to take up a position to the side of p62 PH-D, such that the whole complex structure is well defined as shown in Figure 3A.

Effects of hTFIIEα AC-D mutations on binding to p62 PH-DIn functional studies of hTFIIEα, we made several mutants of hTFIIEα by changing S365, V372, D380, E383, F387, V390 and D393 to alanine (S365A, V372A, D380A, E383A, F387A, V390A and D393A) as well as S365E, V372D, F387E and V390K. These mutants were expressed in Escherichia coli with hexa-histidine (6H) at the N terminus. All were soluble and could therefore be easily purified using a Ni-nitrilotriacetic acid (NTA) agarose column (Figure 4A).The ability of hTFIIEα mutants to bind to GST-tagged p62 PH-D was examined by in vitro binding assay (Figure 4B). The p62 PH-D-binding activity of the hTFIIEα mutants was severely reduced when the AC-D residues, F387 and V390, which fit into shallow pockets of p62 PH-D, were changed to F387A, F387E, V390A and V390K (Figure 4B, second column p62 PH-D, lanes 9–12). We have shown previously that the N terminus of hTFIIEα is essential for binding to hTFIIEβ (Ohkuma et al, 1995). Consistent with these observations is the fact that none of the hTFIIEα mutations affected the binding of hTFIIEα to hTFIIEβ (Figure 4B, third column IIEβ, lanes 3–13).

Effects of hTFIIEα AC-D mutations on binding to p62 PH-D

In functional studies of hTFIIEα, we made several mutants of hTFIIEα by changing S365, V372, D380, E383, F387, V390 and D393 to alanine (S365A, V372A, D380A, E383A, F387A, V390A and D393A) as well as S365E, V372D, F387E and V390K. These mutants were expressed in Escherichia coli with hexa-histidine (6H) at the N terminus. All were soluble and could therefore be easily purified using a Ni-nitrilotriacetic acid (NTA) agarose column (Figure 4A).

The ability of hTFIIEα mutants to bind to GST-tagged p62 PH-D was examined by in vitro binding assay (Figure 4B). The p62 PH-D-binding activity of the hTFIIEα mutants was severely reduced when the AC-D residues, F387 and V390, which fit into shallow pockets of p62 PH-D, were changed to F387A, F387E, V390A and V390K (Figure 4B, second column p62 PH-D, lanes 9–12). We have shown previously that the N terminus of hTFIIEα is essential for binding to hTFIIEβ (Ohkuma et al, 1995). Consistent with these observations is the fact that none of the hTFIIEα mutations affected the binding of hTFIIEα to hTFIIEβ (Figure 4B, third column IIEβ, lanes 3–13).

Functional roles of hTFIIEα AC-D during transcriptionTo further investigate the functional roles of hTFIIEα AC-D, we first checked the effects of the hTFIIEα mutants on basal transcription using the adenovirus major late pML(C2AT)Δ-50 template (Figure 4C). All mutants exhibited defects in transcription. Intriguingly, however, the mutants of S365A, S365E, V372A, V372D, D380A and E383A showed more defects than the p62 PH-D-binding defective mutants (Figure 4C, lanes 3–8 versus lanes 9–13).We next tested the effects on CTD phosphorylation. Each hTFIIEα mutant was mixed with hTFIIH and Pol II. The mixture was analysed by SDS–PAGE and phosphorylated Rpb1 from Pol II was detected by autoradiography (Figure 4D). All mutants failed to stimulate CTD phosphorylation properly compared with wild type (Figure 4D, lanes 3–13 versus lane 2). Phosphorylation profiles of the above-described mutants (S365A, S365E, V372A, V372D, D380A and E383A) phosphorylated CTD but most of Rpb1 was detected at the hypo-phosphorylated IIa position (Figure 4D, lanes 3–8). In contrast, the p62 PH-D-binding defective mutants phosphorylated CTD only weakly (Figure 4D, lanes 9–12).

Functional roles of hTFIIEα AC-D during transcription

To further investigate the functional roles of hTFIIEα AC-D, we first checked the effects of the hTFIIEα mutants on basal transcription using the adenovirus major late pML(C2AT)Δ-50 template (Figure 4C). All mutants exhibited defects in transcription. Intriguingly, however, the mutants of S365A, S365E, V372A, V372D, D380A and E383A showed more defects than the p62 PH-D-binding defective mutants (Figure 4C, lanes 3–8 versus lanes 9–13).

We next tested the effects on CTD phosphorylation. Each hTFIIEα mutant was mixed with hTFIIH and Pol II. The mixture was analysed by SDS–PAGE and phosphorylated Rpb1 from Pol II was detected by autoradiography (Figure 4D). All mutants failed to stimulate CTD phosphorylation properly compared with wild type (Figure 4D, lanes 3–13 versus lane 2). Phosphorylation profiles of the above-described mutants (S365A, S365E, V372A, V372D, D380A and E383A) phosphorylated CTD but most of Rpb1 was detected at the hypo-phosphorylated IIa position (Figure 4D, lanes 3–8). In contrast, the p62 PH-D-binding defective mutants phosphorylated CTD only weakly (Figure 4D, lanes 9–12).

A part of the hTFIIE-binding surface on hTFIIH p62 is shared with transcriptional activator p53Recently, the structure of a complex of the PH-D of Saccharomyces cerevisiae Tfb1, a homologue of human p62, with a TAD2 of activator p53 was determined by NMR spectroscopy (Di Lello et al, 2006). The structure of the Tfb1 PH-D closely resembles that of p62, except for its longer connecting loop between S6 and S7 (Figure 5A and B). Furthermore, the herpes simplex virus protein 16 (VP16) TAD also interacts with virtually identical sites of Tfb1 and p62 PH-D (Di Lello et al, 2005). Interestingly, the binding sites of p62 for p53 TAD2 and VP16 TAD significantly overlap with a part of the binding site for hTFIIEα AC-D. However, their binding mode is entirely different. The binding site of p53 TAD2 peptide is disordered in an unbound state, but it forms a nine-residue amphipathic α-helix upon binding to Tfb1 PH-D and p62 PH-D. The p53 helix contacts the second β-sheet of Tfb1 PH-D through the interactions of I50 (p53)—M59, M88 (Tfb1); E51 (p53)—R61 or R86 (Tfb1); W53 (p53)—K57, M59 (Tfb1); F54 (p53)—Q49, A50, T51, M59, L60, R61 (Tfb1); and E56 (p53)—K57 (Tfb1). Although the N-terminal tail of hTFIIEα AC-D also becomes ordered upon binding to p62 PH-D, it forms a bent extended structure containing a S0 strand, but not α-helix. In spite of such great structural differences, both hTFIIEα AC-D and p53 TAD2 peptide insert phenylalanine residues, F387 of hTFIIEα AC-D and F54 of p53 TAD2, into the equivalent pocket on the second β-sheet of p62 PH-D. Although the p53 TAD2 peptide consisting of residues 20–73 forms no contacts beside this limited area, hTFIIEα AC-D further interacts with p62 PH-D as mentioned above. The binding surface area of hTFIIEα AC-D and p62 PH-D is calculated as ∼2300 Å2, which is much larger than the binding area of p53 TAD2 for Tfb1 calculated as ∼800 Å2.To analyse this interaction biochemically, several point mutants of p62 PH-D were created, bacterially expressed and used in binding studies with hTFIIEα AC-D and p531−73 (Figure 5C). As controls, AC-D-containing hTFIIEα wild type (IIEα wt) and hTFIIEα351−439 (IIEα351−439) were also examined in parallel. As shown, K54, which forms the shallow pocket for F387 of hTFIIEα AC-D with its side chain interacting electrostatically with E389 of hTFIIEα AC-D, was the residue for which mutation to alanine had the largest effect as it prevented binding of all three hTFIIEα proteins tested (Figure 5C, lane 5). In addition, Q66, which also forms the same shallow pocket for F387 of hTFIIEα AC-D and the side chain of Q66 makes contact with F387 through amino-aromatic interaction, was shown to be essential for binding to hTFIIEα AC-D (Figure 5C, lane 6). The adjacent residues, V68, T74 and N76 of p62 PH-D as well as the N-terminal basic residues, K18 and K19, also affected binding but to a lesser extent (Figure 5C, lanes 3, 4, 7–9). A similar but distinct inhibition profile was observed for the N-terminal TAD of p53 (p531−73). In this case, Q66 was also central to the interaction but the essential binding residues were more widespread (Figure 5C, the bottom column, lanes 3 and 6–9).

A part of the hTFIIE-binding surface on hTFIIH p62 is shared with transcriptional activator p53

Recently, the structure of a complex of the PH-D of Saccharomyces cerevisiae Tfb1, a homologue of human p62, with a TAD2 of activator p53 was determined by NMR spectroscopy (Di Lello et al, 2006). The structure of the Tfb1 PH-D closely resembles that of p62, except for its longer connecting loop between S6 and S7 (Figure 5A and B). Furthermore, the herpes simplex virus protein 16 (VP16) TAD also interacts with virtually identical sites of Tfb1 and p62 PH-D (Di Lello et al, 2005). Interestingly, the binding sites of p62 for p53 TAD2 and VP16 TAD significantly overlap with a part of the binding site for hTFIIEα AC-D. However, their binding mode is entirely different. The binding site of p53 TAD2 peptide is disordered in an unbound state, but it forms a nine-residue amphipathic α-helix upon binding to Tfb1 PH-D and p62 PH-D. The p53 helix contacts the second β-sheet of Tfb1 PH-D through the interactions of I50 (p53)—M59, M88 (Tfb1); E51 (p53)—R61 or R86 (Tfb1); W53 (p53)—K57, M59 (Tfb1); F54 (p53)—Q49, A50, T51, M59, L60, R61 (Tfb1); and E56 (p53)—K57 (Tfb1). Although the N-terminal tail of hTFIIEα AC-D also becomes ordered upon binding to p62 PH-D, it forms a bent extended structure containing a S0 strand, but not α-helix. In spite of such great structural differences, both hTFIIEα AC-D and p53 TAD2 peptide insert phenylalanine residues, F387 of hTFIIEα AC-D and F54 of p53 TAD2, into the equivalent pocket on the second β-sheet of p62 PH-D. Although the p53 TAD2 peptide consisting of residues 20–73 forms no contacts beside this limited area, hTFIIEα AC-D further interacts with p62 PH-D as mentioned above. The binding surface area of hTFIIEα AC-D and p62 PH-D is calculated as ∼2300 Å2, which is much larger than the binding area of p53 TAD2 for Tfb1 calculated as ∼800 Å2.

To analyse this interaction biochemically, several point mutants of p62 PH-D were created, bacterially expressed and used in binding studies with hTFIIEα AC-D and p531−73 (Figure 5C). As controls, AC-D-containing hTFIIEα wild type (IIEα wt) and hTFIIEα351−439 (IIEα351−439) were also examined in parallel. As shown, K54, which forms the shallow pocket for F387 of hTFIIEα AC-D with its side chain interacting electrostatically with E389 of hTFIIEα AC-D, was the residue for which mutation to alanine had the largest effect as it prevented binding of all three hTFIIEα proteins tested (Figure 5C, lane 5). In addition, Q66, which also forms the same shallow pocket for F387 of hTFIIEα AC-D and the side chain of Q66 makes contact with F387 through amino-aromatic interaction, was shown to be essential for binding to hTFIIEα AC-D (Figure 5C, lane 6). The adjacent residues, V68, T74 and N76 of p62 PH-D as well as the N-terminal basic residues, K18 and K19, also affected binding but to a lesser extent (Figure 5C, lanes 3, 4, 7–9). A similar but distinct inhibition profile was observed for the N-terminal TAD of p53 (p531−73). In this case, Q66 was also central to the interaction but the essential binding residues were more widespread (Figure 5C, the bottom column, lanes 3 and 6–9).

Replacement of p53 bound to hTFIIH p62 with hTFIIEαAs the overlap of the p62 PH-D-binding region of hTFIIEα AC-D with that of p53 TAD2 was observed and judging from the functional context that p53 may recruit TFIIH at transcriptional activation but at some point TFIIE should take over to recruit TFIIH into the PIC, we then asked whether p53 can be replaced on p62 with hTFIIEα. As shown in Figure 5D, p62 PH-D bound to GST–p53 TAD (p531−73) was removed upon addition of 6H–hTFIIEα wild type (Figure 5D, lanes 2–4). Unbound fractions were then mixed with Ni-NTA resin and FLAG-p62 PH-D bound to 6H–hTFIIEα was detected by western blotting (Figure 5D, lanes 7–9). This clearly demonstrates that p53 binding to p62 can be replaced with hTFIIEα.

Replacement of p53 bound to hTFIIH p62 with hTFIIEα

As the overlap of the p62 PH-D-binding region of hTFIIEα AC-D with that of p53 TAD2 was observed and judging from the functional context that p53 may recruit TFIIH at transcriptional activation but at some point TFIIE should take over to recruit TFIIH into the PIC, we then asked whether p53 can be replaced on p62 with hTFIIEα. As shown in Figure 5D, p62 PH-D bound to GST–p53 TAD (p531−73) was removed upon addition of 6H–hTFIIEα wild type (Figure 5D, lanes 2–4). Unbound fractions were then mixed with Ni-NTA resin and FLAG-p62 PH-D bound to 6H–hTFIIEα was detected by western blotting (Figure 5D, lanes 7–9). This clearly demonstrates that p53 binding to p62 can be replaced with hTFIIEα.

DiscussionInteraction between hTFIIEα AC-D and hTFIIH p62 PH-D and its evolutionary conservationIn the present study, the specific interaction between hTFIIEα AC-D and hTFIIH p62 PH-D was explored, and structures of both the free and PH-D-bound forms of hTFIIEα AC-D were determined. This is the first report of the structural determination of the complex describing the interaction between TFIIE and TFIIH at the molecular level. In the case of hTFIIEα AC-D, its binding site as identified here (residues 378–395) is consistent with the previous report that a deletion mutant of hTFIIEα, Δ377–393 could no longer bind to hTFIIH, whereas a mutant with residues 351–439 could bind (Ohkuma et al, 1995). hTFIIEα possesses another acidic region, the STDE (Ser, Thr, Asp and Glu-rich) region (residues 352–365) immediately before the hTFIIEα AC-D (Figure 1A). To investigate whether the STDE is involved in the binding, we prepared a longer construct (residues 351–439) containing both acidic regions and performed the NMR titration experiment under the same conditions (Supplementary Figure 4). The result was that the NMR signals of STDE showed no significant changes and the Kd of 400±43 nM was almost the same as that estimated using hTFIIEα AC-D. We also examined the binding ability of peptide possessing only an N-terminal tail (AC-D381−394) (Supplementary Figure 5). hTFIIEα AC-D381−394 bound to p62 PH-D with a Kd of 2123±192 nM, which is about ∼6- to 9-fold weaker than that of hTFIIEα AC-D. Although the residues of p62 PH-D whose signals changed significantly were mostly consistent with the binding of AC-D and AC-D381−394, the extents of signal changes in the C-terminal region, to which I395, R432 and M433 of AC-D bind, were reduced. These results clearly indicate that hTFIIEα AC-D, which contains both the core structure and the flexible tail, is necessary and sufficient for the specific binding. This is an entirely new binding mode compared with the canonical binding modes found in some transcriptional activators or repressors, in which an intrinsically disordered region (Dyson and Wright, 2002) of each activation or repression domain binds to a target protein with part of the flexible region forming an ordered structure upon binding to the target. The core structure of hTFIIEα AC-D is essential for its binding to p62 PH-D in addition to its flexible arm.The binding site of hTFIIH p62 PH-D was localized to the second β-sheet (S5, S6 and S7), the loops between S1 and S2 and between S5 and S6 and the C-terminal H1 helix, where a substantial positive cluster is formed. Therefore, it is reasonable to speculate that the N-terminal highly acidic tail of hTFIIEα AC-D strongly binds to the positively charged surface of hTFIIH p62 PH-D. This is supported by the result that the binding is strengthened by removing NaCl from the buffer in the NMR titration experiments. However, given that the first acidic region, STDE, of hTFIIEα has no effect on the binding, the interaction is not merely based on electrostatic interactions. As seen in the complex structure, the highly conserved F387 and V390 residues in the acidic region of hTFIIEα AC-D, not in the STDE region (Figure 1A) make a large contribution to binding. Thus, the combination of electrostatic and hydrophobic interactions is essential for specific binding.It is interesting that hTFIIEα AC-D partially shares the binding surface of hTFIIH p62 PH-D with acidic transcriptional activators, p53 and VP16 TADs (Di Lello et al, 2005, 2006). As p53 and VP16 TADs are able to bind to the PH-Ds of both p62 and Tfb1, their interactions are likely to be evolutionally conserved. For yeast TFIIEα (Tfa1), functional significance of the C-terminal region and specific interaction between Tfa1 and Tfb1 has been reported (Bushnell et al, 1996; Kuldell and Buratowski, 1997). We aligned sequences of TFIIEα AC-D from other species to ascertain whether the interaction observed in human is evolutionally conserved (Figure 1A). As described above, the residues forming the hydrophobic core are well conserved in metazoans, but not in yeast, S. cerevisiae and Schizosaccharomyces pombe. Thus, yeast homologues are unlikely to have a similar core structure to that of human AC-D. However, in yeast similar sequences to the binding site are found in the equivalent positions. Furthermore, in contrast to metazoans, only fungal homologues possess the third acidic region at the C termini. Surprisingly, similar sequences to the binding site are found in the third acidic regions of both S. cerevisiae and S. pombe. Considering that the main binding site of hTFIIEα AC-D is located on the N-terminal tail outside the core structure, Tfa1 does not seem to have a similar core AC-D structure, but the interaction with PH-D is likely to be conserved. This suggests that the interplay between hTFIIEα AC-D, hTFIIH p62 PH-D and p53 TAD2 (and VP16 TAD as well) is evolutionally conserved.Functional roles of hTFIIEα AC-DIn hTFIIEα AC-D, F387 and V390 are the centre of the p62 PH-D-binding region (Figure 4B). As expected, the p62 PH-D-binding defective mutants (F387A, F387E, V390A and V390K) showed defects in in vitro transcription (20–40% reduction from the wild type, Figure 4C, lane 2 versus lanes 9–12). The reason why transcription did not correlate well with the binding and CTD phosphorylation defects might be because there are more than 25 proteins involved in transcription, whereas only limited factors were used for both binding and phosphorylation studies (hTFIIEα AC-D mutants and p62 PH-D were used for binding studies and hTFIIEα AC-D mutants, hTFIIH and Pol II were used for CTD phosphorylation). As a result, other GTFs will be able to support the recruitment of hTFIIH to the correct position in the PIC even in the absence of hTFIIEα AC-D. In addition, bigger defects were observed for mutations of the STDE and in the N-terminal half of hTFIIEα AC-D (S365, V372, D380 and E383) (Figure 4C, lanes 3–8). This effect maybe a result of the action of GTFs with the possibility that this region of hTFIIEα exerts an effect as a binding site for one or more of the general factors. We will test this possibility in the immediate future.Functional implication of the interplay between TFIIE, TFIIH and acidic transcriptional activatorsIn many cases, acidic TADs are disordered in an unbound form, but form an amphipathic helix upon binding to target proteins, for example, p53 TAD2–RPA70 (replication protein A 70) (Bochkareva et al, 2005), p53 TAD1–MDM2 (ubiquitin ligase) (Kussie et al, 1996), VP16 TAD–hTAFII31 (human TBP-associated factor) (Uesugi et al, 1997) complexes as well as the recently determined Tfb1 PH-D–p53 TAD2 complex (Di Lello et al, 2006). In contrast, hTFIIEα AC-D uniquely binds to p62 PH-D through its core structure together with its flexible N-terminal tail. The Kd values for the binding of p53 TAD2 to p62 and Tfb1 PH-Ds determined by isothermal titration calorimetry (ITC) are 3175±570 and 391±74 nM, respectively (Di Lello et al, 2006). In the binding of VP16 TAD to Tfb1 PH-D, the Kd value estimated by NMR titration experiment was ∼4000–7000 nM (Di Lello et al, 2005). Compared with these Kd values, the binding of hTFIIEα AC-D to p62 PH-D is rather strong. One of the well-known functions of transcriptional activators is to promote transcription initiation by increasing recruitment efficiency of Pol II and GTFs (Ptashne and Gann, 1997). TFIIH is recruited through many activator-mediated routes. p62 has been shown to interact with the TADs of not only VP16 and p53 (Xiao et al, 1994) but also E2F-1 (Pearson and Greenblatt, 1997), and the oestrogen receptor α (ERα) (Chen et al, 2000). Of these, it was demonstrated that p53, VP16 and ERα target p62 PH-D. The notable finding from the present study is that hTFIIEα also targets p62 PH-D (Figure 2). To our knowledge, this is the first GTF demonstrated to possess a functional TAD-like motif. As shown in Figure 5D, hTFIIEα could replace p53 TAD and then bind to p62 PH-D. It can be imagined that if TFIIH is recruited by an activator near the promoter through its TAD then the recruited TFIIH could be captured by TFIIE instead of the activator to form the PIC. TFIIE regulates the enzymatic activities of TFIIH, which are necessary for the next stage after the PIC formation, that is, promoter melting or promoter clearance. In contrast, activators cannot directly participate in these processes. Efficient delivery of p62 from activators to TFIIEα is considered to be essential for the final recruitment of TFIIH to form the PIC in vivo. The replacement observed in Figure 5D could be related to this final recruitment. We show in Figure 5A–C that p53 (Di Lello et al, 2006) and VP16 (Di Lello et al, 2005) use part of the binding surface formed between p62 PH-D and hTFIIEα AC-D. The use of an overlapping binding surface on p62 PH-D with p53 TAD2 may be advantageous in the delivery of TFIIH from activators to TFIIE.The binding affinity of p53 TAD2 to p62 PH-D is regulated by S46 and T55 phosphorylation (Di Lello et al, 2006). The Kd values of p53 TAD2 to p62 PH-D are reported for the unphosphorylated form as 3175±570 nM, for the S46-phosphorylated form as 518±92 nM, for the T55-phosphorylated form as 457±75 nM and for both S46- and T55-phosphorylated form as 97±33 nM. Very recent ITC studies demonstrated the Kd value of hTFIIEα336−439 to p62 PH-D to be 45±25 nM (Di Lello et al, 2008). Intriguingly, the binding affinity of hTFIIEα336−439 to p62 PH-D is much stronger than unphosphorylated p53 TAD2 and is comparable to doubly phosphorylated TAD2. As both TFIIH and p53 function not only in transcription but also in DNA repair (Kastan et al, 1991; Drapkin et al, 1994) and p62 PH-D is involved in nucleotide excision repair (Gervais et al, 2004), p62 may serve as a molecular switch between transcription and DNA repair. We imagine that when p53 TAD2 is unphosphorylated, the delivery of p62 from p53 TAD2 to hTFIIEα would be efficient and they would function cooperatively in transcription initiation. However, when p53 TAD2 is doubly phosphorylated at S46 and T55, the affinity of p53 TAD2 for p62 PH-D would be comparable to that for hTFIIEα resulting in p53 and p62 (hTFIIH) functioning in DNA repair or in processes other than transactivation. Further studies are required to verify these possibilities.

Interaction between hTFIIEα AC-D and hTFIIH p62 PH-D and its evolutionary conservationIn the present study, the specific interaction between hTFIIEα AC-D and hTFIIH p62 PH-D was explored, and structures of both the free and PH-D-bound forms of hTFIIEα AC-D were determined. This is the first report of the structural determination of the complex describing the interaction between TFIIE and TFIIH at the molecular level. In the case of hTFIIEα AC-D, its binding site as identified here (residues 378–395) is consistent with the previous report that a deletion mutant of hTFIIEα, Δ377–393 could no longer bind to hTFIIH, whereas a mutant with residues 351–439 could bind (Ohkuma et al, 1995). hTFIIEα possesses another acidic region, the STDE (Ser, Thr, Asp and Glu-rich) region (residues 352–365) immediately before the hTFIIEα AC-D (Figure 1A). To investigate whether the STDE is involved in the binding, we prepared a longer construct (residues 351–439) containing both acidic regions and performed the NMR titration experiment under the same conditions (Supplementary Figure 4). The result was that the NMR signals of STDE showed no significant changes and the Kd of 400±43 nM was almost the same as that estimated using hTFIIEα AC-D. We also examined the binding ability of peptide possessing only an N-terminal tail (AC-D381−394) (Supplementary Figure 5). hTFIIEα AC-D381−394 bound to p62 PH-D with a Kd of 2123±192 nM, which is about ∼6- to 9-fold weaker than that of hTFIIEα AC-D. Although the residues of p62 PH-D whose signals changed significantly were mostly consistent with the binding of AC-D and AC-D381−394, the extents of signal changes in the C-terminal region, to which I395, R432 and M433 of AC-D bind, were reduced. These results clearly indicate that hTFIIEα AC-D, which contains both the core structure and the flexible tail, is necessary and sufficient for the specific binding. This is an entirely new binding mode compared with the canonical binding modes found in some transcriptional activators or repressors, in which an intrinsically disordered region (Dyson and Wright, 2002) of each activation or repression domain binds to a target protein with part of the flexible region forming an ordered structure upon binding to the target. The core structure of hTFIIEα AC-D is essential for its binding to p62 PH-D in addition to its flexible arm.The binding site of hTFIIH p62 PH-D was localized to the second β-sheet (S5, S6 and S7), the loops between S1 and S2 and between S5 and S6 and the C-terminal H1 helix, where a substantial positive cluster is formed. Therefore, it is reasonable to speculate that the N-terminal highly acidic tail of hTFIIEα AC-D strongly binds to the positively charged surface of hTFIIH p62 PH-D. This is supported by the result that the binding is strengthened by removing NaCl from the buffer in the NMR titration experiments. However, given that the first acidic region, STDE, of hTFIIEα has no effect on the binding, the interaction is not merely based on electrostatic interactions. As seen in the complex structure, the highly conserved F387 and V390 residues in the acidic region of hTFIIEα AC-D, not in the STDE region (Figure 1A) make a large contribution to binding. Thus, the combination of electrostatic and hydrophobic interactions is essential for specific binding.It is interesting that hTFIIEα AC-D partially shares the binding surface of hTFIIH p62 PH-D with acidic transcriptional activators, p53 and VP16 TADs (Di Lello et al, 2005, 2006). As p53 and VP16 TADs are able to bind to the PH-Ds of both p62 and Tfb1, their interactions are likely to be evolutionally conserved. For yeast TFIIEα (Tfa1), functional significance of the C-terminal region and specific interaction between Tfa1 and Tfb1 has been reported (Bushnell et al, 1996; Kuldell and Buratowski, 1997). We aligned sequences of TFIIEα AC-D from other species to ascertain whether the interaction observed in human is evolutionally conserved (Figure 1A). As described above, the residues forming the hydrophobic core are well conserved in metazoans, but not in yeast, S. cerevisiae and Schizosaccharomyces pombe. Thus, yeast homologues are unlikely to have a similar core structure to that of human AC-D. However, in yeast similar sequences to the binding site are found in the equivalent positions. Furthermore, in contrast to metazoans, only fungal homologues possess the third acidic region at the C termini. Surprisingly, similar sequences to the binding site are found in the third acidic regions of both S. cerevisiae and S. pombe. Considering that the main binding site of hTFIIEα AC-D is located on the N-terminal tail outside the core structure, Tfa1 does not seem to have a similar core AC-D structure, but the interaction with PH-D is likely to be conserved. This suggests that the interplay between hTFIIEα AC-D, hTFIIH p62 PH-D and p53 TAD2 (and VP16 TAD as well) is evolutionally conserved.

Interaction between hTFIIEα AC-D and hTFIIH p62 PH-D and its evolutionary conservation

In the present study, the specific interaction between hTFIIEα AC-D and hTFIIH p62 PH-D was explored, and structures of both the free and PH-D-bound forms of hTFIIEα AC-D were determined. This is the first report of the structural determination of the complex describing the interaction between TFIIE and TFIIH at the molecular level. In the case of hTFIIEα AC-D, its binding site as identified here (residues 378–395) is consistent with the previous report that a deletion mutant of hTFIIEα, Δ377–393 could no longer bind to hTFIIH, whereas a mutant with residues 351–439 could bind (Ohkuma et al, 1995). hTFIIEα possesses another acidic region, the STDE (Ser, Thr, Asp and Glu-rich) region (residues 352–365) immediately before the hTFIIEα AC-D (Figure 1A). To investigate whether the STDE is involved in the binding, we prepared a longer construct (residues 351–439) containing both acidic regions and performed the NMR titration experiment under the same conditions (Supplementary Figure 4). The result was that the NMR signals of STDE showed no significant changes and the Kd of 400±43 nM was almost the same as that estimated using hTFIIEα AC-D. We also examined the binding ability of peptide possessing only an N-terminal tail (AC-D381−394) (Supplementary Figure 5). hTFIIEα AC-D381−394 bound to p62 PH-D with a Kd of 2123±192 nM, which is about ∼6- to 9-fold weaker than that of hTFIIEα AC-D. Although the residues of p62 PH-D whose signals changed significantly were mostly consistent with the binding of AC-D and AC-D381−394, the extents of signal changes in the C-terminal region, to which I395, R432 and M433 of AC-D bind, were reduced. These results clearly indicate that hTFIIEα AC-D, which contains both the core structure and the flexible tail, is necessary and sufficient for the specific binding. This is an entirely new binding mode compared with the canonical binding modes found in some transcriptional activators or repressors, in which an intrinsically disordered region (Dyson and Wright, 2002) of each activation or repression domain binds to a target protein with part of the flexible region forming an ordered structure upon binding to the target. The core structure of hTFIIEα AC-D is essential for its binding to p62 PH-D in addition to its flexible arm.

The binding site of hTFIIH p62 PH-D was localized to the second β-sheet (S5, S6 and S7), the loops between S1 and S2 and between S5 and S6 and the C-terminal H1 helix, where a substantial positive cluster is formed. Therefore, it is reasonable to speculate that the N-terminal highly acidic tail of hTFIIEα AC-D strongly binds to the positively charged surface of hTFIIH p62 PH-D. This is supported by the result that the binding is strengthened by removing NaCl from the buffer in the NMR titration experiments. However, given that the first acidic region, STDE, of hTFIIEα has no effect on the binding, the interaction is not merely based on electrostatic interactions. As seen in the complex structure, the highly conserved F387 and V390 residues in the acidic region of hTFIIEα AC-D, not in the STDE region (Figure 1A) make a large contribution to binding. Thus, the combination of electrostatic and hydrophobic interactions is essential for specific binding.

It is interesting that hTFIIEα AC-D partially shares the binding surface of hTFIIH p62 PH-D with acidic transcriptional activators, p53 and VP16 TADs (Di Lello et al, 2005, 2006). As p53 and VP16 TADs are able to bind to the PH-Ds of both p62 and Tfb1, their interactions are likely to be evolutionally conserved. For yeast TFIIEα (Tfa1), functional significance of the C-terminal region and specific interaction between Tfa1 and Tfb1 has been reported (Bushnell et al, 1996; Kuldell and Buratowski, 1997). We aligned sequences of TFIIEα AC-D from other species to ascertain whether the interaction observed in human is evolutionally conserved (Figure 1A). As described above, the residues forming the hydrophobic core are well conserved in metazoans, but not in yeast, S. cerevisiae and Schizosaccharomyces pombe. Thus, yeast homologues are unlikely to have a similar core structure to that of human AC-D. However, in yeast similar sequences to the binding site are found in the equivalent positions. Furthermore, in contrast to metazoans, only fungal homologues possess the third acidic region at the C termini. Surprisingly, similar sequences to the binding site are found in the third acidic regions of both S. cerevisiae and S. pombe. Considering that the main binding site of hTFIIEα AC-D is located on the N-terminal tail outside the core structure, Tfa1 does not seem to have a similar core AC-D structure, but the interaction with PH-D is likely to be conserved. This suggests that the interplay between hTFIIEα AC-D, hTFIIH p62 PH-D and p53 TAD2 (and VP16 TAD as well) is evolutionally conserved.

Functional roles of hTFIIEα AC-DIn hTFIIEα AC-D, F387 and V390 are the centre of the p62 PH-D-binding region (Figure 4B). As expected, the p62 PH-D-binding defective mutants (F387A, F387E, V390A and V390K) showed defects in in vitro transcription (20–40% reduction from the wild type, Figure 4C, lane 2 versus lanes 9–12). The reason why transcription did not correlate well with the binding and CTD phosphorylation defects might be because there are more than 25 proteins involved in transcription, whereas only limited factors were used for both binding and phosphorylation studies (hTFIIEα AC-D mutants and p62 PH-D were used for binding studies and hTFIIEα AC-D mutants, hTFIIH and Pol II were used for CTD phosphorylation). As a result, other GTFs will be able to support the recruitment of hTFIIH to the correct position in the PIC even in the absence of hTFIIEα AC-D. In addition, bigger defects were observed for mutations of the STDE and in the N-terminal half of hTFIIEα AC-D (S365, V372, D380 and E383) (Figure 4C, lanes 3–8). This effect maybe a result of the action of GTFs with the possibility that this region of hTFIIEα exerts an effect as a binding site for one or more of the general factors. We will test this possibility in the immediate future.

Functional roles of hTFIIEα AC-D

In hTFIIEα AC-D, F387 and V390 are the centre of the p62 PH-D-binding region (Figure 4B). As expected, the p62 PH-D-binding defective mutants (F387A, F387E, V390A and V390K) showed defects in in vitro transcription (20–40% reduction from the wild type, Figure 4C, lane 2 versus lanes 9–12). The reason why transcription did not correlate well with the binding and CTD phosphorylation defects might be because there are more than 25 proteins involved in transcription, whereas only limited factors were used for both binding and phosphorylation studies (hTFIIEα AC-D mutants and p62 PH-D were used for binding studies and hTFIIEα AC-D mutants, hTFIIH and Pol II were used for CTD phosphorylation). As a result, other GTFs will be able to support the recruitment of hTFIIH to the correct position in the PIC even in the absence of hTFIIEα AC-D. In addition, bigger defects were observed for mutations of the STDE and in the N-terminal half of hTFIIEα AC-D (S365, V372, D380 and E383) (Figure 4C, lanes 3–8). This effect maybe a result of the action of GTFs with the possibility that this region of hTFIIEα exerts an effect as a binding site for one or more of the general factors. We will test this possibility in the immediate future.

Functional implication of the interplay between TFIIE, TFIIH and acidic transcriptional activatorsIn many cases, acidic TADs are disordered in an unbound form, but form an amphipathic helix upon binding to target proteins, for example, p53 TAD2–RPA70 (replication protein A 70) (Bochkareva et al, 2005), p53 TAD1–MDM2 (ubiquitin ligase) (Kussie et al, 1996), VP16 TAD–hTAFII31 (human TBP-associated factor) (Uesugi et al, 1997) complexes as well as the recently determined Tfb1 PH-D–p53 TAD2 complex (Di Lello et al, 2006). In contrast, hTFIIEα AC-D uniquely binds to p62 PH-D through its core structure together with its flexible N-terminal tail. The Kd values for the binding of p53 TAD2 to p62 and Tfb1 PH-Ds determined by isothermal titration calorimetry (ITC) are 3175±570 and 391±74 nM, respectively (Di Lello et al, 2006). In the binding of VP16 TAD to Tfb1 PH-D, the Kd value estimated by NMR titration experiment was ∼4000–7000 nM (Di Lello et al, 2005). Compared with these Kd values, the binding of hTFIIEα AC-D to p62 PH-D is rather strong. One of the well-known functions of transcriptional activators is to promote transcription initiation by increasing recruitment efficiency of Pol II and GTFs (Ptashne and Gann, 1997). TFIIH is recruited through many activator-mediated routes. p62 has been shown to interact with the TADs of not only VP16 and p53 (Xiao et al, 1994) but also E2F-1 (Pearson and Greenblatt, 1997), and the oestrogen receptor α (ERα) (Chen et al, 2000). Of these, it was demonstrated that p53, VP16 and ERα target p62 PH-D. The notable finding from the present study is that hTFIIEα also targets p62 PH-D (Figure 2). To our knowledge, this is the first GTF demonstrated to possess a functional TAD-like motif. As shown in Figure 5D, hTFIIEα could replace p53 TAD and then bind to p62 PH-D. It can be imagined that if TFIIH is recruited by an activator near the promoter through its TAD then the recruited TFIIH could be captured by TFIIE instead of the activator to form the PIC. TFIIE regulates the enzymatic activities of TFIIH, which are necessary for the next stage after the PIC formation, that is, promoter melting or promoter clearance. In contrast, activators cannot directly participate in these processes. Efficient delivery of p62 from activators to TFIIEα is considered to be essential for the final recruitment of TFIIH to form the PIC in vivo. The replacement observed in Figure 5D could be related to this final recruitment. We show in Figure 5A–C that p53 (Di Lello et al, 2006) and VP16 (Di Lello et al, 2005) use part of the binding surface formed between p62 PH-D and hTFIIEα AC-D. The use of an overlapping binding surface on p62 PH-D with p53 TAD2 may be advantageous in the delivery of TFIIH from activators to TFIIE.The binding affinity of p53 TAD2 to p62 PH-D is regulated by S46 and T55 phosphorylation (Di Lello et al, 2006). The Kd values of p53 TAD2 to p62 PH-D are reported for the unphosphorylated form as 3175±570 nM, for the S46-phosphorylated form as 518±92 nM, for the T55-phosphorylated form as 457±75 nM and for both S46- and T55-phosphorylated form as 97±33 nM. Very recent ITC studies demonstrated the Kd value of hTFIIEα336−439 to p62 PH-D to be 45±25 nM (Di Lello et al, 2008). Intriguingly, the binding affinity of hTFIIEα336−439 to p62 PH-D is much stronger than unphosphorylated p53 TAD2 and is comparable to doubly phosphorylated TAD2. As both TFIIH and p53 function not only in transcription but also in DNA repair (Kastan et al, 1991; Drapkin et al, 1994) and p62 PH-D is involved in nucleotide excision repair (Gervais et al, 2004), p62 may serve as a molecular switch between transcription and DNA repair. We imagine that when p53 TAD2 is unphosphorylated, the delivery of p62 from p53 TAD2 to hTFIIEα would be efficient and they would function cooperatively in transcription initiation. However, when p53 TAD2 is doubly phosphorylated at S46 and T55, the affinity of p53 TAD2 for p62 PH-D would be comparable to that for hTFIIEα resulting in p53 and p62 (hTFIIH) functioning in DNA repair or in processes other than transactivation. Further studies are required to verify these possibilities.

Functional implication of the interplay between TFIIE, TFIIH and acidic transcriptional activators

In many cases, acidic TADs are disordered in an unbound form, but form an amphipathic helix upon binding to target proteins, for example, p53 TAD2–RPA70 (replication protein A 70) (Bochkareva et al, 2005), p53 TAD1–MDM2 (ubiquitin ligase) (Kussie et al, 1996), VP16 TAD–hTAFII31 (human TBP-associated factor) (Uesugi et al, 1997) complexes as well as the recently determined Tfb1 PH-D–p53 TAD2 complex (Di Lello et al, 2006). In contrast, hTFIIEα AC-D uniquely binds to p62 PH-D through its core structure together with its flexible N-terminal tail. The Kd values for the binding of p53 TAD2 to p62 and Tfb1 PH-Ds determined by isothermal titration calorimetry (ITC) are 3175±570 and 391±74 nM, respectively (Di Lello et al, 2006). In the binding of VP16 TAD to Tfb1 PH-D, the Kd value estimated by NMR titration experiment was ∼4000–7000 nM (Di Lello et al, 2005). Compared with these Kd values, the binding of hTFIIEα AC-D to p62 PH-D is rather strong. One of the well-known functions of transcriptional activators is to promote transcription initiation by increasing recruitment efficiency of Pol II and GTFs (Ptashne and Gann, 1997). TFIIH is recruited through many activator-mediated routes. p62 has been shown to interact with the TADs of not only VP16 and p53 (Xiao et al, 1994) but also E2F-1 (Pearson and Greenblatt, 1997), and the oestrogen receptor α (ERα) (Chen et al, 2000). Of these, it was demonstrated that p53, VP16 and ERα target p62 PH-D. The notable finding from the present study is that hTFIIEα also targets p62 PH-D (Figure 2). To our knowledge, this is the first GTF demonstrated to possess a functional TAD-like motif. As shown in Figure 5D, hTFIIEα could replace p53 TAD and then bind to p62 PH-D. It can be imagined that if TFIIH is recruited by an activator near the promoter through its TAD then the recruited TFIIH could be captured by TFIIE instead of the activator to form the PIC. TFIIE regulates the enzymatic activities of TFIIH, which are necessary for the next stage after the PIC formation, that is, promoter melting or promoter clearance. In contrast, activators cannot directly participate in these processes. Efficient delivery of p62 from activators to TFIIEα is considered to be essential for the final recruitment of TFIIH to form the PIC in vivo. The replacement observed in Figure 5D could be related to this final recruitment. We show in Figure 5A–C that p53 (Di Lello et al, 2006) and VP16 (Di Lello et al, 2005) use part of the binding surface formed between p62 PH-D and hTFIIEα AC-D. The use of an overlapping binding surface on p62 PH-D with p53 TAD2 may be advantageous in the delivery of TFIIH from activators to TFIIE.

The binding affinity of p53 TAD2 to p62 PH-D is regulated by S46 and T55 phosphorylation (Di Lello et al, 2006). The Kd values of p53 TAD2 to p62 PH-D are reported for the unphosphorylated form as 3175±570 nM, for the S46-phosphorylated form as 518±92 nM, for the T55-phosphorylated form as 457±75 nM and for both S46- and T55-phosphorylated form as 97±33 nM. Very recent ITC studies demonstrated the Kd value of hTFIIEα336−439 to p62 PH-D to be 45±25 nM (Di Lello et al, 2008). Intriguingly, the binding affinity of hTFIIEα336−439 to p62 PH-D is much stronger than unphosphorylated p53 TAD2 and is comparable to doubly phosphorylated TAD2. As both TFIIH and p53 function not only in transcription but also in DNA repair (Kastan et al, 1991; Drapkin et al, 1994) and p62 PH-D is involved in nucleotide excision repair (Gervais et al, 2004), p62 may serve as a molecular switch between transcription and DNA repair. We imagine that when p53 TAD2 is unphosphorylated, the delivery of p62 from p53 TAD2 to hTFIIEα would be efficient and they would function cooperatively in transcription initiation. However, when p53 TAD2 is doubly phosphorylated at S46 and T55, the affinity of p53 TAD2 for p62 PH-D would be comparable to that for hTFIIEα resulting in p53 and p62 (hTFIIH) functioning in DNA repair or in processes other than transactivation. Further studies are required to verify these possibilities.

Materials and methodsConstruction of mutants of hTFIIEα and hTFIIH p62 subunitsBy using the Multi Site-Directed Mutagenesis Kit (Medical Biology Laboratory) as a template with wild-type hTFIIEα cDNA plasmid or a hTFIIH p62 cDNA mutant in which a NdeI site was disrupted by changing the third nucleotide of the 45th histidine codon of wild-type hTFIIH p62 cDNA, T to C, various oligonucleotide-mediated point mutants were created (Kunkel et al, 1987). The mutants were checked by sequencing using an ABI Prism 310 Genetic Analyzer (Applied Biosystems). The oligonucleotides used for mutation are listed in Supplementary Table I. The NdeI–BamHI fragment of mutated hTFIIEα cDNA was subcloned into the pET11d vector (Novagen) making the N-terminal hexa histidine-tagged hTFIIEα (6H–hTFIIEα) expression plasmid. The NdeI–BamHI fragment of mutated hTFIIH p62 cDNA was subcloned into the pET vector making the N-terminal FLAG-tagged hTFIIH p62 PH-D (FLAG–p62 PH) expression plasmid.Purification of hTFIIEα AC-D and hTFIIH p62 PH-DhTFIIEα AC-D (residues 378–439) was expressed as an hexa histidine-tagged product in pET3a vectors (Novagen) in E. coli BL21(DE3)pLysS (Novagen). Lysed supernatant was loaded onto the Ni-NTA-agarose (Qiagen) column. The eluate was then applied onto Q-Sepharose (GE Healthcare). After digestion with thrombin to remove the 6H tag, sample was again loaded onto the Ni-NTA agarose column. Fractions passing through the column were concentrated and applied onto Superdex30 (GE Healthcare). hTFIIH p62 PH-D was purified as described (Gervais et al, 2004).NMR spectroscopyMeasurements of NMR spectra, structural calculations and NMR titration experiments are described in Supplementary data.Purification of recombinant proteinsRecombinant point mutant hTFIIEα proteins were expressed in E. coli Rosetta™(DE3)pLysS (Novagen), and recombinant hTFIIH p62 point mutants were expressed in BL21(DE3)pLysS by induction with isopropyl-β-D-thiogalactopyranoside (Studier et al, 1990). The purification method of these recombinant proteins was as described previously (Watanabe et al, 2003). Typical preparations were >90% pure, judging by Coomassie blue staining of an SDS–polyacrylamide gel.GST pull-down assayGST fusion proteins were used for protein interaction assays (Okamoto et al, 1998). The bound proteins were released by boiling in SDS–PAGE loading buffer, separated by SDS–PAGE and detected by western blotting with anti-hTFIIEα rabbit antiserum (1:3000 dilution), anti-FLAG M2 monoclonal antibody (Sigma) and anti-p53 (DO-1) (Santa Cruz) using the enhanced chemiluminescence detection system (GE Healthcare).In vitro transcription assayRecombinant GTFs as well as native Pol II and hTFIIH were purified as described previously (Watanabe et al, 2003). In vitro transcription was performed as described (Ohkuma et al, 1995). The plasmid pML(C2AT)Δ-50 containing the adenovirus 2 major late promoter, which gives a 390-nt transcript, was used as either a supercoiled or a linearized template for basal transcription assays (Yamamoto et al, 2001). To prepare the linearized template, pML(C2AT)Δ-50 was digested with SmaI. After transcription, radiolabeled transcripts were subjected to urea–PAGE and detected by autoradiography. The transcripts were quantified by a Fuji BAS5000 Bio-Imaging Analyzer. Relative transcription activities of the mutant hTFIIEα proteins were calculated by defining the transcription activity of the wild-type hTFIIEα as 100%.Kinase assayIn addition to transcription factors added as described in the legend of Figure 4D, the kinase reaction mixture (25 μl) contained 5 mM HEPES-KOH (pH 7.8), 20 mM Tris–HCl (pH 7.9 at 4°C), 7 mM MgCl2, 60 mM KCl, 12% (v/v) glycerol, 2% (w/v) polyethylene glycol 8000, 2 mM 2-mercaptoethanol, 0.1 mM EDTA, 240 μg/ml of bovine serum albumin, 5 μM ATP and 3 μCi of [γ-32P]ATP. Phosphorylation reactions were carried out at 30°C for 1 h and stopped by addition of 75 μl of phosphorylation stop solution (10 mM EDTA, 0.1% NP40 and 0.05% SDS). Phosphorylated proteins were precipitated with TCA, analysed by SDS–PAGE (5.5% acrylamide), and detected by autoradiography with Fuji RX-U X-ray film.Accession numbersCoordinates of hTFIIEα AC-D free and in complex with hTFIIH p62 PH-D have been deposited in the Protein Data Bank (www.rcsb.org) under accession codes 2RNQ and 2RNR, respectively.

Materials and methods

Construction of mutants of hTFIIEα and hTFIIH p62 subunitsBy using the Multi Site-Directed Mutagenesis Kit (Medical Biology Laboratory) as a template with wild-type hTFIIEα cDNA plasmid or a hTFIIH p62 cDNA mutant in which a NdeI site was disrupted by changing the third nucleotide of the 45th histidine codon of wild-type hTFIIH p62 cDNA, T to C, various oligonucleotide-mediated point mutants were created (Kunkel et al, 1987). The mutants were checked by sequencing using an ABI Prism 310 Genetic Analyzer (Applied Biosystems). The oligonucleotides used for mutation are listed in Supplementary Table I. The NdeI–BamHI fragment of mutated hTFIIEα cDNA was subcloned into the pET11d vector (Novagen) making the N-terminal hexa histidine-tagged hTFIIEα (6H–hTFIIEα) expression plasmid. The NdeI–BamHI fragment of mutated hTFIIH p62 cDNA was subcloned into the pET vector making the N-terminal FLAG-tagged hTFIIH p62 PH-D (FLAG–p62 PH) expression plasmid.

Construction of mutants of hTFIIEα and hTFIIH p62 subunits

By using the Multi Site-Directed Mutagenesis Kit (Medical Biology Laboratory) as a template with wild-type hTFIIEα cDNA plasmid or a hTFIIH p62 cDNA mutant in which a NdeI site was disrupted by changing the third nucleotide of the 45th histidine codon of wild-type hTFIIH p62 cDNA, T to C, various oligonucleotide-mediated point mutants were created (Kunkel et al, 1987). The mutants were checked by sequencing using an ABI Prism 310 Genetic Analyzer (Applied Biosystems). The oligonucleotides used for mutation are listed in Supplementary Table I. The NdeI–BamHI fragment of mutated hTFIIEα cDNA was subcloned into the pET11d vector (Novagen) making the N-terminal hexa histidine-tagged hTFIIEα (6H–hTFIIEα) expression plasmid. The NdeI–BamHI fragment of mutated hTFIIH p62 cDNA was subcloned into the pET vector making the N-terminal FLAG-tagged hTFIIH p62 PH-D (FLAG–p62 PH) expression plasmid.

Purification of hTFIIEα AC-D and hTFIIH p62 PH-DhTFIIEα AC-D (residues 378–439) was expressed as an hexa histidine-tagged product in pET3a vectors (Novagen) in E. coli BL21(DE3)pLysS (Novagen). Lysed supernatant was loaded onto the Ni-NTA-agarose (Qiagen) column. The eluate was then applied onto Q-Sepharose (GE Healthcare). After digestion with thrombin to remove the 6H tag, sample was again loaded onto the Ni-NTA agarose column. Fractions passing through the column were concentrated and applied onto Superdex30 (GE Healthcare). hTFIIH p62 PH-D was purified as described (Gervais et al, 2004).

Purification of hTFIIEα AC-D and hTFIIH p62 PH-D

hTFIIEα AC-D (residues 378–439) was expressed as an hexa histidine-tagged product in pET3a vectors (Novagen) in E. coli BL21(DE3)pLysS (Novagen). Lysed supernatant was loaded onto the Ni-NTA-agarose (Qiagen) column. The eluate was then applied onto Q-Sepharose (GE Healthcare). After digestion with thrombin to remove the 6H tag, sample was again loaded onto the Ni-NTA agarose column. Fractions passing through the column were concentrated and applied onto Superdex30 (GE Healthcare). hTFIIH p62 PH-D was purified as described (Gervais et al, 2004).

NMR spectroscopyMeasurements of NMR spectra, structural calculations and NMR titration experiments are described in Supplementary data.

NMR spectroscopy

Measurements of NMR spectra, structural calculations and NMR titration experiments are described in Supplementary data.

Purification of recombinant proteinsRecombinant point mutant hTFIIEα proteins were expressed in E. coli Rosetta™(DE3)pLysS (Novagen), and recombinant hTFIIH p62 point mutants were expressed in BL21(DE3)pLysS by induction with isopropyl-β-D-thiogalactopyranoside (Studier et al, 1990). The purification method of these recombinant proteins was as described previously (Watanabe et al, 2003). Typical preparations were >90% pure, judging by Coomassie blue staining of an SDS–polyacrylamide gel.

Purification of recombinant proteins

Recombinant point mutant hTFIIEα proteins were expressed in E. coli Rosetta™(DE3)pLysS (Novagen), and recombinant hTFIIH p62 point mutants were expressed in BL21(DE3)pLysS by induction with isopropyl-β-D-thiogalactopyranoside (Studier et al, 1990). The purification method of these recombinant proteins was as described previously (Watanabe et al, 2003). Typical preparations were >90% pure, judging by Coomassie blue staining of an SDS–polyacrylamide gel.

GST pull-down assayGST fusion proteins were used for protein interaction assays (Okamoto et al, 1998). The bound proteins were released by boiling in SDS–PAGE loading buffer, separated by SDS–PAGE and detected by western blotting with anti-hTFIIEα rabbit antiserum (1:3000 dilution), anti-FLAG M2 monoclonal antibody (Sigma) and anti-p53 (DO-1) (Santa Cruz) using the enhanced chemiluminescence detection system (GE Healthcare).

GST pull-down assay

GST fusion proteins were used for protein interaction assays (Okamoto et al, 1998). The bound proteins were released by boiling in SDS–PAGE loading buffer, separated by SDS–PAGE and detected by western blotting with anti-hTFIIEα rabbit antiserum (1:3000 dilution), anti-FLAG M2 monoclonal antibody (Sigma) and anti-p53 (DO-1) (Santa Cruz) using the enhanced chemiluminescence detection system (GE Healthcare).

In vitro transcription assayRecombinant GTFs as well as native Pol II and hTFIIH were purified as described previously (Watanabe et al, 2003). In vitro transcription was performed as described (Ohkuma et al, 1995). The plasmid pML(C2AT)Δ-50 containing the adenovirus 2 major late promoter, which gives a 390-nt transcript, was used as either a supercoiled or a linearized template for basal transcription assays (Yamamoto et al, 2001). To prepare the linearized template, pML(C2AT)Δ-50 was digested with SmaI. After transcription, radiolabeled transcripts were subjected to urea–PAGE and detected by autoradiography. The transcripts were quantified by a Fuji BAS5000 Bio-Imaging Analyzer. Relative transcription activities of the mutant hTFIIEα proteins were calculated by defining the transcription activity of the wild-type hTFIIEα as 100%.

In vitro transcription assay

Recombinant GTFs as well as native Pol II and hTFIIH were purified as described previously (Watanabe et al, 2003). In vitro transcription was performed as described (Ohkuma et al, 1995). The plasmid pML(C2AT)Δ-50 containing the adenovirus 2 major late promoter, which gives a 390-nt transcript, was used as either a supercoiled or a linearized template for basal transcription assays (Yamamoto et al, 2001). To prepare the linearized template, pML(C2AT)Δ-50 was digested with SmaI. After transcription, radiolabeled transcripts were subjected to urea–PAGE and detected by autoradiography. The transcripts were quantified by a Fuji BAS5000 Bio-Imaging Analyzer. Relative transcription activities of the mutant hTFIIEα proteins were calculated by defining the transcription activity of the wild-type hTFIIEα as 100%.

Kinase assayIn addition to transcription factors added as described in the legend of Figure 4D, the kinase reaction mixture (25 μl) contained 5 mM HEPES-KOH (pH 7.8), 20 mM Tris–HCl (pH 7.9 at 4°C), 7 mM MgCl2, 60 mM KCl, 12% (v/v) glycerol, 2% (w/v) polyethylene glycol 8000, 2 mM 2-mercaptoethanol, 0.1 mM EDTA, 240 μg/ml of bovine serum albumin, 5 μM ATP and 3 μCi of [γ-32P]ATP. Phosphorylation reactions were carried out at 30°C for 1 h and stopped by addition of 75 μl of phosphorylation stop solution (10 mM EDTA, 0.1% NP40 and 0.05% SDS). Phosphorylated proteins were precipitated with TCA, analysed by SDS–PAGE (5.5% acrylamide), and detected by autoradiography with Fuji RX-U X-ray film.

Kinase assay

In addition to transcription factors added as described in the legend of Figure 4D, the kinase reaction mixture (25 μl) contained 5 mM HEPES-KOH (pH 7.8), 20 mM Tris–HCl (pH 7.9 at 4°C), 7 mM MgCl2, 60 mM KCl, 12% (v/v) glycerol, 2% (w/v) polyethylene glycol 8000, 2 mM 2-mercaptoethanol, 0.1 mM EDTA, 240 μg/ml of bovine serum albumin, 5 μM ATP and 3 μCi of [γ-32P]ATP. Phosphorylation reactions were carried out at 30°C for 1 h and stopped by addition of 75 μl of phosphorylation stop solution (10 mM EDTA, 0.1% NP40 and 0.05% SDS). Phosphorylated proteins were precipitated with TCA, analysed by SDS–PAGE (5.5% acrylamide), and detected by autoradiography with Fuji RX-U X-ray film.

Accession numbersCoordinates of hTFIIEα AC-D free and in complex with hTFIIH p62 PH-D have been deposited in the Protein Data Bank (www.rcsb.org) under accession codes 2RNQ and 2RNR, respectively.

Accession numbers

Coordinates of hTFIIEα AC-D free and in complex with hTFIIH p62 PH-D have been deposited in the Protein Data Bank (www.rcsb.org) under accession codes 2RNQ and 2RNR, respectively.

Supplementary MaterialSupplementary Information

Supplementary Material

Supplementary Information
