---
pmid: '22842922'
title: Dissecting DNA damage response pathways by analysing protein localization and
  abundance changes during DNA replication stress.
authors:
- Tkach JM
- Yimit A
- Lee AY
- Riffle M
- Costanzo M
- Jaschob D
- Hendry JA
- Ou J
- Moffat J
- Boone C
- Davis TN
- Nislow C
- Brown GW
journal: Nat Cell Biol
year: '2012'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC3434236
doi: 10.1038/ncb2549
---

# Dissecting DNA damage response pathways by analysing protein localization and abundance changes during DNA replication stress.
**Authors:** Tkach JM, Yimit A, Lee AY, Riffle M, Costanzo M, Jaschob D, Hendry JA, Ou J, Moffat J, Boone C, Davis TN, Nislow C, Brown GW
**Journal:** Nat Cell Biol (2012)
**DOI:** [10.1038/ncb2549](https://doi.org/10.1038/ncb2549)
**PMC:** [PMC3434236](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3434236/)

## Abstract

1. Nat Cell Biol. 2012 Sep;14(9):966-76. doi: 10.1038/ncb2549. Epub 2012 Jul 29.

Dissecting DNA damage response pathways by analysing protein localization and 
abundance changes during DNA replication stress.

Tkach JM(1), Yimit A, Lee AY, Riffle M, Costanzo M, Jaschob D, Hendry JA, Ou J, 
Moffat J, Boone C, Davis TN, Nislow C, Brown GW.

Author information:
(1)Department of Biochemistry, University of Toronto, Toronto, Ontario M5S 1A8, 
Canada.

Comment in
    Nat Cell Biol. 2012 Sep;14(9):900-1. doi: 10.1038/ncb2571.

Relocalization of proteins is a hallmark of the DNA damage response. We use 
high-throughput microscopic screening of the yeast GFP fusion collection to 
develop a systems-level view of protein reorganization following drug-induced 
DNA replication stress. Changes in protein localization and abundance reveal 
drug-specific patterns of functional enrichments. Classification of proteins by 
subcellular destination enables the identification of pathways that respond to 
replication stress. We analysed pairwise combinations of GFP fusions and gene 
deletion mutants to define and order two previously unknown DNA damage 
responses. In the first, Cmr1 forms subnuclear foci that are regulated by the 
histone deacetylase Hos2 and are distinct from the typical Rad52 repair foci. In 
a second example, we find that the checkpoint kinases Mec1/Tel1 and the 
translation regulator Asc1 regulate P-body formation. This method identifies 
response pathways that were not detected in genetic and protein interaction 
screens, and can be readily applied to any form of chemical or genetic stress to 
reveal cellular response pathways.

DOI: 10.1038/ncb2549
PMCID: PMC3434236
PMID: 22842922 [Indexed for MEDLINE]

## Full Text

IntroductionCells detect and respond to changes in their environment in a number of ways. Perhaps the best studied of these are changes in gene transcription1, protein abundance2, 3, and protein modification4, 5, all of which have been subjected to genome-scale analysis. Cells also regulate the intracellular localization of proteins to accommodate different environmental conditions, but this form of regulation has not been analyzed systematically.The DNA damage response consists of transcriptional, translational and post-translational facets, and several lines of evidence suggest that post-translational regulation is particularly important. At the single gene level, there is little if any correlation between transcriptional regulation in response to DNA damage and requirement for drug resistance6-8. Likewise, blocking mRNA translation does not prevent cells from completing S-phase when challenged with the replication inhibitor hydroxyurea (HU), nor does it affect cell viability after HU treatment9, 10. Critical roles of phosphorylation-, ubiquitylation-, and sumoylation-dependent signaling in the DNA damage response have been well characterized11-13. Together, these data suggest that post-translational regulation of existing proteins play a paramount role in the DNA damage response.Regulated protein re-localization is a hallmark of the cellular response to genotoxic drugs that cause DNA damage or DNA replication stress. In yeast, DNA damage response proteins including the single stranded DNA binding complex RPA, the double-strand DNA break processing complex MRX, the DNA damage sensor Ddc2, and proteins involved in homologous recombination relocalize from a diffuse nuclear distribution to form subnuclear foci in cells treated with genotoxic drugs14, 15. In the case of the recombination protein Rad52, these foci co-localize with induced double-stranded breaks suggesting that they represent centers for DNA repair15. Other localization changes occur including the re-localization of the small ribonucleotide reductase (RNR) subunits to the cytoplasm16. Some aspects of the regulated localization of DNA repair proteins to subnuclear foci are conserved, as RPA, the Ddc2 homologue ATRIP, and recombination proteins form foci in response to DNA damage in both yeast and human cells15. Mutations that disrupt phosphorylation of H2AX, or delete the ubiquitin interacting domains of Rad18 or Polη specifically disrupt the accumulation of repair proteins at nuclear foci and render cells sensitive to DNA damaging agents17-20 highlighting the importance of this post-translational regulation.Despite the frequent occurrence, conservation, and importance of protein localization changes in response to DNA damage, they have not been examined systematically in any organism. We used high-throughput microscopic analysis of the GFP-tagged yeast ORF collection to define the total proteome localization and abundance changes that occur in response to drug-induced DNA replication stress, and to identify DNA damage response modules. When combined with high-throughput genetic interaction methods the approach identifies and orders DNA damage response pathways. This method is readily applicable to any chemical or genetic stress in which the re-localization of proteins is suspected to play a role.

Introduction

Cells detect and respond to changes in their environment in a number of ways. Perhaps the best studied of these are changes in gene transcription1, protein abundance2, 3, and protein modification4, 5, all of which have been subjected to genome-scale analysis. Cells also regulate the intracellular localization of proteins to accommodate different environmental conditions, but this form of regulation has not been analyzed systematically.

The DNA damage response consists of transcriptional, translational and post-translational facets, and several lines of evidence suggest that post-translational regulation is particularly important. At the single gene level, there is little if any correlation between transcriptional regulation in response to DNA damage and requirement for drug resistance6-8. Likewise, blocking mRNA translation does not prevent cells from completing S-phase when challenged with the replication inhibitor hydroxyurea (HU), nor does it affect cell viability after HU treatment9, 10. Critical roles of phosphorylation-, ubiquitylation-, and sumoylation-dependent signaling in the DNA damage response have been well characterized11-13. Together, these data suggest that post-translational regulation of existing proteins play a paramount role in the DNA damage response.

Regulated protein re-localization is a hallmark of the cellular response to genotoxic drugs that cause DNA damage or DNA replication stress. In yeast, DNA damage response proteins including the single stranded DNA binding complex RPA, the double-strand DNA break processing complex MRX, the DNA damage sensor Ddc2, and proteins involved in homologous recombination relocalize from a diffuse nuclear distribution to form subnuclear foci in cells treated with genotoxic drugs14, 15. In the case of the recombination protein Rad52, these foci co-localize with induced double-stranded breaks suggesting that they represent centers for DNA repair15. Other localization changes occur including the re-localization of the small ribonucleotide reductase (RNR) subunits to the cytoplasm16. Some aspects of the regulated localization of DNA repair proteins to subnuclear foci are conserved, as RPA, the Ddc2 homologue ATRIP, and recombination proteins form foci in response to DNA damage in both yeast and human cells15. Mutations that disrupt phosphorylation of H2AX, or delete the ubiquitin interacting domains of Rad18 or Polη specifically disrupt the accumulation of repair proteins at nuclear foci and render cells sensitive to DNA damaging agents17-20 highlighting the importance of this post-translational regulation.

Despite the frequent occurrence, conservation, and importance of protein localization changes in response to DNA damage, they have not been examined systematically in any organism. We used high-throughput microscopic analysis of the GFP-tagged yeast ORF collection to define the total proteome localization and abundance changes that occur in response to drug-induced DNA replication stress, and to identify DNA damage response modules. When combined with high-throughput genetic interaction methods the approach identifies and orders DNA damage response pathways. This method is readily applicable to any chemical or genetic stress in which the re-localization of proteins is suspected to play a role.

ResultsGlobal changes in protein abundance and localization following DNA replication stressWe imaged each strain of the yeast GFP collection in the absence of perturbation and in the presence of HU or methylmethanesulfonate (MMS) to determine the spectrum of yeast proteins that undergo localization or abundance changes in response to replication stress (Fig. 1a). HU slows DNA replication by inhibiting RNR and limiting dNTP pools21, while MMS is an alkylating agent that results in a lesion that cannot be bypassed by the replicative DNA polymerases22. Following drug treatment, we observed phosphorylation of histone 2A S129 and Rad53, upregulation of Rnr3 and accumulation of cells in S-phase, all of which indicate that the DNA damage response was activated232425 (Supplementary Information, Figure S1). A total of 74,664 images were collected, and raw image files are available from the Yeast Resource Center Public Image Repository (http://images.yeastrc.org/tkach_brown/replication_stress). To identify proteins that changed in abundance after drug treatment we used a CellProfiler26 analysis pipeline to determine the fluorescence intensity in images of control and drug-treated cells (Supplementary Information, Table S1). We compared the control intensities to the single cell-based fluorescence measurements of the same GFP-fusion collection grown in minimal medium27 and found a significant positive correlation (r =0.890, p < 2.2 ×10−16, Supplementary Information, Fig. S2a) indicating the robustness of our abundance measurement method. Fluorescence intensities were converted to Z-scores relative to the control based on the median of the intensity measurements (Fig. 1b) and cut-offs of -2 and 2 (corresponding to 2 median absolute deviations from the control median value) were applied to identify strains that deviated significantly from the control.We scored localization changes by visual inspection of images, reasoning that some changes might be unanticipated and therefore difficult to score computationally. Ten major localization change classes, and several minor ones each representing two proteins or less, were identified (Fig. 1c, Supplementary Information, Table S2). To assess the accuracy of our subcellular localization designations, we compared our localization calls in unperturbed cells for 323 strains to those previously reported28. The primary localization for 89% of the proteins tested matched those from Huh et al while only 8% differed, indicating that our manual inspection was of high quality. In addition, we assigned localizations to 3% of proteins that were previously characterized as ‘ambiguous’ (Supplementary Information, Table S3). To assess the reproducibility of the localization analysis, we re-screened 252 of the 254 strains that showed a protein localization change in response to drug in the primary screen (Supplementary Information, Table S4). Of these, 74% were positive in the HU re-screen and 78% were positive in the MMS re-screen.A global view of the protein abundance and localization changes induced by replication stress is shown in Figure 1d. In total, 254 proteins underwent one or more localization changes and 356 proteins increased in abundance in response to drug treatment. Abundance changes were more prevalent in MMS than in HU (Fig. 1d and 2a), and only 35 proteins displayed both localization and abundance changes (Fig. 2b). In total, 575 proteins changed localization or abundance following HU or MMS treatment, representing 14% of the proteins screened.Analysis of protein dynamics reveals chemical-specific functional enrichmentsThe sets of proteins identified by localization and abundance changes are largely non-overlapping (Fig. 2b) and thus represent different kinds of cellular responses. Furthermore, the proteins identified in MMS differed from those in HU, particularly in abundance changes (Fig. 2a and 2c), and so might represent useful signatures to distinguish chemical agents. Enrichment analysis revealed that biological processes and protein complexes enriched in the abundance change classes (Fig. 3a and 3b) were distinct from those in the localization change classes (Fig. 3c, 3d and Supplementary Information, Fig. S3a and b). Abundance changes identified functions reminiscent of a global stress response, including iron homeostasis for HU and oxidative stress response for MMS. Interestingly, HU causes loss of iron from the ribonucleotide reductase active site 29 and HU is known to interfere with iron homeostasis in mammalian cells30, whereas MMS depletes mammalian cells of reduced glutathione31 and induces genes involved in cellular redox homeostasis in yeast8. By contrast, localization changes in MMS were enriched for functions with more obvious connections to the response to genotoxic stress, including cell cycle regulation, cell cycle checkpoint, and DNA repair (Fig. 3d). Despite the large overlap between proteins that re-localize in HU and those that re-localize in MMS (Fig. 2c), the enrichments remain specific for each agent (Fig. 3c and 3d). Finally, we find an unanticipated enrichment for mRNA decapping proteins in the HU localization category (Fig. 3c). These data indicate that protein abundance is regulated differently from protein localization, and so each likely performs distinct cellular roles in the response to HU and MMS. Furthermore, the enrichments we observe are specific to each chemical’s mechanism-of-action, and suggest that comprehensive chemical screening by this method could produce useful agent-specific signatures.Protein localization or abundance changes correlate poorly with replication stress resistanceGenes that are transcriptionally upregulated in response to DNA damaging agents do not correspond to those that are required for drug resistance6-8. Consistent with the lack of overlap between MMS sensitivity and mRNA abundance changes6, 7, the overlap between MMS sensitivity (Supplementary Information, Table S5) and protein abundance change was insignificant (Supplementary Information, Fig. S4a). Similarly, there was little overlap between HU sensitivity and protein abundance or localization changes (Supplementary Information, Fig. S4b and S4d). Lastly, a comparison of protein abundance and localization changes and genes identified in screens for chromosome instability32 and increased Rad52 focus formation33 did not reveal large overlaps among the datasets (Supplemental Information, Table S6). We anticipate that drug-induced protein localization changes and genetic requirements for drug resistance and genome instability phenotypes are not strongly predictive of each other due to considerable redundancy in replication stress resistance. This notion is supported by DNA damage-induced epistasis studies in which 379 double mutants exhibited greater MMS sensitivity than the corresponding single mutants34.Protein destination identifies DNA replication stress response modulesWe identified 10 major classes of protein localization changes (Fig. 1c), 9 of which reflect a protein destination and one that reflects movement away from the budneck or bud tip. There was significant overlap between the localization changes in HU and those in MMS (108 proteins re-localize in both drugs, Fig. 2c), and those that re-localized in both drugs moved to the same destination 98% of the time. To arrive at a dynamic view of protein localization changes, we compared the localization of all proteins that move in response to HU or MMS, before and after drug treatment (Fig. 4a). We found similar patterns of re-localization in HU and MMS, with the most populated changes being a reduction in diffuse nuclear localization, increases in localization to the cytoplasm, to cytoplasmic foci and to nuclear foci, and a decrease in localization to the budneck and budtip. Closer examination of proteins that had reduced diffuse nuclear distribution revealed that the reduction was due in part to the recruitment of 24 nuclear proteins into subnuclear foci (Fig. 4b), a well-known response to DNA damage and replication stress. However, there was also an export of 33 proteins from the nucleus to the cytoplasm and to cytoplasmic foci that contributed to the reduction in nuclear localization (Fig. 4b). Import of proteins to the nucleus typically involved further nuclear enrichment of proteins that were located in both the cytoplasm and nucleus, indicating a change in net nuclear import (Fig. 4b). Recruitment of proteins to foci, either in the cytoplasm or the nucleus most commonly reflected movement from a diffuse localization within the same compartment (Figure 4b).There was significant enrichment of biological processes within six of the ten localization change classes (Supplementary Information, Table S7), indicating that the classes might represent biological pathways important for the replication stress response. In particular, we focused on the nuclear foci and cytoplasmic foci localization classes. Localization to nuclear foci is a classic DNA damage response15 and so this class might contain uncharacterized response proteins. The nuclear foci class was highly enriched for the GO term ‘DNA repair’ (p=2×10−14; Fig. 4c and Supplementary Information, Fig. S5), with 16 of 28 proteins in the class annotated with this term. We mined existing databases to determine the extent of genetic interactions among the 28 genes encoding nuclear foci proteins. This analysis revealed a strong enrichment for interactions (p=1.9×10−14; Fig. 5a and 5e), indicating that proteins that share the same localization following replication stress are more likely to share functional biological connections. This further suggests that biological function can be assigned based on re-localization behavior. For example, of three poorly-characterized genes in the nuclear foci class, one of them, CMR1, has extensive genetic and physical interactions with other DNA repair genes and proteins in the class (Fig. 5a and 5b).Localization to cytoplasmic foci following replication stress was an unanticipated localization change. This class had a striking enrichment for mRNA catabolism processes, particularly mRNA decapping (p=2.6×10−16; Fig. 4c and Supplementary Information, Fig. S5). Mining physical interactions among the 41 proteins in the cytoplasmic foci class revealed a highly connected network of interactions, with a 6-fold higher interaction density than expected by chance (p=9.9×10−16; Fig. 5d and 5e). Inspection of the proteins involved revealed that most are components of cytoplasmic mRNA processing bodies (P-bodies) that form when excess non-translating mRNAs are present35, indicating a functional link between DNA replication stress and mRNA processing.CMR1 defines a novel class of replication stress fociWe first mined existing data to identify biological processes connected to CMR1. The genetic interaction similarity profile and physical interaction networks for CMR1 were enriched for DNA repair and homologous recombination processes, respectively (p=1.4×10−3 for DNA repair and p=2.3×10−4 for homologous recombination; Fig. 6a). To systematically explore these functional enrichments we undertook a synthetic genetic array (SGA) analysis36 of CMR1. The negative CMR1 genetic interactions defined in this screen revealed enrichment for recombinational repair (p=3.1×10−4; Fig. 6b and Supplementary Information, Table S8). We found that H2A serine 129 phosphorylation increased almost 2-fold in cmr1Δ cells treated with MMS (Fig. 6c), consistent with a role for Cmr1 in preventing DNA damage during exposure to replication stress.Proteins in the nuclear foci localization class share common functions and genetic and physical interactions, suggesting a functional ‘neighbourhood’ that could be mined for regulatory relationships. We first imaged mini-arrays of the 27 nuclear foci strains as GFP fusions deleted for CMR1 and identified a positive regulator of Cmr1 foci formation, the deacetylase Hos2, and a negative regulator, the molecular chaperone Apj1 (Fig. 6d, Supplementary Information, Fig. S6a). By performing the reciprocal experiment and imaging Cmr1-GFP strains deleted for the 24 non-essential members of its neighbourhood we found that Cmr1 suppressed the ability of Apj1 and the phosphatase Pph21 to form foci. (Fig. 6e, Supplementary Information, Fig. S6b). Finally, we interrogated the relationships among Hos2, Apj1 and Pph21 (Supplementary Information, Fig. 6b), determined that Pph21 foci formation requires Apj1, and ultimately defined the pathway that regulates DNA damage-induced focus formation among this group of proteins (Fig. 6f).Hos2 and Cmr1 foci co-localize (Fig. 6g), suggesting they are recruited to the same structures. Although we noted that the proteins in the Cmr1 pathway formed foci with a distinctive perinuclear location (Fig. 6d and 6e), these foci did not co-localize with the rDNA (Fig. 6i), nor did they co-localize with the canonical DNA repair focus member Rad52 (Fig. 6h). Thus, Cmr1, Hos2, Apj1, and Pph21 define a distinct subnuclear DNA damage response focus.Asc1 and Mec1/Tel1 regulate P-bodies induced by replication stressThe cytoplasmic foci formed following replication stress, particularly HU, were reminiscent of P-bodies, and all known P-body components in our screen formed these foci in HU (Supplementary Information, Fig. S7a, b, and c). The cytoplasmic foci formed by two P-body components, Lsm1 and Dhh1, either co-localized or were found adjacent to each other after HU treatment, consistent with the known distribution of P-body markers35 and indistinguishable from their co-localization after a combination of two typical P-body inducers, osmotic and glucose deprivation stresses35 (Fig. 7a). Deletion of two genes, PUB1 and TIF463237, which are required for the formation of cytoplasmic stress granules, had no effect on HU-induced Lsm1 foci formation (Fig. 7b and Supplementary Information, Fig. S8a). We conclude that the cytoplasmic foci that form in response to DNA replication stress are P-bodies.Combining microscopic screening with SGA analysis is a powerful means of identifying the complement of genes that regulate the subcellular localization of a given protein38, 39. We used SGA36 to cross Lsm1-GFP into the non-essential gene deletion collection and imaged control and HU-treated cultures. These 86,016 raw images are also available from the Yeast Resource Center Public Image Repository (http://images.yeastrc.org/tkach_brown/replication_stress). Positives were re-imaged after treatment with HU or water (Fig. 7c). We found that PAT1 and EDC3 are required for Lsm1 P-body formation in response to osmotic stress/starvation, consistent with their documented roles in this response40, 41. Both genes were also required for P-body formation in HU suggesting that these proteins might control P-body formation in response to diverse stimuli. Deletion of the gene encoding Lsm1 complex member Lsm6 reduced, but did not block Lsm1-GFP focus formation and is consistent with LSM complex members contributing to P-body assembly during glucose starvation40. Of particular interest, we found that the translation regulator Asc1 is required for P-body formation specifically in HU (Fig. 7c, Supplementary Information, Fig. S8a). This indicates that the formation of P-bodies in HU is not a general stress response, as it is regulated in a manner that is distinct from P-body formation following osmotic stress/starvation.Pat1 is a central regulator of P-body formation in the canonical glucose deprivation pathway42, and is itself a component of P-bodies35. Pat1 foci formed in water were unaffected by ASC1 deletion, but completely failed to form in HU (Fig. 7d). Thus, Asc1 is upstream of Pat1 in a HU-specific branch of the P-body pathway (Fig. 7g). The key components of the HU-induced P-body assembly pathway, Pat1, Lsm1, and Asc1, are all encoded by genes that confer HU sensitivity when deleted (Fig. 7e and Supplementary Information, Fig. S7b), connecting this response to HU resistance.The checkpoint kinases Mec1and Tel1 are critical regulators of the response to DNA replication stress11. To test a connection between P-body formation in HU and the checkpoint response, we deleted MEC1 and its homologue TEL1 and assessed the effect on P-body formation. Surprisingly, P-body formation, as measured by both Lsm1 and Pat1 foci, increased in the absence of Mec1 and Tel1, even in untreated cells, indicating that the checkpoint kinases are repressors of P-body formation (Fig. 7f, 7g, and Supplementary Information, Fig. S8b). We propose that activation of Mec1 in response to HU, either directly or indirectly, relieves this repression, allowing Asc1 to activate Pat1 and, subsequently, P-body formation.

Global changes in protein abundance and localization following DNA replication stressWe imaged each strain of the yeast GFP collection in the absence of perturbation and in the presence of HU or methylmethanesulfonate (MMS) to determine the spectrum of yeast proteins that undergo localization or abundance changes in response to replication stress (Fig. 1a). HU slows DNA replication by inhibiting RNR and limiting dNTP pools21, while MMS is an alkylating agent that results in a lesion that cannot be bypassed by the replicative DNA polymerases22. Following drug treatment, we observed phosphorylation of histone 2A S129 and Rad53, upregulation of Rnr3 and accumulation of cells in S-phase, all of which indicate that the DNA damage response was activated232425 (Supplementary Information, Figure S1). A total of 74,664 images were collected, and raw image files are available from the Yeast Resource Center Public Image Repository (http://images.yeastrc.org/tkach_brown/replication_stress). To identify proteins that changed in abundance after drug treatment we used a CellProfiler26 analysis pipeline to determine the fluorescence intensity in images of control and drug-treated cells (Supplementary Information, Table S1). We compared the control intensities to the single cell-based fluorescence measurements of the same GFP-fusion collection grown in minimal medium27 and found a significant positive correlation (r =0.890, p < 2.2 ×10−16, Supplementary Information, Fig. S2a) indicating the robustness of our abundance measurement method. Fluorescence intensities were converted to Z-scores relative to the control based on the median of the intensity measurements (Fig. 1b) and cut-offs of -2 and 2 (corresponding to 2 median absolute deviations from the control median value) were applied to identify strains that deviated significantly from the control.We scored localization changes by visual inspection of images, reasoning that some changes might be unanticipated and therefore difficult to score computationally. Ten major localization change classes, and several minor ones each representing two proteins or less, were identified (Fig. 1c, Supplementary Information, Table S2). To assess the accuracy of our subcellular localization designations, we compared our localization calls in unperturbed cells for 323 strains to those previously reported28. The primary localization for 89% of the proteins tested matched those from Huh et al while only 8% differed, indicating that our manual inspection was of high quality. In addition, we assigned localizations to 3% of proteins that were previously characterized as ‘ambiguous’ (Supplementary Information, Table S3). To assess the reproducibility of the localization analysis, we re-screened 252 of the 254 strains that showed a protein localization change in response to drug in the primary screen (Supplementary Information, Table S4). Of these, 74% were positive in the HU re-screen and 78% were positive in the MMS re-screen.A global view of the protein abundance and localization changes induced by replication stress is shown in Figure 1d. In total, 254 proteins underwent one or more localization changes and 356 proteins increased in abundance in response to drug treatment. Abundance changes were more prevalent in MMS than in HU (Fig. 1d and 2a), and only 35 proteins displayed both localization and abundance changes (Fig. 2b). In total, 575 proteins changed localization or abundance following HU or MMS treatment, representing 14% of the proteins screened.

Global changes in protein abundance and localization following DNA replication stress

We imaged each strain of the yeast GFP collection in the absence of perturbation and in the presence of HU or methylmethanesulfonate (MMS) to determine the spectrum of yeast proteins that undergo localization or abundance changes in response to replication stress (Fig. 1a). HU slows DNA replication by inhibiting RNR and limiting dNTP pools21, while MMS is an alkylating agent that results in a lesion that cannot be bypassed by the replicative DNA polymerases22. Following drug treatment, we observed phosphorylation of histone 2A S129 and Rad53, upregulation of Rnr3 and accumulation of cells in S-phase, all of which indicate that the DNA damage response was activated232425 (Supplementary Information, Figure S1). A total of 74,664 images were collected, and raw image files are available from the Yeast Resource Center Public Image Repository (http://images.yeastrc.org/tkach_brown/replication_stress). To identify proteins that changed in abundance after drug treatment we used a CellProfiler26 analysis pipeline to determine the fluorescence intensity in images of control and drug-treated cells (Supplementary Information, Table S1). We compared the control intensities to the single cell-based fluorescence measurements of the same GFP-fusion collection grown in minimal medium27 and found a significant positive correlation (r =0.890, p < 2.2 ×10−16, Supplementary Information, Fig. S2a) indicating the robustness of our abundance measurement method. Fluorescence intensities were converted to Z-scores relative to the control based on the median of the intensity measurements (Fig. 1b) and cut-offs of -2 and 2 (corresponding to 2 median absolute deviations from the control median value) were applied to identify strains that deviated significantly from the control.

We scored localization changes by visual inspection of images, reasoning that some changes might be unanticipated and therefore difficult to score computationally. Ten major localization change classes, and several minor ones each representing two proteins or less, were identified (Fig. 1c, Supplementary Information, Table S2). To assess the accuracy of our subcellular localization designations, we compared our localization calls in unperturbed cells for 323 strains to those previously reported28. The primary localization for 89% of the proteins tested matched those from Huh et al while only 8% differed, indicating that our manual inspection was of high quality. In addition, we assigned localizations to 3% of proteins that were previously characterized as ‘ambiguous’ (Supplementary Information, Table S3). To assess the reproducibility of the localization analysis, we re-screened 252 of the 254 strains that showed a protein localization change in response to drug in the primary screen (Supplementary Information, Table S4). Of these, 74% were positive in the HU re-screen and 78% were positive in the MMS re-screen.

A global view of the protein abundance and localization changes induced by replication stress is shown in Figure 1d. In total, 254 proteins underwent one or more localization changes and 356 proteins increased in abundance in response to drug treatment. Abundance changes were more prevalent in MMS than in HU (Fig. 1d and 2a), and only 35 proteins displayed both localization and abundance changes (Fig. 2b). In total, 575 proteins changed localization or abundance following HU or MMS treatment, representing 14% of the proteins screened.

Analysis of protein dynamics reveals chemical-specific functional enrichmentsThe sets of proteins identified by localization and abundance changes are largely non-overlapping (Fig. 2b) and thus represent different kinds of cellular responses. Furthermore, the proteins identified in MMS differed from those in HU, particularly in abundance changes (Fig. 2a and 2c), and so might represent useful signatures to distinguish chemical agents. Enrichment analysis revealed that biological processes and protein complexes enriched in the abundance change classes (Fig. 3a and 3b) were distinct from those in the localization change classes (Fig. 3c, 3d and Supplementary Information, Fig. S3a and b). Abundance changes identified functions reminiscent of a global stress response, including iron homeostasis for HU and oxidative stress response for MMS. Interestingly, HU causes loss of iron from the ribonucleotide reductase active site 29 and HU is known to interfere with iron homeostasis in mammalian cells30, whereas MMS depletes mammalian cells of reduced glutathione31 and induces genes involved in cellular redox homeostasis in yeast8. By contrast, localization changes in MMS were enriched for functions with more obvious connections to the response to genotoxic stress, including cell cycle regulation, cell cycle checkpoint, and DNA repair (Fig. 3d). Despite the large overlap between proteins that re-localize in HU and those that re-localize in MMS (Fig. 2c), the enrichments remain specific for each agent (Fig. 3c and 3d). Finally, we find an unanticipated enrichment for mRNA decapping proteins in the HU localization category (Fig. 3c). These data indicate that protein abundance is regulated differently from protein localization, and so each likely performs distinct cellular roles in the response to HU and MMS. Furthermore, the enrichments we observe are specific to each chemical’s mechanism-of-action, and suggest that comprehensive chemical screening by this method could produce useful agent-specific signatures.

Analysis of protein dynamics reveals chemical-specific functional enrichments

The sets of proteins identified by localization and abundance changes are largely non-overlapping (Fig. 2b) and thus represent different kinds of cellular responses. Furthermore, the proteins identified in MMS differed from those in HU, particularly in abundance changes (Fig. 2a and 2c), and so might represent useful signatures to distinguish chemical agents. Enrichment analysis revealed that biological processes and protein complexes enriched in the abundance change classes (Fig. 3a and 3b) were distinct from those in the localization change classes (Fig. 3c, 3d and Supplementary Information, Fig. S3a and b). Abundance changes identified functions reminiscent of a global stress response, including iron homeostasis for HU and oxidative stress response for MMS. Interestingly, HU causes loss of iron from the ribonucleotide reductase active site 29 and HU is known to interfere with iron homeostasis in mammalian cells30, whereas MMS depletes mammalian cells of reduced glutathione31 and induces genes involved in cellular redox homeostasis in yeast8. By contrast, localization changes in MMS were enriched for functions with more obvious connections to the response to genotoxic stress, including cell cycle regulation, cell cycle checkpoint, and DNA repair (Fig. 3d). Despite the large overlap between proteins that re-localize in HU and those that re-localize in MMS (Fig. 2c), the enrichments remain specific for each agent (Fig. 3c and 3d). Finally, we find an unanticipated enrichment for mRNA decapping proteins in the HU localization category (Fig. 3c). These data indicate that protein abundance is regulated differently from protein localization, and so each likely performs distinct cellular roles in the response to HU and MMS. Furthermore, the enrichments we observe are specific to each chemical’s mechanism-of-action, and suggest that comprehensive chemical screening by this method could produce useful agent-specific signatures.

Protein localization or abundance changes correlate poorly with replication stress resistanceGenes that are transcriptionally upregulated in response to DNA damaging agents do not correspond to those that are required for drug resistance6-8. Consistent with the lack of overlap between MMS sensitivity and mRNA abundance changes6, 7, the overlap between MMS sensitivity (Supplementary Information, Table S5) and protein abundance change was insignificant (Supplementary Information, Fig. S4a). Similarly, there was little overlap between HU sensitivity and protein abundance or localization changes (Supplementary Information, Fig. S4b and S4d). Lastly, a comparison of protein abundance and localization changes and genes identified in screens for chromosome instability32 and increased Rad52 focus formation33 did not reveal large overlaps among the datasets (Supplemental Information, Table S6). We anticipate that drug-induced protein localization changes and genetic requirements for drug resistance and genome instability phenotypes are not strongly predictive of each other due to considerable redundancy in replication stress resistance. This notion is supported by DNA damage-induced epistasis studies in which 379 double mutants exhibited greater MMS sensitivity than the corresponding single mutants34.

Protein localization or abundance changes correlate poorly with replication stress resistance

Genes that are transcriptionally upregulated in response to DNA damaging agents do not correspond to those that are required for drug resistance6-8. Consistent with the lack of overlap between MMS sensitivity and mRNA abundance changes6, 7, the overlap between MMS sensitivity (Supplementary Information, Table S5) and protein abundance change was insignificant (Supplementary Information, Fig. S4a). Similarly, there was little overlap between HU sensitivity and protein abundance or localization changes (Supplementary Information, Fig. S4b and S4d). Lastly, a comparison of protein abundance and localization changes and genes identified in screens for chromosome instability32 and increased Rad52 focus formation33 did not reveal large overlaps among the datasets (Supplemental Information, Table S6). We anticipate that drug-induced protein localization changes and genetic requirements for drug resistance and genome instability phenotypes are not strongly predictive of each other due to considerable redundancy in replication stress resistance. This notion is supported by DNA damage-induced epistasis studies in which 379 double mutants exhibited greater MMS sensitivity than the corresponding single mutants34.

Protein destination identifies DNA replication stress response modulesWe identified 10 major classes of protein localization changes (Fig. 1c), 9 of which reflect a protein destination and one that reflects movement away from the budneck or bud tip. There was significant overlap between the localization changes in HU and those in MMS (108 proteins re-localize in both drugs, Fig. 2c), and those that re-localized in both drugs moved to the same destination 98% of the time. To arrive at a dynamic view of protein localization changes, we compared the localization of all proteins that move in response to HU or MMS, before and after drug treatment (Fig. 4a). We found similar patterns of re-localization in HU and MMS, with the most populated changes being a reduction in diffuse nuclear localization, increases in localization to the cytoplasm, to cytoplasmic foci and to nuclear foci, and a decrease in localization to the budneck and budtip. Closer examination of proteins that had reduced diffuse nuclear distribution revealed that the reduction was due in part to the recruitment of 24 nuclear proteins into subnuclear foci (Fig. 4b), a well-known response to DNA damage and replication stress. However, there was also an export of 33 proteins from the nucleus to the cytoplasm and to cytoplasmic foci that contributed to the reduction in nuclear localization (Fig. 4b). Import of proteins to the nucleus typically involved further nuclear enrichment of proteins that were located in both the cytoplasm and nucleus, indicating a change in net nuclear import (Fig. 4b). Recruitment of proteins to foci, either in the cytoplasm or the nucleus most commonly reflected movement from a diffuse localization within the same compartment (Figure 4b).There was significant enrichment of biological processes within six of the ten localization change classes (Supplementary Information, Table S7), indicating that the classes might represent biological pathways important for the replication stress response. In particular, we focused on the nuclear foci and cytoplasmic foci localization classes. Localization to nuclear foci is a classic DNA damage response15 and so this class might contain uncharacterized response proteins. The nuclear foci class was highly enriched for the GO term ‘DNA repair’ (p=2×10−14; Fig. 4c and Supplementary Information, Fig. S5), with 16 of 28 proteins in the class annotated with this term. We mined existing databases to determine the extent of genetic interactions among the 28 genes encoding nuclear foci proteins. This analysis revealed a strong enrichment for interactions (p=1.9×10−14; Fig. 5a and 5e), indicating that proteins that share the same localization following replication stress are more likely to share functional biological connections. This further suggests that biological function can be assigned based on re-localization behavior. For example, of three poorly-characterized genes in the nuclear foci class, one of them, CMR1, has extensive genetic and physical interactions with other DNA repair genes and proteins in the class (Fig. 5a and 5b).Localization to cytoplasmic foci following replication stress was an unanticipated localization change. This class had a striking enrichment for mRNA catabolism processes, particularly mRNA decapping (p=2.6×10−16; Fig. 4c and Supplementary Information, Fig. S5). Mining physical interactions among the 41 proteins in the cytoplasmic foci class revealed a highly connected network of interactions, with a 6-fold higher interaction density than expected by chance (p=9.9×10−16; Fig. 5d and 5e). Inspection of the proteins involved revealed that most are components of cytoplasmic mRNA processing bodies (P-bodies) that form when excess non-translating mRNAs are present35, indicating a functional link between DNA replication stress and mRNA processing.

Protein destination identifies DNA replication stress response modules

We identified 10 major classes of protein localization changes (Fig. 1c), 9 of which reflect a protein destination and one that reflects movement away from the budneck or bud tip. There was significant overlap between the localization changes in HU and those in MMS (108 proteins re-localize in both drugs, Fig. 2c), and those that re-localized in both drugs moved to the same destination 98% of the time. To arrive at a dynamic view of protein localization changes, we compared the localization of all proteins that move in response to HU or MMS, before and after drug treatment (Fig. 4a). We found similar patterns of re-localization in HU and MMS, with the most populated changes being a reduction in diffuse nuclear localization, increases in localization to the cytoplasm, to cytoplasmic foci and to nuclear foci, and a decrease in localization to the budneck and budtip. Closer examination of proteins that had reduced diffuse nuclear distribution revealed that the reduction was due in part to the recruitment of 24 nuclear proteins into subnuclear foci (Fig. 4b), a well-known response to DNA damage and replication stress. However, there was also an export of 33 proteins from the nucleus to the cytoplasm and to cytoplasmic foci that contributed to the reduction in nuclear localization (Fig. 4b). Import of proteins to the nucleus typically involved further nuclear enrichment of proteins that were located in both the cytoplasm and nucleus, indicating a change in net nuclear import (Fig. 4b). Recruitment of proteins to foci, either in the cytoplasm or the nucleus most commonly reflected movement from a diffuse localization within the same compartment (Figure 4b).

There was significant enrichment of biological processes within six of the ten localization change classes (Supplementary Information, Table S7), indicating that the classes might represent biological pathways important for the replication stress response. In particular, we focused on the nuclear foci and cytoplasmic foci localization classes. Localization to nuclear foci is a classic DNA damage response15 and so this class might contain uncharacterized response proteins. The nuclear foci class was highly enriched for the GO term ‘DNA repair’ (p=2×10−14; Fig. 4c and Supplementary Information, Fig. S5), with 16 of 28 proteins in the class annotated with this term. We mined existing databases to determine the extent of genetic interactions among the 28 genes encoding nuclear foci proteins. This analysis revealed a strong enrichment for interactions (p=1.9×10−14; Fig. 5a and 5e), indicating that proteins that share the same localization following replication stress are more likely to share functional biological connections. This further suggests that biological function can be assigned based on re-localization behavior. For example, of three poorly-characterized genes in the nuclear foci class, one of them, CMR1, has extensive genetic and physical interactions with other DNA repair genes and proteins in the class (Fig. 5a and 5b).

Localization to cytoplasmic foci following replication stress was an unanticipated localization change. This class had a striking enrichment for mRNA catabolism processes, particularly mRNA decapping (p=2.6×10−16; Fig. 4c and Supplementary Information, Fig. S5). Mining physical interactions among the 41 proteins in the cytoplasmic foci class revealed a highly connected network of interactions, with a 6-fold higher interaction density than expected by chance (p=9.9×10−16; Fig. 5d and 5e). Inspection of the proteins involved revealed that most are components of cytoplasmic mRNA processing bodies (P-bodies) that form when excess non-translating mRNAs are present35, indicating a functional link between DNA replication stress and mRNA processing.

CMR1 defines a novel class of replication stress fociWe first mined existing data to identify biological processes connected to CMR1. The genetic interaction similarity profile and physical interaction networks for CMR1 were enriched for DNA repair and homologous recombination processes, respectively (p=1.4×10−3 for DNA repair and p=2.3×10−4 for homologous recombination; Fig. 6a). To systematically explore these functional enrichments we undertook a synthetic genetic array (SGA) analysis36 of CMR1. The negative CMR1 genetic interactions defined in this screen revealed enrichment for recombinational repair (p=3.1×10−4; Fig. 6b and Supplementary Information, Table S8). We found that H2A serine 129 phosphorylation increased almost 2-fold in cmr1Δ cells treated with MMS (Fig. 6c), consistent with a role for Cmr1 in preventing DNA damage during exposure to replication stress.Proteins in the nuclear foci localization class share common functions and genetic and physical interactions, suggesting a functional ‘neighbourhood’ that could be mined for regulatory relationships. We first imaged mini-arrays of the 27 nuclear foci strains as GFP fusions deleted for CMR1 and identified a positive regulator of Cmr1 foci formation, the deacetylase Hos2, and a negative regulator, the molecular chaperone Apj1 (Fig. 6d, Supplementary Information, Fig. S6a). By performing the reciprocal experiment and imaging Cmr1-GFP strains deleted for the 24 non-essential members of its neighbourhood we found that Cmr1 suppressed the ability of Apj1 and the phosphatase Pph21 to form foci. (Fig. 6e, Supplementary Information, Fig. S6b). Finally, we interrogated the relationships among Hos2, Apj1 and Pph21 (Supplementary Information, Fig. 6b), determined that Pph21 foci formation requires Apj1, and ultimately defined the pathway that regulates DNA damage-induced focus formation among this group of proteins (Fig. 6f).Hos2 and Cmr1 foci co-localize (Fig. 6g), suggesting they are recruited to the same structures. Although we noted that the proteins in the Cmr1 pathway formed foci with a distinctive perinuclear location (Fig. 6d and 6e), these foci did not co-localize with the rDNA (Fig. 6i), nor did they co-localize with the canonical DNA repair focus member Rad52 (Fig. 6h). Thus, Cmr1, Hos2, Apj1, and Pph21 define a distinct subnuclear DNA damage response focus.

CMR1 defines a novel class of replication stress foci

We first mined existing data to identify biological processes connected to CMR1. The genetic interaction similarity profile and physical interaction networks for CMR1 were enriched for DNA repair and homologous recombination processes, respectively (p=1.4×10−3 for DNA repair and p=2.3×10−4 for homologous recombination; Fig. 6a). To systematically explore these functional enrichments we undertook a synthetic genetic array (SGA) analysis36 of CMR1. The negative CMR1 genetic interactions defined in this screen revealed enrichment for recombinational repair (p=3.1×10−4; Fig. 6b and Supplementary Information, Table S8). We found that H2A serine 129 phosphorylation increased almost 2-fold in cmr1Δ cells treated with MMS (Fig. 6c), consistent with a role for Cmr1 in preventing DNA damage during exposure to replication stress.

Proteins in the nuclear foci localization class share common functions and genetic and physical interactions, suggesting a functional ‘neighbourhood’ that could be mined for regulatory relationships. We first imaged mini-arrays of the 27 nuclear foci strains as GFP fusions deleted for CMR1 and identified a positive regulator of Cmr1 foci formation, the deacetylase Hos2, and a negative regulator, the molecular chaperone Apj1 (Fig. 6d, Supplementary Information, Fig. S6a). By performing the reciprocal experiment and imaging Cmr1-GFP strains deleted for the 24 non-essential members of its neighbourhood we found that Cmr1 suppressed the ability of Apj1 and the phosphatase Pph21 to form foci. (Fig. 6e, Supplementary Information, Fig. S6b). Finally, we interrogated the relationships among Hos2, Apj1 and Pph21 (Supplementary Information, Fig. 6b), determined that Pph21 foci formation requires Apj1, and ultimately defined the pathway that regulates DNA damage-induced focus formation among this group of proteins (Fig. 6f).

Hos2 and Cmr1 foci co-localize (Fig. 6g), suggesting they are recruited to the same structures. Although we noted that the proteins in the Cmr1 pathway formed foci with a distinctive perinuclear location (Fig. 6d and 6e), these foci did not co-localize with the rDNA (Fig. 6i), nor did they co-localize with the canonical DNA repair focus member Rad52 (Fig. 6h). Thus, Cmr1, Hos2, Apj1, and Pph21 define a distinct subnuclear DNA damage response focus.

Asc1 and Mec1/Tel1 regulate P-bodies induced by replication stressThe cytoplasmic foci formed following replication stress, particularly HU, were reminiscent of P-bodies, and all known P-body components in our screen formed these foci in HU (Supplementary Information, Fig. S7a, b, and c). The cytoplasmic foci formed by two P-body components, Lsm1 and Dhh1, either co-localized or were found adjacent to each other after HU treatment, consistent with the known distribution of P-body markers35 and indistinguishable from their co-localization after a combination of two typical P-body inducers, osmotic and glucose deprivation stresses35 (Fig. 7a). Deletion of two genes, PUB1 and TIF463237, which are required for the formation of cytoplasmic stress granules, had no effect on HU-induced Lsm1 foci formation (Fig. 7b and Supplementary Information, Fig. S8a). We conclude that the cytoplasmic foci that form in response to DNA replication stress are P-bodies.Combining microscopic screening with SGA analysis is a powerful means of identifying the complement of genes that regulate the subcellular localization of a given protein38, 39. We used SGA36 to cross Lsm1-GFP into the non-essential gene deletion collection and imaged control and HU-treated cultures. These 86,016 raw images are also available from the Yeast Resource Center Public Image Repository (http://images.yeastrc.org/tkach_brown/replication_stress). Positives were re-imaged after treatment with HU or water (Fig. 7c). We found that PAT1 and EDC3 are required for Lsm1 P-body formation in response to osmotic stress/starvation, consistent with their documented roles in this response40, 41. Both genes were also required for P-body formation in HU suggesting that these proteins might control P-body formation in response to diverse stimuli. Deletion of the gene encoding Lsm1 complex member Lsm6 reduced, but did not block Lsm1-GFP focus formation and is consistent with LSM complex members contributing to P-body assembly during glucose starvation40. Of particular interest, we found that the translation regulator Asc1 is required for P-body formation specifically in HU (Fig. 7c, Supplementary Information, Fig. S8a). This indicates that the formation of P-bodies in HU is not a general stress response, as it is regulated in a manner that is distinct from P-body formation following osmotic stress/starvation.Pat1 is a central regulator of P-body formation in the canonical glucose deprivation pathway42, and is itself a component of P-bodies35. Pat1 foci formed in water were unaffected by ASC1 deletion, but completely failed to form in HU (Fig. 7d). Thus, Asc1 is upstream of Pat1 in a HU-specific branch of the P-body pathway (Fig. 7g). The key components of the HU-induced P-body assembly pathway, Pat1, Lsm1, and Asc1, are all encoded by genes that confer HU sensitivity when deleted (Fig. 7e and Supplementary Information, Fig. S7b), connecting this response to HU resistance.The checkpoint kinases Mec1and Tel1 are critical regulators of the response to DNA replication stress11. To test a connection between P-body formation in HU and the checkpoint response, we deleted MEC1 and its homologue TEL1 and assessed the effect on P-body formation. Surprisingly, P-body formation, as measured by both Lsm1 and Pat1 foci, increased in the absence of Mec1 and Tel1, even in untreated cells, indicating that the checkpoint kinases are repressors of P-body formation (Fig. 7f, 7g, and Supplementary Information, Fig. S8b). We propose that activation of Mec1 in response to HU, either directly or indirectly, relieves this repression, allowing Asc1 to activate Pat1 and, subsequently, P-body formation.

Asc1 and Mec1/Tel1 regulate P-bodies induced by replication stress

The cytoplasmic foci formed following replication stress, particularly HU, were reminiscent of P-bodies, and all known P-body components in our screen formed these foci in HU (Supplementary Information, Fig. S7a, b, and c). The cytoplasmic foci formed by two P-body components, Lsm1 and Dhh1, either co-localized or were found adjacent to each other after HU treatment, consistent with the known distribution of P-body markers35 and indistinguishable from their co-localization after a combination of two typical P-body inducers, osmotic and glucose deprivation stresses35 (Fig. 7a). Deletion of two genes, PUB1 and TIF463237, which are required for the formation of cytoplasmic stress granules, had no effect on HU-induced Lsm1 foci formation (Fig. 7b and Supplementary Information, Fig. S8a). We conclude that the cytoplasmic foci that form in response to DNA replication stress are P-bodies.

Combining microscopic screening with SGA analysis is a powerful means of identifying the complement of genes that regulate the subcellular localization of a given protein38, 39. We used SGA36 to cross Lsm1-GFP into the non-essential gene deletion collection and imaged control and HU-treated cultures. These 86,016 raw images are also available from the Yeast Resource Center Public Image Repository (http://images.yeastrc.org/tkach_brown/replication_stress). Positives were re-imaged after treatment with HU or water (Fig. 7c). We found that PAT1 and EDC3 are required for Lsm1 P-body formation in response to osmotic stress/starvation, consistent with their documented roles in this response40, 41. Both genes were also required for P-body formation in HU suggesting that these proteins might control P-body formation in response to diverse stimuli. Deletion of the gene encoding Lsm1 complex member Lsm6 reduced, but did not block Lsm1-GFP focus formation and is consistent with LSM complex members contributing to P-body assembly during glucose starvation40. Of particular interest, we found that the translation regulator Asc1 is required for P-body formation specifically in HU (Fig. 7c, Supplementary Information, Fig. S8a). This indicates that the formation of P-bodies in HU is not a general stress response, as it is regulated in a manner that is distinct from P-body formation following osmotic stress/starvation.

Pat1 is a central regulator of P-body formation in the canonical glucose deprivation pathway42, and is itself a component of P-bodies35. Pat1 foci formed in water were unaffected by ASC1 deletion, but completely failed to form in HU (Fig. 7d). Thus, Asc1 is upstream of Pat1 in a HU-specific branch of the P-body pathway (Fig. 7g). The key components of the HU-induced P-body assembly pathway, Pat1, Lsm1, and Asc1, are all encoded by genes that confer HU sensitivity when deleted (Fig. 7e and Supplementary Information, Fig. S7b), connecting this response to HU resistance.

The checkpoint kinases Mec1and Tel1 are critical regulators of the response to DNA replication stress11. To test a connection between P-body formation in HU and the checkpoint response, we deleted MEC1 and its homologue TEL1 and assessed the effect on P-body formation. Surprisingly, P-body formation, as measured by both Lsm1 and Pat1 foci, increased in the absence of Mec1 and Tel1, even in untreated cells, indicating that the checkpoint kinases are repressors of P-body formation (Fig. 7f, 7g, and Supplementary Information, Fig. S8b). We propose that activation of Mec1 in response to HU, either directly or indirectly, relieves this repression, allowing Asc1 to activate Pat1 and, subsequently, P-body formation.

DiscussionHU and MMS are commonly used to induce replication stress and DNA damage in yeast. Despite the clear effect of HU on DNA replication (Supplementary Information, Fig. S1b), proteins involved in the DNA damage response were not significantly enriched in either the abundance or localization change HU categories likely due to the lack of DNA damage in HU-treated cells. We did not detect a significant increase in Ddc2 foci, which is a common proxy for DNA damage43, 44, during HU treatment, consistent with previous reports45 and consistent with the absence of HU-induced DNA damage in cells that have an intact checkpoint46. Proteins involved in iron transport were enriched in the HU abundance hits and could counteract the loss of iron at the catalytic RNR subunit29 as has been suggested by transcriptome analysis47. Notably, this suggests that disrupting iron transport might augment the chemotherapeutic efficacy of HU.MMS treatment causes multiple DNA alkylation adducts, including an N3-deoxyadenosine lesion that inhibits DNA polymerase elongation22. The MMS localization change category showed robust enrichment of DNA repair and checkpoint genes, consistent with its major mode of action and distinguishing the MMS response from the HU response. Consistent with MMS increasing cellular reactive oxygen species48, we also see a strong enrichment for oxidative stress response processes in the MMS abundance change category. Together, our results indicate that there is considerable specificity in the functional enrichments, both for different agents, and for localization versus abundance changes. This points to the usefulness of microscopic screening to characterize the biological properties of drugs.The functional enrichments evident in our data are different from those observed when the yeast genome was screened for HU- or MMS-sensitive mutants (for example, see7, 49). Compiling all HU sensitive genes from SGD yields a potent enrichment for DNA damage response, DNA repair, and stress response, but does not reveal the enrichment for iron homeostasis. Similarly, the striking enrichment of mRNA decapping processes in the HU localization response is not evident in the group of HU sensitive strains. Thus, analysis of protein dynamics affords a view of cellular response that is not captured by other methods.Post-transcriptional regulation in the response to MMSComparison of the protein abundance changes that occur during MMS treatment with their corresponding mRNA changes8 yielded a positive correlation for the top 300 abundance changes (r=0.457; Supplementary Information, Fig. S2b), indicating that mRNA changes account for 21% of the variance in protein abundance changes. Thus, many increases in transcript levels did not result in corresponding protein changes, and including the entire set of proteins analyzed resulted in a poorer correlation (r=0.281; Supplementary Information, Figure S2d). These observations suggest that in the case of the MMS response post-transcriptional regulation is a critical determinant of the ultimate changes in protein abundance. This contrasts with the response to osmotic shock50 and rapamycin treatment51 in yeast where 80% and 36% of the protein abundance changes could be explained by cognate changes in mRNA abundance. It appears that the relationship between mRNA abundance and protein abundance varies greatly depending on the cell stress, indicating stress-specific roles for post-transcriptional regulation. It is interesting in this respect that one of the biological modules we identified in HU regulates mRNA translation and stability.Identifying regulators in localization change neighbourhoodsThe high degree of biological process, genetic and physical interaction enrichment in most of the localization classes suggested that each class could represent a functionally connected ‘neighbourhood’ of proteins. Consistent with this possibility, the ‘to nuclear foci’ class was enriched for DNA repair proteins and we connected a protein in that class, Cmr1, with DNA repair in several ways (Fig. 6a, b, and c). We further interrogated the nuclear foci ‘neighbourhood’ to identify regulators of Cmr1 focus formation. It is interesting that our analysis of 27 genes identified three regulators, whereas in the case of Lsm1, screening the entire gene deletion collection of ~4500 genes identified only seven regulators (Fig. 7 and data not shown). While this single case has not yet been extrapolated, it is tempting to speculate that localization change categories will be enriched for regulatory relationships.We also found that analyzing the co-localization of proteins within the nuclear foci class revealed a new kind of sub-nuclear focus consisting of Cmr1, Hos2, Apj1 and Pph21that is not associated with the canonical DNA repair foci represented by proteins like Rad52 and Ddc2. The role of these proteins in the DNA damage response is unclear since the deletion of any one gene does not result in a strong damage sensitivity phenotype. However, one member of this group, Cmr1, contributes to genome stability32 and was recently demonstrated to interact with UV-damaged DNA in vitro and in vivo52. In one scenario, the chromatin remodeling activity of Hos2 might be required to permit Cmr1 to access ‘bulky’ DNA lesions. Alternatively, since Hos2 is required for the activation of DNA damage-inducible genes53, these foci might not represent sites of DNA damage, but rather sites of damage-induced transcription.A novel pathway regulating cytoplasmic P-bodiesThe redistribution of proteins from a diffuse cytoplasmic distribution to cytoplasmic foci formed the most striking relocalization class in our screen and represents the formation of P-bodies (Fig. 7a). We demonstrate that replication stress is a potent inducer of P-body formation, suggesting that replication stress causes an increase in non-translating mRNAs, and indicating an important role for post-transcriptional regulation in the genotoxic stress response. We found that Asc1 is required for P-body formation in the HU response, but not in response to glucose deprivation/osmotic stress, and acts upstream of the key regulator of P-body formation, Pat1. Thus, HU induction of P-bodies is regulated differently than induction by more classical conditions, and so forms a distinct branch of the P-body pathway. Both Pat1 and Lsm1 are required for resistance to hydroxyurea (Supplemental Information, Fig. S7d) and the topoisomerase I poison camptothecin49. It was recently shown that Lsm1 contributes to the turnover of histone mRNA and that loss of this function contributes to the HU-sensitivity of lsm1Δ strains55. Thus, the P-bodies we observe may in part reflect the turnover of histone mRNA in response to replication stress.Asc1, and its mammalian homologue RACK1 are signaling adaptor proteins that regulate diverse cellular processes56. In addition, both Asc1 and RACK1 are stoichiometric components of the ribosome and are thought to recruit regulators to the ribosome to modulate translation56, 57. It will be interesting to determine if RACK1 modulates P-body assembly in response to replication stress in mammalian cells, and whether such a role is relevant to the upregulation of RACK1 that is common in neoplasias56.We provide a comprehensive resource detailing the protein abundance and localization changes that occur during replication stress in yeast. Our data demonstrate the potential of high-throughput microscopic screening to identify novel response pathways and their regulators. The methodology can be readily applied to virtually any genetic or chemical perturbation.

HU and MMS are commonly used to induce replication stress and DNA damage in yeast. Despite the clear effect of HU on DNA replication (Supplementary Information, Fig. S1b), proteins involved in the DNA damage response were not significantly enriched in either the abundance or localization change HU categories likely due to the lack of DNA damage in HU-treated cells. We did not detect a significant increase in Ddc2 foci, which is a common proxy for DNA damage43, 44, during HU treatment, consistent with previous reports45 and consistent with the absence of HU-induced DNA damage in cells that have an intact checkpoint46. Proteins involved in iron transport were enriched in the HU abundance hits and could counteract the loss of iron at the catalytic RNR subunit29 as has been suggested by transcriptome analysis47. Notably, this suggests that disrupting iron transport might augment the chemotherapeutic efficacy of HU.

MMS treatment causes multiple DNA alkylation adducts, including an N3-deoxyadenosine lesion that inhibits DNA polymerase elongation22. The MMS localization change category showed robust enrichment of DNA repair and checkpoint genes, consistent with its major mode of action and distinguishing the MMS response from the HU response. Consistent with MMS increasing cellular reactive oxygen species48, we also see a strong enrichment for oxidative stress response processes in the MMS abundance change category. Together, our results indicate that there is considerable specificity in the functional enrichments, both for different agents, and for localization versus abundance changes. This points to the usefulness of microscopic screening to characterize the biological properties of drugs.

The functional enrichments evident in our data are different from those observed when the yeast genome was screened for HU- or MMS-sensitive mutants (for example, see7, 49). Compiling all HU sensitive genes from SGD yields a potent enrichment for DNA damage response, DNA repair, and stress response, but does not reveal the enrichment for iron homeostasis. Similarly, the striking enrichment of mRNA decapping processes in the HU localization response is not evident in the group of HU sensitive strains. Thus, analysis of protein dynamics affords a view of cellular response that is not captured by other methods.

Post-transcriptional regulation in the response to MMSComparison of the protein abundance changes that occur during MMS treatment with their corresponding mRNA changes8 yielded a positive correlation for the top 300 abundance changes (r=0.457; Supplementary Information, Fig. S2b), indicating that mRNA changes account for 21% of the variance in protein abundance changes. Thus, many increases in transcript levels did not result in corresponding protein changes, and including the entire set of proteins analyzed resulted in a poorer correlation (r=0.281; Supplementary Information, Figure S2d). These observations suggest that in the case of the MMS response post-transcriptional regulation is a critical determinant of the ultimate changes in protein abundance. This contrasts with the response to osmotic shock50 and rapamycin treatment51 in yeast where 80% and 36% of the protein abundance changes could be explained by cognate changes in mRNA abundance. It appears that the relationship between mRNA abundance and protein abundance varies greatly depending on the cell stress, indicating stress-specific roles for post-transcriptional regulation. It is interesting in this respect that one of the biological modules we identified in HU regulates mRNA translation and stability.

Post-transcriptional regulation in the response to MMS

Comparison of the protein abundance changes that occur during MMS treatment with their corresponding mRNA changes8 yielded a positive correlation for the top 300 abundance changes (r=0.457; Supplementary Information, Fig. S2b), indicating that mRNA changes account for 21% of the variance in protein abundance changes. Thus, many increases in transcript levels did not result in corresponding protein changes, and including the entire set of proteins analyzed resulted in a poorer correlation (r=0.281; Supplementary Information, Figure S2d). These observations suggest that in the case of the MMS response post-transcriptional regulation is a critical determinant of the ultimate changes in protein abundance. This contrasts with the response to osmotic shock50 and rapamycin treatment51 in yeast where 80% and 36% of the protein abundance changes could be explained by cognate changes in mRNA abundance. It appears that the relationship between mRNA abundance and protein abundance varies greatly depending on the cell stress, indicating stress-specific roles for post-transcriptional regulation. It is interesting in this respect that one of the biological modules we identified in HU regulates mRNA translation and stability.

Identifying regulators in localization change neighbourhoodsThe high degree of biological process, genetic and physical interaction enrichment in most of the localization classes suggested that each class could represent a functionally connected ‘neighbourhood’ of proteins. Consistent with this possibility, the ‘to nuclear foci’ class was enriched for DNA repair proteins and we connected a protein in that class, Cmr1, with DNA repair in several ways (Fig. 6a, b, and c). We further interrogated the nuclear foci ‘neighbourhood’ to identify regulators of Cmr1 focus formation. It is interesting that our analysis of 27 genes identified three regulators, whereas in the case of Lsm1, screening the entire gene deletion collection of ~4500 genes identified only seven regulators (Fig. 7 and data not shown). While this single case has not yet been extrapolated, it is tempting to speculate that localization change categories will be enriched for regulatory relationships.We also found that analyzing the co-localization of proteins within the nuclear foci class revealed a new kind of sub-nuclear focus consisting of Cmr1, Hos2, Apj1 and Pph21that is not associated with the canonical DNA repair foci represented by proteins like Rad52 and Ddc2. The role of these proteins in the DNA damage response is unclear since the deletion of any one gene does not result in a strong damage sensitivity phenotype. However, one member of this group, Cmr1, contributes to genome stability32 and was recently demonstrated to interact with UV-damaged DNA in vitro and in vivo52. In one scenario, the chromatin remodeling activity of Hos2 might be required to permit Cmr1 to access ‘bulky’ DNA lesions. Alternatively, since Hos2 is required for the activation of DNA damage-inducible genes53, these foci might not represent sites of DNA damage, but rather sites of damage-induced transcription.

Identifying regulators in localization change neighbourhoods

The high degree of biological process, genetic and physical interaction enrichment in most of the localization classes suggested that each class could represent a functionally connected ‘neighbourhood’ of proteins. Consistent with this possibility, the ‘to nuclear foci’ class was enriched for DNA repair proteins and we connected a protein in that class, Cmr1, with DNA repair in several ways (Fig. 6a, b, and c). We further interrogated the nuclear foci ‘neighbourhood’ to identify regulators of Cmr1 focus formation. It is interesting that our analysis of 27 genes identified three regulators, whereas in the case of Lsm1, screening the entire gene deletion collection of ~4500 genes identified only seven regulators (Fig. 7 and data not shown). While this single case has not yet been extrapolated, it is tempting to speculate that localization change categories will be enriched for regulatory relationships.

We also found that analyzing the co-localization of proteins within the nuclear foci class revealed a new kind of sub-nuclear focus consisting of Cmr1, Hos2, Apj1 and Pph21that is not associated with the canonical DNA repair foci represented by proteins like Rad52 and Ddc2. The role of these proteins in the DNA damage response is unclear since the deletion of any one gene does not result in a strong damage sensitivity phenotype. However, one member of this group, Cmr1, contributes to genome stability32 and was recently demonstrated to interact with UV-damaged DNA in vitro and in vivo52. In one scenario, the chromatin remodeling activity of Hos2 might be required to permit Cmr1 to access ‘bulky’ DNA lesions. Alternatively, since Hos2 is required for the activation of DNA damage-inducible genes53, these foci might not represent sites of DNA damage, but rather sites of damage-induced transcription.

A novel pathway regulating cytoplasmic P-bodiesThe redistribution of proteins from a diffuse cytoplasmic distribution to cytoplasmic foci formed the most striking relocalization class in our screen and represents the formation of P-bodies (Fig. 7a). We demonstrate that replication stress is a potent inducer of P-body formation, suggesting that replication stress causes an increase in non-translating mRNAs, and indicating an important role for post-transcriptional regulation in the genotoxic stress response. We found that Asc1 is required for P-body formation in the HU response, but not in response to glucose deprivation/osmotic stress, and acts upstream of the key regulator of P-body formation, Pat1. Thus, HU induction of P-bodies is regulated differently than induction by more classical conditions, and so forms a distinct branch of the P-body pathway. Both Pat1 and Lsm1 are required for resistance to hydroxyurea (Supplemental Information, Fig. S7d) and the topoisomerase I poison camptothecin49. It was recently shown that Lsm1 contributes to the turnover of histone mRNA and that loss of this function contributes to the HU-sensitivity of lsm1Δ strains55. Thus, the P-bodies we observe may in part reflect the turnover of histone mRNA in response to replication stress.Asc1, and its mammalian homologue RACK1 are signaling adaptor proteins that regulate diverse cellular processes56. In addition, both Asc1 and RACK1 are stoichiometric components of the ribosome and are thought to recruit regulators to the ribosome to modulate translation56, 57. It will be interesting to determine if RACK1 modulates P-body assembly in response to replication stress in mammalian cells, and whether such a role is relevant to the upregulation of RACK1 that is common in neoplasias56.We provide a comprehensive resource detailing the protein abundance and localization changes that occur during replication stress in yeast. Our data demonstrate the potential of high-throughput microscopic screening to identify novel response pathways and their regulators. The methodology can be readily applied to virtually any genetic or chemical perturbation.

A novel pathway regulating cytoplasmic P-bodies

The redistribution of proteins from a diffuse cytoplasmic distribution to cytoplasmic foci formed the most striking relocalization class in our screen and represents the formation of P-bodies (Fig. 7a). We demonstrate that replication stress is a potent inducer of P-body formation, suggesting that replication stress causes an increase in non-translating mRNAs, and indicating an important role for post-transcriptional regulation in the genotoxic stress response. We found that Asc1 is required for P-body formation in the HU response, but not in response to glucose deprivation/osmotic stress, and acts upstream of the key regulator of P-body formation, Pat1. Thus, HU induction of P-bodies is regulated differently than induction by more classical conditions, and so forms a distinct branch of the P-body pathway. Both Pat1 and Lsm1 are required for resistance to hydroxyurea (Supplemental Information, Fig. S7d) and the topoisomerase I poison camptothecin49. It was recently shown that Lsm1 contributes to the turnover of histone mRNA and that loss of this function contributes to the HU-sensitivity of lsm1Δ strains55. Thus, the P-bodies we observe may in part reflect the turnover of histone mRNA in response to replication stress.

Asc1, and its mammalian homologue RACK1 are signaling adaptor proteins that regulate diverse cellular processes56. In addition, both Asc1 and RACK1 are stoichiometric components of the ribosome and are thought to recruit regulators to the ribosome to modulate translation56, 57. It will be interesting to determine if RACK1 modulates P-body assembly in response to replication stress in mammalian cells, and whether such a role is relevant to the upregulation of RACK1 that is common in neoplasias56.

We provide a comprehensive resource detailing the protein abundance and localization changes that occur during replication stress in yeast. Our data demonstrate the potential of high-throughput microscopic screening to identify novel response pathways and their regulators. The methodology can be readily applied to virtually any genetic or chemical perturbation.

MethodsStrains and mediaYeast strains used in this study (Supplementary Information, Table S9) are derivatives of BY474158. Unless indicated otherwise, standard yeast media and growth conditions were used59. For high throughput screening, low fluorescence media, YNB (MP Biomedicals) was supplemented with 5 g/L ammonium sulfate, 2% (w/v) glucose and standard amounts of methionine, histidine, leucine and uracil59. For all other microscopy, low fluorescence media containing ammonium sulfate and glucose was supplemented with standard amounts of adenine, arginine, isoleucine, valine, histidine, leucine, lysine, methionine, phenylalanine, threonine, tryptophan, tyrosine and uracil.Screen to identify protein abundance and localization changes in response to replication stressJTY7 containing a NUP49-mCherry::CaURA3 marker was crossed to the yeast GFP collection by SGA36. The resulting strains were grown to saturation (~24 h growth time) in a 96-well format and further subcultured to mid-log phase (~0.3 OD/mL) at 30°C in low fluorescence medium (~16 h growth time). Cells were transferred to a 384-well slide to a final density of 0.045 OD/mL (Perkin-Elmer) and incubated at 30°C for 2 h with additional medium (control), 0.2 M hydroxyurea (HU, SIGMA) or 0.03 % methyl methanesulfonate (MMS, SIGMA). Images from three areas per well in the green (405/488/640 primary dichroic, 540/75 emission bandpass filter, 800 ms exposure) and red channels (405/561/640 primary dichroic, 600/40 emission bandpass filter, 2000 ms exposure) were obtained using the EVOTEC Opera confocal microscope system (PerkinElmer). All raw images are available from the Yeast Resource Center Public Image Repository (http://images.yeastrc.org/tkach_brown/replication_stress).Localization change raw data scoring and refinementThe images were blinded and scored manually for localization changes in drug-treated samples. For each protein undergoing localization change, a brief description of the protein localization in control and drug treated cells was recorded (Supplementary Information, Table S1). For cases where the protein was present in more than one compartment, the compartments are listed in order of phenotypic prominence; a protein located in both the nucleus and cytoplasm but appears more abundant in the nucleus would have the designation “Nucleus, cytoplasm”. In the case where the protein is distributed equally, “and” is used to separate the compartments (eg. Nucleus and cytoplasm). Where distinct populations of cells were observed, ‘or’ was used to separate the descriptions (eg. Nucleus and cytoplasm or nucleus). When assessing the change that occurred after drug treatment, it was possible that the protein was still present in the same compartments but that its relative distribution had changed. For example, a protein present in the nucleus and cytoplasm could become more nuclear after drug treatment while retaining some cytoplasmic signal. In this case, “Nucleus, cytoplasm” indicates a re-distribution to the nucleus. Although we gathered detailed information regarding each protein localization, to facilitate further analysis each localization call was refined to a single term representing the predominant localization (all “Nucleus and cytoplasm” were designated “Nucleus”). The localization class represents the net change in protein distribution between control and drug-treated samples. All classes represent the predominant localization after drug treatment with the exception of the ‘From budneck/tip” category.Automated analysis to determine abundance changesTo determine overall abundance the .flex image files were analyzed using the provided CellProfiler pipeline (ScreenAnalysis.cp; Supplementary Note 1). Briefly, the RFP channel was analyzed for primary objects (nuclei) using global robust background thresholding. For this method, the brightest and dimmest pixel intensities are trimmed by 5% and the threshold is calculated as the mean plus two standard deviations of the remaining pixel values. The primary objects were overlaid onto the corresponding GFP channel and measurements corresponding to the nuclei were obtained. The edge of the nuclear object was extended by 6 pixels to obtain a secondary object referred to as the ‘cytoplasmic ring’. Fluorescence measurements within the cytoplasmic ring were obtained. R scripts ‘ReadExtractCombine.R’, ‘TakeMedian.R’ and ‘CalculateZScore.R’ were used to select relevant output data from CellProfiler and calculate fluorescence intensities and Z-scores (Supplementary Note 1). Based on examination of approximately 200 cells per sample it was estimated that the nucleus comprised approximately 30% and 35% of the cell area in the control and drug-treated images, respectively. The estimated cytoplasmic area was then calculated [nuclear area/ (0.3 or 0.35) – nuclear area] and used to calculate the total cytoplasmic intensity. The sum of the intensities measured for the nucleus and calculated for the cytoplasm represents the total cellular fluorescence. We next compared the median fluorescence intensity of all three control images to the median intensity of all three drug treated images to calculate an abundance change ratio (Supplementary Information, Table S1) for each strain. The median was used to buffer effects from small numbers of cells with fluorescence intensities that were greatly different from the rest of the population, or from spurious objects detected during the automated analysis. The Z-score was calculated based on the medians of the drug and control samples and the median absolute deviation (MAD) of the control sample ((mediandrug – mediancon)/ MADcon). Z-scores of -2 and 2 representing two MADs from the control median were chosen as cutoff values.Screen for regulators of HU-induced Lsm1 P-bodiesAYY5, which expresses Lsm1-GFP from the native LSM1 locus, was crossed to the yeast deletion collection60 by SGA36 and the resulting array was grown and imaged after treatment with media (control) or HU as described above. The images were blinded and scored manually for strains that exhibited defects in Lsm1-GFP P-body formation. Raw images are available from the Yeast Resource Center Public Image Repository (http://images.yeastrc.org/tkach_brown/replication_stress).Gene-set Enrichment Analysis (GSEA) of proteins that change abundanceAn abundance profile was defined such that each gene in the GFP collection was associated with a Z-score as an index of protein abundance change. The profiles were analyzed by GSEA61 v2.07 in pre-rank mode. All default parameters were used except that the minimum and maximum gene set sizes were restricted to 5 and 300, respectively. Biological process and protein complex gene annotations were obtained from Gene Ontology (GO; http://berkeleybop.org/goose) on April 13, 2011. Additional protein complex annotations based on consensus across different studies were obtained from Benschop et. al. 201062. Enrichment maps were generated with the Enrichment Map Plugin v1.163 developed for Cytoscape64 using default parameters. The nodes in the map were clustered with the Markov clustering algorithm65, using the overlap coefficient computed by the plugin as the similarity metric (coefficients less than 0.5 were set to zero) and an inflation of 2.GO enrichment analysis of proteins that change localizationFor Fig. 3c and d, Fig. 4c, Supplementary Information
Fig. S3 and S5: Each gene set was analyzed for enrichment with GO biological processes and protein complexes (using the annotations used for GSEA, except that gene set sizes were only restricted to be ≤ 300). The significance of enrichment was computed using the hypergeometric test, relative to the genes in the GFP collection. FDR values were computed from the resulting P values using the Benjamini and Hochberg method65. For each gene set, an enrichment map was generated to illustrate the significantly enriched categories (FDR ≤ 0.01). Node clustering was performed as described for the GSEA-based enrichment maps. For Fig. 2: Each gene set was analyzed for GO biological process enrichment compared to the GFP collection using the Generic Gene Ontology Term Finder (http://go.princeton.edu/cgibin/GOTermFinder) using Bonferroni correction and all evidence codes. Enriched GO terms were further refined using ReviGO66 with a cut-off of p < 0.01. The top two or three refined terms are listed.Interaction enrichment analysesGeneMANIA (http://www.genemania.org/
67) was used to generate protein-protein and genetic interaction networks. For both networks the ‘equal by network’ network weighting method was used and only input genes were included in the networks (i.e. no related genes were returned). For the protein-protein interaction network, all available data sets were used (GeneMANIA datasets as of June 2011). For the genetic interaction network the Costanzo-Boone-2010_positive/negative_interactions_full datasets were excluded. To calculate the interaction enrichment of the network and the associated p-value, the total number of pairwise interactions indicated by GeneMANIA was compared to the total number of pairwise interactions among the yeast GFP collection genes in GeneMANIA, using a hypergeometric test. Nodes were manually arranged for clarity, but the overall shape of each network was preserved. For the GeneMANIA analysis of CMR1, the Costanzo-Boone-profile-similarity database was used to generate the profile similarity network (top 10 genes returned) and all databases were used to generate the physical interaction network (top 20 genes returned). GeneMANIA datasets were accessed December 2011.SGA analysis of CMR1 was performed as described36. Negative genetic interactions with CMR1, both as the query and array strain, scored as in Baryshnikova et al.68, and using the intermediate cutoff (−0.08) recommended in Constanzo et al.69, were used to construct the CMR1 genetic interaction network in Cytoscape 2.864.Nodes in all networks were colored according to the biological process annotation provided in Costanzo et al.69. Genes absent in this set were manually annotated (Supplementary Information, Table S10).Confocal fluorescence microscopy and image analysisFor P-body analyses, cultures were grown to mid-log phase in YPD at 30°C, washed once in low fluorescence medium, water or low fluorescence medium containing drug. Where indicated, cultures were treated for 15 min in water or 2 h in 2 M HU. For analysis of Cmr1-GFP nuclear foci, cultures were grown to saturation in YPD, diluted into fresh YPD at 0.4 OD/mL and grown for 3 h at 30°C before treating with 0.03% MMS for 2 h. 11 z-slices with a 0.4 μm step size were obtained using Volocity imaging software (PerkinElmer) controlling a Leica DMI6000 microscope with the FITC, Texas Red and DIC filter sets (Quorum Technologies). Where indicated, the resulting maximum Z-projections were analyzed using CellProfiler pipelines (Supplementary Note 1). For P-body analyses: The pipelines PbodyFocusMeasure.cp and PbodyFocusMeasure_mec1tel1.cp) were used. Briefly, total cellular fluorescence was used to identify primary objects using an Otsu global background method. The resulting objects were used to mask the GFP image to ensure that foci were only identified within previously identified objects. Foci were identified using a robust background method on a per object basis; this method detects foci based on their relative intensity compared to the overall fluorescence within a cell and is not affected by variations in total fluorescence between cells or strains. Foci were associated with each parent object and the size and intensity of each focus was measured and output to a spreadsheet. The larger cell size of the mec1Δtel1Δ strain necessitated a modified pipeline to account for this change. For nuclear focus analyses: The pipelines ‘NucFocusIdent_Apj1_Hos2_Ydl.cp’ and ‘NucFocusIdent_Pph21.cp’ were used. The pipelines work essentially as described for P-body analysis except the primary object identification was modified to identify nuclei (‘NucFocusIdent_Apj1_Hos2_Ydl.cp’, for analysis of Apj1, Hos2 and Cmr1) or whole cells (‘NucFocusIdent_Pph21.cp’, for analysis of Pph21).Western blot and drug sensitivity assaysWestern blotting: Cultures were grown to OD ~0.5 in YPD at 30°C. 5 OD of cells were treated as indicated and fixed with 10% trichloroacetic acid (Sigma-Aldrich) and prepared as described9. Samples were separated by SDS polyacrylamide gel electrophoresis, transferred to nitrocellulose and blocked with TBS containing 0.05% Tween-20 (TBST) and 5% skim milk powder. To detect p-H2A: α-p-H2A (Abcam, ab15083) 1:500 overnight at 4°C followed by α-rabbit HRP (Pierce Chemical) 1:10,000 for 1 h at RT. To detect H2A: α-H2A (Abcam, ab13923) 1:2500 overnight at 4°C followed by α-rabbit HRP (Pierce Chemical) 1:10,000 for 1 h at RT. All antibodies were diluted in TBST plus milk. Western blots were developed using SuperSignal ECL (Pierce Chemical), imaged with a Versadoc MP 5000 (BioRad) and quantified using ImageJ (http://imagej.nih.gov/ij/).Drug sensitivityCultures were grown overnight at 30°C in YPD. Cell densities were equalized to OD = 1, serially diluted ten-fold, spotted on the indicated medium and grown for 2-3 d at 30°C before imaging.

Strains and mediaYeast strains used in this study (Supplementary Information, Table S9) are derivatives of BY474158. Unless indicated otherwise, standard yeast media and growth conditions were used59. For high throughput screening, low fluorescence media, YNB (MP Biomedicals) was supplemented with 5 g/L ammonium sulfate, 2% (w/v) glucose and standard amounts of methionine, histidine, leucine and uracil59. For all other microscopy, low fluorescence media containing ammonium sulfate and glucose was supplemented with standard amounts of adenine, arginine, isoleucine, valine, histidine, leucine, lysine, methionine, phenylalanine, threonine, tryptophan, tyrosine and uracil.

Strains and media

Yeast strains used in this study (Supplementary Information, Table S9) are derivatives of BY474158. Unless indicated otherwise, standard yeast media and growth conditions were used59. For high throughput screening, low fluorescence media, YNB (MP Biomedicals) was supplemented with 5 g/L ammonium sulfate, 2% (w/v) glucose and standard amounts of methionine, histidine, leucine and uracil59. For all other microscopy, low fluorescence media containing ammonium sulfate and glucose was supplemented with standard amounts of adenine, arginine, isoleucine, valine, histidine, leucine, lysine, methionine, phenylalanine, threonine, tryptophan, tyrosine and uracil.

Screen to identify protein abundance and localization changes in response to replication stressJTY7 containing a NUP49-mCherry::CaURA3 marker was crossed to the yeast GFP collection by SGA36. The resulting strains were grown to saturation (~24 h growth time) in a 96-well format and further subcultured to mid-log phase (~0.3 OD/mL) at 30°C in low fluorescence medium (~16 h growth time). Cells were transferred to a 384-well slide to a final density of 0.045 OD/mL (Perkin-Elmer) and incubated at 30°C for 2 h with additional medium (control), 0.2 M hydroxyurea (HU, SIGMA) or 0.03 % methyl methanesulfonate (MMS, SIGMA). Images from three areas per well in the green (405/488/640 primary dichroic, 540/75 emission bandpass filter, 800 ms exposure) and red channels (405/561/640 primary dichroic, 600/40 emission bandpass filter, 2000 ms exposure) were obtained using the EVOTEC Opera confocal microscope system (PerkinElmer). All raw images are available from the Yeast Resource Center Public Image Repository (http://images.yeastrc.org/tkach_brown/replication_stress).

Screen to identify protein abundance and localization changes in response to replication stress

JTY7 containing a NUP49-mCherry::CaURA3 marker was crossed to the yeast GFP collection by SGA36. The resulting strains were grown to saturation (~24 h growth time) in a 96-well format and further subcultured to mid-log phase (~0.3 OD/mL) at 30°C in low fluorescence medium (~16 h growth time). Cells were transferred to a 384-well slide to a final density of 0.045 OD/mL (Perkin-Elmer) and incubated at 30°C for 2 h with additional medium (control), 0.2 M hydroxyurea (HU, SIGMA) or 0.03 % methyl methanesulfonate (MMS, SIGMA). Images from three areas per well in the green (405/488/640 primary dichroic, 540/75 emission bandpass filter, 800 ms exposure) and red channels (405/561/640 primary dichroic, 600/40 emission bandpass filter, 2000 ms exposure) were obtained using the EVOTEC Opera confocal microscope system (PerkinElmer). All raw images are available from the Yeast Resource Center Public Image Repository (http://images.yeastrc.org/tkach_brown/replication_stress).

Localization change raw data scoring and refinementThe images were blinded and scored manually for localization changes in drug-treated samples. For each protein undergoing localization change, a brief description of the protein localization in control and drug treated cells was recorded (Supplementary Information, Table S1). For cases where the protein was present in more than one compartment, the compartments are listed in order of phenotypic prominence; a protein located in both the nucleus and cytoplasm but appears more abundant in the nucleus would have the designation “Nucleus, cytoplasm”. In the case where the protein is distributed equally, “and” is used to separate the compartments (eg. Nucleus and cytoplasm). Where distinct populations of cells were observed, ‘or’ was used to separate the descriptions (eg. Nucleus and cytoplasm or nucleus). When assessing the change that occurred after drug treatment, it was possible that the protein was still present in the same compartments but that its relative distribution had changed. For example, a protein present in the nucleus and cytoplasm could become more nuclear after drug treatment while retaining some cytoplasmic signal. In this case, “Nucleus, cytoplasm” indicates a re-distribution to the nucleus. Although we gathered detailed information regarding each protein localization, to facilitate further analysis each localization call was refined to a single term representing the predominant localization (all “Nucleus and cytoplasm” were designated “Nucleus”). The localization class represents the net change in protein distribution between control and drug-treated samples. All classes represent the predominant localization after drug treatment with the exception of the ‘From budneck/tip” category.

Localization change raw data scoring and refinement

The images were blinded and scored manually for localization changes in drug-treated samples. For each protein undergoing localization change, a brief description of the protein localization in control and drug treated cells was recorded (Supplementary Information, Table S1). For cases where the protein was present in more than one compartment, the compartments are listed in order of phenotypic prominence; a protein located in both the nucleus and cytoplasm but appears more abundant in the nucleus would have the designation “Nucleus, cytoplasm”. In the case where the protein is distributed equally, “and” is used to separate the compartments (eg. Nucleus and cytoplasm). Where distinct populations of cells were observed, ‘or’ was used to separate the descriptions (eg. Nucleus and cytoplasm or nucleus). When assessing the change that occurred after drug treatment, it was possible that the protein was still present in the same compartments but that its relative distribution had changed. For example, a protein present in the nucleus and cytoplasm could become more nuclear after drug treatment while retaining some cytoplasmic signal. In this case, “Nucleus, cytoplasm” indicates a re-distribution to the nucleus. Although we gathered detailed information regarding each protein localization, to facilitate further analysis each localization call was refined to a single term representing the predominant localization (all “Nucleus and cytoplasm” were designated “Nucleus”). The localization class represents the net change in protein distribution between control and drug-treated samples. All classes represent the predominant localization after drug treatment with the exception of the ‘From budneck/tip” category.

Automated analysis to determine abundance changesTo determine overall abundance the .flex image files were analyzed using the provided CellProfiler pipeline (ScreenAnalysis.cp; Supplementary Note 1). Briefly, the RFP channel was analyzed for primary objects (nuclei) using global robust background thresholding. For this method, the brightest and dimmest pixel intensities are trimmed by 5% and the threshold is calculated as the mean plus two standard deviations of the remaining pixel values. The primary objects were overlaid onto the corresponding GFP channel and measurements corresponding to the nuclei were obtained. The edge of the nuclear object was extended by 6 pixels to obtain a secondary object referred to as the ‘cytoplasmic ring’. Fluorescence measurements within the cytoplasmic ring were obtained. R scripts ‘ReadExtractCombine.R’, ‘TakeMedian.R’ and ‘CalculateZScore.R’ were used to select relevant output data from CellProfiler and calculate fluorescence intensities and Z-scores (Supplementary Note 1). Based on examination of approximately 200 cells per sample it was estimated that the nucleus comprised approximately 30% and 35% of the cell area in the control and drug-treated images, respectively. The estimated cytoplasmic area was then calculated [nuclear area/ (0.3 or 0.35) – nuclear area] and used to calculate the total cytoplasmic intensity. The sum of the intensities measured for the nucleus and calculated for the cytoplasm represents the total cellular fluorescence. We next compared the median fluorescence intensity of all three control images to the median intensity of all three drug treated images to calculate an abundance change ratio (Supplementary Information, Table S1) for each strain. The median was used to buffer effects from small numbers of cells with fluorescence intensities that were greatly different from the rest of the population, or from spurious objects detected during the automated analysis. The Z-score was calculated based on the medians of the drug and control samples and the median absolute deviation (MAD) of the control sample ((mediandrug – mediancon)/ MADcon). Z-scores of -2 and 2 representing two MADs from the control median were chosen as cutoff values.

Automated analysis to determine abundance changes

To determine overall abundance the .flex image files were analyzed using the provided CellProfiler pipeline (ScreenAnalysis.cp; Supplementary Note 1). Briefly, the RFP channel was analyzed for primary objects (nuclei) using global robust background thresholding. For this method, the brightest and dimmest pixel intensities are trimmed by 5% and the threshold is calculated as the mean plus two standard deviations of the remaining pixel values. The primary objects were overlaid onto the corresponding GFP channel and measurements corresponding to the nuclei were obtained. The edge of the nuclear object was extended by 6 pixels to obtain a secondary object referred to as the ‘cytoplasmic ring’. Fluorescence measurements within the cytoplasmic ring were obtained. R scripts ‘ReadExtractCombine.R’, ‘TakeMedian.R’ and ‘CalculateZScore.R’ were used to select relevant output data from CellProfiler and calculate fluorescence intensities and Z-scores (Supplementary Note 1). Based on examination of approximately 200 cells per sample it was estimated that the nucleus comprised approximately 30% and 35% of the cell area in the control and drug-treated images, respectively. The estimated cytoplasmic area was then calculated [nuclear area/ (0.3 or 0.35) – nuclear area] and used to calculate the total cytoplasmic intensity. The sum of the intensities measured for the nucleus and calculated for the cytoplasm represents the total cellular fluorescence. We next compared the median fluorescence intensity of all three control images to the median intensity of all three drug treated images to calculate an abundance change ratio (Supplementary Information, Table S1) for each strain. The median was used to buffer effects from small numbers of cells with fluorescence intensities that were greatly different from the rest of the population, or from spurious objects detected during the automated analysis. The Z-score was calculated based on the medians of the drug and control samples and the median absolute deviation (MAD) of the control sample ((mediandrug – mediancon)/ MADcon). Z-scores of -2 and 2 representing two MADs from the control median were chosen as cutoff values.

Screen for regulators of HU-induced Lsm1 P-bodiesAYY5, which expresses Lsm1-GFP from the native LSM1 locus, was crossed to the yeast deletion collection60 by SGA36 and the resulting array was grown and imaged after treatment with media (control) or HU as described above. The images were blinded and scored manually for strains that exhibited defects in Lsm1-GFP P-body formation. Raw images are available from the Yeast Resource Center Public Image Repository (http://images.yeastrc.org/tkach_brown/replication_stress).

Screen for regulators of HU-induced Lsm1 P-bodies

AYY5, which expresses Lsm1-GFP from the native LSM1 locus, was crossed to the yeast deletion collection60 by SGA36 and the resulting array was grown and imaged after treatment with media (control) or HU as described above. The images were blinded and scored manually for strains that exhibited defects in Lsm1-GFP P-body formation. Raw images are available from the Yeast Resource Center Public Image Repository (http://images.yeastrc.org/tkach_brown/replication_stress).

Gene-set Enrichment Analysis (GSEA) of proteins that change abundanceAn abundance profile was defined such that each gene in the GFP collection was associated with a Z-score as an index of protein abundance change. The profiles were analyzed by GSEA61 v2.07 in pre-rank mode. All default parameters were used except that the minimum and maximum gene set sizes were restricted to 5 and 300, respectively. Biological process and protein complex gene annotations were obtained from Gene Ontology (GO; http://berkeleybop.org/goose) on April 13, 2011. Additional protein complex annotations based on consensus across different studies were obtained from Benschop et. al. 201062. Enrichment maps were generated with the Enrichment Map Plugin v1.163 developed for Cytoscape64 using default parameters. The nodes in the map were clustered with the Markov clustering algorithm65, using the overlap coefficient computed by the plugin as the similarity metric (coefficients less than 0.5 were set to zero) and an inflation of 2.

Gene-set Enrichment Analysis (GSEA) of proteins that change abundance

An abundance profile was defined such that each gene in the GFP collection was associated with a Z-score as an index of protein abundance change. The profiles were analyzed by GSEA61 v2.07 in pre-rank mode. All default parameters were used except that the minimum and maximum gene set sizes were restricted to 5 and 300, respectively. Biological process and protein complex gene annotations were obtained from Gene Ontology (GO; http://berkeleybop.org/goose) on April 13, 2011. Additional protein complex annotations based on consensus across different studies were obtained from Benschop et. al. 201062. Enrichment maps were generated with the Enrichment Map Plugin v1.163 developed for Cytoscape64 using default parameters. The nodes in the map were clustered with the Markov clustering algorithm65, using the overlap coefficient computed by the plugin as the similarity metric (coefficients less than 0.5 were set to zero) and an inflation of 2.

GO enrichment analysis of proteins that change localizationFor Fig. 3c and d, Fig. 4c, Supplementary Information
Fig. S3 and S5: Each gene set was analyzed for enrichment with GO biological processes and protein complexes (using the annotations used for GSEA, except that gene set sizes were only restricted to be ≤ 300). The significance of enrichment was computed using the hypergeometric test, relative to the genes in the GFP collection. FDR values were computed from the resulting P values using the Benjamini and Hochberg method65. For each gene set, an enrichment map was generated to illustrate the significantly enriched categories (FDR ≤ 0.01). Node clustering was performed as described for the GSEA-based enrichment maps. For Fig. 2: Each gene set was analyzed for GO biological process enrichment compared to the GFP collection using the Generic Gene Ontology Term Finder (http://go.princeton.edu/cgibin/GOTermFinder) using Bonferroni correction and all evidence codes. Enriched GO terms were further refined using ReviGO66 with a cut-off of p < 0.01. The top two or three refined terms are listed.

GO enrichment analysis of proteins that change localization

For Fig. 3c and d, Fig. 4c, Supplementary Information
Fig. S3 and S5: Each gene set was analyzed for enrichment with GO biological processes and protein complexes (using the annotations used for GSEA, except that gene set sizes were only restricted to be ≤ 300). The significance of enrichment was computed using the hypergeometric test, relative to the genes in the GFP collection. FDR values were computed from the resulting P values using the Benjamini and Hochberg method65. For each gene set, an enrichment map was generated to illustrate the significantly enriched categories (FDR ≤ 0.01). Node clustering was performed as described for the GSEA-based enrichment maps. For Fig. 2: Each gene set was analyzed for GO biological process enrichment compared to the GFP collection using the Generic Gene Ontology Term Finder (http://go.princeton.edu/cgibin/GOTermFinder) using Bonferroni correction and all evidence codes. Enriched GO terms were further refined using ReviGO66 with a cut-off of p < 0.01. The top two or three refined terms are listed.

Interaction enrichment analysesGeneMANIA (http://www.genemania.org/
67) was used to generate protein-protein and genetic interaction networks. For both networks the ‘equal by network’ network weighting method was used and only input genes were included in the networks (i.e. no related genes were returned). For the protein-protein interaction network, all available data sets were used (GeneMANIA datasets as of June 2011). For the genetic interaction network the Costanzo-Boone-2010_positive/negative_interactions_full datasets were excluded. To calculate the interaction enrichment of the network and the associated p-value, the total number of pairwise interactions indicated by GeneMANIA was compared to the total number of pairwise interactions among the yeast GFP collection genes in GeneMANIA, using a hypergeometric test. Nodes were manually arranged for clarity, but the overall shape of each network was preserved. For the GeneMANIA analysis of CMR1, the Costanzo-Boone-profile-similarity database was used to generate the profile similarity network (top 10 genes returned) and all databases were used to generate the physical interaction network (top 20 genes returned). GeneMANIA datasets were accessed December 2011.SGA analysis of CMR1 was performed as described36. Negative genetic interactions with CMR1, both as the query and array strain, scored as in Baryshnikova et al.68, and using the intermediate cutoff (−0.08) recommended in Constanzo et al.69, were used to construct the CMR1 genetic interaction network in Cytoscape 2.864.Nodes in all networks were colored according to the biological process annotation provided in Costanzo et al.69. Genes absent in this set were manually annotated (Supplementary Information, Table S10).

Interaction enrichment analyses

GeneMANIA (http://www.genemania.org/
67) was used to generate protein-protein and genetic interaction networks. For both networks the ‘equal by network’ network weighting method was used and only input genes were included in the networks (i.e. no related genes were returned). For the protein-protein interaction network, all available data sets were used (GeneMANIA datasets as of June 2011). For the genetic interaction network the Costanzo-Boone-2010_positive/negative_interactions_full datasets were excluded. To calculate the interaction enrichment of the network and the associated p-value, the total number of pairwise interactions indicated by GeneMANIA was compared to the total number of pairwise interactions among the yeast GFP collection genes in GeneMANIA, using a hypergeometric test. Nodes were manually arranged for clarity, but the overall shape of each network was preserved. For the GeneMANIA analysis of CMR1, the Costanzo-Boone-profile-similarity database was used to generate the profile similarity network (top 10 genes returned) and all databases were used to generate the physical interaction network (top 20 genes returned). GeneMANIA datasets were accessed December 2011.

SGA analysis of CMR1 was performed as described36. Negative genetic interactions with CMR1, both as the query and array strain, scored as in Baryshnikova et al.68, and using the intermediate cutoff (−0.08) recommended in Constanzo et al.69, were used to construct the CMR1 genetic interaction network in Cytoscape 2.864.

Nodes in all networks were colored according to the biological process annotation provided in Costanzo et al.69. Genes absent in this set were manually annotated (Supplementary Information, Table S10).

Confocal fluorescence microscopy and image analysisFor P-body analyses, cultures were grown to mid-log phase in YPD at 30°C, washed once in low fluorescence medium, water or low fluorescence medium containing drug. Where indicated, cultures were treated for 15 min in water or 2 h in 2 M HU. For analysis of Cmr1-GFP nuclear foci, cultures were grown to saturation in YPD, diluted into fresh YPD at 0.4 OD/mL and grown for 3 h at 30°C before treating with 0.03% MMS for 2 h. 11 z-slices with a 0.4 μm step size were obtained using Volocity imaging software (PerkinElmer) controlling a Leica DMI6000 microscope with the FITC, Texas Red and DIC filter sets (Quorum Technologies). Where indicated, the resulting maximum Z-projections were analyzed using CellProfiler pipelines (Supplementary Note 1). For P-body analyses: The pipelines PbodyFocusMeasure.cp and PbodyFocusMeasure_mec1tel1.cp) were used. Briefly, total cellular fluorescence was used to identify primary objects using an Otsu global background method. The resulting objects were used to mask the GFP image to ensure that foci were only identified within previously identified objects. Foci were identified using a robust background method on a per object basis; this method detects foci based on their relative intensity compared to the overall fluorescence within a cell and is not affected by variations in total fluorescence between cells or strains. Foci were associated with each parent object and the size and intensity of each focus was measured and output to a spreadsheet. The larger cell size of the mec1Δtel1Δ strain necessitated a modified pipeline to account for this change. For nuclear focus analyses: The pipelines ‘NucFocusIdent_Apj1_Hos2_Ydl.cp’ and ‘NucFocusIdent_Pph21.cp’ were used. The pipelines work essentially as described for P-body analysis except the primary object identification was modified to identify nuclei (‘NucFocusIdent_Apj1_Hos2_Ydl.cp’, for analysis of Apj1, Hos2 and Cmr1) or whole cells (‘NucFocusIdent_Pph21.cp’, for analysis of Pph21).

Confocal fluorescence microscopy and image analysis

For P-body analyses, cultures were grown to mid-log phase in YPD at 30°C, washed once in low fluorescence medium, water or low fluorescence medium containing drug. Where indicated, cultures were treated for 15 min in water or 2 h in 2 M HU. For analysis of Cmr1-GFP nuclear foci, cultures were grown to saturation in YPD, diluted into fresh YPD at 0.4 OD/mL and grown for 3 h at 30°C before treating with 0.03% MMS for 2 h. 11 z-slices with a 0.4 μm step size were obtained using Volocity imaging software (PerkinElmer) controlling a Leica DMI6000 microscope with the FITC, Texas Red and DIC filter sets (Quorum Technologies). Where indicated, the resulting maximum Z-projections were analyzed using CellProfiler pipelines (Supplementary Note 1). For P-body analyses: The pipelines PbodyFocusMeasure.cp and PbodyFocusMeasure_mec1tel1.cp) were used. Briefly, total cellular fluorescence was used to identify primary objects using an Otsu global background method. The resulting objects were used to mask the GFP image to ensure that foci were only identified within previously identified objects. Foci were identified using a robust background method on a per object basis; this method detects foci based on their relative intensity compared to the overall fluorescence within a cell and is not affected by variations in total fluorescence between cells or strains. Foci were associated with each parent object and the size and intensity of each focus was measured and output to a spreadsheet. The larger cell size of the mec1Δtel1Δ strain necessitated a modified pipeline to account for this change. For nuclear focus analyses: The pipelines ‘NucFocusIdent_Apj1_Hos2_Ydl.cp’ and ‘NucFocusIdent_Pph21.cp’ were used. The pipelines work essentially as described for P-body analysis except the primary object identification was modified to identify nuclei (‘NucFocusIdent_Apj1_Hos2_Ydl.cp’, for analysis of Apj1, Hos2 and Cmr1) or whole cells (‘NucFocusIdent_Pph21.cp’, for analysis of Pph21).

Western blot and drug sensitivity assaysWestern blotting: Cultures were grown to OD ~0.5 in YPD at 30°C. 5 OD of cells were treated as indicated and fixed with 10% trichloroacetic acid (Sigma-Aldrich) and prepared as described9. Samples were separated by SDS polyacrylamide gel electrophoresis, transferred to nitrocellulose and blocked with TBS containing 0.05% Tween-20 (TBST) and 5% skim milk powder. To detect p-H2A: α-p-H2A (Abcam, ab15083) 1:500 overnight at 4°C followed by α-rabbit HRP (Pierce Chemical) 1:10,000 for 1 h at RT. To detect H2A: α-H2A (Abcam, ab13923) 1:2500 overnight at 4°C followed by α-rabbit HRP (Pierce Chemical) 1:10,000 for 1 h at RT. All antibodies were diluted in TBST plus milk. Western blots were developed using SuperSignal ECL (Pierce Chemical), imaged with a Versadoc MP 5000 (BioRad) and quantified using ImageJ (http://imagej.nih.gov/ij/).Drug sensitivityCultures were grown overnight at 30°C in YPD. Cell densities were equalized to OD = 1, serially diluted ten-fold, spotted on the indicated medium and grown for 2-3 d at 30°C before imaging.

Western blot and drug sensitivity assays

Western blotting: Cultures were grown to OD ~0.5 in YPD at 30°C. 5 OD of cells were treated as indicated and fixed with 10% trichloroacetic acid (Sigma-Aldrich) and prepared as described9. Samples were separated by SDS polyacrylamide gel electrophoresis, transferred to nitrocellulose and blocked with TBS containing 0.05% Tween-20 (TBST) and 5% skim milk powder. To detect p-H2A: α-p-H2A (Abcam, ab15083) 1:500 overnight at 4°C followed by α-rabbit HRP (Pierce Chemical) 1:10,000 for 1 h at RT. To detect H2A: α-H2A (Abcam, ab13923) 1:2500 overnight at 4°C followed by α-rabbit HRP (Pierce Chemical) 1:10,000 for 1 h at RT. All antibodies were diluted in TBST plus milk. Western blots were developed using SuperSignal ECL (Pierce Chemical), imaged with a Versadoc MP 5000 (BioRad) and quantified using ImageJ (http://imagej.nih.gov/ij/).

Drug sensitivityCultures were grown overnight at 30°C in YPD. Cell densities were equalized to OD = 1, serially diluted ten-fold, spotted on the indicated medium and grown for 2-3 d at 30°C before imaging.

Drug sensitivity

Cultures were grown overnight at 30°C in YPD. Cell densities were equalized to OD = 1, serially diluted ten-fold, spotted on the indicated medium and grown for 2-3 d at 30°C before imaging.

Supplementary Material1567891011121314234

Supplementary Material
