---
pmid: '26751287'
title: The LINK-A lncRNA activates normoxic HIF1α signalling in triple-negative breast
  cancer.
authors:
- Lin A
- Li C
- Xing Z
- Hu Q
- Liang K
- Han L
- Wang C
- Hawke DH
- Wang S
- Zhang Y
- Wei Y
- Ma G
- Park PK
- Zhou J
- Zhou Y
- Hu Z
- Zhou Y
- Marks JR
- Liang H
- Hung MC
- Lin C
- Yang L
journal: Nat Cell Biol
year: '2016'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC4791069
doi: 10.1038/ncb3295
---

# The LINK-A lncRNA activates normoxic HIF1α signalling in triple-negative breast cancer.
**Authors:** Lin A, Li C, Xing Z, Hu Q, Liang K, Han L, Wang C, Hawke DH, Wang S, Zhang Y, Wei Y, Ma G, Park PK, Zhou J, Zhou Y, Hu Z, Zhou Y, Marks JR, Liang H, Hung MC, Lin C, Yang L
**Journal:** Nat Cell Biol (2016)
**DOI:** [10.1038/ncb3295](https://doi.org/10.1038/ncb3295)
**PMC:** [PMC4791069](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4791069/)

## Abstract

1. Nat Cell Biol. 2016 Feb;18(2):213-24. doi: 10.1038/ncb3295. Epub 2016 Jan 11.

The LINK-A lncRNA activates normoxic HIF1α signalling in triple-negative breast 
cancer.

Lin A(1), Li C(1), Xing Z(1), Hu Q(1), Liang K(1), Han L(2), Wang C(3), Hawke 
DH(4), Wang S(1), Zhang Y(1), Wei Y(1), Ma G(5), Park PK(1), Zhou J(6), Zhou 
Y(7), Hu Z(3), Zhou Y(5), Marks JR(8), Liang H(4)(9), Hung MC(1)(10)(11), Lin 
C(1)(10), Yang L(1)(10)(12).

Author information:
(1)Department of Molecular and Cellular Oncology, The University of Texas M D 
Anderson Cancer Center, Houston, Texas 77030, USA.
(2)Department of Biochemistry and Molecular Biology, The University of Texas 
Health Science Center at Houston McGroven Medical School, Houston, Texas 77030, 
USA.
(3)Department of Epidemiology and Biostatistics and Ministry of Education (MOE), 
School of Public Health, Nanjing Medical University, 210029, China.
(4)Department of System Biology, The University of Texas M D Anderson Cancer 
Center, Houston, Texas 77030, USA.
(5)Center for Translational Cancer Research, Institute of Biosciences and 
Technology, Texas A&M University Health Science Center, Houston, Texas 77030, 
USA.
(6)Department of Molecular Cell Biology and Toxicology, School of Public Health, 
Nanjing Medical University, 140 Hanzhong Road, Nanjing 210029, China.
(7)Department of Oncology, Yixing People's Hospital, 75 Zhenguan Road, 
Yixing 214200, China.
(8)Department of Surgery, Division of Surgical Science, Duke University, School 
of Medicine, Durham, North Carolina 27710, USA.
(9)Department of Bioinformatics and Computational Biology, Division of 
Quantitative Sciences, The University of Texas M D Anderson Cancer Center, 
Houston, Texas 77030, USA.
(10)The Graduate School of Biomedical Sciences, The University of Texas M D 
Anderson Cancer Center, Houston, Texas 77030, USA.
(11)Center for Molecular Medicine and Graduate Institute of Cancer Biology, 
China Medical University, Taichung 404, Taiwan.
(12)Center for RNA Interference and Non-Coding RNAs, The University of Texas M D 
Anderson Cancer Center, Houston, Texas 77030, USA.

Although long non-coding RNAs (lncRNAs) predominately reside in the nucleus and 
exert their functions in many biological processes, their potential involvement 
in cytoplasmic signal transduction remains unexplored. Here, we identify a 
cytoplasmic lncRNA, LINK-A (long intergenic non-coding RNA for kinase 
activation), which mediates HB-EGF-triggered, EGFR:GPNMB heterodimer-dependent 
HIF1α phosphorylation at Tyr 565 and Ser 797 by BRK and LRRK2, respectively. 
These events cause HIF1α stabilization, HIF1α-p300 interaction, and activation 
of HIF1α transcriptional programs under normoxic conditions. Mechanistically, 
LINK-A facilitates the recruitment of BRK to the EGFR:GPNMB complex and BRK 
kinase activation. The BRK-dependent HIF1α Tyr 565 phosphorylation interferes 
with Pro 564 hydroxylation, leading to normoxic HIF1α stabilization. Both LINK-A 
expression and LINK-A-dependent signalling pathway activation correlate with 
triple-negative breast cancer (TNBC), promoting breast cancer glycolysis 
reprogramming and tumorigenesis. Our findings illustrate the magnitude and 
diversity of cytoplasmic lncRNAs in signal transduction and highlight the 
important roles of lncRNAs in cancer.

DOI: 10.1038/ncb3295
PMCID: PMC4791069
PMID: 26751287 [Indexed for MEDLINE]

## Full Text

IntroductionTriple-Negative Breast Cancer (TNBC) continues to be a severe health problem1–3, demanding the consideration of emerging long non-coding RNAs (lncRNAs) as biomarkers and therapeutic targets in combatting this disease4–6. Accumulating evidence demonstrates that lncRNAs have broad functional roles in the nucleus: regulation of transcriptional activation, X chromosome inactivation, heterochromatin formation, and maintenance of telomeres7–14. Alterations of these functions promote tumor formation, progression, and metastasis of many cancer types15–20. However, many known lncRNAs reside either within the cytosol or shuttle between the nucleus and cytoplasm21, playing important roles in modulating messenger RNA translation, decay and cytoplasmic protein trafficking22–24. Intriguingly, many protein kinases and metabolic enzymes bind RNA via their noncanonical RNA-binding domains25–27, raising an important question whether cytoplasmic lncRNAs are relevant in the regulation of fundamental cellular processes.Hypoxia-Inducible Factors (HIFs)–targeted transcriptional program is involved in TNBC progression, recurrence, and metabolic reprogramming28–30. Although it is well known that the hydroxylation of HIF1α mediated by Proline Hydroxylase Domain (PHD) proteins triggers VHL-dependent HIF1α ubiquitination and degradation under normoxic conditions31,32; under certain circumstances in tumor, HIF1α can accumulate under normoxic conditions, promoting angiogenesis and cancer progression33,34. However, the mechanism underlying normoxic HIF1α stabilization in TNBC remains elusive.Here, we identified a highly prognostic lncRNA in TNBC, Long Intergenic Noncoding RNA for Kinase Activation (LINK-A) (also known as LOC339535 and NR_015407), which is critical for growth factor-induced normoxic HIF1α signaling pathway. Mechanistically, LINK-A is required for a HB-EGF-triggered, EGFR:GPNMB heterodimer-mediated recruitment of BRK to GPNMB, and subsequent enzymatic activation of BRK. The activated BRK, together with LRRK2 that is also recruited by LINK-A, phosphorylate HIF1α at Tyr565 and Ser797 respectively. While the phosphorylation at Tyr565 inhibits hydroxylation at the adjacent Pro564, which prevents HIF1α degradation under normoxic conditions, Ser797 phosphorylation facilitates HIF1α-p300 interaction, leading to activation of HIF1α target genes upon HB-EGF stimulation. Furthermore, we demonstrated that LRRK2, a constitutively active kinase in Parkinson’s disease, is a RNA binding kinase that phosphorylates HIF1α in human cancers. Importantly, both LINK-A expression and activation of the LINK-A-mediated normoxic HIF1a signaling pathway correlated with TNBC. Therefore, targeting LINK-A may serve as a favorable strategy to block a normoxic HIF1α signaling pathway in TNBC with promising therapeutic potential.

Introduction

Triple-Negative Breast Cancer (TNBC) continues to be a severe health problem1–3, demanding the consideration of emerging long non-coding RNAs (lncRNAs) as biomarkers and therapeutic targets in combatting this disease4–6. Accumulating evidence demonstrates that lncRNAs have broad functional roles in the nucleus: regulation of transcriptional activation, X chromosome inactivation, heterochromatin formation, and maintenance of telomeres7–14. Alterations of these functions promote tumor formation, progression, and metastasis of many cancer types15–20. However, many known lncRNAs reside either within the cytosol or shuttle between the nucleus and cytoplasm21, playing important roles in modulating messenger RNA translation, decay and cytoplasmic protein trafficking22–24. Intriguingly, many protein kinases and metabolic enzymes bind RNA via their noncanonical RNA-binding domains25–27, raising an important question whether cytoplasmic lncRNAs are relevant in the regulation of fundamental cellular processes.

Hypoxia-Inducible Factors (HIFs)–targeted transcriptional program is involved in TNBC progression, recurrence, and metabolic reprogramming28–30. Although it is well known that the hydroxylation of HIF1α mediated by Proline Hydroxylase Domain (PHD) proteins triggers VHL-dependent HIF1α ubiquitination and degradation under normoxic conditions31,32; under certain circumstances in tumor, HIF1α can accumulate under normoxic conditions, promoting angiogenesis and cancer progression33,34. However, the mechanism underlying normoxic HIF1α stabilization in TNBC remains elusive.

Here, we identified a highly prognostic lncRNA in TNBC, Long Intergenic Noncoding RNA for Kinase Activation (LINK-A) (also known as LOC339535 and NR_015407), which is critical for growth factor-induced normoxic HIF1α signaling pathway. Mechanistically, LINK-A is required for a HB-EGF-triggered, EGFR:GPNMB heterodimer-mediated recruitment of BRK to GPNMB, and subsequent enzymatic activation of BRK. The activated BRK, together with LRRK2 that is also recruited by LINK-A, phosphorylate HIF1α at Tyr565 and Ser797 respectively. While the phosphorylation at Tyr565 inhibits hydroxylation at the adjacent Pro564, which prevents HIF1α degradation under normoxic conditions, Ser797 phosphorylation facilitates HIF1α-p300 interaction, leading to activation of HIF1α target genes upon HB-EGF stimulation. Furthermore, we demonstrated that LRRK2, a constitutively active kinase in Parkinson’s disease, is a RNA binding kinase that phosphorylates HIF1α in human cancers. Importantly, both LINK-A expression and activation of the LINK-A-mediated normoxic HIF1a signaling pathway correlated with TNBC. Therefore, targeting LINK-A may serve as a favorable strategy to block a normoxic HIF1α signaling pathway in TNBC with promising therapeutic potential.

RESULTSLINK-A is a cytoplasmic lncRNA with prognostic value for TNBCTo identify TNBC-relevant lncRNAs, we examined lncRNA expression profile in two stage III TNBC tissues and their paired adjacent noncancerous tissues, finding 21 differentially expressed lncRNAs16. We further searched the expression pattern of these 21 lncRNAs in the TCGA database. Interestingly, statistical analysis of a combined 711 RNA-seq transcriptome profiles indicated that the expression of LINK-A is frequently elevated in TNBC patient cohorts in comparison to cohorts of ERPR−/HER2+, ERPR+/HER2-, and ERPR+/HER2+ patients. Differential LINK-A expression between ERPR−/HER2+, ERPR+/HER2-, and ERPR+/HER2+ cohorts were not statistically significant (Fig. 1a). Consistently, basal-like breast cancer, which lacks or shows low levels of ER, PR, and HER2 proteins35,36, exhibited significantly increased LINK-A expression in comparison to HER2+, LumA, LumB, and normal-like subtypes (Fig. 1b).LINK-A is a ~1.5kb long intergenic non-protein coding RNA37, which was confirmed by our northern blot and RACE analyses in MDA-MB-231 cells (Supplementary Fig. 1a, b). Given that LINK-A has a predicted ORF of 139 amino acids, we performed in vitro translation assays, showing that neither sense nor antisense transcript of LINK-A encodes protein (Supplementary Fig. 1c). We next examined LINK-A expression in breast cancer tissue microarrays (clinicopathological parameters listed in Supplementary Table 1) using RNAScope® 2.0 HD assay. In both training and validation sets of tissue samples, the expression of LINK-A was significantly increased in TNBC tissues compared to normal breast tissues, ERPR−/HER2+, ERPR+/HER2-, and ERPR+/HER2+ subtypes (Fig. 1c, d), demonstrating the strong correlation of LINK-A expression with TNBC. Additionally, we examined the LINK-A expression level in Duke breast cancer cohort, finding that high levels of LINK-A correlated with unfavorable recurrence free survival for breast cancer patients (Fig. 1e). Consistently, LINK-A was highly expressed in TNBC cell lines compared to Estrogen Receptor (ER)- or HER2-positive breast cancer cell lines (Supplementary Fig. 1d).Next, we examined the subcellular localization of LINK-A, finding that LINK-A predominately resides in the cytoplasm or close to the cellular membrane, which was distinct from typical nuclear lncRNAs including BCAR416 and HOTAIR20 (Supplementary Fig. 1e–g). Cell fractionation analysis showed that >90% of LINK-A is localized within the cytosolic fraction compared to the nuclear enrichment of BCAR4 (Supplementary Fig. 1h, i). We hypothesized that LINK-A has important roles in the cytosol.Identification and characterization of LINK-A-protein interactionWe performed an RNA pulldown assay followed by Mass Spectrometry (MS)15,16 to identify LINK-A-associated proteins that might be involved in cytoplasmic processes. Interestingly, sense LINK-A but not antisense and beads control specifically associated with two transmembrane proteins, Epidermal Growth Factor Receptor (EGFR) and Transmembrane Glycoprotein NMB (GPNMB), tyrosine protein kinase 6 (also known as Breast Tumor Kinase, BRK)38,39, leucine-rich repeat kinase 2 (LRRK2)40,41, and HIF1α in the breast cancer cell (Fig. 1f, Supplementary Fig. 1j and Supplementary Table 2). RNA pulldown assay in cell lysate and RNA-protein binding assay using recombinant EGFR, BRK, LRRK2, HIF1α, and GPNMB confirmed that LINK-A associated with all of the proteins mentioned above in vivo, but only BRK and LRRK2 directly interacted with LINK-A (Supplementary Fig. 1k–n). The specific interaction between LINK-A and BRK or LRRK2 was also confirmed by an RNA immunoprecipitation (RIP) assay (Fig. 1g).To map the BRK domains required for LINK-A binding, we generated BRK SH3 (a.a. 11–72), SH2 (a.a. 78–170), kinase domains (a.a. 191–445), and regulatory carboxyl-terminus (a.a. 446–451) deletion mutants (Fig. 1h, bottom panel). Deletion of either the SH3 domain or the carboxyl-terminus region of the kinase domain of BRK alone impaired the interaction between LINK-A and BRK, suggesting that LINK-A interacts with two separate domains of BRK (Supplementary Fig. 1o). Double deletion of these two domains abolished LINK-A-BRK interaction in vitro and in vivo (Fig. 1h and Supplementary Fig. 1p). A similar strategy was utilized to map the domain required for LRRK2-LINK-A interaction, showing that deletion of the WD40 domain, an atypical RNA binding domain16,25, abolished the direct interaction (Fig. 1i and Supplementary Fig. 1q).To map RNA motifs essential for the LINK-A-protein interactions, we conducted an in vitro RNA-protein binding coupled with dot-blot assay15,16, finding that BRK interacted with LINK-A at two regions nt. 481–540 (dot B3) and nt. 781–840 (dot C2) (corresponding to the two domains of BRK at the SH3 domain and the C-terminal tail) (Fig. 1j). LINK-A nt. 1261–1320 (dot D4) interacted with LRRK2 (Fig. 1j). Consistently, double deletion of LINK-A (nt.471–550 and nt. 771–850) abolished the BRK-LINK-A interaction without affecting the LRRK2-LINK-A interaction, while deletion of LINK-A (nt. 1251–1330) specifically abolished LRRK2-LINK-A association (Fig. 1k). The predicted secondary structure of LINK-A indicates that the RNA motifs required for BRK and LRRK2 interactions form individual branching stem-loops, suggesting their contribution to specific RNA-protein interactions (Supplementary Fig. 1r).Characterization of a HB-EGF-triggered, EGFR:GPNMB-dependent and LINK-A-mediated signaling pathway in TNBCOur MS data revealed a series of phosphorylation sites of GPNMB (Tyr525), BRK (Tyr351), and HIF1α (Tyr565 and Ser797) (Fig. 2a, Supplementary Fig. 2a–d and Supplementary Table 2), leading us to generate phosphorylation site-specific antibodies (Supplementary Fig. 2e–i) for investigating whether LINK-A modulates a previously unknown signaling pathway.Given that LINK-A associated with orphan receptor GPNMB and EGFR, which are involved in metastatic TNBC42–44, we reasoned that EGFR and GPNMB may interact with each other in TNBC cells upon EGF family ligands. While all EGFR ligands effectively activated EGFR (Supplementary Fig. 2j), HB-EGF robustly induced the specific interaction between EGFR and GPNMB (Fig. 2b), indicating that EGF ligands could differentially trigger the formation of the EGFR homodimer or the heterodimer between EGFR and other receptors45. To test this, we performed a cross-linking assay, finding that EGF predominately triggered EGFR homodimerization with a lesser degree of EGFR:GPNMB heterodimerization while HB-EGF stimulated EGFR:GPNMB heterodimerization with less EGFR homodimerization (Fig. 2c). Knockdown of LINK-A exhibited minimal effects on the HB-EGF-induced EGFR:GPNMB interaction as well as GPNMB phosphorylation upon ligand stimulation (Supplementary Fig. 2k, l), suggesting that HB-EGF preferentially triggered EGFR:GPNMB heterodimer formation. We further mapped the domains mediating EGFR-GPNMB binding, and found that the kinase domain (KD) in EGFR Intracellular Domains (ICD) interacts with GPNMB ICD (Fig. 2d, e and Supplementary Fig. 2m). HB-EGF robustly induced site-specific phosphorylations of EGFR, GPNMB, BRK, and HIF1α (Fig. 2f) and pretreatment of TNBC cell lines with Cetuximab impaired EGFR-GPNMB interaction (Supplementary Fig. 2n, o). These observations led us to fully characterize this HB-EGF triggered, EGFR:GPNMB-dependent signaling pathway in TNBC.First, an in vitro kinase assay indicated that EGFR, but not BRK, phosphorylated GPNMB at Tyr525 (Fig. 2g) and the exogenously expressed wild-type GPNMB but not Y525F mutant was phosphorylated in vivo upon HB-EGF stimulation (Fig. 2h). Next, we observed the interaction between GPNMB and BRK following ligand stimulation, which was abolished in the presence of GPNMB Y525F mutant (Fig. 2h). Furthermore, the ligand-triggered BRK Tyr351 phosphorylation was abolished in GPNMB Y525F-overexpressing cells (Fig. 2h). Biochemical experiments showed that BRK SH2 domain deletion (a.a 78–170) eliminated the ligand-dependent interaction with Tyr525-phosphorylated GPNMB (Fig. 2i). These data suggest that the EGFR-dependent GPNMB Tyr525 phosphorylation is required for further recruitment of BRK via its SH2 domain and subsequent phosphorylation at Tyr351.LINK-A facilitates the recruitment and activation of BRKWe then conducted an Immuno-RNA FISH assay to examine the proximity of LINK-A to the ligand-bound receptors upon ligand treatment, finding the overlapping between LINK-A and EGFR upon HB-EGF stimulation (Supplementary Fig. 3a), which was further validated by in vivo RIP assay (Supplementary Fig. 3b). We examined the co-localization of BRK and EGFR:GPNMB receptor complex in the presence or absence of LINK-A. Our data indicate that both BRK and phospho-BRK (Tyr351) faithfully co-localized with EGFR upon HB-EGF stimulation (Fig. 3a and Supplementary Fig. 3c). In contrast, depletion of LINK-A abolished the recruitment of BRK to EGFR and subsequent phosphorylation of BRK without affecting the internalization of EGFR (Fig. 3a and Supplementary Fig. 3c). we then performed rescue experiments in which LINK-A was knocked down by Locked Nucleic Acids (LNA) followed by re-introduction of LNA-resistant full-length LINK-A or one of the following deletion mutants: ΔBRK (Δ471–550 and Δ771–850) or ΔLRRK2 (Δ1251–1330) (Fig. 3b, lower panel, Supplementary Fig. 3d, e), finding that knockdown of LINK-A abolished the HB-EGF-induced BRK-GPNMB interaction, as well as BRK Tyr351phosphorylation (Fig. 3c, d); re-introduction of full-length LINK-A or ΔLRRK2 but not ΔBRK mutant rescued these phenotypes (Fig. 3c, d). These data suggest that LINK-A-BRK interaction facilitate the recruitment of BRK to the tyrosine phosphorylated membrane receptor GPNMB, as well as subsequent autophosphorylation of BRK.LINK-A elicits the conformational change of BRK for kinase activationIt has been reported that the activity of BRK is auto-inhibited by interaction between the SH2 domain and the Tyr447-phosphorylated C-terminal domain46–48. Our data indicate that LINK-A interacts with BRK at two regions, SH3 domain and the C-terminal domain (see Fig. 1h and Supplementary Fig. 1p), raising a possible role of LINK-A in eliciting a BRK conformational change that mitigates the conformation required for self-inhibition. Indeed, we found that full-length LINK-A and ΔLRRK2 LINK-A dramatically enhanced the autophosphorylation and kinase activity of BRK while both the control lncRNA and ΔBRK LINK-A showed minimal effects (Fig. 3e, f).We next conducted a protease digestion assay by incubating BRK with Caspase-1 in the presence of full-length LINK-A or ΔBRK LINK-A, finding that Caspase-1 barely cleaved BRK at aa. 397 in the presence of an unrelated lncRNA RP11-383G10.5, but robustly cleaved BRK only in the presence of full-length LINK-A (Fig. 3g), suggesting a potential conformational change occurred in BRK to expose the digestion site upon LINK-A binding. Notably, depletion of either two regions of LINK-A involved in BRK interaction failed to promote the Caspase-1 dependent BRK cleavage (Fig. 3g), suggesting that simultaneous binding of LINK-A to two BRK domains is required to elicit the conformational change in BRK. Our data suggest that the binding of LINK-A to BRK promotes a conformational change, leading to increased accessibility of the SH2 domain and the autophosphorylation sites in the kinase domain. Upon ligand stimulation, these events lead to the recruitment of BRK to Tyr525-phosphorylated GPNMB and activation of BRK upon Tyr351 phosphorylation.LINK-A-interacting BRK and LRRK2 phosphorylate HIF1αWe next performed in vitro phosphorylation assays, finding that activated BRK phosphorylated HIF1α at Tyr565 (Fig. 4a) and LRRK2, another LINK-A interacting protein kinase, phosphorylated HIF1α at Ser797, which was further demonstrated by the marked inhibition of HIF1α phosphorylation in the presence of a S797A point mutant (Fig. 4a). The BRK kinase activity deficient mutant, Y351F, diminished the phosphorylation of HIF1α in vivo (Fig. 4b). Both Tyr565 and Ser797 of HIF1α are conserved (Supplementary Fig. 4a).HB-EGF induced phosphorylation of GPNMB (Tyr525) and BRK (Tyr351), as well as HIF1α protein stabilization under normoxic conditions (Fig. 4c and Supplementary Fig. 4b). Interestingly, knockdown of EGFR abolished the ligand-dependent phosphorylation of GPNMB (Tyr525) and BRK (Tyr351), as well as the stabilization of HIF1α; knockdown of GPNMB abolished HB-EGF induced BRK phosphorylation and HIF1α protein stabilization, but did not affect EGFR phosphorylation (Tyr1068) (Fig. 4d). Knockdown of LINK-A in both MDA-MB-231 and MDA-MB-468 cells eliminated HB-EGF induced BRK phosphorylation and HIF1α stabilization, but not phosphorylation of EGFR or GPNMB (Fig. 4e, f). In contrast, LINK-A knockdown exhibited minimal effects on hypoxia-dependent HIF1α stabilization, and hypoxia failed to trigger phosphorylation of GPNMB and BRK (Fig. 4c–f and Supplementary Fig. 4c). Finally, depletion of BRK decreased ligand-triggered HIF1α protein accumulation but did not affect the phosphorylation status of EGFR or GPNMB (Fig. 4d). Taken together, these data suggest a linear EGFR:GPNMB→LINK-A→BRK/LRRK2→HIF1α signaling cascade upon HB-EGF stimulation under normoxic conditions.Upon HB-EGF stimulation, HIF1α underwent Tyr565 and Ser797 phosphorylations but the hydroxylation at Pro564 was inhibited, which led to HIF1α stabilization (Fig. 4g). Knockdown of LINK-A abolished HB-EGF induced HIF1α Tyr565-phosphorylation and enhanced the Pro564-hydroxylation (Fig.4h and Supplementary Fig. 4d). A similar pattern was observed with EGFR, GPNMB and BRK knockdown (Fig.4i). These data suggest that HB-EGF triggers an lncRNA-dependent signaling pathway to stabilize the HIF1α at protein level.Tyr565 phosphorylation antagonizes Pro564 hydroxylation to stabilize HIF1α under normoxiaAn in vitro hydroxylation assay demonstrated that the HIF1α peptides (aa.557–566) but not Tyr565-phosphorylated peptides can be hydroxylated by PHD1 (Fig. 4j, Supplementary Fig. 4e–j and Supplementary Table 3). An in vitro kinase assay followed by in vitro hydroxylation assay further showed that phosphorylation of wild-type HIF1α but not Y565F mutant by BRK prevented subsequent hydroxylation at Pro564 (Fig. 4k). Consistently, HB-EGF triggered Tyr565 phosphorylation of HIF1α and inhibition of hydroxylation at Pro564, which was abolished by overexpression of the Y565F mutant of HIF1α (Fig. 4l).Cycloheximide (CHX) treatment experiment revealed that upon HB-EGF stimulation, HIF1α protein exhibited ≥4 hours half-life while knocking down of LINK-A reduced it to 1.5 hours (Supplementary Fig. 4k, l). In TNBC cells exogenously expressing wild-type HIF1α or Y565D mutant, Y565D mutant exhibited constitutively prolonged half-life (Supplementary Fig. 4m–o). These data indicate that LINK-A-associated BRK phosphorylated HIF1α at Tyr565, which prevents HIF1α hydroxylation at adjacent Pro564 and stabilizes HIF1α under normoxia.LINK-A-recruited LRRK2 phosphorylates Ser797 of HIF1α to potentiate its transcriptional activityKnockdown of LINK-A, LRRK2, or overexpression of the HIF1α S797A mutant abolished Ser797 phosphorylation of HIF1α as well as its association with p300, which was concurrent with the release of FIH49, a protein that binds to HIF1α and inhibits its transactivation function (Fig. 5a, b). We also examined the kinase activity of LRRK2 in the presence of LINK-A, finding that full-length LINK-A, ΔBRK LINK-A or ΔLRRK2 LINK-A exhibited minimal effect on the kinase activity of LRRK2 (Supplementary Fig. 5a). The rescue experiments indicated that full-length LINK-A fully rescued HIF1α phosphorylations and protein stabilization; ΔBRK LINK-A only rescued HIF1α Ser797 phosphorylation and ΔLRRK2 LINK-A restored HIF1α Tyr565 phosphorylation and protein stabilization, but failed to rescue the phosphorylation of HIF1α at Ser797 (Fig. 5c). Recent studies have shown that certain lncRNAs encode small protein peptides50–52. While our data have demonstrated that a predicted ORF of LINK-A has no protein-coding products in vitro (see Supplementary Fig. 1a–c), we further mutated the predicted translational start codon ATG (nt. 318–321), or the potential stop codon TGA (nt. 732–735) of this ORF in a functional rescue experiments, finding that the phosphorylation of BRK (Tyr351) and HIF1α (Tyr565), two major cellular effects mediated by LINK-A, were fully rescued by wild-type LINK-A as well as ATG→TAG or TGA→TGT mutants of LINK-A (Supplementary Fig. 5b–d). These observations suggested that the cellular effect of LINK-A is mainly dependent on its RNA function instead of the potential translational products. Taken together, we demonstrated that LINK-A, in coordination with two protein kinases BRK and LRRK2, mediated a growth factor-triggered signaling cascade to synergistically regulate the phosphorylation and protein stabilization of HIF1α under normoxia.LINK-A-dependent normoxic HIF1α signaling promotes tumor growth and correlates with TNBCNext, we examined the transcriptional activity of HIF1α upon HB-EGF stimulation by ChIP-seq, finding that under normoxia, HB-EGF triggered the recruitment of HIF1α to the promoters of HIF1α target genes and regulated the HIF1α-dependent transcriptional program (Fig. 5d, e and Supplementary Table 4). Knockdown of LINK-A in TNBC cells impaired HIF1α-target genes expression upon HB-EGF stimulation (Fig. 5f, g and Supplementary Fig. 5e). Consistently, in vitro glucose uptake and lactate production assays confirmed that LINK-A deficiency impaired glycolysis (Supplementary Fig. 5f–l). Consistent with the in vitro colony formation assays (Fig. 5h), the mice with xenografts of LINK-A-depleted tumor cells rarely developed tumor mass in vivo (Fig. 5i, j and Supplementary Fig. 5m).LINK-A-mediated signaling pathway was also activated in TNBC tissues, as evidenced by significantly higher staining density of phospho-GPNMB (Tyr525), phospho-BRK (Tyr351), phospho-HIF1α (Tyr565), and phospho-HIF1α (Ser797) in TNBC samples compared to non-TNBC samples (Fig. 6a–c and Supplementary Fig. 6a). Furthermore, within the TNBC category, breast cancer with advanced lymph-node metastasis showed increased phospho-BRK (Tyr351), phospho-HIF1α (Tyr565), and phospho-GPNMB (Tyr525) levels compared to tissue samples with no lymph-node metastasis (Fig. 6a–c and Fig. 6d–f, upper panel). Importantly, there is a strong correlation between LINK-A expression and the phosphorylation status of BRK, HIF1α, and GPNMB in these TNBC tissues (Fig. 6d–f, lower panel), and breast cancer patients with higher levels of these phosphoproteins exhibited shorter survival time (Fig. 6g–i). Furthermore, the TCGA database revealed that both BRK and LRRK2 are highly expressed in invasive breast carcinoma (Supplementary Fig. 6b). Our data implicate LINK-A and its associated signaling pathway as potential biomarkers and therapeutic targets for TNBC.

LINK-A is a cytoplasmic lncRNA with prognostic value for TNBCTo identify TNBC-relevant lncRNAs, we examined lncRNA expression profile in two stage III TNBC tissues and their paired adjacent noncancerous tissues, finding 21 differentially expressed lncRNAs16. We further searched the expression pattern of these 21 lncRNAs in the TCGA database. Interestingly, statistical analysis of a combined 711 RNA-seq transcriptome profiles indicated that the expression of LINK-A is frequently elevated in TNBC patient cohorts in comparison to cohorts of ERPR−/HER2+, ERPR+/HER2-, and ERPR+/HER2+ patients. Differential LINK-A expression between ERPR−/HER2+, ERPR+/HER2-, and ERPR+/HER2+ cohorts were not statistically significant (Fig. 1a). Consistently, basal-like breast cancer, which lacks or shows low levels of ER, PR, and HER2 proteins35,36, exhibited significantly increased LINK-A expression in comparison to HER2+, LumA, LumB, and normal-like subtypes (Fig. 1b).LINK-A is a ~1.5kb long intergenic non-protein coding RNA37, which was confirmed by our northern blot and RACE analyses in MDA-MB-231 cells (Supplementary Fig. 1a, b). Given that LINK-A has a predicted ORF of 139 amino acids, we performed in vitro translation assays, showing that neither sense nor antisense transcript of LINK-A encodes protein (Supplementary Fig. 1c). We next examined LINK-A expression in breast cancer tissue microarrays (clinicopathological parameters listed in Supplementary Table 1) using RNAScope® 2.0 HD assay. In both training and validation sets of tissue samples, the expression of LINK-A was significantly increased in TNBC tissues compared to normal breast tissues, ERPR−/HER2+, ERPR+/HER2-, and ERPR+/HER2+ subtypes (Fig. 1c, d), demonstrating the strong correlation of LINK-A expression with TNBC. Additionally, we examined the LINK-A expression level in Duke breast cancer cohort, finding that high levels of LINK-A correlated with unfavorable recurrence free survival for breast cancer patients (Fig. 1e). Consistently, LINK-A was highly expressed in TNBC cell lines compared to Estrogen Receptor (ER)- or HER2-positive breast cancer cell lines (Supplementary Fig. 1d).Next, we examined the subcellular localization of LINK-A, finding that LINK-A predominately resides in the cytoplasm or close to the cellular membrane, which was distinct from typical nuclear lncRNAs including BCAR416 and HOTAIR20 (Supplementary Fig. 1e–g). Cell fractionation analysis showed that >90% of LINK-A is localized within the cytosolic fraction compared to the nuclear enrichment of BCAR4 (Supplementary Fig. 1h, i). We hypothesized that LINK-A has important roles in the cytosol.

LINK-A is a cytoplasmic lncRNA with prognostic value for TNBC

To identify TNBC-relevant lncRNAs, we examined lncRNA expression profile in two stage III TNBC tissues and their paired adjacent noncancerous tissues, finding 21 differentially expressed lncRNAs16. We further searched the expression pattern of these 21 lncRNAs in the TCGA database. Interestingly, statistical analysis of a combined 711 RNA-seq transcriptome profiles indicated that the expression of LINK-A is frequently elevated in TNBC patient cohorts in comparison to cohorts of ERPR−/HER2+, ERPR+/HER2-, and ERPR+/HER2+ patients. Differential LINK-A expression between ERPR−/HER2+, ERPR+/HER2-, and ERPR+/HER2+ cohorts were not statistically significant (Fig. 1a). Consistently, basal-like breast cancer, which lacks or shows low levels of ER, PR, and HER2 proteins35,36, exhibited significantly increased LINK-A expression in comparison to HER2+, LumA, LumB, and normal-like subtypes (Fig. 1b).

LINK-A is a ~1.5kb long intergenic non-protein coding RNA37, which was confirmed by our northern blot and RACE analyses in MDA-MB-231 cells (Supplementary Fig. 1a, b). Given that LINK-A has a predicted ORF of 139 amino acids, we performed in vitro translation assays, showing that neither sense nor antisense transcript of LINK-A encodes protein (Supplementary Fig. 1c). We next examined LINK-A expression in breast cancer tissue microarrays (clinicopathological parameters listed in Supplementary Table 1) using RNAScope® 2.0 HD assay. In both training and validation sets of tissue samples, the expression of LINK-A was significantly increased in TNBC tissues compared to normal breast tissues, ERPR−/HER2+, ERPR+/HER2-, and ERPR+/HER2+ subtypes (Fig. 1c, d), demonstrating the strong correlation of LINK-A expression with TNBC. Additionally, we examined the LINK-A expression level in Duke breast cancer cohort, finding that high levels of LINK-A correlated with unfavorable recurrence free survival for breast cancer patients (Fig. 1e). Consistently, LINK-A was highly expressed in TNBC cell lines compared to Estrogen Receptor (ER)- or HER2-positive breast cancer cell lines (Supplementary Fig. 1d).

Next, we examined the subcellular localization of LINK-A, finding that LINK-A predominately resides in the cytoplasm or close to the cellular membrane, which was distinct from typical nuclear lncRNAs including BCAR416 and HOTAIR20 (Supplementary Fig. 1e–g). Cell fractionation analysis showed that >90% of LINK-A is localized within the cytosolic fraction compared to the nuclear enrichment of BCAR4 (Supplementary Fig. 1h, i). We hypothesized that LINK-A has important roles in the cytosol.

Identification and characterization of LINK-A-protein interactionWe performed an RNA pulldown assay followed by Mass Spectrometry (MS)15,16 to identify LINK-A-associated proteins that might be involved in cytoplasmic processes. Interestingly, sense LINK-A but not antisense and beads control specifically associated with two transmembrane proteins, Epidermal Growth Factor Receptor (EGFR) and Transmembrane Glycoprotein NMB (GPNMB), tyrosine protein kinase 6 (also known as Breast Tumor Kinase, BRK)38,39, leucine-rich repeat kinase 2 (LRRK2)40,41, and HIF1α in the breast cancer cell (Fig. 1f, Supplementary Fig. 1j and Supplementary Table 2). RNA pulldown assay in cell lysate and RNA-protein binding assay using recombinant EGFR, BRK, LRRK2, HIF1α, and GPNMB confirmed that LINK-A associated with all of the proteins mentioned above in vivo, but only BRK and LRRK2 directly interacted with LINK-A (Supplementary Fig. 1k–n). The specific interaction between LINK-A and BRK or LRRK2 was also confirmed by an RNA immunoprecipitation (RIP) assay (Fig. 1g).To map the BRK domains required for LINK-A binding, we generated BRK SH3 (a.a. 11–72), SH2 (a.a. 78–170), kinase domains (a.a. 191–445), and regulatory carboxyl-terminus (a.a. 446–451) deletion mutants (Fig. 1h, bottom panel). Deletion of either the SH3 domain or the carboxyl-terminus region of the kinase domain of BRK alone impaired the interaction between LINK-A and BRK, suggesting that LINK-A interacts with two separate domains of BRK (Supplementary Fig. 1o). Double deletion of these two domains abolished LINK-A-BRK interaction in vitro and in vivo (Fig. 1h and Supplementary Fig. 1p). A similar strategy was utilized to map the domain required for LRRK2-LINK-A interaction, showing that deletion of the WD40 domain, an atypical RNA binding domain16,25, abolished the direct interaction (Fig. 1i and Supplementary Fig. 1q).To map RNA motifs essential for the LINK-A-protein interactions, we conducted an in vitro RNA-protein binding coupled with dot-blot assay15,16, finding that BRK interacted with LINK-A at two regions nt. 481–540 (dot B3) and nt. 781–840 (dot C2) (corresponding to the two domains of BRK at the SH3 domain and the C-terminal tail) (Fig. 1j). LINK-A nt. 1261–1320 (dot D4) interacted with LRRK2 (Fig. 1j). Consistently, double deletion of LINK-A (nt.471–550 and nt. 771–850) abolished the BRK-LINK-A interaction without affecting the LRRK2-LINK-A interaction, while deletion of LINK-A (nt. 1251–1330) specifically abolished LRRK2-LINK-A association (Fig. 1k). The predicted secondary structure of LINK-A indicates that the RNA motifs required for BRK and LRRK2 interactions form individual branching stem-loops, suggesting their contribution to specific RNA-protein interactions (Supplementary Fig. 1r).

Identification and characterization of LINK-A-protein interaction

We performed an RNA pulldown assay followed by Mass Spectrometry (MS)15,16 to identify LINK-A-associated proteins that might be involved in cytoplasmic processes. Interestingly, sense LINK-A but not antisense and beads control specifically associated with two transmembrane proteins, Epidermal Growth Factor Receptor (EGFR) and Transmembrane Glycoprotein NMB (GPNMB), tyrosine protein kinase 6 (also known as Breast Tumor Kinase, BRK)38,39, leucine-rich repeat kinase 2 (LRRK2)40,41, and HIF1α in the breast cancer cell (Fig. 1f, Supplementary Fig. 1j and Supplementary Table 2). RNA pulldown assay in cell lysate and RNA-protein binding assay using recombinant EGFR, BRK, LRRK2, HIF1α, and GPNMB confirmed that LINK-A associated with all of the proteins mentioned above in vivo, but only BRK and LRRK2 directly interacted with LINK-A (Supplementary Fig. 1k–n). The specific interaction between LINK-A and BRK or LRRK2 was also confirmed by an RNA immunoprecipitation (RIP) assay (Fig. 1g).

To map the BRK domains required for LINK-A binding, we generated BRK SH3 (a.a. 11–72), SH2 (a.a. 78–170), kinase domains (a.a. 191–445), and regulatory carboxyl-terminus (a.a. 446–451) deletion mutants (Fig. 1h, bottom panel). Deletion of either the SH3 domain or the carboxyl-terminus region of the kinase domain of BRK alone impaired the interaction between LINK-A and BRK, suggesting that LINK-A interacts with two separate domains of BRK (Supplementary Fig. 1o). Double deletion of these two domains abolished LINK-A-BRK interaction in vitro and in vivo (Fig. 1h and Supplementary Fig. 1p). A similar strategy was utilized to map the domain required for LRRK2-LINK-A interaction, showing that deletion of the WD40 domain, an atypical RNA binding domain16,25, abolished the direct interaction (Fig. 1i and Supplementary Fig. 1q).

To map RNA motifs essential for the LINK-A-protein interactions, we conducted an in vitro RNA-protein binding coupled with dot-blot assay15,16, finding that BRK interacted with LINK-A at two regions nt. 481–540 (dot B3) and nt. 781–840 (dot C2) (corresponding to the two domains of BRK at the SH3 domain and the C-terminal tail) (Fig. 1j). LINK-A nt. 1261–1320 (dot D4) interacted with LRRK2 (Fig. 1j). Consistently, double deletion of LINK-A (nt.471–550 and nt. 771–850) abolished the BRK-LINK-A interaction without affecting the LRRK2-LINK-A interaction, while deletion of LINK-A (nt. 1251–1330) specifically abolished LRRK2-LINK-A association (Fig. 1k). The predicted secondary structure of LINK-A indicates that the RNA motifs required for BRK and LRRK2 interactions form individual branching stem-loops, suggesting their contribution to specific RNA-protein interactions (Supplementary Fig. 1r).

Characterization of a HB-EGF-triggered, EGFR:GPNMB-dependent and LINK-A-mediated signaling pathway in TNBCOur MS data revealed a series of phosphorylation sites of GPNMB (Tyr525), BRK (Tyr351), and HIF1α (Tyr565 and Ser797) (Fig. 2a, Supplementary Fig. 2a–d and Supplementary Table 2), leading us to generate phosphorylation site-specific antibodies (Supplementary Fig. 2e–i) for investigating whether LINK-A modulates a previously unknown signaling pathway.Given that LINK-A associated with orphan receptor GPNMB and EGFR, which are involved in metastatic TNBC42–44, we reasoned that EGFR and GPNMB may interact with each other in TNBC cells upon EGF family ligands. While all EGFR ligands effectively activated EGFR (Supplementary Fig. 2j), HB-EGF robustly induced the specific interaction between EGFR and GPNMB (Fig. 2b), indicating that EGF ligands could differentially trigger the formation of the EGFR homodimer or the heterodimer between EGFR and other receptors45. To test this, we performed a cross-linking assay, finding that EGF predominately triggered EGFR homodimerization with a lesser degree of EGFR:GPNMB heterodimerization while HB-EGF stimulated EGFR:GPNMB heterodimerization with less EGFR homodimerization (Fig. 2c). Knockdown of LINK-A exhibited minimal effects on the HB-EGF-induced EGFR:GPNMB interaction as well as GPNMB phosphorylation upon ligand stimulation (Supplementary Fig. 2k, l), suggesting that HB-EGF preferentially triggered EGFR:GPNMB heterodimer formation. We further mapped the domains mediating EGFR-GPNMB binding, and found that the kinase domain (KD) in EGFR Intracellular Domains (ICD) interacts with GPNMB ICD (Fig. 2d, e and Supplementary Fig. 2m). HB-EGF robustly induced site-specific phosphorylations of EGFR, GPNMB, BRK, and HIF1α (Fig. 2f) and pretreatment of TNBC cell lines with Cetuximab impaired EGFR-GPNMB interaction (Supplementary Fig. 2n, o). These observations led us to fully characterize this HB-EGF triggered, EGFR:GPNMB-dependent signaling pathway in TNBC.First, an in vitro kinase assay indicated that EGFR, but not BRK, phosphorylated GPNMB at Tyr525 (Fig. 2g) and the exogenously expressed wild-type GPNMB but not Y525F mutant was phosphorylated in vivo upon HB-EGF stimulation (Fig. 2h). Next, we observed the interaction between GPNMB and BRK following ligand stimulation, which was abolished in the presence of GPNMB Y525F mutant (Fig. 2h). Furthermore, the ligand-triggered BRK Tyr351 phosphorylation was abolished in GPNMB Y525F-overexpressing cells (Fig. 2h). Biochemical experiments showed that BRK SH2 domain deletion (a.a 78–170) eliminated the ligand-dependent interaction with Tyr525-phosphorylated GPNMB (Fig. 2i). These data suggest that the EGFR-dependent GPNMB Tyr525 phosphorylation is required for further recruitment of BRK via its SH2 domain and subsequent phosphorylation at Tyr351.

Characterization of a HB-EGF-triggered, EGFR:GPNMB-dependent and LINK-A-mediated signaling pathway in TNBC

Our MS data revealed a series of phosphorylation sites of GPNMB (Tyr525), BRK (Tyr351), and HIF1α (Tyr565 and Ser797) (Fig. 2a, Supplementary Fig. 2a–d and Supplementary Table 2), leading us to generate phosphorylation site-specific antibodies (Supplementary Fig. 2e–i) for investigating whether LINK-A modulates a previously unknown signaling pathway.

Given that LINK-A associated with orphan receptor GPNMB and EGFR, which are involved in metastatic TNBC42–44, we reasoned that EGFR and GPNMB may interact with each other in TNBC cells upon EGF family ligands. While all EGFR ligands effectively activated EGFR (Supplementary Fig. 2j), HB-EGF robustly induced the specific interaction between EGFR and GPNMB (Fig. 2b), indicating that EGF ligands could differentially trigger the formation of the EGFR homodimer or the heterodimer between EGFR and other receptors45. To test this, we performed a cross-linking assay, finding that EGF predominately triggered EGFR homodimerization with a lesser degree of EGFR:GPNMB heterodimerization while HB-EGF stimulated EGFR:GPNMB heterodimerization with less EGFR homodimerization (Fig. 2c). Knockdown of LINK-A exhibited minimal effects on the HB-EGF-induced EGFR:GPNMB interaction as well as GPNMB phosphorylation upon ligand stimulation (Supplementary Fig. 2k, l), suggesting that HB-EGF preferentially triggered EGFR:GPNMB heterodimer formation. We further mapped the domains mediating EGFR-GPNMB binding, and found that the kinase domain (KD) in EGFR Intracellular Domains (ICD) interacts with GPNMB ICD (Fig. 2d, e and Supplementary Fig. 2m). HB-EGF robustly induced site-specific phosphorylations of EGFR, GPNMB, BRK, and HIF1α (Fig. 2f) and pretreatment of TNBC cell lines with Cetuximab impaired EGFR-GPNMB interaction (Supplementary Fig. 2n, o). These observations led us to fully characterize this HB-EGF triggered, EGFR:GPNMB-dependent signaling pathway in TNBC.

First, an in vitro kinase assay indicated that EGFR, but not BRK, phosphorylated GPNMB at Tyr525 (Fig. 2g) and the exogenously expressed wild-type GPNMB but not Y525F mutant was phosphorylated in vivo upon HB-EGF stimulation (Fig. 2h). Next, we observed the interaction between GPNMB and BRK following ligand stimulation, which was abolished in the presence of GPNMB Y525F mutant (Fig. 2h). Furthermore, the ligand-triggered BRK Tyr351 phosphorylation was abolished in GPNMB Y525F-overexpressing cells (Fig. 2h). Biochemical experiments showed that BRK SH2 domain deletion (a.a 78–170) eliminated the ligand-dependent interaction with Tyr525-phosphorylated GPNMB (Fig. 2i). These data suggest that the EGFR-dependent GPNMB Tyr525 phosphorylation is required for further recruitment of BRK via its SH2 domain and subsequent phosphorylation at Tyr351.

LINK-A facilitates the recruitment and activation of BRKWe then conducted an Immuno-RNA FISH assay to examine the proximity of LINK-A to the ligand-bound receptors upon ligand treatment, finding the overlapping between LINK-A and EGFR upon HB-EGF stimulation (Supplementary Fig. 3a), which was further validated by in vivo RIP assay (Supplementary Fig. 3b). We examined the co-localization of BRK and EGFR:GPNMB receptor complex in the presence or absence of LINK-A. Our data indicate that both BRK and phospho-BRK (Tyr351) faithfully co-localized with EGFR upon HB-EGF stimulation (Fig. 3a and Supplementary Fig. 3c). In contrast, depletion of LINK-A abolished the recruitment of BRK to EGFR and subsequent phosphorylation of BRK without affecting the internalization of EGFR (Fig. 3a and Supplementary Fig. 3c). we then performed rescue experiments in which LINK-A was knocked down by Locked Nucleic Acids (LNA) followed by re-introduction of LNA-resistant full-length LINK-A or one of the following deletion mutants: ΔBRK (Δ471–550 and Δ771–850) or ΔLRRK2 (Δ1251–1330) (Fig. 3b, lower panel, Supplementary Fig. 3d, e), finding that knockdown of LINK-A abolished the HB-EGF-induced BRK-GPNMB interaction, as well as BRK Tyr351phosphorylation (Fig. 3c, d); re-introduction of full-length LINK-A or ΔLRRK2 but not ΔBRK mutant rescued these phenotypes (Fig. 3c, d). These data suggest that LINK-A-BRK interaction facilitate the recruitment of BRK to the tyrosine phosphorylated membrane receptor GPNMB, as well as subsequent autophosphorylation of BRK.

LINK-A facilitates the recruitment and activation of BRK

We then conducted an Immuno-RNA FISH assay to examine the proximity of LINK-A to the ligand-bound receptors upon ligand treatment, finding the overlapping between LINK-A and EGFR upon HB-EGF stimulation (Supplementary Fig. 3a), which was further validated by in vivo RIP assay (Supplementary Fig. 3b). We examined the co-localization of BRK and EGFR:GPNMB receptor complex in the presence or absence of LINK-A. Our data indicate that both BRK and phospho-BRK (Tyr351) faithfully co-localized with EGFR upon HB-EGF stimulation (Fig. 3a and Supplementary Fig. 3c). In contrast, depletion of LINK-A abolished the recruitment of BRK to EGFR and subsequent phosphorylation of BRK without affecting the internalization of EGFR (Fig. 3a and Supplementary Fig. 3c). we then performed rescue experiments in which LINK-A was knocked down by Locked Nucleic Acids (LNA) followed by re-introduction of LNA-resistant full-length LINK-A or one of the following deletion mutants: ΔBRK (Δ471–550 and Δ771–850) or ΔLRRK2 (Δ1251–1330) (Fig. 3b, lower panel, Supplementary Fig. 3d, e), finding that knockdown of LINK-A abolished the HB-EGF-induced BRK-GPNMB interaction, as well as BRK Tyr351phosphorylation (Fig. 3c, d); re-introduction of full-length LINK-A or ΔLRRK2 but not ΔBRK mutant rescued these phenotypes (Fig. 3c, d). These data suggest that LINK-A-BRK interaction facilitate the recruitment of BRK to the tyrosine phosphorylated membrane receptor GPNMB, as well as subsequent autophosphorylation of BRK.

LINK-A elicits the conformational change of BRK for kinase activationIt has been reported that the activity of BRK is auto-inhibited by interaction between the SH2 domain and the Tyr447-phosphorylated C-terminal domain46–48. Our data indicate that LINK-A interacts with BRK at two regions, SH3 domain and the C-terminal domain (see Fig. 1h and Supplementary Fig. 1p), raising a possible role of LINK-A in eliciting a BRK conformational change that mitigates the conformation required for self-inhibition. Indeed, we found that full-length LINK-A and ΔLRRK2 LINK-A dramatically enhanced the autophosphorylation and kinase activity of BRK while both the control lncRNA and ΔBRK LINK-A showed minimal effects (Fig. 3e, f).We next conducted a protease digestion assay by incubating BRK with Caspase-1 in the presence of full-length LINK-A or ΔBRK LINK-A, finding that Caspase-1 barely cleaved BRK at aa. 397 in the presence of an unrelated lncRNA RP11-383G10.5, but robustly cleaved BRK only in the presence of full-length LINK-A (Fig. 3g), suggesting a potential conformational change occurred in BRK to expose the digestion site upon LINK-A binding. Notably, depletion of either two regions of LINK-A involved in BRK interaction failed to promote the Caspase-1 dependent BRK cleavage (Fig. 3g), suggesting that simultaneous binding of LINK-A to two BRK domains is required to elicit the conformational change in BRK. Our data suggest that the binding of LINK-A to BRK promotes a conformational change, leading to increased accessibility of the SH2 domain and the autophosphorylation sites in the kinase domain. Upon ligand stimulation, these events lead to the recruitment of BRK to Tyr525-phosphorylated GPNMB and activation of BRK upon Tyr351 phosphorylation.

LINK-A elicits the conformational change of BRK for kinase activation

It has been reported that the activity of BRK is auto-inhibited by interaction between the SH2 domain and the Tyr447-phosphorylated C-terminal domain46–48. Our data indicate that LINK-A interacts with BRK at two regions, SH3 domain and the C-terminal domain (see Fig. 1h and Supplementary Fig. 1p), raising a possible role of LINK-A in eliciting a BRK conformational change that mitigates the conformation required for self-inhibition. Indeed, we found that full-length LINK-A and ΔLRRK2 LINK-A dramatically enhanced the autophosphorylation and kinase activity of BRK while both the control lncRNA and ΔBRK LINK-A showed minimal effects (Fig. 3e, f).

We next conducted a protease digestion assay by incubating BRK with Caspase-1 in the presence of full-length LINK-A or ΔBRK LINK-A, finding that Caspase-1 barely cleaved BRK at aa. 397 in the presence of an unrelated lncRNA RP11-383G10.5, but robustly cleaved BRK only in the presence of full-length LINK-A (Fig. 3g), suggesting a potential conformational change occurred in BRK to expose the digestion site upon LINK-A binding. Notably, depletion of either two regions of LINK-A involved in BRK interaction failed to promote the Caspase-1 dependent BRK cleavage (Fig. 3g), suggesting that simultaneous binding of LINK-A to two BRK domains is required to elicit the conformational change in BRK. Our data suggest that the binding of LINK-A to BRK promotes a conformational change, leading to increased accessibility of the SH2 domain and the autophosphorylation sites in the kinase domain. Upon ligand stimulation, these events lead to the recruitment of BRK to Tyr525-phosphorylated GPNMB and activation of BRK upon Tyr351 phosphorylation.

LINK-A-interacting BRK and LRRK2 phosphorylate HIF1αWe next performed in vitro phosphorylation assays, finding that activated BRK phosphorylated HIF1α at Tyr565 (Fig. 4a) and LRRK2, another LINK-A interacting protein kinase, phosphorylated HIF1α at Ser797, which was further demonstrated by the marked inhibition of HIF1α phosphorylation in the presence of a S797A point mutant (Fig. 4a). The BRK kinase activity deficient mutant, Y351F, diminished the phosphorylation of HIF1α in vivo (Fig. 4b). Both Tyr565 and Ser797 of HIF1α are conserved (Supplementary Fig. 4a).HB-EGF induced phosphorylation of GPNMB (Tyr525) and BRK (Tyr351), as well as HIF1α protein stabilization under normoxic conditions (Fig. 4c and Supplementary Fig. 4b). Interestingly, knockdown of EGFR abolished the ligand-dependent phosphorylation of GPNMB (Tyr525) and BRK (Tyr351), as well as the stabilization of HIF1α; knockdown of GPNMB abolished HB-EGF induced BRK phosphorylation and HIF1α protein stabilization, but did not affect EGFR phosphorylation (Tyr1068) (Fig. 4d). Knockdown of LINK-A in both MDA-MB-231 and MDA-MB-468 cells eliminated HB-EGF induced BRK phosphorylation and HIF1α stabilization, but not phosphorylation of EGFR or GPNMB (Fig. 4e, f). In contrast, LINK-A knockdown exhibited minimal effects on hypoxia-dependent HIF1α stabilization, and hypoxia failed to trigger phosphorylation of GPNMB and BRK (Fig. 4c–f and Supplementary Fig. 4c). Finally, depletion of BRK decreased ligand-triggered HIF1α protein accumulation but did not affect the phosphorylation status of EGFR or GPNMB (Fig. 4d). Taken together, these data suggest a linear EGFR:GPNMB→LINK-A→BRK/LRRK2→HIF1α signaling cascade upon HB-EGF stimulation under normoxic conditions.Upon HB-EGF stimulation, HIF1α underwent Tyr565 and Ser797 phosphorylations but the hydroxylation at Pro564 was inhibited, which led to HIF1α stabilization (Fig. 4g). Knockdown of LINK-A abolished HB-EGF induced HIF1α Tyr565-phosphorylation and enhanced the Pro564-hydroxylation (Fig.4h and Supplementary Fig. 4d). A similar pattern was observed with EGFR, GPNMB and BRK knockdown (Fig.4i). These data suggest that HB-EGF triggers an lncRNA-dependent signaling pathway to stabilize the HIF1α at protein level.

LINK-A-interacting BRK and LRRK2 phosphorylate HIF1α

We next performed in vitro phosphorylation assays, finding that activated BRK phosphorylated HIF1α at Tyr565 (Fig. 4a) and LRRK2, another LINK-A interacting protein kinase, phosphorylated HIF1α at Ser797, which was further demonstrated by the marked inhibition of HIF1α phosphorylation in the presence of a S797A point mutant (Fig. 4a). The BRK kinase activity deficient mutant, Y351F, diminished the phosphorylation of HIF1α in vivo (Fig. 4b). Both Tyr565 and Ser797 of HIF1α are conserved (Supplementary Fig. 4a).

HB-EGF induced phosphorylation of GPNMB (Tyr525) and BRK (Tyr351), as well as HIF1α protein stabilization under normoxic conditions (Fig. 4c and Supplementary Fig. 4b). Interestingly, knockdown of EGFR abolished the ligand-dependent phosphorylation of GPNMB (Tyr525) and BRK (Tyr351), as well as the stabilization of HIF1α; knockdown of GPNMB abolished HB-EGF induced BRK phosphorylation and HIF1α protein stabilization, but did not affect EGFR phosphorylation (Tyr1068) (Fig. 4d). Knockdown of LINK-A in both MDA-MB-231 and MDA-MB-468 cells eliminated HB-EGF induced BRK phosphorylation and HIF1α stabilization, but not phosphorylation of EGFR or GPNMB (Fig. 4e, f). In contrast, LINK-A knockdown exhibited minimal effects on hypoxia-dependent HIF1α stabilization, and hypoxia failed to trigger phosphorylation of GPNMB and BRK (Fig. 4c–f and Supplementary Fig. 4c). Finally, depletion of BRK decreased ligand-triggered HIF1α protein accumulation but did not affect the phosphorylation status of EGFR or GPNMB (Fig. 4d). Taken together, these data suggest a linear EGFR:GPNMB→LINK-A→BRK/LRRK2→HIF1α signaling cascade upon HB-EGF stimulation under normoxic conditions.

Upon HB-EGF stimulation, HIF1α underwent Tyr565 and Ser797 phosphorylations but the hydroxylation at Pro564 was inhibited, which led to HIF1α stabilization (Fig. 4g). Knockdown of LINK-A abolished HB-EGF induced HIF1α Tyr565-phosphorylation and enhanced the Pro564-hydroxylation (Fig.4h and Supplementary Fig. 4d). A similar pattern was observed with EGFR, GPNMB and BRK knockdown (Fig.4i). These data suggest that HB-EGF triggers an lncRNA-dependent signaling pathway to stabilize the HIF1α at protein level.

Tyr565 phosphorylation antagonizes Pro564 hydroxylation to stabilize HIF1α under normoxiaAn in vitro hydroxylation assay demonstrated that the HIF1α peptides (aa.557–566) but not Tyr565-phosphorylated peptides can be hydroxylated by PHD1 (Fig. 4j, Supplementary Fig. 4e–j and Supplementary Table 3). An in vitro kinase assay followed by in vitro hydroxylation assay further showed that phosphorylation of wild-type HIF1α but not Y565F mutant by BRK prevented subsequent hydroxylation at Pro564 (Fig. 4k). Consistently, HB-EGF triggered Tyr565 phosphorylation of HIF1α and inhibition of hydroxylation at Pro564, which was abolished by overexpression of the Y565F mutant of HIF1α (Fig. 4l).Cycloheximide (CHX) treatment experiment revealed that upon HB-EGF stimulation, HIF1α protein exhibited ≥4 hours half-life while knocking down of LINK-A reduced it to 1.5 hours (Supplementary Fig. 4k, l). In TNBC cells exogenously expressing wild-type HIF1α or Y565D mutant, Y565D mutant exhibited constitutively prolonged half-life (Supplementary Fig. 4m–o). These data indicate that LINK-A-associated BRK phosphorylated HIF1α at Tyr565, which prevents HIF1α hydroxylation at adjacent Pro564 and stabilizes HIF1α under normoxia.

Tyr565 phosphorylation antagonizes Pro564 hydroxylation to stabilize HIF1α under normoxia

An in vitro hydroxylation assay demonstrated that the HIF1α peptides (aa.557–566) but not Tyr565-phosphorylated peptides can be hydroxylated by PHD1 (Fig. 4j, Supplementary Fig. 4e–j and Supplementary Table 3). An in vitro kinase assay followed by in vitro hydroxylation assay further showed that phosphorylation of wild-type HIF1α but not Y565F mutant by BRK prevented subsequent hydroxylation at Pro564 (Fig. 4k). Consistently, HB-EGF triggered Tyr565 phosphorylation of HIF1α and inhibition of hydroxylation at Pro564, which was abolished by overexpression of the Y565F mutant of HIF1α (Fig. 4l).

Cycloheximide (CHX) treatment experiment revealed that upon HB-EGF stimulation, HIF1α protein exhibited ≥4 hours half-life while knocking down of LINK-A reduced it to 1.5 hours (Supplementary Fig. 4k, l). In TNBC cells exogenously expressing wild-type HIF1α or Y565D mutant, Y565D mutant exhibited constitutively prolonged half-life (Supplementary Fig. 4m–o). These data indicate that LINK-A-associated BRK phosphorylated HIF1α at Tyr565, which prevents HIF1α hydroxylation at adjacent Pro564 and stabilizes HIF1α under normoxia.

LINK-A-recruited LRRK2 phosphorylates Ser797 of HIF1α to potentiate its transcriptional activityKnockdown of LINK-A, LRRK2, or overexpression of the HIF1α S797A mutant abolished Ser797 phosphorylation of HIF1α as well as its association with p300, which was concurrent with the release of FIH49, a protein that binds to HIF1α and inhibits its transactivation function (Fig. 5a, b). We also examined the kinase activity of LRRK2 in the presence of LINK-A, finding that full-length LINK-A, ΔBRK LINK-A or ΔLRRK2 LINK-A exhibited minimal effect on the kinase activity of LRRK2 (Supplementary Fig. 5a). The rescue experiments indicated that full-length LINK-A fully rescued HIF1α phosphorylations and protein stabilization; ΔBRK LINK-A only rescued HIF1α Ser797 phosphorylation and ΔLRRK2 LINK-A restored HIF1α Tyr565 phosphorylation and protein stabilization, but failed to rescue the phosphorylation of HIF1α at Ser797 (Fig. 5c). Recent studies have shown that certain lncRNAs encode small protein peptides50–52. While our data have demonstrated that a predicted ORF of LINK-A has no protein-coding products in vitro (see Supplementary Fig. 1a–c), we further mutated the predicted translational start codon ATG (nt. 318–321), or the potential stop codon TGA (nt. 732–735) of this ORF in a functional rescue experiments, finding that the phosphorylation of BRK (Tyr351) and HIF1α (Tyr565), two major cellular effects mediated by LINK-A, were fully rescued by wild-type LINK-A as well as ATG→TAG or TGA→TGT mutants of LINK-A (Supplementary Fig. 5b–d). These observations suggested that the cellular effect of LINK-A is mainly dependent on its RNA function instead of the potential translational products. Taken together, we demonstrated that LINK-A, in coordination with two protein kinases BRK and LRRK2, mediated a growth factor-triggered signaling cascade to synergistically regulate the phosphorylation and protein stabilization of HIF1α under normoxia.

LINK-A-recruited LRRK2 phosphorylates Ser797 of HIF1α to potentiate its transcriptional activity

Knockdown of LINK-A, LRRK2, or overexpression of the HIF1α S797A mutant abolished Ser797 phosphorylation of HIF1α as well as its association with p300, which was concurrent with the release of FIH49, a protein that binds to HIF1α and inhibits its transactivation function (Fig. 5a, b). We also examined the kinase activity of LRRK2 in the presence of LINK-A, finding that full-length LINK-A, ΔBRK LINK-A or ΔLRRK2 LINK-A exhibited minimal effect on the kinase activity of LRRK2 (Supplementary Fig. 5a). The rescue experiments indicated that full-length LINK-A fully rescued HIF1α phosphorylations and protein stabilization; ΔBRK LINK-A only rescued HIF1α Ser797 phosphorylation and ΔLRRK2 LINK-A restored HIF1α Tyr565 phosphorylation and protein stabilization, but failed to rescue the phosphorylation of HIF1α at Ser797 (Fig. 5c). Recent studies have shown that certain lncRNAs encode small protein peptides50–52. While our data have demonstrated that a predicted ORF of LINK-A has no protein-coding products in vitro (see Supplementary Fig. 1a–c), we further mutated the predicted translational start codon ATG (nt. 318–321), or the potential stop codon TGA (nt. 732–735) of this ORF in a functional rescue experiments, finding that the phosphorylation of BRK (Tyr351) and HIF1α (Tyr565), two major cellular effects mediated by LINK-A, were fully rescued by wild-type LINK-A as well as ATG→TAG or TGA→TGT mutants of LINK-A (Supplementary Fig. 5b–d). These observations suggested that the cellular effect of LINK-A is mainly dependent on its RNA function instead of the potential translational products. Taken together, we demonstrated that LINK-A, in coordination with two protein kinases BRK and LRRK2, mediated a growth factor-triggered signaling cascade to synergistically regulate the phosphorylation and protein stabilization of HIF1α under normoxia.

LINK-A-dependent normoxic HIF1α signaling promotes tumor growth and correlates with TNBCNext, we examined the transcriptional activity of HIF1α upon HB-EGF stimulation by ChIP-seq, finding that under normoxia, HB-EGF triggered the recruitment of HIF1α to the promoters of HIF1α target genes and regulated the HIF1α-dependent transcriptional program (Fig. 5d, e and Supplementary Table 4). Knockdown of LINK-A in TNBC cells impaired HIF1α-target genes expression upon HB-EGF stimulation (Fig. 5f, g and Supplementary Fig. 5e). Consistently, in vitro glucose uptake and lactate production assays confirmed that LINK-A deficiency impaired glycolysis (Supplementary Fig. 5f–l). Consistent with the in vitro colony formation assays (Fig. 5h), the mice with xenografts of LINK-A-depleted tumor cells rarely developed tumor mass in vivo (Fig. 5i, j and Supplementary Fig. 5m).LINK-A-mediated signaling pathway was also activated in TNBC tissues, as evidenced by significantly higher staining density of phospho-GPNMB (Tyr525), phospho-BRK (Tyr351), phospho-HIF1α (Tyr565), and phospho-HIF1α (Ser797) in TNBC samples compared to non-TNBC samples (Fig. 6a–c and Supplementary Fig. 6a). Furthermore, within the TNBC category, breast cancer with advanced lymph-node metastasis showed increased phospho-BRK (Tyr351), phospho-HIF1α (Tyr565), and phospho-GPNMB (Tyr525) levels compared to tissue samples with no lymph-node metastasis (Fig. 6a–c and Fig. 6d–f, upper panel). Importantly, there is a strong correlation between LINK-A expression and the phosphorylation status of BRK, HIF1α, and GPNMB in these TNBC tissues (Fig. 6d–f, lower panel), and breast cancer patients with higher levels of these phosphoproteins exhibited shorter survival time (Fig. 6g–i). Furthermore, the TCGA database revealed that both BRK and LRRK2 are highly expressed in invasive breast carcinoma (Supplementary Fig. 6b). Our data implicate LINK-A and its associated signaling pathway as potential biomarkers and therapeutic targets for TNBC.

LINK-A-dependent normoxic HIF1α signaling promotes tumor growth and correlates with TNBC

Next, we examined the transcriptional activity of HIF1α upon HB-EGF stimulation by ChIP-seq, finding that under normoxia, HB-EGF triggered the recruitment of HIF1α to the promoters of HIF1α target genes and regulated the HIF1α-dependent transcriptional program (Fig. 5d, e and Supplementary Table 4). Knockdown of LINK-A in TNBC cells impaired HIF1α-target genes expression upon HB-EGF stimulation (Fig. 5f, g and Supplementary Fig. 5e). Consistently, in vitro glucose uptake and lactate production assays confirmed that LINK-A deficiency impaired glycolysis (Supplementary Fig. 5f–l). Consistent with the in vitro colony formation assays (Fig. 5h), the mice with xenografts of LINK-A-depleted tumor cells rarely developed tumor mass in vivo (Fig. 5i, j and Supplementary Fig. 5m).

LINK-A-mediated signaling pathway was also activated in TNBC tissues, as evidenced by significantly higher staining density of phospho-GPNMB (Tyr525), phospho-BRK (Tyr351), phospho-HIF1α (Tyr565), and phospho-HIF1α (Ser797) in TNBC samples compared to non-TNBC samples (Fig. 6a–c and Supplementary Fig. 6a). Furthermore, within the TNBC category, breast cancer with advanced lymph-node metastasis showed increased phospho-BRK (Tyr351), phospho-HIF1α (Tyr565), and phospho-GPNMB (Tyr525) levels compared to tissue samples with no lymph-node metastasis (Fig. 6a–c and Fig. 6d–f, upper panel). Importantly, there is a strong correlation between LINK-A expression and the phosphorylation status of BRK, HIF1α, and GPNMB in these TNBC tissues (Fig. 6d–f, lower panel), and breast cancer patients with higher levels of these phosphoproteins exhibited shorter survival time (Fig. 6g–i). Furthermore, the TCGA database revealed that both BRK and LRRK2 are highly expressed in invasive breast carcinoma (Supplementary Fig. 6b). Our data implicate LINK-A and its associated signaling pathway as potential biomarkers and therapeutic targets for TNBC.

DISCUSSIONOur study reveals that lncRNA directly interacts with non-receptor tyrosine kinase and facilitate it’s recruitment to membrane-bound receptor complex and subsequent activation upon ligand stimulation, broadening the known mechanisms of lncRNA action (Fig. 6j). The regulatory mechanism of non-receptor tyrosine kinase activation is largely unknown. We propose a model that LINK-A interacts with non-receptor tyrosine kinases to facilitate their activation. At the basal level, BRK, a prototype RNA-binding non-receptor tyrosine kinase, is in a “closed” conformation and its kinase activity is auto-inhibited, mediated by the self-inhibitory interaction between SH2 domain and phospho-C-terminal (Tyr447)46. The binding of LINK-A to both the SH3 domain and the C-terminal region of BRK leads to a more accessible structure of BRK as “prime” condition, which may contribute to higher accessibility by other regulatory proteins and kinases for its activation.Most common cancer types show increased HIF1α protein levels although hypoxic areas are missing53,54. Our study delineates an lncRNA-protein kinase module that regulates normoxic HIF1α stabilization with respect to functional implications in the glycolytic reprogramming and tumorigenesis. The LINK-A-dependent HIF1α signaling cascade and the consequent effects on cancer cell glycolysis implicate LINK-A and LINK-A interacting kinases/receptors as promising therapeutic targets for TNBC. Analyses of the LINK-A expression status in the TCGA database and breast cancer tissues both indicated that LINK-A significantly correlates with TNBC, revealing an lncRNA that can serve as a biomarker for further classification of TNBC.Our study identifies four previous unknown phosphorylation sites of GPNMB, BRK and HIF1α in a LINK-A-regulated signaling pathway for glycolysis reprogramming in TNBC. These phosphorylation events predict worse outcome in TNBC patients, suggesting that LINK-A-dependent signaling pathway plays a critical role in TNBC and may provide wide-ranging therapeutic targets for treating TNBC.

Our study reveals that lncRNA directly interacts with non-receptor tyrosine kinase and facilitate it’s recruitment to membrane-bound receptor complex and subsequent activation upon ligand stimulation, broadening the known mechanisms of lncRNA action (Fig. 6j). The regulatory mechanism of non-receptor tyrosine kinase activation is largely unknown. We propose a model that LINK-A interacts with non-receptor tyrosine kinases to facilitate their activation. At the basal level, BRK, a prototype RNA-binding non-receptor tyrosine kinase, is in a “closed” conformation and its kinase activity is auto-inhibited, mediated by the self-inhibitory interaction between SH2 domain and phospho-C-terminal (Tyr447)46. The binding of LINK-A to both the SH3 domain and the C-terminal region of BRK leads to a more accessible structure of BRK as “prime” condition, which may contribute to higher accessibility by other regulatory proteins and kinases for its activation.

Most common cancer types show increased HIF1α protein levels although hypoxic areas are missing53,54. Our study delineates an lncRNA-protein kinase module that regulates normoxic HIF1α stabilization with respect to functional implications in the glycolytic reprogramming and tumorigenesis. The LINK-A-dependent HIF1α signaling cascade and the consequent effects on cancer cell glycolysis implicate LINK-A and LINK-A interacting kinases/receptors as promising therapeutic targets for TNBC. Analyses of the LINK-A expression status in the TCGA database and breast cancer tissues both indicated that LINK-A significantly correlates with TNBC, revealing an lncRNA that can serve as a biomarker for further classification of TNBC.

Our study identifies four previous unknown phosphorylation sites of GPNMB, BRK and HIF1α in a LINK-A-regulated signaling pathway for glycolysis reprogramming in TNBC. These phosphorylation events predict worse outcome in TNBC patients, suggesting that LINK-A-dependent signaling pathway plays a critical role in TNBC and may provide wide-ranging therapeutic targets for treating TNBC.

MethodsTissue samplesBreast cancer tissue microarrays were purchased from Biomax, Biochain and USbiolabs. Two sets of fresh frozen breast cancer tissues (Nanjing Cohorts and Duke Cohorts) were obtained from Yixing People’s Hospital (Yixing, Jiangsu Province, China) and Duke University respectively. The study protocol was approved by the Institutional Review Board of Nanjing Medical University (Nanjing, China) and Duke University Health System. All tissue samples were collected in compliance with informed consent policy. Detailed clinical information is summarized in Supplementary Table 1.Cell culture, transfection, and lentiviral transductionHuman Breast cancer cell lines, human mammary gland epithelia cell line, and human embryonic kidney cell line were purchased from American Type Culture Collection (ATCC) and Characterized Cell Line Core Facility (MD Anderson Cancer Center). siRNA and plasmid transfections were performed using DharmaFECT4 (Thermo Scientific) and Lipofectamine® 3000 (Life Technologies). Lentiviruses were produced in HEK293T cells with ViraPower Lentiviral Expression System. All of the cell lines were free of mycoplasma contamination tested by vendors using MycoAlert kit from Lonza. No cell lines used in this study are found in the database of commonly misidentified cell lines (ICLAC and NCBI Biosample) based on short tandem repeats (STR) profiling performed by vendors.Cell treatments, cross-linking, fractionation, cell lysis, immunoprecipitation and immunoblottingCells were serum starved overnight followed by growth factor (Peprotech) treatment for 30 minutes at the following concentrations: EGF (10 ng/ml), Amphiregulin (10 ng/ml), Betacellulin (10 ng/ml), Epigen (200 ng/ml), Epiregulin (10 ng/ml), HB-EGF (10 ng/ml), Heregulin-β1 (5 ng/ml), TGF-α (2 ng/ml). Cetuximab (20 µg/ml) was provided by Dr. Scott Kopetz (MD Anderson Cancer Center). Chemical cross-linking was carried out as previously described55 with 1 mM 3,3´-Dithiobis[sulfosuccinimidylpropionate] (DTSSP) (Pierce). In certain experiments, cells were pre-treated with 10 µM InSolution™ MG-132 (EMD Millipore) for 6 hours before growth factor treatment. Nuclear /cytoplasmic fractionation, cell lysis, immunoprecipitation and immunoblotting were performed as previously described16.RNA preparation, northern blot and RACE analysisRNA in vitro transcription and purification were performed as previously described16. Total RNAs from 1×106 MDA-MB-231 cells with or without HB-EGF treatment were analyzed for LINK-A and Beta-Actin expression using Biotin-labeled LNA™ probes (Exiqon, sequence are listed in the Oligonucleotide sequences, probes and primers section) according to NorthernMax® Kit (Ambion). RACE-PCR was performed using SMARTer RACE 5'/3' Kit (Clontech).Cloning proceduresFull-length LINK-A and deletion mutants were constructed by subcloning the gene sequences into pCDNA3.1 (+) backbone (Life Technologies). To generate LNA#5-resistant LINK-A mammalian expression vectors used in the rescue experiments, LNA#5 targeting sequence ACA GCT CAT TTA TCC A was mutated to ACA GGC GAT TTA TCC A.The full-length HIF1α, GPNMB, BRK and LRRK2 mammalian expression vectors were obtained from Origene and Addgene. His-tagged full-length EGFR, extracellular domain (ECD) and intracellular domain (ICD) were provided by Dr. Mien-Chie Hung (MD Anderson Cancer Center). FLAG-tagged full-length GPNMB, ECD+transmembrane domain (TM), and ICD+TM were constructed by subcloning the corresponding gene sequences into SFB-tagged expression vector (provided by Dr. Junjie Chen, MD Anderson Cancer Center) using the Gateway system (Life Technologies).Bacterial expression vectors for His-tagged HIF1α and GPNMB ICD were constructed by subcloning the corresponding gene sequences into pET-DEST42 vector. GST-tagged BRK (wt and mutants) were constructed into pGEX-5X-1 backbone (GE Healthcare). GST-tagged EGFR ICD, EGFR kinase domain (KD), and EGFR C-terminal domain (CTD) in pGEX-6p-1backbone were provided by Dr. Mien-Chie Hung (MD Anderson Cancer Center).All single-point and deletion mutations were generated using QuikChange™ Lightning Site-Directed Mutagenesis Kit (Agilent Technologies).siRNA, shRNA and LNA™Lincode SMARTpool siRNA targeting LINK-A (R-027622) and ON-TARGETplus SMARTpool siRNA targeting EGFR (L-003114), GPNMB (L-011741), PTK6/BRK (L-003166) and LRRK2 (L-006323) from Dharmacon were used in this study. shRNA targeting LINK-A was designed based on Lincode SMARTpool siRNA sequence and cloned into pLKO.1-Puro vector. LNAs targeting LINK-A were designed and synthesized by Exiqon. Detailed sequences were listed in Oligonucleotide sequences, probes and primers section.AntibodiesCell Signaling Technology: anti-EGFR (D38B1) rabbit mAb (4267), anti-GPNMB (E1Y7J) rabbit mAb (13251), anti-LRRK2 (5559), anti-phospho-EGFR (Tyr845) (D63B4) rabbit mAb (6963), anti-phospho-EGFR (Tyr992) (2235), anti-phospho-EGFR (Tyr1045) (2237), anti-phospho-EGFR (Tyr1068) (D7A5) rabbit mAb (3777), anti-phospho-EGFR (Tyr1148) (4404), anti-phospho-EGFR (Tyr1173) (53A5) rabbit mAb (4407), anti-hydroxy-HIF1α (Pro564) (D43B5) rabbit mAb (3434), anti-PHD2 (3293), anti-VHL (2738), anti-LRRK2 (5559), anti-EGFR (D38B1) rabbit mAb (Alexa Fluor® 555 Conjugate) (5108), anti-GST tag (26H1) mouse mAb (2624), anti-His tag (2365) and anti-eIF4B (3592); Santa Cruz Biotechnology: anti-FIH (H-299) (sc-48813), anti-HA (Y-11) (sc-805), anti-BRK (C-18) (sc-1118), anti-GAPDH (6C5) mouse mAb (sc-32233) and anti-HIF1α (28b) mouse mAb (sc-13515); Millipore: anti-phosphotyrosine (4G10) Platinum mouse mAb (05-1050X), anti-Myc tag (4A6) mouse mAb (05-724) and anti-eIF4B (ABS281); Thermo Scientific: anti-EGFR (Ab-13) mouse mAb (MS-609); Sigma-Aldrich: anti-FLAG tag (M2) mouse mAb (F3165); Active Motif: anti-p300 (NM11) mouse mAb (61402); Novus Biologicals: anti-HIF1α (NB100-134); Life Technologies: anti-phosphoserine (Poly-Z-PS1); YenZym Antibodies, LLC: anti-phospho-GPNMB (Tyr525) [p-GPNMB (Tyr525)], anti-phospho-BRK (Tyr351) [p-BRK (Tyr351)], anti-phospho-HIF1α (Tyr565) [p-HIF1α (Tyr565)], and anti-phospho-HIF1α (Ser797) [p-HIF1α (Ser797)]. The specificity of phospho-specific antibodies were confirmed by blocking peptide competition assay. The antibodies were used as 1:1000 dilutions for immunoblotting experiments and 1:200 for immunoprecipitation, immunofluorescence and immunohistochemistry experiments.Protein recombination, purification and in vitro translationRecombinant proteins were expressed in E.coli strain BL21-CodonPlus® (DE3)-RIPL (Agilent Technologies) and purified using Protein Purification Kit (Clontech). Recombinant Flag-GPNMB and PHD1 were purchased from Origene. GST-EGFR was purchased from Active Motif, HIF1α and BRK were purchased from Novus Biologicals. LRRK2 was purchased from SignalChem. Recombinant active Caspase-1 was purchased from R&D Systems. In vitro translation of LINK-A was conducted using TnT® Quick Coupled Transcription/Translation Kit and detection was performed using Transcend™ Non-Radioactive Translation Detection System (Promega).RNA pulldown, mass spectrometry analysis, in vitro RNA-protein binding assay and in vitro RNA-protein binding coupled with dot-blot assayThe cell lysates were freshly prepared using ProteaPrep Zwitterionic Cell Lysis Kit, Mass Spec Grade (Protea®) with Anti-RNase, Protease/ Phosphatase Inhibitor Cocktail, Panobinostat and Methylstat supplemented in the lysis buffer. The BcMag™ Monomer avidin Magnetic Beads (Bioclone) were first prepared according to manufacturer’s instructions and then immediately subjected to RNA (20 µg) capture in RNA capture buffer [20 mM Tris-HCl pH 7.5, 1M NaCl, 1mM EDTA] for 30 minutes at room temperature with agitation. The RNA-captured beads were washed once with NT2 buffer [50 mM Tris-HCl pH 7.4, 150 mM NaCl, 1 mM MgCl2, 0.05% NP-40] and incubated with 30 mg cell lysates diluted in NT2 buffer supplemented with 50 U/mL Anti-RNase, 2 mM dithiothreitol, 30 mM EDTA and Heparin 0.02 mg/ml for 4 hours at 4°C with rotation. The RNA-binding protein complexes were washed sequentially with NT2 buffer (twice), NT2-high salt buffer containing 500 mM NaCl (twice), NT2-high salt buffer containing 1 M NaCl (once), NT2-KSCN buffer containing 750 mM KSCN (twice) and PBS (once) for 5 minutes at 4°C and eluted by 2 mM D-biotin in PBS. The eluted protein complexes were denatured, reduced, alkylated and digested with immobilized trypsin (Promega) for MS analysis at MD Anderson Cancer Center Proteomics Facility. The RNA-protein binding assays and in vitro RNA-protein binding coupled with dot-blot assay were performed as described previously16.RNAScope® assay, RNA FISH, immunofluorescence and immunohistochemistryRNAScope® assay and RNA FISH were performed as previously described16. RNAScope® robes targeting LINK-A (Cat# 412027), BCAR4 (Cat# 407777) or HOTAIR (Cat# 312347) were custom designed or purchased from Advanced Cell Diagnostics. LNA™ FISH probes targeting LINK-A and control probe targeting Beta-Actin (300512-04) were purchased from Exiqon (sequences were listed in Oligonucleotide sequences, probes and primers section).For immuno-RNA FISH, the slide from RNA FISH was further blocked with blocking buffer [1 × PBS, 5% BSA, 0.3% Triton X-100] for 1 hour at room temperature followed by incubation with primary antibodies (diluted 1:200) for 1 hour at room temperature. After incubation with fluorochrome-conjugated secondary antibodies for 1 hour at room temperature in dark, the slide was washed and mounted for detection. Immunofluorescence and immunohistochemistry were performed as previously described16.The quantification of RNAScope® staining densities was measured by RNAscope® SpotStudio v1.0 Software (Advanced Cell Diagnostics). The quantification of IHC staining density was performed by Image-Pro plus 6.0 (Media Cybernetics) and calculated based on the average staining intensity and the percentage of positively stained cells.Computational analysis of TCGA RNA-Seq dataBreast cancer RNA-seq BAM files was downloaded from UCSC Cancer Genomics Hub (CGHub, https://cghub.ucsc.edu/). TCGA BAM files were generated based on MapSplice algorithm for alignment against the hg19 reference genome using default parameters56. We then quantified lncRNA expression LINK-A as RPKM (reads per kilobase per million mapped reads)57 as previously described58, and the analysis was based on log2(RPKM+1). Clinical information, PAM50 subtype, ER, PR, and HER2 status, were obtained from the TCGA marker paper1. We used analysis of variance (ANOVA) or student t-test to detect the statistical difference between two or more groups.In vitro kinase assayWild-type or mutant substrate proteins were incubated with 50 µl of in vitro kinase assay buffer II (SignalChem) containing 100 µM ATP (cold reaction) or 10 µCi [γ-32P] ATP and indicated protein kinase for 1 hour at 30°C. Resulting products were separated by SDS-PAGE and detected by Coomassie Blue staining, autoradiography or immunoblotting with phospho-specific antibodies. The specific BRK and LRRK2 kinase activity were measured using a Universal Kinase Activity Kit (R&D Systems).In vitro HIF1α hydroxylation assayFive µg wild-type His-tagged HIF1α or Y565F mutant was incubated with 1 µg recombinant PHD1 in a reaction buffer containing 10 µM FeSO4, 100 µM 2-oxo-glutarate, 1 mM ascorbate, 100 µM dithiothreitol, and 50 µM Tris-HCl (pH 7.8) at 37°C for 1 hour. HIF1α hydroxylation was analyzed by SDS-PAGE and immunoblotting with specific antibody against Pro564 hydroxylation. For the quantitative peptide hydroxylation assay, 10 µg synthesized unmodified peptide (LDLEMLAPYI) or Tyr565-phosphopeptide (LDLEMLAP-pY-I) and 3 µg recombinant PHD1 were incubated in the same reaction buffer described above or in the same buffer, except containing 100 µM DMOG. Resulting peptides were purified by ZipTip and analyzed by liquid chromatography coupled with tandem MS (LC-MS) to confirm the presence of the Proline-hydroxylated peptides. The acquired MS/MS data were searched against a database to identify hydroxylated Proline sites through a dynamic mass shift for the modified Proline (+15.9949 Da).Chromatin immunoprecipitation (ChIP), RNA immunoprecipitation (RIP), ChIP-seq and data analysisChIP and RIP were performed as previously described16. ChIP-seq and data analysis were performed by ArrayStar. The mapped reads were used for peak detection by MACS v1.4.0 (Model-based Analysis of ChIP-Seq) software. Statistically significant ChIP-enriched regions (peaks) were identified by comparison to a Poisson background model (Cut-off p-value=10−4).Anchorage-independent growth assay, glucose uptake assay and lactate production assayAnchorage-independent growth assay was performed as previously described59. Glucose uptake and lactate production assay were performed using Glucose Uptake Cell-based Assay Kit and L-Lactate Assay Kit (Cayman Chemical) respectively. Lactate production was expressed as lactate concentration per 104 viable cells.In vivo tumorigenesis studyAll animal experiments were performed in accordance with a protocol approved by the Institutional Animal Care and Use Committee of MD Anderson Cancer Center. Animals arrived in our facility were randomly put into cages with five mice each. They were implanted with respective tumor cells in the unit of cages, which were randomly selected. The animal experiment was set up to use 5 mice per group to detect a 2-fold difference with power of 80% and at the significance level of 0.05 by a two-sided test for significant studies. Tumor cells in 30 µl growth medium (mixed with Matrigel at a 1:1 ratio) were injected subcutaneously into the flank of six- to eight-week-old female nude mice. Tumor size was measured every five days using a caliper, and tumor volume was calculated using the standard formula: 0.54×L×W2, where L is the longest diameter and W is the shortest diameter. The tumors were removed, photographed and weighed. The investigators were not blinded to allocation during experiments and outcome assessment.Oligonucleotide sequences (5’-3’), probes (5’-3’) and primers (forward and reverse)qPCR primers for gene expression and RIPLINK-A (TTC CCC CAT TTT TCC TTT TC and CTC TGG TTG GGT GAC TGG TT); GAPDH (GAA GGT GAA GGT CGG AGT and GAA GAT GGT GAT GGG ATT TC); ANGPTL4 (CAC AGC CTG CAG ACA CAA CT and AAA CTG GCT TTG CAG ATG CT); ALDOA (CTG CCA GTA TGT GAC CGA GA and ACA GGA AGG TGA TCC CAG TG); ANKRD37 (GTA GCC AGT GAT GCC CAA AT and CTT CCG AGA CTC CGT TTC TG); BHLHE40 (CCT TGA AGC ATG TGA AAG CA and GCT TGG CCA GAT ACT GAA GC); EGR1 (TGA CCG CAG AGT CTT TTC CT and TGG GTT GGT CAT GCT CAC TA); IGFBP3 (GGG GTG TAC ACA TTC CCA AC and AGG CTG CCC ATA CTT ATC CA); LDHA (TGT GCC TGT ATG GAG TGG AA and AGC ACT CTC AAC CAC CTG CT); MAPK1 (CCA GAG AAC CCT GAG GGA GA and TCG ATG GTT GGT GCT CGA AT); PKM2 (ATC GTC CTC ACC AAG TCT GG and GAA GAT GCC ACG GTA CAG GT); RPLP0 (TGG TCA TCC AGC AGG TGT TCG A and ACA GAC ACT GGC AAC ATT GCG G); HIF1α (GTC TGA GGG GAC AGG AGG AT and CTC CTC AGG TGG CTT GTC AG); B2M (GGC TAT CCA GCG TAC TCC AA and TGG ATG AAA CCC AGA CAC ATA)qPCR primers for LINK-A knockdown-rescue gene expression and RIP experimentsLINK-A (TAT GGA GGA TCG CTG TTT CC and CCA AAG ATG TCG CAG GAC TT)siRNA sequencesLINK-A (UGU CUA AGG UGG AGA UUA C, AGA UGU AGU UCU AGU UCA U, UUA CUG AGG UUG AAU AUG U and GGU CUU CAU UCU UAC GCU U); EGFR (CAA AGU GUG UAA CGG AAU A, CCA UAA AUG CUA CGA AUA U, GUA ACA AGC UCA CGC AGU U, and CAG AGG AUG UUC AAU AAC U); GPNMB (GGA AUU UCA UCU ACG UCU U, AUA UAA CAU UUG CGG UGA A, UGC AAG AAG AGG CGG GAU A, and CCA GAA GAA CGA UCG AAA U); PTK6/BRK (GAG AAA GUC CUG CCC GUU U, CCA UUA AGG UGA UUU CUC G, UGC CCG AGC UUG UGA ACU A, and GGC CAU UAC UCC ACC AAA U), LRRK2 (CAA GUU AUU UCA AGG CAA A, UUA CCG AGA UGC CGU AUU A, GGA GGG AUC UUC UUU AAU U, and GAA AUU AUC AUC CGA CUA U)LNA™ GapmeR sequencesNegative control (AAC ACG TCT ATA CGC); LINK-A #1 (GCG TAA GAA TGA AGA); LINK-A #2 (GTG ATA AGA CTA AGT G); LINK-A #3(GAA TAA GGA TAA GCG T); LINK-A #4 (CCA CAG CTT GAA TTC C) and LINK-A #5(TGG ATA AAT GAG CTG T)LINK-A shRNA sequencesLINK-A sh#1 (TGT CTA AGG TGG AGA TTA C); LINK-A sh#2 (AGA TGT AGT TCT AGT TCA T); LINK-A sh#3 (TTA CTG AGG TTG AAT ATG T); LINK-A sh#4 (GGT CTT CAT TCT TAC GCT T).RNA FISH probesLINK-A (/56-FAM/TGT AGC CAC AGA CAT CAT TAC A), beta-Actin (/Fluorescein/CTC ATT GTA GAA GGT GTG GTG CCA).LNA™ RNA detection probesBeta-Actin (/5Biosg/CTC ATT GTA GAA GGT GTG GTG CCA) and LINK-A (/5Biosg/ ACT AAG TGT TGG CAG GTT ATG T).LINK-A RACE-PCR primers3’-RACE (TGG AAT TCA AGC TGT GGG TG) and 5’-RACE (GCA TTT TTA TTT TAA TTG AGG)qPCR primers for ChIPEGLN3 (CGT GGA GGA CTG GCT CTA AG and GGT GTG CTC GGG TGT G)60, ERRFI1 (GAT TAC AGG CTG GAT GGC AC and TGC TGC CAG ACT GGT ATG AG)61, ARRDC3 (CCC CTG CAG TCA CAC ACT C and TTT GTC ACA TGG GAC TCT TC)31, ANKRD37 (CCA GTT TCC TGG TTA CGT GC and TAA GTC AGT GGG CGT GAG AG)62, RPLP0 (TGA AGA GCA GAG GCG ACC CAC and ATG GGT GTC GGC GTG AC)60.Statistics & ReproducibilityThe experiment was set up to use 3–5 samples/repeats per experiment/group/condition to detect a 2-fold difference with power of 80% and at the significance level of 0.05 by a two-sided test for significant studies. For RNAscope®, immunostaining, immunohistochemical staining, colony formation assay, northern blotting, RACE analysis, and western blotting, the representative images are shown. Each of these experiments was independently repeated for 3–5 times. Relative quantities of gene expression level were normalized to B2M. The relative quantities of ChIP samples were normalized by individual inputs, respectively. Results are reported as mean ± standard error of the mean (S.E.M.) of at least three independent experiments. Each exact n value is indicated in the corresponding figure legend. Comparisons were performed using two tailed paired Student’s t test or two-way ANOVA (*p < 0.05, **p < 0.01, and ***p < 0.001), as indicated in individual figures. Fisher’s exact test was implemented for statistical analyses of the correlation between markers and clinical parameters. For survival analysis, the expression of LINK-A or phosphorylation density of indicated proteins was treated as a binary variant and divided into ‘high’ and ‘low’ level. Kaplan-Meier survival curves were compared using the Gehan-Breslow test with GraphPad Prism (GraphPad Software). The experiments were not randomized. The investigators were not blinded to allocation during experiments and outcome assessment.

Tissue samplesBreast cancer tissue microarrays were purchased from Biomax, Biochain and USbiolabs. Two sets of fresh frozen breast cancer tissues (Nanjing Cohorts and Duke Cohorts) were obtained from Yixing People’s Hospital (Yixing, Jiangsu Province, China) and Duke University respectively. The study protocol was approved by the Institutional Review Board of Nanjing Medical University (Nanjing, China) and Duke University Health System. All tissue samples were collected in compliance with informed consent policy. Detailed clinical information is summarized in Supplementary Table 1.

Tissue samples

Breast cancer tissue microarrays were purchased from Biomax, Biochain and USbiolabs. Two sets of fresh frozen breast cancer tissues (Nanjing Cohorts and Duke Cohorts) were obtained from Yixing People’s Hospital (Yixing, Jiangsu Province, China) and Duke University respectively. The study protocol was approved by the Institutional Review Board of Nanjing Medical University (Nanjing, China) and Duke University Health System. All tissue samples were collected in compliance with informed consent policy. Detailed clinical information is summarized in Supplementary Table 1.

Cell culture, transfection, and lentiviral transductionHuman Breast cancer cell lines, human mammary gland epithelia cell line, and human embryonic kidney cell line were purchased from American Type Culture Collection (ATCC) and Characterized Cell Line Core Facility (MD Anderson Cancer Center). siRNA and plasmid transfections were performed using DharmaFECT4 (Thermo Scientific) and Lipofectamine® 3000 (Life Technologies). Lentiviruses were produced in HEK293T cells with ViraPower Lentiviral Expression System. All of the cell lines were free of mycoplasma contamination tested by vendors using MycoAlert kit from Lonza. No cell lines used in this study are found in the database of commonly misidentified cell lines (ICLAC and NCBI Biosample) based on short tandem repeats (STR) profiling performed by vendors.

Cell culture, transfection, and lentiviral transduction

Human Breast cancer cell lines, human mammary gland epithelia cell line, and human embryonic kidney cell line were purchased from American Type Culture Collection (ATCC) and Characterized Cell Line Core Facility (MD Anderson Cancer Center). siRNA and plasmid transfections were performed using DharmaFECT4 (Thermo Scientific) and Lipofectamine® 3000 (Life Technologies). Lentiviruses were produced in HEK293T cells with ViraPower Lentiviral Expression System. All of the cell lines were free of mycoplasma contamination tested by vendors using MycoAlert kit from Lonza. No cell lines used in this study are found in the database of commonly misidentified cell lines (ICLAC and NCBI Biosample) based on short tandem repeats (STR) profiling performed by vendors.

Cell treatments, cross-linking, fractionation, cell lysis, immunoprecipitation and immunoblottingCells were serum starved overnight followed by growth factor (Peprotech) treatment for 30 minutes at the following concentrations: EGF (10 ng/ml), Amphiregulin (10 ng/ml), Betacellulin (10 ng/ml), Epigen (200 ng/ml), Epiregulin (10 ng/ml), HB-EGF (10 ng/ml), Heregulin-β1 (5 ng/ml), TGF-α (2 ng/ml). Cetuximab (20 µg/ml) was provided by Dr. Scott Kopetz (MD Anderson Cancer Center). Chemical cross-linking was carried out as previously described55 with 1 mM 3,3´-Dithiobis[sulfosuccinimidylpropionate] (DTSSP) (Pierce). In certain experiments, cells were pre-treated with 10 µM InSolution™ MG-132 (EMD Millipore) for 6 hours before growth factor treatment. Nuclear /cytoplasmic fractionation, cell lysis, immunoprecipitation and immunoblotting were performed as previously described16.

Cell treatments, cross-linking, fractionation, cell lysis, immunoprecipitation and immunoblotting

Cells were serum starved overnight followed by growth factor (Peprotech) treatment for 30 minutes at the following concentrations: EGF (10 ng/ml), Amphiregulin (10 ng/ml), Betacellulin (10 ng/ml), Epigen (200 ng/ml), Epiregulin (10 ng/ml), HB-EGF (10 ng/ml), Heregulin-β1 (5 ng/ml), TGF-α (2 ng/ml). Cetuximab (20 µg/ml) was provided by Dr. Scott Kopetz (MD Anderson Cancer Center). Chemical cross-linking was carried out as previously described55 with 1 mM 3,3´-Dithiobis[sulfosuccinimidylpropionate] (DTSSP) (Pierce). In certain experiments, cells were pre-treated with 10 µM InSolution™ MG-132 (EMD Millipore) for 6 hours before growth factor treatment. Nuclear /cytoplasmic fractionation, cell lysis, immunoprecipitation and immunoblotting were performed as previously described16.

RNA preparation, northern blot and RACE analysisRNA in vitro transcription and purification were performed as previously described16. Total RNAs from 1×106 MDA-MB-231 cells with or without HB-EGF treatment were analyzed for LINK-A and Beta-Actin expression using Biotin-labeled LNA™ probes (Exiqon, sequence are listed in the Oligonucleotide sequences, probes and primers section) according to NorthernMax® Kit (Ambion). RACE-PCR was performed using SMARTer RACE 5'/3' Kit (Clontech).

RNA preparation, northern blot and RACE analysis

RNA in vitro transcription and purification were performed as previously described16. Total RNAs from 1×106 MDA-MB-231 cells with or without HB-EGF treatment were analyzed for LINK-A and Beta-Actin expression using Biotin-labeled LNA™ probes (Exiqon, sequence are listed in the Oligonucleotide sequences, probes and primers section) according to NorthernMax® Kit (Ambion). RACE-PCR was performed using SMARTer RACE 5'/3' Kit (Clontech).

Cloning proceduresFull-length LINK-A and deletion mutants were constructed by subcloning the gene sequences into pCDNA3.1 (+) backbone (Life Technologies). To generate LNA#5-resistant LINK-A mammalian expression vectors used in the rescue experiments, LNA#5 targeting sequence ACA GCT CAT TTA TCC A was mutated to ACA GGC GAT TTA TCC A.The full-length HIF1α, GPNMB, BRK and LRRK2 mammalian expression vectors were obtained from Origene and Addgene. His-tagged full-length EGFR, extracellular domain (ECD) and intracellular domain (ICD) were provided by Dr. Mien-Chie Hung (MD Anderson Cancer Center). FLAG-tagged full-length GPNMB, ECD+transmembrane domain (TM), and ICD+TM were constructed by subcloning the corresponding gene sequences into SFB-tagged expression vector (provided by Dr. Junjie Chen, MD Anderson Cancer Center) using the Gateway system (Life Technologies).Bacterial expression vectors for His-tagged HIF1α and GPNMB ICD were constructed by subcloning the corresponding gene sequences into pET-DEST42 vector. GST-tagged BRK (wt and mutants) were constructed into pGEX-5X-1 backbone (GE Healthcare). GST-tagged EGFR ICD, EGFR kinase domain (KD), and EGFR C-terminal domain (CTD) in pGEX-6p-1backbone were provided by Dr. Mien-Chie Hung (MD Anderson Cancer Center).All single-point and deletion mutations were generated using QuikChange™ Lightning Site-Directed Mutagenesis Kit (Agilent Technologies).

Cloning procedures

Full-length LINK-A and deletion mutants were constructed by subcloning the gene sequences into pCDNA3.1 (+) backbone (Life Technologies). To generate LNA#5-resistant LINK-A mammalian expression vectors used in the rescue experiments, LNA#5 targeting sequence ACA GCT CAT TTA TCC A was mutated to ACA GGC GAT TTA TCC A.

The full-length HIF1α, GPNMB, BRK and LRRK2 mammalian expression vectors were obtained from Origene and Addgene. His-tagged full-length EGFR, extracellular domain (ECD) and intracellular domain (ICD) were provided by Dr. Mien-Chie Hung (MD Anderson Cancer Center). FLAG-tagged full-length GPNMB, ECD+transmembrane domain (TM), and ICD+TM were constructed by subcloning the corresponding gene sequences into SFB-tagged expression vector (provided by Dr. Junjie Chen, MD Anderson Cancer Center) using the Gateway system (Life Technologies).

Bacterial expression vectors for His-tagged HIF1α and GPNMB ICD were constructed by subcloning the corresponding gene sequences into pET-DEST42 vector. GST-tagged BRK (wt and mutants) were constructed into pGEX-5X-1 backbone (GE Healthcare). GST-tagged EGFR ICD, EGFR kinase domain (KD), and EGFR C-terminal domain (CTD) in pGEX-6p-1backbone were provided by Dr. Mien-Chie Hung (MD Anderson Cancer Center).

All single-point and deletion mutations were generated using QuikChange™ Lightning Site-Directed Mutagenesis Kit (Agilent Technologies).

siRNA, shRNA and LNA™Lincode SMARTpool siRNA targeting LINK-A (R-027622) and ON-TARGETplus SMARTpool siRNA targeting EGFR (L-003114), GPNMB (L-011741), PTK6/BRK (L-003166) and LRRK2 (L-006323) from Dharmacon were used in this study. shRNA targeting LINK-A was designed based on Lincode SMARTpool siRNA sequence and cloned into pLKO.1-Puro vector. LNAs targeting LINK-A were designed and synthesized by Exiqon. Detailed sequences were listed in Oligonucleotide sequences, probes and primers section.

siRNA, shRNA and LNA™

Lincode SMARTpool siRNA targeting LINK-A (R-027622) and ON-TARGETplus SMARTpool siRNA targeting EGFR (L-003114), GPNMB (L-011741), PTK6/BRK (L-003166) and LRRK2 (L-006323) from Dharmacon were used in this study. shRNA targeting LINK-A was designed based on Lincode SMARTpool siRNA sequence and cloned into pLKO.1-Puro vector. LNAs targeting LINK-A were designed and synthesized by Exiqon. Detailed sequences were listed in Oligonucleotide sequences, probes and primers section.

AntibodiesCell Signaling Technology: anti-EGFR (D38B1) rabbit mAb (4267), anti-GPNMB (E1Y7J) rabbit mAb (13251), anti-LRRK2 (5559), anti-phospho-EGFR (Tyr845) (D63B4) rabbit mAb (6963), anti-phospho-EGFR (Tyr992) (2235), anti-phospho-EGFR (Tyr1045) (2237), anti-phospho-EGFR (Tyr1068) (D7A5) rabbit mAb (3777), anti-phospho-EGFR (Tyr1148) (4404), anti-phospho-EGFR (Tyr1173) (53A5) rabbit mAb (4407), anti-hydroxy-HIF1α (Pro564) (D43B5) rabbit mAb (3434), anti-PHD2 (3293), anti-VHL (2738), anti-LRRK2 (5559), anti-EGFR (D38B1) rabbit mAb (Alexa Fluor® 555 Conjugate) (5108), anti-GST tag (26H1) mouse mAb (2624), anti-His tag (2365) and anti-eIF4B (3592); Santa Cruz Biotechnology: anti-FIH (H-299) (sc-48813), anti-HA (Y-11) (sc-805), anti-BRK (C-18) (sc-1118), anti-GAPDH (6C5) mouse mAb (sc-32233) and anti-HIF1α (28b) mouse mAb (sc-13515); Millipore: anti-phosphotyrosine (4G10) Platinum mouse mAb (05-1050X), anti-Myc tag (4A6) mouse mAb (05-724) and anti-eIF4B (ABS281); Thermo Scientific: anti-EGFR (Ab-13) mouse mAb (MS-609); Sigma-Aldrich: anti-FLAG tag (M2) mouse mAb (F3165); Active Motif: anti-p300 (NM11) mouse mAb (61402); Novus Biologicals: anti-HIF1α (NB100-134); Life Technologies: anti-phosphoserine (Poly-Z-PS1); YenZym Antibodies, LLC: anti-phospho-GPNMB (Tyr525) [p-GPNMB (Tyr525)], anti-phospho-BRK (Tyr351) [p-BRK (Tyr351)], anti-phospho-HIF1α (Tyr565) [p-HIF1α (Tyr565)], and anti-phospho-HIF1α (Ser797) [p-HIF1α (Ser797)]. The specificity of phospho-specific antibodies were confirmed by blocking peptide competition assay. The antibodies were used as 1:1000 dilutions for immunoblotting experiments and 1:200 for immunoprecipitation, immunofluorescence and immunohistochemistry experiments.

Cell Signaling Technology: anti-EGFR (D38B1) rabbit mAb (4267), anti-GPNMB (E1Y7J) rabbit mAb (13251), anti-LRRK2 (5559), anti-phospho-EGFR (Tyr845) (D63B4) rabbit mAb (6963), anti-phospho-EGFR (Tyr992) (2235), anti-phospho-EGFR (Tyr1045) (2237), anti-phospho-EGFR (Tyr1068) (D7A5) rabbit mAb (3777), anti-phospho-EGFR (Tyr1148) (4404), anti-phospho-EGFR (Tyr1173) (53A5) rabbit mAb (4407), anti-hydroxy-HIF1α (Pro564) (D43B5) rabbit mAb (3434), anti-PHD2 (3293), anti-VHL (2738), anti-LRRK2 (5559), anti-EGFR (D38B1) rabbit mAb (Alexa Fluor® 555 Conjugate) (5108), anti-GST tag (26H1) mouse mAb (2624), anti-His tag (2365) and anti-eIF4B (3592); Santa Cruz Biotechnology: anti-FIH (H-299) (sc-48813), anti-HA (Y-11) (sc-805), anti-BRK (C-18) (sc-1118), anti-GAPDH (6C5) mouse mAb (sc-32233) and anti-HIF1α (28b) mouse mAb (sc-13515); Millipore: anti-phosphotyrosine (4G10) Platinum mouse mAb (05-1050X), anti-Myc tag (4A6) mouse mAb (05-724) and anti-eIF4B (ABS281); Thermo Scientific: anti-EGFR (Ab-13) mouse mAb (MS-609); Sigma-Aldrich: anti-FLAG tag (M2) mouse mAb (F3165); Active Motif: anti-p300 (NM11) mouse mAb (61402); Novus Biologicals: anti-HIF1α (NB100-134); Life Technologies: anti-phosphoserine (Poly-Z-PS1); YenZym Antibodies, LLC: anti-phospho-GPNMB (Tyr525) [p-GPNMB (Tyr525)], anti-phospho-BRK (Tyr351) [p-BRK (Tyr351)], anti-phospho-HIF1α (Tyr565) [p-HIF1α (Tyr565)], and anti-phospho-HIF1α (Ser797) [p-HIF1α (Ser797)]. The specificity of phospho-specific antibodies were confirmed by blocking peptide competition assay. The antibodies were used as 1:1000 dilutions for immunoblotting experiments and 1:200 for immunoprecipitation, immunofluorescence and immunohistochemistry experiments.

Protein recombination, purification and in vitro translationRecombinant proteins were expressed in E.coli strain BL21-CodonPlus® (DE3)-RIPL (Agilent Technologies) and purified using Protein Purification Kit (Clontech). Recombinant Flag-GPNMB and PHD1 were purchased from Origene. GST-EGFR was purchased from Active Motif, HIF1α and BRK were purchased from Novus Biologicals. LRRK2 was purchased from SignalChem. Recombinant active Caspase-1 was purchased from R&D Systems. In vitro translation of LINK-A was conducted using TnT® Quick Coupled Transcription/Translation Kit and detection was performed using Transcend™ Non-Radioactive Translation Detection System (Promega).

Protein recombination, purification and in vitro translation

Recombinant proteins were expressed in E.coli strain BL21-CodonPlus® (DE3)-RIPL (Agilent Technologies) and purified using Protein Purification Kit (Clontech). Recombinant Flag-GPNMB and PHD1 were purchased from Origene. GST-EGFR was purchased from Active Motif, HIF1α and BRK were purchased from Novus Biologicals. LRRK2 was purchased from SignalChem. Recombinant active Caspase-1 was purchased from R&D Systems. In vitro translation of LINK-A was conducted using TnT® Quick Coupled Transcription/Translation Kit and detection was performed using Transcend™ Non-Radioactive Translation Detection System (Promega).

RNA pulldown, mass spectrometry analysis, in vitro RNA-protein binding assay and in vitro RNA-protein binding coupled with dot-blot assayThe cell lysates were freshly prepared using ProteaPrep Zwitterionic Cell Lysis Kit, Mass Spec Grade (Protea®) with Anti-RNase, Protease/ Phosphatase Inhibitor Cocktail, Panobinostat and Methylstat supplemented in the lysis buffer. The BcMag™ Monomer avidin Magnetic Beads (Bioclone) were first prepared according to manufacturer’s instructions and then immediately subjected to RNA (20 µg) capture in RNA capture buffer [20 mM Tris-HCl pH 7.5, 1M NaCl, 1mM EDTA] for 30 minutes at room temperature with agitation. The RNA-captured beads were washed once with NT2 buffer [50 mM Tris-HCl pH 7.4, 150 mM NaCl, 1 mM MgCl2, 0.05% NP-40] and incubated with 30 mg cell lysates diluted in NT2 buffer supplemented with 50 U/mL Anti-RNase, 2 mM dithiothreitol, 30 mM EDTA and Heparin 0.02 mg/ml for 4 hours at 4°C with rotation. The RNA-binding protein complexes were washed sequentially with NT2 buffer (twice), NT2-high salt buffer containing 500 mM NaCl (twice), NT2-high salt buffer containing 1 M NaCl (once), NT2-KSCN buffer containing 750 mM KSCN (twice) and PBS (once) for 5 minutes at 4°C and eluted by 2 mM D-biotin in PBS. The eluted protein complexes were denatured, reduced, alkylated and digested with immobilized trypsin (Promega) for MS analysis at MD Anderson Cancer Center Proteomics Facility. The RNA-protein binding assays and in vitro RNA-protein binding coupled with dot-blot assay were performed as described previously16.

RNA pulldown, mass spectrometry analysis, in vitro RNA-protein binding assay and in vitro RNA-protein binding coupled with dot-blot assay

The cell lysates were freshly prepared using ProteaPrep Zwitterionic Cell Lysis Kit, Mass Spec Grade (Protea®) with Anti-RNase, Protease/ Phosphatase Inhibitor Cocktail, Panobinostat and Methylstat supplemented in the lysis buffer. The BcMag™ Monomer avidin Magnetic Beads (Bioclone) were first prepared according to manufacturer’s instructions and then immediately subjected to RNA (20 µg) capture in RNA capture buffer [20 mM Tris-HCl pH 7.5, 1M NaCl, 1mM EDTA] for 30 minutes at room temperature with agitation. The RNA-captured beads were washed once with NT2 buffer [50 mM Tris-HCl pH 7.4, 150 mM NaCl, 1 mM MgCl2, 0.05% NP-40] and incubated with 30 mg cell lysates diluted in NT2 buffer supplemented with 50 U/mL Anti-RNase, 2 mM dithiothreitol, 30 mM EDTA and Heparin 0.02 mg/ml for 4 hours at 4°C with rotation. The RNA-binding protein complexes were washed sequentially with NT2 buffer (twice), NT2-high salt buffer containing 500 mM NaCl (twice), NT2-high salt buffer containing 1 M NaCl (once), NT2-KSCN buffer containing 750 mM KSCN (twice) and PBS (once) for 5 minutes at 4°C and eluted by 2 mM D-biotin in PBS. The eluted protein complexes were denatured, reduced, alkylated and digested with immobilized trypsin (Promega) for MS analysis at MD Anderson Cancer Center Proteomics Facility. The RNA-protein binding assays and in vitro RNA-protein binding coupled with dot-blot assay were performed as described previously16.

RNAScope® assay, RNA FISH, immunofluorescence and immunohistochemistryRNAScope® assay and RNA FISH were performed as previously described16. RNAScope® robes targeting LINK-A (Cat# 412027), BCAR4 (Cat# 407777) or HOTAIR (Cat# 312347) were custom designed or purchased from Advanced Cell Diagnostics. LNA™ FISH probes targeting LINK-A and control probe targeting Beta-Actin (300512-04) were purchased from Exiqon (sequences were listed in Oligonucleotide sequences, probes and primers section).For immuno-RNA FISH, the slide from RNA FISH was further blocked with blocking buffer [1 × PBS, 5% BSA, 0.3% Triton X-100] for 1 hour at room temperature followed by incubation with primary antibodies (diluted 1:200) for 1 hour at room temperature. After incubation with fluorochrome-conjugated secondary antibodies for 1 hour at room temperature in dark, the slide was washed and mounted for detection. Immunofluorescence and immunohistochemistry were performed as previously described16.The quantification of RNAScope® staining densities was measured by RNAscope® SpotStudio v1.0 Software (Advanced Cell Diagnostics). The quantification of IHC staining density was performed by Image-Pro plus 6.0 (Media Cybernetics) and calculated based on the average staining intensity and the percentage of positively stained cells.

RNAScope® assay, RNA FISH, immunofluorescence and immunohistochemistry

RNAScope® assay and RNA FISH were performed as previously described16. RNAScope® robes targeting LINK-A (Cat# 412027), BCAR4 (Cat# 407777) or HOTAIR (Cat# 312347) were custom designed or purchased from Advanced Cell Diagnostics. LNA™ FISH probes targeting LINK-A and control probe targeting Beta-Actin (300512-04) were purchased from Exiqon (sequences were listed in Oligonucleotide sequences, probes and primers section).

For immuno-RNA FISH, the slide from RNA FISH was further blocked with blocking buffer [1 × PBS, 5% BSA, 0.3% Triton X-100] for 1 hour at room temperature followed by incubation with primary antibodies (diluted 1:200) for 1 hour at room temperature. After incubation with fluorochrome-conjugated secondary antibodies for 1 hour at room temperature in dark, the slide was washed and mounted for detection. Immunofluorescence and immunohistochemistry were performed as previously described16.

The quantification of RNAScope® staining densities was measured by RNAscope® SpotStudio v1.0 Software (Advanced Cell Diagnostics). The quantification of IHC staining density was performed by Image-Pro plus 6.0 (Media Cybernetics) and calculated based on the average staining intensity and the percentage of positively stained cells.

Computational analysis of TCGA RNA-Seq dataBreast cancer RNA-seq BAM files was downloaded from UCSC Cancer Genomics Hub (CGHub, https://cghub.ucsc.edu/). TCGA BAM files were generated based on MapSplice algorithm for alignment against the hg19 reference genome using default parameters56. We then quantified lncRNA expression LINK-A as RPKM (reads per kilobase per million mapped reads)57 as previously described58, and the analysis was based on log2(RPKM+1). Clinical information, PAM50 subtype, ER, PR, and HER2 status, were obtained from the TCGA marker paper1. We used analysis of variance (ANOVA) or student t-test to detect the statistical difference between two or more groups.

Computational analysis of TCGA RNA-Seq data

Breast cancer RNA-seq BAM files was downloaded from UCSC Cancer Genomics Hub (CGHub, https://cghub.ucsc.edu/). TCGA BAM files were generated based on MapSplice algorithm for alignment against the hg19 reference genome using default parameters56. We then quantified lncRNA expression LINK-A as RPKM (reads per kilobase per million mapped reads)57 as previously described58, and the analysis was based on log2(RPKM+1). Clinical information, PAM50 subtype, ER, PR, and HER2 status, were obtained from the TCGA marker paper1. We used analysis of variance (ANOVA) or student t-test to detect the statistical difference between two or more groups.

In vitro kinase assayWild-type or mutant substrate proteins were incubated with 50 µl of in vitro kinase assay buffer II (SignalChem) containing 100 µM ATP (cold reaction) or 10 µCi [γ-32P] ATP and indicated protein kinase for 1 hour at 30°C. Resulting products were separated by SDS-PAGE and detected by Coomassie Blue staining, autoradiography or immunoblotting with phospho-specific antibodies. The specific BRK and LRRK2 kinase activity were measured using a Universal Kinase Activity Kit (R&D Systems).

In vitro kinase assay

Wild-type or mutant substrate proteins were incubated with 50 µl of in vitro kinase assay buffer II (SignalChem) containing 100 µM ATP (cold reaction) or 10 µCi [γ-32P] ATP and indicated protein kinase for 1 hour at 30°C. Resulting products were separated by SDS-PAGE and detected by Coomassie Blue staining, autoradiography or immunoblotting with phospho-specific antibodies. The specific BRK and LRRK2 kinase activity were measured using a Universal Kinase Activity Kit (R&D Systems).

In vitro HIF1α hydroxylation assayFive µg wild-type His-tagged HIF1α or Y565F mutant was incubated with 1 µg recombinant PHD1 in a reaction buffer containing 10 µM FeSO4, 100 µM 2-oxo-glutarate, 1 mM ascorbate, 100 µM dithiothreitol, and 50 µM Tris-HCl (pH 7.8) at 37°C for 1 hour. HIF1α hydroxylation was analyzed by SDS-PAGE and immunoblotting with specific antibody against Pro564 hydroxylation. For the quantitative peptide hydroxylation assay, 10 µg synthesized unmodified peptide (LDLEMLAPYI) or Tyr565-phosphopeptide (LDLEMLAP-pY-I) and 3 µg recombinant PHD1 were incubated in the same reaction buffer described above or in the same buffer, except containing 100 µM DMOG. Resulting peptides were purified by ZipTip and analyzed by liquid chromatography coupled with tandem MS (LC-MS) to confirm the presence of the Proline-hydroxylated peptides. The acquired MS/MS data were searched against a database to identify hydroxylated Proline sites through a dynamic mass shift for the modified Proline (+15.9949 Da).

In vitro HIF1α hydroxylation assay

Five µg wild-type His-tagged HIF1α or Y565F mutant was incubated with 1 µg recombinant PHD1 in a reaction buffer containing 10 µM FeSO4, 100 µM 2-oxo-glutarate, 1 mM ascorbate, 100 µM dithiothreitol, and 50 µM Tris-HCl (pH 7.8) at 37°C for 1 hour. HIF1α hydroxylation was analyzed by SDS-PAGE and immunoblotting with specific antibody against Pro564 hydroxylation. For the quantitative peptide hydroxylation assay, 10 µg synthesized unmodified peptide (LDLEMLAPYI) or Tyr565-phosphopeptide (LDLEMLAP-pY-I) and 3 µg recombinant PHD1 were incubated in the same reaction buffer described above or in the same buffer, except containing 100 µM DMOG. Resulting peptides were purified by ZipTip and analyzed by liquid chromatography coupled with tandem MS (LC-MS) to confirm the presence of the Proline-hydroxylated peptides. The acquired MS/MS data were searched against a database to identify hydroxylated Proline sites through a dynamic mass shift for the modified Proline (+15.9949 Da).

Chromatin immunoprecipitation (ChIP), RNA immunoprecipitation (RIP), ChIP-seq and data analysisChIP and RIP were performed as previously described16. ChIP-seq and data analysis were performed by ArrayStar. The mapped reads were used for peak detection by MACS v1.4.0 (Model-based Analysis of ChIP-Seq) software. Statistically significant ChIP-enriched regions (peaks) were identified by comparison to a Poisson background model (Cut-off p-value=10−4).

Chromatin immunoprecipitation (ChIP), RNA immunoprecipitation (RIP), ChIP-seq and data analysis

ChIP and RIP were performed as previously described16. ChIP-seq and data analysis were performed by ArrayStar. The mapped reads were used for peak detection by MACS v1.4.0 (Model-based Analysis of ChIP-Seq) software. Statistically significant ChIP-enriched regions (peaks) were identified by comparison to a Poisson background model (Cut-off p-value=10−4).

Anchorage-independent growth assay, glucose uptake assay and lactate production assayAnchorage-independent growth assay was performed as previously described59. Glucose uptake and lactate production assay were performed using Glucose Uptake Cell-based Assay Kit and L-Lactate Assay Kit (Cayman Chemical) respectively. Lactate production was expressed as lactate concentration per 104 viable cells.

Anchorage-independent growth assay, glucose uptake assay and lactate production assay

Anchorage-independent growth assay was performed as previously described59. Glucose uptake and lactate production assay were performed using Glucose Uptake Cell-based Assay Kit and L-Lactate Assay Kit (Cayman Chemical) respectively. Lactate production was expressed as lactate concentration per 104 viable cells.

In vivo tumorigenesis studyAll animal experiments were performed in accordance with a protocol approved by the Institutional Animal Care and Use Committee of MD Anderson Cancer Center. Animals arrived in our facility were randomly put into cages with five mice each. They were implanted with respective tumor cells in the unit of cages, which were randomly selected. The animal experiment was set up to use 5 mice per group to detect a 2-fold difference with power of 80% and at the significance level of 0.05 by a two-sided test for significant studies. Tumor cells in 30 µl growth medium (mixed with Matrigel at a 1:1 ratio) were injected subcutaneously into the flank of six- to eight-week-old female nude mice. Tumor size was measured every five days using a caliper, and tumor volume was calculated using the standard formula: 0.54×L×W2, where L is the longest diameter and W is the shortest diameter. The tumors were removed, photographed and weighed. The investigators were not blinded to allocation during experiments and outcome assessment.

In vivo tumorigenesis study

All animal experiments were performed in accordance with a protocol approved by the Institutional Animal Care and Use Committee of MD Anderson Cancer Center. Animals arrived in our facility were randomly put into cages with five mice each. They were implanted with respective tumor cells in the unit of cages, which were randomly selected. The animal experiment was set up to use 5 mice per group to detect a 2-fold difference with power of 80% and at the significance level of 0.05 by a two-sided test for significant studies. Tumor cells in 30 µl growth medium (mixed with Matrigel at a 1:1 ratio) were injected subcutaneously into the flank of six- to eight-week-old female nude mice. Tumor size was measured every five days using a caliper, and tumor volume was calculated using the standard formula: 0.54×L×W2, where L is the longest diameter and W is the shortest diameter. The tumors were removed, photographed and weighed. The investigators were not blinded to allocation during experiments and outcome assessment.

Oligonucleotide sequences (5’-3’), probes (5’-3’) and primers (forward and reverse)qPCR primers for gene expression and RIPLINK-A (TTC CCC CAT TTT TCC TTT TC and CTC TGG TTG GGT GAC TGG TT); GAPDH (GAA GGT GAA GGT CGG AGT and GAA GAT GGT GAT GGG ATT TC); ANGPTL4 (CAC AGC CTG CAG ACA CAA CT and AAA CTG GCT TTG CAG ATG CT); ALDOA (CTG CCA GTA TGT GAC CGA GA and ACA GGA AGG TGA TCC CAG TG); ANKRD37 (GTA GCC AGT GAT GCC CAA AT and CTT CCG AGA CTC CGT TTC TG); BHLHE40 (CCT TGA AGC ATG TGA AAG CA and GCT TGG CCA GAT ACT GAA GC); EGR1 (TGA CCG CAG AGT CTT TTC CT and TGG GTT GGT CAT GCT CAC TA); IGFBP3 (GGG GTG TAC ACA TTC CCA AC and AGG CTG CCC ATA CTT ATC CA); LDHA (TGT GCC TGT ATG GAG TGG AA and AGC ACT CTC AAC CAC CTG CT); MAPK1 (CCA GAG AAC CCT GAG GGA GA and TCG ATG GTT GGT GCT CGA AT); PKM2 (ATC GTC CTC ACC AAG TCT GG and GAA GAT GCC ACG GTA CAG GT); RPLP0 (TGG TCA TCC AGC AGG TGT TCG A and ACA GAC ACT GGC AAC ATT GCG G); HIF1α (GTC TGA GGG GAC AGG AGG AT and CTC CTC AGG TGG CTT GTC AG); B2M (GGC TAT CCA GCG TAC TCC AA and TGG ATG AAA CCC AGA CAC ATA)qPCR primers for LINK-A knockdown-rescue gene expression and RIP experimentsLINK-A (TAT GGA GGA TCG CTG TTT CC and CCA AAG ATG TCG CAG GAC TT)siRNA sequencesLINK-A (UGU CUA AGG UGG AGA UUA C, AGA UGU AGU UCU AGU UCA U, UUA CUG AGG UUG AAU AUG U and GGU CUU CAU UCU UAC GCU U); EGFR (CAA AGU GUG UAA CGG AAU A, CCA UAA AUG CUA CGA AUA U, GUA ACA AGC UCA CGC AGU U, and CAG AGG AUG UUC AAU AAC U); GPNMB (GGA AUU UCA UCU ACG UCU U, AUA UAA CAU UUG CGG UGA A, UGC AAG AAG AGG CGG GAU A, and CCA GAA GAA CGA UCG AAA U); PTK6/BRK (GAG AAA GUC CUG CCC GUU U, CCA UUA AGG UGA UUU CUC G, UGC CCG AGC UUG UGA ACU A, and GGC CAU UAC UCC ACC AAA U), LRRK2 (CAA GUU AUU UCA AGG CAA A, UUA CCG AGA UGC CGU AUU A, GGA GGG AUC UUC UUU AAU U, and GAA AUU AUC AUC CGA CUA U)LNA™ GapmeR sequencesNegative control (AAC ACG TCT ATA CGC); LINK-A #1 (GCG TAA GAA TGA AGA); LINK-A #2 (GTG ATA AGA CTA AGT G); LINK-A #3(GAA TAA GGA TAA GCG T); LINK-A #4 (CCA CAG CTT GAA TTC C) and LINK-A #5(TGG ATA AAT GAG CTG T)LINK-A shRNA sequencesLINK-A sh#1 (TGT CTA AGG TGG AGA TTA C); LINK-A sh#2 (AGA TGT AGT TCT AGT TCA T); LINK-A sh#3 (TTA CTG AGG TTG AAT ATG T); LINK-A sh#4 (GGT CTT CAT TCT TAC GCT T).RNA FISH probesLINK-A (/56-FAM/TGT AGC CAC AGA CAT CAT TAC A), beta-Actin (/Fluorescein/CTC ATT GTA GAA GGT GTG GTG CCA).LNA™ RNA detection probesBeta-Actin (/5Biosg/CTC ATT GTA GAA GGT GTG GTG CCA) and LINK-A (/5Biosg/ ACT AAG TGT TGG CAG GTT ATG T).LINK-A RACE-PCR primers3’-RACE (TGG AAT TCA AGC TGT GGG TG) and 5’-RACE (GCA TTT TTA TTT TAA TTG AGG)qPCR primers for ChIPEGLN3 (CGT GGA GGA CTG GCT CTA AG and GGT GTG CTC GGG TGT G)60, ERRFI1 (GAT TAC AGG CTG GAT GGC AC and TGC TGC CAG ACT GGT ATG AG)61, ARRDC3 (CCC CTG CAG TCA CAC ACT C and TTT GTC ACA TGG GAC TCT TC)31, ANKRD37 (CCA GTT TCC TGG TTA CGT GC and TAA GTC AGT GGG CGT GAG AG)62, RPLP0 (TGA AGA GCA GAG GCG ACC CAC and ATG GGT GTC GGC GTG AC)60.

Oligonucleotide sequences (5’-3’), probes (5’-3’) and primers (forward and reverse)

qPCR primers for gene expression and RIPLINK-A (TTC CCC CAT TTT TCC TTT TC and CTC TGG TTG GGT GAC TGG TT); GAPDH (GAA GGT GAA GGT CGG AGT and GAA GAT GGT GAT GGG ATT TC); ANGPTL4 (CAC AGC CTG CAG ACA CAA CT and AAA CTG GCT TTG CAG ATG CT); ALDOA (CTG CCA GTA TGT GAC CGA GA and ACA GGA AGG TGA TCC CAG TG); ANKRD37 (GTA GCC AGT GAT GCC CAA AT and CTT CCG AGA CTC CGT TTC TG); BHLHE40 (CCT TGA AGC ATG TGA AAG CA and GCT TGG CCA GAT ACT GAA GC); EGR1 (TGA CCG CAG AGT CTT TTC CT and TGG GTT GGT CAT GCT CAC TA); IGFBP3 (GGG GTG TAC ACA TTC CCA AC and AGG CTG CCC ATA CTT ATC CA); LDHA (TGT GCC TGT ATG GAG TGG AA and AGC ACT CTC AAC CAC CTG CT); MAPK1 (CCA GAG AAC CCT GAG GGA GA and TCG ATG GTT GGT GCT CGA AT); PKM2 (ATC GTC CTC ACC AAG TCT GG and GAA GAT GCC ACG GTA CAG GT); RPLP0 (TGG TCA TCC AGC AGG TGT TCG A and ACA GAC ACT GGC AAC ATT GCG G); HIF1α (GTC TGA GGG GAC AGG AGG AT and CTC CTC AGG TGG CTT GTC AG); B2M (GGC TAT CCA GCG TAC TCC AA and TGG ATG AAA CCC AGA CAC ATA)

qPCR primers for gene expression and RIP

LINK-A (TTC CCC CAT TTT TCC TTT TC and CTC TGG TTG GGT GAC TGG TT); GAPDH (GAA GGT GAA GGT CGG AGT and GAA GAT GGT GAT GGG ATT TC); ANGPTL4 (CAC AGC CTG CAG ACA CAA CT and AAA CTG GCT TTG CAG ATG CT); ALDOA (CTG CCA GTA TGT GAC CGA GA and ACA GGA AGG TGA TCC CAG TG); ANKRD37 (GTA GCC AGT GAT GCC CAA AT and CTT CCG AGA CTC CGT TTC TG); BHLHE40 (CCT TGA AGC ATG TGA AAG CA and GCT TGG CCA GAT ACT GAA GC); EGR1 (TGA CCG CAG AGT CTT TTC CT and TGG GTT GGT CAT GCT CAC TA); IGFBP3 (GGG GTG TAC ACA TTC CCA AC and AGG CTG CCC ATA CTT ATC CA); LDHA (TGT GCC TGT ATG GAG TGG AA and AGC ACT CTC AAC CAC CTG CT); MAPK1 (CCA GAG AAC CCT GAG GGA GA and TCG ATG GTT GGT GCT CGA AT); PKM2 (ATC GTC CTC ACC AAG TCT GG and GAA GAT GCC ACG GTA CAG GT); RPLP0 (TGG TCA TCC AGC AGG TGT TCG A and ACA GAC ACT GGC AAC ATT GCG G); HIF1α (GTC TGA GGG GAC AGG AGG AT and CTC CTC AGG TGG CTT GTC AG); B2M (GGC TAT CCA GCG TAC TCC AA and TGG ATG AAA CCC AGA CAC ATA)

qPCR primers for LINK-A knockdown-rescue gene expression and RIP experimentsLINK-A (TAT GGA GGA TCG CTG TTT CC and CCA AAG ATG TCG CAG GAC TT)

qPCR primers for LINK-A knockdown-rescue gene expression and RIP experiments

LINK-A (TAT GGA GGA TCG CTG TTT CC and CCA AAG ATG TCG CAG GAC TT)

siRNA sequencesLINK-A (UGU CUA AGG UGG AGA UUA C, AGA UGU AGU UCU AGU UCA U, UUA CUG AGG UUG AAU AUG U and GGU CUU CAU UCU UAC GCU U); EGFR (CAA AGU GUG UAA CGG AAU A, CCA UAA AUG CUA CGA AUA U, GUA ACA AGC UCA CGC AGU U, and CAG AGG AUG UUC AAU AAC U); GPNMB (GGA AUU UCA UCU ACG UCU U, AUA UAA CAU UUG CGG UGA A, UGC AAG AAG AGG CGG GAU A, and CCA GAA GAA CGA UCG AAA U); PTK6/BRK (GAG AAA GUC CUG CCC GUU U, CCA UUA AGG UGA UUU CUC G, UGC CCG AGC UUG UGA ACU A, and GGC CAU UAC UCC ACC AAA U), LRRK2 (CAA GUU AUU UCA AGG CAA A, UUA CCG AGA UGC CGU AUU A, GGA GGG AUC UUC UUU AAU U, and GAA AUU AUC AUC CGA CUA U)

siRNA sequences

LINK-A (UGU CUA AGG UGG AGA UUA C, AGA UGU AGU UCU AGU UCA U, UUA CUG AGG UUG AAU AUG U and GGU CUU CAU UCU UAC GCU U); EGFR (CAA AGU GUG UAA CGG AAU A, CCA UAA AUG CUA CGA AUA U, GUA ACA AGC UCA CGC AGU U, and CAG AGG AUG UUC AAU AAC U); GPNMB (GGA AUU UCA UCU ACG UCU U, AUA UAA CAU UUG CGG UGA A, UGC AAG AAG AGG CGG GAU A, and CCA GAA GAA CGA UCG AAA U); PTK6/BRK (GAG AAA GUC CUG CCC GUU U, CCA UUA AGG UGA UUU CUC G, UGC CCG AGC UUG UGA ACU A, and GGC CAU UAC UCC ACC AAA U), LRRK2 (CAA GUU AUU UCA AGG CAA A, UUA CCG AGA UGC CGU AUU A, GGA GGG AUC UUC UUU AAU U, and GAA AUU AUC AUC CGA CUA U)

LNA™ GapmeR sequencesNegative control (AAC ACG TCT ATA CGC); LINK-A #1 (GCG TAA GAA TGA AGA); LINK-A #2 (GTG ATA AGA CTA AGT G); LINK-A #3(GAA TAA GGA TAA GCG T); LINK-A #4 (CCA CAG CTT GAA TTC C) and LINK-A #5(TGG ATA AAT GAG CTG T)

LNA™ GapmeR sequences

Negative control (AAC ACG TCT ATA CGC); LINK-A #1 (GCG TAA GAA TGA AGA); LINK-A #2 (GTG ATA AGA CTA AGT G); LINK-A #3(GAA TAA GGA TAA GCG T); LINK-A #4 (CCA CAG CTT GAA TTC C) and LINK-A #5(TGG ATA AAT GAG CTG T)

LINK-A shRNA sequencesLINK-A sh#1 (TGT CTA AGG TGG AGA TTA C); LINK-A sh#2 (AGA TGT AGT TCT AGT TCA T); LINK-A sh#3 (TTA CTG AGG TTG AAT ATG T); LINK-A sh#4 (GGT CTT CAT TCT TAC GCT T).

LINK-A shRNA sequences

LINK-A sh#1 (TGT CTA AGG TGG AGA TTA C); LINK-A sh#2 (AGA TGT AGT TCT AGT TCA T); LINK-A sh#3 (TTA CTG AGG TTG AAT ATG T); LINK-A sh#4 (GGT CTT CAT TCT TAC GCT T).

RNA FISH probesLINK-A (/56-FAM/TGT AGC CAC AGA CAT CAT TAC A), beta-Actin (/Fluorescein/CTC ATT GTA GAA GGT GTG GTG CCA).

RNA FISH probes

LINK-A (/56-FAM/TGT AGC CAC AGA CAT CAT TAC A), beta-Actin (/Fluorescein/CTC ATT GTA GAA GGT GTG GTG CCA).

LNA™ RNA detection probesBeta-Actin (/5Biosg/CTC ATT GTA GAA GGT GTG GTG CCA) and LINK-A (/5Biosg/ ACT AAG TGT TGG CAG GTT ATG T).

LNA™ RNA detection probes

Beta-Actin (/5Biosg/CTC ATT GTA GAA GGT GTG GTG CCA) and LINK-A (/5Biosg/ ACT AAG TGT TGG CAG GTT ATG T).

LINK-A RACE-PCR primers3’-RACE (TGG AAT TCA AGC TGT GGG TG) and 5’-RACE (GCA TTT TTA TTT TAA TTG AGG)

LINK-A RACE-PCR primers

3’-RACE (TGG AAT TCA AGC TGT GGG TG) and 5’-RACE (GCA TTT TTA TTT TAA TTG AGG)

qPCR primers for ChIPEGLN3 (CGT GGA GGA CTG GCT CTA AG and GGT GTG CTC GGG TGT G)60, ERRFI1 (GAT TAC AGG CTG GAT GGC AC and TGC TGC CAG ACT GGT ATG AG)61, ARRDC3 (CCC CTG CAG TCA CAC ACT C and TTT GTC ACA TGG GAC TCT TC)31, ANKRD37 (CCA GTT TCC TGG TTA CGT GC and TAA GTC AGT GGG CGT GAG AG)62, RPLP0 (TGA AGA GCA GAG GCG ACC CAC and ATG GGT GTC GGC GTG AC)60.

qPCR primers for ChIP

EGLN3 (CGT GGA GGA CTG GCT CTA AG and GGT GTG CTC GGG TGT G)60, ERRFI1 (GAT TAC AGG CTG GAT GGC AC and TGC TGC CAG ACT GGT ATG AG)61, ARRDC3 (CCC CTG CAG TCA CAC ACT C and TTT GTC ACA TGG GAC TCT TC)31, ANKRD37 (CCA GTT TCC TGG TTA CGT GC and TAA GTC AGT GGG CGT GAG AG)62, RPLP0 (TGA AGA GCA GAG GCG ACC CAC and ATG GGT GTC GGC GTG AC)60.

Statistics & ReproducibilityThe experiment was set up to use 3–5 samples/repeats per experiment/group/condition to detect a 2-fold difference with power of 80% and at the significance level of 0.05 by a two-sided test for significant studies. For RNAscope®, immunostaining, immunohistochemical staining, colony formation assay, northern blotting, RACE analysis, and western blotting, the representative images are shown. Each of these experiments was independently repeated for 3–5 times. Relative quantities of gene expression level were normalized to B2M. The relative quantities of ChIP samples were normalized by individual inputs, respectively. Results are reported as mean ± standard error of the mean (S.E.M.) of at least three independent experiments. Each exact n value is indicated in the corresponding figure legend. Comparisons were performed using two tailed paired Student’s t test or two-way ANOVA (*p < 0.05, **p < 0.01, and ***p < 0.001), as indicated in individual figures. Fisher’s exact test was implemented for statistical analyses of the correlation between markers and clinical parameters. For survival analysis, the expression of LINK-A or phosphorylation density of indicated proteins was treated as a binary variant and divided into ‘high’ and ‘low’ level. Kaplan-Meier survival curves were compared using the Gehan-Breslow test with GraphPad Prism (GraphPad Software). The experiments were not randomized. The investigators were not blinded to allocation during experiments and outcome assessment.

Statistics & Reproducibility

The experiment was set up to use 3–5 samples/repeats per experiment/group/condition to detect a 2-fold difference with power of 80% and at the significance level of 0.05 by a two-sided test for significant studies. For RNAscope®, immunostaining, immunohistochemical staining, colony formation assay, northern blotting, RACE analysis, and western blotting, the representative images are shown. Each of these experiments was independently repeated for 3–5 times. Relative quantities of gene expression level were normalized to B2M. The relative quantities of ChIP samples were normalized by individual inputs, respectively. Results are reported as mean ± standard error of the mean (S.E.M.) of at least three independent experiments. Each exact n value is indicated in the corresponding figure legend. Comparisons were performed using two tailed paired Student’s t test or two-way ANOVA (*p < 0.05, **p < 0.01, and ***p < 0.001), as indicated in individual figures. Fisher’s exact test was implemented for statistical analyses of the correlation between markers and clinical parameters. For survival analysis, the expression of LINK-A or phosphorylation density of indicated proteins was treated as a binary variant and divided into ‘high’ and ‘low’ level. Kaplan-Meier survival curves were compared using the Gehan-Breslow test with GraphPad Prism (GraphPad Software). The experiments were not randomized. The investigators were not blinded to allocation during experiments and outcome assessment.

Supplementary Material12345678

Supplementary Material
