---
pmid: '31461456'
title: A Sir2-regulated locus control region in the recombination enhancer of Saccharomyces
  cerevisiae specifies chromosome III structure.
authors:
- Li M
- Fine RD
- Dinda M
- Bekiranov S
- Smith JS
journal: PLoS Genet
year: '2019'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC6736312
doi: 10.1371/journal.pgen.1008339
---

# A Sir2-regulated locus control region in the recombination enhancer of Saccharomyces cerevisiae specifies chromosome III structure.
**Authors:** Li M, Fine RD, Dinda M, Bekiranov S, Smith JS
**Journal:** PLoS Genet (2019)
**DOI:** [10.1371/journal.pgen.1008339](https://doi.org/10.1371/journal.pgen.1008339)
**PMC:** [PMC6736312](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6736312/)

## Abstract

1. PLoS Genet. 2019 Aug 28;15(8):e1008339. doi: 10.1371/journal.pgen.1008339. 
eCollection 2019 Aug.

A Sir2-regulated locus control region in the recombination enhancer of 
Saccharomyces cerevisiae specifies chromosome III structure.

Li M(1), Fine RD(2), Dinda M(2), Bekiranov S(2), Smith JS(2).

Author information:
(1)Department of Laboratory Medicine, Jilin Medical University, Jilin, China.
(2)Department of Biochemistry and Molecular Genetics, University of Virginia 
School of Medicine, Charlottesville, Virginia, United States of America.

The NAD+-dependent histone deacetylase Sir2 was originally identified in 
Saccharomyces cerevisiae as a silencing factor for HML and HMR, the 
heterochromatic cassettes utilized as donor templates during mating-type 
switching. MATa cells preferentially switch to MATα using HML as the donor, 
which is driven by an adjacent cis-acting element called the recombination 
enhancer (RE). In this study we demonstrate that Sir2 and the condensin complex 
are recruited to the RE exclusively in MATa cells, specifically to the promoter 
of a small gene within the right half of the RE known as RDT1. We also provide 
evidence that the RDT1 promoter functions as a locus control region (LCR) that 
regulates both transcription and long-range chromatin interactions. Sir2 
represses RDT1 transcription until it is removed from the promoter in response 
to a dsDNA break at the MAT locus induced by HO endonuclease during mating-type 
switching. Condensin is also recruited to the RDT1 promoter and is displaced 
upon HO induction, but does not significantly repress RDT1 transcription. 
Instead condensin appears to promote mating-type donor preference by maintaining 
proper chromosome III architecture, which is defined by the interaction of HML 
with the right arm of chromosome III, including MATa and HMR. Remarkably, 
eliminating Sir2 and condensin recruitment to the RDT1 promoter disrupts this 
structure and reveals an aberrant interaction between MATa and HMR, consistent 
with the partially defective donor preference for this mutant. Global condensin 
subunit depletion also impairs mating-type switching efficiency and donor 
preference, suggesting that modulation of chromosome architecture plays a 
significant role in controlling mating-type switching, thus providing a novel 
model for dissecting condensin function in vivo.

DOI: 10.1371/journal.pgen.1008339
PMCID: PMC6736312
PMID: 31461456 [Indexed for MEDLINE]

Conflict of interest statement: The authors have declared that no competing 
interests exist.

## Full Text

IntroductionSince the first descriptions of mating-type switching in budding yeast over 40 years ago, characterization of this process has led to numerous advances in understanding mechanisms of gene silencing (heterochromatin), cell-fate determination (mating-type), and homologous recombination (reviewed in [1]). For example, the NAD+-dependent histone deacetylase, Sir2, and other Silent Information Regulator (SIR) proteins, were genetically identified due to their roles in silencing the heterochromatic HML and HMR loci, which are maintained as silenced copies of the active MATα and MATa loci, respectively [2–4]. The SIR silencing complex (Sir2-Sir3-Sir4) is recruited to cis-acting E and I silencer elements flanking HML and HMR through physical interactions with silencer binding factors Rap1, ORC, and Abf1, as well as histones H3 and H4 (reviewed in [5]).HML and HMR play a critical role in mating-type switching. Haploid cells of the same mating-type cannot mate to form diploids, the preferred cell type in the wild. Therefore, in order to facilitate mating and diploid formation, haploid mother cells switch their mating-type by expressing HO endonuclease, which introduces a programmed DNA double-strand break (DSB) at the MAT locus [6]. The break is then repaired by homologous recombination using either HML or HMR as a donor template for gene conversion [6, 7]. This change in mating-type enables immediate diploid formation between mother and daughter. HO is deleted from most standard lab strains in order to maintain them as haploids, so expression of HO from an inducible promoter such as PGAL1 is commonly used to switch mating-types during strain construction [8].There is a “donor preference” directionality to mating-type switching such that ~90% of the time, the HO-induced DSB is repaired to the opposite mating-type [9]. For example, MATα cells preferentially switch to MATa using HMR as the donor. However, while both silent mating loci can be utilized as a donor template, usage of HML by MATa cells requires a 2.5 kb intergenic region located ~17 kb from HML called the recombination enhancer (RE) [10]. Donor preference activity within the RE has been further narrowed down to a 700 bp segment containing an Mcm1/α2 binding site (DPS1) and multiple Fkh1 binding sites [10]. The RE is active in MATa cells, requiring Mcm1 and Fkh1 activity at their respective binding sites [10–12]. The RE is inactivated in MATα cells due to expression of transcription factor α2 from MATα [13], which forms a repressive heterodimer with Mcm1 (Mcm1/α2) to repress MATa-specific genes [1]. Current models for donor preference posit that Fkh1 at the RE helps position HML in close proximity with MATa by interacting with threonine-phosphorylated H2A (γ-H2AX) and Mph1 DNA helicase at the HO-induced DNA DSB [14, 15].Sir2-dependent silencing of HML and HMR has two known functions related to mating-type switching. First, HML and HMR must be silenced in haploids to prevent formation of the a1/α2 heterodimer, which would otherwise inactivate haploid-specific genes such as HO [16]. Second, heterochromatin structure at HML and HMR blocks cleavage by HO, thus restricting its activity to the fully accessible MAT locus [17, 18]. Here we describe new roles for Sir2 and the condensin complex within the RE during mating-type switching. ChIP-seq analysis revealed strong overlapping binding sites for Sir2 and condensin at the promoter of a small gene within the RE known as RDT1. Here, Sir2 was found to repress the MATa-specific transcription of RDT1, which is also translated into a small 28 amino acid peptide. RDT1 expression is also dramatically upregulated during mating-type switching when Sir2 is lost from the RDT1 promoter and instead associates with the HO-induced DNA DSB at MATa. Furthermore, eliminating Sir2/condensin recruitment to the RDT1 promoter disrupts chromosome III architecture such that donor preference is partially impaired. The RDT1 promoter region therefore functions like a classic locus control region (LCR) in MATa yeast cells, regulating localized transcription as well as long-range chromosome interactions.

Introduction

Since the first descriptions of mating-type switching in budding yeast over 40 years ago, characterization of this process has led to numerous advances in understanding mechanisms of gene silencing (heterochromatin), cell-fate determination (mating-type), and homologous recombination (reviewed in [1]). For example, the NAD+-dependent histone deacetylase, Sir2, and other Silent Information Regulator (SIR) proteins, were genetically identified due to their roles in silencing the heterochromatic HML and HMR loci, which are maintained as silenced copies of the active MATα and MATa loci, respectively [2–4]. The SIR silencing complex (Sir2-Sir3-Sir4) is recruited to cis-acting E and I silencer elements flanking HML and HMR through physical interactions with silencer binding factors Rap1, ORC, and Abf1, as well as histones H3 and H4 (reviewed in [5]).

HML and HMR play a critical role in mating-type switching. Haploid cells of the same mating-type cannot mate to form diploids, the preferred cell type in the wild. Therefore, in order to facilitate mating and diploid formation, haploid mother cells switch their mating-type by expressing HO endonuclease, which introduces a programmed DNA double-strand break (DSB) at the MAT locus [6]. The break is then repaired by homologous recombination using either HML or HMR as a donor template for gene conversion [6, 7]. This change in mating-type enables immediate diploid formation between mother and daughter. HO is deleted from most standard lab strains in order to maintain them as haploids, so expression of HO from an inducible promoter such as PGAL1 is commonly used to switch mating-types during strain construction [8].

There is a “donor preference” directionality to mating-type switching such that ~90% of the time, the HO-induced DSB is repaired to the opposite mating-type [9]. For example, MATα cells preferentially switch to MATa using HMR as the donor. However, while both silent mating loci can be utilized as a donor template, usage of HML by MATa cells requires a 2.5 kb intergenic region located ~17 kb from HML called the recombination enhancer (RE) [10]. Donor preference activity within the RE has been further narrowed down to a 700 bp segment containing an Mcm1/α2 binding site (DPS1) and multiple Fkh1 binding sites [10]. The RE is active in MATa cells, requiring Mcm1 and Fkh1 activity at their respective binding sites [10–12]. The RE is inactivated in MATα cells due to expression of transcription factor α2 from MATα [13], which forms a repressive heterodimer with Mcm1 (Mcm1/α2) to repress MATa-specific genes [1]. Current models for donor preference posit that Fkh1 at the RE helps position HML in close proximity with MATa by interacting with threonine-phosphorylated H2A (γ-H2AX) and Mph1 DNA helicase at the HO-induced DNA DSB [14, 15].

Sir2-dependent silencing of HML and HMR has two known functions related to mating-type switching. First, HML and HMR must be silenced in haploids to prevent formation of the a1/α2 heterodimer, which would otherwise inactivate haploid-specific genes such as HO [16]. Second, heterochromatin structure at HML and HMR blocks cleavage by HO, thus restricting its activity to the fully accessible MAT locus [17, 18]. Here we describe new roles for Sir2 and the condensin complex within the RE during mating-type switching. ChIP-seq analysis revealed strong overlapping binding sites for Sir2 and condensin at the promoter of a small gene within the RE known as RDT1. Here, Sir2 was found to repress the MATa-specific transcription of RDT1, which is also translated into a small 28 amino acid peptide. RDT1 expression is also dramatically upregulated during mating-type switching when Sir2 is lost from the RDT1 promoter and instead associates with the HO-induced DNA DSB at MATa. Furthermore, eliminating Sir2/condensin recruitment to the RDT1 promoter disrupts chromosome III architecture such that donor preference is partially impaired. The RDT1 promoter region therefore functions like a classic locus control region (LCR) in MATa yeast cells, regulating localized transcription as well as long-range chromosome interactions.

ResultsSir2 and condensin associate with the recombination enhancer (RE)We previously characterized global sirtuin distribution using ChIP-Seq to identify novel loci regulated by Sir2 and its homologs [19]. Significant overlap was observed between binding sites for Sir2, Hst1, or Sum1 with previously described condensin binding sites [19, 20], suggesting a possible functional connection. ChIP-Seq was therefore performed on WT and sir2Δ strains in which the condensin subunit Smc4 was C-terminally tagged (13xMyc) (Fig 1A). To avoid “hyper-ChIPable” loci that can appear in yeast ChIP-seq experiments, we also ran nuclear localized GFP controls [21]. Genes closest to Sir2-dependent condensin peaks after subtraction of GFP are listed in S1 Table, and are distributed throughout the genome. One of the strongest peaks overlapped with a Sir2-myc binding site on chromosome III between KAR4 and SPB1 that was not enriched for GFP (Fig 1A). The specificity of Sir2 enrichment at this peak, as opposed to the adjacent SPB1 gene, was independently confirmed by quantitative ChIP using an α-Sir2 antibody (Fig 1B), with enrichment comparable to levels observed at the HML-I silencer (Fig 1A and 1B). Sir2-dependent condensin binding was also confirmed for Myc-tagged Smc4 and Brn1 subunits (Fig 1C). The ~2.5 kb intergenic region between KAR4 and SPB1 was previously defined as a cis-acting recombination enhancer (RE) that specifies donor preference of mating-type switching in MATa cells [10, 13]. Quantitative ChIP assays revealed that Sir2 and Brn1-myc enrichment at the RE was also MATa-specific (Fig 1D and 1E), which was notable because the ChIP-seq datasets in Fig 1A happened to be generated from MATa strains. We next considered whether the condensin binding defect in the MATa sir2Δ mutant was due to HMLALPHA2 expression caused by defective HML silencing. To test this idea, we re-examined Brn1-myc ChIP signal at the RE in strains lacking HML, and found that deleting SIR2 no longer affected condensin recruitment (Fig 1F). Similarly, a MATa condensin mutant (ycs4-1) known to have an HML silencing defect [22] reduced Sir2 recruitment to the RE, but had no effect when HML was also deleted (Fig 1G). Sir2 and condensin are therefore independently recruited to the RE only in MATa cells.10.1371/journal.pgen.1008339.g001Fig 1MATa-specific binding of Sir2 and condensin to the recombination enhancer (RE).(A) ChIP-seq of Smc4-myc, Sir2-myc, and nuclear localized GFP in WT and sir2Δ backgrounds. The left arm of chromosome III is depicted from HML to SPB1. RE indicates the recombination enhancer region. Inset: The minimal 700bp RE element required for donor preference is indicated, as are the two Mcm1/α2 binding sites (DPS1 and DPS2) and RDT1. (B) Sir2 ChIP at the RE, HML-I silencer, and SPB1. (C)
α-Myc ChIP of Brn1-myc and Smc4-myc at the RE. (D) ChIP showing MATa-specific binding of Sir2 to the RE. (E) ChIP showing MATa-specific binding of Brn1-myc to the RE. (F) Brn1-Myc ChIP at RE is not Sir2 dependent. (G) Native Sir2 ChIP at RE is not condensin dependent. The ChIP signal in each panel is relative to input and plotted as the mean of three replicates. Error bars = standard deviation. (**p<0.005).Sir2 regulates a small gene (RDT1) within the REDonor preference activity ascribed to the RE was previously narrowed down to a KAR4 (YCL055W)-proximal 700 bp domain defined by an Mcm1/α2 binding site (Fig 2A, DPS1) [10, 11, 13]. The Sir2 and condensin ChIP-seq peaks we identified were located outside this region, between a second Mcm1/α2 binding site (DPS2) and a small gene of unknown function called RDT1 [23] (Figs 1A and 2A). We noticed the location of RDT1 coincided with the smallest of several putative non-coding RNAs (ncRNA) previously reported as being transcribed from the RE, but not annotated in SGD (Fig 2A, [13]). Quantitative RT-PCR and analysis of publicly available RNA-seq data from BY4741 (MATa) and BY4742 (MATα) revealed that RDT1 expression was indeed MATa specific (Fig 2B and S1A Fig).10.1371/journal.pgen.1008339.g002Fig 2RDT1 is a novel Sir2 regulated gene.(A) Schematic of the RE locus depicting Sir2/Condensin peak locations relative to previously reported R1L/S and R2 (RDT1) RNAs. (B)
RDT1 mRNA expression is MATa specific. RDT1 expression level is calculated relative to the ACT1 control. This RDT1/ACT1 ratio is then normalized to 1.0 for the MATa strain. (C) H4K16ac ChIP signal at RE in SIR complex null strains, relative to input signal. WT is normalized to 1.0. (D) H4K16 deacetylation is dependent on Sir2 catalytic activity. A sir2Δ strain was transformed with the indicated plasmids and ChIP assays performed. ChIP signal is relative to input. (E) Effect of sir2Δ on H3K9/K14ac ChIP at the RDT1 promoter in HML and hmlΔ backgrounds. ChIP signal is relative to input. (F) Differential RDT1 transcriptional regulation by SIR2 is dependent on HML status. RDT1 expression is relative to the ACT1 control, and the RDT1/ACT1 ratio is normalized to 1.0 for WT. (G) Effects of the ycs4-1 mutation on RDT1 expression in HML and hmlΔ backgrounds at 30°C. (*p<0.05; **p<0.005). RDT1 expression is plotted as for panel F.We next asked whether Sir2 and/or condensin regulate histone acetylation and RDT1 expression when recruited to the RE. Sir2 normally represses transcription at HML, HMR, and telomeres as a catalytic subunit of the SIR complex where it preferentially deacetylates H4K16 (reviewed in [5]). Accordingly, deleting SIR2, SIR3, or SIR4 from MATa cells increased H4K16 acetylation at the RDT1 promoter (Fig 2C), consistent with the observed enrichment of Sir3-myc and Sir4-myc at this site (S1B Fig). Furthermore, re-introducing active SIR2 into the sir2Δ mutant restored H4K16 to the hypoacetylated state, whereas catalytically inactive sir2-H364Y did not (Fig 2D). While H4K16ac is a preferred Sir2 substrate for silencing at HML and HMR, the recruited SIR complex also maintains lysine deacetylation of the other N-terminal histone tails [24]. Since RDT1 transcription is MATa specific, and Mcm1/α2 represses MATa-specific genes by recruiting the Ssn6/Tup1 corepressor complex and Class I/II HDACs such as the H3/H2B-specific histone deacetylase HDA1 [25, 26], we also tested the effect of deleting SIR2 on H3K9/14 acetylation at the RDT1 promoter, predicting it may remain hypoacetylated due to the SIR complex being replaced by Ssn6/Tup1/HDA1. Indeed, H3K9/14 acetylation was reduced in the sir2Δ mutant relative to WT, but was significantly elevated in the hmlΔ sir2Δ double mutant (Fig 2E). RDT1 expression was similarly reduced in the sir2Δ mutant and strongly upregulated when HML and SIR2 were both deleted (Fig 2F). An even stronger expression effect was observed in an hmlΔ sir2Δ hst1Δ triple mutant that eliminates any possibility of redundancy between the Sir2 and Hst1 paralogs (S1C Fig). On the other hand, RDT1 was not upregulated in an hmlΔ ycs4-1 condensin mutant (Fig 2G). Taken together, these results provide strong evidence that the SIR complex represses RDT1 in MATa cells by establishing a generally hypoacetylated chromatin environment at the promoter, while condensin has a functional role independent of transcriptional regulation.We next attempted to prevent Sir2 and condensin recruitment to the RDT1 promoter by precisely deleting a 100bp DNA sequence underlying the shared enrichment region (coordinates 30701–30800), while not disturbing the adjacent Mcm1/α2 site (Fig 3A). Sir2 and Brn1-myc binding to the RE as measured by ChIP was greatly diminished in this mutant (Fig 3B and 3C), despite unaltered Sir2, Brn1-myc, or Smc4-myc expression levels (S2A–S2C Fig). Furthermore, RDT1 RNA expression level was significantly increased by the 100bp deletion exclusively in MATa cells (Fig 3D), consistent with the loss of Sir2-mediated repression.10.1371/journal.pgen.1008339.g003Fig 3Identification of a 100bp sequence that recruits Sir2/condensin and represses RDT1 expression.(A) Schematic indicating a 100bp deletion that covers the condensin (red) and Sir2 (blue) peaks. (B) ChIP of Sir2 in the 100bpΔ mutant (ML275). (C) ChIP of Brn1-Myc in the 100bpΔ mutant. ChIP PCR signal in panels B and C is plotted relative to the input signal. (D)
RDT1 transcription in MATa cells is derepressed in the 100bpΔ mutant. The RDT1 level is plotted relative to ACT1 control, and then normalized to 1.0 for the WT MATa strain. (E) Western blot of Rdt1-13xMyc in WT MATα and MATa cells, as well as the MATa 100bpΔ mutant. Predicted fusion protein molecular weight is 19.1 kDa, but it runs ~37 kDa. (F) Comparative RDT1 and R1L/S expression levels in WT, sir2Δ, and 100bpΔ strains. One of the WT RDT1 qPCR replicates was normalized to 1.0 to allow direct comparisons between RDT1 and R1L/S levels. (G) Comparative RDT1, R1L/S, and SPB1 expression levels in WT, sir2Δ, and 100bpΔ strains. As in panel F, one of the WT RDT1 qPCR replicates was normalized to 1.0. (*p<0.05; **p<0.005, compared to WT strain).Because Sir2 and condensin were not present at the RDT1 promoter in MATα cells, we reasoned that their binding should require a MATa specific transcription factor. This made the 2nd Mcm1/α2 binding site (DPS2) upstream of the Sir2/condensin ChIP-seq peaks an ideal candidate, because it had not previously been ascribed a function other than redundancy with DPS1 for donor preference [13]. Deleting MCM1 is lethal, so alternatively, we precisely deleted the 2nd Mcm1/α2 binding site (ChrIII coordinates 30595 to 30626, S3A Fig) and then retested for Sir2 and Brn1-myc enrichment. As shown in S3B and S3C Fig, respectively, Sir2 and Brn1-myc enrichment at the Mcm1/α2 binding site (DPS2) and the RDT1 promoter (defined as the Sir2/condensin peaks) was significantly reduced in the binding site mutant. These results suggest that Mcm1 nucleates a complex that recruits the SIR and condensin complexes to the RDT1 promoter in MATa cells, and also provides a possible mechanism of blocking the recruitment in MATα cells due to the interaction of Mcm1 with α2.RDT1 encodes a translated mRNARibosome Detected Transcript-1 (RDT1) was originally annotated as a newly evolved gene whose transcript was associated with ribosomes and predicted to have a small open reading frame of 28 amino acids [23]. Our work suggested that RDT1 and the putative non-coding R2 transcript were the same (Fig 2A). To determine if RDT1/R2 codes for a small protein, the ORF was C-terminally fused with a 13x-Myc epitope in MATa and MATα cells. As shown in Fig 3E, a fusion protein was exclusively detected in exponentially growing MATa WT cells and also highly expressed in the 100bpΔ strain, correlating with the increased RNA level observed for that mutant in Fig 3D.Since additional MATa-specific RNAs are derived from the minimal 700bp RE domain (Fig 2A; R1L and R1S) [13, 27], we next tested whether Sir2/condensin recruitment to the RDT1 promoter had any effect on expression of these upstream ncRNAs from a distance. Quantitative RT-PCR for the R1L/S transcripts indicated their expression level in the WT strain was comparable to RDT1, and was also reduced in a sir2Δ mutant because of the pseudodiploid phenotype (Fig 3F). However, while RDT1 was strongly upregulated by the 100bp deletion of the Sir2/condensin binding site, R1L/S expression was unaffected (Fig 3F). As a control, we also measured expression of the SPB1 gene located immediately downstream of RDT1 (see Fig 2A schematic). SPB1 expression is not mating-type specific, it encodes a rRNA methyltransferase required for maturation of the large 60S ribosomal subunit [28], and interestingly, also functions in silencing at HML and HMR [29]. It was therefore intriguing that SBP1 expression was increased 2- to 3-fold in the sir2Δ and 100bpΔ mutants (Fig 3G), suggesting that Sir2 at the RDT1 promoter has a modest downstream repressive effect on SPB1, but not on the upstream R1L/S ncRNAs. Notably, steady state RDT1 and R1L/S RNA levels were relatively low compared to SPB1 and the ACT1 loading control, even in the 100bpΔ mutant (Fig 3G).Sir2 and condensin are displaced from the RDT1 promoter during mating-type switchingWe next asked if Sir2 played any role in regulating RDT1 during mating-type switching. Sir2 was previously shown to associate with a HO-induced DSB at the MAT locus during mating-type switching, presumably to effect repair through histone deacetylation [30]. SIR complex association with DSBs occurs at the expense of telomere binding [31], so we hypothesized that the HO-induced DSB at MAT could also trigger loss of Sir2 from the RDT1 promoter, thus facilitating increased RDT1 transcription. To test this idea, HO was induced at time 0 with galactose and then turned off 2 hours later by glucose addition to allow for repair/switching to occur (Fig 4A and 4B). By the 3 hr time point (1 hr after glucose addition), ChIP analysis indicated Sir2 was maximally enriched at the MAT locus (Fig 4C), corresponding to the time of peak mating-type switching ([30] and Fig 4B). Interestingly, Sir2 was significantly depleted from the RDT1 promoter within 1 hr after HO induction, and by 3 hr there was actually stronger enrichment of Sir2 at MAT than RDT1 (Fig 4C). Critically, this shift in Sir2 distribution correlated with maximal induction of RDT1 mRNA and the Myc-tagged Rdt1protein (Fig 4D and 4E, 3 hr). Once switching was completed by 4 hr (2 hr after glucose addition), RDT1 transcription was permanently inactivated and Sir2 binding never returned because most cells were now MATα. The Myc-tagged Rdt1 protein, however, remained elevated for the rest of the time course (Fig 4E), suggesting that it is relatively stable, at least when epitope tagged. A parallel ChIP time course experiment was performed with condensin (Brn1-myc), indicating significant depletion from the RDT1 promoter within 1 hr (Fig 4F), similar to the timing of Sir2 loss. Unlike Sir2, Brn1-myc enrichment at the HO-cleaved MAT site did not increase, suggesting that condensin normally associates with MATa in non-switching cells and is then displaced in response to the HO-induced DSB, perhaps to facilitate structural reorganization associated with switching.10.1371/journal.pgen.1008339.g004Fig 4Dynamics of Sir2 and condensin binding at the RDT1 promoter and MATa locus during mating-type switching.(A, B) Quantitation of a mating-type switching time course where HO was induced by galactose at time 0, then glucose added at 2 hr to stop HO expression and allow for break repair. Switching is maximal at 3 hr. The average ratios of MATa and MATα PCR products relative to an SCR1 loading control PCR product were plotted (n = 3 biological replicates). (C) ChIP of Sir2 at the RDT1 promoter and the HO-induced DSB site (MAT-HO). (D) qRT-PCR of RDT1 expression relative to ACT1 across the mating-type switching time course. The time 0 level is normalized to 1. (E) Rdt1-13xMyc protein expression across the same time course. (F) ChIP of Brn1-myc at the RDT1 promoter and MAT-HO break site across the same time course. ChIP signal PCR is plotted relative to the input signal. (*p<0.05, **p<0.005 compared to time 0).Since RDT1 was highly expressed during mating-type switching, we next asked whether the small protein encoded by this gene had a direct function during the switching process when using the galactose-inducible system employed in this study. The 28 amino acid ORF was precisely deleted using the delitto perfetto method [32], and the efficiency of switching from MATa to MATα was then tested across a time course by PCR (S4A and S4B Fig). No significant differences were observed between the WT and rdt1Δ strains. Next, donor preference was tested using a strain previously developed by the Haber lab [15], whereby HMRa was replaced with HMRα containing a unique BamHI site (HMRα-B) (S4C Fig). Following completion of Gal-HO-induced switching from MATa to MATα, the proportion of donor utilization was determined by BamHI digest of a MATα PCR product. As shown in S4D and S4E Fig, deleting RDT1 also had no effect on donor preference, indicated by low utilization (~9%) of HMRα-B. Therefore, although RDT1 gene expression strongly correlates with switching, a specific function for its gene product remains elusive. Therefore, we shifted our attention to a possible function for the RDT1 promoter.The RDT1 promoter region controls chromosome III architectureThe coupling of Sir2 and condensin distribution with RDT1 transcriptional regulation during mating-type switching was reminiscent of classic locus control regions (LCR) that modulate long-range chromatin interactions [33, 34]. We therefore hypothesized that the RDT1 promoter region functions as an LCR to modulate long-range chromatin interactions of chromosome III. To test this hypothesis, we performed Hi-C analysis with WT, sir2Δ and the 100bpΔ strains. Genomic contact differences between the mutants and WT were quantified using the HOMER Hi-C software suite [35], and the frequency of statistically significant differences for each chromosome calculated (Fig 5A). Chromosome III had the most significant differences in both mutants, so we focused on this chromosome and used HOMER to plot the observed/expected interaction frequency in 10kb bins for each strain as a heat map (Fig 5B). In a WT strain (ML1) there was strong interaction between the left and right ends of chromosome III, mostly centered around the HML (bin 2) and HMR (bin 29) loci. Interestingly, HML (bin 2) also appeared to sample the entire right arm of chromosome III, with the interaction frequency increasing as a gradient from CEN3 to a maximal observed interaction at HMR, thus also encompassing the MATa locus at bin 20. This distinct interaction pattern was completely disrupted in the sir2Δ mutant, whereas some telomere-subtelomere contacts were retained in the 100bpΔ mutant (Fig 5B), suggesting there was still limited interaction between the left and right ends of the chromosome. We confirmed the changes in HML-HMR interaction for these strains using a quantitative 3C-PCR assay to rule out sequencing artifacts (Fig 5C), and to confirm an earlier sir2Δ 3C result from the Dekker lab [36]. Importantly, despite the loss of HML-HMR interaction in the 100bpΔ mutant, heterochromatin at these domains was unaffected based on normal quantitative mating assays (S5A Fig), and unaltered Sir2 association with HML (S5B Fig).10.1371/journal.pgen.1008339.g005Fig 5The Sir2/condensin binding site controls chromosome III architecture.(A) Frequency of significant Hi-C interaction changes identified using HOMER for each chromosome in the sir2Δ (ML25) and 100bpΔ (ML275) strains compared to WT (ML1). (B) HOMER-generated log2 observed/expected Hi-C interaction frequency heat maps (10kb bins) for chromosome III. Solid white lines indicate bins with insufficient read coverage post-filtering. (C) qPCR detection of HML-HMR interaction using 3C analysis, with the WT signal normalized to 1. (*p < 0.05, **<0.005). (D) Iteratively corrected and read-normalized Hi-C heat maps revealing an interaction between HMR (bin 29) and MATa (bin 20) in the sir2Δ and 100bpΔ mutants (black arrow). Solid black lines indicate bins with insufficient read coverage post-filtering. Red arrows indicate the HML-HMR interaction. (E) Summary of large-scale changes in chromosome III architecture. Δ indicates the 100bp deletion.We next analyzed the Hi-C data using an iterative correction method that reduces background to reveal interacting loci that potentially drive the overall chromosomal architecture, rather than passenger locus effects [37]. HML (bin 2) and HMR (bin 29) again formed the dominant interaction pair off the diagonal in WT, which was lost in the sir2Δ or 100bpΔ mutants (Fig 5D, red arrows). Importantly, a prominent new interaction between HMR (bin 29) and MATa (bin 20) appeared in both mutants (Fig 5D, black arrows), as would be predicted if normal donor preference of MATa cells was altered. We conclude that the RDT1 promoter does function like an LCR in MATa yeast cells, regulating localized transcription and establishing a long-range chromatin interaction between HML and HMR that appears to prevent HMR from strongly associating with MATa (Fig 5E).Sir2 and condensin regulate mating-type switchingSir2/condensin binding was observed in the right half of the RE (Fig 1A), but this region was previously reported as dispensable for donor preference activity [10]. Considering that HMR was aberrantly associated with the MATa locus in sir2Δ and 100bpΔ mutants (Fig 5B–5E), we proceeded to test whether these mutants had any alterations in donor preference. As was done for the rdt1Δ mutant in S4C and S4D Fig, the 100bpΔ mutation was introduced into a reporter strain with HMRα-B on the right arm of chromosome III (Fig 6A; [15]). After inducing switching to MATα with galactose, the proportion of HMLα or HMRα-B used for the switching was determined by BamHI digestion of a MATα-specific PCR product (Fig 6B; [15]). As expected for normal donor preference, HMRα-B on the right arm was only utilized ~9% of the time in the WT strain (Fig 6C and 6D). Strikingly, donor preference was lost in the sir2Δ mutant, similar to a control strain with the RE deleted (Fig 6C and 6D), and consistent with the clear interaction between HMR and MATa bins observed for the sir2Δ mutant in Fig 5D and 5E. This interaction was less prominent in the 100bpΔ mutant (Fig 5D), and the corresponding donor preference defect was also less severe (~25% HMRα-B), though still significantly different from WT (Fig 6C and 6D).10.1371/journal.pgen.1008339.g006Fig 6Loss of Sir2 and the Sir2/condensin binding site alters mating-type switching.(A) Schematic of a donor preference assay in which utilization of an artificial HMRα-B cassette as the donor for switching introduces a unique BamHI site to the MAT locus. (B) Locations of primers flanking the BamHI site used for PCR detection of MATα. (C) Representative ethidium bromide stained agarose gel of BamHI-digested MATα PCR products after mating-type switching in WT (XW652), reΔ (XW676), sir2Δ (ML557), and 100bpΔ (SY742) strains. The MATα-B product is digested into 2 smaller bands. (D) Quantifying the percentage of MATα-B PCR product digested by BamHI, from three biological replicates. ImageJ was used for the quantitation. (*p < 0.05, **p < 0.005). (E) Representative time course of switching from MATa to MATα in WT (ML447), 100bpΔ (ML460), and sir2Δ (ML458) strains after HO was induced for 45 min and then shut down with glucose. Aliquots were harvested at 30 min intervals. SCR1 is a control for input genomic DNA. (F) ImageJ quantification of MATα PCR relative to SCR1 for each time point. n = 3 biological replicates.Since the donor preference assay is an endpoint experiment, we next tested whether the sir2Δ or 100bpΔ mutations temporally impacted the efficiency of switching from MATa to MATα in the same ML1 background strains used for Hi-C analysis. As shown in Fig 6E and 6F, switching efficiency was dramatically impaired in the sir2Δ strain, but unaffected in the 100bpΔ mutant. These results suggest a model whereby condensin and Sir2 recruitment to the RDT1 promoter supports donor preference by organizing chromosome III into a structure that limits HMR association with the MATa locus, but is not required for the mechanics of mating-type switching. We suspect at least part of the strong sir2Δ effect on chromosome III organization and donor preference is caused by HMLALPHA2 derepression, which inactivates the RE due to formation of the Mcm1/α2 heterodimer [13]. We also considered a possibility that the sir2Δ heterochromatin defect at HML and HMR could make them highly accessible to HO cleavage [38], which would prevent their usage as donor templates. As a measure of HO cleavage at MATa, HML, or HMR, we assayed for reduced PCR amplification across the recognition site following Gal-HO induction (S6 Fig). While MATa was equally cut by HO in WT and sir2Δ strains (S6A and S6B Fig), HML was only cut in the sir2Δ strain (S6A and S6C Fig), consistent with the idea of reduced HML availability for switching. Unexpectedly, HMR remained available as a template in the sir2Δ strain (S6A and S6D Fig). We confirmed the difference in HO accessibility between HML and HMR using real-time qPCR (S6E and S6F Fig), which could contribute to the extreme sir2Δ donor preference defect (Fig 6C and 6D). In the 100bpΔ mutant, which locally eliminates condensin recruitment at RDT1, the continued maintenance of heterochromatin at HML/HMR and telomeres (S5 Fig), as well as residual telomere clustering (Fig 5D), may partially buffer the resulting donor preference defect by still limiting contact between the subtelomeric HMR locus and MATa.Condensin recruitment to the RDT1 promoter does not appear critical for the mechanics of mating-type switching, as indicated by normal timing of switching in the 100bpΔ mutant (Fig 6E and 6F). However, condensin could still potentially impact the switching process independent of the RE. In order to test this hypothesis, we C-terminally tagged the Brn1 condensin subunit with an auxin-inducible degron (AID) fused with a V5 epitope, which allows for rapid depletion of tagged proteins upon the addition of auxin [39]. Indeed, Brn1-AID was effectively degraded within 15 min of adding auxin to cells, as measured by western blotting (S7A Fig), or ChIP at the RDT1 promoter (S7B Fig). Importantly, even after 1 hr of auxin treatment, there were no significant changes in RDT1 or HMLALPHA2 gene expression as measured by qRT-PCR (S7C and S7D Fig), thus indicating that silencing of HML was unaffected, unlike the ycs4-1 condensin mutant used in Fig 1G [22]. The efficiency of ML1 switching from MATa to MATα was then tested across a time course with or without auxin treatment (Fig 7A). As shown in Fig 7B and 7C, auxin significantly slowed the efficiency of switching to MATα, indicating that the Brn1 subunit of condensin is important for normal mating-type switching.10.1371/journal.pgen.1008339.g007Fig 7Effects of condensin depletion on mating-type switching.(A) Schematic of the time course used to deplete Brn1-AID prior to induction of mating-type switching in the ML1 strain background. Auxin was added 30 min prior to the induction of HO expression by galactose. (B) Representative EtBr stained agarose gel of MATα PCR products amplified from each time point during mating-type switching. SCR1 PCR was used as a control for input DNA. (C) Quantification of the MATα/SCR1 PCR product ratio across the time course from 3 biological replicates. (D) Effect of Brn1-AID depletion on mating-type switching donor preference. A representative biological replicate is shown, along with percentage use of HMRα-B cassette as the donor template. (*p < 0.05).Since the 100bpΔ mutant caused a modest donor preference defect that we partially attributed to a loss of condensin (Fig 6C and 6D), it was important to also test for a donor preference defect when condensin was depleted. Indeed, Brn1-AID depletion produced a significant defect in donor preference using the HMRα-B reporter strain (Fig 7D) that was similar in magnitude to that observed for the 100bpΔ strain (Fig 6D). Taken together, these results support a working model whereby condensin recruitment to the RDT1 promoter in MATa cells organizes chromosome III into a conformation that limits the association of HMR with MATa, thus partially contributing to donor preference regulation. We hypothesize that upon HO cleavage of MATa, the increased expression of RDT1 caused by loss of Sir2, displaces condensin from the promoter, which then allows the left half of the RE to physically direct HML to MATa for use as a donor (Fig 8).10.1371/journal.pgen.1008339.g008Fig 8Model for cooperative regulation of chromosome III structure and donor preference by the bipartite RE.The left half of the RE regulates donor preference through expression of the R1L and R1S long ncRNAs and an array of Fkh1 proteins that interact with threonine-phosphorylated proteins at the double stranded DNA break at MATa. The right half of the RE is bound by condensin and Sir2. Like R1L/S, the expression of RDT1 correlates with mating-type switching. Condensin is hypothesized to organize chromosome III structure through a unidirectional loop extrusion process that eventually places MATa into a chromosome-sized loop that helps insulates it from HML and HMR until condensin is removed at the time of switching. The loop is also likely maintained by clustering of telomeres and subtelomeres (black bars) and the heterochromatin set up by HML and HMR.

Sir2 and condensin associate with the recombination enhancer (RE)We previously characterized global sirtuin distribution using ChIP-Seq to identify novel loci regulated by Sir2 and its homologs [19]. Significant overlap was observed between binding sites for Sir2, Hst1, or Sum1 with previously described condensin binding sites [19, 20], suggesting a possible functional connection. ChIP-Seq was therefore performed on WT and sir2Δ strains in which the condensin subunit Smc4 was C-terminally tagged (13xMyc) (Fig 1A). To avoid “hyper-ChIPable” loci that can appear in yeast ChIP-seq experiments, we also ran nuclear localized GFP controls [21]. Genes closest to Sir2-dependent condensin peaks after subtraction of GFP are listed in S1 Table, and are distributed throughout the genome. One of the strongest peaks overlapped with a Sir2-myc binding site on chromosome III between KAR4 and SPB1 that was not enriched for GFP (Fig 1A). The specificity of Sir2 enrichment at this peak, as opposed to the adjacent SPB1 gene, was independently confirmed by quantitative ChIP using an α-Sir2 antibody (Fig 1B), with enrichment comparable to levels observed at the HML-I silencer (Fig 1A and 1B). Sir2-dependent condensin binding was also confirmed for Myc-tagged Smc4 and Brn1 subunits (Fig 1C). The ~2.5 kb intergenic region between KAR4 and SPB1 was previously defined as a cis-acting recombination enhancer (RE) that specifies donor preference of mating-type switching in MATa cells [10, 13]. Quantitative ChIP assays revealed that Sir2 and Brn1-myc enrichment at the RE was also MATa-specific (Fig 1D and 1E), which was notable because the ChIP-seq datasets in Fig 1A happened to be generated from MATa strains. We next considered whether the condensin binding defect in the MATa sir2Δ mutant was due to HMLALPHA2 expression caused by defective HML silencing. To test this idea, we re-examined Brn1-myc ChIP signal at the RE in strains lacking HML, and found that deleting SIR2 no longer affected condensin recruitment (Fig 1F). Similarly, a MATa condensin mutant (ycs4-1) known to have an HML silencing defect [22] reduced Sir2 recruitment to the RE, but had no effect when HML was also deleted (Fig 1G). Sir2 and condensin are therefore independently recruited to the RE only in MATa cells.10.1371/journal.pgen.1008339.g001Fig 1MATa-specific binding of Sir2 and condensin to the recombination enhancer (RE).(A) ChIP-seq of Smc4-myc, Sir2-myc, and nuclear localized GFP in WT and sir2Δ backgrounds. The left arm of chromosome III is depicted from HML to SPB1. RE indicates the recombination enhancer region. Inset: The minimal 700bp RE element required for donor preference is indicated, as are the two Mcm1/α2 binding sites (DPS1 and DPS2) and RDT1. (B) Sir2 ChIP at the RE, HML-I silencer, and SPB1. (C)
α-Myc ChIP of Brn1-myc and Smc4-myc at the RE. (D) ChIP showing MATa-specific binding of Sir2 to the RE. (E) ChIP showing MATa-specific binding of Brn1-myc to the RE. (F) Brn1-Myc ChIP at RE is not Sir2 dependent. (G) Native Sir2 ChIP at RE is not condensin dependent. The ChIP signal in each panel is relative to input and plotted as the mean of three replicates. Error bars = standard deviation. (**p<0.005).

Sir2 and condensin associate with the recombination enhancer (RE)

We previously characterized global sirtuin distribution using ChIP-Seq to identify novel loci regulated by Sir2 and its homologs [19]. Significant overlap was observed between binding sites for Sir2, Hst1, or Sum1 with previously described condensin binding sites [19, 20], suggesting a possible functional connection. ChIP-Seq was therefore performed on WT and sir2Δ strains in which the condensin subunit Smc4 was C-terminally tagged (13xMyc) (Fig 1A). To avoid “hyper-ChIPable” loci that can appear in yeast ChIP-seq experiments, we also ran nuclear localized GFP controls [21]. Genes closest to Sir2-dependent condensin peaks after subtraction of GFP are listed in S1 Table, and are distributed throughout the genome. One of the strongest peaks overlapped with a Sir2-myc binding site on chromosome III between KAR4 and SPB1 that was not enriched for GFP (Fig 1A). The specificity of Sir2 enrichment at this peak, as opposed to the adjacent SPB1 gene, was independently confirmed by quantitative ChIP using an α-Sir2 antibody (Fig 1B), with enrichment comparable to levels observed at the HML-I silencer (Fig 1A and 1B). Sir2-dependent condensin binding was also confirmed for Myc-tagged Smc4 and Brn1 subunits (Fig 1C). The ~2.5 kb intergenic region between KAR4 and SPB1 was previously defined as a cis-acting recombination enhancer (RE) that specifies donor preference of mating-type switching in MATa cells [10, 13]. Quantitative ChIP assays revealed that Sir2 and Brn1-myc enrichment at the RE was also MATa-specific (Fig 1D and 1E), which was notable because the ChIP-seq datasets in Fig 1A happened to be generated from MATa strains. We next considered whether the condensin binding defect in the MATa sir2Δ mutant was due to HMLALPHA2 expression caused by defective HML silencing. To test this idea, we re-examined Brn1-myc ChIP signal at the RE in strains lacking HML, and found that deleting SIR2 no longer affected condensin recruitment (Fig 1F). Similarly, a MATa condensin mutant (ycs4-1) known to have an HML silencing defect [22] reduced Sir2 recruitment to the RE, but had no effect when HML was also deleted (Fig 1G). Sir2 and condensin are therefore independently recruited to the RE only in MATa cells.

MATa-specific binding of Sir2 and condensin to the recombination enhancer (RE).

(A) ChIP-seq of Smc4-myc, Sir2-myc, and nuclear localized GFP in WT and sir2Δ backgrounds. The left arm of chromosome III is depicted from HML to SPB1. RE indicates the recombination enhancer region. Inset: The minimal 700bp RE element required for donor preference is indicated, as are the two Mcm1/α2 binding sites (DPS1 and DPS2) and RDT1. (B) Sir2 ChIP at the RE, HML-I silencer, and SPB1. (C)
α-Myc ChIP of Brn1-myc and Smc4-myc at the RE. (D) ChIP showing MATa-specific binding of Sir2 to the RE. (E) ChIP showing MATa-specific binding of Brn1-myc to the RE. (F) Brn1-Myc ChIP at RE is not Sir2 dependent. (G) Native Sir2 ChIP at RE is not condensin dependent. The ChIP signal in each panel is relative to input and plotted as the mean of three replicates. Error bars = standard deviation. (**p<0.005).

Sir2 regulates a small gene (RDT1) within the REDonor preference activity ascribed to the RE was previously narrowed down to a KAR4 (YCL055W)-proximal 700 bp domain defined by an Mcm1/α2 binding site (Fig 2A, DPS1) [10, 11, 13]. The Sir2 and condensin ChIP-seq peaks we identified were located outside this region, between a second Mcm1/α2 binding site (DPS2) and a small gene of unknown function called RDT1 [23] (Figs 1A and 2A). We noticed the location of RDT1 coincided with the smallest of several putative non-coding RNAs (ncRNA) previously reported as being transcribed from the RE, but not annotated in SGD (Fig 2A, [13]). Quantitative RT-PCR and analysis of publicly available RNA-seq data from BY4741 (MATa) and BY4742 (MATα) revealed that RDT1 expression was indeed MATa specific (Fig 2B and S1A Fig).10.1371/journal.pgen.1008339.g002Fig 2RDT1 is a novel Sir2 regulated gene.(A) Schematic of the RE locus depicting Sir2/Condensin peak locations relative to previously reported R1L/S and R2 (RDT1) RNAs. (B)
RDT1 mRNA expression is MATa specific. RDT1 expression level is calculated relative to the ACT1 control. This RDT1/ACT1 ratio is then normalized to 1.0 for the MATa strain. (C) H4K16ac ChIP signal at RE in SIR complex null strains, relative to input signal. WT is normalized to 1.0. (D) H4K16 deacetylation is dependent on Sir2 catalytic activity. A sir2Δ strain was transformed with the indicated plasmids and ChIP assays performed. ChIP signal is relative to input. (E) Effect of sir2Δ on H3K9/K14ac ChIP at the RDT1 promoter in HML and hmlΔ backgrounds. ChIP signal is relative to input. (F) Differential RDT1 transcriptional regulation by SIR2 is dependent on HML status. RDT1 expression is relative to the ACT1 control, and the RDT1/ACT1 ratio is normalized to 1.0 for WT. (G) Effects of the ycs4-1 mutation on RDT1 expression in HML and hmlΔ backgrounds at 30°C. (*p<0.05; **p<0.005). RDT1 expression is plotted as for panel F.We next asked whether Sir2 and/or condensin regulate histone acetylation and RDT1 expression when recruited to the RE. Sir2 normally represses transcription at HML, HMR, and telomeres as a catalytic subunit of the SIR complex where it preferentially deacetylates H4K16 (reviewed in [5]). Accordingly, deleting SIR2, SIR3, or SIR4 from MATa cells increased H4K16 acetylation at the RDT1 promoter (Fig 2C), consistent with the observed enrichment of Sir3-myc and Sir4-myc at this site (S1B Fig). Furthermore, re-introducing active SIR2 into the sir2Δ mutant restored H4K16 to the hypoacetylated state, whereas catalytically inactive sir2-H364Y did not (Fig 2D). While H4K16ac is a preferred Sir2 substrate for silencing at HML and HMR, the recruited SIR complex also maintains lysine deacetylation of the other N-terminal histone tails [24]. Since RDT1 transcription is MATa specific, and Mcm1/α2 represses MATa-specific genes by recruiting the Ssn6/Tup1 corepressor complex and Class I/II HDACs such as the H3/H2B-specific histone deacetylase HDA1 [25, 26], we also tested the effect of deleting SIR2 on H3K9/14 acetylation at the RDT1 promoter, predicting it may remain hypoacetylated due to the SIR complex being replaced by Ssn6/Tup1/HDA1. Indeed, H3K9/14 acetylation was reduced in the sir2Δ mutant relative to WT, but was significantly elevated in the hmlΔ sir2Δ double mutant (Fig 2E). RDT1 expression was similarly reduced in the sir2Δ mutant and strongly upregulated when HML and SIR2 were both deleted (Fig 2F). An even stronger expression effect was observed in an hmlΔ sir2Δ hst1Δ triple mutant that eliminates any possibility of redundancy between the Sir2 and Hst1 paralogs (S1C Fig). On the other hand, RDT1 was not upregulated in an hmlΔ ycs4-1 condensin mutant (Fig 2G). Taken together, these results provide strong evidence that the SIR complex represses RDT1 in MATa cells by establishing a generally hypoacetylated chromatin environment at the promoter, while condensin has a functional role independent of transcriptional regulation.We next attempted to prevent Sir2 and condensin recruitment to the RDT1 promoter by precisely deleting a 100bp DNA sequence underlying the shared enrichment region (coordinates 30701–30800), while not disturbing the adjacent Mcm1/α2 site (Fig 3A). Sir2 and Brn1-myc binding to the RE as measured by ChIP was greatly diminished in this mutant (Fig 3B and 3C), despite unaltered Sir2, Brn1-myc, or Smc4-myc expression levels (S2A–S2C Fig). Furthermore, RDT1 RNA expression level was significantly increased by the 100bp deletion exclusively in MATa cells (Fig 3D), consistent with the loss of Sir2-mediated repression.10.1371/journal.pgen.1008339.g003Fig 3Identification of a 100bp sequence that recruits Sir2/condensin and represses RDT1 expression.(A) Schematic indicating a 100bp deletion that covers the condensin (red) and Sir2 (blue) peaks. (B) ChIP of Sir2 in the 100bpΔ mutant (ML275). (C) ChIP of Brn1-Myc in the 100bpΔ mutant. ChIP PCR signal in panels B and C is plotted relative to the input signal. (D)
RDT1 transcription in MATa cells is derepressed in the 100bpΔ mutant. The RDT1 level is plotted relative to ACT1 control, and then normalized to 1.0 for the WT MATa strain. (E) Western blot of Rdt1-13xMyc in WT MATα and MATa cells, as well as the MATa 100bpΔ mutant. Predicted fusion protein molecular weight is 19.1 kDa, but it runs ~37 kDa. (F) Comparative RDT1 and R1L/S expression levels in WT, sir2Δ, and 100bpΔ strains. One of the WT RDT1 qPCR replicates was normalized to 1.0 to allow direct comparisons between RDT1 and R1L/S levels. (G) Comparative RDT1, R1L/S, and SPB1 expression levels in WT, sir2Δ, and 100bpΔ strains. As in panel F, one of the WT RDT1 qPCR replicates was normalized to 1.0. (*p<0.05; **p<0.005, compared to WT strain).Because Sir2 and condensin were not present at the RDT1 promoter in MATα cells, we reasoned that their binding should require a MATa specific transcription factor. This made the 2nd Mcm1/α2 binding site (DPS2) upstream of the Sir2/condensin ChIP-seq peaks an ideal candidate, because it had not previously been ascribed a function other than redundancy with DPS1 for donor preference [13]. Deleting MCM1 is lethal, so alternatively, we precisely deleted the 2nd Mcm1/α2 binding site (ChrIII coordinates 30595 to 30626, S3A Fig) and then retested for Sir2 and Brn1-myc enrichment. As shown in S3B and S3C Fig, respectively, Sir2 and Brn1-myc enrichment at the Mcm1/α2 binding site (DPS2) and the RDT1 promoter (defined as the Sir2/condensin peaks) was significantly reduced in the binding site mutant. These results suggest that Mcm1 nucleates a complex that recruits the SIR and condensin complexes to the RDT1 promoter in MATa cells, and also provides a possible mechanism of blocking the recruitment in MATα cells due to the interaction of Mcm1 with α2.

Sir2 regulates a small gene (RDT1) within the RE

Donor preference activity ascribed to the RE was previously narrowed down to a KAR4 (YCL055W)-proximal 700 bp domain defined by an Mcm1/α2 binding site (Fig 2A, DPS1) [10, 11, 13]. The Sir2 and condensin ChIP-seq peaks we identified were located outside this region, between a second Mcm1/α2 binding site (DPS2) and a small gene of unknown function called RDT1 [23] (Figs 1A and 2A). We noticed the location of RDT1 coincided with the smallest of several putative non-coding RNAs (ncRNA) previously reported as being transcribed from the RE, but not annotated in SGD (Fig 2A, [13]). Quantitative RT-PCR and analysis of publicly available RNA-seq data from BY4741 (MATa) and BY4742 (MATα) revealed that RDT1 expression was indeed MATa specific (Fig 2B and S1A Fig).

RDT1 is a novel Sir2 regulated gene.

(A) Schematic of the RE locus depicting Sir2/Condensin peak locations relative to previously reported R1L/S and R2 (RDT1) RNAs. (B)
RDT1 mRNA expression is MATa specific. RDT1 expression level is calculated relative to the ACT1 control. This RDT1/ACT1 ratio is then normalized to 1.0 for the MATa strain. (C) H4K16ac ChIP signal at RE in SIR complex null strains, relative to input signal. WT is normalized to 1.0. (D) H4K16 deacetylation is dependent on Sir2 catalytic activity. A sir2Δ strain was transformed with the indicated plasmids and ChIP assays performed. ChIP signal is relative to input. (E) Effect of sir2Δ on H3K9/K14ac ChIP at the RDT1 promoter in HML and hmlΔ backgrounds. ChIP signal is relative to input. (F) Differential RDT1 transcriptional regulation by SIR2 is dependent on HML status. RDT1 expression is relative to the ACT1 control, and the RDT1/ACT1 ratio is normalized to 1.0 for WT. (G) Effects of the ycs4-1 mutation on RDT1 expression in HML and hmlΔ backgrounds at 30°C. (*p<0.05; **p<0.005). RDT1 expression is plotted as for panel F.

We next asked whether Sir2 and/or condensin regulate histone acetylation and RDT1 expression when recruited to the RE. Sir2 normally represses transcription at HML, HMR, and telomeres as a catalytic subunit of the SIR complex where it preferentially deacetylates H4K16 (reviewed in [5]). Accordingly, deleting SIR2, SIR3, or SIR4 from MATa cells increased H4K16 acetylation at the RDT1 promoter (Fig 2C), consistent with the observed enrichment of Sir3-myc and Sir4-myc at this site (S1B Fig). Furthermore, re-introducing active SIR2 into the sir2Δ mutant restored H4K16 to the hypoacetylated state, whereas catalytically inactive sir2-H364Y did not (Fig 2D). While H4K16ac is a preferred Sir2 substrate for silencing at HML and HMR, the recruited SIR complex also maintains lysine deacetylation of the other N-terminal histone tails [24]. Since RDT1 transcription is MATa specific, and Mcm1/α2 represses MATa-specific genes by recruiting the Ssn6/Tup1 corepressor complex and Class I/II HDACs such as the H3/H2B-specific histone deacetylase HDA1 [25, 26], we also tested the effect of deleting SIR2 on H3K9/14 acetylation at the RDT1 promoter, predicting it may remain hypoacetylated due to the SIR complex being replaced by Ssn6/Tup1/HDA1. Indeed, H3K9/14 acetylation was reduced in the sir2Δ mutant relative to WT, but was significantly elevated in the hmlΔ sir2Δ double mutant (Fig 2E). RDT1 expression was similarly reduced in the sir2Δ mutant and strongly upregulated when HML and SIR2 were both deleted (Fig 2F). An even stronger expression effect was observed in an hmlΔ sir2Δ hst1Δ triple mutant that eliminates any possibility of redundancy between the Sir2 and Hst1 paralogs (S1C Fig). On the other hand, RDT1 was not upregulated in an hmlΔ ycs4-1 condensin mutant (Fig 2G). Taken together, these results provide strong evidence that the SIR complex represses RDT1 in MATa cells by establishing a generally hypoacetylated chromatin environment at the promoter, while condensin has a functional role independent of transcriptional regulation.

We next attempted to prevent Sir2 and condensin recruitment to the RDT1 promoter by precisely deleting a 100bp DNA sequence underlying the shared enrichment region (coordinates 30701–30800), while not disturbing the adjacent Mcm1/α2 site (Fig 3A). Sir2 and Brn1-myc binding to the RE as measured by ChIP was greatly diminished in this mutant (Fig 3B and 3C), despite unaltered Sir2, Brn1-myc, or Smc4-myc expression levels (S2A–S2C Fig). Furthermore, RDT1 RNA expression level was significantly increased by the 100bp deletion exclusively in MATa cells (Fig 3D), consistent with the loss of Sir2-mediated repression.

Identification of a 100bp sequence that recruits Sir2/condensin and represses RDT1 expression.

(A) Schematic indicating a 100bp deletion that covers the condensin (red) and Sir2 (blue) peaks. (B) ChIP of Sir2 in the 100bpΔ mutant (ML275). (C) ChIP of Brn1-Myc in the 100bpΔ mutant. ChIP PCR signal in panels B and C is plotted relative to the input signal. (D)
RDT1 transcription in MATa cells is derepressed in the 100bpΔ mutant. The RDT1 level is plotted relative to ACT1 control, and then normalized to 1.0 for the WT MATa strain. (E) Western blot of Rdt1-13xMyc in WT MATα and MATa cells, as well as the MATa 100bpΔ mutant. Predicted fusion protein molecular weight is 19.1 kDa, but it runs ~37 kDa. (F) Comparative RDT1 and R1L/S expression levels in WT, sir2Δ, and 100bpΔ strains. One of the WT RDT1 qPCR replicates was normalized to 1.0 to allow direct comparisons between RDT1 and R1L/S levels. (G) Comparative RDT1, R1L/S, and SPB1 expression levels in WT, sir2Δ, and 100bpΔ strains. As in panel F, one of the WT RDT1 qPCR replicates was normalized to 1.0. (*p<0.05; **p<0.005, compared to WT strain).

Because Sir2 and condensin were not present at the RDT1 promoter in MATα cells, we reasoned that their binding should require a MATa specific transcription factor. This made the 2nd Mcm1/α2 binding site (DPS2) upstream of the Sir2/condensin ChIP-seq peaks an ideal candidate, because it had not previously been ascribed a function other than redundancy with DPS1 for donor preference [13]. Deleting MCM1 is lethal, so alternatively, we precisely deleted the 2nd Mcm1/α2 binding site (ChrIII coordinates 30595 to 30626, S3A Fig) and then retested for Sir2 and Brn1-myc enrichment. As shown in S3B and S3C Fig, respectively, Sir2 and Brn1-myc enrichment at the Mcm1/α2 binding site (DPS2) and the RDT1 promoter (defined as the Sir2/condensin peaks) was significantly reduced in the binding site mutant. These results suggest that Mcm1 nucleates a complex that recruits the SIR and condensin complexes to the RDT1 promoter in MATa cells, and also provides a possible mechanism of blocking the recruitment in MATα cells due to the interaction of Mcm1 with α2.

RDT1 encodes a translated mRNARibosome Detected Transcript-1 (RDT1) was originally annotated as a newly evolved gene whose transcript was associated with ribosomes and predicted to have a small open reading frame of 28 amino acids [23]. Our work suggested that RDT1 and the putative non-coding R2 transcript were the same (Fig 2A). To determine if RDT1/R2 codes for a small protein, the ORF was C-terminally fused with a 13x-Myc epitope in MATa and MATα cells. As shown in Fig 3E, a fusion protein was exclusively detected in exponentially growing MATa WT cells and also highly expressed in the 100bpΔ strain, correlating with the increased RNA level observed for that mutant in Fig 3D.Since additional MATa-specific RNAs are derived from the minimal 700bp RE domain (Fig 2A; R1L and R1S) [13, 27], we next tested whether Sir2/condensin recruitment to the RDT1 promoter had any effect on expression of these upstream ncRNAs from a distance. Quantitative RT-PCR for the R1L/S transcripts indicated their expression level in the WT strain was comparable to RDT1, and was also reduced in a sir2Δ mutant because of the pseudodiploid phenotype (Fig 3F). However, while RDT1 was strongly upregulated by the 100bp deletion of the Sir2/condensin binding site, R1L/S expression was unaffected (Fig 3F). As a control, we also measured expression of the SPB1 gene located immediately downstream of RDT1 (see Fig 2A schematic). SPB1 expression is not mating-type specific, it encodes a rRNA methyltransferase required for maturation of the large 60S ribosomal subunit [28], and interestingly, also functions in silencing at HML and HMR [29]. It was therefore intriguing that SBP1 expression was increased 2- to 3-fold in the sir2Δ and 100bpΔ mutants (Fig 3G), suggesting that Sir2 at the RDT1 promoter has a modest downstream repressive effect on SPB1, but not on the upstream R1L/S ncRNAs. Notably, steady state RDT1 and R1L/S RNA levels were relatively low compared to SPB1 and the ACT1 loading control, even in the 100bpΔ mutant (Fig 3G).

RDT1 encodes a translated mRNA

Ribosome Detected Transcript-1 (RDT1) was originally annotated as a newly evolved gene whose transcript was associated with ribosomes and predicted to have a small open reading frame of 28 amino acids [23]. Our work suggested that RDT1 and the putative non-coding R2 transcript were the same (Fig 2A). To determine if RDT1/R2 codes for a small protein, the ORF was C-terminally fused with a 13x-Myc epitope in MATa and MATα cells. As shown in Fig 3E, a fusion protein was exclusively detected in exponentially growing MATa WT cells and also highly expressed in the 100bpΔ strain, correlating with the increased RNA level observed for that mutant in Fig 3D.

Since additional MATa-specific RNAs are derived from the minimal 700bp RE domain (Fig 2A; R1L and R1S) [13, 27], we next tested whether Sir2/condensin recruitment to the RDT1 promoter had any effect on expression of these upstream ncRNAs from a distance. Quantitative RT-PCR for the R1L/S transcripts indicated their expression level in the WT strain was comparable to RDT1, and was also reduced in a sir2Δ mutant because of the pseudodiploid phenotype (Fig 3F). However, while RDT1 was strongly upregulated by the 100bp deletion of the Sir2/condensin binding site, R1L/S expression was unaffected (Fig 3F). As a control, we also measured expression of the SPB1 gene located immediately downstream of RDT1 (see Fig 2A schematic). SPB1 expression is not mating-type specific, it encodes a rRNA methyltransferase required for maturation of the large 60S ribosomal subunit [28], and interestingly, also functions in silencing at HML and HMR [29]. It was therefore intriguing that SBP1 expression was increased 2- to 3-fold in the sir2Δ and 100bpΔ mutants (Fig 3G), suggesting that Sir2 at the RDT1 promoter has a modest downstream repressive effect on SPB1, but not on the upstream R1L/S ncRNAs. Notably, steady state RDT1 and R1L/S RNA levels were relatively low compared to SPB1 and the ACT1 loading control, even in the 100bpΔ mutant (Fig 3G).

Sir2 and condensin are displaced from the RDT1 promoter during mating-type switchingWe next asked if Sir2 played any role in regulating RDT1 during mating-type switching. Sir2 was previously shown to associate with a HO-induced DSB at the MAT locus during mating-type switching, presumably to effect repair through histone deacetylation [30]. SIR complex association with DSBs occurs at the expense of telomere binding [31], so we hypothesized that the HO-induced DSB at MAT could also trigger loss of Sir2 from the RDT1 promoter, thus facilitating increased RDT1 transcription. To test this idea, HO was induced at time 0 with galactose and then turned off 2 hours later by glucose addition to allow for repair/switching to occur (Fig 4A and 4B). By the 3 hr time point (1 hr after glucose addition), ChIP analysis indicated Sir2 was maximally enriched at the MAT locus (Fig 4C), corresponding to the time of peak mating-type switching ([30] and Fig 4B). Interestingly, Sir2 was significantly depleted from the RDT1 promoter within 1 hr after HO induction, and by 3 hr there was actually stronger enrichment of Sir2 at MAT than RDT1 (Fig 4C). Critically, this shift in Sir2 distribution correlated with maximal induction of RDT1 mRNA and the Myc-tagged Rdt1protein (Fig 4D and 4E, 3 hr). Once switching was completed by 4 hr (2 hr after glucose addition), RDT1 transcription was permanently inactivated and Sir2 binding never returned because most cells were now MATα. The Myc-tagged Rdt1 protein, however, remained elevated for the rest of the time course (Fig 4E), suggesting that it is relatively stable, at least when epitope tagged. A parallel ChIP time course experiment was performed with condensin (Brn1-myc), indicating significant depletion from the RDT1 promoter within 1 hr (Fig 4F), similar to the timing of Sir2 loss. Unlike Sir2, Brn1-myc enrichment at the HO-cleaved MAT site did not increase, suggesting that condensin normally associates with MATa in non-switching cells and is then displaced in response to the HO-induced DSB, perhaps to facilitate structural reorganization associated with switching.10.1371/journal.pgen.1008339.g004Fig 4Dynamics of Sir2 and condensin binding at the RDT1 promoter and MATa locus during mating-type switching.(A, B) Quantitation of a mating-type switching time course where HO was induced by galactose at time 0, then glucose added at 2 hr to stop HO expression and allow for break repair. Switching is maximal at 3 hr. The average ratios of MATa and MATα PCR products relative to an SCR1 loading control PCR product were plotted (n = 3 biological replicates). (C) ChIP of Sir2 at the RDT1 promoter and the HO-induced DSB site (MAT-HO). (D) qRT-PCR of RDT1 expression relative to ACT1 across the mating-type switching time course. The time 0 level is normalized to 1. (E) Rdt1-13xMyc protein expression across the same time course. (F) ChIP of Brn1-myc at the RDT1 promoter and MAT-HO break site across the same time course. ChIP signal PCR is plotted relative to the input signal. (*p<0.05, **p<0.005 compared to time 0).Since RDT1 was highly expressed during mating-type switching, we next asked whether the small protein encoded by this gene had a direct function during the switching process when using the galactose-inducible system employed in this study. The 28 amino acid ORF was precisely deleted using the delitto perfetto method [32], and the efficiency of switching from MATa to MATα was then tested across a time course by PCR (S4A and S4B Fig). No significant differences were observed between the WT and rdt1Δ strains. Next, donor preference was tested using a strain previously developed by the Haber lab [15], whereby HMRa was replaced with HMRα containing a unique BamHI site (HMRα-B) (S4C Fig). Following completion of Gal-HO-induced switching from MATa to MATα, the proportion of donor utilization was determined by BamHI digest of a MATα PCR product. As shown in S4D and S4E Fig, deleting RDT1 also had no effect on donor preference, indicated by low utilization (~9%) of HMRα-B. Therefore, although RDT1 gene expression strongly correlates with switching, a specific function for its gene product remains elusive. Therefore, we shifted our attention to a possible function for the RDT1 promoter.

Sir2 and condensin are displaced from the RDT1 promoter during mating-type switching

We next asked if Sir2 played any role in regulating RDT1 during mating-type switching. Sir2 was previously shown to associate with a HO-induced DSB at the MAT locus during mating-type switching, presumably to effect repair through histone deacetylation [30]. SIR complex association with DSBs occurs at the expense of telomere binding [31], so we hypothesized that the HO-induced DSB at MAT could also trigger loss of Sir2 from the RDT1 promoter, thus facilitating increased RDT1 transcription. To test this idea, HO was induced at time 0 with galactose and then turned off 2 hours later by glucose addition to allow for repair/switching to occur (Fig 4A and 4B). By the 3 hr time point (1 hr after glucose addition), ChIP analysis indicated Sir2 was maximally enriched at the MAT locus (Fig 4C), corresponding to the time of peak mating-type switching ([30] and Fig 4B). Interestingly, Sir2 was significantly depleted from the RDT1 promoter within 1 hr after HO induction, and by 3 hr there was actually stronger enrichment of Sir2 at MAT than RDT1 (Fig 4C). Critically, this shift in Sir2 distribution correlated with maximal induction of RDT1 mRNA and the Myc-tagged Rdt1protein (Fig 4D and 4E, 3 hr). Once switching was completed by 4 hr (2 hr after glucose addition), RDT1 transcription was permanently inactivated and Sir2 binding never returned because most cells were now MATα. The Myc-tagged Rdt1 protein, however, remained elevated for the rest of the time course (Fig 4E), suggesting that it is relatively stable, at least when epitope tagged. A parallel ChIP time course experiment was performed with condensin (Brn1-myc), indicating significant depletion from the RDT1 promoter within 1 hr (Fig 4F), similar to the timing of Sir2 loss. Unlike Sir2, Brn1-myc enrichment at the HO-cleaved MAT site did not increase, suggesting that condensin normally associates with MATa in non-switching cells and is then displaced in response to the HO-induced DSB, perhaps to facilitate structural reorganization associated with switching.

Dynamics of Sir2 and condensin binding at the RDT1 promoter and MATa locus during mating-type switching.

(A, B) Quantitation of a mating-type switching time course where HO was induced by galactose at time 0, then glucose added at 2 hr to stop HO expression and allow for break repair. Switching is maximal at 3 hr. The average ratios of MATa and MATα PCR products relative to an SCR1 loading control PCR product were plotted (n = 3 biological replicates). (C) ChIP of Sir2 at the RDT1 promoter and the HO-induced DSB site (MAT-HO). (D) qRT-PCR of RDT1 expression relative to ACT1 across the mating-type switching time course. The time 0 level is normalized to 1. (E) Rdt1-13xMyc protein expression across the same time course. (F) ChIP of Brn1-myc at the RDT1 promoter and MAT-HO break site across the same time course. ChIP signal PCR is plotted relative to the input signal. (*p<0.05, **p<0.005 compared to time 0).

Since RDT1 was highly expressed during mating-type switching, we next asked whether the small protein encoded by this gene had a direct function during the switching process when using the galactose-inducible system employed in this study. The 28 amino acid ORF was precisely deleted using the delitto perfetto method [32], and the efficiency of switching from MATa to MATα was then tested across a time course by PCR (S4A and S4B Fig). No significant differences were observed between the WT and rdt1Δ strains. Next, donor preference was tested using a strain previously developed by the Haber lab [15], whereby HMRa was replaced with HMRα containing a unique BamHI site (HMRα-B) (S4C Fig). Following completion of Gal-HO-induced switching from MATa to MATα, the proportion of donor utilization was determined by BamHI digest of a MATα PCR product. As shown in S4D and S4E Fig, deleting RDT1 also had no effect on donor preference, indicated by low utilization (~9%) of HMRα-B. Therefore, although RDT1 gene expression strongly correlates with switching, a specific function for its gene product remains elusive. Therefore, we shifted our attention to a possible function for the RDT1 promoter.

The RDT1 promoter region controls chromosome III architectureThe coupling of Sir2 and condensin distribution with RDT1 transcriptional regulation during mating-type switching was reminiscent of classic locus control regions (LCR) that modulate long-range chromatin interactions [33, 34]. We therefore hypothesized that the RDT1 promoter region functions as an LCR to modulate long-range chromatin interactions of chromosome III. To test this hypothesis, we performed Hi-C analysis with WT, sir2Δ and the 100bpΔ strains. Genomic contact differences between the mutants and WT were quantified using the HOMER Hi-C software suite [35], and the frequency of statistically significant differences for each chromosome calculated (Fig 5A). Chromosome III had the most significant differences in both mutants, so we focused on this chromosome and used HOMER to plot the observed/expected interaction frequency in 10kb bins for each strain as a heat map (Fig 5B). In a WT strain (ML1) there was strong interaction between the left and right ends of chromosome III, mostly centered around the HML (bin 2) and HMR (bin 29) loci. Interestingly, HML (bin 2) also appeared to sample the entire right arm of chromosome III, with the interaction frequency increasing as a gradient from CEN3 to a maximal observed interaction at HMR, thus also encompassing the MATa locus at bin 20. This distinct interaction pattern was completely disrupted in the sir2Δ mutant, whereas some telomere-subtelomere contacts were retained in the 100bpΔ mutant (Fig 5B), suggesting there was still limited interaction between the left and right ends of the chromosome. We confirmed the changes in HML-HMR interaction for these strains using a quantitative 3C-PCR assay to rule out sequencing artifacts (Fig 5C), and to confirm an earlier sir2Δ 3C result from the Dekker lab [36]. Importantly, despite the loss of HML-HMR interaction in the 100bpΔ mutant, heterochromatin at these domains was unaffected based on normal quantitative mating assays (S5A Fig), and unaltered Sir2 association with HML (S5B Fig).10.1371/journal.pgen.1008339.g005Fig 5The Sir2/condensin binding site controls chromosome III architecture.(A) Frequency of significant Hi-C interaction changes identified using HOMER for each chromosome in the sir2Δ (ML25) and 100bpΔ (ML275) strains compared to WT (ML1). (B) HOMER-generated log2 observed/expected Hi-C interaction frequency heat maps (10kb bins) for chromosome III. Solid white lines indicate bins with insufficient read coverage post-filtering. (C) qPCR detection of HML-HMR interaction using 3C analysis, with the WT signal normalized to 1. (*p < 0.05, **<0.005). (D) Iteratively corrected and read-normalized Hi-C heat maps revealing an interaction between HMR (bin 29) and MATa (bin 20) in the sir2Δ and 100bpΔ mutants (black arrow). Solid black lines indicate bins with insufficient read coverage post-filtering. Red arrows indicate the HML-HMR interaction. (E) Summary of large-scale changes in chromosome III architecture. Δ indicates the 100bp deletion.We next analyzed the Hi-C data using an iterative correction method that reduces background to reveal interacting loci that potentially drive the overall chromosomal architecture, rather than passenger locus effects [37]. HML (bin 2) and HMR (bin 29) again formed the dominant interaction pair off the diagonal in WT, which was lost in the sir2Δ or 100bpΔ mutants (Fig 5D, red arrows). Importantly, a prominent new interaction between HMR (bin 29) and MATa (bin 20) appeared in both mutants (Fig 5D, black arrows), as would be predicted if normal donor preference of MATa cells was altered. We conclude that the RDT1 promoter does function like an LCR in MATa yeast cells, regulating localized transcription and establishing a long-range chromatin interaction between HML and HMR that appears to prevent HMR from strongly associating with MATa (Fig 5E).

The RDT1 promoter region controls chromosome III architecture

The coupling of Sir2 and condensin distribution with RDT1 transcriptional regulation during mating-type switching was reminiscent of classic locus control regions (LCR) that modulate long-range chromatin interactions [33, 34]. We therefore hypothesized that the RDT1 promoter region functions as an LCR to modulate long-range chromatin interactions of chromosome III. To test this hypothesis, we performed Hi-C analysis with WT, sir2Δ and the 100bpΔ strains. Genomic contact differences between the mutants and WT were quantified using the HOMER Hi-C software suite [35], and the frequency of statistically significant differences for each chromosome calculated (Fig 5A). Chromosome III had the most significant differences in both mutants, so we focused on this chromosome and used HOMER to plot the observed/expected interaction frequency in 10kb bins for each strain as a heat map (Fig 5B). In a WT strain (ML1) there was strong interaction between the left and right ends of chromosome III, mostly centered around the HML (bin 2) and HMR (bin 29) loci. Interestingly, HML (bin 2) also appeared to sample the entire right arm of chromosome III, with the interaction frequency increasing as a gradient from CEN3 to a maximal observed interaction at HMR, thus also encompassing the MATa locus at bin 20. This distinct interaction pattern was completely disrupted in the sir2Δ mutant, whereas some telomere-subtelomere contacts were retained in the 100bpΔ mutant (Fig 5B), suggesting there was still limited interaction between the left and right ends of the chromosome. We confirmed the changes in HML-HMR interaction for these strains using a quantitative 3C-PCR assay to rule out sequencing artifacts (Fig 5C), and to confirm an earlier sir2Δ 3C result from the Dekker lab [36]. Importantly, despite the loss of HML-HMR interaction in the 100bpΔ mutant, heterochromatin at these domains was unaffected based on normal quantitative mating assays (S5A Fig), and unaltered Sir2 association with HML (S5B Fig).

The Sir2/condensin binding site controls chromosome III architecture.

(A) Frequency of significant Hi-C interaction changes identified using HOMER for each chromosome in the sir2Δ (ML25) and 100bpΔ (ML275) strains compared to WT (ML1). (B) HOMER-generated log2 observed/expected Hi-C interaction frequency heat maps (10kb bins) for chromosome III. Solid white lines indicate bins with insufficient read coverage post-filtering. (C) qPCR detection of HML-HMR interaction using 3C analysis, with the WT signal normalized to 1. (*p < 0.05, **<0.005). (D) Iteratively corrected and read-normalized Hi-C heat maps revealing an interaction between HMR (bin 29) and MATa (bin 20) in the sir2Δ and 100bpΔ mutants (black arrow). Solid black lines indicate bins with insufficient read coverage post-filtering. Red arrows indicate the HML-HMR interaction. (E) Summary of large-scale changes in chromosome III architecture. Δ indicates the 100bp deletion.

We next analyzed the Hi-C data using an iterative correction method that reduces background to reveal interacting loci that potentially drive the overall chromosomal architecture, rather than passenger locus effects [37]. HML (bin 2) and HMR (bin 29) again formed the dominant interaction pair off the diagonal in WT, which was lost in the sir2Δ or 100bpΔ mutants (Fig 5D, red arrows). Importantly, a prominent new interaction between HMR (bin 29) and MATa (bin 20) appeared in both mutants (Fig 5D, black arrows), as would be predicted if normal donor preference of MATa cells was altered. We conclude that the RDT1 promoter does function like an LCR in MATa yeast cells, regulating localized transcription and establishing a long-range chromatin interaction between HML and HMR that appears to prevent HMR from strongly associating with MATa (Fig 5E).

Sir2 and condensin regulate mating-type switchingSir2/condensin binding was observed in the right half of the RE (Fig 1A), but this region was previously reported as dispensable for donor preference activity [10]. Considering that HMR was aberrantly associated with the MATa locus in sir2Δ and 100bpΔ mutants (Fig 5B–5E), we proceeded to test whether these mutants had any alterations in donor preference. As was done for the rdt1Δ mutant in S4C and S4D Fig, the 100bpΔ mutation was introduced into a reporter strain with HMRα-B on the right arm of chromosome III (Fig 6A; [15]). After inducing switching to MATα with galactose, the proportion of HMLα or HMRα-B used for the switching was determined by BamHI digestion of a MATα-specific PCR product (Fig 6B; [15]). As expected for normal donor preference, HMRα-B on the right arm was only utilized ~9% of the time in the WT strain (Fig 6C and 6D). Strikingly, donor preference was lost in the sir2Δ mutant, similar to a control strain with the RE deleted (Fig 6C and 6D), and consistent with the clear interaction between HMR and MATa bins observed for the sir2Δ mutant in Fig 5D and 5E. This interaction was less prominent in the 100bpΔ mutant (Fig 5D), and the corresponding donor preference defect was also less severe (~25% HMRα-B), though still significantly different from WT (Fig 6C and 6D).10.1371/journal.pgen.1008339.g006Fig 6Loss of Sir2 and the Sir2/condensin binding site alters mating-type switching.(A) Schematic of a donor preference assay in which utilization of an artificial HMRα-B cassette as the donor for switching introduces a unique BamHI site to the MAT locus. (B) Locations of primers flanking the BamHI site used for PCR detection of MATα. (C) Representative ethidium bromide stained agarose gel of BamHI-digested MATα PCR products after mating-type switching in WT (XW652), reΔ (XW676), sir2Δ (ML557), and 100bpΔ (SY742) strains. The MATα-B product is digested into 2 smaller bands. (D) Quantifying the percentage of MATα-B PCR product digested by BamHI, from three biological replicates. ImageJ was used for the quantitation. (*p < 0.05, **p < 0.005). (E) Representative time course of switching from MATa to MATα in WT (ML447), 100bpΔ (ML460), and sir2Δ (ML458) strains after HO was induced for 45 min and then shut down with glucose. Aliquots were harvested at 30 min intervals. SCR1 is a control for input genomic DNA. (F) ImageJ quantification of MATα PCR relative to SCR1 for each time point. n = 3 biological replicates.Since the donor preference assay is an endpoint experiment, we next tested whether the sir2Δ or 100bpΔ mutations temporally impacted the efficiency of switching from MATa to MATα in the same ML1 background strains used for Hi-C analysis. As shown in Fig 6E and 6F, switching efficiency was dramatically impaired in the sir2Δ strain, but unaffected in the 100bpΔ mutant. These results suggest a model whereby condensin and Sir2 recruitment to the RDT1 promoter supports donor preference by organizing chromosome III into a structure that limits HMR association with the MATa locus, but is not required for the mechanics of mating-type switching. We suspect at least part of the strong sir2Δ effect on chromosome III organization and donor preference is caused by HMLALPHA2 derepression, which inactivates the RE due to formation of the Mcm1/α2 heterodimer [13]. We also considered a possibility that the sir2Δ heterochromatin defect at HML and HMR could make them highly accessible to HO cleavage [38], which would prevent their usage as donor templates. As a measure of HO cleavage at MATa, HML, or HMR, we assayed for reduced PCR amplification across the recognition site following Gal-HO induction (S6 Fig). While MATa was equally cut by HO in WT and sir2Δ strains (S6A and S6B Fig), HML was only cut in the sir2Δ strain (S6A and S6C Fig), consistent with the idea of reduced HML availability for switching. Unexpectedly, HMR remained available as a template in the sir2Δ strain (S6A and S6D Fig). We confirmed the difference in HO accessibility between HML and HMR using real-time qPCR (S6E and S6F Fig), which could contribute to the extreme sir2Δ donor preference defect (Fig 6C and 6D). In the 100bpΔ mutant, which locally eliminates condensin recruitment at RDT1, the continued maintenance of heterochromatin at HML/HMR and telomeres (S5 Fig), as well as residual telomere clustering (Fig 5D), may partially buffer the resulting donor preference defect by still limiting contact between the subtelomeric HMR locus and MATa.Condensin recruitment to the RDT1 promoter does not appear critical for the mechanics of mating-type switching, as indicated by normal timing of switching in the 100bpΔ mutant (Fig 6E and 6F). However, condensin could still potentially impact the switching process independent of the RE. In order to test this hypothesis, we C-terminally tagged the Brn1 condensin subunit with an auxin-inducible degron (AID) fused with a V5 epitope, which allows for rapid depletion of tagged proteins upon the addition of auxin [39]. Indeed, Brn1-AID was effectively degraded within 15 min of adding auxin to cells, as measured by western blotting (S7A Fig), or ChIP at the RDT1 promoter (S7B Fig). Importantly, even after 1 hr of auxin treatment, there were no significant changes in RDT1 or HMLALPHA2 gene expression as measured by qRT-PCR (S7C and S7D Fig), thus indicating that silencing of HML was unaffected, unlike the ycs4-1 condensin mutant used in Fig 1G [22]. The efficiency of ML1 switching from MATa to MATα was then tested across a time course with or without auxin treatment (Fig 7A). As shown in Fig 7B and 7C, auxin significantly slowed the efficiency of switching to MATα, indicating that the Brn1 subunit of condensin is important for normal mating-type switching.10.1371/journal.pgen.1008339.g007Fig 7Effects of condensin depletion on mating-type switching.(A) Schematic of the time course used to deplete Brn1-AID prior to induction of mating-type switching in the ML1 strain background. Auxin was added 30 min prior to the induction of HO expression by galactose. (B) Representative EtBr stained agarose gel of MATα PCR products amplified from each time point during mating-type switching. SCR1 PCR was used as a control for input DNA. (C) Quantification of the MATα/SCR1 PCR product ratio across the time course from 3 biological replicates. (D) Effect of Brn1-AID depletion on mating-type switching donor preference. A representative biological replicate is shown, along with percentage use of HMRα-B cassette as the donor template. (*p < 0.05).Since the 100bpΔ mutant caused a modest donor preference defect that we partially attributed to a loss of condensin (Fig 6C and 6D), it was important to also test for a donor preference defect when condensin was depleted. Indeed, Brn1-AID depletion produced a significant defect in donor preference using the HMRα-B reporter strain (Fig 7D) that was similar in magnitude to that observed for the 100bpΔ strain (Fig 6D). Taken together, these results support a working model whereby condensin recruitment to the RDT1 promoter in MATa cells organizes chromosome III into a conformation that limits the association of HMR with MATa, thus partially contributing to donor preference regulation. We hypothesize that upon HO cleavage of MATa, the increased expression of RDT1 caused by loss of Sir2, displaces condensin from the promoter, which then allows the left half of the RE to physically direct HML to MATa for use as a donor (Fig 8).10.1371/journal.pgen.1008339.g008Fig 8Model for cooperative regulation of chromosome III structure and donor preference by the bipartite RE.The left half of the RE regulates donor preference through expression of the R1L and R1S long ncRNAs and an array of Fkh1 proteins that interact with threonine-phosphorylated proteins at the double stranded DNA break at MATa. The right half of the RE is bound by condensin and Sir2. Like R1L/S, the expression of RDT1 correlates with mating-type switching. Condensin is hypothesized to organize chromosome III structure through a unidirectional loop extrusion process that eventually places MATa into a chromosome-sized loop that helps insulates it from HML and HMR until condensin is removed at the time of switching. The loop is also likely maintained by clustering of telomeres and subtelomeres (black bars) and the heterochromatin set up by HML and HMR.

Sir2 and condensin regulate mating-type switching

Sir2/condensin binding was observed in the right half of the RE (Fig 1A), but this region was previously reported as dispensable for donor preference activity [10]. Considering that HMR was aberrantly associated with the MATa locus in sir2Δ and 100bpΔ mutants (Fig 5B–5E), we proceeded to test whether these mutants had any alterations in donor preference. As was done for the rdt1Δ mutant in S4C and S4D Fig, the 100bpΔ mutation was introduced into a reporter strain with HMRα-B on the right arm of chromosome III (Fig 6A; [15]). After inducing switching to MATα with galactose, the proportion of HMLα or HMRα-B used for the switching was determined by BamHI digestion of a MATα-specific PCR product (Fig 6B; [15]). As expected for normal donor preference, HMRα-B on the right arm was only utilized ~9% of the time in the WT strain (Fig 6C and 6D). Strikingly, donor preference was lost in the sir2Δ mutant, similar to a control strain with the RE deleted (Fig 6C and 6D), and consistent with the clear interaction between HMR and MATa bins observed for the sir2Δ mutant in Fig 5D and 5E. This interaction was less prominent in the 100bpΔ mutant (Fig 5D), and the corresponding donor preference defect was also less severe (~25% HMRα-B), though still significantly different from WT (Fig 6C and 6D).

Loss of Sir2 and the Sir2/condensin binding site alters mating-type switching.

(A) Schematic of a donor preference assay in which utilization of an artificial HMRα-B cassette as the donor for switching introduces a unique BamHI site to the MAT locus. (B) Locations of primers flanking the BamHI site used for PCR detection of MATα. (C) Representative ethidium bromide stained agarose gel of BamHI-digested MATα PCR products after mating-type switching in WT (XW652), reΔ (XW676), sir2Δ (ML557), and 100bpΔ (SY742) strains. The MATα-B product is digested into 2 smaller bands. (D) Quantifying the percentage of MATα-B PCR product digested by BamHI, from three biological replicates. ImageJ was used for the quantitation. (*p < 0.05, **p < 0.005). (E) Representative time course of switching from MATa to MATα in WT (ML447), 100bpΔ (ML460), and sir2Δ (ML458) strains after HO was induced for 45 min and then shut down with glucose. Aliquots were harvested at 30 min intervals. SCR1 is a control for input genomic DNA. (F) ImageJ quantification of MATα PCR relative to SCR1 for each time point. n = 3 biological replicates.

Since the donor preference assay is an endpoint experiment, we next tested whether the sir2Δ or 100bpΔ mutations temporally impacted the efficiency of switching from MATa to MATα in the same ML1 background strains used for Hi-C analysis. As shown in Fig 6E and 6F, switching efficiency was dramatically impaired in the sir2Δ strain, but unaffected in the 100bpΔ mutant. These results suggest a model whereby condensin and Sir2 recruitment to the RDT1 promoter supports donor preference by organizing chromosome III into a structure that limits HMR association with the MATa locus, but is not required for the mechanics of mating-type switching. We suspect at least part of the strong sir2Δ effect on chromosome III organization and donor preference is caused by HMLALPHA2 derepression, which inactivates the RE due to formation of the Mcm1/α2 heterodimer [13]. We also considered a possibility that the sir2Δ heterochromatin defect at HML and HMR could make them highly accessible to HO cleavage [38], which would prevent their usage as donor templates. As a measure of HO cleavage at MATa, HML, or HMR, we assayed for reduced PCR amplification across the recognition site following Gal-HO induction (S6 Fig). While MATa was equally cut by HO in WT and sir2Δ strains (S6A and S6B Fig), HML was only cut in the sir2Δ strain (S6A and S6C Fig), consistent with the idea of reduced HML availability for switching. Unexpectedly, HMR remained available as a template in the sir2Δ strain (S6A and S6D Fig). We confirmed the difference in HO accessibility between HML and HMR using real-time qPCR (S6E and S6F Fig), which could contribute to the extreme sir2Δ donor preference defect (Fig 6C and 6D). In the 100bpΔ mutant, which locally eliminates condensin recruitment at RDT1, the continued maintenance of heterochromatin at HML/HMR and telomeres (S5 Fig), as well as residual telomere clustering (Fig 5D), may partially buffer the resulting donor preference defect by still limiting contact between the subtelomeric HMR locus and MATa.

Condensin recruitment to the RDT1 promoter does not appear critical for the mechanics of mating-type switching, as indicated by normal timing of switching in the 100bpΔ mutant (Fig 6E and 6F). However, condensin could still potentially impact the switching process independent of the RE. In order to test this hypothesis, we C-terminally tagged the Brn1 condensin subunit with an auxin-inducible degron (AID) fused with a V5 epitope, which allows for rapid depletion of tagged proteins upon the addition of auxin [39]. Indeed, Brn1-AID was effectively degraded within 15 min of adding auxin to cells, as measured by western blotting (S7A Fig), or ChIP at the RDT1 promoter (S7B Fig). Importantly, even after 1 hr of auxin treatment, there were no significant changes in RDT1 or HMLALPHA2 gene expression as measured by qRT-PCR (S7C and S7D Fig), thus indicating that silencing of HML was unaffected, unlike the ycs4-1 condensin mutant used in Fig 1G [22]. The efficiency of ML1 switching from MATa to MATα was then tested across a time course with or without auxin treatment (Fig 7A). As shown in Fig 7B and 7C, auxin significantly slowed the efficiency of switching to MATα, indicating that the Brn1 subunit of condensin is important for normal mating-type switching.

Effects of condensin depletion on mating-type switching.

(A) Schematic of the time course used to deplete Brn1-AID prior to induction of mating-type switching in the ML1 strain background. Auxin was added 30 min prior to the induction of HO expression by galactose. (B) Representative EtBr stained agarose gel of MATα PCR products amplified from each time point during mating-type switching. SCR1 PCR was used as a control for input DNA. (C) Quantification of the MATα/SCR1 PCR product ratio across the time course from 3 biological replicates. (D) Effect of Brn1-AID depletion on mating-type switching donor preference. A representative biological replicate is shown, along with percentage use of HMRα-B cassette as the donor template. (*p < 0.05).

Since the 100bpΔ mutant caused a modest donor preference defect that we partially attributed to a loss of condensin (Fig 6C and 6D), it was important to also test for a donor preference defect when condensin was depleted. Indeed, Brn1-AID depletion produced a significant defect in donor preference using the HMRα-B reporter strain (Fig 7D) that was similar in magnitude to that observed for the 100bpΔ strain (Fig 6D). Taken together, these results support a working model whereby condensin recruitment to the RDT1 promoter in MATa cells organizes chromosome III into a conformation that limits the association of HMR with MATa, thus partially contributing to donor preference regulation. We hypothesize that upon HO cleavage of MATa, the increased expression of RDT1 caused by loss of Sir2, displaces condensin from the promoter, which then allows the left half of the RE to physically direct HML to MATa for use as a donor (Fig 8).

Model for cooperative regulation of chromosome III structure and donor preference by the bipartite RE.

The left half of the RE regulates donor preference through expression of the R1L and R1S long ncRNAs and an array of Fkh1 proteins that interact with threonine-phosphorylated proteins at the double stranded DNA break at MATa. The right half of the RE is bound by condensin and Sir2. Like R1L/S, the expression of RDT1 correlates with mating-type switching. Condensin is hypothesized to organize chromosome III structure through a unidirectional loop extrusion process that eventually places MATa into a chromosome-sized loop that helps insulates it from HML and HMR until condensin is removed at the time of switching. The loop is also likely maintained by clustering of telomeres and subtelomeres (black bars) and the heterochromatin set up by HML and HMR.

DiscussionSIR2 was identified ~40 years ago as a recessive mutation unlinked from HML and HMR that caused their derepression [3, 4], and has been extensively studied ever since as encoding a heterochromatin factor that functions not only at the HM loci, but also telomeres and the rDNA locus (reviewed in [5]). In this study we describe a previously unidentified Sir2 binding site that overlaps with a major non-rDNA condensin binding site within the RE on chromosome III in MATa cells. Here, Sir2 regulates a small gene of unknown function called RDT1, which is transcriptionally activated during mating-type switching due to loss of repressive Sir2 from the RDT1 promoter that correlates with binding to the HO-induced DSB at MATa. The RDT1 RNA transcript is also translated into a small protein, but we have not yet been able to assign a function to the gene or protein because deleting the 28 amino acid ORF does not measurably alter mating-type switching when using GAL-HO based assays. It remains possible that deleting RDT1 would have a significant effect on switching in the context of native HO expression, which is expressed only in mother cells during late G1, whereas the GAL1-HO is overexpressed in all cells throughout the cell cycle. Furthermore, RDT1 and the R1L/S ncRNAs are themselves cell cycle regulated, similarly showing peak expression around late G1 [27]. Therefore, even with the Gal-HO induced system, the strong RDT1 expression observed during switching (Fig 4D) suggests significant enrichment of G1 cells in the population. It is also possible that RDT1 functions as a non-coding RNA that happens to be translated into a small non-functional peptide. Alternatively, transcription of RDT1 could directly function in chromosome III conformation by altering local chromatin accessibility at the promoter. Such a model was proposed for regulation of donor preference by transcription of the R1S/R1L non-coding RNAs [13, 27]. Dissecting the function(s) of RDT1 through the cell cycle therefore remains an area of active investigation for the lab, and perhaps the key to fully understanding how its promoter functions as an LCR.Functional complexity within the REWhile we do not yet know the molecular function of RDT1 in mating-type regulation or other cellular processes, the promoter region of this gene clearly controls the structure of chromosome III. Three-dimensional chromatin structure has long been proposed to influence donor preference [40, 41]. However, deleting the minimal 700bp (left half) of the RE alters donor preference without a large change in chromosome III conformation. Furthermore, deleting the right half of the RE, which includes RDT1, changes chromosome III conformation without a dramatic change in donor preference [10, 13, 42]. Based on these findings it was proposed that the RE is a bipartite regulatory element [42], with the left half primarily responsible for donor preference activity and the right half for chromosome III structure. Our results support this view and narrow down the structural regulatory domain of the RE to a small (100bp) region of the RDT1 promoter bound by the SIR and condensin complexes. Importantly, deleting this small region not only altered chromosome III structure, but also had a significant effect on donor preference, though not as strong as the sir2Δ mutation.The coordination of RDT1 expression with loss of Sir2/condensin binding at its promoter during mating-type switching, together with the loss of HML-HMR interaction in the 100bpΔ mutant, makes this site intriguingly similar to classic locus control regions (LCRs) in metazoans, which are cis-acting domains that contain a mixture of enhancers, insulators, chromatin opening elements, and tissue-specificity elements [33]. The minimal RE was previously described as an LCR in the context of donor preference [10], and transcription of the R1S/R1L long non-coding RNAs via activation by the 1st Mcm1/α2 binding site (DPS1) appears to be important for this activity in MATa cells [27]. We find that Sir2 indirectly supports donor preference from the left half of the RE in MATa cells by silencing HMLALPHA2 expression, which prevents transcriptional repression by an Mcm1/α2 heterodimer, and by protecting the HML template from HO cleavage. Similarly, the loss of Sir2 also represses RDT1 expression and condensin recruitment in the right half of the RE due to HMLALPHA2 expression. It remains possible that SIR-dependent heterochromatin formation also directly contributes to the HML-HMR interaction through clustering. More clearly, however, Sir2 directly represses RDT1 through localized histone deacetylation. How the loss of RDT1 regulation and condensin recruitment changes chromosome III structure in the sir2Δ mutant remains unknown, but we propose that the HMR-MATa interaction is a default state, while the HML-HMR association has to be actively maintained by condensin and likely additional factors co-localized to this element, as well as SIR-dependent heterochromatin.Interestingly, there also appears to be a functional relationship between the RE and silencing at the HML locus, such that deleting the left half of the RE specifically stabilizes HML silencing in MATa cells [43]. The mechanism involved remains unknown, but we hypothesize that eliminating this part of the RE could potentially allow the SIR and condensin complexes bound at the RDT1 promoter to encroach and somehow enhance the heterochromatic structure at HML. Under this scenario, the left half of the RE could be insulating HML from the chromosomal organizing activity that occurs at the RDT1 promoter.Condensin function in mating-type switchingThe RDT1 promoter is a major condensin binding site identified by ChIP-seq (Fig 1), and given the strong Hi-C interaction between HML and HMR, we initially hypothesized that condensin at the RDT1 promoter would crosslink with another condensin site bound on the right arm of chromosome III. However, this turned out to be unlikely because ChIP-seq of Smc4-myc did not reveal any strong peaks near HMR. The S. cerevisiae condensin complex was recently shown to catalyze ATP-dependent unidirectional loop extrusion using an in vitro single molecule assay [44]. The mechanism involves direct binding of condensin to DNA, followed by one end of the bound DNA being pulled inward as an extruded loop. Applying this model to the strong binding site at the RDT1 promoter, this region could act as an anchor bound by condensin, with DNA to the right being rapidly extruded as a loop until pausing at CEN3. Extrusion would then continue at a slower rate toward HMR, allowing HML time to sample the entire right arm of chromosome III, until clustering with HMR (Fig 8). HOMER analysis of the Hi-C data in Fig 5B provides evidence for such a model because there is an ascending gradient of HML interaction frequency with sequences extending from the centromere region (bin 12) toward HMR, suggesting that HML “samples” the right arm of chromosome III. Once brought in contact, HML and HMR would then remain associated due to their heterochromatic states and shared retention at the nuclear envelope [45]. Formation of this loop appears to limit HMR association with MATa, but since the 100bpΔ mutation has no effect on the timing of switching, we do not believe that condensin at the RDT1 promoter functions directly in the homologous recombination process. Rather, general Brn1 subunit depletion could slow the switching process by affecting chromosome III flexibility or conformational dynamics.MATa specific recruitment of Sir2 and condensin to the RECondensin, and Sir2 each strongly associated with the RDT1 promoter exclusively in MATa cells, though it is not clear if they bind at the same time, or are differentially bound throughout the cell cycle. Since DPS2 was required for Sir2 and condensin recruitment, and derepression of HMLALPHA2 from HML also eliminated binding, we hypothesized and then demonstrated (S3 Fig) that Mcm1 was a key DNA binding factor involved. Mcm1 is a prototypical MADS box combinatorial transcription factor that derives its regulatory specificity through interactions with other factors, such as Ste12 in the case of MATa haploid-specific gene activation, or α2 when repressing the same target genes in MATα cells [46]. This raises the question of whether Mcm1 directly recruits the SIR and condensin complexes, or perhaps additional factors that work with Mcm1 are involved. The latter is likely true because condensin and Sir2 are not recruited to the leftmost Mcm1/α2 binding site in the left half of the RE, as indicated by the ChIP-seq data in Fig 1A. At the RDT1 promoter, specificity for Sir2/condensin recruitment could originate from sequences underlying the condensin/Sir2 peaks. There are no traditional silencer-like sequences for SIR recruitment within the deleted 100bp (coordinates 30702 to 30801), and yeast condensin does not appear to have a consensus DNA binding sequence [47]. Closer inspection of the RDT1 promoter indicates an A/T rich region with consensus binding sites for the transcription factors Fkh1/2 and Ash1, each of which regulates mating-type switching [11, 48, 49]. Fkh1 and Fkh2 also physically associate with Sir2 at the mitotic cyclin CLB2 promoter during stress [50]. Ash1 is intriguing because it represses HO transcription in daughter cells [49, 51], raising the possibility of RDT1 repression in daughter cells. Mcm1 activity in MATa cells could also indirectly establish a chromatin environment that is competent for Sir2/condensin recruitment, rather than direct recruitment through protein-protein interactions. In MATa cells, Mcm1 appears to prevent the strong nucleosome positioning across the RE that occurs in MATα cells [27], and indicative of an actively remodeled chromatin environment. Perhaps condensin is attracted to such regions, which is consistent with the association of condensin with promoters of active genes in mitotic cells, where enrichment was greatest at unwound regions of DNA [52]. Furthermore, nucleosome eviction by transcriptional coactivators was found to assist condensin loading in yeast [53], though the mechanism of loading remains poorly understood. Recruitment of condensin to the RDT1 promoter LCR therefore provides an outstanding opportunity for dissecting mechanisms of condensin loading and function.

SIR2 was identified ~40 years ago as a recessive mutation unlinked from HML and HMR that caused their derepression [3, 4], and has been extensively studied ever since as encoding a heterochromatin factor that functions not only at the HM loci, but also telomeres and the rDNA locus (reviewed in [5]). In this study we describe a previously unidentified Sir2 binding site that overlaps with a major non-rDNA condensin binding site within the RE on chromosome III in MATa cells. Here, Sir2 regulates a small gene of unknown function called RDT1, which is transcriptionally activated during mating-type switching due to loss of repressive Sir2 from the RDT1 promoter that correlates with binding to the HO-induced DSB at MATa. The RDT1 RNA transcript is also translated into a small protein, but we have not yet been able to assign a function to the gene or protein because deleting the 28 amino acid ORF does not measurably alter mating-type switching when using GAL-HO based assays. It remains possible that deleting RDT1 would have a significant effect on switching in the context of native HO expression, which is expressed only in mother cells during late G1, whereas the GAL1-HO is overexpressed in all cells throughout the cell cycle. Furthermore, RDT1 and the R1L/S ncRNAs are themselves cell cycle regulated, similarly showing peak expression around late G1 [27]. Therefore, even with the Gal-HO induced system, the strong RDT1 expression observed during switching (Fig 4D) suggests significant enrichment of G1 cells in the population. It is also possible that RDT1 functions as a non-coding RNA that happens to be translated into a small non-functional peptide. Alternatively, transcription of RDT1 could directly function in chromosome III conformation by altering local chromatin accessibility at the promoter. Such a model was proposed for regulation of donor preference by transcription of the R1S/R1L non-coding RNAs [13, 27]. Dissecting the function(s) of RDT1 through the cell cycle therefore remains an area of active investigation for the lab, and perhaps the key to fully understanding how its promoter functions as an LCR.

Functional complexity within the REWhile we do not yet know the molecular function of RDT1 in mating-type regulation or other cellular processes, the promoter region of this gene clearly controls the structure of chromosome III. Three-dimensional chromatin structure has long been proposed to influence donor preference [40, 41]. However, deleting the minimal 700bp (left half) of the RE alters donor preference without a large change in chromosome III conformation. Furthermore, deleting the right half of the RE, which includes RDT1, changes chromosome III conformation without a dramatic change in donor preference [10, 13, 42]. Based on these findings it was proposed that the RE is a bipartite regulatory element [42], with the left half primarily responsible for donor preference activity and the right half for chromosome III structure. Our results support this view and narrow down the structural regulatory domain of the RE to a small (100bp) region of the RDT1 promoter bound by the SIR and condensin complexes. Importantly, deleting this small region not only altered chromosome III structure, but also had a significant effect on donor preference, though not as strong as the sir2Δ mutation.The coordination of RDT1 expression with loss of Sir2/condensin binding at its promoter during mating-type switching, together with the loss of HML-HMR interaction in the 100bpΔ mutant, makes this site intriguingly similar to classic locus control regions (LCRs) in metazoans, which are cis-acting domains that contain a mixture of enhancers, insulators, chromatin opening elements, and tissue-specificity elements [33]. The minimal RE was previously described as an LCR in the context of donor preference [10], and transcription of the R1S/R1L long non-coding RNAs via activation by the 1st Mcm1/α2 binding site (DPS1) appears to be important for this activity in MATa cells [27]. We find that Sir2 indirectly supports donor preference from the left half of the RE in MATa cells by silencing HMLALPHA2 expression, which prevents transcriptional repression by an Mcm1/α2 heterodimer, and by protecting the HML template from HO cleavage. Similarly, the loss of Sir2 also represses RDT1 expression and condensin recruitment in the right half of the RE due to HMLALPHA2 expression. It remains possible that SIR-dependent heterochromatin formation also directly contributes to the HML-HMR interaction through clustering. More clearly, however, Sir2 directly represses RDT1 through localized histone deacetylation. How the loss of RDT1 regulation and condensin recruitment changes chromosome III structure in the sir2Δ mutant remains unknown, but we propose that the HMR-MATa interaction is a default state, while the HML-HMR association has to be actively maintained by condensin and likely additional factors co-localized to this element, as well as SIR-dependent heterochromatin.Interestingly, there also appears to be a functional relationship between the RE and silencing at the HML locus, such that deleting the left half of the RE specifically stabilizes HML silencing in MATa cells [43]. The mechanism involved remains unknown, but we hypothesize that eliminating this part of the RE could potentially allow the SIR and condensin complexes bound at the RDT1 promoter to encroach and somehow enhance the heterochromatic structure at HML. Under this scenario, the left half of the RE could be insulating HML from the chromosomal organizing activity that occurs at the RDT1 promoter.

Functional complexity within the RE

While we do not yet know the molecular function of RDT1 in mating-type regulation or other cellular processes, the promoter region of this gene clearly controls the structure of chromosome III. Three-dimensional chromatin structure has long been proposed to influence donor preference [40, 41]. However, deleting the minimal 700bp (left half) of the RE alters donor preference without a large change in chromosome III conformation. Furthermore, deleting the right half of the RE, which includes RDT1, changes chromosome III conformation without a dramatic change in donor preference [10, 13, 42]. Based on these findings it was proposed that the RE is a bipartite regulatory element [42], with the left half primarily responsible for donor preference activity and the right half for chromosome III structure. Our results support this view and narrow down the structural regulatory domain of the RE to a small (100bp) region of the RDT1 promoter bound by the SIR and condensin complexes. Importantly, deleting this small region not only altered chromosome III structure, but also had a significant effect on donor preference, though not as strong as the sir2Δ mutation.

The coordination of RDT1 expression with loss of Sir2/condensin binding at its promoter during mating-type switching, together with the loss of HML-HMR interaction in the 100bpΔ mutant, makes this site intriguingly similar to classic locus control regions (LCRs) in metazoans, which are cis-acting domains that contain a mixture of enhancers, insulators, chromatin opening elements, and tissue-specificity elements [33]. The minimal RE was previously described as an LCR in the context of donor preference [10], and transcription of the R1S/R1L long non-coding RNAs via activation by the 1st Mcm1/α2 binding site (DPS1) appears to be important for this activity in MATa cells [27]. We find that Sir2 indirectly supports donor preference from the left half of the RE in MATa cells by silencing HMLALPHA2 expression, which prevents transcriptional repression by an Mcm1/α2 heterodimer, and by protecting the HML template from HO cleavage. Similarly, the loss of Sir2 also represses RDT1 expression and condensin recruitment in the right half of the RE due to HMLALPHA2 expression. It remains possible that SIR-dependent heterochromatin formation also directly contributes to the HML-HMR interaction through clustering. More clearly, however, Sir2 directly represses RDT1 through localized histone deacetylation. How the loss of RDT1 regulation and condensin recruitment changes chromosome III structure in the sir2Δ mutant remains unknown, but we propose that the HMR-MATa interaction is a default state, while the HML-HMR association has to be actively maintained by condensin and likely additional factors co-localized to this element, as well as SIR-dependent heterochromatin.

Interestingly, there also appears to be a functional relationship between the RE and silencing at the HML locus, such that deleting the left half of the RE specifically stabilizes HML silencing in MATa cells [43]. The mechanism involved remains unknown, but we hypothesize that eliminating this part of the RE could potentially allow the SIR and condensin complexes bound at the RDT1 promoter to encroach and somehow enhance the heterochromatic structure at HML. Under this scenario, the left half of the RE could be insulating HML from the chromosomal organizing activity that occurs at the RDT1 promoter.

Condensin function in mating-type switchingThe RDT1 promoter is a major condensin binding site identified by ChIP-seq (Fig 1), and given the strong Hi-C interaction between HML and HMR, we initially hypothesized that condensin at the RDT1 promoter would crosslink with another condensin site bound on the right arm of chromosome III. However, this turned out to be unlikely because ChIP-seq of Smc4-myc did not reveal any strong peaks near HMR. The S. cerevisiae condensin complex was recently shown to catalyze ATP-dependent unidirectional loop extrusion using an in vitro single molecule assay [44]. The mechanism involves direct binding of condensin to DNA, followed by one end of the bound DNA being pulled inward as an extruded loop. Applying this model to the strong binding site at the RDT1 promoter, this region could act as an anchor bound by condensin, with DNA to the right being rapidly extruded as a loop until pausing at CEN3. Extrusion would then continue at a slower rate toward HMR, allowing HML time to sample the entire right arm of chromosome III, until clustering with HMR (Fig 8). HOMER analysis of the Hi-C data in Fig 5B provides evidence for such a model because there is an ascending gradient of HML interaction frequency with sequences extending from the centromere region (bin 12) toward HMR, suggesting that HML “samples” the right arm of chromosome III. Once brought in contact, HML and HMR would then remain associated due to their heterochromatic states and shared retention at the nuclear envelope [45]. Formation of this loop appears to limit HMR association with MATa, but since the 100bpΔ mutation has no effect on the timing of switching, we do not believe that condensin at the RDT1 promoter functions directly in the homologous recombination process. Rather, general Brn1 subunit depletion could slow the switching process by affecting chromosome III flexibility or conformational dynamics.

Condensin function in mating-type switching

The RDT1 promoter is a major condensin binding site identified by ChIP-seq (Fig 1), and given the strong Hi-C interaction between HML and HMR, we initially hypothesized that condensin at the RDT1 promoter would crosslink with another condensin site bound on the right arm of chromosome III. However, this turned out to be unlikely because ChIP-seq of Smc4-myc did not reveal any strong peaks near HMR. The S. cerevisiae condensin complex was recently shown to catalyze ATP-dependent unidirectional loop extrusion using an in vitro single molecule assay [44]. The mechanism involves direct binding of condensin to DNA, followed by one end of the bound DNA being pulled inward as an extruded loop. Applying this model to the strong binding site at the RDT1 promoter, this region could act as an anchor bound by condensin, with DNA to the right being rapidly extruded as a loop until pausing at CEN3. Extrusion would then continue at a slower rate toward HMR, allowing HML time to sample the entire right arm of chromosome III, until clustering with HMR (Fig 8). HOMER analysis of the Hi-C data in Fig 5B provides evidence for such a model because there is an ascending gradient of HML interaction frequency with sequences extending from the centromere region (bin 12) toward HMR, suggesting that HML “samples” the right arm of chromosome III. Once brought in contact, HML and HMR would then remain associated due to their heterochromatic states and shared retention at the nuclear envelope [45]. Formation of this loop appears to limit HMR association with MATa, but since the 100bpΔ mutation has no effect on the timing of switching, we do not believe that condensin at the RDT1 promoter functions directly in the homologous recombination process. Rather, general Brn1 subunit depletion could slow the switching process by affecting chromosome III flexibility or conformational dynamics.

MATa specific recruitment of Sir2 and condensin to the RECondensin, and Sir2 each strongly associated with the RDT1 promoter exclusively in MATa cells, though it is not clear if they bind at the same time, or are differentially bound throughout the cell cycle. Since DPS2 was required for Sir2 and condensin recruitment, and derepression of HMLALPHA2 from HML also eliminated binding, we hypothesized and then demonstrated (S3 Fig) that Mcm1 was a key DNA binding factor involved. Mcm1 is a prototypical MADS box combinatorial transcription factor that derives its regulatory specificity through interactions with other factors, such as Ste12 in the case of MATa haploid-specific gene activation, or α2 when repressing the same target genes in MATα cells [46]. This raises the question of whether Mcm1 directly recruits the SIR and condensin complexes, or perhaps additional factors that work with Mcm1 are involved. The latter is likely true because condensin and Sir2 are not recruited to the leftmost Mcm1/α2 binding site in the left half of the RE, as indicated by the ChIP-seq data in Fig 1A. At the RDT1 promoter, specificity for Sir2/condensin recruitment could originate from sequences underlying the condensin/Sir2 peaks. There are no traditional silencer-like sequences for SIR recruitment within the deleted 100bp (coordinates 30702 to 30801), and yeast condensin does not appear to have a consensus DNA binding sequence [47]. Closer inspection of the RDT1 promoter indicates an A/T rich region with consensus binding sites for the transcription factors Fkh1/2 and Ash1, each of which regulates mating-type switching [11, 48, 49]. Fkh1 and Fkh2 also physically associate with Sir2 at the mitotic cyclin CLB2 promoter during stress [50]. Ash1 is intriguing because it represses HO transcription in daughter cells [49, 51], raising the possibility of RDT1 repression in daughter cells. Mcm1 activity in MATa cells could also indirectly establish a chromatin environment that is competent for Sir2/condensin recruitment, rather than direct recruitment through protein-protein interactions. In MATa cells, Mcm1 appears to prevent the strong nucleosome positioning across the RE that occurs in MATα cells [27], and indicative of an actively remodeled chromatin environment. Perhaps condensin is attracted to such regions, which is consistent with the association of condensin with promoters of active genes in mitotic cells, where enrichment was greatest at unwound regions of DNA [52]. Furthermore, nucleosome eviction by transcriptional coactivators was found to assist condensin loading in yeast [53], though the mechanism of loading remains poorly understood. Recruitment of condensin to the RDT1 promoter LCR therefore provides an outstanding opportunity for dissecting mechanisms of condensin loading and function.

MATa specific recruitment of Sir2 and condensin to the RE

Condensin, and Sir2 each strongly associated with the RDT1 promoter exclusively in MATa cells, though it is not clear if they bind at the same time, or are differentially bound throughout the cell cycle. Since DPS2 was required for Sir2 and condensin recruitment, and derepression of HMLALPHA2 from HML also eliminated binding, we hypothesized and then demonstrated (S3 Fig) that Mcm1 was a key DNA binding factor involved. Mcm1 is a prototypical MADS box combinatorial transcription factor that derives its regulatory specificity through interactions with other factors, such as Ste12 in the case of MATa haploid-specific gene activation, or α2 when repressing the same target genes in MATα cells [46]. This raises the question of whether Mcm1 directly recruits the SIR and condensin complexes, or perhaps additional factors that work with Mcm1 are involved. The latter is likely true because condensin and Sir2 are not recruited to the leftmost Mcm1/α2 binding site in the left half of the RE, as indicated by the ChIP-seq data in Fig 1A. At the RDT1 promoter, specificity for Sir2/condensin recruitment could originate from sequences underlying the condensin/Sir2 peaks. There are no traditional silencer-like sequences for SIR recruitment within the deleted 100bp (coordinates 30702 to 30801), and yeast condensin does not appear to have a consensus DNA binding sequence [47]. Closer inspection of the RDT1 promoter indicates an A/T rich region with consensus binding sites for the transcription factors Fkh1/2 and Ash1, each of which regulates mating-type switching [11, 48, 49]. Fkh1 and Fkh2 also physically associate with Sir2 at the mitotic cyclin CLB2 promoter during stress [50]. Ash1 is intriguing because it represses HO transcription in daughter cells [49, 51], raising the possibility of RDT1 repression in daughter cells. Mcm1 activity in MATa cells could also indirectly establish a chromatin environment that is competent for Sir2/condensin recruitment, rather than direct recruitment through protein-protein interactions. In MATa cells, Mcm1 appears to prevent the strong nucleosome positioning across the RE that occurs in MATα cells [27], and indicative of an actively remodeled chromatin environment. Perhaps condensin is attracted to such regions, which is consistent with the association of condensin with promoters of active genes in mitotic cells, where enrichment was greatest at unwound regions of DNA [52]. Furthermore, nucleosome eviction by transcriptional coactivators was found to assist condensin loading in yeast [53], though the mechanism of loading remains poorly understood. Recruitment of condensin to the RDT1 promoter LCR therefore provides an outstanding opportunity for dissecting mechanisms of condensin loading and function.

MethodsYeast strains, plasmids, and mediaYeast strains were grown at 30°C in YPD or synthetic complete (SC) medium where indicated. The SIR2, or HST1 open reading frames (ORFs) were deleted with kanMX4 using one-step PCR-mediated gene replacement. HML was deleted and replaced with LEU2. Precise deletions of the 100bp condensin/Sir2 binding site within the RDT1 promoter (chrIII coordinates 30701–30800), DPS2 (chrIII coordinates 30557–30626), or the RDT1 ORF (chrIII coordinates 30910–30996) were generated using the delitto perfetto method [32]. Endogenous SIR2, BRN1, or SMC4 genes were C-terminally tagged with the 13xMyc epitope (13-EQKLISEEDL). Deletion and tagged gene combinations were generated through genetic crosses and tetrad dissection, including Brn1 tagged with a V5-AID tag (template plasmids kindly provided by Vincent Guacci). All genetic manipulations were confirmed by PCR, and expression of tagged proteins confirmed by western blotting. The pGAL-HO-URA3 expression plasmid was kindly provided by Jessica Tyler [30]. Strain genotypes are provided in S2 Table and oligonucleotides listed in S3 Table.ChIP-seq analysisSir2 ChIP-seq was previously described [19]. For other ChIP-seq datasets, log-phase YPD cultures were cross-linked with 1% formaldehyde for 20 min, pelleted, washed with Tris-buffered saline (TBS), and then lysed in 600 μl FA140 lysis buffer with glass beads using a mini-beadbeater (BioSpec Products). Lysates were removed from the beads and sonicated for 60 cycles (30s “on” and 30s “off” per cycle) in a Diagenode Bioruptor. Sonicated lysates were pelleted for 5 min at 14000 rpm in a microcentrifuge and the entire supernatant was transferred to a new microfuge tube and incubated overnight at 4°C with 5 μg of anti-Myc antibody (9E10) and 20 μl of protein G magnetic beads (Millipore). Following IP, the beads were washed once with FA140 buffer, twice with FA500 buffer, and twice with LiCl wash buffer. DNA was eluted from the beads in 1% SDS/TE buffer and cross-links were reversed overnight at 65°C. The chromatin was then purified using a Qiagen PCR purification kit. Libraries were constructed using the Illumina Trueseq ChIP Sample Prep kit and TrueSeq standard protocol with 10ng of initial ChIP or Input DNA. Libraries that passed QC on a Bioanalyzer High Sensitivity DNA Chip (Agilent Technologies) were sequenced on an Illumina Miseq (UVA Genomic Analysis and Technology Core).ChIP-seq computational analysisBiological duplicate fastq files were concatenated together and reads mapped to the sacCer3 genome using Bowtie with the following options:—best,—stratum,—nomaqround, and—m10 [54]. The resulting bam files were then converted into bigwig files using BEDTools [55]. As part of the pipeline, chromosome names were changed from the sacCer3 NCBI values to values readable by genomics viewers e.g. "ref|NC_001133|" to "chrI". The raw and processed datasets used in this study have been deposited in NCBI’s GEO and are accessible through the GEO series accession number GSE92717. Downstream GO analysis was performed as follows. MACS2 was used to call peaks with the following options:—broad,—keep-dup, -tz 150, and -m 3, 1000 [56]. GFP peaks in the WT or sir2Δ backgrounds were subtracted from the WT SMC4-13xMyc and sir2Δ SMC4-13xMyc peaks, respectively, using BEDTools “intersect” with the–v option. The resulting normalized peaks were annotated using BEDTools “closest” with the -t all option specified, and in combination with a yeast gene list produced from USCS genome tables. The annotated peaks were then analyzed for GO terms using YeastMine (yeastmine.yeastgenome.org).Hi-C analysisLog-phase cultures were cross-linked with 3% formaldehyde for 20 min and quenched with a 2x volume of 2.5M Glycine. Cell pellets were washed with dH2O and stored at -80°C. Thawed cells were resuspended in 5 ml of 1X NEB2 restriction enzyme buffer (New England Biolabs) and poured into a pre-chilled mortar containing liquid N2. Nitrogen grinding was performed twice as previously described [57], and the lysates were then diluted to an OD600 of 12 in 1x NEB2 buffer. 500 μl of cell lysate was used for each Hi-C library as follows. Lysates were solubilized by the addition of 50 μl 1% SDS and incubation at 65°C for 10 min. 55 μl of 10% TritonX-100 was added to quench the SDS, followed by 10 μl of 10X NEB2 buffer and 15 μl of HindIII (New England Biolabs, 20 U/μl) to digest at 37°C for 2 hr. An additional 10 μl of HindIII was added for digestion overnight. The remainder of the protocol was based on previously published work with minor exceptions [58]. In short, digested chromatin ends were filled-in with Klenow fragment (New England Biolabs) and biotinylated dCTP at 37°C for 1 hr, then heat inactivated at 70°C for 10 min. Ligation reactions with T4 DNA ligase were performed at 16°C for a minimum of 6 hr using the entire Hi-C sample diluted into a total volume of 4 ml. Proteinase K was added and cross-links were reversed overnight at 70°C. The ligated chromatin was phenol:chloroform extracted, ethanol precipitated, then resuspended in 500μl dH2O and treated with RNAse A for 45 min. Following treatment with T4 DNA polymerase to remove biotinylated DNA ends that were unligated, the samples were concentrated with a Clean and Concentrator spin column (Zymogen, D4013) and sheared to approximately 300bp with a Diagenode Bioruptor. Biotinylated fragments were captured with 30 μl pre-washed Streptavidin Dynabeads (Invitrogen), then used for library preparation. Hi-C sequencing libraries were prepared with reagents from an Illumina Nextera Mate Pair Kit (FC-132-1001) using the standard Illumina protocol of End Repair, A-tailing, Adapter Ligation, and 12 cycles of PCR. PCR products were size selected and purified with AMPure XP beads before sequencing with an Illumina Miseq (UVA Genomic Analysis and Technology Core) or Hiseq (HudsonAlpha Institute for Biotechnology, Birmingham, AL). Raw and mapped reads are deposited at GEO (GSE92717).Hi-C computational analysisIteratively corrected heatmaps were produced using python scripts from the Mirny lab hiclib library, http://mirnylab.bitbucket.org/hiclib/index.html. Briefly, reads were mapped using the iterative mapping program, which uses Bowtie2 to map reads and iteratively trims unmapped reads to increase the total number of mapped reads. Mapped reads were then parsed into an hdf5 python data dictionary for storage and further analysis. Mapped reads of the same strains were concatenated using the hiclib library’s “Merge" function. Both individual and concatenated mapped reads have been deposited in GEO. Mapped reads were then run through the fragment filtering program using the default parameters as follows: filterRsiteStart(offset = 5), filterDuplicates, filterLarge, filterExtreme (cutH = 0.005, cutL = 0). Raw heat maps were further filtered to remove diagonal reads and iteratively corrected using the 03 heat map processing program. Finally, the iteratively corrected heatmaps were normalized for read count differences by dividing the sum of each row by the sum of the max row for a given plot, which drives all values towards 1 to make individual heatmaps comparable.Observed/Expected heatmaps were created using HOMER Hi-C analysis software on the BAM file outputs from the iterativemapping program of the hiclib library python package [35]. Tag directories were created using all experimental replicates of a given biological sample and the tbp -1 and illuminaPE options. Homer was also used to score differential chromosome interactions between the WT and mutant Hi-C heatmaps. The resulting list of differential interactions was uploaded into R where the given p-value was adjusted to a qvalue with p.adjust. An FDR cutoff of 0.05 was used to create a histogram of significantly different chromosome interactions in the mutants compared to WT. The histogram was further normalized by dividing the total number of significant differential interactions for a chromosome by total number of interactions called in the WT sample for that chromosome to account for size differences in the chromosomes. Thus, frequency represents the number of interactions that changed out of all possible interactions that could have changed.RNA-seq data analysisRNA-Seq data was acquired from GEO accessions GSE73274 [59] and GSE58319 [60] for the BY4742 (MATα) and BY4741 (MATa) backgrounds, respectively. Reads were then mapped to the sacCer3 genome using Bowtie2 with no further processing of the resulting BAM files visualized in this paper.3C assaysChromosome Conformation Capture (3C) was performed in a similar manner to Hi-C with a few exceptions due to assay-specific quantification via quantitative real-time PCR rather than sequencing. Most notably, digested DNA ends were not filled in with dCTP-biotin before ligation and an un-crosslinked control library was created for each 3C library. Furthermore, all PCR reactions were normalized for starting DNA concentration using a PDC1 intergenic region that is not recognized by HindIII, in addition to PCR of the un-crosslinked control for all tested looping interaction primer pairs.Quantitative reverse transcriptase (RT) PCR assayTotal RNA (1 μg) was used for cDNA synthesis with oligo(dT) and Superscript II reverse transcriptase as previously described [61]. Expression levels are indicated in figures relative to the level of ACT1 mRNA, with this ratio then normalized to 1.0 for a specific strain or condition indicated for each experiment.Western blotsProteins were blotted using standard TCA extraction followed by SDS-PAGE as previously described [19]. Myc-tagged proteins were incubated with an anti-Myc primary antibody 9E10 (Millipore) at a 1:2000 dilution while tubulin was incubated with anti-Tubulin antibody B-5-1-2 (Sigma-Aldrich) at a 1:1500 dilution. The V5-AID tagged Brn1 was detected with anti-V5 antibody (Invitrogen, R96025) at a 1:4000 dilution. Primary antibodies were detected with an anti-mouse secondary antibody conjugated to HRP (Promega) at 1:5000 dilution in 5% fat-free milk. Bands were then visualized with HyGlo (Denville Scientific) capture on autoradiography film (Denville Scientific).Mating-type switching assaysFor tracking the efficiency of switching, strains were transformed with pGAL-HO-URA3, pre-cultured in SC-ura + raffinose (2%) medium overnight, re-inoculated into the same medium (OD600 = 0.05) and then grown into log phase. Galactose (2%) was added to induce HO expression for 45 min. Glucose (2%) was then added and aliquots of the cultures were harvested at indicated time points. Genomic DNA was isolated and 10 ng used for PCR amplification. MATα was detected using primers JS301 and JS302. The SCR1 gene on chromosome V was used as a loading control (primers JS2665 and JS2666). PCR products were separated on a 1% agarose gel stained with ethidium bromide and then quantified using ImageJ. Donor preference with strains containing HMRα-B was performed as previously described [15]. Briefly, MATa was amplified with primers Yalpha105F and MATdist-4R from genomic DNA 90 after switching was completed (90 min), and then digested with BamHI. Ethidium stained bands were quantified using ImageJ. For the conditional V5-AID degron strains, degradation of V5-AID-fused Brn1 protein was induced by addition of 0.5 mM indole-3-acetic acid (Auxin, Sigma # 13750).For assaying HO cleavage across MATa, HML, and HMR, the WT and sir2Δ strains containing the pGAL-HO-URA3 plasmid were induced with galactose for 0 to 2 hrs following growth in raffinose. Genomic DNA was then isolated and PCR across the HO-cleavage site performed with primers specific to each locus, and SCR1 used as a loading control. MATa was detected with JS301 and JS854, HML with JS3101 and JS3103, and HMR with JS3097 and JS3100. PCR was performed in the linear range and bands on ethidium stained agarose gels quantified with ImageJ. Real-time qPCR of HML and HMR with the same genomic DNA was performed with primers JS3101-JS3103, and JS3097-JS3098, respectively.

Yeast strains, plasmids, and mediaYeast strains were grown at 30°C in YPD or synthetic complete (SC) medium where indicated. The SIR2, or HST1 open reading frames (ORFs) were deleted with kanMX4 using one-step PCR-mediated gene replacement. HML was deleted and replaced with LEU2. Precise deletions of the 100bp condensin/Sir2 binding site within the RDT1 promoter (chrIII coordinates 30701–30800), DPS2 (chrIII coordinates 30557–30626), or the RDT1 ORF (chrIII coordinates 30910–30996) were generated using the delitto perfetto method [32]. Endogenous SIR2, BRN1, or SMC4 genes were C-terminally tagged with the 13xMyc epitope (13-EQKLISEEDL). Deletion and tagged gene combinations were generated through genetic crosses and tetrad dissection, including Brn1 tagged with a V5-AID tag (template plasmids kindly provided by Vincent Guacci). All genetic manipulations were confirmed by PCR, and expression of tagged proteins confirmed by western blotting. The pGAL-HO-URA3 expression plasmid was kindly provided by Jessica Tyler [30]. Strain genotypes are provided in S2 Table and oligonucleotides listed in S3 Table.

Yeast strains, plasmids, and media

Yeast strains were grown at 30°C in YPD or synthetic complete (SC) medium where indicated. The SIR2, or HST1 open reading frames (ORFs) were deleted with kanMX4 using one-step PCR-mediated gene replacement. HML was deleted and replaced with LEU2. Precise deletions of the 100bp condensin/Sir2 binding site within the RDT1 promoter (chrIII coordinates 30701–30800), DPS2 (chrIII coordinates 30557–30626), or the RDT1 ORF (chrIII coordinates 30910–30996) were generated using the delitto perfetto method [32]. Endogenous SIR2, BRN1, or SMC4 genes were C-terminally tagged with the 13xMyc epitope (13-EQKLISEEDL). Deletion and tagged gene combinations were generated through genetic crosses and tetrad dissection, including Brn1 tagged with a V5-AID tag (template plasmids kindly provided by Vincent Guacci). All genetic manipulations were confirmed by PCR, and expression of tagged proteins confirmed by western blotting. The pGAL-HO-URA3 expression plasmid was kindly provided by Jessica Tyler [30]. Strain genotypes are provided in S2 Table and oligonucleotides listed in S3 Table.

ChIP-seq analysisSir2 ChIP-seq was previously described [19]. For other ChIP-seq datasets, log-phase YPD cultures were cross-linked with 1% formaldehyde for 20 min, pelleted, washed with Tris-buffered saline (TBS), and then lysed in 600 μl FA140 lysis buffer with glass beads using a mini-beadbeater (BioSpec Products). Lysates were removed from the beads and sonicated for 60 cycles (30s “on” and 30s “off” per cycle) in a Diagenode Bioruptor. Sonicated lysates were pelleted for 5 min at 14000 rpm in a microcentrifuge and the entire supernatant was transferred to a new microfuge tube and incubated overnight at 4°C with 5 μg of anti-Myc antibody (9E10) and 20 μl of protein G magnetic beads (Millipore). Following IP, the beads were washed once with FA140 buffer, twice with FA500 buffer, and twice with LiCl wash buffer. DNA was eluted from the beads in 1% SDS/TE buffer and cross-links were reversed overnight at 65°C. The chromatin was then purified using a Qiagen PCR purification kit. Libraries were constructed using the Illumina Trueseq ChIP Sample Prep kit and TrueSeq standard protocol with 10ng of initial ChIP or Input DNA. Libraries that passed QC on a Bioanalyzer High Sensitivity DNA Chip (Agilent Technologies) were sequenced on an Illumina Miseq (UVA Genomic Analysis and Technology Core).

ChIP-seq analysis

Sir2 ChIP-seq was previously described [19]. For other ChIP-seq datasets, log-phase YPD cultures were cross-linked with 1% formaldehyde for 20 min, pelleted, washed with Tris-buffered saline (TBS), and then lysed in 600 μl FA140 lysis buffer with glass beads using a mini-beadbeater (BioSpec Products). Lysates were removed from the beads and sonicated for 60 cycles (30s “on” and 30s “off” per cycle) in a Diagenode Bioruptor. Sonicated lysates were pelleted for 5 min at 14000 rpm in a microcentrifuge and the entire supernatant was transferred to a new microfuge tube and incubated overnight at 4°C with 5 μg of anti-Myc antibody (9E10) and 20 μl of protein G magnetic beads (Millipore). Following IP, the beads were washed once with FA140 buffer, twice with FA500 buffer, and twice with LiCl wash buffer. DNA was eluted from the beads in 1% SDS/TE buffer and cross-links were reversed overnight at 65°C. The chromatin was then purified using a Qiagen PCR purification kit. Libraries were constructed using the Illumina Trueseq ChIP Sample Prep kit and TrueSeq standard protocol with 10ng of initial ChIP or Input DNA. Libraries that passed QC on a Bioanalyzer High Sensitivity DNA Chip (Agilent Technologies) were sequenced on an Illumina Miseq (UVA Genomic Analysis and Technology Core).

ChIP-seq computational analysisBiological duplicate fastq files were concatenated together and reads mapped to the sacCer3 genome using Bowtie with the following options:—best,—stratum,—nomaqround, and—m10 [54]. The resulting bam files were then converted into bigwig files using BEDTools [55]. As part of the pipeline, chromosome names were changed from the sacCer3 NCBI values to values readable by genomics viewers e.g. "ref|NC_001133|" to "chrI". The raw and processed datasets used in this study have been deposited in NCBI’s GEO and are accessible through the GEO series accession number GSE92717. Downstream GO analysis was performed as follows. MACS2 was used to call peaks with the following options:—broad,—keep-dup, -tz 150, and -m 3, 1000 [56]. GFP peaks in the WT or sir2Δ backgrounds were subtracted from the WT SMC4-13xMyc and sir2Δ SMC4-13xMyc peaks, respectively, using BEDTools “intersect” with the–v option. The resulting normalized peaks were annotated using BEDTools “closest” with the -t all option specified, and in combination with a yeast gene list produced from USCS genome tables. The annotated peaks were then analyzed for GO terms using YeastMine (yeastmine.yeastgenome.org).

ChIP-seq computational analysis

Biological duplicate fastq files were concatenated together and reads mapped to the sacCer3 genome using Bowtie with the following options:—best,—stratum,—nomaqround, and—m10 [54]. The resulting bam files were then converted into bigwig files using BEDTools [55]. As part of the pipeline, chromosome names were changed from the sacCer3 NCBI values to values readable by genomics viewers e.g. "ref|NC_001133|" to "chrI". The raw and processed datasets used in this study have been deposited in NCBI’s GEO and are accessible through the GEO series accession number GSE92717. Downstream GO analysis was performed as follows. MACS2 was used to call peaks with the following options:—broad,—keep-dup, -tz 150, and -m 3, 1000 [56]. GFP peaks in the WT or sir2Δ backgrounds were subtracted from the WT SMC4-13xMyc and sir2Δ SMC4-13xMyc peaks, respectively, using BEDTools “intersect” with the–v option. The resulting normalized peaks were annotated using BEDTools “closest” with the -t all option specified, and in combination with a yeast gene list produced from USCS genome tables. The annotated peaks were then analyzed for GO terms using YeastMine (yeastmine.yeastgenome.org).

Hi-C analysisLog-phase cultures were cross-linked with 3% formaldehyde for 20 min and quenched with a 2x volume of 2.5M Glycine. Cell pellets were washed with dH2O and stored at -80°C. Thawed cells were resuspended in 5 ml of 1X NEB2 restriction enzyme buffer (New England Biolabs) and poured into a pre-chilled mortar containing liquid N2. Nitrogen grinding was performed twice as previously described [57], and the lysates were then diluted to an OD600 of 12 in 1x NEB2 buffer. 500 μl of cell lysate was used for each Hi-C library as follows. Lysates were solubilized by the addition of 50 μl 1% SDS and incubation at 65°C for 10 min. 55 μl of 10% TritonX-100 was added to quench the SDS, followed by 10 μl of 10X NEB2 buffer and 15 μl of HindIII (New England Biolabs, 20 U/μl) to digest at 37°C for 2 hr. An additional 10 μl of HindIII was added for digestion overnight. The remainder of the protocol was based on previously published work with minor exceptions [58]. In short, digested chromatin ends were filled-in with Klenow fragment (New England Biolabs) and biotinylated dCTP at 37°C for 1 hr, then heat inactivated at 70°C for 10 min. Ligation reactions with T4 DNA ligase were performed at 16°C for a minimum of 6 hr using the entire Hi-C sample diluted into a total volume of 4 ml. Proteinase K was added and cross-links were reversed overnight at 70°C. The ligated chromatin was phenol:chloroform extracted, ethanol precipitated, then resuspended in 500μl dH2O and treated with RNAse A for 45 min. Following treatment with T4 DNA polymerase to remove biotinylated DNA ends that were unligated, the samples were concentrated with a Clean and Concentrator spin column (Zymogen, D4013) and sheared to approximately 300bp with a Diagenode Bioruptor. Biotinylated fragments were captured with 30 μl pre-washed Streptavidin Dynabeads (Invitrogen), then used for library preparation. Hi-C sequencing libraries were prepared with reagents from an Illumina Nextera Mate Pair Kit (FC-132-1001) using the standard Illumina protocol of End Repair, A-tailing, Adapter Ligation, and 12 cycles of PCR. PCR products were size selected and purified with AMPure XP beads before sequencing with an Illumina Miseq (UVA Genomic Analysis and Technology Core) or Hiseq (HudsonAlpha Institute for Biotechnology, Birmingham, AL). Raw and mapped reads are deposited at GEO (GSE92717).

Hi-C analysis

Log-phase cultures were cross-linked with 3% formaldehyde for 20 min and quenched with a 2x volume of 2.5M Glycine. Cell pellets were washed with dH2O and stored at -80°C. Thawed cells were resuspended in 5 ml of 1X NEB2 restriction enzyme buffer (New England Biolabs) and poured into a pre-chilled mortar containing liquid N2. Nitrogen grinding was performed twice as previously described [57], and the lysates were then diluted to an OD600 of 12 in 1x NEB2 buffer. 500 μl of cell lysate was used for each Hi-C library as follows. Lysates were solubilized by the addition of 50 μl 1% SDS and incubation at 65°C for 10 min. 55 μl of 10% TritonX-100 was added to quench the SDS, followed by 10 μl of 10X NEB2 buffer and 15 μl of HindIII (New England Biolabs, 20 U/μl) to digest at 37°C for 2 hr. An additional 10 μl of HindIII was added for digestion overnight. The remainder of the protocol was based on previously published work with minor exceptions [58]. In short, digested chromatin ends were filled-in with Klenow fragment (New England Biolabs) and biotinylated dCTP at 37°C for 1 hr, then heat inactivated at 70°C for 10 min. Ligation reactions with T4 DNA ligase were performed at 16°C for a minimum of 6 hr using the entire Hi-C sample diluted into a total volume of 4 ml. Proteinase K was added and cross-links were reversed overnight at 70°C. The ligated chromatin was phenol:chloroform extracted, ethanol precipitated, then resuspended in 500μl dH2O and treated with RNAse A for 45 min. Following treatment with T4 DNA polymerase to remove biotinylated DNA ends that were unligated, the samples were concentrated with a Clean and Concentrator spin column (Zymogen, D4013) and sheared to approximately 300bp with a Diagenode Bioruptor. Biotinylated fragments were captured with 30 μl pre-washed Streptavidin Dynabeads (Invitrogen), then used for library preparation. Hi-C sequencing libraries were prepared with reagents from an Illumina Nextera Mate Pair Kit (FC-132-1001) using the standard Illumina protocol of End Repair, A-tailing, Adapter Ligation, and 12 cycles of PCR. PCR products were size selected and purified with AMPure XP beads before sequencing with an Illumina Miseq (UVA Genomic Analysis and Technology Core) or Hiseq (HudsonAlpha Institute for Biotechnology, Birmingham, AL). Raw and mapped reads are deposited at GEO (GSE92717).

Hi-C computational analysisIteratively corrected heatmaps were produced using python scripts from the Mirny lab hiclib library, http://mirnylab.bitbucket.org/hiclib/index.html. Briefly, reads were mapped using the iterative mapping program, which uses Bowtie2 to map reads and iteratively trims unmapped reads to increase the total number of mapped reads. Mapped reads were then parsed into an hdf5 python data dictionary for storage and further analysis. Mapped reads of the same strains were concatenated using the hiclib library’s “Merge" function. Both individual and concatenated mapped reads have been deposited in GEO. Mapped reads were then run through the fragment filtering program using the default parameters as follows: filterRsiteStart(offset = 5), filterDuplicates, filterLarge, filterExtreme (cutH = 0.005, cutL = 0). Raw heat maps were further filtered to remove diagonal reads and iteratively corrected using the 03 heat map processing program. Finally, the iteratively corrected heatmaps were normalized for read count differences by dividing the sum of each row by the sum of the max row for a given plot, which drives all values towards 1 to make individual heatmaps comparable.Observed/Expected heatmaps were created using HOMER Hi-C analysis software on the BAM file outputs from the iterativemapping program of the hiclib library python package [35]. Tag directories were created using all experimental replicates of a given biological sample and the tbp -1 and illuminaPE options. Homer was also used to score differential chromosome interactions between the WT and mutant Hi-C heatmaps. The resulting list of differential interactions was uploaded into R where the given p-value was adjusted to a qvalue with p.adjust. An FDR cutoff of 0.05 was used to create a histogram of significantly different chromosome interactions in the mutants compared to WT. The histogram was further normalized by dividing the total number of significant differential interactions for a chromosome by total number of interactions called in the WT sample for that chromosome to account for size differences in the chromosomes. Thus, frequency represents the number of interactions that changed out of all possible interactions that could have changed.

Hi-C computational analysis

Iteratively corrected heatmaps were produced using python scripts from the Mirny lab hiclib library, http://mirnylab.bitbucket.org/hiclib/index.html. Briefly, reads were mapped using the iterative mapping program, which uses Bowtie2 to map reads and iteratively trims unmapped reads to increase the total number of mapped reads. Mapped reads were then parsed into an hdf5 python data dictionary for storage and further analysis. Mapped reads of the same strains were concatenated using the hiclib library’s “Merge" function. Both individual and concatenated mapped reads have been deposited in GEO. Mapped reads were then run through the fragment filtering program using the default parameters as follows: filterRsiteStart(offset = 5), filterDuplicates, filterLarge, filterExtreme (cutH = 0.005, cutL = 0). Raw heat maps were further filtered to remove diagonal reads and iteratively corrected using the 03 heat map processing program. Finally, the iteratively corrected heatmaps were normalized for read count differences by dividing the sum of each row by the sum of the max row for a given plot, which drives all values towards 1 to make individual heatmaps comparable.

Observed/Expected heatmaps were created using HOMER Hi-C analysis software on the BAM file outputs from the iterativemapping program of the hiclib library python package [35]. Tag directories were created using all experimental replicates of a given biological sample and the tbp -1 and illuminaPE options. Homer was also used to score differential chromosome interactions between the WT and mutant Hi-C heatmaps. The resulting list of differential interactions was uploaded into R where the given p-value was adjusted to a qvalue with p.adjust. An FDR cutoff of 0.05 was used to create a histogram of significantly different chromosome interactions in the mutants compared to WT. The histogram was further normalized by dividing the total number of significant differential interactions for a chromosome by total number of interactions called in the WT sample for that chromosome to account for size differences in the chromosomes. Thus, frequency represents the number of interactions that changed out of all possible interactions that could have changed.

RNA-seq data analysisRNA-Seq data was acquired from GEO accessions GSE73274 [59] and GSE58319 [60] for the BY4742 (MATα) and BY4741 (MATa) backgrounds, respectively. Reads were then mapped to the sacCer3 genome using Bowtie2 with no further processing of the resulting BAM files visualized in this paper.

RNA-seq data analysis

RNA-Seq data was acquired from GEO accessions GSE73274 [59] and GSE58319 [60] for the BY4742 (MATα) and BY4741 (MATa) backgrounds, respectively. Reads were then mapped to the sacCer3 genome using Bowtie2 with no further processing of the resulting BAM files visualized in this paper.

3C assaysChromosome Conformation Capture (3C) was performed in a similar manner to Hi-C with a few exceptions due to assay-specific quantification via quantitative real-time PCR rather than sequencing. Most notably, digested DNA ends were not filled in with dCTP-biotin before ligation and an un-crosslinked control library was created for each 3C library. Furthermore, all PCR reactions were normalized for starting DNA concentration using a PDC1 intergenic region that is not recognized by HindIII, in addition to PCR of the un-crosslinked control for all tested looping interaction primer pairs.

Chromosome Conformation Capture (3C) was performed in a similar manner to Hi-C with a few exceptions due to assay-specific quantification via quantitative real-time PCR rather than sequencing. Most notably, digested DNA ends were not filled in with dCTP-biotin before ligation and an un-crosslinked control library was created for each 3C library. Furthermore, all PCR reactions were normalized for starting DNA concentration using a PDC1 intergenic region that is not recognized by HindIII, in addition to PCR of the un-crosslinked control for all tested looping interaction primer pairs.

Quantitative reverse transcriptase (RT) PCR assayTotal RNA (1 μg) was used for cDNA synthesis with oligo(dT) and Superscript II reverse transcriptase as previously described [61]. Expression levels are indicated in figures relative to the level of ACT1 mRNA, with this ratio then normalized to 1.0 for a specific strain or condition indicated for each experiment.

Quantitative reverse transcriptase (RT) PCR assay

Total RNA (1 μg) was used for cDNA synthesis with oligo(dT) and Superscript II reverse transcriptase as previously described [61]. Expression levels are indicated in figures relative to the level of ACT1 mRNA, with this ratio then normalized to 1.0 for a specific strain or condition indicated for each experiment.

Western blotsProteins were blotted using standard TCA extraction followed by SDS-PAGE as previously described [19]. Myc-tagged proteins were incubated with an anti-Myc primary antibody 9E10 (Millipore) at a 1:2000 dilution while tubulin was incubated with anti-Tubulin antibody B-5-1-2 (Sigma-Aldrich) at a 1:1500 dilution. The V5-AID tagged Brn1 was detected with anti-V5 antibody (Invitrogen, R96025) at a 1:4000 dilution. Primary antibodies were detected with an anti-mouse secondary antibody conjugated to HRP (Promega) at 1:5000 dilution in 5% fat-free milk. Bands were then visualized with HyGlo (Denville Scientific) capture on autoradiography film (Denville Scientific).

Western blots

Proteins were blotted using standard TCA extraction followed by SDS-PAGE as previously described [19]. Myc-tagged proteins were incubated with an anti-Myc primary antibody 9E10 (Millipore) at a 1:2000 dilution while tubulin was incubated with anti-Tubulin antibody B-5-1-2 (Sigma-Aldrich) at a 1:1500 dilution. The V5-AID tagged Brn1 was detected with anti-V5 antibody (Invitrogen, R96025) at a 1:4000 dilution. Primary antibodies were detected with an anti-mouse secondary antibody conjugated to HRP (Promega) at 1:5000 dilution in 5% fat-free milk. Bands were then visualized with HyGlo (Denville Scientific) capture on autoradiography film (Denville Scientific).

Mating-type switching assaysFor tracking the efficiency of switching, strains were transformed with pGAL-HO-URA3, pre-cultured in SC-ura + raffinose (2%) medium overnight, re-inoculated into the same medium (OD600 = 0.05) and then grown into log phase. Galactose (2%) was added to induce HO expression for 45 min. Glucose (2%) was then added and aliquots of the cultures were harvested at indicated time points. Genomic DNA was isolated and 10 ng used for PCR amplification. MATα was detected using primers JS301 and JS302. The SCR1 gene on chromosome V was used as a loading control (primers JS2665 and JS2666). PCR products were separated on a 1% agarose gel stained with ethidium bromide and then quantified using ImageJ. Donor preference with strains containing HMRα-B was performed as previously described [15]. Briefly, MATa was amplified with primers Yalpha105F and MATdist-4R from genomic DNA 90 after switching was completed (90 min), and then digested with BamHI. Ethidium stained bands were quantified using ImageJ. For the conditional V5-AID degron strains, degradation of V5-AID-fused Brn1 protein was induced by addition of 0.5 mM indole-3-acetic acid (Auxin, Sigma # 13750).For assaying HO cleavage across MATa, HML, and HMR, the WT and sir2Δ strains containing the pGAL-HO-URA3 plasmid were induced with galactose for 0 to 2 hrs following growth in raffinose. Genomic DNA was then isolated and PCR across the HO-cleavage site performed with primers specific to each locus, and SCR1 used as a loading control. MATa was detected with JS301 and JS854, HML with JS3101 and JS3103, and HMR with JS3097 and JS3100. PCR was performed in the linear range and bands on ethidium stained agarose gels quantified with ImageJ. Real-time qPCR of HML and HMR with the same genomic DNA was performed with primers JS3101-JS3103, and JS3097-JS3098, respectively.

Mating-type switching assays

For tracking the efficiency of switching, strains were transformed with pGAL-HO-URA3, pre-cultured in SC-ura + raffinose (2%) medium overnight, re-inoculated into the same medium (OD600 = 0.05) and then grown into log phase. Galactose (2%) was added to induce HO expression for 45 min. Glucose (2%) was then added and aliquots of the cultures were harvested at indicated time points. Genomic DNA was isolated and 10 ng used for PCR amplification. MATα was detected using primers JS301 and JS302. The SCR1 gene on chromosome V was used as a loading control (primers JS2665 and JS2666). PCR products were separated on a 1% agarose gel stained with ethidium bromide and then quantified using ImageJ. Donor preference with strains containing HMRα-B was performed as previously described [15]. Briefly, MATa was amplified with primers Yalpha105F and MATdist-4R from genomic DNA 90 after switching was completed (90 min), and then digested with BamHI. Ethidium stained bands were quantified using ImageJ. For the conditional V5-AID degron strains, degradation of V5-AID-fused Brn1 protein was induced by addition of 0.5 mM indole-3-acetic acid (Auxin, Sigma # 13750).

For assaying HO cleavage across MATa, HML, and HMR, the WT and sir2Δ strains containing the pGAL-HO-URA3 plasmid were induced with galactose for 0 to 2 hrs following growth in raffinose. Genomic DNA was then isolated and PCR across the HO-cleavage site performed with primers specific to each locus, and SCR1 used as a loading control. MATa was detected with JS301 and JS854, HML with JS3101 and JS3103, and HMR with JS3097 and JS3100. PCR was performed in the linear range and bands on ethidium stained agarose gels quantified with ImageJ. Real-time qPCR of HML and HMR with the same genomic DNA was performed with primers JS3101-JS3103, and JS3097-JS3098, respectively.

Supporting informationS1 FigMATa-specific transcription of RDT1 is repressed by Sir2 and Hst1.(A) IGV screenshot of compiled raw RNA-seq read data from BY4741 (MATa) and BY4742 (MATα) strains. The top two blue peaks represent Smc4-myc and Sir2-myc ChIP-seq reads. (B) Quantitative ChIP assay showing additional SIR complex subunit enrichment at the RDT1 promoter. Signals are relative to input. (C) RT-qPCR showing effects of deleting SIR2 and/or HST1 on RDT1 expression (relative to ACT1) when HML is present or deleted (*p<0.05, **p < .005). The WT RDT1/ACT1 ratio is normalized to 1.(TIF)Click here for additional data file.S2 FigDeletion of Sir2 or the RDT1 promoter Sir2/condensin binding site does not affect protein levels of Sir2 or Myc-tagged condensin subunits.(A) Western blot showing steady state Sir2 protein levels in WT (ML1), sir2Δ (ML25), and 100bpΔ (ML275) strains. (B) Western blot with anti-Myc detection of Brn1-13xMyc or Sir2 in WT (ML149), sir2Δ (ML161), and 100bpΔ (ML322) strains. (C) Western blot with anti-Myc detection of Smc4-13xMyc or Sir2 in WT (ML152), sir2Δ (ML160), and 100bpΔ version.(TIF)Click here for additional data file.S3 FigThe RDT1-proximal Mcm1/α2 binding site (DPS2) is important for Sir2 and condensin recruitment.(A) Schematic diagram depicting the location of the DPS2 sequence deletion relative to other elements with the RE, with the deleted chromosome III coordinates indicated in red. (B) Quantitative ChIP of native Sir2 in WT and dps2Δ strains. (C) Quantitative ChIP of Brn1-Myc in WT and dps2Δ strains. @RDT1 promoter indicates enrichment at the Sir2/condensin peak. ChIP signals are plotted relative to the input signal. (**p<0.005).(TIF)Click here for additional data file.S4 FigDeletion of the RDT1 open reading frame has no effect on mating-type switching.(A) Representative time course of switching from MATa to MATα in WT (ML440) and rdt1Δ (ML443) strains. PCR products are specific to MATα and an SCR1 loading control. (B) Quantitation of the average MATα/SCR1 PCR signal ratio from 3 biological replicates. (C) Schematic of chromosome III in the donor preference reporter strain harboring an artificial HMRα-B cassette as the donor for switching, which introduces a unique BamHI site to the MAT locus. (D) Representative ethidium bromide stained agarose gel of BamHI-digested MATα PCR products after mating-type switching in WT (XW652) and rdt1Δ (MD30) strains. The bottom two bands represent HMR-derived switching. (E) Quantitation of average MATα-B utilization from 3 biological replicates.(TIF)Click here for additional data file.S5 FigDeleting the Sir2/condensin binding site within the RE (100bpΔ) does not alter Sir2 function at HMLα.(A) Quantitative mating assay for WT (ML1) and 100bpΔ (ML275) strains. (B) Quantitative ChIP assay showing Sir2 enrichment at HML-I in WT (ML1) and 100bpΔ (ML275) strains. (**p<0.005).(TIF)Click here for additional data file.S6 FigHML is accessible to cleavage by HO endonuclease in sir2Δ strains.(A) PCR amplification of MATa, HML, HMR, and SCR1 loci from MATa WT (ML440) and sir2Δ (MD29) strains containing the pGAL-HO-URA3 expression vector. Times after HO induction with galactose are indicated. A representative experiment is depicted. (B) Quantitation of mean MATa PCR signal, relative to the SCR1 control, from 3 independent biological replicates. Error bars indicate standard deviation. (C) Quantitation of mean HML PCR signal, as done for panel B. (D) Quantitation of mean HMR PCR signal, as done for panel B. (E and F) Real-time qPCR signal, relative to SCR1 control, for HML and HMR, respectively. The PCR signal at time 0 is normalized to 1.0 in each panel.(TIF)Click here for additional data file.S7 FigAuxin inducible degron (AID)-mediated depletion of Brn1 does not derepress RDT1 or HMLα.(A) Western blot time course of auxin induced degradation of Brn1-V5. Time indicates minutes after addition of auxin. (B) ChIP assay showing Brn1-V5 enrichment at the RDT1 promoter with and without auxin addition. The Brn1-V5 signal is relative to input and the ratio from an untagged control strain is normalized to 1. (C) RT-qPCR of RDT1 expression following 30 or 60 minutes of Brn1 depletion by auxin. (D) RT-qPCR of HMLALPHA2 expression following 30 or 60 min of Brn1 depletion by auxin. In panels C and D, the signals are relative to ACT1 control and normalized to 1.0 without auxin.(TIF)Click here for additional data file.S1 TableGenes closest to Sir2-dependent condensin peaks.This Excel spreadsheet lists the systematic ORF names of all genes that were closest to Sir2-dependent condensin peaks, as chosen using MACS.(XLSX)Click here for additional data file.S2 TableYeast strains.List of all Saccharomyces cerevisiae strains used in this study, along with their genotypes and source.(DOCX)Click here for additional data file.S3 TableOligonucleotides.List of oligodeoxynucleotides used in this study.(DOCX)Click here for additional data file.

Supporting information

MATa-specific transcription of RDT1 is repressed by Sir2 and Hst1.

(A) IGV screenshot of compiled raw RNA-seq read data from BY4741 (MATa) and BY4742 (MATα) strains. The top two blue peaks represent Smc4-myc and Sir2-myc ChIP-seq reads. (B) Quantitative ChIP assay showing additional SIR complex subunit enrichment at the RDT1 promoter. Signals are relative to input. (C) RT-qPCR showing effects of deleting SIR2 and/or HST1 on RDT1 expression (relative to ACT1) when HML is present or deleted (*p<0.05, **p < .005). The WT RDT1/ACT1 ratio is normalized to 1.

Click here for additional data file.

Deletion of Sir2 or the RDT1 promoter Sir2/condensin binding site does not affect protein levels of Sir2 or Myc-tagged condensin subunits.

(A) Western blot showing steady state Sir2 protein levels in WT (ML1), sir2Δ (ML25), and 100bpΔ (ML275) strains. (B) Western blot with anti-Myc detection of Brn1-13xMyc or Sir2 in WT (ML149), sir2Δ (ML161), and 100bpΔ (ML322) strains. (C) Western blot with anti-Myc detection of Smc4-13xMyc or Sir2 in WT (ML152), sir2Δ (ML160), and 100bpΔ version.

Click here for additional data file.

The RDT1-proximal Mcm1/α2 binding site (DPS2) is important for Sir2 and condensin recruitment.

(A) Schematic diagram depicting the location of the DPS2 sequence deletion relative to other elements with the RE, with the deleted chromosome III coordinates indicated in red. (B) Quantitative ChIP of native Sir2 in WT and dps2Δ strains. (C) Quantitative ChIP of Brn1-Myc in WT and dps2Δ strains. @RDT1 promoter indicates enrichment at the Sir2/condensin peak. ChIP signals are plotted relative to the input signal. (**p<0.005).

Click here for additional data file.

Deletion of the RDT1 open reading frame has no effect on mating-type switching.

(A) Representative time course of switching from MATa to MATα in WT (ML440) and rdt1Δ (ML443) strains. PCR products are specific to MATα and an SCR1 loading control. (B) Quantitation of the average MATα/SCR1 PCR signal ratio from 3 biological replicates. (C) Schematic of chromosome III in the donor preference reporter strain harboring an artificial HMRα-B cassette as the donor for switching, which introduces a unique BamHI site to the MAT locus. (D) Representative ethidium bromide stained agarose gel of BamHI-digested MATα PCR products after mating-type switching in WT (XW652) and rdt1Δ (MD30) strains. The bottom two bands represent HMR-derived switching. (E) Quantitation of average MATα-B utilization from 3 biological replicates.

Click here for additional data file.

Deleting the Sir2/condensin binding site within the RE (100bpΔ) does not alter Sir2 function at HMLα.

(A) Quantitative mating assay for WT (ML1) and 100bpΔ (ML275) strains. (B) Quantitative ChIP assay showing Sir2 enrichment at HML-I in WT (ML1) and 100bpΔ (ML275) strains. (**p<0.005).

Click here for additional data file.

HML is accessible to cleavage by HO endonuclease in sir2Δ strains.

(A) PCR amplification of MATa, HML, HMR, and SCR1 loci from MATa WT (ML440) and sir2Δ (MD29) strains containing the pGAL-HO-URA3 expression vector. Times after HO induction with galactose are indicated. A representative experiment is depicted. (B) Quantitation of mean MATa PCR signal, relative to the SCR1 control, from 3 independent biological replicates. Error bars indicate standard deviation. (C) Quantitation of mean HML PCR signal, as done for panel B. (D) Quantitation of mean HMR PCR signal, as done for panel B. (E and F) Real-time qPCR signal, relative to SCR1 control, for HML and HMR, respectively. The PCR signal at time 0 is normalized to 1.0 in each panel.

Click here for additional data file.

Auxin inducible degron (AID)-mediated depletion of Brn1 does not derepress RDT1 or HMLα.

(A) Western blot time course of auxin induced degradation of Brn1-V5. Time indicates minutes after addition of auxin. (B) ChIP assay showing Brn1-V5 enrichment at the RDT1 promoter with and without auxin addition. The Brn1-V5 signal is relative to input and the ratio from an untagged control strain is normalized to 1. (C) RT-qPCR of RDT1 expression following 30 or 60 minutes of Brn1 depletion by auxin. (D) RT-qPCR of HMLALPHA2 expression following 30 or 60 min of Brn1 depletion by auxin. In panels C and D, the signals are relative to ACT1 control and normalized to 1.0 without auxin.

Click here for additional data file.

Genes closest to Sir2-dependent condensin peaks.

This Excel spreadsheet lists the systematic ORF names of all genes that were closest to Sir2-dependent condensin peaks, as chosen using MACS.

Click here for additional data file.

Yeast strains.

List of all Saccharomyces cerevisiae strains used in this study, along with their genotypes and source.

Click here for additional data file.

Oligonucleotides.

List of oligodeoxynucleotides used in this study.

Click here for additional data file.
