---
pmid: '9166431'
title: 'V(D)J recombination: modulation of RAG1 and RAG2 cleavage activity on 12/23
  substrates by whole cell extract and DNA-bending proteins.'
authors:
- Sawchuk DJ
- Weis-Garcia F
- Malik S
- Besmer E
- Bustin M
- Nussenzweig MC
- Cortes P
journal: J Exp Med
year: '1997'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC2196333
doi: 10.1084/jem.185.11.2025
---

# V(D)J recombination: modulation of RAG1 and RAG2 cleavage activity on 12/23 substrates by whole cell extract and DNA-bending proteins.
**Authors:** Sawchuk DJ, Weis-Garcia F, Malik S, Besmer E, Bustin M, Nussenzweig MC, Cortes P
**Journal:** J Exp Med (1997)
**DOI:** [10.1084/jem.185.11.2025](https://doi.org/10.1084/jem.185.11.2025)
**PMC:** [PMC2196333](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2196333/)

## Abstract

1. J Exp Med. 1997 Jun 2;185(11):2025-32. doi: 10.1084/jem.185.11.2025.

V(D)J recombination: modulation of RAG1 and RAG2 cleavage activity on 12/23 
substrates by whole cell extract and DNA-bending proteins.

Sawchuk DJ(1), Weis-Garcia F, Malik S, Besmer E, Bustin M, Nussenzweig MC, 
Cortes P.

Author information:
(1)Laboratory of Molecular Immunology, The Rockefeller University, New York 
10021, USA.

Antigen receptor gene rearrangement is directed by DNA motifs consisting of a 
conserved heptamer and nonamer separated by a nonconserved spacer of either 12 
or 23 base pairs (12 or 23 recombination signal sequences [RSS]). V(D)J 
recombination requires that the rearranging DNA segments be flanked by RSSs of 
different spacer lengths, a phenomenon known as the 12/23 rule. Recent studies 
have shown that this restriction operates at the level of DNA cleavage, which is 
mediated by the products of the recombination activating genes RAG1 and RAG2. 
Here, we show that RAG1 and RAG2 are not sufficient for 12/23 dependent 
cleavage, whereas RAG1 and RAG2 complemented with whole cell extract faithfully 
recapitulates the 12/23 rule. In addition, HMG box containing proteins HMG1 and 
HMG2 enhance RAG1- and RAG2-mediated cleavage of substrates containing 23 RSS 
but not of substrates containing only 12 RSS. These results suggest the 
existence of a nucleoprotein complex at the cleavage site, consisting of 
architectural, catalytic, and regulatory components.

DOI: 10.1084/jem.185.11.2025
PMCID: PMC2196333
PMID: 9166431 [Indexed for MEDLINE]

## Full Text

Rearrangement of antigen receptor genes occurs via a  process known as V(D)J recombination, which can be divided into two phases. The first part of the reaction involves recognition and cleavage of the DNA, and the second phase involves resolution and joining (1, 2). Recognition and cleavage are mediated by the lymphoid-specific RAG1 and RAG2 proteins, which are directed in cis by recombination signal sequences (RSSs)1 (3–7). The RSSs consist of well-conserved heptamer and nonamer motifs separated by nonconserved spacers of 12 or 23 bp. V(D)J recombination only occurs between DNA elements that are flanked by RSSs containing different spacers, a phenomenon known as the 12/23 rule (8, 9). In vivo and in vitro studies have shown that the 12/23 rule is imposed during the DNA cleavage reaction (10–12). However, the precise molecular requirements for 12/23-restricted cleavage have not yet been determined. Eastman et al. (10) found that unfractionated lymphoid cell extracts overexpressing RAG1 and RAG2 can mediate 12/23 regulated cleavage in vitro. In contrast, van Gent et al. (12) showed only preferential cleavage of a 12/23 RSS substrate using purified recombinant RAG1 and RAG2 proteins, suggesting that RAG1 and RAG2 alone may not be sufficient for strict adherence to the 12/23 rule.

Cleavage of the RSS by RAG1 and RAG2 in the first phase of the V(D)J recombination reaction results in the production of signal and coding ends (3, 4). The signal ends are blunt and 5′ phosphorylated, whereas coding ends are hairpin-like structures that are generated by a transesterification mechanism (3, 4, 13–17). In the second phase of the V(D)J recombination reaction, the hairpin coding ends are resolved and both signal and coding ends are joined to produce signal and coding joints.

A number of ubiquitously expressed DNA repair proteins have been shown to play a role in the second step of V(D)J recombination. These include XRCC4, Ku-80 antigen, and the large catalytic subunit of DNA-dependent protein kinase (product of scid) (1, 2, 18). Other ubiquitous factors may also play a role in V(D)J rearrangement, including DNA-bending proteins, which are used extensively in prokaryotic recombination systems (for reviews see 19, 20). In these reactions, DNA-bending proteins seem to play an architectural role by bending DNA into conformations that facilitate interactions between other proteins.

Here, we present experiments that explore the molecular requirements for the 12/23 rule. Our results indicate that RAG1 and RAG2 are required but not sufficient to regulate 12/23-dependent cleavage, and that other ubiquitously expressed cellular factors cooperate with RAGs to establish the 12/23 rule. DNA-bending proteins specifically enhance RAG-mediated cleavage of substrates containing a 23 RSS, but are unable to impose 12/23 restriction in vitro.

Materials and MethodsTissue Culture and Transfection.293T cells were plated on Corning 25020 plates and cultured at 37°C and 5% CO2, in DMEM containing 10% heat-inactivated calf serum (GIBCO BRL, Gaithersburg, MD) (21). After a minimum of three passages, cells were transfected, at a confluency of 30%. The 293 cells were cotransfected with core RAG1–GST fusion and core RAG2–GST fusion constructs as described previously (22). 48 h after transfection, cell monolayers were washed and harvested for extract preparation and protein purification.Extract Preparation.Extracts from 293T cells cotransfected with core RAG1–GST and core RAG2–GST were made based on a modification of the protocol described in Eastman et al. (10). In brief, 4 ml pelleted cell volume was resuspended in 15 ml of buffer G, consisting of 20 mM Hepes–NaOH (pH 7.9), 20% glycerol, 1.5 mM MgCl2, 300 mM NaCl, 0.2 mM EDTA, 0.5 mM DTT, 0.5 mM PMSF, and other protease inhibitors. Samples were freeze–thawed five times. After the freeze–thaw cycles, the samples were allowed to extract on a rocking platform for 30 min at 4°C. Samples were centrifuged in a Beckman 50.2Ti rotor at 35,000 rpm for 55 min at 4°C, in a Beckman XL-90 preparative ultracentrifuge. Supernatant was saved and ammonium sulfate was added to 75% saturation at 4°C. After equilibration, samples were centrifuged at 12,000 rpm for 20 min at 4°C, in a Beckman J2-MI high speed centrifuge (JA-20 rotor). The ammonium sulfate pellet was then resuspended in 10 ml of dialysis buffer (pH 8.0), consisting of 100 mM NaCl, 20% glycerol, 50 μM ZnSO4, 25 mM Tris–HCl, 2 mM DTT, and 0.5 mM PMSF. The samples were dialyzed in 12,000 mol/wt cutoff dialysis membrane for 6–8 h, in 100× buffer volume, with three buffer changes. The concentration of the dialyzed extracts was estimated to be 10 mg/ml.Purification of Core RAG1–GST and Core RAG2–GST.900 μl of a 1:1 mix of glutathione beads (Molecular Probes, Oregon) and dialysis buffer was added to 10 ml of dialyzed 293 whole cell extract (WCE) containing coexpressed core RAG1–GST and core RAG2–GST (R1/R2 WCE). After overnight incubation at 4°C, samples were centrifuged at 1,000 rpm in a Beckman GS6KR swinging bucket centrifuge. Glutathione beads were then resuspended and washed in 5 ml of dialysis buffer. This process was repeated 12 times. RAG1 and RAG2 proteins bound to the glutathione beads were then eluted at 4°C in elution buffer consisting of 300 mM NaCl, 20% glycerol, 50 μM ZnSO4, 25 mM Tris-HCl (pH 8.2), 2 mM DTT, 0.5 mM PMSF and 20 mM glutathione, for 2 h. Samples were then centrifuged and eluted RAG proteins were dialyzed. Protein samples were electrophoresed on SDS-PAGE gels, and the concentrations of core RAG1–GST and core RAG2–GST were determined to be 100 and 200 ng/μl, respectively.Preparation of HMG1 and HMG2.HMG2 was prepared from calf thymus by extraction with 0.35 M NaCl followed by chromatography on Sephadex CM-25 (23). The purity of the protein was assessed by electrophoresis in polyacrylamide gels as described (24).Natural human HMG1 was purified from HeLa cell nuclear extracts (25) after chromatography over phosphocellulose (P11) in buffer A (20 mM Tris–HCl, pH 7.9, 0.1 mM EDTA, 20% glycerol, 5 mM DTT) essentially as described (26). The 0.85 M KCl eluate was dialyzed to 0.1 M KCl in buffer A and applied to DEAE–cellulose (DE-52) column. The flow through and the 0.15 M KCl step eluates were pooled, dialyzed to 50 mM KCl in the same buffer, and applied to a second DE-52 column. The column was developed with a linear gradient (0.05–0.25 M KCl). Fractions eluting between 0.075 and 0.12 M KCl were pooled, dialyzed to 0.1 M KCl, and chromatographed over an FPLC MonoS (Pharmacia, Piscataway, NJ) column, which was eluted with a linear gradient (0.1–0.5 M KCl). Fractions eluting at 0.12–0.15 M KCl were virtually homogeneous in HMG1.Histidine-tagged recombinant HMG1 was purified from a bacterial strain (BL21 DE pLysS) harboring the plasmid pET-HMG as others (27). After lysis and chromatography over Ni-NTA resin, the preparation was further enriched over S-Sepharose column. Material eluting at 0.15 M KCl was employed for all studies.Preparation of DNA Cleavage Substrate. Recombination substrates pJH290 12/23 (8), pJH290 12/12 were grown in XL-1 Blue cells (Stratagene, La Jolla, CA) and plasmid DNA was prepared with a Qiagen (Chatsworth, CA) maxi-prep kit. pJH290 23/ 23 deletion recombination substrate was constructed by digesting parental pJH290 12/23 with SalI, releasing a 12-RSS–containing fragment. Two annealed complementary oligonucleotides containing a 23 RSS, oligo400, 5′-TCGACCACAGTGGTAGTACTCCACTGTCTGGCTGTACAAAAACCCTCGG-3′; and oligo401, 5′-TCGACCGAGGGTTTTTGTACAGCCAGACAGTGGAGTACTACCACTGTGG-3′, were then ligated into the digested parental plasmid, yielding a clone containing two 23 RSS, as determined by restriction endonuclease analysis. Clones were then sequenced to confirm orientation. pJH290 23i (single 23 RSS) was generated by digesting parental pJH290 12/23 with SalI, and self-ligating the resulting digested parental vector. All substrates were then digested overnight with SapI and NcoI (New England Biolabs, Beverly, MA). The correct NcoI–SapI fragment was then gel-purified using agarose gel electrophoresis. pJH290 12/23, 12/12, 23/23, and 23i yielded RSS-containing fragments of 1162, 1151, 1172, and 1123 bp, respectively. 50 ng of each fragment was then double-labeled using exo− Klenow (Stratagene, La Jolla, CA), and α-[32P]dCTP and α-[32P]dATP (DuPont–NEN, Boston, MA). Labeled cleavage substrates, in a volume of 300 μl, were then purified using a Sephadex G50 column (Pharmacia, Piscataway, NJ), and scintillation counting was performed. ∼40,000 cpm of each substrate was then used for each cleavage reaction.Cleavage Reactions.Cleavage reaction conditions were essentially as described in Eastman et al. (10). The total volume of the reaction was 50 μl, consisting of copurified RAG1/RAG2 (1 or 2 μl), high mobility group 1 (HMG1) (0.3 or 0.6 μl at 1 μg/μl), recombinant HMG as others (0.3 or 0.6 μl at 200 ng/μl), HMG2 (1.0 or 2.0 μl at 100 ng/μl), integration host factor (IHF) (0.5 or 1.0 μl at 400 ng/μl), HU (0.5 or 1.0 μl at 1 μg/μl), untransfected whole cell extract (0.2 μl), and RAG1/RAG2 coexpressed whole cell extracts (1 or 2 μl), in various combinations. This volume was adjusted to 20 μl with dialysis buffer, and 30 μl of a master mix containing 40,000 counts of cleavage substrate, NaAc2, MgAc2, Na-Hepes, EGTA, dNTPs, and ATP was added. Final concentrations were as follows: 150, 10, 25, and 1 mM, respectively. After preincubaton on ice for 5 min, samples were incubated at 37°C for 1 h. The reactions were terminated by the addition of 0.5% SDS and excess EDTA. Phenol–chloroform extractions were then followed by ethanol precipitation, and resuspension in TE buffer. Samples were loaded and resolved on a 4.5% nondenaturing polyacrylamide gel. Quantitative analysis of the cleavage assays was performed using a Storm phosphorimager and Imagequant software (Molecular Dynamics, Sunnyvale, CA). Dried gels were also autoradiographed.

Materials and Methods

Tissue Culture and Transfection.293T cells were plated on Corning 25020 plates and cultured at 37°C and 5% CO2, in DMEM containing 10% heat-inactivated calf serum (GIBCO BRL, Gaithersburg, MD) (21). After a minimum of three passages, cells were transfected, at a confluency of 30%. The 293 cells were cotransfected with core RAG1–GST fusion and core RAG2–GST fusion constructs as described previously (22). 48 h after transfection, cell monolayers were washed and harvested for extract preparation and protein purification.

Tissue Culture and Transfection.

293T cells were plated on Corning 25020 plates and cultured at 37°C and 5% CO2, in DMEM containing 10% heat-inactivated calf serum (GIBCO BRL, Gaithersburg, MD) (21). After a minimum of three passages, cells were transfected, at a confluency of 30%. The 293 cells were cotransfected with core RAG1–GST fusion and core RAG2–GST fusion constructs as described previously (22). 48 h after transfection, cell monolayers were washed and harvested for extract preparation and protein purification.

Extract Preparation.Extracts from 293T cells cotransfected with core RAG1–GST and core RAG2–GST were made based on a modification of the protocol described in Eastman et al. (10). In brief, 4 ml pelleted cell volume was resuspended in 15 ml of buffer G, consisting of 20 mM Hepes–NaOH (pH 7.9), 20% glycerol, 1.5 mM MgCl2, 300 mM NaCl, 0.2 mM EDTA, 0.5 mM DTT, 0.5 mM PMSF, and other protease inhibitors. Samples were freeze–thawed five times. After the freeze–thaw cycles, the samples were allowed to extract on a rocking platform for 30 min at 4°C. Samples were centrifuged in a Beckman 50.2Ti rotor at 35,000 rpm for 55 min at 4°C, in a Beckman XL-90 preparative ultracentrifuge. Supernatant was saved and ammonium sulfate was added to 75% saturation at 4°C. After equilibration, samples were centrifuged at 12,000 rpm for 20 min at 4°C, in a Beckman J2-MI high speed centrifuge (JA-20 rotor). The ammonium sulfate pellet was then resuspended in 10 ml of dialysis buffer (pH 8.0), consisting of 100 mM NaCl, 20% glycerol, 50 μM ZnSO4, 25 mM Tris–HCl, 2 mM DTT, and 0.5 mM PMSF. The samples were dialyzed in 12,000 mol/wt cutoff dialysis membrane for 6–8 h, in 100× buffer volume, with three buffer changes. The concentration of the dialyzed extracts was estimated to be 10 mg/ml.

Extract Preparation.

Extracts from 293T cells cotransfected with core RAG1–GST and core RAG2–GST were made based on a modification of the protocol described in Eastman et al. (10). In brief, 4 ml pelleted cell volume was resuspended in 15 ml of buffer G, consisting of 20 mM Hepes–NaOH (pH 7.9), 20% glycerol, 1.5 mM MgCl2, 300 mM NaCl, 0.2 mM EDTA, 0.5 mM DTT, 0.5 mM PMSF, and other protease inhibitors. Samples were freeze–thawed five times. After the freeze–thaw cycles, the samples were allowed to extract on a rocking platform for 30 min at 4°C. Samples were centrifuged in a Beckman 50.2Ti rotor at 35,000 rpm for 55 min at 4°C, in a Beckman XL-90 preparative ultracentrifuge. Supernatant was saved and ammonium sulfate was added to 75% saturation at 4°C. After equilibration, samples were centrifuged at 12,000 rpm for 20 min at 4°C, in a Beckman J2-MI high speed centrifuge (JA-20 rotor). The ammonium sulfate pellet was then resuspended in 10 ml of dialysis buffer (pH 8.0), consisting of 100 mM NaCl, 20% glycerol, 50 μM ZnSO4, 25 mM Tris–HCl, 2 mM DTT, and 0.5 mM PMSF. The samples were dialyzed in 12,000 mol/wt cutoff dialysis membrane for 6–8 h, in 100× buffer volume, with three buffer changes. The concentration of the dialyzed extracts was estimated to be 10 mg/ml.

Purification of Core RAG1–GST and Core RAG2–GST.900 μl of a 1:1 mix of glutathione beads (Molecular Probes, Oregon) and dialysis buffer was added to 10 ml of dialyzed 293 whole cell extract (WCE) containing coexpressed core RAG1–GST and core RAG2–GST (R1/R2 WCE). After overnight incubation at 4°C, samples were centrifuged at 1,000 rpm in a Beckman GS6KR swinging bucket centrifuge. Glutathione beads were then resuspended and washed in 5 ml of dialysis buffer. This process was repeated 12 times. RAG1 and RAG2 proteins bound to the glutathione beads were then eluted at 4°C in elution buffer consisting of 300 mM NaCl, 20% glycerol, 50 μM ZnSO4, 25 mM Tris-HCl (pH 8.2), 2 mM DTT, 0.5 mM PMSF and 20 mM glutathione, for 2 h. Samples were then centrifuged and eluted RAG proteins were dialyzed. Protein samples were electrophoresed on SDS-PAGE gels, and the concentrations of core RAG1–GST and core RAG2–GST were determined to be 100 and 200 ng/μl, respectively.

Purification of Core RAG1–GST and Core RAG2–GST.

900 μl of a 1:1 mix of glutathione beads (Molecular Probes, Oregon) and dialysis buffer was added to 10 ml of dialyzed 293 whole cell extract (WCE) containing coexpressed core RAG1–GST and core RAG2–GST (R1/R2 WCE). After overnight incubation at 4°C, samples were centrifuged at 1,000 rpm in a Beckman GS6KR swinging bucket centrifuge. Glutathione beads were then resuspended and washed in 5 ml of dialysis buffer. This process was repeated 12 times. RAG1 and RAG2 proteins bound to the glutathione beads were then eluted at 4°C in elution buffer consisting of 300 mM NaCl, 20% glycerol, 50 μM ZnSO4, 25 mM Tris-HCl (pH 8.2), 2 mM DTT, 0.5 mM PMSF and 20 mM glutathione, for 2 h. Samples were then centrifuged and eluted RAG proteins were dialyzed. Protein samples were electrophoresed on SDS-PAGE gels, and the concentrations of core RAG1–GST and core RAG2–GST were determined to be 100 and 200 ng/μl, respectively.

Preparation of HMG1 and HMG2.HMG2 was prepared from calf thymus by extraction with 0.35 M NaCl followed by chromatography on Sephadex CM-25 (23). The purity of the protein was assessed by electrophoresis in polyacrylamide gels as described (24).Natural human HMG1 was purified from HeLa cell nuclear extracts (25) after chromatography over phosphocellulose (P11) in buffer A (20 mM Tris–HCl, pH 7.9, 0.1 mM EDTA, 20% glycerol, 5 mM DTT) essentially as described (26). The 0.85 M KCl eluate was dialyzed to 0.1 M KCl in buffer A and applied to DEAE–cellulose (DE-52) column. The flow through and the 0.15 M KCl step eluates were pooled, dialyzed to 50 mM KCl in the same buffer, and applied to a second DE-52 column. The column was developed with a linear gradient (0.05–0.25 M KCl). Fractions eluting between 0.075 and 0.12 M KCl were pooled, dialyzed to 0.1 M KCl, and chromatographed over an FPLC MonoS (Pharmacia, Piscataway, NJ) column, which was eluted with a linear gradient (0.1–0.5 M KCl). Fractions eluting at 0.12–0.15 M KCl were virtually homogeneous in HMG1.Histidine-tagged recombinant HMG1 was purified from a bacterial strain (BL21 DE pLysS) harboring the plasmid pET-HMG as others (27). After lysis and chromatography over Ni-NTA resin, the preparation was further enriched over S-Sepharose column. Material eluting at 0.15 M KCl was employed for all studies.

Preparation of HMG1 and HMG2.

HMG2 was prepared from calf thymus by extraction with 0.35 M NaCl followed by chromatography on Sephadex CM-25 (23). The purity of the protein was assessed by electrophoresis in polyacrylamide gels as described (24).

Natural human HMG1 was purified from HeLa cell nuclear extracts (25) after chromatography over phosphocellulose (P11) in buffer A (20 mM Tris–HCl, pH 7.9, 0.1 mM EDTA, 20% glycerol, 5 mM DTT) essentially as described (26). The 0.85 M KCl eluate was dialyzed to 0.1 M KCl in buffer A and applied to DEAE–cellulose (DE-52) column. The flow through and the 0.15 M KCl step eluates were pooled, dialyzed to 50 mM KCl in the same buffer, and applied to a second DE-52 column. The column was developed with a linear gradient (0.05–0.25 M KCl). Fractions eluting between 0.075 and 0.12 M KCl were pooled, dialyzed to 0.1 M KCl, and chromatographed over an FPLC MonoS (Pharmacia, Piscataway, NJ) column, which was eluted with a linear gradient (0.1–0.5 M KCl). Fractions eluting at 0.12–0.15 M KCl were virtually homogeneous in HMG1.

Histidine-tagged recombinant HMG1 was purified from a bacterial strain (BL21 DE pLysS) harboring the plasmid pET-HMG as others (27). After lysis and chromatography over Ni-NTA resin, the preparation was further enriched over S-Sepharose column. Material eluting at 0.15 M KCl was employed for all studies.

Preparation of DNA Cleavage Substrate. Recombination substrates pJH290 12/23 (8), pJH290 12/12 were grown in XL-1 Blue cells (Stratagene, La Jolla, CA) and plasmid DNA was prepared with a Qiagen (Chatsworth, CA) maxi-prep kit. pJH290 23/ 23 deletion recombination substrate was constructed by digesting parental pJH290 12/23 with SalI, releasing a 12-RSS–containing fragment. Two annealed complementary oligonucleotides containing a 23 RSS, oligo400, 5′-TCGACCACAGTGGTAGTACTCCACTGTCTGGCTGTACAAAAACCCTCGG-3′; and oligo401, 5′-TCGACCGAGGGTTTTTGTACAGCCAGACAGTGGAGTACTACCACTGTGG-3′, were then ligated into the digested parental plasmid, yielding a clone containing two 23 RSS, as determined by restriction endonuclease analysis. Clones were then sequenced to confirm orientation. pJH290 23i (single 23 RSS) was generated by digesting parental pJH290 12/23 with SalI, and self-ligating the resulting digested parental vector. All substrates were then digested overnight with SapI and NcoI (New England Biolabs, Beverly, MA). The correct NcoI–SapI fragment was then gel-purified using agarose gel electrophoresis. pJH290 12/23, 12/12, 23/23, and 23i yielded RSS-containing fragments of 1162, 1151, 1172, and 1123 bp, respectively. 50 ng of each fragment was then double-labeled using exo− Klenow (Stratagene, La Jolla, CA), and α-[32P]dCTP and α-[32P]dATP (DuPont–NEN, Boston, MA). Labeled cleavage substrates, in a volume of 300 μl, were then purified using a Sephadex G50 column (Pharmacia, Piscataway, NJ), and scintillation counting was performed. ∼40,000 cpm of each substrate was then used for each cleavage reaction.

Preparation of DNA Cleavage Substrate.

Recombination substrates pJH290 12/23 (8), pJH290 12/12 were grown in XL-1 Blue cells (Stratagene, La Jolla, CA) and plasmid DNA was prepared with a Qiagen (Chatsworth, CA) maxi-prep kit. pJH290 23/ 23 deletion recombination substrate was constructed by digesting parental pJH290 12/23 with SalI, releasing a 12-RSS–containing fragment. Two annealed complementary oligonucleotides containing a 23 RSS, oligo400, 5′-TCGACCACAGTGGTAGTACTCCACTGTCTGGCTGTACAAAAACCCTCGG-3′; and oligo401, 5′-TCGACCGAGGGTTTTTGTACAGCCAGACAGTGGAGTACTACCACTGTGG-3′, were then ligated into the digested parental plasmid, yielding a clone containing two 23 RSS, as determined by restriction endonuclease analysis. Clones were then sequenced to confirm orientation. pJH290 23i (single 23 RSS) was generated by digesting parental pJH290 12/23 with SalI, and self-ligating the resulting digested parental vector. All substrates were then digested overnight with SapI and NcoI (New England Biolabs, Beverly, MA). The correct NcoI–SapI fragment was then gel-purified using agarose gel electrophoresis. pJH290 12/23, 12/12, 23/23, and 23i yielded RSS-containing fragments of 1162, 1151, 1172, and 1123 bp, respectively. 50 ng of each fragment was then double-labeled using exo− Klenow (Stratagene, La Jolla, CA), and α-[32P]dCTP and α-[32P]dATP (DuPont–NEN, Boston, MA). Labeled cleavage substrates, in a volume of 300 μl, were then purified using a Sephadex G50 column (Pharmacia, Piscataway, NJ), and scintillation counting was performed. ∼40,000 cpm of each substrate was then used for each cleavage reaction.

Cleavage Reactions.Cleavage reaction conditions were essentially as described in Eastman et al. (10). The total volume of the reaction was 50 μl, consisting of copurified RAG1/RAG2 (1 or 2 μl), high mobility group 1 (HMG1) (0.3 or 0.6 μl at 1 μg/μl), recombinant HMG as others (0.3 or 0.6 μl at 200 ng/μl), HMG2 (1.0 or 2.0 μl at 100 ng/μl), integration host factor (IHF) (0.5 or 1.0 μl at 400 ng/μl), HU (0.5 or 1.0 μl at 1 μg/μl), untransfected whole cell extract (0.2 μl), and RAG1/RAG2 coexpressed whole cell extracts (1 or 2 μl), in various combinations. This volume was adjusted to 20 μl with dialysis buffer, and 30 μl of a master mix containing 40,000 counts of cleavage substrate, NaAc2, MgAc2, Na-Hepes, EGTA, dNTPs, and ATP was added. Final concentrations were as follows: 150, 10, 25, and 1 mM, respectively. After preincubaton on ice for 5 min, samples were incubated at 37°C for 1 h. The reactions were terminated by the addition of 0.5% SDS and excess EDTA. Phenol–chloroform extractions were then followed by ethanol precipitation, and resuspension in TE buffer. Samples were loaded and resolved on a 4.5% nondenaturing polyacrylamide gel. Quantitative analysis of the cleavage assays was performed using a Storm phosphorimager and Imagequant software (Molecular Dynamics, Sunnyvale, CA). Dried gels were also autoradiographed.

Cleavage Reactions.

Cleavage reaction conditions were essentially as described in Eastman et al. (10). The total volume of the reaction was 50 μl, consisting of copurified RAG1/RAG2 (1 or 2 μl), high mobility group 1 (HMG1) (0.3 or 0.6 μl at 1 μg/μl), recombinant HMG as others (0.3 or 0.6 μl at 200 ng/μl), HMG2 (1.0 or 2.0 μl at 100 ng/μl), integration host factor (IHF) (0.5 or 1.0 μl at 400 ng/μl), HU (0.5 or 1.0 μl at 1 μg/μl), untransfected whole cell extract (0.2 μl), and RAG1/RAG2 coexpressed whole cell extracts (1 or 2 μl), in various combinations. This volume was adjusted to 20 μl with dialysis buffer, and 30 μl of a master mix containing 40,000 counts of cleavage substrate, NaAc2, MgAc2, Na-Hepes, EGTA, dNTPs, and ATP was added. Final concentrations were as follows: 150, 10, 25, and 1 mM, respectively. After preincubaton on ice for 5 min, samples were incubated at 37°C for 1 h. The reactions were terminated by the addition of 0.5% SDS and excess EDTA. Phenol–chloroform extractions were then followed by ethanol precipitation, and resuspension in TE buffer. Samples were loaded and resolved on a 4.5% nondenaturing polyacrylamide gel. Quantitative analysis of the cleavage assays was performed using a Storm phosphorimager and Imagequant software (Molecular Dynamics, Sunnyvale, CA). Dried gels were also autoradiographed.

ResultsRAG1 and RAG2 Complemented with Whole Cell Extracts Show Strict 12/23 Regulation.Experiments with purified recombinant RAG1 and RAG2 proteins have shown preferential, but incomplete 12/23-regulated cleavage by the isolated proteins (12). In contrast, extracts from lymphoid cells overexpressing RAG1 and RAG2 showed strict adherence to the 12/23 rule (10). To clarify the requirements for regulated cleavage, we performed cleavage reactions using crude extracts from cells expressing RAG1 and RAG2, purified RAG1 and RAG2, and combinations of extracts and purified proteins. Extracts containing RAG1 and RAG2 were prepared from 293T cells that were transiently transfected with plasmids that encode RAG1 and RAG2 GST fusion proteins. RAG1 and RAG2 GST fusion proteins were copurified directly from the same transfected 293T cell extracts, and control extracts were from 293T cells that did not express RAG1 or RAG2.In a typical cleavage reaction we used a 1,162 bp DNA fragment from the deletion substrate pJH290 (Fig. 1) (8). This fragment contains a 12 and a 23 RSS and will be referred to as the 12/23 substrate. In all of the experiments, the 12/23 substrate was labeled at both ends with [32P]dNTPs. Upon regulated cleavage, three fragments are produced in equimolar amounts. The two end fragments are labeled (262 and 571 bp), whereas the third internal fragment (329 bp) is not labeled. In contrast with regulated cleavage, unregulated cleavage of the 12/23 substrate at the 12 RSS alone would generate two labeled fragments of 262 and 900 bp, whereas unregulated cleavage at the 23 RSS would produce two labeled fragments of 571 and 591, respectively (Fig. 1). Control substrates containing either 12/12, 23/23, and a single 23 RSS were produced in a similar manner (Fig. 1; Materials and Methods). Whole cell extracts produced from RAG1- and RAG2transfected cells showed strict 12/23-restricted cleavage (Fig. 2, A and B; and reference 10). There was no detectable cleavage on either 12/12 or 23/23 substrates, and there were no unexpected cleavage products with the 12/ 23 substrate (Fig. 2, A and B). In contrast, RAGs purified from the same transfected 293T cell extracts showed only a modest preference for cleaving the 12/23 substrate (Fig. 2, C and E). In contrast with the RAG-containing extracts, the purified proteins were also active on 12/12 and 23/23 RSS substrates. Quantitation with a phosphoimager showed 50% and 30% of the level of cleavage on 12/12 and 23/23 substrates when compared with the 12/23 substrate (Fig. 2, C and E; data not shown). Consistent with the lack of specificity, we also found unexpected cryptic cleavage products on the 12/23 substrate with the purified proteins (Fig. 2
C, asterisks). We conclude that crude cellular extracts containing RAG1 and RAG2 strictly adhere to the 12/23 rule, whereas RAGs purified from the same extracts have only a two- to threefold preference for a 12/23 substrate. To determine whether 12/23-regulated cleavage could be restored to the purified RAG1 and RAG2 proteins with factors found in whole cell extracts, we combined control extracts from untransfected 293T cells with purified RAG proteins. Addition of the untransfected cell extracts to purified RAGs had two prominent effects. First, there was a two- to threefold inhibition of the cleavage activity, resulting in a level of cleavage similar to that obtained with the RAG1/RAG2-transfected cell extracts. Second, the control extracts converted the preferential cleavage reaction seen with purified RAGs to the strictly regulated pattern seen with the extracts from transfected cells. In the presence of the control extracts, the 12/23 substrate was cleaved such that the 262 and 571 bp products of coupled and regulated cleavage were found in the same ratios as obtained with extracts from RAG-transfected cells. In addition, the cryptic cleavage products seen with purified RAGs on the 12/23 substrate were suppressed (Fig. 2, D and E). Further, 12/12 and 23/23 cleavage by purified RAG1 and RAG2 was completely inhibited by the extracts of untransfected 293T cells (Fig. 2, D and E). No cleavage was observed with the 12/12 and 23/23 control substrates even upon prolonged exposure (data not shown). We conclude that untransfected 293T cell extracts contain an activity that complements purified RAGs and promotes stringent adherence to the 12/23 rule.HMG Box–containing Proteins and Prokaryotic Counterparts Enhance RAG-mediated Cleavage of 23 RSS-containing Substrates.DNA-bending proteins have been shown to be important components of a number of recombination reactions that require DNA cleavage and formation of multicomponent DNA–protein complexes (for reviews see references 19, 20). The role of the bending proteins in these reactions made them good candidates to participate in, and potentially regulate, RAG1- and RAG2-mediated DNA cleavage. To investigate the role of DNA-bending proteins on 12/23-regulated cleavage we performed cleavage reactions with purified RAG1 and RAG2 in the presence of four different DNA-bending proteins that are known to bend DNA and to play architectural roles: HMG1, rHMG1 (recombinant), HMG2, IHF, and HU. As indicated in Fig. 3, three of these proteins, HMG1, rHMG1, HMG2, and the prokaryotic HU enhanced the activity of RAG1 and RAG2 on the 12/23 substrate. The magnitude of the enhancement on the 12/23 substrate was three- to fourfold when quantitated. IHF differed from the other bending proteins in that it failed to enhance cleavage and was inhibitory for RAG-mediated cleavage of 12/23 and 12/12 substrates. Despite the enhanced cleavage by the purified RAGs in the presence of DNA-bending proteins there was no significant effect on the stringency of 12/23 regulation. A similar threefold increase in cleavage was also seen with a combination of purified RAG1 and RAG2 and either HMG1, rHMG1, HMG2, or HU on both 23/23 and 23i substrates (Fig. 3
C; data not shown). In contrast to 23RSS–containing substrates, there was no significant enhancement of cleavage by the DNA-bending proteins on a 12/12 substrate as seen in Fig. 3
B and confirmed by quantitation (data not shown). Thus, the addition of the DNAbending proteins to purified RAGs enhanced cleavage of DNA substrates that contained the 23 RSS, but they failed to induce regulated cleavage. Because DNA-bending proteins enhanced cleavage of 23 RSS containing substrates, whereas unfractionated cell extracts confer regulated cleavage, we asked whether we could both enhance and regulate cleavage by combining the two. We found that the extracts had a dominant effect decreasing the overall level of activity and increasing the specificity of the reaction (Fig. 4
A). There was complete inhibition of the 12/12, 23/23, and 23 substrate cleavage even when extract and rHMG1 were combined (Fig. 4, B–D).

RAG1 and RAG2 Complemented with Whole Cell Extracts Show Strict 12/23 Regulation.Experiments with purified recombinant RAG1 and RAG2 proteins have shown preferential, but incomplete 12/23-regulated cleavage by the isolated proteins (12). In contrast, extracts from lymphoid cells overexpressing RAG1 and RAG2 showed strict adherence to the 12/23 rule (10). To clarify the requirements for regulated cleavage, we performed cleavage reactions using crude extracts from cells expressing RAG1 and RAG2, purified RAG1 and RAG2, and combinations of extracts and purified proteins. Extracts containing RAG1 and RAG2 were prepared from 293T cells that were transiently transfected with plasmids that encode RAG1 and RAG2 GST fusion proteins. RAG1 and RAG2 GST fusion proteins were copurified directly from the same transfected 293T cell extracts, and control extracts were from 293T cells that did not express RAG1 or RAG2.In a typical cleavage reaction we used a 1,162 bp DNA fragment from the deletion substrate pJH290 (Fig. 1) (8). This fragment contains a 12 and a 23 RSS and will be referred to as the 12/23 substrate. In all of the experiments, the 12/23 substrate was labeled at both ends with [32P]dNTPs. Upon regulated cleavage, three fragments are produced in equimolar amounts. The two end fragments are labeled (262 and 571 bp), whereas the third internal fragment (329 bp) is not labeled. In contrast with regulated cleavage, unregulated cleavage of the 12/23 substrate at the 12 RSS alone would generate two labeled fragments of 262 and 900 bp, whereas unregulated cleavage at the 23 RSS would produce two labeled fragments of 571 and 591, respectively (Fig. 1). Control substrates containing either 12/12, 23/23, and a single 23 RSS were produced in a similar manner (Fig. 1; Materials and Methods). Whole cell extracts produced from RAG1- and RAG2transfected cells showed strict 12/23-restricted cleavage (Fig. 2, A and B; and reference 10). There was no detectable cleavage on either 12/12 or 23/23 substrates, and there were no unexpected cleavage products with the 12/ 23 substrate (Fig. 2, A and B). In contrast, RAGs purified from the same transfected 293T cell extracts showed only a modest preference for cleaving the 12/23 substrate (Fig. 2, C and E). In contrast with the RAG-containing extracts, the purified proteins were also active on 12/12 and 23/23 RSS substrates. Quantitation with a phosphoimager showed 50% and 30% of the level of cleavage on 12/12 and 23/23 substrates when compared with the 12/23 substrate (Fig. 2, C and E; data not shown). Consistent with the lack of specificity, we also found unexpected cryptic cleavage products on the 12/23 substrate with the purified proteins (Fig. 2
C, asterisks). We conclude that crude cellular extracts containing RAG1 and RAG2 strictly adhere to the 12/23 rule, whereas RAGs purified from the same extracts have only a two- to threefold preference for a 12/23 substrate. To determine whether 12/23-regulated cleavage could be restored to the purified RAG1 and RAG2 proteins with factors found in whole cell extracts, we combined control extracts from untransfected 293T cells with purified RAG proteins. Addition of the untransfected cell extracts to purified RAGs had two prominent effects. First, there was a two- to threefold inhibition of the cleavage activity, resulting in a level of cleavage similar to that obtained with the RAG1/RAG2-transfected cell extracts. Second, the control extracts converted the preferential cleavage reaction seen with purified RAGs to the strictly regulated pattern seen with the extracts from transfected cells. In the presence of the control extracts, the 12/23 substrate was cleaved such that the 262 and 571 bp products of coupled and regulated cleavage were found in the same ratios as obtained with extracts from RAG-transfected cells. In addition, the cryptic cleavage products seen with purified RAGs on the 12/23 substrate were suppressed (Fig. 2, D and E). Further, 12/12 and 23/23 cleavage by purified RAG1 and RAG2 was completely inhibited by the extracts of untransfected 293T cells (Fig. 2, D and E). No cleavage was observed with the 12/12 and 23/23 control substrates even upon prolonged exposure (data not shown). We conclude that untransfected 293T cell extracts contain an activity that complements purified RAGs and promotes stringent adherence to the 12/23 rule.

RAG1 and RAG2 Complemented with Whole Cell Extracts Show Strict 12/23 Regulation.

Experiments with purified recombinant RAG1 and RAG2 proteins have shown preferential, but incomplete 12/23-regulated cleavage by the isolated proteins (12). In contrast, extracts from lymphoid cells overexpressing RAG1 and RAG2 showed strict adherence to the 12/23 rule (10). To clarify the requirements for regulated cleavage, we performed cleavage reactions using crude extracts from cells expressing RAG1 and RAG2, purified RAG1 and RAG2, and combinations of extracts and purified proteins. Extracts containing RAG1 and RAG2 were prepared from 293T cells that were transiently transfected with plasmids that encode RAG1 and RAG2 GST fusion proteins. RAG1 and RAG2 GST fusion proteins were copurified directly from the same transfected 293T cell extracts, and control extracts were from 293T cells that did not express RAG1 or RAG2.

In a typical cleavage reaction we used a 1,162 bp DNA fragment from the deletion substrate pJH290 (Fig. 1) (8). This fragment contains a 12 and a 23 RSS and will be referred to as the 12/23 substrate. In all of the experiments, the 12/23 substrate was labeled at both ends with [32P]dNTPs. Upon regulated cleavage, three fragments are produced in equimolar amounts. The two end fragments are labeled (262 and 571 bp), whereas the third internal fragment (329 bp) is not labeled. In contrast with regulated cleavage, unregulated cleavage of the 12/23 substrate at the 12 RSS alone would generate two labeled fragments of 262 and 900 bp, whereas unregulated cleavage at the 23 RSS would produce two labeled fragments of 571 and 591, respectively (Fig. 1). Control substrates containing either 12/12, 23/23, and a single 23 RSS were produced in a similar manner (Fig. 1; Materials and Methods).

Whole cell extracts produced from RAG1- and RAG2transfected cells showed strict 12/23-restricted cleavage (Fig. 2, A and B; and reference 10). There was no detectable cleavage on either 12/12 or 23/23 substrates, and there were no unexpected cleavage products with the 12/ 23 substrate (Fig. 2, A and B). In contrast, RAGs purified from the same transfected 293T cell extracts showed only a modest preference for cleaving the 12/23 substrate (Fig. 2, C and E). In contrast with the RAG-containing extracts, the purified proteins were also active on 12/12 and 23/23 RSS substrates. Quantitation with a phosphoimager showed 50% and 30% of the level of cleavage on 12/12 and 23/23 substrates when compared with the 12/23 substrate (Fig. 2, C and E; data not shown). Consistent with the lack of specificity, we also found unexpected cryptic cleavage products on the 12/23 substrate with the purified proteins (Fig. 2
C, asterisks). We conclude that crude cellular extracts containing RAG1 and RAG2 strictly adhere to the 12/23 rule, whereas RAGs purified from the same extracts have only a two- to threefold preference for a 12/23 substrate.

To determine whether 12/23-regulated cleavage could be restored to the purified RAG1 and RAG2 proteins with factors found in whole cell extracts, we combined control extracts from untransfected 293T cells with purified RAG proteins. Addition of the untransfected cell extracts to purified RAGs had two prominent effects. First, there was a two- to threefold inhibition of the cleavage activity, resulting in a level of cleavage similar to that obtained with the RAG1/RAG2-transfected cell extracts. Second, the control extracts converted the preferential cleavage reaction seen with purified RAGs to the strictly regulated pattern seen with the extracts from transfected cells. In the presence of the control extracts, the 12/23 substrate was cleaved such that the 262 and 571 bp products of coupled and regulated cleavage were found in the same ratios as obtained with extracts from RAG-transfected cells. In addition, the cryptic cleavage products seen with purified RAGs on the 12/23 substrate were suppressed (Fig. 2, D and E). Further, 12/12 and 23/23 cleavage by purified RAG1 and RAG2 was completely inhibited by the extracts of untransfected 293T cells (Fig. 2, D and E). No cleavage was observed with the 12/12 and 23/23 control substrates even upon prolonged exposure (data not shown). We conclude that untransfected 293T cell extracts contain an activity that complements purified RAGs and promotes stringent adherence to the 12/23 rule.

HMG Box–containing Proteins and Prokaryotic Counterparts Enhance RAG-mediated Cleavage of 23 RSS-containing Substrates.DNA-bending proteins have been shown to be important components of a number of recombination reactions that require DNA cleavage and formation of multicomponent DNA–protein complexes (for reviews see references 19, 20). The role of the bending proteins in these reactions made them good candidates to participate in, and potentially regulate, RAG1- and RAG2-mediated DNA cleavage. To investigate the role of DNA-bending proteins on 12/23-regulated cleavage we performed cleavage reactions with purified RAG1 and RAG2 in the presence of four different DNA-bending proteins that are known to bend DNA and to play architectural roles: HMG1, rHMG1 (recombinant), HMG2, IHF, and HU. As indicated in Fig. 3, three of these proteins, HMG1, rHMG1, HMG2, and the prokaryotic HU enhanced the activity of RAG1 and RAG2 on the 12/23 substrate. The magnitude of the enhancement on the 12/23 substrate was three- to fourfold when quantitated. IHF differed from the other bending proteins in that it failed to enhance cleavage and was inhibitory for RAG-mediated cleavage of 12/23 and 12/12 substrates. Despite the enhanced cleavage by the purified RAGs in the presence of DNA-bending proteins there was no significant effect on the stringency of 12/23 regulation. A similar threefold increase in cleavage was also seen with a combination of purified RAG1 and RAG2 and either HMG1, rHMG1, HMG2, or HU on both 23/23 and 23i substrates (Fig. 3
C; data not shown). In contrast to 23RSS–containing substrates, there was no significant enhancement of cleavage by the DNA-bending proteins on a 12/12 substrate as seen in Fig. 3
B and confirmed by quantitation (data not shown). Thus, the addition of the DNAbending proteins to purified RAGs enhanced cleavage of DNA substrates that contained the 23 RSS, but they failed to induce regulated cleavage. Because DNA-bending proteins enhanced cleavage of 23 RSS containing substrates, whereas unfractionated cell extracts confer regulated cleavage, we asked whether we could both enhance and regulate cleavage by combining the two. We found that the extracts had a dominant effect decreasing the overall level of activity and increasing the specificity of the reaction (Fig. 4
A). There was complete inhibition of the 12/12, 23/23, and 23 substrate cleavage even when extract and rHMG1 were combined (Fig. 4, B–D).

HMG Box–containing Proteins and Prokaryotic Counterparts Enhance RAG-mediated Cleavage of 23 RSS-containing Substrates.

DNA-bending proteins have been shown to be important components of a number of recombination reactions that require DNA cleavage and formation of multicomponent DNA–protein complexes (for reviews see references 19, 20). The role of the bending proteins in these reactions made them good candidates to participate in, and potentially regulate, RAG1- and RAG2-mediated DNA cleavage. To investigate the role of DNA-bending proteins on 12/23-regulated cleavage we performed cleavage reactions with purified RAG1 and RAG2 in the presence of four different DNA-bending proteins that are known to bend DNA and to play architectural roles: HMG1, rHMG1 (recombinant), HMG2, IHF, and HU. As indicated in Fig. 3, three of these proteins, HMG1, rHMG1, HMG2, and the prokaryotic HU enhanced the activity of RAG1 and RAG2 on the 12/23 substrate. The magnitude of the enhancement on the 12/23 substrate was three- to fourfold when quantitated. IHF differed from the other bending proteins in that it failed to enhance cleavage and was inhibitory for RAG-mediated cleavage of 12/23 and 12/12 substrates. Despite the enhanced cleavage by the purified RAGs in the presence of DNA-bending proteins there was no significant effect on the stringency of 12/23 regulation. A similar threefold increase in cleavage was also seen with a combination of purified RAG1 and RAG2 and either HMG1, rHMG1, HMG2, or HU on both 23/23 and 23i substrates (Fig. 3
C; data not shown). In contrast to 23RSS–containing substrates, there was no significant enhancement of cleavage by the DNA-bending proteins on a 12/12 substrate as seen in Fig. 3
B and confirmed by quantitation (data not shown). Thus, the addition of the DNAbending proteins to purified RAGs enhanced cleavage of DNA substrates that contained the 23 RSS, but they failed to induce regulated cleavage.

Because DNA-bending proteins enhanced cleavage of 23 RSS containing substrates, whereas unfractionated cell extracts confer regulated cleavage, we asked whether we could both enhance and regulate cleavage by combining the two. We found that the extracts had a dominant effect decreasing the overall level of activity and increasing the specificity of the reaction (Fig. 4
A). There was complete inhibition of the 12/12, 23/23, and 23 substrate cleavage even when extract and rHMG1 were combined (Fig. 4, B–D).

DiscussionElegant in vivo and in vitro studies demonstrated that the 12/23 rule is established at the level of DNA cleavage (10–12). We have determined that the stringency of the 12/23 rule is enforced by cellular activities other than RAG1 and RAG2. The cofactor(s) that enhance the selectivity of the RAGs for substrates that contain both 12 and 23 RSSs are not lymphoid restricted, but are found in all cell extracts assayed (data not shown), including nonlymphoid 293T cells. These factors have yet to be characterized but are sensitive to heat and proteinase K. Finding that a component of nonlymphoid cell extracts enforces regulated cleavage resolves the apparent discrepancy between experiments with purified RAGs and those performed with extracts from cells expressing high levels of RAG1 and RAG2. Purified RAGs have only a modest two- or threefold preference for 12/23-containing substrates, but the discrimination is absolute when the same proteins are supplemented with extracts from cells that do not produce RAG1 and RAG2. A number of proteins other than RAG1 and RAG2 are known to be involved in V(D)J recombination, but none of these proteins has been suggested as required for 12/23-regulated cleavage. XRCC4, Ku-80, and the DNA-dependent protein kinase, all of which have a function in DNA double-strand break repair, are thought to be involved in the joining of signal and/or coding ends (1, 2, 18, 28, 29).Candidate factors that could be involved in assisting RAG-mediated DNA cleavage include DNA-bending proteins that are widely distributed, and have been shown to participate in site-specific recombination reactions (for review see references 19, 20). There are now many examples of eukaryotic and prokaryotic bending proteins that fulfill a number of essential architectural requirements during replication, transcription, and DNA recombination (30, 31). In these processes, they are thought to stabilize the multicomponent nucleoprotein complexes (32). HU, a prokaryotic DNA-bending protein, is particularly interesting in that it participates during assembly of bacteriophage Mu transposition intermediates, a process that may be distantly related to V(D)J recombination (17, 33). During in vitro Mu transposition HU loops the DNA between the L1 and L2 Mu A binding sites, thereby facilitating the communication between these sites during the assembly of a higher order transposition complex. HMG1, a widely distributed mammalian DNA-bending protein can substitute for HU in this reaction (34), and was thus a good candidate to assist RAGmediated DNA cleavage. Our results show that HMG1, HMG2, and HU all enhanced cleavage of a 12/23-RSS–containing substrate in vitro, but cleavage was not regulated by HMG1, because a similar increase in cleavage activity was also seen with control 23/23-RSS–and 23i-RSS–containing substrates. In contrast with 23-RSS–containing substrates, RAG-mediated cleavage of a 12/12 substrate was not affected by DNA bending proteins, suggesting that the increased cleavage at the 12/23 substrate may reflect a specific effect on the RAG1 and RAG2 complex at the 23 RSS. Modification of the complex formed at the 23 RSS may be sufficient to have a dramatic effect in coordinated 12/23 cleavage when these two different RSS are forming a synaptic complex. DNA-bending proteins may enhance cleavage at the 23 RSS in a number of different ways, including direct architectural effects on the DNA bringing one or more cis elements into closer proximity, or through more complex interactions that directly alter the binding of RAGs to DNA. IHF differed from the other bending proteins in that it was inhibitory for the 12/23 as well as the 12/12 substrate. This inhibition by IHF remains to be explained fully but may be due to a documented preference of this bending protein for specific DNA sequences (reference 35 and references therein). Thus, IHF binding to specific sequences in the substrate might interfere directly with binding of RAGs to the RSS.Although the effects of DNA bending proteins on in vitro cleavage by RAGs are readily demonstrable, the role of these proteins in V(D)J recombination in vivo remains to be elucidated. However, evaluating the role of these proteins in vivo may be a daunting task, because the bending proteins are a large family that is found in all cell types, and members of the family display functional redundancy.DNA-bending proteins and cell extracts have distinct effects on the cleavage reaction. Extracts have an overall inhibitory effect on cleavage and specifically eliminate cleavage activity on isolated 23 RSS, 23/23 RSS, and 12/12 RSS substrate. In contrast, DNA-bending proteins enhance cleavage on all 23-RSS–containing substrates and have no effect on 12/12 substrates. When DNA-bending proteins and the extracts were combined, the extracts were dominant, with cleavage remaining regulated and with no appreciable enhancement of cleavage by the DNA-bending proteins. One way to explain this result is that the high levels of HMGs already present in cellular extracts may be saturating, in which case further addition of DNA-bending proteins would be ineffective. Alternatively, there may be an inhibitory factor(s) that interfere with the effects of the DNA-bending proteins. Complete purification of the 12/23 coregulatory activity will be essential in order to reconstitute 12/23-regulated cleavage in a system where the effect of RAG1, RAG2, DNA-bending proteins, and other factors can be completely analyzed.

Elegant in vivo and in vitro studies demonstrated that the 12/23 rule is established at the level of DNA cleavage (10–12). We have determined that the stringency of the 12/23 rule is enforced by cellular activities other than RAG1 and RAG2. The cofactor(s) that enhance the selectivity of the RAGs for substrates that contain both 12 and 23 RSSs are not lymphoid restricted, but are found in all cell extracts assayed (data not shown), including nonlymphoid 293T cells. These factors have yet to be characterized but are sensitive to heat and proteinase K. Finding that a component of nonlymphoid cell extracts enforces regulated cleavage resolves the apparent discrepancy between experiments with purified RAGs and those performed with extracts from cells expressing high levels of RAG1 and RAG2. Purified RAGs have only a modest two- or threefold preference for 12/23-containing substrates, but the discrimination is absolute when the same proteins are supplemented with extracts from cells that do not produce RAG1 and RAG2. A number of proteins other than RAG1 and RAG2 are known to be involved in V(D)J recombination, but none of these proteins has been suggested as required for 12/23-regulated cleavage. XRCC4, Ku-80, and the DNA-dependent protein kinase, all of which have a function in DNA double-strand break repair, are thought to be involved in the joining of signal and/or coding ends (1, 2, 18, 28, 29).

Candidate factors that could be involved in assisting RAG-mediated DNA cleavage include DNA-bending proteins that are widely distributed, and have been shown to participate in site-specific recombination reactions (for review see references 19, 20). There are now many examples of eukaryotic and prokaryotic bending proteins that fulfill a number of essential architectural requirements during replication, transcription, and DNA recombination (30, 31). In these processes, they are thought to stabilize the multicomponent nucleoprotein complexes (32). HU, a prokaryotic DNA-bending protein, is particularly interesting in that it participates during assembly of bacteriophage Mu transposition intermediates, a process that may be distantly related to V(D)J recombination (17, 33). During in vitro Mu transposition HU loops the DNA between the L1 and L2 Mu A binding sites, thereby facilitating the communication between these sites during the assembly of a higher order transposition complex. HMG1, a widely distributed mammalian DNA-bending protein can substitute for HU in this reaction (34), and was thus a good candidate to assist RAGmediated DNA cleavage. Our results show that HMG1, HMG2, and HU all enhanced cleavage of a 12/23-RSS–containing substrate in vitro, but cleavage was not regulated by HMG1, because a similar increase in cleavage activity was also seen with control 23/23-RSS–and 23i-RSS–containing substrates. In contrast with 23-RSS–containing substrates, RAG-mediated cleavage of a 12/12 substrate was not affected by DNA bending proteins, suggesting that the increased cleavage at the 12/23 substrate may reflect a specific effect on the RAG1 and RAG2 complex at the 23 RSS. Modification of the complex formed at the 23 RSS may be sufficient to have a dramatic effect in coordinated 12/23 cleavage when these two different RSS are forming a synaptic complex. DNA-bending proteins may enhance cleavage at the 23 RSS in a number of different ways, including direct architectural effects on the DNA bringing one or more cis elements into closer proximity, or through more complex interactions that directly alter the binding of RAGs to DNA. IHF differed from the other bending proteins in that it was inhibitory for the 12/23 as well as the 12/12 substrate. This inhibition by IHF remains to be explained fully but may be due to a documented preference of this bending protein for specific DNA sequences (reference 35 and references therein). Thus, IHF binding to specific sequences in the substrate might interfere directly with binding of RAGs to the RSS.

Although the effects of DNA bending proteins on in vitro cleavage by RAGs are readily demonstrable, the role of these proteins in V(D)J recombination in vivo remains to be elucidated. However, evaluating the role of these proteins in vivo may be a daunting task, because the bending proteins are a large family that is found in all cell types, and members of the family display functional redundancy.

DNA-bending proteins and cell extracts have distinct effects on the cleavage reaction. Extracts have an overall inhibitory effect on cleavage and specifically eliminate cleavage activity on isolated 23 RSS, 23/23 RSS, and 12/12 RSS substrate. In contrast, DNA-bending proteins enhance cleavage on all 23-RSS–containing substrates and have no effect on 12/12 substrates. When DNA-bending proteins and the extracts were combined, the extracts were dominant, with cleavage remaining regulated and with no appreciable enhancement of cleavage by the DNA-bending proteins. One way to explain this result is that the high levels of HMGs already present in cellular extracts may be saturating, in which case further addition of DNA-bending proteins would be ineffective. Alternatively, there may be an inhibitory factor(s) that interfere with the effects of the DNA-bending proteins. Complete purification of the 12/23 coregulatory activity will be essential in order to reconstitute 12/23-regulated cleavage in a system where the effect of RAG1, RAG2, DNA-bending proteins, and other factors can be completely analyzed.
