---
pmid: '25851601'
title: The RhoGEF DOCK10 is essential for dendritic spine morphogenesis.
authors:
- Jaudon F
- Raynaud F
- Wehrlé R
- Bellanger JM
- Doulazmi M
- Vodjdani G
- Gasman S
- Fagni L
- Dusart I
- Debant A
- Schmidt S
journal: Mol Biol Cell
year: '2015'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC4472020
doi: 10.1091/mbc.E14-08-1310
---

# The RhoGEF DOCK10 is essential for dendritic spine morphogenesis.
**Authors:** Jaudon F, Raynaud F, Wehrlé R, Bellanger JM, Doulazmi M, Vodjdani G, Gasman S, Fagni L, Dusart I, Debant A, Schmidt S
**Journal:** Mol Biol Cell (2015)
**DOI:** [10.1091/mbc.E14-08-1310](https://doi.org/10.1091/mbc.E14-08-1310)
**PMC:** [PMC4472020](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4472020/)

## Abstract

1. Mol Biol Cell. 2015 Jun 1;26(11):2112-27. doi: 10.1091/mbc.E14-08-1310. Epub 
2015 Apr 7.

The RhoGEF DOCK10 is essential for dendritic spine morphogenesis.

Jaudon F(1), Raynaud F(2), Wehrlé R(3), Bellanger JM(1), Doulazmi M(3), Vodjdani 
G(4), Gasman S(5), Fagni L(2), Dusart I(3), Debant A(6), Schmidt S(6).

Author information:
(1)Centre de Recherche en Biochimie Macromoléculaire, CNRS-UMR 5237, Université 
de Montpellier, 34293 Montpellier, France.
(2)Institute of Functional Genomics, CNRS-UMR 5203, INSERM U661, Université de 
Montpellier, 34094 Montpellier, France.
(3)Université Pierre et Marie Curie, CNRS-UMR 7102, Université Paris 06, 75005 
Paris, France.
(4)PROTECT, Neuroprotection du cerveau en développement, UMR1141-INSERM, 
Université Paris-Diderot, Sorbonne Paris-Cité, 75019 Paris, France.
(5)Institut des Neurosciences Cellulaires et Intégratives, CNRS-UPR 3212, Centre 
de Neurochimie, Université de Strasbourg, 67084 Strasbourg, France.
(6)Centre de Recherche en Biochimie Macromoléculaire, CNRS-UMR 5237, Université 
de Montpellier, 34293 Montpellier, France susanne.schmidt@crbm.cnrs.fr 
anne.debant@crbm.cnrs.fr.

By regulating actin cytoskeleton dynamics, Rho GTPases and their activators 
RhoGEFs are implicated in various aspects of neuronal differentiation, including 
dendritogenesis and synaptogenesis. Purkinje cells (PCs) of the cerebellum, by 
developing spectacular dendrites covered with spines, represent an attractive 
model system in which to decipher the molecular signaling underlying these 
processes. To identify novel regulators of dendritic spine morphogenesis among 
members of the poorly characterized DOCK family of RhoGEFs, we performed gene 
expression profiling of fluorescence-activated cell sorting (FACS)-purified 
murine PCs at various stages of their postnatal differentiation. We found a 
strong increase in the expression of the Cdc42-specific GEF DOCK10. Depleting 
DOCK10 in organotypic cerebellar cultures resulted in dramatic dendritic spine 
defects in PCs. Accordingly, in mouse hippocampal neurons, depletion of DOCK10 
or expression of a DOCK10 GEF-dead mutant led to a strong decrease in spine 
density and size. Conversely, overexpression of DOCK10 led to increased spine 
formation. We show that DOCK10 function in spinogenesis is mediated mainly by 
Cdc42 and its downstream effectors N-WASP and PAK3, although DOCK10 is also able 
to activate Rac1. Our global approach thus identifies an unprecedented function 
for DOCK10 as a novel regulator of dendritic spine morphogenesis via a 
Cdc42-mediated pathway.

© 2015 Jaudon et al. This article is distributed by The American Society for 
Cell Biology under license from the author(s). Two months after publication it 
is available to the public under an Attribution–Noncommercial–Share Alike 3.0 
Unported Creative Commons License 
(http://creativecommons.org/licenses/by-nc-sa/3.0).

DOI: 10.1091/mbc.E14-08-1310
PMCID: PMC4472020
PMID: 25851601 [Indexed for MEDLINE]

## Full Text

INTRODUCTIONRho-family GTPases are potent determinants of cell shape that regulate actin cytoskeleton and microtubule dynamics, membrane dynamics, and vesicular trafficking (Etienne-Manneville and Hall, 2002). They require precise spatiotemporal activation in order to execute their functions. This is in part achieved by their main regulators, the Rho guanine nucleotide exchange factors (GEFs) and the Rho GTPase-activating proteins (GAPs), which stimulate GDP-to-GTP exchange and GTP hydrolysis, respectively. RhoGEFs belong to two distinct classes of proteins: the Dbl family and the evolutionary distinct family of Dedicator of cytokinesis (DOCK) proteins (Schmidt and Hall, 2002; Cote and Vuori, 2007). In mammals, the 11 DOCK proteins activate Rac1 or Cdc42 through their catalytic DOCK-homology-region-2 (DHR-2) domain (Cote and Vuori, 2007). Based on sequence similarity, they have been grouped into four subfamilies. The DOCK-A and DOCK-B subfamilies contain Rac-specific GEFs, the DOCK-C subfamily comprises dual-specificity Rac- and Cdc42-GEFs (Pakes et al., 2013), and the DOCK-D subfamily members, also referred to as zizimins (DOCK9/zizimin1, DOCK10/zizimin3, and DOCK11/zizimin2), have been reported to act as Cdc42-specific GEFs (Nishikimi et al., 2005; Lin et al., 2006).As central to the regulation of cytoskeleton dynamics, Rho GTPase signaling contributes to various cellular processes, including different steps of neuronal differentiation, such as the morphogenesis of dendrites and spines (Govek et al., 2005). Development and maturation of dendrites are complex, multistep processes that must be finely tuned. This has been well described in the Purkinje cells (PCs) of the cerebellum, which have a very elaborate dendritic tree and therefore represent an attractive model system in which to study this process (Urbanska et al., 2008; Sotelo and Dusart, 2009; Tanaka, 2009). The development of PC dendrites follows sequential stages to finally reach, at the end of the third postnatal week in mice, a mature dendritic tree covered with spines. Dendritic spines are small protrusions that constitute the sites of synaptic contacts, leading to the establishment of neuronal circuits (Yuste and Bonhoeffer, 2004). They are highly dynamic actin-rich structures, which undergo continuous remodeling upon sensory and emotional stimuli, such as those arising during learning and memory. Their dynamic assembly is therefore essential for normal brain function. In hippocampal neurons, the GTPases Rac1 and Cdc42 promote formation and maintenance of dendritic spines, whereas RhoA induces their retraction and loss (Tada and Sheng, 2006). Signaling from these GTPases to the actin cytoskeleton is then mediated by specific downstream effectors, such as the neural Wiskott–Aldrich syndrome protein (N-WASP), p21-activated kinases (PAKs), Rho kinase, and mDia (Newey et al., 2005; Kasri and van Aelst, 2008). The physiological importance of this signaling in dendritic spine morphogenesis is further highlighted by the fact that mutations in several genes of Rho pathway components, like the GAP oligophrenin-1, the GEF α-PIX, and the kinase PAK3, are associated with X-linked mental retardation, characterized by an immature morphology of spines (Govek et al., 2005; Newey et al., 2005).A clear picture of the RhoGEFs activating Rho-family GTPases during spinogenesis is lacking. In this study, we sought novel regulators of dendritogenesis and/or spinogenesis among members of the poorly characterized DOCK family of RhoGEFs via gene expression profiling of fluorescence-activated cell sorting (FACS)–purified PCs. We identified the RhoGEF DOCK10 as being essential for dendritic spine morphogenesis both in PCs and in hippocampal neurons via a Cdc42-mediated pathway.

INTRODUCTION

Rho-family GTPases are potent determinants of cell shape that regulate actin cytoskeleton and microtubule dynamics, membrane dynamics, and vesicular trafficking (Etienne-Manneville and Hall, 2002). They require precise spatiotemporal activation in order to execute their functions. This is in part achieved by their main regulators, the Rho guanine nucleotide exchange factors (GEFs) and the Rho GTPase-activating proteins (GAPs), which stimulate GDP-to-GTP exchange and GTP hydrolysis, respectively. RhoGEFs belong to two distinct classes of proteins: the Dbl family and the evolutionary distinct family of Dedicator of cytokinesis (DOCK) proteins (Schmidt and Hall, 2002; Cote and Vuori, 2007). In mammals, the 11 DOCK proteins activate Rac1 or Cdc42 through their catalytic DOCK-homology-region-2 (DHR-2) domain (Cote and Vuori, 2007). Based on sequence similarity, they have been grouped into four subfamilies. The DOCK-A and DOCK-B subfamilies contain Rac-specific GEFs, the DOCK-C subfamily comprises dual-specificity Rac- and Cdc42-GEFs (Pakes et al., 2013), and the DOCK-D subfamily members, also referred to as zizimins (DOCK9/zizimin1, DOCK10/zizimin3, and DOCK11/zizimin2), have been reported to act as Cdc42-specific GEFs (Nishikimi et al., 2005; Lin et al., 2006).

As central to the regulation of cytoskeleton dynamics, Rho GTPase signaling contributes to various cellular processes, including different steps of neuronal differentiation, such as the morphogenesis of dendrites and spines (Govek et al., 2005). Development and maturation of dendrites are complex, multistep processes that must be finely tuned. This has been well described in the Purkinje cells (PCs) of the cerebellum, which have a very elaborate dendritic tree and therefore represent an attractive model system in which to study this process (Urbanska et al., 2008; Sotelo and Dusart, 2009; Tanaka, 2009). The development of PC dendrites follows sequential stages to finally reach, at the end of the third postnatal week in mice, a mature dendritic tree covered with spines. Dendritic spines are small protrusions that constitute the sites of synaptic contacts, leading to the establishment of neuronal circuits (Yuste and Bonhoeffer, 2004). They are highly dynamic actin-rich structures, which undergo continuous remodeling upon sensory and emotional stimuli, such as those arising during learning and memory. Their dynamic assembly is therefore essential for normal brain function. In hippocampal neurons, the GTPases Rac1 and Cdc42 promote formation and maintenance of dendritic spines, whereas RhoA induces their retraction and loss (Tada and Sheng, 2006). Signaling from these GTPases to the actin cytoskeleton is then mediated by specific downstream effectors, such as the neural Wiskott–Aldrich syndrome protein (N-WASP), p21-activated kinases (PAKs), Rho kinase, and mDia (Newey et al., 2005; Kasri and van Aelst, 2008). The physiological importance of this signaling in dendritic spine morphogenesis is further highlighted by the fact that mutations in several genes of Rho pathway components, like the GAP oligophrenin-1, the GEF α-PIX, and the kinase PAK3, are associated with X-linked mental retardation, characterized by an immature morphology of spines (Govek et al., 2005; Newey et al., 2005).

A clear picture of the RhoGEFs activating Rho-family GTPases during spinogenesis is lacking. In this study, we sought novel regulators of dendritogenesis and/or spinogenesis among members of the poorly characterized DOCK family of RhoGEFs via gene expression profiling of fluorescence-activated cell sorting (FACS)–purified PCs. We identified the RhoGEF DOCK10 as being essential for dendritic spine morphogenesis both in PCs and in hippocampal neurons via a Cdc42-mediated pathway.

RESULTSGene expression profiling during postnatal Purkinje neuron developmentTo identify novel RhoGEFs involved in dendritic tree morphogenesis, we undertook gene expression profiling of all DOCK-family RhoGEFs in Purkinje cells. Because these neurons represent only 2–3% of the whole cerebellum, we took advantage of a mouse strain carrying the green fluorescent protein (GFP) under the Purkinje-specific promoter Pcp2 to purify the PCs by FACS (Tomomura et al., 2001). Neurons were isolated from cerebella of postnatal day 3 (P3), P7, P15, and P20 Pcp2-GFP mice, corresponding to the time frame of development of the Purkinje cell dendritic tree. Purity of the Purkinje cell samples was assessed both by visual counting of the calbindin-positive cells (calbindin is a Purkinje cell–specific marker) before and after sorting (Figure 1, A-C) and by real-time quantitative PCR (RT-qPCR) of marker genes specific for the Purkinje cells (calbindin) and for the other main cerebellar cell types, namely granule cells (NeuroD1), astrocytes (glial fibrillary acidic protein [GFAP]), and interneurons (Tcfap2a; Figure 1D). After sorting, Purkinje cells represented ∼78% of the cell population.FIGURE 1:Gene expression profiling of all DOCK RhoGEFs in purified postnatal Purkinje neurons. (A, B) Sorting of GFP-expressing postnatal Purkinje neurons by FACS. Dissociated cells from cerebella of Pcp2-GFP mice at P3, P7, P15, and P20 were plated before (A) and after (B) sorting and fixed. Nuclei were stained with Hoechst stain to visualize the total cell number in the sample, and Purkinje neurons were revealed with a calbindin antibody (red) and by direct fluorescence of the GFP (green). Micrographs show representative sorting results of cerebellar cells from mice at P7. (C) Histogram representing the enrichment of Purkinje neurons within the cell population upon sorting, expressed as relative percentages. Data are expressed as mean enrichment value of all age groups combined ± SD of at least three independent experiments. **p < 0.01 (Student's t test). (D) RT-qPCR performed before and after FACS on the mRNA of specific marker genes of the main cerebellar cell types: calbindin (Purkinje cells), NeuroD1 (cerebellar granule neurons), GFAP (astrocytes), and Tcfap2a (interneurons). Data are expressed as mean ± SD of at least three experiments performed on P7 Pcp2-GFP mice in the example shown. **p < 0.01 and ***p < 0.001 (Student's t test). (E) RT-qPCR performed on purified PC mRNAs of all 11 mammalian DOCK-family RhoGEFs. Data are expressed as mean ± SD of at least three experiments. *p < 0.05, **p < 0.01, and ***p < 0.001 (Student's t test).We then performed RT-qPCR on mRNAs isolated from these cells and analyzed the expression pattern of all 11 members of the mammalian DOCK family of RhoGEFs in the course of development (Figure 1E; see Supplemental Table S1 for primer sequences). The mRNAs of DOCK180/DOCK1, DOCK2, DOCK3, and DOCK8 could not be detected due to too low expression levels. DOCK4, DOCK5, DOCK6, and DOCK7 were expressed, but their mRNA levels did not vary significantly across the four developmental stages we analyzed, except for DOCK4 and DOCK6, whose expression dropped at P20. In contrast, the three DOCK-D subfamily members—DOCK9, DOCK10, and DOCK11—showed a significant increase in their mRNA expression levels, reaching a peak at P15 (DOCK11) or P20 (DOCK9 and DOCK10; Figure 1E). We identified DOCK10 as a promising candidate since its robust increase in expression occurred precisely during the time frame of Purkinje cell spinogenesis during the second and third postnatal weeks and no neuronal function had yet been ascribed to it.DOCK10 is expressed in Purkinje cells during postnatal developmentTo confirm the RT-qPCR data and address the expression of the DOCK10 protein in the postnatal cerebellum, we analyzed the expression pattern of DOCK10 in postnatal mouse brain slices by immunohistochemistry (IHC) using a DOCK10-specific antibody (Figure 2). In whole sections of the cerebellum (P15), the DOCK10 protein was detected in the monolayer of Purkinje cells, nicely overlapping with the Purkinje-specific calbindin staining (Figure 2A). Higher-magnification images showed that at P7, and more strikingly at P15 and later stages, DOCK10 was detected in the soma and the dendritic tree of PCs, although it was only weakly detectable in the latter (Figure 2, A, inset, and B). At P7, DOCK10 immunostaining could also be seen in the external and internal granule cell layer (IGL), but at P15 and P21, IGL staining disappeared and DOCK10 localization seemed to be restricted to the Purkinje cell layer (Figure 2B).FIGURE 2:DOCK10 expression in cerebellar Purkinje cells at postnatal stages. (A) Immunohistochemistry performed on wild-type C57BL/6 mouse cerebellar sections at P15, using a DOCK10 antibody (red) and a calbindin antibody to label the PCs (green). Nuclei were stained with Hoechst stain (blue) to visualize all cells in the slice. Owing to the high density of granule cells, the latter staining highlights mainly the internal granule cell layer. Bottom right, merged image. Note the monolayer-specific staining of PCs, revealed with both DOCK10 antibody and calbindin antibody. These antibodies also label the Purkinje cell dendritic tree in the molecular layer. Inset, higher-magnification image of the boxed region, showing the DOCK10 immunostaining. IGL, internal granule cell layer; ML, molecular layer. PCL, Purkinje cell layer; Scale bar, 250 μm. (B) Immunohistochemistry performed on P7, P15, and P21 C57BL/6 mouse brain sections, using a DOCK10 and a calbindin antibody. Nuclei were stained with Hoechst to visualize all cells in the slice. The various cell layers found in the section are indicated as in A; EGL, external granule cell layer. Note the Purkinje cell–specific staining of DOCK10 at P15 and P21 in the soma and to a lesser extent in the dendritic tree (see also inset in A). Scale bar, 25 μm.Taken together, these data show that, at early postnatal stages, DOCK10 is expressed broadly in the cerebellum and that from P15 on, its expression becomes restricted to the Purkinje cells.Depletion of DOCK10 in Purkinje cells leads to dendritic spine defectsGiven the expression of DOCK10 in differentiating Purkinje neurons, we investigated the function of DOCK10 in these cells by analyzing the effect of knocking down its expression. To do so, we designed and validated two different short hairpin RNAs (shRNAs) targeting DOCK10 (shA and shB) and a control shRNA (shCtrl; see Materials and Methods). Both shRNAs targeting DOCK10 efficiently knocked down endogenous DOCK10 expression (Supplemental Figure S1A) and were ineffective on the closely related DOCK9 and DOCK11 RhoGEFs (Supplemental Figure S1B). We then transduced mouse cerebellar organotypic cultures at P0 with lentiviral vectors expressing these shRNAs and also carrying GFP as a transduction marker. In these organotypic slices, the dendritic development of Purkinje cells is almost similar to that observed in vivo, although in a slightly delayed manner (Boukhtouche et al., 2006; Poulain et al., 2008). Slices were grown until 14 days in vitro (DIV) and, after fixation, PC morphology was analyzed by immunofluorescence staining with a calbindin antibody. On the same slice, transduced (calbindin+/GFP+) versus nontransduced (calbindin+/GFP–) Purkinje cells could be visualized (Figure 3A), allowing for an accurate comparison of the cell morphology, whatever the transduced state. A blinded analysis was then conducted in order to analyze whether the knockdown of DOCK10 affected the general dendritic morphology of Purkinje cells (see Materials and Methods). To do so, all of the Purkinje cells were distributed into three groups, depending on their dendritic morphology as observed by calbindin immunostaining (Figure 3B): PCs with an atrophic morphology (the longest dendrite being smaller than the cell body diameter), PCs with a stellate morphology (harboring more than six perisomatic processes), and PCs with an elaborate dendritic tree.FIGURE 3:Depletion of DOCK10 in Purkinje cells leads to dendritic spine defects. (A) Micrographs of organotypic cultures that received Lv-ShA DOCK10. The arrowhead represents a transduced Purkinje cell (GFP+/calbindin+), whereas the arrow represents a nontransduced Purkinje cell (GFP–/Calbindin+). Scale bar, 18.75 μm. (B) Micrographs illustrating typical calbindin-labeled PCs observed in organotypic cultures after 14 DIV: atrophic, stellate, and elaborate PCs. (C) Histograms representing the distribution of Purkinje cell types 14 DIV after transduction with lentiviral vectors expressing the GFP marker together with either a control shRNA or the two DOCK10 shRNAs shA and shB. Data are presented as relative percentage for each stage in nontransduced (NT) and transduced (T) PCs. For statistical analysis, the distributions of Purkinje cell types were compared using Pearson's χ2 test or Fisher's exact test, depending on the number of PCs per group (nonsignificant differences were detected, p > 0.5). (D) Micrographs of calbindin-labeled Purkinje cells within organotypic cultures after 14 DIV and lentiviral infection. D1 is a nontransduced Purkinje cell, whereas D2 and D3 are Purkinje cells transduced with shA DOCK10 and shB DOCK10, respectively. Arrowheads point to a dendritic area covered with spines; arrow, to a naked dendritic area (devoid of spines). Scale bar, 18.75 μm. (E) Histograms representing the distribution of Purkinje cells distinguishing cells with at least one part of the dendritic tree covered with spines (spines) and cells with a dendritic tree devoid of spines (no spines) after transduction by lentiviral vectors as described in C. Data are presented as relative percentage for each group. Compared are nontransduced Purkinje cells (NT) and transduced PCs (T). For statistical analysis, the distributions of Purkinje cell types were compared using Pearson's χ2 test or Fisher's exact test, depending on the number of PCs per group (ns, nonsignificant, **p < 0.005, ***p < 0.001). (F) Sholl analysis of PCs with an elaborate dendritic tree in the different experimental groups. Sholl analysis was performed on cells with at least one part of the dendritic tree covered with spines (spines) and cells with a dendritic tree devoid of spines (no spines) in the different transduced conditions (shCtlr, shA DOCK10, and shB DOCK10).No statistically significant difference could be detected in the distribution within these three groups of PCs, whether transduced or not with shA or shB DOCK10 or shCtrl lentivectors (Figure 3C), suggesting that knockdown of DOCK10 expression does not have a major effect on overall Purkinje cell dendritic development.More careful analysis of the elaborate Purkinje cells, however, revealed two types of cells: cells presenting at least one part of the dendritic tree covered with spines (Figure 3D, D1) and cells with a dendritic tree totally devoid of spines (Figure 3D, D2 and D3). Nearly all of the nontransduced PCs and the PCs transduced with ShCtrl presented an elaborate dendritic tree with spines (Figure 3, D and E). In contrast, 53 and 23% of Purkinje cells transduced with shA-DOCK10 and shB-DOCK10, respectively, presented a dendritic tree completely devoid of spines (Figure 3, D and E). This defect was observed with both shRNAs, albeit to a lesser extent with Lv-ShB, excluding an off-target effect.To determine whether depletion of DOCK10 acted only on the formation of spines or the observed phenotype was an indirect consequence of defective dendritic tree morphogenesis, we performed a Sholl analysis on these elaborate Purkinje cells to measure the complexity and height of the dendritic tree (see Materials and Methods; Figure 3F). The general distribution of the Sholl index was the same whether the cells were transduced with shCtrl or shA- or shB-DOCK10. The large majority of the branching points were observed between 20 and 40 μm from the soma (Figure 3F). Although the Sholl index at 30 μm from the soma varied between the different lentivector-transduced conditions (from 4.6 to 7.7), these differences were not correlated with the presence or absence of spines. Therefore this analysis revealed no significant effect of any shRNA on the complexity or height of the dendritic tree of elaborate PCs, showing a direct role for DOCK10 in dendritic spine morphogenesis.DOCK10 is required for the formation of dendritic spines in hippocampal neuronsTo better characterize the function of DOCK10 in dendritic spine morphogenesis, we turned to primary cultures of hippocampal neurons, which are a classical model with which to study spine morphogenesis and are more tractable to culture and transfection than Purkinje cells. We first verified that DOCK10 was expressed in the postnatal mouse hippocampus. Indeed, Western blot analysis performed on lysates of different P26 mouse brain areas and on lysates of cultured hippocampal neurons (at 18 DIV) revealed the expression of DOCK10 in the hippocampus but also in the olfactory bulb and the cortex (Figure 4A). The hippocampal expression of DOCK10 was also confirmed by IHC analysis on P15 mouse brain sections, using a DOCK10 antibody (Figure 4C). This revealed that DOCK10 was present in the pyramidal cells of the CA1 and CA3 regions, as well as in the adjacent subiculum and the dentate gyrus.FIGURE 4:DOCK10 is expressed in the hippocampus and localized in dendritic spines in hippocampal neurons. (A) Protein extracts from various P26 mouse brain regions and from cultured hippocampal neurons at 18 DIV were analyzed by Western blot, using a DOCK10 antibody. Tubulin was used as a loading control. (B) Subcellular fractions obtained after fractionation of P27 mouse brain extracts on a sucrose gradient were analyzed by Western blot, using a DOCK10, a PSD-95, and a Cdc42 antibody. Tubulin was used as a loading control. S1, soluble cytoplasmic fraction; P1, vesicular fraction; P2, synaptosomal fraction; P3, membrane fraction. (C) Immunohistochemistry performed on stage P15 mouse hippocampus sections. Immunostaining was performed using a DOCK10 antibody (red). Nuclei were stained with Hoechst stain (blue) to visualize all cells in the slice. Owing to the high density of cells, the latter staining highlights mainly the dentate gyrus. Right, merged image. Underneath each image are higher-magnification images of the region selected (CA1/subiculum region). Scale bar, 40 μm. (D) DOCK10 is localized in dendrites and spines of hippocampal neurons. Micrographs of cultured hippocampal neurons cotransfected with Flag-DOCK10 (detected with an anti-Flag antibody; red) and GFP to visualize the dendrites (green). Right, merged images. Bottom, higher- magnification images of the region selected. Scale bar, 1 μm. Arrows, examples of spine heads labeled by DOCK10. (E) DOCK10 postsynaptic localization is adjacent to presynaptic terminals. Micrographs of hippocampal neurons cotransfected with Flag-DOCK10 (detected with an anti-Flag antibody; red; top right) and GFP to visualize the dendrites (green; top left). Immunostaining was performed using an anti-Bassoon antibody (blue, bottom left), which marks the presynaptic terminals. Bottom right, merged image. Insets, higher-magnification images of a selected region to better visualize the adjacent localization of DOCK10 and Bassoon (arrows).We then studied the specific subcellular localization of endogenous DOCK10 by performing fractionation experiments on a sucrose gradient. This showed that DOCK10 was present in the synaptosomal fraction, where it fractionated together with the postsynaptic density protein PSD-95 (Figure 4B), indicating a synaptic localization for DOCK10. Both proteins were also detected in the membrane fraction. Of interest, Cdc42, the target GTPase of DOCK10, was also detected in these two fractions (Figure 4B). We further confirmed the synaptosomal localization of DOCK10 by immunofluorescence studies by transfecting flag-DOCK10 into cultured hippocampal neurons. Figure 4D shows that DOCK10 was localized in the dendritic shaft and most strikingly in the dendritic spines in hippocampal neurons, which is compatible with a function of DOCK10 in dendritic spine morphogenesis. In addition, DOCK10 immunostaining in the spines localized adjacent to the presynaptic protein Bassoon (Figure 4E). Combined, these data show that DOCK10 is present in the postsynaptic density.We next analyzed the consequences of DOCK10 depletion on spine formation in hippocampal neurons by transfecting them at 8 DIV (before the initial appearance of the spines) and fixing them 3 d later. ShRNA-mediated depletion of DOCK10 led to a dramatic decrease in the number of spines along the dendritic shaft (Figure 5, A and B). This defect was accompanied by a decrease in spine head size in the remaining spines (Figure 5C). In addition, the postsynaptic density protein Homer-1 was hardly detected by immunofluorescence in the remaining protrusions of DOCK10-depleted neurons as compared with control neurons, confirming that spines devoid of DOCK10 are not properly formed (Figure 5, D and E). To confirm the requirement of DOCK10 for spine formation, we performed rescue experiments by reintroducing a shRNA-resistant DOCK10 construct into DOCK10-depleted hippocampal neurons. This construct was designed to be resistant only to shA DOCK10 and not to shB, as demonstrated in Supplemental Figure S2A (see Materials and Methods). As shown in Figure 5, F and G, wild-type (wt) DOCK10shR was able to rescue the decrease in both spine number and spine head size induced by shA-mediated DOCK10 depletion, demonstrating its requirement for spine morphogenesis.FIGURE 5:The GEF activity of DOCK10 is required for the formation of dendritic spines in hippocampal neurons. (A) Depletion of DOCK10 leads to decreased spine density and spine head size. Representative micrographs of hippocampal neurons transfected with a control shRNA (shCtrl) or two different DOCK10 shRNAs (shA and shB). Neurons were transfected at 8 DIV and fixed at 11 DIV (experimental conditions to reveal spine formation). Scale bar, 1 μm. (B, C) Quantification of dendritic spine density (B) and size (C) of hippocampal neurons transfected as described in A, measured by the number of spines/10 μm and the spine head area, respectively. Error bars indicate SEM (n = 3 independent experiments); **p < 0.01 (Student's t test). (D) Representative micrographs of hippocampal neurons transfected with shA DOCK10 or shCtrl (green) and immunostained for postsynaptic protein Homer-1 (red; middle micrographs). Right, merged images. Scale bar, 1 μm. (E) Quantification of the number of protrusions positive for Homer-1 staining in DOCK10-depleted or control neurons, as described in D. Error bars indicate SEM (n = 4 independent experiments); **p = 0.001 (Student's t test). (F, G) A functional GEF activity of DOCK10 is required for spine formation in hippocampal neurons. Quantification of spine density (F) and size (G) of hippocampal neurons cotransfected with shA-DOCK10 and either a control vector or shA-resistant wt DOCK10shR or shA-resistant GEF-dead DOCK10 (DOCK10 GDshR). Error bars indicate SEM (n = 3 independent experiments), *p < 0.05, #p < 0.05 (Student's t test).To determine whether the dependence on DOCK10 for spine formation required its GEF activity, we created a catalytically inactive mutant based on a similar mutant in DOCK9 (Yang et al., 2009). In this mutant, the catalytic DHR2 domain harbors a point mutation abolishing its GEF activity, as assessed by in vitro GEF assays on Cdc42 (Supplemental Figure S2B). Of interest, this GEF-dead mutant DOCK10 (DOCK10shR GD) was unable to rescue the spine defects induced by DOCK10 depletion (Figure 5, F and G), indicating that the GEF activity of DOCK10 is indeed required for proper morphogenesis of dendritic spines.DOCK10 is a GEF for Rac1 and for Cdc42 in vitro and in cellsBoth Cdc42 and Rac1 GTPases are involved in different aspects of spine development, promoting growth and/or stability of dendritic spines (Tada and Sheng, 2006). This prompted us to ask whether Cdc42 and Rac1 could be targets of DOCK10 in this process. Intriguingly, all DOCK-D subfamily GEFs, including DOCK10, have been reported as Cdc42-specific GEFs. In nonneuronal cell lines, DOCK10 has been described as a GEF specific for Cdc42 by both GTPase-binding assays and active GTPase pull-down assays (Nishikimi et al., 2005; Gadea et al., 2008). However, the exchange activity of DOCK10 on Cdc42 has never been formally shown in vitro, in contrast to that of DOCK9 and DOCK11 (Cote and Vuori, 2002; Lin et al., 2006).To clarify this issue, we performed in vitro GEF assays on recombinant Cdc42, Rac1, and RhoA (Figure 6, A and B, and Supplemental Figure S3A). The catalytic DHR2 domains of DOCK9, DOCK10, and DOCK11 (as defined by Cote and Vuori, 2006) were produced as recombinant proteins in bacteria and used in a mant-GTP fluorescence kinetics assay (Bouquier et al., 2009). In this assay, the DHR2 domains of DOCK9, DOCK10, and DOCK11 promoted GDP/GTP exchange on Cdc42 and not on RhoA, as expected from previous data (Figure 6A and Supplemental Figure S3A; Cote and Vuori, 2002; Lin et al., 2006). Surprisingly, DOCK10 DHR2 was also able to activate directly Rac1, in contrast to the other two GEFs, establishing DOCK10 as the sole dual-specificity RhoGEF of the DOCK-D subfamily (Figure 6B).FIGURE 6:DOCK10 is a GEF for Rac1 and Cdc42 in vitro and in cells. (A, B) In vitro GEF assay performed on Cdc42 (A) and Rac1 (B), using GST-DHR2 domains of DOCK9, DOCK10, and DOCK11. GST-Dbl and GST-Trio GEF1 were used as positive control GEFs for Cdc42 and Rac1, respectively, whereas GST alone was used as negative control. Results are expressed as relative fluorescence units (RFU) vs. time. (C–E) Active GTPase pull-down assays. GTP-loaded Cdc42 (C) or GTP-loaded Rac1 (D, E) were pulled down from protein lysates of HEK293 cells expressing wild-type pEGFP-DOCK9, -10 or -11 constructs or DOCK10 GD. GFP vector was used as a negative control. Shown are representative experiments. GTP-bound and total Cdc42 or Rac1 were detected by Western blot, using anti-Cdc42 and anti-Rac1 antibodies, respectively. Protein expression in the cell lysates was verified by immunoblotting with an anti-GFP antibody. Histograms underneath each Western blot show the quantification of the corresponding GTPase activation assays. The ratio of GTP-Rac1 (or Cdc42) over total Rac1 (or Cdc42) was calculated from at least three independent experiments. Error bars indicate SEM, **p < 0.003, ***p < 0.0001.To confirm these in vitro data in intact cells, we performed Cdc42-GTP, Rac1-GTP, and RhoA-GTP pull-down experiments, using the Cdc42/Rac1 interactive binding (CRIB) domain of WASP and PAK1 and the RhoA-binding domain (RBD) of Rhotekin, respectively (Figure 6, C– E, and Supplemental Figure S3B). Cdc42-GTP, but not RhoA-GTP, was efficiently pulled down from HEK293T cells expressing all three full-length DOCK proteins (Figure 6C and Supplemental Figure S3B). Rac1-GTP, in contrast, was only and very efficiently pulled-down from cells expressing full-length DOCK10 (Figure 6D), thus confirming the in vitro data. To rule out the possibility that this activation of Rac1 by DOCK10 in cells was indirect, we used the GEF-dead point mutant of DOCK10 (DOCK10 GD) described earlier. When expressed in cells, this mutant was unable to elicit the robust activation of Rac1 observed with wild-type DOCK10 (Figure 6E). Thus these data show that DOCK10 is a bona fide GEF for Rac1.Taken together, our results reveal DOCK10 as a dual-specific GEF, activating both Cdc42 and Rac1 in vitro and in cells, whereas the other subfamily members, DOCK9 and DOCK11, are strictly specific for Cdc42.Cdc42 GEF activity is required for DOCK10-mediated spine formationNext we asked which of these GTPases would be the target(s) of DOCK10 for its function on spine morphogenesis. To determine this, we transfected cultured hippocampal neurons at 8 DIV with DOCK10 and well-established shRNAs targeting Cdc42 or Rac1 (Momboisse et al., 2009) or with a control shRNA (shCtrl). As expected, expression of shRNAs targeting Rac1 or Cdc42 alone significantly decreased spine number along the dendrites as compared with shCtrl (Figure 7, A and B). Overexpression of DOCK10 (with shCtrl) increased spine density moderately but significantly (Figure 7, A and B), confirming its positive effect on spine formation. In contrast, coexpression of shCdc42 with DOCK10 markedly suppressed this effect, and this was not the case with shRac1. Taken together, these data suggest that DOCK10 signals essentially through Cdc42 for spine formation.FIGURE 7:DOCK10 regulates spine formation via Cdc42-N-WASP and -PAK3 pathways. (A) The effects of DOCK10 on spine formation are mediated by Cdc42. Quantification of dendritic spine density, measured as described in Figure 5B, of hippocampal neurons transfected with shRNAs alone targeting either Cdc42 or Rac1 (Momboisse et al., 2009; or control shRNA) or in combination with DOCK10. Appropriate experimental conditions (transfection at 8 DIV and fixation at 11 DIV) were used to quantify formation of spines. Error bars indicate SEM (n = 3 independent experiments); *p < 0.05, **p < 0.01 (Student's t test). (B) Representative micrographs of hippocampal neurons transfected as described in A. Scale bar, 1 μm. (C) Quantification of dendritic spine density, measured as described in Figure 5B, of hippocampal neurons transfected with N-WASP∆WA, N-WASPWA, or PAK3KD alone or in combination with DOCK10. Appropriate experimental conditions as in A were used to quantify formation of spines. Error bars indicate SEM (n = 3 independent experiments); **p < 0.01, ***p < 0.001 (Student's t test). (D) Quantification of dendritic spine density, measured as described in Figure 5B, of hippocampal neurons transfected with sh Ctrl or shA-DOCK10 in combination with N-WASP or PAK3. Appropriate experimental conditions as in A were used to quantify formation of spines. Error bars indicate SEM (n = 3 independent experiments), ***p < 0.001 (Student's t test).To confirm that DOCK10 acted through Cdc42 for its effect on spine formation, we tested whether dominant-negative forms of Cdc42-specific effectors, such as N-WASP and PAK3, were able to block DOCK10-mediated spinogenesis. N-WASP has been shown to mediate a direct connection between Cdc42 and the Arp2/3 complex for actin polymerization (Rohatgi et al., 1999). Group 1 PAK kinases are effectors of both Cdc42 and Rac1, but PAK3 has been shown to bind preferentially Cdc42 (Kreis et al., 2007). Both N-WASP and PAK3 are critical regulators of spine morphogenesis (Kreis et al., 2007; Wegner et al., 2008). As shown previously, expression of N-WASP∆WA, a construct encoding N-WASP deleted of its C-terminal WA domain and thus unable to bind the Arp2/3 complex and G-actin (Supplemental Figure S4; Wegner et al., 2008), or expression of this domain alone (N-WASP WA) affected the number of spines in cultured hippocampal neurons (Figure 7C). Expression of a kinase-dead form of PAK3 (PAK3 KD; Supplemental Figure S4) also affected spine number (Figure 7C). Strikingly, coexpression of DOCK10 with either N-WASP∆WA or N-WASP WA or PAK3 KD impaired DOCK10's effect on spine formation (Figure 7C).To demonstrate further that DOCK10 activated Cdc42 to promote the formation of spines, we examined whether the poor spine morphology induced by DOCK10 depletion could be rescued by coexpressing wt N-WASP or PAK3. Expression of either N-WASP or PAK3 nicely rescued the defect in spine formation due to DOCK10 depletion (Figure 7D) to a similar extent as did reintroduction of DOCK10shR (Figure 5F).Combined, these data argue for N-WASP– and PAK3–mediated pathways downstream of DOCK10/Cdc42 functioning in spine morphogenesis.

Gene expression profiling during postnatal Purkinje neuron developmentTo identify novel RhoGEFs involved in dendritic tree morphogenesis, we undertook gene expression profiling of all DOCK-family RhoGEFs in Purkinje cells. Because these neurons represent only 2–3% of the whole cerebellum, we took advantage of a mouse strain carrying the green fluorescent protein (GFP) under the Purkinje-specific promoter Pcp2 to purify the PCs by FACS (Tomomura et al., 2001). Neurons were isolated from cerebella of postnatal day 3 (P3), P7, P15, and P20 Pcp2-GFP mice, corresponding to the time frame of development of the Purkinje cell dendritic tree. Purity of the Purkinje cell samples was assessed both by visual counting of the calbindin-positive cells (calbindin is a Purkinje cell–specific marker) before and after sorting (Figure 1, A-C) and by real-time quantitative PCR (RT-qPCR) of marker genes specific for the Purkinje cells (calbindin) and for the other main cerebellar cell types, namely granule cells (NeuroD1), astrocytes (glial fibrillary acidic protein [GFAP]), and interneurons (Tcfap2a; Figure 1D). After sorting, Purkinje cells represented ∼78% of the cell population.FIGURE 1:Gene expression profiling of all DOCK RhoGEFs in purified postnatal Purkinje neurons. (A, B) Sorting of GFP-expressing postnatal Purkinje neurons by FACS. Dissociated cells from cerebella of Pcp2-GFP mice at P3, P7, P15, and P20 were plated before (A) and after (B) sorting and fixed. Nuclei were stained with Hoechst stain to visualize the total cell number in the sample, and Purkinje neurons were revealed with a calbindin antibody (red) and by direct fluorescence of the GFP (green). Micrographs show representative sorting results of cerebellar cells from mice at P7. (C) Histogram representing the enrichment of Purkinje neurons within the cell population upon sorting, expressed as relative percentages. Data are expressed as mean enrichment value of all age groups combined ± SD of at least three independent experiments. **p < 0.01 (Student's t test). (D) RT-qPCR performed before and after FACS on the mRNA of specific marker genes of the main cerebellar cell types: calbindin (Purkinje cells), NeuroD1 (cerebellar granule neurons), GFAP (astrocytes), and Tcfap2a (interneurons). Data are expressed as mean ± SD of at least three experiments performed on P7 Pcp2-GFP mice in the example shown. **p < 0.01 and ***p < 0.001 (Student's t test). (E) RT-qPCR performed on purified PC mRNAs of all 11 mammalian DOCK-family RhoGEFs. Data are expressed as mean ± SD of at least three experiments. *p < 0.05, **p < 0.01, and ***p < 0.001 (Student's t test).We then performed RT-qPCR on mRNAs isolated from these cells and analyzed the expression pattern of all 11 members of the mammalian DOCK family of RhoGEFs in the course of development (Figure 1E; see Supplemental Table S1 for primer sequences). The mRNAs of DOCK180/DOCK1, DOCK2, DOCK3, and DOCK8 could not be detected due to too low expression levels. DOCK4, DOCK5, DOCK6, and DOCK7 were expressed, but their mRNA levels did not vary significantly across the four developmental stages we analyzed, except for DOCK4 and DOCK6, whose expression dropped at P20. In contrast, the three DOCK-D subfamily members—DOCK9, DOCK10, and DOCK11—showed a significant increase in their mRNA expression levels, reaching a peak at P15 (DOCK11) or P20 (DOCK9 and DOCK10; Figure 1E). We identified DOCK10 as a promising candidate since its robust increase in expression occurred precisely during the time frame of Purkinje cell spinogenesis during the second and third postnatal weeks and no neuronal function had yet been ascribed to it.

Gene expression profiling during postnatal Purkinje neuron development

To identify novel RhoGEFs involved in dendritic tree morphogenesis, we undertook gene expression profiling of all DOCK-family RhoGEFs in Purkinje cells. Because these neurons represent only 2–3% of the whole cerebellum, we took advantage of a mouse strain carrying the green fluorescent protein (GFP) under the Purkinje-specific promoter Pcp2 to purify the PCs by FACS (Tomomura et al., 2001). Neurons were isolated from cerebella of postnatal day 3 (P3), P7, P15, and P20 Pcp2-GFP mice, corresponding to the time frame of development of the Purkinje cell dendritic tree. Purity of the Purkinje cell samples was assessed both by visual counting of the calbindin-positive cells (calbindin is a Purkinje cell–specific marker) before and after sorting (Figure 1, A-C) and by real-time quantitative PCR (RT-qPCR) of marker genes specific for the Purkinje cells (calbindin) and for the other main cerebellar cell types, namely granule cells (NeuroD1), astrocytes (glial fibrillary acidic protein [GFAP]), and interneurons (Tcfap2a; Figure 1D). After sorting, Purkinje cells represented ∼78% of the cell population.

Gene expression profiling of all DOCK RhoGEFs in purified postnatal Purkinje neurons. (A, B) Sorting of GFP-expressing postnatal Purkinje neurons by FACS. Dissociated cells from cerebella of Pcp2-GFP mice at P3, P7, P15, and P20 were plated before (A) and after (B) sorting and fixed. Nuclei were stained with Hoechst stain to visualize the total cell number in the sample, and Purkinje neurons were revealed with a calbindin antibody (red) and by direct fluorescence of the GFP (green). Micrographs show representative sorting results of cerebellar cells from mice at P7. (C) Histogram representing the enrichment of Purkinje neurons within the cell population upon sorting, expressed as relative percentages. Data are expressed as mean enrichment value of all age groups combined ± SD of at least three independent experiments. **p < 0.01 (Student's t test). (D) RT-qPCR performed before and after FACS on the mRNA of specific marker genes of the main cerebellar cell types: calbindin (Purkinje cells), NeuroD1 (cerebellar granule neurons), GFAP (astrocytes), and Tcfap2a (interneurons). Data are expressed as mean ± SD of at least three experiments performed on P7 Pcp2-GFP mice in the example shown. **p < 0.01 and ***p < 0.001 (Student's t test). (E) RT-qPCR performed on purified PC mRNAs of all 11 mammalian DOCK-family RhoGEFs. Data are expressed as mean ± SD of at least three experiments. *p < 0.05, **p < 0.01, and ***p < 0.001 (Student's t test).

We then performed RT-qPCR on mRNAs isolated from these cells and analyzed the expression pattern of all 11 members of the mammalian DOCK family of RhoGEFs in the course of development (Figure 1E; see Supplemental Table S1 for primer sequences). The mRNAs of DOCK180/DOCK1, DOCK2, DOCK3, and DOCK8 could not be detected due to too low expression levels. DOCK4, DOCK5, DOCK6, and DOCK7 were expressed, but their mRNA levels did not vary significantly across the four developmental stages we analyzed, except for DOCK4 and DOCK6, whose expression dropped at P20. In contrast, the three DOCK-D subfamily members—DOCK9, DOCK10, and DOCK11—showed a significant increase in their mRNA expression levels, reaching a peak at P15 (DOCK11) or P20 (DOCK9 and DOCK10; Figure 1E). We identified DOCK10 as a promising candidate since its robust increase in expression occurred precisely during the time frame of Purkinje cell spinogenesis during the second and third postnatal weeks and no neuronal function had yet been ascribed to it.

DOCK10 is expressed in Purkinje cells during postnatal developmentTo confirm the RT-qPCR data and address the expression of the DOCK10 protein in the postnatal cerebellum, we analyzed the expression pattern of DOCK10 in postnatal mouse brain slices by immunohistochemistry (IHC) using a DOCK10-specific antibody (Figure 2). In whole sections of the cerebellum (P15), the DOCK10 protein was detected in the monolayer of Purkinje cells, nicely overlapping with the Purkinje-specific calbindin staining (Figure 2A). Higher-magnification images showed that at P7, and more strikingly at P15 and later stages, DOCK10 was detected in the soma and the dendritic tree of PCs, although it was only weakly detectable in the latter (Figure 2, A, inset, and B). At P7, DOCK10 immunostaining could also be seen in the external and internal granule cell layer (IGL), but at P15 and P21, IGL staining disappeared and DOCK10 localization seemed to be restricted to the Purkinje cell layer (Figure 2B).FIGURE 2:DOCK10 expression in cerebellar Purkinje cells at postnatal stages. (A) Immunohistochemistry performed on wild-type C57BL/6 mouse cerebellar sections at P15, using a DOCK10 antibody (red) and a calbindin antibody to label the PCs (green). Nuclei were stained with Hoechst stain (blue) to visualize all cells in the slice. Owing to the high density of granule cells, the latter staining highlights mainly the internal granule cell layer. Bottom right, merged image. Note the monolayer-specific staining of PCs, revealed with both DOCK10 antibody and calbindin antibody. These antibodies also label the Purkinje cell dendritic tree in the molecular layer. Inset, higher-magnification image of the boxed region, showing the DOCK10 immunostaining. IGL, internal granule cell layer; ML, molecular layer. PCL, Purkinje cell layer; Scale bar, 250 μm. (B) Immunohistochemistry performed on P7, P15, and P21 C57BL/6 mouse brain sections, using a DOCK10 and a calbindin antibody. Nuclei were stained with Hoechst to visualize all cells in the slice. The various cell layers found in the section are indicated as in A; EGL, external granule cell layer. Note the Purkinje cell–specific staining of DOCK10 at P15 and P21 in the soma and to a lesser extent in the dendritic tree (see also inset in A). Scale bar, 25 μm.Taken together, these data show that, at early postnatal stages, DOCK10 is expressed broadly in the cerebellum and that from P15 on, its expression becomes restricted to the Purkinje cells.

DOCK10 is expressed in Purkinje cells during postnatal development

To confirm the RT-qPCR data and address the expression of the DOCK10 protein in the postnatal cerebellum, we analyzed the expression pattern of DOCK10 in postnatal mouse brain slices by immunohistochemistry (IHC) using a DOCK10-specific antibody (Figure 2). In whole sections of the cerebellum (P15), the DOCK10 protein was detected in the monolayer of Purkinje cells, nicely overlapping with the Purkinje-specific calbindin staining (Figure 2A). Higher-magnification images showed that at P7, and more strikingly at P15 and later stages, DOCK10 was detected in the soma and the dendritic tree of PCs, although it was only weakly detectable in the latter (Figure 2, A, inset, and B). At P7, DOCK10 immunostaining could also be seen in the external and internal granule cell layer (IGL), but at P15 and P21, IGL staining disappeared and DOCK10 localization seemed to be restricted to the Purkinje cell layer (Figure 2B).

DOCK10 expression in cerebellar Purkinje cells at postnatal stages. (A) Immunohistochemistry performed on wild-type C57BL/6 mouse cerebellar sections at P15, using a DOCK10 antibody (red) and a calbindin antibody to label the PCs (green). Nuclei were stained with Hoechst stain (blue) to visualize all cells in the slice. Owing to the high density of granule cells, the latter staining highlights mainly the internal granule cell layer. Bottom right, merged image. Note the monolayer-specific staining of PCs, revealed with both DOCK10 antibody and calbindin antibody. These antibodies also label the Purkinje cell dendritic tree in the molecular layer. Inset, higher-magnification image of the boxed region, showing the DOCK10 immunostaining. IGL, internal granule cell layer; ML, molecular layer. PCL, Purkinje cell layer; Scale bar, 250 μm. (B) Immunohistochemistry performed on P7, P15, and P21 C57BL/6 mouse brain sections, using a DOCK10 and a calbindin antibody. Nuclei were stained with Hoechst to visualize all cells in the slice. The various cell layers found in the section are indicated as in A; EGL, external granule cell layer. Note the Purkinje cell–specific staining of DOCK10 at P15 and P21 in the soma and to a lesser extent in the dendritic tree (see also inset in A). Scale bar, 25 μm.

Taken together, these data show that, at early postnatal stages, DOCK10 is expressed broadly in the cerebellum and that from P15 on, its expression becomes restricted to the Purkinje cells.

Depletion of DOCK10 in Purkinje cells leads to dendritic spine defectsGiven the expression of DOCK10 in differentiating Purkinje neurons, we investigated the function of DOCK10 in these cells by analyzing the effect of knocking down its expression. To do so, we designed and validated two different short hairpin RNAs (shRNAs) targeting DOCK10 (shA and shB) and a control shRNA (shCtrl; see Materials and Methods). Both shRNAs targeting DOCK10 efficiently knocked down endogenous DOCK10 expression (Supplemental Figure S1A) and were ineffective on the closely related DOCK9 and DOCK11 RhoGEFs (Supplemental Figure S1B). We then transduced mouse cerebellar organotypic cultures at P0 with lentiviral vectors expressing these shRNAs and also carrying GFP as a transduction marker. In these organotypic slices, the dendritic development of Purkinje cells is almost similar to that observed in vivo, although in a slightly delayed manner (Boukhtouche et al., 2006; Poulain et al., 2008). Slices were grown until 14 days in vitro (DIV) and, after fixation, PC morphology was analyzed by immunofluorescence staining with a calbindin antibody. On the same slice, transduced (calbindin+/GFP+) versus nontransduced (calbindin+/GFP–) Purkinje cells could be visualized (Figure 3A), allowing for an accurate comparison of the cell morphology, whatever the transduced state. A blinded analysis was then conducted in order to analyze whether the knockdown of DOCK10 affected the general dendritic morphology of Purkinje cells (see Materials and Methods). To do so, all of the Purkinje cells were distributed into three groups, depending on their dendritic morphology as observed by calbindin immunostaining (Figure 3B): PCs with an atrophic morphology (the longest dendrite being smaller than the cell body diameter), PCs with a stellate morphology (harboring more than six perisomatic processes), and PCs with an elaborate dendritic tree.FIGURE 3:Depletion of DOCK10 in Purkinje cells leads to dendritic spine defects. (A) Micrographs of organotypic cultures that received Lv-ShA DOCK10. The arrowhead represents a transduced Purkinje cell (GFP+/calbindin+), whereas the arrow represents a nontransduced Purkinje cell (GFP–/Calbindin+). Scale bar, 18.75 μm. (B) Micrographs illustrating typical calbindin-labeled PCs observed in organotypic cultures after 14 DIV: atrophic, stellate, and elaborate PCs. (C) Histograms representing the distribution of Purkinje cell types 14 DIV after transduction with lentiviral vectors expressing the GFP marker together with either a control shRNA or the two DOCK10 shRNAs shA and shB. Data are presented as relative percentage for each stage in nontransduced (NT) and transduced (T) PCs. For statistical analysis, the distributions of Purkinje cell types were compared using Pearson's χ2 test or Fisher's exact test, depending on the number of PCs per group (nonsignificant differences were detected, p > 0.5). (D) Micrographs of calbindin-labeled Purkinje cells within organotypic cultures after 14 DIV and lentiviral infection. D1 is a nontransduced Purkinje cell, whereas D2 and D3 are Purkinje cells transduced with shA DOCK10 and shB DOCK10, respectively. Arrowheads point to a dendritic area covered with spines; arrow, to a naked dendritic area (devoid of spines). Scale bar, 18.75 μm. (E) Histograms representing the distribution of Purkinje cells distinguishing cells with at least one part of the dendritic tree covered with spines (spines) and cells with a dendritic tree devoid of spines (no spines) after transduction by lentiviral vectors as described in C. Data are presented as relative percentage for each group. Compared are nontransduced Purkinje cells (NT) and transduced PCs (T). For statistical analysis, the distributions of Purkinje cell types were compared using Pearson's χ2 test or Fisher's exact test, depending on the number of PCs per group (ns, nonsignificant, **p < 0.005, ***p < 0.001). (F) Sholl analysis of PCs with an elaborate dendritic tree in the different experimental groups. Sholl analysis was performed on cells with at least one part of the dendritic tree covered with spines (spines) and cells with a dendritic tree devoid of spines (no spines) in the different transduced conditions (shCtlr, shA DOCK10, and shB DOCK10).No statistically significant difference could be detected in the distribution within these three groups of PCs, whether transduced or not with shA or shB DOCK10 or shCtrl lentivectors (Figure 3C), suggesting that knockdown of DOCK10 expression does not have a major effect on overall Purkinje cell dendritic development.More careful analysis of the elaborate Purkinje cells, however, revealed two types of cells: cells presenting at least one part of the dendritic tree covered with spines (Figure 3D, D1) and cells with a dendritic tree totally devoid of spines (Figure 3D, D2 and D3). Nearly all of the nontransduced PCs and the PCs transduced with ShCtrl presented an elaborate dendritic tree with spines (Figure 3, D and E). In contrast, 53 and 23% of Purkinje cells transduced with shA-DOCK10 and shB-DOCK10, respectively, presented a dendritic tree completely devoid of spines (Figure 3, D and E). This defect was observed with both shRNAs, albeit to a lesser extent with Lv-ShB, excluding an off-target effect.To determine whether depletion of DOCK10 acted only on the formation of spines or the observed phenotype was an indirect consequence of defective dendritic tree morphogenesis, we performed a Sholl analysis on these elaborate Purkinje cells to measure the complexity and height of the dendritic tree (see Materials and Methods; Figure 3F). The general distribution of the Sholl index was the same whether the cells were transduced with shCtrl or shA- or shB-DOCK10. The large majority of the branching points were observed between 20 and 40 μm from the soma (Figure 3F). Although the Sholl index at 30 μm from the soma varied between the different lentivector-transduced conditions (from 4.6 to 7.7), these differences were not correlated with the presence or absence of spines. Therefore this analysis revealed no significant effect of any shRNA on the complexity or height of the dendritic tree of elaborate PCs, showing a direct role for DOCK10 in dendritic spine morphogenesis.

Depletion of DOCK10 in Purkinje cells leads to dendritic spine defects

Given the expression of DOCK10 in differentiating Purkinje neurons, we investigated the function of DOCK10 in these cells by analyzing the effect of knocking down its expression. To do so, we designed and validated two different short hairpin RNAs (shRNAs) targeting DOCK10 (shA and shB) and a control shRNA (shCtrl; see Materials and Methods). Both shRNAs targeting DOCK10 efficiently knocked down endogenous DOCK10 expression (Supplemental Figure S1A) and were ineffective on the closely related DOCK9 and DOCK11 RhoGEFs (Supplemental Figure S1B). We then transduced mouse cerebellar organotypic cultures at P0 with lentiviral vectors expressing these shRNAs and also carrying GFP as a transduction marker. In these organotypic slices, the dendritic development of Purkinje cells is almost similar to that observed in vivo, although in a slightly delayed manner (Boukhtouche et al., 2006; Poulain et al., 2008). Slices were grown until 14 days in vitro (DIV) and, after fixation, PC morphology was analyzed by immunofluorescence staining with a calbindin antibody. On the same slice, transduced (calbindin+/GFP+) versus nontransduced (calbindin+/GFP–) Purkinje cells could be visualized (Figure 3A), allowing for an accurate comparison of the cell morphology, whatever the transduced state. A blinded analysis was then conducted in order to analyze whether the knockdown of DOCK10 affected the general dendritic morphology of Purkinje cells (see Materials and Methods). To do so, all of the Purkinje cells were distributed into three groups, depending on their dendritic morphology as observed by calbindin immunostaining (Figure 3B): PCs with an atrophic morphology (the longest dendrite being smaller than the cell body diameter), PCs with a stellate morphology (harboring more than six perisomatic processes), and PCs with an elaborate dendritic tree.

Depletion of DOCK10 in Purkinje cells leads to dendritic spine defects. (A) Micrographs of organotypic cultures that received Lv-ShA DOCK10. The arrowhead represents a transduced Purkinje cell (GFP+/calbindin+), whereas the arrow represents a nontransduced Purkinje cell (GFP–/Calbindin+). Scale bar, 18.75 μm. (B) Micrographs illustrating typical calbindin-labeled PCs observed in organotypic cultures after 14 DIV: atrophic, stellate, and elaborate PCs. (C) Histograms representing the distribution of Purkinje cell types 14 DIV after transduction with lentiviral vectors expressing the GFP marker together with either a control shRNA or the two DOCK10 shRNAs shA and shB. Data are presented as relative percentage for each stage in nontransduced (NT) and transduced (T) PCs. For statistical analysis, the distributions of Purkinje cell types were compared using Pearson's χ2 test or Fisher's exact test, depending on the number of PCs per group (nonsignificant differences were detected, p > 0.5). (D) Micrographs of calbindin-labeled Purkinje cells within organotypic cultures after 14 DIV and lentiviral infection. D1 is a nontransduced Purkinje cell, whereas D2 and D3 are Purkinje cells transduced with shA DOCK10 and shB DOCK10, respectively. Arrowheads point to a dendritic area covered with spines; arrow, to a naked dendritic area (devoid of spines). Scale bar, 18.75 μm. (E) Histograms representing the distribution of Purkinje cells distinguishing cells with at least one part of the dendritic tree covered with spines (spines) and cells with a dendritic tree devoid of spines (no spines) after transduction by lentiviral vectors as described in C. Data are presented as relative percentage for each group. Compared are nontransduced Purkinje cells (NT) and transduced PCs (T). For statistical analysis, the distributions of Purkinje cell types were compared using Pearson's χ2 test or Fisher's exact test, depending on the number of PCs per group (ns, nonsignificant, **p < 0.005, ***p < 0.001). (F) Sholl analysis of PCs with an elaborate dendritic tree in the different experimental groups. Sholl analysis was performed on cells with at least one part of the dendritic tree covered with spines (spines) and cells with a dendritic tree devoid of spines (no spines) in the different transduced conditions (shCtlr, shA DOCK10, and shB DOCK10).

No statistically significant difference could be detected in the distribution within these three groups of PCs, whether transduced or not with shA or shB DOCK10 or shCtrl lentivectors (Figure 3C), suggesting that knockdown of DOCK10 expression does not have a major effect on overall Purkinje cell dendritic development.

More careful analysis of the elaborate Purkinje cells, however, revealed two types of cells: cells presenting at least one part of the dendritic tree covered with spines (Figure 3D, D1) and cells with a dendritic tree totally devoid of spines (Figure 3D, D2 and D3). Nearly all of the nontransduced PCs and the PCs transduced with ShCtrl presented an elaborate dendritic tree with spines (Figure 3, D and E). In contrast, 53 and 23% of Purkinje cells transduced with shA-DOCK10 and shB-DOCK10, respectively, presented a dendritic tree completely devoid of spines (Figure 3, D and E). This defect was observed with both shRNAs, albeit to a lesser extent with Lv-ShB, excluding an off-target effect.

To determine whether depletion of DOCK10 acted only on the formation of spines or the observed phenotype was an indirect consequence of defective dendritic tree morphogenesis, we performed a Sholl analysis on these elaborate Purkinje cells to measure the complexity and height of the dendritic tree (see Materials and Methods; Figure 3F). The general distribution of the Sholl index was the same whether the cells were transduced with shCtrl or shA- or shB-DOCK10. The large majority of the branching points were observed between 20 and 40 μm from the soma (Figure 3F). Although the Sholl index at 30 μm from the soma varied between the different lentivector-transduced conditions (from 4.6 to 7.7), these differences were not correlated with the presence or absence of spines. Therefore this analysis revealed no significant effect of any shRNA on the complexity or height of the dendritic tree of elaborate PCs, showing a direct role for DOCK10 in dendritic spine morphogenesis.

DOCK10 is required for the formation of dendritic spines in hippocampal neuronsTo better characterize the function of DOCK10 in dendritic spine morphogenesis, we turned to primary cultures of hippocampal neurons, which are a classical model with which to study spine morphogenesis and are more tractable to culture and transfection than Purkinje cells. We first verified that DOCK10 was expressed in the postnatal mouse hippocampus. Indeed, Western blot analysis performed on lysates of different P26 mouse brain areas and on lysates of cultured hippocampal neurons (at 18 DIV) revealed the expression of DOCK10 in the hippocampus but also in the olfactory bulb and the cortex (Figure 4A). The hippocampal expression of DOCK10 was also confirmed by IHC analysis on P15 mouse brain sections, using a DOCK10 antibody (Figure 4C). This revealed that DOCK10 was present in the pyramidal cells of the CA1 and CA3 regions, as well as in the adjacent subiculum and the dentate gyrus.FIGURE 4:DOCK10 is expressed in the hippocampus and localized in dendritic spines in hippocampal neurons. (A) Protein extracts from various P26 mouse brain regions and from cultured hippocampal neurons at 18 DIV were analyzed by Western blot, using a DOCK10 antibody. Tubulin was used as a loading control. (B) Subcellular fractions obtained after fractionation of P27 mouse brain extracts on a sucrose gradient were analyzed by Western blot, using a DOCK10, a PSD-95, and a Cdc42 antibody. Tubulin was used as a loading control. S1, soluble cytoplasmic fraction; P1, vesicular fraction; P2, synaptosomal fraction; P3, membrane fraction. (C) Immunohistochemistry performed on stage P15 mouse hippocampus sections. Immunostaining was performed using a DOCK10 antibody (red). Nuclei were stained with Hoechst stain (blue) to visualize all cells in the slice. Owing to the high density of cells, the latter staining highlights mainly the dentate gyrus. Right, merged image. Underneath each image are higher-magnification images of the region selected (CA1/subiculum region). Scale bar, 40 μm. (D) DOCK10 is localized in dendrites and spines of hippocampal neurons. Micrographs of cultured hippocampal neurons cotransfected with Flag-DOCK10 (detected with an anti-Flag antibody; red) and GFP to visualize the dendrites (green). Right, merged images. Bottom, higher- magnification images of the region selected. Scale bar, 1 μm. Arrows, examples of spine heads labeled by DOCK10. (E) DOCK10 postsynaptic localization is adjacent to presynaptic terminals. Micrographs of hippocampal neurons cotransfected with Flag-DOCK10 (detected with an anti-Flag antibody; red; top right) and GFP to visualize the dendrites (green; top left). Immunostaining was performed using an anti-Bassoon antibody (blue, bottom left), which marks the presynaptic terminals. Bottom right, merged image. Insets, higher-magnification images of a selected region to better visualize the adjacent localization of DOCK10 and Bassoon (arrows).We then studied the specific subcellular localization of endogenous DOCK10 by performing fractionation experiments on a sucrose gradient. This showed that DOCK10 was present in the synaptosomal fraction, where it fractionated together with the postsynaptic density protein PSD-95 (Figure 4B), indicating a synaptic localization for DOCK10. Both proteins were also detected in the membrane fraction. Of interest, Cdc42, the target GTPase of DOCK10, was also detected in these two fractions (Figure 4B). We further confirmed the synaptosomal localization of DOCK10 by immunofluorescence studies by transfecting flag-DOCK10 into cultured hippocampal neurons. Figure 4D shows that DOCK10 was localized in the dendritic shaft and most strikingly in the dendritic spines in hippocampal neurons, which is compatible with a function of DOCK10 in dendritic spine morphogenesis. In addition, DOCK10 immunostaining in the spines localized adjacent to the presynaptic protein Bassoon (Figure 4E). Combined, these data show that DOCK10 is present in the postsynaptic density.We next analyzed the consequences of DOCK10 depletion on spine formation in hippocampal neurons by transfecting them at 8 DIV (before the initial appearance of the spines) and fixing them 3 d later. ShRNA-mediated depletion of DOCK10 led to a dramatic decrease in the number of spines along the dendritic shaft (Figure 5, A and B). This defect was accompanied by a decrease in spine head size in the remaining spines (Figure 5C). In addition, the postsynaptic density protein Homer-1 was hardly detected by immunofluorescence in the remaining protrusions of DOCK10-depleted neurons as compared with control neurons, confirming that spines devoid of DOCK10 are not properly formed (Figure 5, D and E). To confirm the requirement of DOCK10 for spine formation, we performed rescue experiments by reintroducing a shRNA-resistant DOCK10 construct into DOCK10-depleted hippocampal neurons. This construct was designed to be resistant only to shA DOCK10 and not to shB, as demonstrated in Supplemental Figure S2A (see Materials and Methods). As shown in Figure 5, F and G, wild-type (wt) DOCK10shR was able to rescue the decrease in both spine number and spine head size induced by shA-mediated DOCK10 depletion, demonstrating its requirement for spine morphogenesis.FIGURE 5:The GEF activity of DOCK10 is required for the formation of dendritic spines in hippocampal neurons. (A) Depletion of DOCK10 leads to decreased spine density and spine head size. Representative micrographs of hippocampal neurons transfected with a control shRNA (shCtrl) or two different DOCK10 shRNAs (shA and shB). Neurons were transfected at 8 DIV and fixed at 11 DIV (experimental conditions to reveal spine formation). Scale bar, 1 μm. (B, C) Quantification of dendritic spine density (B) and size (C) of hippocampal neurons transfected as described in A, measured by the number of spines/10 μm and the spine head area, respectively. Error bars indicate SEM (n = 3 independent experiments); **p < 0.01 (Student's t test). (D) Representative micrographs of hippocampal neurons transfected with shA DOCK10 or shCtrl (green) and immunostained for postsynaptic protein Homer-1 (red; middle micrographs). Right, merged images. Scale bar, 1 μm. (E) Quantification of the number of protrusions positive for Homer-1 staining in DOCK10-depleted or control neurons, as described in D. Error bars indicate SEM (n = 4 independent experiments); **p = 0.001 (Student's t test). (F, G) A functional GEF activity of DOCK10 is required for spine formation in hippocampal neurons. Quantification of spine density (F) and size (G) of hippocampal neurons cotransfected with shA-DOCK10 and either a control vector or shA-resistant wt DOCK10shR or shA-resistant GEF-dead DOCK10 (DOCK10 GDshR). Error bars indicate SEM (n = 3 independent experiments), *p < 0.05, #p < 0.05 (Student's t test).To determine whether the dependence on DOCK10 for spine formation required its GEF activity, we created a catalytically inactive mutant based on a similar mutant in DOCK9 (Yang et al., 2009). In this mutant, the catalytic DHR2 domain harbors a point mutation abolishing its GEF activity, as assessed by in vitro GEF assays on Cdc42 (Supplemental Figure S2B). Of interest, this GEF-dead mutant DOCK10 (DOCK10shR GD) was unable to rescue the spine defects induced by DOCK10 depletion (Figure 5, F and G), indicating that the GEF activity of DOCK10 is indeed required for proper morphogenesis of dendritic spines.

DOCK10 is required for the formation of dendritic spines in hippocampal neurons

To better characterize the function of DOCK10 in dendritic spine morphogenesis, we turned to primary cultures of hippocampal neurons, which are a classical model with which to study spine morphogenesis and are more tractable to culture and transfection than Purkinje cells. We first verified that DOCK10 was expressed in the postnatal mouse hippocampus. Indeed, Western blot analysis performed on lysates of different P26 mouse brain areas and on lysates of cultured hippocampal neurons (at 18 DIV) revealed the expression of DOCK10 in the hippocampus but also in the olfactory bulb and the cortex (Figure 4A). The hippocampal expression of DOCK10 was also confirmed by IHC analysis on P15 mouse brain sections, using a DOCK10 antibody (Figure 4C). This revealed that DOCK10 was present in the pyramidal cells of the CA1 and CA3 regions, as well as in the adjacent subiculum and the dentate gyrus.

DOCK10 is expressed in the hippocampus and localized in dendritic spines in hippocampal neurons. (A) Protein extracts from various P26 mouse brain regions and from cultured hippocampal neurons at 18 DIV were analyzed by Western blot, using a DOCK10 antibody. Tubulin was used as a loading control. (B) Subcellular fractions obtained after fractionation of P27 mouse brain extracts on a sucrose gradient were analyzed by Western blot, using a DOCK10, a PSD-95, and a Cdc42 antibody. Tubulin was used as a loading control. S1, soluble cytoplasmic fraction; P1, vesicular fraction; P2, synaptosomal fraction; P3, membrane fraction. (C) Immunohistochemistry performed on stage P15 mouse hippocampus sections. Immunostaining was performed using a DOCK10 antibody (red). Nuclei were stained with Hoechst stain (blue) to visualize all cells in the slice. Owing to the high density of cells, the latter staining highlights mainly the dentate gyrus. Right, merged image. Underneath each image are higher-magnification images of the region selected (CA1/subiculum region). Scale bar, 40 μm. (D) DOCK10 is localized in dendrites and spines of hippocampal neurons. Micrographs of cultured hippocampal neurons cotransfected with Flag-DOCK10 (detected with an anti-Flag antibody; red) and GFP to visualize the dendrites (green). Right, merged images. Bottom, higher- magnification images of the region selected. Scale bar, 1 μm. Arrows, examples of spine heads labeled by DOCK10. (E) DOCK10 postsynaptic localization is adjacent to presynaptic terminals. Micrographs of hippocampal neurons cotransfected with Flag-DOCK10 (detected with an anti-Flag antibody; red; top right) and GFP to visualize the dendrites (green; top left). Immunostaining was performed using an anti-Bassoon antibody (blue, bottom left), which marks the presynaptic terminals. Bottom right, merged image. Insets, higher-magnification images of a selected region to better visualize the adjacent localization of DOCK10 and Bassoon (arrows).

We then studied the specific subcellular localization of endogenous DOCK10 by performing fractionation experiments on a sucrose gradient. This showed that DOCK10 was present in the synaptosomal fraction, where it fractionated together with the postsynaptic density protein PSD-95 (Figure 4B), indicating a synaptic localization for DOCK10. Both proteins were also detected in the membrane fraction. Of interest, Cdc42, the target GTPase of DOCK10, was also detected in these two fractions (Figure 4B). We further confirmed the synaptosomal localization of DOCK10 by immunofluorescence studies by transfecting flag-DOCK10 into cultured hippocampal neurons. Figure 4D shows that DOCK10 was localized in the dendritic shaft and most strikingly in the dendritic spines in hippocampal neurons, which is compatible with a function of DOCK10 in dendritic spine morphogenesis. In addition, DOCK10 immunostaining in the spines localized adjacent to the presynaptic protein Bassoon (Figure 4E). Combined, these data show that DOCK10 is present in the postsynaptic density.

We next analyzed the consequences of DOCK10 depletion on spine formation in hippocampal neurons by transfecting them at 8 DIV (before the initial appearance of the spines) and fixing them 3 d later. ShRNA-mediated depletion of DOCK10 led to a dramatic decrease in the number of spines along the dendritic shaft (Figure 5, A and B). This defect was accompanied by a decrease in spine head size in the remaining spines (Figure 5C). In addition, the postsynaptic density protein Homer-1 was hardly detected by immunofluorescence in the remaining protrusions of DOCK10-depleted neurons as compared with control neurons, confirming that spines devoid of DOCK10 are not properly formed (Figure 5, D and E). To confirm the requirement of DOCK10 for spine formation, we performed rescue experiments by reintroducing a shRNA-resistant DOCK10 construct into DOCK10-depleted hippocampal neurons. This construct was designed to be resistant only to shA DOCK10 and not to shB, as demonstrated in Supplemental Figure S2A (see Materials and Methods). As shown in Figure 5, F and G, wild-type (wt) DOCK10shR was able to rescue the decrease in both spine number and spine head size induced by shA-mediated DOCK10 depletion, demonstrating its requirement for spine morphogenesis.

The GEF activity of DOCK10 is required for the formation of dendritic spines in hippocampal neurons. (A) Depletion of DOCK10 leads to decreased spine density and spine head size. Representative micrographs of hippocampal neurons transfected with a control shRNA (shCtrl) or two different DOCK10 shRNAs (shA and shB). Neurons were transfected at 8 DIV and fixed at 11 DIV (experimental conditions to reveal spine formation). Scale bar, 1 μm. (B, C) Quantification of dendritic spine density (B) and size (C) of hippocampal neurons transfected as described in A, measured by the number of spines/10 μm and the spine head area, respectively. Error bars indicate SEM (n = 3 independent experiments); **p < 0.01 (Student's t test). (D) Representative micrographs of hippocampal neurons transfected with shA DOCK10 or shCtrl (green) and immunostained for postsynaptic protein Homer-1 (red; middle micrographs). Right, merged images. Scale bar, 1 μm. (E) Quantification of the number of protrusions positive for Homer-1 staining in DOCK10-depleted or control neurons, as described in D. Error bars indicate SEM (n = 4 independent experiments); **p = 0.001 (Student's t test). (F, G) A functional GEF activity of DOCK10 is required for spine formation in hippocampal neurons. Quantification of spine density (F) and size (G) of hippocampal neurons cotransfected with shA-DOCK10 and either a control vector or shA-resistant wt DOCK10shR or shA-resistant GEF-dead DOCK10 (DOCK10 GDshR). Error bars indicate SEM (n = 3 independent experiments), *p < 0.05, #p < 0.05 (Student's t test).

To determine whether the dependence on DOCK10 for spine formation required its GEF activity, we created a catalytically inactive mutant based on a similar mutant in DOCK9 (Yang et al., 2009). In this mutant, the catalytic DHR2 domain harbors a point mutation abolishing its GEF activity, as assessed by in vitro GEF assays on Cdc42 (Supplemental Figure S2B). Of interest, this GEF-dead mutant DOCK10 (DOCK10shR GD) was unable to rescue the spine defects induced by DOCK10 depletion (Figure 5, F and G), indicating that the GEF activity of DOCK10 is indeed required for proper morphogenesis of dendritic spines.

DOCK10 is a GEF for Rac1 and for Cdc42 in vitro and in cellsBoth Cdc42 and Rac1 GTPases are involved in different aspects of spine development, promoting growth and/or stability of dendritic spines (Tada and Sheng, 2006). This prompted us to ask whether Cdc42 and Rac1 could be targets of DOCK10 in this process. Intriguingly, all DOCK-D subfamily GEFs, including DOCK10, have been reported as Cdc42-specific GEFs. In nonneuronal cell lines, DOCK10 has been described as a GEF specific for Cdc42 by both GTPase-binding assays and active GTPase pull-down assays (Nishikimi et al., 2005; Gadea et al., 2008). However, the exchange activity of DOCK10 on Cdc42 has never been formally shown in vitro, in contrast to that of DOCK9 and DOCK11 (Cote and Vuori, 2002; Lin et al., 2006).To clarify this issue, we performed in vitro GEF assays on recombinant Cdc42, Rac1, and RhoA (Figure 6, A and B, and Supplemental Figure S3A). The catalytic DHR2 domains of DOCK9, DOCK10, and DOCK11 (as defined by Cote and Vuori, 2006) were produced as recombinant proteins in bacteria and used in a mant-GTP fluorescence kinetics assay (Bouquier et al., 2009). In this assay, the DHR2 domains of DOCK9, DOCK10, and DOCK11 promoted GDP/GTP exchange on Cdc42 and not on RhoA, as expected from previous data (Figure 6A and Supplemental Figure S3A; Cote and Vuori, 2002; Lin et al., 2006). Surprisingly, DOCK10 DHR2 was also able to activate directly Rac1, in contrast to the other two GEFs, establishing DOCK10 as the sole dual-specificity RhoGEF of the DOCK-D subfamily (Figure 6B).FIGURE 6:DOCK10 is a GEF for Rac1 and Cdc42 in vitro and in cells. (A, B) In vitro GEF assay performed on Cdc42 (A) and Rac1 (B), using GST-DHR2 domains of DOCK9, DOCK10, and DOCK11. GST-Dbl and GST-Trio GEF1 were used as positive control GEFs for Cdc42 and Rac1, respectively, whereas GST alone was used as negative control. Results are expressed as relative fluorescence units (RFU) vs. time. (C–E) Active GTPase pull-down assays. GTP-loaded Cdc42 (C) or GTP-loaded Rac1 (D, E) were pulled down from protein lysates of HEK293 cells expressing wild-type pEGFP-DOCK9, -10 or -11 constructs or DOCK10 GD. GFP vector was used as a negative control. Shown are representative experiments. GTP-bound and total Cdc42 or Rac1 were detected by Western blot, using anti-Cdc42 and anti-Rac1 antibodies, respectively. Protein expression in the cell lysates was verified by immunoblotting with an anti-GFP antibody. Histograms underneath each Western blot show the quantification of the corresponding GTPase activation assays. The ratio of GTP-Rac1 (or Cdc42) over total Rac1 (or Cdc42) was calculated from at least three independent experiments. Error bars indicate SEM, **p < 0.003, ***p < 0.0001.To confirm these in vitro data in intact cells, we performed Cdc42-GTP, Rac1-GTP, and RhoA-GTP pull-down experiments, using the Cdc42/Rac1 interactive binding (CRIB) domain of WASP and PAK1 and the RhoA-binding domain (RBD) of Rhotekin, respectively (Figure 6, C– E, and Supplemental Figure S3B). Cdc42-GTP, but not RhoA-GTP, was efficiently pulled down from HEK293T cells expressing all three full-length DOCK proteins (Figure 6C and Supplemental Figure S3B). Rac1-GTP, in contrast, was only and very efficiently pulled-down from cells expressing full-length DOCK10 (Figure 6D), thus confirming the in vitro data. To rule out the possibility that this activation of Rac1 by DOCK10 in cells was indirect, we used the GEF-dead point mutant of DOCK10 (DOCK10 GD) described earlier. When expressed in cells, this mutant was unable to elicit the robust activation of Rac1 observed with wild-type DOCK10 (Figure 6E). Thus these data show that DOCK10 is a bona fide GEF for Rac1.Taken together, our results reveal DOCK10 as a dual-specific GEF, activating both Cdc42 and Rac1 in vitro and in cells, whereas the other subfamily members, DOCK9 and DOCK11, are strictly specific for Cdc42.

DOCK10 is a GEF for Rac1 and for Cdc42 in vitro and in cells

Both Cdc42 and Rac1 GTPases are involved in different aspects of spine development, promoting growth and/or stability of dendritic spines (Tada and Sheng, 2006). This prompted us to ask whether Cdc42 and Rac1 could be targets of DOCK10 in this process. Intriguingly, all DOCK-D subfamily GEFs, including DOCK10, have been reported as Cdc42-specific GEFs. In nonneuronal cell lines, DOCK10 has been described as a GEF specific for Cdc42 by both GTPase-binding assays and active GTPase pull-down assays (Nishikimi et al., 2005; Gadea et al., 2008). However, the exchange activity of DOCK10 on Cdc42 has never been formally shown in vitro, in contrast to that of DOCK9 and DOCK11 (Cote and Vuori, 2002; Lin et al., 2006).

To clarify this issue, we performed in vitro GEF assays on recombinant Cdc42, Rac1, and RhoA (Figure 6, A and B, and Supplemental Figure S3A). The catalytic DHR2 domains of DOCK9, DOCK10, and DOCK11 (as defined by Cote and Vuori, 2006) were produced as recombinant proteins in bacteria and used in a mant-GTP fluorescence kinetics assay (Bouquier et al., 2009). In this assay, the DHR2 domains of DOCK9, DOCK10, and DOCK11 promoted GDP/GTP exchange on Cdc42 and not on RhoA, as expected from previous data (Figure 6A and Supplemental Figure S3A; Cote and Vuori, 2002; Lin et al., 2006). Surprisingly, DOCK10 DHR2 was also able to activate directly Rac1, in contrast to the other two GEFs, establishing DOCK10 as the sole dual-specificity RhoGEF of the DOCK-D subfamily (Figure 6B).

DOCK10 is a GEF for Rac1 and Cdc42 in vitro and in cells. (A, B) In vitro GEF assay performed on Cdc42 (A) and Rac1 (B), using GST-DHR2 domains of DOCK9, DOCK10, and DOCK11. GST-Dbl and GST-Trio GEF1 were used as positive control GEFs for Cdc42 and Rac1, respectively, whereas GST alone was used as negative control. Results are expressed as relative fluorescence units (RFU) vs. time. (C–E) Active GTPase pull-down assays. GTP-loaded Cdc42 (C) or GTP-loaded Rac1 (D, E) were pulled down from protein lysates of HEK293 cells expressing wild-type pEGFP-DOCK9, -10 or -11 constructs or DOCK10 GD. GFP vector was used as a negative control. Shown are representative experiments. GTP-bound and total Cdc42 or Rac1 were detected by Western blot, using anti-Cdc42 and anti-Rac1 antibodies, respectively. Protein expression in the cell lysates was verified by immunoblotting with an anti-GFP antibody. Histograms underneath each Western blot show the quantification of the corresponding GTPase activation assays. The ratio of GTP-Rac1 (or Cdc42) over total Rac1 (or Cdc42) was calculated from at least three independent experiments. Error bars indicate SEM, **p < 0.003, ***p < 0.0001.

To confirm these in vitro data in intact cells, we performed Cdc42-GTP, Rac1-GTP, and RhoA-GTP pull-down experiments, using the Cdc42/Rac1 interactive binding (CRIB) domain of WASP and PAK1 and the RhoA-binding domain (RBD) of Rhotekin, respectively (Figure 6, C– E, and Supplemental Figure S3B). Cdc42-GTP, but not RhoA-GTP, was efficiently pulled down from HEK293T cells expressing all three full-length DOCK proteins (Figure 6C and Supplemental Figure S3B). Rac1-GTP, in contrast, was only and very efficiently pulled-down from cells expressing full-length DOCK10 (Figure 6D), thus confirming the in vitro data. To rule out the possibility that this activation of Rac1 by DOCK10 in cells was indirect, we used the GEF-dead point mutant of DOCK10 (DOCK10 GD) described earlier. When expressed in cells, this mutant was unable to elicit the robust activation of Rac1 observed with wild-type DOCK10 (Figure 6E). Thus these data show that DOCK10 is a bona fide GEF for Rac1.

Taken together, our results reveal DOCK10 as a dual-specific GEF, activating both Cdc42 and Rac1 in vitro and in cells, whereas the other subfamily members, DOCK9 and DOCK11, are strictly specific for Cdc42.

Cdc42 GEF activity is required for DOCK10-mediated spine formationNext we asked which of these GTPases would be the target(s) of DOCK10 for its function on spine morphogenesis. To determine this, we transfected cultured hippocampal neurons at 8 DIV with DOCK10 and well-established shRNAs targeting Cdc42 or Rac1 (Momboisse et al., 2009) or with a control shRNA (shCtrl). As expected, expression of shRNAs targeting Rac1 or Cdc42 alone significantly decreased spine number along the dendrites as compared with shCtrl (Figure 7, A and B). Overexpression of DOCK10 (with shCtrl) increased spine density moderately but significantly (Figure 7, A and B), confirming its positive effect on spine formation. In contrast, coexpression of shCdc42 with DOCK10 markedly suppressed this effect, and this was not the case with shRac1. Taken together, these data suggest that DOCK10 signals essentially through Cdc42 for spine formation.FIGURE 7:DOCK10 regulates spine formation via Cdc42-N-WASP and -PAK3 pathways. (A) The effects of DOCK10 on spine formation are mediated by Cdc42. Quantification of dendritic spine density, measured as described in Figure 5B, of hippocampal neurons transfected with shRNAs alone targeting either Cdc42 or Rac1 (Momboisse et al., 2009; or control shRNA) or in combination with DOCK10. Appropriate experimental conditions (transfection at 8 DIV and fixation at 11 DIV) were used to quantify formation of spines. Error bars indicate SEM (n = 3 independent experiments); *p < 0.05, **p < 0.01 (Student's t test). (B) Representative micrographs of hippocampal neurons transfected as described in A. Scale bar, 1 μm. (C) Quantification of dendritic spine density, measured as described in Figure 5B, of hippocampal neurons transfected with N-WASP∆WA, N-WASPWA, or PAK3KD alone or in combination with DOCK10. Appropriate experimental conditions as in A were used to quantify formation of spines. Error bars indicate SEM (n = 3 independent experiments); **p < 0.01, ***p < 0.001 (Student's t test). (D) Quantification of dendritic spine density, measured as described in Figure 5B, of hippocampal neurons transfected with sh Ctrl or shA-DOCK10 in combination with N-WASP or PAK3. Appropriate experimental conditions as in A were used to quantify formation of spines. Error bars indicate SEM (n = 3 independent experiments), ***p < 0.001 (Student's t test).To confirm that DOCK10 acted through Cdc42 for its effect on spine formation, we tested whether dominant-negative forms of Cdc42-specific effectors, such as N-WASP and PAK3, were able to block DOCK10-mediated spinogenesis. N-WASP has been shown to mediate a direct connection between Cdc42 and the Arp2/3 complex for actin polymerization (Rohatgi et al., 1999). Group 1 PAK kinases are effectors of both Cdc42 and Rac1, but PAK3 has been shown to bind preferentially Cdc42 (Kreis et al., 2007). Both N-WASP and PAK3 are critical regulators of spine morphogenesis (Kreis et al., 2007; Wegner et al., 2008). As shown previously, expression of N-WASP∆WA, a construct encoding N-WASP deleted of its C-terminal WA domain and thus unable to bind the Arp2/3 complex and G-actin (Supplemental Figure S4; Wegner et al., 2008), or expression of this domain alone (N-WASP WA) affected the number of spines in cultured hippocampal neurons (Figure 7C). Expression of a kinase-dead form of PAK3 (PAK3 KD; Supplemental Figure S4) also affected spine number (Figure 7C). Strikingly, coexpression of DOCK10 with either N-WASP∆WA or N-WASP WA or PAK3 KD impaired DOCK10's effect on spine formation (Figure 7C).To demonstrate further that DOCK10 activated Cdc42 to promote the formation of spines, we examined whether the poor spine morphology induced by DOCK10 depletion could be rescued by coexpressing wt N-WASP or PAK3. Expression of either N-WASP or PAK3 nicely rescued the defect in spine formation due to DOCK10 depletion (Figure 7D) to a similar extent as did reintroduction of DOCK10shR (Figure 5F).Combined, these data argue for N-WASP– and PAK3–mediated pathways downstream of DOCK10/Cdc42 functioning in spine morphogenesis.

Cdc42 GEF activity is required for DOCK10-mediated spine formation

Next we asked which of these GTPases would be the target(s) of DOCK10 for its function on spine morphogenesis. To determine this, we transfected cultured hippocampal neurons at 8 DIV with DOCK10 and well-established shRNAs targeting Cdc42 or Rac1 (Momboisse et al., 2009) or with a control shRNA (shCtrl). As expected, expression of shRNAs targeting Rac1 or Cdc42 alone significantly decreased spine number along the dendrites as compared with shCtrl (Figure 7, A and B). Overexpression of DOCK10 (with shCtrl) increased spine density moderately but significantly (Figure 7, A and B), confirming its positive effect on spine formation. In contrast, coexpression of shCdc42 with DOCK10 markedly suppressed this effect, and this was not the case with shRac1. Taken together, these data suggest that DOCK10 signals essentially through Cdc42 for spine formation.

DOCK10 regulates spine formation via Cdc42-N-WASP and -PAK3 pathways. (A) The effects of DOCK10 on spine formation are mediated by Cdc42. Quantification of dendritic spine density, measured as described in Figure 5B, of hippocampal neurons transfected with shRNAs alone targeting either Cdc42 or Rac1 (Momboisse et al., 2009; or control shRNA) or in combination with DOCK10. Appropriate experimental conditions (transfection at 8 DIV and fixation at 11 DIV) were used to quantify formation of spines. Error bars indicate SEM (n = 3 independent experiments); *p < 0.05, **p < 0.01 (Student's t test). (B) Representative micrographs of hippocampal neurons transfected as described in A. Scale bar, 1 μm. (C) Quantification of dendritic spine density, measured as described in Figure 5B, of hippocampal neurons transfected with N-WASP∆WA, N-WASPWA, or PAK3KD alone or in combination with DOCK10. Appropriate experimental conditions as in A were used to quantify formation of spines. Error bars indicate SEM (n = 3 independent experiments); **p < 0.01, ***p < 0.001 (Student's t test). (D) Quantification of dendritic spine density, measured as described in Figure 5B, of hippocampal neurons transfected with sh Ctrl or shA-DOCK10 in combination with N-WASP or PAK3. Appropriate experimental conditions as in A were used to quantify formation of spines. Error bars indicate SEM (n = 3 independent experiments), ***p < 0.001 (Student's t test).

To confirm that DOCK10 acted through Cdc42 for its effect on spine formation, we tested whether dominant-negative forms of Cdc42-specific effectors, such as N-WASP and PAK3, were able to block DOCK10-mediated spinogenesis. N-WASP has been shown to mediate a direct connection between Cdc42 and the Arp2/3 complex for actin polymerization (Rohatgi et al., 1999). Group 1 PAK kinases are effectors of both Cdc42 and Rac1, but PAK3 has been shown to bind preferentially Cdc42 (Kreis et al., 2007). Both N-WASP and PAK3 are critical regulators of spine morphogenesis (Kreis et al., 2007; Wegner et al., 2008). As shown previously, expression of N-WASP∆WA, a construct encoding N-WASP deleted of its C-terminal WA domain and thus unable to bind the Arp2/3 complex and G-actin (Supplemental Figure S4; Wegner et al., 2008), or expression of this domain alone (N-WASP WA) affected the number of spines in cultured hippocampal neurons (Figure 7C). Expression of a kinase-dead form of PAK3 (PAK3 KD; Supplemental Figure S4) also affected spine number (Figure 7C). Strikingly, coexpression of DOCK10 with either N-WASP∆WA or N-WASP WA or PAK3 KD impaired DOCK10's effect on spine formation (Figure 7C).

To demonstrate further that DOCK10 activated Cdc42 to promote the formation of spines, we examined whether the poor spine morphology induced by DOCK10 depletion could be rescued by coexpressing wt N-WASP or PAK3. Expression of either N-WASP or PAK3 nicely rescued the defect in spine formation due to DOCK10 depletion (Figure 7D) to a similar extent as did reintroduction of DOCK10shR (Figure 5F).

Combined, these data argue for N-WASP– and PAK3–mediated pathways downstream of DOCK10/Cdc42 functioning in spine morphogenesis.

DISCUSSIONCognitive functions, such as learning and memory, rely on proper morphogenesis of dendritic spines and plasticity of brain synapses. Their establishment is initiated by small, actin-rich protrusions called dendritic filopodia, which are long, thin protrusions abundantly present in developing neurons and can then be transformed morphologically and functionally into spines. In the adult, dendritic spines also undergo continuous remodeling of their shape and number upon sensory and emotional stimulus, which requires reorganization of their actin cytoskeleton. In this study, we show that the atypical RhoGEF DOCK10 is a novel regulator of the formation of dendritic spines in developing neurons. This work is the first report of a neuronal function of DOCK10, whose activity has been implicated so far only in amoeboid invasion of melanoma cells (Gadea et al., 2008). DOCK10 is expressed in postnatal developing Purkinje neurons, when the dendritic tree and spines are being formed. Consistently, we show that DOCK10 is essential for the formation of the spines, not only in Purkinje cells, but also in cultured hippocampal neurons, suggesting that DOCK10 may function as a general regulator of spine morphogenesis. In support of this, a recent human genetics study showed that deletion of the DOCK10 gene, together with the presence of a hemizygous missense variant, is associated with autism spectrum disorders (ASDs; Nava et al., 2013). ASDs are neuropsychiatric diseases that are often characterized by developmental alterations of spines and loss of synaptic plasticity, and many of the genes that are mutated in ASDs are crucial components of the activity-dependent signaling networks that regulate synapse development and plasticity.By controlling actin cytoskeleton dynamics, Rac1 and Cdc42 signaling pathways have been shown to play a key role in regulating spine formation and/or stability in hippocampal neurons (Vadodaria et al., 2013). Cdc42 is important for neural stem/progenitor cell proliferation, initial dendritic development, and spine maturation, whereas Rac1 regulates late steps of dendritic growth and spine maturation. These pathways are regulated by several different signaling complexes, including protein kinases and RhoGEFs. Indeed, RhoGEFs belonging to the Dbl family, mostly the Rac1-specific GEFs such as kalirin-7, βPIX, and Tiam-1, but also the Cdc42-specific intersectin, have been reported for some time now to be localized to dendritic spines and to be implicated in spine development and synaptic plasticity in hippocampal neurons (for review see Kiraly et al., 2010; Tolias et al., 2011). Only recently have DOCK-family RhoGEFs—that is, the Rac1-specific DOCK180 and DOCK4—emerged as being also required in the process of spine formation (Kuramoto et al., 2009; Kim et al., 2011; Ueda et al., 2013). In the case of DOCK180, a signaling module composed of RhoG/ELMO1/DOCK180 has been shown to regulate spine morphogenesis in hippocampal neurons by activating Rac1 (Kim et al., 2011). DOCK4, which also interacts with ELMO2 to activate Rac1, requires an association with cortactin to mediate its effect on hippocampal spine formation (Ueda et al., 2013). Intriguingly, we detected no increase in mRNA expression of either of these two GEFs during Purkinje cell dendritic development. Among the DOCK-D subfamily members, only DOCK9 has been shown to be involved in dendritic formation, but no function in spine morphogenesis has been described (Kuramoto et al., 2009). Our findings now add the RhoGEF DOCK10 to the picture, localizing to dendritic spines and acting as an essential regulator of spine morphogenesis in both the hippocampus and the cerebellum. Our work shows that DOCK10 regulates spine formation by promoting actin polymerization via Cdc42 activation, although we cannot totally exclude a function of Rac1 in this process, as, for example, during spine maintenance. Our data suggest that N-WASP and PAK3 both mediate spine formation downstream of DOCK10-Cdc42. Indeed, N-WASP truncation mutants and PAK3 KD mutant block DOCK10's effect on spine formation, and both wt N-WASP and PAK3 rescue the defects induced by DOCK10 depletion. By activating the actin nucleation complex Arp2/3, N-WASP thus provides a direct connection between DOCK10/Cdc42 and actin cytoskeleton remodeling in spines. This is in agreement with previous findings in melanoma cells showing that N-WASP and DOCK10 are found in the same complex during amoeboid motility (Gadea et al., 2008). PAK3 controls actin cytoskeleton remodeling through LIM kinase–mediated cofilin phosphorylation (Ba et al., 2013), thus linking the DOCK10/Cdc42 module to actin cytoskeleton remodeling in spines. Together, these data argue for N-WASP– and PAK3-mediated pathways downstream of DOCK10/Cdcd42 functioning in spine morphogenesis.DOCK-family RhoGEFs usually exhibit high specificity toward either Rac1 or Cdc42 as compared with the RhoGEFs of the Dbl family, which can often activate several GTPases. Of interest, our data demonstrate that DOCK10 differs from the other two members of the DOCK-D subfamily of GEFs, as it acts as a dual-specificity GEF activating both Rac1 and Cdc42 in vitro and in cells. All three members of the DOCK-D subfamily of GEFs, DOCK9/10/11, have been reported to be specific for Cdc42 (Nishikimi et al., 2005; Lin et al., 2006). DOCK10 therefore emerges as a peculiar GEF of this subfamily. The molecular mechanisms of GDP-GTP exchange and the nature of the determinants responsible for the specificity of DOCK proteins toward their GTPase targets have been proposed following the recent elucidation of the structure of the DHR-2 domains of DOCK9 and DOCK2 in complex with Cdc42 and Rac1, respectively (Yang et al., 2009; Kulkarni et al., 2011). The selectivity of the DHR-2 domain is conferred mainly by two regions in the GTPase—a Phe or Trp residue in position 56 and an Ala or Lys residue at position 27 for Cdc42 and Rac1, respectively. However, the relative divergence between DHR-2 domains precludes an easy identification of the amino acids of the DHR-2 domain participating in the GTPase specificity. For example, Leu-1941 in the Cdc42-specific DOCK9 is a Met in the Rac1-GEF DOCK2, and these amino acids contact Phe-56 or Trp-56 of the Cdc42 and Rac1 GTPases, respectively. However, these residues are not sufficient to account by themselves for the GTPase selectivity, since DOCK10, which activates both Cdc42 and Rac1, harbors a Leu at this position, like DOCK9. This suggests that multiple, crucial amino acids are required for the GTPase selectivity of DHR-2 domains. Further studies are required to define these residues.How DOCK10 activity is regulated remains an open question as well. On a molecular level, most structure–function analyses have been conducted on the DOCK-A and -B subfamilies, whose regulation involves their N-terminal SH3 domain (Lu et al., 2005). Like the other DOCK-D subfamily members, DOCK10 harbors a PH domain, instead of an SH3 domain, within its N-terminus. PH domains have been shown to be important for modulating GEF activity and localization of Dbl-family GEFs via interaction with proteins or phosphoinositides. The precise function of this domain in DOCK-D family GEFs remains to be determined. In addition, the DHR-1 domain may target DOCK10 to the plasma membrane, as has been described for the prototypic DOCK180 GEF (Cote et al., 2005). This could represent a means to specifically localize DOCK10 at the postsynaptic density under the surface membrane of spine heads, for example. Consistently, we detected DOCK10 in the membrane and synaptosomal fractions of brain extracts. In addition, we observed an increased expression of DOCK10 protein during PC development, which could then account for increased activity of the protein in the spines. Upstream signaling pathways that would impinge on and activate DOCK10 also remain unknown, as is the case for the other DOCK-family RhoGEFs, and their identification is essential to gain insights into the overall neuronal spectrum of activity and regulation of DOCK10. Possible candidate upstream regulators of DOCK10 that could be tested include neurotransmitter receptors such as N-methyl-d-aspartate or α-amino-3-hydroxy-5-methyl-4-isoxazolepropionic acid receptors or the EphB receptor, which have been shown, for example, to translocate the Dbl-RhoGEFs Tiam1 and Kalirin-7 to the postsynaptic region and enhance their GEF activity (Penzes et al., 2003; Tolias et al., 2005, 2007).In conclusion, the atypical RhoGEF DOCK10 emerges as a novel, crucial regulator of dendritic spine morphogenesis in the hippocampus and the cerebellum, and, given its strong expression in several other brain areas, it could potentially play additional roles in neuronal development that remain to be explored.

Cognitive functions, such as learning and memory, rely on proper morphogenesis of dendritic spines and plasticity of brain synapses. Their establishment is initiated by small, actin-rich protrusions called dendritic filopodia, which are long, thin protrusions abundantly present in developing neurons and can then be transformed morphologically and functionally into spines. In the adult, dendritic spines also undergo continuous remodeling of their shape and number upon sensory and emotional stimulus, which requires reorganization of their actin cytoskeleton. In this study, we show that the atypical RhoGEF DOCK10 is a novel regulator of the formation of dendritic spines in developing neurons. This work is the first report of a neuronal function of DOCK10, whose activity has been implicated so far only in amoeboid invasion of melanoma cells (Gadea et al., 2008). DOCK10 is expressed in postnatal developing Purkinje neurons, when the dendritic tree and spines are being formed. Consistently, we show that DOCK10 is essential for the formation of the spines, not only in Purkinje cells, but also in cultured hippocampal neurons, suggesting that DOCK10 may function as a general regulator of spine morphogenesis. In support of this, a recent human genetics study showed that deletion of the DOCK10 gene, together with the presence of a hemizygous missense variant, is associated with autism spectrum disorders (ASDs; Nava et al., 2013). ASDs are neuropsychiatric diseases that are often characterized by developmental alterations of spines and loss of synaptic plasticity, and many of the genes that are mutated in ASDs are crucial components of the activity-dependent signaling networks that regulate synapse development and plasticity.

By controlling actin cytoskeleton dynamics, Rac1 and Cdc42 signaling pathways have been shown to play a key role in regulating spine formation and/or stability in hippocampal neurons (Vadodaria et al., 2013). Cdc42 is important for neural stem/progenitor cell proliferation, initial dendritic development, and spine maturation, whereas Rac1 regulates late steps of dendritic growth and spine maturation. These pathways are regulated by several different signaling complexes, including protein kinases and RhoGEFs. Indeed, RhoGEFs belonging to the Dbl family, mostly the Rac1-specific GEFs such as kalirin-7, βPIX, and Tiam-1, but also the Cdc42-specific intersectin, have been reported for some time now to be localized to dendritic spines and to be implicated in spine development and synaptic plasticity in hippocampal neurons (for review see Kiraly et al., 2010; Tolias et al., 2011). Only recently have DOCK-family RhoGEFs—that is, the Rac1-specific DOCK180 and DOCK4—emerged as being also required in the process of spine formation (Kuramoto et al., 2009; Kim et al., 2011; Ueda et al., 2013). In the case of DOCK180, a signaling module composed of RhoG/ELMO1/DOCK180 has been shown to regulate spine morphogenesis in hippocampal neurons by activating Rac1 (Kim et al., 2011). DOCK4, which also interacts with ELMO2 to activate Rac1, requires an association with cortactin to mediate its effect on hippocampal spine formation (Ueda et al., 2013). Intriguingly, we detected no increase in mRNA expression of either of these two GEFs during Purkinje cell dendritic development. Among the DOCK-D subfamily members, only DOCK9 has been shown to be involved in dendritic formation, but no function in spine morphogenesis has been described (Kuramoto et al., 2009). Our findings now add the RhoGEF DOCK10 to the picture, localizing to dendritic spines and acting as an essential regulator of spine morphogenesis in both the hippocampus and the cerebellum. Our work shows that DOCK10 regulates spine formation by promoting actin polymerization via Cdc42 activation, although we cannot totally exclude a function of Rac1 in this process, as, for example, during spine maintenance. Our data suggest that N-WASP and PAK3 both mediate spine formation downstream of DOCK10-Cdc42. Indeed, N-WASP truncation mutants and PAK3 KD mutant block DOCK10's effect on spine formation, and both wt N-WASP and PAK3 rescue the defects induced by DOCK10 depletion. By activating the actin nucleation complex Arp2/3, N-WASP thus provides a direct connection between DOCK10/Cdc42 and actin cytoskeleton remodeling in spines. This is in agreement with previous findings in melanoma cells showing that N-WASP and DOCK10 are found in the same complex during amoeboid motility (Gadea et al., 2008). PAK3 controls actin cytoskeleton remodeling through LIM kinase–mediated cofilin phosphorylation (Ba et al., 2013), thus linking the DOCK10/Cdc42 module to actin cytoskeleton remodeling in spines. Together, these data argue for N-WASP– and PAK3-mediated pathways downstream of DOCK10/Cdcd42 functioning in spine morphogenesis.

DOCK-family RhoGEFs usually exhibit high specificity toward either Rac1 or Cdc42 as compared with the RhoGEFs of the Dbl family, which can often activate several GTPases. Of interest, our data demonstrate that DOCK10 differs from the other two members of the DOCK-D subfamily of GEFs, as it acts as a dual-specificity GEF activating both Rac1 and Cdc42 in vitro and in cells. All three members of the DOCK-D subfamily of GEFs, DOCK9/10/11, have been reported to be specific for Cdc42 (Nishikimi et al., 2005; Lin et al., 2006). DOCK10 therefore emerges as a peculiar GEF of this subfamily. The molecular mechanisms of GDP-GTP exchange and the nature of the determinants responsible for the specificity of DOCK proteins toward their GTPase targets have been proposed following the recent elucidation of the structure of the DHR-2 domains of DOCK9 and DOCK2 in complex with Cdc42 and Rac1, respectively (Yang et al., 2009; Kulkarni et al., 2011). The selectivity of the DHR-2 domain is conferred mainly by two regions in the GTPase—a Phe or Trp residue in position 56 and an Ala or Lys residue at position 27 for Cdc42 and Rac1, respectively. However, the relative divergence between DHR-2 domains precludes an easy identification of the amino acids of the DHR-2 domain participating in the GTPase specificity. For example, Leu-1941 in the Cdc42-specific DOCK9 is a Met in the Rac1-GEF DOCK2, and these amino acids contact Phe-56 or Trp-56 of the Cdc42 and Rac1 GTPases, respectively. However, these residues are not sufficient to account by themselves for the GTPase selectivity, since DOCK10, which activates both Cdc42 and Rac1, harbors a Leu at this position, like DOCK9. This suggests that multiple, crucial amino acids are required for the GTPase selectivity of DHR-2 domains. Further studies are required to define these residues.

How DOCK10 activity is regulated remains an open question as well. On a molecular level, most structure–function analyses have been conducted on the DOCK-A and -B subfamilies, whose regulation involves their N-terminal SH3 domain (Lu et al., 2005). Like the other DOCK-D subfamily members, DOCK10 harbors a PH domain, instead of an SH3 domain, within its N-terminus. PH domains have been shown to be important for modulating GEF activity and localization of Dbl-family GEFs via interaction with proteins or phosphoinositides. The precise function of this domain in DOCK-D family GEFs remains to be determined. In addition, the DHR-1 domain may target DOCK10 to the plasma membrane, as has been described for the prototypic DOCK180 GEF (Cote et al., 2005). This could represent a means to specifically localize DOCK10 at the postsynaptic density under the surface membrane of spine heads, for example. Consistently, we detected DOCK10 in the membrane and synaptosomal fractions of brain extracts. In addition, we observed an increased expression of DOCK10 protein during PC development, which could then account for increased activity of the protein in the spines. Upstream signaling pathways that would impinge on and activate DOCK10 also remain unknown, as is the case for the other DOCK-family RhoGEFs, and their identification is essential to gain insights into the overall neuronal spectrum of activity and regulation of DOCK10. Possible candidate upstream regulators of DOCK10 that could be tested include neurotransmitter receptors such as N-methyl-d-aspartate or α-amino-3-hydroxy-5-methyl-4-isoxazolepropionic acid receptors or the EphB receptor, which have been shown, for example, to translocate the Dbl-RhoGEFs Tiam1 and Kalirin-7 to the postsynaptic region and enhance their GEF activity (Penzes et al., 2003; Tolias et al., 2005, 2007).

In conclusion, the atypical RhoGEF DOCK10 emerges as a novel, crucial regulator of dendritic spine morphogenesis in the hippocampus and the cerebellum, and, given its strong expression in several other brain areas, it could potentially play additional roles in neuronal development that remain to be explored.

MATERIALS AND METHODSDNA constructsIMAGE clones for mouse DOCK10 and DOCK9 were obtained from Source Bioscience (Biovalley, Marne la Vallée, France). DOCK11 plasmid was a kind gift of Mitsuo Maruyama (Aichi, Japan). The cDNAs were subcloned into pEGFP or pmCherry vectors by PCR amplification, flanking the construct with restriction enzymes appropriate for subcloning. The DHR-2 domains of DOCK9/10/11 were subcloned into the pEGFP or the pGEX vector by PCR amplification of the appropriate fragment of the three clones. The GEF-dead DOCK10 mutant was generated by introducing a point mutation at V2055A in the wild-type form of DOCK10 using the QuikChange Site-Directed Mutagenesis Kit (Agilent Technologies, Les Ulis, France) according to the manufacturer's instructions. The shRNA-resistant mutant of DOCK10 was created by introducing four point mutations in the DNA sequence of DOCK10 targeted by shA to render it unrecognizable by the shA shRNA. Details on cloning procedures and on primer sequences can be obtained upon request. RFP-PAK3 and RFP-PAK3 K297L (kinase-dead mutant) were kind gifts of Jean-Vianney Barnier (Centre National de la Recherche Scientifique, Unité Mixte de Recherche 8145, Paris, France). mCherry N-WASP, N-WASP ΔWA, and N-WASP WA construct were kindly provided by Nathalie Morin (Centre de Recherche de Biochimie Macromoléculaire, Montpellier, France) and Michael Way (Cancer Research UK, London Research Institute, London, United Kingdom).AntibodiesThe following primary antibodies were purchased: anti-GFP polyclonal antibody (Clinisciences, Nanterre, France); anti-DOCK10 polyclonal antibody (Bethyl Laboratories, Montgomery, TX); anti–α-tubulin monoclonal antibody (Sigma-Aldrich, St. Louis, MO); anti-Rac1 and anti-Cdc42 monoclonal antibodies (BD Biosciences, Le Pont de Claix, France); anti-calbindin monoclonal antibody (Swant, Marly, Switzerland); anti-PSD95 monoclonal antibody (clone 7E3; Santa Cruz Biotechnology, Dallas, TX); anti-Bassoon monoclonal antibody (Enzo Life Sciences, Villeurbanne, France); anti–Homer-1 monoclonal antibody (Synaptic Systems, Göttingen, Germany); and anti-Flag M2 monoclonal antibody (Sigma-Aldrich).The following secondary antibodies were used: Alexa Fluor 350–, 488–, 546–, and 594–conjugated anti-rabbit/mouse antibodies (Invitrogen, Life Technologies, St-Aubin, France) for immunofluorescence studies and anti-mouse or anti-rabbit immunoglobulin G coupled to horseradish peroxidase for Western immunoblotting (GE Healthcare, Villacoubley, France). Hoechst stain was used to detect nuclei.AnimalsMale and female Pcp2-GFP or wild-type C57BL/6 mice (Jackson Laboratories, Bar Harbor, ME) and Swiss mice (Janvier Labs, St. Berthevin, France) used for organotypic cultures and for primary cultures of hippocampal neurons were housed at the animal house facility of the Institut de Génétique Moléculaire de Montpellier (Montpellier, France) or of the Integrative Biology Institute (Paris, France). Animals had ad libitum access to food and water, with a 12-h light–dark cycle.Purkinje cell sortingPurkinje cells were isolated from Pcp2-GFP mice as described (Tomomura et al., 2001). Briefly, cerebella were removed from mice at the indicated ages. Cubes (0.5 mm3) of cerebella were digested for 10 min at 37°C with 0.025% trypsin (type I; Sigma-Aldrich) in dissociation solution consisting of Ca2+-free Hank's balanced salt solution containing 3 mg/ml bovine serum albumin (BSA), 15 mM 4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid (HEPES), 1.5 mM MgSO4, and 3 mg/ml glucose (pH 7.4). The enzymatic reaction was stopped by the addition of dissociation solution containing 0.25 mg/ml soybean trypsin inhibitor, 40 μg/ml DNase I, 50 μM d-2-amino-5-phosphonovaleric acid (APV), 20 μM 6,7-dinitroquinoxaline-2,3 (1H, 4H)-dione (DNQX), and 0.1 μM tetrodotoxin (TTX) (all from Sigma-Aldrich). Tissues were triturated mildly by sequential passage through wide-bore and fine-tipped pipettes. Cells were filtered through a 40-μm nylon mesh and resuspended in Ca2+- and Mg2+-free dissociation solution at a final concentration of 5 × 106 cells/ml. To label dead cells, Propidium iodide (PI; Sigma-Aldrich) was added at a final concentration of 2 μg/ml. Cell sorting was performed on a FACsAria machine. The sorting decision was based on the measurements of forward-scattered light, PI fluorescence, and GFP fluorescence.RNA preparation and RT-qPCR of purified Purkinje cellsTotal RNA was extracted with TRIzol reagent (Invitrogen) from ∼3 × 104 purified Purkinje cells at the indicated stages. The corresponding cDNAs were prepared by reverse transcription of 100 ng of RNA using the SuperScript III First-Strand Synthesis System (Invitrogen) with an oligo-dT primer according to the manufacturer's instructions. The resulting cDNAs were used as a template for RT-qPCR using a Light Cycler 480 thermocycler (384 plates; Roche Diagnostics, Meylan, France) with a home-made SYBR Green QPCR master mix (Lutfalla and Uze, 2006). Thermal cycling parameters were 2 min at 95°C, followed by 45 cycles of 95°C for 10 s, 64°C for 15 s, and 72°C for 25 s. The relative quantification in gene expression was determined using the ΔΔCt method. To normalize expression data, primers for 10 commonly used housekeeping genes were used, and the normalization factor was determined using the geNorm software, as described in Vandesompele et al. (2002). This led to the selection of the following internal control genes in our assays: glyceraldehyde-3-phosphate dehydrogenase, β-actin, and hypoxanthine phosphoribosyltransferase 1. Sequences of the primers used are listed in Supplemental Table S1.ImmunohistochemistryFor immunohistochemistry, mice were perfused transcardially with 4% paraformaldehyde (PFA) in 0.12 M phosphate buffer, pH 7.4. Brains were removed, postfixed overnight in 4% PFA, cryoprotected in 20% and then 30% sucrose, embedded in OCT, frozen in isopentane (−55°C), and stored at −80°C. Sagittal sections (10 μm) were cut with a cryostat and stored at −20°C before immunostaining. Sections were rehydrated in phosphate-buffered saline (PBS) for 5 min and then fixed and permeabilized with 70% methanol/30% acetone for 15 min at −20°C. After washes in PBS, slices were blocked for 30 min in PBS containing 1% BSA and incubated with primary antibodies diluted in PBS/1% BSA for 2 h at room temperature. After washes in PBS containing 0.25% Tween-20, slices were incubated with secondary antibodies diluted in PBS/1% BSA for 45 min at room temperature. After washes, sections were mounted with Mowiol.Recombinant lentiviral vectors: construction and productionPlasmid constructionsTo inhibit DOCK10 expression, two shRNAs directed against DOCK10 mRNA were selected for their inhibitory efficiency. Their corresponding DNA sequences are as follows:ShA:5′-GATCCCCGCACAGAGCTGAATCCTATTTCAAG­AG­AATAGGATTCAGCTCTGTGCTTTTTA-3′ShB:5′-GATCCCCCAAGGCACGGAATATAACTTTCAAGAGAAG­TTATATTCCGTGCCTTGTTTTTA-3′These two sequences were inserted as a DNA cassette under the control of the polymerase III-dependent H1 promoter into the pFlap-PGK-EGFP-ΔU3 vector (Santamaria et al., 2009), generating the pFlap-H1-shRNA-PGK-EGFP-ΔU3 series of vectors hereafter named shA-DOCK10 and shB-DOCK10. The scrambled control shRNA vector, shScr, corresponding to a random shA-DOCK10 sequence (5′-GATCCCCGGCAAATTAGCCACGTACTTTCAAGAGAAGTACGTGGCTAATTTCGGTTTTTA-3′) and a nonrelevant control shRNA vector, Lv-shRd, consisting of a random sequence (5′-GATCCCCTCGTCATAGCGTGCATAGGTTCAAGAGACCTATGCACGCTATGACGATTTTTGGAAA-3′) were used as controls and collectively referred to as shCtrl throughout the study.Lentiviral productionStocks of viral particles were prepared as described previously (Zennou et al., 2001) by transient cotransfection of HEK293T cells with the p8.91 encapsidation plasmid (Zufferey et al., 1997), the pHCMV-G (vesicular stomatitis virus pseudotype) envelope plasmid, and the pFlap vectors described earlier. Briefly, the supernatants were collected 48 h after transfection, treated with DNase I (Roche Diagnostics), and filtered before ultracentrifugation. Viral pellets were then resuspended in PBS, aliquoted, and stored at −80°C until use. The amount of p24 capsid protein was determined by the HIV-1 p24 ELISA antigen assay (Beckman Coulter, Fullerton, CA). Viruses from different productions averaged 300 ng/μl of p24 antigen and 3 × 108 transducing units/ml.Organotypic slice culture, lentivirus-mediated transduction, and immunostainingCerebellar organotypic cultures were set up from newborn (P0) mice as previously described (Lebrun et al., 2013). Mice were decapitated, and their brains were dissected out in cold Gey's balanced salt solution (Invitrogen) supplemented with 5 mg/ml glucose. Cerebellar parasagittal slices (350 μm thick) were cut on a McIlwain tissue chopper and transferred onto 30-mm-diameter Millipore culture inserts with 0.4-μm pores (Millicell; Millipore, St. Quentin en Yvelines, France). Slices were maintained in culture in six-well plates containing 1 ml of nutrient medium/well at 37°C under a humidified atmosphere containing 5% CO2. The nutrient medium consisted of basal medium with Earle's salts (BME; Invitrogen) containing 1 mM l-glutamine, 5 mg/ml glucose, 0.5 mg/ml BSA (A-4503; Sigma-Aldrich), and supplements including 5 μg/ml insulin, 5 μg/ml transferrin, and 5 μg/ml sodium selenite (I-1884; Sigma-Aldrich). Lentiviral vectors (0.5–2 μl/slice, depending on the lentiviral production) were applied 2–4 h after culture processing of P0 slices. The medium was replaced every 2–3 d.Slices were fixed with 4% PFA in phosphate buffer (0.1 M, pH 7.4) for 1 h at room temperature after 14 DIV. They were washed in PBS, incubated 1 h in PBS 0.25% Triton X-100 (PBST), 0.1 M gelatin (PBSTG), and 0.1% sodium azide and then incubated with primary antibodies in PBSTG overnight at room temperature. After washing in PBST, slices were incubated with secondary antibodies in PBSTG for 2 h at room temperature, washed in PBST, and mounted in Mowiol.Semiautomated image analysis of Purkinje cellsPhotomicrographs of isolated Purkinje cells were acquired (Leica wide-field DM6000 microscope, 40× oil objective), and the conditions (experimental procedure, transduced or not transduced) were recorded for each Purkinje cell in a file. Then a study blinded to the experimental procedures was performed to categorize and analyze the dendritic tree of Purkinje cells using ImagePro software. For each Purkinje cell, the following variables were measured: size of the soma, number of branch points in the dendritic arbor, and length of the longest dendrite connecting to a given cell body. An algorithm was proposed to classify the different types of dendritic tree of Purkinje cells into three groups: atrophic, when the size of the longest dendrite was inferior to the soma diameter; stellate, when more than six primary dendrites of about the same size emerged from the cell body; and elaborate, including all the other types. Among the Purkinje cells presenting an elaborate dendritic tree, we distinguished two types of Purkinje cells: those with spines—that is, presenting at least one part of their dendritic tree covered with spines—and those we called with no spines. On these elaborate Purkinje cells, a Sholl analysis was performed to estimate the distribution and complexity of the dendrites (Gutierrez and Davies, 2007). The Sholl analysis consisted of 1) constructing concentric and equidistantly (10 μm) organized spherical shells centered at the cell body, and 2) counting the intersections of dendrites with the circles of increasing radius. To quantify dendritic length, branch points, and Sholl analysis, Perl scripts were developed for parsing intermediate outputs.Primary cell culture and transfection of hippocampal neuronsNeuronal hippocampal cultures were prepared from embryonic day 17.5 C57BL/6 mice and grown in Neurobasal medium supplemented with B27 and 10% fetal bovine serum (Raynaud et al., 2013). Neurons were transfected at 8 DIV with pSuper plasmids expressing the described shRNA sequences or plasmid DNA constructs of DOCK10, PAK3, or N-WASP and fixed at 11 DIV.Subcellular fractionationSynaptosomal fractions were prepared as follows: brains from killed P27 mice were placed in HEPES buffer (20 mM HEPES, pH 7.4, 0.32 M sucrose, 1 mM EDTA), subjected to Dounce homogenization, and subsequently centrifuged at 1500 × g for 5 min to remove nuclei and cell debris. The supernatant was centrifuged at 100,000 × g for 10 min. The resulting supernatant was saved (fraction S1: cytoplasmic fraction), and the pellet was resuspended in HEPES buffer. This homogenate was carefully layered on top of a 0.85 M sucrose buffer and centrifuged at 9000 × g for 25 min. The resulting upper phase P1 (small vesicles) and middle phase P2 (synaptosomes) were collected and subjected to centrifugation at 100,000 × g for 10 min, and the pellet fraction P3 (membranes) was solubilized in Lysis buffer (20 mM HEPES, pH 7.4, 100 mM NaCl, 2 mM EDTA, 1% Triton). The pellets resulting from the centrifuged P1 and P2 fractions were also solu­bilized in lysis buffer.Immunofluorescence microscopyHippocampal neurons were fixed and permeabilized as previously described (Raynaud et al., 2013). Untreated or shRNA-GFP–expressing neurons were visualized using a Zeiss Axioimager Z1 microscope. For brain slices, we used a Zeiss LMS780 confocal microscope with a 63×/1.4 numerical aperture (NA) oil objective or a Leica wide-field DM6000 microscope with a 100×/1.4 NA oil objective. Images were recorded with a charge-coupled device HQ2 camera (Roper Scientific, Martiensried, Germany) controlled by MetaMorph 7.1 (Universal Imaging).Morphometric analyses were performed in different fields from at least three different cultures using ImageJ software (National Institutes of Health, Bethesda, MD). Spines were defined as dendritic protrusions with a neck and a head. Dendrites were randomly selected and spines manually counted over a 50-μm length of dendrite. Data were then expressed as density of spines/10-μm length of dendrite. The same spines were used to measure the area of their head. Each spine was manually traced and the surface calculated using ImageJ. All experiments were conducted in a double-blinded manner, and dendritic spines were analyzed from at least 20 neurons from at least three independent cultures (n = 3 or 4 experiments, as indicated).In vitro GEF assaysFluorescence-based in vitro guanine-nucleotide exchange assays were performed using Mant-GTP (Molecular Probes, Life Technologies, St-Aubin, France) in an FLX 800 microplate fluorescence reader (BioTek Instruments, Colmar, France) at 25°C, as described (Bouquier et al., 2009). Nucleotide exchange was determined by measuring Mant-GTP loading on GDP-preloaded Rac1, Cdc42, or RhoA GTPases using 1 μM GEF (0.5 μM for Dbl). The relative Mant fluorescence (λex = 360 nm and λem = 460 nm) was monitored for 30 min, and measurements were taken every 15 s.Active GTPase pull downFor the Rac1-GTP glutathione S-transferase (GST) pull-down assay, the CRIB domain of PAK was chosen, as described previously (Briancon-Marjollet et al., 2008), whereas for Cdc42, the CRIB domain of N-WASP was used. Total HEK293T lysates and corresponding pull downs retained on GST-Sepharose beads were processed for Western immunoblotting. RhoA-GTP pull-down assays were performed similarly, using the RBD domain of Rhotekin (Bouquier et al., 2009).Statistical analysisThe distribution of Purkinje cell types was compared using the Pearson χ2 test or Fisher's exact test, depending on the amount. The Kruskal–Wallis test (nonparametric analysis of variance) was used to compare the Sholl index. All analyses were performed using the Statistical Package for the Social Sciences (SPSS), version 20.0. Significance values were set at p < 0.05.

MATERIALS AND METHODS

DNA constructsIMAGE clones for mouse DOCK10 and DOCK9 were obtained from Source Bioscience (Biovalley, Marne la Vallée, France). DOCK11 plasmid was a kind gift of Mitsuo Maruyama (Aichi, Japan). The cDNAs were subcloned into pEGFP or pmCherry vectors by PCR amplification, flanking the construct with restriction enzymes appropriate for subcloning. The DHR-2 domains of DOCK9/10/11 were subcloned into the pEGFP or the pGEX vector by PCR amplification of the appropriate fragment of the three clones. The GEF-dead DOCK10 mutant was generated by introducing a point mutation at V2055A in the wild-type form of DOCK10 using the QuikChange Site-Directed Mutagenesis Kit (Agilent Technologies, Les Ulis, France) according to the manufacturer's instructions. The shRNA-resistant mutant of DOCK10 was created by introducing four point mutations in the DNA sequence of DOCK10 targeted by shA to render it unrecognizable by the shA shRNA. Details on cloning procedures and on primer sequences can be obtained upon request. RFP-PAK3 and RFP-PAK3 K297L (kinase-dead mutant) were kind gifts of Jean-Vianney Barnier (Centre National de la Recherche Scientifique, Unité Mixte de Recherche 8145, Paris, France). mCherry N-WASP, N-WASP ΔWA, and N-WASP WA construct were kindly provided by Nathalie Morin (Centre de Recherche de Biochimie Macromoléculaire, Montpellier, France) and Michael Way (Cancer Research UK, London Research Institute, London, United Kingdom).

DNA constructs

IMAGE clones for mouse DOCK10 and DOCK9 were obtained from Source Bioscience (Biovalley, Marne la Vallée, France). DOCK11 plasmid was a kind gift of Mitsuo Maruyama (Aichi, Japan). The cDNAs were subcloned into pEGFP or pmCherry vectors by PCR amplification, flanking the construct with restriction enzymes appropriate for subcloning. The DHR-2 domains of DOCK9/10/11 were subcloned into the pEGFP or the pGEX vector by PCR amplification of the appropriate fragment of the three clones. The GEF-dead DOCK10 mutant was generated by introducing a point mutation at V2055A in the wild-type form of DOCK10 using the QuikChange Site-Directed Mutagenesis Kit (Agilent Technologies, Les Ulis, France) according to the manufacturer's instructions. The shRNA-resistant mutant of DOCK10 was created by introducing four point mutations in the DNA sequence of DOCK10 targeted by shA to render it unrecognizable by the shA shRNA. Details on cloning procedures and on primer sequences can be obtained upon request. RFP-PAK3 and RFP-PAK3 K297L (kinase-dead mutant) were kind gifts of Jean-Vianney Barnier (Centre National de la Recherche Scientifique, Unité Mixte de Recherche 8145, Paris, France). mCherry N-WASP, N-WASP ΔWA, and N-WASP WA construct were kindly provided by Nathalie Morin (Centre de Recherche de Biochimie Macromoléculaire, Montpellier, France) and Michael Way (Cancer Research UK, London Research Institute, London, United Kingdom).

AntibodiesThe following primary antibodies were purchased: anti-GFP polyclonal antibody (Clinisciences, Nanterre, France); anti-DOCK10 polyclonal antibody (Bethyl Laboratories, Montgomery, TX); anti–α-tubulin monoclonal antibody (Sigma-Aldrich, St. Louis, MO); anti-Rac1 and anti-Cdc42 monoclonal antibodies (BD Biosciences, Le Pont de Claix, France); anti-calbindin monoclonal antibody (Swant, Marly, Switzerland); anti-PSD95 monoclonal antibody (clone 7E3; Santa Cruz Biotechnology, Dallas, TX); anti-Bassoon monoclonal antibody (Enzo Life Sciences, Villeurbanne, France); anti–Homer-1 monoclonal antibody (Synaptic Systems, Göttingen, Germany); and anti-Flag M2 monoclonal antibody (Sigma-Aldrich).The following secondary antibodies were used: Alexa Fluor 350–, 488–, 546–, and 594–conjugated anti-rabbit/mouse antibodies (Invitrogen, Life Technologies, St-Aubin, France) for immunofluorescence studies and anti-mouse or anti-rabbit immunoglobulin G coupled to horseradish peroxidase for Western immunoblotting (GE Healthcare, Villacoubley, France). Hoechst stain was used to detect nuclei.

The following primary antibodies were purchased: anti-GFP polyclonal antibody (Clinisciences, Nanterre, France); anti-DOCK10 polyclonal antibody (Bethyl Laboratories, Montgomery, TX); anti–α-tubulin monoclonal antibody (Sigma-Aldrich, St. Louis, MO); anti-Rac1 and anti-Cdc42 monoclonal antibodies (BD Biosciences, Le Pont de Claix, France); anti-calbindin monoclonal antibody (Swant, Marly, Switzerland); anti-PSD95 monoclonal antibody (clone 7E3; Santa Cruz Biotechnology, Dallas, TX); anti-Bassoon monoclonal antibody (Enzo Life Sciences, Villeurbanne, France); anti–Homer-1 monoclonal antibody (Synaptic Systems, Göttingen, Germany); and anti-Flag M2 monoclonal antibody (Sigma-Aldrich).

The following secondary antibodies were used: Alexa Fluor 350–, 488–, 546–, and 594–conjugated anti-rabbit/mouse antibodies (Invitrogen, Life Technologies, St-Aubin, France) for immunofluorescence studies and anti-mouse or anti-rabbit immunoglobulin G coupled to horseradish peroxidase for Western immunoblotting (GE Healthcare, Villacoubley, France). Hoechst stain was used to detect nuclei.

AnimalsMale and female Pcp2-GFP or wild-type C57BL/6 mice (Jackson Laboratories, Bar Harbor, ME) and Swiss mice (Janvier Labs, St. Berthevin, France) used for organotypic cultures and for primary cultures of hippocampal neurons were housed at the animal house facility of the Institut de Génétique Moléculaire de Montpellier (Montpellier, France) or of the Integrative Biology Institute (Paris, France). Animals had ad libitum access to food and water, with a 12-h light–dark cycle.

Male and female Pcp2-GFP or wild-type C57BL/6 mice (Jackson Laboratories, Bar Harbor, ME) and Swiss mice (Janvier Labs, St. Berthevin, France) used for organotypic cultures and for primary cultures of hippocampal neurons were housed at the animal house facility of the Institut de Génétique Moléculaire de Montpellier (Montpellier, France) or of the Integrative Biology Institute (Paris, France). Animals had ad libitum access to food and water, with a 12-h light–dark cycle.

Purkinje cell sortingPurkinje cells were isolated from Pcp2-GFP mice as described (Tomomura et al., 2001). Briefly, cerebella were removed from mice at the indicated ages. Cubes (0.5 mm3) of cerebella were digested for 10 min at 37°C with 0.025% trypsin (type I; Sigma-Aldrich) in dissociation solution consisting of Ca2+-free Hank's balanced salt solution containing 3 mg/ml bovine serum albumin (BSA), 15 mM 4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid (HEPES), 1.5 mM MgSO4, and 3 mg/ml glucose (pH 7.4). The enzymatic reaction was stopped by the addition of dissociation solution containing 0.25 mg/ml soybean trypsin inhibitor, 40 μg/ml DNase I, 50 μM d-2-amino-5-phosphonovaleric acid (APV), 20 μM 6,7-dinitroquinoxaline-2,3 (1H, 4H)-dione (DNQX), and 0.1 μM tetrodotoxin (TTX) (all from Sigma-Aldrich). Tissues were triturated mildly by sequential passage through wide-bore and fine-tipped pipettes. Cells were filtered through a 40-μm nylon mesh and resuspended in Ca2+- and Mg2+-free dissociation solution at a final concentration of 5 × 106 cells/ml. To label dead cells, Propidium iodide (PI; Sigma-Aldrich) was added at a final concentration of 2 μg/ml. Cell sorting was performed on a FACsAria machine. The sorting decision was based on the measurements of forward-scattered light, PI fluorescence, and GFP fluorescence.

Purkinje cell sorting

Purkinje cells were isolated from Pcp2-GFP mice as described (Tomomura et al., 2001). Briefly, cerebella were removed from mice at the indicated ages. Cubes (0.5 mm3) of cerebella were digested for 10 min at 37°C with 0.025% trypsin (type I; Sigma-Aldrich) in dissociation solution consisting of Ca2+-free Hank's balanced salt solution containing 3 mg/ml bovine serum albumin (BSA), 15 mM 4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid (HEPES), 1.5 mM MgSO4, and 3 mg/ml glucose (pH 7.4). The enzymatic reaction was stopped by the addition of dissociation solution containing 0.25 mg/ml soybean trypsin inhibitor, 40 μg/ml DNase I, 50 μM d-2-amino-5-phosphonovaleric acid (APV), 20 μM 6,7-dinitroquinoxaline-2,3 (1H, 4H)-dione (DNQX), and 0.1 μM tetrodotoxin (TTX) (all from Sigma-Aldrich). Tissues were triturated mildly by sequential passage through wide-bore and fine-tipped pipettes. Cells were filtered through a 40-μm nylon mesh and resuspended in Ca2+- and Mg2+-free dissociation solution at a final concentration of 5 × 106 cells/ml. To label dead cells, Propidium iodide (PI; Sigma-Aldrich) was added at a final concentration of 2 μg/ml. Cell sorting was performed on a FACsAria machine. The sorting decision was based on the measurements of forward-scattered light, PI fluorescence, and GFP fluorescence.

RNA preparation and RT-qPCR of purified Purkinje cellsTotal RNA was extracted with TRIzol reagent (Invitrogen) from ∼3 × 104 purified Purkinje cells at the indicated stages. The corresponding cDNAs were prepared by reverse transcription of 100 ng of RNA using the SuperScript III First-Strand Synthesis System (Invitrogen) with an oligo-dT primer according to the manufacturer's instructions. The resulting cDNAs were used as a template for RT-qPCR using a Light Cycler 480 thermocycler (384 plates; Roche Diagnostics, Meylan, France) with a home-made SYBR Green QPCR master mix (Lutfalla and Uze, 2006). Thermal cycling parameters were 2 min at 95°C, followed by 45 cycles of 95°C for 10 s, 64°C for 15 s, and 72°C for 25 s. The relative quantification in gene expression was determined using the ΔΔCt method. To normalize expression data, primers for 10 commonly used housekeeping genes were used, and the normalization factor was determined using the geNorm software, as described in Vandesompele et al. (2002). This led to the selection of the following internal control genes in our assays: glyceraldehyde-3-phosphate dehydrogenase, β-actin, and hypoxanthine phosphoribosyltransferase 1. Sequences of the primers used are listed in Supplemental Table S1.

RNA preparation and RT-qPCR of purified Purkinje cells

Total RNA was extracted with TRIzol reagent (Invitrogen) from ∼3 × 104 purified Purkinje cells at the indicated stages. The corresponding cDNAs were prepared by reverse transcription of 100 ng of RNA using the SuperScript III First-Strand Synthesis System (Invitrogen) with an oligo-dT primer according to the manufacturer's instructions. The resulting cDNAs were used as a template for RT-qPCR using a Light Cycler 480 thermocycler (384 plates; Roche Diagnostics, Meylan, France) with a home-made SYBR Green QPCR master mix (Lutfalla and Uze, 2006). Thermal cycling parameters were 2 min at 95°C, followed by 45 cycles of 95°C for 10 s, 64°C for 15 s, and 72°C for 25 s. The relative quantification in gene expression was determined using the ΔΔCt method. To normalize expression data, primers for 10 commonly used housekeeping genes were used, and the normalization factor was determined using the geNorm software, as described in Vandesompele et al. (2002). This led to the selection of the following internal control genes in our assays: glyceraldehyde-3-phosphate dehydrogenase, β-actin, and hypoxanthine phosphoribosyltransferase 1. Sequences of the primers used are listed in Supplemental Table S1.

ImmunohistochemistryFor immunohistochemistry, mice were perfused transcardially with 4% paraformaldehyde (PFA) in 0.12 M phosphate buffer, pH 7.4. Brains were removed, postfixed overnight in 4% PFA, cryoprotected in 20% and then 30% sucrose, embedded in OCT, frozen in isopentane (−55°C), and stored at −80°C. Sagittal sections (10 μm) were cut with a cryostat and stored at −20°C before immunostaining. Sections were rehydrated in phosphate-buffered saline (PBS) for 5 min and then fixed and permeabilized with 70% methanol/30% acetone for 15 min at −20°C. After washes in PBS, slices were blocked for 30 min in PBS containing 1% BSA and incubated with primary antibodies diluted in PBS/1% BSA for 2 h at room temperature. After washes in PBS containing 0.25% Tween-20, slices were incubated with secondary antibodies diluted in PBS/1% BSA for 45 min at room temperature. After washes, sections were mounted with Mowiol.

Immunohistochemistry

For immunohistochemistry, mice were perfused transcardially with 4% paraformaldehyde (PFA) in 0.12 M phosphate buffer, pH 7.4. Brains were removed, postfixed overnight in 4% PFA, cryoprotected in 20% and then 30% sucrose, embedded in OCT, frozen in isopentane (−55°C), and stored at −80°C. Sagittal sections (10 μm) were cut with a cryostat and stored at −20°C before immunostaining. Sections were rehydrated in phosphate-buffered saline (PBS) for 5 min and then fixed and permeabilized with 70% methanol/30% acetone for 15 min at −20°C. After washes in PBS, slices were blocked for 30 min in PBS containing 1% BSA and incubated with primary antibodies diluted in PBS/1% BSA for 2 h at room temperature. After washes in PBS containing 0.25% Tween-20, slices were incubated with secondary antibodies diluted in PBS/1% BSA for 45 min at room temperature. After washes, sections were mounted with Mowiol.

Recombinant lentiviral vectors: construction and productionPlasmid constructionsTo inhibit DOCK10 expression, two shRNAs directed against DOCK10 mRNA were selected for their inhibitory efficiency. Their corresponding DNA sequences are as follows:ShA:5′-GATCCCCGCACAGAGCTGAATCCTATTTCAAG­AG­AATAGGATTCAGCTCTGTGCTTTTTA-3′ShB:5′-GATCCCCCAAGGCACGGAATATAACTTTCAAGAGAAG­TTATATTCCGTGCCTTGTTTTTA-3′These two sequences were inserted as a DNA cassette under the control of the polymerase III-dependent H1 promoter into the pFlap-PGK-EGFP-ΔU3 vector (Santamaria et al., 2009), generating the pFlap-H1-shRNA-PGK-EGFP-ΔU3 series of vectors hereafter named shA-DOCK10 and shB-DOCK10. The scrambled control shRNA vector, shScr, corresponding to a random shA-DOCK10 sequence (5′-GATCCCCGGCAAATTAGCCACGTACTTTCAAGAGAAGTACGTGGCTAATTTCGGTTTTTA-3′) and a nonrelevant control shRNA vector, Lv-shRd, consisting of a random sequence (5′-GATCCCCTCGTCATAGCGTGCATAGGTTCAAGAGACCTATGCACGCTATGACGATTTTTGGAAA-3′) were used as controls and collectively referred to as shCtrl throughout the study.Lentiviral productionStocks of viral particles were prepared as described previously (Zennou et al., 2001) by transient cotransfection of HEK293T cells with the p8.91 encapsidation plasmid (Zufferey et al., 1997), the pHCMV-G (vesicular stomatitis virus pseudotype) envelope plasmid, and the pFlap vectors described earlier. Briefly, the supernatants were collected 48 h after transfection, treated with DNase I (Roche Diagnostics), and filtered before ultracentrifugation. Viral pellets were then resuspended in PBS, aliquoted, and stored at −80°C until use. The amount of p24 capsid protein was determined by the HIV-1 p24 ELISA antigen assay (Beckman Coulter, Fullerton, CA). Viruses from different productions averaged 300 ng/μl of p24 antigen and 3 × 108 transducing units/ml.

Recombinant lentiviral vectors: construction and production

Plasmid constructionsTo inhibit DOCK10 expression, two shRNAs directed against DOCK10 mRNA were selected for their inhibitory efficiency. Their corresponding DNA sequences are as follows:ShA:5′-GATCCCCGCACAGAGCTGAATCCTATTTCAAG­AG­AATAGGATTCAGCTCTGTGCTTTTTA-3′ShB:5′-GATCCCCCAAGGCACGGAATATAACTTTCAAGAGAAG­TTATATTCCGTGCCTTGTTTTTA-3′These two sequences were inserted as a DNA cassette under the control of the polymerase III-dependent H1 promoter into the pFlap-PGK-EGFP-ΔU3 vector (Santamaria et al., 2009), generating the pFlap-H1-shRNA-PGK-EGFP-ΔU3 series of vectors hereafter named shA-DOCK10 and shB-DOCK10. The scrambled control shRNA vector, shScr, corresponding to a random shA-DOCK10 sequence (5′-GATCCCCGGCAAATTAGCCACGTACTTTCAAGAGAAGTACGTGGCTAATTTCGGTTTTTA-3′) and a nonrelevant control shRNA vector, Lv-shRd, consisting of a random sequence (5′-GATCCCCTCGTCATAGCGTGCATAGGTTCAAGAGACCTATGCACGCTATGACGATTTTTGGAAA-3′) were used as controls and collectively referred to as shCtrl throughout the study.

Plasmid constructions

To inhibit DOCK10 expression, two shRNAs directed against DOCK10 mRNA were selected for their inhibitory efficiency. Their corresponding DNA sequences are as follows:

5′-GATCCCCGCACAGAGCTGAATCCTATTTCAAG­AG­AATAGGATTCAGCTCTGTGCTTTTTA-3′

5′-GATCCCCCAAGGCACGGAATATAACTTTCAAGAGAAG­TTATATTCCGTGCCTTGTTTTTA-3′

These two sequences were inserted as a DNA cassette under the control of the polymerase III-dependent H1 promoter into the pFlap-PGK-EGFP-ΔU3 vector (Santamaria et al., 2009), generating the pFlap-H1-shRNA-PGK-EGFP-ΔU3 series of vectors hereafter named shA-DOCK10 and shB-DOCK10. The scrambled control shRNA vector, shScr, corresponding to a random shA-DOCK10 sequence (5′-GATCCCCGGCAAATTAGCCACGTACTTTCAAGAGAAGTACGTGGCTAATTTCGGTTTTTA-3′) and a nonrelevant control shRNA vector, Lv-shRd, consisting of a random sequence (5′-GATCCCCTCGTCATAGCGTGCATAGGTTCAAGAGACCTATGCACGCTATGACGATTTTTGGAAA-3′) were used as controls and collectively referred to as shCtrl throughout the study.

Lentiviral productionStocks of viral particles were prepared as described previously (Zennou et al., 2001) by transient cotransfection of HEK293T cells with the p8.91 encapsidation plasmid (Zufferey et al., 1997), the pHCMV-G (vesicular stomatitis virus pseudotype) envelope plasmid, and the pFlap vectors described earlier. Briefly, the supernatants were collected 48 h after transfection, treated with DNase I (Roche Diagnostics), and filtered before ultracentrifugation. Viral pellets were then resuspended in PBS, aliquoted, and stored at −80°C until use. The amount of p24 capsid protein was determined by the HIV-1 p24 ELISA antigen assay (Beckman Coulter, Fullerton, CA). Viruses from different productions averaged 300 ng/μl of p24 antigen and 3 × 108 transducing units/ml.

Lentiviral production

Stocks of viral particles were prepared as described previously (Zennou et al., 2001) by transient cotransfection of HEK293T cells with the p8.91 encapsidation plasmid (Zufferey et al., 1997), the pHCMV-G (vesicular stomatitis virus pseudotype) envelope plasmid, and the pFlap vectors described earlier. Briefly, the supernatants were collected 48 h after transfection, treated with DNase I (Roche Diagnostics), and filtered before ultracentrifugation. Viral pellets were then resuspended in PBS, aliquoted, and stored at −80°C until use. The amount of p24 capsid protein was determined by the HIV-1 p24 ELISA antigen assay (Beckman Coulter, Fullerton, CA). Viruses from different productions averaged 300 ng/μl of p24 antigen and 3 × 108 transducing units/ml.

Organotypic slice culture, lentivirus-mediated transduction, and immunostainingCerebellar organotypic cultures were set up from newborn (P0) mice as previously described (Lebrun et al., 2013). Mice were decapitated, and their brains were dissected out in cold Gey's balanced salt solution (Invitrogen) supplemented with 5 mg/ml glucose. Cerebellar parasagittal slices (350 μm thick) were cut on a McIlwain tissue chopper and transferred onto 30-mm-diameter Millipore culture inserts with 0.4-μm pores (Millicell; Millipore, St. Quentin en Yvelines, France). Slices were maintained in culture in six-well plates containing 1 ml of nutrient medium/well at 37°C under a humidified atmosphere containing 5% CO2. The nutrient medium consisted of basal medium with Earle's salts (BME; Invitrogen) containing 1 mM l-glutamine, 5 mg/ml glucose, 0.5 mg/ml BSA (A-4503; Sigma-Aldrich), and supplements including 5 μg/ml insulin, 5 μg/ml transferrin, and 5 μg/ml sodium selenite (I-1884; Sigma-Aldrich). Lentiviral vectors (0.5–2 μl/slice, depending on the lentiviral production) were applied 2–4 h after culture processing of P0 slices. The medium was replaced every 2–3 d.Slices were fixed with 4% PFA in phosphate buffer (0.1 M, pH 7.4) for 1 h at room temperature after 14 DIV. They were washed in PBS, incubated 1 h in PBS 0.25% Triton X-100 (PBST), 0.1 M gelatin (PBSTG), and 0.1% sodium azide and then incubated with primary antibodies in PBSTG overnight at room temperature. After washing in PBST, slices were incubated with secondary antibodies in PBSTG for 2 h at room temperature, washed in PBST, and mounted in Mowiol.

Organotypic slice culture, lentivirus-mediated transduction, and immunostaining

Cerebellar organotypic cultures were set up from newborn (P0) mice as previously described (Lebrun et al., 2013). Mice were decapitated, and their brains were dissected out in cold Gey's balanced salt solution (Invitrogen) supplemented with 5 mg/ml glucose. Cerebellar parasagittal slices (350 μm thick) were cut on a McIlwain tissue chopper and transferred onto 30-mm-diameter Millipore culture inserts with 0.4-μm pores (Millicell; Millipore, St. Quentin en Yvelines, France). Slices were maintained in culture in six-well plates containing 1 ml of nutrient medium/well at 37°C under a humidified atmosphere containing 5% CO2. The nutrient medium consisted of basal medium with Earle's salts (BME; Invitrogen) containing 1 mM l-glutamine, 5 mg/ml glucose, 0.5 mg/ml BSA (A-4503; Sigma-Aldrich), and supplements including 5 μg/ml insulin, 5 μg/ml transferrin, and 5 μg/ml sodium selenite (I-1884; Sigma-Aldrich). Lentiviral vectors (0.5–2 μl/slice, depending on the lentiviral production) were applied 2–4 h after culture processing of P0 slices. The medium was replaced every 2–3 d.

Slices were fixed with 4% PFA in phosphate buffer (0.1 M, pH 7.4) for 1 h at room temperature after 14 DIV. They were washed in PBS, incubated 1 h in PBS 0.25% Triton X-100 (PBST), 0.1 M gelatin (PBSTG), and 0.1% sodium azide and then incubated with primary antibodies in PBSTG overnight at room temperature. After washing in PBST, slices were incubated with secondary antibodies in PBSTG for 2 h at room temperature, washed in PBST, and mounted in Mowiol.

Semiautomated image analysis of Purkinje cellsPhotomicrographs of isolated Purkinje cells were acquired (Leica wide-field DM6000 microscope, 40× oil objective), and the conditions (experimental procedure, transduced or not transduced) were recorded for each Purkinje cell in a file. Then a study blinded to the experimental procedures was performed to categorize and analyze the dendritic tree of Purkinje cells using ImagePro software. For each Purkinje cell, the following variables were measured: size of the soma, number of branch points in the dendritic arbor, and length of the longest dendrite connecting to a given cell body. An algorithm was proposed to classify the different types of dendritic tree of Purkinje cells into three groups: atrophic, when the size of the longest dendrite was inferior to the soma diameter; stellate, when more than six primary dendrites of about the same size emerged from the cell body; and elaborate, including all the other types. Among the Purkinje cells presenting an elaborate dendritic tree, we distinguished two types of Purkinje cells: those with spines—that is, presenting at least one part of their dendritic tree covered with spines—and those we called with no spines. On these elaborate Purkinje cells, a Sholl analysis was performed to estimate the distribution and complexity of the dendrites (Gutierrez and Davies, 2007). The Sholl analysis consisted of 1) constructing concentric and equidistantly (10 μm) organized spherical shells centered at the cell body, and 2) counting the intersections of dendrites with the circles of increasing radius. To quantify dendritic length, branch points, and Sholl analysis, Perl scripts were developed for parsing intermediate outputs.

Semiautomated image analysis of Purkinje cells

Photomicrographs of isolated Purkinje cells were acquired (Leica wide-field DM6000 microscope, 40× oil objective), and the conditions (experimental procedure, transduced or not transduced) were recorded for each Purkinje cell in a file. Then a study blinded to the experimental procedures was performed to categorize and analyze the dendritic tree of Purkinje cells using ImagePro software. For each Purkinje cell, the following variables were measured: size of the soma, number of branch points in the dendritic arbor, and length of the longest dendrite connecting to a given cell body. An algorithm was proposed to classify the different types of dendritic tree of Purkinje cells into three groups: atrophic, when the size of the longest dendrite was inferior to the soma diameter; stellate, when more than six primary dendrites of about the same size emerged from the cell body; and elaborate, including all the other types. Among the Purkinje cells presenting an elaborate dendritic tree, we distinguished two types of Purkinje cells: those with spines—that is, presenting at least one part of their dendritic tree covered with spines—and those we called with no spines. On these elaborate Purkinje cells, a Sholl analysis was performed to estimate the distribution and complexity of the dendrites (Gutierrez and Davies, 2007). The Sholl analysis consisted of 1) constructing concentric and equidistantly (10 μm) organized spherical shells centered at the cell body, and 2) counting the intersections of dendrites with the circles of increasing radius. To quantify dendritic length, branch points, and Sholl analysis, Perl scripts were developed for parsing intermediate outputs.

Primary cell culture and transfection of hippocampal neuronsNeuronal hippocampal cultures were prepared from embryonic day 17.5 C57BL/6 mice and grown in Neurobasal medium supplemented with B27 and 10% fetal bovine serum (Raynaud et al., 2013). Neurons were transfected at 8 DIV with pSuper plasmids expressing the described shRNA sequences or plasmid DNA constructs of DOCK10, PAK3, or N-WASP and fixed at 11 DIV.

Primary cell culture and transfection of hippocampal neurons

Neuronal hippocampal cultures were prepared from embryonic day 17.5 C57BL/6 mice and grown in Neurobasal medium supplemented with B27 and 10% fetal bovine serum (Raynaud et al., 2013). Neurons were transfected at 8 DIV with pSuper plasmids expressing the described shRNA sequences or plasmid DNA constructs of DOCK10, PAK3, or N-WASP and fixed at 11 DIV.

Subcellular fractionationSynaptosomal fractions were prepared as follows: brains from killed P27 mice were placed in HEPES buffer (20 mM HEPES, pH 7.4, 0.32 M sucrose, 1 mM EDTA), subjected to Dounce homogenization, and subsequently centrifuged at 1500 × g for 5 min to remove nuclei and cell debris. The supernatant was centrifuged at 100,000 × g for 10 min. The resulting supernatant was saved (fraction S1: cytoplasmic fraction), and the pellet was resuspended in HEPES buffer. This homogenate was carefully layered on top of a 0.85 M sucrose buffer and centrifuged at 9000 × g for 25 min. The resulting upper phase P1 (small vesicles) and middle phase P2 (synaptosomes) were collected and subjected to centrifugation at 100,000 × g for 10 min, and the pellet fraction P3 (membranes) was solubilized in Lysis buffer (20 mM HEPES, pH 7.4, 100 mM NaCl, 2 mM EDTA, 1% Triton). The pellets resulting from the centrifuged P1 and P2 fractions were also solu­bilized in lysis buffer.

Subcellular fractionation

Synaptosomal fractions were prepared as follows: brains from killed P27 mice were placed in HEPES buffer (20 mM HEPES, pH 7.4, 0.32 M sucrose, 1 mM EDTA), subjected to Dounce homogenization, and subsequently centrifuged at 1500 × g for 5 min to remove nuclei and cell debris. The supernatant was centrifuged at 100,000 × g for 10 min. The resulting supernatant was saved (fraction S1: cytoplasmic fraction), and the pellet was resuspended in HEPES buffer. This homogenate was carefully layered on top of a 0.85 M sucrose buffer and centrifuged at 9000 × g for 25 min. The resulting upper phase P1 (small vesicles) and middle phase P2 (synaptosomes) were collected and subjected to centrifugation at 100,000 × g for 10 min, and the pellet fraction P3 (membranes) was solubilized in Lysis buffer (20 mM HEPES, pH 7.4, 100 mM NaCl, 2 mM EDTA, 1% Triton). The pellets resulting from the centrifuged P1 and P2 fractions were also solu­bilized in lysis buffer.

Immunofluorescence microscopyHippocampal neurons were fixed and permeabilized as previously described (Raynaud et al., 2013). Untreated or shRNA-GFP–expressing neurons were visualized using a Zeiss Axioimager Z1 microscope. For brain slices, we used a Zeiss LMS780 confocal microscope with a 63×/1.4 numerical aperture (NA) oil objective or a Leica wide-field DM6000 microscope with a 100×/1.4 NA oil objective. Images were recorded with a charge-coupled device HQ2 camera (Roper Scientific, Martiensried, Germany) controlled by MetaMorph 7.1 (Universal Imaging).Morphometric analyses were performed in different fields from at least three different cultures using ImageJ software (National Institutes of Health, Bethesda, MD). Spines were defined as dendritic protrusions with a neck and a head. Dendrites were randomly selected and spines manually counted over a 50-μm length of dendrite. Data were then expressed as density of spines/10-μm length of dendrite. The same spines were used to measure the area of their head. Each spine was manually traced and the surface calculated using ImageJ. All experiments were conducted in a double-blinded manner, and dendritic spines were analyzed from at least 20 neurons from at least three independent cultures (n = 3 or 4 experiments, as indicated).

Immunofluorescence microscopy

Hippocampal neurons were fixed and permeabilized as previously described (Raynaud et al., 2013). Untreated or shRNA-GFP–expressing neurons were visualized using a Zeiss Axioimager Z1 microscope. For brain slices, we used a Zeiss LMS780 confocal microscope with a 63×/1.4 numerical aperture (NA) oil objective or a Leica wide-field DM6000 microscope with a 100×/1.4 NA oil objective. Images were recorded with a charge-coupled device HQ2 camera (Roper Scientific, Martiensried, Germany) controlled by MetaMorph 7.1 (Universal Imaging).

Morphometric analyses were performed in different fields from at least three different cultures using ImageJ software (National Institutes of Health, Bethesda, MD). Spines were defined as dendritic protrusions with a neck and a head. Dendrites were randomly selected and spines manually counted over a 50-μm length of dendrite. Data were then expressed as density of spines/10-μm length of dendrite. The same spines were used to measure the area of their head. Each spine was manually traced and the surface calculated using ImageJ. All experiments were conducted in a double-blinded manner, and dendritic spines were analyzed from at least 20 neurons from at least three independent cultures (n = 3 or 4 experiments, as indicated).

In vitro GEF assaysFluorescence-based in vitro guanine-nucleotide exchange assays were performed using Mant-GTP (Molecular Probes, Life Technologies, St-Aubin, France) in an FLX 800 microplate fluorescence reader (BioTek Instruments, Colmar, France) at 25°C, as described (Bouquier et al., 2009). Nucleotide exchange was determined by measuring Mant-GTP loading on GDP-preloaded Rac1, Cdc42, or RhoA GTPases using 1 μM GEF (0.5 μM for Dbl). The relative Mant fluorescence (λex = 360 nm and λem = 460 nm) was monitored for 30 min, and measurements were taken every 15 s.

In vitro GEF assays

Fluorescence-based in vitro guanine-nucleotide exchange assays were performed using Mant-GTP (Molecular Probes, Life Technologies, St-Aubin, France) in an FLX 800 microplate fluorescence reader (BioTek Instruments, Colmar, France) at 25°C, as described (Bouquier et al., 2009). Nucleotide exchange was determined by measuring Mant-GTP loading on GDP-preloaded Rac1, Cdc42, or RhoA GTPases using 1 μM GEF (0.5 μM for Dbl). The relative Mant fluorescence (λex = 360 nm and λem = 460 nm) was monitored for 30 min, and measurements were taken every 15 s.

Active GTPase pull downFor the Rac1-GTP glutathione S-transferase (GST) pull-down assay, the CRIB domain of PAK was chosen, as described previously (Briancon-Marjollet et al., 2008), whereas for Cdc42, the CRIB domain of N-WASP was used. Total HEK293T lysates and corresponding pull downs retained on GST-Sepharose beads were processed for Western immunoblotting. RhoA-GTP pull-down assays were performed similarly, using the RBD domain of Rhotekin (Bouquier et al., 2009).

Active GTPase pull down

For the Rac1-GTP glutathione S-transferase (GST) pull-down assay, the CRIB domain of PAK was chosen, as described previously (Briancon-Marjollet et al., 2008), whereas for Cdc42, the CRIB domain of N-WASP was used. Total HEK293T lysates and corresponding pull downs retained on GST-Sepharose beads were processed for Western immunoblotting. RhoA-GTP pull-down assays were performed similarly, using the RBD domain of Rhotekin (Bouquier et al., 2009).

Statistical analysisThe distribution of Purkinje cell types was compared using the Pearson χ2 test or Fisher's exact test, depending on the amount. The Kruskal–Wallis test (nonparametric analysis of variance) was used to compare the Sholl index. All analyses were performed using the Statistical Package for the Social Sciences (SPSS), version 20.0. Significance values were set at p < 0.05.

Statistical analysis

The distribution of Purkinje cell types was compared using the Pearson χ2 test or Fisher's exact test, depending on the amount. The Kruskal–Wallis test (nonparametric analysis of variance) was used to compare the Sholl index. All analyses were performed using the Statistical Package for the Social Sciences (SPSS), version 20.0. Significance values were set at p < 0.05.

Supplementary MaterialSupplemental Materials

Supplementary Material

Supplemental Materials
