---
pmid: '29466333'
title: The cryo-electron microscopy structure of huntingtin.
authors:
- Guo Q
- Bin Huang
- Cheng J
- Seefelder M
- Engler T
- Pfeifer G
- Oeckl P
- Otto M
- Moser F
- Maurer M
- Pautsch A
- Baumeister W
- Fernández-Busnadiego R
- Kochanek S
journal: Nature
year: '2018'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC5837020
doi: 10.1038/nature25502
---

# The cryo-electron microscopy structure of huntingtin.
**Authors:** Guo Q, Bin Huang, Cheng J, Seefelder M, Engler T, Pfeifer G, Oeckl P, Otto M, Moser F, Maurer M, Pautsch A, Baumeister W, Fernández-Busnadiego R, Kochanek S
**Journal:** Nature (2018)
**DOI:** [10.1038/nature25502](https://doi.org/10.1038/nature25502)
**PMC:** [PMC5837020](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5837020/)

## Abstract

1. Nature. 2018 Mar 1;555(7694):117-120. doi: 10.1038/nature25502. Epub 2018 Feb 
21.

The cryo-electron microscopy structure of huntingtin.

Guo Q(1), Bin Huang(2), Cheng J(3), Seefelder M(2), Engler T(2), Pfeifer G(1), 
Oeckl P(4), Otto M(4), Moser F(5), Maurer M(5), Pautsch A(5), Baumeister W(1), 
Fernández-Busnadiego R(1), Kochanek S(2).

Author information:
(1)Department of Molecular Structural Biology, Max Planck Institute of 
Biochemistry, 82152 Martinsried, Germany.
(2)Department of Gene Therapy, Ulm University, 89081 Ulm, Germany.
(3)Gene Center, Department of Biochemistry and Center for Integrated Protein 
Science Munich, Ludwig-Maximilians University, 81377 Munich, Germany.
(4)Department of Neurology, Ulm University, 89081 Ulm, Germany.
(5)Department of Medicinal Chemistry, Boehringer Ingelheim Pharma GmbH & Co. KG, 
88397 Biberach an der Riß, Germany.

Huntingtin (HTT) is a large (348 kDa) protein that is essential for embryonic 
development and is involved in diverse cellular activities such as vesicular 
transport, endocytosis, autophagy and the regulation of transcription. Although 
an integrative understanding of the biological functions of HTT is lacking, the 
large number of identified HTT interactors suggests that it serves as a 
protein-protein interaction hub. Furthermore, Huntington's disease is caused by 
a mutation in the HTT gene, resulting in a pathogenic expansion of a 
polyglutamine repeat at the amino terminus of HTT. However, only limited 
structural information regarding HTT is currently available. Here we use 
cryo-electron microscopy to determine the structure of full-length human HTT in 
a complex with HTT-associated protein 40 (HAP40; encoded by three F8A genes in 
humans) to an overall resolution of 4 Å. HTT is largely α-helical and consists 
of three major domains. The amino- and carboxy-terminal domains contain multiple 
HEAT (huntingtin, elongation factor 3, protein phosphatase 2A and lipid kinase 
TOR) repeats arranged in a solenoid fashion. These domains are connected by a 
smaller bridge domain containing different types of tandem repeats. HAP40 is 
also largely α-helical and has a tetratricopeptide repeat-like organization. 
HAP40 binds in a cleft and contacts the three HTT domains by hydrophobic and 
electrostatic interactions, thereby stabilizing the conformation of HTT. These 
data rationalize previous biochemical results and pave the way for improved 
understanding of the diverse cellular functions of HTT.

DOI: 10.1038/nature25502
PMCID: PMC5837020
PMID: 29466333 [Indexed for MEDLINE]

Conflict of interest statement: The authors declare no competing financial 
interests.

## Full Text

Computational and biochemical studies on Htt have predicted a variable number of
HEAT repeats interspersed by unstructured regions8–12. However, attempts to
determine the structure of Htt at high resolution have been hindered by its
flexibility13–15. Most structural studies have focused on an N-terminal fragment
corresponding to the first exon of the Htt gene, while the majority of the protein
(>97 % of its amino acid length) remains largely uncharted14. To overcome this hurdle we searched for interaction partners
that could stabilize the structure of Htt. A first screen using polyQ-expanded
full-length human Htt (46QHtt) expressed at low levels in HEK293 cells identified
abundant binding with HAP40/F8A (Fig. 1a),
previously characterized as a Htt interactor7 that
recruits Htt to early endosomes16. Although a
complex of Htt and HAP40 could not be reconstituted from the individual proteins
in vitro, the complex was purified at high yield from human cells
co-expressing both full-length human 17QHtt and HAP40 (Fig. 1b). Whereas Htt alone formed oligomers and tended to aggregate17, the Htt-HAP40 complex eluted as a symmetric
narrow peak upon size-exclusion chromatography (Fig.
1c). Consistently, ultracentrifugation analysis indicated that the Htt-HAP40
complex was more conformationally homogeneous than Htt alone (Extended Data Fig. 1). The Htt-HAP40 complex - but not its isolated
components - showed a sharp, strong unfolding transition in differential scanning
fluorimetry assays (Fig. 1d), confirming that the
Htt-HAP40 complex was stable and could be amenable to structural studies18.

The conformational heterogeneity of Htt alone prevented high-resolution cryo-EM
analysis. In contrast, the Htt-HAP40 complex was well defined and yielded a globular
structure measuring ~120 x 80 x 100 Å (Fig.
2, Extended Data Fig. 2a, b), to some
extent reminiscent of a recently published negative stain structure of Htt alone9. The global resolution of the map was 4 Å
(Extended Data Fig. 2c, d), sufficient to build
a de novo atomic model by energy minimization using well-resolved large
side chains as landmarks (Fig. 2, Extended Data Fig. 2e, Extended Data Fig. 3, Extended Data Table
1). For both Htt and HAP40, all secondary structure elements resolved in the
model corresponded to α-helices (Extended Data Fig.
4), in agreement with computational predictions using PSIPRED19 (Extended Data
Fig. 5). For Htt, 72 % of the helices were arranged in HEAT or other tandem
repeats. On the other hand, most of the regions not resolved in the map were predicted
to be unstructured. Notably, no density was observed for the Htt exon 1 fragment (aa
1-90; 17QHtt is used for aa numbering throughout the text) even at very low thresholds,
indicating that this region of the protein is extremely flexible. Thus, polyQ length may
have limited influence on the overall architecture of the Htt-HAP40 complex.

The domain organization of Htt has been controversial1,8–12. Our data show that Htt consists of three
domains: N- and C-terminal domains containing multiple HEAT repeats (hereafter
“N-HEAT” and “C-HEAT”) linked by a smaller bridge domain
(Fig. 2). N-HEAT (aa 91-1684) forms a typical
α-solenoid, comprising 21 HEAT repeats arranged as a one and a half turn
right-handed superhelix, whose concave face defines an arch of ~80 Å in
diameter (Fig. 3a). Two putative membrane-binding
regions have been identified in Htt, both within N-HEAT: an exon 1 fragment, especially
aa 1-17, which may form an amphipathic helix20,
and a larger region at aa 168-366, which contains a functionally important
palmitoylation site at C20821,22. While the N-terminus corresponding to exon 1 is
not visible in our structure, N-HEAT repeats 2-4 (aa 160-275) form a positively charged
region at the second putative membrane-binding region in the N-HEAT convex surface
(Fig. 4a). However, a previously reported
putative amphipathic helix (aa 223-240)22 faces
the inner concave side of N-HEAT, with limited accessibility to membrane
interactions.

Consistent with our computational predictions (Extended Data Fig. 5) and previous studies1,14, N-HEAT accommodates a large
disordered insertion (aa 400-674) between N-HEAT repeats 6 and 7 (Fig. 3a). The insertion projects outwards without interrupting the
interactions between HEAT repeats and making this region accessible to protease action.
Indeed, multiple proteolytic cleavage sites have been mapped to this region1,14,23. However, the continuous packing of HEAT repeats
6 and 7 makes it unlikely that such cleavage events would result in an easy release of
N-terminal fragments, in agreement with previous reports15,24. This insertion also harbors
multiple phosphorylation sites that may modulate protein-protein interactions and
proteolytic accessibility1,25–27, perhaps by
regulating the interaction of this insertion with the positively charged region of
N-HEAT. Most other reported posttranslational modifications of Htt are located in
presumably unstructured regions not resolved in our map (Extended Data Fig. 4), including protease cleavage sites that release
N-terminal Htt fragments1,23,28.

C-HEAT (aa 2092-3098) comprises 12 HEAT repeats forming an elliptical ring of
~80 x 30 Å (Fig. 3b). C-HEAT repeats
1 and 12 interact closing the ring. Contrary to canonical HEAT repeats, where the first
helix is exposed to the convex surface of the domain, the first helix of C-HEAT repeat
12 faces the concave surface of C-HEAT. The repeats are interrupted by two insertions.
Insertion 1 (aa 2121-2456) consists of 12 helical segments that separate C-HEAT repeats
1 and 2. On the other hand, insertion 2 (aa 2510-2663) is mostly unstructured and does
not interfere with the interaction between C-HEAT repeats 2 and 3. Both insertions are
harbored in the concave surface of C-HEAT, potentially shielding this region from
protein-protein interactions. In contrast, both the convex and concave surfaces of
N-HEAT are accessible in the structure (Fig. 3a)
and could thus act as cargo binding sites. This may explain why most of the known
binding sites of Htt interactors have been mapped to its N-terminus1,3,29.

N-HEAT and C-HEAT are stacked approximately vertically and connected by the
bridge domain (aa 1685-2091, Fig. 3c). This domain
contains 6 tandem α-helical repeats, of which repeats 3, 4 and 6 are
Armadillo-like. The repeat region is flanked by 5 non-repeat helices and a flexible
C-terminus (aa 2062-2092), which is unresolved. Besides this flexible linkage, N-HEAT
and C-HEAT are only weakly connected via loop interactions (Fig. 3d), explaining the highly dynamic structure of Htt in the
absence of interaction partners such as HAP40.

HAP40 binds within the cleft defined by the two HEAT domains and the bridge
domain, thereby stabilizing the observed Htt conformation. HAP40 consists of 14
α-helices arranged in TPR-like tandem repeats (Fig.
2). Within the complex, Htt and HAP40 share large interfaces with mainly
hydrophobic interactions (Fig. 4a), in agreement
with our differential scanning fluorimetry data suggesting considerably reduced exposure
of hydrophobic areas in the Htt-HAP40 complex (Fig.
1d). The C-terminus of HAP40 contains four negatively charged residues that
interact with a positively charged patch on the bridge domain of Htt (Fig. 4b). In contrast, the N-terminus of HAP40 is
mostly solvent exposed, and consequently helix 1 is not well resolved in our map.
Similarly, a central region of HAP40 (aa 217-258) was not visible, consistent with
biochemical experiments showing that this region was not required for Htt binding (Extended Data Fig. 6).

Although Htt is highly conserved from sea urchins to humans12 (Extended Data Fig. 7),
the Htt ortholog in D. melanogaster bears only little resemblance to
human Htt. Interestingly, no HAP40 homolog appears to be present in this organism,
suggesting that these two proteins may have co-evolved. While many Htt interactors bind
Htt’s N-terminus, HAP40 binding to Htt requires the coordination of all Htt
domains (Fig. 2, Fig.
4). This explains why HAP40 has been identified as a Htt interactor by
previous studies using full-length Htt as bait4,7, but not by others using only Htt
fragments3. Also, it is possible that other
proteins bind to Htt at a similar location. Altogether, our data resolves long-standing
speculations on Htt's architecture, strongly supports the concept that Htt serves
as a multivalent interaction hub1 and invites
future structure-guided studies on the mechanisms by which Htt coordinates its diverse
activities.

MethodsAntibodiesThe following antibodies were used: anti-Flag M2 (Sigma), anti-HAP40
SC-69489 (Santa Cruz), anti-Strep (IBA) and anti-Htt MAB2166 (Millipore).Identification of Htt-interacting proteins2x108 HEK293-based C2.6 cells17 expressing FLAG-tagged full-length polyQ-expanded 46QHtt at low
levels were harvested and lysed within 25 mM Tris, 150 mM NaCl, 0.5 % Tween 20,
1x protease inhibitor (Roche), pH 7.4, followed by centrifugation (14.000 rpm, 1
h). The supernatant was incubated with FLAG beads at 4 °C for 2 h,
followed by three times washing with 25 mM Tris, 150 mM NaCl, 0.02 % Tween 20,
pH 7.4. Proteins bound to the FLAG beads were eluted with 100 mM Glycine, 150 mM
NaCl, 0.02 % Tween, pH 3.5 and immediately neutralized with 1 M Tris (pH 8.0).
The eluted proteins were concentrated and analysed by SDS-PAGE and Coomassie
staining. To identify potential interactors of Htt the lanes were excised,
proteins were in-gel digested using trypsin, and then analyzed by nano liquid
chromatography (C18, 500 x 0.075 mm, 2 µm column, Thermo Fisher
Scientific) and tandem mass spectrometry (QExactive, Thermo Fisher Scientific)
in data-dependent acquisition mode (Top12). Proteins were identified using
Proteome Discoverer 1.4 (Thermo Fisher Scientific) with a peptide FDR ≤
0.01, and enrichment analysis was performed with Perseus 1.4.1.3 using MS1 peak
area for quantification.The identity of FLAG affinity-purified proteins was confirmed by Western
blot analysis following SDS-PAGE with anti-Htt and anti-HAP antibodies.Generation of a stable human cell lines co-expressing 17QHtt and
HAP40B1.21 cells17 are based on HEK293
cells and express full-length wild type 17QHtt upon induction with doxycycline
(Dox). An expression plasmid pBSK/2-CMV-HAP40-TS was constructed to express the
human HAP40 (F8A) protein (NCBI Ref Seq NP_036283.2) with a C-terminal
Twin-Strep-tag under the control of the hCMV promoter. B1.21 cells were
co-transfected with this plasmid together with a puromycin resistance gene. The
resulting stable cell line (B1.21-HAP40TS) expressing HAP40 at high constitutive
levels and Htt upon induction with Dox, was used for the purification of
Htt-HAP40 complex. Generated cell lines have been tested negative for mycoplasma
by PCR. Cell lines have been authenticated by inducibility of Htt expression
with doxycycline and Western blot analysis.Purification of Htt, HAP40 and the Htt-HAP40 complexThe purification of Htt alone has been described17. For purification of the Htt-HAP40 complex,
2x108 B1.21-HAP40TS cells were harvested 72 h after induction
with Dox by centrifugation at 400 g for 10 min. Cells were lysed with 25 mM
HEPES, 300 mM NaCl, 0.5 % Tween 20, protease inhibitor, pH 8.0 by rotation at 4
°C for 30 min followed by centrifugation of the cell lysate at 30,000 g
and clearance by filtration through a 0.2 μm filter. The filtrate was
incubated with Strep-Tactin beads (Qiagen) for 2-3 h at 4 °C. After
washing three times with 25 mM HEPES, 300 mM NaCl, 0.02 % Tween 20, pH 8.0 bound
proteins were eluted with 25 mM HEPES, 300 mM NaCl, 0.02 % Tween 20, 2.5 mM
Desthiobiotin, pH 8.0. The eluate was concentrated using Amicon filters.The Htt-HAP40 complex was further purified by size exclusion
chromatography using a Superose 6 10/300 increase column (GE Healthcare) in
running buffer 25 mM HEPES, 300 mM NaCl, 0.1 % CHAPS and 1 mM DTT, pH 8.0.
Htt-HAP40 eluted in one narrow-based peak and was concentrated with Amicon ultra
100 kDa filters (Millipore).HAP40 was purified from the Htt-HAP40 complex as follows. Htt-HAP40
bound to Strep beads was eluted with 25 mM HEPES, 300 mM NaCl, 0.05 % N-Dodecyl
beta-D-maltoside (DDM) and 2.5 mM Desthiobiotin, pH 8.0. The eluate was
concentrated using Amicon filters. To disrupt the Htt-HAP40 complex, DDM was
added to a final concentration of 0.25 %. After overnight incubation at 4
°C, the Strep eluate was further purified by size exclusion
chromatography using a Superose 6 10/300 increase column in running buffer 25
mM, 300 mM NaCl, 0.1 % CHAPS and 1 mM DTT, pH 8.0 to separate Htt and HAP40.
HAP40 eluted in one narrow-based peak and was concentrated with Amicon ultra 30
kDa filters.Rate-zonal ultracentrifugation5-20 % sucrose gradients in 25 mM HEPES, 300 mM NaCl, 0.1 % CHAPS, pH
8.0 were generated by an automatic gradient maker (Gradient master, Biocomp
instrument). A volume of 120 μl of FLAG-tag-purified Htt or
Strep-tag-purified Htt-HAP40 complex was laid on top of the gradient and
centrifuged at 39,000 rpm for 16 h using a SW41 rotor in a Beckman
ultracentrifuge. Fractions of the sucrose gradient were collected from the
bottom of the tubes in fractions of 0.5 ml to be analyzed by SDS-PAGE and
Coomassie Blue staining and Western blotting.Differential scanning fluorimetryProtein thermostability was assessed by differential scanning
fluorimetry30. Protein unfolding was
monitored by the increase in the fluorescence of SYPRO Orange (Invitrogen).
Prior to use, a 100 mM stock of the dye (stored at -20°C) was diluted
1:20 in DMSO and directly added to the sample to a final concentration of 125
µM. The tested proteins were diluted in sample buffer (25 mM HEPES, 300
mM NaCl, 0.1 % CHAPS, 1 mM DTT and 10 % glycerol) to concentrations of 1.6
µM (Htt, Htt-HAP40 complex) and 2 µM (HAP40). The samples were
heated up with a ramp rate of 1 °C/min over a temperature range of 15-95
°C using the qPCR System MX 3005 P (Stratagene). Measurements were
performed in duplicate.Transient expression of HAP40 and HAP40 fragments and interaction studies
with HttPlasmids were generated expressing, under hCMV promoter control, either
full-length HAP40, a N-terminal HAP40 fragment (HAP40-N, encoding aa 1-222), a
C-terminal HAP40 fragment (HAP40-C, encoding aa 249-371) or a HAP40 fragment, in
which the central proline-rich region had been replaced by a flexible linker
(HAP40∆, encoding aa 1-222 linked by a (GGGGS)3 linker to aa
249-371). All HAP40 variants carried a C-terminal Twin-Strep-tag.B1.21 cells induced with Dox to express 17QHtt were transiently
transfected with the plasmids using PEI transfection. At 48 hours after
transfection, the cells were harvested by centrifugation and lysed in 25 mM
HEPES, 300 mM NaCl, 0.5 % Tween 20, 1x protease inhibitor, pH 8.0, followed by
centrifugation (14.000 rpm, 1 h). The supernatant was incubated with Magstrep
beads (IBA) at 4 °C for 2 hours, followed by three times washing with 25
mM HEPES, 300 mM NaCl, 0.02 % Tween 20, pH 8.0. Thereafter, bound proteins were
eluted using Desthiobiotin in SDS loading buffer, followed by SDS-PAGE and
Western blot analysis using anti-FLAG and anti-Strep antibodies for
detection.Cryo-EM sample preparation and data acquisitionPurified Htt-HAP40 complex was diluted to 0.5 mg/ml with 25mM HEPES, 300
mM NaCl, 0.025 % CHAPS, 1 mM DTT. 4 μl of sample was applied to a
Quantifoil gold grid suspended with monolayer graphene (Graphenea) and vitrified
by plunge-freezing into a liquid ethane/propane mixture using Vitrobot Mark IV
(FEI) with blotting time of 5 s. Data collection was performed on a Titan Krios
microscope (FEI) operated at 300 kV and equipped with a field emission gun, a
Gatan GIF Quantum energy filter and a Gatan K2 Summit direct electron camera.
The calibrated magnification was 105,000 in EFTEM mode, corresponding to a pixel
size of 1.35 Å. Images were collected at a dose rate of 4
electrons/Å2/s, each exposure (8 s exposures time)
comprising 16 sub-frames amounting to a total dose of 32
electrons/Å2/s. Data was recorded using SerialEM31 software and custom macros with defocus
values ranging from -1.4 to -3 μm.Image processingMicrograph movie frame stacks were subjected to beam-induced motion
correction by MotionCor232. Most further
processing was performed using RELION33.
The contrast transfer function parameters for each micrograph were determined
with CTFFIND434, and all micrographs with
a resolution limit worse than 4 Å were discarded. Particles were
initially picked with Gautomatch (http://www.mrc-lmb.cam.ac.uk/kzhang/Gautomatch/Gautomatch_Brief_Manual.pdf)
using a sphere as template, and extracted with a 160-pixel by 160-pixel box.
Reference-free 2D class averaging was performed reiteratively, keeping only
particles with well-resolved 2D average for initial model generation. To
validate the ab initio model, 3D classification was performed
using initial models generated by RELION, VIPER35 or a simple sphere as reference. Identical 3D maps with detailed
features were generated regardless of the reference used (Extended Data Fig. 8). 2D projections of this model were
subsequently used as a reference to re-pick the particles. The resulting
particles were subjected to reiterative reference-free 2D class averaging.
Strict selection of classes showing distinct structural features resulted in a
particle subset used for further three-dimensional classification. The classes
with identical detailed features were merged for further auto-refinement,
applying a soft mask with 6 pixel fall-off around the entire molecule, to
produce the final density map with an overall resolution of 4 Å (Extended
Data 2c). The resolution was estimated based on the gold-standard Fourier shell
correlation (FSC) method using the 0.143 criterion36. The chirality of the final map was validated by model
building of side chains within α helixes. All density maps were sharpened
by applying temperature factor that was estimated using post-processing in
RELION. For visualization, the density maps were filtered based on the local
resolution determined using half-reconstructions as input maps. Chimera37 and PyMOL38 were used for graphic visualization.Model buildingAb initio modeling of the entire Htt-HAP40 complex was
performed in COOT39, using secondary
structure predictions calculated by PSIPRED19 and the densities of bulky side chains to determine registers of
the residues. Regions (1-90, 323-342, 403-660, 960-977, 1049-1057, 1103-1120,
1158-1222, 1319-1347, 1372-1418, 1504-1510, 1549-1556, 1714-1728, 1855-1881,
2063-2091, 2325-2347, 2472-2490, 2580-2582, 2627-2660, 2681-2687, 2926-2944 and
3099-3138) of Htt and regions (1-41, 217-257, 300-313 and 365-371) of HAP40 were
not built in the final model, as no well-resolved densities were present in the
map. Map refinement was carried out using Phenix.real_space_refine40 against the overall map at resolution 4
Å, with secondary structure and Ramachandran restrains. The final model
was validated using MolProbity41 (Extended Data Table 1).Data availabilityThe data supporting the findings of this study are available within this
paper, including source data for figures. The cryo-EM map of the 17QHtt-HAP40
complex has been deposited at the Electron Microscopy Data Bank under accession
code EMD-3984. The modeled structure of the 17QHtt-HAP40 complex has been
deposited at the Protein Data Bank under accession code 6EZ8.

AntibodiesThe following antibodies were used: anti-Flag M2 (Sigma), anti-HAP40
SC-69489 (Santa Cruz), anti-Strep (IBA) and anti-Htt MAB2166 (Millipore).

The following antibodies were used: anti-Flag M2 (Sigma), anti-HAP40
SC-69489 (Santa Cruz), anti-Strep (IBA) and anti-Htt MAB2166 (Millipore).

Identification of Htt-interacting proteins2x108 HEK293-based C2.6 cells17 expressing FLAG-tagged full-length polyQ-expanded 46QHtt at low
levels were harvested and lysed within 25 mM Tris, 150 mM NaCl, 0.5 % Tween 20,
1x protease inhibitor (Roche), pH 7.4, followed by centrifugation (14.000 rpm, 1
h). The supernatant was incubated with FLAG beads at 4 °C for 2 h,
followed by three times washing with 25 mM Tris, 150 mM NaCl, 0.02 % Tween 20,
pH 7.4. Proteins bound to the FLAG beads were eluted with 100 mM Glycine, 150 mM
NaCl, 0.02 % Tween, pH 3.5 and immediately neutralized with 1 M Tris (pH 8.0).
The eluted proteins were concentrated and analysed by SDS-PAGE and Coomassie
staining. To identify potential interactors of Htt the lanes were excised,
proteins were in-gel digested using trypsin, and then analyzed by nano liquid
chromatography (C18, 500 x 0.075 mm, 2 µm column, Thermo Fisher
Scientific) and tandem mass spectrometry (QExactive, Thermo Fisher Scientific)
in data-dependent acquisition mode (Top12). Proteins were identified using
Proteome Discoverer 1.4 (Thermo Fisher Scientific) with a peptide FDR ≤
0.01, and enrichment analysis was performed with Perseus 1.4.1.3 using MS1 peak
area for quantification.The identity of FLAG affinity-purified proteins was confirmed by Western
blot analysis following SDS-PAGE with anti-Htt and anti-HAP antibodies.

Identification of Htt-interacting proteins

2x108 HEK293-based C2.6 cells17 expressing FLAG-tagged full-length polyQ-expanded 46QHtt at low
levels were harvested and lysed within 25 mM Tris, 150 mM NaCl, 0.5 % Tween 20,
1x protease inhibitor (Roche), pH 7.4, followed by centrifugation (14.000 rpm, 1
h). The supernatant was incubated with FLAG beads at 4 °C for 2 h,
followed by three times washing with 25 mM Tris, 150 mM NaCl, 0.02 % Tween 20,
pH 7.4. Proteins bound to the FLAG beads were eluted with 100 mM Glycine, 150 mM
NaCl, 0.02 % Tween, pH 3.5 and immediately neutralized with 1 M Tris (pH 8.0).
The eluted proteins were concentrated and analysed by SDS-PAGE and Coomassie
staining. To identify potential interactors of Htt the lanes were excised,
proteins were in-gel digested using trypsin, and then analyzed by nano liquid
chromatography (C18, 500 x 0.075 mm, 2 µm column, Thermo Fisher
Scientific) and tandem mass spectrometry (QExactive, Thermo Fisher Scientific)
in data-dependent acquisition mode (Top12). Proteins were identified using
Proteome Discoverer 1.4 (Thermo Fisher Scientific) with a peptide FDR ≤
0.01, and enrichment analysis was performed with Perseus 1.4.1.3 using MS1 peak
area for quantification.

The identity of FLAG affinity-purified proteins was confirmed by Western
blot analysis following SDS-PAGE with anti-Htt and anti-HAP antibodies.

Generation of a stable human cell lines co-expressing 17QHtt and
HAP40B1.21 cells17 are based on HEK293
cells and express full-length wild type 17QHtt upon induction with doxycycline
(Dox). An expression plasmid pBSK/2-CMV-HAP40-TS was constructed to express the
human HAP40 (F8A) protein (NCBI Ref Seq NP_036283.2) with a C-terminal
Twin-Strep-tag under the control of the hCMV promoter. B1.21 cells were
co-transfected with this plasmid together with a puromycin resistance gene. The
resulting stable cell line (B1.21-HAP40TS) expressing HAP40 at high constitutive
levels and Htt upon induction with Dox, was used for the purification of
Htt-HAP40 complex. Generated cell lines have been tested negative for mycoplasma
by PCR. Cell lines have been authenticated by inducibility of Htt expression
with doxycycline and Western blot analysis.

Generation of a stable human cell lines co-expressing 17QHtt and
HAP40

B1.21 cells17 are based on HEK293
cells and express full-length wild type 17QHtt upon induction with doxycycline
(Dox). An expression plasmid pBSK/2-CMV-HAP40-TS was constructed to express the
human HAP40 (F8A) protein (NCBI Ref Seq NP_036283.2) with a C-terminal
Twin-Strep-tag under the control of the hCMV promoter. B1.21 cells were
co-transfected with this plasmid together with a puromycin resistance gene. The
resulting stable cell line (B1.21-HAP40TS) expressing HAP40 at high constitutive
levels and Htt upon induction with Dox, was used for the purification of
Htt-HAP40 complex. Generated cell lines have been tested negative for mycoplasma
by PCR. Cell lines have been authenticated by inducibility of Htt expression
with doxycycline and Western blot analysis.

Purification of Htt, HAP40 and the Htt-HAP40 complexThe purification of Htt alone has been described17. For purification of the Htt-HAP40 complex,
2x108 B1.21-HAP40TS cells were harvested 72 h after induction
with Dox by centrifugation at 400 g for 10 min. Cells were lysed with 25 mM
HEPES, 300 mM NaCl, 0.5 % Tween 20, protease inhibitor, pH 8.0 by rotation at 4
°C for 30 min followed by centrifugation of the cell lysate at 30,000 g
and clearance by filtration through a 0.2 μm filter. The filtrate was
incubated with Strep-Tactin beads (Qiagen) for 2-3 h at 4 °C. After
washing three times with 25 mM HEPES, 300 mM NaCl, 0.02 % Tween 20, pH 8.0 bound
proteins were eluted with 25 mM HEPES, 300 mM NaCl, 0.02 % Tween 20, 2.5 mM
Desthiobiotin, pH 8.0. The eluate was concentrated using Amicon filters.The Htt-HAP40 complex was further purified by size exclusion
chromatography using a Superose 6 10/300 increase column (GE Healthcare) in
running buffer 25 mM HEPES, 300 mM NaCl, 0.1 % CHAPS and 1 mM DTT, pH 8.0.
Htt-HAP40 eluted in one narrow-based peak and was concentrated with Amicon ultra
100 kDa filters (Millipore).HAP40 was purified from the Htt-HAP40 complex as follows. Htt-HAP40
bound to Strep beads was eluted with 25 mM HEPES, 300 mM NaCl, 0.05 % N-Dodecyl
beta-D-maltoside (DDM) and 2.5 mM Desthiobiotin, pH 8.0. The eluate was
concentrated using Amicon filters. To disrupt the Htt-HAP40 complex, DDM was
added to a final concentration of 0.25 %. After overnight incubation at 4
°C, the Strep eluate was further purified by size exclusion
chromatography using a Superose 6 10/300 increase column in running buffer 25
mM, 300 mM NaCl, 0.1 % CHAPS and 1 mM DTT, pH 8.0 to separate Htt and HAP40.
HAP40 eluted in one narrow-based peak and was concentrated with Amicon ultra 30
kDa filters.

Purification of Htt, HAP40 and the Htt-HAP40 complex

The purification of Htt alone has been described17. For purification of the Htt-HAP40 complex,
2x108 B1.21-HAP40TS cells were harvested 72 h after induction
with Dox by centrifugation at 400 g for 10 min. Cells were lysed with 25 mM
HEPES, 300 mM NaCl, 0.5 % Tween 20, protease inhibitor, pH 8.0 by rotation at 4
°C for 30 min followed by centrifugation of the cell lysate at 30,000 g
and clearance by filtration through a 0.2 μm filter. The filtrate was
incubated with Strep-Tactin beads (Qiagen) for 2-3 h at 4 °C. After
washing three times with 25 mM HEPES, 300 mM NaCl, 0.02 % Tween 20, pH 8.0 bound
proteins were eluted with 25 mM HEPES, 300 mM NaCl, 0.02 % Tween 20, 2.5 mM
Desthiobiotin, pH 8.0. The eluate was concentrated using Amicon filters.

The Htt-HAP40 complex was further purified by size exclusion
chromatography using a Superose 6 10/300 increase column (GE Healthcare) in
running buffer 25 mM HEPES, 300 mM NaCl, 0.1 % CHAPS and 1 mM DTT, pH 8.0.
Htt-HAP40 eluted in one narrow-based peak and was concentrated with Amicon ultra
100 kDa filters (Millipore).

HAP40 was purified from the Htt-HAP40 complex as follows. Htt-HAP40
bound to Strep beads was eluted with 25 mM HEPES, 300 mM NaCl, 0.05 % N-Dodecyl
beta-D-maltoside (DDM) and 2.5 mM Desthiobiotin, pH 8.0. The eluate was
concentrated using Amicon filters. To disrupt the Htt-HAP40 complex, DDM was
added to a final concentration of 0.25 %. After overnight incubation at 4
°C, the Strep eluate was further purified by size exclusion
chromatography using a Superose 6 10/300 increase column in running buffer 25
mM, 300 mM NaCl, 0.1 % CHAPS and 1 mM DTT, pH 8.0 to separate Htt and HAP40.
HAP40 eluted in one narrow-based peak and was concentrated with Amicon ultra 30
kDa filters.

Rate-zonal ultracentrifugation5-20 % sucrose gradients in 25 mM HEPES, 300 mM NaCl, 0.1 % CHAPS, pH
8.0 were generated by an automatic gradient maker (Gradient master, Biocomp
instrument). A volume of 120 μl of FLAG-tag-purified Htt or
Strep-tag-purified Htt-HAP40 complex was laid on top of the gradient and
centrifuged at 39,000 rpm for 16 h using a SW41 rotor in a Beckman
ultracentrifuge. Fractions of the sucrose gradient were collected from the
bottom of the tubes in fractions of 0.5 ml to be analyzed by SDS-PAGE and
Coomassie Blue staining and Western blotting.

Rate-zonal ultracentrifugation

5-20 % sucrose gradients in 25 mM HEPES, 300 mM NaCl, 0.1 % CHAPS, pH
8.0 were generated by an automatic gradient maker (Gradient master, Biocomp
instrument). A volume of 120 μl of FLAG-tag-purified Htt or
Strep-tag-purified Htt-HAP40 complex was laid on top of the gradient and
centrifuged at 39,000 rpm for 16 h using a SW41 rotor in a Beckman
ultracentrifuge. Fractions of the sucrose gradient were collected from the
bottom of the tubes in fractions of 0.5 ml to be analyzed by SDS-PAGE and
Coomassie Blue staining and Western blotting.

Differential scanning fluorimetryProtein thermostability was assessed by differential scanning
fluorimetry30. Protein unfolding was
monitored by the increase in the fluorescence of SYPRO Orange (Invitrogen).
Prior to use, a 100 mM stock of the dye (stored at -20°C) was diluted
1:20 in DMSO and directly added to the sample to a final concentration of 125
µM. The tested proteins were diluted in sample buffer (25 mM HEPES, 300
mM NaCl, 0.1 % CHAPS, 1 mM DTT and 10 % glycerol) to concentrations of 1.6
µM (Htt, Htt-HAP40 complex) and 2 µM (HAP40). The samples were
heated up with a ramp rate of 1 °C/min over a temperature range of 15-95
°C using the qPCR System MX 3005 P (Stratagene). Measurements were
performed in duplicate.

Differential scanning fluorimetry

Protein thermostability was assessed by differential scanning
fluorimetry30. Protein unfolding was
monitored by the increase in the fluorescence of SYPRO Orange (Invitrogen).
Prior to use, a 100 mM stock of the dye (stored at -20°C) was diluted
1:20 in DMSO and directly added to the sample to a final concentration of 125
µM. The tested proteins were diluted in sample buffer (25 mM HEPES, 300
mM NaCl, 0.1 % CHAPS, 1 mM DTT and 10 % glycerol) to concentrations of 1.6
µM (Htt, Htt-HAP40 complex) and 2 µM (HAP40). The samples were
heated up with a ramp rate of 1 °C/min over a temperature range of 15-95
°C using the qPCR System MX 3005 P (Stratagene). Measurements were
performed in duplicate.

Transient expression of HAP40 and HAP40 fragments and interaction studies
with HttPlasmids were generated expressing, under hCMV promoter control, either
full-length HAP40, a N-terminal HAP40 fragment (HAP40-N, encoding aa 1-222), a
C-terminal HAP40 fragment (HAP40-C, encoding aa 249-371) or a HAP40 fragment, in
which the central proline-rich region had been replaced by a flexible linker
(HAP40∆, encoding aa 1-222 linked by a (GGGGS)3 linker to aa
249-371). All HAP40 variants carried a C-terminal Twin-Strep-tag.B1.21 cells induced with Dox to express 17QHtt were transiently
transfected with the plasmids using PEI transfection. At 48 hours after
transfection, the cells were harvested by centrifugation and lysed in 25 mM
HEPES, 300 mM NaCl, 0.5 % Tween 20, 1x protease inhibitor, pH 8.0, followed by
centrifugation (14.000 rpm, 1 h). The supernatant was incubated with Magstrep
beads (IBA) at 4 °C for 2 hours, followed by three times washing with 25
mM HEPES, 300 mM NaCl, 0.02 % Tween 20, pH 8.0. Thereafter, bound proteins were
eluted using Desthiobiotin in SDS loading buffer, followed by SDS-PAGE and
Western blot analysis using anti-FLAG and anti-Strep antibodies for
detection.

Transient expression of HAP40 and HAP40 fragments and interaction studies
with Htt

Plasmids were generated expressing, under hCMV promoter control, either
full-length HAP40, a N-terminal HAP40 fragment (HAP40-N, encoding aa 1-222), a
C-terminal HAP40 fragment (HAP40-C, encoding aa 249-371) or a HAP40 fragment, in
which the central proline-rich region had been replaced by a flexible linker
(HAP40∆, encoding aa 1-222 linked by a (GGGGS)3 linker to aa
249-371). All HAP40 variants carried a C-terminal Twin-Strep-tag.

B1.21 cells induced with Dox to express 17QHtt were transiently
transfected with the plasmids using PEI transfection. At 48 hours after
transfection, the cells were harvested by centrifugation and lysed in 25 mM
HEPES, 300 mM NaCl, 0.5 % Tween 20, 1x protease inhibitor, pH 8.0, followed by
centrifugation (14.000 rpm, 1 h). The supernatant was incubated with Magstrep
beads (IBA) at 4 °C for 2 hours, followed by three times washing with 25
mM HEPES, 300 mM NaCl, 0.02 % Tween 20, pH 8.0. Thereafter, bound proteins were
eluted using Desthiobiotin in SDS loading buffer, followed by SDS-PAGE and
Western blot analysis using anti-FLAG and anti-Strep antibodies for
detection.

Cryo-EM sample preparation and data acquisitionPurified Htt-HAP40 complex was diluted to 0.5 mg/ml with 25mM HEPES, 300
mM NaCl, 0.025 % CHAPS, 1 mM DTT. 4 μl of sample was applied to a
Quantifoil gold grid suspended with monolayer graphene (Graphenea) and vitrified
by plunge-freezing into a liquid ethane/propane mixture using Vitrobot Mark IV
(FEI) with blotting time of 5 s. Data collection was performed on a Titan Krios
microscope (FEI) operated at 300 kV and equipped with a field emission gun, a
Gatan GIF Quantum energy filter and a Gatan K2 Summit direct electron camera.
The calibrated magnification was 105,000 in EFTEM mode, corresponding to a pixel
size of 1.35 Å. Images were collected at a dose rate of 4
electrons/Å2/s, each exposure (8 s exposures time)
comprising 16 sub-frames amounting to a total dose of 32
electrons/Å2/s. Data was recorded using SerialEM31 software and custom macros with defocus
values ranging from -1.4 to -3 μm.

Cryo-EM sample preparation and data acquisition

Purified Htt-HAP40 complex was diluted to 0.5 mg/ml with 25mM HEPES, 300
mM NaCl, 0.025 % CHAPS, 1 mM DTT. 4 μl of sample was applied to a
Quantifoil gold grid suspended with monolayer graphene (Graphenea) and vitrified
by plunge-freezing into a liquid ethane/propane mixture using Vitrobot Mark IV
(FEI) with blotting time of 5 s. Data collection was performed on a Titan Krios
microscope (FEI) operated at 300 kV and equipped with a field emission gun, a
Gatan GIF Quantum energy filter and a Gatan K2 Summit direct electron camera.
The calibrated magnification was 105,000 in EFTEM mode, corresponding to a pixel
size of 1.35 Å. Images were collected at a dose rate of 4
electrons/Å2/s, each exposure (8 s exposures time)
comprising 16 sub-frames amounting to a total dose of 32
electrons/Å2/s. Data was recorded using SerialEM31 software and custom macros with defocus
values ranging from -1.4 to -3 μm.

Image processingMicrograph movie frame stacks were subjected to beam-induced motion
correction by MotionCor232. Most further
processing was performed using RELION33.
The contrast transfer function parameters for each micrograph were determined
with CTFFIND434, and all micrographs with
a resolution limit worse than 4 Å were discarded. Particles were
initially picked with Gautomatch (http://www.mrc-lmb.cam.ac.uk/kzhang/Gautomatch/Gautomatch_Brief_Manual.pdf)
using a sphere as template, and extracted with a 160-pixel by 160-pixel box.
Reference-free 2D class averaging was performed reiteratively, keeping only
particles with well-resolved 2D average for initial model generation. To
validate the ab initio model, 3D classification was performed
using initial models generated by RELION, VIPER35 or a simple sphere as reference. Identical 3D maps with detailed
features were generated regardless of the reference used (Extended Data Fig. 8). 2D projections of this model were
subsequently used as a reference to re-pick the particles. The resulting
particles were subjected to reiterative reference-free 2D class averaging.
Strict selection of classes showing distinct structural features resulted in a
particle subset used for further three-dimensional classification. The classes
with identical detailed features were merged for further auto-refinement,
applying a soft mask with 6 pixel fall-off around the entire molecule, to
produce the final density map with an overall resolution of 4 Å (Extended
Data 2c). The resolution was estimated based on the gold-standard Fourier shell
correlation (FSC) method using the 0.143 criterion36. The chirality of the final map was validated by model
building of side chains within α helixes. All density maps were sharpened
by applying temperature factor that was estimated using post-processing in
RELION. For visualization, the density maps were filtered based on the local
resolution determined using half-reconstructions as input maps. Chimera37 and PyMOL38 were used for graphic visualization.

Image processing

Micrograph movie frame stacks were subjected to beam-induced motion
correction by MotionCor232. Most further
processing was performed using RELION33.
The contrast transfer function parameters for each micrograph were determined
with CTFFIND434, and all micrographs with
a resolution limit worse than 4 Å were discarded. Particles were
initially picked with Gautomatch (http://www.mrc-lmb.cam.ac.uk/kzhang/Gautomatch/Gautomatch_Brief_Manual.pdf)
using a sphere as template, and extracted with a 160-pixel by 160-pixel box.
Reference-free 2D class averaging was performed reiteratively, keeping only
particles with well-resolved 2D average for initial model generation. To
validate the ab initio model, 3D classification was performed
using initial models generated by RELION, VIPER35 or a simple sphere as reference. Identical 3D maps with detailed
features were generated regardless of the reference used (Extended Data Fig. 8). 2D projections of this model were
subsequently used as a reference to re-pick the particles. The resulting
particles were subjected to reiterative reference-free 2D class averaging.
Strict selection of classes showing distinct structural features resulted in a
particle subset used for further three-dimensional classification. The classes
with identical detailed features were merged for further auto-refinement,
applying a soft mask with 6 pixel fall-off around the entire molecule, to
produce the final density map with an overall resolution of 4 Å (Extended
Data 2c). The resolution was estimated based on the gold-standard Fourier shell
correlation (FSC) method using the 0.143 criterion36. The chirality of the final map was validated by model
building of side chains within α helixes. All density maps were sharpened
by applying temperature factor that was estimated using post-processing in
RELION. For visualization, the density maps were filtered based on the local
resolution determined using half-reconstructions as input maps. Chimera37 and PyMOL38 were used for graphic visualization.

Model buildingAb initio modeling of the entire Htt-HAP40 complex was
performed in COOT39, using secondary
structure predictions calculated by PSIPRED19 and the densities of bulky side chains to determine registers of
the residues. Regions (1-90, 323-342, 403-660, 960-977, 1049-1057, 1103-1120,
1158-1222, 1319-1347, 1372-1418, 1504-1510, 1549-1556, 1714-1728, 1855-1881,
2063-2091, 2325-2347, 2472-2490, 2580-2582, 2627-2660, 2681-2687, 2926-2944 and
3099-3138) of Htt and regions (1-41, 217-257, 300-313 and 365-371) of HAP40 were
not built in the final model, as no well-resolved densities were present in the
map. Map refinement was carried out using Phenix.real_space_refine40 against the overall map at resolution 4
Å, with secondary structure and Ramachandran restrains. The final model
was validated using MolProbity41 (Extended Data Table 1).

Model building

Ab initio modeling of the entire Htt-HAP40 complex was
performed in COOT39, using secondary
structure predictions calculated by PSIPRED19 and the densities of bulky side chains to determine registers of
the residues. Regions (1-90, 323-342, 403-660, 960-977, 1049-1057, 1103-1120,
1158-1222, 1319-1347, 1372-1418, 1504-1510, 1549-1556, 1714-1728, 1855-1881,
2063-2091, 2325-2347, 2472-2490, 2580-2582, 2627-2660, 2681-2687, 2926-2944 and
3099-3138) of Htt and regions (1-41, 217-257, 300-313 and 365-371) of HAP40 were
not built in the final model, as no well-resolved densities were present in the
map. Map refinement was carried out using Phenix.real_space_refine40 against the overall map at resolution 4
Å, with secondary structure and Ramachandran restrains. The final model
was validated using MolProbity41 (Extended Data Table 1).

Data availabilityThe data supporting the findings of this study are available within this
paper, including source data for figures. The cryo-EM map of the 17QHtt-HAP40
complex has been deposited at the Electron Microscopy Data Bank under accession
code EMD-3984. The modeled structure of the 17QHtt-HAP40 complex has been
deposited at the Protein Data Bank under accession code 6EZ8.

Data availability

The data supporting the findings of this study are available within this
paper, including source data for figures. The cryo-EM map of the 17QHtt-HAP40
complex has been deposited at the Electron Microscopy Data Bank under accession
code EMD-3984. The modeled structure of the 17QHtt-HAP40 complex has been
deposited at the Protein Data Bank under accession code 6EZ8.

Extended DataExtended Data Fig. 1Sedimentation analysis by rate-zonal ultracentrifugation.a, FLAG-tag purified Htt (top) and Strep-tag purified
Htt-HAP40 complex (bottom) analysed by rate-zonal ultracentrifugation
followed by SDS-PAGE and Coomassie staining. 25 fractions from 5-20 %
sucrose gradients were collected from the bottom of the tube, here showing
fractions 1-18. While Htt alone was present in fractions 1-18, the Htt-HAP40
complex was found mainly in fractions 15-17, indicating lower conformational
heterogeneity. b, Western blot analysis of fractions 10-18 of
the Htt-HAP40 complex. Independent experiments with similar results (n):
n=3. For gel source data, see Supplementary Figure 1.Extended Data Fig. 2Cryo-EM analysis of the Htt-HAP40 complex.a, Representative micrograph of Htt-HAP40 complex.
b, 2D class averages. c, Fourier shell
correlation (FSC) plots. Cyan, gold-standard FSC curve; orange, FSC curve
calculated between the cryo-EM map and refined atomic model. 0.143 and 0.5
FCS cut-off values, respectively, were used as indicated. The
(initial/final) numbers of micrographs and particles were 707/635 and
418,627/98,310, respectively. d, Final density map of the
Htt-HAP40 complex colored according to local resolution. The map was
low-pass filtered to 4.0 Å and sharpened with a B-factor of -174
Å2. e, Detail of the electron density
maps (mesh) for parts of Htt and HAP40.Extended Data Fig. 3Atomic model of Htt within the Htt-HAP40 complex.The atomic model is shown in ribbon representation with the
indicated rainbow color code from N-terminus (blue arrowhead in d) to C-
terminus (red arrowhead in a). a, b,
c, d show different views of the complex as
indicated. Dashed lines mark unresolved regions.Extended Data Fig. 4Amino acid sequences of 17QHtt and HAP40.Structural elements of the atomic models are indicated as follows:
not visible in the model (red box), unstructured region (no box) and
α-helix (yellow box). The sites of previously reported proteases
cleavage and post-translational modifications of Htt1,21,23,27 are indicated as follows: acetylation (dark blue),
palmitoylation (red), phosphorylation (green) and proteolytic cleavage
(cyan).Extended Data Fig. 5PSIPRED secondary structure predictions for Htt and HAP40.Structural elements are indicated as follows: unstructured region
(no box), α-helix (yellow box) and β-sheet (grey box).Extended Data Fig. 6Truncation analysis of HAP40 binding to Htt.a, Schematic representation of the HAP40 constructs
studied (all C-terminally Strep-tagged). b, HAP40 constructs
were co-expressed with C-terminally FLAG-tagged 17QHtt, immunoprecipitated
using Strep-Tactin beads and analyzed by Western blot. Lanes are indicated
as follows: 1, cell lysates; 2, cell lysates upon incubation with Strep
beads; 3, Strep beads eluates. Note that full-length HAP40 and a construct
lacking the central domain immunoprecipitate Htt, but not deletions of the
N- and C-terminal regions of HAP40. Independent experiments with similar
results (n): n=2. For gel source data, see Supplementary Figure
1.Extended Data Fig. 7Evolutionary analysis of Htt.The human Htt model is shown in ribbon representation, colored
according to sequence conservation across 16 metazoan species (Homo
sapiens, Rattus norvegicus, Mus musculus, Sus scrofa, Bos Taurus, Canis
familiaris, Monodelphis domestica, Gallus gallus, Danio rerio, Tetraodon
nigroviridis, Fugu rubripes, Ciona savignyi, Ciona intestinalis,
Strongylocentrotus purpuratus, Tribolium castaneum, Apis
mellifera), using a previously reported sequence alignment12. At the top, the orientation of the
Htt-HAP40 complex is indicated with respect to Fig. 2.Extended Data Fig. 8Workflow for initial model validation for 3D reconstruction of the
Htt-HAP40 complex.A subset of particles with well-resolved 2D average were used for
initial model generation using RELION33 or SPHIRE35. The
resulting models were used as reference for 3D classification of all the
good particles. A featureless sphere was also used as classification
reference. Most of the particles were classified to identical structures
with sufficient detail, indicating no reference bias in the
reconstruction.Extended Data Table 1Cryo-EM data collection, refinement and validation statistics.Huntingtin-HAP40(EMDB-3984)(PDB 6EZ8)
Data collection and processing
Magnification105kVoltage (kV)300Electron exposure
(e−/Å2)32Defocus range (μm)1.4-3.0Pixel size (Å)1.35Symmetry imposedC1Initial particle images (no.)418,627Final particle images (no.)98,310Map resolution (Å)4.0    FSC
threshold0.143Map resolution range (Å)3.6-4.8
Refinement
Initial model used (PDB code)N/AModel resolution (Å)4.1    FSC
threshold0.5Model resolution range (Å)4.1Map sharpening B factor
(Å2)-174Model composition    Non-hydrogen atoms20,491    Protein
residues2.621B factors
(Å2)    Protein24.57R.m.s. deviations    Bond
lengths (Å)0.007    Bond
angles (°)1.42Validation    MolProbity
score1.68    Clashscore4.87    Poor
rotamers (%)0.83Ramachandran plot    Favored
(%)93.59    Allowed
(%)6.14    Disallowed
(%)0.27

Extended Data

Sedimentation analysis by rate-zonal ultracentrifugation.

a, FLAG-tag purified Htt (top) and Strep-tag purified
Htt-HAP40 complex (bottom) analysed by rate-zonal ultracentrifugation
followed by SDS-PAGE and Coomassie staining. 25 fractions from 5-20 %
sucrose gradients were collected from the bottom of the tube, here showing
fractions 1-18. While Htt alone was present in fractions 1-18, the Htt-HAP40
complex was found mainly in fractions 15-17, indicating lower conformational
heterogeneity. b, Western blot analysis of fractions 10-18 of
the Htt-HAP40 complex. Independent experiments with similar results (n):
n=3. For gel source data, see Supplementary Figure 1.

Cryo-EM analysis of the Htt-HAP40 complex.

a, Representative micrograph of Htt-HAP40 complex.
b, 2D class averages. c, Fourier shell
correlation (FSC) plots. Cyan, gold-standard FSC curve; orange, FSC curve
calculated between the cryo-EM map and refined atomic model. 0.143 and 0.5
FCS cut-off values, respectively, were used as indicated. The
(initial/final) numbers of micrographs and particles were 707/635 and
418,627/98,310, respectively. d, Final density map of the
Htt-HAP40 complex colored according to local resolution. The map was
low-pass filtered to 4.0 Å and sharpened with a B-factor of -174
Å2. e, Detail of the electron density
maps (mesh) for parts of Htt and HAP40.

Atomic model of Htt within the Htt-HAP40 complex.

The atomic model is shown in ribbon representation with the
indicated rainbow color code from N-terminus (blue arrowhead in d) to C-
terminus (red arrowhead in a). a, b,
c, d show different views of the complex as
indicated. Dashed lines mark unresolved regions.

Amino acid sequences of 17QHtt and HAP40.

Structural elements of the atomic models are indicated as follows:
not visible in the model (red box), unstructured region (no box) and
α-helix (yellow box). The sites of previously reported proteases
cleavage and post-translational modifications of Htt1,21,23,27 are indicated as follows: acetylation (dark blue),
palmitoylation (red), phosphorylation (green) and proteolytic cleavage
(cyan).

PSIPRED secondary structure predictions for Htt and HAP40.

Structural elements are indicated as follows: unstructured region
(no box), α-helix (yellow box) and β-sheet (grey box).

Truncation analysis of HAP40 binding to Htt.

a, Schematic representation of the HAP40 constructs
studied (all C-terminally Strep-tagged). b, HAP40 constructs
were co-expressed with C-terminally FLAG-tagged 17QHtt, immunoprecipitated
using Strep-Tactin beads and analyzed by Western blot. Lanes are indicated
as follows: 1, cell lysates; 2, cell lysates upon incubation with Strep
beads; 3, Strep beads eluates. Note that full-length HAP40 and a construct
lacking the central domain immunoprecipitate Htt, but not deletions of the
N- and C-terminal regions of HAP40. Independent experiments with similar
results (n): n=2. For gel source data, see Supplementary Figure
1.

Evolutionary analysis of Htt.

The human Htt model is shown in ribbon representation, colored
according to sequence conservation across 16 metazoan species (Homo
sapiens, Rattus norvegicus, Mus musculus, Sus scrofa, Bos Taurus, Canis
familiaris, Monodelphis domestica, Gallus gallus, Danio rerio, Tetraodon
nigroviridis, Fugu rubripes, Ciona savignyi, Ciona intestinalis,
Strongylocentrotus purpuratus, Tribolium castaneum, Apis
mellifera), using a previously reported sequence alignment12. At the top, the orientation of the
Htt-HAP40 complex is indicated with respect to Fig. 2.

Workflow for initial model validation for 3D reconstruction of the
Htt-HAP40 complex.

A subset of particles with well-resolved 2D average were used for
initial model generation using RELION33 or SPHIRE35. The
resulting models were used as reference for 3D classification of all the
good particles. A featureless sphere was also used as classification
reference. Most of the particles were classified to identical structures
with sufficient detail, indicating no reference bias in the
reconstruction.

Cryo-EM data collection, refinement and validation statistics.

Supplementary MaterialReporting summarySupplementary figure

Supplementary Material
