---
pmid: '37098072'
title: Structure of the human respiratory complex II.
authors:
- Du Z
- Zhou X
- Lai Y
- Xu J
- Zhang Y
- Zhou S
- Feng Z
- Yu L
- Tang Y
- Wang W
- Yu L
- Tian C
- Ran T
- Chen H
- Guddat LW
- Liu F
- Gao Y
- Rao Z
- Gong H
journal: Proc Natl Acad Sci U S A
year: '2023'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC10161127
doi: 10.1073/pnas.2216713120
---

# Structure of the human respiratory complex II.
**Authors:** Du Z, Zhou X, Lai Y, Xu J, Zhang Y, Zhou S, Feng Z, Yu L, Tang Y, Wang W, Yu L, Tian C, Ran T, Chen H, Guddat LW, Liu F, Gao Y, Rao Z, Gong H
**Journal:** Proc Natl Acad Sci U S A (2023)
**DOI:** [10.1073/pnas.2216713120](https://doi.org/10.1073/pnas.2216713120)
**PMC:** [PMC10161127](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10161127/)

## Abstract

1. Proc Natl Acad Sci U S A. 2023 May 2;120(18):e2216713120. doi: 
10.1073/pnas.2216713120. Epub 2023 Apr 25.

Structure of the human respiratory complex II.

Du Z(1), Zhou X(2), Lai Y(1), Xu J(1), Zhang Y(1), Zhou S(1), Feng Z(1), Yu 
L(1), Tang Y(1), Wang W(3), Yu L(4), Tian C(4), Ran T(5), Chen H(5), Guddat 
LW(6), Liu F(5), Gao Y(3), Rao Z(1)(3)(6)(7)(8), Gong H(1).

Author information:
(1)State Key Laboratory of Medicinal Chemical Biology, Frontiers Science Center 
for Cell Responses, College of Life Sciences, Nankai University, Tianjin 300353, 
China.
(2)State Key Laboratory of Reproductive Regulation and Breeding of Grassland 
Livestock, School of Life Sciences, Inner Mongolia University, Hohhot 010070, 
China.
(3)Shanghai Institute for Advanced Immunochemical Studies and School of Life 
Science and Technology, ShanghaiTech University, Shanghai 201210, China.
(4)High Magnetic Field Laboratory, Chinese Academy of Sciences, Hefei 230031, 
China.
(5)Innovative Center For Pathogen Research, Guangzhou Laboratory, Guangzhou 
510005, China.
(6)School of Chemistry and Molecular Biosciences, The University of Queensland, 
Brisbane, QLD 4072, Australia.
(7)National Laboratory of Biomacromolecules, CAS Center for Excellence in 
Biomacromolecules, Institute of Biophysics, Chinese Academy of Sciences, Beijing 
100101, China.
(8)Laboratory of Structural Biology, Tsinghua University, Beijing 100084, China.

Human complex II is a key protein complex that links two essential 
energy-producing processes: the tricarboxylic acid cycle and oxidative 
phosphorylation. Deficiencies due to mutagenesis have been shown to cause 
mitochondrial disease and some types of cancers. However, the structure of this 
complex is yet to be resolved, hindering a comprehensive understanding of the 
functional aspects of this molecular machine. Here, we have determined the 
structure of human complex II in the presence of ubiquinone at 2.86 Å resolution 
by cryoelectron microscopy, showing it comprises two water-soluble subunits, 
SDHA and SDHB, and two membrane-spanning subunits, SDHC and SDHD. This structure 
allows us to propose a route for electron transfer. In addition, clinically 
relevant mutations are mapped onto the structure. This mapping provides a 
molecular understanding to explain why these variants have the potential to 
produce disease.

DOI: 10.1073/pnas.2216713120
PMCID: PMC10161127
PMID: 37098072 [Indexed for MEDLINE]

Conflict of interest statement: The authors declare no competing interest.

## Full Text

The mitochondrial oxidative phosphorylation (OXPHOS) system is central to cellular metabolism during aerobic respiration in eukaryotic cells (1, 2). It comprises five enzymatic complexes (complexes I-V, CI-CV) along with two mobile electron carriers (ubiquinone and cytochrome c) and is located in the inner mitochondrial membrane. The respiratory chain (also called electron transport chain) consists of complexes I-IV. It oxidizes the reducing equivalents in nicotinamide adenine dinucleotide (NADH) and succinate using molecular oxygen and couples the translocation of protons from the mitochondrial matrix into the intermembrane space. This process drives the production of adenosine triphosphatase (ATP) by F1Fo-ATP synthase (complex V).

Respiratory complex II plays a crucial role in the two primary key metabolic pathways for generating ATP: the Krebs cycle (also known as tricarboxylic acid, TCA) and the OXPHOS pathway (3). It can function in either aerobic energy metabolism [termed succinate-ubiquinone reductase (SQR) or succinate dehydrogenase (SDH)] or anaerobic energy metabolism [termed quinol-fumarate reductase (QFR) or fumarate reductase (FRD)] in the mammalian mitochondria (3, 4). Human respiratory complex II (CII) is the source of a broad range of human diseases related to its malfunction. These include several types of cancers (5, 6) and neurodegenerative disorders (7). Thus, understanding how human CII operates can support the development of therapeutic approaches to treat these diseases. To date, the complete structures of complex II from Wolinella succinogenes (8), Escherichia coli (9, 10), Sus scrofa (11), Gallus gallus (12), Ascaris suum (13), and Mycobacterium smegmatis (14, 15) have been determined. Remarkably, although the structural and mechanistic properties of CII from these species have been studied extensively, little is known about the structure and machinery of human CII. Here, we report the cryoelectron microscopy (cryo-EM) structure of this complex, which advances our understanding of how it transports electrons, as well as giving insights into the molecular basis for diseases associated with its defects.

Results and DiscussionPurification and Characterization of Human CII.To isolate human CII, a 3× Flag tag was incorporated at the C terminus of the SDHD subunit. This construct was then inserted into an expression plasmid, which was then transferred into human embryonic kidney (HEK) 293F cells (SI Appendix, Fig. S1A). FLAG affinity chromatography and gel filtration column were then performed to obtain a supramolecular assembly (SI Appendix, Fig. S1B). A single band was shown on blue native polyacrylamide gel electrophoresis (BN-PAGE), which indicated that human CII is present (SI Appendix, Fig. S1C). All the four subunits (SDHA, SDHB, SDHC, and SDHD) were detected by SDS-PAGE (SI Appendix, Fig. S1D) and mass spectrometry (MS) (SI Appendix, Table S1). Electron paramagnetic resonance (EPR) spectra revealed the presence of redox centers, containing the [2Fe-2S] cluster, the [4Fe-4S] cluster, the [3Fe-4S] cluster, and one b-type heme group (SI Appendix, Fig. S1E). Human CII coupled succinate oxidation to ubiquinone reduction gave a catalytic constant (kcat) 0.67 ± 0.02 s−1 (SI Appendix, Table S2), confirming that the purified human CII is a functioning complex.Structure of the Human CII.The structure of human CII was determined by cryo-EM to an overall resolution of 2.86 Å (SI Appendix, Figs. S2 and S3 and Tables S3 and S4). We observed all four subunits in a monomeric assembly (Fig. 1A). This arrangement is similar to that in W. succinogenes QFR (8), E. coli QFR (9), S. scrofa SQR (11), and M. smegmatis SQR (15) with human CII also adopting a mushroom-like shape with a hydrophilic head and a hydrophobic transmembrane-anchored tail (Fig. 1A). The hydrophilic head of human CII consists of the flavin adenine dinucleotide (FAD)–binding protein (SDHA) and the iron–sulfur protein (SDHB) (Fig. 1B). The entire hydrophobic domain contains two membrane-anchored subunits: SDHC and SDHD (Fig. 1B). The SDHA subunit can be divided into four domains (11) termed the FAD-binding domain, capping domain, helical domain, and C-terminal domain (Fig. 2A). The active site for succinate–fumarate interconversion is located at the interface between the FAD-binding domain and the capping domain. SDHB is a small iron–sulfur protein harboring a [2Fe-2S], a [4Fe-4S], and a [3Fe-4S] cluster and organized into two domains in a butterfly-like shape (Fig. 2B). The [2Fe-2S] cluster is coordinated by a loop located in the N-terminal domain, and the [4Fe-4S] and [3Fe-4S] clusters are assembled on the C-terminal domain (Fig. 2B). Overall, these soluble domains (SDHA and SDHB) are similar in fold to the previously reported bacterial and mitochondrial respiratory CIIs (8–15) (SI Appendix, Fig. S4). In contrast, the structures and components of the integral membrane domain of CIIs can vary containing 1, 2, or 3 membrane-bound subunits, possess varying prosthetic groups (heme b or Fe-S cluster), and even have different numbers of heme b groups (0, 1, or 2) (8–15). The two membrane-anchored proteins (SDHC and SDHD) in human CII, each with three transmembrane helices, contain only one heme b group. The transmembrane helices of SDHC and SDHD exhibit a similar fold to each other (Fig. 2C). Notably, the structural features of the membrane domain of human CII are also most similar to that of the E. coli (10) and S. scrofa (11) SQRs (SI Appendix, Fig. S4).Fig. 1.Structure of human respiratory complex II. (A) Single-particle cryo-EM density map of human CII at 2.86 Å resolution. Two views are shown: the front view (Left) and the back view (Right). Structure of the human CII atomic model from the same views is shown in cartoon representation. SDHA, SDHB, SDHC, and SDHD are colored in blue, pink, green, and purple, respectively. The complex has approximate dimensions of 115 by 70 Å (height and width). (B) The soluble region (SDHA and SDHB subunits) and transmembrane anchor region (SDHC and SDHD subunits) are shown in cartoon representation and indicated in an open-book configuration. IMM, inner mitochondrial membrane.Fig. 2.Composition of human CII. (A) SDHA consists of four subdomains: the FAD-binding domain (residues A51 to A315 and A403 to A487), the capping domain (residues A316 to A402), the helical domain (residues A488 to A585), and the C-terminal domain (residues A596 to A664). FAD is presented as a stick model. (B) SDHB contains two domains: the N-terminal domain (residues A35 to A142) and the C-terminal domain (residues A143 to A273). Three FeS clusters are shown in sphere representation. (C) Two integral membrane domains of SDHC and SDHD are shown in two orthogonal views. Ubiquinone (UQ), phosphatidylethanolamine (PE), and heme b are shown as stick models. IMM, inner mitochondrial membrane.Density for a phosphatidylethanolamine (PE) unit is observed (SI Appendix, Fig. S3) and positioned at the interface on the mitochondrial inner membrane side between SDHC and SDHD and located directly below the heme b group (Fig. 2C). PE is also observed in S. scrofa SQR (11) (SI Appendix, Fig. S5A). However, in E. coli SQR (10), the position equivalent to the PE site is occupied by a cardiolipin (CDL) molecule, and in the W. succinogenes QFR (8) and M. smegmatis Sdh2 (14) (SI Appendix, Fig. S5A), the position equivalent to the PE site is occupied by a heme b. Structure determination of membrane proteins has highlighted the many roles played by lipid molecules in influencing overall structural and functional integrity (16). It has been shown that PE lipids play a role in stabilizing the inward conformation of E. coli xylose transporter XylE and lactose permease LacY (17), maintaining the structural stability of the E. coli ABC transporter McjD by providing the correct hydrophobic environment (18) and facilitating the functional dimer formation of Aspergillus nidulans purine transporter UapA (19). CDL was observed to stabilize and facilitate the assembly of the E. coli Na+/H+ antiporter NhaA (20). In the present study, we used molecular dynamics (MD) simulations to understand how PE binding and dissociation events regulate the conformational stability of human CII at the molecular level. The results of the MD simulations suggest that the PE molecule here is also linked to protein stability (SI Appendix, Fig. S5B). In addition, the heme cofactor is also crucial for the heme protein to achieve its proper fold and thus become a stable, functional structure (21). Taken together, the ligand in human CII and its counterparts in other organisms are believed to play crucial roles by modulating the conformational, structural, and functional properties of the protein, and its heterogeneity may be related to species specificity.Electron Transfer Pathway in the Human CII.The prosthetic groups (FAD, [2Fe-2S], [4Fe-4S], [3Fe-4S], and a b-type heme groups) mediating the electron transfer were all unambiguously assigned into the cryo-EM map (SI Appendix, Fig. S3). The edge-to-edge distance between these redox-active prosthetic groups is less than 14 Å (Fig. 3), a distance range that can efficiently support the delivery of electrons between these redox centers (22). The EPR signal (g = 2.0) indicates the presence of ubiquinone (UQ) (SI Appendix, Fig. S1E). UQ is also observed to bind at the entrance of the pocket formed by the transmembrane helix I of SDHC, transmembrane helix II of SDHD, and the C-terminal segment of SDHB. It interacts with Pro-SDHB197, Trp-SDHB201, Ile-SDHB246, Ile-SDHC56, Trp-SDHC61, Met-SDHC65, Ile-SDHC69, and Tyr-SDHD114 (Fig. 3). The edge-to-edge distances between UQ and heme b and [3Fe-4S] are about 6.6 Å and 6.6 Å, respectively, which are efficient to promote electron transfer (22).Fig. 3.Prosthetic groups in human CII and the pathway of electron transfer. (Left) Overall structure of human CII is shown together with the prosthetic groups. A zoomed-in view of UQ-binding site representation indicates the interactions between the UQ and surrounding residues. (Right) Prosthetic groups in the path of electron transfer are labeled with edge-to-edge distances. Five prosthetic groups, FAD, [2Fe-2S], [4Fe-4S], [3Fe-4S], and a heme b are required for electron transfer flow from succinate to ubiquinone. Prosthetic groups are shown in stick models or as spheres. The transmembrane helices are labeled.In E. coli (10) and S. scrofa (11) SQRs, the electrons released from succinate oxidation are transferred to the [3Fe-4S] by sequential delivery through a chain of redox centers consisting of FAD, [2Fe-2S], and [4Fe-4S]. Although the electrons from the [3Fe-4S] cluster can be transferred either to heme b or ubiquinone, ubiquinone is preferable (10, 11). It has been shown that the heme b is neither an absolute structural requirement nor essential for quinone reduction (23, 24). In addition, once the heme b is reduced, it can also in turn reduce ubiquinone (10, 11). Therefore, the heme b is proposed to serve as an electron sink in the electron transfer pathway (25, 26). In this process, quinone likely receives the first electron from the [3Fe-4S] cluster to form the semiquinone, and then, the semiquinone radical is stabilized by rapid electron equilibration between the heme b and ubiquinone (26, 27). Finally, the semiquinone accepts a second electron to complete full reduction, which is transported from either the heme b or the [3Fe-4S] cluster. Given the overall structural features and arrangement of redox centers between human CII and the SQRs from E. coli (10) and S. scrofa (11) (SI Appendix, Fig. S4), they likely have a very similar electron transfer pathway. Based on these structural similarities, we propose that the human CII succinate- and ubiquinone-binding sites are likely to be connected by a similar chain of redox centers that include FAD, [2Fe-2S], [4Fe-4S], [3Fe-4S], and heme b (Fig. 3). However, although the structural features and electron transfer pathway of the human CII appear to be similar to that of E. coli (10) and S. scrofa (11) SQRs, there is some amino acid sequence variability (SI Appendix, Fig. S4B). Therefore, we have estimated how the sequence variability affects the structure using MD simulations. Considering the high sequence identity between human CII and S. scrofa SQR among the available structures of complex II, the potential implication of a slight difference of residue sequence between these two species was investigated (SI Appendix, Fig. S6A). The MD simulations indicate that the structure of human CII is more conformationally stable than that of the S. scrofa SQR (SI Appendix, Fig. S6B), which may be potentially useful for electron transport.Human CII Mutations Related with Diseases.Mutants of human CII result in mitochondrial dysfunction. This can manifest in two clinical diseases: either as a primary mitochondrial disease or as susceptibility to tumorigenesis (28). The clinically affected individuals harbor genetic variants within the SDHA (29–34), SDHB (30, 35–40), and SDHD (41, 42) subunits. However, there are no reports of primary mitochondrial dysfunction involving SDHC variants. Knowledge of the exact location of the mutations is important to explain the molecular basis for a specific disease (Fig. 4). The dominant substitution of Arg451Cys (29) in the subunit SDHA abolishes a hydrogen bond with the FAD cofactor (Fig. 4A), thus potentially affecting FAD binding. The Thr508Ile and Ser509Leu mutants in SDHA have been reported in one individual (30). These mutations are situated at the SDHA–SDHB interface (Fig. 4B) and result in the loss of hydrogen-bonding interactions with neighboring polar residues and are therefore purported to affect heterodimer formation. Arg554 and Gly555 within a helical domain of SDHA are also situated at the SDHA–SDHB interface. The Arg554Trp (31) mutation abrogates a hydrogen bond with Thr551 within SDHA (Fig. 4C). The Gly555Glu (32–34) replacement creates a negative charge at an external loop of SDHA (Fig. 4C). This change may impact the folding of this segment and assembly with SDHB. The Asp48Val substitution in SDHB is the change that is most frequently reported in clinical studies (30, 35–38). Substitution of a negatively charged aspartate with an uncharged valine introduces a hydrophobic residue into a hydrophilic region between the SDHA–SDHB interface, reducing the stability of this contact (Fig. 4D). The substitution of Ala102Thr (39) results in the formation of new polar bonds with Cys189, and Cys191 within the SDHB subunit (Fig. 4E), possibly altering the local tertiary structure around the [4Fe-4S] cluster. Interestingly, an adjacent variant, Met103Thr of SDHB, is also reported in a case of mitochondrial disease (36). Met103 is located proximal to the [2Fe-2S] cluster, and its substitution with threonine may influence the structure of surrounding hydrophobic region (Fig. 4F). The Arg230His variant of SDHB is associated with both mitochondrial complex II deficiency and susceptibility to tumorigenesis growth (38, 40). This substitution removes a salt bridge with Asp224 in the SDHB subunit (Fig. 4G). The Leu257Val (38) substitution in SDHB alters the interaction with the [4Fe-4S] cluster (Fig. 4H), and the Glu69Lys (41) change in SDHD will affect interactions with heme b (Fig. 4I). These two mutations may induce the change in the orientation of the [4Fe-4S] cluster and heme b, respectively, and affect the capacity for electron transfer. The Asp92Gly (42) substitution removes a hydrogen-bonding interaction with the residue Cys150 (Fig. 4J), which may affect the stability of transmembrane helices of the SDHD subunit on the intermembrane space side. In summary, these substitutions tend to occur in regions of the proteins that result in instability of the complex, leading to a functional deterioration of the complex. However, more studies are needed to reveal the exact interactions between these substitutions and the relevant disease states. In addition, as for the human mutations, they could generally be mapped on the available homologous structures from different organisms. It is remarkable, however, that human CII does not have 100% amino acid sequence identity with any of the available structures of complex II (SI Appendix, Figs. S4B, S6A and S7). This suggests that the mutations of human CII do not always accurately map on the existing structures. Hence, the current structure can directly reflect the potential impacts that result from the mutations of human CII. Overall, this structure improves our understanding of the molecular basis for human CII mutation-related disease and lays the foundation for the development of rational treatments for these diseases.Fig. 4.Location of the ten clinically relevant mutations in human CII. (A-J) The mutants are highlighted in green, and the surrounding residues are shown in gray sticks. The central panel shows an overview of the locations of the ten mutations.

Results and Discussion

Purification and Characterization of Human CII.To isolate human CII, a 3× Flag tag was incorporated at the C terminus of the SDHD subunit. This construct was then inserted into an expression plasmid, which was then transferred into human embryonic kidney (HEK) 293F cells (SI Appendix, Fig. S1A). FLAG affinity chromatography and gel filtration column were then performed to obtain a supramolecular assembly (SI Appendix, Fig. S1B). A single band was shown on blue native polyacrylamide gel electrophoresis (BN-PAGE), which indicated that human CII is present (SI Appendix, Fig. S1C). All the four subunits (SDHA, SDHB, SDHC, and SDHD) were detected by SDS-PAGE (SI Appendix, Fig. S1D) and mass spectrometry (MS) (SI Appendix, Table S1). Electron paramagnetic resonance (EPR) spectra revealed the presence of redox centers, containing the [2Fe-2S] cluster, the [4Fe-4S] cluster, the [3Fe-4S] cluster, and one b-type heme group (SI Appendix, Fig. S1E). Human CII coupled succinate oxidation to ubiquinone reduction gave a catalytic constant (kcat) 0.67 ± 0.02 s−1 (SI Appendix, Table S2), confirming that the purified human CII is a functioning complex.

Purification and Characterization of Human CII.

To isolate human CII, a 3× Flag tag was incorporated at the C terminus of the SDHD subunit. This construct was then inserted into an expression plasmid, which was then transferred into human embryonic kidney (HEK) 293F cells (SI Appendix, Fig. S1A). FLAG affinity chromatography and gel filtration column were then performed to obtain a supramolecular assembly (SI Appendix, Fig. S1B). A single band was shown on blue native polyacrylamide gel electrophoresis (BN-PAGE), which indicated that human CII is present (SI Appendix, Fig. S1C). All the four subunits (SDHA, SDHB, SDHC, and SDHD) were detected by SDS-PAGE (SI Appendix, Fig. S1D) and mass spectrometry (MS) (SI Appendix, Table S1). Electron paramagnetic resonance (EPR) spectra revealed the presence of redox centers, containing the [2Fe-2S] cluster, the [4Fe-4S] cluster, the [3Fe-4S] cluster, and one b-type heme group (SI Appendix, Fig. S1E). Human CII coupled succinate oxidation to ubiquinone reduction gave a catalytic constant (kcat) 0.67 ± 0.02 s−1 (SI Appendix, Table S2), confirming that the purified human CII is a functioning complex.

Structure of the Human CII.The structure of human CII was determined by cryo-EM to an overall resolution of 2.86 Å (SI Appendix, Figs. S2 and S3 and Tables S3 and S4). We observed all four subunits in a monomeric assembly (Fig. 1A). This arrangement is similar to that in W. succinogenes QFR (8), E. coli QFR (9), S. scrofa SQR (11), and M. smegmatis SQR (15) with human CII also adopting a mushroom-like shape with a hydrophilic head and a hydrophobic transmembrane-anchored tail (Fig. 1A). The hydrophilic head of human CII consists of the flavin adenine dinucleotide (FAD)–binding protein (SDHA) and the iron–sulfur protein (SDHB) (Fig. 1B). The entire hydrophobic domain contains two membrane-anchored subunits: SDHC and SDHD (Fig. 1B). The SDHA subunit can be divided into four domains (11) termed the FAD-binding domain, capping domain, helical domain, and C-terminal domain (Fig. 2A). The active site for succinate–fumarate interconversion is located at the interface between the FAD-binding domain and the capping domain. SDHB is a small iron–sulfur protein harboring a [2Fe-2S], a [4Fe-4S], and a [3Fe-4S] cluster and organized into two domains in a butterfly-like shape (Fig. 2B). The [2Fe-2S] cluster is coordinated by a loop located in the N-terminal domain, and the [4Fe-4S] and [3Fe-4S] clusters are assembled on the C-terminal domain (Fig. 2B). Overall, these soluble domains (SDHA and SDHB) are similar in fold to the previously reported bacterial and mitochondrial respiratory CIIs (8–15) (SI Appendix, Fig. S4). In contrast, the structures and components of the integral membrane domain of CIIs can vary containing 1, 2, or 3 membrane-bound subunits, possess varying prosthetic groups (heme b or Fe-S cluster), and even have different numbers of heme b groups (0, 1, or 2) (8–15). The two membrane-anchored proteins (SDHC and SDHD) in human CII, each with three transmembrane helices, contain only one heme b group. The transmembrane helices of SDHC and SDHD exhibit a similar fold to each other (Fig. 2C). Notably, the structural features of the membrane domain of human CII are also most similar to that of the E. coli (10) and S. scrofa (11) SQRs (SI Appendix, Fig. S4).Fig. 1.Structure of human respiratory complex II. (A) Single-particle cryo-EM density map of human CII at 2.86 Å resolution. Two views are shown: the front view (Left) and the back view (Right). Structure of the human CII atomic model from the same views is shown in cartoon representation. SDHA, SDHB, SDHC, and SDHD are colored in blue, pink, green, and purple, respectively. The complex has approximate dimensions of 115 by 70 Å (height and width). (B) The soluble region (SDHA and SDHB subunits) and transmembrane anchor region (SDHC and SDHD subunits) are shown in cartoon representation and indicated in an open-book configuration. IMM, inner mitochondrial membrane.Fig. 2.Composition of human CII. (A) SDHA consists of four subdomains: the FAD-binding domain (residues A51 to A315 and A403 to A487), the capping domain (residues A316 to A402), the helical domain (residues A488 to A585), and the C-terminal domain (residues A596 to A664). FAD is presented as a stick model. (B) SDHB contains two domains: the N-terminal domain (residues A35 to A142) and the C-terminal domain (residues A143 to A273). Three FeS clusters are shown in sphere representation. (C) Two integral membrane domains of SDHC and SDHD are shown in two orthogonal views. Ubiquinone (UQ), phosphatidylethanolamine (PE), and heme b are shown as stick models. IMM, inner mitochondrial membrane.Density for a phosphatidylethanolamine (PE) unit is observed (SI Appendix, Fig. S3) and positioned at the interface on the mitochondrial inner membrane side between SDHC and SDHD and located directly below the heme b group (Fig. 2C). PE is also observed in S. scrofa SQR (11) (SI Appendix, Fig. S5A). However, in E. coli SQR (10), the position equivalent to the PE site is occupied by a cardiolipin (CDL) molecule, and in the W. succinogenes QFR (8) and M. smegmatis Sdh2 (14) (SI Appendix, Fig. S5A), the position equivalent to the PE site is occupied by a heme b. Structure determination of membrane proteins has highlighted the many roles played by lipid molecules in influencing overall structural and functional integrity (16). It has been shown that PE lipids play a role in stabilizing the inward conformation of E. coli xylose transporter XylE and lactose permease LacY (17), maintaining the structural stability of the E. coli ABC transporter McjD by providing the correct hydrophobic environment (18) and facilitating the functional dimer formation of Aspergillus nidulans purine transporter UapA (19). CDL was observed to stabilize and facilitate the assembly of the E. coli Na+/H+ antiporter NhaA (20). In the present study, we used molecular dynamics (MD) simulations to understand how PE binding and dissociation events regulate the conformational stability of human CII at the molecular level. The results of the MD simulations suggest that the PE molecule here is also linked to protein stability (SI Appendix, Fig. S5B). In addition, the heme cofactor is also crucial for the heme protein to achieve its proper fold and thus become a stable, functional structure (21). Taken together, the ligand in human CII and its counterparts in other organisms are believed to play crucial roles by modulating the conformational, structural, and functional properties of the protein, and its heterogeneity may be related to species specificity.

Structure of the Human CII.

The structure of human CII was determined by cryo-EM to an overall resolution of 2.86 Å (SI Appendix, Figs. S2 and S3 and Tables S3 and S4). We observed all four subunits in a monomeric assembly (Fig. 1A). This arrangement is similar to that in W. succinogenes QFR (8), E. coli QFR (9), S. scrofa SQR (11), and M. smegmatis SQR (15) with human CII also adopting a mushroom-like shape with a hydrophilic head and a hydrophobic transmembrane-anchored tail (Fig. 1A). The hydrophilic head of human CII consists of the flavin adenine dinucleotide (FAD)–binding protein (SDHA) and the iron–sulfur protein (SDHB) (Fig. 1B). The entire hydrophobic domain contains two membrane-anchored subunits: SDHC and SDHD (Fig. 1B). The SDHA subunit can be divided into four domains (11) termed the FAD-binding domain, capping domain, helical domain, and C-terminal domain (Fig. 2A). The active site for succinate–fumarate interconversion is located at the interface between the FAD-binding domain and the capping domain. SDHB is a small iron–sulfur protein harboring a [2Fe-2S], a [4Fe-4S], and a [3Fe-4S] cluster and organized into two domains in a butterfly-like shape (Fig. 2B). The [2Fe-2S] cluster is coordinated by a loop located in the N-terminal domain, and the [4Fe-4S] and [3Fe-4S] clusters are assembled on the C-terminal domain (Fig. 2B). Overall, these soluble domains (SDHA and SDHB) are similar in fold to the previously reported bacterial and mitochondrial respiratory CIIs (8–15) (SI Appendix, Fig. S4). In contrast, the structures and components of the integral membrane domain of CIIs can vary containing 1, 2, or 3 membrane-bound subunits, possess varying prosthetic groups (heme b or Fe-S cluster), and even have different numbers of heme b groups (0, 1, or 2) (8–15). The two membrane-anchored proteins (SDHC and SDHD) in human CII, each with three transmembrane helices, contain only one heme b group. The transmembrane helices of SDHC and SDHD exhibit a similar fold to each other (Fig. 2C). Notably, the structural features of the membrane domain of human CII are also most similar to that of the E. coli (10) and S. scrofa (11) SQRs (SI Appendix, Fig. S4).

Structure of human respiratory complex II. (A) Single-particle cryo-EM density map of human CII at 2.86 Å resolution. Two views are shown: the front view (Left) and the back view (Right). Structure of the human CII atomic model from the same views is shown in cartoon representation. SDHA, SDHB, SDHC, and SDHD are colored in blue, pink, green, and purple, respectively. The complex has approximate dimensions of 115 by 70 Å (height and width). (B) The soluble region (SDHA and SDHB subunits) and transmembrane anchor region (SDHC and SDHD subunits) are shown in cartoon representation and indicated in an open-book configuration. IMM, inner mitochondrial membrane.

Composition of human CII. (A) SDHA consists of four subdomains: the FAD-binding domain (residues A51 to A315 and A403 to A487), the capping domain (residues A316 to A402), the helical domain (residues A488 to A585), and the C-terminal domain (residues A596 to A664). FAD is presented as a stick model. (B) SDHB contains two domains: the N-terminal domain (residues A35 to A142) and the C-terminal domain (residues A143 to A273). Three FeS clusters are shown in sphere representation. (C) Two integral membrane domains of SDHC and SDHD are shown in two orthogonal views. Ubiquinone (UQ), phosphatidylethanolamine (PE), and heme b are shown as stick models. IMM, inner mitochondrial membrane.

Density for a phosphatidylethanolamine (PE) unit is observed (SI Appendix, Fig. S3) and positioned at the interface on the mitochondrial inner membrane side between SDHC and SDHD and located directly below the heme b group (Fig. 2C). PE is also observed in S. scrofa SQR (11) (SI Appendix, Fig. S5A). However, in E. coli SQR (10), the position equivalent to the PE site is occupied by a cardiolipin (CDL) molecule, and in the W. succinogenes QFR (8) and M. smegmatis Sdh2 (14) (SI Appendix, Fig. S5A), the position equivalent to the PE site is occupied by a heme b. Structure determination of membrane proteins has highlighted the many roles played by lipid molecules in influencing overall structural and functional integrity (16). It has been shown that PE lipids play a role in stabilizing the inward conformation of E. coli xylose transporter XylE and lactose permease LacY (17), maintaining the structural stability of the E. coli ABC transporter McjD by providing the correct hydrophobic environment (18) and facilitating the functional dimer formation of Aspergillus nidulans purine transporter UapA (19). CDL was observed to stabilize and facilitate the assembly of the E. coli Na+/H+ antiporter NhaA (20). In the present study, we used molecular dynamics (MD) simulations to understand how PE binding and dissociation events regulate the conformational stability of human CII at the molecular level. The results of the MD simulations suggest that the PE molecule here is also linked to protein stability (SI Appendix, Fig. S5B). In addition, the heme cofactor is also crucial for the heme protein to achieve its proper fold and thus become a stable, functional structure (21). Taken together, the ligand in human CII and its counterparts in other organisms are believed to play crucial roles by modulating the conformational, structural, and functional properties of the protein, and its heterogeneity may be related to species specificity.

Electron Transfer Pathway in the Human CII.The prosthetic groups (FAD, [2Fe-2S], [4Fe-4S], [3Fe-4S], and a b-type heme groups) mediating the electron transfer were all unambiguously assigned into the cryo-EM map (SI Appendix, Fig. S3). The edge-to-edge distance between these redox-active prosthetic groups is less than 14 Å (Fig. 3), a distance range that can efficiently support the delivery of electrons between these redox centers (22). The EPR signal (g = 2.0) indicates the presence of ubiquinone (UQ) (SI Appendix, Fig. S1E). UQ is also observed to bind at the entrance of the pocket formed by the transmembrane helix I of SDHC, transmembrane helix II of SDHD, and the C-terminal segment of SDHB. It interacts with Pro-SDHB197, Trp-SDHB201, Ile-SDHB246, Ile-SDHC56, Trp-SDHC61, Met-SDHC65, Ile-SDHC69, and Tyr-SDHD114 (Fig. 3). The edge-to-edge distances between UQ and heme b and [3Fe-4S] are about 6.6 Å and 6.6 Å, respectively, which are efficient to promote electron transfer (22).Fig. 3.Prosthetic groups in human CII and the pathway of electron transfer. (Left) Overall structure of human CII is shown together with the prosthetic groups. A zoomed-in view of UQ-binding site representation indicates the interactions between the UQ and surrounding residues. (Right) Prosthetic groups in the path of electron transfer are labeled with edge-to-edge distances. Five prosthetic groups, FAD, [2Fe-2S], [4Fe-4S], [3Fe-4S], and a heme b are required for electron transfer flow from succinate to ubiquinone. Prosthetic groups are shown in stick models or as spheres. The transmembrane helices are labeled.In E. coli (10) and S. scrofa (11) SQRs, the electrons released from succinate oxidation are transferred to the [3Fe-4S] by sequential delivery through a chain of redox centers consisting of FAD, [2Fe-2S], and [4Fe-4S]. Although the electrons from the [3Fe-4S] cluster can be transferred either to heme b or ubiquinone, ubiquinone is preferable (10, 11). It has been shown that the heme b is neither an absolute structural requirement nor essential for quinone reduction (23, 24). In addition, once the heme b is reduced, it can also in turn reduce ubiquinone (10, 11). Therefore, the heme b is proposed to serve as an electron sink in the electron transfer pathway (25, 26). In this process, quinone likely receives the first electron from the [3Fe-4S] cluster to form the semiquinone, and then, the semiquinone radical is stabilized by rapid electron equilibration between the heme b and ubiquinone (26, 27). Finally, the semiquinone accepts a second electron to complete full reduction, which is transported from either the heme b or the [3Fe-4S] cluster. Given the overall structural features and arrangement of redox centers between human CII and the SQRs from E. coli (10) and S. scrofa (11) (SI Appendix, Fig. S4), they likely have a very similar electron transfer pathway. Based on these structural similarities, we propose that the human CII succinate- and ubiquinone-binding sites are likely to be connected by a similar chain of redox centers that include FAD, [2Fe-2S], [4Fe-4S], [3Fe-4S], and heme b (Fig. 3). However, although the structural features and electron transfer pathway of the human CII appear to be similar to that of E. coli (10) and S. scrofa (11) SQRs, there is some amino acid sequence variability (SI Appendix, Fig. S4B). Therefore, we have estimated how the sequence variability affects the structure using MD simulations. Considering the high sequence identity between human CII and S. scrofa SQR among the available structures of complex II, the potential implication of a slight difference of residue sequence between these two species was investigated (SI Appendix, Fig. S6A). The MD simulations indicate that the structure of human CII is more conformationally stable than that of the S. scrofa SQR (SI Appendix, Fig. S6B), which may be potentially useful for electron transport.

Electron Transfer Pathway in the Human CII.

The prosthetic groups (FAD, [2Fe-2S], [4Fe-4S], [3Fe-4S], and a b-type heme groups) mediating the electron transfer were all unambiguously assigned into the cryo-EM map (SI Appendix, Fig. S3). The edge-to-edge distance between these redox-active prosthetic groups is less than 14 Å (Fig. 3), a distance range that can efficiently support the delivery of electrons between these redox centers (22). The EPR signal (g = 2.0) indicates the presence of ubiquinone (UQ) (SI Appendix, Fig. S1E). UQ is also observed to bind at the entrance of the pocket formed by the transmembrane helix I of SDHC, transmembrane helix II of SDHD, and the C-terminal segment of SDHB. It interacts with Pro-SDHB197, Trp-SDHB201, Ile-SDHB246, Ile-SDHC56, Trp-SDHC61, Met-SDHC65, Ile-SDHC69, and Tyr-SDHD114 (Fig. 3). The edge-to-edge distances between UQ and heme b and [3Fe-4S] are about 6.6 Å and 6.6 Å, respectively, which are efficient to promote electron transfer (22).

Prosthetic groups in human CII and the pathway of electron transfer. (Left) Overall structure of human CII is shown together with the prosthetic groups. A zoomed-in view of UQ-binding site representation indicates the interactions between the UQ and surrounding residues. (Right) Prosthetic groups in the path of electron transfer are labeled with edge-to-edge distances. Five prosthetic groups, FAD, [2Fe-2S], [4Fe-4S], [3Fe-4S], and a heme b are required for electron transfer flow from succinate to ubiquinone. Prosthetic groups are shown in stick models or as spheres. The transmembrane helices are labeled.

In E. coli (10) and S. scrofa (11) SQRs, the electrons released from succinate oxidation are transferred to the [3Fe-4S] by sequential delivery through a chain of redox centers consisting of FAD, [2Fe-2S], and [4Fe-4S]. Although the electrons from the [3Fe-4S] cluster can be transferred either to heme b or ubiquinone, ubiquinone is preferable (10, 11). It has been shown that the heme b is neither an absolute structural requirement nor essential for quinone reduction (23, 24). In addition, once the heme b is reduced, it can also in turn reduce ubiquinone (10, 11). Therefore, the heme b is proposed to serve as an electron sink in the electron transfer pathway (25, 26). In this process, quinone likely receives the first electron from the [3Fe-4S] cluster to form the semiquinone, and then, the semiquinone radical is stabilized by rapid electron equilibration between the heme b and ubiquinone (26, 27). Finally, the semiquinone accepts a second electron to complete full reduction, which is transported from either the heme b or the [3Fe-4S] cluster. Given the overall structural features and arrangement of redox centers between human CII and the SQRs from E. coli (10) and S. scrofa (11) (SI Appendix, Fig. S4), they likely have a very similar electron transfer pathway. Based on these structural similarities, we propose that the human CII succinate- and ubiquinone-binding sites are likely to be connected by a similar chain of redox centers that include FAD, [2Fe-2S], [4Fe-4S], [3Fe-4S], and heme b (Fig. 3). However, although the structural features and electron transfer pathway of the human CII appear to be similar to that of E. coli (10) and S. scrofa (11) SQRs, there is some amino acid sequence variability (SI Appendix, Fig. S4B). Therefore, we have estimated how the sequence variability affects the structure using MD simulations. Considering the high sequence identity between human CII and S. scrofa SQR among the available structures of complex II, the potential implication of a slight difference of residue sequence between these two species was investigated (SI Appendix, Fig. S6A). The MD simulations indicate that the structure of human CII is more conformationally stable than that of the S. scrofa SQR (SI Appendix, Fig. S6B), which may be potentially useful for electron transport.

Human CII Mutations Related with Diseases.Mutants of human CII result in mitochondrial dysfunction. This can manifest in two clinical diseases: either as a primary mitochondrial disease or as susceptibility to tumorigenesis (28). The clinically affected individuals harbor genetic variants within the SDHA (29–34), SDHB (30, 35–40), and SDHD (41, 42) subunits. However, there are no reports of primary mitochondrial dysfunction involving SDHC variants. Knowledge of the exact location of the mutations is important to explain the molecular basis for a specific disease (Fig. 4). The dominant substitution of Arg451Cys (29) in the subunit SDHA abolishes a hydrogen bond with the FAD cofactor (Fig. 4A), thus potentially affecting FAD binding. The Thr508Ile and Ser509Leu mutants in SDHA have been reported in one individual (30). These mutations are situated at the SDHA–SDHB interface (Fig. 4B) and result in the loss of hydrogen-bonding interactions with neighboring polar residues and are therefore purported to affect heterodimer formation. Arg554 and Gly555 within a helical domain of SDHA are also situated at the SDHA–SDHB interface. The Arg554Trp (31) mutation abrogates a hydrogen bond with Thr551 within SDHA (Fig. 4C). The Gly555Glu (32–34) replacement creates a negative charge at an external loop of SDHA (Fig. 4C). This change may impact the folding of this segment and assembly with SDHB. The Asp48Val substitution in SDHB is the change that is most frequently reported in clinical studies (30, 35–38). Substitution of a negatively charged aspartate with an uncharged valine introduces a hydrophobic residue into a hydrophilic region between the SDHA–SDHB interface, reducing the stability of this contact (Fig. 4D). The substitution of Ala102Thr (39) results in the formation of new polar bonds with Cys189, and Cys191 within the SDHB subunit (Fig. 4E), possibly altering the local tertiary structure around the [4Fe-4S] cluster. Interestingly, an adjacent variant, Met103Thr of SDHB, is also reported in a case of mitochondrial disease (36). Met103 is located proximal to the [2Fe-2S] cluster, and its substitution with threonine may influence the structure of surrounding hydrophobic region (Fig. 4F). The Arg230His variant of SDHB is associated with both mitochondrial complex II deficiency and susceptibility to tumorigenesis growth (38, 40). This substitution removes a salt bridge with Asp224 in the SDHB subunit (Fig. 4G). The Leu257Val (38) substitution in SDHB alters the interaction with the [4Fe-4S] cluster (Fig. 4H), and the Glu69Lys (41) change in SDHD will affect interactions with heme b (Fig. 4I). These two mutations may induce the change in the orientation of the [4Fe-4S] cluster and heme b, respectively, and affect the capacity for electron transfer. The Asp92Gly (42) substitution removes a hydrogen-bonding interaction with the residue Cys150 (Fig. 4J), which may affect the stability of transmembrane helices of the SDHD subunit on the intermembrane space side. In summary, these substitutions tend to occur in regions of the proteins that result in instability of the complex, leading to a functional deterioration of the complex. However, more studies are needed to reveal the exact interactions between these substitutions and the relevant disease states. In addition, as for the human mutations, they could generally be mapped on the available homologous structures from different organisms. It is remarkable, however, that human CII does not have 100% amino acid sequence identity with any of the available structures of complex II (SI Appendix, Figs. S4B, S6A and S7). This suggests that the mutations of human CII do not always accurately map on the existing structures. Hence, the current structure can directly reflect the potential impacts that result from the mutations of human CII. Overall, this structure improves our understanding of the molecular basis for human CII mutation-related disease and lays the foundation for the development of rational treatments for these diseases.Fig. 4.Location of the ten clinically relevant mutations in human CII. (A-J) The mutants are highlighted in green, and the surrounding residues are shown in gray sticks. The central panel shows an overview of the locations of the ten mutations.

Human CII Mutations Related with Diseases.

Mutants of human CII result in mitochondrial dysfunction. This can manifest in two clinical diseases: either as a primary mitochondrial disease or as susceptibility to tumorigenesis (28). The clinically affected individuals harbor genetic variants within the SDHA (29–34), SDHB (30, 35–40), and SDHD (41, 42) subunits. However, there are no reports of primary mitochondrial dysfunction involving SDHC variants. Knowledge of the exact location of the mutations is important to explain the molecular basis for a specific disease (Fig. 4). The dominant substitution of Arg451Cys (29) in the subunit SDHA abolishes a hydrogen bond with the FAD cofactor (Fig. 4A), thus potentially affecting FAD binding. The Thr508Ile and Ser509Leu mutants in SDHA have been reported in one individual (30). These mutations are situated at the SDHA–SDHB interface (Fig. 4B) and result in the loss of hydrogen-bonding interactions with neighboring polar residues and are therefore purported to affect heterodimer formation. Arg554 and Gly555 within a helical domain of SDHA are also situated at the SDHA–SDHB interface. The Arg554Trp (31) mutation abrogates a hydrogen bond with Thr551 within SDHA (Fig. 4C). The Gly555Glu (32–34) replacement creates a negative charge at an external loop of SDHA (Fig. 4C). This change may impact the folding of this segment and assembly with SDHB. The Asp48Val substitution in SDHB is the change that is most frequently reported in clinical studies (30, 35–38). Substitution of a negatively charged aspartate with an uncharged valine introduces a hydrophobic residue into a hydrophilic region between the SDHA–SDHB interface, reducing the stability of this contact (Fig. 4D). The substitution of Ala102Thr (39) results in the formation of new polar bonds with Cys189, and Cys191 within the SDHB subunit (Fig. 4E), possibly altering the local tertiary structure around the [4Fe-4S] cluster. Interestingly, an adjacent variant, Met103Thr of SDHB, is also reported in a case of mitochondrial disease (36). Met103 is located proximal to the [2Fe-2S] cluster, and its substitution with threonine may influence the structure of surrounding hydrophobic region (Fig. 4F). The Arg230His variant of SDHB is associated with both mitochondrial complex II deficiency and susceptibility to tumorigenesis growth (38, 40). This substitution removes a salt bridge with Asp224 in the SDHB subunit (Fig. 4G). The Leu257Val (38) substitution in SDHB alters the interaction with the [4Fe-4S] cluster (Fig. 4H), and the Glu69Lys (41) change in SDHD will affect interactions with heme b (Fig. 4I). These two mutations may induce the change in the orientation of the [4Fe-4S] cluster and heme b, respectively, and affect the capacity for electron transfer. The Asp92Gly (42) substitution removes a hydrogen-bonding interaction with the residue Cys150 (Fig. 4J), which may affect the stability of transmembrane helices of the SDHD subunit on the intermembrane space side. In summary, these substitutions tend to occur in regions of the proteins that result in instability of the complex, leading to a functional deterioration of the complex. However, more studies are needed to reveal the exact interactions between these substitutions and the relevant disease states. In addition, as for the human mutations, they could generally be mapped on the available homologous structures from different organisms. It is remarkable, however, that human CII does not have 100% amino acid sequence identity with any of the available structures of complex II (SI Appendix, Figs. S4B, S6A and S7). This suggests that the mutations of human CII do not always accurately map on the existing structures. Hence, the current structure can directly reflect the potential impacts that result from the mutations of human CII. Overall, this structure improves our understanding of the molecular basis for human CII mutation-related disease and lays the foundation for the development of rational treatments for these diseases.

Location of the ten clinically relevant mutations in human CII. (A-J) The mutants are highlighted in green, and the surrounding residues are shown in gray sticks. The central panel shows an overview of the locations of the ten mutations.

Materials and MethodsCloning and Expression of Protein.The gene for SDHD was amplified from human cDNA by PCR and cloned into the pQCXIP vector with a 3× Flag tag inserted at the C terminus of SDHD. The primer sequences are listed in SI Appendix, Table S5. 293F cells were transfected with the reconstructed plasmid using polyethylenimine (PEI, Polysciences) and then were subjected to 5 µg/mL puromycin (Sangon Biotech) for 2 wk. The cells were cultivated in SMM 293-TII medium (Sino Biological), harvested at a density of 5 × 106 cells/mL, and collected and rinsed in PBS buffer. Cell pellets were stored at −80 °C for future use.Isolation of Human Mitochondria.The cell pellets were resuspended in ice-cold buffer consisting of 20 mM 3-(N-morpholino)propanesulfonic acid (MOPS), pH 7.4, 250 mM sucrose, and 1 mM EGTA and incubated for 10 min. The suspension was homogenized (40 mL buffer/300 mL pellets) with 20 strokes of a Dounce tissue grinder (Kimble). Cell debris was removed by centrifugation at 1,000 ×g for 10 min at 4 °C. The supernatant was collected and centrifuged (10,000 × g, 4 °C, 10 min) again to isolate the mitochondria. Finally, the pellets were collected as mitochondrial fractions.Purification and Identification of the Respiratory Complex II.To purify complex II, mitochondria were resuspended in buffer containing 20 mM MOPS (pH 7.4), 100 mM NaCl, and 1% (w/v) lauryl maltose neopentyl glycol (LMNG, Anatrace) and incubated for about 30 min with rotation at 4 °C. The lysate was then centrifuged at 40,000 × g for 40 min at 4 °C and incubated with Anti ‐DYKDDDDK G1 Affinity Resin (GenScript) at 4 °C for 1 h. The beads were washed with 10 column volumes (CVs) of washing buffer containing 20 mM MOPS (pH 7.4), 100 mM NaCl, and 0.004% (w/v) LMNG. Then, protein was eluted with 5 CV elution buffer containing 20 mM MOPS (pH 7.4), 100 mM NaCl, and 0.02% (w/v) glyco-diosgenin (GDN, Anatrace) supplemented with 0.2 mg/mL 1× Flag peptides (GenScript). Finally, complex II was concentrated and loaded onto a Superose 6 Increase 10/300 size-exclusion column (GE Healthcare) equilibrated with elution buffer. Peak fractions containing protein were collected and concentrated for protein gel electrophoresis analysis, MS analysis, and electron microscopy studies.To identify the cofactors in the protein sample, low-temperature electron paramagnetic resonance (EPR) spectrum was acquired using a Bruker X-band (9.4 GHz) EMX plus 10/12 spectrometer equipped with an Oxford Instruments ESR 910 continuous helium flow cryostat. A cylindrical resonator (ER4119hs TE011) was used for EPR data collection. The sample was placed into quartz EPR tubes (Wilmad, 707-SQ-250 M, 3 mm i.d., 4 mm o.d.) for measurements. The temperature was set at 10 K. Other experimental conditions were as follows: microwave power, 0.2 mW; modulation amplitude, 5 Gauss; modulation frequency, 100 kHz; and resonance frequency, ~9.404 GHz. For each sample, multiple scans were accumulated to obtain a good S/N ratio.Kinetics of Enzyme Activity Assay.The succinate-quinone oxidoreductase enzymatic activity of human CII was determined by performing a succinate-2,6-dichlorophenolindophenol (DCIP, Sigma) assay in the presence of ubiquinone-1 (UQ1, Sigma) as previously described (43). UQ1 was used as the intermediate electron acceptor to examine succinate dehydrogenase activity. The final reaction mixture contained elution buffer, human complex II (2.29 μg), 0.00625 to 0.8 mM sodium succinate (Sigma), 0.5 mM UQ1, and 200 μM DCIP. Reactions were initiated by addition of succinate, and the absorbance change at 600 nm was monitored over time. All assays were performed at 25 °C by using the Cytation5 imaging reader (BioTek). All the enzymatic experiments were repeated three times, and data were processed with GraphPad Prism 6.Electron Microscopy Sample Preparation and Imaging.First, 3 μL of 2.9 mg/mL human CII was applied to H2/O2 glow-discharged 300-mesh Quantifoil R 1.2/1.3 grids (Quantifoil, Micro Tools GmbH) and subsequently blotted using an FEI Vitrobot and then frozen in liquid ethane. Then, the grids were imaged on a Thermo Fisher Krios G4 microscope (Thermo Fisher Scientific) equipped with a cold field emission gun, a Selectris X energy filter, and a Falcon 4 detector. The energy filter was operated with a slit width of 10 e−V to remove inelastically scattered electrons. Image stacks were collected using EPU software at a pixel size of 0.73 Å/pixel with a total dose of 50 e−/Å2.Imaging Processing.A total of 6,700 image stacks were subjected to beam-induced motion correction and CTF estimation using CryoSPARC (44). A total of 1,788,861 particles were picked automatically and were extracted with a box size of 384 pixels. After several rounds of 2D and 3D classifications, 322,768 particles were used to perform ab initio reconstruction and heterogeneous refinement. Afterward, the candidate model and 114,967 particles were selected and refined using NU refinement to generate the final cryo-EM map at a 2.86 Å resolution. Local resolution ranges were also analyzed within CryoSPARC according to the gold standard Fourier shell correlation (FSC) cutoff of 0.143 (45). All dataset processing is shown in SI Appendix, Fig. S2.Model Building and Refinement.The atomic model of complex II was built based on the structure of SDHA/B/C/D predicted from the AlphaFold Protein Structure Database (46). To start model building, the predicted domains of four subunits were docked into the EM density map by using Chimera (47), followed by manual adjustment of main chains and side chains in Coot (48) and real space refinement in PHENIX (49).Figure Images.The majority of the images presented in the figures were created using UCSF Chimera X (50).Simulation System Construction.The structures of wild-type and mutant human CII were processed on the H++ website (http://newbiophysics.cs.vt.edu/H++/) to analyze the protonation state of ionizable groups in the protein at pH 7.0. Next, the protein/membrane system solvated in water was built using PACKMOL-Memgen software, an automated workflow for building the lipid bilayer of the membrane protein. Here, the POPC (1-palmitoyl-2-oleoyl-sn-glycero-3-phosphocholine) lipid bilayer with dimensions 90 × 90 × 158 Å was applied to embed the transmembrane region of human CII, while the cytoplasmic domain of human CII was exposed to solvents. Na+ and Cl− ions at a physiological concentration of 0.15 M were added to neutralize the system. The force fields ffSB19 and LIPID17 were applied to parameterize the standard amino acids and lipids, respectively, and a TIP3P model was chosen for water parameterization. The parameter files of HEME molecule were downloaded from the AMBER parameter database (http://amber.manchester.ac.uk/) maintained by Richard Bryce. All parameters of the iron–sulfate clusters, i.e., SF4, F3S, and FES, referred to Marcel Swart’s study (51). The parameters of FAD and UQ molecules were both generated using the Antechamber tool, while their electrostatic charges were computed based on the RESP (restrained electrostatic potential) methodology. The PE molecule was constructed as a nonstandard lipid molecule, and its parameters were derived from the lipid force field LIPID17. With the above parameters, the leap tool in the AMBER package was used to generate the molecular topology and coordinate files for simulation.Molecular Dynamics Simulation.Molecular dynamic simulation was performed using AMBER 2020 (52). Before dynamic simulation, 5,000 steps of energy minimization with a force constant of 10 kcal/mol Å2 and 2.5 kcal/mol Å2 on protein and lipids, respectively, were performed on waters to remove potential steric clashes between the solute and solvents. Then, two stages of equilibrium simulations were carried out to relax the protein and lipid molecules by gradually decreasing the restraint force. The first one includes three short simulations with a constant volume and temperature (NVT) ensemble. Lipids were restrained with a force field of 2.5 kcal/mol Å2, while the restraint force on protein was decreased from 10.0 to 1.0 kcal/mol Å2. A time step of 1 fs was set to maintain simulation stability. The second run used three short simulations in total 1.5 ns to completely relax the whole system with an extremely low force. At this stage, constant pressure and temperature (NPT) was applied to the simulation. The time step was increased to 2 fs, while the SHAKE algorithm was used to fix high-frequency hydrogen motions (53). Subsequently, a 10-ns NPT simulation without any restraints was performed to further equilibrate the system. Langevin dynamics was used to control the temperature at 303.15 K in NVT and NPT simulations (54). In the NPT simulations, additional semi-isotropic pressure coupling and constant surface tension were applied to the simulation of lipid bilayers (55). After the equilibrium simulation, a 50 ns of production simulation was carried out to generate the trajectory for the final structure analysis. Coordinates were printed every 100 ps so that in total, 500 frames were contained in the final trajectory. The CPPTRAJ module in AMBER was applied to analyze the trajectory (56). The rmsd of protein backbone atoms was used to measure the overall conformational stability of the structure. It was calculated as the positional difference of all backbone atoms between the frames produced during the simulation and compared with the starting conformation. We used the definition that the smaller the deviation, the more stable was the protein structure.

Materials and Methods

Cloning and Expression of Protein.The gene for SDHD was amplified from human cDNA by PCR and cloned into the pQCXIP vector with a 3× Flag tag inserted at the C terminus of SDHD. The primer sequences are listed in SI Appendix, Table S5. 293F cells were transfected with the reconstructed plasmid using polyethylenimine (PEI, Polysciences) and then were subjected to 5 µg/mL puromycin (Sangon Biotech) for 2 wk. The cells were cultivated in SMM 293-TII medium (Sino Biological), harvested at a density of 5 × 106 cells/mL, and collected and rinsed in PBS buffer. Cell pellets were stored at −80 °C for future use.

Cloning and Expression of Protein.

The gene for SDHD was amplified from human cDNA by PCR and cloned into the pQCXIP vector with a 3× Flag tag inserted at the C terminus of SDHD. The primer sequences are listed in SI Appendix, Table S5. 293F cells were transfected with the reconstructed plasmid using polyethylenimine (PEI, Polysciences) and then were subjected to 5 µg/mL puromycin (Sangon Biotech) for 2 wk. The cells were cultivated in SMM 293-TII medium (Sino Biological), harvested at a density of 5 × 106 cells/mL, and collected and rinsed in PBS buffer. Cell pellets were stored at −80 °C for future use.

Isolation of Human Mitochondria.The cell pellets were resuspended in ice-cold buffer consisting of 20 mM 3-(N-morpholino)propanesulfonic acid (MOPS), pH 7.4, 250 mM sucrose, and 1 mM EGTA and incubated for 10 min. The suspension was homogenized (40 mL buffer/300 mL pellets) with 20 strokes of a Dounce tissue grinder (Kimble). Cell debris was removed by centrifugation at 1,000 ×g for 10 min at 4 °C. The supernatant was collected and centrifuged (10,000 × g, 4 °C, 10 min) again to isolate the mitochondria. Finally, the pellets were collected as mitochondrial fractions.

Isolation of Human Mitochondria.

The cell pellets were resuspended in ice-cold buffer consisting of 20 mM 3-(N-morpholino)propanesulfonic acid (MOPS), pH 7.4, 250 mM sucrose, and 1 mM EGTA and incubated for 10 min. The suspension was homogenized (40 mL buffer/300 mL pellets) with 20 strokes of a Dounce tissue grinder (Kimble). Cell debris was removed by centrifugation at 1,000 ×g for 10 min at 4 °C. The supernatant was collected and centrifuged (10,000 × g, 4 °C, 10 min) again to isolate the mitochondria. Finally, the pellets were collected as mitochondrial fractions.

Purification and Identification of the Respiratory Complex II.To purify complex II, mitochondria were resuspended in buffer containing 20 mM MOPS (pH 7.4), 100 mM NaCl, and 1% (w/v) lauryl maltose neopentyl glycol (LMNG, Anatrace) and incubated for about 30 min with rotation at 4 °C. The lysate was then centrifuged at 40,000 × g for 40 min at 4 °C and incubated with Anti ‐DYKDDDDK G1 Affinity Resin (GenScript) at 4 °C for 1 h. The beads were washed with 10 column volumes (CVs) of washing buffer containing 20 mM MOPS (pH 7.4), 100 mM NaCl, and 0.004% (w/v) LMNG. Then, protein was eluted with 5 CV elution buffer containing 20 mM MOPS (pH 7.4), 100 mM NaCl, and 0.02% (w/v) glyco-diosgenin (GDN, Anatrace) supplemented with 0.2 mg/mL 1× Flag peptides (GenScript). Finally, complex II was concentrated and loaded onto a Superose 6 Increase 10/300 size-exclusion column (GE Healthcare) equilibrated with elution buffer. Peak fractions containing protein were collected and concentrated for protein gel electrophoresis analysis, MS analysis, and electron microscopy studies.To identify the cofactors in the protein sample, low-temperature electron paramagnetic resonance (EPR) spectrum was acquired using a Bruker X-band (9.4 GHz) EMX plus 10/12 spectrometer equipped with an Oxford Instruments ESR 910 continuous helium flow cryostat. A cylindrical resonator (ER4119hs TE011) was used for EPR data collection. The sample was placed into quartz EPR tubes (Wilmad, 707-SQ-250 M, 3 mm i.d., 4 mm o.d.) for measurements. The temperature was set at 10 K. Other experimental conditions were as follows: microwave power, 0.2 mW; modulation amplitude, 5 Gauss; modulation frequency, 100 kHz; and resonance frequency, ~9.404 GHz. For each sample, multiple scans were accumulated to obtain a good S/N ratio.

Purification and Identification of the Respiratory Complex II.

To purify complex II, mitochondria were resuspended in buffer containing 20 mM MOPS (pH 7.4), 100 mM NaCl, and 1% (w/v) lauryl maltose neopentyl glycol (LMNG, Anatrace) and incubated for about 30 min with rotation at 4 °C. The lysate was then centrifuged at 40,000 × g for 40 min at 4 °C and incubated with Anti ‐DYKDDDDK G1 Affinity Resin (GenScript) at 4 °C for 1 h. The beads were washed with 10 column volumes (CVs) of washing buffer containing 20 mM MOPS (pH 7.4), 100 mM NaCl, and 0.004% (w/v) LMNG. Then, protein was eluted with 5 CV elution buffer containing 20 mM MOPS (pH 7.4), 100 mM NaCl, and 0.02% (w/v) glyco-diosgenin (GDN, Anatrace) supplemented with 0.2 mg/mL 1× Flag peptides (GenScript). Finally, complex II was concentrated and loaded onto a Superose 6 Increase 10/300 size-exclusion column (GE Healthcare) equilibrated with elution buffer. Peak fractions containing protein were collected and concentrated for protein gel electrophoresis analysis, MS analysis, and electron microscopy studies.

To identify the cofactors in the protein sample, low-temperature electron paramagnetic resonance (EPR) spectrum was acquired using a Bruker X-band (9.4 GHz) EMX plus 10/12 spectrometer equipped with an Oxford Instruments ESR 910 continuous helium flow cryostat. A cylindrical resonator (ER4119hs TE011) was used for EPR data collection. The sample was placed into quartz EPR tubes (Wilmad, 707-SQ-250 M, 3 mm i.d., 4 mm o.d.) for measurements. The temperature was set at 10 K. Other experimental conditions were as follows: microwave power, 0.2 mW; modulation amplitude, 5 Gauss; modulation frequency, 100 kHz; and resonance frequency, ~9.404 GHz. For each sample, multiple scans were accumulated to obtain a good S/N ratio.

Kinetics of Enzyme Activity Assay.The succinate-quinone oxidoreductase enzymatic activity of human CII was determined by performing a succinate-2,6-dichlorophenolindophenol (DCIP, Sigma) assay in the presence of ubiquinone-1 (UQ1, Sigma) as previously described (43). UQ1 was used as the intermediate electron acceptor to examine succinate dehydrogenase activity. The final reaction mixture contained elution buffer, human complex II (2.29 μg), 0.00625 to 0.8 mM sodium succinate (Sigma), 0.5 mM UQ1, and 200 μM DCIP. Reactions were initiated by addition of succinate, and the absorbance change at 600 nm was monitored over time. All assays were performed at 25 °C by using the Cytation5 imaging reader (BioTek). All the enzymatic experiments were repeated three times, and data were processed with GraphPad Prism 6.

Kinetics of Enzyme Activity Assay.

The succinate-quinone oxidoreductase enzymatic activity of human CII was determined by performing a succinate-2,6-dichlorophenolindophenol (DCIP, Sigma) assay in the presence of ubiquinone-1 (UQ1, Sigma) as previously described (43). UQ1 was used as the intermediate electron acceptor to examine succinate dehydrogenase activity. The final reaction mixture contained elution buffer, human complex II (2.29 μg), 0.00625 to 0.8 mM sodium succinate (Sigma), 0.5 mM UQ1, and 200 μM DCIP. Reactions were initiated by addition of succinate, and the absorbance change at 600 nm was monitored over time. All assays were performed at 25 °C by using the Cytation5 imaging reader (BioTek). All the enzymatic experiments were repeated three times, and data were processed with GraphPad Prism 6.

Electron Microscopy Sample Preparation and Imaging.First, 3 μL of 2.9 mg/mL human CII was applied to H2/O2 glow-discharged 300-mesh Quantifoil R 1.2/1.3 grids (Quantifoil, Micro Tools GmbH) and subsequently blotted using an FEI Vitrobot and then frozen in liquid ethane. Then, the grids were imaged on a Thermo Fisher Krios G4 microscope (Thermo Fisher Scientific) equipped with a cold field emission gun, a Selectris X energy filter, and a Falcon 4 detector. The energy filter was operated with a slit width of 10 e−V to remove inelastically scattered electrons. Image stacks were collected using EPU software at a pixel size of 0.73 Å/pixel with a total dose of 50 e−/Å2.

Electron Microscopy Sample Preparation and Imaging.

First, 3 μL of 2.9 mg/mL human CII was applied to H2/O2 glow-discharged 300-mesh Quantifoil R 1.2/1.3 grids (Quantifoil, Micro Tools GmbH) and subsequently blotted using an FEI Vitrobot and then frozen in liquid ethane. Then, the grids were imaged on a Thermo Fisher Krios G4 microscope (Thermo Fisher Scientific) equipped with a cold field emission gun, a Selectris X energy filter, and a Falcon 4 detector. The energy filter was operated with a slit width of 10 e−V to remove inelastically scattered electrons. Image stacks were collected using EPU software at a pixel size of 0.73 Å/pixel with a total dose of 50 e−/Å2.

Imaging Processing.A total of 6,700 image stacks were subjected to beam-induced motion correction and CTF estimation using CryoSPARC (44). A total of 1,788,861 particles were picked automatically and were extracted with a box size of 384 pixels. After several rounds of 2D and 3D classifications, 322,768 particles were used to perform ab initio reconstruction and heterogeneous refinement. Afterward, the candidate model and 114,967 particles were selected and refined using NU refinement to generate the final cryo-EM map at a 2.86 Å resolution. Local resolution ranges were also analyzed within CryoSPARC according to the gold standard Fourier shell correlation (FSC) cutoff of 0.143 (45). All dataset processing is shown in SI Appendix, Fig. S2.

Imaging Processing.

A total of 6,700 image stacks were subjected to beam-induced motion correction and CTF estimation using CryoSPARC (44). A total of 1,788,861 particles were picked automatically and were extracted with a box size of 384 pixels. After several rounds of 2D and 3D classifications, 322,768 particles were used to perform ab initio reconstruction and heterogeneous refinement. Afterward, the candidate model and 114,967 particles were selected and refined using NU refinement to generate the final cryo-EM map at a 2.86 Å resolution. Local resolution ranges were also analyzed within CryoSPARC according to the gold standard Fourier shell correlation (FSC) cutoff of 0.143 (45). All dataset processing is shown in SI Appendix, Fig. S2.

Model Building and Refinement.The atomic model of complex II was built based on the structure of SDHA/B/C/D predicted from the AlphaFold Protein Structure Database (46). To start model building, the predicted domains of four subunits were docked into the EM density map by using Chimera (47), followed by manual adjustment of main chains and side chains in Coot (48) and real space refinement in PHENIX (49).

Model Building and Refinement.

The atomic model of complex II was built based on the structure of SDHA/B/C/D predicted from the AlphaFold Protein Structure Database (46). To start model building, the predicted domains of four subunits were docked into the EM density map by using Chimera (47), followed by manual adjustment of main chains and side chains in Coot (48) and real space refinement in PHENIX (49).

Figure Images.The majority of the images presented in the figures were created using UCSF Chimera X (50).

Figure Images.

The majority of the images presented in the figures were created using UCSF Chimera X (50).

Simulation System Construction.The structures of wild-type and mutant human CII were processed on the H++ website (http://newbiophysics.cs.vt.edu/H++/) to analyze the protonation state of ionizable groups in the protein at pH 7.0. Next, the protein/membrane system solvated in water was built using PACKMOL-Memgen software, an automated workflow for building the lipid bilayer of the membrane protein. Here, the POPC (1-palmitoyl-2-oleoyl-sn-glycero-3-phosphocholine) lipid bilayer with dimensions 90 × 90 × 158 Å was applied to embed the transmembrane region of human CII, while the cytoplasmic domain of human CII was exposed to solvents. Na+ and Cl− ions at a physiological concentration of 0.15 M were added to neutralize the system. The force fields ffSB19 and LIPID17 were applied to parameterize the standard amino acids and lipids, respectively, and a TIP3P model was chosen for water parameterization. The parameter files of HEME molecule were downloaded from the AMBER parameter database (http://amber.manchester.ac.uk/) maintained by Richard Bryce. All parameters of the iron–sulfate clusters, i.e., SF4, F3S, and FES, referred to Marcel Swart’s study (51). The parameters of FAD and UQ molecules were both generated using the Antechamber tool, while their electrostatic charges were computed based on the RESP (restrained electrostatic potential) methodology. The PE molecule was constructed as a nonstandard lipid molecule, and its parameters were derived from the lipid force field LIPID17. With the above parameters, the leap tool in the AMBER package was used to generate the molecular topology and coordinate files for simulation.

Simulation System Construction.

The structures of wild-type and mutant human CII were processed on the H++ website (http://newbiophysics.cs.vt.edu/H++/) to analyze the protonation state of ionizable groups in the protein at pH 7.0. Next, the protein/membrane system solvated in water was built using PACKMOL-Memgen software, an automated workflow for building the lipid bilayer of the membrane protein. Here, the POPC (1-palmitoyl-2-oleoyl-sn-glycero-3-phosphocholine) lipid bilayer with dimensions 90 × 90 × 158 Å was applied to embed the transmembrane region of human CII, while the cytoplasmic domain of human CII was exposed to solvents. Na+ and Cl− ions at a physiological concentration of 0.15 M were added to neutralize the system. The force fields ffSB19 and LIPID17 were applied to parameterize the standard amino acids and lipids, respectively, and a TIP3P model was chosen for water parameterization. The parameter files of HEME molecule were downloaded from the AMBER parameter database (http://amber.manchester.ac.uk/) maintained by Richard Bryce. All parameters of the iron–sulfate clusters, i.e., SF4, F3S, and FES, referred to Marcel Swart’s study (51). The parameters of FAD and UQ molecules were both generated using the Antechamber tool, while their electrostatic charges were computed based on the RESP (restrained electrostatic potential) methodology. The PE molecule was constructed as a nonstandard lipid molecule, and its parameters were derived from the lipid force field LIPID17. With the above parameters, the leap tool in the AMBER package was used to generate the molecular topology and coordinate files for simulation.

Molecular Dynamics Simulation.Molecular dynamic simulation was performed using AMBER 2020 (52). Before dynamic simulation, 5,000 steps of energy minimization with a force constant of 10 kcal/mol Å2 and 2.5 kcal/mol Å2 on protein and lipids, respectively, were performed on waters to remove potential steric clashes between the solute and solvents. Then, two stages of equilibrium simulations were carried out to relax the protein and lipid molecules by gradually decreasing the restraint force. The first one includes three short simulations with a constant volume and temperature (NVT) ensemble. Lipids were restrained with a force field of 2.5 kcal/mol Å2, while the restraint force on protein was decreased from 10.0 to 1.0 kcal/mol Å2. A time step of 1 fs was set to maintain simulation stability. The second run used three short simulations in total 1.5 ns to completely relax the whole system with an extremely low force. At this stage, constant pressure and temperature (NPT) was applied to the simulation. The time step was increased to 2 fs, while the SHAKE algorithm was used to fix high-frequency hydrogen motions (53). Subsequently, a 10-ns NPT simulation without any restraints was performed to further equilibrate the system. Langevin dynamics was used to control the temperature at 303.15 K in NVT and NPT simulations (54). In the NPT simulations, additional semi-isotropic pressure coupling and constant surface tension were applied to the simulation of lipid bilayers (55). After the equilibrium simulation, a 50 ns of production simulation was carried out to generate the trajectory for the final structure analysis. Coordinates were printed every 100 ps so that in total, 500 frames were contained in the final trajectory. The CPPTRAJ module in AMBER was applied to analyze the trajectory (56). The rmsd of protein backbone atoms was used to measure the overall conformational stability of the structure. It was calculated as the positional difference of all backbone atoms between the frames produced during the simulation and compared with the starting conformation. We used the definition that the smaller the deviation, the more stable was the protein structure.

Molecular Dynamics Simulation.

Molecular dynamic simulation was performed using AMBER 2020 (52). Before dynamic simulation, 5,000 steps of energy minimization with a force constant of 10 kcal/mol Å2 and 2.5 kcal/mol Å2 on protein and lipids, respectively, were performed on waters to remove potential steric clashes between the solute and solvents. Then, two stages of equilibrium simulations were carried out to relax the protein and lipid molecules by gradually decreasing the restraint force. The first one includes three short simulations with a constant volume and temperature (NVT) ensemble. Lipids were restrained with a force field of 2.5 kcal/mol Å2, while the restraint force on protein was decreased from 10.0 to 1.0 kcal/mol Å2. A time step of 1 fs was set to maintain simulation stability. The second run used three short simulations in total 1.5 ns to completely relax the whole system with an extremely low force. At this stage, constant pressure and temperature (NPT) was applied to the simulation. The time step was increased to 2 fs, while the SHAKE algorithm was used to fix high-frequency hydrogen motions (53). Subsequently, a 10-ns NPT simulation without any restraints was performed to further equilibrate the system. Langevin dynamics was used to control the temperature at 303.15 K in NVT and NPT simulations (54). In the NPT simulations, additional semi-isotropic pressure coupling and constant surface tension were applied to the simulation of lipid bilayers (55). After the equilibrium simulation, a 50 ns of production simulation was carried out to generate the trajectory for the final structure analysis. Coordinates were printed every 100 ps so that in total, 500 frames were contained in the final trajectory. The CPPTRAJ module in AMBER was applied to analyze the trajectory (56). The rmsd of protein backbone atoms was used to measure the overall conformational stability of the structure. It was calculated as the positional difference of all backbone atoms between the frames produced during the simulation and compared with the starting conformation. We used the definition that the smaller the deviation, the more stable was the protein structure.

Supplementary MaterialAppendix 01 (PDF)Click here for additional data file.

Supplementary Material

Appendix 01 (PDF)

Click here for additional data file.
