---
pmid: '20551380'
title: Proteomics characterization of extracellular space components in the human
  aorta.
authors:
- Didangelos A
- Yin X
- Mandal K
- Baumert M
- Jahangiri M
- Mayr M
journal: Mol Cell Proteomics
year: '2010'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC2938114
doi: 10.1074/mcp.M110.001693
---

# Proteomics characterization of extracellular space components in the human aorta.
**Authors:** Didangelos A, Yin X, Mandal K, Baumert M, Jahangiri M, Mayr M
**Journal:** Mol Cell Proteomics (2010)
**DOI:** [10.1074/mcp.M110.001693](https://doi.org/10.1074/mcp.M110.001693)
**PMC:** [PMC2938114](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2938114/)

## Abstract

1. Mol Cell Proteomics. 2010 Sep;9(9):2048-62. doi: 10.1074/mcp.M110.001693. Epub
 2010 Jun 15.

Proteomics characterization of extracellular space components in the human 
aorta.

Didangelos A(1), Yin X, Mandal K, Baumert M, Jahangiri M, Mayr M.

Author information:
(1)King's British Heart Foundation Centre, King's College, London, United 
Kingdom.

The vascular extracellular matrix (ECM) is essential for the structural 
integrity of the vessel wall and also serves as a substrate for the binding and 
retention of secreted products of vascular cells as well as molecules coming 
from the circulation. Although proteomics has been previously applied to 
vascular tissues, few studies have specifically targeted the vascular ECM and 
its associated proteins. Thus, its detailed composition remains to be 
characterized. In this study, we describe a methodology for the extraction of 
extracellular proteins from human aortas and their identification by proteomics. 
The approach is based on (a) effective decellularization to enrich for scarce 
extracellular proteins, (b) successful solubilization and deglycosylation of ECM 
proteins, and (c) relative estimation of protein abundance using spectral 
counting. Our three-step extraction approach resulted in the identification of 
103 extracellular proteins of which one-third have never been reported in the 
proteomics literature of vascular tissues. In particular, three glycoproteins 
(podocan, sclerostin, and agrin) were identified for the first time in human 
aortas at the protein level. We also identified extracellular adipocyte 
enhancer-binding protein 1, the cartilage glycoprotein asporin, and a previously 
hypothetical protein, retinal pigment epithelium (RPE) spondin. Moreover, our 
methodology allowed us to screen for proteolysis in the aortic samples based on 
the identification of proteolytic enzymes and their corresponding degradation 
products. For instance, we were able to detect matrix metalloproteinase-9 by 
mass spectrometry and relate its presence to degradation of fibronectin in a 
clinical specimen. We expect this proteomics methodology to further our 
understanding of the composition of the vascular extracellular environment, shed 
light on ECM remodeling and degradation, and provide insights into important 
pathological processes, such as plaque rupture, aneurysm formation, and 
restenosis.

DOI: 10.1074/mcp.M110.001693
PMCID: PMC2938114
PMID: 20551380 [Indexed for MEDLINE]

## Full Text

Vascular cells, in particular vascular smooth muscle cells, produce and maintain a complex meshwork of ECM.1 The ECM is not only the scaffold for the anchorage and mobility of residing cells but also absorbs and transduces the shear and strain forces of the blood flow. It is primarily composed of elastin, collagen, proteoglycans, and glycoproteins. The elastin fibers and type I and III fibrillar collagens form a rigid network of highly cross-linked interstitial matrix. They offer elasticity (elastin) and tensile strength (collagens). Proteoglycans, because of their negative charge, attract water and confer resistance to compression. Finally, glycoproteins participate in matrix organization and are essential for cell attachment.

The vascular ECM also serves as a substrate for the binding and retention of secreted, soluble proteins of vascular cells as well as molecules coming from the circulation, including lipoproteins, growth factors, cytokines, proteases, and protease inhibitors. These components are invariably associated with ECM proteins, especially proteoglycans. Together they comprise the vascular extracellular environment and are pivotal for disease processes, such as atherosclerosis and aneurysm formation (1).

Although proteomics has been previously applied to vascular tissues, only one study has specifically targeted the extracellular vascular environment (2). This study was focused on the isolation of intimal proteoglycans from human carotid arteries. Moreover, most proteomics studies use whole tissue lysates, which are rich in cellular proteins that inevitably mask the identification of the less abundant proteins of the vascular extracellular environment (3–5). Thus, the composition of the vascular ECM and its associated proteins remains poorly defined. In the present study, we used morphologically normal human aortic samples to develop a method for the extraction of proteins present in the extracellular environment, including ECM proteins and proteins attached to the ECM. We had three specific aims: first, to reduce the contamination with cellular proteins, thereby increasing the chance of identifying scarce extracellular proteins; second, to efficiently solubilize and deglycosylate ECM proteins to improve their analysis by liquid chromatography tandem mass spectrometry (LC-MS/MS); and third, to interface the nanoflow LC system to a recently developed injection device, which splits the flow from the analytical column, to allow the reanalysis of the same sample during a single LC-MS/MS run (RePlay, Advion).

Our methodology provides a detailed overview of the aortic ECM and its associated proteins, many reported for the first time in proteomics analysis of the vasculature. Most importantly, this method could be adapted for use with other tissues to further our understanding of the composition of extracellular environment and ECM turnover under various disease conditions.

EXPERIMENTAL PROCEDURESTissue Processing, Decellularization, and Extraction of Extracellular Space ProteinsThe data presented were derived from three human aortic samples. The samples were obtained upon aortotomy performed during routine aortic valve replacement. They were collected from positions of the ascending aorta that were free of macroscopically evident vascular pathology, including atherosclerosis or aneurysm formation. The tissue was immediately snap frozen and subsequently kept in liquid nitrogen for later use. A fourth sample was also obtained upon aortotomy and processed as described below, but the proteomics analysis revealed proteolysis of ECM proteins. Therefore, we used it to examine the potential of our methodology to study proteolytic activity in the tissue. All procedures were approved by our local Regional Ethics Committee Board. Informed consent was obtained from all patients recruited into the study.Before extraction, the tissue pieces were partially thawed and weighed. Approximately 150 mg of tissue per aortic sample were immediately placed in ice-cold phosphate-buffered saline to remove plasma contaminants. Commercially available protease and phosphatase inhibitor mixtures (Sigma-Aldrich) were included according to the manufacturer's instructions to inhibit broad range proteinase activity, and 25 mm EDTA was included to ensure inhibition of metalloproteinases. While the tissue samples were immersed in the cold saline mixture, they were quickly diced with a scalpel into 8–10 smaller pieces. This would enhance (a) the initial removal of plasma contaminants and (b) the effective extraction of extracellular proteins, which is described below. The saline mixture was changed five times (∼30 ml total per sample).Then, the diced samples were incubated with 0.5 m NaCl, 10 mm Tris, pH 7.5 supplemented with proteinase/phosphatase inhibitor mixtures (Sigma-Aldrich) and 25 mm EDTA. The volume of the buffer was adjusted to 10:1 of the tissue weight (i.e. 100 mg in 1 ml), and the samples were mildly vortexed for 4 h at room temperature. The NaCl extracts were then desalted with centrifugation using desalting columns (Zeba Spin, Pierce). Following desalting, the extracts were mixed with 100% acetone (5:1 volume ratio) at −20 °C for 16 h. Proteins were precipitated with centrifugation (16,000 × g for 45 min), and the pellets were dried and redissolved in deglycosylation buffer (see below).Subsequently, the aortic samples were incubated with 0.08% SDS (10:1 buffer volume to tissue weight) supplemented with proteinase/phosphatase inhibitor mixtures and 25 mm EDTA. The samples were mildly vortexed for 4 h at room temperature to achieve removal of cellular components. From our experience with tissue extractions, we took particular care to ensure a low vortexing speed to decrease the possibility of mechanical disruption of the ECM. The SDS solution was then removed and stored frozen for later use.Finally, the samples were incubated in a 4 m guanidine HCl, 50 mm sodium acetate, pH 5.8 buffer (5:1 buffer volume to tissue weight) supplemented with proteinase/phosphatase inhibitor mixtures and 25 mm EDTA. Adjusting the pH to 5.8 is important; more than 30 years ago it was shown that at this pH guanidine is more effective in solubilizing vascular proteoglycans, whereas higher pH is required for the solubilization of the proteoglycans from other ECM-rich tissues, such as cartilage (6). The samples were then incubated for 48 h at room temperature and vortexed vigorously to enhance mechanical disruption of the ECM components. Subsequently, the guanidine extracts were mixed with 100% ethanol (5:1 volume ratio) at −20 °C for 16 h to ensure removal of guanidine (the presence of guanidine would hinder further biochemical processing). Proteins were precipitated with centrifugation (16,000 × g for 45 min), and the pellets were washed with 90% ethanol, dried, and redissolved in deglycosylation buffer (see below).Deglycosylation (removal of glycosaminoglycan side chains) of the different extracts was achieved in a 150 mm NaCl, 50 mm sodium acetate, pH 6.8 buffer supplemented with proteinase/phosphatase inhibitors and 10 mm EDTA for 16 h at 37 °C. The deglycosylation enzymes (0.05 unit) used were chondroitinase ABC from Proteus vulgaris (It catalyzes the removal of polysaccharides containing 1→4-β-d-hexosaminyl and 1→3-β-d-glucuronosyl or 1→3-α-l-iduronosyl linkages to disaccharides containing 4-deoxy-β-d-gluc-4-enuronosyl groups. It acts on chondroitin 4-sulfate, chondroitin 6-sulfate, and dermatan sulfate glycosaminoglycan side chains.), keratanase from Bacteroides fragilis (cleaves internal 1→4-β-galactose linkages in unbranched, repeating poly-N-acetyl-lactosamine and keratan sulfate), and heparitinase II from Flavobacterium heparinum (cleaves heparan sulfate), all purchased from Sigma-Aldrich. Following deglycosylation, the solutions were clarified again with centrifugation (16,000 × g for 10 min) to ensure that the samples were free of turbidity, and protein concentration was estimated by UV absorbance at 280 nm using extinction coefficient of 1.1 of 0.1% mg/ml solution calculated on the basis of the frequency of tyrosine and tryptophan, which are the main UV light-absorbing amino acids at 280 nm in mammalian proteins (7).One-dimensional ElectrophoresisThe NaCl, guanidine, and SDS extracts were denatured and reduced in sample buffer containing 100 mm Tris, pH 6.8, 40% glycerol, 0.2% SDS, 2% β-mercaptoethanol, and 0.02% bromphenol blue and boiled at 96 °C for 10 min. 35 μg of protein per sample were loaded and separated on Bis-Tris discontinuous 4–12% polyacrylamide gradient gels (NuPAGE, Invitrogen). Prestained protein standards were run alongside the samples to allow molecular mass estimation of proteins (All Blue, Precision Plus, Bio-Rad).Nanoflow LC-MS/MS for NaCl and Guanidine ExtractsAfter electrophoresis, gels were stained using the PlusOne Silver staining kit (GE Healthcare). Silver staining was used for band excision to avoid cross-contamination with fainter gel bands. (Coomassie staining will predominantly stain the abundant proteins; hence, fainter gel bands could be missed.). The gel bands were excised in identical parallel positions across lanes, and no “empty” gel pieces were left behind. Subsequently, all gel bands were subjected to in-gel digestion with trypsin using an Investigator ProGest (Genomic Solutions) robotic digestion system. Tryptic peptides from the NaCl and guanidine extracts were separated on a nanoflow LC system (Dionex UltiMate 3000) and eluted with a 40-min gradient (10–25% B in 35 min, 25–40% B in 5 min, 90% B in 10 min, and 2% B in 30min where A is 2% ACN, 0.1% formic acid in HPLC H2O and B is 90% ACN, 0.1% formic acid in HPLC H2O). The column (Dionex PepMap C18, 25-cm length, 75-μm internal diameter, 3-μm particle size) was coupled to a nanospray source (Picoview) using RePlay (Advion). After the direct LC-MS run, the flow was switched, and the portion stored in the capillary of the RePlay device was reanalyzed (“replay run”) (8). Spectra were collected from an ion trap mass analyzer (LTQ-Orbitrap XL, Thermo Fisher Scientific) using full ion scan mode over the mass-to-charge (m/z) range 450–1600. MS/MS was performed on the top six ions in each MS scan using the data-dependent acquisition mode with dynamic exclusion enabled. MS/MS peak lists were generated by extract_msn.exe and matched to a human database (UniProtKB/Swiss-Prot Release 14.6, 20,333 protein entries) using SEQUEST version 28 (revision 13) (Bioworks Browser 3.3.1 SP1, Thermo Fisher Scientific) and X! Tandem (version 2007.01.01.2). Carboxyamidomethylation of cysteine was chosen as a fixed modification, and oxidation of methionine was chosen as a variable modification. The mass tolerance was set at 1.5 amu for the precursor ions and at 1.0 amu for fragment ions. Two missed cleavages were allowed. Scaffold (version 2.0.5, Proteome Software Inc., Portland, OR) was used to calculate the spectral counts and to validate MS/MS-based peptide and protein identifications (9, 10). According to the default values in the Scaffold software, the following peptide thresholds were applied: X! Tandem: −log(Expect scores) > 2.0; SEQUEST: ΔCn > 0.10 and XCorr > 2.5 (2+), 3.5 (3+), and 3.5 (4+). Peptide identifications were accepted if they could be established at greater than 95.0% probability as specified by the Peptide Prophet algorithm (9). Protein identifications were accepted if they could be established at greater than 99.0% probability (10) with at least two independent peptides and a precursor ion mass accuracy ≤10 ppm.Nanoflow LC-MS/MS for SDS ExtractsThe protein concentration in the SDS extracts was estimated using the Groves equation, (A280 × 1.55) − (A260 × 0.76) (11), which takes into account nucleic acid interference. 35 μg of protein per sample were loaded and separated on Bis-Tris discontinuous 4–12% polyacrylamide gradient gels as above. The gel was silver-stained, and protein bands were digested by trypsin as described above. Tryptic peptides were separated on a split-free, nanoflow LC system (EASY-nLC, Proxeon) on a C18 column (Easy-nLC C18, Proxeon, 10-cm length, 75-μm internal diameter, 3-μm particle size) and eluted with an 80-min gradient (10–25% B in 35 min, 25–40% B in 5 min, 90% B in 10 min, and 2% B in 30min where A is 2% ACN, 0.1% formic acid in HPLC H2O and B is 90% ACN, 0.1% formic acid in HPLC H2O). The nanoflow LC system was not coupled to a RePlay device. Spectra were collected from a dual pressure, linear ion trap (LTQ Velos, Thermo Fisher Scientific) using full ion scan mode over the m/z range 300–1800. MS/MS was performed on the top 20 ions in each MS scan using the data-dependent acquisition mode with dynamic exclusion enabled. MS/MS peak lists were generated by extract_msn.exe and matched to a human database (UniProtKB/Swiss-Prot Release 15.13, 20,276 protein entries) using SEQUEST version 28 (revision 13) (Proteome Discoverer 1.2 beta, Thermo Fisher Scientific). Carboxyamidomethylation of cysteine was chosen as a fixed modification, and oxidation of methionine was chosen as a variable modification. The mass tolerance was set at 0.5 amu for the precursor ions and at 0.5 amu for fragment ions. Two missed cleavages were allowed. Scaffold (version 3, Proteome Software Inc.) was used to calculate the spectral counts and to validate MS/MS-based peptide and protein identifications as before. According to the default values in the Scaffold software, the following peptide thresholds were applied: SEQUEST: ΔCn > 0.10 and XCorr > 2.5 (2+), 3.5 (3+), and 3.5 (4+). Peptide identifications were accepted if they could be established at greater than 95.0% probability as specified by the Peptide Prophet algorithm (9). Protein identifications were accepted if they could be established at greater than 99.0% probability (10) with at least two independent peptides.Western BlottingAliquots of the NaCl, guanidine, and SDS extracts were mixed with denaturing sample buffer and boiled. 20 μg of protein per sample were loaded and separated on 4–12% gradient gels as above. Proteins were then transferred on nitrocellulose membranes. Membranes were blocked in 5% fat-free milk powder in PBS and then probed for 16 h at 4 °C with primary antibodies to β-actin (sc-130656), periostin (sc-67233), and fibronectin (sc-56391). The antibodies were purchased from Santa Cruz Biotechnology and were used at a 1:500 dilution in 5% BSA. The membranes were then treated with the appropriate secondary, peroxidase-conjugated antibodies (Dako) at a 1:2000 dilution. Finally, the blots were imaged using enhanced chemiluminescence (ECL; GE Healthcare), and films were developed on a Xograph processor. The densitometry for the lanes from developed blots for periostin and asporin was performed using ImageJ software.Gelatinolytic ZymographyGelatinolytic zymograms were used to detect the presence of MMP-2 and -9 in the 0.5 m NaCl extracts. Extract aliquots (20 μg of protein) were mixed with non-reducing sample buffer containing 100 mm Tris, pH 6.8, 40% glycerol, 0.2% SDS, and 0.02% bromphenol blue and separated by 10% SDS-PAGE in gels containing 0.5 mg/ml gelatin (Sigma-Aldrich). Prestained protein standards were run alongside the samples as before. To allow in-gel MMP activity, SDS was removed from the gels with three washes in a buffer containing 2% Triton X-100, 50 mm Tris, pH 7.4, and 200 mm NaCl for 45 min. Gels were then incubated for 16 h to detect MMP-9 or 36 h to detect MMP-2 at 37 °C in a buffer containing 50 mm Tris, pH 7.4, 200 mm NaCl, and 10 mm CaCl2. Finally, gels were stained for 60 min with Coomassie Brilliant Blue (Sigma-Aldrich) and destained for 2 h to visualize the MMPs.HistochemistrySmall representative pieces from each aortic sample were placed in optimal cutting temperature (OCT) compound (BDH), which was subsequently frozen. The frozen tissue blocks were used to generate consecutive 7-μm-thick cross-sections. The sections were air-dried (30 min) and fixed in 100% acetone for 20 min. Subsequently, sections were used either to generate hematoxylin- and eosin-stained histological slides or for immunohistochemical analysis using antibodies to podocan used at 2 μg/ml (MAB4220) (R&D Systems), sclerostin (sc-130258) used at 1 μg/ml, and agrin (sc-25528) used at used at 2 μg/ml (Santa Cruz Biotechnology). Negative controls were generated with either mouse (podocan) or rabbit (sclerostin and agrin) isotype IgG antibodies (Sigma-Aldrich) used at 2 μg/ml. The immunostaining was generated using the DAB+ (diaminobenzidine) peroxidase-based chromogen system (Dako) according to the manufacturer's instructions. Images were taken by a Zeiss Axioplan 2ie microscope interfaced to AxioVision software (version 3.0.6).Statistical Analysis of NaCl and Guanidine ExtractsFor each clinical sample (n = 4) and biochemical fraction (n = 2) analyzed, we obtained a technical replicate by using the RePlay device. The reproducibility of the RePlay analysis was assessed by calculating the Pearson correlation coefficient for the spectral counts. All spectra from proteins identified with at least two unique peptides were included in the analysis, and no outliers were removed. The p value for the linear dependence between the RePlay runs was obtained using the Fisher transformation. To assess biological variability among the four clinical specimens, we computed the coefficient of variation for all identified proteins using Scaffold software (version 2.0.5, Proteome Software Inc.). A coefficient of variation exceeding 100% was applied as a cutoff. Statistical significance was determined by using analysis of variance (ANOVA). A p value <0.05 was considered significant.

EXPERIMENTAL PROCEDURES

Tissue Processing, Decellularization, and Extraction of Extracellular Space ProteinsThe data presented were derived from three human aortic samples. The samples were obtained upon aortotomy performed during routine aortic valve replacement. They were collected from positions of the ascending aorta that were free of macroscopically evident vascular pathology, including atherosclerosis or aneurysm formation. The tissue was immediately snap frozen and subsequently kept in liquid nitrogen for later use. A fourth sample was also obtained upon aortotomy and processed as described below, but the proteomics analysis revealed proteolysis of ECM proteins. Therefore, we used it to examine the potential of our methodology to study proteolytic activity in the tissue. All procedures were approved by our local Regional Ethics Committee Board. Informed consent was obtained from all patients recruited into the study.Before extraction, the tissue pieces were partially thawed and weighed. Approximately 150 mg of tissue per aortic sample were immediately placed in ice-cold phosphate-buffered saline to remove plasma contaminants. Commercially available protease and phosphatase inhibitor mixtures (Sigma-Aldrich) were included according to the manufacturer's instructions to inhibit broad range proteinase activity, and 25 mm EDTA was included to ensure inhibition of metalloproteinases. While the tissue samples were immersed in the cold saline mixture, they were quickly diced with a scalpel into 8–10 smaller pieces. This would enhance (a) the initial removal of plasma contaminants and (b) the effective extraction of extracellular proteins, which is described below. The saline mixture was changed five times (∼30 ml total per sample).Then, the diced samples were incubated with 0.5 m NaCl, 10 mm Tris, pH 7.5 supplemented with proteinase/phosphatase inhibitor mixtures (Sigma-Aldrich) and 25 mm EDTA. The volume of the buffer was adjusted to 10:1 of the tissue weight (i.e. 100 mg in 1 ml), and the samples were mildly vortexed for 4 h at room temperature. The NaCl extracts were then desalted with centrifugation using desalting columns (Zeba Spin, Pierce). Following desalting, the extracts were mixed with 100% acetone (5:1 volume ratio) at −20 °C for 16 h. Proteins were precipitated with centrifugation (16,000 × g for 45 min), and the pellets were dried and redissolved in deglycosylation buffer (see below).Subsequently, the aortic samples were incubated with 0.08% SDS (10:1 buffer volume to tissue weight) supplemented with proteinase/phosphatase inhibitor mixtures and 25 mm EDTA. The samples were mildly vortexed for 4 h at room temperature to achieve removal of cellular components. From our experience with tissue extractions, we took particular care to ensure a low vortexing speed to decrease the possibility of mechanical disruption of the ECM. The SDS solution was then removed and stored frozen for later use.Finally, the samples were incubated in a 4 m guanidine HCl, 50 mm sodium acetate, pH 5.8 buffer (5:1 buffer volume to tissue weight) supplemented with proteinase/phosphatase inhibitor mixtures and 25 mm EDTA. Adjusting the pH to 5.8 is important; more than 30 years ago it was shown that at this pH guanidine is more effective in solubilizing vascular proteoglycans, whereas higher pH is required for the solubilization of the proteoglycans from other ECM-rich tissues, such as cartilage (6). The samples were then incubated for 48 h at room temperature and vortexed vigorously to enhance mechanical disruption of the ECM components. Subsequently, the guanidine extracts were mixed with 100% ethanol (5:1 volume ratio) at −20 °C for 16 h to ensure removal of guanidine (the presence of guanidine would hinder further biochemical processing). Proteins were precipitated with centrifugation (16,000 × g for 45 min), and the pellets were washed with 90% ethanol, dried, and redissolved in deglycosylation buffer (see below).Deglycosylation (removal of glycosaminoglycan side chains) of the different extracts was achieved in a 150 mm NaCl, 50 mm sodium acetate, pH 6.8 buffer supplemented with proteinase/phosphatase inhibitors and 10 mm EDTA for 16 h at 37 °C. The deglycosylation enzymes (0.05 unit) used were chondroitinase ABC from Proteus vulgaris (It catalyzes the removal of polysaccharides containing 1→4-β-d-hexosaminyl and 1→3-β-d-glucuronosyl or 1→3-α-l-iduronosyl linkages to disaccharides containing 4-deoxy-β-d-gluc-4-enuronosyl groups. It acts on chondroitin 4-sulfate, chondroitin 6-sulfate, and dermatan sulfate glycosaminoglycan side chains.), keratanase from Bacteroides fragilis (cleaves internal 1→4-β-galactose linkages in unbranched, repeating poly-N-acetyl-lactosamine and keratan sulfate), and heparitinase II from Flavobacterium heparinum (cleaves heparan sulfate), all purchased from Sigma-Aldrich. Following deglycosylation, the solutions were clarified again with centrifugation (16,000 × g for 10 min) to ensure that the samples were free of turbidity, and protein concentration was estimated by UV absorbance at 280 nm using extinction coefficient of 1.1 of 0.1% mg/ml solution calculated on the basis of the frequency of tyrosine and tryptophan, which are the main UV light-absorbing amino acids at 280 nm in mammalian proteins (7).One-dimensional ElectrophoresisThe NaCl, guanidine, and SDS extracts were denatured and reduced in sample buffer containing 100 mm Tris, pH 6.8, 40% glycerol, 0.2% SDS, 2% β-mercaptoethanol, and 0.02% bromphenol blue and boiled at 96 °C for 10 min. 35 μg of protein per sample were loaded and separated on Bis-Tris discontinuous 4–12% polyacrylamide gradient gels (NuPAGE, Invitrogen). Prestained protein standards were run alongside the samples to allow molecular mass estimation of proteins (All Blue, Precision Plus, Bio-Rad).Nanoflow LC-MS/MS for NaCl and Guanidine ExtractsAfter electrophoresis, gels were stained using the PlusOne Silver staining kit (GE Healthcare). Silver staining was used for band excision to avoid cross-contamination with fainter gel bands. (Coomassie staining will predominantly stain the abundant proteins; hence, fainter gel bands could be missed.). The gel bands were excised in identical parallel positions across lanes, and no “empty” gel pieces were left behind. Subsequently, all gel bands were subjected to in-gel digestion with trypsin using an Investigator ProGest (Genomic Solutions) robotic digestion system. Tryptic peptides from the NaCl and guanidine extracts were separated on a nanoflow LC system (Dionex UltiMate 3000) and eluted with a 40-min gradient (10–25% B in 35 min, 25–40% B in 5 min, 90% B in 10 min, and 2% B in 30min where A is 2% ACN, 0.1% formic acid in HPLC H2O and B is 90% ACN, 0.1% formic acid in HPLC H2O). The column (Dionex PepMap C18, 25-cm length, 75-μm internal diameter, 3-μm particle size) was coupled to a nanospray source (Picoview) using RePlay (Advion). After the direct LC-MS run, the flow was switched, and the portion stored in the capillary of the RePlay device was reanalyzed (“replay run”) (8). Spectra were collected from an ion trap mass analyzer (LTQ-Orbitrap XL, Thermo Fisher Scientific) using full ion scan mode over the mass-to-charge (m/z) range 450–1600. MS/MS was performed on the top six ions in each MS scan using the data-dependent acquisition mode with dynamic exclusion enabled. MS/MS peak lists were generated by extract_msn.exe and matched to a human database (UniProtKB/Swiss-Prot Release 14.6, 20,333 protein entries) using SEQUEST version 28 (revision 13) (Bioworks Browser 3.3.1 SP1, Thermo Fisher Scientific) and X! Tandem (version 2007.01.01.2). Carboxyamidomethylation of cysteine was chosen as a fixed modification, and oxidation of methionine was chosen as a variable modification. The mass tolerance was set at 1.5 amu for the precursor ions and at 1.0 amu for fragment ions. Two missed cleavages were allowed. Scaffold (version 2.0.5, Proteome Software Inc., Portland, OR) was used to calculate the spectral counts and to validate MS/MS-based peptide and protein identifications (9, 10). According to the default values in the Scaffold software, the following peptide thresholds were applied: X! Tandem: −log(Expect scores) > 2.0; SEQUEST: ΔCn > 0.10 and XCorr > 2.5 (2+), 3.5 (3+), and 3.5 (4+). Peptide identifications were accepted if they could be established at greater than 95.0% probability as specified by the Peptide Prophet algorithm (9). Protein identifications were accepted if they could be established at greater than 99.0% probability (10) with at least two independent peptides and a precursor ion mass accuracy ≤10 ppm.Nanoflow LC-MS/MS for SDS ExtractsThe protein concentration in the SDS extracts was estimated using the Groves equation, (A280 × 1.55) − (A260 × 0.76) (11), which takes into account nucleic acid interference. 35 μg of protein per sample were loaded and separated on Bis-Tris discontinuous 4–12% polyacrylamide gradient gels as above. The gel was silver-stained, and protein bands were digested by trypsin as described above. Tryptic peptides were separated on a split-free, nanoflow LC system (EASY-nLC, Proxeon) on a C18 column (Easy-nLC C18, Proxeon, 10-cm length, 75-μm internal diameter, 3-μm particle size) and eluted with an 80-min gradient (10–25% B in 35 min, 25–40% B in 5 min, 90% B in 10 min, and 2% B in 30min where A is 2% ACN, 0.1% formic acid in HPLC H2O and B is 90% ACN, 0.1% formic acid in HPLC H2O). The nanoflow LC system was not coupled to a RePlay device. Spectra were collected from a dual pressure, linear ion trap (LTQ Velos, Thermo Fisher Scientific) using full ion scan mode over the m/z range 300–1800. MS/MS was performed on the top 20 ions in each MS scan using the data-dependent acquisition mode with dynamic exclusion enabled. MS/MS peak lists were generated by extract_msn.exe and matched to a human database (UniProtKB/Swiss-Prot Release 15.13, 20,276 protein entries) using SEQUEST version 28 (revision 13) (Proteome Discoverer 1.2 beta, Thermo Fisher Scientific). Carboxyamidomethylation of cysteine was chosen as a fixed modification, and oxidation of methionine was chosen as a variable modification. The mass tolerance was set at 0.5 amu for the precursor ions and at 0.5 amu for fragment ions. Two missed cleavages were allowed. Scaffold (version 3, Proteome Software Inc.) was used to calculate the spectral counts and to validate MS/MS-based peptide and protein identifications as before. According to the default values in the Scaffold software, the following peptide thresholds were applied: SEQUEST: ΔCn > 0.10 and XCorr > 2.5 (2+), 3.5 (3+), and 3.5 (4+). Peptide identifications were accepted if they could be established at greater than 95.0% probability as specified by the Peptide Prophet algorithm (9). Protein identifications were accepted if they could be established at greater than 99.0% probability (10) with at least two independent peptides.Western BlottingAliquots of the NaCl, guanidine, and SDS extracts were mixed with denaturing sample buffer and boiled. 20 μg of protein per sample were loaded and separated on 4–12% gradient gels as above. Proteins were then transferred on nitrocellulose membranes. Membranes were blocked in 5% fat-free milk powder in PBS and then probed for 16 h at 4 °C with primary antibodies to β-actin (sc-130656), periostin (sc-67233), and fibronectin (sc-56391). The antibodies were purchased from Santa Cruz Biotechnology and were used at a 1:500 dilution in 5% BSA. The membranes were then treated with the appropriate secondary, peroxidase-conjugated antibodies (Dako) at a 1:2000 dilution. Finally, the blots were imaged using enhanced chemiluminescence (ECL; GE Healthcare), and films were developed on a Xograph processor. The densitometry for the lanes from developed blots for periostin and asporin was performed using ImageJ software.Gelatinolytic ZymographyGelatinolytic zymograms were used to detect the presence of MMP-2 and -9 in the 0.5 m NaCl extracts. Extract aliquots (20 μg of protein) were mixed with non-reducing sample buffer containing 100 mm Tris, pH 6.8, 40% glycerol, 0.2% SDS, and 0.02% bromphenol blue and separated by 10% SDS-PAGE in gels containing 0.5 mg/ml gelatin (Sigma-Aldrich). Prestained protein standards were run alongside the samples as before. To allow in-gel MMP activity, SDS was removed from the gels with three washes in a buffer containing 2% Triton X-100, 50 mm Tris, pH 7.4, and 200 mm NaCl for 45 min. Gels were then incubated for 16 h to detect MMP-9 or 36 h to detect MMP-2 at 37 °C in a buffer containing 50 mm Tris, pH 7.4, 200 mm NaCl, and 10 mm CaCl2. Finally, gels were stained for 60 min with Coomassie Brilliant Blue (Sigma-Aldrich) and destained for 2 h to visualize the MMPs.HistochemistrySmall representative pieces from each aortic sample were placed in optimal cutting temperature (OCT) compound (BDH), which was subsequently frozen. The frozen tissue blocks were used to generate consecutive 7-μm-thick cross-sections. The sections were air-dried (30 min) and fixed in 100% acetone for 20 min. Subsequently, sections were used either to generate hematoxylin- and eosin-stained histological slides or for immunohistochemical analysis using antibodies to podocan used at 2 μg/ml (MAB4220) (R&D Systems), sclerostin (sc-130258) used at 1 μg/ml, and agrin (sc-25528) used at used at 2 μg/ml (Santa Cruz Biotechnology). Negative controls were generated with either mouse (podocan) or rabbit (sclerostin and agrin) isotype IgG antibodies (Sigma-Aldrich) used at 2 μg/ml. The immunostaining was generated using the DAB+ (diaminobenzidine) peroxidase-based chromogen system (Dako) according to the manufacturer's instructions. Images were taken by a Zeiss Axioplan 2ie microscope interfaced to AxioVision software (version 3.0.6).Statistical Analysis of NaCl and Guanidine ExtractsFor each clinical sample (n = 4) and biochemical fraction (n = 2) analyzed, we obtained a technical replicate by using the RePlay device. The reproducibility of the RePlay analysis was assessed by calculating the Pearson correlation coefficient for the spectral counts. All spectra from proteins identified with at least two unique peptides were included in the analysis, and no outliers were removed. The p value for the linear dependence between the RePlay runs was obtained using the Fisher transformation. To assess biological variability among the four clinical specimens, we computed the coefficient of variation for all identified proteins using Scaffold software (version 2.0.5, Proteome Software Inc.). A coefficient of variation exceeding 100% was applied as a cutoff. Statistical significance was determined by using analysis of variance (ANOVA). A p value <0.05 was considered significant.

Tissue Processing, Decellularization, and Extraction of Extracellular Space ProteinsThe data presented were derived from three human aortic samples. The samples were obtained upon aortotomy performed during routine aortic valve replacement. They were collected from positions of the ascending aorta that were free of macroscopically evident vascular pathology, including atherosclerosis or aneurysm formation. The tissue was immediately snap frozen and subsequently kept in liquid nitrogen for later use. A fourth sample was also obtained upon aortotomy and processed as described below, but the proteomics analysis revealed proteolysis of ECM proteins. Therefore, we used it to examine the potential of our methodology to study proteolytic activity in the tissue. All procedures were approved by our local Regional Ethics Committee Board. Informed consent was obtained from all patients recruited into the study.Before extraction, the tissue pieces were partially thawed and weighed. Approximately 150 mg of tissue per aortic sample were immediately placed in ice-cold phosphate-buffered saline to remove plasma contaminants. Commercially available protease and phosphatase inhibitor mixtures (Sigma-Aldrich) were included according to the manufacturer's instructions to inhibit broad range proteinase activity, and 25 mm EDTA was included to ensure inhibition of metalloproteinases. While the tissue samples were immersed in the cold saline mixture, they were quickly diced with a scalpel into 8–10 smaller pieces. This would enhance (a) the initial removal of plasma contaminants and (b) the effective extraction of extracellular proteins, which is described below. The saline mixture was changed five times (∼30 ml total per sample).Then, the diced samples were incubated with 0.5 m NaCl, 10 mm Tris, pH 7.5 supplemented with proteinase/phosphatase inhibitor mixtures (Sigma-Aldrich) and 25 mm EDTA. The volume of the buffer was adjusted to 10:1 of the tissue weight (i.e. 100 mg in 1 ml), and the samples were mildly vortexed for 4 h at room temperature. The NaCl extracts were then desalted with centrifugation using desalting columns (Zeba Spin, Pierce). Following desalting, the extracts were mixed with 100% acetone (5:1 volume ratio) at −20 °C for 16 h. Proteins were precipitated with centrifugation (16,000 × g for 45 min), and the pellets were dried and redissolved in deglycosylation buffer (see below).Subsequently, the aortic samples were incubated with 0.08% SDS (10:1 buffer volume to tissue weight) supplemented with proteinase/phosphatase inhibitor mixtures and 25 mm EDTA. The samples were mildly vortexed for 4 h at room temperature to achieve removal of cellular components. From our experience with tissue extractions, we took particular care to ensure a low vortexing speed to decrease the possibility of mechanical disruption of the ECM. The SDS solution was then removed and stored frozen for later use.Finally, the samples were incubated in a 4 m guanidine HCl, 50 mm sodium acetate, pH 5.8 buffer (5:1 buffer volume to tissue weight) supplemented with proteinase/phosphatase inhibitor mixtures and 25 mm EDTA. Adjusting the pH to 5.8 is important; more than 30 years ago it was shown that at this pH guanidine is more effective in solubilizing vascular proteoglycans, whereas higher pH is required for the solubilization of the proteoglycans from other ECM-rich tissues, such as cartilage (6). The samples were then incubated for 48 h at room temperature and vortexed vigorously to enhance mechanical disruption of the ECM components. Subsequently, the guanidine extracts were mixed with 100% ethanol (5:1 volume ratio) at −20 °C for 16 h to ensure removal of guanidine (the presence of guanidine would hinder further biochemical processing). Proteins were precipitated with centrifugation (16,000 × g for 45 min), and the pellets were washed with 90% ethanol, dried, and redissolved in deglycosylation buffer (see below).Deglycosylation (removal of glycosaminoglycan side chains) of the different extracts was achieved in a 150 mm NaCl, 50 mm sodium acetate, pH 6.8 buffer supplemented with proteinase/phosphatase inhibitors and 10 mm EDTA for 16 h at 37 °C. The deglycosylation enzymes (0.05 unit) used were chondroitinase ABC from Proteus vulgaris (It catalyzes the removal of polysaccharides containing 1→4-β-d-hexosaminyl and 1→3-β-d-glucuronosyl or 1→3-α-l-iduronosyl linkages to disaccharides containing 4-deoxy-β-d-gluc-4-enuronosyl groups. It acts on chondroitin 4-sulfate, chondroitin 6-sulfate, and dermatan sulfate glycosaminoglycan side chains.), keratanase from Bacteroides fragilis (cleaves internal 1→4-β-galactose linkages in unbranched, repeating poly-N-acetyl-lactosamine and keratan sulfate), and heparitinase II from Flavobacterium heparinum (cleaves heparan sulfate), all purchased from Sigma-Aldrich. Following deglycosylation, the solutions were clarified again with centrifugation (16,000 × g for 10 min) to ensure that the samples were free of turbidity, and protein concentration was estimated by UV absorbance at 280 nm using extinction coefficient of 1.1 of 0.1% mg/ml solution calculated on the basis of the frequency of tyrosine and tryptophan, which are the main UV light-absorbing amino acids at 280 nm in mammalian proteins (7).One-dimensional ElectrophoresisThe NaCl, guanidine, and SDS extracts were denatured and reduced in sample buffer containing 100 mm Tris, pH 6.8, 40% glycerol, 0.2% SDS, 2% β-mercaptoethanol, and 0.02% bromphenol blue and boiled at 96 °C for 10 min. 35 μg of protein per sample were loaded and separated on Bis-Tris discontinuous 4–12% polyacrylamide gradient gels (NuPAGE, Invitrogen). Prestained protein standards were run alongside the samples to allow molecular mass estimation of proteins (All Blue, Precision Plus, Bio-Rad).Nanoflow LC-MS/MS for NaCl and Guanidine ExtractsAfter electrophoresis, gels were stained using the PlusOne Silver staining kit (GE Healthcare). Silver staining was used for band excision to avoid cross-contamination with fainter gel bands. (Coomassie staining will predominantly stain the abundant proteins; hence, fainter gel bands could be missed.). The gel bands were excised in identical parallel positions across lanes, and no “empty” gel pieces were left behind. Subsequently, all gel bands were subjected to in-gel digestion with trypsin using an Investigator ProGest (Genomic Solutions) robotic digestion system. Tryptic peptides from the NaCl and guanidine extracts were separated on a nanoflow LC system (Dionex UltiMate 3000) and eluted with a 40-min gradient (10–25% B in 35 min, 25–40% B in 5 min, 90% B in 10 min, and 2% B in 30min where A is 2% ACN, 0.1% formic acid in HPLC H2O and B is 90% ACN, 0.1% formic acid in HPLC H2O). The column (Dionex PepMap C18, 25-cm length, 75-μm internal diameter, 3-μm particle size) was coupled to a nanospray source (Picoview) using RePlay (Advion). After the direct LC-MS run, the flow was switched, and the portion stored in the capillary of the RePlay device was reanalyzed (“replay run”) (8). Spectra were collected from an ion trap mass analyzer (LTQ-Orbitrap XL, Thermo Fisher Scientific) using full ion scan mode over the mass-to-charge (m/z) range 450–1600. MS/MS was performed on the top six ions in each MS scan using the data-dependent acquisition mode with dynamic exclusion enabled. MS/MS peak lists were generated by extract_msn.exe and matched to a human database (UniProtKB/Swiss-Prot Release 14.6, 20,333 protein entries) using SEQUEST version 28 (revision 13) (Bioworks Browser 3.3.1 SP1, Thermo Fisher Scientific) and X! Tandem (version 2007.01.01.2). Carboxyamidomethylation of cysteine was chosen as a fixed modification, and oxidation of methionine was chosen as a variable modification. The mass tolerance was set at 1.5 amu for the precursor ions and at 1.0 amu for fragment ions. Two missed cleavages were allowed. Scaffold (version 2.0.5, Proteome Software Inc., Portland, OR) was used to calculate the spectral counts and to validate MS/MS-based peptide and protein identifications (9, 10). According to the default values in the Scaffold software, the following peptide thresholds were applied: X! Tandem: −log(Expect scores) > 2.0; SEQUEST: ΔCn > 0.10 and XCorr > 2.5 (2+), 3.5 (3+), and 3.5 (4+). Peptide identifications were accepted if they could be established at greater than 95.0% probability as specified by the Peptide Prophet algorithm (9). Protein identifications were accepted if they could be established at greater than 99.0% probability (10) with at least two independent peptides and a precursor ion mass accuracy ≤10 ppm.Nanoflow LC-MS/MS for SDS ExtractsThe protein concentration in the SDS extracts was estimated using the Groves equation, (A280 × 1.55) − (A260 × 0.76) (11), which takes into account nucleic acid interference. 35 μg of protein per sample were loaded and separated on Bis-Tris discontinuous 4–12% polyacrylamide gradient gels as above. The gel was silver-stained, and protein bands were digested by trypsin as described above. Tryptic peptides were separated on a split-free, nanoflow LC system (EASY-nLC, Proxeon) on a C18 column (Easy-nLC C18, Proxeon, 10-cm length, 75-μm internal diameter, 3-μm particle size) and eluted with an 80-min gradient (10–25% B in 35 min, 25–40% B in 5 min, 90% B in 10 min, and 2% B in 30min where A is 2% ACN, 0.1% formic acid in HPLC H2O and B is 90% ACN, 0.1% formic acid in HPLC H2O). The nanoflow LC system was not coupled to a RePlay device. Spectra were collected from a dual pressure, linear ion trap (LTQ Velos, Thermo Fisher Scientific) using full ion scan mode over the m/z range 300–1800. MS/MS was performed on the top 20 ions in each MS scan using the data-dependent acquisition mode with dynamic exclusion enabled. MS/MS peak lists were generated by extract_msn.exe and matched to a human database (UniProtKB/Swiss-Prot Release 15.13, 20,276 protein entries) using SEQUEST version 28 (revision 13) (Proteome Discoverer 1.2 beta, Thermo Fisher Scientific). Carboxyamidomethylation of cysteine was chosen as a fixed modification, and oxidation of methionine was chosen as a variable modification. The mass tolerance was set at 0.5 amu for the precursor ions and at 0.5 amu for fragment ions. Two missed cleavages were allowed. Scaffold (version 3, Proteome Software Inc.) was used to calculate the spectral counts and to validate MS/MS-based peptide and protein identifications as before. According to the default values in the Scaffold software, the following peptide thresholds were applied: SEQUEST: ΔCn > 0.10 and XCorr > 2.5 (2+), 3.5 (3+), and 3.5 (4+). Peptide identifications were accepted if they could be established at greater than 95.0% probability as specified by the Peptide Prophet algorithm (9). Protein identifications were accepted if they could be established at greater than 99.0% probability (10) with at least two independent peptides.Western BlottingAliquots of the NaCl, guanidine, and SDS extracts were mixed with denaturing sample buffer and boiled. 20 μg of protein per sample were loaded and separated on 4–12% gradient gels as above. Proteins were then transferred on nitrocellulose membranes. Membranes were blocked in 5% fat-free milk powder in PBS and then probed for 16 h at 4 °C with primary antibodies to β-actin (sc-130656), periostin (sc-67233), and fibronectin (sc-56391). The antibodies were purchased from Santa Cruz Biotechnology and were used at a 1:500 dilution in 5% BSA. The membranes were then treated with the appropriate secondary, peroxidase-conjugated antibodies (Dako) at a 1:2000 dilution. Finally, the blots were imaged using enhanced chemiluminescence (ECL; GE Healthcare), and films were developed on a Xograph processor. The densitometry for the lanes from developed blots for periostin and asporin was performed using ImageJ software.Gelatinolytic ZymographyGelatinolytic zymograms were used to detect the presence of MMP-2 and -9 in the 0.5 m NaCl extracts. Extract aliquots (20 μg of protein) were mixed with non-reducing sample buffer containing 100 mm Tris, pH 6.8, 40% glycerol, 0.2% SDS, and 0.02% bromphenol blue and separated by 10% SDS-PAGE in gels containing 0.5 mg/ml gelatin (Sigma-Aldrich). Prestained protein standards were run alongside the samples as before. To allow in-gel MMP activity, SDS was removed from the gels with three washes in a buffer containing 2% Triton X-100, 50 mm Tris, pH 7.4, and 200 mm NaCl for 45 min. Gels were then incubated for 16 h to detect MMP-9 or 36 h to detect MMP-2 at 37 °C in a buffer containing 50 mm Tris, pH 7.4, 200 mm NaCl, and 10 mm CaCl2. Finally, gels were stained for 60 min with Coomassie Brilliant Blue (Sigma-Aldrich) and destained for 2 h to visualize the MMPs.HistochemistrySmall representative pieces from each aortic sample were placed in optimal cutting temperature (OCT) compound (BDH), which was subsequently frozen. The frozen tissue blocks were used to generate consecutive 7-μm-thick cross-sections. The sections were air-dried (30 min) and fixed in 100% acetone for 20 min. Subsequently, sections were used either to generate hematoxylin- and eosin-stained histological slides or for immunohistochemical analysis using antibodies to podocan used at 2 μg/ml (MAB4220) (R&D Systems), sclerostin (sc-130258) used at 1 μg/ml, and agrin (sc-25528) used at used at 2 μg/ml (Santa Cruz Biotechnology). Negative controls were generated with either mouse (podocan) or rabbit (sclerostin and agrin) isotype IgG antibodies (Sigma-Aldrich) used at 2 μg/ml. The immunostaining was generated using the DAB+ (diaminobenzidine) peroxidase-based chromogen system (Dako) according to the manufacturer's instructions. Images were taken by a Zeiss Axioplan 2ie microscope interfaced to AxioVision software (version 3.0.6).Statistical Analysis of NaCl and Guanidine ExtractsFor each clinical sample (n = 4) and biochemical fraction (n = 2) analyzed, we obtained a technical replicate by using the RePlay device. The reproducibility of the RePlay analysis was assessed by calculating the Pearson correlation coefficient for the spectral counts. All spectra from proteins identified with at least two unique peptides were included in the analysis, and no outliers were removed. The p value for the linear dependence between the RePlay runs was obtained using the Fisher transformation. To assess biological variability among the four clinical specimens, we computed the coefficient of variation for all identified proteins using Scaffold software (version 2.0.5, Proteome Software Inc.). A coefficient of variation exceeding 100% was applied as a cutoff. Statistical significance was determined by using analysis of variance (ANOVA). A p value <0.05 was considered significant.

Tissue Processing, Decellularization, and Extraction of Extracellular Space ProteinsThe data presented were derived from three human aortic samples. The samples were obtained upon aortotomy performed during routine aortic valve replacement. They were collected from positions of the ascending aorta that were free of macroscopically evident vascular pathology, including atherosclerosis or aneurysm formation. The tissue was immediately snap frozen and subsequently kept in liquid nitrogen for later use. A fourth sample was also obtained upon aortotomy and processed as described below, but the proteomics analysis revealed proteolysis of ECM proteins. Therefore, we used it to examine the potential of our methodology to study proteolytic activity in the tissue. All procedures were approved by our local Regional Ethics Committee Board. Informed consent was obtained from all patients recruited into the study.Before extraction, the tissue pieces were partially thawed and weighed. Approximately 150 mg of tissue per aortic sample were immediately placed in ice-cold phosphate-buffered saline to remove plasma contaminants. Commercially available protease and phosphatase inhibitor mixtures (Sigma-Aldrich) were included according to the manufacturer's instructions to inhibit broad range proteinase activity, and 25 mm EDTA was included to ensure inhibition of metalloproteinases. While the tissue samples were immersed in the cold saline mixture, they were quickly diced with a scalpel into 8–10 smaller pieces. This would enhance (a) the initial removal of plasma contaminants and (b) the effective extraction of extracellular proteins, which is described below. The saline mixture was changed five times (∼30 ml total per sample).Then, the diced samples were incubated with 0.5 m NaCl, 10 mm Tris, pH 7.5 supplemented with proteinase/phosphatase inhibitor mixtures (Sigma-Aldrich) and 25 mm EDTA. The volume of the buffer was adjusted to 10:1 of the tissue weight (i.e. 100 mg in 1 ml), and the samples were mildly vortexed for 4 h at room temperature. The NaCl extracts were then desalted with centrifugation using desalting columns (Zeba Spin, Pierce). Following desalting, the extracts were mixed with 100% acetone (5:1 volume ratio) at −20 °C for 16 h. Proteins were precipitated with centrifugation (16,000 × g for 45 min), and the pellets were dried and redissolved in deglycosylation buffer (see below).Subsequently, the aortic samples were incubated with 0.08% SDS (10:1 buffer volume to tissue weight) supplemented with proteinase/phosphatase inhibitor mixtures and 25 mm EDTA. The samples were mildly vortexed for 4 h at room temperature to achieve removal of cellular components. From our experience with tissue extractions, we took particular care to ensure a low vortexing speed to decrease the possibility of mechanical disruption of the ECM. The SDS solution was then removed and stored frozen for later use.Finally, the samples were incubated in a 4 m guanidine HCl, 50 mm sodium acetate, pH 5.8 buffer (5:1 buffer volume to tissue weight) supplemented with proteinase/phosphatase inhibitor mixtures and 25 mm EDTA. Adjusting the pH to 5.8 is important; more than 30 years ago it was shown that at this pH guanidine is more effective in solubilizing vascular proteoglycans, whereas higher pH is required for the solubilization of the proteoglycans from other ECM-rich tissues, such as cartilage (6). The samples were then incubated for 48 h at room temperature and vortexed vigorously to enhance mechanical disruption of the ECM components. Subsequently, the guanidine extracts were mixed with 100% ethanol (5:1 volume ratio) at −20 °C for 16 h to ensure removal of guanidine (the presence of guanidine would hinder further biochemical processing). Proteins were precipitated with centrifugation (16,000 × g for 45 min), and the pellets were washed with 90% ethanol, dried, and redissolved in deglycosylation buffer (see below).Deglycosylation (removal of glycosaminoglycan side chains) of the different extracts was achieved in a 150 mm NaCl, 50 mm sodium acetate, pH 6.8 buffer supplemented with proteinase/phosphatase inhibitors and 10 mm EDTA for 16 h at 37 °C. The deglycosylation enzymes (0.05 unit) used were chondroitinase ABC from Proteus vulgaris (It catalyzes the removal of polysaccharides containing 1→4-β-d-hexosaminyl and 1→3-β-d-glucuronosyl or 1→3-α-l-iduronosyl linkages to disaccharides containing 4-deoxy-β-d-gluc-4-enuronosyl groups. It acts on chondroitin 4-sulfate, chondroitin 6-sulfate, and dermatan sulfate glycosaminoglycan side chains.), keratanase from Bacteroides fragilis (cleaves internal 1→4-β-galactose linkages in unbranched, repeating poly-N-acetyl-lactosamine and keratan sulfate), and heparitinase II from Flavobacterium heparinum (cleaves heparan sulfate), all purchased from Sigma-Aldrich. Following deglycosylation, the solutions were clarified again with centrifugation (16,000 × g for 10 min) to ensure that the samples were free of turbidity, and protein concentration was estimated by UV absorbance at 280 nm using extinction coefficient of 1.1 of 0.1% mg/ml solution calculated on the basis of the frequency of tyrosine and tryptophan, which are the main UV light-absorbing amino acids at 280 nm in mammalian proteins (7).

Tissue Processing, Decellularization, and Extraction of Extracellular Space Proteins

The data presented were derived from three human aortic samples. The samples were obtained upon aortotomy performed during routine aortic valve replacement. They were collected from positions of the ascending aorta that were free of macroscopically evident vascular pathology, including atherosclerosis or aneurysm formation. The tissue was immediately snap frozen and subsequently kept in liquid nitrogen for later use. A fourth sample was also obtained upon aortotomy and processed as described below, but the proteomics analysis revealed proteolysis of ECM proteins. Therefore, we used it to examine the potential of our methodology to study proteolytic activity in the tissue. All procedures were approved by our local Regional Ethics Committee Board. Informed consent was obtained from all patients recruited into the study.

Before extraction, the tissue pieces were partially thawed and weighed. Approximately 150 mg of tissue per aortic sample were immediately placed in ice-cold phosphate-buffered saline to remove plasma contaminants. Commercially available protease and phosphatase inhibitor mixtures (Sigma-Aldrich) were included according to the manufacturer's instructions to inhibit broad range proteinase activity, and 25 mm EDTA was included to ensure inhibition of metalloproteinases. While the tissue samples were immersed in the cold saline mixture, they were quickly diced with a scalpel into 8–10 smaller pieces. This would enhance (a) the initial removal of plasma contaminants and (b) the effective extraction of extracellular proteins, which is described below. The saline mixture was changed five times (∼30 ml total per sample).

Then, the diced samples were incubated with 0.5 m NaCl, 10 mm Tris, pH 7.5 supplemented with proteinase/phosphatase inhibitor mixtures (Sigma-Aldrich) and 25 mm EDTA. The volume of the buffer was adjusted to 10:1 of the tissue weight (i.e. 100 mg in 1 ml), and the samples were mildly vortexed for 4 h at room temperature. The NaCl extracts were then desalted with centrifugation using desalting columns (Zeba Spin, Pierce). Following desalting, the extracts were mixed with 100% acetone (5:1 volume ratio) at −20 °C for 16 h. Proteins were precipitated with centrifugation (16,000 × g for 45 min), and the pellets were dried and redissolved in deglycosylation buffer (see below).

Subsequently, the aortic samples were incubated with 0.08% SDS (10:1 buffer volume to tissue weight) supplemented with proteinase/phosphatase inhibitor mixtures and 25 mm EDTA. The samples were mildly vortexed for 4 h at room temperature to achieve removal of cellular components. From our experience with tissue extractions, we took particular care to ensure a low vortexing speed to decrease the possibility of mechanical disruption of the ECM. The SDS solution was then removed and stored frozen for later use.

Finally, the samples were incubated in a 4 m guanidine HCl, 50 mm sodium acetate, pH 5.8 buffer (5:1 buffer volume to tissue weight) supplemented with proteinase/phosphatase inhibitor mixtures and 25 mm EDTA. Adjusting the pH to 5.8 is important; more than 30 years ago it was shown that at this pH guanidine is more effective in solubilizing vascular proteoglycans, whereas higher pH is required for the solubilization of the proteoglycans from other ECM-rich tissues, such as cartilage (6). The samples were then incubated for 48 h at room temperature and vortexed vigorously to enhance mechanical disruption of the ECM components. Subsequently, the guanidine extracts were mixed with 100% ethanol (5:1 volume ratio) at −20 °C for 16 h to ensure removal of guanidine (the presence of guanidine would hinder further biochemical processing). Proteins were precipitated with centrifugation (16,000 × g for 45 min), and the pellets were washed with 90% ethanol, dried, and redissolved in deglycosylation buffer (see below).

Deglycosylation (removal of glycosaminoglycan side chains) of the different extracts was achieved in a 150 mm NaCl, 50 mm sodium acetate, pH 6.8 buffer supplemented with proteinase/phosphatase inhibitors and 10 mm EDTA for 16 h at 37 °C. The deglycosylation enzymes (0.05 unit) used were chondroitinase ABC from Proteus vulgaris (It catalyzes the removal of polysaccharides containing 1→4-β-d-hexosaminyl and 1→3-β-d-glucuronosyl or 1→3-α-l-iduronosyl linkages to disaccharides containing 4-deoxy-β-d-gluc-4-enuronosyl groups. It acts on chondroitin 4-sulfate, chondroitin 6-sulfate, and dermatan sulfate glycosaminoglycan side chains.), keratanase from Bacteroides fragilis (cleaves internal 1→4-β-galactose linkages in unbranched, repeating poly-N-acetyl-lactosamine and keratan sulfate), and heparitinase II from Flavobacterium heparinum (cleaves heparan sulfate), all purchased from Sigma-Aldrich. Following deglycosylation, the solutions were clarified again with centrifugation (16,000 × g for 10 min) to ensure that the samples were free of turbidity, and protein concentration was estimated by UV absorbance at 280 nm using extinction coefficient of 1.1 of 0.1% mg/ml solution calculated on the basis of the frequency of tyrosine and tryptophan, which are the main UV light-absorbing amino acids at 280 nm in mammalian proteins (7).

One-dimensional ElectrophoresisThe NaCl, guanidine, and SDS extracts were denatured and reduced in sample buffer containing 100 mm Tris, pH 6.8, 40% glycerol, 0.2% SDS, 2% β-mercaptoethanol, and 0.02% bromphenol blue and boiled at 96 °C for 10 min. 35 μg of protein per sample were loaded and separated on Bis-Tris discontinuous 4–12% polyacrylamide gradient gels (NuPAGE, Invitrogen). Prestained protein standards were run alongside the samples to allow molecular mass estimation of proteins (All Blue, Precision Plus, Bio-Rad).

One-dimensional Electrophoresis

The NaCl, guanidine, and SDS extracts were denatured and reduced in sample buffer containing 100 mm Tris, pH 6.8, 40% glycerol, 0.2% SDS, 2% β-mercaptoethanol, and 0.02% bromphenol blue and boiled at 96 °C for 10 min. 35 μg of protein per sample were loaded and separated on Bis-Tris discontinuous 4–12% polyacrylamide gradient gels (NuPAGE, Invitrogen). Prestained protein standards were run alongside the samples to allow molecular mass estimation of proteins (All Blue, Precision Plus, Bio-Rad).

Nanoflow LC-MS/MS for NaCl and Guanidine ExtractsAfter electrophoresis, gels were stained using the PlusOne Silver staining kit (GE Healthcare). Silver staining was used for band excision to avoid cross-contamination with fainter gel bands. (Coomassie staining will predominantly stain the abundant proteins; hence, fainter gel bands could be missed.). The gel bands were excised in identical parallel positions across lanes, and no “empty” gel pieces were left behind. Subsequently, all gel bands were subjected to in-gel digestion with trypsin using an Investigator ProGest (Genomic Solutions) robotic digestion system. Tryptic peptides from the NaCl and guanidine extracts were separated on a nanoflow LC system (Dionex UltiMate 3000) and eluted with a 40-min gradient (10–25% B in 35 min, 25–40% B in 5 min, 90% B in 10 min, and 2% B in 30min where A is 2% ACN, 0.1% formic acid in HPLC H2O and B is 90% ACN, 0.1% formic acid in HPLC H2O). The column (Dionex PepMap C18, 25-cm length, 75-μm internal diameter, 3-μm particle size) was coupled to a nanospray source (Picoview) using RePlay (Advion). After the direct LC-MS run, the flow was switched, and the portion stored in the capillary of the RePlay device was reanalyzed (“replay run”) (8). Spectra were collected from an ion trap mass analyzer (LTQ-Orbitrap XL, Thermo Fisher Scientific) using full ion scan mode over the mass-to-charge (m/z) range 450–1600. MS/MS was performed on the top six ions in each MS scan using the data-dependent acquisition mode with dynamic exclusion enabled. MS/MS peak lists were generated by extract_msn.exe and matched to a human database (UniProtKB/Swiss-Prot Release 14.6, 20,333 protein entries) using SEQUEST version 28 (revision 13) (Bioworks Browser 3.3.1 SP1, Thermo Fisher Scientific) and X! Tandem (version 2007.01.01.2). Carboxyamidomethylation of cysteine was chosen as a fixed modification, and oxidation of methionine was chosen as a variable modification. The mass tolerance was set at 1.5 amu for the precursor ions and at 1.0 amu for fragment ions. Two missed cleavages were allowed. Scaffold (version 2.0.5, Proteome Software Inc., Portland, OR) was used to calculate the spectral counts and to validate MS/MS-based peptide and protein identifications (9, 10). According to the default values in the Scaffold software, the following peptide thresholds were applied: X! Tandem: −log(Expect scores) > 2.0; SEQUEST: ΔCn > 0.10 and XCorr > 2.5 (2+), 3.5 (3+), and 3.5 (4+). Peptide identifications were accepted if they could be established at greater than 95.0% probability as specified by the Peptide Prophet algorithm (9). Protein identifications were accepted if they could be established at greater than 99.0% probability (10) with at least two independent peptides and a precursor ion mass accuracy ≤10 ppm.

Nanoflow LC-MS/MS for NaCl and Guanidine Extracts

After electrophoresis, gels were stained using the PlusOne Silver staining kit (GE Healthcare). Silver staining was used for band excision to avoid cross-contamination with fainter gel bands. (Coomassie staining will predominantly stain the abundant proteins; hence, fainter gel bands could be missed.). The gel bands were excised in identical parallel positions across lanes, and no “empty” gel pieces were left behind. Subsequently, all gel bands were subjected to in-gel digestion with trypsin using an Investigator ProGest (Genomic Solutions) robotic digestion system. Tryptic peptides from the NaCl and guanidine extracts were separated on a nanoflow LC system (Dionex UltiMate 3000) and eluted with a 40-min gradient (10–25% B in 35 min, 25–40% B in 5 min, 90% B in 10 min, and 2% B in 30min where A is 2% ACN, 0.1% formic acid in HPLC H2O and B is 90% ACN, 0.1% formic acid in HPLC H2O). The column (Dionex PepMap C18, 25-cm length, 75-μm internal diameter, 3-μm particle size) was coupled to a nanospray source (Picoview) using RePlay (Advion). After the direct LC-MS run, the flow was switched, and the portion stored in the capillary of the RePlay device was reanalyzed (“replay run”) (8). Spectra were collected from an ion trap mass analyzer (LTQ-Orbitrap XL, Thermo Fisher Scientific) using full ion scan mode over the mass-to-charge (m/z) range 450–1600. MS/MS was performed on the top six ions in each MS scan using the data-dependent acquisition mode with dynamic exclusion enabled. MS/MS peak lists were generated by extract_msn.exe and matched to a human database (UniProtKB/Swiss-Prot Release 14.6, 20,333 protein entries) using SEQUEST version 28 (revision 13) (Bioworks Browser 3.3.1 SP1, Thermo Fisher Scientific) and X! Tandem (version 2007.01.01.2). Carboxyamidomethylation of cysteine was chosen as a fixed modification, and oxidation of methionine was chosen as a variable modification. The mass tolerance was set at 1.5 amu for the precursor ions and at 1.0 amu for fragment ions. Two missed cleavages were allowed. Scaffold (version 2.0.5, Proteome Software Inc., Portland, OR) was used to calculate the spectral counts and to validate MS/MS-based peptide and protein identifications (9, 10). According to the default values in the Scaffold software, the following peptide thresholds were applied: X! Tandem: −log(Expect scores) > 2.0; SEQUEST: ΔCn > 0.10 and XCorr > 2.5 (2+), 3.5 (3+), and 3.5 (4+). Peptide identifications were accepted if they could be established at greater than 95.0% probability as specified by the Peptide Prophet algorithm (9). Protein identifications were accepted if they could be established at greater than 99.0% probability (10) with at least two independent peptides and a precursor ion mass accuracy ≤10 ppm.

Nanoflow LC-MS/MS for SDS ExtractsThe protein concentration in the SDS extracts was estimated using the Groves equation, (A280 × 1.55) − (A260 × 0.76) (11), which takes into account nucleic acid interference. 35 μg of protein per sample were loaded and separated on Bis-Tris discontinuous 4–12% polyacrylamide gradient gels as above. The gel was silver-stained, and protein bands were digested by trypsin as described above. Tryptic peptides were separated on a split-free, nanoflow LC system (EASY-nLC, Proxeon) on a C18 column (Easy-nLC C18, Proxeon, 10-cm length, 75-μm internal diameter, 3-μm particle size) and eluted with an 80-min gradient (10–25% B in 35 min, 25–40% B in 5 min, 90% B in 10 min, and 2% B in 30min where A is 2% ACN, 0.1% formic acid in HPLC H2O and B is 90% ACN, 0.1% formic acid in HPLC H2O). The nanoflow LC system was not coupled to a RePlay device. Spectra were collected from a dual pressure, linear ion trap (LTQ Velos, Thermo Fisher Scientific) using full ion scan mode over the m/z range 300–1800. MS/MS was performed on the top 20 ions in each MS scan using the data-dependent acquisition mode with dynamic exclusion enabled. MS/MS peak lists were generated by extract_msn.exe and matched to a human database (UniProtKB/Swiss-Prot Release 15.13, 20,276 protein entries) using SEQUEST version 28 (revision 13) (Proteome Discoverer 1.2 beta, Thermo Fisher Scientific). Carboxyamidomethylation of cysteine was chosen as a fixed modification, and oxidation of methionine was chosen as a variable modification. The mass tolerance was set at 0.5 amu for the precursor ions and at 0.5 amu for fragment ions. Two missed cleavages were allowed. Scaffold (version 3, Proteome Software Inc.) was used to calculate the spectral counts and to validate MS/MS-based peptide and protein identifications as before. According to the default values in the Scaffold software, the following peptide thresholds were applied: SEQUEST: ΔCn > 0.10 and XCorr > 2.5 (2+), 3.5 (3+), and 3.5 (4+). Peptide identifications were accepted if they could be established at greater than 95.0% probability as specified by the Peptide Prophet algorithm (9). Protein identifications were accepted if they could be established at greater than 99.0% probability (10) with at least two independent peptides.

Nanoflow LC-MS/MS for SDS Extracts

The protein concentration in the SDS extracts was estimated using the Groves equation, (A280 × 1.55) − (A260 × 0.76) (11), which takes into account nucleic acid interference. 35 μg of protein per sample were loaded and separated on Bis-Tris discontinuous 4–12% polyacrylamide gradient gels as above. The gel was silver-stained, and protein bands were digested by trypsin as described above. Tryptic peptides were separated on a split-free, nanoflow LC system (EASY-nLC, Proxeon) on a C18 column (Easy-nLC C18, Proxeon, 10-cm length, 75-μm internal diameter, 3-μm particle size) and eluted with an 80-min gradient (10–25% B in 35 min, 25–40% B in 5 min, 90% B in 10 min, and 2% B in 30min where A is 2% ACN, 0.1% formic acid in HPLC H2O and B is 90% ACN, 0.1% formic acid in HPLC H2O). The nanoflow LC system was not coupled to a RePlay device. Spectra were collected from a dual pressure, linear ion trap (LTQ Velos, Thermo Fisher Scientific) using full ion scan mode over the m/z range 300–1800. MS/MS was performed on the top 20 ions in each MS scan using the data-dependent acquisition mode with dynamic exclusion enabled. MS/MS peak lists were generated by extract_msn.exe and matched to a human database (UniProtKB/Swiss-Prot Release 15.13, 20,276 protein entries) using SEQUEST version 28 (revision 13) (Proteome Discoverer 1.2 beta, Thermo Fisher Scientific). Carboxyamidomethylation of cysteine was chosen as a fixed modification, and oxidation of methionine was chosen as a variable modification. The mass tolerance was set at 0.5 amu for the precursor ions and at 0.5 amu for fragment ions. Two missed cleavages were allowed. Scaffold (version 3, Proteome Software Inc.) was used to calculate the spectral counts and to validate MS/MS-based peptide and protein identifications as before. According to the default values in the Scaffold software, the following peptide thresholds were applied: SEQUEST: ΔCn > 0.10 and XCorr > 2.5 (2+), 3.5 (3+), and 3.5 (4+). Peptide identifications were accepted if they could be established at greater than 95.0% probability as specified by the Peptide Prophet algorithm (9). Protein identifications were accepted if they could be established at greater than 99.0% probability (10) with at least two independent peptides.

Western BlottingAliquots of the NaCl, guanidine, and SDS extracts were mixed with denaturing sample buffer and boiled. 20 μg of protein per sample were loaded and separated on 4–12% gradient gels as above. Proteins were then transferred on nitrocellulose membranes. Membranes were blocked in 5% fat-free milk powder in PBS and then probed for 16 h at 4 °C with primary antibodies to β-actin (sc-130656), periostin (sc-67233), and fibronectin (sc-56391). The antibodies were purchased from Santa Cruz Biotechnology and were used at a 1:500 dilution in 5% BSA. The membranes were then treated with the appropriate secondary, peroxidase-conjugated antibodies (Dako) at a 1:2000 dilution. Finally, the blots were imaged using enhanced chemiluminescence (ECL; GE Healthcare), and films were developed on a Xograph processor. The densitometry for the lanes from developed blots for periostin and asporin was performed using ImageJ software.

Western Blotting

Aliquots of the NaCl, guanidine, and SDS extracts were mixed with denaturing sample buffer and boiled. 20 μg of protein per sample were loaded and separated on 4–12% gradient gels as above. Proteins were then transferred on nitrocellulose membranes. Membranes were blocked in 5% fat-free milk powder in PBS and then probed for 16 h at 4 °C with primary antibodies to β-actin (sc-130656), periostin (sc-67233), and fibronectin (sc-56391). The antibodies were purchased from Santa Cruz Biotechnology and were used at a 1:500 dilution in 5% BSA. The membranes were then treated with the appropriate secondary, peroxidase-conjugated antibodies (Dako) at a 1:2000 dilution. Finally, the blots were imaged using enhanced chemiluminescence (ECL; GE Healthcare), and films were developed on a Xograph processor. The densitometry for the lanes from developed blots for periostin and asporin was performed using ImageJ software.

Gelatinolytic ZymographyGelatinolytic zymograms were used to detect the presence of MMP-2 and -9 in the 0.5 m NaCl extracts. Extract aliquots (20 μg of protein) were mixed with non-reducing sample buffer containing 100 mm Tris, pH 6.8, 40% glycerol, 0.2% SDS, and 0.02% bromphenol blue and separated by 10% SDS-PAGE in gels containing 0.5 mg/ml gelatin (Sigma-Aldrich). Prestained protein standards were run alongside the samples as before. To allow in-gel MMP activity, SDS was removed from the gels with three washes in a buffer containing 2% Triton X-100, 50 mm Tris, pH 7.4, and 200 mm NaCl for 45 min. Gels were then incubated for 16 h to detect MMP-9 or 36 h to detect MMP-2 at 37 °C in a buffer containing 50 mm Tris, pH 7.4, 200 mm NaCl, and 10 mm CaCl2. Finally, gels were stained for 60 min with Coomassie Brilliant Blue (Sigma-Aldrich) and destained for 2 h to visualize the MMPs.

Gelatinolytic Zymography

Gelatinolytic zymograms were used to detect the presence of MMP-2 and -9 in the 0.5 m NaCl extracts. Extract aliquots (20 μg of protein) were mixed with non-reducing sample buffer containing 100 mm Tris, pH 6.8, 40% glycerol, 0.2% SDS, and 0.02% bromphenol blue and separated by 10% SDS-PAGE in gels containing 0.5 mg/ml gelatin (Sigma-Aldrich). Prestained protein standards were run alongside the samples as before. To allow in-gel MMP activity, SDS was removed from the gels with three washes in a buffer containing 2% Triton X-100, 50 mm Tris, pH 7.4, and 200 mm NaCl for 45 min. Gels were then incubated for 16 h to detect MMP-9 or 36 h to detect MMP-2 at 37 °C in a buffer containing 50 mm Tris, pH 7.4, 200 mm NaCl, and 10 mm CaCl2. Finally, gels were stained for 60 min with Coomassie Brilliant Blue (Sigma-Aldrich) and destained for 2 h to visualize the MMPs.

HistochemistrySmall representative pieces from each aortic sample were placed in optimal cutting temperature (OCT) compound (BDH), which was subsequently frozen. The frozen tissue blocks were used to generate consecutive 7-μm-thick cross-sections. The sections were air-dried (30 min) and fixed in 100% acetone for 20 min. Subsequently, sections were used either to generate hematoxylin- and eosin-stained histological slides or for immunohistochemical analysis using antibodies to podocan used at 2 μg/ml (MAB4220) (R&D Systems), sclerostin (sc-130258) used at 1 μg/ml, and agrin (sc-25528) used at used at 2 μg/ml (Santa Cruz Biotechnology). Negative controls were generated with either mouse (podocan) or rabbit (sclerostin and agrin) isotype IgG antibodies (Sigma-Aldrich) used at 2 μg/ml. The immunostaining was generated using the DAB+ (diaminobenzidine) peroxidase-based chromogen system (Dako) according to the manufacturer's instructions. Images were taken by a Zeiss Axioplan 2ie microscope interfaced to AxioVision software (version 3.0.6).

Histochemistry

Small representative pieces from each aortic sample were placed in optimal cutting temperature (OCT) compound (BDH), which was subsequently frozen. The frozen tissue blocks were used to generate consecutive 7-μm-thick cross-sections. The sections were air-dried (30 min) and fixed in 100% acetone for 20 min. Subsequently, sections were used either to generate hematoxylin- and eosin-stained histological slides or for immunohistochemical analysis using antibodies to podocan used at 2 μg/ml (MAB4220) (R&D Systems), sclerostin (sc-130258) used at 1 μg/ml, and agrin (sc-25528) used at used at 2 μg/ml (Santa Cruz Biotechnology). Negative controls were generated with either mouse (podocan) or rabbit (sclerostin and agrin) isotype IgG antibodies (Sigma-Aldrich) used at 2 μg/ml. The immunostaining was generated using the DAB+ (diaminobenzidine) peroxidase-based chromogen system (Dako) according to the manufacturer's instructions. Images were taken by a Zeiss Axioplan 2ie microscope interfaced to AxioVision software (version 3.0.6).

Statistical Analysis of NaCl and Guanidine ExtractsFor each clinical sample (n = 4) and biochemical fraction (n = 2) analyzed, we obtained a technical replicate by using the RePlay device. The reproducibility of the RePlay analysis was assessed by calculating the Pearson correlation coefficient for the spectral counts. All spectra from proteins identified with at least two unique peptides were included in the analysis, and no outliers were removed. The p value for the linear dependence between the RePlay runs was obtained using the Fisher transformation. To assess biological variability among the four clinical specimens, we computed the coefficient of variation for all identified proteins using Scaffold software (version 2.0.5, Proteome Software Inc.). A coefficient of variation exceeding 100% was applied as a cutoff. Statistical significance was determined by using analysis of variance (ANOVA). A p value <0.05 was considered significant.

Statistical Analysis of NaCl and Guanidine Extracts

For each clinical sample (n = 4) and biochemical fraction (n = 2) analyzed, we obtained a technical replicate by using the RePlay device. The reproducibility of the RePlay analysis was assessed by calculating the Pearson correlation coefficient for the spectral counts. All spectra from proteins identified with at least two unique peptides were included in the analysis, and no outliers were removed. The p value for the linear dependence between the RePlay runs was obtained using the Fisher transformation. To assess biological variability among the four clinical specimens, we computed the coefficient of variation for all identified proteins using Scaffold software (version 2.0.5, Proteome Software Inc.). A coefficient of variation exceeding 100% was applied as a cutoff. Statistical significance was determined by using analysis of variance (ANOVA). A p value <0.05 was considered significant.

RESULTSBiochemical Extraction of Proteins in Extracellular EnvironmentOur extraction methodology was developed on human aortic samples obtained during cardiac surgery for aortic valve replacement of degenerative aortic valve stenosis. Fig. 1A is a schematic summary of the methodology used in this study. The samples (n = 3) were macroscopically classified as normal during surgery, and upon histological examination, they showed no pathological signs. Extraction of the proteins of the vascular extracellular environment and reduction of the contamination with cellular proteins were achieved by a three-step extraction procedure. First, the tissue samples were treated with 0.5 m NaCl. The salt ions induce displacement of polyionic interactions between proteins, thus enabling the extraction of loosely bound extracellular proteins, including newly synthesized ECM proteins and degradation products (12), providing an overview of tissue turnover. The tissue was then decellularized using SDS. In contrast to other decellularization reagents, such as non-ionic detergents (Triton X-100) (13), 0.1% SDS has been previously shown to be very effective in solubilizing cytoplasmic and nuclear membranes, thereby removing cellular components from tissues (14). However, the ionic detergent above its critical micelle concentration can also cause protein denaturation and disrupt native ECM proteins (15).Fig. 1.Identification of extracellular proteins in human aortas. A, schematic summary. Biochemical subfractionation was used for the enrichment of extracellular space proteins before proteomic analysis. B, validation of decellularization by hematoxylin/eosin staining. Representative images of a hematoxylin/eosin-stained human aorta before (left panel) and after decellularization with 0.08% SDS (right panel) are shown. L, lumen; I, intima; M, media. Arrows depict the intima/media border. Note the loss of nuclear (blue; hematoxylin) and cytoplasmic contents (pink; eosin), whereas other eosinophilic structures, such as collagen fibers, are preserved. C, immunoblotting and silver staining. The ECM glycoprotein AEBP1 is found in the NaCl and guanidine (Guan) extracts, which are enriched in proteins of the extracellular space (upper panel). Cytoplasmic β-actin is predominantly found in the SDS extracts (middle panel), demonstrating successful depletion of cellular contents. Silver staining of proteins from all extracts separated by one-dimensional SDS-PAGE shows the pattern of proteins extracted in the three different extraction steps (bottom panel).To substantially remove the cellular material while preserving the ECM and its associated proteins as much as possible, we used 2.8 mm (0.08%) SDS in distilled water. This concentration is well below the critical micelle concentration of SDS in distilled water, which is 8 mm (16). In addition, at this concentration, SDS is effective in solubilizing cell membrane lipid bilayers (17). Treatment with 0.08% SDS was sufficient to remove most cellular components as shown by the marked reduction of nuclear staining (hematoxylin; Fig. 1B, blue). The cytoplasmic staining (eosin; pink) was also reduced, but other eosinophilic structures, such as collagen and elastin, were preserved in the tissue. Finally, the specimens were subjected to denaturing extraction in a 4 m guanidine HCl buffer. Guanidine HCl extractions were developed more than 40 years ago as a very effective way to solubilize most of the strongly bound ECM components, including large aggregating proteoglycans (versican, aggrecan, etc.), small proteoglycans (decorin, biglycan, etc.), cell attachment matrix glycoproteins (such as type VI collagen, fibronectins, and laminins), and basement membrane components (perlecan, type IV collagen, etc.). Guanidine induces disaggregation of ECM components by destabilizing the ionic, disulfide-dependent protein conformations (18). Following dissociative extraction, only the insoluble, heavily cross-linked interstitial matrix, namely type I and III collagens and elastin, remains in the tissue. Western blotting confirmed that β-actin immunoreactivity is predominantly found in the SDS extracts (Fig 1C, middle panel), which also had the highest protein concentration per milliliter (supplemental Fig. 1). On the other hand, the extracellular AEBP1 isoform, an ECM-associated glycoprotein, was detected in the NaCl and guanidine extracts by Western blotting (Fig 1C, upper panel), indicating that these extracts are enriched in proteins of the extracellular space. Silver staining revealed a distinct but reproducible protein pattern in the three different extraction steps (Fig. 1C, bottom panel).The different extracts were subjected to enzymatic deglycosylation to remove chondroitin, keratan, and heparan sulfate glycosaminoglycan (GAG) chains from the matrix glycoproteins. Deglycosylation is a critical step in our methodology. First, GAGs increase the molecular weight of glycoproteins and affect their separation and resolution by one-dimensional electrophoresis. Second, the presence of GAGs greatly reduces the chance for the identification of proteoglycans by mass spectrometry.Proteomics Analysis of ExtractsAfter separation by SDS-PAGE, protein bands were excised from silver-stained gels, subjected to in-gel tryptic digestion, and analyzed by LC-MS/MS. The peptides were injected into a high mass accuracy ion trap (LTQ-Orbitrap XL, Thermo Fisher Scientific) using RePlay (Advion). This injection system splits the gradient from the analytical column and allows the reanalysis of the same sample in a single LC-MS/MS run (8). 272 and 321 proteins were identified in the NaCl and guanidine extracts, respectively. All the peptides identified by LC-MS/MS in the NaCl and guanidine extracts are shown in supplemental Tables 1 and 2. According to their gene ontology annotation (European Molecular Biology Laboratory-European Bioinformatics Institute database, www.uniprot.org), a quarter of these proteins were located in the extracellular space: among the 61 extracellular proteins in the NaCl extracts (Table I) and the 84 extracellular proteins in the guanidine extracts (Table II), proteins highlighted with Footnote b were, to the best of our knowledge, identified for the first time in the vasculature by proteomics. Lipoproteins were included (even though they are plasma-derived) because of their intimate relationship with proteoglycans and their relevance in cardiovascular pathology. The classification of the identified extracellular space proteins in the NaCl and the guanidine extracts is shown in Fig. 2, A and B. As expected, the denaturing guanidine extracts were particularly enriched in ECM proteoglycans and glycoproteins (50%), whereas the mildly dissociative NaCl extracts were enriched in proteases and protease inhibitors (21%), indicating the importance of this subproteome for studying proteolytic processes in tissues.Table IExtracellular space proteins identified by proteomics in three NaCl extractsCOMP, cartilage oligomeric matrix protein; IGF, insulin-like growth factor.Protein nameSwiss-Prot accession nameMolecular massUnique peptides, nUnique spectra, nCoverageTotal spectra,ankDa%Proteoglycans and glycoproteins    LumicanLUM_HUMAN38404020.0249    Tenascin-XTENX_HUMAN46466668.1156    Fibulin-3FBLN3_HUMAN55383822.0155    Adipocyte enhancer-binding protein 1bAEBP1_HUMAN131373911.0153    FibronectinFINC_HUMAN26334347.3111    PerlecanPGBM_HUMAN46916162.550    MimecanMIME_HUMAN34131312.049    VersicanCSPG2_HUMAN37316163.746    TetranectinTETN_HUMAN23202029.039    FibromodulinFMOD_HUMAN43667.426    Fibulin-1FBLN1_HUMAN7714149.125    Latent TGFβ-binding protein 2bLTBP2_HUMAN19515155.223    Thrombospondin-1TSP1_HUMAN12911115.118    PodocanbPODN_HUMAN6910106.917    Fibrillin-1FBN1_HUMAN31210102.116    Fibulin-5FBLN5_HUMAN50667.812    Galectin-1LEG1_HUMAN157726.09    DecorinPGS2_HUMAN403310.08    Latent TGFβ-binding protein 4bLTBP4_HUMAN173331.67    ClusterinCLUS_HUMAN52333.65    COMPCOMP_HUMAN83226.33    TenascinTENA_HUMAN241220.952    DermatopontinDERM_HUMAN24229.52Other ECM-associated proteins    Extracellular superoxide dismutaseSODE_HUMAN26222746.0209    Pigment epithelium-derived factorPEDF_HUMAN46323216.082    Serum amyloid P-componentSAMP_HUMAN25313126.062    IGF-binding protein 7bIBP7_HUMAN29151521.047    Apolipoprotein A-IAPOA1_HUMAN316625.019    Apolipoprotein A-IVAPOA4_HUMAN457717.012    Apolipoprotein HbAPOH_HUMAN388814.011    Hepatoma-derived growth factorbHDGF_HUMAN27667.911    Secreted frizzled-related protein 1bSFRP1_HUMAN35559.29    CeruloplasminCERU_HUMAN122442.39    Secreted frizzled-related protein 3bSFRP3_HUMAN36449.28    Apolipoprotein DAPOD_HUMAN216620.08    Apolipoprotein EAPOE_HUMAN36557.67    LIM and cysteine-rich domains protein 1bLMCD1_HUMAN41667.77    Spondin-1bSPON1_HUMAN91332.76    Target of Nesh-SH3bTARSH_HUMAN119222.66Proteases and protease inhibitors    Tryptase β-1bTRYB1_HUMAN319922.022    Procollagen C-endopeptidase enhancer 1PCOC1_HUMAN48887.818    Cathepsin GbCATG_HUMAN29776.716    CalreticulinCALR_HUMAN489911.013    Cathepsin ZbCATZ_HUMAN34667.310    Plasma glutamate carboxypeptidasebPGCP_HUMAN52554.28    Cathepsin DCATD_HUMAN45557.07    Leukocyte elastase inhibitorILEU_HUMAN43559.07    Proactivator polypeptidebSAP_HUMAN58555.37    MMP-2bMMP2_HUMAN74223.34    KallistatinbKAIN_HUMAN49224.73    Carboxypeptidase-like protein X2bCPXM2_HUMAN86224.62kDa%Collagens    Collagen α-1(XIV)bCOEA1_HUMAN194737314.0233    Collagen α-1(XII)COCA1_HUMAN33320202.841    Collagen α-3(VI)CO6A3_HUMAN34416163.729    Collagen α-1(XVIII)COIA1_HUMAN17811113.722    Collagen α-2(VI)CO6A2_HUMAN109443.06    Collagen α-1(XV)COFA1_HUMAN142332.04    Collagen α-1(I)CO1A1_HUMAN129331.54a Sum of all spectra identified for each protein in the three biological samples, performed with two technical replicates each.b Proteins found for the first time by proteomics in the vasculature.Table IIExtracellular space proteins identified by proteomics in three guanidine extractsIGF, insulin-like growth factor.Protein nameSwiss-Prot accession nameMolecular massUnique peptides, nUnique spectra, nCoverageTotal spectra,ankDa%Proteoglycans and glycoproteins    PerlecanPGBM_HUMAN46936748119.01596    FibronectinFINC_HUMAN26318221820.01028    VersicanCSPG2_HUMAN37376894.4681    AggrecanPGCA_HUMAN25070826.3451    BiglycanPGS1_HUMAN42566236.0352    LactadherinMFGM_HUMAN43646748.0309    ProlarginPRELP_HUMAN44557837.0283    PeriostinPOSTN_HUMAN9310011729.0257    LumicanLUM_HUMAN38535830.0204    DecorinPGS2_HUMAN40435129.0173    ClusterinCLUS_HUMAN52444823.0173    Tenascin-XTENX_HUMAN46479794.5140    Laminin subunit γ-1LAMC1_HUMAN178404418.0129    Link protein 1HPLN1_HUMAN40262933.0117    Laminin subunit β-2LAMB2_HUMAN196343414.0104    VitronectinVTNC_HUMAN54252714.0102    Nidogen-1NID1_HUMAN136566415.0101    Adipocyte enhancer-binding protein 1bAEBP1_HUMAN13134407.694    Laminin subunit α-5LAMA5_HUMAN40039394.691    TenascinTENA_HUMAN24125255.652    MimecanMIME_HUMAN34212115.048    EGF-containing fibulin-like ECM protein 1FBLN3_HUMAN55222415.048    Fibulin-5FBLN5_HUMAN50171714.040    Galectin-1LEG1_HUMAN15172053.040    TGFβ-induced protein ig-h3BGH3_HUMAN75232312.035    EMILIN-1EMIL1_HUMAN107883.030    Fibulin-1FBLN1_HUMAN77171712.029    Latent TGFβ-binding protein 2bLTBP2_HUMAN19513132.722    FibromodulinFMOD_HUMAN438118.222    Microfibril-associated glycoprotein 4MFAP4_HUMAN298812.021    DermatopontinDERM_HUMAN248912.021    Fibrillin-1FBN1_HUMAN31215152.421    Latent TGFβ-binding protein 4bLTBP4_HUMAN17312124.215    Nidogen-2NID2_HUMAN151883.511    AgrinbAGRIN_HUMAN215772.311    AsporinASPN_HUMAN438811.011    Cartilage oligomeric matrix proteinCOMP_HUMAN83443.67    Thrombospondin-1TSP1_HUMAN129331.87    Matrix Gla proteinMGP_HUMAN124413.06    Galectin-3LEG3_HUMAN26226.04    SclerostinbSOST_HUMAN24225.63    Matrilin-2bMATN2_HUMAN107222.33Other ECM-associated proteins    Serum amyloid P-componentSAMP_HUMAN25323333.0169    Tubulointerstitial nephritis antigen-likebTINAL_HUMAN52465521.0131    Extracellular superoxide dismutaseSODE_HUMAN26283525.0102    Target of Nesh-SH3bTARSH_HUMAN11930338.689    IGF-binding protein 7bIBP7_HUMAN29303128.059    Cysteine-rich protein 2CRIP2_HUMAN22202125.057    TGFβ-1-induced transcript 1 proteinbTGFI1_HUMAN50272745    LIM and cysteine-rich domains protein 1bLMCD1_HUMAN41252534.039    Apolipoprotein HbAPOH_HUMAN3810129.032    Galectin-3-binding proteinLG3BP_HUMAN65121212.023    RPE-spondinbRPESP_HUMAN30121222.019kDa%    Carboxypeptidase-like protein X2bCPXM2_HUMAN869117.316    Latent TGFβ-binding protein, isoform 1SbLTB1S_HUMAN153892.713    Pigment epithelium derived factorPEDF_HUMAN46558.68    Secreted frizzled-related protein 1bSFRP1_HUMAN35338.96    ProperdinbPROP_HUMAN51345.15    Spondin-1bSPON1_HUMAN91333.04    Neutrophil defensin 1bDEF1_HUMAN10229.64Proteases and protease inhibitors    Serine protease HTRA1HTRA1_HUMAN51222213.068    ChymasebCMA1_HUMAN27121511.023    Lysyl oxidase homolog 1bLOXL1_HUMAN6312127.323    Metalloproteinase inhibitor 3TIMP3_HUMAN249917.012    Tryptase β-1bTRYB1_HUMAN31679.811    Procollagen C-endopeptidase enhancer 1PCOC1_HUMAN48776.710    Cathepsin GbCATG_HUMAN297715.08    Cathepsin DCATD_HUMAN45442.77    AntileukoproteinasebSLPI_HUMAN14339.16    MyeloblastinbPRTN3_HUMAN28224.74Collagens    Collagen α-3(VI)CO6A3_HUMAN34416418620.0898    Collagen α-1(XIV)bCOEA1_HUMAN19413615020.0529    Collagen α-1(XVIII)COIA1_HUMAN17845485.9374    Collagen α-1(VI)CO6A1_HUMAN109507116.0282    Collagen α-2(VI)CO6A2_HUMAN109313119.0155    Collagen α-2(IV)CO4A2_HUMAN16826313.9139    Collagen α-1(XV)COFA1_HUMAN14232347.5100    Collagen α-1(XII)COCA1_HUMAN33342424.090    Collagen α-2(I)CO1A2_HUMAN12912153.570    Collagen α-1(I)CO1A1_HUMAN13914155.951    Collagen α-1(VIII)CO8A1_HUMAN73551.333    Collagen α-1(V)CO5A1_HUMAN184660.623    Collagen α-2(VIII)CO8A2_HUMAN67443.39    Collagen α-1(IV)CO4A1_HUMAN161551.88a Sum of all spectra identified for each protein in the three biological samples, performed with two technical replicates each.b Proteins found for the first time by proteomics in the vasculature.Fig. 2.Protein categories. The classification and numerical distribution of the identified extracellular proteins in the NaCl (A), guanidine (B), and SDS (C) extracts are shown. The size of the doughnut charts is representative of the number of extracellular space proteins identified in the different extracts.To identify proteins of the extracellular space extracted by 0.08% SDS, we performed proteomics analysis of the SDS extracts. Because of the increased complexity of these samples, tryptic peptides were injected into a dual pressure, linear ion trap (LTQ Velos, Thermo Fisher Scientific) with faster scan speed. As expected, the majority of the identified proteins were cellular (data not shown), and only 28 extracellular space proteins were identified (Table III). The classification of the identified extracellular space proteins in the SDS extracts is shown in Fig. 2C. Interestingly, apart from type XIV collagen, all other extracellular space proteins have been previously identified by proteomics in the vasculature. Also, of note is the absence of proteases and protease inhibitors in these extracts either because they are obscured by abundant cellular proteins or because those that remain in the tissue following 0.5 m NaCl extraction are tightly bound to the ECM and require dissociative conditions to be extracted (i.e. guanidine). Peptides of extracellular space proteins identified by LC-MS/MS in the SDS extracts are shown in supplemental Table 3.Table IIIExtracellular space proteins identified by proteomics in three SDS extractsProtein nameSwiss-Prot accession nameMolecular massUnique peptides, nUnique spectra, nCoverageTotal spectra,ankDa%Proteoglycans and glycoproteins    BiglycanPGS1_HUMAN42242832.6141    DecorinPGS2_HUMAN40242433.185    MimecanMIME_HUMAN34212215.076    LumicanLUM_HUMAN38171824.672    ProlarginPRELP_HUMAN44191922.559    FibronectinFINC_HUMAN26329298.138    VersicanCSPG2_HUMAN37313132.333    LactadherinMFGM_HUMAN43131521.223    PerlecanPGBM_HUMAN46920202.922    ClusterinCLUS_HUMAN527714.218    EGF-containing fibulin-like ECM protein 1FBLN3_HUMAN556610.611    PeriostinPOSTN_HUMAN93774.811    Galectin-1LEG1_HUMAN155523.07    TenascinTENA_HUMAN241552.16    VitronectinVTNC_HUMAN54445.65    Link protein 1HPLN1_HUMAN404413.05    Fibulin-1FBLN1_HUMAN77333.83    Laminin subunit α-4LAMA4_HUMAN203331.93    Fibulin-5FBLN5_HUMAN50336.03Other ECM-associated proteins    Apolipoprotein EAPOE_HUMAN36121217.718    Extracellular superoxide dismutaseSODE_HUMAN26101024.013    Apolipoprotein BAPOD_HUMAN21771.510    Apolipoprotein A-IAPOA1_HUMAN305515.05    Galectin-3-binding proteinLG3BP_HUMAN65336.73Collagens    Collagen α-3(VI)CO6A3_HUMAN34457587.4109    Collagen α-2(VI)CO6A2_HUMAN10911112.420    Collagen α-1(XIV)bCOEA1_HUMAN194773.68    Collagen α-1(XVIII)COIA1_HUMAN178661.57a Sum of all spectra identified for each protein in the three biological samples, performed with two technical replicates each.b Proteins found for the first time by proteomics in the vasculature.Identification of Podocan, Sclerostin, and AgrinAmong the identified proteins in the NaCl and guanidine extracts were three recently discovered glycoproteins that have never been detected in the aortic ECM at the protein level. Podocan, an atypical proteoglycan, which has so far been found in the basement membrane of the glomeruli as a product of kidney podocytes (19), was detected in the NaCl extracts. Immunohistological staining confirmed the presence of podocan in the aortic intima (Fig. 3A; the isotype negative control is shown in Fig. 3B). Sclerostin, a potent bone morphogenetic protein antagonist secreted by osteoblasts and osteocytes (20), was identified in the guanidine extract and was localized in the subendothelial layer of the aortic intima (Fig. 3C; the isotype negative control is shown in Fig. 3D). The function of these glycoproteins in the vasculature is to date unknown. Finally, agrin is a large heparan sulfate proteoglycan. It has been found in the basement membrane of various tissues and is known to participate in the organization of the neuromuscular junction (21). Immunohistochemical staining confirmed the presence of agrin on the aortic endothelium (Fig. 3E; the isotype negative control is shown in Fig. 3F), which is also the site for the accumulation of other heparan sulfate proteoglycans, such as perlecan (22). It is therefore possible that agrin participates in the formation of the vascular endothelial barrier. Representative MS/MS spectra for the three glycoproteins are shown in supplemental Fig. 2.Fig. 3.Confirmation of podocan, sclerostin, and agrin in aortic ECM. Immunohistological staining confirmed the presence of podocan in the aortic intima (A). Immunoreactivity appears in brown (magnification, 20×; inset magnification, 60×). Nuclei are counterstained with Mayer's hematoxylin (blue). Sclerostin (C) was localized in the subendothelial layer of the aortic intima. Finally, immunohistochemical staining confirmed the presence of agrin (E). Agrin is localized on the aortic endothelium. The respective isotype negative controls are shown in B, D, and F, respectively. The specificity of the sclerostin and agrin antibodies was confirmed by immunoblotting (data not shown). The aortic specimens were from a 44-year old female (A, B) and a 55-year old male patient (C–F).Evaluation of RePlayFor the wider application of proteomics in clinical research, it is essential to maximize the information obtained from valuable clinical samples. Given the scarcity of human tissue available, we used the RePlay device (Advion), which allowed us to obtain a technical replicate from each clinical specimen in a single LC-MS/MS run. Representative examples of chromatograms using RePlay are shown in Fig. 4, A and B. Correlation analyses of the spectral counts in the three NaCl extracts and the three guanidine extracts are presented in Fig. 4, C and D. The mean spectral counts for the NaCl and the guanidine extracts were 32.7 ± 9.9 (median, 9.5) and 126.5 ± 41 (median, 40), respectively. The R2 values (Pearson correlation coefficient) from all six extracts exceeded 0.94 (p values <0.001), demonstrating good reproducibility in the duplicate analyses (supplemental Fig. 3).Fig. 4.Evaluation of RePlay analysis. Representative chromatograms for a NaCl (A) and a guanidine extract (B) using RePlay are shown. The RePlay device allows the reanalysis of the same sample during a single LC-MS/MS run. The combined correlation of the spectral counts obtained by RePlay in the three NaCl extracts (C) and in the three guanidine extracts (D) demonstrates the good correlation between the technical replicates (Correlation for each of the six extrats can be found in supplemental Fig. 3.). No RePlay analysis was performed for the SDS extracts.Because of their scarcity, MMPs are rarely detected by tissue proteomics (23). Our subfractionation, however, enabled us to obtain spectral evidence for the presence of MMP-2 in one of the NaCl extracts (aortic sample 1). Consistent with our proteomics data, gelatin zymography confirmed the presence of MMP-2 in this sample (Fig. 5A). A representative MS/MS spectrum for MMP-2 is shown in supplemental Fig. 4A. Importantly, the non-denaturing composition of the NaCl buffer allowed the detection of MMP-2 directly without further processing of the extracts.Fig. 5.Validation of proteomics data. Gelatin zymography confirmed the presence of MMP-2 in one of the three NaCl extracts (A). Periostin (B, upper panel) and asporin (B, bottom panel) were validated by immunoblotting in the guanidine extracts (on the same blotting membrane; first, asporin; and second, reprobing for periostin). The total protein loading control for the extracts can be seen in Fig. 1C, bottom panel. For periostin and asporin, the relative band intensity of the immunoblots was measured, and the values were correlated with the spectral counts of the original and the RePlay analysis (n = 2, mean ± S.D. Error bars denote standard deviation.) (C and D).In the guanidine extracts, we performed Western immunoblotting for the ECM protein periostin, which was unambiguously identified (Fig. 5B, upper panel). Periostin mRNA was recently shown to be expressed in rat carotid arteries following balloon injury (24). We also confirmed the presence of the scarce glycoprotein asporin in the vascular ECM (Fig. 5B, bottom panel). Its biological role is to date unclear, but it was recently shown to inhibit TGFβ-induced chondrogenesis in articular cartilage (25) and is a potential regulator of collagen mineralization (26). Finally, the Western blots for periostin and asporin were analyzed by densitometry, and the densitometry values were correlated with the spectral counts obtained from the RePlay analysis (n = 2, mean ± S.D.) (Fig. 5, C and D). Although the technical replicates obtained using the RePlay device were accurate, there is expected variability in the different clinical samples. Representative MS/MS spectra for periostin and asporin are shown in supplemental Fig. 4, B and C.ProteolysisFinally, we investigated whether the NaCl extract represents a subproteome enriched in degradation products. Currently, our understanding of ECM proteolysis is based on in vitro experiments, which do not necessarily relate to in vivo processes. In our methodology, the original molecular mass distribution of the identified proteins is preserved by the gel LC-MS/MS approach. Thus, our method can be important for advancing our understanding of protein degradation in vivo. For instance, we noted that in a fourth aortic specimen, which was classified as normal upon aortotomy, the coefficient of variation for the number of fibronectin spectra in the NaCl extract exceeded 100%, implying a significant difference in the dispersion of the data among the analyzed samples (p ANOVA <0.05). On the other hand, the coefficient of variation for samples 1–3 is not significant (58%; p ANOVA >0.05). We also noted that there was an increase in fibronectin sequence coverage (Fig. 6, A and B) and total fibronectin spectra (∼4-fold) in the NaCl extract of the fourth specimen. Closer inspection of the molecular mass distribution of fibronectin spectra indicated the breakdown of fibronectin in this particular sample with more than 80% of fibronectin spectra identified in gel bands below 150 kDa. Western blotting using an antibody raised against full-length human fibronectin confirmed extensive laddering of the glycoprotein in both the NaCl and guanidine extracts (Fig. 6, C and D). Fibronectin degradation products have been implicated in the induction of proinflammatory responses in the vasculature and other tissues (1). Interestingly, mass spectrometry provided spectral evidence for MMP-9 only in this particular sample (Fig. 6E), and the presence of MMP-9 was confirmed by gelatin zymography (Fig. 6C, lower panel). Fibronectin is a known substrate for MMP-9 (27), but obviously, we cannot exclude that other proteinases contribute to the breakdown of fibronectin in this specimen in vivo. The example, however, demonstrates the potential of our methodology to analyze proteolytic activity based on the presence of proteolytic enzymes and corresponding degradation products in tissues. Importantly, this key information on protein breakdown within clinical samples that were considered normal macroscopically could be lost by using a conventional shotgun proteomics approach.Fig. 6.Detection of proteolysis. Percent sequence coverage for fibronectin in the NaCl (A) and guanidine extracts (B) is shown. “R” denotes RePlay analysis. Note the increase of fibronectin products in the NaCl extracts of the fourth aortic specimen. Western immunoblotting of fibronectin in the NaCl (C) and guanidine (D) extracts confirmed the laddering of fibronectin. The product ion spectrum of the doubly charged tryptic peptide SLGPALLLLQK was identified as MMP-9 (E) (peptide probability, 95%; SEQUEST XCorr score, 2.67; SEQUEST ΔCn score, 0.40; observed m/z, 576.87; actual peptide mass, 1,151.73 Da; peptide charge, 2; delta amu, 0.0049; delta ppm, 4.3; peptide start index, 66; peptide stop index, 76). MMP-9 was only detected by LC-MS/MS in the NaCl extract of the fourth aortic specimen. Gelatin zymography confirmed higher levels of MMP-9 in this sample (C, bottom panel).

Biochemical Extraction of Proteins in Extracellular EnvironmentOur extraction methodology was developed on human aortic samples obtained during cardiac surgery for aortic valve replacement of degenerative aortic valve stenosis. Fig. 1A is a schematic summary of the methodology used in this study. The samples (n = 3) were macroscopically classified as normal during surgery, and upon histological examination, they showed no pathological signs. Extraction of the proteins of the vascular extracellular environment and reduction of the contamination with cellular proteins were achieved by a three-step extraction procedure. First, the tissue samples were treated with 0.5 m NaCl. The salt ions induce displacement of polyionic interactions between proteins, thus enabling the extraction of loosely bound extracellular proteins, including newly synthesized ECM proteins and degradation products (12), providing an overview of tissue turnover. The tissue was then decellularized using SDS. In contrast to other decellularization reagents, such as non-ionic detergents (Triton X-100) (13), 0.1% SDS has been previously shown to be very effective in solubilizing cytoplasmic and nuclear membranes, thereby removing cellular components from tissues (14). However, the ionic detergent above its critical micelle concentration can also cause protein denaturation and disrupt native ECM proteins (15).Fig. 1.Identification of extracellular proteins in human aortas. A, schematic summary. Biochemical subfractionation was used for the enrichment of extracellular space proteins before proteomic analysis. B, validation of decellularization by hematoxylin/eosin staining. Representative images of a hematoxylin/eosin-stained human aorta before (left panel) and after decellularization with 0.08% SDS (right panel) are shown. L, lumen; I, intima; M, media. Arrows depict the intima/media border. Note the loss of nuclear (blue; hematoxylin) and cytoplasmic contents (pink; eosin), whereas other eosinophilic structures, such as collagen fibers, are preserved. C, immunoblotting and silver staining. The ECM glycoprotein AEBP1 is found in the NaCl and guanidine (Guan) extracts, which are enriched in proteins of the extracellular space (upper panel). Cytoplasmic β-actin is predominantly found in the SDS extracts (middle panel), demonstrating successful depletion of cellular contents. Silver staining of proteins from all extracts separated by one-dimensional SDS-PAGE shows the pattern of proteins extracted in the three different extraction steps (bottom panel).To substantially remove the cellular material while preserving the ECM and its associated proteins as much as possible, we used 2.8 mm (0.08%) SDS in distilled water. This concentration is well below the critical micelle concentration of SDS in distilled water, which is 8 mm (16). In addition, at this concentration, SDS is effective in solubilizing cell membrane lipid bilayers (17). Treatment with 0.08% SDS was sufficient to remove most cellular components as shown by the marked reduction of nuclear staining (hematoxylin; Fig. 1B, blue). The cytoplasmic staining (eosin; pink) was also reduced, but other eosinophilic structures, such as collagen and elastin, were preserved in the tissue. Finally, the specimens were subjected to denaturing extraction in a 4 m guanidine HCl buffer. Guanidine HCl extractions were developed more than 40 years ago as a very effective way to solubilize most of the strongly bound ECM components, including large aggregating proteoglycans (versican, aggrecan, etc.), small proteoglycans (decorin, biglycan, etc.), cell attachment matrix glycoproteins (such as type VI collagen, fibronectins, and laminins), and basement membrane components (perlecan, type IV collagen, etc.). Guanidine induces disaggregation of ECM components by destabilizing the ionic, disulfide-dependent protein conformations (18). Following dissociative extraction, only the insoluble, heavily cross-linked interstitial matrix, namely type I and III collagens and elastin, remains in the tissue. Western blotting confirmed that β-actin immunoreactivity is predominantly found in the SDS extracts (Fig 1C, middle panel), which also had the highest protein concentration per milliliter (supplemental Fig. 1). On the other hand, the extracellular AEBP1 isoform, an ECM-associated glycoprotein, was detected in the NaCl and guanidine extracts by Western blotting (Fig 1C, upper panel), indicating that these extracts are enriched in proteins of the extracellular space. Silver staining revealed a distinct but reproducible protein pattern in the three different extraction steps (Fig. 1C, bottom panel).The different extracts were subjected to enzymatic deglycosylation to remove chondroitin, keratan, and heparan sulfate glycosaminoglycan (GAG) chains from the matrix glycoproteins. Deglycosylation is a critical step in our methodology. First, GAGs increase the molecular weight of glycoproteins and affect their separation and resolution by one-dimensional electrophoresis. Second, the presence of GAGs greatly reduces the chance for the identification of proteoglycans by mass spectrometry.Proteomics Analysis of ExtractsAfter separation by SDS-PAGE, protein bands were excised from silver-stained gels, subjected to in-gel tryptic digestion, and analyzed by LC-MS/MS. The peptides were injected into a high mass accuracy ion trap (LTQ-Orbitrap XL, Thermo Fisher Scientific) using RePlay (Advion). This injection system splits the gradient from the analytical column and allows the reanalysis of the same sample in a single LC-MS/MS run (8). 272 and 321 proteins were identified in the NaCl and guanidine extracts, respectively. All the peptides identified by LC-MS/MS in the NaCl and guanidine extracts are shown in supplemental Tables 1 and 2. According to their gene ontology annotation (European Molecular Biology Laboratory-European Bioinformatics Institute database, www.uniprot.org), a quarter of these proteins were located in the extracellular space: among the 61 extracellular proteins in the NaCl extracts (Table I) and the 84 extracellular proteins in the guanidine extracts (Table II), proteins highlighted with Footnote b were, to the best of our knowledge, identified for the first time in the vasculature by proteomics. Lipoproteins were included (even though they are plasma-derived) because of their intimate relationship with proteoglycans and their relevance in cardiovascular pathology. The classification of the identified extracellular space proteins in the NaCl and the guanidine extracts is shown in Fig. 2, A and B. As expected, the denaturing guanidine extracts were particularly enriched in ECM proteoglycans and glycoproteins (50%), whereas the mildly dissociative NaCl extracts were enriched in proteases and protease inhibitors (21%), indicating the importance of this subproteome for studying proteolytic processes in tissues.Table IExtracellular space proteins identified by proteomics in three NaCl extractsCOMP, cartilage oligomeric matrix protein; IGF, insulin-like growth factor.Protein nameSwiss-Prot accession nameMolecular massUnique peptides, nUnique spectra, nCoverageTotal spectra,ankDa%Proteoglycans and glycoproteins    LumicanLUM_HUMAN38404020.0249    Tenascin-XTENX_HUMAN46466668.1156    Fibulin-3FBLN3_HUMAN55383822.0155    Adipocyte enhancer-binding protein 1bAEBP1_HUMAN131373911.0153    FibronectinFINC_HUMAN26334347.3111    PerlecanPGBM_HUMAN46916162.550    MimecanMIME_HUMAN34131312.049    VersicanCSPG2_HUMAN37316163.746    TetranectinTETN_HUMAN23202029.039    FibromodulinFMOD_HUMAN43667.426    Fibulin-1FBLN1_HUMAN7714149.125    Latent TGFβ-binding protein 2bLTBP2_HUMAN19515155.223    Thrombospondin-1TSP1_HUMAN12911115.118    PodocanbPODN_HUMAN6910106.917    Fibrillin-1FBN1_HUMAN31210102.116    Fibulin-5FBLN5_HUMAN50667.812    Galectin-1LEG1_HUMAN157726.09    DecorinPGS2_HUMAN403310.08    Latent TGFβ-binding protein 4bLTBP4_HUMAN173331.67    ClusterinCLUS_HUMAN52333.65    COMPCOMP_HUMAN83226.33    TenascinTENA_HUMAN241220.952    DermatopontinDERM_HUMAN24229.52Other ECM-associated proteins    Extracellular superoxide dismutaseSODE_HUMAN26222746.0209    Pigment epithelium-derived factorPEDF_HUMAN46323216.082    Serum amyloid P-componentSAMP_HUMAN25313126.062    IGF-binding protein 7bIBP7_HUMAN29151521.047    Apolipoprotein A-IAPOA1_HUMAN316625.019    Apolipoprotein A-IVAPOA4_HUMAN457717.012    Apolipoprotein HbAPOH_HUMAN388814.011    Hepatoma-derived growth factorbHDGF_HUMAN27667.911    Secreted frizzled-related protein 1bSFRP1_HUMAN35559.29    CeruloplasminCERU_HUMAN122442.39    Secreted frizzled-related protein 3bSFRP3_HUMAN36449.28    Apolipoprotein DAPOD_HUMAN216620.08    Apolipoprotein EAPOE_HUMAN36557.67    LIM and cysteine-rich domains protein 1bLMCD1_HUMAN41667.77    Spondin-1bSPON1_HUMAN91332.76    Target of Nesh-SH3bTARSH_HUMAN119222.66Proteases and protease inhibitors    Tryptase β-1bTRYB1_HUMAN319922.022    Procollagen C-endopeptidase enhancer 1PCOC1_HUMAN48887.818    Cathepsin GbCATG_HUMAN29776.716    CalreticulinCALR_HUMAN489911.013    Cathepsin ZbCATZ_HUMAN34667.310    Plasma glutamate carboxypeptidasebPGCP_HUMAN52554.28    Cathepsin DCATD_HUMAN45557.07    Leukocyte elastase inhibitorILEU_HUMAN43559.07    Proactivator polypeptidebSAP_HUMAN58555.37    MMP-2bMMP2_HUMAN74223.34    KallistatinbKAIN_HUMAN49224.73    Carboxypeptidase-like protein X2bCPXM2_HUMAN86224.62kDa%Collagens    Collagen α-1(XIV)bCOEA1_HUMAN194737314.0233    Collagen α-1(XII)COCA1_HUMAN33320202.841    Collagen α-3(VI)CO6A3_HUMAN34416163.729    Collagen α-1(XVIII)COIA1_HUMAN17811113.722    Collagen α-2(VI)CO6A2_HUMAN109443.06    Collagen α-1(XV)COFA1_HUMAN142332.04    Collagen α-1(I)CO1A1_HUMAN129331.54a Sum of all spectra identified for each protein in the three biological samples, performed with two technical replicates each.b Proteins found for the first time by proteomics in the vasculature.Table IIExtracellular space proteins identified by proteomics in three guanidine extractsIGF, insulin-like growth factor.Protein nameSwiss-Prot accession nameMolecular massUnique peptides, nUnique spectra, nCoverageTotal spectra,ankDa%Proteoglycans and glycoproteins    PerlecanPGBM_HUMAN46936748119.01596    FibronectinFINC_HUMAN26318221820.01028    VersicanCSPG2_HUMAN37376894.4681    AggrecanPGCA_HUMAN25070826.3451    BiglycanPGS1_HUMAN42566236.0352    LactadherinMFGM_HUMAN43646748.0309    ProlarginPRELP_HUMAN44557837.0283    PeriostinPOSTN_HUMAN9310011729.0257    LumicanLUM_HUMAN38535830.0204    DecorinPGS2_HUMAN40435129.0173    ClusterinCLUS_HUMAN52444823.0173    Tenascin-XTENX_HUMAN46479794.5140    Laminin subunit γ-1LAMC1_HUMAN178404418.0129    Link protein 1HPLN1_HUMAN40262933.0117    Laminin subunit β-2LAMB2_HUMAN196343414.0104    VitronectinVTNC_HUMAN54252714.0102    Nidogen-1NID1_HUMAN136566415.0101    Adipocyte enhancer-binding protein 1bAEBP1_HUMAN13134407.694    Laminin subunit α-5LAMA5_HUMAN40039394.691    TenascinTENA_HUMAN24125255.652    MimecanMIME_HUMAN34212115.048    EGF-containing fibulin-like ECM protein 1FBLN3_HUMAN55222415.048    Fibulin-5FBLN5_HUMAN50171714.040    Galectin-1LEG1_HUMAN15172053.040    TGFβ-induced protein ig-h3BGH3_HUMAN75232312.035    EMILIN-1EMIL1_HUMAN107883.030    Fibulin-1FBLN1_HUMAN77171712.029    Latent TGFβ-binding protein 2bLTBP2_HUMAN19513132.722    FibromodulinFMOD_HUMAN438118.222    Microfibril-associated glycoprotein 4MFAP4_HUMAN298812.021    DermatopontinDERM_HUMAN248912.021    Fibrillin-1FBN1_HUMAN31215152.421    Latent TGFβ-binding protein 4bLTBP4_HUMAN17312124.215    Nidogen-2NID2_HUMAN151883.511    AgrinbAGRIN_HUMAN215772.311    AsporinASPN_HUMAN438811.011    Cartilage oligomeric matrix proteinCOMP_HUMAN83443.67    Thrombospondin-1TSP1_HUMAN129331.87    Matrix Gla proteinMGP_HUMAN124413.06    Galectin-3LEG3_HUMAN26226.04    SclerostinbSOST_HUMAN24225.63    Matrilin-2bMATN2_HUMAN107222.33Other ECM-associated proteins    Serum amyloid P-componentSAMP_HUMAN25323333.0169    Tubulointerstitial nephritis antigen-likebTINAL_HUMAN52465521.0131    Extracellular superoxide dismutaseSODE_HUMAN26283525.0102    Target of Nesh-SH3bTARSH_HUMAN11930338.689    IGF-binding protein 7bIBP7_HUMAN29303128.059    Cysteine-rich protein 2CRIP2_HUMAN22202125.057    TGFβ-1-induced transcript 1 proteinbTGFI1_HUMAN50272745    LIM and cysteine-rich domains protein 1bLMCD1_HUMAN41252534.039    Apolipoprotein HbAPOH_HUMAN3810129.032    Galectin-3-binding proteinLG3BP_HUMAN65121212.023    RPE-spondinbRPESP_HUMAN30121222.019kDa%    Carboxypeptidase-like protein X2bCPXM2_HUMAN869117.316    Latent TGFβ-binding protein, isoform 1SbLTB1S_HUMAN153892.713    Pigment epithelium derived factorPEDF_HUMAN46558.68    Secreted frizzled-related protein 1bSFRP1_HUMAN35338.96    ProperdinbPROP_HUMAN51345.15    Spondin-1bSPON1_HUMAN91333.04    Neutrophil defensin 1bDEF1_HUMAN10229.64Proteases and protease inhibitors    Serine protease HTRA1HTRA1_HUMAN51222213.068    ChymasebCMA1_HUMAN27121511.023    Lysyl oxidase homolog 1bLOXL1_HUMAN6312127.323    Metalloproteinase inhibitor 3TIMP3_HUMAN249917.012    Tryptase β-1bTRYB1_HUMAN31679.811    Procollagen C-endopeptidase enhancer 1PCOC1_HUMAN48776.710    Cathepsin GbCATG_HUMAN297715.08    Cathepsin DCATD_HUMAN45442.77    AntileukoproteinasebSLPI_HUMAN14339.16    MyeloblastinbPRTN3_HUMAN28224.74Collagens    Collagen α-3(VI)CO6A3_HUMAN34416418620.0898    Collagen α-1(XIV)bCOEA1_HUMAN19413615020.0529    Collagen α-1(XVIII)COIA1_HUMAN17845485.9374    Collagen α-1(VI)CO6A1_HUMAN109507116.0282    Collagen α-2(VI)CO6A2_HUMAN109313119.0155    Collagen α-2(IV)CO4A2_HUMAN16826313.9139    Collagen α-1(XV)COFA1_HUMAN14232347.5100    Collagen α-1(XII)COCA1_HUMAN33342424.090    Collagen α-2(I)CO1A2_HUMAN12912153.570    Collagen α-1(I)CO1A1_HUMAN13914155.951    Collagen α-1(VIII)CO8A1_HUMAN73551.333    Collagen α-1(V)CO5A1_HUMAN184660.623    Collagen α-2(VIII)CO8A2_HUMAN67443.39    Collagen α-1(IV)CO4A1_HUMAN161551.88a Sum of all spectra identified for each protein in the three biological samples, performed with two technical replicates each.b Proteins found for the first time by proteomics in the vasculature.Fig. 2.Protein categories. The classification and numerical distribution of the identified extracellular proteins in the NaCl (A), guanidine (B), and SDS (C) extracts are shown. The size of the doughnut charts is representative of the number of extracellular space proteins identified in the different extracts.To identify proteins of the extracellular space extracted by 0.08% SDS, we performed proteomics analysis of the SDS extracts. Because of the increased complexity of these samples, tryptic peptides were injected into a dual pressure, linear ion trap (LTQ Velos, Thermo Fisher Scientific) with faster scan speed. As expected, the majority of the identified proteins were cellular (data not shown), and only 28 extracellular space proteins were identified (Table III). The classification of the identified extracellular space proteins in the SDS extracts is shown in Fig. 2C. Interestingly, apart from type XIV collagen, all other extracellular space proteins have been previously identified by proteomics in the vasculature. Also, of note is the absence of proteases and protease inhibitors in these extracts either because they are obscured by abundant cellular proteins or because those that remain in the tissue following 0.5 m NaCl extraction are tightly bound to the ECM and require dissociative conditions to be extracted (i.e. guanidine). Peptides of extracellular space proteins identified by LC-MS/MS in the SDS extracts are shown in supplemental Table 3.Table IIIExtracellular space proteins identified by proteomics in three SDS extractsProtein nameSwiss-Prot accession nameMolecular massUnique peptides, nUnique spectra, nCoverageTotal spectra,ankDa%Proteoglycans and glycoproteins    BiglycanPGS1_HUMAN42242832.6141    DecorinPGS2_HUMAN40242433.185    MimecanMIME_HUMAN34212215.076    LumicanLUM_HUMAN38171824.672    ProlarginPRELP_HUMAN44191922.559    FibronectinFINC_HUMAN26329298.138    VersicanCSPG2_HUMAN37313132.333    LactadherinMFGM_HUMAN43131521.223    PerlecanPGBM_HUMAN46920202.922    ClusterinCLUS_HUMAN527714.218    EGF-containing fibulin-like ECM protein 1FBLN3_HUMAN556610.611    PeriostinPOSTN_HUMAN93774.811    Galectin-1LEG1_HUMAN155523.07    TenascinTENA_HUMAN241552.16    VitronectinVTNC_HUMAN54445.65    Link protein 1HPLN1_HUMAN404413.05    Fibulin-1FBLN1_HUMAN77333.83    Laminin subunit α-4LAMA4_HUMAN203331.93    Fibulin-5FBLN5_HUMAN50336.03Other ECM-associated proteins    Apolipoprotein EAPOE_HUMAN36121217.718    Extracellular superoxide dismutaseSODE_HUMAN26101024.013    Apolipoprotein BAPOD_HUMAN21771.510    Apolipoprotein A-IAPOA1_HUMAN305515.05    Galectin-3-binding proteinLG3BP_HUMAN65336.73Collagens    Collagen α-3(VI)CO6A3_HUMAN34457587.4109    Collagen α-2(VI)CO6A2_HUMAN10911112.420    Collagen α-1(XIV)bCOEA1_HUMAN194773.68    Collagen α-1(XVIII)COIA1_HUMAN178661.57a Sum of all spectra identified for each protein in the three biological samples, performed with two technical replicates each.b Proteins found for the first time by proteomics in the vasculature.Identification of Podocan, Sclerostin, and AgrinAmong the identified proteins in the NaCl and guanidine extracts were three recently discovered glycoproteins that have never been detected in the aortic ECM at the protein level. Podocan, an atypical proteoglycan, which has so far been found in the basement membrane of the glomeruli as a product of kidney podocytes (19), was detected in the NaCl extracts. Immunohistological staining confirmed the presence of podocan in the aortic intima (Fig. 3A; the isotype negative control is shown in Fig. 3B). Sclerostin, a potent bone morphogenetic protein antagonist secreted by osteoblasts and osteocytes (20), was identified in the guanidine extract and was localized in the subendothelial layer of the aortic intima (Fig. 3C; the isotype negative control is shown in Fig. 3D). The function of these glycoproteins in the vasculature is to date unknown. Finally, agrin is a large heparan sulfate proteoglycan. It has been found in the basement membrane of various tissues and is known to participate in the organization of the neuromuscular junction (21). Immunohistochemical staining confirmed the presence of agrin on the aortic endothelium (Fig. 3E; the isotype negative control is shown in Fig. 3F), which is also the site for the accumulation of other heparan sulfate proteoglycans, such as perlecan (22). It is therefore possible that agrin participates in the formation of the vascular endothelial barrier. Representative MS/MS spectra for the three glycoproteins are shown in supplemental Fig. 2.Fig. 3.Confirmation of podocan, sclerostin, and agrin in aortic ECM. Immunohistological staining confirmed the presence of podocan in the aortic intima (A). Immunoreactivity appears in brown (magnification, 20×; inset magnification, 60×). Nuclei are counterstained with Mayer's hematoxylin (blue). Sclerostin (C) was localized in the subendothelial layer of the aortic intima. Finally, immunohistochemical staining confirmed the presence of agrin (E). Agrin is localized on the aortic endothelium. The respective isotype negative controls are shown in B, D, and F, respectively. The specificity of the sclerostin and agrin antibodies was confirmed by immunoblotting (data not shown). The aortic specimens were from a 44-year old female (A, B) and a 55-year old male patient (C–F).Evaluation of RePlayFor the wider application of proteomics in clinical research, it is essential to maximize the information obtained from valuable clinical samples. Given the scarcity of human tissue available, we used the RePlay device (Advion), which allowed us to obtain a technical replicate from each clinical specimen in a single LC-MS/MS run. Representative examples of chromatograms using RePlay are shown in Fig. 4, A and B. Correlation analyses of the spectral counts in the three NaCl extracts and the three guanidine extracts are presented in Fig. 4, C and D. The mean spectral counts for the NaCl and the guanidine extracts were 32.7 ± 9.9 (median, 9.5) and 126.5 ± 41 (median, 40), respectively. The R2 values (Pearson correlation coefficient) from all six extracts exceeded 0.94 (p values <0.001), demonstrating good reproducibility in the duplicate analyses (supplemental Fig. 3).Fig. 4.Evaluation of RePlay analysis. Representative chromatograms for a NaCl (A) and a guanidine extract (B) using RePlay are shown. The RePlay device allows the reanalysis of the same sample during a single LC-MS/MS run. The combined correlation of the spectral counts obtained by RePlay in the three NaCl extracts (C) and in the three guanidine extracts (D) demonstrates the good correlation between the technical replicates (Correlation for each of the six extrats can be found in supplemental Fig. 3.). No RePlay analysis was performed for the SDS extracts.Because of their scarcity, MMPs are rarely detected by tissue proteomics (23). Our subfractionation, however, enabled us to obtain spectral evidence for the presence of MMP-2 in one of the NaCl extracts (aortic sample 1). Consistent with our proteomics data, gelatin zymography confirmed the presence of MMP-2 in this sample (Fig. 5A). A representative MS/MS spectrum for MMP-2 is shown in supplemental Fig. 4A. Importantly, the non-denaturing composition of the NaCl buffer allowed the detection of MMP-2 directly without further processing of the extracts.Fig. 5.Validation of proteomics data. Gelatin zymography confirmed the presence of MMP-2 in one of the three NaCl extracts (A). Periostin (B, upper panel) and asporin (B, bottom panel) were validated by immunoblotting in the guanidine extracts (on the same blotting membrane; first, asporin; and second, reprobing for periostin). The total protein loading control for the extracts can be seen in Fig. 1C, bottom panel. For periostin and asporin, the relative band intensity of the immunoblots was measured, and the values were correlated with the spectral counts of the original and the RePlay analysis (n = 2, mean ± S.D. Error bars denote standard deviation.) (C and D).In the guanidine extracts, we performed Western immunoblotting for the ECM protein periostin, which was unambiguously identified (Fig. 5B, upper panel). Periostin mRNA was recently shown to be expressed in rat carotid arteries following balloon injury (24). We also confirmed the presence of the scarce glycoprotein asporin in the vascular ECM (Fig. 5B, bottom panel). Its biological role is to date unclear, but it was recently shown to inhibit TGFβ-induced chondrogenesis in articular cartilage (25) and is a potential regulator of collagen mineralization (26). Finally, the Western blots for periostin and asporin were analyzed by densitometry, and the densitometry values were correlated with the spectral counts obtained from the RePlay analysis (n = 2, mean ± S.D.) (Fig. 5, C and D). Although the technical replicates obtained using the RePlay device were accurate, there is expected variability in the different clinical samples. Representative MS/MS spectra for periostin and asporin are shown in supplemental Fig. 4, B and C.ProteolysisFinally, we investigated whether the NaCl extract represents a subproteome enriched in degradation products. Currently, our understanding of ECM proteolysis is based on in vitro experiments, which do not necessarily relate to in vivo processes. In our methodology, the original molecular mass distribution of the identified proteins is preserved by the gel LC-MS/MS approach. Thus, our method can be important for advancing our understanding of protein degradation in vivo. For instance, we noted that in a fourth aortic specimen, which was classified as normal upon aortotomy, the coefficient of variation for the number of fibronectin spectra in the NaCl extract exceeded 100%, implying a significant difference in the dispersion of the data among the analyzed samples (p ANOVA <0.05). On the other hand, the coefficient of variation for samples 1–3 is not significant (58%; p ANOVA >0.05). We also noted that there was an increase in fibronectin sequence coverage (Fig. 6, A and B) and total fibronectin spectra (∼4-fold) in the NaCl extract of the fourth specimen. Closer inspection of the molecular mass distribution of fibronectin spectra indicated the breakdown of fibronectin in this particular sample with more than 80% of fibronectin spectra identified in gel bands below 150 kDa. Western blotting using an antibody raised against full-length human fibronectin confirmed extensive laddering of the glycoprotein in both the NaCl and guanidine extracts (Fig. 6, C and D). Fibronectin degradation products have been implicated in the induction of proinflammatory responses in the vasculature and other tissues (1). Interestingly, mass spectrometry provided spectral evidence for MMP-9 only in this particular sample (Fig. 6E), and the presence of MMP-9 was confirmed by gelatin zymography (Fig. 6C, lower panel). Fibronectin is a known substrate for MMP-9 (27), but obviously, we cannot exclude that other proteinases contribute to the breakdown of fibronectin in this specimen in vivo. The example, however, demonstrates the potential of our methodology to analyze proteolytic activity based on the presence of proteolytic enzymes and corresponding degradation products in tissues. Importantly, this key information on protein breakdown within clinical samples that were considered normal macroscopically could be lost by using a conventional shotgun proteomics approach.Fig. 6.Detection of proteolysis. Percent sequence coverage for fibronectin in the NaCl (A) and guanidine extracts (B) is shown. “R” denotes RePlay analysis. Note the increase of fibronectin products in the NaCl extracts of the fourth aortic specimen. Western immunoblotting of fibronectin in the NaCl (C) and guanidine (D) extracts confirmed the laddering of fibronectin. The product ion spectrum of the doubly charged tryptic peptide SLGPALLLLQK was identified as MMP-9 (E) (peptide probability, 95%; SEQUEST XCorr score, 2.67; SEQUEST ΔCn score, 0.40; observed m/z, 576.87; actual peptide mass, 1,151.73 Da; peptide charge, 2; delta amu, 0.0049; delta ppm, 4.3; peptide start index, 66; peptide stop index, 76). MMP-9 was only detected by LC-MS/MS in the NaCl extract of the fourth aortic specimen. Gelatin zymography confirmed higher levels of MMP-9 in this sample (C, bottom panel).

Biochemical Extraction of Proteins in Extracellular EnvironmentOur extraction methodology was developed on human aortic samples obtained during cardiac surgery for aortic valve replacement of degenerative aortic valve stenosis. Fig. 1A is a schematic summary of the methodology used in this study. The samples (n = 3) were macroscopically classified as normal during surgery, and upon histological examination, they showed no pathological signs. Extraction of the proteins of the vascular extracellular environment and reduction of the contamination with cellular proteins were achieved by a three-step extraction procedure. First, the tissue samples were treated with 0.5 m NaCl. The salt ions induce displacement of polyionic interactions between proteins, thus enabling the extraction of loosely bound extracellular proteins, including newly synthesized ECM proteins and degradation products (12), providing an overview of tissue turnover. The tissue was then decellularized using SDS. In contrast to other decellularization reagents, such as non-ionic detergents (Triton X-100) (13), 0.1% SDS has been previously shown to be very effective in solubilizing cytoplasmic and nuclear membranes, thereby removing cellular components from tissues (14). However, the ionic detergent above its critical micelle concentration can also cause protein denaturation and disrupt native ECM proteins (15).Fig. 1.Identification of extracellular proteins in human aortas. A, schematic summary. Biochemical subfractionation was used for the enrichment of extracellular space proteins before proteomic analysis. B, validation of decellularization by hematoxylin/eosin staining. Representative images of a hematoxylin/eosin-stained human aorta before (left panel) and after decellularization with 0.08% SDS (right panel) are shown. L, lumen; I, intima; M, media. Arrows depict the intima/media border. Note the loss of nuclear (blue; hematoxylin) and cytoplasmic contents (pink; eosin), whereas other eosinophilic structures, such as collagen fibers, are preserved. C, immunoblotting and silver staining. The ECM glycoprotein AEBP1 is found in the NaCl and guanidine (Guan) extracts, which are enriched in proteins of the extracellular space (upper panel). Cytoplasmic β-actin is predominantly found in the SDS extracts (middle panel), demonstrating successful depletion of cellular contents. Silver staining of proteins from all extracts separated by one-dimensional SDS-PAGE shows the pattern of proteins extracted in the three different extraction steps (bottom panel).To substantially remove the cellular material while preserving the ECM and its associated proteins as much as possible, we used 2.8 mm (0.08%) SDS in distilled water. This concentration is well below the critical micelle concentration of SDS in distilled water, which is 8 mm (16). In addition, at this concentration, SDS is effective in solubilizing cell membrane lipid bilayers (17). Treatment with 0.08% SDS was sufficient to remove most cellular components as shown by the marked reduction of nuclear staining (hematoxylin; Fig. 1B, blue). The cytoplasmic staining (eosin; pink) was also reduced, but other eosinophilic structures, such as collagen and elastin, were preserved in the tissue. Finally, the specimens were subjected to denaturing extraction in a 4 m guanidine HCl buffer. Guanidine HCl extractions were developed more than 40 years ago as a very effective way to solubilize most of the strongly bound ECM components, including large aggregating proteoglycans (versican, aggrecan, etc.), small proteoglycans (decorin, biglycan, etc.), cell attachment matrix glycoproteins (such as type VI collagen, fibronectins, and laminins), and basement membrane components (perlecan, type IV collagen, etc.). Guanidine induces disaggregation of ECM components by destabilizing the ionic, disulfide-dependent protein conformations (18). Following dissociative extraction, only the insoluble, heavily cross-linked interstitial matrix, namely type I and III collagens and elastin, remains in the tissue. Western blotting confirmed that β-actin immunoreactivity is predominantly found in the SDS extracts (Fig 1C, middle panel), which also had the highest protein concentration per milliliter (supplemental Fig. 1). On the other hand, the extracellular AEBP1 isoform, an ECM-associated glycoprotein, was detected in the NaCl and guanidine extracts by Western blotting (Fig 1C, upper panel), indicating that these extracts are enriched in proteins of the extracellular space. Silver staining revealed a distinct but reproducible protein pattern in the three different extraction steps (Fig. 1C, bottom panel).The different extracts were subjected to enzymatic deglycosylation to remove chondroitin, keratan, and heparan sulfate glycosaminoglycan (GAG) chains from the matrix glycoproteins. Deglycosylation is a critical step in our methodology. First, GAGs increase the molecular weight of glycoproteins and affect their separation and resolution by one-dimensional electrophoresis. Second, the presence of GAGs greatly reduces the chance for the identification of proteoglycans by mass spectrometry.Proteomics Analysis of ExtractsAfter separation by SDS-PAGE, protein bands were excised from silver-stained gels, subjected to in-gel tryptic digestion, and analyzed by LC-MS/MS. The peptides were injected into a high mass accuracy ion trap (LTQ-Orbitrap XL, Thermo Fisher Scientific) using RePlay (Advion). This injection system splits the gradient from the analytical column and allows the reanalysis of the same sample in a single LC-MS/MS run (8). 272 and 321 proteins were identified in the NaCl and guanidine extracts, respectively. All the peptides identified by LC-MS/MS in the NaCl and guanidine extracts are shown in supplemental Tables 1 and 2. According to their gene ontology annotation (European Molecular Biology Laboratory-European Bioinformatics Institute database, www.uniprot.org), a quarter of these proteins were located in the extracellular space: among the 61 extracellular proteins in the NaCl extracts (Table I) and the 84 extracellular proteins in the guanidine extracts (Table II), proteins highlighted with Footnote b were, to the best of our knowledge, identified for the first time in the vasculature by proteomics. Lipoproteins were included (even though they are plasma-derived) because of their intimate relationship with proteoglycans and their relevance in cardiovascular pathology. The classification of the identified extracellular space proteins in the NaCl and the guanidine extracts is shown in Fig. 2, A and B. As expected, the denaturing guanidine extracts were particularly enriched in ECM proteoglycans and glycoproteins (50%), whereas the mildly dissociative NaCl extracts were enriched in proteases and protease inhibitors (21%), indicating the importance of this subproteome for studying proteolytic processes in tissues.Table IExtracellular space proteins identified by proteomics in three NaCl extractsCOMP, cartilage oligomeric matrix protein; IGF, insulin-like growth factor.Protein nameSwiss-Prot accession nameMolecular massUnique peptides, nUnique spectra, nCoverageTotal spectra,ankDa%Proteoglycans and glycoproteins    LumicanLUM_HUMAN38404020.0249    Tenascin-XTENX_HUMAN46466668.1156    Fibulin-3FBLN3_HUMAN55383822.0155    Adipocyte enhancer-binding protein 1bAEBP1_HUMAN131373911.0153    FibronectinFINC_HUMAN26334347.3111    PerlecanPGBM_HUMAN46916162.550    MimecanMIME_HUMAN34131312.049    VersicanCSPG2_HUMAN37316163.746    TetranectinTETN_HUMAN23202029.039    FibromodulinFMOD_HUMAN43667.426    Fibulin-1FBLN1_HUMAN7714149.125    Latent TGFβ-binding protein 2bLTBP2_HUMAN19515155.223    Thrombospondin-1TSP1_HUMAN12911115.118    PodocanbPODN_HUMAN6910106.917    Fibrillin-1FBN1_HUMAN31210102.116    Fibulin-5FBLN5_HUMAN50667.812    Galectin-1LEG1_HUMAN157726.09    DecorinPGS2_HUMAN403310.08    Latent TGFβ-binding protein 4bLTBP4_HUMAN173331.67    ClusterinCLUS_HUMAN52333.65    COMPCOMP_HUMAN83226.33    TenascinTENA_HUMAN241220.952    DermatopontinDERM_HUMAN24229.52Other ECM-associated proteins    Extracellular superoxide dismutaseSODE_HUMAN26222746.0209    Pigment epithelium-derived factorPEDF_HUMAN46323216.082    Serum amyloid P-componentSAMP_HUMAN25313126.062    IGF-binding protein 7bIBP7_HUMAN29151521.047    Apolipoprotein A-IAPOA1_HUMAN316625.019    Apolipoprotein A-IVAPOA4_HUMAN457717.012    Apolipoprotein HbAPOH_HUMAN388814.011    Hepatoma-derived growth factorbHDGF_HUMAN27667.911    Secreted frizzled-related protein 1bSFRP1_HUMAN35559.29    CeruloplasminCERU_HUMAN122442.39    Secreted frizzled-related protein 3bSFRP3_HUMAN36449.28    Apolipoprotein DAPOD_HUMAN216620.08    Apolipoprotein EAPOE_HUMAN36557.67    LIM and cysteine-rich domains protein 1bLMCD1_HUMAN41667.77    Spondin-1bSPON1_HUMAN91332.76    Target of Nesh-SH3bTARSH_HUMAN119222.66Proteases and protease inhibitors    Tryptase β-1bTRYB1_HUMAN319922.022    Procollagen C-endopeptidase enhancer 1PCOC1_HUMAN48887.818    Cathepsin GbCATG_HUMAN29776.716    CalreticulinCALR_HUMAN489911.013    Cathepsin ZbCATZ_HUMAN34667.310    Plasma glutamate carboxypeptidasebPGCP_HUMAN52554.28    Cathepsin DCATD_HUMAN45557.07    Leukocyte elastase inhibitorILEU_HUMAN43559.07    Proactivator polypeptidebSAP_HUMAN58555.37    MMP-2bMMP2_HUMAN74223.34    KallistatinbKAIN_HUMAN49224.73    Carboxypeptidase-like protein X2bCPXM2_HUMAN86224.62kDa%Collagens    Collagen α-1(XIV)bCOEA1_HUMAN194737314.0233    Collagen α-1(XII)COCA1_HUMAN33320202.841    Collagen α-3(VI)CO6A3_HUMAN34416163.729    Collagen α-1(XVIII)COIA1_HUMAN17811113.722    Collagen α-2(VI)CO6A2_HUMAN109443.06    Collagen α-1(XV)COFA1_HUMAN142332.04    Collagen α-1(I)CO1A1_HUMAN129331.54a Sum of all spectra identified for each protein in the three biological samples, performed with two technical replicates each.b Proteins found for the first time by proteomics in the vasculature.Table IIExtracellular space proteins identified by proteomics in three guanidine extractsIGF, insulin-like growth factor.Protein nameSwiss-Prot accession nameMolecular massUnique peptides, nUnique spectra, nCoverageTotal spectra,ankDa%Proteoglycans and glycoproteins    PerlecanPGBM_HUMAN46936748119.01596    FibronectinFINC_HUMAN26318221820.01028    VersicanCSPG2_HUMAN37376894.4681    AggrecanPGCA_HUMAN25070826.3451    BiglycanPGS1_HUMAN42566236.0352    LactadherinMFGM_HUMAN43646748.0309    ProlarginPRELP_HUMAN44557837.0283    PeriostinPOSTN_HUMAN9310011729.0257    LumicanLUM_HUMAN38535830.0204    DecorinPGS2_HUMAN40435129.0173    ClusterinCLUS_HUMAN52444823.0173    Tenascin-XTENX_HUMAN46479794.5140    Laminin subunit γ-1LAMC1_HUMAN178404418.0129    Link protein 1HPLN1_HUMAN40262933.0117    Laminin subunit β-2LAMB2_HUMAN196343414.0104    VitronectinVTNC_HUMAN54252714.0102    Nidogen-1NID1_HUMAN136566415.0101    Adipocyte enhancer-binding protein 1bAEBP1_HUMAN13134407.694    Laminin subunit α-5LAMA5_HUMAN40039394.691    TenascinTENA_HUMAN24125255.652    MimecanMIME_HUMAN34212115.048    EGF-containing fibulin-like ECM protein 1FBLN3_HUMAN55222415.048    Fibulin-5FBLN5_HUMAN50171714.040    Galectin-1LEG1_HUMAN15172053.040    TGFβ-induced protein ig-h3BGH3_HUMAN75232312.035    EMILIN-1EMIL1_HUMAN107883.030    Fibulin-1FBLN1_HUMAN77171712.029    Latent TGFβ-binding protein 2bLTBP2_HUMAN19513132.722    FibromodulinFMOD_HUMAN438118.222    Microfibril-associated glycoprotein 4MFAP4_HUMAN298812.021    DermatopontinDERM_HUMAN248912.021    Fibrillin-1FBN1_HUMAN31215152.421    Latent TGFβ-binding protein 4bLTBP4_HUMAN17312124.215    Nidogen-2NID2_HUMAN151883.511    AgrinbAGRIN_HUMAN215772.311    AsporinASPN_HUMAN438811.011    Cartilage oligomeric matrix proteinCOMP_HUMAN83443.67    Thrombospondin-1TSP1_HUMAN129331.87    Matrix Gla proteinMGP_HUMAN124413.06    Galectin-3LEG3_HUMAN26226.04    SclerostinbSOST_HUMAN24225.63    Matrilin-2bMATN2_HUMAN107222.33Other ECM-associated proteins    Serum amyloid P-componentSAMP_HUMAN25323333.0169    Tubulointerstitial nephritis antigen-likebTINAL_HUMAN52465521.0131    Extracellular superoxide dismutaseSODE_HUMAN26283525.0102    Target of Nesh-SH3bTARSH_HUMAN11930338.689    IGF-binding protein 7bIBP7_HUMAN29303128.059    Cysteine-rich protein 2CRIP2_HUMAN22202125.057    TGFβ-1-induced transcript 1 proteinbTGFI1_HUMAN50272745    LIM and cysteine-rich domains protein 1bLMCD1_HUMAN41252534.039    Apolipoprotein HbAPOH_HUMAN3810129.032    Galectin-3-binding proteinLG3BP_HUMAN65121212.023    RPE-spondinbRPESP_HUMAN30121222.019kDa%    Carboxypeptidase-like protein X2bCPXM2_HUMAN869117.316    Latent TGFβ-binding protein, isoform 1SbLTB1S_HUMAN153892.713    Pigment epithelium derived factorPEDF_HUMAN46558.68    Secreted frizzled-related protein 1bSFRP1_HUMAN35338.96    ProperdinbPROP_HUMAN51345.15    Spondin-1bSPON1_HUMAN91333.04    Neutrophil defensin 1bDEF1_HUMAN10229.64Proteases and protease inhibitors    Serine protease HTRA1HTRA1_HUMAN51222213.068    ChymasebCMA1_HUMAN27121511.023    Lysyl oxidase homolog 1bLOXL1_HUMAN6312127.323    Metalloproteinase inhibitor 3TIMP3_HUMAN249917.012    Tryptase β-1bTRYB1_HUMAN31679.811    Procollagen C-endopeptidase enhancer 1PCOC1_HUMAN48776.710    Cathepsin GbCATG_HUMAN297715.08    Cathepsin DCATD_HUMAN45442.77    AntileukoproteinasebSLPI_HUMAN14339.16    MyeloblastinbPRTN3_HUMAN28224.74Collagens    Collagen α-3(VI)CO6A3_HUMAN34416418620.0898    Collagen α-1(XIV)bCOEA1_HUMAN19413615020.0529    Collagen α-1(XVIII)COIA1_HUMAN17845485.9374    Collagen α-1(VI)CO6A1_HUMAN109507116.0282    Collagen α-2(VI)CO6A2_HUMAN109313119.0155    Collagen α-2(IV)CO4A2_HUMAN16826313.9139    Collagen α-1(XV)COFA1_HUMAN14232347.5100    Collagen α-1(XII)COCA1_HUMAN33342424.090    Collagen α-2(I)CO1A2_HUMAN12912153.570    Collagen α-1(I)CO1A1_HUMAN13914155.951    Collagen α-1(VIII)CO8A1_HUMAN73551.333    Collagen α-1(V)CO5A1_HUMAN184660.623    Collagen α-2(VIII)CO8A2_HUMAN67443.39    Collagen α-1(IV)CO4A1_HUMAN161551.88a Sum of all spectra identified for each protein in the three biological samples, performed with two technical replicates each.b Proteins found for the first time by proteomics in the vasculature.Fig. 2.Protein categories. The classification and numerical distribution of the identified extracellular proteins in the NaCl (A), guanidine (B), and SDS (C) extracts are shown. The size of the doughnut charts is representative of the number of extracellular space proteins identified in the different extracts.To identify proteins of the extracellular space extracted by 0.08% SDS, we performed proteomics analysis of the SDS extracts. Because of the increased complexity of these samples, tryptic peptides were injected into a dual pressure, linear ion trap (LTQ Velos, Thermo Fisher Scientific) with faster scan speed. As expected, the majority of the identified proteins were cellular (data not shown), and only 28 extracellular space proteins were identified (Table III). The classification of the identified extracellular space proteins in the SDS extracts is shown in Fig. 2C. Interestingly, apart from type XIV collagen, all other extracellular space proteins have been previously identified by proteomics in the vasculature. Also, of note is the absence of proteases and protease inhibitors in these extracts either because they are obscured by abundant cellular proteins or because those that remain in the tissue following 0.5 m NaCl extraction are tightly bound to the ECM and require dissociative conditions to be extracted (i.e. guanidine). Peptides of extracellular space proteins identified by LC-MS/MS in the SDS extracts are shown in supplemental Table 3.Table IIIExtracellular space proteins identified by proteomics in three SDS extractsProtein nameSwiss-Prot accession nameMolecular massUnique peptides, nUnique spectra, nCoverageTotal spectra,ankDa%Proteoglycans and glycoproteins    BiglycanPGS1_HUMAN42242832.6141    DecorinPGS2_HUMAN40242433.185    MimecanMIME_HUMAN34212215.076    LumicanLUM_HUMAN38171824.672    ProlarginPRELP_HUMAN44191922.559    FibronectinFINC_HUMAN26329298.138    VersicanCSPG2_HUMAN37313132.333    LactadherinMFGM_HUMAN43131521.223    PerlecanPGBM_HUMAN46920202.922    ClusterinCLUS_HUMAN527714.218    EGF-containing fibulin-like ECM protein 1FBLN3_HUMAN556610.611    PeriostinPOSTN_HUMAN93774.811    Galectin-1LEG1_HUMAN155523.07    TenascinTENA_HUMAN241552.16    VitronectinVTNC_HUMAN54445.65    Link protein 1HPLN1_HUMAN404413.05    Fibulin-1FBLN1_HUMAN77333.83    Laminin subunit α-4LAMA4_HUMAN203331.93    Fibulin-5FBLN5_HUMAN50336.03Other ECM-associated proteins    Apolipoprotein EAPOE_HUMAN36121217.718    Extracellular superoxide dismutaseSODE_HUMAN26101024.013    Apolipoprotein BAPOD_HUMAN21771.510    Apolipoprotein A-IAPOA1_HUMAN305515.05    Galectin-3-binding proteinLG3BP_HUMAN65336.73Collagens    Collagen α-3(VI)CO6A3_HUMAN34457587.4109    Collagen α-2(VI)CO6A2_HUMAN10911112.420    Collagen α-1(XIV)bCOEA1_HUMAN194773.68    Collagen α-1(XVIII)COIA1_HUMAN178661.57a Sum of all spectra identified for each protein in the three biological samples, performed with two technical replicates each.b Proteins found for the first time by proteomics in the vasculature.Identification of Podocan, Sclerostin, and AgrinAmong the identified proteins in the NaCl and guanidine extracts were three recently discovered glycoproteins that have never been detected in the aortic ECM at the protein level. Podocan, an atypical proteoglycan, which has so far been found in the basement membrane of the glomeruli as a product of kidney podocytes (19), was detected in the NaCl extracts. Immunohistological staining confirmed the presence of podocan in the aortic intima (Fig. 3A; the isotype negative control is shown in Fig. 3B). Sclerostin, a potent bone morphogenetic protein antagonist secreted by osteoblasts and osteocytes (20), was identified in the guanidine extract and was localized in the subendothelial layer of the aortic intima (Fig. 3C; the isotype negative control is shown in Fig. 3D). The function of these glycoproteins in the vasculature is to date unknown. Finally, agrin is a large heparan sulfate proteoglycan. It has been found in the basement membrane of various tissues and is known to participate in the organization of the neuromuscular junction (21). Immunohistochemical staining confirmed the presence of agrin on the aortic endothelium (Fig. 3E; the isotype negative control is shown in Fig. 3F), which is also the site for the accumulation of other heparan sulfate proteoglycans, such as perlecan (22). It is therefore possible that agrin participates in the formation of the vascular endothelial barrier. Representative MS/MS spectra for the three glycoproteins are shown in supplemental Fig. 2.Fig. 3.Confirmation of podocan, sclerostin, and agrin in aortic ECM. Immunohistological staining confirmed the presence of podocan in the aortic intima (A). Immunoreactivity appears in brown (magnification, 20×; inset magnification, 60×). Nuclei are counterstained with Mayer's hematoxylin (blue). Sclerostin (C) was localized in the subendothelial layer of the aortic intima. Finally, immunohistochemical staining confirmed the presence of agrin (E). Agrin is localized on the aortic endothelium. The respective isotype negative controls are shown in B, D, and F, respectively. The specificity of the sclerostin and agrin antibodies was confirmed by immunoblotting (data not shown). The aortic specimens were from a 44-year old female (A, B) and a 55-year old male patient (C–F).Evaluation of RePlayFor the wider application of proteomics in clinical research, it is essential to maximize the information obtained from valuable clinical samples. Given the scarcity of human tissue available, we used the RePlay device (Advion), which allowed us to obtain a technical replicate from each clinical specimen in a single LC-MS/MS run. Representative examples of chromatograms using RePlay are shown in Fig. 4, A and B. Correlation analyses of the spectral counts in the three NaCl extracts and the three guanidine extracts are presented in Fig. 4, C and D. The mean spectral counts for the NaCl and the guanidine extracts were 32.7 ± 9.9 (median, 9.5) and 126.5 ± 41 (median, 40), respectively. The R2 values (Pearson correlation coefficient) from all six extracts exceeded 0.94 (p values <0.001), demonstrating good reproducibility in the duplicate analyses (supplemental Fig. 3).Fig. 4.Evaluation of RePlay analysis. Representative chromatograms for a NaCl (A) and a guanidine extract (B) using RePlay are shown. The RePlay device allows the reanalysis of the same sample during a single LC-MS/MS run. The combined correlation of the spectral counts obtained by RePlay in the three NaCl extracts (C) and in the three guanidine extracts (D) demonstrates the good correlation between the technical replicates (Correlation for each of the six extrats can be found in supplemental Fig. 3.). No RePlay analysis was performed for the SDS extracts.Because of their scarcity, MMPs are rarely detected by tissue proteomics (23). Our subfractionation, however, enabled us to obtain spectral evidence for the presence of MMP-2 in one of the NaCl extracts (aortic sample 1). Consistent with our proteomics data, gelatin zymography confirmed the presence of MMP-2 in this sample (Fig. 5A). A representative MS/MS spectrum for MMP-2 is shown in supplemental Fig. 4A. Importantly, the non-denaturing composition of the NaCl buffer allowed the detection of MMP-2 directly without further processing of the extracts.Fig. 5.Validation of proteomics data. Gelatin zymography confirmed the presence of MMP-2 in one of the three NaCl extracts (A). Periostin (B, upper panel) and asporin (B, bottom panel) were validated by immunoblotting in the guanidine extracts (on the same blotting membrane; first, asporin; and second, reprobing for periostin). The total protein loading control for the extracts can be seen in Fig. 1C, bottom panel. For periostin and asporin, the relative band intensity of the immunoblots was measured, and the values were correlated with the spectral counts of the original and the RePlay analysis (n = 2, mean ± S.D. Error bars denote standard deviation.) (C and D).In the guanidine extracts, we performed Western immunoblotting for the ECM protein periostin, which was unambiguously identified (Fig. 5B, upper panel). Periostin mRNA was recently shown to be expressed in rat carotid arteries following balloon injury (24). We also confirmed the presence of the scarce glycoprotein asporin in the vascular ECM (Fig. 5B, bottom panel). Its biological role is to date unclear, but it was recently shown to inhibit TGFβ-induced chondrogenesis in articular cartilage (25) and is a potential regulator of collagen mineralization (26). Finally, the Western blots for periostin and asporin were analyzed by densitometry, and the densitometry values were correlated with the spectral counts obtained from the RePlay analysis (n = 2, mean ± S.D.) (Fig. 5, C and D). Although the technical replicates obtained using the RePlay device were accurate, there is expected variability in the different clinical samples. Representative MS/MS spectra for periostin and asporin are shown in supplemental Fig. 4, B and C.ProteolysisFinally, we investigated whether the NaCl extract represents a subproteome enriched in degradation products. Currently, our understanding of ECM proteolysis is based on in vitro experiments, which do not necessarily relate to in vivo processes. In our methodology, the original molecular mass distribution of the identified proteins is preserved by the gel LC-MS/MS approach. Thus, our method can be important for advancing our understanding of protein degradation in vivo. For instance, we noted that in a fourth aortic specimen, which was classified as normal upon aortotomy, the coefficient of variation for the number of fibronectin spectra in the NaCl extract exceeded 100%, implying a significant difference in the dispersion of the data among the analyzed samples (p ANOVA <0.05). On the other hand, the coefficient of variation for samples 1–3 is not significant (58%; p ANOVA >0.05). We also noted that there was an increase in fibronectin sequence coverage (Fig. 6, A and B) and total fibronectin spectra (∼4-fold) in the NaCl extract of the fourth specimen. Closer inspection of the molecular mass distribution of fibronectin spectra indicated the breakdown of fibronectin in this particular sample with more than 80% of fibronectin spectra identified in gel bands below 150 kDa. Western blotting using an antibody raised against full-length human fibronectin confirmed extensive laddering of the glycoprotein in both the NaCl and guanidine extracts (Fig. 6, C and D). Fibronectin degradation products have been implicated in the induction of proinflammatory responses in the vasculature and other tissues (1). Interestingly, mass spectrometry provided spectral evidence for MMP-9 only in this particular sample (Fig. 6E), and the presence of MMP-9 was confirmed by gelatin zymography (Fig. 6C, lower panel). Fibronectin is a known substrate for MMP-9 (27), but obviously, we cannot exclude that other proteinases contribute to the breakdown of fibronectin in this specimen in vivo. The example, however, demonstrates the potential of our methodology to analyze proteolytic activity based on the presence of proteolytic enzymes and corresponding degradation products in tissues. Importantly, this key information on protein breakdown within clinical samples that were considered normal macroscopically could be lost by using a conventional shotgun proteomics approach.Fig. 6.Detection of proteolysis. Percent sequence coverage for fibronectin in the NaCl (A) and guanidine extracts (B) is shown. “R” denotes RePlay analysis. Note the increase of fibronectin products in the NaCl extracts of the fourth aortic specimen. Western immunoblotting of fibronectin in the NaCl (C) and guanidine (D) extracts confirmed the laddering of fibronectin. The product ion spectrum of the doubly charged tryptic peptide SLGPALLLLQK was identified as MMP-9 (E) (peptide probability, 95%; SEQUEST XCorr score, 2.67; SEQUEST ΔCn score, 0.40; observed m/z, 576.87; actual peptide mass, 1,151.73 Da; peptide charge, 2; delta amu, 0.0049; delta ppm, 4.3; peptide start index, 66; peptide stop index, 76). MMP-9 was only detected by LC-MS/MS in the NaCl extract of the fourth aortic specimen. Gelatin zymography confirmed higher levels of MMP-9 in this sample (C, bottom panel).

Biochemical Extraction of Proteins in Extracellular EnvironmentOur extraction methodology was developed on human aortic samples obtained during cardiac surgery for aortic valve replacement of degenerative aortic valve stenosis. Fig. 1A is a schematic summary of the methodology used in this study. The samples (n = 3) were macroscopically classified as normal during surgery, and upon histological examination, they showed no pathological signs. Extraction of the proteins of the vascular extracellular environment and reduction of the contamination with cellular proteins were achieved by a three-step extraction procedure. First, the tissue samples were treated with 0.5 m NaCl. The salt ions induce displacement of polyionic interactions between proteins, thus enabling the extraction of loosely bound extracellular proteins, including newly synthesized ECM proteins and degradation products (12), providing an overview of tissue turnover. The tissue was then decellularized using SDS. In contrast to other decellularization reagents, such as non-ionic detergents (Triton X-100) (13), 0.1% SDS has been previously shown to be very effective in solubilizing cytoplasmic and nuclear membranes, thereby removing cellular components from tissues (14). However, the ionic detergent above its critical micelle concentration can also cause protein denaturation and disrupt native ECM proteins (15).Fig. 1.Identification of extracellular proteins in human aortas. A, schematic summary. Biochemical subfractionation was used for the enrichment of extracellular space proteins before proteomic analysis. B, validation of decellularization by hematoxylin/eosin staining. Representative images of a hematoxylin/eosin-stained human aorta before (left panel) and after decellularization with 0.08% SDS (right panel) are shown. L, lumen; I, intima; M, media. Arrows depict the intima/media border. Note the loss of nuclear (blue; hematoxylin) and cytoplasmic contents (pink; eosin), whereas other eosinophilic structures, such as collagen fibers, are preserved. C, immunoblotting and silver staining. The ECM glycoprotein AEBP1 is found in the NaCl and guanidine (Guan) extracts, which are enriched in proteins of the extracellular space (upper panel). Cytoplasmic β-actin is predominantly found in the SDS extracts (middle panel), demonstrating successful depletion of cellular contents. Silver staining of proteins from all extracts separated by one-dimensional SDS-PAGE shows the pattern of proteins extracted in the three different extraction steps (bottom panel).To substantially remove the cellular material while preserving the ECM and its associated proteins as much as possible, we used 2.8 mm (0.08%) SDS in distilled water. This concentration is well below the critical micelle concentration of SDS in distilled water, which is 8 mm (16). In addition, at this concentration, SDS is effective in solubilizing cell membrane lipid bilayers (17). Treatment with 0.08% SDS was sufficient to remove most cellular components as shown by the marked reduction of nuclear staining (hematoxylin; Fig. 1B, blue). The cytoplasmic staining (eosin; pink) was also reduced, but other eosinophilic structures, such as collagen and elastin, were preserved in the tissue. Finally, the specimens were subjected to denaturing extraction in a 4 m guanidine HCl buffer. Guanidine HCl extractions were developed more than 40 years ago as a very effective way to solubilize most of the strongly bound ECM components, including large aggregating proteoglycans (versican, aggrecan, etc.), small proteoglycans (decorin, biglycan, etc.), cell attachment matrix glycoproteins (such as type VI collagen, fibronectins, and laminins), and basement membrane components (perlecan, type IV collagen, etc.). Guanidine induces disaggregation of ECM components by destabilizing the ionic, disulfide-dependent protein conformations (18). Following dissociative extraction, only the insoluble, heavily cross-linked interstitial matrix, namely type I and III collagens and elastin, remains in the tissue. Western blotting confirmed that β-actin immunoreactivity is predominantly found in the SDS extracts (Fig 1C, middle panel), which also had the highest protein concentration per milliliter (supplemental Fig. 1). On the other hand, the extracellular AEBP1 isoform, an ECM-associated glycoprotein, was detected in the NaCl and guanidine extracts by Western blotting (Fig 1C, upper panel), indicating that these extracts are enriched in proteins of the extracellular space. Silver staining revealed a distinct but reproducible protein pattern in the three different extraction steps (Fig. 1C, bottom panel).The different extracts were subjected to enzymatic deglycosylation to remove chondroitin, keratan, and heparan sulfate glycosaminoglycan (GAG) chains from the matrix glycoproteins. Deglycosylation is a critical step in our methodology. First, GAGs increase the molecular weight of glycoproteins and affect their separation and resolution by one-dimensional electrophoresis. Second, the presence of GAGs greatly reduces the chance for the identification of proteoglycans by mass spectrometry.

Biochemical Extraction of Proteins in Extracellular Environment

Our extraction methodology was developed on human aortic samples obtained during cardiac surgery for aortic valve replacement of degenerative aortic valve stenosis. Fig. 1A is a schematic summary of the methodology used in this study. The samples (n = 3) were macroscopically classified as normal during surgery, and upon histological examination, they showed no pathological signs. Extraction of the proteins of the vascular extracellular environment and reduction of the contamination with cellular proteins were achieved by a three-step extraction procedure. First, the tissue samples were treated with 0.5 m NaCl. The salt ions induce displacement of polyionic interactions between proteins, thus enabling the extraction of loosely bound extracellular proteins, including newly synthesized ECM proteins and degradation products (12), providing an overview of tissue turnover. The tissue was then decellularized using SDS. In contrast to other decellularization reagents, such as non-ionic detergents (Triton X-100) (13), 0.1% SDS has been previously shown to be very effective in solubilizing cytoplasmic and nuclear membranes, thereby removing cellular components from tissues (14). However, the ionic detergent above its critical micelle concentration can also cause protein denaturation and disrupt native ECM proteins (15).

Identification of extracellular proteins in human aortas. A, schematic summary. Biochemical subfractionation was used for the enrichment of extracellular space proteins before proteomic analysis. B, validation of decellularization by hematoxylin/eosin staining. Representative images of a hematoxylin/eosin-stained human aorta before (left panel) and after decellularization with 0.08% SDS (right panel) are shown. L, lumen; I, intima; M, media. Arrows depict the intima/media border. Note the loss of nuclear (blue; hematoxylin) and cytoplasmic contents (pink; eosin), whereas other eosinophilic structures, such as collagen fibers, are preserved. C, immunoblotting and silver staining. The ECM glycoprotein AEBP1 is found in the NaCl and guanidine (Guan) extracts, which are enriched in proteins of the extracellular space (upper panel). Cytoplasmic β-actin is predominantly found in the SDS extracts (middle panel), demonstrating successful depletion of cellular contents. Silver staining of proteins from all extracts separated by one-dimensional SDS-PAGE shows the pattern of proteins extracted in the three different extraction steps (bottom panel).

To substantially remove the cellular material while preserving the ECM and its associated proteins as much as possible, we used 2.8 mm (0.08%) SDS in distilled water. This concentration is well below the critical micelle concentration of SDS in distilled water, which is 8 mm (16). In addition, at this concentration, SDS is effective in solubilizing cell membrane lipid bilayers (17). Treatment with 0.08% SDS was sufficient to remove most cellular components as shown by the marked reduction of nuclear staining (hematoxylin; Fig. 1B, blue). The cytoplasmic staining (eosin; pink) was also reduced, but other eosinophilic structures, such as collagen and elastin, were preserved in the tissue. Finally, the specimens were subjected to denaturing extraction in a 4 m guanidine HCl buffer. Guanidine HCl extractions were developed more than 40 years ago as a very effective way to solubilize most of the strongly bound ECM components, including large aggregating proteoglycans (versican, aggrecan, etc.), small proteoglycans (decorin, biglycan, etc.), cell attachment matrix glycoproteins (such as type VI collagen, fibronectins, and laminins), and basement membrane components (perlecan, type IV collagen, etc.). Guanidine induces disaggregation of ECM components by destabilizing the ionic, disulfide-dependent protein conformations (18). Following dissociative extraction, only the insoluble, heavily cross-linked interstitial matrix, namely type I and III collagens and elastin, remains in the tissue. Western blotting confirmed that β-actin immunoreactivity is predominantly found in the SDS extracts (Fig 1C, middle panel), which also had the highest protein concentration per milliliter (supplemental Fig. 1). On the other hand, the extracellular AEBP1 isoform, an ECM-associated glycoprotein, was detected in the NaCl and guanidine extracts by Western blotting (Fig 1C, upper panel), indicating that these extracts are enriched in proteins of the extracellular space. Silver staining revealed a distinct but reproducible protein pattern in the three different extraction steps (Fig. 1C, bottom panel).

The different extracts were subjected to enzymatic deglycosylation to remove chondroitin, keratan, and heparan sulfate glycosaminoglycan (GAG) chains from the matrix glycoproteins. Deglycosylation is a critical step in our methodology. First, GAGs increase the molecular weight of glycoproteins and affect their separation and resolution by one-dimensional electrophoresis. Second, the presence of GAGs greatly reduces the chance for the identification of proteoglycans by mass spectrometry.

Proteomics Analysis of ExtractsAfter separation by SDS-PAGE, protein bands were excised from silver-stained gels, subjected to in-gel tryptic digestion, and analyzed by LC-MS/MS. The peptides were injected into a high mass accuracy ion trap (LTQ-Orbitrap XL, Thermo Fisher Scientific) using RePlay (Advion). This injection system splits the gradient from the analytical column and allows the reanalysis of the same sample in a single LC-MS/MS run (8). 272 and 321 proteins were identified in the NaCl and guanidine extracts, respectively. All the peptides identified by LC-MS/MS in the NaCl and guanidine extracts are shown in supplemental Tables 1 and 2. According to their gene ontology annotation (European Molecular Biology Laboratory-European Bioinformatics Institute database, www.uniprot.org), a quarter of these proteins were located in the extracellular space: among the 61 extracellular proteins in the NaCl extracts (Table I) and the 84 extracellular proteins in the guanidine extracts (Table II), proteins highlighted with Footnote b were, to the best of our knowledge, identified for the first time in the vasculature by proteomics. Lipoproteins were included (even though they are plasma-derived) because of their intimate relationship with proteoglycans and their relevance in cardiovascular pathology. The classification of the identified extracellular space proteins in the NaCl and the guanidine extracts is shown in Fig. 2, A and B. As expected, the denaturing guanidine extracts were particularly enriched in ECM proteoglycans and glycoproteins (50%), whereas the mildly dissociative NaCl extracts were enriched in proteases and protease inhibitors (21%), indicating the importance of this subproteome for studying proteolytic processes in tissues.Table IExtracellular space proteins identified by proteomics in three NaCl extractsCOMP, cartilage oligomeric matrix protein; IGF, insulin-like growth factor.Protein nameSwiss-Prot accession nameMolecular massUnique peptides, nUnique spectra, nCoverageTotal spectra,ankDa%Proteoglycans and glycoproteins    LumicanLUM_HUMAN38404020.0249    Tenascin-XTENX_HUMAN46466668.1156    Fibulin-3FBLN3_HUMAN55383822.0155    Adipocyte enhancer-binding protein 1bAEBP1_HUMAN131373911.0153    FibronectinFINC_HUMAN26334347.3111    PerlecanPGBM_HUMAN46916162.550    MimecanMIME_HUMAN34131312.049    VersicanCSPG2_HUMAN37316163.746    TetranectinTETN_HUMAN23202029.039    FibromodulinFMOD_HUMAN43667.426    Fibulin-1FBLN1_HUMAN7714149.125    Latent TGFβ-binding protein 2bLTBP2_HUMAN19515155.223    Thrombospondin-1TSP1_HUMAN12911115.118    PodocanbPODN_HUMAN6910106.917    Fibrillin-1FBN1_HUMAN31210102.116    Fibulin-5FBLN5_HUMAN50667.812    Galectin-1LEG1_HUMAN157726.09    DecorinPGS2_HUMAN403310.08    Latent TGFβ-binding protein 4bLTBP4_HUMAN173331.67    ClusterinCLUS_HUMAN52333.65    COMPCOMP_HUMAN83226.33    TenascinTENA_HUMAN241220.952    DermatopontinDERM_HUMAN24229.52Other ECM-associated proteins    Extracellular superoxide dismutaseSODE_HUMAN26222746.0209    Pigment epithelium-derived factorPEDF_HUMAN46323216.082    Serum amyloid P-componentSAMP_HUMAN25313126.062    IGF-binding protein 7bIBP7_HUMAN29151521.047    Apolipoprotein A-IAPOA1_HUMAN316625.019    Apolipoprotein A-IVAPOA4_HUMAN457717.012    Apolipoprotein HbAPOH_HUMAN388814.011    Hepatoma-derived growth factorbHDGF_HUMAN27667.911    Secreted frizzled-related protein 1bSFRP1_HUMAN35559.29    CeruloplasminCERU_HUMAN122442.39    Secreted frizzled-related protein 3bSFRP3_HUMAN36449.28    Apolipoprotein DAPOD_HUMAN216620.08    Apolipoprotein EAPOE_HUMAN36557.67    LIM and cysteine-rich domains protein 1bLMCD1_HUMAN41667.77    Spondin-1bSPON1_HUMAN91332.76    Target of Nesh-SH3bTARSH_HUMAN119222.66Proteases and protease inhibitors    Tryptase β-1bTRYB1_HUMAN319922.022    Procollagen C-endopeptidase enhancer 1PCOC1_HUMAN48887.818    Cathepsin GbCATG_HUMAN29776.716    CalreticulinCALR_HUMAN489911.013    Cathepsin ZbCATZ_HUMAN34667.310    Plasma glutamate carboxypeptidasebPGCP_HUMAN52554.28    Cathepsin DCATD_HUMAN45557.07    Leukocyte elastase inhibitorILEU_HUMAN43559.07    Proactivator polypeptidebSAP_HUMAN58555.37    MMP-2bMMP2_HUMAN74223.34    KallistatinbKAIN_HUMAN49224.73    Carboxypeptidase-like protein X2bCPXM2_HUMAN86224.62kDa%Collagens    Collagen α-1(XIV)bCOEA1_HUMAN194737314.0233    Collagen α-1(XII)COCA1_HUMAN33320202.841    Collagen α-3(VI)CO6A3_HUMAN34416163.729    Collagen α-1(XVIII)COIA1_HUMAN17811113.722    Collagen α-2(VI)CO6A2_HUMAN109443.06    Collagen α-1(XV)COFA1_HUMAN142332.04    Collagen α-1(I)CO1A1_HUMAN129331.54a Sum of all spectra identified for each protein in the three biological samples, performed with two technical replicates each.b Proteins found for the first time by proteomics in the vasculature.Table IIExtracellular space proteins identified by proteomics in three guanidine extractsIGF, insulin-like growth factor.Protein nameSwiss-Prot accession nameMolecular massUnique peptides, nUnique spectra, nCoverageTotal spectra,ankDa%Proteoglycans and glycoproteins    PerlecanPGBM_HUMAN46936748119.01596    FibronectinFINC_HUMAN26318221820.01028    VersicanCSPG2_HUMAN37376894.4681    AggrecanPGCA_HUMAN25070826.3451    BiglycanPGS1_HUMAN42566236.0352    LactadherinMFGM_HUMAN43646748.0309    ProlarginPRELP_HUMAN44557837.0283    PeriostinPOSTN_HUMAN9310011729.0257    LumicanLUM_HUMAN38535830.0204    DecorinPGS2_HUMAN40435129.0173    ClusterinCLUS_HUMAN52444823.0173    Tenascin-XTENX_HUMAN46479794.5140    Laminin subunit γ-1LAMC1_HUMAN178404418.0129    Link protein 1HPLN1_HUMAN40262933.0117    Laminin subunit β-2LAMB2_HUMAN196343414.0104    VitronectinVTNC_HUMAN54252714.0102    Nidogen-1NID1_HUMAN136566415.0101    Adipocyte enhancer-binding protein 1bAEBP1_HUMAN13134407.694    Laminin subunit α-5LAMA5_HUMAN40039394.691    TenascinTENA_HUMAN24125255.652    MimecanMIME_HUMAN34212115.048    EGF-containing fibulin-like ECM protein 1FBLN3_HUMAN55222415.048    Fibulin-5FBLN5_HUMAN50171714.040    Galectin-1LEG1_HUMAN15172053.040    TGFβ-induced protein ig-h3BGH3_HUMAN75232312.035    EMILIN-1EMIL1_HUMAN107883.030    Fibulin-1FBLN1_HUMAN77171712.029    Latent TGFβ-binding protein 2bLTBP2_HUMAN19513132.722    FibromodulinFMOD_HUMAN438118.222    Microfibril-associated glycoprotein 4MFAP4_HUMAN298812.021    DermatopontinDERM_HUMAN248912.021    Fibrillin-1FBN1_HUMAN31215152.421    Latent TGFβ-binding protein 4bLTBP4_HUMAN17312124.215    Nidogen-2NID2_HUMAN151883.511    AgrinbAGRIN_HUMAN215772.311    AsporinASPN_HUMAN438811.011    Cartilage oligomeric matrix proteinCOMP_HUMAN83443.67    Thrombospondin-1TSP1_HUMAN129331.87    Matrix Gla proteinMGP_HUMAN124413.06    Galectin-3LEG3_HUMAN26226.04    SclerostinbSOST_HUMAN24225.63    Matrilin-2bMATN2_HUMAN107222.33Other ECM-associated proteins    Serum amyloid P-componentSAMP_HUMAN25323333.0169    Tubulointerstitial nephritis antigen-likebTINAL_HUMAN52465521.0131    Extracellular superoxide dismutaseSODE_HUMAN26283525.0102    Target of Nesh-SH3bTARSH_HUMAN11930338.689    IGF-binding protein 7bIBP7_HUMAN29303128.059    Cysteine-rich protein 2CRIP2_HUMAN22202125.057    TGFβ-1-induced transcript 1 proteinbTGFI1_HUMAN50272745    LIM and cysteine-rich domains protein 1bLMCD1_HUMAN41252534.039    Apolipoprotein HbAPOH_HUMAN3810129.032    Galectin-3-binding proteinLG3BP_HUMAN65121212.023    RPE-spondinbRPESP_HUMAN30121222.019kDa%    Carboxypeptidase-like protein X2bCPXM2_HUMAN869117.316    Latent TGFβ-binding protein, isoform 1SbLTB1S_HUMAN153892.713    Pigment epithelium derived factorPEDF_HUMAN46558.68    Secreted frizzled-related protein 1bSFRP1_HUMAN35338.96    ProperdinbPROP_HUMAN51345.15    Spondin-1bSPON1_HUMAN91333.04    Neutrophil defensin 1bDEF1_HUMAN10229.64Proteases and protease inhibitors    Serine protease HTRA1HTRA1_HUMAN51222213.068    ChymasebCMA1_HUMAN27121511.023    Lysyl oxidase homolog 1bLOXL1_HUMAN6312127.323    Metalloproteinase inhibitor 3TIMP3_HUMAN249917.012    Tryptase β-1bTRYB1_HUMAN31679.811    Procollagen C-endopeptidase enhancer 1PCOC1_HUMAN48776.710    Cathepsin GbCATG_HUMAN297715.08    Cathepsin DCATD_HUMAN45442.77    AntileukoproteinasebSLPI_HUMAN14339.16    MyeloblastinbPRTN3_HUMAN28224.74Collagens    Collagen α-3(VI)CO6A3_HUMAN34416418620.0898    Collagen α-1(XIV)bCOEA1_HUMAN19413615020.0529    Collagen α-1(XVIII)COIA1_HUMAN17845485.9374    Collagen α-1(VI)CO6A1_HUMAN109507116.0282    Collagen α-2(VI)CO6A2_HUMAN109313119.0155    Collagen α-2(IV)CO4A2_HUMAN16826313.9139    Collagen α-1(XV)COFA1_HUMAN14232347.5100    Collagen α-1(XII)COCA1_HUMAN33342424.090    Collagen α-2(I)CO1A2_HUMAN12912153.570    Collagen α-1(I)CO1A1_HUMAN13914155.951    Collagen α-1(VIII)CO8A1_HUMAN73551.333    Collagen α-1(V)CO5A1_HUMAN184660.623    Collagen α-2(VIII)CO8A2_HUMAN67443.39    Collagen α-1(IV)CO4A1_HUMAN161551.88a Sum of all spectra identified for each protein in the three biological samples, performed with two technical replicates each.b Proteins found for the first time by proteomics in the vasculature.Fig. 2.Protein categories. The classification and numerical distribution of the identified extracellular proteins in the NaCl (A), guanidine (B), and SDS (C) extracts are shown. The size of the doughnut charts is representative of the number of extracellular space proteins identified in the different extracts.To identify proteins of the extracellular space extracted by 0.08% SDS, we performed proteomics analysis of the SDS extracts. Because of the increased complexity of these samples, tryptic peptides were injected into a dual pressure, linear ion trap (LTQ Velos, Thermo Fisher Scientific) with faster scan speed. As expected, the majority of the identified proteins were cellular (data not shown), and only 28 extracellular space proteins were identified (Table III). The classification of the identified extracellular space proteins in the SDS extracts is shown in Fig. 2C. Interestingly, apart from type XIV collagen, all other extracellular space proteins have been previously identified by proteomics in the vasculature. Also, of note is the absence of proteases and protease inhibitors in these extracts either because they are obscured by abundant cellular proteins or because those that remain in the tissue following 0.5 m NaCl extraction are tightly bound to the ECM and require dissociative conditions to be extracted (i.e. guanidine). Peptides of extracellular space proteins identified by LC-MS/MS in the SDS extracts are shown in supplemental Table 3.Table IIIExtracellular space proteins identified by proteomics in three SDS extractsProtein nameSwiss-Prot accession nameMolecular massUnique peptides, nUnique spectra, nCoverageTotal spectra,ankDa%Proteoglycans and glycoproteins    BiglycanPGS1_HUMAN42242832.6141    DecorinPGS2_HUMAN40242433.185    MimecanMIME_HUMAN34212215.076    LumicanLUM_HUMAN38171824.672    ProlarginPRELP_HUMAN44191922.559    FibronectinFINC_HUMAN26329298.138    VersicanCSPG2_HUMAN37313132.333    LactadherinMFGM_HUMAN43131521.223    PerlecanPGBM_HUMAN46920202.922    ClusterinCLUS_HUMAN527714.218    EGF-containing fibulin-like ECM protein 1FBLN3_HUMAN556610.611    PeriostinPOSTN_HUMAN93774.811    Galectin-1LEG1_HUMAN155523.07    TenascinTENA_HUMAN241552.16    VitronectinVTNC_HUMAN54445.65    Link protein 1HPLN1_HUMAN404413.05    Fibulin-1FBLN1_HUMAN77333.83    Laminin subunit α-4LAMA4_HUMAN203331.93    Fibulin-5FBLN5_HUMAN50336.03Other ECM-associated proteins    Apolipoprotein EAPOE_HUMAN36121217.718    Extracellular superoxide dismutaseSODE_HUMAN26101024.013    Apolipoprotein BAPOD_HUMAN21771.510    Apolipoprotein A-IAPOA1_HUMAN305515.05    Galectin-3-binding proteinLG3BP_HUMAN65336.73Collagens    Collagen α-3(VI)CO6A3_HUMAN34457587.4109    Collagen α-2(VI)CO6A2_HUMAN10911112.420    Collagen α-1(XIV)bCOEA1_HUMAN194773.68    Collagen α-1(XVIII)COIA1_HUMAN178661.57a Sum of all spectra identified for each protein in the three biological samples, performed with two technical replicates each.b Proteins found for the first time by proteomics in the vasculature.

Proteomics Analysis of Extracts

After separation by SDS-PAGE, protein bands were excised from silver-stained gels, subjected to in-gel tryptic digestion, and analyzed by LC-MS/MS. The peptides were injected into a high mass accuracy ion trap (LTQ-Orbitrap XL, Thermo Fisher Scientific) using RePlay (Advion). This injection system splits the gradient from the analytical column and allows the reanalysis of the same sample in a single LC-MS/MS run (8). 272 and 321 proteins were identified in the NaCl and guanidine extracts, respectively. All the peptides identified by LC-MS/MS in the NaCl and guanidine extracts are shown in supplemental Tables 1 and 2. According to their gene ontology annotation (European Molecular Biology Laboratory-European Bioinformatics Institute database, www.uniprot.org), a quarter of these proteins were located in the extracellular space: among the 61 extracellular proteins in the NaCl extracts (Table I) and the 84 extracellular proteins in the guanidine extracts (Table II), proteins highlighted with Footnote b were, to the best of our knowledge, identified for the first time in the vasculature by proteomics. Lipoproteins were included (even though they are plasma-derived) because of their intimate relationship with proteoglycans and their relevance in cardiovascular pathology. The classification of the identified extracellular space proteins in the NaCl and the guanidine extracts is shown in Fig. 2, A and B. As expected, the denaturing guanidine extracts were particularly enriched in ECM proteoglycans and glycoproteins (50%), whereas the mildly dissociative NaCl extracts were enriched in proteases and protease inhibitors (21%), indicating the importance of this subproteome for studying proteolytic processes in tissues.

Extracellular space proteins identified by proteomics in three NaCl extracts

COMP, cartilage oligomeric matrix protein; IGF, insulin-like growth factor.

a Sum of all spectra identified for each protein in the three biological samples, performed with two technical replicates each.

b Proteins found for the first time by proteomics in the vasculature.

Extracellular space proteins identified by proteomics in three guanidine extracts

IGF, insulin-like growth factor.

a Sum of all spectra identified for each protein in the three biological samples, performed with two technical replicates each.

b Proteins found for the first time by proteomics in the vasculature.

Protein categories. The classification and numerical distribution of the identified extracellular proteins in the NaCl (A), guanidine (B), and SDS (C) extracts are shown. The size of the doughnut charts is representative of the number of extracellular space proteins identified in the different extracts.

To identify proteins of the extracellular space extracted by 0.08% SDS, we performed proteomics analysis of the SDS extracts. Because of the increased complexity of these samples, tryptic peptides were injected into a dual pressure, linear ion trap (LTQ Velos, Thermo Fisher Scientific) with faster scan speed. As expected, the majority of the identified proteins were cellular (data not shown), and only 28 extracellular space proteins were identified (Table III). The classification of the identified extracellular space proteins in the SDS extracts is shown in Fig. 2C. Interestingly, apart from type XIV collagen, all other extracellular space proteins have been previously identified by proteomics in the vasculature. Also, of note is the absence of proteases and protease inhibitors in these extracts either because they are obscured by abundant cellular proteins or because those that remain in the tissue following 0.5 m NaCl extraction are tightly bound to the ECM and require dissociative conditions to be extracted (i.e. guanidine). Peptides of extracellular space proteins identified by LC-MS/MS in the SDS extracts are shown in supplemental Table 3.

Extracellular space proteins identified by proteomics in three SDS extracts

a Sum of all spectra identified for each protein in the three biological samples, performed with two technical replicates each.

b Proteins found for the first time by proteomics in the vasculature.

Identification of Podocan, Sclerostin, and AgrinAmong the identified proteins in the NaCl and guanidine extracts were three recently discovered glycoproteins that have never been detected in the aortic ECM at the protein level. Podocan, an atypical proteoglycan, which has so far been found in the basement membrane of the glomeruli as a product of kidney podocytes (19), was detected in the NaCl extracts. Immunohistological staining confirmed the presence of podocan in the aortic intima (Fig. 3A; the isotype negative control is shown in Fig. 3B). Sclerostin, a potent bone morphogenetic protein antagonist secreted by osteoblasts and osteocytes (20), was identified in the guanidine extract and was localized in the subendothelial layer of the aortic intima (Fig. 3C; the isotype negative control is shown in Fig. 3D). The function of these glycoproteins in the vasculature is to date unknown. Finally, agrin is a large heparan sulfate proteoglycan. It has been found in the basement membrane of various tissues and is known to participate in the organization of the neuromuscular junction (21). Immunohistochemical staining confirmed the presence of agrin on the aortic endothelium (Fig. 3E; the isotype negative control is shown in Fig. 3F), which is also the site for the accumulation of other heparan sulfate proteoglycans, such as perlecan (22). It is therefore possible that agrin participates in the formation of the vascular endothelial barrier. Representative MS/MS spectra for the three glycoproteins are shown in supplemental Fig. 2.Fig. 3.Confirmation of podocan, sclerostin, and agrin in aortic ECM. Immunohistological staining confirmed the presence of podocan in the aortic intima (A). Immunoreactivity appears in brown (magnification, 20×; inset magnification, 60×). Nuclei are counterstained with Mayer's hematoxylin (blue). Sclerostin (C) was localized in the subendothelial layer of the aortic intima. Finally, immunohistochemical staining confirmed the presence of agrin (E). Agrin is localized on the aortic endothelium. The respective isotype negative controls are shown in B, D, and F, respectively. The specificity of the sclerostin and agrin antibodies was confirmed by immunoblotting (data not shown). The aortic specimens were from a 44-year old female (A, B) and a 55-year old male patient (C–F).

Identification of Podocan, Sclerostin, and Agrin

Among the identified proteins in the NaCl and guanidine extracts were three recently discovered glycoproteins that have never been detected in the aortic ECM at the protein level. Podocan, an atypical proteoglycan, which has so far been found in the basement membrane of the glomeruli as a product of kidney podocytes (19), was detected in the NaCl extracts. Immunohistological staining confirmed the presence of podocan in the aortic intima (Fig. 3A; the isotype negative control is shown in Fig. 3B). Sclerostin, a potent bone morphogenetic protein antagonist secreted by osteoblasts and osteocytes (20), was identified in the guanidine extract and was localized in the subendothelial layer of the aortic intima (Fig. 3C; the isotype negative control is shown in Fig. 3D). The function of these glycoproteins in the vasculature is to date unknown. Finally, agrin is a large heparan sulfate proteoglycan. It has been found in the basement membrane of various tissues and is known to participate in the organization of the neuromuscular junction (21). Immunohistochemical staining confirmed the presence of agrin on the aortic endothelium (Fig. 3E; the isotype negative control is shown in Fig. 3F), which is also the site for the accumulation of other heparan sulfate proteoglycans, such as perlecan (22). It is therefore possible that agrin participates in the formation of the vascular endothelial barrier. Representative MS/MS spectra for the three glycoproteins are shown in supplemental Fig. 2.

Confirmation of podocan, sclerostin, and agrin in aortic ECM. Immunohistological staining confirmed the presence of podocan in the aortic intima (A). Immunoreactivity appears in brown (magnification, 20×; inset magnification, 60×). Nuclei are counterstained with Mayer's hematoxylin (blue). Sclerostin (C) was localized in the subendothelial layer of the aortic intima. Finally, immunohistochemical staining confirmed the presence of agrin (E). Agrin is localized on the aortic endothelium. The respective isotype negative controls are shown in B, D, and F, respectively. The specificity of the sclerostin and agrin antibodies was confirmed by immunoblotting (data not shown). The aortic specimens were from a 44-year old female (A, B) and a 55-year old male patient (C–F).

Evaluation of RePlayFor the wider application of proteomics in clinical research, it is essential to maximize the information obtained from valuable clinical samples. Given the scarcity of human tissue available, we used the RePlay device (Advion), which allowed us to obtain a technical replicate from each clinical specimen in a single LC-MS/MS run. Representative examples of chromatograms using RePlay are shown in Fig. 4, A and B. Correlation analyses of the spectral counts in the three NaCl extracts and the three guanidine extracts are presented in Fig. 4, C and D. The mean spectral counts for the NaCl and the guanidine extracts were 32.7 ± 9.9 (median, 9.5) and 126.5 ± 41 (median, 40), respectively. The R2 values (Pearson correlation coefficient) from all six extracts exceeded 0.94 (p values <0.001), demonstrating good reproducibility in the duplicate analyses (supplemental Fig. 3).Fig. 4.Evaluation of RePlay analysis. Representative chromatograms for a NaCl (A) and a guanidine extract (B) using RePlay are shown. The RePlay device allows the reanalysis of the same sample during a single LC-MS/MS run. The combined correlation of the spectral counts obtained by RePlay in the three NaCl extracts (C) and in the three guanidine extracts (D) demonstrates the good correlation between the technical replicates (Correlation for each of the six extrats can be found in supplemental Fig. 3.). No RePlay analysis was performed for the SDS extracts.Because of their scarcity, MMPs are rarely detected by tissue proteomics (23). Our subfractionation, however, enabled us to obtain spectral evidence for the presence of MMP-2 in one of the NaCl extracts (aortic sample 1). Consistent with our proteomics data, gelatin zymography confirmed the presence of MMP-2 in this sample (Fig. 5A). A representative MS/MS spectrum for MMP-2 is shown in supplemental Fig. 4A. Importantly, the non-denaturing composition of the NaCl buffer allowed the detection of MMP-2 directly without further processing of the extracts.Fig. 5.Validation of proteomics data. Gelatin zymography confirmed the presence of MMP-2 in one of the three NaCl extracts (A). Periostin (B, upper panel) and asporin (B, bottom panel) were validated by immunoblotting in the guanidine extracts (on the same blotting membrane; first, asporin; and second, reprobing for periostin). The total protein loading control for the extracts can be seen in Fig. 1C, bottom panel. For periostin and asporin, the relative band intensity of the immunoblots was measured, and the values were correlated with the spectral counts of the original and the RePlay analysis (n = 2, mean ± S.D. Error bars denote standard deviation.) (C and D).In the guanidine extracts, we performed Western immunoblotting for the ECM protein periostin, which was unambiguously identified (Fig. 5B, upper panel). Periostin mRNA was recently shown to be expressed in rat carotid arteries following balloon injury (24). We also confirmed the presence of the scarce glycoprotein asporin in the vascular ECM (Fig. 5B, bottom panel). Its biological role is to date unclear, but it was recently shown to inhibit TGFβ-induced chondrogenesis in articular cartilage (25) and is a potential regulator of collagen mineralization (26). Finally, the Western blots for periostin and asporin were analyzed by densitometry, and the densitometry values were correlated with the spectral counts obtained from the RePlay analysis (n = 2, mean ± S.D.) (Fig. 5, C and D). Although the technical replicates obtained using the RePlay device were accurate, there is expected variability in the different clinical samples. Representative MS/MS spectra for periostin and asporin are shown in supplemental Fig. 4, B and C.

Evaluation of RePlay

For the wider application of proteomics in clinical research, it is essential to maximize the information obtained from valuable clinical samples. Given the scarcity of human tissue available, we used the RePlay device (Advion), which allowed us to obtain a technical replicate from each clinical specimen in a single LC-MS/MS run. Representative examples of chromatograms using RePlay are shown in Fig. 4, A and B. Correlation analyses of the spectral counts in the three NaCl extracts and the three guanidine extracts are presented in Fig. 4, C and D. The mean spectral counts for the NaCl and the guanidine extracts were 32.7 ± 9.9 (median, 9.5) and 126.5 ± 41 (median, 40), respectively. The R2 values (Pearson correlation coefficient) from all six extracts exceeded 0.94 (p values <0.001), demonstrating good reproducibility in the duplicate analyses (supplemental Fig. 3).

Evaluation of RePlay analysis. Representative chromatograms for a NaCl (A) and a guanidine extract (B) using RePlay are shown. The RePlay device allows the reanalysis of the same sample during a single LC-MS/MS run. The combined correlation of the spectral counts obtained by RePlay in the three NaCl extracts (C) and in the three guanidine extracts (D) demonstrates the good correlation between the technical replicates (Correlation for each of the six extrats can be found in supplemental Fig. 3.). No RePlay analysis was performed for the SDS extracts.

Because of their scarcity, MMPs are rarely detected by tissue proteomics (23). Our subfractionation, however, enabled us to obtain spectral evidence for the presence of MMP-2 in one of the NaCl extracts (aortic sample 1). Consistent with our proteomics data, gelatin zymography confirmed the presence of MMP-2 in this sample (Fig. 5A). A representative MS/MS spectrum for MMP-2 is shown in supplemental Fig. 4A. Importantly, the non-denaturing composition of the NaCl buffer allowed the detection of MMP-2 directly without further processing of the extracts.

Validation of proteomics data. Gelatin zymography confirmed the presence of MMP-2 in one of the three NaCl extracts (A). Periostin (B, upper panel) and asporin (B, bottom panel) were validated by immunoblotting in the guanidine extracts (on the same blotting membrane; first, asporin; and second, reprobing for periostin). The total protein loading control for the extracts can be seen in Fig. 1C, bottom panel. For periostin and asporin, the relative band intensity of the immunoblots was measured, and the values were correlated with the spectral counts of the original and the RePlay analysis (n = 2, mean ± S.D. Error bars denote standard deviation.) (C and D).

In the guanidine extracts, we performed Western immunoblotting for the ECM protein periostin, which was unambiguously identified (Fig. 5B, upper panel). Periostin mRNA was recently shown to be expressed in rat carotid arteries following balloon injury (24). We also confirmed the presence of the scarce glycoprotein asporin in the vascular ECM (Fig. 5B, bottom panel). Its biological role is to date unclear, but it was recently shown to inhibit TGFβ-induced chondrogenesis in articular cartilage (25) and is a potential regulator of collagen mineralization (26). Finally, the Western blots for periostin and asporin were analyzed by densitometry, and the densitometry values were correlated with the spectral counts obtained from the RePlay analysis (n = 2, mean ± S.D.) (Fig. 5, C and D). Although the technical replicates obtained using the RePlay device were accurate, there is expected variability in the different clinical samples. Representative MS/MS spectra for periostin and asporin are shown in supplemental Fig. 4, B and C.

ProteolysisFinally, we investigated whether the NaCl extract represents a subproteome enriched in degradation products. Currently, our understanding of ECM proteolysis is based on in vitro experiments, which do not necessarily relate to in vivo processes. In our methodology, the original molecular mass distribution of the identified proteins is preserved by the gel LC-MS/MS approach. Thus, our method can be important for advancing our understanding of protein degradation in vivo. For instance, we noted that in a fourth aortic specimen, which was classified as normal upon aortotomy, the coefficient of variation for the number of fibronectin spectra in the NaCl extract exceeded 100%, implying a significant difference in the dispersion of the data among the analyzed samples (p ANOVA <0.05). On the other hand, the coefficient of variation for samples 1–3 is not significant (58%; p ANOVA >0.05). We also noted that there was an increase in fibronectin sequence coverage (Fig. 6, A and B) and total fibronectin spectra (∼4-fold) in the NaCl extract of the fourth specimen. Closer inspection of the molecular mass distribution of fibronectin spectra indicated the breakdown of fibronectin in this particular sample with more than 80% of fibronectin spectra identified in gel bands below 150 kDa. Western blotting using an antibody raised against full-length human fibronectin confirmed extensive laddering of the glycoprotein in both the NaCl and guanidine extracts (Fig. 6, C and D). Fibronectin degradation products have been implicated in the induction of proinflammatory responses in the vasculature and other tissues (1). Interestingly, mass spectrometry provided spectral evidence for MMP-9 only in this particular sample (Fig. 6E), and the presence of MMP-9 was confirmed by gelatin zymography (Fig. 6C, lower panel). Fibronectin is a known substrate for MMP-9 (27), but obviously, we cannot exclude that other proteinases contribute to the breakdown of fibronectin in this specimen in vivo. The example, however, demonstrates the potential of our methodology to analyze proteolytic activity based on the presence of proteolytic enzymes and corresponding degradation products in tissues. Importantly, this key information on protein breakdown within clinical samples that were considered normal macroscopically could be lost by using a conventional shotgun proteomics approach.Fig. 6.Detection of proteolysis. Percent sequence coverage for fibronectin in the NaCl (A) and guanidine extracts (B) is shown. “R” denotes RePlay analysis. Note the increase of fibronectin products in the NaCl extracts of the fourth aortic specimen. Western immunoblotting of fibronectin in the NaCl (C) and guanidine (D) extracts confirmed the laddering of fibronectin. The product ion spectrum of the doubly charged tryptic peptide SLGPALLLLQK was identified as MMP-9 (E) (peptide probability, 95%; SEQUEST XCorr score, 2.67; SEQUEST ΔCn score, 0.40; observed m/z, 576.87; actual peptide mass, 1,151.73 Da; peptide charge, 2; delta amu, 0.0049; delta ppm, 4.3; peptide start index, 66; peptide stop index, 76). MMP-9 was only detected by LC-MS/MS in the NaCl extract of the fourth aortic specimen. Gelatin zymography confirmed higher levels of MMP-9 in this sample (C, bottom panel).

Proteolysis

Finally, we investigated whether the NaCl extract represents a subproteome enriched in degradation products. Currently, our understanding of ECM proteolysis is based on in vitro experiments, which do not necessarily relate to in vivo processes. In our methodology, the original molecular mass distribution of the identified proteins is preserved by the gel LC-MS/MS approach. Thus, our method can be important for advancing our understanding of protein degradation in vivo. For instance, we noted that in a fourth aortic specimen, which was classified as normal upon aortotomy, the coefficient of variation for the number of fibronectin spectra in the NaCl extract exceeded 100%, implying a significant difference in the dispersion of the data among the analyzed samples (p ANOVA <0.05). On the other hand, the coefficient of variation for samples 1–3 is not significant (58%; p ANOVA >0.05). We also noted that there was an increase in fibronectin sequence coverage (Fig. 6, A and B) and total fibronectin spectra (∼4-fold) in the NaCl extract of the fourth specimen. Closer inspection of the molecular mass distribution of fibronectin spectra indicated the breakdown of fibronectin in this particular sample with more than 80% of fibronectin spectra identified in gel bands below 150 kDa. Western blotting using an antibody raised against full-length human fibronectin confirmed extensive laddering of the glycoprotein in both the NaCl and guanidine extracts (Fig. 6, C and D). Fibronectin degradation products have been implicated in the induction of proinflammatory responses in the vasculature and other tissues (1). Interestingly, mass spectrometry provided spectral evidence for MMP-9 only in this particular sample (Fig. 6E), and the presence of MMP-9 was confirmed by gelatin zymography (Fig. 6C, lower panel). Fibronectin is a known substrate for MMP-9 (27), but obviously, we cannot exclude that other proteinases contribute to the breakdown of fibronectin in this specimen in vivo. The example, however, demonstrates the potential of our methodology to analyze proteolytic activity based on the presence of proteolytic enzymes and corresponding degradation products in tissues. Importantly, this key information on protein breakdown within clinical samples that were considered normal macroscopically could be lost by using a conventional shotgun proteomics approach.

Detection of proteolysis. Percent sequence coverage for fibronectin in the NaCl (A) and guanidine extracts (B) is shown. “R” denotes RePlay analysis. Note the increase of fibronectin products in the NaCl extracts of the fourth aortic specimen. Western immunoblotting of fibronectin in the NaCl (C) and guanidine (D) extracts confirmed the laddering of fibronectin. The product ion spectrum of the doubly charged tryptic peptide SLGPALLLLQK was identified as MMP-9 (E) (peptide probability, 95%; SEQUEST XCorr score, 2.67; SEQUEST ΔCn score, 0.40; observed m/z, 576.87; actual peptide mass, 1,151.73 Da; peptide charge, 2; delta amu, 0.0049; delta ppm, 4.3; peptide start index, 66; peptide stop index, 76). MMP-9 was only detected by LC-MS/MS in the NaCl extract of the fourth aortic specimen. Gelatin zymography confirmed higher levels of MMP-9 in this sample (C, bottom panel).

DISCUSSIONIn this study, we combined biochemical fractionation with proteomics to comprehensively characterize the protein composition of the vascular extracellular environment. The main advantage of our methodology is the stepwise approach, which results in the isolation of proteins and protein degradation products loosely bound to the ECM in the NaCl fraction, the depletion of cellular proteins in the SDS fraction, and the effective solubilization of ECM proteins in the guanidine fraction.Comprehensive Matrix AnalysisUntil now, a comprehensive analysis of extracellular space components comprising the vascular proteome in humans has not been performed. Previous attempts to utilize the analytical power of proteomics to study ECM components and associated proteins have been limited by the fact that without prefractionation cellular and plasma proteins in mammalian tissues overwhelm the analytical capacities of the instruments and obscure the identification of scarce proteins in the extracellular environment. Therefore, it is not surprising that all extracellular space proteins found in the SDS extracts (except from type XIV collagen) have been reported before by proteomics studies of different vascular tissues.On the other hand, the tissue subfractionation resulted in the identification of numerous proteins in the NaCl and guanidine extracts, which have not been investigated in the vasculature until now. The relative insolubility and the extensive post-translational modifications, especially glycosylation, render ECM proteins difficult to analyze. For example, we identified and validated the presence of three glycoproteins: podocan, originally described as a glycoprotein secreted by kidney podocytes; sclerostin, another small glycoprotein, which has recently received attention because of its ability to act as a potent negative regulator of bone morphogenetic protein signaling and bone mineralization; and agrin, a large basement membrane proteoglycan, which participates in the organization of the synapses in the neuromuscular junction. We also identified and validated the presence of the atypical proteoglycan asporin in our aortic extracts.Moreover, the NaCl and guanidine extracts contained two ECM glycoproteins, tenascin-X and AEBP1. The first is believed to be a modulator of collagen fibrillogenesis (28). Interestingly, genetic deletion of tenascin-X in mice resulted in increased activity of MMP-2 and -9 (29). Less is known about AEBP1; the extracellular isoform was originally found to play a role in wound healing (30), and more recently, Majdalawieh et al. (31) showed that intracellular AEBP1 represses macrophage cholesterol efflux.Among collagens, collagen VI appears to be very abundant in our extracts. It serves mainly as a substrate for the attachment of cells and matrix molecules and also anchors basement membranes to underlying connective tissues (32). More interesting, however, is the identification of type XIV collagen (33). Previous proteomics studies probably failed to identify this collagen because it is heavily modified by chondroitin sulfate saccharides in vivo (34). These modifications greatly increase its size and alter its electrophoretic properties but are removed by the deglycosylation step in our extraction methodology.Finally, many scarce proteins, including LIM and cysteine-rich domains protein 1 (LMCD1), target of Nesh-SH3 (TARSH), and RPE-spondin (a previously hypothetical protein (35)), were all identified for the first time by proteomics in the vasculature, and representative MS/MS spectra are shown in supplemental Fig. 5. Their function in the vasculature is unknown.The subfractionation presented here could be an important tool for studying catabolic processes in tissues. In the NaCl fraction, we found fibronectin degradation products and the presence of MMP-2 and -9 as verified by gelatinolytic zymography. Mass spectrometry also identified two mast cell proteases, namely chymase and tryptase β-1. The first was recently found to participate in the development of mouse abdominal aneurysm (36), whereas the second, an activator of MMP-3 (37), has been shown to degrade the antiatherogenic high density lipoprotein (HDL) (38). Their presence in the aortic extracts might suggest an early involvement of mast cells in aortic pathology (39).Application to Clinical SamplesAs far as vascular research is concerned, the methodical and accurate characterization of the vascular extracellular environment is essential to advance our understanding of the ECM composition in different vascular territories and their predisposition to disease processes. For instance, atherosclerosis, the major cause for morbidity and mortality in the developed world, is intimately related to dynamic changes in the ECM and its associated proteins: the retention of lipoproteins by proteoglycans leads to the development of atherosclerotic lesions, but ultimately, it is the breakdown of ECM that facilitates plaque ruptures, triggering the onset of clinical symptoms. These changes, if comprehensively characterized, could lead to new therapeutic options for cardiovascular disease. Finally, because metabolic labeling cannot be used with clinical samples, spectral counting is an alternative method to assess relative protein abundance (40), but its accuracy is poor for proteins with low spectral counts (41).In conclusion, the proteomics method described allowed (a) the identification of scarce proteins present in the vascular extracellular space, (b) the identification of novel glycoproteins, and (c) the interrogation of proteolytic activity within tissues based on the identification of proteolytic enzymes and corresponding degradation products. Because ECM proteins accumulate in tissues, mRNA transcripts do not correspond to actual protein levels. In contrast, a proteomics-led approach has the potential to provide a more comprehensive analysis of the ECM composition and shed new light on pathological disease processes.

In this study, we combined biochemical fractionation with proteomics to comprehensively characterize the protein composition of the vascular extracellular environment. The main advantage of our methodology is the stepwise approach, which results in the isolation of proteins and protein degradation products loosely bound to the ECM in the NaCl fraction, the depletion of cellular proteins in the SDS fraction, and the effective solubilization of ECM proteins in the guanidine fraction.

Comprehensive Matrix AnalysisUntil now, a comprehensive analysis of extracellular space components comprising the vascular proteome in humans has not been performed. Previous attempts to utilize the analytical power of proteomics to study ECM components and associated proteins have been limited by the fact that without prefractionation cellular and plasma proteins in mammalian tissues overwhelm the analytical capacities of the instruments and obscure the identification of scarce proteins in the extracellular environment. Therefore, it is not surprising that all extracellular space proteins found in the SDS extracts (except from type XIV collagen) have been reported before by proteomics studies of different vascular tissues.On the other hand, the tissue subfractionation resulted in the identification of numerous proteins in the NaCl and guanidine extracts, which have not been investigated in the vasculature until now. The relative insolubility and the extensive post-translational modifications, especially glycosylation, render ECM proteins difficult to analyze. For example, we identified and validated the presence of three glycoproteins: podocan, originally described as a glycoprotein secreted by kidney podocytes; sclerostin, another small glycoprotein, which has recently received attention because of its ability to act as a potent negative regulator of bone morphogenetic protein signaling and bone mineralization; and agrin, a large basement membrane proteoglycan, which participates in the organization of the synapses in the neuromuscular junction. We also identified and validated the presence of the atypical proteoglycan asporin in our aortic extracts.Moreover, the NaCl and guanidine extracts contained two ECM glycoproteins, tenascin-X and AEBP1. The first is believed to be a modulator of collagen fibrillogenesis (28). Interestingly, genetic deletion of tenascin-X in mice resulted in increased activity of MMP-2 and -9 (29). Less is known about AEBP1; the extracellular isoform was originally found to play a role in wound healing (30), and more recently, Majdalawieh et al. (31) showed that intracellular AEBP1 represses macrophage cholesterol efflux.Among collagens, collagen VI appears to be very abundant in our extracts. It serves mainly as a substrate for the attachment of cells and matrix molecules and also anchors basement membranes to underlying connective tissues (32). More interesting, however, is the identification of type XIV collagen (33). Previous proteomics studies probably failed to identify this collagen because it is heavily modified by chondroitin sulfate saccharides in vivo (34). These modifications greatly increase its size and alter its electrophoretic properties but are removed by the deglycosylation step in our extraction methodology.Finally, many scarce proteins, including LIM and cysteine-rich domains protein 1 (LMCD1), target of Nesh-SH3 (TARSH), and RPE-spondin (a previously hypothetical protein (35)), were all identified for the first time by proteomics in the vasculature, and representative MS/MS spectra are shown in supplemental Fig. 5. Their function in the vasculature is unknown.The subfractionation presented here could be an important tool for studying catabolic processes in tissues. In the NaCl fraction, we found fibronectin degradation products and the presence of MMP-2 and -9 as verified by gelatinolytic zymography. Mass spectrometry also identified two mast cell proteases, namely chymase and tryptase β-1. The first was recently found to participate in the development of mouse abdominal aneurysm (36), whereas the second, an activator of MMP-3 (37), has been shown to degrade the antiatherogenic high density lipoprotein (HDL) (38). Their presence in the aortic extracts might suggest an early involvement of mast cells in aortic pathology (39).Application to Clinical SamplesAs far as vascular research is concerned, the methodical and accurate characterization of the vascular extracellular environment is essential to advance our understanding of the ECM composition in different vascular territories and their predisposition to disease processes. For instance, atherosclerosis, the major cause for morbidity and mortality in the developed world, is intimately related to dynamic changes in the ECM and its associated proteins: the retention of lipoproteins by proteoglycans leads to the development of atherosclerotic lesions, but ultimately, it is the breakdown of ECM that facilitates plaque ruptures, triggering the onset of clinical symptoms. These changes, if comprehensively characterized, could lead to new therapeutic options for cardiovascular disease. Finally, because metabolic labeling cannot be used with clinical samples, spectral counting is an alternative method to assess relative protein abundance (40), but its accuracy is poor for proteins with low spectral counts (41).In conclusion, the proteomics method described allowed (a) the identification of scarce proteins present in the vascular extracellular space, (b) the identification of novel glycoproteins, and (c) the interrogation of proteolytic activity within tissues based on the identification of proteolytic enzymes and corresponding degradation products. Because ECM proteins accumulate in tissues, mRNA transcripts do not correspond to actual protein levels. In contrast, a proteomics-led approach has the potential to provide a more comprehensive analysis of the ECM composition and shed new light on pathological disease processes.

Comprehensive Matrix AnalysisUntil now, a comprehensive analysis of extracellular space components comprising the vascular proteome in humans has not been performed. Previous attempts to utilize the analytical power of proteomics to study ECM components and associated proteins have been limited by the fact that without prefractionation cellular and plasma proteins in mammalian tissues overwhelm the analytical capacities of the instruments and obscure the identification of scarce proteins in the extracellular environment. Therefore, it is not surprising that all extracellular space proteins found in the SDS extracts (except from type XIV collagen) have been reported before by proteomics studies of different vascular tissues.On the other hand, the tissue subfractionation resulted in the identification of numerous proteins in the NaCl and guanidine extracts, which have not been investigated in the vasculature until now. The relative insolubility and the extensive post-translational modifications, especially glycosylation, render ECM proteins difficult to analyze. For example, we identified and validated the presence of three glycoproteins: podocan, originally described as a glycoprotein secreted by kidney podocytes; sclerostin, another small glycoprotein, which has recently received attention because of its ability to act as a potent negative regulator of bone morphogenetic protein signaling and bone mineralization; and agrin, a large basement membrane proteoglycan, which participates in the organization of the synapses in the neuromuscular junction. We also identified and validated the presence of the atypical proteoglycan asporin in our aortic extracts.Moreover, the NaCl and guanidine extracts contained two ECM glycoproteins, tenascin-X and AEBP1. The first is believed to be a modulator of collagen fibrillogenesis (28). Interestingly, genetic deletion of tenascin-X in mice resulted in increased activity of MMP-2 and -9 (29). Less is known about AEBP1; the extracellular isoform was originally found to play a role in wound healing (30), and more recently, Majdalawieh et al. (31) showed that intracellular AEBP1 represses macrophage cholesterol efflux.Among collagens, collagen VI appears to be very abundant in our extracts. It serves mainly as a substrate for the attachment of cells and matrix molecules and also anchors basement membranes to underlying connective tissues (32). More interesting, however, is the identification of type XIV collagen (33). Previous proteomics studies probably failed to identify this collagen because it is heavily modified by chondroitin sulfate saccharides in vivo (34). These modifications greatly increase its size and alter its electrophoretic properties but are removed by the deglycosylation step in our extraction methodology.Finally, many scarce proteins, including LIM and cysteine-rich domains protein 1 (LMCD1), target of Nesh-SH3 (TARSH), and RPE-spondin (a previously hypothetical protein (35)), were all identified for the first time by proteomics in the vasculature, and representative MS/MS spectra are shown in supplemental Fig. 5. Their function in the vasculature is unknown.The subfractionation presented here could be an important tool for studying catabolic processes in tissues. In the NaCl fraction, we found fibronectin degradation products and the presence of MMP-2 and -9 as verified by gelatinolytic zymography. Mass spectrometry also identified two mast cell proteases, namely chymase and tryptase β-1. The first was recently found to participate in the development of mouse abdominal aneurysm (36), whereas the second, an activator of MMP-3 (37), has been shown to degrade the antiatherogenic high density lipoprotein (HDL) (38). Their presence in the aortic extracts might suggest an early involvement of mast cells in aortic pathology (39).Application to Clinical SamplesAs far as vascular research is concerned, the methodical and accurate characterization of the vascular extracellular environment is essential to advance our understanding of the ECM composition in different vascular territories and their predisposition to disease processes. For instance, atherosclerosis, the major cause for morbidity and mortality in the developed world, is intimately related to dynamic changes in the ECM and its associated proteins: the retention of lipoproteins by proteoglycans leads to the development of atherosclerotic lesions, but ultimately, it is the breakdown of ECM that facilitates plaque ruptures, triggering the onset of clinical symptoms. These changes, if comprehensively characterized, could lead to new therapeutic options for cardiovascular disease. Finally, because metabolic labeling cannot be used with clinical samples, spectral counting is an alternative method to assess relative protein abundance (40), but its accuracy is poor for proteins with low spectral counts (41).In conclusion, the proteomics method described allowed (a) the identification of scarce proteins present in the vascular extracellular space, (b) the identification of novel glycoproteins, and (c) the interrogation of proteolytic activity within tissues based on the identification of proteolytic enzymes and corresponding degradation products. Because ECM proteins accumulate in tissues, mRNA transcripts do not correspond to actual protein levels. In contrast, a proteomics-led approach has the potential to provide a more comprehensive analysis of the ECM composition and shed new light on pathological disease processes.

Comprehensive Matrix AnalysisUntil now, a comprehensive analysis of extracellular space components comprising the vascular proteome in humans has not been performed. Previous attempts to utilize the analytical power of proteomics to study ECM components and associated proteins have been limited by the fact that without prefractionation cellular and plasma proteins in mammalian tissues overwhelm the analytical capacities of the instruments and obscure the identification of scarce proteins in the extracellular environment. Therefore, it is not surprising that all extracellular space proteins found in the SDS extracts (except from type XIV collagen) have been reported before by proteomics studies of different vascular tissues.On the other hand, the tissue subfractionation resulted in the identification of numerous proteins in the NaCl and guanidine extracts, which have not been investigated in the vasculature until now. The relative insolubility and the extensive post-translational modifications, especially glycosylation, render ECM proteins difficult to analyze. For example, we identified and validated the presence of three glycoproteins: podocan, originally described as a glycoprotein secreted by kidney podocytes; sclerostin, another small glycoprotein, which has recently received attention because of its ability to act as a potent negative regulator of bone morphogenetic protein signaling and bone mineralization; and agrin, a large basement membrane proteoglycan, which participates in the organization of the synapses in the neuromuscular junction. We also identified and validated the presence of the atypical proteoglycan asporin in our aortic extracts.Moreover, the NaCl and guanidine extracts contained two ECM glycoproteins, tenascin-X and AEBP1. The first is believed to be a modulator of collagen fibrillogenesis (28). Interestingly, genetic deletion of tenascin-X in mice resulted in increased activity of MMP-2 and -9 (29). Less is known about AEBP1; the extracellular isoform was originally found to play a role in wound healing (30), and more recently, Majdalawieh et al. (31) showed that intracellular AEBP1 represses macrophage cholesterol efflux.Among collagens, collagen VI appears to be very abundant in our extracts. It serves mainly as a substrate for the attachment of cells and matrix molecules and also anchors basement membranes to underlying connective tissues (32). More interesting, however, is the identification of type XIV collagen (33). Previous proteomics studies probably failed to identify this collagen because it is heavily modified by chondroitin sulfate saccharides in vivo (34). These modifications greatly increase its size and alter its electrophoretic properties but are removed by the deglycosylation step in our extraction methodology.Finally, many scarce proteins, including LIM and cysteine-rich domains protein 1 (LMCD1), target of Nesh-SH3 (TARSH), and RPE-spondin (a previously hypothetical protein (35)), were all identified for the first time by proteomics in the vasculature, and representative MS/MS spectra are shown in supplemental Fig. 5. Their function in the vasculature is unknown.The subfractionation presented here could be an important tool for studying catabolic processes in tissues. In the NaCl fraction, we found fibronectin degradation products and the presence of MMP-2 and -9 as verified by gelatinolytic zymography. Mass spectrometry also identified two mast cell proteases, namely chymase and tryptase β-1. The first was recently found to participate in the development of mouse abdominal aneurysm (36), whereas the second, an activator of MMP-3 (37), has been shown to degrade the antiatherogenic high density lipoprotein (HDL) (38). Their presence in the aortic extracts might suggest an early involvement of mast cells in aortic pathology (39).

Comprehensive Matrix Analysis

Until now, a comprehensive analysis of extracellular space components comprising the vascular proteome in humans has not been performed. Previous attempts to utilize the analytical power of proteomics to study ECM components and associated proteins have been limited by the fact that without prefractionation cellular and plasma proteins in mammalian tissues overwhelm the analytical capacities of the instruments and obscure the identification of scarce proteins in the extracellular environment. Therefore, it is not surprising that all extracellular space proteins found in the SDS extracts (except from type XIV collagen) have been reported before by proteomics studies of different vascular tissues.

On the other hand, the tissue subfractionation resulted in the identification of numerous proteins in the NaCl and guanidine extracts, which have not been investigated in the vasculature until now. The relative insolubility and the extensive post-translational modifications, especially glycosylation, render ECM proteins difficult to analyze. For example, we identified and validated the presence of three glycoproteins: podocan, originally described as a glycoprotein secreted by kidney podocytes; sclerostin, another small glycoprotein, which has recently received attention because of its ability to act as a potent negative regulator of bone morphogenetic protein signaling and bone mineralization; and agrin, a large basement membrane proteoglycan, which participates in the organization of the synapses in the neuromuscular junction. We also identified and validated the presence of the atypical proteoglycan asporin in our aortic extracts.

Moreover, the NaCl and guanidine extracts contained two ECM glycoproteins, tenascin-X and AEBP1. The first is believed to be a modulator of collagen fibrillogenesis (28). Interestingly, genetic deletion of tenascin-X in mice resulted in increased activity of MMP-2 and -9 (29). Less is known about AEBP1; the extracellular isoform was originally found to play a role in wound healing (30), and more recently, Majdalawieh et al. (31) showed that intracellular AEBP1 represses macrophage cholesterol efflux.

Among collagens, collagen VI appears to be very abundant in our extracts. It serves mainly as a substrate for the attachment of cells and matrix molecules and also anchors basement membranes to underlying connective tissues (32). More interesting, however, is the identification of type XIV collagen (33). Previous proteomics studies probably failed to identify this collagen because it is heavily modified by chondroitin sulfate saccharides in vivo (34). These modifications greatly increase its size and alter its electrophoretic properties but are removed by the deglycosylation step in our extraction methodology.

Finally, many scarce proteins, including LIM and cysteine-rich domains protein 1 (LMCD1), target of Nesh-SH3 (TARSH), and RPE-spondin (a previously hypothetical protein (35)), were all identified for the first time by proteomics in the vasculature, and representative MS/MS spectra are shown in supplemental Fig. 5. Their function in the vasculature is unknown.

The subfractionation presented here could be an important tool for studying catabolic processes in tissues. In the NaCl fraction, we found fibronectin degradation products and the presence of MMP-2 and -9 as verified by gelatinolytic zymography. Mass spectrometry also identified two mast cell proteases, namely chymase and tryptase β-1. The first was recently found to participate in the development of mouse abdominal aneurysm (36), whereas the second, an activator of MMP-3 (37), has been shown to degrade the antiatherogenic high density lipoprotein (HDL) (38). Their presence in the aortic extracts might suggest an early involvement of mast cells in aortic pathology (39).

Application to Clinical SamplesAs far as vascular research is concerned, the methodical and accurate characterization of the vascular extracellular environment is essential to advance our understanding of the ECM composition in different vascular territories and their predisposition to disease processes. For instance, atherosclerosis, the major cause for morbidity and mortality in the developed world, is intimately related to dynamic changes in the ECM and its associated proteins: the retention of lipoproteins by proteoglycans leads to the development of atherosclerotic lesions, but ultimately, it is the breakdown of ECM that facilitates plaque ruptures, triggering the onset of clinical symptoms. These changes, if comprehensively characterized, could lead to new therapeutic options for cardiovascular disease. Finally, because metabolic labeling cannot be used with clinical samples, spectral counting is an alternative method to assess relative protein abundance (40), but its accuracy is poor for proteins with low spectral counts (41).In conclusion, the proteomics method described allowed (a) the identification of scarce proteins present in the vascular extracellular space, (b) the identification of novel glycoproteins, and (c) the interrogation of proteolytic activity within tissues based on the identification of proteolytic enzymes and corresponding degradation products. Because ECM proteins accumulate in tissues, mRNA transcripts do not correspond to actual protein levels. In contrast, a proteomics-led approach has the potential to provide a more comprehensive analysis of the ECM composition and shed new light on pathological disease processes.

Application to Clinical Samples

As far as vascular research is concerned, the methodical and accurate characterization of the vascular extracellular environment is essential to advance our understanding of the ECM composition in different vascular territories and their predisposition to disease processes. For instance, atherosclerosis, the major cause for morbidity and mortality in the developed world, is intimately related to dynamic changes in the ECM and its associated proteins: the retention of lipoproteins by proteoglycans leads to the development of atherosclerotic lesions, but ultimately, it is the breakdown of ECM that facilitates plaque ruptures, triggering the onset of clinical symptoms. These changes, if comprehensively characterized, could lead to new therapeutic options for cardiovascular disease. Finally, because metabolic labeling cannot be used with clinical samples, spectral counting is an alternative method to assess relative protein abundance (40), but its accuracy is poor for proteins with low spectral counts (41).

In conclusion, the proteomics method described allowed (a) the identification of scarce proteins present in the vascular extracellular space, (b) the identification of novel glycoproteins, and (c) the interrogation of proteolytic activity within tissues based on the identification of proteolytic enzymes and corresponding degradation products. Because ECM proteins accumulate in tissues, mRNA transcripts do not correspond to actual protein levels. In contrast, a proteomics-led approach has the potential to provide a more comprehensive analysis of the ECM composition and shed new light on pathological disease processes.

Supplementary MaterialSupplemental Data

Supplementary Material

Supplemental Data
