---
pmid: '33147445'
title: Identification of Required Host Factors for SARS-CoV-2 Infection in Human Cells.
authors:
- Daniloski Z
- Jordan TX
- Wessels HH
- Hoagland DA
- Kasela S
- Legut M
- Maniatis S
- Mimitou EP
- Lu L
- Geller E
- Danziger O
- Rosenberg BR
- Phatnani H
- Smibert P
- Lappalainen T
- tenOever BR
- Sanjana NE
journal: Cell
year: '2021'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC7584921
doi: 10.1016/j.cell.2020.10.030
---

# Identification of Required Host Factors for SARS-CoV-2 Infection in Human Cells.
**Authors:** Daniloski Z, Jordan TX, Wessels HH, Hoagland DA, Kasela S, Legut M, Maniatis S, Mimitou EP, Lu L, Geller E, Danziger O, Rosenberg BR, Phatnani H, Smibert P, Lappalainen T, tenOever BR, Sanjana NE
**Journal:** Cell (2021)
**DOI:** [10.1016/j.cell.2020.10.030](https://doi.org/10.1016/j.cell.2020.10.030)
**PMC:** [PMC7584921](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7584921/)

## Abstract

1. Cell. 2021 Jan 7;184(1):92-105.e16. doi: 10.1016/j.cell.2020.10.030. Epub 2020
 Oct 24.

Identification of Required Host Factors for SARS-CoV-2 Infection in Human Cells.

Daniloski Z(1), Jordan TX(2), Wessels HH(1), Hoagland DA(2), Kasela S(3), Legut 
M(1), Maniatis S(4), Mimitou EP(5), Lu L(1), Geller E(1), Danziger O(2), 
Rosenberg BR(2), Phatnani H(6), Smibert P(5), Lappalainen T(3), tenOever BR(7), 
Sanjana NE(8).

Author information:
(1)New York Genome Center, New York, NY, USA; Department of Biology, New York 
University, New York, NY, USA.
(2)Department of Microbiology, Icahn School of Medicine at Mount Sinai, New 
York, NY, USA.
(3)New York Genome Center, New York, NY, USA; Department of Systems Biology, 
Columbia University, New York, NY, USA.
(4)New York Genome Center, New York, NY, USA.
(5)Technology Innovation Lab, New York Genome Center, New York, NY, USA.
(6)New York Genome Center, New York, NY, USA; Department of Neurology, Columbia 
University Irving Medical Center, New York, NY, USA.
(7)Department of Microbiology, Icahn School of Medicine at Mount Sinai, New 
York, NY, USA. Electronic address: benjamin.tenoever@mssm.edu.
(8)New York Genome Center, New York, NY, USA; Department of Biology, New York 
University, New York, NY, USA. Electronic address: neville@sanjanalab.org.

Comment in
    Nat Rev Genet. 2021 Jan;22(1):1. doi: 10.1038/s41576-020-00310-y.
    Nat Rev Mol Cell Biol. 2021 Jan;22(1):2. doi: 10.1038/s41580-020-00319-5.
    Cell. 2021 Jan 7;184(1):15-17. doi: 10.1016/j.cell.2020.12.003.
    J Infect. 2022 Mar;84(3):418-467. doi: 10.1016/j.jinf.2021.12.011.

To better understand host-virus genetic dependencies and find potential 
therapeutic targets for COVID-19, we performed a genome-scale CRISPR 
loss-of-function screen to identify host factors required for SARS-CoV-2 viral 
infection of human alveolar epithelial cells. Top-ranked genes cluster into 
distinct pathways, including the vacuolar ATPase proton pump, Retromer, and 
Commander complexes. We validate these gene targets using several orthogonal 
methods such as CRISPR knockout, RNA interference knockdown, and small-molecule 
inhibitors. Using single-cell RNA-sequencing, we identify shared transcriptional 
changes in cholesterol biosynthesis upon loss of top-ranked genes. In addition, 
given the key role of the ACE2 receptor in the early stages of viral entry, we 
show that loss of RAB7A reduces viral entry by sequestering the ACE2 receptor 
inside cells. Overall, this work provides a genome-scale, quantitative resource 
of the impact of the loss of each host gene on fitness/response to viral 
infection.

Copyright © 2020 Elsevier Inc. All rights reserved.

DOI: 10.1016/j.cell.2020.10.030
PMCID: PMC7584921
PMID: 33147445 [Indexed for MEDLINE]

Conflict of interest statement: Declaration of Interests N.E.S. is an advisor to 
Vertex. T.L. is a member of the scientific advisory boards of Goldfinch Bio and, 
with equity, Variant Bio. The New York Genome Center and New York University 
have applied for patents relating to the work in this article.

## Full Text

IntroductionAs of October 2020, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), the virus that causes COVID-19, has infected 40 million people worldwide and led to the deaths of more than 1 million people, according to the John Hopkins Research Center (Gardner, 2020). SARS-CoV-2 belongs to the family of enveloped viruses known as Coronaviridae and was first reported in late 2019 in China. Over the past two decades, it is the third zoonotic coronavirus to emerge: compared to the other two coronaviruses, SARS-CoV (2002) and Middle East respiratory syndrome (MERS)-CoV (2012), SARS-CoV-2 shows an increased infectivity and lower case-fatality rate, contributing to its wide-spread transmission and resulting in a pandemic (Gates, 2020; Liu et al., 2020). Given that SARS-CoV-2 has already taken a major toll on human life and livelihoods worldwide, many research institutions, governmental organizations, and pharmaceutical companies are working to identify antiviral drugs and develop vaccines. Currently, there are nearly 30 vaccines against SARS-CoV-2 in clinical trials and a Food and Drug Administration (FDA)-approved antiviral drug (remdesivir) that acts as an inhibitor of the SARS-CoV-2 viral RNA-dependent RNA polymerase (Beigel et al., 2020; Funk et al., 2020). A recent study identified small molecules that antagonize SARS-CoV-2 replication and infection by testing ∼12,000 clinical-stage and FDA-approved inhibitors (Riva et al., 2020). Here, we utilize an alternative approach—a genome-scale loss-of-function screen—to identify targets among host genes that are required for SARS-CoV-2 infection. These gene targets (and inhibitors of these genes) may aid in the development of new therapies for COVID-19.SARS-CoV-2 is an enveloped positive-sense RNA virus that relies on host factors for all stages of its life cycle (Kim et al., 2020; Zhou et al., 2020). The viral envelope is coated by Spike protein trimers that bind to angiotensin converting-enzyme 2 (ACE2) receptor, which is required for SARS-CoV-2 infection (Hoffmann et al., 2020a; Zhou et al., 2020). The Spike protein undergoes proteolytic cleavage that is catalyzed by several host proteases, such as furin, TMPRSS2, and cathepsin L, and can occur in the secretory pathway of the host cell or during viral entry in the target cell. Proteolytic cleavage is considered to be required for activation of Spike that in turn allows for viral-host membrane fusion and release of the viral RNA into the host cytoplasm (Hoffmann et al., 2020b). Once in the cytoplasm, the virus utilizes the host and its own machinery to replicate its genetic material and assemble new viral particles. Recent proteomic studies have identified hundreds of host proteins that directly bind to SARS-CoV-2 viral proteins and have mapped changes in the global protein phosphorylation landscape in response to viral infection, highlighting the interest in better understanding of host-virus genetic dependencies (Bouhaddou et al., 2020; Gordon et al., 2020). To date, there are no genome-wide studies that directly identify human genes required for viral infection, which will be of great interest and utility for the broader scientific community.Here, we perform a genome-scale CRISPR loss-of-function screen in human alveolar basal epithelial carcinoma cells to identify genes whose loss confers resistance to SARS-CoV-2 viral infection. We validate that these genes reduce SARS-CoV-2 infection using multiple orthogonal cell perturbations (CRISPR knockout, RNA interference knockdown, and small-molecule inhibitors). For the top gene hits, we explore potential mechanisms of their antiviral activity using single-cell transcriptomics, flow cytometry, and immunofluorescence. Using single-cell transcriptomics, we identified a group of genes (ATP6AP1, ATP6V1A, NPC1, RAB7A, CCDC22, and PIK3C3) whose knockout induced shared transcriptional changes in cholesterol biosynthesis pathway. Perturbation of the cholesterol biosynthesis pathway with the small molecule amlodipine reduced viral infection. In addition, we show that loss of RAB7A reduces viral entry by sequestering ACE2 receptors inside cells through altered endosomal trafficking. Prior to this study, our knowledge of essential host genes for SARS-CoV-2 has been limited to only a handful of genes, such as ACE2 and cathepsin L: this work provides a quantitative resource of the impact of each gene’s loss on response to viral infection for every protein-coding gene in the human genome.

Introduction

As of October 2020, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), the virus that causes COVID-19, has infected 40 million people worldwide and led to the deaths of more than 1 million people, according to the John Hopkins Research Center (Gardner, 2020). SARS-CoV-2 belongs to the family of enveloped viruses known as Coronaviridae and was first reported in late 2019 in China. Over the past two decades, it is the third zoonotic coronavirus to emerge: compared to the other two coronaviruses, SARS-CoV (2002) and Middle East respiratory syndrome (MERS)-CoV (2012), SARS-CoV-2 shows an increased infectivity and lower case-fatality rate, contributing to its wide-spread transmission and resulting in a pandemic (Gates, 2020; Liu et al., 2020). Given that SARS-CoV-2 has already taken a major toll on human life and livelihoods worldwide, many research institutions, governmental organizations, and pharmaceutical companies are working to identify antiviral drugs and develop vaccines. Currently, there are nearly 30 vaccines against SARS-CoV-2 in clinical trials and a Food and Drug Administration (FDA)-approved antiviral drug (remdesivir) that acts as an inhibitor of the SARS-CoV-2 viral RNA-dependent RNA polymerase (Beigel et al., 2020; Funk et al., 2020). A recent study identified small molecules that antagonize SARS-CoV-2 replication and infection by testing ∼12,000 clinical-stage and FDA-approved inhibitors (Riva et al., 2020). Here, we utilize an alternative approach—a genome-scale loss-of-function screen—to identify targets among host genes that are required for SARS-CoV-2 infection. These gene targets (and inhibitors of these genes) may aid in the development of new therapies for COVID-19.

SARS-CoV-2 is an enveloped positive-sense RNA virus that relies on host factors for all stages of its life cycle (Kim et al., 2020; Zhou et al., 2020). The viral envelope is coated by Spike protein trimers that bind to angiotensin converting-enzyme 2 (ACE2) receptor, which is required for SARS-CoV-2 infection (Hoffmann et al., 2020a; Zhou et al., 2020). The Spike protein undergoes proteolytic cleavage that is catalyzed by several host proteases, such as furin, TMPRSS2, and cathepsin L, and can occur in the secretory pathway of the host cell or during viral entry in the target cell. Proteolytic cleavage is considered to be required for activation of Spike that in turn allows for viral-host membrane fusion and release of the viral RNA into the host cytoplasm (Hoffmann et al., 2020b). Once in the cytoplasm, the virus utilizes the host and its own machinery to replicate its genetic material and assemble new viral particles. Recent proteomic studies have identified hundreds of host proteins that directly bind to SARS-CoV-2 viral proteins and have mapped changes in the global protein phosphorylation landscape in response to viral infection, highlighting the interest in better understanding of host-virus genetic dependencies (Bouhaddou et al., 2020; Gordon et al., 2020). To date, there are no genome-wide studies that directly identify human genes required for viral infection, which will be of great interest and utility for the broader scientific community.

Here, we perform a genome-scale CRISPR loss-of-function screen in human alveolar basal epithelial carcinoma cells to identify genes whose loss confers resistance to SARS-CoV-2 viral infection. We validate that these genes reduce SARS-CoV-2 infection using multiple orthogonal cell perturbations (CRISPR knockout, RNA interference knockdown, and small-molecule inhibitors). For the top gene hits, we explore potential mechanisms of their antiviral activity using single-cell transcriptomics, flow cytometry, and immunofluorescence. Using single-cell transcriptomics, we identified a group of genes (ATP6AP1, ATP6V1A, NPC1, RAB7A, CCDC22, and PIK3C3) whose knockout induced shared transcriptional changes in cholesterol biosynthesis pathway. Perturbation of the cholesterol biosynthesis pathway with the small molecule amlodipine reduced viral infection. In addition, we show that loss of RAB7A reduces viral entry by sequestering ACE2 receptors inside cells through altered endosomal trafficking. Prior to this study, our knowledge of essential host genes for SARS-CoV-2 has been limited to only a handful of genes, such as ACE2 and cathepsin L: this work provides a quantitative resource of the impact of each gene’s loss on response to viral infection for every protein-coding gene in the human genome.

ResultsA High-Throughput Screen to Identify Genes Required for SARS-CoV-2 InfectionTo identify key genes required for SARS-CoV-2 infection, we performed a genome-scale loss-of-function screen targeting 19,050 genes in the human genome using the GeCKOv2 CRISPR-Cas9 library (Sanjana et al., 2014). The GeCKOv2 library contains 122,411 CRISPR single-guide RNAs (sgRNAs) (6 guide RNAs per gene) and has previously been used in CRISPR screens for drug resistance, immunotherapy, synthetic lethality, mitochondrial disease, and therapeutic discovery for muscular dystrophy (Erb et al., 2017; Jain et al., 2016; Lek et al., 2020; Patel et al., 2017; Shalem et al., 2014). First, we transduced a human alveolar basal epithelial carcinoma cell line (A549) that constitutively expresses ACE2 (referred to as A549ACE2) with an all-in-one lentiviral vector containing Cas9, guide RNAs from the GeCKOv2 human library, and a puromycin resistance gene. The transduction was performed at a low multiplicity of infection (MOI ∼0.2) to ensure that most cells would receive only one guide RNA construct (Figure 1
A). We then selected with puromycin so that only library-transduced cells remained. We also measured the survival rate after puromycin selection was complete (3 days) to ensure high coverage of the 122,411 guide RNAs (∼1,000 cells per guide RNA). After puromycin selection was complete, we cultured the cells for 9 days to ensure protein depletion after CRISPR gene targeting.Figure 1A Genome-Scale CRISPR Loss-of-Function Screen to Identify Genes that Prevent SARS-CoV-2 Infection of Human Alveolar Epithelial Carcinoma Cells(A) Overview of the genome-scale loss-of-function screen for host factors in human A549ACE2 cells requires for SARS-CoV-2 infection.(B) Immunofluorescence of SARS-CoV-2 nucleocapsid (N) protein and DAPI labeling of human A549ACE2 cells at 24 h post-infection.(C) Percent survival of human A549ACE2 cells transduced with the GeCKOv2 library with the indicated SARS-CoV-2 viral amount (MOI) at 6 days post-infection.(D) Scatterplot of guide RNA read counts from A549ACE2 cells at 6 days post-infection with SARS-CoV-2 (MOI ~0.01) versus cells prior to infection. Read counts are normalized log2 reads.(E) Volcano plot of median fold-change of guide RNAs for each gene and log10 robust rank aggregation (RRA) p values. All genes with |fold-change| > 4 and RRA p < 10−3 are labeled.(F) Overlap of top 50 ranked genes between the MOI 0.01 and MOI 0.3 screen.See also Figure S1 and Table S1.Next, we infected the GeCKOv2 pool of A549ACE2 cells with SARS-CoV-2 virus (Isolate USA-WA1/2020 NR-52281) at either a high (0.3) or a low (0.01) MOI. We verified that SARS-CoV-2 infects A549ACE2 cells by staining for the nucleocapsid (N) protein at 24 h post-infection (Figure 1B), and at day 6 post-infection, we measured cell survival for both the high and low MOI conditions (Figure 1C). As expected, the higher MOI infection resulted in fewer surviving cells at day 6 post-infection. Next, we extracted genomic DNA, and via amplicon sequencing, we quantified guide abundance in each biological condition (Figure 1A). To confirm that library representation was properly maintained, we computed the correlation between the guide representation in the plasmid library and after puromycin selection (r = 0.84) (Figure S1
A). In contrast, after SARS-CoV-2 infection, there was a much greater degree of guide dropout, as expected given that SARS-CoV-2 rapidly kills A549ACE2 cells without CRISPR perturbations (Figures 1C and 1D).Figure S1Genome-wide Loss-of-Function CRISPR Screen Enriched Gene Identification, Related to Figure 1(A) Scatterplot of guide RNA read counts from A549ACE2 cells at 3 days post-transduction with the GeCKOv2 library versus read counts from the library plasmid. Read counts ar4e normalized log2 reads. (B) RRA p-value distribution for all genes in the GeCKOv2 library. (C) Overlap of top-ranked (top 500) genes between 3 different analysis methods (RRA, RIGER, and SBR). 142 genes are found by all 3 methods. (D) Comparison of top-ranked genes between different genome-scale CRISPR screens for SARS-CoV-2 infection.Using robust-rank aggregation (RRA) on the guide relative enrichments, we computed gene-level scores to identify genes where loss-of-function mutations led to enrichment within the pool (Figure 1E; Kolde et al., 2012). We identified ∼1,000 genes with significant RRA enrichment (p < 0.05) (Figure S1B). We also used two other previously published methods to compute gene enrichments (RIGER weighted-sum and second-best rank) and found a high degree of overlap between enriched genes identified by all three methods (Figure S1C; Chen et al., 2015; Luo et al., 2008). We also found a high degree of shared genes across both the low and high SARS-CoV-2 MOI conditions: when examining the top 50 most enriched genes, we found that 27 of them were shared between the low and high MOI conditions (Figure 1F; Table S1), suggesting that several host genes involved in viral pathogenesis function independently of viral dose. An independent genome-scale CRISPR screen for SARS-CoV-2 infection also performed in A549 that overexpress ACE2 but with a different CRISPR library identified similar top-ranked genes (Zhu et al., 2020), highlighting the robustness of our results (Figure S1D).Enriched Genes Are Involved in Multiple Aspects of the Viral Life Cycle and Are Broadly ExpressedUpon close examination of the most enriched genes, we found genes involved in key aspects of viral entry and replication (Figure 2
; Du et al., 2009). For example, the well-established entry receptor angiotensin-converting enzyme 2 (ACE2) receptor was ranked as the 8th most-enriched gene in the low MOI screen and 12th in the high MOI screen (Table S1; Hoffmann et al., 2020a; Zhou et al., 2020). Among the top 50 enriched genes, we identified several sets of related genes that function together in complexes, giving us further confidence in the genome-scale screen (Figures 2 and 3A). We found genes essential for initial attachment and endocytosis (ACE2, RAB7A, and 4 members of the ARP2/3 complex: ACTR2, ACTR3, ARPC3, and ARPC4), Spike protein cleavage and viral membrane fusion (CTSL and 13 members of the vacuolar-ATPase proton pump: ATP6AP1, ATP6AP2, ATP6V0B ATP6V0C, ATP6V0D1, ATP6V1A, ATP6V1B2, ATP6V1C1, ATP6V1E1, ATP6V1G1, ATP6V1H, TMEM199, and TOR1AIP1), endosome recycling (4 members of the endosomal protein sorting Retromer complex: VPS26A, VPS29, VPS35, and SNX27; 4 members of the endosomal trafficking Commander complex: COMMD2, COMMD3, COMMD3-BMI1, and COMMD4; and 3 members of the PI3K pathway: PIK3C3/VPS34, WDR81, and ACP5), ER-Golgi trafficking (DPM3, ERMP1, PPID, and CHST14), and transcriptional modulators (SLTM and SPEN). A consistent theme among the enriched complexes is endosome function and regulation (V-ATPase proton pump, Retromer, Commander, class 3 PI3Ks) (Banerjee and Kane, 2020; Mallam and Marcotte, 2017; McNally and Cullen, 2018). Gene set enrichment analysis on the full ranked list of genes identified significantly enriched Gene Ontology (GO) categories for endosome processing, transport, and acidification and categories related to cytokinesis and virion attachment (false discovery rate [FDR] q < 0.1) (Figures 3B and S2A–S2D; Table S2; Subramanian et al., 2005).Figure 2Top-Ranked Genes from the CRISPR Screen Are Involved in Key Elements of the SARS-CoV-2 Viral Life CycleSchematic of SARS-CoV-2 docking, entry, RNA genome release and transcription, and virion replication, assembly, and release with top-ranked host genes from the CRISPR screen highlighted in red. All genes shown are ranked in the top 50 genes (top ~0.25% of library) in the low MOI CRISPR screen using RRA. Adapted from Du et al. (2009).See also Figure S2 and Table S2.Figure 3Enriched Genes Cluster into Related Pathways, Are Expressed Broadly, Interact Directly with Viral Proteins, and Are Also Involved in Viral Pathogenesis of Pandemic Flu and Zika Virus(A) Classification of genes shown in Figure 2 (top-ranked ~0.25% of the GeCKOv2 library) into specific complexes.(B) Gene set enrichment analysis normalized enrichment scores for all significant (FDR q < 0.1) Gene Ontology (GO) biological processes.(C) Expression of top-ranked genes (same as in A) across the indicated human tissues from GTEx v8. Gene expression color scale is transcripts per million (TPM).(D) RRA fold-change for the low MOI CRISPR screen for the high-confidence protein-protein interaction with the maximum fold-change for each viral gene from the Gordon et al. (2020) mass spectrometry dataset.(E) Clustering of top-ranked GO biological processes for CRISPR screens for Zika virus ZIKV (Li et al., 2019), H1N1 pandemic avian influenza IAV (Li et al., 2020), and SARS-CoV-2 (this study).See also Figure S2 and Table S2.Figure S2Gene Set Enrichment and Overlap of Top-Ranked Genes with Other Viral Infections, Related to Figures 2 and 3(A) – (D) Four of the significant (FDR < 0.1) top-ranked GO biological process terms and the fold-change ranks of their genes in the SARS-CoV-2 low MOI CRISPR screen. (E) Normalized gene ranks of the top 50 genes from the SARS-CoV-2 low MOI CRISPR screen and genome-scale CRISPR screens for Zika virus (ZIKV) and H1N1 avian influenza (IAV).Although we performed our CRISPR screen in human lung cells, we explored whether the expression of host genes whose loss reduces SARS-CoV-2 infection were lung-specific or more broadly expressed. To answer this question, we took the top-ranked genes and examined their expression across 12 tissues using 4,790 RNA-sequencing datasets from the Genotype-Tissue Expression (GTEx) v8 database (Figure 3C; Aguet et al., 2019). Virtually all of the top gene hits were broadly expressed across all tissues, implying that these mechanisms may function independent of cell or tissue type. Among the top-ranked genes, only ACE2 showed tissue-specific expression with a particular enrichment in testis, small intestine, kidney, and heart (Figure 3C).Enriched Genes Have Been Suggested to Interact with Viral Proteins and Are Also Essential for Other Viral PathogensRecently, Gordon et al. (2020) performed an in-depth study of SARS-CoV-2 protein-protein interaction networks by overexpressing affinity-tagged versions of each protein encoded in the viral genome followed by tandem mass spectrometry after pull-down. Their study identified 332 high-confidence SARS-CoV-2-human protein-protein interactions (PPIs). We found that some of the highly ranked genes from our loss-of-function screen were reported to have direct PPI with different viral proteins (Figure 3D; Table S2). For example, two highly ranked subunits of the vacuolar-ATPase proton pump, ATP6AP1 and ATP6V1A, interact with SARS-CoV-2 non-structural protein 6 (nsp6) and membrane (M) protein, respectively. ATP6AP1, which was ranked 2nd in the low MOI CRISPR screen and 4th in the high MOI CRISPR screen, has a very strong PPI interaction with nsp6 (mass spectrometry interaction statistics [MIST] score = 0.99) (Verschueren et al., 2015). Another key endocytosis protein, RAB7A, is ranked in the top 50 genes in both CRISPR screens and interacts strongly with non-structural protein 7 (nsp7) (MIST score = 0.97). We also compared the top-ranked genes with another proteomic study that used proximity labeling in A549 cells overexpressing BioID-tagged viral proteins and found that 22 out of the top 50 low MOI CRISPR screen genes had direct interactions with viral genes—a significant enrichment over randomly chosen genes (p = 2 × 10−4) (Samavarchi-Tehrani et al., 2020).Because similar loss-of-function CRISPR screens have been performed to identify host genes required for other viral pathogens, we next sought to understand whether the hits identified in our SARS-CoV-2 screen were shared with those identified in prior screens for Zika virus (ZIKV) and pandemic H1N1 avian influenza (IAV) (Li et al., 2019, 2020). We examined whether top-ranked hits from the ZIKV and IAV screens shared similar genes and similar functional categories. Overall, there was greater similarity between GO categories of enriched genes for SARS-CoV-2 and ZIKV (Figure 3E; Table S2). When examining the top 50 genes from the SARS-CoV-2 screen, we found several genes that were highly enriched in all 3 viral pathogen screens (Figure S2E). This group included subunits of the vacuolar-ATPase proton pump, a well-known family of genes essential for acidification and endosomal processing (Banerjee and Kane, 2020).Validation of Enriched Genes Using CRISPR Knockout, RNA Interference, and Small Molecule InhibitorsTo test the ability of top-ranked genes to block SARS-CoV-2 viral infection, we picked 30 genes ranked among the top 200 genes in our RRA analysis for independent validation. Each gene was targeted with 3 guide RNAs distinct from the guides present in the GeCKOv2 genome-wide CRISPR library (Table S3). The guides were synthesized individually and cloned into the pCC1 lentiviral vector, a modified version of lentiCRISPRv2 with the F+E optimized guide RNA scaffold (Legut et al., 2020). For a subset of genes, we validated Cas9-mediated loss-of-function by western blot for protein expression (Figure S3
A).Figure S3Validation of Top-Ranked Genes Using CRISPR Perturbations and RNA Interference, Related to Figure 4(A) western blot analysis of RAB7A, CCDC22, ATP6V1A, and ACE2 after transduction of A549ACE2 with the indicated CRISPR guide RNA and selection with puromycin for 7 days. For validation, we designed 3 independent guide RNAs per gene (i.e., distinct guide RNAs from those in the GeCKOv2 library). Beta tubulin was used as loading control. (B) Quantitative PCR (qPCR) of SARS-CoV-2 viral load present in A549ACE2 CRISPR-perturbed cells infected with SARS-CoV-2 at MOI of 0.1. The qPCR was performed on cells collected at the indicated time (hours) post-infection (hpi) (n = 6 biological replicates, error bars indicate s.e.m.). (C) western blot analysis of RAB7A and ACE2 after transduction of Huh7.5ACE2 with the indicated CRISPR guide RNA and selection with puromycin for 7 days. For validation, we designed 3 independent guide RNAs per gene (i.e., distinct guide RNAs from those in the GeCKOv2 library). Beta tubulin was used as loading control. (D) qPCR of SARS-CoV-2 viral load present in Huh7.5ACE2 CRISPR-perturbed cells infected with SARS-CoV-2 at MOI of 0.1. The qPCR was performed on cells collected and fixed at 36 h.p.i (n = 3 guide RNAs with 6 biological replicates each, error bars indicate s.e.m.). (E) Immunofluorescence quantification of SARS-CoV-2 N protein at 36 hours post-infection (hpi) at MOI 0.1 in A549ACE2 cells pretreated with siRNA pools for 48 hours (n = 3 technical replicates, error bars represent s.e.m., NT indicates non-targeting controls).Cas9-perturbed A549ACE2 lines were infected with SARS-CoV-2 at an MOI of 0.1 and the percentage of infected cells was determined by immunofluorescence against the SARS-CoV-2 N protein at 36 h post-infection (hpi). For all of the Cas9-perturbed cell lines, we observed a reduced percentage of infected cells with an up to 10-fold reduction in SARS-CoV-2 infection, compared to the cell lines with non-targeting sgRNAs (Figures 4A and 4B). Among the genes where loss provided the greatest protection against SARS-CoV-2 infection were vesicular trafficking genes like RAB7A, CCDC22, and VPS35, and other genes with well-established roles such as the ACE2 receptor and the protease cathepsin L (CTSL) (Figure 4B). We found a significant negative correlation between the percent infection in the arrayed CRISPR validation and the median fold-change from the genome-wide CRISPR screen (r
s = −0.6, p = 5 × 10−4) (Figure 4C).Figure 4Arrayed Validation of Genome-Scale SARS-CoV-2 Screen and Identification of Druggable Gene Targets(A) Representative immunofluorescence images of A549ACE2 knockout lines infected with SARS-CoV-2 at MOI of 0.1 and fixed 36 h post-infection (hpi). SARS-CoV-2 N protein is shown in red and DAPI in blue.(B) Quantification of SARS-CoV-2 infected A549ACE2 knockout lines immuno-stained with N protein as shown in (A). Each gene was targeted with 3 different guide RNAs represented as diamond symbols (n = 3 biological replicates, error bars indicate SEM).(C) Correlation of log2 median fold change from the genome-scale CRISPR screen (low MOI) and percent of infected cells after individual (arrayed) gene perturbation shown in (B).(D) Druggable genes found in the Drug Gene Interaction database (DGIdb) among highly ranked genes from the genome-scale CRISPR screen (ranked by RRA p value).(E) qPCR of SARS-CoV-2 viral load present in A549ACE2 cells pretreated for 2 h with the indicated small molecule inhibitors at 10 μM and then infected with SARS-CoV-2 at MOI of 0.1. The qPCR was performed at 36 hpi. Red bars indicate inhibitors that yield a >100-fold reduction in viral load. Bars with hatch marks indicate an unreliable viral load measurement due to a large reduction in cell viability (see F). Inhibitors were maintained at the same concentration throughout the experiment (n = 6 biological replicates, error bars indicate SEM).(F) Percent of A549ACE2 viable cells following inhibitor treatments at 10 μM for 36 hpi determined using LIVE/DEAD stain and flow cytometry. Bars with red hatch marks indicate that inhibitor treatment had a large impact on viability (<90% viability). Significance testing for (B), (E), and (F) was performed via a one-way ANOVA with false-discovery rate-corrected follow up tests; for clarity of presentation, all significance testing can be found in Table S6.See also Figures S3 and S4 and Tables S3 and S4.For these genes, we performed a full multi-step viral replication growth curve and found that the CRISPR perturbations decrease viral load across all time points (5, 10, 24, and 48 h) compared to the non-targeting control (Figure S3B). To test if the identified hits block viral infection in other cell lines, for 8 genes we generated polyclonal CRISPR knockouts in a human liver cell line (Huh7.5ACE2), which were then infected with SARS-CoV-2 at an MOI of 0.1. Compared to non-targeting guide RNA controls, we found reduced infection with all 8 genes tested in the Huh7.5ACE2 cells (Figure S3C).As an orthogonal gene perturbation method, we also validated a more extensive list of top-ranked genes via siRNA knockdown for 48 h, followed by infection with SARS-CoV-2 at MOI of 0.1 (Table S3). Quantification of N-protein immunofluorescent images revealed a substantial reduction of the percent infected cells (Figure S3D).By cross-referencing highly ranked genes from the CRISPR screen with the Drug Gene Interaction database (DGIdb), we identified a set of 69 druggable genes (Figure 4D; Table S4; Cotto et al., 2018). We selected 9 genes that were a primary or a secondary target of one of the 26 small molecule inhibitors (Figure 4E). Among the 26 inhibitors, 9 are FDA approved and 7 are in Phase 2 or Phase 3 clinical trials for diverse diseases (Table S4). We pre-treated A549ACE2 cells for 2 h with 10 μM of each inhibitor and then infected with SARS-CoV-2 and analyzed the cells at 36 hpi. As a positive control, we also included remdesivir, which inhibits the viral RNA polymerase and is the only currently approved treatment for COVID-19 in the United States (Beigel et al., 2020). We assessed the efficacy of each inhibitor on blocking viral infection using immunofluorescence and quantitative PCR (qPCR).Seven of the 26 inhibitors that we tested (PIK-III, Compound-19, SAR405, autophinib, ALLN, tamoxifen, and ilomastat) resulted in >100-fold reduction of viral load as measured by qPCR (Figure 4E). Among the best performing inhibitors, 4 of them target the same gene PIK3C3 (also known as VPS34) and 2 inhibitors (autophinib and ALLN) reduced the viral load more than 1000-fold. Similar results were obtained by immunofluorescent imaging of SARS-CoV-2 N protein (Figure S4
A). By testing the top four PIK3C3 inhibitors in combination with CRISPR targeting of PIK3C3, we found that Compound-19, PIK-III and autophinib were specific while SAR405 resulted in greater viral inhibition in PIK3C3 CRISPR-perturbed cells, suggesting some potential off-target activity for SAR405 (Figure S4B).Figure S4Perturbations of Enriched CRISPR Screen Genes with Small-Molecule Inhibitors, Related to Figure 4(A) Immunofluorescence quantification of SARS-CoV-2 N protein at 36 hpi (MOI 0.1) in A549ACE2 cells pretreated for 2 hours with 10 μM of the indicated inhibitors (n = 3 biological replicates, error bars represent s.e.m.) (B) Quantitative PCR (qPCR) of SARS-CoV-2 viral load present in A549ACE2 cells (CRISPR-perturbed with either non-targeting or PIK3C3-targeting guide RNAs) pretreated for 2 hours with the indicated PIK3C3 molecule inhibitors at 10 μM and then infected with SARS-CoV-2 at MOI of 0.1. The qPCR was performed at 36 hours post-infection (hpi). Inhibitors were maintained at the same concentration throughout the experiment (n = 6 biological replicates, error bars indicate s.e.m.). (C) Immunofluorescence quantification of SARS-CoV-2 N protein at 36 hpi (MOI 0.1) in A549ACE2 cells pretreated with a combination of indicated inhibitors at 10 μM each for 2 hours (n = 3 biological replicates, error bars represent s.e.m.).Combinations of some of the top performing inhibitors overall showed an additive effect and further protected the cells from SARS-CoV-2 infection (Figure S4C). Next we measured cell viability at 36 h post-inhibitor treatment using flow cytometry. We observed more than 50% reduction of A549ACE2 cell viability with two pan-HDAC inhibitors, panobinostat and pracinostat (Figure 4F).Single-Cell Sequencing Identifies Cholesterol Biosynthesis as a Common Mechanism Underlying Multiple Enriched Genes from the CRISPR ScreenNext, we sought to understand the mechanisms underlying how individual genes identified in our loss-of-function screen prevent SARS-CoV-2 infection and if host gene loss alters cell transcriptional programs. For this, we utilized the expanded CRISPR-compatible cellular indexing of transcriptomes and epitopes by sequencing (ECCITE-seq) method to couple pooled CRISPR perturbations of our top hit genes with a single-cell transcriptomic and proteomic readout (Mimitou et al., 2019; Figure 5
A; Table S5). ECCITE-seq is a high-throughput approach to identify the molecular mechanisms and cellular pathways that drive infection resistance; the pooled format also provides a more controlled experiment that may be less susceptible to batch variation. Importantly, cells were infected at low MOI to maximize the fraction of cells that express a single guide RNA, and therefore can be assigned a specific gene perturbation. For this, we pooled all individual guide RNA plasmids (3 per target gene) used to validate our genome-scale screen and 9 non-targeting (NT) sgRNAs (Figure 4B; Table S3).Figure 5Single-Cell Transcriptomics (ECCITE-Seq) Identifies Shared Target Gene Signatures for Lipid and Cholesterol Regulation(A) Schematic of pooled CRISPR perturbations with expanded CRISPR-compatible cellular indexing of transcriptomes and epitopes by sequencing (ECCITE-seq). Adapted from Mimitou et al. (2019).(B) Single-cell mRNA expression heatmap showing the 100 most differentially upregulated genes (adjusted p value <0.01) for 200 randomized cells per selected target gene perturbation (for clarity, CCDC22, PIK3C3, RAB7, and TMEM165 perturbations are not shown). Labeled genes are either a top-ranked gene from the genome-wide CRISPR screens (red) or among the top 5 differentially expressed genes for a gene perturbation (black).(C) Heatmap summarizing gene set enrichment analysis results for genes upregulated in any of the indicated target gene-perturbed cells (all genes with p value <0.01 and with a limit to the 300 most differentially expressed genes; all enriched pathways with adjusted p value <10−13).(D) Cholesterol quantification in gene-perturbed cells (with the indicated guide RNAs), normalized to total protein.See also Figure S5 and Tables S5 and S6.In an initial ECCITE-seq experiment, we identified a median of 152 single cells per target gene. We observed specific reduction of target gene expression in cells grouped by target gene, indicating nonsense-mediated decay of transcripts with frameshift indel mutations after CRISPR modification (Figure S5
A). This effect was more pronounced for genes with higher expression. Using differential gene expression analysis between cells with non-targeting guide RNAs and cells with targeting guide RNAs, we identified for 11 of the 30 target genes more than 5 differentially expressed genes with a minimal log fold change (see STAR Methods), implying that loss of these genes results in a detectable transcriptomic shift. It is likely that loss of the other 19 genes results in more subtle changes in only a few genes; however, even for “non-perturbed” genes with sufficient basal expression, we detected clear changes in the CRISPR target gene. We repeated the ECCITE-seq experiment focusing on the 11 genes with detectable transcriptomic shifts upon target gene perturbation. Combining both replicate experiments, we obtained 18,853 cells that expressed only one guide RNA with a median of 1,388 cells per target gene.Figure S5ECCITE-Seq Identifies Cholesterol Gene Signature Shared across Multiple Top-Ranked Genes, Related to Figure 5(A) Stacked violin plot of 11 genes shared in both ECCITE-seq experiments. Single-cells are grouped by unique guide RNA target gene label of cells with a single detected guide RNA. Target gene expression is highlighted in red. (B) Heatmap of Gene Set Enrichment Analysis results for genes downregulated in any of the indicated target gene perturbed cells. (C) Cholesterol (normalized by total protein) in A549ACE2 cells treated with amlodipine or vehicle (DMSO) for 24 hours. (D) Quantitative PCR (qPCR) of SARS-CoV-2 viral load present in A549ACE2 cells treated with amlodipine or DMSO for 24 hours and then infected with SARS-CoV-2 at MOI of 0.1. The qPCR was performed on cells collected at the indicated time (hours) post-infection (hpi) (n = 3 biological replicates, error bars indicate s.e.m.). (E) Plaque assays of SARS-CoV-2 viral load present in A549ACE2 cells treated with amlodipine or DMSO for 24 hours and then infected with SARS-CoV-2 using logarithmically diluted supernatants. (n = 3 biological replicates, error bars indicate s.e.m.). (F) Number of reads mapping to the indicated portion of the viral genome in A549ACE2 cells treated with amlodipine or DMSO. A representative sample is shown for each treatment (n = 3 biological replicate sequencing libraries). (G) Cell viability by Trypan Blue exclusion in A549ACE2 cells treated with amlodipine or DMSO for 24 hours (n = 3 biological replicates, error bars indicate s.e.m.). (H) Distance matrix of RNA-sequencing from A549ACE2 cells treated with amlodipine or DMSO for 24 hours and then infected at MOI 0.1 or mock infection (n = 3 biological replicate sequencing libraries for each treatment-infection group). Read counts were processed with the DESeq2 regularized-log transform before computing distances. (I) k-means clustering (k = 3) of the top 500 most variable genes across all 4 conditions (n = 3 biological replicate sequencing libraries for each treatment-infection group). For each cluster, we label the most enriched pathway (lowest p-value) and, for the genes in that pathway, we label the top 5 most variable genes. No significantly enriched pathways were found for Cluster 3.We found that loss of 6 of the “perturbed” genes—ATP6AP1, ATP6V1A, CCDC22, NPC1, PIK3C3, and RAB7A, which are part of the endosomal entry pathway—yielded similar gene expression signatures among upregulated differentially expressed genes (Figures 5B, 5C, and S5B). These 6 target gene perturbations all led to upregulation of pathways affecting lipid and cholesterol homeostasis (Figure 5C). Recently, we performed a large survey of >20,000 potential pharmacological treatments for COVID-19, and for compounds effective at preventing viral infection, we identified induction of the cholesterol biosynthesis pathway as a potential mechanism of viral inhibition (Hoagland et al., 2020). Loss-of-function mutations in these 6 genes may function through a similar mechanism (induction of cholesterol synthesis) that combats the virus-mediated suppression of cholesterol synthesis. Among the significant differentially expressed genes, we also found 61 genes from the enriched CRISPR screen genes (n = 20 genes upregulated; n = 41 genes downregulated) (Table S5). For example, NPC1, ATP6V1F, and ATP6V1G1 are upregulated in most of the 6 endosomal entry pathway gene-perturbed cells (Figure 5C), suggesting compensatory upregulation of related genes to mitigate target gene loss.To understand how these changes impact lipid production, we measured cholesterol levels in cells after CRISPR perturbation and found that loss of these genes increases cholesterol by between 10% and 50%, depending on the perturbation (Figure 5D). To show that increases in cholesterol leads to increased SARS-CoV-2 resistance, we treated A549ACE2 cells with amlodipine, a calcium-channel antagonist that increases intracellular cholesterol (Mori et al., 1988; Ranganathan et al., 1982). We verified that amlodipine increases cholesterol levels in A549ACE2 cells (Figure S5C) and found that pre-treatment with amlodipine results in reduced SARS-CoV-2 viral infection, as measured by qPCR for nucleocapsid RNA, plaque formation, and number of viral RNA reads from RNA-sequencing, with only a modest impact on cell viability (Figures S5D–S5G). RNA-sequencing of cells treated with amlodipine shows a similar differential gene expression profile as seen in our ECCITE-seq with CRISPR perturbations with the most significant upregulated pathway as cholesterol biosynthesis (Figures S5H and S5I).RAB7A Knockout Results in a Reduced Cell Surface Expression and an Increased Endosomal Accumulation of ACE2Next, we sought to determine if any of the top-ranked genes regulate cell surface expression of ACE2, because surface ACE2 is required for SARS-CoV-2 infection (Hoffmann et al., 2020a). To measure the cell surface expression of ACE2, we performed flow cytometry on A549 wild-type cells and a panel of A549ACE2 CRISPR-perturbed cells. ACE2 expression was detected in A549ACE2, but not in A549 wild-type cells, validating the antibody specificity (Figures 6
A and S6
). Flow cytometry analysis of the A549ACE2 CRISPR-perturbed lines revealed that RAB7A knockout cells have a significantly reduced cell surface expression of ACE2 compared to cells transduced with a non-targeting guide RNA (Figure 6B). Efficient Rab7a protein depletion across the RAB7A CRISPR-perturbed lines used was confirmed by western blot (Figure S6C).Figure 6RAB7A Loss Results in a Reduced Cell Surface Expression and an Increased Endosomal Accumulation of ACE2(A) Representative histograms of flow cytometry analysis to determine cell surface expression of ACE2 on A549 cell lines (A549 wild-type [WT], A549ACE2, and ACE2 with Cas9 and non-targeting [NT] or RAB7A-targeting guide RNAs). The dashed line indicates the gate between the ACE2-negative and -positive cells.(B) Fraction of ACE2+ cells (using gating shown in A). ACE2 expression level was normalized across all samples to the A549ACE2 cells transduced with non-targeting (NT) guides (n = 2–3 guide RNA-transduced lines per gene, error bars are SEM).(C) Representative images of immunofluorescence staining of ACE2 on A549ACE2 transduced with an NT or a RAB7A-targeting guide. In NT cells, ACE2 localizes at the cell membrane and in the cytoplasm, while in RAB7A-targeted cells, ACE2 shows a distinct pattern of localization to vesicles.(D) Percent of cells with ACE2 accumulation in vesicles in NT and RAB7A-transduced A549ACE2 cells (n = 2 biological replicates, error bars are SEM).(E) Representative images of immunofluorescence co-stained for ACE2, EEA1, and LysoTracker in A549ACE2 cells with a CRISPR guide RNA targeting RAB7A. ACE2 shows a distinct colocalization with EEA1 (an early endosome marker) and a less frequent colocalization with LysoTracker (a lysosomal maker).(F) Representative histograms of flow cytometry analysis to determine cell surface expression in Calu-3 cells. The dashed line indicates the gate between the ACE2-negative and -positive cells.(G) Fraction of ACE2+ cells (using gating shown in F). ACE2 expression level was normalized across all samples to the Calu-3 cells transduced with an NT guide (n = 3 biological replicates, error bars are SEM).(H) Representative images of immunofluorescence staining of ACE2 on Caco-2 cells transduced with a NT or a RAB7A-targeting guide. In NT cells, ACE2 localizes at the cell membrane and in the cytoplasm, whereas in RAB7A-targeted cells, ACE2 shows a distinct pattern of localization to vesicles.(I) Mean area of ACE2 foci in Caco-2 cells transduced with a NT or a RAB7A-targeting guide (n = 4 biological replicates, 80–105 cells per replicate were scored, error bars are SEM). Significance testing for (B) and (G) was performed with a one-way ANOVA (B: F = 9.8, p < 10−4; G: Calu-3: F = 378, p < 10−4, Caco-2: F = 222, p < 10−4) with false-discovery rate-corrected post hoc tests. Significance testing for (D) and (I) was performed with an unpaired t test. For all panels, ∗p ≤ 0.05, ∗∗p ≤ 0.01, ∗∗∗p ≤ 0.001, and ∗∗∗∗p ≤ 0.0001.See also Figure S6.Figure S6Flow Cytometry for Cell Surface ACE2 Expression and Protein Analysis of RAB7A Protein after CRISPR Targeting, Related to Figure 6(A) and (B) Flow cytometry gating strategy to quantify cell surface expression of ACE2. (A) Live cells were first gated by the forward and side scatter area, then doublets were excluded by gating with the forward scatter area and width. Viable cells were selected by gating on side scatter area and LIVE/DEAD violet. (B) Gating strategy to determine ACE2+ cells. The gate was position such that < 3% of A549 wild-type and > 85% of A549ACE2 cells were ACE2 positive. The same gating strategy was applied to all samples. (C) western blot on A549ACE2 cells perturbed with non-targeting (NT) or RAB7A-targeting guide RNAs and probed with a RAB7A antibody. GAPDH was used as loading control.Rab7a is a small GTPase that is involved in regulating cellular processes such as vesicular transport and membrane trafficking (Guerra and Bucci, 2016). We hypothesized that the observed reduction of ACE2 at the cell surface in RAB7A knockout cells may be due to impaired vesicular trafficking and accumulation of ACE2 in the cytoplasm. Immunofluorescence in the A549ACE2 polyclonal cell line revealed that ACE2 is detected in most cells. We also found that RAB7A knockout leads to an increased accumulation of ACE2 in the cytoplasm and in vesicle-like hollow structures reminiscent of endo-lysosomes (Figure 6C). We detected these accumulations in about 35% of RAB7A knockout cells with ACE2 staining, compared to the control where ACE2 was primarily localized at the plasma membrane and small cytoplasmic puncta (Figure 6D). Finally, we investigated which cellular compartments accumulate ACE2 in RAB7A knockout cells. Co-immunofluorescence images showed that in RAB7A knockout cells ACE2-containing vesicles often colocalize with EEA1, an early endosomal marker and less frequently with LysoTracker, a lysosome marker (Figure 6E).Because ACE2 was overexpressed in our A549ACE2 cells, we wondered whether Rab7a loss would lead to similar cytoplasmic sequestration in cells with endogenous ACE2 expression. Flow cytometry analysis in Caco-2 colon and Calu-3 lung cells revealed that RAB7A knockout cells have significantly reduced cell surface expression of ACE2 compared to cells transduced with a non-targeting guide RNA (Figures 6F and 6G). We also found that RAB7A knockout in Caco-2 cells results in larger cytoplasmic ACE2 puncta compared to the control cells, further supporting cytoplasmic accumulation of ACE2 in the absence of Rab7a (Figures 6H and 6I).

A High-Throughput Screen to Identify Genes Required for SARS-CoV-2 InfectionTo identify key genes required for SARS-CoV-2 infection, we performed a genome-scale loss-of-function screen targeting 19,050 genes in the human genome using the GeCKOv2 CRISPR-Cas9 library (Sanjana et al., 2014). The GeCKOv2 library contains 122,411 CRISPR single-guide RNAs (sgRNAs) (6 guide RNAs per gene) and has previously been used in CRISPR screens for drug resistance, immunotherapy, synthetic lethality, mitochondrial disease, and therapeutic discovery for muscular dystrophy (Erb et al., 2017; Jain et al., 2016; Lek et al., 2020; Patel et al., 2017; Shalem et al., 2014). First, we transduced a human alveolar basal epithelial carcinoma cell line (A549) that constitutively expresses ACE2 (referred to as A549ACE2) with an all-in-one lentiviral vector containing Cas9, guide RNAs from the GeCKOv2 human library, and a puromycin resistance gene. The transduction was performed at a low multiplicity of infection (MOI ∼0.2) to ensure that most cells would receive only one guide RNA construct (Figure 1
A). We then selected with puromycin so that only library-transduced cells remained. We also measured the survival rate after puromycin selection was complete (3 days) to ensure high coverage of the 122,411 guide RNAs (∼1,000 cells per guide RNA). After puromycin selection was complete, we cultured the cells for 9 days to ensure protein depletion after CRISPR gene targeting.Figure 1A Genome-Scale CRISPR Loss-of-Function Screen to Identify Genes that Prevent SARS-CoV-2 Infection of Human Alveolar Epithelial Carcinoma Cells(A) Overview of the genome-scale loss-of-function screen for host factors in human A549ACE2 cells requires for SARS-CoV-2 infection.(B) Immunofluorescence of SARS-CoV-2 nucleocapsid (N) protein and DAPI labeling of human A549ACE2 cells at 24 h post-infection.(C) Percent survival of human A549ACE2 cells transduced with the GeCKOv2 library with the indicated SARS-CoV-2 viral amount (MOI) at 6 days post-infection.(D) Scatterplot of guide RNA read counts from A549ACE2 cells at 6 days post-infection with SARS-CoV-2 (MOI ~0.01) versus cells prior to infection. Read counts are normalized log2 reads.(E) Volcano plot of median fold-change of guide RNAs for each gene and log10 robust rank aggregation (RRA) p values. All genes with |fold-change| > 4 and RRA p < 10−3 are labeled.(F) Overlap of top 50 ranked genes between the MOI 0.01 and MOI 0.3 screen.See also Figure S1 and Table S1.Next, we infected the GeCKOv2 pool of A549ACE2 cells with SARS-CoV-2 virus (Isolate USA-WA1/2020 NR-52281) at either a high (0.3) or a low (0.01) MOI. We verified that SARS-CoV-2 infects A549ACE2 cells by staining for the nucleocapsid (N) protein at 24 h post-infection (Figure 1B), and at day 6 post-infection, we measured cell survival for both the high and low MOI conditions (Figure 1C). As expected, the higher MOI infection resulted in fewer surviving cells at day 6 post-infection. Next, we extracted genomic DNA, and via amplicon sequencing, we quantified guide abundance in each biological condition (Figure 1A). To confirm that library representation was properly maintained, we computed the correlation between the guide representation in the plasmid library and after puromycin selection (r = 0.84) (Figure S1
A). In contrast, after SARS-CoV-2 infection, there was a much greater degree of guide dropout, as expected given that SARS-CoV-2 rapidly kills A549ACE2 cells without CRISPR perturbations (Figures 1C and 1D).Figure S1Genome-wide Loss-of-Function CRISPR Screen Enriched Gene Identification, Related to Figure 1(A) Scatterplot of guide RNA read counts from A549ACE2 cells at 3 days post-transduction with the GeCKOv2 library versus read counts from the library plasmid. Read counts ar4e normalized log2 reads. (B) RRA p-value distribution for all genes in the GeCKOv2 library. (C) Overlap of top-ranked (top 500) genes between 3 different analysis methods (RRA, RIGER, and SBR). 142 genes are found by all 3 methods. (D) Comparison of top-ranked genes between different genome-scale CRISPR screens for SARS-CoV-2 infection.Using robust-rank aggregation (RRA) on the guide relative enrichments, we computed gene-level scores to identify genes where loss-of-function mutations led to enrichment within the pool (Figure 1E; Kolde et al., 2012). We identified ∼1,000 genes with significant RRA enrichment (p < 0.05) (Figure S1B). We also used two other previously published methods to compute gene enrichments (RIGER weighted-sum and second-best rank) and found a high degree of overlap between enriched genes identified by all three methods (Figure S1C; Chen et al., 2015; Luo et al., 2008). We also found a high degree of shared genes across both the low and high SARS-CoV-2 MOI conditions: when examining the top 50 most enriched genes, we found that 27 of them were shared between the low and high MOI conditions (Figure 1F; Table S1), suggesting that several host genes involved in viral pathogenesis function independently of viral dose. An independent genome-scale CRISPR screen for SARS-CoV-2 infection also performed in A549 that overexpress ACE2 but with a different CRISPR library identified similar top-ranked genes (Zhu et al., 2020), highlighting the robustness of our results (Figure S1D).

A High-Throughput Screen to Identify Genes Required for SARS-CoV-2 Infection

To identify key genes required for SARS-CoV-2 infection, we performed a genome-scale loss-of-function screen targeting 19,050 genes in the human genome using the GeCKOv2 CRISPR-Cas9 library (Sanjana et al., 2014). The GeCKOv2 library contains 122,411 CRISPR single-guide RNAs (sgRNAs) (6 guide RNAs per gene) and has previously been used in CRISPR screens for drug resistance, immunotherapy, synthetic lethality, mitochondrial disease, and therapeutic discovery for muscular dystrophy (Erb et al., 2017; Jain et al., 2016; Lek et al., 2020; Patel et al., 2017; Shalem et al., 2014). First, we transduced a human alveolar basal epithelial carcinoma cell line (A549) that constitutively expresses ACE2 (referred to as A549ACE2) with an all-in-one lentiviral vector containing Cas9, guide RNAs from the GeCKOv2 human library, and a puromycin resistance gene. The transduction was performed at a low multiplicity of infection (MOI ∼0.2) to ensure that most cells would receive only one guide RNA construct (Figure 1
A). We then selected with puromycin so that only library-transduced cells remained. We also measured the survival rate after puromycin selection was complete (3 days) to ensure high coverage of the 122,411 guide RNAs (∼1,000 cells per guide RNA). After puromycin selection was complete, we cultured the cells for 9 days to ensure protein depletion after CRISPR gene targeting.Figure 1A Genome-Scale CRISPR Loss-of-Function Screen to Identify Genes that Prevent SARS-CoV-2 Infection of Human Alveolar Epithelial Carcinoma Cells(A) Overview of the genome-scale loss-of-function screen for host factors in human A549ACE2 cells requires for SARS-CoV-2 infection.(B) Immunofluorescence of SARS-CoV-2 nucleocapsid (N) protein and DAPI labeling of human A549ACE2 cells at 24 h post-infection.(C) Percent survival of human A549ACE2 cells transduced with the GeCKOv2 library with the indicated SARS-CoV-2 viral amount (MOI) at 6 days post-infection.(D) Scatterplot of guide RNA read counts from A549ACE2 cells at 6 days post-infection with SARS-CoV-2 (MOI ~0.01) versus cells prior to infection. Read counts are normalized log2 reads.(E) Volcano plot of median fold-change of guide RNAs for each gene and log10 robust rank aggregation (RRA) p values. All genes with |fold-change| > 4 and RRA p < 10−3 are labeled.(F) Overlap of top 50 ranked genes between the MOI 0.01 and MOI 0.3 screen.See also Figure S1 and Table S1.

A Genome-Scale CRISPR Loss-of-Function Screen to Identify Genes that Prevent SARS-CoV-2 Infection of Human Alveolar Epithelial Carcinoma Cells

(A) Overview of the genome-scale loss-of-function screen for host factors in human A549ACE2 cells requires for SARS-CoV-2 infection.

(B) Immunofluorescence of SARS-CoV-2 nucleocapsid (N) protein and DAPI labeling of human A549ACE2 cells at 24 h post-infection.

(C) Percent survival of human A549ACE2 cells transduced with the GeCKOv2 library with the indicated SARS-CoV-2 viral amount (MOI) at 6 days post-infection.

(D) Scatterplot of guide RNA read counts from A549ACE2 cells at 6 days post-infection with SARS-CoV-2 (MOI ~0.01) versus cells prior to infection. Read counts are normalized log2 reads.

(E) Volcano plot of median fold-change of guide RNAs for each gene and log10 robust rank aggregation (RRA) p values. All genes with |fold-change| > 4 and RRA p < 10−3 are labeled.

(F) Overlap of top 50 ranked genes between the MOI 0.01 and MOI 0.3 screen.

See also Figure S1 and Table S1.

Next, we infected the GeCKOv2 pool of A549ACE2 cells with SARS-CoV-2 virus (Isolate USA-WA1/2020 NR-52281) at either a high (0.3) or a low (0.01) MOI. We verified that SARS-CoV-2 infects A549ACE2 cells by staining for the nucleocapsid (N) protein at 24 h post-infection (Figure 1B), and at day 6 post-infection, we measured cell survival for both the high and low MOI conditions (Figure 1C). As expected, the higher MOI infection resulted in fewer surviving cells at day 6 post-infection. Next, we extracted genomic DNA, and via amplicon sequencing, we quantified guide abundance in each biological condition (Figure 1A). To confirm that library representation was properly maintained, we computed the correlation between the guide representation in the plasmid library and after puromycin selection (r = 0.84) (Figure S1
A). In contrast, after SARS-CoV-2 infection, there was a much greater degree of guide dropout, as expected given that SARS-CoV-2 rapidly kills A549ACE2 cells without CRISPR perturbations (Figures 1C and 1D).Figure S1Genome-wide Loss-of-Function CRISPR Screen Enriched Gene Identification, Related to Figure 1(A) Scatterplot of guide RNA read counts from A549ACE2 cells at 3 days post-transduction with the GeCKOv2 library versus read counts from the library plasmid. Read counts ar4e normalized log2 reads. (B) RRA p-value distribution for all genes in the GeCKOv2 library. (C) Overlap of top-ranked (top 500) genes between 3 different analysis methods (RRA, RIGER, and SBR). 142 genes are found by all 3 methods. (D) Comparison of top-ranked genes between different genome-scale CRISPR screens for SARS-CoV-2 infection.

Genome-wide Loss-of-Function CRISPR Screen Enriched Gene Identification, Related to Figure 1

(A) Scatterplot of guide RNA read counts from A549ACE2 cells at 3 days post-transduction with the GeCKOv2 library versus read counts from the library plasmid. Read counts ar4e normalized log2 reads. (B) RRA p-value distribution for all genes in the GeCKOv2 library. (C) Overlap of top-ranked (top 500) genes between 3 different analysis methods (RRA, RIGER, and SBR). 142 genes are found by all 3 methods. (D) Comparison of top-ranked genes between different genome-scale CRISPR screens for SARS-CoV-2 infection.

Using robust-rank aggregation (RRA) on the guide relative enrichments, we computed gene-level scores to identify genes where loss-of-function mutations led to enrichment within the pool (Figure 1E; Kolde et al., 2012). We identified ∼1,000 genes with significant RRA enrichment (p < 0.05) (Figure S1B). We also used two other previously published methods to compute gene enrichments (RIGER weighted-sum and second-best rank) and found a high degree of overlap between enriched genes identified by all three methods (Figure S1C; Chen et al., 2015; Luo et al., 2008). We also found a high degree of shared genes across both the low and high SARS-CoV-2 MOI conditions: when examining the top 50 most enriched genes, we found that 27 of them were shared between the low and high MOI conditions (Figure 1F; Table S1), suggesting that several host genes involved in viral pathogenesis function independently of viral dose. An independent genome-scale CRISPR screen for SARS-CoV-2 infection also performed in A549 that overexpress ACE2 but with a different CRISPR library identified similar top-ranked genes (Zhu et al., 2020), highlighting the robustness of our results (Figure S1D).

Enriched Genes Are Involved in Multiple Aspects of the Viral Life Cycle and Are Broadly ExpressedUpon close examination of the most enriched genes, we found genes involved in key aspects of viral entry and replication (Figure 2
; Du et al., 2009). For example, the well-established entry receptor angiotensin-converting enzyme 2 (ACE2) receptor was ranked as the 8th most-enriched gene in the low MOI screen and 12th in the high MOI screen (Table S1; Hoffmann et al., 2020a; Zhou et al., 2020). Among the top 50 enriched genes, we identified several sets of related genes that function together in complexes, giving us further confidence in the genome-scale screen (Figures 2 and 3A). We found genes essential for initial attachment and endocytosis (ACE2, RAB7A, and 4 members of the ARP2/3 complex: ACTR2, ACTR3, ARPC3, and ARPC4), Spike protein cleavage and viral membrane fusion (CTSL and 13 members of the vacuolar-ATPase proton pump: ATP6AP1, ATP6AP2, ATP6V0B ATP6V0C, ATP6V0D1, ATP6V1A, ATP6V1B2, ATP6V1C1, ATP6V1E1, ATP6V1G1, ATP6V1H, TMEM199, and TOR1AIP1), endosome recycling (4 members of the endosomal protein sorting Retromer complex: VPS26A, VPS29, VPS35, and SNX27; 4 members of the endosomal trafficking Commander complex: COMMD2, COMMD3, COMMD3-BMI1, and COMMD4; and 3 members of the PI3K pathway: PIK3C3/VPS34, WDR81, and ACP5), ER-Golgi trafficking (DPM3, ERMP1, PPID, and CHST14), and transcriptional modulators (SLTM and SPEN). A consistent theme among the enriched complexes is endosome function and regulation (V-ATPase proton pump, Retromer, Commander, class 3 PI3Ks) (Banerjee and Kane, 2020; Mallam and Marcotte, 2017; McNally and Cullen, 2018). Gene set enrichment analysis on the full ranked list of genes identified significantly enriched Gene Ontology (GO) categories for endosome processing, transport, and acidification and categories related to cytokinesis and virion attachment (false discovery rate [FDR] q < 0.1) (Figures 3B and S2A–S2D; Table S2; Subramanian et al., 2005).Figure 2Top-Ranked Genes from the CRISPR Screen Are Involved in Key Elements of the SARS-CoV-2 Viral Life CycleSchematic of SARS-CoV-2 docking, entry, RNA genome release and transcription, and virion replication, assembly, and release with top-ranked host genes from the CRISPR screen highlighted in red. All genes shown are ranked in the top 50 genes (top ~0.25% of library) in the low MOI CRISPR screen using RRA. Adapted from Du et al. (2009).See also Figure S2 and Table S2.Figure 3Enriched Genes Cluster into Related Pathways, Are Expressed Broadly, Interact Directly with Viral Proteins, and Are Also Involved in Viral Pathogenesis of Pandemic Flu and Zika Virus(A) Classification of genes shown in Figure 2 (top-ranked ~0.25% of the GeCKOv2 library) into specific complexes.(B) Gene set enrichment analysis normalized enrichment scores for all significant (FDR q < 0.1) Gene Ontology (GO) biological processes.(C) Expression of top-ranked genes (same as in A) across the indicated human tissues from GTEx v8. Gene expression color scale is transcripts per million (TPM).(D) RRA fold-change for the low MOI CRISPR screen for the high-confidence protein-protein interaction with the maximum fold-change for each viral gene from the Gordon et al. (2020) mass spectrometry dataset.(E) Clustering of top-ranked GO biological processes for CRISPR screens for Zika virus ZIKV (Li et al., 2019), H1N1 pandemic avian influenza IAV (Li et al., 2020), and SARS-CoV-2 (this study).See also Figure S2 and Table S2.Figure S2Gene Set Enrichment and Overlap of Top-Ranked Genes with Other Viral Infections, Related to Figures 2 and 3(A) – (D) Four of the significant (FDR < 0.1) top-ranked GO biological process terms and the fold-change ranks of their genes in the SARS-CoV-2 low MOI CRISPR screen. (E) Normalized gene ranks of the top 50 genes from the SARS-CoV-2 low MOI CRISPR screen and genome-scale CRISPR screens for Zika virus (ZIKV) and H1N1 avian influenza (IAV).Although we performed our CRISPR screen in human lung cells, we explored whether the expression of host genes whose loss reduces SARS-CoV-2 infection were lung-specific or more broadly expressed. To answer this question, we took the top-ranked genes and examined their expression across 12 tissues using 4,790 RNA-sequencing datasets from the Genotype-Tissue Expression (GTEx) v8 database (Figure 3C; Aguet et al., 2019). Virtually all of the top gene hits were broadly expressed across all tissues, implying that these mechanisms may function independent of cell or tissue type. Among the top-ranked genes, only ACE2 showed tissue-specific expression with a particular enrichment in testis, small intestine, kidney, and heart (Figure 3C).

Enriched Genes Are Involved in Multiple Aspects of the Viral Life Cycle and Are Broadly Expressed

Upon close examination of the most enriched genes, we found genes involved in key aspects of viral entry and replication (Figure 2
; Du et al., 2009). For example, the well-established entry receptor angiotensin-converting enzyme 2 (ACE2) receptor was ranked as the 8th most-enriched gene in the low MOI screen and 12th in the high MOI screen (Table S1; Hoffmann et al., 2020a; Zhou et al., 2020). Among the top 50 enriched genes, we identified several sets of related genes that function together in complexes, giving us further confidence in the genome-scale screen (Figures 2 and 3A). We found genes essential for initial attachment and endocytosis (ACE2, RAB7A, and 4 members of the ARP2/3 complex: ACTR2, ACTR3, ARPC3, and ARPC4), Spike protein cleavage and viral membrane fusion (CTSL and 13 members of the vacuolar-ATPase proton pump: ATP6AP1, ATP6AP2, ATP6V0B ATP6V0C, ATP6V0D1, ATP6V1A, ATP6V1B2, ATP6V1C1, ATP6V1E1, ATP6V1G1, ATP6V1H, TMEM199, and TOR1AIP1), endosome recycling (4 members of the endosomal protein sorting Retromer complex: VPS26A, VPS29, VPS35, and SNX27; 4 members of the endosomal trafficking Commander complex: COMMD2, COMMD3, COMMD3-BMI1, and COMMD4; and 3 members of the PI3K pathway: PIK3C3/VPS34, WDR81, and ACP5), ER-Golgi trafficking (DPM3, ERMP1, PPID, and CHST14), and transcriptional modulators (SLTM and SPEN). A consistent theme among the enriched complexes is endosome function and regulation (V-ATPase proton pump, Retromer, Commander, class 3 PI3Ks) (Banerjee and Kane, 2020; Mallam and Marcotte, 2017; McNally and Cullen, 2018). Gene set enrichment analysis on the full ranked list of genes identified significantly enriched Gene Ontology (GO) categories for endosome processing, transport, and acidification and categories related to cytokinesis and virion attachment (false discovery rate [FDR] q < 0.1) (Figures 3B and S2A–S2D; Table S2; Subramanian et al., 2005).Figure 2Top-Ranked Genes from the CRISPR Screen Are Involved in Key Elements of the SARS-CoV-2 Viral Life CycleSchematic of SARS-CoV-2 docking, entry, RNA genome release and transcription, and virion replication, assembly, and release with top-ranked host genes from the CRISPR screen highlighted in red. All genes shown are ranked in the top 50 genes (top ~0.25% of library) in the low MOI CRISPR screen using RRA. Adapted from Du et al. (2009).See also Figure S2 and Table S2.Figure 3Enriched Genes Cluster into Related Pathways, Are Expressed Broadly, Interact Directly with Viral Proteins, and Are Also Involved in Viral Pathogenesis of Pandemic Flu and Zika Virus(A) Classification of genes shown in Figure 2 (top-ranked ~0.25% of the GeCKOv2 library) into specific complexes.(B) Gene set enrichment analysis normalized enrichment scores for all significant (FDR q < 0.1) Gene Ontology (GO) biological processes.(C) Expression of top-ranked genes (same as in A) across the indicated human tissues from GTEx v8. Gene expression color scale is transcripts per million (TPM).(D) RRA fold-change for the low MOI CRISPR screen for the high-confidence protein-protein interaction with the maximum fold-change for each viral gene from the Gordon et al. (2020) mass spectrometry dataset.(E) Clustering of top-ranked GO biological processes for CRISPR screens for Zika virus ZIKV (Li et al., 2019), H1N1 pandemic avian influenza IAV (Li et al., 2020), and SARS-CoV-2 (this study).See also Figure S2 and Table S2.Figure S2Gene Set Enrichment and Overlap of Top-Ranked Genes with Other Viral Infections, Related to Figures 2 and 3(A) – (D) Four of the significant (FDR < 0.1) top-ranked GO biological process terms and the fold-change ranks of their genes in the SARS-CoV-2 low MOI CRISPR screen. (E) Normalized gene ranks of the top 50 genes from the SARS-CoV-2 low MOI CRISPR screen and genome-scale CRISPR screens for Zika virus (ZIKV) and H1N1 avian influenza (IAV).

Top-Ranked Genes from the CRISPR Screen Are Involved in Key Elements of the SARS-CoV-2 Viral Life Cycle

Schematic of SARS-CoV-2 docking, entry, RNA genome release and transcription, and virion replication, assembly, and release with top-ranked host genes from the CRISPR screen highlighted in red. All genes shown are ranked in the top 50 genes (top ~0.25% of library) in the low MOI CRISPR screen using RRA. Adapted from Du et al. (2009).

See also Figure S2 and Table S2.

Enriched Genes Cluster into Related Pathways, Are Expressed Broadly, Interact Directly with Viral Proteins, and Are Also Involved in Viral Pathogenesis of Pandemic Flu and Zika Virus

(A) Classification of genes shown in Figure 2 (top-ranked ~0.25% of the GeCKOv2 library) into specific complexes.

(B) Gene set enrichment analysis normalized enrichment scores for all significant (FDR q < 0.1) Gene Ontology (GO) biological processes.

(C) Expression of top-ranked genes (same as in A) across the indicated human tissues from GTEx v8. Gene expression color scale is transcripts per million (TPM).

(D) RRA fold-change for the low MOI CRISPR screen for the high-confidence protein-protein interaction with the maximum fold-change for each viral gene from the Gordon et al. (2020) mass spectrometry dataset.

(E) Clustering of top-ranked GO biological processes for CRISPR screens for Zika virus ZIKV (Li et al., 2019), H1N1 pandemic avian influenza IAV (Li et al., 2020), and SARS-CoV-2 (this study).

See also Figure S2 and Table S2.

Gene Set Enrichment and Overlap of Top-Ranked Genes with Other Viral Infections, Related to Figures 2 and 3

(A) – (D) Four of the significant (FDR < 0.1) top-ranked GO biological process terms and the fold-change ranks of their genes in the SARS-CoV-2 low MOI CRISPR screen. (E) Normalized gene ranks of the top 50 genes from the SARS-CoV-2 low MOI CRISPR screen and genome-scale CRISPR screens for Zika virus (ZIKV) and H1N1 avian influenza (IAV).

Although we performed our CRISPR screen in human lung cells, we explored whether the expression of host genes whose loss reduces SARS-CoV-2 infection were lung-specific or more broadly expressed. To answer this question, we took the top-ranked genes and examined their expression across 12 tissues using 4,790 RNA-sequencing datasets from the Genotype-Tissue Expression (GTEx) v8 database (Figure 3C; Aguet et al., 2019). Virtually all of the top gene hits were broadly expressed across all tissues, implying that these mechanisms may function independent of cell or tissue type. Among the top-ranked genes, only ACE2 showed tissue-specific expression with a particular enrichment in testis, small intestine, kidney, and heart (Figure 3C).

Enriched Genes Have Been Suggested to Interact with Viral Proteins and Are Also Essential for Other Viral PathogensRecently, Gordon et al. (2020) performed an in-depth study of SARS-CoV-2 protein-protein interaction networks by overexpressing affinity-tagged versions of each protein encoded in the viral genome followed by tandem mass spectrometry after pull-down. Their study identified 332 high-confidence SARS-CoV-2-human protein-protein interactions (PPIs). We found that some of the highly ranked genes from our loss-of-function screen were reported to have direct PPI with different viral proteins (Figure 3D; Table S2). For example, two highly ranked subunits of the vacuolar-ATPase proton pump, ATP6AP1 and ATP6V1A, interact with SARS-CoV-2 non-structural protein 6 (nsp6) and membrane (M) protein, respectively. ATP6AP1, which was ranked 2nd in the low MOI CRISPR screen and 4th in the high MOI CRISPR screen, has a very strong PPI interaction with nsp6 (mass spectrometry interaction statistics [MIST] score = 0.99) (Verschueren et al., 2015). Another key endocytosis protein, RAB7A, is ranked in the top 50 genes in both CRISPR screens and interacts strongly with non-structural protein 7 (nsp7) (MIST score = 0.97). We also compared the top-ranked genes with another proteomic study that used proximity labeling in A549 cells overexpressing BioID-tagged viral proteins and found that 22 out of the top 50 low MOI CRISPR screen genes had direct interactions with viral genes—a significant enrichment over randomly chosen genes (p = 2 × 10−4) (Samavarchi-Tehrani et al., 2020).Because similar loss-of-function CRISPR screens have been performed to identify host genes required for other viral pathogens, we next sought to understand whether the hits identified in our SARS-CoV-2 screen were shared with those identified in prior screens for Zika virus (ZIKV) and pandemic H1N1 avian influenza (IAV) (Li et al., 2019, 2020). We examined whether top-ranked hits from the ZIKV and IAV screens shared similar genes and similar functional categories. Overall, there was greater similarity between GO categories of enriched genes for SARS-CoV-2 and ZIKV (Figure 3E; Table S2). When examining the top 50 genes from the SARS-CoV-2 screen, we found several genes that were highly enriched in all 3 viral pathogen screens (Figure S2E). This group included subunits of the vacuolar-ATPase proton pump, a well-known family of genes essential for acidification and endosomal processing (Banerjee and Kane, 2020).

Enriched Genes Have Been Suggested to Interact with Viral Proteins and Are Also Essential for Other Viral Pathogens

Recently, Gordon et al. (2020) performed an in-depth study of SARS-CoV-2 protein-protein interaction networks by overexpressing affinity-tagged versions of each protein encoded in the viral genome followed by tandem mass spectrometry after pull-down. Their study identified 332 high-confidence SARS-CoV-2-human protein-protein interactions (PPIs). We found that some of the highly ranked genes from our loss-of-function screen were reported to have direct PPI with different viral proteins (Figure 3D; Table S2). For example, two highly ranked subunits of the vacuolar-ATPase proton pump, ATP6AP1 and ATP6V1A, interact with SARS-CoV-2 non-structural protein 6 (nsp6) and membrane (M) protein, respectively. ATP6AP1, which was ranked 2nd in the low MOI CRISPR screen and 4th in the high MOI CRISPR screen, has a very strong PPI interaction with nsp6 (mass spectrometry interaction statistics [MIST] score = 0.99) (Verschueren et al., 2015). Another key endocytosis protein, RAB7A, is ranked in the top 50 genes in both CRISPR screens and interacts strongly with non-structural protein 7 (nsp7) (MIST score = 0.97). We also compared the top-ranked genes with another proteomic study that used proximity labeling in A549 cells overexpressing BioID-tagged viral proteins and found that 22 out of the top 50 low MOI CRISPR screen genes had direct interactions with viral genes—a significant enrichment over randomly chosen genes (p = 2 × 10−4) (Samavarchi-Tehrani et al., 2020).

Because similar loss-of-function CRISPR screens have been performed to identify host genes required for other viral pathogens, we next sought to understand whether the hits identified in our SARS-CoV-2 screen were shared with those identified in prior screens for Zika virus (ZIKV) and pandemic H1N1 avian influenza (IAV) (Li et al., 2019, 2020). We examined whether top-ranked hits from the ZIKV and IAV screens shared similar genes and similar functional categories. Overall, there was greater similarity between GO categories of enriched genes for SARS-CoV-2 and ZIKV (Figure 3E; Table S2). When examining the top 50 genes from the SARS-CoV-2 screen, we found several genes that were highly enriched in all 3 viral pathogen screens (Figure S2E). This group included subunits of the vacuolar-ATPase proton pump, a well-known family of genes essential for acidification and endosomal processing (Banerjee and Kane, 2020).

Validation of Enriched Genes Using CRISPR Knockout, RNA Interference, and Small Molecule InhibitorsTo test the ability of top-ranked genes to block SARS-CoV-2 viral infection, we picked 30 genes ranked among the top 200 genes in our RRA analysis for independent validation. Each gene was targeted with 3 guide RNAs distinct from the guides present in the GeCKOv2 genome-wide CRISPR library (Table S3). The guides were synthesized individually and cloned into the pCC1 lentiviral vector, a modified version of lentiCRISPRv2 with the F+E optimized guide RNA scaffold (Legut et al., 2020). For a subset of genes, we validated Cas9-mediated loss-of-function by western blot for protein expression (Figure S3
A).Figure S3Validation of Top-Ranked Genes Using CRISPR Perturbations and RNA Interference, Related to Figure 4(A) western blot analysis of RAB7A, CCDC22, ATP6V1A, and ACE2 after transduction of A549ACE2 with the indicated CRISPR guide RNA and selection with puromycin for 7 days. For validation, we designed 3 independent guide RNAs per gene (i.e., distinct guide RNAs from those in the GeCKOv2 library). Beta tubulin was used as loading control. (B) Quantitative PCR (qPCR) of SARS-CoV-2 viral load present in A549ACE2 CRISPR-perturbed cells infected with SARS-CoV-2 at MOI of 0.1. The qPCR was performed on cells collected at the indicated time (hours) post-infection (hpi) (n = 6 biological replicates, error bars indicate s.e.m.). (C) western blot analysis of RAB7A and ACE2 after transduction of Huh7.5ACE2 with the indicated CRISPR guide RNA and selection with puromycin for 7 days. For validation, we designed 3 independent guide RNAs per gene (i.e., distinct guide RNAs from those in the GeCKOv2 library). Beta tubulin was used as loading control. (D) qPCR of SARS-CoV-2 viral load present in Huh7.5ACE2 CRISPR-perturbed cells infected with SARS-CoV-2 at MOI of 0.1. The qPCR was performed on cells collected and fixed at 36 h.p.i (n = 3 guide RNAs with 6 biological replicates each, error bars indicate s.e.m.). (E) Immunofluorescence quantification of SARS-CoV-2 N protein at 36 hours post-infection (hpi) at MOI 0.1 in A549ACE2 cells pretreated with siRNA pools for 48 hours (n = 3 technical replicates, error bars represent s.e.m., NT indicates non-targeting controls).Cas9-perturbed A549ACE2 lines were infected with SARS-CoV-2 at an MOI of 0.1 and the percentage of infected cells was determined by immunofluorescence against the SARS-CoV-2 N protein at 36 h post-infection (hpi). For all of the Cas9-perturbed cell lines, we observed a reduced percentage of infected cells with an up to 10-fold reduction in SARS-CoV-2 infection, compared to the cell lines with non-targeting sgRNAs (Figures 4A and 4B). Among the genes where loss provided the greatest protection against SARS-CoV-2 infection were vesicular trafficking genes like RAB7A, CCDC22, and VPS35, and other genes with well-established roles such as the ACE2 receptor and the protease cathepsin L (CTSL) (Figure 4B). We found a significant negative correlation between the percent infection in the arrayed CRISPR validation and the median fold-change from the genome-wide CRISPR screen (r
s = −0.6, p = 5 × 10−4) (Figure 4C).Figure 4Arrayed Validation of Genome-Scale SARS-CoV-2 Screen and Identification of Druggable Gene Targets(A) Representative immunofluorescence images of A549ACE2 knockout lines infected with SARS-CoV-2 at MOI of 0.1 and fixed 36 h post-infection (hpi). SARS-CoV-2 N protein is shown in red and DAPI in blue.(B) Quantification of SARS-CoV-2 infected A549ACE2 knockout lines immuno-stained with N protein as shown in (A). Each gene was targeted with 3 different guide RNAs represented as diamond symbols (n = 3 biological replicates, error bars indicate SEM).(C) Correlation of log2 median fold change from the genome-scale CRISPR screen (low MOI) and percent of infected cells after individual (arrayed) gene perturbation shown in (B).(D) Druggable genes found in the Drug Gene Interaction database (DGIdb) among highly ranked genes from the genome-scale CRISPR screen (ranked by RRA p value).(E) qPCR of SARS-CoV-2 viral load present in A549ACE2 cells pretreated for 2 h with the indicated small molecule inhibitors at 10 μM and then infected with SARS-CoV-2 at MOI of 0.1. The qPCR was performed at 36 hpi. Red bars indicate inhibitors that yield a >100-fold reduction in viral load. Bars with hatch marks indicate an unreliable viral load measurement due to a large reduction in cell viability (see F). Inhibitors were maintained at the same concentration throughout the experiment (n = 6 biological replicates, error bars indicate SEM).(F) Percent of A549ACE2 viable cells following inhibitor treatments at 10 μM for 36 hpi determined using LIVE/DEAD stain and flow cytometry. Bars with red hatch marks indicate that inhibitor treatment had a large impact on viability (<90% viability). Significance testing for (B), (E), and (F) was performed via a one-way ANOVA with false-discovery rate-corrected follow up tests; for clarity of presentation, all significance testing can be found in Table S6.See also Figures S3 and S4 and Tables S3 and S4.For these genes, we performed a full multi-step viral replication growth curve and found that the CRISPR perturbations decrease viral load across all time points (5, 10, 24, and 48 h) compared to the non-targeting control (Figure S3B). To test if the identified hits block viral infection in other cell lines, for 8 genes we generated polyclonal CRISPR knockouts in a human liver cell line (Huh7.5ACE2), which were then infected with SARS-CoV-2 at an MOI of 0.1. Compared to non-targeting guide RNA controls, we found reduced infection with all 8 genes tested in the Huh7.5ACE2 cells (Figure S3C).As an orthogonal gene perturbation method, we also validated a more extensive list of top-ranked genes via siRNA knockdown for 48 h, followed by infection with SARS-CoV-2 at MOI of 0.1 (Table S3). Quantification of N-protein immunofluorescent images revealed a substantial reduction of the percent infected cells (Figure S3D).By cross-referencing highly ranked genes from the CRISPR screen with the Drug Gene Interaction database (DGIdb), we identified a set of 69 druggable genes (Figure 4D; Table S4; Cotto et al., 2018). We selected 9 genes that were a primary or a secondary target of one of the 26 small molecule inhibitors (Figure 4E). Among the 26 inhibitors, 9 are FDA approved and 7 are in Phase 2 or Phase 3 clinical trials for diverse diseases (Table S4). We pre-treated A549ACE2 cells for 2 h with 10 μM of each inhibitor and then infected with SARS-CoV-2 and analyzed the cells at 36 hpi. As a positive control, we also included remdesivir, which inhibits the viral RNA polymerase and is the only currently approved treatment for COVID-19 in the United States (Beigel et al., 2020). We assessed the efficacy of each inhibitor on blocking viral infection using immunofluorescence and quantitative PCR (qPCR).Seven of the 26 inhibitors that we tested (PIK-III, Compound-19, SAR405, autophinib, ALLN, tamoxifen, and ilomastat) resulted in >100-fold reduction of viral load as measured by qPCR (Figure 4E). Among the best performing inhibitors, 4 of them target the same gene PIK3C3 (also known as VPS34) and 2 inhibitors (autophinib and ALLN) reduced the viral load more than 1000-fold. Similar results were obtained by immunofluorescent imaging of SARS-CoV-2 N protein (Figure S4
A). By testing the top four PIK3C3 inhibitors in combination with CRISPR targeting of PIK3C3, we found that Compound-19, PIK-III and autophinib were specific while SAR405 resulted in greater viral inhibition in PIK3C3 CRISPR-perturbed cells, suggesting some potential off-target activity for SAR405 (Figure S4B).Figure S4Perturbations of Enriched CRISPR Screen Genes with Small-Molecule Inhibitors, Related to Figure 4(A) Immunofluorescence quantification of SARS-CoV-2 N protein at 36 hpi (MOI 0.1) in A549ACE2 cells pretreated for 2 hours with 10 μM of the indicated inhibitors (n = 3 biological replicates, error bars represent s.e.m.) (B) Quantitative PCR (qPCR) of SARS-CoV-2 viral load present in A549ACE2 cells (CRISPR-perturbed with either non-targeting or PIK3C3-targeting guide RNAs) pretreated for 2 hours with the indicated PIK3C3 molecule inhibitors at 10 μM and then infected with SARS-CoV-2 at MOI of 0.1. The qPCR was performed at 36 hours post-infection (hpi). Inhibitors were maintained at the same concentration throughout the experiment (n = 6 biological replicates, error bars indicate s.e.m.). (C) Immunofluorescence quantification of SARS-CoV-2 N protein at 36 hpi (MOI 0.1) in A549ACE2 cells pretreated with a combination of indicated inhibitors at 10 μM each for 2 hours (n = 3 biological replicates, error bars represent s.e.m.).Combinations of some of the top performing inhibitors overall showed an additive effect and further protected the cells from SARS-CoV-2 infection (Figure S4C). Next we measured cell viability at 36 h post-inhibitor treatment using flow cytometry. We observed more than 50% reduction of A549ACE2 cell viability with two pan-HDAC inhibitors, panobinostat and pracinostat (Figure 4F).

Validation of Enriched Genes Using CRISPR Knockout, RNA Interference, and Small Molecule Inhibitors

To test the ability of top-ranked genes to block SARS-CoV-2 viral infection, we picked 30 genes ranked among the top 200 genes in our RRA analysis for independent validation. Each gene was targeted with 3 guide RNAs distinct from the guides present in the GeCKOv2 genome-wide CRISPR library (Table S3). The guides were synthesized individually and cloned into the pCC1 lentiviral vector, a modified version of lentiCRISPRv2 with the F+E optimized guide RNA scaffold (Legut et al., 2020). For a subset of genes, we validated Cas9-mediated loss-of-function by western blot for protein expression (Figure S3
A).Figure S3Validation of Top-Ranked Genes Using CRISPR Perturbations and RNA Interference, Related to Figure 4(A) western blot analysis of RAB7A, CCDC22, ATP6V1A, and ACE2 after transduction of A549ACE2 with the indicated CRISPR guide RNA and selection with puromycin for 7 days. For validation, we designed 3 independent guide RNAs per gene (i.e., distinct guide RNAs from those in the GeCKOv2 library). Beta tubulin was used as loading control. (B) Quantitative PCR (qPCR) of SARS-CoV-2 viral load present in A549ACE2 CRISPR-perturbed cells infected with SARS-CoV-2 at MOI of 0.1. The qPCR was performed on cells collected at the indicated time (hours) post-infection (hpi) (n = 6 biological replicates, error bars indicate s.e.m.). (C) western blot analysis of RAB7A and ACE2 after transduction of Huh7.5ACE2 with the indicated CRISPR guide RNA and selection with puromycin for 7 days. For validation, we designed 3 independent guide RNAs per gene (i.e., distinct guide RNAs from those in the GeCKOv2 library). Beta tubulin was used as loading control. (D) qPCR of SARS-CoV-2 viral load present in Huh7.5ACE2 CRISPR-perturbed cells infected with SARS-CoV-2 at MOI of 0.1. The qPCR was performed on cells collected and fixed at 36 h.p.i (n = 3 guide RNAs with 6 biological replicates each, error bars indicate s.e.m.). (E) Immunofluorescence quantification of SARS-CoV-2 N protein at 36 hours post-infection (hpi) at MOI 0.1 in A549ACE2 cells pretreated with siRNA pools for 48 hours (n = 3 technical replicates, error bars represent s.e.m., NT indicates non-targeting controls).

Validation of Top-Ranked Genes Using CRISPR Perturbations and RNA Interference, Related to Figure 4

(A) western blot analysis of RAB7A, CCDC22, ATP6V1A, and ACE2 after transduction of A549ACE2 with the indicated CRISPR guide RNA and selection with puromycin for 7 days. For validation, we designed 3 independent guide RNAs per gene (i.e., distinct guide RNAs from those in the GeCKOv2 library). Beta tubulin was used as loading control. (B) Quantitative PCR (qPCR) of SARS-CoV-2 viral load present in A549ACE2 CRISPR-perturbed cells infected with SARS-CoV-2 at MOI of 0.1. The qPCR was performed on cells collected at the indicated time (hours) post-infection (hpi) (n = 6 biological replicates, error bars indicate s.e.m.). (C) western blot analysis of RAB7A and ACE2 after transduction of Huh7.5ACE2 with the indicated CRISPR guide RNA and selection with puromycin for 7 days. For validation, we designed 3 independent guide RNAs per gene (i.e., distinct guide RNAs from those in the GeCKOv2 library). Beta tubulin was used as loading control. (D) qPCR of SARS-CoV-2 viral load present in Huh7.5ACE2 CRISPR-perturbed cells infected with SARS-CoV-2 at MOI of 0.1. The qPCR was performed on cells collected and fixed at 36 h.p.i (n = 3 guide RNAs with 6 biological replicates each, error bars indicate s.e.m.). (E) Immunofluorescence quantification of SARS-CoV-2 N protein at 36 hours post-infection (hpi) at MOI 0.1 in A549ACE2 cells pretreated with siRNA pools for 48 hours (n = 3 technical replicates, error bars represent s.e.m., NT indicates non-targeting controls).

Cas9-perturbed A549ACE2 lines were infected with SARS-CoV-2 at an MOI of 0.1 and the percentage of infected cells was determined by immunofluorescence against the SARS-CoV-2 N protein at 36 h post-infection (hpi). For all of the Cas9-perturbed cell lines, we observed a reduced percentage of infected cells with an up to 10-fold reduction in SARS-CoV-2 infection, compared to the cell lines with non-targeting sgRNAs (Figures 4A and 4B). Among the genes where loss provided the greatest protection against SARS-CoV-2 infection were vesicular trafficking genes like RAB7A, CCDC22, and VPS35, and other genes with well-established roles such as the ACE2 receptor and the protease cathepsin L (CTSL) (Figure 4B). We found a significant negative correlation between the percent infection in the arrayed CRISPR validation and the median fold-change from the genome-wide CRISPR screen (r
s = −0.6, p = 5 × 10−4) (Figure 4C).Figure 4Arrayed Validation of Genome-Scale SARS-CoV-2 Screen and Identification of Druggable Gene Targets(A) Representative immunofluorescence images of A549ACE2 knockout lines infected with SARS-CoV-2 at MOI of 0.1 and fixed 36 h post-infection (hpi). SARS-CoV-2 N protein is shown in red and DAPI in blue.(B) Quantification of SARS-CoV-2 infected A549ACE2 knockout lines immuno-stained with N protein as shown in (A). Each gene was targeted with 3 different guide RNAs represented as diamond symbols (n = 3 biological replicates, error bars indicate SEM).(C) Correlation of log2 median fold change from the genome-scale CRISPR screen (low MOI) and percent of infected cells after individual (arrayed) gene perturbation shown in (B).(D) Druggable genes found in the Drug Gene Interaction database (DGIdb) among highly ranked genes from the genome-scale CRISPR screen (ranked by RRA p value).(E) qPCR of SARS-CoV-2 viral load present in A549ACE2 cells pretreated for 2 h with the indicated small molecule inhibitors at 10 μM and then infected with SARS-CoV-2 at MOI of 0.1. The qPCR was performed at 36 hpi. Red bars indicate inhibitors that yield a >100-fold reduction in viral load. Bars with hatch marks indicate an unreliable viral load measurement due to a large reduction in cell viability (see F). Inhibitors were maintained at the same concentration throughout the experiment (n = 6 biological replicates, error bars indicate SEM).(F) Percent of A549ACE2 viable cells following inhibitor treatments at 10 μM for 36 hpi determined using LIVE/DEAD stain and flow cytometry. Bars with red hatch marks indicate that inhibitor treatment had a large impact on viability (<90% viability). Significance testing for (B), (E), and (F) was performed via a one-way ANOVA with false-discovery rate-corrected follow up tests; for clarity of presentation, all significance testing can be found in Table S6.See also Figures S3 and S4 and Tables S3 and S4.

Arrayed Validation of Genome-Scale SARS-CoV-2 Screen and Identification of Druggable Gene Targets

(A) Representative immunofluorescence images of A549ACE2 knockout lines infected with SARS-CoV-2 at MOI of 0.1 and fixed 36 h post-infection (hpi). SARS-CoV-2 N protein is shown in red and DAPI in blue.

(B) Quantification of SARS-CoV-2 infected A549ACE2 knockout lines immuno-stained with N protein as shown in (A). Each gene was targeted with 3 different guide RNAs represented as diamond symbols (n = 3 biological replicates, error bars indicate SEM).

(C) Correlation of log2 median fold change from the genome-scale CRISPR screen (low MOI) and percent of infected cells after individual (arrayed) gene perturbation shown in (B).

(D) Druggable genes found in the Drug Gene Interaction database (DGIdb) among highly ranked genes from the genome-scale CRISPR screen (ranked by RRA p value).

(E) qPCR of SARS-CoV-2 viral load present in A549ACE2 cells pretreated for 2 h with the indicated small molecule inhibitors at 10 μM and then infected with SARS-CoV-2 at MOI of 0.1. The qPCR was performed at 36 hpi. Red bars indicate inhibitors that yield a >100-fold reduction in viral load. Bars with hatch marks indicate an unreliable viral load measurement due to a large reduction in cell viability (see F). Inhibitors were maintained at the same concentration throughout the experiment (n = 6 biological replicates, error bars indicate SEM).

(F) Percent of A549ACE2 viable cells following inhibitor treatments at 10 μM for 36 hpi determined using LIVE/DEAD stain and flow cytometry. Bars with red hatch marks indicate that inhibitor treatment had a large impact on viability (<90% viability). Significance testing for (B), (E), and (F) was performed via a one-way ANOVA with false-discovery rate-corrected follow up tests; for clarity of presentation, all significance testing can be found in Table S6.

See also Figures S3 and S4 and Tables S3 and S4.

For these genes, we performed a full multi-step viral replication growth curve and found that the CRISPR perturbations decrease viral load across all time points (5, 10, 24, and 48 h) compared to the non-targeting control (Figure S3B). To test if the identified hits block viral infection in other cell lines, for 8 genes we generated polyclonal CRISPR knockouts in a human liver cell line (Huh7.5ACE2), which were then infected with SARS-CoV-2 at an MOI of 0.1. Compared to non-targeting guide RNA controls, we found reduced infection with all 8 genes tested in the Huh7.5ACE2 cells (Figure S3C).

As an orthogonal gene perturbation method, we also validated a more extensive list of top-ranked genes via siRNA knockdown for 48 h, followed by infection with SARS-CoV-2 at MOI of 0.1 (Table S3). Quantification of N-protein immunofluorescent images revealed a substantial reduction of the percent infected cells (Figure S3D).

By cross-referencing highly ranked genes from the CRISPR screen with the Drug Gene Interaction database (DGIdb), we identified a set of 69 druggable genes (Figure 4D; Table S4; Cotto et al., 2018). We selected 9 genes that were a primary or a secondary target of one of the 26 small molecule inhibitors (Figure 4E). Among the 26 inhibitors, 9 are FDA approved and 7 are in Phase 2 or Phase 3 clinical trials for diverse diseases (Table S4). We pre-treated A549ACE2 cells for 2 h with 10 μM of each inhibitor and then infected with SARS-CoV-2 and analyzed the cells at 36 hpi. As a positive control, we also included remdesivir, which inhibits the viral RNA polymerase and is the only currently approved treatment for COVID-19 in the United States (Beigel et al., 2020). We assessed the efficacy of each inhibitor on blocking viral infection using immunofluorescence and quantitative PCR (qPCR).

Seven of the 26 inhibitors that we tested (PIK-III, Compound-19, SAR405, autophinib, ALLN, tamoxifen, and ilomastat) resulted in >100-fold reduction of viral load as measured by qPCR (Figure 4E). Among the best performing inhibitors, 4 of them target the same gene PIK3C3 (also known as VPS34) and 2 inhibitors (autophinib and ALLN) reduced the viral load more than 1000-fold. Similar results were obtained by immunofluorescent imaging of SARS-CoV-2 N protein (Figure S4
A). By testing the top four PIK3C3 inhibitors in combination with CRISPR targeting of PIK3C3, we found that Compound-19, PIK-III and autophinib were specific while SAR405 resulted in greater viral inhibition in PIK3C3 CRISPR-perturbed cells, suggesting some potential off-target activity for SAR405 (Figure S4B).Figure S4Perturbations of Enriched CRISPR Screen Genes with Small-Molecule Inhibitors, Related to Figure 4(A) Immunofluorescence quantification of SARS-CoV-2 N protein at 36 hpi (MOI 0.1) in A549ACE2 cells pretreated for 2 hours with 10 μM of the indicated inhibitors (n = 3 biological replicates, error bars represent s.e.m.) (B) Quantitative PCR (qPCR) of SARS-CoV-2 viral load present in A549ACE2 cells (CRISPR-perturbed with either non-targeting or PIK3C3-targeting guide RNAs) pretreated for 2 hours with the indicated PIK3C3 molecule inhibitors at 10 μM and then infected with SARS-CoV-2 at MOI of 0.1. The qPCR was performed at 36 hours post-infection (hpi). Inhibitors were maintained at the same concentration throughout the experiment (n = 6 biological replicates, error bars indicate s.e.m.). (C) Immunofluorescence quantification of SARS-CoV-2 N protein at 36 hpi (MOI 0.1) in A549ACE2 cells pretreated with a combination of indicated inhibitors at 10 μM each for 2 hours (n = 3 biological replicates, error bars represent s.e.m.).

Perturbations of Enriched CRISPR Screen Genes with Small-Molecule Inhibitors, Related to Figure 4

(A) Immunofluorescence quantification of SARS-CoV-2 N protein at 36 hpi (MOI 0.1) in A549ACE2 cells pretreated for 2 hours with 10 μM of the indicated inhibitors (n = 3 biological replicates, error bars represent s.e.m.) (B) Quantitative PCR (qPCR) of SARS-CoV-2 viral load present in A549ACE2 cells (CRISPR-perturbed with either non-targeting or PIK3C3-targeting guide RNAs) pretreated for 2 hours with the indicated PIK3C3 molecule inhibitors at 10 μM and then infected with SARS-CoV-2 at MOI of 0.1. The qPCR was performed at 36 hours post-infection (hpi). Inhibitors were maintained at the same concentration throughout the experiment (n = 6 biological replicates, error bars indicate s.e.m.). (C) Immunofluorescence quantification of SARS-CoV-2 N protein at 36 hpi (MOI 0.1) in A549ACE2 cells pretreated with a combination of indicated inhibitors at 10 μM each for 2 hours (n = 3 biological replicates, error bars represent s.e.m.).

Combinations of some of the top performing inhibitors overall showed an additive effect and further protected the cells from SARS-CoV-2 infection (Figure S4C). Next we measured cell viability at 36 h post-inhibitor treatment using flow cytometry. We observed more than 50% reduction of A549ACE2 cell viability with two pan-HDAC inhibitors, panobinostat and pracinostat (Figure 4F).

Single-Cell Sequencing Identifies Cholesterol Biosynthesis as a Common Mechanism Underlying Multiple Enriched Genes from the CRISPR ScreenNext, we sought to understand the mechanisms underlying how individual genes identified in our loss-of-function screen prevent SARS-CoV-2 infection and if host gene loss alters cell transcriptional programs. For this, we utilized the expanded CRISPR-compatible cellular indexing of transcriptomes and epitopes by sequencing (ECCITE-seq) method to couple pooled CRISPR perturbations of our top hit genes with a single-cell transcriptomic and proteomic readout (Mimitou et al., 2019; Figure 5
A; Table S5). ECCITE-seq is a high-throughput approach to identify the molecular mechanisms and cellular pathways that drive infection resistance; the pooled format also provides a more controlled experiment that may be less susceptible to batch variation. Importantly, cells were infected at low MOI to maximize the fraction of cells that express a single guide RNA, and therefore can be assigned a specific gene perturbation. For this, we pooled all individual guide RNA plasmids (3 per target gene) used to validate our genome-scale screen and 9 non-targeting (NT) sgRNAs (Figure 4B; Table S3).Figure 5Single-Cell Transcriptomics (ECCITE-Seq) Identifies Shared Target Gene Signatures for Lipid and Cholesterol Regulation(A) Schematic of pooled CRISPR perturbations with expanded CRISPR-compatible cellular indexing of transcriptomes and epitopes by sequencing (ECCITE-seq). Adapted from Mimitou et al. (2019).(B) Single-cell mRNA expression heatmap showing the 100 most differentially upregulated genes (adjusted p value <0.01) for 200 randomized cells per selected target gene perturbation (for clarity, CCDC22, PIK3C3, RAB7, and TMEM165 perturbations are not shown). Labeled genes are either a top-ranked gene from the genome-wide CRISPR screens (red) or among the top 5 differentially expressed genes for a gene perturbation (black).(C) Heatmap summarizing gene set enrichment analysis results for genes upregulated in any of the indicated target gene-perturbed cells (all genes with p value <0.01 and with a limit to the 300 most differentially expressed genes; all enriched pathways with adjusted p value <10−13).(D) Cholesterol quantification in gene-perturbed cells (with the indicated guide RNAs), normalized to total protein.See also Figure S5 and Tables S5 and S6.In an initial ECCITE-seq experiment, we identified a median of 152 single cells per target gene. We observed specific reduction of target gene expression in cells grouped by target gene, indicating nonsense-mediated decay of transcripts with frameshift indel mutations after CRISPR modification (Figure S5
A). This effect was more pronounced for genes with higher expression. Using differential gene expression analysis between cells with non-targeting guide RNAs and cells with targeting guide RNAs, we identified for 11 of the 30 target genes more than 5 differentially expressed genes with a minimal log fold change (see STAR Methods), implying that loss of these genes results in a detectable transcriptomic shift. It is likely that loss of the other 19 genes results in more subtle changes in only a few genes; however, even for “non-perturbed” genes with sufficient basal expression, we detected clear changes in the CRISPR target gene. We repeated the ECCITE-seq experiment focusing on the 11 genes with detectable transcriptomic shifts upon target gene perturbation. Combining both replicate experiments, we obtained 18,853 cells that expressed only one guide RNA with a median of 1,388 cells per target gene.Figure S5ECCITE-Seq Identifies Cholesterol Gene Signature Shared across Multiple Top-Ranked Genes, Related to Figure 5(A) Stacked violin plot of 11 genes shared in both ECCITE-seq experiments. Single-cells are grouped by unique guide RNA target gene label of cells with a single detected guide RNA. Target gene expression is highlighted in red. (B) Heatmap of Gene Set Enrichment Analysis results for genes downregulated in any of the indicated target gene perturbed cells. (C) Cholesterol (normalized by total protein) in A549ACE2 cells treated with amlodipine or vehicle (DMSO) for 24 hours. (D) Quantitative PCR (qPCR) of SARS-CoV-2 viral load present in A549ACE2 cells treated with amlodipine or DMSO for 24 hours and then infected with SARS-CoV-2 at MOI of 0.1. The qPCR was performed on cells collected at the indicated time (hours) post-infection (hpi) (n = 3 biological replicates, error bars indicate s.e.m.). (E) Plaque assays of SARS-CoV-2 viral load present in A549ACE2 cells treated with amlodipine or DMSO for 24 hours and then infected with SARS-CoV-2 using logarithmically diluted supernatants. (n = 3 biological replicates, error bars indicate s.e.m.). (F) Number of reads mapping to the indicated portion of the viral genome in A549ACE2 cells treated with amlodipine or DMSO. A representative sample is shown for each treatment (n = 3 biological replicate sequencing libraries). (G) Cell viability by Trypan Blue exclusion in A549ACE2 cells treated with amlodipine or DMSO for 24 hours (n = 3 biological replicates, error bars indicate s.e.m.). (H) Distance matrix of RNA-sequencing from A549ACE2 cells treated with amlodipine or DMSO for 24 hours and then infected at MOI 0.1 or mock infection (n = 3 biological replicate sequencing libraries for each treatment-infection group). Read counts were processed with the DESeq2 regularized-log transform before computing distances. (I) k-means clustering (k = 3) of the top 500 most variable genes across all 4 conditions (n = 3 biological replicate sequencing libraries for each treatment-infection group). For each cluster, we label the most enriched pathway (lowest p-value) and, for the genes in that pathway, we label the top 5 most variable genes. No significantly enriched pathways were found for Cluster 3.We found that loss of 6 of the “perturbed” genes—ATP6AP1, ATP6V1A, CCDC22, NPC1, PIK3C3, and RAB7A, which are part of the endosomal entry pathway—yielded similar gene expression signatures among upregulated differentially expressed genes (Figures 5B, 5C, and S5B). These 6 target gene perturbations all led to upregulation of pathways affecting lipid and cholesterol homeostasis (Figure 5C). Recently, we performed a large survey of >20,000 potential pharmacological treatments for COVID-19, and for compounds effective at preventing viral infection, we identified induction of the cholesterol biosynthesis pathway as a potential mechanism of viral inhibition (Hoagland et al., 2020). Loss-of-function mutations in these 6 genes may function through a similar mechanism (induction of cholesterol synthesis) that combats the virus-mediated suppression of cholesterol synthesis. Among the significant differentially expressed genes, we also found 61 genes from the enriched CRISPR screen genes (n = 20 genes upregulated; n = 41 genes downregulated) (Table S5). For example, NPC1, ATP6V1F, and ATP6V1G1 are upregulated in most of the 6 endosomal entry pathway gene-perturbed cells (Figure 5C), suggesting compensatory upregulation of related genes to mitigate target gene loss.To understand how these changes impact lipid production, we measured cholesterol levels in cells after CRISPR perturbation and found that loss of these genes increases cholesterol by between 10% and 50%, depending on the perturbation (Figure 5D). To show that increases in cholesterol leads to increased SARS-CoV-2 resistance, we treated A549ACE2 cells with amlodipine, a calcium-channel antagonist that increases intracellular cholesterol (Mori et al., 1988; Ranganathan et al., 1982). We verified that amlodipine increases cholesterol levels in A549ACE2 cells (Figure S5C) and found that pre-treatment with amlodipine results in reduced SARS-CoV-2 viral infection, as measured by qPCR for nucleocapsid RNA, plaque formation, and number of viral RNA reads from RNA-sequencing, with only a modest impact on cell viability (Figures S5D–S5G). RNA-sequencing of cells treated with amlodipine shows a similar differential gene expression profile as seen in our ECCITE-seq with CRISPR perturbations with the most significant upregulated pathway as cholesterol biosynthesis (Figures S5H and S5I).

Single-Cell Sequencing Identifies Cholesterol Biosynthesis as a Common Mechanism Underlying Multiple Enriched Genes from the CRISPR Screen

Next, we sought to understand the mechanisms underlying how individual genes identified in our loss-of-function screen prevent SARS-CoV-2 infection and if host gene loss alters cell transcriptional programs. For this, we utilized the expanded CRISPR-compatible cellular indexing of transcriptomes and epitopes by sequencing (ECCITE-seq) method to couple pooled CRISPR perturbations of our top hit genes with a single-cell transcriptomic and proteomic readout (Mimitou et al., 2019; Figure 5
A; Table S5). ECCITE-seq is a high-throughput approach to identify the molecular mechanisms and cellular pathways that drive infection resistance; the pooled format also provides a more controlled experiment that may be less susceptible to batch variation. Importantly, cells were infected at low MOI to maximize the fraction of cells that express a single guide RNA, and therefore can be assigned a specific gene perturbation. For this, we pooled all individual guide RNA plasmids (3 per target gene) used to validate our genome-scale screen and 9 non-targeting (NT) sgRNAs (Figure 4B; Table S3).Figure 5Single-Cell Transcriptomics (ECCITE-Seq) Identifies Shared Target Gene Signatures for Lipid and Cholesterol Regulation(A) Schematic of pooled CRISPR perturbations with expanded CRISPR-compatible cellular indexing of transcriptomes and epitopes by sequencing (ECCITE-seq). Adapted from Mimitou et al. (2019).(B) Single-cell mRNA expression heatmap showing the 100 most differentially upregulated genes (adjusted p value <0.01) for 200 randomized cells per selected target gene perturbation (for clarity, CCDC22, PIK3C3, RAB7, and TMEM165 perturbations are not shown). Labeled genes are either a top-ranked gene from the genome-wide CRISPR screens (red) or among the top 5 differentially expressed genes for a gene perturbation (black).(C) Heatmap summarizing gene set enrichment analysis results for genes upregulated in any of the indicated target gene-perturbed cells (all genes with p value <0.01 and with a limit to the 300 most differentially expressed genes; all enriched pathways with adjusted p value <10−13).(D) Cholesterol quantification in gene-perturbed cells (with the indicated guide RNAs), normalized to total protein.See also Figure S5 and Tables S5 and S6.

Single-Cell Transcriptomics (ECCITE-Seq) Identifies Shared Target Gene Signatures for Lipid and Cholesterol Regulation

(A) Schematic of pooled CRISPR perturbations with expanded CRISPR-compatible cellular indexing of transcriptomes and epitopes by sequencing (ECCITE-seq). Adapted from Mimitou et al. (2019).

(B) Single-cell mRNA expression heatmap showing the 100 most differentially upregulated genes (adjusted p value <0.01) for 200 randomized cells per selected target gene perturbation (for clarity, CCDC22, PIK3C3, RAB7, and TMEM165 perturbations are not shown). Labeled genes are either a top-ranked gene from the genome-wide CRISPR screens (red) or among the top 5 differentially expressed genes for a gene perturbation (black).

(C) Heatmap summarizing gene set enrichment analysis results for genes upregulated in any of the indicated target gene-perturbed cells (all genes with p value <0.01 and with a limit to the 300 most differentially expressed genes; all enriched pathways with adjusted p value <10−13).

(D) Cholesterol quantification in gene-perturbed cells (with the indicated guide RNAs), normalized to total protein.

See also Figure S5 and Tables S5 and S6.

In an initial ECCITE-seq experiment, we identified a median of 152 single cells per target gene. We observed specific reduction of target gene expression in cells grouped by target gene, indicating nonsense-mediated decay of transcripts with frameshift indel mutations after CRISPR modification (Figure S5
A). This effect was more pronounced for genes with higher expression. Using differential gene expression analysis between cells with non-targeting guide RNAs and cells with targeting guide RNAs, we identified for 11 of the 30 target genes more than 5 differentially expressed genes with a minimal log fold change (see STAR Methods), implying that loss of these genes results in a detectable transcriptomic shift. It is likely that loss of the other 19 genes results in more subtle changes in only a few genes; however, even for “non-perturbed” genes with sufficient basal expression, we detected clear changes in the CRISPR target gene. We repeated the ECCITE-seq experiment focusing on the 11 genes with detectable transcriptomic shifts upon target gene perturbation. Combining both replicate experiments, we obtained 18,853 cells that expressed only one guide RNA with a median of 1,388 cells per target gene.Figure S5ECCITE-Seq Identifies Cholesterol Gene Signature Shared across Multiple Top-Ranked Genes, Related to Figure 5(A) Stacked violin plot of 11 genes shared in both ECCITE-seq experiments. Single-cells are grouped by unique guide RNA target gene label of cells with a single detected guide RNA. Target gene expression is highlighted in red. (B) Heatmap of Gene Set Enrichment Analysis results for genes downregulated in any of the indicated target gene perturbed cells. (C) Cholesterol (normalized by total protein) in A549ACE2 cells treated with amlodipine or vehicle (DMSO) for 24 hours. (D) Quantitative PCR (qPCR) of SARS-CoV-2 viral load present in A549ACE2 cells treated with amlodipine or DMSO for 24 hours and then infected with SARS-CoV-2 at MOI of 0.1. The qPCR was performed on cells collected at the indicated time (hours) post-infection (hpi) (n = 3 biological replicates, error bars indicate s.e.m.). (E) Plaque assays of SARS-CoV-2 viral load present in A549ACE2 cells treated with amlodipine or DMSO for 24 hours and then infected with SARS-CoV-2 using logarithmically diluted supernatants. (n = 3 biological replicates, error bars indicate s.e.m.). (F) Number of reads mapping to the indicated portion of the viral genome in A549ACE2 cells treated with amlodipine or DMSO. A representative sample is shown for each treatment (n = 3 biological replicate sequencing libraries). (G) Cell viability by Trypan Blue exclusion in A549ACE2 cells treated with amlodipine or DMSO for 24 hours (n = 3 biological replicates, error bars indicate s.e.m.). (H) Distance matrix of RNA-sequencing from A549ACE2 cells treated with amlodipine or DMSO for 24 hours and then infected at MOI 0.1 or mock infection (n = 3 biological replicate sequencing libraries for each treatment-infection group). Read counts were processed with the DESeq2 regularized-log transform before computing distances. (I) k-means clustering (k = 3) of the top 500 most variable genes across all 4 conditions (n = 3 biological replicate sequencing libraries for each treatment-infection group). For each cluster, we label the most enriched pathway (lowest p-value) and, for the genes in that pathway, we label the top 5 most variable genes. No significantly enriched pathways were found for Cluster 3.

ECCITE-Seq Identifies Cholesterol Gene Signature Shared across Multiple Top-Ranked Genes, Related to Figure 5

(A) Stacked violin plot of 11 genes shared in both ECCITE-seq experiments. Single-cells are grouped by unique guide RNA target gene label of cells with a single detected guide RNA. Target gene expression is highlighted in red. (B) Heatmap of Gene Set Enrichment Analysis results for genes downregulated in any of the indicated target gene perturbed cells. (C) Cholesterol (normalized by total protein) in A549ACE2 cells treated with amlodipine or vehicle (DMSO) for 24 hours. (D) Quantitative PCR (qPCR) of SARS-CoV-2 viral load present in A549ACE2 cells treated with amlodipine or DMSO for 24 hours and then infected with SARS-CoV-2 at MOI of 0.1. The qPCR was performed on cells collected at the indicated time (hours) post-infection (hpi) (n = 3 biological replicates, error bars indicate s.e.m.). (E) Plaque assays of SARS-CoV-2 viral load present in A549ACE2 cells treated with amlodipine or DMSO for 24 hours and then infected with SARS-CoV-2 using logarithmically diluted supernatants. (n = 3 biological replicates, error bars indicate s.e.m.). (F) Number of reads mapping to the indicated portion of the viral genome in A549ACE2 cells treated with amlodipine or DMSO. A representative sample is shown for each treatment (n = 3 biological replicate sequencing libraries). (G) Cell viability by Trypan Blue exclusion in A549ACE2 cells treated with amlodipine or DMSO for 24 hours (n = 3 biological replicates, error bars indicate s.e.m.). (H) Distance matrix of RNA-sequencing from A549ACE2 cells treated with amlodipine or DMSO for 24 hours and then infected at MOI 0.1 or mock infection (n = 3 biological replicate sequencing libraries for each treatment-infection group). Read counts were processed with the DESeq2 regularized-log transform before computing distances. (I) k-means clustering (k = 3) of the top 500 most variable genes across all 4 conditions (n = 3 biological replicate sequencing libraries for each treatment-infection group). For each cluster, we label the most enriched pathway (lowest p-value) and, for the genes in that pathway, we label the top 5 most variable genes. No significantly enriched pathways were found for Cluster 3.

We found that loss of 6 of the “perturbed” genes—ATP6AP1, ATP6V1A, CCDC22, NPC1, PIK3C3, and RAB7A, which are part of the endosomal entry pathway—yielded similar gene expression signatures among upregulated differentially expressed genes (Figures 5B, 5C, and S5B). These 6 target gene perturbations all led to upregulation of pathways affecting lipid and cholesterol homeostasis (Figure 5C). Recently, we performed a large survey of >20,000 potential pharmacological treatments for COVID-19, and for compounds effective at preventing viral infection, we identified induction of the cholesterol biosynthesis pathway as a potential mechanism of viral inhibition (Hoagland et al., 2020). Loss-of-function mutations in these 6 genes may function through a similar mechanism (induction of cholesterol synthesis) that combats the virus-mediated suppression of cholesterol synthesis. Among the significant differentially expressed genes, we also found 61 genes from the enriched CRISPR screen genes (n = 20 genes upregulated; n = 41 genes downregulated) (Table S5). For example, NPC1, ATP6V1F, and ATP6V1G1 are upregulated in most of the 6 endosomal entry pathway gene-perturbed cells (Figure 5C), suggesting compensatory upregulation of related genes to mitigate target gene loss.

To understand how these changes impact lipid production, we measured cholesterol levels in cells after CRISPR perturbation and found that loss of these genes increases cholesterol by between 10% and 50%, depending on the perturbation (Figure 5D). To show that increases in cholesterol leads to increased SARS-CoV-2 resistance, we treated A549ACE2 cells with amlodipine, a calcium-channel antagonist that increases intracellular cholesterol (Mori et al., 1988; Ranganathan et al., 1982). We verified that amlodipine increases cholesterol levels in A549ACE2 cells (Figure S5C) and found that pre-treatment with amlodipine results in reduced SARS-CoV-2 viral infection, as measured by qPCR for nucleocapsid RNA, plaque formation, and number of viral RNA reads from RNA-sequencing, with only a modest impact on cell viability (Figures S5D–S5G). RNA-sequencing of cells treated with amlodipine shows a similar differential gene expression profile as seen in our ECCITE-seq with CRISPR perturbations with the most significant upregulated pathway as cholesterol biosynthesis (Figures S5H and S5I).

RAB7A Knockout Results in a Reduced Cell Surface Expression and an Increased Endosomal Accumulation of ACE2Next, we sought to determine if any of the top-ranked genes regulate cell surface expression of ACE2, because surface ACE2 is required for SARS-CoV-2 infection (Hoffmann et al., 2020a). To measure the cell surface expression of ACE2, we performed flow cytometry on A549 wild-type cells and a panel of A549ACE2 CRISPR-perturbed cells. ACE2 expression was detected in A549ACE2, but not in A549 wild-type cells, validating the antibody specificity (Figures 6
A and S6
). Flow cytometry analysis of the A549ACE2 CRISPR-perturbed lines revealed that RAB7A knockout cells have a significantly reduced cell surface expression of ACE2 compared to cells transduced with a non-targeting guide RNA (Figure 6B). Efficient Rab7a protein depletion across the RAB7A CRISPR-perturbed lines used was confirmed by western blot (Figure S6C).Figure 6RAB7A Loss Results in a Reduced Cell Surface Expression and an Increased Endosomal Accumulation of ACE2(A) Representative histograms of flow cytometry analysis to determine cell surface expression of ACE2 on A549 cell lines (A549 wild-type [WT], A549ACE2, and ACE2 with Cas9 and non-targeting [NT] or RAB7A-targeting guide RNAs). The dashed line indicates the gate between the ACE2-negative and -positive cells.(B) Fraction of ACE2+ cells (using gating shown in A). ACE2 expression level was normalized across all samples to the A549ACE2 cells transduced with non-targeting (NT) guides (n = 2–3 guide RNA-transduced lines per gene, error bars are SEM).(C) Representative images of immunofluorescence staining of ACE2 on A549ACE2 transduced with an NT or a RAB7A-targeting guide. In NT cells, ACE2 localizes at the cell membrane and in the cytoplasm, while in RAB7A-targeted cells, ACE2 shows a distinct pattern of localization to vesicles.(D) Percent of cells with ACE2 accumulation in vesicles in NT and RAB7A-transduced A549ACE2 cells (n = 2 biological replicates, error bars are SEM).(E) Representative images of immunofluorescence co-stained for ACE2, EEA1, and LysoTracker in A549ACE2 cells with a CRISPR guide RNA targeting RAB7A. ACE2 shows a distinct colocalization with EEA1 (an early endosome marker) and a less frequent colocalization with LysoTracker (a lysosomal maker).(F) Representative histograms of flow cytometry analysis to determine cell surface expression in Calu-3 cells. The dashed line indicates the gate between the ACE2-negative and -positive cells.(G) Fraction of ACE2+ cells (using gating shown in F). ACE2 expression level was normalized across all samples to the Calu-3 cells transduced with an NT guide (n = 3 biological replicates, error bars are SEM).(H) Representative images of immunofluorescence staining of ACE2 on Caco-2 cells transduced with a NT or a RAB7A-targeting guide. In NT cells, ACE2 localizes at the cell membrane and in the cytoplasm, whereas in RAB7A-targeted cells, ACE2 shows a distinct pattern of localization to vesicles.(I) Mean area of ACE2 foci in Caco-2 cells transduced with a NT or a RAB7A-targeting guide (n = 4 biological replicates, 80–105 cells per replicate were scored, error bars are SEM). Significance testing for (B) and (G) was performed with a one-way ANOVA (B: F = 9.8, p < 10−4; G: Calu-3: F = 378, p < 10−4, Caco-2: F = 222, p < 10−4) with false-discovery rate-corrected post hoc tests. Significance testing for (D) and (I) was performed with an unpaired t test. For all panels, ∗p ≤ 0.05, ∗∗p ≤ 0.01, ∗∗∗p ≤ 0.001, and ∗∗∗∗p ≤ 0.0001.See also Figure S6.Figure S6Flow Cytometry for Cell Surface ACE2 Expression and Protein Analysis of RAB7A Protein after CRISPR Targeting, Related to Figure 6(A) and (B) Flow cytometry gating strategy to quantify cell surface expression of ACE2. (A) Live cells were first gated by the forward and side scatter area, then doublets were excluded by gating with the forward scatter area and width. Viable cells were selected by gating on side scatter area and LIVE/DEAD violet. (B) Gating strategy to determine ACE2+ cells. The gate was position such that < 3% of A549 wild-type and > 85% of A549ACE2 cells were ACE2 positive. The same gating strategy was applied to all samples. (C) western blot on A549ACE2 cells perturbed with non-targeting (NT) or RAB7A-targeting guide RNAs and probed with a RAB7A antibody. GAPDH was used as loading control.Rab7a is a small GTPase that is involved in regulating cellular processes such as vesicular transport and membrane trafficking (Guerra and Bucci, 2016). We hypothesized that the observed reduction of ACE2 at the cell surface in RAB7A knockout cells may be due to impaired vesicular trafficking and accumulation of ACE2 in the cytoplasm. Immunofluorescence in the A549ACE2 polyclonal cell line revealed that ACE2 is detected in most cells. We also found that RAB7A knockout leads to an increased accumulation of ACE2 in the cytoplasm and in vesicle-like hollow structures reminiscent of endo-lysosomes (Figure 6C). We detected these accumulations in about 35% of RAB7A knockout cells with ACE2 staining, compared to the control where ACE2 was primarily localized at the plasma membrane and small cytoplasmic puncta (Figure 6D). Finally, we investigated which cellular compartments accumulate ACE2 in RAB7A knockout cells. Co-immunofluorescence images showed that in RAB7A knockout cells ACE2-containing vesicles often colocalize with EEA1, an early endosomal marker and less frequently with LysoTracker, a lysosome marker (Figure 6E).Because ACE2 was overexpressed in our A549ACE2 cells, we wondered whether Rab7a loss would lead to similar cytoplasmic sequestration in cells with endogenous ACE2 expression. Flow cytometry analysis in Caco-2 colon and Calu-3 lung cells revealed that RAB7A knockout cells have significantly reduced cell surface expression of ACE2 compared to cells transduced with a non-targeting guide RNA (Figures 6F and 6G). We also found that RAB7A knockout in Caco-2 cells results in larger cytoplasmic ACE2 puncta compared to the control cells, further supporting cytoplasmic accumulation of ACE2 in the absence of Rab7a (Figures 6H and 6I).

RAB7A Knockout Results in a Reduced Cell Surface Expression and an Increased Endosomal Accumulation of ACE2

Next, we sought to determine if any of the top-ranked genes regulate cell surface expression of ACE2, because surface ACE2 is required for SARS-CoV-2 infection (Hoffmann et al., 2020a). To measure the cell surface expression of ACE2, we performed flow cytometry on A549 wild-type cells and a panel of A549ACE2 CRISPR-perturbed cells. ACE2 expression was detected in A549ACE2, but not in A549 wild-type cells, validating the antibody specificity (Figures 6
A and S6
). Flow cytometry analysis of the A549ACE2 CRISPR-perturbed lines revealed that RAB7A knockout cells have a significantly reduced cell surface expression of ACE2 compared to cells transduced with a non-targeting guide RNA (Figure 6B). Efficient Rab7a protein depletion across the RAB7A CRISPR-perturbed lines used was confirmed by western blot (Figure S6C).Figure 6RAB7A Loss Results in a Reduced Cell Surface Expression and an Increased Endosomal Accumulation of ACE2(A) Representative histograms of flow cytometry analysis to determine cell surface expression of ACE2 on A549 cell lines (A549 wild-type [WT], A549ACE2, and ACE2 with Cas9 and non-targeting [NT] or RAB7A-targeting guide RNAs). The dashed line indicates the gate between the ACE2-negative and -positive cells.(B) Fraction of ACE2+ cells (using gating shown in A). ACE2 expression level was normalized across all samples to the A549ACE2 cells transduced with non-targeting (NT) guides (n = 2–3 guide RNA-transduced lines per gene, error bars are SEM).(C) Representative images of immunofluorescence staining of ACE2 on A549ACE2 transduced with an NT or a RAB7A-targeting guide. In NT cells, ACE2 localizes at the cell membrane and in the cytoplasm, while in RAB7A-targeted cells, ACE2 shows a distinct pattern of localization to vesicles.(D) Percent of cells with ACE2 accumulation in vesicles in NT and RAB7A-transduced A549ACE2 cells (n = 2 biological replicates, error bars are SEM).(E) Representative images of immunofluorescence co-stained for ACE2, EEA1, and LysoTracker in A549ACE2 cells with a CRISPR guide RNA targeting RAB7A. ACE2 shows a distinct colocalization with EEA1 (an early endosome marker) and a less frequent colocalization with LysoTracker (a lysosomal maker).(F) Representative histograms of flow cytometry analysis to determine cell surface expression in Calu-3 cells. The dashed line indicates the gate between the ACE2-negative and -positive cells.(G) Fraction of ACE2+ cells (using gating shown in F). ACE2 expression level was normalized across all samples to the Calu-3 cells transduced with an NT guide (n = 3 biological replicates, error bars are SEM).(H) Representative images of immunofluorescence staining of ACE2 on Caco-2 cells transduced with a NT or a RAB7A-targeting guide. In NT cells, ACE2 localizes at the cell membrane and in the cytoplasm, whereas in RAB7A-targeted cells, ACE2 shows a distinct pattern of localization to vesicles.(I) Mean area of ACE2 foci in Caco-2 cells transduced with a NT or a RAB7A-targeting guide (n = 4 biological replicates, 80–105 cells per replicate were scored, error bars are SEM). Significance testing for (B) and (G) was performed with a one-way ANOVA (B: F = 9.8, p < 10−4; G: Calu-3: F = 378, p < 10−4, Caco-2: F = 222, p < 10−4) with false-discovery rate-corrected post hoc tests. Significance testing for (D) and (I) was performed with an unpaired t test. For all panels, ∗p ≤ 0.05, ∗∗p ≤ 0.01, ∗∗∗p ≤ 0.001, and ∗∗∗∗p ≤ 0.0001.See also Figure S6.Figure S6Flow Cytometry for Cell Surface ACE2 Expression and Protein Analysis of RAB7A Protein after CRISPR Targeting, Related to Figure 6(A) and (B) Flow cytometry gating strategy to quantify cell surface expression of ACE2. (A) Live cells were first gated by the forward and side scatter area, then doublets were excluded by gating with the forward scatter area and width. Viable cells were selected by gating on side scatter area and LIVE/DEAD violet. (B) Gating strategy to determine ACE2+ cells. The gate was position such that < 3% of A549 wild-type and > 85% of A549ACE2 cells were ACE2 positive. The same gating strategy was applied to all samples. (C) western blot on A549ACE2 cells perturbed with non-targeting (NT) or RAB7A-targeting guide RNAs and probed with a RAB7A antibody. GAPDH was used as loading control.

RAB7A Loss Results in a Reduced Cell Surface Expression and an Increased Endosomal Accumulation of ACE2

(A) Representative histograms of flow cytometry analysis to determine cell surface expression of ACE2 on A549 cell lines (A549 wild-type [WT], A549ACE2, and ACE2 with Cas9 and non-targeting [NT] or RAB7A-targeting guide RNAs). The dashed line indicates the gate between the ACE2-negative and -positive cells.

(B) Fraction of ACE2+ cells (using gating shown in A). ACE2 expression level was normalized across all samples to the A549ACE2 cells transduced with non-targeting (NT) guides (n = 2–3 guide RNA-transduced lines per gene, error bars are SEM).

(C) Representative images of immunofluorescence staining of ACE2 on A549ACE2 transduced with an NT or a RAB7A-targeting guide. In NT cells, ACE2 localizes at the cell membrane and in the cytoplasm, while in RAB7A-targeted cells, ACE2 shows a distinct pattern of localization to vesicles.

(D) Percent of cells with ACE2 accumulation in vesicles in NT and RAB7A-transduced A549ACE2 cells (n = 2 biological replicates, error bars are SEM).

(E) Representative images of immunofluorescence co-stained for ACE2, EEA1, and LysoTracker in A549ACE2 cells with a CRISPR guide RNA targeting RAB7A. ACE2 shows a distinct colocalization with EEA1 (an early endosome marker) and a less frequent colocalization with LysoTracker (a lysosomal maker).

(F) Representative histograms of flow cytometry analysis to determine cell surface expression in Calu-3 cells. The dashed line indicates the gate between the ACE2-negative and -positive cells.

(G) Fraction of ACE2+ cells (using gating shown in F). ACE2 expression level was normalized across all samples to the Calu-3 cells transduced with an NT guide (n = 3 biological replicates, error bars are SEM).

(H) Representative images of immunofluorescence staining of ACE2 on Caco-2 cells transduced with a NT or a RAB7A-targeting guide. In NT cells, ACE2 localizes at the cell membrane and in the cytoplasm, whereas in RAB7A-targeted cells, ACE2 shows a distinct pattern of localization to vesicles.

(I) Mean area of ACE2 foci in Caco-2 cells transduced with a NT or a RAB7A-targeting guide (n = 4 biological replicates, 80–105 cells per replicate were scored, error bars are SEM). Significance testing for (B) and (G) was performed with a one-way ANOVA (B: F = 9.8, p < 10−4; G: Calu-3: F = 378, p < 10−4, Caco-2: F = 222, p < 10−4) with false-discovery rate-corrected post hoc tests. Significance testing for (D) and (I) was performed with an unpaired t test. For all panels, ∗p ≤ 0.05, ∗∗p ≤ 0.01, ∗∗∗p ≤ 0.001, and ∗∗∗∗p ≤ 0.0001.

See also Figure S6.

Flow Cytometry for Cell Surface ACE2 Expression and Protein Analysis of RAB7A Protein after CRISPR Targeting, Related to Figure 6

(A) and (B) Flow cytometry gating strategy to quantify cell surface expression of ACE2. (A) Live cells were first gated by the forward and side scatter area, then doublets were excluded by gating with the forward scatter area and width. Viable cells were selected by gating on side scatter area and LIVE/DEAD violet. (B) Gating strategy to determine ACE2+ cells. The gate was position such that < 3% of A549 wild-type and > 85% of A549ACE2 cells were ACE2 positive. The same gating strategy was applied to all samples. (C) western blot on A549ACE2 cells perturbed with non-targeting (NT) or RAB7A-targeting guide RNAs and probed with a RAB7A antibody. GAPDH was used as loading control.

Rab7a is a small GTPase that is involved in regulating cellular processes such as vesicular transport and membrane trafficking (Guerra and Bucci, 2016). We hypothesized that the observed reduction of ACE2 at the cell surface in RAB7A knockout cells may be due to impaired vesicular trafficking and accumulation of ACE2 in the cytoplasm. Immunofluorescence in the A549ACE2 polyclonal cell line revealed that ACE2 is detected in most cells. We also found that RAB7A knockout leads to an increased accumulation of ACE2 in the cytoplasm and in vesicle-like hollow structures reminiscent of endo-lysosomes (Figure 6C). We detected these accumulations in about 35% of RAB7A knockout cells with ACE2 staining, compared to the control where ACE2 was primarily localized at the plasma membrane and small cytoplasmic puncta (Figure 6D). Finally, we investigated which cellular compartments accumulate ACE2 in RAB7A knockout cells. Co-immunofluorescence images showed that in RAB7A knockout cells ACE2-containing vesicles often colocalize with EEA1, an early endosomal marker and less frequently with LysoTracker, a lysosome marker (Figure 6E).

Because ACE2 was overexpressed in our A549ACE2 cells, we wondered whether Rab7a loss would lead to similar cytoplasmic sequestration in cells with endogenous ACE2 expression. Flow cytometry analysis in Caco-2 colon and Calu-3 lung cells revealed that RAB7A knockout cells have significantly reduced cell surface expression of ACE2 compared to cells transduced with a non-targeting guide RNA (Figures 6F and 6G). We also found that RAB7A knockout in Caco-2 cells results in larger cytoplasmic ACE2 puncta compared to the control cells, further supporting cytoplasmic accumulation of ACE2 in the absence of Rab7a (Figures 6H and 6I).

DiscussionGiven the current COVID-19 global pandemic, there is an urgent need to better understand the complex relationships between host and virus genetic dependencies. We report a genome-wide loss-of-function screen in human lung cells that identified host genes required for SARS-CoV-2 viral infection. We selected and validated 30 genes that were ranked among the top 200 genes. To support the ability of the screen to identify key dependencies, some of the well-known host genes involved in SARS-CoV-2 Spike protein binding and entry such as the ACE2 receptor and Cathepsin L were among the top-scoring genes (Hoffmann et al., 2020a). One of the validated genes (SIGMAR) encodes the Sigma-1 receptor that was recently identified to be modulated by drugs effective against SARS-CoV-2 in vitro (Gordon et al., 2020). Overall, the top-ranked genes clustered within several protein complexes including vacuolar ATPases, Retromer and endosome, Commander, ARP2/3, PI3K, and others, highlighting both the critical importance of multiple genes within each pathway to viral pathogenesis and the diversity of molecular pathways involved in SARS-CoV-2 infection.Using a “minipool” CRISPR library of perturbations targeting top-ranked genes from the genome-scale CRISPR screen and single-cell transcriptomics, we identified a group of 6 genes (RAB7A, PIK3C3, NPC1, CCDC22, ATP6V1A, and ATP6AP1) that had a similar transcriptional signature—upregulation of the cholesterol synthesis pathway. By measuring the cholesterol levels, we found that CRISPR-driven loss of those 6 genes result in increased cellular cholesterol. Some of the 6 genes have previously been implicated in regulating low-density lipoprotein (LDL) cholesterol. For example, depletion of Rab7a leads to LDL accumulation in endosomes and NPC1 knockout cells show a reduction of cholesterol at the plasma membrane and an accumulation in the late endosome/lysosome compartments (Chang et al., 2005; Girard et al., 2014; Millard et al., 2000; Neufeld et al., 1996). We have recently, in an independent study, identified that SARS-CoV-2 infection negatively downregulates the cholesterol synthesis pathway, and viral infection can be counteracted by drug treatments that upregulate the same pathway (Hoagland et al., 2020). It is possible that changes in lipid composition directly impacts SARS-CoV-2 virion maturation and infectivity, as has been previously shown for hepatitis C and influenza A (Aizaki et al., 2008; Bajimaya et al., 2017). In this study, we showed that amlodipine, a calcium-channel antagonist, upregulates cholesterol levels and blocks SARS-CoV-2 infection. In addition, recent clinical studies have suggested that patients taking amlodipine or similar dihydropyridine calcium channel inhibitors have a reduced COVID-19 case fatality rate (Solaimanzadeh, 2020; Zhang et al., 2020). An important future research direction will be to further understand the relationship between cholesterol synthesis pathways and SARS-CoV-2.Furthermore, we screened a panel of the top-ranked genes and identified that Rab7a regulates cell surface expression of ACE2, likely by sequestering ACE2 in endosomal vesicles. Rab7a is involved in vesicular trafficking and its depletion has been shown to sequester other cell receptors in endosomes (Rush and Ceresa, 2013). Interestingly, RAB7A knockout cell lines showed both altered cholesterol biosynthesis and sequestration of ACE2 receptor. Previous proteomics work showed that Rab7a has a strong interaction with viral protein nsp7 (Gordon et al., 2020). However, there is no nsp7 in the incoming virion, implying a post-entry/post-translational role for Rab7a. Thus, it is possible that loss of Rab7a blocks SARS-CoV-2 pathogenesis via multiple separate pathways, which is supported by the observation that it is the top-performing gene in our arrayed validation.While this study was under review, a few other groups released preprints with loss-of-function CRISPR screens to identify host factors required for SARS-CoV-2 infection (Heaton et al., 2020; Wei et al., 2020; Zhu et al., 2020). Notably, only two studies (ours and Zhu et al. [2020]) have substantial overlap (11 and 14 genes among the top 36 genes from our MOI 0.3 and 0.01 screens, respectively, as shown in Figure S1D and nearly all of the top-ranked gene categories shown in Figure 2A). Given that these independent screens utilized different CRISPR libraries, the corroboration by Zhu et al. (2020) provides further support for our conclusions. The overlap between either of these studies with another screen performed using African green monkey cells (Wei et al., 2020) was limited to ACE2 and CTSL, two genes with established roles in viral entry. The differences in the overlap among the top-ranked genes might be due to technical aspects (such as different CRISPR libraries or variations in guide representation) or biological differences (such as different cell types or different host species). An exciting avenue for future study would be to investigate if SARS-CoV-2 perhaps utilizes multiple cell-type-specific genetic circuits.A key element in our study was harnessing genome-scale loss-of-function to develop more refined therapeutic hypothesis. Our study suggests that PIK3C3 is a promising drug target: four out of the seven PIK3C3 inhibitors resulted in more than 100-fold reduction of SARS-CoV-2 viral load (SAR405, Compound-19, PIK-III, and Autophinib). Using a PIK3C3 polyclonal knockout A549ACE2 cell line, we found that among the top four PIK3C3 inhibitors, SAR405 may have some off-target effects (Figure S4B). Considering that our polyclonal knockout line likely has some residual PIK3C3, future work will be required to test the inhibitor specificity using a PIK3C3 monoclonal knockout cell line. Another drug that shows a substantial reduction in SARS-CoV-2 viral load is tamoxifen. Tamoxifen is an FDA-approved drug given as prophylaxis to patients at risk of breast cancer and works via modulation of the estrogen receptor. Tamoxifen was included in our study as it targets protein kinase C as a secondary target (O’Brian et al., 1985). This mechanism is further supported by the observation that A549 cells have undetectable transcript levels of estrogen receptor 1 (Human Protein Atlas) (Uhlen et al., 2010). Considering that tamoxifen is typically given to patients for years as a cancer therapy and prophylactic (Marchant, 1976), it would be interesting to investigate if patients taking tamoxifen have a reduced risk of SARS-CoV-2 infection and/or display less severe symptoms post-infection.Finally, many approaches for therapeutic discovery have focused on large-scale screens of compound libraries. Even when promising therapeutic candidates are identified, it can be challenging to understand the mechanisms responsible for reducing viral pathogenesis. Our forward-genetics approach allows us to first identify key host genes, which can then be targeted through a diversity of methods such as small-molecule inhibitors, blocking antibodies, or gene knockdown. A key advantage of this approach is that the mechanism of action for any therapeutic is well-established from the outset.Taken together, our integrative study identifies essential host genes in SARS-CoV-2 viral pathogenesis and, through a broad range of analytic and experimental approaches, validates their central role in infection. We also identify potential mechanisms underlying top-ranked genes, including cholesterol synthesis and endosomal function. In addition to guiding new therapeutic targets to help end this pandemic, our study provides a framework for harnessing massively parallel genome editing to understand disease genetics and mechanisms.Limitations of StudyOur study identified host factors required for SARS-CoV-2 infection in human A549 cells that overexpress ACE2. Future work will be needed to explore limitations of our study. (1) Considering the ACE2 overexpression in our screen, it will be interesting to screen human cells expressing endogenous ACE2, which may potentially identify transcriptional regulators of ACE2. (2) The A549 cell line used in our CRISPR screen is a lung adenocarcinoma cell line. Given that various organs are affected by SARS-CoV-2, it will be helpful to understand whether there are tissue-specific host factors. (3) Although we show through multiple, distinct genetic perturbations that upregulation of the cholesterol biosynthesis pathway and increase in cellular cholesterol blocks SARS-CoV-2, the precise mechanisms of how changes in cholesterol disrupt viral infection remain to be elucidated. (4) Recent genome-wide association studies have uncovered human genetic variants associated with COVID-19 risk and severity. Since the majority of such variants are in noncoding regions, integrative analysis of genome-wide CRISPR screens may help pinpoint the causal genes through which these variants function.

Given the current COVID-19 global pandemic, there is an urgent need to better understand the complex relationships between host and virus genetic dependencies. We report a genome-wide loss-of-function screen in human lung cells that identified host genes required for SARS-CoV-2 viral infection. We selected and validated 30 genes that were ranked among the top 200 genes. To support the ability of the screen to identify key dependencies, some of the well-known host genes involved in SARS-CoV-2 Spike protein binding and entry such as the ACE2 receptor and Cathepsin L were among the top-scoring genes (Hoffmann et al., 2020a). One of the validated genes (SIGMAR) encodes the Sigma-1 receptor that was recently identified to be modulated by drugs effective against SARS-CoV-2 in vitro (Gordon et al., 2020). Overall, the top-ranked genes clustered within several protein complexes including vacuolar ATPases, Retromer and endosome, Commander, ARP2/3, PI3K, and others, highlighting both the critical importance of multiple genes within each pathway to viral pathogenesis and the diversity of molecular pathways involved in SARS-CoV-2 infection.

Using a “minipool” CRISPR library of perturbations targeting top-ranked genes from the genome-scale CRISPR screen and single-cell transcriptomics, we identified a group of 6 genes (RAB7A, PIK3C3, NPC1, CCDC22, ATP6V1A, and ATP6AP1) that had a similar transcriptional signature—upregulation of the cholesterol synthesis pathway. By measuring the cholesterol levels, we found that CRISPR-driven loss of those 6 genes result in increased cellular cholesterol. Some of the 6 genes have previously been implicated in regulating low-density lipoprotein (LDL) cholesterol. For example, depletion of Rab7a leads to LDL accumulation in endosomes and NPC1 knockout cells show a reduction of cholesterol at the plasma membrane and an accumulation in the late endosome/lysosome compartments (Chang et al., 2005; Girard et al., 2014; Millard et al., 2000; Neufeld et al., 1996). We have recently, in an independent study, identified that SARS-CoV-2 infection negatively downregulates the cholesterol synthesis pathway, and viral infection can be counteracted by drug treatments that upregulate the same pathway (Hoagland et al., 2020). It is possible that changes in lipid composition directly impacts SARS-CoV-2 virion maturation and infectivity, as has been previously shown for hepatitis C and influenza A (Aizaki et al., 2008; Bajimaya et al., 2017). In this study, we showed that amlodipine, a calcium-channel antagonist, upregulates cholesterol levels and blocks SARS-CoV-2 infection. In addition, recent clinical studies have suggested that patients taking amlodipine or similar dihydropyridine calcium channel inhibitors have a reduced COVID-19 case fatality rate (Solaimanzadeh, 2020; Zhang et al., 2020). An important future research direction will be to further understand the relationship between cholesterol synthesis pathways and SARS-CoV-2.

Furthermore, we screened a panel of the top-ranked genes and identified that Rab7a regulates cell surface expression of ACE2, likely by sequestering ACE2 in endosomal vesicles. Rab7a is involved in vesicular trafficking and its depletion has been shown to sequester other cell receptors in endosomes (Rush and Ceresa, 2013). Interestingly, RAB7A knockout cell lines showed both altered cholesterol biosynthesis and sequestration of ACE2 receptor. Previous proteomics work showed that Rab7a has a strong interaction with viral protein nsp7 (Gordon et al., 2020). However, there is no nsp7 in the incoming virion, implying a post-entry/post-translational role for Rab7a. Thus, it is possible that loss of Rab7a blocks SARS-CoV-2 pathogenesis via multiple separate pathways, which is supported by the observation that it is the top-performing gene in our arrayed validation.

While this study was under review, a few other groups released preprints with loss-of-function CRISPR screens to identify host factors required for SARS-CoV-2 infection (Heaton et al., 2020; Wei et al., 2020; Zhu et al., 2020). Notably, only two studies (ours and Zhu et al. [2020]) have substantial overlap (11 and 14 genes among the top 36 genes from our MOI 0.3 and 0.01 screens, respectively, as shown in Figure S1D and nearly all of the top-ranked gene categories shown in Figure 2A). Given that these independent screens utilized different CRISPR libraries, the corroboration by Zhu et al. (2020) provides further support for our conclusions. The overlap between either of these studies with another screen performed using African green monkey cells (Wei et al., 2020) was limited to ACE2 and CTSL, two genes with established roles in viral entry. The differences in the overlap among the top-ranked genes might be due to technical aspects (such as different CRISPR libraries or variations in guide representation) or biological differences (such as different cell types or different host species). An exciting avenue for future study would be to investigate if SARS-CoV-2 perhaps utilizes multiple cell-type-specific genetic circuits.

A key element in our study was harnessing genome-scale loss-of-function to develop more refined therapeutic hypothesis. Our study suggests that PIK3C3 is a promising drug target: four out of the seven PIK3C3 inhibitors resulted in more than 100-fold reduction of SARS-CoV-2 viral load (SAR405, Compound-19, PIK-III, and Autophinib). Using a PIK3C3 polyclonal knockout A549ACE2 cell line, we found that among the top four PIK3C3 inhibitors, SAR405 may have some off-target effects (Figure S4B). Considering that our polyclonal knockout line likely has some residual PIK3C3, future work will be required to test the inhibitor specificity using a PIK3C3 monoclonal knockout cell line. Another drug that shows a substantial reduction in SARS-CoV-2 viral load is tamoxifen. Tamoxifen is an FDA-approved drug given as prophylaxis to patients at risk of breast cancer and works via modulation of the estrogen receptor. Tamoxifen was included in our study as it targets protein kinase C as a secondary target (O’Brian et al., 1985). This mechanism is further supported by the observation that A549 cells have undetectable transcript levels of estrogen receptor 1 (Human Protein Atlas) (Uhlen et al., 2010). Considering that tamoxifen is typically given to patients for years as a cancer therapy and prophylactic (Marchant, 1976), it would be interesting to investigate if patients taking tamoxifen have a reduced risk of SARS-CoV-2 infection and/or display less severe symptoms post-infection.

Finally, many approaches for therapeutic discovery have focused on large-scale screens of compound libraries. Even when promising therapeutic candidates are identified, it can be challenging to understand the mechanisms responsible for reducing viral pathogenesis. Our forward-genetics approach allows us to first identify key host genes, which can then be targeted through a diversity of methods such as small-molecule inhibitors, blocking antibodies, or gene knockdown. A key advantage of this approach is that the mechanism of action for any therapeutic is well-established from the outset.

Taken together, our integrative study identifies essential host genes in SARS-CoV-2 viral pathogenesis and, through a broad range of analytic and experimental approaches, validates their central role in infection. We also identify potential mechanisms underlying top-ranked genes, including cholesterol synthesis and endosomal function. In addition to guiding new therapeutic targets to help end this pandemic, our study provides a framework for harnessing massively parallel genome editing to understand disease genetics and mechanisms.

Limitations of StudyOur study identified host factors required for SARS-CoV-2 infection in human A549 cells that overexpress ACE2. Future work will be needed to explore limitations of our study. (1) Considering the ACE2 overexpression in our screen, it will be interesting to screen human cells expressing endogenous ACE2, which may potentially identify transcriptional regulators of ACE2. (2) The A549 cell line used in our CRISPR screen is a lung adenocarcinoma cell line. Given that various organs are affected by SARS-CoV-2, it will be helpful to understand whether there are tissue-specific host factors. (3) Although we show through multiple, distinct genetic perturbations that upregulation of the cholesterol biosynthesis pathway and increase in cellular cholesterol blocks SARS-CoV-2, the precise mechanisms of how changes in cholesterol disrupt viral infection remain to be elucidated. (4) Recent genome-wide association studies have uncovered human genetic variants associated with COVID-19 risk and severity. Since the majority of such variants are in noncoding regions, integrative analysis of genome-wide CRISPR screens may help pinpoint the causal genes through which these variants function.

Limitations of Study

Our study identified host factors required for SARS-CoV-2 infection in human A549 cells that overexpress ACE2. Future work will be needed to explore limitations of our study. (1) Considering the ACE2 overexpression in our screen, it will be interesting to screen human cells expressing endogenous ACE2, which may potentially identify transcriptional regulators of ACE2. (2) The A549 cell line used in our CRISPR screen is a lung adenocarcinoma cell line. Given that various organs are affected by SARS-CoV-2, it will be helpful to understand whether there are tissue-specific host factors. (3) Although we show through multiple, distinct genetic perturbations that upregulation of the cholesterol biosynthesis pathway and increase in cellular cholesterol blocks SARS-CoV-2, the precise mechanisms of how changes in cholesterol disrupt viral infection remain to be elucidated. (4) Recent genome-wide association studies have uncovered human genetic variants associated with COVID-19 risk and severity. Since the majority of such variants are in noncoding regions, integrative analysis of genome-wide CRISPR screens may help pinpoint the causal genes through which these variants function.

STAR★MethodsKey Resources Table
REAGENT or RESOURCESOURCEIDENTIFIERAntibodiesSARS-CoV-2 nucleocapsid (N) antibody (clone 1C7C7)Center for Therapeutic Antibody Discovery at the Icahn School of Medicine at Mount SinaiN/AMouse anti-2A (clone 3H4)MilliporeCat#MABS2005Rabbit anti-GAPDH (clone 14C10)Cell SignalingCat#2118S; RRID: AB_561053IRDye 680RD donkey anti-rabbitLI-CORCat#926-68073; RRID: AB_10954442IRDye 800CW donkey anti-mouseLI-CORCat#926-32212; RRID: AB_621847Goat anti-human IgG Alexa Flour 568Thermo Fisher ScientificCat#A-21090; RRID: AB_2535746Goat anti-ACE2R&D SystemsCat#AF933; RRID: AB_355722Anti-Goat APCR&D SystemsCat#F0108; RRID: AB_573124Rabbit anti-EEA1Thermo Fisher ScientificCat#MA5-14794; RRID: AB_10985824Goat anti-Mouse IgG Alexa 488Jackson Immuno ResearchCat#115-545-003; RRID: AB_2338840Goat anti-Rabbit IgG Alexa 594Jackson Immuno ResearchCat#711-585-152; RRID: AB_2340621Rabbit anti-RAB7ANovusBioCat#NBP1-87174; RRID: AB_11004418Rabbit anti-CCDC22ProteintechCat#16636-1-AP; RRID: AB_2072065Rabbit anti-ATP6V1AProteintechCat#17115-1-AP; RRID: AB_2290195Rabbit anti-ACE2Thermo Fisher ScientificCat#MA5-32307; RRID: AB_2809589Mouse anti-beta tubulinThermo Fisher ScientificCat#32-2600; RRID: AB_2533072Bacterial and Virus StrainsNEB Stable CellsNew England BiolabsCat#C3040IChemicals, Peptides, and Recombinant ProteinsPolyethyleneiminePolysciencesCat#23966Taq B polymeraseEnzymaticsCAT#P7250LQ5 High-Fidelity DNA polymeraseNew England BiolabsCat#M0491LPageRuler pre-stained protein ladderThermo Fisher ScientificCat#26616Lipofectamine RNAiMAXThermo Fisher ScientificCat#13778075SerabelisibSelleckchemCat#S8581IdelalisibSelleckchemCat#S2226BuparlisibSelleckchemCat#S2247PIK-IIISelleckchemCat#S7683Compound 19 (VPS34 inhibitor 1)SelleckchemCat#S8456SAR405SelleckchemCat#S7682AutophinibSelleckchemCat#S8596OdanacatibSelleckchemCat#S1115SID 26681509MedChemExpressCat#HY-103353ALLNSelleckchemCat#S7386SotrastaurinSelleckchemCat#S2791EnzastaurinSelleckchemCat#S1055TamoxifenSelleckchemCat#S1238IlomastatSelleckchemCat#S7157Doxycycline HyclateSelleckchemCat#S4163GSK6853MedChemExpressCat#HY-100220OlanzapineSelleckchemCat#S2493Binimetinib (MEK162)SelleckchemCat#S7007MirdametinibSelleckchemCat#S1036GentamicinSelleckchemCat#S4030VorinostatSelleckchemCat#S1047BelinostatSelleckchemCat#S1085PanobinostatSelleckchemCat#S1030PracinostatSelleckchemCat#S1515BRD4354MedChemExpressCat#HY-112719TMP195SelleckchemCat#S8502RemdesivirBiovisionCat#B2997-1000AmlodipineSigmaCat#A5605Critical Commercial AssaysPierce BCA protein assay kitThermo Fisher ScientificCat#23225Trypan BlueThermo Fisher ScientificCat#15250061LysoTracker DeepRedThermo Fisher ScientificCat#L12492Cholesterol-Glo assayPromegaCat#J3190TruSeq Stranded mRNA Library Prep KitIlluminaCat#2004053410x Genomics 5′ kit (Chromium Single Cell Immune Profiling Solution v1.0)10x GenomicsCat #1000014, #1000020, #1000151LIVE/DEAD Fixable Violet Dead Stain KitThermo Fisher ScientificCat#L34864KAPA SYBR FAST One-step Universal qRT-PCR KitRocheCat#KK4652SYBR Fast qPCR (KAPA)RocheCat#KK4602SuperScript Reverse Transcriptase IIThermo Fisher ScientificCat#18064022Deposited DataGeCKOv2 CRISPR Screen in A549-ACE2 infected with SARS-CoV-2This studyGEO accession: GSE158298scRNA seq of A549-ACE CRISPR preturbed cellsThis studyGEO accession: GSE159519RNA seq of DMSO or Amplodipine treated A549-ACE2 cellsThis studyGEO accession: GSE159522Experimental Models: Cell LinesHEK293FTThermo Fisher ScientificCat#R70007Caco-2ATCCCat#HTB-37Vero E6ATCCCat#CRL-1586Calu-3ATCCCat#HTB-55A549-ACE2this studyN/AHuh7.5-ACE2Daniloski et al., 2020N/AOligonucleotidesSARS-2 qPCR N mRNA forward: 5′-CTCTTGTAGATCTGTTCTCTAAACGAAC-3′This studyN/ASARS-2 qPCR N mRNA reverse: 5′-GGTCCACCAAACGTAATGCG-3′This studyN/ABeta Tubulin forward: 5′-GCCTGGACCACAAGTTTGAC-3This studyN/ABeta Tubulin reverse: 5′-TGAAATTCTGGGAGCATGAC-3′This studyN/ARecombinant DNApHR-PGK-hACEThis studyAddgene 161612pCV-EF1a-hACE2-hygroDaniloski et al., 2020Addgene 161758pCC_01 - lenti-hU6-sgRNA(F+E)-EFS-Cas9-NLS2A-PuroLegut et al., 2020Addgene 139086GeCKOv2 A and B librariesSanjana et al., 2014Addgene 1000000048pMD2.GTrono Lab Packaging and Envelope PlasmidsAddgene 12259psPAX2Trono Lab Packaging and Envelope PlasmidsAddgene 12260Software and AlgorithmsMaGeCK pipeline was used for the analysis of the CRISPR screenChen et al., 2018N/AFlowJo v10BD BiosciencesN/AGraphPad Prism 8GraphPadN/ACellranger v3.0.110x GenomicsN/ASeurat R package v3.2Stuart et al., 2019N/ADESeq2Love et al., 2014N/ASTRING toolSzklarczyk et al., 2019N/A
Resource AvailabilityLead ContactFurther information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Neville Sanjana (neville@sanjanalab.org).Materials AvailabilityThe pHR-PGK-ACE2 vector generated in this study is available through Addgene (161612).Data and Code AvailabilityCRISPR screen, single cell RNA-sequencing/ECCITE-seq and bulk RNA-sequencing datasets are available on the GEO repository with accession numbers GSE158298, GSE159519 and GSE159522, respectively.Experimental Models and Subject DetailsMammalian cell lines and culture conditionsHuman alveolar basal epithelial carcinoma cells (A549, ATCC CCL-185), human hepatocellular carcinoma (Huh7.5, a kind gift from C. Rice), human colorectal carcinoma (Caco-2, ATCC HTB-37), lung adenocarcinoma (Calu-3, ATCC HTB-55), monkey kidney epithelial cells (Vero E6, ATCC CRL-1586) and human embryonic kidney cells HEK293FT (Thermo) were used in our study. HEK293FT, Huh7.5, and Caco-2 cells were maintained at 37°C and 5% CO2 in D10 media, which consists of DMEM (Caisson Labs) with 10% Serum Plus II Medium Supplement (Sigma-Aldrich). Calu-3 and Vero E6 cells were maintained in EMEM (ATCC) media with 10% Serum Plus II Medium Supplement (Sigma-Aldrich).Viral strainsSARS-related coronavirus 2 (SARS-CoV-2), isolate USA-WA1/2020 (NR-52281), used in the study (Blanco-Melo et al., 2020; Daniloski et al., 2020
). SARS-CoV-2 was grown in Vero E6 cells in DMEM supplemented with 2% FBS, 4.5 g/L D-glucose, 4 mM L-glutamine, 10 mM non-essential amino acids, 1 mM sodium pyruvate and 10 mM HEPES. Plaque assays were used to determine infectious titers of SARS-CoV-2 by infection of Vero E6 cells in Minimum Essential Media supplemented with 2% FBS, 4 mM L-glutamine, 0.2% BSA, 10 mM HEPES and 0.12% NaHCO3 and 0.7% agar.Method DetailsGeneration of A549ACE2 lineTo generate ACE2 expressing cells, the human ACE2 coding sequence was amplified and cloned into the BamHI site of the lentiviral vector pHR-PGK (Addgene 79120). We generated lentiviral particles (as described below) and transduced 5x104 A549 cells plated in a 12-well plate in the presence of polybrene (8 μg/ml). We confirmed hACE2 expression by western blot analysis (Thermo MA5-32307).Lentiviral production, transduction and SARS-CoV-2 infectionThe Human GeCKOv2 A and B libraries (Addgene 1000000048) were used for genome-scale CRISPR knockout screens (Sanjana et al., 2014). We mixed equal amount of the A and B library plasmids to target each gene with 6 guide RNAs. Briefly, 225cm2 flasks of 80% confluent HEK293FT cells (Thermo) were transfected with 25 μg GeCKOv2 plasmid, 14 μg pMD2.G and 20 μg psPAX2 mixed in 2.5 mL OptiMEM (Thermo) and 175 μL Polyethylenimine (1 mg/ml) (Polysciences). After 6 hours, media was changed to D10 media with 1% bovine serum albumin (Sigma) added to improve virus stability. After 60 hours, viral supernatants were harvested and centrifuged at 3,000 rpm at 4°C for 10 min to pellet cell debris debris and filtered using 45 μm PVDF filters (CellTreat). The supernatant was then ultracentrifuged for 2 hours at 100,000 g (Sorvall Lynx 6000) and the pellet resuspended overnight at 4°C in PBS with 1% BSA.Following lentiviral titration, 330 million A549ACE2 expressing cells were transduced at MOI of 0.5. Cells were selected with 2 μg/uL puromycin (Thermo) for 12 days to ensure proper selection. Throughout the experiment the representation was monitored such that each guide RNA is represented by at least 1000 cells (∼125 M cells).We infected GeCKOv2-transduced cells (∼1000x representation) with SARS-CoV-2 isolate USA-WA1/2020 (NR-52281) at either MOI of 0.01 (low MOI) and 0.3 (high MOI). Surviving cells were collected on day 6 post-infection for genomic DNA isolation.Genomic DNA isolation, guide RNA amplification and quantificationWe used a two-step PCR protocol (PCR1 and PCR2) to amplify the guide RNA cassette for Illumina sequencing from genomic DNA (gDNA). The gDNA was extracted from CRISPR screen cells using the following protocol (Chen et al., 2015): Per 100 million cells, 12 mL of NK Lysis Buffer (50 mM Tris, 50 mM EDTA, 1% SDS, pH 8) were used for cell lysis. Once cells were resuspended, 60 μL of 20 mg/ml Proteinase K (QIAGEN) was added and the sample was incubated at 55°C overnight. The next day, 60 μL of 20 μg/mL RNase A (QIAGEN) was added, mixed, and samples were incubated at 37°C for 30 min. Then 4 mL of pre-chilled 7.5 M ammonium acetate was added, samples were vortexed and spun at 4,000 g for 10 min. The supernatant was placed in a new tube, mixed well with 12 mL isopropanol and spun at 4,000 g for 10 min. DNA pellets were washed with 12 mL of 70% ethanol, spun, dried and pellets were resuspended with 0.2x TE buffer (Sigma).For the first PCR reaction, we used all gDNA available for each sample. We performed the PCR1 using Taq-B polymerase (Enzymatics) and used multiple reactions where each reaction contained up to 10 μg gDNA per 100 μL PCR reaction. PCR1 products for each sample were pooled and used for amplification with barcoded PCR2 primers. For each sample, we performed 12 PCR2 reactions (using 5 μL of the pooled PCR1 product per PCR2 reaction) with Q5 polymerase (NEB). PCR2 products were pooled and then normalized within each biological sample before combining uniquely-barcoded separate biological samples. The pooled product was then gel-purified from a 2% E-gel EX (Life Technologies) using the QiaQuick gel extraction kit (QIAGEN). The purified, pooled library was then quantified with Tapestation 4200 (Agilent Technologies). PCR products were run on a 2% agarose gel and the correct size band was extracted. Sequencing was performed on the NextSeq 550 instrument using the HighOutput Mode v2 with 75 bp paired-end reads (Illumina). For PCR1 the following primer set was used: 5′ GAGGGCCTATTTCCCATGATTC 3′ and 5′ GTTGCGAAAAAGAACGTTCACGG 3′. For PCR2 the following primers were used: 5′ AATGATACGGCGACCACCGAGATCTACACTCTTTCCCTACACGACGCTCTTCCGATCT (N1-9) (BC8) TCTTGTGGAAAGGACGAAACACCG 3′ and 5′ CAAGCAGAAGACGGCATACGAGAT (BC8) GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCT (N1-9) TCTACTATTCTTTCCCCTGCACTGT 3′, where N is a stagger of 1 to 9 nucleotides and BC is a barcode of 6 nucleotides.Sequencing reads were demultiplexed upon sequencing based on Illumina i7 barcodes present in PCR2 reverse primers using Illumina BaseSpace. We performed adaptor trimming by treating the hU6 promoter sequence as a 5′ adaptor, using cutadapt v1.13 [-e 0.2 -O 5 -m 20 -g TCTTGTGGAAAGGACGAAACACCG]. Processed guide RNA sequences were aligned to the GeCKOv2 reference allowing for up to 1 mismatch using bowtie v1.1.2 [-a–best–strata -v 1 –norc] with alignment rates of 81% to 86%.Computational analyses of genome-scale CRISPR screensGuide RNA counts were processed using the MaGeCK pipeline with an output of RRA p-values and gene ranks (Chen et al., 2018). We separately ranked genes using the RIGER (weighted-sum) and second-best rank methods (Chen et al., 2015; Luo et al., 2008). Gene Set Enrichment Analyses were performed using the fgsea package with Gene Ontology for biological processes (c5.bp.v7.1.symbols) (Korotkevich et al., 2019). GTEx v8 tissue specific enrichment was performed using the Multi Gene Query function available on the GTEx website: https://www.gtexportal.org/home/multiGeneQueryPage (accessed August 1st, 2020) (Aguet et al., 2019). GO enrichments for SARS-CoV-2, ZIKV, and IAV CRISPR screens were performed using GOrilla to find all significant enrichments (FDR p-value < 10-3) (Eden et al., 2009). FDR p-values were log10-transformed and normalized to create the heatmap shown in Figure 3E.Generation of gene-perturbed human cell lines and SARS-CoV-2 infectionFor validation by CRISPR knock-out (using 3 independent guide RNAs per gene), we selected genes that were highly-ranked (either in the low or high MOI screen or both) or genes that had not previously been implicated in viral pathogenesis or immune signaling. For each gene of interest, 3 guide RNAs were designed using GUIDES software and were subsequently cloned into an all-in-one vector with an optimized Cas9 scaffold (pCC_01, Addgene 139086) containing Cas9, a guide RNA cassette and a puro resistant cassette (Legut et al., 2020; Meier et al., 2017). Following sequence confirmation by Sanger sequencing, lentivirus was produced individually for each plasmid and the target cells were transduced in presence of polybrene (10 μg/mL, Santa Cruz). Gene-perturbed A549ACE2, Huh7.5ACE2, Calu-3 and Caco-2 cells were selected for at least 10 days with 2 μg/mL (A549ACE2, Calu-3), 3 μg/mL (Caco-2) or 5 μg/mL (Huh7.5ACE2) puromycin (Thermo). To determine the SARS-CoV-2 infection susceptibility of gene-perturbed lines, 10,000 cells were plated per well of 96-well plates. After 24 hours, the cells were infected with SARS-CoV-2 at MOI of 0.1. At 36 hours post-infection, the cells were either fixed and processed for immunofluorescence or cellular RNA was harvested for qRT-PCR analyses. For the multistep growth curves, cells were infected at an MOI of 0.1, and total RNA was harvested from infected cells at indicated times. RNA was processed for qRT-PCR as described above. All infections with SARS-CoV-2 were performed with at least 3 biological replicates.Immunofluorescence of nucleocapsid (N) proteinCells were fixed with 5% formaldehyde and immunostained for nucleocapsid (N) protein and visualized with a secondary antibody labeled with AlexaFlour-568 (Thermo). SARS-CoV-2 nucleocapsid (N) antibody (clone 1C7C7) was obtained from the Center for Therapeutic Antibody Discovery at the Icahn School of Medicine at Mount Sinai. Nuclei were stained with DAPI. Full wells were imaged and quantified for SARS-CoV-2 infected cells using a Celigo imaging cytometer (Nexcelom Biosciences). All infections with SARS-CoV-2 were performed with 3 biological replicates. Representative images from the top gene knockout hits were acquired using the EVOS M5000 Imaging System (Thermo).Identification of druggable genes and drug treatmentsTo identify druggable genes among the top CRISPR screen hits, we cross-referenced highly-ranked genes from the RRA analysis with the data table containing drugs and their gene targets was obtained from Drug Gene Interaction database (DGIdb, retrieved on June 3, 2020) as well as manual literature search (Cotto et al., 2018). Inhibitors that target the genes of interest (and remdesivir) was obtained from SelleckChem and MedChemExpress. The catalog number and vendor information is available in the Key Resource section. Amlodipine was obtained from Sigma (A5605).To test drug efficacy in reducing SARS-CoV-2 infection, 10,000 A549ACE2 cells were seeded per well of a 96-well plate. Cells were treated with inhibitors at 10 μM for two hours before infection and inhibitors were maintained throughout the course of infection. Cells were infected with SARS-CoV-2 at MOI of 0.1 for 36 hours and the cells were collected for analysis via qRT-PCR or processed for immunofluorescence (N protein and quantified by Celigo). For Figure 4E, the remdesivir data was collected in an independent experiment.Cell viability assaysCell viability following drug treatments was performed the same way as described above. Thirty-six hours post drug treatment the cells were collected and stained with LIVE-DEAD Violet (Thermo). Cell acquisition was performed using a Sony SH800S cell sorter with a 100 μm sorting chip. We used the following gating strategy: 1) We excluded the cell debris based on the forward and reverse scatter; 2) Doublets were excluded by plotting FSC height versus FCS area, 3) Dead cells were quantified using live-dead violet stain. For all samples, we recorded at least 5,000 cells that pass the gating criteria described above. Flow cytometry analyses were performed using FlowJo v10. Cell viability for A549-ACE2 treated with DMSO or 10μM amlodipine was performed 24 hours post treatment using Trypan Blue (Thermo Fisher) and automated cell counting (Nexcelom AutoT4).Quantitative reverse-transcription PCR (qRT-PCR) of viral RNARNA was extracted from cells grown in 96-well plates by using the RNeasy 96 Kit (QIAGEN) per the manufacturer’s instructions. RNA was reverse-transcribed and PCR amplified using SYBR Fast One-step Universal qRT-PCR Kit (KAPA/Roche). For amlodipine experiments, RNA was extracted using TRIzol Reagent (Invitrogen) and purified with the Direct-zol RNA Miniprep kit (Zymo Research) as per the manufacturer’s instructions (including the optional DNase I treatment). For amlodipine experiments, the RNA was reverse-transcribed using SuperScript Reverse Transcriptase II (Thermo Fisher) and oligo(dT) primers. The cDNA was diluted 1:20 before qRT-PCR was performed using SYBR Fast qPCR (KAPA/Roche). SARS-CoV-2 replication was assessed by using primers specific to the N mRNA (Forward 5′-CTCTTGTAGATCTGTTCTCTAAACGAAC-3′; Reverse 5′-GGTCCACCAAACGTAATGCG-3′). SARS-CoV-2 N mRNA levels were normalized to beta tubulin (Forward 5′-GCCTGGACCACAAGTTTGAC-3; Reverse 5′-TGAAATTCTGGGAGCATGAC-3′). Reactions were ran and analyzed on a Lightcycler 480 II Instrument (Roche). Relative quantification was calculated by comparing the cycle threshold (C
t) values using ΔΔCt. C
t values of subgenomic nucleocapsid transcript were normalized to C
t values of alpha-tubulin, and those values were normalized to viral transcript levels of DMSO-treated or non-targeting guide RNA-transduced control samples, and displayed as fold-reduction in viral transcript. Significance was determined using a one-way ANOVA with post hoc tests or a two-tailed unpaired Student’s t test, as appropriate.siRNA transfections and SARS-CoV-2 infectionAll of the siRNAs were ordered from Dharmacon and their catalog number can be found in the Key Resources Table. To knockdown individual genes, 10,000 A549ACE2 cells were seeded in 96-well plates and transfected with siRNAs using Lipofectamine RNAiMAX (Thermo) following the manufacturer’s protocol. Forty-eight hours later, the cells were infected with SARS-CoV-2 at an MOI of 0.1 for 36 hours. Cells were fixed with 5% formaldehyde, stained with nucleocapsid protein (clone 1C7C7, ISMMS), and visualized with AlexaFluor-568 conjugated secondary antibody (Thermo). Nuclei were stained with DAPI, and full wells were imaged with a Celigo imaging cytometer (Nexcelom Biosciences).Minipool CRISPR library and lentiviral productionTo generate the minipool, we combined equimolar amounts the same guide RNA vectors (cloned in pCC1) that we used for arrayed validation (see above Generation of gene-perturbed A549ACE2
lines and SARS-CoV-2 infection). In total, we combined 3 guide RNAs for each gene (either, 30 or 11 genes) plus 9 non-targeting guide RNAs. The non-targeting guide RNA plasmids were added at half the molar ratio of the other plasmids. Lentivirus was produced via transfection of the minipool with appropriate packaging plasmids (psPAX2: Addgene 12260; pMD2.G: Addgene 12259) using polyethylenimine (PEI) reagent in HEK293FT. The target A549ACE2 cells were transduced in presence of polybrene (10 μg/mL, Santa Cruz) (ECCITE-Seq experiment 1) or without (ECCITE-seq experiment 2) at very low MOI to ensure entry of a single virus per cell. The survival was below 5% after 2 days of selection with 2 μg/mL puromycin (Thermo) for both ECCITE-seq experiments. On day 10 post-transduction, the cells were collected and processed for ECCITE-seq.Expanded CRISPR-compatible Cellular Indexing of Transcriptomes and Epitopes (ECCITE-seq)To perform cell-hashing and ECCITE-seq (Mimitou et al., 2019; Stoeckius et al., 2018), we separated cells into several sub-pools and, for each pool, we resuspended 1 million cells in 100 μl staining buffer (2% BSA, 0.01% Tween in PBS). We then added 10 μl Human TruStain FcX Fc Receptor Blocking Solution (BioLegend) and incubated on ice for 10 minutes. We then added hashing antibodies (BioLegend), incubated on ice for 30 minutes, and washed cells 3 times with staining buffer. In total, we ran four lanes of a Chromium Single Cell Immune Profiling Solution v1.0 5′ kit (10x Genomics) targeting recovery of 12,000 cells per lane (superloading) in experiment 1 and 20,000 cells in experiment 2. Gene expression (mRNA), hashtags (Hashtag-derived oligos, HTOs) and guide RNA (Guide-derived oligos, GDOs) libraries were constructed (Smibert et al., 2019). Each replicate was sequenced on two NextSeq550 75-cycle high-output runs (Illumina). Sequencing reads from the mRNA library were mapped to the hg38 reference genome (Ensembl v97) using Cellranger (v3.0.1, 10x Genomics). To generate count matrices for HTO and GDO libraries, the CITE-seq-count package was used (https://github.com/Hoohm/CITE-seq-Count v1.4.2). Count matrices were then used as input into the Seurat R package (v3.2) for downstream analyses (Stuart et al., 2019).ECCITE-seq data analysisProcessing of initial ECCITE-seq experiment with 30 target genesCells with low quality metrics, high mitochondrial gene content (> 17.5%) and low number of genes detected (≤1800) were removed. The median number of detected genes was 3309. RNA counts were log-normalized and HTO counts were normalized using the centered log-ratio transformation approach, with margin = 2 (normalizing across cells). To identify cell doublets and assign experimental conditions to cells, we used the HTOseqDemux function (Stoeckius et al., 2018). HTOseqDemux-defined cell doublets and negatives were removed from any downstream analyses.Cellular guide RNA identity for cells in the ECCITE-seq pool was assigned based on GDO unique molecular identifier (UMI) counts. We considered a guide RNA detected with ≥ 16 UMI counts (the median UMI counts per guide RNA was 503 counts, 93% of cells with ≥ 1 sgRNA). We observed that 64% of the cells had > 1 detected guide RNA (29% 1 guide RNA, 27% 2 guide RNAs, 17% 3 guide RNAs). To maximize cell recovery, we retained cells with 1 - 3 guide RNAs per cell (n = 10,265). We collapsed the guide RNA information down to target gene level (3 guide RNA per target gene) in the following way: For all cells (with 1-3 guide RNAs), we assigned a unique target gene if all detected guide RNAs targeted the same gene, or if the additional guide RNA detected was a non-targeting (NT) guide RNA (4,715 cells: 4,013 cells with exactly 1 sgRNA and 702 cells with collapsed target gene assignment). All remaining cells assigned 2 or 3 separate target genes were discarded.Some of the top-ranked genes from the CRISPR screens may be endpoint genes and thus may have subtle phenotypes difficult to detect using single-cell transcriptomics. To address this issue, we used the initial ECCITE-seq experiment (n = 4,715 cells) and grouped cells by target gene. To identify target gene perturbations that lead to systemic transcriptomic changes, we determined the number of differentially expressed genes relative to cells with NT guide RNAs. We found that under stringent filtering conditions (non-adjusted p-value < 10−5 or adjusted p-value < 0.2), only 11 target genes showed more than 5 differentially expressed genes. The second ECCITE-seq experiment was conducted using guide RNAs solely for these.Joint processing of both ECCITE-seq experimentsCells with low quality metrics, high mitochondrial gene content (> 15%) and low or high number of genes detected (≤2000 and > 5000) were removed. The median number of detected genes was 3365, with a median of 13,532 UMIs per cell. HTO counts were normalized using the centered log-ratio transformation approach, with margin = 2 (normalizing across cells). To identify cell doublets and assign experimental conditions to cells, we used the HTOseqDemux function (Stoeckius et al., 2018). HTOseqDemux-defined cell doublets and negatives were removed from any downstream analyses. Guide RNA counts were normalized similar to HTO UMI counts. Cell-wise guide RNA assignment was achieved using the MultiSeqDemux function (McGinnis et al., 2019). We only retained cells assigned with a single HTO and a single guide RNA (ECCITE-seq experiment 1: n = 1,824 cells, ECCITE-seq experiment 2: n = 17,029 cells). Cells from ECCITE-Seq experiment 1 and experiment 2 were harmonized by log-normalizing, followed by the identification of the 2000 most variable features with “vst” selection method. Next, we we used FindIntegrationAnchors and IntegrateData with the first 40 principle components.In the harmonized dataset (n = 18,853 cells), we used FindMarkers to find differentially expressed genes between non-targeting cells and cells that belonged to a targeted gene class. We used up to 100 of the most upregulated genes with adjusted p-value < 0.01 and minimal logeFC threshold > 0.1 filter as input into the heatmap in Figure 5B. The same FindMarkers analysis (but using the non-adjusted p-value < 0.01 with a maximum of the 300 most significant DEGs) was used as input into the EnrichR package to run pathway analysis using the human WikiPathways database (v. 2019) (Kuleshov et al., 2016; Slenter et al., 2018). Figure 5C shows -log10 transformed p-values for the union of all enriched pathways with an adjusted p-value < 0.01 across the 11 target genes with perturbation signatures.Plaque assayInfectious titer in supernatant of SARS-CoV-2 infections (from A549ACE2) was determined by plaque assay in Vero E6 cells. Cells were seeded in 12-well plates before infection with logarithmically diluted supernatants in a total inoculum of 100 ul, rocking plates every 10 minutes for 1 hour. Then an overlay of Minimum Essential Media with 0.12% NaHCO3, 4 mM L-glutamine, 0.2% BSA, 10 mM HEPES and 0.7% OXOID agar (Thermo) was aliquoted into wells and plates were incubated for 48 hours before fixing with 5% formaldehyde. After 24 hours, plaques were stained with crystal violet (1% crystal violet in 20% EtOH) for 15 minutes before washing wells and counting plaques.Bulk RNA-sequencingRNA-sequencing libraries were constructed from 1ug of RNA per sample using the TruSeq Stranded mRNA Library Prep Kit (Illumina) according to the manufacturer’s instructions. Sequencing libraries were sequenced using an Illumina NextSeq 500 platform and fastq files were generated using bcl2fastq (Illumina). Alignment to the SARS-CoV-2 genome (GenBank: MN985325.1) were performed using bowtie (Langmead et al., 2009). Reads were aligned to hg19 using STAR aligner in the Basespace RNA-Seq Alignment application (Illumina) and processed using DESeq2 (Love et al., 2014). Sample distances were calculated using read counts from the regularized log transform in DESeq2. K-means clustering was perfomed using the kmeans function in R/RStudio. Enriched Reactome pathways were identified using the STRING tool (Szklarczyk et al., 2019).Measurement of cellular cholesterolCellular cholesterol was measured using the Cholesterol-Glo assay (Promega). For each cell line (CRISPR-perturbed or drug-treated), we counted 100,000 cells, washed with 1xPBS twice and resuspended in 100 μL cell lysis solution for 30 min at 37°C. Following the incubation, 10 μL of cell lysate was used to measure the cholesterol levels in 96 well plates in a final reaction volume of 100uL per well (without esterase). Luminescence was measured after incubating the plate for 1 hour at room temperature in the dark. Using the same lysis sample, the protein concentration was determined using the BCA assay (Thermo). The cholesterol levels were normalized to the respective protein concentration of each sample. A549ACE cells were incubated with 10 μM amlodipine (Sigma) or DMSO for 24 hours. Cholesterol levels were then determined as described above.Flow cytometry of ACE2 cell surface expressionCells were harvested, counted and about 100,000 cells were washed with Dulbecco’s phosphate-buffered saline (PBS, Caisson Labs) and then stained with LIVE/DEAD Violet stain (Thermo, 34864). Following a wash with 1x PBS, all subsequent washes and antibody dilutions were performed using 1x PBS supplemented with 2% FBS. Wild-type or Cas9-perturbed A549ACE2, Caco-2 and Calu-3 cells were stained for 30 min on ice with 0.25 μg of anti-ACE2 antibody (R&D Systems, AF933) in ∼50 μL residual volume. Following two washes with 1x PBS with 2% FBS, samples were stained on ice for 20 min with 7.5 μL of anti-goat-APC secondary antibody (R&D Systems, F0108). Cell acquisition and sorting was performed using a Sony SH800S cell sorter with a 100 μm sorting chip. We used the following gating strategy: 1) We excluded the cell debris based on the forward and reverse scatter; 2) Doublets were excluded by plotting FSC height versus FCS area, 3) Dead cells were excluded by live-dead stain. We recorded at least 5,000 cells for A549ACE2 and at least 3,000 cells for Calu-3 and Caco-2 that pass the gating criteria described above. Gates to determine ACE2-APC+ cells were set based on control A549 wild-type cells or only secondary antibody stained Calu-3 and Caco-2, where the percent of ACE2 positive cells was set as < 5% (background level). Flow cytometry analyses were performed using FlowJo v10.Immunofluorescence of ACE2 and endo/lysosomal markersA549ACE2 Cas9-transduced (specific gene or non-targeting) cell lines were seeded on poly-D-lysine-coated coverslips (Electron Microscopy Sciences) 24 hours before they were fixed with 4% formaldehyde (Sigma) diluted in 1x PBS (Caisson Labs) (Daniloski et al., 2020). Following 3 washes with 1x PBS, cells were blocked with 1x PBS with 2% BSA for 30 min at room temperature. Cells were stained with mouse anti-2A antibody diluted at 1:250 (clone 3H4, Millipore Sigma, MABS2005) to recognize ACE2-2A, and rabbit anti-EEA1 diluted at 1:100 (Thermo, MA5-14794). To detect ACE2 in Caco-2 cells, goat anti-ACE2 (R&D Systems, AF933) was used at 1.25 μg/mL. Cell were incubated with primary antibodies for 3 hours at room temperature in a moisturized chamber. Coverslips were then washed 5 times with 1x PBS with 2% BSA and then incubated with Alexa-conjugated secondary antibodies (Jackson ImmunoResearch) diluted at 1:1000 for 45 min at room temperature. Coverslips were then washed 5 times with 1x PBS with 2%BSA, and DAPI (Sigma) was added in the fourth wash at a concentration of 0.1 μg/mL. To stain the lysosomes, LysoTracker DeepRed (Thermo) was diluted to 70 nM in culture media and was added to actively growing cells on coverslips and were incubated for 1 hour at 37°C. Following incubation with LysoTracker, cells were processed for immunofluorescence as described above. Confocal images were acquired on a Zeiss LSM 780 with a 20x/0.8 Plan-Apochromat objective or a 63x/1.4 Plan-Apochromat objective (Zeiss). Epifluorescence images were acquired using the same system; both fitted with a Zeiss Axiocam 506 mono (Zeiss). Images were processed using Zen Black 2012 (Zeiss) and FIJI 2.1.0; Java 1.8.0-202 (Schindelin et al., 2012). Linear histogram adjustments were applied uniformly within experiments for clarity of presentation.ACE2 localization was determined on images taken on a Zeiss LSM 780 with a 20x/0.8 Plan-Apochromat objective using Zen Black (Zeiss). The A549ACE2 polyclonal cell line had both ACE2 positive and negative cells. Only ACE2 positive cells were then manually scored if they had a distinct accumulation of cytoplasmic vesicles-like hollow structures, compared to the non-targeting (NT) control. In the NT control, the ACE2 staining was seen on the plasma membrane and diffuse in the cytoplasm. The scoring was performed on 2 biological replicates (that is, two separate non-targeting controls and two separate Rab7A knockout lines engineered using 4 separate Cas9 guide RNAs). We scored ∼430 cells per replicate. To quantify the size of ACE2 foci, confocal microscopy images were thresholded using FIJI’s local-threshold (Phansalkar radius = 15) and then a watershed transform was applied. Foci were counted and measured using FIJI’s Analyze Particles function.Western blotsA549ACE2 or Huh7.5ACE2 cells were collected, washed with 1x PBS and lysed with TNE buffer (10 mM Tris-HCl, pH 7.4, 150 mM NaCl, 1mM EDTA, 1% Nonidet P-40) in presence of protease inhibitor cocktail (Bimake B14001) for 1 hour on ice. Cells lysates were spun for 10 min at 10,000 g, and protein concentration was determined with the BCA assay (Thermo). Equal amounts of cell lysates (20 μg) were denatured in Tris-Glycine SDS Sample buffer (Thermo LC2676), and loaded on a Novex 4 - 20% Tris-Glycine gel (Thermo XP04205BOX). PageRuler pre-stained protein ladder (Thermo 26616) was used to determine the protein size. The gel was run in 1x Tris-Glycine-SDS buffer (IBI Scientific IBI01160) for about 120 min at 120V. Proteins were transferred on a nitrocellulose membrane (BioRad 1620112) in presence of prechilled 1x Tris-Glycine transfer buffer (FisherSci LC3675) supplemented with 20% methanol for 100 min at 100V. Immunoblots were blocked with 5% skim milk dissolved in 1x PBS with 1% Tween-20 (PBST) and incubated overnight at 4°C separately with the following primary antibodies: rabbit anti-RAB7A (0.1 μg/mL, NovusBio, NBP1-87174), rabbit anti-GAPDH (0.1 μg/mL, Cell Signaling, 2118S), rabbit anti-CCDC22 (0.34 μg/mL, Proteintech, 16636-1-AP), rabbit anti-ATP6V1A (0.46 μg/mL, Proteintech, 17115-1-AP), rabbit anti-ACE2 (0.5 μg/mL, Invitrogen, MA5-32307), and mouse anti-beta tubulin (0.5 μg/mL, Invitrogen, 32-2600). Following the primary antibody, the blots were incubated with IRDye 680RD donkey anti-rabbit (0.2 μg/mL, LI-COR 926-68073) or with IRDye 800CW donkey anti-mouse (0.2 μg/mL, LI-COR 926-32212). The blots were imaged using Odyssey CLx (LI-COR).Quantification and Statistical AnalysisStatistical analysisStatistical parameters and details are reported in each figure legend. Generally, experiments were repeated with at least three biological replicates. Each plot includes points for individual biological replicates and mean ± s.e.m. error bars unless otherwise specified. In general, biological replicates consist of replicate viral transductions (CRISPR), replicate RNA transfections (siRNA), or replicate drug treatments (small molecule inhibitors) and are specified in each figure legend. For imaging experiments, the number of cells scored per biological replicate is given in the figure legend.When assessing significance of genetic perturbations and/or inhibitor treatments, we employed first a one-way or two-way ANOVA to assess significance and, when significant, post hoc comparisons were performed using the false-discovery rate method of Benjamini-Krieger-Yekutieli to correct for multiple comparisons. Before applying ANOVA, we first verified the equality of group variances using the Brown-Forsythe test. Details for all ANOVA and post hoc multiple comparisons are reported in Table S6; t tests are reported in the respective figure legends.Throughout the manuscript, we use r to denote the Pearson correlation and r
s to denote the Spearman correlation; all reported correlations were significant using the cor.test function in R. For the BioID interactome analysis, we derived an empirical p value by resampling from a null distribution (n = 10,000 gene shuffling iterations). Statistical analyses were performed in GraphPad Prism 8 and RStudio 1.2.5019.

STAR★Methods

Key Resources Table
REAGENT or RESOURCESOURCEIDENTIFIERAntibodiesSARS-CoV-2 nucleocapsid (N) antibody (clone 1C7C7)Center for Therapeutic Antibody Discovery at the Icahn School of Medicine at Mount SinaiN/AMouse anti-2A (clone 3H4)MilliporeCat#MABS2005Rabbit anti-GAPDH (clone 14C10)Cell SignalingCat#2118S; RRID: AB_561053IRDye 680RD donkey anti-rabbitLI-CORCat#926-68073; RRID: AB_10954442IRDye 800CW donkey anti-mouseLI-CORCat#926-32212; RRID: AB_621847Goat anti-human IgG Alexa Flour 568Thermo Fisher ScientificCat#A-21090; RRID: AB_2535746Goat anti-ACE2R&D SystemsCat#AF933; RRID: AB_355722Anti-Goat APCR&D SystemsCat#F0108; RRID: AB_573124Rabbit anti-EEA1Thermo Fisher ScientificCat#MA5-14794; RRID: AB_10985824Goat anti-Mouse IgG Alexa 488Jackson Immuno ResearchCat#115-545-003; RRID: AB_2338840Goat anti-Rabbit IgG Alexa 594Jackson Immuno ResearchCat#711-585-152; RRID: AB_2340621Rabbit anti-RAB7ANovusBioCat#NBP1-87174; RRID: AB_11004418Rabbit anti-CCDC22ProteintechCat#16636-1-AP; RRID: AB_2072065Rabbit anti-ATP6V1AProteintechCat#17115-1-AP; RRID: AB_2290195Rabbit anti-ACE2Thermo Fisher ScientificCat#MA5-32307; RRID: AB_2809589Mouse anti-beta tubulinThermo Fisher ScientificCat#32-2600; RRID: AB_2533072Bacterial and Virus StrainsNEB Stable CellsNew England BiolabsCat#C3040IChemicals, Peptides, and Recombinant ProteinsPolyethyleneiminePolysciencesCat#23966Taq B polymeraseEnzymaticsCAT#P7250LQ5 High-Fidelity DNA polymeraseNew England BiolabsCat#M0491LPageRuler pre-stained protein ladderThermo Fisher ScientificCat#26616Lipofectamine RNAiMAXThermo Fisher ScientificCat#13778075SerabelisibSelleckchemCat#S8581IdelalisibSelleckchemCat#S2226BuparlisibSelleckchemCat#S2247PIK-IIISelleckchemCat#S7683Compound 19 (VPS34 inhibitor 1)SelleckchemCat#S8456SAR405SelleckchemCat#S7682AutophinibSelleckchemCat#S8596OdanacatibSelleckchemCat#S1115SID 26681509MedChemExpressCat#HY-103353ALLNSelleckchemCat#S7386SotrastaurinSelleckchemCat#S2791EnzastaurinSelleckchemCat#S1055TamoxifenSelleckchemCat#S1238IlomastatSelleckchemCat#S7157Doxycycline HyclateSelleckchemCat#S4163GSK6853MedChemExpressCat#HY-100220OlanzapineSelleckchemCat#S2493Binimetinib (MEK162)SelleckchemCat#S7007MirdametinibSelleckchemCat#S1036GentamicinSelleckchemCat#S4030VorinostatSelleckchemCat#S1047BelinostatSelleckchemCat#S1085PanobinostatSelleckchemCat#S1030PracinostatSelleckchemCat#S1515BRD4354MedChemExpressCat#HY-112719TMP195SelleckchemCat#S8502RemdesivirBiovisionCat#B2997-1000AmlodipineSigmaCat#A5605Critical Commercial AssaysPierce BCA protein assay kitThermo Fisher ScientificCat#23225Trypan BlueThermo Fisher ScientificCat#15250061LysoTracker DeepRedThermo Fisher ScientificCat#L12492Cholesterol-Glo assayPromegaCat#J3190TruSeq Stranded mRNA Library Prep KitIlluminaCat#2004053410x Genomics 5′ kit (Chromium Single Cell Immune Profiling Solution v1.0)10x GenomicsCat #1000014, #1000020, #1000151LIVE/DEAD Fixable Violet Dead Stain KitThermo Fisher ScientificCat#L34864KAPA SYBR FAST One-step Universal qRT-PCR KitRocheCat#KK4652SYBR Fast qPCR (KAPA)RocheCat#KK4602SuperScript Reverse Transcriptase IIThermo Fisher ScientificCat#18064022Deposited DataGeCKOv2 CRISPR Screen in A549-ACE2 infected with SARS-CoV-2This studyGEO accession: GSE158298scRNA seq of A549-ACE CRISPR preturbed cellsThis studyGEO accession: GSE159519RNA seq of DMSO or Amplodipine treated A549-ACE2 cellsThis studyGEO accession: GSE159522Experimental Models: Cell LinesHEK293FTThermo Fisher ScientificCat#R70007Caco-2ATCCCat#HTB-37Vero E6ATCCCat#CRL-1586Calu-3ATCCCat#HTB-55A549-ACE2this studyN/AHuh7.5-ACE2Daniloski et al., 2020N/AOligonucleotidesSARS-2 qPCR N mRNA forward: 5′-CTCTTGTAGATCTGTTCTCTAAACGAAC-3′This studyN/ASARS-2 qPCR N mRNA reverse: 5′-GGTCCACCAAACGTAATGCG-3′This studyN/ABeta Tubulin forward: 5′-GCCTGGACCACAAGTTTGAC-3This studyN/ABeta Tubulin reverse: 5′-TGAAATTCTGGGAGCATGAC-3′This studyN/ARecombinant DNApHR-PGK-hACEThis studyAddgene 161612pCV-EF1a-hACE2-hygroDaniloski et al., 2020Addgene 161758pCC_01 - lenti-hU6-sgRNA(F+E)-EFS-Cas9-NLS2A-PuroLegut et al., 2020Addgene 139086GeCKOv2 A and B librariesSanjana et al., 2014Addgene 1000000048pMD2.GTrono Lab Packaging and Envelope PlasmidsAddgene 12259psPAX2Trono Lab Packaging and Envelope PlasmidsAddgene 12260Software and AlgorithmsMaGeCK pipeline was used for the analysis of the CRISPR screenChen et al., 2018N/AFlowJo v10BD BiosciencesN/AGraphPad Prism 8GraphPadN/ACellranger v3.0.110x GenomicsN/ASeurat R package v3.2Stuart et al., 2019N/ADESeq2Love et al., 2014N/ASTRING toolSzklarczyk et al., 2019N/A

Key Resources Table

REAGENT or RESOURCESOURCEIDENTIFIERAntibodiesSARS-CoV-2 nucleocapsid (N) antibody (clone 1C7C7)Center for Therapeutic Antibody Discovery at the Icahn School of Medicine at Mount SinaiN/AMouse anti-2A (clone 3H4)MilliporeCat#MABS2005Rabbit anti-GAPDH (clone 14C10)Cell SignalingCat#2118S; RRID: AB_561053IRDye 680RD donkey anti-rabbitLI-CORCat#926-68073; RRID: AB_10954442IRDye 800CW donkey anti-mouseLI-CORCat#926-32212; RRID: AB_621847Goat anti-human IgG Alexa Flour 568Thermo Fisher ScientificCat#A-21090; RRID: AB_2535746Goat anti-ACE2R&D SystemsCat#AF933; RRID: AB_355722Anti-Goat APCR&D SystemsCat#F0108; RRID: AB_573124Rabbit anti-EEA1Thermo Fisher ScientificCat#MA5-14794; RRID: AB_10985824Goat anti-Mouse IgG Alexa 488Jackson Immuno ResearchCat#115-545-003; RRID: AB_2338840Goat anti-Rabbit IgG Alexa 594Jackson Immuno ResearchCat#711-585-152; RRID: AB_2340621Rabbit anti-RAB7ANovusBioCat#NBP1-87174; RRID: AB_11004418Rabbit anti-CCDC22ProteintechCat#16636-1-AP; RRID: AB_2072065Rabbit anti-ATP6V1AProteintechCat#17115-1-AP; RRID: AB_2290195Rabbit anti-ACE2Thermo Fisher ScientificCat#MA5-32307; RRID: AB_2809589Mouse anti-beta tubulinThermo Fisher ScientificCat#32-2600; RRID: AB_2533072Bacterial and Virus StrainsNEB Stable CellsNew England BiolabsCat#C3040IChemicals, Peptides, and Recombinant ProteinsPolyethyleneiminePolysciencesCat#23966Taq B polymeraseEnzymaticsCAT#P7250LQ5 High-Fidelity DNA polymeraseNew England BiolabsCat#M0491LPageRuler pre-stained protein ladderThermo Fisher ScientificCat#26616Lipofectamine RNAiMAXThermo Fisher ScientificCat#13778075SerabelisibSelleckchemCat#S8581IdelalisibSelleckchemCat#S2226BuparlisibSelleckchemCat#S2247PIK-IIISelleckchemCat#S7683Compound 19 (VPS34 inhibitor 1)SelleckchemCat#S8456SAR405SelleckchemCat#S7682AutophinibSelleckchemCat#S8596OdanacatibSelleckchemCat#S1115SID 26681509MedChemExpressCat#HY-103353ALLNSelleckchemCat#S7386SotrastaurinSelleckchemCat#S2791EnzastaurinSelleckchemCat#S1055TamoxifenSelleckchemCat#S1238IlomastatSelleckchemCat#S7157Doxycycline HyclateSelleckchemCat#S4163GSK6853MedChemExpressCat#HY-100220OlanzapineSelleckchemCat#S2493Binimetinib (MEK162)SelleckchemCat#S7007MirdametinibSelleckchemCat#S1036GentamicinSelleckchemCat#S4030VorinostatSelleckchemCat#S1047BelinostatSelleckchemCat#S1085PanobinostatSelleckchemCat#S1030PracinostatSelleckchemCat#S1515BRD4354MedChemExpressCat#HY-112719TMP195SelleckchemCat#S8502RemdesivirBiovisionCat#B2997-1000AmlodipineSigmaCat#A5605Critical Commercial AssaysPierce BCA protein assay kitThermo Fisher ScientificCat#23225Trypan BlueThermo Fisher ScientificCat#15250061LysoTracker DeepRedThermo Fisher ScientificCat#L12492Cholesterol-Glo assayPromegaCat#J3190TruSeq Stranded mRNA Library Prep KitIlluminaCat#2004053410x Genomics 5′ kit (Chromium Single Cell Immune Profiling Solution v1.0)10x GenomicsCat #1000014, #1000020, #1000151LIVE/DEAD Fixable Violet Dead Stain KitThermo Fisher ScientificCat#L34864KAPA SYBR FAST One-step Universal qRT-PCR KitRocheCat#KK4652SYBR Fast qPCR (KAPA)RocheCat#KK4602SuperScript Reverse Transcriptase IIThermo Fisher ScientificCat#18064022Deposited DataGeCKOv2 CRISPR Screen in A549-ACE2 infected with SARS-CoV-2This studyGEO accession: GSE158298scRNA seq of A549-ACE CRISPR preturbed cellsThis studyGEO accession: GSE159519RNA seq of DMSO or Amplodipine treated A549-ACE2 cellsThis studyGEO accession: GSE159522Experimental Models: Cell LinesHEK293FTThermo Fisher ScientificCat#R70007Caco-2ATCCCat#HTB-37Vero E6ATCCCat#CRL-1586Calu-3ATCCCat#HTB-55A549-ACE2this studyN/AHuh7.5-ACE2Daniloski et al., 2020N/AOligonucleotidesSARS-2 qPCR N mRNA forward: 5′-CTCTTGTAGATCTGTTCTCTAAACGAAC-3′This studyN/ASARS-2 qPCR N mRNA reverse: 5′-GGTCCACCAAACGTAATGCG-3′This studyN/ABeta Tubulin forward: 5′-GCCTGGACCACAAGTTTGAC-3This studyN/ABeta Tubulin reverse: 5′-TGAAATTCTGGGAGCATGAC-3′This studyN/ARecombinant DNApHR-PGK-hACEThis studyAddgene 161612pCV-EF1a-hACE2-hygroDaniloski et al., 2020Addgene 161758pCC_01 - lenti-hU6-sgRNA(F+E)-EFS-Cas9-NLS2A-PuroLegut et al., 2020Addgene 139086GeCKOv2 A and B librariesSanjana et al., 2014Addgene 1000000048pMD2.GTrono Lab Packaging and Envelope PlasmidsAddgene 12259psPAX2Trono Lab Packaging and Envelope PlasmidsAddgene 12260Software and AlgorithmsMaGeCK pipeline was used for the analysis of the CRISPR screenChen et al., 2018N/AFlowJo v10BD BiosciencesN/AGraphPad Prism 8GraphPadN/ACellranger v3.0.110x GenomicsN/ASeurat R package v3.2Stuart et al., 2019N/ADESeq2Love et al., 2014N/ASTRING toolSzklarczyk et al., 2019N/A

Resource AvailabilityLead ContactFurther information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Neville Sanjana (neville@sanjanalab.org).Materials AvailabilityThe pHR-PGK-ACE2 vector generated in this study is available through Addgene (161612).Data and Code AvailabilityCRISPR screen, single cell RNA-sequencing/ECCITE-seq and bulk RNA-sequencing datasets are available on the GEO repository with accession numbers GSE158298, GSE159519 and GSE159522, respectively.

Resource Availability

Lead ContactFurther information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Neville Sanjana (neville@sanjanalab.org).

Lead Contact

Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Neville Sanjana (neville@sanjanalab.org).

Materials AvailabilityThe pHR-PGK-ACE2 vector generated in this study is available through Addgene (161612).

Materials Availability

The pHR-PGK-ACE2 vector generated in this study is available through Addgene (161612).

Data and Code AvailabilityCRISPR screen, single cell RNA-sequencing/ECCITE-seq and bulk RNA-sequencing datasets are available on the GEO repository with accession numbers GSE158298, GSE159519 and GSE159522, respectively.

Data and Code Availability

CRISPR screen, single cell RNA-sequencing/ECCITE-seq and bulk RNA-sequencing datasets are available on the GEO repository with accession numbers GSE158298, GSE159519 and GSE159522, respectively.

Experimental Models and Subject DetailsMammalian cell lines and culture conditionsHuman alveolar basal epithelial carcinoma cells (A549, ATCC CCL-185), human hepatocellular carcinoma (Huh7.5, a kind gift from C. Rice), human colorectal carcinoma (Caco-2, ATCC HTB-37), lung adenocarcinoma (Calu-3, ATCC HTB-55), monkey kidney epithelial cells (Vero E6, ATCC CRL-1586) and human embryonic kidney cells HEK293FT (Thermo) were used in our study. HEK293FT, Huh7.5, and Caco-2 cells were maintained at 37°C and 5% CO2 in D10 media, which consists of DMEM (Caisson Labs) with 10% Serum Plus II Medium Supplement (Sigma-Aldrich). Calu-3 and Vero E6 cells were maintained in EMEM (ATCC) media with 10% Serum Plus II Medium Supplement (Sigma-Aldrich).Viral strainsSARS-related coronavirus 2 (SARS-CoV-2), isolate USA-WA1/2020 (NR-52281), used in the study (Blanco-Melo et al., 2020; Daniloski et al., 2020
). SARS-CoV-2 was grown in Vero E6 cells in DMEM supplemented with 2% FBS, 4.5 g/L D-glucose, 4 mM L-glutamine, 10 mM non-essential amino acids, 1 mM sodium pyruvate and 10 mM HEPES. Plaque assays were used to determine infectious titers of SARS-CoV-2 by infection of Vero E6 cells in Minimum Essential Media supplemented with 2% FBS, 4 mM L-glutamine, 0.2% BSA, 10 mM HEPES and 0.12% NaHCO3 and 0.7% agar.

Experimental Models and Subject Details

Mammalian cell lines and culture conditionsHuman alveolar basal epithelial carcinoma cells (A549, ATCC CCL-185), human hepatocellular carcinoma (Huh7.5, a kind gift from C. Rice), human colorectal carcinoma (Caco-2, ATCC HTB-37), lung adenocarcinoma (Calu-3, ATCC HTB-55), monkey kidney epithelial cells (Vero E6, ATCC CRL-1586) and human embryonic kidney cells HEK293FT (Thermo) were used in our study. HEK293FT, Huh7.5, and Caco-2 cells were maintained at 37°C and 5% CO2 in D10 media, which consists of DMEM (Caisson Labs) with 10% Serum Plus II Medium Supplement (Sigma-Aldrich). Calu-3 and Vero E6 cells were maintained in EMEM (ATCC) media with 10% Serum Plus II Medium Supplement (Sigma-Aldrich).

Mammalian cell lines and culture conditions

Human alveolar basal epithelial carcinoma cells (A549, ATCC CCL-185), human hepatocellular carcinoma (Huh7.5, a kind gift from C. Rice), human colorectal carcinoma (Caco-2, ATCC HTB-37), lung adenocarcinoma (Calu-3, ATCC HTB-55), monkey kidney epithelial cells (Vero E6, ATCC CRL-1586) and human embryonic kidney cells HEK293FT (Thermo) were used in our study. HEK293FT, Huh7.5, and Caco-2 cells were maintained at 37°C and 5% CO2 in D10 media, which consists of DMEM (Caisson Labs) with 10% Serum Plus II Medium Supplement (Sigma-Aldrich). Calu-3 and Vero E6 cells were maintained in EMEM (ATCC) media with 10% Serum Plus II Medium Supplement (Sigma-Aldrich).

Viral strainsSARS-related coronavirus 2 (SARS-CoV-2), isolate USA-WA1/2020 (NR-52281), used in the study (Blanco-Melo et al., 2020; Daniloski et al., 2020
). SARS-CoV-2 was grown in Vero E6 cells in DMEM supplemented with 2% FBS, 4.5 g/L D-glucose, 4 mM L-glutamine, 10 mM non-essential amino acids, 1 mM sodium pyruvate and 10 mM HEPES. Plaque assays were used to determine infectious titers of SARS-CoV-2 by infection of Vero E6 cells in Minimum Essential Media supplemented with 2% FBS, 4 mM L-glutamine, 0.2% BSA, 10 mM HEPES and 0.12% NaHCO3 and 0.7% agar.

Viral strains

SARS-related coronavirus 2 (SARS-CoV-2), isolate USA-WA1/2020 (NR-52281), used in the study (Blanco-Melo et al., 2020; Daniloski et al., 2020
). SARS-CoV-2 was grown in Vero E6 cells in DMEM supplemented with 2% FBS, 4.5 g/L D-glucose, 4 mM L-glutamine, 10 mM non-essential amino acids, 1 mM sodium pyruvate and 10 mM HEPES. Plaque assays were used to determine infectious titers of SARS-CoV-2 by infection of Vero E6 cells in Minimum Essential Media supplemented with 2% FBS, 4 mM L-glutamine, 0.2% BSA, 10 mM HEPES and 0.12% NaHCO3 and 0.7% agar.

Method DetailsGeneration of A549ACE2 lineTo generate ACE2 expressing cells, the human ACE2 coding sequence was amplified and cloned into the BamHI site of the lentiviral vector pHR-PGK (Addgene 79120). We generated lentiviral particles (as described below) and transduced 5x104 A549 cells plated in a 12-well plate in the presence of polybrene (8 μg/ml). We confirmed hACE2 expression by western blot analysis (Thermo MA5-32307).Lentiviral production, transduction and SARS-CoV-2 infectionThe Human GeCKOv2 A and B libraries (Addgene 1000000048) were used for genome-scale CRISPR knockout screens (Sanjana et al., 2014). We mixed equal amount of the A and B library plasmids to target each gene with 6 guide RNAs. Briefly, 225cm2 flasks of 80% confluent HEK293FT cells (Thermo) were transfected with 25 μg GeCKOv2 plasmid, 14 μg pMD2.G and 20 μg psPAX2 mixed in 2.5 mL OptiMEM (Thermo) and 175 μL Polyethylenimine (1 mg/ml) (Polysciences). After 6 hours, media was changed to D10 media with 1% bovine serum albumin (Sigma) added to improve virus stability. After 60 hours, viral supernatants were harvested and centrifuged at 3,000 rpm at 4°C for 10 min to pellet cell debris debris and filtered using 45 μm PVDF filters (CellTreat). The supernatant was then ultracentrifuged for 2 hours at 100,000 g (Sorvall Lynx 6000) and the pellet resuspended overnight at 4°C in PBS with 1% BSA.Following lentiviral titration, 330 million A549ACE2 expressing cells were transduced at MOI of 0.5. Cells were selected with 2 μg/uL puromycin (Thermo) for 12 days to ensure proper selection. Throughout the experiment the representation was monitored such that each guide RNA is represented by at least 1000 cells (∼125 M cells).We infected GeCKOv2-transduced cells (∼1000x representation) with SARS-CoV-2 isolate USA-WA1/2020 (NR-52281) at either MOI of 0.01 (low MOI) and 0.3 (high MOI). Surviving cells were collected on day 6 post-infection for genomic DNA isolation.Genomic DNA isolation, guide RNA amplification and quantificationWe used a two-step PCR protocol (PCR1 and PCR2) to amplify the guide RNA cassette for Illumina sequencing from genomic DNA (gDNA). The gDNA was extracted from CRISPR screen cells using the following protocol (Chen et al., 2015): Per 100 million cells, 12 mL of NK Lysis Buffer (50 mM Tris, 50 mM EDTA, 1% SDS, pH 8) were used for cell lysis. Once cells were resuspended, 60 μL of 20 mg/ml Proteinase K (QIAGEN) was added and the sample was incubated at 55°C overnight. The next day, 60 μL of 20 μg/mL RNase A (QIAGEN) was added, mixed, and samples were incubated at 37°C for 30 min. Then 4 mL of pre-chilled 7.5 M ammonium acetate was added, samples were vortexed and spun at 4,000 g for 10 min. The supernatant was placed in a new tube, mixed well with 12 mL isopropanol and spun at 4,000 g for 10 min. DNA pellets were washed with 12 mL of 70% ethanol, spun, dried and pellets were resuspended with 0.2x TE buffer (Sigma).For the first PCR reaction, we used all gDNA available for each sample. We performed the PCR1 using Taq-B polymerase (Enzymatics) and used multiple reactions where each reaction contained up to 10 μg gDNA per 100 μL PCR reaction. PCR1 products for each sample were pooled and used for amplification with barcoded PCR2 primers. For each sample, we performed 12 PCR2 reactions (using 5 μL of the pooled PCR1 product per PCR2 reaction) with Q5 polymerase (NEB). PCR2 products were pooled and then normalized within each biological sample before combining uniquely-barcoded separate biological samples. The pooled product was then gel-purified from a 2% E-gel EX (Life Technologies) using the QiaQuick gel extraction kit (QIAGEN). The purified, pooled library was then quantified with Tapestation 4200 (Agilent Technologies). PCR products were run on a 2% agarose gel and the correct size band was extracted. Sequencing was performed on the NextSeq 550 instrument using the HighOutput Mode v2 with 75 bp paired-end reads (Illumina). For PCR1 the following primer set was used: 5′ GAGGGCCTATTTCCCATGATTC 3′ and 5′ GTTGCGAAAAAGAACGTTCACGG 3′. For PCR2 the following primers were used: 5′ AATGATACGGCGACCACCGAGATCTACACTCTTTCCCTACACGACGCTCTTCCGATCT (N1-9) (BC8) TCTTGTGGAAAGGACGAAACACCG 3′ and 5′ CAAGCAGAAGACGGCATACGAGAT (BC8) GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCT (N1-9) TCTACTATTCTTTCCCCTGCACTGT 3′, where N is a stagger of 1 to 9 nucleotides and BC is a barcode of 6 nucleotides.Sequencing reads were demultiplexed upon sequencing based on Illumina i7 barcodes present in PCR2 reverse primers using Illumina BaseSpace. We performed adaptor trimming by treating the hU6 promoter sequence as a 5′ adaptor, using cutadapt v1.13 [-e 0.2 -O 5 -m 20 -g TCTTGTGGAAAGGACGAAACACCG]. Processed guide RNA sequences were aligned to the GeCKOv2 reference allowing for up to 1 mismatch using bowtie v1.1.2 [-a–best–strata -v 1 –norc] with alignment rates of 81% to 86%.Computational analyses of genome-scale CRISPR screensGuide RNA counts were processed using the MaGeCK pipeline with an output of RRA p-values and gene ranks (Chen et al., 2018). We separately ranked genes using the RIGER (weighted-sum) and second-best rank methods (Chen et al., 2015; Luo et al., 2008). Gene Set Enrichment Analyses were performed using the fgsea package with Gene Ontology for biological processes (c5.bp.v7.1.symbols) (Korotkevich et al., 2019). GTEx v8 tissue specific enrichment was performed using the Multi Gene Query function available on the GTEx website: https://www.gtexportal.org/home/multiGeneQueryPage (accessed August 1st, 2020) (Aguet et al., 2019). GO enrichments for SARS-CoV-2, ZIKV, and IAV CRISPR screens were performed using GOrilla to find all significant enrichments (FDR p-value < 10-3) (Eden et al., 2009). FDR p-values were log10-transformed and normalized to create the heatmap shown in Figure 3E.Generation of gene-perturbed human cell lines and SARS-CoV-2 infectionFor validation by CRISPR knock-out (using 3 independent guide RNAs per gene), we selected genes that were highly-ranked (either in the low or high MOI screen or both) or genes that had not previously been implicated in viral pathogenesis or immune signaling. For each gene of interest, 3 guide RNAs were designed using GUIDES software and were subsequently cloned into an all-in-one vector with an optimized Cas9 scaffold (pCC_01, Addgene 139086) containing Cas9, a guide RNA cassette and a puro resistant cassette (Legut et al., 2020; Meier et al., 2017). Following sequence confirmation by Sanger sequencing, lentivirus was produced individually for each plasmid and the target cells were transduced in presence of polybrene (10 μg/mL, Santa Cruz). Gene-perturbed A549ACE2, Huh7.5ACE2, Calu-3 and Caco-2 cells were selected for at least 10 days with 2 μg/mL (A549ACE2, Calu-3), 3 μg/mL (Caco-2) or 5 μg/mL (Huh7.5ACE2) puromycin (Thermo). To determine the SARS-CoV-2 infection susceptibility of gene-perturbed lines, 10,000 cells were plated per well of 96-well plates. After 24 hours, the cells were infected with SARS-CoV-2 at MOI of 0.1. At 36 hours post-infection, the cells were either fixed and processed for immunofluorescence or cellular RNA was harvested for qRT-PCR analyses. For the multistep growth curves, cells were infected at an MOI of 0.1, and total RNA was harvested from infected cells at indicated times. RNA was processed for qRT-PCR as described above. All infections with SARS-CoV-2 were performed with at least 3 biological replicates.Immunofluorescence of nucleocapsid (N) proteinCells were fixed with 5% formaldehyde and immunostained for nucleocapsid (N) protein and visualized with a secondary antibody labeled with AlexaFlour-568 (Thermo). SARS-CoV-2 nucleocapsid (N) antibody (clone 1C7C7) was obtained from the Center for Therapeutic Antibody Discovery at the Icahn School of Medicine at Mount Sinai. Nuclei were stained with DAPI. Full wells were imaged and quantified for SARS-CoV-2 infected cells using a Celigo imaging cytometer (Nexcelom Biosciences). All infections with SARS-CoV-2 were performed with 3 biological replicates. Representative images from the top gene knockout hits were acquired using the EVOS M5000 Imaging System (Thermo).Identification of druggable genes and drug treatmentsTo identify druggable genes among the top CRISPR screen hits, we cross-referenced highly-ranked genes from the RRA analysis with the data table containing drugs and their gene targets was obtained from Drug Gene Interaction database (DGIdb, retrieved on June 3, 2020) as well as manual literature search (Cotto et al., 2018). Inhibitors that target the genes of interest (and remdesivir) was obtained from SelleckChem and MedChemExpress. The catalog number and vendor information is available in the Key Resource section. Amlodipine was obtained from Sigma (A5605).To test drug efficacy in reducing SARS-CoV-2 infection, 10,000 A549ACE2 cells were seeded per well of a 96-well plate. Cells were treated with inhibitors at 10 μM for two hours before infection and inhibitors were maintained throughout the course of infection. Cells were infected with SARS-CoV-2 at MOI of 0.1 for 36 hours and the cells were collected for analysis via qRT-PCR or processed for immunofluorescence (N protein and quantified by Celigo). For Figure 4E, the remdesivir data was collected in an independent experiment.Cell viability assaysCell viability following drug treatments was performed the same way as described above. Thirty-six hours post drug treatment the cells were collected and stained with LIVE-DEAD Violet (Thermo). Cell acquisition was performed using a Sony SH800S cell sorter with a 100 μm sorting chip. We used the following gating strategy: 1) We excluded the cell debris based on the forward and reverse scatter; 2) Doublets were excluded by plotting FSC height versus FCS area, 3) Dead cells were quantified using live-dead violet stain. For all samples, we recorded at least 5,000 cells that pass the gating criteria described above. Flow cytometry analyses were performed using FlowJo v10. Cell viability for A549-ACE2 treated with DMSO or 10μM amlodipine was performed 24 hours post treatment using Trypan Blue (Thermo Fisher) and automated cell counting (Nexcelom AutoT4).Quantitative reverse-transcription PCR (qRT-PCR) of viral RNARNA was extracted from cells grown in 96-well plates by using the RNeasy 96 Kit (QIAGEN) per the manufacturer’s instructions. RNA was reverse-transcribed and PCR amplified using SYBR Fast One-step Universal qRT-PCR Kit (KAPA/Roche). For amlodipine experiments, RNA was extracted using TRIzol Reagent (Invitrogen) and purified with the Direct-zol RNA Miniprep kit (Zymo Research) as per the manufacturer’s instructions (including the optional DNase I treatment). For amlodipine experiments, the RNA was reverse-transcribed using SuperScript Reverse Transcriptase II (Thermo Fisher) and oligo(dT) primers. The cDNA was diluted 1:20 before qRT-PCR was performed using SYBR Fast qPCR (KAPA/Roche). SARS-CoV-2 replication was assessed by using primers specific to the N mRNA (Forward 5′-CTCTTGTAGATCTGTTCTCTAAACGAAC-3′; Reverse 5′-GGTCCACCAAACGTAATGCG-3′). SARS-CoV-2 N mRNA levels were normalized to beta tubulin (Forward 5′-GCCTGGACCACAAGTTTGAC-3; Reverse 5′-TGAAATTCTGGGAGCATGAC-3′). Reactions were ran and analyzed on a Lightcycler 480 II Instrument (Roche). Relative quantification was calculated by comparing the cycle threshold (C
t) values using ΔΔCt. C
t values of subgenomic nucleocapsid transcript were normalized to C
t values of alpha-tubulin, and those values were normalized to viral transcript levels of DMSO-treated or non-targeting guide RNA-transduced control samples, and displayed as fold-reduction in viral transcript. Significance was determined using a one-way ANOVA with post hoc tests or a two-tailed unpaired Student’s t test, as appropriate.siRNA transfections and SARS-CoV-2 infectionAll of the siRNAs were ordered from Dharmacon and their catalog number can be found in the Key Resources Table. To knockdown individual genes, 10,000 A549ACE2 cells were seeded in 96-well plates and transfected with siRNAs using Lipofectamine RNAiMAX (Thermo) following the manufacturer’s protocol. Forty-eight hours later, the cells were infected with SARS-CoV-2 at an MOI of 0.1 for 36 hours. Cells were fixed with 5% formaldehyde, stained with nucleocapsid protein (clone 1C7C7, ISMMS), and visualized with AlexaFluor-568 conjugated secondary antibody (Thermo). Nuclei were stained with DAPI, and full wells were imaged with a Celigo imaging cytometer (Nexcelom Biosciences).Minipool CRISPR library and lentiviral productionTo generate the minipool, we combined equimolar amounts the same guide RNA vectors (cloned in pCC1) that we used for arrayed validation (see above Generation of gene-perturbed A549ACE2
lines and SARS-CoV-2 infection). In total, we combined 3 guide RNAs for each gene (either, 30 or 11 genes) plus 9 non-targeting guide RNAs. The non-targeting guide RNA plasmids were added at half the molar ratio of the other plasmids. Lentivirus was produced via transfection of the minipool with appropriate packaging plasmids (psPAX2: Addgene 12260; pMD2.G: Addgene 12259) using polyethylenimine (PEI) reagent in HEK293FT. The target A549ACE2 cells were transduced in presence of polybrene (10 μg/mL, Santa Cruz) (ECCITE-Seq experiment 1) or without (ECCITE-seq experiment 2) at very low MOI to ensure entry of a single virus per cell. The survival was below 5% after 2 days of selection with 2 μg/mL puromycin (Thermo) for both ECCITE-seq experiments. On day 10 post-transduction, the cells were collected and processed for ECCITE-seq.Expanded CRISPR-compatible Cellular Indexing of Transcriptomes and Epitopes (ECCITE-seq)To perform cell-hashing and ECCITE-seq (Mimitou et al., 2019; Stoeckius et al., 2018), we separated cells into several sub-pools and, for each pool, we resuspended 1 million cells in 100 μl staining buffer (2% BSA, 0.01% Tween in PBS). We then added 10 μl Human TruStain FcX Fc Receptor Blocking Solution (BioLegend) and incubated on ice for 10 minutes. We then added hashing antibodies (BioLegend), incubated on ice for 30 minutes, and washed cells 3 times with staining buffer. In total, we ran four lanes of a Chromium Single Cell Immune Profiling Solution v1.0 5′ kit (10x Genomics) targeting recovery of 12,000 cells per lane (superloading) in experiment 1 and 20,000 cells in experiment 2. Gene expression (mRNA), hashtags (Hashtag-derived oligos, HTOs) and guide RNA (Guide-derived oligos, GDOs) libraries were constructed (Smibert et al., 2019). Each replicate was sequenced on two NextSeq550 75-cycle high-output runs (Illumina). Sequencing reads from the mRNA library were mapped to the hg38 reference genome (Ensembl v97) using Cellranger (v3.0.1, 10x Genomics). To generate count matrices for HTO and GDO libraries, the CITE-seq-count package was used (https://github.com/Hoohm/CITE-seq-Count v1.4.2). Count matrices were then used as input into the Seurat R package (v3.2) for downstream analyses (Stuart et al., 2019).ECCITE-seq data analysisProcessing of initial ECCITE-seq experiment with 30 target genesCells with low quality metrics, high mitochondrial gene content (> 17.5%) and low number of genes detected (≤1800) were removed. The median number of detected genes was 3309. RNA counts were log-normalized and HTO counts were normalized using the centered log-ratio transformation approach, with margin = 2 (normalizing across cells). To identify cell doublets and assign experimental conditions to cells, we used the HTOseqDemux function (Stoeckius et al., 2018). HTOseqDemux-defined cell doublets and negatives were removed from any downstream analyses.Cellular guide RNA identity for cells in the ECCITE-seq pool was assigned based on GDO unique molecular identifier (UMI) counts. We considered a guide RNA detected with ≥ 16 UMI counts (the median UMI counts per guide RNA was 503 counts, 93% of cells with ≥ 1 sgRNA). We observed that 64% of the cells had > 1 detected guide RNA (29% 1 guide RNA, 27% 2 guide RNAs, 17% 3 guide RNAs). To maximize cell recovery, we retained cells with 1 - 3 guide RNAs per cell (n = 10,265). We collapsed the guide RNA information down to target gene level (3 guide RNA per target gene) in the following way: For all cells (with 1-3 guide RNAs), we assigned a unique target gene if all detected guide RNAs targeted the same gene, or if the additional guide RNA detected was a non-targeting (NT) guide RNA (4,715 cells: 4,013 cells with exactly 1 sgRNA and 702 cells with collapsed target gene assignment). All remaining cells assigned 2 or 3 separate target genes were discarded.Some of the top-ranked genes from the CRISPR screens may be endpoint genes and thus may have subtle phenotypes difficult to detect using single-cell transcriptomics. To address this issue, we used the initial ECCITE-seq experiment (n = 4,715 cells) and grouped cells by target gene. To identify target gene perturbations that lead to systemic transcriptomic changes, we determined the number of differentially expressed genes relative to cells with NT guide RNAs. We found that under stringent filtering conditions (non-adjusted p-value < 10−5 or adjusted p-value < 0.2), only 11 target genes showed more than 5 differentially expressed genes. The second ECCITE-seq experiment was conducted using guide RNAs solely for these.Joint processing of both ECCITE-seq experimentsCells with low quality metrics, high mitochondrial gene content (> 15%) and low or high number of genes detected (≤2000 and > 5000) were removed. The median number of detected genes was 3365, with a median of 13,532 UMIs per cell. HTO counts were normalized using the centered log-ratio transformation approach, with margin = 2 (normalizing across cells). To identify cell doublets and assign experimental conditions to cells, we used the HTOseqDemux function (Stoeckius et al., 2018). HTOseqDemux-defined cell doublets and negatives were removed from any downstream analyses. Guide RNA counts were normalized similar to HTO UMI counts. Cell-wise guide RNA assignment was achieved using the MultiSeqDemux function (McGinnis et al., 2019). We only retained cells assigned with a single HTO and a single guide RNA (ECCITE-seq experiment 1: n = 1,824 cells, ECCITE-seq experiment 2: n = 17,029 cells). Cells from ECCITE-Seq experiment 1 and experiment 2 were harmonized by log-normalizing, followed by the identification of the 2000 most variable features with “vst” selection method. Next, we we used FindIntegrationAnchors and IntegrateData with the first 40 principle components.In the harmonized dataset (n = 18,853 cells), we used FindMarkers to find differentially expressed genes between non-targeting cells and cells that belonged to a targeted gene class. We used up to 100 of the most upregulated genes with adjusted p-value < 0.01 and minimal logeFC threshold > 0.1 filter as input into the heatmap in Figure 5B. The same FindMarkers analysis (but using the non-adjusted p-value < 0.01 with a maximum of the 300 most significant DEGs) was used as input into the EnrichR package to run pathway analysis using the human WikiPathways database (v. 2019) (Kuleshov et al., 2016; Slenter et al., 2018). Figure 5C shows -log10 transformed p-values for the union of all enriched pathways with an adjusted p-value < 0.01 across the 11 target genes with perturbation signatures.Plaque assayInfectious titer in supernatant of SARS-CoV-2 infections (from A549ACE2) was determined by plaque assay in Vero E6 cells. Cells were seeded in 12-well plates before infection with logarithmically diluted supernatants in a total inoculum of 100 ul, rocking plates every 10 minutes for 1 hour. Then an overlay of Minimum Essential Media with 0.12% NaHCO3, 4 mM L-glutamine, 0.2% BSA, 10 mM HEPES and 0.7% OXOID agar (Thermo) was aliquoted into wells and plates were incubated for 48 hours before fixing with 5% formaldehyde. After 24 hours, plaques were stained with crystal violet (1% crystal violet in 20% EtOH) for 15 minutes before washing wells and counting plaques.Bulk RNA-sequencingRNA-sequencing libraries were constructed from 1ug of RNA per sample using the TruSeq Stranded mRNA Library Prep Kit (Illumina) according to the manufacturer’s instructions. Sequencing libraries were sequenced using an Illumina NextSeq 500 platform and fastq files were generated using bcl2fastq (Illumina). Alignment to the SARS-CoV-2 genome (GenBank: MN985325.1) were performed using bowtie (Langmead et al., 2009). Reads were aligned to hg19 using STAR aligner in the Basespace RNA-Seq Alignment application (Illumina) and processed using DESeq2 (Love et al., 2014). Sample distances were calculated using read counts from the regularized log transform in DESeq2. K-means clustering was perfomed using the kmeans function in R/RStudio. Enriched Reactome pathways were identified using the STRING tool (Szklarczyk et al., 2019).Measurement of cellular cholesterolCellular cholesterol was measured using the Cholesterol-Glo assay (Promega). For each cell line (CRISPR-perturbed or drug-treated), we counted 100,000 cells, washed with 1xPBS twice and resuspended in 100 μL cell lysis solution for 30 min at 37°C. Following the incubation, 10 μL of cell lysate was used to measure the cholesterol levels in 96 well plates in a final reaction volume of 100uL per well (without esterase). Luminescence was measured after incubating the plate for 1 hour at room temperature in the dark. Using the same lysis sample, the protein concentration was determined using the BCA assay (Thermo). The cholesterol levels were normalized to the respective protein concentration of each sample. A549ACE cells were incubated with 10 μM amlodipine (Sigma) or DMSO for 24 hours. Cholesterol levels were then determined as described above.Flow cytometry of ACE2 cell surface expressionCells were harvested, counted and about 100,000 cells were washed with Dulbecco’s phosphate-buffered saline (PBS, Caisson Labs) and then stained with LIVE/DEAD Violet stain (Thermo, 34864). Following a wash with 1x PBS, all subsequent washes and antibody dilutions were performed using 1x PBS supplemented with 2% FBS. Wild-type or Cas9-perturbed A549ACE2, Caco-2 and Calu-3 cells were stained for 30 min on ice with 0.25 μg of anti-ACE2 antibody (R&D Systems, AF933) in ∼50 μL residual volume. Following two washes with 1x PBS with 2% FBS, samples were stained on ice for 20 min with 7.5 μL of anti-goat-APC secondary antibody (R&D Systems, F0108). Cell acquisition and sorting was performed using a Sony SH800S cell sorter with a 100 μm sorting chip. We used the following gating strategy: 1) We excluded the cell debris based on the forward and reverse scatter; 2) Doublets were excluded by plotting FSC height versus FCS area, 3) Dead cells were excluded by live-dead stain. We recorded at least 5,000 cells for A549ACE2 and at least 3,000 cells for Calu-3 and Caco-2 that pass the gating criteria described above. Gates to determine ACE2-APC+ cells were set based on control A549 wild-type cells or only secondary antibody stained Calu-3 and Caco-2, where the percent of ACE2 positive cells was set as < 5% (background level). Flow cytometry analyses were performed using FlowJo v10.Immunofluorescence of ACE2 and endo/lysosomal markersA549ACE2 Cas9-transduced (specific gene or non-targeting) cell lines were seeded on poly-D-lysine-coated coverslips (Electron Microscopy Sciences) 24 hours before they were fixed with 4% formaldehyde (Sigma) diluted in 1x PBS (Caisson Labs) (Daniloski et al., 2020). Following 3 washes with 1x PBS, cells were blocked with 1x PBS with 2% BSA for 30 min at room temperature. Cells were stained with mouse anti-2A antibody diluted at 1:250 (clone 3H4, Millipore Sigma, MABS2005) to recognize ACE2-2A, and rabbit anti-EEA1 diluted at 1:100 (Thermo, MA5-14794). To detect ACE2 in Caco-2 cells, goat anti-ACE2 (R&D Systems, AF933) was used at 1.25 μg/mL. Cell were incubated with primary antibodies for 3 hours at room temperature in a moisturized chamber. Coverslips were then washed 5 times with 1x PBS with 2% BSA and then incubated with Alexa-conjugated secondary antibodies (Jackson ImmunoResearch) diluted at 1:1000 for 45 min at room temperature. Coverslips were then washed 5 times with 1x PBS with 2%BSA, and DAPI (Sigma) was added in the fourth wash at a concentration of 0.1 μg/mL. To stain the lysosomes, LysoTracker DeepRed (Thermo) was diluted to 70 nM in culture media and was added to actively growing cells on coverslips and were incubated for 1 hour at 37°C. Following incubation with LysoTracker, cells were processed for immunofluorescence as described above. Confocal images were acquired on a Zeiss LSM 780 with a 20x/0.8 Plan-Apochromat objective or a 63x/1.4 Plan-Apochromat objective (Zeiss). Epifluorescence images were acquired using the same system; both fitted with a Zeiss Axiocam 506 mono (Zeiss). Images were processed using Zen Black 2012 (Zeiss) and FIJI 2.1.0; Java 1.8.0-202 (Schindelin et al., 2012). Linear histogram adjustments were applied uniformly within experiments for clarity of presentation.ACE2 localization was determined on images taken on a Zeiss LSM 780 with a 20x/0.8 Plan-Apochromat objective using Zen Black (Zeiss). The A549ACE2 polyclonal cell line had both ACE2 positive and negative cells. Only ACE2 positive cells were then manually scored if they had a distinct accumulation of cytoplasmic vesicles-like hollow structures, compared to the non-targeting (NT) control. In the NT control, the ACE2 staining was seen on the plasma membrane and diffuse in the cytoplasm. The scoring was performed on 2 biological replicates (that is, two separate non-targeting controls and two separate Rab7A knockout lines engineered using 4 separate Cas9 guide RNAs). We scored ∼430 cells per replicate. To quantify the size of ACE2 foci, confocal microscopy images were thresholded using FIJI’s local-threshold (Phansalkar radius = 15) and then a watershed transform was applied. Foci were counted and measured using FIJI’s Analyze Particles function.Western blotsA549ACE2 or Huh7.5ACE2 cells were collected, washed with 1x PBS and lysed with TNE buffer (10 mM Tris-HCl, pH 7.4, 150 mM NaCl, 1mM EDTA, 1% Nonidet P-40) in presence of protease inhibitor cocktail (Bimake B14001) for 1 hour on ice. Cells lysates were spun for 10 min at 10,000 g, and protein concentration was determined with the BCA assay (Thermo). Equal amounts of cell lysates (20 μg) were denatured in Tris-Glycine SDS Sample buffer (Thermo LC2676), and loaded on a Novex 4 - 20% Tris-Glycine gel (Thermo XP04205BOX). PageRuler pre-stained protein ladder (Thermo 26616) was used to determine the protein size. The gel was run in 1x Tris-Glycine-SDS buffer (IBI Scientific IBI01160) for about 120 min at 120V. Proteins were transferred on a nitrocellulose membrane (BioRad 1620112) in presence of prechilled 1x Tris-Glycine transfer buffer (FisherSci LC3675) supplemented with 20% methanol for 100 min at 100V. Immunoblots were blocked with 5% skim milk dissolved in 1x PBS with 1% Tween-20 (PBST) and incubated overnight at 4°C separately with the following primary antibodies: rabbit anti-RAB7A (0.1 μg/mL, NovusBio, NBP1-87174), rabbit anti-GAPDH (0.1 μg/mL, Cell Signaling, 2118S), rabbit anti-CCDC22 (0.34 μg/mL, Proteintech, 16636-1-AP), rabbit anti-ATP6V1A (0.46 μg/mL, Proteintech, 17115-1-AP), rabbit anti-ACE2 (0.5 μg/mL, Invitrogen, MA5-32307), and mouse anti-beta tubulin (0.5 μg/mL, Invitrogen, 32-2600). Following the primary antibody, the blots were incubated with IRDye 680RD donkey anti-rabbit (0.2 μg/mL, LI-COR 926-68073) or with IRDye 800CW donkey anti-mouse (0.2 μg/mL, LI-COR 926-32212). The blots were imaged using Odyssey CLx (LI-COR).

Method Details

Generation of A549ACE2 lineTo generate ACE2 expressing cells, the human ACE2 coding sequence was amplified and cloned into the BamHI site of the lentiviral vector pHR-PGK (Addgene 79120). We generated lentiviral particles (as described below) and transduced 5x104 A549 cells plated in a 12-well plate in the presence of polybrene (8 μg/ml). We confirmed hACE2 expression by western blot analysis (Thermo MA5-32307).

Generation of A549ACE2 line

To generate ACE2 expressing cells, the human ACE2 coding sequence was amplified and cloned into the BamHI site of the lentiviral vector pHR-PGK (Addgene 79120). We generated lentiviral particles (as described below) and transduced 5x104 A549 cells plated in a 12-well plate in the presence of polybrene (8 μg/ml). We confirmed hACE2 expression by western blot analysis (Thermo MA5-32307).

Lentiviral production, transduction and SARS-CoV-2 infectionThe Human GeCKOv2 A and B libraries (Addgene 1000000048) were used for genome-scale CRISPR knockout screens (Sanjana et al., 2014). We mixed equal amount of the A and B library plasmids to target each gene with 6 guide RNAs. Briefly, 225cm2 flasks of 80% confluent HEK293FT cells (Thermo) were transfected with 25 μg GeCKOv2 plasmid, 14 μg pMD2.G and 20 μg psPAX2 mixed in 2.5 mL OptiMEM (Thermo) and 175 μL Polyethylenimine (1 mg/ml) (Polysciences). After 6 hours, media was changed to D10 media with 1% bovine serum albumin (Sigma) added to improve virus stability. After 60 hours, viral supernatants were harvested and centrifuged at 3,000 rpm at 4°C for 10 min to pellet cell debris debris and filtered using 45 μm PVDF filters (CellTreat). The supernatant was then ultracentrifuged for 2 hours at 100,000 g (Sorvall Lynx 6000) and the pellet resuspended overnight at 4°C in PBS with 1% BSA.Following lentiviral titration, 330 million A549ACE2 expressing cells were transduced at MOI of 0.5. Cells were selected with 2 μg/uL puromycin (Thermo) for 12 days to ensure proper selection. Throughout the experiment the representation was monitored such that each guide RNA is represented by at least 1000 cells (∼125 M cells).We infected GeCKOv2-transduced cells (∼1000x representation) with SARS-CoV-2 isolate USA-WA1/2020 (NR-52281) at either MOI of 0.01 (low MOI) and 0.3 (high MOI). Surviving cells were collected on day 6 post-infection for genomic DNA isolation.

Lentiviral production, transduction and SARS-CoV-2 infection

The Human GeCKOv2 A and B libraries (Addgene 1000000048) were used for genome-scale CRISPR knockout screens (Sanjana et al., 2014). We mixed equal amount of the A and B library plasmids to target each gene with 6 guide RNAs. Briefly, 225cm2 flasks of 80% confluent HEK293FT cells (Thermo) were transfected with 25 μg GeCKOv2 plasmid, 14 μg pMD2.G and 20 μg psPAX2 mixed in 2.5 mL OptiMEM (Thermo) and 175 μL Polyethylenimine (1 mg/ml) (Polysciences). After 6 hours, media was changed to D10 media with 1% bovine serum albumin (Sigma) added to improve virus stability. After 60 hours, viral supernatants were harvested and centrifuged at 3,000 rpm at 4°C for 10 min to pellet cell debris debris and filtered using 45 μm PVDF filters (CellTreat). The supernatant was then ultracentrifuged for 2 hours at 100,000 g (Sorvall Lynx 6000) and the pellet resuspended overnight at 4°C in PBS with 1% BSA.

Following lentiviral titration, 330 million A549ACE2 expressing cells were transduced at MOI of 0.5. Cells were selected with 2 μg/uL puromycin (Thermo) for 12 days to ensure proper selection. Throughout the experiment the representation was monitored such that each guide RNA is represented by at least 1000 cells (∼125 M cells).

We infected GeCKOv2-transduced cells (∼1000x representation) with SARS-CoV-2 isolate USA-WA1/2020 (NR-52281) at either MOI of 0.01 (low MOI) and 0.3 (high MOI). Surviving cells were collected on day 6 post-infection for genomic DNA isolation.

Genomic DNA isolation, guide RNA amplification and quantificationWe used a two-step PCR protocol (PCR1 and PCR2) to amplify the guide RNA cassette for Illumina sequencing from genomic DNA (gDNA). The gDNA was extracted from CRISPR screen cells using the following protocol (Chen et al., 2015): Per 100 million cells, 12 mL of NK Lysis Buffer (50 mM Tris, 50 mM EDTA, 1% SDS, pH 8) were used for cell lysis. Once cells were resuspended, 60 μL of 20 mg/ml Proteinase K (QIAGEN) was added and the sample was incubated at 55°C overnight. The next day, 60 μL of 20 μg/mL RNase A (QIAGEN) was added, mixed, and samples were incubated at 37°C for 30 min. Then 4 mL of pre-chilled 7.5 M ammonium acetate was added, samples were vortexed and spun at 4,000 g for 10 min. The supernatant was placed in a new tube, mixed well with 12 mL isopropanol and spun at 4,000 g for 10 min. DNA pellets were washed with 12 mL of 70% ethanol, spun, dried and pellets were resuspended with 0.2x TE buffer (Sigma).For the first PCR reaction, we used all gDNA available for each sample. We performed the PCR1 using Taq-B polymerase (Enzymatics) and used multiple reactions where each reaction contained up to 10 μg gDNA per 100 μL PCR reaction. PCR1 products for each sample were pooled and used for amplification with barcoded PCR2 primers. For each sample, we performed 12 PCR2 reactions (using 5 μL of the pooled PCR1 product per PCR2 reaction) with Q5 polymerase (NEB). PCR2 products were pooled and then normalized within each biological sample before combining uniquely-barcoded separate biological samples. The pooled product was then gel-purified from a 2% E-gel EX (Life Technologies) using the QiaQuick gel extraction kit (QIAGEN). The purified, pooled library was then quantified with Tapestation 4200 (Agilent Technologies). PCR products were run on a 2% agarose gel and the correct size band was extracted. Sequencing was performed on the NextSeq 550 instrument using the HighOutput Mode v2 with 75 bp paired-end reads (Illumina). For PCR1 the following primer set was used: 5′ GAGGGCCTATTTCCCATGATTC 3′ and 5′ GTTGCGAAAAAGAACGTTCACGG 3′. For PCR2 the following primers were used: 5′ AATGATACGGCGACCACCGAGATCTACACTCTTTCCCTACACGACGCTCTTCCGATCT (N1-9) (BC8) TCTTGTGGAAAGGACGAAACACCG 3′ and 5′ CAAGCAGAAGACGGCATACGAGAT (BC8) GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCT (N1-9) TCTACTATTCTTTCCCCTGCACTGT 3′, where N is a stagger of 1 to 9 nucleotides and BC is a barcode of 6 nucleotides.Sequencing reads were demultiplexed upon sequencing based on Illumina i7 barcodes present in PCR2 reverse primers using Illumina BaseSpace. We performed adaptor trimming by treating the hU6 promoter sequence as a 5′ adaptor, using cutadapt v1.13 [-e 0.2 -O 5 -m 20 -g TCTTGTGGAAAGGACGAAACACCG]. Processed guide RNA sequences were aligned to the GeCKOv2 reference allowing for up to 1 mismatch using bowtie v1.1.2 [-a–best–strata -v 1 –norc] with alignment rates of 81% to 86%.

Genomic DNA isolation, guide RNA amplification and quantification

We used a two-step PCR protocol (PCR1 and PCR2) to amplify the guide RNA cassette for Illumina sequencing from genomic DNA (gDNA). The gDNA was extracted from CRISPR screen cells using the following protocol (Chen et al., 2015): Per 100 million cells, 12 mL of NK Lysis Buffer (50 mM Tris, 50 mM EDTA, 1% SDS, pH 8) were used for cell lysis. Once cells were resuspended, 60 μL of 20 mg/ml Proteinase K (QIAGEN) was added and the sample was incubated at 55°C overnight. The next day, 60 μL of 20 μg/mL RNase A (QIAGEN) was added, mixed, and samples were incubated at 37°C for 30 min. Then 4 mL of pre-chilled 7.5 M ammonium acetate was added, samples were vortexed and spun at 4,000 g for 10 min. The supernatant was placed in a new tube, mixed well with 12 mL isopropanol and spun at 4,000 g for 10 min. DNA pellets were washed with 12 mL of 70% ethanol, spun, dried and pellets were resuspended with 0.2x TE buffer (Sigma).

For the first PCR reaction, we used all gDNA available for each sample. We performed the PCR1 using Taq-B polymerase (Enzymatics) and used multiple reactions where each reaction contained up to 10 μg gDNA per 100 μL PCR reaction. PCR1 products for each sample were pooled and used for amplification with barcoded PCR2 primers. For each sample, we performed 12 PCR2 reactions (using 5 μL of the pooled PCR1 product per PCR2 reaction) with Q5 polymerase (NEB). PCR2 products were pooled and then normalized within each biological sample before combining uniquely-barcoded separate biological samples. The pooled product was then gel-purified from a 2% E-gel EX (Life Technologies) using the QiaQuick gel extraction kit (QIAGEN). The purified, pooled library was then quantified with Tapestation 4200 (Agilent Technologies). PCR products were run on a 2% agarose gel and the correct size band was extracted. Sequencing was performed on the NextSeq 550 instrument using the HighOutput Mode v2 with 75 bp paired-end reads (Illumina). For PCR1 the following primer set was used: 5′ GAGGGCCTATTTCCCATGATTC 3′ and 5′ GTTGCGAAAAAGAACGTTCACGG 3′. For PCR2 the following primers were used: 5′ AATGATACGGCGACCACCGAGATCTACACTCTTTCCCTACACGACGCTCTTCCGATCT (N1-9) (BC8) TCTTGTGGAAAGGACGAAACACCG 3′ and 5′ CAAGCAGAAGACGGCATACGAGAT (BC8) GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCT (N1-9) TCTACTATTCTTTCCCCTGCACTGT 3′, where N is a stagger of 1 to 9 nucleotides and BC is a barcode of 6 nucleotides.

Sequencing reads were demultiplexed upon sequencing based on Illumina i7 barcodes present in PCR2 reverse primers using Illumina BaseSpace. We performed adaptor trimming by treating the hU6 promoter sequence as a 5′ adaptor, using cutadapt v1.13 [-e 0.2 -O 5 -m 20 -g TCTTGTGGAAAGGACGAAACACCG]. Processed guide RNA sequences were aligned to the GeCKOv2 reference allowing for up to 1 mismatch using bowtie v1.1.2 [-a–best–strata -v 1 –norc] with alignment rates of 81% to 86%.

Computational analyses of genome-scale CRISPR screensGuide RNA counts were processed using the MaGeCK pipeline with an output of RRA p-values and gene ranks (Chen et al., 2018). We separately ranked genes using the RIGER (weighted-sum) and second-best rank methods (Chen et al., 2015; Luo et al., 2008). Gene Set Enrichment Analyses were performed using the fgsea package with Gene Ontology for biological processes (c5.bp.v7.1.symbols) (Korotkevich et al., 2019). GTEx v8 tissue specific enrichment was performed using the Multi Gene Query function available on the GTEx website: https://www.gtexportal.org/home/multiGeneQueryPage (accessed August 1st, 2020) (Aguet et al., 2019). GO enrichments for SARS-CoV-2, ZIKV, and IAV CRISPR screens were performed using GOrilla to find all significant enrichments (FDR p-value < 10-3) (Eden et al., 2009). FDR p-values were log10-transformed and normalized to create the heatmap shown in Figure 3E.

Computational analyses of genome-scale CRISPR screens

Guide RNA counts were processed using the MaGeCK pipeline with an output of RRA p-values and gene ranks (Chen et al., 2018). We separately ranked genes using the RIGER (weighted-sum) and second-best rank methods (Chen et al., 2015; Luo et al., 2008). Gene Set Enrichment Analyses were performed using the fgsea package with Gene Ontology for biological processes (c5.bp.v7.1.symbols) (Korotkevich et al., 2019). GTEx v8 tissue specific enrichment was performed using the Multi Gene Query function available on the GTEx website: https://www.gtexportal.org/home/multiGeneQueryPage (accessed August 1st, 2020) (Aguet et al., 2019). GO enrichments for SARS-CoV-2, ZIKV, and IAV CRISPR screens were performed using GOrilla to find all significant enrichments (FDR p-value < 10-3) (Eden et al., 2009). FDR p-values were log10-transformed and normalized to create the heatmap shown in Figure 3E.

Generation of gene-perturbed human cell lines and SARS-CoV-2 infectionFor validation by CRISPR knock-out (using 3 independent guide RNAs per gene), we selected genes that were highly-ranked (either in the low or high MOI screen or both) or genes that had not previously been implicated in viral pathogenesis or immune signaling. For each gene of interest, 3 guide RNAs were designed using GUIDES software and were subsequently cloned into an all-in-one vector with an optimized Cas9 scaffold (pCC_01, Addgene 139086) containing Cas9, a guide RNA cassette and a puro resistant cassette (Legut et al., 2020; Meier et al., 2017). Following sequence confirmation by Sanger sequencing, lentivirus was produced individually for each plasmid and the target cells were transduced in presence of polybrene (10 μg/mL, Santa Cruz). Gene-perturbed A549ACE2, Huh7.5ACE2, Calu-3 and Caco-2 cells were selected for at least 10 days with 2 μg/mL (A549ACE2, Calu-3), 3 μg/mL (Caco-2) or 5 μg/mL (Huh7.5ACE2) puromycin (Thermo). To determine the SARS-CoV-2 infection susceptibility of gene-perturbed lines, 10,000 cells were plated per well of 96-well plates. After 24 hours, the cells were infected with SARS-CoV-2 at MOI of 0.1. At 36 hours post-infection, the cells were either fixed and processed for immunofluorescence or cellular RNA was harvested for qRT-PCR analyses. For the multistep growth curves, cells were infected at an MOI of 0.1, and total RNA was harvested from infected cells at indicated times. RNA was processed for qRT-PCR as described above. All infections with SARS-CoV-2 were performed with at least 3 biological replicates.

Generation of gene-perturbed human cell lines and SARS-CoV-2 infection

For validation by CRISPR knock-out (using 3 independent guide RNAs per gene), we selected genes that were highly-ranked (either in the low or high MOI screen or both) or genes that had not previously been implicated in viral pathogenesis or immune signaling. For each gene of interest, 3 guide RNAs were designed using GUIDES software and were subsequently cloned into an all-in-one vector with an optimized Cas9 scaffold (pCC_01, Addgene 139086) containing Cas9, a guide RNA cassette and a puro resistant cassette (Legut et al., 2020; Meier et al., 2017). Following sequence confirmation by Sanger sequencing, lentivirus was produced individually for each plasmid and the target cells were transduced in presence of polybrene (10 μg/mL, Santa Cruz). Gene-perturbed A549ACE2, Huh7.5ACE2, Calu-3 and Caco-2 cells were selected for at least 10 days with 2 μg/mL (A549ACE2, Calu-3), 3 μg/mL (Caco-2) or 5 μg/mL (Huh7.5ACE2) puromycin (Thermo). To determine the SARS-CoV-2 infection susceptibility of gene-perturbed lines, 10,000 cells were plated per well of 96-well plates. After 24 hours, the cells were infected with SARS-CoV-2 at MOI of 0.1. At 36 hours post-infection, the cells were either fixed and processed for immunofluorescence or cellular RNA was harvested for qRT-PCR analyses. For the multistep growth curves, cells were infected at an MOI of 0.1, and total RNA was harvested from infected cells at indicated times. RNA was processed for qRT-PCR as described above. All infections with SARS-CoV-2 were performed with at least 3 biological replicates.

Immunofluorescence of nucleocapsid (N) proteinCells were fixed with 5% formaldehyde and immunostained for nucleocapsid (N) protein and visualized with a secondary antibody labeled with AlexaFlour-568 (Thermo). SARS-CoV-2 nucleocapsid (N) antibody (clone 1C7C7) was obtained from the Center for Therapeutic Antibody Discovery at the Icahn School of Medicine at Mount Sinai. Nuclei were stained with DAPI. Full wells were imaged and quantified for SARS-CoV-2 infected cells using a Celigo imaging cytometer (Nexcelom Biosciences). All infections with SARS-CoV-2 were performed with 3 biological replicates. Representative images from the top gene knockout hits were acquired using the EVOS M5000 Imaging System (Thermo).

Immunofluorescence of nucleocapsid (N) protein

Cells were fixed with 5% formaldehyde and immunostained for nucleocapsid (N) protein and visualized with a secondary antibody labeled with AlexaFlour-568 (Thermo). SARS-CoV-2 nucleocapsid (N) antibody (clone 1C7C7) was obtained from the Center for Therapeutic Antibody Discovery at the Icahn School of Medicine at Mount Sinai. Nuclei were stained with DAPI. Full wells were imaged and quantified for SARS-CoV-2 infected cells using a Celigo imaging cytometer (Nexcelom Biosciences). All infections with SARS-CoV-2 were performed with 3 biological replicates. Representative images from the top gene knockout hits were acquired using the EVOS M5000 Imaging System (Thermo).

Identification of druggable genes and drug treatmentsTo identify druggable genes among the top CRISPR screen hits, we cross-referenced highly-ranked genes from the RRA analysis with the data table containing drugs and their gene targets was obtained from Drug Gene Interaction database (DGIdb, retrieved on June 3, 2020) as well as manual literature search (Cotto et al., 2018). Inhibitors that target the genes of interest (and remdesivir) was obtained from SelleckChem and MedChemExpress. The catalog number and vendor information is available in the Key Resource section. Amlodipine was obtained from Sigma (A5605).To test drug efficacy in reducing SARS-CoV-2 infection, 10,000 A549ACE2 cells were seeded per well of a 96-well plate. Cells were treated with inhibitors at 10 μM for two hours before infection and inhibitors were maintained throughout the course of infection. Cells were infected with SARS-CoV-2 at MOI of 0.1 for 36 hours and the cells were collected for analysis via qRT-PCR or processed for immunofluorescence (N protein and quantified by Celigo). For Figure 4E, the remdesivir data was collected in an independent experiment.

Identification of druggable genes and drug treatments

To identify druggable genes among the top CRISPR screen hits, we cross-referenced highly-ranked genes from the RRA analysis with the data table containing drugs and their gene targets was obtained from Drug Gene Interaction database (DGIdb, retrieved on June 3, 2020) as well as manual literature search (Cotto et al., 2018). Inhibitors that target the genes of interest (and remdesivir) was obtained from SelleckChem and MedChemExpress. The catalog number and vendor information is available in the Key Resource section. Amlodipine was obtained from Sigma (A5605).

To test drug efficacy in reducing SARS-CoV-2 infection, 10,000 A549ACE2 cells were seeded per well of a 96-well plate. Cells were treated with inhibitors at 10 μM for two hours before infection and inhibitors were maintained throughout the course of infection. Cells were infected with SARS-CoV-2 at MOI of 0.1 for 36 hours and the cells were collected for analysis via qRT-PCR or processed for immunofluorescence (N protein and quantified by Celigo). For Figure 4E, the remdesivir data was collected in an independent experiment.

Cell viability assaysCell viability following drug treatments was performed the same way as described above. Thirty-six hours post drug treatment the cells were collected and stained with LIVE-DEAD Violet (Thermo). Cell acquisition was performed using a Sony SH800S cell sorter with a 100 μm sorting chip. We used the following gating strategy: 1) We excluded the cell debris based on the forward and reverse scatter; 2) Doublets were excluded by plotting FSC height versus FCS area, 3) Dead cells were quantified using live-dead violet stain. For all samples, we recorded at least 5,000 cells that pass the gating criteria described above. Flow cytometry analyses were performed using FlowJo v10. Cell viability for A549-ACE2 treated with DMSO or 10μM amlodipine was performed 24 hours post treatment using Trypan Blue (Thermo Fisher) and automated cell counting (Nexcelom AutoT4).

Cell viability assays

Cell viability following drug treatments was performed the same way as described above. Thirty-six hours post drug treatment the cells were collected and stained with LIVE-DEAD Violet (Thermo). Cell acquisition was performed using a Sony SH800S cell sorter with a 100 μm sorting chip. We used the following gating strategy: 1) We excluded the cell debris based on the forward and reverse scatter; 2) Doublets were excluded by plotting FSC height versus FCS area, 3) Dead cells were quantified using live-dead violet stain. For all samples, we recorded at least 5,000 cells that pass the gating criteria described above. Flow cytometry analyses were performed using FlowJo v10. Cell viability for A549-ACE2 treated with DMSO or 10μM amlodipine was performed 24 hours post treatment using Trypan Blue (Thermo Fisher) and automated cell counting (Nexcelom AutoT4).

Quantitative reverse-transcription PCR (qRT-PCR) of viral RNARNA was extracted from cells grown in 96-well plates by using the RNeasy 96 Kit (QIAGEN) per the manufacturer’s instructions. RNA was reverse-transcribed and PCR amplified using SYBR Fast One-step Universal qRT-PCR Kit (KAPA/Roche). For amlodipine experiments, RNA was extracted using TRIzol Reagent (Invitrogen) and purified with the Direct-zol RNA Miniprep kit (Zymo Research) as per the manufacturer’s instructions (including the optional DNase I treatment). For amlodipine experiments, the RNA was reverse-transcribed using SuperScript Reverse Transcriptase II (Thermo Fisher) and oligo(dT) primers. The cDNA was diluted 1:20 before qRT-PCR was performed using SYBR Fast qPCR (KAPA/Roche). SARS-CoV-2 replication was assessed by using primers specific to the N mRNA (Forward 5′-CTCTTGTAGATCTGTTCTCTAAACGAAC-3′; Reverse 5′-GGTCCACCAAACGTAATGCG-3′). SARS-CoV-2 N mRNA levels were normalized to beta tubulin (Forward 5′-GCCTGGACCACAAGTTTGAC-3; Reverse 5′-TGAAATTCTGGGAGCATGAC-3′). Reactions were ran and analyzed on a Lightcycler 480 II Instrument (Roche). Relative quantification was calculated by comparing the cycle threshold (C
t) values using ΔΔCt. C
t values of subgenomic nucleocapsid transcript were normalized to C
t values of alpha-tubulin, and those values were normalized to viral transcript levels of DMSO-treated or non-targeting guide RNA-transduced control samples, and displayed as fold-reduction in viral transcript. Significance was determined using a one-way ANOVA with post hoc tests or a two-tailed unpaired Student’s t test, as appropriate.

Quantitative reverse-transcription PCR (qRT-PCR) of viral RNA

RNA was extracted from cells grown in 96-well plates by using the RNeasy 96 Kit (QIAGEN) per the manufacturer’s instructions. RNA was reverse-transcribed and PCR amplified using SYBR Fast One-step Universal qRT-PCR Kit (KAPA/Roche). For amlodipine experiments, RNA was extracted using TRIzol Reagent (Invitrogen) and purified with the Direct-zol RNA Miniprep kit (Zymo Research) as per the manufacturer’s instructions (including the optional DNase I treatment). For amlodipine experiments, the RNA was reverse-transcribed using SuperScript Reverse Transcriptase II (Thermo Fisher) and oligo(dT) primers. The cDNA was diluted 1:20 before qRT-PCR was performed using SYBR Fast qPCR (KAPA/Roche). SARS-CoV-2 replication was assessed by using primers specific to the N mRNA (Forward 5′-CTCTTGTAGATCTGTTCTCTAAACGAAC-3′; Reverse 5′-GGTCCACCAAACGTAATGCG-3′). SARS-CoV-2 N mRNA levels were normalized to beta tubulin (Forward 5′-GCCTGGACCACAAGTTTGAC-3; Reverse 5′-TGAAATTCTGGGAGCATGAC-3′). Reactions were ran and analyzed on a Lightcycler 480 II Instrument (Roche). Relative quantification was calculated by comparing the cycle threshold (C
t) values using ΔΔCt. C
t values of subgenomic nucleocapsid transcript were normalized to C
t values of alpha-tubulin, and those values were normalized to viral transcript levels of DMSO-treated or non-targeting guide RNA-transduced control samples, and displayed as fold-reduction in viral transcript. Significance was determined using a one-way ANOVA with post hoc tests or a two-tailed unpaired Student’s t test, as appropriate.

siRNA transfections and SARS-CoV-2 infectionAll of the siRNAs were ordered from Dharmacon and their catalog number can be found in the Key Resources Table. To knockdown individual genes, 10,000 A549ACE2 cells were seeded in 96-well plates and transfected with siRNAs using Lipofectamine RNAiMAX (Thermo) following the manufacturer’s protocol. Forty-eight hours later, the cells were infected with SARS-CoV-2 at an MOI of 0.1 for 36 hours. Cells were fixed with 5% formaldehyde, stained with nucleocapsid protein (clone 1C7C7, ISMMS), and visualized with AlexaFluor-568 conjugated secondary antibody (Thermo). Nuclei were stained with DAPI, and full wells were imaged with a Celigo imaging cytometer (Nexcelom Biosciences).

siRNA transfections and SARS-CoV-2 infection

All of the siRNAs were ordered from Dharmacon and their catalog number can be found in the Key Resources Table. To knockdown individual genes, 10,000 A549ACE2 cells were seeded in 96-well plates and transfected with siRNAs using Lipofectamine RNAiMAX (Thermo) following the manufacturer’s protocol. Forty-eight hours later, the cells were infected with SARS-CoV-2 at an MOI of 0.1 for 36 hours. Cells were fixed with 5% formaldehyde, stained with nucleocapsid protein (clone 1C7C7, ISMMS), and visualized with AlexaFluor-568 conjugated secondary antibody (Thermo). Nuclei were stained with DAPI, and full wells were imaged with a Celigo imaging cytometer (Nexcelom Biosciences).

Minipool CRISPR library and lentiviral productionTo generate the minipool, we combined equimolar amounts the same guide RNA vectors (cloned in pCC1) that we used for arrayed validation (see above Generation of gene-perturbed A549ACE2
lines and SARS-CoV-2 infection). In total, we combined 3 guide RNAs for each gene (either, 30 or 11 genes) plus 9 non-targeting guide RNAs. The non-targeting guide RNA plasmids were added at half the molar ratio of the other plasmids. Lentivirus was produced via transfection of the minipool with appropriate packaging plasmids (psPAX2: Addgene 12260; pMD2.G: Addgene 12259) using polyethylenimine (PEI) reagent in HEK293FT. The target A549ACE2 cells were transduced in presence of polybrene (10 μg/mL, Santa Cruz) (ECCITE-Seq experiment 1) or without (ECCITE-seq experiment 2) at very low MOI to ensure entry of a single virus per cell. The survival was below 5% after 2 days of selection with 2 μg/mL puromycin (Thermo) for both ECCITE-seq experiments. On day 10 post-transduction, the cells were collected and processed for ECCITE-seq.

Minipool CRISPR library and lentiviral production

To generate the minipool, we combined equimolar amounts the same guide RNA vectors (cloned in pCC1) that we used for arrayed validation (see above Generation of gene-perturbed A549ACE2
lines and SARS-CoV-2 infection). In total, we combined 3 guide RNAs for each gene (either, 30 or 11 genes) plus 9 non-targeting guide RNAs. The non-targeting guide RNA plasmids were added at half the molar ratio of the other plasmids. Lentivirus was produced via transfection of the minipool with appropriate packaging plasmids (psPAX2: Addgene 12260; pMD2.G: Addgene 12259) using polyethylenimine (PEI) reagent in HEK293FT. The target A549ACE2 cells were transduced in presence of polybrene (10 μg/mL, Santa Cruz) (ECCITE-Seq experiment 1) or without (ECCITE-seq experiment 2) at very low MOI to ensure entry of a single virus per cell. The survival was below 5% after 2 days of selection with 2 μg/mL puromycin (Thermo) for both ECCITE-seq experiments. On day 10 post-transduction, the cells were collected and processed for ECCITE-seq.

Expanded CRISPR-compatible Cellular Indexing of Transcriptomes and Epitopes (ECCITE-seq)To perform cell-hashing and ECCITE-seq (Mimitou et al., 2019; Stoeckius et al., 2018), we separated cells into several sub-pools and, for each pool, we resuspended 1 million cells in 100 μl staining buffer (2% BSA, 0.01% Tween in PBS). We then added 10 μl Human TruStain FcX Fc Receptor Blocking Solution (BioLegend) and incubated on ice for 10 minutes. We then added hashing antibodies (BioLegend), incubated on ice for 30 minutes, and washed cells 3 times with staining buffer. In total, we ran four lanes of a Chromium Single Cell Immune Profiling Solution v1.0 5′ kit (10x Genomics) targeting recovery of 12,000 cells per lane (superloading) in experiment 1 and 20,000 cells in experiment 2. Gene expression (mRNA), hashtags (Hashtag-derived oligos, HTOs) and guide RNA (Guide-derived oligos, GDOs) libraries were constructed (Smibert et al., 2019). Each replicate was sequenced on two NextSeq550 75-cycle high-output runs (Illumina). Sequencing reads from the mRNA library were mapped to the hg38 reference genome (Ensembl v97) using Cellranger (v3.0.1, 10x Genomics). To generate count matrices for HTO and GDO libraries, the CITE-seq-count package was used (https://github.com/Hoohm/CITE-seq-Count v1.4.2). Count matrices were then used as input into the Seurat R package (v3.2) for downstream analyses (Stuart et al., 2019).

Expanded CRISPR-compatible Cellular Indexing of Transcriptomes and Epitopes (ECCITE-seq)

To perform cell-hashing and ECCITE-seq (Mimitou et al., 2019; Stoeckius et al., 2018), we separated cells into several sub-pools and, for each pool, we resuspended 1 million cells in 100 μl staining buffer (2% BSA, 0.01% Tween in PBS). We then added 10 μl Human TruStain FcX Fc Receptor Blocking Solution (BioLegend) and incubated on ice for 10 minutes. We then added hashing antibodies (BioLegend), incubated on ice for 30 minutes, and washed cells 3 times with staining buffer. In total, we ran four lanes of a Chromium Single Cell Immune Profiling Solution v1.0 5′ kit (10x Genomics) targeting recovery of 12,000 cells per lane (superloading) in experiment 1 and 20,000 cells in experiment 2. Gene expression (mRNA), hashtags (Hashtag-derived oligos, HTOs) and guide RNA (Guide-derived oligos, GDOs) libraries were constructed (Smibert et al., 2019). Each replicate was sequenced on two NextSeq550 75-cycle high-output runs (Illumina). Sequencing reads from the mRNA library were mapped to the hg38 reference genome (Ensembl v97) using Cellranger (v3.0.1, 10x Genomics). To generate count matrices for HTO and GDO libraries, the CITE-seq-count package was used (https://github.com/Hoohm/CITE-seq-Count v1.4.2). Count matrices were then used as input into the Seurat R package (v3.2) for downstream analyses (Stuart et al., 2019).

ECCITE-seq data analysisProcessing of initial ECCITE-seq experiment with 30 target genesCells with low quality metrics, high mitochondrial gene content (> 17.5%) and low number of genes detected (≤1800) were removed. The median number of detected genes was 3309. RNA counts were log-normalized and HTO counts were normalized using the centered log-ratio transformation approach, with margin = 2 (normalizing across cells). To identify cell doublets and assign experimental conditions to cells, we used the HTOseqDemux function (Stoeckius et al., 2018). HTOseqDemux-defined cell doublets and negatives were removed from any downstream analyses.Cellular guide RNA identity for cells in the ECCITE-seq pool was assigned based on GDO unique molecular identifier (UMI) counts. We considered a guide RNA detected with ≥ 16 UMI counts (the median UMI counts per guide RNA was 503 counts, 93% of cells with ≥ 1 sgRNA). We observed that 64% of the cells had > 1 detected guide RNA (29% 1 guide RNA, 27% 2 guide RNAs, 17% 3 guide RNAs). To maximize cell recovery, we retained cells with 1 - 3 guide RNAs per cell (n = 10,265). We collapsed the guide RNA information down to target gene level (3 guide RNA per target gene) in the following way: For all cells (with 1-3 guide RNAs), we assigned a unique target gene if all detected guide RNAs targeted the same gene, or if the additional guide RNA detected was a non-targeting (NT) guide RNA (4,715 cells: 4,013 cells with exactly 1 sgRNA and 702 cells with collapsed target gene assignment). All remaining cells assigned 2 or 3 separate target genes were discarded.Some of the top-ranked genes from the CRISPR screens may be endpoint genes and thus may have subtle phenotypes difficult to detect using single-cell transcriptomics. To address this issue, we used the initial ECCITE-seq experiment (n = 4,715 cells) and grouped cells by target gene. To identify target gene perturbations that lead to systemic transcriptomic changes, we determined the number of differentially expressed genes relative to cells with NT guide RNAs. We found that under stringent filtering conditions (non-adjusted p-value < 10−5 or adjusted p-value < 0.2), only 11 target genes showed more than 5 differentially expressed genes. The second ECCITE-seq experiment was conducted using guide RNAs solely for these.Joint processing of both ECCITE-seq experimentsCells with low quality metrics, high mitochondrial gene content (> 15%) and low or high number of genes detected (≤2000 and > 5000) were removed. The median number of detected genes was 3365, with a median of 13,532 UMIs per cell. HTO counts were normalized using the centered log-ratio transformation approach, with margin = 2 (normalizing across cells). To identify cell doublets and assign experimental conditions to cells, we used the HTOseqDemux function (Stoeckius et al., 2018). HTOseqDemux-defined cell doublets and negatives were removed from any downstream analyses. Guide RNA counts were normalized similar to HTO UMI counts. Cell-wise guide RNA assignment was achieved using the MultiSeqDemux function (McGinnis et al., 2019). We only retained cells assigned with a single HTO and a single guide RNA (ECCITE-seq experiment 1: n = 1,824 cells, ECCITE-seq experiment 2: n = 17,029 cells). Cells from ECCITE-Seq experiment 1 and experiment 2 were harmonized by log-normalizing, followed by the identification of the 2000 most variable features with “vst” selection method. Next, we we used FindIntegrationAnchors and IntegrateData with the first 40 principle components.In the harmonized dataset (n = 18,853 cells), we used FindMarkers to find differentially expressed genes between non-targeting cells and cells that belonged to a targeted gene class. We used up to 100 of the most upregulated genes with adjusted p-value < 0.01 and minimal logeFC threshold > 0.1 filter as input into the heatmap in Figure 5B. The same FindMarkers analysis (but using the non-adjusted p-value < 0.01 with a maximum of the 300 most significant DEGs) was used as input into the EnrichR package to run pathway analysis using the human WikiPathways database (v. 2019) (Kuleshov et al., 2016; Slenter et al., 2018). Figure 5C shows -log10 transformed p-values for the union of all enriched pathways with an adjusted p-value < 0.01 across the 11 target genes with perturbation signatures.

ECCITE-seq data analysis

Processing of initial ECCITE-seq experiment with 30 target genesCells with low quality metrics, high mitochondrial gene content (> 17.5%) and low number of genes detected (≤1800) were removed. The median number of detected genes was 3309. RNA counts were log-normalized and HTO counts were normalized using the centered log-ratio transformation approach, with margin = 2 (normalizing across cells). To identify cell doublets and assign experimental conditions to cells, we used the HTOseqDemux function (Stoeckius et al., 2018). HTOseqDemux-defined cell doublets and negatives were removed from any downstream analyses.Cellular guide RNA identity for cells in the ECCITE-seq pool was assigned based on GDO unique molecular identifier (UMI) counts. We considered a guide RNA detected with ≥ 16 UMI counts (the median UMI counts per guide RNA was 503 counts, 93% of cells with ≥ 1 sgRNA). We observed that 64% of the cells had > 1 detected guide RNA (29% 1 guide RNA, 27% 2 guide RNAs, 17% 3 guide RNAs). To maximize cell recovery, we retained cells with 1 - 3 guide RNAs per cell (n = 10,265). We collapsed the guide RNA information down to target gene level (3 guide RNA per target gene) in the following way: For all cells (with 1-3 guide RNAs), we assigned a unique target gene if all detected guide RNAs targeted the same gene, or if the additional guide RNA detected was a non-targeting (NT) guide RNA (4,715 cells: 4,013 cells with exactly 1 sgRNA and 702 cells with collapsed target gene assignment). All remaining cells assigned 2 or 3 separate target genes were discarded.Some of the top-ranked genes from the CRISPR screens may be endpoint genes and thus may have subtle phenotypes difficult to detect using single-cell transcriptomics. To address this issue, we used the initial ECCITE-seq experiment (n = 4,715 cells) and grouped cells by target gene. To identify target gene perturbations that lead to systemic transcriptomic changes, we determined the number of differentially expressed genes relative to cells with NT guide RNAs. We found that under stringent filtering conditions (non-adjusted p-value < 10−5 or adjusted p-value < 0.2), only 11 target genes showed more than 5 differentially expressed genes. The second ECCITE-seq experiment was conducted using guide RNAs solely for these.

Processing of initial ECCITE-seq experiment with 30 target genes

Cells with low quality metrics, high mitochondrial gene content (> 17.5%) and low number of genes detected (≤1800) were removed. The median number of detected genes was 3309. RNA counts were log-normalized and HTO counts were normalized using the centered log-ratio transformation approach, with margin = 2 (normalizing across cells). To identify cell doublets and assign experimental conditions to cells, we used the HTOseqDemux function (Stoeckius et al., 2018). HTOseqDemux-defined cell doublets and negatives were removed from any downstream analyses.

Cellular guide RNA identity for cells in the ECCITE-seq pool was assigned based on GDO unique molecular identifier (UMI) counts. We considered a guide RNA detected with ≥ 16 UMI counts (the median UMI counts per guide RNA was 503 counts, 93% of cells with ≥ 1 sgRNA). We observed that 64% of the cells had > 1 detected guide RNA (29% 1 guide RNA, 27% 2 guide RNAs, 17% 3 guide RNAs). To maximize cell recovery, we retained cells with 1 - 3 guide RNAs per cell (n = 10,265). We collapsed the guide RNA information down to target gene level (3 guide RNA per target gene) in the following way: For all cells (with 1-3 guide RNAs), we assigned a unique target gene if all detected guide RNAs targeted the same gene, or if the additional guide RNA detected was a non-targeting (NT) guide RNA (4,715 cells: 4,013 cells with exactly 1 sgRNA and 702 cells with collapsed target gene assignment). All remaining cells assigned 2 or 3 separate target genes were discarded.

Some of the top-ranked genes from the CRISPR screens may be endpoint genes and thus may have subtle phenotypes difficult to detect using single-cell transcriptomics. To address this issue, we used the initial ECCITE-seq experiment (n = 4,715 cells) and grouped cells by target gene. To identify target gene perturbations that lead to systemic transcriptomic changes, we determined the number of differentially expressed genes relative to cells with NT guide RNAs. We found that under stringent filtering conditions (non-adjusted p-value < 10−5 or adjusted p-value < 0.2), only 11 target genes showed more than 5 differentially expressed genes. The second ECCITE-seq experiment was conducted using guide RNAs solely for these.

Joint processing of both ECCITE-seq experimentsCells with low quality metrics, high mitochondrial gene content (> 15%) and low or high number of genes detected (≤2000 and > 5000) were removed. The median number of detected genes was 3365, with a median of 13,532 UMIs per cell. HTO counts were normalized using the centered log-ratio transformation approach, with margin = 2 (normalizing across cells). To identify cell doublets and assign experimental conditions to cells, we used the HTOseqDemux function (Stoeckius et al., 2018). HTOseqDemux-defined cell doublets and negatives were removed from any downstream analyses. Guide RNA counts were normalized similar to HTO UMI counts. Cell-wise guide RNA assignment was achieved using the MultiSeqDemux function (McGinnis et al., 2019). We only retained cells assigned with a single HTO and a single guide RNA (ECCITE-seq experiment 1: n = 1,824 cells, ECCITE-seq experiment 2: n = 17,029 cells). Cells from ECCITE-Seq experiment 1 and experiment 2 were harmonized by log-normalizing, followed by the identification of the 2000 most variable features with “vst” selection method. Next, we we used FindIntegrationAnchors and IntegrateData with the first 40 principle components.In the harmonized dataset (n = 18,853 cells), we used FindMarkers to find differentially expressed genes between non-targeting cells and cells that belonged to a targeted gene class. We used up to 100 of the most upregulated genes with adjusted p-value < 0.01 and minimal logeFC threshold > 0.1 filter as input into the heatmap in Figure 5B. The same FindMarkers analysis (but using the non-adjusted p-value < 0.01 with a maximum of the 300 most significant DEGs) was used as input into the EnrichR package to run pathway analysis using the human WikiPathways database (v. 2019) (Kuleshov et al., 2016; Slenter et al., 2018). Figure 5C shows -log10 transformed p-values for the union of all enriched pathways with an adjusted p-value < 0.01 across the 11 target genes with perturbation signatures.

Joint processing of both ECCITE-seq experiments

Cells with low quality metrics, high mitochondrial gene content (> 15%) and low or high number of genes detected (≤2000 and > 5000) were removed. The median number of detected genes was 3365, with a median of 13,532 UMIs per cell. HTO counts were normalized using the centered log-ratio transformation approach, with margin = 2 (normalizing across cells). To identify cell doublets and assign experimental conditions to cells, we used the HTOseqDemux function (Stoeckius et al., 2018). HTOseqDemux-defined cell doublets and negatives were removed from any downstream analyses. Guide RNA counts were normalized similar to HTO UMI counts. Cell-wise guide RNA assignment was achieved using the MultiSeqDemux function (McGinnis et al., 2019). We only retained cells assigned with a single HTO and a single guide RNA (ECCITE-seq experiment 1: n = 1,824 cells, ECCITE-seq experiment 2: n = 17,029 cells). Cells from ECCITE-Seq experiment 1 and experiment 2 were harmonized by log-normalizing, followed by the identification of the 2000 most variable features with “vst” selection method. Next, we we used FindIntegrationAnchors and IntegrateData with the first 40 principle components.

In the harmonized dataset (n = 18,853 cells), we used FindMarkers to find differentially expressed genes between non-targeting cells and cells that belonged to a targeted gene class. We used up to 100 of the most upregulated genes with adjusted p-value < 0.01 and minimal logeFC threshold > 0.1 filter as input into the heatmap in Figure 5B. The same FindMarkers analysis (but using the non-adjusted p-value < 0.01 with a maximum of the 300 most significant DEGs) was used as input into the EnrichR package to run pathway analysis using the human WikiPathways database (v. 2019) (Kuleshov et al., 2016; Slenter et al., 2018). Figure 5C shows -log10 transformed p-values for the union of all enriched pathways with an adjusted p-value < 0.01 across the 11 target genes with perturbation signatures.

Plaque assayInfectious titer in supernatant of SARS-CoV-2 infections (from A549ACE2) was determined by plaque assay in Vero E6 cells. Cells were seeded in 12-well plates before infection with logarithmically diluted supernatants in a total inoculum of 100 ul, rocking plates every 10 minutes for 1 hour. Then an overlay of Minimum Essential Media with 0.12% NaHCO3, 4 mM L-glutamine, 0.2% BSA, 10 mM HEPES and 0.7% OXOID agar (Thermo) was aliquoted into wells and plates were incubated for 48 hours before fixing with 5% formaldehyde. After 24 hours, plaques were stained with crystal violet (1% crystal violet in 20% EtOH) for 15 minutes before washing wells and counting plaques.

Plaque assay

Infectious titer in supernatant of SARS-CoV-2 infections (from A549ACE2) was determined by plaque assay in Vero E6 cells. Cells were seeded in 12-well plates before infection with logarithmically diluted supernatants in a total inoculum of 100 ul, rocking plates every 10 minutes for 1 hour. Then an overlay of Minimum Essential Media with 0.12% NaHCO3, 4 mM L-glutamine, 0.2% BSA, 10 mM HEPES and 0.7% OXOID agar (Thermo) was aliquoted into wells and plates were incubated for 48 hours before fixing with 5% formaldehyde. After 24 hours, plaques were stained with crystal violet (1% crystal violet in 20% EtOH) for 15 minutes before washing wells and counting plaques.

Bulk RNA-sequencingRNA-sequencing libraries were constructed from 1ug of RNA per sample using the TruSeq Stranded mRNA Library Prep Kit (Illumina) according to the manufacturer’s instructions. Sequencing libraries were sequenced using an Illumina NextSeq 500 platform and fastq files were generated using bcl2fastq (Illumina). Alignment to the SARS-CoV-2 genome (GenBank: MN985325.1) were performed using bowtie (Langmead et al., 2009). Reads were aligned to hg19 using STAR aligner in the Basespace RNA-Seq Alignment application (Illumina) and processed using DESeq2 (Love et al., 2014). Sample distances were calculated using read counts from the regularized log transform in DESeq2. K-means clustering was perfomed using the kmeans function in R/RStudio. Enriched Reactome pathways were identified using the STRING tool (Szklarczyk et al., 2019).

Bulk RNA-sequencing

RNA-sequencing libraries were constructed from 1ug of RNA per sample using the TruSeq Stranded mRNA Library Prep Kit (Illumina) according to the manufacturer’s instructions. Sequencing libraries were sequenced using an Illumina NextSeq 500 platform and fastq files were generated using bcl2fastq (Illumina). Alignment to the SARS-CoV-2 genome (GenBank: MN985325.1) were performed using bowtie (Langmead et al., 2009). Reads were aligned to hg19 using STAR aligner in the Basespace RNA-Seq Alignment application (Illumina) and processed using DESeq2 (Love et al., 2014). Sample distances were calculated using read counts from the regularized log transform in DESeq2. K-means clustering was perfomed using the kmeans function in R/RStudio. Enriched Reactome pathways were identified using the STRING tool (Szklarczyk et al., 2019).

Measurement of cellular cholesterolCellular cholesterol was measured using the Cholesterol-Glo assay (Promega). For each cell line (CRISPR-perturbed or drug-treated), we counted 100,000 cells, washed with 1xPBS twice and resuspended in 100 μL cell lysis solution for 30 min at 37°C. Following the incubation, 10 μL of cell lysate was used to measure the cholesterol levels in 96 well plates in a final reaction volume of 100uL per well (without esterase). Luminescence was measured after incubating the plate for 1 hour at room temperature in the dark. Using the same lysis sample, the protein concentration was determined using the BCA assay (Thermo). The cholesterol levels were normalized to the respective protein concentration of each sample. A549ACE cells were incubated with 10 μM amlodipine (Sigma) or DMSO for 24 hours. Cholesterol levels were then determined as described above.

Measurement of cellular cholesterol

Cellular cholesterol was measured using the Cholesterol-Glo assay (Promega). For each cell line (CRISPR-perturbed or drug-treated), we counted 100,000 cells, washed with 1xPBS twice and resuspended in 100 μL cell lysis solution for 30 min at 37°C. Following the incubation, 10 μL of cell lysate was used to measure the cholesterol levels in 96 well plates in a final reaction volume of 100uL per well (without esterase). Luminescence was measured after incubating the plate for 1 hour at room temperature in the dark. Using the same lysis sample, the protein concentration was determined using the BCA assay (Thermo). The cholesterol levels were normalized to the respective protein concentration of each sample. A549ACE cells were incubated with 10 μM amlodipine (Sigma) or DMSO for 24 hours. Cholesterol levels were then determined as described above.

Flow cytometry of ACE2 cell surface expressionCells were harvested, counted and about 100,000 cells were washed with Dulbecco’s phosphate-buffered saline (PBS, Caisson Labs) and then stained with LIVE/DEAD Violet stain (Thermo, 34864). Following a wash with 1x PBS, all subsequent washes and antibody dilutions were performed using 1x PBS supplemented with 2% FBS. Wild-type or Cas9-perturbed A549ACE2, Caco-2 and Calu-3 cells were stained for 30 min on ice with 0.25 μg of anti-ACE2 antibody (R&D Systems, AF933) in ∼50 μL residual volume. Following two washes with 1x PBS with 2% FBS, samples were stained on ice for 20 min with 7.5 μL of anti-goat-APC secondary antibody (R&D Systems, F0108). Cell acquisition and sorting was performed using a Sony SH800S cell sorter with a 100 μm sorting chip. We used the following gating strategy: 1) We excluded the cell debris based on the forward and reverse scatter; 2) Doublets were excluded by plotting FSC height versus FCS area, 3) Dead cells were excluded by live-dead stain. We recorded at least 5,000 cells for A549ACE2 and at least 3,000 cells for Calu-3 and Caco-2 that pass the gating criteria described above. Gates to determine ACE2-APC+ cells were set based on control A549 wild-type cells or only secondary antibody stained Calu-3 and Caco-2, where the percent of ACE2 positive cells was set as < 5% (background level). Flow cytometry analyses were performed using FlowJo v10.

Flow cytometry of ACE2 cell surface expression

Cells were harvested, counted and about 100,000 cells were washed with Dulbecco’s phosphate-buffered saline (PBS, Caisson Labs) and then stained with LIVE/DEAD Violet stain (Thermo, 34864). Following a wash with 1x PBS, all subsequent washes and antibody dilutions were performed using 1x PBS supplemented with 2% FBS. Wild-type or Cas9-perturbed A549ACE2, Caco-2 and Calu-3 cells were stained for 30 min on ice with 0.25 μg of anti-ACE2 antibody (R&D Systems, AF933) in ∼50 μL residual volume. Following two washes with 1x PBS with 2% FBS, samples were stained on ice for 20 min with 7.5 μL of anti-goat-APC secondary antibody (R&D Systems, F0108). Cell acquisition and sorting was performed using a Sony SH800S cell sorter with a 100 μm sorting chip. We used the following gating strategy: 1) We excluded the cell debris based on the forward and reverse scatter; 2) Doublets were excluded by plotting FSC height versus FCS area, 3) Dead cells were excluded by live-dead stain. We recorded at least 5,000 cells for A549ACE2 and at least 3,000 cells for Calu-3 and Caco-2 that pass the gating criteria described above. Gates to determine ACE2-APC+ cells were set based on control A549 wild-type cells or only secondary antibody stained Calu-3 and Caco-2, where the percent of ACE2 positive cells was set as < 5% (background level). Flow cytometry analyses were performed using FlowJo v10.

Immunofluorescence of ACE2 and endo/lysosomal markersA549ACE2 Cas9-transduced (specific gene or non-targeting) cell lines were seeded on poly-D-lysine-coated coverslips (Electron Microscopy Sciences) 24 hours before they were fixed with 4% formaldehyde (Sigma) diluted in 1x PBS (Caisson Labs) (Daniloski et al., 2020). Following 3 washes with 1x PBS, cells were blocked with 1x PBS with 2% BSA for 30 min at room temperature. Cells were stained with mouse anti-2A antibody diluted at 1:250 (clone 3H4, Millipore Sigma, MABS2005) to recognize ACE2-2A, and rabbit anti-EEA1 diluted at 1:100 (Thermo, MA5-14794). To detect ACE2 in Caco-2 cells, goat anti-ACE2 (R&D Systems, AF933) was used at 1.25 μg/mL. Cell were incubated with primary antibodies for 3 hours at room temperature in a moisturized chamber. Coverslips were then washed 5 times with 1x PBS with 2% BSA and then incubated with Alexa-conjugated secondary antibodies (Jackson ImmunoResearch) diluted at 1:1000 for 45 min at room temperature. Coverslips were then washed 5 times with 1x PBS with 2%BSA, and DAPI (Sigma) was added in the fourth wash at a concentration of 0.1 μg/mL. To stain the lysosomes, LysoTracker DeepRed (Thermo) was diluted to 70 nM in culture media and was added to actively growing cells on coverslips and were incubated for 1 hour at 37°C. Following incubation with LysoTracker, cells were processed for immunofluorescence as described above. Confocal images were acquired on a Zeiss LSM 780 with a 20x/0.8 Plan-Apochromat objective or a 63x/1.4 Plan-Apochromat objective (Zeiss). Epifluorescence images were acquired using the same system; both fitted with a Zeiss Axiocam 506 mono (Zeiss). Images were processed using Zen Black 2012 (Zeiss) and FIJI 2.1.0; Java 1.8.0-202 (Schindelin et al., 2012). Linear histogram adjustments were applied uniformly within experiments for clarity of presentation.ACE2 localization was determined on images taken on a Zeiss LSM 780 with a 20x/0.8 Plan-Apochromat objective using Zen Black (Zeiss). The A549ACE2 polyclonal cell line had both ACE2 positive and negative cells. Only ACE2 positive cells were then manually scored if they had a distinct accumulation of cytoplasmic vesicles-like hollow structures, compared to the non-targeting (NT) control. In the NT control, the ACE2 staining was seen on the plasma membrane and diffuse in the cytoplasm. The scoring was performed on 2 biological replicates (that is, two separate non-targeting controls and two separate Rab7A knockout lines engineered using 4 separate Cas9 guide RNAs). We scored ∼430 cells per replicate. To quantify the size of ACE2 foci, confocal microscopy images were thresholded using FIJI’s local-threshold (Phansalkar radius = 15) and then a watershed transform was applied. Foci were counted and measured using FIJI’s Analyze Particles function.

Immunofluorescence of ACE2 and endo/lysosomal markers

A549ACE2 Cas9-transduced (specific gene or non-targeting) cell lines were seeded on poly-D-lysine-coated coverslips (Electron Microscopy Sciences) 24 hours before they were fixed with 4% formaldehyde (Sigma) diluted in 1x PBS (Caisson Labs) (Daniloski et al., 2020). Following 3 washes with 1x PBS, cells were blocked with 1x PBS with 2% BSA for 30 min at room temperature. Cells were stained with mouse anti-2A antibody diluted at 1:250 (clone 3H4, Millipore Sigma, MABS2005) to recognize ACE2-2A, and rabbit anti-EEA1 diluted at 1:100 (Thermo, MA5-14794). To detect ACE2 in Caco-2 cells, goat anti-ACE2 (R&D Systems, AF933) was used at 1.25 μg/mL. Cell were incubated with primary antibodies for 3 hours at room temperature in a moisturized chamber. Coverslips were then washed 5 times with 1x PBS with 2% BSA and then incubated with Alexa-conjugated secondary antibodies (Jackson ImmunoResearch) diluted at 1:1000 for 45 min at room temperature. Coverslips were then washed 5 times with 1x PBS with 2%BSA, and DAPI (Sigma) was added in the fourth wash at a concentration of 0.1 μg/mL. To stain the lysosomes, LysoTracker DeepRed (Thermo) was diluted to 70 nM in culture media and was added to actively growing cells on coverslips and were incubated for 1 hour at 37°C. Following incubation with LysoTracker, cells were processed for immunofluorescence as described above. Confocal images were acquired on a Zeiss LSM 780 with a 20x/0.8 Plan-Apochromat objective or a 63x/1.4 Plan-Apochromat objective (Zeiss). Epifluorescence images were acquired using the same system; both fitted with a Zeiss Axiocam 506 mono (Zeiss). Images were processed using Zen Black 2012 (Zeiss) and FIJI 2.1.0; Java 1.8.0-202 (Schindelin et al., 2012). Linear histogram adjustments were applied uniformly within experiments for clarity of presentation.

ACE2 localization was determined on images taken on a Zeiss LSM 780 with a 20x/0.8 Plan-Apochromat objective using Zen Black (Zeiss). The A549ACE2 polyclonal cell line had both ACE2 positive and negative cells. Only ACE2 positive cells were then manually scored if they had a distinct accumulation of cytoplasmic vesicles-like hollow structures, compared to the non-targeting (NT) control. In the NT control, the ACE2 staining was seen on the plasma membrane and diffuse in the cytoplasm. The scoring was performed on 2 biological replicates (that is, two separate non-targeting controls and two separate Rab7A knockout lines engineered using 4 separate Cas9 guide RNAs). We scored ∼430 cells per replicate. To quantify the size of ACE2 foci, confocal microscopy images were thresholded using FIJI’s local-threshold (Phansalkar radius = 15) and then a watershed transform was applied. Foci were counted and measured using FIJI’s Analyze Particles function.

Western blotsA549ACE2 or Huh7.5ACE2 cells were collected, washed with 1x PBS and lysed with TNE buffer (10 mM Tris-HCl, pH 7.4, 150 mM NaCl, 1mM EDTA, 1% Nonidet P-40) in presence of protease inhibitor cocktail (Bimake B14001) for 1 hour on ice. Cells lysates were spun for 10 min at 10,000 g, and protein concentration was determined with the BCA assay (Thermo). Equal amounts of cell lysates (20 μg) were denatured in Tris-Glycine SDS Sample buffer (Thermo LC2676), and loaded on a Novex 4 - 20% Tris-Glycine gel (Thermo XP04205BOX). PageRuler pre-stained protein ladder (Thermo 26616) was used to determine the protein size. The gel was run in 1x Tris-Glycine-SDS buffer (IBI Scientific IBI01160) for about 120 min at 120V. Proteins were transferred on a nitrocellulose membrane (BioRad 1620112) in presence of prechilled 1x Tris-Glycine transfer buffer (FisherSci LC3675) supplemented with 20% methanol for 100 min at 100V. Immunoblots were blocked with 5% skim milk dissolved in 1x PBS with 1% Tween-20 (PBST) and incubated overnight at 4°C separately with the following primary antibodies: rabbit anti-RAB7A (0.1 μg/mL, NovusBio, NBP1-87174), rabbit anti-GAPDH (0.1 μg/mL, Cell Signaling, 2118S), rabbit anti-CCDC22 (0.34 μg/mL, Proteintech, 16636-1-AP), rabbit anti-ATP6V1A (0.46 μg/mL, Proteintech, 17115-1-AP), rabbit anti-ACE2 (0.5 μg/mL, Invitrogen, MA5-32307), and mouse anti-beta tubulin (0.5 μg/mL, Invitrogen, 32-2600). Following the primary antibody, the blots were incubated with IRDye 680RD donkey anti-rabbit (0.2 μg/mL, LI-COR 926-68073) or with IRDye 800CW donkey anti-mouse (0.2 μg/mL, LI-COR 926-32212). The blots were imaged using Odyssey CLx (LI-COR).

Western blots

A549ACE2 or Huh7.5ACE2 cells were collected, washed with 1x PBS and lysed with TNE buffer (10 mM Tris-HCl, pH 7.4, 150 mM NaCl, 1mM EDTA, 1% Nonidet P-40) in presence of protease inhibitor cocktail (Bimake B14001) for 1 hour on ice. Cells lysates were spun for 10 min at 10,000 g, and protein concentration was determined with the BCA assay (Thermo). Equal amounts of cell lysates (20 μg) were denatured in Tris-Glycine SDS Sample buffer (Thermo LC2676), and loaded on a Novex 4 - 20% Tris-Glycine gel (Thermo XP04205BOX). PageRuler pre-stained protein ladder (Thermo 26616) was used to determine the protein size. The gel was run in 1x Tris-Glycine-SDS buffer (IBI Scientific IBI01160) for about 120 min at 120V. Proteins were transferred on a nitrocellulose membrane (BioRad 1620112) in presence of prechilled 1x Tris-Glycine transfer buffer (FisherSci LC3675) supplemented with 20% methanol for 100 min at 100V. Immunoblots were blocked with 5% skim milk dissolved in 1x PBS with 1% Tween-20 (PBST) and incubated overnight at 4°C separately with the following primary antibodies: rabbit anti-RAB7A (0.1 μg/mL, NovusBio, NBP1-87174), rabbit anti-GAPDH (0.1 μg/mL, Cell Signaling, 2118S), rabbit anti-CCDC22 (0.34 μg/mL, Proteintech, 16636-1-AP), rabbit anti-ATP6V1A (0.46 μg/mL, Proteintech, 17115-1-AP), rabbit anti-ACE2 (0.5 μg/mL, Invitrogen, MA5-32307), and mouse anti-beta tubulin (0.5 μg/mL, Invitrogen, 32-2600). Following the primary antibody, the blots were incubated with IRDye 680RD donkey anti-rabbit (0.2 μg/mL, LI-COR 926-68073) or with IRDye 800CW donkey anti-mouse (0.2 μg/mL, LI-COR 926-32212). The blots were imaged using Odyssey CLx (LI-COR).

Quantification and Statistical AnalysisStatistical analysisStatistical parameters and details are reported in each figure legend. Generally, experiments were repeated with at least three biological replicates. Each plot includes points for individual biological replicates and mean ± s.e.m. error bars unless otherwise specified. In general, biological replicates consist of replicate viral transductions (CRISPR), replicate RNA transfections (siRNA), or replicate drug treatments (small molecule inhibitors) and are specified in each figure legend. For imaging experiments, the number of cells scored per biological replicate is given in the figure legend.When assessing significance of genetic perturbations and/or inhibitor treatments, we employed first a one-way or two-way ANOVA to assess significance and, when significant, post hoc comparisons were performed using the false-discovery rate method of Benjamini-Krieger-Yekutieli to correct for multiple comparisons. Before applying ANOVA, we first verified the equality of group variances using the Brown-Forsythe test. Details for all ANOVA and post hoc multiple comparisons are reported in Table S6; t tests are reported in the respective figure legends.Throughout the manuscript, we use r to denote the Pearson correlation and r
s to denote the Spearman correlation; all reported correlations were significant using the cor.test function in R. For the BioID interactome analysis, we derived an empirical p value by resampling from a null distribution (n = 10,000 gene shuffling iterations). Statistical analyses were performed in GraphPad Prism 8 and RStudio 1.2.5019.

Quantification and Statistical Analysis

Statistical analysisStatistical parameters and details are reported in each figure legend. Generally, experiments were repeated with at least three biological replicates. Each plot includes points for individual biological replicates and mean ± s.e.m. error bars unless otherwise specified. In general, biological replicates consist of replicate viral transductions (CRISPR), replicate RNA transfections (siRNA), or replicate drug treatments (small molecule inhibitors) and are specified in each figure legend. For imaging experiments, the number of cells scored per biological replicate is given in the figure legend.When assessing significance of genetic perturbations and/or inhibitor treatments, we employed first a one-way or two-way ANOVA to assess significance and, when significant, post hoc comparisons were performed using the false-discovery rate method of Benjamini-Krieger-Yekutieli to correct for multiple comparisons. Before applying ANOVA, we first verified the equality of group variances using the Brown-Forsythe test. Details for all ANOVA and post hoc multiple comparisons are reported in Table S6; t tests are reported in the respective figure legends.Throughout the manuscript, we use r to denote the Pearson correlation and r
s to denote the Spearman correlation; all reported correlations were significant using the cor.test function in R. For the BioID interactome analysis, we derived an empirical p value by resampling from a null distribution (n = 10,000 gene shuffling iterations). Statistical analyses were performed in GraphPad Prism 8 and RStudio 1.2.5019.

Statistical analysis

Statistical parameters and details are reported in each figure legend. Generally, experiments were repeated with at least three biological replicates. Each plot includes points for individual biological replicates and mean ± s.e.m. error bars unless otherwise specified. In general, biological replicates consist of replicate viral transductions (CRISPR), replicate RNA transfections (siRNA), or replicate drug treatments (small molecule inhibitors) and are specified in each figure legend. For imaging experiments, the number of cells scored per biological replicate is given in the figure legend.

When assessing significance of genetic perturbations and/or inhibitor treatments, we employed first a one-way or two-way ANOVA to assess significance and, when significant, post hoc comparisons were performed using the false-discovery rate method of Benjamini-Krieger-Yekutieli to correct for multiple comparisons. Before applying ANOVA, we first verified the equality of group variances using the Brown-Forsythe test. Details for all ANOVA and post hoc multiple comparisons are reported in Table S6; t tests are reported in the respective figure legends.

Throughout the manuscript, we use r to denote the Pearson correlation and r
s to denote the Spearman correlation; all reported correlations were significant using the cor.test function in R. For the BioID interactome analysis, we derived an empirical p value by resampling from a null distribution (n = 10,000 gene shuffling iterations). Statistical analyses were performed in GraphPad Prism 8 and RStudio 1.2.5019.
